# Evidence for association of two variants of the nociceptin/orphanin FQ receptor gene OPRL1 with vulnerability to develop opiate addiction in Caucasians

## Metadata
**Authors:** Judith A Briant, David A Nielsen, Dmitri Proudnikov, Douglas Londono, Ann Ho, Jurg Ott, Mary Jeanne Kreek
**Journal:** Psychiatric genetics
**Date:** 2010 Apr
**DOI:** [10.1097/YPG.0b013e32833511f6](https://doi.org/10.1097/YPG.0b013e32833511f6)
**PMID:** 20032820
**PMCID:** PMC3832186
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3832186/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC3832186/pdf/nihms213330.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC3832186/pdf/nihms213330.pdf)

## Abstract

**Objectives:** 
The OPRL1 gene encodes the nociceptin/orphanin FQ receptor (NOP-R), which plays a role in regulating tolerance and behavioral responses to morphine. However, there is limited information on whether variants of OPRL1 are associated with vulnerability to develop opiate addiction. In this study, we examined five variants of OPRL1 and their role in determining vulnerability to develop opiate addiction.

**Methods:** 
We recruited 447 subjects: 271 former severe heroin addicts and 176 healthy controls. Using a 5′-fluorogenic exonuclease assay (TaqMan®), we genotyped subjects at five variants in OPRL1. It was then determined whether there was a significant association of allele, genotype, or haplotype frequency with vulnerability to develop opiate addiction.

**Results:** 
When the cohort was stratified by ethnicity, we found that, in Caucasians but not in African Americans or Hispanics, the allele frequency of rs6090041 and rs6090043 were significantly associated point-wise with opiate addiction (P = 0.03 and 0.04, respectively). Of the haplotypes formed by these two variants, one haplotype was found to be associated with protection from developing opiate addiction in both African Americans (point-wise P = 0.04) and Caucasians (point-wise P = 0.04), and another haplotype with vulnerability to develop opiate addiction in Caucasians only (P = 0.020).

**Conclusions:** 
This study provides evidence for an association of two variants of the OPRL1 gene, rs6090041 and rs6090043, with vulnerability to develop opiate addiction, suggesting a role for NOP-R in the development of opiate addiction.

Keywords: OPRL1, ORL1, nociceptin/orphanin FQ receptor, polymorphism, variant, opiate addiction, heroin

### Objectives

The *OPRL1* gene encodes the nociceptin/orphanin FQ receptor (NOP-R), which plays a role in regulating tolerance and behavioral responses to morphine. However, there is limited information on whether variants of *OPRL1* are associated with vulnerability to develop opiate addiction. In this study, we examined five variants of *OPRL1* and their role in determining vulnerability to develop opiate addiction.

### Methods

We recruited 447 subjects: 271 former severe heroin addicts and 176 healthy controls. Using a 5′-fluorogenic exonuclease assay (TaqMan®), we genotyped subjects at five variants in *OPRL1*. It was then determined whether there was a significant association of allele, genotype, or haplotype frequency with vulnerability to develop opiate addiction.

### Results

When the cohort was stratified by ethnicity, we found that, in Caucasians but not in African Americans or Hispanics, the allele frequency of rs6090041 and rs6090043 were significantly associated point-wise with opiate addiction (*P =* 0.03 and 0.04, respectively). Of the haplotypes formed by these two variants, one haplotype was found to be associated with protection from developing opiate addiction in both African Americans (point-wise *P* = 0.04) and Caucasians (point-wise *P* = 0.04), and another haplotype with vulnerability to develop opiate addiction in Caucasians only (*P* = 0.020).

### Conclusions

This study provides evidence for an association of two variants of the *OPRL1* gene, rs6090041 and rs6090043, with vulnerability to develop opiate addiction, suggesting a role for NOP-R in the development of opiate addiction.

## Introduction

The development of opiate addiction is a complex process that is influenced by a variety of factors ([Kreek et al., 2004](#R16); [Kreek et al., 2005a](#R14), [Kreek et al., 2005b](#R17)). It has been established that genetic, as well as social, psychological, and environmental factors, play a role in the development of opiate addiction. For example, family and twin studies have found that the genetic variance contributing to an individual’s vulnerability to develop heroin addiction is estimated to be up to 0.54 ([Tsuang et al., 1998](#R39) and [Kendler et al., 2003](#R11)). However, many of the specific gene variants that are responsible for influencing an individual’s genetic vulnerability to develop addiction have yet to be identified.

The endogenous opioid system is involved in transducing the effects of opiates, such as heroin and morphine ([Kreek, 1996](#R13)). Opioids exert their effects, in part, by binding opioid receptors on the surface of GABA-ergic neurons in the mesocorticolimbic system. GABA-ergic neurons exert a tonic inhibition on the dopaminergic neurons, and the binding of opiates causes inhibition of the GABA-ergic neurons and disinhibition of the dopaminergic system, resulting in an increase in synaptic dopamine levels in the terminal fields of the mesocorticolimbic dopamine system ([Johnson and North, 1992](#R10)). The mesocorticolimbic system projects from the ventral tegmental area to the nucleus accumbens, olfactory tubercle, cingulate cortex, and amygdala. The increase in dopaminergic tone due to opiate inhibition contributes to the rewarding properties of opiates. Blocking opioid receptors in the nucleus accumbens or ventral tegmental area with opiate antagonists ablates the rewarding properties of opiates (reviewed in [Kreek and Koob, 1998](#R15)).

One component related to the endogenous opioid system is the nociceptin/orphanin FQ receptor (NOP-R), encoded by the *OPRL1* gene. The ligand of NOP-R is nociceptin/orphanin FQ (N/OFQ). Both NOP-R and N/OFQ are expressed at various levels throughout brain tissues ([Mollereau and Mouledous, 2000](#R24)). NOP-R is a G protein-coupled receptor that, when activated by N/OFQ, signals via inhibition of adenylyl cyclase, decreases cAMP formation ([Wu et al., 1997](#R45)). The N/OFQ peptide is similar in structure to the endogenous opioid dynorphin ([Houtani et al., 1996](#R7)), and its processing has been reported to be similar to that of dynorphin ([Yu et al., 1996](#R47)). Functionally, N/OFQ has been reported to inhibit the rewarding effect of addictive drugs ([Ciccocioppo et al., 2000](#R5); [Cowen and Lawrence, 2006](#R6); [Reinscheid et al., 2006](#R31)). When N/OFQ is injected into mice, there is a dose-dependent inhibition of mesolimbic dopamine release ([Murphy et al., 1996](#R27)) with a concomitant loss of morphine conditioned place preference ([Murphy et al., 1999](#R26)). N/OFQ inhibits dopaminergic neurons in the ventral tegmental area, potentially via an increase in GABA levels, which in turn decreases dopamine transmission to the nucleus accumbens ([Murphy and Maidment, 1999](#R28)). It has been reported that there is conditioned place aversion to naloxone in mice following administration of N/OFQ ([Sakoori and Murphy, 2004](#R33)). In two studies, nociceptin was found to have no detectable effect on heroin self-administration in rats ([Walker et al., 1998](#R41); [Martin-Fardon et al., 2000](#R20)). In most studies, however, the effects of N/OFQ are similar to those of dynorphin, which decreases basal and drug of abuse-induced increases in dopamine ([Zhang et al., 2004](#R48)). This has been further supported by a study in mice lacking the nociceptin receptor gene, in which a 50% reduction in the development of morphine tolerance was observed ([Ueda et al., 1997](#R40)). Thus, these studies indicate that N/OFQ and its receptor, NOP-R, block some of the rewarding properties of opiates and are involved in the development of opioid tolerance.

Given the role of NOP-R in the endogenous opioid system, we have evaluated whether variants of *OPRL1* are associated with vulnerability to develop opiate addiction. In this study, we evaluated five variants in the coding, the intronic, or upstream regions of the human *OPRL1* gene to examine whether there are significant associations of allele, genotype, or haplotype frequency with vulnerability to develop opiate addiction.

## Methods

### Study Subjects

Subjects were consecutive unrelated volunteers, who were a part of a study on the genetics of drug addiction at the Laboratory of the Biology of Addictive Diseases at The Rockefeller University. The subjects were recruited from February 7, 1995 to January 20, 2000 from methadone maintenance treatment programs in New York City and through advertisements in local media. There were 447 subjects who met the inclusion criteria defined below: former severe heroin addicts stabilized in methadone maintenance pharmacotherapy or healthy controls ([Table 1](#T1)).

### Table 1.

|   | Ethnicity |  |  |  |  |  |  |  |  |  | Total |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| African American |  | Caucasian |  | Hispanic |  | Asian |  | Mixed/Other |  |  |  |  |
|   |  |  |  |  |  |  |  |  |  |  |  |  |
| Classification | N | Male (%) | N | Male (%) | N | Male (%) | N | Male (%) | N | Male (%) | N | Male (%) |
|   |   |  |   |  |   |  |   |  |   |  |   |  |
| Former Severe |   |   |   |   |   |   |   |   |   |   |   |   |
| Heroin Addicts | 59 | 30 (51) | 100 | 65 (65) | 94 | 51 (54) | 1 | 0 (0) | 17 | 6 (35) | 271 | 152 (56) |
| Controls | 44 | 26 (59) | 76 | 41 (54) | 29 | 11 (38) | 20 | 11 (55) | 7 | 6 (86) | 176 | 95 (54) |
|   |   |  |   |  |   |  |   |  |   |  |   |  |
| Total | 103 | 56 (54) | 176 | 106 (60) | 123 | 62 (50) | 21 | 11 (52) | 24 | 12 (50) | 447 | 247 (55) |

Table 1 Caption: Primary ascertainment and classification of subjects

Each subject was evaluated by a psychiatrist, clinical psychologist, or research nurse and signed an informed consent form approved by The Rockefeller University Hospital Institutional Review Board. This form gives consent specifically for genetic studies. The addiction history of each subject was evaluated using the Addiction Severity Index (ASI) ([McLellen et al., 1992](#R21)). Urine samples were obtained and screened for drugs of abuse, and blood samples were taken by venipuncture. Subjects were excluded if they had an ongoing psychotic illness.

All of the former severe heroin addicts met U.S. Federal criteria for methadone maintenance treatment, including demonstration of drug-seeking behavior, self-administration of multiple doses of opiate drugs daily for one year or more, and the acquisition of dependence and tolerance ([Rettig and Yarmolinsky, 1995](#R32)). Control subjects were excluded if they used an illicit drug (except cannabis) within the last 30 days, had any instance of alcohol intoxication in the last 30 days, or had a history of greater than six months of illicit drug or alcohol intoxication more than three times per week as defined by ASI criteria. Control subjects were excluded if they had used cannabis three or more times per week for four years or more.

Each subject provided information about place of birth and the ethnic/cultural background of him/herself, parents, grandparents, and great grandparents. Individuals were classified into five groups: African American, Asian, Caucasian, Hispanic, or Other/Mixed.

The Asian group was not included in the association studies since only one subject was a former severe heroin addict. The 24 subjects who identified themselves as in the mixed/other group were not included in the association studies, to avoid possible confounding factors due to population admixture.

### Genotyping

We decided *a priori* to study five variants of *OPRL1*. Two variants, rs2229205 and rs2295448, were initially selected using the UCSC Genome Browser ([http://genome.ucsc.edu](http://genome.ucsc.edu), Build 36.1, May 2008). rs2229205 was chosen due to its location in the coding region of *OPRL1*. Using the Tagger software ([http://www.broad.mit.edu/mpg/tagger](http://www.broad.mit.edu/mpg/tagger)), seven tagged variants were identified from the CEU (European) HapMap analysis panel (r^2^ = 1.00, minimum allele frequency ≥ 5%) ([Bakker et al., 2005](#R1)). The allele frequencies of these were examined using HapMap data found at dbSNP ([http://www.ncbi.nlm.nih.gov/SNP](http://www.ncbi.nlm.nih.gov/SNP)) (Build 127), and three variants, rs6011280, rs6090041, and rs6090043, were chosen based on allele frequencies greater than 0.05 in the CEU, HCB (Asian), JPT (Asian), and YRI (Sub-Saharan African) populations. The variants in intron 1 (rs6090041 and rs6090043) were evaluated for transcription factor binding sites using TESS: Transcription Element Search System ([Schug and Overton, 1998](#R34)).

A 5′-fluorogenic exonuclease assay (TaqMan®, Applied Biosystems, Foster City, CA) was used to determine genotypes at the five variants. TaqMan® probes were designed for variants rs2229205 and rs2295448 using Primer Express software (Applied Biosystems) and synthesized by Applied Biosystems. We used the following oligonucleotides for genotyping variant rs2229205: 5′-TGTTCACCAGCACCTTCACC-3′ (forward), 5′-ACACCGACAACAGAGGCCAG-3′ (reverse), 5′-AGGCTGTCAATGTG-3′ (FAM-labeled), and 5′-AGGCTGTTAATGTGG-3′ (VIC-labeled), and for genotyping variant 2295448: 5′-GGCTCCCGGGTGGCT-3′ (forward), 5′-GCCGGATCATGAGGCTGTAG-3′ (reverse), 5′-AGGGCACGTGGGC-3′ (FAM-labeled), and 5′-AGGGCACATGGGC-3′ (VIC-labeled). Pre-designed TaqMan® primer-probe sets (Applied Biosystems) were used for genotyping variants rs6011280 (Assay ID C_69507_10), rs6090041 (Assay ID C_29614941_10), and rs6090043 (Assay ID C_26713862_10). PCR amplification was performed in duplicate using Platinum® quantitative PCR SuperMix-UDG (Invitrogen, Carlsbad, CA) on a GeneAmp® PCR system 9700 and the dual 384-well sample block module (Applied Biosystems). Samples were amplified at 50°C for 2 min, 95° C for 10 min, and then 50 cycles of 95°C for 15s and 60°C for 1 min. The amplification products were analyzed using an Applied Biosystems Prism® 7900 sequence detection system and SDS 2.2 software (Applied Biosystems) by an investigator who was unaware of the subjects’ diagnoses.

### Statistical Analysis

The frequency of marker alleles and genotypes in cases and controls were compared using the two-sided Fisher’s exact test. Odds ratios (OR) with 95% confidence intervals were determined. A *P* ≤ 0.05 was considered significant point-wise prior to correcting for multiple comparisons.

To account for any allelic correlation at the locus level, the exact test of Hardy-Weinberg proportions was performed as implemented in the *genetics* library of R v2.4.1 ([Warnes, 2003](#R43)). A *P* ≤ 0.05/15 (five markers on three ethnicities) was considered significant experiment-wise (correcting for multiple testing).

The G-test of independence was applied to the distribution of allele frequencies in the control samples across the different ethnicities to test for hidden structure ([Sokal et al., 1994](#R36)). This is an asymptotically distributed chi-square test with the same number of degrees of freedom as the corresponding chi-square test of independence.

Score tests of association with disease as well as frequencies were computed for haplotypes with a special software [haplo.score from the R package haplo.stats v1.4.3 ([Sinnwell and Schaid, 2009](#R35))]. This program provides a global association test *P* value as well as *P* values resulting from evaluating each haplotype against the rest.

To adjust for multiple testing, single-marker and haplotype results were examined using a special software [R package *qvalue* v1.1 ([Storey and Tibshirani, 2003](#R37))]. This package takes a given set of *P* values and estimates the minimum false discovery rate (FDR) ([Benjamini and Hochberg, 1995](#R3)) that is incurred when calling a particular test significant, which is the q-value of the test ([Storey, 2003](#R38)). An FDR of 0.05 was used as a cut-off for experiment-wise significance, and the proportion of true null hypotheses was estimated by the bootstrap method.

### Pairwise Linkage Disequilibrium (LD)

The pattern of pairwise LD between variants was measured by the standardized disequilibrium value (D′) and r^2^ ([Lewontin, 1964](#R18)) as implemented in Haploview v3.32 ([Barrett et al., 2005](#R2)). D′ varies from −1 to 1. A value of |D′| of 1 implies complete LD, a value of 0 implies linkage equilibrium, and intermediate levels of D′ suggest recombination between the loci or recurrent mutation. Haplotype blocks were selected using the 4-gamete rule definition ([Wang et al., 2002](#R42)) and were reconstructed using the accelerated expectation-maximization (EM) algorithm implemented in Haploview (which is similar to the partition-ligation method) ([Qin et al., 2002](#R29)).

## Results

### Demographics

In this study, there were 447 subjects: 247 (55%) were male; 103 (23%) were African American, 176 (39%) Caucasian, 123 (28%) Hispanic, 21 (5%) Asian, and 24 (5%) Mixed/Other ([Table 1](#T1)). In total, 271 (61%) were methadone-stabilized former severe heroin addicts and 176 (39%) were healthy controls.

### Variant Analysis

The 447 subjects were genotyped at five variants (rs6011280, rs6090041, rs6090043, rs2229205, and rs2295448), the first of which is found 6,348 nucleotides upstream of the *OPRL1* transcription start site (in the 3′ untranslated region of *RGS19*, a gene that encodes a regulator of G-protein signaling ([Ito et al., 2000](#R9))), and the remaining four are found in intron 1, intron 1, exon 4, and intron 4, respectively, of the NOP-R gene *OPRL1* ([Figure 1](#F1)). Variant rs2229205 is located in the coding region of exon 4 but is a synonymous substitution that does not alter amino acid sequence. As is standard convention in most genetics papers, the variant nucleotide indicated is in the sense strand of *OPRL1*.

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a0bd/3832186/1cfea5c567cd/nihms213330f1.jpg)

Organization of the human OPRL1 gene. The location of each variant used in this study is shown with its nucleotide position on chromosome 20 (NCBI Build 36.1). Coding region is shown as and non-coding region as white. ATG transcription start site is indicated.

Genotype frequencies in the control groups were examined in order to determine whether they deviated significantly from Hardy-Weinberg equilibrium (HWE). None of the variants significantly deviated from HWE (*P* < 0.05) in any of the four major ethnic groups: African Americans, Caucasians, Hispanics, and Asians. The allele frequencies of the five variants were evaluated among ethnicities. Allele frequencies differed significantly among ethnicities for variant rs6090041 only (*P =* 0.0004, [Table 2](#T2)). Since one of the five variants was significantly different in allele frequencies among the ethnicities, case-control comparisons were done after stratification by ethnicity.

### Table 2.

| Variant | Allele | African American | Caucasian | Hispanic | Asian | LR χ2 a | P value |
| --- | --- | --- | --- | --- | --- | --- | --- |
|   |  |  |  |  |  |  |  |
|   |   | N (frequency) | N (frequency) | N (frequency) | N (frequency) |   |   |
| rs6011280 | C | 32 (0.41) | 45 (0.36) | 18 (0.39) | 8 (0.20) | 5.91 | 0.116 |
| A | 46 (0.59)[0.62]c | 79 (0.64)[0.72] | 28 (0.61) | 32 (0.80)[0.73] |   |   |  |
| rs6090041 | G | 37 (0.47) | 28 (0.23) | 14 (0.30) | 6 (0.15) | 18.33 | 0.0004b |
| A | 41 (0.53)[0.49] | 94 (0.77)[0.59] | 32 (0.70) | 34 (0.85)[0.69] |   |   |  |
| rs6090043 | C | 34 (0.44) | 51 (0.42) | 21 (0.46) | 15 (0.38) | 0.66 | 0.883 |
| T | 44 (0.56)[0.52] | 71 (0.58)[0.51] | 25 (0.54) | 25 (0.62)[0.63] |   |   |  |
| rs2229205 | T | 6 (0.07) | 28 (0.19) | 8 (0.14) | 4 (0.10) | 7.75 | 0.051 |
| C | 80 (0.93)[0.87] | 118 (0.81)[0.86] | 50 (0.86) | 36 (0.90)[0.87] |   |   |  |
| rs2295448 | T | 12 (0.14) | 18 (0.12) | 4 (0.07) | 4 (0.10) | 2.17 | 0.537 |
| C | 72 (0.86) | 128 (0.88) | 54 (0.93) | 36 (0.90) |   |   |  |

Table 2 Caption: Allele frequencies in controls by ethnicity

### Association of Allele, Genotype, and Haplotype Frequency with Opiate Addiction

The five *OPRL1* variants were analyzed for association with vulnerability to develop opiate addiction separately in each of the three major ethnicities in which there were adequate sample sizes (African Americans, Caucasians, and Hispanics, [Table 3](#T3)). After stratifying by ethnicity, the allele frequencies of rs6090041 and of rs6090043 were found to be associated significantly point-wise with opiate addiction in Caucasians (*P =* 0.03, [Table 3B](#T3); *P =* 0.04, [Table 3C](#T3), respectively). No experiment-wise significance was found after correction for multiple testing. At the genotype level ([Table 3](#T3)), no significant association of *OPRL1* variants with opiate addiction was found in any of the ethnic groups tested.

### Table 3.

| A. Variant rs6011280 |  |  |  |  |  |  |  |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
|   |   |   |  |  |  |   |  |  |  |
|   |   | Genotype |  |  |  | Allele Frequency |  |  |  |
|   |  |  |  |  |  |  |  |  |  |
| Ethnicity | Group | AA | CA | CC | P value | A | C | ORa (95% CI) | P value |
| African American | Opiate | 21 | 25 | 7 | 0.84 | 0.63 | 0.37 | 0.84 (0.44, 1.59) | 0.65 |
| Control | 13 | 20 | 6 |   | 0.59 | 0.41 |   |   |  |
| Caucasian | Opiate | 38 | 35 | 9 | 0.66 | 0.68 | 0.32 | 0.84 (0.50, 1.41) | 0.54 |
| Control | 24 | 31 | 7 |   | 0.64 | 0.36 |   |   |  |
| Hispanic | Opiate | 28 | 43 | 14 | 1.00 | 0.59 | 0.41 | 1.11 (0.55, 2.32) | 0.87 |
| Control | 8 | 12 | 3 |   | 0.61 | 0.39 |   |   |  |

Table 3 Caption: Association of the genotypes of variants of the OPRL1 gene with opiate addiction

Since the *OPRL1* variants rs6090041 and rs6090043 showed point-wise significant association with opiate addiction, association analyses of the haplotypes of these variants with opiate addiction were conducted. Point-wise significant differences were found in the African Americans and in the Caucasians ([Table 4](#T4)). The AT haplotype was found to be associated with protection from developing opiate addiction in both African Americans and Caucasians (point-wise *P* = 0.041 for each). In the Caucasians only, the GC haplotype was found to be associated with vulnerability to develop opiate addiction (point-wise *P* = 0.020).

### Table 4.

| Haplotype | Estimate of haplotype frequency |  | Haplotype P valuea | Global P valueb |
| --- | --- | --- | --- | --- |
| Opiate | Control |  |  |  |
| A. African American |  |  |  |  |
| A T | 0.288 | 0.421 | 0.041c | 0.215 |
| A C | 0.099 | 0.104 | 0.895 |   |
| G C | 0.467 | 0.332 | 0.061 |   |
| G T | 0.146 | 0.143 | 0.939 |   |
| B. Caucasian |  |  |  |  |
| A T | 0.417 | 0.541 | 0.041c | 0.105 |
| A C | 0.229 | 0.229 | 1.000 |   |
| G C | 0.313 | 0.189 | 0.020c |   |
| G T | 0.040 | 0.041 | 0.984 |   |
| C. Hispanic |  |  |  |  |
| A T | 0.373 | 0.464 | 0.231 | 0.406 |
| A C | 0.274 | 0.231 | 0.530 |   |
| G C | 0.308 | 0.225 | 0.228 |   |
| G T | 0.045 | 0.079 | 0.256 |   |

Table 4 Caption: Association of OPRL1 haplotypes of rs6090041 and rs6090043 with opiate addiction

### Linkage Disequilibrium

Linkage disequilibrium among the five variants was evaluated in the controls of the four primary ethnic groups. As a service to the reader, haplotype blocks were determined, but were not used in the statistical analyses. A single haplotype block was found at the 3′ end of the gene in the African American, Caucasian, and Hispanic groups that encompasses rs6090043 in intron 1, rs2229205 in exon 4, and rs2295448 in intron 4, and spans 11,547 nucleotides ([Table 5a, 5b, 5c](#T5)). In the Asian group, in addition to this block, a second haplotype block was found encompassing rs6011280, which is upstream of the *OPRL1* gene (in the 3′ untranslated region of *RGS19*), and rs6090041 in intron 1, and this block spans 7,536 nucleotides ([Table 5d](#T5)). Thus, variants rs6090043, rs2229205, and rs2295448 are in high linkage disequilibrium in all four ethnic groups, while the variants rs6011280 and rs6090041 are in high linkage disequilibrium in Asians only.

### Table 5.

| A. African American |  |  |  |  |  |
| --- | --- | --- | --- | --- | --- |
| Variant | rs6011280 | rs6090041 | rs6090043 | rs2229205 | rs2295448 |
| rs6011280 | --- | 0.48 | 0.83 | 1.00 | 0.33 |
| rs6090041 | 0.12 | --- | 0.56 | 1.00 | 0.53 |
| rs6090043 | 0.41 | 0.26 | --- | 1.00 | 1.00 |
| rs2229205 | 0.05 | 0.09 | 0.08 | --- | 1.00 |
| rs2295448 | 0.03 | 0.04 | 0.17 | 0.01 | --- |

Table 5 Caption: Pairwise linkage disequilibrium among the five OPRL1 variants in the four primary ethnic groupsa

## Discussion

In this study, we evaluated the association of five variants in the *OPRL1* gene with vulnerability to develop opiate addiction in the three ethnicities with adequate sample sizes (African Americans, Caucasians, and Hispanics). Variant rs6011280 is located 6,348 nucleotides upstream of the *OPRL1* transcription start site, variants rs6090041 and rs6090043 are in intron 1, variant rs2229205 is in exon 4, and variant rs22295448 is in intron 4. The variant rs2229205, located in the coding region of exon 4, does not result in an amino acid change.

In the analysis of the 447 subjects stratified by ethnicity, significant point-wise associations were found between the allele frequency of the variants rs6090041 and rs6090043 with vulnerability to develop opiate addiction in Caucasians. One haplotype (AT) formed by these two variants was significantly associated point-wise with protection from developing opiate addiction in both African Americans and Caucasians. Since we had similar results with both ethnicities, this provides further support for a role of this haplotype in protection from opiate addiction. In Caucasians only, another haplotype (GC) was found to be associated with vulnerability to develop opiate addiction.

The two variants that were point-wise significantly associated with opiate addiction, rs6090041 and rs6090043, are located in the first intron of the *OPRL1* gene. These variants, which are in low linkage disequilibrium (r^2^ = 0.26, 0.24, and 0.13 in African Americans, Caucasians, and Hispanics, respectively), may themselves alter function, or they may be markers for nearby functional variants that have a role in determining an individual’s vulnerability to develop opiate addiction. An evaluation of predicted transcription factor binding sites, using TESS ([Schung and Overton, 1998](#R34)), suggests that the sequence surrounding the G allele of rs6090041 predicts a putative AP-2 transcription factor binding site. The presence of the A allele would disrupt this AP-2 binding site. The AP-2 binding site containing the G allele is GCGGTGGG, which conforms to the AP-2 consensus motif CCCMNSSS (TESS - TRANSFAC Site Record [R02121](https://www.ncbi.nlm.nih.gov/nuccore/R02121)) in reverse orientation (SSSNKGGG), while the site is altered to GCGGTGAG, which no longer conforms to the consensus motif and therefore would not bind AP-2. AP-2 has been found to alter the transcription of genes impacting the morphogenesis of the peripheral nervous system during embryogenesis ([Mitchell et al., 1991](#R23)). Similarly, analyses using TESS suggest that the C allele of rs6090043 predicts several transcription factor binding sites that the T allele would disrupt, including binding sites for CREB and AP-1. The CREB binding motif containing the C allele is TGACG, the same sequence as the CREB binding motif that is found in the insulin gene (TESS - TRANSFAC Site Record [R02710](https://www.ncbi.nlm.nih.gov/nuccore/R02710); [Boam et al., 1990](#R4)); when this motif is altered to TGATG, it would not bind CREB. CREB is activated in neurons by a wide range of extracellular stimuli and is involved in the function, development, and plasticity of the nervous system ([Lonze et al., 2002](#R19)). The AP-1 binding motif containing the C allele is TGAC, the same sequence as the AP-1 binding motif found in adenovirus 5 (TESS - TRANSFAC Site Record [R00368](https://www.ncbi.nlm.nih.gov/nuccore/R00368); [Merino et al., 1989](#R22)); when the motif is changed to TGAT, it should not bind AP-1. AP-1 has been shown to be required for efficient axonal regeneration ([Raivich, et al., 2004](#R30)). Thus, both variants may alter transcriptional regulation.

In this study in Caucasians, the G allele at rs6090041 and the C allele at rs609004, which are both associated with an increased risk for developing opiate addiction (see [Table 3](#T3)), each form potential additional transcription factor binding sites that may increase NOP-R expression. Since NOP-R has anti-opioid effects, it is possible that individuals with increased NOP-R expression may require a greater amount of an opiate to experience drug reward and therefore may be more likely to become addicted to opiates.

There are three published genetic studies of the nociceptin system in humans. The first study found that there was no association between the number of copies of a 39-base pair repeat in *OPRL1* and panic disorder in a case-control study in Japanese subjects ([Kobayashi et al., 2007](#R12)). The second study examined the association of *OPRL1* and N/OFQ variants with alcohol, illicit drug, or opiate dependence (Xuei *et al.* 2007). This study examined 10 variants of the *OPRL1* gene, and an adjacent regulatory gene (*RGS19*), and 15 variants of the gene that encodes N/OFQ (*PNOC*). Among the variants of *OPRL1* were rs6011280, rs6090041, rs6090043, and rs2229205, which were examined in our study. rs2295448 was not studied by Xuei *et al*. No experiment-wise significant associations with alcohol or illicit drug dependence were found in that study. When opiate addiction was examined, they found a “marginally significant” association of variants rs6512305 and rs6090043 with opiate addiction (*P* = 0.05). However, they did not state which allele of rs6512305 or rs6090043 was associated with opiate dependence. In our study, we found an association between rs6090043 and opiate addiction that had point-wise significance (*P* = 0.04). The Xuei *et al.* study found that variants rs6512305 and rs6090043 were in high linkage disequilibrium (D′ = 0.99) in European Americans. It may be that rs6512305, or another nearby variant in linkage disequilibrium with rs6090043, may be the functional marker. The convergence of our results with that of Xuei *et al.* may be due the alteration of the overlapping CREB/AP-1 binding sites, resulting in an altered expression of *OPRL1*. We also found that variant rs6090041, which Xuei *et al*. studied, had a significant association with opiate addiction (*P* = 0.03). The lack of significance found in the Xuei *et al.* study may be due to their small number of cases (83 cases). The third genetic study examined the association between *OPRL1* variants and alcohol dependence in a Swedish population ([Huang et al., 2008](#R8)). They studied 15 variants of *OPRL1*, including four of the five variants that we examined (rs6090041, rs6090043, rs2229205, and rs2295448). They found variant rs6010718 (a variant we did not examine) in *OPRL1* and two haplotypes of the *OPRL1* gene to be significantly associated with alcohol dependence. A rare haplotype (5.9% in cases, 0.6% in controls) containing the T allele of rs6090043 was found to be associated with alcohol dependence. We found the common (46% in cases, 58% in controls) T allele of rs6090043 to be protective from opiate addiction in Caucasians. It may be that alterations in function caused by the rs6090043 allele (or a nearby linked functional allele) may influence vulnerability to develop alcohol dependence and opiate addiction in different ways.

We found point-wise significance of an association of *OPRL1* variants with the vulnerability to develop opiate addiction. A future study with a larger sample size should have more power to detect an association. However, our study demonstrates a moderate association of *OPRL1* variants with vulnerability to develop opiate addiction specifically and thereby contributes to the growing evidence that the *OPRL1* gene may contribute to the development of illicit drug and alcohol dependence.

## Conclusions

In this study, we found that two variants of the *OPRL1* gene, rs6090041 and rs6090043, are significantly associated point-wise with vulnerability to develop opiate addiction, and also form a haplotype that was associated with vulnerability to develop opiate addiction and a haplotype that was protective. These two variants, located in the first intron of the gene, may alter transcription factor binding sites, thus altering *OPRL1* gene transcription and, finally, the development of opiate addiction. These two variants, which are not in linkage disequilibrium, may be markers for nearby functional variants that themselves may have a role in an individual’s vulnerability to develop opiate addiction. Our study provides evidence for an association between *OPRL1* variants and vulnerability to develop opiate addiction, which may ultimately lead to better prevention strategies or new therapeutic approaches.

## Acknowledgments

Sources of support: National Institutes of Health (NIH) grants P60-DA005130 (M.J.K.), R01-MH79880, RR UL1RR024143 (B.C.), NIH-MH R01-44292 (J.O.), R01-DA12848 (M.J.K.), NSFC grants 30730057 and 30700442 from the Chinese Government (J.O.), and American Federation for Aging Research Medical Student Training in Aging Research grant (J.A.B.).

We thank all of our clinical staff for recruiting, screening, and assessing study subjects. We also thank Matthew Randesi for technical assistance and Sara Hamon for statistical assistance.

## References

1. De Bakker R, Yelensky R, Pe’er I, Gabriel SB, Daly MJ, Altshuler D. Efficiency and power in genetic association studies. Nature Genetics. 2005;37:1217–1223. doi: 10.1038/ng1669.  [DOI](https://doi.org/10.1038/ng1669) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16244653/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature%20Genetics&title=Efficiency%20and%20power%20in%20genetic%20association%20studies&author=R%20De%20Bakker&author=R%20Yelensky&author=I%20Pe%E2%80%99er&author=SB%20Gabriel&author=MJ%20Daly&volume=37&publication_year=2005&pages=1217-1223&pmid=16244653&doi=10.1038/ng1669&)

2. Barrett JC, Fry B, Maller J, Daly MJ. Haploview: analysis and visualization of LD and haplotype maps. Bioinformatics. 2005;21:263–265. doi: 10.1093/bioinformatics/bth457.  [DOI](https://doi.org/10.1093/bioinformatics/bth457) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15297300/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=Haploview:%20analysis%20and%20visualization%20of%20LD%20and%20haplotype%20maps&author=JC%20Barrett&author=B%20Fry&author=J%20Maller&author=MJ%20Daly&volume=21&publication_year=2005&pages=263-265&pmid=15297300&doi=10.1093/bioinformatics/bth457&)

3. Benjamini Y, Hochberg Y. Controlling the false discovery rate: A practical and powerful approach to multiple testing. Journal of the Royal Statistical Society Series B (Methodological) 1995;57:289–300.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20of%20the%20Royal%20Statistical%20Society%20Series%20B%20(Methodological)&title=Controlling%20the%20false%20discovery%20rate:%20A%20practical%20and%20powerful%20approach%20to%20multiple%20testing&author=Y%20Benjamini&author=Y%20Hochberg&volume=57&publication_year=1995&pages=289-300&)

4. Boam DSW, Clark AR, Docherty K. Positive and negative regulation of the human insulin gene by multiple Trans-acting Factors. J Biol Chem. 1990;265:8285–8296.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/2186040/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=Positive%20and%20negative%20regulation%20of%20the%20human%20insulin%20gene%20by%20multiple%20Trans-acting%20Factors&author=DSW%20Boam&author=AR%20Clark&author=K%20Docherty&volume=265&publication_year=1990&pages=8285-8296&pmid=2186040&)

5. Ciccocioppo R, Angeletti S, Panocka I, Massi M. Nociceptin/orphanin FQ and drugs of abuse. Peptides. 2000;21:1071–1080. doi: 10.1016/s0196-9781(00)00245-x.  [DOI](https://doi.org/10.1016/s0196-9781(00)00245-x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10998542/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Peptides&title=Nociceptin/orphanin%20FQ%20and%20drugs%20of%20abuse&author=R%20Ciccocioppo&author=S%20Angeletti&author=I%20Panocka&author=M%20Massi&volume=21&publication_year=2000&pages=1071-1080&pmid=10998542&doi=10.1016/s0196-9781(00)00245-x&)

6. Cowen MS, Lawrence AJ. Alcoholism and neuropeptides: an update. CNS Neurol Disord Drug Targets. 2006;5:233–239. doi: 10.2174/187152706776359646.  [DOI](https://doi.org/10.2174/187152706776359646) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16611096/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=CNS%20Neurol%20Disord%20Drug%20Targets&title=Alcoholism%20and%20neuropeptides:%20an%20update&author=MS%20Cowen&author=AJ%20Lawrence&volume=5&publication_year=2006&pages=233-239&pmid=16611096&doi=10.2174/187152706776359646&)

7. Houtani T, Nishi M, Takeshima H, Nukada T, Sugimoto T. Structure and regional distribution of nociceptin/orphanin FQ precursor. Biochem Biophys Res Commun. 1996;219:714–719. doi: 10.1006/bbrc.1996.0300.  [DOI](https://doi.org/10.1006/bbrc.1996.0300) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8645247/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem%20Biophys%20Res%20Commun&title=Structure%20and%20regional%20distribution%20of%20nociceptin/orphanin%20FQ%20precursor&author=T%20Houtani&author=M%20Nishi&author=H%20Takeshima&author=T%20Nukada&author=T%20Sugimoto&volume=219&publication_year=1996&pages=714-719&pmid=8645247&doi=10.1006/bbrc.1996.0300&)

8. Huang J, Young B, Pletcher MT, Heilig M, Wahlestedt C. Assocation between the nociceptin receptor gene (OPRL1) single nucleotide polymorphisms and alcohol dependence. Addict Biol. 2008;13:88–94. doi: 10.1111/j.1369-1600.2007.00089.x.  [DOI](https://doi.org/10.1111/j.1369-1600.2007.00089.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18269382/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Addict%20Biol&title=Assocation%20between%20the%20nociceptin%20receptor%20gene%20(OPRL1)%20single%20nucleotide%20polymorphisms%20and%20alcohol%20dependence&author=J%20Huang&author=B%20Young&author=MT%20Pletcher&author=M%20Heilig&author=C%20Wahlestedt&volume=13&publication_year=2008&pages=88-94&pmid=18269382&doi=10.1111/j.1369-1600.2007.00089.x&)

9. Ito E, Xie G, Maruyama K, Palmer PP. A core-promoter region functions bi-directionally for human opioid-receptor-like gene ORL1 and its 5′-adjacent gene GAIP. J Mol Biol. 2000;304:259–270. doi: 10.1006/jmbi.2000.4212.  [DOI](https://doi.org/10.1006/jmbi.2000.4212) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11090272/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Mol%20Biol&title=A%20core-promoter%20region%20functions%20bi-directionally%20for%20human%20opioid-receptor-like%20gene%20ORL1%20and%20its%205%E2%80%B2-adjacent%20gene%20GAIP&author=E%20Ito&author=G%20Xie&author=K%20Maruyama&author=PP%20Palmer&volume=304&publication_year=2000&pages=259-270&pmid=11090272&doi=10.1006/jmbi.2000.4212&)

10. Johnson SW, North RA. Opioids excite dopamine neurons by hyperpolarization of local interneurons. J Neurosci. 1992;12:483–488. doi: 10.1523/JNEUROSCI.12-02-00483.1992.  [DOI](https://doi.org/10.1523/JNEUROSCI.12-02-00483.1992) | [PMC free article](/articles/PMC6575608/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/1346804/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Neurosci&title=Opioids%20excite%20dopamine%20neurons%20by%20hyperpolarization%20of%20local%20interneurons&author=SW%20Johnson&author=RA%20North&volume=12&publication_year=1992&pages=483-488&pmid=1346804&doi=10.1523/JNEUROSCI.12-02-00483.1992&)

11. Kendler KS, Jacobson KC, Prescott CA, Neale MC. Specificity of genetic and environmental risk factors for use and abuse/dependence of cannabis, cocaine, hallucinogens, sedatives, stimulants, and opiates in male twins. Am J Psychiatry. 2003;160:687–695. doi: 10.1176/appi.ajp.160.4.687.  [DOI](https://doi.org/10.1176/appi.ajp.160.4.687) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12668357/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Psychiatry&title=Specificity%20of%20genetic%20and%20environmental%20risk%20factors%20for%20use%20and%20abuse/dependence%20of%20cannabis,%20cocaine,%20hallucinogens,%20sedatives,%20stimulants,%20and%20opiates%20in%20male%20twins&author=KS%20Kendler&author=KC%20Jacobson&author=CA%20Prescott&author=MC%20Neale&volume=160&publication_year=2003&pages=687-695&pmid=12668357&doi=10.1176/appi.ajp.160.4.687&)

12. Kobayashi Y, Akiyoshi J, Kanehisa M, Ichioka S, Tanaka Y, Tsuru J, et al. Lack of polymorphism in genes encoding mGluR 7, mGluR 8, GABA(A) receptor alfa-6 subunit and nociceptin/orphanin FQ receptor and panic disorder. Psychiatric Genetics. 2007;17:9. doi: 10.1097/YPG.0b013e32801118bc.  [DOI](https://doi.org/10.1097/YPG.0b013e32801118bc) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17167337/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Psychiatric%20Genetics&title=Lack%20of%20polymorphism%20in%20genes%20encoding%20mGluR%207,%20mGluR%208,%20GABA(A)%20receptor%20alfa-6%20subunit%20and%20nociceptin/orphanin%20FQ%20receptor%20and%20panic%20disorder&author=Y%20Kobayashi&author=J%20Akiyoshi&author=M%20Kanehisa&author=S%20Ichioka&author=Y%20Tanaka&volume=17&publication_year=2007&pages=9&pmid=17167337&doi=10.1097/YPG.0b013e32801118bc&)

13. Kreek MJ. Opiates, opioids and addiction. Mol Psychiatry. 1996;1:232–254.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/9118348/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Psychiatry&title=Opiates,%20opioids%20and%20addiction&author=MJ%20Kreek&volume=1&publication_year=1996&pages=232-254&pmid=9118348&)

14. Kreek MJ, Bart G, Lilly C, LaForge KS, Nielsen DA. Pharmacogenetics and human molecular genetics of opiate and cocaine addictions and their treatments. Pharmacol Rev. 2005;57:1–26. doi: 10.1124/pr.57.1.1.  [DOI](https://doi.org/10.1124/pr.57.1.1) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15734726/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacol%20Rev&title=Pharmacogenetics%20and%20human%20molecular%20genetics%20of%20opiate%20and%20cocaine%20addictions%20and%20their%20treatments&author=MJ%20Kreek&author=G%20Bart&author=C%20Lilly&author=KS%20LaForge&author=DA%20Nielsen&volume=57&publication_year=2005&pages=1-26&pmid=15734726&doi=10.1124/pr.57.1.1&)

15. Kreek MJ, Koob GF. Drug dependence: stress and dysregulation of brain reward pathways. Drug Alcohol Depend. 1998;51:23–47. doi: 10.1016/s0376-8716(98)00064-7.  [DOI](https://doi.org/10.1016/s0376-8716(98)00064-7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9716928/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Alcohol%20Depend&title=Drug%20dependence:%20stress%20and%20dysregulation%20of%20brain%20reward%20pathways&author=MJ%20Kreek&author=GF%20Koob&volume=51&publication_year=1998&pages=23-47&pmid=9716928&doi=10.1016/s0376-8716(98)00064-7&)

16. Kreek MJ, Nielsen DA, LaForge KS. Genes associated with addiction: alcoholism, opiate, and cocaine addiction. Neuromolecular Med. 2004;5:85–108. doi: 10.1385/NMM:5:1:085.  [DOI](https://doi.org/10.1385/NMM:5:1:085) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15001815/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neuromolecular%20Med&title=Genes%20associated%20with%20addiction:%20alcoholism,%20opiate,%20and%20cocaine%20addiction&author=MJ%20Kreek&author=DA%20Nielsen&author=KS%20LaForge&volume=5&publication_year=2004&pages=85-108&pmid=15001815&doi=10.1385/NMM:5:1:085&)

17. Kreek MJ, Nielsen DA, Butelman ER, LaForge KS. Genetic influences on impulsivity, risk taking, stress responsitivity and vulnerability to drug abuse and addiction. Nat Neurosci. 2005;8:1450–7. doi: 10.1038/nn1583.  [DOI](https://doi.org/10.1038/nn1583) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16251987/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Neurosci&title=Genetic%20influences%20on%20impulsivity,%20risk%20taking,%20stress%20responsitivity%20and%20vulnerability%20to%20drug%20abuse%20and%20addiction&author=MJ%20Kreek&author=DA%20Nielsen&author=ER%20Butelman&author=KS%20LaForge&volume=8&publication_year=2005&pages=1450-7&pmid=16251987&doi=10.1038/nn1583&)

18. Lewontin RC. The interaction of selection and linkage I. General considerations. Genetics. 1964;49:49–67. doi: 10.1093/genetics/49.1.49.  [DOI](https://doi.org/10.1093/genetics/49.1.49) | [PMC free article](/articles/PMC1210557/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17248194/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genetics&title=The%20interaction%20of%20selection%20and%20linkage%20I.%20General%20considerations&author=RC%20Lewontin&volume=49&publication_year=1964&pages=49-67&pmid=17248194&doi=10.1093/genetics/49.1.49&)

19. Lonze BC, Ginty DD. Function and regulation of CREB family transcription factors in the nervous system. Neuron. 2002;35(4):605–623. doi: 10.1016/s0896-6273(02)00828-0.  [DOI](https://doi.org/10.1016/s0896-6273(02)00828-0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12194863/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neuron&title=Function%20and%20regulation%20of%20CREB%20family%20transcription%20factors%20in%20the%20nervous%20system&author=BC%20Lonze&author=DD%20Ginty&volume=35&issue=4&publication_year=2002&pages=605-623&pmid=12194863&doi=10.1016/s0896-6273(02)00828-0&)

20. Martin-Fardon R, Ciccocioppo R, Massi M, Weiss F. Nociceptin prevents stress-induced ethanol- but not cocaine-seeking behavior in rats. Neuroreport. 2000;11:1939–1943. doi: 10.1097/00001756-200006260-00026.  [DOI](https://doi.org/10.1097/00001756-200006260-00026) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10884047/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neuroreport&title=Nociceptin%20prevents%20stress-induced%20ethanol-%20but%20not%20cocaine-seeking%20behavior%20in%20rats&author=R%20Martin-Fardon&author=R%20Ciccocioppo&author=M%20Massi&author=F%20Weiss&volume=11&publication_year=2000&pages=1939-1943&pmid=10884047&doi=10.1097/00001756-200006260-00026&)

21. McLellen AT, Kushner H, Metzger D, Peters R, Smith I, Grisson G, et al. The fifth edition of the Addiction Severity Index. J Subst Abuse Treat. 1992;9:199–213. doi: 10.1016/0740-5472(92)90062-s.  [DOI](https://doi.org/10.1016/0740-5472(92)90062-s) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/1334156/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Subst%20Abuse%20Treat&title=The%20fifth%20edition%20of%20the%20Addiction%20Severity%20Index&author=AT%20McLellen&author=H%20Kushner&author=D%20Metzger&author=R%20Peters&author=I%20Smith&volume=9&publication_year=1992&pages=199-213&pmid=1334156&doi=10.1016/0740-5472(92)90062-s&)

22. Merino A, Buckbinder L, Mermelstein FH, Reinberg D. Phosphorylation of cellular proteins regulates their binding to the cAMP response element. J Biol Chem. 1989;264:21266–21276.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/2556400/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=Phosphorylation%20of%20cellular%20proteins%20regulates%20their%20binding%20to%20the%20cAMP%20response%20element&author=A%20Merino&author=L%20Buckbinder&author=FH%20Mermelstein&author=D%20Reinberg&volume=264&publication_year=1989&pages=21266-21276&pmid=2556400&)

23. Mitchell PJ, Timmons PM, Hebert JM, Rigby PW, Tjian R. Transcription factor AP-2 is expressed in neural crest cell lineages during mouse embryogenesis. Genes Dev. 1991;5:105–119. doi: 10.1101/gad.5.1.105.  [DOI](https://doi.org/10.1101/gad.5.1.105) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/1989904/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genes%20Dev&title=Transcription%20factor%20AP-2%20is%20expressed%20in%20neural%20crest%20cell%20lineages%20during%20mouse%20embryogenesis&author=PJ%20Mitchell&author=PM%20Timmons&author=JM%20Hebert&author=PW%20Rigby&author=R%20Tjian&volume=5&publication_year=1991&pages=105-119&pmid=1989904&doi=10.1101/gad.5.1.105&)

24. Mollereau C, Mouledous L. Tissue distribution of the opioid receptor-like (ORL1) receptor. Peptides. 2000;21:907–917. doi: 10.1016/s0196-9781(00)00227-8.  [DOI](https://doi.org/10.1016/s0196-9781(00)00227-8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10998524/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Peptides&title=Tissue%20distribution%20of%20the%20opioid%20receptor-like%20(ORL1)%20receptor&author=C%20Mollereau&author=L%20Mouledous&volume=21&publication_year=2000&pages=907-917&pmid=10998524&doi=10.1016/s0196-9781(00)00227-8&)

25. Mollereau C, Parmentier M, Mailleux P, Butour J-L, Moisand C, Chalon P, et al. ORL1, a novel member of the opioid receptor family: cloning, functional expression and localization. FEBS Lett. 1994;341:33–38. doi: 10.1016/0014-5793(94)80235-1.  [DOI](https://doi.org/10.1016/0014-5793(94)80235-1) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8137918/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=FEBS%20Lett&title=ORL1,%20a%20novel%20member%20of%20the%20opioid%20receptor%20family:%20cloning,%20functional%20expression%20and%20localization&author=C%20Mollereau&author=M%20Parmentier&author=P%20Mailleux&author=J-L%20Butour&author=C%20Moisand&volume=341&publication_year=1994&pages=33-38&pmid=8137918&doi=10.1016/0014-5793(94)80235-1&)

26. Murphy NP, Lee Y, Maidment NT. Orphanin FQ/nociceptin blocks acquisition of morphine place preference. Brain Res. 1999;832:168–170. doi: 10.1016/s0006-8993(99)01425-0.  [DOI](https://doi.org/10.1016/s0006-8993(99)01425-0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10375664/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Brain%20Res&title=Orphanin%20FQ/nociceptin%20blocks%20acquisition%20of%20morphine%20place%20preference&author=NP%20Murphy&author=Y%20Lee&author=NT%20Maidment&volume=832&publication_year=1999&pages=168-170&pmid=10375664&doi=10.1016/s0006-8993(99)01425-0&)

27. Murphy NP, Ly HT, Maidment NT. Intracerebroventricular orphanin FQ/nociceptin suppresses dopamine release in the nucleus accumbens of anaesthetized rats. Neuroscience. 1996;75:1–4. doi: 10.1016/0306-4522(96)00322-3.  [DOI](https://doi.org/10.1016/0306-4522(96)00322-3) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8923516/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neuroscience&title=Intracerebroventricular%20orphanin%20FQ/nociceptin%20suppresses%20dopamine%20release%20in%20the%20nucleus%20accumbens%20of%20anaesthetized%20rats&author=NP%20Murphy&author=HT%20Ly&author=NT%20Maidment&volume=75&publication_year=1996&pages=1-4&pmid=8923516&doi=10.1016/0306-4522(96)00322-3&)

28. Murphy NP, Maidment NT. Orphanin FQ/nociceptin modulation of mesolimbic dopamine transmission determined by microdialysis. J Neurochem. 1999;73:179–186. doi: 10.1046/j.1471-4159.1999.0730179.x.  [DOI](https://doi.org/10.1046/j.1471-4159.1999.0730179.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10386969/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Neurochem&title=Orphanin%20FQ/nociceptin%20modulation%20of%20mesolimbic%20dopamine%20transmission%20determined%20by%20microdialysis&author=NP%20Murphy&author=NT%20Maidment&volume=73&publication_year=1999&pages=179-186&pmid=10386969&doi=10.1046/j.1471-4159.1999.0730179.x&)

29. Qin ZS, Niu T, Liu JS. Partition-ligation-expectation-maximization algorithm for haplotype inference with single-nucleotide polymorphisms. Am J Hum Genet. 2002;71(5):1242–1247. doi: 10.1086/344207.  [DOI](https://doi.org/10.1086/344207) | [PMC free article](/articles/PMC385113/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12452179/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=Partition-ligation-expectation-maximization%20algorithm%20for%20haplotype%20inference%20with%20single-nucleotide%20polymorphisms&author=ZS%20Qin&author=T%20Niu&author=JS%20Liu&volume=71&issue=5&publication_year=2002&pages=1242-1247&pmid=12452179&doi=10.1086/344207&)

30. Raivich G, Bohatschek M, Da Costa C, Iwata O, Galiano M, Hristova M, et al. The AP-1 transcription factor c-Jun Is required for efficient axonal regeneration. Neuron. 2004;43(1):57–67. doi: 10.1016/j.neuron.2004.06.005.  [DOI](https://doi.org/10.1016/j.neuron.2004.06.005) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15233917/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neuron&title=The%20AP-1%20transcription%20factor%20c-Jun%20Is%20required%20for%20efficient%20axonal%20regeneration&author=G%20Raivich&author=M%20Bohatschek&author=C%20Da%20Costa&author=O%20Iwata&author=M%20Galiano&volume=43&issue=1&publication_year=2004&pages=57-67&pmid=15233917&doi=10.1016/j.neuron.2004.06.005&)

31. Reinscheid RK. The orphanin FQ/nociceptin receptor as a novel drug target in psychiatric disorders. CNS Neurol Disord Drug Targets. 2006;5:219–224. doi: 10.2174/187152706776359628.  [DOI](https://doi.org/10.2174/187152706776359628) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16611094/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=CNS%20Neurol%20Disord%20Drug%20Targets&title=The%20orphanin%20FQ/nociceptin%20receptor%20as%20a%20novel%20drug%20target%20in%20psychiatric%20disorders&author=RK%20Reinscheid&volume=5&publication_year=2006&pages=219-224&pmid=16611094&doi=10.2174/187152706776359628&)

32. Rettig RA, Yarmolinsky A, editors. Federal regulation of methadone treatment. Washington, DC: National Academy Press; 1995.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/25121195/) | [Google Scholar](https://scholar.google.com/scholar_lookup?title=Federal%20regulation%20of%20methadone%20treatment&publication_year=1995&)

33. Sakoori K, Murphy NP. Central administration of nociceptin/orphanin FQ blocks the acquisition of conditioned place preference to morphine and cocaine, but not conditioned place aversion to naloxone in mice. Psychopharmacology (Berl) 2004;172:129–136. doi: 10.1007/s00213-003-1643-3.  [DOI](https://doi.org/10.1007/s00213-003-1643-3) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14624329/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Psychopharmacology%20(Berl)&title=Central%20administration%20of%20nociceptin/orphanin%20FQ%20blocks%20the%20acquisition%20of%20conditioned%20place%20preference%20to%20morphine%20and%20cocaine,%20but%20not%20conditioned%20place%20aversion%20to%20naloxone%20in%20mice&author=K%20Sakoori&author=NP%20Murphy&volume=172&publication_year=2004&pages=129-136&pmid=14624329&doi=10.1007/s00213-003-1643-3&)

34. Schug J, Overton GC. TESS: Transcription element search software on the WWW Technical Report CBIL-TR-1997-1001-v00. Computational Biology and Informatics Laboratory. School of Medicine. University of Pennsylvania; 1998. URL: http://www.cbil.upenn.edu/tess.  [http://www.cbil.upenn.edu/tess](http://www.cbil.upenn.edu/tess) | [Google Scholar](https://scholar.google.com/scholar_lookup?title=TESS:%20Transcription%20element%20search%20software%20on%20the%20WWW%20Technical%20Report%20CBIL-TR-1997-1001-v00&author=J%20Schug&author=GC%20Overton&publication_year=1998&)

35. Sinnwell JP, Schaid DJ. haplo.stats: Statistical analysis of haplotypes with traits and covariates when linkage phase is ambiguous. R package version 1.4.3. 2009 URL: http://mayoresearch.mayo.edu/mayo/research/schaid_lab/software.cfm.  [http://mayoresearch.mayo.edu/mayo/research/schaid_lab/software.cfm](http://mayoresearch.mayo.edu/mayo/research/schaid_lab/software.cfm)

36. Sokal RR, Rohlf FJ. Biometry: the principles and practice of statistics in biological research. 3 New York: 1994.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Biometry:%20the%20principles%20and%20practice%20of%20statistics%20in%20biological%20research&author=RR%20Sokal&author=FJ%20Rohlf&publication_year=1994&)

37. Storey JD, Tibshirani . Statistical significance for genome-wide experiments. 2003.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Statistical%20significance%20for%20genome-wide%20experiments&author=JD%20Storey&author=%20Tibshirani&publication_year=2003&)

38. Storey JD. The positive false discovery rate: A bayesian interpretation and the q-value. The Annals of Statistics. 2003;31:2013–2035.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=The%20Annals%20of%20Statistics&title=The%20positive%20false%20discovery%20rate:%20A%20bayesian%20interpretation%20and%20the%20q-value&author=JD%20Storey&volume=31&publication_year=2003&pages=2013-2035&)

39. Tsuang MT, Lyons MJ, Meyer JM, Doyle T, Eisen SA, Goldberg J, True W, Lin N, Toomey R, Eaves L. Co-occurrence of abuse of different drugs in men. Arch Gen Psychiatry. 1998;55:967–972. doi: 10.1001/archpsyc.55.11.967.  [DOI](https://doi.org/10.1001/archpsyc.55.11.967) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9819064/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arch%20Gen%20Psychiatry&title=Co-occurrence%20of%20abuse%20of%20different%20drugs%20in%20men&author=MT%20Tsuang&author=MJ%20Lyons&author=JM%20Meyer&author=T%20Doyle&author=SA%20Eisen&volume=55&publication_year=1998&pages=967-972&pmid=9819064&doi=10.1001/archpsyc.55.11.967&)

40. Ueda H, Yamaguchi T, Tokuyama S, Inoue M, Nishi M, Takeshima H. Partial loss of tolerance liability to morphine analgesia in mice lacking the nociceptin receptor gene. Neuroscience Lett. 1997;237:136–138. doi: 10.1016/s0304-3940(97)00832-x.  [DOI](https://doi.org/10.1016/s0304-3940(97)00832-x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9453234/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neuroscience%20Lett&title=Partial%20loss%20of%20tolerance%20liability%20to%20morphine%20analgesia%20in%20mice%20lacking%20the%20nociceptin%20receptor%20gene&author=H%20Ueda&author=T%20Yamaguchi&author=S%20Tokuyama&author=M%20Inoue&author=M%20Nishi&volume=237&publication_year=1997&pages=136-138&pmid=9453234&doi=10.1016/s0304-3940(97)00832-x&)

41. Walker JR, Spina M, Terenius L, Koob GF. Nociceptin fails to affect heroin self-administration in the rat. Neuroreport. 1998;9:2243–2247. doi: 10.1097/00001756-199807130-00017.  [DOI](https://doi.org/10.1097/00001756-199807130-00017) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9694207/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neuroreport&title=Nociceptin%20fails%20to%20affect%20heroin%20self-administration%20in%20the%20rat&author=JR%20Walker&author=M%20Spina&author=L%20Terenius&author=GF%20Koob&volume=9&publication_year=1998&pages=2243-2247&pmid=9694207&doi=10.1097/00001756-199807130-00017&)

42. Wang N, Akey JM, Zhang K, Chakraborty R, Jin L. Distribution of recombination crossovers and the origin of haplotype blocks: the interplay of population history, recombination, and mutation. American Journal of Human Genetics. 2002;71:1227–1234. doi: 10.1086/344398.  [DOI](https://doi.org/10.1086/344398) | [PMC free article](/articles/PMC385104/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12384857/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=American%20Journal%20of%20Human%20Genetics&title=Distribution%20of%20recombination%20crossovers%20and%20the%20origin%20of%20haplotype%20blocks:%20the%20interplay%20of%20population%20history,%20recombination,%20and%20mutation&author=N%20Wang&author=JM%20Akey&author=K%20Zhang&author=R%20Chakraborty&author=L%20Jin&volume=71&publication_year=2002&pages=1227-1234&pmid=12384857&doi=10.1086/344398&)

43. Warnes GR. R News. 2003. The genetics package; p. 3.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=R%20News&author=GR%20Warnes&publication_year=2003&)

44. Wu X, Zhu K, Wang Z, Chen S, Yang L. GATA-1, -2, -3 genes expression in bone marrow microenvironment with chronic aplastic anemia. Hematology. 2007;12:331–335. doi: 10.1080/10245330701255288.  [DOI](https://doi.org/10.1080/10245330701255288) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17654061/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hematology&title=GATA-1,%20-2,%20-3%20genes%20expression%20in%20bone%20marrow%20microenvironment%20with%20chronic%20aplastic%20anemia&author=X%20Wu&author=K%20Zhu&author=Z%20Wang&author=S%20Chen&author=L%20Yang&volume=12&publication_year=2007&pages=331-335&pmid=17654061&doi=10.1080/10245330701255288&)

45. Wu YL, Pu L, Ling K, Zhao J, Cheng ZJ, Ma L, Pei G. Molecular characterization and functional expression of opioid receptor-like1 receptor. Cell Res. 1997;7:69–77. doi: 10.1038/cr.1997.8.  [DOI](https://doi.org/10.1038/cr.1997.8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9261564/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell%20Res&title=Molecular%20characterization%20and%20functional%20expression%20of%20opioid%20receptor-like1%20receptor&author=YL%20Wu&author=L%20Pu&author=K%20Ling&author=J%20Zhao&author=ZJ%20Cheng&volume=7&publication_year=1997&pages=69-77&pmid=9261564&doi=10.1038/cr.1997.8&)

46. Xuei X, Flury-Wetherill L, Almasy L, Beirut L, Tischfield J, Schuckit M, et al. Assocation analysis of genes encoding the nociceptin receptor (OPRL1) and its endogenous ligand (PNOC) with alcohol or illicit drug dependence. Addict Biol. 2008;13:80–87. doi: 10.1111/j.1369-1600.2007.00082.x.  [DOI](https://doi.org/10.1111/j.1369-1600.2007.00082.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17910740/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Addict%20Biol&title=Assocation%20analysis%20of%20genes%20encoding%20the%20nociceptin%20receptor%20(OPRL1)%20and%20its%20endogenous%20ligand%20(PNOC)%20with%20alcohol%20or%20illicit%20drug%20dependence&author=X%20Xuei&author=L%20Flury-Wetherill&author=L%20Almasy&author=L%20Beirut&author=J%20Tischfield&volume=13&publication_year=2008&pages=80-87&pmid=17910740&doi=10.1111/j.1369-1600.2007.00082.x&)

47. Yu J, Chait BT, Toll L, Kreek MJ. Nociceptin in vitro biotransformation in human blood. Peptides. 1996;17:873–876. doi: 10.1016/0196-9781(96)00079-4.  [DOI](https://doi.org/10.1016/0196-9781(96)00079-4) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8844779/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Peptides&title=Nociceptin%20in%20vitro%20biotransformation%20in%20human%20blood&author=J%20Yu&author=BT%20Chait&author=L%20Toll&author=MJ%20Kreek&volume=17&publication_year=1996&pages=873-876&pmid=8844779&doi=10.1016/0196-9781(96)00079-4&)

48. Zhang Y, Butelman ER, Schlussman SD, Ho A, Kreek MJ. Effect of the endogenous kappa opioid agonist dynorphin A(1–17) on cocaine-evoked increases in striatal dopamine levels and cocaine-induced place preference in C57BL/6J mice. Psychopharmacology. 2004;172:422–429. doi: 10.1007/s00213-003-1688-3.  [DOI](https://doi.org/10.1007/s00213-003-1688-3) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14712335/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Psychopharmacology&title=Effect%20of%20the%20endogenous%20kappa%20opioid%20agonist%20dynorphin%20A(1%E2%80%9317)%20on%20cocaine-evoked%20increases%20in%20striatal%20dopamine%20levels%20and%20cocaine-induced%20place%20preference%20in%20C57BL/6J%20mice&author=Y%20Zhang&author=ER%20Butelman&author=SD%20Schlussman&author=A%20Ho&author=MJ%20Kreek&volume=172&publication_year=2004&pages=422-429&pmid=14712335&doi=10.1007/s00213-003-1688-3&)
