# Genome-Wide Association Analysis in Asthma Subjects Identifies SPATS2L as a Novel Bronchodilator Response Gene

## Metadata
**Authors:** Blanca E Himes, Xiaofeng Jiang, Ruoxi Hu, Ann C Wu, Jessica A Lasky-Su, Barbara J Klanderman, John Ziniti, Jody Senter-Sylvia, John J Lima, Charles G Irvin, Stephen P Peters, Deborah A Meyers, Eugene R Bleecker, Michiaki Kubo, Mayumi Tamari, Yusuke Nakamura, Stanley J Szefler, Robert F Lemanske, Jr, Robert S Zeiger, Robert C Strunk, Fernando D Martinez, John P Hanrahan, Gerard H Koppelman, Dirkje S Postma, Maartje A E Nieuwenhuis, Judith M Vonk, Reynold A Panettieri, Jr, Amy Markezich, Elliot Israel, Vincent J Carey, Kelan G Tantisira, Augusto A Litonjua, Quan Lu, Scott T Weiss
**Journal:** PLoS Genetics
**Date:** 2012 Jul 5
**DOI:** [10.1371/journal.pgen.1002824](https://doi.org/10.1371/journal.pgen.1002824)
**PMID:** 22792082
**PMCID:** PMC3390407
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3390407/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC3390407/pdf/pgen.1002824.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC3390407/pdf/pgen.1002824.pdf)

## Abstract

Bronchodilator response (BDR) is an important asthma phenotype that measures reversibility of airway obstruction by comparing lung function (i.e. FEV1) before and after the administration of a short-acting β2-agonist, the most common rescue medications used for the treatment of asthma. BDR also serves as a test of β2-agonist efficacy. BDR is a complex trait that is partly under genetic control. A genome-wide association study (GWAS) of BDR, quantified as percent change in baseline FEV1 after administration of a β2-agonist, was performed with 1,644 non-Hispanic white asthmatic subjects from six drug clinical trials: CAMP, LOCCS, LODO, a medication trial conducted by Sepracor, CARE, and ACRN. Data for 469,884 single-nucleotide polymorphisms (SNPs) were used to measure the association of SNPs with BDR using a linear regression model, while adjusting for age, sex, and height. Replication of primary P-values was attempted in 501 white subjects from SARP and 550 white subjects from DAG. Experimental evidence supporting the top gene was obtained via siRNA knockdown and Western blotting analyses. The lowest overall combined P-value was 9.7E-07 for SNP rs295137, near the SPATS2L gene. Among subjects in the primary analysis, those with rs295137 TT genotype had a median BDR of 16.0 (IQR = [6.2, 32.4]), while those with CC or TC genotypes had a median BDR of 10.9 (IQR = [5.0, 22.2]). SPATS2L mRNA knockdown resulted in increased β2-adrenergic receptor levels. Our results suggest that SPATS2L may be an important regulator of β2-adrenergic receptor down-regulation and that there is promise in gaining a better understanding of the biological mechanisms of differential response to β2-agonists through GWAS.

## Introduction

Asthma is a chronic respiratory disease that affects over 20 million Americans and 300 million people worldwide [[1]](#pgen.1002824-American1), [[2]](#pgen.1002824-Global1). A hallmark characteristic of asthma is reversible airway obstruction, which is commonly measured via a bronchodilator response (BDR) test, in which the reduction of bronchoconstriction after administration of a short-acting reliever drug is quantified [[3]](#pgen.1002824-Nelson1). β_2_-agonists, the most common short-acting reliever drugs used during BDR tests and for asthma therapy, act in part by stimulating β_2_-adrenergic receptors (β_2_ARs) on airway smooth muscle cells to reduce bronchoconstriction via subsequent increases in cyclic adenosine monophosphate (cAMP) and protein kinase A (PKA) [[3]](#pgen.1002824-Nelson1). Although a comprehensive pathophysiologic understanding of BDR has not been obtained, it is a complex trait involving interactions among various tissues and cells, including inflammatory [[4]](#pgen.1002824-Loza1), airway epithelium [[5]](#pgen.1002824-Salathe1), smooth muscle [[6]](#pgen.1002824-Shore1), and the autonomic nervous system [[7]](#pgen.1002824-Jartti1). In addition to being used for the diagnosis of asthma, BDR tests can be used to measure whether inhaled β_2_-agonists are effective in patients. Although short-acting β_2_-agonists are widely used clinically as asthma rescue medications, they are variably efficacious among patients [[8]](#pgen.1002824-Drazen1). Studying BDR may thus provide information regarding both the pathophysiology and pharmacogenetics of asthma.

The search for genetic variants that modify asthma susceptibility has resulted in the most recent multi-center asthma genome-wide association studies (GWAS) providing strong statistical evidence for the association of many genes, including the *IKZF3-ZPBP2-GSDMB-ORMDL3* locus, *HLA-DQ*, *IL1RL1*, *IL18RL1*, *IL33*, *TSLP*, *SLC22A5*, *SMAD3*, and *RORA*, with asthma [[9]](#pgen.1002824-Moffatt1), [[10]](#pgen.1002824-Torgerson1). Functional experiments to identify the role that these genes play in asthma pathophysiology are hindered by the complexity of the asthma phenotype. Familial aggregation [[11]](#pgen.1002824-Niu1) and genetic association studies [[12]](#pgen.1002824-Hawkins1) have provided suggestive evidence for a genetic contribution to interindividual differences in BDR. Candidate genes reported to be associated with BDR include β_2_-adrenergic receptor (*ADRB2*) [[13]](#pgen.1002824-Drysdale1), [[14]](#pgen.1002824-Silverman1), adenylyl cyclase type 9 (*ADCY9*) [[15]](#pgen.1002824-Tantisira1), corticotrophin-releasing hormone receptor 2 (*CRHR2*) [[16]](#pgen.1002824-Poon1), and arginase 1 (*ARG1*) [[17]](#pgen.1002824-Litonjua1), [[18]](#pgen.1002824-Vonk1). While BDR is a complex phenotype, functional studies of BDR candidate genes are simpler than those for a general asthma phenotype because this pharmacogenetic phenotype can be readily simulated *in vitro* via stimulation of cells with β_2_-agonists.

In this study, we performed a GWAS of BDR in 1,644 non-Hispanic white asthmatics and found that the strongest evidence of association with BDR was at variants near the Homo sapiens spermatogenesis associated, serine-rich 2-like (*SPATS2L*) gene. We attempted to replicate the primary findings in two independent populations and investigated the function of *SPATS2L* via mRNA knockdown experiments and found evidence to support its involvement in BDR.

## Results

### Primary BDR GWAS

[Figure 1](#pgen-1002824-g001) is an overview of our study design. Characteristics of the subjects used in the primary GWAS are provided in [Table 1](#pgen-1002824-t001). We utilized 1,644 non-Hispanic white subjects from six clinical trials to measure the association of SNPs to BDR. After QC filters, 469,884 SNPs genotyped in CAMP/LOCCS/LODO/Sepracor and either genotyped or imputed in CARE and ACRN were used to test for the association of SNPs to BDR. We utilized genotyped SNPs for CAMP/LOCCS/LODO/Sepracor because these cohorts, who were all genotyped using Illumina platforms, had the largest sample size. Due to the poor overlap of Illumina and Affymetrix platform SNPs, we utilized HapMap Phase 2 imputed SNPs for CARE and ACRN, so that the maximal number of SNPs in all cohorts could be analyzed. The quantile-quantile (QQ) and Manhattan plots revealed that the distribution of association P-values was similar to that expected for a null distribution and that no P-values met genome-wide statistically significant levels ([Figures S1](#pgen.1002824.s001) and [S2](#pgen.1002824.s002)). To expand the primary association results further, all SNPs available in the June 2010 release of the 1000 Genome Project (1000GP) data were imputed using MaCH in each of the three primary groups of genotype data and overall BDR GWAS results were re-computed. Among SNPs contained in the primary GWAS, imputed and genotyped P-values were similar, particularly for those with low P-values ([Figure S3](#pgen.1002824.s003)). Some imputed regions had P-values lower than those of the primary GWAS, but the results in most of these regions were not supported by primary GWAS data ([Figure S2](#pgen.1002824.s002)). Thus, we proceeded to attempt to validate the primary GWAS findings based on the combined genotyped CAMP/LOCCS/LODO/Sepracor SNP results and HapMap Phase 2 imputed CARE and ACRN SNP results. The top five primary GWAS SNPs with P-value<1E-05 are in [Table 2](#pgen-1002824-t002). Further details on these regions and all primary GWAS SNPs with P-value<1E-04 are in [Tables S1](#pgen.1002824.s008) and [S2](#pgen.1002824.s009). Further details on all 1000GP imputed GWAS SNPs with P-value<1E-05 are in [Tables S3](#pgen.1002824.s010)

### Figure 1. Study overview.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5c91/3390407/5f68188ef7b0/pgen.1002824.g001.jpg)

(A) Primary GWAS was conducted using subjects from CAMP, LOCCS, LODO, Sepracor, ACRN, and CARE cohort. Samples genotyped on Illumina platforms (i.e. CAMP/LOCCS/LODO/Sepracor) were pooled and analyzed first. Results from samples genotyped on Affymetrix platforms were analyzed separately and then combined to obtain the primary GWAS results. 1000GP imputed data was utilized to expand the primary GWAS association results. (B) Replication of the top (i.e. P-value<1E-04) SNPs from the primary GWAS were attempted in two independent populations: SARP and DAG. (C) The SPATS2L gene was selected for functional validation based on nominal replication of association results in SARP and analysis of publicly available resources.

### Table 1. Characteristics of GWAS subjects.

|   | CAMP (n = 546) | Sepracor (n = 401) | LOCCS (n = 135) | LODO (n = 114) | CARE§ (n = 207) | ACRN¶ (n = 241) | SARP (n = 552) | DAG (n = 550) |
| --- | --- | --- | --- | --- | --- | --- | --- | --- |
| Age | 8.9 (2.1) [5, 13] | 32.4 (14.0) [12, 80] | 35.8 (14.7) [7, 67] | 43.0 (15.0) [15, 76] | 10.6 (2.9) [6, 18] | 31.7 (10.5) [12, 64] | 36.5 (13.1) [14, 68] | 39.9 (12.6) [7, 75] |
| Male – N (%) | 327 (60) | 196 (49) | 49 (36) | 28 (25) | 129 (62) | 100 (41) | 179 (32) | 247 (45) |
| Wash-out Prior to BDR Test* - Weeks | 4 | 6 | 4–6 on fluticasone | 2 | 0–4 | 0–6 | 0 | 0–2 |
| Albuterol puffs (90 µg/puff) for BDR Test | 2 | 2 | 2 | 2 | 4 | 2 or 4 | 2 to 8 | 800 µg Salbutamol |
| Pre-BD FEV1 | 1.7 (0.5) [0.4, 3.3] | 2.2 (0.5) [1.1, 3.6] | 3.1 (0.8) [1.3, 5.1] | 2.5 (0.7) [0.9, 4.9] | 2.1 (0.7) [0.9, 5.5] | 3.2 (0.8) [1.1, 5.6] | 2.6 (0.9) [0.5, 15.9] | 2.8 (0.9) [0.4, 5.9] |
| Pre-BD FEV1 % Predicted | 94.0 (14.2) [51.0, 138.0] | 57.7 (14.0) [23.3, 81.0] | 87.4 (18.6) [38.6, 121.4] | 76.8 (23.6) [30.3, 126.2] | 98.5 (12.9) [69.6, 133.1] | 87.0 (14.4) [35.0, 137.0] | 76.7 (20.9) [5.9, 117.9] | 80.1 (21.0) [11.6, 131.8] |
| BDR | 10.8 (10.2) [−16.0, 60.0] | 40.6 (20.9) [14.9, 140.5] | 6.0 (6.3) [−11.1, 21.3] | 6.0 (10.5) [−5.6, 60.3] | 9.7 (8.4) [−16.7, 37.6] | 11.6 (10.2) [−2.7, 73.9] | 13.3 (16.1) [−6, 146] | 17.5 (15.5) [−14.5, 131.2] |
| Genotyping Platform** | 550Kv3 or 610 | 610 | 610 | 610 | Affy 6.0 | Affy 6.0 | 610 | 317K or 370K |

Table 1 Caption: Pre-BD = Pre-bronchodilator. SD = Standard deviation. For continuous variables, Mean (SD) and [Range] are shown. *: Subjects were allowed to use rescue medication during wash-out period unless otherwise indicated. **: 550Kv3, 610, 317K, and 370K refer to Illumina BeadChips. §: CARE trials included: CLIC, PACT, MARS. ¶: ACRN trials included: BAGS, DICE, IMPACT, PRICE.

### Table 2. Primary GWAS Top Results (SNPs with Combined P-values <1e-05).

|   |   |   |   | CLLS |  | CARE |  |  | ACRN |  |  |   |   |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| SNP | CHR | BP | Minor Allele | MAF | P-value | MAF | P-value | Rsq | MAF | P-value | Rsq | Combined P-value | Gene |
| rs4452682 | 6 | 3360301 | A | 0.40 | 4.80E-05 | 0.44 | 0.021 | 0.89 | 0.46 | 0.018 | 0.79 | 3.20E-06 | SLC22A23 |
| rs295137 | 2 | 200858285 | T | 0.41 | 4.90E-05 | 0.41 | 1.10E-03 | 0.95 | 0.41 | 0.12 | 0.97 | 3.40E-06 | SPATS2L |
| rs295114 | 2 | 200903847 | T | 0.43 | 3.50E-05 | 0.42 | 1.50E-03 | 0.96 | 0.42 | 0.42 | 0.99 | 6.00E-06 | SPATS2L |
| rs10940113 | 5 | 66850598 | C | 0.37 | 1.00E-06 | 0.34 | 0.73 | 0.90 | 0.35 | 0.72 | 0.91 | 6.10E-06 |   |
| rs4328902 | 4 | 25348267 | T | 0.27 | 2.20E-05 | 0.27 | 0.38 | 0.73 | 0.26 | 0.026 | 0.78 | 6.10E-06 |   |

Table 2 Caption: CLLS = CAMP/LOCCS/LODO/Sepracor. CLLS used genotyped SNP data. CARE and ACRN used HapMap Phase 2 imputed data with Mach ratio of empirically observed dosage variance to the expected dosage variance (Rsq) values indicated.

### Replication of Primary GWAS Findings

We attempted to replicate in SARP all of the SNPs with primary GWAS P-values<1E-04 ([Table S5](#pgen.1002824.s012)). Three had nominally significant P-values (i.e. <0.05), and two of these associations supported the top 5 primary GWAS associations ([Table 3](#pgen-1002824-t003)). The lowest combined P-value for all primary GWAS plus SARP data was 7.7E-07 for rs295137. The region of BDR association spanning this SNP was in the 5′UTR region of *SPATS2L*, a gene of unknown *in vivo* function and paralog of *SPATS2* ([Figure 2](#pgen-1002824-g002)). The effect of the rs295137 genotype on BDR is shown in [Figure 3](#pgen-1002824-g003), and a plot of the residuals of the linear regression fit of BDR while adjusting for age, sex, and height is shown in [Figure S4](#pgen.1002824.s004). We sought further evidence of association for the two *SPATS2L* SNPs with lowest P-values in our primary GWAS in a second independent population: DAG. There was no evidence for association in this cohort (rs295137 P-value = 0.21; rs295114 P-value = 0.21), and combined P-values for these two SNPs across all cohorts were 9.7E-07 and 1.6E-06 ([Table 3](#pgen-1002824-t003)). To investigate whether our top combined association represented a biologically significant finding, we sought experimental evidence that *SPATS2L* was involved in bronchodilator response.

### Table 3. Replication study of top SNPs in two independent populations (SARP and DAG).

|   | SARP |  | DAG |  |   |
| --- | --- | --- | --- | --- | --- |
| SNP | MAF | P-value | MAF | P-value | Combined P-value* |
| rs4452682 | 0.50 | 0.43 | - | - | 1.3E-05 |
| rs295137 | 0.41 | 0.044 | 0.40 | 0.21 | 9.7E-07 |
| rs295114 | 0.43 | 0.043 | 0.42 | 0.21 | 1.6E-06 |
| rs10940113 | 0.37 | 0.16 | - | - | 5.5E-06 |
| rs4328902 | 0.34 | 0.91 | - | - | 2.8E-04 |

Table 3 Caption: *: rs295137 and rs295114 include SARP and DAG P-values. Remaining SNPs include SARP only.

### Figure 2. Region of association near SPATS2L SNPs to BDR.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5c91/3390407/c8e350e53354/pgen.1002824.g002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3390407_pgen.1002824.g002.jpg)

The x-axis denotes position along Chromosome 2. The y-axis denotes −Log10(P) corresponding to 1000GP imputed data P-values. LD between the SNP with the lowest P-value (rs295137) to each SNP in the plot is denoted in colors and was computed according to 1000GP June 2010 CEU data. Plot was created using LocusZoom [44].

### Figure 3. Summary of BDR by genotype of the SNP (rs295137) near SPATS2L with lowest P-value among all subjects in the primary populations.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5c91/3390407/18e2210c42b5/pgen.1002824.g003.jpg)

The TT genotype was associated with higher BDR (median 16.0; inter-quartile range (IQR) = [6.2, 32.4]), than the TC genotype (median 11.2; IQR = [5.2, 22.6]) or the CC genotype (median 10.3; IQR = [4.1, 21.7]).

### SPATS2L mRNA Expression Changes when the β2-Agonist Pathway Is Modified in Human Airway Smooth Muscle Cells

We found one public gene expression array experiment ([GSE13168](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE13168)) that would help to address the question of whether *SPATS2L* is differentially expressed in response to changes in the BDR pathway. We compared the levels of expression of two *SPATS2L* and one *SPATS2* probes in human airway smooth muscle (HASM) cells that stably expressed a PKA inhibitor vs. a GFP control at baseline and when stimulated with the pro-asthmatic cytokines interleukin-1β (IL1β), epidermal growth factor (EGF), or both. A trend of differential expression was observed for the *SPATS2* and one *SPATS2L* probes, but not a second *SPATS2L* probe ([Figure S5](#pgen.1002824.s005)). None of the comparisons met a Benjamini-Hochberg adjusted significance threshold, but nominally significant P-values were obtained for the *SPATS2* probe under all conditions and for one of the *SPATS2L* probes under the condition of EGF and IL1β stimulation ([Table S6](#pgen.1002824.s013)). According to the Gene Enrichment Profiler, the two *SPATS2L* probes are highly expressed in lung, and all three probes are highly expressed in smooth muscle, especially the *SPATS2* one. Overall, there are strikingly different tissue-specific expression patterns for each probe ([Figure S6](#pgen.1002824.s006)).

### Knockdown of SPATS2L mRNA Leads to Increased β2AR Levels

We further investigated the involvement of *SPATS2L* in the β_2_-adrenergic response pathway by knocking down *SPATS2L* mRNA using two different small interfering RNAs (siRNA) and measuring subsequent changes in β_2_AR protein levels. The knockdown efficiency of the siRNAs was >80% reduction of *SPATS2L* mRNA as measured by qRT-PCR, and the corresponding increases in β_2_AR (normalized against the control β-actin protein) levels were 1.88- (SD 0.41) and 1.86- (SD 0.30) fold for the two *SPATS2L* siRNAs ([Figure 4](#pgen-1002824-g004)).

### Figure 4. Effect of siRNA-mediated SPATS2L knockdown on β2-adrenergic receptor levels in HASM cells.

![Figure 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5c91/3390407/7f4480f2b2b4/pgen.1002824.g004.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3390407_pgen.1002824.g004.jpg)

(A) Knockdown efficiency of two SPATS2L siRNAs. Quantitative real-time PCR was done on RNAs extracted from HASM cells transfected with control non-targeting (NT) or SPATS2L-specific siRNAs. (B) Western blot analysis of β2AR protein in SPATS2L knockdown HASM cells. Three independent experiments (siRNA transfection and Western blot analysis) were done in HASM cells. The upper panel is a representative of the three Western blots. Quantification of the β2AR protein amount (normalized against the control β-actin protein) from three Western blots is shown in the lower panel.

### Primary GWAS Results in Previously Identified BDR Candidate Genes

The association of SNPs with BDR at SNPs in/near genes (i.e. *ADRB2*, *ADCY9*, *CRHR2*, *ARG1*) previously reported as being associated with BDR was measured in our primary GWAS imputed data ([Figure S7](#pgen.1002824.s007)). Nominally significant (P-value<0.05) SNPs were found in *ADCY9*, *CRHR2*, and *ARG1*, but not in *ADRB2* ([Tables S7](#pgen.1002824.s014) and [S8](#pgen.1002824.s015)). The SNPs with lowest P-values within 50 kb of these genes were: rs2531988 for *ADCY9* (3.2E-03), rs12533248 for *CRHR2* (0.029), and rs6929820 for *ARG1* (0.012).

## Discussion

In recent years, many GWAS that have successfully identified risk-modifying loci for a wide range of complex diseases have been published, but progress toward understanding how the loci and genes identified are functionally related to diseases has been slow [[19]](#pgen.1002824-Manolio1). The relationship of genes and gene variants to pharmacogenetic traits is often easier to test functionally than that for complex diseases because pharmacogenetic traits are more amenable to *in vitro* testing. However, compared to GWAS of complex diseases, GWAS of pharmacogenetic traits have been challenged by the relatively small size of drug clinical trials, which has caused many studies to be underpowered for obtaining genome-wide significant associations [[20]](#pgen.1002824-Daly1). Nonetheless, successful pharmacogenetic GWAS have led to the identification of loci involved in modulating response to inhaled corticosteroids among asthma patients [[21]](#pgen.1002824-Tantisira2), warfarin dose [[22]](#pgen.1002824-Takeuchi1), and lipid-lowering response to statins [[23]](#pgen.1002824-Barber1).

One of the difficulties specific to BDR GWAS is the complexity of the BDR phenotype. Regardless of how it is quantified, BDR is highly variable among asthma patients due to the time-dependent variation in baseline FEV_1_ and the influence of external environmental factors [[24]](#pgen.1002824-Sharma1). BDR can be quantified in various ways, with slightly varying resulting classification of patients as responders or non-responders. For our study, we selected the definition most widely utilized in clinical and human asthma research settings: percent change in baseline FEV_1_ following administration of a standard dose of short-acting inhaled β_2_-agonist [[25]](#pgen.1002824-American2). We have attempted to control for the known relationship between baseline lung function and BDR [[24]](#pgen.1002824-Sharma1), [[26]](#pgen.1002824-Waalkens1) by (1) selecting a definition of BDR that standardizes the change in FEV_1_ by dividing by baseline FEV_1_ and (2) by using age, sex, and height, which together account for a large portion of the variability in baseline lung function, as covariates in our statistical models. Because BDR tests are routinely performed during asthma clinical trials to use as inclusion criteria and to monitor outcomes among patients, we were able to utilize subjects from several diverse asthma clinical trials that were not specifically designed to study the pharmacogenetics of BDR. Most of these trials included a wash-out period that reduces modification of BDR due to concomitant medication administration, but LOCCS and some CARE and ACRN subjects were administered BDR tests at a time when they were not necessarily off of medications ([Table 1](#pgen-1002824-t001)). Subjects without a placebo washout, and especially those who were on ICS (e.g. LOCCS subjects), may be expected to have less BDR than those on placebo. The relationship of the magnitude of BDR to the various gene loci could therefore be blunted and show a less significant relationship than would be expected if all studies had incorporated a placebo washout.

In addition to variable washout periods, the cohorts had other significant differences in their design. Two trials consisted of children with asthma (i.e. CAMP, CARE), while the others consisted mostly of adults. The gender composition varied from 25% to 62% male. We attempted to control for age, gender and height, all of which are known to influence BDR, by including them as covariates in the association analysis. The mean and range of BDR also varied among trials. Of most significance, because the Sepracor trial used BDR greater than 15% as a criterion for inclusion, its subjects had markedly greater BDR than those of other trials. We attempted to control for this difference and any other trial specific differences among the cohorts that were pooled in the primary analysis by including trial as a covariate in the association model. There were additional differences among trials that were not taken into account. For example, SARP and Sepracor were composed of subjects with more severe asthma than those of other cohorts. Some ACRN, CARE, and SARP subjects were administered a different amount of albuterol during their BDR tests than those of CAMP, Sepracor, LOCCS, and LODO. DAG subjects were administered a different beta-agonist (i.e. Salbutamol) at a different concentration than that used with subjects of all other trials. DAG and SARP subjects were not participants of clinical trials, so there was greater heterogeneity of subjects within those cohorts. Despite the heterogeneity among trials, we utilized as many subjects as possible in an attempt to increase our statistical power to detect associations of SNPs with BDR. We reasoned that any associations detected despite the heterogeneity of the trial populations would be those most likely to generalize to all asthma patients. Another expected consequence of the trial heterogeneity is that our association results do not replicate in all cohorts. While having the largest number of subjects provides the greatest statistical power to detect statistically significant associations that are most generalizable across the clinical trials, we may be missing associations that are specific to the individual trials. For example, the subjects within clinical trials representing different ranges of asthma severity, age, and baseline characteristics may have genetic associations that are unique to subjects with their specific trial characteristics. The small sample size of each individual clinical trial makes detection of trial-specific associations more challenging. Despite the cohort heterogeneity, our meta-analysis identified a strong association that suggests a novel gene is involved in BDR.

Our top association was at SNP rs295137, with a combined P-value across all cohorts of 9.7E-07. This P-value does not meet conventional genome-wide significance thresholds (e.g. Bonferroni corrected minimally significant P-value would be 0.05/469,884 = 1.1E-07), but performing searches through public data sources and the fact that other pharmacogenetic GWAS have discovered biologically important results without genome-wide significant associations led us to pursue our top association further. The region of association surrounding rs295137 is in the 5′UTR of *SPATS2L* ([Figure 2](#pgen-1002824-g002)). This gene maps to chromosome 2 at 2q33.1, covering 176.78 kb from 201170592 to 201347368 (NCBI 37, August 2010). According to data gathered via the AceView [[27]](#pgen.1002824-ThierryMieg1) tool, *SPATS2L* is a complex locus that may have at least 30 spliced variants, its *in vivo* function is unknown, and it is a highly expressed gene in many tissues, with the greatest number of GenBank accessions belonging to lung. In gene-trap experiments in myoblasts, *SPATS2L* (a.k.a. *SGNP*) was found to be involved in ribosomal biogenesis and translational control in response to oxidative stress [[28]](#pgen.1002824-Zhu1).

The availability of one public expression array experiment that utilized HASM cells expressing a PKA inhibitor (PKI) to modify the β_2_-adrenergic pathway allowed us to perform a preliminary search for evidence that *SPATS2L* may be involved in BDR. We found that a probe for *SPATS2*, the paralog of *SPATS2L*, was significantly differentially expressed in PKI vs. control cells at baseline and when stimulated with pro-inflammatory cytokines (EGF, IL1β, or both). One *SPAST2L* probe followed this trend but had a nominally significant P-value only under the condition of stimulation with both EGF and IL1β, while the other *SPATS2L* probe did not exhibit any changes. As illustrated in [Figure S6](#pgen.1002824.s006), the tissue-specific expression patterns of the three probes varied widely. While all were expressed in smooth muscle, the *SPATS2* probe's relative expression in this tissue was markedly greater than that of the *SPATS2L* probes. Taken together, the expression patterns are consistent with tissue and isoform dependent changes in *SPATS2L* gene products. While the public dataset *SPATS2L* results were inconclusive based on the differences among probes, they suggested that *SPATS2L* expression may change when PKA is inhibited in HASM cells.

Knockdown of *SPATS2L* in HASM cells resulted in significantly increased β_2_AR protein levels, suggesting that *SPATS2L* may affect BDR by directly modulating β_2_AR protein expression. In HASM, β_2_-agonists exert their effects exclusively via the β_2_AR [[6]](#pgen.1002824-Shore1). The relaxation of HASM occurs after the binding of β_2_-agonists to β_2_ARs via increased levels of cAMP followed by PKA activation. PKA activation leads to changes in gene transcription via activation of cAMP response element binding protein (CREB). Because β_2_ARs are the gateway to the effects of β_2_-agonists in HASM cells, modulations, such as *SPATS2L* inactivation, that increase the levels of β_2_ARs in HASM cells may lead to both greater relaxation in response to β_2_-agonists in the short term and greater differences in gene transcription in the longer term. Further study is needed to elucidate the precise mechanism by which *SPATS2L* regulates β_2_AR and consequently modifies BDR. Among our primary GWAS subjects, those whose *SPATS2L* SNP rs295137 has the TT genotype have greater BDR than those with CT or TT genotypes (median BDR 16.0 vs. 10.9). In one of the simplest scenarios, it is possible that the increased BDR among subjects with the TT genotype results from this genotype playing a direct role in decreasing transcription of *SPATS2L*, which in turn results in increased β_2_AR levels. Further work is required to understand how specific SNP associations in/near *SPATS2L* affect *SPATS2L* function and/or expression and how such effects impact β_2_AR signaling and BDR. Because the observed influence of our most strongly associated SNP genotype on BDR is relatively small ([Figure 3](#pgen-1002824-g003)), our current data do not support the development of any personalized therapeutics based solely on variants in/near *SPATS2L*.

In addition to studying top primary GWAS SNPs, we attempted to replicate findings from previous BDR candidate gene association studies. Specifically, we measured association between BDR and *ADRB2* [[13]](#pgen.1002824-Drysdale1), [[14]](#pgen.1002824-Silverman1), *ADCY9* [[15]](#pgen.1002824-Tantisira1), *CRHR2* [[16]](#pgen.1002824-Poon1), and *ARG1* [[17]](#pgen.1002824-Litonjua1) variants. Notably, these previous findings are not entirely independent of those from the current GWAS: CAMP was a primary population utilized to identify associations in *ADRB2*, *ADCY9*, *CRHR2*, and *ARG1* in previous reports; LODO and Sepracor were replication populations in the *CRHR2*, and *ARG1* reports; and LOCCS was a replication population in the *ARG1* report. At a nominal significance level, we replicated gene-level associations for all of the candidate genes other than *ADRB2*. This gene, which encodes the β_2_AR, is the most studied gene related to BDR and SNPs and haplotypes in this gene have been related to decreased pulmonary function [[29]](#pgen.1002824-Israel1), response to β_2_-agonist treatment [[30]](#pgen.1002824-Israel2), an increased frequency of asthma exacerbations [[31]](#pgen.1002824-Taylor1), and BDR [[13]](#pgen.1002824-Drysdale1), [[14]](#pgen.1002824-Silverman1). Initial reports of *ADRB2* associations were very promising and suggested that variants of this candidate gene would be reliable markers of BDR pharmacogenetics. However, a meta-analysis of 21 studies of *ADRB2* polymorphisms found that most of the positive associations identified in individual studies cannot be compared to findings in other studies due to different study designs, phenotypes considered and selective reporting, making the number of variants with true replications very small and questioning the utility of *ADRB2* polymorphisms for generalizable pharmacogenetic tests [[32]](#pgen.1002824-ContopoulosIoannidis1). Our inability to find associations with *ADRB2* variants is consistent with the view that BDR genetics are complex: no individual SNPs or genes are responsible for a large proportion of BDR variability observed among all asthmatics. Our results suggest that genes other than the previously identified candidate genes are more strongly associated with BDR and that functional studies of these regions may yield important insights into BDR biology despite not having strong effects or generalizing to all populations.

In summary, a BDR GWAS among asthma patients from eight cohorts found that the most strongly associated SNP, rs295137, had a combined P-value of 9.7E-07. This association led us to *SPATS2L*, a gene of unknown *in vivo* function that we showed may be involved in BDR via the down-regulation of β_2_AR levels. Our results support the notion that there is promise in pursuing GWAS results that do not necessarily reach genome-wide significance and are an example of the way in which results from pharmacogenetic GWAS can be studied functionally.

## Materials and Methods

### Ethics Statement

Each study was approved by the Institutional Review Board of the corresponding institution, which ensured that all procedures followed were in accordance with the ethical standards of the responsible committee on human experimentation. Informed consent was obtained for all study participants.

### Subjects

The primary group of subjects consisted of 1,644 non-Hispanic white asthmatics from the following drug clinical trials: Childhood Asthma Management Program (CAMP) [[33]](#pgen.1002824-Childhood1), Leukotriene Modifier or Corticosteroid Salmeterol study (LOCCS) [[34]](#pgen.1002824-Peters1), Effectiveness of Low Dose Theophylline as an Add-on Treatment in Asthma trial (LODO) [[35]](#pgen.1002824-American3), a medication trial conducted by Sepracor, Inc. [[36]](#pgen.1002824-Silverman2), [[37]](#pgen.1002824-Silverman3), and subsets of clinical trials within the Childhood Asthma Research and Education (CARE) network [[38]](#pgen.1002824-Guilbert1), and the Asthma Clinical Research Network (ACRN) [[39]](#pgen.1002824-Denlinger1) participating in the NHLBI SNP Health Association Resource (SHARe) Asthma Resource project (SHARP). Some basic characteristics of these cohorts are in [Table 1](#pgen-1002824-t001) and further details are provided in [Text S1](#pgen.1002824.s016). BDR tests were performed according to American Thoracic Society criteria with Albuterol as the β_2_-agonist [[25]](#pgen.1002824-American2), unless otherwise noted. Baseline BDR measures were utilized, and BDR was quantified as the percent change in FEV_1_ in response to administration of a β_2_-agonist [i.e. (post-BD FEV_1_ – pre-BD FEV_1_)/pre-BD FEV_1_].

### Genotyping and Quality Control

Genome-wide genotyping for CAMP subjects (n = 546) was performed on the HumanHap550 Genotyping BeadChip or Infinium HD Human610-Quad BeadChip by Illumina, Inc (San Diego, CA) at the Channing Laboratory. LOCCS (n = 135), LODO (n = 114), and Sepracor (n = 401) subjects were genotyped at the Riken Center for Genomic Medicine using the Infinium HD Human610-Quad BeadChip. CARE (n = 207) and ACRN (n = 241) subjects were genotyped on Affymetrix 6.0 genotyping chip by Affymetrix, Inc. (Santa Clara, CA). Data from those subjects genotyped using Illumina technologies was combined into a primary dataset with 469,884 overlapping SNPs having missingness <1%, passing HWE (P-value threshold of 1E-03), and having minor allele frequency (MAF)>0.05. EIGENSTRAT was used to identify 23 outliers (not included in counts above) based on being outside of six standard deviations of the top four principal components during five iterations [[40]](#pgen.1002824-Price1). The genomic inflation factor (λ_GC_) of the remaining 1,196 subjects was 1.002, demonstrating minimal population stratification. CARE and ACRN dataset quality control (QC) also included the removal of four related subjects (i.e. CARE siblings), SNPs with MAF<0.05, missingness >5%, or not passing HWE based on a threshold of 1E-03. The λ_GC_ for CARE and ACRN genotype data were 1.02 and 0.98, demonstrating minimal population stratification among subjects within each group. Comprehensive genotyping and QC measures are provided in [Text S1](#pgen.1002824.s016).

### Statistical Analysis

Due to the poor overlap among SNPs genotyped on the Illumina and Affymetrix platforms, imputation of all SNPs available in HapMap Phase 2 Release 22 CEU data using the Markov Chain Haplotyping software (MaCH) [[41]](#pgen.1002824-Willer1) was performed for ACRN and CARE genotyped data. The primary GWAS consisted in the set of 469,884 SNPs that were successfully genotyped in those cohorts using Illumina arrays (i.e., CAMP/Sepracor/LOCCS/LODO) and imputed with HapMap Phase 2 data in those cohorts genotyped with Affymetrix arrays (i.e., ACRN and CARE) with a ratio of empirically observed dosage variance to the expected (binomial) dosage variance greater than 0.3, indicating good quality of imputation.

To expand the association results, imputation of all SNPs available in the June 2010 release of the 1000 Genome Project (1000GP) data using MaCH was performed for each of the three primary groups of genotype data. An overlapping set of 4,571,615 imputed SNPs had a MAF>0.05 and ratio of empirically observed dosage variance to the expected (binomial) dosage variance greater than 0.5, indicating good quality of imputation.

The association of SNPs with BDR was measured with a linear regression model as implemented in PLINK [[42]](#pgen.1002824-Purcell1) in the three sets of data: 1) CAMP/Sepracor/LOCCS/LODO, 2) ACRN, 3) CARE. Association of imputed SNPs was carried out using dosage data. Covariates for the CAMP/Sepracor/LOCCS/LODO group included age, gender, height, and study. Covariates for the CARE and ACRN groups included age, gender, and height. To get the primary GWAS results, CARE and ACRN P-values were combined with those of the CAMP/Sepracor/LOCCS/LODO group by using Liptak's combined probability method [[43]](#pgen.1002824-Liptak1) where hypothesis tests in CARE and ACRN had one-sided alternatives, based on the direction of the association in CAMP/Sepracor/LOCCS/LODO, so that SNPs with association tests in opposite directions would not produce inappropriately small P-values. The overall λ_GC_ was 1.002 in the primary set of GWAS results and 1.000 in the 1000GP imputed data GWAS.

Plots of association results near specific genes were created using LocusZoom with the hg18/1000 Genomes June 2010 CEU GenomeBuild/LD Population [[44]](#pgen.1002824-Pruim1).

### Replication Studies

#### 1) Severe Asthma Research Program (SARP)

Replication of primary GWAS P-values was attempted in 501 European American subjects from SARP who were recruited to meet the American Thoracic Society (ATS) definition of mild to severe asthmatics, with enrichment for severe asthma [[45]](#pgen.1002824-Moore1). Genotyping was performed on the Illumina Infinium HD Human 610-Quad Bead Chip at Wake Forest University. Tests of BDR association were performed using linear regression in R [[46]](#pgen.1002824-R1) with age, sex, and height as covariates.

#### 2) Dutch Asthma GWAS (DAG)

Replication of two SNPs near *SPATS2L* with lowest P-values in the primary GWAS was attempted in 550 DAG subjects with asthma and BDR data that were phenotyped in a single center (i.e, Beatrixoord, Haren, the Netherlands) [[18]](#pgen.1002824-Vonk1), [[47]](#pgen.1002824-Koppelman1). All patients refrained from bronchodilator use for at least 8 hours, and stopped inhaled steroids for 2 weeks before pulmonary function testing if clinically possible. Reversibility was assessed measuring spirometry before and 15 minutes after inhaling 800 µg of salbutamol. Genotyping was performed using the Hapmap 317K platform or Illumina 370 Duo Chip. Tests of BDR association were performed using linear regression, with age, gender and height as covariates using Predictive Analysis Software (PASW) version 18.0.

### Genome-Wide Gene Expression Data

The publicly available Gene Expression Omnibus (GEO) dataset, [GSE13168](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE13168), corresponding to an experiment in which human airway smooth muscle (HASM) cell cultures were generated from four donor trachea to test for the effects of glucocorticoids and PKA inhibition on the HASM transcriptome using the Affymetrix Human Genome U133A platform was used [[48]](#pgen.1002824-Misior1). We tested for the involvement of our top primary GWAS gene in the β_2_-adrenergic pathway by comparing the differential expression of genes in cells stably expressing a PKA inhibitor (PKI) vs. control at baseline and in the presence of pro-inflammatory cytokines interleukin-1β (IL1β), epidermal growth factor (EGF), or both. The expression array contained two *SPATS2L* probes (i.e., 215617_at, 222154_s_at) and one *SPATS2* probe (i.e., 218324_s_at). The probe for the paralog of *SPATS2L* was included to account for the possibility of non-specific binding of *SPATS2L* mRNA to the *SPATS2* probe. Analyses were conducted in R [[46]](#pgen.1002824-R1). Pre-processing of raw signal intensities was performed with RMA [[49]](#pgen.1002824-Gautier1) and differential expression was quantified using the limma package [[50]](#pgen.1002824-Smyth1). Tissue-specific expression of these probes was assessed using 557 microarrays from 126 human normal primary tissues in the Gene Enrichment Profiler [[51]](#pgen.1002824-Benita1).

### SPATS2L siRNA Knockdown and β2AR Western Blotting Analysis

Primary HASM cells were isolated from aborted lung transplant donors with no chronic illness. The tissue was obtained from the National Disease Resource Interchange (NDRI) and their use approved by the University of Pennsylvania IRB. HASM cell cultivation and characterization were described previously [[52]](#pgen.1002824-Panettieri1), [[53]](#pgen.1002824-Cooper1). Passages 4 to 7 HASM cells maintained in Ham's F12 medium supplemented with 10% FBS were used in all experiments. 2×10^5^ HASM cells were grown overnight and then transfected with 50 nM siRNA by using DharmaFECT 1 reagent (Thermo Scientific, Lafayette, CO, USA). About 72 h post transfection, cells were washed with PBS and lysed with NP-40 lysing buffer (50 mM Tris-HCl pH7.5, 150 mM NaCl, 0.5% Nonidet P-40) containing protease inhibitor cocktail (Roche Diagnostics Corporation, Indianapolis, IN, USA). Protein samples were denatured 10 min at 50°C, separated on NuPAGE 4–12% Bis-Tris gels (Invitrogen, Grand Island, NY, USA) and transferred to PVDF membranes (Bio-Rad Laboratories, Hercules, CA, USA). Immunoblot signals were developed using SuperSignal West Pico (Pierce Protein Research Products, Thermo Fisher Scientific, Rockford, IL, USA) and quantified by ImageJ software. Non-targeting control siRNA, *SPATS2L*-specific siRNA 1 (sense *5′- guc agu cca uug auu guc u*(dT)(dT)-3′, antisense *5′- aga caa uca aug gac uga c*(dT)(dT) -3′) and *SPATS2L*-specific siRNA 2 (sense *5′-caa ccu gug uug uag cag u*(dT)(dT)-3′, antisense *5′- acu gcu aca aca cag guu g*(dT)(dT) -3′) were obtained from Sigma-Aldrich (Mission siRNA; St. Louis, MO, USA). Antibodies for β_2_AR (H20) and β-actin were from Santa Cruz Biotechnology, Inc. (Santa Cruz, CA, USA). Experiments were performed in triplicate.

## Supporting Information

## Acknowledgments

We thank all CAMP subjects for their ongoing participation in this study. We acknowledge the CAMP investigators and research team, supported by the National Heart, Lung, and Blood Institute (NHLBI), for collection of CAMP Genetic Ancillary Study data. All work on data collected from the CAMP Genetic Ancillary Study was conducted at the Channing Laboratory of the Brigham and Women's Hospital under appropriate CAMP policies and human subject's protections. We thank the American Lung Association Asthma Clinical Research Centers (ALA-ACRC) for use of LOCCS and LODO study samples.

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. American Lung Association. Trends in Asthma Morbidity and Mortality. New York, NY: Epidemiology and Statistics Unit, Research and Program Services, American Lung Association; 2006.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Trends%20in%20Asthma%20Morbidity%20and%20Mortality&publication_year=2006&)

2. Global Initiative for Asthma Management and Prevention. NHLBI/WHO Workshop Report. National Institutes of Health, Bethesda: US Department of Health and Human Services; 1995.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=NHLBI/WHO%20Workshop%20Report&publication_year=1995&)

3. Nelson HS. Beta-adrenergic bronchodilators. N Engl J Med. 1995;333:499–506. doi: 10.1056/NEJM199508243330807.  [DOI](https://doi.org/10.1056/NEJM199508243330807) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7623883/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Beta-adrenergic%20bronchodilators.&author=HS%20Nelson&volume=333&publication_year=1995&pages=499-506&pmid=7623883&doi=10.1056/NEJM199508243330807&)

4. Loza MJ, Penn RB. Regulation of T cells in airway disease by beta-agonist. Front Biosci (Schol Ed) 2010;2:969–979. doi: 10.2741/s113.  [DOI](https://doi.org/10.2741/s113) | [PMC free article](/articles/PMC2956062/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20515836/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front%20Biosci%20(Schol%20Ed)&title=Regulation%20of%20T%20cells%20in%20airway%20disease%20by%20beta-agonist.&author=MJ%20Loza&author=RB%20Penn&volume=2&publication_year=2010&pages=969-979&pmid=20515836&doi=10.2741/s113&)

5. Salathe M. Effects of beta-agonists on airway epithelial cells. J Allergy Clin Immunol. 2002;110:S275–281. doi: 10.1067/mai.2002.129412.  [DOI](https://doi.org/10.1067/mai.2002.129412) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12464936/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Allergy%20Clin%20Immunol&title=Effects%20of%20beta-agonists%20on%20airway%20epithelial%20cells.&author=M%20Salathe&volume=110&publication_year=2002&pages=S275-281&pmid=12464936&doi=10.1067/mai.2002.129412&)

6. Shore SA, Moore PE. Regulation of beta-adrenergic responses in airway smooth muscle. Respir Physiol Neurobiol. 2003;137:179–195. doi: 10.1016/s1569-9048(03)00146-0.  [DOI](https://doi.org/10.1016/s1569-9048(03)00146-0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14516725/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Respir%20Physiol%20Neurobiol&title=Regulation%20of%20beta-adrenergic%20responses%20in%20airway%20smooth%20muscle.&author=SA%20Shore&author=PE%20Moore&volume=137&publication_year=2003&pages=179-195&pmid=14516725&doi=10.1016/s1569-9048(03)00146-0&)

7. Jartti T. Asthma, asthma medication and autonomic nervous system dysfunction. Clin Physiol. 2001;21:260–269. doi: 10.1046/j.1365-2281.2001.00323.x.  [DOI](https://doi.org/10.1046/j.1365-2281.2001.00323.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11318835/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Physiol&title=Asthma,%20asthma%20medication%20and%20autonomic%20nervous%20system%20dysfunction.&author=T%20Jartti&volume=21&publication_year=2001&pages=260-269&pmid=11318835&doi=10.1046/j.1365-2281.2001.00323.x&)

8. Drazen JM, Silverman EK, Lee TH. Heterogeneity of therapeutic responses in asthma. Br Med Bull. 2000;56:1054–1070. doi: 10.1258/0007142001903535.  [DOI](https://doi.org/10.1258/0007142001903535) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11359637/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20Med%20Bull&title=Heterogeneity%20of%20therapeutic%20responses%20in%20asthma.&author=JM%20Drazen&author=EK%20Silverman&author=TH%20Lee&volume=56&publication_year=2000&pages=1054-1070&pmid=11359637&doi=10.1258/0007142001903535&)

9. Moffatt MF, Gut IG, Demenais F, Strachan DP, Bouzigon E, et al. A large-scale, consortium-based genomewide association study of asthma. N Engl J Med. 2010;363:1211–1221. doi: 10.1056/NEJMoa0906312.  [DOI](https://doi.org/10.1056/NEJMoa0906312) | [PMC free article](/articles/PMC4260321/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20860503/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=A%20large-scale,%20consortium-based%20genomewide%20association%20study%20of%20asthma.&author=MF%20Moffatt&author=IG%20Gut&author=F%20Demenais&author=DP%20Strachan&author=E%20Bouzigon&volume=363&publication_year=2010&pages=1211-1221&pmid=20860503&doi=10.1056/NEJMoa0906312&)

10. Torgerson DG, Ampleford EJ, Chiu GY, Gauderman WJ, Gignoux CR, et al. Meta-analysis of genome-wide association studies of asthma in ethnically diverse North American populations. Nat Genet. 2011;43:887–892. doi: 10.1038/ng.888.  [DOI](https://doi.org/10.1038/ng.888) | [PMC free article](/articles/PMC3445408/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21804549/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Meta-analysis%20of%20genome-wide%20association%20studies%20of%20asthma%20in%20ethnically%20diverse%20North%20American%20populations.&author=DG%20Torgerson&author=EJ%20Ampleford&author=GY%20Chiu&author=WJ%20Gauderman&author=CR%20Gignoux&volume=43&publication_year=2011&pages=887-892&pmid=21804549&doi=10.1038/ng.888&)

11. Niu T, Rogus JJ, Chen C, Wang B, Yang J, et al. Familial aggregation of bronchodilator response: a community-based study. Am J Respir Crit Care Med. 2000;162:1833–1837. doi: 10.1164/ajrccm.162.5.9908127.  [DOI](https://doi.org/10.1164/ajrccm.162.5.9908127) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11069822/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Respir%20Crit%20Care%20Med&title=Familial%20aggregation%20of%20bronchodilator%20response:%20a%20community-based%20study.&author=T%20Niu&author=JJ%20Rogus&author=C%20Chen&author=B%20Wang&author=J%20Yang&volume=162&publication_year=2000&pages=1833-1837&pmid=11069822&doi=10.1164/ajrccm.162.5.9908127&)

12. Hawkins GA, Weiss ST, Bleecker ER. Clinical consequences of ADRbeta2 polymorphisms. Pharmacogenomics. 2008;9:349–358. doi: 10.2217/14622416.9.3.349.  [DOI](https://doi.org/10.2217/14622416.9.3.349) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18303970/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Clinical%20consequences%20of%20ADRbeta2%20polymorphisms.&author=GA%20Hawkins&author=ST%20Weiss&author=ER%20Bleecker&volume=9&publication_year=2008&pages=349-358&pmid=18303970&doi=10.2217/14622416.9.3.349&)

13. Drysdale CM, McGraw DW, Stack CB, Stephens JC, Judson RS, et al. Complex promoter and coding region beta 2-adrenergic receptor haplotypes alter receptor expression and predict in vivo responsiveness. Proc Natl Acad Sci U S A. 2000;97:10483–10488. doi: 10.1073/pnas.97.19.10483.  [DOI](https://doi.org/10.1073/pnas.97.19.10483) | [PMC free article](/articles/PMC27050/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10984540/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&title=Complex%20promoter%20and%20coding%20region%20beta%202-adrenergic%20receptor%20haplotypes%20alter%20receptor%20expression%20and%20predict%20in%20vivo%20responsiveness.&author=CM%20Drysdale&author=DW%20McGraw&author=CB%20Stack&author=JC%20Stephens&author=RS%20Judson&volume=97&publication_year=2000&pages=10483-10488&pmid=10984540&doi=10.1073/pnas.97.19.10483&)

14. Silverman EK, Kwiatkowski DJ, Sylvia JS, Lazarus R, Drazen JM, et al. Family-based association analysis of beta2-adrenergic receptor polymorphisms in the childhood asthma management program. J Allergy Clin Immunol. 2003;112:870–876. doi: 10.1016/s0091-6749(03)02023-2.  [DOI](https://doi.org/10.1016/s0091-6749(03)02023-2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14610472/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Allergy%20Clin%20Immunol&title=Family-based%20association%20analysis%20of%20beta2-adrenergic%20receptor%20polymorphisms%20in%20the%20childhood%20asthma%20management%20program.&author=EK%20Silverman&author=DJ%20Kwiatkowski&author=JS%20Sylvia&author=R%20Lazarus&author=JM%20Drazen&volume=112&publication_year=2003&pages=870-876&pmid=14610472&doi=10.1016/s0091-6749(03)02023-2&)

15. Tantisira KG, Small KM, Litonjua AA, Weiss ST, Liggett SB. Molecular properties and pharmacogenetics of a polymorphism of adenylyl cyclase type 9 in asthma: interaction between beta-agonist and corticosteroid pathways. Hum Mol Genet. 2005;14:1671–1677. doi: 10.1093/hmg/ddi175.  [DOI](https://doi.org/10.1093/hmg/ddi175) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15879435/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mol%20Genet&title=Molecular%20properties%20and%20pharmacogenetics%20of%20a%20polymorphism%20of%20adenylyl%20cyclase%20type%209%20in%20asthma:%20interaction%20between%20beta-agonist%20and%20corticosteroid%20pathways.&author=KG%20Tantisira&author=KM%20Small&author=AA%20Litonjua&author=ST%20Weiss&author=SB%20Liggett&volume=14&publication_year=2005&pages=1671-1677&pmid=15879435&doi=10.1093/hmg/ddi175&)

16. Poon AH, Tantisira KG, Litonjua AA, Lazarus R, Xu J, et al. Association of corticotropin-releasing hormone receptor-2 genetic variants with acute bronchodilator response in asthma. Pharmacogenet Genomics. 2008;18:373–382. doi: 10.1097/FPC.0b013e3282fa760a.  [DOI](https://doi.org/10.1097/FPC.0b013e3282fa760a) | [PMC free article](/articles/PMC3208318/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18408560/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Association%20of%20corticotropin-releasing%20hormone%20receptor-2%20genetic%20variants%20with%20acute%20bronchodilator%20response%20in%20asthma.&author=AH%20Poon&author=KG%20Tantisira&author=AA%20Litonjua&author=R%20Lazarus&author=J%20Xu&volume=18&publication_year=2008&pages=373-382&pmid=18408560&doi=10.1097/FPC.0b013e3282fa760a&)

17. Litonjua AA, Lasky-Su J, Schneiter K, Tantisira KG, Lazarus R, et al. ARG1 is a novel bronchodilator response gene: screening and replication in four asthma cohorts. Am J Respir Crit Care Med. 2008;178:688–694. doi: 10.1164/rccm.200709-1363OC.  [DOI](https://doi.org/10.1164/rccm.200709-1363OC) | [PMC free article](/articles/PMC2556451/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18617639/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Respir%20Crit%20Care%20Med&title=ARG1%20is%20a%20novel%20bronchodilator%20response%20gene:%20screening%20and%20replication%20in%20four%20asthma%20cohorts.&author=AA%20Litonjua&author=J%20Lasky-Su&author=K%20Schneiter&author=KG%20Tantisira&author=R%20Lazarus&volume=178&publication_year=2008&pages=688-694&pmid=18617639&doi=10.1164/rccm.200709-1363OC&)

18. Vonk JM, Postma DS, Maarsingh H, Bruinenberg M, Koppelman GH, et al. Arginase 1 and arginase 2 variations associate with asthma, asthma severity and beta2 agonist and steroid response. Pharmacogenet Genomics. 2010;20:179–186. doi: 10.1097/FPC.0b013e328336c7fd.  [DOI](https://doi.org/10.1097/FPC.0b013e328336c7fd) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20124949/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Arginase%201%20and%20arginase%202%20variations%20associate%20with%20asthma,%20asthma%20severity%20and%20beta2%20agonist%20and%20steroid%20response.&author=JM%20Vonk&author=DS%20Postma&author=H%20Maarsingh&author=M%20Bruinenberg&author=GH%20Koppelman&volume=20&publication_year=2010&pages=179-186&pmid=20124949&doi=10.1097/FPC.0b013e328336c7fd&)

19. Manolio TA. Genomewide association studies and assessment of the risk of disease. N Engl J Med. 2010;363:166–176. doi: 10.1056/NEJMra0905980.  [DOI](https://doi.org/10.1056/NEJMra0905980) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20647212/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Genomewide%20association%20studies%20and%20assessment%20of%20the%20risk%20of%20disease.&author=TA%20Manolio&volume=363&publication_year=2010&pages=166-176&pmid=20647212&doi=10.1056/NEJMra0905980&)

20. Daly AK. Genome-wide association studies in pharmacogenomics. Nat Rev Genet. 2010;11:241–246. doi: 10.1038/nrg2751.  [DOI](https://doi.org/10.1038/nrg2751) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20300088/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Genet&title=Genome-wide%20association%20studies%20in%20pharmacogenomics.&author=AK%20Daly&volume=11&publication_year=2010&pages=241-246&pmid=20300088&doi=10.1038/nrg2751&)

21. Tantisira KG, Lasky-Su J, Harada M, Murphy A, Litonjua AA, et al. Genomewide association between GLCCI1 and response to glucocorticoid therapy in asthma. N Engl J Med. 2011;365:1173–1183. doi: 10.1056/NEJMoa0911353.  [DOI](https://doi.org/10.1056/NEJMoa0911353) | [PMC free article](/articles/PMC3667396/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21991891/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Genomewide%20association%20between%20GLCCI1%20and%20response%20to%20glucocorticoid%20therapy%20in%20asthma.&author=KG%20Tantisira&author=J%20Lasky-Su&author=M%20Harada&author=A%20Murphy&author=AA%20Litonjua&volume=365&publication_year=2011&pages=1173-1183&pmid=21991891&doi=10.1056/NEJMoa0911353&)

22. Takeuchi F, McGinnis R, Bourgeois S, Barnes C, Eriksson N, et al. A genome-wide association study confirms VKORC1, CYP2C9, and CYP4F2 as principal genetic determinants of warfarin dose. PLoS Genet. 2009;5:e1000433. doi: 10.1371/journal.pgen.1000433. doi: 10.1371/journal.pgen.1000433.  [DOI](http://dx.doi.org/10.1371/journal.pgen.1000433) | [DOI](https://doi.org/10.1371/journal.pgen.1000433) | [PMC free article](/articles/PMC2652833/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19300499/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet&title=A%20genome-wide%20association%20study%20confirms%20VKORC1,%20CYP2C9,%20and%20CYP4F2%20as%20principal%20genetic%20determinants%20of%20warfarin%20dose.&author=F%20Takeuchi&author=R%20McGinnis&author=S%20Bourgeois&author=C%20Barnes&author=N%20Eriksson&volume=5&publication_year=2009&pages=e1000433&pmid=19300499&doi=10.1371/journal.pgen.1000433&)

23. Barber MJ, Mangravite LM, Hyde CL, Chasman DI, Smith JD, et al. Genome-wide association of lipid-lowering response to statins in combined study populations. PLoS ONE. 2010;5:e9763. doi: 10.1371/journal.pone.0009763. doi: 10.1371/journal.pone.0009763.  [DOI](http://dx.doi.org/10.1371/journal.pone.0009763) | [DOI](https://doi.org/10.1371/journal.pone.0009763) | [PMC free article](/articles/PMC2842298/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20339536/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&title=Genome-wide%20association%20of%20lipid-lowering%20response%20to%20statins%20in%20combined%20study%20populations.&author=MJ%20Barber&author=LM%20Mangravite&author=CL%20Hyde&author=DI%20Chasman&author=JD%20Smith&volume=5&publication_year=2010&pages=e9763&pmid=20339536&doi=10.1371/journal.pone.0009763&)

24. Sharma S, Litonjua AA, Tantisira KG, Fuhlbrigge AL, Szefler SJ, et al. Clinical predictors and outcomes of consistent bronchodilator response in the childhood asthma management program. J Allergy Clin Immunol. 2008;122:921–928 e924. doi: 10.1016/j.jaci.2008.09.004.  [DOI](https://doi.org/10.1016/j.jaci.2008.09.004) | [PMC free article](/articles/PMC2947830/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18848350/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Allergy%20Clin%20Immunol&title=Clinical%20predictors%20and%20outcomes%20of%20consistent%20bronchodilator%20response%20in%20the%20childhood%20asthma%20management%20program.&author=S%20Sharma&author=AA%20Litonjua&author=KG%20Tantisira&author=AL%20Fuhlbrigge&author=SJ%20Szefler&volume=122&publication_year=2008&pages=921-928%20e924&pmid=18848350&doi=10.1016/j.jaci.2008.09.004&)

25. American Thoracic Society. Lung function testing: selection of reference values and interpretative strategies. Am Rev Respir Dis. 1991;144:1202–1218. doi: 10.1164/ajrccm/144.5.1202.  [DOI](https://doi.org/10.1164/ajrccm/144.5.1202) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/1952453/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20Rev%20Respir%20Dis&title=Lung%20function%20testing:%20selection%20of%20reference%20values%20and%20interpretative%20strategies.&volume=144&publication_year=1991&pages=1202-1218&pmid=1952453&doi=10.1164/ajrccm/144.5.1202&)

26. Waalkens HJ, Merkus PJ, van Essen-Zandvliet EE, Brand PL, Gerritsen J, et al. Assessment of bronchodilator response in children with asthma. Dutch CNSLD Study Group. Eur Respir J. 1993;6:645–651.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/8519373/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20Respir%20J&title=Assessment%20of%20bronchodilator%20response%20in%20children%20with%20asthma.%20Dutch%20CNSLD%20Study%20Group.&author=HJ%20Waalkens&author=PJ%20Merkus&author=EE%20van%20Essen-Zandvliet&author=PL%20Brand&author=J%20Gerritsen&volume=6&publication_year=1993&pages=645-651&pmid=8519373&)

27. Thierry-Mieg D, Thierry-Mieg J. AceView: a comprehensive cDNA-supported gene and transcripts annotation. Genome Biol. 2006;7(Suppl 1):S12 11–14. doi: 10.1186/gb-2006-7-s1-s12.  [DOI](https://doi.org/10.1186/gb-2006-7-s1-s12) | [PMC free article](/articles/PMC1810549/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16925834/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Biol&title=AceView:%20a%20comprehensive%20cDNA-supported%20gene%20and%20transcripts%20annotation.&author=D%20Thierry-Mieg&author=J%20Thierry-Mieg&volume=7&issue=Suppl%201&publication_year=2006&pages=S12%2011-14&pmid=16925834&doi=10.1186/gb-2006-7-s1-s12&)

28. Zhu CH, Kim J, Shay JW, Wright WE. SGNP: an essential Stress Granule/Nucleolar Protein potentially involved in 5.8s rRNA processing/transport. PLoS ONE. 2008;3:e3716. doi: 10.1371/journal.pone.0003716. doi: 10.1371/journal.pone.0003716.  [DOI](http://dx.doi.org/10.1371/journal.pone.0003716) | [DOI](https://doi.org/10.1371/journal.pone.0003716) | [PMC free article](/articles/PMC2579992/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19005571/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&title=SGNP:%20an%20essential%20Stress%20Granule/Nucleolar%20Protein%20potentially%20involved%20in%205.8s%20rRNA%20processing/transport.&author=CH%20Zhu&author=J%20Kim&author=JW%20Shay&author=WE%20Wright&volume=3&publication_year=2008&pages=e3716&pmid=19005571&doi=10.1371/journal.pone.0003716&)

29. Israel E, Drazen JM, Liggett SB, Boushey HA, Cherniack RM, et al. The effect of polymorphisms of the beta(2)-adrenergic receptor on the response to regular use of albuterol in asthma. Am J Respir Crit Care Med. 2000;162:75–80. doi: 10.1164/ajrccm.162.1.9907092.  [DOI](https://doi.org/10.1164/ajrccm.162.1.9907092) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10903223/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Respir%20Crit%20Care%20Med&title=The%20effect%20of%20polymorphisms%20of%20the%20beta(2)-adrenergic%20receptor%20on%20the%20response%20to%20regular%20use%20of%20albuterol%20in%20asthma.&author=E%20Israel&author=JM%20Drazen&author=SB%20Liggett&author=HA%20Boushey&author=RM%20Cherniack&volume=162&publication_year=2000&pages=75-80&pmid=10903223&doi=10.1164/ajrccm.162.1.9907092&)

30. Israel E, Chinchilli VM, Ford JG, Boushey HA, Cherniack R, et al. Use of regularly scheduled albuterol treatment in asthma: genotype-stratified, randomised, placebo-controlled cross-over trial. Lancet. 2004;364:1505–1512. doi: 10.1016/S0140-6736(04)17273-5.  [DOI](https://doi.org/10.1016/S0140-6736(04)17273-5) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15500895/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Use%20of%20regularly%20scheduled%20albuterol%20treatment%20in%20asthma:%20genotype-stratified,%20randomised,%20placebo-controlled%20cross-over%20trial.&author=E%20Israel&author=VM%20Chinchilli&author=JG%20Ford&author=HA%20Boushey&author=R%20Cherniack&volume=364&publication_year=2004&pages=1505-1512&pmid=15500895&doi=10.1016/S0140-6736(04)17273-5&)

31. Taylor DR, Drazen JM, Herbison GP, Yandava CN, Hancox RJ, et al. Asthma exacerbations during long term beta agonist use: influence of beta(2) adrenoceptor polymorphism. Thorax. 2000;55:762–767. doi: 10.1136/thorax.55.9.762.  [DOI](https://doi.org/10.1136/thorax.55.9.762) | [PMC free article](/articles/PMC1745862/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10950895/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Thorax&title=Asthma%20exacerbations%20during%20long%20term%20beta%20agonist%20use:%20influence%20of%20beta(2)%20adrenoceptor%20polymorphism.&author=DR%20Taylor&author=JM%20Drazen&author=GP%20Herbison&author=CN%20Yandava&author=RJ%20Hancox&volume=55&publication_year=2000&pages=762-767&pmid=10950895&doi=10.1136/thorax.55.9.762&)

32. Contopoulos-Ioannidis DG, Alexiou GA, Gouvias TC, Ioannidis JP. An empirical evaluation of multifarious outcomes in pharmacogenetics: beta-2 adrenoceptor gene polymorphisms in asthma treatment. Pharmacogenet Genomics. 2006;16:705–711. doi: 10.1097/01.fpc.0000236332.11304.8f.  [DOI](https://doi.org/10.1097/01.fpc.0000236332.11304.8f) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17001289/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=An%20empirical%20evaluation%20of%20multifarious%20outcomes%20in%20pharmacogenetics:%20beta-2%20adrenoceptor%20gene%20polymorphisms%20in%20asthma%20treatment.&author=DG%20Contopoulos-Ioannidis&author=GA%20Alexiou&author=TC%20Gouvias&author=JP%20Ioannidis&volume=16&publication_year=2006&pages=705-711&pmid=17001289&doi=10.1097/01.fpc.0000236332.11304.8f&)

33. Childhood Asthma Management Program Research Group. The Childhood Asthma Management Program (CAMP): design, rationale, and methods. Control Clin Trials. 1999;20:91–120.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/10027502/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Control%20Clin%20Trials&title=The%20Childhood%20Asthma%20Management%20Program%20(CAMP):%20design,%20rationale,%20and%20methods.&volume=20&publication_year=1999&pages=91-120&pmid=10027502&)

34. Peters SP, Anthonisen N, Castro M, Holbrook JT, Irvin CG, et al. Randomized comparison of strategies for reducing treatment in mild persistent asthma. N Engl J Med. 2007;356:2027–2039. doi: 10.1056/NEJMoa070013.  [DOI](https://doi.org/10.1056/NEJMoa070013) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17507702/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Randomized%20comparison%20of%20strategies%20for%20reducing%20treatment%20in%20mild%20persistent%20asthma.&author=SP%20Peters&author=N%20Anthonisen&author=M%20Castro&author=JT%20Holbrook&author=CG%20Irvin&volume=356&publication_year=2007&pages=2027-2039&pmid=17507702&doi=10.1056/NEJMoa070013&)

35. American Lung Association Asthma Clinical Research Centers. Clinical trial of low-dose theophylline and montelukast in patients with poorly controlled asthma. Am J Respir Crit Care Med. 2007;175:235–242. doi: 10.1164/rccm.200603-416OC.  [DOI](https://doi.org/10.1164/rccm.200603-416OC) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16998094/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Respir%20Crit%20Care%20Med&title=Clinical%20trial%20of%20low-dose%20theophylline%20and%20montelukast%20in%20patients%20with%20poorly%20controlled%20asthma.&volume=175&publication_year=2007&pages=235-242&pmid=16998094&doi=10.1164/rccm.200603-416OC&)

36. Silverman ES, Palmer LJ, Subramaniam V, Hallock A, Mathew S, et al. Transforming growth factor-beta1 promoter polymorphism C-509T is associated with asthma. Am J Respir Crit Care Med. 2004;169:214–219. doi: 10.1164/rccm.200307-973OC.  [DOI](https://doi.org/10.1164/rccm.200307-973OC) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14597484/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Respir%20Crit%20Care%20Med&title=Transforming%20growth%20factor-beta1%20promoter%20polymorphism%20C-509T%20is%20associated%20with%20asthma.&author=ES%20Silverman&author=LJ%20Palmer&author=V%20Subramaniam&author=A%20Hallock&author=S%20Mathew&volume=169&publication_year=2004&pages=214-219&pmid=14597484&doi=10.1164/rccm.200307-973OC&)

37. Silverman ES, Baron RM, Palmer LJ, Le L, Hallock A, et al. Constitutive and cytokine-induced expression of the ETS transcription factor ESE-3 in the lung. Am J Respir Cell Mol Biol. 2002;27:697–704. doi: 10.1165/rcmb.2002-0011OC.  [DOI](https://doi.org/10.1165/rcmb.2002-0011OC) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12444029/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Respir%20Cell%20Mol%20Biol&title=Constitutive%20and%20cytokine-induced%20expression%20of%20the%20ETS%20transcription%20factor%20ESE-3%20in%20the%20lung.&author=ES%20Silverman&author=RM%20Baron&author=LJ%20Palmer&author=L%20Le&author=A%20Hallock&volume=27&publication_year=2002&pages=697-704&pmid=12444029&doi=10.1165/rcmb.2002-0011OC&)

38. Guilbert TW, Morgan WJ, Krawiec M, Lemanske RF, Jr, Sorkness C, et al. The Prevention of Early Asthma in Kids study: design, rationale and methods for the Childhood Asthma Research and Education network. Control Clin Trials. 2004;25:286–310. doi: 10.1016/j.cct.2004.03.002.  [DOI](https://doi.org/10.1016/j.cct.2004.03.002) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15157730/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Control%20Clin%20Trials&title=The%20Prevention%20of%20Early%20Asthma%20in%20Kids%20study:%20design,%20rationale%20and%20methods%20for%20the%20Childhood%20Asthma%20Research%20and%20Education%20network.&author=TW%20Guilbert&author=WJ%20Morgan&author=M%20Krawiec&author=RF%20Lemanske&author=C%20Sorkness&volume=25&publication_year=2004&pages=286-310&pmid=15157730&doi=10.1016/j.cct.2004.03.002&)

39. Denlinger LC, Sorkness CA, Chinchilli VM, Lemanske RF., Jr Guideline-defining asthma clinical trials of the National Heart, Lung, and Blood Institute's Asthma Clinical Research Network and Childhood Asthma Research and Education Network. J Allergy Clin Immunol. 2007;119:3–11; quiz 12–13. doi: 10.1016/j.jaci.2006.10.015.  [DOI](https://doi.org/10.1016/j.jaci.2006.10.015) | [PMC free article](/articles/PMC2742212/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17141853/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Allergy%20Clin%20Immunol&title=Guideline-defining%20asthma%20clinical%20trials%20of%20the%20National%20Heart,%20Lung,%20and%20Blood%20Institute's%20Asthma%20Clinical%20Research%20Network%20and%20Childhood%20Asthma%20Research%20and%20Education%20Network.&author=LC%20Denlinger&author=CA%20Sorkness&author=VM%20Chinchilli&author=RF%20Lemanske&volume=119&publication_year=2007&pages=3-11;%20quiz%2012%E2%80%9313&pmid=17141853&doi=10.1016/j.jaci.2006.10.015&)

40. Price AL, Patterson NJ, Plenge RM, Weinblatt ME, Shadick NA, et al. Principal components analysis corrects for stratification in genome-wide association studies. Nat Genet. 2006;38:904–909. doi: 10.1038/ng1847.  [DOI](https://doi.org/10.1038/ng1847) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16862161/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Principal%20components%20analysis%20corrects%20for%20stratification%20in%20genome-wide%20association%20studies.&author=AL%20Price&author=NJ%20Patterson&author=RM%20Plenge&author=ME%20Weinblatt&author=NA%20Shadick&volume=38&publication_year=2006&pages=904-909&pmid=16862161&doi=10.1038/ng1847&)

41. Willer CJ, Sanna S, Jackson AU, Scuteri A, Bonnycastle LL, et al. Newly identified loci that influence lipid concentrations and risk of coronary artery disease. Nat Genet. 2008;40:161–169. doi: 10.1038/ng.76.  [DOI](https://doi.org/10.1038/ng.76) | [PMC free article](/articles/PMC5206900/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18193043/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Newly%20identified%20loci%20that%20influence%20lipid%20concentrations%20and%20risk%20of%20coronary%20artery%20disease.&author=CJ%20Willer&author=S%20Sanna&author=AU%20Jackson&author=A%20Scuteri&author=LL%20Bonnycastle&volume=40&publication_year=2008&pages=161-169&pmid=18193043&doi=10.1038/ng.76&)

42. Purcell S, Neale B, Todd-Brown K, Thomas L, Ferreira MA, et al. PLINK: a tool set for whole-genome association and population-based linkage analyses. Am J Hum Genet. 2007;81:559–575. doi: 10.1086/519795.  [DOI](https://doi.org/10.1086/519795) | [PMC free article](/articles/PMC1950838/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17701901/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=PLINK:%20a%20tool%20set%20for%20whole-genome%20association%20and%20population-based%20linkage%20analyses.&author=S%20Purcell&author=B%20Neale&author=K%20Todd-Brown&author=L%20Thomas&author=MA%20Ferreira&volume=81&publication_year=2007&pages=559-575&pmid=17701901&doi=10.1086/519795&)

43. Liptak T. On the combination of independent tests. Magyar Tud Akad Mat Kutato Int Kozl. 1958;3:171–197.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Magyar%20Tud%20Akad%20Mat%20Kutato%20Int%20Kozl&title=On%20the%20combination%20of%20independent%20tests.&author=T%20Liptak&volume=3&publication_year=1958&pages=171-197&)

44. Pruim RJ, Welch RP, Sanna S, Teslovich TM, Chines PS, et al. LocusZoom: regional visualization of genome-wide association scan results. Bioinformatics. 2010;26:2336–2337. doi: 10.1093/bioinformatics/btq419.  [DOI](https://doi.org/10.1093/bioinformatics/btq419) | [PMC free article](/articles/PMC2935401/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20634204/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=LocusZoom:%20regional%20visualization%20of%20genome-wide%20association%20scan%20results.&author=RJ%20Pruim&author=RP%20Welch&author=S%20Sanna&author=TM%20Teslovich&author=PS%20Chines&volume=26&publication_year=2010&pages=2336-2337&pmid=20634204&doi=10.1093/bioinformatics/btq419&)

45. Moore WC, Bleecker ER, Curran-Everett D, Erzurum SC, Ameredes BT, et al. Characterization of the severe asthma phenotype by the National Heart, Lung, and Blood Institute's Severe Asthma Research Program. J Allergy Clin Immunol. 2007;119:405–413. doi: 10.1016/j.jaci.2006.11.639.  [DOI](https://doi.org/10.1016/j.jaci.2006.11.639) | [PMC free article](/articles/PMC2837934/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17291857/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Allergy%20Clin%20Immunol&title=Characterization%20of%20the%20severe%20asthma%20phenotype%20by%20the%20National%20Heart,%20Lung,%20and%20Blood%20Institute's%20Severe%20Asthma%20Research%20Program.&author=WC%20Moore&author=ER%20Bleecker&author=D%20Curran-Everett&author=SC%20Erzurum&author=BT%20Ameredes&volume=119&publication_year=2007&pages=405-413&pmid=17291857&doi=10.1016/j.jaci.2006.11.639&)

46. R Development Core Team. R: A Language and Environment for Statistical Computing. Vienna, Austria: R Foundation for Statistical Computing; 2008.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=R:%20A%20Language%20and%20Environment%20for%20Statistical%20Computing&publication_year=2008&)

47. Koppelman GH, Meyers DA, Howard TD, Zheng SL, Hawkins GA, et al. Identification of PCDH1 as a novel susceptibility gene for bronchial hyperresponsiveness. Am J Respir Crit Care Med. 2009;180:929–935. doi: 10.1164/rccm.200810-1621OC.  [DOI](https://doi.org/10.1164/rccm.200810-1621OC) | [PMC free article](/articles/PMC2778155/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19729670/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Respir%20Crit%20Care%20Med&title=Identification%20of%20PCDH1%20as%20a%20novel%20susceptibility%20gene%20for%20bronchial%20hyperresponsiveness.&author=GH%20Koppelman&author=DA%20Meyers&author=TD%20Howard&author=SL%20Zheng&author=GA%20Hawkins&volume=180&publication_year=2009&pages=929-935&pmid=19729670&doi=10.1164/rccm.200810-1621OC&)

48. Misior AM, Deshpande DA, Loza MJ, Pascual RM, Hipp JD, et al. Glucocorticoid- and protein kinase A-dependent transcriptome regulation in airway smooth muscle. Am J Respir Cell Mol Biol. 2009;41:24–39. doi: 10.1165/rcmb.2008-0266OC.  [DOI](https://doi.org/10.1165/rcmb.2008-0266OC) | [PMC free article](/articles/PMC2701960/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19059887/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Respir%20Cell%20Mol%20Biol&title=Glucocorticoid-%20and%20protein%20kinase%20A-dependent%20transcriptome%20regulation%20in%20airway%20smooth%20muscle.&author=AM%20Misior&author=DA%20Deshpande&author=MJ%20Loza&author=RM%20Pascual&author=JD%20Hipp&volume=41&publication_year=2009&pages=24-39&pmid=19059887&doi=10.1165/rcmb.2008-0266OC&)

49. Gautier L, Cope L, Bolstad BM, Irizarry RA. affy–analysis of Affymetrix GeneChip data at the probe level. Bioinformatics. 2004;20:307–315. doi: 10.1093/bioinformatics/btg405.  [DOI](https://doi.org/10.1093/bioinformatics/btg405) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14960456/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=affy%E2%80%93analysis%20of%20Affymetrix%20GeneChip%20data%20at%20the%20probe%20level.&author=L%20Gautier&author=L%20Cope&author=BM%20Bolstad&author=RA%20Irizarry&volume=20&publication_year=2004&pages=307-315&pmid=14960456&doi=10.1093/bioinformatics/btg405&)

50. Smyth GK. Limma: linear models for microarray data. In: Gentleman R, Carey V, Dudoit S, Irizarry R, Huber W, editors. Bioinformatics and Computational Biology Solutions using R and Bioconductor. New York: Springer; 2005. pp. 397–420.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Bioinformatics%20and%20Computational%20Biology%20Solutions%20using%20R%20and%20Bioconductor&author=GK%20Smyth&publication_year=2005&)

51. Benita Y, Cao Z, Giallourakis C, Li C, Gardet A, et al. Gene enrichment profiles reveal T-cell development, differentiation, and lineage-specific transcription factors including ZBTB25 as a novel NF-AT repressor. Blood. 2010;115:5376–5384. doi: 10.1182/blood-2010-01-263855.  [DOI](https://doi.org/10.1182/blood-2010-01-263855) | [PMC free article](/articles/PMC2902135/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20410506/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Gene%20enrichment%20profiles%20reveal%20T-cell%20development,%20differentiation,%20and%20lineage-specific%20transcription%20factors%20including%20ZBTB25%20as%20a%20novel%20NF-AT%20repressor.&author=Y%20Benita&author=Z%20Cao&author=C%20Giallourakis&author=C%20Li&author=A%20Gardet&volume=115&publication_year=2010&pages=5376-5384&pmid=20410506&doi=10.1182/blood-2010-01-263855&)

52. Panettieri RA, Murray RK, DePalo LR, Yadvish PA, Kotlikoff MI. A human airway smooth muscle cell line that retains physiological responsiveness. Am J Physiol. 1989;256:C329–335. doi: 10.1152/ajpcell.1989.256.2.C329.  [DOI](https://doi.org/10.1152/ajpcell.1989.256.2.C329) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/2645779/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Physiol&title=A%20human%20airway%20smooth%20muscle%20cell%20line%20that%20retains%20physiological%20responsiveness.&author=RA%20Panettieri&author=RK%20Murray&author=LR%20DePalo&author=PA%20Yadvish&author=MI%20Kotlikoff&volume=256&publication_year=1989&pages=C329-335&pmid=2645779&doi=10.1152/ajpcell.1989.256.2.C329&)

53. Cooper PR, Mesaros AC, Zhang J, Christmas P, Stark CM, et al. 20-HETE mediates ozone-induced, neutrophil-independent airway hyper-responsiveness in mice. PLoS ONE. 2010;5:e10235. doi: 10.1371/journal.pone.0010235. doi: 10.1371/journal.pone.0010235.  [DOI](http://dx.doi.org/10.1371/journal.pone.0010235) | [DOI](https://doi.org/10.1371/journal.pone.0010235) | [PMC free article](/articles/PMC2857875/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20422032/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&title=20-HETE%20mediates%20ozone-induced,%20neutrophil-independent%20airway%20hyper-responsiveness%20in%20mice.&author=PR%20Cooper&author=AC%20Mesaros&author=J%20Zhang&author=P%20Christmas&author=CM%20Stark&volume=5&publication_year=2010&pages=e10235&pmid=20422032&doi=10.1371/journal.pone.0010235&)
