# The PPAR Alpha gene is associated with triglyceride, low-density cholesterol, and inflammation marker response to fenofibrate intervention: The GOLDN Study

## Metadata
**Authors:** AC Frazier-Wood, JM Ordovas, RJ Straka, JE Hixson, IB Borecki, HK Tiwari, DK Arnett
**Journal:** The pharmacogenomics journal
**Date:** 2012 May 1
**DOI:** [10.1038/tpj.2012.9](https://doi.org/10.1038/tpj.2012.9)
**PMID:** 22547144
**PMCID:** PMC3410976
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3410976/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC3410976/pdf/nihms-361672.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC3410976/pdf/nihms-361672.pdf)

## Abstract

As a peroxisome proliferator-activated receptor alpha (PPARα) agonist, fenofibrate favorably modulates dyslipidemia and inflammation markers, which are associated with cardiovascular risk. To determine whether variation in the PPARα receptor gene was associated with lipid and inflammatory marker response, we conducted a three week trial of fenofibrate in 861 men and women. Mixed linear models which controlled for age and sex, as well as family pedigree and study-center, were constructed using SNPs in the PPARα gene as predictors and changes in fasting triglycerides (TGs), cholesterol and inflammatory markers as outcomes. Significant associations with low-density cholesterol (LDL-C) and interleukin-2 (IL-2; P<.001) responses to fenofibrate were found. Although there were suggestive associations with tumour necrosis factor-alpha (TNF-α) and TG responses (P<.05), these did not survive the correction for multiple testing. We conclude that variants in the PPARα gene may contribute to future pharmacogenomic paradigms seeking to predict fenofibrate responders from both an anti-dyslipidemic and anti-inflammatory perspective.

Keywords: lipid, inflammation, PPARA, pharmacogenetics, GOLDN, fenofibrate response

## INTRODUCTION

Elevated plasma TG, LDL-C levels, and reduced plasma high-density lipoprotein cholesterol (HDL-C) have each been identified as risk factors for cardiovascular disease (CVD) ([1](#R1),[2](#R2)). Current evidence suggests that a number of inflammation markers are also potentially useful predictors of prevalent or incident CVD, and interact with dyslipidemia to mediate CVD risk([3](#R3),[4](#R4)). Drug therapy which reduces plasma TG and LDL-C and raises high-density cholesterol (HDL-C) has been shown to lower adverse cardiovascular outcomes ([5](#R5),[6](#R6)). Fenofibrate is an efficacious therapeutic agent for hypertriglyceridemia, reducing plasma TG levels by 35–50% while having the collateral benefit of raising HDL-C levels by 10–20% ([7](#R7)). In addition to their effects on TG and cholesterol, fibrates have been shown to modulate the production of inflammation markers ([8](#R8)–[12](#R12)). There is, however, significant inter-individual variation in response to fenofibrate([13](#R13)). Findings from a number of studies have suggested that lipid, lipoprotein([14](#R14)–[16](#R16)) and some inflammation marker ([14](#R14)–[17](#R17)) responses to drug therapy are partially under genetic influence.

Fibrates are amphipathic carboxylic acids and are agonists of PPARα. PPARα receptors heterodimerize with the retinoid X receptor and function as transcription factors, regulating the expression of genes. More than 80 genes in humans and mammalian models are known to be regulated by PPARα including apolipoprotein genes (e.g., APOA1, APOA2, APOA5, APOC3), lipoprotein lipase gene (LPL), phospholipid transfer protein gene (PLTP), and other genes involved in lipid and lipoprotein metabolism ([18](#R18),[19](#R19)). PPARα is also known to modulate transcriptional pathways involved in inflammation responses ([20](#R20)).

Given that PPARα is the molecular target for fenofibrate and its known, albeit incompletely understood, roles in lipid metabolism and inflammation, we investigated whether genetic variation in the PPARα gene was associated with variation in fasting TG, LDL-C, HDL-C, adiponectin and inflammation marker response to a three-week fenofibrate trial in the Genetics of Lipid Lowering Drugs and Diet Network (GOLDN) study.

## MATERIALS AND METHODS

### Subjects and Intervention

GOLDN is part of the PROGENI (PROgram for GENetic Interaction) Network, a group of family intervention studies focusing on gene-environment interactions. The participants in the GOLDN study were mainly re-recruited from two NHLBI Family Heart Study (FHS) field centers: Minneapolis, MN, and Salt Lake City, UT. All subjects were Caucasian and nearly all of European ancestry. Eligibility criteria were: 1) ≥ 18 years of age; 2) fasting TGs < 1500 mg/dL; 3) willing to participate in the study and attend the scheduled clinic exams; 4) member of a family with at least two members in a sibship; 5) AST and ALT results within normal range; and 6) creatinine ≤ 2.0 mg/dL. Exclusion criteria were: 1) history of liver, kidney, pancreas, gall bladder disease, or malabsorption; 2) current pregnancy; 3) insulin use; 4) use of lipid lower drugs (including prescription, OTC and nutraceuticals; volunteers taking these agents were withdrawn from them at least four weeks prior to the study with physician's approval); 5) use of warfarin 6) women of childbearing potential not using an acceptable form of contraception; 7) known hypersensitivity to fenofibrate; and 8) history of pancreatitis within 12 months prior to enrollment. A previous study demonstrated that Caucasians in UT and MN were homogeneous and pooling data across centers would not threaten the validity of this study ([21](#R21)).

The details of the GOLDN visits are shown in **Figure 1**. After granting informed consent, participants underwent a baseline screening visit. This visit included a fasting blood draw and pregnancy test, if applicable. The day before the first clinical exam, participants came to the clinic for a fasting blood draw. On visits 2 and 4, willing participants underwent a high fat (83% of calories) meal challenge, after an 8 hour fast, which required ingesting 700 kilocalories / m2 of body surface area ([22](#R22)). Blood draws were collected before the high fat meal (fasting), 3.5 hours after ingestion (uptake) and 6 hours after ingestion (clearance). The fenofibrate intervention consisted of a three-week treatment period, in which participants took fenofibrate (160 mg) daily. Lipids were measured twice on the last two days of the treatment period after a minimum 8-hour fast.

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/050a/3410976/5e93aba5154b/nihms-361672-f0001.jpg)

Design of the NHLBI Genetics of Lipid-Lowering Drugs and Diet Network (GOLDN) study

### Biochemical Analyses

Protocols for measuring TG, HDL-C, LDL-C, and fenofibric acid have been previously described ([23](#R23),[24](#R24)). All lipids were measured using the Roche/Hitachi 911 Automatic Analyzer (Roche Diagnostics Corporation). TGs were measured using a glycerol blanked enzymatic method, cholesterol was measured using a cholesterol esterase, cholesterol oxidase reaction, and LDL-C was measured by a homogeneous direct method (LDL Direct Liquid Select™ Cholesterol Reagent, Equal Diagnostics, Exton, Pa). HDL-C was calculated after precipitation of non-HDL-C with magnesium/dextran. The interlaboratory coefficients of variation in a pooled plasma control were 2.6%, 1.9%, 3.9%, and 1.8% for triglycerides, cholesterol, HDL-C, and LDL-C, respectively.

Inflammatory markers were measured using the following enzyme-linked immunosorbent assay kits from R&D Systems Inc., (Minneapolis, Minn): Quantikine® High Sensitivity Human IL-6, Quantikine® Human IL-2 sRα, Quantikine® Human MCP-1, and QuantiGlo® Human TNF-α. The interlaboratory coefficients of variation on a pooled plasma control were 12.3%, 5.7%, 7.5%, and 9.7% for IL-6, IL-2, monocyte chemotactic protein-1(MCP-1), and TNF-α, respectively.

### SNP Selection and Genotyping

SNPs were identified through searching public databases such as dbSNP ([http://www.ncbi.nlm.nih.gov/SNP/](http://www.ncbi.nlm.nih.gov/SNP/)). We selected 10 SNPs at the PPARα gene on the basis of the following criteria, in order of importance in our selection scheme: 1) validation status, i.e. experimentally validated in Caucasians; 2) functional relevance and importance, namely the potential ‘functional’ SNPs residing within the transcription factor binding sites in the 5' promoter region, in the mRNA stability regulatory protein binding sites in 3'UTR, in exons that change amino acid sequences, or in exon-intron boundaries that alter mRNA splicing; 3) degree of heterozygosity, i.e., minor allele frequencies (MAF) >= 0.05; and 4) previous evidence of association with lipid measurements. SNP genotyping methods have been described elsewhere ([23](#R23)). The overall genotyping error and missing rate was ~1%. We used the GRR software to detect pedigree errors via graphically inspecting the distribution for marker allele sharing among pairs of family members on all pairs of individuals. SNP allele frequencies were estimated via a maximum-likelihood method ([24](#R24)). For each SNP, a χ^2^ test was used to examine deviation of SNP genotypes from Hardy-Weinberg equilibrium (HWE).

### Statistical Analyses

#### Responses to fenofibrate

Lipid and inflammatory responses to fenofibrate were calculated using growth curve models. Eight lipid and inflammatory marker measures were analyzed: TG, LDL-C, HDL-C, adiponectin, Il-2, Il-6, MCP, and TNF-α. At time point 1 (fasting) TG, HDL –C, adiponectin, IL-2, IL-6, MCP and TNF-α concentrations. Post-prandial lipids were measured at time points 2 (3.5 hours post-ingestion, or uptake) and 3 6 hours post-ingestion, or clearance).TG concentrations were log-transformed to normalize the distribution. The normalized data were used for the growth curve models. Fasting data were used from visits 1 and 2 (pre-fenofibrate) and 3 and 4 (post- fenofibrate; [figure 1](#F1)). For each condition (pre/post) the two draws were treated as repeated measures to reduce error variance in the slope estimation. Growth curve slopes were adjusted for age, age^2^, age^3,^ sex, and data collection center. Compared to a simple delta phenotype, SOLAR estimated heritabilities were much higher for growth curve phenotypes (>50%). In addition to analyzing genotype associations with lipid, lipoprotein, and inflammation marker response phenotypes, we analyzed genotype associations with serum fenofibric acid concentrations. For the fenofibric acid phenotype, we used the partial area under the fenofibric acid concentration-time curve (FA AUC_0-6_) from serum concentrations measured over six hours (at 0, 3.5, and 6 hours after dosing) on the final day of the intervention. To test whether there were significant differences in lipid and inflammatory marker concentrations before and after the fenofibrate trial, t tests were conducted, stratified by sex, on data that had been transformed where necessary, as above.

#### SNP-phenotype associations

We used a linear mixed model implemented in SAS (version 9.1, SAS Institute, Cary, NC). The effects of SNP genotypes (categorical variable with three classes) were treated as fixed effects and the dependencies among members within each family were treated as random effects. Field center, age, age^2^, age^3^ and gender were included in the model as covariates. To control for multiple testing, false discovery rate (FDR ([25](#R25))) procedure was applied to the data, which gives rise to q-values that represent p-values corrected for the number tests run, which seeks to maximize power while correcting for the family-wise error rate in multiple tests. Hardy-Weinberg Equilibrium (HWE) was calculated using Haploview ([26](#R26)), and all SNPs were in HWE with an alpha value of *P*>0.05.

The slopes for Il-2, MCP and TNF-α were log transformed to normality for the genetic analysis. The square root was taken of the FAAUC_0-6_ values in the genetic models, again, to approximate normality.

#### Haplotype analysis

For all traits with more than one significant fasting SNP-phenotype association within a haplotype block (**Figure 2**) we conducted haplotype analysis. Haplotype frequencies were calculated, and included in the association analysis were haplotype frequencies greater than 0.01. Tests of association, controlling for pedigree were conducted using the haplotype association module hbat within the FBAT program ([27](#R27)).

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/050a/3410976/4768f8e4fda7/nihms-361672-f0002.jpg)

Linkage disequilibrium plot for 10 SNPs in the PPARα gene Created using Haploview software (26)

## RESULTS

### Study Sample Characteristics

The number of subjects having screened for eligibility was 1,327 (639 men and 688 women) from 148 families. Of these, 861 subjects (427 men and 434 women) underwent the fenofibrate intervention, had all genotype data, and had all lipid, lipoprotein, and inflammation marker phenotype measures. **Table 1** shows the baseline characteristics of this sample. The mean age of men was 50.6 years, and that of women was 51.1 years. Mean phenotype data are shown stratified by sex in **Table 1**.

### TABLE 1.

|   | Men (n=427) |  |   | Women (n=434) |  |   |
| --- | --- | --- | --- | --- | --- | --- |
|   | Prefenofibrate | Postfenofibrate | p-value | Prefenofibrate | Postfenofibrate | p-value |
| Age, yr | 50.6 ± 15.9 |  |   | 51.1 ± 15.8 |  |   |
| BMI, kg/m2 | 28.72 ± 4.86 |  |   | 28.48 ± 6.33 |  |   |
| Triglyceride, mg/dL | 153.41 ± 142.01 | 99.18 ± 59.83 | <.001 | 125.19 ± 82.22 | 80.67 ± 47.30 | <.001 |
| HDL-C, mg/dL | 41.57 ± 9.84 | 43.67 ± 9.99 | <.001 | 52.29 ± 13.68 | 55.03 ± 14.23 | <.001 |
| LDL-C, mg/dL | 123.17 ± 30.32 | 111.45 ± 32.79 | <.001 | 119.63± 32.15 | 97.19 ± 28.83 | <.001 |
| MCP, pg/mL | 216.67 ± 69.79 | 224.26 ±75.719 | <.001 | 200.86 ± 58.45 | 221.04 ± 74.98 | <.001 |
| TNF, pg/mL | 3.68 ± 7.08 | 3.81 ±5.29 | <.001 | 3.18 ±2.05 | 3.65 ± 2.09 | <.001 |
| Il-2, pg/mL | 1042.60 ± 390.66 | 1132.51 ± 494.82 | <.001 | 1003.72 ±342.21 | 1191.78 ± 561.40 | <.001 |
| IL-6, pg/mL | 2.07 ± 4.13 | 2.31± 4.26 | .049 | 1.89 ± 1.63 | 2.06 ± 2.15 | .165 |
| Adiponectin, pg/mL | 6314.26 ± 3514.82 | 5845.01 ± 3197.56 | <.001 | 10216.32 ± 5064.67 | 9681.73 ± 4672.38 | <.001 |
| FA AUC, (mg/L)*hr* | - | 64.54 ± 27.82 |   | - | 74.98 ± 36.57 |   |

Table 1 Caption: Characteristics of study subjects. Values are listed as mean ± standard deviation.

For both men and women, TG and LDL-C, concentrations were significantly lower after fenofibrate treatment than those before treatment (all *P* < 0.001). HDL-C increased significantly after fenofibrate treatment (*P* < 0.001). Adiponectin was the only marker which was significantly lower in both men (*P*<.0001) and women (*P*=.002); concentrations of IL-2, TNF-α and MCP increased in both men and women (*P*<.001) with IL-6 higher in men (*P*=.049) and not women (*P*=.165).

### Single-SNP Associations

**Supplementary Figure 1** shows the location of our SNPs in the *PPARα* gene and **Figure 2** gives a linkage disequilibrium (LD) plot from haploview for our 10 SNPs. **Table 2** summarizes the significant results of the association analyses between each of the single PPARα SNPs and response to fenofibrate (i.e., growth curve slopes) for all phenotypes. For TG, evidence for association was found for rs4253701 (*P* = 0.025) but this did not survive the FDR correction for multiple testing (q=.321). For LDL-C, there were significant associations for rs135550 (*P* = 0.001) and rs135543 (*P*=0.001; both q=0.030). For inflammation marker response phenotypes, only Il-2 and TNF response to fenofibrate showed associations with any SNP. rs9626730 showed an association with Il-2 (*P* = 0.002; q=0.018), and rs13550, rs15522, rs135543 and rs4253701 showed suggestive associations with TNF response to fenofibrate (*P*<0.05) that did not survive the FDR correction (q>0.05). Significant associations were not observed for FA AUC_0-6_ phenotype, although rs135550 showed a trend association with FA AUC_0-6_ (*P*=0.024; q=0.321).

### TABLE 2.

| SNP | Phenotypic mean ± SE |  |  | F(df)1 | P | Q |
| --- | --- | --- | --- | --- | --- | --- |
| 1/1 | 1/2 | 2/2 |  |  |  |  |
| TG |   |   |   |   |   |   |
| rs4253701r | -0.130(0.085) | -0.042(0.027) | 0.014 (0.013) | 5.06(612) | .025 | 0.321 |
| LDL-C |   |   |   |   |   |   |
| rs135550 | 8.349 (2.550) | -1.617(1.267) | 0.541(1.155) | 7.77(612) | .001 | 0.030 |
| rs135543 | 0.403(1.207) | -1.468(1.237) | 7.898(2.345) | 6.93(614) | .001 | 0.030 |
| Il-2 |   |   |   |   |   |   |
| rs9626730d | 0.130(0.042) | -0.037(0.014) | -0.023(0.008) | 13.64(679) | .0002 | 0.018 |
| TNF |   |   |   |   |   |   |
| rs135550 | -0.099(0.042) | -0.010(0.010) | -0.068(0.017) | 3.33(674) | .036 | 0.500 |
| rs15522r | -0.048(0.028) | -0.021(0.018) | -0.091(0.022) | 3.24(676) | .016 | 0.288 |
| rs135543r | -0.073(0.018) | -0.016(0.019) | -0.056(0.039) | 2.53(676) | .044 | 0.495 |
| rs4253701 | -0.291(0.112) | 0.005(0.030) | -0.055(0.014) | 4.19(673) | .016 | 0.288 |
| FAAUC |   |   |   |   |   |   |
| rs135550d | 7.547(0.243) | 8.187(0.116) | 8.033(0.104) | 5.12(601) | 0.024 | 0.321 |

Table 2 Caption: Significant genetic variant-phenotype associations with fasting data.

We additionally conducted SNP-phenotype associations with the post-prandial data at both time points (**Figure 1**). This confirmed our findings with the same significant SNP-lipid associations, in the expected direction (**Supplementary Table 1**). There were no additional significant SNP-trait associations. *Haplotype analysis* Only LDL-C and TNFα were associated with SNPs within a haplotype block ([Table 2](#T2); **Figure 2**) at P<.05.

#### Association within Haplotype block 1

This consisted of SNPs rs135550 and rs135549 (**Figure 3**). FBAT identified four haplotypes, of which three had frequencies greater than .01.. Haplotypes 1 (T-C) and 2 (C-T) were associated with fasting LDL-C responses to fenofibrate (*P*=.01 and *P*=.03; **Table 3**), but were not associated with fasting TNFα (*P*=.48 and *P*=.44; **Table 3**).

|   |   | P |  |
| --- | --- | --- | --- |
| Haplotype | frequency | LDL-C1 | TNFα1 |
| Block 1 rs135550- rs135549 |  |  |  |
| T-C | .70 | .01 | .48 |
| C-T | .27 | .03 | .46 |
| Block 2 rs135550- rs135549- rs135543 -rs9626730 |  |  |  |
| T-C-A-A | .57 | .02 | .67 |

Table Caption: Significant phenotype associations with haplotype blocks consisting of two (rs135550 and rs135549) and four (rs135550, rs135549, rs135543, rs9626730) marker haplotypes.

#### Association within Haplotype block 2

This consisted of SNPs rs135550 , rs135549, rs135543 and rs9626730 (**Figure 2**). FBAT identified 14 haplotypes, of which 7 had frequency of greater than .01. Only one haplotype (show haplotype) was associated with fasting LDL-C responses to fenofibrate (*P*=0.02; **Table 3**), and none were associated with fasting TNFα (all *P*>0.05).

## DISCUSSION

This study demonstrates that people who carry different PPARα variants responded differently to fenofibrate intervention with respect to reductions in LDL-C and changes in inflammation marker plasma concentrations. Although PPARα has been studied in this context before, this is the first study of its size and scope investigating these associations in a healthy population, and the first study to include the effects of PPARα variants on inflammation marker responses to fenofibrate.

Fenofibrate binds to and activates PPARα. PPARs are specific transcription factors that mediate the gene regulation effects thought to induce the lipid- and inflammation- lowering effects of fibrates. We thus tested whether variants in PPARα are associated with differences in fenofibrate responses by lipids and inflammatory markers. For lipid phenotypes, two variants (rs135550 and rs135543) showed an association with change in LDL-C, both giving rise to a q-value of q=0.015 after an FDR correction for multiple testing. The haplotype analysis of these two markers with a total of four markers within the haplotype block, showed a significant association with LDL-C responses to fenofibrate, and raises the possibility that the true causal variant(s) is located between rs135549 and rs135543. Fibrate treatment induces the formation of larger, more buoyant LDL particles, which show an increased affinity for the LDL receptor over their smaller counterparts ([28](#R28),[29](#R29)). The main mechanism of action responsible for a reduction in LDL particles is thought to be an increase in the rate of catabolism from this increased LDL receptor affinity – in hyperlipidemic patients LDL receptor-mediated uptake increased 43% after fibrate treatment, but receptor-independent catabolism remained unchanged([28](#R28)). This is the first study, to our knowledge, to report associations between the degree of change in LDL-C concentration after fenofibrate treatment and these two variants. rs135550 is in the untranslated region of the messenger ribonucleic acids (mRNA) and here we report a novel association of this variant with a lipid phenotype. Variant rs135543, 2087 kb downstream of rs13550, is in the same untranslated region and has previously been associated with baseline lipids ([29](#R29)) and myocardial infarction (MI) as part of a three-marker haploblock([30](#R30)). Our results were confirmed with similar SNP-phenotype associations, in the expected direction, with the postprandial data.

We did not show any variant-phenotype association with HDL-C response to fenofibrate, and although one SNP (rs4253701) showed an association with TG of p<0.05, this did not survive the correction for multiple testing (q=0.321). We are unaware of any previous associations between PPARα variants and fenofibrate response of HDL-C. Brisson *et al*. report no association with TG response to fenofibrate and rs1800206 (commonly referred to as PPARα Leu162Val; not tested here)([31](#R31)) but Foucher and colleagues([32](#R32)) report an association between rs4253778 and TG response to fenofibrate in diabetics. It is not clear why we did not replicate Foucher *et al*.'s variant-phenotype associations with TG response to fenofibrate on the PPARα. This may relate to the different variants typed in our study, where the closest is 170,0048 bp downstream of variant rs4253778 (SNP Annotation and Proxy Search (SNAP) software tool by the Broad Institute ([http://www.broadinstitute.org/mpg/snap/ldsearch.php](http://www.broadinstitute.org/mpg/snap/ldsearch.php)) which uses HapMap Project data, reports that none of our SNPs are in LD with an r^2^>0.8 with rs4253778). Alternatively, Foucher *et al*.'s variant-phenotype association between TG response to fenofibrate and SNPs in the PPARα gene could be due the population used, which was not enriched for diabetics and had a higher baseline TG level ([32](#R32)). In comparison, GOLDN is a general population sample, with a low percentage (~20%) of diabetics, consequently, baseline TG levels are lower. As baseline TG levels in themselves modulate TG-responses to fenofibrate([33](#R33)) this may partially explain why we did not replicate Foucher *et al.*'s variant-response association.

We examined PPARα variants and their effect on inflammatory marker reductions and adiponectin levels with fenofibrate treatment. No variants were associated with change in adiponectin, MCP or Il-6, although concentrations of these inflammatory markers did increase significantly in both men and women before and after the fenofibrate trial. Two previous studies reported on two of the variants used in this study (rs4253701 and rs4253728) but neither demonstrated an association between either SNP and either myocardial infarctions (MI) ([34](#R34)) or apoC-III levels. ([35](#R35)) However, these studies were seeking associations between PPARα variants and select outcomes, in contrast to our pursuits of PPARα variants as modulators of lipid or inflammatory markers in response to fenofibrate as an agonist of PPARα. We did show a significant mediation of the Il-2 increase in response to fenofibrate by the variant rs9626730 (P=0.0002; q=0.018). Given that the increase in IL-2 was surprising, this highlights the importance of genetic variation in PPARα in protecting against this. Although there were 4 SNPs that showed suggested associations with TNF-α increases (*P*<.05), none of these response-variant associations survived the FDR correction. Fibrates have previously been shown to reduce a number of systemic inflammation markers such as IL-6 and TNF-α ([36](#R36)–[38](#R38)), presumed to act through their effects on PPARα. Other PPARs such as PPARγ activation may also affect inflammation markers, but its role is more controversial and less well understood([39](#R39)). PPARα activation has been shown to interact with nuclear factor-κB (NF-κB) and AP-1 signaling pathways to lead to a reduction in inflammatory markers, such as IL-6 ([39](#R39)). It remains unclear whether this decrease is due solely to the known increase in catabolism of inflammatory markers, or because of independent effects of PPARα activation on their production ([39](#R39)). Notably, reductions in IL-6 with fenofibrate have been shown to occur independently of lipid changes in patients with the metabolic syndrome, suggesting direct effects of fenofibrate and PPARα on inflammation ([36](#R36)). In our study, we show that carriers of the minor allele had reductions in IL-6, yet those homozygous for the major allele had increases in IL-6. This highlights the complexity of this biological pathway and warrants further research.

Fenofibric acid is the major metabolite of fenofibrate. After oral fenofibrate administrations, fenofibrate is rapidly hydrolyzed by esterases to form fenofibric acid, the rest being excreted through urine (primary) and feces (secondary)([40](#R40)). Given that serum fenofibric acid concentration has been correlated with reductions in total, and LDL-C, it is of interest to examine genotype-phenotype associations with serum fenofibric acid after a standardized dose. rs135550 was significantly associated with serum fenofibric acid concentrations (*P*=0.024), and may be interesting given its significant association with change in LDL-C concentrations after fenofibrate treatment, however, as this variant-phenotype association did not survive the correction for multiple testing, the relationship of this variant to the lipid and inflammatory marker lowering potential of fenofibrate acid, and the role of fenofibric acid concentrations in mediating this relationship is unclear. We provide only a preliminary indication that differences in LDL-C reduction in our study, associated with rs135550 may operate through differences in systemic exposure (AUC_0-6_) to fenofibrate acid in the serum. This relationship and others related to exposure are more likely to be elucidated with further consideration of genetic variations in the metabolic pathways of fenofibric acid's elimination.

Our analyses benefited from a strong *a priori* hypothesis, a good sample size for a pharmacogenetic study, and a healthy population, making generalizations to other samples less problematic than with clinical samples. However, there are some limitations: firstly, these associations should be tested in more racially diverse populations, and with sample sizes that allow for the stratification by gender, which was only statistically controlled for in this study. Secondly, it is not clear whether these SNP-phenotype associations would generalize to hyperlipidemic patients, where fenofibrate treatment is targeted. Thirdly, many of our conclusions regarding associations between a variant and both serum fenofibric acid concentrations and responses to fenofibrate did not survive the FDR correction for multiple testing, so should be considered preliminary. Thirdly, it is not clear why some variants in high LD with each other, did not show the same genotype-phenotype associations. This lack of replication within our study suggests that we may have suffered from a power issue with the correction for multiple testing, and replication with a lower number of SNPs, and / or alternative phenotypes would be an important avenue of research. Finally, the finding of an increase in inflammatory markers after fenofibrate treatment was unexpected. Our data suggest that some variants are associated with the expected decrease, while others are associated with an increase in markers of inflammation. As this is the first such finding, we again heavily encourage replication in independent cohorts.

Nonetheless, the SNP-phenotype associations identified in these analyses may have important implications for clinical care. Our data supports results showing fenofibrate to be an effective reducer of TG, LDL-C and a modulator of inflammatory markers, and we further highlight the role of genetic variants on the PPARα gene in modulating these responses. Treating cardiovascular disease, and reducing markers of cardiovascular disease risk, requires complex lifestyle and pharmacologic intervention paradigms. Newer research is aimed at integrating complex genetic information into these multi-modal networks of treatments to increase their efficacy ([41](#R41)). The data here, which show variants in the PPARα gene modulate the response in LDL-C and IL-6 in response to fenofibrate, may be an important contributor to future paradigms that use pharmacogenetic information.

## Supplementary Material

## ACKNOWLEDGEMENTS

We are grateful to the staff of the GOLDN study for the assistance in data collection and management.

*Sources of funding*: This study was funded by NHLBI grant number U01HL072524.

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Krauss RM. Triglycerides and atherogenic lipoproteins: rationale for lipid management. Am J Med. 1998;105(1A):58S–62S. doi: 10.1016/s0002-9343(98)00213-7.  [DOI](https://doi.org/10.1016/s0002-9343(98)00213-7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9707269/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Med&title=Triglycerides%20and%20atherogenic%20lipoproteins:%20rationale%20for%20lipid%20management.&author=RM%20Krauss&volume=105&issue=1A&publication_year=1998&pages=58S-62S&pmid=9707269&doi=10.1016/s0002-9343(98)00213-7&)

2. Wilson PW, Garrison RJ, Castelli WP, et al. Prevalence of coronary heart disease in the Framingham Offspring Study: role of lipoprotein cholesterols. Am J Cardiol. 1980;46(0002-9149):649–654. doi: 10.1016/0002-9149(80)90516-0. (Print)  [DOI](https://doi.org/10.1016/0002-9149(80)90516-0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7416024/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Cardiol&title=Prevalence%20of%20coronary%20heart%20disease%20in%20the%20Framingham%20Offspring%20Study:%20role%20of%20lipoprotein%20cholesterols.&author=PW%20Wilson&author=RJ%20Garrison&author=WP%20Castelli&volume=46&issue=0002-9149&publication_year=1980&pages=649-654&pmid=7416024&doi=10.1016/0002-9149(80)90516-0&)

3. Ballantyne CM, Nambi V. Markers of inflammation and their clinical significance. Atheroscler.Suppl. 2005;6(1567-5688):21–29. doi: 10.1016/j.atherosclerosissup.2005.02.005. (Print)  [DOI](https://doi.org/10.1016/j.atherosclerosissup.2005.02.005) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15823493/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Atheroscler.Suppl&title=Markers%20of%20inflammation%20and%20their%20clinical%20significance.&author=CM%20Ballantyne&author=V%20Nambi&volume=6&issue=1567-5688&publication_year=2005&pages=21-29&pmid=15823493&doi=10.1016/j.atherosclerosissup.2005.02.005&)

4. Cesari M, Penninx BW, Newman AB, et al. Inflammatory markers and onset of cardiovascular events: results from the Health ABC study. Circulation. 2003;108:2317–2322. doi: 10.1161/01.CIR.0000097109.90783.FC.  [DOI](https://doi.org/10.1161/01.CIR.0000097109.90783.FC) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14568895/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circulation&title=Inflammatory%20markers%20and%20onset%20of%20cardiovascular%20events:%20results%20from%20the%20Health%20ABC%20study.&author=M%20Cesari&author=BW%20Penninx&author=AB%20Newman&volume=108&publication_year=2003&pages=2317-2322&pmid=14568895&doi=10.1161/01.CIR.0000097109.90783.FC&)

5. Cullen P. Evidence that triglycerides are an independent coronary heart disease risk factor. Am J Cardiol. 2000;86:943–949. doi: 10.1016/s0002-9149(00)01127-9.  [DOI](https://doi.org/10.1016/s0002-9149(00)01127-9) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11053704/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Cardiol&title=Evidence%20that%20triglycerides%20are%20an%20independent%20coronary%20heart%20disease%20risk%20factor.&author=P%20Cullen&volume=86&publication_year=2000&pages=943-949&pmid=11053704&doi=10.1016/s0002-9149(00)01127-9&)

6. Hausenloy DJ, Yellon DM. Targeting residual cardiovascular risk: raising high-density lipoprotein cholesterol levels. Postgrad.Med J. 2008;84:590–598. doi: 10.1136/hrt.2007.125401.  [DOI](https://doi.org/10.1136/hrt.2007.125401) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19103817/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Postgrad.Med%20J&title=Targeting%20residual%20cardiovascular%20risk:%20raising%20high-density%20lipoprotein%20cholesterol%20levels.&author=DJ%20Hausenloy&author=DM%20Yellon&volume=84&publication_year=2008&pages=590-598&pmid=19103817&doi=10.1136/hrt.2007.125401&)

7. Knopp RH, Walden CE, Warnick GR, et al. Effect of fenofibrate treatment on plasma lipoprotein lipids, high-density lipoprotein cholesterol subfractions, and apolipoproteins B, AI, AII, and E. Am J Med. 1987;83:75–84. doi: 10.1016/0002-9343(87)90875-8.  [DOI](https://doi.org/10.1016/0002-9343(87)90875-8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/3120587/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Med&title=Effect%20of%20fenofibrate%20treatment%20on%20plasma%20lipoprotein%20lipids,%20high-density%20lipoprotein%20cholesterol%20subfractions,%20and%20apolipoproteins%20B,%20AI,%20AII,%20and%20E.&author=RH%20Knopp&author=CE%20Walden&author=GR%20Warnick&volume=83&publication_year=1987&pages=75-84&pmid=3120587&doi=10.1016/0002-9343(87)90875-8&)

8. Sebestjen M, Keber I, Zegura B, et al. Statin and fibrate treatment of combined hyperlipidemia: the effects on some novel risk factors. Thromb Haemost. 2004;92:1129–1135. doi: 10.1160/TH03-04-0250.  [DOI](https://doi.org/10.1160/TH03-04-0250) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15543343/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Thromb%20Haemost&title=Statin%20and%20fibrate%20treatment%20of%20combined%20hyperlipidemia:%20the%20effects%20on%20some%20novel%20risk%20factors.&author=M%20Sebestjen&author=I%20Keber&author=B%20Zegura&volume=92&publication_year=2004&pages=1129-1135&pmid=15543343&doi=10.1160/TH03-04-0250&)

9. Rosenson RS. Effect of fenofibrate on adiponectin and inflammatory biomarkers in metabolic syndrome patients. Obesity. 2009;17:504–509. doi: 10.1038/oby.2008.530.  [DOI](https://doi.org/10.1038/oby.2008.530) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19023279/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Obesity&title=Effect%20of%20fenofibrate%20on%20adiponectin%20and%20inflammatory%20biomarkers%20in%20metabolic%20syndrome%20patients.&author=RS%20Rosenson&volume=17&publication_year=2009&pages=504-509&pmid=19023279&doi=10.1038/oby.2008.530&)

10. Rosenson RS, Huskin AL, Wolff DA, et al. Fenofibrate reduces fasting and postprandial inflammatory responses among hypertriglyceridemia patients with the metabolic syndrome. Atherosclerosis. 2008;198:381–388. doi: 10.1016/j.atherosclerosis.2007.12.007.  [DOI](https://doi.org/10.1016/j.atherosclerosis.2007.12.007) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18242616/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Atherosclerosis&title=Fenofibrate%20reduces%20fasting%20and%20postprandial%20inflammatory%20responses%20among%20hypertriglyceridemia%20patients%20with%20the%20metabolic%20syndrome.&author=RS%20Rosenson&author=AL%20Huskin&author=DA%20Wolff&volume=198&publication_year=2008&pages=381-388&pmid=18242616&doi=10.1016/j.atherosclerosis.2007.12.007&)

11. Jonkers IJ, Mohrschladt MF, Westendorp RG, et al. Severe hypertriglyceridemia with insulin resistance is associated with systemic inflammation: reversal with bezafibrate therapy in a randomized controlled trial. Am J Med. 2002;112:275–280. doi: 10.1016/s0002-9343(01)01123-8.  [DOI](https://doi.org/10.1016/s0002-9343(01)01123-8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11893366/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Med&title=Severe%20hypertriglyceridemia%20with%20insulin%20resistance%20is%20associated%20with%20systemic%20inflammation:%20reversal%20with%20bezafibrate%20therapy%20in%20a%20randomized%20controlled%20trial.&author=IJ%20Jonkers&author=MF%20Mohrschladt&author=RG%20Westendorp&volume=112&publication_year=2002&pages=275-280&pmid=11893366&doi=10.1016/s0002-9343(01)01123-8&)

12. Okopien B, Krysiak R, Kowalski J, et al. The effect of statins and fibrates on interferon-gamma and interleukin-2 release in patients with primary type II dyslipidemia. Atherosclerosis. 2004;176:327–335. doi: 10.1016/j.atherosclerosis.2004.05.009.  [DOI](https://doi.org/10.1016/j.atherosclerosis.2004.05.009) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15380456/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Atherosclerosis&title=The%20effect%20of%20statins%20and%20fibrates%20on%20interferon-gamma%20and%20interleukin-2%20release%20in%20patients%20with%20primary%20type%20II%20dyslipidemia.&author=B%20Okopien&author=R%20Krysiak&author=J%20Kowalski&volume=176&publication_year=2004&pages=327-335&pmid=15380456&doi=10.1016/j.atherosclerosis.2004.05.009&)

13. Keating GM, Ormrod D. Micronised fenofibrate: an updated review of its clinical efficacy in the management of dyslipidaemia. Drugs. 2002;62:1909–1944. doi: 10.2165/00003495-200262130-00013.  [DOI](https://doi.org/10.2165/00003495-200262130-00013) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12215067/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drugs&title=Micronised%20fenofibrate:%20an%20updated%20review%20of%20its%20clinical%20efficacy%20in%20the%20management%20of%20dyslipidaemia.&author=GM%20Keating&author=D%20Ormrod&volume=62&publication_year=2002&pages=1909-1944&pmid=12215067&doi=10.2165/00003495-200262130-00013&)

14. Ordovas JM. Pharmacogenetics of lipid diseases. Hum Genomics. 2004;1(1473-9542):111–125. doi: 10.1186/1479-7364-1-2-111. (Print)  [DOI](https://doi.org/10.1186/1479-7364-1-2-111) | [PMC free article](/articles/PMC3525073/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15601539/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Genomics&title=Pharmacogenetics%20of%20lipid%20diseases.&author=JM%20Ordovas&volume=1&issue=1473-9542&publication_year=2004&pages=111-125&pmid=15601539&doi=10.1186/1479-7364-1-2-111&)

15. Smith JA, Arnett DK, Kelly RJ, et al. The genetic architecture of fasting plasma triglyceride response to fenofibrate treatment. Eur J Hum Genet. 2008;16(1018-4813):603–613. doi: 10.1038/sj.ejhg.5202003. (Print)  [DOI](https://doi.org/10.1038/sj.ejhg.5202003) | [PMC free article](/articles/PMC2546577/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18212815/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Hum%20Genet&title=The%20genetic%20architecture%20of%20fasting%20plasma%20triglyceride%20response%20to%20fenofibrate%20treatment.&author=JA%20Smith&author=DK%20Arnett&author=RJ%20Kelly&volume=16&issue=1018-4813&publication_year=2008&pages=603-613&pmid=18212815&doi=10.1038/sj.ejhg.5202003&)

16. Liu Y, Ordovas JM, Gao G, et al. Pharmacogenetic association of the APOA1/C3/A4/A5 gene cluster and lipid responses to fenofibrate: the genetics of lipid-lowering drugs and diet network study. Pharmacogenet.Genomics. 2009;19(1744-6872):161–169. doi: 10.1097/FPC.0b013e32831e030e. (Print)  [DOI](https://doi.org/10.1097/FPC.0b013e32831e030e) | [PMC free article](/articles/PMC2733171/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19057464/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet.Genomics&title=Pharmacogenetic%20association%20of%20the%20APOA1/C3/A4/A5%20gene%20cluster%20and%20lipid%20responses%20to%20fenofibrate:%20the%20genetics%20of%20lipid-lowering%20drugs%20and%20diet%20network%20study.&author=Y%20Liu&author=JM%20Ordovas&author=G%20Gao&volume=19&issue=1744-6872&publication_year=2009&pages=161-169&pmid=19057464&doi=10.1097/FPC.0b013e32831e030e&)

17. Shen J, Arnett DK, Parnell LD, et al. Association of common C-reactive protein (CRP) gene polymorphisms with baseline plasma CRP levels and fenofibrate response: the GOLDN study. Diabetes Care. 2008;31(1935-5548):910–915. doi: 10.2337/dc07-1687. (Electronic)  [DOI](https://doi.org/10.2337/dc07-1687) | [PMC free article](/articles/PMC4615595/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18285551/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Diabetes%20Care&title=Association%20of%20common%20C-reactive%20protein%20(CRP)%20gene%20polymorphisms%20with%20baseline%20plasma%20CRP%20levels%20and%20fenofibrate%20response:%20the%20GOLDN%20study.&author=J%20Shen&author=DK%20Arnett&author=LD%20Parnell&volume=31&issue=1935-5548&publication_year=2008&pages=910-915&pmid=18285551&doi=10.2337/dc07-1687&)

18. Mandard S, Muller M, Kersten S. Peroxisome proliferator-activated receptor alpha target genes. Cell Mol Life Sci. 2004;61(1420-682X):393–416. doi: 10.1007/s00018-003-3216-3. (Print)  [DOI](https://doi.org/10.1007/s00018-003-3216-3) | [PMC free article](/articles/PMC11138883/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14999402/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell%20Mol%20Life%20Sci&title=Peroxisome%20proliferator-activated%20receptor%20alpha%20target%20genes.&author=S%20Mandard&author=M%20Muller&author=S%20Kersten&volume=61&issue=1420-682X&publication_year=2004&pages=393-416&pmid=14999402&doi=10.1007/s00018-003-3216-3&)

19. Spinelli SL, O'Brien JJ, Bancos S, et al. The PPAR-Platelet Connection: Modulators of Inflammation and Potential Cardiovascular Effects. PPAR.Res. 2008;2008(1687-4757):328172. doi: 10.1155/2008/328172. (Print) LA - eng PT - Journal Article.  [DOI](https://doi.org/10.1155/2008/328172) | [PMC free article](/articles/PMC2233896/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18288284/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PPAR.Res&title=The%20PPAR-Platelet%20Connection:%20Modulators%20of%20Inflammation%20and%20Potential%20Cardiovascular%20Effects.&author=SL%20Spinelli&author=JJ%20O'Brien&author=S%20Bancos&volume=2008&issue=1687-4757&publication_year=2008&pages=328172&pmid=18288284&doi=10.1155/2008/328172&)

20. Rizzo G, Fiorucci S. PPARs and other nuclear receptors in inflammation. Curr Opin Pharmacol. 2006;6(1471-4892):421–427. doi: 10.1016/j.coph.2006.03.012. (Print)  [DOI](https://doi.org/10.1016/j.coph.2006.03.012) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16777482/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Opin%20Pharmacol&title=PPARs%20and%20other%20nuclear%20receptors%20in%20inflammation.&author=G%20Rizzo&author=S%20Fiorucci&volume=6&issue=1471-4892&publication_year=2006&pages=421-427&pmid=16777482&doi=10.1016/j.coph.2006.03.012&)

21. Pankow JS, Province MA, Hunt SC, et al. Regarding “Testing for population subdivision and association in four case-control studies.”. Am J Hum Genet. 2002;71:1478–1480. doi: 10.1086/344582.  [DOI](https://doi.org/10.1086/344582) | [PMC free article](/articles/PMC378599/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12515254/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=Regarding%20%E2%80%9CTesting%20for%20population%20subdivision%20and%20association%20in%20four%20case-control%20studies.%E2%80%9D&author=JS%20Pankow&author=MA%20Province&author=SC%20Hunt&volume=71&publication_year=2002&pages=1478-1480&pmid=12515254&doi=10.1086/344582&)

22. Patsch JR, Miesenböck G, Hopferwieser T, et al. Relation of triglyceride metabolism and coronary artery disease. Studies in the postprandial state. Arteriosclerosis and thrombosis□: a journal of vascular biology / American Heart Association. 1992;12(11):1336–45. doi: 10.1161/01.atv.12.11.1336.  [DOI](https://doi.org/10.1161/01.atv.12.11.1336) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/1420093/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arteriosclerosis%20and%20thrombosis%E2%96%A1:%20a%20journal%20of%20vascular%20biology%20/%20American%20Heart%20Association&title=Relation%20of%20triglyceride%20metabolism%20and%20coronary%20artery%20disease.%20Studies%20in%20the%20postprandial%20state.&author=JR%20Patsch&author=G%20Miesenb%C3%B6ck&author=T%20Hopferwieser&volume=12&issue=11&publication_year=1992&pages=1336-45&pmid=1420093&doi=10.1161/01.atv.12.11.1336&)

23. Liu Y, Ordovas JM, Gao G, et al. The SCARB1 gene is associated with lipid response to dietary and pharmacological interventions. J Hum Genet. 2008;53(1434-5161):709–717. doi: 10.1007/s10038-008-0302-2. (Print)  [DOI](https://doi.org/10.1007/s10038-008-0302-2) | [PMC free article](/articles/PMC3836273/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18542840/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Hum%20Genet&title=The%20SCARB1%20gene%20is%20associated%20with%20lipid%20response%20to%20dietary%20and%20pharmacological%20interventions.&author=Y%20Liu&author=JM%20Ordovas&author=G%20Gao&volume=53&issue=1434-5161&publication_year=2008&pages=709-717&pmid=18542840&doi=10.1007/s10038-008-0302-2&)

24. Straka RJ, Burkhardt RT, Fisher JE. Determination of fenofibric acid concentrations by HPLC after anion exchange solid-phase extraction from human serum. Ther Drug Monit. 2007;29(0163-4356):197–202. doi: 10.1097/ftd.0b013e318030860a. (Print)  [DOI](https://doi.org/10.1097/ftd.0b013e318030860a) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17417074/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ther%20Drug%20Monit&title=Determination%20of%20fenofibric%20acid%20concentrations%20by%20HPLC%20after%20anion%20exchange%20solid-phase%20extraction%20from%20human%20serum.&author=RJ%20Straka&author=RT%20Burkhardt&author=JE%20Fisher&volume=29&issue=0163-4356&publication_year=2007&pages=197-202&pmid=17417074&doi=10.1097/ftd.0b013e318030860a&)

25. Benjamini Y, Hochberg Y. Controlling the False Discovery Rate: A Practical and Powerful Approach to Multiple Testing. Journal of the Royal Statistical Society.Series B (Methodological) 1995;57(1):289–300.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20of%20the%20Royal%20Statistical%20Society.Series%20B%20(Methodological)&title=Controlling%20the%20False%20Discovery%20Rate:%20A%20Practical%20and%20Powerful%20Approach%20to%20Multiple%20Testing.&author=Y%20Benjamini&author=Y%20Hochberg&volume=57&issue=1&publication_year=1995&pages=289-300&)

26. Barrett JC, Fry B, Maller J, et al. Haploview: analysis and visualization of LD and haplotype maps. Bioinformatics (Oxford, England) 2005;21(2):263–5. doi: 10.1093/bioinformatics/bth457.  [DOI](https://doi.org/10.1093/bioinformatics/bth457) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15297300/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics%20(Oxford,%20England)&title=Haploview:%20analysis%20and%20visualization%20of%20LD%20and%20haplotype%20maps.&author=JC%20Barrett&author=B%20Fry&author=J%20Maller&volume=21&issue=2&publication_year=2005&pages=263-5&pmid=15297300&doi=10.1093/bioinformatics/bth457&)

27. Horvath S, Xu X, Lake SL, et al. Family-based tests for associating haplotypes with general phenotype data: application to asthma genetics. Genetic epidemiology. 2004;26(1):61–9. doi: 10.1002/gepi.10295.  [DOI](https://doi.org/10.1002/gepi.10295) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14691957/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genetic%20epidemiology&title=Family-based%20tests%20for%20associating%20haplotypes%20with%20general%20phenotype%20data:%20application%20to%20asthma%20genetics.&author=S%20Horvath&author=X%20Xu&author=SL%20Lake&volume=26&issue=1&publication_year=2004&pages=61-9&pmid=14691957&doi=10.1002/gepi.10295&)

28. Caslake MJ, Packard CJ, Gaw A, et al. Fenofibrate and LDL metabolic heterogeneity in hypercholesterolemia. Arterioscler Thromb. 1993;13(1049-8834):702–711. doi: 10.1161/01.atv.13.5.702. (Print)  [DOI](https://doi.org/10.1161/01.atv.13.5.702) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8485122/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arterioscler%20Thromb&title=Fenofibrate%20and%20LDL%20metabolic%20heterogeneity%20in%20hypercholesterolemia.&author=MJ%20Caslake&author=CJ%20Packard&author=A%20Gaw&volume=13&issue=1049-8834&publication_year=1993&pages=702-711&pmid=8485122&doi=10.1161/01.atv.13.5.702&)

29. Kraja AT, Province MA, Straka RJ, et al. Fenofibrate and metabolic syndrome. Endocr Metab Immune Disord Drug Targets. 2010;10:138–148. doi: 10.2174/187153010791213047. (1871-5303 (Print) LA - eng PT - Journal Article PT - Research Support, N.I.H., Extramural PT - Review RN - 0 (Cholesterol, HDL) RN - 0 (Cholesterol, LDL) RN - 0 (Hydroxymethylglutaryl-CoA Reductase Inhibitors) RN - 0 (Hypolipidemic Agents) RN - 0 (Lipids)):

30. Reinhard W, Stark K, Sedlacek K, et al. Association between PPARalpha gene polymorphisms and myocardial infarction. Clin Sci (Lond) 2008;115(1470-8736):301–308. doi: 10.1042/CS20070391. (Electronic)  [DOI](https://doi.org/10.1042/CS20070391) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18336366/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Sci%20(Lond)&title=Association%20between%20PPARalpha%20gene%20polymorphisms%20and%20myocardial%20infarction.&author=W%20Reinhard&author=K%20Stark&author=K%20Sedlacek&volume=115&issue=1470-8736&publication_year=2008&pages=301-308&pmid=18336366&doi=10.1042/CS20070391&)

31. Brisson D, Ledoux K, Bosse Y, et al. Effect of apolipoprotein E, peroxisome proliferator-activated receptor alpha and lipoprotein lipase gene mutations on the ability of fenofibrate to improve lipid profiles and reach clinical guideline targets among hypertriglyceridemic patients. Pharmacogenetics. 2002;12(0960-314X):313–320. doi: 10.1097/00008571-200206000-00007. (Print)  [DOI](https://doi.org/10.1097/00008571-200206000-00007) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12042669/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=Effect%20of%20apolipoprotein%20E,%20peroxisome%20proliferator-activated%20receptor%20alpha%20and%20lipoprotein%20lipase%20gene%20mutations%20on%20the%20ability%20of%20fenofibrate%20to%20improve%20lipid%20profiles%20and%20reach%20clinical%20guideline%20targets%20among%20hypertriglyceridemic%20patients.&author=D%20Brisson&author=K%20Ledoux&author=Y%20Bosse&volume=12&issue=0960-314X&publication_year=2002&pages=313-320&pmid=12042669&doi=10.1097/00008571-200206000-00007&)

32. Foucher C, Rattier S, Flavell DM, et al. Response to micronized fenofibrate treatment is associated with the peroxisome-proliferator-activated receptors alpha G/C intron7 polymorphism in subjects with type 2 diabetes. Pharmacogenetics. 2004;14(0960-314X):823–829. doi: 10.1097/00008571-200412000-00005. (Print)  [DOI](https://doi.org/10.1097/00008571-200412000-00005) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15608561/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=Response%20to%20micronized%20fenofibrate%20treatment%20is%20associated%20with%20the%20peroxisome-proliferator-activated%20receptors%20alpha%20G/C%20intron7%20polymorphism%20in%20subjects%20with%20type%202%20diabetes.&author=C%20Foucher&author=S%20Rattier&author=DM%20Flavell&volume=14&issue=0960-314X&publication_year=2004&pages=823-829&pmid=15608561&doi=10.1097/00008571-200412000-00005&)

33. Jun M, Foote C, Lv J, et al. Effects of fibrates on cardiovascular outcomes: a systematic review and meta-analysis. Lancet. 2010;375(9729):1875–84. doi: 10.1016/S0140-6736(10)60656-3.  [DOI](https://doi.org/10.1016/S0140-6736(10)60656-3) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20462635/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Effects%20of%20fibrates%20on%20cardiovascular%20outcomes:%20a%20systematic%20review%20and%20meta-analysis.&author=M%20Jun&author=C%20Foote&author=J%20Lv&volume=375&issue=9729&publication_year=2010&pages=1875-84&pmid=20462635&doi=10.1016/S0140-6736(10)60656-3&)

34. Enquobahrie DA, Smith NL, Bis JC, et al. Cholesterol ester transfer protein, interleukin-8, peroxisome proliferator activator receptor alpha, and Toll-like receptor 4 genetic variations and risk of incident nonfatal myocardial infarction and ischemic stroke. Am J Cardiol. 2008;101(0002-9149):1683–1688. doi: 10.1016/j.amjcard.2008.02.052. (Print)  [DOI](https://doi.org/10.1016/j.amjcard.2008.02.052) | [PMC free article](/articles/PMC2577871/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18549840/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Cardiol&title=Cholesterol%20ester%20transfer%20protein,%20interleukin-8,%20peroxisome%20proliferator%20activator%20receptor%20alpha,%20and%20Toll-like%20receptor%204%20genetic%20variations%20and%20risk%20of%20incident%20nonfatal%20myocardial%20infarction%20and%20ischemic%20stroke.&author=DA%20Enquobahrie&author=NL%20Smith&author=JC%20Bis&volume=101&issue=0002-9149&publication_year=2008&pages=1683-1688&pmid=18549840&doi=10.1016/j.amjcard.2008.02.052&)

35. Shin T, Kuboki S, Huber N, et al. Activation of peroxisome proliferator-activated receptor-gamma during hepatic ischemia is age-dependent. J Surg.Res. 2008;147(0022-4804):200–205. doi: 10.1016/j.jss.2008.02.004. (Print)  [DOI](https://doi.org/10.1016/j.jss.2008.02.004) | [PMC free article](/articles/PMC2737330/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18498870/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Surg.Res&title=Activation%20of%20peroxisome%20proliferator-activated%20receptor-gamma%20during%20hepatic%20ischemia%20is%20age-dependent.&author=T%20Shin&author=S%20Kuboki&author=N%20Huber&volume=147&issue=0022-4804&publication_year=2008&pages=200-205&pmid=18498870&doi=10.1016/j.jss.2008.02.004&)

36. Belfort R, Berria R, Cornell J, et al. Fenofibrate reduces systemic inflammation markers independent of its effects on lipid and glucose metabolism in patients with the metabolic syndrome. Journal of Clinical Endocrinology Metabolism. 2010;95(1945-7197):829–836. doi: 10.1210/jc.2009-1487. (Electronic)  [DOI](https://doi.org/10.1210/jc.2009-1487) | [PMC free article](/articles/PMC2840858/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20061429/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Clinical%20Endocrinology%20Metabolism&title=Fenofibrate%20reduces%20systemic%20inflammation%20markers%20independent%20of%20its%20effects%20on%20lipid%20and%20glucose%20metabolism%20in%20patients%20with%20the%20metabolic%20syndrome.&author=R%20Belfort&author=R%20Berria&author=J%20Cornell&volume=95&issue=1945-7197&publication_year=2010&pages=829-836&pmid=20061429&doi=10.1210/jc.2009-1487&)

37. Kon KK, Yeal AJ, Hwan HS, et al. Effects of fenofibrate on lipoproteins, vasomotor function, and serological markers of inflammation, plaque stabilization, and hemostasis. Atherosclerosis. 2004;174(0021-9150):379–383. doi: 10.1016/j.atherosclerosis.2004.01.033. (Print)  [DOI](https://doi.org/10.1016/j.atherosclerosis.2004.01.033) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15136070/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Atherosclerosis&title=Effects%20of%20fenofibrate%20on%20lipoproteins,%20vasomotor%20function,%20and%20serological%20markers%20of%20inflammation,%20plaque%20stabilization,%20and%20hemostasis.&author=KK%20Kon&author=AJ%20Yeal&author=HS%20Hwan&volume=174&issue=0021-9150&publication_year=2004&pages=379-383&pmid=15136070&doi=10.1016/j.atherosclerosis.2004.01.033&)

38. Okopien B, Krysiak R, Herman ZS. Effects of short-term fenofibrate treatment on circulating markers of inflammation and hemostasis in patients with impaired glucose tolerance. Journal of Clinical Endocrinology Metabolism. 2006;91(0021-972X):1770–1778. doi: 10.1210/jc.2005-1615. (Print)  [DOI](https://doi.org/10.1210/jc.2005-1615) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16492702/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Clinical%20Endocrinology%20Metabolism&title=Effects%20of%20short-term%20fenofibrate%20treatment%20on%20circulating%20markers%20of%20inflammation%20and%20hemostasis%20in%20patients%20with%20impaired%20glucose%20tolerance.&author=B%20Okopien&author=R%20Krysiak&author=ZS%20Herman&volume=91&issue=0021-972X&publication_year=2006&pages=1770-1778&pmid=16492702&doi=10.1210/jc.2005-1615&)

39. Delerive P, Fruchart JC, Staels B. Peroxisome proliferator-activated receptors in inflammation control. J Endocrinol. 2001;169(0022-0795):453–459. doi: 10.1677/joe.0.1690453. (Print)  [DOI](https://doi.org/10.1677/joe.0.1690453) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11375115/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Endocrinol&title=Peroxisome%20proliferator-activated%20receptors%20in%20inflammation%20control.&author=P%20Delerive&author=JC%20Fruchart&author=B%20Staels&volume=169&issue=0022-0795&publication_year=2001&pages=453-459&pmid=11375115&doi=10.1677/joe.0.1690453&)

40. Guay DR. Micronized fenofibrate: a new fibric acid hypolipidemic agent . The Annals of Pharmacotherapy . 1999;33(10):1083–1103. doi: 10.1345/aph.18432.  [DOI](https://doi.org/10.1345/aph.18432) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10534222/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=The%20Annals%20of%20Pharmacotherapy&title=Micronized%20fenofibrate:%20a%20new%20fibric%20acid%20hypolipidemic%20agent%20.&author=DR%20Guay&volume=33&issue=10&publication_year=1999&pages=1083-1103&pmid=10534222&doi=10.1345/aph.18432&)

41. Michie S, Bobrow M, Marteau TM. Predictive genetic testing in children and adults: a study of emotional impact. J Med Genet. 2001;38(1468-6244):519–526. doi: 10.1136/jmg.38.8.519. (Electronic)  [DOI](https://doi.org/10.1136/jmg.38.8.519) | [PMC free article](/articles/PMC1734924/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11483640/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Med%20Genet&title=Predictive%20genetic%20testing%20in%20children%20and%20adults:%20a%20study%20of%20emotional%20impact.&author=S%20Michie&author=M%20Bobrow&author=TM%20Marteau&volume=38&issue=1468-6244&publication_year=2001&pages=519-526&pmid=11483640&doi=10.1136/jmg.38.8.519&)

42. Chen Y-H, Liu C-K, Chang S-C, et al. GenoWatch: a disease gene mining browser for association study. Nucleic acids research. 2008;36(Web Server issue):W336–40. doi: 10.1093/nar/gkn214.  [DOI](https://doi.org/10.1093/nar/gkn214) | [PMC free article](/articles/PMC2447740/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18440974/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20acids%20research&title=GenoWatch:%20a%20disease%20gene%20mining%20browser%20for%20association%20study.&author=Y-H%20Chen&author=C-K%20Liu&author=S-C%20Chang&volume=36&issue=Web%20Server%20issue&publication_year=2008&pages=W336-40&pmid=18440974&doi=10.1093/nar/gkn214&)
