# Functional Characterization of Liver Enhancers That Regulate Drug-Associated Transporters

## Metadata
**Authors:** MJ Kim, P Skewes-Cox, H Fukushima, S Hesselson, SW Yee, LB Ramsey, L Nguyen, JL Eshragh, RA Castro, CC Wen, D Stryke, SJ Johns, TE Ferrin, P-Y Kwok, MV Relling, KM Giacomini, DL Kroetz, N Ahituv
**Journal:** Clinical pharmacology and therapeutics
**Date:** 2011 Mar 2
**DOI:** [10.1038/clpt.2010.353](https://doi.org/10.1038/clpt.2010.353)
**PMID:** 21368754
**PMCID:** PMC3227682
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3227682/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC3227682/pdf/nihms337373.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC3227682/pdf/nihms337373.pdf)

## Abstract

Little is known about how genetic variations in enhancers influence drug response. In this study, we investigated whether nucleotide variations in enhancers that regulate drug transporters can alter their expression levels. Using comparative genomics and liver-specific transcription factor binding site (TFBS) analyses, we identified evolutionary conserved regions (ECRs) surrounding nine liver membrane transporters that interact with commonly used pharmaceuticals. The top 50 ECRs were screened for enhancer activity in vivo, of which five—located around ABCB11, SLC10A1, SLCO1B1, SLCO1A2, and SLC47A1—exhibited significant enhancer activity. Common variants identified in a large ethnically diverse cohort (n = 272) were assayed for differential enhancer activity, and three variants were found to have significant effects on reporter activity as compared with the reference allele. In addition, one variant was associated with reduced SLCO1A2 mRNA expression levels in human liver tissues, and another was associated with increased methotrexate (MTX) clearance in patients. This work provides a general model for the rapid characterization of liver enhancers and identifies associations between enhancer variants and drug response.

## Results

Liver membrane transporters have been shown to play an important role in regulating drug absorption, distribution, metabolism, elimination, and transport. For our study, we selected nine liver membrane transporters with well-documented drug substrates ([Table 1](#T1)).

### Computational analysis

Using ECRbase ([http://ecrbase.dcode.org](http://ecrbase.dcode.org)), we identified human–mouse noncoding evolutionary conserved regions (ECRs) within the genomic regions surrounding the nine liver membrane transporters, up to and including the immediate neighboring genes. We identified 621 ECRs ([Supplementary Table S1](#SD1) online) with at least 70% sequence identity between human and mouse that is at least 100 bp (the commonly used conservation parameters^24^).

Next, we prioritized our ECRs according to the presence of liver-associated TFBSs, using MATCH,^25^ a TFBS analysis program. The following TFBSs were chosen, on the basis of their previous implication in liver-specific gene regulation: *AP-1*, *C/ EBPβ*, *HNF-1*, *HNF-3β*, *GATA-3*, and *NF-1*.^26–28^ Each of the 621 ECRs were analyzed for the prevalence of these TFBSs and then ranked by the total number of liver-specific TFBSs per bp. Topranking ECRs were analyzed manually for repetitive sequences and any coding evidence, using the UCSC Genome Browser database ([http://genome.ucsc.edu](http://genome.ucsc.edu)), and ECRs that contained either of these were removed. The top 50 ECRs ([Supplementary Table S2](#SD2) online) were chosen for enhancer assays.

### Enhancer identification screen

The selected ECRs ([Supplementary Table S2](#SD2) online) were cloned into a luciferase reporter enhancer assay vector (pGL4.23; Promega, Madison, WI) and tested for enhancer activity using the *in vivo* hydrodynamic mouse tail vein assay ([Figure 1a](#F1)). In addition to the enhancer construct, we injected a *Renilla* luciferase construct, pGL4.74[*hRluc*/TK] (Promega), to correct for injection efficiency. On each injection day, we also injected an empty pGL4.23[*luc2*] vector and the apolipoprotein E (*APOE*) liver enhancer^29^ as negative and positive controls, respectively. At least three mice were injected per construct, including our controls. Our initial screen resulted in twelve ECRs exhibiting enhancer activity, as defined by an increase in luciferase activity of at least 1.8-fold relative to the negative control ([Figure 1b](#F1)). Each of these twelve ECRs was followed up by reinjection of the construct in at least nine mice in order to establish statistical significance. Following these reinjections, the enhancer activities of ECRs 11, 29, 32, 35, and 48 were found to be statistically significant (*P* < 0.05) as compared with the negative control ([Table 2](#T2)). These ECRs were in the vicinity of *ABCB11* (ECR11), *SLC10A1* (ECR29, ECR32), *SLC47A* (ECR48), and *SLCO1B1* and *SLCO1A2* (ECR35). *SLCO1A2* was not selected in our initial bioinformatic analyses screen; however, it is located downstream of *SLCO1B1* and was therefore a good candidate to consider.

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/fd06/3227682/9561aef40593/nihms337373f1.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3227682_nihms337373f1.jpg)

The hydrodynamic tail vein enhancer assay and results of the initial screen. (a) A schematic illustration of the in vivo hydrodynamic tail vein injection assay. Evolutionary conserved regions (ECRs) were cloned into a luciferase reporter plasmid and injected into the tail vein of the mouse. Relative luciferase activity was assayed 24 hours after the injection. (b) The initial enhancer screen yielded twelve ECRs (9, 11, 15, 16, 23, 25, 29, 32, 33, 34, 35, and 48) that exhibited relative luciferase activity ≥1.8-fold higher than the negative (−) control (pGL4.23). A known liver enhancer of the APOE gene was used as the positive (+) control. Three to five mice were injected for each construct.

### Table 2.

| Construct | Transporter | Initial enhancer screen (n = 3–5) | Follow-up screen (n = 6–9) | Combined (n = 9–12) | Combined P value |
| --- | --- | --- | --- | --- | --- |
| ECR9 | SLC10A1 | 6.78 | 0.02 | 2.17 | 0.453 |
| ECR11 | ABCB11 | 14.09 | 4.05 | 8.07 | 0.035 |
| ECR15 | SLC10A1 | 1.85 | 1.19 | 1.39 | 0.604 |
| ECR16 | SLCO1B1 | 2.00 | 1.38 | 1.58 | 0.117 |
| ECR23 | SLC47A1 | 2.26 | 0.94 | 1.30 | 0.483 |
| ECR25 | ABCC2 | 2.23 | 1.38 | 1.62 | 0.222 |
| ECR29 | SLC10A1 | 1.85 | 1.67 | 1.73 | 0.002 |
| ECR32 | SLC10A1 | 3.29 | 3.48 | 3.42 | <0.0001 |
| ECR33 | ABCB11 | 2.23 | 0.04 | 0.70 | 0.533 |
| ECR34 | SLC47A1 | 2.49 | 1.53 | 1.85 | 0.117 |
| ECR35 | SLCO1B1 | 4.93 | 4.92 | 4.92 | 0.008 |
| ECR48 | SLC47A1 | 4.41 | 2.72 | 3.28 | 0.001 |

Table 2 Caption: Follow-up enhancer assays

### Sequencing and functional variant analysis

We sequenced the five positive liver enhancers in 272 subjects from the SOPHIE (Studies of Pharmacogenetics in Ethnically Diverse Populations) cohort (as described in the Methods section). Common variants with a minor allele frequency *>*1% ([Table 3](#T3)) were selected for differential enhancer activity assays by comparing them to the reference allele (the allele with the largest frequency). Sequencing of the positive enhancers showed that some did not correspond to the reference allele. Consequently, we cloned the reference allele along with the other common alleles into the pGL4.23 enhancer assay vector by using the individuals with the specific haplotype as a template for amplification. Plasmids were verified through sequencing as having the correct haplotype. We injected each of these different haplotypes into at least nine mice. Of the eleven variant/haplotypes tested, 3 exhibited significant differential enhancer activity as compared with their respective reference alleles ([Table 3](#T3)). Two of the three variants were found in ECR35, and the third was found in ECR32.

### Table 3.

| ECR | Variant/haplotype | dbSNP ID | PMT SNP ID | Chromosomal location (hg18) | Nucleotide change | Allele frequency by ethnicity |  |  |  |  | P value (relative to REF) |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| AA | CA | AS | ME | RLA |  |  |  |  |  |  |  |
| 11 | REF |   |   |   |   | 0.937 | 0.984 | 0.893 | 0.935 | 3.152 |   |
|   | *2*3 |   | 5973 | Chr2: 169647588 | G→A | 0 | 0 | 0.024 | 0.065 | 4.536 | NS |
|   |   | rs73026864 | 5972 | Chr2: 169647612 | G→A |  |  |  |  |  |  |
|   | *3*5*6*7 | rs73026864 | 5972 | Chr2: 169647612 | G→A | 0.056 | 0 | 0 | 0 | 1.722 | NS |
|   |   | rs73026864 | 5970 | Chr2: 169647657 | G→A |  |  |  |  |  |  |
|   |   |   | 5969 | Chr2: 169647696 | G→A |  |  |  |  |  |  |
|   |   | rs73026869 | 5968 | Chr2: 169647698 | T→C |  |  |  |  |  |  |
|   | *8 |   | 5967 | Chr2: 169647904 | A→G | 0.024 | 0.008 | 0 | 0.008 | 1.779 | NS |
| 29 | REF |   |   |   |   | 0.890 | 0.993 | 0.978 | 1.000 | 1.462 |   |
|   | *6 | rs729458 | 6669 | Chr14: 69388040 | T→C | 0.081 | 0 | 0 | 0 | 2.086 | NS |
| 32 | REF |   |   |   |   | 0.780 | 0.694 | 0.821 | 0.735 | 0.981 |   |
|   | *1 | rs10162517 | 6677 | Chr14: 69413537 | A→G | 0.205 | 0.306 | 0.179 | 0.228 | 1.932 | 0.0118 |
| 35 | REF |   |   |   |   | 0.455 | 0.676 | 0.662 | 0.787 | 3.727 |   |
|   | *1*3 | rs11045981 | 6437 | Chr12: 21378250 | T→G | 0.037 | 0.147 | 0 | 0.074 | 2.577 | NS |
|   |   | rs4148981 | 6439 | Chr12: 21378335 | C→T |  |  |  |  |  |  |
|   | *2*3 |   | 6438 | Chr12: 21378301 | A→AA | 0.082 | 0.029 | 0 | 0.022 | 0.797 | <0.0001 |
|   |   | rs4148981 | 6439 | Chr12: 21378335 | C→T |  |  |  |  |  |  |
|   | *3 | rs4148981 | 6439 | Chr12: 21378335 | C→T | 0.425 | 0.147 | 0.338 | 0.118 | 1.081 | <0.0001 |
| 48 | REF |   |   |   |   | 0.632 | 0.669 | 0.433 | 0.559 | 2.661 |   |
|   | *1 | rs4646800 | 6459 | Chr17: 19502021 | T→G | 0.331 | 0.301 | 0.567 | 0.412 | 2.205 | NS |
|   | *3 |   | 6461 | Chr17: 19502026 | A→T | 0.044 | 0.007 | 0 | 0.007 | 2.759 | NS |

Table 3 Caption: The various ECR haplotypes and their enhancer activity levels

The results for ECR35 and its variants are particularly interesting. ECR35 has three common variants present in the SOPHIE cohort ([Table 3](#T3)). These SNPs, two of which— ECR35*1 (rs11045981) and ECR35*3 (rs4148981)—have been reported, and one novel SNP, ECR35*2, are present in most ethnic groups. ECR35*3 (rs4148981) is found in all four ethnic groups, with an average minor allele frequency of 26%, making it the most prevalent SNP that altered enhancer activity in our study. Of the four common haplotypes, three share ECR35*3; however, only the latter two haplotypes— ECR35*2,*3 and ECR35*3—exhibited significantly reduced enhancer activity ([Figure 2a](#F2); *P* < 0.0001; one-way analysis of variance with Dunnett’s multiple-comparison test). The ECR35*1*3 haplotype followed a similar trend but was not found to reduce reporter expression to a statistically significant extent.

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/fd06/3227682/1b89d65053ed/nihms337373f2.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3227682_nihms337373f2.jpg)

Enhancer activity of the different ECR35 haplotypes and their association with SLCO1A2 mRNA levels. (a) Differential enhancer activity levels of ECR35 haplotypes. ECR35*2,*3 and ECR*3 haplotypes exhibit significantly decreased enhancer activity as compared to the reference allele (one-way analysis of variance with Dunnett’s multiple-comparison test). (b) ECR35*3 haplotype is associated with a significantly lower expression of SLCO1A2 mRNA levels as compared with the reference allele (P = 0.0133, Kruskal–Wallis test followed by Dunn’s test). (c) The ECR35*1,*3 haplotype is not associated with differences in SLCO1A2 mRNA expression. ECR, evolutionary conserved region; Het, heterozygous; Hom, homozygous; REF, reference allele.

### Association of ECR35 variants with mRNA transporter expression levels in human liver

Next, we tested for association of mRNA expression levels with our functional enhancer variants. Because of the costs and the limited amount of human tissue samples available (*n* = 88), we only analyzed the variants for ECR35, which lies in an intron of *SLCO1A2*, the gene downstream of *SLCO1B1. SLCO1A2* is also known to interact with many prescription drugs such as erythromycin, statins, and methotrexate (MTX).^30^ Using real-time PCR, *SLCO1A2* and *SLCO1B1* mRNA levels were quantified in the 88 Caucasian human liver tissue samples. Extensive interindividual variation (mean ± SD) was noted for the *SLCO1A2* transporter (relative expression 284 ± 722). By contrast, *SLCO1B1* mRNA levels were much less variable (relative expression 22.8 ± 24.9).

Sequencing of all 88 samples revealed that the four ECR35 common haplotypes identified in the SOPHIE cohort were present in our tissue samples. However, the ECR35*2,*3 haplotype was found in only one of our samples and was therefore not analyzed further. We examined whether ECR35*1,*3 and ECR35*3 affected mRNA expression of each transporter. We did not observe any significant *SLCO1B1* mRNA expression level differences associating with either of these haplotypes (data not shown). However, we observed lower *SLCO1A2* mRNA levels associated with increasing numbers of the respective T allele for the ECR35*3 SNP ([Figure 2b](#F2); *P* = 0.0133; Kruskal–Wallis followed by Dunn’s test), as in the enhancer assay results. Comparison of ECR35*3 carriers and noncarriers yielded similar findings (*P* = 0.0128, Mann–Whitney test; data not shown). For the ECR35*1*3 haplotype, we did not observe a significant association with mRNA expression of *SLCO1A2* ([Figure 2c](#F2); *P* = 0.05, Kruskal–Wallis followed by Dunn’s test); this finding was analogous to our observations in the enhancer assays. Similar results were found when comparing ECR35*1*3 carriers and noncarriers (*P* > 0.05, Mann–Whitney test; data not shown).

### ECR35 SNPs and MTX clearance

OATP1A2 (the protein coded from *SLCO1A2*) has been shown to be important for the clearance of MTX, a drug widely used in the treatment of malignant and autoimmune diseases. In addition, through a genome-wide association study (GWAS), the *SLCO1B1*–*SLCO1A2* genomic region was recently associated with MTX clearance and gastrointestinal toxicity.^31^ We examined the ECR35 variants for their involvement in interindividual differences in MTX clearance. We genotyped ECR35 in the same population from the GWAS study,^31^ consisting of 639 children, all of whom received multiple courses of MTX at 2–5 g/m^2^. In this population, ECR35 SNPs were not found to be in strong linkage disequilibrium (LD) with the previously discovered GWAS-associated SNPs ([Figure 3](#F3); [Supplementary Table S3](#SD3) online). We analyzed these SNPs for their association with MTX clearance. Only haplotype ECR35*1*3 showed a significant association with MTX clearance, and the homozygous allele correlated with increased clearance as compared with the reference allele (*P* = 0.045; Kruskal–Wallis test; data not shown). The significance (*P* = 0.0354; Kruskal–Wallis test; data not shown) was also maintained after removing individuals carrying the nonsynonymous SNP, rs4149056 (*SLCO1B1* T521C), the most strongly associated SNP with MTX clearance.^31^ This suggests that the ECR35*1*3 haplotype may influence MTX clearance independent of the GWAS-detected SNPs. Analysis of the ECR35*1,*3 haplotype demonstrated that increasing copies of the ECR35*1,*3 allele significantly associated with increased MTX clearance ([Figure 4](#F4); *P* = 0.0261; Kruskal– Wallis). Interestingly, in this population, ECR35 SNP*1 is also in strong LD ([Figure 3](#F3); D′ = 0.991; *r*^2^ = 0.939) with rs10841795 (*SLC01A2* T38C), a SNP which has been previously shown to increase MTX transport.^32,33^

### Figure 3.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/fd06/3227682/8bd2398ffddc/nihms337373f3.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3227682_nihms337373f3.jpg)

Linkage disequilibrium (LD) plot of ECR35 SNPs with SLCO1B1 and SLCO1A2 SNPs that are associated with methotrexate clearance. ECR35 SNPs are not in LD with any of the previously described SLCO1B1 genome-wide association study SNPs.31 However, there is strong linkage (D′ = 0.99; r2 = 0.939) between ECR35*1 and the SLCO1A2 SNP rs10841795 (refs. 32,33).

### Figure 4.

![Figure 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/fd06/3227682/a8e5e154fc02/nihms337373f4.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3227682_nihms337373f4.jpg)

Haplotype analysis shows an association between the ECR35*1*3 haplotype and increased methotrexate (MTX) clearance. MTX clearance was adjusted for age, sex, race, and treatment arm. Regression analysis of the number of copies of the ECR35*1,*3 haplotype showed a significant association with MTX clearance (P = 0.0261), with the homozygous allele (2) correlating with increased clearance as compared with the reference allele (0). Individuals who are carriers of: *1*1*3*3 = 2; *1*3, *1*2*3*3, and *1*3*3 = 1; *2, *3, *2*3, and wild type = 0. ECR, evolutionary conserved region.

## Discussion

Using comparative genomics, we identified noncoding ECRs in the vicinity of nine liver membrane transporters (*SLC22A1*, *SLCO1B1*, *SLCO1B3*, *ABCC2*, *ABCB11*, *SLC22A7*, *SLCO2B1*, *SLC10A1*, and *SLC47A1*) that have well-characterized drug interactions. These sequences were then ranked for the prevalence of previously characterized liver TFBSs, and the top 50 sequences were assayed for liver enhancer activity using the mouse hydrodynamic tail vein injection technique. The initial screening resulted in twelve ECRs that exhibited luciferase activity at or above the cutoff value (1.8-fold). This value could be more stringent in future screenings because we saw that the majority of ECRs with activity between 1.8 and 2.0-fold decreased in reporter gene activity when followed up and subsequently lost statistical significance.

Five sequences that exhibited significant enhancer activity were further characterized for sequence variation in an ethnically diverse cohort. These were in the vicinity of *ABCB11*, *SLC10A1*, *SLCO1B1*, *SLCO1A2*, and *SLC47A1*. We found eleven common variants within these five ECRs that we assayed for differential enhancer expression. Of these, three showed significant enhancer activity differences as compared with their reference allele. These variants showed significant minor allele frequency differences between the ethnic groups, suggesting that enhancer variants can also contribute to ethnic-specific differences in drug response, similar to what has been observed for coding SNPs. We performed a more extensive analysis of one enhancer, ECR35. This enhancer is located within an intron of *SLCO1A2*, a transporter that interacts with a variety of statins, anticancer and antibacterial drugs.^30^ We identified four different common haplotypes within this ECR, two of which led to significantly reduced enhancer expression levels.

Using human liver tissue samples, we were able to show that ECR35*3 by itself led to reduced *SLCO1A2* mRNA levels. However, this SNP in combination with ECR35*1 did not cause a significant reduction of *SLCO1A2* mRNA levels despite demonstrating a trend to reduced expression. This suggests that ECR35*1 may mitigate the ability of ECR35*3 to reduce expression of *SLCO1A2*. Analysis of ECR35 variants for the six liver-specific TFBSs used to rank the ECRs found no changes in binding sites for either ECR35*1 or ECR35*3.

We observed a correlation for the ECR35 variants between the enhancer assay results and *SLCO1A2* mRNA expression levels. Our studies were limited due to our small human liver tissue sample size (*n* = 88) and the large variation in *SLCO1A2* mRNA expression levels (22.8 ± 24.9). Additional studies would have to be done to explicitly link ECR35 SNP*3 as the causative SNP that leads to reduced *SLCO1A2* expression. We also found another variant that exhibited increased enhancer activity in another enhancer, ECR32, which is near *SLC10A1*, an uptake transporter for bile salts and cyclosporin A.^3^ Performing drug-associated studies with this variant would also be of interest because the ECR32*1 variant led to increased reporter gene activity relative to the reference allele.

With ECR35 variants leading to significantly reduced reporter gene and mRNA expression levels, we analyzed their potential effects on OATP1A2 and its associated drug interactions. Coding mutations in OATP1A2 have been shown to affect the transport of MTX^32^ and other substances,^33^ and the *SLCO1B1–SLCO1A2* genomic region has been found to be associated with reduced MTX clearance and gastrointestinal toxicity.^31^ Potentially, the reduced MTX clearance could be due to lower *SLCO1A2* mRNA expression, attributable to the ECR35 variants. Although we did not observe ECR35 SNPs to be in strong LD with the MTX GWAS-associated SNPs in the *SLCO1B1*–*SLCO1A2* genomic region, we did observe haplotype ECR35*1*3 to be associated with increased MTX clearance. This haplotype led to slightly reduced reporter gene expression in the enhancer assay but did not meet the criteria for significance ([Table 3](#T3)). However, in this population, SNP*1 was found to be in strong LD (D′ = 0.991; *r*^2^ = 0.939) with the nonsynonymous *SLCO1A2* rs10841795 SNP (T38C; Ile13Thr), which was previously shown to lead to increased MTX uptake when expressed in *Xenopus laevis* oocytes.^32^ In the kidney, the predominant organ involved in MTX elimination, OATP1A2 is localized to the apical domain of distal nephrons.^33^ Higher MTX uptake, as observed in association with rs10841795, should lead to increased reabsorption and lower MTX clearance, which is contrary to our observations. The differences between our *in vivo* observations and these functional studies might be due to the assay having been carried out *in vitro* and/or the differences in pH concentrations in the kidney. The pH of the distal tubule is variable, and MTX transport by the reference OATP1A2 allele has been shown to be highly dependent on pH.^32^

Further studies need to be performed to determine the actual *in vivo* functional outcomes of rs10841795 and SNP*1, considering their effect on OATP1A2 in the kidney, liver, intestine, and brain. As for ECR35*3, all of our assays were carried out on liver tissues; therefore, its functional role in the kidney—the major organ associated with MTX clearance—is unknown. It would be interesting to investigate whether ECR35 is important for *SLCO1A2* expression in the kidney and whether the enhancer affects interindividual drug interactions of other OATP1A2- transported drugs in general.

This study is the first to link nucleotide variations in enhancers with drug-associated genes, providing further evidence that regulatory sequences could be important determinants of differential drug response and ADEs. With the advancement of sequencing technologies, individual genome sequences will soon be available at an affordable price to the general public. The ability to obtain the genetic blueprint of an individual will greatly advance the quality of pharmacological treatment by tailoring drugs to fit that blueprint and ultimately reduce ADEs and improve drug efficacy overall. However, the major hurdle will be the development of high-throughput functional assays that can rapidly interpret the pharmacological nature of the nucleotide change(s) in a given individual, both in coding and in regulatory sequences.

## Methods

### Identification and ranking of ECRs

Using ECRbase ([http://ecrbase.dcode.org](http://ecrbase.dcode.org)), we selected intronic and intergenic ECRs for each transporter that were at least 100 bp long and with at least 70% sequence identity between human and mouse. We parsed conservation data from ECRbase using in-house software (available upon request). ECRs were chosen in proximity to the gene all the way to the next neighboring gene including the neighboring gene’s entire genomic sequence. In the case of *ABCB11*, given the close proximity of the 3′ neighboring gene, *G6PC2* (~13 kb), we searched for ECRs two genes away from the transporter, including the genomic region of *SPC25*. This analysis generated 621 unique ECRs ([Supplementary Table S1](#SD1) online). To prioritize ECRs for functional assays, we scanned these ECRs for liverspecific TFBSs: *AP-1*, *C/EBPβ*, *HNF-1*, *HNF-3β*, *GATA-3*, and *NF-1*, using MATCH.^25^ The ECRs were ranked based on the total number of liver-specific TFBS per bp of each ECR.

### Cloning of ECRs

The top 50 ECRs ranked by the number of TFBSs per bp of ECR were tested for enhancer activity. Primers ([Supplementary Table S2](#SD2) online) were designed to have up to 200 bp of sequence flanking each side of the human–mouse conserved sequence (previous experiments have shown this addition to be a reliable method for obtaining positive enhancer activity^22^). ECRs were PCR-amplified using TopTaq (Qiagen, Germantown, MD) from human genomic DNA (Roche, Indianapolis, IN), purified by gel extraction (QIAquick gel extraction kit; Qiagen) and cloned into the pENTR-dTOPO vector (Invitrogen, Carlsbad, CA). After plasmid DNA purification using the QIAprep spin miniprep kit (Qiagen), proper orientation was confirmed by restriction enzyme digest. Clones in correct orientation were subsequently subcloned into the pGL4.23[*luc2*] vector (Promega) containing the Gateway reading frame A (Invitrogen). Orientation of the insert was re-verified by restriction enzyme digest, and endotoxin-free plasmid DNA was isolated using the EndoFree Plasmid Midi prep (Qiagen). ECR19 was not followed up with injections because of cloning difficulties.

### In vivo hydrodynamic tail vein enhancer assay

For the hydrodynamic tail vein assay, each assayed sequence, cloned in the pGL4.23[*luc2*], was initially injected (10 μg) alongside 2 μg of pGL4.74[*hRluc*/TK] vector to correct for injection efficiency, into at least three CD1 mice (Charles River Laboratories, Wilmington, MA) using the *Trans*IT EE hydrodynamic gene delivery system (Mirus Bio, Madison, WI) according to the manufacturer’s protocol. Negative (empty pGL4.23[*luc2*]) and positive (*ApoE* liver enhancer)^29^ controls (*n* = 3–5) were also injected in parallel at each injection date/experiment. After 24 h, the mice were euthanized, and the livers were harvested and homogenized in passive lysis buffer (Promega), followed by centrifugation at 4 °C for 30 min at 14,000 rpm. Firefly luciferase and *Renilla* luciferase activity in the supernatant (diluted 1:20) were measured on a Synergy 2 microplate reader (BioTek Instruments, Winooski, VT) in replicates of six for each liver, using the Dual-Luciferase reporter assay system (Promega). The ratios for firefly luciferase:*Renilla* luciferase were determined and expressed as relative luciferase activity. Experiments for each construct were repeated in at least nine mice if analysis of the initial screen had an average increase in activity that was at least 1.8-fold higher than the negative control. All mouse work was approved by the UCSF Institutional Animal Care and Use Committee.

### Nucleotide variant sequencing and cloning

Selected positive liver enhancers were sequenced in a panel of 272 ethnically diverse subjects (African Americans (41 female, 27 male), Caucasians (34 female, 34 male), Chinese (44 female, 24 male), and Mexicans (49 female, 19 male)) from the SOPHIE cohort.^19^ In addition, human liver tissues and the MTX cohort were also sequenced as previously described.^31^ Enhancers with common variants (minor allele frequency *>*1%) were PCR-amplified from human genomic DNA from SOPHIE cohort individuals encompassing this variant/haplotype as listed in [Table 2](#T2). The amplified DNA was inserted into the pENTR-dTOPO vector as previously described and directly sequenced to verify the variant(s) before being subcloned into the pGL4.23[*luc2*] vector. All enhancer sequencing results can be found on the PMT database ([http://pharmacogenetics.ucsf.edu](http://pharmacogenetics.ucsf.edu)).

### Human liver samples

Liver tissue samples were purchased from Asterand (Detroit, MI) and donated by SRI International (Menlo Park, CA). All tissues were flash frozen immediately after collection and stored at −80 °C. Isolation of total RNA and DNA from the tissues was performed using Qiagen Allprep DNA/RNA kits according to the manufacturer’s protocol. Equal amounts of total RNA were reverse transcribed to cDNA using a high-capacity cDNA reverse transcription kit (Applied Biosystems, Foster City, CA). Quantitative real-time PCRs for *SLCO1A2*, *SLCO1B1*, and β-actin were carried out with ready-to-use TaqMan gene expression assays (Applied Biosystems). The thermal cycling conditions used were as follows: 2 min at 50 °C, 10 min at 95 °C followed by 15 s at 95 °C, and 1 min at 60 °C, for 40 cycles. The real-time PCR efficiencies were evaluated with twofold serial dilutions in a pool of all starting RNAs in order to confirm accuracy and linearity. Efficiency was calculated from the slopes of the calibration curve according to the equation E = 10 ^[−1/slope]^, as previously described.^34^ Real-time PCR efficiencies of *SLCO1A2*, *SLCO1B1*, and β-actin were 1.90 ± 0.05, 1.91 ± 0.08, and 2.02 ± 0.10, respectively, which were consistently similar to a static efficiency of 2. Therefore, expression values of both target genes were normalized to β-actin, and the 2^−ΔCt^ standard method of relative quantification was used. Genotyping was performed by direct sequencing as described above. This study was approved by the UCSF Committee on Human Research.

### Pharmacokinetic data for patients receiving MTX

A total of 639 children with acute lymphoblastic leukemia who were receiving treatment according to St Jude Children’s Research Hospital’s Total XIIIB and Total XV protocols were genotyped for ECR35 by Sanger sequencing.^30^ Germline DNA was extracted from the blood after remission was achieved. MTX clearance was estimated as described previously.^30^

### Statistical analysis

Data are expressed as the mean ratio firefly luciferase:*Renilla* luciferase activity as compared with the negative control (*+* SEM). Statistical analysis for enhancer identification was carried out using the unpaired Student’s *t*-test, assuming equal variance compared to the negative control. The activity levels of the variants were also analyzed using the unpaired *t*-test and compared with the enhancer’s reference luciferase activity. Statistical comparison of luciferase activity for ECR11, ECR35, and ECR48 variants with their respective reference alleles was performed using one-way analysis of variance followed by Dunnett’s multiple- comparison test. Statistical comparisons between different genotypes and haplotypes from the human liver tissue samples were made using the Mann–Whitney and Kruskal–Wallis tests followed by Dunn’s multiple-comparison test. Data were analyzed using GraphPad Prism, version 5.0 (GraphPad, La Jolla, CA). Associations between MTX clearance and ECR haplotypes were evaluated using a general linear model, treating genotype as a numerical variable (0 = AA, 1 = AB, and 2 = BB) and including age, race, sex, and treatment regimen as covariates.^30^ LD was calculated using Haploview ([http://www.broadinstitute.org/haploview](http://www.broadinstitute.org/haploview)). A *P* value ≤0.05 was considered statistically significant.

## Supplementary Material

## Acknowledgments

This research was supported by NIH NIGMS grant GM61390 and by NIH/ NCRR UCSF-CTSI grant UL1 RR024131. N.A. is also supported by award R01HD059862 from the NICHD and award R01HG005058 from the NHGRI. Grant support for L.B.R. and M.V.R. came from NIH R37 CA36401, U01 GM92666, P30 CA21765, and ALSAC (American Lebanese Syrian Associated Charities). We thank the Ahituv lab for helpful comments on the manuscript. We also thank the clinical and research staff at St Jude Children’s Research Hospital, particularly Wenjian Yang, Pamela McGill, and Jeremy Hunt, as well as the patients and their families for their participation. We thank the staff of the St Jude Hartwell Center for Bioinformatics and Biotechnology. The content of this paper is solely the responsibility of the authors and does not necessarily represent the official views of the NIH, the NIGMS, the NICHD, or the NHGRI.

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Murphy SL. Deaths: final data for 1998. Natl Vital Stat Rep. 2000;48:1–105.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/10934859/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Natl%20Vital%20Stat%20Rep&title=Deaths:%20final%20data%20for%201998&author=SL%20Murphy&volume=48&publication_year=2000&pages=1-105&pmid=10934859&)

2. Burt CW, McCaig LF, Rechtsteiner EA. Ambulatory medical care utilization estimates for 2005. Adv Data. 2007;388:1–15.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/17784725/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Adv%20Data&title=Ambulatory%20medical%20care%20utilization%20estimates%20for%202005&author=CW%20Burt&author=LF%20McCaig&author=EA%20Rechtsteiner&volume=388&publication_year=2007&pages=1-15&pmid=17784725&)

3. Sadée W, Dai Z. Pharmacogenetics/genomics and personalized medicine. Hum Mol Genet. 2005;14 (spec 2):R207–R214. doi: 10.1093/hmg/ddi261.  [DOI](https://doi.org/10.1093/hmg/ddi261) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16244319/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mol%20Genet&title=Pharmacogenetics/genomics%20and%20personalized%20medicine&author=W%20Sad%C3%A9e&author=Z%20Dai&volume=14&issue=spec%202&publication_year=2005&pages=R207-R214&pmid=16244319&doi=10.1093/hmg/ddi261&)

4. Schillani G, et al. 5-HTTLPR polymorphism of serotonin transporter and effects of sertraline in terminally ill cancer patients: report of eleven cases. Tumori. 2008;94:563–567. doi: 10.1177/030089160809400419.  [DOI](https://doi.org/10.1177/030089160809400419) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18822694/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Tumori&title=5-HTTLPR%20polymorphism%20of%20serotonin%20transporter%20and%20effects%20of%20sertraline%20in%20terminally%20ill%20cancer%20patients:%20report%20of%20eleven%20cases&author=G%20Schillani&volume=94&publication_year=2008&pages=563-567&pmid=18822694&doi=10.1177/030089160809400419&)

5. Yee SW, et al. Identification and characterization of proximal promoter polymorphisms in the human concentrative nucleoside transporter 2 (SLC28A2) J Pharmacol Exp Ther. 2009;328:699–707. doi: 10.1124/jpet.108.147207.  [DOI](https://doi.org/10.1124/jpet.108.147207) | [PMC free article](/articles/PMC2682268/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19098160/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharmacol%20Exp%20Ther&title=Identification%20and%20characterization%20of%20proximal%20promoter%20polymorphisms%20in%20the%20human%20concentrative%20nucleoside%20transporter%202%20(SLC28A2)&author=SW%20Yee&volume=328&publication_year=2009&pages=699-707&pmid=19098160&doi=10.1124/jpet.108.147207&)

6. Choi JH, et al. Identification and characterization of novel polymorphisms in the basal promoter of the human transporter, MATE1. Pharmacogenet Genomics. 2009;19:770–780. doi: 10.1097/FPC.0b013e328330eeca.  [DOI](https://doi.org/10.1097/FPC.0b013e328330eeca) | [PMC free article](/articles/PMC2976711/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19745787/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Identification%20and%20characterization%20of%20novel%20polymorphisms%20in%20the%20basal%20promoter%20of%20the%20human%20transporter,%20MATE1&author=JH%20Choi&volume=19&publication_year=2009&pages=770-780&pmid=19745787&doi=10.1097/FPC.0b013e328330eeca&)

7. Silva H, et al. Serotonin transporter polymorphism and fluoxetine effect on impulsiveness and aggression in borderline personality disorder. Actas Esp Psiquiatr. 2007;35:387–392.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/17597424/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Actas%20Esp%20Psiquiatr&title=Serotonin%20transporter%20polymorphism%20and%20fluoxetine%20effect%20on%20impulsiveness%20and%20aggression%20in%20borderline%20personality%20disorder&author=H%20Silva&volume=35&publication_year=2007&pages=387-392&pmid=17597424&)

8. Serretti A, Kato M, De Ronchi D, Kinoshita T. Meta-analysis of serotonin transporter gene promoter polymorphism (5-HTTLPR) association with selective serotonin reuptake inhibitor efficacy in depressed patients. Mol Psychiatry. 2007;12:247–257. doi: 10.1038/sj.mp.4001926.  [DOI](https://doi.org/10.1038/sj.mp.4001926) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17146470/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Psychiatry&title=Meta-analysis%20of%20serotonin%20transporter%20gene%20promoter%20polymorphism%20(5-HTTLPR)%20association%20with%20selective%20serotonin%20reuptake%20inhibitor%20efficacy%20in%20depressed%20patients&author=A%20Serretti&author=M%20Kato&author=D%20De%20Ronchi&author=T%20Kinoshita&volume=12&publication_year=2007&pages=247-257&pmid=17146470&doi=10.1038/sj.mp.4001926&)

9. Lankisch TO, et al. Identification and characterization of a functional TATA box polymorphism of the UDP glucuronosyltransferase 1A7 gene. Mol Pharmacol. 2005;67:1732–1739. doi: 10.1124/mol.104.007146.  [DOI](https://doi.org/10.1124/mol.104.007146) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15716465/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Pharmacol&title=Identification%20and%20characterization%20of%20a%20functional%20TATA%20box%20polymorphism%20of%20the%20UDP%20glucuronosyltransferase%201A7%20gene&author=TO%20Lankisch&volume=67&publication_year=2005&pages=1732-1739&pmid=15716465&doi=10.1124/mol.104.007146&)

10. Sadee W. Regulatory polymorphisms in key candidate genes for disease susceptibility and drug response: a mandate for valid genetic biomarkers. Expert Rev Mol Diagn. 2010;10:9–11. doi: 10.1586/erm.09.72.  [DOI](https://doi.org/10.1586/erm.09.72) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20014917/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Expert%20Rev%20Mol%20Diagn&title=Regulatory%20polymorphisms%20in%20key%20candidate%20genes%20for%20disease%20susceptibility%20and%20drug%20response:%20a%20mandate%20for%20valid%20genetic%20biomarkers&author=W%20Sadee&volume=10&publication_year=2010&pages=9-11&pmid=20014917&doi=10.1586/erm.09.72&)

11. Giacomini KM, et al. Membrane transporters in drug development. Nat Rev Drug Discov. 2010;9:215–236. doi: 10.1038/nrd3028.  [DOI](https://doi.org/10.1038/nrd3028) | [PMC free article](/articles/PMC3326076/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20190787/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Drug%20Discov&title=Membrane%20transporters%20in%20drug%20development&author=KM%20Giacomini&volume=9&publication_year=2010&pages=215-236&pmid=20190787&doi=10.1038/nrd3028&)

12. Ieiri I, Higuchi S. Pharmacogenomics: inter-ethnic and intra-ethnic differences in pharmacokinetic and pharmacodynamic profiles of clinically relevant drugs. Yakugaku Zasshi. 2009;129:231–235. doi: 10.1248/yakushi.129.231.  [DOI](https://doi.org/10.1248/yakushi.129.231) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19182452/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Yakugaku%20Zasshi&title=Pharmacogenomics:%20inter-ethnic%20and%20intra-ethnic%20differences%20in%20pharmacokinetic%20and%20pharmacodynamic%20profiles%20of%20clinically%20relevant%20drugs&author=I%20Ieiri&author=S%20Higuchi&volume=129&publication_year=2009&pages=231-235&pmid=19182452&doi=10.1248/yakushi.129.231&)

13. Kerb R. Implications of genetic polymorphisms in drug transporters for pharmacotherapy. Cancer Lett. 2006;234:4–33. doi: 10.1016/j.canlet.2005.06.051.  [DOI](https://doi.org/10.1016/j.canlet.2005.06.051) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16504381/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Lett&title=Implications%20of%20genetic%20polymorphisms%20in%20drug%20transporters%20for%20pharmacotherapy&author=R%20Kerb&volume=234&publication_year=2006&pages=4-33&pmid=16504381&doi=10.1016/j.canlet.2005.06.051&)

14. Gerloff T. Impact of genetic polymorphisms in transmembrane carriersystems on drug and xenobiotic distribution. Naunyn Schmiedebergs Arch Pharmacol. 2004;369:69–77. doi: 10.1007/s00210-003-0813-5.  [DOI](https://doi.org/10.1007/s00210-003-0813-5) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14598019/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Naunyn%20Schmiedebergs%20Arch%20Pharmacol&title=Impact%20of%20genetic%20polymorphisms%20in%20transmembrane%20carriersystems%20on%20drug%20and%20xenobiotic%20distribution&author=T%20Gerloff&volume=369&publication_year=2004&pages=69-77&pmid=14598019&doi=10.1007/s00210-003-0813-5&)

15. Kivistö KT, Niemi M. Influence of drug transporter polymorphisms on pravastatin pharmacokinetics in humans. Pharm Res. 2007;24:239–247. doi: 10.1007/s11095-006-9159-2.  [DOI](https://doi.org/10.1007/s11095-006-9159-2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17177112/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharm%20Res&title=Influence%20of%20drug%20transporter%20polymorphisms%20on%20pravastatin%20pharmacokinetics%20in%20humans&author=KT%20Kivist%C3%B6&author=M%20Niemi&volume=24&publication_year=2007&pages=239-247&pmid=17177112&doi=10.1007/s11095-006-9159-2&)

16. Lang C, et al. Mutations and polymorphisms in the bile salt export pump and the multidrug resistance protein 3 associated with drug-induced liver injury. Pharmacogenet Genomics. 2007;17:47–60. doi: 10.1097/01.fpc.0000230418.28091.76.  [DOI](https://doi.org/10.1097/01.fpc.0000230418.28091.76) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17264802/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Mutations%20and%20polymorphisms%20in%20the%20bile%20salt%20export%20pump%20and%20the%20multidrug%20resistance%20protein%203%20associated%20with%20drug-induced%20liver%20injury&author=C%20Lang&volume=17&publication_year=2007&pages=47-60&pmid=17264802&doi=10.1097/01.fpc.0000230418.28091.76&)

17. Shu Y, et al. Effect of genetic variation in the organic cation transporter 1 (OCT1) on metformin action. J Clin Invest. 2007;117:1422–1431. doi: 10.1172/JCI30558.  [DOI](https://doi.org/10.1172/JCI30558) | [PMC free article](/articles/PMC1857259/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17476361/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Invest&title=Effect%20of%20genetic%20variation%20in%20the%20organic%20cation%20transporter%201%20(OCT1)%20on%20metformin%20action&author=Y%20Shu&volume=117&publication_year=2007&pages=1422-1431&pmid=17476361&doi=10.1172/JCI30558&)

18. Tahara H, et al. Functional genetic variation in the basal promoter of the organic cation/carnitine transporters OCTN1 (SLC22A4) and OCTN2 (SLC22A5) J Pharmacol Exp Ther. 2009;329:262–271. doi: 10.1124/jpet.108.146449.  [DOI](https://doi.org/10.1124/jpet.108.146449) | [PMC free article](/articles/PMC2670604/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19141711/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharmacol%20Exp%20Ther&title=Functional%20genetic%20variation%20in%20the%20basal%20promoter%20of%20the%20organic%20cation/carnitine%20transporters%20OCTN1%20(SLC22A4)%20and%20OCTN2%20(SLC22A5)&author=H%20Tahara&volume=329&publication_year=2009&pages=262-271&pmid=19141711&doi=10.1124/jpet.108.146449&)

19. Hesselson SE, et al. Genetic variation in the proximal promoter of ABC and SLC superfamilies: liver and kidney specific expression and promoter activity predict variation. PLoS ONE. 2009;4:e6942. doi: 10.1371/journal.pone.0006942.  [DOI](https://doi.org/10.1371/journal.pone.0006942) | [PMC free article](/articles/PMC2735003/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19742321/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&title=Genetic%20variation%20in%20the%20proximal%20promoter%20of%20ABC%20and%20SLC%20superfamilies:%20liver%20and%20kidney%20specific%20expression%20and%20promoter%20activity%20predict%20variation&author=SE%20Hesselson&volume=4&publication_year=2009&pages=e6942&pmid=19742321&doi=10.1371/journal.pone.0006942&)

20. Sun C, Southard C, Witonsky DB, Olopade OI, Di Rienzo A. Allelic imbalance (AI) identifies novel tissue-specific cis-regulatory variation for human UGT2B15. Hum Mutat. 2010;31:99–107. doi: 10.1002/humu.21145.  [DOI](https://doi.org/10.1002/humu.21145) | [PMC free article](/articles/PMC2922057/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19847790/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mutat&title=Allelic%20imbalance%20(AI)%20identifies%20novel%20tissue-specific%20cis-regulatory%20variation%20for%20human%20UGT2B15&author=C%20Sun&author=C%20Southard&author=DB%20Witonsky&author=OI%20Olopade&author=A%20Di%20Rienzo&volume=31&publication_year=2010&pages=99-107&pmid=19847790&doi=10.1002/humu.21145&)

21. Woolfe A, et al. Highly conserved non-coding sequences are associated with vertebrate development. PLoS Biol. 2005;3:116–130. doi: 10.1371/journal.pbio.0030007.  [DOI](https://doi.org/10.1371/journal.pbio.0030007) | [PMC free article](/articles/PMC526512/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15630479/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Biol&title=Highly%20conserved%20non-coding%20sequences%20are%20associated%20with%20vertebrate%20development&author=A%20Woolfe&volume=3&publication_year=2005&pages=116-130&pmid=15630479&doi=10.1371/journal.pbio.0030007&)

22. Pennacchio LA, et al. In vivo enhancer analysis of human conserved noncoding sequences. Nature. 2006;444:499–502. doi: 10.1038/nature05295.  [DOI](https://doi.org/10.1038/nature05295) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17086198/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=In%20vivo%20enhancer%20analysis%20of%20human%20conserved%20noncoding%20sequences&author=LA%20Pennacchio&volume=444&publication_year=2006&pages=499-502&pmid=17086198&doi=10.1038/nature05295&)

23. Zhang G, Budker V, Wolff JA. High levels of foreign gene expression in hepatocytes after tail vein injections of naked plasmid DNA. Hum Gene Ther. 1999;10:1735–1737. doi: 10.1089/10430349950017734.  [DOI](https://doi.org/10.1089/10430349950017734) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10428218/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Gene%20Ther&title=High%20levels%20of%20foreign%20gene%20expression%20in%20hepatocytes%20after%20tail%20vein%20injections%20of%20naked%20plasmid%20DNA&author=G%20Zhang&author=V%20Budker&author=JA%20Wolff&volume=10&publication_year=1999&pages=1735-1737&pmid=10428218&doi=10.1089/10430349950017734&)

24. Nóbrega MA, Zhu Y, Plajzer-Frick I, Afzal V, Rubin EM. Megabase deletions of gene deserts result in viable mice. Nature. 2004;431:988–993. doi: 10.1038/nature03022.  [DOI](https://doi.org/10.1038/nature03022) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15496924/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Megabase%20deletions%20of%20gene%20deserts%20result%20in%20viable%20mice&author=MA%20N%C3%B3brega&author=Y%20Zhu&author=I%20Plajzer-Frick&author=V%20Afzal&author=EM%20Rubin&volume=431&publication_year=2004&pages=988-993&pmid=15496924&doi=10.1038/nature03022&)

25. Kel AE, Gössling E, Reuter I, Cheremushkin E, Kel-Margoulis OV, Wingender E. MATCH: A tool for searching transcription factor binding sites in DNA sequences. Nucleic Acids Res. 2003;31:3576–3579. doi: 10.1093/nar/gkg585.  [DOI](https://doi.org/10.1093/nar/gkg585) | [PMC free article](/articles/PMC169193/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12824369/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=MATCH:%20A%20tool%20for%20searching%20transcription%20factor%20binding%20sites%20in%20DNA%20sequences&author=AE%20Kel&author=E%20G%C3%B6ssling&author=I%20Reuter&author=E%20Cheremushkin&author=OV%20Kel-Margoulis&volume=31&publication_year=2003&pages=3576-3579&pmid=12824369&doi=10.1093/nar/gkg585&)

26. Krivan W, Wasserman WW. A predictive model for regulatory sequences directing liver-specific transcription. Genome Res. 2001;11:1559–1566. doi: 10.1101/gr.180601.  [DOI](https://doi.org/10.1101/gr.180601) | [PMC free article](/articles/PMC311083/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11544200/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Res&title=A%20predictive%20model%20for%20regulatory%20sequences%20directing%20liver-specific%20transcription&author=W%20Krivan&author=WW%20Wasserman&volume=11&publication_year=2001&pages=1559-1566&pmid=11544200&doi=10.1101/gr.180601&)

27. Vandenbon A, Nakai K. Modeling tissue-specific structural patterns in human and mouse promoters. Nucleic Acids Res. 2010;38:17–25. doi: 10.1093/nar/gkp866.  [DOI](https://doi.org/10.1093/nar/gkp866) | [PMC free article](/articles/PMC2800225/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19850720/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=Modeling%20tissue-specific%20structural%20patterns%20in%20human%20and%20mouse%20promoters&author=A%20Vandenbon&author=K%20Nakai&volume=38&publication_year=2010&pages=17-25&pmid=19850720&doi=10.1093/nar/gkp866&)

28. Matys V, et al. TRANSFAC: transcriptional regulation, from patterns to profiles. Nucleic Acids Res. 2003;31:374–378. doi: 10.1093/nar/gkg108.  [DOI](https://doi.org/10.1093/nar/gkg108) | [PMC free article](/articles/PMC165555/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12520026/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=TRANSFAC:%20transcriptional%20regulation,%20from%20patterns%20to%20profiles&author=V%20Matys&volume=31&publication_year=2003&pages=374-378&pmid=12520026&doi=10.1093/nar/gkg108&)

29. Simonet WS, Bucay N, Lauer SJ, Taylor JM. A far-downstream hepatocyte-specific control region directs expression of the linked human apolipoprotein E and C-I genes in transgenic mice. J Biol Chem. 1993;268:8221–8229.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/7681840/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=A%20far-downstream%20hepatocyte-specific%20control%20region%20directs%20expression%20of%20the%20linked%20human%20apolipoprotein%20E%20and%20C-I%20genes%20in%20transgenic%20mice&author=WS%20Simonet&author=N%20Bucay&author=SJ%20Lauer&author=JM%20Taylor&volume=268&publication_year=1993&pages=8221-8229&pmid=7681840&)

30. Kalliokoski A, Niemi M. Impact of OATP transporters on pharmacokinetics. Br J Pharmacol. 2009;158:693–705. doi: 10.1111/j.1476-5381.2009.00430.x.  [DOI](https://doi.org/10.1111/j.1476-5381.2009.00430.x) | [PMC free article](/articles/PMC2765590/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19785645/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Pharmacol&title=Impact%20of%20OATP%20transporters%20on%20pharmacokinetics&author=A%20Kalliokoski&author=M%20Niemi&volume=158&publication_year=2009&pages=693-705&pmid=19785645&doi=10.1111/j.1476-5381.2009.00430.x&)

31. Treviño LR, et al. Germline genetic variation in an organic anion transporter polypeptide associated with methotrexate pharmacokinetics and clinical effects. J Clin Oncol. 2009;27:5972–5978. doi: 10.1200/JCO.2008.20.4156.  [DOI](https://doi.org/10.1200/JCO.2008.20.4156) | [PMC free article](/articles/PMC2793040/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19901119/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Germline%20genetic%20variation%20in%20an%20organic%20anion%20transporter%20polypeptide%20associated%20with%20methotrexate%20pharmacokinetics%20and%20clinical%20effects&author=LR%20Trevi%C3%B1o&volume=27&publication_year=2009&pages=5972-5978&pmid=19901119&doi=10.1200/JCO.2008.20.4156&)

32. Badagnani I, et al. Interaction of methotrexate with organic-anion transporting polypeptide 1A2 and its genetic variants. J Pharmacol Exp Ther. 2006;318:521–529. doi: 10.1124/jpet.106.104364.  [DOI](https://doi.org/10.1124/jpet.106.104364) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16702441/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharmacol%20Exp%20Ther&title=Interaction%20of%20methotrexate%20with%20organic-anion%20transporting%20polypeptide%201A2%20and%20its%20genetic%20variants&author=I%20Badagnani&volume=318&publication_year=2006&pages=521-529&pmid=16702441&doi=10.1124/jpet.106.104364&)

33. Lee W, et al. Polymorphisms in human organic anion-transporting polypeptide 1A2 (OATP1A2): implications for altered drug disposition and central nervous system drug entry. J Biol Chem. 2005;280:9610–9617. doi: 10.1074/jbc.M411092200.  [DOI](https://doi.org/10.1074/jbc.M411092200) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15632119/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=Polymorphisms%20in%20human%20organic%20anion-transporting%20polypeptide%201A2%20(OATP1A2):%20implications%20for%20altered%20drug%20disposition%20and%20central%20nervous%20system%20drug%20entry&author=W%20Lee&volume=280&publication_year=2005&pages=9610-9617&pmid=15632119&doi=10.1074/jbc.M411092200&)

34. Rasmussen R. Quantification on the LightCycler. In: Meuer S, Wittwer C, Nakagawara K, editors. Rapid Cycle Real-Time PCR: Methods and Applications. Springer Press; Heidelberg, Germany: 2001. pp. 21–34.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Rapid%20Cycle%20Real-Time%20PCR:%20Methods%20and%20Applications&author=R%20Rasmussen&publication_year=2001&)

35. Mita S, et al. Inhibition of bile acid transport across Na+/taurocholate cotransporting polypeptide (SLC10A1) and bile salt export pump (ABCB 11)-coexpressing LLC-PK1 cells by cholestasis-inducing drugs. Drug Metab Dispos. 2006;34:1575–1581. doi: 10.1124/dmd.105.008748.  [DOI](https://doi.org/10.1124/dmd.105.008748) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16760228/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Inhibition%20of%20bile%20acid%20transport%20across%20Na+/taurocholate%20cotransporting%20polypeptide%20(SLC10A1)%20and%20bile%20salt%20export%20pump%20(ABCB%2011)-coexpressing%20LLC-PK1%20cells%20by%20cholestasis-inducing%20drugs&author=S%20Mita&volume=34&publication_year=2006&pages=1575-1581&pmid=16760228&doi=10.1124/dmd.105.008748&)

36. Ho RH, et al. Effect of drug transporter genotypes on pravastatin disposition in European- and African-American participants. Pharmacogenet Genomics. 2007;17:647–656. doi: 10.1097/FPC.0b013e3280ef698f.  [DOI](https://doi.org/10.1097/FPC.0b013e3280ef698f) | [PMC free article](/articles/PMC4063287/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17622941/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Effect%20of%20drug%20transporter%20genotypes%20on%20pravastatin%20disposition%20in%20European-%20and%20African-American%20participants&author=RH%20Ho&volume=17&publication_year=2007&pages=647-656&pmid=17622941&doi=10.1097/FPC.0b013e3280ef698f&)

37. Arimori K, et al. Effect of P-glycoprotein modulator, cyclosporin A, on the gastrointestinal excretion of irinotecan and its metabolite SN-38 in rats. Pharm Res. 2003;20:910–917. doi: 10.1023/a:1023847521767.  [DOI](https://doi.org/10.1023/a:1023847521767) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12817897/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharm%20Res&title=Effect%20of%20P-glycoprotein%20modulator,%20cyclosporin%20A,%20on%20the%20gastrointestinal%20excretion%20of%20irinotecan%20and%20its%20metabolite%20SN-38%20in%20rats&author=K%20Arimori&volume=20&publication_year=2003&pages=910-917&pmid=12817897&doi=10.1023/a:1023847521767&)

38. Hagenbuch B, Gui C. Xenobiotic transporters of the human organic anion transporting polypeptides (OATP) family. Xenobiotica. 2008;38:778–801. doi: 10.1080/00498250801986951.  [DOI](https://doi.org/10.1080/00498250801986951) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18668430/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Xenobiotica&title=Xenobiotic%20transporters%20of%20the%20human%20organic%20anion%20transporting%20polypeptides%20(OATP)%20family&author=B%20Hagenbuch&author=C%20Gui&volume=38&publication_year=2008&pages=778-801&pmid=18668430&doi=10.1080/00498250801986951&)

39. Ho RH, et al. Drug and bile acid transporters in rosuvastatin hepatic uptake: function, expression, and pharmacogenetics. Gastroenterology. 2006;130:1793–1806. doi: 10.1053/j.gastro.2006.02.034.  [DOI](https://doi.org/10.1053/j.gastro.2006.02.034) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16697742/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gastroenterology&title=Drug%20and%20bile%20acid%20transporters%20in%20rosuvastatin%20hepatic%20uptake:%20function,%20expression,%20and%20pharmacogenetics&author=RH%20Ho&volume=130&publication_year=2006&pages=1793-1806&pmid=16697742&doi=10.1053/j.gastro.2006.02.034&)

40. Jonker JW, et al. Reduced hepatic uptake and intestinal excretion of organic cations in mice with a targeted disruption of the organic cation transporter 1 (Oct1 [Slc22a1]) gene. Mol Cell Biol. 2001;21:5471–5477. doi: 10.1128/MCB.21.16.5471-5477.2001.  [DOI](https://doi.org/10.1128/MCB.21.16.5471-5477.2001) | [PMC free article](/articles/PMC87269/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11463829/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Cell%20Biol&title=Reduced%20hepatic%20uptake%20and%20intestinal%20excretion%20of%20organic%20cations%20in%20mice%20with%20a%20targeted%20disruption%20of%20the%20organic%20cation%20transporter%201%20(Oct1%20%5BSlc22a1%5D)%20gene&author=JW%20Jonker&volume=21&publication_year=2001&pages=5471-5477&pmid=11463829&doi=10.1128/MCB.21.16.5471-5477.2001&)

41. Zhang S, et al. Organic cation transporters are determinants of oxaliplatin cytotoxicity. Cancer Res. 2006;66:8847–8857. doi: 10.1158/0008-5472.CAN-06-0769.  [DOI](https://doi.org/10.1158/0008-5472.CAN-06-0769) | [PMC free article](/articles/PMC2775093/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16951202/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=Organic%20cation%20transporters%20are%20determinants%20of%20oxaliplatin%20cytotoxicity&author=S%20Zhang&volume=66&publication_year=2006&pages=8847-8857&pmid=16951202&doi=10.1158/0008-5472.CAN-06-0769&)

42. Kobayashi Y, Sakai R, Ohshiro N, Ohbayashi M, Kohyama N, Yamamoto T. Possible involvement of organic anion transporter 2 on the interaction of theophylline with erythromycin in the human liver. Drug Metab Dispos. 2005;33:619–622. doi: 10.1124/dmd.104.003301.  [DOI](https://doi.org/10.1124/dmd.104.003301) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15708966/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Possible%20involvement%20of%20organic%20anion%20transporter%202%20on%20the%20interaction%20of%20theophylline%20with%20erythromycin%20in%20the%20human%20liver&author=Y%20Kobayashi&author=R%20Sakai&author=N%20Ohshiro&author=M%20Ohbayashi&author=N%20Kohyama&volume=33&publication_year=2005&pages=619-622&pmid=15708966&doi=10.1124/dmd.104.003301&)

43. Cropp CD, et al. Organic anion transporter 2 (SLC22A7) is a facilitative transporter of cGMP. Mol Pharmacol. 2008;73:1151–1158. doi: 10.1124/mol.107.043117.  [DOI](https://doi.org/10.1124/mol.107.043117) | [PMC free article](/articles/PMC2698938/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18216183/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Pharmacol&title=Organic%20anion%20transporter%202%20(SLC22A7)%20is%20a%20facilitative%20transporter%20of%20cGMP&author=CD%20Cropp&volume=73&publication_year=2008&pages=1151-1158&pmid=18216183&doi=10.1124/mol.107.043117&)

44. Chen Y, et al. Genetic variants in multidrug and toxic compound extrusion-1, hMATE1, alter transport function. Pharmacogenomics J. 2009;9:127–136. doi: 10.1038/tpj.2008.19.  [DOI](https://doi.org/10.1038/tpj.2008.19) | [PMC free article](/articles/PMC2949062/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19172157/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=Genetic%20variants%20in%20multidrug%20and%20toxic%20compound%20extrusion-1,%20hMATE1,%20alter%20transport%20function&author=Y%20Chen&volume=9&publication_year=2009&pages=127-136&pmid=19172157&doi=10.1038/tpj.2008.19&)

45. Tanihara Y, Masuda S, Sato T, Katsura T, Ogawa O, Inui K. Substrate specificity of MATE1 and MATE2-K, human multidrug and toxin extrusions/ H(+)-organic cation antiporters. Biochem Pharmacol. 2007;74:359–371. doi: 10.1016/j.bcp.2007.04.010.  [DOI](https://doi.org/10.1016/j.bcp.2007.04.010) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17509534/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem%20Pharmacol&title=Substrate%20specificity%20of%20MATE1%20and%20MATE2-K,%20human%20multidrug%20and%20toxin%20extrusions/%20H(+)-organic%20cation%20antiporters&author=Y%20Tanihara&author=S%20Masuda&author=T%20Sato&author=T%20Katsura&author=O%20Ogawa&volume=74&publication_year=2007&pages=359-371&pmid=17509534&doi=10.1016/j.bcp.2007.04.010&)

46. Treiber A, Schneiter R, Häusler S, Stieger B. Bosentan is a substrate of human OATP1B1 and OATP1B3: inhibition of hepatic uptake as the common mechanism of its interactions with cyclosporin A, rifampicin, and sildenafil. Drug Metab Dispos. 2007;35:1400–1407. doi: 10.1124/dmd.106.013615.  [DOI](https://doi.org/10.1124/dmd.106.013615) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17496208/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Bosentan%20is%20a%20substrate%20of%20human%20OATP1B1%20and%20OATP1B3:%20inhibition%20of%20hepatic%20uptake%20as%20the%20common%20mechanism%20of%20its%20interactions%20with%20cyclosporin%20A,%20rifampicin,%20and%20sildenafil&author=A%20Treiber&author=R%20Schneiter&author=S%20H%C3%A4usler&author=B%20Stieger&volume=35&publication_year=2007&pages=1400-1407&pmid=17496208&doi=10.1124/dmd.106.013615&)

47. Gui C, et al. Effect of pregnane X receptor ligands on transport mediated by human OATP1B1 and OATP1B3. Eur J Pharmacol. 2008;584:57–65. doi: 10.1016/j.ejphar.2008.01.042.  [DOI](https://doi.org/10.1016/j.ejphar.2008.01.042) | [PMC free article](/articles/PMC2376123/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18321482/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Pharmacol&title=Effect%20of%20pregnane%20X%20receptor%20ligands%20on%20transport%20mediated%20by%20human%20OATP1B1%20and%20OATP1B3&author=C%20Gui&volume=584&publication_year=2008&pages=57-65&pmid=18321482&doi=10.1016/j.ejphar.2008.01.042&)
