# A genome-wide association study of overall survival in pancreatic cancer patients treated with gemcitabine in CALGB 80303

## Metadata
**Authors:** Federico Innocenti, Kouros Owzar, Nancy L Cox, Patrick Evans, Michiaki Kubo, Hitoshi Zembutsu, Chen Jiang, Donna Hollis, Taisei Mushiroda, Liang Li, Paula Friedman, Liewei Wang, Dylan Glubb, Herbert Hurwitz, Kathleen M Giacomini, Howard L McLeod, Richard M Goldberg, Richard L Schilsky, Hedy L Kindler, Yusuke Nakamura, Mark J Ratain
**Journal:** Clinical cancer research : an official journal of the American Association for Cancer Research
**Date:** 2011 Dec 5
**DOI:** [10.1158/1078-0432.CCR-11-1387](https://doi.org/10.1158/1078-0432.CCR-11-1387)
**PMID:** 22142827
**PMCID:** PMC3412624
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3412624/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC3412624/pdf/nihms341305.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC3412624/pdf/nihms341305.pdf)

## Abstract

**Background and Aims:** 
CALGB 80303 was a randomized, phase III study in advanced pancreatic cancer patients treated with gemcitabine plus either bevacizumab or placebo. We prospectively collected germline DNA and conducted a genome-wide association study (GWAS) using overall survival (OS) as the endpoint.

**Methods:** 
DNA from 351 patients was genotyped for >550,000 single nucleotide polymorphisms (SNPs). Associations between OS and SNPs were investigated using the log-linear two-way multiplicative Cox proportional-hazards model. The subset of 294 genetically European patients was used for the primary analysis.

**Results:** 
A nonsynonymous SNP in IL17F (rs763780, H161R) and an intronic SNP in strong linkage disequilibrium (rs7771466) were associated with OS using genome-wide criteria (p≤10−7). Median OS was significantly shorter (p 2.61×10−8) for the rs763780 heterozygotes (3.1 months, 95% CI 2.3–4.3) as compared to the patients without this variant (6.8 months, 5.8–7.3). After adjustment by stratification factors, the p value for the association was 9.51×10−7.

**Conclusions:** 
The variant 161R form of interleukin-17F is a natural antagonist of the anti-angiogenic effects of wild-type 161H interleukin-17F, and angiogenesis may play an important role in the metastatic spread of pancreatic cancer. In this preliminary study, we hypothesize that the angiogenesis potential of pancreatic cancers in patients with variant interleukin-17F is higher than that of tumors in patients with wild-type interleukin-17F, conferring worse prognosis. This exploratory GWAS may provide the foundation for testing the biology and clinical effects of novel genes and their heritable variants through mechanistic and confirmatory studies in pancreatic cancer.

Keywords: GWAS, pharmacogenetics, bevacizumab, pancreatic cancer, gemcitabine

### Background and Aims

CALGB 80303 was a randomized, phase III study in advanced pancreatic cancer patients treated with gemcitabine plus either bevacizumab or placebo. We prospectively collected germline DNA and conducted a genome-wide association study (GWAS) using overall survival (OS) as the endpoint.

### Methods

DNA from 351 patients was genotyped for >550,000 single nucleotide polymorphisms (SNPs). Associations between OS and SNPs were investigated using the log-linear two-way multiplicative Cox proportional-hazards model. The subset of 294 genetically European patients was used for the primary analysis.

### Results

A nonsynonymous SNP in *IL17F* (rs763780, H161R) and an intronic SNP in strong linkage disequilibrium (rs7771466) were associated with OS using genome-wide criteria (p≤10^−7^). Median OS was significantly shorter (p 2.61×10^−8^) for the rs763780 heterozygotes (3.1 months, 95% CI 2.3–4.3) as compared to the patients without this variant (6.8 months, 5.8–7.3). After adjustment by stratification factors, the p value for the association was 9.51×10^−7^.

### Conclusions

The variant 161R form of interleukin-17F is a natural antagonist of the anti-angiogenic effects of wild-type 161H interleukin-17F, and angiogenesis may play an important role in the metastatic spread of pancreatic cancer. In this preliminary study, we hypothesize that the angiogenesis potential of pancreatic cancers in patients with variant interleukin-17F is higher than that of tumors in patients with wild-type interleukin-17F, conferring worse prognosis. This exploratory GWAS may provide the foundation for testing the biology and clinical effects of novel genes and their heritable variants through mechanistic and confirmatory studies in pancreatic cancer.

## INTRODUCTION

Pancreatic cancer has a very poor prognosis and the lowest survival by stage of any solid tumor[[1](#R1)]. Gemcitabine is the cornerstone of chemotherapy in this disease but has a very modest impact[[2](#R2)], and although numerous clinical trials have been conducted, only the combination of gemcitabine and erlotinib achieved a modest increase in median OS over gemcitabine alone[[3](#R3)]. Novel agents and/or novel molecular biomarkers are urgently needed. One approach is to identify novel candidate genes putatively involved in the biology of pancreatic cancer.

It is likely that germline variants will be able to predict the outcome of patients with cancer and there is epidemiologic evidence that prognosis has an inherited component[[4](#R4)]. To test this, we conducted a genome-wide association study (GWAS) in Cancer and Leukemia Group B (CALGB) study 80303, a randomized, double-blind, phase III study in 602 advanced pancreatic cancer patients treated with gemcitabine plus either bevacizumab or placebo. There was no superiority in OS of the gemcitabine-bevacizumab arm compared to the gemcitabine-placebo arm[[5](#R5)].

As part of the study, we prospectively collected germline DNA for pharmacogenetic studies, originally focusing on the association of candidate genes with treatment outcome. We subsequently amended the study to conduct a genome-wide association study (GWAS) in order to identify novel associations. In GWAS, germline DNA of patients can be scanned using high-density single-nucleotide polymorphism (SNP) chips that assess hundreds of thousands of SNP markers[[6](#R6)]. This approach is unbiased and does not rely on a priori knowledge about the role of candidate markers for the outcome of interest. The goal of this study was to identify novel genes associated with OS in pancreatic cancer.

## PATIENTS AND METHODS

### Clinical trial and patients

CALGB 80303 was a double-blind, placebo-controlled randomized (1:1) phase III multi-institution study of bevacizumab, in combination with gemcitabine. Eligible patients had histologically or cytologically confirmed adenocarcinoma of the pancreas not amenable to potentially curative surgery, as previously described[[7](#R7)]. Patients were required to have an ECOG performance status of 0–2 and adequate bone marrow, renal, and hepatic function. Gemcitabine 1000 mg/m^2^ was given intravenously over 30 min on days 1, 8, and 15 of a 28-day cycle. Bevacizumab, 10 mg/kg, or placebo was administered intravenously after gemcitabine on days 1 and 15 of each 28-day cycle. Treatment was discontinued for progressive disease, unacceptable adverse events, or patient withdrawal of consent.

Patients were stratified according to extent of disease (locally advanced vs. metastatic), ECOG performance status (0–1, vs. 2) and prior radiotherapy (yes/no). Patients received a minimum of two cycles of treatment unless unacceptable toxicity or early progression of disease occurred. Patients were evaluated for response according to the Response Evaluation Criteria in Solid Tumors[[8](#R8)] every 2 cycles. Confirmatory scans were obtained at least 4 weeks following initial documentation of objective complete or partial response. OS, with date of randomization as its reference point, was the primary study endpoint.

The companion pharmacogenetic protocol (CALGB 60401) was approved by the Institutional Review Boards of the University of Chicago and the Riken Institute. Only patients who consented to CALGB 60401 were included in this study. The patient and tumor characteristics of the subgroup of patients genotyped in this study are comparable to those of patients in the main clinical trial ([Table 1](#T1)).

### Table 1.

|   |   | All Patients | GWAS Patients | GWAS Patients ofEuropean Ancestry | GWAS Patients ofAfrican Ancestry |
| --- | --- | --- | --- | --- | --- |
| Sample Size |   | 602 | 338 | 294 | 26 |
| Sex | Male | 329 | 185 | 159 | 15 |
| Female | 273 | 153 | 135 | 11 |  |
| Age | Mean (SD) | 64.0 (10.6) | 63.6(10.4) | 63.8 (10.5) | 62.0 (8.5) |
| Median(95%CI) | 64.1(63.2–65.1) | 64(62.8–65.2) | 64(63.0–65.5) | 64(60.9–67.1) |  |
| Race | White | 529 (87.9%) | 301 (89.0%) | 289 (98.3%) | 0 |
| Black | 49 (8.1%) | 26 (7.7%) | 0 | 26 (100%) |  |
| Asian | 10 (1.7%) | 2 (0.6%) | 0 | 0 |  |
| Native Hawaiian | 1 (0.2%) | 1 (0.3%) | 0 | 0 |  |
| American Indian | 3 (0.5%) | 0 | 0 | 0 |  |
| Multiple | 1 (0.2%) | 1 (0.3%) | 0 | 0 |  |
| Unknown | 9 (1.5%) | 7 (2.1%) | 5 (1.7%) | 0 |  |
| Extent ofDisease | Metastatic | 520 (86.4%) | 299 (88.5%) | 256 (87.1%) | 25 (96.2%) |
| Locally Advanced | 69 (11.5%) | 39 (11.5%) | 38 (12.9%) | 1 (3.8%) |  |
| PriorRadiotherapy | No | 526 (87.4%) | 302 (89.3%) | 262 (89.1%) | 26 (92.3%) |
| Yes | 65 (10.8%) | 36 (10.7%) | 32 (10.9%) | 2 (7.7%) |  |
| PerformanceStatus | 0 or 1 | 537 (89.2%) | 305 (90.2%) | 265 (90.1%) | 23 (88.5%) |
| 2 | 65 (10.8%) | 33 (9.8%) | 29 (9.9%) | 3 (11.5%) |  |
| Arm | Placebo | 300 (49.8%) | 157 (46.4%) | 140 (47.6%) | 9 (34.6%) |
| Bevacizumab | 302 (50.2%) | 181 (53.6%) | 154 (52.4%) | 17 (65.4%) |  |
| OS time | Median (months) | 5.85 | 5.86 | 5.93 | 5.83 |
| Mean (months) | 7.71 | 7.75 | 7.82 | 8.07 |  |

Table 1 Caption: Patient demographics.

### DNA samples and genotyping platforms

Of the 602 randomized patients on the main clinical study[[5](#R5)], blood samples were collected from 365 patients who consented to the pharmacogenetic analysis, and were shipped to the CALGB Pathology Coordinating Office (PCO) at Ohio State University. DNA was extracted from a single 5–10 ml peripheral whole blood sample collected using EDTA vacutainer tubes (purple tops) prior to beginning the study treatment using a commercially available kit from Qiagen (Germantown, MD). The concentration and quality of DNA were measured by ultraviolet spectrophotometry (Nanodrop, Wilmington, DE). DNA of sufficient yield and quality (i.e. at least 2.5 µg and a minimum concentration of 50 ng/ml) was obtained on 352 of the blood specimens (96%). DNA samples were randomly placed on a 96-deep well plate, each well containing one sample at a concentration of 50 ng/µl and volume of 50 µl (by dilution with dH_2_O if needed).

The Illumina HumanHap550v3 Genotyping BeadChip was used to genotype >550,000 SNPs in these samples. In addition, >7,000 SNPs in 267 candidate (i.e., hypothesized *a priori* to be drug-related) genes were also genotyped in the same chip[[9](#R9)]. Genotyping was conducted at the Center for Genomic Medicine, Riken Institute (Yokohama, Japan).

### Quality control of the genotyping results and the phenotypic data (Figure 1)

### Figure 1. Flow chart of the quality control process in 352 initial patients genotyped with 561,466 SNPs.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d7a6/3412624/a1d7712dfac1/nihms341305f1.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3412624_nihms341305f1.jpg)

IBS, identity by state; MAF, minor relative allelic frequency; MGC, minor genotypic counts.

Cases excluded from analysis were closely related patients. We assessed the relatedness among patients by IBS, and the only individual removed from the IBS was one set of duplicates (n=1) (as shown in [Figure 1](#F1)). Patients who were not treated or went off study before completing 2 cycles of therapy were also excluded (n=13). The remaining 338 patients formed the basis for the association analyses. Among the 561,466 SNPs typed on the platform, 44,108 SNPs were excluded due to call rates less than 95%. Among the remaining 517,538 SNPs, 21,894 SNPs with minor relative allelic frequencies (MAF) less than 0.01 were removed. Finally, among the 495,464 remaining SNPs, 88 SNPs with strong evidence for departure from Hardy-Weinberg equilibrium (HWE, p<10^−8^) were removed. Among the 495,376 SNPs passing the filter, 484,523 were autosomal. Among these, 330,690 SNPs had a minor genotypic counts (MGC) >9 and were used in the association analyses.

Patient registration, data collection, and data analysis were performed by the CALGB Statistical Center. Data quality was ensured by careful review of data by CALGB Statistical Center staff and the study chairperson.

### Population structure analysis

Self-reported ethnicity information was available for each patient. However, this was confirmed by estimating the genetic ancestral origin of patients using the principal components analysis software implemented in Eigenstrat[[10](#R10)]. This was done by combining our case data with the European, Asian, and African population SNP data from HapMap. Genetically-European patients enrolled in CALGB 80303 were then selected by choosing only those individuals that closely clustered with the European HapMap samples when using all SNPs in the HumanHap550K BeadChip. This resulted in 294 patients of genetically-estimated European ancestry, and in 26 patients of genetically-estimated African ancestry ([Supplemental Material, Figure 1](#SD1)). Also as shown in this figure, the CALGB 80303 samples lined up as expected against the reference HapMap samples. A strong concordance between genetically-estimated ancestry and self-reported race was observed.

### Functional studies

The putative functional effects of the 20 most significant SNPs associated with OS were examined using FastSNP ([http://fastsnp.ibms.sinica.edu.tw/](http://fastsnp.ibms.sinica.edu.tw/)), a web-based bioinformatic application[[11](#R11)]. Fast SNP can identify genetic regulatory regions, non-synonymous and nonsense amino acid changes and determines the effects of SNPs on exon splicing enhancer and silencer motifs, and transcription factor binding sites. For the same purpose, we also have used our genome-wide data of gemcitabine cytotoxicity in lymphoblastoid cell lines[[12](#R12)].

### Statistical analysis of the associations

Our primary analysis determined the association between SNPs and OS in both arms combined, in patients of European ancestry only. For the SNP by OS association analyses, the Cox score (log-rank) test was used, and the analyses were powered against the additive genetic model. The robustness of the genetic associations in the unadjusted analysis (i.e., our primary analysis described above) was tested by including covariates in the model, testing within the framework of a multivariable additive log-linear Cox proportional-hazards model[[13](#R13)]. These covariates were: randomization stratification factors (performance status, prior radiotherapy, and extent of disease), treatment arm, and genetic ancestry (based on the three principal components). In the genome-wide feature selection process for the genetically European population, only SNPs with MGC>9 were considered. The most significant SNP, *IL17F* rs763780 for OS in patients of European ancestry was also tested in the patients of African ancestry, in an unadjusted analysis. The coxph function from the R[[13](#R13)] extension package survival[[14](#R14)] was used. The p values were not adjusted for multiple comparisons. For a Q-Q plot, see [Supplemental Material](#SD1) ([Figure 2](#F2)). We have used 1×10^−7^ (0.05/500,000) as the p value cut-off for genome-wide significance.

### Figure 2. Manhattan plot of SNPs associated with OS in patients of European ancestry with both arms combined.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d7a6/3412624/32db3dd8ab40/nihms341305f2.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3412624_nihms341305f2.jpg)

The observed marginal P-values (minus log base 10 scale) are plotted across the chromosomes.

## RESULTS

The OS (median, 95% CI) in the genotyped patients of European ancestry was 6.3 months (5.1–8.0) in the placebo arm and 5.9 months (4.9–7.1) in the bevacizumab arm, comparable to the median OS observed in the overall clinical study[[5](#R5)]. The number of available SNPs for association with OS was 484,523. Here we present the results of the primary analysis of the SNP vs. OS association in patients of European ancestry (n=294), in both arms combined ([Figure 2](#F2)).

Of the 20 SNPs that showed the most significant association with OS, nine were in annotated genes, one was near a gene, and ten were in intergenic regions ([Table 2](#T2)). All SNPs in genes were intronic, with the exception of the coding SNP rs763780 in *IL17F*. The SNP with the highest statistical significance was rs763780 in *IL17F* (p 2.61×10^−8^), with a MAF of 0.04 ([Figure 2](#F2)). Patients who were rs763780 heterozygous had reduced median OS of 3.1 months (2.3–4.3) compared to patients without the variant (no patients were homozygous for the variant), who had a median OS of 6.8 months (5.8–7.3, [Figure 3A](#F3)). This SNP was also in strong linkage disequilibrium (r^2^ 0.955) with another *IL17F* SNP (rs7771466) having the second highest statistical significance (p 1.66×10^−7^) and a similar effect on OS [3.1 months (2.4–4.3) vs. 6.6 months (5.8–7.2)]. A similar trend was observed in the subset of patients of African ancestry ([Supplemental Material, Figure 3](#SD1)). The associations between the SNPs in *IL17F* and OS, after adjusting for the stratification factors, treatment arm, and genetic ancestry within Europeans, do not meet the criterion for genome-wide statistical significance (1×10^−7^) ([Supplemental Material, Table 1](#SD1)).

### Table 2. Twenty most significant SNPs associated with OS in patients of European ancestry in both arms combined.

| SNP | Chromosome | Gene | Feature | MAF | 5’ flankinggene | 3’ flankinggene | RiskAllele/ControlAllele | P(unadjusted) | HR(95%CI) |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| rs763780 | 6 | IL17F | codingnonsynonymous | 0.039 | LOC730141 | SLC25A20P | G/A | 2.61E-08 | 3.3(2.1–5.1) |
| rs7771466 | 6 | IL17F | intronic | 0.037 | LOC730141 | SLC25A20P | A/C | 1.66E-07 | 3.1(2.0–4.9) |
| rs2900174 | 12 | PRB2 | intronic | 0.029 | PRB1 | hCG_1655019 | G/A | 1.03E-06 | 3.3(2.0–5.5) |
| rs11062040 | 12 | DCP1B | intronic | 0.498 | CACNA2D4 | CACNA1C | A/G | 7.57E-06 | 1.4(1.2–1.7) |
| rs11644322 | 16 | WWOX | intronic | 0.223 | LOC645947 | LOC729251 | A/G | 1.31E-05 | 1.6(1.3–2.0) |
| rs10736526 | 11 | UBASH3B | intronic | 0.197 | GLULL3 | CRTAM | A/G | 1.50E-05 | 1.6(1.3–1.9) |
| rs179619 | 16 | NA | NA | 0.199 | LOC729993 | ERCC4 | G/A | 2.07E-05 | 1.5(1.3–1.9) |
| rs306104 | 5 | CAMK4 | intronic | 0.347 | LOC402287 | STARD4 | A/G | 2.31E-05 | 1.5(1.2–1.8) |
| rs3744311 | 17 | NA | NA | 0.36 | LOC727973 | SLC39A11 | G/A | 2.53E-05 | 1.5(1.2–1.7) |
| rs10489997 | 2 | BAZ2B | intronic | 0.405 | LOC728059 | LOC643072 | A/G | 2.60E-05 | 1.5(1.2–1.7) |
| rs1910236 | 3 | NA | NA | 0.471 | C3orf67 | hCG_1813818 | A/G | 2.71E-05 | 1.4(1.2–1.7) |
| rs7149097 | 14 | NA | NA | 0.192 | CDC42BPB | C14orf73 | A/G | 3.84E-05 | 1.6(1.3–1.9) |
| rs10883617 | 10 | BTRC | near-gene-5 | 0.35 | LOC282992 | BTRC | G/A | 3.94E-05 | 1.5(1.2–1.8) |
| rs490332 | 5 | NA | NA | 0.486 | TSPAN17 | LOC729378 | A/G | 4.32E-05 | 1.5(1.2–1.7) |
| rs2550731 | 16 | WWOX | intronic | 0.346 | LOC645947 | LOC729251 | A/C | 4.44E-05 | 1.4(1.2–1.7) |
| rs12361312 | 11 | NA | NA | 0.206 | LOC441609 | MS4A2 | A/G | 4.62E-05 | 1.5(1.2–1.8) |
| rs4785367 | 16 | NA | NA | 0.498 | ZNF423 | TMEM188 | G/A | 5.06E-05 | 1.4(1.2–1.7) |
| rs814951 | 1 | NA | NA | 0.464 | FAM5C | LOC440704 | A/G | 5.38E-05 | 1.4(1.2–1.7) |
| rs7712169 | 5 | NA | NA | 0.024 | CTNND2 | LOC391738 | A/G | 5.64E-05 | 2.9(1.7–5.0) |
| rs1808458 | 2 | NA | NA | 0.036 | INSIG2 | LOC645733 | A/G | 6.16E-05 | 2.4(1.6–3.8) |

Table 2 Caption: SNP annotation information is according to the SCAN database[32]using dbSNP version 129. The base change is according to the Illumina TOP stranding method for determining strand and allele[33]. The rs10883617 SNP islocated <1 Kb 5’ of BTRC. NA, intergenic SNP; HR, hazard ratio; MAF, minor relative allele frequency. The SNP call rate and HWE are shown in the Supplemental Material (Table 3).

### Figure 3. Kaplan-Meier estimate of OS stratified by (A) rs763780 IL17F, (B) rs11644322 WWOX, and (C) rs10883617 BTRC genotypes in patients of European ancestry with both arms combined.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d7a6/3412624/2503b6573650/nihms341305f3.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3412624_nihms341305f3.jpg)

For rs763780, A/A, non carriers of rs763780; A/G, heterozygotes. For rs11644322, G/G, non carriers of rs11644322; G/A, heterozygotes; A/A, homozygotes. For rs10883617, A/A, non carriers of rs10883617; A/G, heterozygotes; G/G, homozygotes. The base change is according to the Illumina TOP stranding method for determining strand and allele[33].

In silico analysis of the putative function of the intronic *IL17F* rs7771466 variant (in very strong LD with the non-synonymous rs763780) indicates that a) rs7771466 introduces an additional CDX1 transcription factor binding site to one present in the wild-type sequence, and b) rs763780 abolishes an exonic splicing silencer and introduces two exonic splicing enhancers ([Supplementary Material, Table 2](#SD1)).

## DISCUSSION

We interrogated >550,000 heritable variants in patients with advanced pancreatic cancer treated with chemotherapy in CALGB 80303. To our knowledge, this is the first GWAS in a cancer patient population in the context of a randomized, placebo-controlled, clinical trial. This preliminary study generates hypotheses on the role of the *IL17F* gene in the biology of advanced pancreatic cancer. If replicated, the *IL17F* SNPs might have prognostic significance.

*IL17F* encodes interleukin-17F, a cytokine with the ability to induce stromal cells to secrete pro-inflammatory cytokines. The most significant SNP in this study is rs763780 in *IL17F*, a base substitution that alters the histidine to arginine at amino acid 161 (H161R). In vitro functional experiments demonstrated that, in contrast to the wild-type 161H interleukin-17F, the 161R variant form lacks the ability to activate the mitogen-activated protein kinase pathway, thereby restricting cytokine and chemokine production[[15](#R15)]. Wild-type 161H interleukin-17F has also demonstrated a strong anti-angiogenesis effect by markedly inhibiting the angiogenesis of human endothelial cells and inducing them to produce interleukin-2, TGF-beta, and monocyte chemoattractant protein-1[[16](#R16)]. A recent study has also shown the anti-angiogenetic and anti-tumor properties of wild-type interleukin-17F in vivo[[17](#R17)]. With respect to these activities, the variant 161R form of interleukin-17F is a natural antagonist of the anti-angiogenic and pro-inflammatory effects of wild-type 161H interleukin-17F. For example, the 161R variant has been associated with protective effects in Asian patients with inflammatory and autoimmune conditions[[18](#R18), [19](#R19)]. The resulting pro-angiogenic effects of the variant 161R of interleukin-17F could be further magnified by the concomitant increased expression driven by the noncoding rs7771466 *IL17F* variant. This variant is almost in complete linkage disequilibrium with the nonsynonymous rs763780, and its minor allele seems to increase interleukin-17F expression through splicing enhancement and introduction of an additional CDX1 transcription factor binding site. The putative synergistic effect of these *IL17F* variants should be verified in experimental models of SNP functionality and angiogenesis.

As angiogenesis has been thought to play an important role in the growth and metastatic spread of pancreatic cancer[[20](#R20)], we hypothesize that the angiogenesis potential of tumors of patients with the variant 161R interleukin-17F is higher than tumors with wild-type 161H interleukin-17F, conferring worse prognosis. However, other mechanisms related to the pro-inflammatory effects of interleukin-17F cannot be excluded.

The *IL17F* rs763780 is the most important candidate SNP discovered by this study, due to 1) the genome-wide significance, 2) its already established molecular function, 3) the mechanistic hypothesis explaining the association with reduced OS, and 4) the suggestion that a trend could be detected in patients of African ancestry, despite the very small sample size. This study proposes that rs763780 in *IL17F* might have a prognostic effect in advanced pancreatic cancer patients, also because stratification by treatment arm does not seem to negatively affect the association ([Supplemental Material, Table 1](#SD1)). Because gemcitabine is given in both arms, a true interaction between gemcitabine and *IL17F* SNPs cannot be tested. Additionally, a review of the clinical characteristics of the patients heterozygous for *IL17F* rs763780 did not show any obvious difference with respect to the characteristics of the overall population accrued into this study (data not shown).

In addition to *IL17F*, this study proposes additional genes as putatively involved in determining differences in survival among patients with advanced pancreatic cancer. Among the SNPs listed in [Table 2](#T2), rs11644322 in *WWOX* demonstrated a gene-dosage effect, with median OS in the heterozygous patients (5.3 months, 4.3–6.9) that was intermediate between the other two genotype groups (3.3 months, 2.9–5.7, for the variant homozygotes; 7.1 months, 6.0–8.4, for the wild-type homozygotes; p 1.31×10^−5^; [Figure 3B](#F3)). *WWOX* codes for the WW domain-containing oxidoreductase, a tumor suppressor in several tumors, including pancreatic cancer[[21](#R21)]. *WWOX* SNPs showed the strongest linkage for prostate cancer susceptibility in a recent genome-wide scan[[22](#R22)]. In multiple myeloma, loss of heterozygosity of 16q23, the location of *WWOX*, was associated with adverse survival and reduced WWOX expression[[23](#R23)]. Germline variants from a recent study mapped *WWOX* as one of the genes associated with clinical staging in lung cancer[[24](#R24)]. Contrary to the amino acid changing SNP in *IL17F*, the molecular functions of the SNPs in *WWOX* are not known at this time. Due to their association with reduced OS, the established tumor suppressor role of *WWOX*, and their intronic location, we hypothesize that these SNPs might reduce the expression of WWOX, diminishing its tumor suppressor properties, and leading to worse prognosis.

A SNP (rs10883617) near *BTRC* was associated with reduced OS, as homozygote patients had a reduced median OS (3.6 months, 2.7–6.8) compared to heterozygous patients (6.1 months, 4.9–7.3) and patients who were not carriers of this variant (6.9 months, 5.9–8.2; p 3.94×10^−5^, [Figure 3C](#F3)). *BTRC* (beta-transducin repeat containing) encodes a protein involved in the ubiquitination processes that demonstrated an oncogenic activity in several cancers including pancreatic cancer[[25](#R25)–[27](#R27)]. In most tumors, overexpression of BTRC results in the degradation of IKappaB, an inhibitor of the NFkappaB transcription factor, and thus the activation of NFkappaB and the uncontrolled cell proliferation in these tumors. The use of our functional results in lymphoblastoid cell lines treated with gemcitabine indicated that *BTRC* rs10883617 is associated with increased IC_50_ and, hence, resistance to gemcitabine (r 0.21, p 0.008). As inhibiting and silencing NFkappaB have been shown to increase the sensitivity of pancreatic cancer cells to gemcitabine[[28](#R28), [29](#R29)], we hypothesize that *BTRC* rs10883617 might have a predictive value in pancreatic cancer patients treated with gemcitabine, via a NFkappaB-mediated effect.

This study is limited by the large number of multiple comparisons typical of GWAS, increasing the chance of false positive associations. This limitation could be overcome by independent replication of the findings. Replication studies in cancer treatment outcome have intrinsic difficulties, as there may not be an existing trial (to be used for replication) with the same eligibility criteria and drug treatment of the trial used for discovery. In a relatively uncommon disease like pancreatic cancer, the access to a sufficiently powered replication set is particularly challenging. Ideally, validation of our top hits should be conducted in patients treated with gemcitabine and randomized to an experimental treatment, in order to ensure that the populations are comparable. A few published trials[[30](#R30), [31](#R31)] where patient DNA has been already collected may be considered for replication, due to similarity of treatment and/or disease, and randomized treatment. Additionally, the MAF of the variants in *IL17F* is low in Caucasians (0.05 from HapMap), potentially limiting the ability to replicate this association in this population. However, the MAF of the variants in *IL17F* is higher in Asians (0.13).

The results of this study are preliminary because of the limited sample size and the low MAF of the *IL17F* SNP. Due to the refractoriness to treatment of advanced pancreatic cancer and the lack of established markers of survival, the dissemination of these findings to the scientific community could facilitate their replication by others, even as we continue to conduct replication and validation studies. The association of *IL17F* variants with efficacy could be also tested in tumors other than pancreatic cancer. To support this, our data are now available in dbGaP, in accordance with NIH policy.

## Supplementary Material

## ACKNOWLEDGEMENT

The authors would like to acknowledge the helpful suggestions made by Dr. Monica Bertagnolli, Harvard Medical School. This paper was prepared for submission by Michelle Gibeault, University of North Carolina - Chapel Hill.

**Grant Support:**

The research for CALGB 80303 was supported, in part, by grants from the Cancer and Leukemia Group B and the CALGB Statistical Center (CA33601).This study was also supported by K07-CA140390-01, T32 MH200065-06, 5K24-CA113755, CA60138, the U01 GM61393, and the BioBank Japan Project that is funded by the Ministry of Education, Culture, Sports, Science and Technology of the Japanese government. This study was supported by the NIH Pharmacogenomics Research Network (PGRN) – RIKEN Center for Genomic Medicine (CGM) Strategic Alliance.

## Abbreviations

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Siegel RL, Jemal A, Ward EM. Increase in incidence of colorectal cancer among young men and women in the United States. Cancer Epidemiol Biomarkers Prev. 2009;18:1695–1698. doi: 10.1158/1055-9965.EPI-09-0186.  [DOI](https://doi.org/10.1158/1055-9965.EPI-09-0186) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19505901/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Epidemiol%20Biomarkers%20Prev&title=Increase%20in%20incidence%20of%20colorectal%20cancer%20among%20young%20men%20and%20women%20in%20the%20United%20States&author=RL%20Siegel&author=A%20Jemal&author=EM%20Ward&volume=18&publication_year=2009&pages=1695-1698&pmid=19505901&doi=10.1158/1055-9965.EPI-09-0186&)

2. Burris HA, 3rd, Moore MJ, Andersen J, Green MR, Rothenberg ML, Modiano MR, et al. Improvements in survival and clinical benefit with gemcitabine as first-line therapy for patients with advanced pancreas cancer: a randomized trial. J Clin Oncol. 1997;15:2403–2413. doi: 10.1200/JCO.1997.15.6.2403.  [DOI](https://doi.org/10.1200/JCO.1997.15.6.2403) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9196156/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Improvements%20in%20survival%20and%20clinical%20benefit%20with%20gemcitabine%20as%20first-line%20therapy%20for%20patients%20with%20advanced%20pancreas%20cancer:%20a%20randomized%20trial&author=HA%20Burris&author=MJ%20Moore&author=J%20Andersen&author=MR%20Green&author=ML%20Rothenberg&volume=15&publication_year=1997&pages=2403-2413&pmid=9196156&doi=10.1200/JCO.1997.15.6.2403&)

3. Pliarchopoulou K, Pectasides D. Pancreatic cancer: current and future treatment strategies. Cancer Treat Rev. 2009;35:431–436. doi: 10.1016/j.ctrv.2009.02.005.  [DOI](https://doi.org/10.1016/j.ctrv.2009.02.005) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19328630/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Treat%20Rev&title=Pancreatic%20cancer:%20current%20and%20future%20treatment%20strategies&author=K%20Pliarchopoulou&author=D%20Pectasides&volume=35&publication_year=2009&pages=431-436&pmid=19328630&doi=10.1016/j.ctrv.2009.02.005&)

4. Hartman M, Loy EY, Ku CS, Chia KS. Molecular epidemiology and its current clinical use in cancer management. Lancet Oncol. 11:383–390. doi: 10.1016/S1470-2045(10)70005-X.  [DOI](https://doi.org/10.1016/S1470-2045(10)70005-X) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20359664/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet%20Oncol&title=Molecular%20epidemiology%20and%20its%20current%20clinical%20use%20in%20cancer%20management&author=M%20Hartman&author=EY%20Loy&author=CS%20Ku&author=KS%20Chia&volume=11&pages=383-390&pmid=20359664&doi=10.1016/S1470-2045(10)70005-X&)

5. Kindler HL, Niedzwiecki D, Hollis D, Sutherland S, Schrag D, Hurwitz H, et al. Gemcitabine plus bevacizumab compared with gemcitabine plus placebo in patients with advanced pancreatic cancer: phase III trial of the Cancer and Leukemia Group B (CALGB 80303) J Clin Oncol. 28:3617–3622. doi: 10.1200/JCO.2010.28.1386.  [DOI](https://doi.org/10.1200/JCO.2010.28.1386) | [PMC free article](/articles/PMC2917317/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20606091/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Gemcitabine%20plus%20bevacizumab%20compared%20with%20gemcitabine%20plus%20placebo%20in%20patients%20with%20advanced%20pancreatic%20cancer:%20phase%20III%20trial%20of%20the%20Cancer%20and%20Leukemia%20Group%20B%20(CALGB%2080303)&author=HL%20Kindler&author=D%20Niedzwiecki&author=D%20Hollis&author=S%20Sutherland&author=D%20Schrag&volume=28&pages=3617-3622&pmid=20606091&doi=10.1200/JCO.2010.28.1386&)

6. Innocenti F, Cox NJ, Dolan ME. The use of genomic information to optimize cancer chemotherapy. Semin Oncol. 38:186–195. doi: 10.1053/j.seminoncol.2011.01.005.  [DOI](https://doi.org/10.1053/j.seminoncol.2011.01.005) | [PMC free article](/articles/PMC3076508/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21421109/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Semin%20Oncol&title=The%20use%20of%20genomic%20information%20to%20optimize%20cancer%20chemotherapy&author=F%20Innocenti&author=NJ%20Cox&author=ME%20Dolan&volume=38&pages=186-195&pmid=21421109&doi=10.1053/j.seminoncol.2011.01.005&)

7. Kindler HL, Niedzwiecki D, Hollis D, Sutherland S, Schrag D, Hurwitz H, et al. Gemcitabine plus bevacizumab compared with gemcitabine plus placebo in patients with advanced pancreatic cancer: phase III trial of the Cancer and Leukemia Group B (CALGB 80303) J Clin Oncol. 2010;28:3617–3622. doi: 10.1200/JCO.2010.28.1386.  [DOI](https://doi.org/10.1200/JCO.2010.28.1386) | [PMC free article](/articles/PMC2917317/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20606091/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Gemcitabine%20plus%20bevacizumab%20compared%20with%20gemcitabine%20plus%20placebo%20in%20patients%20with%20advanced%20pancreatic%20cancer:%20phase%20III%20trial%20of%20the%20Cancer%20and%20Leukemia%20Group%20B%20(CALGB%2080303)&author=HL%20Kindler&author=D%20Niedzwiecki&author=D%20Hollis&author=S%20Sutherland&author=D%20Schrag&volume=28&publication_year=2010&pages=3617-3622&pmid=20606091&doi=10.1200/JCO.2010.28.1386&)

8. Therasse P, Arbuck SG, Eisenhauer EA, Wanders J, Kaplan RS, Rubinstein L, et al. New guidelines to evaluate the response to treatment in solid tumors. European Organization for Research and Treatment of Cancer, National Cancer Institute of the United States, National Cancer Institute of Canada. J Natl Cancer Inst. 2000;92:205–216. doi: 10.1093/jnci/92.3.205.  [DOI](https://doi.org/10.1093/jnci/92.3.205) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10655437/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Natl%20Cancer%20Inst&title=New%20guidelines%20to%20evaluate%20the%20response%20to%20treatment%20in%20solid%20tumors.%20European%20Organization%20for%20Research%20and%20Treatment%20of%20Cancer,%20National%20Cancer%20Institute%20of%20the%20United%20States,%20National%20Cancer%20Institute%20of%20Canada&author=P%20Therasse&author=SG%20Arbuck&author=EA%20Eisenhauer&author=J%20Wanders&author=RS%20Kaplan&volume=92&publication_year=2000&pages=205-216&pmid=10655437&doi=10.1093/jnci/92.3.205&)

9. Iida A, Saito S, Sekine A, Takahashi A, Kamatani N, Nakamura Y. Japanese single nucleotide polymorphism database for 267 possible drug-related genes. Cancer Sci. 2006;97:16–24. doi: 10.1111/j.1349-7006.2006.00142.x.  [DOI](https://doi.org/10.1111/j.1349-7006.2006.00142.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16367916/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Sci&title=Japanese%20single%20nucleotide%20polymorphism%20database%20for%20267%20possible%20drug-related%20genes&author=A%20Iida&author=S%20Saito&author=A%20Sekine&author=A%20Takahashi&author=N%20Kamatani&volume=97&publication_year=2006&pages=16-24&pmid=16367916&doi=10.1111/j.1349-7006.2006.00142.x&)

10. Price AL, Patterson NJ, Plenge RM, Weinblatt ME, Shadick NA, Reich D. Principal components analysis corrects for stratification in genome-wide association studies. Nat Genet. 2006;38:904–909. doi: 10.1038/ng1847.  [DOI](https://doi.org/10.1038/ng1847) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16862161/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Principal%20components%20analysis%20corrects%20for%20stratification%20in%20genome-wide%20association%20studies&author=AL%20Price&author=NJ%20Patterson&author=RM%20Plenge&author=ME%20Weinblatt&author=NA%20Shadick&volume=38&publication_year=2006&pages=904-909&pmid=16862161&doi=10.1038/ng1847&)

11. Yuan HY, Chiou JJ, Tseng WH, Liu CH, Liu CK, Lin YJ, et al. FASTSNP: an always up-to-date and extendable service for SNP function analysis and prioritization. Nucleic Acids Res. 2006;34:W635–W641. doi: 10.1093/nar/gkl236.  [DOI](https://doi.org/10.1093/nar/gkl236) | [PMC free article](/articles/PMC1538865/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16845089/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=FASTSNP:%20an%20always%20up-to-date%20and%20extendable%20service%20for%20SNP%20function%20analysis%20and%20prioritization&author=HY%20Yuan&author=JJ%20Chiou&author=WH%20Tseng&author=CH%20Liu&author=CK%20Liu&volume=34&publication_year=2006&pages=W635-W641&pmid=16845089&doi=10.1093/nar/gkl236&)

12. Li L, Fridley BL, Kalari K, Jenkins G, Batzler A, Weinshilboum RM, et al. Gemcitabine and arabinosylcytosin pharmacogenomics: genome-wide association and drug response biomarkers. PLoS One. 2009;4:e7765. doi: 10.1371/journal.pone.0007765.  [DOI](https://doi.org/10.1371/journal.pone.0007765) | [PMC free article](/articles/PMC2770319/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19898621/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=Gemcitabine%20and%20arabinosylcytosin%20pharmacogenomics:%20genome-wide%20association%20and%20drug%20response%20biomarkers&author=L%20Li&author=BL%20Fridley&author=K%20Kalari&author=G%20Jenkins&author=A%20Batzler&volume=4&publication_year=2009&pages=e7765&pmid=19898621&doi=10.1371/journal.pone.0007765&)

13. R Development Core Team. A language and environment for statistical computing [Internet] [updated 2006 April 21; cited 2010 July 7]. Available from: http://www.R-project.org.  [http://www.R-project.org](http://www.R-project.org)

14. Therneau TM, Grambsch PM. Modeling survival data: extending the Cox model. New York: Springer; 2000.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Modeling%20survival%20data:%20extending%20the%20Cox%20model&author=TM%20Therneau&author=PM%20Grambsch&publication_year=2000&)

15. Kawaguchi M, Takahashi D, Hizawa N, Suzuki S, Matsukura S, Kokubu F, et al. IL-17F sequence variant (His161Arg) is associated with protection against asthma and antagonizes wild-type IL-17F activity. J Allergy Clin Immunol. 2006;117:795–801. doi: 10.1016/j.jaci.2005.12.1346.  [DOI](https://doi.org/10.1016/j.jaci.2005.12.1346) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16630936/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Allergy%20Clin%20Immunol&title=IL-17F%20sequence%20variant%20(His161Arg)%20is%20associated%20with%20protection%20against%20asthma%20and%20antagonizes%20wild-type%20IL-17F%20activity&author=M%20Kawaguchi&author=D%20Takahashi&author=N%20Hizawa&author=S%20Suzuki&author=S%20Matsukura&volume=117&publication_year=2006&pages=795-801&pmid=16630936&doi=10.1016/j.jaci.2005.12.1346&)

16. Starnes T, Robertson MJ, Sledge G, Kelich S, Nakshatri H, Broxmeyer HE, et al. Cutting edge: IL-17F, a novel cytokine selectively expressed in activated T cells and monocytes, regulates angiogenesis and endothelial cell cytokine production. J Immunol. 2001;167:4137–4140. doi: 10.4049/jimmunol.167.8.4137.  [DOI](https://doi.org/10.4049/jimmunol.167.8.4137) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11591732/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Immunol&title=Cutting%20edge:%20IL-17F,%20a%20novel%20cytokine%20selectively%20expressed%20in%20activated%20T%20cells%20and%20monocytes,%20regulates%20angiogenesis%20and%20endothelial%20cell%20cytokine%20production&author=T%20Starnes&author=MJ%20Robertson&author=G%20Sledge&author=S%20Kelich&author=H%20Nakshatri&volume=167&publication_year=2001&pages=4137-4140&pmid=11591732&doi=10.4049/jimmunol.167.8.4137&)

17. Xie Y, Sheng W, Xiang J, Ye Z, Yang J. Interleukin-17F suppresses hepatocarcinoma cell growth via inhibition of tumor angiogenesis. Cancer Invest. 28:598–607. doi: 10.3109/07357900903287030.  [DOI](https://doi.org/10.3109/07357900903287030) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20210523/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Invest&title=Interleukin-17F%20suppresses%20hepatocarcinoma%20cell%20growth%20via%20inhibition%20of%20tumor%20angiogenesis&author=Y%20Xie&author=W%20Sheng&author=J%20Xiang&author=Z%20Ye&author=J%20Yang&volume=28&pages=598-607&pmid=20210523&doi=10.3109/07357900903287030&)

18. Arisawa T, Tahara T, Shibata T, Nagasaka M, Nakamura M, Kamiya Y, et al. Genetic polymorphisms of molecules associated with inflammation and immune response in Japanese subjects with functional dyspepsia. Int J Mol Med. 2007;20:717–723.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/17912466/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Mol%20Med&title=Genetic%20polymorphisms%20of%20molecules%20associated%20with%20inflammation%20and%20immune%20response%20in%20Japanese%20subjects%20with%20functional%20dyspepsia&author=T%20Arisawa&author=T%20Tahara&author=T%20Shibata&author=M%20Nagasaka&author=M%20Nakamura&volume=20&publication_year=2007&pages=717-723&pmid=17912466&)

19. Arisawa T, Tahara T, Shibata T, Nagasaka M, Nakamura M, Kamiya Y, et al. The influence of polymorphisms of interleukin-17A and interleukin-17F genes on the susceptibility to ulcerative colitis. J Clin Immunol. 2008;28:44–49. doi: 10.1007/s10875-007-9125-8.  [DOI](https://doi.org/10.1007/s10875-007-9125-8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17828618/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Immunol&title=The%20influence%20of%20polymorphisms%20of%20interleukin-17A%20and%20interleukin-17F%20genes%20on%20the%20susceptibility%20to%20ulcerative%20colitis&author=T%20Arisawa&author=T%20Tahara&author=T%20Shibata&author=M%20Nagasaka&author=M%20Nakamura&volume=28&publication_year=2008&pages=44-49&pmid=17828618&doi=10.1007/s10875-007-9125-8&)

20. Korc M. Pathways for aberrant angiogenesis in pancreatic cancer. Mol Cancer. 2003;2:8. doi: 10.1186/1476-4598-2-8.  [DOI](https://doi.org/10.1186/1476-4598-2-8) | [PMC free article](/articles/PMC149422/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12556241/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Cancer&title=Pathways%20for%20aberrant%20angiogenesis%20in%20pancreatic%20cancer&author=M%20Korc&volume=2&publication_year=2003&pages=8&pmid=12556241&doi=10.1186/1476-4598-2-8&)

21. Del Mare S, Salah Z, Aqeilan RI. WWOX: its genomics, partners, and functions. J Cell Biochem. 2009;108:737–745. doi: 10.1002/jcb.22298.  [DOI](https://doi.org/10.1002/jcb.22298) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19708029/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Cell%20Biochem&title=WWOX:%20its%20genomics,%20partners,%20and%20functions&author=S%20Del%20Mare&author=Z%20Salah&author=RI%20Aqeilan&volume=108&publication_year=2009&pages=737-745&pmid=19708029&doi=10.1002/jcb.22298&)

22. Lange EM, Beebe-Dimmer JL, Ray AM, Zuhlke KA, Ellis J, Wang Y, et al. Genome-wide linkage scan for prostate cancer susceptibility from the University of Michigan Prostate Cancer Genetics Project: suggestive evidence for linkage at 16q23. Prostate. 2009;69:385–391. doi: 10.1002/pros.20891.  [DOI](https://doi.org/10.1002/pros.20891) | [PMC free article](/articles/PMC2712837/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19035517/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Prostate&title=Genome-wide%20linkage%20scan%20for%20prostate%20cancer%20susceptibility%20from%20the%20University%20of%20Michigan%20Prostate%20Cancer%20Genetics%20Project:%20suggestive%20evidence%20for%20linkage%20at%2016q23&author=EM%20Lange&author=JL%20Beebe-Dimmer&author=AM%20Ray&author=KA%20Zuhlke&author=J%20Ellis&volume=69&publication_year=2009&pages=385-391&pmid=19035517&doi=10.1002/pros.20891&)

23. Jenner MW, Leone PE, Walker BA, Ross FM, Johnson DC, Gonzalez D, et al. Gene mapping and expression analysis of 16q loss of heterozygosity identifies WWOX and CYLD as being important in determining clinical outcome in multiple myeloma. Blood. 2007;110:3291–3300. doi: 10.1182/blood-2007-02-075069.  [DOI](https://doi.org/10.1182/blood-2007-02-075069) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17609426/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Gene%20mapping%20and%20expression%20analysis%20of%2016q%20loss%20of%20heterozygosity%20identifies%20WWOX%20and%20CYLD%20as%20being%20important%20in%20determining%20clinical%20outcome%20in%20multiple%20myeloma&author=MW%20Jenner&author=PE%20Leone&author=BA%20Walker&author=FM%20Ross&author=DC%20Johnson&volume=110&publication_year=2007&pages=3291-3300&pmid=17609426&doi=10.1182/blood-2007-02-075069&)

24. Frullanti E, Galvan A, Falvella FS, Manenti G, Colombo F, Vannelli A, et al. Multiple genetic Loci modulate lung adenocarcinoma clinical staging. Clin Cancer Res. 17:2410–2416. doi: 10.1158/1078-0432.CCR-10-2394.  [DOI](https://doi.org/10.1158/1078-0432.CCR-10-2394) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21242121/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&title=Multiple%20genetic%20Loci%20modulate%20lung%20adenocarcinoma%20clinical%20staging&author=E%20Frullanti&author=A%20Galvan&author=FS%20Falvella&author=G%20Manenti&author=F%20Colombo&volume=17&pages=2410-2416&pmid=21242121&doi=10.1158/1078-0432.CCR-10-2394&)

25. Gerstein AV, Almeida TA, Zhao G, Chess E, Shih Ie M, Buhler K, et al. APC/CTNNB1 (beta-catenin) pathway alterations in human prostate cancers. Genes Chromosomes Cancer. 2002;34:9–16. doi: 10.1002/gcc.10037.  [DOI](https://doi.org/10.1002/gcc.10037) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11921277/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genes%20Chromosomes%20Cancer&title=APC/CTNNB1%20(beta-catenin)%20pathway%20alterations%20in%20human%20prostate%20cancers&author=AV%20Gerstein&author=TA%20Almeida&author=G%20Zhao&author=E%20Chess&author=M%20Shih%20Ie&volume=34&publication_year=2002&pages=9-16&pmid=11921277&doi=10.1002/gcc.10037&)

26. Wood LD, Parsons DW, Jones S, Lin J, Sjoblom T, Leary RJ, et al. The genomic landscapes of human breast and colorectal cancers. Science. 2007;318:1108–1113. doi: 10.1126/science.1145720.  [DOI](https://doi.org/10.1126/science.1145720) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17932254/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=The%20genomic%20landscapes%20of%20human%20breast%20and%20colorectal%20cancers&author=LD%20Wood&author=DW%20Parsons&author=S%20Jones&author=J%20Lin&author=T%20Sjoblom&volume=318&publication_year=2007&pages=1108-1113&pmid=17932254&doi=10.1126/science.1145720&)

27. Muerkoster S, Arlt A, Sipos B, Witt M, Grossmann M, Kloppel G, et al. Increased expression of the E3-ubiquitin ligase receptor subunit betaTRCP1 relates to constitutive nuclear factor-kappaB activation and chemoresistance in pancreatic carcinoma cells. Cancer Res. 2005;65:1316–1324. doi: 10.1158/0008-5472.CAN-04-1626.  [DOI](https://doi.org/10.1158/0008-5472.CAN-04-1626) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15735017/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=Increased%20expression%20of%20the%20E3-ubiquitin%20ligase%20receptor%20subunit%20betaTRCP1%20relates%20to%20constitutive%20nuclear%20factor-kappaB%20activation%20and%20chemoresistance%20in%20pancreatic%20carcinoma%20cells&author=S%20Muerkoster&author=A%20Arlt&author=B%20Sipos&author=M%20Witt&author=M%20Grossmann&volume=65&publication_year=2005&pages=1316-1324&pmid=15735017&doi=10.1158/0008-5472.CAN-04-1626&)

28. Pan X, Arumugam T, Yamamoto T, Levin PA, Ramachandran V, Ji B, et al. Nuclear factor-kappaB p65/relA silencing induces apoptosis and increases gemcitabine effectiveness in a subset of pancreatic cancer cells. Clin Cancer Res. 2008;14:8143–8151. doi: 10.1158/1078-0432.CCR-08-1539.  [DOI](https://doi.org/10.1158/1078-0432.CCR-08-1539) | [PMC free article](/articles/PMC4403242/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19088029/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&title=Nuclear%20factor-kappaB%20p65/relA%20silencing%20induces%20apoptosis%20and%20increases%20gemcitabine%20effectiveness%20in%20a%20subset%20of%20pancreatic%20cancer%20cells&author=X%20Pan&author=T%20Arumugam&author=T%20Yamamoto&author=PA%20Levin&author=V%20Ramachandran&volume=14&publication_year=2008&pages=8143-8151&pmid=19088029&doi=10.1158/1078-0432.CCR-08-1539&)

29. Guo X, Xu B, Pandey S, Goessl E, Brown J, Armesilla AL, et al. Disulfiram/copper complex inhibiting NFkappaB activity and potentiating cytotoxic effect of gemcitabine on colon and breast cancer cell lines. Cancer Lett. 290:104–113. doi: 10.1016/j.canlet.2009.09.002.  [DOI](https://doi.org/10.1016/j.canlet.2009.09.002) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19782464/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Lett&title=Disulfiram/copper%20complex%20inhibiting%20NFkappaB%20activity%20and%20potentiating%20cytotoxic%20effect%20of%20gemcitabine%20on%20colon%20and%20breast%20cancer%20cell%20lines&author=X%20Guo&author=B%20Xu&author=S%20Pandey&author=E%20Goessl&author=J%20Brown&volume=290&pages=104-113&pmid=19782464&doi=10.1016/j.canlet.2009.09.002&)

30. Van Cutsem E, Vervenne WL, Bennouna J, Humblet Y, Gill S, Van Laethem JL, et al. Phase III trial of bevacizumab in combination with gemcitabine and erlotinib in patients with metastatic pancreatic cancer. J Clin Oncol. 2009;27:2231–2237. doi: 10.1200/JCO.2008.20.0238.  [DOI](https://doi.org/10.1200/JCO.2008.20.0238) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19307500/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Phase%20III%20trial%20of%20bevacizumab%20in%20combination%20with%20gemcitabine%20and%20erlotinib%20in%20patients%20with%20metastatic%20pancreatic%20cancer&author=E%20Van%20Cutsem&author=WL%20Vervenne&author=J%20Bennouna&author=Y%20Humblet&author=S%20Gill&volume=27&publication_year=2009&pages=2231-2237&pmid=19307500&doi=10.1200/JCO.2008.20.0238&)

31. Schneider BP, Wang M, Radovich M, Sledge GW, Badve S, Thor A, et al. Association of vascular endothelial growth factor and vascular endothelial growth factor receptor-2 genetic polymorphisms with outcome in a trial of paclitaxel compared with paclitaxel plus bevacizumab in advanced breast cancer: ECOG 2100. J Clin Oncol. 2008;26:4672–4678. doi: 10.1200/JCO.2008.16.1612.  [DOI](https://doi.org/10.1200/JCO.2008.16.1612) | [PMC free article](/articles/PMC2653128/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18824714/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Association%20of%20vascular%20endothelial%20growth%20factor%20and%20vascular%20endothelial%20growth%20factor%20receptor-2%20genetic%20polymorphisms%20with%20outcome%20in%20a%20trial%20of%20paclitaxel%20compared%20with%20paclitaxel%20plus%20bevacizumab%20in%20advanced%20breast%20cancer:%20ECOG%202100&author=BP%20Schneider&author=M%20Wang&author=M%20Radovich&author=GW%20Sledge&author=S%20Badve&volume=26&publication_year=2008&pages=4672-4678&pmid=18824714&doi=10.1200/JCO.2008.16.1612&)

32. Gamazon ER, Zhang W, Konkashbaev A, Duan S, Kistner EO, Nicolae DL, et al. SCAN: SNP and copy number annotation. Bioinformatics. 26:259–262. doi: 10.1093/bioinformatics/btp644.  [DOI](https://doi.org/10.1093/bioinformatics/btp644) | [PMC free article](/articles/PMC2852202/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19933162/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=SCAN:%20SNP%20and%20copy%20number%20annotation&author=ER%20Gamazon&author=W%20Zhang&author=A%20Konkashbaev&author=S%20Duan&author=EO%20Kistner&volume=26&pages=259-262&pmid=19933162&doi=10.1093/bioinformatics/btp644&)

33. Illumina. "TOP/BOT" strand and "A/B" allele: A guide to Illumina's method for determining Strand and Allele for the GoldenGate and Infinium Assays [Internet] [updated 2006 June 27; cited 2010 December 23]. Available from: http://www.illumina.com/documents/products/technotes/technote_topbot.pdf.  [http://www.illumina.com/documents/products/technotes/technote_topbot.pdf](http://www.illumina.com/documents/products/technotes/technote_topbot.pdf)
