# A Non-Synonymous Single Nucleotide Polymorphism in an OPRM1 Splice Variant Is Associated with Fentanyl-Induced Emesis in Women Undergoing Minor Gynaecological Surgery

## Metadata
**Authors:** Grace Su Yin Pang, Farida Ithnin, Yin Yee Wong, Jing Bo Wang, Yvonne Lim, Alex Tiong Heng Sia, Caroline Guat Lay Lee
**Journal:** PLoS ONE
**Date:** 2012 Nov 7
**DOI:** [10.1371/journal.pone.0048416](https://doi.org/10.1371/journal.pone.0048416)
**PMID:** 23144877
**PMCID:** PMC3492352
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3492352/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC3492352/pdf/pone.0048416.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC3492352/pdf/pone.0048416.pdf)

## Abstract

**Background:** 
Fentanyl-induced emesis (FIE) is a distressing adverse effect in the postoperative setting. The genetic basis of FIE remains largely unknown, therefore, we examined whether it was associated with specific genetic variants of OPRM1, the gene encoding the main receptor target of fentanyl.

**Methods:** 
In this prospective case-control study, 193 women undergoing gynaecological surgery under a standardized anaesthetic with a low emetogenic risk were enrolled. Inclusion and exclusion criteria were designed to select extreme phenotypes as well as to ensure that most major confounders for FIE were either excluded or present in all patients. To control for unforeseen intra- and postoperative confounders for FIE, only 161 patients were further analysed, out of which 10 were categorized as having FIE, defined by the presence of at least one of three symptoms: nausea, vomiting or retching that was likely to be fentanyl-related. To identify SNPs relevant to FIE in our population, DNA from 40 controls and 10 cases was sequenced at the following OPRM1 regions: 3 kbp of the promoter, main and alternative exons as well as 2 kbp of the 3′ downstream region. The genotype of the significant SNP was further determined in the remaining 111 controls.

**Results:** 
The incidence of FIE was 6.2%. Initial sequencing of 10 cases and 40 controls identified 25 SNPs. Only rs540825, a non-synonymous SNP in the splice variant, MOR1X, showed a significant association with FIE post-Bonferroni correction. This SNP was further examined in the remaining 111 controls which confirmed its significant association with FIE (p = 0.019 post-Bonferroni, OR: 5.6, 95% CI: 1.42–21.91).

**Conclusions:** 
This is the first report of an association between the occurrence of FIE in Chinese women undergoing gynaecological surgery and an OPRM1 splice variant SNP, rs540825.

### Background

Fentanyl-induced emesis (FIE) is a distressing adverse effect in the postoperative setting. The genetic basis of FIE remains largely unknown, therefore, we examined whether it was associated with specific genetic variants of *OPRM1*, the gene encoding the main receptor target of fentanyl.

### Methods

In this prospective case-control study, 193 women undergoing gynaecological surgery under a standardized anaesthetic with a low emetogenic risk were enrolled. Inclusion and exclusion criteria were designed to select extreme phenotypes as well as to ensure that most major confounders for FIE were either excluded or present in all patients. To control for unforeseen intra- and postoperative confounders for FIE, only 161 patients were further analysed, out of which 10 were categorized as having FIE, defined by the presence of at least one of three symptoms: nausea, vomiting or retching that was likely to be fentanyl-related. To identify SNPs relevant to FIE in our population, DNA from 40 controls and 10 cases was sequenced at the following *OPRM1* regions: 3 kbp of the promoter, main and alternative exons as well as 2 kbp of the 3′ downstream region. The genotype of the significant SNP was further determined in the remaining 111 controls.

### Results

The incidence of FIE was 6.2%. Initial sequencing of 10 cases and 40 controls identified 25 SNPs. Only rs540825, a non-synonymous SNP in the splice variant, MOR1X, showed a significant association with FIE post-Bonferroni correction. This SNP was further examined in the remaining 111 controls which confirmed its significant association with FIE (p = 0.019 post-Bonferroni, OR: 5.6, 95% CI: 1.42–21.91).

### Conclusions

This is the first report of an association between the occurrence of FIE in Chinese women undergoing gynaecological surgery and an *OPRM1* splice variant SNP, rs540825.

## Introduction

Emesis is a distressing side effect of administering opioids such as fentanyl resulting in adverse consequences [[1]](#pone.0048416-Sia1), [[2]](#pone.0048416-Oderda1) but not all patients who are administered opioids develop emesis. Genetic variants are one possible explanation for inter-individual differences in FIE occurrence [[3]](#pone.0048416-Kolesnikov1), [[4]](#pone.0048416-Zhang1).

Single nucleotide polymorphisms (SNPs) are the commonest variant in the human genome [[5]](#pone.0048416-Lander1) so SNPs are obvious potential candidates for initial study when identifying genetic variants predisposing to FIE. The mu-opioid receptor (MOR), encoded by the gene, *OPRM1*, is the main receptor target for fentanyl. We hypothesized that SNPs in *OPRM1* could explain some of the susceptibility to FIE.

Although SNPs in *OPRM1* can affect *OPRM1* expression and function *in vitro* [[6]](#pone.0048416-Ono1), [[7]](#pone.0048416-Margas1), establishing an association between *OPRM1* genetic variants and opioid-induced emesis (OIE) have been less successful [[8]](#pone.0048416-Walter1). Potential reasons for studies not identifying an association between OIE and any genetic variant include heterogeneity in individuals included in such studies (e.g. mixed ethnic groups), variability of opioids and route of administration, and the concomitant use of non-opioid emetogenic drugs and/or anti-emetics resulting in too many confounders to allow identification of a specific phenotype.

To address the issue of study heterogeneity, the patients' eligibility criteria as well as the study protocol in our case-control study was designed so that all study subjects were similar with respect to confounders of OIE including previously reported major non-opioid emetic risk factors [[9]](#pone.0048416-Gan1), [[10]](#pone.0048416-Leslie1) such as gender as well as motion sickness to allow us to ascertain if the emesis observed was likely to be due to the opioid.

Patients receiving the anaesthetic regime in this study were not expected to develop postoperative emesis. Therefore, patients who developed postoperative emesis could be considered to represent the extreme upper end of the emetic risk spectrum and thus regarded as extreme phenotypes. Extreme phenotypes for adverse drug reactions (ADRs) are known to demonstrate a strong genetic basis [[11]](#pone.0048416-Wilke1). Hence, studying extreme phenotypes in OIE is a useful strategy for us to elucidate the genetic basis of OIE.

The large genetic effect size observed in extreme phenotypes is advantageous as study power can be maintained with a smaller study sample size [[12]](#pone.0048416-Nebert1), [[13]](#pone.0048416-Guey1). For example, Nelson et al demonstrated, in a genomewide association study (GWAS) of 500000 SNPs and abacavir hypersensitivity, that only 14 cases and 200 clinically matched controls were needed to attain 80% statistical power to detect a significant association (p-value<10^−7^) between an adverse drug reaction (ADR) with 5% prevalence and SNP with minor allele frequency (MAF) of 5% but large genetic effect of 30 for a dominant model [[14]](#pone.0048416-Nelson1).

In contrast to previous studies on OIE, which genotyped a few selected SNPs in OPRM1, we elected to sequence functionally important genomic regions of *OPRM1* [[8]](#pone.0048416-Walter1), [[15]](#pone.0048416-Chou1). Sequencing was performed to identify novel SNPs and avoided the *a priori* assumption made in genotyping; that the genotyped SNP was the causative SNP. Sequencing extreme phenotypes has been reported to be a powerful strategy for discovering SNPs associated with complex phenotypes [[16]](#pone.0048416-Fahmi1), [[17]](#pone.0048416-Reichardt1) which OIE is likely to be.

*OPRM1* SNPs significantly associated with OIE could be used as a starting platform for further studies on genetic markers not only for OIE but also postoperative nausea and vomiting (PONV).

## Materials and Methods

Approval was obtained from the National Cancer Centre Singapore (NCCS) and the KK Women's Hospital (KKWH) ethical review committees (CRIB 2005/427/B) before commencing our case-control candidate gene study. Written informed consent was obtained from the patient before enrolment into the study.

### Preoperative Patient Selection Criteria

Our study inclusion and exclusion criteria ([Table 1](#pone-0048416-t001)) was designed to ensure that all study subjects were similar with respect to previously reported non-opioid emetic risk factors such as female gender and history of motion sickness [[9]](#pone.0048416-Gan1), [[10]](#pone.0048416-Leslie1). We did not verify whether patients had a history of prior postoperative PONV as this information is subject to recall bias and a diagnosis of PONV does not imply that emesis is necessarily opioid-related.

### Table 1. Inclusion and exclusion criteria for study subjects.

| INCLUSION | EXCLUSION |
| --- | --- |
| Females aged 18–80 years | Declined to participate |
| Subjects parents and grandparents are of Chinese descent by self-report | Allergic to opioids |
| American Society of Anaesthesiologists (ASA) physical status I and II | Undergoing termination of pregnancy or laparoscopic surgery |
| Undergoing gynaecological day surgery at the Day Surgery Unit of KKWH |   |
| Intact cognition |   |
| Body mass index (BMI)<35 |   |
| No history of emesis or anti-emetic usage for at least 2 days prior to entry into the study |   |
| Opioid-naïve |   |
| Non-smokers |   |
| Not pregnant |   |
| No history of migraine or motion sickness |   |
| No history of alcohol or drug abuse |   |

Table 1 Caption: Abbreviations: KKWH: KK Womens Hospital.

### Perioperative Protocol

The perioperative protocol has been outlined as a flowchart in [Figure 1](#pone-0048416-g001). Propofol-based Total Intravenous Anaesthesia (TIVA) without nitrous oxide was administered. To attain a perioperative plasma propofol concentration of 4–6 ug/ml and 3–6 ug/ml during the induction and maintenance phase of anaesthesia respectively, the Marsh algorithm [[18]](#pone.0048416-Marsh1), pre-programmed in the propofol infusor (Asena PK Syringe Pump, Carefusion USA), was employed to compute the propofol dosing rates based on the weight, age and gender of each patient. Hence, the total propofol dose administered was different amongst different patients.

### Figure 1. Flowchart of study perioperative anaesthetic protocol of low emetogenicity for minor gynaecological day surgery.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7396/3492352/a9082883e15f/pone.0048416.g001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3492352_pone.0048416.g001.jpg)

No prophylactic anti-emetics were given pre and intraoperatively. Total intravenous anaesthesia (TIVA) using an intravenous (IV) target controlled infusion (TCI) of propofol, was administered using a propofol infusor (Asena PK Syringe Pump, Carefusion USA). The infusor was pre-programmed with the Marsh algorithm [18], which computed propofol dosing rates to attain a target perioperative plasma propofol concentration of 4–6 ug/ml and 3–6 ug/ml in the induction and maintenance phase of anaesthesia respectively in every patient.

Intravenous (IV) administration of a single bolus dose of 1.5 ug fentanyl/kg body weight, which is a highly selective MOR agonist with no active metabolites, was used to preclude the possibility of differences in emetic occurrence due to differences in types of opioids [[19]](#pone.0048416-Johnston1), their active metabolites or routes of administration, The drug was administered at induction with no further doses given intraoperatively.

The only additional analgesic allowed by protocol was paracetamol. No other opioid or non-opioid analgesics such as non-steroidal anti-inflammatory drugs (NSAID), or anti-emetics were given pre or intraoperatively.

Postoperatively, patients were assessed for the presence or absence of pain, nausea, vomiting and retching at 3 time points; when the patient was awake, just before discharge from the day surgery unit and the next day via telephone to the patient. Analgesics and anti-emetics were available as rescue medications only when required.

### Analyses

Altogether, 193 patients were recruited in this study. However, 32 out of 193 patients were excluded from further analyses for the following reasons. Firstly, 16 had concurrent pain and emesis making it unclear if the occurrence of emesis was directly related to fentanyl while 6 developed unforeseen intraoperative complications requiring the administration of medications, such as midazolam, which might have anti-emetic properties. The final 10 patients were outliers for total propofol dose and they may have been inadvertently misclassified as controls because propofol has anti-emetic properties [[20]](#pone.0048416-Vasileiou1). Hence, only 161 patients comprising 151 controls and 10 cases, were further analyzed ([Figure 2](#pone-0048416-g002)).

### Figure 2. Flowchart outlining the patients enrolled, analysed, classified as cases or controls and sequenced for OPRM1 SNPs.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7396/3492352/d5b40acf67b9/pone.0048416.g002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3492352_pone.0048416.g002.jpg)

A case was defined as a patient who developed a minimum of 1 out of 3 symptoms of emesis that was likely to be fentanyl-related; namely, nausea, vomiting and retching, in at least 1 of 3 postoperative time points of assessment. A control was defined as a patient who did not develop any of the 3 symptoms of emesis at all 3 time points postoperatively.

Nausea, vomiting and retching was treated as a single variable (emesis) with a dichotomous (absent/present) outcome due to the small number of cases which limits meaningful sub-analyses of the emetic phenotype. The rationale for treating nausea and vomiting as a single phenotype is that there may be overlap between the molecular pathways for fentanyl-related nausea and vomiting even though the physiological pathways for nausea differs from that of vomiting.

### DNA Sequencing

DNA from the venous blood of subjects was extracted using the QIAamp DNA Mini Kit (Qiagen, USA).

To identify SNPs relevant to the study population, sequencing of these *OPRM1* regions was initially undertaken in all 10 cases and 40 controls selected by a technician without preference for any specific sample. A control∶case ratio of 4∶1 was used as the basis for sequencing 40 controls and 10 cases as increasing the control∶case ratio beyond 4 to 1 was reported to yield only marginal increases in study power [[21]](#pone.0048416-Grimes1).

Polymerase chain reaction (PCR) and sequencing primers were designed to cover the following genomic regions:

To ensure full sequencing coverage of the promoter and 3′ downstream regions, the sequencing primers span a mean overlap of 145 nucleotides between 2 sequenced regions ([Figures S1](#pone.0048416.s001) and [S2](#pone.0048416.s002) in the Supporting Information). The identified genomic regions in *OPRM1* were amplified by PCR using 10 ng of sample DNA, 5 µl of PCR mastermix from the Qiagen Multiplex PCR kit (Qiagen, USA) and 0.2 pmol/ul of forward and reverse primers in a 10 µl reaction volume. 10 µl of the PCR product was treated with 1 µl ExoSAP (Exonuclease I (Exo) (New England Biolabs, USA):shrimp alkaline phosphatase (SAP) (Promega, USA) at a ratio of 1∶10) at 37°C for 1 hr and 80°C for 15 minutes to remove unincorporated nucleotides and primers.

Big Dye sequencing reactions were performed using 1 µl of ExoSAP treated PCR product, 1 µl of Big Dye Terminator v3.1 Cycle Sequencing Kit (Applied Biosystems (ABI), Life Technologies, USA) and sequencing primer with a final primer concentration of 0.16 pmol/ul in a 10 µl sequencing reaction volume. DNA was precipitated with 100% ethanol and rinsed with 70% ethanol before resuspension of the DNA pellet in 10 µl of Hi-Di solution (ABI, Life Technologies, USA). The DNA was then sequenced using the Genetic Analyzer 3100XL (ABI, Life Technologies, USA). Details regarding all primers, PCR and sequencing thermocycling conditions are listed in [Tables S1](#pone.0048416.s004) and [S2](#pone.0048416.s005) of the Supporting Information.

Base-calling of peaks in the chromatogram was performed using Sequencing Analysis Software v5.3.1 (ABI, Life Technologies, USA) and ContigExpress in Vector NTI ® Advanced 11 (Invitrogen, USA) with corresponding manual chromatogram checks. Double peaks at the same position in the chromatogram that have not been previously identified as SNP loci in NCBI or SNP loci that deviated from Hardy-Weinberg Equilibrium (HWE) underwent bidirectional sequencing to confirm the identity of the peaks.

As rs540825 was noted to be significantly associated with FIE after the initial sequencing, the alternative exon containing this SNP was sequenced in the remaining 111 controls to obtain genotype information for this SNP.

Information for 4 novel SNPs identified during sequencing was deposited in NCBI dbSNP. The accession numbers (ss numbers) for novel variants 1, 2, 3 and 4 in [Table 2](#pone-0048416-t002) are ss528308597, ss 528308608, ss 528308611 and ss 528308612 respectively.

### Table 2. 25 polymorphic SNPs identified during DNA sequencing of 40 controls and 10 cases.

|   | Gene Loci |   |   |   |   | Genotype Frequency In Our Study (%) |  |  | CA Trend |   | Fishers-Exact |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
|   | According To | rs | MOR | AA |   | Homozygous |   | Homozygous | Test |   | Test |
| SNP | MOR11 | Number | Transcript | Change | Patients | Major3 | Heterozygote3 | Minor3 | p-value4 | MAF | p-value4 |
| 1 | 5′UR/T-2694G 2 | rs12210856 | - |   | Cases | 90.00 | 10.00 | - | 0.197 | 5.00 | 0.46 |
|   |   |   |   |   | Controls | 70.00 | 30.00 | - |   | 15.00 |   |
| 2 | 5′UR/A-2455C | rs12190259 | - |   | Cases | 90.00 | 10.00 | - | 0.197 | 5.00 | 0.46 |
|   |   |   |   |   | Controls | 70.00 | 30.00 | - |   | 15.00 |   |
| 3 | 5′UR/G-1509A | rs12205732 | - |   | Cases | 90.00 | 10.00 | - | 0.197 | 5.00 | 0.46 |
|   |   |   |   |   | Controls | 70.00 | 30.00 | - |   | 15.00 |   |
| 4 | 5′UTR/G-172T | rs6912029 | MOR1 |   | Cases | 90.00 | 10.00 | - | 0.156 | 5.00 | 0.29 |
|   |   |   |   |   | Controls | 67.50 | 32.50 | - |   | 16.25 |   |
| 5 | E1/A118G | rs1799971 | MOR1 | Asn→Asp | Cases | 30.00 | 70.00 | - | 0.643 | 35.00 | 0.80 |
|   |   |   |   |   | Controls | 32.50 | 55.00 | 12.50 |   | 40.00 |   |
| 6 | I1/G-2994A | rs563649 | MOR1K1 |   | Cases | 90.00 | 10.00 | - | 0.247 | 5.00 | 0.45 |
|   |   |   | 5′UTR |   | Controls | 72.50 | 27.50 | - |   | 13.75 |   |
| 7 | I1/G -2259A | rs9322446 | MOR1K1 |   | Cases | 90.00 | 10.00 | - | 1.000 | 5.00 | 1.00 |
|   |   |   | 5′UTR |   | Controls | 90.00 | 10.00 | - |   | 5.00 |   |
| 8 | E3/G877A | NV1 | MOR1 | Val→Ile | Cases | 100.00 | 0.00 | - | 0.471 | 0.00 | 1.00 |
|   |   |   |   |   | Controls | 95.00 | 5.00 | - |   | 2.50 |   |
| 9 | I3/G399T | NV2 | MOR1A |   | Cases | 90.00 | 10.00 | - | 0.043 | 5.00 | 0.20 |
|   |   |   | 3′UTR |   | Controls | 100.00 | 0.00 | - |   | 0.00 |   |
| 10 | I3/A1839T | rs540825 | MOR1X | Glu→His | Cases | 60.00 | 40.00 | - | 0.002 * (0.035 * ) | 20.00 | 0.01 * (0.15) |
|   |   |   | E4 |   | Controls | 95.00 | 5.00 | - |   | 2.50 |   |
| 11 | I3/C1956T | rs675026 | MOR1X | Gly→Gly | Cases | 60.00 | 30.00 | 10.00 | 0.030* (0.456) | 25.00 | 0.04* (0.60) |
|   |   |   | E4 |   | Controls | 87.50 | 10.00 | 2.50 |   | 7.50 |   |
| 12 | I3/A1966G | rs562859 | MOR1X | Leu→Leu | Cases | 70.00 | 20.00 | 10.00 | 0.103 | 20.00 | 0.11 |
|   |   |   | E4 |   | Controls | 87.50 | 10.00 | 2.50 |   | 7.50 |   |
| 13 | I3/C-11152T | NV3 | MOR1B5 | Glu→Stop | Cases | 60.00 | 40.00 | - | 0.042* (0.633) | 20.00 | 0.05 |
|   |   |   |   |   | Controls | 92.50 | 5.00 | 2.50 |   | 5.00 |   |
| 14 | I3/C18957T | rs606545 | MOR1B1 |   | Cases | 60.00 | 40.00 | - | 0.042* (0.633) | 20.00 | 0.05 |
|   |   |   | 3′UTR |   | Controls | 92.50 | 5.00 | 2.50 |   | 5.00 |   |
| 15 | 3′UTR/A725G | rs17181352 | MOR1 |   | Cases | 100.00 | 0.00 | - | 0.614 | 0.00 | 1.00 |
|   |   |   |   |   | Controls | 97.50 | 2.50 | - |   | 1.25 |   |
| 16 | 3′DR/A148G | rs671531 | - |   | Cases | 80.00 | 20.00 | - | 0.459 | 10.00 | 0.60 |
|   |   |   |   |   | Controls | 92.50 | 5.00 | 2.50 |   | 5.00 |   |
| 17 | 3′DR/A633G | rs583664 | - |   | Cases | 70.00 | 30.00 | - | 0.158 | 15.00 | 0.14 |
|   |   |   |   |   | Controls | 92.50 | 5.00 | 2.50 |   | 5.00 |   |
| 18 | 3′DR/C932T | rs658156 | - |   | Cases | 60.00 | 40.00 | - | 0.042* (0.633) | 20.00 | 0.05 |
|   |   |   |   |   | Controls | 92.50 | 5.00 | 2.50 |   | 5.00 |   |
| 19 | 3′DR/G971A | rs9371776 | - |   | Cases | 100.00 | - | - | 0.372 | - | 1.00 |
|   |   |   |   |   | Controls | 92.50 | 7.50 | - |   | 3.75 |   |
| 20 | 3′DR/G1270A | rs558948 | - |   | Cases | 60.00 | 40.00 | - | 0.042* (0.633) | 20.00 | 0.05 |
|   |   |   |   |   | Controls | 92.50 | 5.00 | 2.50 |   | 5.00 |   |
| 21 | 3′DR/T1371C | rs558025 | - |   | Cases | 60.00 | 40.00 | - | 0.042* (0.633) | 20.00 | 0.05 |
|   |   |   |   |   | Controls | 92.50 | 5.00 | 2.50 |   | 5.00 |   |
| 22 | 3′DR/C1510T | rs645027 | - |   | Cases | 80.00 | 20.00 | - | 0.44 | 10.00 | 0.73 |
|   |   |   |   |   | Controls | 67.50 | 32.50 | - |   | 16.25 |   |
| 23 | 3′DR/C1549T | rs598160 | - |   | Cases | 60.00 | 40.00 | - | 0.042* (0.633) | 20.00 | 0.05 |
|   |   |   |   |   | Controls | 92.50 | 5.00 | 2.50 |   | 5.00 |   |
| 24 | 3′DR/C1657G | rs644261 | - |   | Cases | 60.00 | 40.00 | - | 0.042* (0.633) | 20.00 | 0.05 |
|   |   |   |   |   | Controls | 92.50 | 5.00 | 2.50 |   | 5.00 |   |
| 25 | 3′DR/G1665A | NV4 | - |   | Cases | 100.00 | - | - | 0.614 | - | 1.00 |
|   |   |   |   |   | Controls | 97.50 | 2.50 | - |   | 1.25 |   |

Table 2 Caption: Footnote: SNP 10, rs540825, was significantly associated with fentanyl-induced emesis pre- and post- Bonferroni correction. Abbreviations: SNP: Single nucleotide polymorphism; UR: Upstream region; DR: Downstream region; UTR: Untranslated region; E: Exon; I: Intron; NV: Novel variant; MOR: Mu opioid receptor; NCBI: National Centre For Biotechnology Information; CA: Cochran-Armitage; AA: Amino acid; MAF: Minor allele frequency; Asn: Asparigine; Asp: Aspartate; Val: Valine; Ile: Isoleucine; Glu: Glutamine; His: Histidine; Gly: Glycine; Leu: Leucine. Legends: 1MOR1 is the main transcript of OPRM1. Other transcripts of MOR1 e.g. MOR1X and MOR1K1 are splice variants of MOR. 2The letter in capitals refers to the minor allele of the SNP. 3Patients with 2 copies of the major or minor allele respectively. Heterozygote: Patients with 1 copy each of the major and minor allele. 4Non-bracketed p-values for the CA trend test and Fishers-Exact test refers to p-values before Bonferroni correction. Bracketed (p-values) refers to p-values post-Bonferroni correction. *p-value<0.05 indicates statistical significance.

### Statistical Analyses

#### Clinical Data

Incidence of FIE was calculated as the total number of patients (n = 10), who developed emesis that was likely to be fentanyl-related divided by the total number of patients who were at risk of developing FIE (n = 161). Outliers for total propofol dose (TPD≥535 mg) were determined based on the following formula:

| Category |
| --- |
|   |
These outliers were excluded from further analyses since propofol has anti-emetic properties and the patients could have been misclassified [[20]](#pone.0048416-Vasileiou1). The median was used to compare age, total propofol and total fentanyl dose between cases and controls. The statistical significance of differences in median between the 2 groups was determined using the Mann-Whitney test. All basic statistical analyses of the clinical data was performed using Minitab®15 (Minitab Inc,USA).

#### Genotype Data

HWE was calculated for each SNP in the controls. SNPs with HWE<0.001, indicating significant deviation from HWE, were discarded from analyses [[22]](#pone.0048416-Balding1) due to possible errors in base-calling for such SNPs. SNP allele and genotype frequencies were calculated. The Fishers-Exact test was used to test for the significance of allelic frequency differences between cases and controls. The Cochran-Armitage (CA) Trend test for genotypes was used to correlate SNP genotypes with FIE under a co-dominant model [[23]](#pone.0048416-Freidlin1). Haplotypes of SNPs for each identified sequenced genomic region and their frequencies were inferred using the Expectation-Maximization (EM) algorithm [[24]](#pone.0048416-Excoffier1). PLINK, an online tool for genetic association analyses [[25]](#pone.0048416-Purcell1), was used to perform the CA Trend test and haplotype analysis.

Linkage disequilibrium (LD) between 2 SNPs was represented by the pairwise r^2^ association, where r^2^≥0.8 indicates high LD suggesting strong association. The LD profile surrounding the SNP of interest was determined in our study population and compared with other HAPMAP populations including CHB (Han Chinese in Beijing) and CEU (Utah residents with Northern and Western European ancestry) [[26]](#pone.0048416-Consortium1).

Bonferroni multiple test correction was performed to correct for Type I error. The uncorrected p-values were multiplied by a factor of 15, which represents the number of SNPs whose occurrence was considered to be independent of other SNPs. The number of independent SNPs was derived from the LD profile of the sequenced SNPs where 14 out of 25 SNPs could be clustered into 4 groups ([Figure S3](#pone.0048416.s003) of the Supporting Information).

The odds ratio (OR) for SNP-phenotype correlations was calculated as the ratio of the odds of developing FIE (cases) to the odds of not developing FIE (controls) with the SNP allele. The population attributable risk (PAR) is the proportion of patients in our study with a particular allele, genotype or haplotype that significantly correlate with FIE. The PAR was calculated using the formula PAR = P*e*(R-1)/1+P*e*(R-1) where R is the relative risk and P*e* is the proportion of patients with the allele, genotype or haplotype that significantly correlates with FIE [[27]](#pone.0048416-Taylor1).

Logistic regression for a binary outcome (FIE) incorporating the variables of age, total propofol dose, total fentanyl dose, type of surgery (dilation/curettage/hysteroscopy versus other types of minor gynaecological surgery) and SNP genotype was performed for 151 controls and 10 cases using Minitab®15 (Minitab Inc,USA).

## Results

Ten out of 161 patients analysed in this study developed emesis, which was likely to be fentanyl-induced, giving an FIE incidence of 6.2%.

Three variables that may have contributed to differences in emetic occurrence between cases and controls were total propofol dose, total fentanyl dose and age. Total fentanyl and total propofol dose can differ between individuals as these variables are dependent on the patients' weight and age in this study although the fentanyl dose (µg/kg) and the perioperative propofol plasma concentration (µg/ml) is similar amongst all the patients.

However, no significant differences in the median values for these 3 variables between cases and controls were observed ([Figure 3A–C](#pone-0048416-g003)). Hence, these 3 variables were unlikely to contribute significantly to differences in the occurrence of postoperative emesis.

### Figure 3. Boxplots indicating the distribution of values for 3 variables in 151 controls and 10 cases.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7396/3492352/2b9b74892fdd/pone.0048416.g003.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3492352_pone.0048416.g003.jpg)

These 3 variables could contribute to the occurrence of emesis but could not be controlled for in the study design (a) Age (b) Total propofol dose (c) Total fentanyl dose. The median is the horizontal line bisecting the shaded box of the boxplot and the median value placed adjacent to the horizontal line. There are no significant differences (p<0.05) between the median values of controls and cases for the 3 variables.

Altogether, 25 polymorphic SNPs were identified in the sequenced regions of 40 controls and 10 cases ([Table 2](#pone-0048416-t002)).

### rs540825 is a non-synonymous SNP in an alternative exon of OPRM1 that significantly correlates with FIE

Amongst the 25 SNPs, there was a significant difference (p = 0.01) before Bonferroni correction in the minor allele frequency (MAF) of SNP rs540825 between 40 controls and 10 cases ([Table 2](#pone-0048416-t002)). Notably, the CA Trend test [[23]](#pone.0048416-Freidlin1) for genotypes showed a significant correlation between the genotypes of rs540825 and FIE both pre (p = 0.002) as well as post-Bonferroni (p = 0.035) correction in 40 controls and 10 cases ([Table 2](#pone-0048416-t002)).

To determine if the significant correlation would still be observed when more samples were examined, the exon containing this SNP was sequenced in an additional 111 controls since there were no additional cases. Notably, results similar to that observed with the smaller sample size were obtained when more controls were examined (Fishers-Exact test (p = 0.04 (pre-) and p = 0.60 (post-Bonferroni)) as well as the CA Trend test (p = 0.001 (pre-) and p = 0.019 (post-Bonferroni)).

The OR in 151 controls and 10 cases was 5.6 (95% CI: 1.42–21.91) suggesting that individuals carrying this SNP were 5.6 times more likely to develop FIE. In addition, the PAR analyses indicated that 31.2% of FIE in our study could be attributed to the minor allele of rs540825.

Logistic regression of the 5 variables listed in the [Materials and Methods](#s2), that could contribute to the emesis in this study, indicated that the rs540825 genotype was the only variable that significantly affected the risk of developing FIE (OR: 3.96, 95%CI: 1.12–14.08). The OR for other non-genetic variables such as surgery and total propofol dose ranged from 0.97 to 1.05.

This SNP was observed to be in low linkage disequilibrium (LD) (r^2^<0.8) with other sequenced polymorphic SNPs in our study ([Figure S3](#pone.0048416.s003) of the Supporting Information). Moreover, its LD pattern in this study concurred with its LD pattern in the CHB and JPT populations from HAPMAP [[28]](#pone.0048416-Consortium2) suggesting that this was highly likely to be the SNP contributing to FIE.

## Discussion

The 2 main findings in this study were a low incidence of FIE under the specific study conditions and the significant correlation of FIE with a non-synonymous SNP, rs540825. This SNP resides in the alternative last exon encoding the C-terminus of MOR1X, a known splice variant of MOR, which is differentially expressed within the brain [[29]](#pone.0048416-Pan1).

Exonic SNPs encoding the main transcript have been reported to be responsible for inter-individual differences in drug response [[1]](#pone.0048416-Sia1), [[30]](#pone.0048416-Huang1), [[31]](#pone.0048416-Ross1). Our study highlights that non-synonymous SNPs, in alternative exons of splice variants, represent another important genetic source of variation in drug response.

The effect size of this SNP is comparable to the effect sizes for ADRs in other reports where the OR ranges from 2.2 (HLA-A*020 allele in drug induced liver injury) [[32]](#pone.0048416-Daly1) to 9 (rs1805128 allele in *KCNE1* for drug-induced Torsades de Pointes) [[33]](#pone.0048416-Kaab1) and >20 (HLA-A*3101 allele in Stevens–Johnson syndrome and toxic epidermal necrolysis) [[34]](#pone.0048416-McCormack1). The OR observed in our study also reflects our study design where extreme phenotypes occur in small numbers but the genetic basis for the phenotype is strong. The PAR of this SNP (31.2%) is also notably higher than the PAR of other single SNPs implicated in drug response. For example, the PAR of 2 SNPs implicated in thiazolidinedione-related odema was 29.8% and 18.8%, [[35]](#pone.0048416-Chang1) whilst the PAR of the rs7158782 SNP in *TCL1A*, which conferred an increased risk of aromatase inhibitor-related musculoskeletal side effects, was 11% [[36]](#pone.0048416-Ingle1), [[37]](#pone.0048416-Offit1).

Interestingly, the MAF of this SNP in the HAPMAP Caucasian (CEU 27.9%) and the Chinese (Chinese in Singapore (CHS) 6.5%, CHB 4.2%, CEU 27.9%) populations is very different, suggesting that the effect of this SNP on FIE risk may be population specific. Population-specific effects of SNPs have been previously reported for ADRs [[38]](#pone.0048416-Aihara1). Significantly, when F_ST_, which measures the degree of genetic differentiation between 2 populations, was calculated for this SNP using data from the HAPMAP Public Release #27 dataset, the F_ST_ for the CHB-CEU populations (F_ST (CHB-CEU)_) was 0.17, placing it amongst the top 5% of F_st_ values for all SNPs in the CHB-CEU populations (unpublished data courtesy of M. Bachtiar).

This SNP has also previously been associated with a specific subset of citalopram (antidepressant) response that was observed only in non-Hispanic-Whites but not African-Americans or Hispanic-Whites [[39]](#pone.0048416-Garriock1). In contrast, Laugsand *et al*, reported no significant correlation between this SNP and OIE in a study of 1579 European patients with advanced cancer taking opioids for symptom control, where 96 SNPs with MAF≥10% were genotyped [[40]](#pone.0048416-Laugsand1). A plausible explanation for the disparate findings may lie in the different study design employed by the European investigators and us.

In the European study, subjects were heterogeneous for at least 10 variables that may act as confounders for the genotyped SNP and OIE. These confounders included the age, type of cancer, site of metastases, age, gender, type of opioid, stability of opioid dosing, concomitant use of anti-emetics, steroids in the past 24 hours in addition to other medications, past medical history and clinical setting (outpatients or hospital inpatient). The large number of confounders creates difficulty in distinguishing emesis due to opioids from non-opioid causes as well as genetic from non-genetic factors. Notably, if the European study was conducted in populations of Chinese descent where the MAF of this SNP is <10%, this SNP would not have been genotyped.

This SNP, rs540825, changes an amino acid in the C-terminus of the splice variant, MOR1X, from a neutral (glutamine) to basic (histidine) amino acid. MOR1X was previously reported to be functionally different from MOR1, the main transcript [[29]](#pone.0048416-Pan1). In addition, mutations in the C-terminus of MOR1 have been reported to affect agonist potency and affinity [[41]](#pone.0048416-Waldhoer1), [[42]](#pone.0048416-Pan2). Hence, this SNP, which results in a non-synonymous change at the C-terminus of MOR1X, could conceivably affect MOR1X structure and function. Although Garriock *et al* [[39]](#pone.0048416-Garriock1) did not observe significant differences between the major and minor alleles of this SNP for adenylyl cyclase inhibition and ligand-induced receptor endocytocis *in vitro*, the functional effect of the minor allele may be related to established MOR signaling processes such as ion channel coupling, protein kinase phosphorylation [[41]](#pone.0048416-Waldhoer1) and MOR dimerization with other receptors [[43]](#pone.0048416-vanRijn1), which do not involve adenylyl cyclase/cAMP and were not investigated in their study.

The main limitations of this study are the lack of a second study to demonstrate the replication of the association of rs540825 with FIE and the small case numbers. Replication studies may be the norm for genetic association studies of common disease. However, undertaking replication studies for rare ADRs, including FIE in our study, or extreme phenotypes, which tend to be few in number, can be daunting due to the difficulties in collecting large numbers of cases over a reasonable period of time [[44]](#pone.0048416-Daly2). The complexities of emesis also make subgroup analyses difficult due to the small numbers in each subgroup.

This study describes the first association between a non-synonymous SNP in an alternative splice variant of OPRM1, rs540825, and FIE. Our study finding could potentially contribute to the development of future preoperative screening tools to identify individuals at risk of FIE for prophylactic anti-emetics. However, further studies are needed to evaluate this association in different ethnic groups and confirm the clinical utility of predicting the FIE risk.

## Supporting Information

## Funding Statement

This work was supported by grants from the Academic Research Fund – Ministry of Education Tier 2 (MOE T206B3105), the Academic Research Fund – Tier 1 (R-183-000-172-112) to CL through the National University of Singapore; the National Medical Research Council (NMRC) (NMRC/0993/2005), Singapore to CL through the National Cancer Centre, Singapore; Singhealth Foundation Start Up Grant, Singapore, Singapore Millennium Foundation, NMRC Fellowship and Cancer Research Endowment Fund, National Cancer Centre Singapore to GP through the National Cancer Centre as well as Singapore Millennium Foundation and block funding to CL through the National Cancer Centre. The funders had no role in the study design, data collection and analyses, decision to publish, or preparation of the manuscript.

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Sia AT, Lim Y, Lim EC, Goh RW, Law HY, et al. (2008) A118G single nucleotide polymorphism of human mu-opioid receptor gene influences pain perception and patient-controlled intravenous morphine consumption after intrathecal morphine for postcesarean analgesia. Anesthesiology 109: 520–526.  [DOI](https://doi.org/10.1097/ALN.0b013e318182af21) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18719451/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Anesthesiology&title=A118G%20single%20nucleotide%20polymorphism%20of%20human%20mu-opioid%20receptor%20gene%20influences%20pain%20perception%20and%20patient-controlled%20intravenous%20morphine%20consumption%20after%20intrathecal%20morphine%20for%20postcesarean%20analgesia&author=AT%20Sia&author=Y%20Lim&author=EC%20Lim&author=RW%20Goh&author=HY%20Law&volume=109&publication_year=2008&pages=520-526&pmid=18719451&doi=10.1097/ALN.0b013e318182af21&)

2. Oderda GM, Said Q, Evans RS, Stoddard GJ, Lloyd J, et al. (2007) Opioid-related adverse drug events in surgical hospitalizations: impact on costs and length of stay. Ann Pharmacother 41: 400–406.  [DOI](https://doi.org/10.1345/aph.1H386) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17341537/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Pharmacother&title=Opioid-related%20adverse%20drug%20events%20in%20surgical%20hospitalizations:%20impact%20on%20costs%20and%20length%20of%20stay&author=GM%20Oderda&author=Q%20Said&author=RS%20Evans&author=GJ%20Stoddard&author=J%20Lloyd&volume=41&publication_year=2007&pages=400-406&pmid=17341537&doi=10.1345/aph.1H386&)

3. Kolesnikov Y, Gabovits B, Levin A, Voiko E, Veske A (2011) Combined catechol-O-methyltransferase and mu-opioid receptor gene polymorphisms affect morphine postoperative analgesia and central side effects. Anesth Analg 112: 448–453.  [DOI](https://doi.org/10.1213/ANE.0b013e318202cc8d) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21127283/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Anesth%20Analg&title=Combined%20catechol-O-methyltransferase%20and%20mu-opioid%20receptor%20gene%20polymorphisms%20affect%20morphine%20postoperative%20analgesia%20and%20central%20side%20effects&author=Y%20Kolesnikov&author=B%20Gabovits&author=A%20Levin&author=E%20Voiko&author=A%20Veske&volume=112&publication_year=2011&pages=448-453&pmid=21127283&doi=10.1213/ANE.0b013e318202cc8d&)

4. Zhang W, Yuan JJ, Kan QC, Zhang LR, Chang YZ, et al. (2011) Study of the OPRM1 A118G genetic polymorphism associated with postoperative nausea and vomiting induced by fentanyl intravenous analgesia. Minerva Anestesiol 77: 33–39.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/21150856/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Minerva%20Anestesiol&title=Study%20of%20the%20OPRM1%20A118G%20genetic%20polymorphism%20associated%20with%20postoperative%20nausea%20and%20vomiting%20induced%20by%20fentanyl%20intravenous%20analgesia&author=W%20Zhang&author=JJ%20Yuan&author=QC%20Kan&author=LR%20Zhang&author=YZ%20Chang&volume=77&publication_year=2011&pages=33-39&pmid=21150856&)

5. Lander ES, Linton LM, Birren B, Nusbaum C, Zody MC, et al. (2001) Initial sequencing and analysis of the human genome. Nature 409: 860–921.  [DOI](https://doi.org/10.1038/35057062) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11237011/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Initial%20sequencing%20and%20analysis%20of%20the%20human%20genome&author=ES%20Lander&author=LM%20Linton&author=B%20Birren&author=C%20Nusbaum&author=MC%20Zody&volume=409&publication_year=2001&pages=860-921&pmid=11237011&doi=10.1038/35057062&)

6. Ono T, Kaneda T, Muto A, Yoshida T (2009) Positive transcriptional regulation of the human micro opioid receptor gene by poly(ADP-ribose) polymerase-1 and increase of its DNA binding affinity based on polymorphism of G-172→T. J Biol Chem 284: 20175–20183.  [DOI](https://doi.org/10.1074/jbc.M109.019414) | [PMC free article](/articles/PMC2740444/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19447888/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=Positive%20transcriptional%20regulation%20of%20the%20human%20micro%20opioid%20receptor%20gene%20by%20poly(ADP-ribose)%20polymerase-1%20and%20increase%20of%20its%20DNA%20binding%20affinity%20based%20on%20polymorphism%20of%20G-172%E2%86%92T&author=T%20Ono&author=T%20Kaneda&author=A%20Muto&author=T%20Yoshida&volume=284&publication_year=2009&pages=20175-20183&pmid=19447888&doi=10.1074/jbc.M109.019414&)

7. Margas W, Zubkoff I, Schuler HG, Janicki PK, Ruiz-Velasco V (2007) Modulation of Ca2+ channels by heterologously expressed wild-type and mutant human micro-opioid receptors (hMORs) containing the A118G single-nucleotide polymorphism. J Neurophysiol 97: 1058–1067.  [DOI](https://doi.org/10.1152/jn.01007.2006) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17151221/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Neurophysiol&title=Modulation%20of%20Ca2+%20channels%20by%20heterologously%20expressed%20wild-type%20and%20mutant%20human%20micro-opioid%20receptors%20(hMORs)%20containing%20the%20A118G%20single-nucleotide%20polymorphism&author=W%20Margas&author=I%20Zubkoff&author=HG%20Schuler&author=PK%20Janicki&author=V%20Ruiz-Velasco&volume=97&publication_year=2007&pages=1058-1067&pmid=17151221&doi=10.1152/jn.01007.2006&)

8. Walter C, Lotsch J (2009) Meta-analysis of the relevance of the OPRM1 118A>G genetic variant for pain treatment. Pain 146: 270–275.  [DOI](https://doi.org/10.1016/j.pain.2009.07.013) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19683391/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pain&title=Meta-analysis%20of%20the%20relevance%20of%20the%20OPRM1%20118A>G%20genetic%20variant%20for%20pain%20treatment&author=C%20Walter&author=J%20Lotsch&volume=146&publication_year=2009&pages=270-275&pmid=19683391&doi=10.1016/j.pain.2009.07.013&)

9. Gan TJ, Meyer TA, Apfel CC, Chung F, Davis PJ, et al. (2007) Society for Ambulatory Anesthesia guidelines for the management of postoperative nausea and vomiting. Anesth Analg 105: 1615–1628, table of contents.  [DOI](https://doi.org/10.1213/01.ane.0000295230.55439.f4) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18042859/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Anesth%20Analg&title=Society%20for%20Ambulatory%20Anesthesia%20guidelines%20for%20the%20management%20of%20postoperative%20nausea%20and%20vomiting&author=TJ%20Gan&author=TA%20Meyer&author=CC%20Apfel&author=F%20Chung&author=PJ%20Davis&volume=105&publication_year=2007&pages=1615-1628,%20table%20of%20contents&pmid=18042859&doi=10.1213/01.ane.0000295230.55439.f4&)

10. Leslie K, Myles PS, Chan MT, Paech MJ, Peyton P, et al. (2008) Risk factors for severe postoperative nausea and vomiting in a randomized trial of nitrous oxide-based vs nitrous oxide-free anaesthesia. Br J Anaesth 101: 498–505.  [DOI](https://doi.org/10.1093/bja/aen230) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18682411/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Anaesth&title=Risk%20factors%20for%20severe%20postoperative%20nausea%20and%20vomiting%20in%20a%20randomized%20trial%20of%20nitrous%20oxide-based%20vs%20nitrous%20oxide-free%20anaesthesia&author=K%20Leslie&author=PS%20Myles&author=MT%20Chan&author=MJ%20Paech&author=P%20Peyton&volume=101&publication_year=2008&pages=498-505&pmid=18682411&doi=10.1093/bja/aen230&)

11. Wilke RA, Lin DW, Roden DM, Watkins PB, Flockhart D, et al. (2007) Identifying genetic risk factors for serious adverse drug reactions: current progress and challenges. Nat Rev Drug Discov 6: 904–916.  [DOI](https://doi.org/10.1038/nrd2423) | [PMC free article](/articles/PMC2763923/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17971785/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Drug%20Discov&title=Identifying%20genetic%20risk%20factors%20for%20serious%20adverse%20drug%20reactions:%20current%20progress%20and%20challenges&author=RA%20Wilke&author=DW%20Lin&author=DM%20Roden&author=PB%20Watkins&author=D%20Flockhart&volume=6&publication_year=2007&pages=904-916&pmid=17971785&doi=10.1038/nrd2423&)

12. Nebert DW (2000) Extreme discordant phenotype methodology: an intuitive approach to clinical pharmacogenetics. Eur J Pharmacol 410: 107–120.  [DOI](https://doi.org/10.1016/s0014-2999(00)00809-8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11134663/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Pharmacol&title=Extreme%20discordant%20phenotype%20methodology:%20an%20intuitive%20approach%20to%20clinical%20pharmacogenetics&author=DW%20Nebert&volume=410&publication_year=2000&pages=107-120&pmid=11134663&doi=10.1016/s0014-2999(00)00809-8&)

13. Guey LT, Kravic J, Melander O, Burtt NP, Laramie JM, et al. (2011) Power in the phenotypic extremes: a simulation study of power in discovery and replication of rare variants. Genet Epidemiol  [DOI](https://doi.org/10.1002/gepi.20572) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21308769/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genet%20Epidemiol&title=Power%20in%20the%20phenotypic%20extremes:%20a%20simulation%20study%20of%20power%20in%20discovery%20and%20replication%20of%20rare%20variants&author=LT%20Guey&author=J%20Kravic&author=O%20Melander&author=NP%20Burtt&author=JM%20Laramie&publication_year=2011&pmid=21308769&doi=10.1002/gepi.20572&)

14. Nelson MR, Bacanu SA, Mosteller M, Li L, Bowman CE, et al. (2009) Genome-wide approaches to identify pharmacogenetic contributions to adverse drug reactions. Pharmacogenomics J 9: 23–33.  [DOI](https://doi.org/10.1038/tpj.2008.4) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18301416/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=Genome-wide%20approaches%20to%20identify%20pharmacogenetic%20contributions%20to%20adverse%20drug%20reactions&author=MR%20Nelson&author=SA%20Bacanu&author=M%20Mosteller&author=L%20Li&author=CE%20Bowman&volume=9&publication_year=2009&pages=23-33&pmid=18301416&doi=10.1038/tpj.2008.4&)

15. Chou WY, Wang CH, Liu PH, Liu CC, Tseng CC, et al. (2006) Human opioid receptor A118G polymorphism affects intravenous patient-controlled analgesia morphine consumption after total abdominal hysterectomy. Anesthesiology 105: 334–337.  [DOI](https://doi.org/10.1097/00000542-200608000-00016) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16871067/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Anesthesiology&title=Human%20opioid%20receptor%20A118G%20polymorphism%20affects%20intravenous%20patient-controlled%20analgesia%20morphine%20consumption%20after%20total%20abdominal%20hysterectomy&author=WY%20Chou&author=CH%20Wang&author=PH%20Liu&author=CC%20Liu&author=CC%20Tseng&volume=105&publication_year=2006&pages=334-337&pmid=16871067&doi=10.1097/00000542-200608000-00016&)

16. Fahmi S, Yang C, Esmail S, Hobbs HH, Cohen JC (2008) Functional characterization of genetic variants in NPC1L1 supports the sequencing extremes strategy to identify complex trait genes. Hum Mol Genet 17: 2101–2107.  [DOI](https://doi.org/10.1093/hmg/ddn108) | [PMC free article](/articles/PMC2441721/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18413323/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mol%20Genet&title=Functional%20characterization%20of%20genetic%20variants%20in%20NPC1L1%20supports%20the%20sequencing%20extremes%20strategy%20to%20identify%20complex%20trait%20genes&author=S%20Fahmi&author=C%20Yang&author=S%20Esmail&author=HH%20Hobbs&author=JC%20Cohen&volume=17&publication_year=2008&pages=2101-2107&pmid=18413323&doi=10.1093/hmg/ddn108&)

17. Reichardt JK, Mehrian-Shai R (2009) The future of the human SNP identification: which individuals to sequence? Proc Natl Acad Sci U S A 106: E50.  [DOI](https://doi.org/10.1073/pnas.0903681106) | [PMC free article](/articles/PMC2683127/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19416875/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&title=The%20future%20of%20the%20human%20SNP%20identification:%20which%20individuals%20to%20sequence?&author=JK%20Reichardt&author=R%20Mehrian-Shai&volume=106&publication_year=2009&pages=E50&pmid=19416875&doi=10.1073/pnas.0903681106&)

18. Marsh B, White M, Morton N, Kenny GN (1991) Pharmacokinetic model driven infusion of propofol in children. Br J Anaesth 67: 41–48.  [DOI](https://doi.org/10.1093/bja/67.1.41) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/1859758/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Anaesth&title=Pharmacokinetic%20model%20driven%20infusion%20of%20propofol%20in%20children&author=B%20Marsh&author=M%20White&author=N%20Morton&author=GN%20Kenny&volume=67&publication_year=1991&pages=41-48&pmid=1859758&doi=10.1093/bja/67.1.41&)

19. Johnston KD (2010) The potential for mu-opioid receptor agonists to be anti-emetic in humans: a review of clinical data. Acta Anaesthesiol Scand 54: 132–140.  [DOI](https://doi.org/10.1111/j.1399-6576.2009.02115.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19817719/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Acta%20Anaesthesiol%20Scand&title=The%20potential%20for%20mu-opioid%20receptor%20agonists%20to%20be%20anti-emetic%20in%20humans:%20a%20review%20of%20clinical%20data&author=KD%20Johnston&volume=54&publication_year=2010&pages=132-140&pmid=19817719&doi=10.1111/j.1399-6576.2009.02115.x&)

20. Vasileiou I, Xanthos T, Koudouna E, Perrea D, Klonaris C, et al. (2009) Propofol: a review of its non-anaesthetic effects. Eur J Pharmacol 605: 1–8.  [DOI](https://doi.org/10.1016/j.ejphar.2009.01.007) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19248246/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Pharmacol&title=Propofol:%20a%20review%20of%20its%20non-anaesthetic%20effects&author=I%20Vasileiou&author=T%20Xanthos&author=E%20Koudouna&author=D%20Perrea&author=C%20Klonaris&volume=605&publication_year=2009&pages=1-8&pmid=19248246&doi=10.1016/j.ejphar.2009.01.007&)

21. Grimes DA, Schulz KF (2005) Compared to what? Finding controls for case-control studies. Lancet 365: 1429–1433.  [DOI](https://doi.org/10.1016/S0140-6736(05)66379-9) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15836892/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Compared%20to%20what?%20Finding%20controls%20for%20case-control%20studies&author=DA%20Grimes&author=KF%20Schulz&volume=365&publication_year=2005&pages=1429-1433&pmid=15836892&doi=10.1016/S0140-6736(05)66379-9&)

22. Balding DJ (2006) A tutorial on statistical methods for population association studies. Nat Rev Genet 7: 781–791.  [DOI](https://doi.org/10.1038/nrg1916) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16983374/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Genet&title=A%20tutorial%20on%20statistical%20methods%20for%20population%20association%20studies&author=DJ%20Balding&volume=7&publication_year=2006&pages=781-791&pmid=16983374&doi=10.1038/nrg1916&)

23. Freidlin B, Zheng G, Li Z, Gastwirth JL (2002) Trend tests for case-control studies of genetic markers: power, sample size and robustness. Hum Hered 53: 146–152.  [DOI](https://doi.org/10.1159/000064976) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12145550/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Hered&title=Trend%20tests%20for%20case-control%20studies%20of%20genetic%20markers:%20power,%20sample%20size%20and%20robustness&author=B%20Freidlin&author=G%20Zheng&author=Z%20Li&author=JL%20Gastwirth&volume=53&publication_year=2002&pages=146-152&pmid=12145550&doi=10.1159/000064976&)

24. Excoffier L, Slatkin M (1995) Maximum-likelihood estimation of molecular haplotype frequencies in a diploid population. Mol Biol Evol 12: 921–927.  [DOI](https://doi.org/10.1093/oxfordjournals.molbev.a040269) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7476138/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Biol%20Evol&title=Maximum-likelihood%20estimation%20of%20molecular%20haplotype%20frequencies%20in%20a%20diploid%20population&author=L%20Excoffier&author=M%20Slatkin&volume=12&publication_year=1995&pages=921-927&pmid=7476138&doi=10.1093/oxfordjournals.molbev.a040269&)

25. Purcell S, Neale B, Todd-Brown K, Thomas L, Ferreira MA, et al. (2007) PLINK: a tool set for whole-genome association and population-based linkage analyses. Am J Hum Genet 81: 559–575.  [DOI](https://doi.org/10.1086/519795) | [PMC free article](/articles/PMC1950838/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17701901/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=PLINK:%20a%20tool%20set%20for%20whole-genome%20association%20and%20population-based%20linkage%20analyses&author=S%20Purcell&author=B%20Neale&author=K%20Todd-Brown&author=L%20Thomas&author=MA%20Ferreira&volume=81&publication_year=2007&pages=559-575&pmid=17701901&doi=10.1086/519795&)

26. Consortium IH (2003) The International HapMap Project. Nature 426: 789–796.  [DOI](https://doi.org/10.1038/nature02168) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14685227/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=The%20International%20HapMap%20Project&author=IH%20Consortium&volume=426&publication_year=2003&pages=789-796&pmid=14685227&doi=10.1038/nature02168&)

27. Taylor JW (1977) Simple estimation of population attributable risk from case-control studies. Am J Epidemiol 106: 260.  [DOI](https://doi.org/10.1093/oxfordjournals.aje.a112461) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/910794/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Epidemiol&title=Simple%20estimation%20of%20population%20attributable%20risk%20from%20case-control%20studies&author=JW%20Taylor&volume=106&publication_year=1977&pages=260&pmid=910794&doi=10.1093/oxfordjournals.aje.a112461&)

28. Consortium TGP (2010) A map of human genome variation from population scale sequencing. Nature 1061: 1061–1073.  [DOI](https://doi.org/10.1038/nature09534) | [PMC free article](/articles/PMC3042601/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20981092/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=A%20map%20of%20human%20genome%20variation%20from%20population%20scale%20sequencing&volume=1061&publication_year=2010&pages=1061-1073&pmid=20981092&doi=10.1038/nature09534&)

29. Pan YX, Xu J, Mahurter L, Xu M, Gilbert AK, et al. (2003) Identification and characterization of two new human mu opioid receptor splice variants, hMOR-1O and hMOR-1X. Biochem Biophys Res Commun 301: 1057–1061.  [DOI](https://doi.org/10.1016/s0006-291x(03)00089-5) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12589820/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem%20Biophys%20Res%20Commun&title=Identification%20and%20characterization%20of%20two%20new%20human%20mu%20opioid%20receptor%20splice%20variants,%20hMOR-1O%20and%20hMOR-1X&author=YX%20Pan&author=J%20Xu&author=L%20Mahurter&author=M%20Xu&author=AK%20Gilbert&volume=301&publication_year=2003&pages=1057-1061&pmid=12589820&doi=10.1016/s0006-291x(03)00089-5&)

30. Huang RS, Duan S, Kistner EO, Bleibel WK, Delaney SM, et al. (2008) Genetic variants contributing to daunorubicin-induced cytotoxicity. Cancer Res 68: 3161–3168.  [DOI](https://doi.org/10.1158/0008-5472.CAN-07-6381) | [PMC free article](/articles/PMC2714371/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18451141/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=Genetic%20variants%20contributing%20to%20daunorubicin-induced%20cytotoxicity&author=RS%20Huang&author=S%20Duan&author=EO%20Kistner&author=WK%20Bleibel&author=SM%20Delaney&volume=68&publication_year=2008&pages=3161-3168&pmid=18451141&doi=10.1158/0008-5472.CAN-07-6381&)

31. Ross CJ, Katzov-Eckert H, Dube MP, Brooks B, Rassekh SR, et al. (2009) Genetic variants in TPMT and COMT are associated with hearing loss in children receiving cisplatin chemotherapy. Nat Genet 41: 1345–1349.  [DOI](https://doi.org/10.1038/ng.478) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19898482/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Genetic%20variants%20in%20TPMT%20and%20COMT%20are%20associated%20with%20hearing%20loss%20in%20children%20receiving%20cisplatin%20chemotherapy&author=CJ%20Ross&author=H%20Katzov-Eckert&author=MP%20Dube&author=B%20Brooks&author=SR%20Rassekh&volume=41&publication_year=2009&pages=1345-1349&pmid=19898482&doi=10.1038/ng.478&)

32. Daly AK, Day CP (2012) Genetic association studies in drug-induced liver injury. Drug Metab Rev 44: 116–126.  [DOI](https://doi.org/10.3109/03602532.2011.605790) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21913872/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Rev&title=Genetic%20association%20studies%20in%20drug-induced%20liver%20injury&author=AK%20Daly&author=CP%20Day&volume=44&publication_year=2012&pages=116-126&pmid=21913872&doi=10.3109/03602532.2011.605790&)

33. Kaab S, Crawford DC, Sinner MF, Behr ER, Kannankeril PJ, et al. (2011) A Large Candidate Gene Survey Identifies the KCNE1 D85N Polymorphism as a Possible Modulator of Drug-Induced Torsades de Pointes. Circ Cardiovasc Genet  [DOI](https://doi.org/10.1161/CIRCGENETICS.111.960930) | [PMC free article](/articles/PMC3288202/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22100668/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circ%20Cardiovasc%20Genet&title=A%20Large%20Candidate%20Gene%20Survey%20Identifies%20the%20KCNE1%20D85N%20Polymorphism%20as%20a%20Possible%20Modulator%20of%20Drug-Induced%20Torsades%20de%20Pointes&author=S%20Kaab&author=DC%20Crawford&author=MF%20Sinner&author=ER%20Behr&author=PJ%20Kannankeril&publication_year=2011&pmid=22100668&doi=10.1161/CIRCGENETICS.111.960930&)

34. McCormack M, Alfirevic A, Bourgeois S, Farrell JJ, Kasperaviciute D, et al. (2011) HLA-A*3101 and carbamazepine-induced hypersensitivity reactions in Europeans. N Engl J Med 364: 1134–1143.  [DOI](https://doi.org/10.1056/NEJMoa1013297) | [PMC free article](/articles/PMC3113609/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21428769/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=HLA-A*3101%20and%20carbamazepine-induced%20hypersensitivity%20reactions%20in%20Europeans&author=M%20McCormack&author=A%20Alfirevic&author=S%20Bourgeois&author=JJ%20Farrell&author=D%20Kasperaviciute&volume=364&publication_year=2011&pages=1134-1143&pmid=21428769&doi=10.1056/NEJMoa1013297&)

35. Chang TJ, Liu PH, Liang YC, Chang YC, Jiang YD, et al. (2011) Genetic predisposition and nongenetic risk factors of thiazolidinedione-related edema in patients with type 2 diabetes. Pharmacogenet Genomics 21: 829–836.  [DOI](https://doi.org/10.1097/FPC.0b013e32834bfff1) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21934636/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Genetic%20predisposition%20and%20nongenetic%20risk%20factors%20of%20thiazolidinedione-related%20edema%20in%20patients%20with%20type%202%20diabetes&author=TJ%20Chang&author=PH%20Liu&author=YC%20Liang&author=YC%20Chang&author=YD%20Jiang&volume=21&publication_year=2011&pages=829-836&pmid=21934636&doi=10.1097/FPC.0b013e32834bfff1&)

36. Ingle JN, Schaid DJ, Goss PE, Liu M, Mushiroda T, et al. (2010) Genome-wide associations and functional genomic studies of musculoskeletal adverse events in women receiving aromatase inhibitors. J Clin Oncol 28: 4674–4682.  [DOI](https://doi.org/10.1200/JCO.2010.28.5064) | [PMC free article](/articles/PMC3020700/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20876420/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Genome-wide%20associations%20and%20functional%20genomic%20studies%20of%20musculoskeletal%20adverse%20events%20in%20women%20receiving%20aromatase%20inhibitors&author=JN%20Ingle&author=DJ%20Schaid&author=PE%20Goss&author=M%20Liu&author=T%20Mushiroda&volume=28&publication_year=2010&pages=4674-4682&pmid=20876420&doi=10.1200/JCO.2010.28.5064&)

37. Offit K, Robson ME (2010) New pharmacogenomic paradigm in breast cancer treatment. J Clin Oncol 28: 4665–4666.  [DOI](https://doi.org/10.1200/JCO.2010.31.2926) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20876427/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=New%20pharmacogenomic%20paradigm%20in%20breast%20cancer%20treatment&author=K%20Offit&author=ME%20Robson&volume=28&publication_year=2010&pages=4665-4666&pmid=20876427&doi=10.1200/JCO.2010.31.2926&)

38. Aihara M (2011) Pharmacogenetics of cutaneous adverse drug reactions. J Dermatol 38: 246–254.  [DOI](https://doi.org/10.1111/j.1346-8138.2010.01196.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21342226/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Dermatol&title=Pharmacogenetics%20of%20cutaneous%20adverse%20drug%20reactions&author=M%20Aihara&volume=38&publication_year=2011&pages=246-254&pmid=21342226&doi=10.1111/j.1346-8138.2010.01196.x&)

39. Garriock HA, Tanowitz M, Kraft JB, Dang VC, Peters EJ, et al. (2010) Association of mu-opioid receptor variants and response to citalopram treatment in major depressive disorder. Am J Psychiatry 167: 565–573.  [DOI](https://doi.org/10.1176/appi.ajp.2009.08081167) | [PMC free article](/articles/PMC2885766/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20194481/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Psychiatry&title=Association%20of%20mu-opioid%20receptor%20variants%20and%20response%20to%20citalopram%20treatment%20in%20major%20depressive%20disorder&author=HA%20Garriock&author=M%20Tanowitz&author=JB%20Kraft&author=VC%20Dang&author=EJ%20Peters&volume=167&publication_year=2010&pages=565-573&pmid=20194481&doi=10.1176/appi.ajp.2009.08081167&)

40. Laugsand EA, Fladvad T, Skorpen F, Maltoni M, Kaasa S, et al. (2011) Clinical and genetic factors associated with nausea and vomiting in cancer patients receiving opioids. Eur J Cancer 47: 1682–1691.  [DOI](https://doi.org/10.1016/j.ejca.2011.04.014) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21570824/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Cancer&title=Clinical%20and%20genetic%20factors%20associated%20with%20nausea%20and%20vomiting%20in%20cancer%20patients%20receiving%20opioids&author=EA%20Laugsand&author=T%20Fladvad&author=F%20Skorpen&author=M%20Maltoni&author=S%20Kaasa&volume=47&publication_year=2011&pages=1682-1691&pmid=21570824&doi=10.1016/j.ejca.2011.04.014&)

41. Waldhoer M, Bartlett SE, Whistler JL (2004) Opioid receptors. Annu Rev Biochem 73: 953–990.  [DOI](https://doi.org/10.1146/annurev.biochem.73.011303.073940) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15189164/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Annu%20Rev%20Biochem&title=Opioid%20receptors&author=M%20Waldhoer&author=SE%20Bartlett&author=JL%20Whistler&volume=73&publication_year=2004&pages=953-990&pmid=15189164&doi=10.1146/annurev.biochem.73.011303.073940&)

42. Pan YX, Xu J, Bolan E, Moskowitz HS, Xu M, et al. (2005) Identification of four novel exon 5 splice variants of the mouse mu-opioid receptor gene: functional consequences of C-terminal splicing. Mol Pharmacol 68: 866–875.  [DOI](https://doi.org/10.1124/mol.105.011858) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15939800/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Pharmacol&title=Identification%20of%20four%20novel%20exon%205%20splice%20variants%20of%20the%20mouse%20mu-opioid%20receptor%20gene:%20functional%20consequences%20of%20C-terminal%20splicing&author=YX%20Pan&author=J%20Xu&author=E%20Bolan&author=HS%20Moskowitz&author=M%20Xu&volume=68&publication_year=2005&pages=866-875&pmid=15939800&doi=10.1124/mol.105.011858&)

43. van Rijn RM, Whistler JL, Waldhoer M (2010) Opioid-receptor-heteromer-specific trafficking and pharmacology. Curr Opin Pharmacol 10: 73–79.  [DOI](https://doi.org/10.1016/j.coph.2009.09.007) | [PMC free article](/articles/PMC2900797/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19846340/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Opin%20Pharmacol&title=Opioid-receptor-heteromer-specific%20trafficking%20and%20pharmacology&author=RM%20van%20Rijn&author=JL%20Whistler&author=M%20Waldhoer&volume=10&publication_year=2010&pages=73-79&pmid=19846340&doi=10.1016/j.coph.2009.09.007&)

44. Daly AK (2010) Genome-wide association studies in pharmacogenomics. Nat Rev Genet 11: 241–246.  [DOI](https://doi.org/10.1038/nrg2751) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20300088/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Genet&title=Genome-wide%20association%20studies%20in%20pharmacogenomics&author=AK%20Daly&volume=11&publication_year=2010&pages=241-246&pmid=20300088&doi=10.1038/nrg2751&)
