# Genome-wide association study identifies a potent locus associated with human opioid sensitivity

## Metadata
**Authors:** D Nishizawa, K Fukuda, S Kasai, J Hasegawa, Y Aoki, A Nishi, N Saita, Y Koukita, M Nagashima, R Katoh, Y Satoh, M Tagami, S Higuchi, H Ujike, N Ozaki, T Inada, N Iwata, I Sora, M Iyo, N Kondo, M-J Won, N Naruse, K Uehara-Aoyama, M Itokawa, M Koga, T Arinami, Y Kaneko, M Hayashida, K Ikeda
**Journal:** Molecular Psychiatry
**Date:** 2012 Nov 27
**DOI:** [10.1038/mp.2012.164](https://doi.org/10.1038/mp.2012.164)
**PMID:** 23183491
**PMCID:** PMC3873034
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3873034/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC3873034/pdf/mp2012164a.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC3873034/pdf/mp2012164a.pdf)

## Abstract

Opioids, such as morphine and fentanyl, are widely used as effective analgesics for the treatment of acute and chronic pain. In addition, the opioid system has a key role in the rewarding effects of morphine, ethanol, cocaine and various other drugs. Although opioid sensitivity is well known to vary widely among individual subjects, several candidate genetic polymorphisms reported so far are not sufficient for fully understanding the wide range of interindividual differences in human opioid sensitivity. By conducting a multistage genome-wide association study (GWAS) in healthy subjects, we found that genetic polymorphisms within a linkage disequilibrium block that spans 2q33.3–2q34 were strongly associated with the requirements for postoperative opioid analgesics after painful cosmetic surgery. The C allele of the best candidate single-nucleotide polymorphism (SNP), rs2952768, was associated with more analgesic requirements, and consistent results were obtained in patients who underwent abdominal surgery. In addition, carriers of the C allele in this SNP exhibited less vulnerability to severe drug dependence in patients with methamphetamine dependence, alcohol dependence, and eating disorders and a lower ‘Reward Dependence' score on a personality questionnaire in healthy subjects. Furthermore, the C/C genotype of this SNP was significantly associated with the elevated expression of a neighboring gene, CREB1. These results show that SNPs in this locus are the most potent genetic factors associated with human opioid sensitivity known to date, affecting both the efficacy of opioid analgesics and liability to severe substance dependence. Our findings provide valuable information for the personalized treatment of pain and drug dependence.

Keywords: analgesia, dependence, opioids, pharmacogenetics

## Introduction

The opioid system has important roles in both antinociception and reward.^1, 2^ Therefore, opioids, such as morphine and fentanyl, are widely used not only as effective analgesics for the treatment of acute and chronic pain but also as abused drugs. The opioid system is also involved in the rewarding effects of morphine,^3^ ethanol,^4^ cocaine,^5^ and various other drugs^6, 7, 8^ or behaviors. However, opioid sensitivity is well known to vary widely among individual subjects,^9^ resulting in differences in the effectiveness of opioid analgesics and vulnerability to dependence on opioids and other drugs or behaviors. Individual differences may be attributable to both genetic and environmental factors,^10^ although the relative influence of each of these factors can be diverse. To date, several candidate genetic polymorphisms have been reported to be associated with opioid sensitivity in human studies.^10, 11, 12, 13, 14^ However, such polymorphisms have not sufficiently explained the wide range of interindividual variance observed in the sensitivity to opioid analgesics. A genome-wide approach has not yet been adopted to explore the best candidates, although this approach has been applied to other pharmacogenomics-related traits. Several genetic polymorphisms have been found to be associated with the sensitivity to pharmacotherapies.^15, 16, 17^ In this study, we sought to comprehensively identify genetic polymorphisms in the human genome that could greatly contribute to individual differences in opioid sensitivity by conducting a genome-wide association study (GWAS) of healthy subjects and further analyses.

## Materials and methods

### Subjects

Enrolled in this multistage GWAS were 355 healthy patients who were scheduled to undergo cosmetic orthognathic surgery for mandibular prognathism at Tokyo Dental College Suidoubashi Hospital. The surgical protocol and subsequent postoperative pain management were fundamentally the same as a previous study^12^ and are detailed in the [Supplementary Information](#sup1).

The subjects recruited in the additional analysis to confirm the association between the rs2952768 single-nucleotide polymorphism (SNP) and postoperative opioid analgesic requirements were 112 patients who underwent major open abdominal surgery at several related hospitals. The surgical protocol and subsequent postoperative pain management were fundamentally the same as a previous study^11, 18^ and are detailed in the [Supplementary Information](#sup1). Enrolled in the study to investigate the contribution of the rs2952768 SNP to the symptoms of drug dependence or related personality traits were 203 patients with methamphetamine (METH) dependence with clinical data that included their multisubstance abuse status, 438 patients with alcohol dependence with clinical data that included the number of drugs used, 228 patients with eating disorders with clinical data including the presence or absence of other psychiatric disorders such as substance dependence and 500 healthy volunteer subjects with personality profile data from the temperament and character inventory (TCI).^19, 20, 21^ To examine the mRNA expression levels of the *METTL21A* (*FAM119A*) and *CREB1* genes, 100 post-mortem human brain specimens, from which DNA and RNA were extracted for experimental use, were obtained from the Stanley Medical Research Institute (SMRI; Bethesda, MD, USA) as samples independent of those in the association study with opioid sensitivity (SMRI samples).

All of the individuals included in the study originated from Japan, with the exception of those from whom the SMRI samples were obtained, whose racial background was mostly European American (see [Supplementary Information](#sup1)).

The study protocol was approved by the Institutional Review Boards at the related hospitals, Tokyo Institute of Psychiatry (currently Tokyo Metropolitan Institute of Medical Science) and the ethics committee of each participating institute of the Japanese Genetics Initiative for Drug Abuse.^22, 23^ All of the subjects provided informed, written consent for the genetic studies. The detailed demographic and clinical data of the subjects are provided in [Supplementary Tables 1, 5–8 and 10](#sup1).

### Genotyping

After total genomic DNA was extracted from whole-blood samples using standard procedures, whole-genome genotyping was performed using the Infinium assay II with an iScan system (Illumina, San Diego, CA, USA) according to the manufacturer's instructions. The data for the whole-genome genotyped samples were analyzed using BeadStudio or GenomeStudio with the Genotyping module v3.3.7 (Illumina) to evaluate the quality of the results. In the data-cleaning process, the samples with a genotype call rate of <0.95 were excluded from further analyses. As a result, one sample was excluded from further analyses. Markers with a genotype call frequency of <0.95 or ‘Cluster sep' (that is, an index of genotype cluster separation) of <0.1 were excluded from the subsequent association study. A total of 295 036 SNP markers survived the filtration process and were used for the GWAS ([Supplementary Figure S1](#sup1)).

For additional genotyping of the rs2952768 and rs2254137 SNPs, the TaqMan allelic discrimination assay (Life Technologies, Carlsbad, CA, USA) was mostly conducted after total genomic DNA was extracted from whole-blood or oral mucosa samples using standard procedures. For samples that were not appropriately genotyped by this assay, direct sequencing was alternatively adopted to genotype the rs2952768 SNP. A total of 112, 203, 438, 228, 500 and 105 DNA samples from patients who underwent major abdominal surgery, patients with METH dependence/psychosis, patients with alcohol dependence, patients with eating disorders, healthy volunteer subjects and SMRI, respectively, were used to genotype the rs2952768 SNP. In addition, a total of 105 DNA samples from the post-mortem specimens for the expression analysis were used to genotype the rs2254137 SNP, although genotyping this SNP for other samples was not conducted because of the strength of the linkage disequilibrium (LD) with the rs2952768 SNP. The genotype distribution of the rs2952768 SNP in patients with METH dependence/psychosis, patients with alcohol dependence and patients with eating disorders is provided in [Supplementary Table 9](#sup1).

### Quantitative PCR procedure

The SMRI RNA samples were treated with DNase I using the RNase-Free DNase Set (Qiagen, Hilden, Germany), and clean-up was then performed using the RNeasy MinElute Cleanup Kit (Qiagen). First-strand complementary DNA for use in the real-time quantitative PCR was synthesized with the SuperScriptIII First-Strand synthesis system for quantitative reverse transcriptase-PCR (Life Technologies) with 100 ng purified total RNA according to the manufacturer's protocol and diluted properly with diethylpyrocarbonate-treated H_2_O before the experiments.

To perform real-time quantitative PCR with a LightCycler 480 (Roche Diagnostics, Basel, Switzerland), TaqMan Gene Expression Assays (Life Technologies) were used as a probe/primer set specified for the *FAM119A* (*METTL21A*) gene and *CREB1* gene and a probe/primer set for the *ACTB* gene, a house-keeping gene that encodes *β*-actin. The expression level of the *FAM119A* (*METTL21A*) gene or *CREB1* gene was normalized to the expression level of the *ACTB* gene for each sample, and relative mRNA expression levels were compared between the genotype subgroups for each gene. The experiments were performed in triplicate (separate experiments) for each sample, and average values were calculated for normalized expression levels.

### Statistical analysis

A three-stage GWAS was conducted for the patients who underwent painful cosmetic surgery to investigate the association between opioid sensitivity and the 295 036 SNPs that passed the quality control criteria in a total of 353 subjects (118, 117 and 118 subjects for the first-, second- and final-stage analyses, respectively). As an index of opioid sensitivity, postoperative patient-controlled analgesia fentanyl use during the first 24-h postoperative period was used because analgesic requirements likely reflect the efficacy of fentanyl in each individual. To explore the association between the SNPs and phenotype, linear regression analyses were conducted in each stage of the analysis, in which postoperative fentanyl use (μg kg^–1^; log transformed) and the genotype data of each SNP were incorporated as dependent and independent variables, respectively. Additive, dominant and recessive genetic models were used for the analyses because of the previously insufficient knowledge about the genetic factors associated with opioid sensitivity. The GWAS procedure is summarized in [Supplementary Figure S1](#sup1). In the first-stage analysis of 118 subjects, the SNPs that showed statistical *P*-values of <0.05 were selected as the candidate SNPs for the second-stage analysis among the 295 036 SNPs. For these SNPs, the second-stage analysis was conducted; again, the SNPs that showed *P*<0.05 were considered potent candidates and selected for further final-stage analysis. In the final stage of the three stages, the *Q*-values of the false discovery rate were calculated to correct for multiple testing in addition to the *P*-values based on previous reports.^24, 25^ The SNPs that showed *Q*<0.05 in the analysis were considered genome-wide significant.

Additional analyses were conducted using the samples of the patients who underwent major abdominal surgery, patients with METH dependence/psychosis, patients with alcohol dependence, patients with eating disorders, healthy volunteers with personality profile data and post-mortem specimens for the expression analysis. For all of the genotype data used in these analyses, the distributions were checked using the *χ*^2^ test, and the absence of significant deviation from the theoretical distribution expected from Hardy–Weinberg equilibrium was confirmed ([Supplementary Table 13](#sup1)).

In the analysis of the patients who underwent major abdominal surgery, the calculated total dose of rescue analgesics administered during the first 24-h postoperative period was used as an index of opioid sensitivity. To explore the association between the SNPs and phenotype, Student's *t*-test or analysis of variance was performed, in which Bonferroni correction for multiple comparisons was used for the *post hoc* tests. For these analyses, postoperative analgesic use (μg kg^–1^; log transformed) and the genotype data of each SNP were incorporated as dependent and independent variables, respectively. *χ*^2^ Tests were performed to investigate the contribution of the SNPs to the vulnerability to the presence of serious symptoms of substance dependence. For the analyses in which the number of subjects in a cell in 2 × 2 contingency tables was <5, Fisher's exact tests were conducted instead of *χ*^2^ tests. In the analysis of healthy volunteers with personality profile data, raw TCI scores were properly processed (see [Supplementary Information](#sup1)). To explore the association between the SNPs and phenotype, linear regression analyses were conducted, in which the endpoint TCI score (log transformed) and genotype data of each SNP were incorporated as dependent and independent variables, respectively. The correction of multiple tests for the analyses of the seven phenotypes was not performed in the present exploratory study. In the analysis of the post-mortem specimens for the expression analysis, the calculated expression level of the *FAM119A* (*METTL21A*) gene or *CREB1* gene normalized to the *ACTB* gene for each sample was used. To explore the association between the SNPs and phenotype, Student's *t*-test or analysis of variance was performed, in which Bonferroni correction for multiple comparisons was used as the *post hoc* test. For these analyses, the relative expression level and genotype data of each SNP were incorporated as dependent and independent variables, respectively.

For all of the statistical analyses described above, SPSS 18.0J for Windows (International Business Machines Corporation, Armonk, NY, USA), gPLINK v. 2.050, PLINK v. 1.07 ([http://pngu.mgh.harvard.edu/purcell/plink/](http://pngu.mgh.harvard.edu/purcell/plink/); accessed 1 March 2012),^26^ and Haploview v. 4.1^27^ were used. The criterion for significance was set at *P*<0.05, with the exception of the GWAS. Statistical corrections for multiple tests, such as Bonferroni adjustments on the multiple parameters analyzed, were not performed in the present exploratory study after the GWAS because it would be too conservative for genetic association studies,^28^ meaning that the likelihood of type II errors is increased by Bonferroni adjustments, and truly important differences would be deemed nonsignificant.^29^

### Additional in silico analysis

Additional *in silico* analyses and the Internet links for the websites of the databases referenced are provided in the [Supplementary Information](#sup1).

## Results

### Identification of a potent locus associated with human opioid sensitivity by GWAS in subjects who underwent painful cosmetic surgery

We first explored the association between genetic variations and opioid sensitivity in a total of 353 healthy subjects who were scheduled to undergo cosmetic orthognathic surgery (mandibular sagittal split ramus osteotomy) for mandibular prognathism that involved the administration of opioid analgesics ([Supplementary Table S1](#sup1)), in which the surgical procedure was uniform and thus the invasiveness and resultant pain would be regarded as homogeneous among the subjects. A GWAS was conducted as a consecutive three-stage analysis to identify potent SNPs associated with the requirements for an opioid analgesic, fentanyl (μg kg^–1^), during the 24-h postoperative period ([Supplementary Figure S1](#sup1)). Consequently, 9, 12 and 10 SNPs were selected as the top candidates for additive, dominant and recessive models for each minor allele, respectively, after the final stage ([Supplementary Tables S2–4](#sup1)). Among these, several SNPs mapped to 2q33.3–2q34 showed significant associations after the final stage with 24-h postoperative fentanyl requirements in the additive and recessive models (additive model: combined *β*=0.293, nominal *P*=8.044 × 10^−7^; recessive model: combined *β*=0.553, nominal *P*=9.382 × 10^−7^; [Supplementary Tables S2–4](#sup1)). The observed *P*-values of these SNPs, calculated as −log10 (*P*-value), obviously deviated from the expected values from the null hypothesis of uniform distribution in the quantile–quantile plot for the entire sample ([Supplementary Figure S2](#sup1)). The genes located in this region were found to include *METTL21A* (*FAM119A*) and *CREB1*, encoding methyltransferase like 21A and cyclic adenosine 3′,5′-monophosphate (cAMP) responsive element-binding protein 1 (CREB1), respectively.

For further fine mapping of this region, we used the remainder of the genotyped data and imputed the genotype data from the SNPs that spanned 300 kbp, including both the *METTL21A* and *CREB1* genes ([Figure 1a](#fig1)), and analyzed the association. We then identified several other SNPs around the best candidate SNP, rs2952768, with nominal *P*-values from 1 × 10^−6^ to 1 × 10^−4^ ([Figure 1b](#fig1)). After the LD analysis, all of these SNPs were found to be included in an LD block that spanned approximately 134 kbp and were in strong LD (*D*′=1, *r*^2^>0.8) with rs2952768 ([Figure 1c](#fig1)). Total fentanyl use during the 24-h postoperative period was 1.080±0.053, 1.146±0.049 and 1.666±0.083 μg kg^–1^ (log transformed; mean± s.e.m.) in subjects with the T/T, T/C and C/C genotypes, respectively ([Figure 2a](#fig2)).

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/452d/3873034/4e3e30e3c387/mp2012164f1.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3873034_mp2012164f1.jpg)

Candidate locus possibly associated with human opioid sensitivity. (a) Illustration of the genes in the genomic region from position 208 000 000 to 208 300 000 on chromosome 2 in the HapMap database (http://hapmap.ncbi.nlm.nih.gov/index.html.ja; accessed 1 March 2012). (b) Fine mapping of the candidate region after the imputation-based association analysis. The circle, square and triangle plots represent the results from the additive, dominant and recessive models, respectively. The area between the dotted vertical lines represents the genomic position from 208 070 000 to 208 240 000 on chromosome 2. (c) State of linkage disequilibrium (LD) between the SNPs in the genomic position from 208 070 000 to 208 240 000 on chromosome 2, based on the genotype data of the subjects who are derived from the Japanese population and underwent cosmetic orthognathic surgery. Numbers in squares in which two SNPs face represent the percentage of the r2 values calculated from the genotype data of the SNPs. Squares with asterisks represent r2=1. Only the values >0.70 are highlighted.

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/452d/3873034/684cac445596/mp2012164f2.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3873034_mp2012164f2.jpg)

Association analysis between opioid analgesic requirements and the rs2952768 SNP. (a) Total dose of analgesics administered per body weight (μg kg–1; log transformed) during the 24-h postoperative period after cosmetic orthognathic surgery (mandibular sagittal split ramus osteotomy). (b) Total dose of analgesics administered per body weight (μg kg–1; log transformed) during the 24-h postoperative period after major open abdominal surgery. *Q<0.05, greater dose of analgesic administered in the C/C genotype compared with the T/C and T/T genotypes with genome-wide significance; †P<0.05, greater dose of analgesic administered in the C/C genotype compared with the T/C and T/T genotypes with nominal significance. The data are expressed as mean±s.e.m.

### Association of rs2952768 SNP with sensitivity to opioid analgesics in patients who underwent major open abdominal surgery

To examine whether the SNPs identified in our GWAS generally affect individual differences in opioid sensitivity, we attempted to confirm the association between the rs2952768 SNP and postoperative opioid requirements in another cohort that underwent a different surgical procedure. The subjects recruited in this study were 112 patients who underwent major open abdominal surgery under combined general and epidural anesthesia ([Supplementary Table S5](#sup1)),^18^ mostly gastrectomy for gastric cancer and colectomy for colorectal cancer, which involves different modes of invasion from the orthognathic surgery and might cause different pain modalities. Appropriate doses of analgesics, mainly opioids such as morphine, buprenorphine, pentazocine and pethidine, were administered as rescue analgesics at the discretion of the surgeons whenever the patients complained of significant postoperative pain during the postoperative period, and the total dose administered was estimated for the association analysis. As a result, a significant difference in postoperative analgesic requirements was found between the subjects with the combined T/T and T/C genotype and subjects with the C/C genotype in the rs2952768 SNP. Interestingly, the subjects with the C/C genotype required significantly more analgesics than the subjects with the combined T/T and T/C genotype in the rs2952768 SNP (*t*_110_=−2.340, *P*=0.021), a pattern similar to the one observed in the subjects who underwent cosmetic orthognathic surgery. Total analgesic use, equipotent with systemic fentanyl, during the 24-h postoperative period was 0.359±0.073, 0.397±0.068 and 0.741±0.169 μg kg^–1^ (log transformed; mean±s.e.m.) in the subjects with the T/T, T/C and C/C genotypes, respectively ([Figure 2b](#fig2)).

### Association of rs2952768 SNP with severity of drug dependence in patients in several cohorts

The results suggested that the subjects with the C/C genotype in the rs2952768 SNP required more analgesics than the subjects with the other genotypes, attributable to the decreased effectiveness of opioid analgesics in both cohorts. Given the fact that the opioid system is involved in both rewarding and analgesic effects, one could assume that decreased opioid sensitivity may reflect the decreased rewarding effects of various drugs or behaviors and less liability to serious dependence. To test this hypothesis, we investigated the contribution of the rs2952768 SNP to the vulnerability to substance dependence in additional subjects with METH dependence, alcohol dependence and eating disorders ([Supplementary Tables S6–8](#sup1)). In the initial case–control analyses, no associations in the genotypic and allelic distribution of this SNP were found between the subjects with psychiatric disorders and corresponding control subjects ([Supplementary Table S9](#sup1)). However, a significant difference in the genotypic distribution was found between the absent and present subgroups of polydrug use among the patients with METH dependence (*χ*^2^=3.979, *P*=0.046). Indeed, fewer polydrug abusers carried the C/C genotype compared with monodrug users ([Table 1](#tbl1)). A similar result was found in the patients with alcohol dependence. A significant difference in the genotypic distribution was found between the absent and present subgroups of drug use among patients with alcohol dependence (*χ*^2^=3.860, *P*=0.049, and *χ*^2^=3.039, *P*=0.097, in the additive and recessive models, respectively), and a lower proportion of drug abusers carried the C allele or C/C genotype compared with the alcoholics without drug abuse ([Table 1](#tbl1)). Furthermore, a significant difference in allelic distribution was found between the absent and present subgroups of comorbid dependence among the patients with eating disorders (*χ*^2^=3.985, *P*=0.046, and *χ*^2^=4.488, *P*=0.034, in the additive and dominant models, respectively), and a lower proportion of patients with drug dependence carried the C allele compared with the patients without drug dependence ([Table 1](#tbl1)). Although nonsignificant, likely because of the small sample size, a marginal difference was observed in allelic distribution between the absent (*χ*^2^=3.780, *P*=0.052) and present subgroups with a comorbid alcoholic state among the patients with eating disorders, and a lower proportion of patients with an alcoholic state carried the C allele compared with the patients without an alcoholic state ([Table 1](#tbl1)). Altogether, these results showed that carriers of the C allele among the patients with psychiatric disorders, especially the C/C genotype, tended not to abuse polydrugs and not have comorbid alcohol or drug dependence. Although the association was nonsignificant in the recessive model for the C allele in all of the analyses, possibly because of a lack of statistical power caused by the limited sample size, the present results suggest that carriers of the C allele in this SNP have less inclination to abuse drugs, consistent with our assumption that various drugs of abuse have decreased rewarding effects in subjects with decreased opioid sensitivity, making these subjects less liable to the expression of symptoms of serious dependence.

### Table 1. Distribution of genotypes of the rs2952768 polymorphism and odds ratio between the patient subgroups based on clinical status.

|   | n | Genotype |  |  | Allele frequency |  | χ2 |  |  |  | OR (95% CI) |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
|   |   | T/T | T/C | C/C | T | C | P |  |  |  | C allele | C/C+T/C genotype | C/C genotype |
|   |   |   |   |   |   |   | Genotype | Allele | Dominant | Recessive | (vs T) | (vs T/T) | (vs T/T+T/C) |
| Polydrug use |  |  |  |  |  |  |  |  |  |  |  |  |  |
| In METH dependence/psychosis patients |  |  |  |  |  |  |  |  |  |  |  |  |  |
| Absence | 53 | 19 | 22 | 12 | 0.556 | 0.444 | 3.979 | 2.262 | 0.450 | 3.979 | 0.71 | 0.80 | 0.44 |
| Presence | 141 | 58 | 67 | 16 | 0.644 | 0.356 | 0.137 | 0.133 | 0.503 | 0.046* | (0.45–1.11) | (0.42−1.54) | (0.19–1.00) |
|   |   |   |   |   |   |   |   |   |   |   |   |   |   |
| Drug use |  |  |  |  |  |  |  |  |  |  |  |  |  |
| In alcohol dependence patients |  |  |  |  |  |  |  |  |  |  |  |  |  |
| Absence | 391 | 172 | 166 | 53 | 0.652 | 0.348 | 3.908 | 3.860 | 2.180 | 3.039 | 0.61 | 0.63 | 0.30 |
| Presence | 45 | 25 | 18 | 2 | 0.756 | 0.244 | 0.142 | 0.049* | 0.140 | 0.097§† | (0.37–1.00) | (0.34–1.17) | (0.07–1.26) |
|   |   |   |   |   |   |   |   |   |   |   |   |   |   |
| Drug dependence |  |  |  |  |  |  |  |  |  |  |  |  |  |
| In eating disorder patients |  |  |  |  |  |  |  |  |  |  |  |  |  |
| Absence | 200 | 85 | 93 | 22 | 0.658 | 0.343 | 4.552 | 3.985 | 4.488 | 0.793 | 0.45 | 0.37 | 0.40 |
| Presence | 21 | 14 | 6 | 1 | 0.810 | 0.190 | 0.103 | 0.046* | 0.034* | 0.705§ | (0.20–1.00) | (0.14–0.96) | (0.05–3.16) |
|   |   |   |   |   |   |   |   |   |   |   |   |   |   |
| Alcoholic state |  |  |  |  |  |  |  |  |  |  |  |  |  |
| In eating disorder patients |  |  |  |  |  |  |  |  |  |  |  |  |  |
| Absence | 151 | 61 | 72 | 18 | 0.642 | 0.358 | 3.995 | 3.780 | 3.731 | 1.171 | 0.65 | 0.57 | 0.57 |
| Presence | 70 | 38 | 27 | 5 | 0.736 | 0.264 | 0.136 | 0.052† | 0.053† | 0.279 | (0.41–1.01) | (0.32–1.01) | (0.20–1.60) |

Table 1 Caption: Abbreviations: METH, methamphetamine; n, the number of samples; OR, odds ratio; 95% CI, 95% confidence interval. *P<0.05, †0.05⩽P<0.1, §P-value from Fisher's exact test was presented instead of that from χ2 test.

### Association of rs2952768 SNP with ‘reward dependence' score on a personality questionnaire in healthy subjects

Another interest is whether this SNP also affects personality traits related to the reward system in healthy people. To address this issue, we investigated the association between this SNP and data from the TCI, a personality profiling questionnaire, in healthy volunteers ([Supplementary Table S10](#sup1)). Intriguingly, among the seven dimensions of the TCI, a significant association was found only for reward dependence (RD) (novelty seeking: *β*=−0.009055, *P*=0.2995; harm avoidance: *β*=0.004317, *P*=0.6819; RD: *β*=−0.0175, *P*=0.03265; persistence: *β*=−0.007554, *P*=0.6295; self-directedness: *β*=−0.01274, *P*=0.1709; cooperativeness: *β*=−0.004122, *P*=0.5302; self-transcendence: *β*=2.68E−06, *P*=0.9998; [Supplementary Figure S3](#sup1)). The RD value decreased as the copy number of the carried C allele increased among the subjects. The RD value on the TCI was previously shown to be positively correlated with activity of the caudate head,^30^ which has been shown to be associated with reward processing.^31^ Although a future confirmatory study is needed, the present data suggest the possibility that this SNP is one of the predisposing factors that partially contribute to the developmental differentiation of personality traits related to RD.

### Association of rs2952768 SNP with mRNA expression level of the CREB1 gene

Considering the fact that the rs2952768 and all of the neighboring SNPs that showed similarly strong associations with opioid sensitivity were included in the LD block that comprised the *METTL21A* (*FAM119A*) and *CREB1* genes, a subsequent issue is the impact of these SNPs on the function of these genes. Our database search estimated that several of the SNPs could putatively cause functional alterations ([Supplementary Table S11](#sup1)). To pursue this issue, we examined the mRNA expression levels of these genes using real-time quantitative PCR with RNA samples extracted from post-mortem subject specimens and compared the mRNA expression levels between the genotype subgroups for the rs2952768 SNP, which were determined by genotyping the DNA samples extracted from the corresponding subjects. Although no significant association was found in the relative mRNA expression level of the *METTL21A* (*FAM119A*) gene between the genotype subgroups (*F*_2, 97_=0.372, *P*=0.690), a significant association was found in the relative mRNA expression level of the *CREB1* gene between the combined T/T and T/C genotype subgroup and C/C subgroup ([Figures 3a and b](#fig3)). These results showed that the C/C genotype of this SNP was significantly associated with elevated *CREB1* mRNA expression (*t*_98_=−2.561, *P*=0.012).

### Figure 3.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/452d/3873034/861a3dffcc63/mp2012164f3.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3873034_mp2012164f3.jpg)

Relative mRNA expression level of the candidate genes between each genotype subgroup of the rs2952768 SNP in post-mortem brains. (a) Results for the METTL21A gene. (b) Results for the CREB1 gene. NS, no significant association between relative mRNA expression and genotype subgroup (P⩾0.05); *P<0.05, greater level of mRNA expression in the C/C genotype compared with the T/C and T/T genotypes with nominal significance. The data are expressed as mean±s.e.m.

## Discussion

By conducting a multistage GWAS for the first time in healthy subjects who were treated with opioid analgesics, we identified a potent locus for opioid sensitivity that encompasses an LD block that includes the most significant SNP, rs2952768. The C allele of this SNP, especially homozygotes of the C allele, was associated with more analgesic requirements, suggesting the possibility to classify the patients into groups of high responders and low responders or non-responders to the given opioid, which would presumably correspond to the T/T or T/C and C/C genotype groups, respectively. Surprisingly, this SNP was also found to be significantly associated with postoperative analgesic requirements in subjects who also received opioids but underwent different surgical procedures. Moreover, the C allele with likely less opioid sensitivity in this SNP was associated with a lower risk for serious symptoms of substance dependence and a lower RD score on the TCI in healthy subjects. These results show that SNPs in this locus are the most potent genetic factors associated with human opioid sensitivity known to date, affecting both the efficacy of opioid analgesics and liability to severe substance dependence.

Opioids exert their effects by binding to opioid receptors (that is, G-protein-coupled receptors) and triggering signaling transmission to several downstream effectors, including inhibition of adenylyl cyclase, activation of G-protein-activated inwardly rectifying potassium channels, and inhibition of voltage-gated Ca^2+^ channels.^32, 33, 34^ Inhibition of adenylyl cyclase inhibits the production of cAMP, thus decreasing the active form of protein kinase A, phosphorylating CREB, and decreasing gene expression in the nucleus related to the action of analgesia and reward.^35, 36^ Indeed, the administration of cAMP intracerebrally or intravenously antagonized morphine analgesia in nontolerant and tolerant mice.^37^ Moreover, all of the major behavioral effects of morphine, including analgesia, tolerance, reward, and physical dependence and withdrawal symptoms, were attenuated in mice that lacked adenylyl cyclase 5.^38^ Meanwhile, increased CREB function in the nucleus accumbens decreased the sensitivity to the rewarding effects of morphine and cocaine in animals, whereas decreased CREB function had the opposite effect.^35, 36, 39^ Although no change in opioid-binding sites or morphine-induced analgesia was observed in CREBαδ-deficient mice,^40^ alterations in dose-dependent morphine-induced reward were also reported in CREBαδ-deficient mice.^41^ Altogether, these reports indicate the involvement of CREB and the cAMP pathway in the analgesic and rewarding effects of opioids. Higher mRNA expression levels of the *CREB1* gene in subjects with the C/C genotype in the rs2952768 SNP identified in our GWAS may indicate elevated CREB function and decreased sensitivity to the rewarding effects of opioids, resulting in greater postoperative opioid analgesic requirements and less vulnerability to dependence on other drugs.

The best candidate SNP, rs2952768, is located closer to the *METTL21A* (*FAM119A*) gene than to the *CREB1* gene on chromosome 2 ([Figures 1a and b](#fig1)). However, no significant association was found between rs2952768 and *METTL21A* mRNA expression, precluding the attribution of phenotypic alterations related to this SNP to the expression levels of the *METTL21A* gene. Although the precise functions of METTL21A are poorly understood to date, a representative METTL, METTL11A, reportedly exhibited catalytic activity as a histone methyltransferase^42^ and chronic morphine treatment exhibited the acetylation and trimethylation of histones.^43^ Although future studies are imperative, the action of opioids might be partially modulated by histone methylation via METTL functions.

With regard to our initial GWAS conducted as a consecutive three-stage analysis, the lowest combined *P*-value for the entire sample was *P*=8.044 × 10^−7^ ([Supplementary Table S2](#sup1)), which would have been deemed genome-wide nonsignificant if only a single-stage analysis was used to calculate conventional Bonferroni- or false discovery rate-corrected *P*-values for the total samples to determine statistical significance. However, ‘significant' results obtained as conventionally corrected *P*-values will not always represent true associations, meaning that the results may not be necessarily replicated in other studies, and vice versa. For example, data from the National Human Genome Research Institute GWAS catalog (as of 31 January 2009), show 1321 entries of discovered associations with a *P*-value of <10^–5^, but only 550 of these entries have a *P*-value of <5 × 10^–8^,^44^ which is a conventionally corrected conservative threshold for declaring a significant association in a GWAS.^45, 46^ In both cases, truly potent candidate SNPs may be included in the outcome of the studies. Furthermore, GWASs in pharmacogenomics, such as this study, would tend not to yield ‘significant' results obtained as conventionally corrected *P*-values compared with complex-disease GWASs^15^ for several reasons. Among at least 16 different GWASs on drug response since the first was published in late 2007, less than half have shown genome-wide significance, although some potentially interesting associations that come close to significance have been detected in several of the studies in this category.^15^ Altogether, these reports suggest that conventionally corrected *P*-values for the combined samples are not the only criteria to find true associations between SNPs and the phenotypes examined. The SNP we found, rs2952768, appears to be a promising SNP that is associated not only with opioid analgesic sensitivity in two independent surgical operations but also with several dependence-related traits in other subjects, prompting us to consider this SNP as the best candidate SNP known to date that is truly associated with human opioid sensitivity.

Our compelling results suggest the possibility that the association observed in this study can be robustly generalized to various clinical and nonclinical scenarios, although this study is rather exploratory, and independent confirmation of the findings will be required in subsequent studies before various forms of practical clinical utilization of the prediction of opioid sensitivity based on this SNP can be applied. In conclusion, although the underlying mechanisms remain to be fully elucidated in future studies, our findings provide a novel step toward understanding individual differences in opioid sensitivity and stimulating future studies that can open new avenues for the personalized treatment of pain and drug dependence.

## Acknowledgments

We acknowledge Dr Keiji Tanaka for critically reading the manuscript and Mr Michael Arends for his assistance with editing the manuscript. We are grateful to the volunteers for their participation in this study and the anesthesiologists, surgeons and psychiatrists at related hospitals for collecting clinical data. We thank the Stanley Medical Research Institute Brain Collection for donating specimens. This work was supported by grants from the MEXT of Japan (20390162, 23390377), MHLW of Japan (H21-3jigan-ippan-011, H22-Iyaku -015) and Smoking Research Foundation.

**Author contributions**

Conception and design of the experiments: DN, SK and KI. Performance of the experiments: DN, SK, JH and AN. Analysis of the data: DN and YA Contribution of reagents/materials/analysis tools: DN, JH, AN, MK and TA. Writing of the paper: DN and KI. Collection of clinical data and DNA: KF, NS, Y Koukita, MN, RK, YS, MT, SH, HU, NO, TI, NI, IS, M Iyo, NK, MW, NN, KU, M Itokawa and MH. Support of the collection of clinical data and DNA: Y Kaneko.

The authors declare no conflict of interest.

## Footnotes

## Supplementary Material

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Sora I, Takahashi N, Funada M, Ujike H, Revay RS, Donovan DM, et al. Opiate receptor knockout mice define mu receptor roles in endogenous nociceptive responses and morphine-induced analgesia. Proc Natl Acad Sci USA. 1997;94:1544–1549. doi: 10.1073/pnas.94.4.1544.  [DOI](https://doi.org/10.1073/pnas.94.4.1544) | [PMC free article](/articles/PMC19828/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9037090/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20USA&title=Opiate%20receptor%20knockout%20mice%20define%20mu%20receptor%20roles%20in%20endogenous%20nociceptive%20responses%20and%20morphine-induced%20analgesia&author=I%20Sora&author=N%20Takahashi&author=M%20Funada&author=H%20Ujike&author=RS%20Revay&volume=94&publication_year=1997&pages=1544-1549&pmid=9037090&doi=10.1073/pnas.94.4.1544&)

2. Leknes S, Tracey I. A common neurobiology for pain and pleasure. Nat Rev Neurosci. 2008;9:314–320. doi: 10.1038/nrn2333.  [DOI](https://doi.org/10.1038/nrn2333) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18354400/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Neurosci&title=A%20common%20neurobiology%20for%20pain%20and%20pleasure&author=S%20Leknes&author=I%20Tracey&volume=9&publication_year=2008&pages=314-320&pmid=18354400&doi=10.1038/nrn2333&)

3. Sora I, Elmer G, Funada M, Pieper J, Li XF, Hall FS, et al. μ Opiate receptor gene dose effects on different morphine actions: evidence for differential in vivo mu receptor reserve. Neuropsychopharmacology. 2001;25:41–54. doi: 10.1016/S0893-133X(00)00252-9.  [DOI](https://doi.org/10.1016/S0893-133X(00)00252-9) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11377918/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neuropsychopharmacology&title=%CE%BC%20Opiate%20receptor%20gene%20dose%20effects%20on%20different%20morphine%20actions:%20evidence%20for%20differential%20in%20vivo%20mu%20receptor%20reserve&author=I%20Sora&author=G%20Elmer&author=M%20Funada&author=J%20Pieper&author=XF%20Li&volume=25&publication_year=2001&pages=41-54&pmid=11377918&doi=10.1016/S0893-133X(00)00252-9&)

4. Hall FS, Sora I, Uhl GR. Ethanol consumption and reward are decreased in μ-opiate receptor knockout mice. Psychopharmacology (Berl) 2001;154:43–49. doi: 10.1007/s002130000622.  [DOI](https://doi.org/10.1007/s002130000622) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11292005/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Psychopharmacology%20(Berl)&title=Ethanol%20consumption%20and%20reward%20are%20decreased%20in%20%CE%BC-opiate%20receptor%20knockout%20mice&author=FS%20Hall&author=I%20Sora&author=GR%20Uhl&volume=154&publication_year=2001&pages=43-49&pmid=11292005&doi=10.1007/s002130000622&)

5. Hall FS, Goeb M, Li XF, Sora I, Uhl GR. μ-Opioid receptor knockout mice display reduced cocaine conditioned place preference but enhanced sensitization of cocaine-induced locomotion. Brain Res Mol Brain Res. 2004;121:123–130. doi: 10.1016/j.molbrainres.2003.10.024.  [DOI](https://doi.org/10.1016/j.molbrainres.2003.10.024) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14969743/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Brain%20Res%20Mol%20Brain%20Res&title=%CE%BC-Opioid%20receptor%20knockout%20mice%20display%20reduced%20cocaine%20conditioned%20place%20preference%20but%20enhanced%20sensitization%20of%20cocaine-induced%20locomotion&author=FS%20Hall&author=M%20Goeb&author=XF%20Li&author=I%20Sora&author=GR%20Uhl&volume=121&publication_year=2004&pages=123-130&pmid=14969743&doi=10.1016/j.molbrainres.2003.10.024&)

6. Contarino A, Picetti R, Matthes HW, Koob GF, Kieffer BL, Gold LH. Lack of reward and locomotor stimulation induced by heroin in mu-opioid receptor-deficient mice. Eur J Pharmacol. 2002;446:103–109. doi: 10.1016/s0014-2999(02)01812-5.  [DOI](https://doi.org/10.1016/s0014-2999(02)01812-5) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12098591/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Pharmacol&title=Lack%20of%20reward%20and%20locomotor%20stimulation%20induced%20by%20heroin%20in%20mu-opioid%20receptor-deficient%20mice&author=A%20Contarino&author=R%20Picetti&author=HW%20Matthes&author=GF%20Koob&author=BL%20Kieffer&volume=446&publication_year=2002&pages=103-109&pmid=12098591&doi=10.1016/s0014-2999(02)01812-5&)

7. Berrendero F, Kieffer BL, Maldonado R. Attenuation of nicotine-induced antinociception, rewarding effects, and dependence in mu-opioid receptor knock-out mice. J Neurosci. 2002;22:10935–10940. doi: 10.1523/JNEUROSCI.22-24-10935.2002.  [DOI](https://doi.org/10.1523/JNEUROSCI.22-24-10935.2002) | [PMC free article](/articles/PMC6758457/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12486188/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Neurosci&title=Attenuation%20of%20nicotine-induced%20antinociception,%20rewarding%20effects,%20and%20dependence%20in%20mu-opioid%20receptor%20knock-out%20mice&author=F%20Berrendero&author=BL%20Kieffer&author=R%20Maldonado&volume=22&publication_year=2002&pages=10935-10940&pmid=12486188&doi=10.1523/JNEUROSCI.22-24-10935.2002&)

8. Shen X, Purser C, Tien LT, Chiu CT, Paul IA, Baker R, et al. μ-Opioid receptor knockout mice are insensitive to methamphetamine-induced behavioral sensitization. J Neurosci Res. 2010;88:2294–2302. doi: 10.1002/jnr.22386.  [DOI](https://doi.org/10.1002/jnr.22386) | [PMC free article](/articles/PMC3086557/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20209629/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Neurosci%20Res&title=%CE%BC-Opioid%20receptor%20knockout%20mice%20are%20insensitive%20to%20methamphetamine-induced%20behavioral%20sensitization&author=X%20Shen&author=C%20Purser&author=LT%20Tien&author=CT%20Chiu&author=IA%20Paul&volume=88&publication_year=2010&pages=2294-2302&pmid=20209629&doi=10.1002/jnr.22386&)

9. Ikeda K, Ide S, Han W, Hayashida M, Uhl GR, Sora I. How individual sensitivity to opiates can be predicted by gene analyses. Trends Pharmacol Sci. 2005;26:311–317. doi: 10.1016/j.tips.2005.04.001.  [DOI](https://doi.org/10.1016/j.tips.2005.04.001) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15925706/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Trends%20Pharmacol%20Sci&title=How%20individual%20sensitivity%20to%20opiates%20can%20be%20predicted%20by%20gene%20analyses&author=K%20Ikeda&author=S%20Ide&author=W%20Han&author=M%20Hayashida&author=GR%20Uhl&volume=26&publication_year=2005&pages=311-317&pmid=15925706&doi=10.1016/j.tips.2005.04.001&)

10. Coulbault L, Beaussier M, Verstuyft C, Weickmans H, Dubert L, Tregouet D, et al. Environmental and genetic factors associated with morphine response in the postoperative period. Clin Pharmacol Ther. 2006;79:316–324. doi: 10.1016/j.clpt.2006.01.007.  [DOI](https://doi.org/10.1016/j.clpt.2006.01.007) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16580900/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Environmental%20and%20genetic%20factors%20associated%20with%20morphine%20response%20in%20the%20postoperative%20period&author=L%20Coulbault&author=M%20Beaussier&author=C%20Verstuyft&author=H%20Weickmans&author=L%20Dubert&volume=79&publication_year=2006&pages=316-324&pmid=16580900&doi=10.1016/j.clpt.2006.01.007&)

11. Nishizawa D, Nagashima M, Katoh R, Satoh Y, Tagami M, Kasai S, et al. Association between KCNJ6 (GIRK2) gene polymorphisms and postoperative analgesic requirements after major abdominal surgery. PLoS One. 2009;4:e7060. doi: 10.1371/journal.pone.0007060.  [DOI](https://doi.org/10.1371/journal.pone.0007060) | [PMC free article](/articles/PMC2738941/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19756153/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=Association%20between%20KCNJ6%20(GIRK2)%20gene%20polymorphisms%20and%20postoperative%20analgesic%20requirements%20after%20major%20abdominal%20surgery&author=D%20Nishizawa&author=M%20Nagashima&author=R%20Katoh&author=Y%20Satoh&author=M%20Tagami&volume=4&publication_year=2009&pages=e7060&pmid=19756153&doi=10.1371/journal.pone.0007060&)

12. Fukuda K, Hayashida M, Ide S, Saita N, Kokita Y, Kasai S, et al. Association between OPRM1 gene polymorphisms and fentanyl sensitivity in patients undergoing painful cosmetic surgery. Pain. 2009;147:194–201. doi: 10.1016/j.pain.2009.09.004.  [DOI](https://doi.org/10.1016/j.pain.2009.09.004) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19783098/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pain&title=Association%20between%20OPRM1%20gene%20polymorphisms%20and%20fentanyl%20sensitivity%20in%20patients%20undergoing%20painful%20cosmetic%20surgery&author=K%20Fukuda&author=M%20Hayashida&author=S%20Ide&author=N%20Saita&author=Y%20Kokita&volume=147&publication_year=2009&pages=194-201&pmid=19783098&doi=10.1016/j.pain.2009.09.004&)

13. Candiotti KA, Yang Z, Morris R, Yang J, Crescimone NA, Sanchez GC, et al. Polymorphism in the interleukin-1 receptor antagonist gene is associated with serum interleukin-1 receptor antagonist concentrations and postoperative opioid consumption. Anesthesiology. 2011;114:1162–1168. doi: 10.1097/ALN.0b013e318216e9cb.  [DOI](https://doi.org/10.1097/ALN.0b013e318216e9cb) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21455057/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Anesthesiology&title=Polymorphism%20in%20the%20interleukin-1%20receptor%20antagonist%20gene%20is%20associated%20with%20serum%20interleukin-1%20receptor%20antagonist%20concentrations%20and%20postoperative%20opioid%20consumption&author=KA%20Candiotti&author=Z%20Yang&author=R%20Morris&author=J%20Yang&author=NA%20Crescimone&volume=114&publication_year=2011&pages=1162-1168&pmid=21455057&doi=10.1097/ALN.0b013e318216e9cb&)

14. Kumar D, Deb I, Chakraborty J, Mukhopadhyay S, Das S. A polymorphism of the CREB binding protein (CREBBP) gene is a risk factor for addiction. Brain Res. 2011;1406:59–64. doi: 10.1016/j.brainres.2011.05.048.  [DOI](https://doi.org/10.1016/j.brainres.2011.05.048) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21752352/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Brain%20Res&title=A%20polymorphism%20of%20the%20CREB%20binding%20protein%20(CREBBP)%20gene%20is%20a%20risk%20factor%20for%20addiction&author=D%20Kumar&author=I%20Deb&author=J%20Chakraborty&author=S%20Mukhopadhyay&author=S%20Das&volume=1406&publication_year=2011&pages=59-64&pmid=21752352&doi=10.1016/j.brainres.2011.05.048&)

15. Daly AK. Genome-wide association studies in pharmacogenomics. Nat Rev Genet. 2010;11:241–246. doi: 10.1038/nrg2751.  [DOI](https://doi.org/10.1038/nrg2751) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20300088/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Genet&title=Genome-wide%20association%20studies%20in%20pharmacogenomics&author=AK%20Daly&volume=11&publication_year=2010&pages=241-246&pmid=20300088&doi=10.1038/nrg2751&)

16. Wang L, McLeod HL, Weinshilboum RM. Genomics and drug response. N Engl J Med. 2011;364:1144–1153. doi: 10.1056/NEJMra1010600.  [DOI](https://doi.org/10.1056/NEJMra1010600) | [PMC free article](/articles/PMC3184612/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21428770/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Genomics%20and%20drug%20response&author=L%20Wang&author=HL%20McLeod&author=RM%20Weinshilboum&volume=364&publication_year=2011&pages=1144-1153&pmid=21428770&doi=10.1056/NEJMra1010600&)

17. Feero WG, Guttmacher AE, Collins FS. Genomic medicine—an updated primer. N Engl J Med. 2010;362:2001–2011. doi: 10.1056/NEJMra0907175.  [DOI](https://doi.org/10.1056/NEJMra0907175) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20505179/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Genomic%20medicine%E2%80%94an%20updated%20primer&author=WG%20Feero&author=AE%20Guttmacher&author=FS%20Collins&volume=362&publication_year=2010&pages=2001-2011&pmid=20505179&doi=10.1056/NEJMra0907175&)

18. Hayashida M, Nagashima M, Satoh Y, Katoh R, Tagami M, Ide S, et al. Analgesic requirements after major abdominal surgery are associated with OPRM1 gene polymorphism genotype and haplotype. Pharmacogenomics. 2008;9:1605–1616. doi: 10.2217/14622416.9.11.1605.  [DOI](https://doi.org/10.2217/14622416.9.11.1605) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19018716/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Analgesic%20requirements%20after%20major%20abdominal%20surgery%20are%20associated%20with%20OPRM1%20gene%20polymorphism%20genotype%20and%20haplotype&author=M%20Hayashida&author=M%20Nagashima&author=Y%20Satoh&author=R%20Katoh&author=M%20Tagami&volume=9&publication_year=2008&pages=1605-1616&pmid=19018716&doi=10.2217/14622416.9.11.1605&)

19. Cloninger CR. A systematic method for clinical description and classification of personality variants. A proposal. Arch Gen Psychiatry. 1987;44:573–588. doi: 10.1001/archpsyc.1987.01800180093014.  [DOI](https://doi.org/10.1001/archpsyc.1987.01800180093014) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/3579504/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arch%20Gen%20Psychiatry&title=A%20systematic%20method%20for%20clinical%20description%20and%20classification%20of%20personality%20variants.%20A%20proposal&author=CR%20Cloninger&volume=44&publication_year=1987&pages=573-588&pmid=3579504&doi=10.1001/archpsyc.1987.01800180093014&)

20. Cloninger CR, Svrakic DM, Przybeck TR. A psychobiological model of temperament and character. Arch Gen Psychiatry. 1993;50:975–990. doi: 10.1001/archpsyc.1993.01820240059008.  [DOI](https://doi.org/10.1001/archpsyc.1993.01820240059008) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8250684/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arch%20Gen%20Psychiatry&title=A%20psychobiological%20model%20of%20temperament%20and%20character&author=CR%20Cloninger&author=DM%20Svrakic&author=TR%20Przybeck&volume=50&publication_year=1993&pages=975-990&pmid=8250684&doi=10.1001/archpsyc.1993.01820240059008&)

21. Svrakic DM, Whitehead C, Przybeck TR, Cloninger CR. Differential diagnosis of personality disorders by the seven-factor model of temperament and character. Arch Gen Psychiatry. 1993;50:991–999. doi: 10.1001/archpsyc.1993.01820240075009.  [DOI](https://doi.org/10.1001/archpsyc.1993.01820240075009) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8250685/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arch%20Gen%20Psychiatry&title=Differential%20diagnosis%20of%20personality%20disorders%20by%20the%20seven-factor%20model%20of%20temperament%20and%20character&author=DM%20Svrakic&author=C%20Whitehead&author=TR%20Przybeck&author=CR%20Cloninger&volume=50&publication_year=1993&pages=991-999&pmid=8250685&doi=10.1001/archpsyc.1993.01820240075009&)

22. Ujike H, Harano M, Inada T, Yamada M, Komiyama T, Sekine Y, et al. Nine- or fewer repeat alleles in VNTR polymorphism of the dopamine transporter gene is a strong risk factor for prolonged methamphetamine psychosis. Pharmacogenomics J. 2003;3:242–247. doi: 10.1038/sj.tpj.6500189.  [DOI](https://doi.org/10.1038/sj.tpj.6500189) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12931138/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=Nine-%20or%20fewer%20repeat%20alleles%20in%20VNTR%20polymorphism%20of%20the%20dopamine%20transporter%20gene%20is%20a%20strong%20risk%20factor%20for%20prolonged%20methamphetamine%20psychosis&author=H%20Ujike&author=M%20Harano&author=T%20Inada&author=M%20Yamada&author=T%20Komiyama&volume=3&publication_year=2003&pages=242-247&pmid=12931138&doi=10.1038/sj.tpj.6500189&)

23. Ujike H. [Japanese Genetics Initiative for Drug Abuse (JGIDA)] Nihon Shinkei Seishin Yakurigaku Zasshi=Japanese Journal of Psychopharmacology. 2004;24:299–302.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/15658508/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nihon%20Shinkei%20Seishin%20Yakurigaku%20Zasshi=Japanese%20Journal%20of%20Psychopharmacology&title=%5BJapanese%20Genetics%20Initiative%20for%20Drug%20Abuse%20(JGIDA)%5D&author=H%20Ujike&volume=24&publication_year=2004&pages=299-302&pmid=15658508&)

24. Benjamini Y, Hochberg Y. Controlling the false discovery rate: a practical and powerful approach to multiple testing. J R Statist Soc B. 1995;57:289–300.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20R%20Statist%20Soc%20B.&title=Controlling%20the%20false%20discovery%20rate:%20a%20practical%20and%20powerful%20approach%20to%20multiple%20testing&author=Y%20Benjamini&author=Y%20Hochberg&volume=57&publication_year=1995&pages=289-300&)

25. Storey J. The positive false discovery rate: a Bayesian interpretation and the q-value. Ann Statist. 2001;31:2013–2035.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Statist&title=The%20positive%20false%20discovery%20rate:%20a%20Bayesian%20interpretation%20and%20the%20q-value&author=J%20Storey&volume=31&publication_year=2001&pages=2013-2035&)

26. Purcell S, Neale B, Todd-Brown K, Thomas L, Ferreira MA, Bender D, et al. PLINK: a tool set for whole-genome association and population-based linkage analyses. Am J Hum Genet. 2007;81:559–575. doi: 10.1086/519795.  [DOI](https://doi.org/10.1086/519795) | [PMC free article](/articles/PMC1950838/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17701901/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=PLINK:%20a%20tool%20set%20for%20whole-genome%20association%20and%20population-based%20linkage%20analyses&author=S%20Purcell&author=B%20Neale&author=K%20Todd-Brown&author=L%20Thomas&author=MA%20Ferreira&volume=81&publication_year=2007&pages=559-575&pmid=17701901&doi=10.1086/519795&)

27. Barrett JC, Fry B, Maller J, Daly MJ. Haploview: analysis and visualization of LD and haplotype maps. Bioinformatics. 2005;21:263–265. doi: 10.1093/bioinformatics/bth457.  [DOI](https://doi.org/10.1093/bioinformatics/bth457) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15297300/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=Haploview:%20analysis%20and%20visualization%20of%20LD%20and%20haplotype%20maps&author=JC%20Barrett&author=B%20Fry&author=J%20Maller&author=MJ%20Daly&volume=21&publication_year=2005&pages=263-265&pmid=15297300&doi=10.1093/bioinformatics/bth457&)

28. Nyholt DR. Genetic case-control association studies—correcting for multiple testing. Hum Genet. 2001;109:564–567. doi: 10.1007/s00439-001-0611-4.  [DOI](https://doi.org/10.1007/s00439-001-0611-4) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11735033/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Genet&title=Genetic%20case-control%20association%20studies%E2%80%94correcting%20for%20multiple%20testing&author=DR%20Nyholt&volume=109&publication_year=2001&pages=564-567&pmid=11735033&doi=10.1007/s00439-001-0611-4&)

29. Perneger TV. What's wrong with Bonferroni adjustments. BMJ. 1998;316:1236–1238. doi: 10.1136/bmj.316.7139.1236.  [DOI](https://doi.org/10.1136/bmj.316.7139.1236) | [PMC free article](/articles/PMC1112991/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9553006/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMJ&title=What's%20wrong%20with%20Bonferroni%20adjustments&author=TV%20Perneger&volume=316&publication_year=1998&pages=1236-1238&pmid=9553006&doi=10.1136/bmj.316.7139.1236&)

30. Hakamata Y, Iwase M, Iwata H, Kobayashi T, Tamaki T, Nishio M, et al. Regional brain cerebral glucose metabolism and temperament: a positron emission tomography study. Neurosci Lett. 2006;396:33–37. doi: 10.1016/j.neulet.2005.11.017.  [DOI](https://doi.org/10.1016/j.neulet.2005.11.017) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16356648/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neurosci%20Lett&title=Regional%20brain%20cerebral%20glucose%20metabolism%20and%20temperament:%20a%20positron%20emission%20tomography%20study&author=Y%20Hakamata&author=M%20Iwase&author=H%20Iwata&author=T%20Kobayashi&author=T%20Tamaki&volume=396&publication_year=2006&pages=33-37&pmid=16356648&doi=10.1016/j.neulet.2005.11.017&)

31. Zink CF, Pagnoni G, Martin-Skurski ME, Chappelow JC, Berns GS. Human striatal responses to monetary reward depend on saliency. Neuron. 2004;42:509–517. doi: 10.1016/s0896-6273(04)00183-7.  [DOI](https://doi.org/10.1016/s0896-6273(04)00183-7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15134646/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neuron&title=Human%20striatal%20responses%20to%20monetary%20reward%20depend%20on%20saliency&author=CF%20Zink&author=G%20Pagnoni&author=ME%20Martin-Skurski&author=JC%20Chappelow&author=GS%20Berns&volume=42&publication_year=2004&pages=509-517&pmid=15134646&doi=10.1016/s0896-6273(04)00183-7&)

32. Law PY, Wong YH, Loh HH. Molecular mechanisms and regulation of opioid receptor signaling. Annu Rev Pharmacol Toxicol. 2000;40:389–430. doi: 10.1146/annurev.pharmtox.40.1.389.  [DOI](https://doi.org/10.1146/annurev.pharmtox.40.1.389) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10836142/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Annu%20Rev%20Pharmacol%20Toxicol&title=Molecular%20mechanisms%20and%20regulation%20of%20opioid%20receptor%20signaling&author=PY%20Law&author=YH%20Wong&author=HH%20Loh&volume=40&publication_year=2000&pages=389-430&pmid=10836142&doi=10.1146/annurev.pharmtox.40.1.389&)

33. Childers SR. Opioid receptors: pinning down the opiate targets. Curr Biol. 1997;7:R695–R697. doi: 10.1016/s0960-9822(06)00358-7.  [DOI](https://doi.org/10.1016/s0960-9822(06)00358-7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9382783/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Biol&title=Opioid%20receptors:%20pinning%20down%20the%20opiate%20targets&author=SR%20Childers&volume=7&publication_year=1997&pages=R695-R697&pmid=9382783&doi=10.1016/s0960-9822(06)00358-7&)

34. Ikeda K, Kobayashi T, Kumanishi T, Yano R, Sora I, Niki H. Molecular mechanisms of analgesia induced by opioids and ethanol: is the GIRK channel one of the keys. Neurosci Res. 2002;44:121–131. doi: 10.1016/s0168-0102(02)00094-9.  [DOI](https://doi.org/10.1016/s0168-0102(02)00094-9) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12354627/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neurosci%20Res&title=Molecular%20mechanisms%20of%20analgesia%20induced%20by%20opioids%20and%20ethanol:%20is%20the%20GIRK%20channel%20one%20of%20the%20keys&author=K%20Ikeda&author=T%20Kobayashi&author=T%20Kumanishi&author=R%20Yano&author=I%20Sora&volume=44&publication_year=2002&pages=121-131&pmid=12354627&doi=10.1016/s0168-0102(02)00094-9&)

35. Carlezon WA, Thome J, Olson VG, Lane-Ladd SB, Brodkin ES, Hiroi N, et al. Regulation of cocaine reward by CREB. Science. 1998;282:2272–2275. doi: 10.1126/science.282.5397.2272.  [DOI](https://doi.org/10.1126/science.282.5397.2272) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9856954/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=Regulation%20of%20cocaine%20reward%20by%20CREB&author=WA%20Carlezon&author=J%20Thome&author=VG%20Olson&author=SB%20Lane-Ladd&author=ES%20Brodkin&volume=282&publication_year=1998&pages=2272-2275&pmid=9856954&doi=10.1126/science.282.5397.2272&)

36. Nestler EJ. Molecular mechanisms of drug addiction. Neuropharmacology. 2004;47 (Suppl 1:24–32. doi: 10.1016/j.neuropharm.2004.06.031.  [DOI](https://doi.org/10.1016/j.neuropharm.2004.06.031) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15464123/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neuropharmacology&title=Molecular%20mechanisms%20of%20drug%20addiction&author=EJ%20Nestler&volume=47&publication_year=2004&pages=24-32&pmid=15464123&doi=10.1016/j.neuropharm.2004.06.031&)

37. Ho IK, Loh HH, Way EL. Cyclic adenosine monophosphate antagonism of morphine analgesia. J Pharmacol Exp Ther. 1973;185:336–346.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/4350037/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharmacol%20Exp%20Ther&title=Cyclic%20adenosine%20monophosphate%20antagonism%20of%20morphine%20analgesia&author=IK%20Ho&author=HH%20Loh&author=EL%20Way&volume=185&publication_year=1973&pages=336-346&pmid=4350037&)

38. Kim KS, Lee KW, Im JY, Yoo JY, Kim SW, Lee JK, et al. Adenylyl cyclase type 5 (AC5) is an essential mediator of morphine action. Proc Natl Acad Sci USA. 2006;103:3908–3913. doi: 10.1073/pnas.0508812103.  [DOI](https://doi.org/10.1073/pnas.0508812103) | [PMC free article](/articles/PMC1533788/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16537460/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20USA&title=Adenylyl%20cyclase%20type%205%20(AC5)%20is%20an%20essential%20mediator%20of%20morphine%20action&author=KS%20Kim&author=KW%20Lee&author=JY%20Im&author=JY%20Yoo&author=SW%20Kim&volume=103&publication_year=2006&pages=3908-3913&pmid=16537460&doi=10.1073/pnas.0508812103&)

39. Barrot M, Olivier JD, Perrotti LI, DiLeone RJ, Berton O, Eisch AJ, et al. CREB activity in the nucleus accumbens shell controls gating of behavioral responses to emotional stimuli. Proc Natl Acad Sci USA. 2002;99:11435–11440. doi: 10.1073/pnas.172091899.  [DOI](https://doi.org/10.1073/pnas.172091899) | [PMC free article](/articles/PMC123274/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12165570/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20USA&title=CREB%20activity%20in%20the%20nucleus%20accumbens%20shell%20controls%20gating%20of%20behavioral%20responses%20to%20emotional%20stimuli&author=M%20Barrot&author=JD%20Olivier&author=LI%20Perrotti&author=RJ%20DiLeone&author=O%20Berton&volume=99&publication_year=2002&pages=11435-11440&pmid=12165570&doi=10.1073/pnas.172091899&)

40. Maldonado R, Blendy JA, Tzavara E, Gass P, Roques BP, Hanoune J, et al. Reduction of morphine abstinence in mice with a mutation in the gene encoding CREB. Science. 1996;273:657–659. doi: 10.1126/science.273.5275.657.  [DOI](https://doi.org/10.1126/science.273.5275.657) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8662559/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=Reduction%20of%20morphine%20abstinence%20in%20mice%20with%20a%20mutation%20in%20the%20gene%20encoding%20CREB&author=R%20Maldonado&author=JA%20Blendy&author=E%20Tzavara&author=P%20Gass&author=BP%20Roques&volume=273&publication_year=1996&pages=657-659&pmid=8662559&doi=10.1126/science.273.5275.657&)

41. Walters CL, Godfrey M, Li X, Blendy JA. Alterations in morphine-induced reward, locomotor activity, and thermoregulation in CREB-deficient mice. Brain Res. 2005;1032:193–199. doi: 10.1016/j.brainres.2004.11.013.  [DOI](https://doi.org/10.1016/j.brainres.2004.11.013) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15680959/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Brain%20Res&title=Alterations%20in%20morphine-induced%20reward,%20locomotor%20activity,%20and%20thermoregulation%20in%20CREB-deficient%20mice&author=CL%20Walters&author=M%20Godfrey&author=X%20Li&author=JA%20Blendy&volume=1032&publication_year=2005&pages=193-199&pmid=15680959&doi=10.1016/j.brainres.2004.11.013&)

42. Richon VM, Johnston D, Sneeringer CJ, Jin L, Majer CR, Elliston K, et al. Chemogenetic analysis of human protein methyltransferases. Chem Biol Drug Des. 2011;78:199–210. doi: 10.1111/j.1747-0285.2011.01135.x.  [DOI](https://doi.org/10.1111/j.1747-0285.2011.01135.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21564555/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Chem%20Biol%20Drug%20Des&title=Chemogenetic%20analysis%20of%20human%20protein%20methyltransferases&author=VM%20Richon&author=D%20Johnston&author=CJ%20Sneeringer&author=L%20Jin&author=CR%20Majer&volume=78&publication_year=2011&pages=199-210&pmid=21564555&doi=10.1111/j.1747-0285.2011.01135.x&)

43. Wang J, Barke RA, Roy S. Transcriptional and epigenetic regulation of interleukin-2 gene in activated T cells by morphine. J Biol Chem. 2007;282:7164–7171. doi: 10.1074/jbc.M604367200.  [DOI](https://doi.org/10.1074/jbc.M604367200) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17227776/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=Transcriptional%20and%20epigenetic%20regulation%20of%20interleukin-2%20gene%20in%20activated%20T%20cells%20by%20morphine&author=J%20Wang&author=RA%20Barke&author=S%20Roy&volume=282&publication_year=2007&pages=7164-7171&pmid=17227776&doi=10.1074/jbc.M604367200&)

44. Ioannidis JP, Thomas G, Daly MJ. Validating, augmenting and refining genome-wide association signals. Nat Rev Genet. 2009;10:318–329. doi: 10.1038/nrg2544.  [DOI](https://doi.org/10.1038/nrg2544) | [PMC free article](/articles/PMC7877552/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19373277/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Genet&title=Validating,%20augmenting%20and%20refining%20genome-wide%20association%20signals&author=JP%20Ioannidis&author=G%20Thomas&author=MJ%20Daly&volume=10&publication_year=2009&pages=318-329&pmid=19373277&doi=10.1038/nrg2544&)

45. Hirschhorn JN, Daly MJ. Genome-wide association studies for common diseases and complex traits. Nat Rev Genet. 2005;6:95–108. doi: 10.1038/nrg1521.  [DOI](https://doi.org/10.1038/nrg1521) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15716906/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Genet&title=Genome-wide%20association%20studies%20for%20common%20diseases%20and%20complex%20traits&author=JN%20Hirschhorn&author=MJ%20Daly&volume=6&publication_year=2005&pages=95-108&pmid=15716906&doi=10.1038/nrg1521&)

46. Risch N, Merikangas K. The future of genetic studies of complex human diseases. Science. 1996;273:1516–1517. doi: 10.1126/science.273.5281.1516.  [DOI](https://doi.org/10.1126/science.273.5281.1516) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8801636/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=The%20future%20of%20genetic%20studies%20of%20complex%20human%20diseases&author=N%20Risch&author=K%20Merikangas&volume=273&publication_year=1996&pages=1516-1517&pmid=8801636&doi=10.1126/science.273.5281.1516&)
