# Genome Wide Analysis of Drug-Induced Torsades de Pointes: Lack of Common Variants with Large Effect Sizes

## Metadata
**Authors:** Elijah R Behr, Marylyn D Ritchie, Toshihiro Tanaka, Stefan Kääb, Dana C Crawford, Paola Nicoletti, Aris Floratos, Moritz F Sinner, Prince J Kannankeril, Arthur A M Wilde, Connie R Bezzina, Eric Schulze-Bahr, Sven Zumhagen, Pascale Guicheney, Nanette H Bishopric, Vanessa Marshall, Saad Shakir, Chrysoula Dalageorgou, Steve Bevan, Yalda Jamshidi, Rachel Bastiaenen, Robert J Myerburg, Jean-Jacques Schott, A John Camm, Gerhard Steinbeck, Kris Norris, Russ B Altman, Nicholas P Tatonetti, Steve Jeffery, Michiaki Kubo, Yusuke Nakamura, Yufeng Shen, Alfred L George, Jr, Dan M Roden
**Journal:** PLoS ONE
**Date:** 2013 Nov 6
**DOI:** [10.1371/journal.pone.0078511](https://doi.org/10.1371/journal.pone.0078511)
**PMID:** 24223155
**PMCID:** PMC3819377
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3819377/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC3819377/pdf/pone.0078511.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC3819377/pdf/pone.0078511.pdf)

## Abstract

Marked prolongation of the QT interval on the electrocardiogram associated with the polymorphic ventricular tachycardia Torsades de Pointes is a serious adverse event during treatment with antiarrhythmic drugs and other culprit medications, and is a common cause for drug relabeling and withdrawal. Although clinical risk factors have been identified, the syndrome remains unpredictable in an individual patient. Here we used genome-wide association analysis to search for common predisposing genetic variants. Cases of drug-induced Torsades de Pointes (diTdP), treatment tolerant controls, and general population controls were ascertained across multiple sites using common definitions, and genotyped on the Illumina 610k or 1M-Duo BeadChips. Principal Components Analysis was used to select 216 Northwestern European diTdP cases and 771 ancestry-matched controls, including treatment-tolerant and general population subjects. With these sample sizes, there is 80% power to detect a variant at genome-wide significance with minor allele frequency of 10% and conferring an odds ratio of ≥2.7. Tests of association were carried out for each single nucleotide polymorphism (SNP) by logistic regression adjusting for gender and population structure. No SNP reached genome wide-significance; the variant with the lowest P value was rs2276314, a non-synonymous coding variant in C18orf21 (p = 3×10−7, odds ratio = 2, 95% confidence intervals: 1.5–2.6). The haplotype formed by rs2276314 and a second SNP, rs767531, was significantly more frequent in controls than cases (p = 3×10−9). Expanding the number of controls and a gene-based analysis did not yield significant associations. This study argues that common genomic variants do not contribute importantly to risk for drug-induced Torsades de Pointes across multiple drugs.

## Introduction

Marked prolongation of the QT interval on the electrocardiogram can occur in an unpredictable fashion during treatment with both QT interval-prolonging antiarrhythmic drugs as well as with drugs prescribed for non-cardiovascular indications [[1]](#pone.0078511-Fenichel1), [[2]](#pone.0078511-Haverkamp1). Such QT interval prolongation, in turn, is associated with a morphologically-distinctive and potentially fatal polymorphic ventricular tachycardia termed Torsades de Pointes (TdP). This severe adverse event (SAE), drug-induced Torsades de Pointes (diTdP), occurs in 1–5% of patients exposed to QT-prolonging antiarrhythmics (most often prescribed for atrial fibrillation, AF), and is much less common in patients treated with “non-cardiovascular” QT culprit drugs. Clinical risk factors have been identified and these include female gender, hypokalemia, bradycardia, and recent conversion from AF to normal rhythm. Basic electrophysiologic studies have demonstrated that inhibition of a major repolarizing potassium current, I_Kr_, is the common mechanism across multiple drugs and drug classes [[1]](#pone.0078511-Fenichel1)–[[3]](#pone.0078511-Roden1). In addition, for specific drugs, inhibition of metabolism or elimination leading to accumulation of high drug concentrations in plasma has been implicated. This SAE has been a leading cause for drug withdrawal and relabeling, and evaluation of the potential that a new drug entity will carry a risk of diTdP has become a standard component of new drug evaluation [[4]](#pone.0078511-Roden2).

A number of lines of evidence suggest that genomic variants may contribute to variable susceptibility to diTdP. The syndrome phenocopies many of the clinical features of the congenital long QT syndrome, a disease caused by mutations in genes encoding ion channels or their function modifying subunits [[3]](#pone.0078511-Roden1), [[5]](#pone.0078511-The1), and case series have identified mutations in these genes in a minority of patients with diTdP but normal QT intervals at baseline [[6]](#pone.0078511-Yang1)–[[8]](#pone.0078511-Itoh1); this is thought to represent incomplete penetrance of the congenital syndrome. Further evidence for a genetic contribution to diTdP is provided by a report that QT interval responses to challenge with the antiarrhythmic quinidine were greater among first-degree relatives of patients with diTdP than among controls [[9]](#pone.0078511-Kannankeril1). Finally, the duration of the QT interval itself is known to be heritable [[10]](#pone.0078511-Hanson1)-[[13]](#pone.0078511-NewtonCheh1), and multiple genomic loci have been highly significantly and reproducibly associated with this variability [[14]](#pone.0078511-Pfeufer1), [[15]](#pone.0078511-NewtonCheh2).

The genome-wide-association study (GWAS) paradigm has previously been used to identify strong signals for susceptibility to SAEs using small numbers (<100) cases [[16]](#pone.0078511-Daly1)–[[18]](#pone.0078511-Link1). We present here the results of a GWAS to test hypothesis that common genomic variation contributes to risk for diTdP. Cases were accrued by two multicenter collaborations and compared to drug-challenged controls manifesting no major QT interval prolongation as defined below and to population controls. In this analysis, we find no highly statistically significant association between common single nucleotide polymorphisms and risk for diTdP in a Northwestern European population.

## Methods

### Cases

Cases of diTdP reported here were ascertained using the common definition below by two groups: the Trans-Atlantic Alliance Against Sudden Death supported by the Fondation Leducq and the Drug-induced Arrhythmia Risk Evaluation (DARE) study [[5]](#pone.0078511-The1). The Leducq set was collected by six North American and European University Medical centers: Vanderbilt, Miami, Munster, Amsterdam, Nantes, and Munich. The DARE study is a systematic national study of cases of British drug-induced arrhythmia identified prospectively over an initial 5 year period (2003-8).

A common case definition of diTdP was utilized: documented polymorphic ventricular tachycardia; typical pause-dependent initiation; association with exposure to a known or likely culprit drug with resolution of TdP and at least partial resolution of QT prolongation with removal of drug exposure. QT interval prolongation was also required; this was most often documented by a single lead rhythm strip, and was thus not further quantified in the majority of cases. Cases of diTdP occurring in subjects with known or later identified congenital long QT syndrome were not excluded. Those associated with clear acute cardiac ischemia (which may represent a different mechanism) were excluded. Structural heart disease was not an exclusion criterion. Age, gender, culprit drug, presence of atrial fibrillation (AF), and self-reported ethnicity were collected as covariates.

### Controls

There were three control groups: (1) Drug-challenged controls: a set of 470 subjects ascertained at Vanderbilt who were challenged with intravenous ibutilide or oral antiarrhythmics (sotalol, dofetilide, or quinidine) and who did not display any QTc interval >500 msec or any increase in QT >50 msec; and (2) 649 general population control (POPRES) [[19]](#pone.0078511-Nelson1) including Northwestern European subjects and Europeans from other sub-populations. The primary analysis was subsequently supplemented with a further 4231 British subjects from the Wellcome Trust Case Control Consortium phase 2 (WTCCC2), downloaded from [http://www.ebi.ac.uk/ega/page.php](http://www.ebi.ac.uk/ega/page.php). Because diLQTS is rare in the population, using a large population control set increases power even recognizing the (small) potential for misclassification [[20]](#pone.0078511-McCarthy1). Population controls have been successfully used in this fashion in previous studies of rare adverse drug reactions in which GWAS did yield significant associations [[16]](#pone.0078511-Daly1), [[21]](#pone.0078511-Lucena1).

### Ethics statement

Cases recruited at Vanderbilt Medical Center provided written informed consent for participation under a protocol entitled “Screening for Genetic Causes of Cardiac Arrhythmias” approved by the Vanderbilt Human Research Protection Program (date of latest approval Dec. 11, 2012). All patients give their written consent to participate, and the signed consent form is retained in the patients’ research chart to document the consent process. For minors, written consent was obtained from parent/guardian. For minors ages 12 and older, written consent was obtained from the child also. Under the age of 12, verbal assent was obtained from the child. The signed consent form is retained in the patients’ research chart to document the consent process.

Drug-exposed controls provided written consent under a protocol entitled “Pharmacogenomics of Arrhythmia Therapy-Ion Channel Variants and QT-Prolonging Antiarrhythmics” approved by the Vanderbilt Human Research Protection Program (date of latest approval Dec. 13, 2012). All patients give their written consent to participate, and the signed consent form is retained in the patients’ research chart to document the consent process. For minors, written consent was obtained from parent/guardian. For minors ages 12 and older, written consent was obtained from the child also. Under the age of 12, verbal assent was obtained from the child. The signed consent form is retained in the patients’ research chart to document the consent process. Cases accrued by the Leducq network provided written consent approved by local ethics boards. These are: at Nantes: CPP and ministère de la recherche: biocollection de l’institut de thorax intitulée « Génétique et biomarqueurs des pathologies cardiovasculaires, respiratoires et leur facteur de risques ». Declared under the ref n° DC-2011-1399; at Münster: local ethics committee, University Hospital Münster; at Amsterdam: Medisch Ethische Commissie AMC (Medical Ethical Committee Academic Medical Centre, University of Amsterdam, Amsterdam); at Munich: Ethikkommission bei der LMU München; at Miami: University of Miami Institutional Review Boards. Consent for inclusion in the DARE study was given under Multi-Centre Research Ethics Committee approval MREC 02/02/73 on February 24th 2003 with minor amendment approval given on February 3rd 2010.

### Genotyping

The Leducq cases and the drug-exposed controls were genotyped on the Illumina 610k platform by the RIKEN Center for Genomic Medicine (Yokohama, Japan) as part of a collaboration between RIKEN and the National Institutes of Health-supported Pharmacogenetics Research Network. The DARE cases and the POPRES and WTCCC2 controls were genotyped using the Illumina 1M or 1M-Duo chip. We used a consistent QC procedure for each genotyping assay, as previously described [[16]](#pone.0078511-Daly1), [[21]](#pone.0078511-Lucena1): SNPs required a call rate > 0.95, minor allele frequency > 0.01, and Hardy Weinberg Equilibrium (HWE) in controls p>10^−7^, and subjects required a call rate > 0.9. We tested for cryptic relatedness by estimating identity-by-descent (IBD) for all possible pairs of individuals using PLINK [[22]](#pone.0078511-Purcell1).

### Principal components analysis (PCA)

We used the smartPCA program from the EIGENSTRAT package (version 3.0) [[23]](#pone.0078511-Price1) to conduct a PCA in order to expose population structure. SNPs from known regions of long-range linkage disequilibrium [[24]](#pone.0078511-Novembre1) were removed before conducting the PCA. We first analyzed the study genotype data to identify major ancestry groups (Europeans, Asians and Africans). We then conducted a PCA on Europeans to separate sub-populations within this group.

### Statistical analyses

We conducted statistical tests using PLINK v1.05 [[22]](#pone.0078511-Purcell1). The primary analysis compared the cases of Northwestern European ancestry to genetically matched controls. We conducted analyses comparing cases to drug-exposed controls, to the POPRES population controls, to drug-exposed + POPRES controls, and to drug-exposed + POPRES + WTCCC2 controls. The initial analysis was conducted on the set of SNPs common across all sets, and as discussed below, we also extended the analysis to an imputed set. We also analyzed drug-specific sets for the three most common culprit agents, sotalol, amiodarone and quinidine, and searched for associations using gene-specific analyses, as described further below.

We tested the association of single SNPs with diTdP susceptibility using logistic regression, with gender and Eigen score as covariates in an additive model. We set the genome-wide significance threshold of P-value at to 5×10^−8^ to correct for multiple testing. The power calculation with Bonferroni correction was based upon the following assumptions: 265 cases; a low prevalence of diTdP in the general population; a SNP minor allele frequency (MAF) of 0.1; a 0.8 R^2^ coefficient between the SNP and the causal allele; and an odds ratio (OR) in the range of 1.5 and 2.5. The power was calculated based on a chi-square test under an additive model, using a non-centrality parameter [[25]](#pone.0078511-Chapman1). This supported the use of the WTCCC2 controls to achieve increased power (to 29% for OR = 2.0, to 93% for OR = 2.5, and to 99% for OR = 2.7), in the subsequently analysis of the selective cohorts ([Figure 1](#pone-0078511-g001)).

### Figure 1. Plot of power versus odds ratios, using 700 controls (initial analysis) and then >4000 controls.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d890/3819377/b3d0a57ea8af/pone.0078511.g001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3819377_pone.0078511.g001.jpg)

We calculated r^2^ among top associated SNPs located in a 1M bp interval. In the case of two or more SNPs not correlated with each other, we conducted a haplotype analysis using PLINK.

### Imputation

Imputation of missing genotypes was carried out using IMPUTE2 (February 2009 version) [[26]](#pone.0078511-Marchini1), with data from the 1000 Genomes Project (56 individuals, August 2009 release) and HapMap III (146 individuals, CEU and TSI) as the reference panels. All known SNPs with poor quality were removed before the imputation to avoid false positives. Imputed genotypes with a posterior probability of greater than 0.9 were retained [[26]](#pone.0078511-Marchini1). To assess the accuracy of the imputation results, we carried out the following quality control procedures: (1) we tested the difference of missingness between cases and controls; (2) we tested for HWE (p-value << 0.05); 3) we manually inspected signal quality for all the SNPs in the haplotype generating the imputed genotype.

### Gene-based approaches

We also performed a gene-based GWAS analysis. In this analysis we aggregated SNPs by the genes they are near to increase the signal. This approach is based on the idea that multiple variants in a single gene will perturb gene function [[27]](#pone.0078511-Tatonetti1). For each patient in the cases and the controls, we scored each gene by counting the number of minor alleles in that gene. These counts are weighted by the negative log of the frequency that the allele is observed at in the population. This approach up-weights rare alleles and down-weights common alleles [[27]](#pone.0078511-Tatonetti1). We then performed a univariate logistic regression modeling the cases versus controls as a function of these gene scores. We performed an F-test to assess the significance of the fit.

### Repository information

Portions of the Leducq dataset approved by local IRBs for central deposit have been uploaded to dbGaP at [http://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000331.v1.p1](http://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000331.v1.p1). The DARE dataset is held by the International Serious Adverse Events Consortium (iSAEC) and is available on application to the iSAEC data management committee at [http://www.saeconsortium.org/](http://www.saeconsortium.org/).

## Results

There was a total of 264 cases genotyped, 210 from the Leducq network and 54 from the DARE study. The clinical features of these sets are presented in [Table 1](#pone-0078511-t001). They included 185 women (71%), the age range was 7–91, and the most commonly-implicated drugs were sotalol, amiodarone and quinidine ([Table 2](#pone-0078511-t002)). The majority self-declared European ancestry (98% for the DARE study and 92% for the Leducq network).

### Table 1. diTdP Genome-Wide Association Study: Demographic and clinical characteristics of the enrolled cases summarized by cohort.

| Cohort | Number of cases enrolled | Number of North-western European cases | Age at the onset (Mean, SD) | Female (% of total) | Atrial Fibrillation (Yes, % of total) | % self-defined Europeans |
| --- | --- | --- | --- | --- | --- | --- |
| Leducq | 210 | 163 | 61.9±16.5 | 71% | 49% | 92% * |
| DARE | 54 | 53 | 66.8±15.9 | 64% | 52% | 98% |
| Total | 264 | 216 | 62.5±16.3 | 70% | 50% |   |

Table 1 Caption: *In total, 66% of the Leducq cases self-declared ethnicity, among then 92% (61% of total) are Europeans.

### Table 2. diTdP Northwestern European cohort: causal drugs and demographic and clinical characteristics of the diTdP Northwestern European cases summarized by drug group.

| Drugs | # Cases | Atrial Fibrillation (Yes, %of total) | Age (Mean, SD) | % Female |
| --- | --- | --- | --- | --- |
| Amiodarone | 46 | 50% | 62.5±17.1 | 65% |
| Quinidine | 19 | 63% | 55.6±22.3 | 68% |
| Sotalol | 55 | 53% | 55±26.5 | 75% |
| Others | 141 | 36% | 58.2±23.8 | 72% |

Table 2 Caption: Subjects might have experienced diTdP due to more than one causal drug.

### Quality control and population structure

Genotype data from the 264 cases and 1119 drug-exposed and POPRES controls underwent QC as described above. After dropping SNPs with genotyping efficiency ≤95% or MAF ≤1%, the final set for initial analysis across all platforms used included 478,055 SNPs. Of the 1383 samples, 46 sample pairs had identity-by-descent sharing values > 0.2 in the Leducq set, and we removed the sample from each pair with the lower overall SNP calling rate. In total, we selected for further analyses 242 cases from all ethnicities (189 from the Leducq set and 53 from the DARE set) and 1094 controls (435 drug-exposed control and 659 POPRES controls).

After identifying and removing non-Caucasian subjects by PCA, we investigated the genetic structure of Caucasian cases and drug-exposed controls. As shown in [Figure 2](#pone-0078511-g002), most cases and controls clustered in the Northwestern European cohort. We therefore used samples with PC1>-0.03 to make up the final sets for the primary analysis; this included 216 cases, 386 drug-exposed controls, and 385 POPRES controls.

### Figure 2. Population structure of the Caucasian cases and controls.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d890/3819377/87329511269d/pone.0078511.g002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3819377_pone.0078511.g002.jpg)

The red dots represent diTdP cases and blue dots represent controls (drug-exposed patients and population [POPRES] controls). The plot shows the first and second eigen vectors, which clearly separate the Caucasians into a Northwestern group (top) and other groups from Southern and Eastern Europe. The dense cluster on the lower left represents the subjects of Spanish origin from the POPRES collection. The final analysis included subjects with PC1<–0.03.

### Genome-wide statistical analysis

Using logistic regression with the first three significant principal components and gender as covariates ([Figure 3](#pone-0078511-g003) and [4](#pone-0078511-g004)), we found no SNPs with p-values <10^−7^ in separate analyses of cases versus drug-exposed controls, cases versus population controls, and cases versus the combined control set ([Table 3](#pone-0078511-t003)). In the combined analysis ([Figures 3](#pone-0078511-g003) and [4](#pone-0078511-g004)), the SNP with the lowest P value in an additive model (rs2276314, P = 4×10^−7^, OR = 2, 95% CI 1.5–2.7) was a non-synonymous coding variant in *C18orf21*, a gene thought to be transactivated by Hepatitis B virus. There was no significant difference in MAF between the treatment-tolerant and general population groups ([Table 4](#pone-0078511-t004)), or between subjects with or without AF, indicating that the association is not likely to be influenced by confounding factors. Interestingly, the haplotype AA formed by rs2276314 and rs767531, a nearby SNP, was significantly more frequent in controls (79% of controls vs. 65% of cases) with a chi-square p-value of 1.6×10^−9^. Rs767531 is an intragenic SNP close to *GALNT1,* which is thought to mediate O-linked glycosylation in the Golgi.

### Figure 3. QQ plot of the results from logistic regression on the Northwestern European cohort.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d890/3819377/8f950ac1d102/pone.0078511.g003.jpg)

The x axis is –log10 of the expected P-value and the y axis is –log10 of the observed P-values. Black solid lines denote the null distribution. The bulk of the values (red dots) closely follow the expectation under the null model (black line) showing that there is no significant inflation of test statistic due to factors such as population stratification. The tail end shows significant deviation from null model illustrating that there are a few observed significant associations.

### Figure 4. Manhattan Plot of logistic regression on Northwestern European cohort.

![Figure 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d890/3819377/b2cfb8e64b08/pone.0078511.g004.jpg)

Each dot represents a SNP. The x axis represents the position of the SNP on chromosome. The y axis represents the -log10 of logistic regression P-value of the SNP in the case-control association study. rs2276314 and rs4799838 are marked in green with the P-value just below the genome-wide threshold (dashed line).

### Table 3. diTdP Genome-Wide Association Study: top associated SNPs.

| SNP | CHR | Position | functional type | closest gene | OR | P-value |
| --- | --- | --- | --- | --- | --- | --- |
| rs2276314 | 18 | 33557466 | Non synonymous | C18orf21 | 2.0 (1.5–2.6) | 4.0×10−07 |
| rs4799838 | 18 | 33597530 | Intronic variant | RPRD1A | 1.9 (1.5–2.5) | 9.4×10−07 |
| rs9960370 | 18 | 33494636 | Intergenic variant | MIR187 | 2.0 (1.4–2.7) | 6.0×10−06 |
| rs767531 | 18 | 33357868 | Intergenic variant | GALNT1 | 3.0 (1.8–4.8) | 7.1×10−06 |
| rs1789536 | 18 | 33524398 | Intergenic variant | C18orf21 | 2.0 (1.4–2.7) | 7.4×10−06 |
| rs1789532 | 18 | 33532322 | Intergenic variant | C18orf21 | 2.0 (1.4–2.6) | 7.9×10−06 |
| rs10141001 | 14 | 52132201 | Intronic variant | FRMD6 | 2.0 (1.4–2.7) | 1.0×10−05 |
| rs9805984 | 14 | 52131743 | Intronic variant | FRMD6 | 2.0 (1.5–2.7) | 1.0×10−05 |

Table 3 Caption: Abbreviations: OR (Odds Ratio, OR is expressed with a confidence interval of 95%), CHR (chromosome).

### Table 4. Minor Allele Frequency (MAF) for rs2276314, the top associated SNP, in different sample sub-groups.

| sub-groups | MAF |
| --- | --- |
| DC controls (382) | 0.19 |
| POPRES controls (389) | 0.19 |
| subjects with AF (371) | 0.23 |
| subjects without AF (182) | 0.21 |

Table 4 Caption: Abbreviations: AF (atrial fibrillation), MAF (minor allele frequency).

We investigated drug-specific risk alleles by comparing the 771 Northwestern European control subjects with three drug-specific Northwestern European case groups ([Figure 5](#pone-0078511-g005)): sotalol (55 cases), amiodarone (46 cases), and quinidine (19 cases). We found no SNP with genome-wide significance in any of the three comparisons.

### Figure 5. The effect size (OR) of rs2276314 in different drug-specific groups.

![Figure 5](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d890/3819377/7b15b163feca/pone.0078511.g005.jpg)

The numbers in parentheses are the numbers of cases for each group. The horizontal blue lines mark the 95% confidence interval of odds ratios.

An initial analysis of the two cohorts separately did suggest possible signals at or near genome-wide significance, but these signals were not observed in the combined analysis presented here. For example, the initial analysis of the DARE set identified rs2200733 as a potential risk allele. This variant has previously been associated with AF [[28]](#pone.0078511-Gudbjartsson1) and thus may represent the fact that antiarrhythmic drugs, the most common culprit agents for diTdP, are most often prescribed for patients with AF. The association was not seen in the Leducq set or in the combined analysis.

### Alternate analytic approaches

We used imputation to increase the number of SNPs evaluated to 3,542,142 as described in the methods section. We repeated the phenotype and drug specific analyses and we found no genome-wide significant associations. We also obtained no significant associations by including the WTCCC2 controls.

Using the gene-based GWAS analysis method described, we did not find any significant associations at the p<10^−6^. The most significant association encoded for a cytochrome P450 enzyme, CYP2U1 on chromosome 4 (p = 1.2×10^−5^), followed by ubiquitin, USP25 on chromosome 21 (p = 3.4×10^−5^), and a G protein-coupled receptor, OPRL1 on chromosome 20 (p = 9.2×10^−5^).

As mentioned above, small series have identified occasional patients with incompletely penetrant congenital long QT syndrome that becomes manifest with drug exposure [[6]](#pone.0078511-Yang1)–[[8]](#pone.0078511-Itoh1). In a minority of diTdP cases analyzed here (n = 66–132), coding region resequencing in six congenital long QT syndrome disease genes had previously been undertaken. As shown in [Table 5](#pone-0078511-t005), this effort identified 10 rare variants implicated as function-altering likely congenital syndrome mutations [[6]](#pone.0078511-Yang1), [[29]](#pone.0078511-Mohler1), [[30]](#pone.0078511-Hoppe1).

### Table 5. Screening rare variants in congenital long QT Syndrome disease genes in 216 diTdP cases.

|   | KCNQ1 | KCNH2 | SCN5A | ANK2 | KCNE1 | KCNE2 |
| --- | --- | --- | --- | --- | --- | --- |
| Number screened | 131 | 132 | 129 | 66 | 89 | 80 |
| Rare variants identified | 3 | 3 | 1 | 2 | 1 | 0 |
| Rare variants identified | W305X | R784W | A385T | T1404I | D76N |   |
|   | T224M | G934V |   | V1516D |   |   |
|   |   | R954C |   |   |   |   |
## Discussion

Severe adverse events during drug therapy are by their nature unpredictable: if markers of high risk are known, they could be used to reduce or even eliminate risk in an individual subject or for a drug or drug class. The very unpredictable nature of many such events has prompted a search for underlying markers of genomic susceptibility. In the case of diTdP, a contribution by a genomic component is especially appealing given the clinical similarity to the congenital long QT syndrome, evidence that QT interval responses to drug challenge may be exaggerated among relatives of those with diTdP, and the known heritability of QT duration.

One of the major impediments to identifying genomic markers of risk for TdP or other severe adverse drug reaction is the very rarity of the event that precludes accumulation of large numbers of cases [[31]](#pone.0078511-Pirmohamed1). This integrated analysis of two large multi-institutional ascertainment efforts presented here represents the largest such effort for diTdP to date. The analysis did not identify common variants conferring high risk for diTdP. This negative finding suggests that it is unlikely that any single common genomic markers of risk with odds ratios greater than 2.5 contribute. The result leaves open the possibility that predisposition to diTdP is determined by interactions of multiple variants each with modest effect sizes, or that rare variants play a prominent role. Indeed, a recent large-scale candidate gene analysis of the Leducq set implicated an uncommon non-synonymous variant in the potassium channel subunit *KCNE1*: minor allele frequency in control populations was 1.8–2.9% and 8.6% in cases [[32]](#pone.0078511-Kaab1). A trend in the same direction was seen in the DARE set.

In addition, environmental factors may also be important; these include electrolyte abnormalities, alterations in autonomic tone or heart rate, or underlying heart disease. Indeed, the signal observed in the DARE set between AF risk alleles and diTdP may represent confounding by indication (i.e. AF is an indication for the culprit drugs) or may represent an association between homeostatic perturbations induced by AF and subsequent susceptibility to diTdP. Studies of QT interval during AF have identified abnormal control of repolarization (very flat QT-RR relationships; i.e. absence of expected QT prolongation at slower heart rates) during the arrhythmia, with normalization or even super-normalization (i.e. longer than expected QT interval at any given heart rate) after conversion to normal rhythm [[33]](#pone.0078511-Roden3), [[34]](#pone.0078511-Darbar1). The mechanism underlying these changes has not been elucidated, although it is known that the neurohormonal milieu does change shortly after conversion to normal rhythm: for example, catecholamines [[35]](#pone.0078511-Salerno1) and atrial natriuretic peptides [[36]](#pone.0078511-Arakawa1) are higher during AF than after conversion. These changes may contribute to altered QT-RR regulation and susceptibility to diTdP after conversion to normal rhythm [[33]](#pone.0078511-Roden3).

The SNP with the lowest P value is a non-synonymous variant in *C18orf21* that results in a substitution of an alanine for a threonine at position 132 of the protein. The threonine is a predicted phosphorylation site (according to the NetPhos 2.0 website). The function of this transcript is unknown, so this signal may represent new biology or a false positive.

### Limitations

In contrast to the present study, the GWAS paradigm has been used to identify single loci with strong effects in other adverse drug reactions. Carbamazepine-related skin toxicity [[17]](#pone.0078511-McCormack1) and flucloxacillin-related hepatotoxicity [[16]](#pone.0078511-Daly1) were linked to variants in the HLA locus in studies using discovery sets of 65 and 51 cases respectively. Similarly, a GWAS of 85 cases of myotoxicity related to high dose simvastatin identified variants at the *SLCO1B1* locus, encoding a statin drug transporter [[18]](#pone.0078511-Link1). The present study also examined a rare adverse drug reaction and included 216 cases with a well-defined genetic background, but did not identify loci strongly associated with diTdP. We believe this indicates that variants at a single locus do not have a strong influence on diTdP risk across drugs and drug classes. Basic electrophysiologic studies have implicated a common mechanism across multiple drugs, block of the rapid component of the delayed rectifier potassium current I_Kr_, in diTdP [[4]](#pone.0078511-Roden2), [[37]](#pone.0078511-Sanguinetti1) and justify the inclusion of multiple culprit drugs in the present study. This design will not detect drug-specific mechanisms predisposing to diTdP; examples could include alterations in elimination pathways or in drug-specific (e.g. non-I_Kr_-mediated) electrophysiologic pathways leading to diTdP [[38]](#pone.0078511-Lu1). The analysis by drug performed here did not yield any suggestive signals, but the numbers are small; similarly, an analysis by gender did not yield significant associations. We also did not exclude patients with heart failure, which is associated with QT prolongation. A recent study of amiodarone-related diTdP [[39]](#pone.0078511-Jamshidi1), using the multiple SNPs interrogated by the cardiochip, has suggested that variants in the *NOS1AP* locus, known to modulate baseline QT interval [[14]](#pone.0078511-Pfeufer1), [[15]](#pone.0078511-NewtonCheh2), also confer risk for amiodarone-related TdP. Rare variants may contribute to susceptibility, although screening in congenital long QT syndrome disease genes yielded likely contributory mutations in 0–3% (depending on the gene; [Table 5](#pone-0078511-t005)) of subjects.

Small numbers of cases represent a generic problem in the study of rare adverse drug events and support the development of further multi-institutional efforts to accrue cases and controls under uniform protocols and definitions [[31]](#pone.0078511-Pirmohamed1), [[40]](#pone.0078511-Wilke1)._ENREF_24 The present study cannot address whether there are common genomic variants that confer risk in subjects of non-Northwestern European ancestry. One small association study has suggested that the common S1103Y variant in the cardiac sodium channel gene *SCN5A*, seen almost exclusively in subjects of African origin, confers increased risk for a range of arrhythmias, including diTdP [[41]](#pone.0078511-Splawski1). Follow-up clinical and *in vitro* studies have further supported a role for this variant in arrhythmia susceptibility [[42]](#pone.0078511-Plant1), [[43]](#pone.0078511-VanNorstrand1). It is possible that gene-gene interactions drive susceptibility to diTdP but the present study is not powered to detect such associations.

In conclusion, the results described here represent analysis of the largest number of genotyped cases of diTdP reported to date, and effectively rule out any likely role for common variation with anything other than modest effects. Further efforts to define risk for the adverse drug event would benefit from accrual of larger patient sets and extension of analytical approaches to sequence-based analysis to further explore the hypothesis that rare variants (e.g. in congenital arrhythmia syndrome genes) mediate diTdP risk. The precise role of genetic variation in the risk of diTdP remains to be determined.

## Funding Statement

This study was supported by U01HL65962, the Pharmacogenomics of Arrhythmia Therapy site of the Pharmacogenetics Research Network, and by a grant from the Fondation Leducq (Trans-Atlantic Network of Excellence “Alliance Against Sudden Cardiac Death”, 05 CVD 01). Genotyping the Leducq samples described in the text was performed by the RIKEN Center for Genomic Medicine, Yokohama, Japan, which is supported in part by the BioBank Japan Project funded by the Japanese Ministry of Education, Culture, Sports, Science and Technology. The DARE study was funded by the British Heart Foundation under special project grant SP/02/001. The DARE samples were genotyped at Gene Expression Ltd. Further support was provided by a collaboration with the International Serious Adverse Events Consortium, whose membership currently includes Abbott, Amgen, Astra-Zeneca, Cerner, Daiichi-Sankyo, GlaxoSmithKline, Merck, Novartis, Pfizer, Takeda, and the Welcome Trust. Portions of the Leducq dataset have been deposited at dbGaP at [http://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000331.v1.p1](http://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000331.v1.p1). The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.

## References

1. Fenichel RR, Malik M, Antzelevitch C, Sanguinetti M, Roden DM, et al. (2004) Drug-induced torsades de pointes and implications for drug development. J Cardiovasc Electrophysiol 15: 475–495.  [DOI](https://doi.org/10.1046/j.1540-8167.2004.03534.x) | [PMC free article](/articles/PMC1544371/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15090000/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Cardiovasc%20Electrophysiol&title=Drug-induced%20torsades%20de%20pointes%20and%20implications%20for%20drug%20development&author=RR%20Fenichel&author=M%20Malik&author=C%20Antzelevitch&author=M%20Sanguinetti&author=DM%20Roden&volume=15&publication_year=2004&pages=475-495&pmid=15090000&doi=10.1046/j.1540-8167.2004.03534.x&)

2. Haverkamp W, Breithardt G, Camm AJ, Janse MJ, Rosen MR, et al. (2000) The potential for QT prolongation and proarrhythmia by non- antiarrhythmic drugs: clinical and regulatory implications. Report on a policy conference of the European Society of Cardiology. European Heart Journal 21: 1216–1231.  [DOI](https://doi.org/10.1053/euhj.2000.2249) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10924311/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=European%20Heart%20Journal&title=The%20potential%20for%20QT%20prolongation%20and%20proarrhythmia%20by%20non-%20antiarrhythmic%20drugs:%20clinical%20and%20regulatory%20implications.%20Report%20on%20a%20policy%20conference%20of%20the%20European%20Society%20of%20Cardiology&author=W%20Haverkamp&author=G%20Breithardt&author=AJ%20Camm&author=MJ%20Janse&author=MR%20Rosen&volume=21&publication_year=2000&pages=1216-1231&pmid=10924311&doi=10.1053/euhj.2000.2249&)

3. Roden DM, Viswanathan PC (2005) Genetics of acquired long QT syndrome. J ClinInvest 115: 2025–2032.  [DOI](https://doi.org/10.1172/JCI25539) | [PMC free article](/articles/PMC1180553/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16075043/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20ClinInvest&title=Genetics%20of%20acquired%20long%20QT%20syndrome&author=DM%20Roden&author=PC%20Viswanathan&volume=115&publication_year=2005&pages=2025-2032&pmid=16075043&doi=10.1172/JCI25539&)

4. Roden DM (2004) Drug-induced prolongation of the QT Interval. New England Journal of Medicine 350: 1013–1022.  [DOI](https://doi.org/10.1056/NEJMra032426) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14999113/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=New%20England%20Journal%20of%20Medicine&title=Drug-induced%20prolongation%20of%20the%20QT%20Interval&author=DM%20Roden&volume=350&publication_year=2004&pages=1013-1022&pmid=14999113&doi=10.1056/NEJMra032426&)

5. The Psychiatrist (2004) DARE: the Drug-Induced Arrhythmia Risk Evaluation Study. The Psychiatrist 28: 431.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=The%20Psychiatrist&title=DARE:%20the%20Drug-Induced%20Arrhythmia%20Risk%20Evaluation%20Study&author=Psychiatrist%20The&volume=28&publication_year=2004&pages=431&)

6. Yang P, Kanki H, Drolet B, Yang T, Wei J, et al. (2002) Allelic variants in Long QT disease genes in patients with drug-associated Torsades de Pointes. Circulation 105: 1943–1948.  [DOI](https://doi.org/10.1161/01.cir.0000014448.19052.4c) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11997281/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circulation&title=Allelic%20variants%20in%20Long%20QT%20disease%20genes%20in%20patients%20with%20drug-associated%20Torsades%20de%20Pointes&author=P%20Yang&author=H%20Kanki&author=B%20Drolet&author=T%20Yang&author=J%20Wei&volume=105&publication_year=2002&pages=1943-1948&pmid=11997281&doi=10.1161/01.cir.0000014448.19052.4c&)

7. Paulussen AD, Gilissen RA, Armstrong M, Doevendans PA, Verhasselt P, et al. (2004) Genetic variations of KCNQ1, KCNH2, SCN5A, KCNE1, and KCNE2 in drug-induced long QT syndrome patients. JMolMed 82: 182–188.  [DOI](https://doi.org/10.1007/s00109-003-0522-z) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14760488/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JMolMed&title=Genetic%20variations%20of%20KCNQ1,%20KCNH2,%20SCN5A,%20KCNE1,%20and%20KCNE2%20in%20drug-induced%20long%20QT%20syndrome%20patients&author=AD%20Paulussen&author=RA%20Gilissen&author=M%20Armstrong&author=PA%20Doevendans&author=P%20Verhasselt&volume=82&publication_year=2004&pages=182-188&pmid=14760488&doi=10.1007/s00109-003-0522-z&)

8. Itoh H, Sakaguchi T, Ding WG, Watanabe E, Watanabe I, et al. (2009) Latent Genetic Backgrounds and Molecular Pathogenesis in Drug-induced Long QT Syndrome. Circulation: Arrhythmia and Electrophysiology 2: 511–523.  [DOI](https://doi.org/10.1161/CIRCEP.109.862649) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19843919/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circulation:%20Arrhythmia%20and%20Electrophysiology&title=Latent%20Genetic%20Backgrounds%20and%20Molecular%20Pathogenesis%20in%20Drug-induced%20Long%20QT%20Syndrome&author=H%20Itoh&author=T%20Sakaguchi&author=WG%20Ding&author=E%20Watanabe&author=I%20Watanabe&volume=2&publication_year=2009&pages=511-523&pmid=19843919&doi=10.1161/CIRCEP.109.862649&)

9. Kannankeril PJ, Roden DM, Norris KJ, Whalen SP, George AL Jr, et al. (2005) Genetic susceptibility to acquired long QT syndrome: pharmacologic challenge in first-degree relatives. Heart Rhythm 2: 134–140.  [DOI](https://doi.org/10.1016/j.hrthm.2004.10.039) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15851285/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Heart%20Rhythm&title=Genetic%20susceptibility%20to%20acquired%20long%20QT%20syndrome:%20pharmacologic%20challenge%20in%20first-degree%20relatives&author=PJ%20Kannankeril&author=DM%20Roden&author=KJ%20Norris&author=SP%20Whalen&author=AL%20George&volume=2&publication_year=2005&pages=134-140&pmid=15851285&doi=10.1016/j.hrthm.2004.10.039&)

10. Hanson B, Tuna N, Bouchard T, Heston L, Eckert E, et al. (1989) Genetic factors in the electrocardiogram and heart rate of twins reared apart and together. Am J Cardiol 63: 606–609.  [DOI](https://doi.org/10.1016/0002-9149(89)90907-7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/2919564/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Cardiol&title=Genetic%20factors%20in%20the%20electrocardiogram%20and%20heart%20rate%20of%20twins%20reared%20apart%20and%20together&author=B%20Hanson&author=N%20Tuna&author=T%20Bouchard&author=L%20Heston&author=E%20Eckert&volume=63&publication_year=1989&pages=606-609&pmid=2919564&doi=10.1016/0002-9149(89)90907-7&)

11. Akylbekova EL, Crow RS, Johnson WD, Buxbaum SG, Njemanze S, et al. (2009) Clinical Correlates and Heritability of QT Interval Duration in Blacks: The Jackson Heart Study. Circulation: Arrhythmia and Electrophysiology 2: 427–432.  [DOI](https://doi.org/10.1161/CIRCEP.109.858894) | [PMC free article](/articles/PMC2772163/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19808499/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circulation:%20Arrhythmia%20and%20Electrophysiology&title=Clinical%20Correlates%20and%20Heritability%20of%20QT%20Interval%20Duration%20in%20Blacks:%20The%20Jackson%20Heart%20Study&author=EL%20Akylbekova&author=RS%20Crow&author=WD%20Johnson&author=SG%20Buxbaum&author=S%20Njemanze&volume=2&publication_year=2009&pages=427-432&pmid=19808499&doi=10.1161/CIRCEP.109.858894&)

12. Carter N, Snieder H, Jeffery S, Saumarez R, Varma C, et al. (2000) QT interval in twins. JHumHypertens 14: 389–390.  [DOI](https://doi.org/10.1038/sj.jhh.1001026) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10878701/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JHumHypertens&title=QT%20interval%20in%20twins&author=N%20Carter&author=H%20Snieder&author=S%20Jeffery&author=R%20Saumarez&author=C%20Varma&volume=14&publication_year=2000&pages=389-390&pmid=10878701&doi=10.1038/sj.jhh.1001026&)

13. Newton-Cheh C, Larson MG, Corey DC, Benjamin EJ, Herbert AG, et al. (2005) QT interval is a heritable quantitative trait with evidence of linkage to chromosome 3 in a genome-wide linkage analysis: The Framingham Heart Study. Heart Rhythm 2: 277–284.  [DOI](https://doi.org/10.1016/j.hrthm.2004.11.009) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15851319/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Heart%20Rhythm&title=QT%20interval%20is%20a%20heritable%20quantitative%20trait%20with%20evidence%20of%20linkage%20to%20chromosome%203%20in%20a%20genome-wide%20linkage%20analysis:%20The%20Framingham%20Heart%20Study&author=C%20Newton-Cheh&author=MG%20Larson&author=DC%20Corey&author=EJ%20Benjamin&author=AG%20Herbert&volume=2&publication_year=2005&pages=277-284&pmid=15851319&doi=10.1016/j.hrthm.2004.11.009&)

14. Pfeufer A, Sanna S, Arking DE, Muller M, Gateva V, et al. (2009) Common variants at ten loci modulate the QT interval duration in the QTSCD Study. Nature Genetics 41: 407–414.  [DOI](https://doi.org/10.1038/ng.362) | [PMC free article](/articles/PMC2976045/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19305409/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature%20Genetics&title=Common%20variants%20at%20ten%20loci%20modulate%20the%20QT%20interval%20duration%20in%20the%20QTSCD%20Study&author=A%20Pfeufer&author=S%20Sanna&author=DE%20Arking&author=M%20Muller&author=V%20Gateva&volume=41&publication_year=2009&pages=407-414&pmid=19305409&doi=10.1038/ng.362&)

15. Newton-Cheh C, Eijgelsheim M, Rice KM, de Bakker PIW, Yin X, et al. (2009) Common variants at ten loci influence QT interval duration in the QTGEN Study. Nature Genetics 41: 399–406.  [DOI](https://doi.org/10.1038/ng.364) | [PMC free article](/articles/PMC2701449/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19305408/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature%20Genetics&title=Common%20variants%20at%20ten%20loci%20influence%20QT%20interval%20duration%20in%20the%20QTGEN%20Study&author=C%20Newton-Cheh&author=M%20Eijgelsheim&author=KM%20Rice&author=PIW%20de%20Bakker&author=X%20Yin&volume=41&publication_year=2009&pages=399-406&pmid=19305408&doi=10.1038/ng.364&)

16. Daly AK, Donaldson PT, Bhatnagar P, Shen Y, Pe'er I, et al. (2009) HLA-B*5701 genotype is a major determinant of drug-induced liver injury due to flucloxacillin. NatGenet 41: 816–819.  [DOI](https://doi.org/10.1038/ng.379) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19483685/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=NatGenet&title=HLA-B*5701%20genotype%20is%20a%20major%20determinant%20of%20drug-induced%20liver%20injury%20due%20to%20flucloxacillin&author=AK%20Daly&author=PT%20Donaldson&author=P%20Bhatnagar&author=Y%20Shen&author=I%20Pe'er&volume=41&publication_year=2009&pages=816-819&pmid=19483685&doi=10.1038/ng.379&)

17. McCormack M, Alfirevic A, Bourgeois S, Farrell JJ, Kasperaviciute D, et al. (2011) HLA-A*3101 and Carbamazepine-Induced Hypersensitivity Reactions in Europeans. New England Journal of Medicine 364: 1134–1143.  [DOI](https://doi.org/10.1056/NEJMoa1013297) | [PMC free article](/articles/PMC3113609/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21428769/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=New%20England%20Journal%20of%20Medicine&title=HLA-A*3101%20and%20Carbamazepine-Induced%20Hypersensitivity%20Reactions%20in%20Europeans&author=M%20McCormack&author=A%20Alfirevic&author=S%20Bourgeois&author=JJ%20Farrell&author=D%20Kasperaviciute&volume=364&publication_year=2011&pages=1134-1143&pmid=21428769&doi=10.1056/NEJMoa1013297&)

18. Link E, Parish S, Armitage J, Bowman L, Heath S, et al. (2008) SLCO1B1 variants and statin-induced myopathy--a genomewide study. N Engl J Med 359: 789–799.  [DOI](https://doi.org/10.1056/NEJMoa0801936) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18650507/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=SLCO1B1%20variants%20and%20statin-induced%20myopathy--a%20genomewide%20study&author=E%20Link&author=S%20Parish&author=J%20Armitage&author=L%20Bowman&author=S%20Heath&volume=359&publication_year=2008&pages=789-799&pmid=18650507&doi=10.1056/NEJMoa0801936&)

19. Nelson MR, Bryc K, King KS, Indap A, Boyko AR, et al. (2008) The Population Reference Sample, POPRES: a resource for population, disease, and pharmacological genetics research. Am J HumGenet 83: 347–358.  [DOI](https://doi.org/10.1016/j.ajhg.2008.08.005) | [PMC free article](/articles/PMC2556436/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18760391/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20HumGenet&title=The%20Population%20Reference%20Sample,%20POPRES:%20a%20resource%20for%20population,%20disease,%20and%20pharmacological%20genetics%20research&author=MR%20Nelson&author=K%20Bryc&author=KS%20King&author=A%20Indap&author=AR%20Boyko&volume=83&publication_year=2008&pages=347-358&pmid=18760391&doi=10.1016/j.ajhg.2008.08.005&)

20. McCarthy MI, Abecasis GR, Cardon LR, Goldstein DB, Little J, et al. (2008) Genome-wide association studies for complex traits: consensus, uncertainty and challenges. NatRevGenet 9: 356–369.  [DOI](https://doi.org/10.1038/nrg2344) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18398418/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=NatRevGenet&title=Genome-wide%20association%20studies%20for%20complex%20traits:%20consensus,%20uncertainty%20and%20challenges&author=MI%20McCarthy&author=GR%20Abecasis&author=LR%20Cardon&author=DB%20Goldstein&author=J%20Little&volume=9&publication_year=2008&pages=356-369&pmid=18398418&doi=10.1038/nrg2344&)

21. Lucena MI, Molokhia M, Shen Y, Urban TJ, Aithal GP, et al. (2011) Susceptibility to Amoxicillin-Clavulanate-Induced Liver Injury Is Influenced by Multiple HLA Class I and II Alleles. Gastroenterology 141: 338–347.  [DOI](https://doi.org/10.1053/j.gastro.2011.04.001) | [PMC free article](/articles/PMC3129430/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21570397/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gastroenterology&title=Susceptibility%20to%20Amoxicillin-Clavulanate-Induced%20Liver%20Injury%20Is%20Influenced%20by%20Multiple%20HLA%20Class%20I%20and%20II%20Alleles&author=MI%20Lucena&author=M%20Molokhia&author=Y%20Shen&author=TJ%20Urban&author=GP%20Aithal&volume=141&publication_year=2011&pages=338-347&pmid=21570397&doi=10.1053/j.gastro.2011.04.001&)

22. Purcell S, Neale B, Todd-Brown K, Thomas L, Ferreira MA, et al. (2007) PLINK: a tool set for whole-genome association and population-based linkage analyses. AmJHum Genet 81: 559–575.  [DOI](https://doi.org/10.1086/519795) | [PMC free article](/articles/PMC1950838/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17701901/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=AmJHum%20Genet&title=PLINK:%20a%20tool%20set%20for%20whole-genome%20association%20and%20population-based%20linkage%20analyses&author=S%20Purcell&author=B%20Neale&author=K%20Todd-Brown&author=L%20Thomas&author=MA%20Ferreira&volume=81&publication_year=2007&pages=559-575&pmid=17701901&doi=10.1086/519795&)

23. Price AL, Patterson NJ, Plenge RM, Weinblatt ME, Shadick NA, et al. (2006) Principal components analysis corrects for stratification in genome-wide association studies. Nat Genet 38: 904–909.  [DOI](https://doi.org/10.1038/ng1847) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16862161/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Principal%20components%20analysis%20corrects%20for%20stratification%20in%20genome-wide%20association%20studies&author=AL%20Price&author=NJ%20Patterson&author=RM%20Plenge&author=ME%20Weinblatt&author=NA%20Shadick&volume=38&publication_year=2006&pages=904-909&pmid=16862161&doi=10.1038/ng1847&)

24. Novembre J, Johnson T, Bryc K, Kutalik Z, Boyko AR, et al. (2008) Genes mirror geography within Europe. Nature 456: 98–101.  [DOI](https://doi.org/10.1038/nature07331) | [PMC free article](/articles/PMC2735096/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18758442/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Genes%20mirror%20geography%20within%20Europe&author=J%20Novembre&author=T%20Johnson&author=K%20Bryc&author=Z%20Kutalik&author=AR%20Boyko&volume=456&publication_year=2008&pages=98-101&pmid=18758442&doi=10.1038/nature07331&)

25. Chapman JM, Cooper JD, Todd JA, Clayton DG (2003) Detecting disease associations due to linkage disequilibrium using haplotype tags: a class of tests and the determinants of statistical power. Hum Hered 56: 18–31.  [DOI](https://doi.org/10.1159/000073729) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14614235/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Hered&title=Detecting%20disease%20associations%20due%20to%20linkage%20disequilibrium%20using%20haplotype%20tags:%20a%20class%20of%20tests%20and%20the%20determinants%20of%20statistical%20power&author=JM%20Chapman&author=JD%20Cooper&author=JA%20Todd&author=DG%20Clayton&volume=56&publication_year=2003&pages=18-31&pmid=14614235&doi=10.1159/000073729&)

26. Marchini J, Howie B (2010) Genotype imputation for genome-wide association studies. Nat Rev Genet 11: 499–511.  [DOI](https://doi.org/10.1038/nrg2796) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20517342/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Genet&title=Genotype%20imputation%20for%20genome-wide%20association%20studies&author=J%20Marchini&author=B%20Howie&volume=11&publication_year=2010&pages=499-511&pmid=20517342&doi=10.1038/nrg2796&)

27. Tatonetti NP, Dudley JT, Sagreiya H, Butte AJ, Altman RB (2010) An integrative method for scoring candidate genes from association studies: application to warfarin dosing. BMC Bioinformatics 11 Suppl 9S9.  [DOI](https://doi.org/10.1186/1471-2105-11-S9-S9) | [PMC free article](/articles/PMC2967750/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21044367/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Bioinformatics&title=An%20integrative%20method%20for%20scoring%20candidate%20genes%20from%20association%20studies:%20application%20to%20warfarin%20dosing&author=NP%20Tatonetti&author=JT%20Dudley&author=H%20Sagreiya&author=AJ%20Butte&author=RB%20Altman&volume=11&publication_year=2010&pages=S9&pmid=21044367&doi=10.1186/1471-2105-11-S9-S9&)

28. Gudbjartsson DF, Arnar DO, Helgadottir A, Gretarsdottir S, Holm H, et al. (2007) Variants conferring risk of atrial fibrillation on chromosome 4q25. Nature 448: 353–357.  [DOI](https://doi.org/10.1038/nature06007) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17603472/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Variants%20conferring%20risk%20of%20atrial%20fibrillation%20on%20chromosome%204q25&author=DF%20Gudbjartsson&author=DO%20Arnar&author=A%20Helgadottir&author=S%20Gretarsdottir&author=H%20Holm&volume=448&publication_year=2007&pages=353-357&pmid=17603472&doi=10.1038/nature06007&)

29. Mohler PJ, Le Scouarnec S, Denjoy I, Lowe JS, Guicheney P, et al. (2007) Defining the Cellular Phenotype of "Ankyrin-B Syndrome" Variants: Human ANK2 Variants Associated With Clinical Phenotypes Display a Spectrum of Activities in Cardiomyocytes. Circulation 115: 432–441.  [DOI](https://doi.org/10.1161/CIRCULATIONAHA.106.656512) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17242276/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circulation&title=Defining%20the%20Cellular%20Phenotype%20of%20%22Ankyrin-B%20Syndrome%22%20Variants:%20Human%20ANK2%20Variants%20Associated%20With%20Clinical%20Phenotypes%20Display%20a%20Spectrum%20of%20Activities%20in%20Cardiomyocytes&author=PJ%20Mohler&author=S%20Le%20Scouarnec&author=I%20Denjoy&author=JS%20Lowe&author=P%20Guicheney&volume=115&publication_year=2007&pages=432-441&pmid=17242276&doi=10.1161/CIRCULATIONAHA.106.656512&)

30. Hoppe UC, Marban E, Johns DC (2001) Distinct gene-specific mechanisms of arrhythmia revealed by cardiac gene transfer of two long QT disease genes, HERG and KCNE1. Proceedings of the National Academy of Sciences 98: 5335–5340.  [DOI](https://doi.org/10.1073/pnas.091239098) | [PMC free article](/articles/PMC33210/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11320260/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%20National%20Academy%20of%20Sciences&title=Distinct%20gene-specific%20mechanisms%20of%20arrhythmia%20revealed%20by%20cardiac%20gene%20transfer%20of%20two%20long%20QT%20disease%20genes,%20HERG%20and%20KCNE1&author=UC%20Hoppe&author=E%20Marban&author=DC%20Johns&volume=98&publication_year=2001&pages=5335-5340&pmid=11320260&doi=10.1073/pnas.091239098&)

31. Pirmohamed M, Aithal GP, Behr E, Daly A, Roden D (2011) The phenotype standardization project: improving pharmacogenetic studies of serious adverse drug reactions. Clin Pharmacol Ther 89: 784–785.  [DOI](https://doi.org/10.1038/clpt.2011.30) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21593754/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=The%20phenotype%20standardization%20project:%20improving%20pharmacogenetic%20studies%20of%20serious%20adverse%20drug%20reactions&author=M%20Pirmohamed&author=GP%20Aithal&author=E%20Behr&author=A%20Daly&author=D%20Roden&volume=89&publication_year=2011&pages=784-785&pmid=21593754&doi=10.1038/clpt.2011.30&)

32. Kaab S, Crawford DC, Sinner MF, Behr ER, Kannankeril PJ, et al. (2012) A large candidate gene survey identifies the KCNE1 D85N polymorphism as a possible modulator of drug-induced torsades de pointes. Circ Cardiovasc Genet 5: 91–99.  [DOI](https://doi.org/10.1161/CIRCGENETICS.111.960930) | [PMC free article](/articles/PMC3288202/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22100668/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circ%20Cardiovasc%20Genet&title=A%20large%20candidate%20gene%20survey%20identifies%20the%20KCNE1%20D85N%20polymorphism%20as%20a%20possible%20modulator%20of%20drug-induced%20torsades%20de%20pointes&author=S%20Kaab&author=DC%20Crawford&author=MF%20Sinner&author=ER%20Behr&author=PJ%20Kannankeril&volume=5&publication_year=2012&pages=91-99&pmid=22100668&doi=10.1161/CIRCGENETICS.111.960930&)

33. Roden DM, Kannankeril P, Darbar D (2007) On the relationship among QT interval, atrial fibrillation, and torsade de pointes. Europace 9 Suppl 4iv1–iv3.  [DOI](https://doi.org/10.1093/europace/eum165) | [PMC free article](/articles/PMC2599908/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17766319/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Europace&title=On%20the%20relationship%20among%20QT%20interval,%20atrial%20fibrillation,%20and%20torsade%20de%20pointes&author=DM%20Roden&author=P%20Kannankeril&author=D%20Darbar&volume=9&publication_year=2007&pages=iv1-iv3&pmid=17766319&doi=10.1093/europace/eum165&)

34. Darbar D, Hardin B, Harris P, Roden DM (2007) A rate-independent method of assessing QT-RR slope following conversion of atrial fibrillation. J CardiovascElectrophysiol 18: 636–641.  [DOI](https://doi.org/10.1111/j.1540-8167.2007.00817.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17488270/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20CardiovascElectrophysiol&title=A%20rate-independent%20method%20of%20assessing%20QT-RR%20slope%20following%20conversion%20of%20atrial%20fibrillation&author=D%20Darbar&author=B%20Hardin&author=P%20Harris&author=DM%20Roden&volume=18&publication_year=2007&pages=636-641&pmid=17488270&doi=10.1111/j.1540-8167.2007.00817.x&)

35. Salerno DM, Katz A, Dunbar DN, Fjeldos-Sperbeck K (1993) Serum electrolytes and catecholamines after cardioversion from ventricular tachycardia and atrial fibrillation. Pacing and Clinical Electrophysiology 16: 1862–1871.  [DOI](https://doi.org/10.1111/j.1540-8159.1993.tb01821.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7692419/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pacing%20and%20Clinical%20Electrophysiology&title=Serum%20electrolytes%20and%20catecholamines%20after%20cardioversion%20from%20ventricular%20tachycardia%20and%20atrial%20fibrillation&author=DM%20Salerno&author=A%20Katz&author=DN%20Dunbar&author=K%20Fjeldos-Sperbeck&volume=16&publication_year=1993&pages=1862-1871&pmid=7692419&doi=10.1111/j.1540-8159.1993.tb01821.x&)

36. Arakawa M, Miwa H, Kambara K, Ohno M, Kagawa K, et al. (1992) Changes in plasma concentrations of atrial natriuretic peptides after cardioversion of chronic atrial fibrillation. American Journal of Cardiology 70: 550–552.  [DOI](https://doi.org/10.1016/0002-9149(92)91211-l) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/1386493/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=American%20Journal%20of%20Cardiology&title=Changes%20in%20plasma%20concentrations%20of%20atrial%20natriuretic%20peptides%20after%20cardioversion%20of%20chronic%20atrial%20fibrillation&author=M%20Arakawa&author=H%20Miwa&author=K%20Kambara&author=M%20Ohno&author=K%20Kagawa&volume=70&publication_year=1992&pages=550-552&pmid=1386493&doi=10.1016/0002-9149(92)91211-l&)

37. Sanguinetti MC, Bennett PB (2003) Antiarrhythmic drug target choices and screening. Circulation Research 96: 491–499.  [DOI](https://doi.org/10.1161/01.RES.0000091829.63501.A8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14500332/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circulation%20Research&title=Antiarrhythmic%20drug%20target%20choices%20and%20screening&author=MC%20Sanguinetti&author=PB%20Bennett&volume=96&publication_year=2003&pages=491-499&pmid=14500332&doi=10.1161/01.RES.0000091829.63501.A8&)

38. Lu Z, Wu CY, Jiang YP, Ballou LM, Clausen C, et al. (2012) Suppression of Phosphoinositide 3-Kinase Signaling and Alteration of Multiple Ion Currents in Drug-Induced Long QT Syndrome. Sci Transl Med 4: 131ra150.  [DOI](https://doi.org/10.1126/scitranslmed.3003623) | [PMC free article](/articles/PMC3494282/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22539774/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Sci%20Transl%20Med&title=Suppression%20of%20Phosphoinositide%203-Kinase%20Signaling%20and%20Alteration%20of%20Multiple%20Ion%20Currents%20in%20Drug-Induced%20Long%20QT%20Syndrome&author=Z%20Lu&author=CY%20Wu&author=YP%20Jiang&author=LM%20Ballou&author=C%20Clausen&volume=4&publication_year=2012&pages=131ra150&pmid=22539774&doi=10.1126/scitranslmed.3003623&)

39. Jamshidi Y, Nolte IM, Dalageorgou C, Zheng D, Johnson T, et al.. (2012) Common variation in the NOS1AP gene is associated with drug-induced QT prolongation and ventricular arrhythmia. Journal of the American College of Cardiology accepted for publication.  [DOI](https://doi.org/10.1016/j.jacc.2012.03.031) | [PMC free article](/articles/PMC3753216/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22682551/)

40. Wilke RA, Lin DW, Roden DM, Watkins PB, Flockhart D, et al. (2007) Identifying genetic risk factors for serious adverse drug reactions: current progress and challenges. Nat Rev Drug Discov 6: 904–916.  [DOI](https://doi.org/10.1038/nrd2423) | [PMC free article](/articles/PMC2763923/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17971785/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Drug%20Discov&title=Identifying%20genetic%20risk%20factors%20for%20serious%20adverse%20drug%20reactions:%20current%20progress%20and%20challenges&author=RA%20Wilke&author=DW%20Lin&author=DM%20Roden&author=PB%20Watkins&author=D%20Flockhart&volume=6&publication_year=2007&pages=904-916&pmid=17971785&doi=10.1038/nrd2423&)

41. Splawski I, Timothy KW, Tateyama M, Clancy CE, Malhotra A, et al. (2002) Variant of SCN5A sodium channel implicated in risk of cardiac arrhythmia. Science 297: 1333–1336.  [DOI](https://doi.org/10.1126/science.1073569) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12193783/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=Variant%20of%20SCN5A%20sodium%20channel%20implicated%20in%20risk%20of%20cardiac%20arrhythmia&author=I%20Splawski&author=KW%20Timothy&author=M%20Tateyama&author=CE%20Clancy&author=A%20Malhotra&volume=297&publication_year=2002&pages=1333-1336&pmid=12193783&doi=10.1126/science.1073569&)

42. Plant LD, Bowers PN, Liu Q, Morgan T, Zhang T, et al. (2006) A common cardiac sodium channel variant associated with sudden infant death in African Americans, SCN5A S1103Y. Journal of Clinical Investigation 116: 430–435.  [DOI](https://doi.org/10.1172/JCI25618) | [PMC free article](/articles/PMC1359045/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16453024/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Clinical%20Investigation&title=A%20common%20cardiac%20sodium%20channel%20variant%20associated%20with%20sudden%20infant%20death%20in%20African%20Americans,%20SCN5A%20S1103Y&author=LD%20Plant&author=PN%20Bowers&author=Q%20Liu&author=T%20Morgan&author=T%20Zhang&volume=116&publication_year=2006&pages=430-435&pmid=16453024&doi=10.1172/JCI25618&)

43. Van Norstrand DW, Tester DJ, Ackerman MJ (2008) Overrepresentation of the proarrhythmic, sudden death predisposing sodium channel polymorphism S1103Y in a population-based cohort of African-American sudden infant death syndrome. Heart Rhythm 5: 712–715.  [DOI](https://doi.org/10.1016/j.hrthm.2008.02.012) | [PMC free article](/articles/PMC2430043/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18452875/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Heart%20Rhythm&title=Overrepresentation%20of%20the%20proarrhythmic,%20sudden%20death%20predisposing%20sodium%20channel%20polymorphism%20S1103Y%20in%20a%20population-based%20cohort%20of%20African-American%20sudden%20infant%20death%20syndrome&author=DW%20Van%20Norstrand&author=DJ%20Tester&author=MJ%20Ackerman&volume=5&publication_year=2008&pages=712-715&pmid=18452875&doi=10.1016/j.hrthm.2008.02.012&)
