# Genetic and Epigenetic Regulation of the Organic Cation Transporter 3, SLC22A3

## Metadata
**Authors:** Ligong Chen, Chibo Hong, Eugene C Chen, Sook Wah Yee, Lu Xu, Emil U Almof, Christopher Wen, Kacyn Fujii, Susan J Johns, Doug Stryke, Thomas E Ferrin, Jeff Simko, Xin Chen, Joseph F Costello, Kathleen M Giacomini
**Journal:** The pharmacogenomics journal
**Date:** 2012 Jan 10
**DOI:** [10.1038/tpj.2011.60](https://doi.org/10.1038/tpj.2011.60)
**PMID:** 22231567
**PMCID:** PMC3396779
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3396779/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC3396779/pdf/nihms341052.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC3396779/pdf/nihms341052.pdf)

## Abstract

Human organic cation transporter 3 (OCT3, SLC22A3) mediates the uptake of many important endogenous amines and basic drugs in a variety of tissues. OCT3 is identified as one of the important risk loci for prostate cancer and is markedly under-expressed in aggressive prostate cancers. The goal of this study was to identify genetic and epigenetic factors in the promoter region that influence the expression level of OCT3. Haplotypes that contained the common variants, g.-81G>delGA (rs60515630) (minor allele frequency (MAF) 11.5% in African American) and g.-2G>A (rs555754) (MAF>30% in all ethnic groups) showed significant increases in luciferase reporter activities and exhibited stronger transcription factor binding affinity than the haplotypes that contained the major alleles. Consistent with the reporter assays, OCT3 mRNA expression levels were significantly higher in Asian (P<0.001) and Caucasian (P<0.05) liver samples from individuals who were homozygous for g.-2A/A in comparison with those homozygous for the g.-2G/G allele. Studies revealed that the methylation level in the basal promoter region of OCT3 was associated with OCT3 expression level and tumorigenesis capability in various prostate cancer cell lines. The methylation level of the OCT3 promoter was higher in 62% of prostate tumor samples compared with matched normal samples. Our studies demonstrate that genetic polymorphisms in the proximal promoter region of OCT3 alter the transcription rate of the gene and may be associated with altered expression levels of OCT3 in human liver. Aberrant methylation contributes to the reduced expression of OCT3 in prostate cancer.

Keywords: SLC22A3, Polymorphism, Methylation and Prostate Cancer

## Introduction

The organic cation transporter 3, *OCT3* (*SLC22A3*) is increasingly being recognized as an important modulator of human disease and drug response. Recent genome-wide association studies (GWAS) have linked *OCT3* to risk loci for prostate cancer, colorectal cancer as well as for other diseases.^1–6^ In prostate cancer, *OCT3,* which is markedly under-expressed in high-Gleason grade tumors, has been found to be among the lowest expressed genes; in addition, OCT3 expression is associated with progression of prostate cancer.^4,5^ For example, studies have suggested that *OCT3* may serve as an important biomarker for Gleason grade.^4,5^ Because it transports many important monoamines including serotonin, histamine and norepinephrine.^7–10^ *OCT3* appears to play a pleiotropic role in human physiology and pathophysiology.

Although the expression of its paralogs, *OCT1* (*SLC22A1*) and *OCT2* (*SLC22A2*), is restricted mainly to excretory organs such as the liver and kidney, *OCT3* (*SLC22A3*) exhibits a much broader tissue distribution.^7,8,11^ Expressed in heart, brain, liver, skeletal muscle, prostate and placenta, *OCT3* plays a role in the disposition of a variety of cationic substances including endogenous amines and therapeutic and illicit drugs.^7, 11–13^ Important SNPs of *OCT3* have been identified and associated with its mRNA level or risk for prostate cancer, colorectal cancer, coronary artery disease and other human disease.^1–3,6,11,14,15^ Though several recent studies have investigated the effects of promoter region variants in transporter genes on transcription rates, gene expression and drug response,^16–19^ there has been no systematic examination of the genetic variants in the promoter region of *OCT3*.

Epigenetic gene silencing through DNA methylation is one of the important steps in tumorigenesis.^20–26^ Of the various epigenetic modifications, hypermethylation of the promoter regions of tumor suppressor genes, which represses transcription, has been most extensively studied.^21,22,27^ DNA methylation plays a central role in the tissue-specific expression of transporters and regulates the expression of certain transporters in cancer cell lines.^28–32^ *OCT3* shows significantly lower expression levels in several types of cancers, especially in high-Gleason grade prostate cancer.^4,5,13^ However the mechanisms responsible for the reduced expression levels of *OCT3* are not understood. The basal promoter of *OCT3* is located within a large CpG island extending into exon 1 ([Figure S1](#SD1)); aberrant methylation of this region could reduce the expression of *OCT3*.

In this investigation, we examined polymorphisms in the basal promoter region of *OCT3* using data from direct sequencing of a large DNA sample set (n=272) from four racial/ethnic populations to identify *OCT3* variants. The functional significance of *OCT3* promoter variants in luciferase reporter assays was assessed. Further, several prostate cancer cell lines and primary cancer tissues were screened to determine whether methylation in the *OCT3* promoter region results in the under-expression of OCT3 in prostate cancer. Our results demonstrate that a common variant g.-2G>A (rs555754) increases the transcription of *OCT3* in luciferase reporter assays and is associated with high expression levels of *OCT3* in primary human liver tissues. In addition, our results show that epigenetic silencing in the *OCT3* promoter region is an important mechanism to suppress the expression of *OCT3* in several prostate cancer cell lines and tissues. These observations strongly suggest that both genetic and epigenetic factors critically mediate the expression level of *OCT3* and may be important in tumorigenesis and drug responses.

## Method and Materials

### Tissue and cell lines

Primary human prostate healthy and tumor samples were from Caucasian individuals and obtained from the Comprehensive Cancer Institute Tissue Center at the University of California at San Francisco. The Gleason grade of the prostate cancer tissue varies from 4 to 5 examined by histology in the Center. Normal Caucasian liver samples were purchased from Asterand (Detroit, MI) and Capital Biosciences (Rockville, MD). The age of the donors of the liver samples ranges from 36 to 83 years. All the Asian tumor and non-tumor liver samples were collected from liver resections at The University of Hong Kong. The age of the donors of Asian for liver tissues ranges from 38 to 82 years. All samples were obtained with informed consent, and the usage was approved by both the Committee on Human Research at the University of California at San Francisco and the Ethics Committee of the University of Hong Kong. Information concerning the tissue donors is listed in the [supplemental Table 2S](#SD1). For determination of OCT3 expression levels, we used 40 samples from Asians and 29 from Caucasians. In methylation studies, 12 liver samples and 16 prostate samples were used. Five and seven cell lines were used for luciferase assay and methylation studies, respectively. All the cell lines were purchased from ATCC and validated by the UCSF Cell Culture Facility.

### Genetic analysis

Genomic DNA samples were collected from unrelated healthy individuals from four major ethnic groups (68 each from European Americans, African Americans, Chinese Americans, and Mexican Americans) as a part of the Study Of PHarmacogenetics In Ethnically-diverse populations (SOPHIE). To identify polymorphisms in the promoter region of OCT3, a polymerase chain reaction (PCR) fragment (−706 to +223 bp from the translational start site) was amplified and directly sequenced by an automated genetic analyzer using the primers listed in [supplemental Table 1](#SD1).^17^ OCT3 genetic variants and their frequencies are listed in [Table 1A](#T1). Haplotype assembly was performed using the Haploview 4.1 program (Broad Institute, Cambridge, MA), based on a standard expectation–maximization algorithm, to reconstruct individual haplotypes from population genotype data. The haplotypes and their frequencies are listed in [Table 1B](#T2). Nucleotide location number was assigned from the translational start site (TSS) according to the *OCT3* mRNA sequence (GenBank accession number: [NM_021977.2](https://www.ncbi.nlm.nih.gov/nuccore/NM_021977.2)). For genotyping the liver samples from Caucasian and Asian donors, the same primers for cloning the OCT3 with luciferase ([Table S1](#SD1)) were used to amplify the PCR products. The PCR products were processed with ExoSAP-IT (Affymetrix, Santa Clara, CA) for Clean-Up and then were directly sequenced with the forward primer.

### Table 1A.

| dbSNP ID | Genomic Position (hg18) | Transcription position | Nucleotide change | Allele Frequency |  |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- |
| AA (n=136) | EA (n=136) | AS (n=136) | ME (n=136) |  |  |  |  |
| rs58601841 | 160689269 | −146 | C>G | 0.028 | 0.000 | 0.009 | 0.000 |
| rs59711975 | 160689291 | −124 | C>G | 0.019 | 0.000 | 0.000 | 0.000 |
| rs60515630 | 160689334 | −81 | G>delGA | 0.115 | 0.007 | 0.000 | 0.015 |
| rs555754 | 160689413 | −2 | G>A | 0.554 | 0.50000 | 0.323 | 0.385 |
| N/A | 160689416 | 2 | T →insGCGGGCG | 0.000 | 0.000 | 0.077 | 0.000 |

Table 1 Caption: Allele frequencies OCT3 variants in the proximal promoter and 5′-UTR. Data were obtained from DNA samples from 272 unrelated individuals including 68 from each of four major ethnic groups. Position of the variant is based upon the translational start site.

### Table 1B.

| ID | g. -146 | g. -124 | g. -81 | g. −2 | g.+2 | % Haplotypes |  |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| AA | EA | AS | ME |  |  |  |  |  |  |
| H1 | C | C | G | G | T | 44.6 | 50 | 59.2 | 61.5 |
| H2 | G | C | G | G | T | 0 | 0 | 0.8 | 0 |
| H3 | G | C | G | A | T | 2.3 | 0 | 0 | 0 |
| H4 | C | G | G | G | T | 1.5 | 0 | 0 | 0 |
| H5 | C | G | delGA | A | T | 11.5 | 0.7 | 0 | 1.5 |
| H6 | C | C | G | A | T | 40 | 49.3 | 32.3 | 36.9 |
| H7 | C | C | G | G | insGCGGGCG | 0 | 0 | 7.7 | 0 |

Table 1 Caption: Frequencies of the common haplotypes in the OCT3 promoter region. Data were obtained from DNA samples from 272 unrelated individuals including 68 from each of four major ethnic groups. Position of the variant is based upon TSS. Each variant was underlined and bolded in the haplotypes and its transcriptional position was indicated above them.

### Construction of various reporter plasmids containing OCT3 promoter region variants

To construct the reporter plasmids containing the *OCT3* promoter region, the *OCT3* promoter region were amplified from genomic DNA samples from individuals having each haplotype listed in [Table 1B](#T2). The amplified products were inserted into the pGL4.11 [luc2] vectors using the primers containing the restriction endonuclease sites, NheI and HindIII ([Table S1](#SD1)) (Promega Corporation, Madison, WI) and DNA sequences were confirmed by direct sequencing. Hardy-Weinberg Equilibrium probabilities were calculated at each SNP position of each ethnic population in this resequencing study of the OCT3 promoter region.

### Measurement and analysis of OCT3 promoter activity

Reporter plasmids containing the reference *OCT3* promoter (H1) or its variants were transfected into HCT116 (human colon carcinoma), HepG2 (human liver carcinoma), DU145 and PC3 (Prostate cancer cell lines), A460, A549 and H838 (human lung carcinoma) cell lines using Lipofectamine 2000 reagents (Invitrogen) according to the manufacturer’s protocol. The relative luciferase activity was measured as described in previous studies.^16,17^ Using TFSEARCH: Searching Transcription Factor Binding Sites (ver 1.3) (Computational Biology Research Center, AIST, Japan),^33^ the possible TFs in this region were predicted using a threshold score > 75. The 4 TFs with high scores with single or multiple binding sites in the region were specific protein 1 (Sp1), myeloid zinc finger-1 (MZF1), activating enhancer binding protein 4 (Ap-4) and E1A binding protein (p300) ([Figure S2](#SD1)). To examine the effect of Sp1, MZF1, Ap-4 and p300 (Origene, Rockville, MD) on the promoter activity of *OCT3*, reference (H1) reporter plasmids were co-transfected with increasing amounts of transfection-ready plasmid containing the above TFs (50, 100, 200 and 400 ng). After 24 hours, the same measurement as above was used to detect the luciferase activity. The detection of various TFs with Western blotting is described in a previous study^7^ and the specific antibodies against the TFs were from Santa Cruz Biotechnology (Santa Cruz, CA).

### Electrophoretic mobility shift assay (EMSA) of OCT3

Nuclear protein extracts were prepared from HepG2 cells using the Thermo Scientific NE-PER Nuclear and Cytoplasmic Extraction Kit (Thermo Fisher Scientific Inc, Rockford, IL). Briefly, the cells were pelleted from the experimental conditions by centrifugation at 1,000 × g for 5 min, 4°C. Following the manufacture’s protocol, the nuclear protein was extracted by removing the cytoplasmic protein and freshly prepared for the next step of the EMSA assay.

The digoxigenin (DIG) gel shift kit for 3′-end labeling of oligonucleotides (Roche Applied Science, Indianapolis, IN) was used for protein-DNA binding assays. The oligomers ([Table S1](#SD1)) used contained the studied region with or without variants in the *OCT3* promoter region. The oligomers were annealed, labeled, and used in the gel shift reactions according to the manufacturer’s instructions (Roche). For super-shift experiments, 1 μg of antibodies against Sp1, MZF1, Ap4 and p300 was added to the reaction mixture, respectively. Reactions were analyzed by electrophoresis through Novex 6% DNA retardation gels (Invitrogen). After electrotransfer to a positively charged nylon membrane (Roche), bands were detected non-isotopically with a Dig Gel Shift kit, 2nd Generation.^28^

### Quantification of OCT3 gene expression using quantitative real-time reverse transcription PCR

Using the Allprep DNA-RNA isolation kit (Qiagen), total RNA was isolated from liver samples. Single-strand cDNAs were synthesized using a Superscript III first-strand synthesis system for RT-PCR (Invitrogen). Quantitative real-time reverse transcription (qRT-PCR) was performed using a 7500 Fast RT-PCR system (Applied Biosystems, Foster City, CA) with Taqman gene expression assays as described in the previous study.^7^ To compare the expression in the normal tissue samples or cancerous tissue sample or cell lines, the lowest expressed sample was assigned a value of 1.0 in each group. The relative expression level of the samples was normalized to the expression of hGAPDH ([Supplemental Figure S3](#SD1) for representative cell lines and tissues).

### Methylation and Demethylation Studies

For the *in vitro* methylation study, reporter construct (H1) containing the *OCT3* promoter was methylated *in vitro* with 5 U of SssI methylase (New England Biolabs, Beverly, MA) for each microgram of DNA in the presence of 160 μM *S*-adenosylmethionine at 37°C for 3 h following the manufacturer’s protocol.^28^ The methylated reporter constructs were then used for the luciferase assays.

For the demethylation study with 5-aza-2′-deoxycytidine (5′-azadC) (DNA methylation inhibitor) (Sigma, St. Louis, MO), HCT116 cells were precultured to 50% confluence and then cultured for 72 h in medium containing 0, 1, 10, or 100 μM 5′azadC.^28^ Total RNA was prepared from cells by total RNA kit (Qiagen). The RNA was then reverse-transcribed using a first-strand DNA synthesis kit (Invitrogen). PCR was performed with the forward and reverse primers listed in [Table S1](#SD1). The specific RT-PCR for *OCT3* primers is listed in the [Table S1](#SD1) to detect the fragment cDNA.

### Bisulfite sequencing analysis

DNA was treated with sodium bisulfite as described with modifications.^34^ DNA (2 μg) was first digested with *Eco*RV, phenol/chloroform extracted, ethanol precipitated, and resuspended in water. DNA was treated with bisulfite at 55°C for 4 hours. After purification, DNA was dissolved in 50 μL, and 1 μL was used for PCR. Touchdown PCR was done for a total of 35 cycles starting at initial annealing temperature of 68°C and decreasing by 2°C every two cycles to a final annealing temperature of 58°C for the final 25 cycles. The product was cloned into the TOPO TA cloning/pCR2.1 vector (Invitrogen). Individual bacterial colonies were subject to PCR and the products were sequenced. The primers used for the human *OCT3* CpG island are listed in the [Table S1](#SD1).

### Linkage disequilibrium analysis of promoter variant rs555754

Many studies have described significant associations of genetic variants in *SLC22A3* with *SLC22A3* mRNA expression levels and with human diseases ([supplemental Table S2](#SD1)). Pairwise linkage disequilibrium of rs555754 with the single nucleotide polymorphisms (SNPs) listed in [Tables S2](#SD1) and 2 was determined using the available information from SNAP (SNP Annotation and Proxy Search Version 2.2, Broad Institute, [http://www.broadinstitute.org/mpg/snap/ldsearch.php](http://www.broadinstitute.org/mpg/snap/ldsearch.php)). This linkage disequilibrium was determined using the population panel CEU (Caucasians from Utah, USA) and CHB+JPT (Chinese from Beijing plus Japanese from Tokyo) from the 1000 Genomes Pilot 1 SNP data set (low-coverage sequencing pilot). The genotype information of rs555754 is not available in the YRI (Yoruba in Ibadan, Nigeria) population panel and thus linkage disequilibrium could not be determined for the YRI population.

### Statistical analysis

The results of the luciferase assays are presented as the mean ± SD from two to three independent experiments. Functional differences between reference and variant haplotypes were evaluated using one-way analysis of variance followed by Dunnett’s multiple comparison test. A p < 0.05 was considered significant. A D’Agostino and Pearson’s omnibus normality test (GraphPad Prism 5, GraphPad Software, La Jolla, California, USA) was used to determine whether the expression levels of OCT3 in liver samples were normally distributed. One-way analysis of variance or unpaired t-test was used to compare the OCT3 mRNA levels among the three or two genotype groups, respectively using GraphPad Prism version 5.00 (GraphPad Software, San Diego). A p < 0.05 was considered significant.

## Results

### Genetic polymorphism and haplotype analysis of the OCT3 basal promoter region

A total of five polymorphisms in the *OCT3* basal promoter and 5′-untranslated region (UTR) were identified in samples from 272 unrelated individuals from four major ethnic groups ([Table 1A](#T1)). The five polymorphisms are: g.+2T>insGCGGGCG (N/A); g.-2G>A(rs555754); g.-81G>delGA (rs60515630); g.-124 C>G (rs59711975) and g.-146 C>G (rs58601841) (the numbers are the nucleotide position relative to the transcription start site (TSS)). The allele frequencies of each polymorphism in each ethnic group are presented in [Table 1A](#T1). The common SNP, g.-2G>A, was found in all four ethnic groups with high frequency. Seven unambiguous basal promoter haplotypes were identified ([Table 1B](#T2)). In the African American samples, the two common promoter variants, rs60515630 and rs555754, showed linkage disequilibrium with r^2^ = 0.11 and D′ = 1.0. Among the five promoter variants discovered in the PMT cohort, the linkage disequilibrium between each was very weak (r^2^ = 0 to 0.1) in each of the four ethnic groups. All the identified SNPs in the OCT3 promoter region in each of the four ethnic groups were in Hardy-Weinberg Equilibrium (P > 0.05).

### Functional analysis of human OCT3 promoter region variants

To identify the minimal region of the *OCT3* proximal promoter required for its promoter activity, we tested three constructs: −384/+25, −1000/+25 and −2000/+25. The luciferase activity of the shortest fragment (−384/+25) was highest among the three fragments and was selected for subsequent studies ([Figure 1A](#F1)). The longer constructs may contain some negative elements that caused the lower luciferase activity. The highest luciferase activity of the optimal construct (−384/+25) was observed in HCT116 (about 12-fold of the empty vector pGL4.11 (EV)) followed by HepG2 (about 8-fold of EV) ([Figure 1B](#F1)), which were selected for further studies. Two prostate cancer cell lines and three lung cancer cell lines showed modest or low luciferase activity ([Figure 1B](#F1)). In both HepG2 and HCT116, two haplotypes, H3 (g.-81delGA/delGA combined with g.-2A/A) and H5 (g.-146 G/G combined with g.-2A/A), showed markedly enhanced luciferase activities (150 to 200%). Haplotypes H6 (g.-2A/A), H4 (g.-124G/G combined with g.-2A/A) and H2 (g.-146G/G combined with g.-2G/G) showed intermediate increases (125 to 150%) in the luciferase activity compared to haplotype H1(***P<0.01*) ([Figure 1C](#F1)). Not like other genetic variants in this region, H7 (g.+2T>insGCGGGCG) showed a significant reduction in luciferase activity (***P<0.01*) ([Figure 1C](#F1)). Generally, haplotypes containing the common variant g.-2A/A (H2, H3, H4, H5 and H6) had increased luciferase activity.

### Figure 1. Luciferase activities in cell lines expressing reporter constructs containing the basal promoter of OCT3.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/fa9f/3396779/fb26d18c35a0/nihms341052f1.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3396779_nihms341052f1.jpg)

The reporter activity of each construct was compared with that of empty vector (pGL4.11). Data shown represent mean ± SD from triplicate wells in a representative experiment. A, Effect of promoter segment on luciferase activity. Three segments from the promoter region were cloned into the pGL4.11 vector. The basal promoter region (−384 to +25) showed higher luciferase activity than the longer counterparters (−1000 to +24 and −2000 to +25). The reason to include a small fragment of coding region (0 – 25) is to facilitate the cloning because of the high GC percentage in the promoter region. The starting codon in the (−384 to +25) was mutated to AAG to avoid interruption of luciferase expression. B. The luciferase activity of basal promoter reporter construct in different cell lines. The HCT116 and HepG2 had higher luciferase activities than the rest of the cell lines. C. The luciferase activity of 7 Haplotypes (H1–H7) from Table 1B. The g.-2A is contained in H3, H5 and H6. H1, H2, H4 and H7 have the g.-2G at the −2 position. **P<0.01 compared with H1.

### Identification of TFs in the OCT3 basal promoter region and effect of genetic variants on transcription factor binding

Consistent with its broad expression pattern, the *OCT3* promoter region exhibited a wide spectrum of predicted TFs ([Figure S2](#SD1)). Several SNPs fell into predicted TF binding regions including g.-146C>G, which was within a potential binding region of MZF1, and g.-81G>delGA, which was in the predicted p300 binding region ([Figure S2](#SD1)). The common polymorphism, g.-2G>A, was predicted to interact with Sp1. We co-transfected predicted TFs with the *OCT3* promoter-driven luciferase constructs ([Figure 2A](#F2)). As a control, the EV showed no significant increase in luciferase assay when co-transfected with four TFs (Sp1, MZF1, p300 and Ap4) (data not shown). Among the TFs tested, MZF1 (200 ng) strongly stimulated *OCT3* promoter-driven luciferase activity, which resulted in a 3.2-fold increase in luciferase activity (*P<0.05). Higher doses of TFs did not produce higher transcriptional activities, which may reflect lower transfection efficiencies or saturation in the binding of the TFs to their regulatory regions. Sp1 showed modest transcriptional activity and increased luciferase activity from 1.2-fold to 2.4-fold with dosing from 50 to 200 ng (**P<0.05*). Both p300 and Ap4 had weak transcriptional activity. The expression levels of 4 TFs were also tested before and after co-transfection. MZF1 exhibited the highest expression level among the 4 TFs in HCT116. After co-transfection of the *OCT3* promoter with the TF expression vectors, the 4 TFs reached similar expression levels, suggesting that MZF1 had the higher transcriptional activity in the OCT3 promoter. All four TFs were expressed in HepG2, and HCT116 showed highest expression of MZF1, consistent with its higher luciferase assay in comparison to the other cell lines ([Figure 2C](#F2)).

### Figure 2. Effects of four predicted transcriptional factors on the promoter activity of OCT3.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/fa9f/3396779/a37b34d12281/nihms341052f2.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3396779_nihms341052f2.jpg)

A. Dose Effects of transcription factors on luciferase activity. Various doses of four predicted transcription factors (Sp1, MZF1, Ap4 and p300) were co-transfected with the reference construct (H1, reference). Data shown represent mean ± SD from triplicate wells in a representative experiment. *P < 0.05 vs. H1 activity without co-transfection of transcription factor accordingly. B, Expression levels of 4 TFs before co-transfection and after cotransfection. All the expression values were normalized to the lowest expression Sp1 which was set as 1.0. C. Western blots of five cell lines used in the luciferase assay to detect the expression of the four major transcription factors. Four antibodies against the above transcription factors were used to detect the expression of Sp1, MZF1, Ap-4 and p300. MZF1 showed strongest expression in HCT116. Sp1 was only detected in the HepG2 cells.

Using the nuclear protein extract from HepG2, which expressed all 4 major predicted TFs, three common genetic variants (g.-2G>A, g.-81G>delGA and g.-146C>G) were assayed for their binding affinity to the transcription factors. The oligo-nucleotide containing g.-146G/G showed more nuclear protein binding in band 1 than the oligo-nucleotide with g.-146C/C ([Figure 3A](#F3)). For g.-81G>delGA, the deletion of GA greatly increased the density of band 1 suggesting that there was stronger interactions with the enhancer element ([Figure 3A](#F3)). The density of band 1 was slightly increased by variant g.-2G>A ([Figure 3A](#F3)). Bands 1 and 2 were inhibited after adding unlabeled probes in the EMSA assays ([Figure 3B](#F3)). The stronger interactions of these 3 variants, as reflected by increased density of band 1, are consistent with the higher luciferase activities of these variants ([Figure 1C](#F1)).

### Figure 3. Electrophoretic mobility shift analysis of OCT3 reference and variant oligonucleotides and prediction of secondary structure of mRNA.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/fa9f/3396779/ac6b99c0c85b/nihms341052f3a.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3396779_nihms341052f3a.jpg)

A, Three paired oligonucleotide probes corresponding to the references and genetic variants (g.-146C and g.-146C>G, g.-81G and g.-81G>delGA, g.-2G and g.-G>A) were used to compared the nuclear protein binding affinity between the references and variants. Digoxigenin-labeled probes (Table S1) were incubated with nuclear extracts from HepG2 in the presence or absence of a 25-fold excess of unlabeled competitor as indicated. Band 1 is the nuclear protein – DNA complex. Band 2 and 3 are non-specific band and free probe, respectively. B, Supershift analysis with transcription factor specific antibodies. Supershifts were detected for all four transcription factors. The probe was incubated with nuclear extracts from HepG2 cells in the presence or absence of a specific antibody against Sp1, MZF1, Ap-4 and p300 as indicated. Band 1, 2 and 3 are the same annotation as 3C. Band 4 represents the antibody-nuclear protein-DNA complex as super-shift band. C, Prediction of secondary structure of mRNA (1 to 300) of OCT3 without g.+2T (left panel) or with insertion g.+2T>insGCGGGCG (right panel). MFE is listed below the structure.

To further define the TFs that bind to the *OCT3* promoter region, specific antibodies against the four predicted TFs were applied to EMSA. The specific antibodies resulted in the super-shift of band 1 to a higher position (band 4) after forming a DNA-TF-antibody complex ([Figure 3B](#F3)). These results confirmed that MZF1, Sp1, Ap4 and p300 were TFs that interact with the promoter region of *OCT3* with different transcriptional activity. The primers used for the g.-2G>A covered a predicted SP1 site but only partially covered a predicted p300 site. The supershift results showed SP1 was well-shifted and p300 was only shifted partly, likely resulting from the poor binding of the primer to p300. The lower luciferase activity of g.2T>insGCGGGCG in 5′-UTR may be due to the interruption of the secondary structure of mRNA ([Figure 3C](#F3)) which results in a higher minimum free energy (MFE) −18.8 kcal/mol than the g.2T with −34.6 kcal/mol predicted by the RNAfold web server ([http://rna.tbi.univie.ac.at/cgi-bin/RNAfold.cgi](http://rna.tbi.univie.ac.at/cgi-bin/RNAfold.cgi)).

### Association of promoter region variants of OCT3 g.-G>A (rs555275) with expression levels in liver

In 29 Caucasian and 40 Asian (Chinese) normal liver samples, the *OCT3* mRNA level was significantly higher in Caucasian than Asian samples ([Figure 4A](#F4)), suggesting that there may be different transcription rates of *OCT3* in Asian versus Caucasian liver. The relative expression level of representative liver samples from Caucasian and Asian are listed in [supplemental Figure S4](#SD1). Consistent with the higher luciferase activities of H6 (which contains g.-2A/A), the mRNA levels of *OCT3* in liver samples from Chinese with g.-2A/A were significantly higher than liver samples from Chinese with either g.-2G/A or g.-2G/G (n = 40; [Figure 4B](#F4)). Similar results were obtained in 29 liver samples from individuals with European ancestries ([Figure 4C](#F4)). The distribution of the *Oct3* gene expression levels in the liver samples, based on D’Agostino and Pearson omnibus normality test, was not significantly different from normal (*P* > 0.05). The mean expression values were 13.0 ± 1.7 for Caucasian and 8.1 ± 0.74 for Asian. Post- translational modification often affects the protein expression level-the final product of the mRNA. To validate whether the *OCT3* mRNA levels detected in the liver samples correlated with OCT3 proteins levels, Western blotting was used in four representative liver samples that varied from the lowest (set as 1.0), to modest (6.5 and 12.8) to highest (17.2) mRNA levels. Consistent with the mRNA level, OCT3 protein level showed a similar trend with their mRNA levels ([Figure 4D](#F4)).

### Figure 4. The effect of genotype, g. –2G>A (rs555754) on OCT3 expression levels in human liver samples from Asian and Caucasian subjects.

![Figure 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/fa9f/3396779/a4e8cc947094/nihms341052f4.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3396779_nihms341052f4.jpg)

Expression of OCT3 in the human liver tissues was determined by a quantitative real-time RT-PCR and Western blotting. The horizontal bar for each genotype represents the mean value. The P value is calculated using Student’s t test. A, Caucasian liver samples (29) had significantly higher OCT3 mRNA level (P=0.005) than that of Asian (40). The lowest expression sample from one of the Asian sample is set as 1 and the rest of the samples were normalized to it. The mean values are 13.0 ± 1.7 for Caucasian and 8.1 ± 0.74 for Asian. B, The effect of genotype, g. –2G>A on OCT3 expression levels in human liver tissues from Asian (Chinese). The individuals with homozygous g.-2A/A had a significant higher OCT3 mRNA level than those with g.-2G/G and g.-2G/A. C. The effect of genotype, g. –2G>A on OCT3 expression levels in human liver tissues from Caucasian which showed the similar trend as those of Asian. D, Validation of the protein expression level of OCT3 with western blotting from 4 Asian liver samples with the lowest one (set as 1.0), two modest ones (6.5 and 12.8) and highest (17.2).

### Promoter methylation and OCT3 expression in cell lines and primary tumor tissues

The analysis of the *OCT3* basal promoter region showed that it has a very high CG rate, with more than 85% of the CG bases. It contains 66 CpG sites in the examined region ([Figure S1](#SD1)) and a predicted CpG island in the promoter region extending to exon 1 ([Figure 5A](#F5)). These features suggest that methylation can be a major factor influencing its transcription. To test this hypothesis, we firstly performed bisulfite sequencing analysis encompassing the transcription site in 4 prostate cancer cell lines with differential tumorigencity and metastatic capability (LNCaP, DU145, C4-2B and PC3),^35,36^ plus the colon cancer cell line (HCT116). As shown in [Figure 5B](#F5), the methylation level varied from low methylation in LNCaP, moderate methylation in DU 145 and C4-2B to heavy methylation in PC3 and HCT116. The methylation mainly occurred at the 5′-end. Following qRT-PCR, we observed that cell lines with less methylation had higher levels of *OCT3* mRNA transcripts and low metastatic capability generally ([Figure 5B](#F5)). Methylated constructs *in vitro*, using Sss I CpG methylase, had dramatic reductions in luciferase activities from 12-fold to 2-fold of EV in HCT116 cells ([Figure 5C](#F5)). In *OCT3*-negative HCT116, *OCT3* mRNA levels clearly reactivated with increasing treatment doses of 5′-azadC, a demethylation agent ([Figure 5D](#F5)). Collectively, the data demonstrate that methylation of the promoter region plays an important role in expression of *OCT3*.

### Figure 5. Effect of methylation of the OCT3 promoter on its expression in cancer cell lines.

![Figure 5](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/fa9f/3396779/c1946b2a5af4/nihms341052f5.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3396779_nihms341052f5.jpg)

A, Diagram of the 5′ end of the partial OCT3 gene. The rectangles represent the exon 1 of the gene, the CpG islands and the region for bisulfite sequencing assay, respectively. B, The methylation status of the OCT3 promoter in cancer cell lines. Methylation of the OCT3 CpG islands in LNCaP, DU145, C4-2B, PC3 and HCT116 cell lines. Left, each square represents a CpG site from the 1st to 66th; blue, unmethylated; red, methylated; white, data not available. Right, the overall methylation level and the relative OCT3 gene expression of these cell lines. C, Effect of methylase on the luciferase activity of the OCT3 reference promoter (H1) construct in HCT116 and HepG2. As shown, the OCT3 basal promoter exhibits dramatically reduced reporter activity in the presence of CpG mehtylase SssI. D. Demethylation reagent 5′-azadC activated the expression of OCT3 in HCT116 which was highly methylated shown in B. 5′-azadC increased the expression of OCT3 with a dose-dependent manner.

To determine whether methylation may explain the reduced expression level of *OCT3* in prostate cancer tissues,^4^ we examined the methylation in 8 paired samples of normal and cancerous prostate with high Gleason Grade. In prostate cancer samples that had the high Gleason grade from 4 to 5, *OCT3* showed a significantly lower expression than paired normal tissue ([Figure 6A and B](#F6)), consistent with previous observations.^4^ In the paired normal tissues, the basal promoter had a low level of methylation ranging from 0.13% to1.81% ([Figure 6C](#F6)). In contrast, 5 of 8 prostate cancer samples (TP) showed aberrant high methylation levels as TP2 (32.5%), TP3 (18.9%), TP4 (22.4%), TP6 (15.6%) and TP8 (12.7%). The cancerous samples showed much higher methylation levels, primarily at the 5′-end of the promoter ([Figure 6C](#F6)). These data demonstrate that methylation of the *OCT3* promoter region may explain the low expression levels of OCT3 in prostate cancer with high Gleason grades, which might be associated with progression of prostate cancer. In contrast to prostate cancer tissue samples, the liver tumor tissue samples showed similar methylation rates to the paired non-tumor tissues in the *OCT3* promoter region ([Supplemental Figure S5](#SD1)).

### Figure 6. Analysis of expression and methylation in paired normal and cancerous prostate samples.

![Figure 6](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/fa9f/3396779/e75d97baa3b8/nihms341052f6.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3396779_nihms341052f6.jpg)

A and B, Expression levels of mRNA transcripts of OCT3 in 8 matched tumor-normal tissue samples from prostate cancer patients. The lowest expression level sample is set as 1.0 and the rest of the samples were normalized to it. The expression of OCT3 in normal prostate tissue was significantly higher than those in cancerous ones. C. Aberrant methylation of OCT3 promoter in primary prostate cancer samples. The methylation % of each CpG site is plotted for eight prostate cancer samples (upper panel) and eight normal prostate samples (lower panel). Y-axis is the methylation %; X-axis is the individual CpG sites (1–66). The methylation status of the CpG island was analyzed by sequencing subclones of PCR products from bisulfite-treated DNA. Each samples had at least 10 clones. For each CpG site, the methylation percentage was calculated by dividing the number of methylated clones by the total number of clones sequenced. 5 aberrant methylation tumor samples were labeled with *. NP, normal prostate tissue; TP, prostate cancer tissue. Each square represented a CpG site, blue, unmethylated; red, methylated; white, data not available.

## Discussion

Increasingly evidences have recognized OCT3 as the important anti-diabetes and cancer drugs transporter.^7,11,13^ GWAS studies have suggested that SNPs in *OCT3* are important risk factors for prostate cancer and that expression of the gene is dramatically reduced in prostate cancer and particularly in high-Gleason grade ones.^3–5^ These studies have led to speculation that OCT3 may serve as a tumor suppressor gene in prostate cancer. Thus, understanding the mechanisms by which OCT3 expression is regulated is important in elucidating the occurrence and pathogenesis of prostate cancer. In this investigation, we describe both novel genetic and epigenetic factors in the basal promoter of OCT3 that regulate its expression levels in healthy and cancerous tissues. These factors may significantly influence its role in drug uptake efficacy and tumorigenises.

One of the key findings of the study was that a common promoter region variant, g.-2G>A (rs555754), found in all ethnic groups at high allele frequencies >32%, enhanced transcription rates in reporter assays and was associated with higher expression levels of *OCT3* in liver samples obtained from individuals with ancestries in Asia and Europe. In a previous study, SNPs associating with low *OCT3* expression levels in the liver were found;^11^ however, the causative SNP was not identified. In a large sample of human livers, Schadt et al. showed that an intronic SNP (rs518295), which is 6155 bp downstream of rs555754 had a significantly positive association with OCT3 mRNA levels.^37^ The rs518295 is in linkage disequilibrium with rs555754 (r^2^=0.7).

The significant and consistent under-expression of *OCT3* in prostate cancer makes it a candidate tumor suppressor gene.^4,5,15^ The current investigation suggests that the underlying mechanism for reduced expression of the transporter in prostate cancer involves aberrant methylation of the promoter region of *OCT3*. As methylation may occur early in prostate cancer and can be detected in body fluids, it may be of potential use in early detection of tumors and for determining prognosis.^21,22,38^ Further, the level of methylation of the *OCT3* promoter may be applied to tumor samples from prostate cancer patients to determine aggressiveness which might serve as biomarkers for the Gleason scores of prostate cancer. As shown in [Figure 5](#F5), low expression of *OCT3* can be reversed by adding demethylating reagents. If *OCT3* is truly a tumor suppressor in prostate cancer or plays a biologic role in tumor progression, potentially, demethylating drugs such as 5′-azacytidine and decitabine could be used to treat patients with metastatic prostate cancer, combined with other anti-prostate cancer drugs to enhance their length of survival. Although hypermethylation of the promoter region of *OCT3* accounts for the low expression level of *OCT3* in prostate cancer and cell lines, it is not the only event that suppresses the expression of *OCT3* as some of the prostate tumor samples did not show the high methylation level. Other elements in the pre- or post-transcriptional process could also affect the *OCT3* expression like mRNA stability involving microRNA and its long 3′-UTR. Three genetic variants (g.-146C>G, g.-124C>G and g.-81G>delGA) are located in the potential methylation sites (CpG) ([Figure S1](#SD1)). Though speculative these variants, e.g., g.-81G>delGA, could be associated with changes in the methylation status in and around these positions and potentially influence the expression of *OCT3*, leading to increased risk for prostate cancer. Similar to prostate tissue samples, large variation in the expression level of OCT3 in human liver samples was observed. However, in contrast to prostate tumors, the degree of promoter methylation in the paired liver tumor samples was not appreciably higher than in the normal tissues ([Figure S5](#SD1)). It is likely that the methylation of the *OCT3* promoter occurs in a tissue=specific manner with a high degree of methylation in prostate cancer. The promoters of several other transporters also show tissue-specific methylation pattern.^28–32^ The variation of *OCT3* expression in liver tissue may be due to other regulatory elements of gene expression, for example, in the extra-long 3′UTR of the gene.

In summary, the present investigation demonstrates that the common variant, g.-2G>A (rs555754), is associated with a greater transcription rate and higher expression levels of *OCT3* in the liver. Hypermethylation of the *OCT3* promoter region in prostate cancer is revealed as one of the important mechanisms for its reduced expression. These findings add to the growing body of work on the pharmacogenetic and biological role of *OCT3*.

## Supplementary Material

## Acknowledgments

The project is supported by grant from NIH GM61390. We highly appreciate Dr. Suet-Tim Cheung at the University of Hong Kong to provide human liver samples and Dr. Peter Carroll and Dr. Hubert Stoppler in the Cancer Tissue Center in UCSF for the prostate cancer tissues. Lu Xu is a visiting graduate student from University of the Pacific.

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Tregouet DA, Konig IR, Erdmann J, Munteanu A, Braund PS, Hall AS, et al. Genome-wide haplotype association study identifies the SLC22A3-LPAL2-LPA gene cluster as a risk locus for coronary artery disease. Nat Genet. 2009;41:283–5. doi: 10.1038/ng.314.  [DOI](https://doi.org/10.1038/ng.314) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19198611/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Genome-wide%20haplotype%20association%20study%20identifies%20the%20SLC22A3-LPAL2-LPA%20gene%20cluster%20as%20a%20risk%20locus%20for%20coronary%20artery%20disease&author=DA%20Tregouet&author=IR%20Konig&author=J%20Erdmann&author=A%20Munteanu&author=PS%20Braund&volume=41&publication_year=2009&pages=283-5&pmid=19198611&doi=10.1038/ng.314&)

2. Lazar A, Walitza S, Jetter A, Gerlach M, Warnke A, Herpertz-Dahlmann B, et al. Novel mutations of the extraneuronal monoamine transporter gene in children and adolescents with obsessive-compulsive disorder. Int J Neuropsychopharmacol. 2008;11:35–48. doi: 10.1017/S1461145707007742.  [DOI](https://doi.org/10.1017/S1461145707007742) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17477885/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Neuropsychopharmacol&title=Novel%20mutations%20of%20the%20extraneuronal%20monoamine%20transporter%20gene%20in%20children%20and%20adolescents%20with%20obsessive-compulsive%20disorder&author=A%20Lazar&author=S%20Walitza&author=A%20Jetter&author=M%20Gerlach&author=A%20Warnke&volume=11&publication_year=2008&pages=35-48&pmid=17477885&doi=10.1017/S1461145707007742&)

3. Eeles RA, Kote-Jarai Z, Giles GG, Al Olama AA, Guy M, Jugurnauth SK, et al. Multiple newly identified loci associated with prostate cancer susceptibility. Nature Genet. 2008;40:316–21. doi: 10.1038/ng.90.  [DOI](https://doi.org/10.1038/ng.90) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18264097/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature%20Genet&title=Multiple%20newly%20identified%20loci%20associated%20with%20prostate%20cancer%20susceptibility&author=RA%20Eeles&author=Z%20Kote-Jarai&author=GG%20Giles&author=AA%20Al%20Olama&author=M%20Guy&volume=40&publication_year=2008&pages=316-21&pmid=18264097&doi=10.1038/ng.90&)

4. Tomlins SA, Mehra R, Rhodes DR, Cao X, Wang L, Dhanasekaran SM, et al. Integrative molecular concept modeling of prostate cancer progression. Nat Genet. 2007;39:41–51. doi: 10.1038/ng1935.  [DOI](https://doi.org/10.1038/ng1935) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17173048/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Integrative%20molecular%20concept%20modeling%20of%20prostate%20cancer%20progression&author=SA%20Tomlins&author=R%20Mehra&author=DR%20Rhodes&author=X%20Cao&author=L%20Wang&volume=39&publication_year=2007&pages=41-51&pmid=17173048&doi=10.1038/ng1935&)

5. Cui R, Okada Y, Jang SG, Ku JL, Park JG, Kamatani Y, et al. Common variant in 6q26-q27 is associated with distal colon cancer in an Asian population. Gut. 2001 doi: 10.1136/gut.2010.215947. [Epub ahead of print]  [DOI](https://doi.org/10.1136/gut.2010.215947) | [PMC free article](/articles/PMC3095478/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21242260/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gut&title=Common%20variant%20in%206q26-q27%20is%20associated%20with%20distal%20colon%20cancer%20in%20an%20Asian%20population&author=R%20Cui&author=Y%20Okada&author=SG%20Jang&author=JL%20Ku&author=JG%20Park&publication_year=2001&pmid=21242260&doi=10.1136/gut.2010.215947&)

6. True L, Coleman I, Hawley S, Huang CY, Gifford D, Coleman R, et al. A molecular correlate to the Gleason grading system for prostate adenocarcinoma. Proc Natl Acad Sci USA. 2006;103:10991–6. doi: 10.1073/pnas.0603678103.  [DOI](https://doi.org/10.1073/pnas.0603678103) | [PMC free article](/articles/PMC1544162/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16829574/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20USA&title=A%20molecular%20correlate%20to%20the%20Gleason%20grading%20system%20for%20prostate%20adenocarcinoma&author=L%20True&author=I%20Coleman&author=S%20Hawley&author=CY%20Huang&author=D%20Gifford&volume=103&publication_year=2006&pages=10991-6&pmid=16829574&doi=10.1073/pnas.0603678103&)

7. Chen L, Pawlikowski B, Schlessinger A, More SS, Stryke D, Johns SJ, et al. Role of organic cation transporter 3 (SLC22A3) and its missense variants in the pharmacologic action of metformin. Pharmacogenet Genom. 2010;20:687–99. doi: 10.1097/FPC.0b013e32833fe789.  [DOI](https://doi.org/10.1097/FPC.0b013e32833fe789) | [PMC free article](/articles/PMC2976715/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20859243/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genom&title=Role%20of%20organic%20cation%20transporter%203%20(SLC22A3)%20and%20its%20missense%20variants%20in%20the%20pharmacologic%20action%20of%20metformin&author=L%20Chen&author=B%20Pawlikowski&author=A%20Schlessinger&author=SS%20More&author=D%20Stryke&volume=20&publication_year=2010&pages=687-99&pmid=20859243&doi=10.1097/FPC.0b013e32833fe789&)

8. Baganz NL, Horton RE, Calderon AS, Owens WA, Munn JL, Watts LT, et al. Organic cation transporter 3: Keeping the brake on extracellular serotonin in serotonin-transporter-deficient mice. Proc Natl Acad Sci USA. 2008;105:18976–81. doi: 10.1073/pnas.0800466105.  [DOI](https://doi.org/10.1073/pnas.0800466105) | [PMC free article](/articles/PMC2596260/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19033200/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20USA&title=Organic%20cation%20transporter%203:%20Keeping%20the%20brake%20on%20extracellular%20serotonin%20in%20serotonin-transporter-deficient%20mice&author=NL%20Baganz&author=RE%20Horton&author=AS%20Calderon&author=WA%20Owens&author=JL%20Munn&volume=105&publication_year=2008&pages=18976-81&pmid=19033200&doi=10.1073/pnas.0800466105&)

9. Koepsell H, Lips K, Volk C. Polyspecific organic cation transporters: Structure, function, physiological roles, and biopharmaceutical implications. Pharm Res. 2007;24:1227–51. doi: 10.1007/s11095-007-9254-z.  [DOI](https://doi.org/10.1007/s11095-007-9254-z) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17473959/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharm%20Res&title=Polyspecific%20organic%20cation%20transporters:%20Structure,%20function,%20physiological%20roles,%20and%20biopharmaceutical%20implications&author=H%20Koepsell&author=K%20Lips&author=C%20Volk&volume=24&publication_year=2007&pages=1227-51&pmid=17473959&doi=10.1007/s11095-007-9254-z&)

10. Ogasawara M, Yamauchi K, Satoh Y, Yarnaji R, Inui K, Jonker JW, et al. Recent advances in molecular pharmacology of the histamine systems: Organic cation transporters as a histamine transporter and histamine metabolism. J Pharmacol Sci. 2006;101:24–30. doi: 10.1254/jphs.fmj06001x6.  [DOI](https://doi.org/10.1254/jphs.fmj06001x6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16648665/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharmacol%20Sci&title=Recent%20advances%20in%20molecular%20pharmacology%20of%20the%20histamine%20systems:%20Organic%20cation%20transporters%20as%20a%20histamine%20transporter%20and%20histamine%20metabolism&author=M%20Ogasawara&author=K%20Yamauchi&author=Y%20Satoh&author=R%20Yarnaji&author=K%20Inui&volume=101&publication_year=2006&pages=24-30&pmid=16648665&doi=10.1254/jphs.fmj06001x6&)

11. Nies AT, Koepsell H, Winter S, Burk O, Klein K, Kerb R, et al. Expression of organic cation transporters OCT1 (SLC22A1) and OCT3 (SLC22A3) is affected by genetic factors and cholestasis in human liver. Hepatol. 2009;50:1227–40. doi: 10.1002/hep.23103.  [DOI](https://doi.org/10.1002/hep.23103) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19591196/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hepatol&title=Expression%20of%20organic%20cation%20transporters%20OCT1%20(SLC22A1)%20and%20OCT3%20(SLC22A3)%20is%20affected%20by%20genetic%20factors%20and%20cholestasis%20in%20human%20liver&author=AT%20Nies&author=H%20Koepsell&author=S%20Winter&author=O%20Burk&author=K%20Klein&volume=50&publication_year=2009&pages=1227-40&pmid=19591196&doi=10.1002/hep.23103&)

12. Yonezawa A, Inui K. Organic cation transporter OCT/SLC22A and H+/organic cation antiporter MATE/SLC47A are key molecules for nephrotoxicity of platinum agents. Biochem Pharmacol. 2011;81:563–8. doi: 10.1016/j.bcp.2010.11.016.  [DOI](https://doi.org/10.1016/j.bcp.2010.11.016) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21144842/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem%20Pharmacol&title=Organic%20cation%20transporter%20OCT/SLC22A%20and%20H+/organic%20cation%20antiporter%20MATE/SLC47A%20are%20key%20molecules%20for%20nephrotoxicity%20of%20platinum%20agents&author=A%20Yonezawa&author=K%20Inui&volume=81&publication_year=2011&pages=563-8&pmid=21144842&doi=10.1016/j.bcp.2010.11.016&)

13. Yokoo S, Masuda S, Yonezawa A, Terada T, Katsura T, Inui KI. Significance of organic cation transporter 3 (SLC22A3) expression for the cytotoxic effect of oxaliplatin in colorectal cancer. Drug Metab Dispos. 2008;36:2299–306. doi: 10.1124/dmd.108.023168.  [DOI](https://doi.org/10.1124/dmd.108.023168) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18710896/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Significance%20of%20organic%20cation%20transporter%203%20(SLC22A3)%20expression%20for%20the%20cytotoxic%20effect%20of%20oxaliplatin%20in%20colorectal%20cancer&author=S%20Yokoo&author=S%20Masuda&author=A%20Yonezawa&author=T%20Terada&author=T%20Katsura&volume=36&publication_year=2008&pages=2299-306&pmid=18710896&doi=10.1124/dmd.108.023168&)

14. Waters KM, Le Marchand L, Kolonel LN, Monroe KR, Stram DO, Henderson BE, et al. Generalizability of Associations from Prostate Cancer Genome-wide association studies in multiple populations. Cancer Epidem Biomar. 2009;18:1285–9. doi: 10.1158/1055-9965.EPI-08-1142.  [DOI](https://doi.org/10.1158/1055-9965.EPI-08-1142) | [PMC free article](/articles/PMC2917607/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19318432/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Epidem%20Biomar&title=Generalizability%20of%20Associations%20from%20Prostate%20Cancer%20Genome-wide%20association%20studies%20in%20multiple%20populations&author=KM%20Waters&author=L%20Le%20Marchand&author=LN%20Kolonel&author=KR%20Monroe&author=DO%20Stram&volume=18&publication_year=2009&pages=1285-9&pmid=19318432&doi=10.1158/1055-9965.EPI-08-1142&)

15. Kader AK, Sun JL, Isaacs SD, Wiley KE, Yan GF, Kim ST, et al. Individual and Cumulative Effect of Prostate Cancer Risk-Associated Variants on Clinicopathologic Variables in 5,895 Prostate Cancer Patients. Prostate. 2009;69:1195–205. doi: 10.1002/pros.20970.  [DOI](https://doi.org/10.1002/pros.20970) | [PMC free article](/articles/PMC2852875/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19434657/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Prostate&title=Individual%20and%20Cumulative%20Effect%20of%20Prostate%20Cancer%20Risk-Associated%20Variants%20on%20Clinicopathologic%20Variables%20in%205,895%20Prostate%20Cancer%20Patients&author=AK%20Kader&author=JL%20Sun&author=SD%20Isaacs&author=KE%20Wiley&author=GF%20Yan&volume=69&publication_year=2009&pages=1195-205&pmid=19434657&doi=10.1002/pros.20970&)

16. Yee SW, Shima JE, Hesselson S, Nguyen L, De Val S, LaFond RJ, et al. Identification and characterization of proximal promoter polymorphisms in the Human concentrative nucleoside transporter 2 (SLC28A2) J Pharmacol Exp Ther. 2009;328:699–707. doi: 10.1124/jpet.108.147207.  [DOI](https://doi.org/10.1124/jpet.108.147207) | [PMC free article](/articles/PMC2682268/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19098160/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharmacol%20Exp%20Ther&title=Identification%20and%20characterization%20of%20proximal%20promoter%20polymorphisms%20in%20the%20Human%20concentrative%20nucleoside%20transporter%202%20(SLC28A2)&author=SW%20Yee&author=JE%20Shima&author=S%20Hesselson&author=L%20Nguyen&author=S%20De%20Val&volume=328&publication_year=2009&pages=699-707&pmid=19098160&doi=10.1124/jpet.108.147207&)

17. Choi JH, Yee SW, Kim MJ, Nguyen L, Lee JH, Kang JO, et al. Identification and characterization of novel polymorphisms in the basal promoter of the human transporter, MATE1. Pharmacogenet Genom. 2009;19:770–80. doi: 10.1097/FPC.0b013e328330eeca.  [DOI](https://doi.org/10.1097/FPC.0b013e328330eeca) | [PMC free article](/articles/PMC2976711/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19745787/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genom&title=Identification%20and%20characterization%20of%20novel%20polymorphisms%20in%20the%20basal%20promoter%20of%20the%20human%20transporter,%20MATE1&author=JH%20Choi&author=SW%20Yee&author=MJ%20Kim&author=L%20Nguyen&author=JH%20Lee&volume=19&publication_year=2009&pages=770-80&pmid=19745787&doi=10.1097/FPC.0b013e328330eeca&)

18. Hesselson SE, Matsson P, Shima JE, Fukushima H, Yee SW, Kobayashi Y, et al. Genetic variation in the proximal promoter of ABC and SLC superfamilies: Liver and kidney specific expression and promoter activity predict variation. PLoS One. 2009;4:1–12. doi: 10.1371/journal.pone.0006942.  [DOI](https://doi.org/10.1371/journal.pone.0006942) | [PMC free article](/articles/PMC2735003/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19742321/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=Genetic%20variation%20in%20the%20proximal%20promoter%20of%20ABC%20and%20SLC%20superfamilies:%20Liver%20and%20kidney%20specific%20expression%20and%20promoter%20activity%20predict%20variation&author=SE%20Hesselson&author=P%20Matsson&author=JE%20Shima&author=H%20Fukushima&author=SW%20Yee&volume=4&publication_year=2009&pages=1-12&pmid=19742321&doi=10.1371/journal.pone.0006942&)

19. Tahara H, Yee SW, Urban TJ, Hesselson S, Castro RA, Kawamoto M, et al. Functional genetic variation in the basal promoter of the organic cation/carnitine transporters OCTN1 (SLC22A4) and OCTN2 (SLC22A5) J Pharmacol Exp Ther. 2009;329:262–71. doi: 10.1124/jpet.108.146449.  [DOI](https://doi.org/10.1124/jpet.108.146449) | [PMC free article](/articles/PMC2670604/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19141711/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharmacol%20Exp%20Ther&title=Functional%20genetic%20variation%20in%20the%20basal%20promoter%20of%20the%20organic%20cation/carnitine%20transporters%20OCTN1%20(SLC22A4)%20and%20OCTN2%20(SLC22A5)&author=H%20Tahara&author=SW%20Yee&author=TJ%20Urban&author=S%20Hesselson&author=RA%20Castro&volume=329&publication_year=2009&pages=262-71&pmid=19141711&doi=10.1124/jpet.108.146449&)

20. Brena RM, Costello JF. Genome-epigenome interactions in cancer. Hum Mol Genet. 2007;16:R96–105. doi: 10.1093/hmg/ddm073.  [DOI](https://doi.org/10.1093/hmg/ddm073) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17613554/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mol%20Genet&title=Genome-epigenome%20interactions%20in%20cancer&author=RM%20Brena&author=JF%20Costello&volume=16&publication_year=2007&pages=R96-105&pmid=17613554&doi=10.1093/hmg/ddm073&)

21. Das PM, Singal R. DNA methylation and cancer. J Clin Oncol. 2004;22:4632–42. doi: 10.1200/JCO.2004.07.151.  [DOI](https://doi.org/10.1200/JCO.2004.07.151) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15542813/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=DNA%20methylation%20and%20cancer&author=PM%20Das&author=R%20Singal&volume=22&publication_year=2004&pages=4632-42&pmid=15542813&doi=10.1200/JCO.2004.07.151&)

22. Rodríguez-Paredes M, Esteller M. Cancer epigenetics reaches mainstream oncology. Nat Med. 2011;17:330–9. doi: 10.1038/nm.2305.  [DOI](https://doi.org/10.1038/nm.2305) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21386836/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Med&title=Cancer%20epigenetics%20reaches%20mainstream%20oncology&author=M%20Rodr%C3%ADguez-Paredes&author=M%20Esteller&volume=17&publication_year=2011&pages=330-9&pmid=21386836&doi=10.1038/nm.2305&)

23. Ezzeldin HH, Lee AM, Mattison LK, Diasio RB. Methylation of the DPYD promoter: An alternative mechanism for dihydropyrimidine dehydrogenase deficiency in cancer patients. Clin Cancer Res. 2005;11:8699–705. doi: 10.1158/1078-0432.CCR-05-1520.  [DOI](https://doi.org/10.1158/1078-0432.CCR-05-1520) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16361556/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&title=Methylation%20of%20the%20DPYD%20promoter:%20An%20alternative%20mechanism%20for%20dihydropyrimidine%20dehydrogenase%20deficiency%20in%20cancer%20patients&author=HH%20Ezzeldin&author=AM%20Lee&author=LK%20Mattison&author=RB%20Diasio&volume=11&publication_year=2005&pages=8699-705&pmid=16361556&doi=10.1158/1078-0432.CCR-05-1520&)

24. Yu JS, Freimuth RR, Culverhouse R, Marsh S, Watson MA, McLeod HL. DNA methylotype analysis in colorectal cancer. Oncol Rep. 2008;20:921–7.  [PMC free article](/articles/PMC4294550/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18813836/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncol%20Rep&title=DNA%20methylotype%20analysis%20in%20colorectal%20cancer&author=JS%20Yu&author=RR%20Freimuth&author=R%20Culverhouse&author=S%20Marsh&author=MA%20Watson&volume=20&publication_year=2008&pages=921-7&pmid=18813836&)

25. Yu JS, McLeod HL. Methylation of the DPYD promoter and dihydropyrimidine dehydrogenase deficiency. Clin Cancer Res. 2006;12:3864. doi: 10.1158/1078-0432.CCR-06-0549.  [DOI](https://doi.org/10.1158/1078-0432.CCR-06-0549) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16778115/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&title=Methylation%20of%20the%20DPYD%20promoter%20and%20dihydropyrimidine%20dehydrogenase%20deficiency&author=JS%20Yu&author=HL%20McLeod&volume=12&publication_year=2006&pages=3864&pmid=16778115&doi=10.1158/1078-0432.CCR-06-0549&)

26. Teodoridis JM, Hall J, Marsh S, Kannall HD, Smyth C, Curto J, et al. CpG island methylation of DNA damage response genes in advanced ovarian cancer. Cancer Res. 2005;65:8961–7. doi: 10.1158/0008-5472.CAN-05-1187.  [DOI](https://doi.org/10.1158/0008-5472.CAN-05-1187) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16204069/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=CpG%20island%20methylation%20of%20DNA%20damage%20response%20genes%20in%20advanced%20ovarian%20cancer&author=JM%20Teodoridis&author=J%20Hall&author=S%20Marsh&author=HD%20Kannall&author=C%20Smyth&volume=65&publication_year=2005&pages=8961-7&pmid=16204069&doi=10.1158/0008-5472.CAN-05-1187&)

27. Zhang X, Soong R, Wang K, Li L, Davie JR, Guarcello V, et al. Suppression of DPYD expression in RKO Cells via DNA methylation in the regulatory region of the DPYD promoter: a potentially important epigenetic mechanism regulating DPYD expression. Biochem Cell Biol. 2007;85:337–46. doi: 10.1139/o07-009.  [DOI](https://doi.org/10.1139/o07-009) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17612628/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem%20Cell%20Biol&title=Suppression%20of%20DPYD%20expression%20in%20RKO%20Cells%20via%20DNA%20methylation%20in%20the%20regulatory%20region%20of%20the%20DPYD%20promoter:%20a%20potentially%20important%20epigenetic%20mechanism%20regulating%20DPYD%20expression&author=X%20Zhang&author=R%20Soong&author=K%20Wang&author=L%20Li&author=JR%20Davie&volume=85&publication_year=2007&pages=337-46&pmid=17612628&doi=10.1139/o07-009&)

28. Kikuchi R, Kusuhara H, Hattori N, Shiota K, Kim I, Gonzalez FJ, et al. Regulation of the expression of human organic anion transporter 3 by hepatocyte nuclear factor 1 α/β and DNA methylation. Mol Pharmacol. 2006;70:887–96. doi: 10.1124/mol.106.025494.  [DOI](https://doi.org/10.1124/mol.106.025494) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16793932/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Pharmacol&title=Regulation%20of%20the%20expression%20of%20human%20organic%20anion%20transporter%203%20by%20hepatocyte%20nuclear%20factor%201%20%CE%B1/%CE%B2%20and%20DNA%20methylation&author=R%20Kikuchi&author=H%20Kusuhara&author=N%20Hattori&author=K%20Shiota&author=I%20Kim&volume=70&publication_year=2006&pages=887-96&pmid=16793932&doi=10.1124/mol.106.025494&)

29. Ichihara S, Kikuchi R, Kusuhara H, Imai S, Maeda K, Sugiyama Y. DNA Methylation Profiles of Organic Anion Transporting Polypeptide 1B3 in Cancer Cell Lines. Pharm Res. 2010;27:510–6. doi: 10.1007/s11095-010-0064-3.  [DOI](https://doi.org/10.1007/s11095-010-0064-3) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20130966/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharm%20Res&title=DNA%20Methylation%20Profiles%20of%20Organic%20Anion%20Transporting%20Polypeptide%201B3%20in%20Cancer%20Cell%20Lines&author=S%20Ichihara&author=R%20Kikuchi&author=H%20Kusuhara&author=S%20Imai&author=K%20Maeda&volume=27&publication_year=2010&pages=510-6&pmid=20130966&doi=10.1007/s11095-010-0064-3&)

30. Kikuchi R, Yagi S, Kusuhara H, Imai S, Sugiyama Y, Shiota K. Genome-wide analysis of epigenetic signatures for kidney-specific transporters. Kidney Int. 2010;78:569–77. doi: 10.1038/ki.2010.176.  [DOI](https://doi.org/10.1038/ki.2010.176) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20555317/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Kidney%20Int&title=Genome-wide%20analysis%20of%20epigenetic%20signatures%20for%20kidney-specific%20transporters&author=R%20Kikuchi&author=S%20Yagi&author=H%20Kusuhara&author=S%20Imai&author=Y%20Sugiyama&volume=78&publication_year=2010&pages=569-77&pmid=20555317&doi=10.1038/ki.2010.176&)

31. Imai S, Kikuchi R, Kusuhara H, Yagi S, Shiota K, Sugiyama Y. Analysis of DNA Methylation and Histone Modification Profiles of Liver-Specific Transporters. Mol Pharmacol. 2009;75:568–76. doi: 10.1124/mol.108.052589.  [DOI](https://doi.org/10.1124/mol.108.052589) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19047482/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Pharmacol&title=Analysis%20of%20DNA%20Methylation%20and%20Histone%20Modification%20Profiles%20of%20Liver-Specific%20Transporters&author=S%20Imai&author=R%20Kikuchi&author=H%20Kusuhara&author=S%20Yagi&author=K%20Shiota&volume=75&publication_year=2009&pages=568-76&pmid=19047482&doi=10.1124/mol.108.052589&)

32. Terada T, Inui K. Gene expression and regulation of drug transporters in the intestine and kidney. Biochem Pharmacol. 2007;73:440–9. doi: 10.1016/j.bcp.2006.10.010.  [DOI](https://doi.org/10.1016/j.bcp.2006.10.010) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17137557/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem%20Pharmacol&title=Gene%20expression%20and%20regulation%20of%20drug%20transporters%20in%20the%20intestine%20and%20kidney&author=T%20Terada&author=K%20Inui&volume=73&publication_year=2007&pages=440-9&pmid=17137557&doi=10.1016/j.bcp.2006.10.010&)

33. Tokuzawa Y, Kaiho E, Maruyama M, Takahashi K, Mitsui K, Maeda M, et al. Fbx15 is a novel target of Oct3/4 but is dispensable for embryonic stem cell self-renewal and mouse development. Mol Cell Biol. 2003;23:2699–708. doi: 10.1128/MCB.23.8.2699-2708.2003.  [DOI](https://doi.org/10.1128/MCB.23.8.2699-2708.2003) | [PMC free article](/articles/PMC152544/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12665572/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Cell%20Biol&title=Fbx15%20is%20a%20novel%20target%20of%20Oct3/4%20but%20is%20dispensable%20for%20embryonic%20stem%20cell%20self-renewal%20and%20mouse%20development&author=Y%20Tokuzawa&author=E%20Kaiho&author=M%20Maruyama&author=K%20Takahashi&author=K%20Mitsui&volume=23&publication_year=2003&pages=2699-708&pmid=12665572&doi=10.1128/MCB.23.8.2699-2708.2003&)

34. Hong CB, Moorefield KS, Jun P, Aldape KD, Kharbanda S, Phillips HS, et al. Epigenome scans and cancer genome sequencing converge on WNK2, a kinase-independent suppressor of cell growth. Proc Natl Acad Sci U S A. 2007;104:10974–9. doi: 10.1073/pnas.0700683104.  [DOI](https://doi.org/10.1073/pnas.0700683104) | [PMC free article](/articles/PMC1904113/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17578925/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&title=Epigenome%20scans%20and%20cancer%20genome%20sequencing%20converge%20on%20WNK2,%20a%20kinase-independent%20suppressor%20of%20cell%20growth&author=CB%20Hong&author=KS%20Moorefield&author=P%20Jun&author=KD%20Aldape&author=S%20Kharbanda&volume=104&publication_year=2007&pages=10974-9&pmid=17578925&doi=10.1073/pnas.0700683104&)

35. Koeneman KS, Yeung F, Chung LW. Osteomimetic properties of prostate cancer cells: a hypothesis supporting the predilection of prostate cancer metastasis and growth in the bone environment. Prostate. 1999;39:246–61. doi: 10.1002/(sici)1097-0045(19990601)39:4<246::aid-pros5>3.0.co;2-u.  [DOI](https://doi.org/10.1002/(sici)1097-0045(19990601)39:4<246::aid-pros5>3.0.co;2-u) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10344214/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Prostate&title=Osteomimetic%20properties%20of%20prostate%20cancer%20cells:%20a%20hypothesis%20supporting%20the%20predilection%20of%20prostate%20cancer%20metastasis%20and%20growth%20in%20the%20bone%20environment&author=KS%20Koeneman&author=F%20Yeung&author=LW%20Chung&volume=39&publication_year=1999&pages=246-61&pmid=10344214&doi=10.1002/(sici)1097-0045(19990601)39:4<246::aid-pros5>3.0.co;2-u&)

36. Pulukuri SM, Gondi CS, Lakka SS, Jutla A, Estes N, Gujrati M, et al. RNA interference-directed knockdown of urokinase plasminogen activator and urokinase plasminogen activator receptor inhibits prostate cancer cell invasion, survival, and tumorigenicity in vivo. J Biol Chem. 2005;280:36529–40. doi: 10.1074/jbc.M503111200.  [DOI](https://doi.org/10.1074/jbc.M503111200) | [PMC free article](/articles/PMC1351057/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16127174/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=RNA%20interference-directed%20knockdown%20of%20urokinase%20plasminogen%20activator%20and%20urokinase%20plasminogen%20activator%20receptor%20inhibits%20prostate%20cancer%20cell%20invasion,%20survival,%20and%20tumorigenicity%20in%20vivo&author=SM%20Pulukuri&author=CS%20Gondi&author=SS%20Lakka&author=A%20Jutla&author=N%20Estes&volume=280&publication_year=2005&pages=36529-40&pmid=16127174&doi=10.1074/jbc.M503111200&) | [Retracted](/articles/PMC7489898/)

37. Schadt EE, Molony C, Chudin E, Hao K, Yang X, Lum PY, et al. Mapping the genetic architecture of gene expression in human liver. PLoS Biol. 2008;6:1020–32. doi: 10.1371/journal.pbio.0060107.  [DOI](https://doi.org/10.1371/journal.pbio.0060107) | [PMC free article](/articles/PMC2365981/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18462017/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Biol&title=Mapping%20the%20genetic%20architecture%20of%20gene%20expression%20in%20human%20liver&author=EE%20Schadt&author=C%20Molony&author=E%20Chudin&author=K%20Hao&author=X%20Yang&volume=6&publication_year=2008&pages=1020-32&pmid=18462017&doi=10.1371/journal.pbio.0060107&)

38. Brena RM, Plass C, Costello JF. Mining methylation for early detection of common cancers. PLoS Med. 2006;3:2184–5. doi: 10.1371/journal.pmed.0030479.  [DOI](https://doi.org/10.1371/journal.pmed.0030479) | [PMC free article](/articles/PMC1762083/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17194184/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Med&title=Mining%20methylation%20for%20early%20detection%20of%20common%20cancers&author=RM%20Brena&author=C%20Plass&author=JF%20Costello&volume=3&publication_year=2006&pages=2184-5&pmid=17194184&doi=10.1371/journal.pmed.0030479&)
