# The SCARB1 gene is associated with lipid response to dietary and pharmacological interventions

## Metadata
**Authors:** Yongjun Liu, Jose M Ordovas, Guimin Gao, Michael Province, Robert J Straka, Michael Y Tsai, Chao-Qiang Lai, Kui Zhang, Ingrid Borecki, James E Hixson, David B Allison, Donna K Arnett
**Journal:** Journal of human genetics
**Date:** 2008 Jun 10
**DOI:** [10.1007/s10038-008-0302-2](https://doi.org/10.1007/s10038-008-0302-2)
**PMID:** 18542840
**PMCID:** PMC3836273
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3836273/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC3836273/pdf/nihms525275.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC3836273/pdf/nihms525275.pdf)

## Abstract

The scavenger receptor class B type 1 (SCARB1) gene is a key component in the reverse cholesterol transport pathway and thus plays an important role in lipid metabolism. Studies suggested that the SCARB1 gene may contribute to variation in plasma lipid levels at the fasting; however, the results have been inconsistent and it is unclear if SCARB1 may also influence lipid response to dietary and pharmacologic interventions. In this study, we examined genetic variation in the SCARB1 gene in participants of the Genetics of Lipid Lowering Drugs and Diet Network (GOLDN) study for associations with basal lipid levels, changes in lipid measures after dietary fat intake and fenofibrate treatment. We found that the exon 1 variant SCARB1_G2S was significantly associated with post-fenofibrate change for triglyceride (TG) (P = 0.004). Subjects bearing SCARB1_G2S minor allele A tend to have higher responsiveness to fenofibrate in lowering TG. In summary, our study suggested that the SCARB1 gene may serve as a useful marker that predicts variation in baseline lipid levels, postprandial lipid response as well as response to fenofibrate intervention.

Keywords: Scavenger receptor class B type 1, lipid, genetics, fenofibrate, postprandial

## INTRODUCTION

Plasma triglyceride (TG) and cholesterol levels are major risk factors of coronary heart disease (CHD) ([Cullen et al. 1997](#R10)). High-density lipoprotein cholesterol (HDL-C) levels inversely correlated with CHD risk, suggesting HDL-C protects against atherosclerosis ([Gordon and Rifkind 1989](#R14); [Uiterwaal et al. 1994](#R31)). TGs, LDL and HDL particles and particle size are highly reproducible, and are sensitive to dietary manipulation and lipid-lowering drugs ([Tsai et al. 1992](#R29); [Yuan et al. 1994](#R33)). Studies showed that both the fasting lipid levels and the postprandial lipid response are modulated by genetic factors ([Mero et al. 1998](#R19); [Ordovas 2001](#R20)). In addition, several lines of evidence suggested that lipid response to pharmacological interventions is highly variable and has genetic control ([Suter et al. 2001](#R28); [Durstine et al. 2001](#R11); [Kuller et al. 2001](#R17)). However, these underlying genetic factors are largely unknown.

The scavenger receptor class B type 1 (*SCARB1*) gene is the first reported HDL receptor ([Acton et al. 1996](#R5)). *SCARB1* mediates selective uptake of HDL-C without degradation of entire HDL particles ([Acton et al. 1996](#R5)). Over-expression of *SCARB1* in mice resulted in dramatic reductions in plasma HDL-C concentrations ([Kozarsky et al. 1997](#R16)). Conversely, suppression of *SCARB1* expression were related to a marked increase in plasma HDL-C, characterized by enlarged cholesterol-rich HDL particles and impaired HDL-C clearance ([Rigotti et al. 1997](#R26)). As a multilipoprotein receptor, *SCARB1* also regulates the concentrations of LDL-C and very low density lipoprotein (VLDL) ([Acton et al. 1996](#R5); [Ueda et al. 1999](#R30)). Interestingly, *SCARB1* is likely to be involved in intestinal absorption of TGs ([Hauser et al. 1998](#R15)). Importantly, the *SCARB1* gene is located on chromosome 12q24, a region showing significant linkage to plasma HDL-C and TG levels ([Feitosa et al. 2005](#R12)). Several previous studies also reported associations between the *SCARB1* gene polymorphisms and plasma cholesterol levels in humans; however, the results have been inconsistent ([Acton et al. 1999](#R4)) and it is unclear whether the *SCARB1* gene is involved in modulating postprandial lipid response and response to pharmacological interventions.

Stimulated by the importance of *SCARB1* in lipid metabolism, in this study we systemically investigated whether the *SCARB1* gene contributes to baseline lipid levels, lipid response to dietary fat challenge and, for the first time, TG lowering drug therapy in participants of the Genetics of Lipid Lowering Drugs and Diet Network (GOLDN) study.

## MATERIALS AND METHODS

### Study population

The study population came from the GOLDN study ([http://www.biostat.wustl.edu/goldn/](http://www.biostat.wustl.edu/goldn/)). The participants of GOLDN study were mainly re-recruited from two NHLBI Family Heart Study (FHS) field centers: Minneapolis, MN, and Salt Lake City, UT. Nearly all subjects were of European ancestry. Eligibility criteria were: 1) ≥ 18 years of age; 2) fasting TGs < 1500 mg/dL; 3) willing to participate in the study and attend the scheduled clinic exams; 4) member of a family with at least two members in a sibship; 5) AST and ALT results within normal range; and 6) creatinine ≤ 2.0 mg/dL. Exclusion criteria were: 1) history of liver, kidney, pancreas, gall bladder disease, or malabsorption; 2) current pregnancy; 3) insulin use; 4) use of lipid lower drugs (including prescription, OTC and nutraceuticals; volunteers taking these agents were withdrawn from them at least four weeks prior to the study with physician’s approval; 5) use of warfarin; 6) women of childbearing potential not using an acceptable form of contraception; 7) known hypersensitivity to fenofibrate; and 8) history of pancreatitis within 12 months prior to enrollment.

Our previous studies demonstrated that Caucasians in UT and MN were homogeneous and thus pooling data across centers will not threaten the validity of this study ([Pankow et al. 2002](#R24)).

### Dietary and pharmacological Interventions

All family members underwent a baseline screening visit (Visit 0). The eligible subjects were asked to participate in a clinical examination with dietary fat intake and the fenofibrate trial. The dietary fat challenge was conducted to assess baseline and postprandial TGs and related phenotypes. The fat challenge consisted of eating a test meal during a 15-minute period and returning to the clinic 3.5 and 6 hrs later to collect blood samples. The caloric intake of the intervention meal was determined by body surface area. The meal derived 83% of calories from fat and contains 700 calories per m^2^ of body surface area. We measured the total TGs, TG in chylomicrons, VLDL and TG remnant particles, HDL and LDL particle sizes, total cholesterol, LDL-C, and HDL-C at baseline and at 3.5 and 6 hrs after consumption of the meal.

This fat challenge was done twice, once while participants were taking no lipid-lowering medications, and once while taking fenofibrate. The fenofibrate intervention consisted of a three-week treatment period, in which participants took oral fenofibrate 160 mg tablets once daily. We measured lipids twice after a minimum fast of 10 hours on the last two days of the treatment period (Visits 3 and 4). The intervention procedure is displayed in [Supplementary data (supplementary Figure 1)](#SD1).

### Plasma lipid and lipoprotein measurements

The fasting, 3.5, and 6 hr postprandial lipoprotein samples were centrifuged within 20 min of collection at 2000*g* for 15 min at 4 °C. Plasma samples of each participant were stored at 4°C until completion of the treatment period. All samples from a single participant were analyzed within the same batch. TGs were measured using a glycerol blanked enzymatic method (Trig/GB, Roche Diagnostics Corporation, Indianapolis, IN) on the Roche/Hitachi 911 Automatic Analyzer (Roche Diagnostics Corporation). Cholesterol was measured on the Hitachi 911 using a cholesterol esterase, cholesterol oxidase reaction (Chol R1, Roche Diagnostics Corporation). The same reaction was also used to measure HDL-C after precipitation of non-HDL-C with magnesium/dextran. LDL-C was measured by a homogeneous direct method (LDL Direct Liquid Select^™^ Cholesterol Reagent, Equal Diagnostics, Exton, PA) on the Hitachi 911. TGs were measured at all time points, i.e., 0, 3.5, and 6 hrs before/after fenofibrate trial. HDL-C and LDL-C were only measured at baseline before and after fenofibrate exposure.

Plasma lipoprotein particles and subclass distributions were determined by proton nuclear magnetic resonance (NMR) spectroscopy as previously described ([Ordovas et al. 2000](#R21); [Otvos et al. 1992](#R23)). Each profile displays the concentrations of 3 VLDL, 1 IDL, 3 LDL, and 3 HDL subclasses and the weighted-average particle sizes of VLDL, LDL, and HDL. The lipoprotein subclass categories used were the following: large LDL (21.3 to 27.0 nm), intermediate LDL (19.8 to 21.2), small LDL (18.3 to 19.7 nm), large HDL (8.8 to 13.0 nm), intermediate HDL (7.8 to 8.8 nm), and small HDL (7.3 to 7.7 nm). Levels of LDL and HDL subclasses are expressed in nmol units of cholesterol. LDL and HDL subclass distributions determined by gradient gel electrophoresis and NMR have been shown to be closely correlated ([Otvos et al. 1992](#R23)). HDL, LDL, VLDL particles and subclass were measured at all time points (0, 3.5 and 6 hrs) before/after fenofibrate trial.

### SNP Selection and Genotyping

SNPs were identified by searching public databases such as dbSNP ([http://www.ncbi.nlm.nih.gov/SNP/](http://www.ncbi.nlm.nih.gov/SNP/)). We selected 7 SNPs at the *SCARB1* gene locus based on the following criteria, in order of importance in our selection scheme: 1) validation status; 2) functional relevance and importance; 3) degree of heterozygosity, i.e., minor allele frequencies (MAF) > 0.0; and 4) previous evidence of association with lipid measurements.

SNPs were genotyped by using the 5′ nuclease allelic discrimination Taqman assay with allelic specific probes on the ABI Prism 7900HT Sequence Detection System (Applied Biosystems, Foster City, CA, USA) according to standard laboratory protocols. We used GRR software ([Abecasis et al. 2001](#R1)) to detect pedigree errors via graphically inspecting the distribution for marker allele sharing among pairs of family members or all pairs of individuals. The SNP allele frequencies were estimated via a maximum-likelihood method ([Boehnke 1991](#R8)). For each SNP, a χ^2^ test was used to examine deviation of SNP genotypes from Hardy-Weinberg equilibrium (HWE) ([Fisher RA 1934](#R13)). The overall genotyping error and missing rate was ~1%.

### Statistical analyses

Pairwise linkage disequilibrium (LD) between SNPs was calculated using the normalized measure, Lewontin’s *D,* ([LEWONTIN 1964](#R18)), using the publicly available HAPLOVIEW version 3.3 software ([Barrett et al. 2005](#R6)). Normality for continuous variables was checked. After logarithmic transformations for TGs, HDL-C and HDL particle size, they approximately follow normal distribution.

Associations were tested between the *SCARB1* SNPs and a number of lipid profiles, which include: 1) baseline lipid levels (including TGs, HDL-C, LDL-C, HDL and LDL particles and particle sizes); 2) changes of lipid profile between pre- and post-fenofibrate treatments; 3) changes of lipid profile at 0–3.5 hr (which, biologically, represents the absorption phase of TGs after fat meal); and 4) changes of lipid profile at 3.5–6 hr (which, biologically, represents the clearance phase of TGs after fat meal).

Genotype-phenotype association analyses were performed using a linear mixed model that is implemented in SAS (version 9.1, SAS Institute, Cary, NC). In the model, genotypes were treated as fixed effects and the dependencies among members within each family were treated as random effects. Age, age^2^, age^3^, sex, and field center were included as covariates. The model can be defined as:

| yit=α+β×t+γiα+γiβ×t+εit |
| --- |
where *yit* is the phenotype of individual *i* measured at time point *t. α* and *β* are the intercept and slope for the population level effects (fixed effects) respectively, while *γiα* and *γiβ* are for the individual level effects (random effects) respectively. *εit* is the residual. The model was implemented in the PROC MIXED procedure in SAS. Since the growth curve slope (*β +γiβ*) eliminates the noise *εit*, it represents an accurate and stable estimate of lipid changes.

Haplotypes were reconstructed based on the 7 SNPs. The software MERLIN version 1.0.1 ([Abecasis et al. 2002](#R2)) was used for haplotype inference. MERLIN can accommodate LD between SNPs in haplotype inference ([Abecasis and Wigginton 2005](#R3)). Association analysis for haplotypes was conducted using a mixed model similar to that for single SNPs. In addition, a global test was performed to examine whether phenotype differences existed among all haplotypes simultaneously.

## RESULTS

### Study population

Characteristics of the study subjects are shown in [Table 1](#T1). A total of 1,327 subjects (639 men and 688 women) from 148 families were genotyped. Of these 1,327 subjects, 861 subjects (427 men and 434 women) went through the fenofibrate trial and had complete lipid phenotypes (TGs, HDL-C and LDL-C) and genotype data. Men ranged in age from 18.0 to 87.6 (50.6 ± 15.9) yr, and women ranged from 18.0 to 87.2 (51.1 ± 15.8) yr. For both men and women, at all the time points, TG, LDL-C, and LDL particles after fenofibrate treatment were significantly lower than those before fenofibrate treatment (*P* <0.001). Conversely, HDL-C and HDL particles after fenofibrate treatment were significantly higher than those before fenofibrate treatment (*P* < 0.001). Post-fenofibrate HDL sizes generally increased, compared to those before fenofibrate treatment at each corresponding time point for both men and women. The changes of VLDL and LDL particle sizes before/after treatment were not in a consistent pattern for both men and women.

### Table 1.

| Men |  |  |  |  |  |  |
| --- | --- | --- | --- | --- | --- | --- |
| Measure | Pre-fenofibrate |  |  | Post-fenofibrate |  |  |
| Baseline | 3.5 hr | 6 hr | Baseline | 3.5 hr | 6 hr |  |
| Age (yr) | 50.6 (15.9) | – | – | – | – | – |
| Triglyceride (mg/dL) | 153.41 (142.01) | 286.05 (193.49) | 270.41 (219.68) | 99.18 (59.83) | 190.91 (109.88) | 164.41 (107.41) |
| HDL-C (mg/dL) | 41.57 (9.84) | NA | NA | 43.67 (9.99) | NA | NA |
| LDL-C (mg/dL) | 123.17 (30.32) | NA | NA | 111.45 (32.79) | NA | NA |
| HDL particles (nmol/L) | 29.13 (4.98) | 28.64 (4.83) | 29.71 (4.61) | 30.57 (5.36) | 30.47 (4.86) | 31.44 (4.99) |
| Large | 4.71 (2.70) | 4.24 (2.91) | 4.39 (3.06) | 4.42 (2.31) | 4.24 (2.55) | 4.38 (2.71) |
| Intermediate | 1.99 (2.67) | 4.01 (3.58) | 5.80 (4.39) | 4.15 (3.90) | 6.65 (4.53) | 8.78 (5.09) |
| Small | 22.42 (5.02) | 20.39 (5.08) | 19.50 (5.39) | 22.00 (6.06) | 19.56 (5.97) | 18.27 (6.52) |
| LDL particles (nmol/L) | 1445.10 (461.59) | 1420.64 (454.14) | 1429.91 (464.17) | 1275.65 (397.52) | 1267.66 (403.91) | 1267.28 (414.65) |
| Large | 319.12 (228.59) | 357.44 (242.54) | 380.15 (253.89) | 383.93 (205.20) | 402.42 (211.15) | 433.01 (213.67) |
| Intermediate | 48.17 (52.05) | 79.76 (74.32) | 73.59 (69.87) | 35.01 (40.01) | 47.36 (46.97) | 43.42 (39.25) |
| Small | 1077.80 (520.89) | 983.43 (502.99) | 976.16 (533.93) | 856.71 (394.12) | 817.87 (403.30) | 790.84 (423.83) |
| HDL size (nm) | 8.65 (0.38) | 8.66 (0.41) | 8.73 (0.40) | 8.85 (0.40) | 8.95 (0.41) | 9.02 (0.39) |
| LDL size (nm) | 20.48 (0.78) | 20.61 (0.78) | 20.68 (0.80) | 21.01 (0.57) | 21.05 (0.60) | 21.12 (0.60) |
| VLDL size (nm) | 51.38 (7.42) | 55.72 (8.37) | 58.86 (9.88) | 51.56 (8.89) | 55.63 (9.55) | 61.09 (13.25) |

Table 1 Caption: Characteristics of study subjects

### SNP genotyping and linkage disequilibrium

Information of the 7 tested SNPs is summarized in [Supplementary Table 1](#SD1). All the SNPs were in Hardy-Weinberg equilibrium. The allele frequencies of all the SNPs were greater than 0.10. LD coefficients (Lewontin’s *D*′) between each pair of the SNPs are shown in [Supplementary Table 2](#SD1). We found substantial LD (*D*′ ≥ 0.50) among several pairs of SNPs. The strongest LD (*D*′ = 0.904) was found between SCARB1_I51973 (intron 5) and SCARB1_A350A (exon 8).

### Association analyses for single SNPs

[Table 2](#T2) summarizes the association results between SNPs and lipid response to fenofibrate treatment in the total sample. Overall, fenofibrate was associated with a 27.2 mg/dl reduction in TG level. The SNP *SCARB1*_G2S was associated with fenofibrate response for both TG (*P* = 0.004) and HDL-C (*P* = 0.01). The SNP *SCARB1*_I82699 was associated with fenofibrate response for TG (*P* = 0.02). [Figure 1](#F1) displays the changes of HDL-C and TG for *SCARB1*_G2S genotypes. It can be seen that subjects bearing the allele *A* had larger changes for both HDL-C and TG, suggesting higher response to fenofibrate treatment by increasing plasma HDL-C level and lowering TG level.

### Table 2.

|   | P Value * |  |  |
| --- | --- | --- | --- |
| HDL-C Change | LDL-C Change | TG Change |  |
| SCARB1_G2S | 0.01 | 0.65 | 0.004 |
| SCARB1_I9107 | 0.22 | 0.23 | 0.12 |
| SCARB1_19960 | 0.07 | 0.38 | 0.78 |
| SCARB1_I30026 | 0.65 | 0.42 | 0.35 |
| SCARB1_I51973 | 0.56 | 0.89 | 0.85 |
| SCARB1_A350A | 0.77 | 0.86 | 0.09 |
| SCARB1_I82699 | 0.39 | 0.91 | 0.02 |

Table 2 Caption: Association of SCARB1 SNPs with lipid response to fenofibrate treatment *

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/408b/3836273/3b061c9c5070/nihms525275f1.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3836273_nihms525275f1.jpg)

Changes of HDL-C and TG with SCARB1_G2S genotypes in response to fenofibrate treatment. Changes (mean ± SE) of HDL-C and TG (Y axis) were growth curve slopes estimated from the mixed models.

[Table 3](#T3) summarizes the association results of the lipid measures at the baseline (0 hr) and the lipid changes across time points of 0–3.5 hr (0/3.5) and 3.5–6 hr (3.5/6). The most pronounced association was found for the SNP *SCARB1*_G2S. In the pre-fenofibrate phase, a significant association was found between *SCARB1*_G2S and baseline HDL (*P* = 0.001), HDL_0–3.5_ (*P* = 0.04), and changes of HDL particle size at time period 0–3.5 hr (*P* = 0.03). *SCARB1*_G2S also showed associations with baseline TG (*P* = 0.004), TG_0–3.5_ (*P* = 0.005), and baseline LDL (*P* = 0.03) (see [Table 3](#T3)).

### Table 3.

| Pre-fenofibrate |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| SNP | HDL |  |  | LDL |  |  | HDL size |  |  | LDL size |  |  | VLDL size |  |  | TGs |  |  |
| 0 | 0/3.5 | 3.5/6 | 0 | 0/3.5 | 3.5/6 | 0 | 0/3.5 | 3.5/6 | 0 | 0/3.5 | 3.5/6 | 0 | 0/3.5 | 3.5/6 | 0 | 0/3.5 | 3.5/6 |  |
| SCARB1_G2S | 0.001 | 0.04 | 0.05 | 0.03 | 0.13 | 0.82 | 0.83 | 0.05 | 0.41 | 0.54 | 0.25 | 0.41 | 0.12 | 0.31 | 0.50 | 0.004 | 0.005 | 0.31 |
| SCARB1_I9107 | 0.85 | 0.44 | 0.66 | 0.67 | 0.51 | 0.52 | 0.02 | 0.70 | 0.97 | 0.36 | 0.52 | 0.97 | 0.22 | 0.53 | 0.98 | 0.77 | 0.70 | 0.65 |
| SCARB1_19960 | 0.54 | 0.46 | 0.41 | 0.39 | 0.24 | 0.52 | 0.05 | 0.85 | 0.99 | 0.77 | 0.90 | 0.99 | 0.71 | 0.44 | 0.56 | 0.94 | 0.15 | 0.32 |
| SCARB1_I30026 | 0.38 | 0.80 | 0.15 | 0.76 | 0.37 | 0.67 | 0.81 | 0.85 | 0.73 | 0.76 | 0.17 | 0.73 | 0.08 | 0.73 | 0.44 | 0.47 | 0.44 | 0.19 |
| SCARB1_I51973 | 0.65 | 0.92 | 0.73 | 0.90 | 0.71 | 0.95 | 0.55 | 0.65 | 0.21 | 0.81 | 0.86 | 0.21 | 0.57 | 0.26 | 0.58 | 0.43 | 0.21 | 0.31 |
| SCARB1_A350A | 0.20 | 0.13 | 0.53 | 0.10 | 0.68 | 0.02 | 0.40 | 0.67 | 0.41 | 0.42 | 0.21 | 0.41 | 0.14 | 0.87 | 0.71 | 0.28 | 0.36 | 0.48 |
| SCARB1_I82699 | 0.92 | 0.78 | 0.90 | 0.29 | 0.20 | 0.58 | 0.10 | 0.23 | 0.72 | 0.09 | 0.14 | 0.72 | 0.87 | 0.95 | 0.48 | 0.04 | 0.37 | 0.01 |

Table 3 Caption: Association tests of SCARB1 SNPs with baseline lipid measures and response to dietary fat *

For post-fenofibrate lipid measures, significant associations were observed for *SCARB1*_G2S with baseline HDL particles (*P* = 0.002), HDL_3.5–6_, and HDL size at time period of 3.5–6 hr (*P* = 0.0002). However, no association was found between *SCARB1*_G2S and baseline TG or TG response to fat loading.

### Association analyses for haplotypes

A total of 51 haplotypes were observed in our family data. To maximize the statistical power and to minimize potential bias caused by small number of subjects bearing a specific haplotype, we only tested associations between common haplotypes (allele frequencies ≥ 0.05) and lipid measures. [Table 4](#T4) illustrates the frequencies and association information of the 6 common haplotypes (*H1–H6*). No significant association was observed between any single haplotype with response to fenofibrate treatment. However, the haplotype *H1* showed significantly association with pre-fenofibrate baseline HDL (*P* = 0.002) and marginally association with HDL*3.5–6* (*P* = 0.07). *H1* was also association with pre-fenofibrate TG*0–3.5* (*P* = 0.02). The haplotype *H2* was associated with post-fenofibrate HDL*3.5–6* and HDL size*3.5–6*. Other haplotype not specified here showed no significantly association with any lipid phenotypes.

### Table 4.

| Haplotype | SCARB1_G2S | SCARB1_I9107 | SCARB1_19960 | SCARB1_I30026 | SCARB1_I51973 | SCARB1_A350A | SCARB1_I82699 | Haplotype frequency | Association results |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| H1 | G | C | G | G | G | C | G | 0.18 | Pre: HDL0 (P=0.002); HDL3.5–6 (P=0.07); TG0–3.5 (P=0.02)Post: HDL0 (P=0.01); |
| H2 | G | G | A | G | G | T | G | 0.13 | Post: HDL3.5–6 (P=0.03); HDL size3.5–6 (P=0.001) |
| H3 | G | G | G | A | G | C | G | 0.09 | - |
| H4 | G | G | G | G | G | T | G | 0.08 | - |
| H5 | G | G | A | A | G | C | G | 0.06 | - |
| H6 | G | C | G | G | A | T | G | 0.06 | - |

Table 4 Caption: Association tests of SCARB1 haplotypes with lipid phenotypes

## DISCUSSION

In this study we systemically investigated the relationship between the *SCARB1* gene and lipid profiles at the baseline and postprandial states, as well as lipid lowering treatment. Dietary fat challenges and pharmacologic intervention are two potent stimuli that produce large but highly variable changes in lipid profile ([Suter et al. 2001](#R28); [Durstine et al. 2001](#R11); [Kuller et al. 2001](#R17)). We undertook two interventions (i.e., dietary fat challenge and pharmacologic intervention), because by studying contexts that affect both axes (i.e., increasing and decreasing TGs) we may gain novel insight into underlying mechanisms that the *SCARB1* gene may regulate the lipids. The most interesting finding was with the SNP *SCARB1*_G2S. The allele *A* of this SNP, compared to the alternative *G,* was associated with a higher response to fenofibrate by lowering TG levels and increasing plasma HDL-C levels (but the significance with HDL-C disappeared after adjusting for multiple testing).

Several previous studies examined association between the *SCARB1* gene polymorphisms and lipid measures in different human populations. In a white population, male carriers of the 2 allele (1/2) of *SCARB1*_G2S had increased HDL-C and reduced LDL-C and of TGs concentrations at baseline ([Acton et al. 1999](#R4)). In a dietary intervention study, carriers of this allele appeared to be more responsive to changes in dietary saturated fat intake, as they exhibited a greater increase in LDL-C compared with 1/1 individuals ([Perez-Martinez et al. 2003](#R25)). In the Framingham Study, diabetic subjects carrying the *SCARB1*_G2S *A* allele had significantly lower LDL-C and HDL-C concentrations ([Osgood et al. 2003](#R22)). Our results reported herein are in agreement with these earlier findings. An important extension of the present study is that we for the first time found the *SCARB1* gene is also related to lipid response to pharmacological (fenofibrate) treatment.

Mice studies showed that the genetic expression of *SCARB1* modifies the metabolism of HDL-C. Overexpression of *SCARB1* in the liver lowers plasma values for HDL-C and increases the concentration and biliary secretion of cholesterol ([Kozarsky et al. 1997](#R16); [Wang et al. 1998](#R32)). In contrast, the total suppression of its expression increases plasma HDL-C and lowers the cholesterol content in bile and the suprarenal gland. After feeding a meal containing [^14^C] cholesterol and [^3^H] triolein, mice of overexpression of *SCARB1* primarily in the intestine presented a rise in intestinal absorption of both lipids that was not due to a defect in chylomicron clearance nor to a change in the bile flow or the bile acid content ([Bietrix et al. 2006](#R7)), suggesting that *SCARB1* acts as a multi-ligand transporter in the small intestine and is responsible for the accelerated uptake of cholesterol and TG hydrolysis products. The mechanism by which *SCARB1* delivers cholesterol esters to cells is not well understood, but it appears to be different from the well characterized LDL receptor endocytic pathway ([Brown and Goldstein 1986](#R9)). It is likely that *SCARB1* mediates uptake of lipids from both LDL and HDL, and this uptake is associated with an increase in cholesterol esterification ([Stangl et al. 1998](#R27)).

The physiological mechanism of *SCARB1* in the postprandial lipemic response and pharmacologic intervention awaits further molecular/physiological studies to elucidate. Nonetheless, our findings from this study, along with those of others, suggest a potential mechanism that the *SCARB1* gene polymorphisms may be related to a modified hepatic and/or intestinal expression of the *SCARB1* gene, through which *SCARB1* influences lipid absorption and clearance. In particular, the exon 1 polymorphism of *SCARB1*_G2S results in an amino acid change in the protein, which may modify receptor activity. Based on the current understanding of this exon 1variant, it is unlikely that it directly impairs selective cholesterol uptake. This is because selective cholesterol uptake resides primarily in the extracellular domain of the receptor, but the exon 1 amino acid change occurs in its intracellular N-terminus. Further in vitro and in vivo studies are needed to elucidate the exact functional effects of this *SCARB1* exon 1 variant.

A major strength of this study lies in that the participants underwent rigorous interventions of both fat loading dietary challenge and fenofibrate trial, which provided a valuable resource to investigate the *SCARB1* gene regarding its relationship with baseline lipid levels, postprandial lipid response and response to pharmacological treatment. Despite this asset, our study may have some limitations. First, although multiple SNPs were investigated across the gene, the marker density is not sufficient to cover the whole gene. To examine the gene more comprehensively, further studies by testing more SNPs, especially tagSNPs that cover all LDs in *SCARB1*, should be pursued. Second, a number of tests were performed due to the multiple lipid measures and SNPs involved. After correction for multiple testing, some identified associations only reached nominal significance or even disappeared. For example, the association between *SCARB1*_G2S and HDL-C in response to fenofibrate treatment turned out to be non-significant after adjustment for multiple testing. However, since the tested lipid measures are highly correlated and the tested SNPs are in LD, Bonferroni correction may be too conservative. Third, since our analyses were exploratory, the results await replication studies and eventually functional studies to validate. Due to the associated high costs and difficulties in compliance with the strict dietary and pharmacological interventions, we currently lack powerful independent datasets (including those from other groups) to replicate our study findings. We will pursue replication/confirmation in our future endeavors.

In summary, the present study suggests that the *SCARB1* gene polymorphisms may serve as a useful marker to predict baseline lipid measures, postprandial lipid response as well as fenofibrate response. Our results may have significant clinical implications if confirmed by future studies.

## Supplementary Material

## Acknowledgments

We thank the families for their participation in this research. This work was supported by NIH Heart, Lung and Blood Institute grant U 01 HL72524, Genetic and Environmental Determinants of Triglycerides. We wish to acknowledge Abbott Laboratories (Abbott Park, IL) for their supply of study medication for this project. Dr. David B. Allison is supported by NIH grant 3P30DK056336. Dr. Kui Zhang is supported by NIH grant R01GM74913.

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Abecasis GR, Cherny SS, Cookson WO, Cardon LR. GRR: graphical representation of relationship errors. Bioinformatics. 2001;17 (8):742–743. doi: 10.1093/bioinformatics/17.8.742.  [DOI](https://doi.org/10.1093/bioinformatics/17.8.742) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11524377/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=GRR:%20graphical%20representation%20of%20relationship%20errors&author=GR%20Abecasis&author=SS%20Cherny&author=WO%20Cookson&author=LR%20Cardon&volume=17&issue=8&publication_year=2001&pages=742-743&pmid=11524377&doi=10.1093/bioinformatics/17.8.742&)

2. Abecasis GR, Cherny SS, Cookson WO, Cardon LR. Merlin--rapid analysis of dense genetic maps using sparse gene flow trees. Nat Genet. 2002;30 (1):97–101. doi: 10.1038/ng786.  [DOI](https://doi.org/10.1038/ng786) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11731797/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Merlin--rapid%20analysis%20of%20dense%20genetic%20maps%20using%20sparse%20gene%20flow%20trees&author=GR%20Abecasis&author=SS%20Cherny&author=WO%20Cookson&author=LR%20Cardon&volume=30&issue=1&publication_year=2002&pages=97-101&pmid=11731797&doi=10.1038/ng786&)

3. Abecasis GR, Wigginton JE. Handling marker-marker linkage disequilibrium: pedigree analysis with clustered markers. Am J Hum Genet. 2005;77 (5):754–767. doi: 10.1086/497345.  [DOI](https://doi.org/10.1086/497345) | [PMC free article](/articles/PMC1271385/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16252236/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=Handling%20marker-marker%20linkage%20disequilibrium:%20pedigree%20analysis%20with%20clustered%20markers&author=GR%20Abecasis&author=JE%20Wigginton&volume=77&issue=5&publication_year=2005&pages=754-767&pmid=16252236&doi=10.1086/497345&)

4. Acton S, Osgood D, Donoghue M, Corella D, Pocovi M, Cenarro A, Mozas P, Keilty J, Squazzo S, Woolf EA, Ordovas JM. Association of polymorphisms at the SR-BI gene locus with plasma lipid levels and body mass index in a white population. Arterioscler Thromb Vasc Biol. 1999;19 (7):1734–1743. doi: 10.1161/01.atv.19.7.1734.  [DOI](https://doi.org/10.1161/01.atv.19.7.1734) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10397692/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arterioscler%20Thromb%20Vasc%20Biol&title=Association%20of%20polymorphisms%20at%20the%20SR-BI%20gene%20locus%20with%20plasma%20lipid%20levels%20and%20body%20mass%20index%20in%20a%20white%20population&author=S%20Acton&author=D%20Osgood&author=M%20Donoghue&author=D%20Corella&author=M%20Pocovi&volume=19&issue=7&publication_year=1999&pages=1734-1743&pmid=10397692&doi=10.1161/01.atv.19.7.1734&)

5. Acton S, Rigotti A, Landschulz KT, Xu S, Hobbs HH, Krieger M. Identification of scavenger receptor SR-BI as a high density lipoprotein receptor. Science. 1996;271 (5248):518–520. doi: 10.1126/science.271.5248.518.  [DOI](https://doi.org/10.1126/science.271.5248.518) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8560269/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=Identification%20of%20scavenger%20receptor%20SR-BI%20as%20a%20high%20density%20lipoprotein%20receptor&author=S%20Acton&author=A%20Rigotti&author=KT%20Landschulz&author=S%20Xu&author=HH%20Hobbs&volume=271&issue=5248&publication_year=1996&pages=518-520&pmid=8560269&doi=10.1126/science.271.5248.518&)

6. Barrett JC, Fry B, Maller J, Daly MJ. Haploview: analysis and visualization of LD and haplotype maps. Bioinformatics. 2005;21 (2):263–265. doi: 10.1093/bioinformatics/bth457.  [DOI](https://doi.org/10.1093/bioinformatics/bth457) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15297300/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=Haploview:%20analysis%20and%20visualization%20of%20LD%20and%20haplotype%20maps&author=JC%20Barrett&author=B%20Fry&author=J%20Maller&author=MJ%20Daly&volume=21&issue=2&publication_year=2005&pages=263-265&pmid=15297300&doi=10.1093/bioinformatics/bth457&)

7. Bietrix F, Yan D, Nauze M, Rolland C, Bertrand-Michel J, Coméra C, Schaak S, Barbaras R, Groen AK, Perret B, Tercé F, Collet X. Accelerated lipid absorption in mice overexpressing intestinal SR-BI. J Biol Chem. 2006;281 (11):7214–7219. doi: 10.1074/jbc.M508868200.  [DOI](https://doi.org/10.1074/jbc.M508868200) | [PMC free article](/articles/PMC2034750/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16421100/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=Accelerated%20lipid%20absorption%20in%20mice%20overexpressing%20intestinal%20SR-BI&author=F%20Bietrix&author=D%20Yan&author=M%20Nauze&author=C%20Rolland&author=J%20Bertrand-Michel&volume=281&issue=11&publication_year=2006&pages=7214-7219&pmid=16421100&doi=10.1074/jbc.M508868200&)

8. Boehnke M. Allele frequency estimation from data on relatives. Am J Hum Genet. 1991;48 (1):22–25.  [PMC free article](/articles/PMC1682747/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/1985459/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=Allele%20frequency%20estimation%20from%20data%20on%20relatives&author=M%20Boehnke&volume=48&issue=1&publication_year=1991&pages=22-25&pmid=1985459&)

9. Brown MS, Goldstein JL. A receptor-mediated pathway for cholesterol homeostasis. Science. 1986;232 (4746):34–47. doi: 10.1126/science.3513311.  [DOI](https://doi.org/10.1126/science.3513311) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/3513311/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=A%20receptor-mediated%20pathway%20for%20cholesterol%20homeostasis&author=MS%20Brown&author=JL%20Goldstein&volume=232&issue=4746&publication_year=1986&pages=34-47&pmid=3513311&doi=10.1126/science.3513311&)

10. Cullen P, Funke H, Schulte H, Assmann G. Lipoproteins and cardiovascular risk-from genetics to CHD prevention. J Atheroscler Thromb. 1997;4 (2):51–58. doi: 10.5551/jat1994.4.51.  [DOI](https://doi.org/10.5551/jat1994.4.51) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9638514/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Atheroscler%20Thromb&title=Lipoproteins%20and%20cardiovascular%20risk-from%20genetics%20to%20CHD%20prevention&author=P%20Cullen&author=H%20Funke&author=H%20Schulte&author=G%20Assmann&volume=4&issue=2&publication_year=1997&pages=51-58&pmid=9638514&doi=10.5551/jat1994.4.51&)

11. Durstine JL, Grandjean PW, Davis PG, Ferguson MA, Alderson NL, DuBose KD. Blood lipid and lipoprotein adaptations to exercise: a quantitative analysis. Sports Med. 2001;31 (15):1033–1062. doi: 10.2165/00007256-200131150-00002.  [DOI](https://doi.org/10.2165/00007256-200131150-00002) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11735685/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Sports%20Med&title=Blood%20lipid%20and%20lipoprotein%20adaptations%20to%20exercise:%20a%20quantitative%20analysis&author=JL%20Durstine&author=PW%20Grandjean&author=PG%20Davis&author=MA%20Ferguson&author=NL%20Alderson&volume=31&issue=15&publication_year=2001&pages=1033-1062&pmid=11735685&doi=10.2165/00007256-200131150-00002&)

12. Feitosa MF, Rice T, Rankinen T, Province MA, Chagnon YC, Gagnon J, Leon AS, Skinner JS, Wilmore JH, Després JP, Bouchard C, Rao DC, Borecki IB. Evidence of QTLs on chromosomes 13q and 14q for triglycerides before and after 20 weeks of exercise training: the HERITAGE Family Study. Atherosclerosis. 2005;182 (2):349–360. doi: 10.1016/j.atherosclerosis.2005.01.052.  [DOI](https://doi.org/10.1016/j.atherosclerosis.2005.01.052) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16159608/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Atherosclerosis&title=Evidence%20of%20QTLs%20on%20chromosomes%2013q%20and%2014q%20for%20triglycerides%20before%20and%20after%2020%20weeks%20of%20exercise%20training:%20the%20HERITAGE%20Family%20Study&author=MF%20Feitosa&author=T%20Rice&author=T%20Rankinen&author=MA%20Province&author=YC%20Chagnon&volume=182&issue=2&publication_year=2005&pages=349-360&pmid=16159608&doi=10.1016/j.atherosclerosis.2005.01.052&)

13. Fisher RA. Statistical methods for research workers. Oliver and Boyd; Edinburgh: 1934.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Statistical%20methods%20for%20research%20workers&author=RA%20Fisher&publication_year=1934&)

14. Gordon DJ, Rifkind BM. High-density lipoprotein--the clinical implications of recent studies. N Engl J Med. 1989;321 (19):1311–1316. doi: 10.1056/NEJM198911093211907.  [DOI](https://doi.org/10.1056/NEJM198911093211907) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/2677733/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=High-density%20lipoprotein--the%20clinical%20implications%20of%20recent%20studies&author=DJ%20Gordon&author=BM%20Rifkind&volume=321&issue=19&publication_year=1989&pages=1311-1316&pmid=2677733&doi=10.1056/NEJM198911093211907&)

15. Hauser H, Dyer JH, Nandy A, Vega MA, Werder M, Bieliauskaite E, Weber FE, Compassi S, Gemperli A, Boffelli D, Wehrli E, Schulthess G, Phillips MC. Identification of a receptor mediating absorption of dietary cholesterol in the intestine. Biochemistry. 1998;37 (51):17843–17850. doi: 10.1021/bi982404y.  [DOI](https://doi.org/10.1021/bi982404y) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9922151/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochemistry&title=Identification%20of%20a%20receptor%20mediating%20absorption%20of%20dietary%20cholesterol%20in%20the%20intestine&author=H%20Hauser&author=JH%20Dyer&author=A%20Nandy&author=MA%20Vega&author=M%20Werder&volume=37&issue=51&publication_year=1998&pages=17843-17850&pmid=9922151&doi=10.1021/bi982404y&)

16. Kozarsky KF, Donahee MH, Rigotti A, Iqbal SN, Edelman ER, Krieger M. Overexpression of the HDL receptor SR-BI alters plasma HDL and bile cholesterol levels. Nature. 1997;387 (6631):414–417. doi: 10.1038/387414a0.  [DOI](https://doi.org/10.1038/387414a0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9163428/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Overexpression%20of%20the%20HDL%20receptor%20SR-BI%20alters%20plasma%20HDL%20and%20bile%20cholesterol%20levels&author=KF%20Kozarsky&author=MH%20Donahee&author=A%20Rigotti&author=SN%20Iqbal&author=ER%20Edelman&volume=387&issue=6631&publication_year=1997&pages=414-417&pmid=9163428&doi=10.1038/387414a0&)

17. Kuller LH, Simkin-Silverman LR, Wing RR, Meilahn EN, Ives DG. Women’s Healthy Lifestyle Project: A randomized clinical trial: results at 54 months. Circulation. 2001;103 (1):32–37. doi: 10.1161/01.cir.103.1.32.  [DOI](https://doi.org/10.1161/01.cir.103.1.32) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11136682/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circulation&title=Women%E2%80%99s%20Healthy%20Lifestyle%20Project:%20A%20randomized%20clinical%20trial:%20results%20at%2054%20months&author=LH%20Kuller&author=LR%20Simkin-Silverman&author=RR%20Wing&author=EN%20Meilahn&author=DG%20Ives&volume=103&issue=1&publication_year=2001&pages=32-37&pmid=11136682&doi=10.1161/01.cir.103.1.32&)

18. Lewontin RC. The interaction of selection and linkage. ii optimum models. Genetics. 1964;50:757–82. 757–782. doi: 10.1093/genetics/50.4.757.  [DOI](https://doi.org/10.1093/genetics/50.4.757) | [PMC free article](/articles/PMC1210693/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14221879/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genetics&title=The%20interaction%20of%20selection%20and%20linkage.%20ii%20optimum%20models&author=RC%20Lewontin&volume=50&publication_year=1964&pages=757-82&pmid=14221879&doi=10.1093/genetics/50.4.757&)

19. Mero N, Van Tol A, Scheek LM, Van Gent T, Labeur C, Rosseneu M, Taskinen MR. Decreased postprandial high density lipoprotein cholesterol and apolipoproteins A-I and E in normolipidemic smoking men: relations with lipid transfer proteins and LCAT activities. J Lipid Res. 1998;39 (7):1493–1502.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/9684753/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Lipid%20Res&title=Decreased%20postprandial%20high%20density%20lipoprotein%20cholesterol%20and%20apolipoproteins%20A-I%20and%20E%20in%20normolipidemic%20smoking%20men:%20relations%20with%20lipid%20transfer%20proteins%20and%20LCAT%20activities&author=N%20Mero&author=A%20Van%20Tol&author=LM%20Scheek&author=T%20Van%20Gent&author=C%20Labeur&volume=39&issue=7&publication_year=1998&pages=1493-1502&pmid=9684753&)

20. Ordovas JM. Genetics, postprandial lipemia and obesity. Nutr Metab Cardiovasc Dis. 2001;11 (2):118–133.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/11434189/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nutr%20Metab%20Cardiovasc%20Dis&title=Genetics,%20postprandial%20lipemia%20and%20obesity&author=JM%20Ordovas&volume=11&issue=2&publication_year=2001&pages=118-133&pmid=11434189&)

21. Ordovas JM, Cupples LA, Corella D, Otvos JD, Osgood D, Martinez A, Lahoz C, Coltell O, Wilson PW, Schaefer EJ. Association of cholesteryl ester transfer protein-TaqIB polymorphism with variations in lipoprotein subclasses and coronary heart disease risk: the Framingham study. Arterioscler Thromb Vasc Biol. 2000;20 (5):1323–1329. doi: 10.1161/01.atv.20.5.1323.  [DOI](https://doi.org/10.1161/01.atv.20.5.1323) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10807749/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arterioscler%20Thromb%20Vasc%20Biol&title=Association%20of%20cholesteryl%20ester%20transfer%20protein-TaqIB%20polymorphism%20with%20variations%20in%20lipoprotein%20subclasses%20and%20coronary%20heart%20disease%20risk:%20the%20Framingham%20study&author=JM%20Ordovas&author=LA%20Cupples&author=D%20Corella&author=JD%20Otvos&author=D%20Osgood&volume=20&issue=5&publication_year=2000&pages=1323-1329&pmid=10807749&doi=10.1161/01.atv.20.5.1323&)

22. Osgood D, Corella D, Demissie S, Cupples LA, Wilson PW, Meigs JB, Schaefer EJ, Coltell O, Ordovas JM. Genetic variation at the scavenger receptor class B type I gene locus determines plasma lipoprotein concentrations and particle size and interacts with type 2 diabetes: the framingham study. J Clin Endocrinol Metab. 2003;88 (6):2869–2879. doi: 10.1210/jc.2002-021664.  [DOI](https://doi.org/10.1210/jc.2002-021664) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12788901/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Endocrinol%20Metab&title=Genetic%20variation%20at%20the%20scavenger%20receptor%20class%20B%20type%20I%20gene%20locus%20determines%20plasma%20lipoprotein%20concentrations%20and%20particle%20size%20and%20interacts%20with%20type%202%20diabetes:%20the%20framingham%20study&author=D%20Osgood&author=D%20Corella&author=S%20Demissie&author=LA%20Cupples&author=PW%20Wilson&volume=88&issue=6&publication_year=2003&pages=2869-2879&pmid=12788901&doi=10.1210/jc.2002-021664&)

23. Otvos JD, Jeyarajah EJ, Bennett DW, Krauss RM. Development of a proton nuclear magnetic resonance spectroscopic method for determining plasma lipoprotein concentrations and subspecies distributions from a single, rapid measurement. Clin Chem. 1992;38 (9):1632–1638.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/1326420/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Chem&title=Development%20of%20a%20proton%20nuclear%20magnetic%20resonance%20spectroscopic%20method%20for%20determining%20plasma%20lipoprotein%20concentrations%20and%20subspecies%20distributions%20from%20a%20single,%20rapid%20measurement&author=JD%20Otvos&author=EJ%20Jeyarajah&author=DW%20Bennett&author=RM%20Krauss&volume=38&issue=9&publication_year=1992&pages=1632-1638&pmid=1326420&)

24. Pankow JS, Province MA, Hunt SC, Arnett DK. Regarding “Testing for population subdivision and association in four case-control studies”. Am J Hum Genet. 2002;71 (6):1478–1480. doi: 10.1086/344582.  [DOI](https://doi.org/10.1086/344582) | [PMC free article](/articles/PMC378599/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12515254/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=Regarding%20%E2%80%9CTesting%20for%20population%20subdivision%20and%20association%20in%20four%20case-control%20studies%E2%80%9D&author=JS%20Pankow&author=MA%20Province&author=SC%20Hunt&author=DK%20Arnett&volume=71&issue=6&publication_year=2002&pages=1478-1480&pmid=12515254&doi=10.1086/344582&)

25. Pérez-Martínez P, Ordovás JM, López-Miranda J, Gómez P, Marín C, Moreno J, Fuentes F, Fernández de la Puebla RA, Pérez-Jiménez F. Polymorphism exon 1 variant at the locus of the scavenger receptor class B type I gene: influence on plasma LDL cholesterol in healthy subjects during the consumption of diets with different fat contents. Am J Clin Nutr. 2003;77 (4):809–813. doi: 10.1093/ajcn/77.4.809.  [DOI](https://doi.org/10.1093/ajcn/77.4.809) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12663276/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Clin%20Nutr&title=Polymorphism%20exon%201%20variant%20at%20the%20locus%20of%20the%20scavenger%20receptor%20class%20B%20type%20I%20gene:%20influence%20on%20plasma%20LDL%20cholesterol%20in%20healthy%20subjects%20during%20the%20consumption%20of%20diets%20with%20different%20fat%20contents&author=P%20P%C3%A9rez-Mart%C3%ADnez&author=JM%20Ordov%C3%A1s&author=J%20L%C3%B3pez-Miranda&author=P%20G%C3%B3mez&author=C%20Mar%C3%ADn&volume=77&issue=4&publication_year=2003&pages=809-813&pmid=12663276&doi=10.1093/ajcn/77.4.809&)

26. Rigotti A, Trigatti BL, Penman M, Rayburn H, Herz J, Krieger M. A targeted mutation in the murine gene encoding the high density lipoprotein (HDL) receptor scavenger receptor class B type I reveals its key role in HDL metabolism. Proc Natl Acad Sci USA. 1997;94 (23):12610–12615. doi: 10.1073/pnas.94.23.12610.  [DOI](https://doi.org/10.1073/pnas.94.23.12610) | [PMC free article](/articles/PMC25055/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9356497/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20USA&title=A%20targeted%20mutation%20in%20the%20murine%20gene%20encoding%20the%20high%20density%20lipoprotein%20(HDL)%20receptor%20scavenger%20receptor%20class%20B%20type%20I%20reveals%20its%20key%20role%20in%20HDL%20metabolism&author=A%20Rigotti&author=BL%20Trigatti&author=M%20Penman&author=H%20Rayburn&author=J%20Herz&volume=94&issue=23&publication_year=1997&pages=12610-12615&pmid=9356497&doi=10.1073/pnas.94.23.12610&)

27. Stangl SH, Cao G, Wyne KL, Hobbs HH. Scavenger receptor, class B, type I-dependent stimulation of cholesterol esterification by high density lipoproteins, low density lipoproteins, and nonlipoprotein cholesterol. J Biol Chem. 1998;%20;273(47):31002–31008. doi: 10.1074/jbc.273.47.31002.  [DOI](https://doi.org/10.1074/jbc.273.47.31002) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9812997/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=Scavenger%20receptor,%20class%20B,%20type%20I-dependent%20stimulation%20of%20cholesterol%20esterification%20by%20high%20density%20lipoproteins,%20low%20density%20lipoproteins,%20and%20nonlipoprotein%20cholesterol&author=SH%20Stangl&author=G%20Cao&author=KL%20Wyne&author=HH%20Hobbs&volume=%20;273&issue=47&publication_year=1998&pages=31002-31008&pmid=9812997&doi=10.1074/jbc.273.47.31002&)

28. Suter PM, Gerritsen-Zehnder M, Häsler E, Gürtler M, Vetter W, Hänseler E. Effect of alcohol on postprandial lipemia with and without preprandial exercise. J Am Coll Nutr. 2001;20 (1):58–64. doi: 10.1080/07315724.2001.10719015.  [DOI](https://doi.org/10.1080/07315724.2001.10719015) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11293469/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Am%20Coll%20Nutr&title=Effect%20of%20alcohol%20on%20postprandial%20lipemia%20with%20and%20without%20preprandial%20exercise&author=PM%20Suter&author=M%20Gerritsen-Zehnder&author=E%20H%C3%A4sler&author=M%20G%C3%BCrtler&author=W%20Vetter&volume=20&issue=1&publication_year=2001&pages=58-64&pmid=11293469&doi=10.1080/07315724.2001.10719015&)

29. Tsai MY, Yuan J, Hunninghake DB. Effect of gemfibrozil on composition of lipoproteins and distribution of LDL subspecies. Atherosclerosis. 1992;95 (1):35–42. doi: 10.1016/0021-9150(92)90173-e.  [DOI](https://doi.org/10.1016/0021-9150(92)90173-e) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/1642690/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Atherosclerosis&title=Effect%20of%20gemfibrozil%20on%20composition%20of%20lipoproteins%20and%20distribution%20of%20LDL%20subspecies&author=MY%20Tsai&author=J%20Yuan&author=DB%20Hunninghake&volume=95&issue=1&publication_year=1992&pages=35-42&pmid=1642690&doi=10.1016/0021-9150(92)90173-e&)

30. Ueda Y, Royer L, Gong E, Zhang J, Cooper PN, Francone O, Rubin EM. Lower plasma levels and accelerated clearance of high density lipoprotein (HDL) and non-HDL cholesterol in scavenger receptor class B type I transgenic mice. J Biol Chem. 1999;274 (11):7165–7171. doi: 10.1074/jbc.274.11.7165.  [DOI](https://doi.org/10.1074/jbc.274.11.7165) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10066776/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=Lower%20plasma%20levels%20and%20accelerated%20clearance%20of%20high%20density%20lipoprotein%20(HDL)%20and%20non-HDL%20cholesterol%20in%20scavenger%20receptor%20class%20B%20type%20I%20transgenic%20mice&author=Y%20Ueda&author=L%20Royer&author=E%20Gong&author=J%20Zhang&author=PN%20Cooper&volume=274&issue=11&publication_year=1999&pages=7165-7171&pmid=10066776&doi=10.1074/jbc.274.11.7165&)

31. Uiterwaal CS, Grobbee DE, Witteman JC, van Stiphout WA, Krauss XH, Havekes LM, de Bruijn AM, van Tol A, Hofman A. Postprandial triglyceride response in young adult men and familial risk for coronary atherosclerosis. Ann Intern Med. 1994;121 (8):576–583. doi: 10.7326/0003-4819-121-8-199410150-00004.  [DOI](https://doi.org/10.7326/0003-4819-121-8-199410150-00004) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8085689/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Intern%20Med&title=Postprandial%20triglyceride%20response%20in%20young%20adult%20men%20and%20familial%20risk%20for%20coronary%20atherosclerosis&author=CS%20Uiterwaal&author=DE%20Grobbee&author=JC%20Witteman&author=WA%20van%20Stiphout&author=XH%20Krauss&volume=121&issue=8&publication_year=1994&pages=576-583&pmid=8085689&doi=10.7326/0003-4819-121-8-199410150-00004&)

32. Wang N, Arai T, Ji Y, Rinninger F, Tall AR. Liver-specific overexpression of scavenger receptor BI decreases levels of very low density lipoprotein ApoB, low density lipoprotein ApoB, and high density lipoprotein in transgenic mice. J Biol Chem. 1998;273 (49):32920–32926. doi: 10.1074/jbc.273.49.32920.  [DOI](https://doi.org/10.1074/jbc.273.49.32920) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9830042/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=Liver-specific%20overexpression%20of%20scavenger%20receptor%20BI%20decreases%20levels%20of%20very%20low%20density%20lipoprotein%20ApoB,%20low%20density%20lipoprotein%20ApoB,%20and%20high%20density%20lipoprotein%20in%20transgenic%20mice&author=N%20Wang&author=T%20Arai&author=Y%20Ji&author=F%20Rinninger&author=AR%20Tall&volume=273&issue=49&publication_year=1998&pages=32920-32926&pmid=9830042&doi=10.1074/jbc.273.49.32920&)

33. Yuan J, Tsai MY, Hunninghake DB. Changes in composition and distribution of LDL subspecies in hypertriglyceridemic and hypercholesterolemic patients during gemfibrozil therapy. Atherosclerosis. 1994;110 (1):1–11. doi: 10.1016/0021-9150(94)90062-0.  [DOI](https://doi.org/10.1016/0021-9150(94)90062-0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7857363/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Atherosclerosis&title=Changes%20in%20composition%20and%20distribution%20of%20LDL%20subspecies%20in%20hypertriglyceridemic%20and%20hypercholesterolemic%20patients%20during%20gemfibrozil%20therapy&author=J%20Yuan&author=MY%20Tsai&author=DB%20Hunninghake&volume=110&issue=1&publication_year=1994&pages=1-11&pmid=7857363&doi=10.1016/0021-9150(94)90062-0&)
