# Probing Conformational Rescue Induced by a Chemical Corrector of F508del-Cystic Fibrosis Transmembrane Conductance Regulator (CFTR) Mutant

## Metadata
**Authors:** Wilson Yu, Patrick Kim Chiaw, Christine E Bear
**Journal:** The Journal of Biological Chemistry
**Date:** 2011 May 21
**DOI:** [10.1074/jbc.M111.239699](https://doi.org/10.1074/jbc.M111.239699)
**PMID:** 21602569
**PMCID:** PMC3137047
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3137047/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC3137047/pdf/zbc24714.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC3137047/pdf/zbc24714.pdf)

## Abstract

Cystic fibrosis (CF) is caused by mutations in the cystic fibrosis transmembrane conductance regulator (CFTR) gene that cause loss of function of the CFTR channel on the apical surface of epithelial cells. The major CF-causing mutation, F508del-CFTR, is misfolded, retained in the endoplasmic reticulum, and degraded. Small molecule corrector compounds have been identified using high throughput screens, which partially rescue the trafficking defect of F508del-CFTR, allowing a fraction of the mutant protein to escape endoplasmic reticulum retention and traffic to the plasma membrane, where it exhibits partial function as a cAMP-regulated chloride channel. A subset of such corrector compounds binds directly to the mutant protein, prompting the hypothesis that they rescue the biosynthetic defect by inducing improved protein conformation. We tested this hypothesis directly by evaluating the consequences of a corrector compound on the conformation of each nucleotide binding domain (NBD) in the context of the full-length mutant protein in limited proteolytic digest studies. Interestingly, we found that VRT-325 was capable of partially restoring compactness in NBD1. However, VRT-325 had no detectable effect on the conformation of the second half of the molecule. In comparison, ablation of the di-arginine sequence, R553XR555 (F508del-KXK-CFTR), modified protease susceptibility of NBD1, NBD2, and the full-length protein. Singly, each intervention led to a partial correction of the processing defect. Together, these interventions restored processing of F508del-CFTR to near wild type. Importantly, however, a defect in NBD1 conformation persisted, as did a defect in channel activation after the combined interventions. Importantly, this defect in channel activation can be fully corrected by the addition of the potentiator, VX-770.

Keywords: ABC Transporter, Chloride Channels, Protein Conformation, Protein Folding, Protein Sequence, Limited Proteolysis, Protein Conformation, Protein Misfolding

## Introduction

Most cases of cystic fibrosis are caused by the mutation leading to deletion of the phenylalanine at position 508 in the CFTR[5](#FN5) protein (F508del-CFTR) ([1](#B1), [2](#B2)). F508del-CFTR is misfolded, is retained in the endoplasmic reticulum (ER), and fails to be efficiently trafficked to the Golgi and plasma membrane. The absence of the regulated anion channel activity of CFTR at the apical membrane of affected epithelial cells lining the airways, pancreatic ducts, small intestine, and reproductive organs is the primary defect causing cystic fibrosis disease ([3](#B3)).

However, it is clear that the basic defect in biosynthesis and assembly of F508del-CFTR can be partially corrected by the activity of membrane-permeable small molecule compounds called “corrector compounds.” Certain small molecule correctors have been shown to bind directly to F508del-CFTR and partially restore biosynthesis and functional expression on the cell surface ([4](#B4)–[7](#B7)). Unfortunately, correction of the biosynthetic defect using small molecules available to the academic community is relatively poor, prompting efforts to understand the mechanism of action of pharmacological correctors. Such efforts are critical for developing effective treatments.

A high resolution structural model of CFTR is currently lacking. However, CFTR is a member of a large family of transport ATPases, called the ATP binding cassette transporter superfamily, and x-ray crystal structures exist for several of these family members ([8](#B8)–[11](#B11)). Two homology models of CFTR have been generated using the structure of the bacterial ATP binding cassette protein, Sav1866, and these models have been very helpful in providing insight into the assembly of CFTR as a multidomain membrane protein ([12](#B12)–[14](#B14)). These models highlight the inter-domain interfaces essential for coupling the two nucleotide binding domains (NBD1 and NBD2) as well as the interfaces that couple the nucleotide binding domains to the two membrane-spanning domains (MSDs). Electrophysiological studies of the channel activity of CFTR have revealed the importance of ATP binding and hydrolysis occurring at the interface between NBD1 and NBD2 in the regulation of channel activity by CFTR after its phosphorylation by protein kinase A ([15](#B15)). Chemical cross-linking and electrophysiological studies have also revealed the importance of the interface between the NBDs and the loops extending from the MSDs in coupling the catalytic domains to the gating of the anion permeation pathway ([16](#B16), [17](#B17)).

Recent biophysical and biochemical studies have uncovered a defect in the thermodynamic conformation of NBD1 induced by the F508del mutation (*i.e.* an intra-domain defect) ([18](#B18), [19](#B19)). In addition, limited proteolysis and chemical cross-linking studies revealed that there are multiple inter-domain interactions, which are likely perturbed by deletion of Phe^508^, including the interaction between NBD1 and the intracellular loop extending from the second membrane-spanning domain (MSD2) ([20](#B20)) as well as the interactions between the two membrane-spanning domains ([21](#B21)).

Small molecule corrector compounds (VRT-325 and Corr4a), which interact directly with the mutant protein, are thought to “repair” aberrant structural properties of F508del-CFTR. Studies by Clarke and colleagues ([6](#B6)) showed that VRT-325 and Corr4a treatment led to the partial restoration of wild type inter-domain interactions between the first and second halves of the protein via the two MSDs, MSD1 and MSD2. Our previous functional studies of purified and reconstituted full-length F508del-CFTR protein revealed that acute treatment with VRT-325 altered its ATPase activity, an activity of CFTR that is mediated by inter-domain interactions ([22](#B22)).

The efficacy of VRT-325 in promoting biosynthetic rescue is limited to ∼20% of WT-CFTR, suggesting that this intervention is relatively ineffective in mediating conformational rescue of the mutant protein. Potentially, the efficacy of such corrector compounds could be enhanced by targeting distinct steps along the folding pathway. Previous studies pointed to the beneficial effect of mutating a di-arginine sequence (R^553^*X*R^555^ to K^553^*X*K^555^) proximal to the signature motif of NBD1 in partially rescuing the processing of F508del-CFTR protein ([23](#B23)–[27](#B27)). It has been argued that this mutation may prevent retrograde trafficking of F508del-CFTR from the Golgi to the ER ([23](#B23), [25](#B25), [26](#B26)). Alternatively, ablation of the di-arginine sequence in the context of F508del-CFTR may promote improved folding, potentially by modifying the conformation of NBD1 and/or by mediating abnormal inter-domain interactions ([28](#B28)). It remains to be determined whether small molecule correctors mediate similar or distinct changes in conformation of the mutant protein.

In the current study, we show that chemical correction by VRT-325 and ablation of the di-arginine sequence mediate distinct conformational changes in F508del-CFTR and that in combination there is near wild type processing of the major CF mutant. Interestingly, a conformational defect in NBD1 persisted, and this was associated with impaired channel activation. Importantly, VX-770, a clinically effective potentiator compound ([29](#B29), [30](#B30)), completely rescued this residual channel activation defect.

## EXPERIMENTAL PROCEDURES

### 

#### 

Human Embryonic Kidney (HEK)-293 cells stably expressing F508del-CFTR and WT-CFTR obtained from Dr. Rotin (the Hospital for Sick Children, Toronto, Ontario, Canada) were maintained in Dulbecco modified Eagle's medium (DMEM) (MultiCell) supplemented with 10% fetal bovine serum (FBS) (Wisent), 0.1 mm non-essential amino acid (Invitrogen), and 5 μg/ml blasticidin (Invitrogen). HEK-293 cells stably expressing F508del-K*X*K-CFTR (where arginines 553 and 555 were mutated to lysines) were created by transfection of pcDNA 3.1zeo-F508del-K*X*K-CFTR using FuGENE 6 reagent (Roche Applied Science), and selection for specific stable cells was performed using 1 mg/ml of Zeocin (InvivoGen). Baby hamster kidney cells stably transfected with F508del-CFTR were obtained from Dr. Lukacs (McGill University, Montreal, Quebec, Canada) and maintained DMEM/F12 (50/50) supplemented with 10% FBS and methotrexate (200 μg/ml).

Baby hamster kidney cells stably transfected with F508del-CFTR or transiently transfected with F508del-K*X*K-CFTR were grown on 25-mm glass coverslips until a confluency of ∼70% was achieved. Cells were treated with Corr4a (10 μm) for 24 h. Cells were then fixed in 4% paraformaldehyde and permeabilized with 0.1% saponin diluted in PBS 1× solution as described previously ([31](#B31)). The localization of CFTR was detected and visualized using M3A7 antibody (5 μg/ml) (MAB3480 antibody obtained from Millipore, Billerica, ME) and Alexa Fluor 488-conjugated goat-anti-mouse secondary antibody (1:250) (obtained from Molecular Probes, Burlington, Ontario, Canada). Glass coverslips were mounted onto slides using VECTASHIELD mounting medium containing the nuclear stain DAPI (Vector Laboratories). Images were acquired using the Zeiss Axiovert 200 inverted fluorescence microscope equipped with a Hamamatsu back-thinned EM-CCD camera and analyzed using Volocity imaging software.

Experiments were performed as described previously ([32](#B32)) using HEK cells expressing F508del-CFTR, WT-CFTR, and F508del-K*X*K-CFTR. In brief, cells were grown to ∼90–100% confluency. Cells were loaded with NaI loading buffer (3 mm KNO_3_, 2 mm Ca(NO_3_)_2_, 11 mm glucose, 20 mm HEPES, and 136 mm NaI) at 37 °C for 1 h. NaI loading buffer was aspirated, and cells were washed four times in iodide-free efflux buffer (3 mm KNO_3_, 2 mm Ca(NO_3_)_2_, 11 mm glucose, 20 mm HEPES, and 136 mm NaNO_3_). Cells were scraped in 1 ml of iodide-free efflux buffer and collected by centrifugation (350*g* for 5 min at 25 °C). Iodide-free efflux buffer was removed, and the cell pellet was resuspended in 400 μl of iodide-free efflux buffer. Iodide efflux was measured at room temperature using an iodide-sensitive electrode (Lazar Research Laboratories, Los Angeles, CA). CFTR at the cell surface was stimulated using a mixture of 10 μm forskolin, 1000 μm IBMX, and 100 μm CPT-cAMP (8-(4-chlorophenyl)thio-cyclic AMP) (all compounds were purchased from Sigma). Five minutes after activation, Triton X-100 (Sigma) was used to lyse the cells to ensure that iodide was properly loaded. The maximal iodide efflux rate was quantified over a 1-min interval associated with the largest positive slope during the 4–5-min time period after the addition of the activation mixture. Traces were recorded using the Digidata 1320A data acquisition system with Clampex 8 software (Molecular Devices, Sunnyvale, CA).

Crude membranes were isolated from HEK cells stably expressing F508del-CFTR, WT-CFTR, or F508del-K*X*K-CFTR and treated with 10 μm VRT-325 for 48 h at 37 °C or equal volume of DMSO vehicle control as described previously ([33](#B33)). Crude membranes were resuspended in resuspension buffer (40 mm Tris-HCl, pH 7.4, 5 mm MgCl_2_, and 0.1 mm EGTA) and sonicated. Samples were kept on ice, and trypsin (Promega) was added to the samples in the following range of concentration: 0, 1.56, 3.13, 6.25, 12.5, 25, 50, 100 μg/ml. Samples were sonicated again and incubated at 4 °C for 15 min. Proteolysis was terminated with trypsin soybean inhibitor (0.5 mg/ml) (Sigma). Membranes were solubilized in modified radioimmunoprecipitation assay buffer (50 mm Tris-Cl, 150 mm NaCl, 1 mm EDTA, pH 7.4, 0.2% (v/v) SDS, and 0.1% (v/v) Triton X-100) for 14 min, and the soluble fraction was analyzed by SDS-page on a 4–20% gradient gel. After electrophoresis, proteins were transferred onto nitrocellulose membranes and incubated in Odyssey® blocking buffer (LI-COR Biosciences, Lincoln, NE), and protein bands were detected using monoclonal mouse anti-CFTR-NBD2 M3A7 (1:1000) or anti-CFTR-NBD1 L12B4 (1:1000). Fluorescence was detected using the secondary antibody IRDye 800 (goat anti-mouse IgG) (Rockland Immunochemicals, Gilbertsville, PA; 1:20,000). The blots were imaged using the Odyssey system (LI-COR Biosciences) The integrative intensities of the detected bands were obtained by the Odyssey infrared imaging system LI-COR Biosciences software.

The mean pixel intensity of the full-length or proteolytic fragments resulting from the trypsin digestion of each genotype with or without VRT-325 treatment was measured using the ImageJ 1.42 Q software (National Institutes of Health). Mean pixel intensity of full-length CFTR (including Band B and Band C, 150 and 170 kDa, respectively) and proteolytic fragments (NBD2 fragments) were obtained from the doublet (∼28-kDa range) in immunoblots probed with M3A7. The large NBD1 fragments of 34–36 kDa and the small NBD1 fragments obtained below the 28-kDa range in immunoblots probed with L12B4 were normalized to the intensity of full-length CFTR before treatment (0 μg/ml trypsin). The relative intensities of the NBD1 fragments were determined using the following equation.

| Category |
| --- |
|   |
All graphs were then created with Prism 4.0 software (GraphPad Software, San Diego, CA).

All data are represented as mean ± S.E. Prism 4.0 software (GraphPad Software, San Diego, CA) was employed to carry out statistical analysis. Non-paired Student's *t* tests, one-way analysis of variance (ANOVA), and two-way ANOVA were conducted as appropriate, and *p* values of less than 0.05 were considered significant. The number of replicates for each experiment is indicated in the figure legends.

## RESULTS

### 

#### 

Misprocessing of the major CF causing mutant F508del-CFTR results in retention of the protein in the ER. This is evident in SDS-PAGE analyses, where the mutant migrates as a single band (Band B, ∼150 kDa) representing immature CFTR, sensitive to modification by both *N*-glycanase and endoglycosidase H. In some Western blots, an additional lower molecular mass band (Band A, ∼130 kDa) can be detected, and this may represent the product of an alternative transcript. The biochemical phenotype of F508del-CFTR contrasts with the normal (wild type) CFTR protein, which migrates as two bands, Band B and Band C. Band C represents a higher molecular weight protein that has acquired complex sugars in the Golgi and exhibits endoglycosidase H resistance.

In our previously published studies, we showed that substitution of the arginine residues at positions 553 and 555 with lysine residues in F508del-CFTR (F508del-K*X*K-CFTR) led to partial correction of the processing defect in the major mutant ([23](#B23)). We reproduced these findings in transient transfections of HEK cells in the present studies ([Fig. 1](#F1)*A*). Further, we confirmed that the appearance of a higher molecular mass band (170–180 kDa) observed for F508del-K*X*K-CFTR corresponded to Golgi-mediated processing as this band is sensitive to *N*-glycanase but resistant to endoglycosidase H ([Fig. 1](#F1)*B*). In [Fig. 1](#F1), *C* and *D*, we show that the properly processed protein conferred by expression of F508del-K*X*K-CFTR is functional as a cAMP-activated anion channel stimulated with forskolin and IBMX. This activity was measured continuously using an iodide-sensitive electrode. As expected, cAMP-mediated flux was not detected from untransfected cells (HEK) or cells transfected with F508del-CFTR.

![FIGURE 1.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ae48/3137047/4513760bf974/zbc0351170810001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3137047_zbc0351170810001.jpg)

Disruption of R553XR555 partially corrects processing and functional expression of F508del-CFTR. A, Western blotting of WT, F508del-CFTR, and F508del-KXK-CFTR shows partial correction of processing defect of F508del-CFTR caused by the substitution of R553XR555 to K553XK555 with increase in B and C generation (open triangle). B, representative Western blot of WT, F508del-CFTR, and F508del-KXK-CFTR. H shows effect of endoglycosidase H treatment, and F shows the results of N-glycosidase F treatment. C, representative continuous recording iodide efflux trace of untransfected HEK cells (HEK) or HEK cells stably expressing F508del-CFTR or stably expressing F508del-KXK-CFTR and treated with 5 mm sodium butyrate to increase overall protein expression. Activation mixture consists of 10 μm forskolin, 1000 μm IBMX, and 100 μm CPT-cAMP. Triton X-100 was used to lyse the cells and release the remaining iodide. The inset shows the protein expression by cells in the assay chamber determined by Western blotting. The upper panel of the inset shows the blot probed using an anti-CFTR antibody, and the lower panel shows actin expression detected by Western blotting using an anti-actin antibody. D, maximal iodide efflux rate (nm/s ± S.E.). The efflux rate of F508del-KXK-CFTR (ΔFKXK, n = 3) was significantly greater than F508del-CFTR (ΔF, n = 3). ***, p < 0.001, one-way ANOVA.

Interestingly, we found that the disruption of this R^553^*X*R^555^ sequence exerted an additive effect on the efficacy of two commonly studied small molecule corrector compounds in promoting processing and trafficking of F508del-CFTR ([Figs. 2](#F2) and [3](#F3)). As described previously, the long term (48 h) treatment of cells expressing F508del-CFTR with VRT-325 or Corr4a leads to partial and modest rescue of the processing defect and conversion of Band B to Band C. On the other hand, both of these corrector compounds led to an impressive rescue of F508del-K*X*K-CFTR ([Fig. 2](#F2)*A*). We employed a fluorescence-based imaging system (LI-COR Biosciences) to quantify the relative efficacies in processing as conventional immunoblotting and detection by chemiluminescence is less sensitive. There have been several methods for quantifying of the efficacy of F508del-CFTR processing described in the literature ([5](#B5), [34](#B34)–[36](#B36)). We employed two commonly described methods. In [Fig. 2](#F2)*C*, we show the quantification of Band C relative to the total CFTR protein (Band B plus Band C), and in [Fig. 2](#F2)*D*, we show the quantification of Band C relative to Band B. Regardless of the analytical method used, a similar pattern was observed. Specifically, the combination of both interventions, chemical treatment and introduction of the K*X*K mutation, mediated a response, which is additive of both single interventions. This additive effect of both interventions was also observed in immunofluorescence studies ([Fig. 3](#F3)). The confocal images in [Fig. 3](#F3) clearly show the increased abundance of F508del-K*X*K-CFTR after chemical correction with Corr4a relative to F508del-CFTR after the same treatment. Together, these findings suggest that the biosynthetic rescue mediated by these corrector compounds occurs through a different molecular mechanism than the rescue mediated by ablation of the R^553^*X*R^555^ sequence. Importantly, the percentage of rescue, *i.e.* ((C/C+B)×100%), for chemically rescued F508del-K*X*K-CFTR (∼60%) approaches that observed for untreated WT-CFTR (∼80%).

![FIGURE 2.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ae48/3137047/456f01747d38/zbc0351170810002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3137047_zbc0351170810002.jpg)

Efficacy of small molecule correctors enhanced by disruption of R553XR555 in F508del-CFTR (processing). A, representative Western blots illustrating processing of F508del-CFTR and F508del-KXK-CFTR (transiently transfected) in the presence of Corr4a (10 μm) or VRT-325 (10 μm). The core-glycosylated form is denoted by the closed triangle, and the complex-glycosylated form is denoted by the open triangle. B, representative Western blot obtained using the Odyssey infrared imaging system from LI-COR Biosciences. F508del-CFTR and F508del-KXK-CFTR stable cell lines were treated with 10 μm VRT-325. The core-glycosylated form is denoted by the closed triangle, and the complex-glycosylated form is denoted by the open triangle. Actin was detected using an anti-β-actin antibody to serve as an loading control. C, the bar graph shows the mean of several experiments (n = 5) where VRT-325 treatment in F508del-CFTR and F508del-KXK-CFTR significantly increased the Band C/(B+C) percentage. The solid bracket represents the change in immature to mature protein in VRT-325-treated F508del-CFTR when compared with vehicle. The stippled bracket represents the change in immature to mature protein in VRT-325-treated F508del-KXK-CFTR when compared with vehicle. D, the bar graph shows the mean of several experiments (n = 5) where VRT-325 treatment in F508del-CFTR and F508del-KXK-CFTR significantly increased the Band C/B ratio, normalized using β-actin as a loading control. The solid bracket represents the change in immature to mature protein in VRT-325-treated F508del-CFTR when compared with vehicle. The stippled bracket represents the change in immature to mature protein in VRT-325-treated F508del-KXK-CFTR when compared with vehicle. *, p < 0.05, **, p < 0.01, ***, p < 0.001 Unpaired Student's t test statistical analyses were employed.

![FIGURE 3.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ae48/3137047/87a6ad99f24d/zbc0351170810003.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3137047_zbc0351170810003.jpg)

Efficacy of small molecule correctors enhanced by disruption of RXR in F508del-CFTR. Micrographs of baby hamster kidney cells transiently expressing F508del-CFTR or F508del-KXK-CFTR treated with Corr4a (10 μm) for 24 h are shown. CFTR was detected using M3A7 antibody and secondary antibody conjugated to Alexa Fluor 488 (AF-488). The left panel illustrates nuclear staining with DAPI, the middle panel illustrates AF-488 (CFTR)-associated fluorescence, and the right panel is the merge of the two previous panels. The white bar represents 25 μm.

Protease sensitivity has provided a useful tool that is capable of probing the conformational defects caused by the F508del mutation in CFTR. Limited protease digestion studies by Lukacs and colleagues ([37](#B37)) were the first to report a conformational defect in F508del-CFTR as the enhanced susceptibility to degradation of NBD2 (probed using the NBD2-specific antibody, M3A7) relative to the wild type protein. These observations likely reflect a long range defect caused by F508del in NBD1 on the second half of the molecule. In other words, this reflects a defect in inter-domain interactions. We observed a similar susceptibility of NBD2 to proteolysis in our studies as described later in this section (see [Figs. 4](#F4) and [5](#F5)). However, a difference in the protease susceptibility of NBD1 between the WT and F508del-CFTR was not detected in these original studies.

![FIGURE 4.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ae48/3137047/121cb41d8387/zbc0351170810004.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3137047_zbc0351170810004.jpg)

Limited proteolysis reveals defect in intra-domain compactness of NBD1 in F508del-CFTR. A representative Western blot of WT-CFTR and F508del-CFTR treated with increasing levels of trypsin (0, 1.56, 3.13, 6.25, 12.5, 25, 50, 100 μg/ml) was obtained using the Odyssey infrared imaging system from LI-COR Biosciences. L12B4 anti-CFTR antibody (1:1000) was used for immunodetection conjugated to IRDye800 (1:10 000) (Rockland Immunochemical Inc). The relative intensity of the trypsinization pattern for WT-CFTR and F508del-CFTR at 100 μg/ml is depicted using a densitometry graph (ImageJ). The doublet at 34–36 kDa represents larger NBD1 fragments, and the doublet below 28 kDa represents smaller NBD1 fragments.

![FIGURE 5.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ae48/3137047/871a43c6f016/zbc0351170810005.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3137047_zbc0351170810005.jpg)

The effect of chemical correction and ablation of the R553XR555 sequence on conformational correction of F508del-CFTR probed using limited proteolysis. Limited proteolysis of control or chemically corrected F508del-CFTR, F508del-KXK-CFTR, or WT-CFTR with increasing levels of trypsin (0, 1.56, 3.13, 6.25, 12.5, 25, 50, 100 μg/ml) was performed. Digests were analyzed by SDS-PAGE, and immunoreactive bands were detected using the Odyssey infrared imaging system from LI-COR Biosciences. The colored signal was converted to grayscale for the preparation of this figure. A, samples were immunoblotted with anti-CFTR monoclonal antibody, L12B4 (NBD1-specific). Subsequent analyses focused on the larger NBD1 fragments (34–36 kDa) and the smaller NBD1 fragments (lower than 28 kDa). Both sets of fragments were indicated using brackets on the right-hand side of each image. B, samples were immunoblotted with anti-CFTR monoclonal antibody, M3A7 (NBD2-specific). NBD2 fragments of ∼28 kDa were analyzed subsequently (indicated using brackets on the right-hand side of each image).

We were prompted to re-evaluate the differential protease sensitivity of WT NBD1 and F508del-NBD1 in the context of the full-length protein by immunoblotting using the LI-COR detection system given the sensitivity of this fluorescence detection system as discussed previously in this section. As shown in [Fig. 4](#F4), using this sensitive detection system, we did detect a clear difference between the trypsin susceptibility of WT-NBD1 and F508del-NBD1. We monitored the relative intensities of larger and smaller NBD1 fragments (immunoreactive with the anti-NBD1-specific antibody, L12B4) at high trypsin concentrations. As observed in [Fig. 4](#F4), in the presence of 100 μg/ml trypsin, the dominant larger fragments migrate as a doublet (34–36 kDa), with smaller fragments (another doublet) migrating just below the 28-kDa molecular mass marker. Following quantification, it is clear that there is a decrease in the abundance of the higher NBD1 fragments relative to the smaller fragments in the F508del-CFTR mutant protein when compared with wild type CFTR. These findings support the idea that NBD1 in F508del-CFTR lacks the conformational compactness of NBD1 of the wild type protein as F508del-NBD1 is more readily digested.

On the other hand and consistent with the previous reports by Lukacs and colleagues ([37](#B37)), we also found that lower molecular mass fragments (a doublet of ∼28 kDa) that were immunoreactive to an antibody directed toward NBD2, M3A7 (epitope contains residues 1365–1395 ([38](#B38))) were almost completely digested at high trypsin concentrations in F508del-CFTR relative to WT-CFTR ([Fig. 5](#F5)). Hence, as mentioned previously, these findings support those reported by Du *et al.* ([37](#B37)) and suggest that F508del induces a defect in inter-domain interactions, between NBD1 and the second half of the molecule.

To determine the consequences of VRT-325 on the conformation of F508del-CFTR, we compared the protease susceptibility of F508del-CFTR protein obtained from cells previously treated with VRT-325 (10 μm) for 48 h with F508del-CFTR protein obtained from untreated cells. The intra-domain conformational compactness of F508del-NBD1 was quantified in three independent experiments as the proportion of the larger NBD1 fragments (*i.e.* larger/larger + smaller fragments shown in [Fig. 4](#F4)) at various trypsin concentrations expressed relative to the starting undigested full-length F508del-CFTR protein (*i.e.* Band B and Band C, [Fig. 6](#F6)*A*, *panel i*). The mean and S.E. for the above fragment ratio was shown for F508del-CFTR and for the VRT-325-rescued F508del-CFTR in this line graph ([Fig. 6](#F6)*A*, *panel i*). The protein fragments were measured as NBD1 abundance over increasing trypsin concentrations as described under “Experimental Procedures.” F508del-CFTR and F508del-CFTR treated with VRT-325 were analyzed using a two-way ANOVA to detect an overall difference throughout the entire range of trypsin concentrations between treatments. Analysis shows that there is a significant difference between the relative resistance of the NBD1 fragments after F508del-CFTR rescue with VRT-325 (*p* < 0.01). These findings suggest that chemical rescue induces intra-domain compactness in NBD1.

![FIGURE 6.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ae48/3137047/a639d869085d/zbc0351170810006.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3137047_zbc0351170810006.jpg)

Comparison of the effect of chemical correction (VRT-325) with the effect of disrupting R553XR555 on protease susceptibility of F508del-CFTR. A, a comparison of the proteolysis of F508del-CFTR (treated with vehicle or chemically corrected with VRT-325). Panel i, densitometric analysis of NBD1 abundance (determined as described under “Results”); panel ii, NBD2 abundance; and panel iii, full-length CFTR abundance at the different trypsin concentrations (0, 1.56, 3.13, 6.25, 12.5, 25, 50, 100 μg/ml). The intensity of the bands was expressed as a percentage of total full-length CFTR at 0 μg/ml. Two-way ANOVA analysis was performed on each graph with statistical difference between treatments for NBD1 in panel i, **, p < 0.01 (n = 3), but not for NBD2 (panel ii). The data in panel iii were fitted with a monoexponential decay function with half-digestion of full-length F508del-CFTR with trypsin concentrations of 1.45 ± 0.56 μg/ml (n = 3) for F508del-CFTR (DMSO control) and 1.69 ± 0.17 μg/ml (n = 3) for F508del-CFTR treated with VRT-325, respectively. Panel iv, the bars show that there is no significant effect of VRT-325 pretreatment on the trypsin sensitivity of full-length F508del-CFTR. B, a comparison of the limited proteolysis of F508del-CFTR versus F508del-KXK-CFTR. The data for F508del-CFTR were shown previously in A, and a stippled line is representative of these data for visual comparison with F508del-KXK-CFTR. Panel i, densitometric analysis of NBD1 abundance; panel ii, NBD2 abundance; and panel iii, full-length CFTR abundance at the different levels of trypsin (0, 1.56, 3.13, 6.25, 12.5, 25, 50, 100 μg/ml). The intensity of the bands was expressed as a percentage of the full-length CFTR at 0 μg/ml. Two-way ANOVA analysis was performed on each graph with statistical difference between treatments for NBD1 in panel i, ***, p < 0.001 (n = 3) and for NBD2 in panel ii, *, p < 0.05 (n = 3). The data in panel iii were fitted with a monoexponential decay with half-digestion value of full-length F508del-KXK-CFTR with 3.53 ± 0.22 μg/ml (n = 3) trypsin. Panel iv, the bars show that there is a significant effect of disrupting the RXR on resistance of the full-length protein, **, p < 0.01 (n = 3); unpaired Student's t test statistical analysis was used.

In contrast, we did not detect a significant VRT-325-dependent effect on the abundance of the NBD2 fragments (∼28 kDa) cleaved from F508del-CFTR protein. The biphasic curve shown in [Fig. 6](#F6)*A*, *panel ii* (F508del-CFTR) suggests that the accessibility of the trypsin cut site releasing the NBD2 fragments is normally masked within the folded protein until the site becomes accessible in the presence of ∼12 μg/ml trypsin. At higher trypsin concentrations (>12 μg/ml), the released NBD2 fragments are nearly completely digested in F508del-CFTR. We did not measure any significant difference between the abundance of NBD2 fragments generated from F508del-CFTR with or without chemical correction by VRT-325 ([Fig. 6](#F6)*A*, *panel ii*). Hence, chemical correction does not alter the structure that masks trypsin accessibility to the cut site, nor does it protect the liberated NBD2 from digestion. Similarly, the protease resistance of the full-length F508del-CFTR protein was not altered by chemical rescue ([Fig. 6](#F6)*A*, *panel iii*). Protease resistance of the full-length protein was evaluated by fitting the trypsin-dependent changes in the abundance of the full-length protein (total of Bands B and C) relative to the undigested protein (no added trypsin) with a monoexponential decay function. The mean trypsin concentrations at which half the full-length protein remained are plotted in the bars (*n* = 3) in [Fig. 6](#F6)*A*, *panel iv*. Clearly, there was no significant effect of chemical correction on the protease sensitivity of the full-length F508del-CFTR protein. These findings suggest that VRT-325 binding modifies a structural defect in NBD1, but this does not correspond to conformational correction in the rest of the protein, at least not by the methods employed in this study.

To determine the effect of disrupting the R^553^*X*R^555^ sequence on the conformation of F508del-CFTR, we compared the protease susceptibility of F508del-K*X*K-CFTR with that of F508del-CFTR ([Fig. 6](#F6)*B*). There is enhanced protease resistance of F508del-K*X*K-NBD1 relative to F508del-NBD1 (*p* < 0.001) (as analyzed previously) ([Fig. 6](#F6)*B*, *panel i*). Interestingly, there was also enhanced NBD2 protease resistance in F508del-K*X*K-CFTR relative to F508del-CFTR (with trypsin concentrations > 12 μg/ml, *p* < 0.05) ([Fig. 6](#F6)*B*, *panel ii*). Finally, there was a significant increase in protease resistance of the full-length protein upon disruption of the R^553^*X*R^555^ sequence in F508del-K*X*K-CFTR as well ([Fig. 6](#F6)*B*, *panels iii* and *iv*). These findings suggest that conformational compaction of both NBD1 and NBD2 is associated with conformational correction of the full-length protein. Further, these findings argue that unlike the effect of the corrector compound VRT-325, which just modifies NBD1 conformation ([Fig. 6](#F6)*A*), abrogation of the R^553^*X*R^555^ sequence is effective in enabling inter-domain interactions necessary for improved assembly of this multidomain protein.

As shown previously in [Fig. 2](#F2), the combination of chemical correction and ablation of the R^553^*X*R^555^ sequence in F508del-CFTR led to near WT levels of protein processing. Therefore, we were prompted to assess the effect of the combined interventions on protease susceptibility of F508del-CFTR. [Fig. 7](#F7) shows the results of comparative limited proteolysis studies of chemically corrected (treated with VRT-325) F508del-K*X*K-CFTR and WT-CFTR. The lines were analyzed using a two-way ANOVA to detect an overall significance in NBD1 abundance throughout the entire range of trypsin concentrations between genotypes. The trypsin resistance of NBD1 is greater in WT-CFTR than for chemically corrected F508del-K*X*K-CFTR (*p* < 0.001) ([Fig. 7](#F7)*A*). On the other hand, there was no significant difference between WT-CFTR and F508del-K*X*K-CFTR with respect to the proteolytic digestion of NBD2 ([Fig. 7](#F7)*B*) or the full-length protein ([Fig. 7](#F7), *C* and *D*). These findings suggest that despite the correction of processing and the significant conformational changes in F508del-CFTR induced by the combined intervention, there remains a significant defect in the conformational compactness of NBD1.

![FIGURE 7.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ae48/3137047/7d5b9aee4381/zbc0351170810007.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3137047_zbc0351170810007.jpg)

The combined effect of VRT-325 and mutation on F508del-CFTR fails to completely restore wild type conformation in NBD1. A comparison of the limited proteolysis of chemically corrected (+VRT-325) F508del-KXK-CFTR with WT-CFTR (+VRT-325) is shown. A, densitometric analysis of NBD1. B and C, NBD2 abundance (B) and full-length CFTR abundance at the different levels of trypsin (0, 1.56, 3.13, 6.25, 12.5, 25, 50, 100 μg/ml) (C). The density of the polypeptide was expressed as a percentage over total full-length CFTR at 0 μg/ml. Two-way ANOVA analysis was performed for each comparison with statistical difference between treatment in A, ***, p < 0.001 (n = 3). The points in C were fitted with a monoexponential decay with a half-digestion of 4.47 ± 2.57 μg/ml (n = 3) 3.88 ± 2.48 μg/ml (n = 3) of trypsin for F508del-KXK-CFTR and WT-CFTR (both treated with VRT-325), respectively. D, the bars show that there is no significant difference between the two proteins with respect to the protease susceptibility of the full-length form, p > 0.05.

Consistent with the results of the limited proteolysis studies, we found that the well recognized defect in function by F508del-CFTR as a cyclic AMP-regulated chloride channel ([27](#B27), [39](#B39), [40](#B40)) was not fully corrected by these combined interventions. As evident in [Fig. 8](#F8), *A* and *B*, cells expressing chemically (VRT-325) corrected F508del-K*X*K-CFTR failed to mediate the same rate of cyclic AMP-mediated iodide efflux as cells expressing the WT-CFTR protein. Interestingly, the addition of genistein (50 μm), a well known potentiator compound ([41](#B41)), or VX-770 (10 μm), a potentiator compound that recently passed Phase III clinical trials ([29](#B29), [30](#B30)), together with the cAMP stimulation mixture, was successful in fully restoring F508del-K*X*K-CFTR channel activation to rates similar to wild type CFTR channel activation. These findings confirm that chemically corrected F508del-K*X*K-CFTR retains a defect in channel activity but that this can be repaired to wild type levels with the addition of potentiators, particularly VX-770. Further, as the compound mutant, F508del-K*X*K-CFTR, still retains sensitivity to VX-770, the di-arginine sequence may not be a major determinant for binding of this potentiator.

![FIGURE 8.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ae48/3137047/ae5897661991/zbc0351170810008.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3137047_zbc0351170810008.jpg)

Chemically corrected F508del-KXK-CFTR still requires “potentiation” to exhibit WT-CFTR channel activity. A, representative continuous iodide efflux traces comparing cyclic AMP-mediated channel activity by untransfected (HEK) cells, cells stably expressing F508del-KXK-CFTR (VRT-325-treated), or WT-CFTR. The activation mixture consists of 25 μm forskolin, 10 μg/ml IBMX, and 50 μm genistein (G) where appropriate, and Triton X-100 was used to lyse the cells. The inset shows the protein expression by stably transfected cells after VRT-325 correction. The empty arrowhead indicates Band C, and the solid arrowhead indicates Band B. B, graph displays the mean ± S.E. maximal iodide efflux rate (nm/s). The efflux rate of F508del-KXK-CFTR activated with genistein is not significantly different from WT-CFTR but is significantly different from F508del-KXK-CFTR without genistein. C, representative continuous iodide efflux traces comparing cyclic AMP-mediated channel activity by cells stably expressing F508del-KXK-CFTR (VRT-325-treated). The activation mixture consists of 25 μm forskolin, 10 μg/ml IBMX, and 10 μm VRT-532 or 10 μm VX-770 where indicated, and Triton X-100 was used to lyse the cells. D, graph displays the mean ± S.E. maximal iodide efflux rate (nm/s). The efflux rate of F508del-KXK-CFTR-activated VX-770 is significantly different from samples treated without potentiator and VRT-532. *, p < 0.05, (n = 3), ANOVA plus secondary Bonferroni's tests were employed for statistical analyses.

## DISCUSSION

In this study, we investigated the molecular basis for chemical correction of the misprocessing defect exhibited by the major CF causing mutant, F508del-CFTR. Specifically, we asked whether VRT-325, a corrector previously shown to interact directly with the mutant CFTR protein, could induce a conformational change detectable in limited proteolysis experiments. Interestingly, we showed that the protease sensitivity of F508del-NBD1, but not the protease sensitivity of NBD2 nor the full-length F508del-CFTR, protein was modified by the chemical corrector, VRT-325. The significant effect of VRT-325 on F508del-NBD1 conformation implies an important role for NBD1 in the activity of this compound. However, the failure of VRT-325 to cause a detectable change in inter-domain interactions or assembly of the full-length protein may account for its limited efficacy in correcting the processing defect.

### 

#### 

Initial studies comparing the protease susceptibility of the full-length, WT-CFTR, and F508del-CFTR proteins, published by Du *et al.* ([37](#B37)), revealed the utility of this biochemical method for detecting conformational defects caused by the major mutation, F508del-CFTR. The major observation by these authors related to the profound, genotype-specific difference in the protease susceptibility of fragments immunoreactive to M3A7, an antibody generated against an epitope in the core of NBD2, *i.e.* residues 1365–1395. The remarkable protease susceptibility of NBD2 fragments in F508del-CFTR has been observed by several investigators ([35](#B35), [37](#B37)) and in the present study. Increased protease susceptibility of NBD2 in F508del-CFTR reflects the effect of F508del on post-translational, inter-domain assembly of NBD2 with NBD1 ([37](#B37)).

We now know that the F508del mutation also causes a defect in the conformational stability of NBD1 (*i.e.* a defect in intra-domain stability). Several groups ([19](#B19), [36](#B36), [42](#B42)) have shown, using a range of biophysical assays, that the thermodynamic stability of F508del-NBD1 is reduced relative to WT-NBD1. Furthermore, Braakman and colleagues ([43](#B43)) detected a defect in the protease resistance of the isolated F508del-NBD1. Specifically, these authors found that a fragment generated from F508del-NBD1 by proteinase K digestion, and containing an epitope recognized by the antibody, 7D12 (residues 531–540 in the helical subdomain), was particularly susceptible to proteolysis in F508del-NBD1 relative to WT-CFTR-NBD1. Similarly, reduced protease resistance of F508del-NBD1 was observed in the context of the full-length protein as described by Thibodeau *et al.* ([28](#B28)). Our results agree with those of Thibodeau wherein a fragment corresponding to the core (F_1_-ATPase-like) subdomain of ∼25 kDa and containing the epitope for the anti-NBD1 antibody, L12B4 (*i.e.* residues 385–410) is more readily cleaved from the helical subdomain in F508del-CFTR than the WT-CFTR.

Previous studies have suggested that certain corrector compounds may act to rescue the processing defect in F508del-CFTR by acting on NBD1 directly. For example, Sampson *et al.* ([36](#B36)) recently identified a distinct small molecule corrector compound that reverses the defect in thermodynamic stability of the isolated F508del-NBD1 domain. However, the present study is the first to show that conformational correction of NBD1 can be induced by one such corrector, VRT-325, in the context of the full-length F508del-CFTR.

Our current findings that VRT-325 induces conformational correction of NBD1 are consistent with predictions based on our studies of the effect of VRT-325 on the intrinsic ATPase activity of partially purified F508del-CFTR. We found that the ATPase activity of F508del-CFTR was inhibited by treatment with VRT-325 (10 μm) and that this inhibition was associated with a decrease in the apparent affinity for ATP ([22](#B22)). The ATPase activity of CFTR requires ATP binding to the two ATP binding sites formed at the interface of NBD1 and NBD2. Of the two sites, one serves a modulatory role (that site containing the core subdomain of NBD1), whereas the catalytically active site contains the core subdomain of NBD2. We postulated that the inhibition caused by VRT-325 binding may reflect competition of VRT-325 for ATP binding to the core of NBD1, which in turn modifies the activity of the catalytic site in NBD2. Alternatively, VRT-325 binding to an allosteric site may lead to conformational change, which modifies the functional interaction of ATP binding to NBD1 and/or NBD2. Future biophysical studies of direct binding of VRT-325 with the ATP binding sites in F508del-CFTR are required to distinguish between these hypotheses.

Both of the arginines in the di-arginine framed sequence, Arg^553^ and Arg^555^, have been individually mutated, and the consequences of these mutations have been studied in the context of purified NBD1 protein (mouse and human) or in the context of the full-length protein. In the context of the isolated NBD1 domain, the introduction of R^553^Q or R^555^K enabled enhanced protein stability in solution and permitted the generation of crystal structures for both the WT and F508del-NBD1 ([28](#B28)). Hence, these arginine mutants were described as “solubilizing” mutations. In the context of the full-length F508del-CFTR protein, the substitution of R^553^M, or the substitution of arginine to lysine at position 555, led to partial rescue of the primary processing defect, and hence, these were originally described as “suppressor” mutations ([24](#B24), [25](#B25)). This “suppression” phenotype was tentatively attributed to the disruption of a di-arginine framed ER trafficking motif ([23](#B23), [26](#B26), [44](#B44)). To date, however, the role of this sequence in mediating retrograde trafficking of F508del-CFTR from the Golgi to the endoplasmic reticulum remains unclear. On the other hand, several studies point to the potential role of the R^553^*X*R^555^ sequence in modifying the conformational stability and/or dynamics of F508del-NBD1, and consequently, in altering inter-domain interactions in the context of the full-length F508del-CFTR protein ([16](#B16), [19](#B19), [28](#B28), [42](#B42)).

Recent thermal unfolding studies revealed that the F508del mutation reduces the thermodynamic stability of isolated NBD1, and that defect is partially reversed by solubilizing mutations, including mutations in the above di-arginine sequence ([19](#B19)). In addition, Thibodeau *et al.* ([28](#B28)) showed that in the context of the full-length protein, F508del-NBD1 exhibited enhanced protease sensitivity (in limited proteolysis studies) relative to WT-CFTR-NBD1 and further that the solubilizing mutations (G550E, R553M, and R555K) conferred protease resistance to F508del-NBD1. Furthermore, Thibodeau *et al.* ([28](#B28)) found that these solubilizing mutations exerted a protective effect on the protease sensitivity conferred by F508del on NBD2 in the second half of the full-length mutant protein. Together, these studies support a model wherein second site mutations targeting the di-arginine sequence promote stable folding of F508del-NBD1, and consequently, improve the assembly of the first and second halves of the mutant membrane protein.

Although the second site mutations studied here were more conservative than those in the Thibodeau study ([28](#B28)), we observed a similar, protective effect of mutating this di-arginine to a di-lysine sequence on protease sensitivity of F508del-NBD1 and F508del-NBD2, reflecting rescued intra-domain and inter-domain interactions, respectively. Recapitulation of this observation points to the critical role of these arginines in modulating the folding of intact F508del-CFTR protein. The aberrant intra-domain and inter-domain interactions mediated by this di-arginine sequence in the context of F508del-CFTR have yet to be defined but could include electrostatic interactions with other charged residues in NBD1, charged residues in other parts of the protein, and/or phospholipids.

Unique to the current study, we found that the combined effect of chemical correction together with abrogation of the R^553^*X*R^555^ sequence led to an additive effect on the processing and trafficking defect of F508del-CFTR. In fact, the processing of chemically corrected F508del-K*X*K-CFTR was comparable with that of the WT-CFTR protein. This additive effect supports the idea that both interventions act via different mechanisms. Further, these findings support the hypothesis that correction of the processing defect of F508del-CFTR requires more than partial conformational rescue of the intra-domain stability of NBD1 (as mediated by VRT-325) but also the rescue of wild type inter-domain interactions (as mediated by mutation of R^553^*X*R^555^).

It remains unclear whether the additive effect of abrogating the R*X*R by mutation on the chemical correction of F508del-CFTR by VRT-325 solely reflects conformational change in the mutant protein or whether it *also* reflects the loss of a retrograde trafficking motif. Our current studies do not allow us to distinguish between these possibilities. We previously showed that the delivery of a peptide containing this motif was effective in facilitating forward trafficking of F508del-CFTR and suggested that this result was consistent with the competition of R*X*R-mediated retrograde trafficking from the Golgi to the ER ([23](#B23)). However, in light of our current findings of the structural changes associated with mutation of R*X*R in F508del-CFTR, we favor the idea that abrogation of this sequence promotes proper folding. To date, there has been little progress in identifying the molecular basis for retrograde trafficking, thereby limiting the tools with which to further investigate its role in the mistrafficking of F508del-CFTR. In the case of our peptide rescue studies, we suggest that the exogenous R^553^*X*R^555^-bearing peptides may exert a “transcomplementation” effect, similar to that previously described by Cormet-Boyaka *et al.* ([45](#B45), [46](#B46)), wherein WT-CFTR fragments may interact directly with F508del-CFTR to modify aberrant intramolecular interactions during folding.

Interestingly, despite the restoration of WT processing upon chemical intervention and mutagenesis of R^553^*X*R^555^, the channel activity of VRT-325-treated F508del-K*X*K-CFTR remained defective. The maximum rate of activation of the chemically corrected F508del-K*X*K-CFTR protein was only half that of the WT-CFTR. This functional defect may reflect a modest but significant inhibitory activity of the VRT-325 compound itself. We showed previously that VRT-325 exerts acute inhibitory effect on the ATPase activity of purified and reconstituted F508del-CFTR and inhibits the channel activity of this mutant at the relatively high concentration of 25 μm ([22](#B22)). However, VRT-325 was used at a concentration of 10 μm in the present studies, where only a 10% inhibition of the rate of channel activation might be expected. Instead, we suggest that the reduced rate of activation exhibited by VRT-325-treated F508del-K*X*K-CFTR may reflect either less cell surface expression *or* altered channel gating relative to WT-CFTR. We reason that altered gating rather than reduced surface expression likely accounts for this functional defect as the addition of the channel potentiators, genistein or VX-770, leads to wild type halide flux from cells expressing the chemically corrected mutant protein. The results of these functional studies suggest that conformational defects remain, even after chemical correction with VRT-325 and abrogation of the R^553^*X*R^555^ sequence. In fact, we did detect the persistence of a conformational defect in NBD1 ([Fig. 7](#F7)) in chemically corrected F508del-K*X*K-CFTR by limited proteolysis. A recent study by Riordan and colleagues ([34](#B34)) points to an important role of the “regulatory insertion” or “RI” region in F508del-NBD1 as another molecular determinant of misfolding in F508del-CFTR. Therefore, a combination of interventions aimed at correcting these multiple conformational defects may be required to rescue the functional expression of F508del-CFTR. Importantly, potentiators such as VX-770 have the potential of acting acutely on F508del-CFTR, partially rescued as described in the current study, to mediate channel activity that approaches that of the WT-CFTR protein.

## Acknowledgments

We are grateful to S. Molinski, S. Pasyk, and P. Eckford for providing helpful commentary on the text. We are also grateful to Drs. Jeffrey Stack and Frederick Van Goor (of Vertex Pharmaceuticals) for providing VX-770, reading the manuscript, and providing helpful comments and to Dr. Robert Bridges (Rosalind Franklin Institute) and Cystic Fibrosis Foundation Therapeutics (CFFT) for providing the corrector compounds, VRT-325 and Corr4a, for use in our studies.

## References

1. Kerem B., Rommens J. M., Buchanan J. A., Markiewicz D., Cox T. K., Chakravarti A., Buchwald M., Tsui L. C. (1989) Science 245, 1073–1080  [DOI](https://doi.org/10.1126/science.2570460) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/2570460/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&author=B.%20Kerem&author=J.%20M.%20Rommens&author=J.%20A.%20Buchanan&author=D.%20Markiewicz&author=T.%20K.%20Cox&volume=245&publication_year=1989&pages=1073-1080&pmid=2570460&doi=10.1126/science.2570460&)

2. Riordan J. R., Rommens J. M., Kerem B., Alon N., Rozmahel R., Grzelczak Z., Zielenski J., Lok S., Plavsic N., Chou J. L., et al. (1989) Science 245, 1066–1073  [DOI](https://doi.org/10.1126/science.2475911) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/2475911/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&author=J.%20R.%20Riordan&author=J.%20M.%20Rommens&author=B.%20Kerem&author=N.%20Alon&author=R.%20Rozmahel&volume=245&publication_year=1989&pages=1066-1073&pmid=2475911&doi=10.1126/science.2475911&)

3. Rowe S. M., Miller S., Sorscher E. J. (2005) N. Engl. J. Med. 352, 1992–2001  [DOI](https://doi.org/10.1056/NEJMra043184) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15888700/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N.%20Engl.%20J.%20Med.&author=S.%20M.%20Rowe&author=S.%20Miller&author=E.%20J.%20Sorscher&volume=352&publication_year=2005&pages=1992-2001&pmid=15888700&doi=10.1056/NEJMra043184&)

4. Loo T. W., Bartlett M. C., Clarke D. M. (2005) Mol. Pharm. 2, 407–413  [DOI](https://doi.org/10.1021/mp0500521) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16196493/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol.%20Pharm.&author=T.%20W.%20Loo&author=M.%20C.%20Bartlett&author=D.%20M.%20Clarke&volume=2&publication_year=2005&pages=407-413&pmid=16196493&doi=10.1021/mp0500521&)

5. Van Goor F., Straley K. S., Cao D., González J., Hadida S., Hazlewood A., Joubran J., Knapp T., Makings L. R., Miller M., Neuberger T., Olson E., Panchenko V., Rader J., Singh A., Stack J. H., Tung R., Grootenhuis P. D., Negulescu P. (2006) Am. J. Physiol. Lung Cell Mol. Physiol. 290, L1117–L1130  [DOI](https://doi.org/10.1152/ajplung.00169.2005) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16443646/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Physiol.%20Lung%20Cell%20Mol.%20Physiol.&author=F.%20Van%20Goor&author=K.%20S.%20Straley&author=D.%20Cao&author=J.%20Gonz%C3%A1lez&author=S.%20Hadida&volume=290&publication_year=2006&pages=L1117-L1130&pmid=16443646&doi=10.1152/ajplung.00169.2005&)

6. Wang Y., Loo T. W., Bartlett M. C., Clarke D. M. (2007) J. Biol. Chem. 282, 33247–33251  [DOI](https://doi.org/10.1074/jbc.C700175200) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17911111/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem.&author=Y.%20Wang&author=T.%20W.%20Loo&author=M.%20C.%20Bartlett&author=D.%20M.%20Clarke&volume=282&publication_year=2007&pages=33247-33251&pmid=17911111&doi=10.1074/jbc.C700175200&)

7. Wang Y., Loo T. W., Bartlett M. C., Clarke D. M. (2007) Biochem. J. 406, 257–263  [DOI](https://doi.org/10.1042/BJ20070478) | [PMC free article](/articles/PMC1948964/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17535157/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem.%20J.&author=Y.%20Wang&author=T.%20W.%20Loo&author=M.%20C.%20Bartlett&author=D.%20M.%20Clarke&volume=406&publication_year=2007&pages=257-263&pmid=17535157&doi=10.1042/BJ20070478&)

8. Dawson R. J., Locher K. P. (2006) Nature 443, 180–185  [DOI](https://doi.org/10.1038/nature05155) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16943773/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&author=R.%20J.%20Dawson&author=K.%20P.%20Locher&volume=443&publication_year=2006&pages=180-185&pmid=16943773&doi=10.1038/nature05155&)

9. Dawson R. J., Locher K. P. (2007) FEBS Lett. 581, 935–938  [DOI](https://doi.org/10.1016/j.febslet.2007.01.073) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17303126/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=FEBS%20Lett.&author=R.%20J.%20Dawson&author=K.%20P.%20Locher&volume=581&publication_year=2007&pages=935-938&pmid=17303126&doi=10.1016/j.febslet.2007.01.073&)

10. Ward A., Reyes C. L., Yu J., Roth C. B., Chang G. (2007) Proc. Natl. Acad. Sci. U.S.A. 104, 19005–19010  [DOI](https://doi.org/10.1073/pnas.0709388104) | [PMC free article](/articles/PMC2141898/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18024585/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.S.A.&author=A.%20Ward&author=C.%20L.%20Reyes&author=J.%20Yu&author=C.%20B.%20Roth&author=G.%20Chang&volume=104&publication_year=2007&pages=19005-19010&pmid=18024585&doi=10.1073/pnas.0709388104&)

11. Aller S. G., Yu J., Ward A., Weng Y., Chittaboina S., Zhuo R., Harrell P. M., Trinh Y. T., Zhang Q., Urbatsch I. L., Chang G. (2009) Science 323, 1718–1722  [DOI](https://doi.org/10.1126/science.1168750) | [PMC free article](/articles/PMC2720052/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19325113/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&author=S.%20G.%20Aller&author=J.%20Yu&author=A.%20Ward&author=Y.%20Weng&author=S.%20Chittaboina&volume=323&publication_year=2009&pages=1718-1722&pmid=19325113&doi=10.1126/science.1168750&)

12. Mornon J. P., Lehn P., Callebaut I. (2008) Cell. Mol. Life Sci. 65, 2594–2612  [DOI](https://doi.org/10.1007/s00018-008-8249-1) | [PMC free article](/articles/PMC11131860/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18597042/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell.%20Mol.%20Life%20Sci.&author=J.%20P.%20Mornon&author=P.%20Lehn&author=I.%20Callebaut&volume=65&publication_year=2008&pages=2594-2612&pmid=18597042&doi=10.1007/s00018-008-8249-1&)

13. Serohijos A. W., Hegedus T., Aleksandrov A. A., He L., Cui L., Dokholyan N. V., Riordan J. R. (2008) Proc. Natl. Acad. Sci. U.S.A. 105, 3256–3261  [DOI](https://doi.org/10.1073/pnas.0800254105) | [PMC free article](/articles/PMC2265173/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18305154/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.S.A.&author=A.%20W.%20Serohijos&author=T.%20Hegedus&author=A.%20A.%20Aleksandrov&author=L.%20He&author=L.%20Cui&volume=105&publication_year=2008&pages=3256-3261&pmid=18305154&doi=10.1073/pnas.0800254105&)

14. Mornon J. P., Lehn P., Callebaut I. (2009) Cell. Mol. Life Sci.. 66, 3469–3486  [DOI](https://doi.org/10.1007/s00018-009-0133-0) | [PMC free article](/articles/PMC11115851/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19707853/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell.%20Mol.%20Life%20Sci..&author=J.%20P.%20Mornon&author=P.%20Lehn&author=I.%20Callebaut&volume=66&publication_year=2009&pages=3469-3486&pmid=19707853&doi=10.1007/s00018-009-0133-0&)

15. Csanady L., Vergani P., Gadsby D. C. (2010) Proc. Natl. Acad. Sci. U.S.A. 107, 1241–1246  [DOI](https://doi.org/10.1073/pnas.0911061107) | [PMC free article](/articles/PMC2824283/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19966305/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.S.A.&author=L.%20Csanady&author=P.%20Vergani&author=D.%20C.%20Gadsby&volume=107&publication_year=2010&pages=1241-1246&pmid=19966305&doi=10.1073/pnas.0911061107&)

16. He L., Aleksandrov L. A., Cui L., Jensen T. J., Nesbitt K. L., Riordan J. R. (2010) FASEB J. 24, 3103–3112  [DOI](https://doi.org/10.1096/fj.09-141788) | [PMC free article](/articles/PMC2909275/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20233947/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=FASEB%20J.&author=L.%20He&author=L.%20A.%20Aleksandrov&author=L.%20Cui&author=T.%20J.%20Jensen&author=K.%20L.%20Nesbitt&volume=24&publication_year=2010&pages=3103-3112&pmid=20233947&doi=10.1096/fj.09-141788&)

17. He L., Aleksandrov A. A., Serohijos A. W., Hegedus T., Aleksandrov L. A., Cui L., Dokholyan N. V., Riordan J. R. (2008) J. Biol. Chem. 283, 26383–26390  [DOI](https://doi.org/10.1074/jbc.M803894200) | [PMC free article](/articles/PMC2546535/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18658148/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem.&author=L.%20He&author=A.%20A.%20Aleksandrov&author=A.%20W.%20Serohijos&author=T.%20Hegedus&author=L.%20A.%20Aleksandrov&volume=283&publication_year=2008&pages=26383-26390&pmid=18658148&doi=10.1074/jbc.M803894200&)

18. Lewis H. A., Wang C., Zhao X., Hamuro Y., Conners K., Kearins M. C., Lu F., Sauder J. M., Molnar K. S., Coales S. J., Maloney P. C., Guggino W. B., Wetmore D. R., Weber P. C., Hunt J. F. (2010) J. Mol. Biol. 396, 406–430  [DOI](https://doi.org/10.1016/j.jmb.2009.11.051) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19944699/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Mol.%20Biol.&author=H.%20A.%20Lewis&author=C.%20Wang&author=X.%20Zhao&author=Y.%20Hamuro&author=K.%20Conners&volume=396&publication_year=2010&pages=406-430&pmid=19944699&doi=10.1016/j.jmb.2009.11.051&)

19. Wang C., Protasevich I., Yang Z., Seehausen D., Skalak T., Zhao X., Atwell S., Spencer Emtage J., Wetmore D. R., Brouillette C. G., Hunt J. F. (2010) Protein Sci. 19, 1932–1947  [DOI](https://doi.org/10.1002/pro.480) | [PMC free article](/articles/PMC2998727/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20687163/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Protein%20Sci.&author=C.%20Wang&author=I.%20Protasevich&author=Z.%20Yang&author=D.%20Seehausen&author=T.%20Skalak&volume=19&publication_year=2010&pages=1932-1947&pmid=20687163&doi=10.1002/pro.480&)

20. Loo T. W., Bartlett M. C., Clarke D. M. (2008) Biochem. J. 413, 29–36  [DOI](https://doi.org/10.1042/BJ20071690) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18361776/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem.%20J.&author=T.%20W.%20Loo&author=M.%20C.%20Bartlett&author=D.%20M.%20Clarke&volume=413&publication_year=2008&pages=29-36&pmid=18361776&doi=10.1042/BJ20071690&)

21. Chen E. Y., Bartlett M. C., Loo T. W., Clarke D. M. (2004) J. Biol. Chem. 279, 39620–39627  [DOI](https://doi.org/10.1074/jbc.M407887200) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15272010/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem.&author=E.%20Y.%20Chen&author=M.%20C.%20Bartlett&author=T.%20W.%20Loo&author=D.%20M.%20Clarke&volume=279&publication_year=2004&pages=39620-39627&pmid=15272010&doi=10.1074/jbc.M407887200&)

22. Kim Chiaw P., Wellhauser L., Huan L. J., Ramjeesingh M., Bear C. E. (2010) Mol. Pharmacol. 78, 411–418  [DOI](https://doi.org/10.1124/mol.110.065862) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20501743/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol.%20Pharmacol.&author=P.%20Kim%20Chiaw&author=L.%20Wellhauser&author=L.%20J.%20Huan&author=M.%20Ramjeesingh&author=C.%20E.%20Bear&volume=78&publication_year=2010&pages=411-418&pmid=20501743&doi=10.1124/mol.110.065862&)

23. Kim Chiaw P., Huan L. J., Gagnon S., Ly D., Sweezey N., Rotin D., Deber C. M., Bear C. E. (2009) Chem. Biol. 16, 520–530  [DOI](https://doi.org/10.1016/j.chembiol.2009.04.005) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19477416/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Chem.%20Biol.&author=P.%20Kim%20Chiaw&author=L.%20J.%20Huan&author=S.%20Gagnon&author=D.%20Ly&author=N.%20Sweezey&volume=16&publication_year=2009&pages=520-530&pmid=19477416&doi=10.1016/j.chembiol.2009.04.005&)

24. Teem J. L., Berger H. A., Ostedgaard L. S., Rich D. P., Tsui L. C., Welsh M. J. (1993) Cell 73, 335–346  [DOI](https://doi.org/10.1016/0092-8674(93)90233-g) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7682896/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell&author=J.%20L.%20Teem&author=H.%20A.%20Berger&author=L.%20S.%20Ostedgaard&author=D.%20P.%20Rich&author=L.%20C.%20Tsui&volume=73&publication_year=1993&pages=335-346&pmid=7682896&doi=10.1016/0092-8674(93)90233-g&)

25. Teem J. L., Carson M. R., Welsh M. J. (1996) Receptors Channels 4, 63–72  [PubMed](https://pubmed.ncbi.nlm.nih.gov/8723647/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Receptors%20Channels&author=J.%20L.%20Teem&author=M.%20R.%20Carson&author=M.%20J.%20Welsh&volume=4&publication_year=1996&pages=63-72&pmid=8723647&)

26. Chang X. B., Cui L., Hou Y. X., Jensen T. J., Aleksandrov A. A., Mengos A., Riordan J. R. (1999) Mol. Cell 4, 137–142  [DOI](https://doi.org/10.1016/s1097-2765(00)80196-3) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10445036/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol.%20Cell&author=X.%20B.%20Chang&author=L.%20Cui&author=Y.%20X.%20Hou&author=T.%20J.%20Jensen&author=A.%20A.%20Aleksandrov&volume=4&publication_year=1999&pages=137-142&pmid=10445036&doi=10.1016/s1097-2765(00)80196-3&)

27. Hegedus T., Aleksandrov A., Cui L., Gentzsch M., Chang X. B., Riordan J. R. (2006) Biochim. Biophys. Acta 1758, 565–572  [DOI](https://doi.org/10.1016/j.bbamem.2006.03.006) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16624253/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochim.%20Biophys.%20Acta&author=T.%20Hegedus&author=A.%20Aleksandrov&author=L.%20Cui&author=M.%20Gentzsch&author=X.%20B.%20Chang&volume=1758&publication_year=2006&pages=565-572&pmid=16624253&doi=10.1016/j.bbamem.2006.03.006&)

28. Thibodeau P. H., Richardson J. M., 3rd, Wang W., Millen L., Watson J., Mendoza J. L., Du K., Fischman S., Senderowitz H., Lukacs G. L., Kirk K., Thomas P. J. (2010) J. Biol. Chem. 285, 35825–35835  [DOI](https://doi.org/10.1074/jbc.M110.131623) | [PMC free article](/articles/PMC2975206/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20667826/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem.&author=P.%20H.%20Thibodeau&author=J.%20M.%20Richardson&author=W.%20Wang&author=L.%20Millen&author=J.%20Watson&volume=285&publication_year=2010&pages=35825-35835&pmid=20667826&doi=10.1074/jbc.M110.131623&)

29. Accurso F. J., Rowe S. M., Clancy J. P., Boyle M. P., Dunitz J. M., Durie P. R., Sagel S. D., Hornick D. B., Konstan M. W., Donaldson S. H., Moss R. B., Pilewski J. M., Rubenstein R. C., Uluer A. Z., Aitken M. L., Freedman S. D., Rose L. M., Mayer-Hamblett N., Dong Q., Zha J., Stone A. J., Olson E. R., Ordoñez C. L., Campbell P. W., Ashlock M. A., Ramsey B. W. (2010) N. Engl. J. Med. 363, 1991–2003  [DOI](https://doi.org/10.1056/NEJMoa0909825) | [PMC free article](/articles/PMC3148255/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21083385/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N.%20Engl.%20J.%20Med.&author=F.%20J.%20Accurso&author=S.%20M.%20Rowe&author=J.%20P.%20Clancy&author=M.%20P.%20Boyle&author=J.%20M.%20Dunitz&volume=363&publication_year=2010&pages=1991-2003&pmid=21083385&doi=10.1056/NEJMoa0909825&)

30. Van Goor F., Hadida S., Grootenhuis P. D., Burton B., Cao D., Neuberger T., Turnbull A., Singh A., Joubran J., Hazlewood A., Zhou J., McCartney J., Arumugam V., Decker C., Yang J., Young C., Olson E. R., Wine J. J., Frizzell R. A., Ashlock M., Negulescu P. (2009) Proc. Natl. Acad. Sci. U.S.A. 106, 18825–18830  [DOI](https://doi.org/10.1073/pnas.0904709106) | [PMC free article](/articles/PMC2773991/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19846789/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.S.A.&author=F.%20Van%20Goor&author=S.%20Hadida&author=P.%20D.%20Grootenhuis&author=B.%20Burton&author=D.%20Cao&volume=106&publication_year=2009&pages=18825-18830&pmid=19846789&doi=10.1073/pnas.0904709106&)

31. Gentzsch M., Chang X. B., Cui L., Wu Y., Ozols V. V., Choudhury A., Pagano R. E., Riordan J. R. (2004) Mol. Biol. Cell 15, 2684–2696  [DOI](https://doi.org/10.1091/mbc.E04-03-0176) | [PMC free article](/articles/PMC420093/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15075371/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol.%20Biol.%20Cell&author=M.%20Gentzsch&author=X.%20B.%20Chang&author=L.%20Cui&author=Y.%20Wu&author=V.%20V.%20Ozols&volume=15&publication_year=2004&pages=2684-2696&pmid=15075371&doi=10.1091/mbc.E04-03-0176&)

32. Wellhauser L., Kim Chiaw P., Pasyk S., Li C., Ramjeesingh M., Bear C. E. (2009) Mol. Pharmacol. 75, 1430–1438  [DOI](https://doi.org/10.1124/mol.109.055608) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19339490/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol.%20Pharmacol.&author=L.%20Wellhauser&author=P.%20Kim%20Chiaw&author=S.%20Pasyk&author=C.%20Li&author=M.%20Ramjeesingh&volume=75&publication_year=2009&pages=1430-1438&pmid=19339490&doi=10.1124/mol.109.055608&)

33. Aleksandrov L., Mengos A., Chang X., Aleksandrov A., Riordan J. R. (2001) J. Biol. Chem. 276, 12918–12923  [DOI](https://doi.org/10.1074/jbc.M100515200) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11279083/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem.&author=L.%20Aleksandrov&author=A.%20Mengos&author=X.%20Chang&author=A.%20Aleksandrov&author=J.%20R.%20Riordan&volume=276&publication_year=2001&pages=12918-12923&pmid=11279083&doi=10.1074/jbc.M100515200&)

34. Aleksandrov A. A., Kota P., Aleksandrov L. A., He L., Jensen T., Cui L., Gentzsch M., Dokholyan N. V., Riordan J. R. (2010) J. Mol. Biol. 401, 194–210  [DOI](https://doi.org/10.1016/j.jmb.2010.06.019) | [PMC free article](/articles/PMC4361937/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20561529/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Mol.%20Biol.&author=A.%20A.%20Aleksandrov&author=P.%20Kota&author=L.%20A.%20Aleksandrov&author=L.%20He&author=T.%20Jensen&volume=401&publication_year=2010&pages=194-210&pmid=20561529&doi=10.1016/j.jmb.2010.06.019&)

35. Roy G., Chalfin E. M., Saxena A., Wang X. (2010) Mol. Biol. Cell 21, 597–609  [DOI](https://doi.org/10.1091/mbc.E09-05-0427) | [PMC free article](/articles/PMC2820424/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20032308/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol.%20Biol.%20Cell&author=G.%20Roy&author=E.%20M.%20Chalfin&author=A.%20Saxena&author=X.%20Wang&volume=21&publication_year=2010&pages=597-609&pmid=20032308&doi=10.1091/mbc.E09-05-0427&)

36. Sampson H. M., Robert R., Liao J., Matthes E., Carlile G. W., Hanrahan J. W., Thomas D. Y. (2011) Chem. Biol. 18, 231–242  [DOI](https://doi.org/10.1016/j.chembiol.2010.11.016) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21338920/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Chem.%20Biol.&author=H.%20M.%20Sampson&author=R.%20Robert&author=J.%20Liao&author=E.%20Matthes&author=G.%20W.%20Carlile&volume=18&publication_year=2011&pages=231-242&pmid=21338920&doi=10.1016/j.chembiol.2010.11.016&)

37. Du K., Sharma M., Lukacs G. L. (2005) Nat. Struct. Mol. Biol. 12, 17–25  [DOI](https://doi.org/10.1038/nsmb882) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15619635/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Struct.%20Mol.%20Biol.&author=K.%20Du&author=M.%20Sharma&author=G.%20L.%20Lukacs&volume=12&publication_year=2005&pages=17-25&pmid=15619635&doi=10.1038/nsmb882&)

38. Kartner N., Riordan J. R. (1998) Methods Enzymol. 292, 629–652  [DOI](https://doi.org/10.1016/s0076-6879(98)92049-3) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9711588/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Methods%20Enzymol.&author=N.%20Kartner&author=J.%20R.%20Riordan&volume=292&publication_year=1998&pages=629-652&pmid=9711588&doi=10.1016/s0076-6879(98)92049-3&)

39. Bebok Z., Collawn J. F., Wakefield J., Parker W., Li Y., Varga K., Sorscher E. J., Clancy J. P. (2005) J. Physiol. 569, 601–615  [DOI](https://doi.org/10.1113/jphysiol.2005.096669) | [PMC free article](/articles/PMC1464253/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16210354/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Physiol.&author=Z.%20Bebok&author=J.%20F.%20Collawn&author=J.%20Wakefield&author=W.%20Parker&author=Y.%20Li&volume=569&publication_year=2005&pages=601-615&pmid=16210354&doi=10.1113/jphysiol.2005.096669&)

40. Pedemonte N., Lukacs G. L., Du K., Caci E., Zegarra-Moran O., Galietta L. J., Verkman A. S. (2005) J. Clin. Invest. 115, 2564–2571  [DOI](https://doi.org/10.1172/JCI24898) | [PMC free article](/articles/PMC1190372/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16127463/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Invest.&author=N.%20Pedemonte&author=G.%20L.%20Lukacs&author=K.%20Du&author=E.%20Caci&author=O.%20Zegarra-Moran&volume=115&publication_year=2005&pages=2564-2571&pmid=16127463&doi=10.1172/JCI24898&)

41. Hwang T. C., Sheppard D. N. (1999) Trends Pharmacol. Sci. 20, 448–453  [DOI](https://doi.org/10.1016/s0165-6147(99)01386-3) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10542444/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Trends%20Pharmacol.%20Sci.&author=T.%20C.%20Hwang&author=D.%20N.%20Sheppard&volume=20&publication_year=1999&pages=448-453&pmid=10542444&doi=10.1016/s0165-6147(99)01386-3&)

42. Protasevich I., Yang Z., Wang C., Atwell S., Zhao X., Emtage S., Wetmore D., Hunt J. F., Brouillette C. G. (2010) Protein Sci. 19, 1917–1931  [DOI](https://doi.org/10.1002/pro.479) | [PMC free article](/articles/PMC2998726/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20687133/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Protein%20Sci.&author=I.%20Protasevich&author=Z.%20Yang&author=C.%20Wang&author=S.%20Atwell&author=X.%20Zhao&volume=19&publication_year=2010&pages=1917-1931&pmid=20687133&doi=10.1002/pro.479&)

43. Hoelen H., Kleizen B., Schmidt A., Richardson J., Charitou P., Thomas P. J., Braakman I. (2010) PLoS One 5, e15458  [DOI](https://doi.org/10.1371/journal.pone.0015458) | [PMC free article](/articles/PMC2994901/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21152102/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&author=H.%20Hoelen&author=B.%20Kleizen&author=A.%20Schmidt&author=J.%20Richardson&author=P.%20Charitou&volume=5&publication_year=2010&pages=e15458&pmid=21152102&doi=10.1371/journal.pone.0015458&)

44. Zerangue N., Schwappach B., Jan Y. N., Jan L. Y. (1999) Neuron 22, 537–548  [DOI](https://doi.org/10.1016/s0896-6273(00)80708-4) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10197533/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neuron&author=N.%20Zerangue&author=B.%20Schwappach&author=Y.%20N.%20Jan&author=L.%20Y.%20Jan&volume=22&publication_year=1999&pages=537-548&pmid=10197533&doi=10.1016/s0896-6273(00)80708-4&)

45. Cormet-Boyaka E., Hong J. S., Berdiev B. K., Fortenberry J. A., Rennolds J., Clancy J. P., Benos D. J., Boyaka P. N., Sorscher E. J. (2009) FASEB J. 23, 3743–3751  [DOI](https://doi.org/10.1096/fj.08-127878) | [PMC free article](/articles/PMC2775001/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19620404/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=FASEB%20J.&author=E.%20Cormet-Boyaka&author=J.%20S.%20Hong&author=B.%20K.%20Berdiev&author=J.%20A.%20Fortenberry&author=J.%20Rennolds&volume=23&publication_year=2009&pages=3743-3751&pmid=19620404&doi=10.1096/fj.08-127878&)

46. Cormet-Boyaka E., Jablonsky M., Naren A. P., Jackson P. L., Muccio D. D., Kirk K. L. (2004) Proc. Natl. Acad. Sci. U.S.A. 101, 8221–8226  [DOI](https://doi.org/10.1073/pnas.0400459101) | [PMC free article](/articles/PMC419584/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15141088/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.S.A.&author=E.%20Cormet-Boyaka&author=M.%20Jablonsky&author=A.%20P.%20Naren&author=P.%20L.%20Jackson&author=D.%20D.%20Muccio&volume=101&publication_year=2004&pages=8221-8226&pmid=15141088&doi=10.1073/pnas.0400459101&)
