# The Missing Association: Sequencing-Based Discovery of Novel SNPs in VKORC1 and CYP2C9 That Affect Warfarin Dose in African Americans

## Metadata
**Authors:** MA Perera, E Gamazon, LH Cavallari, SR Patel, S Poindexter, RA Kittles, D Nicolae, NJ Cox
**Journal:** Clinical pharmacology and therapeutics
**Date:** 2011 Jan 26
**DOI:** [10.1038/clpt.2010.322](https://doi.org/10.1038/clpt.2010.322)
**PMID:** 21270790
**PMCID:** PMC3625373
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3625373/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC3625373/pdf/nihms-456107.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC3625373/pdf/nihms-456107.pdf)

## Abstract

it is well recognized that the genetic variants VKORC1-1639, CYP2C9*2, and CYP2C9*3 contribute to warfarin dose response. This has led to warfarin dosing algorithms that include these polymorphisms and explains between 47% and 56% of variability in dose in Caucasians. however, these polymorphisms explain significantly less of the variance in dose among African Americans. In order to identify novel variations that affect warfarin dose in African Americans, we used a targeted resequencing strategy that examined evolutionarily conserved sequences and regions of putative transcriptional binding. Through ethnicity-specific warfarin dose model building in 330 African Americans, we identified two novel genetic associations with higher warfarin dose, namely, VKORC1-8191 (rs61162043, P = 0.0041) and 18786 in CYP2C9 (rs7089580, P = 0.035). These novel finds are independent of the previous associations with these genes. Our regression model, encompassing both genetic and clinical variables, explained 40% of the variability in warfarin dose in African-American subjects, significantly more than any model thus far.

## Results

### Patient demographics

The characteristics for both cohorts of the study are shown in [Table 1](#T1). For the discovery cohort, 140 subjects were recruited from the University of Chicago Anticoagulation clinic between June and December of 2008. Of these, 3 were excluded because of poor quality of the DNA sample and 14 failed to attain stable warfarin dose. Ultimately, 122 patients were included in the discovery cohort. The validation cohort consisted of 207 African-American subjects recruited from the University of Illinois Medical Center in Chicago, as previously described.^19^ Of the demographic information collected at both sites, only sex was significantly different between the two cohorts. All subjects were typed for ancestry informative markers to ensure that they were of African descent, with no significant difference in percentage of African ancestry between sites.

### Table 1.

| Variable | Discoverycohort(n = 122) | Validationcohort(n = 207) | P valuea |
| --- | --- | --- | --- |
| Sex |  |  |  |
| Male | 34% | 24% | 0.0486 |
| Female | 66% | 76% |   |
| Age (mean) | 59 | 56 | 0.186 |
| Weight, kg (mean) | 88.3 | 93.6 | 0.095 |
| Height, cm (mean) | 168.8 | 167.1 | 0.156 |
| Warfarin dose, mg/week (mean±SD) | 44.4±21.0 | 44.0±16.9 | 0.853 |
| Smoking sfafus |  |  |  |
| Yes | 11.9% | 8.6% | 0.45 |
| No | 88.1% | 91.4% |   |
| Indicafion |  |  |  |
| DVT/PE | 58% | 50% | 0.13 |
| Atrial fibrillation | 22% | 16% | 0.11 |
| MVR | 8% | 7% | 0.942 |
| Medicafions |  |  |  |
| CYP2C9 inducers | 2% | 3% | 0.773 |
| Aspirin | 28% | 28% | 0.996 |
| Amiodarone | 2% | 2% | 0.844 |
| Statins | 47% | 47% | 0.279 |

Table 1 Caption: Clinical demographics of the discovery and validation cohorts

### Associations in the discovery cohort

On regression analysis, the clinical covariates associated with warfarin dose in the discovery cohort were age (*P* = 12e-06), weight (*P* = 0.00159), and history of venous thromboembolism (*P* = 0.000876). Similarly, in the validation cohort, the covariates associated were age (*P* = 7.22e-09), weight (*P* = 1.06e-06), and indication of deep-vein thrombosis/pulmonary embolism (*P* = 0.0425).

Of the SNPs genotyped in the discovery cohort, six SNPs showed departure from Hardy–Weinberg equilibrium (*P* ≤ 0.05) and were removed from the analysis. Additive linear association in the discovery cohort showed significant associations between warfarin dose and 24 SNPs in both *CYP2C9* and *VKORC1*, without correction for multiple testing (*P* ≤ 0.05). [Table 2](#T2) shows the genotype/phenotype associations found in this cohort. All the *VKORC1* variants previously shown to have been associated with warfarin dose in Caucasians (*VKORC1-1639, 1173, 1542, 7566*, and *3730*) were replicated in this cohort,^14,20^ in agreement with previous studies in African Americans.^21^ Of the known functional *CYP2C9* variants, only *CYP2C9*3, *8*, and **11* showed significant associations. In addition, several novel SNPs were identified as being significantly associated.

### Table 2.

| Gene | snp name/rs numbera | SNP number | Location | MAF (%) | Discoverycohort P valueb | Replicationcohort P valuec |
| --- | --- | --- | --- | --- | --- | --- |
| CYP2C9 | NA | 12479 | Intron 1 | 3 | 0.0357 | 0.184 |
|   | Rs2860905 | 15858 | Intron 3 | 22 | 0.0434 | 0.00007 |
|   | Rs7089580 | 18786 | Intron 3 | 23 | 0.0272 | 0.00019 |
|   | Rs6583964 | 19497 | Intron 3 | 33 | 0.0354 | 0.524 |
|   | NA | 19879 | Intron 3 | 2 | 0.0444 | 0.0459 |
|   | Rs74150726 | 25641 | Intron 5 | 4 | 0.00297 | 0.238 |
|   | Rs35511771 | 25706 | Intron 5 | 24 | 0.0316 | 0.00008 |
|   | Rs2017319 | 62198 | Exon 9 | 14 | 0.00532 | 0.342 |
|   | NA | 65312 | 3' UTR | 2 | 0.0325 | 0.344 |
|   | CYP2C9*2/rs1799853 | 15610 | Exon 3 | 2 | 0.969 | 0.0041 |
|   | CYP2C9*3/rs1057910 | 54616 | Exon 7 | 1 | 0.0507 | 0.0438 |
|   | CYP2C9*5/rs28371686 | 54621 | Exon 7 | 1 | 0.0930 | 0.242 |
|   | CYP2C9*8/rs7900194 | 15629 | Exon 3 | 6 | 0.0514 | 0.0208 |
|   | CYP2C9*11/rs28371685 | 54544 | Exon 7 | 4 | 0.0302 | 0.0791 |
| VKORC1 | VKORC1-8868/rs62057090 | 5374 | 5′ UTR | 29 | 0.000213 | 0.0051 |
|   | NA | 5663 | 5' UTR | 2 | 0.00336 | 0.517 |
|   | VKORC1-8191/rs61162043 | 6051 | 5′ uTR | 47 | 0.00008 | 0.0375 |
|   | rs7187995 | 7475 | 5′ UTR | 32 | 0.00208 | 0.0807 |
|   | rs58826038 | 7507 | 5′ UTR | 14 | 0.0212 | 0.336 |
|   | VKORC1-3427/rs59502288 | 10995 | 5′ uTR | 42 | 0.00906 | 0.0486 |
|   | VKORC1-1639/rs9923231 | 12603 | 5′ UTR | 11 | 0.00633 | 0.00007 |
|   | VKORC1 1173/rs9934438 | 15414 | Intron 1 | 12 | 0.00451 | 0.00007 |
|   | VKORC1 1542/rs8050894 | 15783 | Intron 2 | 30 | 0.00664 | 0.0002 |
|   | VKORC1 7566/rs2359612 | 16497 | Intron 2 | 21 | 0.00526 | 0.0007 |
|   | VKORC1 9041/rs7294 | 17972 | 3′ UTR | 44 | 0.00703 | 0.0031 |
|   | NA | 18005 | 3′ UTR | >1 | 0.00771 | 0.502 |

Table 2 Caption: significant associations found in each cohort

### Associations in the validation cohort

Because of the limited size of the discovery cohort, all the significant SNPs as well as known functional *CYP2C9* SNPs were genotyped in the validation cohort to validate these genetic associations. Of the original 26 SNPs found in the discovery cohort, 7 SNPs in *CYP2C9* and 8 SNPs in *VKORC1* remained significantly associated with dose (after correction for multiple testing). Again, all the SNPs in *VKORC1* previously known to be associated with warfarin dose remained significant, as well as *CYP2C9*2* and **8*, with *CYP2C9*3* and **11* trending toward significance. We were also able to validate the associations of four novel SNPs in *CYP2C9* (15858, 18786, 19879, and 25706) and three novel SNPs in *VKORC1* (*VKORC1-8868, VKORC1-8191*, and *VKORC1-3427*). [Table 2](#T2) highlights these novel SNP associations.

### Regression model in the combined data set

To determine the contribution of both genetic and nongenetic variables to warfarin dose, we conducted a univariate analysis followed by a multivariate regression analysis. We combined the *CYP2C9*2, *3, *5, *8*, and **11* variants into a single group of *CYP2C9* star variant alleles, on the basis of earlier data.^22^ Several clinical covariates showed association with warfarin dose; however, after age and weight were added to the model (the most significant clinical predictors), only indication of deep-vein thrombosis/pulmonary embolism remained significant. After the addition of the clinical covariates to the model, only *VKORC1-1173*, the *CYP2C9* star variants, 18786, and *VKORC1-8191* were retained in the model. Both of the last two SNPs were associated with increased weekly dose, with *VKORC1-8191* showing a 5.2 mg/week increase in dose for each allele and the *CYP2C9* 18786 SNP showing a 3.7 mg/week increase in dose for each allele. [Table 3](#T3) shows the entry of each SNP into the model with the univariate *r*^2^, the multivariate *R*^2^ after entry into the model, and the *P* value of each SNP upon entry. Through this analysis we were able to explain ~40% of the variability in warfarin dose in African Americans.

### Table 3.

| Model | Entry into model | Variable | Gene | Univariate r2 | Adjusted R2after entry | P |
| --- | --- | --- | --- | --- | --- | --- |
| IWPC model | 1 | Age in decades | NA | — | 14.7 | 5.31 E-13 |
|   | 2 | Height (cm) | NA | 4.5 | 15.4 | 0.00298 |
|   | 3 | Weight (kg) | NA | 9.0 | 17.6 | 0.0034 |
|   | 4 | VKORC1-1639 | VKORC1 | 9.1 | 23.3 | 4.86 E-6 |
|   | 5 | CYP2C9 *2 (rs1799853) | CYP2C9 | 1.3 | 24.9 | 0.047 |
|   | 6 | CYP2C9 *3 (rs1057910) | CYP2C9 | 1.6 | 25.3 | 0.11 |
|   | 7 | Race | NA | ND | ND | ND |
|   | 8 | Enzyme inducer status | NA | 2.0 | 25.6 | 0.18 |
|   | 9 | Amiodarone status | NA | 0.003 | 25.8 | 0.24 |
| African-American model | 1 | Age in years | NA | — | 14.7 | 5.31 E-13 |
|   | 2 | Weight (kg) | NA | 9.0 | 20.5 | 2.84 E-6 |
|   | 3 | DVT/PE | NA | 4.1 | 22.0 | 0.00747 |
|   | 4 | VKORC1 1173 | VKORC1 | 9.9 | 29.4 | 1.40 E-8 |
|   | 5 | CYP2C9 star alleles | CYP2C9 | 5.6 | 37.1 | 1.84e-07 |
|   | 6 | VKORC1-8191 (rs61162043) | VKORC1 | 3.2 | 38.6 | 0.00413 |
|   | 7 | 18786 (rs7089580) | CYP2C9 | 0.4 | 40.0 | 0.0346 |

Table 3 Caption: Comparison of warfarin dosing models in African Americans

The most significant genetic predictor in the regression analysis was *VKORC1-1173*, which explained ~9% of the variability in dose, similar to the results of other studies in African Americans.^22^ The presence of a *CYP2C9* star variant allele was the next best predictor of warfarin dose, accounting for ~6% of the variability. Our analysis also shows that two novel SNPs (18786 in *CYP2C9* and *VKORC1-8191*) were both highly predictive of warfarin dose (*P* = 0.00413 and 0.0346, respectively), even after the addition to the model of *VKORC1-1173* and the *CYP2C9* star variants.

### Comparison with the available models

To determine whether our model was significantly better than existing models at estimating the variability in warfarin dose, we first compared it with the algorithm published by the International Warfarin Pharmacogenetic Consortium (IWPC) to determine the amount of variability that this algorithm explained in our data set. This analysis showed that the IWPC algorithm explained 25% of the variability in warfarin dose ([Table 3](#T3)), in agreement with the results of the IWPC’s own African-American subgroup analysis (25.9% in the IWPC discovery set and 19% in the validation set).^14^ We then investigated whether our findings could be replicated by chance alone. We used the covariates that were significantly associated in our data set as well as the known *VKORC1* SNPs and the *CYP2C9* star variants but permuted the genotype/phenotype relationship between our novel SNPs (*VKORC1-8191* and 18786) and warfarin dose to determine whether we would find an *R*^2^ of equal or greater value than our observed *R*^2^. In 10,000 such simulation data sets, we saw that the observed *R*^2^ value equaling the 40% showed a highly significant improvement in percentage of variability explained (*P* = 0.0002). The expected value in 10,000 simulations is 36% (consistent with the maximum percentage of explained variability obtained thus far in prior studies).^22^ This strongly suggests that the significant increase in percentage is attributable to the inclusion of our novel SNP associations and greatly decreases the possibility that our findings were obtained by chance alone.

## DISCUSSION

Most association studies of both *VKORC1* and *CYP2C9* with warfarin dose have been conducted in predominantly Caucasian populations. In the studies that included African Americans, investigators typically genotyped only the variants shown to affect warfarin dose in Caucasians.^14,21,23-25^ Prior to the current study, SNP discovered in African Americans had yet to be used in warfarin pharmacogenetics or in any currently published warfarin algorithms.^14,15,23^ Our study aimed to fill this gap in knowledge.

Previous SNP discovery efforts in *VKORC1*, such as those by Rieder *et al*., involved resequencing the entire gene in three populations.^20^ However, all associations with warfarin dose were made with haplotypes found in Caucasians. Groups of haplotypes associated with high and low warfarin doses were labeled haplogroups A and B, respectively. Most subsequent investigations have genotyped only one or two SNPs in *VKORC1* that differentiated between these haplogroups. In a more comprehensive study, Limdi *et al*. investigated the common SNPs and haplotypes in African Americans^21^ found within 5 kb of the gene start site. The findings of this study, which replicated many of those in the original study by Rieder *et al*., concluded that the best predictors of warfarin dose in African Americans were *VKORC1-1639* or *VKORC1-1173*. However, this study was still restricted to the previously explored regions of *VKORC1* and did not take into account the possibility that regulation may occur further than 5 kb upstream of the gene.

Our study is distinct from previous investigations in that it used a targeted resequencing strategy. This not only allowed for novel SNP discovery but also enabled us to identify SNPs that would not be found through other comprehensive association methods such as genome-wide association analysis. The SNP *VKORC1-8191*, a novel association in *VKORC1*, is not present on any high-throughput SNP platform. Therefore, only through resequencing would this association with warfarin dose be found. In addition, by resequencing we gain a better understanding of the LD structures of these genes in African Americans, outside of the previously investigated regions. Furthermore, our resequencing approach strongly demonstrates that the inclusion of SNPs located further upstream of reproducibly associated genes (as compared with previous studies) significantly increases the proportion of explained variability in warfarin dose requirements.

In the ethnicity-specific model method, we found that two novel SNPs, along with *VKORC1-1173, CYP2C9* star variants, and clinical factors, explained 40% of the variability in warfarin dose. Factors such as smoking status and the use of amiodarone or statin drugs showed no association in our cohorts, in contrast to the findings of previous studies.^14,15^ This discrepancy may suggest either that different demographic and clinical factors affect warfarin dose in African Americans or that these are surrogate markers for unknown causative clinical variables that are not as well correlated in African Americans as in Caucasians. However, the discrepancy may also reflect inadequate power to detect these effects in our cohort.

Previous reports have suggested that *VKORC1-1639* is the causative SNP in the highly correlated LD block in Caucasians.^26^ Because of the high LD between *VKORC1-1173* and *-1639* (*r*^2^ = 0.9), we, like others,^27^ were unable to determine which SNP is the true causative variant in our population. However, our findings are consistent with those of previous studies,^26^ given that after the inclusion of *VKORC1-1173* no other SNPs in this LD block were added to the model. Additionally, the novel variant *VKORC1-8191*, which was significant in all regression analyses, is not in LD with any of the SNPs in this LD block ([Figure 1](#F1)).

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/41a7/3625373/9b7b3566bcc0/nihms-456107-f0001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3625373_nihms-456107-f0001.jpg)

Linkage disequilibrium (LD) of VKORC1 single-nucleotide polymorphisms (SNPs) in the combined data set. The LD map shows the LD (reported as a percentage) with the SNPs in VKORC1 that showed association in the discovery cohort. The shaded boxes represent the locations of the three exons in this gene, with the arrow showing the orientation of the gene. Two blocks of LD are seen, comprising VKORC1-1173 (15414) and VKORC1-1639 (12603) in one block and VKORC1-7566 and VKORC1-9041 in the second block (calculated using D′ by Gabriel et al.).40 LD extends between these two blocks in the combined African-American data set. The novel SNP associated with warfarin dose, VKORC1-8191, is not in LD with either of these blocks. The LD map color intensity is based on the r2, with the darker shades representing higher LD. The values in the boxes represent the pairwise r2 between markers.

We corrected for multiple testing in the validation cohort by means of the permutation method. The Bonferroni correction is commonly used for this purpose, but it assumes that the tested SNPs are independent. Even after binning SNPs according to LD, residual LD remains, invalidating this assumption in Bonferroni correction. Permutation allows us to preserve the LD between tested SNPs while correcting for multiple testing. While not as conservative as the Bonferroni correction, this method is extensively used in genome-wide association studies.^28^

The IWPC published the largest cohort study (*n* = 4,043) examining the predictability of warfarin dose in a diverse population. This study contained 353 African Americans and used *VKORC1-1639* and *CYP2C9*2* and **3* as well as age, weight, height, race, enzyme inducer status, and amiodarone to predict dose. The dosing model including these variables was able to predict 47% of the dose variability in the entire IWPC cohort. However, when examined by race, only 25.9% of the variability in dose (19% in the IWPC validation cohort) was accounted for in African Americans.^14^ We were able to reproduce these findings in our data set, as seen in [Table 3](#T3). This lower *R*^2^ value is not surprising, given that *VKORC1-1639* and *CYP2C9*2* and **3* are present at much lower frequencies in African Americans as compared with Caucasians^29^ and would therefore exert less influence on dose. Also, the IWPC algorithm did not incorporate *CYP2C9*5, *8*, and **11*, which occur at higher frequency, or even exclusively, in African Americans. The inclusion of African American-specific *CYP2C9* star variants into the regression analysis increased the explained percentage of variability to 37%, which is in agreement with the findings of Cavallari *et al.*^22^ Even more importantly, we were able to show, through our simulation study (*N* = 10,000), that the *R*^2^ = 40% explained by our model is significantly greater (*P* = 0.0002) than would be expected by chance alone, with a sample size equivalent to the IWPC’s African-American cohort. While this increase may appear modest, it should be noted that the *CYP4F2* variant contributed only a 2-4% increase in multivariate *R*^2^ (refs. [30](#R30)-[32](#R32)) in Caucasians and has been added to the online database [WarfarinDosing.org](http://WarfarinDosing.org). In addition, we were able to increase the overall *R*^2^ by 15% over the previously published algorithms, providing a more comprehensive picture of the genetic variability affecting warfarin dose in African Americans. It is worth noting that both novel alleles are associated with increase in dose. Given that African Americans require higher warfarin doses than do Caucasians,^24^ these SNPs may improve the prediction of higher doses in African Americans. However, further modeling and validation will be needed to determine the accuracy of these SNPs in predicting warfarin dose.

Given that we used a haplotype-tagging SNP approach, we cannot be certain whether SNP VKORC1-8191 is the true causative SNP. However, our resequencing efforts allowed some interpretation of our findings. SNP VKORC1-8191 is ~8 kb upstream from the transcriptional start site. This SNP has a minor-allele frequency of prevalence of 47% in African Americans and is associated with higher dose in both cohorts. In the Coriell African-American samples, we found that *VKORC1-8191* is in strong LD (*r*^2^ = 0.9) with a novel SNP 9 kb upstream of VKORC1 (termed SNP 5018 in our study, no rs number assigned). *VKORC1-8191* is also in LD with rs7196161 (*R*^2^ = 0.7), found 4.7 kb upstream. However, this known *VKORC1* variant has been shown to have no association with warfarin dose and shows low LD with *VKORC1-1639* and *VKORC1-1173* in our study (see [Figure 2](#F2)) as well as in previously published work.^21^

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/41a7/3625373/55fa1a231a50/nihms-456107-f0002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3625373_nihms-456107-f0002.jpg)

Linkage disequilibrium (LD) of cytochrome P-450 2C9 (CYP2C9) single-nucleotide polymorphisms (SNPs) in the combined data set. The LD map shows the LD (reported as a percentage) with the SNPs in CYP2C9 that showed association in the discovery cohort. The shaded boxes represent the locations of the nine exons in this gene, with the arrow showing the orientation of the gene. Only one LD block is seen, calculated using D′ by Gabriel et al.).40 The SNP 18786 in intron 3 is in strong LD with SNP 25706 in intron 5. There is little or no LD between 18786 and CYP2C9 star variants, CYP2C9*2, *3, *5, *8, and *11. The LD map color intensity is based on the r2, with the darker shades representing higher LD and the solid black box representing perfect LD. The values in the boxes represent the pairwise r2 between markers.

[SNP18786](https://www.ncbi.nlm.nih.gov/protein/SNP18786) (rs7089580) is a novel association found in the intron 3 of *CYP2C9*. This SNP is in LD (*r*^2^ = 0.98) with SNP 12251 (intron 1), 15413 (intron 2), and 25706 (intron 5) found in our resequencing study. From our prediction of putative transcriptional binding sites, 25706 is within 19bp of a predicted transcriptional binding cluster in intron 5. These clusters may signal a gene regulatory region—as distinct from single sites, which occur at random throughout the gene. [Supplementary Figure S2](#SD1) online shows the location of these predicted clusters within *CYP2C9.* Further molecular studies are needed to determine whether this site is a functional regulator of transcription. SNP 18786 is not in LD with any known functional *CYP2C9* star variants (see [Figure 2](#F2)).

In conclusion, we have significantly improved the amount of variability that can be accounted for in warfarin dose variability among African Americans. Even the most current dosing algorithms include only the known *CYP2C9, VKORC1*, and *CYP4F2* variants, even though numerous studies have shown these variants to be less predictive in populations of African ancestry. By using resequencing data, we go beyond the information obtained through genotyping. We are able to throw light on both haplotype structure and potential causative variants in LD with the associated SNP in African Americans. Having performed this study— the first to investigate novel variations in African Americans using resequencing data—we hope to highlight the need for the use of resequencing along with GWAs to discover new associations, as well as to begin modifying warfarin dose algorithms to better accommodate African-American patients.

## METHODS

### Patient population

The discovery cohort was recruited from the University of Chicago’s Anticoagulation clinic. The validation cohort was recruited from the anticoagulation clinics at the University of Illinois Medical Center at Chicago. The inclusion criteria were age ≥18 years, African-American race by self report, and treatment with a stable dose of warfarin, defined as the same dose for at least three consecutive clinic visits that produced an international normalized ratio within the therapeutic range. All international normalized ratio measurements were made on a ProTime Microcoagulation System **(**ITC**,** Edison, NJ), and all values >4 were remeasured via the hospital laboratory services. In the discovery cohort, one 10-ml blood sample was taken from each subject by venipuncture and placed in purple-top Vacutainer tubes containing ethylenediaminetetraacetic acid and frozen at −20 °C for DNA extraction. Buccal cell samples were collected from participants enrolled from the University of Illinois Medical Center, as previously described.^19^ All protocols were reviewed and approved by the institutional review boards of the University of Chicago and the University of Illinois at Chicago. All patients provided written informed consent for the collection of samples and subsequent analysis.

### Resequencing of VKORC1 and CYP2C9

Genetic regions to be resequenced within *CYP2C9* were prioritized using both evolutionary conservation and putative transcriptional cluster prediction. We compared six species, covering 300 million years of evolution, to identify areas of deep sequence conservation. In addition, informatics tools such as the Cister: Cis-Element Cluster Finder^33,34^ were used to identify clusters of hepatic transcription factor binding sites within the noncoding sequence. [Supplementary Figures S1 and S2](#SD1) online show the regions selected for resequencing. *VKORC1* has been completely resequenced in several populations, and the results are available to the public ([http://pga.gs.washington.edu](http://pga.gs.washington.edu)), albeit limited to 5 kb upstream of the gene. To better understand the role of gene regulation, we examined variations up to 10 kb upstream of the gene.

After determination of the priority regions, 24 African-American Coriell samples were resequenced for both genes via PCR amplification of 1 kb overlapping fragments, used for covering contiguous sections as previously described.^35^ Primers were designed using Primer3^36^ when possible. Because of the high sequence similarity between members of the *CYP2C* gene family, some primers were designed by hand to specifically amplify the *CYP2C9* sequence (details of the sequencing primers are in [Supplementary Table S1](#SD1) online).

For DNA sequencing, PCR products were purified using the QIAquick PCR Purification Kit (Qiagen, Valencia, CA), submitted to the Cancer Research Center DNA Sequencing Core Facility at the University of Chicago, and sequenced using ABI 3730XL DNA sequencers (Applied Biosystems, Foster City, CA). The Phred-Phrap-Consed package (Poly-phred, version 6.11) was used to assemble and analyze all sequences, as previously described.^35^ [Supplementary Tables S2 and S3](#SD1) online show all the SNPs found in the resequencing study.

### selecting tag sNPs and genotyping

To provide a more comprehensive approach, we employed a haplotype-tagging SNP design to capture a majority of the variation within *CYP2C9* and *VKORC1*. From the resequencing study, we determined the patterns of LD and haplotype structure using PHASE 2.0.^37^ We then used the program LDSelect,^38^ which bins SNPs together using *r*^2^ = 0.8, to identify tagging SNPs within each of the genes with a minor-allele cutoff of 5%. Tagging SNPs were then genotyped in the discovery cohort along with the known functional variants *CYP2C9*2* (rs1799853), *CYP2C9*3* (rs1057910), *CYP2C9*5* (rs28371686), *CYP2C9*8* (rs7900194), *CYP2C9*11* (28371685), *VKORC1-9041* (rs7294), *VKORC1-7566* (rs2359612), *VKORC1-1542* (rs8050894), *VKORC1-1173* (rs9934438), and *VKORC1-1639* (rs9923231). The discovery cohort was genotyped through SeattleSNPs Genotyping Support ([http://pga.gs.washington.edu/genotyping_support.html](http://pga.gs.washington.edu/genotyping_support.html)) using a resequencing strategy targeting haplotype-tagging SNPs. The validation cohort was genotyped by PCR, pyrosequencing, and fragment analysis, as previously described,^22^ and by the Sequenom multiplex method or via direct sequencing (primers and experimental conditions can be found in [Supplementary Table S4 and S5](#SD1) online). One hundred individuals of known genotype were added to the genotyping platforms to ensure genotyping accuracy. Any SNP that showed discordance was removed from the analysis.

### statistical analysis

Warfarin dose values were log_2_ transformed to improve normality. All covariates were tested in each data set separately for association with warfarin dose, using a multivariate linear regression to determine covariates that were significantly associated with dose and to take into account any interactions between covariates. The covariates tested were age (years); weight (kg); height (cm); smoking status; use of aspirin, amiodarone, carbamazepine, phenytoin, rifampin, rifampicin, any statin drug, or any azole antifungal drug; comorbidities (hypertension and diabetes); indication for warfarin therapy; and percentage of West African ancestry.

SNPs genotyped in both cohorts were tested for departure from Hardy-Weinberg equilibrium, using the χ^2^-test. Any SNP that showed a departure from Hardy-Weinberg equilibrium (*P* ≤ 0.05) was removed from the analysis. The discovery cohort, from the University of Chicago, was tested for additive linear association using a Student’s *t*-test, using age, weight, and indication of deep-vein thrombosis/pulmonary embolism as covariates. SNPs that showed modest association (*P* ≤ 0.05) were then genotyped in the validation cohort and tested for association with log_2_-transformed warfarin dose. All known functional SNPs in both genes were also genotyped in both cohorts to ensure that all associations with known functional variants were tested. All association *P* values in the validation cohort were corrected for multiple testing using a permutation method conducted in PLINK^39^ (PLINK, version 1.07; [http://pngu.mgh.harvard.edu/purcell/plink/](http://pngu.mgh.harvard.edu/purcell/plink/)) and were considered significant at a *P* value ≤0.05. The permutation method preserves the correlations between SNPs (i.e., LD) and yields a less stringent correction for multiple testing than an approach that assumes independence of tests (such as the Bonferroni correction).

To determine the proportion of the phenotype explained by each variable (both genetic and nongenetic), we conducted a stepwise linear regression analysis on the combined data sets (*n* = 330). All SNPs and covariates were first tested for association by means of a univariate analysis. Any variables showing *P* values <0.05 were tested in the forward selection stepwise regression, with the addition of significant clinical covariates first, followed by genotypes. All analyses were done using an additive model. Variables that were significant predictors of warfarin dose (*P* < 0.05) were retained in the model. All tests for statistical association were conducted in PLINK or R ([http://www.r-project.org](http://www.r-project.org)).

To determine whether our findings represented a significant improvement over previous findings regarding associations, we applied the IWPC model to our data set and calculated the resulting *R*^2^. For the purpose of assessing the level of significance of the *R*^2^ explained by the novel associations detected by our resequencing strategy and included in our African-American regression model, we conducted simulations (*N* = 10,000) that created permuted data sets sampled under the null to generate an empirical distribution. Such an approach breaks the genotype-phenotype relationship, but preserves the correlations between SNPs in both the observed and permuted data sets. The number of times that a permuted data set’s *R*^2^ exceeds the observed *R*^2^ yields an empirical *P* value for the significance of the improvement obtained from the novel associations.

## Supplementary Material

## ACKNOWLEDGMENTS

We thank E.P. Gengler for his editorial advice on this article. We would like to thank SeattleSNPs for providing genotyping support.

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Hirsh J, et al. Oral anticoagulants: mechanism of action, clinical effectiveness, and optimal therapeutic range. Chest. 1998;114:445S–69S. doi: 10.1378/chest.114.5_supplement.445s.  [DOI](https://doi.org/10.1378/chest.114.5_supplement.445s) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9822057/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Chest&title=Oral%20anticoagulants:%20mechanism%20of%20action,%20clinical%20effectiveness,%20and%20optimal%20therapeutic%20range&author=J%20Hirsh&volume=114&publication_year=1998&pages=445S-69S&pmid=9822057&doi=10.1378/chest.114.5_supplement.445s&)

2. O’Reilly RA. The pharmacodynamics of the oral anticoagulant drugs. Prog. Hemost. Thromb. 1974;2:175–213.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/4137288/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Prog.%20Hemost.%20Thromb&title=The%20pharmacodynamics%20of%20the%20oral%20anticoagulant%20drugs&author=RA%20O%E2%80%99Reilly&volume=2&publication_year=1974&pages=175-213&pmid=4137288&)

3. Breckenridge A, Orme M, Wesseling H, Lewis RJ, Gibbons R. Pharmacokinetics and pharmacodynamics of the enantiomers of warfarin in man. Clin. Pharmacol. Ther. 1974;15:424–430. doi: 10.1002/cpt1974154424.  [DOI](https://doi.org/10.1002/cpt1974154424) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/4821443/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther&title=Pharmacokinetics%20and%20pharmacodynamics%20of%20the%20enantiomers%20of%20warfarin%20in%20man&author=A%20Breckenridge&author=M%20Orme&author=H%20Wesseling&author=RJ%20Lewis&author=R%20Gibbons&volume=15&publication_year=1974&pages=424-430&pmid=4821443&doi=10.1002/cpt1974154424&)

4. Rettie AE, et al. Hydroxylation of warfarin by human cDNA-expressed cytochrome P-450: a role for P-4502C9 in the etiology of (S)-warfarin-drug interactions. Chem. Res. Toxicol. 1992;5:54–59. doi: 10.1021/tx00025a009.  [DOI](https://doi.org/10.1021/tx00025a009) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/1581537/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Chem.%20Res.%20Toxicol&title=Hydroxylation%20of%20warfarin%20by%20human%20cDNA-expressed%20cytochrome%20P-450:%20a%20role%20for%20P-4502C9%20in%20the%20etiology%20of%20(S)-warfarin-drug%20interactions&author=AE%20Rettie&volume=5&publication_year=1992&pages=54-59&pmid=1581537&doi=10.1021/tx00025a009&)

5. Kaminsky LS, Zhang ZY. Human P450 metabolism of warfarin. Pharmacol. Ther. 1997;73:67–74. doi: 10.1016/s0163-7258(96)00140-4.  [DOI](https://doi.org/10.1016/s0163-7258(96)00140-4) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9014207/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacol.%20Ther&title=Human%20P450%20metabolism%20of%20warfarin&author=LS%20Kaminsky&author=ZY%20Zhang&volume=73&publication_year=1997&pages=67-74&pmid=9014207&doi=10.1016/s0163-7258(96)00140-4&)

6. Kamali F, et al. Contribution of age, body size, and CYP2C9 genotype to anticoagulant response to warfarin. Clin. Pharmacol. Ther. 2004;75:204–212. doi: 10.1016/j.clpt.2003.10.001.  [DOI](https://doi.org/10.1016/j.clpt.2003.10.001) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15001972/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther&title=Contribution%20of%20age,%20body%20size,%20and%20CYP2C9%20genotype%20to%20anticoagulant%20response%20to%20warfarin&author=F%20Kamali&volume=75&publication_year=2004&pages=204-212&pmid=15001972&doi=10.1016/j.clpt.2003.10.001&)

7. Absher RK, Moore ME, Parker MH. Patient-specific factors predictive of warfarin dosage requirements. Ann. Pharmacother. 2002;36:1512–1517. doi: 10.1345/aph.1C025.  [DOI](https://doi.org/10.1345/aph.1C025) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12243598/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann.%20Pharmacother&title=Patient-specific%20factors%20predictive%20of%20warfarin%20dosage%20requirements&author=RK%20Absher&author=ME%20Moore&author=MH%20Parker&volume=36&publication_year=2002&pages=1512-1517&pmid=12243598&doi=10.1345/aph.1C025&)

8. Franco V, Polanczyk CA, Clausell N, Rohde LE. Role of dietary vitamin K intake in chronic oral anticoagulation: prospective evidence from observational and randomized protocols. Am. J. Med. 2004;116:651–656. doi: 10.1016/j.amjmed.2003.12.036.  [DOI](https://doi.org/10.1016/j.amjmed.2003.12.036) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15121490/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Med&title=Role%20of%20dietary%20vitamin%20K%20intake%20in%20chronic%20oral%20anticoagulation:%20prospective%20evidence%20from%20observational%20and%20randomized%20protocols&author=V%20Franco&author=CA%20Polanczyk&author=N%20Clausell&author=LE%20Rohde&volume=116&publication_year=2004&pages=651-656&pmid=15121490&doi=10.1016/j.amjmed.2003.12.036&)

9. Cushman M, Booth SL, Possidente CJ, Davidson KW, Sadowski JA, Bovill EG. The association of vitamin K status with warfarin sensitivity at the onset of treatment. Br. J. Haematol. 2001;112:572–577. doi: 10.1046/j.1365-2141.2001.02635.x.  [DOI](https://doi.org/10.1046/j.1365-2141.2001.02635.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11260056/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br.%20J.%20Haematol&title=The%20association%20of%20vitamin%20K%20status%20with%20warfarin%20sensitivity%20at%20the%20onset%20of%20treatment&author=M%20Cushman&author=SL%20Booth&author=CJ%20Possidente&author=KW%20Davidson&author=JA%20Sadowski&volume=112&publication_year=2001&pages=572-577&pmid=11260056&doi=10.1046/j.1365-2141.2001.02635.x&)

10. Lubetsky A, Dekel-Stern E, Chetrit A, Lubin F, Halkin H. Vitamin K intake and sensitivity to warfarin in patients consuming regular diets. Thromb. Haemost. 1999;81:396–399.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/10102468/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Thromb.%20Haemost&title=Vitamin%20K%20intake%20and%20sensitivity%20to%20warfarin%20in%20patients%20consuming%20regular%20diets&author=A%20Lubetsky&author=E%20Dekel-Stern&author=A%20Chetrit&author=F%20Lubin&author=H%20Halkin&volume=81&publication_year=1999&pages=396-399&pmid=10102468&)

11. Holbrook AM, et al. Systematic overview of warfarin and its drug and food interactions. Arch. Intern. Med. 2005;165:1095–1106. doi: 10.1001/archinte.165.10.1095.  [DOI](https://doi.org/10.1001/archinte.165.10.1095) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15911722/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arch.%20Intern.%20Med&title=Systematic%20overview%20of%20warfarin%20and%20its%20drug%20and%20food%20interactions&author=AM%20Holbrook&volume=165&publication_year=2005&pages=1095-1106&pmid=15911722&doi=10.1001/archinte.165.10.1095&)

12. Gage BF, Fihn SD, White RH. Management and dosing of warfarin therapy. Am. J. Med. 2000;109:481–488. doi: 10.1016/s0002-9343(00)00545-3.  [DOI](https://doi.org/10.1016/s0002-9343(00)00545-3) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11042238/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Med&title=Management%20and%20dosing%20of%20warfarin%20therapy&author=BF%20Gage&author=SD%20Fihn&author=RH%20White&volume=109&publication_year=2000&pages=481-488&pmid=11042238&doi=10.1016/s0002-9343(00)00545-3&)

13. Phillips KA, Veenstra DL, Oren E, Lee JK, Sadee W. Potential role of pharmacogenomics in reducing adverse drug reactions: a systematic review. JAMA. 2001;286:2270–2279. doi: 10.1001/jama.286.18.2270.  [DOI](https://doi.org/10.1001/jama.286.18.2270) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11710893/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA&title=Potential%20role%20of%20pharmacogenomics%20in%20reducing%20adverse%20drug%20reactions:%20a%20systematic%20review&author=KA%20Phillips&author=DL%20Veenstra&author=E%20Oren&author=JK%20Lee&author=W%20Sadee&volume=286&publication_year=2001&pages=2270-2279&pmid=11710893&doi=10.1001/jama.286.18.2270&)

14. Klein TE, et al. Estimation of the warfarin dose with clinical and pharmacogenetic data. N. Engl. J. Med. 2009;360:753–764. doi: 10.1056/NEJMoa0809329.  [DOI](https://doi.org/10.1056/NEJMoa0809329) | [PMC free article](/articles/PMC2722908/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19228618/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N.%20Engl.%20J.%20Med&title=Estimation%20of%20the%20warfarin%20dose%20with%20clinical%20and%20pharmacogenetic%20data&author=TE%20Klein&volume=360&publication_year=2009&pages=753-764&pmid=19228618&doi=10.1056/NEJMoa0809329&)

15. Gage BF, et al. Use of pharmacogenetic and clinical factors to predict the therapeutic dose of warfarin. Clin. Pharmacol. Ther. 2008;84:326–331. doi: 10.1038/clpt.2008.10.  [DOI](https://doi.org/10.1038/clpt.2008.10) | [PMC free article](/articles/PMC2683977/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18305455/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther&title=Use%20of%20pharmacogenetic%20and%20clinical%20factors%20to%20predict%20the%20therapeutic%20dose%20of%20warfarin&author=BF%20Gage&volume=84&publication_year=2008&pages=326-331&pmid=18305455&doi=10.1038/clpt.2008.10&)

16. Blann A, Hewitt J, Siddiqui F, Bareford D. Racial background is a determinant of average warfarin dose required to maintain the INR between 2.0 and 3.0. Br. J. Haematol. 1999;107:207–209. doi: 10.1046/j.1365-2141.1999.01672.x.  [DOI](https://doi.org/10.1046/j.1365-2141.1999.01672.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10520043/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br.%20J.%20Haematol&title=Racial%20background%20is%20a%20determinant%20of%20average%20warfarin%20dose%20required%20to%20maintain%20the%20INR%20between%202.0%20and%203.0&author=A%20Blann&author=J%20Hewitt&author=F%20Siddiqui&author=D%20Bareford&volume=107&publication_year=1999&pages=207-209&pmid=10520043&doi=10.1046/j.1365-2141.1999.01672.x&)

17. Wardrop SL, Brown MA. Identification of two evolutionarily conserved and functional regulatory elements in intron 2 of the human BRCA1 gene. Genomics. 2005;86:316–328. doi: 10.1016/j.ygeno.2005.05.006.  [DOI](https://doi.org/10.1016/j.ygeno.2005.05.006) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15990270/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genomics&title=Identification%20of%20two%20evolutionarily%20conserved%20and%20functional%20regulatory%20elements%20in%20intron%202%20of%20the%20human%20BRCA1%20gene&author=SL%20Wardrop&author=MA%20Brown&volume=86&publication_year=2005&pages=316-328&pmid=15990270&doi=10.1016/j.ygeno.2005.05.006&)

18. Perera MA, et al. Prediction of CYP3A4 enzyme activity using haplotype tag SNPs in African Americans. Pharmacogenomics J. 2009;9:49–60. doi: 10.1038/tpj.2008.13.  [DOI](https://doi.org/10.1038/tpj.2008.13) | [PMC free article](/articles/PMC2754748/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18825162/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=Prediction%20of%20CYP3A4%20enzyme%20activity%20using%20haplotype%20tag%20SNPs%20in%20African%20Americans&author=MA%20Perera&volume=9&publication_year=2009&pages=49-60&pmid=18825162&doi=10.1038/tpj.2008.13&)

19. Momary KM, Shapiro NL, Viana MA, Nutescu EA, Helgason CM, Cavallari LH. Factors influencing warfarin dose requirements in African-Americans. Pharmacogenomics. 2007;8:1535–1544. doi: 10.2217/14622416.8.11.1535.  [DOI](https://doi.org/10.2217/14622416.8.11.1535) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18034618/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Factors%20influencing%20warfarin%20dose%20requirements%20in%20African-Americans&author=KM%20Momary&author=NL%20Shapiro&author=MA%20Viana&author=EA%20Nutescu&author=CM%20Helgason&volume=8&publication_year=2007&pages=1535-1544&pmid=18034618&doi=10.2217/14622416.8.11.1535&)

20. Rieder MJ, et al. Effect of VKORC1 haplotypes on transcriptional regulation and warfarin dose. N. Engl. J. Med. 2005;352:2285–2293. doi: 10.1056/NEJMoa044503.  [DOI](https://doi.org/10.1056/NEJMoa044503) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15930419/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N.%20Engl.%20J.%20Med&title=Effect%20of%20VKORC1%20haplotypes%20on%20transcriptional%20regulation%20and%20warfarin%20dose&author=MJ%20Rieder&volume=352&publication_year=2005&pages=2285-2293&pmid=15930419&doi=10.1056/NEJMoa044503&)

21. Limdi NA, et al. VKORC1 polymorphisms, haplotypes and haplotype groups on warfarin dose among African-Americans and European-Americans. Pharmacogenomics. 2008;9:1445–1458. doi: 10.2217/14622416.9.10.1445.  [DOI](https://doi.org/10.2217/14622416.9.10.1445) | [PMC free article](/articles/PMC2586955/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18855533/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=VKORC1%20polymorphisms,%20haplotypes%20and%20haplotype%20groups%20on%20warfarin%20dose%20among%20African-Americans%20and%20European-Americans&author=NA%20Limdi&volume=9&publication_year=2008&pages=1445-1458&pmid=18855533&doi=10.2217/14622416.9.10.1445&)

22. Cavallari LH, et al. Genetic and clinical predictors of warfarin dose requirements in African Americans. Clin. Pharmacol. Ther. 2010;87:459–464. doi: 10.1038/clpt.2009.223.  [DOI](https://doi.org/10.1038/clpt.2009.223) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20072124/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther&title=Genetic%20and%20clinical%20predictors%20of%20warfarin%20dose%20requirements%20in%20African%20Americans&author=LH%20Cavallari&volume=87&publication_year=2010&pages=459-464&pmid=20072124&doi=10.1038/clpt.2009.223&)

23. Ferder NS, et al. Ability of VKORC1 and CYP2C9 to predict therapeutic warfarin dose during the initial weeks of therapy. J. Thromb. Haemost. 2010;8:95–100. doi: 10.1111/j.1538-7836.2009.03677.x.  [DOI](https://doi.org/10.1111/j.1538-7836.2009.03677.x) | [PMC free article](/articles/PMC3718044/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19874474/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Thromb.%20Haemost&title=Ability%20of%20VKORC1%20and%20CYP2C9%20to%20predict%20therapeutic%20warfarin%20dose%20during%20the%20initial%20weeks%20of%20therapy&author=NS%20Ferder&volume=8&publication_year=2010&pages=95-100&pmid=19874474&doi=10.1111/j.1538-7836.2009.03677.x&)

24. Limdi NA, et al. Influence of CYP2C9 and VKORC1 on warfarin dose, anticoagulation attainment and maintenance among European-Americans and African-Americans. Pharmacogenomics. 2008;9:511–526. doi: 10.2217/14622416.9.5.511.  [DOI](https://doi.org/10.2217/14622416.9.5.511) | [PMC free article](/articles/PMC2757655/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18466099/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Influence%20of%20CYP2C9%20and%20VKORC1%20on%20warfarin%20dose,%20anticoagulation%20attainment%20and%20maintenance%20among%20European-Americans%20and%20African-Americans&author=NA%20Limdi&volume=9&publication_year=2008&pages=511-526&pmid=18466099&doi=10.2217/14622416.9.5.511&)

25. Schelleman H, et al. Dosing algorithms to predict warfarin maintenance dose in Caucasians and African Americans. Clin. Pharmacol. Ther. 2008;84:332–339. doi: 10.1038/clpt.2008.101.  [DOI](https://doi.org/10.1038/clpt.2008.101) | [PMC free article](/articles/PMC2538606/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18596683/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther&title=Dosing%20algorithms%20to%20predict%20warfarin%20maintenance%20dose%20in%20Caucasians%20and%20African%20Americans&author=H%20Schelleman&volume=84&publication_year=2008&pages=332-339&pmid=18596683&doi=10.1038/clpt.2008.101&)

26. Wang D, Chen H, Momary KM, Cavallari LH, Johnson JA, Sadee W. Regulatory polymorphism in vitamin K epoxide reductase complex subunit 1 (VKORC1) affects gene expression and warfarin dose requirement. Blood. 2008;112:1013–1021. doi: 10.1182/blood-2008-03-144899.  [DOI](https://doi.org/10.1182/blood-2008-03-144899) | [PMC free article](/articles/PMC2515137/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18523153/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Regulatory%20polymorphism%20in%20vitamin%20K%20epoxide%20reductase%20complex%20subunit%201%20(VKORC1)%20affects%20gene%20expression%20and%20warfarin%20dose%20requirement&author=D%20Wang&author=H%20Chen&author=KM%20Momary&author=LH%20Cavallari&author=JA%20Johnson&volume=112&publication_year=2008&pages=1013-1021&pmid=18523153&doi=10.1182/blood-2008-03-144899&)

27. Limdi NA, et al. Warfarin pharmacogenetics: a single VKORC1 polymorphism is predictive of dose across 3 racial groups. Blood. 2010;115:3827–3834. doi: 10.1182/blood-2009-12-255992.  [DOI](https://doi.org/10.1182/blood-2009-12-255992) | [PMC free article](/articles/PMC2865873/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20203262/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Warfarin%20pharmacogenetics:%20a%20single%20VKORC1%20polymorphism%20is%20predictive%20of%20dose%20across%203%20racial%20groups&author=NA%20Limdi&volume=115&publication_year=2010&pages=3827-3834&pmid=20203262&doi=10.1182/blood-2009-12-255992&)

28. Cantor RM, Lange K, Sinsheimer JS. Prioritizing GWAS results: A review of statistical methods and recommendations for their application. Am. J. Hum. Genet. 2010;86:6–22. doi: 10.1016/j.ajhg.2009.11.017.  [DOI](https://doi.org/10.1016/j.ajhg.2009.11.017) | [PMC free article](/articles/PMC2801749/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20074509/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Hum.%20Genet&title=Prioritizing%20GWAS%20results:%20A%20review%20of%20statistical%20methods%20and%20recommendations%20for%20their%20application&author=RM%20Cantor&author=K%20Lange&author=JS%20Sinsheimer&volume=86&publication_year=2010&pages=6-22&pmid=20074509&doi=10.1016/j.ajhg.2009.11.017&)

29. Limdi N, Goldstein J, Blaisdell J, Beasley T, Rivers C, Acton R. Influence of CYP2C9 Genotype on warfarin dose among African American and European Americans. Per. Med. 2007;4:157–169. doi: 10.2217/17410541.4.2.157.  [DOI](https://doi.org/10.2217/17410541.4.2.157) | [PMC free article](/articles/PMC2756088/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19802360/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Per.%20Med&title=Influence%20of%20CYP2C9%20Genotype%20on%20warfarin%20dose%20among%20African%20American%20and%20European%20Americans&author=N%20Limdi&author=J%20Goldstein&author=J%20Blaisdell&author=T%20Beasley&author=C%20Rivers&volume=4&publication_year=2007&pages=157-169&pmid=19802360&doi=10.2217/17410541.4.2.157&)

30. Caldwell MD, et al. CYP4F2 genetic variant alters required warfarin dose. Blood. 2008;111:4106–4112. doi: 10.1182/blood-2007-11-122010.  [DOI](https://doi.org/10.1182/blood-2007-11-122010) | [PMC free article](/articles/PMC2288721/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18250228/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=CYP4F2%20genetic%20variant%20alters%20required%20warfarin%20dose&author=MD%20Caldwell&volume=111&publication_year=2008&pages=4106-4112&pmid=18250228&doi=10.1182/blood-2007-11-122010&)

31. Sagrieya H, et al. Extending and evaluating a warfarin dosing algorithm that includes CYP4F2 and pooled rare variants of CYP2C9. Pharmacogenet. Genomics. 2010;20:407–13. doi: 10.1097/FPC.0b013e328338bac2.  [DOI](https://doi.org/10.1097/FPC.0b013e328338bac2) | [PMC free article](/articles/PMC3098751/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20442691/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet.%20Genomics&title=Extending%20and%20evaluating%20a%20warfarin%20dosing%20algorithm%20that%20includes%20CYP4F2%20and%20pooled%20rare%20variants%20of%20CYP2C9&author=H%20Sagrieya&volume=20&publication_year=2010&pages=407-13&pmid=20442691&doi=10.1097/FPC.0b013e328338bac2&)

32. Takeuchi F, et al. A genome-wide association study confirms VKORC1, CYP2C9, and CYP4F2 as principal genetic determinants of warfarin dose. PLoS Genet. 2009;5:e1000433. doi: 10.1371/journal.pgen.1000433.  [DOI](https://doi.org/10.1371/journal.pgen.1000433) | [PMC free article](/articles/PMC2652833/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19300499/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet&title=A%20genome-wide%20association%20study%20confirms%20VKORC1,%20CYP2C9,%20and%20CYP4F2%20as%20principal%20genetic%20determinants%20of%20warfarin%20dose&author=F%20Takeuchi&volume=5&publication_year=2009&pages=e1000433&pmid=19300499&doi=10.1371/journal.pgen.1000433&)

33. Frith MC, Hansen U, Weng Z. Detection of cis-element clusters in higher eukaryotic DNA. Bioinformatics. 2001;17:878–889. doi: 10.1093/bioinformatics/17.10.878.  [DOI](https://doi.org/10.1093/bioinformatics/17.10.878) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11673232/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=Detection%20of%20cis-element%20clusters%20in%20higher%20eukaryotic%20DNA&author=MC%20Frith&author=U%20Hansen&author=Z%20Weng&volume=17&publication_year=2001&pages=878-889&pmid=11673232&doi=10.1093/bioinformatics/17.10.878&)

34. Wingender E, et al. TRANSFAC: an integrated system for gene expression regulation. Nucleic Acids Res. 2000;28:316–319. doi: 10.1093/nar/28.1.316.  [DOI](https://doi.org/10.1093/nar/28.1.316) | [PMC free article](/articles/PMC102445/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10592259/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=TRANSFAC:%20an%20integrated%20system%20for%20gene%20expression%20regulation&author=E%20Wingender&volume=28&publication_year=2000&pages=316-319&pmid=10592259&doi=10.1093/nar/28.1.316&)

35. Thompson EE, Kuttab-Boulos H, Yang L, Roe BA, Di Rienzo A. Sequence diversity and haplotype structure at the human CYP3A cluster. Pharmacogenomics J. 2006;6:105–114. doi: 10.1038/sj.tpj.6500347.  [DOI](https://doi.org/10.1038/sj.tpj.6500347) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16314882/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=Sequence%20diversity%20and%20haplotype%20structure%20at%20the%20human%20CYP3A%20cluster&author=EE%20Thompson&author=H%20Kuttab-Boulos&author=L%20Yang&author=BA%20Roe&author=A%20Di%20Rienzo&volume=6&publication_year=2006&pages=105-114&pmid=16314882&doi=10.1038/sj.tpj.6500347&)

36. Rozen S, Skaletsky H. Primer3 on the WWW for general users and for biologist programmers. Methods Mol. Biol. 2000;132:365–386. doi: 10.1385/1-59259-192-2:365.  [DOI](https://doi.org/10.1385/1-59259-192-2:365) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10547847/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Methods%20Mol.%20Biol&title=Primer3%20on%20the%20WWW%20for%20general%20users%20and%20for%20biologist%20programmers&author=S%20Rozen&author=H%20Skaletsky&volume=132&publication_year=2000&pages=365-386&pmid=10547847&doi=10.1385/1-59259-192-2:365&)

37. Stephens M, Smith NJ, Donnelly P. A new statistical method for haplotype reconstruction from population data. Am. J. Hum. Genet. 2001;68:978–989. doi: 10.1086/319501.  [DOI](https://doi.org/10.1086/319501) | [PMC free article](/articles/PMC1275651/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11254454/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Hum.%20Genet&title=A%20new%20statistical%20method%20for%20haplotype%20reconstruction%20from%20population%20data&author=M%20Stephens&author=NJ%20Smith&author=P%20Donnelly&volume=68&publication_year=2001&pages=978-989&pmid=11254454&doi=10.1086/319501&)

38. Carlson CS, Eberle MA, Rieder MJ, Yi Q, Kruglyak L, Nickerson DA. Selecting a maximally informative set of single-nucleotide polymorphisms for association analyses using linkage disequilibrium. Am. J. Hum. Genet. 2004;74:106–120. doi: 10.1086/381000.  [DOI](https://doi.org/10.1086/381000) | [PMC free article](/articles/PMC1181897/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14681826/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Hum.%20Genet&title=Selecting%20a%20maximally%20informative%20set%20of%20single-nucleotide%20polymorphisms%20for%20association%20analyses%20using%20linkage%20disequilibrium&author=CS%20Carlson&author=MA%20Eberle&author=MJ%20Rieder&author=Q%20Yi&author=L%20Kruglyak&volume=74&publication_year=2004&pages=106-120&pmid=14681826&doi=10.1086/381000&)

39. Purcell S, et al. PLINK: a tool set for whole-genome association and population-based linkage analyses. Am. J. Hum. Genet. 2007;81:559–575. doi: 10.1086/519795.  [DOI](https://doi.org/10.1086/519795) | [PMC free article](/articles/PMC1950838/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17701901/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Hum.%20Genet&title=PLINK:%20a%20tool%20set%20for%20whole-genome%20association%20and%20population-based%20linkage%20analyses&author=S%20Purcell&volume=81&publication_year=2007&pages=559-575&pmid=17701901&doi=10.1086/519795&)

40. Gabriel SB, et al. The structure of haplotype blocks in the human genome. Science. 2002;296:2225–2229. doi: 10.1126/science.1069424.  [DOI](https://doi.org/10.1126/science.1069424) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12029063/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=The%20structure%20of%20haplotype%20blocks%20in%20the%20human%20genome&author=SB%20Gabriel&volume=296&publication_year=2002&pages=2225-2229&pmid=12029063&doi=10.1126/science.1069424&)
