# IL-4 receptor polymorphisms predict reduction in asthma exacerbations during response to an anti–IL-4 receptor α antagonist

## Metadata
**Authors:** Rebecca E Slager, Babatunde A Otulana, Gregory A Hawkins, Yu Ping Yen, Stephen P Peters, Sally E Wenzel, Deborah A Meyers, Eugene R Bleecker
**Journal:** The Journal of allergy and clinical immunology
**Date:** 2012 Apr 26
**DOI:** [10.1016/j.jaci.2012.03.030](https://doi.org/10.1016/j.jaci.2012.03.030)
**PMID:** 22541248
**PMCID:** PMC3992925
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3992925/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC3992925/pdf/nihms373172.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC3992925/pdf/nihms373172.pdf)

## Abstract

**Background:** 
This is the first large pharmacogenetic investigation of the inflammatory IL-4/IL-13 pathway in patients with moderate-to-severe asthma. We analyzed genomic DNA from participants in a 12-week placebo-controlled efficacy trial of pitrakinra (1, 3, or 10 mg twice daily), a novel IL-4/IL-13 pathway antagonist (Clinicaltrials.gov NCT00801853).

**Objectives:** 
The primary hypothesis for this analysis is that amino acid changes in the 3′ end of the IL-4 receptor α gene (IL4RA) or closely proximal variants would predict reductions in asthma exacerbations for subjects randomized to pitrakinra therapy.

**Methods:** 
Nineteen IL4RA single nucleotide polymorphisms (SNPs) were tested in 407 non-Hispanic white subjects for association with the primary clinical end point of asthma exacerbations and changes in secondary end points for asthma symptom scores.

**Results:** 
The most consistent pharmacogenetic associations were observed for the correlated tagging SNPs rs8832 and rs1029489 in the IL4RA 3′ untranslated and proximal regions, respectively. Subjects homozygous for the rs8832 common G allele randomized to pitrakinra (placebo group nonsignificant) had decreased asthma exacerbations and decreased nocturnal awakenings and activities limited by asthma. There was also a significant pitrakinra dose-response relationship (placebo/1 mg/3 mg/10 mg) for exacerbations in subjects homozygous for the common allele in rs1029489 (P = .005) and rs8832 (P = .009) and the intronic SNPs rs3024585, rs3024622, and rs4787956 (P = .03).

**Conclusion:** 
This study demonstrates a significant pharmacogenetic interaction between anti–IL-4 receptor a therapy and IL4RA gene variation, identifying an asthma subgroup that is more responsive to therapy with this antagonist.

Keywords: Pharmacogenetics, pitrakinra, IL-4 receptor, asthma therapy, IL-4 receptor antagonist

### Background

This is the first large pharmacogenetic investigation of the inflammatory IL-4/IL-13 pathway in patients with moderate-to-severe asthma. We analyzed genomic DNA from participants in a 12-week placebo-controlled efficacy trial of pitrakinra (1, 3, or 10 mg twice daily), a novel IL-4/IL-13 pathway antagonist ([Clinicaltrials.gov](http://Clinicaltrials.gov) [NCT00801853](https://clinicaltrials.gov/ct2/show/NCT00801853)).

### Objectives

The primary hypothesis for this analysis is that amino acid changes in the 3′ end of the IL-4 receptor α gene (*IL4RA*) or closely proximal variants would predict reductions in asthma exacerbations for subjects randomized to pitrakinra therapy.

### Methods

Nineteen *IL4RA* single nucleotide polymorphisms (SNPs) were tested in 407 non-Hispanic white subjects for association with the primary clinical end point of asthma exacerbations and changes in secondary end points for asthma symptom scores.

### Results

The most consistent pharmacogenetic associations were observed for the correlated tagging SNPs rs8832 and rs1029489 in the *IL4RA* 3′ untranslated and proximal regions, respectively. Subjects homozygous for the rs8832 common G allele randomized to pitrakinra (placebo group nonsignificant) had decreased asthma exacerbations and decreased nocturnal awakenings and activities limited by asthma. There was also a significant pitrakinra dose-response relationship (placebo/1 mg/3 mg/10 mg) for exacerbations in subjects homozygous for the common allele in rs1029489 (*P* = .005) and rs8832 (*P* = .009) and the intronic SNPs rs3024585, rs3024622, and rs4787956 (*P* = .03).

### Conclusion

This study demonstrates a significant pharmacogenetic interaction between anti–IL-4 receptor a therapy and *IL4RA* gene variation, identifying an asthma subgroup that is more responsive to therapy with this antagonist.

## METHODS

### Clinical trial population

A pharmacogenetic analysis was performed in a phase 2b clinical trial of pitrakinra, an IL-4Rα antagonist. In this trial 534 participants with moderate-to-severe asthma from the intent-to-treat population were enrolled in a 12-week randomized, placebo-controlled, multicenter (sites in the United States and Europe), double-blind, dose-ranging study (1, 3, or 10 mg twice daily) to test the safety and efficacy of pitrakinra. Subjects were evaluated for the primary end point of incidence of asthma exacerbation and several secondary clinical end points. Patients were stabilized for 4 weeks of run-in on inhaled corticosteroids and long-acting β-agonists (LABAs) and then randomized to pitrakinra or placebo for a 12-week treatment period. LABAs were withdrawn at 4 weeks (study day 28), and inhaled corticosteroid withdrawal began 6 weeks after initiation of blinded treatment (stepped down on study days 42 and 56 and stopped on day 70).^15^ Complete participant inclusion and exclusion criteria can be viewed at [Clinicaltrials.gov](http://Clinicaltrials.gov) [NCT00801853](https://clinicaltrials.gov/ct2/show/NCT00801853). Briefly, inclusion criteria defined a population of nonsmoking male and female subjects 18 years of age and older who had moderate-to-severe asthma and whose symptoms were not well controlled with current asthma therapy. All subjects had experienced an asthma exacerbation requiring oral corticosteroid treatment at least once within the preceding 2 years. Because the majority of participants reported non-Hispanic white ethnicity, genetic analysis was performed in 407 non-Hispanic white subjects from the intent-to-treat population, who provided genomic DNA and consent to participate in a genetic study. Although DNA was available from a total of 465 subjects, we analyzed non-Hispanic white subjects only because of the small numbers of subjects with other ethnic backgrounds. Characteristics of participants in the global clinical trial and non-Hispanic white subjects analyzed at study entry (after the 4-week run-in period on combination ≥250/50 μg fluticasone proprionate/salmeterol therapy) are reported in [Table I](#T1). The safety profile for pitrakinra based on adverse events and laboratory measurements was satisfactory.^15^ All subjects provided informed consent, and this pharmacogenetic analysis was approved by the Wake Forest School of Medicine Institutional Review Board.

### TABLE I.

| Characteristic | Non-Hispanic white genotyped subjects (n = 407) | Global intent-to-treat population (n = 534) |
| --- | --- | --- |
| Sex (% female) | 59 | 59 |
| Ethnicity (% non-Hispanic white/African American/Hispanic/other) | — | 88/8/3/2 |
| Recruitment country (% United States/Europe) | 41/59 | 48/52 |
| Age at enrollment (y) | 45.1 ± 13.8 | 45.4 ± 13.7 |
| Body mass index (kg/m2) | 29.4 ± 6.3 | 30.0 ± 7.5 |
| Baseline lung function |   |   |
| FEV1 (% predicted) | 72.3 ± 11.3 | 72.9 ± 11.5 |
| FVC (% predicted) | 84.4 ± 12.3 | 85.3 ± 12.6 |
| FEV1/FVC ratio | 69.0 ± 10.3 | 68.9 ± 10.2 |
| Blood eosinophils (cells/mm3) | 265 ± 216 | 265 ± 210 |
| Total serum IgE (IU/mL*) | 356.5 ± 733 | 405.3 ± 850.1 |
| FENO (ppb) | 19.3 ± 15.6 | 18.9 ± 14.7 |

Table Caption: Baseline characteristics of the global intent-to-treat pitrakinra clinical trial population and genotyped non-Hispanic white participants analyzed in this study

### Outcome definitions

The primary end point of incidence of asthma exacerbation was defined according to the following criteria: a morning peak flow of at least 30% below baseline for 2 or more consecutive days and 6 or more additional uses of short acting β-agonist reliever medication (1 puff = 1 reliever occasion) in a 24-hour period over baseline levels for 2 or more consecutive days; deterioration of asthma requiring treatment with oral corticosteroids; deterioration of asthma requiring treatment increase of 4 or more times the baseline dose of inhaled corticosteroids; deterioration of asthma requiring hospitalization; or deterioration, in the opinion of the investigator, of the patient’s condition significant enough to be considered an asthma exacerbation. When a subject experienced an asthma exacerbation, he or she had completed the primary end point and was withdrawn from the study.^15^ This definition of exacerbation is consistent with other relatively short-duration clinical trials performed by the National Heart, Lung, and Blood Institute’s Asthma Clinical Research Network ([www.acrn.org](http://www.acrn.org)).^18,19^

Additional secondary end points analyzed in this genetic analysis included time (in days) to an asthma exacerbation and changes from baseline in pulmonary function and asthma symptom scores. Pulmonary function, including measurement of FEV_1_ percent predicted, forced vital capacity percent predicted, and FEV_1_/forced vital capacity ratio, were performed according to standard procedures by using Vitalograph, Inc (Buckinghamshire, United Kingdom), spirometric equipment according to American Thoracic Society guidelines.^20^ Asthma symptom scores ranged from 0 to 4 and were reported by the participants in a daily diary. Questions used in this analysis are available in [Table E1](#SD1) in this article’s Online Repository at [www.jacionline.org](http://www.jacionline.org). For all secondary end points, the weekly average change from baseline was evaluated at study completion for participants who finished the study (week 11 for diary symptom scores and visit 8 for in-clinic measurements) or the week before exacerbation or study withdrawal.

### Genotyping and quality control methods

Twenty-one tagging and nonsynonymous SNPs in *IL4RA* were genotyped in 407 non-Hispanic white subjects. SNPs that tagged blocks of correlated *IL4RA* polymorphisms (*r2* > 0.8) were selected based on available genomic information ([hapmap.org](http://hapmap.org) and [ncbi.nih.gov/snp](http://ncbi.nih.gov/snp)) to provide maximum coverage of common polymorphisms in the primarily non-Hispanic white participants who comprised the global clinical trial population. Genomic DNA was isolated and purified by PPD Central Laboratories (Richland Heights, Ky) and then shipped to Wake Forest for genotyping with the Sequenom MassARRAY iPLEX platform (Sequenom, Inc, San Diego, Calif). Standard genotyping quality control procedures were performed by using the publicly available programs Plink version 1.07^21^ and Haploview version 4.2.^22^ On the basis of the quality control criteria of minor allele frequency (MAF) of greater than 5%, genotyping efficiency of greater than 80%, and consistency with Hardy-Weinberg equilibrium (*P* > .01, Fisher exact test), 19 *IL4RA* SNPs were included in genetic association tests. For all SNPs tested, MAFs were consistent across treatment groups. For example, the MAF for rs8832 in the placebo, 1-mg, 3-mg, and 10-mg treatment groups was 0.43, 0.41, 0.48, and 0.42, respectively. A list of *IL4RA* SNPs analyzed is presented in [Table E2](#SD1) in this article’s Online Repository at [www.jacionline.org](http://www.jacionline.org), and a linkage disequilibrium plot and map of these SNPs is included as [Fig E1](#SD1) in this article’s Online Repository at [www.jacionline.org](http://www.jacionline.org).

### Statistical analysis

Pharmacogenetic analyses were performed by using *IL4RA* SNP genotype by exacerbation status contingency tables stratified by treatment status with an additive test for trend. An additive model is consistent with outcome values for heterozygotes intermediate to subjects homozygous for major and minor alleles. We tested the SNP association with the primary clinical outcome: exacerbations in subjects randomized to placebo and all doses of pitrakinra both pooled and each dose separately. To evaluate the pitrakinra dose response (placebo/1 mg/3 mg/10 mg), we developed contingency tables with exacerbation status by treatment assignment stratified by SNP genotypes (heterozygotes and subjects homozygous for minor allele combined). On the basis of these results, we also performed a subgroup analysis of subjects randomized to the 3- and 10-mg doses of pitrakinra.

Because participants with high blood eosinophil counts responded more effectively to pitrakinra,^15,16^ we tested SNP associations with exacerbations stratified by eosinophil counts (using designations of >350 cells/mm^3^ as high and ≤350 cells/mm^3^ as low). For continuous variables, general linear models were developed to test change from baseline by treatment assignment, adjusting for region and baseline values and stratified by genotype, as described above. We also used Kaplan-Meier plots with a log-rank test (each dose compared with placebo) to evaluate differences in time to exacerbation by treatment assignment stratified by SNP genotype. Statistical analyses were performed with Plink version 1.07,^21^ Haploview version 4.2,^22^ and SPSS/PASW version 18 (SPSS, Inc, Chicago, Ill). A 2-sided *P* value of less than .05 was considered statistically significant, and we adjusted for multiple comparisons by using Bonferroni correction based on our primary hypothesis.

## RESULTS

### Demographics

Demographic and baseline clinical characteristics, including lung function, total serum IgE levels, and fraction of exhaled nitric oxide (FENO) levels, of participants were similar for subjects in the overall intent-to-treat trial population (n = 534) and the genotyped non-Hispanic white subjects (n = 407) analyzed in this study ([Table I](#T1)).

### Asthma exacerbations

In this pitrakinra 2b clinical trial the frequency of asthma exacerbations was 19.9% overall, and there were no significant differences in exacerbations in participants assigned to active pitrakinra doses compared with those assigned to placebo.^15^ In 407 genotyped non-Hispanic white subjects, exacerbations occurred in 80 participants (overall, 19.7%), and there were also no statistically significant differences in exacerbations by treatment assignment: 22 (20.6%) exacerbations in the placebo group, 20 (19.6%) exacerbations in the 1-mg group, 24 (23.3%) exacerbations in the 3-mg treatment group, and 14 (14.7%) exacerbations in the 10-mg treatment group. To test for significant differences in exacerbations by genotype, we analyzed *IL4RA* SNP genotype associations with asthma exacerbations stratified by treatment assignment (placebo and all doses of pitrakinra pooled). The frequency of exacerbations in subjects with the common GG genotype in the tagging SNP rs8832 who were receiving active pitrakinra (11%) was significantly reduced compared with that of subjects with the AG (22%) or AA genotypes (25%, *P* = .03), and this SNP was not associated with exacerbations in the placebo group.

Because there was evidence that *IL4RA* tagging SNPs might interact with pitrakinra treatment and identify a therapeutically responsive subpopulation, we then investigated a pitrakinra dose-response relationship (placebo/1 mg/3 mg/10 mg). The correlated SNPs (*r2* = 0.75) rs8832 ([Fig 1, A](#F1)) and rs1029489 ([Fig 1, B](#F1)) showed a similar dose-response relationship for subjects with the common GG genotype for either SNP (*P* =.005–.009). When individual doses of pitrakinra were compared with placebo within the rs8832GG genotype, there was a significant decrease in exacerbations at the 10-mg dose (*P* = .03), corresponding to a 22% overall reduction and an 88% relative reduction. Significant dose-response trends for subjects homozygous for the major allele were also observed for rs3024585, rs3024622, and rs4787956 (*P* = .03, data not shown). There were also no significant differences in baseline demographic or clinical characteristics between subjects with the rs8832GG genotype and subjects with AG/AA genotypes, which would confound these therapeutic associations with asthma exacerbations (see [Table E3](#SD1) in this article’s Online Repository at [www.jacionline.org](http://www.jacionline.org)).

### FIG 1.

![FIG 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0d46/3992925/7a872ad8b307/nihms373172f1.jpg)

Exacerbations by treatment assignment stratified by IL4RA SNP genotype. There was a significant dose-response trend for treatment response to exacerbations in subjects homozygous for the rs8832GG genotype (A) and the GG genotype in the correlated SNP rs1029489 (B).

In a subgroup analysis of participants randomized to 3- and 10-mg doses of pitrakinra only, subjects homozygous for the common alleles in 6 *IL4RA* SNPs were significantly less likely to have an asthma exacerbation than subjects with the minor allele, including rs8832 and rs3024530, which had the lowest *P* values ([Fig 2](#F2), *P* = .007). Inversely, participants with the rs1110470 minor allele were less likely to experience exacerbations compared with subjects with the common allele ([Fig 2](#F2)). These SNPs were not associated with exacerbations in the placebo group. Additionally, a lower *P* value for association with asthma exacerbations in the combined 3- and 10-mg treatment groups was observed for 4 haplotypes that contain the rs8832 G allele (3, 4, 6, and 7 listed in [Table II](#T2)). As indicated by an odds ratio of less than 1, subjects with these haplotypes were less likely to experience exacerbations.

### FIG 2.

![FIG 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0d46/3992925/f40cb32c363a/nihms373172f2.jpg)

IL4RA variants associated with exacerbations. Seven IL4RA SNPs were significantly associated with asthma exacerbations by genotype in participants randomized to 3- and 10-mg doses of pitrakinra. The dashed line indicates overall exacerbation frequency (19.7%) in genotyped non-Hispanic white subjects.

### TABLE II.

| Haplotype | rs3024585 | rs3024622 | rs4787423 | rs1805011 (E400A) | rs1801275 (Q576R) | rs8832 | rs1029489 | rs12102586 | rs4787956 | Frequency | Odds ratio | P value* |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| 1 | G | C | T | A | A |   |   |   |   | 0.4 | 0.35 | .001 |
| 2 |   | G | T | A | A | A |   |   |   | 0.25 | 1.34 | .35 |
| 3 |   | C | T | A | A | G |   |   |   | 0.38 | 0.30 | .0003 |
| 4 |   |   | T | A | A | G | G |   |   | 0.4 | 0.33 | .0006 |
| 5 |   |   |   | A | A | A | A | C |   | 0.34 | 1.65 | .07 |
| 6 |   |   |   | A | A | G | G | C |   | 0.43 | 0.34 | .0006 |
| 7 |   |   |   |   | A | G | G | C | A | 0.42 | 0.38 | .001 |

Table Caption: IL4RA 5-SNP sliding window haplotypes and association with asthma exacerbations in non-Hispanic white subjects randomized to 3 and 10 mg of pitrakinra treatment

### Secondary outcomes

Although there was little overlap between subgroups of participants with high blood eosinophil counts and the *IL4RA*/rs8832GG genotype,^16^ we explored the relationship between the rs8832 genotype and exacerbations, as stratified by eosinophil counts. Subjects with the GG genotype receiving active pitrakinra (all doses combined) in the low-eosinophil-count group only were less likely to have an exacerbation (11%) than subjects with the AG (16%) or AA (24%) genotype (*P* = .04). There was no association with this SNP and asthma exacerbations in subjects with high blood eosinophil counts, although the number of subjects were smaller, and therefore our power to detect associations was lower in this group.

For subjects in the intent-to-treat population with high blood eosinophil counts, there was also a favorable pitrakinra treatment response for secondary end points related to increased time to exacerbations and asthma symptom scores.^15^ Therefore we explored these clinical end points in our pharmacogenetic analysis in all genotyped non-Hispanic white subjects. Kaplan-Meier plots were developed to test for significant differences in time to asthma exacerbation by treatment assignment stratified by *IL4RA* SNP genotypes. As shown in [Fig 3](#F3), for subjects with the rs8832GG genotype, there were significant differences in time to asthma exacerbation for the subjects randomized to 10 mg of pitrakinra compared with those receiving placebo (*P* = .02, log-rank test). Similar results were noted for subjects homozygous for the common alleles in rs1029489 (*P* = .005, placebo vs 10-mg log-rank test) and rs4787956 (*P* = .02).

### FIG 3.

![FIG 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0d46/3992925/de646256865c/nihms373172f3.jpg)

Kaplan-Meier plot for time to exacerbation by treatment assignment for subjects with the rs8832GG genotype. On study day 28, LABAs were withdrawn, and inhaled corticosteroids were stepped down on study days 42 and 56 and stopped completely on day 70. There was a significant difference in time to exacerbation by survival time analysis for subjects with the rs8832GG genotype randomized to 10 mg of pitrakinra compared with placebo. P values are from log-rank tests.

We also investigated *IL4RA* variation and changes in asthma symptoms. There was a dose-dependent reduction in nocturnal awakenings in subjects with the rs8832GG ([Fig 4, A](#F4)) and rs3024622CC ([Fig 4, B](#F4)) genotypes. Furthermore, activities limited by asthma remained similar to baseline or improved in subjects randomized to pitrakinra with the rs8832GG (*P* = .009), rs1029489GG (*P* =.01), and rs4787956AA (*P* =.01) genotypes.

### FIG 4.

![FIG 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0d46/3992925/c3f664ebd2f1/nihms373172f4.jpg)

IL4RA polymorphisms predict change in nighttime awakenings caused by asthma. There was a significant dose-response trend for change in nighttime awakenings by treatment assignment (placebo/1 mg/3 mg/10 mg) observed for subjects with the common rs8832GG genotype (A) and participants with the common rs3024622CC genotype (B). There was no significant trend for participants with the minor allele in these SNPs. CFB, Change from baseline.

Our primary hypothesis was that *IL4RA* amino acid changes or closely proximal SNPs at the 3′ end of this gene would interact with responses to pitrakinra therapy (including rs1805011, rs1801275, rs8832, rs1029489, rs12102586, and rs4797956; see [Fig E1](#SD1)). For our primary dose-response relationship analysis, rs8832 (adjusted *P* = .05) and rs1029489 (adjusted *P* = .03) were associated with a therapeutic response to this intervention after this correction.

## DISCUSSION

This is the first large pharmacogenetic analysis of the T_H_2-related IL-4/IL-13 inflammatory receptor in subjects with moderate-to-severe asthma. In this relatively short (12-week) trial, LABAs were tapered 4 weeks after initiation of blinded IL-4Rα antagonist or placebo treatment, and inhaled corticosteroids were tapered and withdrawn subsequently. Although there was no overall significant difference between active IL-4Rα therapy and placebo,^15,16^ therapeutic efficacy was observed by *IL4RA* genotype in this pharmacogenetic analysis. Consistent with our hypothesis that IL4RA variants were associated with a specific reduction in asthma exacerbations in subjects randomized to anti– IL-4Rα receptor antagonist, the most consistent pharmacogenetic association was observed for the tagging SNP rs8832. Non-Hispanic white subjects homozygous for the rs8832GG common allele randomized to active pitrakinra experienced fewer exacerbations (*P* = .03), nocturnal awakenings (*P* = .04), and asthma limitations in activities (*P* = .009) compared with subjects with the rs8832 A minor allele. More importantly, there was a significant dose-response relationship to this IL-4Rα inhibitor for asthma exacerbations in subjects with the rs8832GG genotype, which was also observed for the correlated SNP rs1029489 (which remained significant after adjustment for multiple testing). A significant clinical response was observed for 4 haplotypes that contained the rs8832 common G allele associated with decreased exacerbations (*P* = .0003–.0006) in subjects randomized to 3 and 10 mg of pitrakinra. These results provide evidence that IL-4Rα variation modulates therapeutic responses in asthmatic subjects and identifies a subgroup of subjects with moderate-to-severe persistent asthma who respond with reduced asthma exacerbations and symptoms. On the basis of extrapolation from the survival curve in [Fig 3](#F3), we speculate that there might be a long-term positive effect for subjects with the rs8832GG genotype receiving active therapy compared with those receiving placebo.

This analysis is based on an earlier pharmacogenetic analysis that we performed on IL-4Rα inhibition with pitrakinra to reduce late-phase allergen responses.^12^ In this prior smaller allergen challenge study of 28 subjects, we identified 2 nonsynonymous variants in *IL4RA* (E400A and Q576R) and several other tagging SNPs that were associated with a reduction in delayed allergen responses. In addition, we supplemented these preliminary pharmacogenetic responses with an analysis of late-phase allergen responses in nonhuman primates, showing that IL-4Rα inhibition reduces bronchial hyperresponsiveness and eosinophilia after antigen airway challenge.^13^ In this current analysis E400A and Q576R polymorphisms were not associated with asthma exacerbations and treatment responses, possibly because of small numbers of subjects in the previous analysis or differences in study design and primary outcome. However, in this pharmacogenetic analysis significant results were noted for the tagging SNPs rs2239347 and rs3024622, which were statistically associated with reduced late-phase antigen response and therapeutic pitra-kinra responses previously^13^ and were also predictors of reduction in exacerbations in subjects randomized to 3- and 10-mg doses of pitrakinra in this study. The most important pharmacogenetic associations were observed for the correlated SNPs rs8832 and rs1029489 (*r2* = 0.75), which were not associated with a therapeutic pitrakinra response in our previous small antigen-challenge studies.^13^ Although differences in the 2 studies can be caused by different therapeutic end points, our primary hypothesis that *IL4RA* variation identifies a subset of asthmatic subjects more responsive to IL-4Rα inhibition was confirmed.

The pharmacologic mechanism for modulation of pitrakinra by rs8832 and rs1029489 is unknown because these tagging SNPs have no known function and reside in the *IL4RA* 3′ untranslated region and the proximal gene region, respectively (see [Fig E1](#SD1)). Additionally, because *IL4RA* SNPs significantly associated with clinical outcomes were common in the non-Hispanic white population (approximately 10% to 45% MAFs) and were chosen to “tag” correlated *IL4RA* variants, these SNPs might only be markers for a different functional variant, and further in-depth sequencing of the *IL4RA* 3′ region might provide additional information regarding the functional mechanism of *IL4RA* interaction with pitrakinra. We speculate that genetic variants in the 3′ untranslated region of the receptor might affect RNA stability or processing that could potentially interact to modulate T_H_2 inflammation. However, an important concept of this analysis is that it identifies a subset of subjects with severe persistent asthma who are responsive (33% of subjects with the rs8832GG genotype) to anti–IL-4Rα therapy and also shows that the other 67% were not responsive. Without this genetic biomarker, these responsive subjects are not easily identified. Other approaches that include use of blood biomarkers, such as eosinophil counts, characterize a somewhat more responsive subset that is not as specific as these pharmacogenetic findings. For example, subjects with the rs8832GG genotype had a greater relative reduction in asthma exacerbations at the 10-mg level compared with placebo than the group with high blood eosinophil counts.^15,16^ This genetic variant might potentially represent an excellent predictive genetic marker for therapeutic responses to anti–IL-4Rα receptor intervention. However, the predictive value of variation in *IL4RA* needs to be further replicated in additional studies with either pitrakinra or other biologic therapies that target IL-4, IL-13, or their receptor, perhaps by using a genotyped stratified trial design.^23,24^ These pharmacogenetic approaches could lead to targeted therapies in this pathway. A similar method has been recently proposed for anti–IL-13 treatment and predictive responses in asthmatic patients with higher serum periostin levels or higher levels of FENO.^10^ These biomarkers identify a subset of asthmatic subjects more responsive to an IL-13 antibody.^5^ Periostin, FENO, and other biomarkers also require replication and could potentially be used in conjunction with pharmacogenetic analysis.

In this important pharmacogenetic study, we were able to take advantage of comprehensive pharmacologic and phenotype information to evaluate the therapeutic interaction between *IL4RA* variants and different doses of pitrakinra, an IL-4 receptor antagonist. These results provide novel evidence that in non-Hispanic white subjects with moderate-to-severe asthma with a common genotype, the rs8832GG genotype treated with this anti–IL-4Rα inhibitor demonstrates a significant dose-dependent reduction in asthma exacerbations and improved outcomes for several other asthma-related clinical end points. Thus these results represent one of the first examples of a pharmacogenetic effect based on receptor variation that identifies a subset of subjects with severe asthma who respond to a new biologic intervention. These analyses provide a basis for understanding therapeutic heterogeneity in asthmatic subjects and are an important step toward paradigms that are based in individualized or stratified treatments in subjects with asthma and other common disorders.

## Supplementary Material

Clinical implications: Pharmacogenetic analysis of the *IL4RA* gene identified a subgroup of asthmatic subjects by genotype who were more responsive to therapy with an IL-4/IL-13 pathway antagonist.

Clinical implications: Pharmacogenetic analysis of the *IL4RA* gene identified a subgroup of asthmatic subjects by genotype who were more responsive to therapy with an IL-4/IL-13 pathway antagonist.

## Acknowledgments

We thank Carla Martin, Shelly Smith, and Siqun (Lilly) Zheng at the Wake Forest Center for Genomics and Personalized Medicine Research for genotyping assistance.

Supported by Aerovance, Bayer Healthcare, and the National Institute of Heart, Lung, and Blood grants HL065899, HL101487, and HL089992.

## Abbreviations used

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Hastie AT, Moore WC, Meyers DA, Vestal PL, Li H, Peters SP, et al. Analyses of asthma severity phenotypes and inflammatory proteins in subjects stratified by sputum granulocytes. J Allergy Clin Immunol. 2010;125:1028–36. e13. doi: 10.1016/j.jaci.2010.02.008.  [DOI](https://doi.org/10.1016/j.jaci.2010.02.008) | [PMC free article](/articles/PMC2878277/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20398920/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Allergy%20Clin%20Immunol&title=Analyses%20of%20asthma%20severity%20phenotypes%20and%20inflammatory%20proteins%20in%20subjects%20stratified%20by%20sputum%20granulocytes&author=AT%20Hastie&author=WC%20Moore&author=DA%20Meyers&author=PL%20Vestal&author=H%20Li&volume=125&publication_year=2010&pages=1028-36&pmid=20398920&doi=10.1016/j.jaci.2010.02.008&)

2. Moore WC, Meyers DA, Wenzel SE, Teague WG, Li H, Li X, et al. Identification of asthma phenotypes using cluster analysis in the Severe Asthma Research Program. Am J Respir Crit Care Med. 2010;181:315–23. doi: 10.1164/rccm.200906-0896OC.  [DOI](https://doi.org/10.1164/rccm.200906-0896OC) | [PMC free article](/articles/PMC2822971/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19892860/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Respir%20Crit%20Care%20Med&title=Identification%20of%20asthma%20phenotypes%20using%20cluster%20analysis%20in%20the%20Severe%20Asthma%20Research%20Program&author=WC%20Moore&author=DA%20Meyers&author=SE%20Wenzel&author=WG%20Teague&author=H%20Li&volume=181&publication_year=2010&pages=315-23&pmid=19892860&doi=10.1164/rccm.200906-0896OC&)

3. Kay AB. The role of T lymphocytes in asthma. Chem Immunol Allergy. 2006;91:59–75. doi: 10.1159/000090230.  [DOI](https://doi.org/10.1159/000090230) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16354949/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Chem%20Immunol%20Allergy&title=The%20role%20of%20T%20lymphocytes%20in%20asthma&author=AB%20Kay&volume=91&publication_year=2006&pages=59-75&pmid=16354949&doi=10.1159/000090230&)

4. Gavett SH, O’Hearn DJ, Karp CL, Patel EA, Schofield BH, Finkelman FD, et al. Interleukin-4 receptor blockade prevents airway responses induced by antigen challenge in mice. Am J Physiol Lung Cell Mol Physiol. 1997;272:L253–61. doi: 10.1152/ajplung.1997.272.2.L253.  [DOI](https://doi.org/10.1152/ajplung.1997.272.2.L253) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9124376/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Physiol%20Lung%20Cell%20Mol%20Physiol&title=Interleukin-4%20receptor%20blockade%20prevents%20airway%20responses%20induced%20by%20antigen%20challenge%20in%20mice&author=SH%20Gavett&author=DJ%20O%E2%80%99Hearn&author=CL%20Karp&author=EA%20Patel&author=BH%20Schofield&volume=272&publication_year=1997&pages=L253-61&pmid=9124376&doi=10.1152/ajplung.1997.272.2.L253&)

5. Grunig G, Warnock M, Wakil AE, Venkayya R, Brombacher F, Rennick DM, et al. Requirement for IL-13 independently of IL-4 in experimental asthma. Science. 1998;282:2261–3. doi: 10.1126/science.282.5397.2261.  [DOI](https://doi.org/10.1126/science.282.5397.2261) | [PMC free article](/articles/PMC3897229/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9856950/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=Requirement%20for%20IL-13%20independently%20of%20IL-4%20in%20experimental%20asthma&author=G%20Grunig&author=M%20Warnock&author=AE%20Wakil&author=R%20Venkayya&author=F%20Brombacher&volume=282&publication_year=1998&pages=2261-3&pmid=9856950&doi=10.1126/science.282.5397.2261&)

6. Oh CK, Geba GP, Molfino N. Investigational therapeutics targeting the IL-4/IL-13/ STAT-6 pathway for the treatment of asthma. Eur Respir Rev. 2010;19:46–54. doi: 10.1183/09059180.00007609.  [DOI](https://doi.org/10.1183/09059180.00007609) | [PMC free article](/articles/PMC9491642/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20956165/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20Respir%20Rev&title=Investigational%20therapeutics%20targeting%20the%20IL-4/IL-13/%20STAT-6%20pathway%20for%20the%20treatment%20of%20asthma&author=CK%20Oh&author=GP%20Geba&author=N%20Molfino&volume=19&publication_year=2010&pages=46-54&pmid=20956165&doi=10.1183/09059180.00007609&)

7. O’Byrne PM, Inman MD, Adelroth E. Reassessing the Th2 cytokine basis of asthma. Trends Pharmacol Sci. 2004;25:244–8. doi: 10.1016/j.tips.2004.03.008.  [DOI](https://doi.org/10.1016/j.tips.2004.03.008) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15120489/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Trends%20Pharmacol%20Sci&title=Reassessing%20the%20Th2%20cytokine%20basis%20of%20asthma&author=PM%20O%E2%80%99Byrne&author=MD%20Inman&author=E%20Adelroth&volume=25&publication_year=2004&pages=244-8&pmid=15120489&doi=10.1016/j.tips.2004.03.008&)

8. Flood-Page P, Swenson C, Faiferman I, Matthews J, Williams M, Brannick L, et al. A study to evaluate safety and efficacy of mepolizumab in patients with moderate persistent asthma. Am J Respir Crit Care Med. 2007;176:1062–71. doi: 10.1164/rccm.200701-085OC.  [DOI](https://doi.org/10.1164/rccm.200701-085OC) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17872493/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Respir%20Crit%20Care%20Med&title=A%20study%20to%20evaluate%20safety%20and%20efficacy%20of%20mepolizumab%20in%20patients%20with%20moderate%20persistent%20asthma&author=P%20Flood-Page&author=C%20Swenson&author=I%20Faiferman&author=J%20Matthews&author=M%20Williams&volume=176&publication_year=2007&pages=1062-71&pmid=17872493&doi=10.1164/rccm.200701-085OC&)

9. Barnes PJ. The cytokine network in asthma and chronic obstructive pulmonary disease. J Clin Invest. 2008;118:3546–56. doi: 10.1172/JCI36130.  [DOI](https://doi.org/10.1172/JCI36130) | [PMC free article](/articles/PMC2575722/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18982161/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Invest&title=The%20cytokine%20network%20in%20asthma%20and%20chronic%20obstructive%20pulmonary%20disease&author=PJ%20Barnes&volume=118&publication_year=2008&pages=3546-56&pmid=18982161&doi=10.1172/JCI36130&)

10. Corren J, Lemanske RF, Hanania NA, Korenblat PE, Parsey MV, Arron JR, et al. Lebrikizumab treatment in adults with asthma. N Engl J Med. 2011;365:1088–98. doi: 10.1056/NEJMoa1106469.  [DOI](https://doi.org/10.1056/NEJMoa1106469) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21812663/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Lebrikizumab%20treatment%20in%20adults%20with%20asthma&author=J%20Corren&author=RF%20Lemanske&author=NA%20Hanania&author=PE%20Korenblat&author=MV%20Parsey&volume=365&publication_year=2011&pages=1088-98&pmid=21812663&doi=10.1056/NEJMoa1106469&)

11. Burmeister Getz E, Fisher DM, Fuller R. Human pharmacokinetics/pharmacodynamics of an interleukin-4 and interleukin-13 dual antagonist in asthma. J Clin Pharmacol. 2009;49:1025–36. doi: 10.1177/0091270009341183.  [DOI](https://doi.org/10.1177/0091270009341183) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19717725/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Pharmacol&title=Human%20pharmacokinetics/pharmacodynamics%20of%20an%20interleukin-4%20and%20interleukin-13%20dual%20antagonist%20in%20asthma&author=E%20Burmeister%20Getz&author=DM%20Fisher&author=R%20Fuller&volume=49&publication_year=2009&pages=1025-36&pmid=19717725&doi=10.1177/0091270009341183&)

12. Wenzel S, Wilbraham D, Fuller R, Getz EB, Longphre M. Effect of an interleukin-4 variant on late phase asthmatic response to allergen challenge in asthmatic patients: results of two phase 2a studies. Lancet. 2007;370:1422–31. doi: 10.1016/S0140-6736(07)61600-6.  [DOI](https://doi.org/10.1016/S0140-6736(07)61600-6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17950857/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Effect%20of%20an%20interleukin-4%20variant%20on%20late%20phase%20asthmatic%20response%20to%20allergen%20challenge%20in%20asthmatic%20patients:%20results%20of%20two%20phase%202a%20studies&author=S%20Wenzel&author=D%20Wilbraham&author=R%20Fuller&author=EB%20Getz&author=M%20Longphre&volume=370&publication_year=2007&pages=1422-31&pmid=17950857&doi=10.1016/S0140-6736(07)61600-6&)

13. Slager RE, Hawkins GA, Ampleford EJ, Bowden A, Stevens LE, Morton MT, et al. IL-4 receptor alpha polymorphisms are predictors of a pharmacogenetic response to a novel IL-4/IL-13 antagonist. J Allergy Clin Immunol. 2010;126:875–8. doi: 10.1016/j.jaci.2010.08.001.  [DOI](https://doi.org/10.1016/j.jaci.2010.08.001) | [PMC free article](/articles/PMC3772526/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20920778/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Allergy%20Clin%20Immunol&title=IL-4%20receptor%20alpha%20polymorphisms%20are%20predictors%20of%20a%20pharmacogenetic%20response%20to%20a%20novel%20IL-4/IL-13%20antagonist&author=RE%20Slager&author=GA%20Hawkins&author=EJ%20Ampleford&author=A%20Bowden&author=LE%20Stevens&volume=126&publication_year=2010&pages=875-8&pmid=20920778&doi=10.1016/j.jaci.2010.08.001&)

14. Tomkinson A, Tepper J, Morton M, Bowden A, Stevens L, Harris P, et al. Inhaled vs subcutaneous effects of a dual IL-4/IL-13 antagonist in a monkey model of asthma. Allergy. 2010;65:69–77. doi: 10.1111/j.1398-9995.2009.02156.x.  [DOI](https://doi.org/10.1111/j.1398-9995.2009.02156.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19796211/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Allergy&title=Inhaled%20vs%20subcutaneous%20effects%20of%20a%20dual%20IL-4/IL-13%20antagonist%20in%20a%20monkey%20model%20of%20asthma&author=A%20Tomkinson&author=J%20Tepper&author=M%20Morton&author=A%20Bowden&author=L%20Stevens&volume=65&publication_year=2010&pages=69-77&pmid=19796211&doi=10.1111/j.1398-9995.2009.02156.x&)

15. Wenzel SE, Ind PW, Otulana BA, Bleecker ER, Kuna P, Yen YP. Inhaled pitrakinra, an IL-4/IL-13 antagonist, reduced exacerbations in patients with eosinophilic asthma. Eur Respir J. 2010;36:P3980.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20Respir%20J&title=Inhaled%20pitrakinra,%20an%20IL-4/IL-13%20antagonist,%20reduced%20exacerbations%20in%20patients%20with%20eosinophilic%20asthma&author=SE%20Wenzel&author=PW%20Ind&author=BA%20Otulana&author=ER%20Bleecker&author=P%20Kuna&volume=36&publication_year=2010&pages=P3980&)

16. Wenzel SE, Ind PW, Otulana BA, Bowden A, Puthukkeril S, Tomkinson A, et al. A phase 2b study of inhaled pitrakinra, an IL-4/IL-13 antagonist, successfully identified responder subpopulations of patients with uncontrolled asthma. Am J Respir Crit Care Med. 2011;183:A6179.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Respir%20Crit%20Care%20Med&title=A%20phase%202b%20study%20of%20inhaled%20pitrakinra,%20an%20IL-4/IL-13%20antagonist,%20successfully%20identified%20responder%20subpopulations%20of%20patients%20with%20uncontrolled%20asthma&author=SE%20Wenzel&author=PW%20Ind&author=BA%20Otulana&author=A%20Bowden&author=S%20Puthukkeril&volume=183&publication_year=2011&pages=A6179&)

17. Wenzel SE, Balzar S, Ampleford E, Hawkins GA, Busse WW, Calhoun WJ, et al. IL4R alpha mutations are associated with asthma exacerbations and mast cell/IgE expression. Am J Respir Crit Care Med. 2007;175:570–6. doi: 10.1164/rccm.200607-909OC.  [DOI](https://doi.org/10.1164/rccm.200607-909OC) | [PMC free article](/articles/PMC1899282/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17170387/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Respir%20Crit%20Care%20Med&title=IL4R%20alpha%20mutations%20are%20associated%20with%20asthma%20exacerbations%20and%20mast%20cell/IgE%20expression&author=SE%20Wenzel&author=S%20Balzar&author=E%20Ampleford&author=GA%20Hawkins&author=WW%20Busse&volume=175&publication_year=2007&pages=570-6&pmid=17170387&doi=10.1164/rccm.200607-909OC&)

18. Lemanske RF, Jr, Sorkness CA, Mauger EA, Lazarus SC, Boushey HA, Fahy JV, et al. Inhaled corticosteroid reduction and elimination in patients with persistent asthma receiving salmeterol: a randomized controlled trial. JAMA. 2001;285:2594–603. doi: 10.1001/jama.285.20.2594.  [DOI](https://doi.org/10.1001/jama.285.20.2594) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11368733/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA&title=Inhaled%20corticosteroid%20reduction%20and%20elimination%20in%20patients%20with%20persistent%20asthma%20receiving%20salmeterol:%20a%20randomized%20controlled%20trial&author=RF%20Lemanske&author=CA%20Sorkness&author=EA%20Mauger&author=SC%20Lazarus&author=HA%20Boushey&volume=285&publication_year=2001&pages=2594-603&pmid=11368733&doi=10.1001/jama.285.20.2594&)

19. Lazarus SC, Boushey HA, Fahy JV, Chinchilli VM, Lemanske RF, Jr, Sorkness CA, et al. Long-acting beta2-agonist monotherapy vs continued therapy with inhaled corticosteroids in patients with persistent asthma: a randomized controlled trial. JAMA. 2001;285:2583–93. doi: 10.1001/jama.285.20.2583.  [DOI](https://doi.org/10.1001/jama.285.20.2583) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11368732/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA&title=Long-acting%20beta2-agonist%20monotherapy%20vs%20continued%20therapy%20with%20inhaled%20corticosteroids%20in%20patients%20with%20persistent%20asthma:%20a%20randomized%20controlled%20trial&author=SC%20Lazarus&author=HA%20Boushey&author=JV%20Fahy&author=VM%20Chinchilli&author=RF%20Lemanske&volume=285&publication_year=2001&pages=2583-93&pmid=11368732&doi=10.1001/jama.285.20.2583&)

20. Miller MR, Hankinson J, Brusasco V, Burgos F, Casaburi R, Coates A, et al. Standardisation of spirometry. Eur Respir J. 2005;26:319–38. doi: 10.1183/09031936.05.00034805.  [DOI](https://doi.org/10.1183/09031936.05.00034805) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16055882/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20Respir%20J&title=Standardisation%20of%20spirometry&author=MR%20Miller&author=J%20Hankinson&author=V%20Brusasco&author=F%20Burgos&author=R%20Casaburi&volume=26&publication_year=2005&pages=319-38&pmid=16055882&doi=10.1183/09031936.05.00034805&)

21. Purcell S, Neale B, Todd-Brown K, Thomas L, Ferreira MA, Bender D, et al. PLINK: a tool set for whole-genome association and population-based linkage analyses. Am J Hum Genet. 2007;81:559–75. doi: 10.1086/519795.  [DOI](https://doi.org/10.1086/519795) | [PMC free article](/articles/PMC1950838/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17701901/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=PLINK:%20a%20tool%20set%20for%20whole-genome%20association%20and%20population-based%20linkage%20analyses&author=S%20Purcell&author=B%20Neale&author=K%20Todd-Brown&author=L%20Thomas&author=MA%20Ferreira&volume=81&publication_year=2007&pages=559-75&pmid=17701901&doi=10.1086/519795&)

22. Barrett JC, Fry B, Maller J, Daly MJ. Haploview: analysis and visualization of LD and haplotype maps. Bioinformatics. 2005;21:263–5. doi: 10.1093/bioinformatics/bth457.  [DOI](https://doi.org/10.1093/bioinformatics/bth457) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15297300/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=Haploview:%20analysis%20and%20visualization%20of%20LD%20and%20haplotype%20maps&author=JC%20Barrett&author=B%20Fry&author=J%20Maller&author=MJ%20Daly&volume=21&publication_year=2005&pages=263-5&pmid=15297300&doi=10.1093/bioinformatics/bth457&)

23. Bleecker ER, Postma DS, Lawrance RM, Meyers DA, Ambrose HJ, Goldman M. Effect of ADRB2 polymorphisms on response to longacting beta2-agonist therapy: a pharmacogenetic analysis of two randomised studies. Lancet. 2007;370:2118–25. doi: 10.1016/S0140-6736(07)61906-0.  [DOI](https://doi.org/10.1016/S0140-6736(07)61906-0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18156033/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Effect%20of%20ADRB2%20polymorphisms%20on%20response%20to%20longacting%20beta2-agonist%20therapy:%20a%20pharmacogenetic%20analysis%20of%20two%20randomised%20studies&author=ER%20Bleecker&author=DS%20Postma&author=RM%20Lawrance&author=DA%20Meyers&author=HJ%20Ambrose&volume=370&publication_year=2007&pages=2118-25&pmid=18156033&doi=10.1016/S0140-6736(07)61906-0&)

24. Wechsler ME, Lehman E, Lazarus SC, Lemanske RF, Jr, Boushey HA, Deykin A, et al. Beta-Adrenergic receptor polymorphisms and response to salmeterol. Am J Respir Crit Care Med. 2006;173:519–26. doi: 10.1164/rccm.200509-1519OC.  [DOI](https://doi.org/10.1164/rccm.200509-1519OC) | [PMC free article](/articles/PMC2662935/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16322642/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Respir%20Crit%20Care%20Med&title=Beta-Adrenergic%20receptor%20polymorphisms%20and%20response%20to%20salmeterol&author=ME%20Wechsler&author=E%20Lehman&author=SC%20Lazarus&author=RF%20Lemanske&author=HA%20Boushey&volume=173&publication_year=2006&pages=519-26&pmid=16322642&doi=10.1164/rccm.200509-1519OC&)
