# Association of SNPs on Chromosome 9p21.3 with Platelet Reactivity: A Potential Mechanism for Increased Vascular Disease

## Metadata
**Authors:** Kiran Musunuru, Wendy S Post, William Herzog, Haiqing Shen, Jeffrey R O’Connell, Patrick F McArdle, Kathleen A Ryan, Quince Gibson, Yu-Ching Cheng, Elizabeth Clearfield, Andrew D Johnson, Geoffrey Tofler, Qiong Yang, Christopher J O’Donnell, Diane M Becker, Lisa R Yanek, Lewis C Becker, Nauder Faraday, Lawrence F Bielak, Patricia A Peyser, Alan R Shuldiner, Braxton D Mitchell
**Journal:** Circulation. Cardiovascular genetics
**Date:** 2010 Sep 21
**DOI:** [10.1161/CIRCGENETICS.109.923508](https://doi.org/10.1161/CIRCGENETICS.109.923508)
**PMID:** 20858905
**PMCID:** PMC3031788
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3031788/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC3031788/pdf/nihms260284.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC3031788/pdf/nihms260284.pdf)

## Abstract

**Background:** 
Genome-wide association studies have identified a locus on chromosome 9p21.3 to be strongly associated with myocardial infarction/coronary artery disease (MI/CAD) and ischemic stroke. To gain insights into the mechanisms underlying these associations, we hypothesized that single nucleotide polymorphisms (SNPs) in this region would be associated with platelet reactivity across multiple populations.

**Methods and Results:** 
Subjects in the initial population included 1,402 asymptomatic Amish adults in whom we measured platelet reactivity (n=788) and/or coronary artery calcification (CAC) (n=939). Platelet reactivity on agonist stimulation was measured by impedence aggregometry, and CAC by electron beam computed tomography. Twenty-nine SNPs at the 9p21.3 locus were genotyped using the Affymetrix 500K array. Twelve correlated SNPs in the locus were significantly associated with platelet reactivity (all p ≤ 0.001). The SNP most strongly associated with platelet reactivity, rs10965219 (p = 0.0002) was also associated with CAC (p = 0.002), along with 9 other SNPs (all p < 0.004). Association of rs10965219 with platelet reactivity persisted after adjustment for CAC, a measure of underlying atherosclerotic burden known to affect platelet reactivity. We then tested rs10965219 for association with platelet function in 2,364 subjects from the Framingham Heart Study (FHS) and 1,169 subjects from the GeneSTAR Study. The rs10965219 G allele (frequency ~ 51% across all three populations) was significantly associated with higher platelet reactivity in FHS (p = 0.001) and trended toward higher reactivity in GeneSTAR (p = 0.087); the combined p-value for meta-analysis was 0.0002.

**Conclusions:** 
These results suggest that risk alleles at 9p21.3 locus may have pleiotropic effects on MI/CAD and stroke risk, possibly through their influence on platelet reactivity.

Keywords: Genetics, cardiovascular diseases, coronary disease, stroke, platelets

### Background

Genome-wide association studies have identified a locus on chromosome 9p21.3 to be strongly associated with myocardial infarction/coronary artery disease (MI/CAD) and ischemic stroke. To gain insights into the mechanisms underlying these associations, we hypothesized that single nucleotide polymorphisms (SNPs) in this region would be associated with platelet reactivity across multiple populations.

### Methods and Results

Subjects in the initial population included 1,402 asymptomatic Amish adults in whom we measured platelet reactivity (n=788) and/or coronary artery calcification (CAC) (n=939). Platelet reactivity on agonist stimulation was measured by impedence aggregometry, and CAC by electron beam computed tomography. Twenty-nine SNPs at the 9p21.3 locus were genotyped using the Affymetrix 500K array. Twelve correlated SNPs in the locus were significantly associated with platelet reactivity (all p ≤ 0.001). The SNP most strongly associated with platelet reactivity, rs10965219 (p = 0.0002) was also associated with CAC (p = 0.002), along with 9 other SNPs (all p < 0.004). Association of rs10965219 with platelet reactivity persisted after adjustment for CAC, a measure of underlying atherosclerotic burden known to affect platelet reactivity. We then tested rs10965219 for association with platelet function in 2,364 subjects from the Framingham Heart Study (FHS) and 1,169 subjects from the GeneSTAR Study. The rs10965219 G allele (frequency ~ 51% across all three populations) was significantly associated with higher platelet reactivity in FHS (p = 0.001) and trended toward higher reactivity in GeneSTAR (p = 0.087); the combined p-value for meta-analysis was 0.0002.

### Conclusions

These results suggest that risk alleles at 9p21.3 locus may have pleiotropic effects on MI/CAD and stroke risk, possibly through their influence on platelet reactivity.

## INTRODUCTION

Myocardial infarction (MI) is one of the leading causes of death and infirmity worldwide, with more than 800,000 individuals suffering an MI annually in the United States alone.[1](#R1) Susceptibility to MI is at least partly inherited, and recent studies have identified several of single nucleotide polymorphisms (SNPs) that are associated with MI.[2](#R2) Among these SNPs is a block of common SNPs located on chromosome 9p21.3 that was associated with MI and/or coronary artery disease (CAD) in three different genome-wide association studies[3](#R3)^–^[5](#R5) (GWAS). Association with these SNPs has now been consistently and robustly replicated in multiple populations,[3](#R3)^–^[16](#R16) and the locus is estimated to account for 10–15% of the population attributable risk for CAD in one study[3](#R3) and up to 21% in a second.[4](#R4) Some of these same MI/CAD-associated SNPs have been associated with other vascular phenotypes, including coronary artery calcification (CAC),[3](#R3)^, ^[7](#R7)^, ^[17](#R17) aortic and intracranial aneurysms,[9](#R9) and more recently with ischemic stroke.[11](#R11)^, ^[18](#R18)

SNPs at the chromosome 9p locus are not known to be associated with traditional cardiovascular disease CVD risk factors, such as blood pressure or lipids, nor have robust associations been reported with subclinical atherosclerosis as measured by carotid arterial wall thickness.[19](#R19)^, ^[20](#R20) It is possible, however, that the chromosome 9p locus may influence mechanisms common to both MI/CAD and stroke. Platelets, for example, play a critical role in vascular repair, and when activated under certain pathological conditions they can lead to thrombus formation and vessel occlusion.[21](#R21) Moreover, therapeutic interventions that decrease platelet reactivity, i.e., aspirin and/or clopidogrel, decrease risk of the vascular occlusive events that underlie MI and stroke.[22](#R22)^, ^[23](#R23) We therefore hypothesized that MI/CAD-associated SNPs at chromosome 9p21.3 would be associated with increased platelet reactivity, as assessed by *ex vivo* platelet aggregation measures. To address this hypothesis, we examined the associations of MI-associated SNPs on chromosome 9p with platelet reactivity in an Amish population and then sought to replicate observed associations in two independent populations. We additionally sought to confirm the associations of chromosome 9p SNPs with CAC in the Amish to replicate previously reported associations,[3](#R3)^, ^[7](#R7)^, ^[17](#R17) and to determine whether CAC predisposing risk alleles would be associated with increased platelet aggregation and if so, whether the effect of 9p SNPs on platelet reactivity would be independent of underlying atherosclerotic burden.

## METHODS

The initial study population included 788 Amish individuals from the Heredity and Phenotype Intervention (HAPI) Heart Study,[24](#R24) in whom platelet reactivity had been measured, as well as a partially overlapping sample of 939 subjects in whom CAC had been measured as part of the Amish Family Calcification Study (AFCS).[25](#R25) Recruitment of subjects for these two studies was carried out between the years of 2002 and 2006. The participant pool was largely healthy and asymptomatic from a CVD perspective. HAPI Heart Study participants ranged in age from 20–80 years; exclusions from participation in the study included severe hypertension (systolic blood pressure > 180 mmHg, diastolic blood pressure > 105 mmHg), kidney disease, liver disease, untreated thyroid disease, and malignancy of any type. Participants of the AFCS aged 30 years and older were recruited without regard to any CVD phenotype. Informed consent was obtained from all participants. The study protocols were approved by the Institutional Review Boards at the University of Maryland School of Medicine and other participating institutions.

Examination procedures for both studies included a medical and family history and focused physical examination at the Amish Research Center in Strasburg, PA, and collection of a morning blood sample following an overnight fast for clinical chemistries and DNA analysis.

Platelet reactivity studies were carried out in 788 HAPI Heart Study subjects.[24](#R24) Prior to enrollment into the platelet reactivity arm of the HAPI Heart Study, subjects were instructed not to take vitamins, herbal supplements, or medications two weeks prior to and during the study period. At an initial clinic visit, a fasting blood sample was obtained for measurement of complete blood count with differential. Subjects with platelet counts between 100,000/μl and 500,000/μl and white cell counts less than 20,000/μl were eligible for platelet studies, in whom a second blood sample was collected in a syringe with sodium citrate anti-coagulant at a final concentration of 0.0105 M for the baseline platelet activity measures. Reactivity studies included whole blood platelet impedence aggregometry with a Chrono-Log four channel aggregometer (Havertown, PA) within three hours after the blood draw, with incubation wells set to 37°C and stirring speed to 1000 rpm. Pre-warmed cuvettes were each filled with equal amounts of citrate anticoagulated whole blood and Hank’s Balanced Salt Solution (Sigma-Aldrich, St. Louis MO). After a five-minute incubation period, a pre-warmed probe was inserted into each cuvette, the aggregation baseline was set to zero, and the impedance circuit was calibrated to 50%. Collagen (Chrono-Log) was added at final concentrations of 0.5, 1.0, 2.0, and 5.0 μg/ml, and peak aggregation at five minutes was measured.

CAC was measured in 939 Amish subjects by electron beam computed tomography on an Imatron C-150 scanner (GE, South San Francisco, CA) using a standard protocol that included thirty to forty 3-mm contiguous transverse slices between the aortic root and the apex of the heart, gated to 80% of the R-R interval obtained during a single breath hold.[25](#R25) CAC was quantified using the Agatston score method, incorporating both density and area. We defined presence of calcification as a CAC score ≥ 1 [density > 130 Hounsfield units in greater than three contiguous pixels (> 1 mm^2^)]. The sum of the scores in the left main, left anterior descending, circumflex, and right coronary arteries was considered the CAC score. Interscan reproducibility for quantification of CAC with this software was previously reported to range from 89% to 94%. The interreader and intrareader reproducibilities were each ~99%.

Genotyping of Amish subjects was performed using the Affymetrix GeneChip® Human Mapping 500K Array set. We selected 42 SNPs in the 175 kb region of interest falling between positions 21,948,524 and 22,124,094 bp on chromosome 9p21.3 (NCBI Map Build 36.3). The GTYPE-generated chip files were analyzed using the BRLMM genotype calling algorithm. Of the 42 SNPs genotyped in the 175 kb region of interest, we excluded from analysis 3 SNPs with call rates < 0.93 and10 SNPs with minor allele frequencies < 0.02. This left a total of 29 analyzable SNPs. The mean genotyping call rate of these SNPs was 98.6%.

### Replication cohorts

#### Framingham Heart Study (FHS)

Platelet aggregation was measured from platelet rich plasma in 2,364 subjects who also underwent a GWAS using the Affymetrix 500K platform, and an additional 50K Affymetrix gene-focused MIPS array. Subjects with prevalent CVD and those using aspirin or anticoagulant medications were excluded from analysis. The extent of platelet aggregation was determined via a four channel aggregometer (BioData Corp., Horsham, PA) as the minimum threshold concentration required to induce a biphasic response within 5 minutes post-epinephrine addition (doses: 0.01, 0.03, 0.05, 0.1, 0.5, 1.0, 3.0, 5.0, 10.0 *u*M); thus, a higher threshold aggregation reflected more agonist required to initiate aggregation and less baseline aggregation. Mean age (± SD) of study subjects was 54.8 ± 9.8 years and 43.6% were male. Details of the assay and trait definition have been previously described.[26](#R26)

#### Genetic Study of Aspirin Responsiveness (GeneSTAR)

Whole blood impedance aggregometry was carried out in 1,169 European American subjects from 327 families who had undergone GWAS analysis using the Illumina 1M platform at deCODE Genetics in Reykjavik, Iceland. Subjects with prevalent CVD were excluded from platelet aggregation studies as were those with history of bleeding disorders, hemorrhagic events, or aspirin intolerance, those with serious comorbidities, and those who were taking aspirin or anticoagulant medications who could not safely discontinue use prior to these studies. Mean age (± SD) of study subjects was 45 ± 13 years and 44.9% were male. Platelet aggregation studies were conducted under a similar protocol as in the Amish HAPI Heart Study, with the exception that the GeneSTAR protocol did not include a 0.5 μg/ml concentration of collagen agonist and therefore for this replication study platelet reactivity was assessed following stimulation with a 1.0 μg/ml concentration of collagen. Details of the assay and trait definition have been previously described.[27](#R27)

To ensure that allele calls were consistent between the Amish/FHS studies, both of which were genotyped on the Affymetrix platform and GeneSTAR, which was genotyped on the Illumina platform, we genotyped rs10965219 in a subset of samples from the Amish study and GeneSTAR via TaqMan (Applied Biosystems, Inc.) and aligned the TaqMan genotype calls with our respective GWAS genotype calls. Because no TaqMan assay was available for rs10965219, we genotyped in its place rs1360590, a SNP in high linkage disequilibrium (LD) (r^2^ = 0.93 in HapMap CEU with rs10965219). Genotyping was carried out in 96 samples from each study. Genotype concordance rates were 98.7% and 94.5% in the Amish study and GeneSTAR, respectively.

### Statistical analysis

The observed distribution of genotypes was tested for deviation from Hardy-Weinberg equilibrium using Pearson’s χ^2^ test. Pairwise LD correlation statistics (r^2^) were computed using the Haploview software program ([www.broad.mit.edu](http://www.broad.mit.edu)). Association analyses of platelet reactivity and CAC quantity in Amish subjects were performed under the measured genotype variance component mixed model that assesses the additive effect of genotype on the quantitative trait, while simultaneously estimating the effects of age, age[2](#R2), gender, and a polygenic component to account for phenotypic correlation due to relatedness.[28](#R28) In the additive model, genotype is coded as the number of copies of the reference allele (0, 1, or 2), and the significance of the genotype effect is assessed using a 1 df test. The polygenic component was modeled using the relationship matrix derived from the pedigree structure constructed from the entire Lancaster Amish settlement dating back 14 generations since all subjects are related. We have implemented this mixed model analysis in an in house software program called Mixed Model Analysis for Pedigrees (MMAP) that is computationally efficient for large pedigrees such as the Amish.[29](#R29) Because the distribution of CAC scores was positively skewed and not all subjects had detectable CAC, the scores were natural log-transformed after adding 1. Age- and sex-residualized CAC scores were approximately normally distributed, as were the distributions of the platelet aggregation measures in response to each dose of collagen agonist. We estimated that our study would have 80% power to detect SNPs accounting for 2% or more of the variation in each of platelet aggregation (n = 788) and CAC score (n = 939) at an alpha level of 0.005 (see below).

Association analyses of the FHS and GeneSTAR replication cohorts were performed within each cohort and then combined for meta-analysis using a weighted z-score-based fixed effects meta-analysis approach. Both FHS and GeneSTAR evaluated age- and sex-adjusted models for aggregation phenotypes. FHS analyzed the log_10_ transformation of epinephrine concentrations that produced a half-maximal response. FHS and GeneSTAR included the principal components (PC) from EIGENSTRAT 2.0[30](#R30) (n=8 and n=2, respectively) as covariates to account for potential population admixture. Linear mixed effects (LME) models were used in the respective studies to test the association under an additive model between a SNP and specific phenotype adjusted for age, sex and PCs. Additional adjustments were made for diabetes, hypertension, current smoking, body mass index, LDL cholesterol and fibrinogen in GeneSTAR. SNP genotypes were included as fixed effects using an additive model (0 for one major allele, 1 for the heterozygote, and 2 for the minor allele homozygote genotype) for the original genotypes and dosage (probabilistic estimations) for the imputed genotypes. We tested whether the SNP additive effects differed from zero. FHS used the R kinship and GWAF packages,[31](#R31) accounting for familial relatedness, while GeneSTAR used PROC MIXED in SAS (v. 9.1.3 for Linux OS) with the option for EMPIRICAL variance and including the family identification number in the random effects to account for relatedness.

Our meta-analysis approach does not compute summary effect sizes across studies (which would be inappropriate given that somewhat different measures of platelet aggregation were used in the Amish, FHS, and GeneSTAR studies). Using the METAL software program ([http://www.sph.umich.edu/csg/abecasis/Metal/index.html](http://www.sph.umich.edu/csg/abecasis/Metal/index.html)), we defined a reference allele and generated a z-statistic summarizing the magnitude of the p-value for association (under the additive model) and direction of effect for each study. An overall z-statistic was then computed as a weighted average of the individual statistics and a corresponding p-value for that statistic was computed. The weights were proportional to the square root of the number of individuals in each study and scaled such that the squared weights summed to 1. In constructing the overall z-statistic, we multiplied the beta coefficient reflecting the SNP association with platelet aggregation in FHS by minus one to make the directions of effect similar between the three studies. In the Amish and in GeneSTAR the measurements reflect platelet aggregation, while in FHS, values reflect the dose of agonist required to initiate aggregation and higher doses reflect lower aggregation.

## RESULTS

Characteristics of the 788 Amish subjects in whom platelet aggregation studies were performed are shown in [Table 1](#T1). This sample included 428 men and 360 women, and the mean age was 42.8 ± 13.5 years. Participating subjects were relatively healthy as evidenced by the low prevalence of hypertension (8%–13%) and diabetes (< 1%). Fewer than 2% of the study subjects reported current use of antihypertensive or cholesterol-lowering medications, and 1.4% of subjects reported a prior CV event (i.e. history of MI, stroke, or a coronary angiogram with a detectable blockage). Mean platelet reactivity was slightly higher in women than in men (11.8 ohms vs. 10.4 ohms with collagen 0.5 μg/ml). CAC was measured in a total of 939 Amish subjects (430 men and 509 women, 325 of whom were included in the platelet aggregation studies). Mean age of the subjects in whom CAC was measured was 50.3 yrs in men and 53.4 yrs in women and 8.2% of subjects reported a prior CV event. The proportion of subjects with detectable CAC was 57.9% (66.7% men and 50.4% women).

### Table 1.

| Trait | Men (n = 428) | Women (n = 360) |
| --- | --- | --- |
| Age, yrs | 41.5 (13.1) | 44.5 (13.8) |
| Body mass index, kg/m2 | 25.5 (3.2) | 27.6 (5.3) |
| Systolic blood pressure, mmHg | 120.9 (11.9) | 120.7 (16.0) |
| Hypertension†, % | 8.6 | 12.8 |
| Diabetes, % | 0.7 | 0.6 |
| Total cholesterol, mg/dL | 203.3 (44.9) | 213.9 (47.4) |
| HDL-cholesterol, mg/dL | 53.1 (12.8) | 59.4 (15.4) |
| LDL-cholesterol, mg/dL | 137.7 (41.5) | 140.2 (44.7) |
| Triglycerides, mg/dL | 62.4 (34.7) | 71.4 (44.1) |
| Current smoking (pipes only), % | 19.9 | 0.0 |
| Presence of CAC, %* | 66.6 | 49.5 |
| Ln(CAC+1)* | 3.36 (2.85) | 2.09 (2.49) |
| Platelet aggregation in response to collagen 0.5 μg/ml in ohms, mean (SE) | 10.4 (2.8) | 11.8 (2.7) |

Table 1 Caption: Characteristics (mean ± SD) of 788 Amish study subjects undergoing platelet reactivity studies

As shown in [Figure 1](#F1), the 29 SNPs in the region of interest formed essentially two large haplotype blocks in Amish subjects, the first extending approximately 49 kb from rs643319 to rs10965219 (SNPs 4-15 in [Figure 1](#F1)) and the second extending approximately 37 kb from rs10757272 to rs1333048 (SNPs 20-24 in [Figure 1](#F1)). The genotype distributions of all SNPs conformed with Hardy-Weinberg expectations (all p > 0.001). SNPs associated with MI/CAD in previously published studies fall in both blocks.

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3d2f/3031788/cdad4e85e159/nihms260284f1.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3031788_nihms260284f1.jpg)

Linkage disequilibrium (LD) structure shown as r2 in the chromosome 9p21.3 region in Amish subjects. All genotyped SNPs passing quality control with minor allele frequencies > 0.02 are shown. Genomic locations are given according to NCBI Map Build 36.3. Figure created with Haploview software.

### SNPs and Platelet Aggregation Associations in the Amish

Since the 29 tested SNPs were largely clustered in two blocks, we did not regard each SNP as being independently tested and so did not apply the Bonferroni correction for the total number of SNPs. Instead we corrected for 12 independent tests, providing a statistical significance threshold of 0.05/12, or p < 0.004. The 12 independent tests were based on subtracting from the 29 SNPs any SNP that was highly correlated in the Amish subjects (i.e., r^2^ > 0.65) with an already selected SNP. Thus, we selected SNPs 1-3, one SNP from Block 1 (SNPS 4-15), SNP 16, 1 SNP from SNP17/SNP18, SNP 19, 1 SNP from Block 2 (SNPS 20-24), SNPs 25-27, and 1 SNP from SNP 28/SNP 29 (see [Figure 1](#F1)). After adjusting for age, age[2](#R2), and gender, all 12 SNPs in Block 1 were associated with variation in platelet reactivity in response to 0.5 μg/ml collagen (p ≤ 0.001). The strongest associations observed for platelet aggregation were with SNPs rs10965212, rs7049105, rs10965215, and rs10965219 (p-values = 0.0001 – 0.0002), all of which were in very high LD (all pairwise r^2^ ≥ 0.97) ([Table 2](#T2)).

### Table 2.

|   |   | SNP rs number | Position (bp) | Minor allele frequency | Ln (CAC+1) | P-value for association with platelet aggregation (0.5 μg/ml collagen) |
| --- | --- | --- | --- | --- | --- | --- |
|   |   |   |  |  |  |  |
|   |   |   |   |   | (n = 939) | (n = 788) |
|   |   | rs717326 | 21,948,524 | 0.070 | 7.43E-01 | 9.44E-01 |
|   |   | rs3731201 | 21,978,896 | 0.082 | 8.20E-02 | 7.00E-02 |
|   |   | rs3217986 | 21,995,330 | 0.074 | 9.76E-01 | 3.43E-01 |
| Block 1 |   | rs643319 | 22,007,836 | 0.434 | 1.13E-03 | 2.10E-04 |
| rs7044859 | 22,008,781 | 0.476 | 2.55E-02 | 1.02E-03 |  |  |
| rs523096 | 22,009,129 | 0.496 | 2.10E-02 | 1.05E-03 |  |  |
| rs518394 | 22,009,673 | 0.489 | 2.70E-02 | 6.24E-04 |  |  |
| rs10757264 | 22,009,732 | 0.477 | 4.45E-02 | 6.08E-04 |  |  |
| rs10965212 | 22,013,795 | 0.473 | 4.34E-03 | 1.37E-04 |  |  |
| rs496892 | 22,014,351 | 0.435 | 8.32E-03 | 3.10E-04 |  |  |
| rs7049105 | 22,018,801 | 0.471 | 2.64E-03 | 1.25E-03 |  |  |
| rs10965215 | 22,019,445 | 0.468 | 3.35E-03 | 1.07E-04 |  |  |
| rs564398 | 22,019,547 | 0.430 | 2.34E-03 | 4.40E-04 |  |  |
| rs7865618 | 22,021,005 | 0.452 | 8.36E-03 | 5.18E-04 |  |  |
| rs10965219 | 22,043,687 | 0.480 | 2.00E-03 | 1.91E-04 |  |  |
|   |   | rs16905599 | 22,059,144 | 0.056 | 8.74E-01 | 1.22E-01 |
|   |   | rs9632884 | 22,062,301 | 0.473 | 4.61E-02 | 2.90E-02 |
|   |   | rs6475606 | 22,071,850 | 0.461 | 8.97E-03 | 1.64E-02 |
|   |   | rs10965228 | 22,072,380 | 0.154 | 2.72E-01 | 4.70E-03 |
| Block 2 |   | rs10757272 | 22,078,260 | 0.497 | 2.97E-04 | 6.79E-02 |
| rs4977574 | 22,088,574 | 0.497 | 2.13E-04 | 8.13E-02 |  |  |
| rs2891168 | 22,088,619 | 0.497 | 2.13E-04 | 8.13E-02 |  |  |
| rs1333042 | 22,093,813 | 0.482 | 9.42E-04 | 8.04E-02 |  |  |
| rs1333048 | 22,115,347 | 0.490 | 3.18E-03 | 1.37E-01 |  |  |
|   |   | rs10965244 | 22,120,389 | 0.188 | 3.11E-01 | 1.47E-01 |
|   |   | rs10965245 | 22,120,515 | 0.088 | 2.88E-01 | 9.01E-01 |
|   |   | rs2891169 | 22,121,825 | 0.456 | 3.36E-01 | 1.23E-02 |
|   |   | rs2383208 | 22,122,076 | 0.127 | 8.27E-01 | 1.27E-01 |
|   |   | rs10811661 | 22,124,094 | 0.143 | 6.01E-01 | 2.96E-01 |

Table 2 Caption: Additive model p-values representing the associations of 29 SNPs on chromosome 9p21.3 with ln transformed (CAC score + 1) and platelet aggregation, adjusted for age, age2, and gender in the Amish

All 12 SNPs in Block 1 were only weakly associated with platelet aggregation in response to collagen agonist at the 1 μg/ml concentration (all p-values 0.01 – 0.07), and no SNPs were associated with platelet aggregation in response to collagen agonist at the 2.0 or 5.0 μg/ml concentrations (data not shown).

### SNPs and CAC Associations in the Amish

After adjusting for age, age[2](#R2), and gender, CAC scores were significantly associated with SNPs in both haplotype blocks ([Table 2](#T2)). In Block 1, associations with CAC quantity were observed with five SNPs in the 36 kb region bounded by rs643319 (22,007,846 bp) and rs10965219 (22,043,687 bp) with p-values of the associated SNPs ranging from 0.001 – 0.004. All five SNPs in Block 2 were also associated with transformed CAC scores, in the 37 kb region bounded by rs10757272 (22,078,260 bp) and rs1333048 (22,115,347 bp) (p-values 0.0002 – 0.003).

Because there is some, albeit modest, LD between SNPs in Blocks 1 and 2 (e.g., 0.10 < r^2^ < 0.28 in Amish subjects), we performed a further analysis that included SNPs in both regions as independent variables to determine whether the associations observed with the Block 1 and Block 2 clusters were independent. The SNP rs564398 was selected as the representative CAC-associated SNP in Block 1 and rs4977574 as the representative CAC-associated SNP in Block 2. In this analysis the effect sizes of both SNPs were markedly reduced, and rs564398 no longer remained associated with CAC at the p < 0.01 significance threshold.

### rs10965219 and CAC and Platelet Aggregation Associations

For each of the SNPs associated with both CAC and platelet aggregation, the same allele that was associated with increased CAC score was also associated with increased platelet aggregation. We selected for more detailed analysis SNP rs10965219, as representative of the rs10965212-rs7049105-rs10965215-rs10965219 SNP cluster that was in near perfect LD and strongly associated with platelet aggregation. [Table 3](#T3) shows the proportion of subjects with any CAC and the mean CAC scores and platelet aggregation values by rs10965219 genotype. The rs10965219 G allele (frequency in Amish = 0.520) was associated with higher presence of any CAC (p = 0.01), higher CAC score (p = 0.0002), and increased platelet aggregation (p = 0.0002). After adjusting for age and gender, this SNP accounted for 1.7% of the variation in pre-aspirin platelet aggregation and 1.1% of the variation in transformed CAC score.

### Table 3.

| Trait | GG genotype | GA genotype | AA genotype | Age- and gender-adjusted p-value |
| --- | --- | --- | --- | --- |
| CAC | [n = 219]* | [n = 508] | [n = 197] |   |
| Presence of CAC**, % | 63.9 | 56.6 | 54.8 | 0.01 |
| ln(CAC + 1), mean (SE) | 3.24 (0.19) | 2.53 (1.63) | 2.56 (1.87) | 0.002 |
| Platelet aggregation measures | [n = 193] | [n = 415] | [n = 162] |   |
| Platelet aggregation in response to collagen 0.5 μg/ml in ohms, mean (SD) | 11.4 (2.6) | 11.2 (2.7) | 10.1 (3.4) | 0.0002 |

Table 3 Caption: Distribution of vascular and platelet phenotypes in 924 Amish subjects according to rs10965219 genotype.

There was no association between transformed CAC scores and platelet aggregation in the 325 subjects in whom both measures were obtained (age- and gender-adjusted r^2^ = 0.019, p = 0.73). Furthermore, the statistically significant association observed between the rs10965219 genotype and platelet aggregation was not diminished by adjusting for CAC scores, nor for any of the conventional CVD risk factors besides age and gender. Genotype for rs10965219 was not associated with body mass index, systolic blood pressure, total, HDL, or LDL cholesterol, or triglycerides in this sample (data not shown). Thus, the genotype - platelet aggregation association does not appear to be mediated by traditional CVD risk factors or by atherosclerotic burden as measured by CAC.

### Replication studies

We assessed the association of rs10965219 genotype with platelet reactivity in the FHS and GeneSTAR studies. This SNP was genotyped directly in FHS, but imputed in GeneSTAR using the MACH imputation program. The imputation quality score (defined as the average posterior probability for the most likely genotype) for this SNP was 0.80, and the r-square (defined as the squared correlation between imputed and true genotypes) was 0.56. The frequencies of the rs10965219 G allele were 0.507 and 0.506 in the Framingham and GeneSTAR studies, compared with 0.520 in the Amish. In each study, the G allele was associated with higher platelet reactivity, with the association achieving statistical significance in the FHS (p = 0.001), although not in GeneSTAR (p = 0.087) (see [Table 4](#T4)). The combined evidence for association across the two replication cohorts via meta-analysis was 0.0002.

### Table 4.

| Study | Amish [n = 788] | Framingham [n = 2,364] | GeneSTAR [n = 1,169] |
| --- | --- | --- | --- |
| Age, years, mean (SD) | 42.8 (13.5) | 54.4 (9.8) | 44.6 (13.2) |
| % male | 54.3 | 43.6 | 44.9 |
| Platelet reactivity measure | Whole blood platelet aggregation in response to collagen 0.5 μg/ml in ohms | Threshold aggregation concentration in response to epinephrine | Whole blood platelet aggregation in response to collagen 1.0 μg/ml in ohms |
| rs10965219 genotype beta (SE)* | −0.541 (0.154) | 0.058 (0.017)* | −0.574 (0.335) |
| Age- and sex-adjusted p-value | 0.0007 | 0.001 | 0.087† |

Table 4 Caption: Association of platelet reactivity with rs10965219 genotype in the Amish, Framingham Heart Study, and GeneSTAR populations.

## DISCUSSION

The chromosome 9p21.3 locus has recently attracted much attention because of the consistent associations observed between SNPs at this locus and CVD-related traits. As in the FHS[17](#R17) and other[3](#R3)^, ^[7](#R7) studies, we found common SNPs at this locus to be associated with CAC, a well-validated marker of atherosclerosis and predictor of MI. The novel contribution of the present study is our demonstration that some of these MI/CAD-associated alleles are also associated with increased platelet aggregation, an important risk factor for both MI and stroke. We found the chromosome 9p SNPs to be associated across three different populations to two different platelet aggregation phenotypes, two reflecting whole blood platelet aggregation in response to collagen and one reflecting threshold aggregation concentration in response to epinephrine agonist in platelet rich plasma. On the one hand, the differences among the platelet aggregation phenotypes among the three studies argue for the robustness of the genotype association with platelet aggregation, although on the other hand, one could also say that no single phenotype association has been replicated. Although differences in the platelet aggregation measures and their scales preclude direct comparison of effect sizes among the three studies, there is a suggestion that the platelet aggregation association may be stronger in the Amish (with collagen dose of 0.5 μg/ml) than in GeneStar (with collagen dose of 1.0 μg/ml). A stronger association in Amish may reflect increased sensitivity to differences in aggregation at a lower dose of collagen agonist. This speculation is supported by the fact that higher doses of collagen cause a more robust platelet activation response than lower doses, including secretion of platelet granules and thromboxane release, which are absent or incomplete at lower doses. It is thus possible that higher doses of collagen may overwhelm gene-association signals of subtle difference in platelet function that are pathophysiologically relevant and are discoverable using lower doses.

The association with platelet aggregation is intriguing, suggesting increased platelet reactivity leading to thrombosis may be a mechanism whereby this locus contributes to MI/CAD. It seems unlikely that the SNP-platelet reactivity association can be explained by associations of these SNPs with CAC since there was virtually no correlation between CAC and platelet aggregation in the Amish. Association with platelet reactivity adds to the growing number of traits that SNP variation at this locus has been shown to influence – a list that now includes in addition to MI/CAD and stroke, abdominal aortic and intracranial aneurysms,[9](#R9) periodontal disease,[32](#R32) and familial melanoma.[33](#R33) Moreover, a SNP in a different LD block approximately 10 kb centromeric to this MI/CAD-associated region has been consistently associated with type 2 diabetes mellitus.[9](#R9)^, ^[34](#R34)^–^[36](#R36) The diabetes-associated locus appears to be entirely distinct from the MI/CAD-associated locus.

Because SNPs in the MI/CAD associated region are in relatively high LD, it has been difficult to pinpoint the specific causal SNPs. In our Amish population, associations with CAC quantity extended over an 86 kb region that includes two large LD blocks, similar to the pattern seen in other Caucasian populations.[19](#R19) However, the most highly associated SNPs in each block were not independently associated with CAC, suggesting that the association could be driven by a single (yet to be identified) variant marked by both SNPs somewhere in this region.

Our data suggest that variant(s) affecting CAC are associated with SNPs in LD with both Blocks 1 and 2, while variants associated with platelet reactivity are associated most strongly to SNPs in Block 1. There are no annotated protein-coding genes that map to the CAC and platelet aggregation-associated SNPs in Block 1, although this region does map to a recently identified noncoding RNA, called *ANRIL*.[37](#R37) Noncoding RNAs can alter expression of associated protein-coding genes through a number of mechanisms.[38](#R38)^, ^[39](#R39) Such mechanisms include the knockdown of messenger RNAs and the alteration of gene transcription via the recruitment of chromatin-modifying enzymes or epigenetic silencing. *ANRIL* has been shown to be expressed in vascular endothelial cells, monocyte-derived macrophages, coronary smooth muscle cells and other cell types known to be affected by atherosclerosis, making it a strong candidate gene for the chromosome 9p disease associations reported.[40](#R40) Recently, Jarinova and colleagues have reported that a conserved sequence within this locus has enhancer activity and that the associated haplotype alters the regulatory sequence of *ANRIL* and changes expression levels of this gene. Through additional experiments, these authors then showed that *ANRIL* expression changes correlated with changes in expression of other genes, particularly those in pathways associated with cell proliferation.[41](#R41) Genotype associations with differential *ANRIL* gene expression have also been independently reported other groups.[42](#R42)^, ^[43](#R43) Thus, risk alleles in the 9p21.3 region may act by altering *ANRIL* expression levels and in turn potentially influence a wide variety of vascular responses. Ultimately, a molecular profiling of the appropriate locus sequences is mandatory to identify the causal variant, which reliably associates with CAD-related phenotypes. Understanding more fully the molecular basis underlying the 9p21.3 association may have important implications for understanding, preventing, and treating heart disease.

In summary, we confirm that common SNPs on chromosome 9p21.3 are associated with CAC, a well-recognized subclinical marker of CAD and a predictor of MI. We find that one mechanism underlying the well-replicated association of this locus with MI/CAC is likely to involve increased platelet reactivity, also a risk factor for CVD events. This observation is consistent with the observation that this locus is a risk factor for thromboembolic disease more generally, including stroke. Additional studies, including in subjects from other ethnic groups and with varying degrees of CVD severity, to further define the importance of platelet reactivity in those who carry the at-risk genotype may provide mechanistic insights towards personalized medicine through genotype-specific interventions that target platelet reactivity.

## Acknowledgments

This work was supported by NIH research grants U01 HL72515 and R01 HL088119, University of Maryland General Clinical Research Center grant (M01 RR 16500), and the Paul Beeson Physician Faculty Scholars in Aging Program of the American Federation of Aging Research. Partial funding was also provided by the Mid-Atlantic Nutrition and Obesity Research Center (P30 DK072488) and the Baltimore Veterans Administration Medical Center Geriatric Research and Education Clinical Center. GeneSTAR was supported by the National Heart, Lung, and Blood Institute through the PROGENI (U01 HL72518) and STAMPEED (R01 HL087698-01) consortia. This research was conducted in part using data and resources from the Framingham Heart Study of the National Heart, Lung and Blood Institute of the National Institutes of Health and Boston University School of Medicine, and was also supported by the National Heart, Lung and Blood Institute’s Framingham Heart Study (Contract No. N01-HC-25195) and its contract with Affymetrix, Inc for genotyping services (Contract No. N02-HL-6-4278), and through R01 HL48157. The analyses reflect intellectual input and resource development from the Framingham Heart Study investigators participating in the SNP Health Association Resource (SHARe) project. A portion of this research utilized the Linux Cluster for Genetic Analysis (LinGA-II) funded by the Robert Dawson Evans Endowment of the Department of Medicine at Boston University School of Medicine and Boston Medical Center.

## Footnotes

## References

1. Thom T, Haase N, Rosamond W, Howard VJ, Rumsfeld J, Manolio T, Zheng ZJ, Flegal K, O’Donnell C, Kittner S, Lloyd-Jones D, Goff DC, Jr, Hong Y, Adams R, Friday G, Furie K, Gorelick P, Kissela B, Marler J, Meigs J, Roger V, Sidney S, Sorlie P, Steinberger J, Wasserthiel-Smoller S, Wilson M, Wolf P American Heart Association Statistics Committee and Stroke Statistics Subcommittee. Heart disease and stroke statistics--2006 update: a report from the American Heart Association Statistics Committee and Stroke Statistics Subcommittee. Circulation. 2006;113:e85–151. doi: 10.1161/CIRCULATIONAHA.105.171600.  [DOI](https://doi.org/10.1161/CIRCULATIONAHA.105.171600) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16407573/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circulation&title=Heart%20disease%20and%20stroke%20statistics--2006%20update:%20a%20report%20from%20the%20American%20Heart%20Association%20Statistics%20Committee%20and%20Stroke%20Statistics%20Subcommittee&author=T%20Thom&author=N%20Haase&author=W%20Rosamond&author=VJ%20Howard&author=J%20Rumsfeld&volume=113&publication_year=2006&pages=e85-151&pmid=16407573&doi=10.1161/CIRCULATIONAHA.105.171600&)

2. Arking DE, Chakravarti A. Understanding cardiovascular disease through the lens of genome-wide association studies. Trends Genet. 2009;25:387–394. doi: 10.1016/j.tig.2009.07.007.  [DOI](https://doi.org/10.1016/j.tig.2009.07.007) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19716196/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Trends%20Genet&title=Understanding%20cardiovascular%20disease%20through%20the%20lens%20of%20genome-wide%20association%20studies&author=DE%20Arking&author=A%20Chakravarti&volume=25&publication_year=2009&pages=387-394&pmid=19716196&doi=10.1016/j.tig.2009.07.007&)

3. McPherson R, Pertsemlidis A, Kavaslar N, Stewart A, Roberts R, Cox DR, Hinds DA, Pennacchio LA, Tybjaerg-Hansen A, Folsom AR, Boerwinkle E, Hobbs HH, Cohen JC. A common allele on chromosome 9 associated with coronary heart disease. Science. 2007;316:1488–1491. doi: 10.1126/science.1142447.  [DOI](https://doi.org/10.1126/science.1142447) | [PMC free article](/articles/PMC2711874/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17478681/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=A%20common%20allele%20on%20chromosome%209%20associated%20with%20coronary%20heart%20disease&author=R%20McPherson&author=A%20Pertsemlidis&author=N%20Kavaslar&author=A%20Stewart&author=R%20Roberts&volume=316&publication_year=2007&pages=1488-1491&pmid=17478681&doi=10.1126/science.1142447&)

4. Helgadottir A, Thorleifsson G, Manolescu A, Gretarsdottir S, Blondal T, Jonasdottir A, Jonasdottir A, Sigurdsson A, Baker A, Palsson A, Masson G, Gudbjartsson DF, Magnusson KP, Andersen K, Levey AI, Backman VM, Matthiasdottir S, Jonsdottir T, Palsson S, Einarsdottir H, Gunnarsdottir S, Gylfason A, Vaccarino V, Hooper WC, Reilly MP, Granger CB, Austin H, Rader DJ, Shah SH, Quyyumi AA, Gulcher JR, Thorgeirsson G, Thorsteinsdottir U, Kong A, Stefansson K. A common variant on chromosome 9p21 affects the risk of myocardial infarction. Science. 2007;316:1491–1493. doi: 10.1126/science.1142842.  [DOI](https://doi.org/10.1126/science.1142842) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17478679/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=A%20common%20variant%20on%20chromosome%209p21%20affects%20the%20risk%20of%20myocardial%20infarction&author=A%20Helgadottir&author=G%20Thorleifsson&author=A%20Manolescu&author=S%20Gretarsdottir&author=T%20Blondal&volume=316&publication_year=2007&pages=1491-1493&pmid=17478679&doi=10.1126/science.1142842&)

5. Samani NJ, Erdmann J, Hall AS, Hengstenberg C, Mangino M, Mayer B, Dixon RJ, Meitinger T, Braund P, Wichmann HE, Barrett JH, Konig IR, Stevens SE, Szymczak S, Tregouet DA, Iles MM, Pahlke F, Pollard H, Lieb W, Cambien F, Fischer M, Ouwehand W, Blankenberg S, Balmforth AJ, Baessler A, Ball SG, Strom TM, Braenne I, Gieger C, Deloukas P, Tobin MD, Ziegler A, Thompson JR, Schunkert H WTCCC and the Cardiogenics Consortium. Genomewide association analysis of coronary artery disease. N Engl J Med. 2007;357:443–453. doi: 10.1056/NEJMoa072366.  [DOI](https://doi.org/10.1056/NEJMoa072366) | [PMC free article](/articles/PMC2719290/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17634449/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Genomewide%20association%20analysis%20of%20coronary%20artery%20disease&author=NJ%20Samani&author=J%20Erdmann&author=AS%20Hall&author=C%20Hengstenberg&author=M%20Mangino&volume=357&publication_year=2007&pages=443-453&pmid=17634449&doi=10.1056/NEJMoa072366&)

6. Abdullah KG, Li L, Shen GQ, Hu Y, Yang Y, Mackinlay KG, Topol EJ, Wang QK. Four SNPS on Chromosome 9p21 Confer Risk to Premature, Familial CAD and MI in an American Caucasian Population (GeneQuest) Ann Hum Genet. 2008;72:654–657. doi: 10.1111/j.1469-1809.2008.00454.x.  [DOI](https://doi.org/10.1111/j.1469-1809.2008.00454.x) | [PMC free article](/articles/PMC2634771/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18505420/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Hum%20Genet&title=Four%20SNPS%20on%20Chromosome%209p21%20Confer%20Risk%20to%20Premature,%20Familial%20CAD%20and%20MI%20in%20an%20American%20Caucasian%20Population%20(GeneQuest)&author=KG%20Abdullah&author=L%20Li&author=GQ%20Shen&author=Y%20Hu&author=Y%20Yang&volume=72&publication_year=2008&pages=654-657&pmid=18505420&doi=10.1111/j.1469-1809.2008.00454.x&)

7. Assimes TL, Knowles JW, Basu A, Iribarren C, Southwick A, Tang H, Absher D, Li J, Fair JM, Rubin GD, Sidney S, Fortmann SP, Go AS, Hlatky MA, Myers RM, Risch N, Quertermous T. Susceptibility locus for clinical and subclinical coronary artery disease at chromosome 9p21 in the multi-ethnic ADVANCE Study. Hum Mol Genet. 2008;17:2320–2328. doi: 10.1093/hmg/ddn132.  [DOI](https://doi.org/10.1093/hmg/ddn132) | [PMC free article](/articles/PMC2733811/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18443000/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mol%20Genet&title=Susceptibility%20locus%20for%20clinical%20and%20subclinical%20coronary%20artery%20disease%20at%20chromosome%209p21%20in%20the%20multi-ethnic%20ADVANCE%20Study&author=TL%20Assimes&author=JW%20Knowles&author=A%20Basu&author=C%20Iribarren&author=A%20Southwick&volume=17&publication_year=2008&pages=2320-2328&pmid=18443000&doi=10.1093/hmg/ddn132&)

8. Chen Z, Qian Q, Ma G, Wang J, Zhang X, Feng Y, Shen C, Yao Y. A common variant on chromosome 9p21 affects the risk of early-onset coronary artery disease. Mol Biol Rep. 2009;36:889–893. doi: 10.1007/s11033-008-9259-7.  [DOI](https://doi.org/10.1007/s11033-008-9259-7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18459066/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Biol%20Rep&title=A%20common%20variant%20on%20chromosome%209p21%20affects%20the%20risk%20of%20early-onset%20coronary%20artery%20disease&author=Z%20Chen&author=Q%20Qian&author=G%20Ma&author=J%20Wang&author=X%20Zhang&volume=36&publication_year=2009&pages=889-893&pmid=18459066&doi=10.1007/s11033-008-9259-7&)

9. Helgadottir A, Thorleifsson G, Magnusson KP, Gretarsdottir S, Steinthorsdottir V, Manolescu A, Jones GT, Rinkel GJ, Blankensteijn JD, Ronkainen A, Jaaskelainen JE, Kyo Y, Lenk GM, Sakalihasan N, Kostulas K, Gottsater A, Flex A, Stefansson H, Hansen T, Andersen G, Weinsheimer S, Borch-Johnsen K, Jorgensen T, Shah SH, Quyyumi AA, Granger CB, Reilly MP, Austin H, Levey AI, Vaccarino V, Palsdottir E, Walters GB, Jonsdottir T, Snorradottir S, Magnusdottir D, Gudmundsson G, Ferrell RE, Sveinbjornsdottir S, Hernesniemi J, Niemela M, Limet R, Andersen K, Sigurdsson G, Benediktsson R, Verhoeven EL, Teijink JA, Grobbee DE, Rader DJ, Collier DA, Pedersen O, Pola R, Hillert J, Lindblad B, Valdimarsson EM, Magnadottir HB, Wijmenga C, Tromp G, Baas AF, Ruigrok YM, van Rij AM, Kuivaniemi H, Powell JT, Matthiasson SE, Gulcher JR, Thorgeirsson G, Kong A, Thorsteinsdottir U, Stefansson K. The same sequence variant on 9p21 associates with myocardial infarction, abdominal aortic aneurysm and intracranial aneurysm. Nat Genet. 2008;40:217–224. doi: 10.1038/ng.72.  [DOI](https://doi.org/10.1038/ng.72) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18176561/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=The%20same%20sequence%20variant%20on%209p21%20associates%20with%20myocardial%20infarction,%20abdominal%20aortic%20aneurysm%20and%20intracranial%20aneurysm&author=A%20Helgadottir&author=G%20Thorleifsson&author=KP%20Magnusson&author=S%20Gretarsdottir&author=V%20Steinthorsdottir&volume=40&publication_year=2008&pages=217-224&pmid=18176561&doi=10.1038/ng.72&)

10. Hinohara K, Nakajima T, Takahashi M, Hohda S, Sasaoka T, Nakahara K, Chida K, Sawabe M, Arimura T, Sato A, Lee BS, Ban JM, Yasunami M, Park JE, Izumi T, Kimura A. Replication of the association between a chromosome 9p21 polymorphism and coronary artery disease in Japanese and Korean populations. J Hum Genet. 2008;53:357–359. doi: 10.1007/s10038-008-0248-4.  [DOI](https://doi.org/10.1007/s10038-008-0248-4) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18264662/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Hum%20Genet&title=Replication%20of%20the%20association%20between%20a%20chromosome%209p21%20polymorphism%20and%20coronary%20artery%20disease%20in%20Japanese%20and%20Korean%20populations&author=K%20Hinohara&author=T%20Nakajima&author=M%20Takahashi&author=S%20Hohda&author=T%20Sasaoka&volume=53&publication_year=2008&pages=357-359&pmid=18264662&doi=10.1007/s10038-008-0248-4&)

11. Matarin M, Brown WM, Singleton A, Hardy JA, Meschia JF ISGS investigators. Whole genome analyses suggest ischemic stroke and heart disease share an association with polymorphisms on chromosome 9p21. Stroke. 2008;39:1586–1589. doi: 10.1161/STROKEAHA.107.502963.  [DOI](https://doi.org/10.1161/STROKEAHA.107.502963) | [PMC free article](/articles/PMC3932672/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18340101/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Stroke&title=Whole%20genome%20analyses%20suggest%20ischemic%20stroke%20and%20heart%20disease%20share%20an%20association%20with%20polymorphisms%20on%20chromosome%209p21&author=M%20Matarin&author=WM%20Brown&author=A%20Singleton&author=JA%20Hardy&author=JF%20Meschia&volume=39&publication_year=2008&pages=1586-1589&pmid=18340101&doi=10.1161/STROKEAHA.107.502963&)

12. Schunkert H, Gotz A, Braund P, McGinnis R, Tregouet DA, Mangino M, Linsel-Nitschke P, Cambien F, Hengstenberg C, Stark K, Blankenberg S, Tiret L, Ducimetiere P, Keniry A, Ghori MJ, Schreiber S, El Mokhtari NE, Hall AS, Dixon RJ, Goodall AH, Liptau H, Pollard H, Schwarz DF, Hothorn LA, Wichmann HE, Konig IR, Fischer M, Meisinger C, Ouwehand W, Deloukas P, Thompson JR, Erdmann J, Ziegler A, Samani NJ Cardiogenics Consortium. Repeated replication and a prospective meta-analysis of the association between chromosome 9p21.3 and coronary artery disease. Circulation. 2008;117:1675–1684. doi: 10.1161/CIRCULATIONAHA.107.730614.  [DOI](https://doi.org/10.1161/CIRCULATIONAHA.107.730614) | [PMC free article](/articles/PMC2689930/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18362232/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circulation&title=Repeated%20replication%20and%20a%20prospective%20meta-analysis%20of%20the%20association%20between%20chromosome%209p21.3%20and%20coronary%20artery%20disease&author=H%20Schunkert&author=A%20Gotz&author=P%20Braund&author=R%20McGinnis&author=DA%20Tregouet&volume=117&publication_year=2008&pages=1675-1684&pmid=18362232&doi=10.1161/CIRCULATIONAHA.107.730614&)

13. Shen GQ, Li L, Rao S, Abdullah KG, Ban JM, Lee BS, Park JE, Wang QK. Four SNPs on chromosome 9p21 in a South Korean population implicate a genetic locus that confers high cross-race risk for development of coronary artery disease. Arterioscler Thromb Vasc Biol. 2008;28:360–365. doi: 10.1161/ATVBAHA.107.157248.  [DOI](https://doi.org/10.1161/ATVBAHA.107.157248) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18048766/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arterioscler%20Thromb%20Vasc%20Biol&title=Four%20SNPs%20on%20chromosome%209p21%20in%20a%20South%20Korean%20population%20implicate%20a%20genetic%20locus%20that%20confers%20high%20cross-race%20risk%20for%20development%20of%20coronary%20artery%20disease&author=GQ%20Shen&author=L%20Li&author=S%20Rao&author=KG%20Abdullah&author=JM%20Ban&volume=28&publication_year=2008&pages=360-365&pmid=18048766&doi=10.1161/ATVBAHA.107.157248&)

14. Shen GQ, Rao S, Martinelli N, Li L, Olivieri O, Corrocher R, Abdullah KG, Hazen SL, Smith J, Barnard J, Plow EF, Girelli D, Wang QK. Association between four SNPs on chromosome 9p21 and myocardial infarction is replicated in an Italian population. J Hum Genet. 2008;53:144–150. doi: 10.1007/s10038-007-0230-6.  [DOI](https://doi.org/10.1007/s10038-007-0230-6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18066490/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Hum%20Genet&title=Association%20between%20four%20SNPs%20on%20chromosome%209p21%20and%20myocardial%20infarction%20is%20replicated%20in%20an%20Italian%20population&author=GQ%20Shen&author=S%20Rao&author=N%20Martinelli&author=L%20Li&author=O%20Olivieri&volume=53&publication_year=2008&pages=144-150&pmid=18066490&doi=10.1007/s10038-007-0230-6&)

15. Talmud PJ, Cooper JA, Palmen J, Lovering R, Drenos F, Hingorani AD, Humphries SE. Chromosome 9p21.3 coronary heart disease locus genotype and prospective risk of CHD in healthy middle-aged men. Clin Chem. 2008;54:467–474. doi: 10.1373/clinchem.2007.095489.  [DOI](https://doi.org/10.1373/clinchem.2007.095489) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18250146/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Chem&title=Chromosome%209p21.3%20coronary%20heart%20disease%20locus%20genotype%20and%20prospective%20risk%20of%20CHD%20in%20healthy%20middle-aged%20men&author=PJ%20Talmud&author=JA%20Cooper&author=J%20Palmen&author=R%20Lovering&author=F%20Drenos&volume=54&publication_year=2008&pages=467-474&pmid=18250146&doi=10.1373/clinchem.2007.095489&)

16. Ye S, Willeit J, Kronenberg F, Xu Q, Kiechl S. Association of genetic variation on chromosome 9p21 with susceptibility and progression of atherosclerosis: a population-based, prospective study. J Am Coll Cardiol. 2008;52:378–384. doi: 10.1016/j.jacc.2007.11.087.  [DOI](https://doi.org/10.1016/j.jacc.2007.11.087) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18652946/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Am%20Coll%20Cardiol&title=Association%20of%20genetic%20variation%20on%20chromosome%209p21%20with%20susceptibility%20and%20progression%20of%20atherosclerosis:%20a%20population-based,%20prospective%20study&author=S%20Ye&author=J%20Willeit&author=F%20Kronenberg&author=Q%20Xu&author=S%20Kiechl&volume=52&publication_year=2008&pages=378-384&pmid=18652946&doi=10.1016/j.jacc.2007.11.087&)

17. O’Donnell CJ, Cupples LA, D’Agostino RB, Fox CS, Hoffmann U, Hwang SJ, Ingellson E, Liu C, Murabito JM, Polak JF, Wolf PA, Demissie S. Genome-wide association study for subclinical atherosclerosis in major arterial territories in the NHLBI’s Framingham Heart Study. BMC Med Genet. 2007;8 (Suppl 1):S4. doi: 10.1186/1471-2350-8-S1-S4.  [DOI](https://doi.org/10.1186/1471-2350-8-S1-S4) | [PMC free article](/articles/PMC1995605/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17903303/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Med%20Genet&title=Genome-wide%20association%20study%20for%20subclinical%20atherosclerosis%20in%20major%20arterial%20territories%20in%20the%20NHLBI%E2%80%99s%20Framingham%20Heart%20Study&author=CJ%20O%E2%80%99Donnell&author=LA%20Cupples&author=RB%20D%E2%80%99Agostino&author=CS%20Fox&author=U%20Hoffmann&volume=8&issue=Suppl%201&publication_year=2007&pages=S4&pmid=17903303&doi=10.1186/1471-2350-8-S1-S4&)

18. Gschwendtner A, Bevan S, Cole JW, Plourde A, Matarin M, Ross-Adams H, Meitinger T, Wichmann E, Mitchell BD, Furie K, Slowik A, Rich SS, Syme PD, MacLeod MJ, Meschia JF, Rosand J, Kittner SJ, Markus HS, Müller-Myhsok B, Dichgans M International Stroke Genetics Consortium. Sequence variants on chromosome 9p21.3 confer risk for atherosclerotic stroke. Ann Neurol. 2009;65:531–539. doi: 10.1002/ana.21590.  [DOI](https://doi.org/10.1002/ana.21590) | [PMC free article](/articles/PMC2702695/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19475673/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Neurol&title=Sequence%20variants%20on%20chromosome%209p21.3%20confer%20risk%20for%20atherosclerotic%20stroke&author=A%20Gschwendtner&author=S%20Bevan&author=JW%20Cole&author=A%20Plourde&author=M%20Matarin&volume=65&publication_year=2009&pages=531-539&pmid=19475673&doi=10.1002/ana.21590&)

19. Samani NJ, Raitakari OT, Sipila K, Tobin MD, Schunkert H, Juonala M, Braund PS, Erdmann J, Viikari J, Moilanen L, Taittonen L, Jula A, Jokinen E, Laitinen T, Hutri-Kahonen N, Nieminen MS, Kesaniemi YA, Hall AS, Hulkkonen J, Kahonen M, Lehtimaki T. Coronary artery disease-associated locus on chromosome 9p21 and early markers of atherosclerosis. Arterioscler Thromb Vasc Biol. 2008;28:1679–1683. doi: 10.1161/ATVBAHA.108.170332.  [DOI](https://doi.org/10.1161/ATVBAHA.108.170332) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18599798/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arterioscler%20Thromb%20Vasc%20Biol&title=Coronary%20artery%20disease-associated%20locus%20on%20chromosome%209p21%20and%20early%20markers%20of%20atherosclerosis&author=NJ%20Samani&author=OT%20Raitakari&author=K%20Sipila&author=MD%20Tobin&author=H%20Schunkert&volume=28&publication_year=2008&pages=1679-1683&pmid=18599798&doi=10.1161/ATVBAHA.108.170332&)

20. Cunnington MS, Mayosi BM, Hall DH, Avery PJ, Farrall M, Vickers MA, Watkins H, Keavney B. Novel genetic variants linked to coronary artery disease by genome-wide association are not associated with carotid artery intima-media thickness or intermediate risk phenotypes. Atherosclerosis. 2009;203:41–44. doi: 10.1016/j.atherosclerosis.2008.06.025.  [DOI](https://doi.org/10.1016/j.atherosclerosis.2008.06.025) | [PMC free article](/articles/PMC2654912/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18675980/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Atherosclerosis&title=Novel%20genetic%20variants%20linked%20to%20coronary%20artery%20disease%20by%20genome-wide%20association%20are%20not%20associated%20with%20carotid%20artery%20intima-media%20thickness%20or%20intermediate%20risk%20phenotypes&author=MS%20Cunnington&author=BM%20Mayosi&author=DH%20Hall&author=PJ%20Avery&author=M%20Farrall&volume=203&publication_year=2009&pages=41-44&pmid=18675980&doi=10.1016/j.atherosclerosis.2008.06.025&)

21. del Zoppo GJ. The role of platelets in ischemic stroke. Neurology. 1998;51:S9–14. doi: 10.1212/wnl.51.3_suppl_3.s9.  [DOI](https://doi.org/10.1212/wnl.51.3_suppl_3.s9) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9744824/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neurology&title=The%20role%20of%20platelets%20in%20ischemic%20stroke&author=GJ%20del%20Zoppo&volume=51&publication_year=1998&pages=S9-14&pmid=9744824&doi=10.1212/wnl.51.3_suppl_3.s9&)

22. Antiplatelet Trialists’ Collaboration. Collaborative overview of randomised trials of antiplatelet therapy--I: Prevention of death, myocardial infarction, and stroke by prolonged antiplatelet therapy in various categories of patients. BMJ. 1994;308:81–106.  [PMC free article](/articles/PMC2539220/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8298418/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMJ&title=Collaborative%20overview%20of%20randomised%20trials%20of%20antiplatelet%20therapy--I:%20Prevention%20of%20death,%20myocardial%20infarction,%20and%20stroke%20by%20prolonged%20antiplatelet%20therapy%20in%20various%20categories%20of%20patients&volume=308&publication_year=1994&pages=81-106&pmid=8298418&)

23. CAPRIE Steering Committee. A randomised, blinded, trial of clopidogrel versus aspirin in patients at risk of ischaemic events (CAPRIE) Lancet. 1996;348:1329–1339. doi: 10.1016/s0140-6736(96)09457-3.  [DOI](https://doi.org/10.1016/s0140-6736(96)09457-3) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8918275/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=A%20randomised,%20blinded,%20trial%20of%20clopidogrel%20versus%20aspirin%20in%20patients%20at%20risk%20of%20ischaemic%20events%20(CAPRIE)&volume=348&publication_year=1996&pages=1329-1339&pmid=8918275&doi=10.1016/s0140-6736(96)09457-3&)

24. Mitchell BD, McArdle PF, Shen H, Rampersaud E, Pollin TI, Bielak LF, Jaquish C, Douglas JA, Roy-Gagnon MH, Sack P, Naglieri R, Hines S, Horenstein RB, Chang YP, Post W, Ryan KA, Brereton NH, Pakyz RE, Sorkin J, Damcott CM, O’Connell JR, Mangano C, Corretti M, Vogel R, Herzog W, Weir MR, Peyser PA, Shuldiner AR. The genetic response to short-term interventions affecting cardiovascular function: rationale and design of the Heredity and Phenotype Intervention (HAPI) Heart Study. Am Heart J. 2008;155:823–828. doi: 10.1016/j.ahj.2008.01.019.  [DOI](https://doi.org/10.1016/j.ahj.2008.01.019) | [PMC free article](/articles/PMC2443415/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18440328/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20Heart%20J&title=The%20genetic%20response%20to%20short-term%20interventions%20affecting%20cardiovascular%20function:%20rationale%20and%20design%20of%20the%20Heredity%20and%20Phenotype%20Intervention%20(HAPI)%20Heart%20Study&author=BD%20Mitchell&author=PF%20McArdle&author=H%20Shen&author=E%20Rampersaud&author=TI%20Pollin&volume=155&publication_year=2008&pages=823-828&pmid=18440328&doi=10.1016/j.ahj.2008.01.019&)

25. Post W, Bielak LF, Ryan KA, Cheng YC, Shen H, Rumberger JA, Sheedy PF, 2nd, Shuldiner AR, Peyser PA, Mitchell BD. Determinants of coronary artery and aortic calcification in the Old Order Amish. Circulation. 2007;115:717–724. doi: 10.1161/CIRCULATIONAHA.106.637512.  [DOI](https://doi.org/10.1161/CIRCULATIONAHA.106.637512) | [PMC free article](/articles/PMC4039563/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17261661/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circulation&title=Determinants%20of%20coronary%20artery%20and%20aortic%20calcification%20in%20the%20Old%20Order%20Amish&author=W%20Post&author=LF%20Bielak&author=KA%20Ryan&author=YC%20Cheng&author=H%20Shen&volume=115&publication_year=2007&pages=717-724&pmid=17261661&doi=10.1161/CIRCULATIONAHA.106.637512&)

26. O’Donnell CJ, Larson MG, Feng D, Sutherland PA, Lindpaintner K, Myers RH, D’Agostino RA, Levy D, Tofler GH Framingham Heart Study. Genetic and environmental contributions to platelet aggregation: the Framingham Heart study. Circulation. 2001;103:3051–3056. doi: 10.1161/01.cir.103.25.3051.  [DOI](https://doi.org/10.1161/01.cir.103.25.3051) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11425767/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circulation&title=Genetic%20and%20environmental%20contributions%20to%20platelet%20aggregation:%20the%20Framingham%20Heart%20study&author=CJ%20O%E2%80%99Donnell&author=MG%20Larson&author=D%20Feng&author=PA%20Sutherland&author=K%20Lindpaintner&volume=103&publication_year=2001&pages=3051-3056&pmid=11425767&doi=10.1161/01.cir.103.25.3051&)

27. Becker DM, Segal J, Vaidya D, Yanek LR, Herrera-Galeano JE, Bray PF, Moy TF, Becker LC, Faraday N. Sex differences in platelet reactivity and response to low-dose aspirin therapy. JAMA. 2006;295:1420–1427. doi: 10.1001/jama.295.12.1420.  [DOI](https://doi.org/10.1001/jama.295.12.1420) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16551714/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA&title=Sex%20differences%20in%20platelet%20reactivity%20and%20response%20to%20low-dose%20aspirin%20therapy&author=DM%20Becker&author=J%20Segal&author=D%20Vaidya&author=LR%20Yanek&author=JE%20Herrera-Galeano&volume=295&publication_year=2006&pages=1420-1427&pmid=16551714&doi=10.1001/jama.295.12.1420&)

28. Boerwinkle E, Chakraborty R, Sing CF. The use of measured genotype information in the analysis of quantitative phenotypes in man. I. Models and analytical methods. Ann Hum Genet. 1986;50(Pt 2):181–194. doi: 10.1111/j.1469-1809.1986.tb01037.x.  [DOI](https://doi.org/10.1111/j.1469-1809.1986.tb01037.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/3435047/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Hum%20Genet&title=The%20use%20of%20measured%20genotype%20information%20in%20the%20analysis%20of%20quantitative%20phenotypes%20in%20man.%20I.%20Models%20and%20analytical%20methods&author=E%20Boerwinkle&author=R%20Chakraborty&author=CF%20Sing&volume=50&issue=Pt%202&publication_year=1986&pages=181-194&pmid=3435047&doi=10.1111/j.1469-1809.1986.tb01037.x&)

29. O’Connell JR. Optimizing measured genotype genome-wide association analysis for quantitative traits in pedigrees. 58th Annual Meeting of the American Society of Human Genetics; 2008. abstract 2409. Abstract.  [Google Scholar](https://scholar.google.com/scholar_lookup?O%E2%80%99Connell%20JR.%20Optimizing%20measured%20genotype%20genome-wide%20association%20analysis%20for%20quantitative%20traits%20in%20pedigrees.%2058th%20Annual%20Meeting%20of%20the%20American%20Society%20of%20Human%20Genetics;%202008.%20abstract%202409.%20Abstract.)

30. Price AL, Patterson NJ, Plenge RM, Weinblatt ME, Shadick NA, Reich D. Principal components analysis corrects for stratification in genome-wide association studies. Nat Gen. 2006;38:904–909. doi: 10.1038/ng1847.  [DOI](https://doi.org/10.1038/ng1847) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16862161/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Gen&title=Principal%20components%20analysis%20corrects%20for%20stratification%20in%20genome-wide%20association%20studies&author=AL%20Price&author=NJ%20Patterson&author=RM%20Plenge&author=ME%20Weinblatt&author=NA%20Shadick&volume=38&publication_year=2006&pages=904-909&pmid=16862161&doi=10.1038/ng1847&)

31. Chen MH, Yang Q. GWAF: an R package for genome-wide association analyses with family data. Bioinformatics. 2010;26:580–581. doi: 10.1093/bioinformatics/btp710.  [DOI](https://doi.org/10.1093/bioinformatics/btp710) | [PMC free article](/articles/PMC2852219/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20040588/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=GWAF:%20an%20R%20package%20for%20genome-wide%20association%20analyses%20with%20family%20data&author=MH%20Chen&author=Q%20Yang&volume=26&publication_year=2010&pages=580-581&pmid=20040588&doi=10.1093/bioinformatics/btp710&)

32. Schaefer AS, Richter GM, Groessner-Schreiber B, Noack B, Nothnagel M, El Mokhtari NE, Loos BG, Jepsen S, Schreiber S. Identification of a shared genetic susceptibility locus for coronary heart disease and periodontitis. PLoS Genet. 2009;5:e1000378. doi: 10.1371/journal.pgen.1000378.  [DOI](https://doi.org/10.1371/journal.pgen.1000378) | [PMC free article](/articles/PMC2632758/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19214202/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet&title=Identification%20of%20a%20shared%20genetic%20susceptibility%20locus%20for%20coronary%20heart%20disease%20and%20periodontitis&author=AS%20Schaefer&author=GM%20Richter&author=B%20Groessner-Schreiber&author=B%20Noack&author=M%20Nothnagel&volume=5&publication_year=2009&pages=e1000378&pmid=19214202&doi=10.1371/journal.pgen.1000378&)

33. Bishop DT, Demenais F, Iles MM, Harland M, Taylor JC, Corda E, Randerson-Moor J, Aitken JF, Avril MF, Azizi E, Bakker B, Bianchi-Scarrà G, Bressac-de Paillerets B, Calista D, Cannon-Albright LA, Chin-A-Woeng T, Debniak T, Galore-Haskel G, Ghiorzo P, Gut I, Hansson J, Hocevar M, Höiom V, Hopper JL, Ingvar C, Kanetsky PA, Kefford RF, Landi MT, Lang J, Lubiñski J, Mackie R, Malvehy J, Mann GJ, Martin NG, Montgomery GW, van Nieuwpoort FA, Novakovic S, Olsson H, Puig S, Weiss M, van Workum W, Zelenika D, Brown KM, Goldstein AM, Gillanders EM, Boland A, Galan P, Elder DE, Gruis NA, Hayward NK, Lathrop GM, Barrett JH, Bishop JA. Genome-wide association identifies three loci associated with melanoma risk. Nat Genet. 2009;41:920–925. doi: 10.1038/ng.411.  [DOI](https://doi.org/10.1038/ng.411) | [PMC free article](/articles/PMC2741419/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19578364/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Genome-wide%20association%20identifies%20three%20loci%20associated%20with%20melanoma%20risk&author=DT%20Bishop&author=F%20Demenais&author=MM%20Iles&author=M%20Harland&author=JC%20Taylor&volume=41&publication_year=2009&pages=920-925&pmid=19578364&doi=10.1038/ng.411&)

34. Diabetes Genetics Initiative of Broad Institute of Harvard and MIT, Lund University, and Novartis Institutes of BioMedical Research. Saxena R, Voight BF, Lyssenko V, Burtt NP, de Bakker PI, Chen H, Roix JJ, Kathiresan S, Hirschhorn JN, Daly MJ, Hughes TE, Groop L, Altshuler D, Almgren P, Florez JC, Meyer J, Ardlie K, Bengtsson Bostrom K, Isomaa B, Lettre G, Lindblad U, Lyon HN, Melander O, Newton-Cheh C, Nilsson P, Orho-Melander M, Rastam L, Speliotes EK, Taskinen MR, Tuomi T, Guiducci C, Berglund A, Carlson J, Gianniny L, Hackett R, Hall L, Holmkvist J, Laurila E, Sjogren M, Sterner M, Surti A, Svensson M, Svensson M, Tewhey R, Blumenstiel B, Parkin M, Defelice M, Barry R, Brodeur W, Camarata J, Chia N, Fava M, Gibbons J, Handsaker B, Healy C, Nguyen K, Gates C, Sougnez C, Gage D, Nizzari M, Gabriel SB, Chirn GW, Ma Q, Parikh H, Richardson D, Ricke D, Purcell S. Genome-wide association analysis identifies loci for type 2 diabetes and triglyceride levels. Science. 2007;316:1331–1336. doi: 10.1126/science.1142358.  [DOI](https://doi.org/10.1126/science.1142358) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17463246/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=Genome-wide%20association%20analysis%20identifies%20loci%20for%20type%202%20diabetes%20and%20triglyceride%20levels&author=R%20Saxena&author=BF%20Voight&author=V%20Lyssenko&author=NP%20Burtt&author=PI%20de%20Bakker&volume=316&publication_year=2007&pages=1331-1336&pmid=17463246&doi=10.1126/science.1142358&)

35. Zeggini E, Weedon MN, Lindgren CM, Frayling TM, Elliott KS, Lango H, Timpson NJ, Perry JR, Rayner NW, Freathy RM, Barrett JC, Shields B, Morris AP, Ellard S, Groves CJ, Harries LW, Marchini JL, Owen KR, Knight B, Cardon LR, Walker M, Hitman GA, Morris AD, Doney AS, McCarthy MI, Hattersley AT Wellcome Trust Case Control Consortium (WTCCC) Replication of genome-wide association signals in UK samples reveals risk loci for type 2 diabetes. Science. 2007;316:1336–1341. doi: 10.1126/science.1142364.  [DOI](https://doi.org/10.1126/science.1142364) | [PMC free article](/articles/PMC3772310/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17463249/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=Replication%20of%20genome-wide%20association%20signals%20in%20UK%20samples%20reveals%20risk%20loci%20for%20type%202%20diabetes&author=E%20Zeggini&author=MN%20Weedon&author=CM%20Lindgren&author=TM%20Frayling&author=KS%20Elliott&volume=316&publication_year=2007&pages=1336-1341&pmid=17463249&doi=10.1126/science.1142364&)

36. Scott LJ, Mohlke KL, Bonnycastle LL, Willer CJ, Li Y, Duren WL, Erdos MR, Stringham HM, Chines PS, Jackson AU, Prokunina-Olsson L, Ding CJ, Swift AJ, Narisu N, Hu T, Pruim R, Xiao R, Li XY, Conneely KN, Riebow NL, Sprau AG, Tong M, White PP, Hetrick KN, Barnhart MW, Bark CW, Goldstein JL, Watkins L, Xiang F, Saramies J, Buchanan TA, Watanabe RM, Valle TT, Kinnunen L, Abecasis GR, Pugh EW, Doheny KF, Bergman RN, Tuomilehto J, Collins FS, Boehnke M. A genome-wide association study of type 2 diabetes in Finns detects multiple susceptibility variants. Science. 2007;316:1341–1345. doi: 10.1126/science.1142382.  [DOI](https://doi.org/10.1126/science.1142382) | [PMC free article](/articles/PMC3214617/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17463248/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=A%20genome-wide%20association%20study%20of%20type%202%20diabetes%20in%20Finns%20detects%20multiple%20susceptibility%20variants&author=LJ%20Scott&author=KL%20Mohlke&author=LL%20Bonnycastle&author=CJ%20Willer&author=Y%20Li&volume=316&publication_year=2007&pages=1341-1345&pmid=17463248&doi=10.1126/science.1142382&)

37. Pasmant E, Laurendeau I, Héron D, Vidaud M, Vidaud D, Bièche I. Characterization of a germ-line deletion, including the entire INK4/ARF locus, in a melanoma-neural system tumor family: identification of ANRIL, an antisense noncoding RNA whose expression coclusters with ARF. Cancer Res. 2007;67:3963–3969. doi: 10.1158/0008-5472.CAN-06-2004.  [DOI](https://doi.org/10.1158/0008-5472.CAN-06-2004) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17440112/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=Characterization%20of%20a%20germ-line%20deletion,%20including%20the%20entire%20INK4/ARF%20locus,%20in%20a%20melanoma-neural%20system%20tumor%20family:%20identification%20of%20ANRIL,%20an%20antisense%20noncoding%20RNA%20whose%20expression%20coclusters%20with%20ARF&author=E%20Pasmant&author=I%20Laurendeau&author=D%20H%C3%A9ron&author=M%20Vidaud&author=D%20Vidaud&volume=67&publication_year=2007&pages=3963-3969&pmid=17440112&doi=10.1158/0008-5472.CAN-06-2004&)

38. Amaral PP, Dinger ME, Mercer TR, Mattick JS. The eukaryotic genome as an RNA machine. Science. 2008;319:1787–1789. doi: 10.1126/science.1155472.  [DOI](https://doi.org/10.1126/science.1155472) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18369136/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=The%20eukaryotic%20genome%20as%20an%20RNA%20machine&author=PP%20Amaral&author=ME%20Dinger&author=TR%20Mercer&author=JS%20Mattick&volume=319&publication_year=2008&pages=1787-1789&pmid=18369136&doi=10.1126/science.1155472&)

39. Ponting CP, Oliver PL, Reik W. Evolution and functions of long noncoding RNAs. Cell. 2009;136:629–641. doi: 10.1016/j.cell.2009.02.006.  [DOI](https://doi.org/10.1016/j.cell.2009.02.006) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19239885/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell&title=Evolution%20and%20functions%20of%20long%20noncoding%20RNAs&author=CP%20Ponting&author=PL%20Oliver&author=W%20Reik&volume=136&publication_year=2009&pages=629-641&pmid=19239885&doi=10.1016/j.cell.2009.02.006&)

40. Broadbent HM, Peden JF, Lorkowski S, Goel A, Ongen H, Green F, Clarke R, Collins R, Franzosi MG, Tognoni G, Seedorf U, Rust S, Eriksson P, Hamsten A, Farrall M, Watkins H PROCARDIS consortium. Susceptibility to coronary artery disease and diabetes is encoded by distinct, tightly linked SNPs in the ANRIL locus on chromosome 9p. Hum Mol Genet. 2008;17:806–814. doi: 10.1093/hmg/ddm352.  [DOI](https://doi.org/10.1093/hmg/ddm352) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18048406/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mol%20Genet&title=Susceptibility%20to%20coronary%20artery%20disease%20and%20diabetes%20is%20encoded%20by%20distinct,%20tightly%20linked%20SNPs%20in%20the%20ANRIL%20locus%20on%20chromosome%209p&author=HM%20Broadbent&author=JF%20Peden&author=S%20Lorkowski&author=A%20Goel&author=H%20Ongen&volume=17&publication_year=2008&pages=806-814&pmid=18048406&doi=10.1093/hmg/ddm352&)

41. Jarinova O, Stewart AF, Roberts R, Wells G, Lau P, Naing T, Buerki C, McLean BW, Cook RC, Parker JS, McPherson R. Functional analysis of the chromosome 9p21.3 coronary artery disease risk locus. Arterioscler Thromb Vasc Biol. 2009;29:1671–1677. doi: 10.1161/ATVBAHA.109.189522.  [DOI](https://doi.org/10.1161/ATVBAHA.109.189522) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19592466/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arterioscler%20Thromb%20Vasc%20Biol&title=Functional%20analysis%20of%20the%20chromosome%209p21.3%20coronary%20artery%20disease%20risk%20locus&author=O%20Jarinova&author=AF%20Stewart&author=R%20Roberts&author=G%20Wells&author=P%20Lau&volume=29&publication_year=2009&pages=1671-1677&pmid=19592466&doi=10.1161/ATVBAHA.109.189522&)

42. Liu Y, Sanoff HK, Cho H, Burd CE, Torrice C, Mohlke KL, Ibrahim JG, Thomas NE, Sharpless NE. INK4/ARF transcript expression is associated with chromosome 9p21 variants linked to atherosclerosis. PLoS ONE. 2009;4:e5027. doi: 10.1371/journal.pone.0005027.  [DOI](https://doi.org/10.1371/journal.pone.0005027) | [PMC free article](/articles/PMC2660422/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19343170/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&title=INK4/ARF%20transcript%20expression%20is%20associated%20with%20chromosome%209p21%20variants%20linked%20to%20atherosclerosis&author=Y%20Liu&author=HK%20Sanoff&author=H%20Cho&author=CE%20Burd&author=C%20Torrice&volume=4&publication_year=2009&pages=e5027&pmid=19343170&doi=10.1371/journal.pone.0005027&)

43. Cunnington MS, Santibanez Koref M, Mayosi BM, Burn J, Keavney B. Chromosome 9p21 SNPs associated with multiple disease phenotypes correlate with ANRIL expression. PLoS Genet. 2010;6:e1000899. doi: 10.1371/journal.pgen.1000899.  [DOI](https://doi.org/10.1371/journal.pgen.1000899) | [PMC free article](/articles/PMC2851566/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20386740/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet&title=Chromosome%209p21%20SNPs%20associated%20with%20multiple%20disease%20phenotypes%20correlate%20with%20ANRIL%20expression&author=MS%20Cunnington&author=M%20Santibanez%20Koref&author=BM%20Mayosi&author=J%20Burn&author=B%20Keavney&volume=6&publication_year=2010&pages=e1000899&pmid=20386740&doi=10.1371/journal.pgen.1000899&)
