# Targeted pharmacogenetic analysis of antipsychotic response in the CATIE study

## Metadata
**Authors:** Qian Liu, Maidar Jamba, Calvin Patrick, III, Saranya Padmanabhan, Mark D Brennan
**Journal:** Pharmacogenomics
**Date:** 2012 Aug
**DOI:** [10.2217/pgs.12.105](https://doi.org/10.2217/pgs.12.105)
**PMID:** 22920393
**PMCID:** PMC3518380
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3518380/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC3518380/pdf/nihms422486.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC3518380/pdf/nihms422486.pdf)

## Abstract

**Aim:** 
This study evaluated the impact of 6789 SNPs on treatment response to antipsychotics in Caucasian patients from the CATIE study.

**Materials & methods:** 
An Illumina (CA, USA) BeadChip was designed that targeted genes potentially impacting disease risk, disease presentation or antipsychotic response. SNPs tagged regions of linkage disequilibrium or functional variants not detectable using previous genotypes for CATIE. Change in Positive and Negative Syndrome scale total score was modeled using a mixed model repeated measures method that assumed a 30-day lag period. Genetic association analysis was performed using linear regression.

**Results:** 
Association analysis identified 20 SNPs with p-values of ≤5 × 10−4. Many of these are in genes previously implicated in schizophrenia and other neuropsychiatric diseases.

**Conclusion:** 
The targeted approach identified SNPs possibly influencing response to antipsychotic drugs in Caucasian patients suffering from schizophrenia. The findings support a biological link between disease risk and presentation and antipsychotic response.

Keywords: neuropsychiatric genes, olanzapine, perphenazine, quetiapine, risperidone, ziprasidone

### Aim

This study evaluated the impact of 6789 SNPs on treatment response to antipsychotics in Caucasian patients from the CATIE study.

### Materials & methods

An Illumina (CA, USA) BeadChip was designed that targeted genes potentially impacting disease risk, disease presentation or antipsychotic response. SNPs tagged regions of linkage disequilibrium or functional variants not detectable using previous genotypes for CATIE. Change in Positive and Negative Syndrome scale total score was modeled using a mixed model repeated measures method that assumed a 30-day lag period. Genetic association analysis was performed using linear regression.

### Results

Association analysis identified 20 SNPs with p-values of ≤5 × 10^−4^. Many of these are in genes previously implicated in schizophrenia and other neuropsychiatric diseases.

### Conclusion

The targeted approach identified SNPs possibly influencing response to antipsychotic drugs in Caucasian patients suffering from schizophrenia. The findings support a biological link between disease risk and presentation and antipsychotic response.

## Materials & methods

### Enrichment for genes impacting psychopathology & psychopharmacology

An initial list of candidate loci was generated based on genetic association analyses using genotypes and phenotypes provided by the Center for Collaborative Genomic Studies on Mental Disorders (CCGSMD; MO, USA). Phenotypes for the CATIE study included baseline psychopathology and drug response variables described in detail by others [[1](#R1), [2](#R2), [15](#R15)]. Case–control status and genotypes for the GAIN schizophrenia (version phs000021.v2.p1) and bipolar disorder (version phs000017.v3.p1) sample sets were obtained from the Database of Genotypes and Phenotypes (dbGaP): National Center for Biotechnology Information, National Library of Medicine (MD, USA) [[14](#R14), [101](#R60)].

Initial screens using the CATIE sample involved genetic association of quantitative traits by linear regression using PLINK (version 1.04) [[16](#R16), [102](#R61)]. The pharmacogenomic phenotypes used for the screen were change in PANSS (percentage relative to baseline) at last observation carried forward [[17](#R17)] and time to ‘all-cause’ discontinuation, the primary end point of the CATIE study [[2](#R2)], for each of the five antipsychotics included in the trial. Genes having one or more SNPs within the transcribed region with p-values of ≤10^−2^ or associated intergenic SNPs with p-values of ≤10^−3^ were included in the initial pharmacogenomic list. Phenotypes relating to baseline psychopathology were derived from PANSS scores for CATIE subjects prior to initiation of antipsychotic treatment. Genes having SNPs within the transcribed region associating with PANSS total score (PANSS-T), or PANSS Positive, Negative or General subscale scores (p ≤ 10^−2^) or with any of the 30 individual PANSS items (p ≤ 10^−3^) were included.

Additional candidate loci were selected by case–control comparisons employing the GAIN consortium schizophrenia and bipolar disorder samples using an additive genetic model in PLINK. Genes with one or more SNPs within the transcribed region with p-values of p ≤ 10^−3^ were included on the initial list.

To further focus the analysis, the final candidate gene list included only those loci with two or more SNPs meeting the above criteria. A total of approximately 2700 genes passed this triage. Of these, approximately 700 contained blocks of linkage disequilibrium (LD; in Caucasians) poorly covered by the original CATIE genotypes (see below).

### SNP selection for candidate loci

To maximize coverage of the transcribed regions of the selected genes, we identified blocks of LD that were poorly represented by the genotypes provided by the CATIE consortium. We constructed databases for SNPs mapping in the transcribed regions and within 5 kb in either direction using both CATIE-provided genotypes and genotypes downloaded from the International HapMap Project [[103](#R62)].

We then used the Haploview program [[18](#R18), [19](#R19)] (version 4.1)separately on each data set to define haplotype blocks and tagging SNPs (using an r^2^ threshold of 0.8 to define tagging SNPs and considering only haplotypes of frequency ≥0.01) and generated a database that compared the resulting LD blocks for both samples in a contiguous manner based on the position of each SNP in the genome. Tagging SNPs from the HapMap Project, having minor allele frequency (MAF) of ≥0.01 and falling between the LD blocks in the CATIE sample, were selected for further analysis.

In addition, 2060 SNPs with possible functional significance were included. Data on the functional class of SNPs (synonymous, nonsynonymous, 3′- or 5′-UTRs) were downloaded from the NCBI database [[104](#R63)]. The list of SNPs potentially affecting miRNA binding sites was obtained from PolymiRTS Database 2.0 [[20](#R20), [21](#R21), [105](#R64)]. SNPs with nonintronic functional annotations and with MAF of ≥0.01, based on NCBI resources, were selected. The list of putative functional SNPs was compared with the list of SNPs used to tag LD blocks and redundancies were removed.

So as to ensure that newly analyzed SNPs would provide the richest possible source of genetic information, as a final triage we excluded most SNPs that could be imputed with high probability using the genotype data provided by the CATIE consortium. Briefly, we used the approximately 450,000 genotypes already available to impute SNP genotypes on a genome-wide basis with the BEAGLE program (version 3.0.4; [[22](#R22)]) using the HapMap Caucasian (Utah residents with Northern and Western European ancestry [CEU]) trios as reference [[106](#R65)]. This produced an output of imputed genotypes along with an assigned probability for each imputed genotype. We next created a database that contained only SNPs with a mean imputed probability of ≥0.8 across all of the CATIE samples and used this as an exclusion list for SNP selection.

### Design of Infinium HD iSelect® custom BeadChip

The above process identified approximately 10,000 SNPs. In addition, for quality control (QC) and confirmatory purposes, we included 281 SNPs previously genotyped by the CATIE group and approximately 500 SNPs previously evaluated by us in non-CATIE schizophrenia or bipolar patients. Finally, to test the feasibility of using Illumina’s (CA, USA) iSelect^®^ BeadChip platform to detect for copy number variant regions, we included 200 SNPs that we had identified as copy number variant in the GAIN sample using the Affymetrix (CA, USA) Genome-Wide Human SNP Array 6.0 platform.

For this study, we designed a 10,000 bead, iSelect BeadChip obtained from Illumina. The assay design requirements (approximately 30% of SNPs require two beads rather than one) required a further reduction in the number of SNPs. To accommodate this, approximately 3500 SNPs were eliminated due to the fact that they were included solely to capture LD blocks in large genes that displayed genetic association with only two, of the many, analyzed phenotypes. Of the approximately 8500 SNPs remaining, approximately 9% could not be accommodated by the iSelect platform as determined by Illumina’s bioinformatics analysis.

In total, 7584 SNPs located in or near 1711 genes were included on the BeadChip (see [Supplementary Table 1](#SD1) [see [www.futuremedicine.com/doi/suppl/10.2217/pgs.12.105](http://www.futuremedicine.com/doi/suppl/10.2217/pgs.12.105)] for complete information). Of this total, 7303 SNPs have not been previously analyzed for the CATIE sample. The majority of these (4719 in or near 638 genes) covered gaps between LD blocks in candidate loci. The remaining 2584 SNPs (in 1445 genes) have putative functional significance or prior evidence in other sample sets suggesting a role in schizophrenia or bipolar disorder.

### Patients, DNA samples & genotyping

The design of the CATIE study, including details of consent for genetic analyses, has been described in detail by others [[1](#R1), [2](#R2), [15](#R15)]. Patients with a Diagnostic and Statistical Manual of Mental Disorders, 4th Edition (DSM-IV) diagnosis of schizophrenia were randomly assigned to olanzapine, perphenazine, quetiapine, risperidone or ziprasidone in Phase I under double-blind conditions. The study employed a flexible dosing regimen wherein the physician could titrate the number of pills (from one to four daily) as required for each patient. Patients who discontinued the assigned treatment in Phase I or II could elect to continue, and if they did so, received a different treatment in the subsequent phase. As Phase III was unblinded, Phase III results were excluded from the current study. Total duration of the study was 18 months. Only retrospective genetic analyses, judged to be exempt from human studies requirements by an institutional review board, were conducted in the current study.

Consented DNA samples were obtained from the Rutgers University Cell and DNA Repository in collaboration with CCGSMD. We genotyped a total of 407 DNA samples from Caucasian patients who participated in the CATIE study, distributed as follows in Phase I of the trial: olanzapine: 93; perphenazine: 76; quetiapine: 94; risperidone: 97; and ziprasidone: 47. All of these patients self-reported as having exclusively European ancestry. This same patient population was described in detail in a previous study that confirmed that there is no hidden population stratification in the sample [[15](#R15)]. We genotyped an additional 429 samples (215 schizophrenia and 214 bipolar patients) from the GAIN consortium for QC purposes to allow comparisons to the previous genotypes obtained from dbGaP for these samples using the Affymetrix Genome-Wide Human SNP Array 6.0 [[101](#R60)].

Genotyping was performed on a fee for service via the NIH Neuroscience Microarray Consortium basis in the laboratory of Mane (Yale University Mircorarray Center, CT, USA) according to Illumina’s standard operating procedures. Raw intensity files were processed using Illumina’s Bead Studio version 1.7.4 software. At the suggested general call threshold of 0.4, a total of 267 SNPs failed initial QC and were not analyzed further. Only two of these had been previously genotyped in the CATIE sample. For the 7317 SNPs that passed this initial QC, the genotyping success rate across all samples was 98.9% (median 99.4%). Eighteen of these SNPs had success rates of <80% and were not used for subsequent genetic analyses. On an individual sample basis, median genotyping success rates for the remaining SNPs across all SNPs averaged 96.1% (median 96.2%). One sample had an overall success rate of only 76% and was eliminated from genetic analysis. The lowest overall success rate for an individual sample used for genetic analysis was 93.8%. To provide access to the research community, genotypes for all 836 samples have been submitted to CCGSMD.

### Genetic analysis

To allow comparison to previously published pharmacogenomic findings for CATIE and to avoid possible bias in *post hoc* selection of a treatment response variable, we used the mixed model repeated measures (MMRM) approach developed by van den Oord and coworkers [[4](#R4), [12](#R12)]. Briefly, this method models random effects by introducing random slopes for treatment effects, allowing treatment effects to be different across subjects. The MMRM approach serves to increase the statistical power to detect genetic associations by increasing the precision in measuring change in PANSS-T by accounting for variance due to baseline PANSS-T, and treatment, as well as smoothing out the random fluctuations in PANSS-T between visits due to various uncontrolled variables.

Change in PANSS-T was modeled for Phases I, Ib and II of the CATIE study using a model that assumed a 30-day lag period with a constant drug effect after that point [[4](#R4)]. Sample sizes for each of the drugs were as follows: olanzapine: 134; perphenazine: 75; quetiapine: 124; risperidone: 134; and ziprasidone: 74. With a type 1 error rate of 0.05, a sample size of 124 gives 80% power for a SNP that explains 6% of the variance in the regression model, and a sample size of 71 gives 80% power for a SNP that explains 10% of the variance in the regression model. Although genotyping results were obtained for 7303 SNPs not previously evaluated for CATIE, genetic association analysis was limited to 6789 of these SNPs passing QC and having minor allele frequencies of ≥3% in the combined sample of 836 CATIE and GAIN Caucasian patients. For these, we tested the null hypothesis that there was no difference in mean PANSS-T change for patients carrying the minor allele of the SNP for the particular antipsychotic drug (additive model). The change in PANSS-T was used as a continuous dependent variable using the SNP and Variation Suite version 7.3.1 software package (Golden Helix, Inc., MT, USA). Quantile–quantile plots were prepared using the R statistical package version 2.14.1 [[107](#R66)]. For comparison purposes, original CATIE-provided SNP genotypes in specific genes were evaluated using the same genetic analysis. Haplotype association integrating newly generated and original genotypes for specific regions was carried out in SNP and Variation Suite using haplotypes blocks predefined by Haploview software.

## Results

[Figure 1](#F1) summarizes the functional classification for the SNPs included on the custom iSelect BeadChip. Most of the SNPs are intronic (68.9%) and were included to cover LD blocks not well represented in the CATIE-provided genotypes. Additionally, the BeadChip had relatively high representations of SNPs categorized as intergenic (12.2%), 3′-UTR (13.1%) and nonsynonymous coding (4.0%). The smallest functional categories were 5′-UTR (1.4%) and synonymous coding (0.4%) variants.

### Figure 1. Summary of functional categories of newly evaluated SNPs on the custom BeadChip, based on NCBI resources.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/528a/3518380/6d5db1589203/nihms422486f1.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3518380_nihms422486f1.jpg)

For full details see Supplementary Table 1. Data taken from [104].

The custom BeadChip produced high-quality genotypes for those SNPs passing initial genotyping QC, as shown by the high concordance rates for SNPs previously genotyped in the same samples by the CATIE and GAIN consortiums. The concordance rate for the 279 SNPs previously genotyped by the CATIE group (n = 407 samples) averaged 99.8% (median 100%). The concordance rate for the 827 SNPs previously genotyped by the GAIN schizophrenia and bipolar disorder consortium (n = 429 samples) averaged 99.6% (median 99.8%).

Using only those SNPs with MAF of ≥3% within the particular drug arm, we conducted genetic association using the MMRM-modeled response variable for change in PANSS-T for Phases I, Ib and II of the CATIE Study (see ‘Materials & methods’ section). [Figure 2](#F2) shows the quantile–quantile plots for each of the five drugs for the 6789 SNPs not previously evaluated in CATIE (MAF ≥3% within the individual drug arm). These results indicate that the custom BeadChip design resulted in a modest enrichment for SNPs influencing response to four of the five antipsychotics, with quetiapine being the sole exception.

### Figure 2. Quantile–quantile plots for log10 transformed observed p-values from the association tests using the mixed model repeated measures-predicted change in Positive and Negative Syndrome scale total score for olanzapine, perphenazine,– quetiapine, risperidone and ziprasidone.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/528a/3518380/266d5f62e956/nihms422486f2.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3518380_nihms422486f2.jpg)

This analysis is limited to SNPs with minor allele frequencies ≥0.03 in the particular drug arm. Gray areas represent 95% CIs. If the slope for observed p-values (circles) is steeper than the baseline assumption (line, y = x), overall the observed p-values are more significant than p-values expected based on a theoretical distribution.

Results for SNPs with MAFs of ≥0.03 and p-values of ≤5 × 10^−4^ are summarized in [Table 1](#T1). This p-value cutoff corresponds to the false-discovery rate of approximately 0.5 previously used by McClay and coworkers in their pharmacogenomic analysis of the CATIE study [[12](#R12), [23](#R23)]. There were four such findings for olanzapine, six for perphenazine, one for quetiapine, five for risperidone and four for ziprasidone. Complete results for all SNPs not previously evaluated in the CATIE sample and having nominal p-values of ≤0.05 are presented in [Supplementary Tables 2](#SD2)–[6](#SD6).

### Table 1.

| SNP | Gene | Allele (function) | Chr | Position† | MAF | R2‡ | p-value | Ref. |
| --- | --- | --- | --- | --- | --- | --- | --- | --- |
| Olanzapine |   |   |   |   |   |   |   |   |
| rs17070785 | CSMD1 | A (intron) | 8 | 4,467,528 | 0.10 | 0.11 | 1.66 × 10−4 | [24–27, 46] |
| rs2247408 | PLAGL1 | C (intron) | 6 | 144,278,262 | 0.37 | 0.10 | 2.07 × 10−4 |   |
| rs3819811 | PLAGL1 | A (intron) | 6 | 144,280,529 | 0.36 | 0.10 | 2.43 × 10−4 |   |
| rs221253 | PTPRN2 | C (3'-UTR) | 7 | 157,332,172 | 0.04 | 0.10 | 3.33 × 10−4 | [28, 37, 39] |
| Perphenazine |   |   |   |   |   |   |   |   |
| rs11774231 | MCPH1 | C (3' intergenic) | 8 | 6,504,316 | 0.05 | 0.25 | 7.72 × 10−6 | [40, 54] |
| rs2278773 | PRKCE | C (3'-UTR) | 2 | 46,412,422 | 0.03 | 0.23 | 1.69 × 10−5 |   |
| rs17570753 | MCPH1 | A (3' intergenic) | 8 | 6,502,359 | 0.06 | 0.23 | 2.25 × 10−5 |   |
| rs2116971 | CDH13 | C (intron) | 16 | 82,666,139 | 0.44 | 0.18 | 2.09 × 10−4 | [29, 37, 38, 46, 48, 49] |
| rs9952628 | SKOR2 | G (intron) | 18 | 44,754,651 | 0.49 | 0.17 | 3.41 × 10−4 |   |
| rs11100483 | MAML3 | A (intron) | 4 | 141,022,317 | 0.25 | 0.16 | 4.83 × 10−4 |   |
| Quetiapine |   |   |   |   |   |   |   |   |
| rs35793 | KCNMA1 | C (intron) | 10 | 79,183,038 | 0.04 | 0.11 | 1.81 × 10−4 | [45] |
| Risperidone |   |   |   |   |   |   |   |   |
| rs9713 | PSMD14 | A (5'-UTR) | 2 | 162,165,008 | 0.31 | 0.12 | 4.99 × 10−5 |   |
| rs874295 | LRP1B | C (intron) | 2 | 142,488,592 | 0.31 | 0.10 | 1.85 × 10−4 |   |
| rs1869295 | AGAP1 | C (intron) | 2 | 236,995,045 | 0.27 | 0.10 | 2.05 × 10−4 | [26, 30, 41] |
| rs1315115 | NPAS3 | C (intron) | 14 | 33,567,750 | 0.12 | 0.10 | 3.04 × 10−4 | [31–33, 35, 44, 50] |
| rs3738883 | TMEFF2 | C (intron) | 2 | 192,922,256 | 0.47 | 0.09 | 4.11 × 10−4 |   |
| Ziprasidone |   |   |   |   |   |   |   |   |
| rs4925300 | CDH4 | A (intron) | 20 | 60,392,038 | 0.34 | 0.21 | 6.29 × 10−5 | [47] |
| rs1546519 | LYN | C (intron) | 8 | 56,824,558 | 0.29 | 0.19 | 1.30 × 10−4 |   |
| rs17194378 | CNTN4 | A (intron/transcription factor) | 3 | 2,465,091 | 0.36 | 0.17 | 4.41 × 10−4 | [42, 43] |
| rs9585618 | NALCN | C (intron) | 13 | 101,724,041 | 0.41 | 0.17 | 4.79 × 10−4 | [34, 36] |

Table 1 Caption: Results for SNPs with p-values ≤5 × 10−4.

As an additional test of the significance of the findings in [Table 1](#T1), we performed bootstrap sampling of ten random draws of 6789 SNPs (MAF ≥3%) from the approximately 450,000 CATIE-provided genotypes and performed the same genetic association analysis with these. Bootstrap expectations of the numbers of SNPs passing the p-value threshold of ≤5 × 10^−4^ ranged from 3.4 to 4.2 for each of the drug arms, thus confirming that there was no prior bias introduced by using the MMRM end point. By this method, perphenazine had somewhat more results than expected by chance (six observed vs 3.4 expected) and quetiapine had somewhat fewer (one observed vs 3.4 expected).

[Table 1](#T1) also highlights previous findings of others relating the corresponding genes to neuropsychiatric diseases and phenotypes. A number of the genes have been associated with schizophrenia, most notably *CSMD1*, which has been reported in several studies, some showing genome-wide significance [[24](#R24)–[27](#R27)]. Genetic variation relating to schizophrenia risk has been reported also for *PTPRN2* [[28](#R28)], *CDH13* [[29](#R29)], *AGAP1* [[26](#R26), [30](#R30)], *NPAS3* [[31](#R31)–[33](#R33)] and *NALCN* [[34](#R34)]. Prior associations with bipolar disorder have been reported for *NPAS3* [[35](#R35)] and *NALCN* [[36](#R36)]; with attention deficit hyperactivity disorder for *PTPRN2* and *CDH13* [[37](#R37), [38](#R38)]; with autism-spectrum disorders for *PTPRN2* [[39](#R39)], *MCPH1* [[40](#R40)], *AGAP1* [[41](#R41)] and *CNTN4* [[42](#R42), [43](#R43)]; with Parkinson’s disease for *NPAS3* [[44](#R44)]; and with Alzheimer’s disease for *KCNMA1* [[45](#R45)].

For several of the genes in [Table 1](#T1), other associations have been reported for various neuropsychiatric phenotypes. These include cognition for *CSMD1*, *CDH13* and *CDH4* [[46](#R46), [47](#R47)]; brain structure for *MCPH1*; and personality traits for *CDH13* [[48](#R48), [49](#R49)]. In addition, it is notable that variation in *NPAS3* has been reported to impact response to iloperidone, an antipsychotic structurally related to risperidone [[50](#R50)].

Additionally, we evaluated the functional significance the SNPs listed in [Table 1](#T1) using a variety of online resources, including SNP nexus [[108](#R67)]. [Table 1](#T1) reports the function of each SNP. None of the SNPs appear to impact RNA processing, influence miRNA binding or alter known enhancer sequences. It is noteworthy that rs17194378, in an intron of *CNTN4*, could potentially impact transcription of the gene. This SNP is located in a putative binding site for the CUX1 transcription factor, a homeodomain-containing protein that has been implicated in cognitive and neurodegenerative disorders [[51](#R51)].

As summarized in [Table 2](#T2), most of the SNPs tag novel haplotypes or genetically isolated regions that could not have been detected or imputed using the original CATIE genotypes. The data in [Table 2](#T2) were generated by integrating the newly generated genotypes with those provided by the CATIE consortium followed by association analysis with the identical patient samples. Twelve of the 20 SNPs define novel haplotypes and 11 of these 12 are sufficient to tag the particular haplotype. One additional SNP, rs2116971, allowed identification of a new haplotype, but only in conjunction with CATIE-provided genotypes. Four other SNPs (rs2247408, rs3819811, rs221253 and rs874295) tag haplotypes that in retrospect could have been identified as multi-SNP combinations using the original genotypes. Only two SNPs (rs9952628 and rs4925300) neither detect novel haplotypes nor are sufficient to tag the haplotypes in which they are included.

### Table 2.

| SNP | Gene | p-value | Novelhaplotype† | Sufficient totag | Original CATIE SNPs | Original SNPs withlower p-values |
| --- | --- | --- | --- | --- | --- | --- |
| Olanzapine |   |   |   |   |   |   |
| rs17070785 | CSMD1 | 1.66 × 10−4 | Yes | Yes | 1408 | 0 |
| rs2247408 | PLAGL1 | 2.07 × 10−4 | No | Yes‡ | 24 | 0 |
| rs3819811 | PLAGL1 | 2.43 × 10−4 | No | Yes‡ | 24 | 0 |
| rs221253 | PTPRN2 | 3.33 × 10−4 | No | Yes | 144 | 0 |
| Perphenazine |   |   |   |   |   |   |
| rs11774231 | MCPH1 | 7.72 × 10−6 | Yes | Yes‡ | 113 | 1 |
| rs2278773 | PRKCE | 1.69 × 10−5 | Yes | Yes§ | 195 | 0 |
| rs17570753 | MCPH1 | 2.25 × 10−5 | Yes | Yes‡ | 113 | 1 |
| rs2116971 | CDH13 | 2.09 × 10−4 | Yes | No | 666 | 0 |
| rs9952628 | SKOR2 | 3.41 × 10−4 | No | No | 66 | 0 |
| rs11100483 | MAML3 | 4.83 × 10−4 | Yes | Yes§ | 89 | 0 |
| Quetiapine |   |   |   |   |   |   |
| rs35793 | KCNMA1 | 1.81 × 10−4 | No | Yes | 233 | 0 |
| Risperidone |   |   |   |   |   |   |
| rs9713 | PSMD14 | 4.99 × 10−5 | Yes | Yes | 14 | 1 |
| rs874295 | LRP1B | 1.85 × 10−4 | No | Yes | 482 | 1 |
| rs1869295 | AGAP1 | 2.05 × 10−4 | Yes | Yes | 125 | 0 |
| rs1315115 | NPAS3 | 3.04 × 10−4 | Yes | Yes§ | 288 | 2 |
| rs3738883 | TMEFF2 | 4.11 × 10−4 | No | Yes | 63 | 0 |
| Ziprasidone |   |   |   |   |   |   |
| rs4925300 | CDH4 | 6.29 × 10−5 | No | No | 327 | 0 |
| rs1546519 | LYN | 1.30 × 10−4 | Yes | Yes | 38 | 0 |
| rs17194378 | CNTN4 | 4.41 × 10−4 | Yes | Yes | 360 | 0 |
| rs9585618 | NALCN | 4.79 × 10−4 | Yes | Yes§ | 165 | 0 |

Table 2 Caption: Characteristics of top-scoring SNPs.

The last two columns in [Table 2](#T2) place the findings for these SNPs within context of the corresponding gene as a whole. Although the results must be interpreted with caution given the *a priori* enrichment for SNPs with low p-values, the fact that few, if any, other SNPs within a particular gene show more significant association indicates that some of the SNPs indeed tag regions of the genes that impact response to antipsychotics.

## Discussion

The targeted genotyping approach described here resulted in several associations for SNPs potentially impacting response to antipsychotic medications for treatment of schizophrenia. The association results are not biased by *post hoc* selection of a response variable, due to the fact that we used a previously published MMRM-based approach to measure treatment response [[4](#R4)]. However, replication will be needed to confirm the clinical utility of these findings. To facilitate replication of our findings by others, and as a service to the scientific community, [Supplementary Tables 2](#SD2)–[6](#SD6) provide association results for all 6789 newly genotyped SNPs with nominal p-values of ≤0.05. Included in these tables are numerous examples of individual SNPs that may impact response to one or more antipsychotic and that can be further evaluated in future studies.

Of the genes we found to have the most significant SNP associations, only *NPAS3* has been previously reported to contain common genetic variation that impacts response to antipsychotic treatment of schizophrenia. In the present study, rs1315115, located in an intron of *NPAS3*, was associated with response to risperidone. Lavedan and coworkers reported the association of SNPs in *NPAS3* with response to the structurally related drug iloperidone [[50](#R50)]. More work will be needed to understand how these findings relate. Based on data available from the International HapMap Project [[103](#R62)], there is no LD between rs1315115 and the SNPs reported to influence response to iloperidone.

Direct comparisons between our findings and other pharmacogenomic studies using the CATIE sample are difficult to make due to differences in study designs. First of all, we limited analysis to the Caucasian patients to minimize effects of population stratification, in contrast to most previous studies, which combined subpopulations and used principal component adjustment for population stratification to increase sample size [[12](#R12), [15](#R15)]. Furthermore, because we intentionally targeted chromosomal regions not previously evaluated for the CATIE sample, we expected results that would complement rather than replicate previous findings for CATIE. Our findings do follow the pattern seen by others in that no single SNP was associated strongly with response to more than one drug [[12](#R12), [13](#R13)]. This is not surprising considering the diverse mechanisms of action for the various antipsychotic drugs evaluated in the CATIE study and the relatively small sample size of each drug arm [[52](#R52)]. Also, the proportions of variance associated with each individual SNP in [Table 1](#T1) (in the order of 10 to 25%) are similar to those observed for the most significant SNPs in a prior genome-wide association study of antipsychotic response for the CATIE Study [[12](#R12)]. If the influence of these SNPs can be replicated and confirmed to be drug specific, they could have clinical utility, as their effect sizes are of the same magnitude as the treatment effect sizes for the drugs [[53](#R53)].

The custom Illumina iSelect BeadChip, was designed to capture common genetic variation, including functional variation, in genes suspected of having an impact on disease presentation or response to antipsychotics. As expected based on the LD information available at the time the BeadChip was designed, most of the SNPs identified and tagged haplotype blocks not detectable using only SNP genotypes provided by the CATIE group. As such, the new genotypes should prove to be a valuable resource for the scientific community to explore other end points for the Caucasian patients in the CATIE study. All of the genotypes described here, for SNPs not previously genotyped as well as for SNPs included for QC purposes, have been submitted to CCGSMD, for distribution to qualified researchers.

It is interesting to note that over half of the genes with top-scoring SNPs (ten of the 18 genes in [Table 1](#T1)) have independent evidence of genetic association with neuropsychiatric diseases or phenotypes that relate to schizophrenia. *CSMD1* is notable in that highly significant associations for schizophrenia have been published [[24](#R24)–[27](#R27)]. Additionally, genetic variation relating to schizophrenia risk has been reported for *PTPRN2* [[28](#R28)], *CDH13* [[29](#R29)], *AGAP1* [[26](#R26), [30](#R30)], *NPAS3* [[31](#R31)–[33](#R33)] and *NALCN* [[34](#R34)]. Prior associations with bipolar disorder have been reported *NPAS3* [[35](#R35)] and *NALCN* [[36](#R36)]; with *ADHD* for *PTPRN2* and *CDH13* [[37](#R37), [38](#R38)]; with autism-spectrum disorders for *PTPRN2* [[39](#R39)], *MCPH1* [[40](#R40)], *AGAP1* [[41](#R41)] and *CNTN4* [[42](#R42), [43](#R43)]; with Parkinson’s disease for *NPAS3* [[44](#R44)]; and with Alzheimer’s disease for *KCNMA1* [[45](#R45)].

Furthermore, genetic variation in some of the genes has been reported to impact neuropsychiatric phenotypes that could relate to the clinical presentation of schizophrenia. These include cognition for *CSMD1*, *CDH13* and *CDH4* [[46](#R46), [47](#R47)]; brain structure for *MCPH1* [[54](#R54)]; and personality traits for *CDH13* [[48](#R48), [49](#R49)]. Interestingly, genetic variation in *CHD4* is associated with tau protein levels in cerebral spinal fluid, a possible biomarker related to Alzheimer’s disease and cognitive performance [[55](#R55), [56](#R56)].

Finally, three of the remaining genes (*SKOR2*, *MAML3* and *PRKCE*), although lacking published reports of genetic associations with neuropsychiatric diseases, could plausibly impact disease presentation or response to antipsychotics. Protein kinase C, epsilon (encoded by *PRKCE*) interacts with Type A γ-aminobuytric acid receptors in the CNS and with a cation channel important for nociception [[57](#R57)]. *SKOR2* is specifically expressed in Purkinje cells and is required for correct development of the cerebellum [[58](#R58)]. Genetic variation in *MAML3* correlates with expression levels of the *COMT* gene, which has been implicated in schizophrenia risk and disease presentation [[59](#R59)].

## Conclusion

The current study describes results for 6789 SNPs in genes potentially impacting disease risk, disease presentation or response to antipsychotics, with a focus on the Caucasian subset of patients from the CATIE study. Most of the SNPs tag regions of LD or represent functional variants that could not have been detected using the original genotypes provided by the CATIE consortium. Association analyses using the mixed model repeated measures approach of van den Oord and coworkers [[4](#R4), [12](#R12)] identified 20 SNPs with p-values of ≤5 × 10^−4^ (four for olanzapine, six for perphenazine, one for quetiapine, five for risperidone and four for ziprasidone). Many of these are in genes previously implicated in risk of or phenotypes associated with schizophrenia and other neuropsychiatric diseases. The findings support a biological link between disease risk and presentation and antipsychotic drug response. As a service to the research community, we report results for all SNPs with nominal p-values of ≤0.05 (in [Supplementary Tables 2](#SD2)–[6](#SD6)) and have made genotypes of all 6789 SNPs available to qualified researchers through the CCGSMD

## Future perspective

The f indings support a link between the underlying biological bases of disease risk and presentation of schizophrenia and response to commonly prescribed antipsychotic drugs. We propose the term ‘pharmacotypic’ to describe genetic variants that influence both disease presentation and response to drug treatment. Future studies are needed to understand how the SNPs might influence specific neuropsychiatric measures, such as improvement in individual PANSS items or subscales. Additionally, evaluating the functional significance of haplo-types tagged by the most significant SNPs could provide insight into the cellular processes impacting response to antipsychotics and better relate these to the underlying etiology of schizophrenia. If the generality of these findings can be confirmed, it is possible that understanding the impact of genetic make up on the clinical course of schizophrenia will allow more appropriate targeting of treatment in the next 5–10 years.

## Supplementary Material

### Background

### Background

### Background

### Background

### Materials & methods

### Results

### Conclusion

## Acknowledgements

The authors thank BW Massey and TL Ramsey for critical reading of the manuscript.

**Financial & competing interests disclosure**

This work was supported by National Institutes of Mental Health grant MH078437, by Kentucky SBIR Phase II Matching Grant KSTC-184-512-07-007 to MD Brennan. The principal investigators of the CATIE study were JA Lieberman, TS Stroup and JP McEvoy. The CATIE study was funded by a grant from the National Institute of Mental Health (N01 MH900001) along with MH074027 (principal investigator PF Sullivan). Prior genotyping expenses for the CATIE study were funded by Eli Lilly and Company. MD Brennan is an equity holder of SureGene, LLC.

No writing assistance was utilized in the production of this manuscript.

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Lieberman JA, Stroup TS, McEvoy JP, et al. Effectiveness of antipsychotic drugs in patients with chronic schizophrenia. N. Engl. J. Med. 2005;353(12):1209–1223. doi: 10.1056/NEJMoa051688. ▪▪ Descriptions of patient sample and primary outcomes for CATIE

2. Stroup TS, McEvoy JP, Swartz MS, et al. The National Institute of Mental Health Clinical Antipsychotic Trials of Intervention Effectiveness (CATIE) project: schizophrenia trial design and protocol development. Schizophr. Bull. 2003;29(1):15–31. doi: 10.1093/oxfordjournals.schbul.a006986.  [DOI](https://doi.org/10.1093/oxfordjournals.schbul.a006986) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12908658/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Schizophr.%20Bull&title=The%20National%20Institute%20of%20Mental%20Health%20Clinical%20Antipsychotic%20Trials%20of%20Intervention%20Effectiveness%20(CATIE)%20project:%20schizophrenia%20trial%20design%20and%20protocol%20development&author=TS%20Stroup&author=JP%20McEvoy&author=MS%20Swartz&volume=29&issue=1&publication_year=2003&pages=15-31&pmid=12908658&doi=10.1093/oxfordjournals.schbul.a006986&)

3. Kane JM. Pharmacologic treatment of schizophrenia. Biol. Psychiatry. 1999;46(10):1396–1408. doi: 10.1016/s0006-3223(99)00059-1.  [DOI](https://doi.org/10.1016/s0006-3223(99)00059-1) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10578454/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biol.%20Psychiatry&title=Pharmacologic%20treatment%20of%20schizophrenia&author=JM%20Kane&volume=46&issue=10&publication_year=1999&pages=1396-1408&pmid=10578454&doi=10.1016/s0006-3223(99)00059-1&)

4. van den Oord EJ, Adkins DE, McClay J, Lieberman J, Sullivan PF. A systematic method for estimating individual responses to treatment with antipsychotics in CATIE. Schizophr. Res. 2009;107(1):13–21. doi: 10.1016/j.schres.2008.09.009.  [DOI](https://doi.org/10.1016/j.schres.2008.09.009) | [PMC free article](/articles/PMC2652489/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18930379/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Schizophr.%20Res&title=A%20systematic%20method%20for%20estimating%20individual%20responses%20to%20treatment%20with%20antipsychotics%20in%20CATIE&author=EJ%20van%20den%20Oord&author=DE%20Adkins&author=J%20McClay&author=J%20Lieberman&author=PF%20Sullivan&volume=107&issue=1&publication_year=2009&pages=13-21&pmid=18930379&doi=10.1016/j.schres.2008.09.009&)

5. Malhotra AK, Murphy GM, Jr, Kennedy JL. Pharmacogenetics of psychotropic drug response. Am. J. Psychiatry. 2004;161(5):780–796. doi: 10.1176/appi.ajp.161.5.780.  [DOI](https://doi.org/10.1176/appi.ajp.161.5.780) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15121641/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Psychiatry&title=Pharmacogenetics%20of%20psychotropic%20drug%20response&author=AK%20Malhotra&author=GM%20Murphy&author=JL%20Kennedy&volume=161&issue=5&publication_year=2004&pages=780-796&pmid=15121641&doi=10.1176/appi.ajp.161.5.780&)

6. Riley B, Kendler KS. Molecular genetic studies of schizophrenia. Eur. J. Hum. Genet. 2006;14(6):669–680. doi: 10.1038/sj.ejhg.5201571.  [DOI](https://doi.org/10.1038/sj.ejhg.5201571) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16721403/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur.%20J.%20Hum.%20Genet&title=Molecular%20genetic%20studies%20of%20schizophrenia&author=B%20Riley&author=KS%20Kendler&volume=14&issue=6&publication_year=2006&pages=669-680&pmid=16721403&doi=10.1038/sj.ejhg.5201571&)

7. Stefansson H, Ophoff RA, Steinberg S, et al. Common variants conferring risk of schizophrenia. Nature. 2009;460(7256):744–747. doi: 10.1038/nature08186.  [DOI](https://doi.org/10.1038/nature08186) | [PMC free article](/articles/PMC3077530/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19571808/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Common%20variants%20conferring%20risk%20of%20schizophrenia&author=H%20Stefansson&author=RA%20Ophoff&author=S%20Steinberg&volume=460&issue=7256&publication_year=2009&pages=744-747&pmid=19571808&doi=10.1038/nature08186&)

8. Allen NC, Bagade S, McQueen MB, et al. Systematic meta-analyses and field synopsis of genetic association studies in schizophrenia: the SzGene database. Nat. Genet. 2008;40(7):827–834. doi: 10.1038/ng.171.  [DOI](https://doi.org/10.1038/ng.171) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18583979/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet&title=Systematic%20meta-analyses%20and%20field%20synopsis%20of%20genetic%20association%20studies%20in%20schizophrenia:%20the%20SzGene%20database&author=NC%20Allen&author=S%20Bagade&author=MB%20McQueen&volume=40&issue=7&publication_year=2008&pages=827-834&pmid=18583979&doi=10.1038/ng.171&)

9. Cook EH, Jr, Scherer SW. Copy-number variations associated with neuropsychiatric conditions. Nature. 2008;455(7215):919–923. doi: 10.1038/nature07458.  [DOI](https://doi.org/10.1038/nature07458) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18923514/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Copy-number%20variations%20associated%20with%20neuropsychiatric%20conditions&author=EH%20Cook&author=SW%20Scherer&volume=455&issue=7215&publication_year=2008&pages=919-923&pmid=18923514&doi=10.1038/nature07458&)

10. Stefansson H, Rujescu D, Cichon S, et al. Large recurrent microdeletions associated with schizophrenia. Nature. 2008;455(7210):232–236. doi: 10.1038/nature07229.  [DOI](https://doi.org/10.1038/nature07229) | [PMC free article](/articles/PMC2687075/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18668039/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Large%20recurrent%20microdeletions%20associated%20with%20schizophrenia&author=H%20Stefansson&author=D%20Rujescu&author=S%20Cichon&volume=455&issue=7210&publication_year=2008&pages=232-236&pmid=18668039&doi=10.1038/nature07229&)

11. Vrijenhoek T, Buizer-Voskamp JE, van der Stelt I, et al. Recurrent CNVs disrupt three candidate genes in schizophrenia patients. Am. J. Hum. Genet. 2008;83(4):504–510. doi: 10.1016/j.ajhg.2008.09.011.  [DOI](https://doi.org/10.1016/j.ajhg.2008.09.011) | [PMC free article](/articles/PMC2561936/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18940311/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Hum.%20Genet&title=Recurrent%20CNVs%20disrupt%20three%20candidate%20genes%20in%20schizophrenia%20patients&author=T%20Vrijenhoek&author=JE%20Buizer-Voskamp&author=I%20van%20der%20Stelt&volume=83&issue=4&publication_year=2008&pages=504-510&pmid=18940311&doi=10.1016/j.ajhg.2008.09.011&)

12. McClay JL, Adkins DE, Aberg K, et al. Genome-wide pharmacogenomic analysis of response to treatment with antipsychotics. Mol. Psychiatry. 2011;16(1):76–85. doi: 10.1038/mp.2009.89. ▪▪ Describes application of the mixed model repeated measures methodology for pharmacogenomic analysis in CATIE and prior identification of SNPs impacting antipsychotic response.

13. Need AC, Keefe RS, Ge D, et al. Pharmacogenetics of antipsychotic response in the CATIE trial: a candidate gene analysis. Eur. J. Hum. Genet. 2009;17(7):946–957. doi: 10.1038/ejhg.2008.264. ▪ Prior candidate gene analysis for CATIE.

14. Manolio TA, Rodriguez LL, Brooks L, et al. New models of collaboration in genome-wide association studies: the Genetic Association Information Network. Nat. Genet. 2007;39(9):1045–1051. doi: 10.1038/ng2127.  [DOI](https://doi.org/10.1038/ng2127) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17728769/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet&title=New%20models%20of%20collaboration%20in%20genome-wide%20association%20studies:%20the%20Genetic%20Association%20Information%20Network&author=TA%20Manolio&author=LL%20Rodriguez&author=L%20Brooks&volume=39&issue=9&publication_year=2007&pages=1045-1051&pmid=17728769&doi=10.1038/ng2127&)

15. Sullivan PF, Lin D, Tzeng JY, et al. Genomewide association for schizophrenia in the CATIE study: results of stage 1. Mol. Psychiatry. 2008;13(6):570–584. doi: 10.1038/mp.2008.25. ▪ Describes prior genotyping of CATIE samples.

16. Purcell S, Neale B, Todd-Brown K, et al. PLINK: a tool set for whole-genome association and population-based linkage analyses. Am. J. Hum. Genet. 2007;81(3):559–575. doi: 10.1086/519795.  [DOI](https://doi.org/10.1086/519795) | [PMC free article](/articles/PMC1950838/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17701901/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Hum.%20Genet&title=PLINK:%20a%20tool%20set%20for%20whole-genome%20association%20and%20population-based%20linkage%20analyses&author=S%20Purcell&author=B%20Neale&author=K%20Todd-Brown&volume=81&issue=3&publication_year=2007&pages=559-575&pmid=17701901&doi=10.1086/519795&)

17. Hamer RM, Simpson PM. Last observation carried forward versus mixed models in the analysis of psychiatric clinical trials. Am. J. Psychiatry. 2009;166(6):639–641. doi: 10.1176/appi.ajp.2009.09040458.  [DOI](https://doi.org/10.1176/appi.ajp.2009.09040458) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19487398/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Psychiatry&title=Last%20observation%20carried%20forward%20versus%20mixed%20models%20in%20the%20analysis%20of%20psychiatric%20clinical%20trials&author=RM%20Hamer&author=PM%20Simpson&volume=166&issue=6&publication_year=2009&pages=639-641&pmid=19487398&doi=10.1176/appi.ajp.2009.09040458&)

18. Barrett JC, Fry B, Maller J, Daly MJ. Haploview: analysis and visualization of LD and haplotype maps. Bioinformatics. 2005;21(2):263–265. doi: 10.1093/bioinformatics/bth457.  [DOI](https://doi.org/10.1093/bioinformatics/bth457) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15297300/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=Haploview:%20analysis%20and%20visualization%20of%20LD%20and%20haplotype%20maps&author=JC%20Barrett&author=B%20Fry&author=J%20Maller&author=MJ%20Daly&volume=21&issue=2&publication_year=2005&pages=263-265&pmid=15297300&doi=10.1093/bioinformatics/bth457&)

19. Barrett JC. Haploview: visualization and analysis of SNP genotype data. Cold Spring Harb. Protoc. 2009;2009(10):pdb.ip71. doi: 10.1101/pdb.ip71.  [DOI](https://doi.org/10.1101/pdb.ip71) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20147036/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cold%20Spring%20Harb.%20Protoc&title=Haploview:%20visualization%20and%20analysis%20of%20SNP%20genotype%20data&author=JC%20Barrett&volume=2009&issue=10&publication_year=2009&pages=pdb.ip71&pmid=20147036&doi=10.1101/pdb.ip71&)

20. Bao L, Zhou M, Wu L, et al. PolymiRTS Database: linking polymorphisms in microRNA target sites with complex traits. Nucleic Acids Res. 2007;35(Database issue):D51–D54. doi: 10.1093/nar/gkl797.  [DOI](https://doi.org/10.1093/nar/gkl797) | [PMC free article](/articles/PMC1669716/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17099235/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=PolymiRTS%20Database:%20linking%20polymorphisms%20in%20microRNA%20target%20sites%20with%20complex%20traits&author=L%20Bao&author=M%20Zhou&author=L%20Wu&volume=35&issue=Database%20issue&publication_year=2007&pages=D51-D54&pmid=17099235&doi=10.1093/nar/gkl797&)

21. Ziebarth JD, Bhattacharya A, Chen A, Cui Y. PolymiRTS Database 2.0: linking polymorphisms in microRNA target sites with human diseases and complex traits. Nucleic Acids Res. 2012;40(Database issue):D216–D221. doi: 10.1093/nar/gkr1026.  [DOI](https://doi.org/10.1093/nar/gkr1026) | [PMC free article](/articles/PMC3245163/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22080514/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=PolymiRTS%20Database%202.0:%20linking%20polymorphisms%20in%20microRNA%20target%20sites%20with%20human%20diseases%20and%20complex%20traits&author=JD%20Ziebarth&author=A%20Bhattacharya&author=A%20Chen&author=Y%20Cui&volume=40&issue=Database%20issue&publication_year=2012&pages=D216-D221&pmid=22080514&doi=10.1093/nar/gkr1026&)

22. Browning BL, Browning SR. A unified approach to genotype imputation and haplotype-phase inference for large data sets of trios and unrelated individuals. Am. J. Hum. Genet. 2009;84(2):210–223. doi: 10.1016/j.ajhg.2009.01.005.  [DOI](https://doi.org/10.1016/j.ajhg.2009.01.005) | [PMC free article](/articles/PMC2668004/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19200528/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Hum.%20Genet&title=A%20unified%20approach%20to%20genotype%20imputation%20and%20haplotype-phase%20inference%20for%20large%20data%20sets%20of%20trios%20and%20unrelated%20individuals&author=BL%20Browning&author=SR%20Browning&volume=84&issue=2&publication_year=2009&pages=210-223&pmid=19200528&doi=10.1016/j.ajhg.2009.01.005&)

23. Benjamini Y, Hochberg Y. Controlling the false discovery rate: a practical and powerful approach to multiple testing. J. R. Stat. Soc. Series B Stat. Methodol. 1995;57(1):289–300.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20R.%20Stat.%20Soc.%20Series%20B%20Stat.%20Methodol&title=Controlling%20the%20false%20discovery%20rate:%20a%20practical%20and%20powerful%20approach%20to%20multiple%20testing&author=Y%20Benjamini&author=Y%20Hochberg&volume=57&issue=1&publication_year=1995&pages=289-300&)

24. Havik B, Le HS, Rietschel M, et al. The complement control-related genes CSMD1 and CSMD2 associate to schizophrenia. Biol. Psychiatry. 2011;70(1):35–42. doi: 10.1016/j.biopsych.2011.01.030.  [DOI](https://doi.org/10.1016/j.biopsych.2011.01.030) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21439553/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biol.%20Psychiatry&title=The%20complement%20control-related%20genes%20CSMD1%20and%20CSMD2%20associate%20to%20schizophrenia&author=B%20Havik&author=HS%20Le&author=M%20Rietschel&volume=70&issue=1&publication_year=2011&pages=35-42&pmid=21439553&doi=10.1016/j.biopsych.2011.01.030&)

25. Need AC, Ge D, Weale ME, et al. A genome-wide investigation of SNPs and CNVs in schizophrenia. PLoS Genet. 2009;5(2):e1000373. doi: 10.1371/journal.pgen.1000373.  [DOI](https://doi.org/10.1371/journal.pgen.1000373) | [PMC free article](/articles/PMC2631150/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19197363/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet&title=A%20genome-wide%20investigation%20of%20SNPs%20and%20CNVs%20in%20schizophrenia&author=AC%20Need&author=D%20Ge&author=ME%20Weale&volume=5&issue=2&publication_year=2009&pages=e1000373&pmid=19197363&doi=10.1371/journal.pgen.1000373&)

26. Ripke S, Sanders AR, Kendler KS, et al. Genome-wide association study identifies five new schizophrenia loci. Nat. Genet. 2011;43(10):969–976. doi: 10.1038/ng.940.  [DOI](https://doi.org/10.1038/ng.940) | [PMC free article](/articles/PMC3303194/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21926974/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet&title=Genome-wide%20association%20study%20identifies%20five%20new%20schizophrenia%20loci&author=S%20Ripke&author=AR%20Sanders&author=KS%20Kendler&volume=43&issue=10&publication_year=2011&pages=969-976&pmid=21926974&doi=10.1038/ng.940&)

27. Suarez BK, Duan J, Sanders AR, et al. Genomewide linkage scan of 409 European-ancestry and African American families with schizophrenia: suggestive evidence of linkage at 8p23.3-p21.2 and 11p13.1-q14.1 in the combined sample. Am. J. Hum. Genet. 2006;78(2):315–333. doi: 10.1086/500272.  [DOI](https://doi.org/10.1086/500272) | [PMC free article](/articles/PMC1380238/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16400611/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Hum.%20Genet&title=Genomewide%20linkage%20scan%20of%20409%20European-ancestry%20and%20African%20American%20families%20with%20schizophrenia:%20suggestive%20evidence%20of%20linkage%20at%208p23.3-p21.2%20and%2011p13.1-q14.1%20in%20the%20combined%20sample&author=BK%20Suarez&author=J%20Duan&author=AR%20Sanders&volume=78&issue=2&publication_year=2006&pages=315-333&pmid=16400611&doi=10.1086/500272&)

28. Curtis D, Vine AE, McQuillin A, et al. Case-case genome-wide association analysis shows markers differentially associated with schizophrenia and bipolar disorder and implicates calcium channel genes. Psychiatr. Genet. 2011;21(1):1–4. doi: 10.1097/YPG.0b013e3283413382.  [DOI](https://doi.org/10.1097/YPG.0b013e3283413382) | [PMC free article](/articles/PMC3024533/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21057379/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Psychiatr.%20Genet&title=Case-case%20genome-wide%20association%20analysis%20shows%20markers%20differentially%20associated%20with%20schizophrenia%20and%20bipolar%20disorder%20and%20implicates%20calcium%20channel%20genes&author=D%20Curtis&author=AE%20Vine&author=A%20McQuillin&volume=21&issue=1&publication_year=2011&pages=1-4&pmid=21057379&doi=10.1097/YPG.0b013e3283413382&)

29. Holmans PA, Riley B, Pulver AE, et al. Genomewide linkage scan of schizophrenia in a large multicenter pedigree sample using single nucleotide polymorphisms. Mol. Psychiatry. 2009;14(8):786–795. doi: 10.1038/mp.2009.11.  [DOI](https://doi.org/10.1038/mp.2009.11) | [PMC free article](/articles/PMC2714870/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19223858/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol.%20Psychiatry&title=Genomewide%20linkage%20scan%20of%20schizophrenia%20in%20a%20large%20multicenter%20pedigree%20sample%20using%20single%20nucleotide%20polymorphisms&author=PA%20Holmans&author=B%20Riley&author=AE%20Pulver&volume=14&issue=8&publication_year=2009&pages=786-795&pmid=19223858&doi=10.1038/mp.2009.11&)

30. Shi J, Levinson DF, Duan J, et al. Common variants on chromosome 6p22.1 are associated with schizophrenia. Nature. 2009;460(7256):753–757. doi: 10.1038/nature08192.  [DOI](https://doi.org/10.1038/nature08192) | [PMC free article](/articles/PMC2775422/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19571809/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Common%20variants%20on%20chromosome%206p22.1%20are%20associated%20with%20schizophrenia&author=J%20Shi&author=DF%20Levinson&author=J%20Duan&volume=460&issue=7256&publication_year=2009&pages=753-757&pmid=19571809&doi=10.1038/nature08192&)

31. Macintyre G, Alford T, Xiong L, Rouleau GA, Tibbo PG, Cox DW. Association of NPAS3 exonic variation with schizophrenia. Schizophr. Res. 2010;120(1–3):143–149. doi: 10.1016/j.schres.2010.04.002.  [DOI](https://doi.org/10.1016/j.schres.2010.04.002) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20466522/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Schizophr.%20Res&title=Association%20of%20NPAS3%20exonic%20variation%20with%20schizophrenia&author=G%20Macintyre&author=T%20Alford&author=L%20Xiong&author=GA%20Rouleau&author=PG%20Tibbo&volume=120&issue=1%E2%80%933&publication_year=2010&pages=143-149&pmid=20466522&doi=10.1016/j.schres.2010.04.002&)

32. Pickard BS, Christoforou A, Thomson PA, et al. Interacting haplotypes at the NPAS3 locus alter risk of schizophrenia and bipolar disorder. Mol. Psychiatry. 2009;14(9):874–884. doi: 10.1038/mp.2008.24.  [DOI](https://doi.org/10.1038/mp.2008.24) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18317462/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol.%20Psychiatry&title=Interacting%20haplotypes%20at%20the%20NPAS3%20locus%20alter%20risk%20of%20schizophrenia%20and%20bipolar%20disorder&author=BS%20Pickard&author=A%20Christoforou&author=PA%20Thomson&volume=14&issue=9&publication_year=2009&pages=874-884&pmid=18317462&doi=10.1038/mp.2008.24&)

33. Huang J, Perlis RH, Lee PH, et al. Cross-disorder genomewide analysis of schizophrenia, bipolar disorder, and depression. Am. J. Psychiatry. 2010;167(10):1254–1263. doi: 10.1176/appi.ajp.2010.09091335.  [DOI](https://doi.org/10.1176/appi.ajp.2010.09091335) | [PMC free article](/articles/PMC3880556/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20713499/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Psychiatry&title=Cross-disorder%20genomewide%20analysis%20of%20schizophrenia,%20bipolar%20disorder,%20and%20depression&author=J%20Huang&author=RH%20Perlis&author=PH%20Lee&volume=167&issue=10&publication_year=2010&pages=1254-1263&pmid=20713499&doi=10.1176/appi.ajp.2010.09091335&)

34. Wang KS, Liu XF, Aragam N. A genome-wide meta-analysis identifies novel loci associated with schizophrenia and bipolar disorder. Schizophr. Res. 2010;124(1–3):192–199. doi: 10.1016/j.schres.2010.09.002.  [DOI](https://doi.org/10.1016/j.schres.2010.09.002) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20889312/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Schizophr.%20Res&title=A%20genome-wide%20meta-analysis%20identifies%20novel%20loci%20associated%20with%20schizophrenia%20and%20bipolar%20disorder&author=KS%20Wang&author=XF%20Liu&author=N%20Aragam&volume=124&issue=1%E2%80%933&publication_year=2010&pages=192-199&pmid=20889312&doi=10.1016/j.schres.2010.09.002&)

35. Ferreira MA, O’Donovan MC, Meng YA, et al. Collaborative genome-wide association analysis supports a role for ANK3 and CACNA1C in bipolar disorder. Nat. Genet. 2008;40(9):1056–1058. doi: 10.1038/ng.209.  [DOI](https://doi.org/10.1038/ng.209) | [PMC free article](/articles/PMC2703780/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18711365/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet&title=Collaborative%20genome-wide%20association%20analysis%20supports%20a%20role%20for%20ANK3%20and%20CACNA1C%20in%20bipolar%20disorder&author=MA%20Ferreira&author=MC%20O%E2%80%99Donovan&author=YA%20Meng&volume=40&issue=9&publication_year=2008&pages=1056-1058&pmid=18711365&doi=10.1038/ng.209&)

36. Baum AE, Akula N, Cabanero M, et al. A genome-wide association study implicates diacylglycerol kinase eta (DGKH) and several other genes in the etiology of bipolar disorder. Mol. Psychiatry. 2008;13(2):197–207. doi: 10.1038/sj.mp.4002012.  [DOI](https://doi.org/10.1038/sj.mp.4002012) | [PMC free article](/articles/PMC2527618/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17486107/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol.%20Psychiatry&title=A%20genome-wide%20association%20study%20implicates%20diacylglycerol%20kinase%20eta%20(DGKH)%20and%20several%20other%20genes%20in%20the%20etiology%20of%20bipolar%20disorder&author=AE%20Baum&author=N%20Akula&author=M%20Cabanero&volume=13&issue=2&publication_year=2008&pages=197-207&pmid=17486107&doi=10.1038/sj.mp.4002012&)

37. Lasky-Su J, Neale BM, Franke B, et al. Genome-wide association scan of quantitative traits for attention deficit hyperactivity disorder identifies novel associations and confirms candidate gene associations. Am. J. Med. Genet. B Neuropsychiatr. Genet. 2008;147B(8):1345–1354. doi: 10.1002/ajmg.b.30867.  [DOI](https://doi.org/10.1002/ajmg.b.30867) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18821565/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Med.%20Genet.%20B%20Neuropsychiatr.%20Genet&title=Genome-wide%20association%20scan%20of%20quantitative%20traits%20for%20attention%20deficit%20hyperactivity%20disorder%20identifies%20novel%20associations%20and%20confirms%20candidate%20gene%20associations&author=J%20Lasky-Su&author=BM%20Neale&author=B%20Franke&volume=147B&issue=8&publication_year=2008&pages=1345-1354&pmid=18821565&doi=10.1002/ajmg.b.30867&)

38. Lesch KP, Timmesfeld N, Renner TJ, et al. Molecular genetics of adult ADHD: converging evidence from genome-wide association and extended pedigree linkage studies. J. Neural Transm. 2008;115(11):1573–1585. doi: 10.1007/s00702-008-0119-3.  [DOI](https://doi.org/10.1007/s00702-008-0119-3) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18839057/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Neural%20Transm&title=Molecular%20genetics%20of%20adult%20ADHD:%20converging%20evidence%20from%20genome-wide%20association%20and%20extended%20pedigree%20linkage%20studies&author=KP%20Lesch&author=N%20Timmesfeld&author=TJ%20Renner&volume=115&issue=11&publication_year=2008&pages=1573-1585&pmid=18839057&doi=10.1007/s00702-008-0119-3&)

39. Ma D, Salyakina D, Jaworski JM, et al. A genome-wide association study of autism reveals a common novel risk locus at 5p14.1. Ann. Hum. Genet. 2009;73(Pt 3):263–273. doi: 10.1111/j.1469-1809.2009.00523.x.  [DOI](https://doi.org/10.1111/j.1469-1809.2009.00523.x) | [PMC free article](/articles/PMC2918410/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19456320/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann.%20Hum.%20Genet&title=A%20genome-wide%20association%20study%20of%20autism%20reveals%20a%20common%20novel%20risk%20locus%20at%205p14.1&author=D%20Ma&author=D%20Salyakina&author=JM%20Jaworski&volume=73&issue=Pt%203&publication_year=2009&pages=263-273&pmid=19456320&doi=10.1111/j.1469-1809.2009.00523.x&)

40. Ozgen HM, van DE, Bolton PF, et al. Copy number changes of the microcephalin 1 gene (MCPH1) in patients with autism spectrum disorders. Clin. Genet. 2009;76(4):348–356. doi: 10.1111/j.1399-0004.2009.01254.x.  [DOI](https://doi.org/10.1111/j.1399-0004.2009.01254.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19793310/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Genet&title=Copy%20number%20changes%20of%20the%20microcephalin%201%20gene%20(MCPH1)%20in%20patients%20with%20autism%20spectrum%20disorders&author=HM%20Ozgen&author=DE%20van&author=PF%20Bolton&volume=76&issue=4&publication_year=2009&pages=348-356&pmid=19793310&doi=10.1111/j.1399-0004.2009.01254.x&)

41. Cannon DS, Miller JS, Robison RJ, et al. Genome-wide linkage analyses of two repetitive behavior phenotypes in Utah pedigrees with autism spectrum disorders. Mol. Autism. 2010;1(1):3. doi: 10.1186/2040-2392-1-3.  [DOI](https://doi.org/10.1186/2040-2392-1-3) | [PMC free article](/articles/PMC2907569/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20678246/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol.%20Autism&title=Genome-wide%20linkage%20analyses%20of%20two%20repetitive%20behavior%20phenotypes%20in%20Utah%20pedigrees%20with%20autism%20spectrum%20disorders&author=DS%20Cannon&author=JS%20Miller&author=RJ%20Robison&volume=1&issue=1&publication_year=2010&pages=3&pmid=20678246&doi=10.1186/2040-2392-1-3&)

42. Roohi J, Montagna C, Tegay DH, et al. Disruption of contactin 4 in three subjects with autism spectrum disorder. J. Med. Genet. 2009;46(3):176–182. doi: 10.1136/jmg.2008.057505.  [DOI](https://doi.org/10.1136/jmg.2008.057505) | [PMC free article](/articles/PMC2643049/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18349135/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Med.%20Genet&title=Disruption%20of%20contactin%204%20in%20three%20subjects%20with%20autism%20spectrum%20disorder&author=J%20Roohi&author=C%20Montagna&author=DH%20Tegay&volume=46&issue=3&publication_year=2009&pages=176-182&pmid=18349135&doi=10.1136/jmg.2008.057505&)

43. Glessner JT, Wang K, Cai G, et al. Autism genome-wide copy number variation reveals ubiquitin and neuronal genes. Nature. 2009;459(7246):569–573. doi: 10.1038/nature07953.  [DOI](https://doi.org/10.1038/nature07953) | [PMC free article](/articles/PMC2925224/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19404257/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Autism%20genome-wide%20copy%20number%20variation%20reveals%20ubiquitin%20and%20neuronal%20genes&author=JT%20Glessner&author=K%20Wang&author=G%20Cai&volume=459&issue=7246&publication_year=2009&pages=569-573&pmid=19404257&doi=10.1038/nature07953&)

44. Edwards TL, Scott WK, Almonte C, et al. Genome-wide association study confirms SNPs in SNCA and the MAPT region as common risk factors for Parkinson disease. Ann. Hum. Genet. 2010;74(2):97–109. doi: 10.1111/j.1469-1809.2009.00560.x.  [DOI](https://doi.org/10.1111/j.1469-1809.2009.00560.x) | [PMC free article](/articles/PMC2853717/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20070850/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann.%20Hum.%20Genet&title=Genome-wide%20association%20study%20confirms%20SNPs%20in%20SNCA%20and%20the%20MAPT%20region%20as%20common%20risk%20factors%20for%20Parkinson%20disease&author=TL%20Edwards&author=WK%20Scott&author=C%20Almonte&volume=74&issue=2&publication_year=2010&pages=97-109&pmid=20070850&doi=10.1111/j.1469-1809.2009.00560.x&)

45. Burns LC, Minster RL, Demirci FY, et al. Replication study of genome-wide associated SNPs with late-onset Alzheimer’s disease. Am. J. Med. Genet. B Neuropsychiatr. Genet. 2011;156B(4):507–512. doi: 10.1002/ajmg.b.31194.  [DOI](https://doi.org/10.1002/ajmg.b.31194) | [PMC free article](/articles/PMC3082594/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21480501/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Med.%20Genet.%20B%20Neuropsychiatr.%20Genet&title=Replication%20study%20of%20genome-wide%20associated%20SNPs%20with%20late-onset%20Alzheimer%E2%80%99s%20disease&author=LC%20Burns&author=RL%20Minster&author=FY%20Demirci&volume=156B&issue=4&publication_year=2011&pages=507-512&pmid=21480501&doi=10.1002/ajmg.b.31194&)

46. Cirulli ET, Kasperaviciute D, Attix DK, et al. Common genetic variation and performance on standardized cognitive tests. Eur. J. Hum. Genet. 2010;18(7):815–820. doi: 10.1038/ejhg.2010.2.  [DOI](https://doi.org/10.1038/ejhg.2010.2) | [PMC free article](/articles/PMC2987367/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20125193/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur.%20J.%20Hum.%20Genet&title=Common%20genetic%20variation%20and%20performance%20on%20standardized%20cognitive%20tests&author=ET%20Cirulli&author=D%20Kasperaviciute&author=DK%20Attix&volume=18&issue=7&publication_year=2010&pages=815-820&pmid=20125193&doi=10.1038/ejhg.2010.2&)

47. Seshadri S, DeStefano AL, Au R, et al. Genetic correlates of brain aging on MRI and cognitive test measures: a genome-wide association and linkage analysis in the Framingham Study. BMC. Med. Genet. 2007;8(Suppl. 1):S15. doi: 10.1186/1471-2350-8-S1-S15.  [DOI](https://doi.org/10.1186/1471-2350-8-S1-S15) | [PMC free article](/articles/PMC1995608/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17903297/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC.%20Med.%20Genet&title=Genetic%20correlates%20of%20brain%20aging%20on%20MRI%20and%20cognitive%20test%20measures:%20a%20genome-wide%20association%20and%20linkage%20analysis%20in%20the%20Framingham%20Study&author=S%20Seshadri&author=AL%20DeStefano&author=R%20Au&volume=8&issue=Suppl.%201&publication_year=2007&pages=S15&pmid=17903297&doi=10.1186/1471-2350-8-S1-S15&)

48. Terracciano A, Sanna S, Uda M, et al. Genome-wide association scan for five major dimensions of personality. Mol. Psychiatry. 2010;15(6):647–656. doi: 10.1038/mp.2008.113.  [DOI](https://doi.org/10.1038/mp.2008.113) | [PMC free article](/articles/PMC2874623/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18957941/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol.%20Psychiatry&title=Genome-wide%20association%20scan%20for%20five%20major%20dimensions%20of%20personality&author=A%20Terracciano&author=S%20Sanna&author=M%20Uda&volume=15&issue=6&publication_year=2010&pages=647-656&pmid=18957941&doi=10.1038/mp.2008.113&)

49. Terracciano A, Tanaka T, Sutin AR, et al. Genome-wide association scan of trait depression. Biol. Psychiatry. 2010;68(9):811–817. doi: 10.1016/j.biopsych.2010.06.030.  [DOI](https://doi.org/10.1016/j.biopsych.2010.06.030) | [PMC free article](/articles/PMC2955852/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20800221/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biol.%20Psychiatry&title=Genome-wide%20association%20scan%20of%20trait%20depression&author=A%20Terracciano&author=T%20Tanaka&author=AR%20Sutin&volume=68&issue=9&publication_year=2010&pages=811-817&pmid=20800221&doi=10.1016/j.biopsych.2010.06.030&)

50. Lavedan C, Licamele L, Volpi S, et al. Association of the NPAS3 gene and five other loci with response to the antipsychotic iloperidone identified in a whole genome association study. Mol. Psychiatry. 2009;14(8):804–819. doi: 10.1038/mp.2008.56.  [DOI](https://doi.org/10.1038/mp.2008.56) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18521090/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol.%20Psychiatry&title=Association%20of%20the%20NPAS3%20gene%20and%20five%20other%20loci%20with%20response%20to%20the%20antipsychotic%20iloperidone%20identified%20in%20a%20whole%20genome%20association%20study&author=C%20Lavedan&author=L%20Licamele&author=S%20Volpi&volume=14&issue=8&publication_year=2009&pages=804-819&pmid=18521090&doi=10.1038/mp.2008.56&)

51. Hulea L, Nepveu A. CUX1 transcription factors: from biochemical activities and cell-based assays to mouse models and human diseases. Gene. 2012;497(1):18–26. doi: 10.1016/j.gene.2012.01.039.  [DOI](https://doi.org/10.1016/j.gene.2012.01.039) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22306263/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gene&title=CUX1%20transcription%20factors:%20from%20biochemical%20activities%20and%20cell-based%20assays%20to%20mouse%20models%20and%20human%20diseases&author=L%20Hulea&author=A%20Nepveu&volume=497&issue=1&publication_year=2012&pages=18-26&pmid=22306263&doi=10.1016/j.gene.2012.01.039&)

52. Meltzer HY, Huang M. In vivo actions of atypical antipsychotic drug on serotonergic and dopaminergic systems. Prog. Brain Res. 2008;172:177–197. doi: 10.1016/S0079-6123(08)00909-6.  [DOI](https://doi.org/10.1016/S0079-6123(08)00909-6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18772033/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Prog.%20Brain%20Res&title=In%20vivo%20actions%20of%20atypical%20antipsychotic%20drug%20on%20serotonergic%20and%20dopaminergic%20systems&author=HY%20Meltzer&author=M%20Huang&volume=172&publication_year=2008&pages=177-197&pmid=18772033&doi=10.1016/S0079-6123(08)00909-6&)

53. Ramsey TL, Meltzer HY, Brock GN, et al. Evidence for a SULT4A1 haplotype correlating with baseline psychopathology and atypical antipsychotic response. Pharmacogenomics. 2011;12(4):471–480. doi: 10.2217/pgs.10.205.  [DOI](https://doi.org/10.2217/pgs.10.205) | [PMC free article](/articles/PMC3168511/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21521020/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Evidence%20for%20a%20SULT4A1%20haplotype%20correlating%20with%20baseline%20psychopathology%20and%20atypical%20antipsychotic%20response&author=TL%20Ramsey&author=HY%20Meltzer&author=GN%20Brock&volume=12&issue=4&publication_year=2011&pages=471-480&pmid=21521020&doi=10.2217/pgs.10.205&)

54. Rimol LM, Agartz I, Djurovic S, et al. Sex-dependent association of common variants of microcephaly genes with brain structure. Proc. Natl Acad. Sci. USA. 2010;107(1):384–388. doi: 10.1073/pnas.0908454107.  [DOI](https://doi.org/10.1073/pnas.0908454107) | [PMC free article](/articles/PMC2806758/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20080800/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl%20Acad.%20Sci.%20USA&title=Sex-dependent%20association%20of%20common%20variants%20of%20microcephaly%20genes%20with%20brain%20structure&author=LM%20Rimol&author=I%20Agartz&author=S%20Djurovic&volume=107&issue=1&publication_year=2010&pages=384-388&pmid=20080800&doi=10.1073/pnas.0908454107&)

55. Han MR, Schellenberg GD, Wang LS. Genome-wide association reveals genetic effects on human Abeta42 and tau protein levels in cerebrospinal fluids: a case control study. BMC. Neurol. 2010;10:90. doi: 10.1186/1471-2377-10-90.  [DOI](https://doi.org/10.1186/1471-2377-10-90) | [PMC free article](/articles/PMC2964649/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20932310/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC.%20Neurol&title=Genome-wide%20association%20reveals%20genetic%20effects%20on%20human%20Abeta42%20and%20tau%20protein%20levels%20in%20cerebrospinal%20fluids:%20a%20case%20control%20study&author=MR%20Han&author=GD%20Schellenberg&author=LS%20Wang&volume=10&publication_year=2010&pages=90&pmid=20932310&doi=10.1186/1471-2377-10-90&)

56. Vemuri P, Weigand SD, Przybelski SA, et al. Cognitive reserve and Alzheimer’s disease biomarkers are independent determinants of cognition. Brain. 2011;134(Pt 5):1479–1492. doi: 10.1093/brain/awr049.  [DOI](https://doi.org/10.1093/brain/awr049) | [PMC free article](/articles/PMC3097887/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21478184/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Brain&title=Cognitive%20reserve%20and%20Alzheimer%E2%80%99s%20disease%20biomarkers%20are%20independent%20determinants%20of%20cognition&author=P%20Vemuri&author=SD%20Weigand&author=SA%20Przybelski&volume=134&issue=Pt%205&publication_year=2011&pages=1479-1492&pmid=21478184&doi=10.1093/brain/awr049&)

57. Newton PM, Messing RO. The substrates and binding partners of protein kinase Cepsilon. Biochem. J. 2010;427(2):189–196. doi: 10.1042/BJ20091302.  [DOI](https://doi.org/10.1042/BJ20091302) | [PMC free article](/articles/PMC2966297/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20350291/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem.%20J&title=The%20substrates%20and%20binding%20partners%20of%20protein%20kinase%20Cepsilon&author=PM%20Newton&author=RO%20Messing&volume=427&issue=2&publication_year=2010&pages=189-196&pmid=20350291&doi=10.1042/BJ20091302&)

58. Wang B, Harrison W, Overbeek PA, Zheng H. Transposon mutagenesis with coat color genotyping identifies an essential role for Skor2 in sonic hedgehog signaling and cerebellum development. Development. 2011;138(20):4487–4497. doi: 10.1242/dev.067264.  [DOI](https://doi.org/10.1242/dev.067264) | [PMC free article](/articles/PMC3177318/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21937600/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Development&title=Transposon%20mutagenesis%20with%20coat%20color%20genotyping%20identifies%20an%20essential%20role%20for%20Skor2%20in%20sonic%20hedgehog%20signaling%20and%20cerebellum%20development&author=B%20Wang&author=W%20Harrison&author=PA%20Overbeek&author=H%20Zheng&volume=138&issue=20&publication_year=2011&pages=4487-4497&pmid=21937600&doi=10.1242/dev.067264&)

59. Xing C, Torres-Caban M, Wang T, Lu Q, Xing G, Elston RC. Linkage studies of catechol-O-methyltransferase (COMT) and dopamine-beta-hydroxylase (DBH) cDNA expression levels. BMC. Proc. 2007;1(Suppl. 1):S95. doi: 10.1186/1753-6561-1-s1-s95.  [DOI](https://doi.org/10.1186/1753-6561-1-s1-s95) | [PMC free article](/articles/PMC2367604/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18466599/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC.%20Proc&title=Linkage%20studies%20of%20catechol-O-methyltransferase%20(COMT)%20and%20dopamine-beta-hydroxylase%20(DBH)%20cDNA%20expression%20levels&author=C%20Xing&author=M%20Torres-Caban&author=T%20Wang&author=Q%20Lu&author=G%20Xing&volume=1&issue=Suppl.%201&publication_year=2007&pages=S95&pmid=18466599&doi=10.1186/1753-6561-1-s1-s95&)
