# Quantitation of Pretreatment Serum IP-10 Improves the Predictive Value of an IL28B Gene Polymorphism for Hepatitis C Treatment Response

## Metadata
**Authors:** Jama M Darling, Jeroen Aerssens, Gregory Fanning, John G McHutchison, David B Goldstein, Alexander J Thompson, Kevin V Shianna, Nezam H Afdhal, Michael L Hudson, Charles D Howell, Willem Talloen, Jacques Bollekens, Mieke De Wit, Annick Scholliers, Michael W Fried
**Journal:** Hepatology (Baltimore, Md.)
**Date:** 2011 Jan
**DOI:** [10.1002/hep.24056](https://doi.org/10.1002/hep.24056)
**PMID:** 21254158
**PMCID:** PMC3083026
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3083026/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC3083026/pdf/nihms247191.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC3083026/pdf/nihms247191.pdf)

## Abstract

**Conclusions:** 
When IL28B genotype is combined with pretreatment serum IP-10 measurement, the predictive value for discrimination between SVR and non-response is significantly improved, especially in non-CC genotypes. This relationship warrants further investigation to elucidate mechanisms of antiviral response and prospective validation.

Keywords: Peginterferon, Ribavirin, Race, Biomarker, Genotype

### Conclusions

When *IL28B* genotype is combined with pretreatment serum IP-10 measurement, the predictive value for discrimination between SVR and non-response is significantly improved, especially in non-CC genotypes. This relationship warrants further investigation to elucidate mechanisms of antiviral response and prospective validation.

## PATIENTS AND METHODS

### Patients

The VIRAHEP-C study was a multicenter study of combination pegIFN and ribavirin therapy of chronic hepatitis C designed to assess the rates and predictors of response among AA and CA with genotype 1 infection, and to identify reasons for nonresponse to therapy. The design and primary outcomes of the VIRAHEP-C trial have been reported elsewhere ([8](#R8)). Adults who were treatment naive, infected with genotype 1, had detectable HCV RNA, and had histologic evidence of chronic HCV were eligible to participate. Patients were classified by race as either African American or Caucasian, and by ethnicity as either Hispanic or non-Hispanic, based on self-report. All participants were required to have been born in the United States. From 8 clinical centers across the United States, 401 patients were enrolled and started on therapy between July 2002 and December 2003.

For the present study, serum samples were acquired from a subset of 272 patients from the total VIRAHEP-C cohort, comprising 157 sustained virological responders (SVR) (104 CA, 53 AA) and 115 non-responders (34 CA, 81 AA). All specimens analyzed in this study were obtained under IRB-approved protocols for which participants provided written informed consent, including consent for genetic testing.

### Treatment

Patients received peginterferon alfa-2a (Pegasys, Roche Pharmaceuticals, Nutley, NJ) 180ug weekly and ribavirin (Copegus, Roche Pharmaceuticals, Nutley, NJ) 1000–1200 mg daily for at least 24 weeks. Patients who became HCV RNA negative by week 24 continued treatment for a total of 48 weeks, whereas those who remained HCV RNA positive stopped treatment and were considered non-responders. The primary endpoint of the trial was SVR, defined as the absence of detectable HCV RNA for at least 24 weeks after stopping therapy.

### HCV RNA quantification

HCV RNA testing was done at a central laboratory (SeraCare BioServices, Gaithersburg, MD) using the Cobas Amplicor Assay (sensitivity 50 IU/ml: Roche Molecular Diagnostics, Alameda, CA). Selected samples were tested for HCV RNA levels by Cobas Amplicor Monitor Assay and for HCV RNA genotype by Versant HCV Genotype Assay (Bayer, Tarrytown, NY).

### Liver histology

All patients had undergone liver biopsy within 18 months of screening, which were read blinded by a central pathologist. All biopsies were assessed for severity of hepatitis C by grading the inflammation and staging the fibrosis using Ishak’s modified histologic activity index (HAI) scoring system.

### Quantification of serum IP-10

Interferon-γ Inducible Protein-10 (IP-10) was measured in serum samples collected at baseline, prior to initiation of treatment, using the commercially available Quantikine human CXCL10/IP-10 immunoassay (R&D Systems). All samples were diluted 1:2 and analyzed in duplicate. The linear dynamic range of the IP-10 measurement in this assay was 8 – 500 pg/ml, with a detection limit at 7.8 pg/ml. Samples with IP-10 concentration above 1000 pg/ml were diluted 1:5 and re-analyzed.

### Genotyping of IL28B

The *IL28B* polymorphic marker rs1297860 was analyzed using the ABI TaqMan allelic discrimination kit and the ABI7900HT Sequence Detection System (Applied Biosystems), as described by Thomas et al ([21](#R21), [24](#R24)–[27](#R27)). DNA samples were genotyped from 210 patients from our cohort.

### Statistical methods

Standard statistical analyses were done using JMP 7.0.2 or SAS version 9.1 software (both from SAS Institute, Inc). IP-10 concentrations were log-transformed before use in statistical tests to meet distribution normality assumptions. Publicly available packages in R (version 2.8.0) were used to assess different classification models (diagonal linear discriminant analysis, random forest, support vector machine, and bagging), as well as receiver operating characteristic (ROC) curve analysis. Fitting logistic regression models and generalized linear models was performed using respectively the *proc logistic* and *proc genmod* procedures in SAS. Graphs were made with the utilized statistical software tools or with GraphPad Prism 4 (GraphPad Software, Inc).

## RESULTS

### Patients included in the study cohort

Serum samples from 157 sustained virologic responders (SVR) and 115 non-responders to antiviral therapy were included from the VIRAHEP-C cohort for this study. The definitions of SVR and non-responder are provided in the methods section. Patients with viral relapse, breakthrough or less than 12 weeks of available virologic data were excluded. The cohort consisted of 134 AA and 138 CA. Baseline patient characteristics of this cohort were as follows: age 48.4 ± 7.4 years; viral load 4.6 ± 5.7 ×10^6^ IU/ml; platelet count 214 ± 73 ×10^6^ cells/mm^3^; alanine transaminase (ALT) 90.9 ± 72.9 IU/l; total bilirubin 0.70 ± 0.35 mg/dl; albumin 4.1 ± 0.40 g/dl; and hematocrit 43.2 ± 3.8 % (all data as mean ± SD, Supplementary Table 1). The cohort included 96 females and 176 males, and 19% with an Ishak fibrosis score of 4–6. Samples from 210 of the 272 patients in our cohort were available for *IL28B* genotyping, 123 SVR and 87 non-responders of whom 111 were CA and 99 AA.

### Baseline serum IP-10 measurement and treatment response

Mean serum IP-10 levels were significantly lower in SVR compared to non-responder patients (437 ± 31 pg/ml versus 704 ± 44 pg/ml, p<0.001) ([Figure 1A](#F1), [Table 1](#T1)). To assess the potential predictive value of IP-10 measurements, we stratified the patients according to a 600 pg/ml threshold value which has been used in prior studies ([15](#R15), [16](#R16), [18](#R18)). Sixty-nine percent (129/188) of individuals with a low baseline IP-10 level (<600 pg/ml) were responders (positive predictive value, PPV = 69%), while 67% (56/83) of the individuals with a high baseline IP-10 level (>600 pg/ml) were non-responders to therapy (negative predictive value, NPV = 67%) ([Figure 1B](#F1)). Overall, this results in a specificity of 82% (129/157) and a sensitivity of 49% (56/115) for a test predictive of therapy response based on pre-treatment serum IP-10 levels. Baseline demographic parameters of the cohort stratified according to pretreatment IP-10 level are shown in Supplementary Table 1. Between high and low IP-10 groups, significant differences were seen for several parameters implying a possible association with IP-10 level. Previous groups have also noted association of race and viral load with IP-10 levels ([16](#R16), [20](#R20)).

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/aae1/3083026/cbe7baa96be2/nihms247191f1.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3083026_nihms247191f1.jpg)

### TABLE 1.

|   | All | SVR | Non-responder | t-test |
| --- | --- | --- | --- | --- |
|   |   |   |  |  |
| All |   | 437 ± 31 (n=157) | 704 ± 44 (n=115) | p<0.001 |
| Caucasian | 504 ± 38 (n=138) | 447 ± 44 (n=104) | 677 ± 69 (n=34) | p<0.001 |
| African-American | 598 ± 38 (n=134) | 418 ± 35 (n=53) | 716 ± 55 (n=81) | p<0.001 |

Table 1 Caption: Pretreatment serum IP-10 levels (pg/ml) in HCV patients stratified according to race and interferon treatment response (SVR). A significant difference was found between CA and AA for the whole cohort (p=0.015), but not within the subgroups of responders or non-responders.

We evaluated the IP-10 threshold value of 600 pg/ml used to identify SVR patients by computing the cutoff value with the best discriminatory ability based on a ROC curve analysis. In our dataset, a threshold concentration of 370 pg/ml revealed the optimal combination of specificity (80%) and sensitivity (56%) in predicting SVR patients. We then determined our optimal IP-10 level to correctly predict both SVR as well as non-response. A threshold value of 550 pg/ml yielded the highest rate of true positives or negatives (69%), and correlated well with the 600 pg/ml cutoff that has been used in the literature (68% true positives or negatives predicted in our dataset). Finally, logistic regression analysis of pre-treatment IP-10 concentrations enabled fitting the probability of SVR for specific IP-10 levels measured in individual patients, and demonstrated a highly significant effect of IP-10 (p<0.0001; [Supplementary Figure 1](#SD1), grey curve).

### Effect of race on serum IP-10 levels

When comparing pre-treatment IP-10 serum levels of CA and AA, no significant differences were observed in separate analyses of the responders (p=0.75) and non-responders (p=0.97) ([Table 1](#T1)). The significant (p=0.015) difference in baseline serum IP-10 level between CA and AA that was observed in the overall study cohort can most likely be explained by the unbalanced composition of the cohort (IFN treatment response rate in the CA subgroup was 75% as compared to 40% in the AA subgroup). The highly significant difference in IP-10 serum level between responders and non-responders to IFN therapy was found in both CA and AA ([Table 1](#T1)). Logistic regression analyses of baseline IP-10 levels were used to generate treatment response curves for CA and AA patients ([Supplementary Figure 1](#SD1)). The response curves for AA and CA patients revealed a significant effect of both IP-10 (p<0.0001) and race (p<0.0001), but no significant interaction between IP-10 and race (p=0.08).

### IL28B genotype and treatment response

Of the 210 patients genotyped, 30% were CC, 49% were CT, and 21% were TT. A significant association between *IL28B* genotype and treatment response was observed: corresponding SVR rates were 87% for CC, 50% for CT, and 39% for TT (p<0.0001) ([Table 2](#T2)). For Caucasian-Americans, 49% were CC with an SVR of 91%, 41% were CT with an SVR of 67% and 10% were TT with an SVR of 45% (p<0.001). For African-Americans, only 9% were CC with an SVR of 67%, 58% were CT with an SVR of 35% and 33% were TT with an SVR of 36% (p=0.20).

### TABLE 2.

| IL28B genotype | CC | CT | TT | Likelihood Ratio Chi2 |
| --- | --- | --- | --- | --- |
| % SVR entire cohort (n SVR / total n) | 87% (55/63) | 50% (51/103) | 39% (17/44) | p < 0.0001 |
| % SVR CA (n SVR CA / total n CA) | 91% (49/54) | 67% (31/46) | 45% (5/11) | p = 0.0008 |
| % SVR AA (n SVR AA / total n AA) | 67% (6/9) | 35% (20/57) | 36% (12/33) | p = 0.20 |

Table 2 Caption: SVR rate according to IL28B genotype in the entire cohort and stratified per racial group.

### Association of IL28B genotype and pretreatment IP-10 levels with treatment response

Mean serum IP-10 levels were similar for all patients regardless of *IL28B* genotype both in Caucasians (p=0.27) and in African-Americans (p=0.58) ([Figure 2](#F2)). This lack of correlation between serum IP-10 and *IL28B* genotype indicates that the associations with SVR observed for both of these markers are independent. Using the 600 pg/ml cutoff for pretreatment IP-10 levels, the SVR rate for our cohort of patients with both serum IP-10 and *IL28B* genotype data available (n=210) was 69% for those with a low IP-10 level (<600 pg/ml) and 35% for those with a high IP-10 level (>600pg/ml) (p<0.0001).

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/aae1/3083026/e64e90d0b8ea/nihms247191f2.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3083026_nihms247191f2.jpg)

Modeling SVR as a function of *IL28B* genotype and serum IP-10 (above or below 600 pg/ml) in a nominal logistic regression revealed a significant additive effect of *IL28B* genotype (p<0.0001) and serum IP-10 (p<0.0015) in predicting SVR (Chi^2^=55, p<0.001), but no interaction between *IL28B* and IP-10 (p=0.66). [Figure 3](#F3) visualizes that baseline IP-10 levels within the *IL28B* genotype groups provided additional and independent information regarding SVR rate. More specifically, baseline IP-10 levels were most helpful in *IL28B* T-allele carriers. The overall response rate for CT carriers was 50% but for those with low IP-10 levels 64% had an SVR versus 24% with high IP-10 levels. For the TT genotype, 39% had an SVR with 48% in the low pretreatment IP-10 group and 20% in the high IP-10 group. Logistic regression modeling of SVR response based on serum IP-10 level treated as a continuous variable and *IL28B* genotype enabled a more individualized prediction of the probability of SVR according to serum IP-10 level, with an additional and significant *IL28B* genotype-dependent shift in response curve ([Supplementary Figure 2](#SD2)). Complementary receiver operating characteristic (ROC) curve analyses, which allow a more quantitative comparison of predictive models, revealed similar ROC area under the curve (AUC) values for the model based on pretreatment serum IP-10 alone (0.71) versus *IL28B* genotype alone (0.70). A much higher ROC AUC value (0.80) was achieved, however, for the model that combined both markers ([Figure 4](#F4)). Together, these data demonstrate that combining *IL28B* genotype with pretreatment serum IP-10 measurements clearly improves the predictive value of SVR, especially in non-CC genotypes.

### Figure 3.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/aae1/3083026/ac3cd2d78f46/nihms247191f3.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3083026_nihms247191f3.jpg)

### Figure 4.

![Figure 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/aae1/3083026/cc7afe9d8b8e/nihms247191f4.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3083026_nihms247191f4.jpg)

The same and significant trend was also found when the analysis was done per racial group ([Table 3](#T3)). For example, in African Americans, the difference with baseline IP-10 levels was even more striking for the CT and TT *IL28B* genotypes. For the CT carriers with low IP-10, SVR was 48% versus 17% with high IP-10, whereas for TT carriers with low IP-10, SVR was 43% versus 25% with high IP-10.

### TABLE 3.

|   | Caucasian-American |  | African-American |  |
| --- | --- | --- | --- | --- |
| IP-10 | low | high | low | high |
| IL28B |   |   |   |   |
| CC | 91% (40/44) | 90% (9/10) | 80% (4/5) | 50% (2/4) |
| CT | 79% (26/33) | 38% (5/13) | 48% (16/33) | 17% (4/24) |
| TT | 62% (5/8) | 0% (0/3) | 43% (9/21) | 25% (3/12) |
|   |  |  |  |  |
|   | Chi2 = 25.5, p<0.0001 |  | Chi2 = 11.8, p=0.037 |  |

Table 3 Caption: SVR rate (%) according to a combination of IL28B genotype and pretreatment serum IP-10 level (<600 pg/ml: low; >600 pg/ml: high), stratified per racial group. Data are expressed as % SVR (n SVR / total n). Likelihood ratio Chi2 analyses were performed to test for response homogeneity across the sample groups.

### Combining IL28B genotype and pretreatment IP-10 levels with other baseline parameters to predict treatment response

We assessed whether other baseline parameters, in addition to *IL28B* genotype and serum IP-10, could significantly improve the prediction of SVR. In this analysis, we added age, gender, race, pretreatment viral load, Ishak fibrosis score, ALT, steatosis, and histological activity index in a logistic regression model. Of all parameters included, only pre-treatment viral load (p<0.0001), *IL28B* genotype (p=0.0004), baseline IP-10 level (p=0.0033), and race (p=0.0011) significantly contributed to the model. No interaction between any pair of variables was significant (all p>0.1). When all variables were treated as categorical variables (eg, IP-10 above or below 600 pg/ml, rather than as a continuous variable), the resulting generalized linear model included the same four significant variables plus ISHAK fibrosis score ([Figure 5](#F5)).

### Figure 5.

![Figure 5](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/aae1/3083026/c528e47072a1/nihms247191f5.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3083026_nihms247191f5.jpg)

ROC curve analyses were performed to compare the predictive power of models that include the most significant variables identified in these multivariate analyses ([Figure 4](#F4)). The AUC of the model that combined *IL28B* genotype and serum IP-10 (AUC 0.80) clearly outperformed the models based on the individual variables, including the model based on *IL28B* genotype alone (AUC 0.70). The addition of race and baseline viral load further improved the model, although the added gain was modest (AUC up to 0.85).

## DISCUSSION

This is the first study to combine the highly useful *IL28B* genotype with baseline IP-10 levels to demonstrate an independent and additive model for predicting SVR with pegIFN and ribavirin treatment. We demonstrated that low pretreatment serum IP-10 is associated with sustained virological response in both Caucasian and African-American HCV genotype 1 patients from the VIRAHEP-C cohort. Using pretreatment serum IP-10 (< or > 600 pg/ml) as a predictive biomarker for treatment response in our cohort revealed a PPV of 69% and a NPV of 67%. These results are in line with other studies confirming that pretreatment IP-10 is lower in patients who subsequently achieve SVR on therapy compared to non-responder patients ([15](#R15)–[20](#R20)). The prognostic utility of baseline IP-10 levels has also been confirmed in other difficult to treat populations such as HCV/HIV co-infection ([15](#R15)) and patients with an elevated viral load and BMI ([18](#R18)).

The current study greatly extends the potential clinical utility of IP-10 and refines the predictive value of *IL28B* gene polymorphisms. Based on five genome-wide association studies, single nucleotide polymorphisms predictive of both spontaneous and treatment-induced viral clearance in HCV genotype 1 have been identified near the *IL28B* gene ([21](#R21), [24](#R24)–[27](#R27)). Carriage of a C-allele at the *IL28B* gene polymorphism (rs12979860) is favorably associated with treatment response to pegIFN and ribavirin in both AA and CA patients ([21](#R21)). This was confirmed in our VIRAHEP-C cohort with SVR rates of 87% with CC, 50% with CT, and 39% with TT genotypes. Most striking, stratification by high or low baseline IP-10 (> or < 600 pg/ml) further improved the predictability of SVR among the genotype groups, especially in *IL28B* T-allele carriers. Multivariate analysis confirmed that *IL28B* genotype and baseline IP-10 levels are independent and additive predictors of HCV treatment response. Likewise, in a predictive model of SVR, serum IP-10 can be used as a “dynamic” variable which complements the “static” *IL-28B* genotype and further individualizes treatment response.

The polymorphisms on chromosome 19 associated with HCV treatment response are in the region which encodes the IFN-λ genes (*IL28A*, *IL28B* and *IL29*). The *IL28B* gene encodes IFN-λ-3 which has a unique signaling receptor as well as a common downstream signaling system with type I IFNs ([22](#R22), [23](#R23)). The role of IFN-λ in control of multiple viral infections including HCV is currently under study. IFN-λ has been used to block HCV replication *in vitro* ([28](#R28)), and human studies with IFN-λ-1 (*IL29*) yielded promising results as an antiviral agent against HCV with limited toxicity in phase 1B studies ([29](#R29)). Further research on the mechanism by which genetic variations near *IL28B* modulate innate immune responses via IFN-λ are ongoing.

It is unclear why high IP-10 levels are associated with nonresponse to HCV therapy. The IP-10 receptor (CXCR3) is upregulated on lymphocytes in chronic HCV and hepatocytes appear to be the predominant source of IP-10 in chronic infection ([16](#R16), [30](#R30), [31](#R31)). While intrahepatic IP-10 levels correlate with necroinflammatory changes and fibrosis in HCV ([30](#R30)), its role in viral clearance is less clear. Low pretreatment IP-10 levels are associated with a rapid decline in HCV viral load during the first 24–48 hours of interferon therapy ([31](#R31)). IP-10 gene expression is transiently elevated immediately post IFN injection in both AA and CA patients ([12](#R12)). In one study, the fold increase in IP-10 after the first pegIFN injection was associated with SVR ([15](#R15)). This is consistent with data that patients with low baseline levels of interferon-stimulated genes (ISG) appear to have a more robust response to exogenous pegIFN and a higher SVR rate. In contrast, those with high baseline ISG expression appear to be refractory to further IFN signaling ([13](#R13), [32](#R32)). High IP-10 levels may be a marker of this refractory state, or excess IP-10 may directly interfere with critical signaling pathways. Baseline hepatic ISG levels have been correlated with *IL28B* polymorphisms and treatment outcomes ([33](#R33)). In vivo, type III interferon IFN-λ-1 can induce IP-10 mRNA expression from PBMC in the absence of other stimuli and independent of type I IFNs ([34](#R34)). There may be a relationship between elevated IP-10 levels and resistance to antiviral effects of type I and type III IFNs which warrants further study.

Interestingly, our data show that at a given pretreatment IP-10 level, the probability of being a responder is also further determined by race. Race has an additive effect on the predictive models of both serum IP-10 and *IL28B* genotype but there is no statistical interaction between race, IP-10 and IL-28B (although allele frequency is race-dependent). This is in line with the observation that AA are generally less responsive to pegIFN and ribavirin treatment when compared to CA and that *IL28B* polymorphisms are not the only factor involved in treatment failure. Our findings confirm those of Butera et al that AA patients with chronic HCV have higher pretreatment IP-10 levels when compared to CA patients, albeit in a much smaller sample than our cohort ([16](#R16)). We did note that only 9% of our AA patients were *IL28B* genotype CC although the additive value for pretreatment IP-10 levels were most pronounced in the CT and TT genotypes in the combined cohort. A potential limitation of our study is that we only included sustained responders and non-responders to treatment, and no patients who relapsed after discontinuation of antiviral therapy (approximately 20% of the treatment population). However, in a previous study no distinct baseline IP-10 pattern associated with viral relapse was identified ([16](#R16)).

In summary, our study may have an impact on how patients with genotype 1 HCV are stratified before starting combination antiviral therapy. *IL28B* genotyping is currently being evaluated in prospective studies including triple therapy with small antiviral molecules combined with pegIFN and ribavirin. Our data show that pretreatment serum IP-10 is a strong positive predictor of SVR in both AA and CA genotype 1 patients, and significantly increases the predictive value of *IL28B* genotyping especially with the CT and TT genotypes. These two markers may prove useful in future algorithms for HCV treatment as pegIFN and ribavirin, both immune modulators, remain the backbone of therapy even with addition of small antiviral molecules. Patients lacking *IL28B* C-allele(s) (rs12979860) in combination with high IP-10 levels may require alteration of therapy type and/or duration. The utility of combining *IL28B* genotyping and baseline serum IP-10 levels to predict SVR warrants prospective validation.

## Supplementary Material

## Acknowledgments

**Financial Support and Conflict of Interest Declaration:**

Dr. Fried is supported in part by NIH Mid-Career Mentoring Award **K24 DK066144 (Fried PI)**. Drs. Aerssens, Fanning, Talloen, Bollekens, De Wit, and Scholliers are employees of Tibotec BVBA. Drs. Goldstein, Shianna, McHutchison and Thompson are co-inventors of an IL28B patent application. Drs. Fried, Fanning, and Aerssens are co-inventors on a patent application utilizing IP-10 and IL28B to predict antiviral response. Dr. Fried serves as an ad hoc consultant to Tibotec BVBA.

The VIRAHEP-C study was conducted by multiple investigators and supported by the National Institute of Digestive and Kidney Diseases (NIDDK). This manuscript was not prepared in collaboration with the investigators of VIRAHEP-C and does not necessarily reflect the opinions or views of the VIRAHEP-C study or the NIDDK. The authors would like to thank the VIRAHEP-C investigators and the NIDDK Biospecimen Repository.

## Abbreviations

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Liang TJ, Rehermann B, Seeff LB, Hoofnagle JH. Pathogenesis, natural history, treatment, and prevention of hepatitis C. Ann Intern Med. 2000;132:296–305. doi: 10.7326/0003-4819-132-4-200002150-00008.  [DOI](https://doi.org/10.7326/0003-4819-132-4-200002150-00008) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10681285/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Intern%20Med&title=Pathogenesis,%20natural%20history,%20treatment,%20and%20prevention%20of%20hepatitis%20C&author=TJ%20Liang&author=B%20Rehermann&author=LB%20Seeff&author=JH%20Hoofnagle&volume=132&publication_year=2000&pages=296-305&pmid=10681285&doi=10.7326/0003-4819-132-4-200002150-00008&)

2. Fried MW, Shiffman ML, Reddy KR, Smith C, Marinos G, Goncales FL, Haussinger D, et al. Peginterferon alfa-2a plus ribavirin for chronic hepatitis C virus infection. New England Journal of Medicine. 2002;347:975–982. doi: 10.1056/NEJMoa020047.  [DOI](https://doi.org/10.1056/NEJMoa020047) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12324553/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=New%20England%20Journal%20of%20Medicine&title=Peginterferon%20alfa-2a%20plus%20ribavirin%20for%20chronic%20hepatitis%20C%20virus%20infection&author=MW%20Fried&author=ML%20Shiffman&author=KR%20Reddy&author=C%20Smith&author=G%20Marinos&volume=347&publication_year=2002&pages=975-982&pmid=12324553&doi=10.1056/NEJMoa020047&)

3. Hadziyannis SJ, Sette H, Jr, Morgan TR, Balan V, Diago M, Marcellin P, Ramadori G, et al. Peginterferon-alpha2a and ribavirin combination therapy in chronic hepatitis C: a randomized study of treatment duration and ribavirin dose. Ann Intern Med. 2004;140:346–355. doi: 10.7326/0003-4819-140-5-200403020-00010.  [DOI](https://doi.org/10.7326/0003-4819-140-5-200403020-00010) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14996676/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Intern%20Med&title=Peginterferon-alpha2a%20and%20ribavirin%20combination%20therapy%20in%20chronic%20hepatitis%20C:%20a%20randomized%20study%20of%20treatment%20duration%20and%20ribavirin%20dose&author=SJ%20Hadziyannis&author=H%20Sette&author=TR%20Morgan&author=V%20Balan&author=M%20Diago&volume=140&publication_year=2004&pages=346-355&pmid=14996676&doi=10.7326/0003-4819-140-5-200403020-00010&)

4. Manns MP, McHutchison JG, Gordon SC, Rustgi VK, Shiffman M, Reindollar R, Goodman ZD, et al. Peginterferon alfa-2b plus ribavirin compared with interferon alfa-2b plus ribavirin for initial treatment of chronic hepatitis C: a randomised trial. Lancet. 2001;358:958–965. doi: 10.1016/s0140-6736(01)06102-5.  [DOI](https://doi.org/10.1016/s0140-6736(01)06102-5) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11583749/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Peginterferon%20alfa-2b%20plus%20ribavirin%20compared%20with%20interferon%20alfa-2b%20plus%20ribavirin%20for%20initial%20treatment%20of%20chronic%20hepatitis%20C:%20a%20randomised%20trial&author=MP%20Manns&author=JG%20McHutchison&author=SC%20Gordon&author=VK%20Rustgi&author=M%20Shiffman&volume=358&publication_year=2001&pages=958-965&pmid=11583749&doi=10.1016/s0140-6736(01)06102-5&)

5. Davis GL, Wong JB, McHutchison JG, Manns MP, Harvey J, Albrecht J. Early virologic response to treatment with peginterferon alfa-2b plus ribavirin in patients with chronic hepatitis C. Hepatology. 2003;38:645–652. doi: 10.1053/jhep.2003.50364.  [DOI](https://doi.org/10.1053/jhep.2003.50364) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12939591/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hepatology&title=Early%20virologic%20response%20to%20treatment%20with%20peginterferon%20alfa-2b%20plus%20ribavirin%20in%20patients%20with%20chronic%20hepatitis%20C&author=GL%20Davis&author=JB%20Wong&author=JG%20McHutchison&author=MP%20Manns&author=J%20Harvey&volume=38&publication_year=2003&pages=645-652&pmid=12939591&doi=10.1053/jhep.2003.50364&)

6. Ferenci P, Fried MW, Shiffman ML, Smith CI, Marinos G, Goncales FL, Jr, Haussinger D, et al. Predicting sustained virological responses in chronic hepatitis C patients treated with peginterferon alfa-2a (40 KD)/ribavirin. J Hepatol. 2005;43:425–433. doi: 10.1016/j.jhep.2005.04.009.  [DOI](https://doi.org/10.1016/j.jhep.2005.04.009) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15990196/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Hepatol&title=Predicting%20sustained%20virological%20responses%20in%20chronic%20hepatitis%20C%20patients%20treated%20with%20peginterferon%20alfa-2a%20(40%20KD)/ribavirin&author=P%20Ferenci&author=MW%20Fried&author=ML%20Shiffman&author=CI%20Smith&author=G%20Marinos&volume=43&publication_year=2005&pages=425-433&pmid=15990196&doi=10.1016/j.jhep.2005.04.009&)

7. Jeffers LJ, Cassidy W, Howell CD, Hu S, Reddy KR. Peginterferon alfa-2a (40 kd) and ribavirin for black American patients with chronic HCV genotype 1. Hepatology. 2004;39:1702–1708. doi: 10.1002/hep.20212.  [DOI](https://doi.org/10.1002/hep.20212) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15185312/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hepatology&title=Peginterferon%20alfa-2a%20(40%20kd)%20and%20ribavirin%20for%20black%20American%20patients%20with%20chronic%20HCV%20genotype%201&author=LJ%20Jeffers&author=W%20Cassidy&author=CD%20Howell&author=S%20Hu&author=KR%20Reddy&volume=39&publication_year=2004&pages=1702-1708&pmid=15185312&doi=10.1002/hep.20212&)

8. Conjeevaram HS, Fried MW, Jeffers LJ, Terrault NA, Wiley-Lucas TE, Afdhal N, Brown RS, et al. Peginterferon and ribavirin treatment in African American and Caucasian American patients with hepatitis C genotype 1. Gastroenterology. 2006;131:470–477. doi: 10.1053/j.gastro.2006.06.008.  [DOI](https://doi.org/10.1053/j.gastro.2006.06.008) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16890601/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gastroenterology&title=Peginterferon%20and%20ribavirin%20treatment%20in%20African%20American%20and%20Caucasian%20American%20patients%20with%20hepatitis%20C%20genotype%201&author=HS%20Conjeevaram&author=MW%20Fried&author=LJ%20Jeffers&author=NA%20Terrault&author=TE%20Wiley-Lucas&volume=131&publication_year=2006&pages=470-477&pmid=16890601&doi=10.1053/j.gastro.2006.06.008&)

9. Muir AJ, Bornstein JD, Killenberg PG. Peginterferon alfa-2b and ribavirin for the treatment of chronic hepatitis C in blacks and non-Hispanic whites. N Engl J Med. 2004;350:2265–2271. doi: 10.1056/NEJMoa032502.  [DOI](https://doi.org/10.1056/NEJMoa032502) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15163776/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Peginterferon%20alfa-2b%20and%20ribavirin%20for%20the%20treatment%20of%20chronic%20hepatitis%20C%20in%20blacks%20and%20non-Hispanic%20whites&author=AJ%20Muir&author=JD%20Bornstein&author=PG%20Killenberg&volume=350&publication_year=2004&pages=2265-2271&pmid=15163776&doi=10.1056/NEJMoa032502&)

10. Conjeevaram HS, Kleiner DE, Everhart JE, Hoofnagle JH, Zacks S, Afdhal NH, Wahed AS. Race, insulin resistance and hepatic steatosis in chronic hepatitis C. Hepatology. 2007;45:80–87. doi: 10.1002/hep.21455.  [DOI](https://doi.org/10.1002/hep.21455) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17187406/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hepatology&title=Race,%20insulin%20resistance%20and%20hepatic%20steatosis%20in%20chronic%20hepatitis%20C&author=HS%20Conjeevaram&author=DE%20Kleiner&author=JE%20Everhart&author=JH%20Hoofnagle&author=S%20Zacks&volume=45&publication_year=2007&pages=80-87&pmid=17187406&doi=10.1002/hep.21455&)

11. Rosen HR, Weston SJ, Im K, Yang H, Burton JR, Jr, Erlich H, Klarquist J, et al. Selective decrease in hepatitis C virus-specific immunity among African Americans and outcome of antiviral therapy. Hepatology. 2007;46:350–358. doi: 10.1002/hep.21714.  [DOI](https://doi.org/10.1002/hep.21714) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17659573/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hepatology&title=Selective%20decrease%20in%20hepatitis%20C%20virus-specific%20immunity%20among%20African%20Americans%20and%20outcome%20of%20antiviral%20therapy&author=HR%20Rosen&author=SJ%20Weston&author=K%20Im&author=H%20Yang&author=JR%20Burton&volume=46&publication_year=2007&pages=350-358&pmid=17659573&doi=10.1002/hep.21714&)

12. Taylor MW, Tsukahara T, Brodsky L, Schaley J, Sanda C, Stephens MJ, McClintick JN, et al. Changes in gene expression during pegylated interferon and ribavirin therapy of chronic hepatitis C virus distinguish responders from nonresponders to antiviral therapy. J Virol. 2007;81:3391–3401. doi: 10.1128/JVI.02640-06.  [DOI](https://doi.org/10.1128/JVI.02640-06) | [PMC free article](/articles/PMC1866036/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17267482/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Virol&title=Changes%20in%20gene%20expression%20during%20pegylated%20interferon%20and%20ribavirin%20therapy%20of%20chronic%20hepatitis%20C%20virus%20distinguish%20responders%20from%20nonresponders%20to%20antiviral%20therapy&author=MW%20Taylor&author=T%20Tsukahara&author=L%20Brodsky&author=J%20Schaley&author=C%20Sanda&volume=81&publication_year=2007&pages=3391-3401&pmid=17267482&doi=10.1128/JVI.02640-06&)

13. Sarasin-Filipowicz M, Oakeley EJ, Duong FH, Christen V, Terracciano L, Filipowicz W, Heim MH. Interferon signaling and treatment outcome in chronic hepatitis C. Proc Natl Acad Sci U S A. 2008;105:7034–7039. doi: 10.1073/pnas.0707882105.  [DOI](https://doi.org/10.1073/pnas.0707882105) | [PMC free article](/articles/PMC2383932/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18467494/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&title=Interferon%20signaling%20and%20treatment%20outcome%20in%20chronic%20hepatitis%20C&author=M%20Sarasin-Filipowicz&author=EJ%20Oakeley&author=FH%20Duong&author=V%20Christen&author=L%20Terracciano&volume=105&publication_year=2008&pages=7034-7039&pmid=18467494&doi=10.1073/pnas.0707882105&)

14. He XS, Ji X, Hale MB, Cheung R, Ahmed A, Guo Y, Nolan GP, et al. Global transcriptional response to interferon is a determinant of HCV treatment outcome and is modified by race. Hepatology. 2006;44:352–359. doi: 10.1002/hep.21267.  [DOI](https://doi.org/10.1002/hep.21267) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16871572/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hepatology&title=Global%20transcriptional%20response%20to%20interferon%20is%20a%20determinant%20of%20HCV%20treatment%20outcome%20and%20is%20modified%20by%20race&author=XS%20He&author=X%20Ji&author=MB%20Hale&author=R%20Cheung&author=A%20Ahmed&volume=44&publication_year=2006&pages=352-359&pmid=16871572&doi=10.1002/hep.21267&)

15. Zeremski M, Markatou M, Brown QB, Dorante G, Cunningham-Rundles S, Talal AH. Interferon gamma-inducible protein 10: a predictive marker of successful treatment response in hepatitis C virus/HIV-coinfected patients. J Acquir Immune Defic Syndr. 2007;45:262–268. doi: 10.1097/QAI.0b013e3180559219.  [DOI](https://doi.org/10.1097/QAI.0b013e3180559219) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17414926/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Acquir%20Immune%20Defic%20Syndr&title=Interferon%20gamma-inducible%20protein%2010:%20a%20predictive%20marker%20of%20successful%20treatment%20response%20in%20hepatitis%20C%20virus/HIV-coinfected%20patients&author=M%20Zeremski&author=M%20Markatou&author=QB%20Brown&author=G%20Dorante&author=S%20Cunningham-Rundles&volume=45&publication_year=2007&pages=262-268&pmid=17414926&doi=10.1097/QAI.0b013e3180559219&)

16. Butera D, Marukian S, Iwamaye AE, Hembrador E, Chambers TJ, Di Bisceglie AM, Charles ED, et al. Plasma chemokine levels correlate with the outcome of antiviral therapy in patients with hepatitis C. Blood. 2005;106:1175–1182. doi: 10.1182/blood-2005-01-0126.  [DOI](https://doi.org/10.1182/blood-2005-01-0126) | [PMC free article](/articles/PMC1895193/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15860662/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Plasma%20chemokine%20levels%20correlate%20with%20the%20outcome%20of%20antiviral%20therapy%20in%20patients%20with%20hepatitis%20C&author=D%20Butera&author=S%20Marukian&author=AE%20Iwamaye&author=E%20Hembrador&author=TJ%20Chambers&volume=106&publication_year=2005&pages=1175-1182&pmid=15860662&doi=10.1182/blood-2005-01-0126&)

17. Diago M, Castellano G, Garcia-Samaniego J, Perez C, Fernandez I, Romero M, Iacono OL, et al. Association of pretreatment serum interferon gamma inducible protein 10 levels with sustained virological response to peginterferon plus ribavirin therapy in genotype 1 infected patients with chronic hepatitis C. Gut. 2006;55:374–379. doi: 10.1136/gut.2005.074062.  [DOI](https://doi.org/10.1136/gut.2005.074062) | [PMC free article](/articles/PMC1856069/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16150856/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gut&title=Association%20of%20pretreatment%20serum%20interferon%20gamma%20inducible%20protein%2010%20levels%20with%20sustained%20virological%20response%20to%20peginterferon%20plus%20ribavirin%20therapy%20in%20genotype%201%20infected%20patients%20with%20chronic%20hepatitis%20C&author=M%20Diago&author=G%20Castellano&author=J%20Garcia-Samaniego&author=C%20Perez&author=I%20Fernandez&volume=55&publication_year=2006&pages=374-379&pmid=16150856&doi=10.1136/gut.2005.074062&)

18. Lagging M, Romero AI, Westin J, Norkrans G, Dhillon AP, Pawlotsky JM, Zeuzem S, et al. IP-10 predicts viral response and therapeutic outcome in difficult-to-treat patients with HCV genotype 1 infection. Hepatology. 2006;44:1617–1625. doi: 10.1002/hep.21407.  [DOI](https://doi.org/10.1002/hep.21407) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17133471/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hepatology&title=IP-10%20predicts%20viral%20response%20and%20therapeutic%20outcome%20in%20difficult-to-treat%20patients%20with%20HCV%20genotype%201%20infection&author=M%20Lagging&author=AI%20Romero&author=J%20Westin&author=G%20Norkrans&author=AP%20Dhillon&volume=44&publication_year=2006&pages=1617-1625&pmid=17133471&doi=10.1002/hep.21407&)

19. Narumi S, Tominaga Y, Tamaru M, Shimai S, Okumura H, Nishioji K, Itoh Y, et al. Expression of IFN-inducible protein-10 in chronic hepatitis. J Immunol. 1997;158:5536–5544.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/9164978/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Immunol&title=Expression%20of%20IFN-inducible%20protein-10%20in%20chronic%20hepatitis&author=S%20Narumi&author=Y%20Tominaga&author=M%20Tamaru&author=S%20Shimai&author=H%20Okumura&volume=158&publication_year=1997&pages=5536-5544&pmid=9164978&)

20. Romero AI, Lagging M, Westin J, Dhillon AP, Dustin LB, Pawlotsky JM, Neumann AU, et al. Interferon (IFN)-gamma-inducible protein-10: association with histological results, viral kinetics, and outcome during treatment with pegylated IFN-alpha 2a and ribavirin for chronic hepatitis C virus infection. J Infect Dis. 2006;194:895–903. doi: 10.1086/507307.  [DOI](https://doi.org/10.1086/507307) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16960776/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Infect%20Dis&title=Interferon%20(IFN)-gamma-inducible%20protein-10:%20association%20with%20histological%20results,%20viral%20kinetics,%20and%20outcome%20during%20treatment%20with%20pegylated%20IFN-alpha%202a%20and%20ribavirin%20for%20chronic%20hepatitis%20C%20virus%20infection&author=AI%20Romero&author=M%20Lagging&author=J%20Westin&author=AP%20Dhillon&author=LB%20Dustin&volume=194&publication_year=2006&pages=895-903&pmid=16960776&doi=10.1086/507307&)

21. Ge D, Fellay J, Thompson AJ, Simon JS, Shianna KV, Urban TJ, Heinzen EL, et al. Genetic variation in IL28B predicts hepatitis C treatment-induced viral clearance. Nature. 2009;461:399–401. doi: 10.1038/nature08309.  [DOI](https://doi.org/10.1038/nature08309) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19684573/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Genetic%20variation%20in%20IL28B%20predicts%20hepatitis%20C%20treatment-induced%20viral%20clearance&author=D%20Ge&author=J%20Fellay&author=AJ%20Thompson&author=JS%20Simon&author=KV%20Shianna&volume=461&publication_year=2009&pages=399-401&pmid=19684573&doi=10.1038/nature08309&)

22. Kotenko SV, Gallagher G, Baurin VV, Lewis-Antes A, Shen M, Shah NK, Langer JA, et al. IFN-lambdas mediate antiviral protection through a distinct class II cytokine receptor complex. Nat Immunol. 2003;4:69–77. doi: 10.1038/ni875.  [DOI](https://doi.org/10.1038/ni875) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12483210/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Immunol&title=IFN-lambdas%20mediate%20antiviral%20protection%20through%20a%20distinct%20class%20II%20cytokine%20receptor%20complex&author=SV%20Kotenko&author=G%20Gallagher&author=VV%20Baurin&author=A%20Lewis-Antes&author=M%20Shen&volume=4&publication_year=2003&pages=69-77&pmid=12483210&doi=10.1038/ni875&)

23. Sheppard P, Kindsvogel W, Xu W, Henderson K, Schlutsmeyer S, Whitmore TE, Kuestner R, et al. IL-28, IL-29 and their class II cytokine receptor IL-28R. Nat Immunol. 2003;4:63–68. doi: 10.1038/ni873.  [DOI](https://doi.org/10.1038/ni873) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12469119/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Immunol&title=IL-28,%20IL-29%20and%20their%20class%20II%20cytokine%20receptor%20IL-28R&author=P%20Sheppard&author=W%20Kindsvogel&author=W%20Xu&author=K%20Henderson&author=S%20Schlutsmeyer&volume=4&publication_year=2003&pages=63-68&pmid=12469119&doi=10.1038/ni873&)

24. Rauch A, Kutalik Z, Descombes P, Cai T, Di Iulio J, Mueller T, Bochud M, et al. Genetic variation in IL28B is associated with chronic hepatitis C and treatment failure: a genome-wide association study. Gastroenterology. 138:1338–1345. doi: 10.1053/j.gastro.2009.12.056. 1345 e1331-1337.  [DOI](https://doi.org/10.1053/j.gastro.2009.12.056) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20060832/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gastroenterology&title=Genetic%20variation%20in%20IL28B%20is%20associated%20with%20chronic%20hepatitis%20C%20and%20treatment%20failure:%20a%20genome-wide%20association%20study&author=A%20Rauch&author=Z%20Kutalik&author=P%20Descombes&author=T%20Cai&author=J%20Di%20Iulio&volume=138&pages=1338-1345&pmid=20060832&doi=10.1053/j.gastro.2009.12.056&)

25. Suppiah V, Moldovan M, Ahlenstiel G, Berg T, Weltman M, Abate ML, Bassendine M, et al. IL28B is associated with response to chronic hepatitis C interferon-alpha and ribavirin therapy. Nat Genet. 2009;41:1100–1104. doi: 10.1038/ng.447.  [DOI](https://doi.org/10.1038/ng.447) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19749758/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=IL28B%20is%20associated%20with%20response%20to%20chronic%20hepatitis%20C%20interferon-alpha%20and%20ribavirin%20therapy&author=V%20Suppiah&author=M%20Moldovan&author=G%20Ahlenstiel&author=T%20Berg&author=M%20Weltman&volume=41&publication_year=2009&pages=1100-1104&pmid=19749758&doi=10.1038/ng.447&)

26. Tanaka Y, Nishida N, Sugiyama M, Kurosaki M, Matsuura K, Sakamoto N, Nakagawa M, et al. Genome-wide association of IL28B with response to pegylated interferon-alpha and ribavirin therapy for chronic hepatitis C. Nat Genet. 2009;41:1105–1109. doi: 10.1038/ng.449.  [DOI](https://doi.org/10.1038/ng.449) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19749757/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Genome-wide%20association%20of%20IL28B%20with%20response%20to%20pegylated%20interferon-alpha%20and%20ribavirin%20therapy%20for%20chronic%20hepatitis%20C&author=Y%20Tanaka&author=N%20Nishida&author=M%20Sugiyama&author=M%20Kurosaki&author=K%20Matsuura&volume=41&publication_year=2009&pages=1105-1109&pmid=19749757&doi=10.1038/ng.449&)

27. Thomas DL, Thio CL, Martin MP, Qi Y, Ge D, O'Huigin C, Kidd J, et al. Genetic variation in IL28B and spontaneous clearance of hepatitis C virus. Nature. 2009;461:798–801. doi: 10.1038/nature08463.  [DOI](https://doi.org/10.1038/nature08463) | [PMC free article](/articles/PMC3172006/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19759533/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Genetic%20variation%20in%20IL28B%20and%20spontaneous%20clearance%20of%20hepatitis%20C%20virus&author=DL%20Thomas&author=CL%20Thio&author=MP%20Martin&author=Y%20Qi&author=D%20Ge&volume=461&publication_year=2009&pages=798-801&pmid=19759533&doi=10.1038/nature08463&)

28. Marcello T, Grakoui A, Barba-Spaeth G, Machlin ES, Kotenko SV, MacDonald MR, Rice CM. Interferons alpha and lambda inhibit hepatitis C virus replication with distinct signal transduction and gene regulation kinetics. Gastroenterology. 2006;131:1887–1898. doi: 10.1053/j.gastro.2006.09.052.  [DOI](https://doi.org/10.1053/j.gastro.2006.09.052) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17087946/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gastroenterology&title=Interferons%20alpha%20and%20lambda%20inhibit%20hepatitis%20C%20virus%20replication%20with%20distinct%20signal%20transduction%20and%20gene%20regulation%20kinetics&author=T%20Marcello&author=A%20Grakoui&author=G%20Barba-Spaeth&author=ES%20Machlin&author=SV%20Kotenko&volume=131&publication_year=2006&pages=1887-1898&pmid=17087946&doi=10.1053/j.gastro.2006.09.052&)

29. Shiffman M, Lawitz E, Zaman A, Vierling J, Yoffe B, Freeman J, Gray T, et al. PEG-IFN-l: Antiviral Activity and Safety Profile in a 4-week Phase 1B Study in Relapsed Genotype 1 Hepatitis C Infection. J Hepatol. 2009;50:a643. (s237)  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Hepatol&title=PEG-IFN-l:%20Antiviral%20Activity%20and%20Safety%20Profile%20in%20a%204-week%20Phase%201B%20Study%20in%20Relapsed%20Genotype%201%20Hepatitis%20C%20Infection&author=M%20Shiffman&author=E%20Lawitz&author=A%20Zaman&author=J%20Vierling&author=B%20Yoffe&volume=50&publication_year=2009&pages=a643&)

30. Zeremski M, Petrovic LM, Chiriboga L, Brown QB, Yee HT, Kinkhabwala M, Jacobson IM, et al. Intrahepatic levels of CXCR3-associated chemokines correlate with liver inflammation and fibrosis in chronic hepatitis C. Hepatology. 2008;48:1440–1450. doi: 10.1002/hep.22500.  [DOI](https://doi.org/10.1002/hep.22500) | [PMC free article](/articles/PMC2579317/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18798334/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hepatology&title=Intrahepatic%20levels%20of%20CXCR3-associated%20chemokines%20correlate%20with%20liver%20inflammation%20and%20fibrosis%20in%20chronic%20hepatitis%20C&author=M%20Zeremski&author=LM%20Petrovic&author=L%20Chiriboga&author=QB%20Brown&author=HT%20Yee&volume=48&publication_year=2008&pages=1440-1450&pmid=18798334&doi=10.1002/hep.22500&)

31. Askarieh G, Alsio A, Pugnale P, Negro F, Ferrari C, Neumann AU, Pawlotsky JM, et al. Systemic and intrahepatic interferon-gamma-inducible protein 10 kDa predicts the first-phase decline in hepatitis C virus RNA and overall viral response to therapy in chronic hepatitis C. Hepatology. 51:1523–1530. doi: 10.1002/hep.23509.  [DOI](https://doi.org/10.1002/hep.23509) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20186843/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hepatology&title=Systemic%20and%20intrahepatic%20interferon-gamma-inducible%20protein%2010%20kDa%20predicts%20the%20first-phase%20decline%20in%20hepatitis%20C%20virus%20RNA%20and%20overall%20viral%20response%20to%20therapy%20in%20chronic%20hepatitis%20C&author=G%20Askarieh&author=A%20Alsio&author=P%20Pugnale&author=F%20Negro&author=C%20Ferrari&volume=51&pages=1523-1530&pmid=20186843&doi=10.1002/hep.23509&)

32. Feld JJ, Nanda S, Huang Y, Chen W, Cam M, Pusek SN, Schweigler LM, et al. Hepatic gene expression during treatment with peginterferon and ribavirin: Identifying molecular pathways for treatment response. Hepatology. 2007;46:1548–1563. doi: 10.1002/hep.21853.  [DOI](https://doi.org/10.1002/hep.21853) | [PMC free article](/articles/PMC2808168/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17929300/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hepatology&title=Hepatic%20gene%20expression%20during%20treatment%20with%20peginterferon%20and%20ribavirin:%20Identifying%20molecular%20pathways%20for%20treatment%20response&author=JJ%20Feld&author=S%20Nanda&author=Y%20Huang&author=W%20Chen&author=M%20Cam&volume=46&publication_year=2007&pages=1548-1563&pmid=17929300&doi=10.1002/hep.21853&)

33. Honda M, Sakai A, Yamashita T, Nakamoto Y, Mizukoshi E, Sakai Y, Nakamura M, et al. Hepatic ISG expression is associated with genetic variation in interleukin 28B and the outcome of IFN therapy for chronic hepatitis C. Gastroenterology. 139:499–509. doi: 10.1053/j.gastro.2010.04.049.  [DOI](https://doi.org/10.1053/j.gastro.2010.04.049) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20434452/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gastroenterology&title=Hepatic%20ISG%20expression%20is%20associated%20with%20genetic%20variation%20in%20interleukin%2028B%20and%20the%20outcome%20of%20IFN%20therapy%20for%20chronic%20hepatitis%20C&author=M%20Honda&author=A%20Sakai&author=T%20Yamashita&author=Y%20Nakamoto&author=E%20Mizukoshi&volume=139&pages=499-509&pmid=20434452&doi=10.1053/j.gastro.2010.04.049&)

34. Pekarek V, Srinivas S, Eskdale J, Gallagher G. Interferon lambda-1 (IFN-lambda1/IL-29) induces ELR(-) CXC chemokine mRNA in human peripheral blood mononuclear cells, in an IFN-gamma-independent manner. Genes Immun. 2007;8:177–180. doi: 10.1038/sj.gene.6364372.  [DOI](https://doi.org/10.1038/sj.gene.6364372) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17252004/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genes%20Immun&title=Interferon%20lambda-1%20(IFN-lambda1/IL-29)%20induces%20ELR(-)%20CXC%20chemokine%20mRNA%20in%20human%20peripheral%20blood%20mononuclear%20cells,%20in%20an%20IFN-gamma-independent%20manner&author=V%20Pekarek&author=S%20Srinivas&author=J%20Eskdale&author=G%20Gallagher&volume=8&publication_year=2007&pages=177-180&pmid=17252004&doi=10.1038/sj.gene.6364372&)
