# The role of KRAS rs61764370 in invasive epithelial ovarian cancer: implications for clinical testing

## Metadata
**Authors:** Paul D P Pharoah, Rachel T Palmieri, Susan J Ramus, Simon A Gayther, Irene L Andrulis, Hoda Anton-Culver, Natalia Antonenkova, Antonis C Antoniou, BCFR Investigators, Mary S Beattie, Matthias W Beckmann, Michael J Birrer, Natalia Bogdanova, Kelly L Bolton, Wendy Brewster, Angela Brooks-Wilson, Robert Brown, Ralf Butzow, Trinidad Caldes, Maria Adelaide Caligo, Ian Campbell, Jenny Chang-Claude, Y Ann Chen, Georgia Chenevix-Trench, Linda S Cook, Fergus J Couch, Daniel W Cramer, Julie M Cunningham, Evelyn Despierre, Jennifer A Doherty, Thilo Dörk, Matthias Dürst, Diana M Eccles, Arif B Ekici, EMBRACE Investigators, Peter A Fasching, Anna de Fazio, David A Fenstermacher, James M Flanagan, Brooke L Fridley, Eitan Friedman, Bo Gao, GEMO Study Collaborators, Aleksandra Gentry-Maharaj, Andrew K Godwin, Ellen L Goode, Marc T Goodman, Jenny Gross, Thomas V O Hansen, Paul Harnett, HEBON Investigators, Tuomas Heikkinen, Rebecca Hein, Claus Høgdall, Estrid Høgdall, Edwin S Iversen, Anna Jakubowska, Sharon E Johnatty, Beth Y Karlan, Noah D Kauff, Stanley B Kaye, kConFab Investigators, Linda E Kelemen, Lambertus A Kiemeney, Susanne Krüger Kjaer, Diether Lambrechts, James P LaPolla, Conxi Lázaro, Nhu D Le, Arto Leminen, Karin Leunen, Douglas A Levine, Yi Lu, Lene Lundvall, Stuart Macgregor, Tamara Marees, Leon F Massuger, John R McLaughlin, Usha Menon, Marco Montagna, Kirsten B Moysich, Steven A Narod, Katherine L Nathanson, Lotte Nedergaard, Roberta B Ness, Heli Nevanlinna, Stefan Nickels, Ana Osorio, Jim Paul, Celeste Leigh Pearce, Catherine M Phelan, Malcolm C Pike, Paolo Radice, Mary Anne Rossing, Joellen M Schildkraut, Thomas A Sellers, Christian F Singer, Honglin Song, Daniel O Stram, Rebecca Sutphen
**Journal:** Clinical cancer research : an official journal of the American Association for Cancer Research
**Date:** 2011 Mar 8
**DOI:** [10.1158/1078-0432.CCR-10-3405](https://doi.org/10.1158/1078-0432.CCR-10-3405)
**PMID:** 21385923
**PMCID:** PMC3107901
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3107901/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC3107901/pdf/nihms271454.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC3107901/pdf/nihms271454.pdf)

## Abstract

**Purpose:** 
An assay for the single nucleotide polymorphism (SNP) rs61764370 has recently been commercially marketed as a clinical test to aid ovarian cancer risk evaluation in women with family histories of the disease. rs67164370 is in a 3′UTR miRNA binding site of the KRAS oncogene, and is a candidate for epithelial ovarian cancer (EOC) susceptibility. However, only one published paper, analyzing fewer than 1,000 subjects in total, has examined this association.

**Experimental Design:** 
Risk association was evaluated in 8,669 cases of invasive EOC and 10,012 controls from nineteen studies participating in the Ovarian Cancer Association Consortium, and in 683 cases and 2,044 controls carrying BRCA1 mutations from studies in the Consortium of Investigators of Modifiers of BRCA1/2. Prognosis association was also examined in a subset of five studies with progression-free survival data and eighteen studies with all-cause mortality data.

**Results:** 
No evidence of association was observed between genotype and risk of unselected EOC (odds ratio (OR)=1.02, 95% confidence interval (CI)=0.95–1.10), serous EOC (OR=1.08, 95%CI=0.98–1.18), familial EOC (OR=1.09, 95%CI=0.78–1.54), or among women carrying deleterious mutations in BRCA1 (OR=1.09, 95%CI=0.88–1.36). There was little evidence for association with survival time among unselected cases (hazard ratio (HR)=1.10, 95%CI=0.99–1.22), among serous cases (HR=1.12, 95%CI=0.99–1.28), or with progression-free survival in 540 cases treated with carboplatin and paclitaxel (HR=1.18, 95%CI=0.93–1.52).

**Conclusions:** 
These data exclude the possibility of an association between rs61764370 and a clinically significant risk of ovarian cancer or of familial ovarian cancer. Use of this SNP for ovarian cancer clinical risk prediction therefore appears unwarranted.

### Purpose

An assay for the single nucleotide polymorphism (SNP) rs61764370 has recently been commercially marketed as a clinical test to aid ovarian cancer risk evaluation in women with family histories of the disease. rs67164370 is in a 3′UTR miRNA binding site of the *KRAS* oncogene, and is a candidate for epithelial ovarian cancer (EOC) susceptibility. However, only one published paper, analyzing fewer than 1,000 subjects in total, has examined this association.

### Experimental Design

Risk association was evaluated in 8,669 cases of invasive EOC and 10,012 controls from nineteen studies participating in the Ovarian Cancer Association Consortium, and in 683 cases and 2,044 controls carrying *BRCA1* mutations from studies in the Consortium of Investigators of Modifiers of *BRCA1/2*. Prognosis association was also examined in a subset of five studies with progression-free survival data and eighteen studies with all-cause mortality data.

### Results

No evidence of association was observed between genotype and risk of unselected EOC (odds ratio (OR)=1.02, 95% confidence interval (CI)=0.95–1.10), serous EOC (OR=1.08, 95%CI=0.98–1.18), familial EOC (OR=1.09, 95%CI=0.78–1.54), or among women carrying deleterious mutations in *BRCA1* (OR=1.09, 95%CI=0.88–1.36). There was little evidence for association with survival time among unselected cases (hazard ratio (HR)=1.10, 95%CI=0.99–1.22), among serous cases (HR=1.12, 95%CI=0.99–1.28), or with progression-free survival in 540 cases treated with carboplatin and paclitaxel (HR=1.18, 95%CI=0.93–1.52).

### Conclusions

These data exclude the possibility of an association between rs61764370 and a clinically significant risk of ovarian cancer or of familial ovarian cancer. Use of this SNP for ovarian cancer clinical risk prediction therefore appears unwarranted.

## INTRODUCTION

Epithelial ovarian cancer (EOC) is the fifth-most common cancer in women. It generally presents as advanced disease with poor prognosis. Family and twin studies have suggested that inherited genetic variation plays an appreciable part in determining individual risk. However, until recently, knowledge of genetic susceptibility was limited to rare, highly-penetrant alleles in a handful of genes including *BRCA1*, *BRCA2* and the mismatch repair genes ([1](#R1)). In the past two years, genome-wide association studies have identified common susceptibility alleles at four loci at highly stringent levels of statistical significance (P<10^−8^), but these alleles have small effects on disease risk (per-allele OR<1.3) and explain a small fraction of the genetic component of disease risk ([2](#R2)–[4](#R4)). Many candidate gene studies have identified possible common ovarian cancer susceptibility alleles, but most are likely to represent false-positive associations as none have been reported at the levels of statistical significance required when testing hypotheses with low prior probabilities of association ([1](#R1)).

In July 2010, a single nucleotide polymorphism (SNP), rs61764370, located in the 3′UTR of the *KRAS* oncogene, was reported to be associated with risk of unselected epithelial ovarian cancer ([5](#R5)). The variant was also reported to be associated with a stronger risk in women carrying *BRCA1* mutations, in women not carrying *BRCA1* or *BRCA2* mutations but with a family history of the disease, as well as associated with poorer progression-free survival ([5](#R5)). This SNP was thought to be a strong candidate for cancer risk as it lies in a miRNA binding site, and associations between miRNA mutations or mis-expression and risk of some human cancers have been seen. These observations suggested that miRNAs can function as tumor suppressors or oncogenes ([6](#R6)). An assay to determine genotype at rs61764370 has subsequently been marketed as a commercial test to determine risk in women with a family history of ovarian cancer ([http://www.miradx.com](http://www.miradx.com)). However, as with other candidate gene studies, the reported association was not at a level of statistical significance that is regarded as definitive for common susceptibility alleles ([7](#R7)), nor was the magnitude of risk sufficient for this SNP to be acceptable as a useful clinical marker of ovarian cancer risk. The present work therefore sought to: i) replicate the association in a robust manner in multiple study populations genotyped to a high standard with stringent quality assurance procedures; ii) assess the association between genotype at this locus and ovarian cancer risk in women with family histories or who carry deleterious mutations in *BRCA1*; and iii) examine the hypothesis that the SNP is associated with differences in post-diagnosis progression-free survival or all-cause mortality.

## METHODS

### Study populations

Nineteen ovarian cancer case-control sets and one case series participating in the Ovarian Cancer Association Consortium (OCAC), and one additional case series, contributed data to the analyses ([Table 1](#T1)). Three of the case-control sets were each comprised of a case series matched to controls from the same geographical region: PVM, UK2 and UK-GWAS. Survival time analysis was based on data from eighteen case series including the additional publicly available data for 359 ovarian cancer cases from The Cancer Genome Atlas ([http://cancergenome.nih.gov/](http://cancergenome.nih.gov/)) that had information on all-cause mortality. The analysis of progression-free survival was based on data from five case series. Finally, data from 683 cases and 2,044 controls enrolled in a stage I project of the Consortium of Investigators of Modifiers of *BRCA1/2* (see reference ([8](#R8)) for details of studies participating in CIMBA) were used to examine risk among women carrying deleterious *BRCA1* mutations. Each study was approved by a governing research ethics committee and all study subjects provided written informed consent. Clinical and questionnaire data included tumor behavior, histology, stage and grade, age at diagnosis (or at comparable date for controls), family history of ovarian cancer, and ethnicity/race.

### Table 1.

| Analysis set | Study name | Study abbrev. | Study population | Study type | Number of subjects |  | Genotyping method |
| --- | --- | --- | --- | --- | --- | --- | --- |
| Controls | Cases |  |  |  |  |  |  |
| Case-control sets |  |  |  |  |  |  |  |
| BEL | Belgium Ovarian Cancer Study | BELa | Belgium | Hospital based | 253 | 173 | Fluidigm |
|   |  |  |  |  |  |  |  |
| BWH | Brigham Women’s Hospital Study | BWH | Boston, USA | Population based | 142 | 137 | Illumina Hap317 |
|   |  |  |  |  |  |  |  |
| DOV | Diseases of the Ovary and their Evaluation Study | DOVb | Washington State, USA | Population based | 721 | 698 | Taqman |
|   |  |  |  |  |  |  |  |
| GER | German Ovarian Cancer Study | GERb | Germany | Population based | 265 | 213 | Fluidigm |
|   |  |  |  |  |  |  |  |
| HMO | Hannover-Minsk Ovarian Cancer Study | HMO | Belarus | Hospital based | 151 | 195 | Fluidigm |
|   |  |  |  |  |  |  |  |
| HJO | Hannover-Jena Ovarian Cancer Study | HJO | Germany | Hospital based | 426 | 259 | Fluidigm |
|   |  |  |  |  |  |  |  |
| HOC | Helsinki Ovarian Cancer Study | HOC | Finland | Population based | 434 | 350 | Fluidigm |
|   |  |  |  |  |  |  |  |
| HOP | Hormones and Ovarian Cancer Prediction Study | HOPb | Pittsburgh, USA | Population based | 368 | 365 | Taqman |
|   |  |  |  |  |  |  |  |
| MAY | Mayo Clinic Ovarian Cancer Study | MAYa | Upper Midwest, USA | Hospital based | 520 | 358 | Illumina 610 Quad |
|   |  |  |  |  |  |  |  |
| NCO | North Carolina Ovarian Cancer Study | NCOb | North Carolina, USA | Population based | 655 | 494 | Illumina 610 Quad |
|   |  |  |  |  |  |  |  |
| NTH | Nijmegen Ovarian Cancer Study | NTH | Netherlands | Population based | 327 | 296 | Fluidigm |
|   |  |  |  |  |  |  |  |
| OVA | Ovarian Cancer Study | OVA | Alberta and British Columbia, Canada | Population based | 416 | 494 | Fluidigm |
|   |  |  |  |  |  |  |  |
| PVM | Pelvic Mass Study | PVDb | Copenhagen, Denmark | Hospital based |   | 201 | Fluidigm |
| Malignant Ovarian Cancer Study | MAL | Copenhagen, Denmark | Population based | 215 |   | Fluidigm |  |
|   |  |  |  |  |  |  |  |
| TBO | Tampa Bay Ovarian Cancer Study | TBOb | Tampa, USA | Population based | 168 | 227 | Illumina 610 Quad |
|   |  |  |  |  |  |  |  |
| TOR | Familial Ovarian Tumour Study | TOR | Ontario, Canada | Population based | 556 | 734 | Illumina 610 Quad |
|   |  |  |  |  |  |  |  |
| UCI | UC Irvine Ovarian Cancer Study | UCIb | California, USA | Population based | 372 | 192 | Fluidigm |
|   |  |  |  |  |  |  |  |
| UK2 | SEARCH | SEAb | UK | Population based | 670 |   | Fluidigm |
| Southampton Ovarian Cancer Study | SOCb | UK | Population based |   | 389 | Fluidigm |  |
| Scottish Randomised Trial in Ovarian Cancer | SROa | UK | Hospital based |   | 866 | Fluidigm |  |
| United Kingdom Ovarian Cancer Population Study | UKOb | UK | Population based | 655 |   | Fluidigm |  |
|   |  |  |  |  |  |  |  |
| UK-GWAS | SEARCH | SEAb | UK | Population based |   | 1089 | Illumina 610 Quad |
| United Kingdom Ovarian Cancer Population Study | UKOb | UK | Population based |   | 500 | Illumina 610 Quad |  |
| Cancer Research UK Familial Ovarian Cancer Register | UCRb | UK | Familial cancer register |   | 32 | Illumina 610 Quad |  |
| Royal Marsden Hospital study | RMH | UK | Hospital based |   | 147 | Illumina 610 Quad |  |
| UK 1958 Birth cohort | 58BC | UK | Cohort | 1438 |   | Illumina Hap550 |  |
| UK Colorectal control | NSCR | UK | Population based | 917 |   | Illumina Hap550 |  |
|   |  |  |  |  |  |  |  |
| USC | Los Angeles County Case-Control Studies of Ovarian Cancer | USC | Los Angeles, USA | Population based | 343 | 260 | Taqman |
|   |  |  |  |  |  |  |  |
| Case-only studies |  |  |  |  |  |  |  |
| LAX | Case series | LAXa | Los Angeles, USA | Hospital based |   | 263 | Fluidigm |
|   |  |  |  |  |  |  |  |
| TCGA | The Cancer Genome Atlas | TCGAa | USA | Hospital based |   | 359 | Illumina 1M Duo |

Table 1 Caption: Description of participating studies

Survival-time data were available for cases from eighteen studies (BEL, DOV, UCR, GER, HOP, LAX, MAY, NCO, PVD, RMH, SEA, SOC, SRO, TBO, TCGA, UCI, UKO, USC) and clinical information on chemotherapy, residual disease after surgery and time to progression was collected in five studies (BEL, LAX, MAY, SRO, TCGA). All of the women included in the analysis of progression-free survival (PFS) had at least four cycles of carboplatin and paclitaxel as part of primary treatment. PFS was defined as the time interval between the date of histologic diagnosis and the first confirmed sign of disease recurrence or progression ([9](#R9)).

### Genotyping

Genotyping of thirteen case-control sets was performed in a single laboratory using a 5′ nuclease Taqman allelic discrimination assay (Applied Biosystems, Foster City, CA) as part of a 96-SNP Fluidigm multiplex (10 studies) or--with the same batch of Taqman reagents--using the 7900HT Sequence Detection Software (Applied Biosystems) (three studies) ([Table 1](#T1)). Details of OCAC’s quality control (QC) criteria have been described previously ([10](#R10)); they include genotyping of a common set of 95 DNAs (90 CEPH trios and five duplicate samples) and comparison to the genotypes for the same samples as reported by HapMap. However, rs61764370 was not genotyped in the HapMap project. Based on sequence data for 57 individuals of European origin from the 1000Genomes project ([http://www.1000genomes.org](http://www.1000genomes.org)), a HapMap SNP, rs17388148, was found to be strongly correlated (r^2^=0.97) with rs61764370. The concordance between the CEPH trio genotype data for rs61764370 and the HapMap genotypes for rs17388148 was 100%. Therefore, data on rs17388148 were obtained from three genome-wide association studies (GWAS), where cases and controls had been genotyped using Illumina genome-wide SNP arrays ([1](#R1), [2](#R2), [4](#R4), [8](#R8)) ([Table 1](#T1)). Neither rs61764370 nor rs17388148 were included on the Illumina arrays used in these GWAS studies, but imputed genotypes were available for rs17388148. These genotypes were provided as the estimated number of rare alleles carried (0 to 2 on a continuous scale). The accuracy of the imputation as calculated by the program MACH of Li and Abecasis ([11](#R11)) for the North American studies (BWH, MAY, NCO, TBO and TOR) was r^2^=0.977. This high accuracy of the imputation was evidently due to the presence of a nearby SNP (rs12305513, 17kb away) in high LD (r^2^=1 in HapMap) with rs17388148.

### Statistical Analyses

Analyses were restricted to white non-Hispanic women based on self-reported ethnic origin for all of the studies, with the exception of the TCGA, MAY, NCO, TOR, TBO and UK-GWAS controls. For these studies, genome-wide genotype data were used to estimate intercontinental ancestry and women of less than 90 percent European ancestry were excluded (see methods in reference ([3](#R3)) for details). Cases with borderline (low malignant potential) epithelial ovarian cancer were also excluded, as were 22 cases from TCGA that had been provided to TCGA as part of the MAY case-control study. Departure of genotype frequencies from those expected under Hardy-Weinberg equilibrium was assessed using a chi-square test for each study that was directly genotyped. The association between SNP and disease risk was evaluated using unconditional logistic regression in which number of copies of the minor (infrequent) allele was treated as a continuous variable. This provides an estimate of the per-allele odds ratio (OR) and 95% confidence interval (95% CI). Models adjusted for age categories (<40, 40–49, 50–59, 60–69 and ≥70 years) were also considered. Each case-control set was analyzed individually and the pooled result was obtained by combining the log odds-ratios using standard inverse variance-weighted meta-analytic methods. Analysis of the *BRCA1* mutation carrier cohort was carried out using a time-to-event analysis framework that models the association between genotype and ovarian cancer risk as a hazard ratio. Because mutation carriers were not sampled randomly with respect to their disease status, standard methods of survival analysis may lead to biased estimates of associations. Therefore, analyses were carried out by modeling the retrospective likelihood of observed genotype conditional on disease phenotype (see references ([8](#R8), [12](#R12)) for details).

Associations between genotype and progression-free survival and all-cause mortality were evaluated using proportional hazards regression. Because the EOC cases were recruited at variable times after diagnosis, regression analysis of all-cause mortality allowed for left truncation, with time at risk starting on date of diagnosis and time under observation beginning at the time of study entry. This method generates an unbiased estimate of the hazard ratio provided that the proportional hazards assumption is reasonably correct ([13](#R13)). Cause-specific mortality was not available for most studies, so the analysis of all-cause mortality was right-censored at five years after diagnosis in order to minimize the proportion of deaths from causes other than ovarian cancer. The analysis of progression-free survival was adjusted for stage and residual disease, and survival time ended at time of progression or was censored at time of last follow-up.

## RESULTS

Details of the nineteen case-control sets used in our analyses are given in [Table 1](#T1). Genotype data from these sets were available for 8,669 cases and 10,012 controls ([Table 2](#T2)). All studies passed the OCAC criteria for genotyping quality. Genotype frequencies were close to those expected under Hardy Weinberg equilibrium in both cases and controls for the thirteen directly genotyped studies. No evidence was found for association between rs61764370 and invasive EOC in univariate analysis (OR = 1.02, 95% CI 0.95–1.10, P = 0.44), with minimal heterogeneity of risk between studies (P = 0.28). Study specific odds ratios are shown in [Figure 1a](#F1). When studies with directly genotyped data and with imputed data were analyzed separately, the overall OR in the genotyped studies was 0.96 (95% CI 0.87–1.06, P = 0.42) compared to 1.08 (95% CI 0.97–1.20, P = 0.15) in the imputed data studies. Adjusting for age at diagnosis/interview made little difference to the results (data not shown). No differences in risk were observed when analyses were restricted to cases who provided blood samples within 18 months of diagnosis (n = 6,550, OR = 1.02, 95% CI 0.93–1.10, P = 0.72), cases with serous tumors ([Figure 1b](#F1), n = 4,706, OR = 1.08, 95% CI 0.98–1.18, P = 0.11), or cases reporting a family history of ovarian cancer in a first-degree relative (from 6 studies) ([Figure 1c](#F1), n = 249, OR = 1.09, 95% CI 0.78–1.54, P = 0.62). Tests for heterogeneity of risk between studies in all analyses were not statistically significant. A similar result was seen for risk of EOC by genotype among 683 cases and 2,044 controls who were carriers of *BRCA1* mutations (HR = 1.09, 95% CI 0.88–1.36, P = 0.40).

### Table 2.

| Study abbreviation | Control genotypesa |  |  |  |  | Case genotypesa |  |  |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| 0 | 1 | 2 | Total | Allele freq.b | 0 | 1 | 2 | Total | Allele freq.b |  |
| BEL | 202 | 49 | 2 | 253 | 0.105 | 142 | 28 | 3 | 173 | 0.098 |
| DOV | 610 | 104 | 7 | 721 | 0.082 | 570 | 123 | 5 | 698 | 0.095 |
| GER | 218 | 46 | 1 | 265 | 0.091 | 184 | 28 | 1 | 213 | 0.070 |
| HJO | 354 | 66 | 6 | 426 | 0.092 | 217 | 42 | 0 | 259 | 0.081 |
| HMO | 130 | 19 | 2 | 151 | 0.076 | 174 | 20 | 1 | 195 | 0.056 |
| HOC | 390 | 43 | 1 | 434 | 0.052 | 321 | 28 | 1 | 350 | 0.043 |
| HOP | 297 | 67 | 4 | 368 | 0.102 | 308 | 54 | 3 | 365 | 0.082 |
| LAX |   |   |   |   |   | 223 | 39 | 1 | 263 | 0.078 |
| NTH | 275 | 47 | 5 | 327 | 0.087 | 249 | 46 | 1 | 296 | 0.081 |
| OVA | 332 | 83 | 1 | 416 | 0.102 | 417 | 74 | 3 | 494 | 0.081 |
| PVM | 181 | 33 | 1 | 215 | 0.081 | 166 | 34 | 1 | 201 | 0.090 |
| UCI | 318 | 52 | 2 | 372 | 0.075 | 150 | 41 | 1 | 192 | 0.112 |
| UK2 | 1,087 | 231 | 7 | 1,325 | 0.092 | 1,039 | 201 | 15 | 1,255 | 0.092 |
| USC | 281 | 59 | 3 | 343 | 0.095 | 214 | 45 | 1 | 260 | 0.090 |
| BWHc | 118 | 23 | 1 | 142 | 0.088 | 115 | 21 | 1 | 137 | 0.084 |
| MAYc | 438 | 78 | 4 | 520 | 0.083 | 283 | 71 | 4 | 358 | 0.110 |
| NCOc | 537 | 112 | 6 | 655 | 0.095 | 398 | 91 | 5 | 494 | 0.102 |
| TBOc | 141 | 26 | 1 | 168 | 0.083 | 181 | 43 | 3 | 227 | 0.108 |
| TCGAc |   |   |   |   |   | 286 | 69 | 4 | 359 | 0.107 |
| TORc | 458 | 93 | 5 | 556 | 0.093 | 587 | 139 | 8 | 734 | 0.106 |
| UK-GWASc | 1,923 | 410 | 22 | 2,355 | 0.096 | 1,446 | 306 | 16 | 1,768 | 0.096 |
| Total | 8,282 | 1,649 | 81 | 10,012 |   | 7,969 | 1,241 | 81 | 9,291 |   |

Table 2 Caption: Genotype frequency numbers of controls and cases by study

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e010/3107901/ff30b60948d2/nihms271454f1.jpg)

Funnel plots of study-specific odds ratios for association between rs61764370 and ovarian cancer risk. (a) all cases; (b) serous cases; (c) cases with a family history of ovarian cancer in a first-degree relative.

Survival-time data were available for 6,002 cases of the 6,826 total from eighteen case series, including 13,696 person-years at risk and 2,044 deaths. Little evidence of association was observed between rs61764370 genotype and all-cause mortality within five years of diagnosis (HR = 1.10, 95% CI 0.99–1.22, P = 0.08), with no evidence of heterogeneity of the hazard ratio between studies (P = 0.89). Results of analyses restricted to serous subtype (HR = 1.12, 95% CI 0.99–1.28, P = 0.08) or adjusted for tumor stage and grade (HR = 1.06, 95% CI 0.96–1.18, P = 0.27) were similar. There was also little evidence for association between genotype and progression-free survival in 540 high-grade serous cases known to have been treated with carboplatin and paclitaxel (HR = 1.18, 95% CI 0.93–1.52, P = 0.16).

## DISCUSSION

This study provides no evidence to support the previously reported associations between rs61764370 and risk of epithelial ovarian cancer. The relative risk given by Ratner and colleagues for their replication data set of unselected cases was 1.70 (95% CI 1.11–2.63, P = 0.016) ([5](#R5)). The confidence intervals in the present study do not overlap with these. The power of the present study to detect an allele conferring a relative risk of 1.3 under either a dominant or log-additive genetic model with a Type 1 error rate of 10^−4^ was greater than 99%, strongly suggesting that the association observed by Ratner and colleagues was a chance finding or possibly due to subtle genotyping errors. This observation is not surprising. Associations with modest P-values that are declared as positive are very likely to be false positives when the prior probability of association is low ([14](#R14)). In genetic association studies, even if evidence exists that a variant has functional effects, the prior probability of association at a relative risk of 1.5 is unlikely to be more than 1:100, and given that only a handful of loci conferring relative risks of more than 1.5 have been found for any cancer, the prior probability is likely to be much less. Based on the methodology of Wacholder and colleagues ([14](#R14)), assuming a prior of 1:100, the probability that the association reported by Ratner and colleagues was a false positive is 86 percent. Under more likely, smaller prior probabilities the false positive probability will approach 100 percent. For familial ovarian cancer, the power of the present study to detect a relative risk of 2.0 was over 95% at a type 1 error rate of 0.05, again suggesting it is unlikely that the present analyses have missed a true association of this magnitude with familial ovarian cancer.

Statistical power to detect an allele with weaker effects is more limited. For example, if the true relative risk were 1.15 under a co-dominant genetic model--similar to the relative risk conferred by an allele with the biggest risk detected in a recent well-powered lung-cancer GWAS ([15](#R15))--power at a type 1 error rate of 0.001 would be reduced to 70%. A very modest association between the rs61764370 variant and a slightly increased risk of ovarian cancer cannot therefore be excluded. Neither has the present study shown much evidence for association between genotype at this locus and other related phenotypes: EOC risk in carriers of deleterious *BRCA1* mutations or in women with family histories of EOC, all-cause mortality after diagnosis of EOC, and progression-free survival in EOC cases treated with carboplatin and paclitaxel.

In summary, the possibility that the minor (infrequent) allele of rs61764340 is associated with an appreciable risk of epithelial ovarian cancer is excluded. Furthermore, it is debatable whether a single risk allele, even one conferring a relative risk as high as 2.0, has clinical utility, particularly in a disease with lifetime risk as low as it is in invasive epithelial ovarian cancer (1 in 70), and even among women with a family history of the disease ([16](#R16)). The marketing of a commercial assay for rs61764370 for clinical use among unselected cases, or women with family histories of ovarian cancer or who are carriers of *BRCA1* mutations, in the absence of convincing evidence for association, is not warranted.

## Acknowledgments

We thank all the individuals who took part in this study and all the researchers, clinicians and administrative staff who have enabled and carried out the many studies contributing to this work. In particular, we thank: E. John, J. L. Hopper, S. S. Buys, M. B. Daly, M. B. Terry, D. Goldgar and M. Southey (BCFR); S. Manoukian, B. Peissel and M. A. Pierotti of the Fondazione IRCCS Istituto Nazionale dei Tumori, Milan, Italy; P. Peterlongo of the Fondazione Istituto FIRC di Oncologia Molecolare, Milan, Italy; M. Barile and B. Bonanni of the Istituto Europeo di Oncologia, Milan, Italy; L. Bernard of the Consortium for Genomic Technologies, Milan, Italy; and A. Viel of the Centro di Riferimento Oncologico, Aviano, Italy (CONSIT); all of the GEMO study collaborating groups for their contributions (GEMO); A.U. Eilber and T. Koehler (GER); L. Gacucova (HMO); P. Schürmann, F. Kramer, W. Zheng, T.-W. Park-Simon, P. Hillemanns, K. Beer-Grondke, D. Schmidt and I. Runnebaum (HJO); A. Teule, G. Capella, I. Blanco, J. Brunet, L. Feliubadalo and M. Salinas (ICO); E. Niedermeyr and H. Thorne (kConFab); I. Tomlinson and R. Houlston (NSCR); the Nijmegen Biomedical Study for sharing questionnaire data and DNA from controls and the gynecologists in the catchment area of the Comprehensive Cancer Center East for recruiting patients (NTH); P. Harrington and the SEARCH team (SEA); Ian Jacobs, Eva Wozniak, Andy Ryan, Jeremy Ford and Nayala Balogun (UKO). We acknowledge the computational resources provided by the University of Cambridge (CamGrid). This study in part makes use of data generated by the Wellcome Trust Case-Control Consortium. A full list of the investigators who contributed to the generation of those data is available at [http://www.wtccc.org.uk](http://www.wtccc.org.uk). The results published here are also in part based upon data generated by The Cancer Genome Atlas Pilot Project established by the National Cancer Institute and National Human Genome Research Institute. Information about TCGA and the investigators and institutions constituting the TCGA research network can be found at [http://cancergenome.nih.gov/](http://cancergenome.nih.gov/). We acknowledge the patients who donated tissues for research use, the tissue source sites, and the TCGA Research Network. Samples used in the BCFR from the NCCC, FCCC, and HCI were processed and distributed by the Coriell Institute for Medical Research. The content of this manuscript does not necessarily reflect the views or policies of the National Cancer Institute or any of the collaborating centers in the BCFR, nor does mention of trade names, commercial products, or organizations imply endorsement by the US Government or the BCFR.

The Ovarian Cancer Association Consortium is supported by a grant from the Ovarian Cancer Research Fund. Funding for the individual participating studies was provided by: cooperative agreements with the Breast Cancer Family Registry including Cancer Care Ontario, and US National Institutes of Health grants 5U01-CA069467, 5U01-CA069398, 5U01-CA069631, 5U01-CA069446, 5U01-CA069417, 5U01-CA069638 and N02-PC45022 (BCFR); the National Cancer Plan – Action 29 for the support of Translational Research (BEL); National Institutes of Health grants 5R01-CA054419 and 5P50-CA105009 (BWH); National Institutes of Health grants 5R01-CA112523 and 5R01-CA087538 (DOV); the Ligue National Contre le Cancer, the Association “Le cancer du sein, parlons-en!” and research grant AICR-07-0454 from the Association for International Cancer Research (GEMO); the German Federal Ministry of Education and Research, the Programme of Clinical Biomedical Research grant 01 GB 9401, the state of Baden-Württemberg through the University of Ulm Medical Faculty grant P.685, and the German Cancer Research Center (GER); the Helsinki University Central Hospital Research Fund, Academy of Finland and the Finnish Cancer Society (HOC); National Institutes of Health grants 5R01-CA095023, 5R01-CA126841, 5R03-CA121881, US Army MRMC grant DAMD 17-02-1-0669, and grants from the Ovarian Cancer Research Foundation and the Roswell Park Alliance Foundation (HOP); grant 628333 from Cancer Australia and grants from the National Breast Cancer Foundation, the National Health and Medical Research Council (NHMRC), the Queensland Cancer Fund, the Cancer Councils of New South Wales, Victoria, Tasmania and South Australia, and the Cancer Foundation of Western Australia (kConFab); American Cancer Society Early Detection Professorship grant SIOP-06-258-01-CCE and the L & S Milken Foundation (LAX); Mermaid 1, The Danish Cancer Society and the National Cancer Institute 5R01-CA061107 (MAL); National Cancer Institute 5R01-CA122443 (MAY); National Cancer Institute 5R01-CA076016 (NCO); an Investment Grant of the Radboud University Nijmegen Medical Centre (NTH); grants from the Canadian Institutes of Health Research (CIHR) and the Workers’ Compensation Board of British Columbia (OVA); Cancer Research UK (SEA); Experimental Cancer Research Centres and Cancer Research UK A6689 (SRO); National Institutes of Health grant 5R01-CA106414, American Cancer Society grant CRTG-00-196-01-CCE and an Advanced Cancer Detection Center Grant, Department of Defense grant DAMD 17-98-1-8659, and funding from the Celma Mastry Ovarian Cancer Foundation (TBO); Canadian Institutes for Health Research, the National Cancer Institute of Canada, the Canadian Cancer Society and the National Institutes of Health grants 5R01-CA063682 and 5R01-CA063678 (TOR); National Cancer Institute grants 5R01-CA058860 and 5K07-CA092044 and the Lon V Smith Foundation LVS-39420 (UCI); Cancer Research UK, the Eve Appeal, the OAK Foundation and the Department of Health’s NIHR Biomedical Research Centre funding scheme (UKO); the California Cancer Research Program grants 00-01389V-20170 and 2110200, National Institutes of Health grants 5P30-CA014089, 5P01-CA017054, 5R01-CA061132, 5R01-CA063464, N01-PC67010 and 5R03-CA113148, and California Department of Health Services sub-contract 050-E8709 (USC); Wellcome Trust award 076113 (WTCCC). Additional support for the TOR, NCO, MAY, TBO, and NCI studies was provided by National Institutes of Health grant 5R01-CA114343. The Consortium of Investigators of Modifiers of *BRCA1/2* GWAS and its participating studies were funded by: the Breast Cancer Research Foundation, Susan G. Komen for the Cure, the Ovarian Cancer Research Fund, the Eileen Stein Jacoby Fund, National Institutes of Health grants 5R01-CA128978 and 1R01-CA140323, Cancer Research UK, the Neve Foundation, Project Hope for Ovarian Cancer Research and Education, the Australian National Breast Cancer Foundation, the National Health and Medical Research Council of Australia.

## References

1. Gayther SA, Pharoah PD. The inherited genetics of ovarian and endometrial cancer. Curr Opin Genet Dev. 2010;20:231–8. doi: 10.1016/j.gde.2010.03.001.  [DOI](https://doi.org/10.1016/j.gde.2010.03.001) | [PMC free article](/articles/PMC3073171/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20456938/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Opin%20Genet%20Dev&title=The%20inherited%20genetics%20of%20ovarian%20and%20endometrial%20cancer&author=SA%20Gayther&author=PD%20Pharoah&volume=20&publication_year=2010&pages=231-8&pmid=20456938&doi=10.1016/j.gde.2010.03.001&)

2. Song H, Ramus SJ, Tyrer J, Bolton KL, Gentry-Maharaj A, Wozniak E, et al. A genome-wide association study identifies a new ovarian cancer susceptibility locus on 9p22.2. Nat Genet. 2009;41:996–1000. doi: 10.1038/ng.424.  [DOI](https://doi.org/10.1038/ng.424) | [PMC free article](/articles/PMC2844110/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19648919/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=A%20genome-wide%20association%20study%20identifies%20a%20new%20ovarian%20cancer%20susceptibility%20locus%20on%209p22.2&author=H%20Song&author=SJ%20Ramus&author=J%20Tyrer&author=KL%20Bolton&author=A%20Gentry-Maharaj&volume=41&publication_year=2009&pages=996-1000&pmid=19648919&doi=10.1038/ng.424&)

3. Bolton KL, Tyrer J, Song H, Ramus SJ, Notaridou M, Jones C, et al. Common variants at 19p13 are associated with susceptibility to ovarian cancer. Nature Genetics. 2010;42:880–4. doi: 10.1038/ng.666.  [DOI](https://doi.org/10.1038/ng.666) | [PMC free article](/articles/PMC3125495/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20852633/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature%20Genetics&title=Common%20variants%20at%2019p13%20are%20associated%20with%20susceptibility%20to%20ovarian%20cancer&author=KL%20Bolton&author=J%20Tyrer&author=H%20Song&author=SJ%20Ramus&author=M%20Notaridou&volume=42&publication_year=2010&pages=880-4&pmid=20852633&doi=10.1038/ng.666&)

4. Goode EL, Chenevix-Trench G, Song H, Ramus SJ, Notaridou M, Lawrenson K, et al. A genome-wide association study identifies susceptibility loci for ovarian cancer at 2q31 and 8q24. Nature Genetics. 2010;42:874–9. doi: 10.1038/ng.668.  [DOI](https://doi.org/10.1038/ng.668) | [PMC free article](/articles/PMC3020231/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20852632/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature%20Genetics&title=A%20genome-wide%20association%20study%20identifies%20susceptibility%20loci%20for%20ovarian%20cancer%20at%202q31%20and%208q24&author=EL%20Goode&author=G%20Chenevix-Trench&author=H%20Song&author=SJ%20Ramus&author=M%20Notaridou&volume=42&publication_year=2010&pages=874-9&pmid=20852632&doi=10.1038/ng.668&)

5. Ratner E, Lu L, Boeke M, Barnett R, Nallur S, Chin LJ, et al. A KRAS-variant in ovarian cancer acts as a genetic marker of cancer risk. Cancer Research. 2010;70:6509–15. doi: 10.1158/0008-5472.CAN-10-0689.  [DOI](https://doi.org/10.1158/0008-5472.CAN-10-0689) | [PMC free article](/articles/PMC2923587/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20647319/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Research&title=A%20KRAS-variant%20in%20ovarian%20cancer%20acts%20as%20a%20genetic%20marker%20of%20cancer%20risk&author=E%20Ratner&author=L%20Lu&author=M%20Boeke&author=R%20Barnett&author=S%20Nallur&volume=70&publication_year=2010&pages=6509-15&pmid=20647319&doi=10.1158/0008-5472.CAN-10-0689&)

6. Esquela-Kerscher A, Slack FJ. Oncomirs - microRNAs with a role in cancer. Nature Reviews Cancer. 2006;6:259–69. doi: 10.1038/nrc1840.  [DOI](https://doi.org/10.1038/nrc1840) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16557279/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature%20Reviews%20Cancer&title=Oncomirs%20-%20microRNAs%20with%20a%20role%20in%20cancer&author=A%20Esquela-Kerscher&author=FJ%20Slack&volume=6&publication_year=2006&pages=259-69&pmid=16557279&doi=10.1038/nrc1840&)

7. Thomas DC, Clayton DG. Betting odds and genetic associations. J Natl Cancer Inst. 2004;96:421–3. doi: 10.1093/jnci/djh094.  [DOI](https://doi.org/10.1093/jnci/djh094) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15026459/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Natl%20Cancer%20Inst&title=Betting%20odds%20and%20genetic%20associations&author=DC%20Thomas&author=DG%20Clayton&volume=96&publication_year=2004&pages=421-3&pmid=15026459&doi=10.1093/jnci/djh094&)

8. Antoniou AC, Wang X, Fredericksen ZS, McGuffog L, Tarrell R, Sinilnikova OM, et al. A locus on 19p13 modifies risk of breast cancer in BRCA1 mutation carriers and is associated with hormone receptor-negative breast cancer in the general population. Nature Genetics. 2010;42:885–92. doi: 10.1038/ng.669.  [DOI](https://doi.org/10.1038/ng.669) | [PMC free article](/articles/PMC3130795/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20852631/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature%20Genetics&title=A%20locus%20on%2019p13%20modifies%20risk%20of%20breast%20cancer%20in%20BRCA1%20mutation%20carriers%20and%20is%20associated%20with%20hormone%20receptor-negative%20breast%20cancer%20in%20the%20general%20population&author=AC%20Antoniou&author=X%20Wang&author=ZS%20Fredericksen&author=L%20McGuffog&author=R%20Tarrell&volume=42&publication_year=2010&pages=885-92&pmid=20852631&doi=10.1038/ng.669&)

9. Johnatty SE, Beesley J, Paul J, Fereday S, Spurdle AB, Webb PM, et al. ABCB1 (MDR 1) polymorphisms and progression-free survival among women with ovarian cancer following paclitaxel/carboplatin chemotherapy. Clinical Cancer Research. 2008;14:5594–601. doi: 10.1158/1078-0432.CCR-08-0606.  [DOI](https://doi.org/10.1158/1078-0432.CCR-08-0606) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18765553/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clinical%20Cancer%20Research&title=ABCB1%20(MDR%201)%20polymorphisms%20and%20progression-free%20survival%20among%20women%20with%20ovarian%20cancer%20following%20paclitaxel/carboplatin%20chemotherapy&author=SE%20Johnatty&author=J%20Beesley&author=J%20Paul&author=S%20Fereday&author=AB%20Spurdle&volume=14&publication_year=2008&pages=5594-601&pmid=18765553&doi=10.1158/1078-0432.CCR-08-0606&)

10. Pearce CL, Near AM, Van Den Berg DJ, Ramus SJ, Gentry-Maharaj A, Menon U, et al. Validating genetic risk associations for ovarian cancer through the international Ovarian Cancer Association Consortium. Br J Cancer. 2009;100:412–20. doi: 10.1038/sj.bjc.6604820.  [DOI](https://doi.org/10.1038/sj.bjc.6604820) | [PMC free article](/articles/PMC2634713/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19127255/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Cancer&title=Validating%20genetic%20risk%20associations%20for%20ovarian%20cancer%20through%20the%20international%20Ovarian%20Cancer%20Association%20Consortium&author=CL%20Pearce&author=AM%20Near&author=DJ%20Van%20Den%20Berg&author=SJ%20Ramus&author=A%20Gentry-Maharaj&volume=100&publication_year=2009&pages=412-20&pmid=19127255&doi=10.1038/sj.bjc.6604820&)

11. Scott LJ, Mohlke KL, Bonnycastle LL, Willer CJ, Li Y, Duren WL, et al. A genome-wide association study of type 2 diabetes in Finns detects multiple susceptibility variants. Science. 2007;316:1341–5. doi: 10.1126/science.1142382.  [DOI](https://doi.org/10.1126/science.1142382) | [PMC free article](/articles/PMC3214617/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17463248/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=A%20genome-wide%20association%20study%20of%20type%202%20diabetes%20in%20Finns%20detects%20multiple%20susceptibility%20variants&author=LJ%20Scott&author=KL%20Mohlke&author=LL%20Bonnycastle&author=CJ%20Willer&author=Y%20Li&volume=316&publication_year=2007&pages=1341-5&pmid=17463248&doi=10.1126/science.1142382&)

12. Antoniou AC, Sinilnikova OM, Simard J, Leone M, Dumont M, Neuhausen SL, et al. RAD51 135G→C modifies breast cancer risk among BRCA2 mutation carriers: results from a combined analysis of 19 studies. Am J Hum Genet. 2007;81:1186–200. doi: 10.1086/522611.  [DOI](https://doi.org/10.1086/522611) | [PMC free article](/articles/PMC2276351/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17999359/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=RAD51%20135G%E2%86%92C%20modifies%20breast%20cancer%20risk%20among%20BRCA2%20mutation%20carriers:%20results%20from%20a%20combined%20analysis%20of%2019%20studies&author=AC%20Antoniou&author=OM%20Sinilnikova&author=J%20Simard&author=M%20Leone&author=M%20Dumont&volume=81&publication_year=2007&pages=1186-200&pmid=17999359&doi=10.1086/522611&)

13. Azzato EM, Greenberg D, Shah M, Blows F, Driver KE, Caporaso NE, et al. Prevalent cases in observational studies of cancer survival: do they bias hazard ratio estimates? Br J Cancer. 2009;100:1806–11. doi: 10.1038/sj.bjc.6605062.  [DOI](https://doi.org/10.1038/sj.bjc.6605062) | [PMC free article](/articles/PMC2695697/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19401693/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Cancer&title=Prevalent%20cases%20in%20observational%20studies%20of%20cancer%20survival:%20do%20they%20bias%20hazard%20ratio%20estimates?&author=EM%20Azzato&author=D%20Greenberg&author=M%20Shah&author=F%20Blows&author=KE%20Driver&volume=100&publication_year=2009&pages=1806-11&pmid=19401693&doi=10.1038/sj.bjc.6605062&)

14. Wacholder S, Chanock S, Garcia-Closas M, El Ghormli L, Rothman N. Assessing the probability that a positive report is false: an approach for molecular epidemiology studies. J Natl Cancer Inst. 2004;96:434–42. doi: 10.1093/jnci/djh075.  [DOI](https://doi.org/10.1093/jnci/djh075) | [PMC free article](/articles/PMC7713993/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15026468/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Natl%20Cancer%20Inst&title=Assessing%20the%20probability%20that%20a%20positive%20report%20is%20false:%20an%20approach%20for%20molecular%20epidemiology%20studies&author=S%20Wacholder&author=S%20Chanock&author=M%20Garcia-Closas&author=L%20El%20Ghormli&author=N%20Rothman&volume=96&publication_year=2004&pages=434-42&pmid=15026468&doi=10.1093/jnci/djh075&)

15. Truong T, Hung RJ, Amos CI, Wu X, Bickeböller H, Rosenberger A, et al. Replication of lung cancer susceptibility loci at chromosomes 15q25, 5p15, and 6p21: a pooled analysis from the International Lung Cancer Consortium. J Natl Cancer Inst. 2010;102:959–71. doi: 10.1093/jnci/djq178.  [DOI](https://doi.org/10.1093/jnci/djq178) | [PMC free article](/articles/PMC2897877/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20548021/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Natl%20Cancer%20Inst&title=Replication%20of%20lung%20cancer%20susceptibility%20loci%20at%20chromosomes%2015q25,%205p15,%20and%206p21:%20a%20pooled%20analysis%20from%20the%20International%20Lung%20Cancer%20Consortium&author=T%20Truong&author=RJ%20Hung&author=CI%20Amos&author=X%20Wu&author=H%20Bickeb%C3%B6ller&volume=102&publication_year=2010&pages=959-71&pmid=20548021&doi=10.1093/jnci/djq178&)

16. Vineis P, Schulte P, McMichael AJ. Misconceptions about the use of genetic tests in populations. Lancet. 2001;357:709–12. doi: 10.1016/S0140-6736(00)04136-2.  [DOI](https://doi.org/10.1016/S0140-6736(00)04136-2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11247571/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Misconceptions%20about%20the%20use%20of%20genetic%20tests%20in%20populations&author=P%20Vineis&author=P%20Schulte&author=AJ%20McMichael&volume=357&publication_year=2001&pages=709-12&pmid=11247571&doi=10.1016/S0140-6736(00)04136-2&)
