# Replication of LDL GWAs hits in PROSPER/PHASE as validation for future (pharmaco)genetic analyses

## Metadata
**Authors:** Stella Trompet, Anton JM de Craen, Iris Postmus, Ian Ford, Naveed Sattar, Muriel Caslake, David J Stott, Brendan M Buckley, Frank Sacks, James J Devlin, P Eline Slagboom, Rudi GJ Westendorp, J Wouter Jukema
**Journal:** BMC Medical Genetics
**Date:** 2011 Oct 6
**DOI:** [10.1186/1471-2350-12-131](https://doi.org/10.1186/1471-2350-12-131)
**PMID:** 21977987
**PMCID:** PMC3207930
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3207930/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC3207930/pdf/1471-2350-12-131.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC3207930/pdf/1471-2350-12-131.pdf)

## Abstract

**Background:** 
The PHArmacogenetic study of Statins in the Elderly at risk (PHASE) is a genome wide association study in the PROspective Study of Pravastatin in the Elderly at risk for vascular disease (PROSPER) that investigates the genetic variation responsible for the individual variation in drug response to pravastatin. Statins lower LDL-cholesterol in general by 30%, however not in all subjects. Moreover, clinical response is highly variable and adverse effects occur in a minority of patients. In this report we first describe the rationale of the PROSPER/PHASE project and second show that the PROSPER/PHASE study can be used to study pharmacogenetics in the elderly.

**Methods:** 
The genome wide association study (GWAS) was conducted using the Illumina 660K-Quad beadchips following manufacturer's instructions. After a stringent quality control 557,192 SNPs in 5,244 subjects were available for analysis. To maximize the availability of genetic data and coverage of the genome, imputation up to 2.5 million autosomal CEPH HapMap SNPs was performed with MACH imputation software. The GWAS for LDL-cholesterol is assessed with an additive linear regression model in PROBABEL software, adjusted for age, sex, and country of origin to account for population stratification.

**Results:** 
Forty-two SNPs reached the GWAS significant threshold of p = 5.0e-08 in 5 genomic loci (APOE/APOC1; LDLR; FADS2/FEN1; HMGCR; PSRC1/CELSR5). The top SNP (rs445925, chromosome 19) with a p-value of p = 2.8e-30 is located within the APOC1 gene and near the APOE gene. The second top SNP (rs6511720, chromosome 19) with a p-value of p = 5.22e-15 is located within the LDLR gene. All 5 genomic loci were previously associated with LDL-cholesterol levels, no novel loci were identified. Replication in WOSCOPS and CARE confirmed our results.

**Conclusion:** 
With the GWAS in the PROSPER/PHASE study we confirm the previously found genetic associations with LDL-cholesterol levels. With this proof-of-principle study we show that the PROSPER/PHASE study can be used to investigate genetic associations in a similar way to population based studies. The next step of the PROSPER/PHASE study is to identify the genetic variation responsible for the variation in LDL-cholesterol lowering in response to statin treatment in collaboration with other large trials.

### Background

The PHArmacogenetic study of Statins in the Elderly at risk (PHASE) is a genome wide association study in the PROspective Study of Pravastatin in the Elderly at risk for vascular disease (PROSPER) that investigates the genetic variation responsible for the individual variation in drug response to pravastatin. Statins lower LDL-cholesterol in general by 30%, however not in all subjects. Moreover, clinical response is highly variable and adverse effects occur in a minority of patients. In this report we first describe the rationale of the PROSPER/PHASE project and second show that the PROSPER/PHASE study can be used to study pharmacogenetics in the elderly.

### Methods

The genome wide association study (GWAS) was conducted using the Illumina 660K-Quad beadchips following manufacturer's instructions. After a stringent quality control 557,192 SNPs in 5,244 subjects were available for analysis. To maximize the availability of genetic data and coverage of the genome, imputation up to 2.5 million autosomal CEPH HapMap SNPs was performed with MACH imputation software. The GWAS for LDL-cholesterol is assessed with an additive linear regression model in PROBABEL software, adjusted for age, sex, and country of origin to account for population stratification.

### Results

Forty-two SNPs reached the GWAS significant threshold of p = 5.0e-08 in 5 genomic loci (APOE/APOC1; LDLR; FADS2/FEN1; HMGCR; PSRC1/CELSR5). The top SNP (rs445925, chromosome 19) with a p-value of p = 2.8e-30 is located within the APOC1 gene and near the APOE gene. The second top SNP (rs6511720, chromosome 19) with a p-value of p = 5.22e-15 is located within the LDLR gene. All 5 genomic loci were previously associated with LDL-cholesterol levels, no novel loci were identified. Replication in WOSCOPS and CARE confirmed our results.

### Conclusion

With the GWAS in the PROSPER/PHASE study we confirm the previously found genetic associations with LDL-cholesterol levels. With this proof-of-principle study we show that the PROSPER/PHASE study can be used to investigate genetic associations in a similar way to population based studies. The next step of the PROSPER/PHASE study is to identify the genetic variation responsible for the variation in LDL-cholesterol lowering in response to statin treatment in collaboration with other large trials.

## Background

Cardiovascular disease is the leading cause of death in industrialized countries at old age. Advancing age is one of the most important risk factors for cardiovascular disease [[1](#B1)]. With the rising number of elderly people in our society cardiovascular disease has a major impact on healthcare [[2](#B2)]. The prevention of cardiovascular disease is critically dependent on lipid lowering therapy including the 3-hydroxymethyl-3-methylglutaryl coenzyme A (HMG-CoA) reductase inhibitors (statins). Statins are the most prescribed class of drugs worldwide and therapy is generally associated with a reduction of cardiovascular events by 20-30%. However, clinical response is highly variable and adverse effects occur in a minority of patients [[3](#B3)]. Recent research provides evidence that genetic variation contributes importantly to this variable drug response [[4](#B4)].

Pharmacogenomics focuses on unraveling the genetic determinants of such variable drug responses, both in intended, beneficial effects and unintended, adverse effects [[5](#B5)]. Therefore, we here present the PHArmacogenetic study of Statin in the Elderly at risk (PHASE) a genome wide association study (GWAS) in the PROspective Study of Pravastatin in the Elderly at Risk for vascular disease (PROSPER)[[6](#B6)] investigating the genetic variation responsible for the individual variation in drug response funded by the European Union's Seventh Framework Programme. To validate the GWAS performed in the PHASE study, we executed a proof-of-principle study to investigate the underlying genetic variation in LDL cholesterol levels.

Recent GWA studies have identified several new loci that influence circulating levels of blood lipids with around 95 loci showing statistical associations with circulating total cholesterol levels, HDL cholesterol, LDL cholesterol, and triglycerides [[7](#B7)]. These GWA studies are executed in population based studies with various age groups, however the elderly (age > 75 years) are rarely represented in these studies. With this proof-of-principle study we provide a testing frame to show that the PROSPER/PHASE study has sufficient statistical power to find genome wide statistical significant associations in quantitative traits such as LDL cholesterol in an elderly population. We replicated our findings from the PROSPER/PHASE study in two independent cohorts to validate that our results contain no false positive findings.

## Methods

### Study population

PROSPER was an investigator-driven, prospective multi-national randomized placebo-controlled trial to assess whether treatment with pravastatin diminishes the risk of major vascular events in the elderly [6;8]. Between December 1997 and May 1999, we screened and enrolled subjects in Scotland, Ireland, and the Netherlands. Men and women aged 70-82 years were recruited if they had pre-existing vascular disease or were at increased risk of such disease because of smoking, hypertension, or diabetes. A total number of 5804 subjects, of whom more than 50% was female, were randomly assigned to pravastatin or placebo. Various clinical laboratory measurements were carried out like inflammatory markers (CRP and various cytokines) and other biochemical substrates (e.g. glucose, leptin) at baseline and during follow-up. The protocol of the PROSPER study meets the criteria of the Declaration of Helsinki and was approved by the Medical Ethics Committees of each participating institution. Written informed consent was obtained from all participating subjects.

### LDL cholesterol

Plasma lipids and lipoproteins were measured twice during the screening phase, i.e. at the beginning and end of the single-blind, placebo "run-in" phase according to the standardized Lipid Research Clinics protocol. Baseline LDL cholesterol levels were taken as the average of these 2 determinations prior to randomization to statin treatment. Total cholesterol (TC), HDL cholesterol, and triglycerides were assessed after an overnight fast, LDL cholesterol was calculated by the Friedewald formula, as previously described [[8](#B8)].

### Genotyping

The genotyping was conducted using the Illumina 660-Quad beadchips following manufacturer's instructions. These beadchips contain 657,366 single nucleotide polymorphism (SNP) and copy number variants (CNV) probes. After genotyping, samples and genetic markers were subjected to a stringent quality control protocol. From the 5763 samples with DNA available that underwent genotyping, 519 samples (9%) were excluded during the quality control (Figure [1](#F1)). Excluded were 18 duplicated samples, 219 samples with a call rate < 97.5%, 11 samples with an excess for heterozygosity, 40 samples of non-caucasian origin, 170 samples with familiar relationships (IBD > 0.35), and 61 samples with a gender mismatch. From the 657,366 probes on the beadchips, 95,876 probes were filtered based on CNV intensity. Moreover, 4,298 SNPs were excluded with a call rate < 95%, leaving us with 557,192 SNPs for analysis. To maximize the availability of genetic data and coverage of the genome, imputation up to 2.5 million autosomal CEPH HapMap SNPs was performed with MACH imputation software based on the Hapmap built II release 23. To assess accuracy of the imputed genotypes, we compared the imputation output with SNPs that had been previously genotyped on other platforms.

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f342/3207930/9703381fabf7/1471-2350-12-131-1.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3207930_1471-2350-12-131-1.jpg)

Flow chart of the Quality Control of the PROSPER/PHASE study.

### Statistical Analysis

Genome wide association analysis was performed with PROBABEL software specialized in genetic association analysis with imputed data taking the probability of the genotype into account ([http://www.genabel.org](http://www.genabel.org)/). With analyzing imputed genotypes, the observed allele count is replaced by the imputation's estimated dosage. For the continuous trait, baseline LDL cholesterol levels, an additive linear regression model was used to assess estimates and standard errors. The model was adjusted for sex and age, and country to correct for the within-study population structure. Standard errors for the regression estimates were calculated with model-robust methods. The analysis of 2.5 million SNPs at once poses a multiple testing problem. After the use of a Bonferroni correction, the threshold for genome wide significant results was set at 5.0e-08.

### Replication

Associations with a genome-wide significant p-value of 5.0e-08 were replicated in two independent cohorts, the West of Scotland Coronary Prevention Study (WOSCOPS)[[9](#B9)] and the Cholesterol and Recurrent Events (CARE) trial [[10](#B10)]. The WOSCOPS study was a double blind randomized placebo-controlled clinical trial in which 6595 men (age range 45-64 years)with hypercholesterolemia and no history of myocardial infarction were treated with 40 mg pravastatin (N = 3302) or placebo (N = 3293). GWAS data and baseline LDL cholesterol levels were available for 431 subjects. The CARE study was a double blind randomized placebo-controlled clinical trial in which 4159 patients (age range 21-75 years) were treated with 40 mg pravastatin (N = 2081) or placebo (N = 2078). GWAS data and baseline LDL cholesterol levels were available for 751 subjects. The significance level for the replication SNPs was set at p-value < 0.05.

## Results

Table [1](#T1) shows the baseline characteristics of the subjects participating in the PROSPER and the PROSPER/PHASE study. This table shows that the genotyped subjects in the PROSPER/PHASE study are representative of the total study population of the PROSPER study, since no major discrepancies exist between the two study sets. The mean age of all subjects at study entry was 75.3 years and about 50% of the participants were female.

### Table 1.

|   | PROSPER study (n = 5804) | PROSPER/PHASE study (n = 5244) |
| --- | --- | --- |
| Continuous variables (mean, SD) |   |   |
| Age (years) | 75.3 (3.3) | 75.3 (3.4) |
| Education (years) | 15.1 (2.0) | 15.1 (2.0) |
| Systolic blood pressure (mmHg) | 154.7 (21.8) | 154.6 (21.9) |
| Diastolic blood pressure (mmHg) | 83.8 (11.5) | 83.7 (11.4) |
| Height (cm) | 165.2 (9.4) | 165.2 (9.4) |
| Weight (kg) | 73.4 (13.4) | 73.3 (13.4) |
| Body mass index (kg/m2) | 26.8 (4.2) | 26.8 (4.2) |
| Total cholesterol (mmol/L) | 5.7 (0.9) | 5.7 (0.9) |
| LDL cholesterol (mmol/L) | 3.8 (0.8) | 3.8 (0.8) |
| HDL cholesterol (mmol/L) | 1.3 (0.3) | 1.3 (0.4) |
| Triglycerides (mmol/L) | 1.5 (0.7) | 1.5 (0.7) |
|   |  |  |
| Categorical variables (n, %) |   |   |
| Males | 2804 (48.3) | 2524 (48.1) |
| Current smoker | 1558 (26.8) | 1392 (26.5) |
| History of diabetes | 623 (10.7) | 544 (10.4) |
| History of hypertension | 3592 (61.9) | 3257 (62.1) |
| History of angina | 1559 (26.9) | 1424 (27.2) |
| History of claudication | 390 (6.7) | 354 (6.8) |
| History of myocardial infarction | 776 (13.4) | 708 (13.5) |
| History of stroke or TIA | 649 (11.2) | 586 (11.2) |
| History of vascular disease* | 2565 (44.2) | 2336 (44.5) |

Table 1 Caption: Baseline characteristics of the PROSPER/PHASE study

In Figure [2](#F2) the QQ-plot of the genome-wide association study with baseline LDL levels within the PROSPER/PHASE study is shown. In this plot it is shown that no genomic inflation has occurred in this analyses (lambda = 1.077) and that population stratification is sufficiently controlled for. In Figure [3](#F3) the results of the genome-wide association study with baseline LDL cholesterol levels within the PROSPER/PHASE study are depicted in a Manhattan plot. Forty-two SNPs in five genomic loci, APOE/APOC1, LDLR, FADS2/FEN1, HMGCR, and PSRC1/CELSR5, reached the genome-wide significant p-value of 5.0e-08. In table [2](#T2) a summary of the five genomic loci and their corresponding SNPs is given. The top SNP (rs445925, Chr. 19) with a p-value of p = 2.8e-30 is located within the APOC1 gene and near the APOE gene. Sixteen other SNPs in the same genomic region were also found to be associated with LDL cholesterol levels. The second top SNP (rs6511720, Chr. 19) with a p-value of p = 5.22e-15 is located within the LDLR gene. The three other genomic regions included the HMGCR (Chr.5), FADS2/FEN1 (Chr. 11), PSRC1/CELSR5 (Chr. 1) genes. All 5 genomic loci were previously found in association with LDL cholesterol levels and no novel loci were identified.

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f342/3207930/0ed78ba87219/1471-2350-12-131-2.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3207930_1471-2350-12-131-2.jpg)

QQ-plot for the GWAS on baseline LDL cholesterol in the PROSPER/PHASE study.

### Figure 3.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f342/3207930/8ec4d746045a/1471-2350-12-131-3.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3207930_1471-2350-12-131-3.jpg)

Manhattan plot for the GWAS on baseline LDL cholesterol in the PROSPER/PHASE study.

### Table 2.

| Chr. | Gene | Number of SNPs | TopSNP | Variant | MAF | Beta | SE | p-value | Ref* |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| 19 | APOEAPOC1 | 17 | rs445925 | G > A | 0.11 | -0.33 | 0.03 | 2.8e-30 | (7;11-14;18;19)] |
| 19 | LDLR | 5 | rs6511720 | G > T | 0.13 | -0.19 | 0.02 | 5.2e-15 | (7;11;13;14;19)] |
| 5 | HMGCR | 5 | rs258494 | G > C | 0.38 | 0.10 | 0.02 | 1.3e-09 | (7;11;13;14;19)] |
| 11 | FADS2FEN1 | 14 | rs174541 | C > T | 0.38 | -0.10 | 0.02 | 1.1e-08 | (7;11;13;19)] |
| 1 | PSRC1CELSR5 | 1 | rs602633 | G > T | 0.23 | -0.11 | 0.02 | 5.0e-08 | (7;11-14;16-19)] |

Table 2 Caption: Genomic loci with a genome wide significant p-value < = 5

We replicated the positive associations with genome-wide significant p-values in two independent cohorts, the WOSCOPS study and the CARE trial (table [3](#T3)). Of our five genomic loci that were significantly associated with baseline LDL cholesterol levels we selected the top SNP for replication in both replication cohorts. If the SNP was not genotyped in their GWAS analysis, we chose a proxy in high linkage disequilibrium (r2 > 0.5%) for that SNP. These SNPs were associated with baseline LDL levels before randomisation to statin treatment in both studies. Three out of the five loci (APOE/APOC1; HMGCR; PSRC1/CELSR5) replicated in one or two replication cohorts (p < 0.05). The two other loci (LDLR and FADS2/FEN1) showed similar trends as shown in the discovery cohort, although they did not reach statistical significance (table [3](#T3)).

### Table 3.

|   |   |   | WOSCOPSN = 431 |  |  | CAREN = 751 |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- |
|   |   |   |   |  |  |  |  |  |
| SNP | Gene | Chr. | beta | se | p-value | beta | se | p-value |
| rs445925 | APOE APOC1 | 19 | 0.07 | 0.05 | 0.164 | -0.10 | 0.04 | 0.006 |
| rs6511720 | LDLR | 19 | -0.03 | 0.05 | 0.657 | -0.03 | 0.03 | 0.411 |
| rs258494*1 | HMGCR | 5 | 0.06 | 0.03 | 0.044 | 0.03 | 0.02 | 0.147 |
| rs174541*2 | FADS2FEN1 | 11 | -0.04 | 0.03 | 0.264 | -0.03 | 0.02 | 0.134 |
| rs602633*3 | PSRC1CELSR5 | 1 | -0.09 | 0.04 | 0.026 | -0.05 | 0.02 | 0.035 |

Table 3 Caption: Replication of the 5 significant loci in the WOSCOPS trial and CARE study in association with baseline LDL cholesterol levels

## Discussion

With this first proof-of principle study we show that the PROSPER/PHASE GWAS can confirm previously found genetic associations with LDL-cholesterol levels. This proof-of-principle study indicates that the PROSPER/PHASE study is likely to be capable of detecting genomic regions responsible for the variation in various other quantitative traits. With almost 6000 samples in the PROSPER/PHASE study and access to various replication studies, the PROSPER/PHASE study can provide a good testing frame to identify the genetic variation responsible for the variation in LDL-cholesterol lowering in response to statin treatment.

The main locus responsible for the person-to-person variation in LDL-cholesterol levels is the chromosome 19 locus, which contains the APOE, APOC1, and LDLR genes. Other important loci included the HMGCR locus on chromosome 5, FADS2/FEN1 locus on chromosome 11, and the PSRC1/CELSR5 locus on chromosome 1. The five genomic loci that were associated with variation in LDL-cholesterol levels in the PHASE GWAS study were all genomic regions that were previously reported with LDL cholesterol variation [7;11-19]. Three out of the five loci were replicated in the WOSCOPS study and the CARE trial. The LDLR and FADS2/FEN1 loci were not replicated, however these loci were repeatedly found to be associated with LDL cholesterol levels in various other studies with large number of participants [7;11-14;16;19]. Moreover, both the WOSCOPS and CARE studies had genotype data available in a small number of subjects. Therefore, the lack of replication of these loci in WOSCOPS and CARE was most likely due to lack of statistical power. Finally, since we used in the replication studies a proxy SNP for some of the topSNPs, this may have diluted the effect.

## Conclusions

With this proof-of-principle study we show that the PROSPER/PHASE study can be used to investigate genetic associations in a similar way to population based studies. Moreover, we can also assume from these results that the PROSPER/PHASE study is likely to have sufficient power to detect genome-wide significant hits with large effects for other quantitative traits. The next step of the PROSPER/PHASE study is to identify the genetic variation responsible for the variation in LDL-cholesterol lowering in response to statin treatment.

## Competing interests

The authors declare that they have no competing interests.

## Authors' contributions

ST performed statistical analysis, interpretation of data, and drafted the manuscript. AdC performed statistical analysis, interpretation of data, and drafted the manuscript. IP performed statistical analysis and drafted the manuscript. IF, NS, DS, BB, JD, FS participated in design of the study and collected the data. MC carried out genotyping analyses. PS supervised the laboratory analysis and manuscript editing. RW participated in the design of the study and manuscript editing. JWJ participated in the design of the study, interpretation of the data, and manuscript editing. All authors read and approved the final manuscript.

## Pre-publication history

The pre-publication history for this paper can be accessed here:

[http://www.biomedcentral.com/1471-2350/12/131/prepub](http://www.biomedcentral.com/1471-2350/12/131/prepub)

## Contributor Information

Stella Trompet, Email: s.trompet@lumc.nl.

Anton JM de Craen, Email: craen@lumc.nl.

Iris Postmus, Email: i.postmus@lumc.nl.

Ian Ford, Email: Ian.Ford@glasgow.ac.uk.

Naveed Sattar, Email: Naveed.Sattar@glasgow.ac.uk.

Muriel Caslake, Email: mc2g@clinmed.gla.ac.uk.

David J Stott, Email: David.J.Stott@glasgow.ac.uk.

Brendan M Buckley, Email: brenbuck@ucc.ie.

Frank Sacks, Email: fsacks@hsph.harvard.edu.

James J Devlin, Email: James.Devlin@celera.com.

P Eline Slagboom, Email: p.slagboom@lumc.nl.

Rudi GJ Westendorp, Email: r.g.j.westendorp@lumc.nl.

J Wouter Jukema, Email: j.w.jukema@lumc.nl.

## Acknowledgements and Funding

The research leading to these results has received funding from the European Union's Seventh Framework Programme (FP7/2007-2013) under grant agreement n° HEALTH-F2-2009-223004. For a part of the genotyping we received funding from the Netherlands Consortium of Healthy Aging (NGI: 05060810). This work was performed as part of an ongoing collaboration of the PROSPER study group in the universities of Leiden, Glasgow and Cork. Prof. Dr. J.W. Jukema is an Established Clinical Investigator of the Netherlands Heart Foundation (2001 D 032).

## References

1. McGovern PG, Pankow JS, Shahar E, Doliszny KM, Folsom AR, Blackburn H, Luepker RV. Recent trends in acute coronary heart disease--mortality, morbidity, medical care, and risk factors. The Minnesota Heart Survey Investigators. N Engl J Med. 1996;334(14):884–90. doi: 10.1056/NEJM199604043341403.  [DOI](https://doi.org/10.1056/NEJM199604043341403) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8596571/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Recent%20trends%20in%20acute%20coronary%20heart%20disease--mortality,%20morbidity,%20medical%20care,%20and%20risk%20factors.%20The%20Minnesota%20Heart%20Survey%20Investigators&author=PG%20McGovern&author=JS%20Pankow&author=E%20Shahar&author=KM%20Doliszny&author=AR%20Folsom&volume=334&issue=14&publication_year=1996&pages=884-90&pmid=8596571&doi=10.1056/NEJM199604043341403&)

2. Kalantzi KJ, Milionis HJ, Mikhailidis DP, Goudevenos JA. Lipid lowering therapy in the elderly: is there a benefit? Curr Pharm Des. 2006;12(30):3945–60. doi: 10.2174/138161206778559669.  [DOI](https://doi.org/10.2174/138161206778559669) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17073689/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Pharm%20Des&title=Lipid%20lowering%20therapy%20in%20the%20elderly:%20is%20there%20a%20benefit?&author=KJ%20Kalantzi&author=HJ%20Milionis&author=DP%20Mikhailidis&author=JA%20Goudevenos&volume=12&issue=30&publication_year=2006&pages=3945-60&pmid=17073689&doi=10.2174/138161206778559669&)

3. Kreisberg RA, Oberman A. Clinical review 141: lipids and atherosclerosis: lessons learned from randomized controlled trials of lipid lowering and other relevant studies. J Clin Endocrinol Metab. 2002;87(2):423–37. doi: 10.1210/jc.87.2.423.  [DOI](https://doi.org/10.1210/jc.87.2.423) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11836262/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Endocrinol%20Metab&title=Clinical%20review%20141:%20lipids%20and%20atherosclerosis:%20lessons%20learned%20from%20randomized%20controlled%20trials%20of%20lipid%20lowering%20and%20other%20relevant%20studies&author=RA%20Kreisberg&author=A%20Oberman&volume=87&issue=2&publication_year=2002&pages=423-37&pmid=11836262&doi=10.1210/jc.87.2.423&)

4. Johnson JA, Cavallari LH. Cardiovascular pharmacogenomics. Exp Physiol. 2005;90(3):283–9. doi: 10.1113/expphysiol.2004.028506.  [DOI](https://doi.org/10.1113/expphysiol.2004.028506) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15778411/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Exp%20Physiol&title=Cardiovascular%20pharmacogenomics&author=JA%20Johnson&author=LH%20Cavallari&volume=90&issue=3&publication_year=2005&pages=283-9&pmid=15778411&doi=10.1113/expphysiol.2004.028506&)

5. Goldstein DB, Tate SK, Sisodiya SM. Pharmacogenetics goes genomic. Nat Rev Genet. 2003;4(12):937–47. doi: 10.1038/nrg1229.  [DOI](https://doi.org/10.1038/nrg1229) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14631354/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Genet&title=Pharmacogenetics%20goes%20genomic&author=DB%20Goldstein&author=SK%20Tate&author=SM%20Sisodiya&volume=4&issue=12&publication_year=2003&pages=937-47&pmid=14631354&doi=10.1038/nrg1229&)

6. Shepherd J, Blauw GJ, Murphy MB, Bollen EL, Buckley BM, Cobbe SM, Ford I, Gaw A, Hyland M, Jukema JW, Kamper AM, Macfarlane PW, Meinders AE, Norrie J, Packard CJ, Perry IJ, Stott DJ, Sweeney BJ, Twomey C, Westendorp RG. Pravastatin in elderly individuals at risk of vascular disease (PROSPER): a randomised controlled trial. Lancet. 2002;360(9346):1623–30. doi: 10.1016/S0140-6736(02)11600-X.  [DOI](https://doi.org/10.1016/S0140-6736(02)11600-X) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12457784/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Pravastatin%20in%20elderly%20individuals%20at%20risk%20of%20vascular%20disease%20(PROSPER):%20a%20randomised%20controlled%20trial&author=J%20Shepherd&author=GJ%20Blauw&author=MB%20Murphy&author=EL%20Bollen&author=BM%20Buckley&volume=360&issue=9346&publication_year=2002&pages=1623-30&pmid=12457784&doi=10.1016/S0140-6736(02)11600-X&)

7. Teslovich TM, Musunuru K, Smith AV, Edmondson AC, Stylianou IM, Koseki M, Pirruccello JP, Ripatti S, Chasman DI, Willer CJ, Johansen CT, Fouchier SW, Isaacs A, Peloso GM, Barbalic M, Ricketts SL, Bis JC, Aulchenko YS, Thorleifsson G, Feitosa MF, Chambers J, Orho-Melander M, Melander O, Johnson T, Li X, Guo X, Li M, Shin Cho Y, Jin Go M, Jin Kim Y. et al. Biological, clinical and population relevance of 95 loci for blood lipids. Nature. 2010;466(7307):707–13. doi: 10.1038/nature09270.  [DOI](https://doi.org/10.1038/nature09270) | [PMC free article](/articles/PMC3039276/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20686565/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Biological,%20clinical%20and%20population%20relevance%20of%2095%20loci%20for%20blood%20lipids&author=TM%20Teslovich&author=K%20Musunuru&author=AV%20Smith&author=AC%20Edmondson&author=IM%20Stylianou&volume=466&issue=7307&publication_year=2010&pages=707-13&pmid=20686565&doi=10.1038/nature09270&)

8. Shepherd J, Blauw GJ, Murphy MB, Cobbe SM, Bollen EL, Buckley BM, Ford I, Jukema JW, Hyland M, Gaw A, Lagaay AM, Perry IJ, Macfarlane PW, Meinders AE, Sweeney BJ, Packard CJ, Westendorp RG, Twomey C, Stott DJ. The design of a prospective study of Pravastatin in the Elderly at Risk (PROSPER). PROSPER Study Group. PROspective Study of Pravastatin in the Elderly at Risk. Am J Cardiol. 1999;84(10):1192–7. doi: 10.1016/S0002-9149(99)00533-0.  [DOI](https://doi.org/10.1016/S0002-9149(99)00533-0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10569329/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Cardiol&title=The%20design%20of%20a%20prospective%20study%20of%20Pravastatin%20in%20the%20Elderly%20at%20Risk%20(PROSPER).%20PROSPER%20Study%20Group.%20PROspective%20Study%20of%20Pravastatin%20in%20the%20Elderly%20at%20Risk&author=J%20Shepherd&author=GJ%20Blauw&author=MB%20Murphy&author=SM%20Cobbe&author=EL%20Bollen&volume=84&issue=10&publication_year=1999&pages=1192-7&pmid=10569329&doi=10.1016/S0002-9149(99)00533-0&)

9. Shepherd J, Cobbe SM, Ford I, Isles CG, Lorimer AR, Macfarlane PW, McKillop JH, Packard CJ. Prevention of coronary heart disease with pravastatin in men with hypercholesterolemia. West of Scotland Coronary Prevention Study Group. N Engl J Med. 1995;333(20):1301–7. doi: 10.1056/NEJM199511163332001.  [DOI](https://doi.org/10.1056/NEJM199511163332001) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7566020/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Prevention%20of%20coronary%20heart%20disease%20with%20pravastatin%20in%20men%20with%20hypercholesterolemia.%20West%20of%20Scotland%20Coronary%20Prevention%20Study%20Group&author=J%20Shepherd&author=SM%20Cobbe&author=I%20Ford&author=CG%20Isles&author=AR%20Lorimer&volume=333&issue=20&publication_year=1995&pages=1301-7&pmid=7566020&doi=10.1056/NEJM199511163332001&)

10. Sacks FM, Pfeffer MA, Moye LA, Rouleau JL, Rutherford JD, Cole TG, Brown L, Warnica JW, Arnold JM, Wun CC, Davis BR, Braunwald E. The effect of pravastatin on coronary events after myocardial infarction in patients with average cholesterol levels. Cholesterol and Recurrent Events Trial investigators. N Engl J Med. 1996;335(14):1001–9. doi: 10.1056/NEJM199610033351401.  [DOI](https://doi.org/10.1056/NEJM199610033351401) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8801446/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=The%20effect%20of%20pravastatin%20on%20coronary%20events%20after%20myocardial%20infarction%20in%20patients%20with%20average%20cholesterol%20levels.%20Cholesterol%20and%20Recurrent%20Events%20Trial%20investigators&author=FM%20Sacks&author=MA%20Pfeffer&author=LA%20Moye&author=JL%20Rouleau&author=JD%20Rutherford&volume=335&issue=14&publication_year=1996&pages=1001-9&pmid=8801446&doi=10.1056/NEJM199610033351401&)

11. Aulchenko YS, Ripatti S, Lindqvist I, Boomsma D, Heid IM, Pramstaller PP, Penninx BW, Janssens AC, Wilson JF, Spector T, Martin NG, Pedersen NL, Kyvik KO, Kaprio J, Hofman A, Freimer NB, Jarvelin MR, Gyllensten U, Campbell H, Rudan I, Johansson A, Marroni F, Hayward C, Vitart V, Jonasson I, Pattaro C, Wright A, Hastie N, Pichler I, Hicks AA. et al. Loci influencing lipid levels and coronary heart disease risk in 16 European population cohorts. Nat Genet. 2009;41(1):47–55. doi: 10.1038/ng.269.  [DOI](https://doi.org/10.1038/ng.269) | [PMC free article](/articles/PMC2687074/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19060911/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Loci%20influencing%20lipid%20levels%20and%20coronary%20heart%20disease%20risk%20in%2016%20European%20population%20cohorts&author=YS%20Aulchenko&author=S%20Ripatti&author=I%20Lindqvist&author=D%20Boomsma&author=IM%20Heid&volume=41&issue=1&publication_year=2009&pages=47-55&pmid=19060911&doi=10.1038/ng.269&)

12. Barber MJ, Mangravite LM, Hyde CL, Chasman DI, Smith JD, McCarty CA, Li X, Wilke RA, Rieder MJ, Williams PT, Ridker PM, Chatterjee A, Rotter JI, Nickerson DA, Stephens M, Krauss RM. Genome-wide association of lipid-lowering response to statins in combined study populations. PLoS One. 2010;5(3):e9763.. doi: 10.1371/journal.pone.0009763.  [DOI](https://doi.org/10.1371/journal.pone.0009763) | [PMC free article](/articles/PMC2842298/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20339536/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=Genome-wide%20association%20of%20lipid-lowering%20response%20to%20statins%20in%20combined%20study%20populations&author=MJ%20Barber&author=LM%20Mangravite&author=CL%20Hyde&author=DI%20Chasman&author=JD%20Smith&volume=5&issue=3&publication_year=2010&pages=e9763.&pmid=20339536&doi=10.1371/journal.pone.0009763&)

13. Kathiresan S, Melander O, Guiducci C, Surti A, Burtt NP, Rieder MJ, Cooper GM, Roos C, Voight BF, Havulinna AS, Wahlstrand B, Hedner T, Corella D, Tai ES, Ordovas JM, Berglund G, Vartiainen E, Jousilahti P, Hedblad B, Taskinen MR, Newton-Cheh C, Salomaa V, Peltonen L, Groop L, Altshuler DM, Orho-Melander M. Six new loci associated with blood low-density lipoprotein cholesterol, high-density lipoprotein cholesterol or triglycerides in humans. Nat Genet. 2008;40(2):189–97. doi: 10.1038/ng.75.  [DOI](https://doi.org/10.1038/ng.75) | [PMC free article](/articles/PMC2682493/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18193044/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Six%20new%20loci%20associated%20with%20blood%20low-density%20lipoprotein%20cholesterol,%20high-density%20lipoprotein%20cholesterol%20or%20triglycerides%20in%20humans&author=S%20Kathiresan&author=O%20Melander&author=C%20Guiducci&author=A%20Surti&author=NP%20Burtt&volume=40&issue=2&publication_year=2008&pages=189-97&pmid=18193044&doi=10.1038/ng.75&)

14. Kathiresan S, Willer CJ, Peloso GM, Demissie S, Musunuru K, Schadt EE, Kaplan L, Bennett D, Li Y, Tanaka T, Voight BF, Bonnycastle LL, Jackson AU, Crawford G, Surti A, Guiducci C, Burtt NP, Parish S, Clarke R, Zelenika D, Kubalanza KA, Morken MA, Scott LJ, Stringham HM, Galan P, Swift AJ, Kuusisto J, Bergman RN, Sundvall J, Laakso M. et al. Common variants at 30 loci contribute to polygenic dyslipidemia. Nat Genet. 2009;41(1):56–65. doi: 10.1038/ng.291.  [DOI](https://doi.org/10.1038/ng.291) | [PMC free article](/articles/PMC2881676/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19060906/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Common%20variants%20at%2030%20loci%20contribute%20to%20polygenic%20dyslipidemia&author=S%20Kathiresan&author=CJ%20Willer&author=GM%20Peloso&author=S%20Demissie&author=K%20Musunuru&volume=41&issue=1&publication_year=2009&pages=56-65&pmid=19060906&doi=10.1038/ng.291&)

15. Ma L, Yang J, Runesha HB, Tanaka T, Ferrucci L, Bandinelli S, Da Y. Genome-wide association analysis of total cholesterol and high-density lipoprotein cholesterol levels using the Framingham heart study data. BMC Med Genet. 2010;11:55. doi: 10.1186/1471-2350-11-55.  [DOI](https://doi.org/10.1186/1471-2350-11-55) | [PMC free article](/articles/PMC2867786/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20370913/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Med%20Genet&title=Genome-wide%20association%20analysis%20of%20total%20cholesterol%20and%20high-density%20lipoprotein%20cholesterol%20levels%20using%20the%20Framingham%20heart%20study%20data&author=L%20Ma&author=J%20Yang&author=HB%20Runesha&author=T%20Tanaka&author=L%20Ferrucci&volume=11&publication_year=2010&pages=55&pmid=20370913&doi=10.1186/1471-2350-11-55&)

16. Sandhu MS, Waterworth DM, Debenham SL, Wheeler E, Papadakis K, Zhao JH, Song K, Yuan X, Johnson T, Ashford S, Inouye M, Luben R, Sims M, Hadley D, McArdle W, Barter P, Kesäniemi YA, Mahley RW, McPherson R, Grundy SM. Wellcome Trust Case Control Consortium; Bingham SA, Khaw KT, Loos RJ, Waeber G, Barroso I, Strachan DP, Deloukas P, Vollenweider P, Wareham NJ, Mooser V. LDL-cholesterol concentrations: a genome-wide association study. Lancet. 2008;371(9611):483–91. doi: 10.1016/S0140-6736(08)60208-1.  [DOI](https://doi.org/10.1016/S0140-6736(08)60208-1) | [PMC free article](/articles/PMC2292820/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18262040/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=LDL-cholesterol%20concentrations:%20a%20genome-wide%20association%20study&author=MS%20Sandhu&author=DM%20Waterworth&author=SL%20Debenham&author=E%20Wheeler&author=K%20Papadakis&volume=371&issue=9611&publication_year=2008&pages=483-91&pmid=18262040&doi=10.1016/S0140-6736(08)60208-1&)

17. Wallace C, Newhouse SJ, Braund P, Zhang F, Tobin M, Falchi M Ahmadi K, Dobson RJ, Marçano AC, Hajat C, Burton P, Deloukas P, Brown M, Connell JM, Dominiczak A, Lathrop GM, Webster J, Farrall M, Spector T, Samani NJ, Caulfield MJ, Munroe PB. Genome-wide association study identifies genes for biomarkers of cardiovascular disease: serum urate and dyslipidemia. Am J Hum Genet. 2008;82(1):139–49. doi: 10.1016/j.ajhg.2007.11.001.  [DOI](https://doi.org/10.1016/j.ajhg.2007.11.001) | [PMC free article](/articles/PMC2253977/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18179892/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=Genome-wide%20association%20study%20identifies%20genes%20for%20biomarkers%20of%20cardiovascular%20disease:%20serum%20urate%20and%20dyslipidemia&author=C%20Wallace&author=SJ%20Newhouse&author=P%20Braund&author=F%20Zhang&author=M%20Tobin&volume=82&issue=1&publication_year=2008&pages=139-49&pmid=18179892&doi=10.1016/j.ajhg.2007.11.001&)

18. Waterworth DM, Ricketts SL, Song K, Chen L, Zhao JH, Ripatti S, Aulchenko YS, Zhang W, Yuan X, Lim N, Luan J, Ashford S, Wheeler E, Young EH, Hadley D, Thompson JR, Braund PS, Johnson T, Struchalin M, Surakka I, Luben R, Khaw KT, Rodwell SA, Loos RJ, Boekholdt SM, Inouye M, Deloukas P, Elliott P, Schlessinger D, Sanna S. et al. Genetic variants influencing circulating lipid levels and risk of coronary artery disease. Arterioscler Thromb Vasc Biol. 2010;30(11):2264–76. doi: 10.1161/ATVBAHA.109.201020.  [DOI](https://doi.org/10.1161/ATVBAHA.109.201020) | [PMC free article](/articles/PMC3891568/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20864672/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arterioscler%20Thromb%20Vasc%20Biol&title=Genetic%20variants%20influencing%20circulating%20lipid%20levels%20and%20risk%20of%20coronary%20artery%20disease&author=DM%20Waterworth&author=SL%20Ricketts&author=K%20Song&author=L%20Chen&author=JH%20Zhao&volume=30&issue=11&publication_year=2010&pages=2264-76&pmid=20864672&doi=10.1161/ATVBAHA.109.201020&)

19. Willer CJ, Sanna S, Jackson AU, Scuteri A, Bonnycastle LL, Clarke R, Heath SC, Timpson NJ, Najjar SS, Stringham HM, Strait J, Duren WL, Maschio A, Busonero F, Mulas A, Albai G, Swift AJ, Morken MA, Narisu N, Bennett D, Parish S, Shen H, Galan P, Meneton P, Hercberg S, Zelenika D, Chen WM, Li Y, Scott LJ, Scheet PA. et al. Newly identified loci that influence lipid concentrations and risk of coronary artery disease. Nat Genet. 2008;40(2):161–9. doi: 10.1038/ng.76.  [DOI](https://doi.org/10.1038/ng.76) | [PMC free article](/articles/PMC5206900/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18193043/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Newly%20identified%20loci%20that%20influence%20lipid%20concentrations%20and%20risk%20of%20coronary%20artery%20disease&author=CJ%20Willer&author=S%20Sanna&author=AU%20Jackson&author=A%20Scuteri&author=LL%20Bonnycastle&volume=40&issue=2&publication_year=2008&pages=161-9&pmid=18193043&doi=10.1038/ng.76&)
