# Genetic Variations in Multiple Drug Action Pathways and Survival in Advanced-Stage Non-small Cell Lung Cancer Treated with Chemotherapy

## Metadata
**Authors:** Yafei Li, Zhifu Sun, Julie M Cunningham, Marie C Aubry, Jason A Wampfler, Gary A Croghan, Cassandra Johnson, Danli Wu, Jeremiah A Aakre, Julian Molina, Liewei Wang, V Shane Pankratz, Ping Yang
**Journal:** Clinical cancer research : an official journal of the American Association for Cancer Research
**Date:** 2011 Jun 1
**DOI:** [10.1158/1078-0432.CCR-10-2877](https://doi.org/10.1158/1078-0432.CCR-10-2877)
**PMID:** 21636554
**PMCID:** PMC3124814
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3124814/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC3124814/pdf/nihms-290601.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC3124814/pdf/nihms-290601.pdf)

## Abstract

**Purpose:** 
Variations in genes related to anticancer drugs' biologic activity could influence treatment responses and lung cancer prognosis. Genetic variants in four biological pathways, i.e., glutathione metabolism, DNA repair, cell cycle, and EGFR, were systematically investigated to examine their association with survival in advanced-stage NSCLC treated with chemotherapy.

**Experimental Design:** 
A total of 894 tagging single-nucleotide polymorphisms (tagSNPs) in 70 genes from the four pathways were genotyped and analyzed in a 1076-patient cohort. Association with overall survival was analyzed at single-SNP and whole-gene levels within all patients and major chemotherapy agent combination groups.

**Results:** 
A poorer overall survival was observed in patients with genetic variations in GSS (glutathione pathway) and MAP3K1 (EGFR pathway) (HR=1.45, 95% CI=1.20–1.70 and HR=1.25, 95% CI=1.05–1.50, respectively). In stratified analysis on patients receiving platinum plus taxane treatment, we observed a hazardous effect on overall survival by MAP3K1 variant (HR=1.38, 95% CI =1.11–1.72) and a protective effect by RAF1 (HR=0.64, 95% CI=0.5–0.82) in the EGFR pathway. In patients receiving platinum plus gemcitabine treatment, RAF and GPX5 (glutathione pathway) genetic variations showed protective effects on survival (HR=0.54, 95% CI=0.38–0.77; HR=0.67, 95% CI=0.52–0.85, respectively); in contrast, NRAS (EGFR pathway) and GPX7 (glutathione pathway) variations showed hazardous effects on overall survival (HR=1.91, 95% CI=1.30–2.80; HR=1.83, 95% CI=1.27–2.63, respectively). All genes that harbored these significant SNPs remained significant by whole-gene analysis.

**Conclusion:** 
Common genetic variations in genes of EGFR and glutathione pathways may be associated with overall survival among patients with advanced-stage NSCLC treated with platinum, taxane, and/or gemicitabine combinations.

Keywords: non-small cell lung cancer, survival, single-nucleotide polymorphisms, pathway, chemotherapy

### Purpose

Variations in genes related to anticancer drugs' biologic activity could influence treatment responses and lung cancer prognosis. Genetic variants in four biological pathways, i.e., glutathione metabolism, DNA repair, cell cycle, and EGFR, were systematically investigated to examine their association with survival in advanced-stage NSCLC treated with chemotherapy.

### Experimental Design

A total of 894 tagging single-nucleotide polymorphisms (tagSNPs) in 70 genes from the four pathways were genotyped and analyzed in a 1076-patient cohort. Association with overall survival was analyzed at single-SNP and whole-gene levels within all patients and major chemotherapy agent combination groups.

### Results

A poorer overall survival was observed in patients with genetic variations in *GSS* (glutathione pathway) and *MAP3K1* (EGFR pathway) (HR=1.45, 95% CI=1.20–1.70 and HR=1.25, 95% CI=1.05–1.50, respectively). In stratified analysis on patients receiving platinum plus taxane treatment, we observed a hazardous effect on overall survival by *MAP3K1* variant (HR=1.38, 95% CI =1.11–1.72) and a protective effect by *RAF1* (HR=0.64, 95% CI=0.5–0.82) in the EGFR pathway. In patients receiving platinum plus gemcitabine treatment, *RAF* and *GPX5* (glutathione pathway) genetic variations showed protective effects on survival (HR=0.54, 95% CI=0.38–0.77; HR=0.67, 95% CI=0.52–0.85, respectively); in contrast, *NRAS* (EGFR pathway) and *GPX7* (glutathione pathway) variations showed hazardous effects on overall survival (HR=1.91, 95% CI=1.30–2.80; HR=1.83, 95% CI=1.27–2.63, respectively). All genes that harbored these significant SNPs remained significant by whole-gene analysis.

### Conclusion

Common genetic variations in genes of *EGFR* and glutathione pathways may be associated with overall survival among patients with advanced-stage NSCLC treated with platinum, taxane, and/or gemicitabine combinations.

## Introduction

Non-small cell lung cancer (NSCLC) represents more than 80% of lung cancer diagnoses.([1](#R1)) For advanced-stage (stage III and stage IV) NSCLC, 5-year survival varies widely (3–50%) depending on the number of lymph nodes involved, resectability, and tumor histology.([2](#R2)) Established prognostic factors include stage, gender, age, and performance status.([3](#R3)) Chemotherapy remains the major component of the standard care in conjunction with radiation therapy and supportive care for patients with advanced-stage lung cancer. For a given treatment, usually prescribed by a standard dosing protocol, survival time widely varies even after stratification by tumor stage, histology, and other clinical information, highlighting the need for improved predictive markers. In addition to clinical and biological behaviors of the tumor, multiple drug metabolism systems of the host also have an impact on the outcomes for lung cancer patients. Pharmacogenomic studies have defined gene variations responsible for varied drug efficacy. Patient-specific genetic profiles of genetic variants related to drugs' biological activity may help improve drug selection.

The glutathione metabolic pathway (GSH pathway) is directly involved in the detoxification or inactivation of multiple anticancer drugs, with evolving clinical relevance in lung cancer treatment.([4](#R4)) The efficacy of anticancer drugs is highly influenced by cellular DNA repair capacity; increasing evidence shows reduced DNA repair capacity resulting from genetic polymorphisms of various DNA repair genes being associated with improved survival after platinum-based chemotherapy.([5](#R5)–[10](#R10)) Taxanes (paclitaxel and docetaxel) are a class of anticancer agents, which exert their cytotoxic effects on spindle microtubules dynamics causing cell cycle arrest and apoptosis.([11](#R11), [12](#R12)) These agents have been widely used as active chemotherapeutic agents in the treatment of NSCLC. Genes that regulate the cell cycle are thought to be involved in the biological activities of taxanes, although the impact of genetic variants in cell cycle pathway genes on clinic outcomes is unclear. The epidermal growth factor receptor (EGFR) is a tyrosine kinase transmembrane receptor in the ErB family of receptors expressed on the surface of epithelial cells, including NSCLC, regulating important processes in cell survival, cell cycle progression, tumor invasion, and angiogenesis.([13](#R13), [14](#R14)) The EGFR pathway has been recognized as an important drug target. The EGFR tyrosine kinase inhibitors (TKIs), namely Gefitinib (IRESSA) and Erlotinib (Tarceva) are extensively studied in NSCLC and are currently incorporated into clinical practice. The EGFR pathway is associated with platinum-chemotherapy sensitivity in cancer patients in a recent study.([15](#R15))

A variety of chemotherapeutic drugs are used, and most advanced-stage NSCLC patients receive more than one drug; therefore, genetic variants of genes in multiple major drug action pathways may have combined effects on outcomes. However, most studies thus far have focused on limited candidate SNPs from a few selected genes. The influence of genetic variation in multiple pathways on NSCLC prognosis remains poorly understood. In this study, genetic variants of 70 key genes in four candidate pathways related to major chemotherapy drug actions, i.e., GSH, DNA repair, cell cycle, and EGFR pathways ([Supplementary Figure 1](#SD14)), were systematically investigated in a prospectively followed cohort of advanced-stage NSCLC patients. Our goal was to examine whether SNP markers from the selected pathways are collectively associated with lung cancer overall survival among NSCLC patients receiving chemotherapy.

## Materials and Methods

### Patient Cohort

Lung cancer patients were identified and enrolled between 1997 and 2008. Detailed procedures of patient enrollment, diagnosis, data collection, and follow-up have been previously described.([16](#R16)–[18](#R18)) Briefly, new cases diagnosed with lung cancer were identified by a daily electronic pathology reporting system. Once identified, patients were invited to participate and enrolled after their consent. The overall participation and blood sample donation rates were 87%. Demographic and tobacco history data were obtained from medical records and interview. When the patient received any therapy elsewhere, authorization for the release of medical information was requested, and copies of the relevant medical records were abstracted. Treatment information was obtained from oncology records, treatment data, outside medical records, and follow up questionnaires. Surgery was defined as a treatment when the patient had any pulmonary resection for the primary tumor, including pneumonectomy, bilobectomy, lobectomy, segmentectomy, and wedge resection. Chemotherapy agents, dosage, and date treatment started and ended were abstracted. Radiation was categorized as radiation to the chest only, radiation to another place with specific location the radiation occurred, gamma knife, and PCI; elapsed days for radiation, energy, radiation dose, times per day, and fractions were also abstracted. Clinical staging and recurrence or progression was determined by results from available chest radiography, computerized tomography, bone scans, position emission tomography scans, and magnetic resonance imaging. All patients were actively followed beginning within 6 months of diagnosis, with subsequent annual follow-up by mailed questionnaires. Annual verification of patients' vital status was accomplished through the use of Mayo Clinic's electronic medical notes and registration database, next-of-kin reports, death certificates, and obituary documents filed in the patients' medical records, as well as through the Mayo Clinic Tumor Registry and Social Security Death Index website. Well-trained abstractors conducted information abstraction and data entry. Research protocols were approved by the Mayo Clinic's Institutional Review Board.

### Pathway, gene and SNP selection

We focused on four common candidate pathways related to major chemotherapy drug actions, i.e., GSH, DNA repair for platinum based agents, cell cycle for taxanes, and EGFR pathways following a review of the literature in lung cancer therapy. Note that each of the four pathways may relate to multiple drug actions, particularly for individual genes; specifically, a gene can participate in multiple drug action pathways other than the one that was selected. Seventy genes were selected from four pathways following a review of the literature ([Supplementary Table 1](#SD1)). TagSNPs ([19](#R19), [20](#R20)) were identified via SNPApp, a tagSNP selection program developed by the Bioinformatics Core at Mayo Clinic, Rochester, MN. SNPApp queries multiple public SNP data repositories (Hapmap, SeattleSNPs, and National Institute for Environmental Health Science SNPs) that contain information on known SNPs, using ldSelect([21](#R21)) to identify tagSNPs for the genes or regions of interest. SNPs within 5kb of each gene with a minor allele frequency (MAF) ≥ 0.05 for European populations were used as candidate SNPs, and tagSNPs were identified with a pair-wise linkage disequilibrium threshold of r^2^ ≥ 0.8. For a given gene, the SNP selection procedure used the SNP data repository with the greatest number of SNPs with a MAF ≥ 0.05 and the greatest number of linkage disequilibrium bins that met Illumina Golden Gate Assay quality score thresholds. Nonsynomyous SNPs were preferentially selected as tags when they were identified, provided that their metrics were equivalent to those of alternative tagSNPs.

### Genotyping and Quality control

A total of 1025 tagSNPs were genotyped in the Mayo Clinic Genomics Shared Resource using a custom-designed Illumina GoldenGate panel. Concordance among the three genomic control DNA samples present in duplicate was 100%. Subjects with a call rate was over 90% for all subjects; 111 SNPs failed genotyping. Of the SNPs with genotyping data, the call rate was >95%, and the minor allele frequency was >0.001 for all. SNPs with a Hardy-Weinberg equilibrium test p-value<1×10^−7^ (n=11) and/or monomorphic (n=9) were excluded, resulting in 894 SNPs in the analyses.

### Chemotherapy agent information

Chemotherapy regimens mostly fall into four groups ([Supplementary Table 2](#SD1)): (1) platinum agents (P) including carboplatin and cisplatin, (2) taxane agents (T), paclitaxel and docetaxel, (3) gemcitabine (G), and (4) EGFR inhibitors (E), Gefitinib (IRESSA) and Erlotinib (Tarceva). Only a very small group of patients received drugs outside of these four groups; therefore, patients were divided into two subgroups based on the most commonly used combinations: platinum plus taxanes with or without other agents (PT group), platinum plus gemcitabine with or without other agents (PG group). We performed subsequent analyses on the significant SNPs in more restricted drug combinations: (1) platinum plus taxanes alone (PT-only group); (2) platinum plus any other agents except taxanes (P/no-T group); (3) platinum plus any other agents except gemcitabine (P/no-G group); (4) platinum plus taxanes without EGFR inhibitors (PT/no-E); (5) platinum plus gemcitabine without EGFR inhibitors (PG/no-E); and (6) EGFR inhibitors alone or plus any other agents (E group).

### Outcomes

Overall survival time was used as the primary endpoint, defined as the time from lung cancer diagnosis to either death or the last known date alive. Patients known to be alive were censored at the time of last contact.

### Statistical analysis

#### Single-SNP association analysis

A backward selection process was used to screen potential confounders: age at diagnosis, sex, race, smoking status, stage, histological types, comorbidity, and treatment modality. The significant variables were retained as covariates in all subsequent analyses. A Cox regression model was used to assess the associations between each SNP's genotypes and overall survival. The primary test of association was based on a genetic model free scheme (2 degrees of freedom). In subgroups, for SNPs where five or fewer minor allele homozygotes were observed, homozygote genotypes were combined with heterozygotes. If combined frequency was still five or fewer, then the SNP was removed. Hazard ratios (HRs) and 95% confidence intervals (CI) were estimated, comparing patients carrying one and two minor alleles individually to patients carrying two major alleles. To account for multiple comparisons in the SNP-based analysis, q-values set at 0.20 were computed using the single-SNP p-values to quantify the probability that a p-value might be a false positive,([22](#R22)) accepting a false discovery rate of 20%. The significant SNPs were further tested in refined treatment groups using the same methods outlined above. Kaplan-Meier curves and log-rank tests were used to assess the differences in survival time by individual SNPs.

#### Principal Components Analysis (PCA) by whole-gene

In order to assess whether different analytical approaches resulted in consistent findings, we performed whole-gene tests of association using a PCA approach. We utilized a genetic model free scheme that used two indicator variables for each SNP, one for the heterozygous and one for the homozygous minor allele carriers and extracted sufficient principal components to capture 90% of the SNP. The resulting collection of principal component variables was used for an omnibus test of significance for the association between each gene and survival in the multivariable Cox proportional hazards regression models. P-values for the global tests were obtained, along with summaries of the PCA.

All reported p-values were based on a two-sided test. All statistical analyses were performed using SAS, version 9 (SAS Institute, Inc.) or R-project software.

## Results

[Table 1](#T1) provides descriptive characteristics of 1076 advanced-stage NSCLC patients. The median overall survival time was 1.7 years with 83.8% deceased within the 11-year follow-up period. Of the 1076 patients, 962 (89.4%) had chemotherapy, of which 657 (68.3%) were treated with PT, and 305 (31.7%) with PG ([Supplementary Table 2](#SD1)). Eighty-eight percent of the patients had documented lung cancer progression or recurrence. In the PT group, gender and treatment modality were significant confounders; and in the PG group, gender, histological types, and treatment modality were significant confounders ([Supplementary Table 3](#SD1)).

### Table 1.

| Patient characteristics | All patients n=1076 (%) | Patients by treatment groups |  |  |
| --- | --- | --- | --- | --- |
| PT groupa n=657 (%) | PG groupb n=305 (%) | Other patientsc n=133 (%) |  |  |
| Alive/death | 174/902 (16.2/83.8) | 108/549 (16.4/83.6) | 27/278 (8.9/91.2) | 23/110 (17.3/82.7) |
| Survival time (years) |   |   |   |   |
| Median (range) | 1.7 (0.1–11.0) | 1.8 (0.1–10.7) | 1.9 (0.2–10.1) | 1.4 (0.2–8.2) |
| Mean(SD) | 2.4(2.1) | 2.6(2.2) | 2.3(1.7) | 2.1(1.6) |
| Age at diagnosis (years) |   |   |   |   |
| Mean(SD) | 62.4(11.0) | 61.0(10.5) | 61.0(10.9) | 64.9(11.4) |
| ≤ 50 | 176(16.4) | 125(19.0) | 59(19.3) | 16(12.0) |
| 50–70 | 592(55.0) | 383(58.3) | 169(55.4) | 67(50.4) |
| ≥70 | 308(28.6) | 149(22.7) | 77(25.3) | 50(37.6) |
| Men/women (%) | 597/479 (55.5/44.5) | 360/297 (54.8/45.2) | 159/146 (52.1/47.9) | 74/59 (55.6/44.4) |
| Race |   |   |   |   |
| Non-Caucasian | 57(5.3) | 37(5.6) | 14(4.6) | 6(4.5) |
| Caucasian | 1019(94.7) | 620(94.4) | 291(95.4) | 127(95.5) |
| Smoking history at diagnosis (%) |   |   |   |   |
| Never | 197(18.3) | 133(20.2) | 79(25.9) | 24(18.1) |
| Former | 542(50.4) | 314(47.8) | 142(46.6) | 69(51.9) |
| Current | 337(31.3) | 210(32.0) | 84(27.5) | 40(30.1) |
| Histological type |   |   |   |   |
| Adenocarcinoma/BACd | 637(59.2) | 387(58.9) | 193(63.3) | 81(60.9) |
| Squamous | 197(18.3) | 118(18.0) | 43(14.1) | 22(16.5) |
| Other and unclassified NSCLCe | 242(22.5) | 152(23.1) | 69(22.6) | 30(22.6) |
| Stage |   |   |   |   |
| IIIA | 304(28.3) | 182(27.7) | 60(19.7) | 29(21.8) |
| IIIB | 263(24.4) | 160(24.4) | 81(26.6) | 29(21.8) |
| IV | 509(47.3) | 315(48.0) | 164(53.8) | 75(56.4) |
| Comorbidity diseases |   |   |   |   |
| Pulmonary diseases (%) |   |   |   |   |
| Yes | 542(50.4) | 322(49.0) | 123(40.3) | 67(50.4) |
| No | 534(49.6) | 335(51.0) | 182(59.7) | 66(49.6) |
| Non-pulmonary diseases (%) |   |   |   |   |
| Yes | 692(64.3) | 434(66.1) | 187(61.3) | 72(54.1) |
| No | 384(35.7) | 223(33.9) | 118(38.7) | 61(45.9) |
| Other cancersf (%) |   |   |   |   |
| Yes | 292(27.1) | 172(26.2) | 72(23.6) | 35(26.3) |
| No | 784(72.9) | 485(73.8) | 233(76.4) | 98(73.7) |
| Lung cancer recurrence/ progression/new primary |   |   |   |   |
| Yes (%) | 947(88.0) | 581(88.4) | 290(95.1) | 113(85.0) |
| No (%) | 129(12.0) | 76(11.6) | 15(4.9) | 20(15.0) |
| Treatment modality |   |   |   |   |
| Only chemotherapy (%) | 319(29.7) | 209(31.8) | 116(38.0) | 52(39.1) |
| Only radiation (%) | 36(3.4) | 0(0) | 0(0) | 0(0) |
| Both chemotherapy and radiation (%) | 358(33.3) | 250(38.1) | 118(38.7) | 49(36.8) |
| Both chemotherapy and surgery (%) | 86(8.0) | 62(9.4) | 21(6.9) | 11(8.3) |
| Both radiation and surgery (%) | 42(3.9) | 0(0) | 0(0) | 0(0) |
| Chemotherapy, radiation, and surgery (%) | 199(18.5) | 136(20.7) | 50(16.4) | 21(15.8) |
| Other treatmentg | 36(3.4) | 0(0) | 0(0) | 0(0) |

Table 1 Caption: Characteristics of the patients

The analytic strategy is presented in [Figure 1](#F1). Significant SNP associations in each treatment group are summarized in [Table 2](#T2). A poorer overall survival was observed in all patients with minor alleles of rs17309872 in *GSS* from the GSH pathway and rs17661089 in *MAP3K1* from the EGFR pathway [HR=1.45, 95% CI=1.20–1.7 and HR=1.25, 95% CI=1.05–1.50, respectively] ([Table 2](#T2) and [Supplementary Figure 1](#SD14)). By gene-level tests, *GSS* and *MAP3K1* remained significant (p=3.76×10^−2^ and 1.80×10^−2^, respectively).

### Figure 1. Study design and key findings.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1102/3124814/ebc4019df35e/nihms-290601-f0001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3124814_nihms-290601-f0001.jpg)

PT group, platinum plus taxanes with or without other drugs; PG group, platinum plus gemcitabine with or without other agents; PT-only group, platinum plus taxanes alone; P/no-T group, platinum plus any other agents except taxanes; PT/no-E group, both platinum and taxanes without EGFR inhibitors; E group, EGFR inhibitors alone or plus any other agents; P/no-G group, platinum plus any other agents except gemcitabine; PG/ no-E group, both platinum and gemcitabine without EGFR inhibitors.

### Table 2.

| Pathway | Gene | SNP | Genotype | Alive/Death | MAFa | Median survival (years) | HR (95% CI)b | p-valueb | q-value |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| All patients (n=1076) |  |   |   |   |   |   |   |   |   |
| GSH | GSS | rs17309872 |   | 174/902 | 0.06 |   |   |   |   |
|   |   |   | TT | 161/783 |   | 1.80 | Ref |   |   |
|   |   |   | TA | 13/117 |   | 1.46 | 1.44(1,19–1.76) | 2.39E-04 |   |
|   |   |   | TA+AA |   |   |   | 1.45(1.20–1.77) | 1.47E-04 |   |
|   |   |   | Model free testd |   |   |   |   | 3.28E-04 | 1.50E-01 |
| EGFR | MAP3K1 | rs17661089 |   | 174/902 | 0.08 |   |   |   |   |
|   |   |   | AA | 151/750 |   | 1.79 | Ref |   |   |
|   |   |   | AG | 23/145 |   | 1.55 | 1.22(1.02–1.46) | 2.92E-02 |   |
|   |   |   | GG | 0/7 |   | 0.86 | 4.27(1.90–9.59) | 4.32E-04 |   |
|   |   |   | AG+GG |   |   |   | 1.25(1.05–1.50) | 1.14E-02 |   |
|   |   |   | Model free testd |   |   |   |   | 2.59E-04 | 1.50E-01 |
| PT group (n=657) c |  |   |   |   |   |   |   |   |   |
| EGFR | MAP3K1 | rs17661089 |   | 108/549 | 0.09 |   |   |   |   |
|   |   |   | AA | 96/449 |   | 1.92 | Ref |   |   |
|   |   |   | AG | 12/94 |   | 1.58 | 1.32(1.06, 1.65) | 1.46E-02 |   |
|   |   |   | GG | 0/6 |   | 0.83 | 4.92(2.18, 11.12) | 1.24E-04 |   |
|   |   |   | AG+GG |   |   |   | 1.38(1.11, 1.72) | 3.79E-03 |   |
|   |   |   | Model free testd |   |   |   |   | 5.24E-05 | 4.23E-02 |
| EGFR | MAP3K1 | rs16886403 |   | 108/549 | 0.11 |   |   |   |   |
|   |   |   | AA | 94/422 |   | 1.92 | Ref |   |   |
|   |   |   | AG | 14/117 |   | 1.68 | 1.31(1.06, 1.61) | 1.10E-02 |   |
|   |   |   | GG | 0/10 |   | 0.99 | 2.62(1.39, 4.92) | 2.79E-03 |   |
|   |   |   | AG+GG |   |   |   | 1.36(1.11, 1.66) | 2.61E-03 |   |
|   |   |   | Model free testd |   |   |   |   | 7.82E-04 | 1.58E-01 |
| EGFR | MAP3K1 | rs726501 |   | 108/549 | 0.11 |   |   |   |   |
|   |   |   | GG | 94/422 |   | 1.92 | Ref |   |   |
|   |   |   | GA | 14/117 |   | 1.68 | 1.31(1.06, 1.61) | 1.10E-02 |   |
|   |   |   | AA | 0/10 |   | 0.99 | 2.62(1.39, 4.92) | 2.79E-03 |   |
|   |   |   | GA+AA |   |   |   | 1.36(1.11, 1.66) | 2.61E-03 |   |
|   |   |   | Model free testd |   |   |   |   | 7.82E-04 | 1.58E-01 |
| EGFR | RAF1 | rs11710163 |   | 108/549 | 0.08 |   |   |   |   |
|   |   |   | AA | 86/472 |   | 1.80 | Ref |   |   |
|   |   |   | AG | 22/73 |   | 2.18 | 0.62(0.49, 0.8) | 2.02E-04 |   |
|   |   |   | AG+GG |   |   |   | 0.64(0.50, 0.82) | 3.78E-04 |   |
|   |   |   | Model free testd |   |   |   |   | 6.70E-04 | 1.58E-01 |
| PG Group (n=305) e |  |   |   |   |   |   |   |   |   |
| EGFR | RAF1 | rs11710163 |   | 27/278 | 0.08 |   |   |   |   |
|   |   |   | AA | 20/239 |   | 1.83 | Ref |   |   |
|   |   |   | AG | 6/38 |   | 2.23 | 0.55(0.38, 0.79) | 1.09E-03 |   |
|   |   |   | AG+GG |   |   |   | 0.54(0.38, 0.77) | 6.84E-04 |   |
|   |   |   | Model free testd |  |   |   |   | 6.84E-04 | 1.98-01 |
| EGFR | NRAS | rs1065634 |   | 27/278 | 0.05 |   |   |   |   |
|   |   |   | AA | 27/247 |   | 1.92 | Ref |   |   |
|   |   |   | AG | 0/30 |   | 1.67 | 1.92(1.30–2.83) | 1.03E-03 |   |
|   |   |   | AG+GG |   |   |   | 1.91(1.30, 2.80) | 1.00E-03 |   |
|   |   |   | Model free testd |  |   |   |   | 1.00E-03 | 1.98-01 |
| GSH | GPX5 | rs451774 |   | 27/278 | 0.29 |   |   |   |   |
|   |   |   | AA | 9/148 |   | 1.66 | Ref |   |   |
|   |   |   | AG | 15/104 |   | 2.27 | 0.61(0.47, 0.79) | 2.06E-04 |   |
|   |   |   | GG | 3/26 |   | 2.15 | 0.99(0.65, 1.51) | 9.55E-01 |   |
|   |   |   | AG+GG |   |   |   | 0.67(0.52, 0.85) | 1.06E-03 |   |
|   |   |   | Model free testd |  |   |   |   | 7.25E-04 | 1.98-01 |
| GSH | GPX7 | rs12118636 |   | 27/278 | 0.06 |   |   |   |   |
|   |   |   | GG | 27/241 |   | 2.00 | Ref |   |   |
|   |   |   | GA | 0/37 |   | 1.28 | 1.83(1.27, 2.63) | 1.21E-03 |   |
|   |   |   | Model free testd |  |   |   |   | 1.21E-03 | 1.98-01 |

Table 2 Caption: Chemotherapy agent-relevant pathway gene polymorphisms and overall survival

In stratified analysis of the PT group, 4 SNPs, rs17661089, rs16886403, rs726501 (in the *MAP3K1* gene of the *EGFR* pathway), and rs11710163 (in *RAF1* of *EGFR* pathway) were significantly associated with overall survival. Patients with rare homozygote genotypes of these variants in *MAP3K1* had significantly poorer overall survival, while patients with heterozygotes at rs11710163 (in *RAF1*) had better overall survival compared to having common homozygotes ([Table 2](#T2) and [Supplementary Figure 3](#SD6)). By gene-level tests, *MAP3K1* and *RAF1* were the top two significant genes (p=6.10×10^−3^ and 3.14×10^−2^, respectively) in the PT group, consistent with the single-SNP level analysis ([Table 3](#T3)). We further analyzed the four significant SNPs in the PT-only, P/no-T, P/no-E, and E groups, accordingly. All of the four SNPs were replicated in the P/no-E group; three SNPs in *MAP3K1* were replicated in the PT-only group; none showed significance in the P/no-T or E groups ([Supplementary Table 4](#SD1)).

### Table 3.

| Pathway | Gene | Number of SNPs | Number of SNP captured 90% of the variability | p-valuea |
| --- | --- | --- | --- | --- |
| All patients (n=1077) |   |   |   |   |
| EGFR | NRAS | 14 | 10 | 6.09E-04 |
| EGFR | MAP3K1 | 26 | 10 | 1.80E-02 |
| GSH | GSTM1 | 2 | 2 | 2.75E-02 |
| GSH | GSS | 18 | 10 | 3.76E-02 |
| DNA | RAD52 | 30 | 10 | 4.63E-02 |
|   |  |  |  |  |
| PT groupb(n=657) |   |   |   |   |
| EGFR | MAP3K1 | 26 | 10 | 6.10E-03 |
| EGFR | RAF1 | 22 | 9 | 3.14E-02 |
| GSH | GPX5 | 12 | 8 | 3.18E-02 |
| DNA | APEX1 | 10 | 8 | 3.20E-02 |
| GSH | GSS | 18 | 10 | 4.24E-02 |
| GSH | GSTA5 | 14 | 9 | 4.34E-02 |
|   |  |  |  |  |
| PG groupc(n=305) |   |   |   |   |
| GSH | GPX6 | 18 | 8 | 8.33E-05 |
| GSH | GPX5 | 12 | 8 | 2.22E-04 |
| EGFR | NRAS | 14 | 10 | 3.50E-03 |
| GSH | GPX7 | 12 | 6 | 4.70E-03 |
| GSH | GCLC | 40 | 10 | 2.93E-02 |
| GSH | GSTM3 | 10 | 5 | 4.09E-02 |
| DNA | RAD52 | 30 | 10 | 4.38E-02 |
| EGFR | RAF1 | 22 | 9 | 4.49E-02 |
| Taxa | MAPK9 | 38 | 10 | 4.87E-02 |

Table 3 Caption: Chemotherapy agent-relevant pathway gene and overall survival

In the PG group, rs11710163 in *RAF1* was also found to be significant; patients with a heterozygote genotype showed a nearly doubled overall survival compared to patients with a common homozygote genotype (HR=0.54, 95% CI=0.38–0.77). The heterozygote genotype at rs451774 in the *GPX5* gene also had a protective effect on overall survival, with the HR=0.61 (95% CI=0.47–0.79). In contrast, heterozygote genotypes at rs1065634 in *NRAS* and rs12118636 in *GPX7* showed a nearly doubled risk for death (HR=1.91; 95% CI=1.30–2.80 and HR=1.83; 95% CI=1.27–2.63, respectively) ([Table 2](#T2) and [Supplementary Figure 4](#SD10)). The genes, *RAF1*, *GPX5*, *GPX7*, and *NRAS*, which harbored the significant SNPs, were also confirmed to be significant ([Table 3](#T3)). We analyzed the four significant SNPs in the P/no-G, PG/no-E, and E groups, and all four SNPs were replicated in the PG/no-E group; however, none remained significant in the P/no-G or E groups ([Supplementary Table 4](#SD1)).

## Discussion

In this study, we systematically investigated the associations between 894 SNPs in 70 key genes from four drug action pathways and overall survival in patient groups receiving different chemotherapy agent combinations. Genetic variations in *MAP3K1* in the EGFR pathway and *GSS* in the GSH pathway were associated with overall survival in the analysis of all patients. In the stratified analysis on patients receiving platinum plus taxane treatment, *MAP3K1* and *RAF1* variations in the EGFR pathway were associated with survival. In patients receiving platinum plus gemcitabine treatment, *RAF* and *GPX5* (GSH pathway), *NRAS* (EGFR pathway), and *GPX7* (GSH pathway) showed predictive effects on overall survival.

The glutathione synthetase gene (*GSS*) encodes the enzyme glutathione synthetase. Mutations in *GSS* prevent cells from making adequate levels of glutathione, leading to glutathione synthetase deficiency ([23](#R23)). Genetic variations in *GSS* were also found to be associated with overall survival of small cell lung cancer after treatment ([24](#R24)). It is interesting to observe *GSS* variants, instead of other commonly reported genetic variations, such as *GSTM1, GSTP1*, and *GSTT1*, as the most significant predictive markers in our systematic GSH pathway SNP analysis. It is proactive to observe that genetic variations in *MAP3K1* from the EGFR pathway are also significant predictive markers for survival in patients receiving the most commonly used doublet agents, cisplatin or carboplatin and paclitaxel or docetaxel, even after restricting down to patients who did not receive EGFR inhibitor agents.

The single-SNP and whole-gene analyses consistently indicated that genetic variations in *MAP3K1* and *RAF1* were significantly associated with overall survival in the PT group. Three SNPs, rs17661089, rs16886403, and rs726501 in *MAP3K1*, were significant. The associations of these SNPs in *MAP3K1* were further replicated in the PT-only group, suggesting that polymorphic variants in the *MAP3K1* were specifically associated with overall survival in patients receiving platinum and taxanes. All four SNPs were replicated in the P/non-E group, but not in the E group, suggesting the associations of these SNPs with overall survival may be independent of the effect of the EGFR inhibitors, although both *MAP3K1* and *RAF1* regulate the EGFR pathway.

*MAP3K1* has a pivotal role in a network of phosphorylating enzymes integrating cellular responses to a number of mitogenic and metabolic stimuli. *MAP3K1* phosphorylates and activates MAPK kinase (MAPK2), which in turn phosphorylates MAPK/ERK to produce downstream signaling effects on a variety of cancer genes ([Supplementary Figure 1](#SD14)).([25](#R25), [26](#R26)) Researchers have recently reported an association between the EGFR pathway and platinum-chemotherapy sensitivity in colorectal cancer.([15](#R15)) Other genes in the EGFR pathway, such as *EGFR* itself, also have been demonstrated to be associated with lung cancer outcomes.([13](#R13), [14](#R14)) For the three significant SNPs in *MAP3K1*, rs17661089 and rs16886403 are in relatively high linkage disequilibrium with rs726501 (r^2^=0.64 and 0.72, respectively). rs17661089 is located in the flanking region of 5'-UTR, and the other two SNPs are located in intron 1 of the gene. rs17661089 was predicted to be in the transcription factor binding sites (TFBS); this SNP may be a functional SNP or other variants in high linkage disequilibrium with it possibly influence the expression level of the gene product, requiring further functional analyses.

The RAF family is composed of three related serine/threonine protein kinases - RAF1, A-RAF and B-RAF— which act, in part, as downstream effectors of the RAS pathway.([27](#R27)) Activated RAS interacts directly with the amino-terminal regulatory domain of the RAF kinase, which results in a cascade of reactions including direct activation of MEK.([28](#R28)–[30](#R30)) Constitutively active mutated RAF can transform cells *in vitro*.([31](#R31)) RAF may play a broader role in tumorigenesis and promotes the expression of the multi-drug resistance gene MDR1.([30](#R30)) RAF1 holds an important role in cell growth, proliferation, and cell survival.([32](#R32), [33](#R33)) The RAF1 protein has been found to be amplified in different lung cancer cell lines. One study showed that the RAF expression level is critical in tumor development.([34](#R34)) Ravi *et al* reported that activated RAF1 causes growth arrest in human small cell lung cancer cells. ([35](#R35)) Immunohistochemical staining indicated that RAF1 was present in 49/53 ovarian adenocarcinomas and high c-RAF expression correlated significantly with poor survival in ovarian cancer.([36](#R36)) The SNP, rs11706408, that has high linkage disequilibrium (r2=1) with rs11710163 in RAF1 was predicted to be in the TFBS; their functional roles in NSCLC chemotherapy agent actions and patient survival need to be further investigated.

Four SNPs in *RAF1*, *GPX5*, *GPX7*, and *NRAS* were found to be significantly associated overall survival in the PG group. All four SNPs were replicated in the PG/non-E group but were not replicated in the E group, indicating the association of being independent of the effect of the EGFR inhibitors. None of the four SNPs remained significant in the P/non-G group, suggesting the association of the variations in *RAF1*, *GPX5*, *GPX7*, and *NRAS* with overall survival in the PG group may reflect the effect of gemcitabine.

The GSH pathway has been related with detoxification or inactivation of platinum drugs. Glutathione peroxidases (GPXs) are a major antioxidative damage enzyme family that catalyzes the reduction of hydrogen peroxide, organic hydroperoxide, and lipid peroxides by reduced glutathione.([37](#R37)–[39](#R39)) GPX has at least seven allozymes, distributed in different organs. In this study, *GPX5* rs451774 and *GPX7* rs12118636 were found to be associated overall survival in the PG group. The association of *GPX5* and *GPX7* were also confirmed by whole gene-based analysis. Function prediction indicated that rs451774 located at 3' UTR of *GPX5* was in both TFBS and miRNA binding sites, and is a potential functional SNP. *GPX5* has structural similarity to GPX1.([40](#R40)) A recent study reported *GPX1* may be an inherited factor in predicting patients' quality of life.([41](#R41)) In previous studies, variation in *GPX1* was linked tumor recurrence of bladder cancer.([42](#R42)) *GPX7* shares a similar structural domain to *GPX4*. Reported findings suggest a potential role for *GPX7* in alleviating oxidative stress induced by dietary consumption of fatty acids in breast cancer cells.([43](#R43)) Common variations in *GPX4* were associated with prognosis after a diagnosis of breast cancer.([44](#R44)) However, no reports are currently available on the association of *GPX5* and *GPX7* variations and clinical outcomes. Our findings showed that genetic variations in *GPX5* tend to predict a better survival, while variations in *GPX7* were associated with worse survival. Current evidence also indicates the diversified roles of the individual *GPX*;([45](#R45)) individual *GPX* might have different impact on the clinical outcome of lung cancer.

The SNP, rs1065634, is located at 5' near the gene region of *NRAS*. This SNP also was found to be significant in the PG group. *NRAS* is a member of the *RAS* proto-oncogene family. Point mutations of the *RAS* proto-oncogene family members are among the most frequent genetic alterations found in human cancer.([46](#R46), [47](#R47)) Until now, *NRAS* gene mutations were mainly associated with hematopoietic malignancies([48](#R48)), melanomas([49](#R49)), and bladder cancers.([50](#R50)) rs1065634 was predicted to be in both TFBS and miRNA binding sites, and may be an important functional SNP; its potential functional significance is required to further study.

To our knowledge, our study presents the first and largest effort to comprehensively characterize the associations of gene alterations in four chemotherapy drug action pathways and survival in advanced-stage lung cancer patients. Platinum drugs, combined with other cytotoxic agents, continue to be the first-line chemotherapy for advanced-stage lung cancers and other metastatic cancers. Our study is strengthened by using different agent combinatorial groups for stratified analysis, which more closely mimics clinic practice than studies using single drug-based groups. We also used different analytic strategies and subgroups to confirm our results.

There are several limitations. First, despite efforts to account for potential confounding factors and careful adjustment for multiple tests in the analyses, false-positive associations cannot be completely ruled out. Second, although the significant SNPs were carefully analyzed in different combinatorial agent subgroups, chemotherapy heterogeneity may also modify the true association between genetic variation and clinical outcomes. Third, although our patient cohort represents one of the largest among all published combinatorial agent pharmacogenetics studies for late stage NSCLC, the power of this study to detect association was still modest, and it is possible that we missed some real associations with a small effect. For example, in the PG group, using a type I error level of 6×10^−5^ (0.05/984), we had 86% power to detect effects from SNPs with a minor allele frequency of 0.3, which confers a per-allele HR of 1.6. If the per-allele HR is instead 1.4, the power is only 35%. Fourth, a thorough evaluation of the gemcitabine pathway SNPs was not included in this study. An additional limitation was the lack of patients treated with first-line EGFR inhibitor drugs, so the specific SNP effects could not be detected or evaluated relevant to these drugs. Our study focused on late-stage NSCLC patients who were treated with platinum drugs with or without other treatments, which will limit the ability to compare our results to other data sets. Nonetheless, our study is useful for testing a novel hypothesis, which may be confirmed in future specifically-designed clinical studies.

In summary, our results suggest that variations of *MAP3k1* and *RAF1* may be associated with overall survival among patients treated with platinum and taxane agents. Polymorphisms in the *RAF1*, *NRAS*, *GPX5*, and *GPX7* genes were found to be associated with overall survival among patients treated with platinum and gemcitabine agents. In the current clinical practice, oncologists usually select patients who they expect to benefit from chemotherapy based on available clinical information. Validated biomarkers, even with small effects, may have utility in optimizing patient therapies. Therefore, our findings could be considered in prospective biomarker-stratified clinical trials.

The prognosis for patients with advanced-stage non-small cell lung cancer (NSCLC) remains very poor. Chemotherapy remains the major component of the standard care for patients with advanced-stage lung cancer. This study comprehensively characterized the associations of gene alterations in four chemotherapy drug action pathways i.e., glutathione, DNA repair, cell cycle, and EGFR, and survival in advanced-stage NSCLC patients. Our results suggest that variations of *MAP3K1* and *RAF1* may be associated with overall survival among patients treated with platinum and taxane agents. Polymorphisms in the *RAF1*, *NRAS*, *GPX5*, and *GPX7* genes were found to be associated with overall survival among patients treated with platinum and gemcitabine agents. These genetic variations may be applied to future prospective biomarker-stratified clinical trials, and help in designing patient-specific treatment and in predicting patients' survival.

The prognosis for patients with advanced-stage non-small cell lung cancer (NSCLC) remains very poor. Chemotherapy remains the major component of the standard care for patients with advanced-stage lung cancer. This study comprehensively characterized the associations of gene alterations in four chemotherapy drug action pathways i.e., glutathione, DNA repair, cell cycle, and EGFR, and survival in advanced-stage NSCLC patients. Our results suggest that variations of *MAP3K1* and *RAF1* may be associated with overall survival among patients treated with platinum and taxane agents. Polymorphisms in the *RAF1*, *NRAS*, *GPX5*, and *GPX7* genes were found to be associated with overall survival among patients treated with platinum and gemcitabine agents. These genetic variations may be applied to future prospective biomarker-stratified clinical trials, and help in designing patient-specific treatment and in predicting patients' survival.

## Supplementary Material

## Acknowledgments

We thank Susan Ernst, M.A. for her technical assistance with the manuscript and David N. Rider (Division of Biomedical Statistics and Informatics, Mayo Clinic) for his assistance in SNP selection.

**Research Support:** This study was supported by U.S. National Institutes of Health (NIH) research grants, R01-CA80127 (to PY) and R01-CA84354 (to PY), and Mayo Foundation funds (to PY). Support was also provided by the NIH grant, P30 CA15083 (Mayo Clinic Comprehensive Cancer Center-Gene Analysis Shared Resource).

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Walker S. Updates in non-small cell lung cancer. Clin J Oncol Nurs. 2008;12:587–96. doi: 10.1188/08.CJON.587-596.  [DOI](https://doi.org/10.1188/08.CJON.587-596) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18676326/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20J%20Oncol%20Nurs&title=Updates%20in%20non-small%20cell%20lung%20cancer&author=S%20Walker&volume=12&publication_year=2008&pages=587-96&pmid=18676326&doi=10.1188/08.CJON.587-596&)

2. Spira A, Ettinger DS. Multidisciplinary management of lung cancer. N Engl J Med. 2004;350:379–92. doi: 10.1056/NEJMra035536.  [DOI](https://doi.org/10.1056/NEJMra035536) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14736930/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Multidisciplinary%20management%20of%20lung%20cancer&author=A%20Spira&author=DS%20Ettinger&volume=350&publication_year=2004&pages=379-92&pmid=14736930&doi=10.1056/NEJMra035536&)

3. Moore DJ, Lee J. In: Lung Cancer: Principles and Practice. Pass H, Mitchell J, Johnson D, et al., editors. Lipincott-Raven Publishers; Philadelphia: 1996. pp. 495–509.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Lung%20Cancer:%20Principles%20and%20Practice&author=DJ%20Moore&author=J%20Lee&publication_year=1996&)

4. Yang P, Ebbert JO, Sun Z, Weinshilboum RM. Role of the glutathione metabolic pathway in lung cancer treatment and prognosis: a review. J Clin Oncol. 2006;24:1761–9. doi: 10.1200/JCO.2005.02.7110.  [DOI](https://doi.org/10.1200/JCO.2005.02.7110) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16603718/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Role%20of%20the%20glutathione%20metabolic%20pathway%20in%20lung%20cancer%20treatment%20and%20prognosis:%20a%20review&author=P%20Yang&author=JO%20Ebbert&author=Z%20Sun&author=RM%20Weinshilboum&volume=24&publication_year=2006&pages=1761-9&pmid=16603718&doi=10.1200/JCO.2005.02.7110&)

5. Kalikaki A, Kanaki M, Vassalou H, Souglakos J, Voutsina A, Georgoulias V, et al. DNA repair gene polymorphisms predict favorable clinical outcome in advanced non-small-cell lung cancer. Clin Lung Cancer. 2009;10:118–23. doi: 10.3816/CLC.2009.n.015.  [DOI](https://doi.org/10.3816/CLC.2009.n.015) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19362955/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Lung%20Cancer&title=DNA%20repair%20gene%20polymorphisms%20predict%20favorable%20clinical%20outcome%20in%20advanced%20non-small-cell%20lung%20cancer&author=A%20Kalikaki&author=M%20Kanaki&author=H%20Vassalou&author=J%20Souglakos&author=A%20Voutsina&volume=10&publication_year=2009&pages=118-23&pmid=19362955&doi=10.3816/CLC.2009.n.015&)

6. Gurubhagavatula S, Liu G, Park S, Zhou W, Su L, Wain JC, et al. XPD and XRCC1 genetic polymorphisms are prognostic factors in advanced non-small-cell lung cancer patients treated with platinum chemotherapy. J Clin Oncol. 2004;22:2594–601. doi: 10.1200/JCO.2004.08.067.  [DOI](https://doi.org/10.1200/JCO.2004.08.067) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15173214/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=XPD%20and%20XRCC1%20genetic%20polymorphisms%20are%20prognostic%20factors%20in%20advanced%20non-small-cell%20lung%20cancer%20patients%20treated%20with%20platinum%20chemotherapy&author=S%20Gurubhagavatula&author=G%20Liu&author=S%20Park&author=W%20Zhou&author=L%20Su&volume=22&publication_year=2004&pages=2594-601&pmid=15173214&doi=10.1200/JCO.2004.08.067&)

7. Ryu JS, Hong YC, Han HS, Lee JE, Kim S, Park YM, et al. Association between polymorphisms of ERCC1 and XPD and survival in non-small-cell lung cancer patients treated with cisplatin combination chemotherapy. Lung Cancer. 2004;44:311–6. doi: 10.1016/j.lungcan.2003.11.019.  [DOI](https://doi.org/10.1016/j.lungcan.2003.11.019) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15140544/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lung%20Cancer&title=Association%20between%20polymorphisms%20of%20ERCC1%20and%20XPD%20and%20survival%20in%20non-small-cell%20lung%20cancer%20patients%20treated%20with%20cisplatin%20combination%20chemotherapy&author=JS%20Ryu&author=YC%20Hong&author=HS%20Han&author=JE%20Lee&author=S%20Kim&volume=44&publication_year=2004&pages=311-6&pmid=15140544&doi=10.1016/j.lungcan.2003.11.019&)

8. Isla D, Sarries C, Rosell R, Alonso G, Domine M, Taron M, et al. Single nucleotide polymorphisms and outcome in docetaxel-cisplatin-treated advanced non-small-cell lung cancer. Ann Oncol. 2004;15:1194–203. doi: 10.1093/annonc/mdh319.  [DOI](https://doi.org/10.1093/annonc/mdh319) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15277258/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Oncol&title=Single%20nucleotide%20polymorphisms%20and%20outcome%20in%20docetaxel-cisplatin-treated%20advanced%20non-small-cell%20lung%20cancer&author=D%20Isla&author=C%20Sarries&author=R%20Rosell&author=G%20Alonso&author=M%20Domine&volume=15&publication_year=2004&pages=1194-203&pmid=15277258&doi=10.1093/annonc/mdh319&)

9. Zhou W, Gurubhagavatula S, Liu G, Park S, Neuberg DS, Wain JC, et al. Excision repair cross-complementation group 1 polymorphism predicts overall survival in advanced non-small cell lung cancer patients treated with platinum-based chemotherapy. Clin Cancer Res. 2004;10:4939–43. doi: 10.1158/1078-0432.CCR-04-0247.  [DOI](https://doi.org/10.1158/1078-0432.CCR-04-0247) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15297394/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&title=Excision%20repair%20cross-complementation%20group%201%20polymorphism%20predicts%20overall%20survival%20in%20advanced%20non-small%20cell%20lung%20cancer%20patients%20treated%20with%20platinum-based%20chemotherapy&author=W%20Zhou&author=S%20Gurubhagavatula&author=G%20Liu&author=S%20Park&author=DS%20Neuberg&volume=10&publication_year=2004&pages=4939-43&pmid=15297394&doi=10.1158/1078-0432.CCR-04-0247&)

10. Wu X, Lu C, Ye Y, Chang J, Yang H, Lin J, et al. Germline genetic variations in drug action pathways predict clinical outcomes in advanced lung cancer treated with platinum-based chemotherapy. Pharmacogenet Genomics. 2008;18:955–65. doi: 10.1097/FPC.0b013e32830efdd4.  [DOI](https://doi.org/10.1097/FPC.0b013e32830efdd4) | [PMC free article](/articles/PMC2665725/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18854777/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Germline%20genetic%20variations%20in%20drug%20action%20pathways%20predict%20clinical%20outcomes%20in%20advanced%20lung%20cancer%20treated%20with%20platinum-based%20chemotherapy&author=X%20Wu&author=C%20Lu&author=Y%20Ye&author=J%20Chang&author=H%20Yang&volume=18&publication_year=2008&pages=955-65&pmid=18854777&doi=10.1097/FPC.0b013e32830efdd4&)

11. Gligorov J, Lotz JP. Preclinical pharmacology of the taxanes: implications of the differences. Oncologist. 2004;9(Suppl 2):3–8. doi: 10.1634/theoncologist.9-suppl_2-3.  [DOI](https://doi.org/10.1634/theoncologist.9-suppl_2-3) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15161985/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncologist&title=Preclinical%20pharmacology%20of%20the%20taxanes:%20implications%20of%20the%20differences&author=J%20Gligorov&author=JP%20Lotz&volume=9&issue=Suppl%202&publication_year=2004&pages=3-8&pmid=15161985&doi=10.1634/theoncologist.9-suppl_2-3&)

12. Zhao J, Kim JE, Reed E, Li QQ. Molecular mechanism of antitumor activity of taxanes in lung cancer (Review) Int J Oncol. 2005;27:247–56.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/15942666/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Oncol&title=Molecular%20mechanism%20of%20antitumor%20activity%20of%20taxanes%20in%20lung%20cancer%20(Review)&author=J%20Zhao&author=JE%20Kim&author=E%20Reed&author=QQ%20Li&volume=27&publication_year=2005&pages=247-56&pmid=15942666&)

13. Nicholson RI, Gee JM, Harper ME. EGFR and cancer prognosis. Eur J Cancer. 2001;37(Suppl 4):S9–15. doi: 10.1016/s0959-8049(01)00231-3.  [DOI](https://doi.org/10.1016/s0959-8049(01)00231-3) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11597399/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Cancer&title=EGFR%20and%20cancer%20prognosis&author=RI%20Nicholson&author=JM%20Gee&author=ME%20Harper&volume=37&issue=Suppl%204&publication_year=2001&pages=S9-15&pmid=11597399&doi=10.1016/s0959-8049(01)00231-3&)

14. Onn A, Correa AM, Gilcrease M, Isobe T, Massarelli E, Bucana CD, et al. Synchronous overexpression of epidermal growth factor receptor and HER2-neu protein is a predictor of poor outcome in patients with stage I non-small cell lung cancer. Clin Cancer Res. 2004;10:136–43. doi: 10.1158/1078-0432.ccr-0373-3.  [DOI](https://doi.org/10.1158/1078-0432.ccr-0373-3) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14734462/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&title=Synchronous%20overexpression%20of%20epidermal%20growth%20factor%20receptor%20and%20HER2-neu%20protein%20is%20a%20predictor%20of%20poor%20outcome%20in%20patients%20with%20stage%20I%20non-small%20cell%20lung%20cancer&author=A%20Onn&author=AM%20Correa&author=M%20Gilcrease&author=T%20Isobe&author=E%20Massarelli&volume=10&publication_year=2004&pages=136-43&pmid=14734462&doi=10.1158/1078-0432.ccr-0373-3&)

15. Zhang W, Stoehlmacher J, Park DJ, Yang D, Borchard E, Gil J, et al. Gene polymorphisms of epidermal growth factor receptor and its downstream effector, interleukin-8, predict oxaliplatin efficacy in patients with advanced colorectal cancer. Clin Colorectal Cancer. 2005;5:124–31. doi: 10.3816/ccc.2005.n.025.  [DOI](https://doi.org/10.3816/ccc.2005.n.025) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16098254/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Colorectal%20Cancer&title=Gene%20polymorphisms%20of%20epidermal%20growth%20factor%20receptor%20and%20its%20downstream%20effector,%20interleukin-8,%20predict%20oxaliplatin%20efficacy%20in%20patients%20with%20advanced%20colorectal%20cancer&author=W%20Zhang&author=J%20Stoehlmacher&author=DJ%20Park&author=D%20Yang&author=E%20Borchard&volume=5&publication_year=2005&pages=124-31&pmid=16098254&doi=10.3816/ccc.2005.n.025&)

16. Yang P, Bamlet WR, Sun Z, Ebbert JO, Aubry MC, Krowka MJ, et al. Alpha1-antitrypsin and neutrophil elastase imbalance and lung cancer risk. Chest. 2005;128:445–52. doi: 10.1378/chest.128.1.445.  [DOI](https://doi.org/10.1378/chest.128.1.445) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16002971/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Chest&title=Alpha1-antitrypsin%20and%20neutrophil%20elastase%20imbalance%20and%20lung%20cancer%20risk&author=P%20Yang&author=WR%20Bamlet&author=Z%20Sun&author=JO%20Ebbert&author=MC%20Aubry&volume=128&publication_year=2005&pages=445-52&pmid=16002971&doi=10.1378/chest.128.1.445&)

17. Yang P, Sun Z, Krowka MJ, Aubry MC, Bamlet WR, Wampfler JA, et al. Alpha1-antitrypsin deficiency carriers, tobacco smoke, chronic obstructive pulmonary disease, and lung cancer risk. Arch Intern Med. 2008;168:1097–103. doi: 10.1001/archinte.168.10.1097.  [DOI](https://doi.org/10.1001/archinte.168.10.1097) | [PMC free article](/articles/PMC2562773/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18504338/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arch%20Intern%20Med&title=Alpha1-antitrypsin%20deficiency%20carriers,%20tobacco%20smoke,%20chronic%20obstructive%20pulmonary%20disease,%20and%20lung%20cancer%20risk&author=P%20Yang&author=Z%20Sun&author=MJ%20Krowka&author=MC%20Aubry&author=WR%20Bamlet&volume=168&publication_year=2008&pages=1097-103&pmid=18504338&doi=10.1001/archinte.168.10.1097&)

18. Yang P, Wentzlaff KA, Katzmann JA, Marks RS, Allen MS, Lesnick TG, et al. Alpha1-antitrypsin deficiency allele carriers among lung cancer patients. Cancer Epidemiol Biomarkers Prev. 1999;8:461–5.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/10350443/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Epidemiol%20Biomarkers%20Prev&title=Alpha1-antitrypsin%20deficiency%20allele%20carriers%20among%20lung%20cancer%20patients&author=P%20Yang&author=KA%20Wentzlaff&author=JA%20Katzmann&author=RS%20Marks&author=MS%20Allen&volume=8&publication_year=1999&pages=461-5&pmid=10350443&)

19. Johnson GC, Esposito L, Barratt BJ, Smith AN, Heward J, Di Genova G, et al. Haplotype tagging for the identification of common disease genes. Nat Genet. 2001;29:233–7. doi: 10.1038/ng1001-233.  [DOI](https://doi.org/10.1038/ng1001-233) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11586306/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Haplotype%20tagging%20for%20the%20identification%20of%20common%20disease%20genes&author=GC%20Johnson&author=L%20Esposito&author=BJ%20Barratt&author=AN%20Smith&author=J%20Heward&volume=29&publication_year=2001&pages=233-7&pmid=11586306&doi=10.1038/ng1001-233&)

20. Carlson CS, Eberle MA, Rieder MJ, Yi Q, Kruglyak L, Nickerson DA. Selecting a maximally informative set of single-nucleotide polymorphisms for association analyses using linkage disequilibrium. Am J Hum Genet. 2004;74:106–20. doi: 10.1086/381000.  [DOI](https://doi.org/10.1086/381000) | [PMC free article](/articles/PMC1181897/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14681826/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=Selecting%20a%20maximally%20informative%20set%20of%20single-nucleotide%20polymorphisms%20for%20association%20analyses%20using%20linkage%20disequilibrium&author=CS%20Carlson&author=MA%20Eberle&author=MJ%20Rieder&author=Q%20Yi&author=L%20Kruglyak&volume=74&publication_year=2004&pages=106-20&pmid=14681826&doi=10.1086/381000&)

21. Carlson CS, Eberle MA, Kruglyak L, Nickerson DA. Mapping complex disease loci in whole-genome association studies. Nature. 2004;429:446–52. doi: 10.1038/nature02623.  [DOI](https://doi.org/10.1038/nature02623) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15164069/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Mapping%20complex%20disease%20loci%20in%20whole-genome%20association%20studies&author=CS%20Carlson&author=MA%20Eberle&author=L%20Kruglyak&author=DA%20Nickerson&volume=429&publication_year=2004&pages=446-52&pmid=15164069&doi=10.1038/nature02623&)

22. Storey JD, Tibshirani R. Statistical significance for genomewide studies. Proc Natl Acad Sci U S A. 2003;100:9440–5. doi: 10.1073/pnas.1530509100.  [DOI](https://doi.org/10.1073/pnas.1530509100) | [PMC free article](/articles/PMC170937/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12883005/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&title=Statistical%20significance%20for%20genomewide%20studies&author=JD%20Storey&author=R%20Tibshirani&volume=100&publication_year=2003&pages=9440-5&pmid=12883005&doi=10.1073/pnas.1530509100&)

23. Njalsson R, Ristoff E, Carlsson K, Winkler A, Larsson A, Norgren S. Genotype, enzyme activity, glutathione level, and clinical phenotype in patients with glutathione synthetase deficiency. Hum Genet. 2005;116:384–9. doi: 10.1007/s00439-005-1255-6.  [DOI](https://doi.org/10.1007/s00439-005-1255-6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15717202/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Genet&title=Genotype,%20enzyme%20activity,%20glutathione%20level,%20and%20clinical%20phenotype%20in%20patients%20with%20glutathione%20synthetase%20deficiency&author=R%20Njalsson&author=E%20Ristoff&author=K%20Carlsson&author=A%20Winkler&author=A%20Larsson&volume=116&publication_year=2005&pages=384-9&pmid=15717202&doi=10.1007/s00439-005-1255-6&)

24. Sun Z, Chen J, Aakre J, Marks RS, Garces YY, Jiang R, et al. Genetic variation in glutathione metabolism and DNA repair genes predicts survival of small-cell lung cancer patients. Ann Oncol. 2010;21:2011–6. doi: 10.1093/annonc/mdq212.  [DOI](https://doi.org/10.1093/annonc/mdq212) | [PMC free article](/articles/PMC2946864/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20439344/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Oncol&title=Genetic%20variation%20in%20glutathione%20metabolism%20and%20DNA%20repair%20genes%20predicts%20survival%20of%20small-cell%20lung%20cancer%20patients&author=Z%20Sun&author=J%20Chen&author=J%20Aakre&author=RS%20Marks&author=YY%20Garces&volume=21&publication_year=2010&pages=2011-6&pmid=20439344&doi=10.1093/annonc/mdq212&)

25. Witowsky JA, Johnson GL. Ubiquitylation of MEKK1 inhibits its phosphorylation of MKK1 and MKK4 and activation of the ERK1/2 and JNK pathways. J Biol Chem. 2003;278:1403–6. doi: 10.1074/jbc.C200616200.  [DOI](https://doi.org/10.1074/jbc.C200616200) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12456688/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=Ubiquitylation%20of%20MEKK1%20inhibits%20its%20phosphorylation%20of%20MKK1%20and%20MKK4%20and%20activation%20of%20the%20ERK1/2%20and%20JNK%20pathways&author=JA%20Witowsky&author=GL%20Johnson&volume=278&publication_year=2003&pages=1403-6&pmid=12456688&doi=10.1074/jbc.C200616200&)

26. Xu S, Cobb MH. MEKK1 binds directly to the c-Jun N-terminal kinases/stress-activated protein kinases. J Biol Chem. 1997;272:32056–60. doi: 10.1074/jbc.272.51.32056.  [DOI](https://doi.org/10.1074/jbc.272.51.32056) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9405400/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=MEKK1%20binds%20directly%20to%20the%20c-Jun%20N-terminal%20kinases/stress-activated%20protein%20kinases&author=S%20Xu&author=MH%20Cobb&volume=272&publication_year=1997&pages=32056-60&pmid=9405400&doi=10.1074/jbc.272.51.32056&)

27. Kerkhoff E, Rapp UR. Cell cycle targets of Ras/Raf signalling. Oncogene. 1998;17:1457–62. doi: 10.1038/sj.onc.1202185.  [DOI](https://doi.org/10.1038/sj.onc.1202185) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9779991/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncogene&title=Cell%20cycle%20targets%20of%20Ras/Raf%20signalling&author=E%20Kerkhoff&author=UR%20Rapp&volume=17&publication_year=1998&pages=1457-62&pmid=9779991&doi=10.1038/sj.onc.1202185&)

28. Beeram M, Patnaik A, Rowinsky EK. Regulation of c-Raf-1: therapeutic implications. Clin Adv Hematol Oncol. 2003;1:476–81.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/16258435/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Adv%20Hematol%20Oncol&title=Regulation%20of%20c-Raf-1:%20therapeutic%20implications&author=M%20Beeram&author=A%20Patnaik&author=EK%20Rowinsky&volume=1&publication_year=2003&pages=476-81&pmid=16258435&)

29. Kolch W, Heidecker G, Kochs G, Hummel R, Vahidi H, Mischak H, et al. Protein kinase C alpha activates RAF-1 by direct phosphorylation. Nature. 1993;364:249–52. doi: 10.1038/364249a0.  [DOI](https://doi.org/10.1038/364249a0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8321321/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Protein%20kinase%20C%20alpha%20activates%20RAF-1%20by%20direct%20phosphorylation&author=W%20Kolch&author=G%20Heidecker&author=G%20Kochs&author=R%20Hummel&author=H%20Vahidi&volume=364&publication_year=1993&pages=249-52&pmid=8321321&doi=10.1038/364249a0&)

30. Cornwell MM, Smith DE. A signal transduction pathway for activation of the mdr1 promoter involves the proto-oncogene c-raf kinase. J Biol Chem. 1993;268:15347–50.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/8101839/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=A%20signal%20transduction%20pathway%20for%20activation%20of%20the%20mdr1%20promoter%20involves%20the%20proto-oncogene%20c-raf%20kinase&author=MM%20Cornwell&author=DE%20Smith&volume=268&publication_year=1993&pages=15347-50&pmid=8101839&)

31. Stanton VP, Jr., Cooper GM. Activation of human raf transforming genes by deletion of normal amino-terminal coding sequences. Mol Cell Biol. 1987;7:1171–9. doi: 10.1128/mcb.7.3.1171.  [DOI](https://doi.org/10.1128/mcb.7.3.1171) | [PMC free article](/articles/PMC365190/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/3561413/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Cell%20Biol&title=Activation%20of%20human%20raf%20transforming%20genes%20by%20deletion%20of%20normal%20amino-terminal%20coding%20sequences&author=VP%20Stanton&author=GM%20Cooper&volume=7&publication_year=1987&pages=1171-9&pmid=3561413&doi=10.1128/mcb.7.3.1171&)

32. Morrison DK, Cutler RE. The complexity of Raf-1 regulation. Curr Opin Cell Biol. 1997;9:174–9. doi: 10.1016/s0955-0674(97)80060-9.  [DOI](https://doi.org/10.1016/s0955-0674(97)80060-9) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9069260/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Opin%20Cell%20Biol&title=The%20complexity%20of%20Raf-1%20regulation&author=DK%20Morrison&author=RE%20Cutler&volume=9&publication_year=1997&pages=174-9&pmid=9069260&doi=10.1016/s0955-0674(97)80060-9&)

33. Kolch W. Meaningful relationships: the regulation of the Ras/Raf/MEK/ERK pathway by protein interactions. Biochem J. 2000;351(Pt 2):289–305.  [PMC free article](/articles/PMC1221363/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11023813/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem%20J&title=Meaningful%20relationships:%20the%20regulation%20of%20the%20Ras/Raf/MEK/ERK%20pathway%20by%20protein%20interactions&author=W%20Kolch&volume=351&issue=Pt%202&publication_year=2000&pages=289-305&pmid=11023813&)

34. Kerkhoff E, Fedorov LM, Siefken R, Walter AO, Papadopoulos T, Rapp UR. Lung-targeted expression of the c-Raf-1 kinase in transgenic mice exposes a novel oncogenic character of the wild-type protein. Cell Growth Differ. 2000;11:185–90.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/10775035/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell%20Growth%20Differ&title=Lung-targeted%20expression%20of%20the%20c-Raf-1%20kinase%20in%20transgenic%20mice%20exposes%20a%20novel%20oncogenic%20character%20of%20the%20wild-type%20protein&author=E%20Kerkhoff&author=LM%20Fedorov&author=R%20Siefken&author=AO%20Walter&author=T%20Papadopoulos&volume=11&publication_year=2000&pages=185-90&pmid=10775035&)

35. Ravi RK, Weber E, McMahon M, Williams JR, Baylin S, Mal A, et al. Activated Raf-1 causes growth arrest in human small cell lung cancer cells. J Clin Invest. 1998;101:153–9. doi: 10.1172/JCI831.  [DOI](https://doi.org/10.1172/JCI831) | [PMC free article](/articles/PMC508551/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9421477/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Invest&title=Activated%20Raf-1%20causes%20growth%20arrest%20in%20human%20small%20cell%20lung%20cancer%20cells&author=RK%20Ravi&author=E%20Weber&author=M%20McMahon&author=JR%20Williams&author=S%20Baylin&volume=101&publication_year=1998&pages=153-9&pmid=9421477&doi=10.1172/JCI831&)

36. McPhillips F, Mullen P, Monia BP, Ritchie AA, Dorr FA, Smyth JF, et al. Association of c-Raf expression with survival and its targeting with antisense oligonucleotides in ovarian cancer. Br J Cancer. 2001;85:1753–8. doi: 10.1054/bjoc.2001.2139.  [DOI](https://doi.org/10.1054/bjoc.2001.2139) | [PMC free article](/articles/PMC2363986/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11742498/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Cancer&title=Association%20of%20c-Raf%20expression%20with%20survival%20and%20its%20targeting%20with%20antisense%20oligonucleotides%20in%20ovarian%20cancer&author=F%20McPhillips&author=P%20Mullen&author=BP%20Monia&author=AA%20Ritchie&author=FA%20Dorr&volume=85&publication_year=2001&pages=1753-8&pmid=11742498&doi=10.1054/bjoc.2001.2139&)

37. Halliwell B. Antioxidant defence mechanisms: from the beginning to the end (of the beginning) Free Radic Res. 1999;31:261–72. doi: 10.1080/10715769900300841.  [DOI](https://doi.org/10.1080/10715769900300841) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10517532/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Free%20Radic%20Res&title=Antioxidant%20defence%20mechanisms:%20from%20the%20beginning%20to%20the%20end%20(of%20the%20beginning)&author=B%20Halliwell&volume=31&publication_year=1999&pages=261-72&pmid=10517532&doi=10.1080/10715769900300841&)

38. Miyamoto Y, Koh YH, Park YS, Fujiwara N, Sakiyama H, Misonou Y, et al. Oxidative stress caused by inactivation of glutathione peroxidase and adaptive responses. Biol Chem. 2003;384:567–74. doi: 10.1515/BC.2003.064.  [DOI](https://doi.org/10.1515/BC.2003.064) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12751786/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biol%20Chem&title=Oxidative%20stress%20caused%20by%20inactivation%20of%20glutathione%20peroxidase%20and%20adaptive%20responses&author=Y%20Miyamoto&author=YH%20Koh&author=YS%20Park&author=N%20Fujiwara&author=H%20Sakiyama&volume=384&publication_year=2003&pages=567-74&pmid=12751786&doi=10.1515/BC.2003.064&)

39. Ray G, Husain SA. Oxidants, antioxidants and carcinogenesis. Indian J Exp Biol. 2002;40:1213–32.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/13677623/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Indian%20J%20Exp%20Biol&title=Oxidants,%20antioxidants%20and%20carcinogenesis&author=G%20Ray&author=SA%20Husain&volume=40&publication_year=2002&pages=1213-32&pmid=13677623&)

40. Vernet P, Rock E, Mazur A, Rayssiguier Y, Dufaure JP, Drevet JR. Selenium-independent epididymis-restricted glutathione peroxidase 5 protein (GPX5) can back up failing Se-dependent GPXs in mice subjected to selenium deficiency. Mol Reprod Dev. 1999;54:362–70. doi: 10.1002/(SICI)1098-2795(199912)54:4<362::AID-MRD6>3.0.CO;2-#.  [DOI](https://doi.org/10.1002/(SICI)1098-2795(199912)54:4<362::AID-MRD6>3.0.CO;2-#) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10542376/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Reprod%20Dev&title=Selenium-independent%20epididymis-restricted%20glutathione%20peroxidase%205%20protein%20(GPX5)%20can%20back%20up%20failing%20Se-dependent%20GPXs%20in%20mice%20subjected%20to%20selenium%20deficiency&author=P%20Vernet&author=E%20Rock&author=A%20Mazur&author=Y%20Rayssiguier&author=JP%20Dufaure&volume=54&publication_year=1999&pages=362-70&pmid=10542376&doi=10.1002/(SICI)1098-2795(199912)54:4<362::AID-MRD6>3.0.CO;2-#&)

41. Yang P, Mandrekar SJ, Hillman SH, Allen Ziegler KL, Sun Z, Wampfler JA, et al. Evaluation of glutathione metabolic genes on outcomes in advanced non-small cell lung cancer patients after initial treatment with platinum-based chemotherapy: an NCCTG-97-24-51 based study. J Thorac Oncol. 2009;4:479–85. doi: 10.1097/jto.0b013e31819c7a2c.  [DOI](https://doi.org/10.1097/jto.0b013e31819c7a2c) | [PMC free article](/articles/PMC2998042/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19347979/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Thorac%20Oncol&title=Evaluation%20of%20glutathione%20metabolic%20genes%20on%20outcomes%20in%20advanced%20non-small%20cell%20lung%20cancer%20patients%20after%20initial%20treatment%20with%20platinum-based%20chemotherapy:%20an%20NCCTG-97-24-51%20based%20study&author=P%20Yang&author=SJ%20Mandrekar&author=SH%20Hillman&author=KL%20Allen%20Ziegler&author=Z%20Sun&volume=4&publication_year=2009&pages=479-85&pmid=19347979&doi=10.1097/jto.0b013e31819c7a2c&)

42. Zhao H, Liang D, Grossman HB, Wu X. Glutathione peroxidase 1 gene polymorphism and risk of recurrence in patients with superficial bladder cancer. Urology. 2005;66:769–74. doi: 10.1016/j.urology.2005.04.033.  [DOI](https://doi.org/10.1016/j.urology.2005.04.033) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16230136/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Urology&title=Glutathione%20peroxidase%201%20gene%20polymorphism%20and%20risk%20of%20recurrence%20in%20patients%20with%20superficial%20bladder%20cancer&author=H%20Zhao&author=D%20Liang&author=HB%20Grossman&author=X%20Wu&volume=66&publication_year=2005&pages=769-74&pmid=16230136&doi=10.1016/j.urology.2005.04.033&)

43. Utomo A, Jiang X, Furuta S, Yun J, Levin DS, Wang YC, et al. Identification of a novel putative non-selenocysteine containing phospholipid hydroperoxide glutathione peroxidase (NPGPx) essential for alleviating oxidative stress generated from polyunsaturated fatty acids in breast cancer cells. J Biol Chem. 2004;279:43522–9. doi: 10.1074/jbc.M407141200.  [DOI](https://doi.org/10.1074/jbc.M407141200) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15294905/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=Identification%20of%20a%20novel%20putative%20non-selenocysteine%20containing%20phospholipid%20hydroperoxide%20glutathione%20peroxidase%20(NPGPx)%20essential%20for%20alleviating%20oxidative%20stress%20generated%20from%20polyunsaturated%20fatty%20acids%20in%20breast%20cancer%20cells&author=A%20Utomo&author=X%20Jiang&author=S%20Furuta&author=J%20Yun&author=DS%20Levin&volume=279&publication_year=2004&pages=43522-9&pmid=15294905&doi=10.1074/jbc.M407141200&)

44. Udler M, Maia AT, Cebrian A, Brown C, Greenberg D, Shah M, et al. Common germline genetic variation in antioxidant defense genes and survival after diagnosis of breast cancer. J Clin Oncol. 2007;25:3015–23. doi: 10.1200/JCO.2006.10.0099.  [DOI](https://doi.org/10.1200/JCO.2006.10.0099) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17634480/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Common%20germline%20genetic%20variation%20in%20antioxidant%20defense%20genes%20and%20survival%20after%20diagnosis%20of%20breast%20cancer&author=M%20Udler&author=AT%20Maia&author=A%20Cebrian&author=C%20Brown&author=D%20Greenberg&volume=25&publication_year=2007&pages=3015-23&pmid=17634480&doi=10.1200/JCO.2006.10.0099&)

45. Brigelius-Flohe R, Kipp A. Glutathione peroxidases in different stages of carcinogenesis. Biochim Biophys Acta. 2009;1790:1555–68. doi: 10.1016/j.bbagen.2009.03.006.  [DOI](https://doi.org/10.1016/j.bbagen.2009.03.006) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19289149/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochim%20Biophys%20Acta&title=Glutathione%20peroxidases%20in%20different%20stages%20of%20carcinogenesis&author=R%20Brigelius-Flohe&author=A%20Kipp&volume=1790&publication_year=2009&pages=1555-68&pmid=19289149&doi=10.1016/j.bbagen.2009.03.006&)

46. Bos JL. ras oncogenes in human cancer: a review. Cancer Res. 1989;49:4682–9.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/2547513/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=ras%20oncogenes%20in%20human%20cancer:%20a%20review&author=JL%20Bos&volume=49&publication_year=1989&pages=4682-9&pmid=2547513&)

47. Barbacid M. ras genes. Annu Rev Biochem. 1987;56:779–827. doi: 10.1146/annurev.bi.56.070187.004023.  [DOI](https://doi.org/10.1146/annurev.bi.56.070187.004023) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/3304147/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Annu%20Rev%20Biochem&title=ras%20genes&author=M%20Barbacid&volume=56&publication_year=1987&pages=779-827&pmid=3304147&doi=10.1146/annurev.bi.56.070187.004023&)

48. Bartram CR, Ludwig WD, Hiddemann W, Lyons J, Buschle M, Ritter J, et al. Acute myeloid leukemia: analysis of ras gene mutations and clonality defined by polymorphic X-linked loci. Leukemia. 1989;3:247–56.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/2564452/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Leukemia&title=Acute%20myeloid%20leukemia:%20analysis%20of%20ras%20gene%20mutations%20and%20clonality%20defined%20by%20polymorphic%20X-linked%20loci&author=CR%20Bartram&author=WD%20Ludwig&author=W%20Hiddemann&author=J%20Lyons&author=M%20Buschle&volume=3&publication_year=1989&pages=247-56&pmid=2564452&)

49. Ball NJ, Yohn JJ, Morelli JG, Norris DA, Golitz LE, Hoeffler JP. Ras mutations in human melanoma: a marker of malignant progression. J Invest Dermatol. 1994;102:285–90. doi: 10.1111/1523-1747.ep12371783.  [DOI](https://doi.org/10.1111/1523-1747.ep12371783) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8120410/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Invest%20Dermatol&title=Ras%20mutations%20in%20human%20melanoma:%20a%20marker%20of%20malignant%20progression&author=NJ%20Ball&author=JJ%20Yohn&author=JG%20Morelli&author=DA%20Norris&author=LE%20Golitz&volume=102&publication_year=1994&pages=285-90&pmid=8120410&doi=10.1111/1523-1747.ep12371783&)

50. Karimianpour N, Mousavi-Shafaei P, Ziaee AA, Akbari MT, Pourmand G, Abedi A, et al. Mutations of RAS gene family in specimens of bladder cancer. Urol J. 2008;5:237–42.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/19101897/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Urol%20J&title=Mutations%20of%20RAS%20gene%20family%20in%20specimens%20of%20bladder%20cancer&author=N%20Karimianpour&author=P%20Mousavi-Shafaei&author=AA%20Ziaee&author=MT%20Akbari&author=G%20Pourmand&volume=5&publication_year=2008&pages=237-42&pmid=19101897&)
