# Resequencing of Nicotinic Acetylcholine Receptor Genes and Association of Common and Rare Variants with the Fagerström Test for Nicotine Dependence

## Metadata
**Authors:** Jennifer Wessel, Sarah M McDonald, David A Hinds, Renee P Stokowski, Harold S Javitz, Michael Kennemer, Ruth Krasnow, William Dirks, Jill Hardin, Steven J Pitts, Martha Michel, Lisa Jack, Dennis G Ballinger, Jennifer B McClure, Gary E Swan, Andrew W Bergen
**Journal:** Neuropsychopharmacology
**Date:** 2010 Aug 25
**DOI:** [10.1038/npp.2010.120](https://doi.org/10.1038/npp.2010.120)
**PMID:** 20736995
**PMCID:** PMC3055324
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3055324/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC3055324/pdf/npp2010120a.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC3055324/pdf/npp2010120a.pdf)

## Abstract

Common single-nucleotide polymorphisms (SNPs) at nicotinic acetylcholine receptor (nAChR) subunit genes have previously been associated with measures of nicotine dependence. We investigated the contribution of common SNPs and rare single-nucleotide variants (SNVs) in nAChR genes to Fagerström test for nicotine dependence (FTND) scores in treatment-seeking smokers. Exons of 10 genes were resequenced with next-generation sequencing technology in 448 European-American participants of a smoking cessation trial, and CHRNB2 and CHRNA4 were resequenced by Sanger technology to improve sequence coverage. A total of 214 SNP/SNVs were identified, of which 19.2% were excluded from analyses because of reduced completion rate, 73.9% had minor allele frequencies <5%, and 48.1% were novel relative to dbSNP build 129. We tested associations of 173 SNP/SNVs with the FTND score using data obtained from 430 individuals (18 were excluded because of reduced completion rate) using linear regression for common, the cohort allelic sum test and the weighted sum statistic for rare, and the multivariate distance matrix regression method for both common and rare SNP/SNVs. Association testing with common SNPs with adjustment for correlated tests within each gene identified a significant association with two CHRNB2 SNPs, eg, the minor allele of rs2072660 increased the mean FTND score by 0.6 Units (P=0.01). We observed a significant evidence for association with the FTND score of common and rare SNP/SNVs at CHRNA5 and CHRNB2, and of rare SNVs at CHRNA4. Both common and/or rare SNP/SNVs from multiple nAChR subunit genes are associated with the FTND score in this sample of treatment-seeking smokers.

Keywords: Fagerström test for nicotine dependence, single-nucleotide polymorphism, candidate gene association scan, treatment-seeking smokers

## INTRODUCTION

Cigarette smoking is the primary risk factor for many chronic diseases ([Bergen and Caporaso, 1999](#bib4)). Genetic epidemiological studies have estimated the heritability of nicotine dependence to range from 40 to 75% ([Broms, 2007](#bib10) no. 1225; [Li, 2003](#bib45) no. 39; [Maes, 2004](#bib55) no. 402; [Vink, 2005](#bib84) no. 480; [Lessov, 2004](#bib42) no. 472). Recent candidate gene association studies ([Baker et al, 2009](#bib2); [Bergen et al, 2009](#bib5); [Bierut et al, 2008](#bib6); [Breitling et al, 2009a](#bib8), [2009b](#bib9); [Chen et al, 2008](#bib13), [2009](#bib14); [Etter et al, 2009](#bib23); [Freathy et al, 2009](#bib27); [Greenbaum et al, 2006](#bib30); [Hoft et al, 2009](#bib35); [Keskitalo et al, 2009](#bib38); [Le Marchand et al, 2008](#bib40); [Li et al, 2008](#bib46), [2009](#bib47); [Perkins et al, 2008a](#bib60), [2008b](#bib61), [2009](#bib62); [Philibert et al, 2009](#bib63); [Rigbi et al, 2008](#bib66); [Saccone et al, 2009a](#bib68), [2009b](#bib69); [Sherva et al, 2008](#bib74); [Spitz et al, 2008](#bib76); [Stevens et al, 2008](#bib77); [Wang et al, 2009](#bib86); [Weiss et al, 2008](#bib88)), and genome-wide association scans (GWASs) ([Drgon et al, 2009a](#bib20), [2009b](#bib21); [Liu et al, 2010](#bib48), [2009](#bib49); [TAG, 2010](#bib81); [Thorgeirsson et al, 2008](#bib82), [2010](#bib83)), with many published candidate gene and GWAS studies through 2009 referenced by [Wang and Li (2010)](#bib85), have searched for, and, at varying levels of significance, identified, common variants associated with measures of response to tobacco, tobacco consumption, nicotine dependence, nicotine metabolism, or smoking cessation. However, most of the attributable fraction assumed to be due to heredity cannot be explained by these common single-nucleotide polymorphisms (SNPs), suggesting that rare variants in candidate genes, ie, those with minor allele frequencies (MAFs) substantially <5%, may also contribute to complex disease ([Bodmer and Bonilla, 2008](#bib7); [Frazer et al, 2009](#bib26); [Schork et al, 2009](#bib72)). In contrast to common SNP associations, which typically exhibit odds ratios in case–control studies substantially less than two, rare variants at the genetic loci known to contribute to specific diseases exhibit substantially greater odds ratios, leading to the suggestion that despite their reduced allele frequency, rare variants might account for a substantial minor fraction of the population attributable risk due to genetic variation ([Bodmer and Bonilla, 2008](#bib7)). Recent technological advances in high-throughput, next-generation sequencing enable direct sequencing of candidate genes, which is the optimal way to identify rare variants ([Harismendy et al, 2009](#bib31)). A number of methods for identifying associations between rare variants and common diseases have been developed ([Li and Leal, 2008](#bib43); [Madsen and Browning, 2009](#bib54); [Schork et al, 2008](#bib73)), which share the approach of combining information obtained from multiple variants.

In light of recent evidence from GWAS and candidate gene results, we chose 10 presynaptic neuronal nicotinic acetylcholine receptor (nAChR) subunit genes (namely *CHRNA2*, *CHRNA3*, *CHRNA4*, *CHRNA5*, *CHRNA6*, *CHRNA7*, *CHRNB1*, *CHRNB2*, *CHRNB3*, and *CHRNB4*) for resequencing to identify both common and rare variants for association analyses to measures of smoking behavior collected in a sample of treatment-seeking smokers involved in a randomized behavioral and pharmacological intervention for smoking cessation ([Swan et al, 2010](#bib80)). Collectively, these genes code for receptors that bind the exogenous ligand nicotine, responsible for nicotine addiction ([Changeux, 2009](#bib12); [Dani and Bertrand, 2007](#bib18); [Stolerman and Jarvis, 1995](#bib78)). On the basis of recent studies in the literature ([Saccone et al, 2009a](#bib68); [Saccone et al, 2007](#bib70)), we hypothesized that common variation in these 10 nAChR subunit genes might contribute to variation in a commonly assessed measure of nicotine dependence, the Fagerström test for nicotine dependence (FTND) ([Heatherton et al, 1991](#bib32)). In addition, on the basis of the emerging appreciation for the potential of rare variants to contribute to common disease (the common disease:rare variant hypothesis), we hypothesized that rare variation at these genes might also contribute to variation in this measure of nicotine dependence in this same sample. We employed a recently developed method and a traditional method ([Margulies et al, 2005](#bib56); [Sanger et al, 1977](#bib71)) of resequencing to identify both common and rare sequence variations, and several recently developed methods for analysis of sequence variation that assesses the association between common and/or rare variants and phenotype combining information obtained from multiple variants to evaluate these hypotheses ([Li and Leal, 2008](#bib43); [Madsen and Browning, 2009](#bib54); [Schork et al, 2008](#bib73)). Herein, we assess the contribution of common and/or rare sequence variation identified in 10 nAChR subunit genes in a sample of treatment-seeking smokers with the FTND score collected at the time of intake in a randomized smoking cessation trial.

## MATERIALS AND METHODS

### Population and Phenotype

Current smokers (⩾10 cigarettes per day over the past year, *N*=1202) were recruited from the Group Health (GH), a consumer-governed nonprofit health-care organization that serves >630 000 residents of Washington and Idaho, to participate in a smoking cessation randomized behavioral intervention combined with the FDA-approved course of varenicline. Recruitment and treatment methods for the Comprehensive Medication Program and Support Services (COMPASS) study, a randomized trial ([NCT00301145](https://clinicaltrials.gov/ct2/show/NCT00301145)) sponsored by the National Cancer Institute, have been described previously ([Swan et al, 2010](#bib80)). Nicotine dependence was measured at baseline with the FTND. Scores on this scale range from 0 to 10 with a higher score indicating greater dependence; although the FTND has suboptimal psychometric properties, it continues to be used in part because of the amount of previous clinical, descriptive, and genetic research that has used it, and in part because of the modest associations observed with the clinical outcome of smoking cessation ([Piper et al, 2006](#bib64)). DNA was extracted from saliva samples provided by COMPASS participants ([Nishita et al, 2009](#bib59)), and DNA obtained from 448 individuals who self-identified as belonging to non-Hispanic White race/ethnicity, who had never used varenicline previously, and who had questionnaire data on smoking habits, was used for genetic analyses. All protocols were reviewed and approved by the Institutional Review Boards of the GH and SRI International.

### 454 Resequencing

We designed target regions (amplicons) to amplify nAChR subunit gene exons and flanking regions including splice sites and selected 5′ and 3′ untranslated regions (UTRs) and SNPs upstream and downstream of transcription units. Primer design, sample preparation, and sequencing followed the protocols supplied by Roche for the 454 Genome Sequencer System (Supplementary File 1). We used the Roche GS Reference Mapper alignment tool to align sequence reads to a target consisting of the amplified intervals from the reference genome (NCBI Build 36). We separately used BLAST ([Altschul et al, 1990](#bib1)) to align each read against the entire reference genome, and removed reads with the best BLAST hits that did not match the GS Reference Mapper alignment to the target. Candidate single-nucleotide variants (SNVs) were identified by parsing the GS Reference Mapper alignments and identifying positions at which at least one individual had reads of the same alternate base in forward and reverse orientations. We then used a Bayesian algorithm similar to that used in the study by [Craig et al (2008)](#bib17) to assign genotypes and quality scores to all sequenced individuals at each putative variant position based on read counts for the reference and alternate base. This approach assigns genotypes taking into account genotype frequencies and error rates estimated at each variant position. Genotypes that were less than twice as likely as the next most likely genotype were coded as missing. Positions that were substantially out of the Hardy–Weinberg equilibrium (*P*<10^−7^) were excluded from analysis; the Hardy–Weinberg equilibrium test has power to detect only disequilibrium with common SNPs.

### Sample Quality Control

We excluded 18 DNA samples with overall call rates <90%. The amplified regions included 29 SNPs in the Phase II HapMap release, 24 that were polymorphic in the CEU (Utah residents with Northern and Western European ancestry from the CEPH collection) panel. All of these SNPs were identified as polymorphic in the sequencing data, although four failed a quality control (QC) threshold as a result of fold coverage <10 × . Allele frequencies for the remaining 25 were within 6% of HapMap CEU frequencies. We included 10 additional amplicons containing 1 QC SNP each and a single Y-chromosome amplicon as control amplicons. Each of the sequenced DNA samples was also genotyped with a custom high-density oligonucleotide array containing 312 SNPs that contained the 10 QC SNPs ([Hinds et al, 2004](#bib34)). Principal components analysis of the SNP array data did not indicate the presence of population structure, and confirmed that all samples were of European ancestry. No samples were excluded from association analyses because of a high discordance rate between the genotypes for the 10 QC SNPs from the array and sequencing data sets (data not shown).

### Sanger Resequencing

Selected regions of *CHRNB2* and *CHRNA4* with low read depth (<10X) from 454 resequencing were resequenced using Sanger sequencing methods ([Sanger et al, 1977](#bib71)), primers as developed by [Weiss et al (2008)](#bib88) or as optimized in this study, enzymatic methods for PCR as optimized in this study, and using BigDye Terminator v3.1 Cycle Sequencing protocols on a Applied Biosystems 3130xl Genetic Analyzer. Variant detection was performed using Mutation Surveyor v3.24 (SoftGenetics, LLC, Penn State, Pennsylvania), Reference Sequence (RefSeq) [NM_000748.2](https://www.ncbi.nlm.nih.gov/nuccore/NM_000748.2) for *CHRNB2* and [NM_000744.5](https://www.ncbi.nlm.nih.gov/nuccore/NM_000744.5) for *CHRNA4* and reference files generated in the Mutation Surveyor using the RefSeq and the ‘generate synthetic chromatograms' function. See Supplementary File 1 for further details.

### Association Analyses

Common variants (MAF ⩾5%) were analyzed separately for association with normally distributed FTND using a generalized linear model. Genotypes were coded in additive (Add) or dominant (Dom) models. The significance of regression models is reported for each SNP and with adjustment for correlated tests (*P*_ACT_) ([Conneely and Boehnke, 2007](#bib15)) for the most significantly associated SNP within each gene and model to a *P*_ACT_ value threshold of 0.10. For rare variants, gene-based association tests were performed by the cohort allelic sum test (CAST) and by the weighted sum statistic (WSS) ([Madsen and Browning, 2009](#bib54)). CAST tests for association between FTND and counts of rare alleles were based on two fixed thresholds for ‘rareness' (MAF <0.01 and <0.05). The WSS was used to test for association between FTND and weighted counts of minor alleles (MAF <0.05), with an inverse relationship between weights and the MAF of minor alleles. Both tests are applied only to rare variants under the assumption that rare variants are more likely to be deleterious than common variants ([Kryukov et al, 2009](#bib39)). Linear regression coefficients, *P*-values from likelihood ratio tests, and empirical *P*-values from permutation testing are reported. Multivariate distance-based matrix regression (MDMR) was also employed to test associations of common and rare (MAF <0.05) variants with FTND, with either identical-by-state allele sharing across individuals and variants in each gene, and with allele sharing weighted by the Lynch–Ritland calculation, with 100 000 permutations. The latter approach gives more weight to rare variants ([Nievergelt et al, 2007](#bib58); [Schork et al, 2008](#bib73); [Wessel and Schork, 2006](#bib89)). For nAChR subunit genes that are clustered in the genome (*CHRNB3* and *CHRNA6* at chr8p11, and *CHRNA5*, *CHRNA3*, and *CHRNB4* at chr15q25.1), we also performed association analyses to evaluate all sequenced SNPs available in these genes together. When MDMR analyses with both common and rare SNPs identified a significant association, we performed and report results from two *post hoc* tests: analysis of rare SNPs alone and analysis of common SNPs alone. Power to detect effect sizes with parameters given, *α*=0.05 and a two-sided test was estimated using Quanto software ([Gauderman, 2002](#bib28)). Pairwise linkage disequilibrium (LD) (*D*′ and *r*^2^) values were calculated from the COMPASS sequence data using Haploview ([Barrett et al, 2005](#bib3)). Genes and SNPs are ordered in tables by the chromosome and coordinate, respectively.

## RESULTS

### nAChR Exome Sequencing and SNP Discovery

We attempted to resequence all coding exons, splice sites, and selected SNPs at 10 nAChR subunit genes comprising 18 kb of target sequence in 448 European ancestry treatment-seeking participants of the COMPASS clinical trial using 454 sequencing supplemented by Sanger sequencing for two genes (Supplementary File 1). For the 10 nAChR subunit genes sequenced by 454 technology, 66% of targeted base pairs in these amplicons had >10 reads, with 3 genes accounting for most of the shortfall in target coverage (*CHRNA4*, 0% *CHRNB2*, 22%, and *CHRNA7*, 39%, *vs* an average for the 7 other genes of 81%, please see Table 3 and [Figure 1](#fig1) in Supplementary File 1). A total of 167 unique SNPs were identified through 454 sequencing. Before analyses, we excluded 18 DNA samples and 24 SNPs that had call rates <90% and 5 SNPs that were monomorphic in the remaining 430 samples. The mean (SD) Sanger amplicon resequencing read depth was 1.98 (0.20) and 1.88 (0.43) for *CHRNB2* and *CHRNA4*, respectively, and SNPs were scored using default base-calling parameters. Sanger resequencing identified 15 SNPs at *CHRNB2* and 32 SNPs at *CHRNA4* (Supplementary File 1 and [Table 1](#tbl1)). Six SNPs found in *CHRNA4* exon 5 and *CHRNB2* exon 5 by 454 resequencing were not found by Sanger resequencing and hence were excluded. One SNP, found by both methods, was unreliable in both methods, because of poor coverage in 454 resequencing and unclear reads in Sanger resequencing and hence was also excluded. Using the two methods, a total of 172 SNPs were identified in 430 samples (Supplementary Table 1). The observed distribution of MAFs in the total sample is strongly skewed to low MAF values ([Figure 1](#fig1) and Supplementary Table 1), in which 73.9% have MAF <5% and 65.3% have MAF <1%, with MAFs ranging from 0.1 to 49.2%. Percentages and types of variants identified (Supplementary Table 1) are 4.1% 5′ of the 5′ UTR, 4.1% 5′ UTR, 51.4% coding (26.0% nonsynonymous, 25.4% synonymous), 23.1% intronic, 14.5% 3′ UTR, and 2.9% 3′ of the 3′ UTR.

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d96f/3055324/6504f55e5083/npp2010120f1.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3055324_npp2010120f1.jpg)

Minor allele frequencies (x axis), proportion (left y axis) and cumulative fraction (right y axis) of nAChR SNPs identified through resequencing. Bars represent the percentage of SNPs within each category of minor allele frequency (MAF) with the figure under each bar being the maximum MAF within that category. The line represents the cumulative frequency of SNPs.

### Table 1. Common nAChR Subunit Gene Variant Associations with the FTND Score.

| SNP ID | A1a | A2b | Gene | Type | MAF | Add Pc | Dom Pd |
| --- | --- | --- | --- | --- | --- | --- | --- |
| rs2280781 | C | T | CHRNB2 | 5′ UTR | 0.101 | 0.221 | 0.280 |
| rs4845378 | G | T | CHRNB2 | Intron | 0.097 | 0.258 | 0.258 |
| rs2072660 | T | C | CHRNB2 | 3′ UTR | 0.244 | 0.003 | 0.006 |
| rs2072661 | G | A | CHRNB2 | 3′ UTR | 0.245 | 0.002 | 0.004 |
| rs4292956 | C | T | CHRNB2 | 3′ UTR | 0.066 | 0.472 | 0.476 |
| rs2472553 | G | A | CHRNA2 | Non-syn | 0.139 | 0.571 | 0.493 |
| rs13277254 | G | A | CHRNB3 | Up | 0.234 | 0.829 | 0.875 |
| rs13280301 | A | G | CHRNB3 | Up | 0.178 | 0.675 | 0.798 |
| rs6474413 | C | T | CHRNB3 | Up | 0.232 | 0.746 | 0.702 |
| rs4950 | G | A | CHRNB3 | 5′ UTR | 0.236 | 0.808 | 0.854 |
| rs2304297 | G | C | CHRNA6 | 3′ UTR | 0.244 | 0.557 | 0.985 |
| ss161151744 | C | T | CHRNA7 | Syn | 0.066 | 0.314 | 0.314 |
| rs569207 | C | T | CHRNA5 | Intron | 0.196 | 0.134 | 0.149 |
| rs16969968 | G | A | CHRNA5 | Non-syn | 0.367 | 0.433 | 0.322 |
| rs615470 | T | C | CHRNA5 | 3′ UTR | 0.381 | 0.330 | 0.329 |
| rs8192482 | C | T | CHRNA5 | 3′ UTR | 0.368 | 0.489 | 0.388 |
| rs564585 | A | G | CHRNA5 | 3′ UTR | 0.237 | 0.080 | 0.139 |
| rs12899226 | T | G | CHRNA3 | Down | 0.052 | 0.104 | 0.104 |
| rs660652 | A | G | CHRNA3 | 3′ UTR | 0.383 | 0.283 | 0.329 |
| rs472054 | A | G | CHRNA3 | 3′ UTR | 0.379 | 0.381 | 0.367 |
| rs578776 | A | G | CHRNA3 | 3′ UTR | 0.247 | 0.053 | 0.112 |
| rs1051730 | G | A | CHRNA3 | Syn | 0.367 | 0.455 | 0.359 |
| rs3743075 | T | C | CHRNA3 | Syn | 0.378 | 0.292 | 0.305 |
| rs3743074 | G | A | CHRNA3 | Intron | 0.378 | 0.336 | 0.324 |
| rs8040868 | T | C | CHRNA3 | Syn | 0.429 | 0.981 | 0.813 |
| rs8192475 | C | T | CHRNA3 | Non-syn | 0.050 | 0.094 | 0.094 |
| rs12914008 | G | A | CHRNB4 | Non-syn | 0.051 | 0.100 | 0.100 |
| rs3813567 | G | A | CHRNB4 | Up | 0.157 | 0.370 | 0.275 |
| rs2302765 | T | C | CHRNB1 | Intron | 0.159 | 0.729 | 1.000 |
| rs12452047 | A | G | CHRNB1 | Intron | 0.166 | 0.616 | 0.808 |
| rs7210231 | C | A | CHRNB1 | Intron | 0.199 | 0.124 | 0.017 |
| rs2302761 | C | T | CHRNB1 | Intron | 0.202 | 0.233 | 0.078 |
| rs2302763 | T | C | CHRNB1 | Intron | 0.164 | 0.679 | 0.889 |
| rs2302764 | T | C | CHRNB1 | 3′ UTR | 0.160 | 0.049 | 0.045 |
| rs3827020 | T | C | CHRNA4 | Intron | 0.143 | 0.521 | 0.865 |
| rs45442394 | G | A | CHRNA4 | Intron | 0.060 | 0.431 | 0.717 |
| rs1044397 | C | T | CHRNA4 | Syn | 0.492 | 0.305 | 0.608 |
| rs1044396 | G | A | CHRNA4 | Syn | 0.488 | 0.215 | 0.467 |
| rs2229960 | A | G | CHRNA4 | Syn | 0.057 | 0.870 | 0.758 |
| rs2229959 | C | A | CHRNA4 | Syn | 0.119 | 0.837 | 0.806 |
| rs1044394 | A | G | CHRNA4 | Syn | 0.067 | 0.784 | 0.678 |
| rs6090384 | T | C | CHRNA4 | Intron | 0.060 | 0.830 | 0.933 |
| rs2273505 | C | T | CHRNA4 | Intron | 0.062 | 0.740 | 0.725 |
| rs2273506 | G | A | CHRNA4 | Syn | 0.061 | 0.580 | 0.562 |

Table 1 Caption: aA1=allele 1. bA2=allele 2. cAdd=additive model. dDom=dominant model. The bold values are for P<0.05.

### Treatment-Seeking Smoker Characteristics

The 430 individuals included in association analyses of nAChR subunit gene SNPs and the FTND score are mostly female (68%), with a mean (SD) age of 49.0 years (11.3) and of cigarettes smoked per day 20.4 (8.5). In this sample of 430 treatment-seeking smokers, gender and age were not significantly associated with FTND (*P*>0.05). The sample had mean (SD) FTND scores of 5.2 (2.1), indicating moderate addiction to tobacco ([Heatherton et al, 1991](#bib32)). This mean FTND score is higher than population-based samples ([Fagerstrom and Furberg, 2008](#bib24)), but typical of treatment-seeking smokers from this patient population ([Swan et al, 2003](#bib79); [Swan et al, 2010](#bib80)) and other treatment-seeking smoker samples ([Bergen et al, 2009](#bib5)).

### Common Variation Association Analyses

In all, 44 SNPs with MAF ⩾5% were tested for association with FTND using two transmission models. Significant (*P*<0.05) unadjusted associations were found between *CHRNB1* (rs2302764 and rs7210231, *D*′=1, *r*^2^=0.049) and *CHRNB2* (rs2072660 and rs2072661, *D*′=1, *r*^2^=0.918) SNPs with FTND ([Table 1](#tbl1)). After adjustment for multiple correlated tests within each gene, significant associations remained between two *CHRNB2* SNPs located in the 3′ UTR and the FTND (rs2072660, *P*_ACT, Additive_=0.013, and rs2072661, *P*_ACT, Additive_=0.009). Individuals with one or two copies of the minor allele of these *CHRNB2* SNPs exhibit a 0.6 unit increase in the mean FTND score (for rs2072661, FTND mean (SD)=5.5 (0.2) *vs* 4.9 (0.1)). After adjustment for correlated tests, the minor allele of the *CHRNB1* SNP rs7210231 was nonsignificantly (*P*_ACT, Dominant_=0.082) associated with a decrease in the FTND score. In a *post hoc* test of mean FTND scores, individuals with the rs7210231 heterozygote genotype exhibit significantly lower mean FTND scores than do individuals with homozygote genotypes (heterozygote mean (SE)=4.70 (0.19), major allele homozygote=5.77 (0.50), minor allele homozygote mean (SE)=5.36 (0.13), with heterozygote to major homozygote *P*=0.044 and heterozygote to minor homozygote *P*=0.004).

### Common and Rare Variation Analyses

Association with rare SNPs within a gene and the FTND are reported in [Table 2](#tbl2) for WSS (MAF<0.05) and CAST (two thresholds, MAF <0.01 and <0.05) analyses, and associations with common and rare (MAF<0.05) SNPs and FTND score are reported in [Table 3](#tbl3) for MDMR analyses, along with *post hoc* MDMR tests. Significant (*P*<0.05) association with rare variation in *CHRNA4* and the FTND score was observed using the CAST tests, but not the WSS test ([Table 2](#tbl2)). Significant associations between common and rare variants and the FTND score were identified at *CHRNB2*, *CHRNA5*, and at the chr15q25.1nAChR locus using the MDMR method ([Table 3](#tbl3)). When either common or rare variants at these genes were removed before association analysis, associations between rare or common SNPs at *CHRNA5* and at the chr15q25.1 nAChR locus and the FTND score, and those between rare *CHRNB2* SNPs and the FTND score, were no longer significant. Some association results that did not reach the threshold of statistical significance used herein, eg, *P*-values between 0.05 and 0.10, suggest that common variation at *CHRNA3* ([Table 1](#tbl1)), rare variation at *CHRNB2* and *CHRNA6* ([Table 2](#tbl2)), common and rare variations at *CHRNA2* ([Table 3](#tbl3)), and common and/or rare variation at *CHRNB1* ([Tables 1](#tbl1) and [3](#tbl3)) may also contribute to the FTND score in this sample of treatment-seeking smokers. We note that *CHRNA6* and *CHRNA7* had very limited numbers of variants available for analysis, and thus tests for association at these genes that combine variants will have low power.

### Table 2. Rare nAChR Subunit Gene Variant Associations with the FTND Score.

|   | CASTβa | P1b | P2c | CASTβd | P1 | P2 | WSSβe | P1 | P2 |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| CHRNB2 | −1.51 | 0.09 | 0.08 | −1.51 | 0.09 | 0.08 | −0.0067 | 0.09 | 0.10 |
| CHRNA2 | 0.48 | 0.53 | 0.55 | 0.48 | 0.53 | 0.49 | 0.0007 | 0.48 | 0.53 |
| CHRNB3 | 0.85 | 0.49 | 0.52 | 0.85 | 0.49 | 0.54 | 0.0038 | 0.49 | 0.59 |
| CHRNA6 | 2.86 | 0.06 | 0.12 | 2.86 | 0.06 | 0.08 | 0.0127 | 0.06 | 0.08 |
| chr8p11 | 1.66 | 0.08 | 0.07 | 1.66 | 0.08 | 0.08 | 0.0074 | 0.08 | 0.07 |
| CHRNA7 | 1.35 | 0.37 | 0.33 | 1.35 | 0.37 | 0.35 | 0.0060 | 0.38 | 0.32 |
| CHRNA5 | 0.10 | 0.93 | 0.94 | 0.61 | 0.19 | 0.23 | −0.0005 | 0.54 | 0.60 |
| CHRNA3 | −2.17 | 0.15 | 0.10 | −2.17 | 0.15 | 0.22 | 0.0022 | 0.15 | 0.19 |
| CHRNB4 | 0.49 | 0.45 | 0.40 | −0.13 | 0.75 | 0.67 | 0.0013 | 0.31 | 0.23 |
| chr15q25.1 | 0.08 | 0.88 | 0.86 | 0.10 | 0.74 | 0.74 | 0.0002 | 0.78 | 0.77 |
| CHRNB1 | −1.17 | 0.28 | 0.33 | −1.17 | 0.28 | 0.32 | −0.0052 | 0.28 | 0.25 |
| CHRNA4 | −1.70 | 0.04 | 0.04 | −1.70 | 0.04 | 0.03 | −0.0011 | 0.45 | 0.43 |

Table 2 Caption: aCAST-β, MAF <0.01. bP1=P-value from standard F-test. cP2=P-value from permutation testing. dCAST-β, MAF <0.05. eWSS-β. The bold values are for P<0.05.

### Table 3. Common and Rare nAChR Subunit Gene Variant Associations with the FTND Score.

| Gene | N SNPs | Allele sharing |  |  | Weighted allele sharing |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- |
|   |   | Pseudo-F | P-value | % Variation | Pseudo-F | P-value | % Variation |
| CHRNB2 | 24 | 7.57 | 0.002 | 0.017 | 4.53 | 0.019 | 0.011 |
| CHRNB2a | 19 | 0.46 | 0.586 | 0.001 | 2.22 | 0.137 | 0.005 |
| CHRNB2b | 5 | 7.51 | 0.003 | 0.018 | 5.48 | 0.010 | 0.013 |
| CHRNA2 | 11 | 0.69 | 0.503 | 0.002 | 23.23 | 0.095 | 0.052 |
| CHRNB3 | 12 | 0.11 | 0.822 | 0.000 | 0.20 | 0.677 | 0.001 |
| CHRNA6 | 3 | 0.63 | 0.436 | 0.002 | 4.24 | 0.361 | 0.010 |
| chr8p11 | 15 | 0.09 | 0.838 | 0.000 | 0.20 | 0.702 | 0.001 |
| CHRNA7 | 4 | 1.24 | 0.275 | 0.003 | −12.02 | 0.814 | −0.029 |
| CHRNA5 | 15 | 1.27 | 0.299 | 0.003 | 3.92 | 0.046 | 0.009 |
| CHRNA5a | 10 | −0.48 | 0.886 | −0.001 | 0.55 | 0.556 | 0.001 |
| CHRNA5b | 5 | 1.85 | 0.177 | 0.005 | 2.64 | 0.124 | 0.007 |
| CHRNA3 | 33 | 1.59 | 0.203 | 0.004 | 1.86 | 0.199 | 0.004 |
| CHRNB4 | 15 | 1.16 | 0.343 | 0.003 | 0.12 | 0.807 | 0.000 |
| chr15q25.1 | 63 | 1.40 | 0.249 | 0.003 | 3.44 | 0.044 | 0.008 |
| chr15q25.1a | 47 | 0.911 | 0.543 | 0.002 | −0.18 | 0.896 | 0.000 |
| chr15q25.1b | 16 | 1.66 | 0.198 | 0.004 | 2.74 | 0.107 | 0.007 |
| CHRNB1 | 25 | 1.67 | 0.206 | 0.004 | 2.84 | 0.064 | 0.007 |
| CHRNA4 | 31 | 0.53 | 0.616 | 0.001 | 21.55 | 0.169 | 0.050 |

Table 3 Caption: aPost hoc MDMR test performed with rare variants only. bPost hoc MDMR test performed with common variants only. The bold values are for P<0.05.

## DISCUSSION

We resequenced the exons of 10 highly relevant candidate genes for the study of nicotine dependence ([Changeux, 2009](#bib12)), using DNA extracted from saliva ([Nishita et al, 2009](#bib59)), provided by treatment-seeking participants of a randomized smoking cessation trial ([Swan et al, 2010](#bib80)), using a next-generation sequencing method ([Margulies et al, 2005](#bib56)), together with the Sanger method ([Sanger et al, 1977](#bib71)). We identified sequence variation and employed standard regression and specialized analysis methods ([Madsen and Browning, 2009](#bib54); [Wessel and Schork, 2006](#bib89)) for analyzing association of this variation with the FTND score. The latter analysis methods group SNPs from a gene together into a summary variable to test for an excess of variants associated with the phenotype ([Li and Leal, 2008](#bib43)). We selected methods to analyze only common (regression), only rare (CAST and WSS), and both common and rare (MDMR) variations at a gene to evaluate the possible influence of common and rare variations under the common disease:common variant and the common disease:rare variant hypotheses ([Bodmer and Bonilla, 2008](#bib7); [Schork et al, 2009](#bib72)).

In the sample of 430 DNA samples obtained from treatment-seeking smokers and 173 nAChR subunit gene SNPs analyzed for association, we observed that common variants at *CHRNB2* and rare variants at *CHRNA4* were significantly associated with the FTND score, after correction for multiple correlated tests and by permutation testing within each gene, respectively. Employment of a method (allele sharing and weighted allele sharing MDMR with permutation testing) that assesses the contribution of common and rare variants combined, and *post hoc* analyses with rare or common variants alone, identified a significant association with variation in the FTND score at *CHRNB2*, *CHRNA5*, and at the chr15q25.1nAChR locus with both types of variation, and with the combination of common variants alone at *CHRNB2*.

There are at least two possible mechanisms underlying the latter findings described above, ie, the observation that the combination of common SNPs and rare variants, or of common SNPs alone, at a gene or gene region, are associated with FTND, but that rare variants do not exhibit association. One possible mechanism is the interaction between common variants at the same gene or within the same gene region, which is the mechanism that has been elucidated by the studies by Li *et al* for the chr15q25.1 nAChR cluster, in which the interaction among *CHRNA3* SNPs was observed in association with measures of nicotine dependence ([Li et al, 2009](#bib47); [Lou et al, 2008](#bib50)). Formal interaction analyses within genes or within multigene regions will be necessary to explore the possible role of this mechanism in the observed association of common and rare or common variants alone with the FTND score in the COMPASS sample. Another possible mechanism is the phenomenon of common SNPs acting as tagging SNPs for multiple rare causal variants that exist in the same LD region, ie, so-called synthetic associations ([Dickson et al, 2010](#bib19); [Wang et al, 2010](#bib87)). When we removed common SNPs from the association test as we did when testing only rare variants in the MDMR analysis, we may have removed the only SNPs that have the power to detect the association of rare causal variants with the FTND score. It is well known that the chr15q25.1 nAChR locus exhibits extensive LD across a region of >200 000 bp, and that the chr1q21.3 *CHRNB2* gene region exhibits strong LD extending ∼150 000 bp in the HapMap CEU sample 3′ from *CHRNB2* through *ADAR* to the 3′ exons of *KCNN3*; thus conditions exist for synthetic associations to be present at the chr15q25.1 and chr1q21.3 nAChR loci. Resequencing of selected individuals based on a common SNP allele or haplotype content, followed by common and rare variant analyses, will be necessary to test the synthetic association hypothesis in these two regions.

Three other groups have resequenced nAChR subunit genes to identify variants for subsequent association analysis. The first group used DNA obtained from 192 European Americans (144 smokers and 48 nonsmokers sampled from low (0–4) and high (6–10) FTND strata) to resequence 10 genes and identified 262 SNPs ([Weiss et al, 2008](#bib88)). They identified the chr15q25.1nAChR locus to be significantly associated with the FTND strata in 144 smokers (72 high *vs* 72 low FTND strata) and genotyped 87 tag SNPs selected with MAF >0.05 across the 10 genes in 3 larger cohorts (total *N*=2827), and a significant association between 6 chr15q25.1 nAChR locus SNPs with the FTND strata was observed among individuals initiating smoking at age ⩽16 years. The size of the COMPASS sample is approximately one-sixth that of the sample used by Weiss *et al* to characterize the effect size of chr15q25.1nAChR locus haplotypes. Replication of the Weiss *et al* finding will require the use of a larger sample of treatment-seeking smokers than available in the COMPASS sample. [Sabatelli et al (2009)](#bib67) resequenced 3 nAChR subunit genes (namely *CHRNA3*, *CHRNA4*, and *CHRNB4*) in 245 sporadic amyotrophic lateral sclerosis (SALS) cases and 450 controls from Italy identifying 55 SNPs; significantly more missense variants in the SALS cases compared with controls were observed in the intracellular loop region of nAChRs (*P*=0.0001). In our analyses, we identified and evaluated 45 nonsynonymous variants at 10 nAChR subunit genes for association with the FNTD score, most of which are rare (38 with MAF <0.01 and 41 with MAF <0.05). Using several methods to assess the association with rare variants in the COMPASS sample, we identified evidence for a significant association with rare variants at *CHRNA4* with the FTND score. We observed a total of 21 rare (20 variants with MAF <0.01 and 1 variant with MAF <0.05) *CHRNA4* variants, with 5 nonsynonymous substitutions, including a nonsense variant, in the COMPASS sample analyzed. Further stratification of association testing of nAChR subunit gene variants in the COMPASS sample by variant type, nAChR structural region, and evaluation of function by *in vitro* expression and electrophysiology would be necessary to test the finding of Sabetelli *et al.* [Rana et al (2009)](#bib65) resequenced the 3 chr15q25.1nAChR subunit genes in 80 individuals from 5 ethnicities identifying 63 variants; upon genotyping of 6 tag SNPs, significant associations in 370 European-American twins with *CHRNA3* SNPs rs3743075 and rs3743074 (in strong LD) were found with plasma catestatin and systolic blood pressure levels. These common *CHRNA3* SNPs were not associated with the FTND score in the COMPASS sample.

It is noteworthy that three recent meta-analyses of GWAS of smoking phenotypes in DNA samples obtained from population and clinically based data sets exceeding 140 000 individuals *in toto* have identified several common SNPs at the chr15q25.1 nAChR locus (rs55853698 and rs6495308 ([Liu et al, 2010](#bib48)); rs1051730 and rs16969968 ([TAG, 2010](#bib81)); and rs1051730 ([Thorgeirsson et al, 2010](#bib83))) associated with smoking quantity. In addition, one of these meta-analyses also identified common SNPs at the chr8p11nAChR locus (rs6474412 and rs13280604 ([Thorgeirsson et al, 2010](#bib83))) associated with smoking quantity. Of these six SNPs, our resequencing study identified only rs1051730 and rs16969968, and neither of these SNPs was statistically significantly associated with the FTND score in our sample. However, we did observe an association with multiple common SNPs at *CHRNA5* and at the chr15q25.1 nAChR locus using the MDMR approach, although none of the 16 common chr15q25.1 nAChR SNPs showed a nominally significant association (*P*<0.05) in individual regression models.

Our finding of a significant association of *CHRNB2* common variants with a measure of nicotine dependence is supported by some previous candidate gene studies reporting significant associations with the FTND and related phenotypes in treatment-seeking, in laboratory, and in population and clinically based data sets distinct from the treatment-seeking smokers we studied ([Swan et al, 2010](#bib80)). An analysis of 417 treatment-seeking smokers from a placebo-controlled randomized clinical trial of bupropion and placebo ([Lerman et al, 2006](#bib41)) reported significant associations with *CHRNB2* SNPs rs2072660 and rs2072661 and abstinence in a logistic model incorporating main effects and interaction with treatment at the end of treatment (*P*=0.01) and at 6 months (*P*=0.0002) ([Conti et al, 2008](#bib16)). A short-term test of nicotine *vs* placebo patch effects in 156 smokers reported a significant association between rs2072661 and the number of abstinent days ([Perkins et al, 2009](#bib62)). In an analysis of *CHRNA4* and *CHRNB2* SNPs in a sample of 1068 older adolescents, rs2072660 was reported to be significantly associated with sensitivity to tobacco (subjective effects of ‘nauseous' or ‘dizzy') ([Ehringer et al, 2007](#bib22)).

However, candidate gene studies evaluating the association of *CHRNB2* SNPs rs2072660 and/or rs2072661 with smoking status and measures of nicotine dependence have more often failed to identify significant associations. These included studies recruiting individuals, twins, or families from the community, including 2037 individuals from 602 multiplex heavy-smoking pedigrees ([Li et al, 2005](#bib44)), 1929 ever-smoking unrelated individuals ([Saccone et al, 2009a](#bib68); [Saccone et al, 2007](#bib70)), 872 unrelated members of a population-based twin sample ([Silverman et al, 2000](#bib75)), 742 unrelated individuals recruited for behavioral studies ([Lueders et al, 2002](#bib53)), 621 heavy-smoking men from 206 families recruited for nicotine addiction studies ([Feng et al, 2004](#bib25)), 516 individuals recruited for behavioral studies ([Philibert et al, 2009](#bib63)), in addition to treatment-seeking smokers ([Bergen et al, 2009](#bib5)). In this last study, a combined analysis of 821 treatment-seeking participants of two smoking cessation trials ([Lerman et al, 2006](#bib41)) did not identify significant associations between rs2072660 and rs2072661 and baseline nicotine dependence as measured by the FTND score (uncorrected *P*-values of 0.280 and 0.195 for rs2072660 and rs2072661, respectively) ([Bergen et al, 2009](#bib5)). Gene–gene interaction analyses did not identify significant interactions of *CHRNB2* SNPs (including rs2072660 and rs2072661) with SNPs from *CHRNA4*, *NTRK2*, and *BDNF* in association with smoking status in a sample of 191 unrelated smokers and 191 unrelated nonsmokers ([Lou et al, 2007](#bib51)), but did identify a significant interaction of *CHRNB2* SNPs (including rs2072660 and rs2072661) and *CHRNA4* SNPs with smoking status in a sample of 275 unrelated smokers with FTND scores ⩾4 and 348 unrelated nonsmokers ([Li et al, 2008](#bib46)).

A large candidate gene association study in 1929 ever smokers that included 119 SNPs at 16 nAChR subunit genes reported significant associations with SNPs at the *CHRNB3/CHRNA6*, *CHRNA5/CHRNA3/CHRNB4*, and *CHRND*/*CHRNG* multigene loci and a dichotomized measure of nicotine dependence ([Saccone et al, 2007](#bib70)). A follow-up study using the same sample and phenotype, and analyzing a total of 226 SNPs at 16 nAChR subunit genes, reported significant associations with SNPs within the same multigene loci, and nominal association (not associated after multiple test correction) with SNPs at *CHRNB1* and at *CHRNA4* ([Saccone et al, 2009a](#bib68)). We identified nominal associations between *CHRNB1* SNPs and the FTND score in our analyses of 430 treatment-seeking smokers, including at some *CHRNB1* SNPs studied by [Saccone et al (2009a)](#bib68) and [Lou et al (2006)](#bib52); however, these nominal associations became nonsignificant after correction for correlated tests at *CHRNB1*.

We identified a significant negative association between rare *CHRNA4* SNPs and the FTND score in the CAST test, which remained significant after permutation testing. Previous results of candidate gene association with *CHRNA4* ([Breitling et al, 2009b](#bib9); [Feng et al, 2004](#bib25); [Hutchison et al, 2007](#bib36); [Li et al, 2005](#bib44); [Saccone et al, 2009a](#bib68)) have suggested that multiple common *CHRNA4* SNPs may be associated with nicotine dependence-related phenotypes. The lack of a significant association of common *CHRNA4* SNPs in this sample with FTND could be because of a lack of power and/or differences in the distribution of common and/or rare SNPs in our sample of treatment-seeking smokers relative to previous samples. Differences in results between different tests of association of rare *CHRNA4* variation with the FTND score in this sample (CAST *vs* WSS) may be because of the large number of rare variants available at *CHRNA4*, which improves the power of the CAST test. However, the nominal significance observed in the CAST test and the lack of association observed in the WSS and MDMR tests suggest that this result should be regarded as preliminary and in need of replication.

One or more differences in study design, ascertainment criteria, assessment of nicotine dependence, case and control definitions, MAF thresholds, demographic variables, or sample sizes could be contributing to differences in results of association of nAChR subunit gene SNPs with measures of nicotine dependence or smoking status among the various studies. For example, the treatment-seeking smokers in our sample have a minimum past year history of ⩾10 cigarettes smoked per day, are mostly female, have an average age of 49 years and are screened for various physical and mental disorders typical of exclusion criteria for smoking cessation clinical trials. These smoking behavior characteristics differ somewhat from population- or family-based samples ascertained with smoking behavior criteria with lower or higher smoking intensities than those observed among treatment-seeking smokers. It is of interest that FTND scores seemed to be normally distributed (data not shown) in this treatment-seeking smoker sample, as in a previous investigation in this patient population ([Swan et al, 2003](#bib79)) and in other treatment-seeking smoker samples recently investigated ([Bergen et al, 2009](#bib5)). We note that the sequencing coverage varied across nAChR subunit genes in this study and that some genes had more common and/or rare SNPs identified than other genes in these analyses. Chance is another factor that could be responsible for association differences among studies.

There were limitations to our analysis. One limitation is the use of saliva-extracted DNA obtained from participants of a smoking cessation trial, in which genotyping performance using DNA extracted from saliva may be reduced by increased nonhuman DNA content ([Herraez and Stoneking, 2008](#bib33)). Three genes (namely *CHRNA4*, *CHRNB2*, and *CHRNA7*) exhibited reduced 454 resequencing coverage due to targeted regions of these genes with GC content that exceeds 80% (*CHRNA4*, 5′UTR, exon 1, exon 5, exon 6, 3′ UTR; *CHRNB2*, 5′UTR, exon 1, exon 2, exon 5, exon 6, 3′ UTR; *CHRNA7*, exon 1). For two of these genes (*CHRNA4* and *CHRNB2*), we supplemented next-generation sequencing with Sanger sequencing protocols, which increased the number of rare and common SNPs available for analysis at these genes. The *CHRNA7* region contains a partial duplication ([Gault et al, 1998](#bib29)), which may have contributed to reduced resequencing coverage of this gene.

Power analyses to discover *de novo* rare variant associations from sequencing efforts have cited sample sizes from 100 to 10 000 depending on a number of factors, including subject sampling criteria, effect sizes, variant frequency, strength of purifying selection, and the probability of detecting variants ([Kryukov et al, 2009](#bib39); [Li and Leal, 2008](#bib43)). Including this study, four groups have resequenced 3–10 nAChR genes using 80–675 individuals identifying 55–262 SNPs ([Rana et al, 2009](#bib65); [Sabatelli et al, 2009](#bib67); [Weiss et al, 2008](#bib88)). Each of the four studies identified previously unreported variants; however, the power to identify associations with rare variants is limited by the lower sample sizes suggesting the need for larger sample sizes and more efficient approaches toward identifying rare, potentially functional variants. Small sample sizes leading to underpowered candidate gene studies have been an often-cited reason for inconsistent results across studies ([Ioannidis, 2008](#bib37)). In our sample of 430 individuals with a mean (SD) FTND score of 5.2 (2.1), the power of single SNP tests (double sided with *α*=0.05) to detect *β*-values of 0.5, 1.0, 1.5, and 2.0 with a MAF=0.01 and a dominant model is 11, 28, 54, and 78%, whereas the *post hoc* power of the linear regression approach to detect the effect of the common *CHRNB2* variants significantly associated with FTND score is 70%.

In conclusion, there are numerous previous GWASs and candidate gene studies supporting the hypothesis that common variation in several nAChR subunit genes contributes to variation in nicotine dependence as measured by the FTND or by measures of smoking intensity. In our resequencing study of DNA from a sample of treatment-seeking smokers, we found a significant evidence for the contribution of common variants at *CHRNB2* and rare variants at *CHRNA4* to variation in a measure of nicotine dependence. We note that in mouse knockout studies, the *β*_2_ and *α*_4_ nicotinic receptor subunit genes have been shown to be necessary and sufficient to observe the reinforcing properties of nicotine ([Mineur and Picciotto, 2008](#bib57)), and that the human *α*_4_*β*_2_ nAChR is upregulated upon nicotine administration ([Buisson and Bertrand, 2001](#bib11)), providing support for the involvement of these two nAChR subunits in the etiology of nicotine dependence. The heteropentameric structure of the nAChR suggests that future gene–gene interaction analyses of nAChR subunit variants will be a useful approach to identify nAChR sequence variation influencing smoking behavior. Analysis of both common and rare variants in the COMPASS samples suggests that the combination of common and rare variants at *CHRNB2*, *CHRNA5*, and at the chr15q25.1 nAChR locus, also contribute to nicotine dependence. The chr15q25.1 and *CHRNA5* findings are concordant with multiple previous association results in both GWASs and candidate gene studies, whereas the *CHRNB2* finding may reflect the common SNP association findings already observed in this sample with SNP-wise analysis. Resequencing of DNA from nAChR subunit genes targeting coding and regulatory regions, using larger samples sizes and multiple approaches to statistical genetic analysis, including analyses of interaction and synthetic association hypotheses, will be necessary to further characterize the relationship between common and rare nAChR subunit gene sequence variation and smoking-related behaviors in treatment-seeking participants of smoking cessation trials.

## Acknowledgments

This research was supported by R01 CA071358, U01 DA020830, and a Material Transfer and Research Agreement between SRI International and Perlegen Sciences. We thank the participants of the COMPASS smoking cessation clinical trial (CA071358, PI Gary Swan). We thank David V Conti for helpful comments, and we acknowledge the efforts of Ondrej Libiger at Scripps Genomic Medicine, Susan Zbikowski and Tim McAfee of Free & Clear, Julie Richards and the staff of the Group Health Research Institute's Survey Research Program, and Martha Agreda, Mary McElroy, Gaye Courtney, Peggy Giacalone, and Trish McLeod of SRI International. We thank the three anonymous reviewers for their thoughtful and detailed comments and suggestions.

*Related to this work*. The COMPASS study was funded by the National Cancer Institute grant R01 CA071358 and is registered at [Clinicaltrials.gov](http://Clinicaltrials.gov) ([NCT00301145](https://clinicaltrials.gov/ct2/show/NCT00301145)). Varenicline and nominal support for recruiting COMPASS participants was provided by Pfizer. Neither entity had any role in the COMPASS study design or in the collection, analysis, and interpretation of COMPASS data. Authors Wessel, McDonald, Hinds, Stokowski, Javitz, Kennemer, Krasnow, Dirks, Hardin, Pitts, Michel, Jack, Ballinger, McClure, Swan, and Bergen declare that all financial and material support for this work was provided by their primary employer. *Related to all other compensation*. Authors Wessel, McDonald, Stokowski, Javitz, Krasnow, Dirks, Hardin, Michel, Jack, and McClure declare that, except for income provided from their primary employer, no financial support or compensation has been received from any individual or corporate entity over the past 3 years for research or professional services and there are no personal financial holdings that could be perceived as constituting a potential conflict of interest. Dr Hinds is currently employed by 23 and Me, Dr Kennemer is currently employed by Complete Genetics as Genomics Applications Analyst, Dr Pitts is currently employed by Pfizer, in its Applied Quantitative Genotherapeutics group, and Dr Ballinger is currently employed by Complete Genomics as Vice President of Genomics. Dr Swan received financial support from Pfizer to attend a 1-day advisory meeting in 2008. In the past 3 years, Dr Bergen has received compensation for professional services from the National Institutes of Health and the National Coalition for Health Professional Education in Genetics.

## Footnotes

## Supplementary Material

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Altschul SF, Gish W, Miller W, Myers EW, Lipman DJ. Basic local alignment search tool. J Mol Biol. 1990;215:403–410. doi: 10.1016/S0022-2836(05)80360-2.  [DOI](https://doi.org/10.1016/S0022-2836(05)80360-2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/2231712/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Mol%20Biol&title=Basic%20local%20alignment%20search%20tool&author=SF%20Altschul&author=W%20Gish&author=W%20Miller&author=EW%20Myers&author=DJ%20Lipman&volume=215&publication_year=1990&pages=403-410&pmid=2231712&doi=10.1016/S0022-2836(05)80360-2&)

2. Baker TB, Weiss RB, Bolt D, von Niederhausern A, Fiore MC, Dunn DM, et al. Human neuronal acetylcholine receptor A5-A3-B4 haplotypes are associated with multiple nicotine dependence phenotypes. Nicotine Tob Res. 2009;11:785–796. doi: 10.1093/ntr/ntp064.  [DOI](https://doi.org/10.1093/ntr/ntp064) | [PMC free article](/articles/PMC2699926/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19436041/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nicotine%20Tob%20Res&title=Human%20neuronal%20acetylcholine%20receptor%20A5-A3-B4%20haplotypes%20are%20associated%20with%20multiple%20nicotine%20dependence%20phenotypes&author=TB%20Baker&author=RB%20Weiss&author=D%20Bolt&author=A%20von%20Niederhausern&author=MC%20Fiore&volume=11&publication_year=2009&pages=785-796&pmid=19436041&doi=10.1093/ntr/ntp064&)

3. Barrett JC, Fry B, Maller J, Daly MJ. Haploview: analysis and visualization of LD and haplotype maps. Bioinformatics. 2005;21:263–265. doi: 10.1093/bioinformatics/bth457.  [DOI](https://doi.org/10.1093/bioinformatics/bth457) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15297300/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=Haploview:%20analysis%20and%20visualization%20of%20LD%20and%20haplotype%20maps&author=JC%20Barrett&author=B%20Fry&author=J%20Maller&author=MJ%20Daly&volume=21&publication_year=2005&pages=263-265&pmid=15297300&doi=10.1093/bioinformatics/bth457&)

4. Bergen AW, Caporaso N. Cigarette smoking. J Natl Cancer Inst. 1999;91:1365–1375. doi: 10.1093/jnci/91.16.1365.  [DOI](https://doi.org/10.1093/jnci/91.16.1365) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10451441/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Natl%20Cancer%20Inst&title=Cigarette%20smoking&author=AW%20Bergen&author=N%20Caporaso&volume=91&publication_year=1999&pages=1365-1375&pmid=10451441&doi=10.1093/jnci/91.16.1365&)

5. Bergen AW, Conti DV, Van Den Berg D, Lee W, Liu J, Li D, et al. Dopamine genes and nicotine dependence in treatment-seeking and community smokers. Neuropsychopharmacology. 2009;34:2252–2264. doi: 10.1038/npp.2009.52.  [DOI](https://doi.org/10.1038/npp.2009.52) | [PMC free article](/articles/PMC3558036/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19494806/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neuropsychopharmacology&title=Dopamine%20genes%20and%20nicotine%20dependence%20in%20treatment-seeking%20and%20community%20smokers&author=AW%20Bergen&author=DV%20Conti&author=D%20Van%20Den%20Berg&author=W%20Lee&author=J%20Liu&volume=34&publication_year=2009&pages=2252-2264&pmid=19494806&doi=10.1038/npp.2009.52&)

6. Bierut LJ, Stitzel JA, Wang JC, Hinrichs AL, Grucza RA, Xuei X, et al. Variants in nicotinic receptors and risk for nicotine dependence. Am J Psychiatry. 2008;165:1163–1171. doi: 10.1176/appi.ajp.2008.07111711.  [DOI](https://doi.org/10.1176/appi.ajp.2008.07111711) | [PMC free article](/articles/PMC2574742/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18519524/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Psychiatry&title=Variants%20in%20nicotinic%20receptors%20and%20risk%20for%20nicotine%20dependence&author=LJ%20Bierut&author=JA%20Stitzel&author=JC%20Wang&author=AL%20Hinrichs&author=RA%20Grucza&volume=165&publication_year=2008&pages=1163-1171&pmid=18519524&doi=10.1176/appi.ajp.2008.07111711&)

7. Bodmer W, Bonilla C. Common and rare variants in multifactorial susceptibility to common diseases. Nat Genet. 2008;40:695–701. doi: 10.1038/ng.f.136.  [DOI](https://doi.org/10.1038/ng.f.136) | [PMC free article](/articles/PMC2527050/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18509313/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Common%20and%20rare%20variants%20in%20multifactorial%20susceptibility%20to%20common%20diseases&author=W%20Bodmer&author=C%20Bonilla&volume=40&publication_year=2008&pages=695-701&pmid=18509313&doi=10.1038/ng.f.136&)

8. Breitling LP, Dahmen N, Mittelstrass K, Illig T, Rujescu D, Raum E, et al. Smoking cessation and variations in nicotinic acetylcholine receptor subunits alpha-5, alpha-3, and beta-4 genes. Biol Psychiatry. 2009a;65:691–695. doi: 10.1016/j.biopsych.2008.10.004.  [DOI](https://doi.org/10.1016/j.biopsych.2008.10.004) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18996504/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biol%20Psychiatry&title=Smoking%20cessation%20and%20variations%20in%20nicotinic%20acetylcholine%20receptor%20subunits%20alpha-5,%20alpha-3,%20and%20beta-4%20genes&author=LP%20Breitling&author=N%20Dahmen&author=K%20Mittelstrass&author=T%20Illig&author=D%20Rujescu&volume=65&publication_year=2009a&pages=691-695&pmid=18996504&doi=10.1016/j.biopsych.2008.10.004&)

9. Breitling LP, Dahmen N, Mittelstrass K, Rujescu D, Gallinat J, Fehr C, et al. Association of nicotinic acetylcholine receptor subunit alpha 4 polymorphisms with nicotine dependence in 5500 Germans. Pharmacogenomics J. 2009b;9:219–224. doi: 10.1038/tpj.2009.6.  [DOI](https://doi.org/10.1038/tpj.2009.6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19290018/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=Association%20of%20nicotinic%20acetylcholine%20receptor%20subunit%20alpha%204%20polymorphisms%20with%20nicotine%20dependence%20in%205500%20Germans&author=LP%20Breitling&author=N%20Dahmen&author=K%20Mittelstrass&author=D%20Rujescu&author=J%20Gallinat&volume=9&publication_year=2009b&pages=219-224&pmid=19290018&doi=10.1038/tpj.2009.6&)

10. Broms U, Madden PA, Heath AC, Pergadia ML, Shiffman S, Kaprio J. The Nicotine Dependence Syndrome Scale in Finnish smokers. Drug Alcohol Depend. 2007;89:42–51. doi: 10.1016/j.drugalcdep.2006.11.017.  [DOI](https://doi.org/10.1016/j.drugalcdep.2006.11.017) | [PMC free article](/articles/PMC1950147/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17174039/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Alcohol%20Depend&title=The%20Nicotine%20Dependence%20Syndrome%20Scale%20in%20Finnish%20smokers&author=U%20Broms&author=PA%20Madden&author=AC%20Heath&author=ML%20Pergadia&author=S%20Shiffman&volume=89&publication_year=2007&pages=42-51&pmid=17174039&doi=10.1016/j.drugalcdep.2006.11.017&)

11. Buisson B, Bertrand D. Chronic exposure to nicotine upregulates the human (alpha)4((beta)2 nicotinic acetylcholine receptor function. J Neurosci. 2001;21:1819–1829. doi: 10.1523/JNEUROSCI.21-06-01819.2001.  [DOI](https://doi.org/10.1523/JNEUROSCI.21-06-01819.2001) | [PMC free article](/articles/PMC6762627/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11245666/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Neurosci&title=Chronic%20exposure%20to%20nicotine%20upregulates%20the%20human%20(alpha)4((beta)2%20nicotinic%20acetylcholine%20receptor%20function&author=B%20Buisson&author=D%20Bertrand&volume=21&publication_year=2001&pages=1819-1829&pmid=11245666&doi=10.1523/JNEUROSCI.21-06-01819.2001&)

12. Changeux JP. Nicotinic receptors and nicotine addiction. C R Biol. 2009;332:421–425. doi: 10.1016/j.crvi.2009.02.005.  [DOI](https://doi.org/10.1016/j.crvi.2009.02.005) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19393973/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=C%20R%20Biol&title=Nicotinic%20receptors%20and%20nicotine%20addiction&author=JP%20Changeux&volume=332&publication_year=2009&pages=421-425&pmid=19393973&doi=10.1016/j.crvi.2009.02.005&)

13. Chen GB, Payne TJ, Lou XY, Ma JZ, Zhu J, Li MD. Association of amyloid precursor protein-binding protein, family B, member 1 with nicotine dependence in African and European American smokers. Hum Genet. 2008;124:393–398. doi: 10.1007/s00439-008-0558-9.  [DOI](https://doi.org/10.1007/s00439-008-0558-9) | [PMC free article](/articles/PMC5576171/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18777128/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Genet&title=Association%20of%20amyloid%20precursor%20protein-binding%20protein,%20family%20B,%20member%201%20with%20nicotine%20dependence%20in%20African%20and%20European%20American%20smokers&author=GB%20Chen&author=TJ%20Payne&author=XY%20Lou&author=JZ%20Ma&author=J%20Zhu&volume=124&publication_year=2008&pages=393-398&pmid=18777128&doi=10.1007/s00439-008-0558-9&)

14. Chen X, Chen J, Williamson VS, An SS, Hettema JM, Aggen SH, et al. Variants in nicotinic acetylcholine receptors alpha5 and alpha3 increase risks to nicotine dependence. Am J Med Genet B Neuropsychiatr Genet. 2009;150B:926–933. doi: 10.1002/ajmg.b.30919.  [DOI](https://doi.org/10.1002/ajmg.b.30919) | [PMC free article](/articles/PMC3081884/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19132693/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Med%20Genet%20B%20Neuropsychiatr%20Genet&title=Variants%20in%20nicotinic%20acetylcholine%20receptors%20alpha5%20and%20alpha3%20increase%20risks%20to%20nicotine%20dependence&author=X%20Chen&author=J%20Chen&author=VS%20Williamson&author=SS%20An&author=JM%20Hettema&volume=150B&publication_year=2009&pages=926-933&pmid=19132693&doi=10.1002/ajmg.b.30919&)

15. Conneely KN, Boehnke M. So many correlated tests, so little time! rapid adjustment of P values for multiple correlated tests. Am J Hum Genet. 2007;81:1158–1168. doi: 10.1086/522036.  [DOI](https://doi.org/10.1086/522036) | [PMC free article](/articles/PMC2276357/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17966093/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=So%20many%20correlated%20tests,%20so%20little%20time!%20rapid%20adjustment%20of%20P%20values%20for%20multiple%20correlated%20tests&author=KN%20Conneely&author=M%20Boehnke&volume=81&publication_year=2007&pages=1158-1168&pmid=17966093&doi=10.1086/522036&)

16. Conti DV, Lee W, Li D, Liu J, Van Den Berg D, Thomas PD, et al. Nicotinic acetylcholine receptor beta2 subunit gene implicated in a systems-based candidate gene study of smoking cessation. Hum Mol Genet. 2008;17:2834–2848. doi: 10.1093/hmg/ddn181.  [DOI](https://doi.org/10.1093/hmg/ddn181) | [PMC free article](/articles/PMC2525499/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18593715/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mol%20Genet&title=Nicotinic%20acetylcholine%20receptor%20beta2%20subunit%20gene%20implicated%20in%20a%20systems-based%20candidate%20gene%20study%20of%20smoking%20cessation&author=DV%20Conti&author=W%20Lee&author=D%20Li&author=J%20Liu&author=D%20Van%20Den%20Berg&volume=17&publication_year=2008&pages=2834-2848&pmid=18593715&doi=10.1093/hmg/ddn181&)

17. Craig DW, Pearson JV, Szelinger S, Sekar A, Redman M, Corneveaux JJ, et al. Identification of genetic variants using bar-coded multiplexed sequencing. Nat Methods. 2008;5:887–893. doi: 10.1038/nmeth.1251.  [DOI](https://doi.org/10.1038/nmeth.1251) | [PMC free article](/articles/PMC3171277/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18794863/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Methods&title=Identification%20of%20genetic%20variants%20using%20bar-coded%20multiplexed%20sequencing&author=DW%20Craig&author=JV%20Pearson&author=S%20Szelinger&author=A%20Sekar&author=M%20Redman&volume=5&publication_year=2008&pages=887-893&pmid=18794863&doi=10.1038/nmeth.1251&)

18. Dani JA, Bertrand D. Nicotinic acetylcholine receptors and nicotinic cholinergic mechanisms of the central nervous system. Annu Rev Pharmacol Toxicol. 2007;47:699–729. doi: 10.1146/annurev.pharmtox.47.120505.105214.  [DOI](https://doi.org/10.1146/annurev.pharmtox.47.120505.105214) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17009926/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Annu%20Rev%20Pharmacol%20Toxicol&title=Nicotinic%20acetylcholine%20receptors%20and%20nicotinic%20cholinergic%20mechanisms%20of%20the%20central%20nervous%20system&author=JA%20Dani&author=D%20Bertrand&volume=47&publication_year=2007&pages=699-729&pmid=17009926&doi=10.1146/annurev.pharmtox.47.120505.105214&)

19. Dickson SP, Wang K, Krantz I, Hakonarson H, Goldstein DB. Rare variants create synthetic genome-wide associations. PLoS Biol. 2010;8:e1000294. doi: 10.1371/journal.pbio.1000294.  [DOI](https://doi.org/10.1371/journal.pbio.1000294) | [PMC free article](/articles/PMC2811148/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20126254/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Biol&title=Rare%20variants%20create%20synthetic%20genome-wide%20associations&author=SP%20Dickson&author=K%20Wang&author=I%20Krantz&author=H%20Hakonarson&author=DB%20Goldstein&volume=8&publication_year=2010&pages=e1000294&pmid=20126254&doi=10.1371/journal.pbio.1000294&)

20. Drgon T, Johnson C, Walther D, Albino AP, Rose JE, Uhl GR. Genome-wide association for smoking cessation success: participants in a trial with adjunctive denicotinized cigarettes. Mol Med. 2009a;15:268–274. doi: 10.2119/molmed.2009.00040.  [DOI](https://doi.org/10.2119/molmed.2009.00040) | [PMC free article](/articles/PMC2707518/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19593411/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Med&title=Genome-wide%20association%20for%20smoking%20cessation%20success:%20participants%20in%20a%20trial%20with%20adjunctive%20denicotinized%20cigarettes&author=T%20Drgon&author=C%20Johnson&author=D%20Walther&author=AP%20Albino&author=JE%20Rose&volume=15&publication_year=2009a&pages=268-274&pmid=19593411&doi=10.2119/molmed.2009.00040&)

21. Drgon T, Montoya I, Johnson C, Liu QR, Walther D, Hamer D, et al. Genome-wide association for nicotine dependence and smoking cessation success in NIH research volunteers. Mol Med. 2009b;15:21–27. doi: 10.2119/molmed.2008.00096.  [DOI](https://doi.org/10.2119/molmed.2008.00096) | [PMC free article](/articles/PMC2581605/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19009022/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Med&title=Genome-wide%20association%20for%20nicotine%20dependence%20and%20smoking%20cessation%20success%20in%20NIH%20research%20volunteers&author=T%20Drgon&author=I%20Montoya&author=C%20Johnson&author=QR%20Liu&author=D%20Walther&volume=15&publication_year=2009b&pages=21-27&pmid=19009022&doi=10.2119/molmed.2008.00096&)

22. Ehringer MA, Clegg HV, Collins AC, Corley RP, Crowley T, Hewitt JK, et al. Association of the neuronal nicotinic receptor beta2 subunit gene (CHRNB2) with subjective responses to alcohol and nicotine. Am J Med Genet B Neuropsychiatr Genet. 2007;144B:596–604. doi: 10.1002/ajmg.b.30464.  [DOI](https://doi.org/10.1002/ajmg.b.30464) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17226798/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Med%20Genet%20B%20Neuropsychiatr%20Genet&title=Association%20of%20the%20neuronal%20nicotinic%20receptor%20beta2%20subunit%20gene%20(CHRNB2)%20with%20subjective%20responses%20to%20alcohol%20and%20nicotine&author=MA%20Ehringer&author=HV%20Clegg&author=AC%20Collins&author=RP%20Corley&author=T%20Crowley&volume=144B&publication_year=2007&pages=596-604&pmid=17226798&doi=10.1002/ajmg.b.30464&)

23. Etter JF, Hoda JC, Perroud N, Munafo M, Buresi C, Duret C, et al. Association of genes coding for the alpha-4, alpha-5, beta-2 and beta-3 subunits of nicotinic receptors with cigarette smoking and nicotine dependence. Addict Behav. 2009;34:772–775. doi: 10.1016/j.addbeh.2009.05.010.  [DOI](https://doi.org/10.1016/j.addbeh.2009.05.010) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19482438/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Addict%20Behav&title=Association%20of%20genes%20coding%20for%20the%20alpha-4,%20alpha-5,%20beta-2%20and%20beta-3%20subunits%20of%20nicotinic%20receptors%20with%20cigarette%20smoking%20and%20nicotine%20dependence&author=JF%20Etter&author=JC%20Hoda&author=N%20Perroud&author=M%20Munafo&author=C%20Buresi&volume=34&publication_year=2009&pages=772-775&pmid=19482438&doi=10.1016/j.addbeh.2009.05.010&)

24. Fagerstrom K, Furberg H. A comparison of the Fagerstrom test for nicotine dependence and smoking prevalence across countries. Addiction. 2008;103:841–845. doi: 10.1111/j.1360-0443.2008.02190.x.  [DOI](https://doi.org/10.1111/j.1360-0443.2008.02190.x) | [PMC free article](/articles/PMC2914535/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18412764/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Addiction&title=A%20comparison%20of%20the%20Fagerstrom%20test%20for%20nicotine%20dependence%20and%20smoking%20prevalence%20across%20countries&author=K%20Fagerstrom&author=H%20Furberg&volume=103&publication_year=2008&pages=841-845&pmid=18412764&doi=10.1111/j.1360-0443.2008.02190.x&)

25. Feng Y, Niu T, Xing H, Xu X, Chen C, Peng S, et al. A common haplotype of the nicotine acetylcholine receptor alpha 4 subunit gene is associated with vulnerability to nicotine addiction in men. Am J Hum Genet. 2004;75:112–121. doi: 10.1086/422194.  [DOI](https://doi.org/10.1086/422194) | [PMC free article](/articles/PMC1181994/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15154117/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=A%20common%20haplotype%20of%20the%20nicotine%20acetylcholine%20receptor%20alpha%204%20subunit%20gene%20is%20associated%20with%20vulnerability%20to%20nicotine%20addiction%20in%20men&author=Y%20Feng&author=T%20Niu&author=H%20Xing&author=X%20Xu&author=C%20Chen&volume=75&publication_year=2004&pages=112-121&pmid=15154117&doi=10.1086/422194&)

26. Frazer KA, Murray SS, Schork NJ, Topol EJ. Human genetic variation and its contribution to complex traits. Nat Rev Genet. 2009;10:241–251. doi: 10.1038/nrg2554.  [DOI](https://doi.org/10.1038/nrg2554) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19293820/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Genet&title=Human%20genetic%20variation%20and%20its%20contribution%20to%20complex%20traits&author=KA%20Frazer&author=SS%20Murray&author=NJ%20Schork&author=EJ%20Topol&volume=10&publication_year=2009&pages=241-251&pmid=19293820&doi=10.1038/nrg2554&)

27. Freathy RM, Ring SM, Shields B, Galobardes B, Knight B, Weedon MN, et al. A common genetic variant in the 15q24 nicotinic acetylcholine receptor gene cluster (CHRNA5-CHRNA3-CHRNB4) is associated with a reduced ability of women to quit smoking in pregnancy. Hum Mol Genet. 2009;18:2922–2927. doi: 10.1093/hmg/ddp216.  [DOI](https://doi.org/10.1093/hmg/ddp216) | [PMC free article](/articles/PMC2706684/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19429911/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mol%20Genet&title=A%20common%20genetic%20variant%20in%20the%2015q24%20nicotinic%20acetylcholine%20receptor%20gene%20cluster%20(CHRNA5-CHRNA3-CHRNB4)%20is%20associated%20with%20a%20reduced%20ability%20of%20women%20to%20quit%20smoking%20in%20pregnancy&author=RM%20Freathy&author=SM%20Ring&author=B%20Shields&author=B%20Galobardes&author=B%20Knight&volume=18&publication_year=2009&pages=2922-2927&pmid=19429911&doi=10.1093/hmg/ddp216&)

28. Gauderman WJ. Sample size requirements for matched case-control studies of gene-environment interaction. Stat Med. 2002;21:35–50. doi: 10.1002/sim.973.  [DOI](https://doi.org/10.1002/sim.973) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11782049/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Stat%20Med&title=Sample%20size%20requirements%20for%20matched%20case-control%20studies%20of%20gene-environment%20interaction&author=WJ%20Gauderman&volume=21&publication_year=2002&pages=35-50&pmid=11782049&doi=10.1002/sim.973&)

29. Gault J, Robinson M, Berger R, Drebing C, Logel J, Hopkins J, et al. Genomic organization and partial duplication of the human alpha7 neuronal nicotinic acetylcholine receptor gene (CHRNA7) Genomics. 1998;52:173–185. doi: 10.1006/geno.1998.5363.  [DOI](https://doi.org/10.1006/geno.1998.5363) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9782083/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genomics&title=Genomic%20organization%20and%20partial%20duplication%20of%20the%20human%20alpha7%20neuronal%20nicotinic%20acetylcholine%20receptor%20gene%20(CHRNA7)&author=J%20Gault&author=M%20Robinson&author=R%20Berger&author=C%20Drebing&author=J%20Logel&volume=52&publication_year=1998&pages=173-185&pmid=9782083&doi=10.1006/geno.1998.5363&)

30. Greenbaum L, Kanyas K, Karni O, Merbl Y, Olender T, Horowitz A, et al. 2006Why do young women smoke? I. Direct and interactive effects of environment, psychological characteristics and nicotinic cholinergic receptor genes Mol Psychiatry 11312–322.223.  [DOI](https://doi.org/10.1038/sj.mp.4001774) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16314871/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Psychiatry&title=Why%20do%20young%20women%20smoke?%20I.%20Direct%20and%20interactive%20effects%20of%20environment,%20psychological%20characteristics%20and%20nicotinic%20cholinergic%20receptor%20genes&author=L%20Greenbaum&author=K%20Kanyas&author=O%20Karni&author=Y%20Merbl&author=T%20Olender&volume=11&publication_year=2006&pages=312-322&pmid=16314871&doi=10.1038/sj.mp.4001774&)

31. Harismendy O, Ng PC, Strausberg RL, Wang X, Stockwell TB, Beeson KY, et al. Evaluation of next generation sequencing platforms for population targeted sequencing studies. Genome Biol. 2009;10:R32. doi: 10.1186/gb-2009-10-3-r32.  [DOI](https://doi.org/10.1186/gb-2009-10-3-r32) | [PMC free article](/articles/PMC2691003/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19327155/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Biol&title=Evaluation%20of%20next%20generation%20sequencing%20platforms%20for%20population%20targeted%20sequencing%20studies&author=O%20Harismendy&author=PC%20Ng&author=RL%20Strausberg&author=X%20Wang&author=TB%20Stockwell&volume=10&publication_year=2009&pages=R32&pmid=19327155&doi=10.1186/gb-2009-10-3-r32&)

32. Heatherton TF, Kozlowski LT, Frecker RC, Fagerstrom KO. The Fagerstrom test for nicotine dependence: a revision of the Fagerstrom Tolerance Questionnaire. Br J Addict. 1991;86:1119–1127. doi: 10.1111/j.1360-0443.1991.tb01879.x.  [DOI](https://doi.org/10.1111/j.1360-0443.1991.tb01879.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/1932883/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Addict&title=The%20Fagerstrom%20test%20for%20nicotine%20dependence:%20a%20revision%20of%20the%20Fagerstrom%20Tolerance%20Questionnaire&author=TF%20Heatherton&author=LT%20Kozlowski&author=RC%20Frecker&author=KO%20Fagerstrom&volume=86&publication_year=1991&pages=1119-1127&pmid=1932883&doi=10.1111/j.1360-0443.1991.tb01879.x&)

33. Herraez DL, Stoneking M. High fractions of exogenous DNA in human buccal samples reduce the quality of large-scale genotyping. Anal Biochem. 2008;383:329–331. doi: 10.1016/j.ab.2008.08.015.  [DOI](https://doi.org/10.1016/j.ab.2008.08.015) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18804445/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Anal%20Biochem&title=High%20fractions%20of%20exogenous%20DNA%20in%20human%20buccal%20samples%20reduce%20the%20quality%20of%20large-scale%20genotyping&author=DL%20Herraez&author=M%20Stoneking&volume=383&publication_year=2008&pages=329-331&pmid=18804445&doi=10.1016/j.ab.2008.08.015&)

34. Hinds DA, Stokowski RP, Patil N, Konvicka K, Kershenobich D, Cox DR, et al. Matching strategies for genetic association studies in structured populations. Am J Hum Genet. 2004;74:317–325. doi: 10.1086/381716.  [DOI](https://doi.org/10.1086/381716) | [PMC free article](/articles/PMC1181929/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14740319/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=Matching%20strategies%20for%20genetic%20association%20studies%20in%20structured%20populations&author=DA%20Hinds&author=RP%20Stokowski&author=N%20Patil&author=K%20Konvicka&author=D%20Kershenobich&volume=74&publication_year=2004&pages=317-325&pmid=14740319&doi=10.1086/381716&)

35. Hoft NR, Corley RP, McQueen MB, Schlaepfer IR, Huizinga D, Ehringer MA. Genetic association of the CHRNA6 and CHRNB3 genes with tobacco dependence in a nationally representative sample. Neuropsychopharmacology. 2009;34:698–706. doi: 10.1038/npp.2008.122.  [DOI](https://doi.org/10.1038/npp.2008.122) | [PMC free article](/articles/PMC2915837/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18704094/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neuropsychopharmacology&title=Genetic%20association%20of%20the%20CHRNA6%20and%20CHRNB3%20genes%20with%20tobacco%20dependence%20in%20a%20nationally%20representative%20sample&author=NR%20Hoft&author=RP%20Corley&author=MB%20McQueen&author=IR%20Schlaepfer&author=D%20Huizinga&volume=34&publication_year=2009&pages=698-706&pmid=18704094&doi=10.1038/npp.2008.122&)

36. Hutchison KE, Allen DL, Filbey FM, Jepson C, Lerman C, Benowitz NL, et al. CHRNA4 and tobacco dependence: from gene regulation to treatment outcome. Arch Gen Psychiatry. 2007;64:1078–1086. doi: 10.1001/archpsyc.64.9.1078.  [DOI](https://doi.org/10.1001/archpsyc.64.9.1078) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17768273/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arch%20Gen%20Psychiatry&title=CHRNA4%20and%20tobacco%20dependence:%20from%20gene%20regulation%20to%20treatment%20outcome&author=KE%20Hutchison&author=DL%20Allen&author=FM%20Filbey&author=C%20Jepson&author=C%20Lerman&volume=64&publication_year=2007&pages=1078-1086&pmid=17768273&doi=10.1001/archpsyc.64.9.1078&)

37. Ioannidis JP. Why most discovered true associations are inflated. Epidemiology. 2008;19:640–648. doi: 10.1097/EDE.0b013e31818131e7.  [DOI](https://doi.org/10.1097/EDE.0b013e31818131e7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18633328/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Epidemiology&title=Why%20most%20discovered%20true%20associations%20are%20inflated&author=JP%20Ioannidis&volume=19&publication_year=2008&pages=640-648&pmid=18633328&doi=10.1097/EDE.0b013e31818131e7&)

38. Keskitalo K, Broms U, Heliovaara M, Ripatti S, Surakka I, Perola M, et al. Association of serum cotinine level with a cluster of three nicotinic acetylcholine receptor genes (CHRNA3/CHRNA5/CHRNB4) on chromosome 15. Hum Mol Genet. 2009;18:4007–4012. doi: 10.1093/hmg/ddp322.  [DOI](https://doi.org/10.1093/hmg/ddp322) | [PMC free article](/articles/PMC2748889/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19628476/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mol%20Genet&title=Association%20of%20serum%20cotinine%20level%20with%20a%20cluster%20of%20three%20nicotinic%20acetylcholine%20receptor%20genes%20(CHRNA3/CHRNA5/CHRNB4)%20on%20chromosome%2015&author=K%20Keskitalo&author=U%20Broms&author=M%20Heliovaara&author=S%20Ripatti&author=I%20Surakka&volume=18&publication_year=2009&pages=4007-4012&pmid=19628476&doi=10.1093/hmg/ddp322&)

39. Kryukov GV, Shpunt A, Stamatoyannopoulos JA, Sunyaev SR. Power of deep, all-exon resequencing for discovery of human trait genes. Proc Natl Acad Sci USA. 2009;106:3871–3876. doi: 10.1073/pnas.0812824106.  [DOI](https://doi.org/10.1073/pnas.0812824106) | [PMC free article](/articles/PMC2656172/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19202052/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20USA&title=Power%20of%20deep,%20all-exon%20resequencing%20for%20discovery%20of%20human%20trait%20genes&author=GV%20Kryukov&author=A%20Shpunt&author=JA%20Stamatoyannopoulos&author=SR%20Sunyaev&volume=106&publication_year=2009&pages=3871-3876&pmid=19202052&doi=10.1073/pnas.0812824106&)

40. Le Marchand L, Derby KS, Murphy SE, Hecht SS, Hatsukami D, Carmella SG, et al. Smokers with the CHRNA lung cancer-associated variants are exposed to higher levels of nicotine equivalents and a carcinogenic tobacco-specific nitrosamine. Cancer Res. 2008;68:9137–9140. doi: 10.1158/0008-5472.CAN-08-2271.  [DOI](https://doi.org/10.1158/0008-5472.CAN-08-2271) | [PMC free article](/articles/PMC2587068/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19010884/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=Smokers%20with%20the%20CHRNA%20lung%20cancer-associated%20variants%20are%20exposed%20to%20higher%20levels%20of%20nicotine%20equivalents%20and%20a%20carcinogenic%20tobacco-specific%20nitrosamine&author=L%20Le%20Marchand&author=KS%20Derby&author=SE%20Murphy&author=SS%20Hecht&author=D%20Hatsukami&volume=68&publication_year=2008&pages=9137-9140&pmid=19010884&doi=10.1158/0008-5472.CAN-08-2271&)

41. Lerman C, Jepson C, Wileyto EP, Epstein LH, Rukstalis M, Patterson F, et al. Role of functional genetic variation in the dopamine D2 receptor (DRD2) in response to bupropion and nicotine replacement therapy for tobacco dependence: results of two randomized clinical trials. Neuropsychopharmacology. 2006;31:231–242. doi: 10.1038/sj.npp.1300861.  [DOI](https://doi.org/10.1038/sj.npp.1300861) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16123753/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neuropsychopharmacology&title=Role%20of%20functional%20genetic%20variation%20in%20the%20dopamine%20D2%20receptor%20(DRD2)%20in%20response%20to%20bupropion%20and%20nicotine%20replacement%20therapy%20for%20tobacco%20dependence:%20results%20of%20two%20randomized%20clinical%20trials&author=C%20Lerman&author=C%20Jepson&author=EP%20Wileyto&author=LH%20Epstein&author=M%20Rukstalis&volume=31&publication_year=2006&pages=231-242&pmid=16123753&doi=10.1038/sj.npp.1300861&)

42. Lessov CN, Martin NG, Statham DJ, Todorov AA, Slutske WS, Bucholz KK, et al. Defining nicotine dependence for genetic research: evidence from Australian twins. Psychol Med. 2004;34:865–879. doi: 10.1017/s0033291703001582.  [DOI](https://doi.org/10.1017/s0033291703001582) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15500307/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Psychol%20Med&title=Defining%20nicotine%20dependence%20for%20genetic%20research:%20evidence%20from%20Australian%20twins&author=CN%20Lessov&author=NG%20Martin&author=DJ%20Statham&author=AA%20Todorov&author=WS%20Slutske&volume=34&publication_year=2004&pages=865-879&pmid=15500307&doi=10.1017/s0033291703001582&)

43. Li B, Leal SM. Methods for detecting associations with rare variants for common diseases: application to analysis of sequence data. Am J Hum Genet. 2008;83:311–321. doi: 10.1016/j.ajhg.2008.06.024.  [DOI](https://doi.org/10.1016/j.ajhg.2008.06.024) | [PMC free article](/articles/PMC2842185/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18691683/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=Methods%20for%20detecting%20associations%20with%20rare%20variants%20for%20common%20diseases:%20application%20to%20analysis%20of%20sequence%20data&author=B%20Li&author=SM%20Leal&volume=83&publication_year=2008&pages=311-321&pmid=18691683&doi=10.1016/j.ajhg.2008.06.024&)

44. Li MD, Beuten J, Ma JZ, Payne TJ, Lou XY, Garcia V, et al. Ethnic- and gender-specific association of the nicotinic acetylcholine receptor alpha4 subunit gene (CHRNA4) with nicotine dependence. Hum Mol Genet. 2005;14:1211–1219. doi: 10.1093/hmg/ddi132.  [DOI](https://doi.org/10.1093/hmg/ddi132) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15790597/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mol%20Genet&title=Ethnic-%20and%20gender-specific%20association%20of%20the%20nicotinic%20acetylcholine%20receptor%20alpha4%20subunit%20gene%20(CHRNA4)%20with%20nicotine%20dependence&author=MD%20Li&author=J%20Beuten&author=JZ%20Ma&author=TJ%20Payne&author=XY%20Lou&volume=14&publication_year=2005&pages=1211-1219&pmid=15790597&doi=10.1093/hmg/ddi132&)

45. Li MD, Cheng R, Ma JZ, Swan GE. A meta-analysis of estimated genetic and environmental effects on smoking behavior in male and female adult twins. Addiction. 2003;98:23–31. doi: 10.1046/j.1360-0443.2003.00295.x.  [DOI](https://doi.org/10.1046/j.1360-0443.2003.00295.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12492752/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Addiction&title=A%20meta-analysis%20of%20estimated%20genetic%20and%20environmental%20effects%20on%20smoking%20behavior%20in%20male%20and%20female%20adult%20twins&author=MD%20Li&author=R%20Cheng&author=JZ%20Ma&author=GE%20Swan&volume=98&publication_year=2003&pages=23-31&pmid=12492752&doi=10.1046/j.1360-0443.2003.00295.x&)

46. Li MD, Lou XY, Chen G, Ma JZ, Elston RC. Gene-gene interactions among CHRNA4, CHRNB2, BDNF, and NTRK2 in nicotine dependence. Biol Psychiatry. 2008;64:951–957. doi: 10.1016/j.biopsych.2008.04.026.  [DOI](https://doi.org/10.1016/j.biopsych.2008.04.026) | [PMC free article](/articles/PMC2592606/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18534558/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biol%20Psychiatry&title=Gene-gene%20interactions%20among%20CHRNA4,%20CHRNB2,%20BDNF,%20and%20NTRK2%20in%20nicotine%20dependence&author=MD%20Li&author=XY%20Lou&author=G%20Chen&author=JZ%20Ma&author=RC%20Elston&volume=64&publication_year=2008&pages=951-957&pmid=18534558&doi=10.1016/j.biopsych.2008.04.026&)

47. Li MD, Xu Q, Lou XY, Payne TJ, Niu T, Ma JZ. Association and interaction analysis of variants in CHRNA5/CHRNA3/CHRNB4 gene cluster with nicotine dependence in African and European Americans. Am J Med Genet B Neuropsychiatr Genet. 2009;153B:745–756. doi: 10.1002/ajmg.b.31043.  [DOI](https://doi.org/10.1002/ajmg.b.31043) | [PMC free article](/articles/PMC2924635/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19859904/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Med%20Genet%20B%20Neuropsychiatr%20Genet&title=Association%20and%20interaction%20analysis%20of%20variants%20in%20CHRNA5/CHRNA3/CHRNB4%20gene%20cluster%20with%20nicotine%20dependence%20in%20African%20and%20European%20Americans&author=MD%20Li&author=Q%20Xu&author=XY%20Lou&author=TJ%20Payne&author=T%20Niu&volume=153B&publication_year=2009&pages=745-756&pmid=19859904&doi=10.1002/ajmg.b.31043&)

48. Liu JZ, Tozzi F, Waterworth DM, Pillai SG, Muglia P, Middleton L, et al. Meta-analysis and imputation refines the association of 15q25 with smoking quantity. Nat Genet. 2010;42:436–440. doi: 10.1038/ng.572.  [DOI](https://doi.org/10.1038/ng.572) | [PMC free article](/articles/PMC3612983/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20418889/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Meta-analysis%20and%20imputation%20refines%20the%20association%20of%2015q25%20with%20smoking%20quantity&author=JZ%20Liu&author=F%20Tozzi&author=DM%20Waterworth&author=SG%20Pillai&author=P%20Muglia&volume=42&publication_year=2010&pages=436-440&pmid=20418889&doi=10.1038/ng.572&)

49. Liu YZ, Pei YF, Guo YF, Wang L, Liu XG, Yan H, et al. Genome-wide association analyses suggested a novel mechanism for smoking behavior regulated by IL15. Mol Psychiatry. 2009;14:668–680. doi: 10.1038/mp.2009.3.  [DOI](https://doi.org/10.1038/mp.2009.3) | [PMC free article](/articles/PMC2700850/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19188921/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Psychiatry&title=Genome-wide%20association%20analyses%20suggested%20a%20novel%20mechanism%20for%20smoking%20behavior%20regulated%20by%20IL15&author=YZ%20Liu&author=YF%20Pei&author=YF%20Guo&author=L%20Wang&author=XG%20Liu&volume=14&publication_year=2009&pages=668-680&pmid=19188921&doi=10.1038/mp.2009.3&)

50. Lou XY, Chen GB, Yan L, Ma JZ, Mangold JE, Zhu J, et al. A combinatorial approach to detecting gene-gene and gene-environment interactions in family studies. Am J Hum Genet. 2008;83:457–467. doi: 10.1016/j.ajhg.2008.09.001.  [DOI](https://doi.org/10.1016/j.ajhg.2008.09.001) | [PMC free article](/articles/PMC2561932/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18834969/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=A%20combinatorial%20approach%20to%20detecting%20gene-gene%20and%20gene-environment%20interactions%20in%20family%20studies&author=XY%20Lou&author=GB%20Chen&author=L%20Yan&author=JZ%20Ma&author=JE%20Mangold&volume=83&publication_year=2008&pages=457-467&pmid=18834969&doi=10.1016/j.ajhg.2008.09.001&)

51. Lou XY, Chen GB, Yan L, Ma JZ, Zhu J, Elston RC, et al. A generalized combinatorial approach for detecting gene-by-gene and gene-by-environment interactions with application to nicotine dependence. Am J Hum Genet. 2007;80:1125–1137. doi: 10.1086/518312.  [DOI](https://doi.org/10.1086/518312) | [PMC free article](/articles/PMC1867100/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17503330/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=A%20generalized%20combinatorial%20approach%20for%20detecting%20gene-by-gene%20and%20gene-by-environment%20interactions%20with%20application%20to%20nicotine%20dependence&author=XY%20Lou&author=GB%20Chen&author=L%20Yan&author=JZ%20Ma&author=J%20Zhu&volume=80&publication_year=2007&pages=1125-1137&pmid=17503330&doi=10.1086/518312&)

52. Lou XY, Ma JZ, Payne TJ, Beuten J, Crew KM, Li MD. Gene-based analysis suggests association of the nicotinic acetylcholine receptor beta1 subunit (CHRNB1) and M1 muscarinic acetylcholine receptor (CHRM1) with vulnerability for nicotine dependence. Hum Genet. 2006;120:381–389. doi: 10.1007/s00439-006-0229-7.  [DOI](https://doi.org/10.1007/s00439-006-0229-7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16874522/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Genet&title=Gene-based%20analysis%20suggests%20association%20of%20the%20nicotinic%20acetylcholine%20receptor%20beta1%20subunit%20(CHRNB1)%20and%20M1%20muscarinic%20acetylcholine%20receptor%20(CHRM1)%20with%20vulnerability%20for%20nicotine%20dependence&author=XY%20Lou&author=JZ%20Ma&author=TJ%20Payne&author=J%20Beuten&author=KM%20Crew&volume=120&publication_year=2006&pages=381-389&pmid=16874522&doi=10.1007/s00439-006-0229-7&)

53. Lueders KK, Hu S, McHugh L, Myakishev MV, Sirota LA, Hamer DH. Genetic and functional analysis of single nucleotide polymorphisms in the beta2-neuronal nicotinic acetylcholine receptor gene (CHRNB2) Nicotine Tob Res. 2002;4:115–125. doi: 10.1080/14622200110098419.  [DOI](https://doi.org/10.1080/14622200110098419) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11906688/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nicotine%20Tob%20Res&title=Genetic%20and%20functional%20analysis%20of%20single%20nucleotide%20polymorphisms%20in%20the%20beta2-neuronal%20nicotinic%20acetylcholine%20receptor%20gene%20(CHRNB2)&author=KK%20Lueders&author=S%20Hu&author=L%20McHugh&author=MV%20Myakishev&author=LA%20Sirota&volume=4&publication_year=2002&pages=115-125&pmid=11906688&doi=10.1080/14622200110098419&)

54. Madsen BE, Browning SR. A group-wise association test for rare mutations using a weighted sum statistic. PLoS Genet. 2009;5:e1000384. doi: 10.1371/journal.pgen.1000384.  [DOI](https://doi.org/10.1371/journal.pgen.1000384) | [PMC free article](/articles/PMC2633048/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19214210/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet&title=A%20group-wise%20association%20test%20for%20rare%20mutations%20using%20a%20weighted%20sum%20statistic&author=BE%20Madsen&author=SR%20Browning&volume=5&publication_year=2009&pages=e1000384&pmid=19214210&doi=10.1371/journal.pgen.1000384&)

55. Maes HH, Sullivan PF, Bulik CM, Neale MC, Prescott CA, Eaves LJ, et al. A twin study of genetic and environmental influences on tobacco initiation, regular tobacco use and nicotine dependence. Psychol Med. 2004;34:1251–1261. doi: 10.1017/s0033291704002405.  [DOI](https://doi.org/10.1017/s0033291704002405) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15697051/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Psychol%20Med&title=A%20twin%20study%20of%20genetic%20and%20environmental%20influences%20on%20tobacco%20initiation,%20regular%20tobacco%20use%20and%20nicotine%20dependence&author=HH%20Maes&author=PF%20Sullivan&author=CM%20Bulik&author=MC%20Neale&author=CA%20Prescott&volume=34&publication_year=2004&pages=1251-1261&pmid=15697051&doi=10.1017/s0033291704002405&)

56. Margulies M, Egholm M, Altman WE, Attiya S, Bader JS, Bemben LA, et al. Genome sequencing in microfabricated high-density picolitre reactors. Nature. 2005;437:376–380. doi: 10.1038/nature03959.  [DOI](https://doi.org/10.1038/nature03959) | [PMC free article](/articles/PMC1464427/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16056220/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Genome%20sequencing%20in%20microfabricated%20high-density%20picolitre%20reactors&author=M%20Margulies&author=M%20Egholm&author=WE%20Altman&author=S%20Attiya&author=JS%20Bader&volume=437&publication_year=2005&pages=376-380&pmid=16056220&doi=10.1038/nature03959&)

57. Mineur YS, Picciotto MR. Genetics of nicotinic acetylcholine receptors: relevance to nicotine addiction. Biochem Pharmacol. 2008;75:323–333. doi: 10.1016/j.bcp.2007.06.010.  [DOI](https://doi.org/10.1016/j.bcp.2007.06.010) | [PMC free article](/articles/PMC2212607/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17632086/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem%20Pharmacol&title=Genetics%20of%20nicotinic%20acetylcholine%20receptors:%20relevance%20to%20nicotine%20addiction&author=YS%20Mineur&author=MR%20Picciotto&volume=75&publication_year=2008&pages=323-333&pmid=17632086&doi=10.1016/j.bcp.2007.06.010&)

58. Nievergelt CM, Libiger O, Schork NJ. Generalized analysis of molecular variance. PLoS Genet. 2007;3:e51. doi: 10.1371/journal.pgen.0030051.  [DOI](https://doi.org/10.1371/journal.pgen.0030051) | [PMC free article](/articles/PMC1847693/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17411342/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet&title=Generalized%20analysis%20of%20molecular%20variance&author=CM%20Nievergelt&author=O%20Libiger&author=NJ%20Schork&volume=3&publication_year=2007&pages=e51&pmid=17411342&doi=10.1371/journal.pgen.0030051&)

59. Nishita DM, Jack LM, McElroy M, McClure JB, Richards J, Swan GE, et al. Clinical trial participant characteristics and saliva and DNA metrics. BMC Med Res Methodol. 2009;9:71. doi: 10.1186/1471-2288-9-71.  [DOI](https://doi.org/10.1186/1471-2288-9-71) | [PMC free article](/articles/PMC2776600/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19874586/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Med%20Res%20Methodol&title=Clinical%20trial%20participant%20characteristics%20and%20saliva%20and%20DNA%20metrics&author=DM%20Nishita&author=LM%20Jack&author=M%20McElroy&author=JB%20McClure&author=J%20Richards&volume=9&publication_year=2009&pages=71&pmid=19874586&doi=10.1186/1471-2288-9-71&)

60. Perkins KA, Lerman C, Coddington S, Jetton C, Karelitz JL, Wilson A, et al. Gene and gene by sex associations with initial sensitivity to nicotine in nonsmokers. Behav Pharmacol. 2008a;19:630–640. doi: 10.1097/FBP.0b013e32830c3621.  [DOI](https://doi.org/10.1097/FBP.0b013e32830c3621) | [PMC free article](/articles/PMC2743299/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18690117/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Behav%20Pharmacol&title=Gene%20and%20gene%20by%20sex%20associations%20with%20initial%20sensitivity%20to%20nicotine%20in%20nonsmokers&author=KA%20Perkins&author=C%20Lerman&author=S%20Coddington&author=C%20Jetton&author=JL%20Karelitz&volume=19&publication_year=2008a&pages=630-640&pmid=18690117&doi=10.1097/FBP.0b013e32830c3621&)

61. Perkins KA, Lerman C, Grottenthaler A, Ciccocioppo MM, Milanak M, Conklin CA, et al. Dopamine and opioid gene variants are associated with increased smoking reward and reinforcement owing to negative mood. Behav Pharmacol. 2008b;19:641–649. doi: 10.1097/FBP.0b013e32830c367c.  [DOI](https://doi.org/10.1097/FBP.0b013e32830c367c) | [PMC free article](/articles/PMC2717609/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18690118/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Behav%20Pharmacol&title=Dopamine%20and%20opioid%20gene%20variants%20are%20associated%20with%20increased%20smoking%20reward%20and%20reinforcement%20owing%20to%20negative%20mood&author=KA%20Perkins&author=C%20Lerman&author=A%20Grottenthaler&author=MM%20Ciccocioppo&author=M%20Milanak&volume=19&publication_year=2008b&pages=641-649&pmid=18690118&doi=10.1097/FBP.0b013e32830c367c&)

62. Perkins KA, Lerman C, Mercincavage M, Fonte CA, Briski JL. Nicotinic acetylcholine receptor beta2 subunit (CHRNB2) gene and short-term ability to quit smoking in response to nicotine patch. Cancer Epidemiol Biomarkers Prev. 2009;18:2608–2612. doi: 10.1158/1055-9965.EPI-09-0166.  [DOI](https://doi.org/10.1158/1055-9965.EPI-09-0166) | [PMC free article](/articles/PMC2759850/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19755656/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Epidemiol%20Biomarkers%20Prev&title=Nicotinic%20acetylcholine%20receptor%20beta2%20subunit%20(CHRNB2)%20gene%20and%20short-term%20ability%20to%20quit%20smoking%20in%20response%20to%20nicotine%20patch&author=KA%20Perkins&author=C%20Lerman&author=M%20Mercincavage&author=CA%20Fonte&author=JL%20Briski&volume=18&publication_year=2009&pages=2608-2612&pmid=19755656&doi=10.1158/1055-9965.EPI-09-0166&)

63. Philibert RA, Todorov A, Andersen A, Hollenbeck N, Gunter T, Heath A, et al. Examination of the nicotine dependence (NICSNP) consortium findings in the Iowa adoption studies population. Nicotine Tob Res. 2009;11:286–292. doi: 10.1093/ntr/ntn034.  [DOI](https://doi.org/10.1093/ntr/ntn034) | [PMC free article](/articles/PMC2666378/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19307444/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nicotine%20Tob%20Res&title=Examination%20of%20the%20nicotine%20dependence%20(NICSNP)%20consortium%20findings%20in%20the%20Iowa%20adoption%20studies%20population&author=RA%20Philibert&author=A%20Todorov&author=A%20Andersen&author=N%20Hollenbeck&author=T%20Gunter&volume=11&publication_year=2009&pages=286-292&pmid=19307444&doi=10.1093/ntr/ntn034&)

64. Piper ME, McCarthy DE, Baker TB. Assessing tobacco dependence: a guide to measure evaluation and selection. Nicotine Tob Res. 2006;8:339–351. doi: 10.1080/14622200600672765.  [DOI](https://doi.org/10.1080/14622200600672765) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16801292/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nicotine%20Tob%20Res&title=Assessing%20tobacco%20dependence:%20a%20guide%20to%20measure%20evaluation%20and%20selection&author=ME%20Piper&author=DE%20McCarthy&author=TB%20Baker&volume=8&publication_year=2006&pages=339-351&pmid=16801292&doi=10.1080/14622200600672765&)

65. Rana BK, Wessel J, Mahboubi V, Rao F, Haeller J, Gayen JR, et al. Natural variation within the neuronal nicotinic acetylcholine receptor cluster on human chromosome 15q24: influence on heritable autonomic traits in twin pairs. J Pharmacol Exp Ther. 2009;331:419–428. doi: 10.1124/jpet.109.157271.  [DOI](https://doi.org/10.1124/jpet.109.157271) | [PMC free article](/articles/PMC2775266/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19671882/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharmacol%20Exp%20Ther&title=Natural%20variation%20within%20the%20neuronal%20nicotinic%20acetylcholine%20receptor%20cluster%20on%20human%20chromosome%2015q24:%20influence%20on%20heritable%20autonomic%20traits%20in%20twin%20pairs&author=BK%20Rana&author=J%20Wessel&author=V%20Mahboubi&author=F%20Rao&author=J%20Haeller&volume=331&publication_year=2009&pages=419-428&pmid=19671882&doi=10.1124/jpet.109.157271&)

66. Rigbi A, Kanyas K, Yakir A, Greenbaum L, Pollak Y, Ben-Asher E, et al. Why do young women smoke? V. Role of direct and interactive effects of nicotinic cholinergic receptor gene variation on neurocognitive function. Genes Brain Behav. 2008;7:164–172. doi: 10.1111/j.1601-183X.2007.00329.x.  [DOI](https://doi.org/10.1111/j.1601-183X.2007.00329.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17559419/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genes%20Brain%20Behav&title=Why%20do%20young%20women%20smoke?%20V.%20Role%20of%20direct%20and%20interactive%20effects%20of%20nicotinic%20cholinergic%20receptor%20gene%20variation%20on%20neurocognitive%20function&author=A%20Rigbi&author=K%20Kanyas&author=A%20Yakir&author=L%20Greenbaum&author=Y%20Pollak&volume=7&publication_year=2008&pages=164-172&pmid=17559419&doi=10.1111/j.1601-183X.2007.00329.x&)

67. Sabatelli M, Eusebi F, Al-Chalabi A, Conte A, Madia F, Luigetti M, et al. Rare missense variants of neuronal nicotinic acetylcholine receptor altering receptor function are associated with sporadic amyotrophic lateral sclerosis. Hum Mol Genet. 2009;18:3997–4006. doi: 10.1093/hmg/ddp339.  [DOI](https://doi.org/10.1093/hmg/ddp339) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19628475/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mol%20Genet&title=Rare%20missense%20variants%20of%20neuronal%20nicotinic%20acetylcholine%20receptor%20altering%20receptor%20function%20are%20associated%20with%20sporadic%20amyotrophic%20lateral%20sclerosis&author=M%20Sabatelli&author=F%20Eusebi&author=A%20Al-Chalabi&author=A%20Conte&author=F%20Madia&volume=18&publication_year=2009&pages=3997-4006&pmid=19628475&doi=10.1093/hmg/ddp339&)

68. Saccone NL, Saccone SF, Hinrichs AL, Stitzel JA, Duan W, Pergadia ML, et al. Multiple distinct risk loci for nicotine dependence identified by dense coverage of the complete family of nicotinic receptor subunit (CHRN) genes. Am J Med Genet B Neuropsychiatr Genet. 2009a;150B:453–466. doi: 10.1002/ajmg.b.30828.  [DOI](https://doi.org/10.1002/ajmg.b.30828) | [PMC free article](/articles/PMC2693307/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19259974/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Med%20Genet%20B%20Neuropsychiatr%20Genet&title=Multiple%20distinct%20risk%20loci%20for%20nicotine%20dependence%20identified%20by%20dense%20coverage%20of%20the%20complete%20family%20of%20nicotinic%20receptor%20subunit%20(CHRN)%20genes&author=NL%20Saccone&author=SF%20Saccone&author=AL%20Hinrichs&author=JA%20Stitzel&author=W%20Duan&volume=150B&publication_year=2009a&pages=453-466&pmid=19259974&doi=10.1002/ajmg.b.30828&)

69. Saccone NL, Wang JC, Breslau N, Johnson EO, Hatsukami D, Saccone SF, et al. The CHRNA5-CHRNA3-CHRNB4 nicotinic receptor subunit gene cluster affects risk for nicotine dependence in African-Americans and in European-Americans. Cancer Res. 2009b;69:6848–6856. doi: 10.1158/0008-5472.CAN-09-0786.  [DOI](https://doi.org/10.1158/0008-5472.CAN-09-0786) | [PMC free article](/articles/PMC2874321/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19706762/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=The%20CHRNA5-CHRNA3-CHRNB4%20nicotinic%20receptor%20subunit%20gene%20cluster%20affects%20risk%20for%20nicotine%20dependence%20in%20African-Americans%20and%20in%20European-Americans&author=NL%20Saccone&author=JC%20Wang&author=N%20Breslau&author=EO%20Johnson&author=D%20Hatsukami&volume=69&publication_year=2009b&pages=6848-6856&pmid=19706762&doi=10.1158/0008-5472.CAN-09-0786&)

70. Saccone SF, Hinrichs AL, Saccone NL, Chase GA, Konvicka K, Madden PA, et al. Cholinergic nicotinic receptor genes implicated in a nicotine dependence association study targeting 348 candidate genes with 3713 SNPs. Hum Mol Genet. 2007;16:36–49. doi: 10.1093/hmg/ddl438.  [DOI](https://doi.org/10.1093/hmg/ddl438) | [PMC free article](/articles/PMC2270437/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17135278/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mol%20Genet&title=Cholinergic%20nicotinic%20receptor%20genes%20implicated%20in%20a%20nicotine%20dependence%20association%20study%20targeting%20348%20candidate%20genes%20with%203713%20SNPs&author=SF%20Saccone&author=AL%20Hinrichs&author=NL%20Saccone&author=GA%20Chase&author=K%20Konvicka&volume=16&publication_year=2007&pages=36-49&pmid=17135278&doi=10.1093/hmg/ddl438&)

71. Sanger F, Nicklen S, Coulson AR. DNA sequencing with chain-terminating inhibitors. Proc Natl Acad Sci USA. 1977;74:5463–5467. doi: 10.1073/pnas.74.12.5463.  [DOI](https://doi.org/10.1073/pnas.74.12.5463) | [PMC free article](/articles/PMC431765/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/271968/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20USA&title=DNA%20sequencing%20with%20chain-terminating%20inhibitors&author=F%20Sanger&author=S%20Nicklen&author=AR%20Coulson&volume=74&publication_year=1977&pages=5463-5467&pmid=271968&doi=10.1073/pnas.74.12.5463&)

72. Schork NJ, Murray SS, Frazer KA, Topol EJ. Common vs rare allele hypotheses for complex diseases. Curr Opin Genet Dev. 2009;19:212–219. doi: 10.1016/j.gde.2009.04.010.  [DOI](https://doi.org/10.1016/j.gde.2009.04.010) | [PMC free article](/articles/PMC2914559/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19481926/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Opin%20Genet%20Dev&title=Common%20vs%20rare%20allele%20hypotheses%20for%20complex%20diseases&author=NJ%20Schork&author=SS%20Murray&author=KA%20Frazer&author=EJ%20Topol&volume=19&publication_year=2009&pages=212-219&pmid=19481926&doi=10.1016/j.gde.2009.04.010&)

73. Schork NJ, Wessel J, Malo N. DNA sequence-based phenotypic association analysis. Adv Genet. 2008;60:195–217. doi: 10.1016/S0065-2660(07)00409-9.  [DOI](https://doi.org/10.1016/S0065-2660(07)00409-9) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18358322/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Adv%20Genet&title=DNA%20sequence-based%20phenotypic%20association%20analysis&author=NJ%20Schork&author=J%20Wessel&author=N%20Malo&volume=60&publication_year=2008&pages=195-217&pmid=18358322&doi=10.1016/S0065-2660(07)00409-9&)

74. Sherva R, Wilhelmsen K, Pomerleau CS, Chasse SA, Rice JP, Snedecor SM, et al. Association of a single nucleotide polymorphism in neuronal acetylcholine receptor subunit alpha 5 (CHRNA5) with smoking status and with ‘pleasurable buzz' during early experimentation with smoking. Addiction. 2008;103:1544–1552. doi: 10.1111/j.1360-0443.2008.02279.x.  [DOI](https://doi.org/10.1111/j.1360-0443.2008.02279.x) | [PMC free article](/articles/PMC2582398/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18783506/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Addiction&title=Association%20of%20a%20single%20nucleotide%20polymorphism%20in%20neuronal%20acetylcholine%20receptor%20subunit%20alpha%205%20(CHRNA5)%20with%20smoking%20status%20and%20with%20%E2%80%98pleasurable%20buzz'%20during%20early%20experimentation%20with%20smoking&author=R%20Sherva&author=K%20Wilhelmsen&author=CS%20Pomerleau&author=SA%20Chasse&author=JP%20Rice&volume=103&publication_year=2008&pages=1544-1552&pmid=18783506&doi=10.1111/j.1360-0443.2008.02279.x&)

75. Silverman MA, Neale MC, Sullivan PF, Harris-Kerr C, Wormley B, Sadek H, et al. Haplotypes of four novel single nucleotide polymorphisms in the nicotinic acetylcholine receptor beta2-subunit (CHRNB2) gene show no association with smoking initiation or nicotine dependence. Am J Med Genet. 2000;96:646–653.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/11054772/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Med%20Genet&title=Haplotypes%20of%20four%20novel%20single%20nucleotide%20polymorphisms%20in%20the%20nicotinic%20acetylcholine%20receptor%20beta2-subunit%20(CHRNB2)%20gene%20show%20no%20association%20with%20smoking%20initiation%20or%20nicotine%20dependence&author=MA%20Silverman&author=MC%20Neale&author=PF%20Sullivan&author=C%20Harris-Kerr&author=B%20Wormley&volume=96&publication_year=2000&pages=646-653&pmid=11054772&)

76. Spitz MR, Amos CI, Dong Q, Lin J, Wu X. The CHRNA5-A3 region on chromosome 15q24-25.1 is a risk factor both for nicotine dependence and for lung cancer. J Natl Cancer Inst. 2008;100:1552–1556. doi: 10.1093/jnci/djn363.  [DOI](https://doi.org/10.1093/jnci/djn363) | [PMC free article](/articles/PMC2720751/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18957677/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Natl%20Cancer%20Inst&title=The%20CHRNA5-A3%20region%20on%20chromosome%2015q24-25.1%20is%20a%20risk%20factor%20both%20for%20nicotine%20dependence%20and%20for%20lung%20cancer&author=MR%20Spitz&author=CI%20Amos&author=Q%20Dong&author=J%20Lin&author=X%20Wu&volume=100&publication_year=2008&pages=1552-1556&pmid=18957677&doi=10.1093/jnci/djn363&)

77. Stevens VL, Bierut LJ, Talbot JT, Wang JC, Sun J, Hinrichs AL, et al. Nicotinic receptor gene variants influence susceptibility to heavy smoking. Cancer Epidemiol Biomarkers Prev. 2008;17:3517–3525. doi: 10.1158/1055-9965.EPI-08-0585.  [DOI](https://doi.org/10.1158/1055-9965.EPI-08-0585) | [PMC free article](/articles/PMC2614129/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19029397/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Epidemiol%20Biomarkers%20Prev&title=Nicotinic%20receptor%20gene%20variants%20influence%20susceptibility%20to%20heavy%20smoking&author=VL%20Stevens&author=LJ%20Bierut&author=JT%20Talbot&author=JC%20Wang&author=J%20Sun&volume=17&publication_year=2008&pages=3517-3525&pmid=19029397&doi=10.1158/1055-9965.EPI-08-0585&)

78. Stolerman IP, Jarvis MJ.1995The scientific case that nicotine is addictive Psychopharmacology (Berl) 1172–10.discussion 14–20.  [DOI](https://doi.org/10.1007/BF02245088) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7724697/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Psychopharmacology%20(Berl)&title=The%20scientific%20case%20that%20nicotine%20is%20addictive&author=IP%20Stolerman&author=MJ%20Jarvis&volume=117&publication_year=1995&pages=2-10&pmid=7724697&doi=10.1007/BF02245088&)

79. Swan GE, McAfee T, Curry SJ, Jack LM, Javitz H, Dacey S, et al. Effectiveness of bupropion sustained release for smoking cessation in a health care setting: a randomized trial. Arch Intern Med. 2003;163:2337–2344. doi: 10.1001/archinte.163.19.2337.  [DOI](https://doi.org/10.1001/archinte.163.19.2337) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14581254/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arch%20Intern%20Med&title=Effectiveness%20of%20bupropion%20sustained%20release%20for%20smoking%20cessation%20in%20a%20health%20care%20setting:%20a%20randomized%20trial&author=GE%20Swan&author=T%20McAfee&author=SJ%20Curry&author=LM%20Jack&author=H%20Javitz&volume=163&publication_year=2003&pages=2337-2344&pmid=14581254&doi=10.1001/archinte.163.19.2337&)

80. Swan GE, McClure JB, Jack LM, Zbikowski SM, Javitz H, Catz SL, et al. Behavioral counseling and varenicline treatment for smoking cessation. Am J Prev Med. 2010;38:482–490. doi: 10.1016/j.amepre.2010.01.024.  [DOI](https://doi.org/10.1016/j.amepre.2010.01.024) | [PMC free article](/articles/PMC2879135/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20409497/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Prev%20Med&title=Behavioral%20counseling%20and%20varenicline%20treatment%20for%20smoking%20cessation&author=GE%20Swan&author=JB%20McClure&author=LM%20Jack&author=SM%20Zbikowski&author=H%20Javitz&volume=38&publication_year=2010&pages=482-490&pmid=20409497&doi=10.1016/j.amepre.2010.01.024&)

81. TAG Genome-wide meta-analyses identify multiple loci associated with smoking behavior. Nat Genet. 2010;42:441–447. doi: 10.1038/ng.571.  [DOI](https://doi.org/10.1038/ng.571) | [PMC free article](/articles/PMC2914600/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20418890/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Genome-wide%20meta-analyses%20identify%20multiple%20loci%20associated%20with%20smoking%20behavior&volume=42&publication_year=2010&pages=441-447&pmid=20418890&doi=10.1038/ng.571&)

82. Thorgeirsson TE, Geller F, Sulem P, Rafnar T, Wiste A, Magnusson KP, et al. A variant associated with nicotine dependence, lung cancer and peripheral arterial disease. Nature. 2008;452:638–642. doi: 10.1038/nature06846.  [DOI](https://doi.org/10.1038/nature06846) | [PMC free article](/articles/PMC4539558/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18385739/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=A%20variant%20associated%20with%20nicotine%20dependence,%20lung%20cancer%20and%20peripheral%20arterial%20disease&author=TE%20Thorgeirsson&author=F%20Geller&author=P%20Sulem&author=T%20Rafnar&author=A%20Wiste&volume=452&publication_year=2008&pages=638-642&pmid=18385739&doi=10.1038/nature06846&)

83. Thorgeirsson TE, Gudbjartsson DF, Surakka I, Vink JM, Amin N, Geller F, et al. Sequence variants at CHRNB3-CHRNA6 and CYP2A6 affect smoking behavior. Nat Genet. 2010;42:448–453. doi: 10.1038/ng.573.  [DOI](https://doi.org/10.1038/ng.573) | [PMC free article](/articles/PMC3080600/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20418888/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Sequence%20variants%20at%20CHRNB3-CHRNA6%20and%20CYP2A6%20affect%20smoking%20behavior&author=TE%20Thorgeirsson&author=DF%20Gudbjartsson&author=I%20Surakka&author=JM%20Vink&author=N%20Amin&volume=42&publication_year=2010&pages=448-453&pmid=20418888&doi=10.1038/ng.573&)

84. Vink JM, Willemsen G, Boomsma DI. Heritability of smoking initiation and nicotine dependence. Behav Genet. 2005;35:397–406. doi: 10.1007/s10519-004-1327-8.  [DOI](https://doi.org/10.1007/s10519-004-1327-8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15971021/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Behav%20Genet&title=Heritability%20of%20smoking%20initiation%20and%20nicotine%20dependence&author=JM%20Vink&author=G%20Willemsen&author=DI%20Boomsma&volume=35&publication_year=2005&pages=397-406&pmid=15971021&doi=10.1007/s10519-004-1327-8&)

85. Wang J, Li MD. Common and unique biological pathways associated with smoking initiation/progression, nicotine dependence, and smoking cessation. Neuropsychopharmacology. 2010;35:702–719. doi: 10.1038/npp.2009.178.  [DOI](https://doi.org/10.1038/npp.2009.178) | [PMC free article](/articles/PMC2814000/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19890259/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neuropsychopharmacology&title=Common%20and%20unique%20biological%20pathways%20associated%20with%20smoking%20initiation/progression,%20nicotine%20dependence,%20and%20smoking%20cessation&author=J%20Wang&author=MD%20Li&volume=35&publication_year=2010&pages=702-719&pmid=19890259&doi=10.1038/npp.2009.178&)

86. Wang JC, Grucza R, Cruchaga C, Hinrichs AL, Bertelsen S, Budde JP, et al. Genetic variation in the CHRNA5 gene affects mRNA levels and is associated with risk for alcohol dependence. Mol Psychiatry. 2009;14:501–510. doi: 10.1038/mp.2008.42.  [DOI](https://doi.org/10.1038/mp.2008.42) | [PMC free article](/articles/PMC4381434/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18414406/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Psychiatry&title=Genetic%20variation%20in%20the%20CHRNA5%20gene%20affects%20mRNA%20levels%20and%20is%20associated%20with%20risk%20for%20alcohol%20dependence&author=JC%20Wang&author=R%20Grucza&author=C%20Cruchaga&author=AL%20Hinrichs&author=S%20Bertelsen&volume=14&publication_year=2009&pages=501-510&pmid=18414406&doi=10.1038/mp.2008.42&)

87. Wang K, Dickson SP, Stolle CA, Krantz ID, Goldstein DB, Hakonarson H. Interpretation of association signals and identification of causal variants from genome-wide association studies. Am J Hum Genet. 2010;86:730–742. doi: 10.1016/j.ajhg.2010.04.003.  [DOI](https://doi.org/10.1016/j.ajhg.2010.04.003) | [PMC free article](/articles/PMC2869011/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20434130/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=Interpretation%20of%20association%20signals%20and%20identification%20of%20causal%20variants%20from%20genome-wide%20association%20studies&author=K%20Wang&author=SP%20Dickson&author=CA%20Stolle&author=ID%20Krantz&author=DB%20Goldstein&volume=86&publication_year=2010&pages=730-742&pmid=20434130&doi=10.1016/j.ajhg.2010.04.003&)

88. Weiss RB, Baker TB, Cannon DS, von Niederhausern A, Dunn DM, Matsunami N, et al. A candidate gene approach identifies the CHRNA5-A3-B4 region as a risk factor for age-dependent nicotine addiction. PLoS Genet. 2008;4:e1000125. doi: 10.1371/journal.pgen.1000125.  [DOI](https://doi.org/10.1371/journal.pgen.1000125) | [PMC free article](/articles/PMC2442220/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18618000/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet&title=A%20candidate%20gene%20approach%20identifies%20the%20CHRNA5-A3-B4%20region%20as%20a%20risk%20factor%20for%20age-dependent%20nicotine%20addiction&author=RB%20Weiss&author=TB%20Baker&author=DS%20Cannon&author=A%20von%20Niederhausern&author=DM%20Dunn&volume=4&publication_year=2008&pages=e1000125&pmid=18618000&doi=10.1371/journal.pgen.1000125&)

89. Wessel J, Schork NJ. Generalized genomic distance-based regression methodology for multilocus association analysis. Am J Hum Genet. 2006;79:792–806. doi: 10.1086/508346.  [DOI](https://doi.org/10.1086/508346) | [PMC free article](/articles/PMC1698575/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17033957/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=Generalized%20genomic%20distance-based%20regression%20methodology%20for%20multilocus%20association%20analysis&author=J%20Wessel&author=NJ%20Schork&volume=79&publication_year=2006&pages=792-806&pmid=17033957&doi=10.1086/508346&)
