# The Relative Contributions of CYP3A4 and CYP3A5 to the Metabolism of Vinorelbine

## Metadata
**Authors:** Ariel R Topletz, Jennifer B Dennison, Robert J Barbuch, Chad E Hadden, Stephen D Hall, Jamie L Renbarger
**Journal:** Drug Metabolism and Disposition
**Date:** 2013 Sep
**DOI:** [10.1124/dmd.113.051094](https://doi.org/10.1124/dmd.113.051094)
**PMID:** 23780963
**PMCID:** PMC3876810
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3876810/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC3876810/pdf/dmd.113.051094.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC3876810/pdf/dmd.113.051094.pdf)

## Abstract

Vinorelbine is a semisynthetic vinca alkaloid used in the treatment of advanced breast and non-small cell lung cancers. Vincristine, a related vinca alkaloid, is 9-fold more efficiently metabolized by CYP3A5 than by CYP3A4 in vitro. This study quantified the relative contribution of CYP3A4 and CYP3A5 to the metabolism of vinorelbine in vitro using cDNA-expressed human cytochrome P450s (P450s) and human liver microsomes (HLMs). CYP3A4 and CYP3A5 were identified as the P450s capable of oxidizing vinorelbine using a panel of human enzymes and selective P450 inhibitors in HLMs. For CYP3A4 coexpressed with cytochrome b5 (CYP3A4+b5) and CYP3A5+b5, the Michaelis-Menten constants for vinorelbine were 2.6 and 3.6 μM, respectively, but the Vmax of 1.4 pmol/min/pmol was common to both enzymes. In HLMs, the intrinsic clearance of vinorelbine metabolism was highly correlated with CYP3A4 activity, and there was no significant difference in intrinsic clearance between CYP3A5 high and low expressers. When radiolabeled vinorelbine substrate was used, there were clear qualitative differences in metabolite formation fingerprints between CYP3A4+b5 and CYP3A5+b5 as determined by NMR and mass spectrometry analysis. One major metabolite (M2), a didehydro-vinorelbine, was present in both recombinant and microsomal systems but was more abundant in CYP3A4+b5 incubations. We conclude that despite the equivalent efficiency of recombinant CYP3A4 and CYP3A5 in vinorelbine metabolism the polymorphic expression of CYP3A5, as shown by the kinetics with HLMs, may have a minimal effect on systemic clearance of vinorelbine.

## Introduction

Vinorelbine is a semisynthetic vinca alkaloid used as chemotherapy primarily in the treatment of advanced breast and non-small-cell lung cancers (NSCLC). The advantages of vinorelbine over the naturally occurring vinca alkaloids such as vincristine include less overall drug-related toxicity owing to a lower therapeutic dose (4- to 10-fold) and a broader antitumor spectrum ([Leveque and Jehl, 1996](#B16)).

Clinical pharmacokinetic studies of vinorelbine reveal significant interpatient variability ([Gauvin et al., 2002a](#B8),[b](#B9); [Qian et al., 2011](#B22)). [Wong et al. (2006)](#B27) reported no association of vinorelbine pharmacokinetic parameters with *ABCB1* or *CYP3A5* genotypes; however, the study was not adequately powered to truly evaluate for genetics. Furthermore, this study found an association between patient body surface area and myelosuppression; however, they found no association of either of these factors with vinorelbine pharmacokinetics. As such, while there are a number of factors that may be contributing to the significant variability in vinorelbine pharmacokinetics ([Baker and Sparreboom, 2006](#B1); [Wong et al., 2006](#B27)), one that remains to be systematically investigated is the potential impact of variability of genes encoding important drug metabolizing enzymes on vinorelbine pharmacokinetics. Cytochrome P450 (P450) enzymes are a superfamily of hemoproteins responsible for most oxidative metabolic drug clearance in vivo, including metabolism of the vinca alkaloids. The CYP3A family includes CYP3A4, which alone comprises over 30% of hepatic enzymes and is involved in the metabolism for over 50% of marketed drugs that rely on metabolic elimination. CYP3A5 is structurally similar to CYP3A4, yet the substrate selectivity of these highly homologous proteins differs unpredictably ([Kuehl et al., 2001](#B13); [Lamba et al., 2002](#B14); [Xie et al., 2004](#B28)). Genetic polymorphisms of *CYP3A5* affect protein expression and activity, and consequently alter the intrinsic clearances of drugs selectively metabolized by CYP3A5 ([Kuehl et al., 2001](#B13); [Lamba et al., 2002](#B14); [Xie et al., 2004](#B28)). Subjects homozygous or heterozygous for the *CYP3A5*1* allele are characteristically high expressers of the functional protein ([Kuehl et al., 2001](#B13)) whereas the *CYP3A5*3*, **6*, and **7* alleles result in greatly diminished expression of the enzyme ([Hustert et al., 2001](#B11); [Kuehl et al., 2001](#B13); [Lamba et al., 2002](#B14); [Xie et al., 2004](#B28)). CYP3A5 is expressed by approximately 55% of African Americans but only 10–20% of Caucasians ([Kuehl et al., 2001](#B13); [Roy et al., 2005](#B25)).

Vincristine, a related vinca alkaloid, is metabolized by the CYP3A enzymes CYP3A4 and CYP3A5 ([Dennison et al., 2006](#B6)). Vincristine is highly selectively metabolized by CYP3A5 in vitro, suggesting the possible need for an individualized therapeutic approach ([Dennison et al., 2006](#B6)). It is well established that African Americans have poorer overall survival rates compared with Caucasians in a number of malignancies for which vincristine is a core chemotherapeutic agent ([Longo et al., 1986](#B17); [Pollock et al., 2000](#B21)). Individuals who express CYP3A5 may metabolize vincristine more efficiently than nonexpressers, resulting in lower vincristine exposure and thus potentially less drug efficacy and toxicity. Clinical data demonstrate less vincristine-induced peripheral neuropathy in African Americans, and a recent retrospective study in children diagnosed with acute lymphoblastic leukemia and treated with vincristine revealed greater neuropathy in children with CYP3A5 low expresser genotypes ([Renbarger et al., 2008](#B23); [Egbelakin et al., 2011](#B7)).

Vinorelbine, as a member of the vinca alkaloid family, could be hypothesized to be similar to vincristine in the contribution of CYP3A5-mediated metabolism with analogous clinical outcomes. A study using human liver microsomes (HLMs) with known P450 enzyme protein concentrations suggested that vinorelbine is extensively metabolized by CYP3A4 but not by CYP2D6; the role of CYP3A5 was not clearly defined ([Beulz-Riche et al., 2005](#B3)).

Preliminary retrospective clinical studies have explored the possible association between *CYP3A5* genotype and clinical response to vinorelbine. [Wong et al. (2006)](#B27) only found a weak correlation between vinorelbine clearance and both CYP3A5 expression status and a common ABCB1 genotype. In a study with NSCLC patients, the presence of a *CYP3A5*1* allele was found to correlate to a slightly higher overall chemotherapy response to vinorelbine ([Pan et al., 2008](#B20)). To clarify the effect of CYP3A5 expression, we quantified the relative contribution of CYP3A4 and CYP4A5 to the metabolic clearance of vinorelbine in vitro and used tandem mass spectrometry (MS/MS) and NMR to identify the major oxidative metabolites specific to CYP3A4.

## Materials and Methods

### 

#### Chemicals and Enzymes.

Vinorelbine tartrate, ketoconazole, omeprazole, quinidine, trimethoprim, sulfaphenazole, *α*-naphthoflavone, pilocarpine, diethyldithiocarbamate, thioTEPA, and NADPH were purchased from Sigma-Aldrich (St. Louis, MO). Vinorelbine [^3^H(G)] (5 Ci/mmol) was obtained from American Radiolabeled Chemicals, Inc. (St. Louis, MO). All other reagents were of high-performance liquid chromatography (HPLC) grade and were purchased from Fisher Scientific (Pittsburgh, PA).

Supersomes containing cDNA-expressed cytochrome P450s coexpressed with cytochrome P450 reductase with (CYP1A1, 1A2, 2B6, 2C19, 2D6, 2J2, 3A4, 3A5, 3A7, and 4A11) and without (CYP2A6, 2C8, 2C9, and 2E1) coexpressed cytochrome *b*_5_ (+*b*_5_) were purchased from the BD Gentest (Woburn, MA). The manufacturer provided the cytochrome P450-reductase activities, protein concentrations, and the cytochrome P450 content of the Supersomes.

#### Incubations with cDNA-Expressed Cytochrome P450s.

Preliminary experiments with cDNA-expressed CYP3A4+*b*_5_ and CYP3A5+*b*_5_ significantly depleted the parent drug but did not result in quantifiable metabolite formation using HPLC with UV detection at a 1 *μ*M substrate concentration. Consequently, in all subsequent experiments, the rates of vinorelbine metabolism were quantified by substrate depletion. Linearity of time and protein content was used to determine incubation conditions suitable for all substrate concentrations, and the percentage of vinorelbine depletion ranged from 8 to 33% from initial concentrations as determined from the controls without NADPH (−NADPH); statistical significance between samples and −NADPH controls was determined. Less than 5% day-to-day variability was observed in HPLC-UV analysis as compared with QC controls, and the limit of quantification of vinorelbine was 0.1 *µ*M.

Incubations were performed under conditions previously developed to study the oxidation of vincristine ([Dennison et al., 2006](#B6)). Samples were incubated in an oscillating water bath at 37°C for a specified time as determined per experiment. Vinorelbine in methanol (MeOH) was added to each 3 ml polypropylene reaction vial and evaporated to dryness in a centrifuge under vacuum before all incubations (Thermo Electron Corporation Savant, SC210A SpeedVac Concentrator; Thermo Fisher, Waltham, MA). We added 200 *μ*l of 100 mM Na_2_HPO_4_ buffer with 5 mM MgCl_2_, pH 7.4, to each vial and vortexed to ensure the drug was in solution before incubation. Each vial was placed in the 37°C water bath and preincubated for 3 minutes. Enzyme was then added, followed by NADPH after a further 3 minutes to start the reaction. To terminate the reaction, an equal volume (200 *μ*l) of acetonitrile was added to each vial at the appropriate time to quench the incubation. Vials were immediately placed on ice, and the internal standard, ketoconazole (1 *µ*M in MeOH), was added. Samples were then placed into a −80°C freezer for a minimum of 30 minutes, and they were subsequently thawed and centrifuged for 10 minutes (3000*g*, 20°C) to precipitate any excess salt. The clear liquid was collected and stored at −80°C until HPLC analysis.

A panel of 14 human drug metabolizing, cDNA-expressed P450s (1A1, 1A2, 2B6, 2C19, 2D6, 2J2, 3A4, 3A5, 3A7, 4A11, 2A6, 2C8, 2C9, and 2E1) were incubated with 1 *μ*M vinorelbine and 30 pmol/ml of each enzyme for 12 minutes in duplicate. The control reactions did not contain NADPH.

To determine the Michaelis-Menten parameters of vinorelbine for CYP3A4+*b*_5_ and CYP3A5+*b*_5_, incubations were performed in triplicate at six vinorelbine concentrations of 1, 2, 3, 5, 7, and 10 *μ*M. Incubation controls were also performed in triplicate and lacked NADPH. Nonincubated controls consisted of vinorelbine in buffer. Samples were incubated with 30 pmol/ml of recombinant enzyme for 5 minutes. Concentrations for each sample were measured after a 5-minute incubation; however, as there was no metabolism observed in the −NADPH control samples, the −NADPH controls were also used to indicate initial substrate concentrations at each time point. The rate of vinorelbine metabolism was calculated for each supersome from the pmol of parent drug lost per minute per pmol of CYP3A4+*b*_5_ or CYP3A5+*b*_5_. Substrate concentrations were corrected for depletion using the −NADPH controls at each respective concentration as a 0-minute time-point concentration before calculating the Michaelis-Menten parameters of vinorelbine metabolism. Corrected substrate concentrations were calculated as the average vinorelbine concentration over 5 minutes.

#### Incubations with Human Liver Microsomes.

HLMs (*n* = 56) were prepared from human liver tissues from the Indiana University Liver Bank (IUL) (Indianapolis, IN) as described previously elsewhere ([Gorski et al., 1998](#B10)). Each HLM was prepared from tissue that had been genotyped for *CYP3A5,*3,*6,* and **7* as described in detail previously elsewhere ([Dennison et al., 2007](#B5)). Microsomal protein concentrations and CYP3A4 and CYP3A5 abundances of the HLMs were reported previously ([Leveque and Jehl, 1996](#B16)). From this bank of 56 livers, 20 were chosen to cover a broad range of CYP3A4 and CYP3A5 contents. Protein contents of the enzymes per HLM were quantified by Western blot analysis ([Dennison et al., 2007](#B5)). The activity of CYP3A4 was quantified from the rates of itraconazole hydroxylation and testosterone 6*β*-hydroxylation; CYP3A5 activity was quantified by the vincristine M1 formation rate ([Dennison et al., 2007](#B5)). The formation rate of 6*β*-hydroxylation from testosterone has been shown to be 27-fold faster by CYP3A4 than CYP3A5, suggesting that 6*β*-hydroxylation formation could be used to determine CYP3A4 activity in HLMs ([Leeder et al., 2005](#B15)). Likewise, OH-itrazonazole formation has been previously shown to be CYP3A4 but not CYP3A5-mediated in supersomes ([Isoherranen et al., 2004](#B12)). The 20 HLMs were incubated with 1 *μ*M vinorelbine with 0.8 mg/ml of protein for 12 minutes as described earlier. Each sample was run in triplicate, and the controls were incubated without an NADPH addition. The rate of vinorelbine depletion (and metabolism) was calculated for each HLM from the pmoles of parent drug lost per minute per mg of microsomal protein. The sample preparation was as described earlier.

#### HLM Incubations Using Selective P450 Inhibitors.

A pool of HLMs was incubated with vinorelbine and selective inhibitors of P450s above *K*_i_ concentrations as previously described to determine contributions of the major human drug metabolizing P450s to vinorelbine metabolism ([Dennison et al., 2007](#B5)). Vinorelbine at 1 *μ*M was incubated with omeprazole (CYP2C19; 10 *μ*M), quinidine (CYP2D6; 1 *μ*M), trimethoprim (CYP2C8; 50 *μ*M), sulfaphenazole (CYP2C9; 10 *μ*M), ketoconazole (CYP3A; 1 *μ*M), *α*-naphthoflavone (CYP1A2; 1 *μ*M), pilocarpine (CYP2A6; 50 *μ*M), diethyldithiocarbamate (CYP2E1; 50 *μ*M), and thioTEPA (CYP2B6; 50 *μ*M) for 12 minutes ([Rodrigues, 1999](#B24); [Bapiro et al., 2001](#B2); [Dennison et al., 2007](#B5)) with 0.8 mg/ml of protein from pooled HLMs. The HLM pool consisted of five livers pooled to be equally represented on the basis of total microsomal protein concentration; two of the five HLMs contained high concentrations of CYP3A5. Each incubation was run in triplicate and compared with controls that lacked inhibitor or both inhibitor and NADPH. Sample preparation was as described earlier.

#### Selective Inhibition of CYP3A4-Mediated Metabolism of Vinorelbine by Cyclosporine A in Supersomes and HLMs.

Recombinant CYP3A4+*b*_5_ and CYP3A5+*b*_5_ were incubated with vinorelbine and cyclosporine A (CsA) to confirm selectivity of CYP3A4 versus CYP3A5-mediated vinorelbine metabolism. Two human liver microsomes with similarly high CYP3A4 activities (>50% drug depletion under incubation conditions of 0.16 mg HLM protein and 1 *μ*M vinorelbine for 12 minutes) were selected for this inhibition study. An HLM with high CYP3A5 protein content (IUL-73) and one with low protein content (IUL-55) were incubated in triplicate with vinorelbine and 25 *μ*M CsA at 0.3% MeOH (v/v). Controls contained 0.3% MeOH (v/v) and no CsA. Samples were incubated with 1 *μ*M vinorelbine for 12 minutes with either 30 pmol/ml of recombinant enzyme or 0.16 mg HLM protein.

#### High-Pressure Liquid Chromatography Analysis.

High-pressure liquid chromatography (HPLC) was used to quantify the parent drug depletion and metabolite formation. The HPLC system consisted of a high pressure gradient binary pump and autosampler (1100 series; Agilent Technologies, Wilmington, DE) and an UV absorbance detector (1050 series; Hewlett Packard, Wilmington, DE). Postincubation samples (as described earlier) were diluted with an equal volume of 0.2% formic acid and injected onto a reverse phase column (C_18_ column, Inertsil ODS3, 3.0 × 150 mm, 5-*µ*m particle size; MetaChem Technologies Inc., Torrance, CA). The mobile phases A and B consisted of 0.2% formic acid in MeOH with ratios of 80:20 and 20:80, respectively, flowing at a rate of 0.4 ml/min. A linear gradient over 25 minutes was used for simple drug depletion quantification: 0 minutes/0% B, 20 minutes/100% B, 20.1 minutes/0% B. A longer linear gradient over 75 minutes was used to distinguish metabolites formed: 0 minutes/0% B, 7 minutes/0% B, 57.1 minutes/70% B, 57.2 minute/100% B, 67.2 minutes/100% B, 67.3 minutes/0% B. The parent drug, metabolites, and internal standard were detected by UV absorbance at a wavelength of 254 nm.

#### Radiolabeled Metabolite Analysis.

Generally tritiated vinorelbine was purified using a HPLC no more than a day before incubations. ^3^H vinorelbine [2 × 10^6^ disintegrations per minute (dpm); 0.9 *µ*M + 5 *µ*M cold vinorelbine] was incubated for 40 minute with recombinant CYP3A4+*b*_5_ and CYP3A5+*b*_5_ enzymes (100 pmol/ml), and for 20 minute with IUL-73 (0.5 mg, high CYP3A5 protein content) and IUL-55 (0.5 mg, low CYP3A5 protein content). Incubation time and enzyme content were chosen to ensure extensive metabolite formation. Samples were injected neat and manually onto the HPLC; aliquots of eluant were collected every 20 seconds into 7 ml scintillation vials. Scintillation fluid was added for a total volume of approximately 5 ml. Radioactive dpm were quantified using a liquid scintillation counter (Tri-Carb 2100TR; Packard Instrument Company, Meriden, CT) with quench correction.

#### Metabolite Identification.

Metabolite structures were identified using liquid chromatography–MS/MS and NMR analysis as described previously elsewhere ([Dennison et al., 2006](#B6)). Vinorelbine metabolites formed by CYP3A4+*b*_5_ were isolated using a C_18_ column (Phenomenex 150 × 4.60 mm, 5-*µ*m particle size; Phenomenex, Inc., Torrance, CA). NMR samples were dissolved in ∼150 *µ*l of dimethylsulfoxide-*d*_6_ and transferred to a Wilmad 335 NMR tube. Data were acquired on a Varian INOVA 500 NMR spectrometer with a Varian gradient, triple-resonance cold probe. The suite of experiments included a standard proton, two-dimensional homonuclear gradient-selected correlation spectroscopy (gCOSY) and total correlation spectroscopy, direct-correlation multiplicity-edited heteronuclear single quantum coherence adiabatic (HSQCAD), and long-range Heteronuclear Multiple Bond Correlation coherence adiabatic (gHMBCAD). Water suppression for all experiments was accomplished with the wet1D element at the beginning of each pulse sequence. Data were acquired at 25°C and referenced to the solvent at 2.49 ppm for ^1^H and 39.5 ppm for ^13^C.

#### Data Analyses.

Michaelis-Menten constants (*K*_m_) and maximal rates of metabolism (*V*_max_) for each enzyme were determined by fitting the data with nonlinear least squares regression (WinNonlin 4.0; Pharsight, Mountain View, CA). The intrinsic clearance (CL_int_) of vinorelbine for CYP3A4+*b*_5_ and CYP3A5+*b*_5_ was calculated ([eq. 1](#eq1)).

|   | (1) |
| --- | --- |
The intrinsic clearance of vinorelbine (CL_int_) in HLMs was predicted from the individual intrinsic clearances of expressed CYP3A4+b_5_ and CYP3A5+*b*_5_ and the previously determined enzyme concentrations in each HLM ([eq. 2](#eq2)). This approach assumes that the only difference between cDNA expressed P450 and microsomal P450 is in abundance and was previously successful in the case of vincristine ([Dennison et al., 2007](#B5)). The average CYP3A4 and CYP3A5 (expressers only) concentration was 51.89 ± 38.5 pmol/mg and 29.05 ± 24.7 pmol/mg, respectively ([Dennison et al., 2007](#B5)).

|   | (2) |
| --- | --- |
The specific contribution of CYP3A5 to the intrinsic clearance of vinorelbine was estimated by subtracting the CYP3A4 component using the relationship between CYP3A4 activity (hydroxy-itraconazole formation) and rate of vinorelbine metabolism in low expressers of CYP3A5 ([Dennison et al., 2007](#B5)). The two-sided Student *t* test using was used to compare the velocities of vinorelbine metabolism by supersomes and in microsomes that contained either high or low contents of CYP3A5. *P* < 0.05 was considered statistically significant.

## Results

### 

#### Determination of P450 Enzymes That Contribute to Vinorelbine Metabolism by Recombinant Enzymes and HLMs.

Incubations with a panel of P450s ([Fig. 1A](#F1)) indicated that vinorelbine was selectively metabolized by CYP3A4+*b*_5_ (43.9% depletion) and CYP3A5+*b*_5_ (43.6% depletion).

![Fig. 1.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8584/3876810/c8401befdb1e/dmd.113.051094f1.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3876810_dmd.113.051094f1.jpg)

(A) Percentage of depletion from control for a panel of cytochrome P450 enzymes incubated with vinorelbine. Incubations were performed in duplicate, using 5 µM vinorelbine and 200 pmol/ml of P450 for 60 minutes. The control was insect cell protein that did not express P450 enzymes. CYP3A4 and CYP3A5 were the only enzymes that showed a statistically significant difference in vinorelbine depletion from the control (*P < 0.001). (B) Remaining percentage of active depletion of vinorelbine by HLMs normalized to control after chemical inhibition by hepatic P450s. Vinorelbine (1 µM) was incubated for 12 minutes with 0.8 mg/ml of protein from pooled HLMs and each chemical inhibitor at their respective Ki concentrations. The control contained no chemical inhibitors. Only the inhibition of CYP3A by ketoconazole (1 µM) showed significantly diminished activity (*P < 0.001). (C) Inhibition of vinorelbine metabolism by cyclosporine A in CYP3A4 and CYP3A5 supersomes and HLMs with high CYP3A5 (IUL-73) and low CYP3A5 (IUL-55) expression. (***P < 0.001).

A pooled set of HLMs was incubated with vinorelbine (1 *μ*M) and selective P450 inhibitors to determine contribution of each enzyme to vinorelbine metabolism ([Fig. 1B](#F1)). The percentage of remaining relative to no inhibitor control was: omeprazole, 89.6% (CYP2C19; 10 *μ*M); quinidine, 99.1% (CYP2D6; 1 *μ*M); trimethoprim, 94.4% (CYP2C8; 50 *μ*M); sulfaphenazole, 117.5% (CYP2C9; 10 *μ*M); ketoconazole, 5.1% (CYP3A4/5; 1 *μ*M); *α*-naphthoflavone, 96.5% (CYP1A2; 1 *μ*M); pilocarpine, 94.5% (CYP2A6; 50 *μ*M); diethyldithiocarbamate, 102.4% (CYP2E1; 50 *μ*M); and thioTEPA = 108.0% (CYP2B6; 50 *μ*M). These data are consistent with the cDNA-expressed enzyme panel and indicate that only CYP3A enzymes are involved in vinorelbine microsomal oxidation.

CsA (25 *µ*M) inhibited >80% of CYP3A4-mediated vinorelbine metabolism but only 20% of CYP3A5-mediated vinorelbine metabolism in supersomes ([Fig. 1C](#F1)). These data indicate that CsA (25 *µ*M) can be used to selectively inhibit CYP3A4-mediated vinorelbine metabolism in HLMs. There was no difference in vinorelbine depletion in HLMs with comparable CYP3A4 but high (IUL-73) and low (IUL-55) CYP3A5 contents (26 and 28% remaining activity compared with no-inhibitor controls, respectively) ([Fig. 1C](#F1)).

#### Determination of Michaelis-Menten Parameters of Vinorelbine Metabolism by Recombinant CYP3A4 and CYP3A5.

The percentage of vinorelbine depleted at each substrate concentration point after incubating with CYP3A4+*b*_5_ and CYP3A5+*b*_5_ supersomes was statistically significantly different compared with the −NADPH controls ([Supplemental Table 1](http://dmd.aspetjournals.org/lookup/suppl/doi:10.1124/dmd.113.051094/-/DC1)). CYP3A4+*b*_5_ and CYP3A5+*b*_5_ were found to have low *K*_m_ values (CYP3A4 2.63 ± 0.48 *μ*M, CYP3A5 3.64 ± 0.96 *μ*M) and similar *V*_max_ values [CYP3A4 1.37 ± 0.09 pmol vinorelbine (VRL)/min/pmol P450, CYP3A5 1.40 ± 0.15 pmol VRL/min/pmol P450] ([Fig. 2, A and B](#F2); [Table 1](#T1)). The corresponding intrinsic clearances were 0.52 ± 0.06 *μ*l/min/pmol P450 and 0.38 ± 0.06 *μ*l/min/pmol P450 for CYP3A4 and CYP3A5 respectively ([Table 1](#T1)). The *K*_m_, *V*_max_, and CL_int_ vinorelbine metabolism by CYP3A4 and CYP3A5 did not statistically significantly differ (*P* > 0.05 for each parameter).

![Fig. 2.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8584/3876810/bf74c8258cad/dmd.113.051094f2.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3876810_dmd.113.051094f2.jpg)

(A) Vinorelbine depletion rate plotted against vinorelbine concentration when incubated with CYP3A4+b5 (○) or CYP3A5+b5 (●). (B) Eadie-Hofstee plots of CYP3A4+b5 (○) or CYP3A5+b5 (●)-mediated metabolism of vinorelbine. All incubations were performed in triplicate with 30 pmol/ml CYP3A4+b5 or CYP3A5+b5 for 5 minutes. Lines represent the best fit to the Michaelis-Menten equation for vinorelbine metabolism by CYP3A4+b5 (dotted line) or CYP3A5+b5 (solid line) as determined using WinNonLin.

|   | Km | Vmax | CLint |
| --- | --- | --- | --- |
|   | µM | pmol VRL/min/pmol P450 | µl/min/pmol P450 |
| CYP3A4 | 2.63 ± 0.48 | 1.37 ± 0.09 | 0.52 ± 0.06 |
| CYP3A5 | 3.64 ± 0.96 | 1.40 ± 0.15 | 0.38 ± 0.06 |

Table Caption: Michaelis-Menten parameters for vinorelbine metabolism by CYP3A4 and CYP3A5 supersomes.

In this study, the presence of coexpressed cytochrome *b*_5_ greatly increased the overall rate of vinorelbine depletion. When incubated with vinorelbine, CYP3A5 without cytochrome *b*_5_ metabolized the parent at one-third of the rate of CYP3A5+*b*_5_; CYP3A4 without cytochrome *b*_5_ did not detectably metabolize vinorelbine (unpublished data).

#### Vinorelbine Metabolism in Human Liver Microsomes Expressing High and Low CYP3A5 Content.

To determine the relative contributions of CYP3A4 and CYP3A5 to the metabolism of vinorelbine, HLMs (*n* = 20) previously characterized for CYP3A activities were incubated with vinorelbine. There was a positive association between the rate of vinorelbine depletion (pmol vinorelbine depletion/min/mg protein) and two highly selective CYP3A4 activities, itraconazole hydroxylation (*r*^2^ = 0.93, *P* < 0.001) and testosterone 6*β*-hydroxylation (*r*^2^ = 0.78, *P* < 0.001) as shown ([Fig. 3, A and B](#F3)). The rate of vinorelbine depletion was highly correlated (*r*^2^ = 0.89, *P* < 0.001) with the rate of vincristine M1 formation in CYP3A5 low expressers ([Fig. 3D](#F3)), but these activities were weakly correlated (*r*^2^ = 0.23, *P* = 0.224) in high CYP3A5 expressers in which both CYP3A4 and CYP3A5 contribute to vincristine M1 formation ([Fig. 3C](#F3)). This correlation analysis indicates that in HLMs, CYP3A4 is primary enzyme responsible for the depletion of vinorelbine.

![Fig. 3.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8584/3876810/229945b23516/dmd.113.051094f3.jpg)

Relationships between rate of vinorelbine metabolism by HLMs and the rates of hydroxy-itraconazole (OH-ITZ) formation from itraconazole (A; r2 = 0.93, P < 0.001), testosterone 6β -hydroxylation (B; r2 = 0.78, P < 0.001) and vincristine M1 formation in HLMs with high CYP3A5 protein content (C; r2 = 0.23, P = 0.224) and low protein content (D; r2 = 0.89, P < 0.001). Microsomes are designated by CYP3A5 protein content: high (●) and low (○). All incubations were performed in triplicate; 1 μM vinorelbine and 0.8 mg/ml protein content per HLM were incubated for 12 minutes.

#### Predicted Hepatic Clearance of Vinorelbine.

The observed mean (± S.D.) rate of vinorelbine depletion by CYP3A5 high expressers (31.0 ± 18.7 pmol VRL/min/mg HLM) and low expressers (31.7 ± 18.9 pmol VRL/min/mg HLM) were not statistically different (*P* = 0.941; [Table 2](#T2)). The corresponding CL_int_ values (after accounting for total microsomal protein in a 70-kg man) were 2546.7 ± 1534.2 ml/min and 2599.9 ± 1548.2 ml/min for CYP3A5 high expressers and low expressers, respectively. The relationship between the intrinsic clearance of vinorelbine depletion and itraconazole hydroxylation in low expressers was used to determine the contribution of CYP3A5 to vinorelbine intrinsic clearance in high expressers as previously described for vincristine ([Fig. 4](#F4)) ([Dennison et al., 2007](#B5)). The clearance of vinorelbine depletion by CYP3A5 after correcting for CYP3A4 contribution was 2.5 ± 4.7 (*µ*l/min/mg protein) in high expressers, and this was not statistically significantly different to zero (*P* = 0.27).

| HLM | CYP3A5 Genotypea | Gender | CYP3A4 Protein Content | CYP3A5 Protein Content | v [1 µM Vinorelbine] | Intrinsic Clearanceb | Hepatic Clearancec |
| --- | --- | --- | --- | --- | --- | --- | --- |
|   |   |   | pmol/mg |  | pmol/min/mg | ml/min |  |
| CYP3A5 high expressers |  |  |  |  |  |  |  |
| IUL-40 | *1/*1 | M | 103.8 | 89.4 | 48.2 | 3952.3 | 337.1 |
| IUL-41 | *1/*3 | F | 14.7 | 20.5 | 33.1 | 2716.9 | 249.2 |
| IUL-42 | *1/*3 | F | 23.7 | 16.1 | 17.2 | 1414.9 | 141.0 |
| IUL-59 | *1/*3 | NA | 36.3 | 14.7 | 29.0 | 2379.3 | 222.8 |
| IUL-66 | *1/*3 | M | 24.4 | 20.6 | 24.4 | 1999.5 | 191.8 |
| IUL-73 | *1/*3 | F | 96.1 | 20.1 | 67.6 | 5546.3 | 433.7 |
| IUL-74 | *1/*6 | F | 10.7 | 26.2 | 11.2 | 915.4 | 94.4 |
| IUL-85 | *1/*3 | M | 85.5 | 24.8 | 17.7 | 1448.9 | 144.1 |
| CYP3A5 low expressers |  |  |  |  |  |  |  |
| IUL-6 | *3/*3 | NA | 19.0 | 0.8 | 13.9 | 1137.1 | 115.5 |
| IUL-49 | *3/*3 | M | 48.8 | 0.8 | 28.9 | 2370.3 | 222.1 |
| IUL-52 | *3/*3 | M | 14.8 | 0.0 | 10.3 | 846.2 | 87.6 |
| IUL-55 | *3/*3 | NA | 130.4 | 0.8 | 57.6 | 4722.5 | 385.8 |
| IUL-57 | *1/*3 | M | 40.6 | 0.8 | 19.5 | 1598.6 | 157.4 |
| IUL-65 | *3/*3 | M | 15.6 | 0.6 | 15.0 | 1232.8 | 124.4 |
| IUL-71 | *1/*7 | F | 10.9 | 1.1 | 16.7 | 1371.5 | 137.1 |
| IUL-72 | *3/*3 | F | 25.3 | 0.9 | 17.5 | 1434.7 | 142.8 |
| IUL-75 | *3/*3 | NA | 78.2 | 2.3 | 46.2 | 3788.5 | 326.1 |
| IUL-78 | *3/*3 | F | 109.5 | 3.1 | 52.4 | 4296.9 | 359.4 |
| IUL-81 | *3/*3 | M | 71.7 | 2.0 | 40.8 | 3350.7 | 295.9 |
| IUL-86 | *3/*3 | NA | 77.8 | 2.1 | 61.5 | 5049.2 | 405.3 |
| Mean |   |   |   |   |   |   |   |
| CYP3A5 high expressers |   |   |   |   | 31.0 ± 18.7 | 2546.7 ± 1534.2 | 226.8 ± 112.4 |
| CYP3A5 low expressers |   |   |   |   | 31.7 ± 18.9 | 2599.9 ± 1548.2 | 229.9 ± 117.4 |
| P value |   |   |   |   | 0.941 | 0.941 | 0.952 |

Table Caption: Genotypes of HLMs and vinorelbine metabolism rates

![Fig. 4.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8584/3876810/5b635a48ef07/dmd.113.051094f4.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3876810_dmd.113.051094f4.jpg)

Formation of hydroxy-itraconazole (OH-ITZ) (µl/min/mg protein) plotted against vinorelbine clearance (µl/min/mg protein) between HLMs with high (●) and low (○) CYP3A5 protein content. Regression line and 95% confidence intervals represent the correlation between HLMs with low CYP3A5 content (○), in which clearance is predominantly CYP3A4-mediated. The majority of HLMs regardless of CYP3A5 expression fell within the 95% confidence intervals. After correcting for CYP3A4 contribution, CYP3A5 did not significantly contribute to vinorelbine clearance in HLMs.

The predicted rates of vinorelbine depletion in the HLMs were calculated using the additive CL_int_ model for cDNA-expressed CYP3A4+*b*_5_ and CYP3A5+*b*_5_ ([eq. 2](#eq2)) and were found to be 1.3-fold higher for the CYP3A5 high expressers (high, 27.4 ± 19.9 pmol VRL/min/mg HLM; low, 20.6 ± 15.2 pmol VRL/min/mg HLM) but not statistically significantly different (*P* = 0.428) ([Table 2](#T2)). The predicted clearance using the cDNA-expressed enzyme Michaelis-Menten parameters overpredicted the contribution of CYP3A5 (predicted to be 36% higher than low CYP3A5-expressing HLMs) to the metabolism of vinorelbine in HLMs.

#### Radiolabeled Metabolite Detection.

Recombinant enzymes CYP3A4+*b*_5_, CYP3A5+*b*_5_, and an insect control were incubated with ^3^H vinorelbine to detect metabolites by liquid scintillation count. Three common metabolites were detected after incubations with recombinant enzymes CYP3A4 and CYP3A5. Metabolite 2 (M2) ([Fig. 5, A and B](#F5)) was the most prevalent (as identified by HPLC/MS/MS). In the radiochromatogram depicting metabolism by CYP3A5 ([Fig. 5B](#F5)), M2 had a 3-fold lower peak area (dpm) than for CYP3A4 ([Fig. 5A](#F5)). However, the overall depletion of ^3^H vinorelbine was similar for both (CYP3A4 = 69.4%, CYP3A5 = 71.2%), perhaps suggesting that subsequent metabolism of M2 by CYP3A5 may be occurring.

![Fig. 5.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8584/3876810/f1e7caa43b85/dmd.113.051094f5.jpg)

Radiochromatograms of 3H vinorelbine (dpm) incubated with CYP3A4+b5 (A), CYP3A5+b5 (B), or non-P450-expressing insect control (C) supersomes. Fractions were separated by HPLC, collected in 20-second intervals, and quantified by scintillation count. The relative formation of M2 appears to be 3-fold higher when vinorelbine is incubated with CYP3A4+b5 than with CYP3A5+b5. M2, metabolite 2.

Two HLMs were incubated with ^3^H vinorelbine, both with similarly high CYP3A4 activity and different CYP3A5 protein contents: IUL-55 (minimal CYP3A5 content, [Fig. 6A](#F6)) and IUL-73 (high CYP3A5 content, [Fig. 6B](#F6)). The control was IUL-73 lacking NADPH ([Fig. 6C](#F6)). The overall depletion of ^3^H vinorelbine was similar for both (IUL-55 = 59.4%, IUL-73 = 64.1%), consistent with predominant metabolism of vinorelbine by CYP3A4. In the radiochromatogram depicting vinorelbine metabolism by IUL-55, the M2 peak was 1.5 fold larger than that of the M2 peak formed with IUL-73, similar to the higher ratio of M2 detected within the CYP3A4 versus CYP3A5 supersomes.

![Fig. 6.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8584/3876810/790a8cef90b2/dmd.113.051094f6.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3876810_dmd.113.051094f6.jpg)

Radiochromatograms of 3H vinorelbine (dpm) incubated with HLMs IUL-55 (low CYP3A5 protein content; A), IUL-73 (high CYP3A5 protein content; B), or IUL-73 lacking NADPH as control (C). Fractions were separated by HPLC, collected in 20-second intervals, and quantified by scintillation count. The relative detection of M2 is 1.5-fold higher for IUL-55 than by IUL-73 (M2, metabolite 2). Based upon supersome data, and that both IUL-55 and IUL-73 contained similar amounts of CYP3A4 protein, the difference in M2 detection may be due to subsequent metabolism of M2 by CYP3A5 in IUL-73.

#### Metabolite Structure Identification.

Metabolite structures were identified using MS/MS and NMR analysis. The structures of vinorelbine and the proposed metabolite structures are shown in [Table 3](#T3) along with the product ions and description of the key product ions for vinorelbine. The N-oxide of vinorelbine, M1, was determined from the NMR data. Significant downfield shifts to the carbons adjacent to N4′, compared with parent, clearly indicate the presence of an N-oxide. M1, however, is thought to be a byproduct of the extraction method, as it was not detected when the incubated sample supernatants were analyzed without additional sample preparation.

| ID | M+H+ | MS/MS Product Ions | Proposed Metabolite Structure |
| --- | --- | --- | --- |
| Vinorelbine | 779 | 719,701,510,469,457,323,122 |   |
| M1 | 795 | 735,702,646,526,469,457,339,202,138,122 |   |
| M2 | 777 | 717,686,580,419,321,311 |   |
| M3 | 795 | 735,717,674,642,510.485,379,323,291,122 |   |
| M4 | 811 | 526,508,485,473,397,345,323,202 |   |
| M5 | 781 | 610,455,443,138,122 |   |
| M6 | 795 | 777,717,633,526,457,403,202,122,108 |   |
| M7 | 763 | 704,672,566,535,455,423,321 |   |
| M8 | 777 | 718,717,658,657,467,389 |   |

Table Caption: Vinorelbine and its CYP3A4 metabolites based on liquid chromatography–MS/MS and NMR data

The main metabolite of interest, M2, was present in both the recombinant and microsomal systems. M2 has been identified as a didehydro metabolite. Based on its protonated molecular ion at *m*/*z* 777 and its MS/MS and NMR data, the additional double bond is on the tetrahydropyridyl ring of the “catharanthine-like” side of the molecule. Key MS/MS product ions used to support the structure included *m*/*z* 321, consistent with the catharanthine-like side minus two hydrogens and *m*/*z* 469, consistent with no change to the vindoline side of the molecule. The resultant double bond could only exist in one of three places; between C3′–C14′, C14′–C17′, and C19′–C18′. Although, the exact location of this double bond could not be determined, the 18′–19′ ethyl moiety is clearly observed in the NMR data, indicating that the dehydration occurred with C14′ and an adjacent carbon.

Other more minor metabolites were tentatively identified from their MS/MS product ion data, and their proposed structures are shown in [Table 3](#T3). From the MS/MS data, oxidation occurred on both the vinorelbine and catharanthine-like side of the molecule. Oxidative demethylation was also observed on the vinorelbine moiety.

## Discussion

In this study we tested the hypothesis that vinorelbine would be more efficiently metabolized by CYP3A5 than CYP3A4 in view of the finding that a related compound, vincristine, displayed this unusual selectivity. Similar to vincristine metabolism, only cDNA-expressed CYP3A4 and CYP3A5 were able to metabolize vinorelbine. A library of highly selective P450 inhibitors confirmed that only CYP3A enzymes catalyzed the loss of vinorelbine from HLMs. However, in contrast to vincristine metabolism, CYP3A5 did not selectively metabolize vinorelbine; the two CYP3A enzymes displayed similar efficiencies toward vinorelbine depletion in vitro. Therefore, CYP3A5 expression would not be predicted to affect systemic clearance of vinorelbine.

Consistent with this hypothesis, there was little difference in vinorelbine intrinsic clearance between the high expresser (*CYP3A5 *1/*1*, **1/*3, *1/*6*) and low expresser (*CYP3A5 *3/*3*) genotype HLMs. It is notable that two livers (IUL-57 and IUL-71) were found to be *1 carriers, which would be expected to be high expressers of CYP3A5 hepatic protein; however, these livers with both found to have low expression of CYP3A4 protein and as such were classified as low expressers. CYP3A5*1 carrier status was determined by process of elimination with testing for *3, *6, and *7 included. We anticipate two potential explanations for the low CYP3A5 protein expression in these livers, including: 1) an undetected genetic polymorphism for these cases that promoted loss of CYP3A5 protein expression or 2) perhaps the donors were in critical condition or chronically ill, thus resulting in altered protein production. The predicted human hepatic clearance was essentially the same in high expressers and low expressers. Correlation analysis also indicated that vinorelbine intrinsic clearance was highly associated with CYP3A4 activities in HLMs but not CYP3A5 activity. This is consistent with the expectation that CYP3A5 genotype does significantly impact vinorelbine hepatic clearance.

Recombinant supersome incubations determined CYP3A4 and CYP3A5 to have similar *K*_M_ and *V*_max_ values for vinorelbine. Therefore, we hypothesized that microsomes expressing CYP3A5 protein would have statistically higher drug depletion rates after normalization to CYP3A4 activity. However, despite protein levels of CYP3A5 accounting for 41% of total CYP3A content on average (in HLMs expressing CYP3A5), vinorelbine depletion in HLM was shown to be predominantly mediated by CYP3A4. Vinorelbine metabolism was inhibited to similar extents by CsA in HLMs with high and low CYP3A5 content and similar amounts of CYP3A4 protein ([Fig. 1C](#F1)), suggesting that there was minimal contribution of CYP3A5 to vinorelbine metabolism in the high CYP3A5-expressing HLM.

Our recombinant enzyme data suggest that CYP3A5 may contribute if CYP3A5 protein levels are high enough in the liver. However, this in vitro prediction was probably not correct based on the HLM data. In HLMs with high CYP3A5 (in several cases higher than CYP3A4), the contribution of CYP3A5 was not detected. As such, we would conclude that the contribution of CYP3A5 is insignificant. Together, these data indicate that CYP3A5 expression may not play a clinically significant role in vinorelbine metabolism in vivo, unlike as previously shown with vincristine ([Dennison et al., 2006](#B6)).

Based on the findings of this in vitro investigation, we would not expect vinorelbine metabolism to differ based on the CYP3A5 genotype. Consistent with the results of our study, clinical data from [Wong et al. (2006)](#B27) support a lack of CYP3A5 contribution to clearance of vinorelbine. In contrast, CYP3A5*1 genotype has even been shown to be moderately associated with better overall median survival rates in patients with NSCLC ([Pan et al., 2007](#B19)). In the study of P450 genotypes and clinical outcomes in patients with NSCLC, they found that patients with at least one CYP3A5*1 allele had slower rates of tumor growth and a higher associated median survival rate than did patients without a CYP3A5*1 allele. As such, the patients in the two genotype groups in the NSCLC clinical trial would be expected to have similar vinorelbine clearance rates.

Any number of confounding factors could be contributing to therapeutic efficacy in CYP3A5*1 subjects. One possible explanation for the finding of this clinical trial is that there is an unidentified variable factor between the two genotype groups studied, resulting in a difference in vinorelbine such as volume of distribution, active transport, or non-P450 parallel elimination pathways between groups, resulting in greater vinorelbine exposure in the CYP3A5*1 genotype group. Another possible reason for the difference observed in this study is that CYP3A5 may be sequentially metabolizing an inactive vinorelbine metabolite that competes with vinorelbine at the level of the tumor, resulting in greater vinorelbine exposure at the level of the tumor.

A noticeable difference between the metabolites formed by CYP3A4 and CYP3A5 was apparent when studying the radiochromatograms of metabolite profiles resulting from incubation of the parent drug with individual recombinant enzymes. Metabolite 2 (M2) was observed when vinorelbine was incubated with both the recombinant and microsomal systems ([Figs. 5](#F5) and [6](#F6)). However, formation of M2 was significantly lower in the presence of CYP3A5. In the recombinant enzyme system, M2 formation with CYP3A4 was 3-fold higher than with CYP3A5. The formation of M2 by IUL-55 (low CYP3A5 content) was 1.5-fold higher than by IUL-73 (high CYP3A5 content). By comparing the HLMs with similarly high CYP3A4 activity levels, it was assumed that a similar amount of M2 was formed by CYP3A4. A plausible hypothesis to address this discrepancy of M2 detection between systems expressing or not expressing CYP3A5 could be the sequential metabolism of M2 by CYP3A5.

In this study, MS/MS analysis of in vitro incubations with both recombinant enzymes and HLMs demonstrated M2 (didehydro-vinorelbine on the velbenamine side of the molecule) as the major vinorelbine metabolite formed by CYP3A4. This major metabolite of vinorelbine M2 is distinct from the major CYP3A-mediated metabolite of vincristine M3, an unusual metabolite formed after an intramolecular amidation reaction ([Dennison et al., 2006](#B6)). The molecular rearrangement of vincristine by CYP3A metabolism would not be expected for vinorelbine because of the chemical differences between the molecules on the catharanthine side.

As for other possible metabolites of vinorelbine, previous clinical studies of vinorelbine have shown the presence of the metabolite deacetyl-vinorelbine (deacetyl-navelbine) in the urine ([Nicot et al., 1990](#B18); [Van Heugen et al., 2001](#B26)). However, deacetyl-vinorelbine has only been detected in trace amounts, representing only 0.25% of the vinorelbine dose (30 mg/m^2^). Although it has been detected in trace amounts in vivo within blood and feces ([Leveque and Jehl, 1996](#B16); [Van Heugen et al., 2001](#B26)), vinorelbine N-oxide in our study was determined to be a byproduct of the extraction process (unpublished data) and is therefore not thought to occur via CYP3A4-mediated oxidation. Recombinant P450s and HLMs are only responsible for oxidative metabolism of vinorelbine, so deacetyl-vinorelbine was not detected in this in vitro study. Recently, a number of vinorelbine metabolites, including vinorelbine N-oxide, were identified using liquid chromatography–MS/MS from human biologic fluids and categorized as being found in blood, urine, or feces ([de Graeve et al., 2008](#B4)). A comparison of the metabolites (M13, M7, M10, M5, M15, and M2) identified from human biologic fluids ([de Graeve et al., 2008](#B4)) showed strong similarities with metabolites M2, M3, M4, M6, M7, and M8, respectively ([Table 3](#T3)), identified in this study as being CYP3A4 mediated. These metabolites thus may be biologically relevant, especially M2 because it was the predominant metabolite formed in HLMs lacking CYP3A5.

In conclusion, based on our in vitro findings, CYP3A enzymes are predicted to metabolize vinorelbine, primarily to a didehydro-metabolite M2 ([Table 3](#T3)). However, polymorphic CYP3A5 expression is not expected to significantly contribute to vinorelbine metabolism in vivo because, unlike vincristine, vinorelbine is not selectively metabolized by CYP3A5.

## Supplementary Material

## Abbreviations

## Authorship Contributions

*Participated in research design:* Topletz, Dennison, Hall, Renbarger.

*Conducted experiments:* Topletz, Barbuch, Hadden.

*Contributed new reagents or analytic tools:* Topletz, Barbuch, Hadden, Hall, Renbarger.

*Performed data analysis:* Topletz, Barbuch, Hadden, Dennison, Hall, Renbarger.

*Wrote or contributed to the writing of the manuscript:* Topletz, Dennison, Barbuch, Hall, Renbarger.

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Baker SD, Sparreboom A. (2006) Predicting vinorelbine disposition and toxicity: does BSA provide more than a “bad statistical association”? J Clin Oncol 24:2412–2413  [DOI](https://doi.org/10.1200/JCO.2005.05.4494) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16651645/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Predicting%20vinorelbine%20disposition%20and%20toxicity:%20does%20BSA%20provide%20more%20than%20a%20%E2%80%9Cbad%20statistical%20association%E2%80%9D?&author=SD%20Baker&author=A%20Sparreboom&volume=24&publication_year=2006&pages=2412-2413&pmid=16651645&doi=10.1200/JCO.2005.05.4494&)

2. Bapiro TE, Egnell AC, Hasler JA, Masimirembwa CM. (2001) Application of higher throughput screening (HTS) inhibition assays to evaluate the interaction of antiparasitic drugs with cytochrome P450s. Drug Metab Dispos 29:30–35  [PubMed](https://pubmed.ncbi.nlm.nih.gov/11124226/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Application%20of%20higher%20throughput%20screening%20(HTS)%20inhibition%20assays%20to%20evaluate%20the%20interaction%20of%20antiparasitic%20drugs%20with%20cytochrome%20P450s&author=TE%20Bapiro&author=AC%20Egnell&author=JA%20Hasler&author=CM%20Masimirembwa&volume=29&publication_year=2001&pages=30-35&pmid=11124226&)

3. Beulz-Riché D, Grudé P, Puozzo C, Sautel F, Filaquier C, Riché C, Ratanasavanh D. (2005) Characterization of human cytochrome P450 isoenzymes involved in the metabolism of vinorelbine. Fundam Clin Pharmacol 19:545–553  [DOI](https://doi.org/10.1111/j.1472-8206.2005.00367.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16176333/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Fundam%20Clin%20Pharmacol&title=Characterization%20of%20human%20cytochrome%20P450%20isoenzymes%20involved%20in%20the%20metabolism%20of%20vinorelbine&author=D%20Beulz-Rich%C3%A9&author=P%20Grud%C3%A9&author=C%20Puozzo&author=F%20Sautel&author=C%20Filaquier&volume=19&publication_year=2005&pages=545-553&pmid=16176333&doi=10.1111/j.1472-8206.2005.00367.x&)

4. de Graeve J, van Heugen JC, Zorza G, Fahy J, Puozzo C. (2008) Metabolism pathway of vinorelbine (Navelbine) in human: characterisation of the metabolites by HPLC-MS/MS. J Pharm Biomed Anal 47:47–58  [DOI](https://doi.org/10.1016/j.jpba.2007.12.006) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18242035/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharm%20Biomed%20Anal&title=Metabolism%20pathway%20of%20vinorelbine%20(Navelbine)%20in%20human:%20characterisation%20of%20the%20metabolites%20by%20HPLC-MS/MS&author=J%20de%20Graeve&author=JC%20van%20Heugen&author=G%20Zorza&author=J%20Fahy&author=C%20Puozzo&volume=47&publication_year=2008&pages=47-58&pmid=18242035&doi=10.1016/j.jpba.2007.12.006&)

5. Dennison JB, Jones DR, Renbarger JL, Hall SD. (2007) Effect of CYP3A5 expression on vincristine metabolism with human liver microsomes. J Pharmacol Exp Ther 321:553–563  [DOI](https://doi.org/10.1124/jpet.106.118471) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17272675/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharmacol%20Exp%20Ther&title=Effect%20of%20CYP3A5%20expression%20on%20vincristine%20metabolism%20with%20human%20liver%20microsomes&author=JB%20Dennison&author=DR%20Jones&author=JL%20Renbarger&author=SD%20Hall&volume=321&publication_year=2007&pages=553-563&pmid=17272675&doi=10.1124/jpet.106.118471&)

6. Dennison JB, Kulanthaivel P, Barbuch RJ, Renbarger JL, Ehlhardt WJ, Hall SD. (2006) Selective metabolism of vincristine in vitro by CYP3A5. Drug Metab Dispos 34:1317–1327  [DOI](https://doi.org/10.1124/dmd.106.009902) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16679390/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Selective%20metabolism%20of%20vincristine%20in%20vitro%20by%20CYP3A5&author=JB%20Dennison&author=P%20Kulanthaivel&author=RJ%20Barbuch&author=JL%20Renbarger&author=WJ%20Ehlhardt&volume=34&publication_year=2006&pages=1317-1327&pmid=16679390&doi=10.1124/dmd.106.009902&)

7. Egbelakin A, Ferguson MJ, MacGill EA, Lehmann AS, Topletz AR, Quinney SK, Li L, McCammack KC, Hall SD, Renbarger JL. (2011) Increased risk of vincristine neurotoxicity associated with low CYP3A5 expression genotype in children with acute lymphoblastic leukemia. Pediatr Blood Cancer 56:361–367  [DOI](https://doi.org/10.1002/pbc.22845) | [PMC free article](/articles/PMC3020258/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21225912/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pediatr%20Blood%20Cancer&title=Increased%20risk%20of%20vincristine%20neurotoxicity%20associated%20with%20low%20CYP3A5%20expression%20genotype%20in%20children%20with%20acute%20lymphoblastic%20leukemia&author=A%20Egbelakin&author=MJ%20Ferguson&author=EA%20MacGill&author=AS%20Lehmann&author=AR%20Topletz&volume=56&publication_year=2011&pages=361-367&pmid=21225912&doi=10.1002/pbc.22845&)

8. Gauvin A, Pinguet F, Culine S, Astre C, Cupissol D, Bressolle F. (2002a) Blood and plasma pharmacokinetics of vinorelbine in elderly patients with advanced metastatic cancer. Cancer Chemother Pharmacol 49:48–56  [DOI](https://doi.org/10.1007/s00280-001-0378-2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11855752/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Chemother%20Pharmacol&title=Blood%20and%20plasma%20pharmacokinetics%20of%20vinorelbine%20in%20elderly%20patients%20with%20advanced%20metastatic%20cancer&author=A%20Gauvin&author=F%20Pinguet&author=S%20Culine&author=C%20Astre&author=D%20Cupissol&volume=49&publication_year=2002&pages=48-56&pmid=11855752&doi=10.1007/s00280-001-0378-2&)

9. Gauvin A, Pinguet F, Culine S, Astre C, Gomeni R, Bressolle F. (2002b) A limited-sampling strategy to estimate individual pharmacokinetic parameters of vinorelbine in elderly patients with advanced metastatic cancer. Anticancer Drugs 13:473–480  [DOI](https://doi.org/10.1097/00001813-200206000-00005) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12045458/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Anticancer%20Drugs&title=A%20limited-sampling%20strategy%20to%20estimate%20individual%20pharmacokinetic%20parameters%20of%20vinorelbine%20in%20elderly%20patients%20with%20advanced%20metastatic%20cancer&author=A%20Gauvin&author=F%20Pinguet&author=S%20Culine&author=C%20Astre&author=R%20Gomeni&volume=13&publication_year=2002&pages=473-480&pmid=12045458&doi=10.1097/00001813-200206000-00005&)

10. Gorski JC, Jones DR, Haehner-Daniels BD, Hamman MA, O’Mara EM, Jr, Hall SD. (1998) The contribution of intestinal and hepatic CYP3A to the interaction between midazolam and clarithromycin. Clin Pharmacol Ther 64:133–143  [DOI](https://doi.org/10.1016/S0009-9236(98)90146-1) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9728893/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=The%20contribution%20of%20intestinal%20and%20hepatic%20CYP3A%20to%20the%20interaction%20between%20midazolam%20and%20clarithromycin&author=JC%20Gorski&author=DR%20Jones&author=BD%20Haehner-Daniels&author=MA%20Hamman&author=EM%20O%E2%80%99Mara&volume=64&publication_year=1998&pages=133-143&pmid=9728893&doi=10.1016/S0009-9236(98)90146-1&)

11. Hustert E, Haberl M, Burk O, Wolbold R, He YQ, Klein K, Nuessler AC, Neuhaus P, Klattig J, Eiselt R, et al. (2001) The genetic determinants of the CYP3A5 polymorphism. Pharmacogenetics 11:773–779  [DOI](https://doi.org/10.1097/00008571-200112000-00005) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11740341/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=The%20genetic%20determinants%20of%20the%20CYP3A5%20polymorphism&author=E%20Hustert&author=M%20Haberl&author=O%20Burk&author=R%20Wolbold&author=YQ%20He&volume=11&publication_year=2001&pages=773-779&pmid=11740341&doi=10.1097/00008571-200112000-00005&)

12. Isoherranen N, Kunze KL, Allen KE, Nelson WL, Thummel KE. (2004) Role of itraconazole metabolites in CYP3A4 inhibition. Drug Metab Dispos 32:1121–1131  [DOI](https://doi.org/10.1124/dmd.104.000315) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15242978/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Role%20of%20itraconazole%20metabolites%20in%20CYP3A4%20inhibition&author=N%20Isoherranen&author=KL%20Kunze&author=KE%20Allen&author=WL%20Nelson&author=KE%20Thummel&volume=32&publication_year=2004&pages=1121-1131&pmid=15242978&doi=10.1124/dmd.104.000315&)

13. Kuehl P, Zhang J, Lin Y, Lamba J, Assem M, Schuetz J, Watkins PB, Daly A, Wrighton SA, Hall SD, et al. (2001) Sequence diversity in CYP3A promoters and characterization of the genetic basis of polymorphic CYP3A5 expression. Nat Genet 27:383–391  [DOI](https://doi.org/10.1038/86882) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11279519/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Sequence%20diversity%20in%20CYP3A%20promoters%20and%20characterization%20of%20the%20genetic%20basis%20of%20polymorphic%20CYP3A5%20expression&author=P%20Kuehl&author=J%20Zhang&author=Y%20Lin&author=J%20Lamba&author=M%20Assem&volume=27&publication_year=2001&pages=383-391&pmid=11279519&doi=10.1038/86882&)

14. Lamba JK, Lin YS, Schuetz EG, Thummel KE. (2002) Genetic contribution to variable human CYP3A-mediated metabolism. Adv Drug Deliv Rev 54:1271–1294  [DOI](https://doi.org/10.1016/s0169-409x(02)00066-2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12406645/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Adv%20Drug%20Deliv%20Rev&title=Genetic%20contribution%20to%20variable%20human%20CYP3A-mediated%20metabolism&author=JK%20Lamba&author=YS%20Lin&author=EG%20Schuetz&author=KE%20Thummel&volume=54&publication_year=2002&pages=1271-1294&pmid=12406645&doi=10.1016/s0169-409x(02)00066-2&)

15. Leeder JS, Gaedigk R, Marcucci KA, Gaedigk A, Vyhlidal CA, Schindel BP, Pearce RE. (2005) Variability of CYP3A7 expression in human fetal liver. J Pharmacol Exp Ther 314:626–635  [DOI](https://doi.org/10.1124/jpet.105.086504) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15845858/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharmacol%20Exp%20Ther&title=Variability%20of%20CYP3A7%20expression%20in%20human%20fetal%20liver&author=JS%20Leeder&author=R%20Gaedigk&author=KA%20Marcucci&author=A%20Gaedigk&author=CA%20Vyhlidal&volume=314&publication_year=2005&pages=626-635&pmid=15845858&doi=10.1124/jpet.105.086504&)

16. Levêque D, Jehl F. (1996) Clinical pharmacokinetics of vinorelbine. Clin Pharmacokinet 31:184–197  [DOI](https://doi.org/10.2165/00003088-199631030-00003) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8877249/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacokinet&title=Clinical%20pharmacokinetics%20of%20vinorelbine&author=D%20Lev%C3%AAque&author=F%20Jehl&volume=31&publication_year=1996&pages=184-197&pmid=8877249&doi=10.2165/00003088-199631030-00003&)

17. Longo DL, Young RC, Wesley M, Hubbard SM, Duffey PL, Jaffe ES, DeVita VT., Jr (1986) Twenty years of MOPP therapy for Hodgkin’s disease. J Clin Oncol 4:1295–1306  [DOI](https://doi.org/10.1200/JCO.1986.4.9.1295) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/3528400/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Twenty%20years%20of%20MOPP%20therapy%20for%20Hodgkin%E2%80%99s%20disease&author=DL%20Longo&author=RC%20Young&author=M%20Wesley&author=SM%20Hubbard&author=PL%20Duffey&volume=4&publication_year=1986&pages=1295-1306&pmid=3528400&doi=10.1200/JCO.1986.4.9.1295&)

18. Nicot G, Lachatre G, Marquet P, Bonnaud F, Vallette JP, Rocca JL. (1990) High-performance liquid chromatographic determination of navelbine in human plasma and urine. J Chromatogr A 528:258–266  [DOI](https://doi.org/10.1016/s0378-4347(00)82384-1) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/2384562/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Chromatogr%20A&title=High-performance%20liquid%20chromatographic%20determination%20of%20navelbine%20in%20human%20plasma%20and%20urine&author=G%20Nicot&author=G%20Lachatre&author=P%20Marquet&author=F%20Bonnaud&author=JP%20Vallette&volume=528&publication_year=1990&pages=258-266&pmid=2384562&doi=10.1016/s0378-4347(00)82384-1&)

19. Pan JH, Han JX, Wu JM, Sheng LJ, Huang HN. (2007) CYP450 polymorphisms predict clinic outcomes to vinorelbine-based chemotherapy in patients with non-small-cell lung cancer. Acta Oncol 46:361–366  [DOI](https://doi.org/10.1080/02841860600902197) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17450472/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Acta%20Oncol&title=CYP450%20polymorphisms%20predict%20clinic%20outcomes%20to%20vinorelbine-based%20chemotherapy%20in%20patients%20with%20non-small-cell%20lung%20cancer&author=JH%20Pan&author=JX%20Han&author=JM%20Wu&author=LJ%20Sheng&author=HN%20Huang&volume=46&publication_year=2007&pages=361-366&pmid=17450472&doi=10.1080/02841860600902197&)

20. Pan JH, Han JX, Wu JM, Sheng LJ, Huang HN, Yu QZ. (2008) MDR1 single nucleotide polymorphisms predict response to vinorelbine-based chemotherapy in patients with non-small cell lung cancer. Respiration 75:380–385 DOI: 10.1159/000108407.  [DOI](https://doi.org/10.1159/000108407) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17851225/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Respiration&title=MDR1%20single%20nucleotide%20polymorphisms%20predict%20response%20to%20vinorelbine-based%20chemotherapy%20in%20patients%20with%20non-small%20cell%20lung%20cancer&author=JH%20Pan&author=JX%20Han&author=JM%20Wu&author=LJ%20Sheng&author=HN%20Huang&volume=75&publication_year=2008&pages=380-385&pmid=17851225&doi=10.1159/000108407&)

21. Pollock BH, DeBaun MR, Camitta BM, Shuster JJ, Ravindranath Y, Pullen DJ, Land VJ, Mahoney DH, Jr, Lauer SJ, Murphy SB. (2000) Racial differences in the survival of childhood B-precursor acute lymphoblastic leukemia: a Pediatric Oncology Group Study. J Clin Oncol 18:813–823  [DOI](https://doi.org/10.1200/JCO.2000.18.4.813) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10673523/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Racial%20differences%20in%20the%20survival%20of%20childhood%20B-precursor%20acute%20lymphoblastic%20leukemia:%20a%20Pediatric%20Oncology%20Group%20Study&author=BH%20Pollock&author=MR%20DeBaun&author=BM%20Camitta&author=JJ%20Shuster&author=Y%20Ravindranath&volume=18&publication_year=2000&pages=813-823&pmid=10673523&doi=10.1200/JCO.2000.18.4.813&)

22. Qian J, Wang Y, Chang J, Zhang J, Wang J, Hu X. (2011) Rapid and sensitive determination of vinorelbine in human plasma by liquid chromatography-tandem mass spectrometry and its pharmacokinetic application. J Chromatogr B Analyt Technol Biomed Life Sci 879:662–668  [DOI](https://doi.org/10.1016/j.jchromb.2011.01.039) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21342795/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Chromatogr%20B%20Analyt%20Technol%20Biomed%20Life%20Sci&title=Rapid%20and%20sensitive%20determination%20of%20vinorelbine%20in%20human%20plasma%20by%20liquid%20chromatography-tandem%20mass%20spectrometry%20and%20its%20pharmacokinetic%20application&author=J%20Qian&author=Y%20Wang&author=J%20Chang&author=J%20Zhang&author=J%20Wang&volume=879&publication_year=2011&pages=662-668&pmid=21342795&doi=10.1016/j.jchromb.2011.01.039&)

23. Renbarger JL, McCammack KC, Rouse CE, Hall SD. (2008) Effect of race on vincristine-associated neurotoxicity in pediatric acute lymphoblastic leukemia patients. Pediatr Blood Cancer 50:769–771  [DOI](https://doi.org/10.1002/pbc.21435) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18085684/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pediatr%20Blood%20Cancer&title=Effect%20of%20race%20on%20vincristine-associated%20neurotoxicity%20in%20pediatric%20acute%20lymphoblastic%20leukemia%20patients&author=JL%20Renbarger&author=KC%20McCammack&author=CE%20Rouse&author=SD%20Hall&volume=50&publication_year=2008&pages=769-771&pmid=18085684&doi=10.1002/pbc.21435&)

24. Rodrigues AD. (1999) Integrated cytochrome P450 reaction phenotyping: attempting to bridge the gap between cDNA-expressed cytochromes P450 and native human liver microsomes. Biochem Pharmacol 57:465–480  [DOI](https://doi.org/10.1016/s0006-2952(98)00268-8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9952310/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem%20Pharmacol&title=Integrated%20cytochrome%20P450%20reaction%20phenotyping:%20attempting%20to%20bridge%20the%20gap%20between%20cDNA-expressed%20cytochromes%20P450%20and%20native%20human%20liver%20microsomes&author=AD%20Rodrigues&volume=57&publication_year=1999&pages=465-480&pmid=9952310&doi=10.1016/s0006-2952(98)00268-8&)

25. Roy JN, Lajoie J, Zijenah LS, Barama A, Poirier C, Ward BJ, Roger M. (2005) CYP3A5 genetic polymorphisms in different ethnic populations. Drug Metab Dispos 33:884–887  [DOI](https://doi.org/10.1124/dmd.105.003822) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15833928/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=CYP3A5%20genetic%20polymorphisms%20in%20different%20ethnic%20populations&author=JN%20Roy&author=J%20Lajoie&author=LS%20Zijenah&author=A%20Barama&author=C%20Poirier&volume=33&publication_year=2005&pages=884-887&pmid=15833928&doi=10.1124/dmd.105.003822&)

26. Van Heugen JC, De Graeve J, Zorza G, Puozzo C. (2001) New sensitive liquid chromatography method coupled with tandem mass spectrometric detection for the clinical analysis of vinorelbine and its metabolites in blood, plasma, urine and faeces. J Chromatogr A 926:11–20  [DOI](https://doi.org/10.1016/s0021-9673(01)00993-1) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11554404/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Chromatogr%20A&title=New%20sensitive%20liquid%20chromatography%20method%20coupled%20with%20tandem%20mass%20spectrometric%20detection%20for%20the%20clinical%20analysis%20of%20vinorelbine%20and%20its%20metabolites%20in%20blood,%20plasma,%20urine%20and%20faeces&author=JC%20Van%20Heugen&author=J%20De%20Graeve&author=G%20Zorza&author=C%20Puozzo&volume=926&publication_year=2001&pages=11-20&pmid=11554404&doi=10.1016/s0021-9673(01)00993-1&)

27. Wong M, Balleine RL, Blair EY, McLachlan AJ, Ackland SP, Garg MB, Evans S, Farlow D, Collins M, Rivory LP, et al. (2006) Predictors of vinorelbine pharmacokinetics and pharmacodynamics in patients with cancer. J Clin Oncol 24:2448–2455  [DOI](https://doi.org/10.1200/JCO.2005.02.1295) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16651648/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Predictors%20of%20vinorelbine%20pharmacokinetics%20and%20pharmacodynamics%20in%20patients%20with%20cancer&author=M%20Wong&author=RL%20Balleine&author=EY%20Blair&author=AJ%20McLachlan&author=SP%20Ackland&volume=24&publication_year=2006&pages=2448-2455&pmid=16651648&doi=10.1200/JCO.2005.02.1295&)

28. Xie HGWA, Wood AJ, Kim RB, Stein CM, Wilkinson GR. (2004) Genetic variability in CYP3A5 and its possible consequences. Pharmacogenomics 5:243–272  [DOI](https://doi.org/10.1517/phgs.5.3.243.29833) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15102541/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Genetic%20variability%20in%20CYP3A5%20and%20its%20possible%20consequences&author=HGWA%20Xie&author=AJ%20Wood&author=RB%20Kim&author=CM%20Stein&author=GR%20Wilkinson&volume=5&publication_year=2004&pages=243-272&pmid=15102541&doi=10.1517/phgs.5.3.243.29833&)
