# Genome-wide Association Identifies the T Gene as a Novel Asthma Pharmacogenetic Locus

## Metadata
**Authors:** Kelan G Tantisira, Amy Damask, Stanley J Szefler, Brooke Schuemann, Amy Markezich, Jessica Su, Barbara Klanderman, Jody Sylvia, Rongling Wu, Fernando Martinez, Homer A Boushey, Vernon M Chinchilli, Dave Mauger, Scott T Weiss, Elliot Israel
**Journal:** American Journal of Respiratory and Critical Care Medicine
**Date:** 2012 Jun 15
**DOI:** [10.1164/rccm.201111-2061OC](https://doi.org/10.1164/rccm.201111-2061OC)
**PMID:** 22538805
**PMCID:** PMC3381232
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3381232/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC3381232/pdf/AJRCCM185121286.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC3381232/pdf/AJRCCM185121286.pdf)

## Abstract

Rationale: To date, most studies aimed at discovering genetic factors influencing treatment response in asthma have focused on biologic candidate genes. Genome-wide association studies (GWAS) can rapidly identify novel pharmacogenetic loci.

Objectives: To investigate if GWAS can identify novel pharmacogenetic loci in asthma.

Methods: Using phenotypic and GWAS genotype data available through the NHLBI-funded Single-nucleotide polymorphism Health association-Asthma Resource Project, we analyzed differences in FEV1 in response to inhaled corticosteroids in 418 white subjects with asthma. Of the 444,088 single nucleotide polymorphisms (SNPs) analyzed, the lowest 50 SNPs by P value were genotyped in an independent clinical trial population of 407 subjects with asthma.

Measurements and Main Results: The lowest P value for the GWAS analysis was 2.09 × 10−6. Of the 47 SNPs successfully genotyped in the replication population, three were associated under the same genetic model in the same direction, including two of the top four SNPs ranked by P value. Combined P values for these SNPs were 1.06 × 10−5 for rs3127412 and 6.13 × 10−6 for rs6456042. Although these two were not located within a gene, they were tightly correlated with three variants mapping to potentially functional regions within the T gene. After genotyping, each T gene variant was also associated with lung function response to inhaled corticosteroids in each of the trials associated with rs3127412 and rs6456042 in the initial GWAS analysis. On average, there was a twofold to threefold difference in FEV1 response for those subjects homozygous for the wild-type versus mutant alleles for each T gene SNP.

Conclusions: Genome-wide association has identified the T gene as a novel pharmacogenetic locus for inhaled corticosteroid response in asthma.

Keywords: polymorphism, genome, pharmacogenomics, glucocorticoid

## AT A GLANCE COMMENTARY

## AT A GLANCE COMMENTARY

### Scientific Knowledge on the Subject

There is clear cut interindividual variability in the response to inhaled corticosteroid medications in asthma with a significant number of nonresponders. Pharmacogenetics seeks to understand the genetic determinants of interindividual variability in drug response, but relatively few asthma pharmacogenetic loci have been identified to date.

### What This Study Adds to the Field

This study provides the results of a genome-wide association analysis focusing on identifying novel determinants of inhaled corticosteroid response in asthma. By using correlation within the genome, this analysis has led to the identification of a novel harmacogenetic locus, the *T* gene.

## Methods

Detailed methods, including SHARP and replication clinical trial descriptions, are described in the online supplement.

### Study Populations

SHARP included Childhood Asthma Management Program (CAMP) ([16](#bib16)), Childhood Asthma Research and Education (CARE) network ([6](#bib6), [17](#bib17)), and Asthma Clinical Research Network (ACRN) ([18](#bib18)–[21](#bib21)) participants. CAMP, two of five CARE trials, and four of six ACRN trials enrolled subjects with ICS monotherapy treatment response and were included. Further information on SHARP can be found in dbGAP ([http://www.ncbi.nlm.nih.gov/sites/entrez?Db=gap](http://www.ncbi.nlm.nih.gov/sites/entrez?Db=gap)).

Replication was performed in an 8-week clinical trial conducted by Forest pharmaceuticals, which has been detailed in other pharmacogenetic analyses ([10](#bib10), [13](#bib13)).

### Phenotype

The primary outcome phenotype was prebronchodilator FEV_1_ while on ICS minus prebronchodilator FEV_1_ off ICS, typically the difference between a placebo run-in period to a point 6–12 weeks later after ICS therapy. Additionally, a “reversal group” of individuals on ICS therapy who were tapered off therapy during a trial protocol and a “mixed group” of individuals with FEV_1_ measured at the start and end of an ICS run-in were included. The mean (± SD) FEV_1_% difference was nearly identical for all three groups (4 ± 7.9, 4.3 ± 10.9, and 4.1 ± 11.8 percent for standard, reverse, and mixed groups, respectively) (*see* Figure E1 in the online supplement). Of 792 subjects with ICS response data, 458 were white, forming the phenotypic analysis group.

### Genotyping

GWAS genotyping was performed by Affymetrix (Santa Clara, CA) using a Human SNP Array 6.0. Marker quality control (QC) is detailed in the online supplement. Overall, 444,088 SNPs and 4,113 genotyped white subjects passed QC. After merger with the phenotypic dataset, the final ICS analysis group was 418: 179 ACRN, 147 CAMP, and 92 CARE.

Replication genotyping was performed using Sequenom (San Diego, CA). *T* SNP genotyping was performed using Taqman (Applied Biosystems, Foster City, CA). Because additional DNA was not available in CARE and ACRN, corresponding *T* genotypes were inferred using imputation with Markov Chain Haplotyping ([22](#bib22)) based on HapMap Phase 2 Release 22 data ([23](#bib23)). The ratio of empirically observed to expected (binomial) dosage variance for these SNPs was greater than 0.9, indicating good quality of imputation.

### Statistical Analysis

GWAS analyses were performed using PLINK ([24](#bib24)). Given similarities between CAMP and CARE phenotypic data, these two datasets were pooled. ACRN data were analyzed separately. The separate β-estimates and standard errors were then used to create combined β-estimates and *P* values, using a random-effects approach ([25](#bib25)).

The ICS analysis comprised two phenotypic definitions: difference in raw FEV_1_ scores (prefev_on___ICS_–prefev_off_ICS_) and difference in percent predicted FEV_1_ scores (prefev%_on_ICS_–prefev%_off_ICS_). The primary outcome, raw differences, was adjusted for age, height, sex, study group, and six significant principal components. The percent predicted differences were adjusted for study group and six significant principal components. ACRN analyses were further adjusted for ICS therapy timing (i.e., standard, reverse, and mixed).

Generalized linear models evaluated the association between the selected SNPs and FEV_1_ change in the replication population_,_ adjusting for age, sex, and height using SAS (SAS Institute, Cary, NC). Analyses assumed an additive effect of the minor allele. A replication was defined by a nominal *P* less than 0.05.

## Results

### Populations

A total of 418 SHARP subjects met phenotypic and genotypic inclusion criteria ([Table 1](#tbl1)). Of these, 239 pediatric CAMP and CARE subjects were pooled before analysis, whereas 179 adult ACRN subjects were analyzed separately. In addition to age, the adult SHARP subjects had a greater proportion of females and lower baseline FEV_1_ as a percent of predicted before ICS therapy compared with the pediatric subjects. To limit potential for associations related to occult population stratification, we limited analysis to subjects self-designated as whites. Characteristics of our replication population, Forest, are also shown in [Table 1](#tbl1). Overall, this population was similar to the SHARP ACRN subjects at baseline, with a more robust response to ICS therapy. This may be related, in part, to the low baseline FEV_1_ associated with this population.

### TABLE 1.

|   | ACRN* | CAMP* | CARE* | Forest† |
| --- | --- | --- | --- | --- |
| N | 179‡ | 147 | 92 | 407 |
| Age ± SD | 33.8 ± 11.2 | 8.7 ± 2.1 | 11 ± 3.3 | 39.4 ± 13.6 |
| Sex (male) | 66 (37%) | 82 (56%) | 55 (60%) | 170 (42%) |
| Race (white)§ | 100% | 100% | 100% | 100% |
| Baseline FEV1% | 79 ± 15.8 | 94.2 ± 14.5 | 99.4 ± 12.6 | 71.7 ± 12.9 |
| Duration of ICS therapy | 6 wk | 8 wk | 6–8 wk | 8 wk |
| Mean FEV1% change after ICS | 4.1 ± 10.4 | 5.2 ± 11.5 | 4.6 ± 8.7 | 7.2 ± 19.7 |

Table 1 Caption: NETWORK POPULATION DESCRIPTORS

### Initial GWAS Association

GWAS of the FEV_1_ difference related to ICS therapy was conducted on 444,088 SNPs in CAMP/CARE pooled samples and separately in the ACRN subjects. [Figure 1](#fig1) demonstrates a Manhattan plot of the combined association *P* values. For internal consistency, we then checked for association of the SNPs with differences in FEV_1_ as a percent predicted related to ICS therapy. The top 50 *P* values from the raw FEV_1_ difference analysis that were also associated with differences in predicted FEV_1_ in the combined analysis are shown in [Table 2](#tbl2) and were taken forward for genotyping in the replication population. Four SNPs had *P* values less than 1 × 10^−5^, although this fell short of the threshold for genome-wide significance.

### Figure 1.

![Figure 1.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3281/3381232/9b9df13dd861/AJRCCM185121286fig1.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3381232_AJRCCM185121286fig1.jpg)

Manhattan plot of genome-wide association study results. Plotted are the 444,088 P values (as negative log P values for association with difference in FEV1 after inhaled corticosteroid therapy) versus their chromosomal position on the genome.

### TABLE 2.

| SNP | Closest Gene* | ACRN P Value | CC P Value | SHARP P Value† |
| --- | --- | --- | --- | --- |
| rs10218533 | RP5-1027O11.1 | 0.023 | 4.67 × 10−5 | 2.09 × 10−6 |
| rs6456042 | RP11-459F1.2 | 0.013 | 1.95 × 10−4 | 6.67 × 10−6 |
| rs6413915 | RP5-894H24.2 | 0.011 | 3.09 × 10−4 | 9.39 × 10−6 |
| rs3127412 | RP11-459F1.2 | 0.034 | 1.35 × 10−4 | 9.68 × 10−6 |
| rs12773521 | RP11-31E13.1 | 0.082 | 7.63 × 10−5 | 1.14 × 10−5 |
| rs2065705 | PRNT | 0.055 | 1.17 × 10−4 | 1.26 × 10−5 |
| rs2065704 | PRNT | 0.055 | 1.21 × 10−4 | 1.30 × 10−5 |
| rs12684262 | SMARCA2 | 0.051 | 1.85 × 10−4 | 1.88 × 10−5 |
| rs189243 | TMED9 | 0.120 | 9.43 × 10−5 | 2.02 × 10−5 |
| rs4752352 | SEC23IP | 0.121 | 9.77 × 10−5 | 2.14 × 10−5 |
| rs13169820 | B4GALT7 | 0.107 | 1.46 × 10−4 | 2.85 × 10−5 |
| rs424727 | TMED9 | 0.134 | 1.74 × 10−4 | 4.22 × 10−5 |
| rs1322239 | BMP6 | 0.364 | 6.20 × 10−5 | 4.27 × 10−5 |
| rs7156058 | C14orf101 | 0.014 | 9.78 × 10−4 | 4.31 × 10−5 |
| rs12156034 | POU5F1B | 0.026 | 6.20 × 10−4 | 4.83 × 10−5 |
| rs2271124 | SEC23IP | 0.156 | 1.73 × 10−4 | 4.88 × 10−5 |
| rs16938276 | SMARCA2 | 0.055 | 4.33 × 10−4 | 4.94 × 10−5 |
| rs17203177 | LASS6 | 0.056 | 4.29 × 10−4 | 4.99 × 10−5 |
| rs6084858 | PRNT | 0.042 | 5.86 × 10−4 | 5.81 × 10−5 |
| rs6052833 | PRNT | 0.031 | 7.88 × 10−4 | 6.33 × 10−5 |
| rs7012345 | JPH1 | 0.045 | 6.48 × 10−4 | 6.63 × 10−5 |
| rs2247701 | AC068483.1 | 0.041 | 7.25 × 10−4 | 6.70 × 10−5 |
| rs1529018 | RP11-497D6.4 | 0.050 | 5.57 × 10−4 | 6.76 × 10−5 |
| rs2934181 | SEMA6D | 0.084 | 4.23 × 10−4 | 6.98 × 10−5 |
| rs6016963 | PTPRT | 0.069 | 5.13 × 10−4 | 7.20 × 10−5 |
| rs2502562 | COL19A1 | 0.009 | 2.20 × 10−3 | 7.48 × 10−5 |
| rs234375 | TMED9 | 0.412 | 8.70 × 10−5 | 7.63 × 10−5 |
| rs6584356 | PKD2L1 | 0.054 | 6.97 × 10−4 | 8.05 × 10−5 |
| rs10834433 | LUZP2 | 0.381 | 1.06 × 10−4 | 8.17 × 10−5 |
| rs12591739 | FAM169B | 0.216 | 2.29 × 10−4 | 9.03 × 10−5 |
| rs12760824 | PLEKHA6 | 0.023 | 1.56 × 10−3 | 9.27 × 10−5 |
| rs420828 | AC091947.4 | 0.242 | 1.86 × 10−4 | 9.28 × 10−5 |
| rs1121121 | AC023481.6 | 0.290 | 1.87 × 10−4 | 9.68 × 10−5 |
| rs11014839 | RP11-435M3.2 | 0.019 | 1.98 × 10−3 | 1.06 × 10−4 |
| rs8061182 | ADCY9 | 0.111 | 5.42 × 10−4 | 1.16 × 10−4 |
| rs615625 | KCNK1 | 0.173 | 3.57 × 10−4 | 1.19 × 10−4 |
| rs11110465 | AC010209.14 | 0.193 | 3.57 × 10−4 | 1.23 × 10−4 |
| rs968433 | RASSF2 | 0.081 | 7.56 × 10−4 | 1.25 × 10−4 |
| rs11944371 | ANxA5 | 0.145 | 4.75 × 10−4 | 1.26 × 10−4 |
| rs9955411 | AC007922.13-1 | 0.050 | 1.11 × 10−3 | 1.28 × 10−4 |
| rs12811075 | KCNA5 | 0.329 | 2.51 × 10−4 | 1.35 × 10−4 |
| rs1557281 | CADM1 | 0.023 | 1.85 × 10−3 | 1.38 × 10−4 |
| rs2344752 | C14orf101 | 0.032 | 1.69 × 10−3 | 1.39 × 10−4 |
| rs2374053 | C12orf42 | 0.230 | 3.03 × 10−4 | 1.42 × 10−4 |
| rs7567056 | AC010096.2 | 0.113 | 6.62 × 10−4 | 1.45 × 10−4 |
| rs2456714 | C10orf119 | 0.059 | 1.18 × 10−3 | 1.56 × 10−4 |
| rs2658746 | ZNF236 | 0.139 | 6.14 × 10−4 | 1.59 × 10−4 |

Table 2 Caption: TOP 50 SHARP P VALUES FOR FEV1 CHANGE

### Replication Population Association

Of the 50 SNPs, 47 were successfully genotyped in the replication population. Three SNPs were associated with FEV_1_ difference in response to flunisolide, under the assumption of an additive model ([Table 3](#tbl3)) in the same direction as the original SHARP analysis. [Table 3](#tbl3) also demonstrates the *P* value for overall association, reflecting the combined association *P* values for the ACRN, CAMP/CARE, and Forest populations, which was calculated using Stouffer z-transform test. Of note, two of the top four SHARP SNPs as ranked by *P* value (rs6456042 and rs3127412) were among the three SNPs replicating in the Forest population. After replication, the lowest combined *P* value was 6.13 × 10^−6^, which is below the threshold for genome-wide significance (1.13 × 10^−7^).

### TABLE 3.

|   | rs3127412 |  | rs6456042 |  | rs9955411 |  |
| --- | --- | --- | --- | --- | --- | --- |
|   | Beta | P Value | Beta | P Value | Beta | P Value |
| SHARP (overall) | 0.078 | 9.68 × 10−6 | 0.079 | 6.67 × 10−6 | 0.080 | 0.00013 |
| ACRN | 0.090 | 0.034 | 0.103 | 0.013 | 0.096 | 0.05 |
| CAMP/CARE | 0.076 | 0.00013 | 0.074 | 0.0002 | 0.076 | 0.0011 |
| Forest | 0.071 | 0.02 | 0.072 | 0.02 | 0.092 | 0.02 |
| Combined P* |   | 1.06 × 10−5 |   | 6.13 × 10−6 |   | 4.64 × 10−5 |

Table 3 Caption: ASSOCIATED REPLICATION SNPS

### Mapping of Top SNPs

Rs6456042 and rs3127412 are located on chromosome 6q27; both map to within 50 kb downstream of a transcription factor known as the *T* gene (Entrez Gene: 6,862). Using the CEU HapMap population, linkage disquilibrium (LD) (i.e., correlation between genotypes) measures were obtained for SNPs mapping to the *T* gene with rs6456042 and rs3127412. For these two SNPs, which are in tight LD, three *T* gene SNPs were correlated, with an *r*^2^ (correlation coefficient) greater than or equal to 0.66 (*see* Figure E2). Furthermore, each of these three *T* gene SNPs are located within potential functional regions of the gene: rs3099266 maps to the promoter region, rs1134481 to the 3′ untranslated region, and rs2305089 is a nonsynonymous variant coding for a glycine to aspartic acid amino acid change at position 177 within the gene. The SHARP genotype dataset did not contain any of these variants. Therefore, to elucidate whether the initial GWAS association actually represented a downstream correlation with a true association within the *T* gene, we genotyped these three SNPs in the CAMP and Forest populations, and analyzed imputed genotypes for the SNPs in ACRN and CARE.

### T Gene Analysis

Association analysis within CAMP and Forest demonstrated that each of the three *T* SNPs were also associated with FEV_1_ response to ICS therapy, with *P* values for CAMP and Forest, respectively, as follows: rs1134481 (0.0009 and 0.01); rs2305089 (0.004 and 0.02); and rs3099266 (0.004 and 0.01) ([Table 4](#tbl4)). The strongest association noted was with rs1134481. For this variant, homozygous wild-type subjects had an improvement in FEV_1_ as a percent of predicted of 2.8 ± 1.3 and 2.7 ± 1 versus 9.9 ± 2.1 and 5.3 ± 1.5 for those homozygous mutant for CAMP and Forest, respectively. Overall, after adjustment for age, sex, and height, each *T* SNP demonstrated a twofold to threefold difference in FEV_1_ response for those subjects homozygous for the wild-type versus mutant alleles in both populations (*see* Figure 3). Imputed analyses of the *T* variants were also consistent with association through LD, with a significant association noted for ACRN. Although the imputed *T* gene variants were not associated with ICS response in CARE, this is consistent with the primary SHARP GWAS results for the two SNPs in LD with the *T* gene variants (*see* Table E1). Although CARE demonstrated the highest baseline level of FEV_1_ ([Table 1](#tbl1)), adjustment for baseline FEV_1_ did not significantly alter the associations with the *T* gene SNPs (*see* Table E2).

### TABLE 4.

| SNP | Role | CAMP* | Forest* | ACRN† | CARE† | Combined‡ |
| --- | --- | --- | --- | --- | --- | --- |
| rs1134481 | 3′ UTR | 0.0017 | 0.041 | 0.0037 | 0.58 | 1.57 × 10−5 |
| rs2305089 | Coding exon (Gly177Asp) | 0.0077 | 0.061 | 0.013 | 0.73 | 2.3 × 10−4 |
| rs3099266 | Promoter | 0.0087 | 0.036 | 0.0028 | 0.98 | 1.1 × 10−4 |

Table 4 Caption: ASSOCIATION OF T GENE VARIANTS WITH ICS RESPONSE

## Discussion

GWAS holds tremendous promise because of the ability to discover new biology through simultaneous interrogation of variants across the entire genome. This is especially applicable to the field of drug response pharmacogenetics, where candidate gene studies have failed to explain enough of the genetic variation to allow predictive modeling. Through the SHARP project, we interrogated 444,088 SNP variants for their association with lung function response to ICS in a compilation of seven asthma clinical trials. We were able to replicate 3 of the top 50 GWAS SNPs in an independent population, including two of the top four SNPs.

Although none of the replicated SNPs were located within a gene, the top two SNPs were in LD (correlated) with three known variants in the *T* gene. Using LD to identify disease genes takes advantage of the historical preservation of regions within the genome as they are passed down through generations ([26](#bib26), [27](#bib27)). Each of the three correlated *T* gene variants maps to a potentially functional region of the gene, with rs2305089 encoding for an amino acid change (Gly177Asp) within the T protein. Each was also significantly associated with ICS response, suggesting that the true biologic effect is likely mediated by this gene. There was a twofold to threefold difference in FEV_1_ response for those subjects homozygous for the wild-type versus mutant alleles for each *T* gene SNP in each of the genotyped populations (*see* Figure E3).

Despite the compilation of studies within SHARP, our study design remains underpowered for finding associations of genome-wide significance. However, our approach used the GWAS as an initial step to subsequently target variation within the *T* gene locus, resulting in an effect that was replicated in a larger replication cohort and the combined population. Each of the three interrogated *T* SNPs was associated with ICS response even after correction for multiple comparisons.

The *T* gene, which has not previously been implicated in either asthma or corticosteroid pathobiology, is a founding member of an ancient family of genes containing a common protein motif, the T locus ([28](#bib28)). The T locus encodes a product with DNA binding activity that likely plays a role in the development of all vertebrate organisms. The *T* gene itself (*Brachyury homolog*) encodes for a key mesodermal developmental transcription factor crucial to notochord development ([29](#bib29)). However, the expression of *T* is sufficiently early in the developmental process that it has the potential to influence mesodermal development overall ([30](#bib30)), including within the lung. Corticosteroids are also crucial to lung development; alterations in corticosteroid-responsive genes during development may thus influence later asthma susceptibility and treatment response. *T* is also expressed in normal adult lung tissue ([31](#bib31)), although little is known about its transcriptional regulation of other genes. *T* and other T-box genes have been implicated in the function of bone morphogenetic protein (BMP) and fibroblast growth factor (FGF) pathways ([32](#bib32)). Specifically, decreased *T* expression inhibits chondrogenesis mediated by *BMP2* and *FGFR3* ([33](#bib33)). Notably, the *BMP2* receptor and *FGFR3* have been independently implicated in corticosteroid resistance ([34](#bib34), [35](#bib35)), supporting a mechanistic basis for the *T* gene in ICS response in asthma.

To further explore the potential effects of the *T* gene on response to corticosteroids, we conducted a pathway analysis specifically focusing on potential *T* gene–*NR3C1* (glucocorticoid receptor) interactions using the program GeneMania ([http://genemania.org/](http://genemania.org/)) (*see* Figure E4). Although not conclusive, the results indicate that the *T* gene is coexpressed with three other genes (*NRIP1*, *FOXA2*, and *TTPA*) that directly interact or are predicted to interact with *NR3C1*.

Although variations in *T* have the potential to directly cause alterations in response to ICS therapy, the familial lineage of *T* also offers intriguing mechanistic possibilities. *T* is phylogenetically conserved with 50 known orthologs. Moreover, 16 other members of the T family are paralogous (i.e., the result of a prior gene duplication within the same ancestral chromosome), including genes encoding for a number of the T-box domain proteins, such as *TBX21* (data from Ensembl [www.ensembl.org]). It is possible that there is sufficient overlap in the genomic structure of *T* and one of these paralogs that the mechanism of action is actually by the paralog; we have previously reported an independent ICS pharmacogenetic association with airways responsiveness in asthma resulting from variation in *TBX21* ([11](#bib11)). *T* is known to interact with other members of the T-box family ([36](#bib36)), yielding an additional mechanism through which a *T* may affect ICS response. Additional work in defining the role of genetic variation within *T* and the role of *T* on asthma treatment response is warranted.

Our top GWAS hits from this analysis contrast with those that we have recently identified by a family-based screening algorithm ([37](#bib37)). This is expected because the current analysis, given its larger sample size, ranked the SNPs by *P* value, whereas our prior study ranked SNPs by relative power for replication, which is only possible in a family-based study design. We also did not identify the *GLCCI1* SNP from our prior study ([37](#bib37)) as one of our most significant variants in this study; however, neither of the *GLCCI1* variants associated in the previous study is incorporated on the genotyping chip used in this study. Specifically, because our *GLCCI1* resequencing revealed only one other variant, rs37973, in tight LD with the original associated variant (rs37972) and because neither of these SNPs is present on the Affymetrix platform used for the current analysis, we did not *a priori* expect to replicate the *GLCCI1* association.

There are several potential limitations to our study. As noted, we do not report any association exceeding the genome-wide significance threshold (Bonferroni adjusted). However, this adjustment may be overly conservative, because it does not account for LD within the genotyped populations. Additionally, our independent replication of the findings is reassuring of a true positive association. Moreover, the association of upstream variants within the *T* gene that were not in complete LD (*r*^2^ = 0.66–0.82) with the initial association lends further credence to the validity of the association. Although we used a compilation of different studies, the treatment response distribution based on timing of ICS usage, age (adult vs. child), and clinical network was similar (*see* Figure E1), suggesting that the data can be combined for genetic analyses. Moreover, findings noted across these distributions support the generalization of our results to multiple clinical scenarios.

In conclusion, GWAS study of ICS response in SHARP has allowed the identification of a novel pharmacogenetic locus, the *T* gene. For variants within this gene, there was a twofold to threefold difference in FEV_1_ response for those subjects homozygous for the wild-type versus mutant alleles. As genome-wide studies identify additional variants influencing drug treatment response in asthma and other diseases, we will continue to draw closer to the era of truly personalized medicine.

## Supplementary Material

## Acknowledgments

The full Single-Nucleotide Polymorphism Health Association-Asthma Resource Project acknowledgment can be found in the online supplement. The authors specifically acknowledge James Kiley for his leadership and funding contribution to this manuscript.

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Masoli M, Fabian D, Holt S, Beasley R. The global burden of asthma: executive summary of the GINA dissemination committee report. Allergy 2004;59:469–478  [DOI](https://doi.org/10.1111/j.1398-9995.2004.00526.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15080825/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Allergy&title=The%20global%20burden%20of%20asthma:%20executive%20summary%20of%20the%20GINA%20dissemination%20committee%20report&author=M%20Masoli&author=D%20Fabian&author=S%20Holt&author=R%20Beasley&volume=59&publication_year=2004&pages=469-478&pmid=15080825&doi=10.1111/j.1398-9995.2004.00526.x&)

2. Inglis GC, Ingram MC, Holloway CD, Swan L, Birnie D, Hillis WS, Davies E, Fraser R, Connell JM. Familial pattern of corticosteroids and their metabolism in adult human subjects: the Scottish Adult Twin Study. J Clin Endocrinol Metab 1999;84:4132–4137  [DOI](https://doi.org/10.1210/jcem.84.11.6146) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10566661/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Endocrinol%20Metab&title=Familial%20pattern%20of%20corticosteroids%20and%20their%20metabolism%20in%20adult%20human%20subjects:%20the%20Scottish%20Adult%20Twin%20Study&author=GC%20Inglis&author=MC%20Ingram&author=CD%20Holloway&author=L%20Swan&author=D%20Birnie&volume=84&publication_year=1999&pages=4132-4137&pmid=10566661&doi=10.1210/jcem.84.11.6146&)

3. Ober C, Abney M, McPeek MS. The genetic dissection of complex traits in a founder population. Am J Hum Genet 2001;69:1068–1079  [DOI](https://doi.org/10.1086/324025) | [PMC free article](/articles/PMC1274354/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11590547/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=The%20genetic%20dissection%20of%20complex%20traits%20in%20a%20founder%20population&author=C%20Ober&author=M%20Abney&author=MS%20McPeek&volume=69&publication_year=2001&pages=1068-1079&pmid=11590547&doi=10.1086/324025&)

4. Schwartz JT, Reuling FH, Feinleib M, Garrison RJ, Collie DJ. Twin heritability study of the effect of corticosteroids on intraocular pressure. J Med Genet 1972;9:137–143  [DOI](https://doi.org/10.1136/jmg.9.2.137) | [PMC free article](/articles/PMC1469045/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/5065285/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Med%20Genet&title=Twin%20heritability%20study%20of%20the%20effect%20of%20corticosteroids%20on%20intraocular%20pressure&author=JT%20Schwartz&author=FH%20Reuling&author=M%20Feinleib&author=RJ%20Garrison&author=DJ%20Collie&volume=9&publication_year=1972&pages=137-143&pmid=5065285&doi=10.1136/jmg.9.2.137&)

5. Szefler SJ, Martin RJ, King TS, Boushey HA, Cherniack RM, Chinchilli VM, Craig TJ, Dolovich M, Drazen JM, Fagan JK, et al. Significant variability in response to inhaled corticosteroids for persistent asthma. J Allergy Clin Immunol 2002;109:410–418  [DOI](https://doi.org/10.1067/mai.2002.122635) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11897984/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Allergy%20Clin%20Immunol&title=Significant%20variability%20in%20response%20to%20inhaled%20corticosteroids%20for%20persistent%20asthma&author=SJ%20Szefler&author=RJ%20Martin&author=TS%20King&author=HA%20Boushey&author=RM%20Cherniack&volume=109&publication_year=2002&pages=410-418&pmid=11897984&doi=10.1067/mai.2002.122635&)

6. Szefler SJ, Phillips BR, Martinez FD, Chinchilli VM, Lemanske RF, Strunk RC, Zeiger RS, Larsen G, Spahn JD, Bacharier LB, et al. Characterization of within-subject responses to fluticasone and montelukast in childhood asthma. J Allergy Clin Immunol 2005;115:233–242  [DOI](https://doi.org/10.1016/j.jaci.2004.11.014) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15696076/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Allergy%20Clin%20Immunol&title=Characterization%20of%20within-subject%20responses%20to%20fluticasone%20and%20montelukast%20in%20childhood%20asthma&author=SJ%20Szefler&author=BR%20Phillips&author=FD%20Martinez&author=VM%20Chinchilli&author=RF%20Lemanske&volume=115&publication_year=2005&pages=233-242&pmid=15696076&doi=10.1016/j.jaci.2004.11.014&)

7. Drazen JM, Silverman EK, Lee TH. Heterogeneity of therapeutic responses in asthma. Br Med Bull 2000;56:1054–1070  [DOI](https://doi.org/10.1258/0007142001903535) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11359637/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20Med%20Bull&title=Heterogeneity%20of%20therapeutic%20responses%20in%20asthma&author=JM%20Drazen&author=EK%20Silverman&author=TH%20Lee&volume=56&publication_year=2000&pages=1054-1070&pmid=11359637&doi=10.1258/0007142001903535&)

8. Drazen JM, Yandava CN, Dube L, Szczerback N, Hippensteel R, Pillari A, Israel E, Schork N, Silverman ES, Katz DA, et al. Pharmacogenetic association between alox5 promoter genotype and the response to anti-asthma treatment. Nat Genet 1999;22:168–170  [DOI](https://doi.org/10.1038/9680) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10369259/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Pharmacogenetic%20association%20between%20alox5%20promoter%20genotype%20and%20the%20response%20to%20anti-asthma%20treatment&author=JM%20Drazen&author=CN%20Yandava&author=L%20Dube&author=N%20Szczerback&author=R%20Hippensteel&volume=22&publication_year=1999&pages=168-170&pmid=10369259&doi=10.1038/9680&)

9. Israel E, Chinchilli VM, Ford JG, Boushey HA, Cherniack R, Craig TJ, Deykin A, Fagan JK, Fahy JV, Fish J, et al. Use of regularly scheduled albuterol treatment in asthma: genotype-stratified, randomised, placebo-controlled cross-over trial. Lancet 2004;364:1505–1512  [DOI](https://doi.org/10.1016/S0140-6736(04)17273-5) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15500895/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Use%20of%20regularly%20scheduled%20albuterol%20treatment%20in%20asthma:%20genotype-stratified,%20randomised,%20placebo-controlled%20cross-over%20trial&author=E%20Israel&author=VM%20Chinchilli&author=JG%20Ford&author=HA%20Boushey&author=R%20Cherniack&volume=364&publication_year=2004&pages=1505-1512&pmid=15500895&doi=10.1016/S0140-6736(04)17273-5&)

10. Tantisira KG, Lake S, Silverman ES, Palmer LJ, Lazarus R, Silverman EK, Liggett SB, Gelfand EW, Richter B, Israel E, et al. Corticosteroid pharmacogenetics: association of sequence variants in crhr1 with improved lung function in asthmatics treated with inhaled corticosteroids. Hum Mol Genet 2004;13:1353–1359  [DOI](https://doi.org/10.1093/hmg/ddh149) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15128701/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mol%20Genet&title=Corticosteroid%20pharmacogenetics:%20association%20of%20sequence%20variants%20in%20crhr1%20with%20improved%20lung%20function%20in%20asthmatics%20treated%20with%20inhaled%20corticosteroids&author=KG%20Tantisira&author=S%20Lake&author=ES%20Silverman&author=LJ%20Palmer&author=R%20Lazarus&volume=13&publication_year=2004&pages=1353-1359&pmid=15128701&doi=10.1093/hmg/ddh149&)

11. Tantisira KG, Hwang ES, Raby BA, Silverman ES, Lake SL, Richter BG, Peng SL, Drazen JM, Glimcher LH, Weiss ST. Tbx21: a functional variant predicts improvement in asthma with the use of inhaled corticosteroids. Proc Natl Acad Sci U S A 2004;101:18099–18104  [DOI](https://doi.org/10.1073/pnas.0408532102) | [PMC free article](/articles/PMC539815/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15604153/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&title=Tbx21:%20a%20functional%20variant%20predicts%20improvement%20in%20asthma%20with%20the%20use%20of%20inhaled%20corticosteroids&author=KG%20Tantisira&author=ES%20Hwang&author=BA%20Raby&author=ES%20Silverman&author=SL%20Lake&volume=101&publication_year=2004&pages=18099-18104&pmid=15604153&doi=10.1073/pnas.0408532102&)

12. Tantisira KG, Silverman ES, Mariani TJ, Xu J, Richter BG, Klanderman BJ, Litonjua AA, Lazarus R, Rosenwasser LJ, Fuhlbrigge AL, et al. Fcer2: a pharmacogenetic basis for severe exacerbations in children with asthma. J Allergy Clin Immunol 2007;120:1285–1291  [DOI](https://doi.org/10.1016/j.jaci.2007.09.005) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17980418/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Allergy%20Clin%20Immunol&title=Fcer2:%20a%20pharmacogenetic%20basis%20for%20severe%20exacerbations%20in%20children%20with%20asthma&author=KG%20Tantisira&author=ES%20Silverman&author=TJ%20Mariani&author=J%20Xu&author=BG%20Richter&volume=120&publication_year=2007&pages=1285-1291&pmid=17980418&doi=10.1016/j.jaci.2007.09.005&)

13. Hawkins GA, Lazarus R, Smith RS, Tantisira KG, Meyers DA, Peters SP, Weiss ST, Bleecker ER. The glucocorticoid receptor heterocomplex gene stip1 is associated with improved lung function in asthmatic subjects treated with inhaled corticosteroids. J Allergy Clin Immunol 2009;123:1376–1383  [DOI](https://doi.org/10.1016/j.jaci.2009.01.049) | [PMC free article](/articles/PMC4317788/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19254810/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Allergy%20Clin%20Immunol&title=The%20glucocorticoid%20receptor%20heterocomplex%20gene%20stip1%20is%20associated%20with%20improved%20lung%20function%20in%20asthmatic%20subjects%20treated%20with%20inhaled%20corticosteroids&author=GA%20Hawkins&author=R%20Lazarus&author=RS%20Smith&author=KG%20Tantisira&author=DA%20Meyers&volume=123&publication_year=2009&pages=1376-1383&pmid=19254810&doi=10.1016/j.jaci.2009.01.049&)

14. Tantisira KG, Small KM, Litonjua AA, Weiss ST, Liggett SB. Molecular properties and pharmacogenetics of a polymorphism of adenylyl cyclase type 9 in asthma: interaction between {beta}-agonist and corticosteroid pathways. Hum Mol Genet 2005;14:1671–1677  [DOI](https://doi.org/10.1093/hmg/ddi175) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15879435/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mol%20Genet&title=Molecular%20properties%20and%20pharmacogenetics%20of%20a%20polymorphism%20of%20adenylyl%20cyclase%20type%209%20in%20asthma:%20interaction%20between%20%7Bbeta%7D-agonist%20and%20corticosteroid%20pathways&author=KG%20Tantisira&author=KM%20Small&author=AA%20Litonjua&author=ST%20Weiss&author=SB%20Liggett&volume=14&publication_year=2005&pages=1671-1677&pmid=15879435&doi=10.1093/hmg/ddi175&)

15. Murphy A, Lasky-Su J, Tantisira KG, Litonjua AA, Lange C, Weiss ST. Genome-wide association studies of family data in pharmacogenetics: a case study. Curr Pharm Des 2009;15:3764–3772  [DOI](https://doi.org/10.2174/138161209789649493) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19925427/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Pharm%20Des&title=Genome-wide%20association%20studies%20of%20family%20data%20in%20pharmacogenetics:%20a%20case%20study&author=A%20Murphy&author=J%20Lasky-Su&author=KG%20Tantisira&author=AA%20Litonjua&author=C%20Lange&volume=15&publication_year=2009&pages=3764-3772&pmid=19925427&doi=10.2174/138161209789649493&)

16. Long-term effects of budesonide or nedocromil in children with asthma. The Childhood Asthma Management Program Research Group. N Engl J Med 2000;343:1054–1063  [DOI](https://doi.org/10.1056/NEJM200010123431501) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11027739/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Long-term%20effects%20of%20budesonide%20or%20nedocromil%20in%20children%20with%20asthma.%20The%20Childhood%20Asthma%20Management%20Program%20Research%20Group&volume=343&publication_year=2000&pages=1054-1063&pmid=11027739&doi=10.1056/NEJM200010123431501&)

17. Sorkness CA, Lemanske RF, Jr, Mauger DT, Boehmer SJ, Chinchilli VM, Martinez FD, Strunk RC, Szefler SJ, Zeiger RS, Bacharier LB, et al. Long-term comparison of 3 controller regimens for mild-moderate persistent childhood asthma: the Pediatric Asthma Controller trial. J Allergy Clin Immunol 2007;119:64–72  [DOI](https://doi.org/10.1016/j.jaci.2006.09.042) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17140647/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Allergy%20Clin%20Immunol&title=Long-term%20comparison%20of%203%20controller%20regimens%20for%20mild-moderate%20persistent%20childhood%20asthma:%20the%20Pediatric%20Asthma%20Controller%20trial&author=CA%20Sorkness&author=RF%20Lemanske&author=DT%20Mauger&author=SJ%20Boehmer&author=VM%20Chinchilli&volume=119&publication_year=2007&pages=64-72&pmid=17140647&doi=10.1016/j.jaci.2006.09.042&)

18. Boushey HA, Sorkness CA, King TS, Sullivan SD, Fahy JV, Lazarus SC, Chinchilli VM, Craig TJ, Dimango EA, Deykin A, et al. Daily versus as-needed corticosteroids for mild persistent asthma. N Engl J Med 2005;352:1519–1528  [DOI](https://doi.org/10.1056/NEJMoa042552) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15829533/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Daily%20versus%20as-needed%20corticosteroids%20for%20mild%20persistent%20asthma&author=HA%20Boushey&author=CA%20Sorkness&author=TS%20King&author=SD%20Sullivan&author=JV%20Fahy&volume=352&publication_year=2005&pages=1519-1528&pmid=15829533&doi=10.1056/NEJMoa042552&)

19. Lazarus SC, Boushey HA, Fahy JV, Chinchilli VM, Lemanske RF, Jr, Sorkness CA, Kraft M, Fish JE, Peters SP, Craig T, et al. Long-acting beta2-agonist monotherapy vs continued therapy with inhaled corticosteroids in patients with persistent asthma: a randomized controlled trial. JAMA 2001;285:2583–2593  [DOI](https://doi.org/10.1001/jama.285.20.2583) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11368732/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA&title=Long-acting%20beta2-agonist%20monotherapy%20vs%20continued%20therapy%20with%20inhaled%20corticosteroids%20in%20patients%20with%20persistent%20asthma:%20a%20randomized%20controlled%20trial&author=SC%20Lazarus&author=HA%20Boushey&author=JV%20Fahy&author=VM%20Chinchilli&author=RF%20Lemanske&volume=285&publication_year=2001&pages=2583-2593&pmid=11368732&doi=10.1001/jama.285.20.2583&)

20. Lemanske RF, Jr, Sorkness CA, Mauger EA, Lazarus SC, Boushey HA, Fahy JV, Drazen JM, Chinchilli VM, Craig T, Fish JE, et al. Inhaled corticosteroid reduction and elimination in patients with persistent asthma receiving salmeterol: a randomized controlled trial. JAMA 2001;285:2594–2603  [DOI](https://doi.org/10.1001/jama.285.20.2594) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11368733/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA&title=Inhaled%20corticosteroid%20reduction%20and%20elimination%20in%20patients%20with%20persistent%20asthma%20receiving%20salmeterol:%20a%20randomized%20controlled%20trial&author=RF%20Lemanske&author=CA%20Sorkness&author=EA%20Mauger&author=SC%20Lazarus&author=HA%20Boushey&volume=285&publication_year=2001&pages=2594-2603&pmid=11368733&doi=10.1001/jama.285.20.2594&)

21. Martin RJ, Szefler SJ, King TS, Kraft M, Boushey HA, Chinchilli VM, Craig TJ, Dimango EA, Deykin A, Fahy JV, et al. The Predicting Response to Inhaled Corticosteroid Efficacy (PRICE) trial. J Allergy Clin Immunol 2007;119:73–80  [DOI](https://doi.org/10.1016/j.jaci.2006.10.035) | [PMC free article](/articles/PMC2872157/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17208587/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Allergy%20Clin%20Immunol&title=The%20Predicting%20Response%20to%20Inhaled%20Corticosteroid%20Efficacy%20(PRICE)%20trial&author=RJ%20Martin&author=SJ%20Szefler&author=TS%20King&author=M%20Kraft&author=HA%20Boushey&volume=119&publication_year=2007&pages=73-80&pmid=17208587&doi=10.1016/j.jaci.2006.10.035&)

22. Willer CJ, Sanna S, Jackson AU, Scuteri A, Bonnycastle LL, Clarke R, Heath SC, Timpson NJ, Najjar SS, Stringham HM, et al. Newly identified loci that influence lipid concentrations and risk of coronary artery disease. Nat Genet 2008;40:161–169  [DOI](https://doi.org/10.1038/ng.76) | [PMC free article](/articles/PMC5206900/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18193043/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Newly%20identified%20loci%20that%20influence%20lipid%20concentrations%20and%20risk%20of%20coronary%20artery%20disease&author=CJ%20Willer&author=S%20Sanna&author=AU%20Jackson&author=A%20Scuteri&author=LL%20Bonnycastle&volume=40&publication_year=2008&pages=161-169&pmid=18193043&doi=10.1038/ng.76&)

23. A haplotype map of the human genome. Nature 2005;437:1299–1320  [DOI](https://doi.org/10.1038/nature04226) | [PMC free article](/articles/PMC1880871/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16255080/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=A%20haplotype%20map%20of%20the%20human%20genome&volume=437&publication_year=2005&pages=1299-1320&pmid=16255080&doi=10.1038/nature04226&)

24. Purcell S, Neale B, Todd-Brown K, Thomas L, Ferreira MA, Bender D, Maller J, Sklar P, de Bakker PI, Daly MJ, et al. Plink: a tool set for whole-genome association and population-based linkage analyses. Am J Hum Genet 2007;81:559–575  [DOI](https://doi.org/10.1086/519795) | [PMC free article](/articles/PMC1950838/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17701901/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=Plink:%20a%20tool%20set%20for%20whole-genome%20association%20and%20population-based%20linkage%20analyses&author=S%20Purcell&author=B%20Neale&author=K%20Todd-Brown&author=L%20Thomas&author=MA%20Ferreira&volume=81&publication_year=2007&pages=559-575&pmid=17701901&doi=10.1086/519795&)

25. DerSimonian R, Laird N. Meta-analysis in clinical trials. Control Clin Trials 1986;7:177–188  [DOI](https://doi.org/10.1016/0197-2456(86)90046-2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/3802833/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Control%20Clin%20Trials&title=Meta-analysis%20in%20clinical%20trials&author=R%20DerSimonian&author=N%20Laird&volume=7&publication_year=1986&pages=177-188&pmid=3802833&doi=10.1016/0197-2456(86)90046-2&)

26. Cordell HJ, Clayton DG. Genetic association studies. Lancet 2005;366:1121–1131  [DOI](https://doi.org/10.1016/S0140-6736(05)67424-7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16182901/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Genetic%20association%20studies&author=HJ%20Cordell&author=DG%20Clayton&volume=366&publication_year=2005&pages=1121-1131&pmid=16182901&doi=10.1016/S0140-6736(05)67424-7&)

27. Jorde LB. Linkage disequilibrium as a gene-mapping tool. Am J Hum Genet 1995;56:11–14  [PMC free article](/articles/PMC1801317/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7825565/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=Linkage%20disequilibrium%20as%20a%20gene-mapping%20tool&author=LB%20Jorde&volume=56&publication_year=1995&pages=11-14&pmid=7825565&)

28. Bollag RJ, Siegfried Z, Cebra-Thomas JA, Garvey N, Davison EM, Silver LM. An ancient family of embryonically expressed mouse genes sharing a conserved protein motif with the T locus. Nat Genet 1994;7:383–389  [DOI](https://doi.org/10.1038/ng0794-383) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7920656/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=An%20ancient%20family%20of%20embryonically%20expressed%20mouse%20genes%20sharing%20a%20conserved%20protein%20motif%20with%20the%20T%20locus&author=RJ%20Bollag&author=Z%20Siegfried&author=JA%20Cebra-Thomas&author=N%20Garvey&author=EM%20Davison&volume=7&publication_year=1994&pages=383-389&pmid=7920656&doi=10.1038/ng0794-383&)

29. Herrmann BG, Labeit S, Poustka A, King TR, Lehrach H. Cloning of the T gene required in mesoderm formation in the mouse. Nature 1990;343:617–622  [DOI](https://doi.org/10.1038/343617a0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/2154694/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Cloning%20of%20the%20T%20gene%20required%20in%20mesoderm%20formation%20in%20the%20mouse&author=BG%20Herrmann&author=S%20Labeit&author=A%20Poustka&author=TR%20King&author=H%20Lehrach&volume=343&publication_year=1990&pages=617-622&pmid=2154694&doi=10.1038/343617a0&)

30. Showell C, Binder O, Conlon FL. T-box genes in early embryogenesis. Dev Dyn 2004;229:201–218  [DOI](https://doi.org/10.1002/dvdy.10480) | [PMC free article](/articles/PMC1635809/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14699590/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Dev%20Dyn&title=T-box%20genes%20in%20early%20embryogenesis&author=C%20Showell&author=O%20Binder&author=FL%20Conlon&volume=229&publication_year=2004&pages=201-218&pmid=14699590&doi=10.1002/dvdy.10480&)

31. Park JC, Chae YK, Son CH, Kim MS, Lee J, Ostrow K, Sidransky D, Hoque MO, Moon C. Epigenetic silencing of human T (brachyury homologue) gene in non-small-cell lung cancer. Biochem Biophys Res Commun 2008;365:221–226  [DOI](https://doi.org/10.1016/j.bbrc.2007.10.144) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17980147/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem%20Biophys%20Res%20Commun&title=Epigenetic%20silencing%20of%20human%20T%20(brachyury%20homologue)%20gene%20in%20non-small-cell%20lung%20cancer&author=JC%20Park&author=YK%20Chae&author=CH%20Son&author=MS%20Kim&author=J%20Lee&volume=365&publication_year=2008&pages=221-226&pmid=17980147&doi=10.1016/j.bbrc.2007.10.144&)

32. Naiche LA, Harrelson Z, Kelly RG, Papaioannou VE. T-box genes in vertebrate development. Annu Rev Genet 2005;39:219–239  [DOI](https://doi.org/10.1146/annurev.genet.39.073003.105925) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16285859/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Annu%20Rev%20Genet&title=T-box%20genes%20in%20vertebrate%20development&author=LA%20Naiche&author=Z%20Harrelson&author=RG%20Kelly&author=VE%20Papaioannou&volume=39&publication_year=2005&pages=219-239&pmid=16285859&doi=10.1146/annurev.genet.39.073003.105925&)

33. Hoffmann A, Czichos S, Kaps C, Bachner D, Mayer H, Kurkalli BG, Zilberman Y, Turgeman G, Pelled G, Gross G, et al. The T-box transcription factor brachyury mediates cartilage development in mesenchymal stem cell line c3h10t1/2. J Cell Sci 2002;115:769–781  [DOI](https://doi.org/10.1242/jcs.115.4.769) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11865033/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Cell%20Sci&title=The%20T-box%20transcription%20factor%20brachyury%20mediates%20cartilage%20development%20in%20mesenchymal%20stem%20cell%20line%20c3h10t1/2&author=A%20Hoffmann&author=S%20Czichos&author=C%20Kaps&author=D%20Bachner&author=H%20Mayer&volume=115&publication_year=2002&pages=769-781&pmid=11865033&doi=10.1242/jcs.115.4.769&)

34. Donn R, Berry A, Stevens A, Farrow S, Betts J, Stevens R, Clayton C, Wang J, Warnock L, Worthington J, et al. Use of gene expression profiling to identify a novel glucocorticoid sensitivity determining gene, bmprii. FASEB J 2007;21:402–414  [DOI](https://doi.org/10.1096/fj.06-7236com) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17185747/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=FASEB%20J&title=Use%20of%20gene%20expression%20profiling%20to%20identify%20a%20novel%20glucocorticoid%20sensitivity%20determining%20gene,%20bmprii&author=R%20Donn&author=A%20Berry&author=A%20Stevens&author=S%20Farrow&author=J%20Betts&volume=21&publication_year=2007&pages=402-414&pmid=17185747&doi=10.1096/fj.06-7236com&)

35. Pollett JB, Trudel S, Stern D, Li ZH, Stewart AK. Overexpression of the myeloma-associated oncogene fibroblast growth factor receptor 3 confers dexamethasone resistance. Blood 2002;100:3819–3821  [DOI](https://doi.org/10.1182/blood-2002-02-0608) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12393593/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Overexpression%20of%20the%20myeloma-associated%20oncogene%20fibroblast%20growth%20factor%20receptor%203%20confers%20dexamethasone%20resistance&author=JB%20Pollett&author=S%20Trudel&author=D%20Stern&author=ZH%20Li&author=AK%20Stewart&volume=100&publication_year=2002&pages=3819-3821&pmid=12393593&doi=10.1182/blood-2002-02-0608&)

36. Goering LM, Hoshijima K, Hug B, Bisgrove B, Kispert A, Grunwald DJ. An interacting network of T-box genes directs gene expression and fate in the zebrafish mesoderm. Proc Natl Acad Sci USA 2003;100:9410–9415  [DOI](https://doi.org/10.1073/pnas.1633548100) | [PMC free article](/articles/PMC170932/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12883008/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20USA&title=An%20interacting%20network%20of%20T-box%20genes%20directs%20gene%20expression%20and%20fate%20in%20the%20zebrafish%20mesoderm&author=LM%20Goering&author=K%20Hoshijima&author=B%20Hug&author=B%20Bisgrove&author=A%20Kispert&volume=100&publication_year=2003&pages=9410-9415&pmid=12883008&doi=10.1073/pnas.1633548100&)

37. Tantisira KG, Lasky-Su J, Harada M, Murphy A, Litonjua AA, Himes BE, Lange C, Lazarus R, Sylvia J, Klanderman B, et al. Genome-wide association of GLCCI1 with asthma steroid treatment response. N Engl J Med 2011;365:1173–1183  [DOI](https://doi.org/10.1056/NEJMoa0911353) | [PMC free article](/articles/PMC3667396/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21991891/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Genome-wide%20association%20of%20GLCCI1%20with%20asthma%20steroid%20treatment%20response&author=KG%20Tantisira&author=J%20Lasky-Su&author=M%20Harada&author=A%20Murphy&author=AA%20Litonjua&volume=365&publication_year=2011&pages=1173-1183&pmid=21991891&doi=10.1056/NEJMoa0911353&)
