# Atenolol Induced HDL-C Change in the Pharmacogenomic Evaluation of Antihypertensive Responses (PEAR) Study

## Metadata
**Authors:** Caitrin W McDonough, Nancy K Gillis, Abdullah Alsultan, Shin-Wen Chang, Marina Kawaguchi-Suzuki, Jason E Lang, Mohamed Hossam A Shahin, Thomas W Buford, Nihal M El Rouby, Ana CC Sá, Taimour Y Langaee, John G Gums, Arlene B Chapman, Rhonda M Cooper-DeHoff, Stephen T Turner, Yan Gong, Julie A Johnson
**Journal:** PLoS ONE
**Date:** 2013 Oct 7
**DOI:** [10.1371/journal.pone.0076984](https://doi.org/10.1371/journal.pone.0076984)
**PMID:** 24116192
**PMCID:** PMC3792156
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3792156/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC3792156/pdf/pone.0076984.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC3792156/pdf/pone.0076984.pdf)

## Abstract

We sought to identify novel pharmacogenomic markers for HDL-C response to atenolol in participants with mild to moderate hypertension. We genotyped 768 hypertensive participants from the Pharmacogenomic Evaluation of Antihypertensive Responses (PEAR) study on the Illumina HumanCVD Beadchip. During PEAR, participants were randomized to receive atenolol or hydrochlorothiazide. Blood pressure and cholesterol levels were evaluated at baseline and after treatment. This study focused on participants treated with atenolol monotherapy. Association with atenolol induced HDL-C change was evaluated in 232 whites and 152 African Americans using linear regression. No SNPs achieved a Bonferroni corrected P-value. However, we identified 13 regions with consistent association across whites and African Americans. The most interesting of these regions were seven with prior associations with HDL-C, other metabolic traits, or functional implications in the lipid pathway: GALNT2, FTO, ABCB1, LRP5, STARD3NL, ESR1, and LIPC. Examples are rs2144300 in GALNT2 in whites (P=2.29x10-4, β=-1.85 mg/dL) and rs12595985 in FTO in African Americans (P=2.90x10-4, β=4.52 mg/dL), both with consistent regional association (P<0.05) in the other race group. Additionally, baseline GALNT2 expression differed by rs2144300 genotype in whites (P=0.0279). In conclusion, we identified multiple gene regions associated with atenolol induced HDL-C change that were consistent across race groups, several with functional implications or prior associations with HDL-C.

## Introduction

Hypertension is a major risk factor for cardiovascular disease (CVD), which is the leading cause of morbidity and mortality in the United States [[1](#B1)]. Over 76 million Americans have hypertension, of which an estimated 71% are taking antihypertensive medications [[1](#B1)]. Thiazide diuretics, β-blockers, angiotensin converting enzyme inhibitors, angiotensin receptor blockers, and calcium channel blockers are all acceptable first-line treatments for hypertension [[2](#B2)]. However, there has been debate about the first-line role of β-blockers and diuretics due to their adverse metabolic effects on glucose, insulin, and lipid levels [[3](#B3)]. For example, β-blockers have been shown to significantly decrease high-density lipoprotein cholesterol (HDL-C) levels, but the underlying mechanism is unclear [[4](#B4)]. These metabolic effects, including HDL-C reduction, are also strong contributors to the risk of coronary heart disease and stroke [[1](#B1)].

Data from four large, prospective studies, including The Framingham Heart Study, suggest that every 1 mg/dL (0.026 mmol/L) decrease in HDL-C is associated with a 2-3% increase in risk for coronary heart disease [[5](#B5)]. Additionally, post hoc analysis of the Treating to New Targets (TNT) trial demonstrated that low levels of HDL-C were predictive of higher rates of CVD, even at very low levels of low-density lipoprotein cholesterol (LDL-C) (< 70 mg/dL) [[6](#B6)] and low HDL-C levels were also predictive of major cardiovascular events in patients treated with statins [7]. However, randomized controlled trials designed to determine if increases in HDL-C prevent coronary heart disease and major cardiovascular events demonstrate inconsistent results [8,9]. Additionally, it remains unclear if decreases in HDL-C in response to β-blocker therapy are harmful in a similar way as low HDL-C observed in epidemiological studies.

Clear data exist regarding the influence of genetics on HDL-C levels. In a study of the sex specific genetic architecture of many quantitative traits, the heritability of HDL-C was estimated at 52% in males and 70% in females [[10](#B10)]. HDL-C is a sexually dimorphic trait, with higher HDL-C levels typically observed in females [[10](#B10)]. There have been multiple genome-wide association studies (GWAS) investigating the effect of genetic variants on blood lipid levels. In a large-scale meta-analysis including many of these GWAS, 45 loci were found to influence HDL-C [[11](#B11)]. When examining the entire set of single nucleotide polymorphisms (SNPs) associated with HDL-C, 12.1% of the total variance in HDL-C was explained, and ~25-30% of the genetic variance was explained [[11](#B11)].

While there have been many studies investigating genetic influences on blood lipid levels, including HDL-C [[11](#B11)], the genetics of HDL-C response to β-blockers remains unexplored. In order to evaluate potential genetic effects, we assessed SNPs across the genome from the Illumina HumanCVD chip [[12](#B12)] for association with atenolol induced HDL-C change in the Pharmacogenomic Evaluation of Antihypertensive Responses (PEAR) study [[13](#B13)].

## Methods

### Ethics Statement

All study participants provided written informed consent and the study protocol was approved by the Institutional Review Boards at all participating clinical trial study sites: University of Florida, Mayo Clinic, and Emory University. The study was conducted according to the principles expressed in the Declaration of Helsinki.

### Study Design and Participants

The PEAR study was a prospective, multicenter, randomized, open-label study (clinicaltrials.gov [NCT00246519](https://clinicaltrials.gov/ct2/show/NCT00246519)). The design of the PEAR study has been described previously [[13](#B13)]. Briefly, PEAR included 768 participants with mild to moderate hypertension between the ages of 17 to 65, without other comorbidities including diabetes. Participants were included if they had an average home diastolic blood pressure (DBP) between 86 and 110 mmHg and office DBP between 91 and 110 mmHg at the end of an average 4 week washout period. After washout, study participants were randomized to receive atenolol 50 mg or hydrochlorothiazide (HCTZ) 12.5 mg daily. If BP remained greater than 120/70 mmHg after three weeks of treatment, doses were titrated to atenolol 100 mg or HCTZ 25 mg daily, and treatment was continued for an additional 6 weeks. The other agent was then added based on BP > 120/70 mmHg, with similar dose titration for 6-9 weeks of combination treatment. During atenolol monotherapy, >82% of whites and >88% of African Americans underwent the dose titration step. In addition to blood pressure measures, laboratory measures including HDL-C were acquired at baseline, after monotherapy, and after combination therapy [[13](#B13)].

### Determination of Blood Lipid Concentration

Fasting serum levels of lipids (including HDL-C) were determined using a Hitachi 911 Chemistry Analyzer (Roche Diagnostics, Indianapolis, IN). All laboratory parameters were measured at a central laboratory at the Mayo Clinic. All samples were tested in duplicate, with the means of the duplicate samples reported.

### SNP Genotyping and Quality Control

DNA from study participants was genotyped on the Illumina HumanCVD Beadchip (Illumina, San Diego, CA). This is a gene-centric array containing ~50,000 SNPs in ~2,100 genes involved in cardiovascular, inflammatory, and metabolic processes [[12](#B12)]. Genotyping was performed on Illumina’s iScan System using the Infinium II Assay (Illumina, San Diego, CA). Genotypes were called using GenomeStudio Software version 2011.1 and the Genotyping Module version 1.9 calling algorithm (Illumina, San Diego, CA). Samples were excluded if genotype call rates were below 90% and SNPs were excluded if genotype call rates were below 90%. Eighty-one blind duplicates were included in genotyping and had a concordance rate of 99.992%. Gender was confirmed from X chromosome genotype data, and those who were discordant were excluded (*n*=1). Cryptic relatedness was estimated by pairwise identity-by-descent (IBD) analysis implemented using PLINK ([http://pngu.mgh.harvard.edu/purcell/plink/](http://pngu.mgh.harvard.edu/purcell/plink/)) [[14](#B14)]. One pair of monozygotic twins was identified in this analysis and one twin was removed (*n*=1). Five pairs of samples were identified as first-degree relatives; these individuals were kept for the analyses. Heterozygosity was assessed using PLINK, by estimating the inbreeding coefficient, F. One sample had F values > 4 negative standard deviations from the mean and was excluded. The final genetic dataset consisted of 765 samples. This analysis focused on the 232 whites and 152 African-Americans randomized to the atenolol monotherapy arm.

### Statistical Analysis

To address the issue of population substructure and admixture in our racially and ethnically diverse population, a Principal Component Analysis (PCA) was performed in all samples on a linkage disequilibrium (LD) pruned dataset using the EIGENSTRAT method [[15](#B15)]. Race/ethnic groups were confirmed with PCA clustering results, and participants who identified with a race/ethnic group other than white or African American were collapsed into the group they clustered nearest. The top principal components (PCs 1-2) that provided the best separation of ancestry clusters were selected to be included as covariates for analysis.

Before analysis, all SNPs with a minor allele frequency (MAF) < 3% were removed from the data set due to limited power. This was performed separately for each race group, leaving 33,785 SNPs for analysis in the white race group, and 39,718 SNPs for analysis in the African Americans. SNPs were tested for departure from Hardy-Weinberg Equilibrium (HWE) in each race group using an exact test. Linear regression was used to model SNP effects on HDL-C response to atenolol monotherapy using PLINK ([http://pngu.mgh.harvard.edu/purcell/plink/](http://pngu.mgh.harvard.edu/purcell/plink/)) [[14](#B14)]. The primary analysis was based on the additive genetic model. Analyses were adjusted for age, sex, baseline HDL-C, and principal components for ancestry. Additional covariates were investigated (BMI, exercise, smoking status, alcohol use, baseline LDL, baseline glucose, lipid lowering medications, and fish oil supplementation), but none were shown to be predictive of HDL-C response in our population, and therefore, were not included as covariates.

Statistical significance was defined as a Bonferroni corrected *P*-value for the number of independent SNPs (*P*<2.4x10^-6^ in whites and *P*<1.9x10^-6^ in African Americans) [[16](#B16)]. Since no SNPs achieved these *P*-values, top hits of interest were identified as regions showing evidence of association in both race groups. The screening threshold was set at *P*<1x10^-3^ in either race group. SNPs meeting the screening *P*-value were examined in the other race group for evidence of association. Next, the gene region (gene ± 10kb) was also examined for evidence of association, as patterns of LD often differ between race groups and different tag SNPs could be observed in each race group. Regional SNPs were not required to have effects in the same direction, as due to differences in racial LD and allele frequency, the different alleles could be tagging the same unknown causal variant. The significance threshold for validation within the gene region in the second race group was *P*<0.005, or *P*<0.05 if there was evidence from previous HDL-C studies or the encoded protein plays a clear role in HDL-C/lipid trafficking/metabolism. Adjusted mean change by genotype was calculated for the top hits using the general linear models procedure in SAS version 9.2 (SAS Institute Inc., Cary, North Carolina, USA). Additionally, top hits with low minor allele frequencies (<0.05) were investigated to determine if single individuals were strongly contributing to the observed association. One white participant was identified with an extreme HDL-C response, standardized residual value from linear regression including non-genetic covariates outside 4 standard deviations, and was removed in a sensitivity analysis. Similarly, for rs12595985 in *FTO*, sensitivity analyses were also run using a dominant genetic model, collapsing the AA homozygote with the AC heterozygotes, and using an additive genetic model after removing the single African American AA homozygote.

FastSNP and SNPNexus were used to predict the potential functional consequences of the top hits [[17](#B17),[18](#B18)]. Results of association were displayed using Haploview, WGAViewer, and LocusZoom [[19](#B19)-[21](#B21)].

### Gene Expression Analysis

Gene expression analysis was performed for the top genes that were 1) previously associated with HDL-C or lipid trafficking/metabolism, 2) expressed in lymphocytes, and 3) had RNA available for at least 10 minor allele homozygotes and/or 10 minor allele carriers. Three genes met these criteria: *GALNT2*, *FTO*, and *LRP5*; however, *LRP5* was expressed at extremely low levels in lymphocytes and was not tested further. Expression of *GALNT2* was measured in 34 PEAR whites (11 AA, 12 GA, and 11 GG at rs2144300) and expression of *FTO* was measured in 35 PEAR African Americans (11 AC, and 24 CC at rs12595985). RNA was isolated from whole blood using the PAXgene Blood RNA Kit IVD (Qiagen, Valenica, CA, USA), before atenolol treatment, and converted to cDNA. Gene expression was measured by quantitative real-time RT-PCR using Taqman Gene Expression Assays and the Taqman 7900HT Real Time PCR System (Applied Biosystems, Foster City, CA, USA). Expression levels were normalized to the reference gene β-2-microglobulin. Relative gene expression was calculated using the 2^-ΔCt^ method [[22](#B22)]. Expression levels between genotype groups at baseline (before atenolol treatment) were compared using a trend test (*GALNT2*) or an unpaired *t*-test (*FTO*). The significance threshold was set at *P*<0.05.

## Results

The results presented are for PEAR study participants on atenolol monotherapy (i.e. the atenolol arm). Baseline characteristics and demographics are shown in [Table 1](#pone-0076984-t001). Among a total of 384 study participants in the atenolol group, 58.6% were white and 38.3% were African American. Participants were overweight (average BMI = 30.8 kg/m^2^), and 12.8% were current smokers. The average baseline HDL-C was 49.7 mg/dL.

### Table 1. Baseline characteristics and demographics of PEAR study participants treated with atenolol monotherapy.

| Characteristic | Atenolol (n=384) |
| --- | --- |
| Age (years) | 48.6 ± 9.2 |
| Female, n (%) | 215 (56.0) |
| BMI (kg/m2) | 30.8 ± 5.9 |
| Waist circumference (cm) | 97.6 ± 13.1 |
| Race/ethnicity |   |
| African American, n (%) | 147 (38.3) |
| White, n (%) | 225 (58.6) |
| Other, n (%) | 12 (3.1) |
| Genetic Race/ethnicity |   |
| African American, n (%) | 152 (39.6) |
| White, n (%) | 232 (60.4) |
| Current smoker, n (%) | 49 (12.8) |
| Clinic Blood Pressure (mm Hg) |   |
| Systolic | 151.2 ± 12.3 |
| Diastolic | 98.3 ± 6.2 |
| Fasting Laboratory Measures |   |
| Glucose (mg/dL) | 91.1 ± 11.9 |
| Triglycerides (mg/dL) | 122.4 ± 84.2 |
| LDL-C (mg/dL) | 119.1 ± 33.4 |
| HDL-C (mg/dL) | 49.7 ± 14.8 |

Table 1 Caption: Values are presented as mean ± standard deviation, unless otherwise noted. BMI: Body Mass Index, LDL-C: low-density lipoprotein cholesterol, HDL-C: high-density lipoprotein cholesterol

The Manhattan plots and corresponding Q-Q plots for HDL-C response to atenolol are shown in [Figure S1](#pone.0076984.s001) and [Figure S2](#pone.0076984.s002), respectively. No SNPs achieved a Bonferroni correct *P*-value. Overall, 25 SNPs met the screening *P*-value (*P*<1x10^-3^) in whites and 42 SNPs met the screening *P*-value in the African Americans; however, no SNP replicated in the other race group, in the same direction. After examining the identified gene regions in the other race for evidence of consistent association, thirteen regions met our threshold for association (*P*<0.005 or *P*<0.05 with prior HDL-C or lipid association, [Table 2](#pone-0076984-t002)). The allele frequencies, genotype counts, and Hardy-Weinberg equilibrium *P*-values for these SNPs are shown in [Table S1](#pone.0076984.s003). All SNPs identified in these gene regions were in Hardy-Weinberg Equilibrium (Hardy-Weinberg *P*-value > 0.01).

### Table 2. Top gene regions associated with atenolol induced change in HDL-Ca.

| Locus |  |  |  | Initial Signal |  |  |  |  | Regional Validation Signal |  |  |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| Chr | Position (Mb) | Gene | Prior HDL or Lipid Association [Ref] | Lead SNP | MA | MAF | Effect (mg/dl) | P-value | Lead SNP | MA | MAF | Effect (mg/dl) | P-value |
|   |   |   |   | Whites |  |  |  |  | African Americans |  |  |  |  |
| 1 | 185.2 | PLA2G4A |   | rs10157410 | C | 0.101 | 3.46 | 6.66E-05 | rs4648287 | G | 0.049 | 3.21 | 0.0040 |
| 1 | 228.4 | GALNT2 | Yes [11] | rs2144300 | G | 0.422 | -1.85 | 2.29E-04 | rs2144297 | A | 0.345 | -1.92 | 0.0233 |
| 7 | 38.2 | STARD3NL | Yes [28] | rs10240718 | A | 0.078 | 3.23 | 3.98E-04 | rs7795499 | A | 0.329 | 1.76 | 0.0350 |
| 11 | 68.0 | LRP5 | Yes [27] | rs3736228 | A | 0.136 | 2.64 | 5.07E-04 | rs4988331 | A | 0.168 | 2.12 | 0.0281 |
| 13 | 77.4 | EDNRB |   | rs3818416 | A | 0.241 | 2.21 | 2.97E-04 | rs3818416 | A | 0.329 | -2.24 | 0.0028 |
| 15 | 56.6 | LIPC | Yes [11] | rs9652472 | G | 0.041 | -4.40 | 6.02E-04 | rs10518978 | A | 0.123 | -3.01 | 0.0101 |
| 16 | 65.5 | CDH16 |   | rs3743725 | A | 0.030 | 5.23 | 7.51E-04 | rs13336470 | A | 0.322 | 2.63 | 0.0041 |
|   |   |   |   | African Americans |  |  |  |  | Whites |  |  |  |  |
| 4 | 57.5 | REST |   | rs6847086 | A | 0.461 | 2.67 | 7.14E-04 | rs4109037 | A | 0.136 | 2.33 | 0.0025 |
| 6 | 152.5 | ESR1 | Yes [29] | rs3020384 | C | 0.487 | 2.65 | 7.63E-04 | rs12199198 | C | 0.067 | -2.07 | 0.0372 |
| 7 | 87.1 | ABCB1 | Yes [26] | rs3213619 | G | 0.072 | 5.22 | 2.74E-04 | rs10267099 | G | 0.220 | 1.62 | 0.0059 |
| 8 | 10.1 | MSRA |   | rs2975721 | T | 0.385 | 2.74 | 4.69E-04 | rs6601419 | A | 0.338 | 1.66 | 0.0029 |
| 13 | 94.5 | ABCC4 |   | rs7319001 | A | 0.095 | 4.92 | 5.39E-04 | rs1189470 | C | 0.154 | 2.12 | 0.0036 |
| 16 | 52.4 | FTO | Yes [25] | rs12595985 | A | 0.109 | 4.52 | 2.90E-04 | rs9940629 | A | 0.448 | 1.56 | 0.0041 |

Table 2 Caption: Chr: Chromosome; Mb: Megabase; MA: Minor Allele; MAF: Minor Allele Frequency aRegional validation SNPs were not required to have effects in the same direction, as due to differences in racial LD and allele frequency, the different alleles could be tagging the same unknown causal variant.

In whites, the strongest association was rs10157410 in *PLA2G4A* (*P*=6.66x10^-5^, beta=3.46, [Table 2](#pone-0076984-t002)), with consistent regional association at rs4648287 in the African Americans (*P*=0.0040, beta=3.21). *PLA2G4A* is located at chromosome 1q31.1 and encodes the cytosolic phospholipase A2. In African Americans, the strongest association was seen at rs3213619 on chromosome 7q21.12, with a *P*-value of 2.74x10^-4^ and a beta of 5.22 ([Table 2](#pone-0076984-t002)). rs3213619 lies in *ABCB1*, ATP-binding cassette sub-family B member 1, and consistent regional association was also observed at rs10267099 in whites (*P*=0.0059, beta=1.62).

Two other regions are of particular note. First, *GALNT2*, polypeptide N-acetylgalactosaminyltransferase 2, which has been previously associated in GWAS studies with circulating HDL-C levels [[11](#B11)], was associated with atenolol induced HDL-C change. In whites the G allele of rs2144300, one of the same SNPs identified from GWAS studies [[23](#B23)], was associated with an atenolol induced decrease in HDL-C (*P*=2.29x10^-4^, [Table 2](#pone-0076984-t002), [Figure 1A](#pone-0076984-g001)), with a mean HDL-C change of -4.18 mg/dL, -3.03 mg/dL, and -0.63 mg/dL for the GG, GA, and AA genotypes, respectively ([Figure 1B](#pone-0076984-g001)). In African Americans, a signal was observed in *GALNT2* with the A allele of rs2144297 and a decrease in HDL-C in response to atenolol (*P*=0.0233, [Table 2](#pone-0076984-t002), [Figure 1C](#pone-0076984-g001)). The mean change in HDL-C was -5.39 mg/dL, -3.61 mg/dL, and -1.63 mg/dL for the AA, AG, and GG genotypes, respectively ([Figure 1D](#pone-0076984-g001)).

### Figure 1. GALNT2 Regional Plots and adjusted HDL-C response by genotype.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/acba/3792156/0b358d51dace/pone.0076984.g001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3792156_pone.0076984.g001.jpg)

(A) Regional Plot in PEAR whites. (B) HDL-C response in PEAR whites by rs2144300 genotype. (C) Regional Plot in PEAR African Americans. (D) HDL-C response in PEAR African Americans by rs2144297 genotype. Error bars represent standard error.

The second gene, *FTO*, the fat mass and obesity associated gene, has been previously associated with BMI, obesity and type 2 diabetes in GWAS [[24](#B24)], and was recently associated with HDL-C in a large, gene-centric analysis of the CVD array used herein [[25](#B25)]. In African Americans, the A allele of rs12595985 was associated with an increase in HDL-C in response to atenolol treatment, with an increase of 13.03 mg/dL in the AA homozygote, an average decrease of 0.07 mg/dL in AC heterozygotes, and an average decrease of 3.97 mg/dL in CC homozygotes (*P*=2.90x10^-4^, [Table 2](#pone-0076984-t002), [Figures 2A](#pone-0076984-g002) and [2B](#pone-0076984-g002)). The observed signal in whites was with rs9940629, with the G allele associated with a greater reduction in HDL-C after atenolol therapy (*P*=0.0041, [Table 2](#pone-0076984-t002), [Figure 2C](#pone-0076984-g002)). The mean change in HDL-C was -1.09 mg/dL, -1.90 mg/dL, and -4.04 mg/dL for the AA, AG, and GG genotypes, respectively ([Figure 2D](#pone-0076984-g002)).

### Figure 2. FTO Regional Plots and adjusted HDL-C response by genotype.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/acba/3792156/51d8dc460860/pone.0076984.g002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3792156_pone.0076984.g002.jpg)

(A) Regional Plot in PEAR African Americans. (B) HDL-C response in PEAR African Americans by rs12595985 genotype. (C) Regional Plot in PEAR whites. (D) HDL-C response in PEAR whites by rs9940629 genotype. Error bars represent standard error.

Top initial signals with low minor allele frequencies were further investigated to determine if single study participants with an extreme HDL-C response to atenolol were driving the observed signals. One white participant with an HDL-C change of -36.4 mg/dL was identified and the individual was removed in a sensitivity analysis ([Table S2](#pone.0076984.s004)). All of the initial signals in whites remained except for *LIPC* rs9652472 (sensitivity *P*-value=0.3998). Additionally, sensitivity analyses were conducted in African Americans at rs12595985 in *FTO* ([Table S3](#pone.0076984.s005)). The signal remained under both a dominant genetic model, collapsing the AA homozygote with the AC heterozygotes (sensitivity *P*-value=9.71x10^-4^) and an additive genetic model, after removing the single AA homozygote, (sensitivity *P*-value=3.18x10^-3^).

The putative functional implications of the lead SNPs from the initial signals in the 13 gene regions are shown in [Table S4](#pone.0076984.s006). rs3736228, in *LRP5* (low density lipoprotein receptor-related protein 5), is a non-synonymous SNP causing a conservative amino acid change of an alanine to valine at amino acid position 1330. Additionally, rs3736228 was predicted by FastSNP to affect splicing regulation. All of the other SNPs identified were located in intronic regions, or upstream or downstream of the nearest gene.

In order to examine the possible function of two the top SNPs, we measured gene expression at baseline (before atenolol treatment) of *GALNT2* by rs2144300 genotype in 34 whites and of *FTO* by rs12595985 genotype in 35 African Americans. For *GALNT2*, study participants with one or two copies of the G allele had a 1.59 and 1.75 fold increase in relative expression at baseline, respectively, compared to AA homozygotes (*P*=0.0279, [Figure 3](#pone-0076984-g003)). There was no difference in baseline expression between genotype groups for *FTO* (data not shown).

### Figure 3. Plot of relative gene expression of GALNT2 by rs2144300 genotype.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/acba/3792156/fd984992a49e/pone.0076984.g003.jpg)

Gene expression analysis performed in 34 PEAR whites, normalized to β-2-microglobulin. Error bars represent standard error.

## Discussion

We report, for the first time to our knowledge, a genome-spanning analysis using the HumanCVD Beadchip, which contains cardiovascular, metabolic and inflammatory candidate genes, to identify genetic variants associated with atenolol induced changes in HDL-C in hypertensive study participants. While no SNPs achieved a Bonferroni corrected *P*-value, using a staged analysis approach that required consistent regional association in white and African American participants, we identified 7 SNPs associated with HDL-C response to atenolol in whites that showed evidence of consistent regional association in African Americans. Additionally, 6 SNPs were found in African Americans with consistent regional association in whites. Most notable were seven gene regions with prior associations with HDL-C or lipid metabolism/trafficking: *GALNT2, FTO*, *ABCB1*, *LRP5*, *STARD3NL*, *ESR1*, and *LIPC* [[11](#B11),[25](#B25)-[29](#B29)].

*GALNT2*, located at 1q42.13, is a member of the GalNAc-transferase family, which transfers N-acetyl galactosamine to the hydroxyl group of a serine or threonine residue in the first step of O-linked oligosaccharide biosynthesis [[30](#B30)]. Common variants in *GALNT2* have previously been associated with both HDL-C and triglyceride levels in numerous GWAS analyses, and subsequently validated [[11](#B11),[23](#B23)]. Additionally, altered expression of *Galnt2* in mouse models, both knockdown and over expression, has been inversely correlated with altered HDL-C levels [[11](#B11)]. This is similar to what we observed; where those individuals with higher gene expression at baseline (G carriers at rs2144300), had greater decreases in HDL-C in response to atenolol, resulting in lower HDL-C levels. These prior data, the association of common SNPs in this gene with HDL-C response to atenolol in whites and African Americans, and the difference in *GALNT2* expression by rs2144300 genotype in whites suggest *GALNT2* may be an important mediator of the observed atenolol associated HDL-C response.

*FTO*, located at 16q12.2, is thought to be involved in the regulation of food intake and to affect lipolysis in adipose tissue [[31](#B31)]. Other SNPs in *FTO* have been previously associated with BMI, obesity and type 2 diabetes [[24](#B24)]. Additionally, *FTO* was recently associated with HDL-C in a large, gene centric analysis [[25](#B25)], and it was suggested that this association with *FTO* and HDL-C may be mediated through *FTO*’ s association with BMI, since the observed association with *FTO* and HDL-C lost significance after BMI adjustment [[25](#B25)]. However, in our data, when we add BMI to the model, rs12595985 remains associated with HDL-C response to atenolol (*P*=3x10^-4^), suggesting the association may be through a different mechanism. Additionally, while we did not observe a significant difference in *FTO* expression by rs12595985 genotype at baseline, this does not preclude that expression differences might exist in more relevant tissues (e.g. liver).

Other top genes are also of note. *ABCB1*, also known as *MDR1* (multi-drug resistance protein 1), belongs to the MDR/TAP subfamily responsible for transporting various molecules across cell membranes. Evidence indicates a link of *ABCB1* variants with circulating lipid profiles, mainly LDL-C, and the efficacy of statins [[26](#B26)]. Furthermore, *ABCB1* 3435C>T (rs1045642) has been associated with efavirenz induced changes in HDL-C [[32](#B32)]. We did not observe association with HDL-C response to atenolol at this SNP in our study; however, because we did observe association in this gene, *ABCB1* and drug associated HDL-C change warrants further investigation. Next, *LRP5*, low-density lipoprotein receptor-related protein 5, encodes a transmembrane low-density lipoprotein receptor. Mice lacking the production of LPR5 had increased total plasma cholesterol levels and impaired glucose tolerance [[27](#B27)]. STARD3 N-terminal L, *STARD3NL*, encodes a late-endosomal protein that binds cholesterol and may assist in endosomal cholesterol transport [[28](#B28)], a process important in maintaining HDL cholesterol levels [[33](#B33)]. Also, *ESR1*, the estrogen receptor 1 gene, was shown associated with HDL-C response to atenolol. Variants in *ESR1* have been associated with many traits including type 2 diabetes in African Americans [[34](#B34)], and HDL-C response to atorvastatin in women [[29](#B29)]. The last gene of particular interest is *LIPC*, hepatic lipase, which is involved in lipoprotein metabolism, and other SNPs in the gene have also been associated with HDL-C in GWAS [[11](#B11)].

Many of the top gene regions identified have shown prior evidence of association with circulating HDL-C levels, HDL-C levels in response to drug therapy, or other metabolic traits or are involved in lipid metabolism. While this analysis was based on a candidate gene chip, the chip includes many genes involved in cardiovascular and inflammatory processes as well as genes involved in metabolic processes [[12](#B12)]. Thus, it is particularly interesting that a connection to HDL-C could be made for many of the gene regions meeting the consistent regional association requirement.

Like any study, the present investigation is not without limitations. For example, we had a relatively small sample size and did not have an external replication cohort. However, to our knowledge, there is not another suitable replication population with data on HDL-C response to a β-blocker at the present time. Also, due to our study design, we only focused on gene regions that showed a signal in both race groups. While true functional variants should be relevant to all race groups, it is very possible that we missed regions of interest due to our consistent regional association requirement and the differences in LD between the two race groups. Additionally, while we have identified gene regions of interest, we have likely not identified the functional variants in these regions. It is more likely we have identified SNPs in LD with the functional variant.

β-blockers are appropriate first-line therapy for hypertension [[2](#B2)]; however, there is growing concern regarding use in patients without compelling indications (i.e., heart failure or myocardial infarction), as the benefits associated with use of β-blockers may not outweigh the potential metabolic and other risks, including a decrease in HDL-C levels [[3](#B3),[4](#B4)]. Our study investigated the possible genetic contributions to atenolol induced change in HDL-C in hypertensive study participants. We found 13 gene regions that showed initial evidence of association in one race (whites or African Americans), and consistent regional association in the other. Of particular interest were seven gene regions with prior associations with HDL-C or other metabolic traits, or functional implications in the lipid pathway: *GALNT2, FTO*, *ABCB1, LRP5*, *STARD3NL*, *ESR1*, and *LIPC*. We also observed differences in *GALNT2* baseline expression by rs2144300 genotype. These results require further investigation in order to confirm the associations, identify potential functional variants, elucidate their role, and establish the mechanism through which they act. However, our findings provide initial insight into genes that may influence atenolol induced change in HDL-C.

## Supporting Information

## Acknowledgments

We acknowledge and thank the participants in the PEAR study. We also acknowledge and thank the PEAR support staff, and study physicians: Drs. George Baramidze, R. Whit Curry, Karen Hall, Frederic Rabari-Oskoui, Dan Rubin, Siegfried Schmidt, and Michel Diab.

## Funding Statement

PEAR was supported by the National Institute of Health Pharmacogenetics Research Network grant U01-GM074492 and the National Center for Advancing Translational Sciences under the award number UL1 TR000064 (University of Florida); UL1 TR000454 (Emory University) and UL1 TR000135 (Mayo Clinic). PEAR efforts at the Mayo Clinic were also supported by funds from the Mayo Foundation. [http://www.nih.gov/](http://www.nih.gov/)[http://www.mayoclinic.com/](http://www.mayoclinic.com/). The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Go AS, Mozaffarian D, Roger VL, Benjamin EJ, Berry JD et al. (2013) Executive summary: heart disease and stroke statistics--2013 update: a report from the American Heart Association. Circulation 127: 143-152. doi: 10.1161/CIR.0b013e318282ab8f. PubMed: 23283859.  [23283859](http://www.ncbi.nlm.nih.gov/pubmed/23283859) | [DOI](https://doi.org/10.1161/CIR.0b013e318282ab8f) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23283859/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circulation&title=Executive%20summary:%20heart%20disease%20and%20stroke%20statistics--2013%20update:%20a%20report%20from%20the%20American%20Heart%20Association&author=AS%20Go&author=D%20Mozaffarian&author=VL%20Roger&author=EJ%20Benjamin&author=JD%20Berry&volume=127&publication_year=2013&pages=143-152&pmid=23283859&doi=10.1161/CIR.0b013e318282ab8f&)

2. Chobanian AV, Bakris GL, Black HR, Cushman WC, Green LA et al. (2003) The Seventh Report of the Joint National Committee on Prevention, Detection, Evaluation, and Treatment of High Blood Pressure: the JNC 7 report. JAMA 289: 2560-2572. doi: 10.1001/jama.289.19.2560. PubMed: 12748199.  [12748199](http://www.ncbi.nlm.nih.gov/pubmed/12748199) | [DOI](https://doi.org/10.1001/jama.289.19.2560) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12748199/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA&title=The%20Seventh%20Report%20of%20the%20Joint%20National%20Committee%20on%20Prevention,%20Detection,%20Evaluation,%20and%20Treatment%20of%20High%20Blood%20Pressure:%20the%20JNC%207%20report&author=AV%20Chobanian&author=GL%20Bakris&author=HR%20Black&author=WC%20Cushman&author=LA%20Green&volume=289&publication_year=2003&pages=2560-2572&pmid=12748199&doi=10.1001/jama.289.19.2560&)

3. Messerli FH, Bangalore S, Julius S (2008) Risk /benefit assessment of beta-blockers and diuretics precludes their use for first-line therapy in hypertension. Circulation 117: 2706-2715; discussion  [DOI](https://doi.org/10.1161/CIRCULATIONAHA.107.695007) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18490538/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circulation&title=Risk%20/benefit%20assessment%20of%20beta-blockers%20and%20diuretics%20precludes%20their%20use%20for%20first-line%20therapy%20in%20hypertension&author=FH%20Messerli&author=S%20Bangalore&author=S%20Julius&volume=117&publication_year=2008&pages=2706-2715&pmid=18490538&doi=10.1161/CIRCULATIONAHA.107.695007&)

4. Lehtonen A, Marniemi J (1984) Effect of atenolol on plasma HDL cholesterol subfractions. Atherosclerosis 51: 335-338. doi: 10.1016/0021-9150(84)90181-3. PubMed: 6743386.  [6743386](http://www.ncbi.nlm.nih.gov/pubmed/6743386) | [DOI](https://doi.org/10.1016/0021-9150(84)90181-3) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/6743386/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Atherosclerosis&title=Effect%20of%20atenolol%20on%20plasma%20HDL%20cholesterol%20subfractions&author=A%20Lehtonen&author=J%20Marniemi&volume=51&publication_year=1984&pages=335-338&pmid=6743386&doi=10.1016/0021-9150(84)90181-3&)

5. Gordon DJ, Rifkind BM (1989) High-density lipoprotein--the clinical implications of recent studies. N Engl J Med 321: 1311-1316. doi: 10.1056/NEJM198911093211907. PubMed: 2677733.  [2677733](http://www.ncbi.nlm.nih.gov/pubmed/2677733) | [DOI](https://doi.org/10.1056/NEJM198911093211907) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/2677733/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=High-density%20lipoprotein--the%20clinical%20implications%20of%20recent%20studies&author=DJ%20Gordon&author=BM%20Rifkind&volume=321&publication_year=1989&pages=1311-1316&pmid=2677733&doi=10.1056/NEJM198911093211907&)

6. Barter P, Gotto AM, LaRosa JC, Maroni J, Szarek M et al. (2007) HDL cholesterol, very low levels of LDL cholesterol, and cardiovascular events. N Engl J Med 357: 1301-1310. doi: 10.1056/NEJMoa064278. PubMed: 17898099.  [17898099](http://www.ncbi.nlm.nih.gov/pubmed/17898099) | [DOI](https://doi.org/10.1056/NEJMoa064278) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17898099/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=HDL%20cholesterol,%20very%20low%20levels%20of%20LDL%20cholesterol,%20and%20cardiovascular%20events&author=P%20Barter&author=AM%20Gotto&author=JC%20LaRosa&author=J%20Maroni&author=M%20Szarek&volume=357&publication_year=2007&pages=1301-1310&pmid=17898099&doi=10.1056/NEJMoa064278&)

7. Jafri H, Alsheikh-Ali AA, Karas RH (2010) Meta-analysis: statin therapy does not alter the association between low levels of high-density lipoprotein cholesterol and increased cardiovascular risk. Ann Intern Med 153: 800-808. doi: 10.7326/0003-4819-153-12-201012210-00006. PubMed: 21173414.  [21173414](http://www.ncbi.nlm.nih.gov/pubmed/21173414) | [DOI](https://doi.org/10.7326/0003-4819-153-12-201012210-00006) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21173414/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Intern%20Med&title=Meta-analysis:%20statin%20therapy%20does%20not%20alter%20the%20association%20between%20low%20levels%20of%20high-density%20lipoprotein%20cholesterol%20and%20increased%20cardiovascular%20risk&author=H%20Jafri&author=AA%20Alsheikh-Ali&author=RH%20Karas&volume=153&publication_year=2010&pages=800-808&pmid=21173414&doi=10.7326/0003-4819-153-12-201012210-00006&)

8. Barter PJ, Caulfield M, Eriksson M, Grundy SM, Kastelein JJ et al. (2007) Effects of torcetrapib in patients at high risk for coronary events. N Engl J Med 357: 2109-2122. doi: 10.1056/NEJMoa0706628. PubMed: 17984165.  [17984165](http://www.ncbi.nlm.nih.gov/pubmed/17984165) | [DOI](https://doi.org/10.1056/NEJMoa0706628) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17984165/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Effects%20of%20torcetrapib%20in%20patients%20at%20high%20risk%20for%20coronary%20events&author=PJ%20Barter&author=M%20Caulfield&author=M%20Eriksson&author=SM%20Grundy&author=JJ%20Kastelein&volume=357&publication_year=2007&pages=2109-2122&pmid=17984165&doi=10.1056/NEJMoa0706628&)

9. Boden WE, Probstfield JL, Anderson T, Chaitman BR, Desvignes-Nickens P et al. (2011) Niacin in patients with low HDL cholesterol levels receiving intensive statin therapy. N Engl J Med 365: 2255-2267. doi: 10.1056/NEJMoa1107579. PubMed: 22085343.  [22085343](http://www.ncbi.nlm.nih.gov/pubmed/22085343) | [DOI](https://doi.org/10.1056/NEJMoa1107579) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22085343/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Niacin%20in%20patients%20with%20low%20HDL%20cholesterol%20levels%20receiving%20intensive%20statin%20therapy&author=WE%20Boden&author=JL%20Probstfield&author=T%20Anderson&author=BR%20Chaitman&author=P%20Desvignes-Nickens&volume=365&publication_year=2011&pages=2255-2267&pmid=22085343&doi=10.1056/NEJMoa1107579&)

10. Weiss LA, Pan L, Abney M, Ober C (2006) The sex-specific genetic architecture of quantitative traits in humans. Nat Genet 38: 218-222. doi: 10.1038/ng1726. PubMed: 16429159.  [16429159](http://www.ncbi.nlm.nih.gov/pubmed/16429159) | [DOI](https://doi.org/10.1038/ng1726) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16429159/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=The%20sex-specific%20genetic%20architecture%20of%20quantitative%20traits%20in%20humans&author=LA%20Weiss&author=L%20Pan&author=M%20Abney&author=C%20Ober&volume=38&publication_year=2006&pages=218-222&pmid=16429159&doi=10.1038/ng1726&)

11. Teslovich TM, Musunuru K, Smith AV, Edmondson AC, Stylianou IM et al. (2010) Biological, clinical and population relevance of 95 loci for blood lipids. Nature 466: 707-713. doi: 10.1038/nature09270. PubMed: 20686565.  [20686565](http://www.ncbi.nlm.nih.gov/pubmed/20686565) | [DOI](https://doi.org/10.1038/nature09270) | [PMC free article](/articles/PMC3039276/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20686565/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Biological,%20clinical%20and%20population%20relevance%20of%2095%20loci%20for%20blood%20lipids&author=TM%20Teslovich&author=K%20Musunuru&author=AV%20Smith&author=AC%20Edmondson&author=IM%20Stylianou&volume=466&publication_year=2010&pages=707-713&pmid=20686565&doi=10.1038/nature09270&)

12. Keating BJ, Tischfield S, Murray SS, Bhangale T, Price TS et al. (2008) Concept, design and implementation of a cardiovascular gene-centric 50 k SNP array for large-scale genomic association studies. PLOS ONE 3: e3583. doi: 10.1371/journal.pone.0003583. PubMed: 18974833.  [18974833](http://www.ncbi.nlm.nih.gov/pubmed/18974833) | [DOI](https://doi.org/10.1371/journal.pone.0003583) | [PMC free article](/articles/PMC2571995/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18974833/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLOS%20ONE&title=Concept,%20design%20and%20implementation%20of%20a%20cardiovascular%20gene-centric%2050%20k%20SNP%20array%20for%20large-scale%20genomic%20association%20studies&author=BJ%20Keating&author=S%20Tischfield&author=SS%20Murray&author=T%20Bhangale&author=TS%20Price&volume=3&publication_year=2008&pages=e3583&pmid=18974833&doi=10.1371/journal.pone.0003583&)

13. Johnson JA, Boerwinkle E, Zineh I, Chapman AB, Bailey K et al. (2009) Pharmacogenomics of antihypertensive drugs: rationale and design of the Pharmacogenomic Evaluation of Antihypertensive Responses (PEAR) study. Am Heart J 157: 442-449. doi: 10.1016/j.ahj.2008.11.018. PubMed: 19249413.  [19249413](http://www.ncbi.nlm.nih.gov/pubmed/19249413) | [DOI](https://doi.org/10.1016/j.ahj.2008.11.018) | [PMC free article](/articles/PMC2671287/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19249413/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20Heart%20J&title=Pharmacogenomics%20of%20antihypertensive%20drugs:%20rationale%20and%20design%20of%20the%20Pharmacogenomic%20Evaluation%20of%20Antihypertensive%20Responses%20(PEAR)%20study&author=JA%20Johnson&author=E%20Boerwinkle&author=I%20Zineh&author=AB%20Chapman&author=K%20Bailey&volume=157&publication_year=2009&pages=442-449&pmid=19249413&doi=10.1016/j.ahj.2008.11.018&)

14. Purcell S, Neale B, Todd-Brown K, Thomas L, Ferreira MA et al. (2007) PLINK: a tool set for whole-genome association and population-based linkage analyses. Am J Hum Genet 81: 559-575. doi: 10.1086/519795. PubMed: 17701901.  [17701901](http://www.ncbi.nlm.nih.gov/pubmed/17701901) | [DOI](https://doi.org/10.1086/519795) | [PMC free article](/articles/PMC1950838/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17701901/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=PLINK:%20a%20tool%20set%20for%20whole-genome%20association%20and%20population-based%20linkage%20analyses&author=S%20Purcell&author=B%20Neale&author=K%20Todd-Brown&author=L%20Thomas&author=MA%20Ferreira&volume=81&publication_year=2007&pages=559-575&pmid=17701901&doi=10.1086/519795&)

15. Price AL, Patterson NJ, Plenge RM, Weinblatt ME, Shadick NA et al. (2006) Principal components analysis corrects for stratification in genome-wide association studies. Nat Genet 38: 904-909. doi: 10.1038/ng1847. PubMed: 16862161.  [16862161](http://www.ncbi.nlm.nih.gov/pubmed/16862161) | [DOI](https://doi.org/10.1038/ng1847) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16862161/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Principal%20components%20analysis%20corrects%20for%20stratification%20in%20genome-wide%20association%20studies&author=AL%20Price&author=NJ%20Patterson&author=RM%20Plenge&author=ME%20Weinblatt&author=NA%20Shadick&volume=38&publication_year=2006&pages=904-909&pmid=16862161&doi=10.1038/ng1847&)

16. Saxena R, Elbers CC, Guo Y, Peter I, Gaunt TR et al. (2012) Large-scale gene-centric meta-analysis across 39 studies identifies type 2 diabetes loci. Am J Hum Genet 90: 410-425. doi: 10.1016/j.ajhg.2011.12.022. PubMed: 22325160.  [22325160](http://www.ncbi.nlm.nih.gov/pubmed/22325160) | [DOI](https://doi.org/10.1016/j.ajhg.2011.12.022) | [PMC free article](/articles/PMC3309185/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22325160/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=Large-scale%20gene-centric%20meta-analysis%20across%2039%20studies%20identifies%20type%202%20diabetes%20loci&author=R%20Saxena&author=CC%20Elbers&author=Y%20Guo&author=I%20Peter&author=TR%20Gaunt&volume=90&publication_year=2012&pages=410-425&pmid=22325160&doi=10.1016/j.ajhg.2011.12.022&)

17. Yuan HY, Chiou JJ, Tseng WH, Liu CH, Liu CK et al. (2006) FASTSNP: an always up-to-date and extendable service for SNP function analysis and prioritization. Nucleic Acids Res 34: W635-W641. doi: 10.1093/nar/gkj469. PubMed: 16845089.  [16845089](http://www.ncbi.nlm.nih.gov/pubmed/16845089) | [DOI](https://doi.org/10.1093/nar/gkj469) | [PMC free article](/articles/PMC1538865/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16845089/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=FASTSNP:%20an%20always%20up-to-date%20and%20extendable%20service%20for%20SNP%20function%20analysis%20and%20prioritization&author=HY%20Yuan&author=JJ%20Chiou&author=WH%20Tseng&author=CH%20Liu&author=CK%20Liu&volume=34&publication_year=2006&pages=W635-W641&pmid=16845089&doi=10.1093/nar/gkj469&)

18. Chelala C, Khan A, Lemoine NR (2009) SNPnexus: a web database for functional annotation of newly discovered and public domain single nucleotide polymorphisms. Bioinformatics 25: 655-661. doi: 10.1093/bioinformatics/btn653. PubMed: 19098027.  [19098027](http://www.ncbi.nlm.nih.gov/pubmed/19098027) | [DOI](https://doi.org/10.1093/bioinformatics/btn653) | [PMC free article](/articles/PMC2647830/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19098027/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=SNPnexus:%20a%20web%20database%20for%20functional%20annotation%20of%20newly%20discovered%20and%20public%20domain%20single%20nucleotide%20polymorphisms&author=C%20Chelala&author=A%20Khan&author=NR%20Lemoine&volume=25&publication_year=2009&pages=655-661&pmid=19098027&doi=10.1093/bioinformatics/btn653&)

19. Ge D, Zhang K, Need AC, Martin O, Fellay J et al. (2008) WGAViewer: software for genomic annotation of whole genome association studies. Genome Res 18: 640-643. doi: 10.1101/gr.071571.107. PubMed: 18256235.  [18256235](http://www.ncbi.nlm.nih.gov/pubmed/18256235) | [DOI](https://doi.org/10.1101/gr.071571.107) | [PMC free article](/articles/PMC2279251/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18256235/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Res&title=WGAViewer:%20software%20for%20genomic%20annotation%20of%20whole%20genome%20association%20studies&author=D%20Ge&author=K%20Zhang&author=AC%20Need&author=O%20Martin&author=J%20Fellay&volume=18&publication_year=2008&pages=640-643&pmid=18256235&doi=10.1101/gr.071571.107&)

20. Barrett JC, Fry B, Maller J, Daly MJ (2005) Haploview: analysis and visualization of LD and haplotype maps. Bioinformatics 21: 263-265. doi: 10.1093/bioinformatics/bth457. PubMed: 15297300.  [15297300](http://www.ncbi.nlm.nih.gov/pubmed/15297300) | [DOI](https://doi.org/10.1093/bioinformatics/bth457) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15297300/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=Haploview:%20analysis%20and%20visualization%20of%20LD%20and%20haplotype%20maps&author=JC%20Barrett&author=B%20Fry&author=J%20Maller&author=MJ%20Daly&volume=21&publication_year=2005&pages=263-265&pmid=15297300&doi=10.1093/bioinformatics/bth457&)

21. Pruim RJ, Welch RP, Sanna S, Teslovich TM, Chines PS et al. (2010) LocusZoom: regional visualization of genome-wide association scan results. Bioinformatics 26: 2336-2337. doi: 10.1093/bioinformatics/btq419. PubMed: 20634204.  [20634204](http://www.ncbi.nlm.nih.gov/pubmed/20634204) | [DOI](https://doi.org/10.1093/bioinformatics/btq419) | [PMC free article](/articles/PMC2935401/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20634204/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=LocusZoom:%20regional%20visualization%20of%20genome-wide%20association%20scan%20results&author=RJ%20Pruim&author=RP%20Welch&author=S%20Sanna&author=TM%20Teslovich&author=PS%20Chines&volume=26&publication_year=2010&pages=2336-2337&pmid=20634204&doi=10.1093/bioinformatics/btq419&)

22. Livak KJ, Schmittgen TD (2001) Analysis of relative gene expression data using real-time quantitative PCR and the 2(-Delta Delta C(T)) Method. Methods 25: 402-408. doi: 10.1006/meth.2001.1262. PubMed: 11846609.  [11846609](http://www.ncbi.nlm.nih.gov/pubmed/11846609) | [DOI](https://doi.org/10.1006/meth.2001.1262) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11846609/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Methods&title=Analysis%20of%20relative%20gene%20expression%20data%20using%20real-time%20quantitative%20PCR%20and%20the%202(-Delta%20Delta%20C(T))%20Method&author=KJ%20Livak&author=TD%20Schmittgen&volume=25&publication_year=2001&pages=402-408&pmid=11846609&doi=10.1006/meth.2001.1262&)

23. Willer CJ, Sanna S, Jackson AU, Scuteri A, Bonnycastle LL et al. (2008) Newly identified loci that influence lipid concentrations and risk of coronary artery disease. Nat Genet 40: 161-169. doi: 10.1038/ng.76. PubMed: 18193043.  [18193043](http://www.ncbi.nlm.nih.gov/pubmed/18193043) | [DOI](https://doi.org/10.1038/ng.76) | [PMC free article](/articles/PMC5206900/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18193043/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Newly%20identified%20loci%20that%20influence%20lipid%20concentrations%20and%20risk%20of%20coronary%20artery%20disease&author=CJ%20Willer&author=S%20Sanna&author=AU%20Jackson&author=A%20Scuteri&author=LL%20Bonnycastle&volume=40&publication_year=2008&pages=161-169&pmid=18193043&doi=10.1038/ng.76&)

24. Frayling TM, Timpson NJ, Weedon MN, Zeggini E, Freathy RM et al. (2007) A common variant in the FTO gene is associated with body mass index and predisposes to childhood and adult obesity. Science 316: 889-894. doi: 10.1126/science.1141634. PubMed: 17434869.  [17434869](http://www.ncbi.nlm.nih.gov/pubmed/17434869) | [DOI](https://doi.org/10.1126/science.1141634) | [PMC free article](/articles/PMC2646098/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17434869/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=A%20common%20variant%20in%20the%20FTO%20gene%20is%20associated%20with%20body%20mass%20index%20and%20predisposes%20to%20childhood%20and%20adult%20obesity&author=TM%20Frayling&author=NJ%20Timpson&author=MN%20Weedon&author=E%20Zeggini&author=RM%20Freathy&volume=316&publication_year=2007&pages=889-894&pmid=17434869&doi=10.1126/science.1141634&)

25. Asselbergs FW, Guo Y, van Iperen EP, Sivapalaratnam S, Tragante V et al. (2012) Large-scale gene-centric meta-analysis across 32 studies identifies multiple lipid loci. Am J Hum Genet 91: 823-838. doi: 10.1016/j.ajhg.2012.08.032. PubMed: 23063622.  [23063622](http://www.ncbi.nlm.nih.gov/pubmed/23063622) | [DOI](https://doi.org/10.1016/j.ajhg.2012.08.032) | [PMC free article](/articles/PMC3487124/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23063622/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=Large-scale%20gene-centric%20meta-analysis%20across%2032%20studies%20identifies%20multiple%20lipid%20loci&author=FW%20Asselbergs&author=Y%20Guo&author=EP%20van%20Iperen&author=S%20Sivapalaratnam&author=V%20Tragante&volume=91&publication_year=2012&pages=823-838&pmid=23063622&doi=10.1016/j.ajhg.2012.08.032&)

26. Fiegenbaum M, da Silveira FR, Van der Sand CR, Van der Sand LC, Ferreira ME et al. (2005) The role of common variants of ABCB1, CYP3A4, and CYP3A5 genes in lipid-lowering efficacy and safety of simvastatin treatment. Clin Pharmacol Ther 78: 551-558. doi: 10.1016/j.clpt.2005.08.003. PubMed: 16321621.  [16321621](http://www.ncbi.nlm.nih.gov/pubmed/16321621) | [DOI](https://doi.org/10.1016/j.clpt.2005.08.003) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16321621/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=The%20role%20of%20common%20variants%20of%20ABCB1,%20CYP3A4,%20and%20CYP3A5%20genes%20in%20lipid-lowering%20efficacy%20and%20safety%20of%20simvastatin%20treatment&author=M%20Fiegenbaum&author=FR%20da%20Silveira&author=CR%20Van%20der%20Sand&author=LC%20Van%20der%20Sand&author=ME%20Ferreira&volume=78&publication_year=2005&pages=551-558&pmid=16321621&doi=10.1016/j.clpt.2005.08.003&)

27. Fujino T, Asaba H, Kang MJ, Ikeda Y, Sone H et al. (2003) Low-density lipoprotein receptor-related protein 5 (LRP5) is essential for normal cholesterol metabolism and glucose-induced insulin secretion. Proc Natl Acad Sci U S A 100: 229-234. doi: 10.1073/pnas.0133792100. PubMed: 12509515.  [12509515](http://www.ncbi.nlm.nih.gov/pubmed/12509515) | [DOI](https://doi.org/10.1073/pnas.0133792100) | [PMC free article](/articles/PMC140935/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12509515/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&title=Low-density%20lipoprotein%20receptor-related%20protein%205%20(LRP5)%20is%20essential%20for%20normal%20cholesterol%20metabolism%20and%20glucose-induced%20insulin%20secretion&author=T%20Fujino&author=H%20Asaba&author=MJ%20Kang&author=Y%20Ikeda&author=H%20Sone&volume=100&publication_year=2003&pages=229-234&pmid=12509515&doi=10.1073/pnas.0133792100&)

28. Alpy F, Tomasetto C (2006) MLN64 and MENTHO, two mediators of endosomal cholesterol transport. Biochem Soc Trans 34: 343-345. doi: 10.1042/BST0340343. PubMed: 16709157.  [16709157](http://www.ncbi.nlm.nih.gov/pubmed/16709157) | [DOI](https://doi.org/10.1042/BST0340343) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16709157/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem%20Soc%20Trans&title=MLN64%20and%20MENTHO,%20two%20mediators%20of%20endosomal%20cholesterol%20transport&author=F%20Alpy&author=C%20Tomasetto&volume=34&publication_year=2006&pages=343-345&pmid=16709157&doi=10.1042/BST0340343&)

29. Kajinami K, Brousseau ME, Lamon-Fava S, Ordovas JM, Schaefer EJ (2005) Gender-specific effects of estrogen receptor alpha gene haplotype on high-density lipoprotein cholesterol response to atorvastatin: interaction with apolipoprotein AI gene polymorphism. Atherosclerosis 178: 331-338. doi: 10.1016/j.atherosclerosis.2004.08.034. PubMed: 15694942.  [15694942](http://www.ncbi.nlm.nih.gov/pubmed/15694942) | [DOI](https://doi.org/10.1016/j.atherosclerosis.2004.08.034) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15694942/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Atherosclerosis&title=Gender-specific%20effects%20of%20estrogen%20receptor%20alpha%20gene%20haplotype%20on%20high-density%20lipoprotein%20cholesterol%20response%20to%20atorvastatin:%20interaction%20with%20apolipoprotein%20AI%20gene%20polymorphism&author=K%20Kajinami&author=ME%20Brousseau&author=S%20Lamon-Fava&author=JM%20Ordovas&author=EJ%20Schaefer&volume=178&publication_year=2005&pages=331-338&pmid=15694942&doi=10.1016/j.atherosclerosis.2004.08.034&)

30. Kingsley DM, Kozarsky KF, Hobbie L, Krieger M (1986) Reversible defects in O-linked glycosylation and LDL receptor expression in a UDP-Gal/UDP-GalNAc 4-epimerase deficient mutant. Cell 44: 749-759. doi: 10.1016/0092-8674(86)90841-X. PubMed: 3948246.  [3948246](http://www.ncbi.nlm.nih.gov/pubmed/3948246) | [DOI](https://doi.org/10.1016/0092-8674(86)90841-X) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/3948246/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell&title=Reversible%20defects%20in%20O-linked%20glycosylation%20and%20LDL%20receptor%20expression%20in%20a%20UDP-Gal/UDP-GalNAc%204-epimerase%20deficient%20mutant&author=DM%20Kingsley&author=KF%20Kozarsky&author=L%20Hobbie&author=M%20Krieger&volume=44&publication_year=1986&pages=749-759&pmid=3948246&doi=10.1016/0092-8674(86)90841-X&)

31. Loos RJ, Bouchard C (2008) FTO: the first gene contributing to common forms of human obesity. Obes Rev 9: 246-250. doi: 10.1111/j.1467-789X.2008.00481.x. PubMed: 18373508.  [18373508](http://www.ncbi.nlm.nih.gov/pubmed/18373508) | [DOI](https://doi.org/10.1111/j.1467-789X.2008.00481.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18373508/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Obes%20Rev&title=FTO:%20the%20first%20gene%20contributing%20to%20common%20forms%20of%20human%20obesity&author=RJ%20Loos&author=C%20Bouchard&volume=9&publication_year=2008&pages=246-250&pmid=18373508&doi=10.1111/j.1467-789X.2008.00481.x&)

32. Mahungu TW, Nair D, Smith CJ, Egan D, Youle M et al. (2009) The relationships of ABCB1 3435C>T and CYP2B6 516GT with high-density lipoprotein cholesterol in HIV-infected patients receiving Efavirenz. Clin Pharmacol Ther 86: 204-211. doi: 10.1038/clpt.2009.78. PubMed: 19474786.  [19474786](http://www.ncbi.nlm.nih.gov/pubmed/19474786) | [DOI](https://doi.org/10.1038/clpt.2009.78) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19474786/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=The%20relationships%20of%20ABCB1%203435C>T%20and%20CYP2B6%20516GT%20with%20high-density%20lipoprotein%20cholesterol%20in%20HIV-infected%20patients%20receiving%20Efavirenz&author=TW%20Mahungu&author=D%20Nair&author=CJ%20Smith&author=D%20Egan&author=M%20Youle&volume=86&publication_year=2009&pages=204-211&pmid=19474786&doi=10.1038/clpt.2009.78&)

33. Schmitz G, Grandl M (2009) The molecular mechanisms of HDL and associated vesicular trafficking mechanisms to mediate cellular lipid homeostasis. Arterioscler Thromb Vasc Biol 29: 1718-1722. doi: 10.1161/ATVBAHA.108.179507. PubMed: 19846837.  [19846837](http://www.ncbi.nlm.nih.gov/pubmed/19846837) | [DOI](https://doi.org/10.1161/ATVBAHA.108.179507) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19846837/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arterioscler%20Thromb%20Vasc%20Biol&title=The%20molecular%20mechanisms%20of%20HDL%20and%20associated%20vesicular%20trafficking%20mechanisms%20to%20mediate%20cellular%20lipid%20homeostasis&author=G%20Schmitz&author=M%20Grandl&volume=29&publication_year=2009&pages=1718-1722&pmid=19846837&doi=10.1161/ATVBAHA.108.179507&)

34. Keene KL, Mychaleckyj JC, Smith SG, Leak TS, Perlegas PS et al. (2008) Comprehensive evaluation of the estrogen receptor alpha gene reveals further evidence for association with type 2 diabetes enriched for nephropathy in an African American population. Hum Genet 123: 333-341. doi: 10.1007/s00439-008-0482-z. PubMed: 18305958.  [18305958](http://www.ncbi.nlm.nih.gov/pubmed/18305958) | [DOI](https://doi.org/10.1007/s00439-008-0482-z) | [PMC free article](/articles/PMC2752813/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18305958/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Genet&title=Comprehensive%20evaluation%20of%20the%20estrogen%20receptor%20alpha%20gene%20reveals%20further%20evidence%20for%20association%20with%20type%202%20diabetes%20enriched%20for%20nephropathy%20in%20an%20African%20American%20population&author=KL%20Keene&author=JC%20Mychaleckyj&author=SG%20Smith&author=TS%20Leak&author=PS%20Perlegas&volume=123&publication_year=2008&pages=333-341&pmid=18305958&doi=10.1007/s00439-008-0482-z&)
