# Catechol O-methyltransferase pharmacogenomics and selective serotonin reuptake inhibitor response

## Metadata
**Authors:** Yuan Ji, Joanna Biernacka, Karen Snyder, Maureen Drews, Linda L Pelleymounter, Colin Colby, Liewei Wang, David A Mrazek, Richard M Weinshilboum
**Journal:** The Pharmacogenomics Journal
**Date:** 2010 Sep 28
**DOI:** [10.1038/tpj.2010.69](https://doi.org/10.1038/tpj.2010.69)
**PMID:** 20877297
**PMCID:** PMC3113454
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3113454/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC3113454/pdf/nihms275936.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC3113454/pdf/nihms275936.pdf)

## Abstract

We applied a systematic pharmacogenetic approach to investigate the role of genetic variation in the gene encoding catechol O-methyltransferase (COMT) in individual variation in selective serotonin reuptake inhibitor (SSRI) response among depressed patients. Twenty-three single nucleotide polymorphisms (SNPs) in COMT were genotyped using DNA from the Sequenced Treatment Alternatives to Relieve Depression (STAR*D) study (N=1914). One SNP, rs13306278, located in the distal promoter region of COMT, showed significant association with remission in White Non-Hispanic (WNH) subjects (P = 0.038). Electromobility shift assay for rs13306278 showed alternation in the ability of the variant sequence to bind nuclear proteins. A replication study was performed using samples from the Mayo Clinic PGRN Citalopram/Escitalopram Pharmacogenomic study (N=422) that demonstrated a similar trend for association. Our findings suggest that novel genetic markers in the COMT distal promoter may influence SSRI response phenotypes.

Keywords: Pharmacogenetics, catechol O-methyltransferase, COMT, selective serotonin reuptake inhibitor, major depressive disorder, STAR*D

## Introduction

Major depressive disorder (MDD) is a common and serious illness with an estimated annual incidence in the United States of 16%.[1](#R1) Selective serotonin reuptake inhibitors (SSRIs) are the most widely prescribed class of antidepressants.[2](#R2) There are large individual variations in response to SSRIs, with reported remission rates of less than 50%.[2](#R2) We set out to test the hypothesis that common variation in the gene encoding catechol O-methyltransferase (COMT, EC 2.1.1.6), might influence response to SSRI therapy of MDD patients.

Biologically active monoamines such as norepinephrine and serotonin contribute to the regulation of mood and emotions.[3](#R3) Decades ago, Schildkraut proposed a “chemical imbalance” theory which suggested that diminished levels of monoamine neurotransmitters might increase risk for depression.[4](#R4) COMT catalyzes the O-methylation of catecholamine neurotransmitters.[5](#R5) Therefore, variation in the activity of COMT could, in theory, modulate the efficacy of antidepressants that are designed to increase neurotransmitters at post-synaptic receptors. The *COMT* gene was cloned in 1989[6](#R6) and maps to chromosome 22q11,[7](#R7) a region of interest for several psychiatric disorders.[8](#R8) Two isoforms, a soluble cytoplasmic (S-COMT) and a membrane-bound (MB-COMT) isoform are encoded by one gene with two promoters. A “proximal promoter” for S-COMT is located in intron 2, while a distal promoter for MB-COMT is found at the 5′-end of the gene.[9](#R9)^, ^[10](#R10) MB-COMT has an additional 50 hydrophobic amino acids at its N-terminus and is thought to be the predominant isoform expressed in the brain.[10](#R10)

Genetic studies of *COMT* and its possible role in risk for diseases that range from depression to breast cancer date back over three decades.[11](#R11) Our laboratory first described common genetic polymorphisms that influence COMT activity over 30 years ago[12](#R12)^–^[14](#R14) and – more recently – a common COMT Val108/158Met (S/MB) polymorphism that is associated with the autosomal codominant regulation of level of COMT activity.[15](#R15) The Val108/158Met polymorphism has been studied extensively in psychiatric disease[16](#R16) as well as diseases such as breast cancer.[17](#R17)^–^[19](#R19) However, those results have often been inconsistent results. We recently conducted a study which suggested that functional polymorphisms in the distal promoter for MB-COMT might be of greater importance for breast cancer risk than the Val158Met polymorphism.[20](#R20)

Therefore, we set out to systematically investigate the possible role of *COMT* genetic variation in response to the SSRI therapy of MDD. Twenty-three SNPs across the *COMT* gene (~28 kb) were selected based on gene resequencing data[21](#R21)^, ^[22](#R22) and public database information for use in genotyping 1914 DNA samples from the Sequenced Treatment Alternatives to Relieve Depression (STAR*D) study.[23](#R23)^–^[25](#R25) Only 33% of patients in the STAR*D study achieved complete “remission” after citalopram therapy, i.e., QIDS-C_16_ ≤ 5, and only 47% “responded”, defined as a 50% reduction in QIDS-C_16_ at the last visit.[25](#R25) We then preformed functional genomic studies and a replication study using 422 samples from the ongoing Mayo Clinic PGRN Citalopram/Escitalopram Pharmacogenomic Study (Mayo PGRN SSRI Study). This systematic approach enabled us to identify a novel SNP in the *COMT* distal promoter that appears to be associated with remission during SSRI therapy.

## Materials and Methods

### Study design and samples

An initial association analysis of 23 *COMT* SNPs was performed using samples from the STAR*D study. STAR*D patients were all treated initially with citalopram. 1914 DNA samples from 4,041 eligible STAR*D subjects were available genotyping. The primary STAR*D outcome was “remission”, a QIDS-C_16_ score ≤ 5 at the last clinic visit.[26](#R26) The Mayo PGRN SSRI Study is an ongoing clinical trial designed to parallel the STAR*D design. Specifically, MDD patients from the Mayo Clinic practice are treated with either citalopram or escitalopram and must have a 17-item Hamilton Depression Rating Scale (HRD-D17) score of >14 to enter the study. Exclusion criteria are also similar to those used in STAR*D. Patients are seen at entry, week 4 and week 8 for clinical evaluation and to obtain blood samples. Over 600 patients have been enrolled to date and 422 DNA samples were available for genotyping when current study was performed. The Mayo PGRN SSRI Study was reviewed and approved by the Mayo Clinic Institutional Review Board, and all participants provided written informed consent.

### SNP selection and genotyping

Twenty-three SNPs were selected from previous resequencing data[21](#R21)^, ^[22](#R22) using haplotype tagging,[27](#R27) or linkage disequilibrium (LD) tagging approaches[28](#R28) as well as on the basis of previous reports from clinical association studies.[29](#R29)^, ^[30](#R30) Genotyping for the STAR*D discovery study was performed using the Illumina Veracode (Bead Xpress) platform (Illumina, San Diego, CA) and Taqman (Applied Biosystems, Foster City, CA) genotyping was used for the Mayo validation study.

### Single SNP associations with remission

Samples with < 90% call rates, ambiguous calls or data for patients with insufficient baseline depressive symptoms (QIDS-C_16_ score <10) were removed from the analysis. Tests for Hardy-Weinberg Equilibrium (HWE) were performed for the entire sample, followed by HWE tests for all samples and within self-reported ethnic groups. For binary phenotypes, i.e. remitter vs. nonremitter, tests of association between remission and each SNP were performed in each ethnic subgroup after excluding 33 subjects who failed to have sufficient depressive symptoms at baseline (QIDS-C_16_ score of < 10). A *P*-value was calculated for each SNP based on a logistic regression model, assuming a log-additive allele effect on the odds of remission. For consistency with earlier STAR*D publications, the analysis was repeated for a subset of patients that included patients who remained in the study for at least 6 weeks and were not “non-compliant”.[26](#R26)

Remission is strongly associated with medication “tolerance”.[31](#R31) Therefore, we also performed univariate tests of association between remission and each SNP in ethnic subgroups, adjusted for medication tolerance. All patients who continued citalopram therapy at the end of STAR*D phase 1 treatment were considered tolerant, while patients who refused to continue citalopram or left because of side effects were considered intolerant. Genetic effects were adjusted for tolerance by including tolerance as a covariate in the model. We also performed an analysis with an adjustment for tolerance, days in study, and medication dose.

Since remission is the goal of treatment, most STAR*D analyses have focused on this outcome.[23](#R23)^, ^[24](#R24)^, ^[26](#R26) However, quantitative change in QIDS-C_16_ scores might provide additional information with regard to drug response, and analyses of QIDS-C_16_ values might provide enhanced power. Therefore, we also determined associations between *COMT* SNPs and QIDS-C_16_ scores at the end of the treatment using linear regression. The final QIDS-C_16_ score or the % change in QIDS-C_16_ were also considered as outcomes, with baseline QIDS-C_16_ as a covariate.

### Electromobility shift (EMS) assays

Since the SNP that was associated with SSRI response in our discovery study is located in the distal promoter region of *COMT*, rather than in the open reading frame, we set out to perform EMS assays to determine the possible effect of this SNP on transcription regulation by determing whether nuclear protein(s) might bind to the SNP locus and also whether alternation in the SNP nucleotide at this locus could alter its ability to bind nuclear protein(s). Previous reports have demonstrated that, in the brain, COMT is expressed mainly in glial cells.[32](#R32) Therefore, the EMS assays were performed using human U-87MG glioma cell nuclear extract (Active Motif, Carlsbad, CA). Specifically, biotin-labeled and unlabeled oligonucleotides were synthesized by IDT Integrated DNA technologies (Coralville, IW). Oligonucleotide sequences for the *COMT* 5′-FR(−485) SNP were WT-5′gccccagtttcCccacctgggaa3′ and variant-5′gccccagtttcTccacctgggaa3′. Details of the EMS assays have been described elsewhere.[20](#R20)

### Bioinformatic analyses

rVISTA, a transcription factor website database that includes comparative sequence analysis web-based “tools”, indicated that the 5′-FR(−485) SNP was located within a 2711 bp region between chromosome 22 positions 18308090 to 18310800 that shared 92.6 % identity with the sequence of the rhesus macaque and contained a cluster of predicted transcription factor binding sites.[33](#R33)^, ^[34](#R34) In silico analysis using AliBaba 2.1[35](#R35) predicted that the region surrounding *COMT* 5′-FR(−485) might contain an Sp1 binding motif and that a C to T alteration at this locus, although remaining a partial Sp1 site, would result in a new transcription factor binding motif, C/EBPα.

## Results

### STAR*D genotyping

Twenty-three SNPs located across the entire *COMT* gene (~28 kb) were genotyped in DNA samples from the STAR*D study. The gene structure of *COMT* is depicted graphically in [Figure 1A](#F1), with locations and allele frequencies for the SNPs genotyped in STAR*D samples as well as those identified during our resequencing of European American (EA) DNA samples. Of the 1903 samples from STAR*D subjects that were genotyped successfully, 33 had inadequate phenotypic data, so 1870 samples were analyzed, including 1232 WNH, 287 Black, 238 White Hispanic (WH) and 113 of “other” self-reported ethnicity. There was some evidence of departure from HWE for rs165774 in WNH subjects (*P-*value = 0.0085), and, rs165722 and rs3810595 in Black subjects (*P* -values = 0.0057 and 0.0029, respectively).

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8b9d/3113454/17fb979fde06/nihms275936f1.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3113454_nihms275936f1.jpg)

Human COMT genetic polymorphisms and linkage disequilibrium in European-American (EA) subjects. (A) Human COMT genetic polymorphisms. Arrows, locations and frequencies of polymorphisms. Black and gray rectangles, coding exons, with the gray area specific for the MB-COMT open reading frame (ORF). Open rectangles, noncoding exons. I/D, insertion/deletion. The well characterized Val108/158Met polymorphism and the 5′-FR(−485) polymorphism are boxed. * = polymorphisms that were genotyped in the STAR*D samples. (B) COMT linkage disequilibrium (LD) displayed by the use of Haploview 3.3. On the left is the COMT LD structure in the STAR*D White Non-Hispanic population (WNH), and on the right is COMT LD structure derived from polymorphisms identified during our COMT resequencing studies performed with 60 EA DNA samples.21, 22 Polymorphisms with minor allele frequencies of greater than 5% were included in the LD analyses.

### Association of remission with COMT SNPs

Remission status was determined on the basis of QIDS-C_16_ at the last clinic visit.[25](#R25)^, ^[26](#R26) Of the STAR*D samples genotyped, there were 541 WNH remitters (44% of the WNH subjects), 93 Black remitters (32% of Black subjects), and 78 WH remitters (33% of the WH subjects). Associations of *COMT* SNPs with remission status for the STAR*D patients are summarized in [Table 1](#T1). One SNP, 5′-FR(−485)C/T (rs13306278) located in the *COMT* distal promoter, appeared to be associated with lack of remission (unadjusted *P-*value = 0.038) in WNH subjects, with an allele-specific odds ratio (OR) of 0.78 (95% CI, 0.62–0.99). A similar trend of association was also present for this same SNP in the WH subjects, with an OR of 0.56 (95% CI, 0.26–1.18). However, this SNP had a very low minor allele frequency (2–3%) in the Black subjects and was not associated with remission in that ethnic group. In addition, a SNP in the 3′-FR of the gene, 3′-FR(23) (rs9332381), displayed a significant association (*P-*value = 0.006) with remission in WNH subjects, with an OR of 1.71 (95% CI, 1.16–2.51), but was not significant in the other two ethnic groups, especially Black patients in whom the minor allele frequency (MAF) was an order of magnitude higher than in WNH subjects. These two SNPs displayed similar association trends when the analysis was performed within a subset that included only patients who were “compliant” and remained in the study for at least 6 weeks (*P-*values for rs13306278 and rs9332381 of 0.015 and 0.028, respectively) ([Supplementary Table 1](#SD1)). When the effects of “tolerance”, days in study, and final drug dose were taken into account, the association between rs13306278 and remission was even more significant (*P*-value = 0.003) ([Supplementary Table 2](#SD1)). However, rs9332381 failed to show an association for this analysis.

### Table 1.

| SNP | rs# | WNH (N = 1232) |  |  |  |  | Black (N = 287) |  |  |  |  | WH (N = 238) |  |  |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| Association |  |  | MAF |  | Association |  |  | MAF |  | Association |  |  | MAF |  |  |  |
| P -value* | OR (allelic) | 95% CI | Nonremitters N = 691 | Remitters N = 541 | P -value* | OR (allelic) | 95% CI | Nonremitters N = 194 | Remitters N = 93 | P -value* | OR (allelic) | 95% CI | Nonremitters N = 160 | Remitters N = 78 |  |  |
| 5′-FR(706) | rs4646310 | 0.236 | 1.13 | 0.92, 1.38 | 0.18 | 0.20 | 0.801 | 0.92 | 0.50, 1.70 | 0.10 | 0.09 | 0.525 | 0.81 | 0.43, 1.53 | 0.116 | 0.096 |
| 5′-FR(−628)** | rs2020917 | 0.060 | 0.84 | 0.70, 1.01 | 0.29 | 0.26 | 0.833 | 0.94 | 0.55, 1.63 | 0.11 | 0.10 | 0.147 | 0.69 | 0.42, 1.14 | 0.225 | 0.167 |
|   |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |
| 5′-FR(−485) | rs13306278 | 0.038 | 0.78 | 0.62, 0.99 | 0.15 | 0.12 | 0.360 | 1.58 | 0.59, 4.19 | 0.02 | 0.03 | 0.125 | 0.56 | 0.26, 1.18 | 0.107 | 0.064 |
|   |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |
| Intron1(99) | rs45454096 | 0.566 | 1.12 | 0.76, 1.66 | 0.04 | 0.05 | 0.560 | 0.78 | 0.33, 1.82 | 0.05 | 0.04 | 0.930 | 0.96 | 0.43, 2.17 | 0.060 | 0.058 |
| Intron1(689)** | rs737866 | 0.058 | 0.84 | 0.70, 1.01 | 0.29 | 0.26 | 0.862 | 0.96 | 0.58, 1.57 | 0.15 | 0.14 | 0.127 | 0.68 | 0.42, 1.12 | 0.228 | 0.167 |
| Intron1(1987) | rs933271 | 0.091 | 1.16 | 0.98, 1.39 | 0.27 | 0.30 | 0.265 | 0.81 | 0.57, 1.17 | 0.42 | 0.37 | 0.405 | 0.85 | 0.57, 1.25 | 0.444 | 0.404 |
| Intron1(8218) | rs5993883 | 0.892 | 0.99 | 0.85, 1.16 | 0.49 | 0.49 | 0.131 | 0.76 | 0.53, 1.09 | 0.49 | 0.44 | 0.055 | 0.69 | 0.47, 1.01 | 0.469 | 0.372 |
| Intron1(15757) | rs740603 | 0.098 | 0.87 | 0.75, 1.03 | 0.47 | 0.49 | 0.145 | 1.32 | 0.91, 1.91 | 0.43 | 0.49 | 0.064 | 0.71 | 0.49, 1.02 | 0.481 | 0.385 |
| Intron2(201) | rs165722 | 0.374 | 0.93 | 0.80, 1.09 | 0.48 | 0.46 | 0.441 | 1.14 | 0.82, 1.58 | 0.47 | 0.49 | 0.090 | 0.71 | 0.47, 1.06 | 0.363 | 0.442 |
| Intron2(832) | rs3810595 | 0.119 | 0.88 | 0.75, 1.03 | 0.40 | 0.37 | 0.491 | 0.89 | 0.63, 1.25 | 0.37 | 0.34 | 0.085 | 0.7 | 0.46, 1.05 | 0.364 | 0.282 |
| Intron2(935) | rs11569716 | 0.525 | 0.64 | 0.16, 2.56 | 0.00 | 0.00 | 0.717 | 1.11 | 0.62, 2.00 | 0.10 | 0.11 | 0.976 | 1.03 | 0.18, 5.73 | 0.013 | 0.013 |
| Exon3(186) | rs4633 | 0.469 | 1.06 | 0.91, 1.24 | 0.48 | 0.46 | 0.083 | 0.72 | 0.50, 1.04 | 0.37 | 0.29 | 0.092 | 1.42 | 0.94, 2.13 | 0.350 | 0.429 |
| Exon4(304) | rs5031015 | N/A | N/A | N/A | 0.00 | 0.00 | 0.517 | 0.59 | 0.12, 2.90 | 0.02 | 0.01 | N/A | N/A | N/A | 0.000 | 0.000 |
| Exon4(408) | rs4818 | 0.119 | 0.88 | 0.75, 1.03 | 0.39 | 0.36 | 0.850 | 0.96 | 0.64, 1.45 | 0.21 | 0.20 | 0.069 | 0.67 | 0.43, 1.03 | 0.331 | 0.250 |
|   |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |
| Exon4(472) | rs4680 | 0.449 | 1.06 | 0.91, 1.24 | 0.48 | 0.47 | 0.354 | 0.84 | 0.58, 1.21 | 0.32 | 0.28 | 0.061 | 1.48 | 0.98, 2.24 | 0.344 | 0.429 |
|   |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |
| Exon5(597) | rs769224 | 0.493 | 1.2 | 0.71, 2.04 | 0.02 | 0.03 | 0.395 | 1.28 | 0.72, 2.26 | 0.09 | 0.11 | 0.145 | 0.32 | 0.07, 1.48 | 0.038 | 0.013 |
| Intron5(75) | rs4646315 | 0.680 | 1.05 | 0.85, 1.29 | 0.17 | 0.18 | 0.084 | 0.62 | 0.36, 1.07 | 0.15 | 0.09 | 0.125 | 1.42 | 0.91, 2.24 | 0.188 | 0.250 |
| Intron5(310) | rs4646316 | 0.466 | 0.93 | 0.77, 1.12 | 0.24 | 0.23 | 0.142 | 1.38 | 0.90, 2.12 | 0.15 | 0.20 | 0.116 | 0.66 | 0.39, 1.11 | 0.220 | 0.160 |
| Intron5(739) | rs165774 | 0.643 | 1.04 | 0.88, 1.23 | 0.32 | 0.33 | 0.360 | 0.82 | 0.55, 1.25 | 0.23 | 0.20 | 0.322 | 1.27 | 0.79, 2.04 | 0.167 | 0.205 |
| Intron5(1354) | rs174696 | 0.152 | 1.15 | 0.95, 1.39 | 0.21 | 0.24 | 0.825 | 1.04 | 0.74, 1.47 | 0.42 | 0.41 | 0.901 | 1.02 | 0.71, 1.49 | 0.494 | 0.500 |
| Intron5(3870) | rs9332377 | 0.693 | 0.96 | 0.77, 1.19 | 0.16 | 0.15 | 0.761 | 1.06 | 0.72, 1.58 | 0.32 | 0.33 | 0.089 | 0.58 | 0.31, 1.09 | 0.153 | 0.096 |
| 3′-FR(23) | rs9332381 | 0.006 | 1.71 | 1.16, 2.51 | 0.03 | 0.06 | 0.584 | 0.89 | 0.59, 1.35 | 0.26 | 0.24 | 0.156 | 0.5 | 0.19, 1.31 | 0.066 | 0.032 |
| 3′-FR(1338) | rs165599 | 0.352 | 1.08 | 0.92, 1.28 | 0.31 | 0.33 | 0.970 | 1.01 | 0.70, 1.45 | 0.31 | 0.31 | 0.061 | 0.68 | 0.46, 1.02 | 0.494 | 0.404 |

Table 1 Caption: Association of COMT SNPs with remission in STAR* D samples. The numbering scheme for the SNPs was adapted from our previous study. Specifically, numbering of nucleotides in the coding region began at the “A” in the “ATG” for MB-COMT, with nucleotides 3′ to that position assigned positive numbers. Nucleotides located within the two upstream noncoding exons, and the 5′-FR were assigned negative numbers, with the final nucleotide of exon 2 designed as “−1”. Nucleotides located within introns were numbered on the basis of distance form the nearest splice site, using positive and negative numbers for distance to 5′- and 3′-splice sites, respectively. The numbering scheme for amino acids is based on their distance from the MB-COMT translation start codon

Although the associations that we observed were not significant after correcting for multiple comparisons, this initial discovery study had identified “SNP signals” that could be pursued. Therefore, we conducted functional genomic experiments for rs13306278 and, subsequently, a validation study using additional clinical samples. We also performed haplotype analyses, but those results showed that rs13306278 was responsible for most of the haplotype effects. Those analyses are described in the [Supplementary Materials](#SD1). Finally, it should be noted that these analyses did not indicate that the functionally significant *COMT* Val108/158Met polymorphism was a factor in SSRI response.

### Association of a quantitative trait with COMT SNPs

Quantitative trait analysis performed using final QIDS-C_16_ score as an “outcome” indicated that rs9332381 was significantly associated with this phenotype (*P*-value ≤ 0.001) in WNH subjects, but not in the Black or WH subjects. In addition, rs4633 in *COMT* Exon 3 showed some evidence of association, but only in Black subjects (*P*-value = 0.013). The rs13306278 SNP was not significantly associated with this measure of response to SSRI therapy ([Supplementary Table 3](#SD1)). However, it was associated with baseline quantitative QIDS score in the WNH subjects, with the minor allele being associated with higher baseline QIDS score, representing more severe depression at study entry (p=0.013).

### Functional genomic studies

EMS assay was performed for the rs13306278 SNP using human glioma U87G cell nuclear extract. [Figure 2A](#F2) shows that glioma cell nuclear protein(s) bound to oligonucleotides containing the wild type (WT) nucleotide at the 5′-FR(−485) locus, and that a C to T change at this nucleotide resulted in a striking reduction in binding. rVISTA predicted that this region of the gene included many possible transcription factor binding motifs. Alibaba 2.1 suggested that this area might be an Sp1 binding site in the WT sequence that would be partially replaced by a C/EBPα (CCAAT/enhancer-binding protein) binding motif in the variant sequence ([Figure 2B](#F2)). C/EBPα regulates gene expression in a variety of tissues[36](#R36) and its expression is detectable in brain.[37](#R37) This evidence of biological plausibility stimulated us to attempt to validate our STAR*D observations using samples from an independent SSRI study.

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8b9d/3113454/6248d51324ef/nihms275936f2.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3113454_nihms275936f2.jpg)

COMT functional genomic studies. (A) EMS assay for the 5′-FR(−485) COMT SNP. U87G glioma cell nuclear extract was incubated with biotin-labeled oligonucleotides. (B) In silico analysis by AliBaba 2.1 for possible transcription binding motif(s) within the 5′-FR(−485) locus.

### Replication study

The Mayo PGRN SSRI Study is an ongoing SSRI pharmacogenomic clinical trial that was designed to parallel STAR*D. A total of 605 patients have been enrolled thus far. The overall remission rate determined by the 8-week QIDS-C_16_ score was 48.4%, similar to the 44% remission rate observed among WNH subjects in the STAR*D study.[26](#R26) Based on our observations with DNA from the STAR*D study and our functional genomic experiments, we genotyped only a single SNP, rs1330628, for validation. The call rate for this TaqMan assay was over 99%. Of the 422 DNA samples genotyped for the Mayo study, 391 were from WNH subjects. 356 subjects had remission status determined on the basis of 8 or 4-week QIDS-C_16_ scores, including 160 remitters and 196 nonremitters. Statistical analysis was performed using remission status at the last visit (either the 8-week QIDS-C_16_ score or the 4-week QIDS-C_16_ score) as the outcome. Assuming a log-additive allele effect, the odds ratio for association between the 5′-FR(−485) locus and remission status was 0.68 (95% CI, 0.42–1.09) ([Table 2](#T2)). Although these results were not statistically significant (*P*-value = 0.11), which is not surprising given the smaller sample size as compared with STAR*D, the trend of association was similar to what we observed for the STAR*D WNH subjects. In addition, MAFs for rs1330628 in the Mayo PGRN SSRI WNH subjects were consistent with those that we had observed in the STAR*D study (15% in nonremitters and 12% in remitters, [Table 2](#T2)). In addition, as in the STAR*D WNH sample, the minor allele of this SNP in the replication samples was once again associated with higher baseline QIDS scores (p=0.004).

### Table 2.

|   | N | Association |  |  | MAF |  |
| --- | --- | --- | --- | --- | --- | --- |
|   | P -value* | OR (allelic) | 95% CI | Nonremitters | Remitters |  |
| STAR*D, WNH group |  |  |  |  |  |  |
| Subjects with remission status at their last visit | 1232 | 0.04 | 0.78 | 0.62, 0.99 | 0.15 | 0.12 |
|   |  |  |  |  |  |  |
| Mayo PGRN SSRI Study, WNH group |  |  |  |  |  |  |
| Subjects with 8-week or 4-week remission status | 356 | 0.11 | 0.68 | 0.42, 1.09 | 0.15 | 0.11 |
| Subjects with 8-week remission status | 313 | 0.27 | 0.75 | 0.46, 1.24 | 0.14 | 0.11 |

Table 2 Caption: Association of COMT 5′-FR(−485) SNP with remission in WNH patients for both STAR* D and Mayo PGRN SSRI studies

## Discussion

SSRIs are the most widely prescribed class of antidepressants.[2](#R2) However, only two-thirds of MDD patients respond to SSRI therapy and even fewer are able to achieve remission.[38](#R38) Therefore, many candidate pharmacogenetic studies have been performed with SSRIs with the goal of identifying genetic markers that might help to predict variation in response to prior to treatment.[31](#R31)^, ^[39](#R39)^–^[42](#R42)

COMT catalyzes the metabolism of catecholamine neurotransmitters that are thought to play a role in mood regulation, based on the “chemical imbalance” theory for depression.[4](#R4) The fact that *COMT* maps to the long arm of chromosome 22, a region of interest for several psychiatric phenotypes, has also served to stimulate studies that focus on the possible role of *COMT* genetic polymorphisms in neuropsychiatric diseases and response to the treatment of those disorders.[16](#R16) The common functional *COMT* Val108/158Met polymorphism that is associated with altered enzyme activity and thermal stability during *in vitro* assays is responsible, in large part, for a tri-modal distribution of COMT enzyme activity in red blood cells, lymphocytes and liver.[11](#R11)^, ^[15](#R15)^, ^[21](#R21)^, ^[22](#R22) Therefore, most studies designed to examine associations between *COMT* polymorphisms and clinical phenotypes have focused on this SNP. However, the results of those studies have been inconsistent. Three of those studies reported a similar trend of association with antidepressant treatment outcome, i.e., that carriers of the Val allele might have better antidepressant treatment outcomes than patients homozygous for Met108/158.[43](#R43)^–^[45](#R45) Since the Val allele is associated with elevated COMT activity, this result seemed to indicate that decreased catecholamine neurotransmitters might enhance drug response – in contrast to our understanding of the role of monoamines in the pathophysiology of depression. Because most of those studies were performed with relatively small numbers of patients and several reported associations that were difficult to explain mechanistically, it has been difficult to draw conclusions about a possible role for *COMT* in SSRI response. Therefore, we set out to systematically examine the possible role of *COMT* sequence variation in individual variation in response to SSRI therapy.

As an initial “discovery” effort, a total of 23 SNPs across the entire length of *COMT* were genotyped in STAR*D subjects. The STAR*D study was not designed to evaluate the efficacy of SSRIs, but – like a large number of observational pharmacogenomic studies – factors that might influence SSRI outcome. The same is true of the Mayo-PGRN SSRI study which was specifically designed to address pharmacogenomic influences from both pharmacokinetic and pharmacodynamic factors on SSRI outcome. SNP rs13306278 in the distal promoter of *COMT* was associated with a decreased rate of remission, was validated functionally and a similar trend was observed in an independent SSRI clinical trial. We also observed an association with a SNP at 3′-FR(23). However, that association was only observed in WNHs, an ethnic group in which this SNP is present with a low MAF (3 to 5%). As a result, we were unable to pursue that SNP further. The heavily studied Val108/158Met polymorphism did not appear to be associated with remission in our study. Haplotype analysis (see [Supplementary Material](#SD1)) provided further evidence that the *COMT* 5′-FR and intron 1, the region that contains the distal promoter, was of importance for SSRI response. However, that association also appeared to be driven, largely, by the rs13306278 SNP. Furthermore, DNA sequence conservation and transcription factor binding site analyses suggested that the area from 5′-FR(−1422) to Intron 1(1380) of the *COMT* gene contained a cluster of transcription factor binding motifs and was highly conserved between rhesus macaque and humans, suggesting a possible functional role for this region. In a recently published genome-wide association study (GWAS) performed with STAR*D samples,[46](#R46) no association was observed between rs13306278 and remission. That is not surprising since the STAR*D GWAS was performed with Affymetrix platforms that do not contain rs1330278. Therefore, the approach used here of applying in-depth gene resequencing data to perform an association study has demonstrated its utility in terms of the discovery of novel variants in candidate genes.

MAFs for rs13306278 in WNH subjects in the STAR*D population were 0.12 for remitters and 0.15 for nonremitters, respectively, resulting in an odds ratio of 0.78 and *P* = 0.038 for a log-additive effect. Assuming that this is a “true” effect size, and with similar MAFs in remitters and nonremitters among the patients in the Mayo PGRN samples, we were not surprised that we were unable to observe a significant association in the replication population of 422 subjects. However, the observed association of SNP rs13306278 with remission in the Mayo study was consistent with our observations for STAR*D, both with regard to direction and effect size.

Both the STAR*D and the PGRN SSRI studies utilized a single-arm design, which is commonly and effectively used in many pharmacogenetic studies. However, with these single arm studies, it is not possible to determine whether a particular SNP influences SSRI response specifically, or is related to depression treatment outcomes or disease prognosis more generally. Nevertheless, from our results we can conclude that, in two large studies, the minor allele at SNP rs13306278 was associated with lower rates of remission of depression after SSRI therapy. Our results also indicated that this allele was associated with higher baseline QIDS scores, indicating greater severity of depression at study entry. These results suggest that the lower rates of remission associated with this SNP may be, at least partially, a reflection of more severe depression. Further studies evaluating the association of this SNP with depression severity and prognosis following other forms of treatment would clearly be of interest.

In summary, our studies have identified a distal promoter SNP in *COMT* that may play a role in individual variation in response to SSRI treatment. These findings indicate that polymorphisms in the distal promoter region of *COMT* might influence response to SSRI therapy, and as a result, they have provided novel candidate polymorphisms for studies of both antidepressant pharmacogenetics and, possibly, depression risk.

## Supplementary Material

## Acknowledgments

This research was supported, in part, by NIH grants RO1 GM28157, U01 GM61388 (The Pharmacogenomics Research Network) and P20 AA17830 as well as a PhRMA Foundation Center of Excellence in Clinical Pharmacology Award. (ClinicalTrials.gov number [NCT00613470](https://clinicaltrials.gov/ct2/show/NCT00613470)) We thank Luanne Wussow for her assistance with the preparation of the manuscript.

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Kessler RC, Berglund P, Demler O, Jin R, Koretz D, Merikangas KR, et al. The epidemiology of major depressive disorder: results from the National Comorbidity Survey Replication (NCS-R) JAMA. 2003;289:3095–3105. doi: 10.1001/jama.289.23.3095.  [DOI](https://doi.org/10.1001/jama.289.23.3095) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12813115/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA&title=The%20epidemiology%20of%20major%20depressive%20disorder:%20results%20from%20the%20National%20Comorbidity%20Survey%20Replication%20(NCS-R)&author=RC%20Kessler&author=P%20Berglund&author=O%20Demler&author=R%20Jin&author=D%20Koretz&volume=289&publication_year=2003&pages=3095-3105&pmid=12813115&doi=10.1001/jama.289.23.3095&)

2. Preskorn S, Staga C, Feighner J, Ross R, editors. Antidepressants: Past, Present and Future. Springer; 2004. p. 242.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Antidepressants:%20Past,%20Present%20and%20Future&publication_year=2004&)

3. Rang HD, MM, Ritter JM. Pharmacology. 4. Churchill Livingstone; 1999. p. 830.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Pharmacology&author=HD%20Rang&author=%20MM&author=JM%20Ritter&publication_year=1999&)

4. Schildkraut JJ. The catecholamine hypothesis of affective disorders: a review of supporting evidence. Am J Psychiatry. 1965;122:509–522. doi: 10.1176/ajp.122.5.509.  [DOI](https://doi.org/10.1176/ajp.122.5.509) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/5319766/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Psychiatry&title=The%20catecholamine%20hypothesis%20of%20affective%20disorders:%20a%20review%20of%20supporting%20evidence&author=JJ%20Schildkraut&volume=122&publication_year=1965&pages=509-522&pmid=5319766&doi=10.1176/ajp.122.5.509&)

5. Axelrod J, Tomchick R. Enzymatic O-methylation of epinephrine and other catechols. J Biol Chem. 1958;233:702–705.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/13575440/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=Enzymatic%20O-methylation%20of%20epinephrine%20and%20other%20catechols&author=J%20Axelrod&author=R%20Tomchick&volume=233&publication_year=1958&pages=702-705&pmid=13575440&)

6. Grossman MH, Creveling CR, Breakefield XO. Isolation of the mRNA encoding rat liver catechol-O-methyltransferase. Biochem Biophys Res Commun. 1989;158:776–782. doi: 10.1016/0006-291x(89)92789-7.  [DOI](https://doi.org/10.1016/0006-291x(89)92789-7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/2645868/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem%20Biophys%20Res%20Commun&title=Isolation%20of%20the%20mRNA%20encoding%20rat%20liver%20catechol-O-methyltransferase&author=MH%20Grossman&author=CR%20Creveling&author=XO%20Breakefield&volume=158&publication_year=1989&pages=776-782&pmid=2645868&doi=10.1016/0006-291x(89)92789-7&)

7. Grossman MH, Emanuel BS, Budarf ML. Chromosomal mapping of the human catechol-O-methyltransferase gene to 22q11.1----q11.2. Genomics. 1992;12:822–825. doi: 10.1016/0888-7543(92)90316-k.  [DOI](https://doi.org/10.1016/0888-7543(92)90316-k) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/1572656/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genomics&title=Chromosomal%20mapping%20of%20the%20human%20catechol-O-methyltransferase%20gene%20to%2022q11.1----q11.2&author=MH%20Grossman&author=BS%20Emanuel&author=ML%20Budarf&volume=12&publication_year=1992&pages=822-825&pmid=1572656&doi=10.1016/0888-7543(92)90316-k&)

8. Badner JA, Gershon ES. Meta-analysis of whole-genome linkage scans of bipolar disorder and schizophrenia. Mol Psychiatry. 2002;7:405–411. doi: 10.1038/sj.mp.4001012.  [DOI](https://doi.org/10.1038/sj.mp.4001012) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11986984/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Psychiatry&title=Meta-analysis%20of%20whole-genome%20linkage%20scans%20of%20bipolar%20disorder%20and%20schizophrenia&author=JA%20Badner&author=ES%20Gershon&volume=7&publication_year=2002&pages=405-411&pmid=11986984&doi=10.1038/sj.mp.4001012&)

9. Lundstrom K, Tenhunen J, Tilgmann C, Karhunen T, Panula P, Ulmanen I. Cloning, expression and structure of catechol-O-methyltransferase. Biochim Biophys Acta. 1995;1251:1–10. doi: 10.1016/0167-4838(95)00071-2.  [DOI](https://doi.org/10.1016/0167-4838(95)00071-2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7647086/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochim%20Biophys%20Acta&title=Cloning,%20expression%20and%20structure%20of%20catechol-O-methyltransferase&author=K%20Lundstrom&author=J%20Tenhunen&author=C%20Tilgmann&author=T%20Karhunen&author=P%20Panula&volume=1251&publication_year=1995&pages=1-10&pmid=7647086&doi=10.1016/0167-4838(95)00071-2&)

10. Tenhunen J, Salminen M, Lundstrom K, Kiviluoto T, Savolainen R, Ulmanen I. Genomic organization of the human catechol O-methyltransferase gene and its expression from two distinct promoters. Eur J Biochem. 1994;223:1049–1059. doi: 10.1111/j.1432-1033.1994.tb19083.x.  [DOI](https://doi.org/10.1111/j.1432-1033.1994.tb19083.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8055944/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Biochem&title=Genomic%20organization%20of%20the%20human%20catechol%20O-methyltransferase%20gene%20and%20its%20expression%20from%20two%20distinct%20promoters&author=J%20Tenhunen&author=M%20Salminen&author=K%20Lundstrom&author=T%20Kiviluoto&author=R%20Savolainen&volume=223&publication_year=1994&pages=1049-1059&pmid=8055944&doi=10.1111/j.1432-1033.1994.tb19083.x&)

11. Weinshilboum RM. Pharmacogenomics: catechol O-methyltransferase to thiopurine S-methyltransferase. Cell Mol Neurobiol. 2006;26:539–561. doi: 10.1007/s10571-006-9095-z.  [DOI](https://doi.org/10.1007/s10571-006-9095-z) | [PMC free article](/articles/PMC11520626/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16807786/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell%20Mol%20Neurobiol&title=Pharmacogenomics:%20catechol%20O-methyltransferase%20to%20thiopurine%20S-methyltransferase&author=RM%20Weinshilboum&volume=26&publication_year=2006&pages=539-561&pmid=16807786&doi=10.1007/s10571-006-9095-z&)

12. Weinshilboum RM, Raymond FA. Inheritance of low erythrocyte catechol-o-methyltransferase activity in man. Am J Hum Genet. 1977;29:125–135.  [PMC free article](/articles/PMC1685255/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/848488/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=Inheritance%20of%20low%20erythrocyte%20catechol-o-methyltransferase%20activity%20in%20man&author=RM%20Weinshilboum&author=FA%20Raymond&volume=29&publication_year=1977&pages=125-135&pmid=848488&)

13. Scanlon PD, Raymond FA, Weinshilboum RM. Catechol-O-methyltransferase: thermolabile enzyme in erythrocytes of subjects homozygous for allele for low activity. Science. 1979;203:63–65. doi: 10.1126/science.758679.  [DOI](https://doi.org/10.1126/science.758679) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/758679/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=Catechol-O-methyltransferase:%20thermolabile%20enzyme%20in%20erythrocytes%20of%20subjects%20homozygous%20for%20allele%20for%20low%20activity&author=PD%20Scanlon&author=FA%20Raymond&author=RM%20Weinshilboum&volume=203&publication_year=1979&pages=63-65&pmid=758679&doi=10.1126/science.758679&)

14. Spielman RS, Weinshilboum RM. Genetics of red cell COMT activity: analysis of thermal stability and family data. Am J Med Genet. 1981;10:279–290. doi: 10.1002/ajmg.1320100311.  [DOI](https://doi.org/10.1002/ajmg.1320100311) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7304673/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Med%20Genet&title=Genetics%20of%20red%20cell%20COMT%20activity:%20analysis%20of%20thermal%20stability%20and%20family%20data&author=RS%20Spielman&author=RM%20Weinshilboum&volume=10&publication_year=1981&pages=279-290&pmid=7304673&doi=10.1002/ajmg.1320100311&)

15. Lachman HM, Papolos DF, Saito T, Yu YM, Szumlanski CL, Weinshilboum RM. Human catechol-O-methyltransferase pharmacogenetics: description of a functional polymorphism and its potential application to neuropsychiatric disorders. Pharmacogenetics. 1996;6:243–250. doi: 10.1097/00008571-199606000-00007.  [DOI](https://doi.org/10.1097/00008571-199606000-00007) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8807664/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=Human%20catechol-O-methyltransferase%20pharmacogenetics:%20description%20of%20a%20functional%20polymorphism%20and%20its%20potential%20application%20to%20neuropsychiatric%20disorders&author=HM%20Lachman&author=DF%20Papolos&author=T%20Saito&author=YM%20Yu&author=CL%20Szumlanski&volume=6&publication_year=1996&pages=243-250&pmid=8807664&doi=10.1097/00008571-199606000-00007&)

16. Craddock N, Owen MJ, O’Donovan MC. The catechol-O-methyl transferase (COMT) gene as a candidate for psychiatric phenotypes: evidence and lessons. Mol Psychiatry. 2006;11:446–458. doi: 10.1038/sj.mp.4001808.  [DOI](https://doi.org/10.1038/sj.mp.4001808) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16505837/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Psychiatry&title=The%20catechol-O-methyl%20transferase%20(COMT)%20gene%20as%20a%20candidate%20for%20psychiatric%20phenotypes:%20evidence%20and%20lessons&author=N%20Craddock&author=MJ%20Owen&author=MC%20O%E2%80%99Donovan&volume=11&publication_year=2006&pages=446-458&pmid=16505837&doi=10.1038/sj.mp.4001808&)

17. Goodman JE, Lavigne JA, Hengstler JG, Tanner B, Helzlsouer KJ, Yager JD. Catechol-O-methyltransferase polymorphism is not associated with ovarian cancer risk. Cancer Epidemiol Biomarkers Prev. 2000;9:1373–1376.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/11142424/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Epidemiol%20Biomarkers%20Prev&title=Catechol-O-methyltransferase%20polymorphism%20is%20not%20associated%20with%20ovarian%20cancer%20risk&author=JE%20Goodman&author=JA%20Lavigne&author=JG%20Hengstler&author=B%20Tanner&author=KJ%20Helzlsouer&volume=9&publication_year=2000&pages=1373-1376&pmid=11142424&)

18. Millikan RC, Pittman GS, Tse CK, Duell E, Newman B, Savitz D, et al. Catechol-O-methyltransferase and breast cancer risk. Carcinogenesis. 1998;19:1943–1947. doi: 10.1093/carcin/19.11.1943.  [DOI](https://doi.org/10.1093/carcin/19.11.1943) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9855007/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Carcinogenesis&title=Catechol-O-methyltransferase%20and%20breast%20cancer%20risk&author=RC%20Millikan&author=GS%20Pittman&author=CK%20Tse&author=E%20Duell&author=B%20Newman&volume=19&publication_year=1998&pages=1943-1947&pmid=9855007&doi=10.1093/carcin/19.11.1943&)

19. Thompson PA, Shields PG, Freudenheim JL, Stone A, Vena JE, Marshall JR, et al. Genetic polymorphisms in catechol-O-methyltransferase, menopausal status, and breast cancer risk. Cancer Res. 1998;58:2107–2110.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/9605753/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=Genetic%20polymorphisms%20in%20catechol-O-methyltransferase,%20menopausal%20status,%20and%20breast%20cancer%20risk&author=PA%20Thompson&author=PG%20Shields&author=JL%20Freudenheim&author=A%20Stone&author=JE%20Vena&volume=58&publication_year=1998&pages=2107-2110&pmid=9605753&)

20. Ji Y, Olson J, Zhang J, Hildebrandt M, Wang L, Ingle J, et al. Breast cancer risk reduction and membrane-bound catechol O-methyltransferase genetic polymorphisms. Cancer Res. 2008;68:5997–6005. doi: 10.1158/0008-5472.CAN-08-0043.  [DOI](https://doi.org/10.1158/0008-5472.CAN-08-0043) | [PMC free article](/articles/PMC2518124/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18632656/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=Breast%20cancer%20risk%20reduction%20and%20membrane-bound%20catechol%20O-methyltransferase%20genetic%20polymorphisms&author=Y%20Ji&author=J%20Olson&author=J%20Zhang&author=M%20Hildebrandt&author=L%20Wang&volume=68&publication_year=2008&pages=5997-6005&pmid=18632656&doi=10.1158/0008-5472.CAN-08-0043&)

21. Zhang J, Ji Y, Moon I, Pelleymounter LL, Salavaggione OE, Wu Y, Jenkins GD, Batzler AJ, Schaid DJ, Weinshilboum RM. Catechol O-methyltransferase pharmacogenomics: human liver genotype-phenotype correlation and proximal promoter studies. Pharmacogenetics and Genomics. 2009;19:577–587. doi: 10.1097/FPC.0b013e32832c15c6.  [DOI](https://doi.org/10.1097/FPC.0b013e32832c15c6) | [PMC free article](/articles/PMC3052748/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19641441/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics%20and%20Genomics&title=Catechol%20O-methyltransferase%20pharmacogenomics:%20human%20liver%20genotype-phenotype%20correlation%20and%20proximal%20promoter%20studies&author=J%20Zhang&author=Y%20Ji&author=I%20Moon&author=LL%20Pelleymounter&author=OE%20Salavaggione&volume=19&publication_year=2009&pages=577-587&pmid=19641441&doi=10.1097/FPC.0b013e32832c15c6&)

22. Shield AJ, Thomae BA, Eckloff BW, Wieben ED, Weinshilboum RM. Human catechol O-methyltransferase genetic variation: gene resequencing and functional characterization of variant allozymes. Mol Psychiatry. 2004;9:151–160. doi: 10.1038/sj.mp.4001386.  [DOI](https://doi.org/10.1038/sj.mp.4001386) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14966473/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Psychiatry&title=Human%20catechol%20O-methyltransferase%20genetic%20variation:%20gene%20resequencing%20and%20functional%20characterization%20of%20variant%20allozymes&author=AJ%20Shield&author=BA%20Thomae&author=BW%20Eckloff&author=ED%20Wieben&author=RM%20Weinshilboum&volume=9&publication_year=2004&pages=151-160&pmid=14966473&doi=10.1038/sj.mp.4001386&)

23. Howland RH. Sequenced Treatment Alternatives to Relieve Depression (STAR*D). Part 2: Study outcomes. J Psychosoc Nurs Ment Health Serv. 2008;46:21–24. doi: 10.3928/02793695-20081001-05.  [DOI](https://doi.org/10.3928/02793695-20081001-05) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18935932/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Psychosoc%20Nurs%20Ment%20Health%20Serv&title=Sequenced%20Treatment%20Alternatives%20to%20Relieve%20Depression%20(STAR*D).%20Part%202:%20Study%20outcomes&author=RH%20Howland&volume=46&publication_year=2008&pages=21-24&pmid=18935932&doi=10.3928/02793695-20081001-05&)

24. Zisook S, Ganadjian K, Moutier C, Prather R, Rao S. Sequenced Treatment Alternatives to Relieve Depression (STAR*D): lessons learned. J Clin Psychiatry. 2008;69:1184–1185. doi: 10.4088/jcp.v69n0719.  [DOI](https://doi.org/10.4088/jcp.v69n0719) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18687018/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Psychiatry&title=Sequenced%20Treatment%20Alternatives%20to%20Relieve%20Depression%20(STAR*D):%20lessons%20learned&author=S%20Zisook&author=K%20Ganadjian&author=C%20Moutier&author=R%20Prather&author=S%20Rao&volume=69&publication_year=2008&pages=1184-1185&pmid=18687018&doi=10.4088/jcp.v69n0719&)

25. Trivedi MH, Rush AJ, Wisniewski SR, Nierenberg AA, Warden D, Ritz L, et al. Evaluation of outcomes with citalopram for depression using measurement-based care in STAR*D: implications for clinical practice. Am J Psychiatry. 2006;163:28–40. doi: 10.1176/appi.ajp.163.1.28.  [DOI](https://doi.org/10.1176/appi.ajp.163.1.28) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16390886/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Psychiatry&title=Evaluation%20of%20outcomes%20with%20citalopram%20for%20depression%20using%20measurement-based%20care%20in%20STAR*D:%20implications%20for%20clinical%20practice&author=MH%20Trivedi&author=AJ%20Rush&author=SR%20Wisniewski&author=AA%20Nierenberg&author=D%20Warden&volume=163&publication_year=2006&pages=28-40&pmid=16390886&doi=10.1176/appi.ajp.163.1.28&)

26. Mrazek DA, Rush AJ, Biernacka JM, O’Kane DJ, Cunningham JM, Wieben ED, et al. SLC6A4 variation and citalopram response. Am J Med Genet B Neuropsychiatr Genet. 2008;150B:341–351. doi: 10.1002/ajmg.b.30816.  [DOI](https://doi.org/10.1002/ajmg.b.30816) | [PMC free article](/articles/PMC2660379/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18618621/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Med%20Genet%20B%20Neuropsychiatr%20Genet&title=SLC6A4%20variation%20and%20citalopram%20response&author=DA%20Mrazek&author=AJ%20Rush&author=JM%20Biernacka&author=DJ%20O%E2%80%99Kane&author=JM%20Cunningham&volume=150B&publication_year=2008&pages=341-351&pmid=18618621&doi=10.1002/ajmg.b.30816&)

27. Stram DO, Haiman CA, Hirschhorn JN, Altshuler D, Kolonel LN, Henderson BE, et al. Choosing haplotype-tagging SNPS based on unphased genotype data using a preliminary sample of unrelated subjects with an example from the Multiethnic Cohort Study. Hum Hered. 2003;55:27–36. doi: 10.1159/000071807.  [DOI](https://doi.org/10.1159/000071807) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12890923/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Hered&title=Choosing%20haplotype-tagging%20SNPS%20based%20on%20unphased%20genotype%20data%20using%20a%20preliminary%20sample%20of%20unrelated%20subjects%20with%20an%20example%20from%20the%20Multiethnic%20Cohort%20Study&author=DO%20Stram&author=CA%20Haiman&author=JN%20Hirschhorn&author=D%20Altshuler&author=LN%20Kolonel&volume=55&publication_year=2003&pages=27-36&pmid=12890923&doi=10.1159/000071807&)

28. Carlson CS, Eberle MA, Rieder MJ, Yi Q, Kruglyak L, Nickerson DA. Selecting a maximally informative set of single-nucleotide polymorphisms for association analyses using linkage disequilibrium. Am J Hum Genet. 2004;74:106–120. doi: 10.1086/381000.  [DOI](https://doi.org/10.1086/381000) | [PMC free article](/articles/PMC1181897/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14681826/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=Selecting%20a%20maximally%20informative%20set%20of%20single-nucleotide%20polymorphisms%20for%20association%20analyses%20using%20linkage%20disequilibrium&author=CS%20Carlson&author=MA%20Eberle&author=MJ%20Rieder&author=Q%20Yi&author=L%20Kruglyak&volume=74&publication_year=2004&pages=106-120&pmid=14681826&doi=10.1086/381000&)

29. Nackley AG, Shabalina SA, Tchivileva IE, Satterfield K, Korchynskyi O, Makarov SS, et al. Human catechol-O-methyltransferase haplotypes modulate protein expression by altering mRNA secondary structure. Science. 2006;314:1930–1933. doi: 10.1126/science.1131262.  [DOI](https://doi.org/10.1126/science.1131262) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17185601/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=Human%20catechol-O-methyltransferase%20haplotypes%20modulate%20protein%20expression%20by%20altering%20mRNA%20secondary%20structure&author=AG%20Nackley&author=SA%20Shabalina&author=IE%20Tchivileva&author=K%20Satterfield&author=O%20Korchynskyi&volume=314&publication_year=2006&pages=1930-1933&pmid=17185601&doi=10.1126/science.1131262&)

30. Shifman S, Bronstein M, Sternfeld M, Pisante-Shalom A, Lev-Lehman E, Weizman A, et al. A highly significant association between a COMT haplotype and schizophrenia. Am J Hum Genet. 2002;71:1296–1302. doi: 10.1086/344514.  [DOI](https://doi.org/10.1086/344514) | [PMC free article](/articles/PMC378567/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12402217/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=A%20highly%20significant%20association%20between%20a%20COMT%20haplotype%20and%20schizophrenia&author=S%20Shifman&author=M%20Bronstein&author=M%20Sternfeld&author=A%20Pisante-Shalom&author=E%20Lev-Lehman&volume=71&publication_year=2002&pages=1296-1302&pmid=12402217&doi=10.1086/344514&)

31. Peters EJ, Slager SL, Kraft JB, Jenkins GD, Reinalda MS, McGrath PJ, et al. Pharmacokinetic genes do not influence response or tolerance to citalopram in the STAR*D sample. PLoS ONE. 2008;3:e1872. doi: 10.1371/journal.pone.0001872.  [DOI](https://doi.org/10.1371/journal.pone.0001872) | [PMC free article](/articles/PMC2268970/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18382661/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&title=Pharmacokinetic%20genes%20do%20not%20influence%20response%20or%20tolerance%20to%20citalopram%20in%20the%20STAR*D%20sample&author=EJ%20Peters&author=SL%20Slager&author=JB%20Kraft&author=GD%20Jenkins&author=MS%20Reinalda&volume=3&publication_year=2008&pages=e1872&pmid=18382661&doi=10.1371/journal.pone.0001872&)

32. Kaakkola S, Gordin A, Mannisto PT. General properties and clinical possibilities of new selective inhibitors of catechol O-methyltransferase. Gen Pharmacol. 1994;25:813–824. doi: 10.1016/0306-3623(94)90082-5.  [DOI](https://doi.org/10.1016/0306-3623(94)90082-5) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7835624/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gen%20Pharmacol&title=General%20properties%20and%20clinical%20possibilities%20of%20new%20selective%20inhibitors%20of%20catechol%20O-methyltransferase&author=S%20Kaakkola&author=A%20Gordin&author=PT%20Mannisto&volume=25&publication_year=1994&pages=813-824&pmid=7835624&doi=10.1016/0306-3623(94)90082-5&)

33. Loots GG, Ovcharenko I, Pachter L, Dubchak I, Rubin EM. rVista for comparative sequence-based discovery of functional transcription factor binding sites. Genome Res. 2002;12:832–839. doi: 10.1101/gr.225502.  [DOI](https://doi.org/10.1101/gr.225502) | [PMC free article](/articles/PMC186580/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11997350/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Res&title=rVista%20for%20comparative%20sequence-based%20discovery%20of%20functional%20transcription%20factor%20binding%20sites&author=GG%20Loots&author=I%20Ovcharenko&author=L%20Pachter&author=I%20Dubchak&author=EM%20Rubin&volume=12&publication_year=2002&pages=832-839&pmid=11997350&doi=10.1101/gr.225502&)

34. Loots GG, Locksley RM, Blankespoor CM, Wang ZE, Miller W, Rubin EM, et al. Identification of a coordinate regulator of interleukins 4, 13, and 5 by cross-species sequence comparisons. Science. 2000;288:136–140. doi: 10.1126/science.288.5463.136.  [DOI](https://doi.org/10.1126/science.288.5463.136) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10753117/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=Identification%20of%20a%20coordinate%20regulator%20of%20interleukins%204,%2013,%20and%205%20by%20cross-species%20sequence%20comparisons&author=GG%20Loots&author=RM%20Locksley&author=CM%20Blankespoor&author=ZE%20Wang&author=W%20Miller&volume=288&publication_year=2000&pages=136-140&pmid=10753117&doi=10.1126/science.288.5463.136&)

35. Grabe N. AliBaba2: context specific identification of transcription factor binding sites. In Silico Biol. 2002;2:S1–15.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/11808873/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=In%20Silico%20Biol&title=AliBaba2:%20context%20specific%20identification%20of%20transcription%20factor%20binding%20sites&author=N%20Grabe&volume=2&publication_year=2002&pages=S1-15&pmid=11808873&)

36. Landschulz WH, Johnson PF, Adashi EY, Graves BJ, McKnight SL. Isolation of a recombinant copy of the gene encoding C/EBP. Genes Dev. 1988;2:786–800. doi: 10.1101/gad.2.7.786.  [DOI](https://doi.org/10.1101/gad.2.7.786) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/2850264/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genes%20Dev&title=Isolation%20of%20a%20recombinant%20copy%20of%20the%20gene%20encoding%20C/EBP&author=WH%20Landschulz&author=PF%20Johnson&author=EY%20Adashi&author=BJ%20Graves&author=SL%20McKnight&volume=2&publication_year=1988&pages=786-800&pmid=2850264&doi=10.1101/gad.2.7.786&)

37. Birkenmeier EH, Gwynn B, Howard S, Jerry J, Gordon JI, Landschulz WH, et al. Tissue-specific expression, developmental regulation, and genetic mapping of the gene encoding CCAAT/enhancer binding protein. Genes Dev. 1989;3:1146–1156. doi: 10.1101/gad.3.8.1146.  [DOI](https://doi.org/10.1101/gad.3.8.1146) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/2792758/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genes%20Dev&title=Tissue-specific%20expression,%20developmental%20regulation,%20and%20genetic%20mapping%20of%20the%20gene%20encoding%20CCAAT/enhancer%20binding%20protein&author=EH%20Birkenmeier&author=B%20Gwynn&author=S%20Howard&author=J%20Jerry&author=JI%20Gordon&volume=3&publication_year=1989&pages=1146-1156&pmid=2792758&doi=10.1101/gad.3.8.1146&)

38. Thase ME, Haight BR, Richard N, Rockett CB, Mitton M, Modell JG, et al. Remission rates following antidepressant therapy with bupropion or selective serotonin reuptake inhibitors: a meta-analysis of original data from 7 randomized controlled trials. J Clin Psychiatry. 2005;66:974–981. doi: 10.4088/jcp.v66n0803.  [DOI](https://doi.org/10.4088/jcp.v66n0803) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16086611/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Psychiatry&title=Remission%20rates%20following%20antidepressant%20therapy%20with%20bupropion%20or%20selective%20serotonin%20reuptake%20inhibitors:%20a%20meta-analysis%20of%20original%20data%20from%207%20randomized%20controlled%20trials&author=ME%20Thase&author=BR%20Haight&author=N%20Richard&author=CB%20Rockett&author=M%20Mitton&volume=66&publication_year=2005&pages=974-981&pmid=16086611&doi=10.4088/jcp.v66n0803&)

39. Serretti A, Artioli P. The pharmacogenomics of selective serotonin reuptake inhibitors. Pharmacogenomics J. 2004;4:233–244. doi: 10.1038/sj.tpj.6500250.  [DOI](https://doi.org/10.1038/sj.tpj.6500250) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15111987/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=The%20pharmacogenomics%20of%20selective%20serotonin%20reuptake%20inhibitors&author=A%20Serretti&author=P%20Artioli&volume=4&publication_year=2004&pages=233-244&pmid=15111987&doi=10.1038/sj.tpj.6500250&)

40. Kirchheiner J, Nickchen K, Bauer M, Wong ML, Licinio J, Roots I, et al. Pharmacogenetics of antidepressants and antipsychotics: the contribution of allelic variations to the phenotype of drug response. Mol Psychiatry. 2004;9:442–473. doi: 10.1038/sj.mp.4001494.  [DOI](https://doi.org/10.1038/sj.mp.4001494) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15037866/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Psychiatry&title=Pharmacogenetics%20of%20antidepressants%20and%20antipsychotics:%20the%20contribution%20of%20allelic%20variations%20to%20the%20phenotype%20of%20drug%20response&author=J%20Kirchheiner&author=K%20Nickchen&author=M%20Bauer&author=ML%20Wong&author=J%20Licinio&volume=9&publication_year=2004&pages=442-473&pmid=15037866&doi=10.1038/sj.mp.4001494&)

41. Kato M, Serretti A. Review and meta-analysis of antidepressant pharmacogenetic findings in major depressive disorder. Mol Psychiatry. 2008;15:473–500. doi: 10.1038/mp.2008.116.  [DOI](https://doi.org/10.1038/mp.2008.116) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18982004/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Psychiatry&title=Review%20and%20meta-analysis%20of%20antidepressant%20pharmacogenetic%20findings%20in%20major%20depressive%20disorder&author=M%20Kato&author=A%20Serretti&volume=15&publication_year=2008&pages=473-500&pmid=18982004&doi=10.1038/mp.2008.116&)

42. Peters EJ, Slager SL, Jenkins GD, Reinalda MS, Garriock HA, Shyn SI, et al. Resequencing of serotonin-related genes and association of tagging SNPs to citalopram response. Pharmacogenet Genomics. 2009;19:1–10. doi: 10.1097/FPC.0b013e3283163ecd.  [DOI](https://doi.org/10.1097/FPC.0b013e3283163ecd) | [PMC free article](/articles/PMC2896826/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19077664/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Resequencing%20of%20serotonin-related%20genes%20and%20association%20of%20tagging%20SNPs%20to%20citalopram%20response&author=EJ%20Peters&author=SL%20Slager&author=GD%20Jenkins&author=MS%20Reinalda&author=HA%20Garriock&volume=19&publication_year=2009&pages=1-10&pmid=19077664&doi=10.1097/FPC.0b013e3283163ecd&)

43. Szegedi A, Rujescu D, Tadic A, Muller MJ, Kohnen R, Stassen HH, et al. The catechol-O-methyltransferase Val108/158Met polymorphism affects short-term treatment response to mirtazapine, but not to paroxetine in major depression. Pharmacogenomics J. 2005;5:49–53. doi: 10.1038/sj.tpj.6500289.  [DOI](https://doi.org/10.1038/sj.tpj.6500289) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15520843/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=The%20catechol-O-methyltransferase%20Val108/158Met%20polymorphism%20affects%20short-term%20treatment%20response%20to%20mirtazapine,%20but%20not%20to%20paroxetine%20in%20major%20depression&author=A%20Szegedi&author=D%20Rujescu&author=A%20Tadic&author=MJ%20Muller&author=R%20Kohnen&volume=5&publication_year=2005&pages=49-53&pmid=15520843&doi=10.1038/sj.tpj.6500289&)

44. Arias B, Serretti A, Lorenzi C, Gasto C, Catalan R, Fananas L. Analysis of COMT gene (Val 158 Met polymorphism) in the clinical response to SSRIs in depressive patients of European origin. J Affect Disord. 2006;90:251–256. doi: 10.1016/j.jad.2005.11.008.  [DOI](https://doi.org/10.1016/j.jad.2005.11.008) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16356553/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Affect%20Disord&title=Analysis%20of%20COMT%20gene%20(Val%20158%20Met%20polymorphism)%20in%20the%20clinical%20response%20to%20SSRIs%20in%20depressive%20patients%20of%20European%20origin&author=B%20Arias&author=A%20Serretti&author=C%20Lorenzi&author=C%20Gasto&author=R%20Catalan&volume=90&publication_year=2006&pages=251-256&pmid=16356553&doi=10.1016/j.jad.2005.11.008&)

45. Yoshida K, Higuchi H, Takahashi H, Kamata M, Sato K, Inoue K, et al. Influence of the tyrosine hydroxylase val81met polymorphism and catechol-O-methyltransferase val158met polymorphism on the antidepressant effect of milnacipran. Hum Psychopharmacol. 2008;23:121–128. doi: 10.1002/hup.907.  [DOI](https://doi.org/10.1002/hup.907) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18023073/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Psychopharmacol&title=Influence%20of%20the%20tyrosine%20hydroxylase%20val81met%20polymorphism%20and%20catechol-O-methyltransferase%20val158met%20polymorphism%20on%20the%20antidepressant%20effect%20of%20milnacipran&author=K%20Yoshida&author=H%20Higuchi&author=H%20Takahashi&author=M%20Kamata&author=K%20Sato&volume=23&publication_year=2008&pages=121-128&pmid=18023073&doi=10.1002/hup.907&)

46. Garriock HA, Kraft JB, Shyn SI, Peters EJ, Yokoyama JS, Jenkins GD, et al. A genomewide association study of citalopram response in major depressive disorder. Biol Psychiatry. 2010;67:133–138. doi: 10.1016/j.biopsych.2009.08.029.  [DOI](https://doi.org/10.1016/j.biopsych.2009.08.029) | [PMC free article](/articles/PMC2794921/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19846067/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biol%20Psychiatry&title=A%20genomewide%20association%20study%20of%20citalopram%20response%20in%20major%20depressive%20disorder&author=HA%20Garriock&author=JB%20Kraft&author=SI%20Shyn&author=EJ%20Peters&author=JS%20Yokoyama&volume=67&publication_year=2010&pages=133-138&pmid=19846067&doi=10.1016/j.biopsych.2009.08.029&)
