# PXR and CAR single nucleotide polymorphisms influence plasma efavirenz levels in South African HIV/AIDS patients

## Metadata
**Authors:** Marelize Swart, Heather Whitehorn, Yuan Ren, Peter Smith, Rajkumar S Ramesar, Collet Dandara
**Journal:** BMC Medical Genetics
**Date:** 2012 Nov 22
**DOI:** [10.1186/1471-2350-13-112](https://doi.org/10.1186/1471-2350-13-112)
**PMID:** 23173844
**PMCID:** PMC3523080
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3523080/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC3523080/pdf/1471-2350-13-112.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC3523080/pdf/1471-2350-13-112.pdf)

## Abstract

**Background:** 
This study investigated variation in NR1I2 and NR1I3 and its effect on plasma efavirenz levels in HIV/AIDS patients. Variability in plasma drug levels has largely led research on identifying causative variants in drug metabolising enzyme (DME) genes, with little focus on the nuclear receptor genes NR1I2 and NR1I3, coding for PXR and CAR, respectively, that are involved in regulating DMEs.

**Methods:** 
464 Bantu-speaking South Africans comprising of HIV/AIDS patients on efavirenz-based treatment (n=301) and 163 healthy subjects were genotyped for 6 SNPs in NR1I2 and NR1I3. 32 of the 301 patients had their DNA binding domains (DBDs) in NR1I2 and NR1I3 sequenced.

**Results:** 
Significantly decreased efavirenz plasma concentrations were observed in patients carrying the NR1I3 rs3003596C/C and T/C genotypes (P=0.015 and P=0.010, respectively). Sequencing resulted in the discovery of a further 13 SNPs, 3 of which are novel variants in the DBD of NR1I2. There were significant differences in the distribution of NR1I2 and NR1I3 SNPs between South Africans when compared to Caucasian, Asian and Yoruba population groups.

**Conclusion:** 
For the realisation of personalised medicine, PXR and CAR genetic variation should be taken into consideration because of their involvement in the regulation of DMEs.

Keywords: CAR, Efavirenz, HIV/AIDS, PXR, Pharmacogenetics, South Africa

### Background

This study investigated variation in *NR1I2* and *NR1I3* and its effect on plasma efavirenz levels in HIV/AIDS patients. Variability in plasma drug levels has largely led research on identifying causative variants in drug metabolising enzyme (DME) genes, with little focus on the nuclear receptor genes *NR1I2* and *NR1I3*, coding for PXR and CAR, respectively, that are involved in regulating DMEs.

### Methods

464 Bantu-speaking South Africans comprising of HIV/AIDS patients on efavirenz-based treatment (n=301) and 163 healthy subjects were genotyped for 6 SNPs in *NR1I2* and *NR1I3*. 32 of the 301 patients had their DNA binding domains (DBDs) in *NR1I2* and *NR1I3* sequenced.

### Results

Significantly decreased efavirenz plasma concentrations were observed in patients carrying the *NR1I3 rs3003596C/C* and *T/C* genotypes (P=0.015 and P=0.010, respectively). Sequencing resulted in the discovery of a further 13 SNPs, 3 of which are novel variants in the DBD of *NR1I2*. There were significant differences in the distribution of *NR1I2* and *NR1I3* SNPs between South Africans when compared to Caucasian, Asian and Yoruba population groups.

### Conclusion

For the realisation of personalised medicine, PXR and CAR genetic variation should be taken into consideration because of their involvement in the regulation of DMEs.

## Background

The pregnane X receptor (PXR) and the constitutive androstane receptor (CAR) are members of the orphan nuclear receptor (NR) superfamily and function primarily as sensors of xenobiotics by up-regulating the expression of drug metabolising enzymes (DMEs), facilitating the elimination of xenobiotics from the body [[1](#B1),[2](#B2)]. PXR and CAR are transcriptional regulators of a wide range of genes whose products metabolise a wide range of drugs [[2](#B2),[3](#B3)].

PXR and CAR are localised in the cytoplasm in their inactive state, bound to histone deacetylase complexes (HDACs). Upon ligand binding, PXR and CAR dissociate from the HDACs. PXR translocates into the nucleus where it heterodimerises with the retinoid X receptor (RXR), while CAR is nuclear translocated through a phosphorylation-dependent mechanism and subsequently heterodimerises with RXR [[4](#B4)]. In the nucleus, the heterodimers PXR/RXR and CAR/RXR bind to nuclear receptor response elements upstream of genes coding for DMEs) [[4](#B4)].

*NR1I2* gene, which encodes PXR, consists of 10 exons and is located on chromosome 3q13-21 [[5](#B5)]. The *NR1I2* DNA binding domain (DBD) is encoded by exons 3 and 4 whereas exons 5–10 code for the ligand binding domain (LBD). The DBD and LBD are separated by a hinge region encoded by a small portion of exon 5 [[5](#B5)]. Several SNPs have been reported in *NR1I2* and some are associated with changes in PXR function*.* For example, *NR1I2 63396C>T (rs2472677)*, which is located in a putative transcription factor binding site, has been associated with increased *NR1I2* expression in the presence of the *63396T* variant, which leads to increased *CYP3A4* expression leading to decreased atazanavir (ATV) plasma concentrations [[6](#B6)-[8](#B8)]. Three SNPs in exon 2 have been reported, namely *NR1I2 52G>A (E18K; rs59371185), 79C>T (P27S; rs12721613)* and *106G>A (G36R; rs12721607)*. The *NR1I2 79T* and *NR1I2 106A* alleles are associated with decreased *NR1I2* expression [[5](#B5)]. Another SNP, *NR1I2 7635A>G (rs6785049)*, is present in intron 5 and the *7635G* allele has been associated with increased expression of *CYP3A4* in the presence of rifampicin [[9](#B9)].

The *NR1I3* gene, which encodes CAR, is located on chromosome 1q21-23, and consists of 9 exons. Twenty-two unique *NR1I3* variants have been described where each isoform results from a different combination of splicing events. Some isoforms produce non-functional proteins due to the presence of nonsense mutations [[10](#B10),[11](#B11)]. *NR1I3* isoform-3 has been suggested as the wild-type and produces a 348 amino acid protein [[11](#B11)]. The *NR1I3* DBD is encoded by exons 2, 3 and the 5’ portion of exon 4 [[12](#B12)]. Previously characterised SNPs in *NR1I3* include *NR1I3 rs2307424C>T*, of which the *rs2307424T* allele has been associated with low efavirenz plasma concentrations and the *NR1I3 rs2307424C/C* genotype has been associated with early discontinuation of efavirenz-containing anti-retroviral therapy (ART) in Caucasian HIV/AIDS patients [[13](#B13)].

Genetic characterization of indigenous African populations is slowly building up. This study aimed to further contribute to the genetic characterization of African populations by genotyping *NR1I2* and *NR1I3* and evaluating the effects of their variants on the response to efavirenz treatment in HIV/AIDS Bantu-speaking South African patients. In order to accelerate discovery of novel SNPs, the DBD of both *NR1I2* and *NR1I3* were targeted for sequencing [[1](#B1)].

## Methods

### Study subjects

The study cohort consisted of four-hundred and sixty-four (n=464) Bantu-speaking South Africans made up of healthy subjects (n=163) and HIV/AIDS patients (n=301) undergoing efavirenz-based treatment for at least six months. The subjects were recruited from Gauteng and Cape Town. Written informed consent was obtained and each participant provided demographic information such as 1) their ethnic group, 2) health status, 3) dietary habits, 4) smoking habits, and 5) home language were captured using a questionnaire. The study was approved by the Research Ethics Committee of the Faculty of Health Sciences at the University of Cape Town and the University of Witwatersrand Human Research Ethics Committee, Gauteng, South Africa and was performed in accordance with the guidelines of the Helsinki Declaration of 2008.

Two blood samples were obtained for DNA extraction and plasma efavirenz levels, respectively. DNA isolation was performed according to the method adapted from Gustafson *et al.,*[[14](#B14)] or the GenElute^TM^ Blood Genomic DNA Kit (Sigma-Aldrich, St. Louis, MO, USA) was used when blood sample volumes were limited. Steady state efavirenz plasma levels were available for 137 of the 301 HIV/AIDS patients and were collected 12–16 hours post-dose. Efavirenz concentrations were determined by the use of LC/MS/MS (API 4000 triple quadrupole MS/MS Applied Biosystems, South Africa) according to the method by Chi *et al.*, [[15](#B15)].

### Selection of SNPs and genotyping methods used

Three SNPs in *NR1I2* [GenBank: [AF364606](https://www.ncbi.nlm.nih.gov/nuccore/AF364606)] and a further three SNPs in *NR1I3* [GenBank: [BC069626.1](https://www.ncbi.nlm.nih.gov/nuccore/BC069626.1)] were investigated in this study. The six SNPs were selected based on previous reports of high minor allele frequencies in African-American and other African populations. SNPs were genotyped using either SNaPshot mini-sequencing or the PCR-RFLP method designed for *NR1I2 rs2472677C>T* (Additional file [1](#S1): Table S1).

PCR amplification was performed using the following conditions: initial denaturation at 94°C for 3 min, followed by 40 cycles of denaturation at 94°C for 30s, annealing at the specific temperature for each SNP for 30s, primer extension at 72°C for 20-45s depending on the primer sets and final extension at 72°C for 10 min. A “MyCycler Thermal cycler” (Bio-Rad, Hercules, USA) was used and the PCR reaction contained the following reagents; 50–100 ng of genomic DNA, 1X Green GoTaq Flexi Reaction Buffer (Promega Corporation, Madison, USA), 0.20 mM of each of the deoxynucleotide triphosphates (dNTPs) (Bioline, London, UK), 1.5 mM MgCl_2_ (Promega Corporation, Madison, USA), 40 pmol of the forward and reverse primers (Integrated DNA Technologies, Inc., Coralville, USA), 1U of GoTaq Flexi DNA Polymerase (Promega Corporation, Madison, USA). PCR amplification was followed by digestion using 3U *Hpy188I* (New England BioLabs, Inc., Ipswich) in the presence of 1X NEBuffer 4 (New England BioLabs, Inc., Ipswich) when genotyping for the *NR1I2 rs2472677C>T* polymorphism.

### SNaPshot multiplex

Five separate PCR amplification reactions were carried out. PCR products (5 μl) were then pooled for SNaPshot genotyping. The pooled PCR products were cleaned using 1.5U shrimp alkaline phosphatase (Fermentas Life Sciences, Burlington, Canada) and 2U *ExonucleaseI* (Fermentas Life Sciences, Burlington, Canada) to remove unincorporated primers and dNTPs. SNaPshot single base extension was performed using the “GeneAmp® PCR System 9700 version 3.08” (Applied Biosystems, Carlsbad, USA) under the following conditions; denaturation at 96°C for 10s, followed by 25 cycles of primer annealing at 50°C for 5s and primer extension at 60°C for 30s. To the 1 μl ABI Prism® SNaPshot™ Multiplex Kit (Applied Biosystems, California, USA), primers (Integrated DNA Technologies, Inc., Coralville, USA) for the pooled PCR products were added. The clean-up reaction was repeated using 1U shrimp alkaline phosphatase. An ABI 3130xl Genetic Analyzer (Applied Biosystems, Carlsbad, USA) was used for capillary electrophoresis and GeneMapper© Software version 4.1 (Applied Biosystems, Carlsbad, USA) was used to analyse results.

### Identification of novel SNPs

The *NR1I2* and *NR1I3* DNA binding domains (DBDs) were sequenced in 32 of the 301 HIV/AIDS patients to search for novel SNPs. The sequencing reaction used the ABI Prism® BigDye® Terminator Cycle Sequencing v3.1 Kit (Applied Biosystems, Carlsbad, CA, USA), which included 1 μl Terminator mix and 1X Sequencing buffer, together with the PCR fragment, and 1 μM of the forward or reverse primer. Analysis of the sequencing data was performed using BioEdit Sequence Alignment Editor v7.0.0. The novel SNPs were assessed for functional significance with the Functional Analysis of Novel SNPs (FANS) program ( [http://genepipe.ncgm.sinica.edu.tw/fans/input.do](http://genepipe.ncgm.sinica.edu.tw/fans/input.do)) [[16](#B16)] and ESE finder v3.0.

### Statistical analysis

Statistical analyses were performed using the Graphpad Prism statistical program (Version 5, GraphPad Software Inc., San Diego, CA), Statistica v10.0 (StatSoft, USA) and Phase v2.1 [[17](#B17)-[19](#B19)]. Pearson’s *χ*^2^-test and Fisher’s exact test was used to compare the genotype and allele frequencies between the healthy participants and the HIV/AIDS patients as well as the allele frequencies in the South Africans to those of other populations with results in literature. The SHEsis statistical program was used for linkage disequilibrium (LD, D’ and r^2^) [[20](#B20),[21](#B21)] analysis and Phase v2.1 for inferring of *NR1I2* and *NR1I3* haplotypes. Statistical significance was defined as *P* <0.05 and all statistical tests were performed two tailed.

## Results

### Demographic characteristics

The healthy subjects had a mean age of 35.8 years (SD ± 14.0 years), while the HIV/AIDS patients (consisting of 71 males, 226 females and 4 patients where gender were not recorded) had a mean age of 41.3 years (SD ±9.3 years). Among the HIV/AIDS patients, efavirenz plasma concentrations were available in 137 subjects. A summary of the baseline characteristics of the study cohort is outlined in Table [1](#T1). The efavirenz plasma concentration in the South African HIV/AIDS patients showed a large degree of variation (36-fold), ranging between 0.59 and 22 μg/mL, suggesting extensive inter-individual variability in efavirenz drug metabolism and disposition.

### Table 1.

|   | HIV/AIDS patients (n=301) |  |
| --- | --- | --- |
| Gender ratio (Male: Female) | 1:3.8 |   |
| Mean age, years (range) | 41.3 ±9.3 | (22–75) |
| Tobacco smoking |   |   |
| Yes | 20 | (0.07) |
| No | 268 | (0.93) |
| Alcohol consumption |   |   |
| Yes | 29 | (0.10) |
| No | 259 | (0.90) |
| Mean BMI, kg/m2 (range) | 23.23 ±4.74 | (11.7-40.6) |
| Average viral load at baseline, copies/ml (range) | 26917.71 ±27133.50 | (25–98400) |
| Average CD4 count at baseline, cells/μl (range) | 136.09 ±113.24 | (2–605) |
| Efavirenz-containing ARV regimens |   |   |
| 3TC_TDF_EFV | 9 | (0.03) |
| AZT_3TC_EFV | 11 | (0.04) |
| d4T_3TC_EFV | 222 | (0.74) |
| Average plasma efavirenz concentration, μg/mL (range) | 4.64 | (0.59 – 22) |
| Mean Haemoglobin level, IU/l (range) | 11.58 ±2.14 | (5.4-16.3) |
| Median AST level, IU/l (range) | 32 | (15 – 228) |
| Median ALT level, IU/l (range) | 23.5 | (6 – 176) |
|   | Healthy subjects (n=163) |  |
| Gender ratio (Male: Female) | 1:2.3 |   |
| Mean age, years (range) | 35.8 ±13.96 | (21–63) |
| Tobacco smoking |   |   |
| Yes | 8 | (0.15) |
| No | 44 | (0.85) |
| Alcohol consumption |   |   |
| Yes | 22 | (0.42) |
| No | 30 | (0.58) |

Table 1 Caption: Demographic characteristics of the healthy participants and HIV/AIDS patients

### Genotype frequencies

Genotype frequencies were compared between the healthy subjects and HIV/AIDS patients for the six SNPs, three each in *NR1I2* and *NR1I3,* genotyped using SNaPshot or PCR-RFLP. The genotypes of the healthy subjects were all in HWE (P>0.05) for the six SNPs. However, the *NR1I2 rs3732356T>G* (P=0.020) genotype frequencies deviated from HWE in the HIV/AIDS patients. Polymorphic variation was observed in all six SNPs and all genotypes were observed in both healthy subjects and HIV/AIDS patients except for the *NR1I2 rs6785049A/A* genotype, which was absent in the HIV/AIDS patients and the *NR1I3 rs2307424T/T* genotype, which was not observed in both the healthy subjects and HIV/AIDS patients (Table [2](#T2)). The distribution of *NR1I2 rs3732356T>G* and *NR1I2 rs6785049G>A* genotypes were significantly different between the healthy subjects and HIV/AIDS patients (P=0.031 and 0.002, respectively) (Table [2](#T2)). Although gender differences are known to result in differences in drug disposition, no association of gender with plasma efavirenz levels were observed in this study with average plasma efavirenz levels of 5.25 μg/mL and 4.43 μg/mL in males and females, respectively (P=0.307).

### Table 2.

| SNP | Genotype | Location/functional effect | Healthy subjects | HIV/AIDS patients | Global p-value |
| --- | --- | --- | --- | --- | --- |
| n (freq) | n (freq) |  |  |  |  |
| NR1I2 rs3732356T>G | T/T | Intron 3 | 95 (0.605) | 162 (0.549) | 0.031 |
| T/G |   | 50 (0.318) | 123 (0.417) |   |  |
| G/G |   | 12 (0.076) | 10 (0.034) |   |  |
| NR1I2 rs2472677C>T | C/C | Intron 1, Disrupts HNF3B TFB site | 57 (0.368) | 97 (0.365) | 0.917 |
| C/T | 75 (0.484) | 133 (0.500) |   |  |  |
| T/T | 23 (0.148) | 36 (0.135) |   |  |  |
| NR1I2 rs6785049G>A | G/G | Intron 5 | 128 (0.853) | 264 (0.950) | 0.002 |
| G/A | 21 (0.140) | 14 (0.050) |   |  |  |
| A/A | 1 (0.007) | 0 (0.000) |   |  |  |
| NR1I3 rs2307424C>T | C/C | Exon 5, Pro180Pro | 138 (0.879) | 270 (0.915) | 0.216 |
| C/T | 19 (0.121) | 25 (0.085) |   |  |  |
| NR1I3 rs3003596T>C | T/T | Intron 3 | 51 (0.325) | 100 (0.341) | 0.659 |
| T/C | 72 (0.459) | 140 (0.478) |   |  |  |
| C/C | 34 (0.217) | 53 (0.181) |   |  |  |
| NR1I3 rs2502815C>T | C/C | Intron 3 | 88 (0.564) | 180 (0.610) | 0.173 |
| C/T | 53 (0.340) | 100 (0.339) |   |  |  |
| T/T | 15 (0.096) | 15 (0.051) |   |  |  |

Table 2 Caption: Comparison of the genotype frequencies between control subjects and HIV/AIDS patients

### Variants discovered through targeted sequencing of NR1I2 and NR1I3 DNA binding domains (DBD) in 32 HIV/AIDS patients

Targeted sequencing of the DBDs in *NR1I2* [GenBank: [AF364606](https://www.ncbi.nlm.nih.gov/nuccore/AF364606)] and *NR1I3* [GenBank: [BC069626.1](https://www.ncbi.nlm.nih.gov/nuccore/BC069626.1)] in 32 HIV/AIDS patients identified a total of 13 genetic variants (Table [3](#T3))*.* Only three of these were novel variants discovered in the DBD of *NR1I2 (36726T>C,* in intron 1*; 36857G>A* and *36905C>T,* both in exon 2) (Additional file [2](#S2): Figure S1). Using the FANS program, the functional significance of the novel SNPs was predicted. While *NR1I2 36726T>C* change was predicted to be of little functional significance, *NR1I2 36857A* variant was associated with increased binding affinity of SRp40 splicing proteins compared to the *NR1I2 36857G* variant.

### Table 3.

| SNP ID | Location/Functional effect | Minor allele | Minor allele n (freq) |
| --- | --- | --- | --- |
| NR1I2 rs12721601T>C | Previously reported, Intron 1 | C | 1 (0.016) |
| NR1I2 rs59371185G>A | Previously reported, Exon 2, Glu18Lys | A | 1 (0.016) |
| NR1I2 rs12721613C>T | Previously reported, Exon 2, Pro27Ser | T | 4 (0.064) |
| NR1I2 rs1464603C>T | Previously reported, Intron 2, Disrupts HNF1 TFB site | T | 1 (0.016) |
| NR1I2 rs1464602C>T | Previously reported, Intron 2 | T | 20 (0.313) |
| NR1I2 rs80320762G>A | Previously reported, Intron 2 | A | 3 (0.048) |
| NR1I2 rs12721616C>T | Previously reported, Intron 2 | T | 14 (0.219) |
| NR1I2 rs112813596G>A | Previously reported, Intron 3 | A | 1 (0.016) |
| NR1I3 rs35205211C>G | Previously reported, Exon 4, Ala86Ala | G | 2 (0.032) |
| NR1I3 rs34161743C>T | Previously reported, Exon 4, Arg97Trp | T | 1 (0.016) |
| NR1I2 36726T>C | Novel, Intron 1 | C | 5 (0.078) |
| NR1I2 36857G>A* | Novel, Exon 2 | A | 1 (0.016) |
| NR1I2 36905C>T | Novel, Exon 2 | T | 2 (0.031) |

Table 3 Caption: NR1I2 and NR1I3 genetic variants in 32 HIV/AIDS patients following targeted sequencing of the NR1I2 and NR1I3 DNA binding domains

### Correlation of NR1I2 and NR1I3 variants with plasma efavirenz concentrations, change in treatment regimens and effects of CYP2B6 516G>T SNP

The *NR1I3 rs3003596C/C* and *T/C* genotypes were associated with significantly reduced plasma efavirenz concentrations compared to the *NR1I3 rs3003596T/T* genotype with P-values of 0.015 and 0.010, respectively and remained significant after Bonferroni’s correction for multiple comparison tests for the three *NR1I3* SNPs with significant P<0.017 (Figure [1](#F1)D). Three of the twenty-two (14%) individuals with the *rs3003596C/C* genotype had plasma efavirenz concentrations above 4 μg/mL, while twenty-four of the fifty (48%) individuals with the *rs3003596T/T* genotype had plasma efavirenz concentrations above 4 μg/mL. The trend towards reduced plasma efavirenz levels associated with *NR1I3 rs3003596C/C* and *T/C* genotypes remained despite stratification according to *CYP2B6 G516T* genotypes (Figure [2](#F2)). CYP2B6 is the main enzyme involved in the metabolism of efavirenz and the *CYP2B6 516T* variant is associated with reduced CYP2B6 activity and inversely, increased plasma efavirenz levels. The effect of the *rs3003596C* variant is clearly demonstrated in Figure [2](#F2)C where, in the absence of CYP2B6 substantial activity, the variant is associated with significantly decreased efavirenz levels. This points to possible increased *NR1I3* expression in the presence of the *NR1I3 rs3003596C* variant, eliciting its effects through other enzymes that participate in efavirenz metabolism.

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/096f/3523080/194f5f9f74b1/1471-2350-13-112-1.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3523080_1471-2350-13-112-1.jpg)

A-H: Effects of NR1I2 and NR1I3 variation on plasma efavirenz concentration. Only significant P-values are indicated.

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/096f/3523080/5a4766497eaa/1471-2350-13-112-2.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3523080_1471-2350-13-112-2.jpg)

A-C: Effects of NR1I3 rs3003596 and CYP2B6 G516T variation on plasma efavirenz concentration.A) NR1I3 rs3003596 stratified by CYP2B6 516G/G. B) Stratified by CYP2B6 516G/T.C) Stratified by CYP2B6 516T/T. Only significant P-values are indicated.

Although not statistically significant, *NR1I2 rs3732356G/G* genotype was associated with high plasma efavirenz concentrations while *NR1I3 rs2502815T/T* and *NR1I3 rs2307424C/T* genotypes were associated with reduced plasma efavirenz levels. It was observed that the *NR1I2 rs3732356G/G* genotype was associated with the least likelihood of changing treatment regimens at 3, 6 and 12 months (0%, 0%, 0%), while 4%, 11%, 24% and 3%, 6%, 21% in patients with the *T/T* and *T/G* genotypes, respectively, had their treatment regimens changed (Table [4](#T4)). However, this finding could have been influenced by the small sample size (n=9) of the *NR1I2 rs3732356G/G* genotype group (Figure [1](#F1)A). It was also observed that *NR1I3 rs2307424C/T* genotype was associated with the highest number (12%, 20% and 32%) of patients switching from efavirenz-based treatment regimen at all three time intervals, respectively (Table [4](#T4)).

### Table 4.

| Gene | ARV regimen or Genotype | Treatment initiation (n) | 3 months | 6 months | 12 months |
| --- | --- | --- | --- | --- | --- |
| ARV regimen |  |  |  |  |  |
|   | 3TC_TDF_EFV | 9 | 0.11 | 0.11 | 0.22 |
|   | AZT_3TC_EFV | 11 | 0.00 | 0.09 | 0.27 |
|   | d4T_3TC_EFV | 222 | 0.03 | 0.08 | 0.22 |
| NR1I2 |   |   |   |   |   |
|   | rs3732356T/T | 156 | 0.04 | 0.11 | 0.24 |
|   | rs3732356T/G | 117 | 0.03 | 0.06 | 0.21 |
|   | rs3732356G/G | 9 | 0.00 | 0.00 | 0.00* |
|   | rs2472677C/C | 92 | 0.05 | 0.06 | 0.22 |
|   | rs2472677C/T | 128 | 0.02 | 0.11 | 0.23 |
|   | rs2472677T/T | 36 | 0.03 | 0.11 | 0.17 |
|   | rs6785049G/G | 251 | 0.03 | 0.08 | 0.22 |
|   | rs6785049G/A | 14 | 0.00 | 0.07 | 0.29 |
| NR1I3 |   |   |   |   |   |
|   | rs2307424C/C | 257 | 0.02 | 0.07 | 0.21 |
|   | rs2307424C/T | 25 | 0.12 | 0.20 | 0.32 |
|   | rs3003596T/T | 96 | 0.04 | 0.08 | 0.23 |
|   | rs3003596T/C | 132 | 0.03 | 0.10 | 0.20 |
|   | rs3003596C/C | 52 | 0.02 | 0.06 | 0.23 |
|   | rs2502815C/C | 169 | 0.02 | 0.09 | 0.22 |
|   | rs2502815C/T | 98 | 0.05 | 0.08 | 0.22 |
|   | rs2502815T/T | 15 | 0.00 | 0.00 | 0.20 |

Table 4 Caption: Frequency of HIV/AIDS patients changing ART regimens within 3, 6 or 12 months post-initiation of treatment

Variants associated with reduced plasma efavirenz levels could possibly act through increased transcription of PXR or CAR and therefore increased transcription of downstream DME genes such as *CYP2B6* and *CYP1A2* leading to increased efavirenz clearance through metabolism. Thus, the *NR1I3 rs2307424T* variant may result in the decrease in efavirenz levels to concentrations that are too low for effective viral suppresion, necessitating the change in treatment regimen when physicians note poor viral load decreases. On the other hand, variants associated with increased plasma efavirenz could be acting through decreased transcription of DMEs and are associated with treatment changes due to ADRs. As confirmed from the patient records, the major cause for treatment change in the *NR1I3 rs2307424C/T* genotype group was ineffective viral suppresion, while the major reason in the *NR1I3 rs2307424C/C* genotype group was adverse drug events. Although the amino acid change is synonymous (Pro180Pro), codon usage is known to affect the rate of transcription and to some extent protein folding [[22](#B22)].

### Allele frequencies and their distribution in different populations

The frequencies of the genetic variants detected in *NR1I2* and *NR1I3* among South African subjects were compared to allele frequencies in populations of Caucasian, Asian and Yoruba origin obtained from HapMap or dbSNP (Table [5](#T5)). Statistically significant differences were observed between the allele frequencies in the South African cohort and the Caucasian, Asian and Yoruba populations for eleven of the twelve SNPs in *NR1I2* and *NR1I3* (P<0.05). The Bantu-speaking South African population showed differences in the distribution of 8 SNPs when compared to the Yoruba population (Table [5](#T5)), which renders problematic the use of one African population to predict drug response in another.

### Table 5.

| SNP ID | Minor allele | South Africans this study (n=464) | YRI HapMap (n=226) | CEU HapMap (n=226) | HCB HapMap (n=86) | Bushman dbSNP (n=3) |
| --- | --- | --- | --- | --- | --- | --- |
| NR1I2 rs3732356T>G | G | 0.234 | 0.254 | 0.033** | N/A | 0.000** |
| NR1I2 rs2472677C>T | T | 0.351 | 0.356 | 0.636** | 0.544* | N/A |
| NR1I2 rs6785049G>A | A | 0.040 | 0.000** | 0.673** | 0.453** | N/A |
| NR1I3 rs2307424C>T | T | 0.047 | 0.119** | 0.336** | 0.512** | N/A |
| NR1I3 rs3003596T>C | C | 0.416 | 0.612** | 0.407 | 0.535 | 0.500 |
| NR1I3 rs2502815C>T | T | 0.230 | 0.406** | 0.248 | 0.442** | 0.500** |
|   |   | n=32# | n=226 | n=226 | n=86 |   |
| NR1I2 rs12721601T>C | C | 0.016 | 0.008 | N/A | N/A | N/A |
| NR1I2 rs59371185G>A | A | 0.016 | 0.000 | N/A | N/A | N/A |
| NR1I2 rs12721613C>T | T | 0.064 | 0.183* | 0.000* | 0.000* | N/A |
| NR1I2 rs1464603C>T | T | 0.016 | N/A | 0.292** | 0.325** | 0.000 |
| NR1I2 rs1464602C>T | T | 0.313 | 0.186* | 0.717** | 0.675** | 0.667** |
| NR1I2 rs80320762G>A | A | 0.048 | 0.144* | N/A | N/A | N/A |
| NR1I2 rs12721616C>T | T | 0.219 | 0.042** | N/A | N/A | N/A |
| NR1I2 rs112813596G>A | A | 0.016 | N/A | N/A | N/A | 0.000 |
| NR1I3 rs35205211C>G | G | 0.032 | N/A | N/A | N/A | 0.005 |
| NR1I3 rs34161743C>T | T | 0.016 | N/A | N/A | N/A | 0.005 |
| NR1I2 36726T>C# | C | 0.078 | N/A | N/A | N/A | N/A |
| NR1I2 36857G>A# | A | 0.016 | N/A | N/A | N/A | N/A |
| NR1I2 36905C>T# | T | 0.031 | N/A | N/A | N/A | N/A |

Table 5 Caption: Comparison of the allele frequencies between the South Africans and other populations

### Haplotype analysis

Haplotype frequencies were compared between the healthy subjects and the HIV/AIDS patients. The haplotype frequencies in *NR1I2* were significantly different between the healthy subjects and HIV/AIDS patients (P=0.015) (Table [6](#T6)). However, the difference was marginally significant where the haplotype analysis was corrected for multiple testing with significant P<0.017. By observation, the *NR1I2 T-G-G* haplotype (with respect to *rs2472677-rs3732356-rs6785049*), which occurs in about 3% of the HIV/AIDS patients, was associated with efavirenz levels greater than 4 μg/mL, and this may influence treatment regimen change (Figure [3](#F3)A).

### Table 6.

| Haplotypes | Healthy subjects n (freq) | HIV/AIDS patients n (freq) | Efavirenz levels (μg/mL)* | Global p-value# |
| --- | --- | --- | --- | --- |
| NR1I2 haplotypes [rs2472677-rs3732356-rs6785049] |  |  |  |  |
| C-T-A | 7 (0.025) | 6 (0.012) | 2.13 |   |
| C-T-G | 109 (0.367) | 190 (0.379) | 4.00 |   |
| C-G-G | 64 (0.216) | 114 (0.226) | 6.69 | 0.015 |
| T-T-A | 16 (0.052) | 6 (0.011) | 4.94 |   |
| T-T-G | 96 (0.321) | 172 (0.343) | 4.23 |   |
| T-G-G | 6 (0.019) | 13 (0.026) | 4.55 |   |
| NR1I3 haplotypes [rs2307424-rs2502815-rs3003596] |  |  |  |  |
| C-C-T | 153 (0.489) | 310 (0.530) | 4.97 |   |
| C-C-C | 58 (0.187) | 122 (0.208) | 3.45 |   |
| C-T-T | 7 (0.023) | 8 (0.013) | 3.53 | 0.424 |
| C-T-C | 75 (0.240) | 121 (0.207) | 3.87 |   |
| T-C-T | 14 (0.045) | 22 (0.037) | 4.42 |   |
| T-C-C | 4 (0.012) | 3 (0.005) | 1.89 |   |

Table 6 Caption: Comparison of the haplotype frequencies between the healthy subjects and the HIV/AIDS patients

### Figure 3.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/096f/3523080/aead64be4fd7/1471-2350-13-112-3.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3523080_1471-2350-13-112-3.jpg)

A-B: Effects of NR1I2 and NR1I3 haplotypes on plasma efavirenz concentration. The therapeutic range for efavirenz is shown. NR1I2 C-T-T and NR1I3 T-C-C haplotypes were only observed once and thus excluded from the analysis.

## Discussion

Many factors, including environmental and genetic factors, determine how individual patients respond to therapeutic drugs. Pharmacogenetics is concerned with understanding individual genetic variability and how it affects response to treatment. Most of the early work has focused on DME genes because of their direct involvement in conversion of drugs and their removal from the body. However, in order to gain a better understanding, variation in the NRs that affect the regulation of DMEs was investigated. This research provides a window into how the variation in the NRs, PXR and CAR, can indirectly affect plasma drug levels and ultimate response to treatment. The report also documents the frequencies of variants for these two genes in the South African population and adds to the growing literature on the genetic characterization of African populations [[23](#B23),[24](#B24)]. To our knowledge, this is the first report on baseline frequencies for the six SNPs in *NR1I2* and *NR1I3* in a Bantu-speaking South African population.

### Gene variant frequencies in the South African population and comparison to other world populations

The genotype frequencies between the healthy subjects and HIV/AIDS patients differed significantly for the *NR1I2 rs6785049G>A* and *NR1I2 rs3732356T>G* SNPs (P=0.002 and 0.031, respectively). The difference in genotype frequencies for the *NR1I2 rs3732356T>G* SNP between the healthy subjects and HIV/AIDS patients may be explained by the deviation from HWE in the HIV/AIDS patients (P=0.020). Among the HIV/AIDS patients, the *NR1I2 rs6785049A/A* genotype was not observed and the *rs6785049A* variant occurred at a frequency of 2.5% compared to the 7% among healthy subjects. On the other hand, the *NR1I2 rs3732356G* allele frequency was higher among HIV/AIDS patients (48%) compared to healthy subjects (22%). The above findings lead us to postulate that PXR may have a role, not only in the metabolism of drugs, but also indirectly in signalling pathways active during acute stages of infection. Gu *et al.,*[[25](#B25)] showed that NF-kB directly interacts with the DBD of RXR; this in turn prevents RXR binding to its consensus DNA sequences and PXR.

*NR1I2 rs2472677C>T, NR1I2 rs6785049G>A* and *NR1I3 rs2307424C>T* SNPs displayed significant differences in allele frequencies between the South African population and the Caucasian and Asian populations. As a result, therapeutic drugs such as efavirenz that are ligands for NRs, may result in different drug effects in different populations. Similarly, African populations cannot be regarded as homogeneous due to the genetic diversity existing between the sub-populations. For example, three SNPs in *NR1I3* and *NR1I2 rs6785049* showed a significant difference in allele frequencies between the South African population and the Yoruba population from Ibadan, Nigeria.

The SNPs in *NR1I2, rs3732356T>G, rs2472677C>T* and *rs6785049G>A*, as well as in *NR1I3, rs2307424C>T*, *rs3003596T>C* and *rs2502815C>T*, displayed no evidence of LD, which is in contrast to the strong LD for *NR1I3 rs2502815C>T* and *NR1I3 rs2307424C>T* reported in a Caucasian population (D’=1.00; P= 0.01) [[26](#B26)]. This finding is consistent with the weak LD reported in African populations, due to the large degree of genetic diversity in African populations [[27](#B27)].

### Comment on drug discovery and relevance of this knowledge

Sequencing of the *NR1I2* and *NR1I3* DBDs identified several previously characterised SNPs as well as three novel variants in the *NR1I2* DBD. The allele frequency (0.016) of the *NR1I2 52A* allele (*rs59371185*) observed in the 32 sequenced South African individuals was similar to allele frequencies reported in Africans from the Ivory Coast and sub-Saharan Africa [[9](#B9),[28](#B28)]. The *NR1I2 70C>T* SNP was observed at a frequency of 0.063 in the sequenced samples, but has however, been reported at frequencies of 0.126 and 0.002 in sub-Saharan Africans and Caucasians, respectively [[9](#B9),[28](#B28)]. *NR1I2 36726T>C* is predicted to be of little significance, since it is present in an intron and is not situated within a GT-AG splice site. Although *NR1I2 36857G>A* and *NR1I2 36905C>T* in exon 2 are both synonymous amino acid substitutions, they may be associated with differential PXR expression due to codon usage [[22](#B22)]. The *NR1I2 36905T* variant was also predicted to affect the structure of PXR. Further analysis (with ESE finder v3.0) revealed that *NR1I2 36857A* affects the binding affinity of the SRp40 splicing protein, which regulates pre-ribosome assembly and transport. Destabilisation of the DBD of NRs is likely to influence the binding of these NRs to promoter regions of target DMEs and possibly alter transcription and expression. Alternatively spliced *NR1I2* mRNA isoforms can differ in their patterns of expression, biological function (either gain- or loss-of-function), activation of target genes like DMEs, DNA binding and tissue-specific expression [[29](#B29)], which may contribute to inter-individual variability in *NR1I2* expression and ultimately efavirenz metabolism.

### Implications for disease or drug treatment and possible development of diagnostic tools

A significantly low average plasma efavirenz concentration was observed among patients with the *NR1I3 rs3003596C/C* and *T/C* genotypes compared to patients with the *rs3003596T/T* genotype (P=0.015 and P=0.010, respectively), using a dominant genetic model. The low efavirenz concentrations may point to possible functional effects of the change on CAR, expression or activity, and regulation of multiple target genes encoding DMEs. Efavirenz induces CYP2B6 activity primarily through CAR and genetic variation in *NR1I3* may therefore significantly influence efavirenz plasma concentrations via the induction of CYP2B6. Due to the availability of resources in Western and Asian countries, these populations have been well studied in comparison to the African populations [[30](#B30),[31](#B31)]. Thus, most drug-discovery and development is premised on Caucasian and Asian populations, and as a result, new drugs may make it to the market without having been exposed to most of the genetic variability within African populations. The effect of this variability only becomes obvious when adverse drug events are noticed [[30](#B30)]. For example, the ART guidelines initially established by the World Health Organization, on which the South African guidelines are modelled, was based on clinical data obtained primarily from Caucasian and Asian individuals. Understanding the consequence of genetic variation in *NR1I2* and *NR1I3* adds to the pharmacogenetics knowledge and improves the move towards personalised medication. This data, taken together with variation in DMEs that metabolise efavirenz (e.g. CYP2B6 and CYP2A6), may aid in the designing of appropriate genotyping assays that could prove useful in individualized efavirenz dosing regimens.

## Conclusion

The finding of novel variants in *NR1I2* after sequencing a very small portion of the DBD is further testimony that there is still “missing” genetic heritability that can be discovered when sequencing as many African genomes as possible. In addition, our data highlights the role (though indirect) that variation in NR genes can play in drug treatment response.

## Competing interests

All authors declare that they have no competing interest.

## Authors’ contributions

MS carried out 40% the molecular genetic studies and drafted the manuscript. HW carried out 60% of the molecular genetic studies. YR and PS both carried out the LC/MS/MS analysis of plasma efavirenz concentration. RR contributed towards reagents and equipment. CD conceived of the study, designed, coordinated the study, assisted with statistical data analysis and helped to draft the manuscript and approved the final version. All authors read and approved the final manuscript.

## Pre-publication history

The pre-publication history for this paper can be accessed here:

[http://www.biomedcentral.com/1471-2350/13/112/prepub](http://www.biomedcentral.com/1471-2350/13/112/prepub)

## Supplementary Material

## Contributor Information

Marelize Swart, Email: marelize.swart@uct.ac.za.

Heather Whitehorn, Email: heather.whitehorn@uct.ac.za.

Yuan Ren, Email: yuan.ren@uct.ac.za.

Peter Smith, Email: peter.smith@uct.ac.za.

Rajkumar S Ramesar, Email: raj.ramesar@uct.ac.za.

Collet Dandara, Email: collet.dandara@uct.ac.za.

## Acknowledgements

We thank the South African Medical Research Council (MRC), the National Research Foundation of South Africa (NRF) and the University of Cape Town for grants supporting CD’s research programme; the Poliomyelitis Research Foundation (PRF) for funding. We also thank Professor Patrick McPhail for facilitating sample collection from the Clinical HIV Research Unit at Themba Lethu Clinic, Helen Joseph Hospital, Johannesburg, South Africa.

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Wang H, LeCluyse EL. Role of orphan nuclear receptors in the regulation of drug-metabolising enzymes. Clin Pharmacokinet. 2003;42(15):1331–1357. doi: 10.2165/00003088-200342150-00003.  [DOI](https://doi.org/10.2165/00003088-200342150-00003) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14674787/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacokinet&title=Role%20of%20orphan%20nuclear%20receptors%20in%20the%20regulation%20of%20drug-metabolising%20enzymes&author=H%20Wang&author=EL%20LeCluyse&volume=42&issue=15&publication_year=2003&pages=1331-1357&pmid=14674787&doi=10.2165/00003088-200342150-00003&)

2. Chen Y, Tang Y, Guo C, Wang J, Boral D, Nie D. Nuclear receptors in the multidrug esistance through the regulation of drug-metabolizing enzymes and drug transporters. Biochem Pharmacol. 2012;56:2153–2157. doi: 10.1016/j.bcp.2012.01.030.  [DOI](https://doi.org/10.1016/j.bcp.2012.01.030) | [PMC free article](/articles/PMC3339266/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22326308/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem%20Pharmacol&title=Nuclear%20receptors%20in%20the%20multidrug%20esistance%20through%20the%20regulation%20of%20drug-metabolizing%20enzymes%20and%20drug%20transporters&author=Y%20Chen&author=Y%20Tang&author=C%20Guo&author=J%20Wang&author=D%20Boral&volume=56&publication_year=2012&pages=2153-2157&pmid=22326308&doi=10.1016/j.bcp.2012.01.030&)

3. Piedade R, Schaeffeler E, Winter S, Asimus S, Schwab M, Ashton M, Burk O, Gil JP. PXR variants and artemisinin use in Vietnamese subjects: Frequency distribution and impact on the inter-individual variability of CYP3A induction by artemisinin. Antimicrob Agents Chemother. 2012;83:1112–1126. doi: 10.1128/AAC.06009-11.  [DOI](https://doi.org/10.1128/AAC.06009-11) | [PMC free article](/articles/PMC3318324/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22252826/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Antimicrob%20Agents%20Chemother&title=PXR%20variants%20and%20artemisinin%20use%20in%20Vietnamese%20subjects:%20Frequency%20distribution%20and%20impact%20on%20the%20inter-individual%20variability%20of%20CYP3A%20induction%20by%20artemisinin&author=R%20Piedade&author=E%20Schaeffeler&author=S%20Winter&author=S%20Asimus&author=M%20Schwab&volume=83&publication_year=2012&pages=1112-1126&pmid=22252826&doi=10.1128/AAC.06009-11&)

4. Handschin C, Meyer UA. Induction of drug metabolism: the role of nuclear receptors. Pharmacol Rev. 2003;55(4):649–673. doi: 10.1124/pr.55.4.2.  [DOI](https://doi.org/10.1124/pr.55.4.2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14657421/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacol%20Rev&title=Induction%20of%20drug%20metabolism:%20the%20role%20of%20nuclear%20receptors&author=C%20Handschin&author=UA%20Meyer&volume=55&issue=4&publication_year=2003&pages=649-673&pmid=14657421&doi=10.1124/pr.55.4.2&)

5. Zhang J, Kuehl P, Green ED, Touchman JW, Watkins PB, Daly A, Hall SD, Maurel P, Relling M, Brimer C. et al. The human pregnane X receptor: genomic structure and identification and functional characterization of natural allelic variants. Pharmacogenetics. 2001;11(7):555–572. doi: 10.1097/00008571-200110000-00003.  [DOI](https://doi.org/10.1097/00008571-200110000-00003) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11668216/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=The%20human%20pregnane%20X%20receptor:%20genomic%20structure%20and%20identification%20and%20functional%20characterization%20of%20natural%20allelic%20variants&author=J%20Zhang&author=P%20Kuehl&author=ED%20Green&author=JW%20Touchman&author=PB%20Watkins&volume=11&issue=7&publication_year=2001&pages=555-572&pmid=11668216&doi=10.1097/00008571-200110000-00003&)

6. Healan-Greenberg C, Waring JF, Kempf DJ, Blomme EA, Tirona RG, Kim RB. A human immunodeficiency virus protease inhibitor is a novel functional inhibitor of human pregnane X receptor. Drug Metab Dispos. 2008;36(3):500–507. doi: 10.1124/dmd.107.019547.  [DOI](https://doi.org/10.1124/dmd.107.019547) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18096673/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=A%20human%20immunodeficiency%20virus%20protease%20inhibitor%20is%20a%20novel%20functional%20inhibitor%20of%20human%20pregnane%20X%20receptor&author=C%20Healan-Greenberg&author=JF%20Waring&author=DJ%20Kempf&author=EA%20Blomme&author=RG%20Tirona&volume=36&issue=3&publication_year=2008&pages=500-507&pmid=18096673&doi=10.1124/dmd.107.019547&)

7. Lehmann JM, McKee DD, Watson MA, Willson TM, Moore JT, Kliewer SA. The human orphan nuclear receptor PXR is activated by compounds that regulate CYP3A4 gene expression and cause drug interactions. J Clin Invest. 1998;102(5):1016–1023. doi: 10.1172/JCI3703.  [DOI](https://doi.org/10.1172/JCI3703) | [PMC free article](/articles/PMC508967/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9727070/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Invest&title=The%20human%20orphan%20nuclear%20receptor%20PXR%20is%20activated%20by%20compounds%20that%20regulate%20CYP3A4%20gene%20expression%20and%20cause%20drug%20interactions&author=JM%20Lehmann&author=DD%20McKee&author=MA%20Watson&author=TM%20Willson&author=JT%20Moore&volume=102&issue=5&publication_year=1998&pages=1016-1023&pmid=9727070&doi=10.1172/JCI3703&)

8. Bertilsson G, Heidrich J, Svensson K, Asman M, Jendeberg L, Sydow-Backman M, Ohlsson R, Postlind H, Blomquist P, Berkenstam A. Identification of a human nuclear receptor defines a new signaling pathway for CYP3A induction. Proc Natl Acad Sci U S A. 1998;95(21):12208–12213. doi: 10.1073/pnas.95.21.12208.  [DOI](https://doi.org/10.1073/pnas.95.21.12208) | [PMC free article](/articles/PMC22810/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9770465/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&title=Identification%20of%20a%20human%20nuclear%20receptor%20defines%20a%20new%20signaling%20pathway%20for%20CYP3A%20induction&author=G%20Bertilsson&author=J%20Heidrich&author=K%20Svensson&author=M%20Asman&author=L%20Jendeberg&volume=95&issue=21&publication_year=1998&pages=12208-12213&pmid=9770465&doi=10.1073/pnas.95.21.12208&)

9. Hustert E, Zibat A, Presecan-Siedel E, Eiselt R, Mueller R, Fuss C, Brehm I, Brinkmann U, Eichelbaum M, Wojnowski L. et al. Natural protein variants of pregnane X receptor with altered transactivation activity toward CYP3A4. Drug Metab Dispos. 2001;29(11):1454–1459.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/11602521/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Natural%20protein%20variants%20of%20pregnane%20X%20receptor%20with%20altered%20transactivation%20activity%20toward%20CYP3A4&author=E%20Hustert&author=A%20Zibat&author=E%20Presecan-Siedel&author=R%20Eiselt&author=R%20Mueller&volume=29&issue=11&publication_year=2001&pages=1454-1459&pmid=11602521&)

10. Auerbach SS, Ramsden R, Stoner MA, Verlinde C, Hassett C, Omiecinski CJ. Alternatively spliced isoforms of the human constitutive androstane receptor. Nucleic Acids Res. 2003;31(12):3194–3207. doi: 10.1093/nar/gkg419.  [DOI](https://doi.org/10.1093/nar/gkg419) | [PMC free article](/articles/PMC162252/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12799447/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=Alternatively%20spliced%20isoforms%20of%20the%20human%20constitutive%20androstane%20receptor&author=SS%20Auerbach&author=R%20Ramsden&author=MA%20Stoner&author=C%20Verlinde&author=C%20Hassett&volume=31&issue=12&publication_year=2003&pages=3194-3207&pmid=12799447&doi=10.1093/nar/gkg419&)

11. Lamba JK, Lamba V, Yasuda K, Lin YS, Assem M, Thompson E, Strom S, Schuetz E. Expression of constitutive androstane receptor splice variants in human tissues and their functional consequences. J Pharmacol Exp Ther. 2004;311(2):811–821. doi: 10.1124/jpet.104.069310.  [DOI](https://doi.org/10.1124/jpet.104.069310) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15194709/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharmacol%20Exp%20Ther&title=Expression%20of%20constitutive%20androstane%20receptor%20splice%20variants%20in%20human%20tissues%20and%20their%20functional%20consequences&author=JK%20Lamba&author=V%20Lamba&author=K%20Yasuda&author=YS%20Lin&author=M%20Assem&volume=311&issue=2&publication_year=2004&pages=811-821&pmid=15194709&doi=10.1124/jpet.104.069310&)

12. Pascussi JM, Busson-Le Coniat M, Maurel P, Vilarem MJ. Transcriptional analysis of the orphan nuclear receptor constitutive androstane receptor (NR1I3) gene promoter: identification of a distal glucocorticoid response element. Mol Endocrinol. 2003;17(1):42–55. doi: 10.1210/me.2002-0244.  [DOI](https://doi.org/10.1210/me.2002-0244) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12511605/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Endocrinol&title=Transcriptional%20analysis%20of%20the%20orphan%20nuclear%20receptor%20constitutive%20androstane%20receptor%20(NR1I3)%20gene%20promoter:%20identification%20of%20a%20distal%20glucocorticoid%20response%20element&author=JM%20Pascussi&author=M%20Busson-Le%20Coniat&author=P%20Maurel&author=MJ%20Vilarem&volume=17&issue=1&publication_year=2003&pages=42-55&pmid=12511605&doi=10.1210/me.2002-0244&)

13. Wyen C, Hendra H, Siccardi M, Platten M, Jaeger H, Harrer T, Esser S, Bogner JR, Brockmeyer NH, Bieniek B. et al. Cytochrome P450 2B6 (CYP2B6) and constitutive androstane receptor (CAR) polymorphisms are associated with early discontinuation of efavirenz-containing regimens. J Antimicrob Chemother. 2011;66(9):2092–2098. doi: 10.1093/jac/dkr272.  [DOI](https://doi.org/10.1093/jac/dkr272) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21715435/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Antimicrob%20Chemother&title=Cytochrome%20P450%202B6%20(CYP2B6)%20and%20constitutive%20androstane%20receptor%20(CAR)%20polymorphisms%20are%20associated%20with%20early%20discontinuation%20of%20efavirenz-containing%20regimens&author=C%20Wyen&author=H%20Hendra&author=M%20Siccardi&author=M%20Platten&author=H%20Jaeger&volume=66&issue=9&publication_year=2011&pages=2092-2098&pmid=21715435&doi=10.1093/jac/dkr272&)

14. Gustafson S, Proper JA, Bowie EJ, Sommer SS. Parameters affecting the yield of DNA from human blood. Anal Biochem. 1987;165(2):294–299. doi: 10.1016/0003-2697(87)90272-7.  [DOI](https://doi.org/10.1016/0003-2697(87)90272-7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/3425899/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Anal%20Biochem&title=Parameters%20affecting%20the%20yield%20of%20DNA%20from%20human%20blood&author=S%20Gustafson&author=JA%20Proper&author=EJ%20Bowie&author=SS%20Sommer&volume=165&issue=2&publication_year=1987&pages=294-299&pmid=3425899&doi=10.1016/0003-2697(87)90272-7&)

15. Chi J, Jayewardene AL, Stone JA, Motoya T, Aweeka FT. Simultaneous determination of five HIV protease inhibitors nelfinavir, indinavir, ritonavir, saquinavir and amprenavir in human plasma by LC/MS/MS. J Pharm Biomed Anal. 2002;30(3):675–684. doi: 10.1016/S0731-7085(02)00357-6.  [DOI](https://doi.org/10.1016/S0731-7085(02)00357-6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12367693/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharm%20Biomed%20Anal&title=Simultaneous%20determination%20of%20five%20HIV%20protease%20inhibitors%20nelfinavir,%20indinavir,%20ritonavir,%20saquinavir%20and%20amprenavir%20in%20human%20plasma%20by%20LC/MS/MS&author=J%20Chi&author=AL%20Jayewardene&author=JA%20Stone&author=T%20Motoya&author=FT%20Aweeka&volume=30&issue=3&publication_year=2002&pages=675-684&pmid=12367693&doi=10.1016/S0731-7085(02)00357-6&)

16. Liu CK, Chen YH, Tang CY, Chang SC, Lin YJ, Tsai MF, Chen YT, Yao A. Functional analysis of novel SNPs and mutations in human and mouse genomes. BMC Bioinformatics. 2008;9(Suppl 12):S10. doi: 10.1186/1471-2105-9-S12-S10.  [DOI](https://doi.org/10.1186/1471-2105-9-S12-S10) | [PMC free article](/articles/PMC2638150/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19091009/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Bioinformatics&title=Functional%20analysis%20of%20novel%20SNPs%20and%20mutations%20in%20human%20and%20mouse%20genomes&author=CK%20Liu&author=YH%20Chen&author=CY%20Tang&author=SC%20Chang&author=YJ%20Lin&volume=9&issue=Suppl%2012&publication_year=2008&pages=S10&pmid=19091009&doi=10.1186/1471-2105-9-S12-S10&)

17. Stephens M, Donnelly P. A comparison of bayesian methods for haplotype reconstruction from population genotype data. Am J Hum Genet. 2003;73(5):1162–1169. doi: 10.1086/379378.  [DOI](https://doi.org/10.1086/379378) | [PMC free article](/articles/PMC1180495/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14574645/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=A%20comparison%20of%20bayesian%20methods%20for%20haplotype%20reconstruction%20from%20population%20genotype%20data&author=M%20Stephens&author=P%20Donnelly&volume=73&issue=5&publication_year=2003&pages=1162-1169&pmid=14574645&doi=10.1086/379378&)

18. Stephens M, Scheet P. Accounting for decay of linkage disequilibrium in haplotype inference and missing-data imputation. Am J Hum Genet. 2005;76(3):449–462. doi: 10.1086/428594.  [DOI](https://doi.org/10.1086/428594) | [PMC free article](/articles/PMC1196397/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15700229/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=Accounting%20for%20decay%20of%20linkage%20disequilibrium%20in%20haplotype%20inference%20and%20missing-data%20imputation&author=M%20Stephens&author=P%20Scheet&volume=76&issue=3&publication_year=2005&pages=449-462&pmid=15700229&doi=10.1086/428594&)

19. Stephens M, Smith NJ, Donnelly P. A new statistical method for haplotype reconstruction from population data. Am J Hum Genet. 2001;68(4):978–989. doi: 10.1086/319501.  [DOI](https://doi.org/10.1086/319501) | [PMC free article](/articles/PMC1275651/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11254454/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=A%20new%20statistical%20method%20for%20haplotype%20reconstruction%20from%20population%20data&author=M%20Stephens&author=NJ%20Smith&author=P%20Donnelly&volume=68&issue=4&publication_year=2001&pages=978-989&pmid=11254454&doi=10.1086/319501&)

20. Shi YY, He L. SHEsis, a powerful software platform for analyses of linkage disequilibrium, haplotype construction, and genetic association at polymorphism loci. Cell Res. 2005;15(2):97–98. doi: 10.1038/sj.cr.7290272.  [DOI](https://doi.org/10.1038/sj.cr.7290272) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15740637/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell%20Res&title=SHEsis,%20a%20powerful%20software%20platform%20for%20analyses%20of%20linkage%20disequilibrium,%20haplotype%20construction,%20and%20genetic%20association%20at%20polymorphism%20loci&author=YY%20Shi&author=L%20He&volume=15&issue=2&publication_year=2005&pages=97-98&pmid=15740637&doi=10.1038/sj.cr.7290272&)

21. Li Z, Zhang Z, He Z, Tang W, Li T, Zeng Z, He L, Shi Y. A partition-ligation-combination-subdivision EM algorithm for haplotype inference with multiallelic markers: update of the SHEsis (http://analysis.bio-x.cn) Cell Res. 2009;19(4):519–523. doi: 10.1038/cr.2009.33.  [DOI](https://doi.org/10.1038/cr.2009.33) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19290020/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell%20Res&title=A%20partition-ligation-combination-subdivision%20EM%20algorithm%20for%20haplotype%20inference%20with%20multiallelic%20markers:%20update%20of%20the%20SHEsis%20(http://analysis.bio-x.cn)&author=Z%20Li&author=Z%20Zhang&author=Z%20He&author=W%20Tang&author=T%20Li&volume=19&issue=4&publication_year=2009&pages=519-523&pmid=19290020&doi=10.1038/cr.2009.33&)

22. Gu W, Zhou T, Ma J, Sun X, Lu Z. The relationship between synonymous codon usage and protein structure in Escherichia coli and Homo sapiens. Biosystems. 2004;73(2):89–97. doi: 10.1016/j.biosystems.2003.10.001.  [DOI](https://doi.org/10.1016/j.biosystems.2003.10.001) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15013221/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biosystems&title=The%20relationship%20between%20synonymous%20codon%20usage%20and%20protein%20structure%20in%20Escherichia%20coli%20and%20Homo%20sapiens&author=W%20Gu&author=T%20Zhou&author=J%20Ma&author=X%20Sun&author=Z%20Lu&volume=73&issue=2&publication_year=2004&pages=89-97&pmid=15013221&doi=10.1016/j.biosystems.2003.10.001&)

23. Dandara C, Lombard Z, Du Plooy I, McLellan T, Norris SA, Ramsay M. Genetic variants in CYP (−1A2, -2C9, -2C19, -3A4 and -3A5), VKORC1 and ABCB1 genes in a black South African population: a window into diversity. Pharmacogenomics. 2011;12(12):1663–1670. doi: 10.2217/pgs.11.106.  [DOI](https://doi.org/10.2217/pgs.11.106) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22118051/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Genetic%20variants%20in%20CYP%20(%E2%88%921A2,%20-2C9,%20-2C19,%20-3A4%20and%20-3A5),%20VKORC1%20and%20ABCB1%20genes%20in%20a%20black%20South%20African%20population:%20a%20window%20into%20diversity&author=C%20Dandara&author=Z%20Lombard&author=I%20Du%20Plooy&author=T%20McLellan&author=SA%20Norris&volume=12&issue=12&publication_year=2011&pages=1663-1670&pmid=22118051&doi=10.2217/pgs.11.106&)

24. Ikediobi O, Aouizerat B, Xiao Y, Gandhi M, Gebhardt S, Warnich L. Analysis of pharmacogenetic traits in two distinct South African populations. Hum Genomics. 2011;5(4):265–282. doi: 10.1186/1479-7364-5-4-265.  [DOI](https://doi.org/10.1186/1479-7364-5-4-265) | [PMC free article](/articles/PMC3525241/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21712189/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Genomics&title=Analysis%20of%20pharmacogenetic%20traits%20in%20two%20distinct%20South%20African%20populations&author=O%20Ikediobi&author=B%20Aouizerat&author=Y%20Xiao&author=M%20Gandhi&author=S%20Gebhardt&volume=5&issue=4&publication_year=2011&pages=265-282&pmid=21712189&doi=10.1186/1479-7364-5-4-265&)

25. Gu X, Ke S, Liu D, Sheng T, Thomas PE, Rabson AB, Gallo MA, Xie W, Tian Y. Role of NF-kappaB in regulation of PXR-mediated gene expression: a mechanism for the suppression of cytochrome P-450 3A4 by proinflammatory agents. J Biol Chem. 2006;281(26):17882–17889. doi: 10.1074/jbc.M601302200.  [DOI](https://doi.org/10.1074/jbc.M601302200) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16608838/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=Role%20of%20NF-kappaB%20in%20regulation%20of%20PXR-mediated%20gene%20expression:%20a%20mechanism%20for%20the%20suppression%20of%20cytochrome%20P-450%203A4%20by%20proinflammatory%20agents&author=X%20Gu&author=S%20Ke&author=D%20Liu&author=T%20Sheng&author=PE%20Thomas&volume=281&issue=26&publication_year=2006&pages=17882-17889&pmid=16608838&doi=10.1074/jbc.M601302200&)

26. Oliver P, Lubomirov R, Carcas A. Genetic polymorphisms of CYP1A2, CYP3A4, CYP3A5, pregnane/steroid X receptor and constitutive androstane receptor in 207 healthy Spanish volunteers. Clin Chem Lab Med. 2010;48(5):635–639. doi: 10.1515/CCLM.2010.130.  [DOI](https://doi.org/10.1515/CCLM.2010.130) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20218903/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Chem%20Lab%20Med&title=Genetic%20polymorphisms%20of%20CYP1A2,%20CYP3A4,%20CYP3A5,%20pregnane/steroid%20X%20receptor%20and%20constitutive%20androstane%20receptor%20in%20207%20healthy%20Spanish%20volunteers&author=P%20Oliver&author=R%20Lubomirov&author=A%20Carcas&volume=48&issue=5&publication_year=2010&pages=635-639&pmid=20218903&doi=10.1515/CCLM.2010.130&)

27. Gabriel SB, Schaffner SF, Nguyen H, Moore JM, Roy J, Blumenstiel B, Higgins J, DeFelice M, Lochner A, Faggart M. et al. The structure of haplotype blocks in the human genome. Science. 2002;296(5576):2225–2229. doi: 10.1126/science.1069424.  [DOI](https://doi.org/10.1126/science.1069424) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12029063/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=The%20structure%20of%20haplotype%20blocks%20in%20the%20human%20genome&author=SB%20Gabriel&author=SF%20Schaffner&author=H%20Nguyen&author=JM%20Moore&author=J%20Roy&volume=296&issue=5576&publication_year=2002&pages=2225-2229&pmid=12029063&doi=10.1126/science.1069424&)

28. Svard J, Spiers JP, Mulcahy F, Hennessy M. Nuclear receptor-mediated induction of CYP450 by antiretrovirals: functional consequences of NR1I2 (PXR) polymorphisms and differential prevalence in whites and sub-Saharan Africans. J Acquir Immune Defic Syndr. 2010;55(5):536–549. doi: 10.1097/QAI.0b013e3181f52f0c.  [DOI](https://doi.org/10.1097/QAI.0b013e3181f52f0c) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20861742/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Acquir%20Immune%20Defic%20Syndr&title=Nuclear%20receptor-mediated%20induction%20of%20CYP450%20by%20antiretrovirals:%20functional%20consequences%20of%20NR1I2%20(PXR)%20polymorphisms%20and%20differential%20prevalence%20in%20whites%20and%20sub-Saharan%20Africans&author=J%20Svard&author=JP%20Spiers&author=F%20Mulcahy&author=M%20Hennessy&volume=55&issue=5&publication_year=2010&pages=536-549&pmid=20861742&doi=10.1097/QAI.0b013e3181f52f0c&)

29. Keightley MC. Steroid receptor isoforms: exception or rule? Mol Cell Endocrinol. 1998;137(1):1–5. doi: 10.1016/S0303-7207(97)00236-0.  [DOI](https://doi.org/10.1016/S0303-7207(97)00236-0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9607722/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Cell%20Endocrinol&title=Steroid%20receptor%20isoforms:%20exception%20or%20rule?&author=MC%20Keightley&volume=137&issue=1&publication_year=1998&pages=1-5&pmid=9607722&doi=10.1016/S0303-7207(97)00236-0&)

30. Tishkoff SA, Williams SM. Genetic analysis of African populations: human evolution and complex disease. Nat Rev Genet. 2002;3(8):611–621. doi: 10.1038/nrg865.  [DOI](https://doi.org/10.1038/nrg865) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12154384/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Genet&title=Genetic%20analysis%20of%20African%20populations:%20human%20evolution%20and%20complex%20disease&author=SA%20Tishkoff&author=SM%20Williams&volume=3&issue=8&publication_year=2002&pages=611-621&pmid=12154384&doi=10.1038/nrg865&)

31. Lonjou C, Collins A, Morton NE. Allelic association between marker loci. Proc Natl Acad Sci U S A. 1999;96(4):1621–1626. doi: 10.1073/pnas.96.4.1621.  [DOI](https://doi.org/10.1073/pnas.96.4.1621) | [PMC free article](/articles/PMC15538/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9990074/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&title=Allelic%20association%20between%20marker%20loci&author=C%20Lonjou&author=A%20Collins&author=NE%20Morton&volume=96&issue=4&publication_year=1999&pages=1621-1626&pmid=9990074&doi=10.1073/pnas.96.4.1621&)
