# Genome-wide association study identifies germline polymorphisms associated with relapse of childhood acute lymphoblastic leukemia

## Metadata
**Authors:** Jun J Yang, Cheng Cheng, Meenakshi Devidas, Xueyuan Cao, Dario Campana, Wenjian Yang, Yiping Fan, Geoff Neale, Nancy Cox, Paul Scheet, Michael J Borowitz, Naomi J Winick, Paul L Martin, W Paul Bowman, Bruce Camitta, Gregory H Reaman, William L Carroll, Cheryl L Willman, Stephen P Hunger, William E Evans, Ching-Hon Pui, Mignon Loh, Mary V Relling
**Journal:** Blood
**Date:** 2012 Sep 24
**DOI:** [10.1182/blood-2012-07-440107](https://doi.org/10.1182/blood-2012-07-440107)
**PMID:** 23007406
**PMCID:** PMC3501717
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3501717/

## Abstract

With the use of risk-directed therapy for childhood acute lymphoblastic leukemia (ALL), outcome has improved dramatically in the past 40 years. However, a substantial portion of patients, many of whom have no known risk factors, experience relapse. Taking a genome-wide approach, in the present study, we evaluated the relationships between genotypes at 444 044 single nucleotide polymorphisms (SNPs) with the risk of relapse in 2535 children with newly diagnosed ALL after adjusting for genetic ancestry and treatment regimen. We identified 134 SNPs that were reproducibly associated with ALL relapse. Of 134 relapse SNPs, 133 remained prognostic after adjusting for all known relapse risk factors, including minimal residual disease, and 111 were significant even among patients who were negative for minimal residual disease after remission induction therapy. The C allele at rs7142143 in the PYGL gene was associated with 3.6-fold higher risk of relapse than the T allele (P = 6.7 × 10−9). Fourteen of the 134 relapse SNPs, including variants in PDE4B and ABCB1, were also associated with antileukemic drug pharmacokinetics and/or pharmacodynamics. In the present study, we systematically identified host genetic variations related to treatment outcome of childhood ALL, most of which were prognostic independent of known risk factors for relapse, and some of which also influenced outcome by affecting host dis-position of antileukemic drugs. All trials are registered at www.clinicaltrials.gov or www.cancer.gov (COG P9904: NCT00005585; COG P9905: NCT00005596; COG P9906: NCT00005603; St Jude Total XIIIB: NCI-T93-0101D; and St Jude Total XV: NCT00137111).

## Introduction

In childhood acute lymphoblastic leukemia (ALL), risk-adapted combination therapy has led to dramatic improvements in outcome, with 5-year survival rates of more than 85% in most industrialized countries.^1^ However, ALL relapse still carries a very poor prognosis and the majority of relapses continue to occur in patients without apparent “high-risk” features.^2–4^ Therefore, more accurate risk classification of newly diagnosed disease is needed to reduce relapses and improve the overall outcome of ALL.

Prior efforts to improve risk stratification primarily focused on genetic variations of the tumor or on assessment of early antileukemic response; for example, the upfront prednisone response^5^ and minimal residual disease (MRD) after remission induction.^6^ The role of inherited genetic variations on treatment response in children with ALL has received increasing attention.^7^ For example, candidate gene studies have suggested that genetic polymorphisms in metabolizing enzymes (*GSTM1*) and transporters (*ABCC4*) of antileukemic agents affect treatment outcome.^8,9^ Recent studies have also identified associations between sequence variations in cytokine and chemokine genes and early treatment response,^7,10^ indicating tumor-microenvironment interaction as one plausible mechanism by which host genetic variations affect ALL outcome.^11^

Prognostic features of ALL are strongly dependent on treatment, and the lack of “replication” of pharmacogenetic findings in chemotherapy trials (and particularly outside of the context of clinical trials) may be attributed to modest differences in therapy among trials. To maximize the opportunity of identifying genotype-phenotype associations generalizable to diverse treatment regimens, in the present study, we applied an iterative “discovery versus replication” genomic screen to a core panel of frontline ALL trials in the Children's Oncology Group (COG) and at St Jude Children's Research Hospital. Our goal was to agnostically test single nucleotide polymorphisms (SNPs) throughout the genome for their association with the risk of relapse in a large group of children with ALL (N = 2535) from diverse racial/ethnic backgrounds.

## Methods

### Patients and treatment

Germline DNA samples were collected at remission in children with newly diagnosed ALL treated on St Jude Children's Research Hospital Total Therapy XIIIB^12^ and XV^13^ or the COG P9906^14^ or COG P9904/P9905 studies^14^ ([Table 1](#T1)). The studies were approved by the institutional review boards and informed consent was obtained from the parents, the guardians, or the patients in accordance with the Declaration of Helsinki. ALL treatment regimens were as described for the St Jude^12,13^ and COG^14^ protocols.

### Table 1.

|   | N | 5-y incidence of relapse in % (SD)¶ | P value for association with ALL relapse |
| --- | --- | --- | --- |
| Treatment protocol |   |   | 1.35 × 10−14# |
| COG P9904/9905 | 1606 | 14.4 (0.89) |   |
| COG P9906 | 222 | 33.5 (3.22) |   |
| St Jude Total Therapy XIIIB | 222 | 12.2 (2.21) |   |
| St Jude Total Therapy XV | 485 | 8.97 (1.35) |   |
| Self-reported race/ethnicity* |   |   | .0052 |
| Asian | 76 | 20.8 (4.85) |   |
| Black | 251 | 15.1 (2.29) |   |
| Hispanic | 405 | 20.7 (2.05) |   |
| Others | 116 | 17.0 (3.58) |   |
| White | 1687 | 13.1 (0.84) |   |
| Sex |   |   | .0094 |
| Female | 1163 | 13.1 (1.01) |   |
| Male | 1372 | 16.4 (1.02) |   |
| Age at diagnosis, y |   |   | 2.37 × 10−9 |
| < 10 | 1998 | 12.7 (0.76) |   |
| ≥ 10 | 537 | 23.4 (1.87) |   |
| Leukocyte count at diagnosis,/μL |   |   | 1.11 × 10−10 |
| < 50 000 | 2104 | 12.9 (0.75) |   |
| ≥ 50 000 | 431 | 24.8 (2.11) |   |
| CNS status† |   |   | .56 |
| CNS3 or traumatic | 86 | 17.0 (4.21) |   |
| CNS1 | 2124 | 14.4 (0.78) |   |
| CNS2 | 324 | 17.8 (2.17) |   |
| Missing | 1 | 0 (0) |   |
| Lineage and molecular subtype of ALL |   |   | 1.16 × 10−6 |
| B-other | 1673 | 16.0 (0.92) |   |
| BCR-ABL | 14 | 53.6 (15.77) |   |
| TCF3-PBX1 | 126 | 16.1 (3.3) |   |
| MLL rearrangement | 29 | 31.2 (8.83) |   |
| T cell | 109 | 19.5 (3.85) |   |
| ETV6-RUNX1 | 584 | 9 (1.2) |  |
| DNA index‡ |   |   | 1.04 × 10−5 |
| < 1.16 | 1798 | 16.7 (0.9) |   |
| ≥ 1.16 | 682 | 10.6 (1.21) |   |
| Missing | 55 | 9.1 (3.93) |   |
| End-of-induction MRD§ |   |   | 1 × 10−16 |
| < 0.01% | 1835 | 10.1 (0.72) |   |
| ≥ 0.01% & < 1% | 373 | 30.9 (2.45) |   |
| ≥ 1% | 81 | 59.8 (5.89) |   |
| Missing | 246 | 12.5 (2.13) |   |

Table 1 Caption: Patient characteristics

### Genotyping and genetic ancestry

Genotyping and quality control were performed as described previously. Briefly, germline DNA was genotyped using either the Affymetrix GeneChip human Mapping 500K sets or SNP6.0 array. Genotyping calls were determined using the BRLMM^15^ or the Birdseed algorithms^16^ for the 500K array and the SNP6.0 array, respectively. Genotypes were coded assuming an additive genetic model (based on the number of B alleles). Only SNPs common to both Affymetrix array platforms were investigated in this study. We also excluded 26 303 SNPs because of low minor allele frequency (< 0.5%) or poor call rate (< 95%), and 136 patients because of poor genotyping (call rate < 95%).

European, African, Asian, and Native-American genetic ancestries were estimated using STRUCTURE,^17^ with HapMap CEU, YRI, CHB/JPT, and a cohort of Native Americans^18^ as reference ancestral populations.

### GWAS for germline SNP genotypes related to risk of relapse

For this genome-wide association study (GWAS), relapse was defined as disease recurrence in bone marrow and/or extramedullary sites. Lineage switch, second malignancy, and death in remission were incorporated in the analyses as competing events. All genotype-relapse association analyses were stratified by 9 risk-adapted treatment arms: St Jude Total XIIIB low risk,^12^ St Jude Total XIIIB high risk,^12^ St Jude Total XV low risk,^13^ St Jude Total XV standard/high risk,^13^ COG P9906, and COG P9904/9905 regimens A, B, C, and D.^14^ Known risk factors (leukocyte count [≥ vs < 50 000/μL [age ≥ vs < 10 years], and the presence or absence of molecular ALL subtypes [including *MLL* rearrangements; *ETV6-RUNX1*, *TCF3-PBX1*, or *BCR-ABL1*; DNA index [≥ vs < 1.16]; and MRD [negative, positive, and high positive as defined below]) were included in the multivariate analysis together with SNP genotype.

To maximize the statistical power for detecting associations between SNP genotype and relapse risk, we performed a GWAS inclusive of all 2535 patients. SNPs were individually tested for association with relapse using a 2-step procedure. First, the Gray test (stratifying on treatment arms) was applied to each SNP and, second, those SNPs achieving *P* < .05 were subsequently analyzed using the Fine and Gray hazard rate regression model^19^ including both genetic ancestry (European, African, Asian, and Native American) and treatment arm as covariates, and genotypes were treated as ordinal variables under an additive genetic model. To balance false-positive and false-negative error rates, we examined the profile information criteria^20^ and determined that the *P* value threshold was 4.4 × 10^−3^ to declare SNPs significantly related to relapse hazard. To further prioritize the top relapse SNPs, we applied an iterative resampling procedure to determine whether a SNP was reproducibly associated with relapse ([Figure 1](#F1)). We first split 2535 patients at a 1:1 ratio into a discovery and a replication cohort, balancing on treatment arm, genetic ancestry, CNS disease status, and treatment outcome. SNPs were individually tested in the discovery cohort for association with relapse by the 2-step procedure (Gray test followed by the Fine and Gray hazard regression model) as described in this paragraph. Significant SNPs (at *P* < 4.4 × 10^−3^) from the discovery cohort GWAS were tested for replication. In the replication cohort, a SNP was considered validated if its genotype was associated with relapse at the 0.05 significance level by both the Gray test and the hazard rate regression model. This discovery-replication procedure was repeated 100 times ([Figure 1](#F1)), with each of iterations generating a list of replicated SNPs. Those SNPs that were replicated at least 10 times exceeded the frequency that would be expected by chance (*P* = .028 by the binomial model [100, 0.05]) and were designated as “relapse SNPs.” These SNPs were selected for further analyses with additional phenotypes.

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c6f3/3501717/9c9c793aac43/zh89991299130001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3501717_zh89991299130001.jpg)

Iterative resampling approach to identify 134 SNPs reproducibly associated with ALL relapse. A total of 2535 children with newly diagnosed ALL were split into a discovery and a validation cohort at a 1:1 ratio with balanced representation of treatment and clinical features. GWAS was performed on the discovery cohort and then on filtered SNPs based on replication in the remaining patients (replication cohort). Resampling was performed for 100 iterations and 134 SNPs were selected as “relapse SNPs” because they were successfully replicated in multiple rounds of resampling. Ip indicates information profiling (see “GWAS for germline SNP genotypes related to risk of relapse”).

### Association of relapse SNPs with additional prognostic phenotypes

#### Clinical presenting features.

Patient characteristics at diagnosis have been associated with relapse risk. In the present study, we assessed whether genotypes associated with relapse were also related to these prognostic patient characteristics. For each of 134 relapse SNPs, we tested the association between genotype and presenting leukocyte count ≥ versus < 50 000/μL, age ≥ versus < 10 years, and ALL blast DNA index ≥ versus < 1.16 by Fisher test. The association between SNP genotypes and molecular ALL subtypes (*MLL* rearrangements, *ETV6-RUNX1*, *TCF3-PBX1*, or *BCR-ABL1*, or T-ALL) was tested using the χ^2^ test.

#### MRD.

Of 2535 patients included in the GWAS of relapse, MRD status was available at the end of remission induction therapy (day 28 in the COG studies and day 46 in the St Jude studies) in 2289 children. MRD was measured by flow cytometry and classified as negative (< 0.01%), positive (≥ 0.01% but < 1%), or high-positive (≥ 1%) for St Jude patients. In COG patients, the MRD classification was nearly identical: negative (≤ 0.01%), positive (> 0.01% but ≤ 1%), or high-positive (> 1%). The Spearman rank correlation test was used to determine the association between genotypes and MRD, both of which were treated as ordinal variables.

#### Antileukemic drug disposition.

SNPs that have been previously associated with 4 pharmacokinetic and pharmacodynamic phenotypes were analyzed for their overlap with relapse SNPs in the present analyses. These SNPs were identified in GWASs from a subset of patients enrolled on St Jude Total Therapy XIIIB and XV protocols. Methotrexate plasma clearance was assessed in 699 children with ALL,^21^ and the GWAS adjusted for ancestry, dosage schedule, and sex. Those SNPs with *P* < .05 for their association with clearance were assessed for overlap with SNPs associated with relapse. Intracellular methotrexate polyglutamate accumulation in ALL blasts at 44 hours after upfront methotrexate therapy was determined in 144 newly diagnosed patients who received preinduction therapy with IV methotrexate.^22^ Those SNPs with *P* < .05 for their association with methotrexate polyglutamates (adjusting for subtype, ancestry, and treatment) were assessed for overlap with relapse SNPs. Dexamethasone apparent oral plasma clearance was determined in 334 patients at week 7 of continuation therapy^23^; those SNPs with *P* < .05 for their association with apparent oral clearance (adjusting for age, ancestry, and treatment arm) were assessed for overlap with relapse SNPs. Asparaginase antibody levels were measured in 403 patients using ELISA serially during therapy, and the total cumulative asparaginase Ab area under the curve was estimated^23^; those SNPs with *P* < .05 for their association with Ab exposure were assessed for overlap with SNPs associated with relapse. Associations between genotype and all 4 pharmacokinetic phenotypes were evaluated by linear regression tests.

### Statistical analysis

Statistical and computational analyses were performed using “R” Version 2.9.1 software ([www.r-project.org](http://www.r-project.org)) and SAS Version 9.2 software.

## Results

### Identification of genomic loci associated with ALL relapse risk

Using an iterative resampling strategy, we investigated which of the 444 044 germline SNP genotypes were reproducibly associated with relapse across treatment regimens ([Figure 1](#F1)). Dividing the 2535 patients into a discovery and a replication cohort at a 1:1 ratio, we used the discovery cohort to perform a genome-wide screen and then filtered SNPs based on their association in the replication cohort ([Figure 1](#F1)). A total of 134 SNPs were successfully replicated in at least 10 of 100 rounds of discovery-replication tests (the top 25 SNPs are shown in [Table 2](#T2) and a full listing is provided in supplemental Table 1, available on the *Blood* Web site; see the Supplemental Materials link at the top of the online article). These 134 SNPs, representing 88 independent genomic loci (pairwise *r*^2^ < 0.5, supplemental Figure 1), were also significant in a GWAS inclusive of all 2535 patients (*P* < .004) and thus were prioritized for subsequent analyses. A total of 73 of 134 SNPs were annotated to 43 genes (within 10 kb of a gene), all of which were intronic except for a synonymous coding variant in the *ATP8A2* gene (rs6491066).

### Table 2.

| SNP information |  |  |  | Association with ALL relapse in GWAS |  |  |  |  | P value for independent prognostic value |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| SNP ID | Chr | Position | Gene | Risk allele frequency | Risk allele | P | HR (95% CI) | Validation frequency | Adjusting for known risk factors | Within MRD− patients |
| rs7142143 | 14 | 50473281 | PYGL | 1% | C | 6.72 × 10−9 | 3.61 (2.34-5.57) | 79% | 1.20 × 10−5 | 2.9 × 10−4 |
| rs17079534 | 3 | 39822076 | MYRIP | 0.5% | A | 1.65 × 10−7 | 4.07 (2.4-6.87) | 74% | 2 × 10−4 | 4.74 × 10−5 |
| rs10170236 | 2 | 150165870 | LOC642340 | 25.6% | C | 3.92 × 10−6 | 1.45 (1.24-1.69) | 66% | 5.07 × 10−7 | .0035 |
| rs6683977 | 1 | 66541688 | PDE4B | 44.9% | C | 5.13 × 10−6 | 1.41 (1.22-1.64) | 63% | 5.9 × 10−4 | .081 |
| rs1402612 | 12 | 77517496 |   | 2.8% | T | 1.85 × 10−5 | 2.45 (1.63-3.69) | 59% | .0059 | 1.1 × 10−4 |
| rs7578361 | 2 | 150105464 |   | 27.6% | C | 8.40 × 10−6 | 1.4 (1.21-1.63) | 59% | 1.65 × 10−6 | .0033 |
| rs17837497 | 7 | 139349062 | TBXAS1 | 1.6% | A | 1.89 × 10−6 | 2.34 (1.65-3.31) | 56% | 6.63 × 10−5 | 7.8 × 10−4 |
| rs1496766 | 7 | 78169401 | MAGI2 | 1.4% | C | 5.45 × 10−6 | 2.84 (1.81-4.44) | 55% | 2.41 × 10−5 | 6.45 × 10−6 |
| rs9958208 | 18 | 38845082 | RIT2 | 9.9% | A | 5.12 × 10−6 | 1.62 (1.32-1.99) | 55% | 1.93 × 10−5 | 4.68 × 10−4 |
| rs9578871 | 13 | 24977627 | ATP8A2 | 33.4% | T | 2.08 × 10−5 | 1.37 (1.18-1.58) | 52% | 7.10 × 10−6 | .0016 |
| rs41322152 | 10 | 71710811 | NPFFR1 | 1.2% | C | 8.33 × 10−6 | 2.52 (1.68-3.79) | 52% | 2.02 × 10−6 | .0027 |
| rs7320729 | 13 | 24962148 | ATP8A2 | 47.2% | T | 1.35 × 10−5 | 1.39 (1.2-1.61) | 51% | 2.69 × 10−6 | 7.77 × 10−5 |
| rs546784 | 1 | 66535054 | PDE4B | 45.7% | A | 9.22 × 10−6 | 1.4 (1.2-1.62) | 51% | .0019 | .087 |
| rs7099696 | 10 | 102946382 |   | 1.7% | T | 1.17 × 10−5 | 2.57 (1.68-3.92) | 47% | 1.32 × 10−4 | 9.37 × 10−5 |
| rs12464261 | 2 | 69227954 | ANTXR1 | 46.5% | A | 2.79 × 10−5 | 1.37 (1.18-1.59) | 47% | 6.86 × 10−6 | 7.1 × 10−4 |
| rs879286 | 14 | 32777047 | NPAS3 | 3.3% | G | 2.34 × 10−5 | 1.95 (1.43-2.66) | 46% | 3.49 × 10−6 | 4.9 × 10−4 |
| rs7320702 | 13 | 24975538 | ATP8A2 | 33.5% | A | 2.22 × 10−5 | 1.37 (1.19-1.59) | 45% | 4.20 × 10−6 | 0.0013 |
| rs992435 | 7 | 96888739 |   | 2.5% | G | 4.66 × 10−5 | 2.12 (1.48-3.04) | 40% | 3.38 × 10−6 | 4.47 × 10−6 |
| rs6953777 | 7 | 10927258 |   | 1.2% | C | 4.91 × 10−5 | 2.62 (1.64-4.16) | 39% | 2.4 × 10−4 | .0049 |
| rs696518 | 3 | 137602990 | STAG1 | 40.9% | C | 4.71 × 10−5 | 1.36 (1.17-1.58) | 39% | .0016 | .015 |
| rs4297844 | 2 | 150108354 |   | 25.3% | C | 7.60 × 10−5 | 1.36 (1.17-1.58) | 38% | 1.64 × 10−5 | .028 |
| rs10153318 | 17 | 13265827 |   | 24.8% | A | 2.10 × 10−5 | 1.41 (1.21-1.66) | 38% | 6.29 × 10−5 | .00965 |
| rs6538901 | 12 | 98043535 | ANKS1B | 36.5% | G | 6.56 × 10−5 | 1.38 (1.18-1.62) | 37% | .0049 | .011 |
| rs12018180 | 13 | 25035957 | ATP8A2 | 33.2% | A | 7.38 × 10−5 | 1.34 (1.16-1.55) | 37% | 1.7 × 10−4 | 3.2 × 10−4 |
| rs9624873 | 22 | 24260784 |   | 4.4% | T | 5.50 × 10−5 | 1.8 (1.35-2.39) | 37% | .0058 | .14 |

Table 2 Caption: Top 25 SNPs associated with ALL relapse identified through genome-wide association and multiple rounds of resampling

Across the genome, the strongest association with relapse risk was observed at 14q22.1 in the *PYGL* gene (rs7142143). Each copy of the C allele at this *PYGL* intronic SNP (rs7142143) rendered a 3.6-fold increase in the risk of relapse (*P* = 6.7 × 10^−9^, [Figure 2](#F2)) and an association of this SNP with relapse was replicated in 79 of 100 rounds of resampling ([Table 2](#T2)).

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c6f3/3501717/5607c534c7ba/zh89991299130002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3501717_zh89991299130002.jpg)

Association of genotypes at the PYGL SNP (rs7142143) with the risk of ALL relapse. The cumulative incidence of any relapse was compared for each genotype group at rs7142143 (CC/CT or TT) in all patients (A) and in those patients negative for MRD at the end of remission induction (B). The P value was estimated using the Fine and Gray hazard regression model.

Many GWAS investigations of treatment response neglect to consider the confounding of germline SNP genotypes with known prognostic factors. In the present study, of 134 relapse SNPs, 73 were associated with one or more clinical presenting features that are recognized as prognostic in childhood ALL (*P* < .05, supplemental Table 2). For example, 32 SNPs had alleles that were related to leukocyte count ≥ 50 000/μL at diagnosis and to relapse risk; genotypes at 19 SNPs were enriched in children older than 10 years and associated with relapse; 16 SNPs were associated with hyperdiploid ALL (DNA index ≥ 1.16) and with a lower risk of relapse. In addition, 61 of 134 SNPs were associated with ALL molecular subtype and/or lineage (*MLL* rearrangements; *ETV6-RUNX1*, *TCF3-PBX1*, or *BCR-ABL1*; or T-cell ALL). However, 133 of 134 relapse SNPs remained prognostic after adjusting for all known risk factors (presenting features and MRD, supple-mental [Table 1](#T1)), indicating their independent associations with relapse risk.

### Associations of relapse SNPs with MRD

Early reduction of leukemia burden, as determined by MRD status at the end of remission induction therapy, is highly prognostic for ALL relapse. However, the majority of relapses still occur among patients with negative MRD status.^14^ Therefore, in the present study, we characterized the relationships between the 134 relapse SNPs with this early treatment response measure and determined the prognostic value of germline polymorphisms. Of 134 relapse SNPs, 133 (99%) remained prognostic after adjusting for MRD (*P* < .05) and 110 (82%) were significantly associated with relapse even within MRD^−^ patients (eg, rs7142143, which is shown in [Figure 2](#F2)B with full listing in supplemental Table 1), strongly suggesting that these genetic variants substantially contribute to interpatient variability in outcome beyond what is explained by MRD. Of 134 relapse SNPs, 34 were significantly associated with MRD, with the alleles related to higher MRD status always linked to higher relapse risk. For example, genotypes at rs10834571 in the *LUZP2* gene were most strongly associated with MRD (*P* = 3 × 10^−4^). Although the G allele at this SNP was related to higher MRD at the end of remission induction and a subsequent higher risk of relapse, it also identified patients at risk of relapse even within the MRD^−^ patient population (supplemental Figure 2).

### Relationships between relapse SNPs and pharmacologic phenotypes

To explore the mechanisms by which SNPs might influence treatment outcome of ALL, we examined the association of the 134 relapse SNPs with 4 pharmacokinetic and pharmacodynamic endophenotypes available in a subset of patients: methotrexate plasma clearance, intracellular accumulation of polyglutamated (active) methotrexate, dexamethasone plasma clearance, and asparaginase antibody levels. Three relapse SNPs were associated with methotrexate plasma clearance after high-dose intravenous methotrexate, and all 3 genotypes were related to low clearance (more drug exposure) and lower risk of relapse. Genotypes at 7 SNPs were associated with polyglutamated methotrexate levels in ALL blasts, 5 of which were associated with reduced levels of methotrexate polyglutamate and a higher relapse risk, including 3 SNPs within the *PDE4B* gene. Dexamethasone plasma clearance differed significantly by genotype at 4 relapse SNPs, 2 of which were within the ABC transporter gene *ABCB1* and both associated with higher dexamethasone clearance and a higher relapse risk ([Figure 3](#F3)). Finally, 5 relapse SNPs were related to higher asparaginase Ab levels, 2 of which were associated with a higher relapse rate. As summarized in [Table 3](#T3), 14 of 134 relapse SNPs were significant for at least 1 of the 4 pharmacologic phenotypes in a manner consistent with a pharmacokinetically intuitive association with relapse (ie, lower drug exposure translated into a higher risk of relapse).

### Figure 3.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c6f3/3501717/1242322d6df2/zh89991299130003.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3501717_zh89991299130003.jpg)

An example of relapse-associated SNPs affecting pharmacokinetics and pharmacodynamics of antileukemic agents. Genotype at ABCB1 SNP rs10264856 is associated with both ALL relapse (A) and dexamethasone apparent oral plasma clearance (B). Note that the C allele is linked to lower dexamethasone clearance and also lower cumulative incidence of relapse. The association of SNP genotype with clearance and with relapse was estimated by linear regression and the Fine and Gray hazard regression model, respectively. Dexamethasone clearance was determined in St Jude Total XV protocol at week 7 of continuation therapy, which is shown for those in the standard-/high-risk arm.

### Table 3.

| SNP information |  |  |  |  | Association with ALL relapse |  | Association with asparaginase antibody |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- |
| SNP ID | Chr | Position | Gene | MAF | P | Risk allele | P | The allele with higher antibody |
| rs251177 | 5 | 141007068 | FCHSD1 | 0.22 | 6.53 × 10−5 | C | .0053 | C |
| rs6773449 | 3 | 26322367 |   | 0.03 | 2.15 × 10−4 | C | .043 | C |

Table 3 Caption: Relationships between relapse-associated SNPs and pharmacokinetics and pharmacodynamics of antileukemic drugs

## Discussion

Although current risk classification schemes identify children with more aggressive ALL, a substantial portion of these high-risk patients still do not respond to more intensive chemotherapy. Conversely, a substantial portion of the children with ALL who eventually relapse are initially classified as being low risk at diagnosis.^24^ For example, even for MRD (the best clinical predictor of relapse in childhood ALL), more than 50% of relapse cases exhibit undetectable levels of leukemia burden after initial therapy and therefore cannot be reclassified for more intense treatment.^3,4,14^ For these reasons, it was particularly impressive that in the present study, germline variations distinguished patients at risk of relapse even within the group showing negative MRD, indicating that these genetic factors can significantly improve current models of risk stratification.

Capitalizing on the nearly population-based capture of childhood ALL patients in the United States to clinical trials led by the COG and St Jude, our present study systematically identified germline genetic variations associated with risk of relapse among 2535 children with ALL, the largest group of children with cancer ever studied for genetic determinants of outcome. Nevertheless, the number of patients receiving identical therapy is too small to separate patients into simple “discovery versus replication” cohorts for traditional genome-wide outcome analyses. Because therapy so strongly affects the likelihood of relapse, each discovery and replication cohort must be balanced for therapy arms. Therefore, we developed an iterative resampling approach^25,26^ in which patients were divided into a discovery and a replication cohort 100 times, each time balancing on treatment arm, ancestry, CNS status, and treatment outcome. By conducting the study in this manner, the replicated SNPs would be expected to be robust for multiple types of therapy. In fact, 133 of 134 relapse SNPs selected using this approach maintained prognostic significance after adjusting for treatment regimens and patient characteristics.

The top-ranked SNP is in the *PYGL* gene. *PYGL*, glycogen phosphorylase, is a target of adenosine monophosphate ([http://www.drugbank.ca/drugs/DB00131](http://www.drugbank.ca/drugs/DB00131)), which plays a critical role in response to antileukemic agents such as mercaptopurine and methotrexate.^27^ Interestingly, *PYGL* expression in diagnostic ALL blasts was shown to be positively correlated with in vitro response to prednisolone in 173 children with ALL (*P* = .002).^28^ *PYGL* has also been shown to be significantly overexpressed in a multidrug-resistant cancer cell line.^29^ Another interesting hit among the top-ranking genes is *PDE4B*, which encodes phosphodiesterase 4B, the primary regulator of cyclic AMP signaling in B lymphocytes.^30,31^ Prior studies have already shown that inhibition of PDE4B induces apoptosis in chronic lymphoblastic leukemia and diffuse large cell lymphoma^32,33^ and sensitizes cells to glucocorticoid-induced cell death.^34,35^ In ALL, pharmacologic inhibition of the PDE4 results in growth suppression and dexamethasone sensitivity,^36^ suggesting glucocorticoid response as a plausible mechanism by which PDE4B is linked to ALL relapse.

Over the past 50 years of studying prognostic features for ALL, many have been associated with relapse (eg, high leukocyte count, age ≥ 10 years, and blasts that are not hyperdiploid or carry unfavorable translations such as *MLL* rearrangements or *BCR-ABL1*); however, the mechanisms by which these features confer a higher relapse risk after chemotherapy remains mostly unclear.^1^ In the present study, we did find that a majority (73 of 134) of relapse SNPs were related to one or more clinical presenting features. Although nearly all SNPs remained prognostic after adjusting for those features, there may nevertheless be mechanisms by which germline polymorphisms associate with both poor outcome and with presenting ALL features.

We found herein that 14 of the 134 relapse SNPs (11%) were also associated with unfavorable pharmacokinetics of commonly used antileukemic agents, suggesting that some inherited variation is likely to affect relapse by affecting host disposition of antileukemic agents. Two SNPs annotated to *ABCB1*, rs10264856 and rs4728709, are in linkage disequilibrium with each other (*r*^2^ > 0.9) and were associated with increased dexamethasone apparent oral clearance. This would translate into lower plasma levels of dexamethasone, a phenotype that we have shown previously to be related to higher risk of ALL relapse.^23^ Interestingly, many drugs that are used to treat ALL, including glucocorticoids such as dexamethasone,^37^ anthracyclines,^38^ and vincristine,^38^ are substrates for the ABCB1 transporter (also known as MDR1 or multidrug resistance protein), and thus this association may be plausibly linked to both mechanisms of intrinsic blast resistance and to unfavorable host pharmacokinetic characteristics. Candidate SNPs in *ABCB1* have been shown previously to be associated with outcome in ALL.^39,40^

We conclude that germline SNPs were associated with risk of ALL relapse in this large cohort of newly diagnosed children with ALL, even among patients who were negative for MRD, so inherited SNP genotypes may be a useful additional feature for risk classification in this disease.

## Supplementary Material

## Acknowledgments

The authors thank all patients and their parents who participated in the St Jude Total Therapy XIIIB/XV and COG P9906/P9904/P9905 studies; clinicians and research staff at the St Jude and COG institutions; Dr Jeannette Pullen from the University of Mississippi at Jackson and Dr Andrew Carroll from the University of Alabama at Birmingham for assistance in classification of patients with ALL; and Dr Mark Shriver at the Pennsylvania State University for sharing SNP genotype data of the 105 Native-American references.

This work was supported by the National Institutes of Health (grants CA142665, CA21765, CA158568, CA156449, CA36401, CA98543, CA114766, CA140729, and U01GM92666) and the American Lebanese Syrian Associated Charities (ALSAC). The Jeffrey Pride Foundation and the National Childhood Cancer Foundation provided support for genome-wide genotyping of the COG specimens. S.P.H. holds the Ergen Family Chair in Pediatric Cancer. J.J.Y. is supported by an American Society of Hematology Scholar Award and an Alex's Lemonade Stand Foundation for Childhood Cancer Young Investigator Grant. These sponsors provided financial support for the St Jude and the COG clinical trials, as well as for the genetic studies, but were not directly involved in the design or execution of the study, genomic data collection/management/analysis, or review/approval of this manuscript.

## Footnotes

## Authorship

Contribution: J.J.Y., C.C., and M.V.R. conceived and designed the study; M.D., D.C., Y.F., G.N., M.J.B., N.J.W., P.L.M., W.P.B., B.C., C.L.W., S.P.H., W.E.E., C.-H.P., M.L., and M.V.R. acquired the data; J.J.Y. and M.V.R. wrote the manuscript; J.J.Y., D.C., G.N., C.L.W., S.P.H., W.E.E., C.-H.P., and M.V.R. critically revised the manuscript for important intellectual content; C.C., M.D., X.C., W.Y., N.C., and P.S. performed the statistical analysis; and D.C., M.J.B., G.H.R., W.L.C., S.P.H., W.E.E., and M.V.R. obtained funding for the study.

Conflict-of-interest disclosure: The authors declare no competing financial interests.

Correspondence: Mary V. Relling, St Jude Children's Research Hospital, Department of Pharmaceutical Sciences, 262 Danny Thomas Pl, Memphis, TN 38105; e-mail: mary.relling@stjude.org.

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Pui CH, Evans WE. Treatment of acute lymphoblastic leukemia. N Engl J Med. 2006;354(2):166–178. doi: 10.1056/NEJMra052603.  [DOI](https://doi.org/10.1056/NEJMra052603) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16407512/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Treatment%20of%20acute%20lymphoblastic%20leukemia.&author=CH%20Pui&author=WE%20Evans&volume=354&issue=2&publication_year=2006&pages=166-178&pmid=16407512&doi=10.1056/NEJMra052603&)

2. Stanulla M, Schrappe M. Treatment of childhood acute lymphoblastic leukemia. Semin Hematol. 2009;46(1):52–63. doi: 10.1053/j.seminhematol.2008.09.007.  [DOI](https://doi.org/10.1053/j.seminhematol.2008.09.007) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19100368/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Semin%20Hematol&title=Treatment%20of%20childhood%20acute%20lymphoblastic%20leukemia.&author=M%20Stanulla&author=M%20Schrappe&volume=46&issue=1&publication_year=2009&pages=52-63&pmid=19100368&doi=10.1053/j.seminhematol.2008.09.007&)

3. Schrappe M, Reiter A, Ludwig WD, et al. Improved outcome in childhood acute lymphoblastic leukemia despite reduced use of anthracyclines and cranial radiotherapy: results of trial ALL-BFM 90. German-Austrian-Swiss ALL-BFM Study Group. Blood. 2000;95(11):3310–3322.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/10828010/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Improved%20outcome%20in%20childhood%20acute%20lymphoblastic%20leukemia%20despite%20reduced%20use%20of%20anthracyclines%20and%20cranial%20radiotherapy:%20results%20of%20trial%20ALL-BFM%2090.%20German-Austrian-Swiss%20ALL-BFM%20Study%20Group.&author=M%20Schrappe&author=A%20Reiter&author=WD%20Ludwig&volume=95&issue=11&publication_year=2000&pages=3310-3322&pmid=10828010&)

4. Schrappe M, Reiter A, Zimmermann M, et al. Long-term results of four consecutive trials in childhood ALL performed by the ALL-BFM study group from 1981 to 1995. Berlin-Frankfurt-Munster. Leukemia. 2000;14(12):2205–2222. doi: 10.1038/sj.leu.2401973.  [DOI](https://doi.org/10.1038/sj.leu.2401973) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11187912/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Leukemia&title=Long-term%20results%20of%20four%20consecutive%20trials%20in%20childhood%20ALL%20performed%20by%20the%20ALL-BFM%20study%20group%20from%201981%20to%201995.%20Berlin-Frankfurt-Munster.&author=M%20Schrappe&author=A%20Reiter&author=M%20Zimmermann&volume=14&issue=12&publication_year=2000&pages=2205-2222&pmid=11187912&doi=10.1038/sj.leu.2401973&)

5. Dördelmann M, Reiter A, Borkhardt A, et al. Prednisone response is the strongest predictor of treatment outcome in infant acute lymphoblastic leukemia. Blood. 1999;94(4):1209–1217.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/10438708/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Prednisone%20response%20is%20the%20strongest%20predictor%20of%20treatment%20outcome%20in%20infant%20acute%20lymphoblastic%20leukemia.&author=M%20D%C3%B6rdelmann&author=A%20Reiter&author=A%20Borkhardt&volume=94&issue=4&publication_year=1999&pages=1209-1217&pmid=10438708&)

6. Campana D. Minimal residual disease in acute lymphoblastic leukemia. Semin Hematol. 2009;46(1):100–106. doi: 10.1053/j.seminhematol.2008.09.001.  [DOI](https://doi.org/10.1053/j.seminhematol.2008.09.001) | [PMC free article](/articles/PMC2632881/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19100372/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Semin%20Hematol&title=Minimal%20residual%20disease%20in%20acute%20lymphoblastic%20leukemia.&author=D%20Campana&volume=46&issue=1&publication_year=2009&pages=100-106&pmid=19100372&doi=10.1053/j.seminhematol.2008.09.001&)

7. Yang JJ, Cheng C, Yang W, et al. Genome-wide interrogation of germline genetic variation associated with treatment response in childhood acute lymphoblastic leukemia. JAMA. 2009;301(4):393–403. doi: 10.1001/jama.2009.7.  [DOI](https://doi.org/10.1001/jama.2009.7) | [PMC free article](/articles/PMC2664534/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19176441/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA&title=Genome-wide%20interrogation%20of%20germline%20genetic%20variation%20associated%20with%20treatment%20response%20in%20childhood%20acute%20lymphoblastic%20leukemia.&author=JJ%20Yang&author=C%20Cheng&author=W%20Yang&volume=301&issue=4&publication_year=2009&pages=393-403&pmid=19176441&doi=10.1001/jama.2009.7&)

8. Ansari M, Sauty G, Labuda M, et al. Polymorphisms in multidrug resistance-associated protein gene 4 is associated with outcome in childhood acute lymphoblastic leukemia. Blood. 2009;114(7):1383–1386. doi: 10.1182/blood-2008-11-191098.  [DOI](https://doi.org/10.1182/blood-2008-11-191098) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19515727/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Polymorphisms%20in%20multidrug%20resistance-associated%20protein%20gene%204%20is%20associated%20with%20outcome%20in%20childhood%20acute%20lymphoblastic%20leukemia.&author=M%20Ansari&author=G%20Sauty&author=M%20Labuda&volume=114&issue=7&publication_year=2009&pages=1383-1386&pmid=19515727&doi=10.1182/blood-2008-11-191098&)

9. Rocha JC, Cheng C, Liu W, et al. Pharmacogenetics of outcome in children with acute lymphoblastic leukemia. Blood. 2005;105(12):4752–4758. doi: 10.1182/blood-2004-11-4544.  [DOI](https://doi.org/10.1182/blood-2004-11-4544) | [PMC free article](/articles/PMC1895006/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15713801/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Pharmacogenetics%20of%20outcome%20in%20children%20with%20acute%20lymphoblastic%20leukemia.&author=JC%20Rocha&author=C%20Cheng&author=W%20Liu&volume=105&issue=12&publication_year=2005&pages=4752-4758&pmid=15713801&doi=10.1182/blood-2004-11-4544&)

10. Davies SM, Borowitz MJ, Rosner GL, et al. Pharmacogenetics of minimal residual disease response in children with B-precursor acute lymphoblastic leukemia: a report from the Children's Oncology Group. Blood. 2008;111(6):2984–2990. doi: 10.1182/blood-2007-09-114082.  [DOI](https://doi.org/10.1182/blood-2007-09-114082) | [PMC free article](/articles/PMC2265447/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18182569/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Pharmacogenetics%20of%20minimal%20residual%20disease%20response%20in%20children%20with%20B-precursor%20acute%20lymphoblastic%20leukemia:%20a%20report%20from%20the%20Children's%20Oncology%20Group.&author=SM%20Davies&author=MJ%20Borowitz&author=GL%20Rosner&volume=111&issue=6&publication_year=2008&pages=2984-2990&pmid=18182569&doi=10.1182/blood-2007-09-114082&)

11. Iwamoto S, Mihara K, Downing JR, Pui CH, Campana D. Mesenchymal cells regulate the response of acute lymphoblastic leukemia cells to asparaginase. J Clin Invest. 2007;117(4):1049–1057. doi: 10.1172/JCI30235.  [DOI](https://doi.org/10.1172/JCI30235) | [PMC free article](/articles/PMC1821067/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17380207/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Invest&title=Mesenchymal%20cells%20regulate%20the%20response%20of%20acute%20lymphoblastic%20leukemia%20cells%20to%20asparaginase.&author=S%20Iwamoto&author=K%20Mihara&author=JR%20Downing&author=CH%20Pui&author=D%20Campana&volume=117&issue=4&publication_year=2007&pages=1049-1057&pmid=17380207&doi=10.1172/JCI30235&)

12. Pui CH, Sandlund JT, Pei D, et al. Improved outcome for children with acute lymphoblastic leukemia: results of Total Therapy Study XIIIB at St Jude Children's Research Hospital. Blood. 2004;104(9):2690–2696. doi: 10.1182/blood-2004-04-1616.  [DOI](https://doi.org/10.1182/blood-2004-04-1616) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15251979/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Improved%20outcome%20for%20children%20with%20acute%20lymphoblastic%20leukemia:%20results%20of%20Total%20Therapy%20Study%20XIIIB%20at%20St%20Jude%20Children's%20Research%20Hospital.&author=CH%20Pui&author=JT%20Sandlund&author=D%20Pei&volume=104&issue=9&publication_year=2004&pages=2690-2696&pmid=15251979&doi=10.1182/blood-2004-04-1616&)

13. Pui CH, Campana D, Pei D, et al. Treating childhood acute lymphoblastic leukemia without cranial irradiation. N Engl J Med. 2009;360(26):2730–2741. doi: 10.1056/NEJMoa0900386.  [DOI](https://doi.org/10.1056/NEJMoa0900386) | [PMC free article](/articles/PMC2754320/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19553647/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Treating%20childhood%20acute%20lymphoblastic%20leukemia%20without%20cranial%20irradiation.&author=CH%20Pui&author=D%20Campana&author=D%20Pei&volume=360&issue=26&publication_year=2009&pages=2730-2741&pmid=19553647&doi=10.1056/NEJMoa0900386&)

14. Borowitz MJ, Devidas M, Hunger SP, et al. Clinical significance of minimal residual disease in childhood acute lymphoblastic leukemia and its relationship to other prognostic factors: a Children's Oncology Group study. Blood. 2008;111(12):5477–5485. doi: 10.1182/blood-2008-01-132837.  [DOI](https://doi.org/10.1182/blood-2008-01-132837) | [PMC free article](/articles/PMC2424148/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18388178/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Clinical%20significance%20of%20minimal%20residual%20disease%20in%20childhood%20acute%20lymphoblastic%20leukemia%20and%20its%20relationship%20to%20other%20prognostic%20factors:%20a%20Children's%20Oncology%20Group%20study.&author=MJ%20Borowitz&author=M%20Devidas&author=SP%20Hunger&volume=111&issue=12&publication_year=2008&pages=5477-5485&pmid=18388178&doi=10.1182/blood-2008-01-132837&)

15. Rabbee N, Speed TP. A genotype calling algorithm for affymetrix SNP arrays. Bioinformatics. 2006;22(1):7–12. doi: 10.1093/bioinformatics/bti741.  [DOI](https://doi.org/10.1093/bioinformatics/bti741) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16267090/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=A%20genotype%20calling%20algorithm%20for%20affymetrix%20SNP%20arrays.&author=N%20Rabbee&author=TP%20Speed&volume=22&issue=1&publication_year=2006&pages=7-12&pmid=16267090&doi=10.1093/bioinformatics/bti741&)

16. Korn JM, Kuruvilla FG, McCarroll SA, et al. Integrated genotype calling and association analysis of SNPs, common copy number polymorphisms and rare CNVs. Nat Genet. 2008;40(10):1253–1260. doi: 10.1038/ng.237.  [DOI](https://doi.org/10.1038/ng.237) | [PMC free article](/articles/PMC2756534/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18776909/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Integrated%20genotype%20calling%20and%20association%20analysis%20of%20SNPs,%20common%20copy%20number%20polymorphisms%20and%20rare%20CNVs.&author=JM%20Korn&author=FG%20Kuruvilla&author=SA%20McCarroll&volume=40&issue=10&publication_year=2008&pages=1253-1260&pmid=18776909&doi=10.1038/ng.237&)

17. Pritchard JK, Stephens M, Donnelly P. Inference of population structure using multilocus genotype data. Genetics. 2000;155(2):945–959. doi: 10.1093/genetics/155.2.945.  [DOI](https://doi.org/10.1093/genetics/155.2.945) | [PMC free article](/articles/PMC1461096/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10835412/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genetics&title=Inference%20of%20population%20structure%20using%20multilocus%20genotype%20data.&author=JK%20Pritchard&author=M%20Stephens&author=P%20Donnelly&volume=155&issue=2&publication_year=2000&pages=945-959&pmid=10835412&doi=10.1093/genetics/155.2.945&)

18. Mao X, Bigham AW, Mei R, et al. A genomewide admixture mapping panel for Hispanic/Latino populations. Am J Hum Genet. 2007;80(6):1171–1178. doi: 10.1086/518564.  [DOI](https://doi.org/10.1086/518564) | [PMC free article](/articles/PMC1867104/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17503334/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=A%20genomewide%20admixture%20mapping%20panel%20for%20Hispanic/Latino%20populations.&author=X%20Mao&author=AW%20Bigham&author=R%20Mei&volume=80&issue=6&publication_year=2007&pages=1171-1178&pmid=17503334&doi=10.1086/518564&)

19. Fine JP, Gray RJ. A proportional hazards model for the subdistribution of a competing risk. J Am Stat Assoc. 1999;94(446):496–509.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Am%20Stat%20Assoc&title=A%20proportional%20hazards%20model%20for%20the%20subdistribution%20of%20a%20competing%20risk.&author=JP%20Fine&author=RJ%20Gray&volume=94&issue=446&publication_year=1999&pages=496-509&)

20. Cheng C, Pounds SB, Boyett JM, Pei D, Kuo ML, Roussel MF. Statistical significance threshold criteria for analysis of microarray gene expression data. Stat Appl Genet Mol Biol. 2004;3 doi: 10.2202/1544-6115.1064. Article36.  [DOI](https://doi.org/10.2202/1544-6115.1064) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16646816/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Stat%20Appl%20Genet%20Mol%20Biol&title=Statistical%20significance%20threshold%20criteria%20for%20analysis%20of%20microarray%20gene%20expression%20data.&author=C%20Cheng&author=SB%20Pounds&author=JM%20Boyett&author=D%20Pei&author=ML%20Kuo&volume=3&publication_year=2004&pmid=16646816&doi=10.2202/1544-6115.1064&)

21. Ramsey LB, Bruun GH, Yang W, et al. Rare versus common variants in pharmacogenetics: SLCO1B1 variation and methotrexate disposition. Genome Res. 2012;22(1):1–8. doi: 10.1101/gr.129668.111.  [DOI](https://doi.org/10.1101/gr.129668.111) | [PMC free article](/articles/PMC3246196/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22147369/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Res&title=Rare%20versus%20common%20variants%20in%20pharmacogenetics:%20SLCO1B1%20variation%20and%20methotrexate%20disposition.&author=LB%20Ramsey&author=GH%20Bruun&author=W%20Yang&volume=22&issue=1&publication_year=2012&pages=1-8&pmid=22147369&doi=10.1101/gr.129668.111&)

22. French D, Yang W, Cheng C, et al. Acquired variation outweighs inherited variation in whole genome analysis of methotrexate polyglutamate accumulation in leukemia. Blood. 2009;113(19):4512–4520. doi: 10.1182/blood-2008-07-172106.  [DOI](https://doi.org/10.1182/blood-2008-07-172106) | [PMC free article](/articles/PMC2680361/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19066393/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Acquired%20variation%20outweighs%20inherited%20variation%20in%20whole%20genome%20analysis%20of%20methotrexate%20polyglutamate%20accumulation%20in%20leukemia.&author=D%20French&author=W%20Yang&author=C%20Cheng&volume=113&issue=19&publication_year=2009&pages=4512-4520&pmid=19066393&doi=10.1182/blood-2008-07-172106&)

23. Kawedia JD, Liu C, Pei D, et al. Dexamethasone exposure and asparaginase antibodies affect relapse risk in acute lymphoblastic leukemia. Blood. 2012;119(7):1658–1664. doi: 10.1182/blood-2011-09-381731.  [DOI](https://doi.org/10.1182/blood-2011-09-381731) | [PMC free article](/articles/PMC3286344/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22117041/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Dexamethasone%20exposure%20and%20asparaginase%20antibodies%20affect%20relapse%20risk%20in%20acute%20lymphoblastic%20leukemia.&author=JD%20Kawedia&author=C%20Liu&author=D%20Pei&volume=119&issue=7&publication_year=2012&pages=1658-1664&pmid=22117041&doi=10.1182/blood-2011-09-381731&)

24. Pui CH, Jeha S. New therapeutic strategies for the treatment of acute lymphoblastic leukaemia. Nat Rev Drug Discov. 2007;6(2):149–165. doi: 10.1038/nrd2240.  [DOI](https://doi.org/10.1038/nrd2240) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17268486/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Drug%20Discov&title=New%20therapeutic%20strategies%20for%20the%20treatment%20of%20acute%20lymphoblastic%20leukaemia.&author=CH%20Pui&author=S%20Jeha&volume=6&issue=2&publication_year=2007&pages=149-165&pmid=17268486&doi=10.1038/nrd2240&)

25. Den Boer ML, van Slegtenhorst M, De Menezes RX, et al. A subtype of childhood acute lymphoblastic leukaemia with poor treatment outcome: a genome-wide classification study. Lancet Oncol. 2009;10(2):125–134. doi: 10.1016/S1470-2045(08)70339-5.  [DOI](https://doi.org/10.1016/S1470-2045(08)70339-5) | [PMC free article](/articles/PMC2707020/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19138562/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet%20Oncol&title=A%20subtype%20of%20childhood%20acute%20lymphoblastic%20leukaemia%20with%20poor%20treatment%20outcome:%20a%20genome-wide%20classification%20study.&author=ML%20Den%20Boer&author=M%20van%20Slegtenhorst&author=RX%20De%20Menezes&volume=10&issue=2&publication_year=2009&pages=125-134&pmid=19138562&doi=10.1016/S1470-2045(08)70339-5&)

26. Freidlin B, Jiang W, Simon R. The cross-validated adaptive signature design. Clin Cancer Res. 2010;16(2):691–698. doi: 10.1158/1078-0432.CCR-09-1357.  [DOI](https://doi.org/10.1158/1078-0432.CCR-09-1357) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20068112/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&title=The%20cross-validated%20adaptive%20signature%20design.&author=B%20Freidlin&author=W%20Jiang&author=R%20Simon&volume=16&issue=2&publication_year=2010&pages=691-698&pmid=20068112&doi=10.1158/1078-0432.CCR-09-1357&)

27. Zaza G, Cheok M, Yang W, et al. Gene expression and thioguanine nucleotide disposition in acute lymphoblastic leukemia after in vivo mercaptopurine treatment. Blood. 2005;106(5):1778–1785. doi: 10.1182/blood-2005-01-0143.  [DOI](https://doi.org/10.1182/blood-2005-01-0143) | [PMC free article](/articles/PMC1895216/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15905191/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Gene%20expression%20and%20thioguanine%20nucleotide%20disposition%20in%20acute%20lymphoblastic%20leukemia%20after%20in%20vivo%20mercaptopurine%20treatment.&author=G%20Zaza&author=M%20Cheok&author=W%20Yang&volume=106&issue=5&publication_year=2005&pages=1778-1785&pmid=15905191&doi=10.1182/blood-2005-01-0143&)

28. Holleman A, Cheok MH, den Boer ML, et al. Gene-expression patterns in drug-resistant acute lymphoblastic leukemia cells and response to treatment. N Engl J Med. 2004;351(6):533–542. doi: 10.1056/NEJMoa033513.  [DOI](https://doi.org/10.1056/NEJMoa033513) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15295046/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Gene-expression%20patterns%20in%20drug-resistant%20acute%20lymphoblastic%20leukemia%20cells%20and%20response%20to%20treatment.&author=A%20Holleman&author=MH%20Cheok&author=ML%20den%20Boer&volume=351&issue=6&publication_year=2004&pages=533-542&pmid=15295046&doi=10.1056/NEJMoa033513&)

29. Heim S, Lage H. Transcriptome analysis of different multidrug-resistant gastric carcinoma cells. In Vivo. 2005;19(3):583–590.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/15875780/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=In%20Vivo&title=Transcriptome%20analysis%20of%20different%20multidrug-resistant%20gastric%20carcinoma%20cells.&author=S%20Heim&author=H%20Lage&volume=19&issue=3&publication_year=2005&pages=583-590&pmid=15875780&)

30. Houslay MD, Adams DR. PDE4 cAMP phosphodiesterases: modular enzymes that orchestrate signalling cross-talk, desensitization and compartmentalization. Biochem J. 2003;370(Pt 1):1–18. doi: 10.1042/BJ20021698.  [DOI](https://doi.org/10.1042/BJ20021698) | [PMC free article](/articles/PMC1223165/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12444918/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem%20J&title=PDE4%20cAMP%20phosphodiesterases:%20modular%20enzymes%20that%20orchestrate%20signalling%20cross-talk,%20desensitization%20and%20compartmentalization.&author=MD%20Houslay&author=DR%20Adams&volume=370&issue=Pt%201&publication_year=2003&pages=1-18&pmid=12444918&doi=10.1042/BJ20021698&)

31. Conti M, Richter W, Mehats C, Livera G, Park JY, Jin C. Cyclic AMP-specific PDE4 phosphodiesterases as critical components of cyclic AMP signaling. J Biol Chem. 2003;278(8):5493–5496. doi: 10.1074/jbc.R200029200.  [DOI](https://doi.org/10.1074/jbc.R200029200) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12493749/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=Cyclic%20AMP-specific%20PDE4%20phosphodiesterases%20as%20critical%20components%20of%20cyclic%20AMP%20signaling.&author=M%20Conti&author=W%20Richter&author=C%20Mehats&author=G%20Livera&author=JY%20Park&volume=278&issue=8&publication_year=2003&pages=5493-5496&pmid=12493749&doi=10.1074/jbc.R200029200&)

32. Smith PG, Wang F, Wilkinson KN, et al. The phosphodiesterase PDE4B limits cAMP-associated PI3K/AKT-dependent apoptosis in diffuse large B-cell lymphoma. Blood. 2005;105(1):308–316. doi: 10.1182/blood-2004-01-0240.  [DOI](https://doi.org/10.1182/blood-2004-01-0240) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15331441/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=The%20phosphodiesterase%20PDE4B%20limits%20cAMP-associated%20PI3K/AKT-dependent%20apoptosis%20in%20diffuse%20large%20B-cell%20lymphoma.&author=PG%20Smith&author=F%20Wang&author=KN%20Wilkinson&volume=105&issue=1&publication_year=2005&pages=308-316&pmid=15331441&doi=10.1182/blood-2004-01-0240&)

33. Meyers JA, Su DW, Lerner A. Chronic lymphocytic leukemia and B and T cells differ in their response to cyclic nucleotide phosphodiesterase inhibitors. J Immunol. 2009;182(9):5400–5411. doi: 10.4049/jimmunol.0804255.  [DOI](https://doi.org/10.4049/jimmunol.0804255) | [PMC free article](/articles/PMC2676892/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19380787/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Immunol&title=Chronic%20lymphocytic%20leukemia%20and%20B%20and%20T%20cells%20differ%20in%20their%20response%20to%20cyclic%20nucleotide%20phosphodiesterase%20inhibitors.&author=JA%20Meyers&author=DW%20Su&author=A%20Lerner&volume=182&issue=9&publication_year=2009&pages=5400-5411&pmid=19380787&doi=10.4049/jimmunol.0804255&)

34. Kim SW, Rai D, Aguiar RC. Gene set enrichment analysis unveils the mechanism for the phosphodiesterase 4B control of glucocorticoid response in B-cell lymphoma. Clin Cancer Res. 2011;17(21):6723–6732. doi: 10.1158/1078-0432.CCR-11-0770.  [DOI](https://doi.org/10.1158/1078-0432.CCR-11-0770) | [PMC free article](/articles/PMC4651210/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21742807/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&title=Gene%20set%20enrichment%20analysis%20unveils%20the%20mechanism%20for%20the%20phosphodiesterase%204B%20control%20of%20glucocorticoid%20response%20in%20B-cell%20lymphoma.&author=SW%20Kim&author=D%20Rai&author=RC%20Aguiar&volume=17&issue=21&publication_year=2011&pages=6723-6732&pmid=21742807&doi=10.1158/1078-0432.CCR-11-0770&)

35. Meyers JA, Taverna J, Chaves J, Makkinje A, Lerner A. Phosphodiesterase 4 inhibitors augment levels of glucocorticoid receptor in B cell chronic lymphocytic leukemia but not in normal circulating hematopoietic cells. Clin Cancer Res. 2007;13(16):4920–4927. doi: 10.1158/1078-0432.CCR-07-0276.  [DOI](https://doi.org/10.1158/1078-0432.CCR-07-0276) | [PMC free article](/articles/PMC2656255/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17699872/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&title=Phosphodiesterase%204%20inhibitors%20augment%20levels%20of%20glucocorticoid%20receptor%20in%20B%20cell%20chronic%20lymphocytic%20leukemia%20but%20not%20in%20normal%20circulating%20hematopoietic%20cells.&author=JA%20Meyers&author=J%20Taverna&author=J%20Chaves&author=A%20Makkinje&author=A%20Lerner&volume=13&issue=16&publication_year=2007&pages=4920-4927&pmid=17699872&doi=10.1158/1078-0432.CCR-07-0276&)

36. Ogawa R, Streiff MB, Bugayenko A, Kato GJ. Inhibition of PDE4 phosphodiesterase activity induces growth suppression, apoptosis, glucocorticoid sensitivity, p53, and p21(WAF1/CIP1) proteins in human acute lymphoblastic leukemia cells. Blood. 2002;99(9):3390–3397. doi: 10.1182/blood.v99.9.3390.  [DOI](https://doi.org/10.1182/blood.v99.9.3390) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11964308/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Inhibition%20of%20PDE4%20phosphodiesterase%20activity%20induces%20growth%20suppression,%20apoptosis,%20glucocorticoid%20sensitivity,%20p53,%20and%20p21(WAF1/CIP1)%20proteins%20in%20human%20acute%20lymphoblastic%20leukemia%20cells.&author=R%20Ogawa&author=MB%20Streiff&author=A%20Bugayenko&author=GJ%20Kato&volume=99&issue=9&publication_year=2002&pages=3390-3397&pmid=11964308&doi=10.1182/blood.v99.9.3390&)

37. Yates CR, Chang C, Kearbey JD, et al. Structural determinants of P-glycoprotein-mediated transport of glucocorticoids. Pharm Res. 2003;20(11):1794–1803. doi: 10.1023/b:pham.0000003377.39548.f6.  [DOI](https://doi.org/10.1023/b:pham.0000003377.39548.f6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14661924/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharm%20Res&title=Structural%20determinants%20of%20P-glycoprotein-mediated%20transport%20of%20glucocorticoids.&author=CR%20Yates&author=C%20Chang&author=JD%20Kearbey&volume=20&issue=11&publication_year=2003&pages=1794-1803&pmid=14661924&doi=10.1023/b:pham.0000003377.39548.f6&)

38. van Asperen J, van Tellingen O, Beijnen JH. The role of mdr1a P-glycoprotein in the biliary and intestinal secretion of doxorubicin and vinblastine in mice. Drug Metab Dispos. 2000;28(3):264–267.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/10681369/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=The%20role%20of%20mdr1a%20P-glycoprotein%20in%20the%20biliary%20and%20intestinal%20secretion%20of%20doxorubicin%20and%20vinblastine%20in%20mice.&author=J%20van%20Asperen&author=O%20van%20Tellingen&author=JH%20Beijnen&volume=28&issue=3&publication_year=2000&pages=264-267&pmid=10681369&)

39. Stanulla M, Schaffeler E, Arens S, et al. GSTP1 and MDR1 genotypes and central nervous system relapse in childhood acute lymphoblastic leukemia. Int J Hematol. 2005;81(1):39–44. doi: 10.1532/ijh97.e0418.  [DOI](https://doi.org/10.1532/ijh97.e0418) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15717687/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Hematol&title=GSTP1%20and%20MDR1%20genotypes%20and%20central%20nervous%20system%20relapse%20in%20childhood%20acute%20lymphoblastic%20leukemia.&author=M%20Stanulla&author=E%20Schaffeler&author=S%20Arens&volume=81&issue=1&publication_year=2005&pages=39-44&pmid=15717687&doi=10.1532/ijh97.e0418&)

40. Jamroziak K, Mlynarski W, Balcerczak E, et al. Functional C3435T polymorphism of MDR1 gene: an impact on genetic susceptibility and clinical outcome of childhood acute lymphoblastic leukemia. Eur J Haematol. 2004;72(5):314–321. doi: 10.1111/j.1600-0609.2004.00228.x.  [DOI](https://doi.org/10.1111/j.1600-0609.2004.00228.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15059065/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Haematol&title=Functional%20C3435T%20polymorphism%20of%20MDR1%20gene:%20an%20impact%20on%20genetic%20susceptibility%20and%20clinical%20outcome%20of%20childhood%20acute%20lymphoblastic%20leukemia.&author=K%20Jamroziak&author=W%20Mlynarski&author=E%20Balcerczak&volume=72&issue=5&publication_year=2004&pages=314-321&pmid=15059065&doi=10.1111/j.1600-0609.2004.00228.x&)
