# Genomewide Pharmacogenetics of Bisphosphonate-Induced Osteonecrosis of the Jaw: The Role of RBMS3

## Metadata
**Authors:** Paola Nicoletti, Vassiliki M Cartsos, Penelope K Palaska, Yufeng Shen, Aris Floratos, Athanasios I Zavras
**Journal:** The Oncologist
**Date:** 2012 Jan 20
**DOI:** [10.1634/theoncologist.2011-0202](https://doi.org/10.1634/theoncologist.2011-0202)
**PMID:** 22267851
**PMCID:** PMC3286178
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3286178/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC3286178/pdf/onc279.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC3286178/pdf/onc279.pdf)

## Abstract

A genomewide association study was conducted to search for genetic variants that increase the risk for bisphosphonate-related osteonecrosis of the jaw. Findings suggest that genetic susceptibility plays a role in the pathophysiology of bisphosphonate-related osteonecrosis of the jaw, with RBMS3 having a significant effect in the risk.

Keywords: Osteonecrosis, Jaw, Bisphosphonates, Pharmacogenetics, Screening, IGFBP7, ABCC4, RBMS3, Zoledronic acid, Genetic susceptibility, Genomewide association study, ONJ, GWAS

### Methods.

We ascertained BRONJ cases according to the diagnostic criteria of the American Association of Oral and Maxillofacial Surgeons. We genotyped cases and a set of treatment-matched controls using Illumina Human Omni Express 12v1 chip (733,202 markers). To maximize the power of the study, we expanded the initial control set by including population and treatment-tolerant controls from publicly available sources. Imputation at the whole-genome level was performed to increase the number of single nucleotide polymorphisms (SNPs) investigated. Tests of association were carried out by logistic regression, adjusting for population structure. We also examined a list of candidate genes comprising genes potentially involved in the pathogenesis of BRONJ and genes related to drug absorption, distribution, metabolism, and excretion.

### Results.

Based on principal component analysis, we initially analyzed 30 white cases and 17 treatment-tolerant controls. We subsequently expanded the control set to include 60 genetically matched controls per case. Association testing identified a significant marker in the *RBMS3* gene, rs17024608 (*p*-value < 7 × 10^−8^); individuals positive for the SNP were 5.8× more likely to develop BRONJ (odds ratio, 5.8; 95% confidence interval, 3.1–11.1). Candidate gene analysis further identified SNPs in *IGFBP7* and *ABCC4* as potentially implicated in BRONJ risk.

### Conclusion.

Our findings suggest that genetic susceptibility plays a role in the pathophysiology of BRONJ, with *RBMS3* having a significant effect in the risk.

## Background

Bisphosphonates (BPs) are widely prescribed antiosteoclastic medications. The i.v. administered BPs pamidronate and zoledronic acid are used in oncology to control bone metastasis and hypercalcemia. Oral BPs are used to control or prevent bone loss in osteoporosis. An estimated three million American women are currently on oral BPs [[1](#B1)–[3](#B3)]. BPs are synthetic analogs of pyrophosphate that localize to bones as a result of their affinity for hydroxyapatite and reduce osteoclastic activity. They are not readily metabolized, and thus have long-lasting effects that might extend for several years. BPs are especially attracted to, and localize in, areas of the bone undergoing inflammation or resorption. They are subsequently phagocytozed and internalized by osteoclasts. These internalized BPs, in turn, trigger apoptosis (cell death) of the osteoclasts, thus inhibiting osteoclast-mediated bone resorption [[1](#B1), [2](#B2)]. Osteoclasts seem to be affected both in terms of number and function. Animal studies have also demonstrated some antiangiogenic properties, which may partially explain the development of osteonecrosis resulting from the limited healing ability of the bone because of reduced vasculature [[4](#B4)].

BPs, especially zoledronic acid, have been associated with a serious adverse effect, osteonecrosis of the jaw. According to the American Association of Oral and Maxillofacial Surgeons (AAOMS), BP-related osteonecrosis of the jaw (BRONJ) is defined as a maxillofacial exposed bone lesion for ≥8 weeks in patients under BP treatment and with no prior history of radiation therapy of the jaws [[5](#B5)]. The nonhealing exposed necrotic lesions may involve the mandible or the maxilla or both, and can be painful, persistent, and resistant to treatment. The incidence of BRONJ is in the range of less than one in 10,000 to 10%, depending on the type of BP, drug administration route [[3](#B3), [6](#B6)], dose and duration of use, comorbidities, and treated condition [[1](#B1), [3](#B3), [6](#B6)]. Cancer patients are a group with a higher risk for BRONJ, whereas, among all BPs on the market, zoledronic acid seems to be the most frequently implicated drug [[1](#B1), [2](#B2)]. BRONJ affects as many as 5%–10% of zoledronic acid users and far fewer users of oral BPs. Age, race, smoking, obesity, cancer diagnosis, and poor oral health have been shown to be predisposing factors for BRONJ, but they explain only a small percentage of the entire risk [[6](#B6)–[8](#B8)]. Previous studies have suggested that genetic factors could be involved in BRONJ risk [[1](#B1), [9](#B9), [10](#B10)]. A genetic test capable of screening subjects for genetic susceptibility to BRONJ prior to initiating BP therapy would have great clinical utility, especially for cancer patients; it would reduce the incidence of osteonecrosis by restricting use to nonsusceptible individuals and would lead to improvements in quality of care [[1](#B1)]. We designed a pharmacogenetic genomewide association study to identify highly penetrant polymorphisms associated with BRONJ across multiple drugs. We recruited patients who had a definite BRONJ diagnosis. We looked across the whole genome for susceptibility single nucleotide polymorphisms (SNPs) and copy number variation (CNV) markers using a dense DNA array with >733,000 markers. Imputation analysis allowed the further expansion of the genomewide marker panel to include ∼3.5 million SNPs. Candidate SNPs in the insulin-like growth factor (IGF) gene family (*IGF1*, *IGF2*, *IGF1R*, *IGF2R*, *IGFBP*s) as well as in genes related to absorption, distribution, metabolism, and excretion of drugs (ADME genes) were specifically inspected [[11](#B11)].

## Methods

### Subject Recruitment

This research involved a hospital-based case–control study. The research protocol was reviewed and approved by the institutional review boards of the recruiting institutions. All the enrolled subjects signed a written informed consent form. The subjects were enrolled in the clinics of the Massachusetts General Hospital, Brigham & Women's Hospital, the Harvard School of Dental Medicine and its affiliated clinics, and Nova University Dental School in Florida. Initially, we searched electronic medical records and clinical notes to identify BP users. Among the BP users, we identified confirmed BRONJ cases according to the AAOMS diagnostic criteria and unaffected exposed controls. Cases were considered to have developed BRONJ if all the following three clinical characteristics were present: (a) the patient was under BP treatment when they developed ONJ, (b) the exposed, necrotic bone in the maxillofacial region persisted for >8 weeks, and (c) the patient had no history of radiation therapy to the jaws. Controls were patients currently under treatment with a BP who had no signs or symptoms of BRONJ, verified via clinical examination.

Potential participants were contacted by letter and were invited to participate in the study. Patients (cases) with confirmed BRONJ status were offered the alternative to participate via mail or to visit the clinic. Controls all visited the clinic to be examined to ensure their non-BRONJ status. After signing the consent form, participants were asked to answer a standardized questionnaire and to provide a saliva sample. The questionnaire contained questions on demographic characteristics as well as environmental factors and habits that might influence the risk for developing BRONJ.

### Genotyping

We collected a saliva sample from each recruited patient, using the Oragene DNA collection kit (DNA Genotek, Kanata, Canada). The saliva kits were mailed in one batch to the subcontracting genotyping facility at SABiosciences (Frederick, MD). DNA was extracted following the manufacturer's recommended protocol. High-throughput genotyping was performed using the Human Omni Express 12v1.0 Beadchip (Illumina, San Diego, CA). The Human Omni Express 12v1.0 Beadchip captures 731,442 markers.

### Genotype Quality Control

Raw data were processed with Illumina's GenomeStudio software and the downstream analysis was performed using the PLINK software [[12](#B12)]. We discarded markers that failed the following quality control criteria: (a) call rate >95%, (b) minor allele frequency (MAF) >1%, (c) a *p*-value for Hardy–Weinberg equilibrium (HWE) >.0000001 in controls (if applicable). We also confirmed that the call rate per sample was >95%. We tested for cryptic relatedness by estimating the identity-by-descent for all possible pairs of individuals.

### Additional Population and Drug-Treated Controls

To increase the power of our analyses, we augmented the initial BRONJ group with publicly available population controls. We selected white subjects from the population reference sample (POPRES) [[13](#B13)], Wellcome Trust Case Control Consortium (WTCCC) [[14](#B14)], Illumina iControlDB [[15](#B15)], and international Serious Adverse Events Consortium (iSAEC) [[16](#B16)] collections. All subjects, except the ones from the iControlDB, were genotyped using Illumina 1M or 1M-duo chips; the subjects from the iControlDB were genotyped using the Illumina 550K chip ([Table 1](#T1)). After applying standard quality control procedures, the controls were combined with the initial BRONJ group to produce the “population control group.” The effect of population structure was assessed through principal components analysis (PCA) using the smartPCA program from the EIGENSTRAT package (version 3.0) [[17](#B17)]. SNPs from known regions of long-range linkage disequilibrium were removed before running the PCA [[18](#B18)]. Because the population control group contained a disproportionally large set of north European subjects, we chose a fixed cases/control ratio of ∼1/60, selecting the best genetically matched controls based on eigenvalues of the significant PC axes.

### Table 1.

Table 1 Caption: Composition of control study cohorts

Additionally, in order to test the effect of possible confounding factors, which may be related to either BP exposure or clinical diagnosis, we downloaded a set of treatment-tolerant cancer sample data from the phs000210.v1.p1 cohort from dbGAP [[19](#B19)]. This cohort is composed of 878 breast cancer patients genotyped by the Illumina 610K chip. We selected 107 subjects who had been BP users by reviewing the patients' clinical data (form: pht000898.v1.p1.c1) included in the downloaded datasets. After selecting the white BP users by PCA, these controls were combined with the initial BRONJ treatment-tolerant controls to produce the “treatment-tolerant group.”

[](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3286178_onc00212-0946-t01.jpg)

### Imputation

Imputation was carried out using IMPUTE2 (version February 2009) with data from the 1000 Genomes Project (112 individuals, release number March 2010) and HapMap III (June 2010, all ethnicities) as the reference panels [[20](#B20)]. SNPs with poor quality were pruned before the imputation to avoid false positives. We divided the genome in 5000-bp long segments and used the “png-miss” option to fill in the missing genotypes. We used an ethnic mixed panel to improve the quality of the imputation for rare variants [[21](#B21)]. We retained imputed genotypes with: (a) posterior probability >0.9 [[20](#B20)], (b) no significant difference in missingness between cases and controls (χ^2^ test, *p*-value >.05), and (c) no significant deviation from HWE (*p*-value >.05).

### Statistical Analysis

We conducted statistical tests using the PLINK software [[12](#B12)]. We tested the association of single SNPs using logistic regression with the PCA eigenvalues as covariates under an additive model. We set the genomewide significance *p*-value threshold to 5 × 10^−8^ to correct for multiple testing (Bonferroni correction). When top results were imputed, we assessed the accuracy of the imputation manually, confirming that the quality of the signal intensity was within the range of acceptance for all SNPs in the haplotype generating the imputed genotype. For the candidate gene analysis, we reviewed the literature and identified genes possibly involved in the pathogenesis of BRONJ, including the IGF gene family and genes belonging to vitamin D metabolism. We also included ADME genes from a list compiled specifically for pharmacogenetic studies [[11](#B11)]. Appropriate Bonferroni correction was applied to the candidate gene analyses.

### CNV Analysis

We inferred CNVs from SNP chip data using the PennCNV software (April 2009 version) [[22](#B22)]. To ensure the accuracy of CNV calling, we applied stringent sample and CNV filtering procedures. We included all samples that had a log2 ratio standard deviation <0.5, maximum number of total CNV calls <50, bioaccumulation factor (BAF) median >0.55 or <0.45, BAF drift >0.01, or waviness factor >0.05 or <−0.05 (recommended parameters). Additionally, to ensure high-confidence CNVs, we excluded individual CNVs with a PennCNV-generated confidence score <10, those with calls based on <10 SNPs or CNV probes, and those with a span within 1 Mb from centromeres or telomeres.

We performed burden and common CNV association analysis whereby any CNV that was present in at least three subjects was considered to be common. Associations were tested with the PLINK software [[12](#B12)] using a two-tailed permuted Fisher's exact test (10^5^ permutations). Duplications and deletions were analyzed separately [[23](#B23)]. We also investigated singleton CNVs >500 kb to find evidence for individual predisposition to BRONJ. We adopted a coverage cutoff excluding all CNVs that had coverage <20 genetic markers. Finally, we selected the top 150 genes most frequently involved in CNVs (both duplications and deletions). We used the David software [[24](#B24)] to perform enrichment analysis (Fisher exact test) of the Kyoto Encyclopedia of Genes and Genomes (KEGG) Pathway Database (released December 2010) [[25](#B25)].

## Results

### Recruitment

We recruited a total of 67 individuals in the period 2008–2009. Of those, 32 were female cases with a mean age of 62.8 years, 15 were female controls with a mean age of 64.8 years, five were male controls with a mean age of 63.6 years, and 15 were male cases with a mean age of 64.8 years. The majority of the cases (28 of 47) and controls (13 of 20) had received zoledronic acid, with an average duration of 22.5 months. The mean number of months on zoledronic acid was higher in cases than in controls, but the difference was not statistically significant. Similarly, there was no significant difference between cases and controls in the mean number of months on zoledronic acid for the 14 subjects who reported a positive history of osteoporosis. Of the 67 individuals who participated in the study, we were able to extract DNA from 53 samples: 35 patients with osteonecrosis of the jaw and 18 treatment-tolerant controls. In what follows we refer to these 53 samples as the “BRONJ group.”

### Population Structure and Selection of Genetically Matched Controls

We applied PCA to expose the population structure of the BRONJ group and to find additional genetically matched population controls from publicly available collections. In order to confirm the self-reported ethnicity of the members of the BRONJ group, we merged their genotype data with that of 987 HapMap III subjects, which included subjects from 11 populations [[26](#B26)]. We found six individuals not clustering with the white HapMap III samples (CEU and TSI). For the remaining 47 white subjects, we attempted to refine ethnicity resolution by clustering them with population controls from the POPRES, WTCCC, and iSAEC collections, representing several European subpopulations. This analysis showed that the white BRONJ study subjects clustered with individuals of northwestern, southern, and eastern European descent ([Fig. 1](#F1)). To further increase the number of eastern European controls, we added 2,978 white samples from the iControlDB dataset. We formed the population control group by selecting the 60 closest controls for each case based on the eigenvalues of the first six principal components. The case/control ratio was chosen to maximize the total number of controls while keeping the ratio comparable among the three major clusters. In addition to the population control group, we identified publically available GWAS data on a set of breast cancer patients who had been treated with BPs without developing BRONJ. We used PCA to select 101 white treatment-tolerant cancer subjects from the 107 drug-exposed controls from the phs000210.v1.p1 cohort [[19](#B19)] and we merged them with the 17 white BRONJ treatment-tolerant controls in a treatment-tolerant group. PCA showed that all the three major ethnicity clusters (northwestern, southern, and eastern Europeans) were equally balanced in the treatment-tolerant group.

### Figure 1.

![Figure 1.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3d8d/3286178/8d16bb2aead2/onc9991109460001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3286178_onc9991109460001.jpg)

Population structure of the BRONJ group. The red circles represent the initial BRONJ group, blue dots represent CEU Hap Map III, orange dots represent TSI Hap Map III, and gray dots represent the European Collections (international Serious Adverse Events Consortium, Wellcome Trust Case Control Consortium, POPRES). First and third eigenvectors clearly separate the Europeans into northern (top), southern (lower center), and eastern (lower right) clusters. The gray cluster on the lower left represents the subjects of Spanish origin who belong to the POPRES collection. Abbreviations: BRONJ, bisphosphonate-related osteonecrosis of the jaw; CEU, Utah residents with Northern and Western European ancestry from the CEPH collection in HapMap III; PC, principal component; POPRES, population reference sample; TSI, Toscans in Italy in HapMap III.

### Genomewide Association Analysis

The white BRONJ group contained 30 white cases and 17 white treatment-exposed controls. In total, 631,507 SNPs passed quality control. In order to maximize the power of the study, we grouped each of the 30 white BRONJ cases with 60 of their closest genetically matched population controls, resulting in a study sample comprising 30 cases and 1,743 controls (724 males, 1,049 females). Because not all population controls were genotyped with the same chip, only 287,434 SNPs were shared by all subjects. We imputed this dataset using reference panels from HapMap 3 and the 1000 Genomes Project. In total, 3,542,142 markers passed quality control procedures specific for the imputation (see Methods). We tested the association of single SNPs using logistic regression with the first six eigenscores as covariates under an additive model. rs17024608, located in an intron of *RBMS3*, was found to be significantly associated with BRONJ (*p*-value = 7.4 × 10^−8^); individuals positive for rs17024608 had a fivefold higher risk for developing BRONJ than negative individuals. We note that this SNP was present in all genotyping platforms, but 2% of the controls had a missing call for this marker; consequently, only the missing genotypes were predicted by the imputation. [Table 2](#T2) summarizes the top findings from the logistic regression on the imputed data. [Figures 2](#F2) and [3](#F3), respectively, present the quantile-quantile plot of logistic regression and the Manhattan plot of the region (± 1 Mbp) surrounding rs17024608.

### Table 2.

Table 2 Caption: BRONJ genomewide association study: top associated SNPs

### Figure 2.

![Figure 2.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3d8d/3286178/8550c80b32db/onc9991109460002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3286178_onc9991109460002.jpg)

Quantile-quantile plot for logistic regression on the population control group, On the x-axis is −log10 of the expected p-values of an equally sized set of single nucleotide polymorphisms under a uniform distribution. The y-axis is −log10 of the observed p-values. Black solid lines denote the uniform null distribution. The bulk of the values (red dots) closely follows the expectation under the null model (black line), showing that there is no systematic artifact of population stratification. The tail end shows significant deviation from the null model, illustrating that there are a few observed significant associations.

### Figure 3.

![Figure 3.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3d8d/3286178/e09331153c06/onc9991109460003.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3286178_onc9991109460003.jpg)

Manhattan plot of the region surrounding rs17024608 (about ±1 Mb). Each dot represents a single nucleotide polymorphism (SNP). The x-axis represents the position of the SNP on the chromosome. The y-axis represents −log10 of the logistic regression p-value of the SNP in the case–control association study. Light blue dots represent imputed markers; dark blue dots represent genotyped markers. rs17024608 is marked in green with the p-value just below the genomewide threshold (dashed line). For this marker, we were able to impute the genotype for 2% of controls who had a missing value for this marker.

The treatment-tolerant group contained 118 treatment-tolerant controls (101 individuals from the breast cancer cohort and 17 from the BRONJ group). There was no difference in the minor allele frequency of rs17024608 between the treatment-tolerant group and the population controls (Fisher's exact test, *p*-value = .2) ([Table 3](#T3)). This finding is consistent with the hypothesis that this SNP is truly associated with the BRONJ phenotype because its association is unlikely to be a result of confounding factors related to BP exposure or clinical diagnosis.

### Table 3.

Table 3 Caption: MAF of rs17024608 in different white subgroups

[](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3286178_onc00212-0946-t02.jpg)

[](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3286178_onc00212-0946-t03.jpg)

### SNPs in Candidate Genes

Previous work suggests that inherited genetic variations in the IGF gene family or in ADME genes may play a role in the pathophysiology of BRONJ [[1](#B1), [9](#B9), [10](#B10)]. With regard to the IGF gene family, we examined 1,083 SNPs located within 20 kb downstream and upstream from the longer transcripts of 40 putative causal genes (hereafter, putative causal gene list). We also examined a list of 4,564 SNPs compiled for pharmacogenetic studies related to ADME genes (hereafter, the ADME list) [[11](#B11)]. The same two panels of markers were inspected in the population control group. No SNP reached significance after Bonferroni correction. In the putative causal gene list, the most significantly associated SNP was rs11934877, located in the intronic region of *IGFBP7*(odds ratio [OR], 2.9; 95% confidence interval [CI], 1.7–5.2; *p*-value = .00022). In the ADME list, rs1678387, intronic within *ABCC4*, was the top associated, with borderline significance after multiple testing correction (OR, 5.3; 95% CI, 2.4–11.4; *p*-value = 2.0 × 10^−5^). For the SNPs (or their proxies), there was no difference in MAF between population and drug-exposed control subjects ([Table 4](#T4)).

### Table 4.

Table 4 Caption: Top associated SNPs from the candidate genes analysis

[](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3286178_onc00212-0946-t04.jpg)

### CNV Association Analysis

All analyses were performed on the initial 52 white subjects. Fifty-two individuals (33 cases and 19 controls) passed our stringent quality control criteria for CNV calling; 431 CNVs were identified, of which 71 were duplications and 360 were deletions. Cases and controls did not differ significantly in their rate of CNV for both deletions and duplications. After multiple-test correction, none of the common CNVs had a significant association. We found two unique oversized (>700 kb) duplications in cases and none in controls. The duplications were found on chromosome 2 (925,407 bp; starting on rs4850234 and ending on rs16837705) and chromosome 22 (730,236 bp; starting on rs6003971 and ending on rs2845421).

Oversized singleton CNVs, as the ones predicted, might explain individual predisposition to the phenotype. In particular, the 730-kb heterozygous duplication covers the *SLC7A4* gene; the gene codes for a solute carrier transporter, which may be relevant for the bioavailability of BPs. The most enriched KEGG pathways in genes within CNVs were the Notch signaling pathway, involving five genes (*NOTCH1*, *PSEN1*, *NUMB*, *NOTCH4*, and *DVL1*) with an enrichment score of six and a Fisher's exact *p*-value of .009, the retinol pathway, involving five genes (*CYP3A7*, *CYP2C19*, *ADH6*, *CYP2A6*, *CYP2A7*) with an enrichment score of 5.2 and a *p*-value of .014, the P450 pathway, involving five genes (*CYP3A7*, *CYP2C19*, *ADH6*, *CYP2A6*, *CYP2A7*) with an enrichment score of 4.6 and a *p*-value of .023, and the drug metabolism pathway, involving four genes (*CES2*, *CYP3A7*, *CYP2A6*, *CYP2A7*) with an enrichment score of 1.0 and a *p*-value of .039.

## Discussion

Osteonecrosis of the jaw is a serious adverse effect of BPs, especially among cancer patients [[1](#B1), [3](#B3)]. In this vulnerable group, BRONJ negatively affects patient quality of life [[27](#B27)]. Differences in BRONJ incidence among ethnic groups and previously published results from pharmacogenetic studies indicate that genetic factors might be central in BRONJ predisposition, besides the known risk factors [[7](#B7)–[9](#B9)]. The goal of our GWAS was to identify high-penetrance genetic biomarkers associated with BRONJ. Using genetically matched population controls, we were able to identify rs17024608 in *RBMS3* as significantly associated with the risk for osteonecrosis, controlling for multiple comparisons. Furthermore, because there was no statistical difference in MAF between the treatment-tolerant group and the general population controls, the association of rs17024608 with BRONJ is unlikely to be a result of potential confounding factors related to either BP exposure or clinical diagnosis.

The MAF of rs17024608 in our control population matches with that previously reported for the white population (MAF, 0.09) in the SNP database dbSNP and is comparable among European countries. However, the risk allele is less frequent in the African population. This may partly explain why BRONJ seems to be more frequent in whites than Africans [[7](#B7)].

Independent biological evidence suggests that *RBMS3* might have a pivotal role in BRONJ etiology. RBMS3 is a binding protein for Prx1, a homeobox transcriptional factor that upregulates collagen type I in fibroblasts [[28](#B28)]. Type I collagen, coded by the COL1A gene family, is the main part of the bone matrix. Mutations in those genes produce genetic bone disorders characterized by fragile bones such as osteogenesis imperfecta [[29](#B29)]. Variations in RB*MS3* (rs10510628) and *COL1A* (rs1800012) have previously been associated with a decrease in bone mass and osteoporotic fractures, linking both genes with bone turnover [[30](#B30), [31](#B31)]. One of the possible BRONJ etiopathogenic mechanisms assumes that it can be caused by BP-associated suppressed bone turnover that leads to decreased blood flow, bone cell necrosis, and apoptosis [[32](#B32)]. Recently it was also shown that BPs downregulated collagen type I synthesis in human gingival fibroblasts and osteoblasts [[33](#B33)]. Hence, our finding suggests that BRONJ genetic susceptibility could affect bone turnover, enhancing the BP-associated suppressed effect on bone apposition.

Moreover, our study did not identify relevant signal on the major histocompatibility complex region. Human leukocyte antigen (HLA) haplotype variation is often associated with adverse drug reactions that have an immune-related pathogenesis [[34](#B34)–[36](#B36)]. HLA variants are mainly related to a drug-specific predisposition and can also be detected by GWAS with a small number of affected cases [[34](#B34), [35](#B35)]. Given the absence of such a signal, we could speculate that this adverse drug reaction (ADR) is more likely to be a toxic ADR, also corroborated by the fact that patients exposed to higher cumulative doses of BPs are at a greater risk for developing BRONJ. Co-occurring mutations on ADME genes might further augment BPs' intrinsic toxic effects, enhancing drug bioavailability. Indeed, the candidate SNP analyses led to interesting signals related to *ABCC4*, for which there was no significant difference in MAF between exposed and nonexposed control populations. The signal on ABCC4 is particularly intriguing. *ABCC4* codes for multidrug resistance transporter (MRP)4 (ABCC4); these transporters efflux endogenous and xenobiotic substrates out of cells, having a protective role, especially in the bone marrow, spleen, thymus, and gastrointestinal tract [[37](#B37)]. Inherited variation in these genes has been associated with the occurrence of toxic serious adverse events (e.g., cyclophosphamide-induced leukopenia/neutropenia) [[38](#B38)]. Currently, there is no published information on MRP4 and BPs. Moreover, CNVs may also predispose to the phenotype by disrupting genes belonging to drug metabolism pathways (*CYP3A7*, *CYP2A7*, *CYP3A6*, *SLC7A4*).

Finally *IGFBP7*, from the putative causal gene list analysis, showed a suggestive association with the phenotype. IGFs, especially IGF1 with its tyrosine kinase domain, are growth factors with potent signal transduction capabilities. IGFs are molecules with an important role in normal growth and development. IGF1-deficient children fail to achieve appropriate height, and pharmacologic therapies now exist to correct such deficiencies [[39](#B39)].

IGF1 and IGF2 are able to influence the replication and differentiation of bone cells through activation of their receptors, especially IGF1R whereas IGF-binding proteins, produced by bone cells, compete with the receptors in binding the ligands and thus affect the bioavailability of IGF1 and IGF2 [[40](#B40), [41](#B41)].

Although the *RBMS3* finding is compelling and was tested against different sets of controls, the reader should note that more research is needed to validate the finding in an independent set of study participants. With evolving observations about the effects of osteonecrosis in different races beyond whites, validation studies are also needed in subjects of various lineages, and in particular in subjects of Chinese Han origin or descent.

## Conclusion

Our pharmacogenetic genomewide association analysis revealed that one SNP on *RBMS3*, rs17024608, is significantly associated with BRONJ. Variations in *RBMS3* and *COLA1* have previously been associated with bone density, suggesting a role in bone turnover. The effect of the specific polymorphism in the etiology of osteonecrosis is currently unknown. It is plausible that RBMS3 may be involved in reduced collagen formation and the disruption of bone turnover, thus increasing the toxic effect of BPs. Candidate gene analyses further suggested that *IGFBP7 and ABCC4* might be implicated in BRONJ pathophysiology. More studies are needed to validate and replicate these results as well as to elucidate their functional relevance.

## Acknowledgments

The authors thank Drs. Thomas Dodson, Sook Bin Woo, Salvatore Ruggiero, and Cesar Migliorati for their help in recruiting study subjects. The authors wish to express their gratitude to all study participants for volunteering their time and effort to make this research possible. This research was supported by the National Institute of Dental and Craniofacial Research, National Institutes of Health (grant 7R21DE018143-03).

P.M. and V.M.C. contributed equally in this project.

## Footnotes

## Author Contributions

**Conception/Design:** Athanasios I. Zavras, Vassiliki M. Cartsos

**Provision of study material or patients:** Athanasios I. Zavras, Vassiliki M. Cartsos, Pinelopi K. Palaska

**Collection and/or assembly of data:** Athanasios I. Zavras, Vassiliki M. Cartsos, Pinelopi K. Palaska

**Data analysis and interpretation:** Athanasios I. Zavras, Paola Nicoletti, Vassiliki M. Cartsos, Yufeng Shen, Aris Floratos

**Manuscript writing:** Athanasios I. Zavras, Paola Nicoletti, Vassiliki M. Cartsos, Pinelopi K. Palaska, Yufeng Shen, Aris Floratos

**Final approval of manuscript:** Athanasios I. Zavras

**Other:** Principal Investigator: Athanasios I. Zavras

## References

1. Zavras AI. The impact of bisphosphonates on oral health: Lessons from the past and opportunities for the future. Ann N Y Acad Sci. 2011;1218:55–61. doi: 10.1111/j.1749-6632.2010.05876.x.  [DOI](https://doi.org/10.1111/j.1749-6632.2010.05876.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21291477/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20N%20Y%20Acad%20Sci&title=The%20impact%20of%20bisphosphonates%20on%20oral%20health:%20Lessons%20from%20the%20past%20and%20opportunities%20for%20the%20future&author=AI%20Zavras&volume=1218&publication_year=2011&pages=55-61&pmid=21291477&doi=10.1111/j.1749-6632.2010.05876.x&)

2. Borromeo GL, Tsao CE, Darby IB, et al. A review of the clinical implications of bisphosphonates in dentistry. Aust Dent J. 2011;56:2–9. doi: 10.1111/j.1834-7819.2010.01283.x.  [DOI](https://doi.org/10.1111/j.1834-7819.2010.01283.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21332734/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Aust%20Dent%20J&title=A%20review%20of%20the%20clinical%20implications%20of%20bisphosphonates%20in%20dentistry&author=GL%20Borromeo&author=CE%20Tsao&author=IB%20Darby&volume=56&publication_year=2011&pages=2-9&pmid=21332734&doi=10.1111/j.1834-7819.2010.01283.x&)

3. Reid IR. Osteonecrosis of the jaw: Who gets it, and why? Bone. 2009;44:4–10. doi: 10.1016/j.bone.2008.09.012.  [DOI](https://doi.org/10.1016/j.bone.2008.09.012) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18948230/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bone&title=Osteonecrosis%20of%20the%20jaw:%20Who%20gets%20it,%20and%20why?&author=IR%20Reid&volume=44&publication_year=2009&pages=4-10&pmid=18948230&doi=10.1016/j.bone.2008.09.012&)

4. Fournier P, Boissier S, Filleur S, et al. Bisphosphonates inhibit angiogenesis in vitro and testosterone-stimulated vascular regrowth in the ventral prostate in castrated rats. Cancer Res. 2002;62:6538–6544.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/12438248/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=Bisphosphonates%20inhibit%20angiogenesis%20in%20vitro%20and%20testosterone-stimulated%20vascular%20regrowth%20in%20the%20ventral%20prostate%20in%20castrated%20rats&author=P%20Fournier&author=S%20Boissier&author=S%20Filleur&volume=62&publication_year=2002&pages=6538-6544&pmid=12438248&)

5. American Association of Oral and Maxillofacial Surgeons. Position Paper on Bisphosphonate-Related Osteonecrosis of the Jaw—2009 Update. [accessed February 10, 2011)]. Available at http://www.aaoms.org/docs/position_papers/bronj_update.pdf.  [http://www.aaoms.org/docs/position_papers/bronj_update.pdf](http://www.aaoms.org/docs/position_papers/bronj_update.pdf) | [DOI](https://doi.org/10.1111/j.1747-4477.2009.00213.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19961450/)

6. Hoff AO, Toth B, Hu M, et al. Epidemiology and risk factors for osteonecrosis of the jaw in cancer patients. Ann N Y Acad Sci. 2011;1218:47–54. doi: 10.1111/j.1749-6632.2010.05771.x.  [DOI](https://doi.org/10.1111/j.1749-6632.2010.05771.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20946574/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20N%20Y%20Acad%20Sci&title=Epidemiology%20and%20risk%20factors%20for%20osteonecrosis%20of%20the%20jaw%20in%20cancer%20patients&author=AO%20Hoff&author=B%20Toth&author=M%20Hu&volume=1218&publication_year=2011&pages=47-54&pmid=20946574&doi=10.1111/j.1749-6632.2010.05771.x&)

7. Badros A, Weikel D, Salama A, et al. Osteonecrosis of the jaw in multiple myeloma patients: Clinical features and risk factors. J Clin Oncol. 2006;24:945–952. doi: 10.1200/JCO.2005.04.2465.  [DOI](https://doi.org/10.1200/JCO.2005.04.2465) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16484704/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Osteonecrosis%20of%20the%20jaw%20in%20multiple%20myeloma%20patients:%20Clinical%20features%20and%20risk%20factors&author=A%20Badros&author=D%20Weikel&author=A%20Salama&volume=24&publication_year=2006&pages=945-952&pmid=16484704&doi=10.1200/JCO.2005.04.2465&)

8. Wessel JH, Dodson TB, Zavras AI. Zoledronate, smoking, and obesity are strong risk factors for osteonecrosis of the jaw: A case-control study. J Oral Maxillofac Surg. 2008;66:625–631. doi: 10.1016/j.joms.2007.11.032.  [DOI](https://doi.org/10.1016/j.joms.2007.11.032) | [PMC free article](/articles/PMC2435366/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18355585/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Oral%20Maxillofac%20Surg&title=Zoledronate,%20smoking,%20and%20obesity%20are%20strong%20risk%20factors%20for%20osteonecrosis%20of%20the%20jaw:%20A%20case-control%20study&author=JH%20Wessel&author=TB%20Dodson&author=AI%20Zavras&volume=66&publication_year=2008&pages=625-631&pmid=18355585&doi=10.1016/j.joms.2007.11.032&)

9. Gong L, Altman RB, Klein TE. Bisphosphonates pathway. Pharmacogenet Genomics. 2011;21:50–53. doi: 10.1097/FPC.0b013e328335729c.  [DOI](https://doi.org/10.1097/FPC.0b013e328335729c) | [PMC free article](/articles/PMC3086066/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20023594/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Bisphosphonates%20pathway&author=L%20Gong&author=RB%20Altman&author=TE%20Klein&volume=21&publication_year=2011&pages=50-53&pmid=20023594&doi=10.1097/FPC.0b013e328335729c&)

10. Sarasquete ME, García-Sanz R, Marín L, et al. Bisphosphonate-related osteonecrosis of the jaw is associated with polymorphisms of the cytochrome P450 CYP2C8 in multiple myeloma: A genome-wide single nucleotide polymorphism analysis. Blood. 2008;112:2709–2712. doi: 10.1182/blood-2008-04-147884.  [DOI](https://doi.org/10.1182/blood-2008-04-147884) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18594024/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Bisphosphonate-related%20osteonecrosis%20of%20the%20jaw%20is%20associated%20with%20polymorphisms%20of%20the%20cytochrome%20P450%20CYP2C8%20in%20multiple%20myeloma:%20A%20genome-wide%20single%20nucleotide%20polymorphism%20analysis&author=ME%20Sarasquete&author=R%20Garc%C3%ADa-Sanz&author=L%20Mar%C3%ADn&volume=112&publication_year=2008&pages=2709-2712&pmid=18594024&doi=10.1182/blood-2008-04-147884&)

11. Ahmadi KR, Weale ME, Xue ZY, et al. A single-nucleotide polymorphism tagging set for human drug metabolism and transport. Nat Genet. 2005;37:84–89. doi: 10.1038/ng1488.  [DOI](https://doi.org/10.1038/ng1488) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15608640/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=A%20single-nucleotide%20polymorphism%20tagging%20set%20for%20human%20drug%20metabolism%20and%20transport&author=KR%20Ahmadi&author=ME%20Weale&author=ZY%20Xue&volume=37&publication_year=2005&pages=84-89&pmid=15608640&doi=10.1038/ng1488&)

12. Purcell S, Neale B, Todd-Brown K, et al. PLINK: A tool set for whole-genome association and population-based linkage analyses. Am J Hum Genet. 2007;81:559–575. doi: 10.1086/519795.  [DOI](https://doi.org/10.1086/519795) | [PMC free article](/articles/PMC1950838/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17701901/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=PLINK:%20A%20tool%20set%20for%20whole-genome%20association%20and%20population-based%20linkage%20analyses&author=S%20Purcell&author=B%20Neale&author=K%20Todd-Brown&volume=81&publication_year=2007&pages=559-575&pmid=17701901&doi=10.1086/519795&)

13. Nelson MR, Bryc K, King KS, et al. The population reference sample, POPRES: A resource for population, disease, and pharmacological genetics research. Am J Hum Genet. 2008;83:347–358. doi: 10.1016/j.ajhg.2008.08.005.  [DOI](https://doi.org/10.1016/j.ajhg.2008.08.005) | [PMC free article](/articles/PMC2556436/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18760391/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=The%20population%20reference%20sample,%20POPRES:%20A%20resource%20for%20population,%20disease,%20and%20pharmacological%20genetics%20research&author=MR%20Nelson&author=K%20Bryc&author=KS%20King&volume=83&publication_year=2008&pages=347-358&pmid=18760391&doi=10.1016/j.ajhg.2008.08.005&)

14. Wellcome Trust Case Control Consortium. Genome-wide association study of 14,000 cases of seven common diseases and 3,000 shared controls. Nature. 2007;447:661–678. doi: 10.1038/nature05911.  [DOI](https://doi.org/10.1038/nature05911) | [PMC free article](/articles/PMC2719288/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17554300/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Genome-wide%20association%20study%20of%2014,000%20cases%20of%20seven%20common%20diseases%20and%203,000%20shared%20controls&volume=447&publication_year=2007&pages=661-678&pmid=17554300&doi=10.1038/nature05911&)

15. illumina®. Science/Illumina iControlDB. [accessed November 19, 2011]. Available at http://www.illumina.com/science/icontroldb.ilmn.  [http://www.illumina.com/science/icontroldb.ilmn](http://www.illumina.com/science/icontroldb.ilmn)

16. Shen Y, Nicoletti P, Floratos A, et al. Genome-wide association study of serious blistering skin rash caused by drugs. Pharmacogenomics J. 2011 Jan 11; doi: 10.1038/tpj.2010.84. [Epub ahead of print]  [DOI](https://doi.org/10.1038/tpj.2010.84) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21221126/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=Genome-wide%20association%20study%20of%20serious%20blistering%20skin%20rash%20caused%20by%20drugs&author=Y%20Shen&author=P%20Nicoletti&author=A%20Floratos&publication_year=2011&pmid=21221126&doi=10.1038/tpj.2010.84&)

17. Price AL, Patterson NJ, Plenge RM, et al. Principal components analysis corrects for stratification in genome-wide association studies. Nat Genet. 2006;38:904–909. doi: 10.1038/ng1847.  [DOI](https://doi.org/10.1038/ng1847) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16862161/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Principal%20components%20analysis%20corrects%20for%20stratification%20in%20genome-wide%20association%20studies&author=AL%20Price&author=NJ%20Patterson&author=RM%20Plenge&volume=38&publication_year=2006&pages=904-909&pmid=16862161&doi=10.1038/ng1847&)

18. Novembre J, Johnson T, Bryc K, et al. Genes mirror geography within Europe. Nature. 2008;456:98–101. doi: 10.1038/nature07331.  [DOI](https://doi.org/10.1038/nature07331) | [PMC free article](/articles/PMC2735096/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18758442/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Genes%20mirror%20geography%20within%20Europe&author=J%20Novembre&author=T%20Johnson&author=K%20Bryc&volume=456&publication_year=2008&pages=98-101&pmid=18758442&doi=10.1038/nature07331&)

19. Ingle JN, Schaid DJ, Goss PE, et al. Genome-wide associations and functional genomic studies of musculoskeletal adverse events in women receiving aromatase inhibitors. J Clin Oncol. 2010;28:4674–4682. doi: 10.1200/JCO.2010.28.5064.  [DOI](https://doi.org/10.1200/JCO.2010.28.5064) | [PMC free article](/articles/PMC3020700/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20876420/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Genome-wide%20associations%20and%20functional%20genomic%20studies%20of%20musculoskeletal%20adverse%20events%20in%20women%20receiving%20aromatase%20inhibitors&author=JN%20Ingle&author=DJ%20Schaid&author=PE%20Goss&volume=28&publication_year=2010&pages=4674-4682&pmid=20876420&doi=10.1200/JCO.2010.28.5064&)

20. Marchini J, Howie B. Genotype imputation for genome-wide association studies. Nat Rev Genet. 2010;11:499–511. doi: 10.1038/nrg2796.  [DOI](https://doi.org/10.1038/nrg2796) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20517342/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Genet&title=Genotype%20imputation%20for%20genome-wide%20association%20studies&author=J%20Marchini&author=B%20Howie&volume=11&publication_year=2010&pages=499-511&pmid=20517342&doi=10.1038/nrg2796&)

21. Howie BN, Donnelly P, Marchini J. A flexible and accurate genotype imputation method for the next generation of genome-wide association studies. PLoS Genet. 2009;5:e1000529. doi: 10.1371/journal.pgen.1000529.  [DOI](https://doi.org/10.1371/journal.pgen.1000529) | [PMC free article](/articles/PMC2689936/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19543373/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet&title=A%20flexible%20and%20accurate%20genotype%20imputation%20method%20for%20the%20next%20generation%20of%20genome-wide%20association%20studies&author=BN%20Howie&author=P%20Donnelly&author=J%20Marchini&volume=5&publication_year=2009&pages=e1000529&pmid=19543373&doi=10.1371/journal.pgen.1000529&)

22. Wang K, Li M, Hadley D, et al. PennCNV: An integrated hidden Markov model designed for high-resolution copy number variation detection in whole-genome SNP genotyping data. Genome Res. 2007;17:1665–1674. doi: 10.1101/gr.6861907.  [DOI](https://doi.org/10.1101/gr.6861907) | [PMC free article](/articles/PMC2045149/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17921354/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Res&title=PennCNV:%20An%20integrated%20hidden%20Markov%20model%20designed%20for%20high-resolution%20copy%20number%20variation%20detection%20in%20whole-genome%20SNP%20genotyping%20data&author=K%20Wang&author=M%20Li&author=D%20Hadley&volume=17&publication_year=2007&pages=1665-1674&pmid=17921354&doi=10.1101/gr.6861907&)

23. Need AC, Ge D, Weale ME, et al. A genome-wide investigation of SNPs and CNVs in schizophrenia. PLoS Genet. 2009;5:e1000373. doi: 10.1371/journal.pgen.1000373.  [DOI](https://doi.org/10.1371/journal.pgen.1000373) | [PMC free article](/articles/PMC2631150/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19197363/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet&title=A%20genome-wide%20investigation%20of%20SNPs%20and%20CNVs%20in%20schizophrenia&author=AC%20Need&author=D%20Ge&author=ME%20Weale&volume=5&publication_year=2009&pages=e1000373&pmid=19197363&doi=10.1371/journal.pgen.1000373&)

24. Sherman BT, Huang da W, Tan Q, et al. DAVID Knowledgebase: A gene-centered database integrating heterogeneous gene annotation resources to facilitate high-throughput gene functional analysis. BMC Bioinformatics. 2007;8:426. doi: 10.1186/1471-2105-8-426.  [DOI](https://doi.org/10.1186/1471-2105-8-426) | [PMC free article](/articles/PMC2186358/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17980028/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Bioinformatics&title=DAVID%20Knowledgebase:%20A%20gene-centered%20database%20integrating%20heterogeneous%20gene%20annotation%20resources%20to%20facilitate%20high-throughput%20gene%20functional%20analysis&author=BT%20Sherman&author=W%20Huang%20da&author=Q%20Tan&volume=8&publication_year=2007&pages=426&pmid=17980028&doi=10.1186/1471-2105-8-426&)

25. Kanehisa M, Goto S, Furumichi M, et al. KEGG for representation and analysis of molecular networks involving diseases and drugs. Nucleic Acids Res. 2010;38(suppl 1):D355–D360. doi: 10.1093/nar/gkp896.  [DOI](https://doi.org/10.1093/nar/gkp896) | [PMC free article](/articles/PMC2808910/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19880382/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=KEGG%20for%20representation%20and%20analysis%20of%20molecular%20networks%20involving%20diseases%20and%20drugs&author=M%20Kanehisa&author=S%20Goto&author=M%20Furumichi&volume=38&issue=suppl%201&publication_year=2010&pages=D355-D360&pmid=19880382&doi=10.1093/nar/gkp896&)

26. Altshuler DM, Gibbs RA, Peltonen L, et al. International HapMap 3 Consortium. Integrating common and rare genetic variation in diverse human populations. Nature. 2010;467:52–58. doi: 10.1038/nature09298.  [DOI](https://doi.org/10.1038/nature09298) | [PMC free article](/articles/PMC3173859/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20811451/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=International%20HapMap%203%20Consortium.%20Integrating%20common%20and%20rare%20genetic%20variation%20in%20diverse%20human%20populations&author=DM%20Altshuler&author=RA%20Gibbs&author=L%20Peltonen&volume=467&publication_year=2010&pages=52-58&pmid=20811451&doi=10.1038/nature09298&)

27. Miksad RA, Lai KC, Dodson TB, et al. Quality of life implications of bisphosphonate-associated osteonecrosis of the jaw. The Oncologist. 2011;16:121–132. doi: 10.1634/theoncologist.2010-0183.  [DOI](https://doi.org/10.1634/theoncologist.2010-0183) | [PMC free article](/articles/PMC3228058/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21212433/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=The%20Oncologist&title=Quality%20of%20life%20implications%20of%20bisphosphonate-associated%20osteonecrosis%20of%20the%20jaw&author=RA%20Miksad&author=KC%20Lai&author=TB%20Dodson&volume=16&publication_year=2011&pages=121-132&pmid=21212433&doi=10.1634/theoncologist.2010-0183&)

28. Fritz D, Stefanovic B. RNA-binding protein RBMS3 is expressed in activated hepatic stellate cells and liver fibrosis and increases expression of transcription factor Prx1. J Mol Biol. 2007;371:585–595. doi: 10.1016/j.jmb.2007.06.006.  [DOI](https://doi.org/10.1016/j.jmb.2007.06.006) | [PMC free article](/articles/PMC1976254/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17586524/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Mol%20Biol&title=RNA-binding%20protein%20RBMS3%20is%20expressed%20in%20activated%20hepatic%20stellate%20cells%20and%20liver%20fibrosis%20and%20increases%20expression%20of%20transcription%20factor%20Prx1&author=D%20Fritz&author=B%20Stefanovic&volume=371&publication_year=2007&pages=585-595&pmid=17586524&doi=10.1016/j.jmb.2007.06.006&)

29. Rauch F, Glorieux FH. Osteogenesis imperfecta. Lancet. 2004;363:1377–1385. doi: 10.1016/S0140-6736(04)16051-0.  [DOI](https://doi.org/10.1016/S0140-6736(04)16051-0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15110498/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Osteogenesis%20imperfecta&author=F%20Rauch&author=FH%20Glorieux&volume=363&publication_year=2004&pages=1377-1385&pmid=15110498&doi=10.1016/S0140-6736(04)16051-0&)

30. Kiel D, Demissie S, Dupuis J, et al. Genome-wide association with bone mass and geometry in the Framingham Heart Study. BMC Med Genet. 2007;8(suppl 1):S14. doi: 10.1186/1471-2350-8-S1-S14.  [DOI](https://doi.org/10.1186/1471-2350-8-S1-S14) | [PMC free article](/articles/PMC1995606/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17903296/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Med%20Genet&title=Genome-wide%20association%20with%20bone%20mass%20and%20geometry%20in%20the%20Framingham%20Heart%20Study&author=D%20Kiel&author=S%20Demissie&author=J%20Dupuis&volume=8&issue=suppl%201&publication_year=2007&pages=S14&pmid=17903296&doi=10.1186/1471-2350-8-S1-S14&)

31. Qureshi AM, Herd RJ, Blake GM, et al. COLIA1 Sp1 polymorphism predicts response of femoral neck bone density to cyclical etidronate therapy. Calcif Tissue Int. 2002;70:158–163. doi: 10.1007/s00223-001-1035-9.  [DOI](https://doi.org/10.1007/s00223-001-1035-9) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11907712/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Calcif%20Tissue%20Int&title=COLIA1%20Sp1%20polymorphism%20predicts%20response%20of%20femoral%20neck%20bone%20density%20to%20cyclical%20etidronate%20therapy&author=AM%20Qureshi&author=RJ%20Herd&author=GM%20Blake&volume=70&publication_year=2002&pages=158-163&pmid=11907712&doi=10.1007/s00223-001-1035-9&)

32. Rizzoli R, Burlet N, Cahall D, et al. Osteonecrosis of the jaw and bisphosphonate treatment for osteoporosis. Bone. 2008;42:841–847. doi: 10.1016/j.bone.2008.01.003.  [DOI](https://doi.org/10.1016/j.bone.2008.01.003) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18314405/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bone&title=Osteonecrosis%20of%20the%20jaw%20and%20bisphosphonate%20treatment%20for%20osteoporosis&author=R%20Rizzoli&author=N%20Burlet&author=D%20Cahall&volume=42&publication_year=2008&pages=841-847&pmid=18314405&doi=10.1016/j.bone.2008.01.003&)

33. Simon M, Niehoff P, Kimmig B, et al. Expression profile and synthesis of different collagen types I, II, III, and V of human gingival fibroblasts, osteoblasts, and SaOS-2 cells after bisphosphonate treatment. Clin Oral Investig. 2010;14:51–58. doi: 10.1007/s00784-009-0312-2.  [DOI](https://doi.org/10.1007/s00784-009-0312-2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19597852/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Oral%20Investig&title=Expression%20profile%20and%20synthesis%20of%20different%20collagen%20types%20I,%20II,%20III,%20and%20V%20of%20human%20gingival%20fibroblasts,%20osteoblasts,%20and%20SaOS-2%20cells%20after%20bisphosphonate%20treatment&author=M%20Simon&author=P%20Niehoff&author=B%20Kimmig&volume=14&publication_year=2010&pages=51-58&pmid=19597852&doi=10.1007/s00784-009-0312-2&)

34. Chung WH, Hung SI, Hong HS, et al. Medical genetics: A marker for Stevens-Johnson syndrome. Nature. 2004;428:486. doi: 10.1038/428486a.  [DOI](https://doi.org/10.1038/428486a) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15057820/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Medical%20genetics:%20A%20marker%20for%20Stevens-Johnson%20syndrome&author=WH%20Chung&author=SI%20Hung&author=HS%20Hong&volume=428&publication_year=2004&pages=486&pmid=15057820&doi=10.1038/428486a&)

35. Daly AK, Donaldson PT, Bhatnagar P, et al. HLA-B*5701 genotype is a major determinant of drug-induced liver injury due to flucloxacillin. Nat Genet. 2009;41:816–819. doi: 10.1038/ng.379.  [DOI](https://doi.org/10.1038/ng.379) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19483685/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=HLA-B*5701%20genotype%20is%20a%20major%20determinant%20of%20drug-induced%20liver%20injury%20due%20to%20flucloxacillin&author=AK%20Daly&author=PT%20Donaldson&author=P%20Bhatnagar&volume=41&publication_year=2009&pages=816-819&pmid=19483685&doi=10.1038/ng.379&)

36. Lonjou C, Borot N, Sekula P, et al. A European study of HLA-B in Stevens-Johnson syndrome and toxic epidermal necrolysis related to five high-risk drugs. Pharmacogenet Genomics. 2008;18:99–107. doi: 10.1097/FPC.0b013e3282f3ef9c.  [DOI](https://doi.org/10.1097/FPC.0b013e3282f3ef9c) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18192896/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=A%20European%20study%20of%20HLA-B%20in%20Stevens-Johnson%20syndrome%20and%20toxic%20epidermal%20necrolysis%20related%20to%20five%20high-risk%20drugs&author=C%20Lonjou&author=N%20Borot&author=P%20Sekula&volume=18&publication_year=2008&pages=99-107&pmid=18192896&doi=10.1097/FPC.0b013e3282f3ef9c&)

37. Abla N, Chinn LW, Nakamura T, et al. The human multidrug resistance protein 4 (MRP4, ABCC4): Functional analysis of a highly polymorphic gene. J Pharmacol Exp Ther. 2008;325:859–868. doi: 10.1124/jpet.108.136523.  [DOI](https://doi.org/10.1124/jpet.108.136523) | [PMC free article](/articles/PMC2612728/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18364470/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharmacol%20Exp%20Ther&title=The%20human%20multidrug%20resistance%20protein%204%20(MRP4,%20ABCC4):%20Functional%20analysis%20of%20a%20highly%20polymorphic%20gene&author=N%20Abla&author=LW%20Chinn&author=T%20Nakamura&volume=325&publication_year=2008&pages=859-868&pmid=18364470&doi=10.1124/jpet.108.136523&)

38. Low SK, Kiyotani K, Mushiroda T, et al. Association study of genetic polymorphism in ABCC4 with cyclophosphamide-induced adverse drug reactions in breast cancer patients. J Hum Genet. 2009;54:564–571. doi: 10.1038/jhg.2009.79.  [DOI](https://doi.org/10.1038/jhg.2009.79) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19696793/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Hum%20Genet&title=Association%20study%20of%20genetic%20polymorphism%20in%20ABCC4%20with%20cyclophosphamide-induced%20adverse%20drug%20reactions%20in%20breast%20cancer%20patients&author=SK%20Low&author=K%20Kiyotani&author=T%20Mushiroda&volume=54&publication_year=2009&pages=564-571&pmid=19696793&doi=10.1038/jhg.2009.79&)

39. U.S. Food and Drug Administration. Mecasermin (Increlex, Tercica) prescribing and safety information. [accessed November 19, 2011]. http://www.accessdata.fda.gov/drugsatfda_docs/label/2011/021839s007lbl.pdf.  [http://www.accessdata.fda.gov/drugsatfda_docs/label/2011/021839s007lbl.pdf](http://www.accessdata.fda.gov/drugsatfda_docs/label/2011/021839s007lbl.pdf)

40. Andreassen TT, Oxlund H. The effects of growth hormone on cortical and cancellous bone. J Musculoskelet Neuronal Interact. 2001;2:49–58.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/15758476/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Musculoskelet%20Neuronal%20Interact&title=The%20effects%20of%20growth%20hormone%20on%20cortical%20and%20cancellous%20bone&author=TT%20Andreassen&author=H%20Oxlund&volume=2&publication_year=2001&pages=49-58&pmid=15758476&)

41. Gordeladze JO, Reseland JE, Drevon CA. Pharmacological interference with transcriptional control of osteoblasts: A possible role for leptin and fatty acids in maintaining bone strength and body lean mass. Curr Pharm Des. 2001;7:275–290. doi: 10.2174/1381612013398167.  [DOI](https://doi.org/10.2174/1381612013398167) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11254890/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Pharm%20Des&title=Pharmacological%20interference%20with%20transcriptional%20control%20of%20osteoblasts:%20A%20possible%20role%20for%20leptin%20and%20fatty%20acids%20in%20maintaining%20bone%20strength%20and%20body%20lean%20mass&author=JO%20Gordeladze&author=JE%20Reseland&author=CA%20Drevon&volume=7&publication_year=2001&pages=275-290&pmid=11254890&doi=10.2174/1381612013398167&)

42. Canalis E. Skeletal growth factors. In: Marcus R, Feldman D, Kelsey J, editors. Osteoporosis. San Diego: Academic Press; 1996. pp. 266–273.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Osteoporosis&author=E%20Canalis&publication_year=1996&)
