# Predisposition to abacavir hypersensitivity conferred by HLA-B*5701 and a haplotypic Hsp70-Hom variant

## Metadata
**Authors:** Annalise M Martin, David Nolan, Silvana Gaudieri, Coral Ann Almeida, Richard Nolan, Ian James, Filipa Carvalho, Elizabeth Phillips, Frank T Christiansen, Anthony W Purcell, James McCluskey, Simon Mallal
**Journal:** Proceedings of the National Academy of Sciences of the United States of America
**Date:** 2004 Mar 15
**DOI:** [10.1073/pnas.0307067101](https://doi.org/10.1073/pnas.0307067101)
**PMID:** 15024131
**PMCID:** PMC384715
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC384715/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC384715/pdf/1014180.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC384715/pdf/1014180.pdf)

## Abstract

Susceptibility to a clinically significant drug hypersensitivity syndrome associated with abacavir use seems to have a strong genetic component. We have previously shown that the presence of HLA-B*5701 strongly predicts abacavir hypersensitivity and have identified a potential susceptibility locus within a 300-kb region between the MEGT1 and C4A6 loci in the central MHC. We now report the results of fine recombinant genetic mapping in an expanded patient population of 248 consecutive, fully ascertained, abacavir-exposed individuals in the Western Australian HIV Cohort Study, in which 18 cases of definite abacavir hypersensitivity (7.3%) and 230 tolerant controls were identified. Haplotype mapping within patients with allelic markers of the 57.1 ancestral haplotype suggests a susceptibility locus within the 14-kb Hsp70 gene cluster. HLA-B*5701 was present in 94.4% of hypersensitive cases compared with 1.7% of controls (odds ratio, 960; P < 0.00001). A haplotypic nonsynonymous polymorphism of Hsp70-Hom (HspA1L, resulting from the substitution of residue M493T in the peptide-binding subunit) was found in combination with HLA-B*5701 in 94.4% of hypersensitive cases and 0.4% of controls (odds ratio, 3,893; P < 0.00001). Individuals with abacavir hypersensitivity demonstrated increased monocyte tumor necrosis factor expression in response to ex vivo abacavir stimulation, which was abrogated with CD8+ T cell depletion. These data indicate that the concurrence of HLA-B*5701 and Hsp70-Hom M493T alleles is necessary for the development of abacavir hypersensitivity, which is likely to be mediated by an HLA-B*5701-restricted immune response to abacavir.

Keywords: ancestral haplotype, microsatellite, human leukocyte antigen, single-nucleotide polymorphism

## Materials and Methods

**Subjects.** **Retrospective study.** The first 200 participants of the Western Australian HIV Cohort Study, to whom abacavir was prescribed before December 31, 2001, and who were included in our previous study ([1](#ref1)), were reclassified based on updated diagnostic criteria. Clinical diagnostic criteria included the occurrence of classical symptoms suggestive of multisystem involvement within 6 weeks of exposure. An appropriate temporal relationship between symptom resolution and cessation of abacavir was required by these diagnostic criteria. In cases where the clinical diagnostic criteria were fulfilled but an alternative explanation for these symptoms was present, epicutaneous patch testing and assessment of the specific immunological response to *ex vivo* abacavir stimulation were undertaken to clarify the role of abacavir.

**Prospective study.** Forty-eight individuals exposed to abacavir since the introduction of prospective genetic testing for *HLA-B*5701* and/or *HLA-DRB1*0701, HLA-DQ3* in December 2001 were classified by a clinician blinded to the results of genetic analysis using the updated diagnostic criteria.

**Patch Test.** Epicutaneous patch testing was undertaken, and reactions ([14](#ref14)) were read after 24 and 48 h of application. A positive result required absence of an allergic response to the control vehicle and demonstration of typical skin changes consisting of erythema and vesicular rash limited to the patch area. Attempts were made to patch test all individuals previously classified as abacavirhypersensitive cases, including those carrying markers of the 57.1 AH (*n* = 14, with 9 consenting to patch testing) as well as those who did not (*n* = 4, with 3 consenting to patch testing). Within the latter group, three individuals who had an alternative explanation for their symptoms (concurrent nevirapine therapy) were tested. The remaining individual who fulfilled diagnostic criteria for abacavir hypersensitivity in the absence of an alternative explanation had relocated and was unavailable for further testing.

Within the group clinically designated as “abacavir hypersensitivity not excluded” (*n* = 15) in our previous study ([1](#ref1)), three individuals consented to patch testing. Testing also was performed on abacavir-tolerant individuals carrying markers of the 57.1 AH (*n* = 12, with 4 consenting to patch testing), including two who carried the *HLA-B*5701* allele, and on two individuals carrying the full 57.1 AH (*HLA-B*5701, C4A6*, and *HLA-DRB1*0701, HLA-DQ3*) who had not been exposed to abacavir. Results were negative in all nine individuals.

**Typing of MHC Markers and Assignment of Alleles Specific to the 57.1 AH.** Typing of MHC markers was carried out by using standard genetic assays and sequencing. Single-nucleotide polymorphisms (SNPs) within genes located in the *C4A6*–*MEGT1* region were examined by using the National Center for Biotechnology Information's SNP database ([www.ncbi.nlm.nih.gov/SNP](http://www.ncbi.nlm.nih.gov/SNP)). Primers were designed based on oligonucleotide sequences reported within the database, and primers and conditions used in the assay are available on request. A set of well characterized 10 IHWS Epstein–Barr virus-transformed homozygous B cell lines representing commonly occurring European AHs was used to determine alleles carried on the 57.1 AH. Full-length sequencing across the *Hsp70* gene cluster was performed by using overlapping primer pairs.

**Measurement of the TNF Response to Abacavir.** Intracellular expression of TNF was assessed in cultured peripheral blood mononuclear cells (PBMCs) grown in the presence and absence of abacavir sulfate at 4 μg/ml (Ziagen 300-mg commercial grade tablet dissolved in PBS, pH 7.4) by means of three-color flow cytometry ([15](#ref15)). The PBMCs were stained with CD45-ECD, CD14-FITC, or IgG1-FITC (isotype control) antibodies (all obtained from Immunotech, Marseille, France) and analyzed on a Coulter EPICS XL-MCL flow cytometer by using the listmode fcs2 data analysis program (Coulter) ([15](#ref15)).

**CD4 and CD8 Cell Depletion.** TNF levels were measured in supernatants of 1 ml of whole blood cultured in the absence and presence of abacavir (1 μg/ml, Moravek Biochemicals, Brea, CA) for 24 h at 37°C. Whole blood was depleted of either CD4^+^ or CD8^+^ T cells by using the Dynabead kit (Dynal, Oslo).

**Statistical Analyses.** Odds ratios (ORs) were calculated by using Haldane's modification, which adds 0.5 to accommodate possible zero counts ([16](#ref16)). Comparisons of demographic and related data in abacavir-hypersensitive (*n* = 18) and abacavir-tolerant (*n* = 230) groups were made by using methods previously described ([1](#ref1)). Positive and negative predictive values were calculated within the retrospective abacavir-exposed cohort (*n* = 200) to exclude potential confounding caused by the active exclusion of abacavir use among those carrying 57.1 AH markers in the prospective cohort. Absolute and relative LD between *HLA-B*5701* and *Hsp70-Hom M493T* was calculated by using the formulas of Cavalli-Sforza and Bodmer ([17](#ref17)) and Baur and Danilov ([18](#ref18)). Significance of LD was measured by using Fisher's exact test.

## Results

**Prevalence of Abacavir Hypersensitivity.** In this abacavir-exposed cohort of 248 individuals, 18 cases with abacavir-hypersensitive reactions were identified by applying restrictive diagnostic criteria to identify “definite” cases of hypersensitivity for the purposes of genetic analysis. This group included 16 definite abacavirhypersensitive cases from the previously reported cohort of 200 consecutive abacavir-exposed individuals ([1](#ref1)), providing a prevalence of 8%. Two other cases were identified among 48 individuals treated with abacavir after prospective genetic testing.

In our previous study ([1](#ref1)), 18 of 200 individuals were clinically classified as abacavir-hypersensitive. Among these, three individuals were receiving concurrent nevirapine therapy, which also may cause hypersensitive reactions. These patients had a negative response to the epicutaneous abacavir skin patch test after 48 h of application and did not develop a cytokine response to *ex vivo* abacavir stimulation (see below). Subsequently, two of these patients have been rechallenged successfully with abacavir without developing abacavir hypersensitivity, whereas the third patient has developed a similar hypersensitive reaction to efavirenz (a drug within the same class as nevirapine). The negative epicutaneous patch test results for these three individuals contrast with the positive results obtained from the nine remaining patients previously classified as abacavir-hypersensitive who underwent testing (see *Materials and Methods* for details), whereas negative results were obtained in all six abacavir-tolerant individuals tested. On these grounds, these individuals did not meet the clinical and immunological diagnostic criteria used in this study. Another individual from this retrospective cohort (case 14) was classified as abacavir-hypersensitive, having been clinically designated as “abacavir hypersensitivity not excluded.” This individual developed typical and severe symptoms that fulfilled the clinical diagnostic criteria within a few days of exposure to abacavir, and no alternative explanation for these symptoms was identified. However, the diagnosis of abacavir hypersensitivity was not made, and the symptoms gradually resolved over a period of days with intensive supportive inpatient care at a peripheral hospital. The individual was therefore likely to have become desensitized by the continued administration of abacavir and had a negative response to epicutaneous patch testing, also in keeping with desensitization to the drug.

In our previous study ([1](#ref1)), a clinical diagnostic category of “abacavir hypersensitivity not excluded” was applied to 15 individuals who developed any symptoms within 6 weeks of the introduction of abacavir therapy ([1](#ref1)), whereas the remaining, abacavirtolerant group (*n* = 167) was entirely asymptomatic. One case described above was classified here as abacavir-hypersensitive. In the remaining 14 individuals, an alternative cause of symptoms was present in five cases (three were treated concurrently with efavirenz, one with nevirapine, and one with amprenavir). In these individuals, symptoms did not have an appropriate temporal relationship with abacavir therapy and/or recurred on rechallenge with the alternative antiretroviral drug. Minor symptoms described by nine individuals did not fulfill clinical diagnostic criteria. These cases were therefore classified as abacavir-tolerant for the purposes of genetic analysis.

Since our initial study ([1](#ref1)), 48 individuals have been HLA-typed before exposure to abacavir. All patients with *HLA-B*5701* and *HLA-DRB1***0701, HLA-DQ3* alleles were actively excluded from exposure to abacavir-based regimens. Two of 48 patients (4.2%) developed a hypersensitive reaction, whereas the remaining 46 patients remained asymptomatic on abacavir therapy. One individual carried the *HLA-B*5701, C4A6*, and *HLA-DRB1*0701, HLA-DQ3* alleles and was prescribed abacavir before review of the results of the HLA typing; another individual, who carried the *HLA-B*5701* but not the *C4A6* allele or the *HLA-DRB1*0701, HLA-DQ3* combination, made an informed choice to undertake abacavir therapy. Hence, the incidence of abacavir hypersensitivity among *HLA-B*5701*-negative individuals in this prospective cohort was 0% (95% confidence interval, 0–0.075%).

Demographic data for the entire cohort of 248 individuals is given in [Table 1](#tbl1). Within the abacavir-exposed cohort (*n* = 248), 214 participants were classified as patients of predominantly European or Asian Indian descent, and the rest consisted of 11 patients of African origin, 15 indigenous Australians, and 8 patients of Asian origin.

### Table 1. Details of patient demographics and immunological status in the abacavir-hypersensitive and -tolerant groups.

|   | Sex, n (%) |  | Ethnicity, n (%) |  |   |   |   |
| --- | --- | --- | --- | --- | --- | --- | --- |
|   | M | F | E | NE | Mean age, years (SD) | Mean CD4+ count, cells per μl (SD) | Mean CD8+ proportion, % (SD) |
| Hypersensitive, n = 18 | 15 (83) | 3 (17) | 18 (100) | 0 | 45.0 (10.3) | 446 (299) | 51.8 (12.1) |
| Tolerant, n = 230 | 197 (86) | 33 (14.3) | 196 (85) | 34 (15) | 42.6 (10.3) | 435 (284) | 51.5 (12.1) |

Table 1 Caption: P values follow for male (M) abacavir-hypersensitive and -tolerant individuals, abacavir-hypersensitive and -tolerant individuals of European descent (E), mean age, mean CD4+ T cell count, and mean CD8+ T cell proportion: 0.73, 0.15, 0.35, 0.85, and 0.92, respectively. F, female; NE, individuals of non-European descent.

**MHC Marker Frequency.** The distribution of MHC alleles was significantly different in the abacavir-hypersensitive, compared with the abacavir-tolerant, group ([Table 2](#tbl2) and [Fig. 1](#fig1)). The positive and negative predictive values presented in [Table 2](#tbl2) were calculated for the retrospective cohort of 200 individuals exposed to abacavir before December 2001. With regard to alleles specific to the 57.1 AH, *HLA-B*5701* was present in 17 of 18 (94.4%) cases with abacavir hypersensitivity and 4 of 230 (1.7%) abacavir-tolerant controls [OR = 960, *P* value corrected for the number of alleles examined (*P*_c_) < 0.0001]. *C4A6* was present in 14 of 18 (77.8%) cases and 7 of 230 (3.0%) controls (OR = 111, *P*_c_ < 0.0001), and the haplospecific combination of *HLA-DRB1*0701* and *HLA-DQ3* was present in 14 of 18 (77.8%) cases and 11 of 230 (4.8%) controls (OR = 70, *P*_c_ < 0.0001). Hence, *HLA-B*5701* provided the strongest univariate association with susceptibility to abacavir hypersensitivity. When these markers were combined, the previously described *HLA-B*5701, C4A6, HLA-DRB1*0701, HLA-DQ3* haplotype ([1](#ref1)) was present in 14 of 18 (77.8%) cases and in none of the 230 controls (OR = 1,485, *P*_c_ < 0.0001).

### Table 2. MHC alleles observed within the abacavir-hypersensitive and -tolerant controls.

| MHC marker | Hypersensitive, n (%) | Tolerant, n (%) | OR | Positive predictive value, % | Negative predictive value, % |
| --- | --- | --- | --- | --- | --- |
| HLA-B*5701 | 17 (94.4) | 4 (1.7) | 960.0 | 78.9 | 99.4 |
| C4A6 | 14 (77.8) | 7 (3.0) | 111.5 | 72.2 | 98.3 |
| HLA-DRB1*0701, HLA-DQ3 | 14 (77.8) | 11 (4.8) | 69.7 | 65.0 | 98.3 |
| Hsp70-Hom M493T | 17 (94.4) | 51 (22.2) | 59.7 | 25.4 | 99.3 |
| HLA-B*5701, C4A6 | 14 (77.8) | 0 | 1,485.0 | 100.0 | 98.4 |
| HLA-B*5701, HLA-DRB1*0701, HLA-DQ3 | 14 (77.8) | 0 | 1,485.0 | 100.0 | 98.4 |
| HLA-B*5701 Hsp70-Hom M493T | 17 (94.4) | 1 (0.43) | 3,893.0 | 93.8 | 99.5 |

Table 2 Caption: The HLA-B*5701, HLA-DRB1*0701, HLA-DQ3, and C4A6 typing was performed as described in our previous study (1). The Hsp70-Hom M493T SNPs were typed by using a combination of restriction fragment-length polymorphism and sequence-specific primer polymorphism assays. For all MHC markers, corrected P values are <0.00001. Positive and negative predictive values were calculated within the retrospective abacavir-exposed cohort (n = 200), to exclude potential confounding caused by the active exclusion of abacavir use among those carrying 57.1 AH markers in the prospective cohort.

### Fig. 1.

![Fig. 1.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/245a/384715/fbd795321f09/zpq0090440320001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=384715_zpq0090440320001.jpg)

Increased frequency of markers of the 57.1 AH among abacavirhypersensitive individuals. As shown from left to right, MHC markers including HLA-B*5701, C4A6, and HLA-DR7, HLA-DQ3 were compared in the healthy controls of the Western Australian Bone Marrow Donor Registry (n = 3,212), HIV-positive cohort (n = 381), abacavir-exposed cohort (n = 248), abacavirhypersensitive group (n = 18), and abacavir-tolerant group (n = 230).

**Mapping of Putative Susceptibility Loci Within the MHC.** To map the putative susceptibility locus/loci within the MHC, polymorphic markers typically found on the 57.1 AH were required. Such markers within the central MHC region between *C4A6* and *MEGT1* were identified by comparing examples of the 57.1 AH with a subset of well characterized Epstein–Barr virus-transformed cell lines representative of common European AHs ([Fig. 2](#fig2) and Table 3, which is published as supporting information on the PNAS web site). The sequenced 8.1, 7.1, and 18.2 AHs (The Wellcome Trust Sanger Institute MHC Haplotype Project, [www.sanger.ac.uk/HGP/Chr6/MHC/index.shtml](http://www.sanger.ac.uk/HGP/Chr6/MHC/index.shtml)) also were compared. The carriage of these markers was examined in the cases (*n* = 18) and controls (*n* = 12) recombinant for the 57.1 AH to determine the presence of these markers of the 57.1 AH and to map the extent of the haplotype within the patients ([Fig. 2](#fig2)). Three additional 57.1 AH recombinant abacavir-tolerant controls subsequently identified in the prospective cohort were included in [Table 2](#tbl2) but not in the fine mapping study ([Fig. 2](#fig2)). With reference to abacavir-hypersensitive individuals, the centromeric boundary of the putative susceptibility region was marked by the three SNPs present on the 57.1 AH and absent in three *HLA-B*5701*-positive, *C4A6*-negative recombinant cases (cases 15–17) suggestive of decay of the 57.1 AH centromeric of *Hsp70.2* in *HLA-B*5701*-negative individuals ([Fig. 2](#fig2)). One of these cases (case 16) carried the *HLA-B*5701* allele and the *Hsp70* region of the 57.1 AH but lacked the region from *snRNP* to *MEGT1*, expressing non-57.1 AH alleles at one SNP and two microsatellite loci in this region. The translated sequences of *Hsp70.1* and *Hsp70.2* were similar in abacavir-hypersensitive individuals (cases 15–17) and controls with recombinant 57.1 AHs (cases 19–22) and were therefore excluded as candidate susceptibility loci.

### Fig. 2.

![Fig. 2.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/245a/384715/2bb2b6b6f101/zpq0090440320002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=384715_zpq0090440320002.jpg)

Recombinant mapping identifies the Hsp70 subgenomic region in combination with HLA-B*5701 as the most parsimonious susceptibility region to abacavir hypersensitivity. The region carrying the putative susceptibility loci has been mapped to the MHC class III region on chromosome 6p21.3 (1). Typing of genetic markers within the C4A6–MEGT1 interval present on the 57.1 AH was carried out on abacavir-hypersensitive cases (n = 18) and abacavir-tolerant controls (n = 12) recombinant for the 57.1 AH. For each individual, shading indicates presence of the marker carried by the 57.1 AH.

The presence of previously reported nonsynonymous substitutions within genes in the region telomeric of the *Hsp70* cluster between *MEGT1* and *snRNP* was examined in the 57.1 AH homozygous cell line and compared with the MHC sequence of the 8.1, 7.1, and 18.2 AH sequences submitted to GenBank (Table 3). We did not identify unique nonsynonymous substitutions in this relatively nonpolymorphic interval. Hence, mapping of recombinant 57.1 AHs among abacavir-hypersensitive and -tolerant cases identified a candidate susceptibility region including the *Hsp70* gene cluster that narrowed the putative susceptibility region to the *Hsp70-Hom* locus. The only nucleotide substitution in the *Hsp70-Hom* gene identified in most of the abacavir-hypersensitive cases (94.4%) involvedaTtoC transition, which results in a change from methionine to threonine at amino acid residue 493 [hereafter referred to as *M493T* but also known as the *Hsp70-Hom 2437CT* allele ([19](#ref19)) or *rs2227956CT* ([www.ncbi.nlm.nih.gov/SNP/snp_ref.cgi?locusId=3305](http://www.ncbi.nlm.nih.gov/SNP/snp_ref.cgi?locusId=3305)]. The *Hsp70-Hom M493T* allele was assessed in all 248 patients of the cohort by using an allele-specific sequence-specific primer and/or the standard *Nco*I restriction fragment-length polymorphism assays. We observed that the *Hsp70-Hom M493T* allele, carried on the 57.1 AH, was detected in 94.4% of the hypersensitive group compared with 22.2% of tolerant controls (17 of 18 individuals vs. 51 of 230 individuals, OR = 59.7, *P*_c_ < 0.00001). However, in combination with *HLA-B*5701*, the *Hsp70-Hom M493T* allele was strongly associated with abacavir hypersensitivity (17 of 18 individuals vs. 1 of 230 individuals, OR = 3,893, *P*_c_ < 0.00001) ([Table 2](#tbl2)). Consistent with a role for both of these genetic markers in determining susceptibility to abacavir hypersensitivity, inclusion of the *Hsp70-Hom M493T*, in addition to the *HLA-B*5701*, allele provided significantly improved discrimination between abacavir-hypersensitive cases and -tolerant controls compared with the *HLA-B*5701* allele alone (*P* = 0.003, Fisher's exact test).

**Expression of Inflammatory Cytokine TNF in Abacavir-Hypersensitive Individuals.** Cytokine response was examined by measuring intracellular TNF levels in cultures of abacavir- or phorbol 12-myristate 13-acetate-stimulated PBMCs from abacavir-hypersensitive, -tolerant, and unexposed HIV-positive patients. The proportion of monocytes accumulating TNF was measured in a rapid (4 h) abacavir-stimulated *ex vivo*-cultured PBMC assay. The proportion of TNF-positive cells was higher in patients with abacavir hypersensitivity (*n* = 8) than in abacavir-tolerant controls (*n* = 9) with a median 13.4-fold increase [interquartile range (IQR) = 13.0] compared with a median 8.9-fold decrease in proportion of TNF-positive cells in tolerant controls (IQR = 24.1, *P* = 0.008, Mann–Whitney test) (data not shown). The three individuals classified as abacavir-tolerant on the basis of a negative epicutaneous patch test and an alternative explanation for their symptoms also had no increase in TNF-positive cells in response to abacavir stimulation.

**Phenotype of T Cells Expressing TNF After Abacavir Stimulation.** Further phenotypic characterization of the T cell response to abacavir was carried out on whole blood cultures of two patients who were selected for analysis because they both carried the predictive *HLA-B*5701* and *Hsp70-Hom M493T* alleles yet were divergent in their clinical responses to abacavir. Definite abacavir hypersensitivity occurred in case 17 within 1 week of drug exposure, whereas case 21 tolerated this drug without any clinical symptoms suggestive of hypersensitivity. Involvement of CD4^+^ and CD8^+^ T cells in the secretion of TNF in culture supernatants was determined after stimulation with abacavir. Extracellular levels of TNF were higher in abacavir-stimulated whole blood cultures of the abacavir-hypersensitive individual compared with those of the abacavir-tolerant control ([Fig. 3](#fig3)). In addition, TNF levels were attenuated in the abacavir-stimulated blood of this abacavirhypersensitive patient when CD8^+^ T cells were depleted, compared with undepleted or CD4^+^ T cell-depleted cultures, suggesting the involvement of MHC class I molecules and CD8^+^ T cells in the development of this immune reaction.

### Fig. 3.

![Fig. 3.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/245a/384715/986033a6f906/zpq0090440320005.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=384715_zpq0090440320005.jpg)

Abrogation of secreted TNF levels in supernatants of abacavir-stimulated whole blood cultures depleted of CD8+ T cells. Secreted TNF was measured by the Chemiluminescent IMMULITE (Diagnostic Products, Los Angeles) method in whole blood culture supernatants from an abacavir (ABC)-hypersensitive (HSR) and an abacavir-tolerant individual carrying HLA-B*5701 and Hsp70-Hom M493T alleles after stimulation with (filled bars, abacavir-hypersensitive; filled stippled bar, abacavir-tolerant) or without (open bars, abacavir-hypersensitive; open stippled bar, abacavir-tolerant) abacavir.

## Discussion

In this study we have used recombinant haplotype mapping to confirm a strong genetic association between the 57.1 AH and definite abacavir hypersensitivity, with further evidence that the concurrence of the *HLA-B*5701* allele and a haplotypic variant of the *Hsp70-Hom* allele represents a highly predictive marker of susceptibility. Through the application of a restrictive definition of definite hypersensitivity, this study is likely to have selected cases with a highly stereotypical immunological response to abacavir exposure, and it must be acknowledged that abacavir also was discontinued in nine individuals (3.6%) with minor symptoms that were not diagnostic of definite abacavir hypersensitivity. These data are consistent with another study involving a fully ascertained cohort of abacavir-exposed individuals in France (*n* = 331), in which the prevalence of likely hypersensitivity was 8.5%, whereas a further 4.2% ceased abacavir treatment within 6 months because of doubtful hypersensitivity reactions ([4](#ref4)).

Several lines of evidence strongly suggest the involvement of both *Hsp70-Hom* and *HLA-B*5701*. The centromeric boundary of the susceptibility region 3′ of *Hsp70.2* was demarcated by the presence of three non-57.1 AH alleles, one in each of the recombinant abacavir-hypersensitive individuals (cases 15–17). In addition, the telomeric end of the hypersensitivity region is flanked by *snRNP*,as an abacavir-hypersensitive individual (case 16) carried non-57.1 AH alleles between *snRNP* and *MEGT1* (identified by non-57.1 AH alleles at two microsatellites and one SNP loci). Additional nonsynonymous polymorphisms in genes within the *snRNP*–*MEGT1* region were not detected in the homozygous 57.1 AH cell line. A unique gene carried on the 57.1 AH was not identified in the *Hsp70.2–Hsp70-Hom* region after direct sequencing, and the identical translated sequences of *Hsp70.1* and *Hsp70.2* genes carried in the homozygous 57.1 AH cell line and by abacavir-tolerant controls carrying *HLA-B*5701* (cases 19–22) also eliminate these genes as possible candidates. However, our mapping studies are based on the presence of reported microsatellite and SNP analysis in a limited number of patients recombinant for the 57.1 AH. Despite our extensive search for informative SNPs by screening and direct sequencing, there may be an unidentified gene within the *HLA-B*5701–Hsp70-Hom M493T* interval that confers susceptibility to abacavir hypersensitivity. Moreover, there may be other determinants in addition to *HLA-B*5701* and *Hsp70-Hom M493T* required for the manifestation of this syndrome.

In this study, the presence of *HLA-B*5701* provides specificity, whereas the combination of *HLA-B*5701* and *Hsp70-Hom M493T* alleles, which are in strong LD in our abacavir-exposed cohort (LD = 0.030, *P* < 0.00001, relative LD = 0.82), significantly increased the ability to discriminate between hypersensitive cases and tolerant controls compared with *HLA-B*5701* testing alone; the combination occurred in 94.4% of patients and in only 0.4% of controls. Hence, in this study, susceptibility to abacavir hypersensitivity appears to be conferred by *Hsp70-Hom M493T* in concurrence with *HLA-B*5701* within the 57.1 AH. However, the small number of informative patients recombinant for the 57.1 AH necessitates further functional studies to confirm an interaction of the allelic products of the two loci. The involvement of *HLA-B*5701* in determining a MHC class I-restricted immune response to abacavir is in keeping with the CD8^+^ T cell-dependent production of TNF by mononuclear cells demonstrated in this study and with the prominent CD8^+^ cell infiltrate associated previously with epicutaneous patch testing ([14](#ref14)). In addition, patients with a higher CD8^+^ T cell count (>850 cells) at the time of exposure to abacavir have an increased risk of developing hypersensitivity ([20](#ref20)). *HLA-B*5701* is an effective antigen-presentation molecule and is associated with slow progression of HIV and induction of potent cytotoxic T lymphocyte responses ([21](#ref21)), and Hsp70 proteins are also known to bind peptides ([22](#ref22)) and to play a role in antigen presentation, particularly cross presentation of exogenous antigen to CD8^+^ T cells ([23](#ref23)–[28](#ref28)) and cytokine response ([29](#ref29)). In this context, the *Hsp70-Hom M493T* polymorphism is predicted to affect substrate binding, because residue 493 resides on the floor of the peptide-binding domain of the homologous Hsc70 molecule ([23](#ref23)). Determining the specific role of residue 493 in contributing to the specificity of peptide binding and/or antigen processing will require biochemical and structural analysis. However, it is tempting to speculate that *HLA-B*5701* and *Hsp70-Hom* cooperate during antigen presentation to confer susceptibility to abacavir hypersensitivity (see proposed model in [Fig. 4](#fig4)). Preliminary data suggest that indeed HLA-B57 and Hsp70 molecules colocalize within discrete vesicles in the CD14^+^ monocyte population of *ex vivo*-cultured abacavir-exposed PBMCs of abacavir-hypersensitive patients (A.M.M., C.A.A., and S.M., unpublished work).

### Fig. 4.

![Fig. 4.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/245a/384715/a1dafbba3109/zpq0090440320004.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=384715_zpq0090440320004.jpg)

Pathogenic model for the generation of a hypersensitive response to abacavir. Constitutively expressed (19) Hsp70-Hom molecules may facilitate loading of an abacavir haptenated peptide onto HLA-B*5701. Abacavir undergoes sequential intracellular phosphorylation to carbovir monophosphate (CBV-MP) and carbovir triphosphate (CBV-TP) and subsequently inhibits HIV reverse transcriptase (HIV-RT).

We hypothesize that abacavir or its metabolites may be involved in the haptenation of endogenous peptides and subsequent presentation of “altered self” in the context of *HLA-B*5701*, thus inducing vigorous T cell responses. It is noteworthy that Hsp70-associated peptides can furnish MHC class I-restricted determinants to class I molecules ([22](#ref22)–[28](#ref28)). It is therefore conceivable that Hsp70 plays a direct role in the selection of *HLA-B*5701*-restricted peptide substrates that are potentially haptenated through an abacavir-dependent mechanism. Haptenation of proteins and cellsurface peptides is well documented and can lead to MHCrestricted T cell immunity resembling auto- and alloimmune responses ([28](#ref28)). The involvement of HLA molecules in presentation of drugs and/or drug metabolites in the pathogenesis of a number of drug hypersensitivity reactions has been previously reported ([30](#ref30)–[33](#ref33)), although this is one of the few reports implicating HLA class I molecules in the development of drug hypersensitivity. In the case of abacavir, different chemical pathways could contribute to haptenation reactions, although the generation of a carboxylate derivative after oxidation of primary βγ alcohol ([34](#ref34)) may be implicated, as this reaction may proceed by means of a reactive aldehyde intermediate that could mediate covalent binding to proteins and peptides through a Schiff base-type reaction ([34](#ref34)) or 1,4 nucleophilic addition.

The presence of both *HLA-B*5701* and the *Hsp70-Hom M493T* variant in one abacavir-tolerant individual, who also demonstrated an absent immunological response to *ex vivo* abacavir stimulation, suggests that there may be other necessary determinants involved in the etiology of this syndrome. Further studies are needed to investigate the functional determinants of abacavir-specific antigen processing and presentation, which should clarify the pathways involved, and the role of modulating factors.

We also noted that, although increased intracellular TNF expression in response to abacavir stimulation occurred in most patients with a hypersensitivity reaction, two patients with definite hypersensitivity did not have detectable increases in TNF expression. This is unlikely to be attributable to clinical misclassification, because both patients had positive epicutaneous patch test results and one had previously developed a diagnostic syndrome on a second occasion after being rechallenged with abacavir. Therefore, measurement of the immunological response to *ex vivo* abacavir stimulation according to our method does not have sufficient sensitivity to be used as a basis for rechallenging a patient with abacavir in cases where a previous hypersensitivity reaction is suspected.

From a clinical perspective, these data indicate that the prospective use of a genetic test involving *HLA-B*5701* alone or in combination with the *Hsp70-Hom M493T* variant would reduce the prevalence of definite abacavir hypersensitivity in our cohort from 8% to 0.4%. When using *HLA-B*5701* alone, an estimated 1.6% of the tested population would be inappropriately denied access to abacavir. Testing for the presence of *HLA-B*5701* and the *Hsp70-Hom M493T* variant would reduce this percentage to 0.4%, based on our data. Given the *HLA-B*5701* phenotype frequency of 8.5% in our cohort, we can predict that ≈14 individuals would need to be screened for *HLA-B*5701* to prevent one case of abacavir hypersensitivity. Taken together, these data indicate that prospective genetic testing for abacavir hypersensitivity is likely to be highly predictive, and, at least in populations of European descent, in which the *HLA-B*5701* allele is relatively common, costeffective ([35](#ref35)).

In conclusion, we have identified the *HLA-B*5701* and *Hsp70-Hom M493T* alleles as highly predictive genetic markers of susceptibility to abacavir hypersensitivity. These findings have significant implications both in terms of the clinical management of abacavir-exposed HIV-infected patients and the elucidation of basic pathophysiological mechanisms underlying this and other idiosyncratic drug hypersensitivity reactions.

## Supplementary Material

## Acknowledgments

We thank the participants and clinical staff of the Western Australian HIV Cohort Study; E. Freitas, M. Stoklasa, A. Pretzelj, and B. Downsborough for DNA sequencing; D. Sayer, C. Witt, and members of the Department of Clinical Immunology and Biochemical Genetics for HLA typing; C. Witt for calculation of LD; and J. Trowsdale, P. Price, S. Temple, S. Hermann, and N. Keane for helpful comments. This work was supported by Australian National Health and Medical Research Council Project Grant 237408. S.G. is supported by a Healy Postdoctoral Fellowship from the Raine Medical Research Foundation.

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Mallal, S., Nolan, D., Witt, C., Masel, G., Martin, A. M., Moore, C., Sayer, D., Castley, A., Mamotte, C., Maxwell, D., et al. (2002) Lancet 359, 727-732.  [DOI](https://doi.org/10.1016/s0140-6736(02)07873-x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11888582/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&volume=359&publication_year=2002&pages=727&pmid=11888582&doi=10.1016/s0140-6736(02)07873-x&)

2. Hetherington, S., Hughes, A. R., Mostelle, M., Shortino, D., Baker, K. L., Spreen, W., Lai, E., Davies, K., Handley, A., Dow, D. J., et al. (2002) Lancet 359, 1121-1122.  [DOI](https://doi.org/10.1016/S0140-6736(02)08158-8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11943262/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&volume=359&publication_year=2002&pages=1121&pmid=11943262&doi=10.1016/S0140-6736(02)08158-8&)

3. Hetherington, S., McGuirk, S., Powell, G., Cutrell, A., Naderer, O., Spreen, B., Lafon, S., Pearce, G. & Steel, H. (2001) Clin. Ther. 23, 1603-1614.  [DOI](https://doi.org/10.1016/s0149-2918(01)80132-6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11726000/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Ther.&volume=23&publication_year=2001&pages=1603&pmid=11726000&doi=10.1016/s0149-2918(01)80132-6&)

4. Peyriere, H., Guillemin, V., Lotthe, A., Baillat, V., Fabre, J., Favier, C., Atoui, N., Hansel, S., Hillaire-Buys, D. & Reynes, J. (2003) Ann. Pharmacother. 37, 1392-1397.  [DOI](https://doi.org/10.1345/aph.1C523) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14519040/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann.%20Pharmacother.&volume=37&publication_year=2003&pages=1392&pmid=14519040&doi=10.1345/aph.1C523&)

5. Escaut, L., Liotier, J. Y., Albengres, E., Cheminot, N. & Vittecoq, D. (1999) AIDS 13, 1419-1420.  [DOI](https://doi.org/10.1097/00002030-199907300-00026) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10449301/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=AIDS&volume=13&publication_year=1999&pages=1419&pmid=10449301&doi=10.1097/00002030-199907300-00026&)

6. Symonds, W., Cutrell, A., Edwards, M., Steel, H., Spreen, B., Powell, G., McGuirk, S. & Hetherington, S. (2002) Clin. Ther. 24, 565-573.  [DOI](https://doi.org/10.1016/s0149-2918(02)85132-3) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12017401/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Ther.&volume=24&publication_year=2002&pages=565&pmid=12017401&doi=10.1016/s0149-2918(02)85132-3&)

7. Peyriere, H., Nicolas, J., Siffert, M., Demoly, P., Hillaire-Buys, D. & Reynes, J. (2001) Ann. Pharmacother. 35, 1291-1292.  [DOI](https://doi.org/10.1345/aph.1A022) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11675863/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann.%20Pharmacother.&volume=35&publication_year=2001&pages=1291&pmid=11675863&doi=10.1345/aph.1A022&)

8. Martin, A. M., Gaudieri, S., Witt, C., Sayer, D., Castley, A., Mamotte, C., Nolan, D., James, I., Christiansen, F. T. & Mallal, S. (2004) in HLA 2004: Immunobiology of the Human MHC, Proc. of the 13th IHWS, eds. Hanson, J. A. & Dupont, B. (IHWG Press, Seattle), in press.

9. Gabriel, S. B., Schaffner, S. F., Nguyen, H., Moore, J. M., Roy, J., Blumenstiel, B., Higgins, J., DeFelice, M., Lochner, A., Faggart, M., et al. (2002) Science 296, 2225-2229.  [DOI](https://doi.org/10.1126/science.1069424) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12029063/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&volume=296&publication_year=2002&pages=2225&pmid=12029063&doi=10.1126/science.1069424&)

10. Ahmad, T., Neville, M., Marshall, S. E., Armuzzi, A., Mulcahy-Hawes, K., Crawshaw, J., Sato, H., Ling, K. L., Barnardo, M., Goldthorpe, S., et al. (2003) Hum. Mol. Genet. 12, 647-656.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/12620970/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum.%20Mol.%20Genet.&volume=12&publication_year=2003&pages=647&pmid=12620970&)

11. Mizuki, N., Ota, M., Kimura, M., Ohno, S., Ando, H., Katsuyama, Y., Yamazaki, M., Watanabe, K., Goto, K., Nakamura, S., et al. (1997) Proc. Natl. Acad. Sci. USA 94, 1298-1303.  [DOI](https://doi.org/10.1073/pnas.94.4.1298) | [PMC free article](/articles/PMC19785/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9037047/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&volume=94&publication_year=1997&pages=1298&pmid=9037047&doi=10.1073/pnas.94.4.1298&)

12. Sebastiani, P., Lazarus, R., Weiss, S. T., Kunkel, L. M., Kohane, I. S. & Ramoni, M. F. (2003) Proc. Natl. Acad. Sci. USA 100, 9900-9905.  [DOI](https://doi.org/10.1073/pnas.1633613100) | [PMC free article](/articles/PMC187880/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12900503/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&volume=100&publication_year=2003&pages=9900&pmid=12900503&doi=10.1073/pnas.1633613100&)

13. Degli-Esposti, M. A., Andreas, A., Christiansen, F. T., Schalke, B., Albert, E. & Dawkins, R. L. (1992) Immunogenetics 35, 355-364.  [DOI](https://doi.org/10.1007/BF00179791) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/1349584/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Immunogenetics&volume=35&publication_year=1992&pages=355&pmid=1349584&doi=10.1007/BF00179791&)

14. Phillips, E. J., Sullivan, J. R., Knowles, S. R. & Shear, N. H. (2002) AIDS 16, 2223-2225.  [DOI](https://doi.org/10.1097/00002030-200211080-00017) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12409746/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=AIDS&volume=16&publication_year=2002&pages=2223&pmid=12409746&doi=10.1097/00002030-200211080-00017&)

15. Temple, S. E., Cheong, K. Y., Almeida, C. M., Price, P. & Waterer, G. W. (2003) Genes Immun. 4, 283-288.  [DOI](https://doi.org/10.1038/sj.gene.6363963) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12761565/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genes%20Immun.&volume=4&publication_year=2003&pages=283&pmid=12761565&doi=10.1038/sj.gene.6363963&)

16. Haldane, J. B. S. (1956) Ann. Hum. Genet. 20, 309-311.  [DOI](https://doi.org/10.1111/j.1469-1809.1955.tb01285.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/13314400/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann.%20Hum.%20Genet.&volume=20&publication_year=1956&pages=309&pmid=13314400&doi=10.1111/j.1469-1809.1955.tb01285.x&)

17. Cavalli-Sforza, L. L. & Bodmer, W. F., eds. (1971) The Genetics of Human Populations (Freeman, San Francisco), p. 256.

18. Baur, M. P. & Danilov, J. A. (1980) in Histocompatability Testing, ed., Terasaki, P. I. (UCLA Tissue Typing Lab., Los Angeles), p. 955-957.

19. Milner, C. M. & Campbell, R. D. (1990) Immunogenetics 32, 242-251.  [DOI](https://doi.org/10.1007/BF00187095) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/1700760/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Immunogenetics&volume=32&publication_year=1990&pages=242&pmid=1700760&doi=10.1007/BF00187095&)

20. Easterbrook, P. J., Waters, A., Murad, S., Ives, N., Taylor, C., King, D., Vyakarnam, A. & Thorburn, D. (2003) HIV Med. 4, 321-324.  [DOI](https://doi.org/10.1046/j.1468-1293.2003.00166.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14525543/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=HIV%20Med.&volume=4&publication_year=2003&pages=321&pmid=14525543&doi=10.1046/j.1468-1293.2003.00166.x&)

21. Klein, M. R., van der Burg, S. H., Hovenkamp, E., Holwerda, A. M., Drijfhout, J. W., Melief, C. J. & Miedema, F. (1998) J. Gen. Virol. 79, 2191-2201.  [DOI](https://doi.org/10.1099/0022-1317-79-9-2191) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9747728/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Gen.%20Virol.&volume=79&publication_year=1998&pages=2191&pmid=9747728&doi=10.1099/0022-1317-79-9-2191&)

22. Fourie, A. M., Sambrook, J. F. & Gething, M. J. (1994) J. Biol. Chem. 269, 30470-30478.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/7982963/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem.&volume=269&publication_year=1994&pages=30470&pmid=7982963&)

23. Morshauser, R. C., Hu, W., Wang, H., Pang, Y., Flynn, G. C. & Zuiderweg, E. R. (1999) J. Mol. Biol. 289, 1387-1403.  [DOI](https://doi.org/10.1006/jmbi.1999.2776) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10373374/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Mol.%20Biol.&volume=289&publication_year=1999&pages=1387&pmid=10373374&doi=10.1006/jmbi.1999.2776&)

24. Castellino, F., Boucher, P. E., Eichelberg, K., Mayhew, M., Rothman, J. E., Houghton, A. N. & Germain, R. N. (2000) J. Exp. Med. 191, 1957-1964.  [DOI](https://doi.org/10.1084/jem.191.11.1957) | [PMC free article](/articles/PMC2213527/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10839810/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Exp.%20Med.&volume=191&publication_year=2000&pages=1957&pmid=10839810&doi=10.1084/jem.191.11.1957&)

25. Ishii, T., Udono, H., Yamano, T., Ohta, H., Uenaka, A., Ono, T., Hizuta, A., Tanaka, N., Srivastava, P. K. & Nakayama, E. (1999) J. Immunol. 162, 1303-1309.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/9973383/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Immunol.&volume=162&publication_year=1999&pages=1303&pmid=9973383&)

26. Binder, R. J., Blachere, N. E. & Srivastava, P. K. (2001) J. Biol. Chem. 276, 17163-17171.  [DOI](https://doi.org/10.1074/jbc.M011547200) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11278929/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem.&volume=276&publication_year=2001&pages=17163&pmid=11278929&doi=10.1074/jbc.M011547200&)

27. Blachere, N. E., Li, Z., Chandawarkar, R. Y., Suto, R., Jaikaria, N. S., Basu, S., Udono, H. & Srivastava, P. K. (1997) J. Exp. Med. 186, 1315-1322.  [DOI](https://doi.org/10.1084/jem.186.8.1315) | [PMC free article](/articles/PMC2199095/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9334371/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Exp.%20Med.&volume=186&publication_year=1997&pages=1315&pmid=9334371&doi=10.1084/jem.186.8.1315&)

28. Srivastava, P. K., Udono, H., Blachere, N. E. & Li, Z. (1994) Immunogenetics 39, 93-98.  [DOI](https://doi.org/10.1007/BF00188611) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8276462/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Immunogenetics&volume=39&publication_year=1994&pages=93&pmid=8276462&doi=10.1007/BF00188611&)

29. Schroder, O., Schulte, K. M., Ostermann, P., Roher, H. D., Ekkernkamp, A. & Laun, R. A. (2003) Crit. Care Med. 31, 73-79, and erratum (2003) 31, 1296.  [DOI](https://doi.org/10.1097/00003246-200301000-00011) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12544996/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Crit.%20Care%20Med.&volume=31&publication_year=2003&pages=73&pmid=12544996&doi=10.1097/00003246-200301000-00011&)

30. Britschgi, M., von Greyerz, S., Burkhart, C. & Pichler, W. J. (2003) Current Drug Targets 4, 1-11.  [DOI](https://doi.org/10.2174/1389450033347082) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12528985/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Current%20Drug%20Targets&volume=4&publication_year=2003&pages=1&pmid=12528985&doi=10.2174/1389450033347082&)

31. Park, B. K., Naisbitt, D. J., Gordon, S. F., Kitteringham, N. R. & Pirmohamed, M. (2001) Toxicology 158, 11-23.  [DOI](https://doi.org/10.1016/s0300-483x(00)00397-8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11164988/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Toxicology&volume=158&publication_year=2001&pages=11&pmid=11164988&doi=10.1016/s0300-483x(00)00397-8&)

32. Zanni, M. P., von Greyerz, S., Schnyder, B., Brander, K. A., Frutig, K., Hari, Y., Valitutti, S. & Pichler, W. J. (1998) J. Clin. Invest. 102, 1591-1598.  [DOI](https://doi.org/10.1172/JCI3544) | [PMC free article](/articles/PMC509010/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9788973/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Invest.&volume=102&publication_year=1998&pages=1591&pmid=9788973&doi=10.1172/JCI3544&)

33. Schnyder, B., Burkhart, C., Schnyder-Frutig, K., von Greyerz, S., Naisbitt, D. J., Pirmohamed, M., Park, B. K. & Pichler, W. J. (2000) J. Immunol. 164, 6647-6654.  [DOI](https://doi.org/10.4049/jimmunol.164.12.6647) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10843725/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Immunol.&volume=164&publication_year=2000&pages=6647&pmid=10843725&doi=10.4049/jimmunol.164.12.6647&)

34. Walsh, J. S., Reese, M. J. & Thurmond, L. M. (2002) Chem. Biol. Interact. 142, 135-154.  [DOI](https://doi.org/10.1016/s0009-2797(02)00059-5) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12399160/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Chem.%20Biol.%20Interact.&volume=142&publication_year=2002&pages=135&pmid=12399160&doi=10.1016/s0009-2797(02)00059-5&)

35. Nolan, D., Gaudieri, S. & Mallal, S. (2003) J. HIV Ther. 8, 36-41.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/12838163/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20HIV%20Ther.&volume=8&publication_year=2003&pages=36&pmid=12838163&)
