# A genome-wide association study of bronchodilator response in asthmatics

## Metadata
**Authors:** Qing Ling Duan, Jessica Lasky-Su, Blanca E Himes, Weiliang Qiu, Augusto A Litonjua, Amy Damask, Ross Lazarus, Barbara Klanderman, Charles G Irvin, Stephen P Peters, John P Hanrahan, John J Lima, Fernando D Martinez, David Mauger, Vernon M Chinchilli, Manuel Soto-Quiros, Lydiana Avila, Juan C Celedón, Christoph Lange, Scott T Weiss, Kelan G Tantisira
**Journal:** The pharmacogenomics journal
**Date:** 2013 Mar 19
**DOI:** [10.1038/tpj.2013.5](https://doi.org/10.1038/tpj.2013.5)
**PMID:** 23508266
**PMCID:** PMC3706515
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3706515/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC3706515/pdf/nihms442409.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC3706515/pdf/nihms442409.pdf)

## Abstract

Reversibility of airway obstruction in response to β2-agonists is highly variable among asthmatics, which is partially attributed to genetic factors. In a genome-wide association study of acute bronchodilator response (BDR) to inhaled albuterol, 534,290 single nucleotide polymorphisms (SNPs) were tested in 403 white trios from the Childhood Asthma Management Program using five statistical models to determine the most robust genetic associations. The primary replication phase included 1397 polymorphisms in three asthma trials (pooled n=764). The second replication phase tested 13 SNPs in three additional asthma populations (n=241, n=215, and n=592). An intergenic SNP on chromosome 10, rs11252394, proximal to several excellent biological candidates, significantly replicated (p=1.98×10−7) in the primary replication trials. An intronic SNP (rs6988229) in the collagen (COL22A1) locus also provided strong replication signals (p=8.51×10−6). This study applied a robust approach for testing the genetic basis of BDR and identified novel loci associated with this drug response in asthmatics.

Keywords: pharmacogenetics, asthma, bronchodilator response, genome-wide association study, albuterol

## Introduction

Asthma is a complex respiratory disease characterized by hyper-responsiveness of the bronchial muscles, chronic inflammation and reversible narrowing of the airways. It affects approximately 300 million individuals worldwide and its prevalence is expected to increase to 400 million by 2025.^1^ Asthma is the most common chronic illness in children,^2,3^ accounting for half a million hospitalizations a year in the United States. In 2007, asthma related health care costs in the US were estimated to be $56 billion, with the majority attributed to medications and hospitalizations.^4^ Taken together, asthma has a significant public health impact and steps towards its prevention or better management will decrease the overall disease burden.

β_2_-agonists are the most commonly used drugs for treating asthma.^2^ The therapeutic effects result from binding to the transmembrane *β*_2_-adrenergic receptor (*β*_2_-AR) located on airway smooth muscle cells to relieve bronchoconstriction. These are available as short-acting β_2_-agonists (SABA; e.g. albuterol) for rescuing acute asthma symptoms or as long-acting β_2_-agonists (LABA; e.g. salmeterol and formoterol) for controlling chronic asthma that is usually administered in combination with an inhaled corticosteroid. The reversibility of airway obstruction in response to these medications, known as bronchodilator response (BDR), may be measured as a change in lung function (forced expiratory volume in one second (FEV_1_)) or as fall in peak expiratory flow rate (PEFR), indicating a down-regulation of *B*_2_-agonist responsivity (tachyphylaxis) with prolonged drug use. Inter-individual variability in response to these drugs have been previously described and research suggests that genetic variants are major contributing factors.^3^ The identification of genetic loci associated with BDR to β_2_-agonists will help to facilitate personalized asthma treatment regimens.

Pharmacogenetic investigations of BDR have identified a number of genetic associations for this variable drug response. The majority had been candidate gene studies, which reported genetic associations to SNPs and/or haplotypes in the arginase 1 (*ARG1*) locus,^5^ the β_2_-adrenergic receptor (*ADRB2*) gene,^6–9^ the corticotropin-releasing hormone receptor (*CRHR*)-2 locus,^10^ and the adenylyl cyclase type 9 (*AC9*) gene.^11^ A recent genome-wide association study (GWAS) of BDR by our group identified a functional variant in the serine-rich 2-like (*SPATS2L*) gene, albeit the mechanism by which it regulates BDR remains unknown.^12^ In this manuscript, we expand on the previous literature by using a novel approach to identify genetic associations with BDR (defined by a change in lung function) whereby we apply five statistical models in a GWAS of this drug response phenotype to decrease the likelihood of false positive associations. Novel aspects of the current GWAS include use of genetic data from the parents of asthmatics in a family-based test, which is more robust against population stratification, as well as analysis of 11 BDR measures for each subject taken over a four year period in addition to BDR at randomization (taken upon entry into the clinical trial). Moreover, we considered both additive and recessive transmissions of the associated alleles. We then pooled the results from these multiple genome-wide analytical models to identify common genetic association signals to carry forward for replication analysis in additional asthma populations. This manuscript describes the findings of our innovative GWAS of BDR in asthmatic subjects.

## Methods

### Asthma Trial Populations

The asthma trial populations are summarized in [Table 1](#T1) and details are available in the [Supplemental Material](#SD1). All patients or their legal guardians consented to each study protocol and ancillary genetic testing. All studies were approved by the respective Institutional Review Boards and/or Ethics Committees of the participating institutions.

### Table 1.

|   | CAMP | CAMP (Placebo) | AT | LOCCS | LODO | AT/LOCCS/LODO | CARE | ACRN | GACRS |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
|   | n=403 | n=171 | n=444 | n=165 | n=155 | n = 764 | n=215 | n=241 | n=592 |
| Age, mean (sd) | 8.8 (2.1) | 8.7 (2.1) | 32.4 (13.6) | 34.4 (15.3) | 42.9 (14.7) | 34.9 (14.8) | 10.6 (2.9) | 31.7 (42.2) | 9.0 (1.8) |
| Range | 5.2–13.2 | 5.2 – 13.2 | 12.0 – 80 | 7 – 71 | 15 – 76 | 7 – 80 | 6–17.8 | 12.4–63.7 | 6.0–14.2 |
| Gender, n(%) |   |   |   |   |   |   |   |   |   |
| male | 254 (63) | 109 (60) | 222 (50.0) | 58 (35.2) | 39 (25.2) | 319 (41.8) | 132 (61.4) | 100 (41.49) | 351 (59.3) |
| Wash-out prior to BDR test*, weeks | 4 | 4 | 6 | 4–6 (fluticasone) | 2 | 2–6 | 0–4 | 0–6 | 4 |
| Albuterol puffs (90ug/puff) | 2 | 2 | 2 | 2 | 2 | 2 | 4 | 2–4 | 2 |
| pre-BD FEV1 pp, mean (sd) | 93.4(14.0) | 94.7(13.3) | 61.5 (6.8) | 84.3 (12.3) | 78.8 (17.7) | 69.8 (14.7) | 99.3 (12.6) | 85.9 (13.5) | 99.8 (17.2) |
| BDR, mean (sd) | 11(10) | 12 (11) | 40.15 (20.9) | 6.4 (6.1) | 9.7 (11.1) | 26.7 (23.2) | 9.5 (8.4) | 11.6 (21.8) | 5.7 (9.2) |

Table 1 Caption: Baseline Characteristics of Participants in the Asthma Populations used in this Analysis.

#### Initial GWAS Population

A total of 403 non-Hispanic white asthmatic children and their parents from the Childhood Asthma Management Program (CAMP)^13,14^ were successfully genotyped on the Illumina HumanHap550v3 BeadChip (San Diego, CA).^15^ BDR at randomization were conducted for each proband upon entry into the trial following 2 inhalations of albuterol. A total of 11 longitudinal BDR values were measured in a subset of 171 asthmatics randomized to inhaled albuterol therapy as needed over four years of this clinical trial. Genome-wide association analysis included 534,290 autosomal SNPs that had passed quality control metrics (see [Supplemental Material](#SD1)).

#### Primary Replication Populations

A total of 1536 SNPs were selected for genotyping, of which 1397 were successful, in three non-Hispanic white adult asthma trials (pooled n=764) using the Illumina GoldenGate Custom Array (Illumina Inc., San Diego, CA). SNP selection criteria are detailed below (**Statistical Methodology**). These replication populations included: 1) the Asthma Trial (AT, n=444)^16,17^; 2) the Leukotriene modifier or Corticosteroid or Corticosteroid Salmeterol (LOCCS, n=165) trial^18^; and 3) the Effectiveness of Low Dose Theophylline as Add-on Treatment in Asthma (LODO, n=155) trial.^19^ BDR at randomization was conducted for each subject upon entry into these clinical trials.

#### Secondary Replication Populations

A total of 13 SNPs with one-sided p-values < 0.05 (based on the direction of association in CAMP) in the primary replication analysis were further tested in two additional asthma trials: 1) the Childhood Asthma Research and Education Network (CARE, n=215) and 2) the Asthma Clinical Research Network (ACRN, n=241).^20^ As these individuals had been genotyped on the Affymetrix Genome-Wide Human SNP Array 6.0 (Santa Clara, CA), imputed data was used that was generated for the HapMap Phase 2 Release 22 SNPs^21^ by applying the Markov Chain Haplotyping (MaCH) software.^22^ Finally, eight of these 13 SNPs were further tested in the Genetics of Asthma in Costa Rica Study (GACRS), which were successfully genotyped on the HumanOmniExpress-12v1_A chip.^23^ BDR at randomization was conducted for each subject upon entry into these clinical trials.

### Statistical Methodology

The primary outcome measure of all analyses was BDR to the inhaled s_2_-agonist albuterol, which was calculated as the percent change in forced expiratory volume in one second (FEV_1_): BDR=100 × [(postFEV_1_-preFEV_1_)/preFEV_1_], where preFEV1 is the lung function before albuterol treatment (baseline) and postFEV1 is the lung function following albuterol treatment. The overall analysis strategy is presented in [Figure 1](#F1). To compensate for the limited statistical power given the small sample size of the CAMP trial, we used five statistical models to identify the most robust genetic associations: generalized linear model of BDR in 403 probands, using recessive (1) and additive (2) models; mixed model of 11 repeated measures of BDR over four years in 171 individuals randomized to as-needed inhaled β_2_-agonist, using recessive (3) and additive (4) models; and family-based association test (FBAT) of BDR at randomization in 403 CAMP parent-offspring trios (5). All models were adjusted for age, sex, and baseline preFEV_1_ and model 5 was additionally adjusted for height. Each SNP was given a score of 0 to 5 based on the total number of p-values below 0.05 from all five association tests. All SNPs scoring 5 (n = 437) were carried forward for genotyping in the primary replication cohort but those scoring 4 were then ranked according to their p-values from the FBAT analysis as this model is robust against population stratification. No SNPs scoring below 4 were included for replication. All tests using generalized linear (additive and recessive) models were performed in PLINK ([http://pngu.mgh.harvard.edu/purcell/plink/](http://pngu.mgh.harvard.edu/purcell/plink/))^24^ and included SNPs with minor allele frequencies (MAF) ≥ 0.05. FBAT applied a pedigree-based analysis tool (PBAT) previously described.^25^ All replication analyses used a single measure of BDR at randomization, with adjustments for age, sex, height and baseline preFEV_1_. Multiple comparisons were adjusted using the Liptak weighted Z method.^26^ Additional details are available in the [Supplemental Material](#SD1).

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1c95/3706515/859d8a5e040f/nihms442409f1.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3706515_nihms442409f1.jpg)

An overview of the genome-wide analyses methods and replication strategies used. The initial GWAS in CAMP applied five statistical models (linear regression of BDR at randomization in 403 asthmatics using additive and recessive models, longitudinal mixed models of 11 repeated BDR measures in 171 probands using additive and recessive models, and a family-based association test of BDR at randomization in 403 trios). A total of 1536 SNPs providing p-values < 0.05 from five or four of these models (the latter rankd by FBAT p-values) were selected for genotyping and replication in LOCCS/LODO/AT (n=764). The 13 replicated SNPs (one sided p-values < 0.05) were further tested in ACRN, CARE and GACRS.

### Expression Quantitative Trait Analysis

Microarray data from immortalized lymphoblastoid cell lines of 117 asthmatics (non-Hispanic white CAMP subjects), spotted on the Illumina HumanRef8v2 microarray BeadChips, were used to test the correlation of genetic variants with gene expression. These cells were cultured and treated with ethanol (sham) as a control for differential analysis with corticosteroid (dexamethasone) treated cells for a separate pharmacogenetic investigation (unpublished data). The microarray data from the sham arm of this experiment was vst-transformed and quantile normalized using the Lumi package in Bioconductor.^27^ A cis-expression quantitative trait locus (eQTL) was defined as a SNP that was correlated with the expression of a gene within 50 kb. A trans-eQTL was a SNP correlation with a transcript located more than 50 kb away or on a separate chromosome entirely.

## Results

The baseline characteristics of all asthma populations including CAMP, the three primary replication trials and the three secondary replication populations are shown in [Table 1](#T1). Whereas the initial GWAS using CAMP consisted of childhood asthmatics, the replication populations included both childhood (CARE and GACRS) and adult asthmatics (pooled AT/LOCCS/LODO and ACRN). It is also notable that the adult asthma populations had fewer males and lower pre-bronchodilator FEV_1_ percent predicted (Pre-BD FEV_1_pp) values, which was previously correlated with higher BDR.^28^ To compensate for the variability in baseline preFEV_1_ across populations, we adjusted each association test for this variable in addition to accounting for it in our phenotype definition (BDR=100×[(postFEV_1_-preFEV_1_)/preFEV_1_]).

### Genome-wide analysis in CAMP

A plot of the –Log_10_(p-values) against the chromosomal location of each SNP from the family-based association (FBAT) analysis is shown in [Figure 2](#F2). A quantile-quantile plot of the expected p-values of the FBAT analysis under the null hypothesis and the actual observed p-values illustrates that the majority of p-values were greater than expected by chance, suggesting that the test was conservative [[Supplemental Figure 1](#SD2)]. However, there are several p-values less than what was expected by chance. For example, the lowest p-value was 5.28×10^−7^ for rs8112048 located 3′ of the zinc finger protein 14 (*ZNF14*) gene but this did not meet genome-wide significance. In addition, we noted that many SNPs in previously implicated genes (*ARG1, ADRB2, CRHR-2,* and *AC9)*^5–11^ were absent from our GWAS due to differences in genotyping platforms. Of the four markers included in our GWAS (rs1042713 in *ADRB2*, rs4723002 and rs226716 in *CRHR2*, and rs2230739 in *AC9*), nominal association was found for rs1042713 in *ADRB2* (p < 0.02), which is the most investigated locus for BDR. Finally, the genomic inflation factor estimate was 1.01, demonstrating minimal population stratification.

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1c95/3706515/73fbf66a7706/nihms442409f2.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3706515_nihms442409f2.jpg)

The distribution of BDR at randomization across all asthma trial populations. BDR is defined as a percent change in lung function (FEV1) in response to inhaled albuterol across all asthma trial populations.

### Replication Analyses

Data for the 1397 replication SNPs from the three adult asthma trials were pooled for analysis to maximize the statistical power for detecting associations. A total of 13 SNPs replicated in the same direction as the initial GWAS population (CAMP) and were carried forward for analysis in the secondary replication phase ([Table 2](#T2)). The intergenic SNP, rs11252394, with a p-value of 0.0099 (beta = 3.1) from the additive model in CAMP, had a one-sided p-value of 1.21×10^−6^ in the primary replication phase, which remained significant following Bonferroni correction for multiple comparisons. However, this SNP did not replicate in the secondary replication phase. Next, nominal association signals (p-values < 0.05) were derived for an intronic SNP, rs6988229, in the collagen type XXII alpha 1 (*COL22A1*) gene in CAMP (recessive p-value = 0.004, beta = 3.26). This SNP further replicated across all asthma populations except for CARE (Liptak combined p = 8.51E-06). Finally, five additional SNPs showed marginal association (p < 0.05) in the primary replication and one of the three secondary replication populations: rs166330, rs166332, rs17495520, rs6002674, and rs1522113. The latter marker (additive p-value = 0.014 and beta = 3.23 in CAMP), is located in intron 8 of *CLOCK* and in perfect linkage disequilibrium (correlation coefficient (r^2^) of 1.0 in CAMP) with a non-synonymous variant (rs34897046; Serine208Cysteine (S208C)) in exon 9 of the same gene.^29^ The top 13 SNPs explain 23.8% of the overall genetic variance in BDR, based on the correlation coefficient for each analysis. This calculation assumed that the genetic contribution of each SNP is independent of the other genetic associations.

### Table 2.

|   |   |   |   |   |   |   |   |  |  |  |   |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
|   |   |   |   |   |   |   | Replication #1 | Replication #2 |  |  |   |
|   |   |   |   |   |   |   |   |  |  |  |   |
| SNP | Chr | MA | MAF | Gene | Model | CAMP GWAS (effect estimate) | Pooled AT/LOCCS/LODO | ACRN | CARE | GACRS | Liptak Combined P val |
| rs11252394 | 10 | A | 0.08 |   | Additive | 0.0099b (+) | 1.21E-06 | 0.4173 | 0.3472 | - | 1.98E-07 |
| rs6988229 | 8 | T | 0.20 | COL22A1 | Recessive | 0.0004c (+) | 0.0050 | 0.0505 | 0.9283 | 0.0139 | 8.51E-06 |
| rs9552679 | 13 | C | 0.26 |   | Additive | 0.0007d (−) | 0.0004 | 0.7884 | 0.4410 | - | 3.02E-05 |
| rs1663330 | 14 | G | 0.33 |   | Additive | 0.0020b (+) | 0.0028 | 0.0234 | 0.7544 | - | 4.54E-05 |
| rs1663332 | 14 | T | 0.37 |   | Additive | 0.0028b(+) | 0.0006 | 0.0374 | 0.6874 | 0.3595 | 8.07E-05 |
| rs17495520 | 5 | T | 0.14 |   | Additive | 0.038b (+) | 0.0110 | 0.0030 | 0.3587 | - | 0.0003 |
| rs10511905 | 9 | G | 0.22 |   | Additive | 0.006a (−) | 0.0087 | 0.1388 | 0.3149 | - | 0.0003 |
| rs518350 | 22 | T | 0.12 |   | Recessive | 0.0065e (+) | 0.0010 | 0.4987 | 0.2090 | 0.3334 | 2.70E-04 |
| rs17701271 | 4 | A | 0.22 |   | Recessive | 0.004c (+) | 0.0026 | 0.1371 | 0.8756 | 0.2031 | 0.0003 |
| rs6002674 | 22 | C | 0.15 |   | Recessive | 0.0012e (−) | 0.0040 | 0.9808 | 0.0310 | 0.5675 | 0.0027 |
| rs1419555 | 7 | T | 0.37 |   | Additive | 0.0043d (+) | 0.0060 | 0.7723 | 0.6879 | 0.4932 | 0.0068 |
| rs1423515 | 5 | A | 0.05 |   | Additive | 0.027b (+) | 0.0400 | 0.4104 | 0.4710 | - | 0.0089 |
| rs1522113 | 4 | A | 0.05 | CLOCK | Additive | 0.014b (+) | 0.0037 | 0.9260 | 0.0427 | 0.8633 | 0.0177 |

Table 2 Caption: Summary of GWAS and replication analyses in all asthma clinical trials.

Analysis of microarray data from lymphoblastoid cell lines from a subset of CAMP subjects determined that the missense variant in *CLOCK* is associated with variable gene expression of both *CLOCK* (p-value = 0.05) and one of its downstream effectors Period 2 gene (*PER2,* p-value = 0.003) [[Supplemental Figure 2](#SD3)]. Individuals with one mutant allele (CG genotype, n = 20) had greater expression of both *CLOCK* and *PER2* compared to individuals without this minor allele (GG genotype, n = 94). The SNP rs6988229 in the *COL22A1* locus on the other hand did not demonstrate any cis-regulatory effects, however, it is correlated with the expression of multiple other genes (trans-acting effects on gene expression). This includes another member of the G protein-coupled receptor superfamily (*GPR110*). The top five trans-effects of each of the 13 SNPs from [Table 2](#T2) are shown in [Supplemental Table 1](#SD1). These results did not suggest a regulatory role for the intergenic SNP on chromosome 10 (rs11252394). While these associations with gene expression suggest functional effects of some of our associated polymorphisms, further investigation is necessary to validate their functional effects and the mechanism by which they might regulate BDR.

## Discussion

This manuscript describes a comprehensive GWAS of treatment response to β_2_-agonists in asthmatics, which identifies novel pharmacogenetic loci associated with clinical response variability. Due to the limited size of the asthma drug trial populations, which is common in pharmacogenetic investigations, we implemented a novel strategy to select SNPs for replication. Specifically, we prioritized SNPs by evaluating p-values from 5 different statistical models, thereby taking advantage of the longitudinal nature of the phenotypic data, the entire sample at randomization, as well as the genotype data from the parents. SNPs with the lowest p-values (< 0.05) across all five statistical models were judged to represent the most robust associations, followed by SNPs yielding p-values < 0.05 in four of the five analyses. The latter were prioritized by FBAT p-values for replication analysis. A total of 1397 were successfully genotyped and tested for replication in three independent clinical trials. The top 13 replicated SNPs were subsequently tested for association with BDR in three secondary asthma populations ([Table 2](#T2)). While only one intergenic SNP significantly replicated in the primary phase, six SNPs provided nominal p-values < 0.05 in both the primary replication phase and in one or more of the secondary replication populations, including intronic SNPs in the *COL22A1* and *CLOCK* genes.

The use of five statistical models in our initial GWAS is an innovative approach for identifying genetic associations for BDR in asthma. As each statistical model has unique strengths and weaknesses, our rationale for ranking SNPs for replication based on p-values from all five models was to identify the most robust associations (i.e. those most likely to replicate and represent true pharmacogenetic associations). For example, population-based tests are more powerful to detect associations by including more individuals than the number of informative families used in the FBAT, but the former is more vulnerable to population stratification. Thus, FBAT allows us to confirm SNP associations that are not influenced by population stratification. In addition, we were able to take advantage of the longitudinal BDR data recorded at 11 time points over the four year clinical trial for a subset of our population to confirm associations that are repeatable within individuals over time. Moreover, we opted to include a recessive model because while an additive genetic model can easily identify dominant transmissions, it does not identify recessive transmissions as easily. We believe that this novel approach reduced the likelihood of false-positive association signals.

The strongest association signal that significantly replicated in the primary replication phase, albeit not associated across the secondary replication populations, was an intergenic SNP rs11252394 (Liptak p-value = 1.98E-07). Despite it being not proximal to a gene within 50 kb, a closer look at this genomic region revealed several excellent biological candidates within 2.5 Mb including Protein Kinase C theta (*PRKCQ*), inter-leukin receptors (*IL15RA*, *IL2RA*) and Krüppel-like factor 6 (*KLF6*). All four genes have been previously reported to regulate pulmonary inflammation using *in vitro* cellular and murine models. In fact, a *PRKCQ* antagonist was investigated by Wyeth Research as a novel treatment for asthma given the role of this gene in airway inflammation and hyper-responsiveness.^30–32^ Inhibition of *IL15RA* and *IL2RA* in mice demonstrated decreased lung inflammation.^33,34^ Finally, blocking of *KLF6 in vitro* decreased Transforming Growth Factor β (TGFβ) production that is correlated with airway remodeling and asthma development.^35^ While rs11252394 is not known to regulate the expression of any of these genes, nor is it known to be in LD with SNPs within these loci, further investigation is warranted to identify the causative variant, if any, in this genomic region which may underlie this association signal.

Another association signal that replicated, albeit only marginally, in the primary replication phase and across two of the secondary replication trials, was an intronic SNP (rs6988229) in the *COL22A1* gene. Little is known about this gene other than it encodes a protein that acts as a cell adhesion ligand for skin epithelial cells and fibroblasts, further investigations are necessary to determine how genetic variants at this locus might influence BDR. Cis-eQTL analysis indicates that this SNP does not regulate expression of the *COL22A1* transcript (p= 0.86). However, this SNP is significantly correlated with the expression of multiple other genes [[Supplemental Table 1](#SD1)]. This includes another member of G protein-coupled receptor superfamily (*GPR110*), to which the β2-adrenergic receptor also belongs, which is known to regulate smooth muscle contractions and relaxations.^36^ Multiple splice variants of this gene, like many other members of this large gene family, has been shown to be expressed at significantly higher levels in airway smooth muscle cells.^37^

While the polymorphism in the *CLOCK* gene (rs1522113) was only marginally associated with BDR at randomization in CAMP using the additive model (p-value = 0.014), and nominally replicated in AT/LOCCS/LODO and CARE, it is an excellent biological candidate for regulating bronchodilator response in asthmatics. Previous studies suggest that *CLOCK* expression and β_2_-agonists affect the expression of circadian rhythm genes, which regulate asthma symptoms. Embryonic fibroblast cells from mice homozygous for mutant *CLOCK* expressed circadian rhythm genes in a non-cyclic manner, a phenotype that was rescued by ectopic expression of *CLOCK*.^38^ DeBruyne et al. reported that the circadian rhythm in peripheral tissues such as the liver and lung are also regulated by *CLOCK*.^39^ CLOCK binds to the E-box enhancer located 5′ of circadian genes such as the Periods (*PER*) 1, 2, and 3 to regulate their expression.^40^β_2-_agonists have also been shown to induce the expression of human period 1 (*hPER1*) gene in bronchial epithelial BEAS-2B cells.^41^ Furthermore, the administration of β_2_-agonists, particularly long-acting, reduces nocturnal asthma.^42,43^β_2_-agonists have also been shown to regulate the expression of these circadian rhythm genes through the phosphorylation of cAMP responsive element binding (CREB) protein which bind to CRE 5′ of these genes.^40^ The role of the circadian rhythm in asthma is apparent in that the narrowing of the airways are more severe between midnight and early morning hours.^44^ In addition, nocturnal asthma exacerbations are commonly experienced between 4 AM and 8 AM,^43^ which may be the combined effect of the circadian clock and the diminishing effect of asthma medications throughout the night.

In addition to a genetic association between rs1522113 and BDR in asthma, we determined that this intronic SNP is in perfect linkage disequilibrium with a missense variant in exon 9 (rs34897046; S208C), which is predicted to result in the loss of a (Serine) phosphorylation site.^45^ This coding SNP is predicted to be “deleterious” by SIFT (Sorting Intolerant From Tolerant)^46^ or “possibly damaging” by PolyPhen2.^47^ Finally, analysis of microarray data from lymphoblastoid cell lines of CAMP subjects indicates a marginal association between the mutant allele (208C) and increased expression of *CLOCK* (p value = 0.054), as well as increased expression of a downstream circadian rhythm gene Period 2 (*PER2,* p value = 0.003) [[Supplemental Figure 2](#SD3)]. While this suggests that the associated polymorphism in *CLOCK* may be functional, further experiments are necessary to investigate the regulatory potential of this variant in the *CLOCK* pathway and the mechanism by which it modulates BDR.

While this manuscript represents a comprehensive GWAS of BDR response in asthmatics aimed at identifying the most robust genetic associations for replication in additional asthma trials, there were several limitations. First, our initial GWAS used the phenotype of acute response to a short-acting β_2_-agonist (BDR at randomization) that was taken in all CAMP probands at the start of the study as well as repeated measures of BDR in a subset of the CAMP probands who were randomized to β_2_-agonist as needed over the four years of the trial. For the replication cohorts, however, we only used BDR measured upon entry into the respective studies as our replication samples did not have longitudinal data. Therefore, our replication results may not identify BDR associations in asthma patients taking β_2_-agonist over long periods of time. Second, the mean pre-bronchodilator FEV_1_ percent predicted values varied across the asthma populations ([Table 1](#T1)). Specifically, those for all childhood asthma trials (CAMP, CARE and GACRS) were noticeably higher than those of adult asthma trials (AT, LOCCS, LODO and ACRN), which was expected. However, pre-bronchodilator FEV_1_ was adjusted for in our definition of BDR as well in all statistical analyses by including it as a covariate. Finally, baseline medications and recruitment criteria varied across some of the populations. For example, the LOCCS trial had completed a run-in period of 4–6 weeks during which they were administered an inhaled corticosteroid that might have improved their lung function, resulting in reduced BDR. Finally, all participants of the AT trial had a minimum BDR of 15% or greater. We addressed these differences across our trial populations in the pooled analysis of AT/LOCCS/LODO by coding each trial differently. Some of the trials (CAMP, AT, LODO, GACRS) had wash-out periods during which they were taken off their regular asthma therapies but were permitted to use rescue medications as needed. Others such as CARE and ACRN had no wash-out periods. Thus, differences in medical histories may have influenced BDR. However, we believe that these differences further demonstrate the generalizability of our association results.

Although the aim of this GWAS was to identify novel loci for BDR, we noted that this study does not replicate all of the prior associated SNPs.^5–11^ These results were expected as it is unusual for all of the candidate genes to be significant in any one replication population. For some of these previously associated loci, the genetic effect sizes were very modest, making these genetic variants more difficult to identify. Power simulations, based on our sample sizes (n = 403, 764, and 1,048) and the number of statistical tests, estimated that we had sufficient power (>90%) to identify common SNPs (MAF > 0.1) with effect estimates of 3 percent or greater. In addition, there was not always adequate LD coverage for some of the SNPs that were previously identified at candidate genes. Therefore, it was difficult to assess these genetic associations in our CAMP samples. Specifically, additional variants were genotyped in earlier studies using custom platforms that were not included on the Illumina HapMap550K Beadchip array used for the current GWAS. For example, none of the previously associated SNPs in *ARGI* were tested in the current GWAS. In fact, only four of the dozen SNPs previously implicated in the remaining three genes (*ADRB2*, *CRHR2* and *AC9)* were directly tested in our GWAS. However, these did not yield high ranking scores for replication because we had selected SNPs based on p-values across five different statistical models. Furthermore, some of the earlier studies had reported a haplotype effect that was not tested in our study. Finally, our GWAS did not replicate the findings of the previous BDR GWAS, which reported association with *SPATS2L*.^12^ A major difference in the current study is the combined analysis of longitudinal BDR measures as well family-based data in addition to BDR at randomization using five statistical models, while the previous GWAS applied only one test of BDR at randomization.

Using a novel genome-wide association analysis method for investigating BDR in asthma, we have identified several genetic loci for further investigation. Among these findings is an intergenic SNP, rs11252394, that is located near multiple genes previously correlated with lung inflammation and therefore, are potential regulators of asthma. Other potentially interesting associations that were marginally associated with our drug response phenotype across multiple trials were intronic SNPs within the *COL22A1* and *CLOCK* loci. While microarray data indicate potential cis- or trans-effects of these SNPs, further investigation is merited to determine their biological significance and potential roles in modulating bronchodilator response to β_2_-agonists.

## Supplementary Material

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1c95/3706515/5f0890ebbde1/nihms442409f3.jpg)

Manhattan plot of –Log10(p-value) for the FBAT analysis of BDR using 403 parent-offspring trios with 534,290 SNPs. Similar plots were generated for the other four statistical models. The analysis was adjusted for age, gender, height, and baseline preFEV1.

## Acknowledgments

This work was supported by U01 HL65899 and P01 HL083069 from the National Heart, Lung, and Blood Institute (NHLBI). We thank all families for their enthusiastic participation in the CAMP Genetics Ancillary Study and the CAMP investigators and research teams, who were supported by the NHLBI N01 HR16049. Additional support for this research came from NHLBI grants N01 HR16044, HR16045, HR16046, HR16047, HR16048, HR16049, HR16050, HR16051, and HR16052. All data collection from the CAMP Genetic Ancillary Study was conducted at the Channing Laboratory of the Brigham and Women’s Hospital under appropriate CAMP policies and human subject’s protections. The CAMP Genetics Ancillary Study is supported by U01 HL075419, U01 HL65899, P01 HL083069, R01 HL086601, and T32 HL07427 from the NIH/NHLBI. Collection of microarray data from immortalized lymphoblastoid cell lines of CAMP subjects was supported by K23 HG003983 and R01 HL092197 from the NIH/NHGRI. We acknowledge the American Lung Association (ALA) and the ALA’s Asthma Clinical Research Centers investigators and research teams for use of LOCCS and LODO data, with additional funding from HL071394 and HL074755 from the NHLBI, and Nemours Children’s’ Clinic. GlaxoSmithKline supported the conduct of the LOCCS Trial by an unrestricted grant to the ALA. We acknowledge Sepracor, Inc. for use of the Asthma Trial data. The Single-Nucleotide Polymorphism Health Association Asthma Resource Project (SHARP) was funded by grants from the NHLBI U01 HL51510, U01 HL51834, U01 HL51831, U01 HL51845, U01 HL51843, M01 RR00079, and M01 RR03186 and was carried out by researchers from the Asthma Clinical Research Network (ACRN), CAMP, and Childhood Asthma Research and Education (CARE) Network. Details are available in the [Online Repository](#SD1) and on the dbGaP (database of Genotypes and Phenotypes) website: [www.ncbi.nlm.nih.gov/sites/entrez?Db=gap](http://www.ncbi.nlm.nih.gov/sites/entrez?Db=gap). The GACRS was supported by HL04370 and HL66289 from the NIH.

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Masoli M, Fabian D, Holt S, Beasley R. The global burden of asthma: executive summary of the GINA Dissemination Committee report. Allergy. 2004;59(5):469–478. doi: 10.1111/j.1398-9995.2004.00526.x.  [DOI](https://doi.org/10.1111/j.1398-9995.2004.00526.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15080825/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Allergy&title=The%20global%20burden%20of%20asthma:%20executive%20summary%20of%20the%20GINA%20Dissemination%20Committee%20report&author=M%20Masoli&author=D%20Fabian&author=S%20Holt&author=R%20Beasley&volume=59&issue=5&publication_year=2004&pages=469-478&pmid=15080825&doi=10.1111/j.1398-9995.2004.00526.x&)

2. Sears MR, Lotvall J. Past, present and future--beta2-adrenoceptor agonists in asthma management. Respir Med. 2005;99(2):152–170. doi: 10.1016/j.rmed.2004.07.003.  [DOI](https://doi.org/10.1016/j.rmed.2004.07.003) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15715182/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Respir%20Med&title=Past,%20present%20and%20future--beta2-adrenoceptor%20agonists%20in%20asthma%20management&author=MR%20Sears&author=J%20Lotvall&volume=99&issue=2&publication_year=2005&pages=152-170&pmid=15715182&doi=10.1016/j.rmed.2004.07.003&)

3. Drazen JM, Silverman EK, Lee TH. Heterogeneity of therapeutic responses in asthma. Br Med Bull. 2000;56(4):1054–1070. doi: 10.1258/0007142001903535.  [DOI](https://doi.org/10.1258/0007142001903535) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11359637/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20Med%20Bull&title=Heterogeneity%20of%20therapeutic%20responses%20in%20asthma&author=JM%20Drazen&author=EK%20Silverman&author=TH%20Lee&volume=56&issue=4&publication_year=2000&pages=1054-1070&pmid=11359637&doi=10.1258/0007142001903535&)

4. Barnett SB, Nurmagambetov TA. Costs of asthma in the United States: 2002–2007. J Allergy Clin Immunol. 127(1):145–152. doi: 10.1016/j.jaci.2010.10.020.  [DOI](https://doi.org/10.1016/j.jaci.2010.10.020) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21211649/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Allergy%20Clin%20Immunol&title=Costs%20of%20asthma%20in%20the%20United%20States:%202002%E2%80%932007&author=SB%20Barnett&author=TA%20Nurmagambetov&volume=127&issue=1&pages=145-152&pmid=21211649&doi=10.1016/j.jaci.2010.10.020&)

5. Litonjua AA, Lasky-Su J, Schneiter K, Tantisira KG, Lazarus R, Klanderman B, et al. ARG1 is a novel bronchodilator response gene: screening and replication in four asthma cohorts. Am J Respir Crit Care Med. 2008;178(7):688–694. doi: 10.1164/rccm.200709-1363OC.  [DOI](https://doi.org/10.1164/rccm.200709-1363OC) | [PMC free article](/articles/PMC2556451/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18617639/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Respir%20Crit%20Care%20Med&title=ARG1%20is%20a%20novel%20bronchodilator%20response%20gene:%20screening%20and%20replication%20in%20four%20asthma%20cohorts&author=AA%20Litonjua&author=J%20Lasky-Su&author=K%20Schneiter&author=KG%20Tantisira&author=R%20Lazarus&volume=178&issue=7&publication_year=2008&pages=688-694&pmid=18617639&doi=10.1164/rccm.200709-1363OC&)

6. Martinez FD, Graves PE, Baldini M, Solomon S, Erickson R. Association between genetic polymorphisms of the beta2-adrenoceptor and response to albuterol in children with and without a history of wheezing. J Clin Invest. 1997;100(12):3184–3188. doi: 10.1172/JCI119874.  [DOI](https://doi.org/10.1172/JCI119874) | [PMC free article](/articles/PMC508532/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9399966/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Invest&title=Association%20between%20genetic%20polymorphisms%20of%20the%20beta2-adrenoceptor%20and%20response%20to%20albuterol%20in%20children%20with%20and%20without%20a%20history%20of%20wheezing&author=FD%20Martinez&author=PE%20Graves&author=M%20Baldini&author=S%20Solomon&author=R%20Erickson&volume=100&issue=12&publication_year=1997&pages=3184-3188&pmid=9399966&doi=10.1172/JCI119874&)

7. Green SA, Turki J, Innis M, Liggett SB. Amino-terminal polymorphisms of the human beta 2-adrenergic receptor impart distinct agonist-promoted regulatory properties. Biochemistry. 1994;33(32):9414–9419. doi: 10.1021/bi00198a006.  [DOI](https://doi.org/10.1021/bi00198a006) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7915137/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochemistry&title=Amino-terminal%20polymorphisms%20of%20the%20human%20beta%202-adrenergic%20receptor%20impart%20distinct%20agonist-promoted%20regulatory%20properties&author=SA%20Green&author=J%20Turki&author=M%20Innis&author=SB%20Liggett&volume=33&issue=32&publication_year=1994&pages=9414-9419&pmid=7915137&doi=10.1021/bi00198a006&)

8. Lima JJ, Thomason DB, Mohamed MH, Eberle LV, Self TH, Johnson JA. Impact of genetic polymorphisms of the beta2-adrenergic receptor on albuterol bronchodilator pharmacodynamics. Clin Pharmacol Ther. 1999;65(5):519–525. doi: 10.1016/S0009-9236(99)70071-8.  [DOI](https://doi.org/10.1016/S0009-9236(99)70071-8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10340917/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Impact%20of%20genetic%20polymorphisms%20of%20the%20beta2-adrenergic%20receptor%20on%20albuterol%20bronchodilator%20pharmacodynamics&author=JJ%20Lima&author=DB%20Thomason&author=MH%20Mohamed&author=LV%20Eberle&author=TH%20Self&volume=65&issue=5&publication_year=1999&pages=519-525&pmid=10340917&doi=10.1016/S0009-9236(99)70071-8&)

9. Martin AC, Zhang G, Rueter K, Khoo SK, Bizzintino J, Hayden CM, et al. Beta2-adrenoceptor polymorphisms predict response to beta2-agonists in children with acute asthma. J Asthma. 2008;45(5):383–388. doi: 10.1080/02770900801971792.  [DOI](https://doi.org/10.1080/02770900801971792) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18569231/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Asthma&title=Beta2-adrenoceptor%20polymorphisms%20predict%20response%20to%20beta2-agonists%20in%20children%20with%20acute%20asthma&author=AC%20Martin&author=G%20Zhang&author=K%20Rueter&author=SK%20Khoo&author=J%20Bizzintino&volume=45&issue=5&publication_year=2008&pages=383-388&pmid=18569231&doi=10.1080/02770900801971792&)

10. Poon AH, Tantisira KG, Litonjua AA, Lazarus R, Xu J, Lasky-Su J, et al. Association of corticotropin-releasing hormone receptor-2 genetic variants with acute bronchodilator response in asthma. Pharmacogenet Genomics. 2008;18(5):373–382. doi: 10.1097/FPC.0b013e3282fa760a.  [DOI](https://doi.org/10.1097/FPC.0b013e3282fa760a) | [PMC free article](/articles/PMC3208318/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18408560/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Association%20of%20corticotropin-releasing%20hormone%20receptor-2%20genetic%20variants%20with%20acute%20bronchodilator%20response%20in%20asthma&author=AH%20Poon&author=KG%20Tantisira&author=AA%20Litonjua&author=R%20Lazarus&author=J%20Xu&volume=18&issue=5&publication_year=2008&pages=373-382&pmid=18408560&doi=10.1097/FPC.0b013e3282fa760a&)

11. Tantisira KG, Small KM, Litonjua AA, Weiss ST, Liggett SB. Molecular properties and pharmacogenetics of a polymorphism of adenylyl cyclase type 9 in asthma: interaction between beta-agonist and corticosteroid pathways. Hum Mol Genet. 2005;14(12):1671–1677. doi: 10.1093/hmg/ddi175.  [DOI](https://doi.org/10.1093/hmg/ddi175) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15879435/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mol%20Genet&title=Molecular%20properties%20and%20pharmacogenetics%20of%20a%20polymorphism%20of%20adenylyl%20cyclase%20type%209%20in%20asthma:%20interaction%20between%20beta-agonist%20and%20corticosteroid%20pathways&author=KG%20Tantisira&author=KM%20Small&author=AA%20Litonjua&author=ST%20Weiss&author=SB%20Liggett&volume=14&issue=12&publication_year=2005&pages=1671-1677&pmid=15879435&doi=10.1093/hmg/ddi175&)

12. Himes BE, Jiang X, Hu R, Wu AC, Lasky-Su J, Klanderman B, et al. Genome-wide Association Analysis in Asthma Subjects Identifies SPATS2L as a Novel Bronchodilator Response Gene. PLoS Genet. doi: 10.1371/journal.pgen.1002824. In press.  [DOI](https://doi.org/10.1371/journal.pgen.1002824) | [PMC free article](/articles/PMC3390407/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22792082/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet&title=Genome-wide%20Association%20Analysis%20in%20Asthma%20Subjects%20Identifies%20SPATS2L%20as%20a%20Novel%20Bronchodilator%20Response%20Gene&author=BE%20Himes&author=X%20Jiang&author=R%20Hu&author=AC%20Wu&author=J%20Lasky-Su&pmid=22792082&doi=10.1371/journal.pgen.1002824&)

13. Long-term effects of budesonide or nedocromil in children with asthma. The Childhood Asthma Management Program Research Group. N Engl J Med. 2000;343(15):1054–1063. doi: 10.1056/NEJM200010123431501.  [DOI](https://doi.org/10.1056/NEJM200010123431501) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11027739/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Long-term%20effects%20of%20budesonide%20or%20nedocromil%20in%20children%20with%20asthma&volume=343&issue=15&publication_year=2000&pages=1054-1063&pmid=11027739&doi=10.1056/NEJM200010123431501&)

14. The Childhood Asthma Management Program (CAMP): design, rationale, and methods. Childhood Asthma Management Program Research Group. Control Clin Trials. 1999;20 (1):91–120.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/10027502/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Control%20Clin%20Trials&title=The%20Childhood%20Asthma%20Management%20Program%20(CAMP):%20design,%20rationale,%20and%20methods&volume=20&issue=1&publication_year=1999&pages=91-120&pmid=10027502&)

15. Himes BE, Hunninghake GM, Baurley JW, Rafaels NM, Sleiman P, Strachan DP, et al. Genome-wide association analysis identifies PDE4D as an asthma-susceptibility gene. Am J Hum Genet. 2009;84(5):581–593. doi: 10.1016/j.ajhg.2009.04.006.  [DOI](https://doi.org/10.1016/j.ajhg.2009.04.006) | [PMC free article](/articles/PMC2681010/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19426955/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=Genome-wide%20association%20analysis%20identifies%20PDE4D%20as%20an%20asthma-susceptibility%20gene&author=BE%20Himes&author=GM%20Hunninghake&author=JW%20Baurley&author=NM%20Rafaels&author=P%20Sleiman&volume=84&issue=5&publication_year=2009&pages=581-593&pmid=19426955&doi=10.1016/j.ajhg.2009.04.006&)

16. Baron RM, Palmer LJ, Tantisira K, Gabriel S, Sonna LA, Le L, et al. DNA sequence variants in epithelium-specific ETS-2 and ETS-3 are not associated with asthma. Am J Respir Crit Care Med. 2002;166(7):927–932. doi: 10.1164/rccm.200201-048OC.  [DOI](https://doi.org/10.1164/rccm.200201-048OC) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12359648/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Respir%20Crit%20Care%20Med&title=DNA%20sequence%20variants%20in%20epithelium-specific%20ETS-2%20and%20ETS-3%20are%20not%20associated%20with%20asthma&author=RM%20Baron&author=LJ%20Palmer&author=K%20Tantisira&author=S%20Gabriel&author=LA%20Sonna&volume=166&issue=7&publication_year=2002&pages=927-932&pmid=12359648&doi=10.1164/rccm.200201-048OC&)

17. Silverman ES, Palmer LJ, Subramaniam V, Hallock A, Mathew S, Vallone J, et al. Transforming growth factor-beta1 promoter polymorphism C-509T is associated with asthma. Am J Respir Crit Care Med. 2004;169(2):214–219. doi: 10.1164/rccm.200307-973OC.  [DOI](https://doi.org/10.1164/rccm.200307-973OC) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14597484/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Respir%20Crit%20Care%20Med&title=Transforming%20growth%20factor-beta1%20promoter%20polymorphism%20C-509T%20is%20associated%20with%20asthma&author=ES%20Silverman&author=LJ%20Palmer&author=V%20Subramaniam&author=A%20Hallock&author=S%20Mathew&volume=169&issue=2&publication_year=2004&pages=214-219&pmid=14597484&doi=10.1164/rccm.200307-973OC&)

18. Peters SP, Anthonisen N, Castro M, Holbrook JT, Irvin CG, Smith LJ, et al. Randomized comparison of strategies for reducing treatment in mild persistent asthma. N Engl J Med. 2007;356(20):2027–2039. doi: 10.1056/NEJMoa070013.  [DOI](https://doi.org/10.1056/NEJMoa070013) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17507702/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Randomized%20comparison%20of%20strategies%20for%20reducing%20treatment%20in%20mild%20persistent%20asthma&author=SP%20Peters&author=N%20Anthonisen&author=M%20Castro&author=JT%20Holbrook&author=CG%20Irvin&volume=356&issue=20&publication_year=2007&pages=2027-2039&pmid=17507702&doi=10.1056/NEJMoa070013&)

19. Clinical trial of low-dose theophylline and montelukast in patients with poorly controlled asthma. Am J Respir Crit Care Med. 2007;175(3):235–242. doi: 10.1164/rccm.200603-416OC.  [DOI](https://doi.org/10.1164/rccm.200603-416OC) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16998094/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Respir%20Crit%20Care%20Med&title=Clinical%20trial%20of%20low-dose%20theophylline%20and%20montelukast%20in%20patients%20with%20poorly%20controlled%20asthma&volume=175&issue=3&publication_year=2007&pages=235-242&pmid=16998094&doi=10.1164/rccm.200603-416OC&)

20. Israel E, Chinchilli VM, Ford JG, Boushey HA, Cherniack R, Craig TJ, et al. Use of regularly scheduled albuterol treatment in asthma: genotype-stratified, randomised, placebo-controlled cross-over trial. Lancet. 2004;364(9444):1505–1512. doi: 10.1016/S0140-6736(04)17273-5.  [DOI](https://doi.org/10.1016/S0140-6736(04)17273-5) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15500895/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Use%20of%20regularly%20scheduled%20albuterol%20treatment%20in%20asthma:%20genotype-stratified,%20randomised,%20placebo-controlled%20cross-over%20trial&author=E%20Israel&author=VM%20Chinchilli&author=JG%20Ford&author=HA%20Boushey&author=R%20Cherniack&volume=364&issue=9444&publication_year=2004&pages=1505-1512&pmid=15500895&doi=10.1016/S0140-6736(04)17273-5&)

21. A haplotype map of the human genome. Nature. 2005;437(7063):1299–1320. doi: 10.1038/nature04226.  [DOI](https://doi.org/10.1038/nature04226) | [PMC free article](/articles/PMC1880871/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16255080/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=A%20haplotype%20map%20of%20the%20human%20genome&volume=437&issue=7063&publication_year=2005&pages=1299-1320&pmid=16255080&doi=10.1038/nature04226&)

22. Willer CJ, Sanna S, Jackson AU, Scuteri A, Bonnycastle LL, Clarke R, et al. Newly identified loci that influence lipid concentrations and risk of coronary artery disease. Nat Genet. 2008;40(2):161–169. doi: 10.1038/ng.76.  [DOI](https://doi.org/10.1038/ng.76) | [PMC free article](/articles/PMC5206900/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18193043/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Newly%20identified%20loci%20that%20influence%20lipid%20concentrations%20and%20risk%20of%20coronary%20artery%20disease&author=CJ%20Willer&author=S%20Sanna&author=AU%20Jackson&author=A%20Scuteri&author=LL%20Bonnycastle&volume=40&issue=2&publication_year=2008&pages=161-169&pmid=18193043&doi=10.1038/ng.76&)

23. Hunninghake GM, Soto-Quiros ME, Avila L, Su J, Murphy A, Demeo DL, et al. Polymorphisms in IL13, total IgE, eosinophilia, and asthma exacerbations in childhood. J Allergy Clin Immunol. 2007;120(1):84–90. doi: 10.1016/j.jaci.2007.04.032.  [DOI](https://doi.org/10.1016/j.jaci.2007.04.032) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17561245/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Allergy%20Clin%20Immunol&title=Polymorphisms%20in%20IL13,%20total%20IgE,%20eosinophilia,%20and%20asthma%20exacerbations%20in%20childhood&author=GM%20Hunninghake&author=ME%20Soto-Quiros&author=L%20Avila&author=J%20Su&author=A%20Murphy&volume=120&issue=1&publication_year=2007&pages=84-90&pmid=17561245&doi=10.1016/j.jaci.2007.04.032&)

24. Purcell S, Neale B, Todd-Brown K, Thomas L, Ferreira MA, Bender D, et al. PLINK: a tool set for whole-genome association and population-based linkage analyses. Am J Hum Genet. 2007;81(3):559–575. doi: 10.1086/519795.  [DOI](https://doi.org/10.1086/519795) | [PMC free article](/articles/PMC1950838/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17701901/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=PLINK:%20a%20tool%20set%20for%20whole-genome%20association%20and%20population-based%20linkage%20analyses&author=S%20Purcell&author=B%20Neale&author=K%20Todd-Brown&author=L%20Thomas&author=MA%20Ferreira&volume=81&issue=3&publication_year=2007&pages=559-575&pmid=17701901&doi=10.1086/519795&)

25. Lange C, DeMeo D, Silverman EK, Weiss ST, Laird NM. PBAT: tools for family-based association studies. Am J Hum Genet. 2004;74(2):367–369. doi: 10.1086/381563.  [DOI](https://doi.org/10.1086/381563) | [PMC free article](/articles/PMC1181934/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14740322/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=PBAT:%20tools%20for%20family-based%20association%20studies&author=C%20Lange&author=D%20DeMeo&author=EK%20Silverman&author=ST%20Weiss&author=NM%20Laird&volume=74&issue=2&publication_year=2004&pages=367-369&pmid=14740322&doi=10.1086/381563&)

26. Liptak T. On the combination of independent tests. Magyar Tud Akad Mat Kutato Int Kozl. 1958;3:171–197.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Magyar%20Tud%20Akad%20Mat%20Kutato%20Int%20Kozl&title=On%20the%20combination%20of%20independent%20tests&author=T%20Liptak&volume=3&publication_year=1958&pages=171-197&)

27. Du P, Kibbe WA, Lin SM. lumi: a pipeline for processing Illumina microarray. Bioinformatics. 2008;24(13):1547–1548. doi: 10.1093/bioinformatics/btn224.  [DOI](https://doi.org/10.1093/bioinformatics/btn224) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18467348/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=lumi:%20a%20pipeline%20for%20processing%20Illumina%20microarray&author=P%20Du&author=WA%20Kibbe&author=SM%20Lin&volume=24&issue=13&publication_year=2008&pages=1547-1548&pmid=18467348&doi=10.1093/bioinformatics/btn224&)

28. Tantisira KG, Fuhlbrigge AL, Tonascia J, Van Natta M, Zeiger RS, Strunk RC, et al. Bronchodilation and bronchoconstriction: predictors of future lung function in childhood asthma. J Allergy Clin Immunol. 2006;117(6):1264–1271. doi: 10.1016/j.jaci.2006.01.050.  [DOI](https://doi.org/10.1016/j.jaci.2006.01.050) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16750985/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Allergy%20Clin%20Immunol&title=Bronchodilation%20and%20bronchoconstriction:%20predictors%20of%20future%20lung%20function%20in%20childhood%20asthma&author=KG%20Tantisira&author=AL%20Fuhlbrigge&author=J%20Tonascia&author=M%20Van%20Natta&author=RS%20Zeiger&volume=117&issue=6&publication_year=2006&pages=1264-1271&pmid=16750985&doi=10.1016/j.jaci.2006.01.050&)

29. Hawkins GA, Meyers DA, Bleecker ER, Pack AI. Identification of coding polymorphisms in human circadian rhythm genes PER1, PER2, PER3, CLOCK, ARNTL, CRY1, CRY2 and TIMELESS in a multi-ethnic screening panel. DNA Seq. 2008;19(1):44–49. doi: 10.1080/10425170701322197.  [DOI](https://doi.org/10.1080/10425170701322197) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17852344/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=DNA%20Seq&title=Identification%20of%20coding%20polymorphisms%20in%20human%20circadian%20rhythm%20genes%20PER1,%20PER2,%20PER3,%20CLOCK,%20ARNTL,%20CRY1,%20CRY2%20and%20TIMELESS%20in%20a%20multi-ethnic%20screening%20panel&author=GA%20Hawkins&author=DA%20Meyers&author=ER%20Bleecker&author=AI%20Pack&volume=19&issue=1&publication_year=2008&pages=44-49&pmid=17852344&doi=10.1080/10425170701322197&)

30. Chaudhary D, Kasaian M. PKCtheta: A potential therapeutic target for T-cell-mediated diseases. Curr Opin Investig Drugs. 2006;7(5):432–437.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/16729719/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Opin%20Investig%20Drugs&title=PKCtheta:%20A%20potential%20therapeutic%20target%20for%20T-cell-mediated%20diseases&author=D%20Chaudhary&author=M%20Kasaian&volume=7&issue=5&publication_year=2006&pages=432-437&pmid=16729719&)

31. Mosyak L, Xu Z, Joseph-McCarthy D, Brooijmans N, Somers W, Chaudhary D. Structure-based optimization of PKCtheta inhibitors. Biochem Soc Trans. 2007;35(Pt 5):1027–1031. doi: 10.1042/BST0351027.  [DOI](https://doi.org/10.1042/BST0351027) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17956269/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem%20Soc%20Trans&title=Structure-based%20optimization%20of%20PKCtheta%20inhibitors&author=L%20Mosyak&author=Z%20Xu&author=D%20Joseph-McCarthy&author=N%20Brooijmans&author=W%20Somers&volume=35&issue=Pt%205&publication_year=2007&pages=1027-1031&pmid=17956269&doi=10.1042/BST0351027&)

32. Cole DC, Asselin M, Brennan A, Czerwinski R, Ellingboe JW, Fitz L, et al. Identification, characterization and initial hit-to-lead optimization of a series of 4-arylamino-3-pyridinecarbonitrile as protein kinase C theta (PKCtheta) inhibitors. J Med Chem. 2008;51(19):5958–5963. doi: 10.1021/jm800214a.  [DOI](https://doi.org/10.1021/jm800214a) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18783200/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Med%20Chem&title=Identification,%20characterization%20and%20initial%20hit-to-lead%20optimization%20of%20a%20series%20of%204-arylamino-3-pyridinecarbonitrile%20as%20protein%20kinase%20C%20theta%20(PKCtheta)%20inhibitors&author=DC%20Cole&author=M%20Asselin&author=A%20Brennan&author=R%20Czerwinski&author=JW%20Ellingboe&volume=51&issue=19&publication_year=2008&pages=5958-5963&pmid=18783200&doi=10.1021/jm800214a&)

33. Ruckert R, Brandt K, Braun A, Hoymann HG, Herz U, Budagian V, et al. Blocking IL-15 prevents the induction of allergen-specific T cells and allergic inflammation in vivo. J Immunol. 2005;174(9):5507–5515. doi: 10.4049/jimmunol.174.9.5507.  [DOI](https://doi.org/10.4049/jimmunol.174.9.5507) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15843549/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Immunol&title=Blocking%20IL-15%20prevents%20the%20induction%20of%20allergen-specific%20T%20cells%20and%20allergic%20inflammation%20in%20vivo&author=R%20Ruckert&author=K%20Brandt&author=A%20Braun&author=HG%20Hoymann&author=U%20Herz&volume=174&issue=9&publication_year=2005&pages=5507-5515&pmid=15843549&doi=10.4049/jimmunol.174.9.5507&)

34. Doganci A, Karwot R, Maxeiner JH, Scholtes P, Schmitt E, Neurath MF, et al. IL-2 receptor beta-chain signaling controls immunosuppressive CD4+ T cells in the draining lymph nodes and lung during allergic airway inflammation in vivo. J Immunol. 2008;181(3):1917–1926. doi: 10.4049/jimmunol.181.3.1917.  [DOI](https://doi.org/10.4049/jimmunol.181.3.1917) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18641329/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Immunol&title=IL-2%20receptor%20beta-chain%20signaling%20controls%20immunosuppressive%20CD4+%20T%20cells%20in%20the%20draining%20lymph%20nodes%20and%20lung%20during%20allergic%20airway%20inflammation%20in%20vivo&author=A%20Doganci&author=R%20Karwot&author=JH%20Maxeiner&author=P%20Scholtes&author=E%20Schmitt&volume=181&issue=3&publication_year=2008&pages=1917-1926&pmid=18641329&doi=10.4049/jimmunol.181.3.1917&)

35. Mgbemena V, Segovia J, Chang T, Bose S. Kruppel-like factor 6 regulates transforming growth factor-beta gene expression during human respiratory syncytial virus infection. Virol J. 8:409. doi: 10.1186/1743-422X-8-409.  [DOI](https://doi.org/10.1186/1743-422X-8-409) | [PMC free article](/articles/PMC3170303/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21849067/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Virol%20J&title=Kruppel-like%20factor%206%20regulates%20transforming%20growth%20factor-beta%20gene%20expression%20during%20human%20respiratory%20syncytial%20virus%20infection&author=V%20Mgbemena&author=J%20Segovia&author=T%20Chang&author=S%20Bose&volume=8&pages=409&pmid=21849067&doi=10.1186/1743-422X-8-409&)

36. McGraw DW, Elwing JM, Fogel KM, Wang WC, Glinka CB, Mihlbachler KA, et al. Crosstalk between Gi and Gq/Gs pathways in airway smooth muscle regulates bronchial contractility and relaxation. J Clin Invest. 2007;117(5):1391–1398. doi: 10.1172/JCI30489.  [DOI](https://doi.org/10.1172/JCI30489) | [PMC free article](/articles/PMC1838924/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17415415/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Invest&title=Crosstalk%20between%20Gi%20and%20Gq/Gs%20pathways%20in%20airway%20smooth%20muscle%20regulates%20bronchial%20contractility%20and%20relaxation&author=DW%20McGraw&author=JM%20Elwing&author=KM%20Fogel&author=WC%20Wang&author=CB%20Glinka&volume=117&issue=5&publication_year=2007&pages=1391-1398&pmid=17415415&doi=10.1172/JCI30489&)

37. Einstein R, Jordan H, Zhou W, Brenner M, Moses EG, Liggett SB. Alternative splicing of the G protein-coupled receptor superfamily in human airway smooth muscle diversifies the complement of receptors. Proc Natl Acad Sci U S A. 2008;105(13):5230–5235. doi: 10.1073/pnas.0801319105.  [DOI](https://doi.org/10.1073/pnas.0801319105) | [PMC free article](/articles/PMC2278184/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18362331/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&title=Alternative%20splicing%20of%20the%20G%20protein-coupled%20receptor%20superfamily%20in%20human%20airway%20smooth%20muscle%20diversifies%20the%20complement%20of%20receptors&author=R%20Einstein&author=H%20Jordan&author=W%20Zhou&author=M%20Brenner&author=EG%20Moses&volume=105&issue=13&publication_year=2008&pages=5230-5235&pmid=18362331&doi=10.1073/pnas.0801319105&)

38. Pando MP, Morse D, Cermakian N, Sassone-Corsi P. Phenotypic rescue of a peripheral clock genetic defect via SCN hierarchical dominance. Cell. 2002;110(1):107–117. doi: 10.1016/s0092-8674(02)00803-6.  [DOI](https://doi.org/10.1016/s0092-8674(02)00803-6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12151001/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell&title=Phenotypic%20rescue%20of%20a%20peripheral%20clock%20genetic%20defect%20via%20SCN%20hierarchical%20dominance&author=MP%20Pando&author=D%20Morse&author=N%20Cermakian&author=P%20Sassone-Corsi&volume=110&issue=1&publication_year=2002&pages=107-117&pmid=12151001&doi=10.1016/s0092-8674(02)00803-6&)

39. DeBruyne JP, Weaver DR, Reppert SM. Peripheral circadian oscillators require CLOCK. Curr Biol. 2007;17(14):R538–539. doi: 10.1016/j.cub.2007.05.067.  [DOI](https://doi.org/10.1016/j.cub.2007.05.067) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17637349/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Biol&title=Peripheral%20circadian%20oscillators%20require%20CLOCK&author=JP%20DeBruyne&author=DR%20Weaver&author=SM%20Reppert&volume=17&issue=14&publication_year=2007&pages=R538-539&pmid=17637349&doi=10.1016/j.cub.2007.05.067&)

40. Travnickova-Bendova Z, Cermakian N, Reppert SM, Sassone-Corsi P. Bimodal regulation of mPeriod promoters by CREB-dependent signaling and CLOCK/BMAL1 activity. Proc Natl Acad Sci U S A. 2002;99(11):7728–7733. doi: 10.1073/pnas.102075599.  [DOI](https://doi.org/10.1073/pnas.102075599) | [PMC free article](/articles/PMC124335/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12032351/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&title=Bimodal%20regulation%20of%20mPeriod%20promoters%20by%20CREB-dependent%20signaling%20and%20CLOCK/BMAL1%20activity&author=Z%20Travnickova-Bendova&author=N%20Cermakian&author=SM%20Reppert&author=P%20Sassone-Corsi&volume=99&issue=11&publication_year=2002&pages=7728-7733&pmid=12032351&doi=10.1073/pnas.102075599&)

41. Takata M, Burioka N, Ohdo S, Fukuoka Y, Miyata M, Endo M, et al. Beta2-adrenoceptor agonists induce the mammalian clock gene, hPer1, mRNA in cultured human bronchial epithelium cells in vitro. Chronobiol Int. 2005;22(4):777–783. doi: 10.1080/07420520500179167.  [DOI](https://doi.org/10.1080/07420520500179167) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16147906/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Chronobiol%20Int&title=Beta2-adrenoceptor%20agonists%20induce%20the%20mammalian%20clock%20gene,%20hPer1,%20mRNA%20in%20cultured%20human%20bronchial%20epithelium%20cells%20in%20vitro&author=M%20Takata&author=N%20Burioka&author=S%20Ohdo&author=Y%20Fukuoka&author=M%20Miyata&volume=22&issue=4&publication_year=2005&pages=777-783&pmid=16147906&doi=10.1080/07420520500179167&)

42. Petrie GR, Chookang JY, Hassan WU, Morrison JF, O’Reilly JF, Pearson SB, et al. Bambuterol: effective in nocturnal asthma. Respir Med. 1993;87(8):581–585. doi: 10.1016/s0954-6111(05)80260-4.  [DOI](https://doi.org/10.1016/s0954-6111(05)80260-4) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8290741/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Respir%20Med&title=Bambuterol:%20effective%20in%20nocturnal%20asthma&author=GR%20Petrie&author=JY%20Chookang&author=WU%20Hassan&author=JF%20Morrison&author=JF%20O%E2%80%99Reilly&volume=87&issue=8&publication_year=1993&pages=581-585&pmid=8290741&doi=10.1016/s0954-6111(05)80260-4&)

43. Bergholtz B. Nocturnal asthma. Causes and treatment. Tidsskr Nor Laegeforen. 1989;109(16):1796–1797.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/2749657/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Tidsskr%20Nor%20Laegeforen&title=Nocturnal%20asthma.%20Causes%20and%20treatment&author=B%20Bergholtz&volume=109&issue=16&publication_year=1989&pages=1796-1797&pmid=2749657&)

44. Hetzel MR, Clark TJ. Comparison of normal and asthmatic circadian rhythms in peak expiratory flow rate. Thorax. 1980;35(10):732–738. doi: 10.1136/thx.35.10.732.  [DOI](https://doi.org/10.1136/thx.35.10.732) | [PMC free article](/articles/PMC471372/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7466721/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Thorax&title=Comparison%20of%20normal%20and%20asthmatic%20circadian%20rhythms%20in%20peak%20expiratory%20flow%20rate&author=MR%20Hetzel&author=TJ%20Clark&volume=35&issue=10&publication_year=1980&pages=732-738&pmid=7466721&doi=10.1136/thx.35.10.732&)

45. Wong YH, Lee TY, Liang HK, Huang CM, Wang TY, Yang YH, et al. KinasePhos 2. 0: a web server for identifying protein kinase-specific phosphorylation sites based on sequences and coupling patterns. Nucleic Acids Res. 2007;35(Web Server issue):W588–594. doi: 10.1093/nar/gkm322.  [DOI](https://doi.org/10.1093/nar/gkm322) | [PMC free article](/articles/PMC1933228/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17517770/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=KinasePhos%202.%200:%20a%20web%20server%20for%20identifying%20protein%20kinase-specific%20phosphorylation%20sites%20based%20on%20sequences%20and%20coupling%20patterns&author=YH%20Wong&author=TY%20Lee&author=HK%20Liang&author=CM%20Huang&author=TY%20Wang&volume=35&issue=Web%20Server%20issue&publication_year=2007&pages=W588-594&pmid=17517770&doi=10.1093/nar/gkm322&)

46. Ng PC, Henikoff S. SIFT: Predicting amino acid changes that affect protein function. Nucleic Acids Res. 2003;31(13):3812–3814. doi: 10.1093/nar/gkg509.  [DOI](https://doi.org/10.1093/nar/gkg509) | [PMC free article](/articles/PMC168916/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12824425/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=SIFT:%20Predicting%20amino%20acid%20changes%20that%20affect%20protein%20function&author=PC%20Ng&author=S%20Henikoff&volume=31&issue=13&publication_year=2003&pages=3812-3814&pmid=12824425&doi=10.1093/nar/gkg509&)

47. Sunyaev S, Ramensky V, Koch I, Lathe W, 3rd, Kondrashov AS, Bork P. Prediction of deleterious human alleles. Hum Mol Genet. 2001;10(6):591–597. doi: 10.1093/hmg/10.6.591.  [DOI](https://doi.org/10.1093/hmg/10.6.591) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11230178/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mol%20Genet&title=Prediction%20of%20deleterious%20human%20alleles&author=S%20Sunyaev&author=V%20Ramensky&author=I%20Koch&author=W%20Lathe&author=AS%20Kondrashov&volume=10&issue=6&publication_year=2001&pages=591-597&pmid=11230178&doi=10.1093/hmg/10.6.591&)
