# Accuracy of various human NAT2 SNP genotyping panels to infer rapid, intermediate and slow acetylator phenotypes

## Metadata
**Authors:** David W Hein, Mark A Doll
**Journal:** Pharmacogenomics
**Date:** 2011 Nov 17
**DOI:** [10.2217/pgs.11.122](https://doi.org/10.2217/pgs.11.122)
**PMID:** 22092036
**PMCID:** PMC3285565
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3285565/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC3285565/pdf/nihms356750.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC3285565/pdf/nihms356750.pdf)

## Abstract

**Aim:** 
Humans exhibit genetic polymorphism in NAT2 resulting in rapid, intermediate and slow acetylator phenotypes. Over 65 NAT2 variants possessing one or more SNPs in the 870-bp NAT2 coding region have been reported. The seven most frequent SNPs are rs1801279 (191G>A), rs1041983 (282C>T), rs1801280 (341T>C), rs1799929 (481C>T), rs1799930 (590G>A), rs1208 (803A>G) and rs1799931 (857G>A). The majority of studies investigate the NAT2 genotype assay for three SNPs: 481C>T, 590G>A and 857G>A. A tag-SNP (rs1495741) recently identified in a genome-wide association study has also been proposed as a biomarker for the NAT2 phenotype.

**Materials & methods:** 
Sulfamethazine N-acetyltransferase catalytic activities were measured in cryopreserved human hepatocytes from a convenience sample of individuals in the USA with an ethnic frequency similar to the 2010 US population census. These activities were segregated by the tag-SNP rs1495741 and each of the seven SNPs described above. We assessed the accuracy of the tag-SNP and various two-, three-, four- and seven-SNP genotyping panels for their ability to accurately infer NAT2 phenotype.

**Results:** 
The accuracy of the various NAT2 SNP genotype panels to infer NAT2 phenotype were as follows: seven-SNP: 98.4%; tag-SNP: 77.7%; two-SNP: 96.1%; three-SNP: 92.2%; and four-SNP: 98.4%.

**Conclusion:** 
A NAT2 four-SNP genotype panel of rs1801279 (191G>A), rs1801280 (341T>C), rs1799930 (590G>A) and rs1799931 (857G>A) infers NAT2 acetylator phenotype with high accuracy, and is recommended over the tag-, two-, three- and (for economy of scale) the seven-SNP genotyping panels, particularly in populations of non-European ancestry.

Keywords: acetylator genotype, acetylator phenotype, cryopreserved, human hepatocyte, NAT2, SNP

### Aim

Humans exhibit genetic polymorphism in *NAT2* resulting in rapid, intermediate and slow acetylator phenotypes. Over 65 *NAT2* variants possessing one or more SNPs in the 870-bp *NAT2* coding region have been reported. The seven most frequent SNPs are rs1801279 (191G>A), rs1041983 (282C>T), rs1801280 (341T>C), rs1799929 (481C>T), rs1799930 (590G>A), rs1208 (803A>G) and rs1799931 (857G>A). The majority of studies investigate the *NAT2* genotype assay for three SNPs: 481C>T, 590G>A and 857G>A. A tag-SNP (rs1495741) recently identified in a genome-wide association study has also been proposed as a biomarker for the *NAT2* phenotype.

### Materials & methods

Sulfamethazine *N*-acetyltransferase catalytic activities were measured in cryopreserved human hepatocytes from a convenience sample of individuals in the USA with an ethnic frequency similar to the 2010 US population census. These activities were segregated by the tag-SNP rs1495741 and each of the seven SNPs described above. We assessed the accuracy of the tag-SNP and various two-, three-, four- and seven-SNP genotyping panels for their ability to accurately infer *NAT2* phenotype.

### Results

The accuracy of the various *NAT2* SNP genotype panels to infer *NAT2* phenotype were as follows: seven-SNP: 98.4%; tag-SNP: 77.7%; two-SNP: 96.1%; three-SNP: 92.2%; and four-SNP: 98.4%.

### Conclusion

A *NAT2* four-SNP genotype panel of rs1801279 (191G>A), rs1801280 (341T>C), rs1799930 (590G>A) and rs1799931 (857G>A) infers *NAT2* acetylator phenotype with high accuracy, and is recommended over the tag-, two-, three- and (for economy of scale) the seven-SNP genotyping panels, particularly in populations of non-European ancestry.

## Materials & methods

### Source & ethnicity of cryopreserved human hepatocytes

Over 250 cryopreserved human hepatocyte samples were collected for public sale and received from Celsis *in vitro* Technologies (MD, USA). They were not collected for this study. Upon receipt, hepatocytes were stored in liquid nitrogen until use. The subject identifier or any information regarding the source of the hepatocytes was not provided. Information on gender, age, race, cause of death, substance abuse, serology and other enzyme activities for the hepatocytes was accessed at [[102](#R53)]. Ethnicity was available for over 99% of the samples. The ethnic frequency in our convenience sampling from 256 individuals was 79.3% Caucasian, 10.55% African–American, 7.03% Hispanic, 1.17% Asian, 1.17% Polynesian and 0.78% unknown. The convenience sample was obtained from individuals in the USA and the ethnic frequency is similar to the 2010 US population census [[103](#R54)].

## DNA isolation & SNP determinations

Upon removal from the liquid nitrogen, the hepatocytes were thawed and centrifuged as described previously [[17](#R17)]. Genomic DNA was isolated from pelleted cells prepared as described above using the QIAamp^®^ DNA Mini Kit (Qiagen, MD, USA) according to manufacturer’s instructions. SNPs in the *NAT2* coding region and their corresponding alleles and haplotypes were determined with the seven-SNP panel assay as previously described [[18](#R18)]. Briefly, SNP-specific PCR primers and fluorogenic probes were designed using Primer Express™ (Applied Biosystems, CA, USA). The fluorogenic probes are labeled with a reporter dye (either carboxyfluorescein or VIC^®^, Applied Biosystems) and are specific for one of the two possible bases identified at the seven SNPs rs1801279 (191G>A), rs1041983 (282C>T), rs1801280 (341T>C), rs1799929 (481C>T), rs1799930 (590G>A), rs1208 (803A>G) and rs1799931 (857G>A) in the *NAT2* coding region.

The presence of the rs1495741 SNP was determined using a predeveloped TaqMan^®^ SNP Genotyping Assays, Assay ID C_86841101–0 (Applied Biosystems) according to manufacturer’s instructions. Briefly the 20× primer and probe mix supplied (Assay ID C_8684110_10) was used at a 1× concentration with a half volume of 2× TaqMan Universal PCR Master Mix and 12.0 ng of genomic sample per well. Assays were designed and optimized to work with TaqMan^®^ Universal PCR Master Mix (with or without AmpErase^®^ UNG) using the same thermal cycling conditions, which were 60°C for 30 s, 95°C for 10 min followed by 40 cycles of 95°C for 15 s and 60°C for 1 min. The PCR amplification and end point reading were performed on StepOnePlus™ real-time instrument (Applied Biosystems).

Controls (no DNA template) were run to ensure that there was no amplification of contaminating DNA.

## NAT2 genotype & inferred phenotype

Determination of the seven *NAT2* coding region SNPs described above, define 34 of the reported human *NAT2* alleles or haplotypes ([Table 1](#T1)). *NAT2* alleles possessing additional SNPs in the *NAT2* coding region are quite rare [[101](#R52)]. Individuals possessing two of the *NAT2* alleles associated with rapid acetylation activity (*NAT2*4, NAT2*11, NAT2*12* and *NAT2*13* clusters) were classified as rapid acetylators; individuals possessing one of these alleles and one allele associated with slow acetylation (*NAT2*5*, *NAT2*6*, *NAT2*7* and *NAT2*14* clusters) were classified as intermediate acetylators, and those individuals that possessed two slow acetylation alleles were classified as slow acetylators as previously described [[18](#R18)].

### Table 1.

| NAT2 allele (haplotype) | Nucleotide change(s) and rs identifiers | Amino acid change(s) |
| --- | --- | --- |
| NAT2*4 | Reference | Reference |
|   |  |  |
| NAT2*5A | 341T>C (rs1801280)† | I114T† |
| 481C>T (rs1799929) | L161L (synonymous) |  |
|   |  |  |
| NAT2*5B | 341T>C (rs1801280)† | I114T† |
| 481C>T (rs1799929) | L161L (synonymous) |  |
| 803A>G (rs1208) | K268R |  |
|   |  |  |
| NAT2*5C | 341T>C (rs1801280)† | I114T† |
| 803A>G (rs1208) | K268R |  |
|   |  |  |
| NAT2*5D | 341T>C (rs1801280)† | I114T† |
|   |  |  |
| NAT2*5E | 341T>C (rs1801280)† | I114T† |
| 590G>A (rs1799930) | R197Q |  |
|   |  |  |
| NAT2*5G | 282C>T (rs1041983) | Y94Y (synonymous) |
| 341T>C (rs1801280)† | I114T† |  |
| 481C>T (rs1799929) | L161L (synonymous) |  |
| 803A>G (rs1208) | K268R |  |
|   |  |  |
| NAT2*5J | 282C>T (rs1041983) | Y94Y (synonymous) |
| 341T>C (rs1801280)† | I114T† |  |
| 590G>A (rs1799930) | R197Q |  |
|   |  |  |
| NAT2*5K | 282C>T (rs1041983) | Y94Y (synonymous) |
| 341T>C (rs1801280)† | I114T† |  |
|   |  |  |
| NAT2*6A | 282C>T (rs1041983) | Y94Y (synonymous) |
| 590G>A (rs1799930)† | R197Q† |  |
|   |  |  |
| NAT2*6B | 590G>A (rs1799930)† | R197Q† |
|   |  |  |
| NAT2*6C | 282C>T (rs1041983) | Y94Y (synonymous) |
| 590G>A (rs1799930)† | R197Q† |  |
| 803A>G (rs1208) | K268R |  |
|   |  |  |
| NAT2*6E | 481C>T (rs1799929) | L161L (synonymous) |
| 590G>A (rs1799930)† | R197Q† |  |
|   |  |  |
| NAT2*6F | 590G>A (rs1799930)† | R197Q† |
| 803A>G (rs1208) | K268R |  |
|   |  |  |
| NAT2*6J | 282C>T (rs1041983) | Y94Y (synonymous) |
| 590G>A (rs1799930)† | R197Q† |  |
| 857G>A (rs1799931) | G286E |  |
|   |  |  |
| NAT2*6N | 282C>T (rs1041983) | Y94Y (synonymous) |
| 481C>T (rs1799929) | L161L (synonymous) |  |
| 590G>A (rs1799930)† | R197Q† |  |
|   |  |  |
| NAT2*7A | 857G>A (rs1799931)† | G286E† |
|   |  |  |
| NAT2*7B | 282C>T (rs1041983) | Y94Y (synonymous) |
| 857G>A (rs1799931)† | G286E† |  |
|   |  |  |
| NAT2*7C | 282C>T (rs1041983) | Y94Y (synonymous) |
| 803A>G (rs1208) | K268R |  |
| 857G>A (rs1799931)† | G286E† |  |
|   |  |  |
| NAT2*7D | 191G>A (rs1801279) | R64Q |
| 282C>T (rs1041983) | Y94Y (synonymous) |  |
| 857G>A (rs1799931)† | G286E† |  |
|   |  |  |
| NAT2*7E | 282C>T (rs1041983) | Y94Y (synonymous) |
| 481C>T (rs1799929) | L161L (synonymous) |  |
| 857G>A (rs1799931)† | G286E† |  |
|   |  |  |
| NAT2*11A | 481C>T (rs1799929)† | L161L (synonymous)† |
|   |  |  |
| NAT2*12A | 803A>G (rs1208)† | K268R† |
|   |  |  |
| NAT2*12B | 282C>T (rs1041983) | Y94Y (synonymous) |
| 803A>G (rs1208)† | K268R† |  |
|   |  |  |
| NAT2*12C | 481C>T (rs1799929) | L161L (synonymous) |
| 803A>G (rs1208)† | K268R† |  |
|   |  |  |
| NAT2*13A | 282C>T (rs1041983)† | Y94Y (synonymous)† |
|   |  |  |
| NAT2*14A | 191G>A (rs1801279)† | R64Q† |
|   |  |  |
| NAT2*14B | 191G>A (rs1801279)† | R64Q† |
| 282C>T (rs1041983) | Y94Y (synonymous) |  |
|   |  |  |
| NAT2*14C | 191G>A (rs1801279)† | R64Q† |
| 341T>C (rs1801280) | I114T |  |
| 481C>T (rs1799929) | L161L (synonymous) |  |
| 803A>G (rs1208) | K268R |  |
|   |  |  |
| NAT2*14D | 191G>A (rs1801279)† | R64Q† |
| 282C>T (rs1041983) | Y94Y (synonymous) |  |
| 590G>A (rs1799930) | R197Q |  |
|   |  |  |
| NAT2*14E | 191G>A (rs1801279)† | R64Q† |
| 803A>G (rs1208) | K268R |  |
|   |  |  |
| NAT2*14F | 191G>A (rs1801279)† | R64Q† |
| 341T>C (rs1801280) | I114T |  |
| 803A>G (rs1208) | K268R |  |
|   |  |  |
| NAT2*14G | 191G>A (rs1801279)† | R64Q† |
| 282C>T (rs1041983) | Y94Y (synonymous) |  |
| 803A>G (rs1208) | K268R |  |
|   |  |  |
| NAT2*14I | 191G>A (rs1801279)† | R64Q† |
| 481C>T (rs1799929) | L161L (synonymous) |  |
| 803A>G (rs1208) | K268R |  |

Table 1 Caption: Common NAT2 single nucleotide pairs and their corresponding alleles/haplotypes.

The two-SNP inferred *NAT2* acetylator phenotype was determined by analysis of two SNPs: rs1041983 (282C>T) and rs1801280 (341T>C). The three-SNP inferred *NAT2* acetylator phenotype was determined by analysis of three SNPs: rs1799929 (481C>T), rs1799930 (590G>A) and rs1799931 (857G>A). The four-SNP inferred *NAT2* acetylator phenotype was determined by an assay of four SNPs: rs1801279 (191G>A), rs1801280 (341T>C), rs1799930 (590G>A) and rs1799931 (857G>A). For each of the two-, three- and four-SNP panel assays, samples homozygous common for all SNPs were classified as rapid acetylator phenotype, samples heterozygous for any one of the SNPs were classified as intermediate acetylator phenotype and samples homozygous variant for one or more SNPs or heterozygous for two or more SNPs were classified as slow acetylator phenotype.

## NAT2 catalytic activity

Upon removal from the liquid nitrogen, the hepatocytes were thawed and centrifuged as described previously [[17](#R17)]. Hepatocytes were lysed in 20 mM sodium phosphate (pH 7.4), 1 mM EDTA, 1 mM dithiothreitol, 100 μM phenylmethanesulfonyl fluoride, 1 μg/ml aprotinin and 1 μM pepstatin A by three rounds of freeze (−70°C)/thawing (37°C). Lysates were centrifuged at 15,000 × g for 20 min at 4°C and the resulting supernatant assayed for protein and enzymatic activity as described later.

SMZ *N*-acetyltransferase assays were conducted as previously described [[20](#R20)]. Briefly, reactions containing the supernatant of hepatocyte lysate (<2 mg/ml protein), SMZ (300 μM) and acetyl coenzyme A (1 mM) were incubated at 37°C. Reactions were terminated at 10 min by the addition of 1/10 volume 1 M acetic acid. The reaction tubes were centrifuged to precipitate the protein. The reaction tubes were centrifuged to precipitate the protein and supernatant was injected onto a Lichrospher^®^ 100 RP-18 (5 mM) reversed phase column. Samples were eluted using a Beckman System Gold^®^ HPLC system equipped with a model 126 programmable solvent module pump and a model 166 programmable UV-visible detector, and the absorbance was measured at 260 nm. The minimum level of detection of *N*-acetyl-SMZ was 5 pmol. Briefly, the substrates and acetylated products were separated with a mobile-phase of 20 mM sodium perchlorate (pH 2.5; solvent A) and acetonitrile (solvent B). A gradient of solvent B from 9 to 29%, over 5 min at a flow rate of 2 ml/min was used to separate SMZ and *N*-acetyl-SMZ. Under these conditions SMZ and *N*-acetyl-SMZ were eluted at 5.2 and 8.4 min, respectively.

## Data analysis

Differences in NAT2 catalytic activity among SNPs or haplotypes were analyzed for significance by one-way analysis of variance and Tukey *post hoc* tests using GraphPad Prism^®^ Software (Graphpad Software, CA, USA). *NAT2* acetylator genotype/phenotype misclassification was estimated by visual inspection of the data.

## Results

SMZ *N*-acetyltransferase catalytic activities in cryopreserved human hepatocytes were analyzed for each individual *NAT2* SNP. Many of the SNPs (282C>T, 341T>C, 481T>C, 590G>A, 803G>A and rs1495741G>A) differed significantly (p < 0.0001) in SMZ *N*-acetyltransferase activities among homozygous common, heterozygote and homozygous variant samples following one way analysis of variance ([Figure 1](#F1)).

### Figure 1. Sulfamethazine N-acetyltransferase catalytic activities (mean ± standard error of the mean) in cryopreserved human hepatocytes are plotted on the ordinate versus each individual NAT2 SNP on the abscissa.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0824/3285565/b2aa71fe2987/nihms356750f1.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3285565_nihms356750f1.jpg)

The white bar is not shown for 191G>A and 857G>A because no homozygous variant samples were available. Sulfamethazine N-acetyltransferase activities for 282C>T, 341T>C, 481T>C, 590G>A, 803G>A and rs1495741G>A all differed significantly (p < 0.0001) among homozygous common, heterozygote and homozygous variant following one-way analysis of variance. For each of these SNPs, homozygous common was significantly higher than heterozygote (p < 0.05), which was significantly higher than homozygous variant samples (p < 0.05) following one-way analysis of variance and Tukey post hoc tests. Homozygous common was not significantly different than the heterozygote for 191G>A (p = 0.6522) or 857G>A (p = 0.7185).

SMZ *N*-acetyltransferase catalytic activities were segregated by individual *NAT2* SNPs and no individual SNP, including rs1495741 yielded a nonoverlapping trimodal distribution ([Figure 2](#F2)) of rapid, intermediate and slow acetylator phenotypes. However, when SMZ *N*-acetyltransferase activities were segregated by *NAT2* genotype ([Figure 3](#F3)), a clear trimodal and nonoverlapping distribution of rapid, intermediate and slow acetylator phenotypes was observed and differences among *NAT2* acetylator phenotypes within an *NAT2* genotype group were trivial.

### Figure 2. Scatter plots of sulfamethazine N-acetyltransferase catalytic activities in cryopreserved human hepatocytes are plotted on the ordinate versus each NAT2 single nucleotide pair on the abscissa.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0824/3285565/0c5e9ab4812b/nihms356750f2.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3285565_nihms356750f2.jpg)

Each homozygous common sample is illustrated with an open square, each heterozygote is illustrated with an open circle, and each homozygous variant is illustrated with an open triangle. Mean activities are shown by the horizontal lines. Statistical analysis of these data is shown in the legend to Figure 1.

### Figure 3. Sulfamethazine N-acetyltransferase catalytic activities (mean ± standard error of the mean) in cryopreserved human hepatocytes are plotted on the ordinate versus NAT2 genotype(s) on the abscissa.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0824/3285565/80c2e1c42ea4/nihms356750f3.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3285565_nihms356750f3.jpg)

The number of samples in each NAT2 acetylator genotype group is shown and related NAT2 genotypes were combined when sample sizes were less than 4.

In the absence of a gold-standard reference, determinations of specificity and selectivity cannot be calculated with certainty. However, based on visual inspection of the data, *NAT2* acetylator genotype/phenotype misclassification can be estimated ([Figure 4](#F4)). Use of the rs1495741 genotype alone resulted in an inferred phenotype accuracy of 199/256 or 77.7% of the hepatocytes sampled in this study. The misclassification included 32 rapid acetylators, 22 intermediate acetylators and only three slow acetylators. Thus, the rs1495741 SNP misclassified rapid and intermediate acetylators to a much greater extent than it did slow acetylators. Use of the historical three-SNP rs1799929 (481C>T), rs1799930 (590G>A) and rs1799931 (857G>A) genotyping protocol resulted in an inferred phenotype accuracy of 236/256 or 92.2% of the hepatocytes sampled this study. Misclassification was almost entirely intermediate acetylators (19/20; 95%). A two-SNP rs1041983 (282C>T) and rs1801280 (341T>C) genotyping protocol resulted in an inferred phenotype accuracy of 246/256 or 96.1% of the hepatocytes sampled in this study. Misclassification with the two-SNP genotype protocol was equally divided between slow (five) and intermediate (five) acetylators. A four-SNP panel of rs1801279 (191G>A), rs1801280 (341T>C), rs1799930 (590G>A) and rs1799931 (857G>A) resulted in an inferred NAT2 acetylator phenotype accuracy of 252/256 or 98.4% of the hepatocytes sampled this study. The misclassification included no rapid acetylators, three intermediate acetylators and one slow acetylator. The seven-SNP assay resulted in an identical inferred *NAT2* acetylator phenotype accuracy determined by the four-SNP assay. The extent of acetylator phenotype misclassification with these *NAT2* genotyping protocols is illustrated in [Figure 5](#F5).

### Figure 4. Scatter plots of SMZ N-acetyltransferase catalytic activities plotted on the abscissa versus inferred acetylator phenotype as determined by various NAT2 genotyping protocols.

![Figure 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0824/3285565/1c11f17eec9b/nihms356750f4.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3285565_nihms356750f4.jpg)

Each NAT2 rapid acetylator sample is illustrated with an open square, each NAT2 intermediate acetylator sample is illustrated with an open circle, and each NAT2 slow acetylator sample is illustrated with an open triangle. Mean activities are shown by the vertical lines. Since the sulfamethazine N-acetyltransferase catalytic activity distribution was not explicitly trimodal in our population of samples, the identification of misclassified inferred NAT2 phenotypes was estimated by visual inspection of the data (illustrated by the inserted boxes).

### Figure 5.

![Figure 5](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0824/3285565/267cf6b9223e/nihms356750f5.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3285565_nihms356750f5.jpg)

Misclassification (percentage) of inferred acetylator phenotype as determined with various NAT2 SNPs genotyping protocols.

## Discussion

*N*-acetylation capacities determined *in vivo* have primarily discriminated between rapid versus slow acetylator phenotypes. Although less commonly reported, trimodal (rapid, intermediate and slow acetylator phenotype) distributions of *N*-acetylation capacity *in vivo* have been reported in humans following administrations of SMZ [[21](#R21),[22](#R22)], isoniazid [[23](#R23)–[26](#R26)], sulfasalazine [[27](#R27)] and caffeine [[28](#R28)–[29](#R29)].

Hepatic *N*-acetylation capacity *in vitro* often segregates into rapid, intermediate, and slow acetylator phenotypes as reported in humans [[17](#R17),[23](#R23)], mice [[30](#R30)], Syrian hamsters [[31](#R31),[32](#R32)] and rats [[33](#R33),[34](#R34)]. NAT2 is highly expressed in cryopreserved liver hepatocytes [[17](#R17)] and SMZ is a highly selective substrate for NAT2 versus NAT1 [[35](#R35),[36](#R36)], facilitating comparisons of NAT2 phenotype accuracy inferred from different *NAT2* SNP genotype panels.

In our study, we assessed the relative accuracy of various *NAT2* SNP genotyping panels to accurately distinguish among rapid, intermediate and slow acetylator phenotypes. Previous studies include calculations of sensitivity and specificity for various assays. We chose not to analyze our data in this manner because:

Thus, our genotyping ‘accuracy’ comparisons are based upon the phenotype method chosen (SMZ *N*-acetyltransferase catalytic activities in cryopreserved human hepatocytes) and the ethnic composition of our sample set.

SMZ *N*-acetyltransferase catalytic activities in cryopreserved human hepatocytes were segregated by individual *NAT2* SNPs and no individual SNP yielded a nonoverlapping trimodal distribution. The effects of each SNP on NAT2 activity was consistent with the effects that have been shown following recombinant expression of human *NAT2* alleles in bacteria [[37](#R37)–[39](#R39)], yeast [[40](#R40),[41](#R41)] and mammalian cells [[4](#R4),[42](#R42),[43](#R43)].

Use of the rs1495741 genotype alone resulted in an inferred phenotype accuracy of 77.7% of the hepatocytes sampled in this study. The rs1495741 is located within a large linkage disequilibrium block including each of the seven SNPs determined our study. As reported by Garcia-Closas *et al.*, linkage disequilibrium of the seven *NAT2* SNPs to the rs1495741 SNP was highest for 341T>C (pairwise r^2^ = 0.24), 481T>C (pairwise r^2^ = 0 0.23) and 803A>G (pairwise r^2^ = 0.21) [[16](#R16)]. These three SNPs define the *NAT2*5B* haplotype ([Table 1](#T1)), which is the most common slow acetylator haplotype in European populations. By contrast, linkage disequilibrium of the 857G>A SNP, common in Asian populations was dramatically less (pairwise r^2^ = 0.01), suggesting that the rs1495741 SNP alone would be much less accurate in the inference of *NAT2* phenotype in populations of Asian ancestry. The three-SNP panel and the rs1495741 SNP should be more accurate to infer *NAT2* phenotype if restricted to populations of European ancestry.

Subsequent to the submission of our manuscript, a recent study proposed that a two-SNP (282C>T, 341T>C) assay was more accurate than the tag-SNP and nearly as accurate as the seven-SNP assay for discriminating rapid versus slow acetylator *NAT2* phenotype as determined *in vivo* with caffeine [[44](#R44)]. We determined the accuracy of this two-SNP assay for discriminating rapid, intermediate or slow acetylator phenotypes as measured by SMZ NAT2 catalytic activity in cryopreserved human hepatocytes and also found it to be more accurate than both the tag-SNP and the historical three-SNP assay.

However, we found that a four-SNP panel rs1801279 (191G>A), rs1801280 (341T>C), rs1799930 (590G>A) and rs1799931 (857G>A) was still more accurate for inferring SMZ *N*-acetyltransferase activity phenotypes in cryopreserved human hepatocytes. This four-SNP assay measures the common SNPs that directly confer reductions in NAT2 catalytic activity. Unlike the two-SNP assay, it does not misclassify *NAT2*13A* alleles as slow acetylators. *NAT2*13A* allele frequency is substantially more common among populations of African than European ancestry [[45](#R45)–[47](#R47)], indicating this misclassification takes on greater significance in non-European populations.

Determination of acetylator genotype has been shown to improve efficacy and reduce toxicity in tuberculosis patients treated with isoniazid [[48](#R48)–[50](#R50)]. Indeed, *NAT2* genotyping may be superior to therapeutic drug monitoring for isoniazid dose individualization in tuberculosis therapy as specific and distinct dosing recommendations have been proposed for rapid, intermediate and slow acetylator genotypes [[26](#R26)] The *NAT2* tag-SNP rs1495741, is strongly associated with urinary bladder cancer [[15](#R15)], plasma triglyceride levels and coronary artery disease [[51](#R51)] in recent genome-wide association studies. Although DNA sequencing of the entire coding exon would provide the most comprehensive haplotype and genotype information, our results suggest that use of the four-SNP panel will infer NAT2 acetylator phenotype with high accuracy, and is recommended over the tag-, two-, three- and (for economy of scale) the seven-SNP genotyping panels, particularly in populations that include individuals of non-European ancestry.

### Background

### Background

### Background

### Background

### Materials & methods

### Results

### Conclusion

## Footnotes

## References

1. Evans DA, Manley KA, McKusick VA. Genetic control of isoniazid metabolism in man. Br Med J. 1960;2:485–491. doi: 10.1136/bmj.2.5197.485.  [DOI](https://doi.org/10.1136/bmj.2.5197.485) | [PMC free article](/articles/PMC2097108/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/13820968/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20Med%20J&title=Genetic%20control%20of%20isoniazid%20metabolism%20in%20man&author=DA%20Evans&author=KA%20Manley&author=VA%20McKusick&volume=2&publication_year=1960&pages=485-491&pmid=13820968&doi=10.1136/bmj.2.5197.485&)

2. Weber WW, Hein DW. N-acetylation pharmacogenetics. Pharmacol Rev. 1985;37:25–79.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/2860675/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacol%20Rev&title=N-acetylation%20pharmacogenetics&author=WW%20Weber&author=DW%20Hein&volume=37&publication_year=1985&pages=25-79&pmid=2860675&)

3. Cascorbi I, Brockmoller J, Mrozikiewicz PM, Muller A, Roots I. Arylamine N-acetyltransferase activity in man. Drug Metab Rev. 1999;31:489–502. doi: 10.1081/dmr-100101932.  [DOI](https://doi.org/10.1081/dmr-100101932) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10335449/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Rev&title=Arylamine%20N-acetyltransferase%20activity%20in%20man&author=I%20Cascorbi&author=J%20Brockmoller&author=PM%20Mrozikiewicz&author=A%20Muller&author=I%20Roots&volume=31&publication_year=1999&pages=489-502&pmid=10335449&doi=10.1081/dmr-100101932&)

4. Blum M, Demierre A, Grant DM, Heim M, Meyer UA. Molecular mechanism of slow acetylation of drugs and carcinogens in humans. Proc Natl Acad Sci USA. 1991;88:5237–5241. doi: 10.1073/pnas.88.12.5237. Described the initial report into the functional effects of three common NAT2 SNPs and formed the basis for the historical three-SNP assay.  [DOI](https://doi.org/10.1073/pnas.88.12.5237) | [PMC free article](/articles/PMC51847/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/1675794/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20USA&title=Molecular%20mechanism%20of%20slow%20acetylation%20of%20drugs%20and%20carcinogens%20in%20humans&author=M%20Blum&author=A%20Demierre&author=DM%20Grant&author=M%20Heim&author=UA%20Meyer&volume=88&publication_year=1991&pages=5237-5241&pmid=1675794&doi=10.1073/pnas.88.12.5237&)

5. Vatsis KP, Martell KJ, Weber WW. Diverse point mutations in the human gene for polymorphic N-acetyltransferase. Proc Natl Acad Sci USA. 1991;88:6333–6337. doi: 10.1073/pnas.88.14.6333.  [DOI](https://doi.org/10.1073/pnas.88.14.6333) | [PMC free article](/articles/PMC52077/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/2068113/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20USA&title=Diverse%20point%20mutations%20in%20the%20human%20gene%20for%20polymorphic%20N-acetyltransferase&author=KP%20Vatsis&author=KJ%20Martell&author=WW%20Weber&volume=88&publication_year=1991&pages=6333-6337&pmid=2068113&doi=10.1073/pnas.88.14.6333&)

6. Boukouvala S, Fakis G. Arylamine N-acetyltransferases: what we learn from genes and genomes. Drug Metab Rev. 2005;37:511–564. doi: 10.1080/03602530500251204.  [DOI](https://doi.org/10.1080/03602530500251204) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16257833/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Rev&title=Arylamine%20N-acetyltransferases:%20what%20we%20learn%20from%20genes%20and%20genomes&author=S%20Boukouvala&author=G%20Fakis&volume=37&publication_year=2005&pages=511-564&pmid=16257833&doi=10.1080/03602530500251204&)

7. Grant DM, Goodfellow GH, Sugamori K, Durette K. Pharmacogenetics of the human arylamine N-acetyltransferases. Pharmacology. 2000;61:204–211. doi: 10.1159/000028402.  [DOI](https://doi.org/10.1159/000028402) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10971207/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacology&title=Pharmacogenetics%20of%20the%20human%20arylamine%20N-acetyltransferases&author=DM%20Grant&author=GH%20Goodfellow&author=K%20Sugamori&author=K%20Durette&volume=61&publication_year=2000&pages=204-211&pmid=10971207&doi=10.1159/000028402&)

8. Sim E, Lack N, Wang CJ, et al. Arylamine N-acetyltransferases: structural and functional implications of polymorphisms. Toxicology. 2008;254:170–183. doi: 10.1016/j.tox.2008.08.022.  [DOI](https://doi.org/10.1016/j.tox.2008.08.022) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18852012/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Toxicology&title=Arylamine%20N-acetyltransferases:%20structural%20and%20functional%20implications%20of%20polymorphisms&author=E%20Sim&author=N%20Lack&author=CJ%20Wang&volume=254&publication_year=2008&pages=170-183&pmid=18852012&doi=10.1016/j.tox.2008.08.022&)

9. Hein DW, Doll MA, Fretland AJ, et al. Molecular genetics and epidemiology of the NAT1 and NAT2 acetylation polymorphisms. Cancer Epidemiol Biomarkers Prev. 2000;9:29–42. One of the first comprehensive reviews on the role of NAT2 polymorphisms in cancer risk. The paper also discusses possible genotyping misclassification with the historical three-SNP genotype assay.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/10667461/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Epidemiol%20Biomarkers%20Prev&title=Molecular%20genetics%20and%20epidemiology%20of%20the%20NAT1%20and%20NAT2%20acetylation%20polymorphisms&author=DW%20Hein&author=MA%20Doll&author=AJ%20Fretland&volume=9&publication_year=2000&pages=29-42&pmid=10667461&)

10. Agundez JA. Polymorphisms of human N-acetyltransferases and cancer risk. Curr Drug Metab. 2008;9:520–531. doi: 10.2174/138920008784892083.  [DOI](https://doi.org/10.2174/138920008784892083) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18680472/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Drug%20Metab&title=Polymorphisms%20of%20human%20N-acetyltransferases%20and%20cancer%20risk&author=JA%20Agundez&volume=9&publication_year=2008&pages=520-531&pmid=18680472&doi=10.2174/138920008784892083&)

11. Walraven JM, Zang Y, Trent JO, Hein DW. Structure/function evaluations of single nucleotide polymorphisms in human N-acetyltransferase 2. Curr Drug Metab. 2008;9:471–486. doi: 10.2174/138920008784892065.  [DOI](https://doi.org/10.2174/138920008784892065) | [PMC free article](/articles/PMC2507886/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18680467/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Drug%20Metab&title=Structure/function%20evaluations%20of%20single%20nucleotide%20polymorphisms%20in%20human%20N-acetyltransferase%202&author=JM%20Walraven&author=Y%20Zang&author=JO%20Trent&author=DW%20Hein&volume=9&publication_year=2008&pages=471-486&pmid=18680467&doi=10.2174/138920008784892065&)

12. Hein DW. N-acetyltransferase SNPs: emerging concepts serve as a paradigm for understanding complexities of personalized medicine. Expert Opin Drug Metab Toxicol. 2009;5:353–366. doi: 10.1517/17425250902877698. Summarizes the functional effects of NAT2 SNPs.  [DOI](https://doi.org/10.1517/17425250902877698) | [PMC free article](/articles/PMC2762189/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19379125/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Expert%20Opin%20Drug%20Metab%20Toxicol&title=N-acetyltransferase%20SNPs:%20emerging%20concepts%20serve%20as%20a%20paradigm%20for%20understanding%20complexities%20of%20personalized%20medicine&author=DW%20Hein&volume=5&publication_year=2009&pages=353-366&pmid=19379125&doi=10.1517/17425250902877698&)

13. Deitz AC, Rothman N, Rebbeck TR, et al. Impact of misclassification in genotype-exposure interaction studies: example of N-acetyltransferase 2 (NAT2), smoking, and bladder cancer. Cancer Epidemiol Biomarkers Prev. 2004;13:1543–1546. Reports NAT2 genotyping misclassification with the historical three-SNP assay and the consequences for gene–environmental interaction analyses.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/15342459/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Epidemiol%20Biomarkers%20Prev&title=Impact%20of%20misclassification%20in%20genotype-exposure%20interaction%20studies:%20example%20of%20N-acetyltransferase%202%20(NAT2),%20smoking,%20and%20bladder%20cancer&author=AC%20Deitz&author=N%20Rothman&author=TR%20Rebbeck&volume=13&publication_year=2004&pages=1543-1546&pmid=15342459&)

14. Doll MA, Fretland AJ, Deitz AC, Hein DW. Determination of human NAT2 acetylator genotype by restriction fragment-length polymorphism and allele-specific amplification. Anal Biochem. 1995;231:413–420. doi: 10.1006/abio.1995.9978.  [DOI](https://doi.org/10.1006/abio.1995.9978) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8594994/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Anal%20Biochem&title=Determination%20of%20human%20NAT2%20acetylator%20genotype%20by%20restriction%20fragment-length%20polymorphism%20and%20allele-specific%20amplification&author=MA%20Doll&author=AJ%20Fretland&author=AC%20Deitz&author=DW%20Hein&volume=231&publication_year=1995&pages=413-420&pmid=8594994&doi=10.1006/abio.1995.9978&)

15. Rothman N, Garcia-Closas M, Chatterjee N, et al. A multi-stage genome-wide association study of bladder cancer identifies multiple susceptibility loci. Nat Genet. 2010;42:978–984. doi: 10.1038/ng.687. Describes the genome-wide association study that identified a new tag-SNP for slow acetylation and increased risk for urinary bladder cancer.  [DOI](https://doi.org/10.1038/ng.687) | [PMC free article](/articles/PMC3049891/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20972438/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=A%20multi-stage%20genome-wide%20association%20study%20of%20bladder%20cancer%20identifies%20multiple%20susceptibility%20loci&author=N%20Rothman&author=M%20Garcia-Closas&author=N%20Chatterjee&volume=42&publication_year=2010&pages=978-984&pmid=20972438&doi=10.1038/ng.687&)

16. Garcia-Closas M, Hein DW, Silverman D, et al. A single nucleotide polymorphism tags variation in the arylamine N-acetyltransferase 2 phenotype in populations of European background. Pharmacogenet Genomics. 2011;21:231–236. doi: 10.1097/FPC.0b013e32833e1b54. Reported associations between the newly identified tag-SNP and arylamine N-acetyltransferase activities in human hepatocytes from a population of individuals of European ancestry.  [DOI](https://doi.org/10.1097/FPC.0b013e32833e1b54) | [PMC free article](/articles/PMC3003749/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20739907/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=A%20single%20nucleotide%20polymorphism%20tags%20variation%20in%20the%20arylamine%20N-acetyltransferase%202%20phenotype%20in%20populations%20of%20European%20background&author=M%20Garcia-Closas&author=DW%20Hein&author=D%20Silverman&volume=21&publication_year=2011&pages=231-236&pmid=20739907&doi=10.1097/FPC.0b013e32833e1b54&)

17. Doll MA, Zang Y, Moeller T, Hein DW. Codominant expression of N-acetylation and O-acetylation activities catalyzed by N-acetyltransferase 2 in human hepatocytes. J Pharmacol Exp Ther. 2010;334:540–544. doi: 10.1124/jpet.110.168567.  [DOI](https://doi.org/10.1124/jpet.110.168567) | [PMC free article](/articles/PMC2913773/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20430842/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharmacol%20Exp%20Ther&title=Codominant%20expression%20of%20N-acetylation%20and%20O-acetylation%20activities%20catalyzed%20by%20N-acetyltransferase%202%20in%20human%20hepatocytes&author=MA%20Doll&author=Y%20Zang&author=T%20Moeller&author=DW%20Hein&volume=334&publication_year=2010&pages=540-544&pmid=20430842&doi=10.1124/jpet.110.168567&)

18. Doll MA, Hein DW. Comprehensive human NAT2 genotype method using single nucleotide polymorphism-specific polymerase chain reaction primers and fluorogenic probes. Anal Biochem. 2001;288:106–108. doi: 10.1006/abio.2000.4892.  [DOI](https://doi.org/10.1006/abio.2000.4892) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11141315/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Anal%20Biochem&title=Comprehensive%20human%20NAT2%20genotype%20method%20using%20single%20nucleotide%20polymorphism-specific%20polymerase%20chain%20reaction%20primers%20and%20fluorogenic%20probes&author=MA%20Doll&author=DW%20Hein&volume=288&publication_year=2001&pages=106-108&pmid=11141315&doi=10.1006/abio.2000.4892&)

19. Hein DW. N-acetyltransferase 2 genetic polymorphism: effects of carcinogen and haplotype on urinary bladder cancer risk. Oncogene. 2006;25:1649–1658. doi: 10.1038/sj.onc.1209374.  [DOI](https://doi.org/10.1038/sj.onc.1209374) | [PMC free article](/articles/PMC1434721/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16550165/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncogene&title=N-acetyltransferase%202%20genetic%20polymorphism:%20effects%20of%20carcinogen%20and%20haplotype%20on%20urinary%20bladder%20cancer%20risk&author=DW%20Hein&volume=25&publication_year=2006&pages=1649-1658&pmid=16550165&doi=10.1038/sj.onc.1209374&)

20. Leff MA, Epstein PN, Doll MA, et al. Prostate-specific human N-acetyltransferase 2 (NAT2) expression in the mouse. J Pharmacol Exp Ther. 1999;290:182–187.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/10381774/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharmacol%20Exp%20Ther&title=Prostate-specific%20human%20N-acetyltransferase%202%20(NAT2)%20expression%20in%20the%20mouse&author=MA%20Leff&author=PN%20Epstein&author=MA%20Doll&volume=290&publication_year=1999&pages=182-187&pmid=10381774&)

21. Chapron DJ, Kramer PA, Mercik SA. Kinetic discrimination of three sulfamethazine acetylation phenotypes. Clin Pharmacol Ther. 1980;27:104–113. doi: 10.1038/clpt.1980.16.  [DOI](https://doi.org/10.1038/clpt.1980.16) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7351110/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Kinetic%20discrimination%20of%20three%20sulfamethazine%20acetylation%20phenotypes&author=DJ%20Chapron&author=PA%20Kramer&author=SA%20Mercik&volume=27&publication_year=1980&pages=104-113&pmid=7351110&doi=10.1038/clpt.1980.16&)

22. Chen B, Zhang WX, Cai WM. The influence of various genotypes on the metabolic activity of NAT2 in a Chinese population. Eur J Clin Pharmacol. 2006;62:355–359. doi: 10.1007/s00228-006-0110-6.  [DOI](https://doi.org/10.1007/s00228-006-0110-6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16570187/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Clin%20Pharmacol&title=The%20influence%20of%20various%20genotypes%20on%20the%20metabolic%20activity%20of%20NAT2%20in%20a%20Chinese%20population&author=B%20Chen&author=WX%20Zhang&author=WM%20Cai&volume=62&publication_year=2006&pages=355-359&pmid=16570187&doi=10.1007/s00228-006-0110-6&)

23. Deguchi T, Mashimo M, Suzuki T. Correlation between acetylator phenotypes and genotypes of polymorphic arylamine N-acetyltransferase in human liver. J Biol Chem. 1990;265:12757–12760.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/2376572/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=Correlation%20between%20acetylator%20phenotypes%20and%20genotypes%20of%20polymorphic%20arylamine%20N-acetyltransferase%20in%20human%20liver&author=T%20Deguchi&author=M%20Mashimo&author=T%20Suzuki&volume=265&publication_year=1990&pages=12757-12760&pmid=2376572&)

24. Parkin DP, Vandenplas S, Botha FJ, et al. Trimodality of isoniazid elimination: phenotype and genotype in patients with tuberculosis. Am J Respir Crit Care Med. 1997;155:1717–1722. doi: 10.1164/ajrccm.155.5.9154882.  [DOI](https://doi.org/10.1164/ajrccm.155.5.9154882) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9154882/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Respir%20Crit%20Care%20Med&title=Trimodality%20of%20isoniazid%20elimination:%20phenotype%20and%20genotype%20in%20patients%20with%20tuberculosis&author=DP%20Parkin&author=S%20Vandenplas&author=FJ%20Botha&volume=155&publication_year=1997&pages=1717-1722&pmid=9154882&doi=10.1164/ajrccm.155.5.9154882&)

25. Smith CA, Wadelius M, Gough AC, Harrison DJ, Wolf CR, Rane A. A simplified assay for the arylamine N-acetyltransferase 2 polymorphism validated by phenotyping with isoniazid. J Med Genet. 1997;34:758–760. doi: 10.1136/jmg.34.9.758.  [DOI](https://doi.org/10.1136/jmg.34.9.758) | [PMC free article](/articles/PMC1051062/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9321764/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Med%20Genet&title=A%20simplified%20assay%20for%20the%20arylamine%20N-acetyltransferase%202%20polymorphism%20validated%20by%20phenotyping%20with%20isoniazid&author=CA%20Smith&author=M%20Wadelius&author=AC%20Gough&author=DJ%20Harrison&author=CR%20Wolf&volume=34&publication_year=1997&pages=758-760&pmid=9321764&doi=10.1136/jmg.34.9.758&)

26. Kinzig-Schippers M, Tomalik-Scharte D, Jetter A, et al. Should we use N-acetyltransferase type 2 genotyping to personalize isoniazid doses? Antimicrob Agents Chemother. 2005;49:1733–1738. doi: 10.1128/AAC.49.5.1733-1738.2005. Describes the importance of NAT2 genotyping in tuberculosis therapy.  [DOI](https://doi.org/10.1128/AAC.49.5.1733-1738.2005) | [PMC free article](/articles/PMC1087660/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15855489/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Antimicrob%20Agents%20Chemother&title=Should%20we%20use%20N-acetyltransferase%20type%202%20genotyping%20to%20personalize%20isoniazid%20doses?&author=M%20Kinzig-Schippers&author=D%20Tomalik-Scharte&author=A%20Jetter&volume=49&publication_year=2005&pages=1733-1738&pmid=15855489&doi=10.1128/AAC.49.5.1733-1738.2005&)

27. Ma JJ, Liu CG, Li JH, Cao XM, Sun SL, Yao X. Effects of NAT2 polymorphism on SASP pharmacokinetics in Chinese population. Clin Chim Acta. 2009;407:30–35. doi: 10.1016/j.cca.2009.06.025.  [DOI](https://doi.org/10.1016/j.cca.2009.06.025) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19560446/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Chim%20Acta&title=Effects%20of%20NAT2%20polymorphism%20on%20SASP%20pharmacokinetics%20in%20Chinese%20population&author=JJ%20Ma&author=CG%20Liu&author=JH%20Li&author=XM%20Cao&author=SL%20Sun&volume=407&publication_year=2009&pages=30-35&pmid=19560446&doi=10.1016/j.cca.2009.06.025&)

28. Grant DM, Tang BK, Kalow W. A simple test for acetylator phenotype using caffeine. Br J Clin Pharmacol. 1984;17:459–464. doi: 10.1111/j.1365-2125.1984.tb02372.x.  [DOI](https://doi.org/10.1111/j.1365-2125.1984.tb02372.x) | [PMC free article](/articles/PMC1463406/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/6721992/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Clin%20Pharmacol&title=A%20simple%20test%20for%20acetylator%20phenotype%20using%20caffeine&author=DM%20Grant&author=BK%20Tang&author=W%20Kalow&volume=17&publication_year=1984&pages=459-464&pmid=6721992&doi=10.1111/j.1365-2125.1984.tb02372.x&)

29. Cascorbi I, Drakoulis N, Brockmoller J, Maurer A, Sperling K, Roots I. Arylamine N-acetyltransferase (NAT2) mutations and their allelic linkage in unrelated Caucasian individuals: correlation with phenotypic activity. Am J Hum Genet. 1995;57:581–592.  [PMC free article](/articles/PMC1801274/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7668286/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=Arylamine%20N-acetyltransferase%20(NAT2)%20mutations%20and%20their%20allelic%20linkage%20in%20unrelated%20Caucasian%20individuals:%20correlation%20with%20phenotypic%20activity&author=I%20Cascorbi&author=N%20Drakoulis&author=J%20Brockmoller&author=A%20Maurer&author=K%20Sperling&volume=57&publication_year=1995&pages=581-592&pmid=7668286&)

30. Hein DW, Trinidad A, Yerokun T, Ferguson RJ, Kirlin WG, Weber WW. Genetic control of acetyl coenzyme A-dependent arylamine N-acetyltransferase, hydrazine N-acetyltransferase, and N-hydroxy-arylamine O-acetyltransferase enzymes in C57BL/6J, A/J, AC57F1, and the rapid and slow acetylator A.B6 and B6. A congenic inbred mouse. Drug Metab Dispos. 1988;16:341–347.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/2900723/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Genetic%20control%20of%20acetyl%20coenzyme%20A-dependent%20arylamine%20N-acetyltransferase,%20hydrazine%20N-acetyltransferase,%20and%20N-hydroxy-arylamine%20O-acetyltransferase%20enzymes%20in%20C57BL/6J,%20A/J,%20AC57F1,%20and%20the%20rapid%20and%20slow%20acetylator%20A.B6%20and%20B6.%20A%20congenic%20inbred%20mouse&author=DW%20Hein&author=A%20Trinidad&author=T%20Yerokun&author=RJ%20Ferguson&author=WG%20Kirlin&volume=16&publication_year=1988&pages=341-347&pmid=2900723&)

31. Hein DW, Rustan TD, Bucher KD, Miller LS. Polymorphic and monomorphic expression of arylamine carcinogen N-acetyltransferase isozymes in tumor target organ cytosols of Syrian-hamsters congenic at the polymorphic acetyltransferase locus. J Pharmacol Exp Ther. 1991;259:699–704.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/1941618/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharmacol%20Exp%20Ther&title=Polymorphic%20and%20monomorphic%20expression%20of%20arylamine%20carcinogen%20N-acetyltransferase%20isozymes%20in%20tumor%20target%20organ%20cytosols%20of%20Syrian-hamsters%20congenic%20at%20the%20polymorphic%20acetyltransferase%20locus&author=DW%20Hein&author=TD%20Rustan&author=KD%20Bucher&author=LS%20Miller&volume=259&publication_year=1991&pages=699-704&pmid=1941618&)

32. Hein DW, Doll MA, Rustan TD, Gray K, Ferguson RJ, Feng Y. Construction of Syrian hamster lines congenic at the polymorphic acetyltransferase locus (NAT2): acetylator genotype-dependent N- and O-acetylation of arylamine carcinogens. Toxicol Appl Pharmacol. 1994;124:16–24. doi: 10.1006/taap.1994.1003.  [DOI](https://doi.org/10.1006/taap.1994.1003) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8291057/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Toxicol%20Appl%20Pharmacol&title=Construction%20of%20Syrian%20hamster%20lines%20congenic%20at%20the%20polymorphic%20acetyltransferase%20locus%20(NAT2):%20acetylator%20genotype-dependent%20N-%20and%20O-acetylation%20of%20arylamine%20carcinogens&author=DW%20Hein&author=MA%20Doll&author=TD%20Rustan&author=K%20Gray&author=RJ%20Ferguson&volume=124&publication_year=1994&pages=16-24&pmid=8291057&doi=10.1006/taap.1994.1003&)

33. Hein DW, Rustan TD, Bucher KD, Martin WJ, Furman EJ. Acetylator phenotype-dependent and -independent expression of arylamine N-acetyltransferase isozymes in rapid and slow acetylator inbred rat liver. Drug Metab Dispos. 1991;19:933–937.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/1686239/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Acetylator%20phenotype-dependent%20and%20-independent%20expression%20of%20arylamine%20N-acetyltransferase%20isozymes%20in%20rapid%20and%20slow%20acetylator%20inbred%20rat%20liver&author=DW%20Hein&author=TD%20Rustan&author=KD%20Bucher&author=WJ%20Martin&author=EJ%20Furman&volume=19&publication_year=1991&pages=933-937&pmid=1686239&)

34. Hein DW, Bendaly J, Neale JR, Doll MA. Systemic functional expression of N-acetyltransferase polymorphism in the F344 Nat2 congenic rat. Drug Metab Dispos. 2008;36:2452–2459. doi: 10.1124/dmd.108.023960.  [DOI](https://doi.org/10.1124/dmd.108.023960) | [PMC free article](/articles/PMC2596689/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18799801/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Systemic%20functional%20expression%20of%20N-acetyltransferase%20polymorphism%20in%20the%20F344%20Nat2%20congenic%20rat&author=DW%20Hein&author=J%20Bendaly&author=JR%20Neale&author=MA%20Doll&volume=36&publication_year=2008&pages=2452-2459&pmid=18799801&doi=10.1124/dmd.108.023960&)

35. Grant DM, Blum M, Beer M, Meyer UA. Monomorphic and polymorphic human arylamine N-acetyltransferases: a comparison of liver isozymes and expressed products of two cloned genes. Mol Pharmacol. 1991;39:184–191.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/1996083/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Pharmacol&title=Monomorphic%20and%20polymorphic%20human%20arylamine%20N-acetyltransferases:%20a%20comparison%20of%20liver%20isozymes%20and%20expressed%20products%20of%20two%20cloned%20genes&author=DM%20Grant&author=M%20Blum&author=M%20Beer&author=UA%20Meyer&volume=39&publication_year=1991&pages=184-191&pmid=1996083&)

36. Hein DW, Doll MA, Rustan TD, et al. Metabolic activation and deactivation of arylamine carcinogens by recombinant human NAT1 and polymorphic NAT2 acetyltransferases. Carcinogenesis. 1993;14:1633–1638. doi: 10.1093/carcin/14.8.1633.  [DOI](https://doi.org/10.1093/carcin/14.8.1633) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8353847/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Carcinogenesis&title=Metabolic%20activation%20and%20deactivation%20of%20arylamine%20carcinogens%20by%20recombinant%20human%20NAT1%20and%20polymorphic%20NAT2%20acetyltransferases&author=DW%20Hein&author=MA%20Doll&author=TD%20Rustan&volume=14&publication_year=1993&pages=1633-1638&pmid=8353847&doi=10.1093/carcin/14.8.1633&)

37. Hein DW, Ferguson RJ, Doll MA, Rustan TD, Gray K. Molecular genetics of human polymorphic N-acetyltransferase: enzymatic analysis of 15 recombinant wild-type, mutant, and chimeric NAT2 allozymes. Hum Mol Genet. 1994;3:729–734. doi: 10.1093/hmg/3.5.729.  [DOI](https://doi.org/10.1093/hmg/3.5.729) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8081359/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mol%20Genet&title=Molecular%20genetics%20of%20human%20polymorphic%20N-acetyltransferase:%20enzymatic%20analysis%20of%2015%20recombinant%20wild-type,%20mutant,%20and%20chimeric%20NAT2%20allozymes&author=DW%20Hein&author=RJ%20Ferguson&author=MA%20Doll&author=TD%20Rustan&author=K%20Gray&volume=3&publication_year=1994&pages=729-734&pmid=8081359&doi=10.1093/hmg/3.5.729&)

38. Hein DW, Doll MA, Rustan TD, Ferguson RJ. Metabolic activation of N-hydroxyarylamines and N-hydroxyarylamides by 16 recombinant human NAT2 allozymes: effects of 7 specific NAT2 nucleic acid substitutions. Cancer Res. 1995;55:3531–3536.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/7627960/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=Metabolic%20activation%20of%20N-hydroxyarylamines%20and%20N-hydroxyarylamides%20by%2016%20recombinant%20human%20NAT2%20allozymes:%20effects%20of%207%20specific%20NAT2%20nucleic%20acid%20substitutions&author=DW%20Hein&author=MA%20Doll&author=TD%20Rustan&author=RJ%20Ferguson&volume=55&publication_year=1995&pages=3531-3536&pmid=7627960&)

39. Grant DM, Hughes NC, Janezic SA, et al. Human acetyltransferase polymorphisms. Mutat Res. 1997;376:61–70. doi: 10.1016/s0027-5107(97)00026-2.  [DOI](https://doi.org/10.1016/s0027-5107(97)00026-2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9202739/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mutat%20Res&title=Human%20acetyltransferase%20polymorphisms&author=DM%20Grant&author=NC%20Hughes&author=SA%20Janezic&volume=376&publication_year=1997&pages=61-70&pmid=9202739&doi=10.1016/s0027-5107(97)00026-2&)

40. Fretland AJ, Leff MA, Doll MA, Hein DW. Functional characterization of human N-acetyltransferase 2 (NAT2) single nucleotide polymorphisms. Pharmacogenetics. 2001;11:207–215. doi: 10.1097/00008571-200104000-00004.  [DOI](https://doi.org/10.1097/00008571-200104000-00004) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11337936/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=Functional%20characterization%20of%20human%20N-acetyltransferase%202%20(NAT2)%20single%20nucleotide%20polymorphisms&author=AJ%20Fretland&author=MA%20Leff&author=MA%20Doll&author=DW%20Hein&volume=11&publication_year=2001&pages=207-215&pmid=11337936&doi=10.1097/00008571-200104000-00004&)

41. Hein DW, Fretland AJ, Doll MA. Effects of single nucleotide polymorphisms in human N-acetyltransferase 2 on metabolic activation (O-acetylation) of heterocyclic amine carcinogens. Int J Cancer. 2006;119:1208–1211. doi: 10.1002/ijc.21957.  [DOI](https://doi.org/10.1002/ijc.21957) | [PMC free article](/articles/PMC2582010/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16570281/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Cancer&title=Effects%20of%20single%20nucleotide%20polymorphisms%20in%20human%20N-acetyltransferase%202%20on%20metabolic%20activation%20(O-acetylation)%20of%20heterocyclic%20amine%20carcinogens&author=DW%20Hein&author=AJ%20Fretland&author=MA%20Doll&volume=119&publication_year=2006&pages=1208-1211&pmid=16570281&doi=10.1002/ijc.21957&)

42. Zang Y, Zhao S, Doll MA, States JC, Hein DW. The T341C (Ile114Thr) polymorphism of N-acetyltransferase 2 yields slow acetylator phenotype by enhanced protein degradation. Pharmacogenetics. 2004;14:717–723. doi: 10.1097/00008571-200411000-00002.  [DOI](https://doi.org/10.1097/00008571-200411000-00002) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15564878/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=The%20T341C%20(Ile114Thr)%20polymorphism%20of%20N-acetyltransferase%202%20yields%20slow%20acetylator%20phenotype%20by%20enhanced%20protein%20degradation&author=Y%20Zang&author=S%20Zhao&author=MA%20Doll&author=JC%20States&author=DW%20Hein&volume=14&publication_year=2004&pages=717-723&pmid=15564878&doi=10.1097/00008571-200411000-00002&)

43. Zang Y, Doll MA, Zhao S, States JC, Hein DW. Functional characterization of single-nucleotide polymorphisms and haplotypes of human N-acetyltransferase 2. Carcinogenesis. 2007;28:1665–1671. doi: 10.1093/carcin/bgm085.  [DOI](https://doi.org/10.1093/carcin/bgm085) | [PMC free article](/articles/PMC2085371/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17434923/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Carcinogenesis&title=Functional%20characterization%20of%20single-nucleotide%20polymorphisms%20and%20haplotypes%20of%20human%20N-acetyltransferase%202&author=Y%20Zang&author=MA%20Doll&author=S%20Zhao&author=JC%20States&author=DW%20Hein&volume=28&publication_year=2007&pages=1665-1671&pmid=17434923&doi=10.1093/carcin/bgm085&)

44. Selinski S, Blaszkewicz M, Lehmann ML, et al. Genotyping NAT2 with only two SNPs (rs1041983 and rs1801280) outperforms the tagging SNP rs1495741 and is equivalent to the conventional 7-SNP NAT2 genotype. Pharmacogenet Genomics. 2011;21(10):673–678. doi: 10.1097/FPC.0b013e3283493a23.  [DOI](https://doi.org/10.1097/FPC.0b013e3283493a23) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21750470/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Genotyping%20NAT2%20with%20only%20two%20SNPs%20(rs1041983%20and%20rs1801280)%20outperforms%20the%20tagging%20SNP%20rs1495741%20and%20is%20equivalent%20to%20the%20conventional%207-SNP%20NAT2%20genotype&author=S%20Selinski&author=M%20Blaszkewicz&author=ML%20Lehmann&volume=21&issue=10&publication_year=2011&pages=673-678&pmid=21750470&doi=10.1097/FPC.0b013e3283493a23&)

45. Loktionov A, Moore W, Spencer SP, et al. Differences in N-acetylation genotypes between Caucasians and black South Africans: implications for cancer prevention. Cancer Detect Prev. 2002;26:15–22. doi: 10.1016/s0361-090x(02)00010-7.  [DOI](https://doi.org/10.1016/s0361-090x(02)00010-7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12088198/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Detect%20Prev&title=Differences%20in%20N-acetylation%20genotypes%20between%20Caucasians%20and%20black%20South%20Africans:%20implications%20for%20cancer%20prevention&author=A%20Loktionov&author=W%20Moore&author=SP%20Spencer&volume=26&publication_year=2002&pages=15-22&pmid=12088198&doi=10.1016/s0361-090x(02)00010-7&)

46. Sabbagh A, Langaney A, Darlu P, Gerard N, Krishnamoorthy R, Poloni ES. Worldwide distribution of NAT2 diversity: implications for NAT2 evolutionary history. BMC Genet. 2008;9:21. doi: 10.1186/1471-2156-9-21.  [DOI](https://doi.org/10.1186/1471-2156-9-21) | [PMC free article](/articles/PMC2292740/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18304320/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Genet&title=Worldwide%20distribution%20of%20NAT2%20diversity:%20implications%20for%20NAT2%20evolutionary%20history&author=A%20Sabbagh&author=A%20Langaney&author=P%20Darlu&author=N%20Gerard&author=R%20Krishnamoorthy&volume=9&publication_year=2008&pages=21&pmid=18304320&doi=10.1186/1471-2156-9-21&)

47. Sabbagh A, Darlu P, Crouau-Roy B, Poloni ES. Arylamine N-acetyltransferase 2 (NAT2) genetic diversity and traditional subsistence: a worldwide population survey. PLoS ONE. 2011;6:e18507. doi: 10.1371/journal.pone.0018507.  [DOI](https://doi.org/10.1371/journal.pone.0018507) | [PMC free article](/articles/PMC3071824/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21494681/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&title=Arylamine%20N-acetyltransferase%202%20(NAT2)%20genetic%20diversity%20and%20traditional%20subsistence:%20a%20worldwide%20population%20survey&author=A%20Sabbagh&author=P%20Darlu&author=B%20Crouau-Roy&author=ES%20Poloni&volume=6&publication_year=2011&pages=e18507&pmid=21494681&doi=10.1371/journal.pone.0018507&)

48. Ohno M, Yamaguchi I, Yamamoto I, et al. Slow N-acetyltransferase 2 genotype affects the incidence of isoniazid and rifampicin-induced hepatotoxicity. Int J Tuberc Lung Dis. 2000;4:256–261.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/10751073/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Tuberc%20Lung%20Dis&title=Slow%20N-acetyltransferase%202%20genotype%20affects%20the%20incidence%20of%20isoniazid%20and%20rifampicin-induced%20hepatotoxicity&author=M%20Ohno&author=I%20Yamaguchi&author=I%20Yamamoto&volume=4&publication_year=2000&pages=256-261&pmid=10751073&)

49. Hiratsuka M, Kishikawa Y, Takekuma Y, et al. Genotyping of the N-acetyltransferase 2 polymorphism in the prediction of adverse drug reactions to isoniazid in Japanese patients. Drug Metab Pharmacokinet. 2002;17:357–362. doi: 10.2133/dmpk.17.357.  [DOI](https://doi.org/10.2133/dmpk.17.357) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15618686/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Pharmacokinet&title=Genotyping%20of%20the%20N-acetyltransferase%202%20polymorphism%20in%20the%20prediction%20of%20adverse%20drug%20reactions%20to%20isoniazid%20in%20Japanese%20patients&author=M%20Hiratsuka&author=Y%20Kishikawa&author=Y%20Takekuma&volume=17&publication_year=2002&pages=357-362&pmid=15618686&doi=10.2133/dmpk.17.357&)

50. Cho HJ, Koh WJ, Ryu YJ, et al. Genetic polymorphisms of NAT2 and CYP2E1 associated with antituberculosis drug-induced hepatotoxicity in Korean patients with pulmonary tuberculosis. Tuberculosis. 2007;87:551–556. doi: 10.1016/j.tube.2007.05.012.  [DOI](https://doi.org/10.1016/j.tube.2007.05.012) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17950035/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Tuberculosis&title=Genetic%20polymorphisms%20of%20NAT2%20and%20CYP2E1%20associated%20with%20antituberculosis%20drug-induced%20hepatotoxicity%20in%20Korean%20patients%20with%20pulmonary%20tuberculosis&author=HJ%20Cho&author=WJ%20Koh&author=YJ%20Ryu&volume=87&publication_year=2007&pages=551-556&pmid=17950035&doi=10.1016/j.tube.2007.05.012&)

51. Teslovich TM, Musunuru K, Smith AV, et al. Biological, clinical and population relevance of 95 loci for blood lipids. Nature. 2010;466:7071–7073. doi: 10.1038/nature09270.  [DOI](https://doi.org/10.1038/nature09270) | [PMC free article](/articles/PMC3039276/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20686565/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Biological,%20clinical%20and%20population%20relevance%20of%2095%20loci%20for%20blood%20lipids&author=TM%20Teslovich&author=K%20Musunuru&author=AV%20Smith&volume=466&publication_year=2010&pages=7071-7073&pmid=20686565&doi=10.1038/nature09270&)
