# META-ANALYSIS OF CYP2D6 METABOLIZER PHENOTYPE AND METOPROLOL PHARMACOKINETICS

## Metadata
**Authors:** CM Blake, ED Kharasch, M Schwab, P Nagele
**Journal:** Clinical pharmacology and therapeutics
**Date:** 2013 May 9
**DOI:** [10.1038/clpt.2013.96](https://doi.org/10.1038/clpt.2013.96)
**PMID:** 23665868
**PMCID:** PMC3818912
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3818912/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC3818912/pdf/nihms-524157.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC3818912/pdf/nihms-524157.pdf)

## Abstract

Metoprolol, a commonly prescribed beta-blocker, is primarily metabolized by cytochrome P450 2D6 (CYP2D6), an enzyme with substantial genetic heterogeneity. Several smaller studies have shown that metoprolol pharmacokinetics is influenced by CYP2D6 genotype and metabolizer phenotype. To increase robustness of metoprolol pharmacokinetic estimates, a systematic review and meta-analysis of pharmacokinetic studies that administered a single oral dose of immediate release metoprolol was performed. Pooled analysis (n= 264) demonstrated differences in peak plasma metoprolol concentration, area under the concentration-time curve, elimination half-life, and apparent oral clearance that were 2.3-, 4.9-, 2.3-, and 5.9-fold between extensive and poor metabolizers, respectively, and 5.3-, 13-, 2.6-, and 15-fold between ultra-rapid and poor metabolizers (all p<0.001). Enantiomer-specific analysis revealed genotype-dependent enantio-selective metabolism, with nearly 40% greater R- vs S-metoprolol metabolism in ultra-rapid and extensive metabolizers. This study demonstrates a marked effect of CYP2D6 metabolizer phenotype on metoprolol pharmacokinetics and confirms enantiomer specific metabolism of metoprolol.

Keywords: metoprolol, CYP2D6, poor metabolizer, pharmacokinetics, pharmacogenetics, pharmacogenomics, meta-analysis

## INTRODUCTION

Metoprolol is one of the most commonly used beta-blockers in medicine and primarily used in the treatment of hypertension and heart failure. The cytochrome P450 2D6 (CYP2D6) enzyme is predominantly involved (approximately 70-80%) in the hepatic metabolism of metoprolol. ([1](#R1), [2](#R2)) Since the first description of a CYP2D6 sparteine-debrisoquin polymorphism in the mid-1970’s, substantial genetic heterogeneity has been reported with close to 100 different polymorphisms identified. ([3](#R3)-[6](#R6)) Some CYP2D6 polymorphisms render the enzyme completely inactive while others do not modify the activity of the enzyme. Based on these *CYP2D6* gene variants, four metabolizer phenotypes are used to characterize drug metabolism via CYP2D6 *in vivo*: ultra-rapid metabolizer (UM), extensive metabolizer (EM), intermediate metabolizer (IM) and poor metabolizer (PM) phenotype. As a result of these metabolizer phenotypes, the plasma concentration of metoprolol available for target effects may range from subtherapeutic levels in the UM group to supratherapeutic and potentially toxic concentrations in the PM group, increasing the probability of adverse effects such as hypotension and bradycardia. ([7](#R7), [8](#R8))

Pharmacokinetic studies often have small sample sizes that can result in less precise parameter estimates. Systematic reviews and meta-analyses have been developed to provide more robust estimates by pooling available data from several studies. Therefore, a pooled analysis of available pharmacokinetic data was performed in order to present a more accurate estimate of CYP2D6 metabolizer phenotype-dependent effects on metoprolol pharmacokinetics.

## RESULTS

The systematic review of the literature identified 13 independent studies that met the pre-defined search criteria ([6](#R6), [9](#R9)-[21](#R21)), with a total sample size of 264 participants ([Figure 1](#F1)). Two major analyses were performed: one for studies quantifying racemic metoprolol pharmacokinetics (n=172) and another for studies that distinguished between R- and S-metoprolol enantiomers (n=128). One study (n=36) provided data that were used in both analyses.

### Figure 1. PRISMA study selection flow diagram.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/cb25/3818912/4027f4edcb5d/nihms-524157-f0001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3818912_nihms-524157-f0001.jpg)

The Preferred Reporting Items for Systematic Reviews and Meta-Analyses (PRISMA) flow diagram depicting the steps undertaken in the systematic review prior to meta-analysis, including the number of records identified, included and excluded. Detailed reasons for exclusion can be found in Supplemental Table 3.

A meta-analysis of the pooled racemic data demonstrated a clear effect in all queried pharmacokinetic (PK) parameters. Comparing both extreme phenotypes, ultra-rapid metabolizers (UM) and poor metabolizers (PM), a 5.3-fold difference in peak dose-normalized plasma metoprolol concentration (C-_max_/dose; 90% CI 3.9 – 6.9-fold, p<0.001), a 13-fold difference in dose-normalized area under the plasma concentration vs time curve (AUC/dose; 90% CI 9.4 – 19.7-fold, p<0.001), a 2.6-fold difference in elimination half-life (t1/2; 90% CI 2.1 – 3.2-fold, p<0.001) and a 15-fold difference in apparent oral clearance (CL/F; 90% CI 14.0 – 16.6-fold, p<0.001) was observed (see [Table 1](#T1) and [Figure 2](#F2)). Between extensive metabolizers (EM, i.e., the majority of the population and the reference group) and poor metabolizers a 2.3-fold difference in C_max_/dose (90% CI 2.2 – 2.4-fold, p<0.001), 4.9-fold difference in AUC/dose (90% CI 4.7 – 5.0-fold, p<0.001), 2.3-fold difference in t1/2 (90%CI 2.3 – 2.3-fold, p<0.001) and a 5.9-fold difference CL/F (90% CI 5.6 – 6.1-fold, p<0.001) was found. Between EM and UM, a 2.3-fold difference in C_max_/dose (90% CI 1.9 – 2.9-fold, p=0.11), 2.7-fold difference in AUC/dose (90% CI 1.9 – 4.0-fold, p=0.17), 1.1-fold difference in t1/2 (90% CI 0.7 – 3.6-fold, p=0.94) and a 2.6-fold difference CL/F (90% CI 1.9 – 3.3-fold, p<0.001) was found.

### Table 1. Pooled analysis of metoprolol pharmacokinetics stratified by CYP2D6 phenotype.

|   | n | Cmax/dose(ng/mL/mg) | AUC/dose(μg*h/L/mg) | t 1/2 (h) | CL/F (L/h) |
| --- | --- | --- | --- | --- | --- |
| UM | 12 | 0.67 [0.55 – 0.79] | 2.73 [1.71 – 3.75] | 2.8 [0.5 – 5.1] | 367 [259 – 474] |
| EM | 122 | 1.56 [1.25 – 1.86] | 7.31 [5.96 – 8.66] | 3.1 [2.8 – 3.4] | 141 [127 – 157] |
| IM | 11 | 2.56 [0.88 – 4.25] | 18.46 [5.19 – 31.73] | 4.8 [4.5 – 5.2] | 95 [60 -130] |
| PM | 27 | 3.55 [2.60 – 4.50] | 35.53 [28.97 – 42.09] | 7.2 [7.0 – 7.5] | 24 [17 – 30] |

Table 1 Caption: Data presented as mean [95% confidence interval]. UM- ultra-rapid metabolizer, EM- extensive metabolizer, IM- intermediate metabolizer, PM- poor metabolizer, Cmax/dose- peak metoprolol concentration divided by the dose of metoprolol given, AUC/dose- area under the curve divided by the dose of metoprolol given, t½ - half-life, CL/F- oral clearance

### Figure 2. A-D. Effects of CYP2D6 metabolizer phenotype on selected pharmacokinetic parameters.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/cb25/3818912/e6c7bd18033d/nihms-524157-f0002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3818912_nihms-524157-f0002.jpg)

A Peak dose-normalized plasma metoprolol concentration (Cmax/dose); B Dose-normalized area under the plasma concentration (AUC/dose); C Elimination half-life (t1/2); D Apparent oral clearance (CL/F) The difference between ultra-rapid metabolizers and poor metabolizers in all four parameters is highly statistically significant (all p<0.001). Error bars represent standard deviations.

Overall, the observed heterogeneity between the studies, as indicated by the *I*^2^ statistics (ranges from 0% to 100%; the lower the number, the lower the degree of heterogeneity), was high and ranged between 80% and 95%, In addition to the gene-dose effect noted in the analysis of racemic metoprolol, a stereoselective metabolism of metoprolol that becomes more pronounced with an increasing number of functional alleles ([http://www.cypalleles.ki.se/cyp2d6.htm](http://www.cypalleles.ki.se/cyp2d6.htm)) was observed in the pooled enantiomer-specific analysis (see [Table 2](#T2)). As a result of genotype-dependent metabolism of the S-enantiomer, peak S-metoprolol concentrations and AUC are approximately 40% higher than those of R-metoprolol in the UM and EM groups (approximate S/R ratios: 1.5 and 1.5 for UM; 1.3 and 1.5 in EM, respectively) and are approximately 20% higher for the IM group (approximate S/R ratios: 1.1 and 1.3 in IM, respectively). The preference for the metabolism of R-metoprolol is also seen in the different fold-changes between the UM and PM phenotypes regarding different pharmacokinetic parameters (5.3 vs. 3.8-fold difference in peak metoprolol concentration, 14.5 vs. 9.5-fold difference in AUC and 30.4 vs. 22.4-fold difference in CL/F for R vs. S-enantiomer, respectively).

### Table 2. Pooled analysis of R- and S-metoprolol pharmacokinetics stratified by CYP2D6 phenotype.

|   | n | Cmax/dose (ng/mL/mg) |  | AUC/dose (ug*h/L/mg) |  | t 1/2 (h) |  | CL/F (L/h) |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
|   |  |  |  |  |  |  |  |  |  |
|   |   | R | S | R | S | R | S | R | S |
| UM | 12 | 0.32[0.24–0.40] | 0.47[0.33-0.61] | 1.27[0.86-1.68] | 1.90[1.34-2.46] | ND | ND | 547[393-701] | 380[272-488] |
| EM | 90 | 0.66[0.50-0.81] | 0.89[0.69-1.09] | 2.94[2.31-3.58] | 4.27[3.53-5.01] | 3.1[2.7-3.6] | 3.3[2.7-3.8] | 170[131-208] | 104[91-117] |
| IM | 12 | 1.09[0.89-1.29] | 1.24[1.04-1.44] | 7.61[5.54-9.68] | 9.59[7.31-11.87] | 4.6[3.9-5.3] | 5.2[4.5-5.9] | 61[46-75] | 48[36-60] |
| PM | 14 | 1.69[122-216] | 1.77[129-225] | 18.39[16.55-20.24] | 18.03[16.23-19.84] | 7.8[7.3-8.3] | 6.8[6.3-7.3] | 18[9-27] | 17[4-31] |

Table 2 Caption: Data presented as mean [95% confidence interval]. UM- ultra-rapid metabolizer, EM- extensive metabolizer, IM- intermediate metabolizer, PM- poor metabolizer, Cmax/dose- peak metoprolol concentration divided by the dose of metoprolol given, AUC/dose- area under the curve divided by the dose of metoprolol given, t½ - half-life, CL/F- oral clearance, ND- not done

Because the extensive metabolizer group has been broadly defined as having “at least one” active allele, an analysis was performed to determine if a semi-quantitative dose effect could be observed amongst different allele combinations.([13](#R13), [15](#R15)) After assigning a score of 1 for each fully active allele, 0.5 for alleles with decreased activity, and 0 for inactive alleles/deletions, a clear trend towards decreased metabolism with lower semi-quantitative dose was noted with regards to peak metoprolol concentration and AUC (see [Table 3](#T3)).

### Table 3. Extensive metabolizer semiquantitative gene dose sub-analysis of metoprolol pharmacokinetics stratified by CYP2D6 phenotype.

|   | n | Cmax/dose(ng/mL/mg) | AUC/dose(ug*h/L/mg) | t 1/2 (h) | CL/F (L/h) |
| --- | --- | --- | --- | --- | --- |
| Two fully active alleles (SQD=2) | 25 | 1.14 [0.94-1.33] | 5.39 [4.28-6.49] | 3.4 [2.7-4.2] | 168 [136-200] |
| One fully active allele, one semiactive allele (SQD=1.5) | 7 | 2.18 [1.65-2.71] | 9.96 [7.58-12.34] | 3.2 [2.2-4.2] | ND |
| One fully active allele, one inactive allele (SQD =1) | 1 | 3.32 | 34.17 | 8.2 | ND |

Table 3 Caption: Data presented as mean [95% confidence interval]. SQD- Semiquantitative gene dose, Cmax/dose- peak metoprolol concentration divided by the dose of metoprolol given, AUC/dose- area under the curve divided by the dose of metoprolol given, t½ - half-life, CL/F- oral clearance, ND- not done

Lastly, data regarding the influence of CYP2D6 metabolizer phenotype on the clinical effects of metoprolol were extracted. In these studies, exercise-induced heart rates of healthy volunteers were determined before and after administration of metoprolol. Twenty participants from one study ([17](#R17)) were excluded, as the data appeared duplicated from an earlier study.([16](#R16)) A greater change in heart rate was observed in PM, an effect that continued virtually unchanged for more than 12 hours while in EM the hemodynamic effects of metoprolol markedly decreased (see [Table 4](#T4)). This effect was also noted in the 12-hour area under the effect curve.

### Table 4. Effect of metabolizer phenotype on changes in exercise induced heart rate.

| Study | N | % chg HR(1.5 h.) | % chg HR(12 h.) | 12 h. Heart rate AUEC |
| --- | --- | --- | --- | --- |
| Hamelin | EM- 10 | 21% | 5% +/− 7% | ND |
|   | PM- 6 | 24% | 14% +/− 2% | ND |
| Hemeryck | EM- 8 | ND | ND | 203 +/− 75 (%*h) |
| Sharma, 2005 | EM- 16 | 24% | 8% | 275 +/− 63 (beats*h/min) |
|   | PM- 4 | 31% | 26% | 423 +/− 85 (beats*h/min) |

Table 4 Caption: Data presented as mean +/− standard deviation where applicable. EM- extensive metabolizer, PM- poor metabolizer, % chg HR- percent change in heart rate, AUEC- area under the effect curve, ND- not done

## DISCUSSION

This meta-analysis provides a more robust evidence of the effects of different CYP2D6 metabolizer phenotypes on metoprolol pharmacokinetics A clear gene-dose effect was observed regarding metoprolol pharmacokinetics, whereby metabolism of metoprolol was proportional to the number of active *CYP2D6* alleles present, as evident in the four PK parameters measured (peak metoprolol concentration [C_max_], area under the curve [AUC], half-life [t1/2] and oral clearance [CL/F]). In addition, CYP2D6 exhibits a preference towards metabolism of the R-enantiomer of metoprolol. The magnitude of this effect is dependent upon the degree of metabolism exerted by the enzyme: poor metabolizers show no preference while the difference between the enantiomers increases gradually through IM, EM and UM phenotypes. Lastly, this pooled systematic analysis shows that these pharmacokinetic differences influence clinical effects of metoprolol such as heart rate and blood pressure.

With varying degrees of statistical significance, each of the studies included in this meta-analysis demonstrated a gene-dose effect of CYP2D6 metabolizer phenotype on metoprolol pharmacokinetics. Consequently, the pooled analysis also demonstrates this effect, however, with greater statistical robustness given the substantially larger sample size of 264 participants while the largest individual study only consisted of 36 participants. In addition, no individual study presented pharmacokinetic data for all four metabolizer groups. Two studies presented oral clearance ([13](#R13)) and AUC([15](#R15)) as a semiquantitative gene dose to provide a better understanding of individual allele-specific effects. The pooled analysis, however, now allows for a direct comparison of four PK parameters stratified by four major metabolizer phenotypes, with an additional semiquantitative gene dose subanalysis of the extensive metabolizer group.

The results of this study have potential implications for clinical practice. First, additional evidence supporting the validity of *CYP2D6* genotyping was demonstrated by a clear correlation between genotype, metabolizer phenotype and metoprolol pharmacokinetics. Genotyping patients before initiating therapy with metoprolol would allow for the identification of PM and UM, thus potentially avoiding adverse events such as hypotension, syncope and bradycardia in the former or a lack of effect in the latter. Armed with this foreknowledge, adjustment of the initial metoprolol dose or the use of alternative drugs which are not major substrates for CYP2D6 may be considered. Alternative beta-blockers that do not depend on CYP2D6 metabolism are bisoprolol, a lipophilic drug that has been successfully tested in heart failure ([22](#R22)), or atenolol which is hydrophilic but has fewer positive endpoint studies. Carvedilol, a newer “third-generation” beta-blocker with unique vasodilating properties may also be a good alternative, although carvedilol undergoes some CYP2D6-dependent metabolism. Carvedilol has been shown in several studies (e.g. the COMET trial) to be superior to metoprolol in heart failure treatment. ([23](#R23), [24](#R24)) Given the significant effects the CYP2D6 phenotype has on metoprolol pharmacokinetics as well as the substantial prevalence of CYP2D6 ultra-rapid and poor metabolizers (combined prevalence probably >10% population), it may be possible that some of the observed differences in clinical trials comparing metoprolol to other beta-blockers may have been due to underlying CYP2D6 genotype and metabolizer phenotype. Furthermore, in the future it may be possible to target and adjust metoprolol dosing according to the CYP2D6 genotype/phenotype.

While this study observed a robust *CYP2D6* gene-dose effect on metoprolol pharmacokinetics, there are several limitations. First, the results of any meta-analysis are limited by the parameters of the studies that comprise the meta-analysis. Therefore, several of the limitations inherent to the original studies are carried forward. For example, key ethnic groups were not represented in this study as only white and Asian participants were reported. The lack of diversity in this analysis could limit the generalizability of the results. Second, additional alleles or confounding genetic variation in the form of yet uncovered haplotypes could influence the conclusions drawn from the data. Lastly, the meta-analysis showed a substantial degree of heterogeneity between the pooled studies, which limits the generalizability of the findings.

In summary, this systematic review and meta-analysis provided robust evidence for the importance of CYP2D6 metabolizer phenotype in influencing plasma metoprolol pharmacokinetics and also confirmed the enantiomer-specific metabolism of metoprolol, which is dependent upon the degree of metabolism present. These results could have further implications for pharmacogenetics-oriented personalized beta-blocker therapy.

## METHODS

### Data Sources and Study Selection Criteria

This study followed the PRISMA (Preferred Reporting Items for Systematic reviews and Meta-Analyses) guidelines ([25](#R25)) A comprehensive search of the PubMed database was conducted for articles published up to October 2012 using the terms *metoprolol* and *CYP2D6*. All articles published in English that evaluated the association between CYP2D6 metabolizer phenotype and metoprolol pharmacokinetics were screened. In addition, the bibliography of all articles retrieved was scanned for the inclusion of further articles. To be included in the systematic review, the articles had to fulfill the following inclusion criteria: ([1](#R1)) study participants must have received a single dose of racemic, immediate acting metoprolol and did not receive any other medications (prescribed or study drugs), ([2](#R2)) study participants must be genotyped for CYP2D6, ([3](#R3)) metoprolol pharmacokinetic data were stratified by metabolizer phenotype, and ([4](#R4)) at least one of the four pharmacokinetic parameters (peak metoprolol concentration, area under the curve, half-life and oral clearance) must have been present. A total of153 studies were screened, of which 13 met the inclusion criteria.

### Genotype/Phenotype Data and Data Compilation

For each study, participants were assigned to a distinct CYP2D6 metabolizer phenotype according to previously published criteria ([26](#R26)-[28](#R28)) using either phenotypic data (i.e. debrisoquine or dextromethorphan metabolizer status) and/or genotyping data for *CYP2D6*. Regarding genotyping for *CYP2D6*, subjects with a gene duplication resulting in more than two active *CYP2D6* alleles (i.e. defined as *1, *2, *33, *35) were classified as ultra-rapid metabolizers (UM), while those with at least one active allele were extensive metabolizers (EM). Subjects carrying two alleles of substrate-dependent decreased activity (i.e. *9, *10, *17, *29, *36, *41) or compound heterozygotes for one decreased activity allele in combination with a null allele (i.e. *3 to *8, *11 to *16, *19, *20, *21, *38, *40, *42, *44, *56 and *62) were termed as intermediate metabolizers (IM). A combination of two null alleles in a homozygous variant or compound heterozygous manner was classified as poor metabolizer (PM) phenotype. ([13](#R13), [29](#R29), [30](#R30)) The analysis focused on four common pharmacokinetic parameters: peak plasma metoprolol concentration (C_max_), area under the concentration vs time curve (AUC), elimination half-life (t1/2) and oral clearance (CL/F). Data were acquired from published tables and results, and converted to commonly reported units (C_max_(ng/mL), AUC (ng*h/mL), t1/2 (h), CL/F (L/h)) as needed. Because different doses of metoprolol were administered, C_max_ and AUC were normalized by dividing by the dose given (C_max_/dose and AUC/dose).

### Statistical Analysis

The statistical analysis was performed with Comprehensive Meta Analysis software, version 2.2.064 (Biostat, Englewood, NJ). Pooled pharmacokinetic parameters were calculated using a random-effects model due to the substantial heterogeneity between studies. The inverse variance method was used for weighing studies. Heterogeneity between studies was formally assessed by the *I2* statistics. Non-normality of the data distribution was considered and tested for, but found to be not serious enough to warrant special treatment. Comparison of the means of pooled pharmacokinetic parameters between metabolizer phenotype groups was done by one-way ANOVA (means and 95% CI). P-values are not adjusted for multiple comparisons. The calculation for the quotient between two means and its corresponding 90% confidence interval was done using Fieller’s method ([31](#R31)) which was incorporated in an online calculator ([http://www.graphpad.com/quickcalcs](http://www.graphpad.com/quickcalcs)). A p-value of <0.05 was considered statistically significant. GraphPad Prism 6.0.2 (GraphPad, La Jolla, CA) was used for additional statistical analyses.

## Supplementary Material

**What is the current knowledge on this topic?** The metabolizer phenotype of CYP2D6 has been shown to influence metoprolol pharmacokinetics in several small studies.

**What question did this study address?** This meta-analysis of eleven clinical trials that utilized plasma samples from healthy volunteers after a single oral dose of metoprolol addresses the impact of CYP2D6 metabolizer phenotype on metoprolol pharmacokinetics.

**What this study adds to our knowledge:** The pooled analysis (n= 264) demonstrated a 5.3-fold difference in peak plasma metoprolol concentration, a 13-fold difference in area under the concentration-time curve, a 2.6-fold difference in elimination half-life and a 15-fold difference in apparent oral clearance between ultra-rapid metabolizers and poor metabolizers. Enantiomer-specific analysis revealed genotype-dependent enantio-selective metabolism, with nearly 40% greater R- vs S-metoprolol metabolism in ultra-rapid and extensive metabolizers

How this might change clinical pharmacology and therapeutics: These results could have further implications in a pharmacogenetics-oriented, personalized beta-blocker therapy.

**What is the current knowledge on this topic?** The metabolizer phenotype of CYP2D6 has been shown to influence metoprolol pharmacokinetics in several small studies.

**What question did this study address?** This meta-analysis of eleven clinical trials that utilized plasma samples from healthy volunteers after a single oral dose of metoprolol addresses the impact of CYP2D6 metabolizer phenotype on metoprolol pharmacokinetics.

**What this study adds to our knowledge:** The pooled analysis (n= 264) demonstrated a 5.3-fold difference in peak plasma metoprolol concentration, a 13-fold difference in area under the concentration-time curve, a 2.6-fold difference in elimination half-life and a 15-fold difference in apparent oral clearance between ultra-rapid metabolizers and poor metabolizers. Enantiomer-specific analysis revealed genotype-dependent enantio-selective metabolism, with nearly 40% greater R- vs S-metoprolol metabolism in ultra-rapid and extensive metabolizers

How this might change clinical pharmacology and therapeutics: These results could have further implications in a pharmacogenetics-oriented, personalized beta-blocker therapy.

## ACKNOWLEDGEMENTS

MS is supported by the German Federal Ministry of Education and Research (Virtual Liver Network grant 2318 0315755 and grant 03 IS 2061C), the Deutsche Forschungsgemeinschaft 2319 (grant SCHW 858/1-1) and the Robert-Bosch Foundation, Stuttgart, Germany. These funding agencies had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.

The study was supported, in parts, by grants from the National Institutes of Health, Bethesda, MD (NIHK23 GM087534 to PN, K24DA00417 to EDK, and UL1RR024992 to Washington University Institute of Clinical and Translational Sciences), and the Division of Clinical and Translational Research, Department of Anesthesiology, Washington University.

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Lennard MS, Silas JH, Freestone S, Ramsay LE, Tucker GT, Woods HF. Oxidation phenotype--a major determinant of metoprolol metabolism and response. N Engl J Med. 1982;307:1558–60. doi: 10.1056/NEJM198212163072505.  [DOI](https://doi.org/10.1056/NEJM198212163072505) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7144837/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Oxidation%20phenotype--a%20major%20determinant%20of%20metoprolol%20metabolism%20and%20response&author=MS%20Lennard&author=JH%20Silas&author=S%20Freestone&author=LE%20Ramsay&author=GT%20Tucker&volume=307&publication_year=1982&pages=1558-60&pmid=7144837&doi=10.1056/NEJM198212163072505&)

2. Gardiner SJ, Begg EJ. Pharmacogenetics, drug-metabolizing enzymes, and clinical practice. Pharmacol Rev. 2006;58:521–90. doi: 10.1124/pr.58.3.6.  [DOI](https://doi.org/10.1124/pr.58.3.6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16968950/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacol%20Rev&title=Pharmacogenetics,%20drug-metabolizing%20enzymes,%20and%20clinical%20practice&author=SJ%20Gardiner&author=EJ%20Begg&volume=58&publication_year=2006&pages=521-90&pmid=16968950&doi=10.1124/pr.58.3.6&)

3. Nagele P, Liggett SB. Genetic variation, β-blockers, and perioperative myocardial infarction. Anesthesiology. 2011;115:1316–27. doi: 10.1097/ALN.0b013e3182315eb2.  [DOI](https://doi.org/10.1097/ALN.0b013e3182315eb2) | [PMC free article](/articles/PMC3226841/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21918425/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Anesthesiology&title=Genetic%20variation,%20%CE%B2-blockers,%20and%20perioperative%20myocardial%20infarction&author=P%20Nagele&author=SB%20Liggett&volume=115&publication_year=2011&pages=1316-27&pmid=21918425&doi=10.1097/ALN.0b013e3182315eb2&)

4. Daly AK, et al. Nomenclature for human CYP2D6 alleles. Pharmacogenetics. 1996;6:193–201. doi: 10.1097/00008571-199606000-00001.  [DOI](https://doi.org/10.1097/00008571-199606000-00001) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8807658/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=Nomenclature%20for%20human%20CYP2D6%20alleles&author=AK%20Daly&volume=6&publication_year=1996&pages=193-201&pmid=8807658&doi=10.1097/00008571-199606000-00001&)

5. McDonagh EM, Whirl-Carrillo M, Garten Y, Altman RB, Klein TE. From pharmacogenomic knowledge acquisition to clinical applications: the PharmGKB as a clinical pharmacogenomic biomarker resource. Biomark Med. 2011;5:795–806. doi: 10.2217/bmm.11.94.  [DOI](https://doi.org/10.2217/bmm.11.94) | [PMC free article](/articles/PMC3339046/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22103613/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biomark%20Med&title=From%20pharmacogenomic%20knowledge%20acquisition%20to%20clinical%20applications:%20the%20PharmGKB%20as%20a%20clinical%20pharmacogenomic%20biomarker%20resource&author=EM%20McDonagh&author=M%20Whirl-Carrillo&author=Y%20Garten&author=RB%20Altman&author=TE%20Klein&volume=5&publication_year=2011&pages=795-806&pmid=22103613&doi=10.2217/bmm.11.94&)

6. Zanger UM, Schwab M. Cytochrome P450 enzymes in drug metabolism: Regulation of gene expression, enzyme activities, and impact of genetic variation. Pharmacology & therapeutics. 2013 doi: 10.1016/j.pharmthera.2012.12.007.  [DOI](https://doi.org/10.1016/j.pharmthera.2012.12.007) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23333322/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacology%20&%20therapeutics&title=Cytochrome%20P450%20enzymes%20in%20drug%20metabolism:%20Regulation%20of%20gene%20expression,%20enzyme%20activities,%20and%20impact%20of%20genetic%20variation&author=UM%20Zanger&author=M%20Schwab&publication_year=2013&pmid=23333322&doi=10.1016/j.pharmthera.2012.12.007&)

7. Rau T, et al. Impact of the CYP2D6 genotype on the clinical effects of metoprolol: a prospective longitudinal study. Clin Pharmacol Ther. 2009;85:269–72. doi: 10.1038/clpt.2008.218.  [DOI](https://doi.org/10.1038/clpt.2008.218) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19037197/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Impact%20of%20the%20CYP2D6%20genotype%20on%20the%20clinical%20effects%20of%20metoprolol:%20a%20prospective%20longitudinal%20study&author=T%20Rau&volume=85&publication_year=2009&pages=269-72&pmid=19037197&doi=10.1038/clpt.2008.218&)

8. Nagele P, Liggett SB. Genetic variation, beta-blockers, and perioperative myocardial infarction. Anesthesiology. 2011;115:1316–27. doi: 10.1097/ALN.0b013e3182315eb2.  [DOI](https://doi.org/10.1097/ALN.0b013e3182315eb2) | [PMC free article](/articles/PMC3226841/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21918425/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Anesthesiology&title=Genetic%20variation,%20beta-blockers,%20and%20perioperative%20myocardial%20infarction&author=P%20Nagele&author=SB%20Liggett&volume=115&publication_year=2011&pages=1316-27&pmid=21918425&doi=10.1097/ALN.0b013e3182315eb2&)

9. Hamelin BA, et al. Significant interaction between the nonprescription antihistamine diphenhydramine and the CYP2D6 substrate metoprolol in healthy men with high or low CYP2D6 activity. Clin Pharmacol Ther. 2000;67:466–77. doi: 10.1067/mcp.2000.106464.  [DOI](https://doi.org/10.1067/mcp.2000.106464) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10824625/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Significant%20interaction%20between%20the%20nonprescription%20antihistamine%20diphenhydramine%20and%20the%20CYP2D6%20substrate%20metoprolol%20in%20healthy%20men%20with%20high%20or%20low%20CYP2D6%20activity&author=BA%20Hamelin&volume=67&publication_year=2000&pages=466-77&pmid=10824625&doi=10.1067/mcp.2000.106464&)

10. Hemeryck A, Lefebvre RA, De Vriendt C, Belpaire FM. Paroxetine affects metoprolol pharmacokinetics and pharmacodynamics in healthy volunteers. Clin Pharmacol Ther. 2000;67:283–91. doi: 10.1067/mcp.2000.104788.  [DOI](https://doi.org/10.1067/mcp.2000.104788) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10741632/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Paroxetine%20affects%20metoprolol%20pharmacokinetics%20and%20pharmacodynamics%20in%20healthy%20volunteers&author=A%20Hemeryck&author=RA%20Lefebvre&author=C%20De%20Vriendt&author=FM%20Belpaire&volume=67&publication_year=2000&pages=283-91&pmid=10741632&doi=10.1067/mcp.2000.104788&)

11. Huang J, Chuang SK, Cheng CL, Lai ML. Pharmacokinetics of metoprolol enantiomers in Chinese subjects of major CYP2D6 genotypes. Clin Pharmacol Ther. 1999;65:402–7. doi: 10.1016/S0009-9236(99)70134-7.  [DOI](https://doi.org/10.1016/S0009-9236(99)70134-7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10223777/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Pharmacokinetics%20of%20metoprolol%20enantiomers%20in%20Chinese%20subjects%20of%20major%20CYP2D6%20genotypes&author=J%20Huang&author=SK%20Chuang&author=CL%20Cheng&author=ML%20Lai&volume=65&publication_year=1999&pages=402-7&pmid=10223777&doi=10.1016/S0009-9236(99)70134-7&)

12. Jin SK, et al. Influence of CYP2D6*10 on the pharmacokinetics of metoprolol in healthy Korean volunteers. J Clin Pharm Ther. 2008;33:567–73. doi: 10.1111/j.1365-2710.2008.00945.x.  [DOI](https://doi.org/10.1111/j.1365-2710.2008.00945.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18834373/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Pharm%20Ther&title=Influence%20of%20CYP2D6*10%20on%20the%20pharmacokinetics%20of%20metoprolol%20in%20healthy%20Korean%20volunteers&author=SK%20Jin&volume=33&publication_year=2008&pages=567-73&pmid=18834373&doi=10.1111/j.1365-2710.2008.00945.x&)

13. Kirchheiner J, et al. Impact of the ultrarapid metabolizer genotype of cytochrome P450 2D6 on metoprolol pharmacokinetics and pharmacodynamics. Clin Pharmacol Ther. 2004;76:302–12. doi: 10.1016/j.clpt.2004.07.002.  [DOI](https://doi.org/10.1016/j.clpt.2004.07.002) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15470329/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Impact%20of%20the%20ultrarapid%20metabolizer%20genotype%20of%20cytochrome%20P450%202D6%20on%20metoprolol%20pharmacokinetics%20and%20pharmacodynamics&author=J%20Kirchheiner&volume=76&publication_year=2004&pages=302-12&pmid=15470329&doi=10.1016/j.clpt.2004.07.002&)

14. Parker RB, Soberman JE. Effects of paroxetine on the pharmacokinetics and pharmacodynamics of immediate-release and extended-release metoprolol. Pharmacotherapy. 2011;31:630–41. doi: 10.1592/phco.31.7.630.  [DOI](https://doi.org/10.1592/phco.31.7.630) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21923449/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacotherapy&title=Effects%20of%20paroxetine%20on%20the%20pharmacokinetics%20and%20pharmacodynamics%20of%20immediate-release%20and%20extended-release%20metoprolol&author=RB%20Parker&author=JE%20Soberman&volume=31&publication_year=2011&pages=630-41&pmid=21923449&doi=10.1592/phco.31.7.630&)

15. Seeringer A, Brockmoller J, Bauer S, Kirchheiner J. Enantiospecific pharmacokinetics of metoprolol in CYP2D6 ultra-rapid metabolizers and correlation with exercise-induced heart rate. Eur J Clin Pharmacol. 2008;64:883–8. doi: 10.1007/s00228-008-0504-8.  [DOI](https://doi.org/10.1007/s00228-008-0504-8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18545991/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Clin%20Pharmacol&title=Enantiospecific%20pharmacokinetics%20of%20metoprolol%20in%20CYP2D6%20ultra-rapid%20metabolizers%20and%20correlation%20with%20exercise-induced%20heart%20rate&author=A%20Seeringer&author=J%20Brockmoller&author=S%20Bauer&author=J%20Kirchheiner&volume=64&publication_year=2008&pages=883-8&pmid=18545991&doi=10.1007/s00228-008-0504-8&)

16. Sharma A, et al. Modulation of metoprolol pharmacokinetics and hemodynamics by diphenhydramine coadministration during exercise testing in healthy premenopausal women. J Pharmacol Exp Ther. 2005;313:1172–81. doi: 10.1124/jpet.104.081109.  [DOI](https://doi.org/10.1124/jpet.104.081109) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15718288/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharmacol%20Exp%20Ther&title=Modulation%20of%20metoprolol%20pharmacokinetics%20and%20hemodynamics%20by%20diphenhydramine%20coadministration%20during%20exercise%20testing%20in%20healthy%20premenopausal%20women&author=A%20Sharma&volume=313&publication_year=2005&pages=1172-81&pmid=15718288&doi=10.1124/jpet.104.081109&)

17. Sharma A, et al. Toward optimal treatment in women: the effect of sex on metoprolol-diphenhydramine interaction. J Clin Pharmacol. 2010;50:214–25. doi: 10.1177/0091270009340417.  [DOI](https://doi.org/10.1177/0091270009340417) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19948945/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Pharmacol&title=Toward%20optimal%20treatment%20in%20women:%20the%20effect%20of%20sex%20on%20metoprolol-diphenhydramine%20interaction&author=A%20Sharma&volume=50&publication_year=2010&pages=214-25&pmid=19948945&doi=10.1177/0091270009340417&)

18. Somer M, Kallio J, Pesonen U, Pyykkö K, Huupponen R, Scheinin M. Influence of hydroxychloroquine on the bioavailability of oral metoprolol. Br J Clin Pharmacol. 2000;49:549–54. doi: 10.1046/j.1365-2125.2000.00197.x.  [DOI](https://doi.org/10.1046/j.1365-2125.2000.00197.x) | [PMC free article](/articles/PMC2015042/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10848718/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Clin%20Pharmacol&title=Influence%20of%20hydroxychloroquine%20on%20the%20bioavailability%20of%20oral%20metoprolol&author=M%20Somer&author=J%20Kallio&author=U%20Pesonen&author=K%20Pyykk%C3%B6&author=R%20Huupponen&volume=49&publication_year=2000&pages=549-54&pmid=10848718&doi=10.1046/j.1365-2125.2000.00197.x&)

19. Wang Y, et al. Effects of imatinib (Glivec) on the pharmacokinetics of metoprolol, a CYP2D6 substrate, in Chinese patients with chronic myelogenous leukaemia. Br J Clin Pharmacol. 2008;65:885–92. doi: 10.1111/j.1365-2125.2008.03150.x.  [DOI](https://doi.org/10.1111/j.1365-2125.2008.03150.x) | [PMC free article](/articles/PMC2485214/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18384443/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Clin%20Pharmacol&title=Effects%20of%20imatinib%20(Glivec)%20on%20the%20pharmacokinetics%20of%20metoprolol,%20a%20CYP2D6%20substrate,%20in%20Chinese%20patients%20with%20chronic%20myelogenous%20leukaemia&author=Y%20Wang&volume=65&publication_year=2008&pages=885-92&pmid=18384443&doi=10.1111/j.1365-2125.2008.03150.x&)

20. Werner U, et al. Valdecoxib does not interfere with the CYP2D6 substrate metoprolol. Int J Clin Pharmacol Ther. 2006;44:397–400. doi: 10.5414/cpp44397.  [DOI](https://doi.org/10.5414/cpp44397) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16995327/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Clin%20Pharmacol%20Ther&title=Valdecoxib%20does%20not%20interfere%20with%20the%20CYP2D6%20substrate%20metoprolol&author=U%20Werner&volume=44&publication_year=2006&pages=397-400&pmid=16995327&doi=10.5414/cpp44397&)

21. Werner U, Werner D, Rau T, Fromm MF, Hinz B, Brune K. Celecoxib inhibits metabolism of cytochrome P450 2D6 substrate metoprolol in humans. Clin Pharmacol Ther. 2003;74:130–7. doi: 10.1016/S0009-9236(03)00120-6.  [DOI](https://doi.org/10.1016/S0009-9236(03)00120-6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12891223/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Celecoxib%20inhibits%20metabolism%20of%20cytochrome%20P450%202D6%20substrate%20metoprolol%20in%20humans&author=U%20Werner&author=D%20Werner&author=T%20Rau&author=MF%20Fromm&author=B%20Hinz&volume=74&publication_year=2003&pages=130-7&pmid=12891223&doi=10.1016/S0009-9236(03)00120-6&)

22. The Cardiac Insufficiency Bisoprolol Study II (CIBIS-II): a randomised trial. Lancet. 1999;353:9–13.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/10023943/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=The%20Cardiac%20Insufficiency%20Bisoprolol%20Study%20II%20(CIBIS-II):%20a%20randomised%20trial&volume=353&publication_year=1999&pages=9-13&pmid=10023943&)

23. DiNicolantonio JJ, Lavie CJ, Fares H, Menezes AR, O’Keefe JH. Meta-Analysis of Carvedilol Versus Beta 1 Selective Beta-Blockers (Atenolol, Bisoprolol, Metoprolol, and Nebivolol) The American Journal of Cardiology. 2013;111:765–9. doi: 10.1016/j.amjcard.2012.11.031.  [DOI](https://doi.org/10.1016/j.amjcard.2012.11.031) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23290925/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=The%20American%20Journal%20of%20Cardiology&title=Meta-Analysis%20of%20Carvedilol%20Versus%20Beta%201%20Selective%20Beta-Blockers%20(Atenolol,%20Bisoprolol,%20Metoprolol,%20and%20Nebivolol)&author=JJ%20DiNicolantonio&author=CJ%20Lavie&author=H%20Fares&author=AR%20Menezes&author=JH%20O%E2%80%99Keefe&volume=111&publication_year=2013&pages=765-9&pmid=23290925&doi=10.1016/j.amjcard.2012.11.031&)

24. Poole-Wilson PA, et al. Comparison of carvedilol and metoprolol on clinical outcomes in patients with chronic heart failure in the Carvedilol Or Metoprolol European Trial (COMET): randomised controlled trial. Lancet. 2003;362:7–13. doi: 10.1016/S0140-6736(03)13800-7.  [DOI](https://doi.org/10.1016/S0140-6736(03)13800-7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12853193/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Comparison%20of%20carvedilol%20and%20metoprolol%20on%20clinical%20outcomes%20in%20patients%20with%20chronic%20heart%20failure%20in%20the%20Carvedilol%20Or%20Metoprolol%20European%20Trial%20(COMET):%20randomised%20controlled%20trial&author=PA%20Poole-Wilson&volume=362&publication_year=2003&pages=7-13&pmid=12853193&doi=10.1016/S0140-6736(03)13800-7&)

25. Moher D, Liberati A, Tetzlaff J, Altman DG, Group t.P. Preferred Reporting Items for Systematic Reviews and Meta-Analyses: The PRISMA Statement. Annals of Internal Medicine. 2009;151:264–9. doi: 10.7326/0003-4819-151-4-200908180-00135.  [DOI](https://doi.org/10.7326/0003-4819-151-4-200908180-00135) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19622511/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Annals%20of%20Internal%20Medicine&title=Preferred%20Reporting%20Items%20for%20Systematic%20Reviews%20and%20Meta-Analyses:%20The%20PRISMA%20Statement&author=D%20Moher&author=A%20Liberati&author=J%20Tetzlaff&author=DG%20Altman&author=t.P.%20Group&volume=151&publication_year=2009&pages=264-9&pmid=19622511&doi=10.7326/0003-4819-151-4-200908180-00135&)

26. Sachse C, Brockmoller J, Bauer S, Roots I. Cytochrome P450 2D6 variants in a Caucasian population: allele frequencies and phenotypic consequences. American journal of human genetics. 1997;60:284–95.  [PMC free article](/articles/PMC1712396/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9012401/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=American%20journal%20of%20human%20genetics&title=Cytochrome%20P450%202D6%20variants%20in%20a%20Caucasian%20population:%20allele%20frequencies%20and%20phenotypic%20consequences&author=C%20Sachse&author=J%20Brockmoller&author=S%20Bauer&author=I%20Roots&volume=60&publication_year=1997&pages=284-95&pmid=9012401&)

27. Griese EU, et al. Assessment of the predictive power of genotypes for the in-vivo catalytic function of CYP2D6 in a German population. Pharmacogenetics. 1998;8:15–26. doi: 10.1097/00008571-199802000-00003.  [DOI](https://doi.org/10.1097/00008571-199802000-00003) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9511177/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=Assessment%20of%20the%20predictive%20power%20of%20genotypes%20for%20the%20in-vivo%20catalytic%20function%20of%20CYP2D6%20in%20a%20German%20population&author=EU%20Griese&volume=8&publication_year=1998&pages=15-26&pmid=9511177&doi=10.1097/00008571-199802000-00003&)

28. Zanger UM, et al. Comprehensive analysis of the genetic factors determining expression and function of hepatic CYP2D6. Pharmacogenetics. 2001;11:573–85. doi: 10.1097/00008571-200110000-00004.  [DOI](https://doi.org/10.1097/00008571-200110000-00004) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11668217/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=Comprehensive%20analysis%20of%20the%20genetic%20factors%20determining%20expression%20and%20function%20of%20hepatic%20CYP2D6&author=UM%20Zanger&volume=11&publication_year=2001&pages=573-85&pmid=11668217&doi=10.1097/00008571-200110000-00004&)

29. Zhou SF. Polymorphism of human cytochrome P450 2D6 and its clinical significance: Part I. Clin Pharmacokinet. 2009;48:689–723. doi: 10.2165/11318030-000000000-00000.  [DOI](https://doi.org/10.2165/11318030-000000000-00000) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19817501/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacokinet&title=Polymorphism%20of%20human%20cytochrome%20P450%202D6%20and%20its%20clinical%20significance:%20Part%20I&author=SF%20Zhou&volume=48&publication_year=2009&pages=689-723&pmid=19817501&doi=10.2165/11318030-000000000-00000&)

30. Swen JJ, et al. Pharmacogenetics: from bench to byte--an update of guidelines. Clin Pharmacol Ther. 2011;89:662–73. doi: 10.1038/clpt.2011.34.  [DOI](https://doi.org/10.1038/clpt.2011.34) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21412232/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Pharmacogenetics:%20from%20bench%20to%20byte--an%20update%20of%20guidelines&author=JJ%20Swen&volume=89&publication_year=2011&pages=662-73&pmid=21412232&doi=10.1038/clpt.2011.34&)

31. Fieller E. The biological standardization of Insulin. J R Statist Soc. 1940;7:S1–46.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20R%20Statist%20Soc&title=The%20biological%20standardization%20of%20Insulin&author=E%20Fieller&volume=7&publication_year=1940&pages=S1-46&)
