# The UDP-Glucuronosyltransferase (UGT) 1A Polymorphism c.2042C>G (rs8330) Is Associated with Increased Human Liver Acetaminophen Glucuronidation, Increased UGT1A Exon 5a/5b Splice Variant mRNA Ratio, and Decreased Risk of Unintentional Acetaminophen-Induced Acute Liver Failure

## Metadata
**Authors:** Michael H Court, Marina Freytsis, Xueding Wang, Inga Peter, Chantal Guillemette, Suwagmani Hazarika, Su X Duan, David J Greenblatt, William M Lee, the Acute Liver Failure Study Group
**Journal:** The Journal of Pharmacology and Experimental Therapeutics
**Date:** 2013 May
**DOI:** [10.1124/jpet.112.202010](https://doi.org/10.1124/jpet.112.202010)
**PMID:** 23408116
**PMCID:** PMC3629801
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3629801/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC3629801/pdf/jpet.112.202010.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC3629801/pdf/jpet.112.202010.pdf)

## Abstract

Acetaminophen is cleared primarily by hepatic glucuronidation. Polymorphisms in genes encoding the acetaminophen UDP-glucuronosyltransferase (UGT) enzymes could explain interindividual variability in acetaminophen glucuronidation and variable risk for liver injury after acetaminophen overdose. In this study, human liver bank samples were phenotyped for acetaminophen glucuronidation activity and genotyped for the major acetaminophen-glucuronidating enzymes (UGTs 1A1, 1A6, 1A9, and 2B15). Of these, only three linked single nucleotide polymorphisms (SNPs) located in the shared UGT1A-3′UTR region (rs10929303, rs1042640, rs8330) were associated with acetaminophen glucuronidation activity, with rs8330 consistently showing higher acetaminophen glucuronidation at all the tested concentrations of acetaminophen. Mechanistic studies using luciferase-UGT1A-3′UTR reporters indicated that these SNPs do not alter mRNA stability or translation efficiency. However, there was evidence for allelic imbalance and a gene-dose proportional increase in the amount of exon 5a versus exon 5b containing UGT1A mRNA spliced transcripts in livers with the rs8330 variant allele. Cotransfection studies demonstrated an inhibitory effect of exon 5b containing cDNAs on acetaminophen glucuronidation by UGT1A1 and UGT1A6 cDNAs containing exon 5a. In silico analysis predicted that rs8330 creates an exon splice enhancer site that could favor exon 5a (over exon 5b) utilization during splicing. Finally, the prevalence of rs8330 was significantly lower (P = 0.027, χ2 test) in patients who had acute liver failure from unintentional acetaminophen overdose compared with patients with acute liver failure from other causes or a race- or ethnicity-matched population. Together, these findings suggest that rs8330 is an important determinant of acetaminophen glucuronidation and could affect an individual’s risk for acetaminophen-induced liver injury.

## Introduction

Acetaminophen is one of the most widely used nonprescription antipyretic and analgesic drugs worldwide. Although considered very safe when used at recommended dosages, excessive dosing of acetaminophen can lead to serious liver injury. Studies by the Acute Liver Failure Study Group have identified acetaminophen as the leading single cause of acute liver failure (ALF) in the United States ([Larson et al., 2005](#B26)). About half of the cases result from intentional acute overdose, most likely in an attempt to elicit serious self-injury; the remaining cases appear to be unintentional resulting from chronic administration of multiple doses.

The mechanisms underlying acetaminophen-induced hepatotoxicity have been studied extensively ([Larson, 2007](#B25)). At therapeutic doses, acetaminophen is cleared primarily by conjugative metabolism (glucuronidation and sulfation). However, after higher (toxic) doses, conjugative pathways are overwhelmed and increased metabolism by the alternate cytochrome P450 oxidative pathway results in significant formation of the toxic electrophilic metabolite N-acetyl-*p*-benzoquinoneimine (NAPQI). NAPQI is normally rapidly detoxified by conjugation with glutathione, but excessive NAPQI formation exhausts available glutathione supplies. Acetaminophen overdose is effectively treated by administration of the glutathione precursor N-acetylcysteine. In support of this mechanism, various risk factors can influence susceptibility to acetaminophen hepatoxicity via effects on these metabolic pathways. For example, chronic alcohol use can enhance toxicity through increased CYP2E1–mediated NAPQI formation ([Larson, 2007](#B25)).

Although it has been speculated that genetic variability may predispose some individuals to an increased risk of acetaminophen-induced ALF ([Patel et al., 1992](#B32); [Court et al., 2001](#B10); [Rauchschwalbe et al., 2004](#B33); [Zhao and Pickering, 2011](#B39)), to date, only one published study has addressed this hypothesis. In that report, the association of polymorphisms in genes encoding three different glutathione S-transferase (GST) enzymes was investigated in 104 patients who had overdosed on acetaminophen ([Buchard et al., 2012](#B7)). Although none of the GST genotypes evaluated was associated with serum alanine aminotransferase levels, a sensitive biomarker of hepatocellular injury, a borderline significant (*P* = 0.05) association was found between copy number variation of the *GST-T1* gene and trough prothrombin time (an indicator of clinical outcome). However, the direction of the association (gene deletion resulting in a better predicted outcome) was the opposite of what would be expected if glutathione conjugation of toxic acetaminophen metabolites by the GST enzymes protected against liver injury. This finding was suggested by the authors to be the result of increased glutathione availability in individuals lacking the *GST-T1* gene since a previous mouse model study showed resistance to acetaminophen hepatotoxicity with higher glutathione levels in mice lacking the *GST-P1* gene ([Henderson et al., 2000](#B19)).

Hepatic glucuronidation is one of the principal mechanisms by which acetaminophen is detoxified and cleared from the human body. Studies in our laboratory and others have identified genetic polymorphisms that alter hepatic glucuronidation enzyme expression and function ([Girard et al., 2004](#B17), [2005](#B18); [Krishnaswamy et al., 2005b](#B24); [Court, 2010](#B9)). The primary aim of the current study was to use a well-established in vitro model of interindividual variability of drug glucuronidation to identify genetic polymorphisms associated with variable acetaminophen glucuronidation in human liver. We also established the most likely mechanism by which the identified polymorphisms influence drug glucuronidation and then determined the frequencies of these polymorphisms in patients who had developed ALF as a consequence of acetaminophen use. Our results indicate that a common single nucleotide polymorphism (SNP) in the UDP-glucuronosyltransferase 1A (UGT1A) gene (rs8330) is associated with increased hepatic acetaminophen glucuronidation, possibly through effects on UGT1A gene splicing. Furthermore, this putative protective gene variant appears to be present at a lower frequency in patients who had unintentionally developed acetaminophen-induced ALF compared with a population with a similar racial or ethnic background.

## Materials and Methods

### 

#### Reagents.

UDP-glucuronic acid (sodium salt), alamethicin, acetaminophen, and acetaminophen glucuronide were purchased from Sigma-Aldrich (St. Louis, MO). All reagents were of analytical or better grade. Oligonucleotide primers used for sequencing and real-time polymerase chain reaction (PCR) assay were synthesized by the Tufts Core Facility (Tufts University, Boston, MA).

#### Human Liver Tissues.

Liver samples from 48 donors with no known liver disease were obtained from either the National Disease Research Interchange (Philadelphia, PA) or the Liver Tissue Procurement and Distribution Service (Minneapolis, MN) with the approval of the Tufts University Institutional Review Board. All livers were either intended for transplantation but had failed to tissue match or were normal tissue adjacent to surgical biopsies. Donors were self-identified non-Hispanic whites and included 37 male subjects and 11 female subjects with a mean age of 43 years (range 2–75 years). Smoking history was positive for 16 donors, and significant alcohol use (defined as 14 or more drinks per week) was positive for 11 donors. Complete details of donor demographics, including available medication history for individual livers, have been reported elsewhere ([Court, 2010](#B9)).

#### Microsome Preparation.

Human liver microsomes were prepared by differential ultracentrifugation as previously described ([Court et al., 1997](#B12)). Microsomal pellets were suspended in 0.1 M potassium phosphate buffer (pH 7.5) containing 20% glycerol and kept at −80°C until use. The protein concentration of human liver microsome samples was determined using the bicinchoninic acid protein assay (Pierce, Rockford, IL) with bovine serum albumin as the standard. The quality of the liver samples was ascertained by reference to at least 10 other glucuronidation activities measured in this laboratory using the same set of livers. Livers that consistently showed low activity values (>2-fold lower for all measured activities) relative to the median activity value for the entire liver set were excluded from study.

#### Acetaminophen Glucuronidation Activities.

The rates of in vitro glucuronidation were determined by high-performance liquid chromatography for all liver microsome samples using the method we have described previously ([Court et al., 2001](#B10)) at three different acetaminophen concentrations (0.1, 2, and 40 mM) with a fixed UDP-glucuronic acid concentration (20 mM). Activities were measured in duplicate, and results were averaged.

#### DNA from Acute Liver Failure Patients.

DNA samples were obtained with appropriate consent from patients enrolled by the Acute Liver Failure Study Group, a consortium of U.S. liver centers established in 1998 to define more completely the causes and outcomes of ALF ([Ostapowicz et al., 2002](#B31)). Standard entry criteria for acute liver failure were used. All patients were considered to have had an acute hepatic injury of less than 26 weeks’ duration and demonstrated an international normalized ratio of ≥1.5 accompanied by any degree of hepatic encephalopathy as a result. Samples analyzed included those from 260 white patients meeting this definition, of which 79 patients were considered to have unintentional acetaminophen overdoses and 78 patients to have intentional overdoses; the remaining 103 patients had developed ALF from a variety of other causes. The definitions for these patient categories have previously been established ([Schiodt et al., 1997](#B35); [Larson et al., 2005](#B26); [Khandelwal et al., 2011](#B23)). In brief, acetaminophen toxicity patients all met clinical criteria, including alanine aminotransferase levels of ≥1000 IU/l, the presence of any level of the parent compound, acetaminophen in serum, and a history of >4 g per day of acetaminophen ingested, with two of three criteria qualifying for inclusion. The definition used to determine an unintentional ingestion is that acetaminophen was taken over days, with a specific cause of pain elicited and denial of suicidal intent ([Schiodt et al., 1997](#B35)). By contrast, patients who were considered to have taken a suicidal (intentional) overdose had taken acetaminophen at one time point, denied a cause for pain, and admitted to intent of self-harm. Causes for the other 103 patients varied greatly and included acute fatty liver of pregnancy (2 cases), autoimmune hepatitis (8 cases), Budd-Chiari syndrome (2 cases), drug-induced liver injury (23 cases), hepatitis A (2 cases), hepatitis B (11 cases), hepatitis E (1 case), indeterminate (27 cases), other (11 cases), shock or ischemia (15 cases), and Wilson’s disease (1 case).

#### Genotyping.

The UGT1A-3′UTR SNPs (rs10929303, c.1813C>T; rs1042640, c.1941C>G; rs8330, c.2042C>G) were genotyped by resequencing DNA obtained from the human liver bank samples (*n* = 48) and patients with ALF (*n* = 260). Briefly, PCR amplification was performed with forward primer Pri-639 (5′- GCA TAA ATT AAT CAG CCC CAG AGT GC -3′) and reverse primer Pri-640 (5′- CAC CAC CCA CCA ATT TCA TAG CAT C -3′) using Platinum taq Hifi supermix (Invitrogen, Carlsbad, CA), and the resultant product was sequenced with Pri-567 (5′- CCA TTC ATT CAT TTC ACC TAC ACT -3′). The entire UGT1A exon 3 to 5a gene region was also resequenced in selected liver samples using an additional 10 sets of primers given in [Table 1](#T1). Genotypes (including methods) for the other polymorphisms in the liver bank samples were reported previously, including UGT1A1*28 (rs34815109, -53TA6 > 7) ([Girard et al., 2005](#B18)), UGT1A6*2 allele (rs6759892, S7A; rs2070959, T181A; and rs1105879, R184S) ([Krishnaswamy et al., 2005b](#B24)), UGT1A9 -275T>A (rs6714486) and UGT1A9*22 (rs45625337, T9 > T10) ([Girard et al., 2004](#B17)), and UGT2B15*2 (rs1902023, D85Y) ([Court et al., 2004](#B11)). Linkage disequilibrium and haplotype block analysis was performed using these SNP data and the Haploview program ([Barrett et al., 2005](#B2)).

| Primer ID | Sequence (5′ to 3′) | Direction | Locationa |
| --- | --- | --- | --- |
| Pri 639 | GCATAAATTAATCAGCCCCAGAGTGC | Forward | Segment 1 |
| Pri 640 | CACCACCCACCAATTTCATAGCATC | Reverse | Segment 1 |
| Pri 818 | CTTTCCTTGCTTCCTTCCCTCCTTCT | Forward | Segment 2 |
| Pri 819 | AAACTCCACCCAGAACACGGCCAG | Reverse | Segment 2 |
| Pri 820 | TAGCACCAAGGGTTGAAGCACCTAAC | Forward | Segment 3 |
| Pri 821 | CTTGAAAAAGAAGGAAGAGAAGGAGGGA | Reverse | Segment 3 |
| Pri 822 | ACAGGAAGCTTAGTGCTGACATCACTTG | Forward | Segment 4 |
| Pri 823 | GCTTCAACCCTTGGTGCTACAGGAAT | Reverse | Segment 4 |
| Pri 824 | AAAAAAATTACCCAGGCATGGTGGTGT | Forward | Segment 5 |
| Pri 825 | CTTTTGCCATGTGACATGCAAGCT | Reverse | Segment 5 |
| Pri 826 | TTGAAGCTCCTTCTTGAGGCTCACA | Forward | Segment 6 |
| Pri 827 | GCATACACCACCATGCCTGGGTAA | Reverse | Segment 6 |
| Pri 828 | GACACACCAGCTTGAGCAAGGGAC | Forward | Segment 7 |
| Pri 829 | GGACTTTCACCAAGTTGTTCTTAGGTTACA | Reverse | Segment 7 |
| Pri 830 | GCTTTTTGCAGCACTGGAACCTGT | Forward | Segment 8 |
| Pri 831 | CCTGTTGTCCCTTGCTCAAGCTGG | Reverse | Segment 8 |
| Pri 848 | ACCTCCCACGTTCAAGCAGTTCT | Forward | Segment 9 |
| Pri 849 | TGGAGTATTTCTTTCTGTGCAGG | Reverse | Segment 9 |
| Pri 834 | GCTGACATCCTCCCTATTTTGCATCTC | Forward | Segment 10 |
| Pri 835 | AGGCAGAACTGCTTGAACGTGGGA | Reverse | Segment 10 |
| Pri 836 | CCTCCAAAACAAGATGCCGGAAGT | Forward | Segment 11 |
| Pri 837 | GGGTGACCTGAGATGCAAAATAGGG | Reverse | Segment 11 |

Table Caption: Polymerase chain reaction primers used to amplify and sequence the UGT1A gene exons 3–5 region in human liver DNA samples

#### UGT1A-3′UTR Luciferase Reporter Assays.

Plasmid luciferase 3′UTR reporter constructs containing the entire UGT1A-3′UTR reference and major variant haplotypes (CCC and TGG for rs10929303, rs1042640, and rs8330, respectively) were created using the methods as previously described ([Oleson et al., 2010](#B29)). Human embryonic kidney 293T (HEK293T) and human colon adenoma LS180 cells were from American Type Culture Collection (Manassas, VA). Hepatocarcinoma Huh7 cells were a gift from Dr. Curt Omiecinski, Pennsylvania State University. Primary human hepatocytes in collagen coated 96-well clear-bottom, white-sided plates (Corning, Tewksbury, MA) were provided by the Liver Tissue Cell Distribution System (Stephen Strom, Department of Pathology, University of Pittsburgh, Pittsburgh, PA). Cells were from three different donors, including a 28-year-old woman, a 53-year-old woman, and a 65-year-old man. HEK293T and Huh7 cells were maintained in Dulbecco’s modified Eagle’s medium with high glucose and l-glutamine (Gibco/Invitrogen) supplemented with 10% fetal bovine serum (HyClone Laboratories, Logan, UT), and penicillin/streptomycin (Gibco/Invitrogen). The LS180 cells were maintained in minimum essential medium (Gibco/Invitrogen) supplemented with 10% fetal bovine serum, penicillin or streptomycin, nonessential amino acids, and sodium pyruvate (Gibco/Invitrogen).

For cell lines, approximately 50,000 cells per well were seeded onto 96-well clear bottom, white-sided plates (Corning) the day before transfection. Cells were cotransfected with 50 ng (5 ng for HEK293 cells) of the UGT1A-3′UTR plasmids and 20 ng (5 ng for HEK293 cells) of the renilla luciferase pRL-CMV transfection control plasmid (Promega, Madison, WI) with Lipofectamine 2000 reagent (Invitrogen). After a further 24 hours, cells were assayed for luciferase and renilla activities using the Dual-Glo assay kit (Promega). Each assay was conducted using four wells averaged per data point, and the final results were derived from at least three independent experiments.

#### Allelic Imbalance Assay.

The relative amounts of allele specific mRNA were determined in 12 of the human liver samples that were heterozygous for all three of the UGT1A-3′UTR SNPs by the quantitative Sanger sequencing method as described previously ([Ge et al., 2005](#B16)). Briefly, PCR was conducted using the same method used for genotyping the UGT1A-3′UTR SNPs described previously herein using primers Pri-639 and Pri-640. The PCR template included both cDNA (after reverse transcription of total RNA with Qiagen Omniscript; Qiagen, Germantown, MD) and gDNA obtained from the same livers. PCR products were sequenced with Pri-567.

#### UGT1A Exon 5a and 5b mRNA Quantitation.

UGT exon 5a and 5b splice variant mRNA content relative to *β*-actin mRNA content was quantified by real-time PCR in 40 of the 48 total human livers that had available good-quality extracted total RNA. Sybr Green based real-time PCR on reverse transcribed liver total RNA was conducted as previously described, including the same beta-actin primers ([Court et al., 2012](#B13)). Primers for both exon 5a and 5b spanned the exon 4-5 junctions and included exon 5a forward primer Pri-761 (5′-GGA GCT GGA GTG ACC CTG AAT GTT C-3′) exon 5a reverse primer Pri-479 (5′-CCC TTG TGC CTC ATC ACA AAC TCC-3′), exon 5b forward primer Pri-761 and exon 5b reverse primer Pri-762 (5′-TTG CTC AAG CTG GTG TGT CCC C-3′).

#### Coexpression of UGT1A Isoforms 1 and 2 Variants in Cell Lines.

Plasmids encoding UGT1A6 isoform 1 (UGT1A6_i1 containing exon 5a) and isoform 2 (UGT1A6_i2 containing splice variant exon 5b), constructed as previously described ([Benoit-Biancamano et al., 2009](#B5)), were cotransfected using Lipfectamine 2000 into six-well plates containing HEK293T cells at approximately 80% confluency. A *β*-gal–expressing plasmid (20 ng) was also included to control for transfection efficiency differences and nonspecific effects of plasmid overexpression as described previously ([Oleson et al., 2010](#B29); [Volak and Court, 2010](#B38)). Three milliliters of growth media was added after 4–6 hours. Cells were harvested 48 hours after transfection, washed with phosphate-buffered saline, disrupted by gentle sonication, and the lysates assayed for acetaminophen glucuronidation activity using 20 mM acetaminophen, 5 mM UDP-glucuronic acid, and 3 hours of incubation time. Data were normalized to *β*-gal activity measured using the same lysates. Microsomes for HEK293 stable cell lines expressing either UGT1A1_i1 alone or coexpressing UGT1A1_i1 and UGT1A1_i2 were obtained as previously described ([Bellemare et al., 2010a](#B3)). By immunoblot, the ratio of UGT1A1_i1 to UGT1A1_i2 was 1:0.33. Acetaminophen glucuronidation activities were measured by high-performance liquid chromatography as described already herein, and the final data were normalized to the UGT1A1_i1 content of each preparation as determined by immunoblotting.

#### Splicing Regulatory Site Consensus Sequence Analysis.

The effect of each of the UGT1A 3′UTR SNPs on binding site consensus sequences for the Srp family of splicing regulatory proteins was evaluated using the ESEfinder program (release 3.0; Cold Spring Harbor Laboratory, Cold Spring Harbor, NY) ([Cartegni et al., 2003](#B8)). The default values of 1.956, 2.383, 2.670, and 2.676 were used as the threshold scores for the SF2/ASF, SC35, SRp40, and SRp55 consensus binding sites, respectively.

#### Statistical Analyses.

Unless otherwise stated, statistical analyses were performed using Sigmaplot 11 software (Systat, San Jose, CA). Associations between genotype and liver bank acetaminophen glucuronidation activities, and between genotype and exon 5a/5b splicing ratios, were evaluated by analysis of variance (ANOVA) on log-transformed data with post-hoc pairwise testing by the Student-Newman-Keuls multiple comparisons test. Differences in allele-specific mRNA content in heterozygous livers were evaluated by paired *t* test. The effects of the UGT1A6_i2 form on UGT1A6_i1 activities were evaluated by repeated-measures ANOVA with multiple comparisons with the control group (no UGT1A6_i2) using the Holm-Sidak test. The effect of UGT1A1_i2 coexpression on UGT1A1_i1 acetaminophen glucuronidation activity was evaluated by *t* test. Differences in genotype frequencies of novel SNPs identified in the UGT1A gene exons 3–5a region between livers with low and high exon 5a/exon5b splicing ratio were evaluated by Fisher’s exact test. Finally, differences in UGT1A-3′UTR genotype frequencies (reference and variant carrier genotypes) between acute liver failure patients grouped by cause were evaluated by *χ*^2^ test with one degree of freedom ([http://ihg.gsf.de/cgi-bin/hw/hwa1.pl](http://ihg.gsf.de/cgi-bin/hw/hwa1.pl)). A *P* value of less than 0.05 was considered significant.

## Results

### 

#### UGT1A-3′UTR SNPs Are Associated with Higher Acetaminophen Glucuronidation Activities.

SNP genotyping was conducted using DNA from our human liver bank samples focusing on genes encoding important human liver acetaminophen glucuronidation enzymes, including UGT1A1, 1A6, and 1A9 ([Bock et al., 1993](#B6); [Court et al., 2001](#B10)). UGT2B15 variants were also evaluated since one study showed significant acetaminophen glucuronidation by recombinant enzyme ([Mutlib et al., 2006](#B27)). Genotypes were then correlated with glucuronidation activities measured in the same liver bank samples at acetaminophen concentrations representing plasma concentrations typically associated with therapeutic use (100 *µ*M), overdose (40 mM), and an intermediate concentration (2 mM) approximating the *K*_m_ value of UGT1A6, the high-affinity acetaminophen UGT ([Court et al., 2001](#B10)). As shown in [Table 2](#T2), only the UGT1A gene 3′UTR SNPs (rs10929303, rs1042640, and rs8330) were associated with acetaminophen glucuronidation activities. Interestingly, there appeared to be an acetaminophen concentration effect on the SNP association in that rs8330 was the only SNP associated with acetaminophen glucuronidation at the lowest concentration (0.1 mM acetaminophen), whereas all three SNPs were significantly associated with glucuronidation at a concentration of 2 and 40 mM. As shown in [Fig. 1](#F1), rs8330 was associated with higher acetaminophen glucuronidation activities (measured at 40 mM substrate concentration), depending on the number of variant alleles. Specifically, rs8330 cg heterozygotes had mean acetaminophen glucuronidation activities that were significantly higher (*P* = 0.045) than the reference rs8330 cc homozygotes and were also significantly lower (*P* = 0.026) than the variant rs8330 gg homozygotes.

|   |   |   | Acetaminophen Glucuronidation (pmol/min/mg Protein) |  |  |  |  |  |  |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
|   |   |   | 0.1 mM |  |  | 2 mM |  |  | 40 mM |  |  |
| Genotype |  | n | Mean | S.D. | P Value | Mean | S.D. | P Value | Mean | S.D. | P Value |
| UGT1A9 -275T>A | T/T | 37 | 16.0 | 11.7 | 0.57 | 935 | 558 | 0.89 | 3272 | 1541 | 0.97 |
| (rs6714486) | T/A | 5 | 19.7 | 17.1 |   | 885 | 420 |   | 3299 | 2026 |   |
|   | A/A | 0 |   |   |   |   |   |   |   |   |   |
| UGT1A9*22 | *1/*1 | 16 | 12.3 | 6.7 | 0.10 | 782 | 401 | 0.056 | 2915 | 1299 | 0.053 |
| (rs45625337) | *1/*22 | 20 | 18.4 | 16.0 |   | 901 | 591 |   | 3142 | 1745 |   |
|   | *22/*22 | 6 | 20.6 | 6.2 |   | 1414 | 470 |   | 4679 | 944 |   |
| UGT1A6*2 | *1/*1 | 21 | 13 | 7.6 | 0.30 | 884 | 550 | 0.69 | 3157 | 1591 | 0.69 |
|   | *1/*2 | 14 | 19 | 17 |   | 762 | 595 |   | 2801 | 1572 |   |
|   | *2/*2 | 7 | 9.9 | 3.5 |   | 771 | 415 |   | 2807 | 1182 |   |
| UGT1A1*28 | *1/*1 | 6 | 10.4 | 3.6 | 0.18 | 774 | 455 | 0.8 | 2839 | 1292 | 0.77 |
| (rs34815109) | *1/*28 | 14 | 20.7 | 15.9 |   | 942 | 610 |   | 3310 | 1521 |   |
|   | *28/*28 | 22 | 15.3 | 10.5 |   | 963 | 529 |   | 3372 | 1720 |   |
| UGT1A 1813C>T | C/C | 34 | 13.5 | 11.5 | 0.07 | 722 | 481 | #0.01 | 2704 | 1271 | #0.001 |
| (rs10929303) | C/T | 11 | 19.9 | 13.3 |   | 1113 | 502 |   | 3787 | 1652 |   |
|   | T/T | 3 | 22.1 | 5.9 |   | 1551 | 540 |   | 5621 | 1197 |   |
| UGT1A 1941C>G | C/C | 33 | 13.7 | 11.7 | 0.17 | 733 | 484 | #0.046 | 2710 | 1291 | #0.01 |
| (rs1042640) | C/G | 13 | 19.5 | 13.0 |   | 1118 | 563 |   | 3919 | 1772 |   |
|   | G/G | 2 | 19.1 | 3.6 |   | 1353 | 590 |   | 5041 | 921 |   |
| UGT1A 2042C>G | C/C | 31 | 12.9 | 11.6 | #0.035 | 718 | 490 | #0.016 | 2681 | 1294 | #0.002 |
| (rs8330) | C/G | 14 | 19.8 | 12.4 |   | 1037 | 499 |   | 3602 | 1569 |   |
|   | G/G | 3 | 22.1 | 5.9 |   | 1551 | 540 |   | 5621 | 1197 |   |
| UGT2B15*2 | *1/*1 | 8 | 22.6 | 19.8 | 0.30 | 1284 | 778 | 0.32 | 4067 | 1857 | 0.14 |
| (rs1902023) | *1/*2 | 27 | 13.3 | 7.8 |   | 767 | 462 |   | 2830 | 1500 |   |
|   | *2/*2 | 13 | 15.6 | 12.1 |   | 804 | 411 |   | 3202 | 1336 |   |

Table Caption: Association of UGT polymorphisms with microsomal acetaminophen glucuronidation activity measured at three different substrate concentrations in 42 human liver bank samples

![Fig. 1.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c5d3/3629801/74362794ce35/jpet.112.202010f1.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3629801_jpet.112.202010f1.jpg)

Association of UGT1A-3′UTR rs8330 SNP genotype (CC, CG, and GG) with microsomal acetaminophen glucuronidation activities (40 mM acetaminophen concentration) measured in a bank of human livers obtained from 48 white donors. Shown are activities for each liver (filled circle), geometric mean values for each genotype group (horizontal dashed line), and also P values for post-hoc pairwise testing on log transformed data by the Student-Newman-Keuls multiple comparisons test (P = 0.002, ANOVA). *P < 0.05; **P < 0.01.

Genetic linkage analysis of the evaluated SNPs revealed the presence of two distinct haplotype blocks incorporating the UGT1A6 and UGT1A1 exons 1 SNPs (block 1) and the UGT1A-3′UTR SNPs (block 2) ([Fig. 2](#F2)). However, analysis of genotype associations with glucuronidation activity was not enhanced by the use of UGT1A6/UGT1A1 or UGT1A-3′UTR haplotypes compared with analysis by individual SNPs ([see Supplemental Table 1](http://jpet.aspetjournals.org/lookup/suppl/doi:10.1124/jpet.112.202010/-/DC1)).

![Fig. 2.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c5d3/3629801/cad7761eba42/jpet.112.202010f2.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3629801_jpet.112.202010f2.jpg)

Results of pairwise linkage disequilibrium and haplotype block analysis of 9 UGT1A gene SNPs and one UGT2B15 gene SNP genotyped using DNA extracted from human liver bank samples obtained from 48 white donors. Shown above are the approximate locations of each of the nine SNPs (identified by their dbSNP rs number) relative to the unique exons 1 and the shared exons 2 to 5 in the UGT1A gene. Shown below is a matrix of linkage disequilibrium r-squared values (as percent) for each pairwise comparison. Two distinct haplotype blocks were identified, including one block incorporating the UGT1A6 and UGT1A1 SNPs (block 1) and another block incorporating the UGT1A-3′UTR SNPs (block 2).

#### UGT1A-3′UTR SNPs Do Not Affect Luciferase-3′UTR Reporter Activity.

SNPs located in the 3′UTR region could affect mRNA degradation rate or translational efficiency such as through altering (creating or eliminating) response elements for microRNAs. Consequently, luciferase-3′UTR reporter constructs were generated containing each of the common UGT1A-3′UTR haplotypes (CCC-UGT1A-reference; TGG-UGT1A-variant). These were transfected into cell lines derived from three different human tissues (kidney, liver, and intestines) and also primary human hepatocytes. As shown in [Fig. 3](#F3), the variant UGT1A-3′UTR did not differ in reporter activity compared with the reference UGT1A-3′UTR in all cell types examined. Interestingly, compared with constructs that lacked any of the UGT1A-3′UTRs, constructs containing the UGT1A-3′UTR (regardless of haplotype) showed 60–70% lower reporter activity in LS-180 intestinal cells (*P* < 0.003), whereas reporter activity was 2- to 3-fold higher with the UGT1A-3′UTR in primary human hepatocytes (*P* < 0.003).

![Fig. 3.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c5d3/3629801/22aa0196f8cf/jpet.112.202010f3.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3629801_jpet.112.202010f3.jpg)

Effect of the UGT1A-3′UTR reference (CCC) and variant (TGG) haplotype sequences on luciferase-UGT1A-3′UTR reporter activities when transfected into human kidney (A), liver (B), intestinal (C) cell lines, and primary human hepatocytes (D). Bars indicate the mean and S.D. of results of either four independent assays (cell lines) or three different donors (hepatocytes). Luciferase data were normalized to a renilla transfection control and expressed as a ratio relative to wells transfected with a plasmid construct lacking the UGT1A-3′UTR (reference or variant). Also shown for each set of data are the results of ANOVA with post-hoc pairwise testing by the Student-Newman-Keuls multiple comparisons test. n.s., not significant.

#### UGT1A-3′UTR SNPs May Cause Allelic Expression Imbalance.

Since the UGT1A-3′UTR SNPs are present in the mature mRNA transcript, allele-specific assay of mRNA levels can be used to determine whether there is allelic expression imbalance that would verify the presence of a *cis*-acting element (either the SNPs themselves or another linked variant) that would explain increased UGT1A-mediated glucuronidation. Variant to reference allele mRNA ratios were determined for 12 of the liver samples that were heterozygous for each of the three UGT1A-3′UTR SNPs (i.e., the CCC/TGG diplotype). The results showed a trend for higher variant allele mRNA levels compared with the reference allele with mean (S.D.) allele-specific ratios of 1.09 (0.18), 1.13 (0.14), and 1.14 (0.23) for rs10929303, rs1042640, and rs8330, respectively (individual data are shown in [Supplemental Fig. 1](http://jpet.aspetjournals.org/lookup/suppl/doi:10.1124/jpet.112.202010/-/DC1)). Higher variant mRNA levels were statistically significant only for rs1042640 (*P* = 0.007, paired *t* test), although rs8330 approached statistical significance (*P* = 0.075).

#### UGT1A-3′UTR SNPs Are Associated with Increased Exon 5a to Exon 5b mRNA Variant Ratios.

Splicing of the primary UGT1A transcript can result in mRNA variants that incorporate an alternate exon 5 (termed exon 5b) that encodes for a truncated protein (termed isoform 2 variant) that lacks the normal C-terminal *trans*-membrane domain (see [Fig. 6B](#F6)). Although these proteins are enzymatically inactive, they appear to act as repressors of enzyme activity, and it is possible that the UGT1A-3′UTR SNPs that are located in exon 5a might influence this splicing. Consequently, we measured the amount of UGT1A transcripts containing the normally spliced exon 4–5a relative to the amount of exon 4–5b alternatively spliced transcripts and determined whether this ratio was correlated with the UGT1A-3′UTR SNPs. Exon 5a/5b mRNA ratios varied greatly, ranging from as low as 0.020 (about 50-fold more exon 5b versus exon 5a) to as high as 28.2 (about 28-fold more exon 5b than exon 5a). Importantly, there was a clear association between rs8330 and exon 5a/5b mRNA ratio (*P* = 0.008, ANOVA) with about 7-fold higher mean 5a/5b ratios in rs8330 cg heterozygotes (*P* = 0.013, Student-Newman-Keuls test) and 11-fold higher in rs8330 gg homozygotes (*P* = 0.017) compared with rs8330 cc homozygotes. Similar associations were also observed for the rs10929303 and rs1042640 SNPs ([Table 3](#T3)).

![Fig. 6.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c5d3/3629801/b016363262d9/jpet.112.202010f6.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3629801_jpet.112.202010f6.jpg)

Putative effect of the UGT1A-3′UTR SNPs on gene splicing. The top panel (A) shows the results of binding site consensus sequence analysis for the Srp family of splicing regulatory proteins using the ESEfinder program (release 3.0). Shown are the sequences surrounding the reference (above) and variant (below) sequences surrounding each SNP and the predicted consensus binding sites and scores in parentheses. No ESE elements were predicted for the rs8330 reference sequence. The bottom panel (B) is a proposed model explaining the effect of the UGT1A3′-UTR SNPs on glucuronidation by UGT1A isoforms through effects on gene splicing. The presence of the UGT1A-3′UTR SNPs would favor utilization of exon 5a over the exon 5b splicing donor site, resulting in more UGT1A_i1 relative to UGT1A_i2 protein variants and less post-translation repression of UGT1A_i1 enzyme activities.

|   |   |   | Exon 5a/Exon 5b mRNA Ratio |  |  |
| --- | --- | --- | --- | --- | --- |
| UGT1A Genotype |  | n | Mean | S.D. | P Value* |
| rs10929303 | C/C | 27 | 1.5 | 3.1 | *0.015 |
|   | C/T | 10 | 7.7 | 9.4 |   |
|   | T/T | 3 | 11.0 | 10.8 |   |
| rs1042640 | C/C | 26 | 1.5 | 3.2 | *0.012 |
|   | C/G | 12 | 6.5 | 8.9 |   |
|   | G/G | 2 | 15.9 | 9.5 |   |
| rs8330 | C/C | 24 | 1.0 | 1.6 | *0.008 |
|   | C/G | 13 | 7.1 | 8.9 |   |
|   | G/G | 3 | 11.0 | 10.8 |   |

Table Caption: Association of UGT1A-3′UTR single nucleotide polymorphism genotype with ratios of exon 5a mRNA to exon 5b mRNA splice variant content measured in 40 human liver bank samples

#### UGT1A Isoform 2 Variants Are Repressors of Acetaminophen Glucuronidation.

Although UGT1A isoform 2 (UGT1A_i2) variants expressed in cell lines have been shown to decrease the glucuronidation activity of the corresponding isoform 1 (UGT1A_i1) enzymes for a number of different model substrates, effects on acetaminophen glucuronidation have not been reported. Consequently, we proceeded to evaluate the effects of UGT1A6_i2 (0–2 *µ*g plasmid) on acetaminophen glucuronidation by UGT1A6_i1 (2 *µ*g plasmid) transiently coexpressed in HEK293 cells. As shown in [Fig. 4A](#F4), there was a profound dose-dependent effect of UGT1A6_i2 on acetaminophen glucuronidation, with nearly 90% inhibition observed with 1.5 *µ*g or greater amounts of cotransfected plasmid (*P* < 0.05, repeated measures ANOVA with Holm-Sidak post-hoc testing versus control). Similar experiments were conducted to determine whether the UGT1A1_i2 variant also inhibited UGT1A1_i1-mediated acetaminophen glucuronidation using HEK293 cell lines that stably expressed either UGT1A1_i1 alone or both UGT1A1_i1 and UGT1A1_i2 (immunoblot ratio of 3:1). Stable expression tends to result in lower expressed protein levels that are likely closer to levels observed in human hepatocytes. UGT1A1_i2 also resulted in significant inhibition of UGT1A1_i1-mediated glucuronidation ([Fig. 4B](#F4)), although the effect was somewhat smaller, approaching a 30% decrease (*P* = 0.02, *t* test).

![Fig. 4.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c5d3/3629801/e7c05129b9d5/jpet.112.202010f4.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3629801_jpet.112.202010f4.jpg)

Repressive effect of UGT1A_i2 variant on UGT1A_i1-mediated acetaminophen glucuronidation activity. Top panel (A) shows acetaminophen glucuronidation activities measured in HEK293 cells transiently transfected with plasmids encoding UGT1A6_i1 (2 µg) and increasing amounts of UGT1A6_i2 (0 to 2 µg). Data were expressed as a percentage of activities in control wells that lacked UGT1A6_i2. Bars represent the mean and S.D. of three independent experiments conducted in duplicate. Also shown are the P values for significant (P < 0.05) pairwise multiple comparisons to the control group using the Holm-Sidak test (P = 0.02, ANOVA). Bottom panel (B) shows acetaminophen glucuronidation activities measured in HEK293 cell lines stably expressing either UGT1A1_i1 alone or both UGT1A1_i1 and UGT1A1_i2 (ratio of 1:0.33 by immunoblot). Data were expressed as a percentage of activities in cell lines expressing only UGT1A1_i1 and were adjusted for the relative UGT1A1_i1 content of each cell line. Bars represent the mean and SD of four independent experiments. Also shown is the P value for the comparison in activities between cell lines by t test.

#### UGT1A Exon 5a/5b Splicing Is Not Associated with Other Variants Between Exon 3 and Exon 5a.

The apparent effect of the 3′UTR SNPs on exon 5a versus exon 5b splicing could be explained by another genetic variant in high-linkage disequilibrium and located in an exon 5a or 5b splice regulatory site on the primary transcript. Consequently, the UGT1A gene region from exons 3–5a (including introns and exon 5b) was scanned by resequencing liver samples with high (*n* = 8) and low (*n* = 8) hepatic UGT1A exon 5a/5b mRNA splicing ratios (see [Fig. 5](#F5)). Seven additional variants were identified by analyzing sequence chromatograms from these samples with allele frequencies ranging from 0.13 to 0.34 ([Fig. 5](#F5)). All were SNPs that had been submitted previously to the dbSNP database. One additional SNP (rs1247441) referenced in dbSNP as being located in the intron between exon 5b and exon 5a showed no genetic diversity in the 16 samples assayed. Although, as expected, a clear association between each of the 3′UTR SNPs and exon 5a/5b mRNA splicing ratios (*P* < 0.006, Fisher's exact test), was found, none of the newly identified SNPs was associated with splicing phenotype (*P* > 0.05) or was in linkage with the 3′UTR SNPs.

![Fig. 5.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c5d3/3629801/95c52a4b0abd/jpet.112.202010f5.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3629801_jpet.112.202010f5.jpg)

SNPs identified by resequencing DNA from human livers with high (n = 8) and low (n = 8) exon 5a to exon 5b variant mRNA ratios. Shown below are a graphic depiction of the UGT1A shared exons 2 to 5 region (5′ to 3′ with introns), the locations of primer pairs (given in Table 1) used for resequencing, and the locations of SNPs (and their dbSNP accession numbers) that were identified. Shown above are the P values for a Fisher's exact test comparing SNP genotype frequencies in the low mRNA ratio versus the high mRNA ratio livers. Only the UGT1A-3′UTR SNPs (rs10929303, rs1042640, and rs8330) were significantly associated with mRNA ratios (P < 0.05).

#### UGT1A-3′UTR SNPs Create Novel Splice-Enhancer Sites.

Genetic variants can modulate gene splicing through creation or abolition of splice enhancer sites. Consequently, possible effects of the UGT1A 3′UTR SNPs on binding site consensus sequences for the Srp family of splicing regulatory proteins (SF2/ASF, SC35, SRp40, and SRp55) were explored using an in silico approach. As shown in [Fig. 6A](#F6), this analysis indicated that compared with the reference allele, the rs1042640 G allele variant creates a novel SRp55 site, whereas the rs8330 C allele variant creates two novel sites (SF2/ASF and SRp40). These findings suggest a model, shown in [Fig. 6B](#F6), whereby the presence of the 3′UTR SNP variants favors utilization of exon 4–5a splicing over the exon 4–5b splicing pathway. This results in increased levels of the active UGT1A_i1 isoforms relative to the repressor UGT1A_i2 repressors and overall increased UGT1A-mediated glucuronidation.

#### UGT1A-3′UTR SNPs Are Associated with Acetaminophen-Induced Acute Liver Failure.

Glucuronidation is the principal clearance mechanism for acetaminophen, and so higher acetaminophen glucuronidation would be expected to protect individuals from the well-known hepatotoxic effects of acetaminophen. Consequently, we used DNA samples from an ongoing large multicenter trial of ALF to compare UGT1A-3′UTR SNP genotype frequencies in patients who had developed ALF either unintentionally with chronic use of acetaminophen, intentionally with an acute acetaminophen overdose, or from causes other than acetaminophen. As shown in [Table 4](#T4), genotype frequency differences were found between these subgroups of ALF patients. Specifically, all the UGT1A-3′UTR variant carrier genotypes were substantially underrepresented by about 2-fold in the unintentional acetaminophen hepatotoxicity subgroup compared with the other two subgroups with odds ratios (95% confidence intervals) of 0.52 (0.29–0.93, *P* = 0.025), 0.56 (0.31–0.99, *P* = 0.045), and 0.53 (0.30–0.94, *P* = 0.027) for rs10929303, rs1042640, and rs8330, respectively.

|   |   |   |  | Acetaminophen |  |  |  |  |  |  |   |   |   |   |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
|   |   | Unintentional |  |  | Intentional |  |  | Other Causes |  |  |  |  |  |  |
| Genotype |  | N | (%) | P Value | N | (%) | P Value | N | (%) | P Value |  |  |  |  |
| rs10929303 | C/C | 57 | (72) | *0.025 | 43 | (54) | 0.094 | 62 | (60) | 0.61 |  |  |  |  |
|   | C/T+T/T | 22 | (28) |   | 36 | (46) |   | 41 | (40) |   |  |  |  |  |
| rs1042640 | C/C | 57 | (72) | *0.045 | 47 | (59) | 0.41 | 61 | (59) | 0.28 |  |  |  |  |
|   | C/G+G/G | 22 | (28) |   | 32 | (41) |   | 42 | (41) |   |  |  |  |  |
| rs8330 | C/C | 56 | (71) | *0.027 | 42 | (54) | 0.13 | 60 | (58) | 0.50 |  |  |  |  |
|   | C/G+G/G | 23 | (29) |   | 36 | (46) |   | 43 | (42) |   |  |  |  |  |

Table Caption: Comparisons of UGT1A-3′UTR single nucleotide polymorphism genotype frequencies determined in patients (N = 261 total) who had developed acute liver failure either unintentionally from chronic acetaminophen use (n = 79), intentionally from acute acetaminophen overdose (n = 79), or from causes other than acetaminophen (n = 103)

#### Rs8330 Allele Frequency Varies between Populations with Different Geographic Origin.

Since the phenotypes (i.e., acetaminophen glucuronidation and risk of acetaminophen-induced hepatotoxicity) that we have associated with the rs8330 SNP could vary between populations in relation to their geographic origin, we compared the minor allele frequencies determined for rs8330 in this study to available published and database frequencies reported for other populations. As shown in [Table 5](#T5), substantial geographic differences with low frequencies were observed for East Asian populations (0.11–0.16), with high frequencies for African populations (0.39 and 0.50) and intermediate frequencies for ancestral European (“white”) populations (0.21–0.25). Consistent with the reported race or ethnicity of the DNA sample donors used in this study, allele frequencies for the human liver bank samples (0.21), the intentional acetaminophen-induced ALF subjects (0.22), and the subjects with ALF from other causes (0.26) most closely resembled those of other European ancestral populations, whereas the unintentional acetaminophen-induced ALF subjects had a considerably lower frequency (0.16).

| Sample Source | Population | No. of Alleles | MAF | Reference |
| --- | --- | --- | --- | --- |
| Unintentional acetaminophen- induced acute liver failure patients | White American | 158 | 0.16 | This study |
| Intentional acetaminophen -induced acute liver failure patients | White American | 158 | 0.26 | This study |
| Acute liver failure patients from causes other than acetaminophen | White American | 206 | 0.22 | This study |
| Tufts liver bank samples | White American | 96 | 0.21 | This study |
| French Canadian cohort | French Canadian | 508 | 0.22 | Menard et al., 2009 |
| 1000Genomes | European | 758 | 0.22 | 1000 Genomesb |
| Hapmap | Utah residents from Northern & Western Europe | 120 | 0.23 | dbSNPa |
| 1000Genomes | Admixed American | 362 | 0.25 | 1000 Genomesb |
| 1000Genomes | African | 492 | 0.39 | 1000 Genomesb |
| Hapmap | Yoruba in Ibadan, Nigeria | 120 | 0.50 | dbSNPa |
| Patients given irinotecan or antiarrhythmics | Japanese | 602 | 0.11 | Saeki et al., 2006 |
| Seon-Young Kim, Korea Research Institute of Bioscience and Biotechnology | Korean | 178 | 0.13 | dbSNP a |
| 1000Genomes | East Asian | 572 | 0.13 | 1000 Genomesb |
| Hapmap | Japanese in Tokyo, Japan | 88 | 0.16 | dbSNPa |
| Hapmap | Han Chinese in Beijing | 90 | 0.16 | dbSNP a |

Table Caption: Comparison of rs8330 minor allele frequencies (MAF) determined for subjects in the current study with published or data base values for different human populations

## Discussion

To our knowledge, this is the first study to identify genetic polymorphisms associated with interindividual variability in acetaminophen glucuronidation in the human liver. Of the SNPs evaluated, only those located in the UGT1A-3′UTR were associated with variable acetaminophen glucuronidation. The UGT1A-3′UTR SNPs, rather than those located within individual UGT1A isoform genes, have the greatest potential to impact the disposition of drugs (such as acetaminophen) that are glucuronidated by multiple UGT1A isoforms, since SNPs affecting only an individual UGT1A enzyme could be compensated for by acetaminophen glucuronidation by other unaffected UGT1A isoforms. Of the three UGT1A-3′UTR SNPs, rs8330 was most consistently associated with glucuronidation phenotype at all three acetaminophen concentrations tested (0.1, 2, and 40 mM). Consequently, this SNP is likely to be the most robust biomarker of acetaminophen glucuronidation over a wide range of acetaminophen dosages (clinical to toxic).

Few studies have evaluated the association between the UGT1A-3′UTR SNPs and UGT1A glucuronidation phenotype. A prior study comprising 85 Japanese cancer patients receiving irinotecan found a trend for higher total bilirubin levels in individuals with the variant 3′UTR TGG haplotype (termed **1B*) after stratifying by *UGT1A1*28* genotype, presumably reflecting decreased UGT1A1-mediated bilirubin glucuronidation ([Sai et al., 2004](#B41)). However, in the same subjects, UGT1A haplotype had no effect on metabolic ratios of SN-38 glucuronide to SN-38, an irinotecan metabolite glucuronidated by hepatic UGT1A1. Furthermore, in a somewhat larger study of 125 African American and white women, rs10929303 (c.1813C>T) was not associated with altered bilirubin levels, even after controlling for UGT1A1*28 genotype effect ([Hong et al., 2007](#B21)). Finally, a weak (although statistically significant) association was reported between a UGT1A-3′UTR SNP and elevated liver function tests in patients taking tolcapone, presumably because of decreased tolcapone glucuronidation ([Acuna et al., 2002](#B1)). Unfortunately, the exact identity of this SNP is unclear since it was given as “C908G” in the report and could be either rs1042640 or rs8330.

None of the UGT1A isoform-specific SNPs evaluated here was associated with altered hepatic acetaminophen glucuronidation. This finding is in agreement with the results of a pharmacokinetic study that showed no association of UGT1A1*28 genotype with urinary acetaminophen glucuronide to acetaminophen ratios in 23 healthy male subjects ([Rauchschwalbe et al., 2004](#B33)). However, another study in patients with *β*-thalassemia showed decreased acetaminophen area under the plasma concentration-time curve, presumably reflecting increased clearance by glucuronidation, in patients carrying UGT1A6*2 and lacking UGT1A1*28 compared with subjects who carried neither variant allele ([Tankanitlert et al., 2007](#B37)). Unfortunately, that study was limited by small sample size (only five subjects per group), possible effect of disease, and also conflicting evidence in that both plasma acetaminophen glucuronide and sulfate plasma concentration-time curves were also decreased in the UGT1A6*2 carriers, which is not consistent with enhanced glucuronidation. Finally, a recent study in 66 healthy subjects showed a trend for increased urinary acetaminophen glucuronide to parent ratios (by almost 30%) in UGT1A6*2 carriers ([Navarro et al., 2011](#B28)). Given the results of the present study, these weak associations could be the result of partial linkage of the UGT1A1 and UGT1A6 SNPs to the UGT1A-3′UTR SNPs. Consequently, future acetaminophen pharmacokinetic studies should evaluate associations with the UGT1A-3′UTR SNPs. The latter study ([Navarro et al., 2011](#B28)) also showed a somewhat stronger gene-dose dependent association between UGT2B15*2 and decreased urinary acetaminophen glucuronide to parent ratios (by about 30%), which contrasts with our results showing no association of this allele with hepatic acetaminophen glucuronidation. The reason for the difference is unclear; however, it should be noted that saliva levels, rather than blood levels of acetaminophen, were reported in that study, and so it was not possible to calculate partial metabolite clearances, which may be a more independent index of metabolic capacity.

The association of acetaminophen glucuronidation with genotype was strongest for the rs8330, suggesting that rs8330 might be the causal SNP rather than another linked variant on the same UGT1A-3′UTR haplotype block. Consequently, we conducted a series of mechanistic studies to explore this hypothesis. We found no evidence that the UGT1A-3′UTR SNPs influence mRNA stability or translation efficiency such as through altering binding sites for microRNAs or other post-transcriptional regulators. However, allele-specific assay of heterozygous liver RNA suggested that there was a statistically significant (albeit small) imbalance with higher variant versus reference allele mRNA levels. This observation was further corroborated by quantitation of UGT1A exon 5 mRNA splice variants, which indicated that the rs8330-containing allele was associated with increased levels of exon 5a versus exon 5b mRNA transcripts.

Exon 5b containing UGT1A transcripts encode for enzymatically inactive truncated proteins lacking the C-terminal transmembrane domain (termed isoform 2 variant or i2), which might play a role in the regulation of the isoform 1 (i1) enzyme through hetero-oligomerization and repression (see proposed model in [Fig. 6B](#F6)). This contention is supported by our i1/i2 coexpression studies that showed a clear repressive effect of coexpression of the i2 variant on acetaminophen glucuronidation mediated by both of the i1 forms of UGT1A1 and UGT1A6. A prior study also showed about 60% lower deferiprone glucuronidation in HEK293 cells stably expressing UGT1A6i1 plus UGT1A6i2 versus UGT1A6i1 alone (normalized by i1 immunoblot content) ([Benoit-Biancamano et al., 2009](#B5)). Although a recent paper ([Jones et al., 2012](#B22)) suggests that UGT1A i2 levels may be too low in human liver to affect UGT1A i1 activity, these authors measured only mRNA levels, and it is unclear whether i2 protein levels are correspondingly low. Interestingly, they did show significant enhancement of raloxifene glucuronidation in HepG2 (hepatocellular carcinoma G2) cells by siRNA-mediated knockdown of UGT1a i2 mRNA. An earlier study also using siRNA-mediated knockdown of endogenous i2 also demonstrated upregulation of cellular glucuronidation activities in colon cancer cells for various substrates of UGT1A enzymes ([Bellemare et al., 2010b](#B4)).

With regard to the mechanism by which the UGT1A-3′UTR SNPs might regulate exon 5a/5b splicing, our in silico analysis indicated that these SNPs alter predicted consensus binding elements for splice regulatory enhancer proteins, with the greatest effect observed for rs8330. Further gene sequencing failed to identify another gene variant that could affect exon 5 splicing, although it is possible that such a variant may exist either upstream of exon 3 or downstream of exon 5a. Although beyond the scope of the current work, a direct method to prove the role of one or more of the 3′UTR SNPs in regulating UGT1A splicing would be through analysis of allelic variants of exon-containing minigene constructs, as was recently reported for the CYP2B6 gene ([Hofmann et al., 2008](#B20)).

This is also the first study to identify genetic variants associated with susceptibility to ALF in patients with a history of chronically consuming large amounts of acetaminophen. The lower frequency of the UGT1A-3′UTR genotypes in these patients is consistent with a protective effect of the variant allele resulting from more extensive detoxification of acetaminophen via glucuronidation. However, we did not observe this association in patients who had developed ALF from intentional acetaminophen overdose. One possible reason for this difference is that the intentional acetaminophen overdose patients tended to ingest quite large amounts of acetaminophen (median of 26 g per patient in this study) as a single dose such that detoxification via glucuronidation may have been overwhelmed regardless of genotype. In contrast, although the unintentional toxicity patients tended to consume even higher total amounts of acetaminophen (median of 45 g per patient in this study), they did so over a longer period (median of 7 days in this study), resulting in a lower daily dose (median of 6.5 g per day) that might not have overwhelmed the glucuronidation pathway, allowing for an influence of the rs8330 variant.

A weakness of the current study is that we were unable to identify an appropriate comparator group, such as individuals who had consumed similar doses of acetaminophen over a similar time course as the unintentional acetaminophen toxicity patients had but had not subsequently developed ALF. However, comparisons with published data suggested that the rs8330 allele frequency in the unintentional acetaminophen toxicity patients was lower than expected for a white population of European origin. Compared with European populations, data for rs8330 also predict that African populations should have higher acetaminophen glucuronidation rates associated with a higher rs8330 frequency, whereas East Asian populations with a lower rs8330 frequency should have lower acetaminophen glucuronidation rates. Interestingly, higher acetaminophen glucuronide excretion rates have been reported for Ghanaian (West African) and Kenyan (East African) versus white populations ([Critchley et al., 1986](#B15)); no differences were reported in another study of a Venda (South African) versus a white population ([Sommers et al., 1987](#B36)). Lower acetaminophen glucuronidation was reported in Hong Kong Chinese versus white subjects in one study ([Critchley et al., 2005](#B14)), although differences (versus whites) were not observed in a study of Chinese living in Australia ([Osborne et al., 1991](#B30)) or for “Orientals” living in Canada ([Patel et al., 1992](#B32)). Further studies are needed to clarify the relationship between genetic polymorphisms, acetaminophen metabolism, and susceptibility to acetaminophen toxicity in different human populations.

## Supplementary Material

## Abbreviations

## Authorship Contributions

*Participated in research design*: Court, Greenblatt, Lee.

*Conducted experiments*: Court, Freytsis, Wang, Hazarika, Duan.

*Contributed new reagents or analytic tools*: Guillemette.

*Performed data analysis*: Court, Freytsis, Wang, Peter, Duan, Hazarika.

*Wrote or contributed to the writing of the manuscript*: Court, Peter, Guillemette, Greenblatt, Lee.

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Acuña G, Foernzler D, Leong D, Rabbia M, Smit R, Dorflinger E, Gasser R, Hoh J, Ott J, Borroni E, et al. (2002) Pharmacogenetic analysis of adverse drug effect reveals genetic variant for susceptibility to liver toxicity. Pharmacogenomics J 2:327–334  [DOI](https://doi.org/10.1038/sj.tpj.6500123) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12439739/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=Pharmacogenetic%20analysis%20of%20adverse%20drug%20effect%20reveals%20genetic%20variant%20for%20susceptibility%20to%20liver%20toxicity&author=G%20Acu%C3%B1a&author=D%20Foernzler&author=D%20Leong&author=M%20Rabbia&author=R%20Smit&volume=2&publication_year=2002&pages=327-334&pmid=12439739&doi=10.1038/sj.tpj.6500123&)

2. Barrett JC, Fry B, Maller J, Daly MJ. (2005) Haploview: analysis and visualization of LD and haplotype maps. Bioinformatics 21:263–265  [DOI](https://doi.org/10.1093/bioinformatics/bth457) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15297300/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=Haploview:%20analysis%20and%20visualization%20of%20LD%20and%20haplotype%20maps&author=JC%20Barrett&author=B%20Fry&author=J%20Maller&author=MJ%20Daly&volume=21&publication_year=2005&pages=263-265&pmid=15297300&doi=10.1093/bioinformatics/bth457&)

3. Bellemare J, Rouleau M, Harvey M, Guillemette C. (2010a) Modulation of the human glucuronosyltransferase UGT1A pathway by splice isoform polypeptides is mediated through protein-protein interactions. J Biol Chem 285:3600–3607  [DOI](https://doi.org/10.1074/jbc.M109.083139) | [PMC free article](/articles/PMC2823500/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19996319/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=Modulation%20of%20the%20human%20glucuronosyltransferase%20UGT1A%20pathway%20by%20splice%20isoform%20polypeptides%20is%20mediated%20through%20protein-protein%20interactions&author=J%20Bellemare&author=M%20Rouleau&author=M%20Harvey&author=C%20Guillemette&volume=285&publication_year=2010&pages=3600-3607&pmid=19996319&doi=10.1074/jbc.M109.083139&)

4. Bellemare J, Rouleau M, Harvey M, Têtu B, Guillemette C. (2010b) Alternative-splicing forms of the major phase II conjugating UGT1A gene negatively regulate glucuronidation in human carcinoma cell lines. Pharmacogenomics J 10:431–441  [DOI](https://doi.org/10.1038/tpj.2009.64) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19997083/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=Alternative-splicing%20forms%20of%20the%20major%20phase%20II%20conjugating%20UGT1A%20gene%20negatively%20regulate%20glucuronidation%20in%20human%20carcinoma%20cell%20lines&author=J%20Bellemare&author=M%20Rouleau&author=M%20Harvey&author=B%20T%C3%AAtu&author=C%20Guillemette&volume=10&publication_year=2010&pages=431-441&pmid=19997083&doi=10.1038/tpj.2009.64&)

5. Benoit-Biancamano MO, Connelly J, Villeneuve L, Caron P, Guillemette C. (2009) Deferiprone glucuronidation by human tissues and recombinant UDP glucuronosyltransferase 1A6: an in vitro investigation of genetic and splice variants. Drug Metab Dispos 37:322–329  [DOI](https://doi.org/10.1124/dmd.108.023101) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18971318/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Deferiprone%20glucuronidation%20by%20human%20tissues%20and%20recombinant%20UDP%20glucuronosyltransferase%201A6:%20an%20in%20vitro%20investigation%20of%20genetic%20and%20splice%20variants&author=MO%20Benoit-Biancamano&author=J%20Connelly&author=L%20Villeneuve&author=P%20Caron&author=C%20Guillemette&volume=37&publication_year=2009&pages=322-329&pmid=18971318&doi=10.1124/dmd.108.023101&)

6. Bock KW, Forster A, Gschaidmeier H, Brück M, Münzel P, Schareck W, Fournel-Gigleux S, Burchell B. (1993) Paracetamol glucuronidation by recombinant rat and human phenol UDP-glucuronosyltransferases. Biochem Pharmacol 45:1809–1814  [DOI](https://doi.org/10.1016/0006-2952(93)90437-2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8494539/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem%20Pharmacol&title=Paracetamol%20glucuronidation%20by%20recombinant%20rat%20and%20human%20phenol%20UDP-glucuronosyltransferases&author=KW%20Bock&author=A%20Forster&author=H%20Gschaidmeier&author=M%20Br%C3%BCck&author=P%20M%C3%BCnzel&volume=45&publication_year=1993&pages=1809-1814&pmid=8494539&doi=10.1016/0006-2952(93)90437-2&)

7. Buchard A, Eefsen M, Semb S, Andersen SE, Morling N, Bendtsen F, Larsen FS, Dalhoff K. (2012) The role of the glutathione S-transferase genes GSTT1, GSTM1, and GSTP1 in acetaminophen-poisoned patients. Clin Toxicol (Phila) 50:27–33  [DOI](https://doi.org/10.3109/15563650.2011.639713) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22175791/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Toxicol%20(Phila)&title=The%20role%20of%20the%20glutathione%20S-transferase%20genes%20GSTT1,%20GSTM1,%20and%20GSTP1%20in%20acetaminophen-poisoned%20patients&author=A%20Buchard&author=M%20Eefsen&author=S%20Semb&author=SE%20Andersen&author=N%20Morling&volume=50&publication_year=2012&pages=27-33&pmid=22175791&doi=10.3109/15563650.2011.639713&)

8. Cartegni L, Wang J, Zhu Z, Zhang MQ, Krainer AR. (2003) ESEfinder: a web resource to identify exonic splicing enhancers. Nucleic Acids Res 31:3568–3571  [DOI](https://doi.org/10.1093/nar/gkg616) | [PMC free article](/articles/PMC169022/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12824367/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=ESEfinder:%20a%20web%20resource%20to%20identify%20exonic%20splicing%20enhancers&author=L%20Cartegni&author=J%20Wang&author=Z%20Zhu&author=MQ%20Zhang&author=AR%20Krainer&volume=31&publication_year=2003&pages=3568-3571&pmid=12824367&doi=10.1093/nar/gkg616&)

9. Court MH. (2010) Interindividual variability in hepatic drug glucuronidation: studies into the role of age, sex, enzyme inducers, and genetic polymorphism using the human liver bank as a model system. Drug Metab Rev 42:209–224  [DOI](https://doi.org/10.3109/03602530903209288) | [PMC free article](/articles/PMC6174030/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19821798/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Rev&title=Interindividual%20variability%20in%20hepatic%20drug%20glucuronidation:%20studies%20into%20the%20role%20of%20age,%20sex,%20enzyme%20inducers,%20and%20genetic%20polymorphism%20using%20the%20human%20liver%20bank%20as%20a%20model%20system&author=MH%20Court&volume=42&publication_year=2010&pages=209-224&pmid=19821798&doi=10.3109/03602530903209288&)

10. Court MH, Duan SX, von Moltke LL, Greenblatt DJ, Patten CJ, Miners JO, Mackenzie PI. (2001) Interindividual variability in acetaminophen glucuronidation by human liver microsomes: identification of relevant acetaminophen UDP-glucuronosyltransferase isoforms. J Pharmacol Exp Ther 299:998–1006  [PubMed](https://pubmed.ncbi.nlm.nih.gov/11714888/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharmacol%20Exp%20Ther&title=Interindividual%20variability%20in%20acetaminophen%20glucuronidation%20by%20human%20liver%20microsomes:%20identification%20of%20relevant%20acetaminophen%20UDP-glucuronosyltransferase%20isoforms&author=MH%20Court&author=SX%20Duan&author=LL%20von%20Moltke&author=DJ%20Greenblatt&author=CJ%20Patten&volume=299&publication_year=2001&pages=998-1006&pmid=11714888&)

11. Court MH, Hao Q, Krishnaswamy S, Bekaii-Saab T, Al-Rohaimi A, von Moltke LL, Greenblatt DJ. (2004) UDP-glucuronosyltransferase (UGT) 2B15 pharmacogenetics: UGT2B15 D85Y genotype and gender are major determinants of oxazepam glucuronidation by human liver. J Pharmacol Exp Ther 310:656–665  [DOI](https://doi.org/10.1124/jpet.104.067660) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15044558/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharmacol%20Exp%20Ther&title=UDP-glucuronosyltransferase%20(UGT)%202B15%20pharmacogenetics:%20UGT2B15%20D85Y%20genotype%20and%20gender%20are%20major%20determinants%20of%20oxazepam%20glucuronidation%20by%20human%20liver&author=MH%20Court&author=Q%20Hao&author=S%20Krishnaswamy&author=T%20Bekaii-Saab&author=A%20Al-Rohaimi&volume=310&publication_year=2004&pages=656-665&pmid=15044558&doi=10.1124/jpet.104.067660&)

12. Court MH, Von Moltke LL, Shader RI, Greenblatt DJ. (1997) Biotransformation of chlorzoxazone by hepatic microsomes from humans and ten other mammalian species. Biopharm Drug Dispos 18:213–226  [DOI](https://doi.org/10.1002/(sici)1099-081x(199704)18:3<213::aid-bdd15>3.0.co;2-0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9113344/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biopharm%20Drug%20Dispos&title=Biotransformation%20of%20chlorzoxazone%20by%20hepatic%20microsomes%20from%20humans%20and%20ten%20other%20mammalian%20species&author=MH%20Court&author=LL%20Von%20Moltke&author=RI%20Shader&author=DJ%20Greenblatt&volume=18&publication_year=1997&pages=213-226&pmid=9113344&doi=10.1002/(sici)1099-081x(199704)18:3<213::aid-bdd15>3.0.co;2-0&)

13. Court MH, Zhang X, Ding X, Yee KK, Hesse LM, Finel M. (2012) Quantitative distribution of mRNAs encoding the 19 human UDP-glucuronosyltransferase enzymes in 26 adult and 3 fetal tissues. Xenobiotica 42:266–277  [DOI](https://doi.org/10.3109/00498254.2011.618954) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21995321/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Xenobiotica&title=Quantitative%20distribution%20of%20mRNAs%20encoding%20the%2019%20human%20UDP-glucuronosyltransferase%20enzymes%20in%2026%20adult%20and%203%20fetal%20tissues&author=MH%20Court&author=X%20Zhang&author=X%20Ding&author=KK%20Yee&author=LM%20Hesse&volume=42&publication_year=2012&pages=266-277&pmid=21995321&doi=10.3109/00498254.2011.618954&)

14. Critchley JA, Critchley LA, Anderson PJ, Tomlinson B. (2005) Differences in the single-oral-dose pharmacokinetics and urinary excretion of paracetamol and its conjugates between Hong Kong Chinese and Caucasian subjects. J Clin Pharm Ther 30:179–184  [DOI](https://doi.org/10.1111/j.1365-2710.2004.00626.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15811172/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Pharm%20Ther&title=Differences%20in%20the%20single-oral-dose%20pharmacokinetics%20and%20urinary%20excretion%20of%20paracetamol%20and%20its%20conjugates%20between%20Hong%20Kong%20Chinese%20and%20Caucasian%20subjects&author=JA%20Critchley&author=LA%20Critchley&author=PJ%20Anderson&author=B%20Tomlinson&volume=30&publication_year=2005&pages=179-184&pmid=15811172&doi=10.1111/j.1365-2710.2004.00626.x&)

15. Critchley JA, Nimmo GR, Gregson CA, Woolhouse NM, Prescott LF. (1986) Inter-subject and ethnic differences in paracetamol metabolism. Br J Clin Pharmacol 22:649–657  [DOI](https://doi.org/10.1111/j.1365-2125.1986.tb02953.x) | [PMC free article](/articles/PMC1401198/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/3567011/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Clin%20Pharmacol&title=Inter-subject%20and%20ethnic%20differences%20in%20paracetamol%20metabolism&author=JA%20Critchley&author=GR%20Nimmo&author=CA%20Gregson&author=NM%20Woolhouse&author=LF%20Prescott&volume=22&publication_year=1986&pages=649-657&pmid=3567011&doi=10.1111/j.1365-2125.1986.tb02953.x&)

16. Ge B, Gurd S, Gaudin T, Dore C, Lepage P, Harmsen E, Hudson TJ, Pastinen T. (2005) Survey of allelic expression using EST mining. Genome Res 15:1584–1591  [DOI](https://doi.org/10.1101/gr.4023805) | [PMC free article](/articles/PMC1310646/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16251468/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Res&title=Survey%20of%20allelic%20expression%20using%20EST%20mining&author=B%20Ge&author=S%20Gurd&author=T%20Gaudin&author=C%20Dore&author=P%20Lepage&volume=15&publication_year=2005&pages=1584-1591&pmid=16251468&doi=10.1101/gr.4023805&)

17. Girard H, Court MH, Bernard O, Fortier LC, Villeneuve L, Hao Q, Greenblatt DJ, von Moltke LL, Perussed L, Guillemette C. (2004) Identification of common polymorphisms in the promoter of the UGT1A9 gene: evidence that UGT1A9 protein and activity levels are strongly genetically controlled in the liver. Pharmacogenetics 14:501–515  [DOI](https://doi.org/10.1097/01.fpc.0000114754.08559.27) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15284532/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=Identification%20of%20common%20polymorphisms%20in%20the%20promoter%20of%20the%20UGT1A9%20gene:%20evidence%20that%20UGT1A9%20protein%20and%20activity%20levels%20are%20strongly%20genetically%20controlled%20in%20the%20liver&author=H%20Girard&author=MH%20Court&author=O%20Bernard&author=LC%20Fortier&author=L%20Villeneuve&volume=14&publication_year=2004&pages=501-515&pmid=15284532&doi=10.1097/01.fpc.0000114754.08559.27&)

18. Girard H, Thibaudeau J, Court MH, Fortier LC, Villeneuve L, Caron P, Hao Q, von Moltke LL, Greenblatt DJ, Guillemette C. (2005) UGT1A1 polymorphisms are important determinants of dietary carcinogen detoxification in the liver. Hepatology 42:448–457  [DOI](https://doi.org/10.1002/hep.20770) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15986396/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hepatology&title=UGT1A1%20polymorphisms%20are%20important%20determinants%20of%20dietary%20carcinogen%20detoxification%20in%20the%20liver&author=H%20Girard&author=J%20Thibaudeau&author=MH%20Court&author=LC%20Fortier&author=L%20Villeneuve&volume=42&publication_year=2005&pages=448-457&pmid=15986396&doi=10.1002/hep.20770&)

19. Henderson CJ, Wolf CR, Kitteringham N, Powell H, Otto D, Park BK. (2000) Increased resistance to acetaminophen hepatotoxicity in mice lacking glutathione S-transferase Pi. Proc Natl Acad Sci USA 97:12741–12745  [DOI](https://doi.org/10.1073/pnas.220176997) | [PMC free article](/articles/PMC18834/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11058152/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20USA&title=Increased%20resistance%20to%20acetaminophen%20hepatotoxicity%20in%20mice%20lacking%20glutathione%20S-transferase%20Pi&author=CJ%20Henderson&author=CR%20Wolf&author=N%20Kitteringham&author=H%20Powell&author=D%20Otto&volume=97&publication_year=2000&pages=12741-12745&pmid=11058152&doi=10.1073/pnas.220176997&)

20. Hofmann MH, Blievernicht JK, Klein K, Saussele T, Schaeffeler E, Schwab M, Zanger UM. (2008) Aberrant splicing caused by single nucleotide polymorphism c.516G>T [Q172H], a marker of CYP2B6*6, is responsible for decreased expression and activity of CYP2B6 in liver. J Pharmacol Exp Ther 325:284–292  [DOI](https://doi.org/10.1124/jpet.107.133306) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18171905/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharmacol%20Exp%20Ther&title=Aberrant%20splicing%20caused%20by%20single%20nucleotide%20polymorphism%20c.516G>T%20%5BQ172H%5D,%20a%20marker%20of%20CYP2B6*6,%20is%20responsible%20for%20decreased%20expression%20and%20activity%20of%20CYP2B6%20in%20liver&author=MH%20Hofmann&author=JK%20Blievernicht&author=K%20Klein&author=T%20Saussele&author=E%20Schaeffeler&volume=325&publication_year=2008&pages=284-292&pmid=18171905&doi=10.1124/jpet.107.133306&)

21. Hong AL, Huo D, Kim HJ, Niu Q, Fackenthal DL, Cummings SA, John EM, West DW, Whittemore AS, Das S, et al. (2007) UDP-Glucuronosyltransferase 1A1 gene polymorphisms and total bilirubin levels in an ethnically diverse cohort of women. Drug Metab Dispos 35:1254–1261  [DOI](https://doi.org/10.1124/dmd.106.014183) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17478602/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=UDP-Glucuronosyltransferase%201A1%20gene%20polymorphisms%20and%20total%20bilirubin%20levels%20in%20an%20ethnically%20diverse%20cohort%20of%20women&author=AL%20Hong&author=D%20Huo&author=HJ%20Kim&author=Q%20Niu&author=DL%20Fackenthal&volume=35&publication_year=2007&pages=1254-1261&pmid=17478602&doi=10.1124/dmd.106.014183&)

22. Jones NR, Sun D, Freeman WM, Lazarus P. (2012) Quantification of hepatic UDP glucuronosyltransferase 1A splice variant expression and correlation of UDP glucuronosyltransferase 1A1 variant expression with glucuronidation activity. J Pharmacol Exp Ther 342:720–729  [DOI](https://doi.org/10.1124/jpet.112.192658) | [PMC free article](/articles/PMC3422519/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22661630/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharmacol%20Exp%20Ther&title=Quantification%20of%20hepatic%20UDP%20glucuronosyltransferase%201A%20splice%20variant%20expression%20and%20correlation%20of%20UDP%20glucuronosyltransferase%201A1%20variant%20expression%20with%20glucuronidation%20activity&author=NR%20Jones&author=D%20Sun&author=WM%20Freeman&author=P%20Lazarus&volume=342&publication_year=2012&pages=720-729&pmid=22661630&doi=10.1124/jpet.112.192658&)

23. Khandelwal N, James LP, Sanders C, Larson AM, Lee WM, Acute Liver Failure Study Group (2011) Unrecognized acetaminophen toxicity as a cause of indeterminate acute liver failure. Hepatology 53:567–576  [DOI](https://doi.org/10.1002/hep.24060) | [PMC free article](/articles/PMC3208905/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21274877/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hepatology&title=Unrecognized%20acetaminophen%20toxicity%20as%20a%20cause%20of%20indeterminate%20acute%20liver%20failure&author=N%20Khandelwal&author=LP%20James&author=C%20Sanders&author=AM%20Larson&author=WM%20Lee&volume=53&publication_year=2011&pages=567-576&pmid=21274877&doi=10.1002/hep.24060&)

24. Krishnaswamy S, Hao Q, Al-Rohaimi A, Hesse LM, von Moltke LL, Greenblatt DJ, Court MH. (2005b) UDP glucuronosyltransferase (UGT) 1A6 pharmacogenetics: I. Identification of polymorphisms in the 5′-regulatory and exon 1 regions, and association with human liver UGT1A6 gene expression and glucuronidation. J Pharmacol Exp Ther 313:1331–1339  [DOI](https://doi.org/10.1124/jpet.104.081950) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15761114/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharmacol%20Exp%20Ther&title=UDP%20glucuronosyltransferase%20(UGT)%201A6%20pharmacogenetics:%20I.%20Identification%20of%20polymorphisms%20in%20the%205%E2%80%B2-regulatory%20and%20exon%201%20regions,%20and%20association%20with%20human%20liver%20UGT1A6%20gene%20expression%20and%20glucuronidation&author=S%20Krishnaswamy&author=Q%20Hao&author=A%20Al-Rohaimi&author=LM%20Hesse&author=LL%20von%20Moltke&volume=313&publication_year=2005&pages=1331-1339&pmid=15761114&doi=10.1124/jpet.104.081950&)

25. Larson AM. (2007) Acetaminophen hepatotoxicity. Clin Liver Dis 11:525–548, vi vi.  [DOI](https://doi.org/10.1016/j.cld.2007.06.006) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17723918/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Liver%20Dis&title=Acetaminophen%20hepatotoxicity&author=AM%20Larson&volume=11&publication_year=2007&pages=525-548,%20vi&pmid=17723918&doi=10.1016/j.cld.2007.06.006&)

26. Larson AM, Polson J, Fontana RJ, Davern TJ, Lalani E, Hynan LS, Reisch JS, Schiødt FV, Ostapowicz G, Shakil AO, et al. Acute Liver Failure Study Group (2005) Acetaminophen-induced acute liver failure: results of a United States multicenter, prospective study. Hepatology 42:1364–1372  [DOI](https://doi.org/10.1002/hep.20948) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16317692/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hepatology&title=Acetaminophen-induced%20acute%20liver%20failure:%20results%20of%20a%20United%20States%20multicenter,%20prospective%20study&author=AM%20Larson&author=J%20Polson&author=RJ%20Fontana&author=TJ%20Davern&author=E%20Lalani&volume=42&publication_year=2005&pages=1364-1372&pmid=16317692&doi=10.1002/hep.20948&)

27. Menard V, Girard H, Harvey M, Perusse L, Guillemette C. (2009) Analysis of inherited genetic variations at the UGT1 locus in the French-Canadian population. Hum Mutat 30:677–687  [DOI](https://doi.org/10.1002/humu.20946) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19204906/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mutat&title=Analysis%20of%20inherited%20genetic%20variations%20at%20the%20UGT1%20locus%20in%20the%20French-Canadian%20population&author=V%20Menard&author=H%20Girard&author=M%20Harvey&author=L%20Perusse&author=C%20Guillemette&volume=30&publication_year=2009&pages=677-687&pmid=19204906&doi=10.1002/humu.20946&)

28. Mutlib AE, Goosen TC, Bauman JN, Williams JA, Kulkarni S, Kostrubsky S. (2006) Kinetics of acetaminophen glucuronidation by UDP-glucuronosyltransferases 1A1, 1A6, 1A9 and 2B15: potential implications in acetaminophen-induced hepatotoxicity. Chem Res Toxicol 19:701–709  [DOI](https://doi.org/10.1021/tx050317i) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16696573/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Chem%20Res%20Toxicol&title=Kinetics%20of%20acetaminophen%20glucuronidation%20by%20UDP-glucuronosyltransferases%201A1,%201A6,%201A9%20and%202B15:%20potential%20implications%20in%20acetaminophen-induced%20hepatotoxicity&author=AE%20Mutlib&author=TC%20Goosen&author=JN%20Bauman&author=JA%20Williams&author=S%20Kulkarni&volume=19&publication_year=2006&pages=701-709&pmid=16696573&doi=10.1021/tx050317i&)

29. Navarro SL, Chen Y, Li L, Li SS, Chang JL, Schwarz Y, King IB, Potter JD, Bigler J, Lampe JW. (2011) UGT1A6 and UGT2B15 polymorphisms and acetaminophen conjugation in response to a randomized, controlled diet of select fruits and vegetables. Drug Metab Dispos 39:1650–1657  [DOI](https://doi.org/10.1124/dmd.111.039149) | [PMC free article](/articles/PMC3164274/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21666065/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=UGT1A6%20and%20UGT2B15%20polymorphisms%20and%20acetaminophen%20conjugation%20in%20response%20to%20a%20randomized,%20controlled%20diet%20of%20select%20fruits%20and%20vegetables&author=SL%20Navarro&author=Y%20Chen&author=L%20Li&author=SS%20Li&author=JL%20Chang&volume=39&publication_year=2011&pages=1650-1657&pmid=21666065&doi=10.1124/dmd.111.039149&)

30. Oleson L, von Moltke LL, Greenblatt DJ, Court MH. (2010) Identification of polymorphisms in the 3′-untranslated region of the human pregnane X receptor (PXR) gene associated with variability in cytochrome P450 3A (CYP3A) metabolism. Xenobiotica 40:146–162  [DOI](https://doi.org/10.3109/00498250903420243) | [PMC free article](/articles/PMC3786868/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20082578/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Xenobiotica&title=Identification%20of%20polymorphisms%20in%20the%203%E2%80%B2-untranslated%20region%20of%20the%20human%20pregnane%20X%20receptor%20(PXR)%20gene%20associated%20with%20variability%20in%20cytochrome%20P450%203A%20(CYP3A)%20metabolism&author=L%20Oleson&author=LL%20von%20Moltke&author=DJ%20Greenblatt&author=MH%20Court&volume=40&publication_year=2010&pages=146-162&pmid=20082578&doi=10.3109/00498250903420243&)

31. Osborne NJ, Tonkin AL, Miners JO. (1991) Interethnic differences in drug glucuronidation: a comparison of paracetamol metabolism in Caucasians and Chinese. Br J Clin Pharmacol 32:765–767  [PMC free article](/articles/PMC1368561/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/1768572/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Clin%20Pharmacol&title=Interethnic%20differences%20in%20drug%20glucuronidation:%20a%20comparison%20of%20paracetamol%20metabolism%20in%20Caucasians%20and%20Chinese&author=NJ%20Osborne&author=AL%20Tonkin&author=JO%20Miners&volume=32&publication_year=1991&pages=765-767&pmid=1768572&)

32. Ostapowicz G, Fontana RJ, Schiødt FV, Larson A, Davern TJ, Han SH, McCashland TM, Shakil AO, Hay JE, Hynan L,, et al. U.S. Acute Liver Failure Study Group (2002) Results of a prospective study of acute liver failure at 17 tertiary care centers in the United States. Ann Intern Med 137:947–954  [DOI](https://doi.org/10.7326/0003-4819-137-12-200212170-00007) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12484709/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Intern%20Med&title=Results%20of%20a%20prospective%20study%20of%20acute%20liver%20failure%20at%2017%20tertiary%20care%20centers%20in%20the%20United%20States&author=G%20Ostapowicz&author=RJ%20Fontana&author=FV%20Schi%C3%B8dt&author=A%20Larson&author=TJ%20Davern&volume=137&publication_year=2002&pages=947-954&pmid=12484709&doi=10.7326/0003-4819-137-12-200212170-00007&)

33. Patel M, Tang BK, Kalow W. (1992) Variability of acetaminophen metabolism in Caucasians and Orientals. Pharmacogenetics 2:38–45  [DOI](https://doi.org/10.1097/00008571-199202000-00007) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/1302042/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=Variability%20of%20acetaminophen%20metabolism%20in%20Caucasians%20and%20Orientals&author=M%20Patel&author=BK%20Tang&author=W%20Kalow&volume=2&publication_year=1992&pages=38-45&pmid=1302042&doi=10.1097/00008571-199202000-00007&)

34. Rauchschwalbe SK, Zühlsdorf MT, Wensing G, Kuhlmann J. (2004) Glucuronidation of acetaminophen is independent of UGT1A1 promotor genotype. Int J Clin Pharmacol Ther 42:73–77  [DOI](https://doi.org/10.5414/cpp42073) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15180166/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Clin%20Pharmacol%20Ther&title=Glucuronidation%20of%20acetaminophen%20is%20independent%20of%20UGT1A1%20promotor%20genotype&author=SK%20Rauchschwalbe&author=MT%20Z%C3%BChlsdorf&author=G%20Wensing&author=J%20Kuhlmann&volume=42&publication_year=2004&pages=73-77&pmid=15180166&doi=10.5414/cpp42073&)

35. Saeki M, Saito Y, Jinno H, Sai K, Ozawa S, Kurose K, Kaniwa N, Komamura K, Kotake T, Morishita H, et al. (2006) Haplotype structures of the UGT1A gene complex in a Japanese population. Pharmacogenomics J 6:63–75  [DOI](https://doi.org/10.1038/sj.tpj.6500335) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16314888/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=Haplotype%20structures%20of%20the%20UGT1A%20gene%20complex%20in%20a%20Japanese%20population&author=M%20Saeki&author=Y%20Saito&author=H%20Jinno&author=K%20Sai&author=S%20Ozawa&volume=6&publication_year=2006&pages=63-75&pmid=16314888&doi=10.1038/sj.tpj.6500335&)

36. Sai K, Saeki M, Saito Y, Ozawa S, Katori N, Jinno H, Hasegawa R, Kaniwa N, Sawada J, Komamura K, et al. (2004) UGT1A1 haplotypes associated with reduced glucuronidation and increased serum bilirubin in irinotecan-administered Japanese patients with cancer. Clin Pharmacol Ther 75:501–515  [DOI](https://doi.org/10.1016/j.clpt.2004.01.010) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15179405/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=UGT1A1%20haplotypes%20associated%20with%20reduced%20glucuronidation%20and%20increased%20serum%20bilirubin%20in%20irinotecan-administered%20Japanese%20patients%20with%20cancer&author=K%20Sai&author=M%20Saeki&author=Y%20Saito&author=S%20Ozawa&author=N%20Katori&volume=75&publication_year=2004&pages=501-515&pmid=15179405&doi=10.1016/j.clpt.2004.01.010&)

37. Schiødt FV, Rochling FA, Casey DL, Lee WM. (1997) Acetaminophen toxicity in an urban county hospital. N Engl J Med 337:1112–1117  [DOI](https://doi.org/10.1056/NEJM199710163371602) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9329933/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Acetaminophen%20toxicity%20in%20an%20urban%20county%20hospital&author=FV%20Schi%C3%B8dt&author=FA%20Rochling&author=DL%20Casey&author=WM%20Lee&volume=337&publication_year=1997&pages=1112-1117&pmid=9329933&doi=10.1056/NEJM199710163371602&)

38. Sommers DK, Moncrieff J, Avenant JC. (1987) Paracetamol conjugation: an interethnic and dietary study. Hum Toxicol 6:407–409  [DOI](https://doi.org/10.1177/096032718700600512) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/3679251/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Toxicol&title=Paracetamol%20conjugation:%20an%20interethnic%20and%20dietary%20study&author=DK%20Sommers&author=J%20Moncrieff&author=JC%20Avenant&volume=6&publication_year=1987&pages=407-409&pmid=3679251&doi=10.1177/096032718700600512&)

39. Tankanitlert J, Morales NP, Howard TA, Fucharoen P, Ware RE, Fucharoen S, Chantharaksri U. (2007) Effects of combined UDP-glucuronosyltransferase (UGT) 1A1*28 and 1A6*2 on paracetamol pharmacokinetics in beta-thalassemia/HbE. Pharmacology 79:97–103  [DOI](https://doi.org/10.1159/000097908) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17164591/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacology&title=Effects%20of%20combined%20UDP-glucuronosyltransferase%20(UGT)%201A1*28%20and%201A6*2%20on%20paracetamol%20pharmacokinetics%20in%20beta-thalassemia/HbE&author=J%20Tankanitlert&author=NP%20Morales&author=TA%20Howard&author=P%20Fucharoen&author=RE%20Ware&volume=79&publication_year=2007&pages=97-103&pmid=17164591&doi=10.1159/000097908&)

40. Volak LP, Court MH. (2010) Role for protein kinase C delta in the functional activity of human UGT1A6: implications for drug-drug interactions between PKC inhibitors and UGT1A6. Xenobiotica 40:306–318  [DOI](https://doi.org/10.3109/00498251003596817) | [PMC free article](/articles/PMC6170201/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20196639/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Xenobiotica&title=Role%20for%20protein%20kinase%20C%20delta%20in%20the%20functional%20activity%20of%20human%20UGT1A6:%20implications%20for%20drug-drug%20interactions%20between%20PKC%20inhibitors%20and%20UGT1A6&author=LP%20Volak&author=MH%20Court&volume=40&publication_year=2010&pages=306-318&pmid=20196639&doi=10.3109/00498251003596817&)

41. Zhao L, Pickering G. (2011) Paracetamol metabolism and related genetic differences. Drug Metab Rev 43:41–52  [DOI](https://doi.org/10.3109/03602532.2010.527984) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21108564/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Rev&title=Paracetamol%20metabolism%20and%20related%20genetic%20differences&author=L%20Zhao&author=G%20Pickering&volume=43&publication_year=2011&pages=41-52&pmid=21108564&doi=10.3109/03602532.2010.527984&)
