# Most Common SNPs Associated with Rheumatoid Arthritis in Subjects of European Ancestry Confer Risk of Rheumatoid Arthritis in African-Americans

## Metadata
**Authors:** Laura B Hughes, Richard J Reynolds, Elizabeth E Brown, James M Kelley, Brian Thomson, Doyt L Conn, Beth L Jonas, Andrew O Westfall, Miguel A Padilla, Leigh F Callahan, Edwin A Smith, Richard D Brasington, Jr, Jeffrey C Edberg, Robert P Kimberly, Larry W Moreland, Robert M Plenge, S Louis Bridges, Jr
**Journal:** Arthritis and Rheumatism
**Date:** 2010 Dec
**DOI:** [10.1002/art.27732](https://doi.org/10.1002/art.27732)
**PMID:** 21120996
**PMCID:** PMC3030622
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3030622/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC3030622/pdf/nihms232158.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC3030622/pdf/nihms232158.pdf)

## Abstract

**Objective:** 
Large-scale genetic association studies have identified over 20 rheumatoid arthritis (RA) risk alleles among individuals of European ancestry. The influence of these risk alleles has not been comprehensively studied in African-Americans. We therefore sought to examine whether these validated RA risk alleles are associated with RA in an African-American population.

**Methods:** 
27 candidate SNPs were genotyped in 556 autoantibody-positive African-Americans with RA and 791 healthy African-American controls. Odds ratios (OR) and 95% confidence intervals (CI) for each SNP were compared to previously published ORs of RA patients of European ancestry. We then calculated a composite Genetic Risk Score (GRS) for each individual based on the sum of all risk alleles.

**Results:** 
There was overlap in the OR and 95% CI between the European and African-American populations in 24 of the 27 candidate SNPs. Conversely, 3 of the 27 SNPs (CCR6 rs3093023, TAGAP rs394581, TNFAIP3 rs6920220) demonstrated an OR in the opposite direction from those reported in RA patients of European ancestry. The GRS analysis indicated a small but highly significant probability that African-American cases were enriched for the European RA risk alleles relative to controls (p=0.00005).

**Conclusion:** 
The majority of RA risk alleles previously validated among European ancestry RA patients showed similar ORs in our population of African-Americans with RA. Furthermore, the aggregate GRS supports the hypothesis that these SNPs are risk alleles for RA in the African-American population. Future large-scale genetic studies are needed to validate these risk alleles and identify novel risk alleles for RA in African-Americans.

### Objective

Large-scale genetic association studies have identified over 20 rheumatoid arthritis (RA) risk alleles among individuals of European ancestry. The influence of these risk alleles has not been comprehensively studied in African-Americans. We therefore sought to examine whether these validated RA risk alleles are associated with RA in an African-American population.

### Methods

27 candidate SNPs were genotyped in 556 autoantibody-positive African-Americans with RA and 791 healthy African-American controls. Odds ratios (OR) and 95% confidence intervals (CI) for each SNP were compared to previously published ORs of RA patients of European ancestry. We then calculated a composite Genetic Risk Score (GRS) for each individual based on the sum of all risk alleles.

### Results

There was overlap in the OR and 95% CI between the European and African-American populations in 24 of the 27 candidate SNPs. Conversely, 3 of the 27 SNPs (*CCR6* rs3093023, *TAGAP* rs394581, *TNFAIP3* rs6920220) demonstrated an OR in the opposite direction from those reported in RA patients of European ancestry. The GRS analysis indicated a small but highly significant probability that African-American cases were enriched for the European RA risk alleles relative to controls (p=0.00005).

### Conclusion

The majority of RA risk alleles previously validated among European ancestry RA patients showed similar ORs in our population of African-Americans with RA. Furthermore, the aggregate GRS supports the hypothesis that these SNPs are risk alleles for RA in the African-American population. Future large-scale genetic studies are needed to validate these risk alleles and identify novel risk alleles for RA in African-Americans.

## PATIENTS AND METHODS

### Study subjects

We analyzed 27 SNPs in 556 African-Americans with autoantibody-positive RA, defined as a positive serum rheumatoid factor (RF) or a positive serum anti-CCP antibody. The analysis was limited to autoantibody-positive subjects because the risk alleles tested here were those previously validated in autoantibody-positive patients of European ancestry ([2](#R2),[5](#R5),[6](#R6),[7](#R7)). All RA subjects were participants in the Consortium for the Longitudinal Evaluation of African-Americans with Early Rheumatoid Arthritis (CLEAR) Registry. The CLEAR Registry enrolls self-identified African-Americans who meet the American College of Rheumatology (ACR) 1987 diagnostic criteria for RA ([8](#R8)). CLEAR participants were recruited from the University of Alabama at Birmingham (UAB) (coordinating site); Emory University / Grady Hospital (Atlanta, GA); University of North Carolina at Chapel Hill; Medical University of South Carolina (Charleston, SC); and Washington University (St. Louis, MO). There are two phases of the CLEAR Registry. The longitudinal arm (CLEAR I) enrolled patients with early RA (<2 year disease duration) from 2000 to 2007 and follows them longitudinally until 5 years disease duration. CLEAR II started in 2007 and is an ongoing one-time evaluation of patients with any disease duration. In this study, we analyzed only the autoantibody-positive participants: 228 RA patients from CLEAR I and 328 RA patients from CLEAR II. Detailed demographic and clinical data and DNA samples are available on all CLEAR participants (see ([http://medicine.uab.edu/rheum/70918/](http://medicine.uab.edu/rheum/70918/) for details).

Healthy African-American controls with similar sex, age, and geographic location were recruited through the CLEAR study. This included 132 controls from the CLEAR I study and 171 controls from the CLEAR II (total = 303). The remaining 501 African-American controls were recruited from the Birmingham, Alabama area. All participants were recruited with informed consent under the approval of each respective Institutional Review Board. Anti-CCP antibodies and RF were determined using methods as previously described ([9](#R9)).

### Genotyping

SNP selection was based on previously identified and validated risk alleles in autoantibody-positive RA subjects of European ancestry ([5](#R5),[6](#R6),[7](#R7),[10](#R10)–[22](#R22)), as well as those identified in a recent GWAS meta-analysis by Stahl et al., which included over 5,000 autoantibody-positive European RA cases and 20,000 controls ([Table 2](#T2)) ([7](#R7)).

### Table 2. Comparison of ORs for Risk Alleles in the European (7) and African-American Populations.

| SNP | Chr | Gene | Min/MajAllele* | MAF(cases) | MAF(controls) | P value | African-AmericanOR | European OR | MAFEuropeanCases | StatisticalPower |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| rs3087243 | 2 | CTLA4 | A/G | 0.19 | 0.24 | 0.003 | 0.75 (0.62–0.91) | 0.87 (0.83–0.91) | 0.44 | 0.340 |
| rs11889341** | 2 | STAT4 | T/C | 0.15 | 0.13 | 0.071 | 1.22 (0.98–1.53) | 1.16 (1.10–1.23) | 0.22 | 0.231 |
| rs1980422 | 2 | CD28 | C/T | 0.22 | 0.19 | 0.073 | 1.19 (0.98–1.44) | 1.12 (1.06–1.18) | 0.24 | 0.216 |
| rs3761847 | 9 | TRAF1-C5 | A/G | 0.41 | 0.38 | 0.128 | 1.13 (0.97–1.32) | 1.13 (1.08–1.18) | 0.43 | 0.326 |
| rs6859219 | 5 | ANKRD55 | T/C | 0.03 | 0.04 | 0.144 | 0.73 (0.48–1.12) | 0.85 (0.78–0.93) | 0.18 | 0.139 |
| rs10499194† | 6 | TNFAIP3 | T/C | 0.18 | 0.16 | 0.164 | 1.16 (0.94–1.42) | 0.91 (0.87–0.96) | 0.27 | 0.148 |
| rs13031237 | 2 | REL | T/G | 0.10 | 0.08 | 0.185 | 1.20 (0.92–1.56) | 1.13 (1.07–1.18) | 0.37 | 0.146 |
| rs934734 | 2 | SPRED2 | G/A | 0.46 | 0.49 | 0.231 | 1.10 (0.94–1.28) | 1.13 (1.06–1.21) | 0.52 | 0.346 |
| rs2812378 | 9 | CCL21 | G/A | 0.45 | 0.42 | 0.232 | 1.10 (0.94–1.28) | 1.10 (1.05–1.16) | 0.34 | 0.228 |
| rs3890745 | 1 | TNFRSF14 | C/T*** | 0.52 | 0.54 | 0.270 | 0.92 (0.79–1.07) | 0.89 (0.85–0.94) | 0.32 | 0.318 |
| rs11586238† | 1 | CD58 | G/C | 0.09 | 0.10 | 0.281 | 0.87 (0.67–1.13) | 1.13 (1.07–1.19) | 0.24 | 0.164 |
| rs10919563† | 1 | PTPRC | A/G | 0.40 | 0.38 | 0.342 | 1.08 (0.92–1.26) | 0.88 (0.82–0.94) | 0.13 | 0.362 |
| rs548234 | 6 | PRDM1 | C/T | 0.08 | 0.07 | 0.436 | 1.12 (0.84–1.49) | 1.10 (1.05–1.16) | 0.33 | 0.100 |
| rs2476601 | 1 | PTPN22 | A/G | 0.02 | 0.01 | 0.507 | 1.23 (0.66–2.29) | 1.94 (1.81–2.08) | 0.10 | 0.665 |
| rs10488631 | 7 | IRF5 | T/C | 0.03 | 0.03 | 0.530 | 1.16 (0.72–1.87) | 1.25 (1.14–1.37) | 0.13 | 0.156 |
| rs3218253 | 22 | IL2RB | A/G | 0.15 | 0.13 | 0.575 | 1.07 (0.85–1.33) | 1.09 (1.03–1.15) | 0.26 | 0.121 |
| rs706778 | 10 | IL2RA | C/T | 0.50 | 0.49 | 0.656 | 1.04 (0.89–1.21) | 1.11 (1.06–1.17) | 0.44 | 0.268 |
| rs26232 | 5 | C5orf13, | T/C | 0.29 | 0.30 | 0.687 | 0.97 (0.82–1.14) | 0.93 (0.88–0.98) | 0.29 | 0.138 |
| rs540386 | 11 | TRAF6 | T/C | 0.25 | 0.25 | 0.756 | 0.97 (0.81–1.16) | 0.88 (0.83–0.94) | 0.14 | 0.307 |
| rs874040 | 4 | RBPJ | C/G | 0.34 | 0.34 | 0.771 | 1.02 (0.87–1.21) | 1.18 (1.12–1.24) | 0.33 | 0.530 |
| rs13315591† | 3 | PXK | C/T | 0.34 | 0.35 | 0.799 | 0.98 (0.83–1.15) | 1.13 (1.04–1.23) | 0.10 | 0.326 |
| rs4750316† | 10 | PRKCQ | C/G | 0.39 | 0.38 | 0.813 | 1.02 (0.87–1.19) | 0.87 (0.82–0.92) | 0.19 | 0.417 |
| rs6822844† | 4 | IL2, IL21 | T/G | 0.02 | 0.02 | 0.829 | 1.06 (0.63–1.77) | 0.90 (0.84–0.95) | 0.18 | 0.069 |
| rs4810485† | 20 | CD40 | T/G | 0.07 | 0.07 | 0.908 | 1.02 (0.75–1.38) | 0.85 (0.80–0.90) | 0.25 | 0.196 |
| A |  |  |  |  |  |  |  |  |  |  |
| SNP | Chr | Gene | Min/MajAllele | MAF(cases) | MAF(controls) | Pvalue | African-AmericanOR | European OR | MAFEuropeanCases | StatisticalPower |
| rs3093023 | 6 | CCR6 | A/G | 0.14 | 0.17 | 0.035 | 0.79 (0.64–0.98) | 1.11 (1.06–1.16) | 0.47 | 0.175 |
| rs394581 | 6 | TAGAP | C/T | 0.47 | 0.43 | 0.080 | 1.15 (0.98–1.34) | 0.91 (0.87–0.96) | 0.30 | 0.225 |
| rs6920220 | 6 | TNFAIP3 | A/G | 0.10 | 0.11 | 0.321 | 0.88 (0.68–1.13) | 1.22 (1.16–1.29) | 0.22 | 0.383 |
| B |  |  |  |  |  |  |  |  |  |  |

Table 2 Caption: A. Odds Ratios for 24 risk alleles (SNPs) for which ORs were consistent (overlapping confidence intervals) between African-American and European populations. B. Odds Ratios for 3 risk alleles (SNPs) for which ORs were inconsistent between African-American and European populations.

SNP genotyping was performed at the Broad Institute using Sequenom iPlex, as previously described ([15](#R15)). For quality control we required that each SNP pass the following criteria: (1) genotype missing rate < 10%; (2) minor allele frequency > 1%; and (3) Hardy-Weinberg equilibrium with p>0.001. We then excluded individuals with data missing for > 10% of SNPs passing quality control.

### Statistical Analysis

We performed SNP associations with RA risk using the software package Plink v1.06 ([http://pngu.mgh.harvard.edu/purcell/plink](http://pngu.mgh.harvard.edu/purcell/plink)) ([23](#R23)). ORs reflected the differences between cases and controls of minor allele frequencies in the African-American population, and they were tested with Pearson’s Chi square. The analysis included 556 autoantibody-positive RA cases and 791 controls after filtering was done for quality control. We did not perform a Bonferroni correction because each of the SNPs tested is a validated risk allele for RA in the European population. Had we used a Bonferroni correction for multiple comparisons, we would not have declared statistical significance unless the p-value was less than the nominal alpha level of 0.002 (0.05 divided by 27).

We compared ORs in European ancestry RA to those in our African-American RA population. The OR and 95% confidence intervals from European subjects were derived from a meta-analysis of 5,539 autoantibody positive RA cases and 20,169 controls of European descent ([7](#R7)). Differences in ORs were declared significant between the European and African-American populations if the OR 95% confidence limits did not overlap. The statistical power to detect a significant association of SNPs with RA in African-Americans was calculated using the Genetic Power Calculator ([24](#R24)). These power calculations were based on: the ORs for association with RA in the European population; the sample size of African-Americans; and the minor allele frequencies in the African-American population.

As a final step, we derived an aggregate genetic risk score (GRS) in our population of autoantibody positive African-American RA and controls ([4](#R4)). For each individual the sum (or count) of the number of risk alleles across the 27 SNPs was calculated (possible range of values from 0 to 54). We chose a GRS count approach rather than a weighted GRS (where each SNP in the GRS is weighted by the OR), as we did not want to assume that the OR in African-Americans was the same as the OR in individuals of European ancestry. The total number of risk vs. non-risk alleles between cases and controls was then assessed using contingency table analysis, and the hypothesis that the type of allele was independent of case-control status was assessed with Pearson's chi square statistic. Equivalently, the GRS score (probability that the individual had risk alleles) was modeled as a binomial response and case-control status as a dichotomous predictor using generalized linear models. Both approaches were implemented in the statistical package R ([25](#R25)).

## RESULTS

The characteristics of the RA participants in CLEAR I and CLEAR II are shown in [Table 1](#T1). Demographic characteristics were similar with the exception of the mean age of RA onset and disease duration, which were significantly higher in CLEAR II than in CLEAR I because of differences in inclusion criteria for the two CLEAR arms.

### Table 1. Characteristics of 556 autoantibody RA patient participants in CLEAR I (longitudinal evaluation of early RA) and CLEAR II (cross-sectional evaluation of established RA).

|   | CLEAR I (n=228) | CLEAR II (n=328) | P value |
| --- | --- | --- | --- |
| Age at RA onset, mean (SD) | 48.8 (13.4) | 43.4 (14.0) | <0.0001 |
| Disease duration at enrollment, months (SD) | 12.8 (7.3) | 133.8 (112.7) | <0.0001 |
| Gender (female), % | 84.2 | 83.5 (N = 324) | 0.83 |
| Anti-CCP antibodies, % positive | 85.1 | 87.1 (N = 271) | 0.52 |
| Rheumatoid factor, % positive | 95.7 (N = 211) | 96.6 (N = 322) | 0.62 |
| Both CCP and RF positive (%) | 80.6 (N = 211) | 82.6 (N = 265) | 0.56 |
| HLA SE positive (%) | 45.9 (N = 218) | 39.5 (N = 258) | 0.16 |
| Smoker (current) (%) | 29.8 | 25.3 | 0.24 |
| Smoker (ever) (%) | 52.2 | 54.6 | 0.58 |

Table 1 Caption: Significant differences in demographic parameters between study groups were tested with T-tests for independent samples (Age at RA, Disease duration) and normal approximations for testing differences between large sample proportions (remaining variables). P-values for these tests are listed in the right column. Where data was unavailable for some of the subjects, sample sizes showing the numbers of subjects available for analysis are shown within the table

### Association between genetic risk alleles and RA in African-Americans

The allele frequencies of each of the candidate SNPs tested for association with RA risk in our African-American cohort are shown in [Table 2](#T2). Notably, some RA risk alleles were present at a substantially lower frequency in African-American than European populations (e.g., *ANKRD55*, *PTPN22*, *IRF5*, *IL2-IL21*). We have previously reported allele frequencies and ORs of 5 SNPs in the *STAT4* region (rs11889341; rs10931481; rs7574865; rs8179673; and rs10181656) in 723 of African-American RA patients (including the 556 autoantibody-positive patients reported here, plus autoantibody-negative patients) and 660 of these same controls ([26](#R26)). The MAFs for African-American RA patients and controls for the *STAT4* rs11889341 T allele were 0.141 and 0.133, respectively using ABI TaqMan assays. These values are consistent with the results obtained on the Sequenom platform used in the current study.

We compared the ORs and 95% CI in our African-American population to those in a recently published meta-analysis of European ancestry RA patients ([7](#R7)). Of the 27 SNPs tested, 24 demonstrated OR and 95% CI that overlap between the European and African-American populations ([Table 2A](#T2), [Figure 1](#F1)). Only 1 of the 24 SNPs, *CTLA4* rs3087243, showed a statistically significant association with RA in African-Americans (OR 0.75, [95% CI 0.62–0.91] p=0.003). Seven of the 24 SNPs (*TNFAIP3* rs10499194, *CD58* rs11586238, *PTPRC* rs10919563, *PXK* rs13315591, *PRKCQ* rs4750316, *IL2,IL21* rs6822844, *CD40* 4810485) demonstrated OR in the opposite direction to that seen in European ancestry RA, but the 95% CIs overlapped between the two populations, and there was no statistically significant difference between African-American RA and controls.

### Figure 1. Scatter plot of the odds ratios of SNPs with replicated association in European populations on the ordinate and OR of the same SNPs tested in the African-American (A-A) population on the abscissa.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/334d/3030622/878d69bcfa1c/nihms232158f1.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3030622_nihms232158f1.jpg)

Solid markers indicate OR that are consistent between populations (Table 2A); open markers are those SNPs in which the OR are inconsistent between populations (Table 2B). The two triangle markers are those SNPs (rs3087243, CTLA4 and rs3093023, CCR6) with statistically significant association with RA in this African-American sample. The diamond at the top of the figure is PTPN22, which has an OR=1.94 and OR=1.23 in European ancestry RA patients and African-American RA patients, respectively.

As shown in [Table 2B](#T2) and [Figure 1](#F1), we found evidence for differences in ORs between the two ethnic groups in 3 of the 27 SNPs (*CCR6* rs3093023, *TAGAP* rs394581, and *TNFAIP3* rs6920220). Of these three SNPs, only *CCR6* rs3093023 showed a statistically significant association with RA in African-Americans (OR 0.79, [95% CI 0.64–0.98] p=0.035). All three of these risk alleles had OR that were in the opposite directions between populations.

To overcome the limitation in statistical power due to the relatively small size of our African-American RA cohort, we calculated a genetic risk score (GRS) based on all RA risk alleles for each person in the study. This approach combines the effect of each risk allele into a single aggregate score, which can then be used to test whether the GRS differentiates between RA cases and controls. The GRS analysis demonstrated that the probability of individuals having risk alleles was higher in cases (mean GRS = 0.43) than in controls (0.41) (p=0.00005, [Figure 2](#F2)). To exclude the possibility that the observed difference was due to a single SNP with a large effect size, we repeated the GRS analysis without the *CTLA4 rs3087243* SNP; the result remains significant (0.41 cases, 0.40 controls, p=0.0003). This observation supports the hypothesis that many risk alleles found in European RA patients are also found in African-Americans with RA.

### Figure 2. Cumulative distribution of the GRS scores between cases (red) and controls (blue).

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/334d/3030622/67938528eaa1/nihms232158f2.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3030622_nihms232158f2.jpg)

The y axis, P (x <= X), is the cumulative probability that individual cases or controls have less than the given GRS score. The GRS score is defined here as the individual proportion of risk alleles carried for the 27 SNPs which have validated association with RA in populations of European ancestry. The mean is 0.43 for cases and 0.41 for controls.

## Discussion

To date, the majority of GWAS and subsequent meta-analyses of GWAS data in RA have focused on subjects of European and East Asian ancestry. It has become clear from these and other large scale genetic studies of complex diseases such as RA that genetic risk loci can differ among these different ethnic groups ([18](#R18);[22](#R22);[27](#R27)). There is a paucity of data from well-characterized, large groups of African-Americans with RA. In this study we sought to test the hypothesis that RA risk alleles validated in populations of European ancestry would also be associated with RA in African-Americans. The power to test individual risk alleles in our African American sample is limited compared to studies in European RA. The results of this study, however, demonstrated that cumulatively (via the GRS analysis), risk alleles for RA in Europeans also confer risk for African Americans. Therefore, we conclude that the two populations are best characterized as genetically homogeneous with respect to validated risk alleles for RA.

We found that the ORs between European and African-American populations were consistent for 24 of 27 European RA risk loci. One interpretation of this finding is that the genetic etiology of RA risk in the two populations is very similar. However, only one European RA risk allele achieved statistically significant association with risk of RA in the African-American population (*CTLA4* rs3087243). Although the CLEAR cohort is the largest group of African-American RA patients currently available for analysis, we had limited power to detect genetic associations. This limited power, as shown in [Table 2](#T2), can be attributed to the small effect size of many individual risk alleles and the low frequency of some of the risk alleles in the African-American population. The limited power to detect association signals also affected our ability to demonstrate between population inconsistencies in OR. For example, the low frequency *PTNP22* SNP rs2476601 (MAF = 0.01) showed the largest between-population difference ([Figure 1](#F1)), with OR of 1.23 in African-Americans and 1.94 in Europeans. However, due to the large confidence interval limit on the African-American OR, we were unable conclude that these OR were different between populations ([Table 2](#T2)).

We observed differences in direction of OR between European and African American populations for three loci (*CCR6* rs3093023, *TAGAP* rs394581, and *TNFAIP3* rs6920220). There are at least three explanations for these differences. First, inconsistent ORs might be due to weakened correlations between tag SNPs and the causal allele (not yet identified) with actual risk with RA. It is known from the International Haplotype Mapping Project (HapMap) that LD structure between European and African American populations may be different for common SNPs in any given locus. Such an effect may be particularly striking if the causal allele is rare in the population (e.g., frequency <5%)([28](#R28)). The second possible explanation is that these differences may be explained by genetic heterogeneity; the European risk allele may operate differently in its effect on RA risk for African-Americans. Such population specific differences may reflect gene by gene or gene by environmental interactions. The third possible explanation is that the inconsistency may simply be due to chance, given the number of hypotheses tested. Further investigation of these alleles among individuals of African ancestry is needed in explore these possibilities.

Although we had limited statistical power because of our small sample size, when the SNP risk alleles are viewed together in the GRS analysis, there is evidence that the European risk alleles are also risk alleles in the African American population. The GRS was found to be significantly different between RA cases and controls, even after excluding the *CTLA4* SNP with the largest effect size. However this approach indicates that even though most of the individual markers were not statistically associated with RA in African-Americans, they may contribute to a panel of alleles that collectively confer risk.

Future studies of large, well-characterized cohorts of African-American RA patients and controls are needed to definitively determine whether the European RA risk alleles are associated with RA risk in African-Americans. Large scale GWAS in African-American RA patients are also needed in order to explore novel risk alleles among this genetically admixed ethnic group. We believe that detailed genetic studies of African-Americans with RA will lead to important insights into the pathogenesis of this disease.

## Acknowledgements

The authors gratefully acknowledge all the study patients for their contribution to this work. We also thank Drs. David Allison, Hemant Tiwari, Maria Danila, Monica Crawford, and Jeffrey Faggard for helpful discussions and review of the manuscript. This research was supported by NIH grants T32 HL072757. The CLEAR Registry is supported by NIH contract N01-AR-6-2278. RMP is supported by grants from NIAMS-NIH (R01-AR056768 and R01-AR057108) and holds a Career Award for Medical Scientists from the Burroughs Wellcome Fund.

## References

1. Mahdi H, Fisher BA, Kallberg H, Plant D, Malmstrom V, Ronnelid J, et al. Specific interaction between genotype, smoking and autoimmunity to citrullinated alpha-enolase in the etiology of rheumatoid arthritis. Nat Genet. 2009;41(12):1319–1324. doi: 10.1038/ng.480.  [DOI](https://doi.org/10.1038/ng.480) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19898480/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Specific%20interaction%20between%20genotype,%20smoking%20and%20autoimmunity%20to%20citrullinated%20alpha-enolase%20in%20the%20etiology%20of%20rheumatoid%20arthritis&author=H%20Mahdi&author=BA%20Fisher&author=H%20Kallberg&author=D%20Plant&author=V%20Malmstrom&volume=41&issue=12&publication_year=2009&pages=1319-1324&pmid=19898480&doi=10.1038/ng.480&)

2. Raychaudhuri S, Thomson BP, Remmers EF, Eyre S, Hinks A, Guiducci C, et al. Genetic variants at CD28, PRDM1 and CD2/CD58 are associated with rheumatoid arthritis risk. Nat Genet. 2009;41(12):1313–1318. doi: 10.1038/ng.479.  [DOI](https://doi.org/10.1038/ng.479) | [PMC free article](/articles/PMC3142887/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19898481/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Genetic%20variants%20at%20CD28,%20PRDM1%20and%20CD2/CD58%20are%20associated%20with%20rheumatoid%20arthritis%20risk&author=S%20Raychaudhuri&author=BP%20Thomson&author=EF%20Remmers&author=S%20Eyre&author=A%20Hinks&volume=41&issue=12&publication_year=2009&pages=1313-1318&pmid=19898481&doi=10.1038/ng.479&)

3. Plenge RM. Recent progress in rheumatoid arthritis genetics: one step towards improved patient care. Curr Opin Rheumatol. 2009;21(3):262–271. doi: 10.1097/BOR.0b013e32832a2e2d.  [DOI](https://doi.org/10.1097/BOR.0b013e32832a2e2d) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19365266/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Opin%20Rheumatol&title=Recent%20progress%20in%20rheumatoid%20arthritis%20genetics:%20one%20step%20towards%20improved%20patient%20care&author=RM%20Plenge&volume=21&issue=3&publication_year=2009&pages=262-271&pmid=19365266&doi=10.1097/BOR.0b013e32832a2e2d&)

4. Karlson EW, Chibnik LB, Kraj P, Cui J, Keenan BT, Ding B, et al. Cumulative Association of Twenty-Two Genetic Variants with Seropositive Rheumatoid Arthritis Risk. Ann Rheum Dis. 2010 doi: 10.1136/ard.2009.120170. In press.  [DOI](https://doi.org/10.1136/ard.2009.120170) | [PMC free article](/articles/PMC2933175/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20233754/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Rheum%20Dis&title=Cumulative%20Association%20of%20Twenty-Two%20Genetic%20Variants%20with%20Seropositive%20Rheumatoid%20Arthritis%20Risk&author=EW%20Karlson&author=LB%20Chibnik&author=P%20Kraj&author=J%20Cui&author=BT%20Keenan&publication_year=2010&pmid=20233754&doi=10.1136/ard.2009.120170&)

5. Raychaudhuri S, Remmers EF, Lee AT, Hackett R, Guiducci C, Burtt NP, et al. Common variants at CD40 and other loci confer risk of rheumatoid arthritis. Nat Genet. 2008;40(10):1216–1223. doi: 10.1038/ng.233.  [DOI](https://doi.org/10.1038/ng.233) | [PMC free article](/articles/PMC2757650/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18794853/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Common%20variants%20at%20CD40%20and%20other%20loci%20confer%20risk%20of%20rheumatoid%20arthritis&author=S%20Raychaudhuri&author=EF%20Remmers&author=AT%20Lee&author=R%20Hackett&author=C%20Guiducci&volume=40&issue=10&publication_year=2008&pages=1216-1223&pmid=18794853&doi=10.1038/ng.233&)

6. Gregersen PK, Amos CI, Lee AT, Lu Y, Remmers EF, Kastner DL, et al. REL, encoding a member of the NF-kappaB family of transcription factors, is a newly defined risk locus for rheumatoid arthritis. Nat Genet. 2009;41(7):820–823. doi: 10.1038/ng.395.  [DOI](https://doi.org/10.1038/ng.395) | [PMC free article](/articles/PMC2705058/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19503088/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=REL,%20encoding%20a%20member%20of%20the%20NF-kappaB%20family%20of%20transcription%20factors,%20is%20a%20newly%20defined%20risk%20locus%20for%20rheumatoid%20arthritis&author=PK%20Gregersen&author=CI%20Amos&author=AT%20Lee&author=Y%20Lu&author=EF%20Remmers&volume=41&issue=7&publication_year=2009&pages=820-823&pmid=19503088&doi=10.1038/ng.395&)

7. Stahl EA, Raychaudhuri S, Remmers EF, Xie G, Eyre S, Thomson B, et al. Genome-wide association study meta-analysis identifies 7 novel rheumatoid arthritis risk loci. Nat Genet. 2010;42(6):508–514. doi: 10.1038/ng.582.  [DOI](https://doi.org/10.1038/ng.582) | [PMC free article](/articles/PMC4243840/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20453842/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Genome-wide%20association%20study%20meta-analysis%20identifies%207%20novel%20rheumatoid%20arthritis%20risk%20loci&author=EA%20Stahl&author=S%20Raychaudhuri&author=EF%20Remmers&author=G%20Xie&author=S%20Eyre&volume=42&issue=6&publication_year=2010&pages=508-514&pmid=20453842&doi=10.1038/ng.582&)

8. Arnett FC, Edworthy SM, Bloch DA, McShane DJ, Fries JF, Cooper NS, et al. The American Rheumatism Association 1987 revised criteria for the classification of rheumatoid arthritis. Arthritis Rheum. 1988;31:315–324. doi: 10.1002/art.1780310302.  [DOI](https://doi.org/10.1002/art.1780310302) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/3358796/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arthritis%20Rheum&title=The%20American%20Rheumatism%20Association%201987%20revised%20criteria%20for%20the%20classification%20of%20rheumatoid%20arthritis&author=FC%20Arnett&author=SM%20Edworthy&author=DA%20Bloch&author=DJ%20McShane&author=JF%20Fries&volume=31&publication_year=1988&pages=315-324&pmid=3358796&doi=10.1002/art.1780310302&)

9. Mikuls TR, Holers VM, Parrish L, Kuhn KA, Alarcón GS, Conn DL, et al. Anti-cyclic citrullinated peptide antibody and rheumatoid factor isotypes in African Americans with early rheumatoid arthritis. Arthritis Rheum. 2006;54(9):3057–3059. doi: 10.1002/art.22200.  [DOI](https://doi.org/10.1002/art.22200) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16948136/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arthritis%20Rheum&title=Anti-cyclic%20citrullinated%20peptide%20antibody%20and%20rheumatoid%20factor%20isotypes%20in%20African%20Americans%20with%20early%20rheumatoid%20arthritis&author=TR%20Mikuls&author=VM%20Holers&author=L%20Parrish&author=KA%20Kuhn&author=GS%20Alarc%C3%B3n&volume=54&issue=9&publication_year=2006&pages=3057-3059&pmid=16948136&doi=10.1002/art.22200&)

10. Barton A, Thomson W, Ke X, Eyre S, Hinks A, Bowes J, et al. Rheumatoid arthritis susceptibility loci at chromosomes 10p15, 12q13 and 22q13. Nat Genet. 2008;40(10):1156–1159. doi: 10.1038/ng.218.  [DOI](https://doi.org/10.1038/ng.218) | [PMC free article](/articles/PMC2662493/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18794857/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Rheumatoid%20arthritis%20susceptibility%20loci%20at%20chromosomes%2010p15,%2012q13%20and%2022q13&author=A%20Barton&author=W%20Thomson&author=X%20Ke&author=S%20Eyre&author=A%20Hinks&volume=40&issue=10&publication_year=2008&pages=1156-1159&pmid=18794857&doi=10.1038/ng.218&)

11. Barton A, Thomson W, Ke X, Eyre S, Hinks A, Bowes J, et al. Re-evaluation of putative rheumatoid arthritis susceptibility genes in the post-genome wide association study era and hypothesis of a key pathway underlying susceptibility. Hum Mol Genet. 2008;17(15):2274–2279. doi: 10.1093/hmg/ddn128.  [DOI](https://doi.org/10.1093/hmg/ddn128) | [PMC free article](/articles/PMC2465799/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18434327/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mol%20Genet&title=Re-evaluation%20of%20putative%20rheumatoid%20arthritis%20susceptibility%20genes%20in%20the%20post-genome%20wide%20association%20study%20era%20and%20hypothesis%20of%20a%20key%20pathway%20underlying%20susceptibility&author=A%20Barton&author=W%20Thomson&author=X%20Ke&author=S%20Eyre&author=A%20Hinks&volume=17&issue=15&publication_year=2008&pages=2274-2279&pmid=18434327&doi=10.1093/hmg/ddn128&)

12. Begovich AB, Carlton VE, Honigberg LA, Schrodi SJ, Chokkalingam AP, Alexander HC, et al. A missense single-nucleotide polymorphism in a gene encoding a protein tyrosine phosphatase (PTPN22) is associated with rheumatoid arthritis. Am J Hum Genet. 2004;75(2):330–337. doi: 10.1086/422827.  [DOI](https://doi.org/10.1086/422827) | [PMC free article](/articles/PMC1216068/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15208781/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=A%20missense%20single-nucleotide%20polymorphism%20in%20a%20gene%20encoding%20a%20protein%20tyrosine%20phosphatase%20(PTPN22)%20is%20associated%20with%20rheumatoid%20arthritis&author=AB%20Begovich&author=VE%20Carlton&author=LA%20Honigberg&author=SJ%20Schrodi&author=AP%20Chokkalingam&volume=75&issue=2&publication_year=2004&pages=330-337&pmid=15208781&doi=10.1086/422827&)

13. Chang M, Rowland CM, Garcia VE, Schrodi SJ, Catanese JJ, van der Helm-van Mil AH, et al. A large-scale rheumatoid arthritis genetic study identifies association at chromosome 9q33.2. PLoS Genet. 2008;4(6):e1000107. doi: 10.1371/journal.pgen.1000107.  [DOI](https://doi.org/10.1371/journal.pgen.1000107) | [PMC free article](/articles/PMC2481282/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18648537/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet&title=A%20large-scale%20rheumatoid%20arthritis%20genetic%20study%20identifies%20association%20at%20chromosome%209q33.2&author=M%20Chang&author=CM%20Rowland&author=VE%20Garcia&author=SJ%20Schrodi&author=JJ%20Catanese&volume=4&issue=6&publication_year=2008&pages=e1000107&pmid=18648537&doi=10.1371/journal.pgen.1000107&)

14. Kurreeman FA, Padyukov L, Marques RB, Schrodi SJ, Seddighzadeh M, Stoeken-Rijsbergen G, et al. A candidate gene approach identifies the TRAF1/C5 region as a risk factor for rheumatoid arthritis. PLoS Med. 2007;4(9):e278. doi: 10.1371/journal.pmed.0040278.  [DOI](https://doi.org/10.1371/journal.pmed.0040278) | [PMC free article](/articles/PMC1976626/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17880261/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Med&title=A%20candidate%20gene%20approach%20identifies%20the%20TRAF1/C5%20region%20as%20a%20risk%20factor%20for%20rheumatoid%20arthritis&author=FA%20Kurreeman&author=L%20Padyukov&author=RB%20Marques&author=SJ%20Schrodi&author=M%20Seddighzadeh&volume=4&issue=9&publication_year=2007&pages=e278&pmid=17880261&doi=10.1371/journal.pmed.0040278&)

15. Plenge RM, Cotsapas C, Davies L, Price AL, de Bakker PI, Maller J, et al. Two independent alleles at 6q23 associated with risk of rheumatoid arthritis. Nat Genet. 2007;39(12):1477–1482. doi: 10.1038/ng.2007.27.  [DOI](https://doi.org/10.1038/ng.2007.27) | [PMC free article](/articles/PMC2652744/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17982456/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Two%20independent%20alleles%20at%206q23%20associated%20with%20risk%20of%20rheumatoid%20arthritis&author=RM%20Plenge&author=C%20Cotsapas&author=L%20Davies&author=AL%20Price&author=PI%20de%20Bakker&volume=39&issue=12&publication_year=2007&pages=1477-1482&pmid=17982456&doi=10.1038/ng.2007.27&)

16. Plenge RM, Seielstad M, Padyukov L, Lee AT, Remmers EF, Ding B, et al. TRAF1-C5 as a risk locus for rheumatoid arthritis--a genomewide study. N Engl J Med. 2007;357(12):1199–1209. doi: 10.1056/NEJMoa073491.  [DOI](https://doi.org/10.1056/NEJMoa073491) | [PMC free article](/articles/PMC2636867/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17804836/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=TRAF1-C5%20as%20a%20risk%20locus%20for%20rheumatoid%20arthritis--a%20genomewide%20study&author=RM%20Plenge&author=M%20Seielstad&author=L%20Padyukov&author=AT%20Lee&author=EF%20Remmers&volume=357&issue=12&publication_year=2007&pages=1199-1209&pmid=17804836&doi=10.1056/NEJMoa073491&)

17. Remmers EF, Plenge RM, Lee AT, Graham RR, Hom G, Behrens TW, et al. STAT4 and the risk of rheumatoid arthritis and systemic lupus erythematosus. N Engl J Med. 2007;357(10):977–986. doi: 10.1056/NEJMoa073003.  [DOI](https://doi.org/10.1056/NEJMoa073003) | [PMC free article](/articles/PMC2630215/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17804842/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=STAT4%20and%20the%20risk%20of%20rheumatoid%20arthritis%20and%20systemic%20lupus%20erythematosus&author=EF%20Remmers&author=RM%20Plenge&author=AT%20Lee&author=RR%20Graham&author=G%20Hom&volume=357&issue=10&publication_year=2007&pages=977-986&pmid=17804842&doi=10.1056/NEJMoa073003&)

18. Suzuki A, Yamada R, Chang X, Tokuhiro S, Sawada T, Suzuki M, et al. Functional haplotypes of PADI4, encoding citrullinating enzyme peptidylarginine deiminase 4, are associated with rheumatoid arthritis. Nat Genet. 2003;34(4):395–402. doi: 10.1038/ng1206.  [DOI](https://doi.org/10.1038/ng1206) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12833157/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Functional%20haplotypes%20of%20PADI4,%20encoding%20citrullinating%20enzyme%20peptidylarginine%20deiminase%204,%20are%20associated%20with%20rheumatoid%20arthritis&author=A%20Suzuki&author=R%20Yamada&author=X%20Chang&author=S%20Tokuhiro&author=T%20Sawada&volume=34&issue=4&publication_year=2003&pages=395-402&pmid=12833157&doi=10.1038/ng1206&)

19. Thomson W, Barton A, Ke X, Eyre S, Hinks A, Bowes J, et al. Rheumatoid arthritis association at 6q23. Nat Genet. 2007;39(12):1431–1433. doi: 10.1038/ng.2007.32.  [DOI](https://doi.org/10.1038/ng.2007.32) | [PMC free article](/articles/PMC2674282/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17982455/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Rheumatoid%20arthritis%20association%20at%206q23&author=W%20Thomson&author=A%20Barton&author=X%20Ke&author=S%20Eyre&author=A%20Hinks&volume=39&issue=12&publication_year=2007&pages=1431-1433&pmid=17982455&doi=10.1038/ng.2007.32&)

20. Plenge RM, Padyukov L, Remmers EF, Purcell S, Lee AT, Karlson EW, et al. Replication of putative candidate-gene associations with rheumatoid arthritis in >4,000 samples from North America and Sweden: association of susceptibility with PTPN22, CTLA4, and PADI4. Am J Hum Genet. 2005;77(6):1044–1060. doi: 10.1086/498651.  [DOI](https://doi.org/10.1086/498651) | [PMC free article](/articles/PMC1285162/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16380915/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=Replication%20of%20putative%20candidate-gene%20associations%20with%20rheumatoid%20arthritis%20in%20>4,000%20samples%20from%20North%20America%20and%20Sweden:%20association%20of%20susceptibility%20with%20PTPN22,%20CTLA4,%20and%20PADI4&author=RM%20Plenge&author=L%20Padyukov&author=EF%20Remmers&author=S%20Purcell&author=AT%20Lee&volume=77&issue=6&publication_year=2005&pages=1044-1060&pmid=16380915&doi=10.1086/498651&)

21. Zhernakova A, Alizadeh BZ, Bevova M, van Leeuwen MA, Coenen MJ, Franke B, et al. Novel association in chromosome 4q27 region with rheumatoid arthritis and confirmation of type 1 diabetes point to a general risk locus for autoimmune diseases. Am J Hum Genet. 2007;81(6):1284–1288. doi: 10.1086/522037.  [DOI](https://doi.org/10.1086/522037) | [PMC free article](/articles/PMC2276343/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17999365/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=Novel%20association%20in%20chromosome%204q27%20region%20with%20rheumatoid%20arthritis%20and%20confirmation%20of%20type%201%20diabetes%20point%20to%20a%20general%20risk%20locus%20for%20autoimmune%20diseases&author=A%20Zhernakova&author=BZ%20Alizadeh&author=M%20Bevova&author=MA%20van%20Leeuwen&author=MJ%20Coenen&volume=81&issue=6&publication_year=2007&pages=1284-1288&pmid=17999365&doi=10.1086/522037&)

22. Suzuki A, Yamada R, Kochi Y, Sawada T, Okada Y, Matsuda K, et al. Functional SNPs in CD244 increase the risk of rheumatoid arthritis in a Japanese population. Nat Genet. 2008;40(10):1224–1229. doi: 10.1038/ng.205.  [DOI](https://doi.org/10.1038/ng.205) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18794858/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Functional%20SNPs%20in%20CD244%20increase%20the%20risk%20of%20rheumatoid%20arthritis%20in%20a%20Japanese%20population&author=A%20Suzuki&author=R%20Yamada&author=Y%20Kochi&author=T%20Sawada&author=Y%20Okada&volume=40&issue=10&publication_year=2008&pages=1224-1229&pmid=18794858&doi=10.1038/ng.205&)

23. Purcell S, Neale B, Todd-Brown K, Thomas L, Ferreira MA, Bender D, et al. PLINK: a tool set for whole-genome association and population-based linkage analyses. Am J Hum Genet. 2007;81(3):559–575. doi: 10.1086/519795.  [DOI](https://doi.org/10.1086/519795) | [PMC free article](/articles/PMC1950838/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17701901/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=PLINK:%20a%20tool%20set%20for%20whole-genome%20association%20and%20population-based%20linkage%20analyses&author=S%20Purcell&author=B%20Neale&author=K%20Todd-Brown&author=L%20Thomas&author=MA%20Ferreira&volume=81&issue=3&publication_year=2007&pages=559-575&pmid=17701901&doi=10.1086/519795&)

24. Purcell S, Cherny SS, Sham PC. Genetic Power Calculator: design of linkage and association genetic mapping studies of complex traits. Bioinformatics. 2003;19(1):149–150. doi: 10.1093/bioinformatics/19.1.149.  [DOI](https://doi.org/10.1093/bioinformatics/19.1.149) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12499305/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=Genetic%20Power%20Calculator:%20design%20of%20linkage%20and%20association%20genetic%20mapping%20studies%20of%20complex%20traits&author=S%20Purcell&author=SS%20Cherny&author=PC%20Sham&volume=19&issue=1&publication_year=2003&pages=149-150&pmid=12499305&doi=10.1093/bioinformatics/19.1.149&)

25. R Development Core Team. R: A language and environment for statistical computing. Vienna, Austria: R Foundation for Statistical Computing; 2008. Ref Type: Online Source.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=R:%20A%20language%20and%20environment%20for%20statistical%20computing&publication_year=2008&)

26. Kelley JM, Hughes LB, Malik A, Danila MI, Edberg Y, Alarcón GS, et al. Genetic variants of STAT4 associated with rheumatoid arthritis in persons of Asian and European ancestry do not replicate in African-Americans. Ann Rheum Dis. 2010 doi: 10.1136/ard.2009.113183. In press.  [DOI](https://doi.org/10.1136/ard.2009.113183) | [PMC free article](/articles/PMC3133745/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20237121/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Rheum%20Dis&title=Genetic%20variants%20of%20STAT4%20associated%20with%20rheumatoid%20arthritis%20in%20persons%20of%20Asian%20and%20European%20ancestry%20do%20not%20replicate%20in%20African-Americans&author=JM%20Kelley&author=LB%20Hughes&author=A%20Malik&author=MI%20Danila&author=Y%20Edberg&publication_year=2010&pmid=20237121&doi=10.1136/ard.2009.113183&)

27. Lee HS, Korman BD, Le JM, Kastner DL, Remmers EF, Gregersen PK, et al. Genetic risk factors for rheumatoid arthritis differ in Caucasian and Korean populations. Arthritis Rheum. 2009;60(2):364–371. doi: 10.1002/art.24245.  [DOI](https://doi.org/10.1002/art.24245) | [PMC free article](/articles/PMC2770844/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19180477/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arthritis%20Rheum&title=Genetic%20risk%20factors%20for%20rheumatoid%20arthritis%20differ%20in%20Caucasian%20and%20Korean%20populations&author=HS%20Lee&author=BD%20Korman&author=JM%20Le&author=DL%20Kastner&author=EF%20Remmers&volume=60&issue=2&publication_year=2009&pages=364-371&pmid=19180477&doi=10.1002/art.24245&)

28. Dickson SP, Wang K, Krantz I, Hakonarson H, Goldstein DB. Rare variants create synthetic genome- wide associations. PLoS Biol. 2010;8(1):e1000294. doi: 10.1371/journal.pbio.1000294.  [DOI](https://doi.org/10.1371/journal.pbio.1000294) | [PMC free article](/articles/PMC2811148/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20126254/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Biol&title=Rare%20variants%20create%20synthetic%20genome-%20wide%20associations&author=SP%20Dickson&author=K%20Wang&author=I%20Krantz&author=H%20Hakonarson&author=DB%20Goldstein&volume=8&issue=1&publication_year=2010&pages=e1000294&pmid=20126254&doi=10.1371/journal.pbio.1000294&)
