# Glycine and a Glycine Dehydrogenase (GLDC) SNP as Citalopram/Escitalopram Response Biomarkers in Depression: Pharmacometabolomics-informed Pharmacogenomics

## Metadata
**Authors:** Yuan Ji, Scott Hebbring, Hongjie Zhu, Gregory D Jenkins, Joanna Biernacka, Karen Snyder, Maureen Drews, Oliver Fiehn, Zhaobang Zeng, Daniel Schaid, David A Mrazek, Rima Kaddurah-Daouk, Richard M Weinshilboum
**Journal:** Clinical pharmacology and therapeutics
**Date:** 2010 Nov 24
**DOI:** [10.1038/clpt.2010.250](https://doi.org/10.1038/clpt.2010.250)
**PMID:** 21107318
**PMCID:** PMC3034442
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3034442/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC3034442/pdf/nihms264812.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC3034442/pdf/nihms264812.pdf)

## Abstract

Major depressive disorder (MDD) is a common psychiatric disease. Selective serotonin reuptake inhibitors (SSRIs) are an important class of drugs used to treat MDD. However, many patients do not respond adequately to SSRI therapy. We used a pharmacometabolomics-informed pharmacogenomic research strategy to identify citalopram/escitalopram treatment outcome biomarkers. Metabolomic assay of plasma samples from 20 escitalopram remitters and 20 non-remitters showed that glycine was negatively associated with treatment outcome (p=0.0054). That observation was pursued by genotyping tag single nucleotide polymorphisms (SNPs) for genes encoding glycine synthesis and degradation enzymes using 529 DNA samples from SSRI-treated MDD patients. The rs10975641 SNP in the glycine dehydrogenase gene was associated with treatment outcome phenotypes. Rs10975641 was then genotyped and was significant (p=0.02) in DNA from 1245 MDD patients in the STAR*D depression study. These results highlight both a possible role for glycine in SSRI response and the use of pharmacometabolomics to “inform” pharmacogenomics.

Keywords: Selective serotonin reuptake inhibitors, SSRIs, major depressive disorder, MDD, pharmacometabolomics, metabolomics, pharmacogenomics, glycine, glycine dehydrogenase, GLDC, escitalopram, citalopram

## RESULTS

### Pharmacometabolomic profiling of MDD patients treated with SSRIs

MDD patients enrolled in the Mayo Clinic-NIH Pharmacogenetics Research Network (PGRN) Citalopram/Escitalopram Pharmacogenomic (Mayo-PGRN SSRI) study were selected for metabolomic profiling based on their remission status at the week 8 clinic visit. Plasma samples had been collected for this study at time zero and at weeks 4 and 8. For the current study, baseline metabolomic profiles were assayed for 20 SSRI “remitters” and 20 “non-remitters”. These patients were receiving 10 mg (N=16) or 20 mg (N=24) of the SSRI escitalopram daily. Thirty percent of the subjects were male. The average entry QIDS-C scores for “remitters” and “non-remitters” were not significantly different (p>0.05), but after 8 weeks of SSRI therapy, the average QIDS-C score was 3.1 ±1.7 for “remitters” and 9.3 ± 2.9 for “non-remitters” (p<0.01). There were no significant differences between the 40 subjects selected for metabolomic profiling and the remaining Mayo-PGRN SSRI study subjects for various demographic and clinical factors including age, race, gender, initial QIDS-C score, or final QIDS-C score after 8 weeks of treatment, except that all the profiled patients completed 8 weeks of treatment, while 78% of the remaining subjects in the study did.

A gas chromatography (GC)-MS metabolomics platform was used to profile these 40 plasma samples. This GC-MS platform measured 251 metabolites, 97 of which were chemically identified. The metabolites measured were matched against a laboratory-constructed library of mass spectra and retention index for 1,200 metabolites, as well as data from the NIST05 commercial library.

### Metabolomic statistical and pathway analysis

A quantitative phenotype, percentage change in QIDS-C, was used as the initial SSRI treatment outcome phenotype for statistical correlation and pathway analysis. The average decrease in QIDS-C for remitters was 77.9 ± 11.8% and 36.9 ± 21.8% (mean ± SD) for non-remitters. Gender, age, and ethnicity were not significantly associated with treatment outcome. Using QQ normal and histogram plots, we examined metabolite distributions and observed significant right skewness. As a result, rank-based tests and log-transformation of observed metabolite concentrations were used in the analysis.

Marginal correlation analyses of single metabolites suggested that several metabolites assayed at zero time were significantly associated with subsequent percentage change in QIDS-C after SSRI therapy. Since those associations did not necessarily point to specific metabolic pathways, we used a pathway-based regression analysis to identify significant pathways ([Figure 1B](#F1)). Pathway-specific regression analysis identified “nitrogen metabolism” as the most significant pathway, a pathway with several metabolites that were significantly associated with SSRI therapy outcome ([Table 1](#T1)). Among the six metabolites that we mapped to that pathway, five were jointly and significantly associated with drug response (regression model adjusted R²=0.55). A positive regression coefficient in [Table 1](#T1) indicates that a lower concentration of the metabolite was associated with better response, i.e. a larger reduction in QIDS-C score after SSRI therapy.

### Table 1.

| Name of Compounds | Simple Regression (p-value) | Partial Regression (p-value) |
| --- | --- | --- |
| Glycine | 2.71×10−5 (0.0054) | 3.68×10−5 (<0.0001) |
|   |  |  |
| Glutamic acid | 2.44×10−4 (0.22) | 4.54×10−4 (0.0088) |
| Aspartic acid | −2.29×10−3 (0.069) | −2.41×10−3 (0.029) |
| Asparagine | −9.46×10−5 (0.70) | −6.04×10−4 (0.0057) |
| Hydroxylamine | −1.33×10−4 (0.0022) | −1.02×10−4 (0.0035) |

Table 1 Caption: Simple and partial regression coefficients for compounds in the “ nitrogen metabolism pathway” on changes in QIDS-C score after escitalopram therapy.

Since glycine was the most significant metabolite in the nitrogen metabolism pathway, we also compared baseline glycine levels between responders and non-responders as well as “remitters” and “non-remitters” ([Figure 2](#F2)). There were significant differences in baseline glycine levels between “responders” and “non-responders” (p=0.005), with a marginally significant difference for the remission phenotype (p=0.058). This trend for association of plasma glycine levels with drug treatment outcome phenotypes was consistent with the results of the pathway analysis. The purpose of this preliminary pharmacometabolomic study was to identify signals that might then be pursued using pharmacogenomic techniques. Since glycine is an inhibitory neurotransmitter – as well as a key metabolite in the Folate Cycle – we focused on glycine as a candidate metabolite. The next step was to determine which genes and which single nucleotide polymorphisms (SNPs) should be genotyped using DNA from citalopram/escitalopram-treated MDD patients.

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f9be/3034442/a95462244ecc/nihms264812f2.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3034442_nihms264812f2.jpg)

Glycine levels at zero time in MDD patient groups classified on the basis of response or remission status after SSRI therapy. P-values were calculated with Student’s t-test.

### Glycine-related candidate genes

The biosynthesis and metabolic degradation of glycine in humans is depicted schematically in [Figure 3](#F3). Glycine is synthesized from serine in a reaction catalyzed by two serine hydroxymethyltransferase (SHMT) isoforms ([18](#R18), [19](#R19)), and it is degraded by a multi-enzyme “glycine cleavage system” ([20](#R20), [21](#R21)). This glycine cleavage system includes aminomethyltransferase, AMT; dihydrolipoamide dehydrogenase, DLD; glycine cleavage system protein H, GCSH, and glycine dehydrogenase, GLDC. As a first step in pharmacogenomic pursuit of the glycine metabolomic signal for SSRI treatment outcome, we genotyped *SHMT1*, *SHMT2*, *AMT*, *DLD*, *GCSH* and *GLDC* using a “tag SNP” approach. Specifically, 144 tag SNPs selected for these genes were genotyped using DNA from 529 subjects enrolled in the Mayo-PGRN SSRI study, the same study population from which the subjects for metabolomic profiling had been selected. We excluded 15 DNA samples from subjects with ethnicities other than White Non-Hispanic (WNH). Three SNPs failed during genotyping.

### Figure 3.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f9be/3034442/de0265f24e87/nihms264812f3.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3034442_nihms264812f3.jpg)

Glycine synthesis and degradation. The figure shows a schematic representation of glycine synthesis and degradation. SHMTs are the two serine hydroxymethyltransferase isoforms that catalyze the synthesis of glycine from serine, a reaction coupled with the conversion of tetrahydrofolate (THF) to 5,10-methylene THF. AMT, aminomethyltransferase; DLD, dihydrolipoamide dehydrogenase; GCSH, glycine cleavage system protein H and GLDC, glycine dehydrogenase, are components of the multiple-enzyme glycine cleavage system, the major pathway for glycine degradation.

Quality control (QC) analyses showed no discrepancies between genotypes from duplicate samples and no Mendelian errors in control CEPH family trio DNA samples. Four SNPs were removed from the analysis because of low MAFs (≤ 0.01) and two because of low call rates (<95%). Two DNA samples were removed due to low SNP call rates (≤ 0.95). Therefore, 135 SNPs in 512 DNA samples from NHW subjects remained for inclusion in the association analyses.

### SNP association analyses for the Mayo-PGRN SSRI Study

“Remission” is a major SSRI treatment outcome, and this phenotype was modeled as a binary trait for our study, based on whether a patient had achieved remission at their last visit, i.e. at week 8, or, at week 4 if week 8 remission status was unavailable. The “remission” rate in the Mayo-PGRN SSRI Study was 27.7% at 4 weeks and 48.8% at 8 weeks. Among the 135 SNPs analyzed, the top 4 SNPs (p<0.05) were all within the *GLDC* gene on chromosome 9, and the rs10975641 SNP was most significantly associated with remission status (p=0.008, [Table 2A](#T2)). This SNP was not in strong linkage disequilibrium (LD) with any of the other genotyped SNPs. When only subjects with remission status at week 8 were considered, the same SNP continued to be significant (p=0.009). When analyses were performed using the binary SSRI “response” phenotype (≥50% reduction in QIDS-C) or the continuous variable of percentage change in QIDS-C, we also observed associations with several SNPs in *GLDC,* including rs10975641 ([Table 2B](#T2)), as well as two *SHMT2* SNPs. Associations for all of the genotyped markers are listed in [Supplemental Table 1](#SD1). P-values shown in the tables are not corrected for multiple testing. However, the minimum p-value based on a permutation-based method to adjust for multiple testing was 0.25 (for the reduction in QIDS-C phenotype). Since the same SNP, rs10975641, had been highly associated with multiple SSRI treatment outcome phenotypes, we proceeded to a replication study with this SNP.

### Table 2.

| (A) SNPs associated with “remission” after SSRI treatment outcome. |  |  |  |  |  |  |  |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
|   |   |   |   | Remission (last visit) |  |  | Remission (week 8) |  |  |
| rs# | chr | gene | location | MAF (non-remitters/remitters) | OR (95% CI) | p-value | MAF (non-remitters/remit) | OR (95% CI) | p-value |
| rs1658957 | 9 | GLDC | 5′-flanking | 0.47/0.54 | 1.3 (1.0,1.7) | 0.026 | 0.48/0.54 | 1.3 (0.97,1.7) | 0.084 |
| rs10975734 | 9 | GLDC | 5′-flanking | 0.26/0.33 | 1.4 (1.1,1.9) | 0.013 | 0.25/0.33 | 1.5 (1.1,2.1) | 0.007 |
| rs11789777 | 9 | GLDC | intron9 | 0.34/0.28 | 0.76 (0.58,0.99) | 0.043 | 0.33/0.28 | 0.77 (0.58,1.0) | 0.083 |
|   |  |  |  |  |  |  |  |  |  |
| rs10975641 | 9 | GLDC | intron17 | 0.43/0.34 | 0.70 (0.53,0.91) | 0.008 | 0.43/0.34 | 0.68 (0.51,0.91) | 0.009 |

Table 2 Caption: Association results with p-values <0.05 for citalopram/escitalopram treatment outcomes.

### STAR*D replication study

DNA samples from 1926 STAR*D patients were genotyped for the rs10975641 SNP. Genotyping included 150 duplicate samples and 22 non-template controls. Twelve of the 1926 STAR*D samples were excluded (5 because of lack of clinical outcome data, 3 unintended duplicates and 4 gender mismatches). Another 114 subjects with ethnicity other than African-American (AA), WNH were excluded. Twenty-one of the remaining 1800 subjects failed genotyping. Thus 1779 samples were included in the genotype association analysis, including 293 from AA subjects, 1245 from NHW and 241 from Hispanic White subjects. The minor allele frequency for rs10975641 was similar among the 3 ethnic groups (0.36 to 0.40), and there were no deviations from Hardy Weinberg Equilibrium (HWE) in any of the ethnically-defined subgroups. The association analysis showed that rs10975641 was significantly associated with the binary phenotype of “response” in WNH subjects (p=0.016), with an odds ratio (OR) of 0.110 (95% CI, 0.013–0.949), as well as in all subjects when the data were adjusted for ethnicity (p=0.023, OR 0.179, 95% CI, 0.035–0.907). There was not a significant association in the STAR*D patient population with the other SSRI treatment outcome phenotypes, i.e. remission or percentage reduction in QIDS-C score.

To determine whether the rs10975641 SNP might be in LD with other ungenotyped SNPs, we sequenced a 5.8 kb region of *GLDC* surrounding this SNP ([Figure 4A](#F4)) using DNA from 96 Caucasian subjects – 48 homozygous for CC at rs10975641 and 48 homozygous for GG. A total of 35 polymorphisms, including 19 that were novel, were identified, 27 of which had not been genotyped in our initial study ([Figure 4B](#F4)). LD across the 5.8 kb resequenced region was consistent with that derived from the HapMap data for CEU Caucasian samples. Of the 27 ungenotyped polymorphisms identified, four were in weak LD with rs10975641 (r^2^ =0.13–0.26). Therefore, no highly plausible candidates other that the original SNP were identified during resequencing. We next proceeded to functional genomic experiments to determine whether the rs10975641 SNP might have functional implications.

### Figure 4.

![Figure 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f9be/3034442/5b638bb95054/nihms264812f4.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3034442_nihms264812f4.jpg)

The GLDC gene and its linkage disequilibrium relationships. (A) A region of chromosome 9 spanning the portion of the GLDC gene from which tag SNPs were selected for genotyping is shown. The box indicates a region around rs10975641 that was resequenced for purpose of fine-mapping. A Haploview (39) plot was generated using data from CEPH HapMap samples (CEU). (B) A “closer” view of the LD structure in the region of interest. Both genotyped (red) tag SNPs and SNPs identified during the resequencing effort (9) are plotted.

### Electrophoretic Mobility Shift (EMS) assays

EMS assays were performed using6 different cell lines, including the human brain-derived cell lines, U-87 MG, U-138 MG and U251, and 3 non-brain derived cell lines, HepG2, HEK293T and lymphoblastoid cells. [Figure 5](#F5) shows that nuclear extract binding resulted in a “shift” for both wild type (WT) and variant nucleotides at the rs10975641 locus, but the C to G change associated with the SNP resulted in a striking increase in the intensity of binding. This difference in nuclear protein binding was present in all three of the human brain-derived cell lines but not the other three cell lines, which all showed less binding and failed to show a striking difference in binding patterns between WT and variant oligonucleotides.

### Figure 5.

![Figure 5](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f9be/3034442/7218da4afdf0/nihms264812f5.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3034442_nihms264812f5.jpg)

EMS assays for the rs10975641 SNP. Nuclear extract was prepared from three human brain-derived cell lines, glioblastoma U-87 MG, U-138 MG and U251 cell lines, as well as non-brain derived cell lines, human embryonic kidney 293T, HepG2 hepatocellular carcinoma cells and a pool of lymphoblastoid cells, and were used to perform EMS assays.

## DISCUSSION

Large variation in treatment response remains a major challenge in the therapy of MDD ([2](#R2), [3](#R3)). Currently, there are few reliable biomarkers to help predict this variation. Pharmacome-tabolomics is an emerging field that uses metabolomic techniques to help discover biomarkers for drug response or exposure ([4](#R4)–[6](#R6)). Previous studies of central nervous system (CNS) disease have suggested that study of the metabolome can reflect individual difference in CNS disease stage or progression ([22](#R22)–[27](#R27)). A search for pharmacometabolomic “signatures” for SSRI antidepressant therapy outcome might not only identify potential biomarkers for treatment outcome, but could also help identify underlying genomic variation associated with disease pathophysiology that contributes to variation in response to drug therapy. Therefore, we performed a pharmacometabolomics-informed pharmacogenomic study using samples from MDD patients enrolled in two large SSRI clinical trials, the Mayo-PGRN SSRI study and the STAR*D study.

As a first step, we used a GC-MS-based metabolomic platform to profile plasma samples obtained prior to escitalopram therapy from remitter and non-remitter MDD patients. The goal was to determine whether, prior to drug therapy, any metabolite might be associated with variation in drug response. Pathway-specific regression analysis resulted in the identification of several metabolites in the nitrogen metabolism pathway, including glycine, that were significantly associated with percentage reduction in QIDS-C score, a quantitative measure of SSRI treatment outcome ([Table 1](#T1)). The fact that these associations were observed using pre-treatment metabolomic profiles led us to test the hypothesis that genetic variation might contribute to the observed variation in this metabolomic “signature”. That pharmacogenomic study focused on genes encoding glycine synthesis and degradation enzymes. Glycine is an inhibitory neurotransmitter in the CNS in addition to its role in a variety of essential cellular processes, e.g., one-carbon metabolism ([16](#R16), [17](#R17)). Several previous studies have attempted to correlate plasma glycine concentrations with risk for depression (not with treatment outcome) – with contradictory results ([28](#R28)–[30](#R30)). Our metabolomic data indicated that baseline plasma glycine level was negatively associated with escitalopram treatment outcome, compatible with the conclusion that elevated glycine might be a marker for decreased SSRI response.

Our initial glycine synthesis and degradation pathway-based pharmacogenomic study involved genotyping 144 markers for candidate genes using DNA from 529 MDD patients enrolled in the Mayo-PGRN SSRI study, followed by validation using DNA samples from patients enrolled in the initial phase of the STAR*D study. By using samples from these two large antidepressant clinical trials with similar designs, we identified one marker, rs10975641, a SNP in the *GLDC* ([31](#R31)), that was associated with multiple SSRI treatment outcome phenotypes in the Mayo-PGRN SSRI study and with SSRI “response” in the STAR*D study. Because of differences between STAR*D subjects who contributed DNA and those who did not ([32](#R32), [33](#R33)), the present observations cannot be considered generalizable without further replication. Additional fine-mapping performed by resequencing a portion of the *GLDC* gene that included the region containing rs10975641 failed to identify any markers in high LD with this SNP. Finally, when we performed EMS assays using nuclear extract from six different cell lines, we observed a striking difference in the pattern of nuclear protein binding between WT and variant oligonucleotides at the rs10975641 locus, as well as a significant difference between human brain-derived and non-brain-derived cell lines ([Figure 5](#F5)). Genomic variation, including deletion of the *GLDC* gene, has been associated with disease related to glycine metabolism, e.g., nonketotic hyperglycinaemia ([34](#R34), [35](#R35)). Our results raise the possibility that a change from C to G at rs10975641 in intron 17 of *GLDC* might result in an alternation in nuclear protein binding that could influence gene function in a tissue-specific fashion, thereby influencing glycine metabolism within the CNS. Obviously, these results will have to be replicated, hopefully in larger populations of patients with MDD who are treated with citalopram/escitalopram.

Pharmacological studies of SSRIs have focused mainly on the serotonin system, based on the monoamine neurotransmitter theory of depression ([16](#R16), [36](#R36)). The same has been true for pharmacogenetic studies of SSRIs ([8](#R8), [10](#R10)), at least until the very recently reported GWA studies of SSRI treatment outcome ([11](#R11)–[13](#R13)). Our use of metabolomics to “inform” pharmacogenomics might provide a novel approach to biomarker discovery for the individualized therapy of MDD and other psychiatric illnesses. In this case, it has served to focus our attention on the possible role of the *GLDC* gene in SSRI therapy outcomes in patients with MDD.

## METHODS

### Study subjects

Samples for metabolomic experiments were obtained from patients enrolled in the Mayo-PGRN SSRI study. Enrollment required a diagnosis of MDD without psychosis or mania and a Hamilton Depression Rating Scale score of >14. Inclusion and exclusion criteria were similar to those of the STAR*D study ([15](#R15)). In the Mayo-PGRN SSRI study, patients received escitalopram (10 mg) or citalopram (20 mg) at time zero, with the possibility of dose escalation to 20 mg of escitalopram or 40 mg of citalopram at week 4. Blood samples were obtained at zero time and weeks 4 and 8. Blood samples were drawn into 10 mL EDTA tubes, followed by centrifugation at 8000 ×g for 20 min. Plasma was removed immediately, flash frozen and stored in 1 mL aliquots.

Citalopram/escitalopram treatment outcome was determined using QIDS-C scores, with “remission” defined as QIDS-C ≤5 and “response” as QIDS-C reduction of ≥50%. Samples for metabolomic analysis were obtained from 40 MDD patients in the Mayo-PGRN SSRI study. Those patients included 20 “remitters” (final QIDS-C ≤ 5) and 20 “non-remitters” with QIDS-C>5. Samples for metabolomic profiling were obtained at zero time. All patients provided written informed consent. The study protocol was reviewed and approved by the Mayo Clinic Institutional Review Board.

### Metabolomic profiling, statistical and pathway analysis

GC-MS metabolomic profiles were analyzed using plasma samples from each of 40 patients (see [Supplementary Methods](#SD1)). For purposes of the metabolomic analysis, SSRI treatment outcome was percentage change in QIDS-C score after drug exposure. The data were log-transformed before fitting regression models for metabolites with right-skewed distributions and/or those with large values at the extremes.

To determine whether metabolic profiles at baseline might help to predict SSRI response, Spearman rank correlation coefficients between each metabolite at baseline and the percent change of QIDS-C score and the corresponding p-values of significance test were calculated. Q-values were also obtained to control for the false discovery rate ([37](#R37)). A marginal correlation test was then enhanced by pathway-specific regression analysis. Students’ t-test was also performed and plotted using JMP® Statistical Discovery Software (SAS, Cary, NC) to compare glycine levels at time zero between escitalopram “remitters” (N=20) and “non-remitters” (N=19) and between “responders” (N=26) and “non-responders” (N=13). One significant outlier non-remitter sample was excluded from this analysis.

### Pharmacogenomic study populations

We then performed an SSRI pharmacogenomic association study using 529 DNA samples from patients enrolled in the ongoing Mayo-PGRN SSRI study – the same study used to obtained samples for the metabolomic analyses. DNA samples from all subjects enrolled at the time of the present study were genotyped. The focus was on tag SNPs in genes encoding enzymes that catalyze glycine synthesis and degradation, followed by replication performed with DNA samples from the initial phase of the STAR*D study in which all patients were treated with citalopram (20–80 mg/day) ([15](#R15)). The Mayo-PGRN SSRI study was designed to parallel the STAR*D study. 1926 STAR*D DNA samples (1245 from WNH subjects), together with clinical data, were obtained from the STAR*D Data Coordinating Center.

### SNP genotyping and DNA sequencing

Genotype data for genes encoding glycine synthesis enzymes (*SHMT1* and *SHMT2*) and glycine degradation enzymes (*AMT*, *DLD, GCSH* and *GLDC*) were utilized for SNP selection (see [Supplementary Methods](#SD1)). A panel of 144 SNPs, including two QC SNPs, was used to genotype the Mayo-PGRN SSRI study utilizing the Illumina Golden Gate platform (Illumina, San Diego, CA). A Taqman genotyping assay (Applied Biosystems, Foster City, CA) was used to perform the STAR*D validation genotyping study. Details of the Sanger “resequencing” of a portion of *GLDC* are listed in the [Supplementary Methods](#SD1).

### Single SNP associations with SSRI treatment outcome

After data QC (See [Supplementary Material](#SD1)) single SNP association analyses were performed with SSRI treatment outcome phenotypes, including remission (QIDS-C≤ 5) at week 8 or week 4 if week 8 remission status was not available; remission at week 8 only, or response, as well as percentage change in QIDS-C score from baseline to week 8, which was analyzed as a continuous variable. For binary phenotypes, i.e. remission vs. non-remission and response vs. non-response, a p-value was calculated for each SNP based on a logistic regression model, assuming a log-additive allele effect. P-values were adjusted for population stratification using eigenvectors calculated with the EIGENSOFT software EIGENSTRAT (smartpca) based on a set of genome-wide SNPs in low or no LD with each other (data not presented). Using the corresponding eigen values, a Tracy-Widom test was conducted to determine the number of eigenvectors to be used to adjust analyses for population stratification. For the reduction in QIDS-C phenotype, the eigenvectors representing population stratification were regressed on the phenotype, and resulting residuals were used to calculate the Spearman rank correlation with individual SNPs modeled as minor allele counts. A step-up permutation based method was used to adjust p-values for multiple testing.

### Association analysis for the STAR*D validation study

An analysis for association similar to that used during the “discovery” study was applied, and a linear-regression model was used for the quantitative percentage reduction in QIDS-C score.

### EMS assays

We performed EMS assays for the rs10975641 SNP to determine whether this SNP might result in an alternation in ability to bind nuclear protein(s). Nuclear extracts were obtained from each of three human brain-derived cell lines, specifically, the glioblastoma cell lines U-87 MG, U-138 MG and U251, as well as three human cell lines from tissues other than brain, human embryonic kidney 293T, HepG2 hepatocellular carcinoma cells and a pool of lymphoblastoid cells from anonymized healthy individuals. All of these cell lines were obtained from the American Type Culture Collection (ATCC, Manassas, VA) except for the lymphoblastoid cell lines, which were obtained from the Coriell Institute (Camden, NJ). The EMS assays were performed as described previously ([38](#R38)). Oligonucleotide sequences for the rs10975641 SNP probes were WT-5′taagatcctttCaggctaggtct3′ and variant-5′ taagatcctttGaggctaggtct 3′.

## Supplementary Material

## Acknowledgments

This research was supported, in part, by NIH grants RO1 GM28157, U01 GM61388 (The Pharmacogenomics Research Network), P20 1P20AA017830-01 (The Mayo Clinic Center for Individualized Treatment of Alcohol Dependence), a PhRMA Foundation Center of Excellence in Clinical Pharmacology Award, as well as NIH grant R24 GM78233 (The Metabolomic Research Network for Drug Response Phenotypes). Dr. Yuan Ji’s work was partially supported by a KL2 Mentored Career Development Award (NCRR Grant 1 KL2 RR024151) and a Gerstner Family Mayo Career Development Award in Individualized Medicine. We thank the staff of the Mayo Clinic Rochester Department of Psychiatry and Psychology for their effort in recruiting the patients engaged in the Mayo-PGRN SSRI Study and Luanne Wussow for her assistance with the preparation of the manuscript. The authors are also grateful for the effort that the STAR*D investigators put into that study and for their generosity in sharing both their clinical data and DNA samples.

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Kessler RC, et al. The epidemiology of major depressive disorder: results from the National Comorbidity Survey Replication (NCS-R) Jama. 2003;289:3095–3105. doi: 10.1001/jama.289.23.3095.  [DOI](https://doi.org/10.1001/jama.289.23.3095) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12813115/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Jama&title=The%20epidemiology%20of%20major%20depressive%20disorder:%20results%20from%20the%20National%20Comorbidity%20Survey%20Replication%20(NCS-R)&author=RC%20Kessler&volume=289&publication_year=2003&pages=3095-3105&pmid=12813115&doi=10.1001/jama.289.23.3095&)

2. Preskorn S, Staga C, Feighner J, Ross R, editors. Antidepressants: Past, Present and Future. Springer; 2004.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Antidepressants:%20Past,%20Present%20and%20Future&publication_year=2004&)

3. Thase ME, et al. Remission rates following antidepressant therapy with bupropion or selective serotonin reuptake inhibitors: a meta-analysis of original data from 7 randomized controlled trials. J Clin Psychiatry. 2005;66:974–981. doi: 10.4088/jcp.v66n0803.  [DOI](https://doi.org/10.4088/jcp.v66n0803) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16086611/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Psychiatry&title=Remission%20rates%20following%20antidepressant%20therapy%20with%20bupropion%20or%20selective%20serotonin%20reuptake%20inhibitors:%20a%20meta-analysis%20of%20original%20data%20from%207%20randomized%20controlled%20trials&author=ME%20Thase&volume=66&publication_year=2005&pages=974-981&pmid=16086611&doi=10.4088/jcp.v66n0803&)

4. Kaddurah-Daouk R, Kristal BS, Weinshilboum RM. Metabolomics: a global biochemical approach to drug response and disease. Annu Rev Pharmacol Toxicol. 2008;48:653–683. doi: 10.1146/annurev.pharmtox.48.113006.094715.  [DOI](https://doi.org/10.1146/annurev.pharmtox.48.113006.094715) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18184107/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Annu%20Rev%20Pharmacol%20Toxicol&title=Metabolomics:%20a%20global%20biochemical%20approach%20to%20drug%20response%20and%20disease&author=R%20Kaddurah-Daouk&author=BS%20Kristal&author=RM%20Weinshilboum&volume=48&publication_year=2008&pages=653-683&pmid=18184107&doi=10.1146/annurev.pharmtox.48.113006.094715&)

5. Schmidt C. Metabolomics takes its place as latest up-and-coming “omic” science. J Natl Cancer Inst. 2004;96:732–734. doi: 10.1093/jnci/96.10.732.  [DOI](https://doi.org/10.1093/jnci/96.10.732) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15150298/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Natl%20Cancer%20Inst&title=Metabolomics%20takes%20its%20place%20as%20latest%20up-and-coming%20%E2%80%9Comic%E2%80%9D%20science&author=C%20Schmidt&volume=96&publication_year=2004&pages=732-734&pmid=15150298&doi=10.1093/jnci/96.10.732&)

6. Lindon JC, Holmes E, Bollard ME, Stanley EG, Nicholson JK. Metabonomics technologies and their applications in physiological monitoring, drug safety assessment and disease diagnosis. Biomarkers. 2004;9:1–31. doi: 10.1080/13547500410001668379.  [DOI](https://doi.org/10.1080/13547500410001668379) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15204308/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biomarkers&title=Metabonomics%20technologies%20and%20their%20applications%20in%20physiological%20monitoring,%20drug%20safety%20assessment%20and%20disease%20diagnosis&author=JC%20Lindon&author=E%20Holmes&author=ME%20Bollard&author=EG%20Stanley&author=JK%20Nicholson&volume=9&publication_year=2004&pages=1-31&pmid=15204308&doi=10.1080/13547500410001668379&)

7. Weinshilboum RM, Wang L. Pharmacogenetics and pharmacogenomics: development, science, and translation. Annu Rev Genomics Hum Genet. 2006;7:223–245. doi: 10.1146/annurev.genom.6.080604.162315.  [DOI](https://doi.org/10.1146/annurev.genom.6.080604.162315) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16948615/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Annu%20Rev%20Genomics%20Hum%20Genet&title=Pharmacogenetics%20and%20pharmacogenomics:%20development,%20science,%20and%20translation&author=RM%20Weinshilboum&author=L%20Wang&volume=7&publication_year=2006&pages=223-245&pmid=16948615&doi=10.1146/annurev.genom.6.080604.162315&)

8. Kato M, Serretti A. Review and meta-analysis of antidepressant pharmacogenetic findings in major depressive disorder. Mol Psychiatry. 2008;15:473–500. doi: 10.1038/mp.2008.116.  [DOI](https://doi.org/10.1038/mp.2008.116) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18982004/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Psychiatry&title=Review%20and%20meta-analysis%20of%20antidepressant%20pharmacogenetic%20findings%20in%20major%20depressive%20disorder&author=M%20Kato&author=A%20Serretti&volume=15&publication_year=2008&pages=473-500&pmid=18982004&doi=10.1038/mp.2008.116&)

9. Mrazek DA, et al. SLC6A4 variation and citalopram response. Am J Med Genet B Neuropsychiatr Genet. 2009;150B:341–351. doi: 10.1002/ajmg.b.30816.  [DOI](https://doi.org/10.1002/ajmg.b.30816) | [PMC free article](/articles/PMC2660379/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18618621/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Med%20Genet%20B%20Neuropsychiatr%20Genet&title=SLC6A4%20variation%20and%20citalopram%20response&author=DA%20Mrazek&volume=150B&publication_year=2009&pages=341-351&pmid=18618621&doi=10.1002/ajmg.b.30816&)

10. Serretti A, Artioli P. The pharmacogenomics of selective serotonin reuptake inhibitors. Pharmacogenomics J. 2004;4:233–244. doi: 10.1038/sj.tpj.6500250.  [DOI](https://doi.org/10.1038/sj.tpj.6500250) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15111987/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=The%20pharmacogenomics%20of%20selective%20serotonin%20reuptake%20inhibitors&author=A%20Serretti&author=P%20Artioli&volume=4&publication_year=2004&pages=233-244&pmid=15111987&doi=10.1038/sj.tpj.6500250&)

11. Garriock HA, et al. A genomewide association study of citalopram response in major depressive disorder. Biol Psychiatry. 2010;67:133–138. doi: 10.1016/j.biopsych.2009.08.029.  [DOI](https://doi.org/10.1016/j.biopsych.2009.08.029) | [PMC free article](/articles/PMC2794921/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19846067/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biol%20Psychiatry&title=A%20genomewide%20association%20study%20of%20citalopram%20response%20in%20major%20depressive%20disorder&author=HA%20Garriock&volume=67&publication_year=2010&pages=133-138&pmid=19846067&doi=10.1016/j.biopsych.2009.08.029&)

12. Uher R, et al. Genome-wide pharmacogenetics of antidepressant response in the GENDEP Project. Am J Psychiatry. 2010;167:555–564. doi: 10.1176/appi.ajp.2009.09070932.  [DOI](https://doi.org/10.1176/appi.ajp.2009.09070932) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20360315/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Psychiatry&title=Genome-wide%20pharmacogenetics%20of%20antidepressant%20response%20in%20the%20GENDEP%20Project&author=R%20Uher&volume=167&publication_year=2010&pages=555-564&pmid=20360315&doi=10.1176/appi.ajp.2009.09070932&)

13. Ising M, et al. A genomewide association study points to multiple loci that predict antidepressant drug treatment outcome in depression. Arch Gen Psychiatry. 2009;66:966–975. doi: 10.1001/archgenpsychiatry.2009.95.  [DOI](https://doi.org/10.1001/archgenpsychiatry.2009.95) | [PMC free article](/articles/PMC4465570/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19736353/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arch%20Gen%20Psychiatry&title=A%20genomewide%20association%20study%20points%20to%20multiple%20loci%20that%20predict%20antidepressant%20drug%20treatment%20outcome%20in%20depression&author=M%20Ising&volume=66&publication_year=2009&pages=966-975&pmid=19736353&doi=10.1001/archgenpsychiatry.2009.95&)

14. Rush AJ, et al. The 16-Item Quick Inventory of Depressive Symptomatology (QIDS), clinician rating (QIDS-C), and self-report (QIDS-SR): a psychometric evaluation in patients with chronic major depression. Biol Psychiatry. 2003;54:573–583. doi: 10.1016/s0006-3223(02)01866-8.  [DOI](https://doi.org/10.1016/s0006-3223(02)01866-8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12946886/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biol%20Psychiatry&title=The%2016-Item%20Quick%20Inventory%20of%20Depressive%20Symptomatology%20(QIDS),%20clinician%20rating%20(QIDS-C),%20and%20self-report%20(QIDS-SR):%20a%20psychometric%20evaluation%20in%20patients%20with%20chronic%20major%20depression&author=AJ%20Rush&volume=54&publication_year=2003&pages=573-583&pmid=12946886&doi=10.1016/s0006-3223(02)01866-8&)

15. Trivedi MH, et al. Evaluation of outcomes with citalopram for depression using measurement-based care in STAR*D: implications for clinical practice. Am J Psychiatry. 2006;163:28–40. doi: 10.1176/appi.ajp.163.1.28.  [DOI](https://doi.org/10.1176/appi.ajp.163.1.28) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16390886/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Psychiatry&title=Evaluation%20of%20outcomes%20with%20citalopram%20for%20depression%20using%20measurement-based%20care%20in%20STAR*D:%20implications%20for%20clinical%20practice&author=MH%20Trivedi&volume=163&publication_year=2006&pages=28-40&pmid=16390886&doi=10.1176/appi.ajp.163.1.28&)

16. Leslie Iversen SI, Bloom Floyd E, Roth Robert H. Introduction to Neuropsychopharmacology. 1. Oxford University Press; 2008.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Introduction%20to%20Neuropsychopharmacology&author=SI%20Leslie%20Iversen&author=Floyd%20E%20Bloom&author=Robert%20H%20Roth&publication_year=2008&)

17. Hall JC. Glycine. JPEN J Parenter Enteral Nutr. 1998;22:393–398. doi: 10.1177/0148607198022006393.  [DOI](https://doi.org/10.1177/0148607198022006393) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9829614/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JPEN%20J%20Parenter%20Enteral%20Nutr&title=Glycine&author=JC%20Hall&volume=22&publication_year=1998&pages=393-398&pmid=9829614&doi=10.1177/0148607198022006393&)

18. McNeil JB, McIntosh EM, Taylor BV, Zhang FR, Tang S, Bognar AL. Cloning and molecular characterization of three genes, including two genes encoding serine hydroxymethyltransferases, whose inactivation is required to render yeast auxotrophic for glycine. J Biol Chem. 1994;269:9155–9165.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/8132653/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=Cloning%20and%20molecular%20characterization%20of%20three%20genes,%20including%20two%20genes%20encoding%20serine%20hydroxymethyltransferases,%20whose%20inactivation%20is%20required%20to%20render%20yeast%20auxotrophic%20for%20glycine&author=JB%20McNeil&author=EM%20McIntosh&author=BV%20Taylor&author=FR%20Zhang&author=S%20Tang&volume=269&publication_year=1994&pages=9155-9165&pmid=8132653&)

19. Turner SR, Ireland R, Morgan C, Rawsthorne S. Identification and localization of multiple forms of serine hydroxymethyltransferase in pea (Pisum sativum) and characterization of a cDNA encoding a mitochondrial isoform. J Biol Chem. 1992;267:13528–13534.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/1618853/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=Identification%20and%20localization%20of%20multiple%20forms%20of%20serine%20hydroxymethyltransferase%20in%20pea%20(Pisum%20sativum)%20and%20characterization%20of%20a%20cDNA%20encoding%20a%20mitochondrial%20isoform&author=SR%20Turner&author=R%20Ireland&author=C%20Morgan&author=S%20Rawsthorne&volume=267&publication_year=1992&pages=13528-13534&pmid=1618853&)

20. Ichinohe A, et al. Glycine cleavage system in neurogenic regions. Eur J Neurosci. 2004;19:2365–2370. doi: 10.1111/j.0953-816X.2004.03345.x.  [DOI](https://doi.org/10.1111/j.0953-816X.2004.03345.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15128390/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Neurosci&title=Glycine%20cleavage%20system%20in%20neurogenic%20regions&author=A%20Ichinohe&volume=19&publication_year=2004&pages=2365-2370&pmid=15128390&doi=10.1111/j.0953-816X.2004.03345.x&)

21. Kikuchi G, Motokawa Y, Yoshida T, Hiraga K. Glycine cleavage system: reaction mechanism, physiological significance, and hyperglycinemia. Proc Jpn Acad Ser B Phys Biol Sci. 2008;84:246–263. doi: 10.2183/pjab/84.246.  [DOI](https://doi.org/10.2183/pjab/84.246) | [PMC free article](/articles/PMC3666648/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18941301/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Jpn%20Acad%20Ser%20B%20Phys%20Biol%20Sci&title=Glycine%20cleavage%20system:%20reaction%20mechanism,%20physiological%20significance,%20and%20hyperglycinemia&author=G%20Kikuchi&author=Y%20Motokawa&author=T%20Yoshida&author=K%20Hiraga&volume=84&publication_year=2008&pages=246-263&pmid=18941301&doi=10.2183/pjab/84.246&)

22. Kaddurah-Daouk R, Krishnan KR. Metabolomics: a global biochemical approach to the study of central nervous system diseases. Neuropsychopharmacology. 2009;34:173–186. doi: 10.1038/npp.2008.174.  [DOI](https://doi.org/10.1038/npp.2008.174) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18843269/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neuropsychopharmacology&title=Metabolomics:%20a%20global%20biochemical%20approach%20to%20the%20study%20of%20central%20nervous%20system%20diseases&author=R%20Kaddurah-Daouk&author=KR%20Krishnan&volume=34&publication_year=2009&pages=173-186&pmid=18843269&doi=10.1038/npp.2008.174&)

23. Quinones MP, Kaddurah-Daouk R. Metabolomics tools for identifying biomarkers for neuropsychiatric diseases. Neurobiol Dis. 2009;35:165–176. doi: 10.1016/j.nbd.2009.02.019.  [DOI](https://doi.org/10.1016/j.nbd.2009.02.019) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19303440/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neurobiol%20Dis&title=Metabolomics%20tools%20for%20identifying%20biomarkers%20for%20neuropsychiatric%20diseases&author=MP%20Quinones&author=R%20Kaddurah-Daouk&volume=35&publication_year=2009&pages=165-176&pmid=19303440&doi=10.1016/j.nbd.2009.02.019&)

24. Yao JK, et al. Altered interactions of tryptophan metabolites in first-episode neuroleptic-naive patients with schizophrenia. Mol Psychiatry. 2009 doi: 10.1038/mp.2009.33. in press.  [DOI](https://doi.org/10.1038/mp.2009.33) | [PMC free article](/articles/PMC2953575/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19401681/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Psychiatry&title=Altered%20interactions%20of%20tryptophan%20metabolites%20in%20first-episode%20neuroleptic-naive%20patients%20with%20schizophrenia&author=JK%20Yao&publication_year=2009&pmid=19401681&doi=10.1038/mp.2009.33&)

25. Kaddurah-Daouk R, et al. Metabolomic mapping of atypical antipsychotic effects in schizophrenia. Mol Psychiatry. 2007;12:934–945. doi: 10.1038/sj.mp.4002000.  [DOI](https://doi.org/10.1038/sj.mp.4002000) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17440431/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Psychiatry&title=Metabolomic%20mapping%20of%20atypical%20antipsychotic%20effects%20in%20schizophrenia&author=R%20Kaddurah-Daouk&volume=12&publication_year=2007&pages=934-945&pmid=17440431&doi=10.1038/sj.mp.4002000&)

26. Mannelli P, Patkar A, Rozen S, Matson W, Krishnan R, Kaddurah-Daouk R. Opioid use affects antioxidant activity and purine metabolism: preliminary results. Hum Psychopharmacol. 2009;24:666–675. doi: 10.1002/hup.1068.  [DOI](https://doi.org/10.1002/hup.1068) | [PMC free article](/articles/PMC3183957/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19760630/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Psychopharmacol&title=Opioid%20use%20affects%20antioxidant%20activity%20and%20purine%20metabolism:%20preliminary%20results&author=P%20Mannelli&author=A%20Patkar&author=S%20Rozen&author=W%20Matson&author=R%20Krishnan&volume=24&publication_year=2009&pages=666-675&pmid=19760630&doi=10.1002/hup.1068&)

27. Patkar AA, et al. Alterations in tryptophan and purine metabolism in cocaine addiction: a metabolomic study. Psychopharmacology (Berl) 2009;206:479–489. doi: 10.1007/s00213-009-1625-1.  [DOI](https://doi.org/10.1007/s00213-009-1625-1) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19649617/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Psychopharmacology%20(Berl)&title=Alterations%20in%20tryptophan%20and%20purine%20metabolism%20in%20cocaine%20addiction:%20a%20metabolomic%20study&author=AA%20Patkar&volume=206&publication_year=2009&pages=479-489&pmid=19649617&doi=10.1007/s00213-009-1625-1&)

28. Altamura C, Maes M, Dai J, Meltzer HY. Plasma concentrations of excitatory amino acids, serine, glycine, taurine and histidine in major depression. Eur Neuropsychopharmacol. 1995;5 (Suppl):71–75. doi: 10.1016/0924-977x(95)00033-l.  [DOI](https://doi.org/10.1016/0924-977x(95)00033-l) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8775762/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20Neuropsychopharmacol&title=Plasma%20concentrations%20of%20excitatory%20amino%20acids,%20serine,%20glycine,%20taurine%20and%20histidine%20in%20major%20depression&author=C%20Altamura&author=M%20Maes&author=J%20Dai&author=HY%20Meltzer&volume=5&issue=Suppl&publication_year=1995&pages=71-75&pmid=8775762&doi=10.1016/0924-977x(95)00033-l&)

29. Sumiyoshi T, Anil AE, Jin D, Jayathilake K, Lee M, Meltzer HY. Plasma glycine and serine levels in schizophrenia compared to normal controls and major depression: relation to negative symptoms. Int J Neuropsychopharmacol. 2004;7:1–8. doi: 10.1017/S1461145703003900.  [DOI](https://doi.org/10.1017/S1461145703003900) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14720317/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Neuropsychopharmacol&title=Plasma%20glycine%20and%20serine%20levels%20in%20schizophrenia%20compared%20to%20normal%20controls%20and%20major%20depression:%20relation%20to%20negative%20symptoms&author=T%20Sumiyoshi&author=AE%20Anil&author=D%20Jin&author=K%20Jayathilake&author=M%20Lee&volume=7&publication_year=2004&pages=1-8&pmid=14720317&doi=10.1017/S1461145703003900&)

30. Mitani H, Shirayama Y, Yamada T, Maeda K, Ashby CR, Jr, Kawahara R. Correlation between plasma levels of glutamate, alanine and serine with severity of depression. Prog Neuropsychopharmacol Biol Psychiatry. 2006;30:1155–1158. doi: 10.1016/j.pnpbp.2006.03.036.  [DOI](https://doi.org/10.1016/j.pnpbp.2006.03.036) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16707201/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Prog%20Neuropsychopharmacol%20Biol%20Psychiatry&title=Correlation%20between%20plasma%20levels%20of%20glutamate,%20alanine%20and%20serine%20with%20severity%20of%20depression&author=H%20Mitani&author=Y%20Shirayama&author=T%20Yamada&author=K%20Maeda&author=CR%20Ashby&volume=30&publication_year=2006&pages=1155-1158&pmid=16707201&doi=10.1016/j.pnpbp.2006.03.036&)

31. Takayanagi M, et al. Human glycine decarboxylase gene (GLDC) and its highly conserved processed pseudogene (psiGLDC): their structure and expression, and the identification of a large deletion in a family with nonketotic hyperglycinemia. Hum Genet. 2000;106:298–305. doi: 10.1007/s004390051041.  [DOI](https://doi.org/10.1007/s004390051041) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10798358/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Genet&title=Human%20glycine%20decarboxylase%20gene%20(GLDC)%20and%20its%20highly%20conserved%20processed%20pseudogene%20(psiGLDC):%20their%20structure%20and%20expression,%20and%20the%20identification%20of%20a%20large%20deletion%20in%20a%20family%20with%20nonketotic%20hyperglycinemia&author=M%20Takayanagi&volume=106&publication_year=2000&pages=298-305&pmid=10798358&doi=10.1007/s004390051041&)

32. Kraft JB, et al. Analysis of association between the serotonin transporter and antidepressant response in a large clinical sample. Biol Psychiatry. 2007;61:734–742. doi: 10.1016/j.biopsych.2006.07.017.  [DOI](https://doi.org/10.1016/j.biopsych.2006.07.017) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17123473/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biol%20Psychiatry&title=Analysis%20of%20association%20between%20the%20serotonin%20transporter%20and%20antidepressant%20response%20in%20a%20large%20clinical%20sample&author=JB%20Kraft&volume=61&publication_year=2007&pages=734-742&pmid=17123473&doi=10.1016/j.biopsych.2006.07.017&)

33. McMahon FJ, et al. Variation in the gene encoding the serotonin 2A receptor is associated with outcome of antidepressant treatment. Am J Hum Genet. 2006;78:804–814. doi: 10.1086/503820.  [DOI](https://doi.org/10.1086/503820) | [PMC free article](/articles/PMC1474035/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16642436/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=Variation%20in%20the%20gene%20encoding%20the%20serotonin%202A%20receptor%20is%20associated%20with%20outcome%20of%20antidepressant%20treatment&author=FJ%20McMahon&volume=78&publication_year=2006&pages=804-814&pmid=16642436&doi=10.1086/503820&)

34. Kanno J, et al. Genomic deletion within GLDC is a major cause of non-ketotic hyperglycinaemia. J Med Genet. 2007;44:e69. doi: 10.1136/jmg.2006.043448.  [DOI](https://doi.org/10.1136/jmg.2006.043448) | [PMC free article](/articles/PMC2598024/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17361008/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Med%20Genet&title=Genomic%20deletion%20within%20GLDC%20is%20a%20major%20cause%20of%20non-ketotic%20hyperglycinaemia&author=J%20Kanno&volume=44&publication_year=2007&pages=e69&pmid=17361008&doi=10.1136/jmg.2006.043448&)

35. Sellner L, Edkins E, Greed L, Lewis B. Detection of mutations in the glycine decarboxylase gene in patients with nonketotic hyperglycinaemia. Mol Genet Metab. 2005;84:167–171. doi: 10.1016/j.ymgme.2004.10.001.  [DOI](https://doi.org/10.1016/j.ymgme.2004.10.001) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15670722/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Genet%20Metab&title=Detection%20of%20mutations%20in%20the%20glycine%20decarboxylase%20gene%20in%20patients%20with%20nonketotic%20hyperglycinaemia&author=L%20Sellner&author=E%20Edkins&author=L%20Greed&author=B%20Lewis&volume=84&publication_year=2005&pages=167-171&pmid=15670722&doi=10.1016/j.ymgme.2004.10.001&)

36. Rang HDMM, Ritter JM. Pharmacology. 4. Churchill Livingstone; 1999.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Pharmacology&author=HDMM%20Rang&author=JM%20Ritter&publication_year=1999&)

37. Storey JD, Tibshirani R. Statistical significance for genomewide studies. Proc Natl Acad Sci U S A. 2003;100:9440–9445. doi: 10.1073/pnas.1530509100.  [DOI](https://doi.org/10.1073/pnas.1530509100) | [PMC free article](/articles/PMC170937/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12883005/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&title=Statistical%20significance%20for%20genomewide%20studies&author=JD%20Storey&author=R%20Tibshirani&volume=100&publication_year=2003&pages=9440-9445&pmid=12883005&doi=10.1073/pnas.1530509100&)

38. Ji Y, et al. Breast cancer risk reduction and membrane-bound catechol O-methyltransferase genetic polymorphisms. Cancer Res. 2008;68:5997–6005. doi: 10.1158/0008-5472.CAN-08-0043.  [DOI](https://doi.org/10.1158/0008-5472.CAN-08-0043) | [PMC free article](/articles/PMC2518124/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18632656/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=Breast%20cancer%20risk%20reduction%20and%20membrane-bound%20catechol%20O-methyltransferase%20genetic%20polymorphisms&author=Y%20Ji&volume=68&publication_year=2008&pages=5997-6005&pmid=18632656&doi=10.1158/0008-5472.CAN-08-0043&)

39. Barrett JC, Fry B, Maller J, Daly MJ. Haploview: analysis and visualization of LD and haplotype maps. Bioinformatics. 2005;21:263–265. doi: 10.1093/bioinformatics/bth457.  [DOI](https://doi.org/10.1093/bioinformatics/bth457) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15297300/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=Haploview:%20analysis%20and%20visualization%20of%20LD%20and%20haplotype%20maps&author=JC%20Barrett&author=B%20Fry&author=J%20Maller&author=MJ%20Daly&volume=21&publication_year=2005&pages=263-265&pmid=15297300&doi=10.1093/bioinformatics/bth457&)
