# Discordant Associations between SLCO1B1 521T→C and Plasma Levels of Ritonavir-boosted Protease Inhibitors in AIDS Clinical Trials Group Study A5146

## Metadata
**Authors:** Xinyan Zhang, Camlin Tierney, Mary Albrecht, Lisa M Demeter, Gene Morse, Robin DiFrancesco, Carrie Dykes, Hongyu Jiang, David W Haas
**Journal:** Therapeutic drug monitoring
**Date:** 2013 Apr
**DOI:** [10.1097/FTD.0b013e318280d0ad](https://doi.org/10.1097/FTD.0b013e318280d0ad)
**PMID:** 23503447
**PMCID:** PMC3603284
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3603284/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC3603284/pdf/nihms432280.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC3603284/pdf/nihms432280.pdf)

## Abstract

**Objective:** 
Among HIV-positive patients prescribed ritonavir-boosted lopinavir, SLCO1B1 521T→C (rs4149056) is associated with increased plasma lopinavir exposure. Protease inhibitors are also substrates for cytochrome P450 (CYP) 3A and ABCB1, which are induced by NR1I2. We characterized relationships between ABCB1, CYP3A4, CYP3A5, NR1I2 and SLCO1B1 polymorphisms and trough protease inhibitor concentrations among AIDS Clinical Trials Group study A5146 participants.

**Methods:** 
At study entry, subjects with virologic failure on protease inhibitor-containing regimens initiated new ritonavir-boosted protease inhibitor regimens. We studied associations between week 2 protease inhibitor plasma trough concentrations and 143 polymorphisms in these genes, including 4 targeted polymorphisms.

**Results:** 
Among 275 subjects with both drug concentrations and genetic data, allelic frequencies of SLCO1B1 521T→C were 15%, 1%, and 8% in whites, blacks, and Hispanics, respectively. Further analyses were limited to 268 white, black, or Hispanic subjects who initiated ritonavir-boosted lopinavir (n=98), fosamprenavir (n=69), or saquinavir (n=99). Of targeted polymorphisms, SLCO1B1 521T→C tended to be associated with higher lopinavir concentrations, with a 1.38-fold increase in the mean per C allele (95% CI 0.97, 1.96; n=98; p=0.07). With fosamprenavir, SLCO1B1 521T→C was associated with lower amprenavir concentrations, with a 35% decrease in the mean per C allele (geometric mean ratio 0.65, 95% CI 0.44, 0.94; n=69; adjusted p=0.02). There was no significant association with saquinavir concentrations, and none of the remaining 139 exploratory polymorphisms were statistically significant after correcting for multiple comparisons.

**Conclusions:** 
With ritonavir-boosted protease inhibitors, a SLCO1B1 polymorphism that predicts higher lopinavir trough concentrations appears to predict lower amprenavir trough concentrations. The mechanism underlying this discordant association is uncertain.

Keywords: pharmacogenomics, protease inhibitors, HIV therapy

### Objective

Among HIV-positive patients prescribed ritonavir-boosted lopinavir, *SLCO1B1* 521T→C (rs4149056) is associated with increased plasma lopinavir exposure. Protease inhibitors are also substrates for cytochrome P450 (CYP) 3A and ABCB1, which are induced by NR1I2. We characterized relationships between *ABCB1, CYP3A4, CYP3A5*, *NR1I2* and *SLCO1B1* polymorphisms and trough protease inhibitor concentrations among AIDS Clinical Trials Group study A5146 participants.

### Methods

At study entry, subjects with virologic failure on protease inhibitor-containing regimens initiated new ritonavir-boosted protease inhibitor regimens. We studied associations between week 2 protease inhibitor plasma trough concentrations and 143 polymorphisms in these genes, including 4 targeted polymorphisms.

### Results

Among 275 subjects with both drug concentrations and genetic data, allelic frequencies of *SLCO1B1* 521T→C were 15%, 1%, and 8% in whites, blacks, and Hispanics, respectively. Further analyses were limited to 268 white, black, or Hispanic subjects who initiated ritonavir-boosted lopinavir (n=98), fosamprenavir (n=69), or saquinavir (n=99). Of targeted polymorphisms, *SLCO1B1* 521T→C tended to be associated with higher lopinavir concentrations, with a 1.38-fold increase in the mean per C allele (95% CI 0.97, 1.96; n=98; p=0.07). With fosamprenavir, *SLCO1B1* 521T→C was associated with lower amprenavir concentrations, with a 35% decrease in the mean per C allele (geometric mean ratio 0.65, 95% CI 0.44, 0.94; n=69; adjusted p=0.02). There was no significant association with saquinavir concentrations, and none of the remaining 139 exploratory polymorphisms were statistically significant after correcting for multiple comparisons.

### Conclusions

With ritonavir-boosted protease inhibitors, a *SLCO1B1* polymorphism that predicts higher lopinavir trough concentrations appears to predict lower amprenavir trough concentrations. The mechanism underlying this discordant association is uncertain.

## INTRODUCTION

Protease inhibitors (PIs) are widely prescribed for human immunodeficiency virus (HIV)-1 infection. Many of these drugs undergo hepatic metabolism by cytochrome P450 (CYP) 3A isoforms, and are substrates for membrane transporters including P-glycoprotein (also known as ATP-binding cassette transporter B1 [ABCB1]) [[1](#R1), [2](#R2)] and organic anion-transporting polypeptide 1B1 (OATP1B1, also known as solute carrier organic anion transporter B1 [SLCO1B1]) [[3](#R3)]. Genes that encode CYP isoforms and membrane transporters are highly polymorphic, and numerous functional genetic polymorphisms are well described. Variable expression of these genes is largely regulated by pregnane X receptor (PXR, also known as nuclear receptor 1I2 [NR1/2]), which is also induced by PIs [[4](#R4)–[7](#R7)].

The drug transporter OATP1B1 is localized to hepatic sinusoids where it facilitates uptake of drugs into hepatocytes to undergo metabolism and biliary excretion [[8](#R8)]. Several studies have associated a *SLCO1B1* polymorphism (521T→C, rs4149056) with increased plasma lopinavir concentrations [[3](#R3), [9](#R9)–[11](#R11)]. The *SLCO1B1* 521 C allele is more frequent with European ancestry than with African ancestry [[12](#R12)]. Although *SLCO1B1* transports other PIs *in vitro* (e.g., saquinavir and darunavir) [[3](#R3)], *in vivo* associations between 521T→C and PIs other than lopinavir have not been reported.

Total hepatic CYP3A activity in adults comprises CYP3A5 and CYP3A4 isoforms, which possess similar catalytic activities with some, but not all substrates [[13](#R13)]. A variant of *CYP3A5* (rs776746, also called *CYP3A5*3*) results in aberrant splicing and non-functional enzyme [[14](#R14)]. Among individuals homozygous for the *CYP3A5*3* allele (A), CYP3A5 comprises only 5% of hepatic CYP3A expression, compared to as much as 50% among individuals carrying at least one copy of the *CYP3A5*1* reference allele (G) [[15](#R15)]. Because *CYP3A5*3* is much more frequent with European ancestry than with African ancestry [[12](#R12)], CYP3A5 comprises a smaller proportion of total hepatic CYP3A activity in the former population. Among HIV-infected patients on PIs without ritonavir boosting, *CYP3A5*3* has been associated with decreased plasma clearance of atazanavir [[16](#R16)], indinavir [[17](#R17), [18](#R18)] and saquinavir [[19](#R19)]. Hepatic CYP3A4 expression, both basal and inducible, also varies markedly among individuals. There is, however, little evidence that frequent *CYP3A4* polymorphisms affect gene expression or enzyme activity. Reported associations between a promoter polymorphism (rs2740574, also called *CYP3A4*1B*) and various phenotypes (e.g. tacrolimus concentrations [[20](#R20)]) may reflect its linkage with *CYP3A5*3*.

The efflux transporter ABCB1 decreases drug absorption from the gut and enhances elimination in bile and urine [[21](#R21)–[23](#R23)]. Polymorphisms of *ABCB1* (3435C→T [rs1045642] and/or 2677G→T [rs2032582]) may correlate with intestinal *ABCB1* expression and activity. However, efforts to replicate associations with these *ABCB1* polymorphisms have yielded inconsistent results. For example regarding pharmacokinetics of various drugs, the 3435 T allele has been associated with increased drug exposure [[24](#R24), [25](#R25)], decreased drug exposure [[26](#R26)], or no effect [[24](#R24), [27](#R27)–[30](#R30)].

The NR1I2 gene is activated by various drugs to induce transcription of *CYP3A4*, *CYP3A5, ABCB1*, *SLCO1B1* and other genes involved in drug disposition [[31](#R31)–[33](#R33)]. It has been suggested that promoter and intron 1 polymorphisms of *NR1I2* are associated with variable hepatic CYP3A4 activity [[34](#R34)].

In AIDS Clinical Trials Group (ACTG) protocol A5146, patients with persistent plasma viremia despite being prescribed a PI-containing regimen were enrolled to test whether PI therapeutic drug monitoring (TDM) followed by dosage escalation would improve virologic response [[35](#R35)]. The present analyses characterize associations between *ABCB1*, *CYP3A4, CYP3A5*, *NR1I2* and *SLCO1B1* polymorphisms and trough PI concentrations before dosage escalation in A5146. In a small subset of subjects who underwent dosage escalation and who had available data, we also explore associations with post-escalation change in trough PI concentrations, and with post-escalation virologic response.

## Materials and Methods

### Study Participants

These analyses include data on ACTG study A5146 participants [[35](#R35)] who had DNA obtained under genetic consent protocol A5128 [[36](#R36)]. The design of A5146 is described elsewhere [[35](#R35)]. Briefly, eligible participants were HIV-infected and at screening had virologic failure on at least one antiretroviral regimen that contained a PI, plasma HIV-1 RNA ≥1000 copies/mL, and a virtual phenotype resistance test (vircoTYPE HIV-1, Janssen Diagnostics) showing resistance to at least one drug on the failing regimen. A new PI-containing regimen was begun at study entry. Recruitment was from June 2002 through May 2006 at 45 ACTG sites in the US and Puerto Rico. Institutional review boards approved the protocol, and all subjects provided written informed consent. The Vanderbilt Committee for the Protection of Human Subjects and ACTG approved this use of DNA.

### Normalized Inhibitory Quotient

Protocol A5146 used a normalized inhibitory quotient (NIQ, an index of the ratio of measured plasma drug concentration to measured susceptibility of the patient’s virus to that drug) to guide dosage adjustment as described elsewhere [[35](#R35)]. Subjects with NIQs ≤1 within 2 weeks after initiating a new PI-containing regimen were randomized at week 4 to PI dose escalation with subsequent TDM, or to the standard of care. Trough concentrations were measured 2 and 6 weeks after randomization in both arms. Clinical and laboratory evaluations were performed at study entry and 2 weeks later; at randomization; and 2, 6, 12, 16, and 20 weeks after randomization. Timed plasma PI trough concentrations were obtained 2 weeks after study entry (the primary endpoint of the present analysis), and 2 and 6 weeks after randomization (10–14 h after dosing for twice-daily PIs; 22–26 h for once-daily atazanavir). Protease inhibitor concentrations were measured using HPLC. Subjects initiated protocol-specified doses of PIs, given with low-dose ritonavir to boost plasma concentrations. Dual PI regimens with no known adverse pharmacokinetic interactions were allowed. Dose escalation algorithms were developed for each regimen.

### Identification of Genetic Polymorphisms

Laboratory personnel with no knowledge of clinical data performed genotyping. A total of 143 polymorphisms (1 in *SLCO1B1*, 33 in *CYP3A4*, 1 in *CYP3A5*, 43 in *NR1I2*, and 65 in *ABCB1*) were assayed using MassARRAY® iPLEX Gold (Sequenom, Inc.). For *CYP3A4*, *CYP3A5*, *ABCB1,* and *NR1I2* we tagged each gene entirely using SeattleSNPs [[37](#R37)], including 20 kB in each 5′ and 3′ untranslated regions (UTR), using a cosmopolitan strategy across populations (Yoruba, Asian, African-American, European-American, and Hispanic) with a 5% allelic frequency cut-off, a 0.80 threshold for r^2^, 85% data convergence for tagging polymorphisms, and 70% data convergence for clustering. Based on putative functional associations we also included *ABCB1* 3435C→T (rs1045642), 2677G→T/A (rs2032582), and *CYP3A5* 6986A→G (rs776746). All genotypes were confirmed by visual inspection of plots. TaqMan^™^ genotyping of *SLCO1B1* 521T→C (rs11045819) was with ABI PRISM 7900 HT Sequence Detection System (Applied Biosystems, Inc., Foster City, CA). Ample duplicate and blank assays were included to assure validity.

### Statistical Analyses

These analyses comprise two components: 1) pharmacokinetic association analyses involving all subjects with week 2 trough plasma PI assay data from A5146, and 2) virologic response association analyses involving subjects randomized to TDM. The self-identified race/ethnicity categories “white, non-Hispanic”, “black, non-Hispanic”, and “Hispanic” in accordance with National Institutes of Health standards [[38](#R38)] are hereafter referred to as white, black, and Hispanic, respectively. Because whites, blacks, and Hispanics comprised 97% of participants from A5146 who provided DNA for analysis, we limited subsequent analyses to these populations ([Table 1](#T1)).

### Table 1.

| Characteristic | Genetic analysis subjects (N=268)b | Other subjects (N=143) |
| --- | --- | --- |
| Race/Ethnicity |  |  |
| White | 140 (51%) | 44 (31%) |
| Black | 73 (27%) | 57 (40%) |
| Hispanic | 55 (21%) | 30 (21%) |
| Other | 0 (0%) | 12 (8%) |
| Male sex | 232 (87%) | 123 (86%) |
| Age in years, mean (SD) | 45.5 (7.6) | 44.0 (8.9) |
| Weight in kilograms, mean (SD) | 78 (14) | 81 (18) |
| CD4 T cells/mm3, mean (SD) | 218 (169) | 190 (175) |
| HIV-1 RNA log10 copies/mL, mean (SD) | 4.35 (0.68) | 4.42 (0.80) |
| Lopinavir-containing regimens | 102 (38%) | 28 (19%) |
| lopinavir | 63 (24%) | 10 (7%) |
| lopinavir + saquinavir | 30 (11%) | 11 (8%) |
| lopinavir + amprenavir | 5 (2%) | 2 (1%) |
| lopinavir + indinavir | 4 (1%) | 5 (3%) |
| Fos-amprenavir-containing regimens | 70 (26%) | 18 (12%) |
| fos-amprenavir | 41 (15%) | 9 (6%) |
| fos-amprenavir + saquinavir | 29 (11%) | 9 (6%) |
| Saquinavir-containing regimensc | 43 (17%) | 15 (10%) |
| saquinavir | 31 (12%) | 9 (6%) |
| saquinavir + amprenavir | 10 (4%) | 6 (4%) |
| saquinavir + indinavir | 2 (1%) | 0 (0%) |
| Other regimens |  |  |
| atazanavir | 16 (6%) | 7 (5%) |
| amprenavir | 14 (5%) | 3 (2%) |
| tipranavir | 14 (5%) | 1 (1%) |
| indinavir | 9 (3%) | 2 (1%) |
| nelfinavir | 0 (0%) | 1 (1%) |
|   | 0 (0%) | 68 (48%) |

Table 1 Caption: Baseline characteristics and protease inhibitor regimens initiated in A5146.a

Analyses were performed separately for fosamprenavir, lopinavir, and saquinavir, where each was boosted with ritonavir. Other PIs infrequently prescribed in A5146 (atazanavir, indinavir, nelfinavir and tipranavir) were excluded from analyses due to small sample sizes. Genotype frequency differences between self-identified race/ethnicity groups were assessed by Fisher’s exact test. Analyses were performed separately among these three groups, and among all three groups combined and adjusting for race/ethnicity. Deviations from Hardy Weinberg equilibrium were evaluated within race/ethnicity groups by an exact test. Genetic associations with endpoints were assessed separately for target polymorphisms *SLCO1B1* rs4149056, *CYP3A5* rs776746, *ABCB1* rs1045642, and *ABCB1* rs2032582. All statistical tests were 2-sided with a significance level of 0.05. Nominal p values are presented for target polymorphisms without adjustment for multiple comparisons. Analyses of the remaining 139 exploratory polymorphisms utilized the Benjamini Hochberg procedure [[39](#R39)] to control for the overall false discovery rate (FDR), and were considered significant if FDR-adjusted p values were ≤0.05.

Genetic associations with week 2 trough PI concentration, and with change in trough PI concentration following dosage adjustment, were assessed a nonparametric Jonckheere-Terpstra test to assess for an ordered difference in clearance by the number of variant alleles. Trough concentrations, as a function of clearance, were assumed to be log normally distributed and were log transformed. Considering the small sample sizes in genotype groups within each race/ethnicity group, a Monte Carlo enumeration method was used to obtain exact p values. Analysis of the association of change in trough PI concentration with change from baseline in plasma HIV-1 RNA were restricted to subjects who were randomized to undergo dosage adjustment based on therapeutic drug monitoring (TDM arm). Genetic associations with change in plasma HIV-1 RNA from randomization to week 20 were compared by Gehan-Wilcoxon test.

Additional secondary analysis applied linear regression to natural log-transformed week 2 trough PI concentrations across populations, adjusting for race/ethnicity, and for use of concomitant PIs other than ritonavir. We also performed analyses among subjects receiving no concomitant PI other than ritonavir. Genetic associations were assumed to be linear (additive) on the log-transformed value.

In *post-hoc* analysis, we evaluated genetic associations with week 2 trough ritonavir concentrations. Because 86% of the ritonavir measures were below the assay limit of quantification (200 ng/ml), we treated this as a binary variable, either above or below the limit of quantification. For ritonavir, genetic associations were compared within each race/ethnicity group for each PI with a Fisher’s exact test, and overall with logistic regression adjusting for race/ethnicity and concomitant PI status.

Linkage disequilibrium plots and r^2^ values were generated by Haploview statistical software. All other analyses were performed using SAS version 9.2.

## Results

### Study Population

A total of 411 subjects initiated PI-containing regimens in A5146. A subset of 350 consented for genetic analyses, of which 328 had trough PI assay data at week 2, and 268 (65% of 411) whites, blacks, or Hispanics had both week 2 trough PI assay and genetic assay results. Baseline characteristics of study participants and PI regimens initiated at study entry are shown in [Table 1](#T1). Subjects from A5146 who participated in pharmacogenetic analyses were generally similar to other A5146 participants. We hereafter focus on genetic analysis participants unless otherwise specified. Approximately one-half of subjects included in these analyses were white. The most frequently initiated PIs were lopinavir, fosamprenavir, and saquinavir. Of 105 lopinavir recipients, 65 (62%) received no additional PI other than ritonavir. Of 72 fosamprenavir recipients, 43 (60%) received no additional PI other than ritonavir. Among fosamprenavir recipients, doses were fosamprenavir 700 mg every 12 hours with ritonavir 100 mg every 12 hours. Of 103 saquinavir recipients, only 31 (30%) received no additional PI other than ritonavir.

### Characterization of genetic polymorphisms

We assayed 143 polymorphisms in *CYP3A4*, *CYP3A5*, *NR1I2*, *ABCB1*, and *SLCO1B1*. A total of 107 polymorphisms differed in frequency by race/ethnicity groups at p<0.05 (unadjusted for multiple comparisons). A total of 18 polymorphisms deviated from Hardy Weinberg equilibrium in at least one race/ethnicity group at p < 0.05, suggesting possible non-random assortment of alleles. However, this should be interpreted with caution given the large number of tests. Linkage disequilibrium (LD) plots for *CYP3A4/CYP3A5*, *NR1I2*, and *ABCB1* among whites, blacks, and Hispanics are provided in Figure, [Supplemental Digital Content 1](#SD1), [http://links.lww.com/TDM/A43](http://links.lww.com/TDM/A43).

### Associations with week 2 PI trough concentrations

Targeted analyses focused on four polymorphisms with *a priori* rationale for associations with PI pharmacokinetics. All univariate association summaries between *SLCO1B1* 521T→C, *CYP3A5*3* and trough PI concentrations among whites, blacks, and Hispanics analyzed separately are presented in [Table 2](#T2). Summaries for *ABCB1* 3435C→T and 2677G→T/A are included in Table, [Supplemental Digital Content 2](#SD2), [http://links.lww.com/TDM/A44](http://links.lww.com/TDM/A44). The *SLCO1B1* 521T→C polymorphism was most frequent among whites, in whom the C allele was significantly associated with increased trough lopinavir concentrations (p=0.01). Although only 1/31 black subject and 1/17 Hispanic subjects receiving lopinavir were heterozygous for *SLCO1B1* 521T→C, these subjects also had among the highest trough lopinavir concentrations. Relationships between *SLCO1B1* 521T→C and trough lopinavir concentrations are shown in [Figure 1](#F1) (top). In contrast, among fosamprenavir recipients the *SLCO1B1* 521 C allele was significantly associated with lower trough amprenavir concentrations in whites (p=0.03). (Fosamprenavir is a prodrug rapidly converted to amprenavir *in vivo*). Although only 3 Hispanic subjects were heterozygous for *SLCO1B1* 521T→C, these subjects also tended to have lower trough amprenavir concentrations (p=0.10). This association was not apparent in blacks, in whom only one subject carried a C allele ([Figure 1](#F1), middle). There was no apparent association between *SLCO1B1* 521T→C and trough saquinavir concentrations ([Figure 1](#F1), bottom).

### Table 2.

| Polymorphism (Drug) | Genotype | White |  |  | Black |  |  | Hispanic |  |  | Linear Regression a |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| n | median Level (ln ng/mL) | P-valueb | n | Median Level (ln ng/mL) | P-valueb | n | Median Level (ln ng/mL) | P-valueb | Geometric mean ratio c | 95% confidence interval | P-value |  |  |
| SLCO1B1 |  |  |  |  |  |  |  |  |  |  |  |  |  |
| rs4149056 |  |  |  |  |  |  |  |  |  |  |  |  |  |
| lopinavir | CC | 2 | 9.15 | 0.01 | 0 | n/a | 0.19 | 0 | n/a | 0.1 | 1.38 | [0.77, 1.15] | 0.07 |
|   | CT | 16 | 8.85 |   | 1 | 9.28 |   | 1 | 9.56 |   |   |   |   |
|   | TT | 32 | 8.54 |   | 30 | 8.69 |   | 16 | 8.66 |   |   |   |   |
| fos-amprenavir | CC | 0 | n/a | 0.03 | 0 | n/a | 0.59 | 0 | n/a | 0.1 | 0.65 | [0.44, 0.94] | 0.02 |
|   | CT | 7 | 7.26 |   | 1 | 7.73 |   | 3 | 7.15 |   |   |   |   |
|   | TT | 30 | 7.65 |   | 16 | 7.61 |   | 12 | 7.70 |   |   |   |   |
| saquinavir | CC | 0 | n/a | 0.98 | 0 | n/a | 0.88 | 0 | n/a | 0.61 | 1.25 | [0.77, 2.03] | 0.37 |
|   | CT | 13 | 6.29 |   | 1 | 6.46 |   | 5 | 6.42 |   |   |   |   |
|   | TT | 35 | 6.46 |   | 24 | 5.99 |   | 21 | 6.44 |   |   |   |   |
| CYP3A5 |  |  |  |  |  |  |  |  |  |  |  |  |  |
| rs776746 |  |  |  |  |  |  |  |  |  |  |  |  |  |
| lopinavir | AA | 0 | n/a | 0.03 | 18 | 8.78 | 0.45 | 2 | 8.65 | 0.9 | 0.94 | [0.77, 1.15] | 0.53 |
|   | AG | 9 | 8.40 |   | 12 | 8.56 |   | 6 | 8.53 |   |   |   |   |
|   | GG | 43 | 8.84 |   | 1 | 8.88 |   | 10 | 8.66 |   |   |   |   |
| fos-amprenavir | AA | 1 | 8.15 | 0.23 | 10 | 7.61 | 0.61 | 0 | n/a | 0.46 | 0.99 | [0.82, 1.21] | 0.96 |
|   | AG | 3 | 7.02 |   | 5 | 7.46 |   | 7 | 7.62 |   |   |   |   |
|   | GG | 34 | 7.65 |   | 2 | 7.72 |   | 8 | 7.64 |   |   |   |   |
| saquinavir | AA | 0 | n/a | 0.87 | 0 | n/a | 0.91 | 0 | n/a | 0.38 | 1.25 | [0.70, 1.15] | 0.37 |
|   | AG | 13 | 6.29 |   | 1 | 6.46 |   | 5 | 6.42 |   |   |   |   |
|   | GG | 35 | 6.46 |   | 24 | 5.99 |   | 21 | 6.44 |   |   |   |   |

Table 2 Caption: Associations between targeted polymorphisms and week 2 plasma trough protease inhibitor concentrations.

### Figure 1. Associations between SLCO1B1 genotype and trough protease inhibitor concentrations at week 2.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8ed1/3603284/772a2dca5e27/nihms432280f1.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3603284_nihms432280f1.jpg)

Each subject contributed a single datapoint to the figure within each drug. Horizontal lines represent medians.

The *CYP3A5*3* loss-of-function G allele was far more frequent among whites than among blacks. It was associated with higher trough lopinavir concentrations among whites (p=0.03), although no whites were homozygous for the A allele. There was no significant association between *CYP3A5*3* genotypes and trough lopinavir concentrations among blacks or Hispanics ([Figure 2](#F2), top), and there were no significant associations between *CYP3A5**3 and trough drug concentrations of amprenavir or saquinavir in any race/ethnicity group ([Figure 2](#F2), middle and bottom).

### Figure 2. Associations between CYP3A5 genotype and trough protease inhibitor concentrations at week 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8ed1/3603284/8d5dda7d8ba8/nihms432280f2.jpg)

Each subject contributed a single datapoint to the figure within each drug. Horizontal lines represent medians.

Regarding *ABCB1* 3435C→T and 2677G→T/A, there was no consistent relationship identified with trough drug concentrations for lopinavir, fosamprenavir, or saquinavir. There appeared to be associations between *ABCB1* rs2032582 and trough amprenavir concentrations among whites (p = 0.02) and among Hispanics (P = 0.03), but in opposite directions, with the G allele associated with lower amprenavir concentrations in whites but higher concentrations in blacks (data not shown).

To further assess these relationships we performed linear regression secondary analyses on log-transformed trough drug concentration among all participants, adjusting for race/ethnicity and concomitant PI other than ritonavir. In these analyses, *SLCO1B1* 521T→C still appeared to be associated with higher trough lopinavir concentrations (p = 0.07), and lower trough amprenavir concentrations (p = 0.02). There were no other apparent associations ([Table 2](#T2)) restricting analysis to the smaller subset of subjects who received no concomitant PI other than ritonavir. In these analyses among whites, *SLCO1B1* 521T→C remained associated with increased lopinavir concentrations (p = 0.02) and decreased amprenavir concentrations (p=0.05), and *CYP3A5*3* with somewhat increased lopinavir concentrations (p=0.07) (data not shown). There were no other apparent associations.

Associations with trough PI concentrations were explored for the other 139 genetic polymorphisms, using the same approach as for the targeted polymorphisms. [Supplemental Online Table 1](#SD1) presents all nominal p-values for univariate associations between each polymorphism and trough PI concentrations among whites, blacks, and Hispanics analyzed separately, as well as regression parameter estimates and nominal p-values for linear regression analyses on log-transformed trough drug concentration among all participants, adjusting for race/ethnicity and concomitant PI other than ritonavir. In these analyses no significant associations were observed when controlling for the false discovery rate.

Associations between *SLCO1B1* 521T→C and trough PI concentrations could be mediated indirectly through effects of 521T→C on the hepatic disposition of ritonavir. We therefore *post-hoc* explored relationships between *SLCO1B1* 521T→C and week 2 trough ritonavir concentrations. These analyses were limited considerably by missing data for ritonavir, as collection of such data was not required in A5146, and many results were below the limit of assay detection. However, among 127 subjects with week 2 trough ritonavir data (67 white, 29 black, 31 Hispanic), there was no significant association between *SLCO1B1* 521T→C and the probability of having ritonavir trough concentration above the limit of assay detection (all p values ≥0.28).

### Association Analyses of Post Randomization Data in the TDM arm

In A5146, 92 (22.4%) of 411 participants qualified for randomization with an NIQ ≤ 1 and were randomized and underwent dose escalation based on NIQ data [[35](#R35)]. As previously reported, subjects in all three study arms of A5146 (standard of care, TDM, and observational) had similar week 2 PI trough concentrations. Approximately three-fourths of subjects in the TDM arm underwent PI dose escalation at week 4, which generally conferred increased PI trough concentrations and increased NIQs compared to the standard of care arm. Despite achieving higher NIQs, however, this did not confer benefit with regard to week 24 virologic response.

Only 3 of 5 possible race/ethnicity and drug combination groups had at least 10 evaluable participants for such genetic association analyses (18 white subjects on saquinavir, 10 Hispanic subjects on saquinavir, and 13 white subjects on fosamprenavir). There was no significant association between any polymorphism and virologic response in these three groups (all p values ≥0.15). In these same subjects who underwent dose escalation, we also explored associations between polymorphisms and change in trough PI concentrations from week 2 to week 10, and also found no significant associations (all p values ≥ 0.10).

## Discussion

The present study characterized relationships between human genetic polymorphisms and trough drug concentrations among treatment experienced HIV-infected subjects who initiated a new ritonavir-boosted PI containing regimen after virologic failure. A provocative finding was that, among white subjects receiving ritonavir-boosted fosamprenavir, *SLCO1B1* T→C was associated with *decreased* plasma amprenavir trough concentrations, albeit with moderate evidence, given that 4 targeted SNPs were assessed without multiple comparison adjustment. This finding was unexpected, since this polymorphism was associated with *increased* plasma steady-state lopinavir exposure, consistent with previous reports [[3](#R3), [9](#R9), [10](#R10), [11](#R11)]. We found no significant association between this polymorphism and saquinavir trough concentrations.

The drug uptake transporter OATP1B1 is localized to the sinusoidal membrane of hepatocytes, where it facilitates uptake of drugs into hepatocytes to undergo intracellular metabolism and biliary excretion [[8](#R8)]. It has been shown to transport various drugs *in vitro* [[40](#R40)], including the PIs lopinavir, saquinavir and darunavir [[3](#R3)]. There is ample evidence that *SLCO1B1* 521T→C is associated with functional effects. In addition to lopinavir concentrations [[3](#R3), [9](#R9), [10](#R10), [11](#R11)], *SLCO1B1* 521T→C has also been associated with increased plasma exposure of numerous other drugs including the statin drugs pravastatin [[41](#R41)], pitavastatin [[42](#R42)], simvastatin [[43](#R43)] atorvastatin [[44](#R44)] and rosuvastatin [[45](#R45)], the anti-diabetes drug repaglinide [[46](#R46)], the anti-cancer drugs atrasentan [[47](#R47)] and irinotecan [[48](#R48)], and the lipid-lowering drug ezetimibe [[49](#R49)].

The allelic frequency of *SLCO1B1* 521T→C varies considerably depending upon geographic region of ancestry. Frequencies of this polymorphism in the present study (15%, 1%, and 8% in self-identified whites, blacks, and Hispanics, respectively) were consistent with what has been previously described. With European ancestry its reported allelic frequency has been 12–20% (homozygosity in approximately 1–2%), whereas with African ancestry its allelic frequency has been only 1–4% (homozygosity extremely rare) [[40](#R40)]. Among Mexicans in HapMap phase 3 its allelic frequency was 8% (homozygosity rare) [[12](#R12)]. This polymorphism may therefore be most likely to have clinical relevance among individuals of European descent, and least likely among individuals of African descent.

At present, while this may be a spurious association, we can only speculate as to the mechanism for the opposite directional association between *SLCO1B1* 521T→C and plasma trough amprenavir and lopinavir concentrations. One possible explanation is that decreased transporter activity of *SLCO1B1* 521T→C has multiple effects. Ritonavir boosts plasma concentrations of PIs primarily by competitively inhibiting hepatic CYP3A4, the enzyme primary responsible for metabolism of PIs and many other drugs. By decreasing hepatocyte uptake of substrate drugs, 521T→C may directly maintain higher plasma drugs concentrations by keeping drug within hepatic sinusoids, both during first pass from portal to hepatic venules and during subsequent systemic clearance as drug passes from hepatic arterioles to venules. In addition, by decreasing intracellular drug concentrations in hepatocytes, 521T→C may alter the balance between CYP3A induction and inhibition. It may be that *SLCO1B1* 521T→C reduces intracellular concentrations of ritonavir, which for lopinavir may still be ample for effective boosting because ritonavir so effectively competes with lopinavir for CYP3A4 (ritonavir and lopinavir have almost identical structures), and because lopinavir is an excellent substrate for OATP1B1 [[3](#R3)]. In contrast, reduced intracellular concentrations of ritonavir due to *SLCO1B1* 521T→C may cause less effective inhibition of CYP3A4 toward amprenavir. Furthermore, amprenavir is a potent inducer of CYP3A4. It is not known whether amprenavir is a good substrate for OATP1B1. We hypothesize that potent induction of CYP3A4 by amprenavir, decreased ritonavir-mediated inhibition of CYP3A4, and low affinity of OATP1B1 for amprenavir combine to result in increased clearance of amprenavir. Our findings need to be replicated in other studies, ideally with robust ritonavir assay data which were limited in our dataset.

The present study has several implications. Drug-drug interactions that involve drugs that may both induce and inhibit enzymes and transporters involved in drug disposition may be complex and *a priori* unpredictable, depending on the balance between induction and inhibition. It is therefore not surprising that some drug-drug interactions may differ qualitatively depending on the presence of polymorphisms that affect drug disposition of at least one of the interacting drugs. Although ritonavir-boosted fosamprenavir is now infrequently prescribed, the present study suggests that plasma trough amprenavir concentrations with this combination are reduced with *SLCO1B1* 521T→C. Importantly, this finding was not predicted by prior studies of this variant. Studies designed to characterize pharmacokinetic interactions between drugs known to be affected by genetic polymorphisms should ideally address whether such polymorphisms substantially alter the drug-drug interaction.

There were some important limitations to the present study. The small sample sizes limited statistical power to detect true and possibly clinically meaningful genetic associations. The provocative association between *SLCO1B1* 521T→C and reduced plasma amprenavir trough concentrations is of moderate evidence, may be spurious and needs to be replicated in other cohorts, in particular those with larger non-white representation. Only a small subset of A5146 subjects with pharmacokinetic data from the PI initiation step of the study were randomized to undergo dose escalation, so the relevance of these polymorphisms for virologic response could not be adequately addressed.

In summary, among subjects prescribed ritonavir-boosted PIs, the *SLCO1B1* 521T→C polymorphism that is associated with higher lopinavir trough concentrations appears to be associated with lower amprenavir trough concentrations. Predicting PI interactions with ritonavir may be informed by considering pharmacogenetics. We found no significant association between any polymorphism and virologic response to TDM-guided dose escalation.

## Supplementary Material

## Acknowledgments

The authors are grateful to the many persons with HIV infection who volunteered for A5146 and A5128. In addition, they acknowledge the contributions of many ACTG personnel involved in protocols A5146 and A5128.

Financial Support:

Grant support for the authors included AI069439, RR000095, AI54999 (DWH), AI38858, AI68634 (XZ, CT), NS32228, AI25903 (DBC), BRS-ACURE-06-00140-T001 (GDM), AI69419 (RMG), AI51966 (RMG, BRS), AI069472, AI062435 (GKR). The project described was supported by Award Number U01AI068636 from the National Institute of Allergy and Infectious Diseases and supported by National Institute of Mental Health (NIMH), National Institute of Dental and Craniofacial Research (NIDCR). Protocol A5146 was supported by Award Number U01AI068636 from the National Institute of Allergy and Infectious Diseases, as well as previous grants AI38858 and AI68634. Clinical Research Sites that participated in ACTG protocols A5146 and collected DNA under protocol A5128, were supported by the following NIH grants: AI069428, AI069484, AI069494, A1069447, AI69501, AI069471, AI069415, AI069474, Al27665; Al069532, AI069513, AI46370, U01AI069511, RR00044, AI69450, AI069419, RR024996, AI-69501, AI069472, AI69432, AI069502, AI069439, AI69418, AI050409, AI69467, A1069424, AI069495, AI069477, AI073961, AI27661, AI069452, AI069423, RR025747, AI050410, AI069434, AI27757, AI069470, AI069556, AI069511, RR00044. Janssen Diagnostics provided virtual phenotype resistance testing in A5146.

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Lee CG, Gottesman MM, Cardarelli CO, et al. HIV-1 protease inhibitors are substrates for the MDR1 multidrug transporter. Biochemistry. 1998;37:3594–3601. doi: 10.1021/bi972709x.  [DOI](https://doi.org/10.1021/bi972709x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9530286/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochemistry&title=HIV-1%20protease%20inhibitors%20are%20substrates%20for%20the%20MDR1%20multidrug%20transporter&author=CG%20Lee&author=MM%20Gottesman&author=CO%20Cardarelli&volume=37&publication_year=1998&pages=3594-3601&pmid=9530286&doi=10.1021/bi972709x&)

2. Kim RB, Fromm MF, Wandel C, et al. The drug transporter P-glycoprotein limits oral absorption and brain entry of HIV-1 protease inhibitors. J Clin Invest. 1998;101:289–294. doi: 10.1172/JCI1269.  [DOI](https://doi.org/10.1172/JCI1269) | [PMC free article](/articles/PMC508566/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9435299/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Invest&title=The%20drug%20transporter%20P-glycoprotein%20limits%20oral%20absorption%20and%20brain%20entry%20of%20HIV-1%20protease%20inhibitors&author=RB%20Kim&author=MF%20Fromm&author=C%20Wandel&volume=101&publication_year=1998&pages=289-294&pmid=9435299&doi=10.1172/JCI1269&)

3. Hartkoorn RC, Kwan WS, Shallcross V, et al. HIV protease inhibitors are substrates for OATP1A2, OATP1B1 and OATP1B3 and lopinavir plasma concentrations are influenced by SLCO1B1 polymorphisms. Pharmacogenet Genom. 2010;20:112–120. doi: 10.1097/FPC.0b013e328335b02d.  [DOI](https://doi.org/10.1097/FPC.0b013e328335b02d) | [PMC free article](/articles/PMC4859410/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20051929/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genom&title=HIV%20protease%20inhibitors%20are%20substrates%20for%20OATP1A2,%20OATP1B1%20and%20OATP1B3%20and%20lopinavir%20plasma%20concentrations%20are%20influenced%20by%20SLCO1B1%20polymorphisms&author=RC%20Hartkoorn&author=WS%20Kwan&author=V%20Shallcross&volume=20&publication_year=2010&pages=112-120&pmid=20051929&doi=10.1097/FPC.0b013e328335b02d&)

4. Luo G, Cunningham M, Kim S, et al. CYP3A4 induction by drugs: correlation between a pregnane X receptor reporter gene assay and CYP3A4 expression in human hepatocytes. Drug Metab Disposit. 2002;30:795–804. doi: 10.1124/dmd.30.7.795.  [DOI](https://doi.org/10.1124/dmd.30.7.795) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12065438/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Disposit&title=CYP3A4%20induction%20by%20drugs:%20correlation%20between%20a%20pregnane%20X%20receptor%20reporter%20gene%20assay%20and%20CYP3A4%20expression%20in%20human%20hepatocytes&author=G%20Luo&author=M%20Cunningham&author=S%20Kim&volume=30&publication_year=2002&pages=795-804&pmid=12065438&doi=10.1124/dmd.30.7.795&)

5. Huang L, Wring SA, Woolley JL, Brouwer KR, Serabjit-Singh C, Polli JW. Induction of P-glycoprotein and cytochrome P450 3A by HIV protease inhibitors. Drug Metab Disposit. 2001;29:754–60.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/11302944/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Disposit&title=Induction%20of%20P-glycoprotein%20and%20cytochrome%20P450%203A%20by%20HIV%20protease%20inhibitors&author=L%20Huang&author=SA%20Wring&author=JL%20Woolley&author=KR%20Brouwer&author=C%20Serabjit-Singh&volume=29&publication_year=2001&pages=754-60&pmid=11302944&)

6. Gupta A, Mugundu GM, Desai PB, Thummel KE, Unadkat JD. Intestinal human colon adenocarcinoma cell line LS180 is an excellent model to study pregnane X receptor, but not constitutive androstane receptor, mediated CYP3A4 and multidrug resistance transporter 1 induction: studies with anti-human immunodeficiency virus protease inhibitors. Drug Metab Disposit. 2008;36:1172–1180. doi: 10.1124/dmd.107.018689.  [DOI](https://doi.org/10.1124/dmd.107.018689) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18332086/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Disposit&title=Intestinal%20human%20colon%20adenocarcinoma%20cell%20line%20LS180%20is%20an%20excellent%20model%20to%20study%20pregnane%20X%20receptor,%20but%20not%20constitutive%20androstane%20receptor,%20mediated%20CYP3A4%20and%20multidrug%20resistance%20transporter%201%20induction:%20studies%20with%20anti-human%20immunodeficiency%20virus%20protease%20inhibitors&author=A%20Gupta&author=GM%20Mugundu&author=PB%20Desai&author=KE%20Thummel&author=JD%20Unadkat&volume=36&publication_year=2008&pages=1172-1180&pmid=18332086&doi=10.1124/dmd.107.018689&)

7. Dussault I, Lin M, Hollister K, Wang EH, Synold TW, Forman BM. Peptide mimetic HIV protease inhibitors are ligands for the orphan receptor SXR. J Biol Chem. 2001;276:33309–33312. doi: 10.1074/jbc.C100375200.  [DOI](https://doi.org/10.1074/jbc.C100375200) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11466304/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=Peptide%20mimetic%20HIV%20protease%20inhibitors%20are%20ligands%20for%20the%20orphan%20receptor%20SXR&author=I%20Dussault&author=M%20Lin&author=K%20Hollister&author=EH%20Wang&author=TW%20Synold&volume=276&publication_year=2001&pages=33309-33312&pmid=11466304&doi=10.1074/jbc.C100375200&)

8. Niemi M. Role of OATP transporters in the disposition of drugs. Pharmacogenomics. 2007;8:787–802. doi: 10.2217/14622416.8.7.787.  [DOI](https://doi.org/10.2217/14622416.8.7.787) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18240907/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Role%20of%20OATP%20transporters%20in%20the%20disposition%20of%20drugs&author=M%20Niemi&volume=8&publication_year=2007&pages=787-802&pmid=18240907&doi=10.2217/14622416.8.7.787&)

9. Lubomirov R, di Iulio J, Fayet A, et al. ADME pharmacogenetics: investigation of the pharmacokinetics of the antiretroviral agent lopinavir coformulated with ritonavir. Pharmacogenet Genom. 2010;20:217–230. doi: 10.1097/FPC.0b013e328336eee4.  [DOI](https://doi.org/10.1097/FPC.0b013e328336eee4) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20139798/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genom&title=ADME%20pharmacogenetics:%20investigation%20of%20the%20pharmacokinetics%20of%20the%20antiretroviral%20agent%20lopinavir%20coformulated%20with%20ritonavir&author=R%20Lubomirov&author=J%20di%20Iulio&author=A%20Fayet&volume=20&publication_year=2010&pages=217-230&pmid=20139798&doi=10.1097/FPC.0b013e328336eee4&)

10. Kohlrausch FB, de Cassia Estrela R, Barroso PF, Suarez-Kurtz G. The impact of SLCO1B1 polymorphisms on the plasma concentration of lopinavir and ritonavir in HIV-infected men. Br J Clin Pharm. 2010;69:95–98. doi: 10.1111/j.1365-2125.2009.03551.x.  [DOI](https://doi.org/10.1111/j.1365-2125.2009.03551.x) | [PMC free article](/articles/PMC2830602/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20078617/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Clin%20Pharm&title=The%20impact%20of%20SLCO1B1%20polymorphisms%20on%20the%20plasma%20concentration%20of%20lopinavir%20and%20ritonavir%20in%20HIV-infected%20men&author=FB%20Kohlrausch&author=R%20de%20Cassia%20Estrela&author=PF%20Barroso&author=G%20Suarez-Kurtz&volume=69&publication_year=2010&pages=95-98&pmid=20078617&doi=10.1111/j.1365-2125.2009.03551.x&)

11. Rakhmanina NY, Neely MN, Van Schaik RH, et al. CYP3A5, ABCB1, and SLCO1B1 polymorphisms and pharmacokinetics and virologic outcome of lopinavir/ritonavir in HIV-infected children. Therap Drug Monitor. 2011;33:417–424. doi: 10.1097/FTD.0b013e318225384f.  [DOI](https://doi.org/10.1097/FTD.0b013e318225384f) | [PMC free article](/articles/PMC3139013/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21743379/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Therap%20Drug%20Monitor&title=CYP3A5,%20ABCB1,%20and%20SLCO1B1%20polymorphisms%20and%20pharmacokinetics%20and%20virologic%20outcome%20of%20lopinavir/ritonavir%20in%20HIV-infected%20children&author=NY%20Rakhmanina&author=MN%20Neely&author=RH%20Van%20Schaik&volume=33&publication_year=2011&pages=417-424&pmid=21743379&doi=10.1097/FTD.0b013e318225384f&)

12. dbSNP - Short Genetic Variations. Available at: http://www.ncbi.nlm.nih.gov/projects/SNP/  [http://www.ncbi.nlm.nih.gov/projects/SNP/](http://www.ncbi.nlm.nih.gov/projects/SNP/)

13. Williams JA, Ring BJ, Cantrell VE, et al. Comparative metabolic capabilities of CYP3A4, CYP3A5, and CYP3A7. Drug Metab Disposit. 2002;30:883–891. doi: 10.1124/dmd.30.8.883.  [DOI](https://doi.org/10.1124/dmd.30.8.883) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12124305/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Disposit&title=Comparative%20metabolic%20capabilities%20of%20CYP3A4,%20CYP3A5,%20and%20CYP3A7&author=JA%20Williams&author=BJ%20Ring&author=VE%20Cantrell&volume=30&publication_year=2002&pages=883-891&pmid=12124305&doi=10.1124/dmd.30.8.883&)

14. Kuehl P, Zhang J, Lin Y, et al. Sequence diversity in CYP3A promoters and characterization of the genetic basis of polymorphic CYP3A5 expression. Nat Genet. 2001;27:383–391. doi: 10.1038/86882.  [DOI](https://doi.org/10.1038/86882) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11279519/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Sequence%20diversity%20in%20CYP3A%20promoters%20and%20characterization%20of%20the%20genetic%20basis%20of%20polymorphic%20CYP3A5%20expression&author=P%20Kuehl&author=J%20Zhang&author=Y%20Lin&volume=27&publication_year=2001&pages=383-391&pmid=11279519&doi=10.1038/86882&)

15. Hustert E, Haberl M, Burk O, et al. The genetic determinants of the CYP3A5 polymorphism. Pharmacogenetics. 2001;11:773–779. doi: 10.1097/00008571-200112000-00005.  [DOI](https://doi.org/10.1097/00008571-200112000-00005) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11740341/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=The%20genetic%20determinants%20of%20the%20CYP3A5%20polymorphism&author=E%20Hustert&author=M%20Haberl&author=O%20Burk&volume=11&publication_year=2001&pages=773-779&pmid=11740341&doi=10.1097/00008571-200112000-00005&)

16. Anderson PL, Aquilante CL, Gardner EM, et al. Atazanavir pharmacokinetics in genetically determined CYP3A5 expressors versus non-expressors. J Antimicrob Chemother. 2009;64:1071–1079. doi: 10.1093/jac/dkp317.  [DOI](https://doi.org/10.1093/jac/dkp317) | [PMC free article](/articles/PMC2760462/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19710077/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Antimicrob%20Chemother&title=Atazanavir%20pharmacokinetics%20in%20genetically%20determined%20CYP3A5%20expressors%20versus%20non-expressors&author=PL%20Anderson&author=CL%20Aquilante&author=EM%20Gardner&volume=64&publication_year=2009&pages=1071-1079&pmid=19710077&doi=10.1093/jac/dkp317&)

17. Anderson PL, Lamba J, Aquilante CL, Schuetz E, Fletcher CV. Pharmacogenetic characteristics of indinavir, zidovudine, and lamivudine therapy in HIV-infected adults: a pilot study. J Acquir Immune Defic Syndr. 2006;42:441–449. doi: 10.1097/01.qai.0000225013.53568.69.  [DOI](https://doi.org/10.1097/01.qai.0000225013.53568.69) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16791115/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Acquir%20Immune%20Defic%20Syndr&title=Pharmacogenetic%20characteristics%20of%20indinavir,%20zidovudine,%20and%20lamivudine%20therapy%20in%20HIV-infected%20adults:%20a%20pilot%20study&author=PL%20Anderson&author=J%20Lamba&author=CL%20Aquilante&author=E%20Schuetz&author=CV%20Fletcher&volume=42&publication_year=2006&pages=441-449&pmid=16791115&doi=10.1097/01.qai.0000225013.53568.69&)

18. Solas C, Simon N, Drogoul MP, et al. Minimal effect of MDR1 and CYP3A5 genetic polymorphisms on the pharmacokinetics of indinavir in HIV-infected patients. Br J Clin Pharmacol. 2007;64:353–362. doi: 10.1111/j.1365-2125.2007.02903.x.  [DOI](https://doi.org/10.1111/j.1365-2125.2007.02903.x) | [PMC free article](/articles/PMC2000655/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17517050/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Clin%20Pharmacol&title=Minimal%20effect%20of%20MDR1%20and%20CYP3A5%20genetic%20polymorphisms%20on%20the%20pharmacokinetics%20of%20indinavir%20in%20HIV-infected%20patients&author=C%20Solas&author=N%20Simon&author=MP%20Drogoul&volume=64&publication_year=2007&pages=353-362&pmid=17517050&doi=10.1111/j.1365-2125.2007.02903.x&)

19. Mouly SJ, Matheny C, Paine MF, et al. Variation in oral clearance of saquinavir is predicted by CYP3A5*1 genotype but not by enterocyte content of cytochrome P450 3A5. Clin Pharmacol Therapeut. 2005;78:605–618. doi: 10.1016/j.clpt.2005.08.014.  [DOI](https://doi.org/10.1016/j.clpt.2005.08.014) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16338276/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Therapeut&title=Variation%20in%20oral%20clearance%20of%20saquinavir%20is%20predicted%20by%20CYP3A5*1%20genotype%20but%20not%20by%20enterocyte%20content%20of%20cytochrome%20P450%203A5&author=SJ%20Mouly&author=C%20Matheny&author=MF%20Paine&volume=78&publication_year=2005&pages=605-618&pmid=16338276&doi=10.1016/j.clpt.2005.08.014&)

20. Birdwell KA, Grady B, Choi L, et al. The use of a DNA biobank linked to electronic medical records to characterize pharmacogenomic predictors of tacrolimus dose requirement in kidney transplant recipients. Pharmacogenet Genom. 2012;22:32–42. doi: 10.1097/FPC.0b013e32834e1641.  [DOI](https://doi.org/10.1097/FPC.0b013e32834e1641) | [PMC free article](/articles/PMC3237759/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22108237/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genom&title=The%20use%20of%20a%20DNA%20biobank%20linked%20to%20electronic%20medical%20records%20to%20characterize%20pharmacogenomic%20predictors%20of%20tacrolimus%20dose%20requirement%20in%20kidney%20transplant%20recipients&author=KA%20Birdwell&author=B%20Grady&author=L%20Choi&volume=22&publication_year=2012&pages=32-42&pmid=22108237&doi=10.1097/FPC.0b013e32834e1641&)

21. Hoffmeyer S, Burk O, von Richter O, et al. Functional polymorphisms of the human multidrug-resistance gene: multiple sequence variations and correlation of one allele with P-glycoprotein expression and activity in vivo. Proc Natl Acad Sci USA. 2000;97:3473–3478. doi: 10.1073/pnas.050585397.  [DOI](https://doi.org/10.1073/pnas.050585397) | [PMC free article](/articles/PMC16264/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10716719/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20USA&title=Functional%20polymorphisms%20of%20the%20human%20multidrug-resistance%20gene:%20multiple%20sequence%20variations%20and%20correlation%20of%20one%20allele%20with%20P-glycoprotein%20expression%20and%20activity%20in%20vivo&author=S%20Hoffmeyer&author=O%20Burk&author=O%20von%20Richter&volume=97&publication_year=2000&pages=3473-3478&pmid=10716719&doi=10.1073/pnas.050585397&)

22. Yu DK. The contribution of P-glycoprotein to pharmacokinetic drug-drug interactions. J Clin Pharmacol. 1999;39:1203–1211. doi: 10.1177/00912709922012006.  [DOI](https://doi.org/10.1177/00912709922012006) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10586385/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Pharmacol&title=The%20contribution%20of%20P-glycoprotein%20to%20pharmacokinetic%20drug-drug%20interactions&author=DK%20Yu&volume=39&publication_year=1999&pages=1203-1211&pmid=10586385&doi=10.1177/00912709922012006&)

23. Fromm MF. P-glycoprotein: a defense mechanism limiting oral bioavailability and CNS accumulation of drugs. Int J Clin Pharmacol Therapeut. 2000;38:69–74. doi: 10.5414/cpp38069.  [DOI](https://doi.org/10.5414/cpp38069) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10706193/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Clin%20Pharmacol%20Therapeut&title=P-glycoprotein:%20a%20defense%20mechanism%20limiting%20oral%20bioavailability%20and%20CNS%20accumulation%20of%20drugs&author=MF%20Fromm&volume=38&publication_year=2000&pages=69-74&pmid=10706193&doi=10.5414/cpp38069&)

24. Drescher S, Schaeffeler E, Hitzl M, et al. MDR1 gene polymorphisms and disposition of the P-glycoprotein substrate fexofenadine. Br J Clin Pharmacol. 2002;53:526–534. doi: 10.1046/j.1365-2125.2002.01591.x.  [DOI](https://doi.org/10.1046/j.1365-2125.2002.01591.x) | [PMC free article](/articles/PMC1874364/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11994059/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Clin%20Pharmacol&title=MDR1%20gene%20polymorphisms%20and%20disposition%20of%20the%20P-glycoprotein%20substrate%20fexofenadine&author=S%20Drescher&author=E%20Schaeffeler&author=M%20Hitzl&volume=53&publication_year=2002&pages=526-534&pmid=11994059&doi=10.1046/j.1365-2125.2002.01591.x&)

25. Zhu D, Taguchi-Nakamura H, Goto M, et al. Influence of single-nucleotide polymorphisms in the multidrug resistance-1 gene on the cellular export of nelfinavir and its clinical implication for highly active antiretroviral therapy. Antiviral Ther. 2004;9:929–935.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/15651752/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Antiviral%20Ther&title=Influence%20of%20single-nucleotide%20polymorphisms%20in%20the%20multidrug%20resistance-1%20gene%20on%20the%20cellular%20export%20of%20nelfinavir%20and%20its%20clinical%20implication%20for%20highly%20active%20antiretroviral%20therapy&author=D%20Zhu&author=H%20Taguchi-Nakamura&author=M%20Goto&volume=9&publication_year=2004&pages=929-935&pmid=15651752&)

26. Rodriguez NS, Barreiro P, Rendon A, et al. Plasma levels of atazanavir and the risk of hyperbilirubinemia are predicted by the 3435C-->T polymorphism at the multidrug resistance gene 1. Clin Infect Dis. 2006;42:291–295. doi: 10.1086/499056.  [DOI](https://doi.org/10.1086/499056) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16355344/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Infect%20Dis&title=Plasma%20levels%20of%20atazanavir%20and%20the%20risk%20of%20hyperbilirubinemia%20are%20predicted%20by%20the%203435C-->T%20polymorphism%20at%20the%20multidrug%20resistance%20gene%201&author=NS%20Rodriguez&author=P%20Barreiro&author=A%20Rendon&volume=42&publication_year=2006&pages=291-295&pmid=16355344&doi=10.1086/499056&)

27. Gerloff T, Schaefer M, Johne A, et al. MDR1 genotypes do not influence the absorption of a single oral dose of 1 mg digoxin in healthy white males. Br J Clin Pharmacol. 2002;54:610–616. doi: 10.1046/j.1365-2125.2002.01691.x.  [DOI](https://doi.org/10.1046/j.1365-2125.2002.01691.x) | [PMC free article](/articles/PMC1874502/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12492608/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Clin%20Pharmacol&title=MDR1%20genotypes%20do%20not%20influence%20the%20absorption%20of%20a%20single%20oral%20dose%20of%201%20mg%20digoxin%20in%20healthy%20white%20males&author=T%20Gerloff&author=M%20Schaefer&author=A%20Johne&volume=54&publication_year=2002&pages=610-616&pmid=12492608&doi=10.1046/j.1365-2125.2002.01691.x&)

28. Pauli-Magnus C, Feiner J, Brett C, Lin E, Kroetz DL. No effect of MDR1 C3435T variant on loperamide disposition and central nervous system effects. Clin Pharmacol Therapeut. 2003;74:487–98. doi: 10.1016/S0009-9236(03)00234-0.  [DOI](https://doi.org/10.1016/S0009-9236(03)00234-0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14586389/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Therapeut&title=No%20effect%20of%20MDR1%20C3435T%20variant%20on%20loperamide%20disposition%20and%20central%20nervous%20system%20effects&author=C%20Pauli-Magnus&author=J%20Feiner&author=C%20Brett&author=E%20Lin&author=DL%20Kroetz&volume=74&publication_year=2003&pages=487-98&pmid=14586389&doi=10.1016/S0009-9236(03)00234-0&)

29. Mai I, Perloff ES, Bauer S, et al. MDR1 haplotypes derived from exons 21 and 26 do not affect the steady-state pharmacokinetics of tacrolimus in renal transplant patients. Br J Clin Pharmacol. 2004;58:548–553. doi: 10.1111/j.1365-2125.2004.02182.x.  [DOI](https://doi.org/10.1111/j.1365-2125.2004.02182.x) | [PMC free article](/articles/PMC1884628/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15521904/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Clin%20Pharmacol&title=MDR1%20haplotypes%20derived%20from%20exons%2021%20and%2026%20do%20not%20affect%20the%20steady-state%20pharmacokinetics%20of%20tacrolimus%20in%20renal%20transplant%20patients&author=I%20Mai&author=ES%20Perloff&author=S%20Bauer&volume=58&publication_year=2004&pages=548-553&pmid=15521904&doi=10.1111/j.1365-2125.2004.02182.x&)

30. Haas DW, Smeaton LM, Shafer RW, et al. Pharmacogenetics of long-term responses to antiretroviral regimens containing Efavirenz and/or Nelfinavir: an Adult Aids Clinical Trials Group Study. J Infect Dis. 2005;192:1931–1942. doi: 10.1086/497610.  [DOI](https://doi.org/10.1086/497610) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16267764/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Infect%20Dis&title=Pharmacogenetics%20of%20long-term%20responses%20to%20antiretroviral%20regimens%20containing%20Efavirenz%20and/or%20Nelfinavir:%20an%20Adult%20Aids%20Clinical%20Trials%20Group%20Study&author=DW%20Haas&author=LM%20Smeaton&author=RW%20Shafer&volume=192&publication_year=2005&pages=1931-1942&pmid=16267764&doi=10.1086/497610&)

31. Synold TW, Dussault I, Forman BM. The orphan nuclear receptor SXR coordinately regulates drug metabolism and efflux. Nat Med. 2001;7:584–590. doi: 10.1038/87912.  [DOI](https://doi.org/10.1038/87912) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11329060/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Med&title=The%20orphan%20nuclear%20receptor%20SXR%20coordinately%20regulates%20drug%20metabolism%20and%20efflux&author=TW%20Synold&author=I%20Dussault&author=BM%20Forman&volume=7&publication_year=2001&pages=584-590&pmid=11329060&doi=10.1038/87912&)

32. Meyer zu Schwabedissen HE, Kim RB. Hepatic OATP1B transporters and nuclear receptors PXR and CAR: interplay, regulation of drug disposition genes, and single nucleotide polymorphisms. Mol Pharmacol. 2009;6:1644–1661. doi: 10.1021/mp9000298.  [DOI](https://doi.org/10.1021/mp9000298) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19558188/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Pharmacol&title=Hepatic%20OATP1B%20transporters%20and%20nuclear%20receptors%20PXR%20and%20CAR:%20interplay,%20regulation%20of%20drug%20disposition%20genes,%20and%20single%20nucleotide%20polymorphisms&author=HE%20Meyer%20zu%20Schwabedissen&author=RB%20Kim&volume=6&publication_year=2009&pages=1644-1661&pmid=19558188&doi=10.1021/mp9000298&)

33. Burk O, Koch I, Raucy J, et al. The induction of cytochrome P450 3A5 (CYP3A5) in the human liver and intestine is mediated by the xenobiotic sensors pregnane X receptor (PXR) and constitutively activated receptor (CAR) J Biol Chem. 2004;10;279:38379–38385. doi: 10.1074/jbc.M404949200.  [DOI](https://doi.org/10.1074/jbc.M404949200) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15252010/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=The%20induction%20of%20cytochrome%20P450%203A5%20(CYP3A5)%20in%20the%20human%20liver%20and%20intestine%20is%20mediated%20by%20the%20xenobiotic%20sensors%20pregnane%20X%20receptor%20(PXR)%20and%20constitutively%20activated%20receptor%20(CAR)&author=O%20Burk&author=I%20Koch&author=J%20Raucy&volume=10;279&publication_year=2004&pages=38379-38385&pmid=15252010&doi=10.1074/jbc.M404949200&)

34. Lamba J, Lamba V, Strom S, Venkataramanan R, Schuetz E. Novel single nucleotide polymorphisms in the promoter and intron 1 of human pregnane X receptor/NR1I2 and their association with CYP3A4 expression. Drug Metab Dispos. 2008;36:169–181. doi: 10.1124/dmd.107.016600.  [DOI](https://doi.org/10.1124/dmd.107.016600) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17925385/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Novel%20single%20nucleotide%20polymorphisms%20in%20the%20promoter%20and%20intron%201%20of%20human%20pregnane%20X%20receptor/NR1I2%20and%20their%20association%20with%20CYP3A4%20expression&author=J%20Lamba&author=V%20Lamba&author=S%20Strom&author=R%20Venkataramanan&author=E%20Schuetz&volume=36&publication_year=2008&pages=169-181&pmid=17925385&doi=10.1124/dmd.107.016600&)

35. Demeter LM, Jiang H, Mukherjee AL, et al. A randomized trial of therapeutic drug monitoring of protease inhibitors in antiretroviral-experienced, HIV-1-infected patients. AIDS. 2009;23:357–368. doi: 10.1097/QAD.0b013e32831f9148.  [DOI](https://doi.org/10.1097/QAD.0b013e32831f9148) | [PMC free article](/articles/PMC2798571/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19114860/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=AIDS&title=A%20randomized%20trial%20of%20therapeutic%20drug%20monitoring%20of%20protease%20inhibitors%20in%20antiretroviral-experienced,%20HIV-1-infected%20patients&author=LM%20Demeter&author=H%20Jiang&author=AL%20Mukherjee&volume=23&publication_year=2009&pages=357-368&pmid=19114860&doi=10.1097/QAD.0b013e32831f9148&)

36. Haas DW, Wilkinson GR, Kuritzkes DR, et al. A multi-investigator/institutional DNA bank for AIDS-related human genetic studies: AACTG Protocol A5128. HIV Clin Trials. 2003;4:287–300. doi: 10.1310/MUQC-QXBC-8118-BPM5.  [DOI](https://doi.org/10.1310/MUQC-QXBC-8118-BPM5) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14583845/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=HIV%20Clin%20Trials&title=A%20multi-investigator/institutional%20DNA%20bank%20for%20AIDS-related%20human%20genetic%20studies:%20AACTG%20Protocol%20A5128&author=DW%20Haas&author=GR%20Wilkinson&author=DR%20Kuritzkes&volume=4&publication_year=2003&pages=287-300&pmid=14583845&doi=10.1310/MUQC-QXBC-8118-BPM5&)

37. SeattleSNPs. NHLBI Program for Genomic Applications. SeattleSNPs; Seattle, WA: Available at: pga.gs.washington.edu/  [pga.gs.washington.edu/](http://pga.gs.washington.edu/) | [Google Scholar](https://scholar.google.com/scholar_lookup?title=NHLBI%20Program%20for%20Genomic%20Applications&)

38. NIH Policy on Reporting Race and Ethnicity Data: Subjects in Clinical Research. Available at: http://www.niams.nih.gov/rtac/funding/grants/notice/notod01-053.htm.  [http://www.niams.nih.gov/rtac/funding/grants/notice/notod01-053.htm](http://www.niams.nih.gov/rtac/funding/grants/notice/notod01-053.htm)

39. Benjamini Y, Hochberg Y. Controlling the false discovery rate: a practical and powerful approach to multiple testing. J Royal Stat Soc B. 1995;57:12.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Royal%20Stat%20Soc%20B&title=Controlling%20the%20false%20discovery%20rate:%20a%20practical%20and%20powerful%20approach%20to%20multiple%20testing&author=Y%20Benjamini&author=Y%20Hochberg&volume=57&publication_year=1995&pages=12&)

40. Oshiro C, Mangravite L, Klein T, Altman R. PharmGKB very important pharmacogene: SLCO1B1. Pharmacogenet Genom. 2010;20:211–216. doi: 10.1097/FPC.0b013e328333b99c.  [DOI](https://doi.org/10.1097/FPC.0b013e328333b99c) | [PMC free article](/articles/PMC3086841/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19952871/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genom&title=PharmGKB%20very%20important%20pharmacogene:%20SLCO1B1&author=C%20Oshiro&author=L%20Mangravite&author=T%20Klein&author=R%20Altman&volume=20&publication_year=2010&pages=211-216&pmid=19952871&doi=10.1097/FPC.0b013e328333b99c&)

41. Nishizato Y, Ieiri I, Suzuki H, et al. Polymorphisms of OATP-C (SLC21A6) and OAT3 (SLC22A8) genes: consequences for pravastatin pharmacokinetics. Clin Pharmacol Therapeut. 2003;73:554–565. doi: 10.1016/S0009-9236(03)00060-2.  [DOI](https://doi.org/10.1016/S0009-9236(03)00060-2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12811365/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Therapeut&title=Polymorphisms%20of%20OATP-C%20(SLC21A6)%20and%20OAT3%20(SLC22A8)%20genes:%20consequences%20for%20pravastatin%20pharmacokinetics&author=Y%20Nishizato&author=I%20Ieiri&author=H%20Suzuki&volume=73&publication_year=2003&pages=554-565&pmid=12811365&doi=10.1016/S0009-9236(03)00060-2&)

42. Chung JY, Cho JY, Yu KS, et al. Effect of OATP1B1 (SLCO1B1) variant alleles on the pharmacokinetics of pitavastatin in healthy volunteers. Clin Pharmacol Therapeut. 2005;78:342–350. doi: 10.1016/j.clpt.2005.07.003.  [DOI](https://doi.org/10.1016/j.clpt.2005.07.003) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16198653/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Therapeut&title=Effect%20of%20OATP1B1%20(SLCO1B1)%20variant%20alleles%20on%20the%20pharmacokinetics%20of%20pitavastatin%20in%20healthy%20volunteers&author=JY%20Chung&author=JY%20Cho&author=KS%20Yu&volume=78&publication_year=2005&pages=342-350&pmid=16198653&doi=10.1016/j.clpt.2005.07.003&)

43. Pasanen MK, Neuvonen M, Neuvonen PJ, Niemi M. SLCO1B1 polymorphism markedly affects the pharmacokinetics of simvastatin acid. Pharmacogenet Genom. 2006;16:873–879. doi: 10.1097/01.fpc.0000230416.82349.90.  [DOI](https://doi.org/10.1097/01.fpc.0000230416.82349.90) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17108811/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genom&title=SLCO1B1%20polymorphism%20markedly%20affects%20the%20pharmacokinetics%20of%20simvastatin%20acid&author=MK%20Pasanen&author=M%20Neuvonen&author=PJ%20Neuvonen&author=M%20Niemi&volume=16&publication_year=2006&pages=873-879&pmid=17108811&doi=10.1097/01.fpc.0000230416.82349.90&)

44. Pasanen MK, Fredrikson H, Neuvonen PJ, Niemi M. Different effects of SLCO1B1 polymorphism on the pharmacokinetics of atorvastatin and rosuvastatin. Clin Pharmacol Therapeut. 2007;82:726–733. doi: 10.1038/sj.clpt.6100220.  [DOI](https://doi.org/10.1038/sj.clpt.6100220) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17473846/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Therapeut&title=Different%20effects%20of%20SLCO1B1%20polymorphism%20on%20the%20pharmacokinetics%20of%20atorvastatin%20and%20rosuvastatin&author=MK%20Pasanen&author=H%20Fredrikson&author=PJ%20Neuvonen&author=M%20Niemi&volume=82&publication_year=2007&pages=726-733&pmid=17473846&doi=10.1038/sj.clpt.6100220&)

45. Lee E, Ryan S, Birmingham B, et al. Rosuvastatin pharmacokinetics and pharmacogenetics in white and Asian subjects residing in the same environment. Clin Pharmacol Therapeut. 2005;78:330–341. doi: 10.1016/j.clpt.2005.06.013.  [DOI](https://doi.org/10.1016/j.clpt.2005.06.013) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16198652/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Therapeut&title=Rosuvastatin%20pharmacokinetics%20and%20pharmacogenetics%20in%20white%20and%20Asian%20subjects%20residing%20in%20the%20same%20environment&author=E%20Lee&author=S%20Ryan&author=B%20Birmingham&volume=78&publication_year=2005&pages=330-341&pmid=16198652&doi=10.1016/j.clpt.2005.06.013&)

46. Niemi M, Backman JT, Kajosaari LI, et al. Polymorphic organic anion transporting polypeptide 1B1 is a major determinant of repaglinide pharmacokinetics. Clin Pharmacol Therapeut. 2005;77:468–478. doi: 10.1016/j.clpt.2005.01.018.  [DOI](https://doi.org/10.1016/j.clpt.2005.01.018) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15961978/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Therapeut&title=Polymorphic%20organic%20anion%20transporting%20polypeptide%201B1%20is%20a%20major%20determinant%20of%20repaglinide%20pharmacokinetics&author=M%20Niemi&author=JT%20Backman&author=LI%20Kajosaari&volume=77&publication_year=2005&pages=468-478&pmid=15961978&doi=10.1016/j.clpt.2005.01.018&)

47. Katz DA, Carr R, Grimm DR, et al. Organic anion transporting polypeptide 1B1 activity classified by SLCO1B1 genotype influences atrasentan pharmacokinetics. Clin Pharmacol Therapeut. 2006;79:186–196. doi: 10.1016/j.clpt.2005.11.003.  [DOI](https://doi.org/10.1016/j.clpt.2005.11.003) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16513443/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Therapeut&title=Organic%20anion%20transporting%20polypeptide%201B1%20activity%20classified%20by%20SLCO1B1%20genotype%20influences%20atrasentan%20pharmacokinetics&author=DA%20Katz&author=R%20Carr&author=DR%20Grimm&volume=79&publication_year=2006&pages=186-196&pmid=16513443&doi=10.1016/j.clpt.2005.11.003&)

48. Xiang X, Jada SR, Li HH, et al. Pharmacogenetics of SLCO1B1 gene and the impact of *1b and *15 haplotypes on irinotecan disposition in Asian cancer patients. Pharmacogenet Genom. 2006;16:683–691. doi: 10.1097/01.fpc.0000230420.05221.71.  [DOI](https://doi.org/10.1097/01.fpc.0000230420.05221.71) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16906022/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genom&title=Pharmacogenetics%20of%20SLCO1B1%20gene%20and%20the%20impact%20of%20*1b%20and%20*15%20haplotypes%20on%20irinotecan%20disposition%20in%20Asian%20cancer%20patients&author=X%20Xiang&author=SR%20Jada&author=HH%20Li&volume=16&publication_year=2006&pages=683-691&pmid=16906022&doi=10.1097/01.fpc.0000230420.05221.71&)

49. Oswald S, Scheuch E, Cascorbi I, Siegmund W. A LC-MS/MS method to quantify the novel cholesterol lowering drug ezetimibe in human serum, urine and feces in healthy subjects genotyped for SLCO1B1. J Chromatog B. 2006;830:143–150. doi: 10.1016/j.jchromb.2005.10.034.  [DOI](https://doi.org/10.1016/j.jchromb.2005.10.034) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16280261/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Chromatog%20B&title=A%20LC-MS/MS%20method%20to%20quantify%20the%20novel%20cholesterol%20lowering%20drug%20ezetimibe%20in%20human%20serum,%20urine%20and%20feces%20in%20healthy%20subjects%20genotyped%20for%20SLCO1B1&author=S%20Oswald&author=E%20Scheuch&author=I%20Cascorbi&author=W%20Siegmund&volume=830&publication_year=2006&pages=143-150&pmid=16280261&doi=10.1016/j.jchromb.2005.10.034&)
