# Pharmacogenomics of selective serotonin reuptake inhibitor treatment for major depressive disorder: genome-wide associations and functional genomics

## Metadata
**Authors:** Yuan Ji, Joanna M Biernacka, Scott Hebbring, Yubo Chai, Gregory D Jenkins, Anthony Batzler, Karen A Snyder, Maureen S Drews, Zeruesenay Desta, David Flockhart, Taisei Mushiroda, Michiaki Kubo, Yusuke Nakamura, Naoyuki Kamatani, Daniel Schaid, Richard M Weinshilboum, David A Mrazek
**Journal:** The pharmacogenomics journal
**Date:** 2012 Aug 21
**DOI:** [10.1038/tpj.2012.32](https://doi.org/10.1038/tpj.2012.32)
**PMID:** 22907730
**PMCID:** PMC3941038
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3941038/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC3941038/pdf/nihms394558.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC3941038/pdf/nihms394558.pdf)

## Abstract

A genome-wide association (GWA) study of treatment outcomes (response and remission) of selective serotonin reuptake inhibitors (SSRIs) was conducted using 529 subjects with major depressive disorder (MDD). While no SNP associations reached the genome-wide level of significance, 14 SNPs of interest were identified for functional analysis. The rs11144870 SNP in riboflavin kinase (RFK) gene on chromosome 9 was associated with eight week treatment response (OR = 0.42, p = 1.04×10−6). The rs915120 SNP in the G protein-coupled receptor kinase 5 (GRK5) gene on chromosome 10 was associated with eight week remission (OR = 0.50, p = 1.15×10−5). Both SNPs were shown to influence transcription by a reporter gene assay and to alter nuclear protein binding using an electrophoretic mobility shift assay. This report represents an example of joining functional genomics with traditional GWA study results derived from a GWA analysis of SSRI treatment outcomes. The goal of this analytic strategy is to provide insights into the potential relevance of biologically plausible observed associations.

Keywords: selective serotonin reuptake inhibitors, SSRI, genome-wide association study, GWA, functional genomics, major depressive disorder

## Introduction

Major depressive disorder (MDD) is a serious and prevalent psychiatric illness.^1^ Selective serotonin reuptake inhibitors (SSRIs) are an effective treatment for MDD, but treatment response is highly variable.^2^ Consequently, the identification of pharmacogenetic predictors of variable drug response phenotypes has become a major objective of MDD research.^3–5^

Genome-wide association (GWA) studies provide an agnostic approach which can identify novel genetic variants that may contribute to variation in drug response phenotypes.^6^ Three GWA studies of SSRI antidepressant outcomes have been reported, but none have identified associations of SNPs with treatment outcomes that reached genome-wide statistical significance or were validated in a separate population.^7–9^ This failure to demonstrate genome-wide significance may be related to (1) insufficient statistical power to detect small effect sizes for associated variants, (2) inconsistently defined phenotypes, (3) confounding effects from non-genetic factors, or (4) inconsistencies in treatment protocols.^10, 11^ However, the conduct of functional genomic studies of the SNPs identified by GWA analyses provides a novel strategy to identify genomic variants that may influence antidepressant treatment.

The present study is a report of GWA analyses of MDD patients who participated in a single-site pharmacogenomic study of SSRI therapy. Following the GWA analyses, a reporter gene assay and an electrophoretic mobility shift assay were performed to determine whether any identified candidate SNPs were associated with a change in function.

## Materials/Subjects and Methods

### Study Design and Samples

The Mayo Clinic Pharmacogenomic Research Network Antidepressant Medication Pharmacogenomic Study (PGRN-AMPS) was supported by the NIGMS-Pharmacogenomics Research Network (PGRN), which has been described elsewhere.^12, 13^ The PGRN-AMPS is an ongoing eight week outpatient SSRI clinical trial that was performed at the Mayo Clinic in Rochester, MN. Patients enrolled in the study met diagnostic criteria for MDD without psychosis or mania and had a 17-item Hamilton Depression Rating Scale (HAMD-17) score ≥14. The study was designed with inclusion and exclusion criteria similar to those used in the Sequenced Treatment Alternatives to Relieve Depression study (STAR*D).^14^ Potential study subjects taking an antidepressant, antipsychotic or mood stabilizing medication were not eligible for enrollment. Patients with MDD initially received either 10 mg of escitalopram or 20 mg of citalopram. SSRI efficacy was determined using the 16-item Quick Inventory of Depressive Symptomatology (QIDS-C16) scores after four weeks and then eight weeks of SSRI therapy. At four weeks after the initiation of treatment, the dose could be increased to 20 mg of escitalopram or 40 mg of citalopram after a clinical assessment of the subject. Unless there was a contraindication, dose was increased if the QIDS-C16 score at the follow-up visit was ≥9, and possibly following a clinical evaluation if the score was between 6 and 8. The dose could also be decreased, or treatment could be discontinued, if a patient developed persistent side effects. Blood samples were obtained at baseline for DNA extraction, and then again at weeks four and eight for assays of drug and metabolite levels. All patients provided written informed consent. The study protocol was approved by the Mayo Clinic Institutional Review Board.

### Treatment Outcomes

The two primary outcome phenotypes in the GWA analyses were “response” (defined as ≥50% reduction in QIDS-C16 score from baseline to the last visit) and “remission” (defined as a QIDS-C16 score of ≤5 at the last visit). For each of these two outcomes, analyses were performed using two strategies. The primary analyses included only subjects that were evaluated at the eight-week visit. The secondary analyses were performed with outcomes based on the final visit QIDS-C16 scores, referred to as the “last visit” assessment. These analyses included subjects who had completed the full eight-week study as well as those who dropped out of the study prior to the eight-week assessment.

### Genotyping and QC

DNA from 529 patients was genotyped by the RIKEN Center for Genomic Medicine (Yokohama, Japan) using Illumina Human610-Quad BeadChips (Illumina, San Diego, CA). Taqman genotyping assays (Applied Biosystems, Foster City, CA) were used to perform genotyping for the replication study using STAR*D samples, and the Illumina Goldengate platform was used to genotype selected top SNPs identified in the published STAR*D GWAS ^7^ using Mayo PGRN-AMPS DNA samples.

Quality control assessments included overall genotype concordance rates based on duplicate sample genotyping and Mendelian inheritance checks based on genotyping of a CEPH trio of two parents and their child. For each SNP, the minor allele frequency, call rate, and departure from Hardy-Weinberg Equilibrium were evaluated. Observed call rates, total heterozygosity and inbreeding coefficients were assessed for each subject using PLINK.^15^ Sex-checks based on X-chromosome heterozygosity were performed, and tests of identity-by-descent were used to identify potentially related subjects.

More than 97% of study participants were of white non-Hispanic (WNH) ancestry. Consequently, genotype-phenotype association analyses were restricted to the WNH subjects. A subset of 4,855 independent SNPs with low local linkage disequilibrium (r^2^ < 0.063) was used to verify self-reported ancestry of the subjects using the software STRUCTURE.^16^ In addition to the study population, 287 DNA samples from the “Human Variation Panel” of lymphoblastoid cell lines (sample sets HD100CAU, HD100AA and HD100CHI) obtained from the Coriell Institute (Camden, NJ) were included in this analysis. Probabilities of membership in each of three known ancestral groups of the cell lines were calculated for each study subject. These analyses were used to verify self-reported race and to assign race to subjects with unknown race. 509 subjects identified themselves as WNH. A further five subjects who self-identified their race as “other” or “multiracial” were found to have >75% European ancestry based on the STRUCTURE analysis. Thus, 514 WNH subjects were included in subsequent analyses.

### Statistical Analyses

Prior to genotype-phenotype association analyses, EigenStrat software was used to determine eigenvalues for the SNP correlation matrix.^17^ Eigenvalues that differed statistically from zero were determined based on Tracy-Widom test p-values, and the corresponding eigenvectors were used as covariates in the genetic analyses to account for differences in ancestry within the WNH set of subjects.

Of the 514 WNH subjects, 15 patients were found to have very low blood drug levels, suggesting non-adherence, and were excluded from analysis, resulting in 499 subjects who were included in the primary GWA analyses. Logistic regression was used to test for associations between the binary outcome variables, response and remission, and the genotype of each SNP. Genotypes were coded as the “dose” of the minor allele. These analyses assume log-additive allele effects on response or remission. All GWA analyses were adjusted for eigenvectors that captured population stratification. Gender and age (categorized into four quartiles) were also considered as covariates for the GWA analyses. Age was not significantly associated with response or remission (p > 0.10 for both outcomes). Gender was not associated with response (p > 0.10 for both 8-week and final-visit response), but was marginally associated with remission (p = 0.057 for 8-week remission, p = 0.036 for final-visit remission). We therefore performed response and remission GWA analyses both with and without gender as a covariate. As the results were nearly identical, we report only findings from the analyses not adjusted for gender.

Analyses were performed in R ([http://www.R-project.org](http://www.R-project.org)), SAS (SAS Institute Inc.) and PLINK.^15^ Odds ratios (ORs), 95% confidence intervals (CIs), and p-values were calculated based on the logistic regression models. P-values < 10^−7^ were considered statistically significant at the genome-wide level.

Prior to performing the GWA analyses, the power to detect SNP effects was estimated, assuming 500 subjects with 30% failing to achieve remission. It was determined that with a type 1 error rate of 10^−7^, the data would provide >80% power to detect ORs of at least 1.8 for common alleles (MAF 0.10–0.20).

### Functional Genomics

On review of the GWA analyses, 14 candidate SNPs were selected based on having some evidence of an association (p < 9.0 × 10^−5^) and being located within or near a candidate gene. SNPs in locations that could potentially influence gene transcription were preferentially selected. Reporter gene assays were used to demonstrate the effects of these SNPs on transcription and electrophoretic mobility shift (EMS) assays were used to assess DNA-protein binding.

For the reporter gene assays, 200 to 300 bp DNA sequences that included a candidate SNP were cloned into a pGL3-promoter luciferase reporter vector that contained an SV40 promoter upstream of the luciferase gene (Promega Corporation, Fitchburg, WI). One microgram of each reporter gene construct was co-transfected with 20 ng of the pRL-TK renilla luciferase vector as a control for transfection efficiency into a human neuroblastoma cell line, SK-N-BE(2) and two human glioblastoma cell lines, U-87 MG and U251 (American Type Culture Collection, ATCC, Manassas, VA), followed by dual-luciferase assays performed 24 hrs after transfection (Promega). Two independent transfections were performed for each reporter gene construct, with triplicate transfections for each construct in each experiment. Values for relative activity were expressed as percentages of the pGL3-promoter activity for vector without an insert. Comparisons were then made between pGL3 reporter gene constructs containing wild type and variant nucleotide for each candidate SNP. DNA samples that were used to amplify SNP regions were selected from the Coriell “Human Variation Panel” DNA samples (Camden, NJ).^18^ Sequences of primers used to amplify genomic regions containing the selected SNPs are listed in [Supplemental Table 1](#SD1).

EMS assays were used to determine whether a nucleotide change altered the ability of oligonucleotides to bind nuclear proteins. Nuclear extracts were prepared from SK-N-BE(2), U87 MG and U251 cells using the Nuclear Extract Kit (Active Motif, Carlsbad, CA). Biotin-labeled and unlabeled oligonucleotides were synthesized by IDT Integrated DNA Technologies (Coralville, IA). EMS assays were performed using the LightShift® Chemiluminescent EMSA Kit (PIERCE, Rockford, IL). Oligonucleotide sequences used to perform the EMS assays are listed in [Supplemental Table 2](#SD1).

### Replication

The 14 candidate SNPs included in the functional genomic experiments were genotyped in the STAR*D DNA samples. In addition, the 25 SNPs reported in the STAR*D GWA study^7^ to have the greatest likelihood of being associated with treatment outcomes were genotyped in the PGRN-AMPS samples. The results of these analyses are described in the [Supplemental Materials](#SD1).

## Results

### Demographic and Clinical Characteristics of the Study Subjects

Of the 529 participants in the Mayo PGRN-AMPS study, 15 subjects who were not identified as white and non-Hispanic and 15 non-adherent subjects were excluded, resulting in 499 WNH subjects for the secondary “last visit” GWA analyses. The primary “eight week” GWA analyses were conducted using a subset of 398 WNH subjects who were protocol adherent. Demographic information, baseline clinical characteristics and treatment outcomes for both the eight week and the last visit cohorts are summarized in [Table 1](#T1).

### Table 1.

| Characteristic and Measure | Last visit (N = 499) | Eight weeks (N = 398) |
| --- | --- | --- |
| Demographics |  |  |
| Age | 39.91 (±13.8) | 40.64 (±13.5) |
| Education (years) | 14.68 (±2.4) | 14.91 (±2.3) |
| Female gender | 312 (62.5%) | 255 (64.1%) |
|   |  |  |
| Clinical Characteristics |  |  |
| QIDS-C Baseline | 15.08 (±3.47) | 14.99 (±3.31) |
| QIDS-C Week Four | 8.46 (±4.41) | 8.34 (±4.41) |
| QIDS-C Week Eight | 6.24 (±4.05) | 6.24 (±4.05) |
| Age at Onset of First Episode of Depression | 24.05 (±13.82) | 24.70 (±14.11) |
| Recurrent Depression (>1 episode) | 483 (96.8%) | 386 (96.9%) |
|   |  |  |
| Baseline Medication |  |  |
| Citalopram | 155 (31.2%) | 124 (31.2%) |
| Escitalopram | 342 (68.8%) | 274 (68.8%) |
|   |  |  |
| Last known dose (mg) |  |  |
| Citalopram | 28.10 (±10.02) | 30.28 (±9.76) |
| Escitalopram | 14.56 (±5.06) | 15.75 (±4.96) |
|   |  |  |
| Outcomes |  |  |
| Remitter (QIDS ≤5) | 206 (41.3%) | 198 (49.7%) |
| Response (% reduction QIDS ≥50%) | 287 (57.5%) | 274 (68.8%) |

Table 1 Caption: Baseline clinical and demographic characteristics and outcomes for medication-adherent white non-Hispanic subjects included in the last-visit and eight-week analyses

### Quality Control

Quality control analyses were performed for 592,236 genotyped SNPs. The genotypes had very high concordance rates in duplicate samples (> 99.99%) and very low rates of Mendelian inheritance errors (< 0.01%). 30,700 SNPs failed genotyping. Another 28,610 SNPs had allele frequencies below the preset threshold of 0.01 and, consequently, were excluded from analysis due to low power. An additional 49 SNPs were excluded due to significant departures from Hardy-Weinberg Equilibrium (p < 10^−6^). The remaining SNPs (n = 532,877) had call rates exceeding 98%. One person who was reported to be female appeared to be genetically male based on no heterozygosity of X-chromosome SNPs. Identity-by-descent analysis revealed several pairs of subjects that appear to be related (two likely parent-child pairs, one possible pair of siblings, and three pairs that may be more distant relatives). Genome-wide heterozygosity was within the expected range.

### Genetic Association Analysis of SSRI Treatment Outcomes

Separate GWA analyses based on the two primary outcomes of response and remission were conducted for both the eight week protocol adherent sample (n = 398) and the last visit sample (n = 499). Four eigenvectors identified through EigenStrat analysis were used as adjusting covariates in the analyses. While none of these analyses demonstrated genome wide significance (p < 10^−7^), the 25 SNPs with the most significant associations with the four outcome phenotypes are listed in [Table 2](#T2). Manhattan plots of p-values from the analyses for eight week outcomes are shown in [Figure 1](#F1). Manhattan plots for last visit outcomes are included in [Figure S1](#SD1).

### Table 2.

|   |   |   |   | MAF |  | Association with Response/Remission |  |   |   |   |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
|   |  |  |  |  |  |  |  |  |  |  |
| Rank | SNP | Chr. | Position | Responder/Remitter | Non-responder/Non-remitter | OR | P-value | Gene | Gene Region | Distance to Gene (bp) |
| Response at Eight Weeks |  |  |  |  |  |  |  |  |  |  |
| 1 | rs11144870a | 9 | 78194033 | 0.22 | 0.39 | 0.43 | 1.04E-06 | RFK | intron | −641 |
| 2 | rs11144905a | 9 | 78255119 | 0.26 | 0.43 | 0.45 | 1.43E-06 | GCNT1 | flanking_5UTR | −8847 |
| 3 | rs9775457 | 9 | 139386750 | 0.33 | 0.17 | 2.63 | 3.15E-06 | NA | NA | NA |
| 4 | rs785916a | 9 | 78290826 | 0.21 | 0.36 | 0.46 | 7.50E-06 | GCNT1 | intron | −14826 |
| 5 | rs1045074 | 6 | 4899709 | 0.47 | 0.30 | 2.13 | 1.18E-05 | CDYL | 3UTR | [419/1068] |
| 6 | rs3906767 | 2 | 12222673 | 0.13 | 0.25 | 0.40 | 1.30E-05 | LPIN1 | flanking_3UTR | −337687 |
| 7 | rs3954862 | 2 | 12222204 | 0.13 | 0.25 | 0.40 | 1.30E-05 | LPIN1 | flanking_3UTR | −337218 |
| 8 | rs731395 | 2 | 12214033 | 0.13 | 0.26 | 0.41 | 1.58E-05 | LPIN1 | flanking_3UTR | −329047 |
| 9 | rs1869733 | 8 | 63970954 | 0.13 | 0.25 | 0.41 | 2.19E-05 | FAM77D | intron | −22612 |
| 10 | rs1428241 | 5 | 25579023 | 0.12 | 0.23 | 0.41 | 3.10E-05 | CDH10 | flanking_5UTR | −898355 |
| 11 | rs2837754 | 21 | 40912929 | 0.05 | 0.14 | 0.32 | 3.72E-05 | DSCAM | intron | −73677 |
| 12 | rs4711589 | 6 | 39338549 | 0.25 | 0.40 | 0.50 | 3.82E-05 | KCNK5 | flanking_5UTR | −33345 |
| 13 | rs2283859 | 22 | 28234519 | 0.20 | 0.33 | 0.47 | 3.99E-05 | C22orf19 | intron | −10 |
| 14 | rs1182864 | 6 | 39338285 | 0.40 | 0.25 | 2.07 | 4.04E-05 | KCNK5 | flanking_5UTR | −33081 |
| 15 | rs1455583 | 8 | 63975240 | 0.13 | 0.25 | 0.42 | 4.06E-05 | FAM77D | intron | −18326 |
| 16 | rs6493270 | 15 | 45400695 | 0.12 | 0.23 | 0.43 | 4.44E-05 | SEMA6D | flanking_5UTR | −397283 |
| 17 | rs4649163 | 1 | 24252188 | 0.24 | 0.38 | 0.50 | 4.70E-05 | MYOM3 | flanking_3UTR | −2934 |
| 18 | rs2595372 | 18 | 27492355 | 0.16 | 0.30 | 0.49 | 5.07E-05 | B4GALT6 | intron | −305 |
| 19 | rs680448 | 9 | 78316773 | 0.54 | 0.40 | 1.93 | 5.64E-05 | GCNT1 | flanking_3UTR | −4621 |
| 20 | rs714679 | 11 | 44610240 | 0.38 | 0.54 | 0.53 | 5.71E-05 | CD82 | flanking_3UTR | −12325 |
| 21 | rs4242365 | 8 | 125307549 | 0.25 | 0.40 | 0.51 | 5.90E-05 | C8orf78 | flanking_5UTR | −54605 |
| 22 | rs10485285 | 6 | 81535326 | 0.29 | 0.45 | 0.52 | 6.75E-05 | BCKDHB | flanking_3UTR | −422620 |
| 23 | rs13014477 | 2 | 29649492 | 0.33 | 0.19 | 2.15 | 7.50E-05 | ALK | intron | −41006 |
| 24 | rs16987839 | 22 | 28313138 | 0.15 | 0.28 | 0.49 | 7.93E-05 | NIPSNAP1 | flanking_5UTR | −5810 |
| 25 | rs3804024 | 21 | 40915577 | 0.06 | 0.14 | 0.35 | 8.10E-05 | DSCAM | intron | −71029 |
| Response at Last Visit |  |  |  |  |  |  |  |  |  |  |
| 1 | rs2248399a | 13 | 105440474 | 0.12 | 0.23 | 0.46 | 1.39E-05 | DAOA | flanking_3UTR | −499090 |
| 2 | rs6501039 | 16 | 7920533 | 0.34 | 0.22 | 1.90 | 2.06E-05 | A2BP1 | flanking_3UTR | −218033 |
| 3 | rs2831007 | 21 | 27863838 | 0.31 | 0.19 | 1.92 | 2.08E-05 | C21orf94 | flanking_5UTR | −443734 |
| 4 | rs13014477 | 2 | 29649492 | 0.34 | 0.22 | 1.89 | 2.64E-05 | ALK | intron | −41006 |
| 5 | rs1751915 | 10 | 21484609 | 0.15 | 0.26 | 0.49 | 2.90E-05 | C10orf113 | flanking_5UTR | −9176 |
| 6 | rs3811541 | 2 | 127129605 | 0.30 | 0.43 | 0.55 | 3.14E-05 | GYPC | flanking_5UTR | −549 |
| 7 | rs12450499 | 17 | 10109041 | 0.11 | 0.20 | 0.45 | 3.36E-05 | MYH13 | flanking_3UTR | −35867 |
| 8 | rs2248174a | 13 | 105442451 | 0.11 | 0.21 | 0.46 | 3.48E-05 | DAOA | flanking_3UTR | −501068 |
| 9 | rs11144870a | 9 | 78194033 | 0.22 | 0.33 | 0.55 | 3.70E-05 | RFK | intron | −641 |
| 10 | rs7378714 | 5 | 111921446 | 0.29 | 0.18 | 1.96 | 4.64E-05 | FLJ11235 | flanking_3UTR | −136878 |
| 11 | rs4620377 | 9 | 14623299 | 0.37 | 0.50 | 0.58 | 4.89E-05 | ZDHHC21 | intron | −6595 |
| 12 | rs12328118 | 2 | 82811461 | 0.26 | 0.15 | 2.02 | 4.93E-05 | LOC388965 | flanking_5UTR | −1559857 |
| 13 | rs4835490 | 4 | 149309776 | 0.24 | 0.14 | 2.07 | 4.95E-05 | NR3C2 | intron | −14274 |
| 14 | rs3762876 | 4 | 95808500 | 0.42 | 0.30 | 1.75 | 5.08E-05 | PDLIM5 | flanking_3UTR | −100 |
| 15 | rs3761555 | 23 | 122144118 | 0.22 | 0.38 | 0.52 | 5.16E-05 | GRIA3 | flanking_5UTR | −1659 |
| 16 | rs2549151 | 16 | 81417750 | 0.40 | 0.53 | 0.58 | 5.21E-05 | CDH13 | intron | −31718 |
| 17 | rs6493270 | 15 | 45400695 | 0.13 | 0.23 | 0.50 | 5.58E-05 | SEMA6D | flanking_5UTR | −397283 |
| 18 | rs1902763 | 12 | 50943932 | 0.21 | 0.12 | 2.19 | 5.62E-05 | KRT7 | flanking_3UTR | −14956 |
| 19 | rs6104012 | 20 | 43167985 | 0.15 | 0.25 | 0.51 | 5.91E-05 | WFDC5 | flanking_3UTR | −3522 |
| 20 | rs11692714 | 2 | 82835165 | 0.25 | 0.14 | 2.03 | 6.20E-05 | LOC388965 | flanking_5UTR | −1536153 |
| 21 | rs17022383 | 2 | 82845146 | 0.25 | 0.14 | 2.03 | 6.20E-05 | NA | NA | NA |
| 22 | rs9828629 | 3 | 71613036 | 0.41 | 0.29 | 1.78 | 6.36E-05 | FOXP1 | intron | −12231 |
| 23 | Rs1945300 | 12 | 50953187 | 0.22 | 0.12 | 2.16 | 6.80E-05 | KRTHB1 | flanking_3UTR | −12777 |
| 24 | rs1998560a | 13 | 105441526 | 0.15 | 0.26 | 0.51 | 6.80E-05 | DAOA | flanking_3UTR | −500134 |
| 25 | rs2608017 | 12 | 50962184 | 0.21 | 0.12 | 2.20 | 6.81E-05 | KRTHB1 | flanking_3UTR | −3780 |
| Remission at Eight Weeks |  |  |  |  |  |  |  |  |  |  |
| 1 | rs1379887a | 6 | 78407561 | 0.27 | 0.42 | 0.49 | 9.05E-06 | HTR1B | flanking_5UTR | −177722 |
| 2 | rs915120a | 10 | 121180103 | 0.32 | 0.48 | 0.50 | 1.15E-05 | GRK5 | intron | −163 |
| 3 | rs10899655 | 11 | 78994647 | 0.11 | 0.23 | 0.41 | 1.62E-05 | NARS2 | flanking_5UTR | −1031280 |
| 4 | rs7738598a | 6 | 78671581 | 0.52 | 0.36 | 1.87 | 1.67E-05 | HTR1B | flanking_5UTR | −441742 |
| 5 | rs898040a | 6 | 78408635 | 0.27 | 0.41 | 0.51 | 1.69E-05 | HTR1B | flanking_5UTR | −178796 |
| 6 | rs790575 | 12 | 91055739 | 0.17 | 0.30 | 0.47 | 2.03E-05 | BTG1 | flanking_3UTR | −5291 |
| 7 | rs6793050 | 3 | 66468459 | 0.09 | 0.19 | 0.40 | 3.58E-05 | NA | NA | NA |
| 8 | rs978814 | 6 | 81016631 | 0.37 | 0.52 | 0.54 | 3.82E-05 | BCKDHB | intron | −22940 |
| 9 | rs12413418 | 10 | 42493575 | 0.03 | 0.10 | 0.23 | 3.89E-05 | ZNF11B | flanking_5UTR | −39577 |
| 10 | rs16840617 | 4 | 7661440 | 0.40 | 0.26 | 1.91 | 4.05E-05 | SORCS2 | intron | −29515 |
| 11 | rs12254134a | 10 | 121161872 | 0.35 | 0.49 | 0.52 | 4.24E-05 | GRK5 | intron | −10796 |
| 12 | rs12963758 | 18 | 7814091 | 0.10 | 0.03 | 4.35 | 4.44E-05 | PTPRM | intron | −49822 |
| 13 | rs1924939 | 13 | 71998057 | 0.29 | 0.44 | 0.54 | 4.67E-05 | FLJ22624 | flanking_5UTR | −201986 |
| 14 | rs1359873 | 13 | 74194810 | 0.36 | 0.24 | 2.01 | 4.76E-05 | LOC400145 | flanking_5UTR | −303557 |
| 15 | rs9436797 | 1 | 68340006 | 0.16 | 0.28 | 0.47 | 4.81E-05 | GPR177 | intron | −2982 |
| 16 | rs3812121 | 6 | 81025446 | 0.37 | 0.52 | 0.55 | 4.94E-05 | BCKDHB | intron | −14125 |
| 17 | rs6414150 | 2 | 196603015 | 0.30 | 0.18 | 2.01 | 4.98E-05 | DNAH7 | intron | −1999 |
| 18 | rs9343638 | 6 | 78438553 | 0.29 | 0.43 | 0.53 | 4.99E-05 | HTR1B | flanking_5UTR | −208714 |
| 19 | rs757240 | 4 | 7658657 | 0.39 | 0.26 | 1.89 | 5.33E-05 | SORCS2 | intron | −32298 |
| 20 | rs2252523 | 16 | 1937615 | 0.25 | 0.39 | 0.53 | 5.49E-05 | RPL3L | intron | −233 |
| 21 | rs9542990 | 13 | 71993272 | 0.22 | 0.35 | 0.52 | 6.20E-05 | FLJ22624 | flanking_5UTR | −206771 |
| 22 | rs9352817 | 6 | 81082598 | 0.38 | 0.52 | 0.55 | 6.70E-05 | BCKDHB | intron | −27502 |
| 23 | rs1936158 | 6 | 78241094 | 0.48 | 0.35 | 1.87 | 6.89E-05 | HTR1B | flanking_5UTR | −11255 |
| 24 | rs351294 | 5 | 154428932 | 0.05 | 0.13 | 0.33 | 7.40E-05 | MRPL22 | flanking_3UTR | −102208 |
| 25 | rs17777705 | 6 | 78551483 | 0.15 | 0.27 | 0.49 | 7.62E-05 | HTR1B | flanking_5UTR | −321644 |
| Remission at Last Visit |  |  |  |  |  |  |  |  |  |  |
| 1 | rs7439567a | 4 | 138684292 | 0.32 | 0.46 | 0.51 | 3.11E-06 | PCDH18 | flanking_5UTR | −11213 |
| 2 | rs1379887a | 6 | 78407561 | 0.27 | 0.41 | 0.52 | 7.88E-06 | HTR1B | flanking_5UTR | −177722 |
| 3 | rs9542990 | 13 | 71993272 | 0.22 | 0.35 | 0.53 | 1.60E-05 | FLJ22624 | flanking_5UTR | −206771 |
| 4 | rs7738598a | 6 | 78671581 | 0.52 | 0.38 | 1.76 | 1.66E-05 | HTR1B | flanking_5UTR | −441742 |
| 5 | rs898040a | 6 | 78408635 | 0.27 | 0.40 | 0.54 | 1.81E-05 | HTR1B | flanking_5UTR | −178796 |
| 6 | rs9761827a | 4 | 138635961 | 0.30 | 0.43 | 0.55 | 2.27E-05 | PCDH18 | flanking_3UTR | −24324 |
| 7 | rs1924939 | 13 | 71998057 | 0.30 | 0.44 | 0.56 | 2.36E-05 | FLJ22624 | flanking_5UTR | −201986 |
| 8 | rs7058787 | 23 | 45583663 | 0.45 | 0.27 | 1.94 | 2.44E-05 | LOC100128442 | gene | 0 |
| 9 | rs12963758 | 18 | 7814091 | 0.10 | 0.03 | 3.36 | 2.56E-05 | PTPRM | intron | −49822 |
| 10 | rs1945300 | 12 | 50953187 | 0.24 | 0.14 | 2.11 | 3.28E-05 | KRTHB1 | flanking_3UTR | −12777 |
| 11 | rs12809631 | 12 | 129611143 | 0.17 | 0.28 | 0.50 | 3.33E-05 | RIMBP2 | flanking_5UTR | −42780 |
| 12 | rs2385296 | 8 | 124655486 | 0.53 | 0.40 | 1.75 | 3.42E-05 | FBXO32 | flanking_5UTR | −32859 |
| 13 | rs1902763 | 12 | 50943932 | 0.23 | 0.13 | 2.11 | 3.46E-05 | KRT7 | flanking_3UTR | −14956 |
| 14 | rs3752183 | 19 | 61804990 | 0.46 | 0.33 | 1.78 | 3.54E-05 | ZNF71 | intron | −219 |
| 15 | rs4947035 | 6 | 110352448 | 0.14 | 0.06 | 2.58 | 3.69E-05 | GPR6 | flanking_5UTR | −54539 |
| 16 | rs3807736 | 7 | 77571209 | 0.42 | 0.55 | 0.58 | 4.46E-05 | MAGI2 | intron | −21738 |
| 17 | rs17541104 | 15 | 91664471 | 0.37 | 0.25 | 1.80 | 4.48E-05 | UNQ9370 | flanking_3UTR | −112190 |
| 18 | rs6580873 | 12 | 50968192 | 0.21 | 0.11 | 2.14 | 4.61E-05 | KRTHB1 | coding | [157/7] |
| 19 | rs9294091 | 6 | 78712418 | 0.37 | 0.25 | 1.80 | 5.73E-05 | HTR1B | flanking_5UTR | −482579 |
| 20 | rs2085144 | 8 | 99022252 | 0.16 | 0.27 | 0.51 | 6.09E-05 | MATN2 | intron | −929 |
| 21 | rs1877531 | 6 | 78618795 | 0.40 | 0.27 | 1.76 | 6.24E-05 | HTR1B | flanking_5UTR | −388956 |
| 22 | rs1655645 | 15 | 27479972 | 0.50 | 0.38 | 1.73 | 6.62E-05 | NDNL2 | flanking_5UTR | −130663 |
| 23 | rs2336413 | 2 | 205839562 | 0.42 | 0.30 | 1.78 | 6.65E-05 | ALS2CR19 | intron | −20771 |
| 24 | rs9343638 | 6 | 78438553 | 0.29 | 0.42 | 0.57 | 7.17E-05 | HTR1B | flanking_5UTR | −208714 |
| 25 | rs12722898 | 1 | 26593874 | 0.48 | 0.35 | 1.72 | 7.28E-05 | LIN28 | flanking_5UTR | −15982 |

Table 2 Caption: Top 25 association results for each of the four outcomes: response at eight weeks and last-visit and remission at eight weeks and last visit (based on logistic regression adjusting for four eigenvectors)

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dd57/3941038/35ef073726e0/nihms394558f1.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3941038_nihms394558f1.jpg)

Manhattan plots of −log10(p-values) from logistic regression adjusted for eigenvectors versus chromosomal position of SNPs. (a) Manhattan plot of −log10(p-values) for GWA analysis of eight-week SSRI response outcome. (b) Manhattan plot of −log10(p-values) for GWA analysis of eight-week SSRI remission outcome. Red dots represent SNPs with p-values ≤ 10−5 and blue dots represent SNPs with p-values ≤ 10−4.

### Eight Week Response Analyses

GWA analysis of the eight week response outcome identified a region on chromosome 9 containing three interesting SNPs with p-values < 8.0 × 10^−6^. The rs11144870 SNP is located in intron 2 of the riboflavin kinase(*RFK*) gene. The rs11144905 and rs785916 SNPs were mapped to introns in the adjacent glucosaminyl N-acetyl transferase 1 (*GCNT1*) gene and were in linkage disequilibrium with rs11144870 (r^2^ = 0.78 and 0.73, respectively).

### Eight Week Remission Analyses

Three SNPs, rs1379887, rs7738598 and rs898040, that were located in the 5′ UTR region of the 5-hydroxytryptamine serotonin receptor 1B gene (*HTR1B*) were associated with eight week remission at the p < 2 × 10^−5^ level. The rs915120 and rs12116187 SNPs in the G protein-coupled receptor (GPCR) kinase gene 5 (*GRK5*) on chromosome 10 were also identified as candidate SNPs of interest.

### Analysis of Secondary Outcomes: Last Visit Response and Remission

The most significant association in the last visit response analysis was obtained for rs2248399, which is located in an intergenic region on chromosome 13 (p = 1.0 × 10^−5^). This SNP is located approximately 500 kb from the D-amino acid oxidase activator gene (*DAOA*). The three SNPs near the 5-hydroxytryptamine (serotonin) receptor 1B gene (*HTR1B*), rs1379887, rs7738598 and rs898040, that were associated with remission in the eight week analysis, were also identified in the last visit remission analysis (p-values = 7.9 × 10^−6^, 1.7 × 10^−5^ and 1.8 × 10^−5^, respectively). The last visit remission analysis also identified the rs7439567 (p = 3.1 × 10^−6^) and the rs9761827 (p = 2.3 × 10^−5^) SNPs in the protocadherin-18 gene (*PCDH18*) on chromosome 4.

### Functional Genomics

As described previously, 14 candidate SNPs were chosen for functional studies based on the significance level of their association with treatment outcomes and their location within or near a candidate gene of interest ([Figure 2A](#F2)). The reporter gene assays identified significant differences in the function of WT and variant SNP sequences for five of these SNPs based on differences in relative luciferase activity between pGL3 constructs that contained the sequences in all three CNS-derived cell lines ([Figure 2B](#F2)). When cloned into the pGL3-Promoter vector, sequences containing the SNPs of interest appeared to function as either an enhancer element that significantly increased promoter activity when compared with activity for the pGL3-Promoter construct alone (p-values < 0.05 in both cell lines) or a silencer element depending on the cell line used. Results for the neuroblastoma SK-N-BE(2) cells are shown in [Figure 2b](#F2), and results for U-87 MG and U251 cell lines in [Figure S2a](#SD1). The patterns of differences in luciferase activity between WT and variant alleles were consistent for SNPs rs11144870 (*RFK)*, rs11144905 (*GCNT1*), rs2248399 (*DAOA*), rs915120 (*GRK5*) and rs12254134 (*GRK5*) in all 3 cell lines tested (p-values < 0.01).

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dd57/3941038/3d341abb1dac/nihms394558f2.jpg)

Functional genomics of candidate SNPs. (a) A table to summarize the 14 SNPs that were selected for functional genomic studies. (b) Results of dual-luciferase reporter gene assays for SNPs rs11144870, rs11144905, rs2248399, rs915120 and rs12254134, performed in the neuroblastoma SK-N-BE(2) cell line. Each bar represents the average of relative luciferase activity reported as a % of the pGL3-Promoter construct activity obtained during 6 independent transfections (mean ± S.E.M). ** represents p-values < 0.01 compared to values for pGL3-WT activity. (b) Electrophoretic mobility shift (EMS) assays for SNPs rs11144870, rs785916 (GCNT1), rs2248399 (DAOA), rs2248174 (DAOA), rs1998560 (DAOA), rs915120 (GRK5) and rs12254134 (GRK5) with nuclear extract prepared from SK-N-BE(2) cells.

EMS assays identified seven SNPs that altered the ability of these sequences to bind nuclear proteins in SK-N-BE(2) cells. [Figure 2c](#F2) shows the results for EMS assays for these seven SNPs, including rs11144870, rs785916 (*GCNT1*), rs2248399, rs2248714, rs1998560 (*DAOA*), rs915120 and rs12254134. All of these SNPs showed significant differences in nuclear protein binding patterns between WT and variant sequence oligonucleotides in SK-N-BE(2) cells. Similar patterns were observed for all of these SNPs in U-87 MG and U251 cells ([Figure S2b](#SD1)). *HTR1B* SNP rs7738598 displayed altered DNA-protein binding only in the two glioblastoma cells ([Figure S2b](#SD1)), but not in the neuroblastoma cells.

Four candidate SNPs, rs11144870 (*RFK*), rs915120 (*GRK5*), rs12254134 (*GRK5*) and rs2248399 (*DAOA*) showed differences between WT and variant SNP sequences in both reporter gene and EMS assays performed with all three cell lines.

### Replication

The SNPs selected for functional studies based on GWA analyses of the PGRN-AMPS outcomes data were also genotyped using DNA from the STAR*D study. Association analyses were performed for the same four treatment outcomes with 12 of the 14 SNPs, as two SNPs failed genotyping, using DNA from WNH STAR*D subjects who had HAMD-17 scores ≥14 prior to SSRI therapy. None of the associations for these SNPs were statistically significant in the STAR*D samples ([Supplemental Table S7](#SD1)). The top 25 SNPs reported for the STAR*D GWA analyses^7^ were also evaluated for possible association with treatment response in the PGRN-AMPS samples. None of these associations were statistically significant ([Supplemental Table S8](#SD1)).

## Discussion

Four GWA studies of antidepressant treatment outcomes have been conducted including the PGRN-AMPS study.^7–9^ None of these GWA analyses was able to identify associations that reached genome-wide statistical significance. Furthermore, no specific SNPs or genes have been associated with outcomes across any two of these studies. Given that replication has not been reported, functional validation studies provide an alternative strategy to take the next step toward deciphering the underlying biology of candidate SNPs that might influence antidepressant response.

The association of SNP rs11144870 in *RFK* with SSRI outcomes is intriguing as riboflavin kinase is an essential enzyme that catalyzes the phosphorylation of riboflavin (vitamin B2) to form flavin mononucleotide (FMN). This is a critical step for both vitamin B2 metabolism and flavin cofactor synthesis. Riboflavin and its flavin cofactors influence the folate and methionine cycles since riboflavin functions as a cofactor for methylene tetrahydrofolate reductase (MTHFR). MTHFR is the enzyme that converts 5,10-methylenetetrahydrofolate to 5-methyltetrahydrofolate.^19^ Previous studies have suggested that components of the folate and methionine cycles may be involved in increasing the risk for developing MDD and might influence treatment outcomes.^20–24^ Since insufficient dietary B vitamins, including riboflavin, have been associated with depressive symptoms,^25^ an alternation in the transcription of *RFK* might result in elevated levels of RFK protein which might indirectly influence the intensity of depressive symptoms and the effects of SSRI therapy.

The *GCNT1* gene is a member of the beta-1,6-N-acetylglucosaminyltransferase gene family that has not previously been shown to be associated with MDD treatment outcomes. However, *GCNT1* is highly expressed in brain, and further research focusing on variants in *GCNT1* would be of interest (see the Nervous System database, [http://www.itb.cnr.it/gncdb/](http://www.itb.cnr.it/gncdb/)).

A SNP in an intergenic region near the *DAOA* gene (rs2248399) was identified by these analyses and may be potentially functionally significant based on the reporter gene assay. This SNP and two other *DAOA* SNPs were also shown to have the potential to affect the binding of nuclear proteins. None of these SNPs have been included in previous candidate gene studies of *DAOA* for schizophrenia or bipolar disease.^26–31^

Six SNPs that mapped to an intergenic region ~150 to 500 kb distant from the *HTR1B* gene were associated with eight-week and last visit remission (see [Table 2](#T2) and [Supplemental Tables S5 and S6](#SD1)). The *HTR1B* gene has been reported to be associated with psychiatric phenotypes as well as response to SSRI treatment.^32–35^ EMS assays were performed with these three SNPs and with rs7738598 were found to display a difference between WT and variant sequences in nuclear protein binding in the two glioblastoma cells that were tested.

Two of the intronic SNPs in *GRK5* (rs915120 and rs12254134) that were associated with remission also altered function. A different member of the G protein-coupled receptor kinase family, *GRK2*, has been found to be upregulated during antidepressant treatment^36^ and *GRK5* has been shown to regulate GPCR receptors such as the β_1_-adrenergic receptor^37^ and the dopamine D1A receptor.^38^ GRK5 is highly expressed in many tissues, including human heart and brain.^39, 40^ A single functional *GRK5* polymorphism, rs17098707, that results in a Gln41Leu change in amino acid sequence has been reported to regulate cardiac function.^41, 42^ The intronic SNPs identified in this GWA study are not in linkage disequilibrium with the Gln41Leu polymorphism, suggesting that these novel *GRK5* SNPs might function independently from the Gln41Leu polymorphism. Since more than 90% of GPCRs are expressed in the brain, the identification of functional *GRK5* SNPs may provide novel directions for future studies of variation in antidepressant response.

In the present study, the selection of the SNPs for functional assessment was limited to those identified during our GWA analyses. While other experimental approaches are available to assess the functional consequences of genetic variants, the application of these two commonly used *in vitro* functional assays has highlighted nine candidate SNPs that may be worthy of further mechanistic pursuit using alternative methods. While many GWA studies have been performed with psychiatric phenotypes, few have identified genomic loci that were replicated and could successfully be used as robust “biomarkers” in clinical psychiatric practice. A lack of reliable model systems for functional genomic studies of the biological mechanism underlying the association is among the factors that have prevented the translation of genomic research to psychiatric practices. The use of pluripotent stem (iPS) cells represents a novel and promising tool for functional validation and mechanistic studies of genomic loci identified through GWA studies.^43^

Limitations of our study also include the fact that detailed information on certain clinical factors, such as comorbid psychiatric diagnoses, was not available. In addition, the influence of potentially important covariates such as drug dose and blood levels has not yet been fully explored. However, important associations between genetic variants and clinical outcomes can be missed by adjusting for covariates such as blood drug levels that serve as intermediate factors. Subsequent analyses will be focused on blood drug levels and their association with genetic variation and treatment outcomes. Our top findings were not replicated by an analysis of samples from the STAR*D study and we also failed to replicate the best association findings from the STAR*D GWA analyses. Similarly, the PGRN-AMPS analyses did not replicate the rs1126757 in the *IL11* gene that was reported to have some association with escitalopram response in the GENDEP project.^9^ Finally, the association of the rs6989467 SNP reported for the MARS GWA study^8^ was not replicated in the PGRN-AMPs analyses. Several important differences between the studies may have contributed to the lack of replication, including differences in baseline clinical characteristics of participants in the two studies.

GWA studies have been shown to be a powerful approach for the identification of novel genomic markers for disease risk and pharmacogenomic phenotypes.^44–46^ However, GWA studies have not been as informative when applied to psychiatric phenotypes of disease risk or drug response. There have now been four GWA studies conducted for antidepressant outcomes in MDD patients. None of them has demonstrated genome-wide significant associations nor have these studies replicated each other. The PGRN-AMPS is unique in that it is the only analysis that used blood drug assays to evaluate treatment adherence and is the only study that analyzed the functional significance of identified SNPs of interest. Given the underlying phenotypic heterogeneity (which is typical of studies of psychiatric illnesses) it is particularly challenging to identify homogenous treatment samples. A possible solution is to subclassify patients using biomarkers such as pharmacometabolomic characteristics to identify more biologically similar patients.^12^

## Supplementary Material

## Acknowledgments

This research was supported, in part, by NIH grants RO1 GM28157, U19 GM61388 (The Pharmacogenomics Research Network), P20 1P20AA017830-01 (The Mayo Clinic Center for Individualized Treatment of Alcohol Dependence), and a PhRMA Foundation Center of Excellence in Clinical Pharmacology Award. Dr. Yuan Ji’s work was supported by a KL2 Mentored Career Development Award (NCRR Grant KL2 RR024151) and a Gerstner Family Mayo Career Development Award in Individualized Medicine. This publication was supported by NIH/NCRR/NCATS CTSA Grant Number UL1 RR024150. Its contents are solely the responsibility of the authors and do not necessarily represent the official views of the NIH. We thank the staff of the Mayo Clinic Rochester Department of Psychiatry and Psychology for their effort in recruiting the patients to the Mayo PGRN-AMPS, Dr. Ryan Abo for his assistance in gene annotation, and Lori Solmonson for her assistance with the preparation of the manuscript. The authors also wish to thank the STAR*D investigators for both their effort and generosity in sharing the clinical data and DNA samples.

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Kessler RC, Berglund P, Demler O, Jin R, Koretz D, Merikangas KR, et al. The epidemiology of major depressive disorder: results from the National Comorbidity Survey Replication (NCS-R) JAMA. 2003;289(23):3095–3105. doi: 10.1001/jama.289.23.3095.  [DOI](https://doi.org/10.1001/jama.289.23.3095) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12813115/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA&title=The%20epidemiology%20of%20major%20depressive%20disorder:%20results%20from%20the%20National%20Comorbidity%20Survey%20Replication%20(NCS-R)&author=RC%20Kessler&author=P%20Berglund&author=O%20Demler&author=R%20Jin&author=D%20Koretz&volume=289&issue=23&publication_year=2003&pages=3095-3105&pmid=12813115&doi=10.1001/jama.289.23.3095&)

2. Thase ME, Haight BR, Richard N, Rockett CB, Mitton M, Modell JG, et al. Remission rates following antidepressant therapy with bupropion or selective serotonin reuptake inhibitors: a meta-analysis of original data from 7 randomized controlled trials. J Clin Psychiatry. 2005;66(8):974–981. doi: 10.4088/jcp.v66n0803.  [DOI](https://doi.org/10.4088/jcp.v66n0803) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16086611/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Psychiatry&title=Remission%20rates%20following%20antidepressant%20therapy%20with%20bupropion%20or%20selective%20serotonin%20reuptake%20inhibitors:%20a%20meta-analysis%20of%20original%20data%20from%207%20randomized%20controlled%20trials&author=ME%20Thase&author=BR%20Haight&author=N%20Richard&author=CB%20Rockett&author=M%20Mitton&volume=66&issue=8&publication_year=2005&pages=974-981&pmid=16086611&doi=10.4088/jcp.v66n0803&)

3. Kato M, Serretti A. Review and meta-analysis of antidepressant pharmacogenetic findings in major depressive disorder. Mol Psychiatry. 2008 doi: 10.1038/mp.2008.116.  [DOI](https://doi.org/10.1038/mp.2008.116) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18982004/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Psychiatry&title=Review%20and%20meta-analysis%20of%20antidepressant%20pharmacogenetic%20findings%20in%20major%20depressive%20disorder&author=M%20Kato&author=A%20Serretti&publication_year=2008&pmid=18982004&doi=10.1038/mp.2008.116&)

4. Mrazek DA. Psychiatric Pharmacogenomics. Oxford University Press; New York, NY: 2010.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Psychiatric%20Pharmacogenomics&author=DA%20Mrazek&publication_year=2010&)

5. Serretti A, Artioli P. The pharmacogenomics of selective serotonin reuptake inhibitors. Pharmacogenomics J. 2004;4(4):233–244. doi: 10.1038/sj.tpj.6500250.  [DOI](https://doi.org/10.1038/sj.tpj.6500250) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15111987/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=The%20pharmacogenomics%20of%20selective%20serotonin%20reuptake%20inhibitors&author=A%20Serretti&author=P%20Artioli&volume=4&issue=4&publication_year=2004&pages=233-244&pmid=15111987&doi=10.1038/sj.tpj.6500250&)

6. Motsinger-Reif AA, Jorgenson E, Relling MV, Kroetz DL, Weinshilboum R, Cox NJ, et al. Genome-wide association studies in pharmacogenomics: successes and lessons. Pharmacogenet Genomics. 2010 doi: 10.1097/FPC.0b013e32833d7b45.  [DOI](https://doi.org/10.1097/FPC.0b013e32833d7b45) | [PMC free article](/articles/PMC3003940/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20639796/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Genome-wide%20association%20studies%20in%20pharmacogenomics:%20successes%20and%20lessons&author=AA%20Motsinger-Reif&author=E%20Jorgenson&author=MV%20Relling&author=DL%20Kroetz&author=R%20Weinshilboum&publication_year=2010&pmid=20639796&doi=10.1097/FPC.0b013e32833d7b45&)

7. Garriock HA, Kraft JB, Shyn SI, Peters EJ, Yokoyama JS, Jenkins GD, et al. A genomewide association study of citalopram response in major depressive disorder. Biol Psychiatry. 2010;67(2):133–138. doi: 10.1016/j.biopsych.2009.08.029.  [DOI](https://doi.org/10.1016/j.biopsych.2009.08.029) | [PMC free article](/articles/PMC2794921/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19846067/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biol%20Psychiatry&title=A%20genomewide%20association%20study%20of%20citalopram%20response%20in%20major%20depressive%20disorder&author=HA%20Garriock&author=JB%20Kraft&author=SI%20Shyn&author=EJ%20Peters&author=JS%20Yokoyama&volume=67&issue=2&publication_year=2010&pages=133-138&pmid=19846067&doi=10.1016/j.biopsych.2009.08.029&)

8. Ising M, Lucae S, Binder EB, Bettecken T, Uhr M, Ripke S, et al. A genomewide association study points to multiple loci that predict antidepressant drug treatment outcome in depression. Arch Gen Psychiatry. 2009;66(9):966–975. doi: 10.1001/archgenpsychiatry.2009.95.  [DOI](https://doi.org/10.1001/archgenpsychiatry.2009.95) | [PMC free article](/articles/PMC4465570/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19736353/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arch%20Gen%20Psychiatry&title=A%20genomewide%20association%20study%20points%20to%20multiple%20loci%20that%20predict%20antidepressant%20drug%20treatment%20outcome%20in%20depression&author=M%20Ising&author=S%20Lucae&author=EB%20Binder&author=T%20Bettecken&author=M%20Uhr&volume=66&issue=9&publication_year=2009&pages=966-975&pmid=19736353&doi=10.1001/archgenpsychiatry.2009.95&)

9. Uher R, Perroud N, Ng MY, Hauser J, Henigsberg N, Maier W, et al. Genome-wide pharmacogenetics of antidepressant response in the GENDEP project. Am J Psychiatry. 2010;167(5):555–564. doi: 10.1176/appi.ajp.2009.09070932.  [DOI](https://doi.org/10.1176/appi.ajp.2009.09070932) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20360315/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Psychiatry&title=Genome-wide%20pharmacogenetics%20of%20antidepressant%20response%20in%20the%20GENDEP%20project&author=R%20Uher&author=N%20Perroud&author=MY%20Ng&author=J%20Hauser&author=N%20Henigsberg&volume=167&issue=5&publication_year=2010&pages=555-564&pmid=20360315&doi=10.1176/appi.ajp.2009.09070932&)

10. Laje G, McMahon FJ. Genome-wide association studies of antidepressant outcome: A brief review. Prog Neuropsychopharmacol Biol Psychiatry. 2011 doi: 10.1016/j.pnpbp.2010.11.031.  [DOI](https://doi.org/10.1016/j.pnpbp.2010.11.031) | [PMC free article](/articles/PMC3125482/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21115088/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Prog%20Neuropsychopharmacol%20Biol%20Psychiatry&title=Genome-wide%20association%20studies%20of%20antidepressant%20outcome:%20A%20brief%20review&author=G%20Laje&author=FJ%20McMahon&publication_year=2011&pmid=21115088&doi=10.1016/j.pnpbp.2010.11.031&)

11. Malhotra AK. The pharmacogenetics of depression: enter the GWAS. Am J Psychiatry. 2010;167(5):493–495. doi: 10.1176/appi.ajp.2010.10020244.  [DOI](https://doi.org/10.1176/appi.ajp.2010.10020244) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20439394/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Psychiatry&title=The%20pharmacogenetics%20of%20depression:%20enter%20the%20GWAS&author=AK%20Malhotra&volume=167&issue=5&publication_year=2010&pages=493-495&pmid=20439394&doi=10.1176/appi.ajp.2010.10020244&)

12. Ji Y, Hebbring S, Zhu H, Jenkins GD, Biernacka J, Snyder K, et al. Glycine and a glycine dehydrogenase (GLDC) SNP as citalopram/escitalopram response biomarkers in depression: pharmacometabolomics-informed pharmacogenomics. Clin Pharmacol Ther. 2010;89(1):97–104. doi: 10.1038/clpt.2010.250.  [DOI](https://doi.org/10.1038/clpt.2010.250) | [PMC free article](/articles/PMC3034442/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21107318/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Glycine%20and%20a%20glycine%20dehydrogenase%20(GLDC)%20SNP%20as%20citalopram/escitalopram%20response%20biomarkers%20in%20depression:%20pharmacometabolomics-informed%20pharmacogenomics&author=Y%20Ji&author=S%20Hebbring&author=H%20Zhu&author=GD%20Jenkins&author=J%20Biernacka&volume=89&issue=1&publication_year=2010&pages=97-104&pmid=21107318&doi=10.1038/clpt.2010.250&)

13. Ji Y, Biernacka J, Snyder K, Drews M, Pelleymounter LL, Colby C, et al. Catechol O-methyltransferase pharmacogenomics and selective serotonin reuptake inhibitor response. Pharmacogenomics J. 2010 doi: 10.1038/tpj.2010.69. [Epub ahead of print]  [DOI](https://doi.org/10.1038/tpj.2010.69) | [PMC free article](/articles/PMC3113454/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20877297/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=Catechol%20O-methyltransferase%20pharmacogenomics%20and%20selective%20serotonin%20reuptake%20inhibitor%20response&author=Y%20Ji&author=J%20Biernacka&author=K%20Snyder&author=M%20Drews&author=LL%20Pelleymounter&publication_year=2010&pmid=20877297&doi=10.1038/tpj.2010.69&)

14. Trivedi MH, Rush AJ, Wisniewski SR, Nierenberg AA, Warden D, Ritz L, et al. Evaluation of outcomes with citalopram for depression using measurement-based care in STAR*D: implications for clinical practice. Am J Psychiatry. 2006;163(1):28–40. doi: 10.1176/appi.ajp.163.1.28.  [DOI](https://doi.org/10.1176/appi.ajp.163.1.28) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16390886/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Psychiatry&title=Evaluation%20of%20outcomes%20with%20citalopram%20for%20depression%20using%20measurement-based%20care%20in%20STAR*D:%20implications%20for%20clinical%20practice&author=MH%20Trivedi&author=AJ%20Rush&author=SR%20Wisniewski&author=AA%20Nierenberg&author=D%20Warden&volume=163&issue=1&publication_year=2006&pages=28-40&pmid=16390886&doi=10.1176/appi.ajp.163.1.28&)

15. Purcell S, Neale B, Todd-Brown K, Thomas L, Ferreira MA, Bender D, et al. PLINK: a tool set for whole-genome association and population-based linkage analyses. Am J Hum Genet. 2007;81(3):559–575. doi: 10.1086/519795.  [DOI](https://doi.org/10.1086/519795) | [PMC free article](/articles/PMC1950838/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17701901/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=PLINK:%20a%20tool%20set%20for%20whole-genome%20association%20and%20population-based%20linkage%20analyses&author=S%20Purcell&author=B%20Neale&author=K%20Todd-Brown&author=L%20Thomas&author=MA%20Ferreira&volume=81&issue=3&publication_year=2007&pages=559-575&pmid=17701901&doi=10.1086/519795&)

16. Pritchard JK, Stephens M, Donnelly P. Inference of population structure using multilocus genotype data. Genetics. 2000;155(2):945–959. doi: 10.1093/genetics/155.2.945.  [DOI](https://doi.org/10.1093/genetics/155.2.945) | [PMC free article](/articles/PMC1461096/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10835412/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genetics&title=Inference%20of%20population%20structure%20using%20multilocus%20genotype%20data&author=JK%20Pritchard&author=M%20Stephens&author=P%20Donnelly&volume=155&issue=2&publication_year=2000&pages=945-959&pmid=10835412&doi=10.1093/genetics/155.2.945&)

17. Price AL, Patterson NJ, Plenge RM, Weinblatt ME, Shadick NA, Reich D. Principal components analysis corrects for stratification in genome-wide association studies. Nat Genet. 2006;38(8):904–909. doi: 10.1038/ng1847.  [DOI](https://doi.org/10.1038/ng1847) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16862161/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Principal%20components%20analysis%20corrects%20for%20stratification%20in%20genome-wide%20association%20studies&author=AL%20Price&author=NJ%20Patterson&author=RM%20Plenge&author=ME%20Weinblatt&author=NA%20Shadick&volume=38&issue=8&publication_year=2006&pages=904-909&pmid=16862161&doi=10.1038/ng1847&)

18. Niu N, Qin Y, Fridley BL, Hou J, Kalari KR, Zhu M, et al. Radiation pharmacogenomics: a genome-wide association approach to identify radiation response biomarkers using human lymphoblastoid cell lines. Genome Res. 2010;20(11):1482–1492. doi: 10.1101/gr.107672.110.  [DOI](https://doi.org/10.1101/gr.107672.110) | [PMC free article](/articles/PMC2963812/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20923822/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Res&title=Radiation%20pharmacogenomics:%20a%20genome-wide%20association%20approach%20to%20identify%20radiation%20response%20biomarkers%20using%20human%20lymphoblastoid%20cell%20lines&author=N%20Niu&author=Y%20Qin&author=BL%20Fridley&author=J%20Hou&author=KR%20Kalari&volume=20&issue=11&publication_year=2010&pages=1482-1492&pmid=20923822&doi=10.1101/gr.107672.110&)

19. Hustad S, Midttun O, Schneede J, Vollset SE, Grotmol T, Ueland PM. The methylenetetrahydrofolate reductase 677C-->T polymorphism as a modulator of a B vitamin network with major effects on homocysteine metabolism. Am J Hum Genet. 2007;80(5):846–855. doi: 10.1086/513520.  [DOI](https://doi.org/10.1086/513520) | [PMC free article](/articles/PMC1852731/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17436239/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=The%20methylenetetrahydrofolate%20reductase%20677C-->T%20polymorphism%20as%20a%20modulator%20of%20a%20B%20vitamin%20network%20with%20major%20effects%20on%20homocysteine%20metabolism&author=S%20Hustad&author=O%20Midttun&author=J%20Schneede&author=SE%20Vollset&author=T%20Grotmol&volume=80&issue=5&publication_year=2007&pages=846-855&pmid=17436239&doi=10.1086/513520&)

20. Smythies JR, Alarcon RD, Morere D, Monti JA, Steele M, Tolbert LC, et al. Abnormalities of one-carbon metabolism in psychiatric disorders: study of methionine adenosyltransferase kinetics and lipid composition of erythrocyte membranes. Biol Psychiatry. 1986;21(14):1391–1398. doi: 10.1016/0006-3223(86)90330-6.  [DOI](https://doi.org/10.1016/0006-3223(86)90330-6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/3790625/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biol%20Psychiatry&title=Abnormalities%20of%20one-carbon%20metabolism%20in%20psychiatric%20disorders:%20study%20of%20methionine%20adenosyltransferase%20kinetics%20and%20lipid%20composition%20of%20erythrocyte%20membranes&author=JR%20Smythies&author=RD%20Alarcon&author=D%20Morere&author=JA%20Monti&author=M%20Steele&volume=21&issue=14&publication_year=1986&pages=1391-1398&pmid=3790625&doi=10.1016/0006-3223(86)90330-6&)

21. Alarcon RD, Tolbert LC, Monti JA, Morere DA, Walter-Ryan WG, Kemp B, et al. One-carbon metabolism disturbances in affective disorders. A preliminary report. J Affect Disord. 1985;9(3):297–301. doi: 10.1016/0165-0327(85)90060-6.  [DOI](https://doi.org/10.1016/0165-0327(85)90060-6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/2934462/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Affect%20Disord&title=One-carbon%20metabolism%20disturbances%20in%20affective%20disorders.%20A%20preliminary%20report&author=RD%20Alarcon&author=LC%20Tolbert&author=JA%20Monti&author=DA%20Morere&author=WG%20Walter-Ryan&volume=9&issue=3&publication_year=1985&pages=297-301&pmid=2934462&doi=10.1016/0165-0327(85)90060-6&)

22. Reynolds EH, Carney MW, Toone BK. Methylation and mood. Lancet. 1984;2(8396):196–198. doi: 10.1016/s0140-6736(84)90482-3.  [DOI](https://doi.org/10.1016/s0140-6736(84)90482-3) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/6146753/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Methylation%20and%20mood&author=EH%20Reynolds&author=MW%20Carney&author=BK%20Toone&volume=2&issue=8396&publication_year=1984&pages=196-198&pmid=6146753&doi=10.1016/s0140-6736(84)90482-3&)

23. Stanger O, Fowler B, Piertzik K, Huemer M, Haschke-Becher E, Semmler A, et al. Homocysteine, folate and vitamin B12 in neuropsychiatric diseases: review and treatment recommendations. Expert Rev Neurother. 2009;9(9):1393–1412. doi: 10.1586/ern.09.75.  [DOI](https://doi.org/10.1586/ern.09.75) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19769453/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Expert%20Rev%20Neurother&title=Homocysteine,%20folate%20and%20vitamin%20B12%20in%20neuropsychiatric%20diseases:%20review%20and%20treatment%20recommendations&author=O%20Stanger&author=B%20Fowler&author=K%20Piertzik&author=M%20Huemer&author=E%20Haschke-Becher&volume=9&issue=9&publication_year=2009&pages=1393-1412&pmid=19769453&doi=10.1586/ern.09.75&)

24. Lewis SJ, Lawlor DA, Davey Smith G, Araya R, Timpson N, Day IN, et al. The thermolabile variant of MTHFR is associated with depression in the British Women’s Heart and Health Study and a meta-analysis. Mol Psychiatry. 2006;11(4):352–360. doi: 10.1038/sj.mp.4001790.  [DOI](https://doi.org/10.1038/sj.mp.4001790) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16402130/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Psychiatry&title=The%20thermolabile%20variant%20of%20MTHFR%20is%20associated%20with%20depression%20in%20the%20British%20Women%E2%80%99s%20Heart%20and%20Health%20Study%20and%20a%20meta-analysis&author=SJ%20Lewis&author=DA%20Lawlor&author=G%20Davey%20Smith&author=R%20Araya&author=N%20Timpson&volume=11&issue=4&publication_year=2006&pages=352-360&pmid=16402130&doi=10.1038/sj.mp.4001790&)

25. Murakami K, Miyake Y, Sasaki S, Tanaka K, Arakawa M. Dietary folate, riboflavin, vitamin B-6, and vitamin B-12 and depressive symptoms in early adolescence: the Ryukyus Child Health Study. Psychosom Med. 2010;72(8):763–768. doi: 10.1097/PSY.0b013e3181f02f15.  [DOI](https://doi.org/10.1097/PSY.0b013e3181f02f15) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20716710/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Psychosom%20Med&title=Dietary%20folate,%20riboflavin,%20vitamin%20B-6,%20and%20vitamin%20B-12%20and%20depressive%20symptoms%20in%20early%20adolescence:%20the%20Ryukyus%20Child%20Health%20Study&author=K%20Murakami&author=Y%20Miyake&author=S%20Sasaki&author=K%20Tanaka&author=M%20Arakawa&volume=72&issue=8&publication_year=2010&pages=763-768&pmid=20716710&doi=10.1097/PSY.0b013e3181f02f15&)

26. Chumakov I, Blumenfeld M, Guerassimenko O, Cavarec L, Palicio M, Abderrahim H, et al. Genetic and physiological data implicating the new human gene G72 and the gene for D-amino acid oxidase in schizophrenia. Proc Natl Acad Sci U S A. 2002;99(21):13675–13680. doi: 10.1073/pnas.182412499.  [DOI](https://doi.org/10.1073/pnas.182412499) | [PMC free article](/articles/PMC129739/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12364586/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&title=Genetic%20and%20physiological%20data%20implicating%20the%20new%20human%20gene%20G72%20and%20the%20gene%20for%20D-amino%20acid%20oxidase%20in%20schizophrenia&author=I%20Chumakov&author=M%20Blumenfeld&author=O%20Guerassimenko&author=L%20Cavarec&author=M%20Palicio&volume=99&issue=21&publication_year=2002&pages=13675-13680&pmid=12364586&doi=10.1073/pnas.182412499&)

27. Wang X, He G, Gu N, Yang J, Tang J, Chen Q, et al. Association of G72/G30 with schizophrenia in the Chinese population. Biochem Biophys Res Commun. 2004;319(4):1281–1286. doi: 10.1016/j.bbrc.2004.05.119.  [DOI](https://doi.org/10.1016/j.bbrc.2004.05.119) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15194506/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem%20Biophys%20Res%20Commun&title=Association%20of%20G72/G30%20with%20schizophrenia%20in%20the%20Chinese%20population&author=X%20Wang&author=G%20He&author=N%20Gu&author=J%20Yang&author=J%20Tang&volume=319&issue=4&publication_year=2004&pages=1281-1286&pmid=15194506&doi=10.1016/j.bbrc.2004.05.119&)

28. Schumacher J, Jamra RA, Freudenberg J, Becker T, Ohlraun S, Otte AC, et al. Examination of G72 and D-amino-acid oxidase as genetic risk factors for schizophrenia and bipolar affective disorder. Mol Psychiatry. 2004;9(2):203–207. doi: 10.1038/sj.mp.4001421.  [DOI](https://doi.org/10.1038/sj.mp.4001421) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14966479/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Psychiatry&title=Examination%20of%20G72%20and%20D-amino-acid%20oxidase%20as%20genetic%20risk%20factors%20for%20schizophrenia%20and%20bipolar%20affective%20disorder&author=J%20Schumacher&author=RA%20Jamra&author=J%20Freudenberg&author=T%20Becker&author=S%20Ohlraun&volume=9&issue=2&publication_year=2004&pages=203-207&pmid=14966479&doi=10.1038/sj.mp.4001421&)

29. Nixon DC, Prust MJ, Sambataro F, Tan HY, Mattay VS, Weinberger DR, et al. Interactive effects of DAOA (G72) and catechol-O-methyltransferase on neurophysiology in prefrontal cortex. Biol Psychiatry. 2011;69(10):1006–1008. doi: 10.1016/j.biopsych.2010.10.031.  [DOI](https://doi.org/10.1016/j.biopsych.2010.10.031) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21215384/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biol%20Psychiatry&title=Interactive%20effects%20of%20DAOA%20(G72)%20and%20catechol-O-methyltransferase%20on%20neurophysiology%20in%20prefrontal%20cortex&author=DC%20Nixon&author=MJ%20Prust&author=F%20Sambataro&author=HY%20Tan&author=VS%20Mattay&volume=69&issue=10&publication_year=2011&pages=1006-1008&pmid=21215384&doi=10.1016/j.biopsych.2010.10.031&)

30. Gaysina D, Cohen-Woods S, Chow PC, Martucci L, Schosser A, Ball HA, et al. Association analysis of DAOA and DAO in bipolar disorder: results from two independent case-control studies. Bipolar Disord. 2010;12(5):579–581. doi: 10.1111/j.1399-5618.2010.00837.x.  [DOI](https://doi.org/10.1111/j.1399-5618.2010.00837.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20712760/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bipolar%20Disord&title=Association%20analysis%20of%20DAOA%20and%20DAO%20in%20bipolar%20disorder:%20results%20from%20two%20independent%20case-control%20studies&author=D%20Gaysina&author=S%20Cohen-Woods&author=PC%20Chow&author=L%20Martucci&author=A%20Schosser&volume=12&issue=5&publication_year=2010&pages=579-581&pmid=20712760&doi=10.1111/j.1399-5618.2010.00837.x&)

31. Addington AM, Gornick M, Sporn AL, Gogtay N, Greenstein D, Lenane M, et al. Polymorphisms in the 13q33. 2 gene G72/G30 are associated with childhood-onset schizophrenia and psychosis not otherwise specified. Biol Psychiatry. 2004;55(10):976–980. doi: 10.1016/j.biopsych.2004.01.024.  [DOI](https://doi.org/10.1016/j.biopsych.2004.01.024) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15121480/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biol%20Psychiatry&title=Polymorphisms%20in%20the%2013q33.%202%20gene%20G72/G30%20are%20associated%20with%20childhood-onset%20schizophrenia%20and%20psychosis%20not%20otherwise%20specified&author=AM%20Addington&author=M%20Gornick&author=AL%20Sporn&author=N%20Gogtay&author=D%20Greenstein&volume=55&issue=10&publication_year=2004&pages=976-980&pmid=15121480&doi=10.1016/j.biopsych.2004.01.024&)

32. Villafuerte SM, Vallabhaneni K, Sliwerska E, McMahon FJ, Young EA, Burmeister M. SSRI response in depression may be influenced by SNPs in HTR1B and HTR1A. Psychiatr Genet. 2009;19(6):281–291. doi: 10.1097/YPG.0b013e32832a506e.  [DOI](https://doi.org/10.1097/YPG.0b013e32832a506e) | [PMC free article](/articles/PMC2783179/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19829169/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Psychiatr%20Genet&title=SSRI%20response%20in%20depression%20may%20be%20influenced%20by%20SNPs%20in%20HTR1B%20and%20HTR1A&author=SM%20Villafuerte&author=K%20Vallabhaneni&author=E%20Sliwerska&author=FJ%20McMahon&author=EA%20Young&volume=19&issue=6&publication_year=2009&pages=281-291&pmid=19829169&doi=10.1097/YPG.0b013e32832a506e&)

33. Mekli K, Payton A, Miyajima F, Platt H, Thomas E, Downey D, et al. The HTR1A and HTR1B receptor genes influence stress-related information processing. Eur Neuropsychopharmacol. 2011;21(1):129–139. doi: 10.1016/j.euroneuro.2010.06.013.  [DOI](https://doi.org/10.1016/j.euroneuro.2010.06.013) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20638825/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20Neuropsychopharmacol&title=The%20HTR1A%20and%20HTR1B%20receptor%20genes%20influence%20stress-related%20information%20processing&author=K%20Mekli&author=A%20Payton&author=F%20Miyajima&author=H%20Platt&author=E%20Thomas&volume=21&issue=1&publication_year=2011&pages=129-139&pmid=20638825&doi=10.1016/j.euroneuro.2010.06.013&)

34. Dickel DE, Veenstra-VanderWeele J, Bivens NC, Wu X, Fischer DJ, Van Etten-Lee M, et al. Association studies of serotonin system candidate genes in early-onset obsessive-compulsive disorder. Biol Psychiatry. 2007;61(3):322–329. doi: 10.1016/j.biopsych.2006.09.030.  [DOI](https://doi.org/10.1016/j.biopsych.2006.09.030) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17241828/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biol%20Psychiatry&title=Association%20studies%20of%20serotonin%20system%20candidate%20genes%20in%20early-onset%20obsessive-compulsive%20disorder&author=DE%20Dickel&author=J%20Veenstra-VanderWeele&author=NC%20Bivens&author=X%20Wu&author=DJ%20Fischer&volume=61&issue=3&publication_year=2007&pages=322-329&pmid=17241828&doi=10.1016/j.biopsych.2006.09.030&)

35. Cao JX, Hu J, Ye XM, Xia Y, Haile CA, Kosten TR, et al. Association between the 5-HTR1B gene polymorphisms and alcohol dependence in a Han Chinese population. Brain Res. 2011;1376:1–9. doi: 10.1016/j.brainres.2010.12.039.  [DOI](https://doi.org/10.1016/j.brainres.2010.12.039) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21172311/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Brain%20Res&title=Association%20between%20the%205-HTR1B%20gene%20polymorphisms%20and%20alcohol%20dependence%20in%20a%20Han%20Chinese%20population&author=JX%20Cao&author=J%20Hu&author=XM%20Ye&author=Y%20Xia&author=CA%20Haile&volume=1376&publication_year=2011&pages=1-9&pmid=21172311&doi=10.1016/j.brainres.2010.12.039&)

36. Garcia-Sevilla JA, Alvaro-Bartolome M, Diez-Alarcia R, Ramos-Miguel A, Puigdemont D, Perez V, et al. Reduced platelet G protein-coupled receptor kinase 2 in major depressive disorder: antidepressant treatment-induced upregulation of GRK2 protein discriminates between responder and non-responder patients. Eur Neuropsychopharmacol. 2010;20(10):721–730. doi: 10.1016/j.euroneuro.2010.04.008.  [DOI](https://doi.org/10.1016/j.euroneuro.2010.04.008) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20493668/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20Neuropsychopharmacol&title=Reduced%20platelet%20G%20protein-coupled%20receptor%20kinase%202%20in%20major%20depressive%20disorder:%20antidepressant%20treatment-induced%20upregulation%20of%20GRK2%20protein%20discriminates%20between%20responder%20and%20non-responder%20patients&author=JA%20Garcia-Sevilla&author=M%20Alvaro-Bartolome&author=R%20Diez-Alarcia&author=A%20Ramos-Miguel&author=D%20Puigdemont&volume=20&issue=10&publication_year=2010&pages=721-730&pmid=20493668&doi=10.1016/j.euroneuro.2010.04.008&)

37. Hu LA, Chen W, Premont RT, Cong M, Lefkowitz RJ. G protein-coupled receptor kinase 5 regulates beta 1-adrenergic receptor association with PSD-95. J Biol Chem. 2002;277(2):1607–1613. doi: 10.1074/jbc.M107297200.  [DOI](https://doi.org/10.1074/jbc.M107297200) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11700307/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=G%20protein-coupled%20receptor%20kinase%205%20regulates%20beta%201-adrenergic%20receptor%20association%20with%20PSD-95&author=LA%20Hu&author=W%20Chen&author=RT%20Premont&author=M%20Cong&author=RJ%20Lefkowitz&volume=277&issue=2&publication_year=2002&pages=1607-1613&pmid=11700307&doi=10.1074/jbc.M107297200&)

38. Tiberi M, Nash SR, Bertrand L, Lefkowitz RJ, Caron MG. Differential regulation of dopamine D1A receptor responsiveness by various G protein-coupled receptor kinases. J Biol Chem. 1996;271(7):3771–3778. doi: 10.1074/jbc.271.7.3771.  [DOI](https://doi.org/10.1074/jbc.271.7.3771) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8631993/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=Differential%20regulation%20of%20dopamine%20D1A%20receptor%20responsiveness%20by%20various%20G%20protein-coupled%20receptor%20kinases&author=M%20Tiberi&author=SR%20Nash&author=L%20Bertrand&author=RJ%20Lefkowitz&author=MG%20Caron&volume=271&issue=7&publication_year=1996&pages=3771-3778&pmid=8631993&doi=10.1074/jbc.271.7.3771&)

39. Rockman HA, Choi DJ, Rahman NU, Akhter SA, Lefkowitz RJ, Koch WJ. Receptor-specific in vivo desensitization by the G protein-coupled receptor kinase-5 in transgenic mice. Proc Natl Acad Sci U S A. 1996;93(18):9954–9959. doi: 10.1073/pnas.93.18.9954.  [DOI](https://doi.org/10.1073/pnas.93.18.9954) | [PMC free article](/articles/PMC38536/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8790438/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&title=Receptor-specific%20in%20vivo%20desensitization%20by%20the%20G%20protein-coupled%20receptor%20kinase-5%20in%20transgenic%20mice&author=HA%20Rockman&author=DJ%20Choi&author=NU%20Rahman&author=SA%20Akhter&author=RJ%20Lefkowitz&volume=93&issue=18&publication_year=1996&pages=9954-9959&pmid=8790438&doi=10.1073/pnas.93.18.9954&)

40. Gainetdinov RR, Bohn LM, Walker JK, Laporte SA, Macrae AD, Caron MG, et al. Muscarinic supersensitivity and impaired receptor desensitization in G protein-coupled receptor kinase 5-deficient mice. Neuron. 1999;24(4):1029–1036. doi: 10.1016/s0896-6273(00)81048-x.  [DOI](https://doi.org/10.1016/s0896-6273(00)81048-x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10624964/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neuron&title=Muscarinic%20supersensitivity%20and%20impaired%20receptor%20desensitization%20in%20G%20protein-coupled%20receptor%20kinase%205-deficient%20mice&author=RR%20Gainetdinov&author=LM%20Bohn&author=JK%20Walker&author=SA%20Laporte&author=AD%20Macrae&volume=24&issue=4&publication_year=1999&pages=1029-1036&pmid=10624964&doi=10.1016/s0896-6273(00)81048-x&)

41. Liggett SB, Cresci S, Kelly RJ, Syed FM, Matkovich SJ, Hahn HS, et al. A GRK5 polymorphism that inhibits beta-adrenergic receptor signaling is protective in heart failure. Nat Med. 2008;14(5):510–517. doi: 10.1038/nm1750.  [DOI](https://doi.org/10.1038/nm1750) | [PMC free article](/articles/PMC2596476/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18425130/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Med&title=A%20GRK5%20polymorphism%20that%20inhibits%20beta-adrenergic%20receptor%20signaling%20is%20protective%20in%20heart%20failure&author=SB%20Liggett&author=S%20Cresci&author=RJ%20Kelly&author=FM%20Syed&author=SJ%20Matkovich&volume=14&issue=5&publication_year=2008&pages=510-517&pmid=18425130&doi=10.1038/nm1750&)

42. Lobmeyer MT, Wang L, Zineh I, Turner ST, Gums JG, Chapman AB, et al. Polymorphisms in genes coding for GRK2 and GRK5 and response differences in antihypertensive-treated patients. Pharmacogenet Genomics. 2011;21(1):42–49. doi: 10.1097/FPC.0b013e328341e911.  [DOI](https://doi.org/10.1097/FPC.0b013e328341e911) | [PMC free article](/articles/PMC3028503/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21127457/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Polymorphisms%20in%20genes%20coding%20for%20GRK2%20and%20GRK5%20and%20response%20differences%20in%20antihypertensive-treated%20patients&author=MT%20Lobmeyer&author=L%20Wang&author=I%20Zineh&author=ST%20Turner&author=JG%20Gums&volume=21&issue=1&publication_year=2011&pages=42-49&pmid=21127457&doi=10.1097/FPC.0b013e328341e911&)

43. Marchetto MC, Brennand KJ, Boyer LF, Gage FH. Induced pluripotent stem cells (iPSCs) and neurological disease modeling: progress and promises. Hum Mol Genet. 2011;20(R2):R109–115. doi: 10.1093/hmg/ddr336.  [DOI](https://doi.org/10.1093/hmg/ddr336) | [PMC free article](/articles/PMC4447776/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21828073/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mol%20Genet&title=Induced%20pluripotent%20stem%20cells%20(iPSCs)%20and%20neurological%20disease%20modeling:%20progress%20and%20promises&author=MC%20Marchetto&author=KJ%20Brennand&author=LF%20Boyer&author=FH%20Gage&volume=20&issue=R2&publication_year=2011&pages=R109-115&pmid=21828073&doi=10.1093/hmg/ddr336&)

44. Takeuchi F, McGinnis R, Bourgeois S, Barnes C, Eriksson N, Soranzo N, et al. A genome-wide association study confirms VKORC1, CYP2C9, and CYP4F2 as principal genetic determinants of warfarin dose. PLoS genetics. 2009;5(3):e1000433. doi: 10.1371/journal.pgen.1000433.  [DOI](https://doi.org/10.1371/journal.pgen.1000433) | [PMC free article](/articles/PMC2652833/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19300499/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20genetics&title=A%20genome-wide%20association%20study%20confirms%20VKORC1,%20CYP2C9,%20and%20CYP4F2%20as%20principal%20genetic%20determinants%20of%20warfarin%20dose&author=F%20Takeuchi&author=R%20McGinnis&author=S%20Bourgeois&author=C%20Barnes&author=N%20Eriksson&volume=5&issue=3&publication_year=2009&pages=e1000433&pmid=19300499&doi=10.1371/journal.pgen.1000433&)

45. Shuldiner AR, O’Connell JR, Bliden KP, Gandhi A, Ryan K, Horenstein RB, et al. Association of cytochrome P450 2C19 genotype with the antiplatelet effect and clinical efficacy of clopidogrel therapy. Jama. 2009;302(8):849–857. doi: 10.1001/jama.2009.1232.  [DOI](https://doi.org/10.1001/jama.2009.1232) | [PMC free article](/articles/PMC3641569/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19706858/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Jama&title=Association%20of%20cytochrome%20P450%202C19%20genotype%20with%20the%20antiplatelet%20effect%20and%20clinical%20efficacy%20of%20clopidogrel%20therapy&author=AR%20Shuldiner&author=JR%20O%E2%80%99Connell&author=KP%20Bliden&author=A%20Gandhi&author=K%20Ryan&volume=302&issue=8&publication_year=2009&pages=849-857&pmid=19706858&doi=10.1001/jama.2009.1232&)

46. Ingle JN, Schaid DJ, Goss PE, Liu M, Mushiroda T, Chapman JA, et al. Genome-wide associations and functional genomic studies of musculoskeletal adverse events in women receiving aromatase inhibitors. J Clin Oncol. 28(31):4674–4682. doi: 10.1200/JCO.2010.28.5064.  [DOI](https://doi.org/10.1200/JCO.2010.28.5064) | [PMC free article](/articles/PMC3020700/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20876420/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Genome-wide%20associations%20and%20functional%20genomic%20studies%20of%20musculoskeletal%20adverse%20events%20in%20women%20receiving%20aromatase%20inhibitors&author=JN%20Ingle&author=DJ%20Schaid&author=PE%20Goss&author=M%20Liu&author=T%20Mushiroda&volume=28&issue=31&pages=4674-4682&pmid=20876420&doi=10.1200/JCO.2010.28.5064&)
