# Genome-wide study of methotrexate clearance replicates SLCO1B1

## Metadata
**Authors:** Laura B Ramsey, John C Panetta, Colton Smith, Wenjian Yang, Yiping Fan, Naomi J Winick, Paul L Martin, Cheng Cheng, Meenakshi Devidas, Ching-Hon Pui, William E Evans, Stephen P Hunger, Mignon Loh, Mary V Relling
**Journal:** Blood
**Date:** 2012 Dec 11
**DOI:** [10.1182/blood-2012-08-452839](https://doi.org/10.1182/blood-2012-08-452839)
**PMID:** 23233662
**PMCID:** PMC3567337
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3567337/

## Abstract

A genome-wide study of the association of over 5 million SNPs with methotrexate clearance in 1279 patients treated with HDMTX in multicenter COG trials 9904 and 9905.

We replicated the finding that inherited variations in SLCO1B1 are the most important genetic variations influencing methotrexate clearance.

## Introduction

High-dose methotrexate (HDMTX) is an important element of chemotherapy in the treatment of acute lymphoblastic leukemia (ALL) and other malignancies.^1–4^ Mikkelsen et al^5^ showed that a 1 g/m^2^ dose given over 24 hours resulted in significantly greater active methotrexate polyglutamates in leukemic cells than the same dose given over 4 hours. In a randomized clinical trial, investigators from the Children's Oncology Group (COG) tested the efficacy and toxicity of a 2 g/m^2^ dose over 4 hours versus a 1 g/m^2^ dose over 24 hours.^6,7^ The systemic exposure to methotrexate (ie, plasma concentration over time) is related to cure and toxicity in children with ALL.^8,9^ In a group of children with ALL who were treated and monitored at a single institution (St Jude Children's Research Hospital), with extensive prospective therapeutic drug monitoring, even including pharmacokinetically guided dosage adjustments, we identified variants in the *SLCO1B1* gene that were associated with methotrexate clearance in a genome-wide association study (GWAS).^10,11^ Herein, we sought to test whether we could replicate these GWAS findings in a large cohort of patients treated with alternative HDMTX schedules on the COG multi-institutional trials P9904 and P9905.

## Methods

This study was approved by the institutional review boards of all participating institutions, and informed consent was obtained in accordance with the Declaration of Helsinki. Patients in P9904 included National Cancer Institute (NCI) standard-risk (age 1.00-9.99 years and initial white blood cell count [WBC]< 50 000/μL) patients with an *ETV6-RUNX1* translocation or simultaneous trisomies of chromosomes 4 and 10, whereas patients in P9905 included a mixture of NCI standard-risk patients without favorable genetic lesions, NCI high-risk (age ≥ 10 years and/or initial WBC ≥ 50 000/μL) patients with favorable genetic changes, and other NCI high-risk patients who did not meet age-, WBC-, and sex-specific criteria for especially high-risk disease originally described by Borowitz et al^6^ and Shuster et al.^12^ Patients were randomized in a 2 × 2 manner to 1 of 4 arms for consolidation: (A) 24-hour methotrexate infusion (1 g/m^2^ given as a 200 mg/m^2^ bolus over 20 minutes, followed by 800 mg/m^2^ over 23.6 hours) and no delayed intensification (DI) phase; (B) 4-hour methotrexate infusion (2 g/m^2^) with no DI; (C) 24-hour methotrexate infusion with DI; and (D) 4-hour methotrexate infusion with DI. Leucovorin at 10 mg/m^2^ was given every 6 hours (3 doses), beginning at 42 hours after the start of the infusion and continued until the plasma methotrexate level was less than 0.2μM. Plasma methotrexate concentrations were requested to be drawn at the end of infusion (4 or 24 hours) and 24 hours later (postinfusion), and leucovorin was increased based on plasma methotrexate concentration (details in supplemental Methods, available on the *Blood* Web site; see the Supplemental Materials link at the top of the online article). All patients with t(1;19)(q23;p13) were in P9905 as a separate stratum—all were assigned to receive a DI phase and randomized to arms C or D for the methotrexate infusion. Patients with simultaneous trisomies of chromosomes 4 and 10 all were assigned to receive no DI and randomized to arms A or B for the methotrexate infusion. Patients who did not respond to induction chemotherapy were not eligible.

Data were retrieved from the COG Statistical and Data Center research databases. Of the 1883 patients treated with HDMTX on P9904 and P9905, 1588 were eligible and had germline DNA evaluable.^6^ Patients who had congenital abnormalities (usually Down syndrome, which is known to affect methotrexate clearance)^13–15^ or who did not have any courses that passed methotrexate quality control were excluded ([Figure 1](#F1)). Demographic information on the patients included can be found in supplemental Table 1. Although participating institutions in COG P9904/9905 were asked to submit data on plasma methotrexate concentrations, the data had not previously undergone any analysis or quality control. Therefore, it was necessary to establish a quality control approach to exclude courses with implausible data; for example, courses with undetectable plasma concentrations of methotrexate at the end of infusion were considered implausible and likely due to errors in drawing samples or in data entry. Courses with only 1 sample were excluded, as were courses for which the concentration of the postinfusion sample (28 or 48 hours) was greater than the end of infusion sample (4 or 24 hours). In addition, samples were removed that fell outside the expected 1-99th percentile range of methotrexate concentrations predicted with the use of the methotrexate pharmacokinetic parameters observed in nearly 500 prospectively monitored patients from St Jude's Total XV study (supplemental Table 2).^10,16^

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9bf7/3567337/a4416e998a1d/zh89991301740001.jpg)

Flow chart of patients included in this analysis. A total of 1883 patients were treated with HDMTX on P9904 and P9905, and 1588 were eligible and had germline DNA evaluable. Patients who had congenital abnormalities (usually Down syndrome) were excluded because they may not have received the full methotrexate dose. Patients with implausible or no evaluable methotrexate plasma concentrations were excluded (see “Methods” for details).

In practice, this step resulted in the following: for the 24-hour infusion arms, end of infusion samples less than 2μM or greater than 38.5μM were excluded, and 48-hour samples greater than 7.5μM were excluded; for the 4-hour infusion arms, end of infusion samples less than 25μM or greater than 310μM were excluded, and 28-hour samples greater than 25μM were excluded. Low postinfusion samples were not excluded because they were considered plausible, whereas undetectable concentrations at the end of infusion ([Figure 2](#F2)) must be related to sampling or data entry errors. Thus, 252 of 4205 24-hour infusion courses (15 of 730 patients) and 653 of 4277 4-hour infusion courses (39 of 749 patients) were excluded on the basis of implausible pharmacokinetic data (likely due to data entry or sample acquisition error; [Figure 2](#F2)). Overall, the median clearance for all courses after excluding the outliers was similar to the median clearance prior to excluding the outliers (144.6 vs 141.2 mL/min/m^2^ for the 4-hour and 162.5 vs 162.3 mL/min/m^2^ for the 24-hour infusions).

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9bf7/3567337/6dfe6ccd48f6/zh89991301740002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3567337_zh89991301740002.jpg)

Concentration versus time plots. (A) Twenty-four-hour methotrexate infusion of 1 g/m2. Concentrations shown were scheduled to be drawn at 24 hours (end of infusion) and at 48 hours (data are jittered [horizontal offset added] for better visualization). The pink lines represent the cut-offs (based on the range of the St Jude pharmacokinetic simulation) outside of which samples were excluded. There is no lower threshold for exclusion (pink line) for postinfusion samples. (B) Plasma concentrations were estimated for St Jude Total XV patients on the basis of a simulated dose of 1 g/m2 methotrexate over 24 hours (solid black line) and COG patients (dotted black line) receiving the same dose. The 1-99th percentile range is shown with shading for the St Jude Total XV patients (blue) and COG patients (red). (C) 4-hour methotrexate infusion of 2 g/m2. Concentrations shown were scheduled to be drawn at 4 hours (end of infusion) and at 28 hours (data are jittered for better visualization). (D) Plasma concentrations were estimated for St Jude Total XV patients based on a simulated dose of 2 g/m2 methotrexate over 4 hours (solid black line) and for COG patients (dotted black line) receiving the same dose. The 1-99th percentile range is shown with shading for the St Jude Total XV patients (blue) and COG patients (red).

Nonlinear mixed-effects modeling (Monolix 3.1; Lixoft) was used to determine population and individual posthoc methotrexate pharmacokinetic parameters. Specifically, a 2-compartment pharmacokinetic model with first-order elimination was fit to the data. The distribution of the parameters (both interoccasion and interindividual) was assumed log-normal. A proportional residual error model was used assuming normal distribution of the residuals. Furthermore, because of the sparse nature of the sampling in the current study, we included as a prior distribution the methotrexate pharmacokinetics from the St Jude Total XV study (supplemental Table 2). For the genome-wide analysis, we used the average methotrexate clearance value per patient.^10^

Germline DNA was isolated at the time of remission from peripheral blood. DNA was extracted and genotyped using the Genome-Wide Human SNP Array 6.0 as described (Affymetrix, Santa Clara, CA).^17^ DNA also was genotyped by the Illumina GoldenGate platform for 7 single-nucleotide polymorphisms (SNP; SNP Center, Johns Hopkins University). We did not perform sequencing or genotype for any rare variants. Genotype calls were coded as 0, 1, or 2 for the number of B alleles (AA, AB, BB). SNPs with a minor allele frequency less than 1% or a call rate less than 95% were excluded. Imputation of SNPs in the 1000 Genomes dataset not represented on the Affymetrix SNP 6.0 array was performed on each ancestral group independently using MACH1.0 (see supplemental Methods).^18^

A GWAS between germline SNP genotypes and log2-transformed methotrexate clearance was performed with the use of a general linear model assuming an additive effect of allelic dosage, with all models including age, genetic ancestry, sex, and treatment regimen as covariates, as we previously described.^10,11^ Genetic ancestry was estimated with STRUCTURE as previously described (see supplemental Methods).^17^ Age was coded in years and treated as a continuous variable. Treatment regimen was coded as a categorical variable. Adjusted methotrexate clearance refers to the residual from a linear regression including the clinical covariates. A meta-analysis was performed by combining the statistics from the current GWAS with those obtained from the GWAS of methotrexate clearance in St Jude patients.^10^ We calculated the combined *P* values from the meta-analysis using the Stouffer method.^19,20^

## Results

We initially tested covariates for their association with methotrexate clearance in a multivariate general linear model ([Table 1](#T1)). Receipt of the 4-hour (rather than the 24-hour infusion; *P* < 2 × 10^−16^; [Figure 3](#F3)A) treatment arm, including a DI phase (*P* = .0010; supplemental Figure 1), older age (*P* = 9 × 10^−7^; supplemental Figure 2, consistent with previous reports),^10^ and female sex (*P* = .000 28, supplemental Figure 3) were all associated with lower clearance. Methotrexate clearance was not related to genetically defined patient ancestry in this study (supplemental Figure 4). In sum, we were able to explain 42.1% of the interindividual variability in methotrexate clearance with the clinical covariates and 1 SNP in *SLCO1B1*, rs4149056 ([Table 1](#T1)), in the same range as in our previous study.^11^

### Table 1.

| Factor | Direction | P* | R2* | P† | R2† |
| --- | --- | --- | --- | --- | --- |
| 24-hour infusion (vs 4 hour) | 24-hour higher | < 2 × 10−16 | 38% | < 2 × 10−16 | 38% |
| DI (vs no DI) | DI lower | .0010 | 0.5% | .0022 | 0.4% |
| Age | Older lower | 9 × 10−7 | 1.1% | 7 × 10−7 | 1.2% |
| Sex | Boys greater | .00028 | 0.6% | .00027 | 0.5% |
| rs4149056 T > C | C lower |   |   | 2.1 × 10−11 | 2% |

Table 1 Caption: Covariates related to methotrexate clearance in a multivariate general linear model

### Figure 3.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9bf7/3567337/40f6113936e1/zh89991301740003.jpg)

Histogram of each patient's average methotrexate clearance. (A) Unadjusted methotrexate clearance for the 655 COG patients receiving the 4-hour infusion (blue), and the 624 receiving the 24-hour infusion (red). (B) Methotrexate clearance in all COG patients (n = 1279), adjusted for age, sex, race, and treatment arm (residual from a linear regression model).

When we adjusted for age, sex, and treatment arm, methotrexate clearance displayed a unimodal population distribution ([Figure 3](#F3)B). A GWAS was performed for SNP-genotype associations with clearance, including age, sex, genetic ancestry, and treatment arm as covariates. The GWAS showed a strong association between methotrexate clearance and SNPs in *SLCO1B1* ([Figure 4](#F4)A; [Table 2](#T2)). There were 4 typed SNPs that reached a genome-wide significance threshold of *P* < 5 × 10^−8^ (rs4149081, rs4149056, rs11045879, and rs11045821). rs4149056 is a nonsynonymous coding SNP that has been shown to have reduced transport of methotrexate in vitro.^11^ Infusion length, treatment including a DI, age, and sex remained significantly associated with clearance in a multivariate model, including the rs4149056 genotype ([Table 1](#T1)); it is unclear why there was an association of DI with methotrexate clearance because it persisted after we adjusted for the sex, race, and age factors that might have confounded the association. The other 3 significant SNPs are in linkage disequilibrium (LD) with rs4149056. With each copy of the C allele at rs4149056, the adjusted methotrexate clearance was reduced by 12 mL/min/m^2^ ([Table 2](#T2); supplemental Figure 5); thus, clearance is (on average) approximately13% lower in patients with CC versus TT genotypes at rs4149056, a magnitude of difference in exposure that could have significance for efficacy or adverse events.^9^

### Figure 4.

![Figure 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9bf7/3567337/ab1a8906d1e0/zh89991301740004.jpg)

Manhattan plot of P values for a genome-wide association with methotrexate clearance. Each chromosome is plotted along the x-axis, whereas the y-axis plots the negative log10 of the P value for the association of each SNP with methotrexate clearance. Higher points on the graph indicate a stronger association with methotrexate clearance. Typed and imputed SNPs are shown in all panels (5.2 million SNPs). Methotrexate clearance is adjusted for age, sex, race, and treatment arm. (A) Association of each SNP with methotrexate clearance in COG patients (n = 1279). (B) Meta-analysis of St Jude (n = 699) and COG patients (n = 1279). (C) Meta-analysis of St Jude and COG patients, after we adjusted for rs4149056.

### Table 2.

| rsID | Meta P | A/B alleles | Gene | COG P | COG effect size | COG source | COG B allele freq | SJ P | SJ effect size | SJ source | SJ B allele freq |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| rs11045879 | 3.13 × 10−19 | T/C | SLCO1B1 | 8.31 × 10−10 | −11.69 | SNPCHIP | 0.17 | 6.23 × 10−11 | −11.27 | SNPCHIP | 0.16 |
| rs4149056 | 5.72 × 10−19 | T/C | SLCO1B1 | 2.01 × 10−10 | −13.29 | SNPCHIP | 0.14 | 4.90 × 10−10 | −12.00 | GoldenGate | 0.12 |
| rs4149081 | 9.74 × 10−19 | G/A | SLCO1B1 | 1.58 × 10−10 | −12.19 | SNPCHIP | 0.18 | 1.05 × 10−9 | −10.61 | SNPCHIP | 0.16 |
| rs11045813 | 4.04 × 10−13 | A/G | SLCO1B1 | 2.95 × 10−7 | −12.89 | GoldenGate | 0.88 | 2.85 × 10−7 | −9.65 | GoldenGate | 0.87 |
| rs11045818 | 1.69 × 10−12 | A/G | SLCO1B1 | 1.35 × 10−6 | −11.66 | SNPCHIP | 0.88 | 2.63 × 10−7 | −9.88 | SNPCHIP | 0.87 |
| rs11045872 | 2.42 × 10−12 | A/G | SLCO1B1 | 3.77 × 10−7 | 11.49 | SNPCHIP | 0.15 | 1.36 × 10−6 | 8.70 | SNPCHIP | 0.15 |
| rs11045821 | 5.59 × 10−12 | G/A | SLCO1B1 | 3.33 × 10−8 | 13.49 | SNPCHIP | 0.13 | 2.44 × 10−5 | 8.46 | Imputed | 0.14 |
| rs11045870 | 3.81 × 10−11 | A/G | SLCO1B1 | 9.17 × 10−6 | −10.51 | GoldenGate | 0.86 | 8.94 × 10−7 | -8.74 | GoldenGate | 0.85 |
| rs10841753 | 5.31 × 10−11 | T/C | SLCO1B1 | 6.32 × 10−6 | 8.83 | SNPCHIP | 0.21 | 1.89 × 10−6 | 7.87 | SNPCHIP | 0.18 |
| rs11045892 | 6.60 × 10−11 | A/G | SLCO1B1 | 3.27 × 10−5 | 10.64 | GoldenGate | 0.14 | 3.76 × 10−7 | 8.98 | GoldenGate | 0.15 |
| rs11045825 | 7.80 × 10−11 | T/C | SLCO1B1 | 9.10 × 10−8 | 12.94 | SNPCHIP | 0.13 | 1.16 × 10−4 | 8.12 | Imputed | 0.16 |
| rs2169969 | 1.67 × 10−10 | A/G | SLCO1B1 | 4.35 × 10−7 | 11.75 | SNPCHIP | 0.15 | 6.84 × 10−5 | 8.23 | Imputed | 0.17 |
| rs16923647 | 7.64 × 10−10 | T/C | SLCO1A2 | 1.31 × 10−4 | 8.98 | SNPCHIP | 0.88 | 1.09 × 10−6 | 10.18 | SNPCHIP | 0.90 |

Table 2 Caption: Highest ranked SNPs associated with methotrexate clearance in a meta-analysis (typed in 1279 COG patients, typed or imputed in 699 SJ patients)

In a meta-analysis in which we combined patients treated with the COG and St Jude protocols, *SLCO1B1* remained significantly related to methotrexate clearance ([Figure 4](#F4)B). Of the 103 typed or imputed SNPs that reached a Bonferroni-corrected genome-wide significance threshold of *P* < 9.62 × 10^−9^ (supplemental Table 3), 101 were in *SLCO1B1*. The meta-analysis *P* value for the top SNP in *SLCO1B1*, rs4149080, is 5.61 × 10^−21^ (supplemental Table 3). This SNP is in LD (r^2^ = 0.897) with rs4149056 in European patients and was imputed in both cohorts. The SNP typed in both the COG and St Jude cohorts that had the smallest *P* value was rs11045879, which is also in LD with rs4149056 (r^2^ = 0.865), and had a *P* value of 3.13 × 10^−19^ in the meta-analysis. Two SNPs in *SLCO1A2* also were related to methotrexate clearance at the genome-wide level (7.6 × 10^−10^ and 1.9 × 10^−9^); however, this is most likely because they are very near *SLCO1B1* on chromosome 12 and in LD with the rs4149056 SNP in *SLCO1B1* ([Figure 5](#F5)).^21,22^ They are not in LD with the rs11045891 and rs4148981 SNPs that are *SLCO1A2* enhancers.^23^ SLCO1B3 is a closely related family member that also transports methotrexate.^24,25^ Although they did not reach genome-wide significance, 2 SNPs in *SLCO1B3* were related to methotrexate clearance (*P* < 3 × 10^−6^), and the association remained after we adjusted for SNPs in *SLCO1B1*. Four SNPs in *SLC19A1* (the reduced folate carrier) were interrogated in this study on the Affymetrix array (2 are in LD with functional polymorphisms rs1051266 and rs1131596, r^2^ > 0.9), but none reached genome-wide significance for an association with methotrexate clearance.

### Figure 5.

![Figure 5](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9bf7/3567337/61642237ee24/zh89991301740005.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3567337_zh89991301740005.jpg)

LD of SNPs with P < 10−5 in meta-analysis spanning SLCO1B3, LST-3TM12, SLCO1B1, and SLCO1A2 generated with the use of Haploview software21 in European-ancestry patients only (COG n = 806 and SJ n = 459). The darkness of each box represents the association between 2 SNPs (R2), with white being R2 = 0, black being R2 = 1, and 0 < R2 < 1 being shades of gray. LD blocks, highlighted with the yellow lines, were defined by confidence intervals.22 Assembly hg18 from http://genome.ucsc.edu used for display.

The nonsynonymous rs4149056 (T521C) SNP has been extensively characterized in relation to other phenotypes^26–28^ and shown to reduce methotrexate transport in vitro.^11^ After we adjusted for the rs4149056 genotype, a significant association remained in *SLCO1B1* at the genome-wide significance level ([Figure 4](#F4)C). The SNP with the lowest *P* value after adjustment for rs4149056 was an intronic SNP, rs4149040 (*P* = 1.2 × 10^−12^, supplemental Table 4), which is in LD with the nonsynonymous SNP rs2306283. Together, rs2306283 and rs4149056 define the most common haplotypes of *SLCO1B1* (*1a, *1b, *5, and *15)^29^ and are in partial LD (D′ = 0.91, r^2^ = 0.25). As we previously demonstrated, the relationship between methotrexate clearance and rs2306283 genotype is evident only in an analysis stratified for rs4149056 genotype (supplemental Figure 6, *P* = 4 × 10^−11^),^11^ indicating a highly reproducible SNP-by-SNP interaction. Patients with at least 1 C allele at rs4149056 have lower clearance than those with 2 T alleles (supplemental Figure 5); however, within each rs4149056 genotypic group, the A allele at rs2306283 was associated with even lower clearance, whereas the variant G allele was associated with increased clearance (supplemental Figure 6). We also replicated that the rs11045872 variant G allele was associated with greater methotrexate clearance (*P* = 3.8 × 10^−7^ in COG patients, supplemental Table 3; supplemental Figure 7).

## Discussion

We have replicated the association between *SLCO1B1* and methotrexate clearance^10,11^ with 2 additional schedules of intravenous methotrexate administration in the setting of a cooperative group trial. We estimated methotrexate clearance using 2 plasma methotrexate concentrations per course via a Bayesian pharmacokinetic modeling approach. The mean and range of clearances that we estimated from only 2 plasma concentrations was similar to that estimated from an entirely different group of patients with more samples per course (St Jude Total XV participants).^10,11^ Further external validity of methotrexate clearance estimates from these large multicenter trials is provided by the fact that we replicated the association of methotrexate clearance with age, sex, and treatment regimen. We previously found that black patients had the greatest methotrexate clearance^10^; although this association was not replicated here, there were relatively few black patients (5%) in this COG cohort.

SLCO1B1 is an organic anion transporter that is known to transport methotrexate; minor alleles for SNPs in the *SLCO1B1* gene are associated with both increased and decreased transporter function and span multiple functional domains in the gene.^11^ We found SNPs associated with both increased and decreased methotrexate clearance in the COG cohort, replicating our previous finding in the St Jude cohort (supplemental Figures 5-7).^11^ We improved our power to detect significant associations by performing a meta-analysis combining the COG and St Jude cohorts. The meta-analysis showed that only SNPs on chromosome 12 near *SLCO1B1* were related to methotrexate clearance in these patients, many of which were not significant at the genome-wide threshold in either cohort alone. The nonsynonymous rs4149056 (T521C) SNP has been the subject of multiple clinical studies of statins and other agents.^26,30–34^ However, even after adjustment for the rs4149056 SNP, other SNPs in *SLCO1B1* remained significantly related to methotrexate clearance, indicating that there are multiple variants in *SLCO1B1* that can influence methotrexate clearance. In this independent cohort, we replicated our previous finding, that is, that the rs2306283 G allele was associated with greater clearance after we adjusted for the rs4149056 genotype.^11^

Variants in *SLCO1B1* could be clinically useful in identifying patients at risk of low methotrexate clearance, particularly in settings in which plasma methotrexate concentrations are not readily available. Toxicity associated with 1 g/m^2^ and 2 g/m^2^ doses has been reported.^35,36^ One possible intervention would be to increase IV hydration and/or alkalinization in the subset of patients with a “low-clearance” *SLCO1B1* genotype to force greater clearance of methotrexate, but the effects of this intervention on toxicity are not known. Tools for aggregation of *SLCO1B1* variants into clinically important haplotypes are available ([http://www.pharmgkb.org/gene/PA134865839](http://www.pharmgkb.org/gene/PA134865839)), and such haplotypes are already used for dosing decisions for other drugs, such as statins.^37^ In addition, it is likely that the coadministration of HDMTX with other SLCO1B1 substrates or inhibitors will result in increased plasma methotrexate concentrations.^38–40^

In summary, we have replicated the finding that *SLCO1B1* is related to methotrexate clearance. This validates our previous GWAS^10^ in a separate group of patients who received different dosages of HDMTX in a multi-institutional clinical trial, solidifying the robustness of this pharmacogenomic determinant of methotrexate clearance.

## Supplementary Material

## Acknowledgments

The authors are grateful to physicians and clinical research associates at the COG institutions participating in this study for cooperation with the sample submission requirements. The authors thank Shaherah Rankins for assistance in preparation of the manuscript.

This work was supported by grants CA 36401, GM 92666, CA 98453, CA98413, CA114766, and CA 21765 from the National Institutes of Health and the American Lebanese Syrian Associated Charities. The authors thank the Jeffrey Pride Foundation and the National Childhood Cancer Foundation for their financial support for genome-wide genotyping of COG specimens. S.P.H. is the Ergen Family Chair in Pediatric Cancer.

## Footnotes

## Authorship

Contribution: L.B.R., C-H.P., W.E.E., and M.V.R. designed research, analyzed and interpreted data, and drafted the manuscript; J.C.P., C.S., W.Y., Y.F., C.C., and M.D. analyzed and interpreted data and performed statistical analysis; and N.J.W., P.L.M., S.P.H., and M.L. collected, analyzed and interpreted data.

Conflict-of-interest disclosure: The authors declare no competing financial interests.

Correspondence: Mary V. Relling, PharmD, St Jude Children's Research Hospital, 262 Danny Thomas Place, MS 313, Memphis, TN 38105; e-mail: mary.relling@stjude.org.

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Whitehead VM, Shuster JJ, Vuchich MJ, et al. Accumulation of methotrexate and methotrexate polyglutamates in lymphoblasts and treatment outcome in children with B-progenitor-cell acute lymphoblastic leukemia: a Pediatric Oncology Group study. Leukemia. 2005;19(4):533–536. doi: 10.1038/sj.leu.2403703.  [DOI](https://doi.org/10.1038/sj.leu.2403703) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15716987/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Leukemia&title=Accumulation%20of%20methotrexate%20and%20methotrexate%20polyglutamates%20in%20lymphoblasts%20and%20treatment%20outcome%20in%20children%20with%20B-progenitor-cell%20acute%20lymphoblastic%20leukemia:%20a%20Pediatric%20Oncology%20Group%20study.&author=VM%20Whitehead&author=JJ%20Shuster&author=MJ%20Vuchich&volume=19&issue=4&publication_year=2005&pages=533-536&pmid=15716987&doi=10.1038/sj.leu.2403703&)

2. Asselin BL, Devidas M, Wang C, et al. Effectiveness of high-dose methotrexate in T-cell lymphoblastic leukemia and advanced-stage lymphoblastic lymphoma: a randomized study by the Children's Oncology Group (POG 9404). Blood. 2011;118(4):874–883. doi: 10.1182/blood-2010-06-292615.  [DOI](https://doi.org/10.1182/blood-2010-06-292615) | [PMC free article](/articles/PMC3292437/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21474675/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Effectiveness%20of%20high-dose%20methotrexate%20in%20T-cell%20lymphoblastic%20leukemia%20and%20advanced-stage%20lymphoblastic%20lymphoma:%20a%20randomized%20study%20by%20the%20Children's%20Oncology%20Group%20(POG%209404).&author=BL%20Asselin&author=M%20Devidas&author=C%20Wang&volume=118&issue=4&publication_year=2011&pages=874-883&pmid=21474675&doi=10.1182/blood-2010-06-292615&)

3. Clarke M, Gaynon P, Hann I, et al. CNS-directed therapy for childhood acute lymphoblastic leukemia: Childhood ALL Collaborative Group overview of 43 randomized trials. J Clin Oncol. 2003;21(9):1798–1809. doi: 10.1200/JCO.2003.08.047.  [DOI](https://doi.org/10.1200/JCO.2003.08.047) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12721257/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=CNS-directed%20therapy%20for%20childhood%20acute%20lymphoblastic%20leukemia:%20Childhood%20ALL%20Collaborative%20Group%20overview%20of%2043%20randomized%20trials.&author=M%20Clarke&author=P%20Gaynon&author=I%20Hann&volume=21&issue=9&publication_year=2003&pages=1798-1809&pmid=12721257&doi=10.1200/JCO.2003.08.047&)

4. Matloub Y, Bostrom BC, Hunger SP, et al. Escalating intravenous methotrexate improves event-free survival in children with standard-risk acute lymphoblastic leukemia: a report from the Children's Oncology Group. Blood. 2011;118(2):243–251. doi: 10.1182/blood-2010-12-322909.  [DOI](https://doi.org/10.1182/blood-2010-12-322909) | [PMC free article](/articles/PMC3138679/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21562038/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Escalating%20intravenous%20methotrexate%20improves%20event-free%20survival%20in%20children%20with%20standard-risk%20acute%20lymphoblastic%20leukemia:%20a%20report%20from%20the%20Children's%20Oncology%20Group.&author=Y%20Matloub&author=BC%20Bostrom&author=SP%20Hunger&volume=118&issue=2&publication_year=2011&pages=243-251&pmid=21562038&doi=10.1182/blood-2010-12-322909&)

5. Mikkelsen TS, Sparreboom A, Cheng C, et al. Shortening infusion time for high-dose methotrexate alters antileukemic effects: a randomized prospective clinical trial. J Clin Oncol. 2011;29(13):1771–1778. doi: 10.1200/JCO.2010.32.5340.  [DOI](https://doi.org/10.1200/JCO.2010.32.5340) | [PMC free article](/articles/PMC3107765/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21444869/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Shortening%20infusion%20time%20for%20high-dose%20methotrexate%20alters%20antileukemic%20effects:%20a%20randomized%20prospective%20clinical%20trial.&author=TS%20Mikkelsen&author=A%20Sparreboom&author=C%20Cheng&volume=29&issue=13&publication_year=2011&pages=1771-1778&pmid=21444869&doi=10.1200/JCO.2010.32.5340&)

6. Borowitz MJ, Devidas M, Hunger SP, et al. Clinical significance of minimal residual disease in childhood acute lymphoblastic leukemia and its relationship to other prognostic factors: a Children's Oncology Group study. Blood. 2008;111(12):5477–5485. doi: 10.1182/blood-2008-01-132837.  [DOI](https://doi.org/10.1182/blood-2008-01-132837) | [PMC free article](/articles/PMC2424148/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18388178/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Clinical%20significance%20of%20minimal%20residual%20disease%20in%20childhood%20acute%20lymphoblastic%20leukemia%20and%20its%20relationship%20to%20other%20prognostic%20factors:%20a%20Children's%20Oncology%20Group%20study.&author=MJ%20Borowitz&author=M%20Devidas&author=SP%20Hunger&volume=111&issue=12&publication_year=2008&pages=5477-5485&pmid=18388178&doi=10.1182/blood-2008-01-132837&)

7. Xu H, Cheng C, Devidas M, et al. ARID5B genetic polymorphisms contribute to racial disparities in the incidence and treatment outcome of childhood acute lymphoblastic leukemia. J Clin Oncol. 2012;30(7):751–757. doi: 10.1200/JCO.2011.38.0345.  [DOI](https://doi.org/10.1200/JCO.2011.38.0345) | [PMC free article](/articles/PMC3295551/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22291082/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=ARID5B%20genetic%20polymorphisms%20contribute%20to%20racial%20disparities%20in%20the%20incidence%20and%20treatment%20outcome%20of%20childhood%20acute%20lymphoblastic%20leukemia.&author=H%20Xu&author=C%20Cheng&author=M%20Devidas&volume=30&issue=7&publication_year=2012&pages=751-757&pmid=22291082&doi=10.1200/JCO.2011.38.0345&)

8. Evans WE, Crom WR, Stewart CF, et al. Methotrexate systemic clearance influences probability of relapse in children with standard-risk acute lymphocytic leukaemia. Lancet. 1984;1:359–362. doi: 10.1016/s0140-6736(84)90411-2.  [DOI](https://doi.org/10.1016/s0140-6736(84)90411-2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/6141424/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Methotrexate%20systemic%20clearance%20influences%20probability%20of%20relapse%20in%20children%20with%20standard-risk%20acute%20lymphocytic%20leukaemia.&author=WE%20Evans&author=WR%20Crom&author=CF%20Stewart&volume=1&publication_year=1984&pages=359-362&pmid=6141424&doi=10.1016/s0140-6736(84)90411-2&)

9. Evans WE, Crom WR, Abromowitch M, et al. Clinical pharmacodynamics of high-dose methotrexate in acute lymphocytic leukemia. Identification of a relation between concentration and effect. N Engl J Med. 1986;314(8):471–477. doi: 10.1056/NEJM198602203140803.  [DOI](https://doi.org/10.1056/NEJM198602203140803) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/3456079/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Clinical%20pharmacodynamics%20of%20high-dose%20methotrexate%20in%20acute%20lymphocytic%20leukemia.%20Identification%20of%20a%20relation%20between%20concentration%20and%20effect.&author=WE%20Evans&author=WR%20Crom&author=M%20Abromowitch&volume=314&issue=8&publication_year=1986&pages=471-477&pmid=3456079&doi=10.1056/NEJM198602203140803&)

10. Trevino LR, Shimasaki N, Yang W, et al. Germline genetic variation in an organic anion transporter polypeptide associated with methotrexate pharmacokinetics and clinical effects. J Clin Oncol. 2009;27(35):5972–5978. doi: 10.1200/JCO.2008.20.4156.  [DOI](https://doi.org/10.1200/JCO.2008.20.4156) | [PMC free article](/articles/PMC2793040/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19901119/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Germline%20genetic%20variation%20in%20an%20organic%20anion%20transporter%20polypeptide%20associated%20with%20methotrexate%20pharmacokinetics%20and%20clinical%20effects.&author=LR%20Trevino&author=N%20Shimasaki&author=W%20Yang&volume=27&issue=35&publication_year=2009&pages=5972-5978&pmid=19901119&doi=10.1200/JCO.2008.20.4156&)

11. Ramsey LB, Bruun GH, Yang W, et al. Rare versus common variants in pharmacogenetics: SLCO1B1 variation and methotrexate disposition. Genome Res. 2012;22(1):1–8. doi: 10.1101/gr.129668.111.  [DOI](https://doi.org/10.1101/gr.129668.111) | [PMC free article](/articles/PMC3246196/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22147369/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Res&title=Rare%20versus%20common%20variants%20in%20pharmacogenetics:%20SLCO1B1%20variation%20and%20methotrexate%20disposition.&author=LB%20Ramsey&author=GH%20Bruun&author=W%20Yang&volume=22&issue=1&publication_year=2012&pages=1-8&pmid=22147369&doi=10.1101/gr.129668.111&)

12. Shuster J, Camitta B, Pullen J, et al. Identification of newly diagnosed children with acute lymphocytic leukemia at high risk for relapse. Cancer Res Ther Control. 1999;9(1):101–107.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res%20Ther%20Control&title=Identification%20of%20newly%20diagnosed%20children%20with%20acute%20lymphocytic%20leukemia%20at%20high%20risk%20for%20relapse.&author=J%20Shuster&author=B%20Camitta&author=J%20Pullen&volume=9&issue=1&publication_year=1999&pages=101-107&)

13. Garre ML, Relling MV, Kalwinsky D, et al. Pharmacokinetics and toxicity of methotrexate in children with Down syndrome and acute lymphocytic leukemia. J Pediatr. 1987;111:606–612. doi: 10.1016/s0022-3476(87)80131-2.  [DOI](https://doi.org/10.1016/s0022-3476(87)80131-2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/2958611/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pediatr&title=Pharmacokinetics%20and%20toxicity%20of%20methotrexate%20in%20children%20with%20Down%20syndrome%20and%20acute%20lymphocytic%20leukemia.&author=ML%20Garre&author=MV%20Relling&author=D%20Kalwinsky&volume=111&publication_year=1987&pages=606-612&pmid=2958611&doi=10.1016/s0022-3476(87)80131-2&)

14. Dordelmann M, Schrappe M, Reiter A, et al. Down's syndrome in childhood acute lymphoblastic leukemia: clinical characteristics and treatment outcome in four consecutive BFM trials. Berlin-Frankfurt-Munster Group. Leukemia. 1998;12(5):645–651. doi: 10.1038/sj.leu.2400989.  [DOI](https://doi.org/10.1038/sj.leu.2400989) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9593260/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Leukemia&title=Down's%20syndrome%20in%20childhood%20acute%20lymphoblastic%20leukemia:%20clinical%20characteristics%20and%20treatment%20outcome%20in%20four%20consecutive%20BFM%20trials.%20Berlin-Frankfurt-Munster%20Group.&author=M%20Dordelmann&author=M%20Schrappe&author=A%20Reiter&volume=12&issue=5&publication_year=1998&pages=645-651&pmid=9593260&doi=10.1038/sj.leu.2400989&)

15. Buitenkamp TD, Mathot RA, de Haas V, Pieters R, Zwaan CM. Methotrexate-induced side effects are not due to differences in pharmacokinetics in children with Down syndrome and acute lymphoblastic leukemia. Haematologica. 2010;95(7):1106–1113. doi: 10.3324/haematol.2009.019778.  [DOI](https://doi.org/10.3324/haematol.2009.019778) | [PMC free article](/articles/PMC2895034/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20418240/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Haematologica&title=Methotrexate-induced%20side%20effects%20are%20not%20due%20to%20differences%20in%20pharmacokinetics%20in%20children%20with%20Down%20syndrome%20and%20acute%20lymphoblastic%20leukemia.&author=TD%20Buitenkamp&author=RA%20Mathot&author=V%20de%20Haas&author=R%20Pieters&author=CM%20Zwaan&volume=95&issue=7&publication_year=2010&pages=1106-1113&pmid=20418240&doi=10.3324/haematol.2009.019778&)

16. Panetta JC, Sparreboom A, Pui CH, Relling MV, Evans WE. Modeling mechanisms of in vivo variability in methotrexate accumulation and folate pathway inhibition in acute lymphoblastic leukemia cells. PLoS Comput Biol. 2010;6(12):e1001019. doi: 10.1371/journal.pcbi.1001019.  [DOI](https://doi.org/10.1371/journal.pcbi.1001019) | [PMC free article](/articles/PMC2996318/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21152005/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Comput%20Biol&title=Modeling%20mechanisms%20of%20in%20vivo%20variability%20in%20methotrexate%20accumulation%20and%20folate%20pathway%20inhibition%20in%20acute%20lymphoblastic%20leukemia%20cells.&author=JC%20Panetta&author=A%20Sparreboom&author=CH%20Pui&author=MV%20Relling&author=WE%20Evans&volume=6&issue=12&publication_year=2010&pages=e1001019&pmid=21152005&doi=10.1371/journal.pcbi.1001019&)

17. Yang JJ, Cheng C, Devidas M, et al. Ancestry and pharmacogenomics of relapse in acute lymphoblastic leukemia. Nat Genet. 2011;43(3):237–241. doi: 10.1038/ng.763.  [DOI](https://doi.org/10.1038/ng.763) | [PMC free article](/articles/PMC3104508/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21297632/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Ancestry%20and%20pharmacogenomics%20of%20relapse%20in%20acute%20lymphoblastic%20leukemia.&author=JJ%20Yang&author=C%20Cheng&author=M%20Devidas&volume=43&issue=3&publication_year=2011&pages=237-241&pmid=21297632&doi=10.1038/ng.763&)

18. Li Y, Abecasis GR. Mach 1.0: Rapid haplotype reconstruction and missing genotype inference. Am J Hum Genet. 2006;S79:2290.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=Mach%201.0:%20Rapid%20haplotype%20reconstruction%20and%20missing%20genotype%20inference.&author=Y%20Li&author=GR%20Abecasis&volume=S79&publication_year=2006&pages=2290&)

19. Stouffer SA, Suchman EA, DeVinney LC, Star SA, Williams RM., Jr . Adjustment During Army Life. Princeton, NJ: Princeton University Press; 1949.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Adjustment%20During%20Army%20Life&author=SA%20Stouffer&author=EA%20Suchman&author=LC%20DeVinney&author=SA%20Star&author=RM%20Williams&publication_year=1949&)

20. Willer CJ, Li Y, Abecasis GR. METAL: fast and efficient meta-analysis of genomewide association scans. Bioinformatics. 2010;26(17):2190–2191. doi: 10.1093/bioinformatics/btq340.  [DOI](https://doi.org/10.1093/bioinformatics/btq340) | [PMC free article](/articles/PMC2922887/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20616382/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=METAL:%20fast%20and%20efficient%20meta-analysis%20of%20genomewide%20association%20scans.&author=CJ%20Willer&author=Y%20Li&author=GR%20Abecasis&volume=26&issue=17&publication_year=2010&pages=2190-2191&pmid=20616382&doi=10.1093/bioinformatics/btq340&)

21. Barrett JC, Fry B, Maller J, Daly MJ. Haploview: analysis and visualization of LD and haplotype maps. Bioinformatics. 2005;21(2):263–265. doi: 10.1093/bioinformatics/bth457.  [DOI](https://doi.org/10.1093/bioinformatics/bth457) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15297300/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=Haploview:%20analysis%20and%20visualization%20of%20LD%20and%20haplotype%20maps.&author=JC%20Barrett&author=B%20Fry&author=J%20Maller&author=MJ%20Daly&volume=21&issue=2&publication_year=2005&pages=263-265&pmid=15297300&doi=10.1093/bioinformatics/bth457&)

22. Gabriel SB, Schaffner SF, Nguyen H, et al. The structure of haplotype blocks in the human genome. Science. 2002;296(5576):2225–2229. doi: 10.1126/science.1069424.  [DOI](https://doi.org/10.1126/science.1069424) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12029063/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=The%20structure%20of%20haplotype%20blocks%20in%20the%20human%20genome.&author=SB%20Gabriel&author=SF%20Schaffner&author=H%20Nguyen&volume=296&issue=5576&publication_year=2002&pages=2225-2229&pmid=12029063&doi=10.1126/science.1069424&)

23. Kim MJ, Skewes-Cox P, Fukushima H, et al. Functional characterization of liver enhancers that regulate drug-associated transporters. Clin Pharmacol Ther. 2011;89(4):571–578. doi: 10.1038/clpt.2010.353.  [DOI](https://doi.org/10.1038/clpt.2010.353) | [PMC free article](/articles/PMC3227682/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21368754/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Functional%20characterization%20of%20liver%20enhancers%20that%20regulate%20drug-associated%20transporters.&author=MJ%20Kim&author=P%20Skewes-Cox&author=H%20Fukushima&volume=89&issue=4&publication_year=2011&pages=571-578&pmid=21368754&doi=10.1038/clpt.2010.353&)

24. Abe T, Unno M, Onogawa T, et al. LST-2, a human liver-specific organic anion transporter, determines methotrexate sensitivity in gastrointestinal cancers. Gastroenterology. 2001;120(7):1689–1699. doi: 10.1053/gast.2001.24804.  [DOI](https://doi.org/10.1053/gast.2001.24804) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11375950/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gastroenterology&title=LST-2,%20a%20human%20liver-specific%20organic%20anion%20transporter,%20determines%20methotrexate%20sensitivity%20in%20gastrointestinal%20cancers.&author=T%20Abe&author=M%20Unno&author=T%20Onogawa&volume=120&issue=7&publication_year=2001&pages=1689-1699&pmid=11375950&doi=10.1053/gast.2001.24804&)

25. Chae YJ, Lee KR, Noh CK, et al. Functional consequences of genetic variations in the human organic anion transporting polypeptide 1B3 (OATP1B3) in the Korean population. J Pharm Sci. 2012;101(3):1302–1313. doi: 10.1002/jps.23005.  [DOI](https://doi.org/10.1002/jps.23005) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22147445/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharm%20Sci&title=Functional%20consequences%20of%20genetic%20variations%20in%20the%20human%20organic%20anion%20transporting%20polypeptide%201B3%20(OATP1B3)%20in%20the%20Korean%20population.&author=YJ%20Chae&author=KR%20Lee&author=CK%20Noh&volume=101&issue=3&publication_year=2012&pages=1302-1313&pmid=22147445&doi=10.1002/jps.23005&)

26. Link E, Parish S, Armitage J, et al. SLCO1B1 variants and statin-induced myopathy—a genomewide study. N Engl J Med. 2008;359(8):789–799. doi: 10.1056/NEJMoa0801936.  [DOI](https://doi.org/10.1056/NEJMoa0801936) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18650507/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=SLCO1B1%20variants%20and%20statin-induced%20myopathy%E2%80%94a%20genomewide%20study.&author=E%20Link&author=S%20Parish&author=J%20Armitage&volume=359&issue=8&publication_year=2008&pages=789-799&pmid=18650507&doi=10.1056/NEJMoa0801936&)

27. Deng JW, Song IS, Shin HJ, et al. The effects of SLCO1B1*15 on the disposition of pravastatin and pitavastatin is substrate dependent: the contribution of transporting activity changes by SLCO1B1*15. Pharmacogenet Genomics. 2008;18(5):424–433. doi: 10.1097/FPC.0b013e3282fb02a3.  [DOI](https://doi.org/10.1097/FPC.0b013e3282fb02a3) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18408565/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=The%20effects%20of%20SLCO1B1*15%20on%20the%20disposition%20of%20pravastatin%20and%20pitavastatin%20is%20substrate%20dependent:%20the%20contribution%20of%20transporting%20activity%20changes%20by%20SLCO1B1*15.&author=JW%20Deng&author=IS%20Song&author=HJ%20Shin&volume=18&issue=5&publication_year=2008&pages=424-433&pmid=18408565&doi=10.1097/FPC.0b013e3282fb02a3&)

28. Xiang X, Jada SR, Li HH, et al. Pharmacogenetics of SLCO1B1 gene and the impact of *1b and *15 haplotypes on irinotecan disposition in Asian cancer patients. Pharmacogenet Genomics. 2006;16(9):683–691. doi: 10.1097/01.fpc.0000230420.05221.71.  [DOI](https://doi.org/10.1097/01.fpc.0000230420.05221.71) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16906022/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Pharmacogenetics%20of%20SLCO1B1%20gene%20and%20the%20impact%20of%20*1b%20and%20*15%20haplotypes%20on%20irinotecan%20disposition%20in%20Asian%20cancer%20patients.&author=X%20Xiang&author=SR%20Jada&author=HH%20Li&volume=16&issue=9&publication_year=2006&pages=683-691&pmid=16906022&doi=10.1097/01.fpc.0000230420.05221.71&)

29. Pasanen MK, Neuvonen PJ, Niemi M. Global analysis of genetic variation in SLCO1B1. Pharmacogenomics. 2008;9(1):19–33. doi: 10.2217/14622416.9.1.19.  [DOI](https://doi.org/10.2217/14622416.9.1.19) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18154446/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Global%20analysis%20of%20genetic%20variation%20in%20SLCO1B1.&author=MK%20Pasanen&author=PJ%20Neuvonen&author=M%20Niemi&volume=9&issue=1&publication_year=2008&pages=19-33&pmid=18154446&doi=10.2217/14622416.9.1.19&)

30. Johnson AD, Kavousi M, Smith AV, et al. Genome- wide association meta-analysis for total serum bilirubin levels. Hum Mol Genet. 2009;18(14):2700–2710. doi: 10.1093/hmg/ddp202.  [DOI](https://doi.org/10.1093/hmg/ddp202) | [PMC free article](/articles/PMC2701336/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19414484/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mol%20Genet&title=Genome-%20wide%20association%20meta-analysis%20for%20total%20serum%20bilirubin%20levels.&author=AD%20Johnson&author=M%20Kavousi&author=AV%20Smith&volume=18&issue=14&publication_year=2009&pages=2700-2710&pmid=19414484&doi=10.1093/hmg/ddp202&)

31. Innocenti F, Kroetz DL, Schuetz E, et al. Comprehensive pharmacogenetic analysis of irinotecan neutropenia and pharmacokinetics. J Clin Oncol. 2009;27(16):2604–2614. doi: 10.1200/JCO.2008.20.6300.  [DOI](https://doi.org/10.1200/JCO.2008.20.6300) | [PMC free article](/articles/PMC2690389/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19349540/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Comprehensive%20pharmacogenetic%20analysis%20of%20irinotecan%20neutropenia%20and%20pharmacokinetics.&author=F%20Innocenti&author=DL%20Kroetz&author=E%20Schuetz&volume=27&issue=16&publication_year=2009&pages=2604-2614&pmid=19349540&doi=10.1200/JCO.2008.20.6300&)

32. Kalliokoski A, Neuvonen M, Neuvonen PJ, Niemi M. Different effects of SLCO1B1 polymorphism on the pharmacokinetics and pharmacodynamics of repaglinide and nateglinide. J Clin Pharmacol. 2008;48(3):311–321. doi: 10.1177/0091270007311569.  [DOI](https://doi.org/10.1177/0091270007311569) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18187595/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Pharmacol&title=Different%20effects%20of%20SLCO1B1%20polymorphism%20on%20the%20pharmacokinetics%20and%20pharmacodynamics%20of%20repaglinide%20and%20nateglinide.&author=A%20Kalliokoski&author=M%20Neuvonen&author=PJ%20Neuvonen&author=M%20Niemi&volume=48&issue=3&publication_year=2008&pages=311-321&pmid=18187595&doi=10.1177/0091270007311569&)

33. Ni W, Ji J, Dai Z, et al. Flavopiridol pharmacogenetics: clinical and functional evidence for the role of SLCO1B1/OATP1B1 in flavopiridol disposition. PLoS One. 2010;5(11):e13792. doi: 10.1371/journal.pone.0013792.  [DOI](https://doi.org/10.1371/journal.pone.0013792) | [PMC free article](/articles/PMC2967470/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21072184/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=Flavopiridol%20pharmacogenetics:%20clinical%20and%20functional%20evidence%20for%20the%20role%20of%20SLCO1B1/OATP1B1%20in%20flavopiridol%20disposition.&author=W%20Ni&author=J%20Ji&author=Z%20Dai&volume=5&issue=11&publication_year=2010&pages=e13792&pmid=21072184&doi=10.1371/journal.pone.0013792&)

34. Pasanen MK, Neuvonen M, Neuvonen PJ, Niemi M. SLCO1B1 polymorphism markedly affects the pharmacokinetics of simvastatin acid. Pharmacogenet Genomics. 2006;16(12):873–879. doi: 10.1097/01.fpc.0000230416.82349.90.  [DOI](https://doi.org/10.1097/01.fpc.0000230416.82349.90) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17108811/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=SLCO1B1%20polymorphism%20markedly%20affects%20the%20pharmacokinetics%20of%20simvastatin%20acid.&author=MK%20Pasanen&author=M%20Neuvonen&author=PJ%20Neuvonen&author=M%20Niemi&volume=16&issue=12&publication_year=2006&pages=873-879&pmid=17108811&doi=10.1097/01.fpc.0000230416.82349.90&)

35. Camitta B, Mahoney D, Leventhal B, et al. Intensive intravenous methotrexate and mercaptopurine treatment of higher-risk non-T, non-B acute lymphocytic leukemia: a Pediatric Oncology Group study. J Clin Oncol. 1994;12:1383–1389. doi: 10.1200/JCO.1994.12.7.1383.  [DOI](https://doi.org/10.1200/JCO.1994.12.7.1383) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8021728/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Intensive%20intravenous%20methotrexate%20and%20mercaptopurine%20treatment%20of%20higher-risk%20non-T,%20non-B%20acute%20lymphocytic%20leukemia:%20a%20Pediatric%20Oncology%20Group%20study.&author=B%20Camitta&author=D%20Mahoney&author=B%20Leventhal&volume=12&publication_year=1994&pages=1383-1389&pmid=8021728&doi=10.1200/JCO.1994.12.7.1383&)

36. Kishi S, Cheng C, French D, et al. Ancestry and pharmacogenetics of antileukemic drug toxicity. Blood. 2007;109(10):4151–4157. doi: 10.1182/blood-2006-10-054528.  [DOI](https://doi.org/10.1182/blood-2006-10-054528) | [PMC free article](/articles/PMC1885506/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17264302/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Ancestry%20and%20pharmacogenetics%20of%20antileukemic%20drug%20toxicity.&author=S%20Kishi&author=C%20Cheng&author=D%20French&volume=109&issue=10&publication_year=2007&pages=4151-4157&pmid=17264302&doi=10.1182/blood-2006-10-054528&)

37. Wilke RA, Ramsey LB, Johnson SG, et al. The clinical pharmacogenomics implementation consortium: CPIC Guideline for SLCO1B1 and simvastatin-induced myopathy. Clin Pharmacol Ther. 2012;92(1):112–117. doi: 10.1038/clpt.2012.57.  [DOI](https://doi.org/10.1038/clpt.2012.57) | [PMC free article](/articles/PMC3384438/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22617227/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=The%20clinical%20pharmacogenomics%20implementation%20consortium:%20CPIC%20Guideline%20for%20SLCO1B1%20and%20simvastatin-induced%20myopathy.&author=RA%20Wilke&author=LB%20Ramsey&author=SG%20Johnson&volume=92&issue=1&publication_year=2012&pages=112-117&pmid=22617227&doi=10.1038/clpt.2012.57&)

38. Niemi M, Pasanen MK, Neuvonen PJ. Organic anion transporting polypeptide 1B1: a genetically polymorphic transporter of major importance for hepatic drug uptake. Pharmacol Rev. 2011;63(1):157–181. doi: 10.1124/pr.110.002857.  [DOI](https://doi.org/10.1124/pr.110.002857) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21245207/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacol%20Rev&title=Organic%20anion%20transporting%20polypeptide%201B1:%20a%20genetically%20polymorphic%20transporter%20of%20major%20importance%20for%20hepatic%20drug%20uptake.&author=M%20Niemi&author=MK%20Pasanen&author=PJ%20Neuvonen&volume=63&issue=1&publication_year=2011&pages=157-181&pmid=21245207&doi=10.1124/pr.110.002857&)

39. Karlgren M, Ahlin G, Bergstrom CA, Svensson R, Palm J, Artursson P. In vitro and in silico strategies to identify OATP1B1 inhibitors and predict clinical drug-drug interactions. Pharm Res. 2012;29(2):411–426. doi: 10.1007/s11095-011-0564-9.  [DOI](https://doi.org/10.1007/s11095-011-0564-9) | [PMC free article](/articles/PMC3264873/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21861202/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharm%20Res&title=In%20vitro%20and%20in%20silico%20strategies%20to%20identify%20OATP1B1%20inhibitors%20and%20predict%20clinical%20drug-drug%20interactions.&author=M%20Karlgren&author=G%20Ahlin&author=CA%20Bergstrom&author=R%20Svensson&author=J%20Palm&volume=29&issue=2&publication_year=2012&pages=411-426&pmid=21861202&doi=10.1007/s11095-011-0564-9&)

40. Fox RI, Morgan SL, Smith HT, Robbins BA, Choc MG, Baggott JE. Combined oral cyclosporin and methotrexate therapy in patients with rheumatoid arthritis elevates methotrexate levels and reduces 7-hydroxymethotrexate levels compared with methotrexate alone. Rheumatology (Oxford) 2003;42(8):989–994. doi: 10.1093/rheumatology/keg277.  [DOI](https://doi.org/10.1093/rheumatology/keg277) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12730515/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Rheumatology%20(Oxford)&title=Combined%20oral%20cyclosporin%20and%20methotrexate%20therapy%20in%20patients%20with%20rheumatoid%20arthritis%20elevates%20methotrexate%20levels%20and%20reduces%207-hydroxymethotrexate%20levels%20compared%20with%20methotrexate%20alone.&author=RI%20Fox&author=SL%20Morgan&author=HT%20Smith&author=BA%20Robbins&author=MG%20Choc&volume=42&issue=8&publication_year=2003&pages=989-994&pmid=12730515&doi=10.1093/rheumatology/keg277&)
