# Common Variants in 40 Genes Assessed for Diabetes Incidence and Response to Metformin and Lifestyle Intervention in the Diabetes Prevention Program

## Metadata
**Authors:** Kathleen A Jablonski, Jarred B McAteer, Paul IW de Bakker, Paul W Franks, Toni I Pollin, Robert L Hanson, Richa Saxena, Sarah Fowler, Alan R Shuldiner, William C Knowler, David Altshuler, Jose C Florez, for the Diabetes Prevention Program Research Group
**Journal:** Diabetes
**Date:** 2010 Aug 3
**DOI:** [10.2337/db10-0543](https://doi.org/10.2337/db10-0543)
**PMID:** 20682687
**PMCID:** PMC3279522
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3279522/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC3279522/pdf/zdb2672.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC3279522/pdf/zdb2672.pdf)

## Abstract

**OBJECTIVE:** 
Genome-wide association studies have begun to elucidate the genetic architecture of type 2 diabetes. We examined whether single nucleotide polymorphisms (SNPs) identified through targeted complementary approaches affect diabetes incidence in the at-risk population of the Diabetes Prevention Program (DPP) and whether they influence a response to preventive interventions.

**RESEARCH DESIGN AND METHODS:** 
We selected SNPs identified by prior genome-wide association studies for type 2 diabetes and related traits, or capturing common variation in 40 candidate genes previously associated with type 2 diabetes, implicated in monogenic diabetes, encoding type 2 diabetes drug targets or drug-metabolizing/transporting enzymes, or involved in relevant physiological processes. We analyzed 1,590 SNPs for association with incident diabetes and their interaction with response to metformin or lifestyle interventions in 2,994 DPP participants. We controlled for multiple hypothesis testing by assessing false discovery rates.

**RESULTS:** 
We replicated the association of variants in the metformin transporter gene SLC47A1 with metformin response and detected nominal interactions in the AMP kinase (AMPK) gene STK11, the AMPK subunit genes PRKAA1 and PRKAA2, and a missense SNP in SLC22A1, which encodes another metformin transporter. The most significant association with diabetes incidence occurred in the AMPK subunit gene PRKAG2 (hazard ratio 1.24, 95% CI 1.09–1.40, P = 7 × 10−4). Overall, there were nominal associations with diabetes incidence at 85 SNPs and nominal interactions with the metformin and lifestyle interventions at 91 and 69 mostly nonoverlapping SNPs, respectively. The lowest P values were consistent with experiment-wide 33% false discovery rates.

**CONCLUSIONS:** 
We have identified potential genetic determinants of metformin response. These results merit confirmation in independent samples.

### OBJECTIVE

Genome-wide association studies have begun to elucidate the genetic architecture of type 2 diabetes. We examined whether single nucleotide polymorphisms (SNPs) identified through targeted complementary approaches affect diabetes incidence in the at-risk population of the Diabetes Prevention Program (DPP) and whether they influence a response to preventive interventions.

### RESEARCH DESIGN AND METHODS

We selected SNPs identified by prior genome-wide association studies for type 2 diabetes and related traits, or capturing common variation in 40 candidate genes previously associated with type 2 diabetes, implicated in monogenic diabetes, encoding type 2 diabetes drug targets or drug-metabolizing/transporting enzymes, or involved in relevant physiological processes. We analyzed 1,590 SNPs for association with incident diabetes and their interaction with response to metformin or lifestyle interventions in 2,994 DPP participants. We controlled for multiple hypothesis testing by assessing false discovery rates.

### RESULTS

We replicated the association of variants in the metformin transporter gene *SLC47A1* with metformin response and detected nominal interactions in the AMP kinase (AMPK) gene *STK11*, the AMPK subunit genes *PRKAA1* and *PRKAA2*, and a missense SNP in *SLC22A1*, which encodes another metformin transporter. The most significant association with diabetes incidence occurred in the AMPK subunit gene *PRKAG2* (hazard ratio 1.24, 95% CI 1.09–1.40, *P* = 7 × 10^−4^). Overall, there were nominal associations with diabetes incidence at 85 SNPs and nominal interactions with the metformin and lifestyle interventions at 91 and 69 mostly nonoverlapping SNPs, respectively. The lowest *P* values were consistent with experiment-wide 33% false discovery rates.

### CONCLUSIONS

We have identified potential genetic determinants of metformin response. These results merit confirmation in independent samples.

## RESEARCH DESIGN AND METHODS

### The DPP.

The DPP was a 27-center randomized clinical trial in the U.S. that assessed whether metformin or lifestyle interventions prevent or delay development of diabetes in high-risk individuals. The DPP enrolled 3,234 overweight or obese people without diabetes but with impaired glucose tolerance and elevated fasting glucose and randomized them to placebo, metformin (850 mg twice daily), or a lifestyle intervention program consisting of individual and group counseling sessions conducted by dietary and exercise professionals aimed at ≥7% weight loss and ≥150 min of physical activity per week. A fourth arm of 585 subjects assigned to troglitazone (400 mg daily) was stopped because of hepatotoxicity ([7](#B7)). The primary end point was development of diabetes, ascertained by semi-annual measurement of fasting glucose or an annual 75-g oral glucose tolerance test, either of which was confirmed on a second occasion. The metformin and lifestyle interventions reduced the incidence of diabetes by 31% (95% CI 17–43) and 58% (95% CI 48–66), respectively, versus placebo ([6](#B6)). The 2,994 participants in the placebo, metformin, and lifestyle arms who gave informed consent for genetic investigation are the subjects of this study, which was approved by institutional review boards at each of the 27 participating sites. Their demographic characteristics are shown in [Table 1](#T1).

### TABLE 1.

|   | Overall | Placebo | Metformin | Lifestyle |
| --- | --- | --- | --- | --- |
| n | 2,994 | 1,000 | 990 | 1,004 |
| Sex |   |   |   |   |
| Male | 982 (32.8) | 311 (31.1) | 344 (34.7) | 327 (32.6) |
| Female | 2,012 (67.2) | 689 (68.9) | 646 (65.3) | 677 (67.4) |
| Race or ethnic group |   |   |   |   |
| White | 1,674 (55.9) | 557 (55.7) | 570 (57.6) | 547 (54.5) |
| African American | 612 (20.4) | 210 (21.0) | 206 (20.8) | 196 (19.5) |
| Hispanic | 498 (16.6) | 164 (16.4) | 158 (16.0) | 176 (17.5) |
| Asian/Pacific Islander | 128 (4.3) | 39 (3.9) | 33 (3.3) | 56 (5.6) |
| American Indian | 82 (2.7) | 30 (3.0) | 23 (2.3) | 29 (2.9) |
| Family history of diabetes | 2,089 (69.8) | 703 (70.4) | 683 (69.0) | 703 (70.1) |
| History of gestational diabetes mellitus | 321 (16.0) | 105 (15.2) | 104 (16.1) | 112 (16.5) |
| Quantitative traits |   |   |   |   |
| Age (years) | 50.8 ± 10.7 | 50.5 ± 10.5 | 51.0 ± 10.4 | 50.7 ± 11.4 |
| Weight (kg) | 94.6 ± 20.3 | 94.8 ± 20.2 | 94.6 ± 20.0 | 94.5 ± 20.8 |
| BMI (kg/m2) | 34.1 ± 6.7 | 34.3 ± 6.7 | 34.0 ± 6.7 | 34.0 ± 6.8 |
| Waist circumference (cm) | 105.2 ± 14.6 | 105.3 ± 14.5 | 105.0 ± 14.6 | 105.3 ± 14.9 |
| Waist-to-hip ratio | 0.92 ± 0.09 | 0.92 ± 0.08 | 0.93 ± 0.09 | 0.93 ± 0.09 |
| Plasma glucose (mg/dl) |   |   |   |   |
| In the fasting state | 106.7 ± 8.2 | 107.0 ± 8.4 | 106.7 ± 8.4 | 106.5 ± 7.9 |
| 2 h after an oral glucose load | 164.7 ± 17.1 | 164.6 ± 17.2 | 165.1 ± 17.2 | 164.4 ± 16.9 |
| Glycated hemoglobin (%) | 5.91 ± 0.51 | 5.91 ± 0.50 | 5.91 ± 0.51 | 5.91 ± 0.51 |
| Leisure physical activitiy (MET h/week) | 16.1 ± 25.9 | 16.6 ± 29.1 | 16.4 ± 25.9 | 15.4 ± 22.4 |

Table 1 Caption: Baseline characteristics of the DPP participants enrolled in the genetic study

### SNP selection.

We selected SNPs in two ways: *1*) SNPs in high-likelihood candidate genes and *2*) SNPs identified by ongoing GWASs for type 2 diabetes or related metabolic traits. The 40 candidate genes were tentatively associated with type 2 diabetes, implicated in monogenic forms of diabetes, known to encode type 2 diabetes drug targets or drug-metabolizing/transporting enzymes, or involved in cellular metabolism, hormonal regulation, or response to exercise ([Table 2](#T2)). We used *Tagger* ([8](#B8)) to capture (at *r*^2^ ≥ 0.8) all common (minor allele frequency >5%) variations in European (CEU) and African (YRI) HapMap populations in these candidate genes. For seven additional genes (*ACE*, *CASQ1*, *GCKR*, *IRS1*, *KCNQ1*, *LIPC*, and *NOS3*), rather than attempting full coverage of genetic variation, we selected a limited number of SNPs previously associated with the phenotypes of interest. As the study evolved, it became obvious that previous reports of genetic association provided an equally compelling—or perhaps even higher—prior probability of true association with type 2 diabetes traits than biological function alone; thus, we also focused on GWASs whose results were available at the time this custom-made genotyping array was designed: SNPs associated with type 2 diabetes in the Diabetes Genetics Initiative ([9](#B9)), DIAGRAM ([10](#B10)), or three smaller 100K SNP GWASs in which we participated ([11](#B11)–[13](#B13)); SNPs tentatively associated with quantitative glycemic traits (fasting glucose, the insulinogenic index, and insulin resistance by homeostasis model assessment) in the Diabetes Genetics Initiative; or SNPs associated with obesity ([14](#B14),[15](#B15)) or lipid traits ([16](#B16)–[18](#B18)). For quality control and analytical reasons, we also included some SNPs previously genotyped in these samples, as well as ancestry-informative markers to derive a global proportion of geographic ancestry in African American ([19](#B19)) or Hispanic ([20](#B20)) participants. Finally, we included a small number of SNPs provided by investigators leading ancillary studies approved by the DPP ancillary studies and genetics subcommittees. The total number of SNPs analyzed for each category is shown in [Table 2](#T2).

### TABLE 2.

| Category | n |
| --- | --- |
| Candidate genes (n SNPs) | 1,256 |
| Comprehensive coverage | 1,241 |
| Monogenic diabetes | 317 |
| ABCC8 (84), GCK (37), HNF1A (20), HNF1B (74), HNF4A (66), KCNJ11 (9), NEUROD1 (14), PDX1 (13) |   |
| Monogenic obesity | 21 |
| MC4R (21) |   |
| Previously associated with type 2 diabetes | 60 |
| CAPN10 (33), PTPN1 (27) |   |
| Drug targets | 190 |
| PPARG (59), PRKAA1 (9), PRKAA2 (18), PRKAB1 (10), PRKAB2 (13), PRKAG1 (7), PRKAG2 (53), PRKAG3 (11), STK11 (10) |   |
| Drug-metabolizing enzymes and transporters | 135 |
| CYP3A4 (15), SLC22A1 (47), SLC22A2 (44), SLC47A1 (29) |   |
| Hormonal regulation | 117 |
| ADIPOQ (25), ADIPOR1 (22), ADIPOR2 (31), GCG (13), ITLN1 (11), ITLN2 (15) |   |
| Cellular energy | 334 |
| FOXO1A (35), PCK1 (37), PCK2 (14), PKLR (10), PPARA (63), PPARGC1A (79), PPARGC1B (96) |   |
| Response to exercise | 67 |
| CREB1 (13), MEF2A (38), MEF2D (16) |   |
| Select SNPs in candidate genes | 15 |
| ACE (1), CASQ1 (2), GCKR (2), IRS1 (2), KCNQ1 (1), LIPC (1), NOS3 (6) |   |
| Ancillary studies | 26 |
| ALOX5 (15), IL6 (1), renin-angiotensin system (7), USF1 (2), TNF (1) |   |
| GWASs | 248 |
| Amish 100K GWAS (13): Type 2 diabetes | 46 |
| Framingham Heart Study 100K GWAS (11): Type 2 diabetes | 38 |
| Pima 100K GWAS (12): Type 2 diabetes | 12 |
| Obesity (14,15) | 11 |
| Diabetes Genetics Initiative (9) |   |
| Fasting glucose | 37 |
| Insulin resistance by homeostasis model assessment | 16 |
| Insulinogenic index | 17 |
| Type 2 diabetes | 30 |
| DIAGRAM (10): Type 2 diabetes | 6 |
| Lipids (16–18) | 35 |
| Quality control | 60 |
| African American ancestry informative markers (19) | 29 |
| Hispanic ancestry informative markers (20) | 29 |
| Previously genotyped in DPP | 2 |
| Total | 1,590 |

Table 2 Caption: Number of SNPs analyzed per selection category

### Genotyping.

We initially designed a 1,536-SNP oligonucleotide pool array for the Illumina BeadArray platform (Illumina, San Diego, CA). In the 1,445 SNPs that passed quality control metrics, the sample pass rate was 99.8% and the average genotyping call rate per SNP was 98.5%. Because 91 SNPs failed genotyping on the oligonucleotide pool array, we assessed the adequacy of the coverage afforded by the successfully genotyped SNPs in each region. To rescue relevant SNPs, we used linkage disequilibrium (LD) to select proxy SNPs highly correlated to those that had failed and genotyped them on a Sequenom iPLEX platform. After quality control, 1,590 SNPs were available for analysis.

### Statistical analysis.

We tested the effect of each SNP on diabetes incidence under an additive genetic model by Cox proportional hazards models, using age, sex, ethnicity, and treatment arm as covariates and including treatment (metformin or lifestyle) × genotype interaction terms. In secondary analyses, we stratified participants by treatment arm; if the interaction *P* value was nominally significant, only stratified analyses were considered. We used the MACH software ([21](#B21)) and the HapMap CEU population to impute allelic calls at SNPs not directly genotyped in the DPP. Because of concerns regarding the accuracy of imputation methods in admixed populations, we restricted this procedure to individuals of self-described non-Hispanic white ethnicity. Genotype-phenotype correlations on imputed data were considered confirmatory of prior associations, as well as an initial fine-mapping exploration. Using the program STRUCTURE ([22](#B22)), we applied these markers trained on the HapMap populations to assign a proportion of global European ancestry to each DPP participant.

We considered two sequential approaches to correct for multiple hypothesis testing based on the number of SNPs examined ([23](#B23)). We first ran 1,000 permutations in which diabetes outcome was randomly assigned to an individual's genotype within each ethnicity and treatment group (keeping sex and age together with genotype, and BMI with diabetes outcome). The *P* value for the overall null hypothesis is the fraction of permutations (*n*/1,000) for which the scalar statistic is at least as extreme as that observed for the data ([24](#B24)). To estimate the expected proportion of type I errors among the rejected hypotheses, we also computed false discovery rates (FDRs) as in Benjamini and Hochberg ([25](#B25)).

## RESULTS

Supplementary Table 1 (available in an online appendix at [http://diabetes.diabetesjournals.org/cgi/content/full/db10-0543/DC1](http://diabetes.diabetesjournals.org/cgi/content/full/db10-0543/DC1)) shows that we achieved adequate coverage of all 40 genes in the two targeted populations, with 37 genes reaching at least 80% of common variants captured at *r*^2^ ≥ 0.8 in Europeans and all 40 reaching at least 70% of common variants captured at that level (comparable numbers were obtained in Africans). The average proportion of European ancestry among the DPP self-described white participants, as determined by ancestry-informative markers, was 98.9%, and the average proportion of West-African ancestry among DPP self-described African American participants was 89.3%. Given these results, we used self-described ethnicity as a covariate for these analyses. The full set of results is available in supplementary Table 2.

[Table 3](#T3) shows the candidate gene regions harboring variants nominally associated with diabetes incidence in the treatment-adjusted models for the full study (i.e., there was no evidence for interaction with either intervention); only the top SNP within each gene region (out of 85 nominal associations) is given. The most significant associations occurred at SNPs in the AMP kinase (AMPK) subunit gene *PRKAG2* (hazard ratio [HR] 1.24, 95% CI 1.09–1.40, *P* = 7.0 × 10^−4^ for the top SNP rs5017427, which is consistent with an experiment-wide 34% FDR). Twelve other *PRKAG2* SNPs were nominally associated with diabetes (five in the top ten). Although most of them are in moderate to high LD with the index SNP (*r*^2^ ranging from 0.49 to 1.0 in HapMap CEU), at least two of them (rs954482 and rs2727537) are only weakly correlated with rs5017427 (*r*^2^ 0.07 and 0.05, respectively). Nevertheless, the consistency of the association signal in this region provides reassurance with regard to the absence of genotyping artifacts in our dataset. Of SNPs previously associated with type 2 diabetes in the 100K Amish, Framingham, or Pima GWASs, three (rs1422930 in *ODZ2*, rs1859441 near *COL2A1* and *SENP1*, and rs385909 near *SH3YL1*) had consistent nominal associations with diabetes incidence in the DPP, and two had nominally significant associations (rs10520926 and rs3136279) in the opposite direction. On the other hand, none of the six SNPs selected from the DIAGRAM meta-analysis (original odds ratio [OR] ranging from 1.05 to 1.15) were nominally significant in the DPP. Fifteen SNPs in genes that cause either maturity-onset diabetes of the young or neonatal diabetes were nominally associated with diabetes incidence; one of them, rs11868513 in *HNF1B* (not in LD with the previously type 2 diabetes–associated SNP rs757210), was strongly associated with diabetes incidence in the placebo arm (HR 1.69, 95% CI 1.36–2.10, *P* = 2 × 10^−6^). Finally, 14 SNPs in genes that encode metformin transporters (*SLC22A1*, *SLC22A2*, and *SLC47A1*) were nominally associated with diabetes incidence. Of the 85 nominal associations with diabetes incidence in DPP, only two SNPs (rs651164 in *SLC22A1* and rs3736265 in *PPARGC1A*) were nominally associated with type 2 diabetes in DIAGRAM in a consistent direction (OR 1.08, 95% CI 1.02–1.16, *P* = 0.01, and OR 1.15, 95% CI 1.01–1.31, *P* = 0.04, respectively), with 60 other SNPs not being nominally significantly associated in DIAGRAM and 23 SNPs not captured in that dataset.

### TABLE 3.

| SNP | Chromosome | Position (NCBI 36) | Gene | Alleles (m/M) | Overall, treatment-adjusted |  |  |  | Placebo |  | Metformin |  | Lifestyle |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| HR (95% CI) | Observed P | Permuted P | FDR Q | HR (95% CI) | Observed P | HR (95% CI) | Observed P | HR (95% CI) | Observed P |  |  |  |  |  |
| rs5017427 | 7 | 150886136 | PRKAG2 | T/C | 1.24 (1.09–1.40) | 0.0007 | 0.001 | 0.34 | 1.26 (1.04–1.53) | 0.02 | 1.15 (0.93–1.42) | 0.20 | 1.34 (1.05–1.71) | 0.02 |
| rs2453583 | 17 | 19382628 | SLC47A1 | T/A | 0.81 (0.72–0.92) | 0.001 | 0.003 | 0.34 | 0.84 (0.70–1.02) | 0.08 | 0.68 (0.54–0.86) | 0.001 | 0.94 (0.73–1.21) | 0.65 |
| rs315978 | 6 | 160572848 | SLC22A2 | T/C | 0.73 (0.60–0.89) | 0.002 | 0.002 | 0.34 | 0.75 (0.57–1.00) | 0.05 | 0.67 (0.47–0.96) | 0.03 | 0.74 (0.49–1.11) | 0.14 |
| rs4273018 | 15 | 97910079 | MEF2A | T/C | 1.20 (1.07–1.35) | 0.002 | 0.005 | 0.50 | 1.20 (1.00–1.44) | 0.05 | 1.37 (1.11–1.67) | 0.003 | 0.96 (0.76–1.23) | 0.76 |
| rs9551419 | 13 | 27378356 | PDX1 | T/C | 0.77 (0.65–0.92) | 0.004 | 0.003 | 0.34 | 0.77 (0.59–1.00) | 0.05 | 0.85 (0.63–1.14) | 0.27 | 0.67 (0.45–1.00) | 0.05 |
| rs1042531 | 20 | 55574386 | PCK1 | C/A | 0.84 (0.74–0.94) | 0.004 | 0.002 | 0.34 | 0.83 (0.69–1.00) | 0.05 | 0.98 (0.80–1.20) | 0.85 | 0.64 (0.49–0.84) | 0.001 |
| rs1342387 | 1 | 201180979 | ADIPOR1 | A/G | 1.17 (1.05–1.31) | 0.006 | 0.009 | 0.55 | 1.22 (1.02–1.45) | 0.03 | 1.07 (0.88–1.30) | 0.51 | 1.24 (0.99–1.56) | 0.07 |
| rs1388332 | 4 | 23438412 | PPARGC1A | G/A | 1.32 (1.08–1.61) | 0.006 | 0.007 | 0.53 | 1.17 (0.85–1.60) | 0.33 | 1.33 (0.94–1.88) | 0.10 | 1.51 (1.03–2.21) | 0.03 |
| rs6093976 | 20 | 42469194 | HNF4A | A/G | 0.78 (0.65–0.94) | 0.008 | 0.008 | 0.53 | 0.76 (0.57–1.00) | 0.05 | 0.85 (0.62–1.16) | 0.31 | 0.75 (0.51–1.10) | 0.14 |
| rs651164 | 6 | 160501364 | SLC22A1 | T/C | 1.18 (1.04–1.33) | 0.008 | 0.007 | 0.53 | 1.08 (0.90–1.30) | 0.39 | 1.14 (0.92–1.42) | 0.23 | 1.40 (1.09–1.78) | 0.01 |
| rs12330015 | 22 | 44968942 | PPARA | C/T | 0.79 (0.66–0.95) | 0.01 | 0.01 | 0.58 | 0.82 (0.63–1.07) | 0.15 | 0.78 (0.57–1.08) | 0.14 | 0.72 (0.49–1.06) | 0.10 |
| rs2755209 | 13 | 40035804 | FOXO1 | C/A | 1.16 (1.04–1.31) | 0.01 | 0.01 | 0.58 | 1.05 (0.89–1.25) | 0.56 | 1.29 (1.05–1.58) | 0.02 | 1.25 (0.97–1.60) | 0.08 |
| rs10875552 | 5 | 149169682 | PPARGC1B | C/T | 0.84 (0.74–0.96) | 0.01 | 0.008 | 0.53 | 0.91 (0.75–1.12) | 0.37 | 0.89 (0.72–1.11) | 0.32 | 0.65 (0.48–0.86) | 0.003 |
| rs739690 | 11 | 17373454 | KCNJ11 | C/G | 1.87 (1.15–3.05) | 0.01 | 0.01 | 0.66 | 1.16 (0.48–2.79) | 0.74 | 2.66 (1.33–5.31) | 0.01 | 2.23 (0.76–6.56) | 0.15 |
| rs7811022 | 7 | 99223619 | CYP3A4 | G/C | 1.33 (1.06–1.67) | 0.01 | 0.02 | 0.67 | 1.16 (0.81–1.65) | 0.42 | 1.99 (1.38–2.87) | 0.0002 | 0.95 (0.56–1.60) | 0.84 |
| rs832646 | 2 | 182266649 | NEUROD1 | C/T | 0.57 (0.35–0.91) | 0.02 | 0.02 | 0.68 | 0.59 (0.31–1.13) | 0.11 | 0.79 (0.37–1.72) | 0.56 | 0.24 (0.06–1.01) | 0.05 |
| rs11836547 | 12 | 1756708 | ADIPOR2 | C/G | 1.37 (1.05–1.78) | 0.02 | 0.02 | 0.67 | 1.60 (1.09–2.34) | 0.02 | 0.92 (0.56–1.52) | 0.75 | 1.72 (1.00–2.95) | 0.05 |
| rs916829 | 11 | 17397049 | ABCC8 | A/G | 0.80 (0.66–0.97) | 0.03 | 0.04 | 0.78 | 0.76 (0.56–1.02) | 0.06 | 0.89 (0.63–1.25) | 0.50 | 0.78 (0.52–1.16) | 0.21 |
| rs12951345 | 17 | 33151976 | HNF1B | G/T | 1.17 (1.02–1.34) | 0.03 | 0.03 | 0.73 | 1.13 (0.91–1.40) | 0.27 | 1.20 (0.95–1.52) | 0.12 | 1.20 (0.91–1.59) | 0.20 |
| rs709159 | 3 | 12456203 | PPARG | C/A | 1.16 (1.01–1.33) | 0.04 | 0.04 | 0.82 | 1.06 (0.86–1.31) | 0.56 | 1.21 (0.95–1.56) | 0.13 | 1.27 (0.96–1.68) | 0.09 |
| rs12058717 | 1 | 159108473 | ITLN1 | T/C | 1.28 (1.01–1.63) | 0.04 | 0.049 | 0.87 | 1.24 (0.89–1.74) | 0.20 | 1.35 (0.87–2.09) | 0.18 | 1.32 (0.77–2.28) | 0.31 |

Table 3 Caption: Candidate gene variants nominally associated with diabetes incidence in the DPP

[Table 4](#T4) shows the candidate gene regions harboring variants that have a nominally significant genotype × metformin interaction; only the top SNP within each gene region is given (out of 91 nominal associations). The best result was consistent with a study-wide 33% FDR. At rs8065082 in *SLC47A1*, there was a nominal interaction with metformin (*P* = 0.006), with the minor allele associated with lower diabetes incidence in the metformin arm (HR 0.78, 95% CI 0.64–0.96, *P* = 0.02) but not in the placebo arm (1.15, 0.97–1.37, *P* = 0.11). At this locus, major allele homozygotes did not benefit from metformin with regard to diabetes prevention (HR 1.07, 95% CI 0.77–1.50, vs. placebo, *P* = 0.68), whereas minor allele carriers did (0.58, 0.46–0.73, vs. placebo, *P* < 0.001; [Fig. 1](#F1)). We also noted a nominally significant interaction of a missense SNP in *SLC22A1* (rs683369, encoding L160F) with metformin, with the major allele protecting from diabetes in the metformin arm (HR 0.69, 95% CI 0.53–0.89, *P* = 0.004) but not the placebo arm (1.01, 0.79–1.30, *P* = 0.91); the major allele is therefore associated with 31% risk reduction in diabetes incidence but only under the action of metformin. In this arm, the likelihood of developing diabetes depended on the number of phenylalanine alleles (HR 0.72, 95% CI 0.59–0.88, vs. placebo for LL homozygotes; 0.92, 0.66–1.28, for heterozygotes; and 1.44, 0.56–3.67, for FF homozygotes). There were five nominally significant interactions at SNPs encoding putative drug targets for metformin, in the gene encoding the AMPK kinase *STK11* and the AMPK subunit genes *PRKAA1*, *PRKAA2*, and *PRKAB2*, respectively. A total of 22 SNPs in the *ABCC8-KCNJ11* region also had nominally significant interactions with metformin, including rs5215, which is tightly linked to the widely replicated type 2 diabetes–associated missense SNP rs5219 (E23K) in *KCNJ11*.

### TABLE 4.

| SNP | Chromosome | Position (NCBI 36) | Gene | Alleles (m/M) | Interaction genotype * metformin |  |  | Placebo |  | Metformin |  | Lifestyle |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| Observed P | Permuted P | FDR Q | HR (95% CI) | Observed P | HR (95% CI) | Observed P | HR (95% CI) | Observed P |  |  |  |  |  |
| rs11868513 | 17 | 33126805 | HNF1B | A/G | 0.0007 | 0.001 | 0.33 | 1.69 (1.36–2.10) | 0.000002 | 0.87 (0.65–1.16) | 0.33 | 1.08 (0.79–1.47) | 0.64 |
| rs4148609 | 11 | 17441307 | ABCC8 | A/G | 0.002 | 0.002 | 0.33 | 1.24 (1.04–1.48) | 0.02 | 0.79 (0.63–0.98) | 0.03 | 1.06 (0.83–1.37) | 0.64 |
| rs11086926 | 20 | 42492111 | HNF4A | G/T | 0.002 | 0.001 | 0.33 | 0.82 (0.61–1.11) | 0.20 | 1.81 (1.35–2.43) | 0.0001 | 1.29 (0.91–1.83) | 0.15 |
| rs10213440 | 4 | 23475437 | PPARGC1A | C/T | 0.002 | 0.002 | 0.33 | 0.76 (0.59–0.97) | 0.03 | 1.31 (1.03–1.66) | 0.03 | 1.07 (0.80–1.43) | 0.65 |
| rs4424892 | 15 | 97921908 | MEF2A | G/A | 0.003 | 0.006 | 0.38 | 0.88 (0.72–1.09) | 0.24 | 1.43 (1.14–1.80) | 0.002 | 0.78 (0.58–1.05) | 0.10 |
| rs8065082 | 17 | 19405783 | SLC47A1 | T/C | 0.006 | 0.008 | 0.41 | 1.15 (0.97–1.37) | 0.11 | 0.78 (0.64–0.96) | 0.02 | 1.09 (0.86–1.38) | 0.46 |
| rs6666307 | 1 | 154719358 | MEF2D | T/A | 0.009 | 0.005 | 0.38 | 0.54 (0.26–1.09) | 0.08 | 2.15 (1.22–3.80) | 0.01 | 1.11 (0.44–2.77) | 0.83 |
| rs3792269 | 2 | 241180152 | CAPN10 | G/A | 0.01 | 0.01 | 0.41 | 1.61 (1.28–2.02) | 0.00005 | 0.95 (0.70–1.28) | 0.73 | 0.81 (0.55–1.18) | 0.27 |
| rs758027 | 12 | 1662034 | ADIPOR2 | C/T | 0.01 | 0.002 | 0.33 | 0.34 (0.15–0.75) | 0.01 | 1.31 (0.76–2.29) | 0.33 | 0.75 (0.26–2.15) | 0.59 |
| rs7124355 | 11 | 17369536 | KCNJ11 | A/G | 0.01 | 0.01 | 0.41 | 0.84 (0.68–1.04) | 0.10 | 1.25 (0.99–1.59) | 0.06 | 0.89 (0.68–1.17) | 0.41 |
| rs662301 | 6 | 160616909 | SLC22A2 | T/C | 0.02 | 0.02 | 0.47 | 0.78 (0.49–1.22) | 0.27 | 1.57 (1.09–2.27) | 0.02 | 0.60 (0.31–1.17) | 0.13 |
| rs2908289 | 7 | 44190467 | GCK | A/G | 0.02 | 0.02 | 0.47 | 0.86 (0.69–1.08) | 0.19 | 1.29 (1.02–1.64) | 0.04 | 0.97 (0.73–1.31) | 0.86 |
| rs741765 | 19 | 1172545 | STK11 | T/C | 0.02 | 0.02 | 0.47 | 1.17 (0.96–1.43) | 0.13 | 0.82 (0.64–1.04) | 0.10 | 0.95 (0.72–1.26) | 0.73 |
| rs6701920 | 1 | 159181541 | ITLN2 | A/G | 0.02 | 0.01 | 0.41 | 1.86 (1.19–2.91) | 0.01 | 0.49 (0.17–1.40) | 0.19 | 0.69 (0.27–1.78) | 0.45 |
| rs741579 | 5 | 149165563 | PPARGC1B | G/A | 0.02 | 0.01 | 0.38 | 2.41 (1.08–5.37) | 0.03 | 0.23 (0.03–1.62) | 0.14 | 0.68 (0.17–2.71) | 0.58 |
| rs4810083 | 20 | 55553677 | PCK1 | T/C | 0.03 | 0.03 | 0.53 | 1.14 (0.95–1.37) | 0.15 | 0.84 (0.69–1.02) | 0.08 | 1.14 (0.90–1.44) | 0.27 |
| rs9803799 | 1 | 56952660 | PRKAA2 | G/T | 0.03 | 0.02 | 0.49 | 1.17 (0.85–1.62) | 0.34 | 0.66 (0.43–1.02) | 0.06 | 1.13 (0.72–1.78) | 0.59 |
| rs17367421 | 1 | 153553865 | PKLR | C/G | 0.03 | 0.03 | 0.53 | 0.47 (0.23–0.96) | 0.04 | 1.21 (0.73–1.99) | 0.46 | 1.36 (0.77–2.38) | 0.29 |
| rs683369 | 6 | 160471194 | SLC22A1 | G/C | 0.03 | 0.03 | 0.53 | 0.99 (0.77–1.27) | 0.91 | 1.45 (1.12–1.88) | 0.004 | 0.82 (0.58–1.17) | 0.27 |
| rs249429 | 5 | 40817996 | PRKAA1 | C/T | 0.04 | 0.03 | 0.53 | 1.22 (1.01–1.46) | 0.04 | 0.89 (0.71–1.13) | 0.34 | 1.24 (0.97–1.59) | 0.09 |
| rs6733736 | 2 | 162704406 | GCG | G/A | 0.04 | 0.01 | 0.43 | 0.65 (0.20–2.13) | 0.48 | 3.70 (1.56–8.80) | 0.003 | 0.59 (0.08–4.45) | 0.61 |
| rs4253652 | 22 | 44947503 | PPARA | G/A | 0.04 | 0.02 | 0.47 | 0.60 (0.27–1.33) | 0.20 | 2.29 (1.10–4.76) | 0.03 | 1.07 (0.44–2.65) | 0.88 |
| rs6690158 | 1 | 145123867 | PRKAB2 | T/C | 0.04 | 0.03 | 0.53 | 0.58 (0.33–1.03) | 0.06 | 1.42 (0.85–2.36) | 0.18 | 1.56 (0.87–2.81) | 0.14 |

Table 4 Caption: Candidate gene variants showing a nominally significant interaction with the metformin intervention in the DPP

### FIG. 1.

![FIG. 1.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/af58/3279522/bd22dde2d31d/zdb0101063160001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3279522_zdb0101063160001.jpg)

Diabetes incidence in the DPP, by genotype at rs8065082 in the SLC47A1 gene. This SNP is in tight LD with rs2289669 (r2 ∼0.8), whose major allele predicts a poorer response to metformin (5). In the DPP, major allele homozygotes at rs8065082 did not benefit from metformin with regard to diabetes prevention, whereas minor allele carriers did (P < 0.001).

[Table 5](#T5) shows the candidate gene regions harboring variants that have a nominally significant interaction with the lifestyle intervention; only the top SNP within each gene region is given (out of 69 nominal associations). The best result was consistent with an experiment-wide 84% FDR. Twelve of the top findings were in four AMPK subunit genes (*PRKAA2*, *PRKAB2*, *PRKAG1*, and *PRKAG2*), and 11 SNPs clustered around the peroxisome proliferator–associated receptor γ coactivators 1α and 1β (*PPARGC1A* and *PPARGC1B*, respectively).

### TABLE 5.

| SNP | Chromosome | Position (NCBI 36) | Gene | Alleles (m/M) | Interaction genotype * lifestyle |  |  | Placebo |  | Metformin |  | Lifestyle |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| Observed P | Permuted P | FDR Q | HR (95% CI) | Observed P | HR (95% CI) | Observed P | HR (95% CI) | Observed P |  |  |  |  |  |
| rs3792269 | 2 | 241180152 | CAPN10 | G/A | 0.005 | 0.008 | 0.84 | 1.61 (1.28–2.02) | 0.00005 | 0.95 (0.70–1.28) | 0.73 | 0.81 (0.55–1.18) | 0.27 |
| rs2425640 | 20 | 42461451 | HNF4A | A/G | 0.01 | 0.01 | 0.84 | 0.83 (0.69–1.01) | 0.06 | 0.93 (0.75–1.16) | 0.51 | 1.23 (0.96–1.58) | 0.11 |
| rs1342514 | 1 | 56959948 | PRKAA2 | G/C | 0.01 | 0.005 | 0.84 | 1.07 (0.90–1.29) | 0.44 | 1.14 (0.92–1.41) | 0.25 | 0.71 (0.54–0.92) | 0.01 |
| rs4725408 | 7 | 150881542 | PRKAG2 | C/T | 0.02 | 0.005 | 0.84 | 0.82 (0.39–1.73) | 0.61 | 1.02 (0.41–2.55) | 0.97 | 2.90 (1.52–5.56) | 0.001 |
| rs7599142 | 2 | 162701737 | GCG | A/G | 0.02 | 0.01 | 0.84 | 1.02 (0.84–1.23) | 0.86 | 1.00 (0.81–1.22) | 0.97 | 0.66 (0.51–0.86) | 0.002 |
| rs17367421 | 1 | 153553865 | PKLR | C/G | 0.02 | 0.01 | 0.84 | 0.47 (0.23–0.96) | 0.04 | 1.21 (0.73–1.99) | 0.46 | 1.36 (0.77–2.38) | 0.29 |
| rs12374408 | 4 | 23463581 | PPARGC1A | T/C | 0.02 | 0.03 | 0.84 | 1.13 (0.94–1.36) | 0.20 | 0.94 (0.76–1.17) | 0.60 | 0.77 (0.59–1.02) | 0.07 |
| rs3751151 | 12 | 119926582 | HNF1A | T/A | 0.02 | 0.02 | 0.84 | 0.89 (0.73–1.09) | 0.26 | 0.82 (0.65–1.04) | 0.10 | 1.30 (1.01–1.67) | 0.04 |
| rs11024298 | 11 | 17448407 | ABCC8 | T/G | 0.02 | 0.01 | 0.84 | 1.17 (0.82–1.67) | 0.39 | 1.01 (0.65–1.58) | 0.95 | 0.44 (0.21–0.92) | 0.03 |
| rs6008306 | 22 | 45018756 | PPARA | T/C | 0.02 | 0.01 | 0.84 | 0.52 (0.25–1.08) | 0.08 | 0.96 (0.49–1.91) | 0.91 | 2.05 (0.97–4.33) | 0.06 |
| rs1422429 | 5 | 149146627 | PPARGC1B | A/G | 0.02 | 0.02 | 0.84 | 0.83 (0.69–0.99) | 0.04 | 0.96 (0.79–1.17) | 0.70 | 1.19 (0.94–1.50) | 0.15 |
| rs1054442 | 12 | 47675587 | PRKAG1 | C/A | 0.03 | 0.02 | 0.84 | 1.22 (1.02–1.45) | 0.03 | 1.08 (0.88–1.32) | 0.47 | 0.87 (0.68–1.12) | 0.27 |
| rs9965495 | 18 | 56184656 | MC4R | T/C | 0.03 | 0.02 | 0.84 | 1.09 (0.90–1.31) | 0.39 | 1.20 (0.97–1.49) | 0.09 | 0.75 (0.58–0.98) | 0.04 |
| rs1008284 | 17 | 33136571 | HNF1B | T/C | 0.03 | 0.03 | 0.84 | 1.30 (1.08–1.58) | 0.01 | 0.91 (0.71–1.15) | 0.43 | 0.89 (0.68–1.18) | 0.42 |
| rs2018675 | 17 | 19382123 | SLC47A1 | T/C | 0.03 | 0.02 | 0.84 | 1.18 (0.99–1.40) | 0.06 | 1.13 (0.92–1.38) | 0.24 | 0.88 (0.69–1.13) | 0.33 |
| rs8032587 | 15 | 98068933 | MEF2A | C/T | 0.03 | 0.02 | 0.84 | 1.19 (0.72–1.95) | 0.50 | 1.07 (0.56–2.05) | 0.83 | 0.31 (0.10–1.00) | 0.05 |
| rs7626560 | 3 | 12450088 | PPARG | A/G | 0.04 | 0.04 | 0.84 | 1.16 (0.93–1.45) | 0.19 | 1.12 (0.88–1.44) | 0.36 | 0.76 (0.55–1.06) | 0.10 |
| rs17161829 | 7 | 99187763 | CYP3A4 | A/G | 0.04 | 0.01 | 0.84 | 1.05 (0.69–1.60) | 0.82 | 1.24 (0.77–1.98) | 0.38 | 0.32 (0.11–0.93) | 0.04 |
| rs6701920 | 1 | 159181541 | ITLN2 | A/G | 0.04 | 0.02 | 0.84 | 1.86 (1.19–2.91) | 0.01 | 0.49 (0.17–1.40) | 0.19 | 0.69 (0.27–1.78) | 0.45 |
| rs461473 | 6 | 160463552 | SLC22A1 | T/C | 0.04 | 0.04 | 0.84 | 0.92 (0.64–1.33) | 0.67 | 0.93 (0.64–1.36) | 0.72 | 1.50 (1.01–2.22) | 0.04 |
| rs17159890 | 1 | 145094998 | PRKAB2 | C/A | 0.046 | 0.05 | 0.84 | 0.84 (0.56–1.26) | 0.40 | 1.01 (0.62–1.63) | 0.98 | 1.88 (1.13–3.14) | 0.02 |
| rs11904814 | 2 | 208135043 | CREB1 | C/A | 0.046 | 0.05 | 0.84 | 1.03 (0.86–1.23) | 0.78 | 1.15 (0.93–1.42) | 0.18 | 0.74 (0.57–0.96) | 0.03 |
| rs17373414 | 3 | 188068221 | ADIPOQ | T/C | 0.049 | 0.06 | 0.84 | 0.82 (0.57–1.18) | 0.28 | 0.90 (0.63–1.29) | 0.57 | 1.34 (0.90–1.98) | 0.15 |

Table 5 Caption: Candidate gene variants showing a nominally significant interaction with the lifestyle intervention in the DPP

Review of 1,609 SNPs imputed in non-Hispanic white DPP participants (supplementary Table 3) revealed the nominal association of other *PRKAG2* SNPs with diabetes incidence (best *P* = 5 × 10^−5^). Imputed SNPs in the *PRKAA1*, *PRKAA2*, and *ABCC8-KCNJ11* regions also had nominally significant interactions with metformin.

## DISCUSSION

We conducted a large-scale genotyping study in the DPP, with the aim to test whether common variants in candidate genes involved in major spheres of human physiology predict diabetes incidence or response to preventive interventions in a multiethnic at-risk population. Our secondary purpose was to characterize the mechanism of action of previously associated variants. We provide evidence supporting a previously reported association of variants in the metformin transporter gene *SLC47A1* with weaker metformin response, here defined as the reduced ability of metformin to lower diabetes incidence ([5](#B5)). We identified a number of nominal associations with diabetes incidence or metformin response in several compelling candidate genes; however, none stand strict statistical correction for multiple hypothesis testing by FDRs.

Correction for multiple tests requires careful consideration in genetic association studies ([26](#B26)). When large numbers of SNPs are tested, methods that are valid in the presence of correlations due to LD, such as permutation methods or evaluation of FDR, are preferred over those that assume independence of SNPs. The scope of the present analysis is guided by technological convenience, and it might be argued that the number of distinct scientific hypotheses formulated, rather than the physical size of the genotyping array, is most relevant to the interpretation of results. However, what constitutes a single hypothesis (e.g., an SNP, a gene, an entire pathway, or a constellation of phenotypes) is subjective. On the other hand, correcting for the equivalent of the universe of independent common variants in the human genome (empirically estimated at ∼1 million [[27](#B27)]) is gaining increasing favor among genetic statisticians. In this context, the novel findings reported here should be viewed as hypothesis-generating.

We previously quantified the power of the DPP to detect modest genetic effects on diabetes incidence ([28](#B28)). Assuming there are no gene-treatment interactions, these calculations show that the overall DPP cohort has 83% power to detect a previously reported effect size of ∼20% for an SNP of 10% frequency at an α level of 0.05, while the placebo, lifestyle modification, and metformin arms have 53, 34, and 44% power, respectively. The DPP has inadequate power for detecting an effect size of <10%. Thus, it is not surprising that the DPP does not replicate all GWAS-derived findings in that range or that it fails to reach genome-wide significance in discovery efforts. Our null results on diabetes incidence for truly associated variants may be due to the high-risk population at baseline, the short time of follow-up (3.2 years on average), and/or the use of interventions effective in reducing diabetes incidence. On the other hand, considering the number of variants likely to influence the phenotypes under study, even submaximal power is likely to provide a number of true positive associations. In this context, genotyped and imputed SNPs in the gene encoding the AMPK γ2 subunit (*PRKAG2*) merit further consideration. While the association of SNPs in genes that encode metformin transporters with type 2 diabetes in the entire DPP cohort (if real) requires explanation, this could be due to a sufficiently strong effect in the metformin arm alone. Alternatively, SNPs in this region could be capturing variants in other nearby genes: for instance, immediately upstream of *SLC22A1* and *SLC22A2* in chromosome 6 lies the gene encoding the insulin-like growth factor 2 receptor (*IGF2R*), an excellent biological candidate.

This study constitutes the first large-scale prospective pharmacogenetic evaluation of metformin action in a controlled clinical trial. The UK Prospective Diabetes Study ([29](#B29)) and A Diabetes Outcome Progression Trial (ADOPT) ([30](#B30)) investigators independently showed that a substantial proportion of patients with type 2 diabetes eventually fail metformin therapy, defined by a need for additional pharmacotherapy to control hyperglycemia. Given the higher prior probability afforded by the known biological role of *SLC47A1* in disposing of metformin and the previously reported genetic association of the major allele at SNP rs2289669 with poorer metformin response ([5](#B5)), validation in the DPP can be convincing without achieving the levels of statistical significance required for novel findings. Our index SNP (rs8065082) is in tight LD with rs2289669 (*r*^2^ ∼0.8 in HapMap CEU) and the direction of effect is consistent in DPP, a cohort nearly 10-fold larger than the one documented in the original report from Rotterdam ([5](#B5)). Thus, our findings confirm those of Becker et al. ([5](#B5)) and suggest that major allele homozygotes at this locus (∼30% of the European population) may experience suboptimal responses to metformin treatment.

Our findings on the *SLC22A1* locus and metformin response are less robust. While our noted association with a missense SNP appears compelling, it is not among the most functional human variants described by Shu et al. ([3](#B3)), and it is in weak LD with rs622342 (*r*^2^ ∼0.14 in HapMap CEU), a *SLC22A1* SNP associated with metformin response in another report from Rotterdam ([31](#B31)). SNP rs622342 was included among our tag SNPs but showed no evidence of an interaction with metformin (nominal *P* = 0.69) or an effect on diabetes incidence in any arm, raising the possibility that the original finding may have been spurious. Similarly, the *SLC22A2* missense SNP rs316019 (A270S), reported to influence metformin renal excretion and affect its plasma concentrations ([32](#B32)), did not significantly interact with metformin in the DPP (nominal *P* = 0.35). Our novel findings in the putative metformin drug targets *STK11* and AMPK require confirmation, as do those in *MEF2A* and *MEF2D*, themselves regulated by AMPK ([33](#B33)). One of the most significant interactions with metformin occurred at an SNP in *HNF4A*; given its role in hepatic gluconeogenesis ([34](#B34)), this intriguing result deserves further exploration. In contrast, the multiple interactions noted in the *ABCC8-KCNJ11* locus reported previously ([35](#B35)) do not offer a clear mechanism of action. Finally, nominal associations with response to lifestyle modification should be replicated in cohorts that underwent a similar intervention.

In summary, we have conducted a large-scale genetic association study in the DPP and replicated the association of a polymorphism in a metformin transporter with metformin response. Other hypothesis-generating results require more detailed characterization in the DPP and follow-up in independent samples. A focus on likely functional variants may uncover loci with stronger effects.

## Supplementary Material

## ACKNOWLEDGMENTS

The National Institute of Diabetes and Digestive and Kidney Diseases (NIDDK) of the National Institutes of Health provided funding to the clinical centers and the coordinating center for the design and conduct of the study and collection, management, analysis, and interpretation of the data. The Southwestern American Indian Centers were supported directly by the NIDDK and the Indian Health Service. The General Clinical Research Center Program, National Center for Research Resources, and the Department of Veterans Affairs supported data collection at many of the clinical centers. Funding for data collection and participant support was also provided by the Office of Research on Minority Health, the National Institute of Child Health and Human Development, the National Institute on Aging, the Office of Research on Women's Health, the Centers for Disease Control and Prevention, and the American Diabetes Association. Bristol-Myers Squibb and Parke-Davis provided medication. This research was also supported in part by the intramural research program of the NIDDK. LifeScan, Health O Meter, Hoechst Marion Roussel, Merck-Medco Managed Care, Merck and Company, Nike Sports Marketing, Slim Fast Foods, and Quaker Oats donated materials, equipment, or medicines for concomitant conditions. McKesson BioServices, Matthews Media Group, and the Henry M. Jackson Foundation provided support services under subcontract with the coordinating center. A complete list of centers, investigators, and staff can be found in the online appendix.

This work was funded by R01 DK072041 to K.A.J., T.I.P., A.R.S., D.A., and J.C.F. J.C.F. is also supported by the Massachusetts General Hospital and a Clinical Scientist Development Award by the Doris Duke Charitable Foundation. This work was partially supported by a Doris Duke Charitable Foundation Distinguished Scientist Clinical Award to D.A. P.W.F. was supported in part by grants from Novo Nordisk, the Swedish Heart-Lung Foundation, the Swedish Diabetes Association, Påhlssons Foundation, the Swedish Research Council, and a Career Development Award from Umeå University. J.C.F. received consulting honoraria from Publicis Healthcare, Merck, bioStrategies, XOMA, and Daiichi-Sankyo and has been a paid invited speaker at internal scientific seminars hosted by Pfizer and Alnylam Pharmaceuticals.

No other potential conflicts of interest relevant to this article were reported.

R.L.H. and W.C.K. contributed to the participant recruitment, interventions, and outcomes assessment. P.W.F., T.I.P., R.L.H., R.S., A.R.S., W.C.K., D.A., and J.C.F. compiled the list of candidate genes. J.B.M. and P.I.W.d.B. performed the tagging procedure in European and African populations. P.W.F., T.I.P., R.L.H., R.S., A.R.S., and J.C.F. selected SNPs within those genes. J.B.M. directed the genotyping with supervision from J.C.F. K.A.J., P.I.W.d.B., T.I.P., R.L.H., and J.C.F. constructed the analytical pipeline. K.A.J. conducted all statistical analyses, with input from P.I.W.d.B., T.I.P., R.L.H., S.F., W.C.K., and D.A. K.A.J., R.L.H., and P.I.W.d.B. implemented the permutation procedure, with input from T.I.P., D.A., and J.C.F. J.B.M. and P.I.W.d.B. derived individual estimates of global ancestry and carried out SNP imputation. J.C.F. wrote the manuscript. All authors contributed to the discussion and reviewed and edited the manuscript.

The investigators gratefully acknowledge the commitment and dedication of the participants of the DPP.

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Prokopenko I, McCarthy MI, Lindgren CM. Type 2 diabetes: new genes, new understanding. Trends Genet 2008;24:613–621  [DOI](https://doi.org/10.1016/j.tig.2008.09.004) | [PMC free article](/articles/PMC7116807/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18952314/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Trends%20Genet&title=Type%202%20diabetes:%20new%20genes,%20new%20understanding&author=I%20Prokopenko&author=MI%20McCarthy&author=CM%20Lindgren&volume=24&publication_year=2008&pages=613-621&pmid=18952314&doi=10.1016/j.tig.2008.09.004&)

2. Pearson ER. Pharmacogenetics in diabetes. Curr Diab Rep 2009;9:172–181  [DOI](https://doi.org/10.1007/s11892-009-0028-3) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19323963/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Diab%20Rep&title=Pharmacogenetics%20in%20diabetes&author=ER%20Pearson&volume=9&publication_year=2009&pages=172-181&pmid=19323963&doi=10.1007/s11892-009-0028-3&)

3. Shu Y, Sheardown SA, Brown C, Owen RP, Zhang S, Castro RA, Ianculescu AG, Yue L, Lo JC, Burchard EG, Brett CM, Giacomini KM. Effect of genetic variation in the organic cation transporter 1 (OCT1) on metformin action. J Clin Invest 2007;117:1422–1431  [DOI](https://doi.org/10.1172/JCI30558) | [PMC free article](/articles/PMC1857259/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17476361/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Invest&title=Effect%20of%20genetic%20variation%20in%20the%20organic%20cation%20transporter%201%20(OCT1)%20on%20metformin%20action&author=Y%20Shu&author=SA%20Sheardown&author=C%20Brown&author=RP%20Owen&author=S%20Zhang&volume=117&publication_year=2007&pages=1422-1431&pmid=17476361&doi=10.1172/JCI30558&)

4. Zhou K, Donnelly LA, Kimber CH, Donnan PT, Doney AS, Leese G, Hattersley AT, McCarthy MI, Morris AD, Palmer CN, Pearson ER. Reduced-function SLC22A1 polymorphisms encoding organic cation transporter 1 and glycemic response to metformin: a GoDARTS study. Diabetes 2009;58:1434–1439  [DOI](https://doi.org/10.2337/db08-0896) | [PMC free article](/articles/PMC2682689/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19336679/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Diabetes&title=Reduced-function%20SLC22A1%20polymorphisms%20encoding%20organic%20cation%20transporter%201%20and%20glycemic%20response%20to%20metformin:%20a%20GoDARTS%20study&author=K%20Zhou&author=LA%20Donnelly&author=CH%20Kimber&author=PT%20Donnan&author=AS%20Doney&volume=58&publication_year=2009&pages=1434-1439&pmid=19336679&doi=10.2337/db08-0896&)

5. Becker ML, Visser LE, van Schaik RH, Hofman A, Uitterlinden AG, Stricker BH. Genetic variation in the multidrug and toxin extrusion 1 transporter protein influences the glucose-lowering effect of metformin in patients with diabetes: a preliminary study. Diabetes 2009;58:745–749  [DOI](https://doi.org/10.2337/db08-1028) | [PMC free article](/articles/PMC2646075/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19228809/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Diabetes&title=Genetic%20variation%20in%20the%20multidrug%20and%20toxin%20extrusion%201%20transporter%20protein%20influences%20the%20glucose-lowering%20effect%20of%20metformin%20in%20patients%20with%20diabetes:%20a%20preliminary%20study&author=ML%20Becker&author=LE%20Visser&author=RH%20van%20Schaik&author=A%20Hofman&author=AG%20Uitterlinden&volume=58&publication_year=2009&pages=745-749&pmid=19228809&doi=10.2337/db08-1028&)

6. Knowler WC, Barrett-Connor E, Fowler SE, Hamman RF, Lachin JM, Walker EA, Nathan DM. Diabetes Prevention Program Research Group Reduction in the incidence of type 2 diabetes with lifestyle intervention or metformin. N Engl J Med 2002;346:393–403  [DOI](https://doi.org/10.1056/NEJMoa012512) | [PMC free article](/articles/PMC1370926/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11832527/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Reduction%20in%20the%20incidence%20of%20type%202%20diabetes%20with%20lifestyle%20intervention%20or%20metformin&author=WC%20Knowler&author=E%20Barrett-Connor&author=SE%20Fowler&author=RF%20Hamman&author=JM%20Lachin&volume=346&publication_year=2002&pages=393-403&pmid=11832527&doi=10.1056/NEJMoa012512&)

7. Diabetes Prevention Program Research Group The Diabetes Prevention Program: Design and methods for a clinical trial in the prevention of type 2 diabetes. Diabetes Care 1999;22:623–634  [DOI](https://doi.org/10.2337/diacare.22.4.623) | [PMC free article](/articles/PMC1351026/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10189543/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Diabetes%20Care&title=The%20Diabetes%20Prevention%20Program:%20Design%20and%20methods%20for%20a%20clinical%20trial%20in%20the%20prevention%20of%20type%202%20diabetes&volume=22&publication_year=1999&pages=623-634&pmid=10189543&doi=10.2337/diacare.22.4.623&)

8. de Bakker PI, Yelensky R, Pe'er I, Gabriel SB, Daly MJ, Altshuler D. Efficiency and power in genetic association studies. Nat Genet 2005;37:1217–1223  [DOI](https://doi.org/10.1038/ng1669) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16244653/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Efficiency%20and%20power%20in%20genetic%20association%20studies&author=PI%20de%20Bakker&author=R%20Yelensky&author=I%20Pe'er&author=SB%20Gabriel&author=MJ%20Daly&volume=37&publication_year=2005&pages=1217-1223&pmid=16244653&doi=10.1038/ng1669&)

9. Diabetes Genetics Initiative of Broad Institute of Harvard and MIT, Lund University, and Novartis Institutes of BioMedical Research Saxena R, Voight BF, Lyssenko V, Burtt NP, de Bakker PI, Chen H, Roix JJ, Kathiresan S, Hirschhorn JN, Daly MJ, Hughes TE, Groop L, Altshuler D, Almgren P, Florez JC, Meyer J, Ardlie K, Bengtsson Boström K, Isomaa B, Lettre G, Lindblad U, Lyon HN, Melander O, Newton-Cheh C, Nilsson P, Orho-Melander M, Råstam L, Speliotes EK, Taskinen MR, Tuomi T, Guiducci C, Berglund A, Carlson J, Gianniny L, Hackett R, Hall L, Holmkvist J, Laurila E, Sjögren M, Sterner M, Surti A, Svensson M, Svensson M, Tewhey R, Blumenstiel B, Parkin M, Defelice M, Barry R, Brodeur W, Camarata J, Chia N, Fava M, Gibbons J, Handsaker B, Healy C, Nguyen K, Gates C, Sougnez C, Gage D, Nizzari M, Gabriel SB, Chirn GW, Ma Q, Parikh H, Richardson D, Ricke D, Purcell S. Genome-wide association analysis identifies loci for type 2 diabetes and triglyceride levels. Science 2007;316:1331–1336  [DOI](https://doi.org/10.1126/science.1142358) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17463246/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=Genome-wide%20association%20analysis%20identifies%20loci%20for%20type%202%20diabetes%20and%20triglyceride%20levels&author=R%20Saxena&author=BF%20Voight&author=V%20Lyssenko&author=NP%20Burtt&author=PI%20de%20Bakker&volume=316&publication_year=2007&pages=1331-1336&pmid=17463246&doi=10.1126/science.1142358&)

10. Zeggini E, Scott LJ, Saxena R, Voight BF, Marchini JL, Hu T, de Bakker PI, Abecasis GR, Almgren P, Andersen G, Ardlie K, Boström KB, Bergman RN, Bonnycastle LL, Borch-Johnsen K, Burtt NP, Chen H, Chines PS, Daly MJ, Deodhar P, Ding CJ, Doney AS, Duren WL, Elliott KS, Erdos MR, Frayling TM, Freathy RM, Gianniny L, Grallert H, Grarup N, Groves CJ, Guiducci C, Hansen T, Herder C, Hitman GA, Hughes TE, Isomaa B, Jackson AU, Jørgensen T, Kong A, Kubalanza K, Kuruvilla FG, Kuusisto J, Langenberg C, Lango H, Lauritzen T, Li Y, Lindgren CM, Lyssenko V, Marvelle AF, Meisinger C, Midthjell K, Mohlke KL, Morken MA, Morris AD, Narisu N, Nilsson P, Owen KR, Palmer CN, Payne F, Perry JR, Pettersen E, Platou C, Prokopenko I, Qi L, Qin L, Rayner NW, Rees M, Roix JJ, Sandbaek A, Shields B, Sjögren M, Steinthorsdottir V, Stringham HM, Swift AJ, Thorleifsson G, Thorsteinsdottir U, Timpson NJ, Tuomi T, Tuomilehto J, Walker M, Watanabe RM, Weedon MN, Willer CJ. Wellcome Trust Case Control Consortium Illig T, Hveem K, Hu FB, Laakso M, Stefansson K, Pedersen O, Wareham NJ, Barroso I, Hattersley AT, Collins FS, Groop L, McCarthy MI, Boehnke M, Altshuler D. Meta-analysis of genome-wide association data and large-scale replication identifies additional susceptibility loci for type 2 diabetes. Nat Genet 2008;40:638–645  [DOI](https://doi.org/10.1038/ng.120) | [PMC free article](/articles/PMC2672416/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18372903/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Meta-analysis%20of%20genome-wide%20association%20data%20and%20large-scale%20replication%20identifies%20additional%20susceptibility%20loci%20for%20type%202%20diabetes&author=E%20Zeggini&author=LJ%20Scott&author=R%20Saxena&author=BF%20Voight&author=JL%20Marchini&volume=40&publication_year=2008&pages=638-645&pmid=18372903&doi=10.1038/ng.120&)

11. Florez JC, Manning AK, Dupuis J, McAteer J, Irenze K, Gianniny L, Mirel DB, Fox CS, Cupples LA, Meigs JB. A 100K genome-wide association scan for diabetes and related traits in the Framingham Heart Study: replication and integration with other genome-wide datasets. Diabetes 2007;56:3063–3074  [DOI](https://doi.org/10.2337/db07-0451) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17848626/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Diabetes&title=A%20100K%20genome-wide%20association%20scan%20for%20diabetes%20and%20related%20traits%20in%20the%20Framingham%20Heart%20Study:%20replication%20and%20integration%20with%20other%20genome-wide%20datasets&author=JC%20Florez&author=AK%20Manning&author=J%20Dupuis&author=J%20McAteer&author=K%20Irenze&volume=56&publication_year=2007&pages=3063-3074&pmid=17848626&doi=10.2337/db07-0451&)

12. Hanson RL, Bogardus C, Duggan D, Kobes S, Knowlton M, Infante AM, Marovich L, Benitez D, Baier LJ, Knowler WC. A search for variants associated with young-onset type 2 diabetes in American Indians in a 100K genotyping array. Diabetes 2007;56:3045–3052  [DOI](https://doi.org/10.2337/db07-0462) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17846125/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Diabetes&title=A%20search%20for%20variants%20associated%20with%20young-onset%20type%202%20diabetes%20in%20American%20Indians%20in%20a%20100K%20genotyping%20array&author=RL%20Hanson&author=C%20Bogardus&author=D%20Duggan&author=S%20Kobes&author=M%20Knowlton&volume=56&publication_year=2007&pages=3045-3052&pmid=17846125&doi=10.2337/db07-0462&)

13. Rampersaud E, Damcott CM, Fu M, Shen H, McArdle P, Shi X, Shelton J, Yin J, Chang YP, Ott SH, Zhang L, Zhao Y, Mitchell BD, O'Connell J, Shuldiner AR. Identification of novel candidate genes for type 2 diabetes from a genome-wide association scan in the Old Order Amish: evidence for replication from diabetes-related quantitative traits and from independent populations. Diabetes 2007;56:3053–3062  [DOI](https://doi.org/10.2337/db07-0457) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17846126/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Diabetes&title=Identification%20of%20novel%20candidate%20genes%20for%20type%202%20diabetes%20from%20a%20genome-wide%20association%20scan%20in%20the%20Old%20Order%20Amish:%20evidence%20for%20replication%20from%20diabetes-related%20quantitative%20traits%20and%20from%20independent%20populations&author=E%20Rampersaud&author=CM%20Damcott&author=M%20Fu&author=H%20Shen&author=P%20McArdle&volume=56&publication_year=2007&pages=3053-3062&pmid=17846126&doi=10.2337/db07-0457&)

14. Willer CJ, Speliotes EK, Loos RJ, Li S, Lindgren CM, Heid IM, Berndt SI, Elliott AL, Jackson AU, Lamina C, Lettre G, Lim N, Lyon HN, McCarroll SA, Papadakis K, Qi L, Randall JC, Roccasecca RM, Sanna S, Scheet P, Weedon MN, Wheeler E, Zhao JH, Jacobs LC, Prokopenko I, Soranzo N, Tanaka T, Timpson NJ, Almgren P, Bennett A, Bergman RN, Bingham SA, Bonnycastle LL, Brown M, Burtt NP, Chines P, Coin L, Collins FS, Connell JM, Cooper C, Smith GD, Dennison EM, Deodhar P, Elliott P, Erdos MR, Estrada K, Evans DM, Gianniny L, Gieger C, Gillson CJ, Guiducci C, Hackett R, Hadley D, Hall AS, Havulinna AS, Hebebrand J, Hofman A, Isomaa B, Jacobs KB, Johnson T, Jousilahti P, Jovanovic Z, Khaw KT, Kraft P, Kuokkanen M, Kuusisto J, Laitinen J, Lakatta EG, Luan J, Luben RN, Mangino M, McArdle WL, Meitinger T, Mulas A, Munroe PB, Narisu N, Ness AR, Northstone K, O'Rahilly S, Purmann C, Rees MG, Ridderstråle M, Ring SM, Rivadeneira F, Ruokonen A, Sandhu MS, Saramies J, Scott LJ, Scuteri A, Silander K, Sims MA, Song K, Stephens J, Stevens S, Stringham HM, Tung YC, Valle TT, Van Duijn CM, Vimaleswaran KS, Vollenweider P, Waeber G, Wallace C, Watanabe RM, Waterworth DM, Watkins N. Wellcome Trust Case Control Consortium Witteman JC, Zeggini E, Zhai G, Zillikens MC, Altshuler D, Caulfield MJ, Chanock SJ, Farooqi IS, Ferrucci L, Guralnik JM, Hattersley AT, Hu FB, Jarvelin MR, Laakso M, Mooser V, Ong KK, Ouwehand WH, Salomaa V, Samani NJ, Spector TD, Tuomi T, Tuomilehto J, Uda M, Uitterlinden AG, Wareham NJ, Deloukas P, Frayling TM, Groop LC, Hayes RB, Hunter DJ, Mohlke KL, Peltonen L, Schlessinger D, Strachan DP, Wichmann HE, McCarthy MI, Boehnke M, Barroso I, Abecasis GR, Hirschhorn JN. Genetic Investigation of ANthropometric Traits Consortium Six new loci associated with body mass index highlight a neuronal influence on body weight regulation. Nat Genet 2009;41:25–34  [DOI](https://doi.org/10.1038/ng.287) | [PMC free article](/articles/PMC2695662/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19079261/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Six%20new%20loci%20associated%20with%20body%20mass%20index%20highlight%20a%20neuronal%20influence%20on%20body%20weight%20regulation&author=CJ%20Willer&author=EK%20Speliotes&author=RJ%20Loos&author=S%20Li&author=CM%20Lindgren&volume=41&publication_year=2009&pages=25-34&pmid=19079261&doi=10.1038/ng.287&)

15. Thorleifsson G, Walters GB, Gudbjartsson DF, Steinthorsdottir V, Sulem P, Helgadottir A, Styrkarsdottir U, Gretarsdottir S, Thorlacius S, Jonsdottir I, Jonsdottir T, Olafsdottir EJ, Olafsdottir GH, Jonsson T, Jonsson F, Borch-Johnsen K, Hansen T, Andersen G, Jorgensen T, Lauritzen T, Aben KK, Verbeek AL, Roeleveld N, Kampman E, Yanek LR, Becker LC, Tryggvadottir L, Rafnar T, Becker DM, Gulcher J, Kiemeney LA, Pedersen O, Kong A, Thorsteinsdottir U, Stefansson K. Genome-wide association yields new sequence variants at seven loci that associate with measures of obesity. Nat Genet 2009;41:18–24  [DOI](https://doi.org/10.1038/ng.274) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19079260/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Genome-wide%20association%20yields%20new%20sequence%20variants%20at%20seven%20loci%20that%20associate%20with%20measures%20of%20obesity&author=G%20Thorleifsson&author=GB%20Walters&author=DF%20Gudbjartsson&author=V%20Steinthorsdottir&author=P%20Sulem&volume=41&publication_year=2009&pages=18-24&pmid=19079260&doi=10.1038/ng.274&)

16. Kathiresan S, Melander O, Guiducci C, Surti A, Burtt NP, Rieder MJ, Cooper GM, Roos C, Voight BF, Havulinna AS, Wahlstrand B, Hedner T, Corella D, Tai ES, Ordovas JM, Berglund G, Vartiainen E, Jousilahti P, Hedblad B, Taskinen MR, Newton-Cheh C, Salomaa V, Peltonen L, Groop L, Altshuler DM, Orho-Melander M. Six new loci associated with blood low-density lipoprotein cholesterol, high-density lipoprotein cholesterol or triglycerides in humans. Nat Genet 2008;40:189–197  [DOI](https://doi.org/10.1038/ng.75) | [PMC free article](/articles/PMC2682493/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18193044/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Six%20new%20loci%20associated%20with%20blood%20low-density%20lipoprotein%20cholesterol,%20high-density%20lipoprotein%20cholesterol%20or%20triglycerides%20in%20humans&author=S%20Kathiresan&author=O%20Melander&author=C%20Guiducci&author=A%20Surti&author=NP%20Burtt&volume=40&publication_year=2008&pages=189-197&pmid=18193044&doi=10.1038/ng.75&)

17. Willer CJ, Sanna S, Jackson AU, Scuteri A, Bonnycastle LL, Clarke R, Heath SC, Timpson NJ, Najjar SS, Stringham HM, Strait J, Duren WL, Maschio A, Busonero F, Mulas A, Albai G, Swift AJ, Morken MA, Narisu N, Bennett D, Parish S, Shen H, Galan P, Meneton P, Hercberg S, Zelenika D, Chen WM, Li Y, Scott LJ, Scheet PA, Sundvall J, Watanabe RM, Nagaraja R, Ebrahim S, Lawlor DA, Ben-Shlomo Y, Davey-Smith G, Shuldiner AR, Collins R, Bergman RN, Uda M, Tuomilehto J, Cao A, Collins FS, Lakatta E, Lathrop GM, Boehnke M, Schlessinger D, Mohlke KL, Abecasis GR. Newly identified loci that influence lipid concentrations and risk of coronary artery disease. Nat Genet 2008;40:161–169  [DOI](https://doi.org/10.1038/ng.76) | [PMC free article](/articles/PMC5206900/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18193043/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Newly%20identified%20loci%20that%20influence%20lipid%20concentrations%20and%20risk%20of%20coronary%20artery%20disease&author=CJ%20Willer&author=S%20Sanna&author=AU%20Jackson&author=A%20Scuteri&author=LL%20Bonnycastle&volume=40&publication_year=2008&pages=161-169&pmid=18193043&doi=10.1038/ng.76&)

18. Kathiresan S, Willer CJ, Peloso GM, Demissie S, Musunuru K, Schadt EE, Kaplan L, Bennett D, Li Y, Tanaka T, Voight BF, Bonnycastle LL, Jackson AU, Crawford G, Surti A, Guiducci C, Burtt NP, Parish S, Clarke R, Zelenika D, Kubalanza KA, Morken MA, Scott LJ, Stringham HM, Galan P, Swift AJ, Kuusisto J, Bergman RN, Sundvall J, Laakso M, Ferrucci L, Scheet P, Sanna S, Uda M, Yang Q, Lunetta KL, Dupuis J, de Bakker PI, O'Donnell CJ, Chambers JC, Kooner JS, Hercberg S, Meneton P, Lakatta EG, Scuteri A, Schlessinger D, Tuomilehto J, Collins FS, Groop L, Altshuler D, Collins R, Lathrop GM, Melander O, Salomaa V, Peltonen L, Orho-Melander M, Ordovas JM, Boehnke M, Abecasis GR, Mohlke KL, Cupples LA. Common variants at 30 loci contribute to polygenic dyslipidemia. Nat Genet 2009;41:56–65  [DOI](https://doi.org/10.1038/ng.291) | [PMC free article](/articles/PMC2881676/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19060906/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Common%20variants%20at%2030%20loci%20contribute%20to%20polygenic%20dyslipidemia&author=S%20Kathiresan&author=CJ%20Willer&author=GM%20Peloso&author=S%20Demissie&author=K%20Musunuru&volume=41&publication_year=2009&pages=56-65&pmid=19060906&doi=10.1038/ng.291&)

19. Smith MW, Patterson N, Lautenberger JA, Truelove AL, McDonald GJ, Waliszewska A, Kessing BD, Malasky MJ, Scafe C, Le E, De Jager PL, Mignault AA, Yi Z, De The G, Essex M, Sankale JL, Moore JH, Poku K, Phair JP, Goedert JJ, Vlahov D, Williams SM, Tishkoff SA, Winkler CA, De La Vega FM, Woodage T, Sninsky JJ, Hafler DA, Altshuler D, Gilbert DA, O'Brien SJ, Reich D. A high-density admixture map for disease gene discovery in African Americans. Am J Hum Genet 2004;74:1001–1013  [DOI](https://doi.org/10.1086/420856) | [PMC free article](/articles/PMC1181963/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15088270/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=A%20high-density%20admixture%20map%20for%20disease%20gene%20discovery%20in%20African%20Americans&author=MW%20Smith&author=N%20Patterson&author=JA%20Lautenberger&author=AL%20Truelove&author=GJ%20McDonald&volume=74&publication_year=2004&pages=1001-1013&pmid=15088270&doi=10.1086/420856&)

20. Price AL, Patterson N, Yu F, Cox DR, Waliszewska A, McDonald GJ, Tandon A, Schirmer C, Neubauer J, Bedoya G, Duque C, Villegas A, Bortolini MC, Salzano FM, Gallo C, Mazzotti G, Tello-Ruiz M, Riba L, Aguilar-Salinas CA, Canizales-Quinteros S, Menjivar M, Klitz W, Henderson B, Haiman CA, Winkler C, Tusie-Luna T, Ruiz-Linares A, Reich D. A genomewide admixture map for Latino populations. Am J Hum Genet 2007;80:1024–1036  [DOI](https://doi.org/10.1086/518313) | [PMC free article](/articles/PMC1867092/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17503322/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=A%20genomewide%20admixture%20map%20for%20Latino%20populations&author=AL%20Price&author=N%20Patterson&author=F%20Yu&author=DR%20Cox&author=A%20Waliszewska&volume=80&publication_year=2007&pages=1024-1036&pmid=17503322&doi=10.1086/518313&)

21. Li Y, Abecasis GR. Mach 1.0: Rapid haplotype reconstruction and missing genotype inference. Am J Hum Genet S 2006;79:2290  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet%20S&title=Mach%201.0:%20Rapid%20haplotype%20reconstruction%20and%20missing%20genotype%20inference&author=Y%20Li&author=GR%20Abecasis&volume=79&publication_year=2006&pages=2290&)

22. Pritchard JK, Stephens M, Donnelly P. Inference of population structure using multilocus genotype data. Genetics 2000;155:945–959  [DOI](https://doi.org/10.1093/genetics/155.2.945) | [PMC free article](/articles/PMC1461096/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10835412/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genetics&title=Inference%20of%20population%20structure%20using%20multilocus%20genotype%20data&author=JK%20Pritchard&author=M%20Stephens&author=P%20Donnelly&volume=155&publication_year=2000&pages=945-959&pmid=10835412&doi=10.1093/genetics/155.2.945&)

23. Dudoit S, Shaffer JP, Boldrick JC. Multiple hypothesis testing in microarray experiments. Statistical Science 2003;18:71–103  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Statistical%20Science&title=Multiple%20hypothesis%20testing%20in%20microarray%20experiments&author=S%20Dudoit&author=JP%20Shaffer&author=JC%20Boldrick&volume=18&publication_year=2003&pages=71-103&)

24. Potter DM. Ominibus permutation tests of the association of an ensemble of genetic markers with disease in case-control studies. Genet Epidemiol 2006;30:438–446  [DOI](https://doi.org/10.1002/gepi.20155) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16671109/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genet%20Epidemiol&title=Ominibus%20permutation%20tests%20of%20the%20association%20of%20an%20ensemble%20of%20genetic%20markers%20with%20disease%20in%20case-control%20studies&author=DM%20Potter&volume=30&publication_year=2006&pages=438-446&pmid=16671109&doi=10.1002/gepi.20155&)

25. Benjamini Y, Hochberg Y. Controlling the false discovery rate: a practical and powerful approach to multiple testing. J R Stat Soc 1995;57:289–300  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20R%20Stat%20Soc&title=Controlling%20the%20false%20discovery%20rate:%20a%20practical%20and%20powerful%20approach%20to%20multiple%20testing&author=Y%20Benjamini&author=Y%20Hochberg&volume=57&publication_year=1995&pages=289-300&)

26. Hattersley AT, McCarthy MI. What makes a good genetic association study? Lancet 2005;366:1315–1323  [DOI](https://doi.org/10.1016/S0140-6736(05)67531-9) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16214603/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=What%20makes%20a%20good%20genetic%20association%20study?&author=AT%20Hattersley&author=MI%20McCarthy&volume=366&publication_year=2005&pages=1315-1323&pmid=16214603&doi=10.1016/S0140-6736(05)67531-9&)

27. Pe'er I, Yelensky R, Altshuler D, Daly MJ. Estimation of the multiple testing burden for genomewide association studies of nearly all common variants. Genet Epidemiol 2008;32:381–385  [DOI](https://doi.org/10.1002/gepi.20303) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18348202/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genet%20Epidemiol&title=Estimation%20of%20the%20multiple%20testing%20burden%20for%20genomewide%20association%20studies%20of%20nearly%20all%20common%20variants&author=I%20Pe'er&author=R%20Yelensky&author=D%20Altshuler&author=MJ%20Daly&volume=32&publication_year=2008&pages=381-385&pmid=18348202&doi=10.1002/gepi.20303&)

28. Moore AF, Jablonski KA, McAteer JB, Saxena R, Pollin TI, Franks PW, Hanson RL, Shuldiner AR, Knowler WC, Altshuler D, Florez JC. Diabetes Prevention Program Research Group Extension of type 2 diabetes genome-wide association scan results in the Diabetes Prevention Program. Diabetes 2008;57:2503–2510  [DOI](https://doi.org/10.2337/db08-0284) | [PMC free article](/articles/PMC2518503/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18544707/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Diabetes&title=Extension%20of%20type%202%20diabetes%20genome-wide%20association%20scan%20results%20in%20the%20Diabetes%20Prevention%20Program&author=AF%20Moore&author=KA%20Jablonski&author=JB%20McAteer&author=R%20Saxena&author=TI%20Pollin&volume=57&publication_year=2008&pages=2503-2510&pmid=18544707&doi=10.2337/db08-0284&)

29. Turner RC, Cull CA, Frighi V, Holman RR. Glycemic control with diet, sulfonylurea, metformin, or insulin in patients with type 2 diabetes mellitus: progressive requirement for multiple therapies (UKPDS 49): UK Prospective Diabetes Study (UKPDS) Group. JAMA 1999;281:2005–2012  [DOI](https://doi.org/10.1001/jama.281.21.2005) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10359389/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA&title=Glycemic%20control%20with%20diet,%20sulfonylurea,%20metformin,%20or%20insulin%20in%20patients%20with%20type%202%20diabetes%20mellitus:%20progressive%20requirement%20for%20multiple%20therapies%20(UKPDS%2049):%20UK%20Prospective%20Diabetes%20Study%20(UKPDS)%20Group&author=RC%20Turner&author=CA%20Cull&author=V%20Frighi&author=RR%20Holman&volume=281&publication_year=1999&pages=2005-2012&pmid=10359389&doi=10.1001/jama.281.21.2005&)

30. Kahn SE, Haffner SM, Heise MA, Herman WH, Holman RR, Jones NP, Kravitz BG, Lachin JM, O'Neill MC, Zinman B, Viberti G. ADOPT Study Group Glycemic durability of rosiglitazone, metformin, or glyburide monotherapy. N Engl J Med 2006;355:2427–2443  [DOI](https://doi.org/10.1056/NEJMoa066224) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17145742/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Glycemic%20durability%20of%20rosiglitazone,%20metformin,%20or%20glyburide%20monotherapy&author=SE%20Kahn&author=SM%20Haffner&author=MA%20Heise&author=WH%20Herman&author=RR%20Holman&volume=355&publication_year=2006&pages=2427-2443&pmid=17145742&doi=10.1056/NEJMoa066224&)

31. Becker ML, Visser LE, van Schaik RH, Hofman A, Uitterlinden AG, Stricker BH. Genetic variation in the organic cation transporter 1 is associated with metformin response in patients with diabetes mellitus. Pharmacogenomics J 2009;9:242–247  [DOI](https://doi.org/10.1038/tpj.2009.15) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19381165/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=Genetic%20variation%20in%20the%20organic%20cation%20transporter%201%20is%20associated%20with%20metformin%20response%20in%20patients%20with%20diabetes%20mellitus&author=ML%20Becker&author=LE%20Visser&author=RH%20van%20Schaik&author=A%20Hofman&author=AG%20Uitterlinden&volume=9&publication_year=2009&pages=242-247&pmid=19381165&doi=10.1038/tpj.2009.15&)

32. Song IS, Shin HJ, Shim EJ, Jung IS, Kim WY, Shon JH, Shin JG. Genetic variants of the organic cation transporter 2 influence the disposition of metformin. Clin Pharmacol Ther 2008;84:559–562  [DOI](https://doi.org/10.1038/clpt.2008.61) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18401339/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Genetic%20variants%20of%20the%20organic%20cation%20transporter%202%20influence%20the%20disposition%20of%20metformin&author=IS%20Song&author=HJ%20Shin&author=EJ%20Shim&author=IS%20Jung&author=WY%20Kim&volume=84&publication_year=2008&pages=559-562&pmid=18401339&doi=10.1038/clpt.2008.61&)

33. Holmes BF, Sparling DP, Olson AL, Winder WW, Dohm GL. Regulation of muscle GLUT4 enhancer factor and myocyte enhancer factor 2 by AMP-activated protein kinase. Am J Physiol Endocrinol Metab 2005;289:E1071–E1076  [DOI](https://doi.org/10.1152/ajpendo.00606.2004) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16105857/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Physiol%20Endocrinol%20Metab&title=Regulation%20of%20muscle%20GLUT4%20enhancer%20factor%20and%20myocyte%20enhancer%20factor%202%20by%20AMP-activated%20protein%20kinase&author=BF%20Holmes&author=DP%20Sparling&author=AL%20Olson&author=WW%20Winder&author=GL%20Dohm&volume=289&publication_year=2005&pages=E1071-E1076&pmid=16105857&doi=10.1152/ajpendo.00606.2004&)

34. Rhee J, Inoue Y, Yoon JC, Puigserver P, Fan M, Gonzalez FJ, Spiegelman BM. Regulation of hepatic fasting response by PPARγ coactivator-1α (PGC-1): requirement for hepatocyte nuclear factor 4α in gluconeogenesis. Proc Natl Acad Sci U S A 2003;100:4012–4017  [DOI](https://doi.org/10.1073/pnas.0730870100) | [PMC free article](/articles/PMC153039/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12651943/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&title=Regulation%20of%20hepatic%20fasting%20response%20by%20PPAR%CE%B3%20coactivator-1%CE%B1%20(PGC-1):%20requirement%20for%20hepatocyte%20nuclear%20factor%204%CE%B1%20in%20gluconeogenesis&author=J%20Rhee&author=Y%20Inoue&author=JC%20Yoon&author=P%20Puigserver&author=M%20Fan&volume=100&publication_year=2003&pages=4012-4017&pmid=12651943&doi=10.1073/pnas.0730870100&)

35. Florez JC, Jablonski KA, Kahn SE, Franks PW, Dabelea D, Hamman RF, Knowler WC, Nathan DM, Altshuler D. Type 2 diabetes-associated missense polymorphisms KCNJ11 E23K and ABCC8 A1369S influence progression to diabetes and response to interventions in the Diabetes Prevention Program. Diabetes 2007;56:531–536  [DOI](https://doi.org/10.2337/db06-0966) | [PMC free article](/articles/PMC2267937/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17259403/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Diabetes&title=Type%202%20diabetes-associated%20missense%20polymorphisms%20KCNJ11%20E23K%20and%20ABCC8%20A1369S%20influence%20progression%20to%20diabetes%20and%20response%20to%20interventions%20in%20the%20Diabetes%20Prevention%20Program&author=JC%20Florez&author=KA%20Jablonski&author=SE%20Kahn&author=PW%20Franks&author=D%20Dabelea&volume=56&publication_year=2007&pages=531-536&pmid=17259403&doi=10.2337/db06-0966&)
