# Genetic risk signatures of opioid-induced respiratory depression following pediatric tonsillectomy

## Metadata
**Authors:** Jacek Biesiada, Vidya Chidambaran, Michael Wagner, Xue Zhang, Lisa J Martin, Jaroslaw Meller, Senthilkumar Sadhasivam
**Journal:** Pharmacogenomics
**Date:** 2014 Nov
**DOI:** [10.2217/pgs.14.137](https://doi.org/10.2217/pgs.14.137)
**PMID:** 25493568
**PMCID:** PMC4287371
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4287371/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC4287371/pdf/nihms-652641.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC4287371/pdf/nihms-652641.pdf)

## Abstract

**Background:** 
Respiratory depression is a clinically and economically important but preventable complication of opioids. Genetic factors can help identify patients with high risk for respiratory depression.

**Methods:** 
In this prospective genotype blinded clinical study, we evaluated the effect of a panel of variants in candidate genes on opioid-related respiratory depression in 347 children following tonsillectomy.

**Results:** 
Using unsupervised hierarchical clustering and a combination of candidate genotypes and clinical variables, we identified several distinct clusters of patients at high risk (36–38%) and low risk (10–17%) of respiratory depression; the relative risk of respiratory depression for high versus low risk clusters was 2.1–3.8 (p = 0.003).

**Conclusion:** 
Genetic risk predictions (genetic signatures) along with clinical risk factors effectively identify children at higher and lower risks of opioid-induced respiratory depression. Genetic signatures of respiratory depression offer strategies for improved clinical decision support to guide clinicians to balance the risks of opioid adverse effects with analgesia.

Keywords: children, depression, genetic signature, morphine, opioid, pediatric tonsillectomy, pharmacogenetics, respiratory, surgical pain

### Background

Respiratory depression is a clinically and economically important but preventable complication of opioids. Genetic factors can help identify patients with high risk for respiratory depression.

### Methods

In this prospective genotype blinded clinical study, we evaluated the effect of a panel of variants in candidate genes on opioid-related respiratory depression in 347 children following tonsillectomy.

### Results

Using unsupervised hierarchical clustering and a combination of candidate genotypes and clinical variables, we identified several distinct clusters of patients at high risk (36–38%) and low risk (10–17%) of respiratory depression; the relative risk of respiratory depression for high versus low risk clusters was 2.1–3.8 (p = 0.003).

### Conclusion

Genetic risk predictions (genetic signatures) along with clinical risk factors effectively identify children at higher and lower risks of opioid-induced respiratory depression. Genetic signatures of respiratory depression offer strategies for improved clinical decision support to guide clinicians to balance the risks of opioid adverse effects with analgesia.

## Background

Opioids remain one of the most commonly used analgesics to manage surgical pain in both adult patients and children. However, due to narrow therapeutic indices, large interindividual variations in opioid responses and significant risk of adverse effects, including opioid induced respiratory depression, postoperative pain management with opioids remains challenging, especially in children. Respiratory depression (RD) is a serious, potentially life threatening, yet preventable complication of opioids that impacts overall clinical (e.g., surgical recovery) and economic outcomes (e.g., hospital costs and length of stay) [[1](#R1)]. Genetic factors that may contribute to variability in opioid-induced RD, as well as other adverse effects, such as nausea and analgesia, have been implicated in previous studies [[2](#R2),[3](#R3)].

Previous studies implicated a number of candidate genes that were found to contribute to the risk of opioid-related adverse effects [[4](#R4)–[15](#R15)]. Major and common concerns with genetic association studies are underpowered studies looking at one or two genetic variants with overestimation of risk (‘winner's curse’), heterogeneous population and lack of external replication of findings. Therefore, further systematic studies of important genetic risk factors (and their interactions) for increased opioid-induced RD are needed to elucidate mechanistic aspects of opioid-related adverse effects, and to provide a basis for clinical decision support in the context of perioperative pain management with opioids. This study focuses on multiple genetic components and their associations with clinical responses to morphine in a cohort of healthy children undergoing outpatient tonsillectomy.

In this work, specific genetic variants of several genes previously implicated with opioid effects and adverse effects are combined together with clinical risk factors of opioid-related RD such as obstructive sleep apnea (OSA), into multivariate risk signatures that can be used to predict pediatric patients with a higher (or lower) risk of RD. Following an approach used to define gene-expression signatures of cancer subtypes and diseases progression [[16](#R16)], and with the goal of accounting for interactions between genes and stratification by clinical risk variables, these multivariate risk signatures were derived using unsupervised clustering and further assessed using supervised machine learning approaches. The resulting composite risk signatures aim to provide insights into signaling pathways underlying opioid-induced RD, and to become a basis for improved clinical decision rules.

## Methods

### Study design & setting

A prospective clinical observational study in 273 children undergoing tonsillectomy was conducted with standard perioperative anesthetic and surgical practices as well as standard postoperative nursing care. The study was approved by the institutional review board at Cincinnati Children's Hospital Medical Center and written informed consents and assents (from children >11 years) were obtained from all parents and participating children. Though children of all races were enrolled in this study, given the demographics of patients for tonsillectomy at our hospital, the vast majority of children were white.

### Participants

Children aged 6–15 years undergoing elective outpatient tonsillectomy were eligible for the study and recruited on the day of surgery. Race was self-reported by either the parent or child; self-report is well accepted for identifying race [[17](#R17),[18](#R18)]. Sample inclusion criteria were children designated to have an American Society of Anesthesiologists (ASA) physical status 1 or 2 scheduled for tonsillectomy because of recurrent tonsillitis, adenotonsillar hypertrophy or OSA. Clinical criteria for OSA designation included sleep disordered breathing with a history of snoring and either sleep pauses lasting more than 10 s or daytime drowsiness. Children were excluded from the study if they or their parents were non-English speaking. Children who were allergic to morphine or who had developmental delay, liver or renal disease, or preoperative pain requiring opioid analgesics (e.g., chronic tonsillitis) were excluded.

### Standard care & study procedures

All participants received standard perioperative care. All clinical care providers and research staff were blinded to genotypes of subjects, as genetic analyses were carried out after clinical care and clinical data collection.

Anesthesia was induced using sevoflurane followed by a propofol (2 mg/kg) bolus to facilitate endotracheal intubation. Anesthesia was maintained with sevoflurane without the use of neuromuscular blockade. Patients received morphine prior to surgical incision. Children with OSA history received 0.1 mg/kg morphine while those without OSA diagnosis received 0.2 mg/kg. Tonsillectomy was carried out using electrocautery surgical technique in all children. If there were any signs suggestive of pain (clinically significant increase in heart rate and blood pressure) following surgical incision and cauterization, the clinical anesthesia team provided additional morphine at 0.05 mg/kg increments intraoperatively as necessary. All children received prophylactic ondansetron (0.1 mg/kg) and dexamethasone (0.1 mg/kg) intraoperatively. Signifi-cant postoperative pain measured with facial expression; leg movement; activity; cry; and consolability (FLACC) pain score [[19](#R19)] of ≥4/10 was managed in the postoperative anesthesia care unit (PACU) with rescue doses of morphine (0.05 mg/kg increments).

### Clinical outcome measures

Metrics for analgesic effectiveness and opioid-related adverse effects were recorded for each participant. For the metrics for opioid-related adverse effects, we focused on clinically and economically important outcome: clinical RD. In our study, we defined clinical RD as a persistent (>1 min) oxygen desaturation <90% or respiratory rate <8 breaths per minute or oxygen desaturation <94% along with respiratory rate <10 per min requiring supplemental oxygen to maintain SpO_2_ >94% in the absence of clinically obvious upper airway obstruction [[20](#R20)]. Metrics for analgesic effectiveness included maximum postoperative pain scores, intravenous analgesic interventions in PACU (yes/no) and postoperative morphine doses (mg/kg) in patients requiring analgesic interventions. The maximum postoperative pain score for each child was generated from postoperative pain scores assessed in PACU at multiple time points using a 0–10 FLACC pain scale [[19](#R19)].

### Genotyping

Blood was collected from each participant in the operating room under anesthesia and the DNA was isolated the same day and stored at −20°C. Genotyping was performed after clinical data was entered on all study participants. The TaqMan® allelic discrimination system assays and Genotyper Software Version 1·0·1 (Life Technologies, Applied Biosystems, CA, USA) were used to analyze and report polymorphisms of candidate genes which were reported to significantly influence responses to drugs including opioids [[15](#R15)]. A panel of 42 SNP markers in a set of candidate genes shown to influence opioid effects, compiled based on previous studies [[4](#R4)–[15](#R15)], including *ABCB1* (rs2032582, rs2229109, rs9282564, rs1045642, rs1128503), *COMT* (rs4680, rs6269, rs4633, rs4818), *OPRM1* (rs1799971, rs2234918), *FAAH* (rs2295632, rs4141964, rs3766246, rs324420, rs324419, rs932816), *UGT2B7* (rs7668258, rs7439366), *ADRB2* (rs11958940, rs1432623, rs2400707, rs1042713, rs1042714, rs1042717), *GCH1* (rs4411417, rs3783641, rs8007267, rs752688), *TRPA1* (rs222747, rs13279503, rs1947913, rs13255063, rs3735942, rs3735943, rs1443952, rs1025928, rs1198795), *DRD2 (*rs2734838, rs6279), *ANKK1* (rs1800497), *MC1R* (rs1805005), was used to map variants associated with post operative outcomes in children undergoing tonsillectomy. All SNPs considered here were typed in both discovery and validation cohorts of 273 and 74 patients, respectively, using TaqMan assay, and subsequently assessed for their individual associations with RD.

### Data analyses

An initial 273 children were part of the discovery cohort and a subsequent 74 children were part of the validation cohort. Allelic and genotype association tests, as implemented in Plink [[21](#R21),[22](#R22)] were used to perform analysis of associations of individual SNPs with RD. Hap-Map [[23](#R23),[24](#R24)] and 1000 Genome Project [[25](#R25)] data were used for assessment and comparison of allelic frequencies and patterns of linkage disequilibrium within relevant ancestries [[23](#R23)]. SNP that were found to be in strong linkage disequilibrium (r^2^ > 0.95) were deemed redundant. Software implemented using the open-source R language [[26](#R26),[27](#R27)] was used to perform further testing of associations, including χ^2^ test for composite categorical variables considered. R and TreeView [[28](#R28)] were used to perform hierarchical clustering with Hamming and Jackard distances [[26](#R26),[29](#R29)]. PVCLUST [[30](#R30)] was used to perform bootstrapping for the assessment of the stability of clustering patterns. R (CTC package) was also used to derive and assess decision rules and k-NN classifiers [[31](#R31)]. These supervised approaches were optimized and evaluated using crossvalidation and over-/under-sampling using the SMOTE package [[32](#R32),[33](#R33)]. To train and assess support vector machine-based classifiers, SVMlib [[34](#R34)] was used.

### Genotype encoding

We used recursive partitioning [[35](#R35)] to optimize SNP selection in the context of confounding variables, such as OSA and race that can be used to explicitly stratify the data. Genotypes for biallelic SNPs are often encoded as categorical variable, for example, with values 0, 1, 2 corresponding to the three different genotypes. Standard decision trees, such as Classification And Regression Trees (CART) used here [[35](#R35)], treat each distinct value of a categorical variable as independent. As a result, heterozygous genotypes, for example, CA, and potential risk (or protective) effect of either of the alleles are decoupled from those observed for homozygous genotypes, making it difficult to account for dominant/recessive effects.

To address this problem, we used an alternative encoding, in which every SNP gives rise to two coupled binary variables. For example, if the possible geno types are CC, AC and AA, the first variable (denoted as C_yes) will be set to 1 if allele C is present, or zero otherwise. Thus, C_yes = 1 accounts for CC and AC, whereas C_yes = 0 for AA, respectively. The corresponding A_yes variable enables capturing relationships between AA and AC genotypes. Together, these two variables describe unambiguously the genotype, while facilitating the use of decision trees to select SNPs that stratify the set of patients into distinct subsets with consistent patterns of genotypes. For consistency, we also used the same encoding for unsupervised clustering and risk signatures.

## Results

### Demographics

A total of 347 children were analyzed in this study. The first 273 children were included in the discovery cohort and the next 74 children were included in the validation cohort ([Table 1](#T1) & [Figure 1](#F1)). Participants were primarily Caucasian ancestry while African–American and other races were also recruited for comparison and to account for ancestry-based stratification. Age, weight, race of children and presence of OSA were similar between the discovery and validation cohorts ([Table 1](#T1)). However, in the validation cohort, we observed a higher number of boys than girls, probably due to random recruitment in a relatively small validation cohort.

### Table 1.

|   | Discovery cohort (n = 273) | Validation cohort (n = 74) | p-value |
| --- | --- | --- | --- |
| Age (years), median (IQR) | 8.6 (7.1–11.0) | 8.4 (7.2–11.0) | 0.718 |
|   |  |  |  |
| Weight (kg), median (IQR) | 34.6 (25.9–47.1) | 29.7 (24.5–42.5) | 0.065 |
|   |  |  |  |
| BMI z scores | 0.79 (−0.16–1.67) | 0.45 (−0.33–1.46) | 0.130 |
|   |  |  |  |
| Race, n (%) |   |   | 0.444 |
| Caucasian | 219 (80) | 62 (84) |   |
| African–American | 44 (16) | 8 (11) |   |
| Other | 10 (4) | 4 (5) |   |
|   |  |  |  |
| Sex, n (%) |   |   |   |
| Male | 129 (47) | 47 (64) |   |
|   |  |  |  |
| OSA, n (%) |   |   |   |
| Yes | 131 (48) | 31 (42%) |   |
|   |  |  |  |
| Morphine requirement (mg/kg), median (IQR) |  |   |   |
| Intraoperative morphine requirement | 0.19 (0.17–0.21) | 0.19 (0.16–0.24) | 0.497 |
| Total morphine requirement | 0.24 (0.20–0.30) | 0.24 (0.19–0.31) | 0.761 |

Table 1 Caption: Demographic and clinical characteristics of children in discovery and validation cohorts.

### Figure 1. The consort diagram.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3e25/4287371/0b8d19d58f54/nihms-652641-f0001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4287371_nihms-652641-f0001.jpg)

Illustrates the flow of study participants through this clinical trial. Eligible participants, reasons for exclusions, enrolled and analyzed patients are reported. IRB: Institutional review board.

### SNP signatures identify high risk for opioid-induced RD

The overall number of RD cases in the discovery cohort was 53 among white and black children. The overall pattern of association between morphine dose and RD, as well as several clinical variables, including sex, race, OSA, is summarized in [Figure 2](#F2). As shown in [Figure 2](#F2), the cumulative morphine dose exhibits the strongest correlation with RD.

### Figure 2. Correlation of total morphine dose with race, sex, obstructive sleep apnea and respiratory depression.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3e25/4287371/7a2a5a875bf4/nihms-652641-f0002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4287371_nihms-652641-f0002.jpg)

In the discovery cohort, total morphine dose (mg/kg) was correlated with race, sex (female and male), OSA (no or yes) and RD (no or yes). Children with OSA received overall less total morphine. Higher total morphine dose was associated with RD. Standard box plots are used with horizontal bars corresponding to median values, boxes and whiskers representing two middle quartiles and 1.5 of the interquartile ranges, respectively. Outliers are denoted by circles. AA: African–American; C: Caucasian; O: Other race; OSA: Obstructive sleep apnea; RD: Respiratory depression.

Consistent with literature, high dose of morphine is clearly the most important risk factor for RD. Pain management aims to ameliorate the pain while keeping the dose as low as possible to minimize the risk of RD. Since OSA is a known clinical risk factor for RD, patients with OSA obtain a lower baseline dose, which is subsequently augmented in postoperative pain management if necessary. This results in lower overall morphine dose for the majority of OSA patients, even if some of them required postoperative interventions (additional doses of morphine intraoperatively or in the recovery room). This leads to a complex pattern of interactions between morphine dose, OSA, and race and sex, as OSA is more prevalent in black children and boys.

In order to account for these effects, we considered the risk of RD in patients who required an additional postoperative dose of morphine to ameliorate their pain (DCint). This subset consisted of 170 (or 62.3%) patients, of which 45 had RD (of which 31 were Caucasian). Since 45 out of 53 (84.9%) cases of RD were associated with such interventions, the resulting subset is significantly enriched in RD cases and less imbalanced. In DCint, RD cases constitute 26.5% of the set, as opposed to 19.4% for the entire discovery cohort (DC). In addition, in order to further account for race as confounding factor, we separately analyzed cohorts of patients of Caucasian ancestry (DC-cau; 218 subjects, of which 37 had RD) and DCint-cau (128 subjects, of which 31 had RD).

Thus, stratifying the set of patients by post operative intervention allowed us to better take into account interactions between higher pain sensitivity, race and OSA, while minimizing the effect of the otherwise strongly imbalanced data set. We would like to comment that qualitatively similar results are obtained when stratifying by high morphine dose.

### Unsupervised risk signatures

Using unsupervised clustering, we identified five clusters of patients that were characterized by different risk of RD. These distinct clusters of patients are shown in [Figures 3](#F3) & [4](#F4), and they are associated with statistically significant differences in either enrichment or under-representation of RD cases.

### Figure 3. Gene signatures of varying respiratory depression risks.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3e25/4287371/1a315038dee3/nihms-652641-f0003.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4287371_nihms-652641-f0003.jpg)

Genetic variant signatures identify high respiratory depression risk versus low respiratory depression risk subtypes among children requiring postsurgical opioid interventions in the recovery room. Clustering of children in the discovery cohort was done using a set of candidate SNPs and clinical variables as features; Hamming distance was used to define similarity. The main high and low respiratory depression clusters were subsequently subdivided into five clusters with progressively increasing risk of respiratory depression. Percentage of respiratory depression incidence in each cluster is shown.

### Figure 4. Hierarchical clustering of respiratory depression in children based on underlying genetic variants and clinical factors.

![Figure 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3e25/4287371/397320bab337/nihms-652641-f0004.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4287371_nihms-652641-f0004.jpg)

Overall hierarchical (bi-) clustering of patients (columns) and SNPs/clinical variables (rows) for a subset of patients who needed additional intervention (DCint cohort). In the heat map, green, black and gray are used to indicate different possible values for SNP and other variables. For each cluster, risk is indicated by the color of a box delineating that cluster: high risk by red, intermediate risk by orange and low risk by blue boxes, respectively. The number of respiratory depression cases and the overall number of subjects in each cluster are indicated in each box. SNPs contributing most to the clustering pattern (rs2295632, rs441417, rs3766246, rs324420 and rs1045642) are indicated by a horizontal purple oval. (A) Results for the entire DCint cohort (same as in Figure 3 – note that that the two main clusters are flipped), using SNP genotypes and several clinical variables. (B) Results for a subset of patients of Caucasian descent (DCint-cau), using SNP genotypes only. Please see color figure at www.futuremedicine.com/doi/pdf/10.2217/pgs.14.137

When considering the entire DC cohort, the frequency of RD cases increases gradually from approximately 10% for the ‘low-risk cluster’ to approximately 40% for the ‘high-risk cluster’. Consequently, the relative risk of RD for the highest versus lowest RD risk clusters in [Figure 3](#F3) is approximately 3.8 (p = 0.04). Merging each of the two extreme clusters results in three clusters associated with low, intermediate and high risk of RD, while increasing the number of RD cases in low- (RD frequency of 13.2%) and high-risk (RD frequency of 36.2%) clusters. The relative risk between these two clusters is somewhat lower (2.7), but statistically significant (p = 0.003).

All nonredundant SNPs were included in the analysis at this stage, while SNP genotypes were encoded using two binary variables, as described in the ‘Methods’ section. In addition to genetic variants, OSA, race, sex and morphine dose were also included for clustering to test for their effect on implicit stratification with respect to RD risk. Though these clinical or demographic variables were used in the analysis, they did not show an obvious association with the clusters identified, with the exception of African–American ancestry.

Children of African–American ancestry predominantly (33 out of 39) clustered as a subcluster of the large high-risk cluster with 36% RD frequency among the total of 62 patients in this cluster ([Figures 3](#F3) & [4A](#F4)). The lowest risk cluster (with just 10% of RD cases) consisted of almost exclusively white children (28 out of 29 in the cluster), whereas the highest RD risk cluster (with RD frequency of 38%) consisted of 18 Caucasian and three non-Caucasian children.

The fraction of patients who obtained the highest dose of morphine (>0.3 mg/kg) was actually somewhat lower for the highest risk cluster (28%) in comparison with the lowest risk cluster (25%). The fraction of OSA cases and ratio between males and females were also similar in these two extreme clusters. Thus, clearly factors other than race or morphine dose differentiate between high and low RD risk clusters.

The role of genetic factors in risk of RD is further supported by a qualitatively similar clustering pattern obtained using just SNP genotypes. In order to better elucidate the role of race in RD risk, clustering analysis was also performed separately for the subset of children of Caucasian descent only. In the latter case, using just SNP genotypes without any clinical variables, four clusters with increasing fraction of RD cases can be distinguished ([Figure 4B](#F4)). Exclusion of patients of African–American ancestry dissolves the racially mixed second high-risk cluster, as can be seen by comparing [Figure 4A & B](#F4). On the other hand, the highest risk Caucasian only cluster largely overlaps with the highest risk cluster observed using the entire DC cohort (and in conjunction with clinical and demographic variables). Importantly, an even higher cohesion and stronger association with RD is observed for the highest risk cluster, when limiting the analysis to Caucasians only, while using SNP genotypes only (50 vs 38% of RD cases, respectively).

As can be qualitatively discerned from [Figure 4](#F4) (and which is further quantified in the next section), there are a number of SNPs that contribute to the observed clustering pattern of RD risk. These SNPs include rs441417, rs3766246 and rs2295632 that were found to individually have the strongest association with RD ([Table 2](#T2)), as well as several other SNPs that are either somewhat redundant (and thus contribute to the robustness of the resulting clustering-based risk assignment) or complementary, such as rs324420 and rs1045642.

### Table 2.

| SNP | Gene position | Allele 1 A1 | F A1 (RD) | F A1 (no RD) | Allele 2 A2 | Odds ratio | p-value | Gene |
| --- | --- | --- | --- | --- | --- | --- | --- | --- |
| rs3766246 | 46638258 | T | 0.576 | 0.427 | C | 1.817 | 0.006 | FAAH |
|   |   |   | 0.487 | 0.373 |   | 1.593 | 0.068 |   |
|   |  |  |  |  |  |  |  |  |
| rs2295632 | 46879562 | A | 0.481 | 0.334 | C | 1.848 | 0.005 | FAAH |
|   |   |   | 0.405 | 0.271 |   | 1.837 | 0.020 |   |
|   |  |  |  |  |  |  |  |  |
| rs1947913 | 73089568 | A | 0.330 | 0.232 | T | 1.634 | 0.036 | TRPA1 |
|   |   |   | 0.297 | 0.213 |   | 1.566 | 0.114 |   |
|   |  |  |  |  |  |  |  |  |
| rs4411417 | 54390313 | C | 0.132 | 0.218 | T | 0.545 | 0.047 | GCH1 |
|   |   |   | 0.135 | 0.229 |   | 0.525 | 0.072 |   |

Table 2 Caption: Genetic variants individually associated with respiratory depression.

Thus, the risk of RD could be identified using cluster assignment based on the relevant SNPs, with clinical and demographic variables contributing to more robust predictions. This approach resembles ‘gene signatures’-based clustering of patients for cancer subtype classification [[16](#R16)]. Using centroids of RD risk clusters identified in [Figure 3](#F3) yielded a classification accuracy of approximately 82% for high- versus intermediate- versus low-risk assignment. This result can also be regarded as a form of validation of the stability of clustering, with most of the centroid-based ‘misclassification’ concerning patients from the intermediate cluster.

To further assess the stability of clustering patterns observed, we used subset sampling and bootstrapping. We observed that support for the low- and high-risk clusters obtained using the DC was relatively high, while the intermediate-risk cluster and the overall order of clusters were much less stable. This is also reflected in consistent clustering of high-risk versus less consistent clustering of intermediate- and low-risk patients of Caucasian ancestry, when comparing results with and without inclusion of African–American ancestry patients. Further investigation with more data from a larger sample will be required to better assess whether these intermediate cases form a coherent subtype. It should also be noted the least informative SNPs and other variables used here could be eliminated while still preserving the overall clustering pattern. This is further analyzed in the context of feature selection for supervised classification in the next section.

### Supervised stratification & decision rules for the prediction of RD risk

Analysis of clusters summarized in [Figures 3](#F3) & [4](#F4) reveals that the most of the discriminatory signals come from just a handful of SNPs. This is highlighted by the fact that a combination of just three SNPs, namely rs2295632(*FAAH*), rs1045642(*ABCB1*) and rs1042713(*ADRB2*), improves risk discrimination of RD cases ([Table 3](#T3)).

### Table 3.

| SNPs | Genes (cohort) | p-value | Balanced classification accuracy (%) |
| --- | --- | --- | --- |
| rs2295632 | FAAH (DC) | 0.00005 | 65.5 |
|   |  |  |  |
| rs2295632 | FAAH (DCint) | 0.0002 | 66.7 |
|   |  |  |  |
| rs2295632 + rs1045642 | FAAH + ABCB1 (DCint) | 0.00002 | 68.2 |
|   |  |  |  |
| rs2295632 + rs1045642 + rs1042713 | FAAH + ABCB1 + ADRB2 (DCint) | 0.000003 | 69.3 |
|   |  |  |  |
| rs2295632 + rs1045642 + rs1042713 | FAAH + ABCB1 + ADRB2 (DCint-cau) | 0.0001 | 69.2 |

Table 3 Caption: Genetic variants based respiratory depression prediction.

As can be seen from [Figure 5](#F5), these SNPs are found as top discriminating features using recursive partitioning [[35](#R35)] and genotype encoding described in the ‘Methods’ section. Combining these three SNPs into a multivariate decision rule yields a p-value of 0.000003, as opposed to 0.0002 for the top discriminating SNP (i.e., rs2295632) alone ([Table 3](#T3)). The decision tree included in [Figure 5](#F5) can be interpreted in terms of the following decision rule that allows one to identify low versus high RD risk groups (strata): if ([rs2295632 = CC] or [rs1045642 = CC] or [rs1042713=AA]) then RD = no (low risk of RD) otherwise RD = yes (high risk of RD).

### Figure 5. Recursive partitioning respiratory depression decision tree.

![Figure 5](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3e25/4287371/d1640ef0aa6a/nihms-652641-f0005.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4287371_nihms-652641-f0005.jpg)

Recursive partitioning of children in the discovery cohort who received additional opioid interventions in recovery room (DCint) for their risk of respiratory depression (yes = high risk; no = low risk) using a combination of three top SNPs, rs2295632, rs1045642 and rs1042713. FAAH SNP, rs2295632 was the most discriminative for high versus low respiratory depression risk. These three SNPs were also found to contribute significantly to the clustering patterns shown in Figures 3 & 4. In Figure 5, these SNPs are combined into a simple logical rule that classifies children into low- versus high-risk strata ‘rs2295632_A = N’ denotes the lack of A allele, that is, CC genotype, as opposed to rs2295632 = Y, which corresponds to CA and AA genotypes that contain an A allele.

The above decision rule yields accuracy of approxiamtely 76% and balanced accuracy of approximately 69% on the set of patients who required post operative intervention (see [Table 3](#T3)). Note, that high-risk patients are identified as those who have risk alleles at all three loci included, which corresponds to the right-most side of the tree in [Figure 5](#F5), with 26 cases of RD = yes and 24 cases RD = no. It should also be noted that while using the most discriminative individual SNP (i.e., rs2295632) results in higher sensitivity (82%) and lower specificity (51%), the combined rule improves specificity to approximately 80% while lowering sensitivity to approximately 58% and improving the overall balanced accuracy. Thus, the multivariate rule might prove useful if false positives are of concern, while a simple test based on rs2295632 genotype might be better if false negatives are of concern.

### Further analysis of stratification

When applying the above decision rule to the subset of Caucasian patients only (DCint-cau cohort), 91 patients are classified as low risk (of which 13 have RD and are thus misclassified) and 37 as high risk (of which 18 have RD and are thus correctly classified), yielding a classification accuracy of 75% and balanced accuracy of 69%. Since these accuracies are nearly identical to those obtained on the entire DCint cohort, we conclude that the combination of SNPs selected here identifies patients with increased risk of RD independent of ancestry.

As shown in [Table 4](#T4), a significant over-representation of RD cases for rs2295632 AA/AC genotype (25.5%), and under-representation for CC genotype (9.5%) with respect to expected (population) risk of 17% is observed among children of Caucasian ancestry in the DC. On the other hand, only nine out of 55 African–American children had CC genotype. However, none of them had RD, suggesting that CC genotype could be associated with a protective effect against RD irrespective of ancestry.

### Table 4.

| FAAH rs2295632 genotype | RD (DC-cau) | No RD (DC-cau) | RD (DC-aa) | No RD (DC-aa) | RD (DC) | No RD (DC) | RD (DCint) | No RD (DCint) |
| --- | --- | --- | --- | --- | --- | --- | --- | --- |
| AC/AA | 26 | 76 | 16 | 30 | 42 | 106 | 37 | 61 |
|   |  |  |  |  |  |  |  |  |
| CC | 11 | 105 | 0 | 9 | 11 | 114 | 8 | 64 |

Table 4 Caption: Distribution of genetic variants in children with and without respiratory depression.

Overall, though, RD was observed more frequently among African–American (24.6%) compared with children of Caucasian descent (17%). It should be noted, however, that the apparently protective CC geno type for rs2295632 occurs with much lower frequency in African–American versus Caucasian populations. In reference 1000 Genome cohorts, these frequencies are 16% for African–American (specifically African ancestry in Southwest USA [ASW]) and 53% for Caucasian (specifically Utah residents with Northern and Western European ancestry [CEU]) ancestries, respectively.

This relatively low overall frequency of CC genotype could therefore partially account for higher risk of RD in African–American, compared with Caucasians. However, it should be emphasized that due to small sample size and low frequency of CC genotype, associations in African–American children analyzed here remain insignificant.

### Validation cohort

Overall hierarchical clustering of patients for a subset of patients who needed additional intervention for a joint set of discovery and a set of 74 validation cases qualitatively reproduced clustering patterns observed in the DC cohort. When clustered together with patients from the DC, twice as many RD cases from the validation set are found associated with the high-risk cluster, compared with the low-risk cluster. Of nine patients from the validation cohort who needed intervention, three and six were found in the low- and high-risk cluster, respectively. While the number of RD cases in the validation cohort is very small (making the observed differences statistically insignificant), their distribution within low- versus high-risk clusters is consistent with trends observed for patients included in the DC.

This is further supported by centroid-based assignment of RD risk. All the patients in the validation cohort were classified based on their distances to centroids of three clusters from the DC. As before, these three clusters were obtained by merging two low-risk and two high-risk clusters, while retaining the intermediate-risk cluster as a separate subtype. Four RD and 26 no RD cases were closest to the centroid of the low-risk cluster (and thus classified as low risk), four RD and 13 no RD were closest to the centroid of the intermediate-risk cluster, whereas six RD and 21 no RD cases were closest to the centroid of the high-risk cluster, and thus classified as high risk. Thus, the relative risk for the high risk class (with 23% of RD cases) versus the low risk class (with 13% of RD cases) is close to two.

### Leave-one-out k nearest neighbor assessment

We further assessed the proposed risk signature using k nearest neighbors (k-NN) classification approach, which can be viewed as complementary to the unsupervised clustering-based approach. k-NN classifier finds k-NN in a set of labelled prototypes; the majority class among such neighbors is then used to assign a class label to a new data point to be classified. Thus, k-NN results are expected to be driven by clustering patterns in the data, and can be used as additional validation (and also an alternative basis for the identification of high versus low RD risk groups) of the clustering-based approach.

The results for leave-one-out 7-NN (k = 7) crossvalidation on the primary cohort are compared with those on the validation set in [Figure 6](#F6). In the leave-one-out assessment, seven nearest neighbors among all other data points are selected for each data point (patient) from the primary cohort, and the number of RD cases among those most similar (in terms of their SNP and clinical profile used here) patients is counted. For the new data points from the validation cohort, prototypes from the primary cohort were used as the training set for the classification. As can be seen from [Figure 6](#F6), RD cases are in general found in close proximity to other RD patients, as indicated by the location of the maximum of the histograms for RD versus no RD cases. Based on this pattern, k-NN-based classification could be used to identify groups with high versus low RD risk, for example, with 0 or 1 RD neighbor identifying low risk versus four or more neighbors pointing towards high-risk cases (for 7-NN used here).

### Figure 6. Crossvalidation of respiratory depression risk prediction using k-nearest neighbors.

![Figure 6](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3e25/4287371/68403ccc0e6b/nihms-652641-f0006.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4287371_nihms-652641-f0006.jpg)

Nearest neighborhood crossvalidation of respiratory depression (RD) risk distribution of the number of RD neighbors in the leave-one-out crossvalidation among seven nearest neighbors for RD (B & D) versus no RD cases (A & C) in the training cohort (A & B) and in the control set (C & D). The corresponding distributions for non-RD cases are shifted to the left, enabling prototype-based classification of high RD risk cases found to be similar (as defined by the Hamming distance in the space of SNP and clinical variables used for Figure 4) to other RD cases.

This trend is also observed in the validation cohort, even though the overall counts of the number of RD cases are much smaller (see [Figure 6C & D](#F6)). Thus, we conclude that the high and low RD risk clusters, as well as prototype-based classification, observed for the primary discovery cohort appear to persist in the validation cohort, despite variations in allele frequencies for individual SNPs between the cohorts. This provides additional support for incorporating multiple SNPs (in some cases partially redundant), as well as the relevant clinical variables that contribute to the identification of a more robust clustering pattern.

## Discussion

Postoperative RD is a clinically and economically significant and potentially life threatening, yet preventable complication of opioids [[1](#R1)]. Associations of individual genes (markers) with RD and other opioid adverse effects are rather weak [[15](#R15)], underscoring the complex nature of the interactions between opioids and respiratory rhythm centers in the brain. Identifying genetic risk factors for RD is further complicated by gene–gene interactions, and by interactions with race, ancestry, and clinical risk factors, such as OSA. In our study, we showed that interaction of genetic variants of *FAAH*, *ABCB1* and *ADRB2* contributed to discrimination of low and high risk for morphine-related postoperative depression in children undergoing tonsillectomy. We also showed that SNP genotypes can be combined with relevant clinical variables to provide effective signatures of RD risk in those patients.

We used an unsupervised clustering approach to analyze patterns of similarity between patients in terms of their SNP genotypes and identify groups (clusters) of patients with distinct combination of risk or protective variants (of candidate genes considered in this study) and overall risk of RD. Thus, we effectively tested the hypothesis that unsupervised clustering could reveal groups of patients that share a pattern of genetic polymorphisms associated with elevated (or diminished) risk of RD. SNPs that contribute most to defining such distinct clusters (in conjunction with clinical variables, such as OSA) can be regarded as ‘SNP signatures’ of those clusters (subtypes), in analogy with gene signatures that define molecular subtypes of cancer [[16](#R16),[36](#R36),[37](#R37)].

Recursive partitioning using decision trees, on the other hand, explicitly stratifies the data based on the discriminatory power of individual variables and their combination. The goal is to learn a classifier that optimizes classification accuracy using explicitly class labels, while selecting the most predictive SNPs and other variables (and their combinations). Strategies to stratify patients into distinct subtypes are expected to help identify stronger (and possibly distinct in each subtype) pattern of associations between SNPs (or other markers) and RD. In addition, decision trees provide simple to interpret decision rules that can be used as a basis for clinical decision support. Using complementary classification (recursive partitioning) and clustering (SNP signature) strategies can also facilitate the validation of the results and uncover hidden structure in the data.

We found that results of the two approaches yielded largely consistent results. Several SNPs in *FAAH*,*ABCB1* and *ADRB2* were found to contribute to both distinct clusters, characterized by significantly different risks of RD, and decision rules that can be used to predict the risk of RD. Relative risk for RD between high-risk clusters (36–38% of RD incidence) and low-risk clusters (10–17% of RD incidence) ranged from 2.1–3.8 (p = 0.003), while decision rules that combine those SNPs yield overall accuracy of approximately 75% on the discovery cohort.

The strongest associations with RD are observed for SNPs in the *FAAH* gene in our study. Those SNPs are consequently contributing to risk signatures and decision rules derived using our discovery cohort. FAAH has been associated with pain sensitivity [[38](#R38)], opioid and drug abuse/addiction [[39](#R39)–[44](#R44)], and modulation of opioid effects [[45](#R45)]. A model has been proposed in which deregulation or loss of function of FAAH leads to accumulation of endogenous cannabinoids that act through CB1 receptors. These receptors are widely distributed throughout the CNS, including the brainstem, and modulate a variety of functions, including breathing. In addition, anandamide, if protected from degradation by a decrease in FAAH catalytic activity, had been hypothesized to act via the CB1 receptor and modulate morphine's analgesia by interactions with κ-opioid receptors [[46](#R46)]. In neonatal mice, activation of cannabinoid CB1 receptor with anandamide had been shown to depress the medullary respiratory rhythm generator [[47](#R47)]. Variations in *ABCB1* efflux pump P-gp transporter at the blood brain barrier can significantly alter cerebral pharmacokinetics and respiratory depressive effects of opioids [[11](#R11),[48](#R48)]. Additional associations with RD are independently observed for SNPs in the *ABCB1* gene, which encodes a key transporter involved in opioids’ (including morphine) central analgesic and adverse effects [[49](#R49)].

The advantage of ‘SNP signatures’ presented here is that they were derived using an unsupervised approach (and therefore did not require optimization of parameters in the model on the training set), which bodes well for their generalization. On the other hand, the accuracies for the SNP-based decision rules reported here on the DC are training set accuracies. Therefore, although they remain stable with respect to subset sampling ([Table 4](#T4)), they should not be overinterpreted. Here, these rules are primarily used as evidence of the consistency between implicit (similarity-based) and explicit (decision-tree-based) stratification and the resulting differentiation in terms of RD risk.

## Conclusion & future perspective

This study represents one of the first attempts to use genetic risk signatures for the prediction of the risk of side effects from opioids. Further studies will be required to assess generalization and applicability of our early RD risk signatures presented here. Contingent upon validation on a larger control cohort, both clustering-based patterns, especially for extreme clusters with the highest and lowest risk of RD, and the corresponding k-NN-based classification could pro-Nearest vide a basis for clinical decisions. Such techniques have previously been used successfully to define cancer subtypes and progression, and upon validation, our results will be a basis for clinically applicable rules for predrug risk assessment in children receiving opioids.

Interindividual difference in morphine-related postoperative depression risk can be attributed to certain genetic factors

**Inter-racial difference in morphine-related RD**

Multiple clinical & genetic risk factors contribute to differentiating patients at lower & higher risks of morphine-related RD

Interindividual difference in morphine-related postoperative depression risk can be attributed to certain genetic factors

**Inter-racial difference in morphine-related RD**

Multiple clinical & genetic risk factors contribute to differentiating patients at lower & higher risks of morphine-related RD

## Acknowledgments

This work was supported in part by USPHS Grant #UL1 RR026314 from the National Center for Research Resources, NIH and with the Place Outcomes Research Award (principal investigator [PI]: S Sadhasivam), Innovation Fund (PI: S Sadhasivam) and Translational Research Award (PI: S Sadhasivam), Cincinnati Children's Hospital Medical Center, Cincinnati, OH, USA. Additional research funding support was provided by the Department of Anesthesia, Cincinnati Children's Hospital Medical Center, Cincinnati, OH, USA. No financial support except departmental salary support for the authors. This pharmacogenetic study was designed and undertaken by the authors. The sponsor of this study, the Cincinnati Children's Hospital Medical Center (CCHMC) provided funding support for the genetic analyses and supported salary of the research team. The authors directed and had access to all the analyses and the full clinical and genetic database, wrote all drafts of the report, decided to publish the results, and attest for the accuracy and completeness of the data. In addition, J Meller received support from the NIH (P30ES006096, UL1RR026314, T32GM105526-01 and R21AI097936-01).

## Footnotes

## References

1. Oderda GM, Said Q, Evans RS, et al. Opioid-related adverse drug events in surgical hospitalizations: impact on costs and length of stay. Ann. Pharmacother. 2007;41(3):400–406. doi: 10.1345/aph.1H386. [Highlights clinical and economic impacts of opioid-related adverse effects including quality of postsurgical recovery and length of hospital stay in patients undergoing surgery.]  [DOI](https://doi.org/10.1345/aph.1H386) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17341537/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann.%20Pharmacother&title=Opioid-related%20adverse%20drug%20events%20in%20surgical%20hospitalizations:%20impact%20on%20costs%20and%20length%20of%20stay.&author=GM%20Oderda&author=Q%20Said&author=RS%20Evans&volume=41&issue=3&publication_year=2007&pages=400-406&pmid=17341537&doi=10.1345/aph.1H386&)

2. Angst MS, Lazzeroni LC, Phillips NG, et al. Aversive and reinforcing opioid effects: a pharmacogenomic twin study. Anesthesiology. 2012;117(1):22–37. doi: 10.1097/ALN.0b013e31825a2a4e.  [DOI](https://doi.org/10.1097/ALN.0b013e31825a2a4e) | [PMC free article](/articles/PMC3428265/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22713632/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Anesthesiology&title=Aversive%20and%20reinforcing%20opioid%20effects:%20a%20pharmacogenomic%20twin%20study.&author=MS%20Angst&author=LC%20Lazzeroni&author=NG%20Phillips&volume=117&issue=1&publication_year=2012&pages=22-37&pmid=22713632&doi=10.1097/ALN.0b013e31825a2a4e&)

3. Angst MS, Phillips NG, Drover DR, et al. Pain sensitivity and opioid analgesia: a pharmacogenomic twin study. Pain. 2012;153(7):1397–1409. doi: 10.1016/j.pain.2012.02.022.  [DOI](https://doi.org/10.1016/j.pain.2012.02.022) | [PMC free article](/articles/PMC3377769/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22444188/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pain&title=Pain%20sensitivity%20and%20opioid%20analgesia:%20a%20pharmacogenomic%20twin%20study.&author=MS%20Angst&author=NG%20Phillips&author=DR%20Drover&volume=153&issue=7&publication_year=2012&pages=1397-1409&pmid=22444188&doi=10.1016/j.pain.2012.02.022&)

4. Oertel BG, Schmidt R, Schneider A, Geisslinger G, Lotsch J. The mu-opioid receptor gene polymorphism 118A>G depletes alfentanil-induced analgesia and protects against respiratory depression in homozygous carriers. Pharmacogenet. Genomics. 2006;16(9):625–636. doi: 10.1097/01.fpc.0000220566.90466.a2.  [DOI](https://doi.org/10.1097/01.fpc.0000220566.90466.a2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16906017/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet.%20Genomics&title=The%20mu-opioid%20receptor%20gene%20polymorphism%20118A>G%20depletes%20alfentanil-induced%20analgesia%20and%20protects%20against%20respiratory%20depression%20in%20homozygous%20carriers.&author=BG%20Oertel&author=R%20Schmidt&author=A%20Schneider&author=G%20Geisslinger&author=J%20Lotsch&volume=16&issue=9&publication_year=2006&pages=625-636&pmid=16906017&doi=10.1097/01.fpc.0000220566.90466.a2&)

5. Voronov P, Przybylo HJ, Jagannathan N. Apnea in a child after oral codeine: a genetic variant – an ultra-rapid metabolizer. Paediatr. Anaesth. 2007;17(7):684–687. doi: 10.1111/j.1460-9592.2006.02182.x.  [DOI](https://doi.org/10.1111/j.1460-9592.2006.02182.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17564651/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Paediatr.%20Anaesth&title=Apnea%20in%20a%20child%20after%20oral%20codeine:%20a%20genetic%20variant%20%E2%80%93%20an%20ultra-rapid%20metabolizer.&author=P%20Voronov&author=HJ%20Przybylo&author=N%20Jagannathan&volume=17&issue=7&publication_year=2007&pages=684-687&pmid=17564651&doi=10.1111/j.1460-9592.2006.02182.x&)

6. Voronov P. Codeine induced respiratory depression in a child: Author's reply. Paediatr. Anaesth. 2008;18(3):273–274. doi: 10.1111/j.1460-9592.2008.02415.x.  [DOI](https://doi.org/10.1111/j.1460-9592.2008.02415.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18230075/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Paediatr.%20Anaesth&title=Codeine%20induced%20respiratory%20depression%20in%20a%20child:%20Author's%20reply.&author=P%20Voronov&volume=18&issue=3&publication_year=2008&pages=273-274&pmid=18230075&doi=10.1111/j.1460-9592.2008.02415.x&)

7. Ciszkowski C, Madadi P, Phillips MS, Lauwers AE, Koren G. Codeine, ultrarapid-metabolism genotype, and postoperative death. N. Engl. J. Med. 2009;361(8):827–828. doi: 10.1056/NEJMc0904266. [Reports codeine postoperative death in an ultrarapid CYP2D6 metabolizer with relatively high morphine formation resulting in respiratory depression.]  [DOI](https://doi.org/10.1056/NEJMc0904266) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19692698/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N.%20Engl.%20J.%20Med&title=Codeine,%20ultrarapid-metabolism%20genotype,%20and%20postoperative%20death.&author=C%20Ciszkowski&author=P%20Madadi&author=MS%20Phillips&author=AE%20Lauwers&author=G%20Koren&volume=361&issue=8&publication_year=2009&pages=827-828&pmid=19692698&doi=10.1056/NEJMc0904266&)

8. Hermanns-Clausen M, Weinmann W, Auwarter V, et al. Drug dosing error with drops: severe clinical course of codeine intoxication in twins. Eur. J. Pediatr. 2009;168(7):819–824. doi: 10.1007/s00431-008-0842-7.  [DOI](https://doi.org/10.1007/s00431-008-0842-7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18936971/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur.%20J.%20Pediatr&title=Drug%20dosing%20error%20with%20drops:%20severe%20clinical%20course%20of%20codeine%20intoxication%20in%20twins.&author=M%20Hermanns-Clausen&author=W%20Weinmann&author=V%20Auwarter&volume=168&issue=7&publication_year=2009&pages=819-824&pmid=18936971&doi=10.1007/s00431-008-0842-7&)

9. Kelly L, Rieder M, Van Den Anker J, et al. More codeine fatalities after tonsillectomy in North American children. Pediatrics. 2012;129(5):e1343–e1347. doi: 10.1542/peds.2011-2538.  [DOI](https://doi.org/10.1542/peds.2011-2538) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22492761/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pediatrics&title=More%20codeine%20fatalities%20after%20tonsillectomy%20in%20North%20American%20children.&author=L%20Kelly&author=M%20Rieder&author=J%20Van%20Den%20Anker&volume=129&issue=5&publication_year=2012&pages=e1343-e1347&pmid=22492761&doi=10.1542/peds.2011-2538&)

10. Coulbault L, Beaussier M, Verstuyft C, et al. Environmental and genetic factors associated with morphine response in the postoperative period. Clin. Pharmacol. Ther. 2006;79(4):316–324. doi: 10.1016/j.clpt.2006.01.007.  [DOI](https://doi.org/10.1016/j.clpt.2006.01.007) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16580900/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther&title=Environmental%20and%20genetic%20factors%20associated%20with%20morphine%20response%20in%20the%20postoperative%20period.&author=L%20Coulbault&author=M%20Beaussier&author=C%20Verstuyft&volume=79&issue=4&publication_year=2006&pages=316-324&pmid=16580900&doi=10.1016/j.clpt.2006.01.007&)

11. Park HJ, Shinn HK, Ryu SH, Lee HS, Park CS, Kang JH. Genetic polymorphisms in the ABCB1 gene and the effects of fentanyl in Koreans. Clin. Pharmacol. Ther. 2007;81(4):539–546. doi: 10.1038/sj.clpt.6100046. [This article in Korean adults receiving a different opioid, fentanyl, is supportive of our pediatric postoperative respiratory depression with morphine results in that genetic variants of ABCB1 are associated with slow respiratory rates.]  [DOI](https://doi.org/10.1038/sj.clpt.6100046) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17192767/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther&title=Genetic%20polymorphisms%20in%20the%20ABCB1%20gene%20and%20the%20effects%20of%20fentanyl%20in%20Koreans.&author=HJ%20Park&author=HK%20Shinn&author=SH%20Ryu&author=HS%20Lee&author=CS%20Park&volume=81&issue=4&publication_year=2007&pages=539-546&pmid=17192767&doi=10.1038/sj.clpt.6100046&)

12. Fujita KI, Ando Y, Yamamoto W, et al. Association of UGT2B7 and ABCB1 genotypes with morphine-induced adverse drug reactions in Japanese patients with cancer. Cancer Chemother. Pharmacol. 2009;65(2):251–258. doi: 10.1007/s00280-009-1029-2.  [DOI](https://doi.org/10.1007/s00280-009-1029-2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19466410/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Chemother.%20Pharmacol&title=Association%20of%20UGT2B7%20and%20ABCB1%20genotypes%20with%20morphine-induced%20adverse%20drug%20reactions%20in%20Japanese%20patients%20with%20cancer.&author=KI%20Fujita&author=Y%20Ando&author=W%20Yamamoto&volume=65&issue=2&publication_year=2009&pages=251-258&pmid=19466410&doi=10.1007/s00280-009-1029-2&)

13. Kolesnikov Y, Gabovits B, Levin A, Voiko E, Veske A. Combined catechol-O-methyltransferase and mu-opioid receptor gene polymorphisms affect morphine postoperative analgesia and central side effects. Anesth. Analg. 2011;112(2):448–453. doi: 10.1213/ANE.0b013e318202cc8d.  [DOI](https://doi.org/10.1213/ANE.0b013e318202cc8d) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21127283/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Anesth.%20Analg&title=Combined%20catechol-O-methyltransferase%20and%20mu-opioid%20receptor%20gene%20polymorphisms%20affect%20morphine%20postoperative%20analgesia%20and%20central%20side%20effects.&author=Y%20Kolesnikov&author=B%20Gabovits&author=A%20Levin&author=E%20Voiko&author=A%20Veske&volume=112&issue=2&publication_year=2011&pages=448-453&pmid=21127283&doi=10.1213/ANE.0b013e318202cc8d&)

14. Zwisler ST, Enggaard TP, Noehr-Jensen L, et al. The antinociceptive effect and adverse drug reactions of oxycodone in human experimental pain in relation to genetic variations in the OPRM1 and ABCB1 genes. Fundam. Clin. Pharmacol. 2010;24(4):517–524. doi: 10.1111/j.1472-8206.2009.00781.x.  [DOI](https://doi.org/10.1111/j.1472-8206.2009.00781.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19845769/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Fundam.%20Clin.%20Pharmacol&title=The%20antinociceptive%20effect%20and%20adverse%20drug%20reactions%20of%20oxycodone%20in%20human%20experimental%20pain%20in%20relation%20to%20genetic%20variations%20in%20the%20OPRM1%20and%20ABCB1%20genes.&author=ST%20Zwisler&author=TP%20Enggaard&author=L%20Noehr-Jensen&volume=24&issue=4&publication_year=2010&pages=517-524&pmid=19845769&doi=10.1111/j.1472-8206.2009.00781.x&)

15. Sadhasivam S, Chidambaran V. Pharmacogenomics of opioids and perioperative pain management. Pharmacogenomics. 2012;13(15):1719–1740. doi: 10.2217/pgs.12.152.  [DOI](https://doi.org/10.2217/pgs.12.152) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23171337/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Pharmacogenomics%20of%20opioids%20and%20perioperative%20pain%20management.&author=S%20Sadhasivam&author=V%20Chidambaran&volume=13&issue=15&publication_year=2012&pages=1719-1740&pmid=23171337&doi=10.2217/pgs.12.152&)

16. Miller LD, Smeds J, George J, et al. An expression signature for p53 status in human breast cancer predicts mutation status, transcriptional effects, and patient survival. Proc. Natl Acad. Sci. USA. 2005;102(38):13550–13555. doi: 10.1073/pnas.0506230102.  [DOI](https://doi.org/10.1073/pnas.0506230102) | [PMC free article](/articles/PMC1197273/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16141321/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl%20Acad.%20Sci.%20USA&title=An%20expression%20signature%20for%20p53%20status%20in%20human%20breast%20cancer%20predicts%20mutation%20status,%20transcriptional%20effects,%20and%20patient%20survival.&author=LD%20Miller&author=J%20Smeds&author=J%20George&volume=102&issue=38&publication_year=2005&pages=13550-13555&pmid=16141321&doi=10.1073/pnas.0506230102&)

17. Boehmer U, Kressin NR, Berlowitz DR, Christiansen CL, Kazis LE, Jones JA. Self-reported vs administrative race/ethnicity data and study results. Am. J. Public Health. 2002;92(9):1471–1472. doi: 10.2105/ajph.92.9.1471.  [DOI](https://doi.org/10.2105/ajph.92.9.1471) | [PMC free article](/articles/PMC1447261/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12197976/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Public%20Health&title=Self-reported%20vs%20administrative%20race/ethnicity%20data%20and%20study%20results.&author=U%20Boehmer&author=NR%20Kressin&author=DR%20Berlowitz&author=CL%20Christiansen&author=LE%20Kazis&volume=92&issue=9&publication_year=2002&pages=1471-1472&pmid=12197976&doi=10.2105/ajph.92.9.1471&)

18. Lalovic B, Phillips B, Risler LL, Howald W, Shen DD. Quantitative contribution of CYP2D6 and CYP3A to oxycodone metabolism in human liver and intestinal microsomes. Drug Metab. Dispos. 2004;32(4):447–454. doi: 10.1124/dmd.32.4.447.  [DOI](https://doi.org/10.1124/dmd.32.4.447) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15039299/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab.%20Dispos&title=Quantitative%20contribution%20of%20CYP2D6%20and%20CYP3A%20to%20oxycodone%20metabolism%20in%20human%20liver%20and%20intestinal%20microsomes.&author=B%20Lalovic&author=B%20Phillips&author=LL%20Risler&author=W%20Howald&author=DD%20Shen&volume=32&issue=4&publication_year=2004&pages=447-454&pmid=15039299&doi=10.1124/dmd.32.4.447&)

19. Merkel SI, Voepel-Lewis T, Shayevitz JR, Malviya S. The FLACC: a behavioral scale for scoring postoperative pain in young children. Pediatr. Nurs. 1997;23(3):293–297.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/9220806/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pediatr.%20Nurs&title=The%20FLACC:%20a%20behavioral%20scale%20for%20scoring%20postoperative%20pain%20in%20young%20children.&author=SI%20Merkel&author=T%20Voepel-Lewis&author=JR%20Shayevitz&author=S%20Malviya&volume=23&issue=3&publication_year=1997&pages=293-297&pmid=9220806&)

20. Sadhasivam S, Chidambaran V, Ngamprasertwong P, et al. Race and unequal burden of perioperative pain and opioid related adverse effects in children. Pediatrics. 2012;129(5):832–838. doi: 10.1542/peds.2011-2607. [Our previous clinical study showing racial differences in perioperative morphine-related outcomes in African–American and Caucasian children.]  [DOI](https://doi.org/10.1542/peds.2011-2607) | [PMC free article](/articles/PMC3340593/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22529273/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pediatrics&title=Race%20and%20unequal%20burden%20of%20perioperative%20pain%20and%20opioid%20related%20adverse%20effects%20in%20children.&author=S%20Sadhasivam&author=V%20Chidambaran&author=P%20Ngamprasertwong&volume=129&issue=5&publication_year=2012&pages=832-838&pmid=22529273&doi=10.1542/peds.2011-2607&)

21. Purcell S, Neale B, Todd-Brown K, et al. PLINK: a tool set for whole-genome association and population-based linkage analyses. Am. J. Hum. Genet. 2007;81(3):559–575. doi: 10.1086/519795.  [DOI](https://doi.org/10.1086/519795) | [PMC free article](/articles/PMC1950838/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17701901/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Hum.%20Genet&title=PLINK:%20a%20tool%20set%20for%20whole-genome%20association%20and%20population-based%20linkage%20analyses.&author=S%20Purcell&author=B%20Neale&author=K%20Todd-Brown&volume=81&issue=3&publication_year=2007&pages=559-575&pmid=17701901&doi=10.1086/519795&)

22. PLINK. http://pngu.mgh.harvard.edu/purcell/plink.  [http://pngu.mgh.harvard.edu/purcell/plink](http://pngu.mgh.harvard.edu/purcell/plink)

23. International Hapmap Consortium. Altshuler DM, Gibbs RA, et al. Integrating common and rare genetic variation in diverse human populations. Nature. 2010;467(7311):52–58. doi: 10.1038/nature09298.  [DOI](https://doi.org/10.1038/nature09298) | [PMC free article](/articles/PMC3173859/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20811451/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Integrating%20common%20and%20rare%20genetic%20variation%20in%20diverse%20human%20populations.&author=DM%20Altshuler&author=RA%20Gibbs&volume=467&issue=7311&publication_year=2010&pages=52-58&pmid=20811451&doi=10.1038/nature09298&)

24. International Hapmap Consortium The International HapMap Project. Nature. 2003;426(6968):789–796. doi: 10.1038/nature02168.  [DOI](https://doi.org/10.1038/nature02168) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14685227/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=The%20International%20HapMap%20Project.&volume=426&issue=6968&publication_year=2003&pages=789-796&pmid=14685227&doi=10.1038/nature02168&)

25. 1000 Genomes Project Consortium. Abecasis GR, Altshuler D, et al. A map of human genome variation from population-scale sequencing. Nature. 2010;467(7319):1061–1073. doi: 10.1038/nature09534.  [DOI](https://doi.org/10.1038/nature09534) | [PMC free article](/articles/PMC3042601/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20981092/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=A%20map%20of%20human%20genome%20variation%20from%20population-scale%20sequencing.&author=GR%20Abecasis&author=D%20Altshuler&volume=467&issue=7319&publication_year=2010&pages=1061-1073&pmid=20981092&doi=10.1038/nature09534&)

26. Torgo L. Data mining with R: learning with case studies. Chapman & Hall/CRC Press; FL, USA: 2010.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Data%20mining%20with%20R:%20learning%20with%20case%20studies&author=L%20Torgo&publication_year=2010&)

27. The R Project for Statistical Computing. www.R-project.org.  [www.R-project.org](http://www.R-project.org)

28. TreeView. www.princeton.edu/~abarysh/treeview.  [www.princeton.edu/~abarysh/treeview](http://www.princeton.edu/~abarysh/treeview)

29. Becker RA, Chambers JM, Wilks AR. The new S language. Wadsworth & Brooks; CA, USA: 1988.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=The%20new%20S%20language&author=RA%20Becker&author=JM%20Chambers&author=AR%20Wilks&publication_year=1988&)

30. Suzuki R, Shimodaira H. Pvclust: an R package for assessing the uncertainty in hierarchical clustering. Bioinformatics. 2006;22(12):1540–1542. doi: 10.1093/bioinformatics/btl117.  [DOI](https://doi.org/10.1093/bioinformatics/btl117) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16595560/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=Pvclust:%20an%20R%20package%20for%20assessing%20the%20uncertainty%20in%20hierarchical%20clustering.&author=R%20Suzuki&author=H%20Shimodaira&volume=22&issue=12&publication_year=2006&pages=1540-1542&pmid=16595560&doi=10.1093/bioinformatics/btl117&)

31. Lucas A, Jasson S. Using amap and ctc packages for huge clustering. The Newsletter of the R Project Volume 6/5. 2006 1985 Dec;6(3):58.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=The%20Newsletter%20of%20the%20R%20Project%20Volume%206/5&title=Using%20amap%20and%20ctc%20packages%20for%20huge%20clustering.&author=A%20Lucas&author=S%20Jasson&volume=6&issue=3&publication_year=2006&pages=58&)

32. Chawla NV, Bowyer KW, Hall LO, Kegelmeyer WP. SMOTE: synthetic minority over-sampling technique. J. Artificial Intelligence Res. 2002;16:321–357.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Artificial%20Intelligence%20Res&title=SMOTE:%20synthetic%20minority%20over-sampling%20technique.&author=NV%20Chawla&author=KW%20Bowyer&author=LO%20Hall&author=WP%20Kegelmeyer&volume=16&publication_year=2002&pages=321-357&)

33. Chawla NV, Lazarevic A, Hall LO, Bowyer KW. In: Knowledge Discovery in Databases: PKDD 2003. Springer; NY, USA: 2003. SMOTEBoost: improving prediction of the minority class in boosting. pp. 107–119.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=In:%20Knowledge%20Discovery%20in%20Databases:%20PKDD%202003&author=NV%20Chawla&author=A%20Lazarevic&author=LO%20Hall&author=KW%20Bowyer&publication_year=2003&)

34. Chang C-C, Lin C-J. LIBSVM: a library for support vector machines. ACM Transactions on Intelligent Systems and Technology (TIST) 2011;2(3):27.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=ACM%20Transactions%20on%20Intelligent%20Systems%20and%20Technology%20(TIST)&title=LIBSVM:%20a%20library%20for%20support%20vector%20machines.&author=C-C%20Chang&author=C-J%20Lin&volume=2&issue=3&publication_year=2011&pages=27&)

35. Breiman L, Friedman J, Stone CJ, Olshen RA. Classification and Regression Trees (Wadsworth Statistics/ Probability) Chapman & Hall/CRC Press; FL, USA: 1984.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Classification%20and%20Regression%20Trees%20(Wadsworth%20Statistics/%20Probability)&author=L%20Breiman&author=J%20Friedman&author=CJ%20Stone&author=RA%20Olshen&publication_year=1984&)

36. Collins FS, Barker AD. Mapping the cancer genome. Pinpointing the genes involved in cancer will help chart a new course across the complex landscape of human malignancies. Sci. Am. 2007;296(3):50–57. [Genetic signatures of cancers are being shown to help uncover traces of the mutagenic influences, define clinically relevant subtypes for prognosis and therapeutic management, and enable the development of new cancer therapies.]  [PubMed](https://pubmed.ncbi.nlm.nih.gov/17348159/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Sci.%20Am&title=Mapping%20the%20cancer%20genome.%20Pinpointing%20the%20genes%20involved%20in%20cancer%20will%20help%20chart%20a%20new%20course%20across%20the%20complex%20landscape%20of%20human%20malignancies.&author=FS%20Collins&author=AD%20Barker&volume=296&issue=3&publication_year=2007&pages=50-57&pmid=17348159&)

37. Cancer Genome Atlas Network Comprehensive molecular portraits of human breast tumours. Nature. 2012;490(7418):61–70. doi: 10.1038/nature11412.  [DOI](https://doi.org/10.1038/nature11412) | [PMC free article](/articles/PMC3465532/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23000897/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Comprehensive%20molecular%20portraits%20of%20human%20breast%20tumours.&volume=490&issue=7418&publication_year=2012&pages=61-70&pmid=23000897&doi=10.1038/nature11412&)

38. Kim H, Mittal DP, Iadarola MJ, Dionne RA. Genetic predictors for acute experimental cold and heat pain sensitivity in humans. J. Med. Genet. 2006;43(8):e40. doi: 10.1136/jmg.2005.036079.  [DOI](https://doi.org/10.1136/jmg.2005.036079) | [PMC free article](/articles/PMC2564596/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16882734/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Med.%20Genet&title=Genetic%20predictors%20for%20acute%20experimental%20cold%20and%20heat%20pain%20sensitivity%20in%20humans.&author=H%20Kim&author=DP%20Mittal&author=MJ%20Iadarola&author=RA%20Dionne&volume=43&issue=8&publication_year=2006&pages=e40&pmid=16882734&doi=10.1136/jmg.2005.036079&)

39. Sipe JC, Chiang K, Gerber AL, Beutler E, Cravatt BF. A missense mutation in human fatty acid amide hydrolase associated with problem drug use. Proc. Natl Acad. Sci. USA. 2002;99(12):8394–8399. doi: 10.1073/pnas.082235799.  [DOI](https://doi.org/10.1073/pnas.082235799) | [PMC free article](/articles/PMC123078/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12060782/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl%20Acad.%20Sci.%20USA&title=A%20missense%20mutation%20in%20human%20fatty%20acid%20amide%20hydrolase%20associated%20with%20problem%20drug%20use.&author=JC%20Sipe&author=K%20Chiang&author=AL%20Gerber&author=E%20Beutler&author=BF%20Cravatt&volume=99&issue=12&publication_year=2002&pages=8394-8399&pmid=12060782&doi=10.1073/pnas.082235799&)

40. Chiang KP, Gerber AL, Sipe JC, Cravatt BF. Reduced cellular expression and activity of the P129T mutant of human fatty acid amide hydrolase: evidence for a link between defects in the endocannabinoid system and problem drug use. Hum. Mol. Genet. 2004;13(18):2113–2119. doi: 10.1093/hmg/ddh216.  [DOI](https://doi.org/10.1093/hmg/ddh216) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15254019/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum.%20Mol.%20Genet&title=Reduced%20cellular%20expression%20and%20activity%20of%20the%20P129T%20mutant%20of%20human%20fatty%20acid%20amide%20hydrolase:%20evidence%20for%20a%20link%20between%20defects%20in%20the%20endocannabinoid%20system%20and%20problem%20drug%20use.&author=KP%20Chiang&author=AL%20Gerber&author=JC%20Sipe&author=BF%20Cravatt&volume=13&issue=18&publication_year=2004&pages=2113-2119&pmid=15254019&doi=10.1093/hmg/ddh216&)

41. Flanagan JM, Gerber AL, Cadet JL, Beutler E, Sipe JC. The fatty acid amide hydrolase 385 A/A (P129T) variant: haplotype analysis of an ancient missense mutation and validation of risk for drug addiction. Hum. Genet. 2006;120(4):581–588. doi: 10.1007/s00439-006-0250-x.  [DOI](https://doi.org/10.1007/s00439-006-0250-x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16972078/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum.%20Genet&title=The%20fatty%20acid%20amide%20hydrolase%20385%20A/A%20(P129T)%20variant:%20haplotype%20analysis%20of%20an%20ancient%20missense%20mutation%20and%20validation%20of%20risk%20for%20drug%20addiction.&author=JM%20Flanagan&author=AL%20Gerber&author=JL%20Cadet&author=E%20Beutler&author=JC%20Sipe&volume=120&issue=4&publication_year=2006&pages=581-588&pmid=16972078&doi=10.1007/s00439-006-0250-x&)

42. Dlugos AM, Hamidovic A, Hodgkinson CA, Goldman D, Palmer AA, De Wit H. More aroused, less fatigued: fatty acid amide hydrolase gene polymorphisms influence acute response to amphetamine. Neuropsychopharmacology. 2010;35(3):613–622. doi: 10.1038/npp.2009.166.  [DOI](https://doi.org/10.1038/npp.2009.166) | [PMC free article](/articles/PMC2945903/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19890266/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neuropsychopharmacology&title=More%20aroused,%20less%20fatigued:%20fatty%20acid%20amide%20hydrolase%20gene%20polymorphisms%20influence%20acute%20response%20to%20amphetamine.&author=AM%20Dlugos&author=A%20Hamidovic&author=CA%20Hodgkinson&author=D%20Goldman&author=AA%20Palmer&volume=35&issue=3&publication_year=2010&pages=613-622&pmid=19890266&doi=10.1038/npp.2009.166&)

43. Filbey FM, Schacht JP, Myers US, Chavez RS, Hutchison KE. Individual and additive effects of the CNR1 and FAAH genes on brain response to marijuana cues. Neuropsychopharmacology. 2010;35(4):967–975. doi: 10.1038/npp.2009.200.  [DOI](https://doi.org/10.1038/npp.2009.200) | [PMC free article](/articles/PMC2820137/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20010552/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neuropsychopharmacology&title=Individual%20and%20additive%20effects%20of%20the%20CNR1%20and%20FAAH%20genes%20on%20brain%20response%20to%20marijuana%20cues.&author=FM%20Filbey&author=JP%20Schacht&author=US%20Myers&author=RS%20Chavez&author=KE%20Hutchison&volume=35&issue=4&publication_year=2010&pages=967-975&pmid=20010552&doi=10.1038/npp.2009.200&)

44. Lopez-Moreno JA, Echeverry-Alzate V, Buhler KM. The genetic basis of the endocannabinoid system and drug addiction in humans. J. Psychopharmacol. 2012;26(1):133–143. doi: 10.1177/0269881111416689.  [DOI](https://doi.org/10.1177/0269881111416689) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21937688/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Psychopharmacol&title=The%20genetic%20basis%20of%20the%20endocannabinoid%20system%20and%20drug%20addiction%20in%20humans.&author=JA%20Lopez-Moreno&author=V%20Echeverry-Alzate&author=KM%20Buhler&volume=26&issue=1&publication_year=2012&pages=133-143&pmid=21937688&doi=10.1177/0269881111416689&)

45. Scavone JL, Sterling RC, Van Bockstaele EJ. Cannabinoid and opioid interactions: implications for opiate dependence and withdrawal. Neuroscience. 2013;248:637–654. doi: 10.1016/j.neuroscience.2013.04.034.  [DOI](https://doi.org/10.1016/j.neuroscience.2013.04.034) | [PMC free article](/articles/PMC3742578/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23624062/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neuroscience&title=Cannabinoid%20and%20opioid%20interactions:%20implications%20for%20opiate%20dependence%20and%20withdrawal.&author=JL%20Scavone&author=RC%20Sterling&author=EJ%20Van%20Bockstaele&volume=248&publication_year=2013&pages=637-654&pmid=23624062&doi=10.1016/j.neuroscience.2013.04.034&)

46. Haller VL, Stevens DL, Welch SP. Modulation of opioids via protection of anandamide degradation by fatty acid amide hydrolase. Eur. J. Pharmacol. 2008;600(1–3):50–58. doi: 10.1016/j.ejphar.2008.08.005.  [DOI](https://doi.org/10.1016/j.ejphar.2008.08.005) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18762181/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur.%20J.%20Pharmacol&title=Modulation%20of%20opioids%20via%20protection%20of%20anandamide%20degradation%20by%20fatty%20acid%20amide%20hydrolase.&author=VL%20Haller&author=DL%20Stevens&author=SP%20Welch&volume=600&issue=1%E2%80%933&publication_year=2008&pages=50-58&pmid=18762181&doi=10.1016/j.ejphar.2008.08.005&)

47. Tree K, Caravagna C, Hilaire G, Peyronnet J, Cayetanot F. Anandamide centrally depresses the respiratory rhythm generator of neonatal mice. Neuroscience. 2010;170(4):1098–1109. doi: 10.1016/j.neuroscience.2010.08.045. [Mechaistically supports our postoperative morphine-related respiratory depression in children with genetic variants of FAAH, which catalizes anandamide.]  [DOI](https://doi.org/10.1016/j.neuroscience.2010.08.045) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20800658/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neuroscience&title=Anandamide%20centrally%20depresses%20the%20respiratory%20rhythm%20generator%20of%20neonatal%20mice.&author=K%20Tree&author=C%20Caravagna&author=G%20Hilaire&author=J%20Peyronnet&author=F%20Cayetanot&volume=170&issue=4&publication_year=2010&pages=1098-1109&pmid=20800658&doi=10.1016/j.neuroscience.2010.08.045&)

48. Kesimci E, Engin AB, Kanbak O, Karahalil B. Association between ABCB1 gene polymorphisms and fentanyl's adverse effects in Turkish patients undergoing spinal anesthesia. Gene. 2012;493(2):273–277. doi: 10.1016/j.gene.2011.11.040.  [DOI](https://doi.org/10.1016/j.gene.2011.11.040) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22197686/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gene&title=Association%20between%20ABCB1%20gene%20polymorphisms%20and%20fentanyl's%20adverse%20effects%20in%20Turkish%20patients%20undergoing%20spinal%20anesthesia.&author=E%20Kesimci&author=AB%20Engin&author=O%20Kanbak&author=B%20Karahalil&volume=493&issue=2&publication_year=2012&pages=273-277&pmid=22197686&doi=10.1016/j.gene.2011.11.040&)

49. Sadhasivam S, Chidambaran V, Zhang X, et al. Opioid-induced respiratory depression: ABCB1 transporter pharmacogenetics. Pharmacogenomics J. 2014 doi: 10.1038/tpj.2014.56. doi:10.1038/tpj.2014.56 (In Press). [Our clinical study demonstrating ABCB1 to be a significant contributor to morphine-induced respiratory depression and prolonged hospital stays in children.]  [DOI](https://doi.org/10.1038/tpj.2014.56) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25311385/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=Opioid-induced%20respiratory%20depression:%20ABCB1%20transporter%20pharmacogenetics.&author=S%20Sadhasivam&author=V%20Chidambaran&author=X%20Zhang&publication_year=2014&pmid=25311385&doi=10.1038/tpj.2014.56&)
