# Prognostic role of the LCS6 KRAS variant in locally advanced rectal cancer: results of the EXPERT-C trial

## Metadata
**Authors:** F Sclafani, I Chau, D Cunningham, C Peckitt, A Lampis, JC Hahne, C Braconi, J Tabernero, B Glimelius, A Cervantes, R Begum, D Gonzalez De Castro, S Hulkki Wilson, Z Eltahir, A Wotherspoon, D Tait, G Brown, J Oates, N Valeri
**Journal:** Annals of Oncology
**Date:** 2015 Jul 10
**DOI:** [10.1093/annonc/mdv285](https://doi.org/10.1093/annonc/mdv285)
**PMID:** 26162609
**PMCID:** PMC4551162
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4551162/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC4551162/pdf/mdv285.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC4551162/pdf/mdv285.pdf)

## Abstract

KRAS mutation has been reported as a marker of radio-resistance in rectal cancer and unfavourable outcome in both colon and rectal cancer. This study suggests that a single-nucleotide polymorphism of the KRAS gene (LCS-6 variant) may predict response to neoadjuvant treatment and mitigate the poor prognosis associated with KRAS mutation in locally advanced rectal cancer.

Keywords: LCS-6 KRAS variant, single-nucleotide polymorphism, let-7, KRAS, cetuximab, rectal cancer

### Background

Lethal-7 (let-7) is a tumour suppressor miRNA which acts by down-regulating several oncogenes including *KRAS*. A single-nucleotide polymorphism (rs61764370, T > G base substitution) in the let-7 complementary site 6 (LCS-6) of *KRAS* mRNA has been shown to predict prognosis in early-stage colorectal cancer (CRC) and benefit from anti-epidermal growth factor receptor monoclonal antibodies in metastatic CRC.

### Patients and methods

We analysed rs61764370 in EXPERT-C, a randomised phase II trial of neoadjuvant CAPOX followed by chemoradiotherapy, surgery and adjuvant CAPOX plus or minus cetuximab in locally advanced rectal cancer. DNA was isolated from formalin-fixed paraffin-embedded tumour tissue and genotyped using a PCR-based commercially available assay. Kaplan–Meier method and Cox regression analysis were used to calculate survival estimates and compare treatment arms.

### Results

A total of 155/164 (94.5%) patients were successfully analysed, of whom 123 (79.4%) and 32 (20.6%) had the LCS-6 TT and LCS-6 TG genotype, respectively. Carriers of the G allele were found to have a statistically significantly higher rate of complete response (CR) after neoadjuvant therapy (28.1% versus 10.6%; *P* = 0.020) and a trend for better 5-year progression-free survival (PFS) [77.4% versus 64.5%: hazard ratio (HR) 0.56; *P* = 0.152] and overall survival (OS) rates (80.3% versus 71.9%: HR 0.59; *P* = 0.234). Both CR and survival outcomes were independent of the use of cetuximab. The negative prognostic effect associated with *KRAS* mutation appeared to be stronger in patients with the LCS-6 TT genotype (HR PFS 1.70, *P* = 0.078; HR OS 1.79, *P* = 0.082) compared with those with the LCS-6 TG genotype (HR PFS 1.33, *P* = 0.713; HR OS 1.01, *P* = 0.995).

### Conclusion

This analysis suggests that rs61764370 may be a biomarker of response to neoadjuvant treatment and an indicator of favourable outcome in locally advanced rectal cancer possibly by mitigating the poor prognosis of *KRAS* mutation. In this setting, however, this polymorphism does not appear to predict cetuximab benefit.

## introduction

miRNAs are short, non-coding, sequences of nucleotides which regulate gene expression by binding to complementary sites in the 3′-untranslated region (3′UTRs) of target mRNAs [[1](#MDV285C1)]. Approximately 2000 miRNAs have been described in humans so far and mounting evidence suggests that these molecules may play an important role in the mechanisms of cell proliferation, differentiation, carcinogenesis, tumour progression and response to treatment [[1](#MDV285C1)–[4](#MDV285C4)].

The lethal-7 (let-7) family members are among the most studied miRNAs in human malignancies. They generally act as tumour suppressors by down-regulating oncogenes involved in the control of the cell cycle or intracellular signalling cascades [[5](#MDV285C5)]. *KRAS* is an established target of let-7, several complementary sites for this miRNA being described in the 3′UTR of the mRNA [[6](#MDV285C6)]. A single-nucleotide polymorphism (SNP) (rs61764370, T > G base substitution) in the let-7 complementary site 6 (LCS-6) has been reported in ∼18% of Caucasians with colorectal cancer (CRC) [[7](#MDV285C7)]. This polymorphism modifies the let-7 binding affinity for *KRAS* ultimately leading to reduced *KRAS* inhibition and increased tumour proliferation [[8](#MDV285C8)].

A number of studies investigated the role of the LCS-6 variant either as a prognostic marker in early CRC or as a predictive marker for anti-epidermal growth factor receptor (EGFR) therapies in metastatic CRC [[7](#MDV285C7), [9](#MDV285C9)–[16](#MDV285C16)]. The results have been largely inconsistent possibly due to a significant inter-study heterogeneity with regard to sample size, patient characteristics and treatment. Notably, although the prognostic relevance of *KRAS* mutation appears greater in rectal cancer compared with colon cancer [[17](#MDV285C17), [18](#MDV285C18)], studies addressing the role of this polymorphism in a homogeneous series of rectal cancer patients are lacking.

We analysed the LCS-6 variant in EXPERT-C, an international, multicentre, randomised phase II trial investigating the addition of cetuximab to a sequential treatment with neoadjuvant capecitabine and oxaliplatin (CAPOX) followed by chemo-radiotherapy (CRT), surgery and adjuvant CAPOX in patients with locally advanced rectal cancer (LARC) [[19](#MDV285C19)].

## methods

The EXPERT-C trial included LARC patients with at least one of the following magnetic resonance imaging high-risk features: tumour within 1 mm of mesorectal fascia, T3 distal (at/below levators) tumour, T3c/T3d tumour (extramural extension ≥5 mm), T4 tumour, extramural vascular invasion [[19](#MDV285C19)]. Patients were randomised to four cycles of neoadjuvant CAPOX followed by capecitabine-based CRT, surgery and four cycles of adjuvant CAPOX or the same treatment plus cetuximab (Figure [1](#MDV285F1)) [[19](#MDV285C19)]. All patients provided written informed consent.

### Figure 1.

![Figure 1.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d384/4551162/1c9c835142df/mdv28501.jpg)

EXPERT-C trial design. R, randomisation; CAPOX, capecitabine and oxaliplatin; C, cetuximab; Cape, capecitabine; RT, radiotherapy.

### molecular analysis

DNA was isolated from formalin-fixed paraffin-embedded tumour tissue from pre-treatment biopsies and/or resection samples using the QIAamp DNA FFPE Tissue Kit (Qiagen, Hilden, Germany). Samples were genotyped using custom Taqman assay (Life Technologies, Carlsbad, CA) (probes available upon request). Cases, negative controls and duplicate samples were processed in a random order. Both inter- and intra-plate duplicates (10% of the samples) were 100% concordant. Analysis of *KRAS* (exons 2–4), *NRAS* (exons 2–4) and *BRAF* (codon 600) was carried out as previously described [[19](#MDV285C19), [20](#MDV285C20)].

### statistical considerations

The primary end point of the EXPERT-C trial was complete response (CR) in patients with *KRAS*/*BRAF* wild-type tumours. Hardy–Weinberg equilibrium was assessed using the *χ*^2^ test. The Kaplan–Meier method was used to calculate survival estimates and comparison between the treatment arms was carried out using a log-rank analysis. Hazard ratios (HRs) and 95% confidence intervals (CIs) were obtained from Cox regression. An interaction term between treatment arm and LCS-6 genotype was included in the Cox regression. Multivariate Cox regression was used to assess whether a significant interaction remained significant after addition of prognostic variables. Variables were included in the multivariate model using forward selection if *P* value <0.1.

## results

One hundred and sixty-four patients were enrolled into the EXPERT-C trial. Of these, 155 (94.5%) had tumour tissue available for LCS-6 genotyping, 77 in the CAPOX-C arm and 78 in the CAPOX arm. Table [1](#MDV285TB1) shows patient characteristics. No significant differences, overall and by treatment arm, were observed compared with the original EXPERT-C trial population (data not shown).

### Table 1.

|   | LCS6 TT genotype (N = 123) |  | LCS6 TG genotype (N = 32) |  | P value |
| --- | --- | --- | --- | --- | --- |
| n | % | n | % |  |  |
| Gender |  |  |  |  |  |
| Male | 75 | 61.0 | 19 | 59.4 | 0.869 |
| Female | 48 | 39.0 | 13 | 40.6 |   |
| Age [mean (SD) and range] | 60 (10.5) | 38–74 | 61 (11.1) | 28–79 | 0.084 |
| WHO performance status |  |  |  |  |  |
| 0 | 57 | 46.3 | 16 | 50.0 | 0.712 |
| ≥1 | 66 | 53.7 | 16 | 50.0 |   |
| MRI-defined high-risk features |  |  |  |  |  |
| T3c–T3d | 81 | 65.9 | 17 | 53.1 | 0.183 |
| T4 | 29 | 23.6 | 10 | 31/3 | 0.373 |
| CRM involved/at risk | 73 | 59.3 | 16 | 50.0 | 0.341 |
| EMVI positive | 90 | 73.2 | 24 | 75.0 | 0.834 |
| Low-lying tumour | 87 | 70.7 | 27 | 84.4 | 0.119 |
| Mutations |  |  |  |  |  |
| KRAS | 49 | 41.5 | 17 | 54.8 | 0.184 |
| NRAS | 5 | 4.3 | 1 | 3.2 | 1.00 |
| KRAS/NRAS | 54 | 45.8 | 18 | 58.1 | 0.223 |
| BRAF | 2 | 1.7 | 2 | 6.5 | 0.191 |
| PIK3CA | 8 | 6.8 | 2 | 6.5 | 1.00 |
| TP53 | 69 | 52.6 | 15 | 50.0 | 0.797 |

Table 1 Caption: Baseline patient characteristics by LCS6 genotype

Genotyping was successful in all assessable patients. One hundred and twenty-three patients (79.4%) had the LCS-6 TT genotype (CAPOX = 65; CAPOX-C = 58) while 32 (20.6%) had the LCS-6 TG genotype (CAPOX = 13; CAPOX-C = 19). Hardy–Weinberg equilibrium was observed (*P* = 0.152). There was no association between the LCS-6 genotype and baseline characteristics including demographics and clinico-pathological features. More patients in the LCS-6 variant group had tumours harbouring *KRAS* (54.8% versus 41.5%), *KRAS/NRAS* (58.1% versus 45.8%) and *BRAF* mutation (6.5% versus 1.7%). These differences however were not statistically significant.

After neoadjuvant treatment, 13/123 patients (10.6%) in the LCS-6 TT genotype group achieved CR compared with 9/32 (28.1%) in the LCS-6 TG genotype group (*P* = 0.020, adjusted *P* = 0.044). In both groups, no significant differences in CR rate were observed with or without cetuximab (10.3% versus 10.8%, *P* = 1.00 and 31.6% versus 23.1%, *P* = 0.704, respectively, treatment*LCS-6 *P* interaction = 0.638) or in *KRAS* wild-type versus *KRAS* mutant tumours (10.1% versus 8.2%, *P* = 1.00 and 21.4% and 29.4%, *P* = 0.698, *KRAS**LCS-6 *P* interaction = 0.534).

After a median follow-up of 64.9 months (95% CI 62.8–67.2), numerically higher 5-year progression-free survival (PFS) [77.4% versus 64.5%, HR 0.56 (95% CI 0.25–1.24), *P* = 0.152] and 5-year overall survival (OS) rates [80.3% versus 71.9%, HR 0.59 (95% CI 0.25–1.41), *P* = 0.234] favouring the LCS-6 TG genotype group were observed in the entire population (Figure [2](#MDV285F2)). In cetuximab-treated patients, survival outcomes were independent of the genotype group. The 5-year PFS and OS rates in patients with the LCS-6 TT genotype treated in the CAPOX-C arm were 66.7% and 77.0%, respectively, compared with 62.7% and 67.4% in patients with the same genotype treated in the CAPOX arm [HR PFS 0.78 (95% CI 0.43–1.42) *P* = 0.420; HR OS 0.62 (95% CI 0.32–1.21) *P* = 0.159]. The 5-year PFS and OS rates in patients with the LCS-6 TG genotype treated in the CAPOX-C arm were 78.9% and 83.9%, respectively, compared with 75.0% and 75.0% in patients with the same genotype treated in the CAPOX arm [HR PFS 0.80 (95% CI 0.19–3.70) *P* = 0.804; HR OS 0.65 (95% CI 0.13–3.22) *P* = 0.597] (Figure [3](#MDV285F3)). No interaction between cetuximab and LCS-6 genotype was observed for both PFS (*P* = 0.937) and OS (*P* = 0.973).

### Figure 2.

![Figure 2.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d384/4551162/80d43f2779b7/mdv28502.jpg)

Progression-free survival (A) and overall survival (B) by LCS6 genotype in the entire study population. HR, hazard ratio; CI, confidence interval.

### Figure 3.

![Figure 3.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d384/4551162/eaa59e49bf8c/mdv28503.jpg)

Progression-free survival (A) and overall survival (B) by LCS6 genotype and treatment arm. a, patients with LCS6 TT genotype treated in the CAPOX arm; b, patients with LCS6 TT genotype treated in the CAPOX-C arm; c, patients with LCS6 TG genotype treated in the CAPOX arm; d, patients with LCS6 TG genotype treated in the CAPOX-C arm. HR, hazard ratio; CI, confidence interval.

Thirty-one patients (25.2%) in the LCS-6 TT genotype group and 5 patients (15.6%) in the LCS-6 TG genotype group had tumour recurrence. The most common sites of disease recurrence were liver (41.7%), lung (44.4%), peritoneum (19.4%) and lymph nodes (16.7%). The rate of liver relapse was different between the two groups: 15 patients out of 123 (12.2%) in the LCS-6 TT genotype group were diagnosed with liver metastases (accounting for 48.4% of all relapsed patients in this group) compared with 0/32 patients with the LCS-6 TG genotype (*P* = 0.038).

When the survival outcomes were analysed by *KRAS* status, the prognostic trend associated with the LCS-6 TG genotype appeared to be stronger in patients with *KRAS* mutant tumours [5-year PFS 75.0% versus 54.5%, HR 0.49 (95% CI 0.17–1.43), *P* = 0.192; 5-year OS 81.3% versus 62.4%, HR 0.42 (95% CI 0.12–1.41), *P* = 0.158] compared with patients with *KRAS* wild-type tumours [5-year PFS 78.6% versus 69.0%, HR 0.59 (95% CI 0.17–1.96), *P* = 0.385; 5-year OS 78.6% versus 76.5%, HR 0.78 (95% CI 0.23–2.69), *P* = 0.703] (Figure [4](#MDV285F4)). *KRAS* mutation was found to have a negative prognostic impact in patients with the LCS-6 TT genotype [HR PFS: 1.70 (95% CI 0.94–3.08), *P* = 0.078; HR OS: 1.79 (95% CI 0.93–3.44), *P* = 0.082] but not in those with the LCS-6 TG genotype [HR PFS: 1.33 (95% CI 0.30–5.92), *P* = 0.713; HR OS: 1.01 (95% CI 0.20–4.99), *P* = 0.995]. However, possibly due to the small numbers, the interaction test did not show any interaction between the LCS-6 genotype and *KRAS* status for both PFS (*P* = 0.765) and OS (*P* = 0.473). Similar results were observed in the analysis by *RAS* (i.e. *KRAS* and *NRAS*) or *RAS/BRAF* status (data not shown).

### Figure 4.

![Figure 4.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d384/4551162/aead401d8adf/mdv28504.jpg)

Progression-free survival (A) and overall survival (B) by LCS6 genotype and KRAS mutational status. a, patients with LCS6 TT genotype and KRAS wild-type tumour; b, patients with LCS6 TT genotype and KRAS mutant tumour; c, patients with LCS6 TG genotype and KRAS wild-type tumour; d, patients with LCS6 TG genotype and KRAS mutant tumour. HR, hazard ratio; CI, confidence interval.

## discussion

In this retrospective analysis of the EXPERT-C trial, we showed that rs61764370 was not a predictive factor for cetuximab in LARC. However, this polymorphic variant was associated with a higher rate of CR to neoadjuvant therapy and a trend towards better survival outcomes, especially in the subgroup of patients with *KRAS* mutant tumours.

To our knowledge, this is the first report on the role of the LCS-6 variant in a homogeneous series of LARC. Previous studies investigating this SNP were either restricted to patients with colon cancer or conducted in unselected CRC populations [[7](#MDV285C7), [9](#MDV285C9)–[16](#MDV285C16)]. However, differences exist between colon and rectal cancers with regards to tumour biology and treatment approach including frequency of microsatellite instability and *BRAF* mutation, prognostic relevance of *KRAS* mutation, miRNA expression profile and routine use of radiotherapy in LARC [[21](#MDV285C21)].

We assessed the role of the LCS-6 variant with an aim to validate two intriguing hypotheses generated by previous analyses. The first hypothesis is that this variant may predict benefit from anti-EGFR agents and refine the selection of metastatic CRC patients who are candidate for these therapies [[16](#MDV285C16)]. The second hypothesis is that the LCS-6 G allele may be a favourable prognostic factor in the non-metastatic setting and be used in the decision-making process regarding adjuvant treatment [[14](#MDV285C14)]. To this end, we used a prospective series of LARC patients who were treated with systemic chemotherapy followed by CRT plus or minus cetuximab in a randomised phase II trial [[19](#MDV285C19)].

The incidence of the LCS-6 variant in our population was in line with what has been previously reported [[7](#MDV285C7), [16](#MDV285C16)]. We could not find an association between this polymorphism and patient characteristics. Of note, our study is the first to include data on *RAS* (*KRAS* and *NRAS*, exon 2–4) mutation in this setting and, although a higher incidence of *RAS* and *BRAF* mutation was observed in carriers of the G allele, this was not statistically significant.

The main and novel finding of our analysis is that patients with the LCS-6 TG genotype had a higher rate of CR after preoperative treatment. This translated into numerically higher, but not statistically significantly improved, survival outcomes possibly due to the relatively small sample size, the low number of events and the limited statistical power of the study. Of note, the better prognosis of the LCS-6 variant group was independent of cetuximab. These results do not confirm, at least in the setting of LARC, an association between LCS-6 genotype and benefit from anti-EGFR agents. However, they suggest that the LCS-6 variant may act as a prognostic factor possibly by modulating the cytotoxic effects of CRT. In support of this contention, preclinical studies showed that the expression of let-7 changed in response to irradiation. Moreover, manipulating the expression of specific let-7 miRNAs was associated with radiosensitivity (i.e. when increasing let-7a or let-7b levels) or radio-resistance (i.e. when reducing let-7 g levels) in *RAS* mutant pancreatic and lung cancer cell lines [[22](#MDV285C22)–[24](#MDV285C24)]. We acknowledge that our results are hypothesis-generating. A better understanding of the relationship between let-7 and radiotherapy as well as validation of our findings in independent series is certainly needed. Also, it should be noted that both treatment arms of the EXPERT-C trial were investigational in that oxaliplatin-based chemotherapy was administered before CRT. Given the absence of a control group treated with standard fluoropyrimidine-based CRT only, we cannot rule out that the improved outcome of the G carriers could be secondary to increased sensitivity to induction systemic chemotherapy. In this regard, it has been previously shown that, in tumour cells harbouring the LCS-6 variant allele, the functional effects of chemotherapy (as measured by KRAS expression) were different among cytotoxic agents [[16](#MDV285C16)].

In subgroup analyses by *KRAS* status, we observed that the favourable prognostic effect of the LCS-6 variant was limited to patients with *KRAS* mutant tumours. Interestingly, this finding is consistent with previous reports. In early-stage CRC patients, Smits et al. showed that G carriers with stage I–II tumours harbouring *KRAS* mutation had a better cancer-specific survival compared with *KRAS* mutant patients with the LCS-6 TT genotype [[14](#MDV285C14)]. In contrast, patients with *KRAS* wild-type tumours had intermediate prognosis regardless of the LCS-6 genotype. An association between LCS-6 G allele and better prognosis was also reported in a series of stage III–IV CRC patients enriched with *KRAS* mutation [[15](#MDV285C15)]. Finally, in a recent analysis of the NCCTG N0147 trial, the outcome of G carriers with *KRAS* mutant stage III colon cancer appeared to be more similar to that of patients with *KRAS* wild-type tumours than that of patients with LCS-6 TT genotype and *KRAS* mutant tumour [[16](#MDV285C16)]. Altogether, these data seem to indicate that the LCS-6 variant may mitigate the unfavourable prognosis associated with *KRAS* mutation in the non-metastatic setting [[17](#MDV285C17), [18](#MDV285C18)]. It has been proposed that this effect may be secondary to induction of cellular senescence through overexpression of KRAS and increased signalling through the MAP-K cascade [[14](#MDV285C14)]. However, there are currently no data to confirm this hypothesis and the broad spectrum of cancer-related genes which are regulated by let-7 suggests that other mechanisms may be involved.

We recognise the limitations of our study including the retrospective design, the small numbers and the analysis of patients who were treated with investigational therapeutic strategies that do not reflect the current standard of care in this setting. Moreover, robust biological hypotheses to explain the study results are lacking. However, this analysis explores for the first time the predictive and prognostic role of the LCS-6 variant in LARC and provides another piece of the puzzle on the relationship between this SNP and anti-EGFR agents.

The management of LARC is orphan of established biomarkers that could lead to optimisation of patient selection, implementation of molecularly selected treatment approaches and improved outcomes. So far, studies investigating putative biomarkers in this setting have been largely unsuccessful. As a result, conventional clinico-pathological prognostic factors still remain the only available tools for individual patient risk assessment and treatment decision. Although genetic variations associated with SNPs have been reported to influence cancer risk, response to treatment and tumour prognosis in a number of tumour types including CRC, their value in routine practice is yet to be demonstrated.

Our analysis suggests that, in a Caucasian population, the rs61764370 SNP may influence the prognostic relevance of *KRAS* mutation which has been increasingly reported as a marker of resistance to CRT in LARC and poor prognosis in distal colon cancer and rectal cancer [[17](#MDV285C17), [18](#MDV285C18)]. If our findings are confirmed, testing for *KRAS* and rs61764370 could potentially provide useful data for patient stratification in clinical trials of (neo)adjuvant treatment of LARC. Further studies to elucidate the mechanisms whereby this SNP may increase tumour (chemo)radiosensitivity and mitigate the unfavourable prognosis of *KRAS* mutant tumours are warranted.

## funding

The work was supported by Cancer Research UK (CEA A18052), European Union FP7 (CIG 334261) and the National Institute for Health Research (NIHR) Biomedical Research Centre (BRC) at The Royal Marsden NHS Foundation Trust and The Institute of Cancer Research (grant A62) to NV.

## disclosure

DC received research funding from: Roche, Amgen, Celgene, Sanofi, Merck-Serono, Novartis, AstraZeneca, Bayer, Merrimack and MedImmune. CP has had advisory roles with Sanofi. JT has had advisory roles with Amgen, Roche, Sanofi-Aventis and Merck. AC has had advisory roles with Merck-Serono and Roche. He has received research funding from Roche and honoraria from Roche and Merck-Serono. IC has had advisory roles with Merck-Serono, Roche, Sanofi Oncology, Bristol Myers Squibb, Eli-Lilly, Novartis, Gilead Science. He has received research funding from Merck-Serono, Novartis, Roche and Sanofi Oncology, and honoraria from Roche, Sanofi Oncology, Eli-Lilly, Taiho. All other authors declare that they have no conflicts of interest.

## acknowledgements

We acknowledge support from the NIHR BRC at The Royal Marsden NHS Foundation Trust and The Institute of Cancer Research and from the Peter Stebbings Memorial Charity. We thank Brid M. Ryan (Laboratory of Human Carcinogenesis, National Cancer Institute, Bethesda, USA) for technical advice on probe design. The EXPERT-C trial was supported by the NIHR BRC at The Royal Marsden NHS Foundation Trust and The Institute of Cancer Research, the Peter Stebbings Memorial Charity, and the Pelican Cancer Foundation and by a grant from Merck & Co. Merck & Co. provided cetuximab, and Sanofi-Aventis provided oxaliplatin; neither company was involved in study design, data analysis, or manuscript preparation or had access to study data. The EXPERT-C trial was endorsed by Cancer Research UK.

## References

1. Meltzer PS. Cancer genomics: small RNAs with big impacts. Nature 2005; 435: 745–746.  [DOI](https://doi.org/10.1038/435745a) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15944682/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Cancer%20genomics:%20small%20RNAs%20with%20big%20impacts&author=PS%20Meltzer&volume=435&publication_year=2005&pages=745-746&pmid=15944682&doi=10.1038/435745a&)

2. Valeri N, Braconi C, Gasparini P et al. MicroRNA-135b promotes cancer progression by acting as a downstream effector of oncogenic pathways in colon cancer. Cancer Cell 2014; 25: 469–483.  [DOI](https://doi.org/10.1016/j.ccr.2014.03.006) | [PMC free article](/articles/PMC3995091/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24735923/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Cell&title=MicroRNA-135b%20promotes%20cancer%20progression%20by%20acting%20as%20a%20downstream%20effector%20of%20oncogenic%20pathways%20in%20colon%20cancer&author=N%20Valeri&author=C%20Braconi&author=P%20Gasparini&volume=25&publication_year=2014&pages=469-483&pmid=24735923&doi=10.1016/j.ccr.2014.03.006&)

3. Valeri N, Gasparini P, Braconi C et al. MicroRNA-21 induces resistance to 5-fluorouracil by down-regulating human DNA MutS homolog 2 (hMSH2). Proc Natl Acad Sci USA 2010; 107: 21098–21103.  [DOI](https://doi.org/10.1073/pnas.1015541107) | [PMC free article](/articles/PMC3000294/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21078976/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20USA&title=MicroRNA-21%20induces%20resistance%20to%205-fluorouracil%20by%20down-regulating%20human%20DNA%20MutS%20homolog%202%20(hMSH2)&author=N%20Valeri&author=P%20Gasparini&author=C%20Braconi&volume=107&publication_year=2010&pages=21098-21103&pmid=21078976&doi=10.1073/pnas.1015541107&)

4. Kozomara A, Griffiths-Jones S. miRBase: annotating high confidence microRNAs using deep sequencing data. Nucleic Acids Res 2014; 42: D68–D73.  [DOI](https://doi.org/10.1093/nar/gkt1181) | [PMC free article](/articles/PMC3965103/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24275495/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=miRBase:%20annotating%20high%20confidence%20microRNAs%20using%20deep%20sequencing%20data&author=A%20Kozomara&author=S%20Griffiths-Jones&volume=42&publication_year=2014&pages=D68-D73&pmid=24275495&doi=10.1093/nar/gkt1181&)

5. Su JL, Chen PS, Johansson G, Kuo ML. Function and regulation of let-7 family microRNAs. MicroRNA 2012; 1: 34–39.  [DOI](https://doi.org/10.2174/2211536611201010034) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25048088/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=MicroRNA&title=Function%20and%20regulation%20of%20let-7%20family%20microRNAs&author=JL%20Su&author=PS%20Chen&author=G%20Johansson&author=ML%20Kuo&volume=1&publication_year=2012&pages=34-39&pmid=25048088&doi=10.2174/2211536611201010034&)

6. Johnson SM, Grosshans H, Shingara J et al. RAS is regulated by the let-7 microRNA family. Cell 2005; 120: 635–647.  [DOI](https://doi.org/10.1016/j.cell.2005.01.014) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15766527/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell&title=RAS%20is%20regulated%20by%20the%20let-7%20microRNA%20family&author=SM%20Johnson&author=H%20Grosshans&author=J%20Shingara&volume=120&publication_year=2005&pages=635-647&pmid=15766527&doi=10.1016/j.cell.2005.01.014&)

7. Sha D, Lee AM, Shi Q et al. Association study of the let-7 miRNA-complementary site variant in the 3′ untranslated region of the KRAS gene in stage III colon cancer (NCCTG N0147 Clinical Trial). Clin Cancer Res 2014; 20: 3319–3327.  [DOI](https://doi.org/10.1158/1078-0432.CCR-14-0069) | [PMC free article](/articles/PMC4084689/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24727325/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&title=Association%20study%20of%20the%20let-7%20miRNA-complementary%20site%20variant%20in%20the%203%E2%80%B2%20untranslated%20region%20of%20the%20KRAS%20gene%20in%20stage%20III%20colon%20cancer%20(NCCTG%20N0147%20Clinical%20Trial)&author=D%20Sha&author=AM%20Lee&author=Q%20Shi&volume=20&publication_year=2014&pages=3319-3327&pmid=24727325&doi=10.1158/1078-0432.CCR-14-0069&)

8. Johnson CD, Esquela-Kerscher A, Stefani G et al. The let-7 microRNA represses cell proliferation pathways in human cells. Cancer Res 2007; 67: 7713–7722.  [DOI](https://doi.org/10.1158/0008-5472.CAN-07-1083) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17699775/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=The%20let-7%20microRNA%20represses%20cell%20proliferation%20pathways%20in%20human%20cells&author=CD%20Johnson&author=A%20Esquela-Kerscher&author=G%20Stefani&volume=67&publication_year=2007&pages=7713-7722&pmid=17699775&doi=10.1158/0008-5472.CAN-07-1083&)

9. Graziano F, Canestrari E, Loupakis F et al. Genetic modulation of the Let-7 microRNA binding to KRAS 3′-untranslated region and survival of metastatic colorectal cancer patients treated with salvage cetuximab-irinotecan. Pharmacogenomics J 2010; 10: 458–464.  [DOI](https://doi.org/10.1038/tpj.2010.9) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20177422/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=Genetic%20modulation%20of%20the%20Let-7%20microRNA%20binding%20to%20KRAS%203%E2%80%B2-untranslated%20region%20and%20survival%20of%20metastatic%20colorectal%20cancer%20patients%20treated%20with%20salvage%20cetuximab-irinotecan&author=F%20Graziano&author=E%20Canestrari&author=F%20Loupakis&volume=10&publication_year=2010&pages=458-464&pmid=20177422&doi=10.1038/tpj.2010.9&)

10. Zhang W, Winder T, Ning YHJ et al. A let-7 microRNA-binding site polymorphism in 3′-untranslated region of KRAS gene predicts response in wild-type KRAS patients with metastatic colorectal cancer treated with cetuximab monotherapy. Ann Oncol 2011; 22: 104–109.  [DOI](https://doi.org/10.1093/annonc/mdq315) | [PMC free article](/articles/PMC8890483/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20603437/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Oncol&title=A%20let-7%20microRNA-binding%20site%20polymorphism%20in%203%E2%80%B2-untranslated%20region%20of%20KRAS%20gene%20predicts%20response%20in%20wild-type%20KRAS%20patients%20with%20metastatic%20colorectal%20cancer%20treated%20with%20cetuximab%20monotherapy&author=W%20Zhang&author=T%20Winder&author=YHJ%20Ning&volume=22&publication_year=2011&pages=104-109&pmid=20603437&doi=10.1093/annonc/mdq315&)

11. Kjersem JB, Ikdahl T, Guren T et al. Let-7 miRNA-binding site polymorphism in the KRAS 3′UTR; colorectal cancer screening population prevalence and influence on clinical outcome in patients with metastatic colorectal cancer treated with 5-fluorouracil and oxaliplatin +/- cetuximab. BMC Cancer 2012; 12: 534.  [DOI](https://doi.org/10.1186/1471-2407-12-534) | [PMC free article](/articles/PMC3526507/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23167843/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Cancer&title=Let-7%20miRNA-binding%20site%20polymorphism%20in%20the%20KRAS%203%E2%80%B2UTR;%20colorectal%20cancer%20screening%20population%20prevalence%20and%20influence%20on%20clinical%20outcome%20in%20patients%20with%20metastatic%20colorectal%20cancer%20treated%20with%205-fluorouracil%20and%20oxaliplatin%20+/-%20cetuximab&author=JB%20Kjersem&author=T%20Ikdahl&author=T%20Guren&volume=12&publication_year=2012&pages=534&pmid=23167843&doi=10.1186/1471-2407-12-534&)

12. Sebio A, Pare L, Paez D et al. The LCS6 polymorphism in the binding site of let-7 microRNA to the KRAS 3′-untranslated region: its role in the efficacy of anti-EGFR-based therapy in metastatic colorectal cancer patients. Pharmacogenet Genomics 2013; 23: 142–147.  [DOI](https://doi.org/10.1097/FPC.0b013e32835d9b0b) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23324806/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=The%20LCS6%20polymorphism%20in%20the%20binding%20site%20of%20let-7%20microRNA%20to%20the%20KRAS%203%E2%80%B2-untranslated%20region:%20its%20role%20in%20the%20efficacy%20of%20anti-EGFR-based%20therapy%20in%20metastatic%20colorectal%20cancer%20patients&author=A%20Sebio&author=L%20Pare&author=D%20Paez&volume=23&publication_year=2013&pages=142-147&pmid=23324806&doi=10.1097/FPC.0b013e32835d9b0b&)

13. Winder T, Zhang W, Khoueiry AE et al. Association of a germline variant in the K-ras 3′ untranslated region with response and progression-free survival in patients with mCRC treated with single-agent cetuximab (IMCL-0144) or in combination with cetuximab (EPIC) independent of K-ras mutation status. J Clin Oncol 2009; 27: 15s (suppl); abstr 4061.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Association%20of%20a%20germline%20variant%20in%20the%20K-ras%203%E2%80%B2%20untranslated%20region%20with%20response%20and%20progression-free%20survival%20in%20patients%20with%20mCRC%20treated%20with%20single-agent%20cetuximab%20(IMCL-0144)%20or%20in%20combination%20with%20cetuximab%20(EPIC)%20independent%20of%20K-ras%20mutation%20status&author=T%20Winder&author=W%20Zhang&author=AE%20Khoueiry&volume=27&publication_year=2009&pages=15s&)

14. Smits KM, Paranjape T, Nallur S et al. A let-7 microRNA SNP in the KRAS 3′UTR is prognostic in early-stage colorectal cancer. Clin Cancer Res 2011; 17: 7723–7731.  [DOI](https://doi.org/10.1158/1078-0432.CCR-11-0990) | [PMC free article](/articles/PMC5919188/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21994416/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&title=A%20let-7%20microRNA%20SNP%20in%20the%20KRAS%203%E2%80%B2UTR%20is%20prognostic%20in%20early-stage%20colorectal%20cancer&author=KM%20Smits&author=T%20Paranjape&author=S%20Nallur&volume=17&publication_year=2011&pages=7723-7731&pmid=21994416&doi=10.1158/1078-0432.CCR-11-0990&)

15. Ryan BM, Robles AI, Harris CC. KRAS-LCS6 genotype as a prognostic marker in early stage CRC-letter. Clin Cancer Res 2012; 18: 3487–3488.  [DOI](https://doi.org/10.1158/1078-0432.CCR-12-0250) | [PMC free article](/articles/PMC4132878/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22669132/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&title=KRAS-LCS6%20genotype%20as%20a%20prognostic%20marker%20in%20early%20stage%20CRC-letter&author=BM%20Ryan&author=AI%20Robles&author=CC%20Harris&volume=18&publication_year=2012&pages=3487-3488&pmid=22669132&doi=10.1158/1078-0432.CCR-12-0250&)

16. Saridaki Z, Weidhaas JB, Lenz HJ et al. A let-7 microRNA-binding site polymorphism in KRAS predicts improved outcome in patients with metastatic colorectal cancer treated with salvage cetuximab/panitumumab monotherapy. Clin Cancer Res 2014; 20: 4499–4510.  [DOI](https://doi.org/10.1158/1078-0432.CCR-14-0348) | [PMC free article](/articles/PMC4155520/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25183481/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&title=A%20let-7%20microRNA-binding%20site%20polymorphism%20in%20KRAS%20predicts%20improved%20outcome%20in%20patients%20with%20metastatic%20colorectal%20cancer%20treated%20with%20salvage%20cetuximab/panitumumab%20monotherapy&author=Z%20Saridaki&author=JB%20Weidhaas&author=HJ%20Lenz&volume=20&publication_year=2014&pages=4499-4510&pmid=25183481&doi=10.1158/1078-0432.CCR-14-0348&)

17. Hutchins G, Southward K, Handley K et al. Value of mismatch repair, KRAS, and BRAF mutations in predicting recurrence and benefits from chemotherapy in colorectal cancer. J Clin Oncol 2011; 29: 1261–1270.  [DOI](https://doi.org/10.1200/JCO.2010.30.1366) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21383284/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Value%20of%20mismatch%20repair,%20KRAS,%20and%20BRAF%20mutations%20in%20predicting%20recurrence%20and%20benefits%20from%20chemotherapy%20in%20colorectal%20cancer&author=G%20Hutchins&author=K%20Southward&author=K%20Handley&volume=29&publication_year=2011&pages=1261-1270&pmid=21383284&doi=10.1200/JCO.2010.30.1366&)

18. Sinicrope FA, Yoon HH, Mahoney MR et al. Overall survival result and outcomes by KRAS, BRAF, and DNA mismatch repair in relation to primary tumor site in colon cancers from a randomized trial of adjuvant chemotherapy: NCCTG (Alliance) N0147. J Clin Oncol 2014; 32: 5s (suppl); abstr 3525.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Overall%20survival%20result%20and%20outcomes%20by%20KRAS,%20BRAF,%20and%20DNA%20mismatch%20repair%20in%20relation%20to%20primary%20tumor%20site%20in%20colon%20cancers%20from%20a%20randomized%20trial%20of%20adjuvant%20chemotherapy:%20NCCTG%20(Alliance)%20N0147&author=FA%20Sinicrope&author=HH%20Yoon&author=MR%20Mahoney&volume=32&issue=(suppl)&publication_year=2014&pages=5s&)

19. Dewdney A, Cunningham D, Tabernero J et al. Multicenter randomized phase II clinical trial comparing neoadjuvant oxaliplatin, capecitabine, and preoperative radiotherapy with or without cetuximab followed by total mesorectal excision in patients with high-risk rectal cancer (EXPERT-C). J Clin Oncol 2012; 30: 1620–1627.  [DOI](https://doi.org/10.1200/JCO.2011.39.6036) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22473163/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Multicenter%20randomized%20phase%20II%20clinical%20trial%20comparing%20neoadjuvant%20oxaliplatin,%20capecitabine,%20and%20preoperative%20radiotherapy%20with%20or%20without%20cetuximab%20followed%20by%20total%20mesorectal%20excision%20in%20patients%20with%20high-risk%20rectal%20cancer%20(EXPERT-C)&author=A%20Dewdney&author=D%20Cunningham&author=J%20Tabernero&volume=30&publication_year=2012&pages=1620-1627&pmid=22473163&doi=10.1200/JCO.2011.39.6036&)

20. Sclafani F, Gonzalez D, Cunningham D et al. RAS mutations and cetuximab in locally advanced rectal cancer: results of the EXPERT-C trial. Eur J Cancer 2014; 50: 1430–1436.  [DOI](https://doi.org/10.1016/j.ejca.2014.02.002) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24582914/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Cancer&title=RAS%20mutations%20and%20cetuximab%20in%20locally%20advanced%20rectal%20cancer:%20results%20of%20the%20EXPERT-C%20trial&author=F%20Sclafani&author=D%20Gonzalez&author=D%20Cunningham&volume=50&publication_year=2014&pages=1430-1436&pmid=24582914&doi=10.1016/j.ejca.2014.02.002&)

21. Hong TS, Clark JW, Haigis KM. Cancers of the colon and rectum: identical or fraternal twins? Cancer Discov 2012; 2: 117–121.  [DOI](https://doi.org/10.1158/2159-8290.CD-11-0315) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22585856/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Discov&title=Cancers%20of%20the%20colon%20and%20rectum:%20identical%20or%20fraternal%20twins?&author=TS%20Hong&author=JW%20Clark&author=KM%20Haigis&volume=2&publication_year=2012&pages=117-121&pmid=22585856&doi=10.1158/2159-8290.CD-11-0315&)

22. Weidhaas JB, Babar I, Nallur S et al. MicroRNAs as potential agents to alter resistance to cytotoxic anticancer therapy. Cancer Res 2007; 67: 11111–11116.  [DOI](https://doi.org/10.1158/0008-5472.CAN-07-2858) | [PMC free article](/articles/PMC6070379/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18056433/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=MicroRNAs%20as%20potential%20agents%20to%20alter%20resistance%20to%20cytotoxic%20anticancer%20therapy&author=JB%20Weidhaas&author=I%20Babar&author=S%20Nallur&volume=67&publication_year=2007&pages=11111-11116&pmid=18056433&doi=10.1158/0008-5472.CAN-07-2858&)

23. Simone NL, Soule BP, Ly D et al. Ionizing radiation-induced oxidative stress alters miRNA expression. PLoS One 2009; 4: e6377.  [DOI](https://doi.org/10.1371/journal.pone.0006377) | [PMC free article](/articles/PMC2712071/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19633716/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=Ionizing%20radiation-induced%20oxidative%20stress%20alters%20miRNA%20expression&author=NL%20Simone&author=BP%20Soule&author=D%20Ly&volume=4&publication_year=2009&pages=e6377&pmid=19633716&doi=10.1371/journal.pone.0006377&)

24. Oh JS, Kim JJ, Byun JY, Kim IA. Lin28-Let7 modulates radiosensitivity of human cancer cells with activation of K-RAS. Int J Radiat Oncol Biol Phys 2010; 76: 5–8.  [DOI](https://doi.org/10.1016/j.ijrobp.2009.08.028) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20005451/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Radiat%20Oncol%20Biol%20Phys&title=Lin28-Let7%20modulates%20radiosensitivity%20of%20human%20cancer%20cells%20with%20activation%20of%20K-RAS&author=JS%20Oh&author=JJ%20Kim&author=JY%20Byun&author=IA%20Kim&volume=76&publication_year=2010&pages=5-8&pmid=20005451&doi=10.1016/j.ijrobp.2009.08.028&)
