# Genetic variants associated with response to lithium treatment in bipolar disorder: a genome-wide association study

## Metadata
**Authors:** Liping Hou, Urs Heilbronner, Franziska Degenhardt, Mazda Adli, Kazufumi Akiyama, Nirmala Akula, Raffaella Ardau, Bárbara Arias, Lena Backlund, Claudio EM Banzato, Antoni Benabarre, Susanne Bengesser, Abesh Kumar Bhattacharjee, Joanna M Biernacka, Armin Birner, Clara Brichant-Petitjean, Elise T Bui, Pablo Cervantes, Guo-Bo Chen, Hsi-Chung Chen, Caterina Chillotti, Sven Cichon, Scott R Clark, Francesc Colom, David Cousins, Cristiana Cruceanu, Piotr M Czerski, Clarissa R Dantas, Alexandre Dayer, Bruno Étain, Peter Falkai, Andreas J Forstner, Louise Frisén, Janice M Fullerton, Sébastien Gard, Julie S Garnham, Fernando S Goes, Paul Grof, Oliver Gruber, Ryota Hashimoto, Joanna Hauser, Stefan Herms, Per Hoffmann, Andrea Hofmann, Stephane Jamain, Esther Jiménez, Jean-Pierre Kahn, Layla Kassem, Sarah Kittel-Schneider, Sebastian Kliwicki, Barbara König, Ichiro Kusumi, Nina Lackner, Gonzalo Laje, Mikael Landén, Catharina Lavebratt, Marion Leboyer, Susan G Leckband, Carlos A López Jaramillo, Glenda MacQueen, Mirko Manchia, Lina Martinsson, Manuel Mattheisen, Michael J McCarthy, Susan McElroy, Marina Mitjans, Francis M Mondimore, Palmiero Monteleone, Caroline M Nievergelt, Markus M Nöthen, Urban Ösby, Norio Ozaki, Roy H Perlis, Andrea Pfennig, Daniela Reich-Erkelenz, Guy A Rouleau, Peter R Schofield, K Oliver Schubert, Barbara W Schweizer, Florian Seemüller, Giovanni Severino, Tatyana Shekhtman, Paul D Shilling, Kazutaka Shimoda, Christian Simhandl, Claire M Slaney, Jordan W Smoller, Alessio Squassina, Thomas Stamm, Pavla Stopkova, Sarah K Tighe, Alfonso Tortorella, Gustavo Turecki, Julia Volkert, Stephanie Witt, Adam Wright, L Trevor Young, Peter P Zandi, James B Potash, Jay Raymond DePaulo
**Journal:** Lancet (London, England)
**Date:** 2016 Jan 22
**DOI:** [10.1016/S0140-6736(16)00143-4](https://doi.org/10.1016/S0140-6736(16)00143-4)
**PMID:** 26806518
**PMCID:** PMC4814312
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4814312/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC4814312/pdf/nihms755576.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC4814312/pdf/nihms755576.pdf)

## Abstract

**Background:** 
Lithium remains a first-line treatment in bipolar disorder, but individual response is variable. Previous studies have suggested that lithium response is a heritable trait. However, no genetic markers have been reproducibly identified.

**Methods:** 
Here we report the results of a genome-wide association study of lithium response in 2,563 patients collected by 22 participating sites from the International Consortium on Lithium Genetics (ConLiGen); the largest attempted so far. Data from over 6 million common single nucleotide polymorphisms (SNPs) were tested for association with categorical and continuous ratings of lithium response of known reliability.

**Findings:** 
A single locus of four linked SNPs on chromosome 21 met genome-wide significance criteria for association with lithium response (rs79663003: p=1·37×10−8; rs78015114: p=1·31×10−8; rs74795342: p=3·31×10−9; rs75222709: p=3·50×10−9). In an independent, prospective study of 73 patients treated with lithium monotherapy for a period of up to two years, carriers of the response-associated alleles had a significantly lower rate of relapse than carriers of the alternate alleles (p=0·03, hazard ratio = 3·8).

**Interpretation:** 
The response-associated region contains two genes coding for long non-coding RNAs (lncRNAs), AL157359.3 and AL157359.4. LncRNAs are increasingly appreciated as important regulators of gene expression, particularly in the CNS. Further studies are needed to establish the biological context of these findings and their potential clinical utility. Confirmed biomarkers of lithium response would constitute an important step forward in the clinical management of bipolar disorder.

### Background

Lithium remains a first-line treatment in bipolar disorder, but individual response is variable. Previous studies have suggested that lithium response is a heritable trait. However, no genetic markers have been reproducibly identified.

### Methods

Here we report the results of a genome-wide association study of lithium response in 2,563 patients collected by 22 participating sites from the International Consortium on Lithium Genetics (ConLiGen); the largest attempted so far. Data from over 6 million common single nucleotide polymorphisms (SNPs) were tested for association with categorical and continuous ratings of lithium response of known reliability.

### Findings

A single locus of four linked SNPs on chromosome 21 met genome-wide significance criteria for association with lithium response (rs79663003: p=1·37×10^−8^; rs78015114: p=1·31×10^−8^; rs74795342: p=3·31×10^−9^; rs75222709: p=3·50×10^−9^). In an independent, prospective study of 73 patients treated with lithium monotherapy for a period of up to two years, carriers of the response-associated alleles had a significantly lower rate of relapse than carriers of the alternate alleles (p=0·03, hazard ratio = 3·8).

### Interpretation

The response-associated region contains two genes coding for long non-coding RNAs (lncRNAs), *AL157359.3* and *AL157359.4*. LncRNAs are increasingly appreciated as important regulators of gene expression, particularly in the CNS. Further studies are needed to establish the biological context of these findings and their potential clinical utility. Confirmed biomarkers of lithium response would constitute an important step forward in the clinical management of bipolar disorder.

## INTRODUCTION

Bipolar disorder (BD) is an often devastating psychiatric illness characterized by disruptive mood swings, with intervals of partial or full recovery. BD type I and II affect at least 2% of the world’s population; subthreshold forms afflict another 2%^1^. BD consumes a substantial portion of mental health resources. Worldwide, the direct and indirect costs are large, with an estimated US$151 billion spent in the US alone in 2009^2^. Moreover, up to 15% of sufferers die by suicide^3^.

Mood stabilizers are a primary mode of medication treatment for BD^4^. Among these drugs, lithium stands out as a preventive agent for manic episodes^5^ and suicide^6^. Consequently, lithium is still recommended as a first-line treatment for BD, even though individual response is variable. Many patients show a robust improvement with lithium and a subset is highly responsive^7–9^, with near total resolution of symptoms. On the other hand, at least 30% of patients are only partially responsive, and more than 30% have no clinical response to lithium.

Evidence suggests that some of the variability in lithium response has a genetic basis, but sample sizes in such studies have been limited. Good responders are more likely to have a family history of BD than poor responders^10^. Patients who stabilized on lithium tend to aggregate within families^11, 12^. A twin study reported better lithium prophylaxis in twins whose co-twin also had BD^13^.

Genetic markers of lithium response could provide insight into the biological mechanism of lithium action and might be valuable for treatment planning. However, few pharmacogenetic studies of lithium have been published, and those have generally employed small samples and variable definitions of response. Candidate gene studies have focused on genes purported to be involved in the therapeutic action of lithium, but replicable results have not emerged^14, 15^. Three genome-wide association studies (GWAS) of lithium response have been published. The first was from the Systematic Treatment Enhancement Program for Bipolar Disorder (STEP-BD) cohort^16^, in which 458 BD-I/II patients were treated with lithium and response was evaluated as time to recurrence during lithium treatment. No genome-wide significant results were identified. A second GWAS^17^ was carried out in 204 Sardinian BD patients (only 52 were genotyped with SNP arrays). No SNPs reached genome-wide significance. Most recently, Chen and colleagues^18^ performed a GWAS on 294 highly treatment adherent individuals of Asian ancestry selected from a larger set of about 2000 treated for BD-I with lithium monotherapy. The authors reported genome-wide significant association with a cluster of SNPs at 3p24.1. However, to date, all reported studies have failed to replicate these findings in either Asian or European ancestry samples^19–21^.

In order to overcome the problems inherent in smaller sample sizes, we established the international Consortium on Lithium Genetics ([www.ConLiGen.org](http://www.ConLiGen.org)) in 2008^22^. Here we report the results of an initial GWAS of lithium response in 2563 BD patients – by far the largest sample to date – using phenotype and genotype data from 22 ConLiGen sites from four continents (Europe, America, Asia and Australia; for details, see [Table S1; Supplementary Materials](#SD1)). The Taiwanese sample included is independent from the sample studied by Chen and colleagues^18^. Four linked SNPs met genome-wide significance criteria for association with a quantitative measure of lithium response. The associated locus has been annotated with two long, non-coding RNA (lncRNA) genes. If replicated, these findings would constitute a novel genetic marker and could implicate lncRNAs in the mechanism of lithium response.

## MATERIALS AND METHODS

### Participants

Written informed consent was obtained from all participants. Study sites were largely non-overlapping ([Table S1; Supplementary Materials](#SD1)). Over the timeframe of this study (phenotyping between 2008 and 2013), available samples were collected and genotyped in two distinct phases. We thus analyzed the data as two distinct GWAS, referred to herein as ‘GWAS 1’ and ‘GWAS 2’ (for a detailed rationale, see [Supplementary Materials](#SD1)). The analysis pipeline is shown in [Figure S1 (Supplementary Materials)](#SD2). Briefly, a total of 3193 participants were genotyped; 2935 remained after quality control (QC).

A DSM-III or DSM-IV diagnosis of a bipolar spectrum disorder (see [Supplementary Materials](#SD1)) was required, along with data on gender and total score on the Alda scale (see below). We included all patients in whom response could be reliably evaluated: patients were required to have taken lithium for a minimum of 6 months with no additional mood stabilizer added. Comorbid conditions were not among the exclusion criteria. After this step, 1162 individuals were included in GWAS 1 and 1401 were included in GWAS 2.

### Phenotypes

We used the “Retrospective Criteria of Long-Term Treatment Response in Research Subjects with Bipolar Disorder” (Alda scale^12^) for the evaluation of long-term treatment response to lithium. This scale measures the change in illness episodes in the course of treatment with lithium. Briefly, the Alda scale quantifies symptom improvement in the course of treatment (A score, range 0–10), which is then weighted against five criteria (B score) that assess confounding factors, each scored 0, 1 or 2. The total score is then derived by subtracting the total B score from the A score. Negative scores are set to 0 by default so that the total score ranges from 0 to 10.

ConLiGen previously conducted a multi-stage inter-rater reliability study^23^ aimed at finding the optimal way in which Alda subscale values can be combined for response evaluation. We evaluated two main phenotypes for lithium response: a dichotomous phenotype (good/poor response to lithium), that has been successfully used in previous studies^9, 13^, and a continuous phenotype (range 0 to 10). We found the most reliable dichotomous phenotype to be that which designated all subjects with a total score ≥ 7 as “responders”. The most reliable continuous phenotype was found to be one that used the A score but excluded all subjects with a total B score > 4. Descriptive statistics of the phenotypes of the total sample analyzed in the present study can be found in [Table 1](#T1); excluded participants are detailed in [Table S2 (Supplementary Materials)](#SD1).

### Table 1.

|   | GWAS 1 | GWAS 2 |
| --- | --- | --- |
| All subjects |  |  |
| N | 1162 | 1401 |
| Percent female | 59·30 | 56·17 |
| Age at interview | 47·80 (13·99) | 46·84 (13·83) |
| ALDA scale A score | 6·03 (3·14) | 6·35 (2·90) |
| ALDA scale total B score | 2·11 (1·63) | 2·86 (1·68) |
| ALDA scale total score | 4·29 (3·32) | 3·90 (3·02) |
| Dichotomous phenotype: Good response (ALDA scale total score 7 or greater) |  |  |
| N | 361 | 342 |
| Age at interview | 51·72 (14·27) | 48·92 (14·80) |
| Percent female | 56·23 | 51·75 |
| ALDA scale A score | 9·21 (0·82) | 9·36 (0·77) |
| ALDA scale total B score | 0·88 (0·84) | 1·38 (0·96) |
| ALDA scale total score | 8·33 (1·10) | 7·99 (1·01) |
| Dichotomous phenotype: Poor response (ALDA scale total score 6 or less) |  |  |
| N | 801 | 1059 |
| Age at interview | 45·86 (13·44) | 46·17 (13·44) |
| Percent female | 60·67 | 57·60 |
| ALDA scale A score | 4·60 (2·71) | 5·38 (2·66) |
| ALDA scale total B score | 2·66 (1·59) | 3·34 (1·58) |
| ALDA scale total score | 2·47 (2·19) | 2·58 (2·14) |
| Continuous phenotype (ALDA scale A score, all subjects with a total B score greater 4 excluded) |  |  |
| N | 1065 | 1168 |
| Age at interview | 48·12 (14·00) | 46·97 (13·84) |
| Percent female | 59·91 | 56·34 |
| ALDA scale A score | 6·13 (3·13) | 6·52 (2·87) |
| ALDA scale total B score | 1·78 (1·26) | 2·35 (1·16) |
| ALDA scale total score | 4·59 (3·28) | 4·40 (2·94) |

Table 1 Caption: Phenotypic characteristics of individuals used for the analyses

### Genotyping, quality control and imputation

DNA was extracted from peripheral blood samples. Samples were genotyped at the NIMH, Life & Brain Center at the University of Bonn, or Broad Institute using either Affymetrix or Illumina SNP arrays ([Table S1; Supplementary Materials](#SD1)), according to the manufacturers’ protocols.

Quality control and imputation were carried out in batches corresponding to distinct SNP arrays and ethnicities. Six batches of data were used in GWAS 1, including five of European ancestry (Affymetrix 6·0, Human610/660W, HumanOmniExpress, HumanOmni1-Quad, HumanOmni2.5), and one of Japanese ancestry (HumanOmni2.5). Five batches of data were used in GWAS2, including four European-ancestry data sets (Affymetrix 6·0, Human660W, HumanOmni1-Quad, HumanOmniExpress), and one Taiwanese data set (HumanOmniExpress) not overlapping with the sample studied by Chen and colleagues^18^. Quality control parameters for retaining SNPs and subjects, including relatedness checking and population stratification analysis are detailed in the [Supplementary Materials](#SD1).

Genotype imputation was performed using the prephasing/imputation strategy^24^ implemented by SHAPEIT2^25^ and minimac^26^. The full 1000 Genomes Project data set was used as the reference panel. Imputation was performed separately for each SNP array and ancestry group. Gene dosages for all markers with imputation r^2^ ≥ 0.5 in all batches were used for the final association tests.

### Statistical analysis

Association testing was carried out separately in European-ancestry and Asian-ancestry samples. We analyzed both the categorical and quantitative response phenotypes. Using PLINK v1·07^27^, the association between allele dosages and the dichotomous phenotype was evaluated by logistic regression and the association between allele dosages and the quantitative phenotype was evaluated by linear regression. Genotyping platform was used as a covariate and, in the European-ancestry samples, the first four principal components of population structure were also included in the model to control for population stratification ([Supplementary Materials](#SD1)). Site of collection was not included as a covariate because it was highly co-linear with genotyping platform. Results across GWAS 1 and GWAS 2 were combined by meta-analysis using METAL^28^, under a fixed-effects model with heterogeneity testing.

Overall results in GWAS 1 were compared to those in GWAS 2 by use of the Sign Test ([Supplementary Materials](#SD1)). If there were no association between SNPs and traits, the expectation is that 50% of the β coefficients would have the same sign. The significance of the observed proportion was evaluated under the binomial distribution. To investigate the contribution of the BD risk profile scores (RPS) to lithium response, we used the LD clumped complete result file of 108 835 SNPs from the PGC bipolar GWAS^29^ to calculate –log (OR) weighted RPS in each of the two European ancestry samples. Regression was then used to test whether the calculated RPS scores had any effect on the association between SNP dosages and lithium response by adding the RPS scores as an additional covariate in the regression model.

## ROLE OF THE FUNDING SOURCES

The funding bodies had no role in study design, data collection, data analysis, data interpretation, or writing of the report. LH, UH, FJM, and TGS had full access to all the data. The corresponding authors FJM and TGS had final responsibility for the decision to submit for publication.

## RESULTS

Our principal goal was to identify common genetic variants associated with differential response to lithium. Neither GWAS 1 nor GWAS 2 alone detected a genome-wide significant (p<5×10^−8^) result. However, there was greater than chance consistency between GWAS 1 and GWAS 2 in the overall direction of association. For the continuous phenotype, of 606 independent SNPs in GWAS 1 with p<0·001, 326 (54%) had the same sign in GWAS 2. This represents a significantly greater agreement than chance alone (p=0·028). For the dichotomous phenotype, of 555 independent SNPs in GWAS 1 with p<0·001, 317 (57%) had the same sign in GWAS 2, significantly (p=0·0003) greater than chance (for the complete list of SNPs used in this test, see [Table S4, Supplementary Materials](#SD1)).

When both studies were combined by meta-analysis, genome-wide significance was attained for the continuous phenotype ([Figures 1](#F1)–[3](#F3)). [Table 2](#T2) summarizes the top results (p<1×10^−6^) with the continuous phenotype (see [Table S5](#SD1) for the top results with the dichotomous phenotype). A region on chromosome 21 contained four SNPs that showed genome-wide significant association with lithium response ([Figure 2](#F2)) (minimum p=3·31×10^−9^). These four SNPs are in very strong LD with each other and have similar minor allele frequencies. The same 4 SNPs were associated with the dichotomous definition of lithium response at a p~0·01. These four SNPs also reached significance when only the European-ancestry population was considered ([Table 2](#T2)). The imputation quality for these 4 SNPs was excellent and was supported by direct genotyping in a subset of the total sample (see *assay validation* on page 2 of [Supplementary Materials](#SD1) and [Table S3](#SD1)).

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/fabe/4814312/a7fbe230af03/nihms755576f1.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4814312_nihms755576f1.jpg)

Meta-analysis results of dichotomous and continuous lithium response phenotypes in all participants (CEU plus Asian samples). Genome-wide significant association can be detected with the continuous phenotype. SNPs in green are in linkage disequilibrium (r2>0·6) with the index SNPs (rs74795342).

### Figure 3.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/fabe/4814312/ebf47c451173/nihms755576f3.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4814312_nihms755576f3.jpg)

Forest plots for the most significant SNP, rs74795342. [Panel A, dichotomous phenotype; Panel B, continuous phenotype].

### Table 2.

| Chr | Positionb | SNP | A1c | A2d | FRQe | Gene | P | Directionsf | BETA (CI)g | Heterogeneity P-valh |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
|   |   |   |   |   |   | Both Populations |  |   |   |   |
| 21 | 20310893 | rs79663003 | T | C | 0·94 | AL157359.4 | 1·37E-8 | +++- | 1·04 (0·68 – 1·40) | 0·30 |
| 21 | 20312612 | rs78015114 | T | C | 0·94 | AL157359.4 | 1·31E-8 | +++- | 1·04 (0·68 – 1·40) | 0·31 |
| 21 | 20326336 | rs74795342 | G | A | 0·94 | AL157359.3 | 3·31E-9 | +++- | 1·10 (0·74 – 1·47) | 0·46 |
| 21 | 20327427 | rs75222709 | T | G | 0·94 | AL157359.3 | 3·50E-9 | +++- | 1·10 (0·73 – 1·46) | 0·46 |
|   |  |  |  |  |  |  |  |  |  |  |
|   |   |   |   |   |   | European ancestry only |  |   |   |   |
| 1 | 34604567 | rs9662615 | T | C | 0·44 | CSMD2 | 5·26E-7 | ++ | 0·45 (0·27 – 0·62) | 0·83 |
| 1 | 34608545 | rs771148 | C | T | 0·40 | CSMD2 | 7·01E-7 | ++ | 0·45 (0·27 – 0·63) | 0·64 |
| 7 | 18444419 | rs61549860 | T | A | 0·22 | HDAC9 | 5·44E-7 | ++ | 0·59 (0·36 – 0·83) | 0·92 |
| 21 | 20310893 | rs79663003 | T | C | 0·94 | AL157359.4 | 1·30E-8 | ++ | 1·10 (0·72 – 1·48) | 0·41 |
| 21 | 20312612 | rs78015114 | T | C | 0·94 | AL157359.4 | 1·25E-8 | ++ | 1·10 (0·72 – 1·48) | 0·41 |
| 21 | 20326336 | rs74795342 | G | A | 0·94 | AL157359.3 | 3·00E-9 | ++ | 1·16 (0·78 – 1·54) | 0·63 |
| 21 | 20327427 | rs75222709 | T | G | 0·94 | AL157359.3 | 3·14E-9 | ++ | 1·16 (0·78 – 1·54) | 0·64 |
|   |  |  |  |  |  |  |  |  |  |  |
|   |   |   |   |   |   | Asian ancestry only |  |   |   |   |
| 8 | 21662848 | rs7833426 | A | G | 0·92 | GFRA2 | 2·10E-7 | ++ | 3·66 (2·37 – 4·94) | 0·23 |

Table 2 Caption: Regionsa of the genome showing the strongest association signals with the continuous trait

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/fabe/4814312/a707780c65a3/nihms755576f2.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4814312_nihms755576f2.jpg)

Regional association plot of the region on chromosome 21 in which the genome-wide significant SNPs are located. Imputation quality for the top 4 SNPs was excellent (Table S3, Supplementary Materials).

The associated chromosomal region contains no known protein coding genes. Two lncRNAs have been identified in the region, ENSG00000232193 (AL157359.4) and ENSG00000226204 (AL157359.3). Two of the SNPs (rs74795342 and rs75222709) are located in the intronic region of the gene, AL157359.3. The other two SNPs (rs79663003 and rs78015114) lie between these two lncRNA genes.

In the smaller Asian-ancestry samples, only rs7833426 on chromosome 8 had a p-value <10^−6^. This SNP lies within an intron of *GFRA2*, which codes for a glial cell line-derived neurotrophic factor (GDNF) receptor. This SNP did not pass QC in the European samples due to a minor allele frequency < 5%.

When the GWAS 1 and GWAS 2 were meta-analyzed under the dichotomous phenotype definition, there were no genome-wide significant results ([Figure 1](#F1)). The SNP with the lowest p-value (p=8·10×10^−7^) lies near an annotated lncRNA (ENSG00000258081) on chromosome 14 ([Table S5, Supplementary Materials](#SD1)).

We calculated the power of this study to detect the observed association findings under an additive genetic model using Quanto (v1.2.4, [http://biostats.usc.edu/Quanto.html](http://biostats.usc.edu/Quanto.html)). The sample had ~65% power to detect the reported association signal at an alpha of 5·0×10^−8^. For the dichotomous trait, however, the sample size still lacked power to identify genome-wide significant association, even for common SNPs (MAF=0·2) with relatively large effect sizes (OR=2).

It is possible that lithium response is related to the overall genetic risk burden for BD rather than to lithium per se. To assess this, we reevaluated the association between the most significant SNPs in a model that corrected for differences in overall BD risk burden (Risk Profile Score, RPS) in the European-ancestry samples. Similar results were obtained ([Table S6, Supplementary Materials](#SD1)). The four SNPs on chromosome 21 continued to show genome-wide significant association with lithium response. There was also no detectable relationship between RPS and Alda Score in this sample (data not shown). These results demonstrate that the findings are specific to lithium response, and do not reflect genetic risk for BD.

We assessed genetic association of lithium response in the subset of patients diagnosed with bipolar I disorder. This narrower phenotype comprised about 79% of all participants. Results showed robust association of the same four SNPs on chromosome 21 with the continuous lithium response trait, suggesting that these SNPs play a role in lithium response in subjects with more narrowly defined BD.

Retrospective assessment of lithium response, while reliable in previous studies and when assessed within ConLiGen^23^, is limited by recall bias, incomplete information, and other sources of unmeasured variance. To evaluate the potential impact of these sources of error and test the identified SNPs in an independent sample, we genotyped all four SNPs in samples of BD patients treated with lithium monotherapy and assessed prospectively. The sample was recruited entirely from the San Diego Veterans Affairs Medical Center. A total of 89 bipolar disorder patients participated in the prospective study. After screening for eligibility and initial assessment, patients were started on lithium and entered the stabilization phase. The goal in this phase was to stabilize patients within three months on lithium monotherapy. Following this, patients were observed for one month to assure stabilization after discontinuation of other medications. Patients then entered the maintenance phase and were followed at two to four month intervals for two years. Basic characteristics of this sample can be found in [Table S7](#SD1) (for details on genotyping and statistical analysis, see [Supplementary Materials](#SD1)). After excluding 16 subjects due to screening failure, diagnosis change, voluntary withdrawal, and non-compliance, 73 bipolar patients (BPI:65, BPII:8) were used for the final data analyses.

After correction for several factors known to influence relapse (see [Supplementary Materials](#SD1).) heterozygote carriers of the alleles associated with poorer lithium response showed a significantly higher rate of relapse compared to carriers of the alternate alleles (p=0·03, hazard ratio = 3·8; [Figure S5](#SD2) & [table S8](#SD1)).

## DISCUSSION

This is the largest GWAS of lithium response in BD published to date. In a sample of more than 2500 individuals, we detected genome-wide significant evidence of association with SNPs at a locus on chromosome 21. Further support for this finding was detected in a small, independent, prospectively ascertained sample of patients on lithium monotherapy. The associated locus is thought to encode two long, non-coding RNA genes. This finding could have important implications for our understanding of lithium’s mechanism of action in BD. Replication in independent samples is needed. Personal treatment planning based on genetic data depends on identification of additional markers and their total contribution to differences between individuals in response to treatment. Detection of genome-wide significant markers for a phenotype is the first step in demonstrating if such a goal is achievable.

This study has several limitations. ConLiGen relies on retrospective ratings of treatment response, which lack precision and are subject to recall bias. However, response was rated using a well-validated instrument^12^, previously shown to be reliable by members of the ConLiGen Consortium^23^, and the results were supported in a prospectively assessed, independent sample. The ConLiGen sample encompassed a variety of patients from a range of ancestries and clinical settings. This is more representative of real-world clinical situations, where patients present at various stages of BD and with a range of illness severity and underlines the robustness of our results. As for any GWAS of a complex trait, sample size is critical. The ConLiGen sample size appears small when compared to GWAS of categorical disease phenotypes, where samples on the order of 10,000 are often required. However, common alleles have been found to exert larger effects upon pharmacogenomic traits^30, 31^, where samples of 2500 cases are relatively large. On the other hand, the statistically significant excess agreement in the direction of association between GWAS 1 and GWAS 2 that we observed suggests that additional genome-wide significant associations might emerge from larger sample sizes.

Our results do not support previous reports of individual genes strongly associated with lithium response^16, 18^. Some of those reports were based on smaller samples that may not be comparable to those we studied. They may also represent false positives. Much larger sample sizes would be needed to exclude any particular genes in a GWAS, however.

The main findings seem to implicate lncRNA genes. This implication is causally uncertain, since we have not yet linked allelic variation at the associated SNPs to expression or function of either transcript. There has been an increasing appreciation of the role of lncRNAs in gene regulation, especially in the CNS. An ongoing study of gene expression in peripheral blood during and after acute episodes of BD found a trend toward decreased expression of one of the lncRNAs identified within the association region (AL157359.3; p=0·08, fold change=1·17) after an acute manic episode (Po-Hsiu Kuo, personal communication, 12/2014).

Even if confirmed, the clinical importance of these findings may be limited. The relatively low frequency of the response-associated alleles means that genetic testing would be uninformative in most patients. These and additional genetic markers from future studies could ultimately lead to a clinically informative test, but additional information from established predictors such as family history may be needed, as has been observed for other phenotypes^32^. In line with similar approaches in the field, polygenic score analyses to predict lithium response may prove to be especially informative, provided that larger, adequately phenotyped samples become available. Clinical utility is a high bar, but the current dearth of good biomarkers of lithium response means that any robust genetic markers could constitute a real step forward.

Any GWAS is subject to experimental error. Type I error can occur, although stringent levels of genome-wide significance keep this to a minimum. Association findings might reflect unobserved variables. The alleles found to be associated with poor lithium response in this study could actually reflect something else, such as treatment adherence. Supportive results in a longitudinal, prospectively-rated sample are encouraging, but due to distinct methods of rating lithium response this cannot be viewed as a replication of the ConLiGen results. On the other hand, relapse over the course of two years on lithium monotherapy is in some ways a better phenotype than that assessed by the Alda Scale, which relies on retrospective ratings. The fact that the same alleles were associated with both retrospective response and prospective relapse may actually increase the importance of the findings and their potential clinical relevance.

GWAS are best viewed as an important starting point for additional investigations. Before embarking on functional studies, future work will need to replicate and extend these findings using comparable ratings of lithium response in large samples. As we may have missed some additional true positive markers due to power constraints, such studies should also target the longer list of SNPs that were associated with lithium response at less significant p-values than were formally reported here. Summary results for SNPs with p values <5x10^−5^ are posted at [www.conligen.org](http://www.conligen.org); the corresponding authors can be contacted for more complete summary results. Additional experimental work is needed to establish the functional SNP(s) and their biological impact, if any, in cellular or animal models. Such models could facilitate screening for other drugs that mimic lithium, thus generating novel therapeutic candidates suitable for further study.

## Supplementary Material

### Evidence before this study

### Evidence before this study

### Evidence before this study

### Evidence before this study

Lithium is a mainstay in the treatment of bipolar disorder (BD), also known as manic-depressive illness, and may exert neuroprotective effects in neurodegenerative disorders. However, little is known about lithium’s mechanism of action. Individual response in BD varies from excellent to very poor, with about 30% of patients considered good responders^7–9^. Many genetic association studies of lithium response have been performed, but samples were small, and replicable findings have not emerged^14, 15^. Three genome-wide association studies (GWAS) of lithium response have been published to date, each implicating different loci.^16–18^

### Added value of this study

The international Consortium on Lithium Genetics ([www.ConLiGen.org](http://www.ConLiGen.org)) has assembled the largest GWAS on lithium response in BD to date, totaling over 2,500 individuals. We now present genome-wide significant evidence of association between lithium response and common genetic variants on chromosome 21. The genetic region associated with response contains two long non-coding RNA genes, which are increasingly appreciated as important regulators of gene expression, particularly in the CNS. These findings suggest a novel potential mechanism of action for lithium. In an independent, prospectively followed clinical sample, the identified genetic markers also helped predict relapse during lithium treatment.

### Implications of all the available evidence

This study suggests that a better understanding of drug mechanisms and response can be achieved through international cooperative efforts that leverage clinical expertise with large-scale genomics. The genetic markers identified here show predictive value in a prospective clinical sample, but further studies are needed to establish the potential clinical utility of these findings and their biological context. Confirmed biomarkers of lithium response would constitute an important step forward in the clinical management of BD.

## Acknowledgments

We are greatly indebted to all the study participants without whom this research would not have been possible.

We are grateful to the members of our Scientific Advisory Board ([www.conligen.org/sab.html](http://www.conligen.org/sab.html)) for critical input over the course of the project.

This work was in part funded by the Deutsche Forschungsgemeinschaft (DFG; grant no. RI 908/7-1) and the Intramural Research Program of the National Institute of Mental Health (ZIA-MH00284311; [NCT00001174](https://clinicaltrials.gov/ct2/show/NCT00001174)).

The genotyping was in part funded by the German Federal Ministry of Education and Research (BMBF) through the Integrated Network IntegraMent (Integrated Understanding of Causes and Mechanisms in Mental Disorders), under the auspices of the e:Med Programme (grants awarded to Thomas G. Schulze, Marcella Rietschel, and Markus M. Nöthen).

Markus M. Nöthen received support from the Alfried Krupp von Bohlen und Halbach-Stiftung. Franziska Degenhardt received support from the BONFOR Programme of the University of Bonn, Germany.

Some data and biomaterials were collected as part of eleven projects (Study 40) that participated in the National Institute of Mental Health (NIMH) Bipolar Disorder Genetics Initiative. From 2003–2007, the Principal Investigators and Co-Investigators were: Indiana University, Indianapolis, IN, R01 MH59545, John Nurnberger, M.D., Ph.D., Marvin J. Miller, M.D., Elizabeth S. Bowman, M.D., N. Leela Rau, M.D., P. Ryan Moe, M.D., Nalini Samavedy, M.D., Rif El-Mallakh, M.D. (at University of Louisville), Husseini Manji, M.D. (at Johnson and Johnson), Debra A. Glitz, M.D. (at Wayne State University), Eric T. Meyer, Ph.D., M.S. (at Oxford University, UK), Carrie Smiley, R.N., Tatiana Foroud, Ph.D., Leah Flury, M.S., Danielle M. Dick, Ph.D (at Virginia Commonwealth University), Howard Edenberg, Ph.D.; Washington University, St. Louis, MO, R01 MH059534, John Rice, Ph.D, Theodore Reich, M.D., Allison Goate, Ph.D., Laura Bierut, M.D. K02 DA21237; Johns Hopkins University, Baltimore, M.D., R01 MH59533, Melvin McInnis, M.D., J. Raymond DePaulo, Jr., M.D., Dean F. MacKinnon, M.D., Francis M. Mondimore, M.D., James B. Potash, M.D., Peter P. Zandi, Ph.D, Dimitrios Avramopoulos, and Jennifer Payne; University of Pennsylvania, PA, R01 MH59553, Wade Berrettini, M.D., Ph.D.; University of California at San Francisco, CA, R01 MH60068, William Byerley, M.D., and Sophia Vinogradov, M.D.; University of Iowa, IA, R01 MH059548, William Coryell, M.D., and Raymond Crowe, M.D.; University of Chicago, IL, R01 MH59535, Elliot Gershon, M.D., Judith Badner, Ph.D., Francis McMahon, M.D., Chunyu Liu, Ph.D., Alan Sanders, M.D., Maria Caserta, Steven Dinwiddie, M.D., Tu Nguyen, Donna Harakal; University of California at San Diego, CA, R01 MH59567, John Kelsoe, M.D., Rebecca McKinney, B.A.; Rush University, IL, R01 MH059556, William Scheftner, M.D., Howard M. Kravitz, D.O., M.P.H., Diana Marta, B.S., Annette Vaughn-Brown, M.S.N., R.N., and Laurie Bederow, M.A.; NIMH Intramural Research Program, Bethesda, MD, 1Z01MH002810-01, Francis J. McMahon, M.D., Layla Kassem, Psy.D., Sevilla Detera-Wadleigh, Ph.D, Lisa Austin, Ph.D, Dennis L. Murphy, M.D.; Howard University, William B. Lawson, M.D., Ph.D., Evarista Nwulia, M.D., and Maria Hipolito, M.D. This work was supported by the NIH grants P50CA89392 from the National Cancer Institute and 5K02DA021237 from the National Institute of Drug Abuse.

The Canadian part of the study was supported by a grant #64410 from the Canadian Institutes of Health Research to MA. We wish to thank Ms. Joanne Petite and Ms. Giselle Kraus for assistance with data collection.

Collection and phenotyping of the Australian UNSW sample, by Philip B. Mitchell, Peter R. Schofield, Janice M. Fullerton and Adam Wright, was funded by an Australian NHMRC Program Grant (No. 1037196)

The collection of the Barcelona sample was supported by the Centro de Investigación en Red de Salud Mental (CIBERSAM) and IDIBAPS (grant numbers PI080247, PI1200906, PI12/00018, 2014SGR1636, and 2014SGR398). Jean-Michel Aubry and Alexandre Dayer were supported by the Swiss National Science Foundation (grant number 32003B_125469. David Cousins was supported by a Medical Research Council Clinician Scientist Fellowship Award (MR/L006642/1). Louise Frisén was supported by the Swedish Research Council (grant no. 523-2011-3807). Maria Grigoroiu-Serbanescu was supported by UEFISCDI, Romania, grant no. 89/2012. Po-Hsiu Kuo was funded by the Taiwan Ministry of Science and Technology (grant no. MST 99-2314-B-002-140-MY3 and 102-2314-B-002-117-MY3). Carlos A. López-Jaramillo was funded by by the “Estrategia de Sostenibilidad 2014-2015” program of the University of Antioquia. Tomas Novak was supported by the Ministry of Health of the Czech Republic (grant no. IGA NT1389). James B. Potash (Funding): James Wah Fund and Project MATCH. Thomas G. Schulze and Urs Heilbronner received support from the Dr.-Lisa-Oehler-Foundation (Kassel, Germany). Alessio Squassina has a post-doctoral fellowship funded by the Sardinia Regional Government POR Sardegna FSE Operational Program of the Autonomous Region of Sardinia, European Social Fund 2007–2013—Axis IV Human Resources, Objective l.3, Line of Activity l.3.1 Naomi R Wray was funded by Australian NHMRC Fellowships 613602 and 1078901.

This study utilized the high-performance computational capabilities of the Biowulf Linux cluster at the National Institutes of Health, Bethesda, MD. ([http://biowulf.nih.gov](http://biowulf.nih.gov)).

Genotyping for part of the Swedish sample was funded by the Stanley Center for Psychiatric Research at the Broad Institute.

**Primary sources of funding:** Deutsche Forschungsgemeinschaft (DFG), National Institute of Mental Health (NIMH) Intramural Research Program

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Merikangas KR, Akiskal HS, Angst J, et al. Lifetime and 12-month prevalence of bipolar spectrum disorder in the National Comorbidity Survey replication. Archives of general psychiatry. 2007;64(5):543–52. doi: 10.1001/archpsyc.64.5.543.  [DOI](https://doi.org/10.1001/archpsyc.64.5.543) | [PMC free article](/articles/PMC1931566/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17485606/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Archives%20of%20general%20psychiatry&title=Lifetime%20and%2012-month%20prevalence%20of%20bipolar%20spectrum%20disorder%20in%20the%20National%20Comorbidity%20Survey%20replication&author=KR%20Merikangas&author=HS%20Akiskal&author=J%20Angst&volume=64&issue=5&publication_year=2007&pages=543-52&pmid=17485606&doi=10.1001/archpsyc.64.5.543&)

2. Dilsaver SC. An estimate of the minimum economic burden of bipolar I and II disorders in the United States: 2009. Journal of affective disorders. 2011;129(1–3):79–83. doi: 10.1016/j.jad.2010.08.030.  [DOI](https://doi.org/10.1016/j.jad.2010.08.030) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20888048/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20of%20affective%20disorders&title=An%20estimate%20of%20the%20minimum%20economic%20burden%20of%20bipolar%20I%20and%20II%20disorders%20in%20the%20United%20States:%202009&author=SC%20Dilsaver&volume=129&issue=1%E2%80%933&publication_year=2011&pages=79-83&pmid=20888048&doi=10.1016/j.jad.2010.08.030&)

3. Goodwin FK, Jaison KR. Manic-Depressive Illness: Bipolar Disorders and Recurrent Depression. 2. 2007.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Manic-Depressive%20Illness:%20Bipolar%20Disorders%20and%20Recurrent%20Depression&author=FK%20Goodwin&author=KR%20Jaison&publication_year=2007&)

4. Geddes JR, Miklowitz DJ. Treatment of bipolar disorder. Lancet. 2013;381(9878):1672–82. doi: 10.1016/S0140-6736(13)60857-0.  [DOI](https://doi.org/10.1016/S0140-6736(13)60857-0) | [PMC free article](/articles/PMC3876031/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23663953/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Treatment%20of%20bipolar%20disorder&author=JR%20Geddes&author=DJ%20Miklowitz&volume=381&issue=9878&publication_year=2013&pages=1672-82&pmid=23663953&doi=10.1016/S0140-6736(13)60857-0&)

5. Bauer MS, Mitchner L. What is a "mood stabilizer"? An evidence-based response The American journal of psychiatry. 2004;161(1):3–18. doi: 10.1176/appi.ajp.161.1.3.  [DOI](https://doi.org/10.1176/appi.ajp.161.1.3) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14702242/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=An%20evidence-based%20response%20The%20American%20journal%20of%20psychiatry&title=What%20is%20a%20%22mood%20stabilizer%22?&author=MS%20Bauer&author=L%20Mitchner&volume=161&issue=1&publication_year=2004&pages=3-18&pmid=14702242&doi=10.1176/appi.ajp.161.1.3&)

6. Tondo L, Hennen J, Baldessarini RJ. Lower suicide risk with long-term lithium treatment in major affective illness: a meta-analysis. Acta psychiatrica Scandinavica. 2001;104(3):163–72. doi: 10.1034/j.1600-0447.2001.00464.x.  [DOI](https://doi.org/10.1034/j.1600-0447.2001.00464.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11531653/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Acta%20psychiatrica%20Scandinavica&title=Lower%20suicide%20risk%20with%20long-term%20lithium%20treatment%20in%20major%20affective%20illness:%20a%20meta-analysis&author=L%20Tondo&author=J%20Hennen&author=RJ%20Baldessarini&volume=104&issue=3&publication_year=2001&pages=163-72&pmid=11531653&doi=10.1034/j.1600-0447.2001.00464.x&)

7. Baldessarini RJ, Tondo L. Does lithium treatment still work? Evidence of stable responses over three decades Archives of general psychiatry. 2000;57(2):187–90. doi: 10.1001/archpsyc.57.2.187.  [DOI](https://doi.org/10.1001/archpsyc.57.2.187) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10665622/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Evidence%20of%20stable%20responses%20over%20three%20decades%20Archives%20of%20general%20psychiatry&title=Does%20lithium%20treatment%20still%20work?&author=RJ%20Baldessarini&author=L%20Tondo&volume=57&issue=2&publication_year=2000&pages=187-90&pmid=10665622&doi=10.1001/archpsyc.57.2.187&)

8. Rybakowski JK, Chlopocka-Wozniak M, Suwalska A. The prophylactic effect of long-term lithium administration in bipolar patients entering treatment in the 1970s and 1980s. Bipolar disorders. 2001;3(2):63–7. doi: 10.1034/j.1399-5618.2001.030203.x.  [DOI](https://doi.org/10.1034/j.1399-5618.2001.030203.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11333064/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bipolar%20disorders&title=The%20prophylactic%20effect%20of%20long-term%20lithium%20administration%20in%20bipolar%20patients%20entering%20treatment%20in%20the%201970s%20and%201980s&author=JK%20Rybakowski&author=M%20Chlopocka-Wozniak&author=A%20Suwalska&volume=3&issue=2&publication_year=2001&pages=63-7&pmid=11333064&doi=10.1034/j.1399-5618.2001.030203.x&)

9. Garnham J, Munro A, Slaney C, et al. Prophylactic treatment response in bipolar disorder: results of a naturalistic observation study. Journal of affective disorders. 2007;104(1–3):185–90. doi: 10.1016/j.jad.2007.03.003.  [DOI](https://doi.org/10.1016/j.jad.2007.03.003) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17442400/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20of%20affective%20disorders&title=Prophylactic%20treatment%20response%20in%20bipolar%20disorder:%20results%20of%20a%20naturalistic%20observation%20study&author=J%20Garnham&author=A%20Munro&author=C%20Slaney&volume=104&issue=1%E2%80%933&publication_year=2007&pages=185-90&pmid=17442400&doi=10.1016/j.jad.2007.03.003&)

10. Kleindienst N, Engel R, Greil W. Which clinical factors predict response to prophylactic lithium? A systematic review for bipolar disorders. Bipolar disorders. 2005;7(5):404–17. doi: 10.1111/j.1399-5618.2005.00244.x.  [DOI](https://doi.org/10.1111/j.1399-5618.2005.00244.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16176433/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bipolar%20disorders&title=Which%20clinical%20factors%20predict%20response%20to%20prophylactic%20lithium?%20A%20systematic%20review%20for%20bipolar%20disorders&author=N%20Kleindienst&author=R%20Engel&author=W%20Greil&volume=7&issue=5&publication_year=2005&pages=404-17&pmid=16176433&doi=10.1111/j.1399-5618.2005.00244.x&)

11. Duffy A, Alda M, Milin R, Grof P. A consecutive series of treated affected offspring of parents with bipolar disorder: is response associated with the clinical profile? Canadian journal of psychiatry Revue canadienne de psychiatrie. 2007;52(6):369–76. doi: 10.1177/070674370705200606.  [DOI](https://doi.org/10.1177/070674370705200606) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17696023/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Canadian%20journal%20of%20psychiatry%20Revue%20canadienne%20de%20psychiatrie&title=A%20consecutive%20series%20of%20treated%20affected%20offspring%20of%20parents%20with%20bipolar%20disorder:%20is%20response%20associated%20with%20the%20clinical%20profile?&author=A%20Duffy&author=M%20Alda&author=R%20Milin&author=P%20Grof&volume=52&issue=6&publication_year=2007&pages=369-76&pmid=17696023&doi=10.1177/070674370705200606&)

12. Grof P, Duffy A, Cavazzoni P, et al. Is response to prophylactic lithium a familial trait? The Journal of clinical psychiatry. 2002;63(10):942–7. doi: 10.4088/jcp.v63n1013.  [DOI](https://doi.org/10.4088/jcp.v63n1013) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12416605/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=The%20Journal%20of%20clinical%20psychiatry&title=Is%20response%20to%20prophylactic%20lithium%20a%20familial%20trait?&author=P%20Grof&author=A%20Duffy&author=P%20Cavazzoni&volume=63&issue=10&publication_year=2002&pages=942-7&pmid=12416605&doi=10.4088/jcp.v63n1013&)

13. Mendlewicz J, Verbanck P, Linkowski P, Wilmotte J. Lithium accumulation in erythrocytes of manic-depressive patients: an in vivo twin study. The British journal of psychiatry : the journal of mental science. 1978;133:436–44. doi: 10.1192/bjp.133.5.436.  [DOI](https://doi.org/10.1192/bjp.133.5.436) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/569523/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=The%20British%20journal%20of%20psychiatry%20:%20the%20journal%20of%20mental%20science&title=Lithium%20accumulation%20in%20erythrocytes%20of%20manic-depressive%20patients:%20an%20in%20vivo%20twin%20study&author=J%20Mendlewicz&author=P%20Verbanck&author=P%20Linkowski&author=J%20Wilmotte&volume=133&publication_year=1978&pages=436-44&pmid=569523&doi=10.1192/bjp.133.5.436&)

14. McCarthy MJ, Leckband SG, Kelsoe JR. Pharmacogenetics of lithium response in bipolar disorder. Pharmacogenomics. 2010;11(10):1439–65. doi: 10.2217/pgs.10.127.  [DOI](https://doi.org/10.2217/pgs.10.127) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21047205/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Pharmacogenetics%20of%20lithium%20response%20in%20bipolar%20disorder&author=MJ%20McCarthy&author=SG%20Leckband&author=JR%20Kelsoe&volume=11&issue=10&publication_year=2010&pages=1439-65&pmid=21047205&doi=10.2217/pgs.10.127&)

15. Can A, Schulze TG, Gould TD. Molecular actions and clinical pharmacogenetics of lithium therapy. Pharmacology, biochemistry, and behavior. 2014 doi: 10.1016/j.pbb.2014.02.004.  [DOI](https://doi.org/10.1016/j.pbb.2014.02.004) | [PMC free article](/articles/PMC4220538/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24534415/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacology,%20biochemistry,%20and%20behavior&title=Molecular%20actions%20and%20clinical%20pharmacogenetics%20of%20lithium%20therapy&author=A%20Can&author=TG%20Schulze&author=TD%20Gould&publication_year=2014&pmid=24534415&doi=10.1016/j.pbb.2014.02.004&)

16. Perlis RH, Smoller JW, Ferreira MA, et al. A genomewide association study of response to lithium for prevention of recurrence in bipolar disorder. The American journal of psychiatry. 2009;166(6):718–25. doi: 10.1176/appi.ajp.2009.08111633.  [DOI](https://doi.org/10.1176/appi.ajp.2009.08111633) | [PMC free article](/articles/PMC3908470/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19448189/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=The%20American%20journal%20of%20psychiatry&title=A%20genomewide%20association%20study%20of%20response%20to%20lithium%20for%20prevention%20of%20recurrence%20in%20bipolar%20disorder&author=RH%20Perlis&author=JW%20Smoller&author=MA%20Ferreira&volume=166&issue=6&publication_year=2009&pages=718-25&pmid=19448189&doi=10.1176/appi.ajp.2009.08111633&)

17. Squassina A, Manchia M, Borg J, et al. Evidence for association of an ACCN1 gene variant with response to lithium treatment in Sardinian patients with bipolar disorder. Pharmacogenomics. 2011;12(11):1559–69. doi: 10.2217/pgs.11.102.  [DOI](https://doi.org/10.2217/pgs.11.102) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21961650/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Evidence%20for%20association%20of%20an%20ACCN1%20gene%20variant%20with%20response%20to%20lithium%20treatment%20in%20Sardinian%20patients%20with%20bipolar%20disorder&author=A%20Squassina&author=M%20Manchia&author=J%20Borg&volume=12&issue=11&publication_year=2011&pages=1559-69&pmid=21961650&doi=10.2217/pgs.11.102&)

18. Chen CH, Lee CS, Lee MT, et al. Variant GADL1 and response to lithium therapy in bipolar I disorder. The New England journal of medicine. 2014;370(2):119–28. doi: 10.1056/NEJMoa1212444.  [DOI](https://doi.org/10.1056/NEJMoa1212444) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24369049/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=The%20New%20England%20journal%20of%20medicine&title=Variant%20GADL1%20and%20response%20to%20lithium%20therapy%20in%20bipolar%20I%20disorder&author=CH%20Chen&author=CS%20Lee&author=MT%20Lee&volume=370&issue=2&publication_year=2014&pages=119-28&pmid=24369049&doi=10.1056/NEJMoa1212444&)

19. Consortium on Lithium Genetics. Hou L, Heilbronner U, et al. Variant GADL1 and response to lithium in bipolar I disorder. The New England journal of medicine. 2014;370(19):1857–9. doi: 10.1056/NEJMc1401817.  [DOI](https://doi.org/10.1056/NEJMc1401817) | [PMC free article](/articles/PMC6639012/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24806176/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=The%20New%20England%20journal%20of%20medicine&title=Variant%20GADL1%20and%20response%20to%20lithium%20in%20bipolar%20I%20disorder&author=L%20Hou&author=U%20Heilbronner&volume=370&issue=19&publication_year=2014&pages=1857-9&pmid=24806176&doi=10.1056/NEJMc1401817&)

20. Ikeda M, Kondo K, Iwata N. Variant GADL1 and response to lithium in bipolar I disorder. The New England journal of medicine. 2014;370(19):1856–7. doi: 10.1056/NEJMc1401817.  [DOI](https://doi.org/10.1056/NEJMc1401817) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24806175/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=The%20New%20England%20journal%20of%20medicine&title=Variant%20GADL1%20and%20response%20to%20lithium%20in%20bipolar%20I%20disorder&author=M%20Ikeda&author=K%20Kondo&author=N%20Iwata&volume=370&issue=19&publication_year=2014&pages=1856-7&pmid=24806175&doi=10.1056/NEJMc1401817&)

21. Cruceanu C, Alda M, Dion PA, Turecki G, Rouleau GA. No evidence for GADL1 variation as a bipolar disorder susceptibility factor in a Caucasian lithium-responsive cohort. The American journal of psychiatry. 2015;172(1):94–5. doi: 10.1176/appi.ajp.2014.14070855.  [DOI](https://doi.org/10.1176/appi.ajp.2014.14070855) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25553497/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=The%20American%20journal%20of%20psychiatry&title=No%20evidence%20for%20GADL1%20variation%20as%20a%20bipolar%20disorder%20susceptibility%20factor%20in%20a%20Caucasian%20lithium-responsive%20cohort&author=C%20Cruceanu&author=M%20Alda&author=PA%20Dion&author=G%20Turecki&author=GA%20Rouleau&volume=172&issue=1&publication_year=2015&pages=94-5&pmid=25553497&doi=10.1176/appi.ajp.2014.14070855&)

22. Schulze TG, Alda M, Adli M, et al. The International Consortium on Lithium Genetics (ConLiGen): an initiative by the NIMH and IGSLI to study the genetic basis of response to lithium treatment. Neuropsychobiology. 2010;62(1):72–8. doi: 10.1159/000314708.  [DOI](https://doi.org/10.1159/000314708) | [PMC free article](/articles/PMC2889682/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20453537/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neuropsychobiology&title=The%20International%20Consortium%20on%20Lithium%20Genetics%20(ConLiGen):%20an%20initiative%20by%20the%20NIMH%20and%20IGSLI%20to%20study%20the%20genetic%20basis%20of%20response%20to%20lithium%20treatment&author=TG%20Schulze&author=M%20Alda&author=M%20Adli&volume=62&issue=1&publication_year=2010&pages=72-8&pmid=20453537&doi=10.1159/000314708&)

23. Manchia M, Adli M, Akula N, et al. Assessment of Response to Lithium Maintenance Treatment in Bipolar Disorder: A Consortium on Lithium Genetics (ConLiGen) Report. PloS one. 2013;8(6):e65636. doi: 10.1371/journal.pone.0065636.  [DOI](https://doi.org/10.1371/journal.pone.0065636) | [PMC free article](/articles/PMC3686769/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23840348/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PloS%20one&title=Assessment%20of%20Response%20to%20Lithium%20Maintenance%20Treatment%20in%20Bipolar%20Disorder:%20A%20Consortium%20on%20Lithium%20Genetics%20(ConLiGen)%20Report&author=M%20Manchia&author=M%20Adli&author=N%20Akula&volume=8&issue=6&publication_year=2013&pages=e65636&pmid=23840348&doi=10.1371/journal.pone.0065636&)

24. Howie B, Fuchsberger C, Stephens M, Marchini J, Abecasis GR. Fast and accurate genotype imputation in genome-wide association studies through pre-phasing. Nature genetics. 2012;44(8):955–9. doi: 10.1038/ng.2354.  [DOI](https://doi.org/10.1038/ng.2354) | [PMC free article](/articles/PMC3696580/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22820512/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature%20genetics&title=Fast%20and%20accurate%20genotype%20imputation%20in%20genome-wide%20association%20studies%20through%20pre-phasing&author=B%20Howie&author=C%20Fuchsberger&author=M%20Stephens&author=J%20Marchini&author=GR%20Abecasis&volume=44&issue=8&publication_year=2012&pages=955-9&pmid=22820512&doi=10.1038/ng.2354&)

25. Delaneau O, Zagury JF, Marchini J. Improved whole-chromosome phasing for disease and population genetic studies. Nature methods. 2013;10(1):5–6. doi: 10.1038/nmeth.2307.  [DOI](https://doi.org/10.1038/nmeth.2307) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23269371/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature%20methods&title=Improved%20whole-chromosome%20phasing%20for%20disease%20and%20population%20genetic%20studies&author=O%20Delaneau&author=JF%20Zagury&author=J%20Marchini&volume=10&issue=1&publication_year=2013&pages=5-6&pmid=23269371&doi=10.1038/nmeth.2307&)

26. Fuchsberger C, Abecasis GR, Hinds DA. minimac2: faster genotype imputation. Bioinformatics. 2015;31(5):782–4. doi: 10.1093/bioinformatics/btu704.  [DOI](https://doi.org/10.1093/bioinformatics/btu704) | [PMC free article](/articles/PMC4341061/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25338720/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=minimac2:%20faster%20genotype%20imputation&author=C%20Fuchsberger&author=GR%20Abecasis&author=DA%20Hinds&volume=31&issue=5&publication_year=2015&pages=782-4&pmid=25338720&doi=10.1093/bioinformatics/btu704&)

27. Purcell S, Neale B, Todd-Brown K, et al. PLINK: a tool set for whole-genome association and population-based linkage analyses. American journal of human genetics. 2007;81(3):559–75. doi: 10.1086/519795.  [DOI](https://doi.org/10.1086/519795) | [PMC free article](/articles/PMC1950838/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17701901/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=American%20journal%20of%20human%20genetics&title=PLINK:%20a%20tool%20set%20for%20whole-genome%20association%20and%20population-based%20linkage%20analyses&author=S%20Purcell&author=B%20Neale&author=K%20Todd-Brown&volume=81&issue=3&publication_year=2007&pages=559-75&pmid=17701901&doi=10.1086/519795&)

28. Willer CJ, Li Y, Abecasis GR. METAL: fast and efficient meta-analysis of genomewide association scans. Bioinformatics. 2010;26(17):2190–1. doi: 10.1093/bioinformatics/btq340.  [DOI](https://doi.org/10.1093/bioinformatics/btq340) | [PMC free article](/articles/PMC2922887/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20616382/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=METAL:%20fast%20and%20efficient%20meta-analysis%20of%20genomewide%20association%20scans&author=CJ%20Willer&author=Y%20Li&author=GR%20Abecasis&volume=26&issue=17&publication_year=2010&pages=2190-1&pmid=20616382&doi=10.1093/bioinformatics/btq340&)

29. Psychiatric GWAS Consortium Bipolar Disorder Working Group. Large-scale genome-wide association analysis of bipolar disorder identifies a new susceptibility locus near ODZ4. Nature genetics. 2011;43(10):977–83. doi: 10.1038/ng.943.  [DOI](https://doi.org/10.1038/ng.943) | [PMC free article](/articles/PMC3637176/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21926972/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature%20genetics&title=Large-scale%20genome-wide%20association%20analysis%20of%20bipolar%20disorder%20identifies%20a%20new%20susceptibility%20locus%20near%20ODZ4&volume=43&issue=10&publication_year=2011&pages=977-83&pmid=21926972&doi=10.1038/ng.943&)

30. Aberg K, Adkins DE, Bukszar J, et al. Genomewide association study of movement-related adverse antipsychotic effects. Biological psychiatry. 2010;67(3):279–82. doi: 10.1016/j.biopsych.2009.08.036.  [DOI](https://doi.org/10.1016/j.biopsych.2009.08.036) | [PMC free article](/articles/PMC3388725/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19875103/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biological%20psychiatry&title=Genomewide%20association%20study%20of%20movement-related%20adverse%20antipsychotic%20effects&author=K%20Aberg&author=DE%20Adkins&author=J%20Bukszar&volume=67&issue=3&publication_year=2010&pages=279-82&pmid=19875103&doi=10.1016/j.biopsych.2009.08.036&)

31. Malhotra AK, Correll CU, Chowdhury NI, et al. Association between common variants near the melanocortin 4 receptor gene and severe antipsychotic drug-induced weight gain. Archives of general psychiatry. 2012;69(9):904–12. doi: 10.1001/archgenpsychiatry.2012.191.  [DOI](https://doi.org/10.1001/archgenpsychiatry.2012.191) | [PMC free article](/articles/PMC4166499/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22566560/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Archives%20of%20general%20psychiatry&title=Association%20between%20common%20variants%20near%20the%20melanocortin%204%20receptor%20gene%20and%20severe%20antipsychotic%20drug-induced%20weight%20gain&author=AK%20Malhotra&author=CU%20Correll&author=NI%20Chowdhury&volume=69&issue=9&publication_year=2012&pages=904-12&pmid=22566560&doi=10.1001/archgenpsychiatry.2012.191&)

32. Do CB, Hinds DA, Francke U, Eriksson N. Comparison of family history and SNPs for predicting risk of complex disease. PLoS genetics. 2012;8(10):e1002973. doi: 10.1371/journal.pgen.1002973.  [DOI](https://doi.org/10.1371/journal.pgen.1002973) | [PMC free article](/articles/PMC3469463/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23071447/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20genetics&title=Comparison%20of%20family%20history%20and%20SNPs%20for%20predicting%20risk%20of%20complex%20disease&author=CB%20Do&author=DA%20Hinds&author=U%20Francke&author=N%20Eriksson&volume=8&issue=10&publication_year=2012&pages=e1002973&pmid=23071447&doi=10.1371/journal.pgen.1002973&)
