# IFNL4-ΔG Genotype Is Associated With Slower Viral Clearance in Hepatitis C, Genotype-1 Patients Treated With Sofosbuvir and Ribavirin

## Metadata
**Authors:** Eric G Meissner, Dimitra Bon, Ludmila Prokunina-Olsson, Wei Tang, Henry Masur, Thomas R O'Brien, Eva Herrmann, Shyamasundaran Kottilil, Anuoluwapo Osinusi
**Journal:** The Journal of Infectious Diseases
**Date:** 2013 Dec 23
**DOI:** [10.1093/infdis/jit827](https://doi.org/10.1093/infdis/jit827)
**PMID:** 24367041
**PMCID:** PMC4017364
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4017364/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC4017364/pdf/jit827.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC4017364/pdf/jit827.pdf)

## Abstract

Response to pegylated interferon-alpha and ribavirin (IFN-α/RBV) treatment for chronic hepatitis C virus (HCV) infection is influenced by host genetic factors, but their role for IFN-α–free, direct-acting antiviral (DAA) regimens is unclear. An exonic deletion allele (IFNL4-ΔG) bolsters the established association with IFN-α/RBV therapy treatment outcome of another IFNL4 variant, rs12979860, which is located upstream of IFNL3 (IL28B). We report that in patients treated with the DAA sofosbuvir along with RBV, IFNL4-ΔG is associated with slower early viral decay, due to slower loss of free virus (P = .039) and decreased drug efficacy (P = .048), suggesting functional relevance of IFN-λ4 in IFN-α–free DAA therapies.

Keywords: viral kinetics, pharmacokinetics, IL28B, IFNL4, haplotype, SVR, hepatitis C virus, DAA therapy, relapse

## EXPERIMENTAL PROCEDURES

### Clinical Trial Design

As previously described, treatment-naive, chronic HCV patients infected with HCV genotype 1 were treated for 24 weeks with sofosbuvir (Gilead Sciences, Foster City, CA) plus low-dose or weight-based RBV (clinicaltrials.gov identifier [NCT01441180](https://clinicaltrials.gov/ct2/show/NCT01441180)) [[6](#JIT827C6)]. The National Institute of Allergy and Infectious Diseases Institutional Review Board–approved written or oral informed consent was obtained from all participants. Of 60 patients who enrolled, 54 patients completed 24 weeks of therapy, 1 patient completed 12 weeks, and 5 patients dropped out of the study before week 8 [[6](#JIT827C6)]. Twenty-five patients participated in a VK–PK substudy, as previously described, with 1 dropout at week 3 [[6](#JIT827C6)]. Of these patients, 20 were African American, 4 were white, and 1 was Hispanic by self-report. Ten patients received low-dose RBV while 15 received weight-based RBV.

### Viral Kinetics

Levels of sofosbuvir and its active metabolite GS-331007 were measured using high-performance liquid chromatography–mass spectrometry (QPS, LLC, Newark, Deleware) at 0, 1, 2, 4, 8, 12, 24, and 36 hours after administration. Plasma HCV RNA levels were measured at hours 0, 1, 2, 4, 8, 12, and 24; days 3, 5, 7, 10, and 14; and weeks 3 and 4 by quantitative reverse-transcriptase polymerase chain reaction (Abbott Laboratories, Abbott Park, IL) with a lower limit of quantification of 12 IU/mL and a lower limit of detection of 3 IU/mL.

### Genetic Analysis

Genotyping of *IFNL4* variants rs12979860 and rs368234815 was performed on genomic DNA with custom TaqMan assays as previously described [[4](#JIT827C4)]. Haploview 4.2 was used to estimate haplotype frequencies and LD between the *IFNL4* markers (D′ and *r*^2^) [[7](#JIT827C7)].

### VK–PK Model

VK, PK, and pharmacodynamic data for the 25 patients were fitted using a VK–PK model as previously described [[8](#JIT827C8)] and as used by Osinusi et al [[6](#JIT827C6)]. In brief, a Bateman function was used to fit the levels of GS-331007 through hour 36. Afterward, an ordinary differential equation system was used to describe viral kinetics and treatment effects of RBV and sofosbuvir. In contrast to the model used in [[8](#JIT827C8)], we used a slightly different association between drug levels and antiviral efficacy ε. As levels of GS-331007 are highly variable, the treatment efficacy factor ε of blocking viral production of sofosbuvir was assumed to depend, through the Hill function, only indirectly on levels of GS-331007 through an intermediate compartment Z, calculated as *dZ*/*dt* (*t*) = *a*(*C*(*t*) – *Z*(*t*)). In this equation, *C* is the concentration of GS-331007 and *a* is the rate linking the drug concentration compartment, as it is quantified, and the effectively active drug compartment. With this approach, treatment efficacy depends on PK of GS-331007, but the variation is slightly damped and delayed. Log HCV RNA levels over the first 30 days of treatment were fitted using a maximum likelihood approach that accounts for data below the quantitation limits [[9](#JIT827C9)]. We did not use a delay parameter *t*_0_, as this is unnecessary with this model, which already allows a delay in drug effectiveness by the PK of GS-331007 as well as by the intermediate compartment Z.

### Statistical Analysis

To assess potential differences in viral kinetics by *IFNL4*-ΔG or rs12979860 genotypes, fitted model parameters were compared with the Jonckheere and Terpstra trend test, which accounts for the ordinal scale of the genotypes and provides a general *P* value without the need for a multiple test correction.

## RESULTS

Baseline characteristics and treatment outcomes for the 25 patients analyzed in the VK–PK substudy were representative of all patients enrolled in the study ([Supplementary Table 1](http://jid.oxfordjournals.org/lookup/suppl/doi:10.1093/infdis/jit827/-/DC1)). Due to the overall high LD between *IFNL4* variants rs12979860 and rs368234815 in the full patient set (n = 59, D′ = 1.0, *r*^2^ = 0.87) and the VK–PK subset (n = 25, D′ = 1.0, *r*^2^ = 0.85), in most patients the CC, CT, and TT genotypes of rs12979860 corresponded to the TT/TT, TT/ΔG, and ΔG/ΔG genotypes of rs368234815 ([Supplementary Table 2](http://jid.oxfordjournals.org/lookup/suppl/doi:10.1093/infdis/jit827/-/DC1)). Four patients were discordant for these 2 markers; 2 had a favorable CC rs12979860 genotype but an unfavorable heterozygous TT/ΔG rs368234815 genotype (1 dropout, 1 relapser), while 2 patients had an unfavorable heterozygous CT rs12979860 genotype and homozygous ΔG/ΔG rs368234815 genotype (1 SVR_24_, 1 relapser). All 4 patients with discordant genotypes were African Americans, which is consistent with lower LD between these markers in individuals of African ancestry [[4](#JIT827C4)].

In a fully fitted VK–PK model, the loss rate of free virus (*c*) was slower for carriers of *IFNL4*-ΔG (*IFNL4*-ΔG/ΔG, *c* = 4.9; *IFNL4*-TT/ΔG, *c* = 4.7) than for patients who do not carry this variant (*IFNL4*-TT/TT, *c* = 5.8; *P* = .039; Figure [1](#JIT827F1)*A*, Table [1](#JIT827TB1)). Drug efficacy (ε) during early treatment also varied by genotype (*IFNL4*-ΔG/ΔG, ε_mean_ = 0.93; *IFNL4*-TT/ΔG, ε_mean_ = 0.97; *IFNL4*-TT/TT, ε_mean_ = 0.98; *P* = .048; Table [1](#JIT827TB1)), but the loss rate of infected cells (delta) did not (*IFNL4*-ΔG/ΔG, delta = 0.16; *IFNL4*-TT/ΔG, delta = 0.20; *IFNL4*-TT/TT, delta = 0.19; *P* = .31; Table [1](#JIT827TB1)). Similar results were obtained when the analysis was restricted to the 20 African-American patients (data not shown). Notably, classification of *IFNL4* genotype using the rs12979860 SNP did not show significance for *c* (*P* = .13), although ε retained significance (*P* = .027; Figure [1](#JIT827F1)*B*, Table [1](#JIT827TB1)).

### Figure 1.

![Figure 1.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ae94/4017364/3fdd499b6cff/jit82701.jpg)

Viral kinetic (VK) decline, determined by a fully fitted VK-PK (pharmacokinetic) model of hepatitis C virus (HCV) treatment with sofosbuvir/ribavirin, is affected by IFNL4 genotype as determined by (A) rs368234815 and (B) rs12979860. As shown in Figure 1A, slower HCV viral decline is associated with the IFNL4-ΔG rs368234815 allele as compared with other alleles.

### Table 1.

| Parameter | rs368234815 |  |  |  |
| --- | --- | --- | --- | --- |
| TT/TT (n = 6) | TT/ΔG (n = 9) | ΔG/ΔG (n = 10) | P Value |  |
| c | 5.8 (5.2–6.1) | 4.7 (2.2–7.1) | 4.9 (1.8–5.8) | .039 |
| Delta | 0.19 (0.02–0.40) | 0.20 (0.02–0.38) | 0.16 (0.12–0.43) | .31 |
| Epsilon mean | 0.98 (0.88–0.99) | 0.97 (0.68–0.99) | 0.93 (0.69–0.98) | .048 |
| Epsilon max | 0.98 (0.90–0.99) | 0.98 (0.73–0.99) | 0.94 (0.75–0.98) | .048 |
| GS-7977 AUC | 84 (38–151) | 97 (41–169) | 73 (50–146) | .48 |
| GS-331007 AUC | 517 (252–1070) | 502 (276–1035) | 662 (513–793) | .087 |
| Parameter | rs12979860 |  |  |  |
| CC (n = 7) | CT (n = 9) | TT (n = 9) | P Value |  |
| c | 5.7 (3.5–6.1) | 4.7 (2.2–7.1) | 5.1 (1.8–5.8) | .13 |
| Delta | 0.23 (0.02–0.40) | 0.16 (0.02–0.38) | 0.17 (0.12–0.43) | .31 |
| Epsilon mean | 0.98 (0.88–0.99) | 0.98 (0.68–0.99) | 0.92 (0.69–0.97) | .027 |
| Epsilon max | 0.98 (0.90–0.99) | 0.98 (0.73–0.99) | 0.94 (0.75–98) | .027 |
| GS-7977 AUC | 77 (38–151) | 97 (41–169) | 74 (51–146) | .30 |
| GS-331007 AUC | 476 (252–1070) | 574 (344–1035) | 689 (513–793) | .034 |

Table 1 Caption: Association of IFNL4 Variants With Viral Kinetic Decline and Pharmacokinetics

Seven of 8 patients (87.5%; 95% confidence interval [CI], 47.4%–99.7%) with the *IFNL4*-TT/TT genotype achieved SVR_24_ as compared with 31 of 47 with TT/ΔG or ΔG/ΔG genotypes (66.7%; 95% CI, 51.7%–77.8%), but this difference was not statistically significant in this small study (*P* = .4, Fisher exact test).

## DISCUSSION

Genotypes of 2 genetic markers within a novel *IFNL4* gene—an intronic SNP rs12979860, previously referred to as an *IFNL3* (*IL28B*) variant, and an exonic IFN-λ4 protein-creating variant, rs368234815—are predictive of VK decline and treatment outcome for chronic HCV patients treated with IFN-α/RBV therapy [[4](#JIT827C4), [10](#JIT827C10)]. Here we show a significant association of *IFNL4*-ΔG genotype with slower early viral decline for HCV genotype-1–infected patients treated with the IFN-α–free DAA regimen of sofosbuvir/RBV. A fully fitted VK–PK model showed that patients carrying at least 1 *IFNL4*-ΔG allele had significantly slower loss rate of free virus (*c*) and drug efficacy (ε) compared with patients homozygous for the *IFNL4*-TT allele.

Although this trial had insufficient statistical power for a meaningful analysis of the association between *IFNL4*-ΔG genotype and SVR, a previous analysis revealed slower early VK decline in patients who later relapsed [[6](#JIT827C6)]. Together, these data suggest that *IFNL4*-ΔG genotype, which affords the ability to make IFN-λ4, impacts viral decline and could affect treatment outcome for this DAA-based regimen.

Consistent with our findings, rs12979860 genotype was associated with favorable treatment outcome for another DAA regimen in HCV genotype-1 infection (faldaprevir, deleobuvir, and RBV) [[11](#JIT827C11)], suggesting the effect may not be regimen specific. Phase 3 trials of sofosbuvir/RBV with or without IFN-α for HCV genotypes 1, 2, and 3 were recently conducted, and while an association of rs12979860-CC genotype was found in the NEUTRINO study in genotype-1–infected patients treated with sofosbuvir and IFN-α/RBV, no association with treatment outcome was observed in the IFN-α–free arms in genotype-2 and -3–infected patients [[12](#JIT827C12), [13](#JIT827C13)]. The predictive value of *IFNL4* genetic variants for IFN-α–free DAA therapies requires further assessment in larger trials.

VK decline in association with rs12979860 was previously assessed during DAA therapy using a biphasic model rather than the fully fitted model used here. Chu et al found a mixed effect on phase 1 decline with slower velocity but longer duration in patients with the rs12979860-CC genotype who were treated with mericitabine and danoprevir for 14 days [[14](#JIT827C14)]. In contrast, our findings show slower early viral decline with reduced slope and magnitude for patients carrying at least 1 *IFNL4*-ΔG allele in comparison with *IFNL4*-TT/TT homozygotes. Differences in our findings may be explained by the use of a mechanistic and fully fitted VK model rather than a descriptive biphasic model or they may reflect differences in the relative potencies of the DAA regimens.

It is unclear how IFN-λ4 impairs response to IFN-α–containing and IFN-α–free treatment regimens. Similar to other members of the IFN-λ family, IFN-λ4 signals through the IFN-λ receptor complex, induces expression of IFN-stimulated genes (ISGs) via JAK-STAT signaling, and has antiviral properties, yet IFN-λ4 differs in that it is poorly secreted [[4](#JIT827C4), [15](#JIT827C15)]. We hypothesize that in individuals who carry *IFNL4*-ΔG, induced IFN-λ4 expression may contribute to persistent hepatic ISG activation, which could have a negative cross-regulatory effect on the immunological response to HCV infection.

Strengths of this study include the detailed VK–PK data collected in 25 patients, as well as data on the recently discovered *IFNL4*-ΔG variant. The main limitation of the study is the small number of participants, which precludes the ability to assess the effect of *IFNL4* genotype on achieving SVR.

In summary, we provide evidence that the *IFNL4*-ΔG allele is associated with slower early VK decline for the IFN-α–free regimen of sofosbuvir/RBV in HCV genotype-1–infected patients. Knowledge of *IFNL4* genotype could potentially be useful for guiding duration or intensity of successful DAA therapy. The biological mechanism by which IFN-λ4 affects the response to IFN-α–containing and IFN-α–free treatment of HCV is under investigation. The impact of *IFNL4* genotype and IFN-λ4 protein on HCV clearance merits testing in subsequent DAA studies with incomplete treatment responses.

## Supplementary Data

[Supplementary materials](http://jid.oxfordjournals.org/lookup/suppl/doi:10.1093/infdis/jit827/-/DC1) are available at *The Journal of Infectious Diseases* online ([http://jid.oxfordjournals.org/](http://jid.oxfordjournals.org/)). Supplementary materials consist of data provided by the author that are published to benefit the reader. The posted materials are not copyedited. The contents of all supplementary data are the sole responsibility of the authors. Questions or messages regarding errors should be addressed to the author.

## Notes

**Acknowledgments.** We thank Gilead Sciences for provision of sofosbuvir.

**Financial support.** This work was supported in part, with federal funds from the National Cancer Institute, National Institutes of Health, under contract HHSN261200800001E, the National Institutes of Health Clinical Center, and the Division of Cancer Epidemiology and Genetics of the National Cancer Institute. This research was supported, in part, by the intramural program of the National Institute of Allergy and Infectious Diseases. The study was also supported, in part, by the German Research Foundation (DFG) by the clinical research unit KFO 129.

**Potential conflicts of interest.** L. P. -O. and T. R. O'B. are inventors on patent applications filed by the National Cancer Institute for the *IFNL4*-ΔG (rs368234815) genotype-based test and for the IFN-λ4 protein. E. H. served as a research consultant of Roche Pharma and Novartis.

All authors have submitted the ICMJE Form for Disclosure of Potential Conflicts of Interest. Conflicts that the editors consider relevant to the content of the manuscript have been disclosed.

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Liang TJ, Ghany MG. Current and future therapies for hepatitis C virus infection. N Engl J Med. 2013;368:1907–17. doi: 10.1056/NEJMra1213651.  [DOI](https://doi.org/10.1056/NEJMra1213651) | [PMC free article](/articles/PMC3893124/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23675659/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Current%20and%20future%20therapies%20for%20hepatitis%20C%20virus%20infection&author=TJ%20Liang&author=MG%20Ghany&volume=368&publication_year=2013&pages=1907-17&pmid=23675659&doi=10.1056/NEJMra1213651&)

2. Shah N, Pierce T, Kowdley KV. Review of direct-acting antiviral agents for the treatment of chronic hepatitis C. Expert Opin Investig Drugs. 2013;22:1107–21. doi: 10.1517/13543784.2013.806482.  [DOI](https://doi.org/10.1517/13543784.2013.806482) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23735127/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Expert%20Opin%20Investig%20Drugs&title=Review%20of%20direct-acting%20antiviral%20agents%20for%20the%20treatment%20of%20chronic%20hepatitis%20C&author=N%20Shah&author=T%20Pierce&author=KV%20Kowdley&volume=22&publication_year=2013&pages=1107-21&pmid=23735127&doi=10.1517/13543784.2013.806482&)

3. Holmes JA, Desmond PV, Thompson AJ. Does IL28B genotyping still have a role in the era of direct-acting antiviral therapy for chronic hepatitis C infection? J Viral Hepat. 2012;19:677–84. doi: 10.1111/jvh.12003.  [DOI](https://doi.org/10.1111/jvh.12003) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22967098/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Viral%20Hepat&title=Does%20IL28B%20genotyping%20still%20have%20a%20role%20in%20the%20era%20of%20direct-acting%20antiviral%20therapy%20for%20chronic%20hepatitis%20C%20infection?&author=JA%20Holmes&author=PV%20Desmond&author=AJ%20Thompson&volume=19&publication_year=2012&pages=677-84&pmid=22967098&doi=10.1111/jvh.12003&)

4. Prokunina-Olsson L, Muchmore B, Tang W, et al. A variant upstream of IFNL3 (IL28B) creating a new interferon gene IFNL4 is associated with impaired clearance of hepatitis C virus. Nat Genet. 2013;45:164–71. doi: 10.1038/ng.2521.  [DOI](https://doi.org/10.1038/ng.2521) | [PMC free article](/articles/PMC3793390/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23291588/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=A%20variant%20upstream%20of%20IFNL3%20(IL28B)%20creating%20a%20new%20interferon%20gene%20IFNL4%20is%20associated%20with%20impaired%20clearance%20of%20hepatitis%20C%20virus&author=L%20Prokunina-Olsson&author=B%20Muchmore&author=W%20Tang&volume=45&publication_year=2013&pages=164-71&pmid=23291588&doi=10.1038/ng.2521&)

5. Bibert S, Roger T, Calandra T, et al. IL28B expression depends on a novel TT/-G polymorphism which improves HCV clearance prediction. J Exp Med. 2013;210:1109–16. doi: 10.1084/jem.20130012.  [DOI](https://doi.org/10.1084/jem.20130012) | [PMC free article](/articles/PMC3674704/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23712427/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Exp%20Med&title=IL28B%20expression%20depends%20on%20a%20novel%20TT/-G%20polymorphism%20which%20improves%20HCV%20clearance%20prediction&author=S%20Bibert&author=T%20Roger&author=T%20Calandra&volume=210&publication_year=2013&pages=1109-16&pmid=23712427&doi=10.1084/jem.20130012&)

6. Osinusi A, Meissner EG, Lee YJ, et al. Sofosbuvir and ribavirin for hepatitis C genotype 1 in patients with unfavorable treatment characteristics: a randomized clinical trial. JAMA. 2013;310:804–11. doi: 10.1001/jama.2013.109309.  [DOI](https://doi.org/10.1001/jama.2013.109309) | [PMC free article](/articles/PMC4254410/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23982366/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA&title=Sofosbuvir%20and%20ribavirin%20for%20hepatitis%20C%20genotype%201%20in%20patients%20with%20unfavorable%20treatment%20characteristics:%20a%20randomized%20clinical%20trial&author=A%20Osinusi&author=EG%20Meissner&author=YJ%20Lee&volume=310&publication_year=2013&pages=804-11&pmid=23982366&doi=10.1001/jama.2013.109309&)

7. Barrett JC, Fry B, Maller J, Daly MJ. Haploview: analysis and visualization of LD and haplotype maps. Bioinformatics. 2005;21:263–5. doi: 10.1093/bioinformatics/bth457.  [DOI](https://doi.org/10.1093/bioinformatics/bth457) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15297300/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=Haploview:%20analysis%20and%20visualization%20of%20LD%20and%20haplotype%20maps&author=JC%20Barrett&author=B%20Fry&author=J%20Maller&author=MJ%20Daly&volume=21&publication_year=2005&pages=263-5&pmid=15297300&doi=10.1093/bioinformatics/bth457&)

8. Naggie S, Osinusi A, Katsounas A, et al. Dysregulation of innate immunity in hepatitis C virus genotype 1 IL28B-unfavorable genotype patients: impaired viral kinetics and therapeutic response. Hepatology. 2012;56:444–54. doi: 10.1002/hep.25647.  [DOI](https://doi.org/10.1002/hep.25647) | [PMC free article](/articles/PMC3361636/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22331604/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hepatology&title=Dysregulation%20of%20innate%20immunity%20in%20hepatitis%20C%20virus%20genotype%201%20IL28B-unfavorable%20genotype%20patients:%20impaired%20viral%20kinetics%20and%20therapeutic%20response&author=S%20Naggie&author=A%20Osinusi&author=A%20Katsounas&volume=56&publication_year=2012&pages=444-54&pmid=22331604&doi=10.1002/hep.25647&)

9. Guedj J, Thiebaut R, Commenges D. Maximum likelihood estimation in dynamical models of HIV. Biometrics. 2007;63:1198–206. doi: 10.1111/j.1541-0420.2007.00812.x.  [DOI](https://doi.org/10.1111/j.1541-0420.2007.00812.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17489970/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biometrics&title=Maximum%20likelihood%20estimation%20in%20dynamical%20models%20of%20HIV&author=J%20Guedj&author=R%20Thiebaut&author=D%20Commenges&volume=63&publication_year=2007&pages=1198-206&pmid=17489970&doi=10.1111/j.1541-0420.2007.00812.x&)

10. Bochud PY, Bibert S, Negro F, et al. IL28B polymorphisms predict reduction of HCV RNA from the first day of therapy in chronic hepatitis C. J Hepatol. 2011;55:980–8. doi: 10.1016/j.jhep.2011.01.050.  [DOI](https://doi.org/10.1016/j.jhep.2011.01.050) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21354446/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Hepatol&title=IL28B%20polymorphisms%20predict%20reduction%20of%20HCV%20RNA%20from%20the%20first%20day%20of%20therapy%20in%20chronic%20hepatitis%20C&author=PY%20Bochud&author=S%20Bibert&author=F%20Negro&volume=55&publication_year=2011&pages=980-8&pmid=21354446&doi=10.1016/j.jhep.2011.01.050&)

11. Zeuzem S, Soriano V, Asselah T, et al. Faldaprevir and deleobuvir for HCV genotype 1 infection. N Engl J Med. 2013;369:630–9. doi: 10.1056/NEJMoa1213557.  [DOI](https://doi.org/10.1056/NEJMoa1213557) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23944300/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Faldaprevir%20and%20deleobuvir%20for%20HCV%20genotype%201%20infection&author=S%20Zeuzem&author=V%20Soriano&author=T%20Asselah&volume=369&publication_year=2013&pages=630-9&pmid=23944300&doi=10.1056/NEJMoa1213557&)

12. Jacobson IM, Gordon SC, Kowdley KV, et al. Sofosbuvir for hepatitis C genotype 2 or 3 in patients without treatment options. N Engl J Med. 2013;368:1867–77. doi: 10.1056/NEJMoa1214854.  [DOI](https://doi.org/10.1056/NEJMoa1214854) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23607593/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Sofosbuvir%20for%20hepatitis%20C%20genotype%202%20or%203%20in%20patients%20without%20treatment%20options&author=IM%20Jacobson&author=SC%20Gordon&author=KV%20Kowdley&volume=368&publication_year=2013&pages=1867-77&pmid=23607593&doi=10.1056/NEJMoa1214854&)

13. Lawitz E, Mangia A, Wyles D, et al. Sofosbuvir for previously untreated chronic hepatitis C infection. N Engl J Med. 2013;368:1878–87. doi: 10.1056/NEJMoa1214853.  [DOI](https://doi.org/10.1056/NEJMoa1214853) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23607594/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Sofosbuvir%20for%20previously%20untreated%20chronic%20hepatitis%20C%20infection&author=E%20Lawitz&author=A%20Mangia&author=D%20Wyles&volume=368&publication_year=2013&pages=1878-87&pmid=23607594&doi=10.1056/NEJMoa1214853&)

14. Chu TW, Kulkarni R, Gane EJ, et al. Effect of IL28B genotype on early viral kinetics during interferon-free treatment of patients with chronic hepatitis C. Gastroenterology. 2012;142:790–5. doi: 10.1053/j.gastro.2011.12.057.  [DOI](https://doi.org/10.1053/j.gastro.2011.12.057) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22248659/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gastroenterology&title=Effect%20of%20IL28B%20genotype%20on%20early%20viral%20kinetics%20during%20interferon-free%20treatment%20of%20patients%20with%20chronic%20hepatitis%20C&author=TW%20Chu&author=R%20Kulkarni&author=EJ%20Gane&volume=142&publication_year=2012&pages=790-5&pmid=22248659&doi=10.1053/j.gastro.2011.12.057&)

15. Hamming OJ, Terczynska-Dyla E, Vieyres G, et al. Interferon lambda 4 signals via the IFN lambda receptor to regulate antiviral activity against HCV and coronaviruses. EMBO J. 2013;32:3055–65. doi: 10.1038/emboj.2013.232.  [DOI](https://doi.org/10.1038/emboj.2013.232) | [PMC free article](/articles/PMC3844954/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24169568/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=EMBO%20J&title=Interferon%20lambda%204%20signals%20via%20the%20IFN%20lambda%20receptor%20to%20regulate%20antiviral%20activity%20against%20HCV%20and%20coronaviruses&author=OJ%20Hamming&author=E%20Terczynska-Dyla&author=G%20Vieyres&volume=32&publication_year=2013&pages=3055-65&pmid=24169568&doi=10.1038/emboj.2013.232&)
