# Aromatase Inhibitor-Associated Bone Fractures: A Case-Cohort GWAS and Functional Genomics

## Metadata
**Authors:** Mohan Liu, Paul E Goss, James N Ingle, Michiaki Kubo, Yoichi Furukawa, Anthony Batzler, Gregory D Jenkins, Erin E Carlson, Yusuke Nakamura, Daniel J Schaid, Judy-Anne W Chapman, Lois E Shepherd, Matthew J Ellis, Sundeep Khosla, Liewei Wang, Richard M Weinshilboum
**Journal:** Molecular Endocrinology
**Date:** 2014 Aug 22
**DOI:** [10.1210/me.2014-1147](https://doi.org/10.1210/me.2014-1147)
**PMID:** 25148458
**PMCID:** PMC4179631
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4179631/

## Abstract

Bone fractures are a major consequence of osteoporosis. There is a direct relationship between serum estrogen concentrations and osteoporosis risk. Aromatase inhibitors (AIs) greatly decrease serum estrogen levels in postmenopausal women, and increased incidence of fractures is a side effect of AI therapy. We performed a discovery case-cohort genome-wide association study (GWAS) using samples from 1071 patients, 231 cases and 840 controls, enrolled in the MA.27 breast cancer AI trial to identify genetic factors involved in AI-related fractures, followed by functional genomic validation. Association analyses identified 20 GWAS single nucleotide polymorphism (SNP) signals with P < 5E-06. After removal of signals in gene deserts and those composed entirely of imputed SNPs, we applied a functional validation “decision cascade” that resulted in validation of the CTSZ-SLMO2-ATP5E, TRAM2-TMEM14A, and MAP4K4 genes. These genes all displayed estradiol (E2)-dependent induction in human fetal osteoblasts transfected with estrogen receptor-α, and their knockdown altered the expression of known osteoporosis-related genes. These same genes also displayed SNP-dependent variation in E2 induction that paralleled the SNP-dependent induction of known osteoporosis genes, such as osteoprotegerin. In summary, our case-cohort GWAS identified SNPs in or near CTSZ-SLMO2-ATP5E, TRAM2-TMEM14A, and MAP4K4 that were associated with risk for bone fracture in estrogen receptor-positive breast cancer patients treated with AIs. These genes displayed E2-dependent induction, their knockdown altered the expression of genes related to osteoporosis, and they displayed SNP genotype-dependent variation in E2 induction. These observations may lead to the identification of novel mechanisms associated with fracture risk in postmenopausal women treated with AIs.

## Materials and Methods

### Source of patients and description of MA.27

Cases and controls were obtained from the MA.27 trial conducted by the NCIC Clinical Trials Group (coordinating group), Cancer and Leukemia Group B, Eastern Cooperative Oncology Group (ECOG), North Central Cancer Treatment Group, Southwest Oncology Group, and the International Breast Cancer Study Group (ClinicalTrials.gov number [NCT00066573](https://clinicaltrials.gov/ct2/show/NCT00066573)). Details of MA.27 have been published previously ([11](#B11)). In brief, MA.27 recruited postmenopausal women with histologically confirmed and completely resected stage I–III breast cancer (AJCC version 6) that was ER and/or progesterone receptor positive. Patients were randomized to 5 years of anastrozole or exemestane. Accrual of 6827 North American patients occurred between May 2003 and July 2008, with most patients providing DNA and consent for genetic testing. Non-North American patients (n = 693) entered by the International Breast Cancer Study Group did not contribute DNA. MA.27 initially included a second randomization to celecoxib (400 mg twice daily) or placebo, but this step was discontinued in December 2004 after the entry of 1622 patients because of reports of cardiovascular adverse events associated with celecoxib. The final results from MA.27, as mentioned previously ([11](#B11)), revealed no difference in efficacy between anastrozole and exemestane. We defined cases as patients who experienced a new fragility fracture at any time while they were on the trial that might be due to AI-related bone effects, specifically those in the spine, forearm, humerus, or proximal femur, from whom DNA was available and consent for genetic testing was given. The cases were selected by a committee (P.E.G., J.N.I., J.-A.W.C., L.E.S., S.K.) that included a specialist in osteoporosis (S.K.).

The case-cohort design was chosen so that future GWAS on different phenotypes would be economical by reusing a randomly chosen subcohort from the main clinical trial. The random subcohort included 876 randomly chosen subjects, of whom 36 had a fracture; all remaining 195 fracture cases were also included in analyses.

### Genotypes and quality control

Of the 1071 subjects included in the study, 184 had been genotyped previously for a study of musculoskeletal adverse events (MS-AEs) using Illumina Human610-Quad BeadChips ([12](#B12)), and the remaining 887 subjects for this study were genotyped using Illumina OmniExpress BeadChips. A total of 729 758 SNPs were genotyped with OmniExpress, but 37 489 (5.1%) of the SNPs were considered failures by the laboratory. Of the remaining 692 269 SNPs, 61 081 SNPs with a minor allele frequency (MAF) < 0.01 were excluded because of limited power for association analyses. An additional 1394 SNPs were removed based on the Hardy Weinberg Equilibrium exact test in PLINK based on controls only. For 2 pairs of duplicate samples, the percentage of discordant genotypes was less than 1.4E-3%, whereas 1 duplicate had 42% discordant genotypes, suggesting a sample mix-up and was excluded from further analyses. The percentage of Mendelian transmission errors was 4.3E-2%.

Because subjects had been genotyped on 2 different platforms, imputation was used to create a common set of SNPs for all subjects. The 1094 subjects from the “1000 Genome” reference population (11/23/2010 release) were used as reference. The software BEAGLE v3.3.1 was used, with 20-MB regions with 1-MB buffer on each side. Imputed genotypes with a dosage R2 < 0.3 were excluded from analyses. The total number of typed and imputed SNPs for the present fractures genotype study was 7 926 078. A similar process was used to impute SNPs for the previous MS-AE study ([12](#B12)), resulting in a total of 7 458 289 typed and imputed SNPs. After merging the 2 studies, the number of measured and imputed SNPs with MAF > 1% used for analysis was 7 560 631 SNPs.

### Statistical analyses

To control for potential population stratification, SNPs were chosen that were uncorrelated with each other to avoid local genomic linkage disequilibrium having undesirable impact on global genomic estimates of population stratification. SNPs were considered uncorrelated when the absolute value of the Pearson correlation was less than 0.063. This resulted in the use of 6476 SNPs to evaluate population structure. Eigenvectors and eigenvalues for the SNP correlations were computed by EIGENSTRAT ([23](#B23)). Eigenvectors corresponding to eigenvalues determined to be statistically different from 0 based on Tracy-Widom test (*P* < .05) were included as covariates in the analysis ([24](#B24)). This resulted in 5 potential eigenvectors to adjust for substructure and population stratification.

The end point for GWAS analyses was time from randomization to fracture, allowing for censored times. The Cox regression model (coxph in R software) with weights and robust variance estimates was used for the case-cohort design, as described by Therneau and Li ([25](#B25)). The next covariates were evaluated for their association with time to fracture: treatment (exemestane vs anastrazole), previous chemotherapy (yes vs no), age (<65 vs 65+ y), ECOG performance status (0, 1, and 2), previous surgery (mastectomy vs partial mastectomy), study (MS-AE vs current study), previous fractures (yes vs no), use of raloxifene (yes vs no), use of bisphosphonate (yes vs no), BMI (quantile cutpoints of 24.8, 28.4, and 32.2), TNM (tumor, nodes, metastases) stage (I, II, and III), and the first 5 eigenvectors (chosen because each had Tracy-Widom *P* < .05). After backward selection of covariates (*P* < .05), the following remaining covariates were used in the GWAS adjusted analyses: age, previous fractures, bisphosphonate use, and the first 3 eigenvectors.

### Patient characteristics

From the initial sample of 1115 subjects, 44 were removed for the following reasons: failed genotyping ([5](#B5)), sample mix-up ([2](#B2)), cancer recurrence before fracture ([11](#B11)), unknown date of fracture ([25](#B25)), and no follow-up after randomization ([1](#B1)). Among the 1071 subjects included in the analyses, 231 had a fracture and 840 were free of fracture.

### Cell lines

We used the Coriell Institute Human Variation Panel of LCLs obtained from 100 European-American (EA), 100 African-American (AA), and 100 Han Chinese-American healthy subjects to perform many of the functional genomic studies. These cell lines were generated from blood samples obtained by the National Institute of General Medical Sciences, followed by deposit in the Coriell Institute. We had previously genotyped DNA from these cell lines for genome-wide SNPs using Illumina 550K and 510S SNP BeadChips (Illumina). The Coriell Institute genotyped DNA from the same cell lines using the Affymetrix SNP Array 6.0 (Affymetrix), for a total of approximately 1.3 million unique genotyped SNPs per cell line ([17](#B17)). In addition, 1000 Genome data were used to impute approximately 7 million SNPs per cell line. We also generated basal Affymetrix U133 2.0 Plus GeneChip expression array data for all of the cell lines. This genomic data-rich LCL genomic model system has been described in detail elsewhere ([17](#B17)). The microarray and SNP data for these LCLs have been deposited in the National Center for Biotechnology Information Gene Expression Omnibus [http://www.ncbi.nlm.nih.gov/geo/](http://www.ncbi.nlm.nih.gov/geo/) under SuerSeries [GEO:*GSE24277*].

The human fetal osteoblast (hFOB)-ERα cell line was generated by stably transfecting hFOB cells with ERα constructs in the laboratory of Thomas Spelsberg, PhD, at the Mayo Clinic (Dr Spelsberg generously provided these cells for use in our studies). John Hawse, PhD, of the Mayo Clinic, provided consultative advice with regard to the hFOB-ERα cell line ([26](#B26)).

### Estradiol (E2) induction

Subconfluent cultures of hFOB-ERα cells were incubated in serum-free medium (DMEM) for 24 hours, followed by DMEM with 5% charcoal-stripped fetal bovine serum (FBS) for another 48 hours. Before exposure to E2, cells were grown in DMEM without serum for 24 hours, followed by 0.1 nmol/L E2 for 0, 1, 2, 4, 8, 12, 18, 24, 30, 36, 40, and 48 hours. Total RNA was then isolated from the cells with the RNeasy mini kit (QIAGEN). Two hundred nanograms of total RNA were used to perform quantitative RT-PCR (qRT-PCR) with primers for *TH1L*, *CTSZ*, *ATP5E*, *SLMO2*, *TRAM2*, *TMEM14A*, and *MAP4K4* (QIAGEN) as well as primers for β-actin (QIAGEN) as a control. All gene expression data was “normalized” on the basis of β-actin expression in the hFOB-ERα cells.

### LCL culture

Three LCLs from EA subjects homozygous for variant genotypes as well as 3 homozygous for wild type (WT) sequences for the *CTSZ-SLMO2-ATP5E* rs10485828 SNP, 3 LCLs from EA subjects homozygous for variant genotypes as well as 3 homozygous for WT sequences for the *TRAM2-TMEM14A* rs6901146 SNP, and 3 LCLs from AA subjects homozygous for variant as well as 3 homozygous for WT sequences for the *MAP4K4* rs4550690 SNP were selected for SNP-dependent induction analyses. These LCLs were cultured in RPMI 1640 media containing 15% (vol/vol) FBS. Before E2 treatment, 5E-06 cells for each cell line were cultured for 24 hours in RPMI 1640 containing 5% (vol/vol) charcoal-stripped FBS (Invitrogen), followed by culture in the same medium without FBS for an additional 24 hours. All cells were then cultured for 24 hours in 12-well plates with RPMI 1640 medium that contained 0nM, 0.01nM, 0.1nM, 1nM, and 10nM E2. Total RNA was isolated from the cells with the RNeasy mini kit (QIAGEN). Two hundred nanograms of total RNA were then used to perform qRT-PCR with *CTSZ*, *SLMO2*, *ATP5E*, *TH1L*, *TRAM2*, *TMEM14A*, *MAP4K4*, osteoprotegerin (*OPG*), *RANK*, and *RANKL* primers (QIAGEN). All gene expression levels were normalized on the basis of ERα expression in each of the cell lines studied.

### Gene knockdown

To perform knockdown studies, hFOB-ERα cells were transfected with *CTSZ*, *SLMO2*, *ATP5E*, *TH1L*, *TRAM2*, *TMEM14A*, and *MAP4K4* small interfering RNA smart pools and negative small interfering RNA (Dharmacon) using lipofectamin 2000 (Invitrogen) for 48 hours. Total RNA was isolated from the hFOB-ERα-transfected cells using the Quick RNA miniprep kit (Zymo Research), and 200 ng of that RNA were used to perform qRT-PCR with primers for all 91 genes on the “vitamin D receptor in regulation of genes involved in osteoporosis” panel (Bio-Rad).

## Results

### MA.27 bone fracture GWAS

The final GWAS involved 231 cases with fragility fractures and 840 controls. The characteristics of these patients are listed in [Table 1](#T1). GWAS genotyping was performed using the Illumina Omni Express for 887 patients (83%). As described in Materials and Methods, GWAS genotyping for 184 of the patients (17%) ([12](#B12)) had been performed previously with the Human610 Quad Beadchip. Even though we used samples from the largest adjuvant AI clinical trial available, we assumed that many of the “signals” observed during the GWAS would be false positives, so we planned from the beginning to apply a series of predetermined functional genomic criteria that narrowed our initial list of “SNP signals” to a final group of signals that met the criteria of a functional genomic “decision cascade” that we established, thus opening the way for future exploration of the possible role of these genes in bone biology and in the pathophysiology of osteoporosis.

### Table 1.

|   | Cases (n = 231) | Controls (n = 840) |
| --- | --- | --- |
| Age, years |   |   |
| Median | 68.7 | 64.2 |
| Range | 46.1-89.8 | 35.9-88.9 |
| Body mass index |   |   |
| Number with data | 227 | 836 |
| Median | 28.6 | 28.4 |
| Quartile 1, quartile 4 | 24.8, 32.4 | 24.9, 32.2 |
| Range | 17.4-66.8 | 16.5-61.3 |
| Race |   |   |
| Asian | 2 (0.9%) | 12 (1.4%) |
| Black | 5 (2.2%) | 22 (2.6%) |
| Hawaiian or Pacific Islander | 0 (0.0%) | 1 (0.1%) |
| Unknown | 0 (0.0%) | 3 (0.4%) |
| White | 224 (97.0%) | 802 (95.5%) |
| Previous fracture |   |   |
| No | 186 (80.5%) | 758 (90.2%) |
| Yes | 45 (19.5%) | 82 (9.8%) |
| Previous chemotherapy |   |   |
| Yes | 57 (24.7%) | 255 (30.4%) |
| No | 174 (75.3%) | 585 (69.6%) |
| Surgery |   |   |
| Mastectomy | 75 (32.5%) | 274 (32.6%) |
| Partial mastectomy | 156 (67.5%) | 566 (67.4%) |
| Treatment |   |   |
| Exemestane | 116 (50.2%) | 420 (50.0%) |
| Anastrozole | 115 (49.8%) | 420 (50.0%) |
| ECOG performance score |   |   |
| 0 | 178 (77.1%) | 699 (83.2%) |
| 1 | 51 (22.1%) | 132 (15.7%) |
| 2 | 2 (0.9%) | 9 (1.1%) |
| TNM stage |   |   |
| I | 126 (54.5%) | 502 (59.8%) |
| II | 81 (35.1%) | 270 (32.1%) |
| III | 19 (8.2%) | 51 (6.1%) |
| T1-3NXM0 | 5 (2.2%) | 13 (1.5%) |
| TXN0M0 | 0 (0.0%) | 4 (0.5%) |
| Previous raloxifene |   |   |
| Ever | 4 (1.7%) | 14 (1.7%) |
| Never | 227 (98.3%) | 819 (97.5%) |
| Missing | 0 (0%) | 7 (0.8%) |
| Bisphosphonate use |   |   |
| Ever | 122 (52.8%) | 270 (32.1%) |
| Never | 94 (40.7%) | 507 (60.4%) |
| Missing | 15 (6.5%) | 63 (7.5%) |

Table 1 Caption: Patient Characteristics

### Functional genomic studies

As a first step in the functional validation process for GWAS signals, we selected the “top” SNP signals from the Manhattan plot shown in [Figure 1](#F1)A. The quantile-quantile plot for these data is shown in [Supplemental Figure 1](http://press.endocrine.org/doi/suppl/10.1210/me.2014-1147/suppl_file/me-14-1147.pdf). In [Figure 1](#F1)A, we have highlighted the SNP signals and associated genes that passed through the decision cascade successfully. There were a total of 20 signals with *P* < 5E-06 and 22 genes that mapped in or near those signals ([Table 2](#T2)). If the SNP signal was over a gene cluster, we listed all of the genes in that cluster in [Table 2](#T2), eg, *CTSZ-SLMO2-ATP5E-TH1L* for the chromosome 20 SNP signal. [Figure 1](#F1), B–D, shows LocusZoom plots for the 3 SNP signals and related genes that subsequently were validated functionally. None of the original 20 SNP signals met the generally accepted value for genome-wide significance of 5.0E-08, which was not surprising because of the relatively small number of “cases” in the trial, even though MA.27 represented the largest adjuvant AI trial available. It should be emphasized that we have previously pursued GWAS SNP signals that were not genome-wide significant to successfully demonstrate the biological importance of those signals ([12](#B12), [21](#B21)) but always with the understanding that many of these signals may represent false positives. Based on that experience, we developed a series of criteria for functional pursuit of the signals observed in the present study ([Figure 2](#F2)).

### Figure 1.

![Figure 1.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/adcf/4179631/8e55b4933952/zmg9991454300001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4179631_zmg9991454300001.jpg)

A, GWAS Manhattan plot GWAS performed with P values adjusted for age, previous fracture, bisphosphonate use, and the first 3 eigenvectors. The 3 SNP signals that underwent functional genomic validation are highlighted. B, LocusZoom plot of the chromosome 20 region surrounding the THIL-TUBB1-CTSZ-SLMO2-ATP5E genes showing a plot of −log10 (P values) for observed (circles) and imputed SNPs (diamonds). C, LocusZoom plot of the chromosome 6 region surrounding the TRAM2-TMEM14A genes. D, LocusZoom plot of the chromosome 2 region surrounding the MAP4K4 gene.

### Table 2.

| Top SNPs in Signals | Chr | P Values | Genes | Validation Outcome |  |
| --- | --- | --- | --- | --- | --- |
| 1. rs10967942 | Chr 9 | 2.01E-07 | KCNV2 | Not expressed in LCLs |   |
| 2. rs10485828 | Chr 20 | 2.56E-07 | TH1L, TUBB1, | CTSZ-SLMO2-ATP5E final validated genes, |  |
|   |   |   | CTSZ, SLMO2, | TUBB1 not expressed in LCLs, |  |
|   |   |   | ATP5E | TH1L failed functional validation |  |
| 3. rs12248467 | Chr 10 | 2.93E-07 | PRKG1 | Not expressed in LCLs |  |
| 4. rs76996285 | Chr 13 | 3.81E-07 | PABPC3 | All SNPs were imputed |  |
| 5. rs8092654 | Chr 18 | 4.41E-07 | DLGAP1 | Not expressed in LCLs |  |
| 6. rs6842571 | Chr 4 | 8.32E-07 | ANTXR2, PRDM8 | Not expressed in LCLs |  |
| 7. rs66520040 | Chr 13 | 8.52E-07 | FLT3 | Not expressed in LCLs |  |
| 8. rs6901146 | Chr 6 | 1.15E-06 | TRAM2, TMEM14A | TRAM2-TMEM14A final validated genes |  |
| 9. rs6591876 | Chr 11 | 1.47E-06 |   | Gene desert |  |
| 10. rs112830684 | Chr 3 | 1.57E-06 | ASB14, DNAH12 | Not expressed in LCLs |  |
| 11. rs7999876 | Chr 13 | 2.10E-06 |   | Gene desert |  |
| 12. c13.82322180.B37P0 | Chr 13 | 2.19E-06 |   | Gene desert |  |
| 13. rs73728712 | Chr 7 | 2.25E-06 | DGKI | Not expressed in LCLs |  |
| 14. rs61399156 | Chr 7 | 2.87E-06 | DOCK4 | Not expressed in LCLs |  |
| 15. rs4550690 | Chr 2 | 2.89E-06 | MAP4K4 | MAP4K4 final validated gene |  |
| 16. rs35112095 | Chr 21 | 4.02E-06 | SLC37A1 | Not expressed in LCLs |  |
| 17. rs9878448 | Chr 3 | 4.02E-06 | GADL1 | Not expressed in LCLs |  |
| 18. rs17015762 | Chr 3 | 4.11E-06 |   | Gene desert |  |
| 19. rs73511817 | Chr 9 | 4.54E-06 |   | Gene desert |  |
| 20. rs117996576 | Chr 8 | 4.64E-06 | TUSC3 | Not expressed in LCLs |  |

Table 2 Caption: GWAS Signals and Related Genes from the MA.27 GWAS

### Figure 2.

![Figure 2.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/adcf/4179631/d84c0e0f8442/zmg9991454300002.jpg)

Validation decision cascade used to test GWAS signals. The association analysis shown graphically in the Manhattan plot in Figure 1A resulted in the identification of 20 SNP signals with P < 5E-06. The figure depicts graphically the process for SNP signal and associated gene functional validation.

Several of the steps in the validation cascade depicted graphically in [Figure 2](#F2) were based on our practical ability to experimentally test genes identified during the GWAS using cell line systems, especially our genomic data-rich LCL model system that has repeatedly demonstrated its power both to generate and test pharmacogenomic hypotheses ([16](#B16)[–](#B17)[21](#B21)). One of the major advantages of the LCL system is the ability that it gives us to test common SNP genotypes, because each cell line has been genotyped for approximately 1.3 million SNPs, with a final total of 7 million SNPs after imputation. This LCL system also makes it possible to correlate expression among candidate genes and genes known to play a role in disease pathophysiology or drug response. However, the LCL system, despite its repeated success, also has limitations, including the fact that these are not osteoblastic cells. Therefore, we also used additional cell line systems, eg, hFOB cells (osteoblasts), in our studies. Finally, we were also able to take into account the possible relationship of genes identified in our GWAS to genes identified during previous osteoporosis GWAS performed using postmenopausal women who had not been exposed to AIs.

As diagrammed graphically in [Figure 2](#F2), if a signal was composed entirely of imputed SNPs or if it mapped to a gene desert (no annotated genes within ±100 kb), we did not pursue it, reducing the number of signals from 20 to 14 and the number of genes to 21 ([Figure 2](#F2)). We next removed 11 SNP signals and 14 genes from consideration that were not expressed in our LCL model system, making it impossible to take advantage of the power of this cell line system to perform functional genomic studies, leaving 3 SNP signals and 7 genes. The 7 remaining genes were then subjected to a series of functional genomic tests, resulting in a final group of 3 SNP signals and 6 genes that were identified during the GWAS and that also successfully passed through the validation cascade ([Table 2](#T2)). All 6 of those genes were induced by E2 (ie, they were linked to the drug action), their expression paralleled the expression of known osteoporosis related genes (ie, they were linked to the known biology of osteoporosis), and their E2 induction occurred in a SNP genotype-dependent fashion, thus linking them to the GWAS SNP signals that originally resulted in their identification. Subsequent paragraphs will describe each these steps in detail.

### Genes that displayed estrogen-dependent induction

The patients enrolled in MA.27 had been treated with AIs to block estrogen synthesis. Therefore, as a first step in our functional pharmacogenomic studies, we took advantage of the known drug effect to determine whether mRNA expression for *TH1L-CTSZ-SLMO2-ATP5E*, *TRAM2-TEME14A*, or *MAP4K4* might be E2 dependent, thus linking the candidate genes identified during the GWAS to the drug effect. These experiments involved exposing hFOB cells stably transfected with ERα (hFOB-ERα) to 0.1 nmol/L E2 for varying times. We observed a greater than 8-fold increase in ATP5E (ATP synthase, H^+^ transporting mitochondrial F1 complex, Epsilon Subunit) expression, an 8-fold increase in cathepsin Z (CTSZ) expression, a 7-fold increase in Slowmo Homolog 2 (Drosophila) (SLMO2) mRNA expression, a 35-fold increase in Trihydrophobin 1 (TH1L) mRNA expression, a 27-fold increase in Translational associated membrane protein 2 (TRAM2) mRNA expression, a 5-fold increase in transmembrane protein 14A (TMEM14A) mRNA expression, and a 31-fold increase in MAPK kinase kinase kinase 4 (MAP4K4) mRNA expression after 48 hours of incubation with 0.1nM E2 ([Figure 3](#F3)). The results shown in [Figure 3](#F3) demonstrated that the expression of these 7 genes, *ATP5E*, *CTSZ*, *SLMO2*, *TH1L*, *TRAM2*, *TMEM14A*, and *MAP4K4* in hFOB-ERα cells was estrogen dependent, so all of these genes were moved to the next step in the functional validation process. By using the TRANSFAC-AliBaba 2.1 and ENCODE databases, we were unable to identify any estrogen response elements that were disrupted or created by the top SNPs in these 3 SNP signals (see Supplemental Table 1). The mechanism(s) responsible for the E2-dependent induction of the expression of these 7 genes remains unclear.

### Figure 3.

![Figure 3.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/adcf/4179631/31109225c034/zmg9991454300003.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4179631_zmg9991454300003.jpg)

E2 induction time course for THIL, CTSZ, SLMO2, ATP5E, TRAM2, TEME14A, and MAP4K4 mRNA expression in hFOB-ERα cells exposed to 0.1 nmol/L E2 for differing periods of time. The values shown represent means of duplicate assays.

### Correlation of expression with that of osteoporosis panel genes

Receptor activator of nuclear factor κB (RANK)-RANK ligand (RANKL)-OPG and the Wnt pathways are all known to play important roles in the pathophysiology of osteoporosis ([27](#B27)), so we used a Bio-Rad Osteoporosis Genes Primer Panel as the next screening test. This panel had been constructed to allow rapid testing of the expression of a group of genes known to play a role in bone biology. For example, when the expression of *SLMO2*, one of our candidate genes, was knocked down to 23% of its basal level in hFOB-ERα cells, the expression of several osteoporosis-related genes in the panel, but especially that of RANK, displayed a decrease in expression to less than 50% of the basal level, whereas others, including RANKL, increased ([Figure 4](#F4)A). When the expression of TRAM2, another of our candidate genes, was knocked down to 9% of its basal level in the same cells, the expression of RANKL decreased more than 5-fold ([Figure 4](#F4)B). Patterns of expression after the knockdown of the remaining candidate genes in hFOB-ERα cells are shown in Supplemental Figure 2. These results led us to move on to the LCL model system to determine whether there might be a correlation between levels of expression of our 7 GWAS candidate genes with genes that had been identified in the course of previous osteoporosis GWAS performed with postmenopausal women, a setting lacking the striking pharmacological provocation represented by prolonged AI therapy.

### Figure 4.

![Figure 4.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/adcf/4179631/19b10ccd3162/zmg9991454300004.jpg)

Effect of knockdown of SLMO2 (A) and TRAM2 (B) on the relative mRNA expression of genes involved in the osteoporosis genes primer panel in hFOB-ERα cells. Red lines represent basal control levels of expression. Data for the RANK-RANKL-OPG genes are highlighted with red rectangles.

### Genes with expression that correlated with that of osteoporosis-related genes in the Human Variation Panel LCL model system

As the next step in functional validation, we asked whether the expression of genes identified during previous osteoporosis risk GWAS performed using samples from postmenopausal women who had not been exposed to AI therapy ([28](#B28)) might be correlated with variation in the expression of the 7 candidate genes from our AI-associated fracture GWAS, ie, *CTSZ*, *SLMO2*, *ATP5E*, *TH1L*, *TMEME14A*, *TRAM2*, and *MAP4K4*. To test that possibility, we used baseline mRNA expression data for 54000 Affymetrix gene expression probes for all 300 of the LCLs in the Human Variation Panel. Those data made it possible to determine whether any of the 7 genes identified during our GWAS might have expression that was correlated with the expression of genes identified during previous osteoporosis GWAS ([15](#B15)), with the clear understanding that LCLs are obviously neither osteoblasts nor osteoclasts. By using a cut-off *P* < 5E-06, all 7 of the remaining MA.27 GWAS genes displayed significant correlations with the expression of genes previously reported to be related to osteoporosis risk (see [Table 3](#T3)). Furthermore, variation in the expression of the many of these genes was highly correlated with that of the RANK-RANKL-OPG genes. In addition, the expression of genes within or near the chromosome 20 SNP signal (*CTSZ*, *SLMO2*, and *ATP5E*) as well as 2 genes near the chromosome 6 SNP signal (*TEME14A* and *TRAM2*) were also correlated ([Table 4](#T4)). It of interest that 2 of our final SNP signals involved gene clusters. It is possible that a single SNP signal can be associated with the expression of several genes in such a gene “cluster.” For example, the LCL model system data indicated that *ATP5E* expression was highly correlated with the expression of 2 other genes, *SLMO2* and *CTSZ*, in the same cluster ([Table 4](#T4) and [Figure 1](#F1)B). In a similar fashion, *TRAM2* expression was highly correlated with the expression of *TMEM14A*. Both of these genes mapped near the chromosome 6 SNP signal ([Table 4](#T4) and [Figure 1](#F1)C). As the next step in our analysis, we took advantage of the genome-wide SNP genotype data that we had generated for these 300 LCLs to determine whether different SNP genotypes might be related to variation in expression of the candidate genes during E2 exposure.

### Table 3.

| Genes | Genes | Spearman ρ | P Values |
| --- | --- | --- | --- |
| MA.27 GWAS | Osteoporosis GWAS |   |   |
| CTSZ | TNFRSF11A (RANK) | −0.3462 | 1.7E-09 |
| CTSZ | SPTBN1 | −0.3341 | 6.6E-09 |
| CTSZ | WNT10B | 0.3079 | 1.0E-07 |
| SLMO2 | FUBP3 | 0.4043 | 1.0E-12 |
| SLMO2 | SPTBN1 | −0.3369 | 4.8E-09 |
| TH1L | C17orf53 | 0.3104 | 7.9E-08 |
| TH1L | TNFRSF11A (RANK) | −0.3179 | 3.7E-08 |
| ATP5E | SPTBN1 | −0.3620 | 3.0E-10 |
| TMEM14A | TNFRSF11A (RANK) | 0.3697 | 1.0E-10 |
| TMEM14A | SLC25A13 | 0.3004 | 2.1E-07 |
| TRAM2 | SPTBN1 | 0.2803 | 1.4E-06 |
| TRAM2 | C18orf192 | −0.2887 | 6.5E-07 |
| TRAM2 | TNFRSF11A (RANK) | −0.3357 | 5.5E-09 |
| TRAM2 | TNFRSF11B (OPG) | −0.2645 | 5.6E-06 |
| TRAM2 | SLC25A131 | −0.3324 | 7.9E-09 |
| MAP4K4 | CTNNB1 | 0.2643 | 5.7E-06 |
| MAP4K4 | WNT6 | 0.3407 | 3.1E-09 |

Table 3 Caption: Correlation Between the Expression of MA.27 GWAS Candidate Genes and Previously Reported Osteoporosis Risk GWAS Genes in the Human Variation Panel LCL Model System

### Table 4.

| Genes MA.27 GWAS | Genes MA.27 GWAS | Spearman ρ | Prob> ρ |
| --- | --- | --- | --- |
| ATP5E | SLMO2 | 0.3665 | 1.00E-10 |
| ATP5E | CTSZ | 0.4431 | 3.00E-15 |
| SLMO2 | CTSZ | −0.1883 | 1.30E-03 |
| TMEM14A | TRAM2 | −0.2553 | 1.20E-05 |

Table 4 Caption: Correlations Among the Expression of the 7 MA.27 GWAS Candidate Genes in the Human Variation Panel LCL Model System

### Genes with SNP- and E2-dependent expression correlated with that of osteoporosis-related genes

We next performed an important test of the possible relationship of GWAS SNPs in or near the 7 remaining candidate genes, *CTSZ-SLMO2-ATP5E-TH1L*, *TRAM2-TMEM14A*, and *MAP4K4*. Specifically, we exposed LCLs from 3 EA subjects who were homozygous for WT SNP genotypes and 3 LCLs that were homozygous for variant genotypes in the signals from the *CTSZ-SLMO2-ATP5E-TH1L* gene cluster (see [Figure 1](#F1)B) to increasing concentrations of E2 for 24 hours. We also exposed LCLs from 3 EA subjects who were homozygous for WT SNP genotypes and 3 who were homozygous for variant SNP genotypes for the *TRAM2-TMEM14A* signal to increasing concentrations of E2 for 24 hours. For the *MAP4K4* SNPs, we exposed LCLs from 3 AA subjects who were homozygous for WT SNP genotypes and 3 AA subjects who were homozygous for variant genotypes to increasing concentration of E2 for 24 hours. We used cell lines from AA subjects for the *MAP4K4* experiments because of the higher MAF for these SNPs in AA subjects. The purpose of these experiments was to determine whether differences in SNP genotypes might result in differential E2-dependent induction of expression of the candidate genes and, in parallel, the expression of genes known to be related to osteoporosis.

Cell lines homozygous for WT SNP genotypes showed significantly greater E2 dose-dependent induction of *CTSZ*, *SLMO2*, and *ATP5E* expression after exposure to increasing concentrations of E2 than did LCLs homozygous for variant genotypes after exposure to E2 for 24 hours ([Figures 5](#F5)A). However, that was not the case for *TH1L*. Of importance, the *OPG* gene also showed greater induction in cell lines that carried the WT genotypes for the *CTSZ-SLMO2-ATP5E* gene cluster than did those homozygous for variant SNP genotypes. We included OPG in these experiments because of its important role in bone physiology ([29](#B29)). We should emphasize that the SNPs being studied were in or near the *CTSZ*, *SLMO2*, and *ATP5E* genes. They were not in the *OPG* gene, which is on a different chromosome, chromosome 8, but these SNP genotypes were associated with differential E2-dependent induction of *OPG*. Because it failed to display SNP-dependent estrogen induction, we removed *TH1L* from our list of candidate genes. In a similar fashion, cell lines homozygous for WT genotypes showed significantly greater E2 dose-dependent induction of *TMEM14A-TRAM2* expression after exposure to increasing concentrations of E2 than did LCLs homozygous for variant genotypes after 24 hours of E2 exposure ([Figure 5](#F5)B). In this case, the *OPG* gene showed greater induction in cell lines that carried the variant genotypes for *TMEM14A-TRAM2* than did those homozygous for WT SNP genotypes. Finally, cell lines homozygous for WT genotypes for the *MAP4K4* SNPs also showed significantly greater E2 dose-dependent induction of *MAP4K4* and, in parallel, greater *OPG* expression after exposure to increasing concentrations of E2 than did LCLs homozygous for variant genotypes after 24 hours ([Figure 5](#F5)C).

### Figure 5.

![Figure 5.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/adcf/4179631/8dd5f8a844dd/zmg9991454300005.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4179631_zmg9991454300005.jpg)

LCL SNP-dependent variation of candidate gene expression in response to increasing concentrations of E2 for 3 homozygous W/W (blue lines) and 3 homozygous V/V (pink lines) LCLs. A, Relative CTSZ, SLMO2, and ATP5E mRNA expression (upper panels) and relative mRNA expression for OPG and THIL (lower panels). B, SNP-dependent variation in TMEM14A-TRAM2 expression in response to increasing concentrations of E2 for 3 homozygous W/W (blue lines) and 3 homozygous V/V (pink lines) LCLs (upper panels) and relative mRNA expression level for OPG (lower left panel). C, SNP-dependent variation in MAP4K4 expression in response to increasing concentrations of E2 for 3 homozygous W/W (blue lines) and 3 homozygous V/V (pink lines) AA LCLs (upper panel) and relative mRNA expression level for OPG (lower panel). *, P < .05 and **, P < .01 when compared with the alternative genotype at the same concentration of E2.

## Discussion

Osteoporosis and bone fractures associated with osteoporosis represent a significant cause of morbidity and mortality worldwide ([30](#B30)). There is a direct relationship between serum estrogen concentrations and risk for osteoporosis ([2](#B2)). AI therapy decreases estrogen levels in postmenopausal women by more than 98% ([31](#B31)). As a result, the occurrence of bone fracture represents a significant side effect of the use of AIs to treat ER-positive breast cancer patients ([10](#B10)). In the present study, we took advantage of a unique opportunity presented by DNA samples from patients enrolled in MA.27, the largest adjuvant AI breast cancer trial available, to perform a case-cohort discovery GWAS to identify possible biomarkers for AI treatment-associated bone fractures and then to validate those signals functionally. Specifically, we preselected a series of criteria designed to make it practically possible to focus our attention on potential true positive GWAS signals, with the understanding that taking this approach meant that we might miss meaningful signals. We started with a total of 20 SNP signals. After moving through the validation criteria shown graphically in [Figure 2](#F2), we identified 3 SNP signals and 6 genes that met all of the criteria listed in the figure. Each of those genes, *CTSZ-SLMO2-ATP5E*, *TRAM2-TMEM14A*, and *MAP4K4*, displayed E2-dependent induction of mRNA expression ([Figure 3](#F3)), and when we knocked down their mRNA, expression of genes in pathways associated with osteoporosis risk and/or bone biology was altered ([Figure 4](#F4) and Supplemental Figure 2). Furthermore, the expression of these 6 genes was correlated with the expression of genes identified during previous osteoporosis GWAS ([Tables 3](#T3) and [4](#T4)). Finally, and most important, the E2 induction of these 6 genes was SNP dependent and paralleled the E2-dependent induction of the osteoporosis-related gene encoding OPG, even though the SNPs were in or near the genes identified during our AI fracture GWAS ([Figure 5](#F5)). The approach applied here is similar to a strategy that we applied previously, in which we used a GWAS to identify genes and SNPs associated with severe musculoskeletal pain during AI therapy to identify a SNP signal near the *TCL1A* gene that led to the identification of a novel estrogen-dependent and SNP-dependent mechanism for the regulation of cytokine and chemokine expression ([12](#B12)[–](#B13)[14](#B14)).

Specifically, we began by linking the candidate genes identified during the GWAS to the effect of AI treatment by studying possible estrogen effect on the mRNA expression of the genes ([Figure 3](#F3)). We then determined a possible relationship of the expression of the candidate genes to the expression of genes known to be involved in bone physiology or osteoporosis biology ([Figure 4](#F4) and Table), thus linking the genes to the clinical phenotype, bone fracture. Finally, we linked the SNP genotypes in the GWAS signals to the SNP-dependent, estrogen-dependent induction of expression of the candidate genes and, in parallel, of the osteoporosis-related gene *OPG*, presumably “downstream” of the induction of our candidate genes ([Figure 5](#F5)).

It was encouraging that one of the genes identified during our GWAS, *CTSZ*, cathepsin Z, a member of cysteine proteinase family, had already been associated with bone remodeling in osteoclasts ([32](#B32)[–](#B33)[37](#B37)), although the mechanism is not well understood. At present, mechanisms for the possible relationship of the remaining genes, *SLMO2*, *ATP5E*, *TMEM14A*, *TRAM2*, and *MAP4K4*, to AI-dependent bone fracture risk remain unclear. However, TRAM2 is a known target for Runt-related transcription factor 2, the “master” osteoblast differentiation transcription factor for bone morphogenetic protein signaling in the control of the α1(I) collagen promoter, leading to low bone mass and spontaneous bone fractures when Runt-related transcription factor 2 is overexpressed ([38](#B38)). MAP4K4 positively regulates TNFα and nuclear factor κB signaling pathways, both of which play important roles in osteoporosis ([39](#B39), [40](#B40)). We should point out that none of the genes that we observed and functionally validated have been identified during previous osteoporosis risk GWAS. We should also emphasize that bone fracture during 5 years of AI therapy represents a phenotype that differs from postmenopausal osteoporosis, because AI treatment decreases circulating estrogen concentrations in these postmenopausal women to levels that are significantly lower than those observed in the absence of drug therapy ([4](#B4)). In addition, most osteoporosis risk GWASs have used BMD as a phenotype, with only a few using fracture risk as a phenotype ([20](#B20)).

In summary, our case-cohort discovery GWAS has identified SNPs in or close to *CTSZ-SLMO2-ATP5E*, *TMEM14A-TRAM2*, and *MAP4K4* that were associated with risk for bone fracture in postmenopausal breast cancer patients treated with AIs for 5 years. The effect of these SNPs may be related to SNP-dependent, estrogen-dependent regulation of the expression of these genes and the possible downstream regulation of the expression of RANK-RANKL-OPG and other genes related to osteoporosis. Our results may not only provide information with regard to the pathophysiology of and biomarkers for AI treatment-associated bone fractures in postmenopausal breast cancer patients treated with AIs, but they may also, based on the results of appropriate follow-up studies, provide insight into novel biological mechanisms related to estrogen effect in bone biology and pathophysiology.

## Acknowledgments

We thank the contributions of the women who participated in MA.27 and provided DNA and consent for its use in genetic studies.

This work was supported in part by National Institutes of Health Grants U19 GM61388 (The Pharmacogenomics Research Network), U01 HG005137, R01 CA138461, R01 CA133049, and P50 CA166201 (Mayo Clinic Breast Cancer Specialized Program of Research Excellence); a Pharmaceutical Research and Manufacturers of America Foundation Center of Excellence in Clinical Pharmacology award; the Mayo Center for Individualized Medicine; and the Rikagaku Kenkyūsho Center for Integrative Medical Science and the Biobank Japan Project funded by the Ministry of Education, Culture, Sports, and Technology, Japan. P.E.G. is supported in part by the Avon Foundation, New York; ClinicalTrials.gov study [NCT00283608](https://clinicaltrials.gov/ct2/show/NCT00283608).

Disclosure Summary: The authors have nothing to disclose.

## Footnotes

## References

1. Lewiecki EM, Binkley N. Evidence-based medicine, clinical practice guidelines, and common sense in the management of osteoporosis. Endocr Pract. 2009;15:573–579  [DOI](https://doi.org/10.4158/EP09107.RA) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19491062/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Endocr%20Pract&title=Evidence-based%20medicine,%20clinical%20practice%20guidelines,%20and%20common%20sense%20in%20the%20management%20of%20osteoporosis&author=EM%20Lewiecki&author=N%20Binkley&volume=15&publication_year=2009&pages=573-579&pmid=19491062&doi=10.4158/EP09107.RA&)

2. Mendoza N, Quereda F, Presa J, et al. Estrogen-related genes and postmenopausal osteoporosis risk. Climacteric. 2012;15:587–593  [DOI](https://doi.org/10.3109/13697137.2012.656160) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22335445/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Climacteric&title=Estrogen-related%20genes%20and%20postmenopausal%20osteoporosis%20risk&author=N%20Mendoza&author=F%20Quereda&author=J%20Presa&volume=15&publication_year=2012&pages=587-593&pmid=22335445&doi=10.3109/13697137.2012.656160&)

3. Grimm SW, Dyroff MC. Inhibition of human drug metabolizing cytochromes P450 by anastrozole, a potent and selective inhibitor of aromatase. Drug Metab Dispos. 1997;25:598–602  [PubMed](https://pubmed.ncbi.nlm.nih.gov/9152599/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Inhibition%20of%20human%20drug%20metabolizing%20cytochromes%20P450%20by%20anastrozole,%20a%20potent%20and%20selective%20inhibitor%20of%20aromatase&author=SW%20Grimm&author=MC%20Dyroff&volume=25&publication_year=1997&pages=598-602&pmid=9152599&)

4. Brueggemeier RW. Aromatase, aromatase inhibitors, and breast cancer. Am J Ther. 2001;8:333–344  [DOI](https://doi.org/10.1097/00045391-200109000-00007) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11550075/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Ther&title=Aromatase,%20aromatase%20inhibitors,%20and%20breast%20cancer&author=RW%20Brueggemeier&volume=8&publication_year=2001&pages=333-344&pmid=11550075&doi=10.1097/00045391-200109000-00007&)

5. Confavreux CB, Fontana A, Guastalla JP, Munoz F, Brun J, Delmas PD. Estrogen-dependent increase in bone turnover and bone loss in postmenopausal women with breast cancer treated with anastrozole. Prevention with bisphosphonates. Bone. 2007;41:346–352  [DOI](https://doi.org/10.1016/j.bone.2007.06.004) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17618847/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bone&title=Estrogen-dependent%20increase%20in%20bone%20turnover%20and%20bone%20loss%20in%20postmenopausal%20women%20with%20breast%20cancer%20treated%20with%20anastrozole.%20Prevention%20with%20bisphosphonates&author=CB%20Confavreux&author=A%20Fontana&author=JP%20Guastalla&author=F%20Munoz&author=J%20Brun&volume=41&publication_year=2007&pages=346-352&pmid=17618847&doi=10.1016/j.bone.2007.06.004&)

6. Coleman RE, Banks LM, Girgis SI, et al. Skeletal effects of exemestane on bone-mineral density, bone biomarkers, and fracture incidence in postmenopausal women with early breast cancer participating in the Intergroup Exemestane Study (IES): a randomised controlled study. Lancet Oncol. 2007;8:119–127  [DOI](https://doi.org/10.1016/S1470-2045(07)70003-7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17267326/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet%20Oncol&title=Skeletal%20effects%20of%20exemestane%20on%20bone-mineral%20density,%20bone%20biomarkers,%20and%20fracture%20incidence%20in%20postmenopausal%20women%20with%20early%20breast%20cancer%20participating%20in%20the%20Intergroup%20Exemestane%20Study%20(IES):%20a%20randomised%20controlled%20study&author=RE%20Coleman&author=LM%20Banks&author=SI%20Girgis&volume=8&publication_year=2007&pages=119-127&pmid=17267326&doi=10.1016/S1470-2045(07)70003-7&)

7. Ingle JN. Overview of adjuvant trials of aromatase inhibitors in early breast cancer. Steroids. 2011;76:765–767  [DOI](https://doi.org/10.1016/j.steroids.2011.02.021) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21382394/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Steroids&title=Overview%20of%20adjuvant%20trials%20of%20aromatase%20inhibitors%20in%20early%20breast%20cancer&author=JN%20Ingle&volume=76&publication_year=2011&pages=765-767&pmid=21382394&doi=10.1016/j.steroids.2011.02.021&)

8. Cuzick J, Sestak I, Baum M, et al. Effect of anastrozole and tamoxifen as adjuvant treatment for early-stage breast cancer: 10-year analysis of the ATAC trial. Lancet Oncol. 2010;11:1135–1141  [DOI](https://doi.org/10.1016/S1470-2045(10)70257-6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21087898/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet%20Oncol&title=Effect%20of%20anastrozole%20and%20tamoxifen%20as%20adjuvant%20treatment%20for%20early-stage%20breast%20cancer:%2010-year%20analysis%20of%20the%20ATAC%20trial&author=J%20Cuzick&author=I%20Sestak&author=M%20Baum&volume=11&publication_year=2010&pages=1135-1141&pmid=21087898&doi=10.1016/S1470-2045(10)70257-6&)

9. Rabaglio M, Sun Z, Price KN, et al. Bone fractures among postmenopausal patients with endocrine-responsive early breast cancer treated with 5 years of letrozole or tamoxifen in the BIG 1-98 trial. Ann Oncol. 2009;20:1489–1498  [DOI](https://doi.org/10.1093/annonc/mdp033) | [PMC free article](/articles/PMC2731016/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19474112/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Oncol&title=Bone%20fractures%20among%20postmenopausal%20patients%20with%20endocrine-responsive%20early%20breast%20cancer%20treated%20with%205%20years%20of%20letrozole%20or%20tamoxifen%20in%20the%20BIG%201-98%20trial&author=M%20Rabaglio&author=Z%20Sun&author=KN%20Price&volume=20&publication_year=2009&pages=1489-1498&pmid=19474112&doi=10.1093/annonc/mdp033&)

10. Amir E, Seruga B, Niraula S, Carlsson L, Ocaña A. Toxicity of adjuvant endocrine therapy in postmenopausal breast cancer patients: a systematic review and meta-analysis. J Natl Cancer Inst. 2011;103:1299–1309  [DOI](https://doi.org/10.1093/jnci/djr242) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21743022/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Natl%20Cancer%20Inst&title=Toxicity%20of%20adjuvant%20endocrine%20therapy%20in%20postmenopausal%20breast%20cancer%20patients:%20a%20systematic%20review%20and%20meta-analysis&author=E%20Amir&author=B%20Seruga&author=S%20Niraula&author=L%20Carlsson&author=A%20Oca%C3%B1a&volume=103&publication_year=2011&pages=1299-1309&pmid=21743022&doi=10.1093/jnci/djr242&)

11. Goss PE, Ingle JN, Pritchard KI, et al. Exemestane versus anastrozole in postmenopausal women with early breast cancer: NCIC CTG MA.27-a randomized controlled phase III trial. J Clin Oncol. 2013;31:1398–1404  [DOI](https://doi.org/10.1200/JCO.2012.44.7805) | [PMC free article](/articles/PMC3612593/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23358971/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Exemestane%20versus%20anastrozole%20in%20postmenopausal%20women%20with%20early%20breast%20cancer:%20NCIC%20CTG%20MA.27-a%20randomized%20controlled%20phase%20III%20trial&author=PE%20Goss&author=JN%20Ingle&author=KI%20Pritchard&volume=31&publication_year=2013&pages=1398-1404&pmid=23358971&doi=10.1200/JCO.2012.44.7805&)

12. Ingle JN, Schaid DJ, Goss PE, et al. Genome-wide associations and functional genomic studies of musculoskeletal adverse events in women receiving aromatase inhibitors. J Clin Oncol. 2010;28:4674–4682  [DOI](https://doi.org/10.1200/JCO.2010.28.5064) | [PMC free article](/articles/PMC3020700/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20876420/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Genome-wide%20associations%20and%20functional%20genomic%20studies%20of%20musculoskeletal%20adverse%20events%20in%20women%20receiving%20aromatase%20inhibitors&author=JN%20Ingle&author=DJ%20Schaid&author=PE%20Goss&volume=28&publication_year=2010&pages=4674-4682&pmid=20876420&doi=10.1200/JCO.2010.28.5064&)

13. Liu M, Wang L, Bongartz T, et al. Aromatase inhibitors, estrogens and musculoskeletal pain: estrogen-dependent T-cell leukemia 1A (TCL1A) gene-mediated regulation of cytokine expression. Breast Cancer Res. 2012;14:R41.  [DOI](https://doi.org/10.1186/bcr3137) | [PMC free article](/articles/PMC3446375/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22405131/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Breast%20Cancer%20Res&title=Aromatase%20inhibitors,%20estrogens%20and%20musculoskeletal%20pain:%20estrogen-dependent%20T-cell%20leukemia%201A%20(TCL1A)%20gene-mediated%20regulation%20of%20cytokine%20expression&author=M%20Liu&author=L%20Wang&author=T%20Bongartz&volume=14&publication_year=2012&pages=R41&pmid=22405131&doi=10.1186/bcr3137&)

14. Ho MF, Liu M, Wang L, Weinshilboum R, Bongartz T. Aromatase inhibitor treatment and musculoskeletal adverse events: SNP modulated, estrogen-dependent variation in CCR6/CCL20 expression. Ann Rheum Dis. 2014;73(suppl 2):34923361084  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Rheum%20Dis&title=Aromatase%20inhibitor%20treatment%20and%20musculoskeletal%20adverse%20events:%20SNP%20modulated,%20estrogen-dependent%20variation%20in%20CCR6/CCL20%20expression&author=MF%20Ho&author=M%20Liu&author=L%20Wang&author=R%20Weinshilboum&author=T%20Bongartz&volume=73&issue=suppl%202&publication_year=2014&pages=349&)

15. Estrada K, Styrkarsdottir U, Evangelou E, et al. Genome-wide meta-analysis identifies 56 bone mineral density loci and reveals 14 loci associated with risk of fracture. Nat Genet. 2012;44:491–501  [DOI](https://doi.org/10.1038/ng.2249) | [PMC free article](/articles/PMC3338864/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22504420/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Genome-wide%20meta-analysis%20identifies%2056%20bone%20mineral%20density%20loci%20and%20reveals%2014%20loci%20associated%20with%20risk%20of%20fracture&author=K%20Estrada&author=U%20Styrkarsdottir&author=E%20Evangelou&volume=44&publication_year=2012&pages=491-501&pmid=22504420&doi=10.1038/ng.2249&)

16. Li L, Wang H, Yang ES, Arteaga CL, Xia F. Erlotinib attenuates homologous recombinational repair of chromosomal breaks in human breast cancer cells. Cancer Res. 2008;68:9141–9146  [DOI](https://doi.org/10.1158/0008-5472.CAN-08-1127) | [PMC free article](/articles/PMC2586988/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19010885/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=Erlotinib%20attenuates%20homologous%20recombinational%20repair%20of%20chromosomal%20breaks%20in%20human%20breast%20cancer%20cells&author=L%20Li&author=H%20Wang&author=ES%20Yang&author=CL%20Arteaga&author=F%20Xia&volume=68&publication_year=2008&pages=9141-9146&pmid=19010885&doi=10.1158/0008-5472.CAN-08-1127&)

17. Niu N, Qin Y, Fridley BL, et al. Radiation pharmacogenomics: a genome-wide association approach to identify radiation response biomarkers using human lymphoblastoid cell lines. Genome Res. 2010;20:1482–1492  [DOI](https://doi.org/10.1101/gr.107672.110) | [PMC free article](/articles/PMC2963812/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20923822/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Res&title=Radiation%20pharmacogenomics:%20a%20genome-wide%20association%20approach%20to%20identify%20radiation%20response%20biomarkers%20using%20human%20lymphoblastoid%20cell%20lines&author=N%20Niu&author=Y%20Qin&author=BL%20Fridley&volume=20&publication_year=2010&pages=1482-1492&pmid=20923822&doi=10.1101/gr.107672.110&)

18. Pei H, Li L, Fridley BL, et al. FKBP51 affects cancer cell response to chemotherapy by negatively regulating Akt. Cancer Cell. 2009;16:259–266  [DOI](https://doi.org/10.1016/j.ccr.2009.07.016) | [PMC free article](/articles/PMC2755578/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19732725/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Cell&title=FKBP51%20affects%20cancer%20cell%20response%20to%20chemotherapy%20by%20negatively%20regulating%20Akt&author=H%20Pei&author=L%20Li&author=BL%20Fridley&volume=16&publication_year=2009&pages=259-266&pmid=19732725&doi=10.1016/j.ccr.2009.07.016&)

19. Ingle JN, Liu M, Wickerham DL, et al. Selective estrogen receptor modulators and pharmacogenomic variation in ZNF423 regulation of BRCA1 expression: individualized breast cancer prevention. Cancer Discov. 2013;3:812–825  [DOI](https://doi.org/10.1158/2159-8290.CD-13-0038) | [PMC free article](/articles/PMC3710533/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23764426/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Discov&title=Selective%20estrogen%20receptor%20modulators%20and%20pharmacogenomic%20variation%20in%20ZNF423%20regulation%20of%20BRCA1%20expression:%20individualized%20breast%20cancer%20prevention&author=JN%20Ingle&author=M%20Liu&author=DL%20Wickerham&volume=3&publication_year=2013&pages=812-825&pmid=23764426&doi=10.1158/2159-8290.CD-13-0038&)

20. Richards JB, Zheng HF, Spector TD. Genetics of osteoporosis from genome-wide association studies: advances and challenges. Nat Rev Genet. 2012;13:576–588  [DOI](https://doi.org/10.1038/nrg3228) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22805710/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Genet&title=Genetics%20of%20osteoporosis%20from%20genome-wide%20association%20studies:%20advances%20and%20challenges&author=JB%20Richards&author=HF%20Zheng&author=TD%20Spector&volume=13&publication_year=2012&pages=576-588&pmid=22805710&doi=10.1038/nrg3228&)

21. Liu M, Ingle JN, Fridley BL, et al. TSPYL5 SNPs: association with plasma estradiol concentrations and aromatase expression. Mol Endocrinol. 2013;27:657–670  [DOI](https://doi.org/10.1210/me.2012-1397) | [PMC free article](/articles/PMC3607698/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23518928/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Endocrinol&title=TSPYL5%20SNPs:%20association%20with%20plasma%20estradiol%20concentrations%20and%20aromatase%20expression&author=M%20Liu&author=JN%20Ingle&author=BL%20Fridley&volume=27&publication_year=2013&pages=657-670&pmid=23518928&doi=10.1210/me.2012-1397&)

22. Wang L, McLeod HL, Weinshilboum RM. Genomics and drug response. N Engl J Med. 2011;364:1144–1153  [DOI](https://doi.org/10.1056/NEJMra1010600) | [PMC free article](/articles/PMC3184612/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21428770/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Genomics%20and%20drug%20response&author=L%20Wang&author=HL%20McLeod&author=RM%20Weinshilboum&volume=364&publication_year=2011&pages=1144-1153&pmid=21428770&doi=10.1056/NEJMra1010600&)

23. Price AL, Patterson NJ, Plenge RM, Weinblatt ME, Shadick NA, Reich D. Principal components analysis corrects for stratification in genome-wide association studies. Nat Genet. 2006;38:904–909  [DOI](https://doi.org/10.1038/ng1847) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16862161/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Principal%20components%20analysis%20corrects%20for%20stratification%20in%20genome-wide%20association%20studies&author=AL%20Price&author=NJ%20Patterson&author=RM%20Plenge&author=ME%20Weinblatt&author=NA%20Shadick&volume=38&publication_year=2006&pages=904-909&pmid=16862161&doi=10.1038/ng1847&)

24. Patterson N, Price AL, Reich D. Population structure and eigenanalysis. PLoS Genet. 2006;2:e190.  [DOI](https://doi.org/10.1371/journal.pgen.0020190) | [PMC free article](/articles/PMC1713260/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17194218/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet&title=Population%20structure%20and%20eigenanalysis&author=N%20Patterson&author=AL%20Price&author=D%20Reich&volume=2&publication_year=2006&pages=e190&pmid=17194218&doi=10.1371/journal.pgen.0020190&)

25. Therneau TM, Li H. Computing the Cox model for case cohort designs. Lifetime Data Anal. 1999;5:99–112  [DOI](https://doi.org/10.1023/a:1009691327335) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10408179/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lifetime%20Data%20Anal&title=Computing%20the%20Cox%20model%20for%20case%20cohort%20designs&author=TM%20Therneau&author=H%20Li&volume=5&publication_year=1999&pages=99-112&pmid=10408179&doi=10.1023/a:1009691327335&)

26. Harris SA, Tau KR, Enger RJ, Toft DO, Riggs BL, Spelsberg TC. Estrogen response in the hFOB 1.19 human fetal osteoblastic cell line stably transfected with the human estrogen receptor gene. J Cell Biochem. 1995;59:193–201  [DOI](https://doi.org/10.1002/jcb.240590209) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8904314/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Cell%20Biochem&title=Estrogen%20response%20in%20the%20hFOB%201.19%20human%20fetal%20osteoblastic%20cell%20line%20stably%20transfected%20with%20the%20human%20estrogen%20receptor%20gene&author=SA%20Harris&author=KR%20Tau&author=RJ%20Enger&author=DO%20Toft&author=BL%20Riggs&volume=59&publication_year=1995&pages=193-201&pmid=8904314&doi=10.1002/jcb.240590209&)

27. Boyce BF, Xing L. Biology of RANK, RANKL, and osteoprotegerin. Arthritis Res Ther. 2007;9(suppl 1):S1.  [DOI](https://doi.org/10.1186/ar2165) | [PMC free article](/articles/PMC1924516/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17634140/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arthritis%20Res%20Ther&title=Biology%20of%20RANK,%20RANKL,%20and%20osteoprotegerin&author=BF%20Boyce&author=L%20Xing&volume=9&issue=suppl%201&publication_year=2007&pages=S1&pmid=17634140&doi=10.1186/ar2165&)

28. Deng FY, Zhao LJ, Pei YF, et al. Genome-wide copy number variation association study suggested VPS13B gene for osteoporosis in Caucasians. Osteoporos Int. 2010;21:579–587  [DOI](https://doi.org/10.1007/s00198-009-0998-7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19680589/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Osteoporos%20Int&title=Genome-wide%20copy%20number%20variation%20association%20study%20suggested%20VPS13B%20gene%20for%20osteoporosis%20in%20Caucasians&author=FY%20Deng&author=LJ%20Zhao&author=YF%20Pei&volume=21&publication_year=2010&pages=579-587&pmid=19680589&doi=10.1007/s00198-009-0998-7&)

29. Marie P, Halbout P. [OPG/RANKL: role and therapeutic target in osteoporosis]. Med Sci (Paris). 2008;24:105–110  [DOI](https://doi.org/10.1051/medsci/2008241105) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18198124/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Med%20Sci%20(Paris)&title=%5BOPG/RANKL:%20role%20and%20therapeutic%20target%20in%20osteoporosis%5D&author=P%20Marie&author=P%20Halbout&volume=24&publication_year=2008&pages=105-110&pmid=18198124&doi=10.1051/medsci/2008241105&)

30. Cree MW, Juby AG, Carriere KC. Mortality and morbidity associated with osteoporosis drug treatment following hip fracture. Osteoporos Int. 2003;14:722–727  [DOI](https://doi.org/10.1007/s00198-003-1430-3) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12904833/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Osteoporos%20Int&title=Mortality%20and%20morbidity%20associated%20with%20osteoporosis%20drug%20treatment%20following%20hip%20fracture&author=MW%20Cree&author=AG%20Juby&author=KC%20Carriere&volume=14&publication_year=2003&pages=722-727&pmid=12904833&doi=10.1007/s00198-003-1430-3&)

31. Goss P, Bondarenko IN, Manikhas GN, et al. Phase III, double-blind, controlled trial of atamestane plus toremifene compared with letrozole in postmenopausal women with advanced receptor-positive breast cancer. J Clin Oncol. 2007;25:4961–4966  [DOI](https://doi.org/10.1200/JCO.2006.09.5455) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17971594/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Phase%20III,%20double-blind,%20controlled%20trial%20of%20atamestane%20plus%20toremifene%20compared%20with%20letrozole%20in%20postmenopausal%20women%20with%20advanced%20receptor-positive%20breast%20cancer&author=P%20Goss&author=IN%20Bondarenko&author=GN%20Manikhas&volume=25&publication_year=2007&pages=4961-4966&pmid=17971594&doi=10.1200/JCO.2006.09.5455&)

32. Holzer G, Noske H, Lang T, Holzer L, Willinger U. Soluble cathepsin K: a novel marker for the prediction of nontraumatic fractures? J Lab Clin Med. 2005;146:13–17  [DOI](https://doi.org/10.1016/j.lab.2005.03.016) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16025086/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Lab%20Clin%20Med&title=Soluble%20cathepsin%20K:%20a%20novel%20marker%20for%20the%20prediction%20of%20nontraumatic%20fractures?&author=G%20Holzer&author=H%20Noske&author=T%20Lang&author=L%20Holzer&author=U%20Willinger&volume=146&publication_year=2005&pages=13-17&pmid=16025086&doi=10.1016/j.lab.2005.03.016&)

33. Lang TH, Willinger U, Holzer G. Soluble cathepsin-L: a marker of bone resorption and bone density? J Lab Clin Med. 2004;144:163–166  [DOI](https://doi.org/10.1016/j.lab.2004.06.001) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15454886/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Lab%20Clin%20Med&title=Soluble%20cathepsin-L:%20a%20marker%20of%20bone%20resorption%20and%20bone%20density?&author=TH%20Lang&author=U%20Willinger&author=G%20Holzer&volume=144&publication_year=2004&pages=163-166&pmid=15454886&doi=10.1016/j.lab.2004.06.001&)

34. Kneissel M, Luong-Nguyen NH, Baptist M, et al. Everolimus suppresses cancellous bone loss, bone resorption, and cathepsin K expression by osteoclasts. Bone. 2004;35:1144–1156  [DOI](https://doi.org/10.1016/j.bone.2004.07.013) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15542040/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bone&title=Everolimus%20suppresses%20cancellous%20bone%20loss,%20bone%20resorption,%20and%20cathepsin%20K%20expression%20by%20osteoclasts&author=M%20Kneissel&author=NH%20Luong-Nguyen&author=M%20Baptist&volume=35&publication_year=2004&pages=1144-1156&pmid=15542040&doi=10.1016/j.bone.2004.07.013&)

35. Han SH, Chae SW, Choi JY, Kim EC, Chae HJ, Kim HR. Acidic pH environments increase the expression of cathepsin B in osteoblasts: the significance of ER stress in bone physiology. Immunopharmacol Immunotoxicol. 2009;31:428–431  [DOI](https://doi.org/10.1080/08923970902751651) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19242848/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Immunopharmacol%20Immunotoxicol&title=Acidic%20pH%20environments%20increase%20the%20expression%20of%20cathepsin%20B%20in%20osteoblasts:%20the%20significance%20of%20ER%20stress%20in%20bone%20physiology&author=SH%20Han&author=SW%20Chae&author=JY%20Choi&author=EC%20Kim&author=HJ%20Chae&volume=31&publication_year=2009&pages=428-431&pmid=19242848&doi=10.1080/08923970902751651&)

36. Wilson TJ, Nannuru KC, Singh RK. Cathepsin G-mediated activation of pro-matrix metalloproteinase 9 at the tumor-bone interface promotes transforming growth factor-β signaling and bone destruction. Mol Cancer Res. 2009;7:1224–1233  [DOI](https://doi.org/10.1158/1541-7786.MCR-09-0028) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19671689/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Cancer%20Res&title=Cathepsin%20G-mediated%20activation%20of%20pro-matrix%20metalloproteinase%209%20at%20the%20tumor-bone%20interface%20promotes%20transforming%20growth%20factor-%CE%B2%20signaling%20and%20bone%20destruction&author=TJ%20Wilson&author=KC%20Nannuru&author=RK%20Singh&volume=7&publication_year=2009&pages=1224-1233&pmid=19671689&doi=10.1158/1541-7786.MCR-09-0028&)

37. Vakulenko VI, Sapargel'dyev NB, Golub GB. [The protein content and cathepsin D activity of the mandibular bone in rats during fracture healing]. Stomatologiia (Mosk). 1990;69:9–11  [PubMed](https://pubmed.ncbi.nlm.nih.gov/2202091/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Stomatologiia%20(Mosk)&title=%5BThe%20protein%20content%20and%20cathepsin%20D%20activity%20of%20the%20mandibular%20bone%20in%20rats%20during%20fracture%20healing%5D&author=VI%20Vakulenko&author=NB%20Sapargel'dyev&author=GB%20Golub&volume=69&publication_year=1990&pages=9-11&pmid=2202091&)

38. Geoffroy V, Kneissel M, Fournier B, Boyde A, Matthias P. High bone resorption in adult aging transgenic mice overexpressing cbfa1/runx2 in cells of the osteoblastic lineage. Mol Cell Biol. 2002;22:6222–6233  [DOI](https://doi.org/10.1128/MCB.22.17.6222-6233.2002) | [PMC free article](/articles/PMC134019/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12167715/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Cell%20Biol&title=High%20bone%20resorption%20in%20adult%20aging%20transgenic%20mice%20overexpressing%20cbfa1/runx2%20in%20cells%20of%20the%20osteoblastic%20lineage&author=V%20Geoffroy&author=M%20Kneissel&author=B%20Fournier&author=A%20Boyde&author=P%20Matthias&volume=22&publication_year=2002&pages=6222-6233&pmid=12167715&doi=10.1128/MCB.22.17.6222-6233.2002&)

39. Wang M, Amano SU, Flach RJ, Chawla A, Aouadi M, Czech MP. Identification of Map4k4 as a novel suppressor of skeletal muscle differentiation. Mol Cell Biol. 2013;33:678–687  [DOI](https://doi.org/10.1128/MCB.00618-12) | [PMC free article](/articles/PMC3571342/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23207904/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Cell%20Biol&title=Identification%20of%20Map4k4%20as%20a%20novel%20suppressor%20of%20skeletal%20muscle%20differentiation&author=M%20Wang&author=SU%20Amano&author=RJ%20Flach&author=A%20Chawla&author=M%20Aouadi&volume=33&publication_year=2013&pages=678-687&pmid=23207904&doi=10.1128/MCB.00618-12&)

40. Bouzakri K, Zierath JR. MAP4K4 gene silencing in human skeletal muscle prevents tumor necrosis factor-α-induced insulin resistance. J Biol Chem. 2007;282:7783–7789  [DOI](https://doi.org/10.1074/jbc.M608602200) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17227768/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=MAP4K4%20gene%20silencing%20in%20human%20skeletal%20muscle%20prevents%20tumor%20necrosis%20factor-%CE%B1-induced%20insulin%20resistance&author=K%20Bouzakri&author=JR%20Zierath&volume=282&publication_year=2007&pages=7783-7789&pmid=17227768&doi=10.1074/jbc.M608602200&)
