# A Potential Role for Human UDP-Glucuronosyltransferase 1A4 Promoter Single Nucleotide Polymorphisms in the Pharmacogenomics of Tamoxifen and Its Derivatives

## Metadata
**Authors:** Aleksandra K Greer, Centdrika R Dates, Athena Starlard-Davenport, Vineetha K Edavana, Stacie M Bratton, Ishwori B Dhakal, Moshe Finel, Susan A Kadlubar, Anna Radominska-Pandya
**Journal:** Drug Metabolism and Disposition
**Date:** 2014 Sep
**DOI:** [10.1124/dmd.114.058016](https://doi.org/10.1124/dmd.114.058016)
**PMID:** 24917585
**PMCID:** PMC4152870
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4152870/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC4152870/pdf/dmd.114.058016.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC4152870/pdf/dmd.114.058016.pdf)

## Abstract

Tamoxifen (Tam) is a selective estrogen receptor modulator used to inhibit breast tumor growth. Tam can be directly N-glucuronidated via the tertiary amine group or O-glucuronidated after cytochrome P450–mediated hydroxylation. In this study, the glucuronidation of Tam and its hydroxylated and/or chlorinated derivatives [4-hydroxytamoxifen (4OHTam), toremifene (Tor), and 4-hydroxytoremifene (4OHTor)] was examined using recombinant human UDP-glucuronosyltransferases (UGTs) from the 1A subfamily and human hepatic microsomes. Recombinant UGT1A4 catalyzed the formation of N-glucuronides of Tam and its derivatives and was the most active UGT enzyme toward these compounds. Therefore, it was hypothesized that single nucleotide polymorphisms (SNPs) in the promoter region of UGT1A4 have the ability to significantly decrease the glucuronidation rates of Tam metabolites in the human liver. In vitro activity of 64 genotyped human liver microsomes was used to determine the association between the UGT1A4 promoter and coding region SNPs and the glucuronidation rates of Tam, 4OHTam, Tor, and 4OHTor. Significant decreases in enzymatic activity were observed in microsomes for individuals heterozygous for −163G/A and −217T/G. These alterations in glucuronidation may lead to prolonged circulating half-lives and may potentially modify the effectiveness of these drugs in the treatment of breast cancer.

## Introduction

Tamoxifen [Tam; (*Z*)-2-[4-(1,2-diphenylbut-1-enyl)phenoxy]-*N*,*N*-dimethylethanamine] is prescribed within the United States as a component of adjuvant therapy to prevent estrogen receptor (ER)–positive breast cancer recurrence, as well as to treat metastatic breast cancer and to prevent disease in high-risk populations and in women with ductal carcinoma in situ ([Sakorafas et al., 2008](#B35)). Phase I metabolism of Tam is complex; it occurs via hepatic cytochrome P450 (P450) and involves many enzymes of this superfamily. The major metabolite of P450 metabolism of Tam is *N*-desmethyltamoxifen (*N*-desTam), which is produced by several P450s. This compound can then be converted to endoxifen, and CYP2D6 is the major enzyme involved in this process. In addition, Tam can be metabolized into its active metabolite 4-hydroxytamoxifen (4OHTam) by several P450s, including CYP2D6 ([Hoskins et al., 2009](#B16)). It has also been demonstrated that human flavin-containing monooxygenases FMO1 and FMO3 catalyze Tam *N*-oxidation to form Tam-*N*-oxide and that many P450s, but not FMOs, reduce Tam-*N*-oxide back to Tam, with CYP2A6, CYP1A1, and CYP3A4 producing the greatest reduction ([Parte and Kupfer, 2005](#B31)).

Glucuronidation, a phase II metabolism reaction, is one of the major means of biotransformation and elimination of Tam and its major active metabolites ([Ogura et al., 2006](#B30); [Sun et al., 2006](#B36), [2007](#B37); [Mazzarino et al., 2013](#B28)). Glucuronidation, catalyzed by UDP-glucuronosyltransferases (UGTs), is an important biochemical process that leads to the conjugation of a number of hydrophobic endogenous and exogenous substrates with UDP-glucuronic acid (UDP-GlcUA; the cosubstrate in the UGT-catalyzed reactions) to generally inactive, more hydrophilic compounds ([Dutton, 1980](#B10); [Tukey and Strassburg, 2000](#B38)). UGT1A4 exhibits the highest enzymatic activity toward Tam catalyzing the formation of the *N*-linked glucuronide; however, it was previously demonstrated that UGT1A3 and UGT2B10 can also form Tam-*N*-glucuronides ([Sun et al., 2007](#B37); [Kato et al., 2013](#B20)). Therefore, interindividual variability in UGT1A4 genes and expression levels may impact the clinical efficacy of this valuable drug.

The single nucleotide polymorphism (SNP) is most common type of genetic polymorphism, with the total number of variant positions among humans estimated to be more than 10,000,000 ([Nussbaum et al., 2007](#B29)). SNPs identified in UGT genes are associated with subpopulations of individuals that are more susceptible to various diseases and may play an important role in individual pharmacological responses to various xenobiotics and drugs ([Tukey and Strassburg, 2001](#B39); [Burchell, 2003](#B5); [Crettol et al., 2010](#B7); [Ghotbi et al., 2010](#B14); [Gulcebi et al., 2011](#B15); [López et al., 2011](#B27); [Zhou et al., 2011](#B41)). Specifically, studies of recombinant *UGT1A4* variants showed that these functional polymorphisms can significantly alter the glucuronidation rates of Tam and its metabolites ([Sun et al., 2006](#B36); [Benoit-Biancamano et al., 2009](#B1)).

This study focuses on SNPs found in the promoter region of UGT1A4, and the 3′ untranslated region (UTR) that is common to all UGT1A enzymes. Since these SNPs are not in the coding region, they are not thought to affect the structure or activity of the expressed UGT1A4 protein. However, these promoter SNPs are shown to affect UGT transcription rates leading to altered mRNA levels ([Erichsen et al., 2008](#B13); [Edavana et al., 2013](#B11)). We previously showed that these altered mRNA levels are correlated with glucuronidation activity, implying that UGT1A4 protein expression levels are also affected ([Edavana et al., 2013](#B11)). We also previously showed that none of these UGT1A4 promoter SNPs are in linkage disequilibrium either with each other or with coding region SNPs ([Edavana et al., 2013](#B11)). The rs8330 UGT1A SNP studied here is located within the 3′UTR of UGT1A isoforms and is shared among all UGT1A isoforms. SNPs in the 3′UTR may impact mRNA stability and translation efficiency leading to impaired UGT1A expression and activity ([Court et al., 2013](#B6)).

In this work, we carried out extensive studies on the glucuronidation of Tam, its chlorinated derivative toremifene [Tor; 2-{4-[(1*Z*)-4-chloro-1,2-diphenylbut-1-en-1-yl]phenoxy}-*N*,*N*-dimethylethanamine], and their hydroxylated metabolites 4OHTam and 4-hydroxytoremifene (4OHTor), using recombinant human UGTs from the 1A subfamily and a set of human liver microsomes from patients that were previously genotyped to identify SNPs located in the *UGT1A4* promoter ([Edavana et al., 2013](#B11)). These studies were driven by the hypothesis that these SNPs in the promoter region of *UGT1A4* have the ability to significantly decrease the glucuronidation rates of Tam and its derivatives in the human liver. Because of the importance of these drugs in cancer treatment and prevention, the information presented here could serve as a tool for better understanding variations in Tam metabolism and could lead to improved breast cancer treatment.

## Materials and Methods

All chemicals used for this study were of at least reagent grade. Tam and 4OHTam were purchased from Sigma-Aldrich (St. Louis, MO). Tor and 4OHTor were procured from Orion Pharma (Helsinki, Finland). Ethyl alcohol (95%) was purchased from AAPER (Shelbyville, KY). Unless otherwise specified, all other chemicals and reagents were of reagent grade and were purchased from Sigma-Aldrich. Recombinant human UGTs were produced in baculovirus-infected insect cells as previously described ([Kurkela et al., 2003](#B22); [Kuuranne et al., 2003](#B23)).

### 

#### Human Liver Microsomes.

The human liver microsomes used for this study were previously described ([Edavana et al., 2012](#B12)). In brief, human normal liver specimens (*n* = 65; from noncancerous subjects, collected and frozen 0–3 hours after surgical excision) were obtained from the Cooperative Human Tissue Network (Birmingham, AL). All specimens assayed in this study were from donors of white or unknown race. Donors ranged in age from 26 to 102 years and comprised 54 men, 36 women, and 6 individuals of unknown sex. Microsomes were prepared from human liver tissue as previously described ([Turesky et al., 1991](#B40)) and were stored frozen at −80°C until assayed. Liver microsomal protein levels were determined using the [Bradford (1976)](#B3) method with bovine serum albumin as a standard.

#### Human Liver Microsome and Recombinant UGT Screening Incubations.

Human liver microsomes (15 *µ*g) or recombinant UGT membrane protein (5 *µ*g) were assayed for activity toward Tam, 4OHTam, Tor, and 4OHTor. The cloning and expression and characterization of UGT1A1, UGT1A3, UGT1A4, and UGT1A6–UGT1A10 in baculovirus-infected Sf9 insect cells as His-tagged proteins and the preparation of enriched membrane fractions were previously reported ([Kurkela et al., 2003](#B22); [Kuuranne et al., 2003](#B23)). These preparations were shown to contain similar amounts of protein by Western blot analysis using an anti-His antibody directed at the His tag on each of these recombinant enzymes. Each enzyme tested in this study is known to be active toward substrates specific for that enzyme, and metabolite formation was previously shown to be linear with respect to both protein concentration and incubation time up to 2 mg protein/ml and 180 minutes, respectively ([Kaivosaari et al., 2008](#B18)). UGT2B10 activity was assayed using cells homogenates, not enriched membranes, as was previously reported ([Kato et al., 2013](#B20)).

Protein was incubated in 100 *µ*M Tris-HCl (pH 7.4)/5 mM MgCl_2_/5 mM saccharolactone with 100 or 200 *µ*M substrate in a total final volume of 30 *μ*l. Substrates were dissolved in H_2_O/EtOH/dimethylsulfoxide at a ratio of 25:50:25 with a final reaction concentration of 2% dimethylsulfoxide and 1% ethanol, and controls omitting substrate/cosubstrate were run with each assay. No additional detergents or other activators were used with recombinant proteins; however, human liver microsomes were activated with the addition of alamethicin (2.5 *µ*g). Reactions were started by the addition of UDP-GlcUA (2 mM) and incubated for 180 minutes at 37°C. The reactions were stopped by addition of 30 *μ*l ethanol, followed by centrifugation at 14,000 rpm for 8 minutes to pellet the protein.

High-pressure liquid chromatography (HPLC) analyses of the supernatants were performed using an HP1050 HPLC system equipped with the Agilent ChemStation software package (Agilent Technologies, Santa Clara, CA). Samples were separated using an Agilent Eclipse XDB-C18 (4.6 × 150 mm, 5 *μ*m) with an Eclipse XDB-CN Analytical Guard Column (4.6 × 12.5 mm, 5 *μ*m) at 37°C. The solvent system consisted of 10 mM ammonium acetate (A), and acetonitrile (B). For assays with Tam and Tor, products were separated at a flow rate of 1.25 ml/min, with the following elution gradient: 75% A (5 minutes), linear gradient from 75% A to 75% B (5–15 minutes), 75% B (15–23 minutes), and 75% A (23.01–24 minutes). The column was re-equilibrated at initial conditions for 10 minutes between runs. For assays using the hydroxylated derivatives as substrates, the reaction products were separated using a flow rate of 1.00 ml/min, with the following elution gradient: 75% A (5 minutes), linear gradient from 75% A to 75% B (5–20 minutes), and 75% B (20–23 minutes). The elution of each metabolite was monitored with a UV-Vis Diode Array Detector (Agilent Technologies, Santa Clara, CA) at 280 nm. Primary standards for the glucuronidated metabolites are not available; however, it was previously shown that the addition of the glucuronic acid moiety does not alter the extinction coefficient from that of the unreacted substrate ([Doerge et al., 2000](#B9)). Therefore, product concentrations were calculated using the external standard response for each substrate under identical conditions. Concentration curves were established for each of the substrates based on peak areas measured from increasing concentrations of substrate dissolved in reaction mix, incubated at identical conditions as the reactions themselves, and separated using the appropriate HPLC methods. Each concentration was assessed in triplicate and the data were fit to a linear equation.

#### Steady-State Enzyme Kinetics Assays.

Kinetic parameters were determined by incubating recombinant UGT1A4 (5 *µ*g total membrane protein) in the presence of varying concentrations of substrate (5–500 *µ*M) at a fixed concentration of UDP-GlcUA (3 mM) for 180 minutes. All other conditions were identical to those of the screening experiments. Curve fitting was conducted utilizing GraphPad Prism v4.0b software (GraphPad Software Inc., San Diego, CA). Kinetic constants were obtained by fitting experimental data to the uncompetitive substrate inhibition (USI) model, where *K*_s_ is the inhibition constant describing the reduction in rate as follows:

| Category |
| --- |
|   |
The fit of the data for each model was assessed from the standard error, 95% confidence intervals, and *R*^2^ values. Kinetic curves were also analyzed as Eadie–Hofstee plots to support kinetic models. Kinetic constants were reported as the mean ± standard error.

#### Genotyping.

Genotyping for *UGT1A4* 5′ sequence SNPs (−36G/A, −163G/A, −217T/G, and −219C/T) and 3′UTR SNPs were performed as previously described ([Saeki et al., 2005](#B34); [Benoit-Biancamano et al., 2009](#B1)) using an ABI Prism 3700 DNA analyzer (Applied Biosystems, Foster City, CA). The *UGT1A*-3′UTR SNP (rs8330) was genotyped by sequencing DNA obtained from this same group of human liver samples (*n* = 36). Briefly, PCR amplification was performed using forward (5′-GAA ACA TGG CCT GTT TG-3′) and reverse (5′-CAA AGC AAG AAA TCA TAT GC-3′) primers using Jumpstart PCR Master mix (Invitrogen, Carlsbad, CA) with the following PCR conditions: 95°C for 5 minutes, followed by 40 cycles at 95°C for 30 seconds, 60°C for 30 seconds, and 72°C for 1 minute, followed by a final extension at 72°C for 7 minutes. PCR products were purified using the DNA Clean & Concentrator Kit (Zymoresearch, Irvine, CA), followed by dye terminator incorporation (DTCS Quick Start Kit; GenomeLab, Fullerton, CA) and ethanol precipitation. The resultant product was sequenced on a Beckman Coulter Genome Laboratory GeXP Genetic Analysis System (Beckman Coulter, Fullerton, CA).

#### Statistical Analysis.

Genotyping data were examined for possible genotyping errors by testing deviation from the Hardy–Weinberg equilibrium. Heterozygous and homozygous variant genotypes were combined as one genotype group in analysis for SNPs with < 5 subjects per genotype group. Median enzymatic activity levels and standard deviations were summarized by genotype and activity and were compared using the Wilcoxon rank sum test and the Kruskal–Wallis test for differences between genotypes ([Table 1](#T1)). Statistical analyses were performed with SAS software (version 9.2; SAS Institute Inc., Cary, NC). A *P* value < 0.05 (two-sided) was considered statistically significant.

| SNP | Tamox-N-Gluc |  | 4OHTam-N-Gluc |  | 4OHTam-O-Gluc |  | Tor-N-Gluc |  | 4OHTor-N-Gluc |  | 4OHTor-O-Gluc |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| Median | P | Median | P | Median | P | Median | P | Median | P | Median | P |  |
|   | nmol/mg protein per min |  |  |  |  |  |  |  |  |  |  |  |
| −36G/A |   |   |   |   |   |   |   |   |   |   |   |   |
| AA + AG | 0.11 (0.06) |   | 0.12 (0.07) |   | 0.13 (0.06) |   | 0.17 (0.07) |   | 0.09 (0.03) |   | 0.25 (0.16) |   |
| GG | 0.14 (0.07) | 0.45 | 0.15 (0.07) | 0.21 | 0.15 (0.11) | 0.11 | 0.18 (0.09) | 0.50 | 0.09 (0.04) | 0.96 | 0.27 (0.15) | 0.85 |
| −163G/A |   |   |   |   |   |   |   |   |   |   |   |   |
| AA | 0.16 (0.05) |   | 0.21 (0.06) |   | 0.12 (0.15) |   | 0.21 (0.08) |   | 0.13 (0.05) |   | 0.14 (0.26) |   |
| AG | 0.08 (0.08) |   | 0.10 (0.07) |   | 0.12 (0.12) |   | 0.11 (0.11) |   | 0.07 (0.03) |   | 0.25 (0.14) |   |
| GG | 0.16 (0.05) | <0.0001a | 0.17 (0.04) | 0.0002a | 0.18 (0.09) | 0.052 | 0.19 (0.06) | 0.007a | 0.10 (0.02) | <0.0001a | 0.29 (0.13) | 0.29 |
| −217T/G |   |   |   |   |   |   |   |   |   |   |   |   |
| GG + TG | 0.07 (0.05) |   | 0.09 (0.06) |   | 0.10 (0.05) |   | 0.09 (0.07) |   | 0.06 (0.04) |   | 0.21 (0.13) |   |
| AA | 0.16 (0.06) | <0.0001a | 0.17 (0.05) | <0.0001a | 0.19 (0.11) | <0.0001a | 0.19 (0.08) | <0.0001a | 0.10 (0.03) | 0.001a | 0.29 (0.15) | 0.022a |
| −219C/T |   |   |   |   |   |   |   |   |   |   |   |   |
| CC | 0.16 (0.06) |   | 0.16 (0.05) |   | 0.16 (0.09) |   | 0.18 (0.06) |   | 0.10 (0.03) |   | 0.29 (0.12) |   |
| CT | 0.13 (0.08) |   | 0.15 (0.08) |   | 0.13 (0.14) |   | 0.16 (0.11) |   | 0.08 (0.03) |   | 0.24 (0.18) |   |
| TT | 0.12 (0.07) | 0.13 | 0.14 (0.07) | 0.53 | 0.12 (0.06) | 0.30 | 0.17 (0.10) | 0.56 | 0.08 (0.07) | 0.047a | 0.19 (0.11) | 0.23 |
| 3′UTR |   |   |   |   |   |   |   |   |   |   |   |   |
| CC | 0.12 (0.06) |   | 0.12 (0.07) |   | 0.13 (0.07) |   | 0.17 (0.09) |   | 0.09 (0.03) |   | 0.26 (0.11) |   |
| GG + CG | 0.14 (0.05) | 0.79 | 0.15 (0.05 | 0.67 | 0.22 (0.10) | 0.019a | 0.18 (0.07) | 0.47 | 0.08 (0.02) | 0.49 | 0.31 (0.14) | 0.053 |

Table Caption: Correlations between genotypes and enzymatic activity Data are presented as the median (standard deviation) of specific enzymatic activity. P values were calculated using nonparametric analysis of variance (Kruskal–Wallis test or Wilcoxon rank sum test).

## Results

### 

#### Glucuronidation by Recombinant UGTs.

As an initial screen for glucuronidation activity toward Tam, eight His-tagged human recombinant UGT1A subfamily enzymes were evaluated for their ability to glucuronidate 100 and 200 *µ*M Tam, 4OHTam, Tor, and 4OHTor ([Fig. 1](#F1)). The results revealed that UGT1A4 was the most active enzyme and catalyzed direct *N*-glucuronidation of each compound. Glucuronidation by UGT1A3 was specific for the hydroxylated derivatives, and resulted predominantly in *O*-glucuronide formation with very little *N*-glucuronide formed. UGT1A1, UGT1A8, UGT1A9, and UGT1A10 also catalyzed *O*-glucuronidation in at least one of the hydroxylated derivatives, but UGT1A3 was the most active enzyme toward both compounds. The relative activity of UGT1A3 varied between the hydroxylated compounds, but the UGT1A4-catalyzed glucuronidation rates were very consistent among all of the compounds tested. Of the two concentrations tested, higher glucuronidation rates were observed with the lower substrate concentration, suggesting substrate inhibition at higher concentrations. This information helped us choose suitable concentrations for more detailed steady-state kinetics experiments. UGT2B10 (cell homogenate) was screened for activity but none was seen using our HPLC–diode array detector method. This is reasonable based on the small amount of activity (approximately 15- and 200-fold less *N*-glucuronide formed than with UGT1A3 and UGT1A4, respectively) measured by [Kato et al. (2013)](#B20), in which product formation was assessed using liquid chromatography coupled to tandem mass spectrometry, which is known to be much more sensitive. We also screened these UGTs for activity toward *N*-desTam, but no glucuronidation activity was seen (data not shown).

![Fig. 1.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b0d3/4152870/0ef6afc2b0e3/dmd.114.058016f1.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4152870_dmd.114.058016f1.jpg)

Glucuronidation activity screening. Selected recombinant UGT1A subfamily enzymes were screened for activity toward Tam, 4OHTam, Tor, and 4OHTor. Glucuronidation activities were measured by incubating recombinant protein (5 µg) with substrate (100 or 200 µM) and UDP-GlcUA (2 mM).

#### Steady-State Kinetics for Glucuronidation by Recombinant UGTs.

Based on the specific activity screening results, we selected UGT1A4 for further catalytic studies to determine the respective steady-state parameters for glucuronidation of these compounds ([Fig. 2](#F2)). Each of the four experiments resulted in data that best fit the USI model, with *K*_s_ values of 162, 18.2, 41.2, and 29.5 *μ*M for Tam, 4OHTam, Tor, and 4OHTor, respectively. The USI model suggests the existence of multiple binding sites within UGT1A4, which fits well with previously published research, which analyzed the steady-state kinetics of UGT1A4 toward dihydrotestosterone and *trans*-androsterone and compared them with those for Tam to support the hypothesis that UGT1A4 has multiple substrate binding sites ([Zhou et al., 2010](#B42)), and suggests that hydroxylation and/or chlorination of Tam does not affect this phenomenon.

![Fig. 2.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b0d3/4152870/cbc31de1c6f2/dmd.114.058016f2.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4152870_dmd.114.058016f2.jpg)

Steady-state glucuronidation kinetics of Tam, 4OHTam, Tor, and 4OHTor with UGT1A4. Glucuronidation activities of recombinant proteins were measured by incubating membrane fractions containing recombinant UGT1A4 (5 µg) with increasing concentrations (shown in the figure) of substrate at a constant concentration of UDP-GlcUA (3 mM). Curve fits and kinetic constants were determined using GraphPad Prism 4 software and the resulting parameters are shown.

#### Effects of UGT1A4 Genotypes on Glucuronidation in Human Liver Microsomes.

To determine the contribution of the *UGT1A*-3′UTR SNP (rs8330) and promoter SNPs to Tam, 4OHTam, Tor, and 4OHTor glucuronidation, activity assays were performed using human liver microsomes (64 human subjects), which had undergone dideoxy sequencing of their genomic DNA ([Table 1](#T1)). We previously examined SNPs in the promoter region of *UGT1A4*, and sequencing analysis revealed the existence of four different SNPs of allele frequencies: −36G/A (0.08), −163G/A (0.32), −217T/G (0.18), and −219C/T (0.32) ([Edavana et al., 2013](#B11)). Here, SNP −163G/A was significantly associated with Tam-*N*-gluc (*P* < 0.0001), 4OHTam-*N*-gluc (*P* = 0.002), 4OHTam-*O*-gluc (0.052), Tor-*N*-gluc (0.007), and 4OHTor-*N*-gluc (*P* < 0.0001). SNP −217T/G was also significantly correlated with Tam-*N*-gluc (*P* < 0.0001), 4OHTam-*N*-gluc (*P* < 0.0001), 4OHTam-*O*-gluc (*P* < 0.0001), Tor-*N*-gluc (*P* < 0.0001), 4OHTor-*N*-gluc (*P* = 0.001), and 4OHTor-*O*-gluc (*P* < 0.022). Microsomes from individuals with the homozygous GG or AA alleles at position −163 show significantly higher *N*-glucuronidation activity toward all four compounds tested than those with the heterozygous GA allele at this position ([Fig. 3](#F3)). In addition, microsomes from samples with the common homozygous TT allele at position −217 exhibited significantly greater glucuronidation activity compared with microsomes from samples with either the TG or GG allele at this position ([Fig. 4](#F4)). Further analysis of the data revealed that glucuronidation activity was significantly decreased only in samples that were heterozygous at both the −163 and −217 positions ([Fig. 5](#F5)). There was no significant difference in activity between samples from individuals who were homozygous at both positions (whether common or uncommon alleles) or from those that were heterozygous at only one of these two alleles.

![Fig. 3.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b0d3/4152870/0d21bcc40954/dmd.114.058016f3.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4152870_dmd.114.058016f3.jpg)

Correlation between −163G/A promoter SNP genotypes and phenotypes. Glucuronidation activity toward Tam, 4OHTam, Tor, and 4OHTor among human liver microsomes was compared among livers of known genotypes at the −163G/A position. Data are displayed as box and whisker plots and were analyzed using nonparametric analysis of variance (Kruskal–Wallis test) to determine overall significance of the effect of this SNP with a Dunn’s multiple comparison post test to determine the significance of the activity difference between each genotype. No significant difference was seen between the activities of samples with either of the two homozygous alleles.

![Fig. 4.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b0d3/4152870/3c5ca62fb963/dmd.114.058016f4.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4152870_dmd.114.058016f4.jpg)

Correlation between −217T/G promoter SNP genotypes and phenotypes. Glucuronidation activity toward Tam, 4OHTam, Tor, and 4OHTor among human liver microsomes was compared among livers of known genotypes at the −217T/G position. Because of the limited number of samples with the GG genotype, data were grouped as TT versus TG + GG. Data are displayed as box and whisker plots and were analyzed using a nonparametric t test (Wilcoxon rank sum test) to determine the overall significance of the effect of this SNP.

![Fig. 5.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b0d3/4152870/53dadcbf3313/dmd.114.058016f5.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4152870_dmd.114.058016f5.jpg)

Combined effect of −163G/A and −217T/G promoter SNP genotypes on phenotype. N-Glucuronidation activity of human liver microsomes toward Tam, 4OHTam, Tor, and 4OHTor was compared among livers of known genotypes at both the −163G/A and −217T/G positions. Samples were grouped as follows: homozygous (HO) at both positions (two HO alleles), heterozygous (HT) at both positions (two HT alleles), or one homozygous and one heterozygous allele (one HO allele and one HT allele). Data are displayed as box and whisker plots and were analyzed using nonparametric analysis of variance (Kruskal–Wallis test) and a Dunn’s multiple comparison post test to determine the significance of the activity difference between each genotype group. No significant difference was seen between the activities of the group with two HO alleles and the groups with one HO allele and one HT allele.

In addition to the *UGT1A4* promoter SNPs investigated, the *UGT1A*-3′UTR SNP (rs8330) genotypes from human liver DNA specimens were correlated with 4OHTam and 4OHTor glucuronidation activities from 36 microsomal samples ([Fig. 6](#F6)). Microsomes from individuals homozygous for the common allele (CC) had significantly lower *O*-glucuronidation of 4OHTam than individuals with either of the variant alleles (GG or CG) (*P* < 0.05). A similar increase in 4OHTor-*O*-glucuronidation was seen in individuals with variant 3′UTR alleles; however, this increase did not quite reach statistically significant levels based on our data set (*P* = 0.053).

![Fig. 6.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b0d3/4152870/68156b0954f9/dmd.114.058016f6.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4152870_dmd.114.058016f6.jpg)

Correlation between 3′UTR SNP genotypes and phenotypes. O-Glucuronidation activity toward 4OHTam and 4OHTor by human liver microsomes was compared among livers of known 3′UTR SNP genotypes. Because of the limited number of samples with the GG genotype, data were grouped as CC versus CG + GG. Data are displayed as box and whisker plots and were analyzed using a nonparametric t test (Wilcoxon rank sum test) to determine the overall significance of the effect of this SNP. No significant differences were seen unless otherwise indicated.

Little or no significant effect of SNP −36G/A, −219C/T, or *UGT1A*-3′UTR was seen on the *N*-glucuronidation of these compounds ([Supplemental Fig. 1](http://dmd.aspetjournals.org/lookup/suppl/doi:10.1124/dmd.114.058016/-/DC1)), and *O*-glucuronidation of 4OHTam and 4OHTor was only affected by the SNP at position −217 ([Supplemental Fig. 2](http://dmd.aspetjournals.org/lookup/suppl/doi:10.1124/dmd.114.058016/-/DC1)). Of the *UGT1A4* SNPs studied, only −163G/A conformed to the Hardy–Weinberg equilibrium, which is likely due to unknown population stratification. Furthermore, there was no statistical evidence of linkage of the *UGT1A4* promoter SNPs with each other or with the 3′UTR SNPs ([Table 1](#T1)).

## Discussion

Breast cancer is the most common form of cancer in women. In two-thirds of patients with breast cancer, endocrine therapy is the standard treatment. There are two classes of endocrine therapy: selective ER modulators, which act on the ER, and aromatase inhibitors, which either block the production of estrogens or block the action of estrogens on the ER ([Ingle, 2013](#B17)). Tam (a selective ER modulator) significantly decreases relapse and mortality in steroid hormone receptor–positive breast cancer and is currently prescribed in >120 countries worldwide ([Brauch et al., 2009](#B4)). Tam is also used as a chemotherapeutic agent in healthy women who are at high risk of developing ER-positive breast cancer ([Ogura et al., 2006](#B30)).

Variants of the *CYP2D6* gene result in a protein with reduced or absent enzyme activity that may be associated with lower plasma levels of active Tam metabolites ([de Leon et al., 2006](#B8); [Lim et al.. 2007](#B26)). Given the integral role of this gene in the production of active Tam metabolites, many studies examining the relationship between *CYP2D6* genotype and tamoxifen response have been performed, but have generated controversial results ([Kiyotani et al., 2013](#B21)). *N*-desTam is primarily a product of CYP3A4 metabolism; although it is the major metabolite of Tam, like Tam itself, *N*-desTam has little affinity for the ER. 4OHTan is produced by multiple P450s, and has 100 times greater affinity for the ER than Tam or *N*-desTam. Endoxifen is produced primarily by CYP2D6 and has similar affinity for the ER as 4OHTam, and its circulating levels are approximately eight times higher than 4OHTam; thus, it is considered by some to be the major active metabolite of Tam.

Recent studies from the Breast International 1-98 Collaborative Group as well as the ATAC (Arimidex, Tamoxifen, Alone or in Combination) trial investigators ([Rae et al., 2012](#B32); [Regan et al., 2012](#B33)), however, did not support the hypothesis that *CYP2D6* genotype is a biomarker of Tam response. Other drug-metabolizing enzymes, including UGTs, play a major role in metabolizing Tam ([Kaku et al., 2004](#B19); [Ogura et al., 2006](#B30); [Sun et al., 2006](#B36), [2007](#B37)). Thus, genetic variation in these enzymes may merit investigation as additional biomarkers of Tam efficacy and toxicity.

In this work, we investigated the glucuronidation of Tam and three of its derivatives, with a set of well defined recombinant human UGT enzymes from the UGT1A subfamily. We confirmed that UGT1A4 is the major enzyme involved in the glucuronidation of all four of the selected compounds. We recently identified that the −163G/A, −219T/G, and −217C/T SNPs located in the *UGT1A4* promoter affect the glucuronidation of another breast cancer drug, anastrozole ([Edavana et al., 2013](#B11)). We observed that microsomes isolated from heterozygous individuals had significantly lower glucuronidation activity toward this drug compared with those individuals who were homozygous for the common allele, and these changes correlated with altered UGT1A4 mRNA expression ([Edavana et al., 2013](#B11)). Therefore, we focused our attention of the role of *UGT1A4* promoter and 3′UTR SNPs in the glucuronidation of these compounds.

Here we used this same well characterized set of human liver microsomes to assess the glucuronidation of our selected Tam derivatives. We have examined the association between *UGT1A4* promoter and 3′UTR SNPs and the N-glucuroindation of Tam, 4OHTam, Tor, and 4OHTor as well as the O-glucuronidation of 4OHTam and 4OHTor. We found similar decreases in the glucuronidation of Tam and its derivatives in heterozygous individuals. In addition, we observed that microsomes isolated from individuals who were heterozygous at both the −163G/A and −217T/G positions had significantly lower glucuronidation activity toward Tam and its derivatives compared with those who were homozygous at these two positions or who were heterozygous at only the −163 or −217 allele. These data suggest that there is some redundancy in the *UGT1A4* promoter region to ensure adequate levels of UGT1A4 activity. However, if both of these positions are altered, it has a significant effect on the activity of this enzyme. This variability in glucuronidation of various xenobiotics in human subjects suggests that the pharmacogenomics of different breast cancer therapy xenobiotics could be greatly affected by genetic *UGT1A4* variants that affect the levels of this enzyme expressed in hepatic tissue.

In addition to examining SNPs located in the promoter region of *UGT1A4*, we determined the association of the *UGT1A*-3′UTR SNP (rs8330) with glucuronidation activity toward Tam and its derivatives. The importance of SNPs located within the common 3′UTR of UGT1A subfamily enzymes lies in their potential to significantly impact mRNA stability and translation efficiency, thus affecting transcript levels and enzyme activity. A recent study by [Court et al. (2013)](#B6) determined the association of rs8330 genotypes with glucuronidation of acetaminophen, which has both an amide and an alcohol linkage, and demonstrated that the rs8330 SNP was associated with acetaminophen glucuronidation activities in human liver bank samples (*n* = 48). In that study, individuals who were heterozygous (CG) for the rs8330 had mean acetaminophen glucuronidation activities that were significantly higher than the common rs8330 CC homozygotes, but significantly lower than the variant rs8330 GG homozygotes (CC < CG < GG). Our findings with *O*-4OHTam glucuronidation are similar. We demonstrate that individuals homozygous for the rs8330 common allele (CC) had significantly lower glucuronidation activity toward our selected compounds than individuals with either the CG or GG allele. Importantly, the 3′UTR SNP was found to significantly affect only *O*-4OHTam glucuronidation. The fact that this SNP is in the 3′UTR, which is common to all UGT1A subfamily enzymes, and that only *O*-glucuronidation was altered by changes at this position indicates that this effect was most likely due to alterations in another UGT1A enzyme, such as UGT1A1 or UGT1A3 ([Fig. 1](#F1)), and not by UGT1A4.

Although the effects of functional UGT polymorphisms on the glucuronidation of Tam were the focus of previous research ([Sun et al., 2006](#B36); [Blevins-Primeau et al., 2009](#B2); [Lazarus et al., 2009](#B24); [Lazarus and Sun, 2010](#B25)), to our knowledge, this study is the first to describe the association of *UGT1A4* promoter SNPs and glucuronidation of *N*-Tam, *N*-4OHTam, *O*-4OHTam *N*-Tor, *N*-4OHTor, and *O*-4OHTor. Here we demonstrated a significant impact of genetic variability in the *UGT1A4* promoter region on xenobiotic glucuronidation, which highlights the importance of *UGT1A4* genotype in breast cancer pharmacogenomic studies. The novel data presented in this work indicate that genetic variation in UGT1A4, which affects the expression levels of this enzyme, could potentially contribute to the bioavailability and disposition of Tam and its derivatives in patients, could be a potential biomarker of Tam response, and should be further investigated as a phase II biomarker of Tam efficacy and toxicity.

## Supplementary Material

## Abbreviations

## Authorship Contributions

*Participated in research design:* Kadlubar, Radominska-Pandya.

*Conducted experiments:* Greer, Dates, Starlard-Davenport, Edavana.

*Contributed new reagents or analytic tools:* Finel.

*Performed data analysis:* Bratton, Dhakal.

*Wrote or contributed to the writing of the manuscript:* Greer, Dates, Starlard-Davenport, Edavana, Bratton, Dhakal, Finel_,_ Kadlubar, Radominska-Pandya.

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Benoit-Biancamano MO, Adam JP, Bernard O, Court MH, Leblanc MH, Caron P, Guillemette C. (2009) A pharmacogenetics study of the human glucuronosyltransferase UGT1A4. Pharmacogenet Genomics 19:945–954  [DOI](https://doi.org/10.1097/FPC.0b013e3283331637) | [PMC free article](/articles/PMC6177227/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19890225/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=A%20pharmacogenetics%20study%20of%20the%20human%20glucuronosyltransferase%20UGT1A4&author=MO%20Benoit-Biancamano&author=JP%20Adam&author=O%20Bernard&author=MH%20Court&author=MH%20Leblanc&volume=19&publication_year=2009&pages=945-954&pmid=19890225&doi=10.1097/FPC.0b013e3283331637&)

2. Blevins-Primeau AS, Sun D, Chen G, Sharma AK, Gallagher CJ, Amin S, Lazarus P. (2009) Functional significance of UDP-glucuronosyltransferase variants in the metabolism of active tamoxifen metabolites. Cancer Res 69:1892–1900  [DOI](https://doi.org/10.1158/0008-5472.CAN-08-3708) | [PMC free article](/articles/PMC2683344/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19244109/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=Functional%20significance%20of%20UDP-glucuronosyltransferase%20variants%20in%20the%20metabolism%20of%20active%20tamoxifen%20metabolites&author=AS%20Blevins-Primeau&author=D%20Sun&author=G%20Chen&author=AK%20Sharma&author=CJ%20Gallagher&volume=69&publication_year=2009&pages=1892-1900&pmid=19244109&doi=10.1158/0008-5472.CAN-08-3708&)

3. Bradford MM. (1976) A rapid and sensitive method for the quantitation of microgram quantities of protein utilizing the principle of protein-dye binding. Anal Biochem 72:248–254  [DOI](https://doi.org/10.1016/0003-2697(76)90527-3) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/942051/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Anal%20Biochem&title=A%20rapid%20and%20sensitive%20method%20for%20the%20quantitation%20of%20microgram%20quantities%20of%20protein%20utilizing%20the%20principle%20of%20protein-dye%20binding&author=MM%20Bradford&volume=72&publication_year=1976&pages=248-254&pmid=942051&doi=10.1016/0003-2697(76)90527-3&)

4. Brauch H, Mürdter TE, Eichelbaum M, Schwab M. (2009) Pharmacogenomics of tamoxifen therapy. Clin Chem 55:1770–1782  [DOI](https://doi.org/10.1373/clinchem.2008.121756) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19574470/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Chem&title=Pharmacogenomics%20of%20tamoxifen%20therapy&author=H%20Brauch&author=TE%20M%C3%BCrdter&author=M%20Eichelbaum&author=M%20Schwab&volume=55&publication_year=2009&pages=1770-1782&pmid=19574470&doi=10.1373/clinchem.2008.121756&)

5. Burchell B. (2003) Genetic variation of human UDP-glucuronosyltransferase: implications in disease and drug glucuronidation. Am J Pharmacogenomics 3:37–52  [DOI](https://doi.org/10.2165/00129785-200303010-00006) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12562215/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Pharmacogenomics&title=Genetic%20variation%20of%20human%20UDP-glucuronosyltransferase:%20implications%20in%20disease%20and%20drug%20glucuronidation&author=B%20Burchell&volume=3&publication_year=2003&pages=37-52&pmid=12562215&doi=10.2165/00129785-200303010-00006&)

6. Court MH, Freytsis M, Wang X, Peter I, Guillemette C, Hazarika S, Duan SX, Greenblatt DJ, Lee WM, Acute Liver Failure Study Group (2013) The UDP-glucuronosyltransferase (UGT) 1A polymorphism c.2042C>G (rs8330) is associated with increased human liver acetaminophen glucuronidation, increased UGT1A exon 5a/5b splice variant mRNA ratio, and decreased risk of unintentional acetaminophen-induced acute liver failure. J Pharmacol Exp Ther 345:297–307  [DOI](https://doi.org/10.1124/jpet.112.202010) | [PMC free article](/articles/PMC3629801/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23408116/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharmacol%20Exp%20Ther&title=The%20UDP-glucuronosyltransferase%20(UGT)%201A%20polymorphism%20c.2042C>G%20(rs8330)%20is%20associated%20with%20increased%20human%20liver%20acetaminophen%20glucuronidation,%20increased%20UGT1A%20exon%205a/5b%20splice%20variant%20mRNA%20ratio,%20and%20decreased%20risk%20of%20unintentional%20acetaminophen-induced%20acute%20liver%20failure&author=MH%20Court&author=M%20Freytsis&author=X%20Wang&author=I%20Peter&author=C%20Guillemette&volume=345&publication_year=2013&pages=297-307&pmid=23408116&doi=10.1124/jpet.112.202010&)

7. Crettol S, Petrovic N, Murray M. (2010) Pharmacogenetics of phase I and phase II drug metabolism. Curr Pharm Des 16:204–219  [DOI](https://doi.org/10.2174/138161210790112674) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19835560/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Pharm%20Des&title=Pharmacogenetics%20of%20phase%20I%20and%20phase%20II%20drug%20metabolism&author=S%20Crettol&author=N%20Petrovic&author=M%20Murray&volume=16&publication_year=2010&pages=204-219&pmid=19835560&doi=10.2174/138161210790112674&)

8. de Leon J, Susce MT, Murray-Carmichael E. (2006) The AmpliChip CYP450 genotyping test: Integrating a new clinical tool. Mol Diagn Ther 10:135–151  [DOI](https://doi.org/10.1007/BF03256453) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16771600/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Diagn%20Ther&title=The%20AmpliChip%20CYP450%20genotyping%20test:%20Integrating%20a%20new%20clinical%20tool&author=J%20de%20Leon&author=MT%20Susce&author=E%20Murray-Carmichael&volume=10&publication_year=2006&pages=135-151&pmid=16771600&doi=10.1007/BF03256453&)

9. Doerge DR, Chang HC, Churchwell MI, Holder CL. (2000) Analysis of soy isoflavone conjugation in vitro and in human blood using liquid chromatography-mass spectrometry. Drug Metab Dispos 28:298–307  [PubMed](https://pubmed.ncbi.nlm.nih.gov/10681374/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Analysis%20of%20soy%20isoflavone%20conjugation%20in%20vitro%20and%20in%20human%20blood%20using%20liquid%20chromatography-mass%20spectrometry&author=DR%20Doerge&author=HC%20Chang&author=MI%20Churchwell&author=CL%20Holder&volume=28&publication_year=2000&pages=298-307&pmid=10681374&)

10. Dutton GJ. (1980) Glucuronidation of Drugs and Other Compounds, CRC Press Inc., Boca Raton, FL.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Glucuronidation%20of%20Drugs%20and%20Other%20Compounds&author=GJ%20Dutton&publication_year=1980&)

11. Edavana VK, Dhakal IB, Williams S, Penney R, Boysen G, Yao-Borengasser A, Kadlubar S. (2013) Potential role of UGT1A4 promoter SNPs in anastrozole pharmacogenomics. Drug Metab Dispos 41:870–877  [DOI](https://doi.org/10.1124/dmd.112.048157) | [PMC free article](/articles/PMC3608453/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23371966/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Potential%20role%20of%20UGT1A4%20promoter%20SNPs%20in%20anastrozole%20pharmacogenomics&author=VK%20Edavana&author=IB%20Dhakal&author=S%20Williams&author=R%20Penney&author=G%20Boysen&volume=41&publication_year=2013&pages=870-877&pmid=23371966&doi=10.1124/dmd.112.048157&)

12. Edavana VK, Dhakal IB, Yu X, Williams S, Kadlubar S. (2012) Sulfation of 4-hydroxy toremifene: individual variability, isoform specificity, and contribution to toremifene pharmacogenomics. Drug Metab Dispos 40:1210–1215  [DOI](https://doi.org/10.1124/dmd.111.044040) | [PMC free article](/articles/PMC3362788/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22434874/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Sulfation%20of%204-hydroxy%20toremifene:%20individual%20variability,%20isoform%20specificity,%20and%20contribution%20to%20toremifene%20pharmacogenomics&author=VK%20Edavana&author=IB%20Dhakal&author=X%20Yu&author=S%20Williams&author=S%20Kadlubar&volume=40&publication_year=2012&pages=1210-1215&pmid=22434874&doi=10.1124/dmd.111.044040&)

13. Erichsen TJ, Ehmer U, Kalthoff S, Lankisch TO, Müller TM, Munzel PA, Manns MP, Strassburg CP. (2008) Genetic variability of aryl hydrocarbon receptor (AhR)-mediated regulation of the human UDP glucuronosyltransferase (UGT) 1A4 gene. Toxicol Appl Pharmacol 230:252–260  [DOI](https://doi.org/10.1016/j.taap.2008.02.020) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18433817/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Toxicol%20Appl%20Pharmacol&title=Genetic%20variability%20of%20aryl%20hydrocarbon%20receptor%20(AhR)-mediated%20regulation%20of%20the%20human%20UDP%20glucuronosyltransferase%20(UGT)%201A4%20gene&author=TJ%20Erichsen&author=U%20Ehmer&author=S%20Kalthoff&author=TO%20Lankisch&author=TM%20M%C3%BCller&volume=230&publication_year=2008&pages=252-260&pmid=18433817&doi=10.1016/j.taap.2008.02.020&)

14. Ghotbi R, Mannheimer B, Aklillu E, Suda A, Bertilsson L, Eliasson E, Osby U. (2010) Carriers of the UGT1A4 142T>G gene variant are predisposed to reduced olanzapine exposure—an impact similar to male gender or smoking in schizophrenic patients. Eur J Clin Pharmacol 66:465–474  [DOI](https://doi.org/10.1007/s00228-009-0783-8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20143052/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Clin%20Pharmacol&title=Carriers%20of%20the%20UGT1A4%20142T>G%20gene%20variant%20are%20predisposed%20to%20reduced%20olanzapine%20exposure%E2%80%94an%20impact%20similar%20to%20male%20gender%20or%20smoking%20in%20schizophrenic%20patients&author=R%20Ghotbi&author=B%20Mannheimer&author=E%20Aklillu&author=A%20Suda&author=L%20Bertilsson&volume=66&publication_year=2010&pages=465-474&pmid=20143052&doi=10.1007/s00228-009-0783-8&)

15. Gulcebi MI, Ozkaynakcı A, Goren MZ, Aker RG, Ozkara C, Onat FY. (2011) The relationship between UGT1A4 polymorphism and serum concentration of lamotrigine in patients with epilepsy. Epilepsy Res 95:1–8  [DOI](https://doi.org/10.1016/j.eplepsyres.2011.01.016) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21601426/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Epilepsy%20Res&title=The%20relationship%20between%20UGT1A4%20polymorphism%20and%20serum%20concentration%20of%20lamotrigine%20in%20patients%20with%20epilepsy&author=MI%20Gulcebi&author=A%20Ozkaynakc%C4%B1&author=MZ%20Goren&author=RG%20Aker&author=C%20Ozkara&volume=95&publication_year=2011&pages=1-8&pmid=21601426&doi=10.1016/j.eplepsyres.2011.01.016&)

16. Hoskins JM, Carey LA, McLeod HL. (2009) CYP2D6 and tamoxifen: DNA matters in breast cancer. Nat Rev Cancer 9:576–586  [DOI](https://doi.org/10.1038/nrc2683) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19629072/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Cancer&title=CYP2D6%20and%20tamoxifen:%20DNA%20matters%20in%20breast%20cancer&author=JM%20Hoskins&author=LA%20Carey&author=HL%20McLeod&volume=9&publication_year=2009&pages=576-586&pmid=19629072&doi=10.1038/nrc2683&)

17. Ingle JN. (2013) Pharmacogenomics of endocrine therapy in breast cancer. J Hum Genet 58:306–312  [DOI](https://doi.org/10.1038/jhg.2013.35) | [PMC free article](/articles/PMC3926210/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23635953/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Hum%20Genet&title=Pharmacogenomics%20of%20endocrine%20therapy%20in%20breast%20cancer&author=JN%20Ingle&volume=58&publication_year=2013&pages=306-312&pmid=23635953&doi=10.1038/jhg.2013.35&)

18. Kaivosaari S, Toivonen P, Aitio O, Sipilä J, Koskinen M, Salonen JS, Finel M. (2008) Regio- and stereospecific N-glucuronidation of medetomidine: the differences between UDP glucuronosyltransferase (UGT) 1A4 and UGT2B10 account for the complex kinetics of human liver microsomes. Drug Metab Dispos 36:1529–1537  [DOI](https://doi.org/10.1124/dmd.108.021709) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18474681/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Regio-%20and%20stereospecific%20N-glucuronidation%20of%20medetomidine:%20the%20differences%20between%20UDP%20glucuronosyltransferase%20(UGT)%201A4%20and%20UGT2B10%20account%20for%20the%20complex%20kinetics%20of%20human%20liver%20microsomes&author=S%20Kaivosaari&author=P%20Toivonen&author=O%20Aitio&author=J%20Sipil%C3%A4&author=M%20Koskinen&volume=36&publication_year=2008&pages=1529-1537&pmid=18474681&doi=10.1124/dmd.108.021709&)

19. Kaku T, Ogura K, Nishiyama T, Ohnuma T, Muro K, Hiratsuka A. (2004) Quaternary ammonium-linked glucuronidation of tamoxifen by human liver microsomes and UDP-glucuronosyltransferase 1A4. Biochem Pharmacol 67:2093–2102  [DOI](https://doi.org/10.1016/j.bcp.2004.02.014) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15135306/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem%20Pharmacol&title=Quaternary%20ammonium-linked%20glucuronidation%20of%20tamoxifen%20by%20human%20liver%20microsomes%20and%20UDP-glucuronosyltransferase%201A4&author=T%20Kaku&author=K%20Ogura&author=T%20Nishiyama&author=T%20Ohnuma&author=K%20Muro&volume=67&publication_year=2004&pages=2093-2102&pmid=15135306&doi=10.1016/j.bcp.2004.02.014&)

20. Kato Y, Izukawa T, Oda S, Fukami T, Finel M, Yokoi T, Nakajima M. (2013) Human UDP-glucuronosyltransferase (UGT) 2B10 in drug N-glucuronidation: substrate screening and comparison with UGT1A3 and UGT1A4. Drug Metab Dispos 41:1389–1397  [DOI](https://doi.org/10.1124/dmd.113.051565) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23611809/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Human%20UDP-glucuronosyltransferase%20(UGT)%202B10%20in%20drug%20N-glucuronidation:%20substrate%20screening%20and%20comparison%20with%20UGT1A3%20and%20UGT1A4&author=Y%20Kato&author=T%20Izukawa&author=S%20Oda&author=T%20Fukami&author=M%20Finel&volume=41&publication_year=2013&pages=1389-1397&pmid=23611809&doi=10.1124/dmd.113.051565&)

21. Kiyotani K, Mushiroda T, Zembutsu H, Nakamura Y. (2013) Important and critical scientific aspects in pharmacogenomics analysis: lessons from controversial results of tamoxifen and CYP2D6 studies. J Hum Genet 58:327–333  [DOI](https://doi.org/10.1038/jhg.2013.39) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23657426/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Hum%20Genet&title=Important%20and%20critical%20scientific%20aspects%20in%20pharmacogenomics%20analysis:%20lessons%20from%20controversial%20results%20of%20tamoxifen%20and%20CYP2D6%20studies&author=K%20Kiyotani&author=T%20Mushiroda&author=H%20Zembutsu&author=Y%20Nakamura&volume=58&publication_year=2013&pages=327-333&pmid=23657426&doi=10.1038/jhg.2013.39&)

22. Kurkela M, García-Horsman JA, Luukkanen L, Mörsky S, Taskinen J, Baumann M, Kostiainen R, Hirvonen J, Finel M. (2003) Expression and characterization of recombinant human UDP-glucuronosyltransferases (UGTs). UGT1A9 is more resistant to detergent inhibition than other UGTs and was purified as an active dimeric enzyme. J Biol Chem 278:3536–3544  [DOI](https://doi.org/10.1074/jbc.M206136200) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12435745/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=Expression%20and%20characterization%20of%20recombinant%20human%20UDP-glucuronosyltransferases%20(UGTs).%20UGT1A9%20is%20more%20resistant%20to%20detergent%20inhibition%20than%20other%20UGTs%20and%20was%20purified%20as%20an%20active%20dimeric%20enzyme&author=M%20Kurkela&author=JA%20Garc%C3%ADa-Horsman&author=L%20Luukkanen&author=S%20M%C3%B6rsky&author=J%20Taskinen&volume=278&publication_year=2003&pages=3536-3544&pmid=12435745&doi=10.1074/jbc.M206136200&)

23. Kuuranne T, Kurkela M, Thevis M, Schänzer W, Finel M, Kostiainen R. (2003) Glucuronidation of anabolic androgenic steroids by recombinant human UDP-glucuronosyltransferases. Drug Metab Dispos 31:1117–1124  [DOI](https://doi.org/10.1124/dmd.31.9.1117) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12920167/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Glucuronidation%20of%20anabolic%20androgenic%20steroids%20by%20recombinant%20human%20UDP-glucuronosyltransferases&author=T%20Kuuranne&author=M%20Kurkela&author=M%20Thevis&author=W%20Sch%C3%A4nzer&author=M%20Finel&volume=31&publication_year=2003&pages=1117-1124&pmid=12920167&doi=10.1124/dmd.31.9.1117&)

24. Lazarus P, Blevins-Primeau AS, Zheng Y, Sun D. (2009) Potential role of UGT pharmacogenetics in cancer treatment and prevention: focus on tamoxifen. Ann N Y Acad Sci 1155:99–111  [DOI](https://doi.org/10.1111/j.1749-6632.2009.04114.x) | [PMC free article](/articles/PMC2694135/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19250197/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20N%20Y%20Acad%20Sci&title=Potential%20role%20of%20UGT%20pharmacogenetics%20in%20cancer%20treatment%20and%20prevention:%20focus%20on%20tamoxifen&author=P%20Lazarus&author=AS%20Blevins-Primeau&author=Y%20Zheng&author=D%20Sun&volume=1155&publication_year=2009&pages=99-111&pmid=19250197&doi=10.1111/j.1749-6632.2009.04114.x&)

25. Lazarus P, Sun D. (2010) Potential role of UGT pharmacogenetics in cancer treatment and prevention: focus on tamoxifen and aromatase inhibitors. Drug Metab Rev 42:182–194  [DOI](https://doi.org/10.3109/03602530903208652) | [PMC free article](/articles/PMC3072888/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19821643/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Rev&title=Potential%20role%20of%20UGT%20pharmacogenetics%20in%20cancer%20treatment%20and%20prevention:%20focus%20on%20tamoxifen%20and%20aromatase%20inhibitors&author=P%20Lazarus&author=D%20Sun&volume=42&publication_year=2010&pages=182-194&pmid=19821643&doi=10.3109/03602530903208652&)

26. Lim HS, Ju Lee H, Seok Lee K, Sook Lee E, Jang IJ, Ro J. (2007) Clinical implications of CYP2D6 genotypes predictive of tamoxifen pharmacokinetics in metastatic breast cancer. J Clin Oncol 25:3837–3845  [DOI](https://doi.org/10.1200/JCO.2007.11.4850) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17761971/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Clinical%20implications%20of%20CYP2D6%20genotypes%20predictive%20of%20tamoxifen%20pharmacokinetics%20in%20metastatic%20breast%20cancer&author=HS%20Lim&author=H%20Ju%20Lee&author=K%20Seok%20Lee&author=E%20Sook%20Lee&author=IJ%20Jang&volume=25&publication_year=2007&pages=3837-3845&pmid=17761971&doi=10.1200/JCO.2007.11.4850&)

27. López M, Dorado P, Monroy N, Alonso ME, Jung-Cook H, Machín E, Peñas-Lledó E, Llerena A. (2011) Pharmacogenetics of the antiepileptic drugs phenytoin and lamotrigine. Drug Metabol Drug Interact 26:5–12  [DOI](https://doi.org/10.1515/DMDI.2011.008) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21557672/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metabol%20Drug%20Interact&title=Pharmacogenetics%20of%20the%20antiepileptic%20drugs%20phenytoin%20and%20lamotrigine&author=M%20L%C3%B3pez&author=P%20Dorado&author=N%20Monroy&author=ME%20Alonso&author=H%20Jung-Cook&volume=26&publication_year=2011&pages=5-12&pmid=21557672&doi=10.1515/DMDI.2011.008&)

28. Mazzarino M, Biava M, de la Torre X, Fiacco I, Botrè F. (2013) Characterization of the biotransformation pathways of clomiphene, tamoxifen and toremifene as assessed by LC-MS/(MS) following in vitro and excretion studies. Anal Bioanal Chem 405:5467–5487  [DOI](https://doi.org/10.1007/s00216-013-6961-7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23604525/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Anal%20Bioanal%20Chem&title=Characterization%20of%20the%20biotransformation%20pathways%20of%20clomiphene,%20tamoxifen%20and%20toremifene%20as%20assessed%20by%20LC-MS/(MS)%20following%20in%20vitro%20and%20excretion%20studies&author=M%20Mazzarino&author=M%20Biava&author=X%20de%20la%20Torre&author=I%20Fiacco&author=F%20Botr%C3%A8&volume=405&publication_year=2013&pages=5467-5487&pmid=23604525&doi=10.1007/s00216-013-6961-7&)

29. Nussbaum R, McInnes RR, Willard HF. (2007) The human genome and the chromosomal basis of heredity, in Thompson and Thompson Genetics in Medicine (Nussbaum R, McInnes RR, Willard HF. eds) pp 5–25, Elsevier Saunders, Philadelphia  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Thompson%20and%20Thompson%20Genetics%20in%20Medicine&author=R%20Nussbaum&author=RR%20McInnes&author=HF%20Willard&publication_year=2007&)

30. Ogura K, Ishikawa Y, Kaku T, Nishiyama T, Ohnuma T, Muro K, Hiratsuka A. (2006) Quaternary ammonium-linked glucuronidation of trans-4-hydroxytamoxifen, an active metabolite of tamoxifen, by human liver microsomes and UDP-glucuronosyltransferase 1A4. Biochem Pharmacol 71:1358–1369  [DOI](https://doi.org/10.1016/j.bcp.2006.01.004) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16480962/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem%20Pharmacol&title=Quaternary%20ammonium-linked%20glucuronidation%20of%20trans-4-hydroxytamoxifen,%20an%20active%20metabolite%20of%20tamoxifen,%20by%20human%20liver%20microsomes%20and%20UDP-glucuronosyltransferase%201A4&author=K%20Ogura&author=Y%20Ishikawa&author=T%20Kaku&author=T%20Nishiyama&author=T%20Ohnuma&volume=71&publication_year=2006&pages=1358-1369&pmid=16480962&doi=10.1016/j.bcp.2006.01.004&)

31. Parte P, Kupfer D. (2005) Oxidation of tamoxifen by human flavin-containing monooxygenase (FMO) 1 and FMO3 to tamoxifen-N-oxide and its novel reduction back to tamoxifen by human cytochromes P450 and hemoglobin. Drug Metab Dispos 33:1446–1452  [DOI](https://doi.org/10.1124/dmd.104.000802) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15987777/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Oxidation%20of%20tamoxifen%20by%20human%20flavin-containing%20monooxygenase%20(FMO)%201%20and%20FMO3%20to%20tamoxifen-N-oxide%20and%20its%20novel%20reduction%20back%20to%20tamoxifen%20by%20human%20cytochromes%20P450%20and%20hemoglobin&author=P%20Parte&author=D%20Kupfer&volume=33&publication_year=2005&pages=1446-1452&pmid=15987777&doi=10.1124/dmd.104.000802&)

32. Rae JM, Drury S, Hayes DF, Stearns V, Thibert JN, Haynes BP, Salter J, Sestak I, Cuzick J, Dowsett M, ATAC trialists (2012) CYP2D6 and UGT2B7 genotype and risk of recurrence in tamoxifen-treated breast cancer patients. J Natl Cancer Inst 104:452–460  [DOI](https://doi.org/10.1093/jnci/djs126) | [PMC free article](/articles/PMC3611934/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22395643/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Natl%20Cancer%20Inst&title=CYP2D6%20and%20UGT2B7%20genotype%20and%20risk%20of%20recurrence%20in%20tamoxifen-treated%20breast%20cancer%20patients&author=JM%20Rae&author=S%20Drury&author=DF%20Hayes&author=V%20Stearns&author=JN%20Thibert&volume=104&publication_year=2012&pages=452-460&pmid=22395643&doi=10.1093/jnci/djs126&)

33. Regan MM, Leyland-Jones B, Bouzyk M, Pagani O, Tang W, Kammler R, Dell’orto P, Biasi MO, Thürlimann B, Lyng MB, et al. Breast International Group (BIG) 1-98 Collaborative Group (2012) CYP2D6 genotype and tamoxifen response in postmenopausal women with endocrine-responsive breast cancer: the Breast International Group 1-98 trial. J Natl Cancer Inst 104:441–451  [DOI](https://doi.org/10.1093/jnci/djs125) | [PMC free article](/articles/PMC3309132/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22395644/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Natl%20Cancer%20Inst&title=CYP2D6%20genotype%20and%20tamoxifen%20response%20in%20postmenopausal%20women%20with%20endocrine-responsive%20breast%20cancer:%20the%20Breast%20International%20Group%201-98%20trial&author=MM%20Regan&author=B%20Leyland-Jones&author=M%20Bouzyk&author=O%20Pagani&author=W%20Tang&volume=104&publication_year=2012&pages=441-451&pmid=22395644&doi=10.1093/jnci/djs125&)

34. Saeki M, Saito Y, Jinno H, Sai K, Hachisuka A, Kaniwa N, Ozawa S, Kawamoto M, Kamatani N, Shirao K, et al. (2005) Genetic variations and haplotypes of UGT1A4 in a Japanese population. Drug Metab Pharmacokinet 20:144–151  [DOI](https://doi.org/10.2133/dmpk.20.144) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15855727/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Pharmacokinet&title=Genetic%20variations%20and%20haplotypes%20of%20UGT1A4%20in%20a%20Japanese%20population&author=M%20Saeki&author=Y%20Saito&author=H%20Jinno&author=K%20Sai&author=A%20Hachisuka&volume=20&publication_year=2005&pages=144-151&pmid=15855727&doi=10.2133/dmpk.20.144&)

35. Sakorafas GH, Farley DR, Peros G. (2008) Recent advances and current controversies in the management of DCIS of the breast. Cancer Treat Rev 34:483–497  [DOI](https://doi.org/10.1016/j.ctrv.2008.03.001) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18490111/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Treat%20Rev&title=Recent%20advances%20and%20current%20controversies%20in%20the%20management%20of%20DCIS%20of%20the%20breast&author=GH%20Sakorafas&author=DR%20Farley&author=G%20Peros&volume=34&publication_year=2008&pages=483-497&pmid=18490111&doi=10.1016/j.ctrv.2008.03.001&)

36. Sun D, Chen G, Dellinger RW, Duncan K, Fang JL, Lazarus P. (2006) Characterization of tamoxifen and 4-hydroxytamoxifen glucuronidation by human UGT1A4 variants. Breast Cancer Res 8:R50.  [DOI](https://doi.org/10.1186/bcr1539) | [PMC free article](/articles/PMC1779459/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16884532/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Breast%20Cancer%20Res&title=Characterization%20of%20tamoxifen%20and%204-hydroxytamoxifen%20glucuronidation%20by%20human%20UGT1A4%20variants&author=D%20Sun&author=G%20Chen&author=RW%20Dellinger&author=K%20Duncan&author=JL%20Fang&volume=8&publication_year=2006&pages=R50&pmid=16884532&doi=10.1186/bcr1539&)

37. Sun D, Sharma AK, Dellinger RW, Blevins-Primeau AS, Balliet RM, Chen G, Boyiri T, Amin S, Lazarus P. (2007) Glucuronidation of active tamoxifen metabolites by the human UDP glucuronosyltransferases. Drug Metab Dispos 35:2006–2014  [DOI](https://doi.org/10.1124/dmd.107.017145) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17664247/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Glucuronidation%20of%20active%20tamoxifen%20metabolites%20by%20the%20human%20UDP%20glucuronosyltransferases&author=D%20Sun&author=AK%20Sharma&author=RW%20Dellinger&author=AS%20Blevins-Primeau&author=RM%20Balliet&volume=35&publication_year=2007&pages=2006-2014&pmid=17664247&doi=10.1124/dmd.107.017145&)

38. Tukey RH, Strassburg CP. (2000) Human UDP-glucuronosyltransferases: metabolism, expression, and disease. Annu Rev Pharmacol Toxicol 40:581–616  [DOI](https://doi.org/10.1146/annurev.pharmtox.40.1.581) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10836148/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Annu%20Rev%20Pharmacol%20Toxicol&title=Human%20UDP-glucuronosyltransferases:%20metabolism,%20expression,%20and%20disease&author=RH%20Tukey&author=CP%20Strassburg&volume=40&publication_year=2000&pages=581-616&pmid=10836148&doi=10.1146/annurev.pharmtox.40.1.581&)

39. Tukey RH, Strassburg CP. (2001) Genetic multiplicity of the human UDP-glucuronosyltransferases and regulation in the gastrointestinal tract. Mol Pharmacol 59:405–414  [DOI](https://doi.org/10.1124/mol.59.3.405) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11179432/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Pharmacol&title=Genetic%20multiplicity%20of%20the%20human%20UDP-glucuronosyltransferases%20and%20regulation%20in%20the%20gastrointestinal%20tract&author=RH%20Tukey&author=CP%20Strassburg&volume=59&publication_year=2001&pages=405-414&pmid=11179432&doi=10.1124/mol.59.3.405&)

40. Turesky RJ, Lang NP, Butler MA, Teitel CH, Kadlubar FF. (1991) Metabolic activation of carcinogenic heterocyclic aromatic amines by human liver and colon. Carcinogenesis 12:1839–1845  [DOI](https://doi.org/10.1093/carcin/12.10.1839) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/1934265/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Carcinogenesis&title=Metabolic%20activation%20of%20carcinogenic%20heterocyclic%20aromatic%20amines%20by%20human%20liver%20and%20colon&author=RJ%20Turesky&author=NP%20Lang&author=MA%20Butler&author=CH%20Teitel&author=FF%20Kadlubar&volume=12&publication_year=1991&pages=1839-1845&pmid=1934265&doi=10.1093/carcin/12.10.1839&)

41. Zhou J, Argikar UA, Remmel RP. (2011) Functional analysis of UGT1A4(P24T) and UGT1A4(L48V) variant enzymes. Pharmacogenomics 12:1671–1679  [DOI](https://doi.org/10.2217/pgs.11.105) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22047493/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Functional%20analysis%20of%20UGT1A4(P24T)%20and%20UGT1A4(L48V)%20variant%20enzymes&author=J%20Zhou&author=UA%20Argikar&author=RP%20Remmel&volume=12&publication_year=2011&pages=1671-1679&pmid=22047493&doi=10.2217/pgs.11.105&)

42. Zhou J, Tracy TS, Remmel RP. (2010) Glucuronidation of dihydrotestosterone and trans-androsterone by recombinant UDP-glucuronosyltransferase (UGT) 1A4: evidence for multiple UGT1A4 aglycone binding sites. Drug Metab Dispos 38:431–440  [DOI](https://doi.org/10.1124/dmd.109.028712) | [PMC free article](/articles/PMC2835394/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20007295/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Glucuronidation%20of%20dihydrotestosterone%20and%20trans-androsterone%20by%20recombinant%20UDP-glucuronosyltransferase%20(UGT)%201A4:%20evidence%20for%20multiple%20UGT1A4%20aglycone%20binding%20sites&author=J%20Zhou&author=TS%20Tracy&author=RP%20Remmel&volume=38&publication_year=2010&pages=431-440&pmid=20007295&doi=10.1124/dmd.109.028712&)
