# A Heroin Addiction Severity-Associated Intronic Single Nucleotide Polymorphism Modulates Alternative Pre-mRNA Splicing of the μ Opioid Receptor Gene OPRM1 via hnRNPH Interactions

## Metadata
**Authors:** Jin Xu, Zhigang Lu, Mingming Xu, Ling Pan, Yi Deng, Xiaohu Xie, Huifen Liu, Shixiong Ding, Yasmin L Hurd, Gavril W Pasternak, Robert J Klein, Luca Cartegni, Wenhua Zhou, Ying-Xian Pan
**Journal:** The Journal of Neuroscience
**Date:** 2014 Aug 13
**DOI:** [10.1523/JNEUROSCI.3986-13.2014](https://doi.org/10.1523/JNEUROSCI.3986-13.2014)
**PMID:** 25122903
**PMCID:** PMC4131016
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4131016/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC4131016/pdf/zns11048.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC4131016/pdf/zns11048.pdf)

## Abstract

Single nucleotide polymorphisms (SNPs) in the OPRM1 gene have been associated with vulnerability to opioid dependence. The current study identifies an association of an intronic SNP (rs9479757) with the severity of heroin addiction among Han-Chinese male heroin addicts. Individual SNP analysis and haplotype-based analysis with additional SNPs in the OPRM1 locus showed that mild heroin addiction was associated with the AG genotype, whereas severe heroin addiction was associated with the GG genotype. In vitro studies such as electrophoretic mobility shift assay, minigene, siRNA, and antisense morpholino oligonucleotide studies have identified heterogeneous nuclear ribonucleoprotein H (hnRNPH) as the major binding partner for the G-containing SNP site. The G-to-A transition weakens hnRNPH binding and facilitates exon 2 skipping, leading to altered expressions of OPRM1 splice-variant mRNAs and hMOR-1 proteins. Similar changes in splicing and hMOR-1 proteins were observed in human postmortem prefrontal cortex with the AG genotype of this SNP when compared with the GG genotype. Interestingly, the altered splicing led to an increase in hMOR-1 protein levels despite decreased hMOR-1 mRNA levels, which is likely contributed by a concurrent increase in single transmembrane domain variants that have a chaperone-like function on MOR-1 protein stability. Our studies delineate the role of this SNP as a modifier of OPRM1 alternative splicing via hnRNPH interactions, and suggest a functional link between an SNP-containing splicing modifier and the severity of heroin addiction.

Keywords: addiction, heroin, hnRNPH, opioid receptor, SNP, splicing

## Introduction

The μ opioid receptor gene *OPRM1* undergoes extensive alternative pre-mRNA splicing, creating an array of splice variants that are conserved from rodent to human. These splice variants can be categorized into the following three major types based on receptor structure: (1) full-length C-terminal variants with 7-transmembrane (TM) domains; (2) truncated variants containing 6-TM domains; and (3) truncated variants containing a single TM ([Fig. 1](#F1)). Increasing evidence suggests that these splice variants play important roles in mediating the complex actions of various μ opioids ([Pan, 2005](#B35); [Pasternak and Pan, 2013](#B40)).

### Figure 1.

![Figure 1.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c31c/4131016/eb23a83c23d4/zns9991459970001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4131016_zns9991459970001.jpg)

Schematic of the human OPRM1 gene and alternative splicing of the single TM splice variants generated by exon 2 skipping. Exons and introns are shown by colored boxes and horizontal lines, respectively. Exons 1 and 11 promoters are indicated by red arrows. Intron sizes were labeled below the intron line. Transmembrane domain (TM) is shown as curved bars under the corresponding exons. Splicing is indicated by tilted lines connecting exons. Translation start and stop codons are shown on the top of the designated exons. Predicted protein structures were drawn at left side. Some alternative exons and their associated splice variants are omitted. The complete human OPRM1 gene structure and alternative splicing was described in literature (Pan, 2005; Pasternak and Pan, 2013).

Serving as the primary target for most clinical opioids, the *OPRM1* gene has been considered as the predominant genetic candidate responsible for vulnerability to opioid dependence ([Bond et al., 1998](#B6); [Uhl et al., 1999](#B49); [Hoehe et al., 2000](#B17); [Luo et al., 2003](#B30); [Ikeda et al., 2005](#B21); [Kreek et al., 2005](#B25)), as well as dependence on other substances of abuse, such as alcohol and nicotine ([Kim et al., 2004](#B23); [Lerman et al., 2004](#B29); [Ray and Hutchison, 2004](#B45); [Bart et al., 2005](#B1); [Berrettini and Lerman, 2005](#B4); [Zhang et al., 2006a](#B55)). Of the 4536 single nucleotide polymorphisms (SNPs) in the human *OPRM1* gene identified in the dbSNP database, rs1799971 (A118G) has been the most extensively studied in association with dependence and addiction to drugs of abuse, such as heroin.

An SNP located in intron 2 (rs9479757 or IVS2 + 31A/G) was initially reported to be associated with heroin consumption in a Han-Chinese population ([Shi et al., 2002](#B46)). In a twin population, this SNP was also associated with smoking initiation and nicotine dependence ([Zhang et al., 2006b](#B56)). Additionally, rs9479757 was found to be associated with pressure pain sensitivity in healthy Han-Chinese women ([Huang et al., 2008](#B19)), and with the severity of HIV infection as well as the response to HIV treatment ([Proudnikov et al., 2012](#B43)).

Studies have suggested the functional significance of several *OPRM1* SNPs. For example, A118G increases the affinity of MOR-1 for β-endorphin ([Bond et al., 1998](#B6); [Kroslak et al., 2007](#B26)), and lowers expression of MOR-1 mRNA and protein likely via altering an mRNA secondary structure ([Zhang et al., 2005](#B57)), or reducing the *N*-glycosylation and protein stability of MOR-1 ([Huang et al., 2012](#B20)). The roles of *OPRM1* SNPs have also been implicated in modulating μ agonist-induced receptor signaling and promoter activities ([Befort et al., 2001](#B3); [Kraus et al., 2001](#B24); [Wang et al., 2001](#B50); [Bayerer et al., 2007](#B2); [Ravindranathan et al., 2009](#B44); [Fortin et al., 2010](#B14)). However, little is known about the effect of SNPs on *OPRM1* alternative splicing.

In the current study, we identify an association of SNP rs9479757 with heroin addiction severity among Han-Chinese male heroin addicts, and explore the role of this SNP on *OPRM1* alternative splicing. Our data suggest that the G-containing SNP site functions as an intronic splicing enhancer (ISE) that mainly binds heterogeneous nuclear ribonucleoprotein H (hnRNPH), and that a G-to-A transition of the SNP weakens ISE via reducing hnRNPH binding. Consequently, weakened ISE-hnRNPH binding promotes exon 2 skipping, leading to altered expressions of *OPRM1* splice-variant mRNAs and hMOR-1 proteins. We also provide data on potential mechanisms by which hnRNPH regulates alternative splicing through the SNP-containing ISE.

## Materials and Methods

### 

#### 

[γ-^32^P]ATP (3000 Ci/mmol), [^35^S]GTPγS, and [^3^H] [d-Ala^2^, *N*-Me Phe^4^, Gly-ol-enkephalin (DAMGO)] were obtained from PerkinElmer. All oligodeoxynucleotides (oligos) used in bacterial artificial chromosome (BAC) minigene constructs and PCRs and RNA oligos were synthesized by Sigma-Aldrich and Integrated DNA Technologies, respectively. Ambion Silencer Select siRNAs were obtained from Life Technologies. Vivo-morpholino oligos were synthesized by Gene Tools. Be(2)C cells were obtained from American Type Culture Collection. The rabbit anti-hnRNPH antibody was a gift from Dr. Douglas Black (University of California, Los Angeles, Los Angeles, CA). The goat anti-hnRNPH (N-16), anti-hnRNPF (N-15), anti-hnRNPA1 (E-17), anti-hnRNP A2/B1 (E-2), anti-hnRNPU (C-15), anti-hMOR-1 exon 4 (C-20), anti-actin (I-19), and anti-GAPDH (6C5) antibodies were obtained from Santa Cruz Biotechnology. All other materials were obtained from the sources listed.

A total of 332 male heroin addicts (case group) were recruited from Ningbo city through the Ningbo Addiction Research and Treatment Center of Zhejiang Province, People's Republic of China, based on *Diagnostic and Statistical Manual of Mental Disorders*, fourth edition (DSM-IV) criteria for substance abuse and dependence. We did not include female subjects in this study because there were fewer female heroin addicts in the Ningbo city area. All subjects were males of Han-Chinese origin with a mean age of 31.8 ± 7.5 years, and had no serious psychiatric symptoms, or a history of hematological diseases, or severe impairment of the liver or kidneys. A total of 190 healthy individuals (control group) were recruited from among voluntary blood donors in the Ningbo blood center of Zhejiang Province. All subjects were males of Han-Chinese origin and had no history of drug abuse or mental illness. There was no significant difference in age between the case group (31.8 ± 7.5 years) and the control group (30.7 ± 8.4 years). The study protocol was approved by the Ethics Committee of Ningbo Addiction Research and Treatment Center, and written informed consent was obtained from all subjects.

Heroin addicts were classified into three groups (mild, moderate, and severe) using standard criteria previously described for heroin addiction severity ([Xie et al., 2013](#B53)), and performed through a questionnaire that collected information including age, education, employment, marriage, and specific information related to heroin abuse, such as drug route, duration, and daily self-administrated dosage. Addicts who smoked <0.5 g/d heroin for <1 year were classified into the mild group, while addicts who smoked >1.5 g/d of heroin for >2 years or injected >0.5 g of heroin for >6 months were grouped into the severe group. The remaining addicts were placed into the moderate group. There was no significant difference in age among the mild (31.2 ± 6.8 years), moderate (31.7 ± 7.8 years), and severe (32.1 ± 7.3 years) groups.

Genomic DNA was extracted from leukocytes of all blood samples provided using the standard phenol-chloroform method. rs9479757 in case and control subjects was genotyped using TaqMan SNP assay (Applied Biosystems) according to the manufacturer's protocol. Genotype calls were determined using Allelic Discrimination software (Applied Biosystems). Five percent of the samples were duplicated to verify the accuracy rate of genotyping. Any deviation from the Hardy–Weinberg equilibrium and the association between genotype/allele frequencies and phenotype were determined using a Pearson χ^2^ test (chisq.test, R version 3.0.2).

To identify an optimal set of tagging SNPs for the *OPRM1* region, we extracted all SNPs in a ∼105 kb *OPRM1* region (chromosome 6: 154359443–154464655, build 37) with minor allele frequency of >5% in individuals of Asian ancestry, using data from the 1000 Genomes Project. Using a greedy algorithm, we identified 30 SNPs that maximally captured all common variants in the region with pairwise tagging *r*^2^ > 0.8. Genotyping was performed by Shanghai Benegene Biotechnology Co., Ltd. using the Sequenom MassARRAY system (Sequenom Inc.), a chip-based assay using mass spectrometry. Seven of the 30 SNPs were excluded due to sequence-based complexity. Twenty-three SNPs, including rs9479757 and rs1799971, were genotyped in all samples from the 332 heroin addicts in the case group. SpectroTYPER software (Sequenome Inc.) was used to analyze the data. Associations among the three categorical phenotypes and each SNP were analyzed using the Pearson χ^2^ test (chisq.test, R version 3.0.2).

Fresh frozen human postmortem prefrontal cortex (PFC) samples were obtained from a postmortem collection of Caucasian Swedish and Hungarian subjects processed under conditions that have been previously described ([Drakenberg et al., 2006](#B12); [Jacobs et al., 2013](#B22)). Briefly, these subjects were recruited at the National Institute of Forensic Medicine, Karolinska Institute, Stockholm, Sweden, as well as from the Department of Forensic Medicine at Semmelweis University (Budapest, Hungary) under guidelines approved by the local ethics committees. Tissue punches from 27 male subjects with a mean age 27.5 ± 7.2 years who died from myocardial infarction and had negative toxicology results for psychoactive substances or a history of psychiatric illness were used for RNA and protein extraction.

An (HIS)_6_-GGGS tag was fused into the N terminus of hnRNPH in pCMV6-XL4 (OriGene) by using Change-IT Mutagenesis Kit (Affymetrix). The resulting plasmid was used as template to make (HIS)_6_-tagged hnRNPH protein by a T7 transcription/translation coupled (TnT) system (Promega) according to the manufacturer's protocol. (HIS)_6_-tagged hnRNPH was purified by using the MagZ system (Promega). Briefly, 375 μl of TnT mixture transcribed/translated from 7.5 μg of plasmid were incubated with 420 μl of MagZ Binding Particles in Mag-Binding/Washing buffer (Mag-B/W buffer, 20 mm sodium phosphate, pH 7.4, 500 mm NaCl, 0.3% Triton X-100) at room temperature for 30 min. After washing four times with Mag-B/W buffer containing 10 mm imidazole, bound proteins were eluted with 1 m imidazole, pH 7.5, and used in an electrophoretic mobility shift assay (EMSA).

EMSA was performed as described previously ([Lefave et al., 2011](#B28)). Briefly, a ^32^P-labeled 26 mer RNA oligo (86 fmol) containing the G allele (G31, 5′-GCC UGA GGG AAG GAG GGU UCA CAG CC-3′) was incubated with 10 μg of nuclear extract (NE) isolated from Be(2)C cells as previously described ([Pan, 2002](#B36)) or HIS-tagged hnRNPH with or without unlabeled RNA oligo in binding buffer [20 mm HEPES-KOH, pH 7.9, 100 mm KCl, 3.2 mm MgOAc, 20 mm creatine phosphate, 1.3 mm ATP, 12% glycerol, 1 mm DTT, amd 10 μg of poly(rA)] at 30°C for 18 min. For supershifting assay, an hnRNPH antibody (a gift from Dr. Douglas Black, University of California, Los Angeles) or a rabbit IgG control (Santa Cruz Biotechnology) was first incubated with nuclear extract at 30°C for 8 min before adding the probe. The reaction mixture was then separated on a 4% acrylamide gel in 0.5 × TBE buffer (45 mm Tris base, 45 mm boric acid, 1 mm EDTA) containing 2.5% glycerol. The gel was exposed on film (BioMax MS, Kodak) at −80°C. For competition assay, ∼86 fmol ^32^P-labeled G31 was incubated with 10 μg of Be(2)C NE in the absence or presence of unlabeled RNA oligos at various concentrations from 4.4 to 709 nm. Bands on film were imaged with ChemiDoc MP System (Bio-Rad) and quantified with ImageLab 4.0 (Bio-Rad). The data were analyzed by nonlinear regression (Prism 5.0) to obtain IC_50_ values.

A 26 mer RNA oligo with the G allele or the A allele of rs9479757 was covalently linked with biotin-triethylene glycol (a triethylene glycol spacer) at the 5′ site during synthesis (Integrated DNA Technology). The 500 pmol biotinylated RNA oligo were bound to 250 μl of streptavidin-coupled magnetic beads (M280-streptavidin, Life Technologies) in Binding & Washing (B&W) buffer (5 mm Tris/HCl, pH 7.5, 0.5 mm EDTA, and 1 m NaCl) following the manufacturer's protocol. RNA oligo-bound beads were incubated with 2.6 mg of NE from Be(2)C cells in binding buffer (see EMSA) at 30°C for 26 min and then at 4°C for 30 min. The mixture was transferred into one well in a six-well plate, and was UV cross-linked at 254 nm for 18 min in Stratagene UV Stratalinker 2400 (Agilent). After washing four times with B&W buffer containing 0.3% SDS, the mixture was digested with 25 μg of RNase A (Affymetrix) and 400 U Ambion RNase I (Life Technologies) at 37°C for 30 min. The digested mixture was separated in 4–20% gradient SDS-PAGE and transferred onto PVDF membranes for Western blot analysis.

Samples from the UV cross-linking study and proteins from Be(2)C cells and the PFC tissues were separated in 4–20% gradient SDS-PAGE and transferred onto PVDF membranes. The PVDF membranes were blocked in a blocking solution containing TBST buffer (10 mm Tris-HCl, pH 7.4, 150 mm NaCl, and 0.05% Tween 20), 4% nonfat dried milk, and 1% BSA at room temperature for 1 h, and incubated with an anti-hnRNPH antibody (1:1000 dilution), anti-hnRNPA1 antibody (1:200 dilution), anti-hnRNPA2/B1 antibody (1:200 dilution), anti-hnRNPU antibody (1:200 dilution), or anti-hnRNPC antibody (1:200 dilution) and anti-hnRNPF antibody (1:1000 dilution), anti-hMOR-1 exon 4 antibody (1:1000), anti-actin antibody (1:3000), and anti-GAPDH antibody (1:3000) in the blocking solution at 4°C overnight. After washing with TBST buffer, the membrane was incubated with appropriate peroxidase-conjugated secondary antibodies in TBST buffer at room temperature for 1 h. After washing with TBST buffer, signals were detected by using ChemiGrow reagents (ProteinSimple), imaged with ChemiDoc MP, and quantified using ImageLab 4.0.

A human BAC clone (CH17-143P18) covering an ∼207 kb *OPRM1* region from 23 kb upstream of exon 1 to 95 kb downstream of exon 4 was obtained from BACPAC Resources, and used as a template for making BAC minigene constructs with Recombineering System obtained from the National Cancer Institute Biological Resources Branch ([Lee et al., 2001](#B27)) and a modified positive/negative (Rpsl-kanmycin) selection system ([Wang et al., 2009](#B52)) following the described procedures. Briefly, CH17-143P18 was first transformed into the SW105 strain by electroporation. All recombineering procedures were performed in the SW105 transformats. The original CH17-143P18 clone contained the G allele at positions 24, 25, 29, 31, and 32 of intron 2. To make BAC clones with the A allele at these positions, we inserted the rpsl-kana cassette at these desired positions through a positive kanamycin selection. The rpsl-kana cassette flanking ∼50 bp of the designated sequences for homologous recombination was generated by PCR, and electroporated into the competent SW105-CH17-143P18 cells in which a recombinase was induced by incubating the cells at 42°C for 15 min. Positive clones were selected on LB plates containing 20 μg/ml kanamycin and 12 μg/ml chloramphenicol, and verified by PCR with appropriate primers. In a negative selection, a fragment containing the A allele was electroporated into induced positive cells with the rpsl-kana cassette and selected on LB plates containing 12 μg/ml chloramphenicol and 1 mg/ml streptomycin for replacing the rpsl-kana cassette. The final BAC constructs with a single A mutation were confirmed by DNA sequencing. All electroporations were performed using a 0.1 cm cuvette and Gene Pulser II (1.75 kV, 25 μF, and 200 Ω; Bio-Rad). The integrity and accuracy of the BAC constructs in each step were verified by digestion patterns with several restriction enzymes. We inserted a tdTomato/pgk-neo/CMV cassette upstream of the exon 1 using positive selection. tdTomato (Clontech) was used for monitoring transfection efficiency, and pgk-neo was used for selection. The CMV promoter from pcDNA3 (Life Technologies) is used for the following two purposes: (1) to drive expression of *OPRM1* variant mRNAs since the exon 1 promoter activity in the BAC constructs was very low; and (2) to provide unique sequences for the design of a sense primer in PCR to detect expressions of the *OPRM1* variants only from the BAC constructs so that expressions of the endogenously expressed *OPRM1* variants in Be(2)C cells can be excluded.

Be(2)C cells were maintained in MEM plus NEAA:F12 medium containing 10% fetal calf serum at 37°C in a 5% CO_2_/95% air-humidified atmosphere. BAC DNA transfection was optimized using NeuroMag reagent (OZ Biosciences). Briefly, 4 μg of BAC DNA purified by Maxiprep (Qiagen) was incubated with 14 μl of NeuroMag reagent at room temperature for 20 min. The mixture was then added into cells (on a 12-well plate) for incubating at 37°C for 20 min under a magnetic plate. We usually obtain ∼50–70% transfection efficiency determined by tdTomato fluorescence under microscopy. Cells were harvested after 48 h of transfection for RNA and protein extraction.

Total RNA was extracted from cells and brain tissues using miRNeasy Kit (Qiagen) according to the manufacturer's protocol. Five micrograms of total RNA were treated with Turbo DNase I (Life Technologies) and reverse transcribed with 0.8 μg of random hexamer (GE Life Sciences) and 200 U of Superscript III reverse transcriptase (Life Technologies) according to the manufacturer's protocols. The first-strand cDNA was then used as a template in regular PCR or quantitative PCR (qPCR). Regular PCR was performed using Platinum TaqDNA polymerase (Life Technologies) to amplify exogenous (transfected BAC minigene constructs) hMOR-1, hMOR-1S, and hMOR-1Z with SP-1 (5′-GGC TAC CCA TAC GAC GTG CCA G-3′) and AP-1 (5′-GCT TGG TGA AGG TCG GAA TGG CAT G-3′) primers, and endogenous hMOR-1, hMOR-1S, and hMOR-1Z with SP-2 (5′-GGT GCT CCT GGC TAC CTC GCA C-3′) and AP-1 primers, as well as human glyceraldehyde 3-phosphate dehydrogenase (hG3PDH) with SP-3 (5′-ACC ACA GTC CAT GCC ATCC AC-3′) and AP-2 (5′-TCC ACC ACC CTG TTG CTG TA-3′) primers for loading control. PCR products were analyzed on 1.5% agarose gel, stained with ethidium bromide, and imaged using the ChemiDoc MP system. qPCR was performed using HotStart SYBR Green Master Mix (Affymetrix) as previously described ([Xu et al., 2013](#B54)). A sense primer, SP-1, and three antisense primers, AP-3 (5′-CTT CAT CTT GGT GTA TCT GAC AAT CAC ATA C-3′), AP-4 (5′-CTT CCA GAT TTT CTA GCT GAC AAT CAC ATA C-3′), and AP-5 (5′-GTG TAC AAT CTA TGG AAC CTG ACA ATC ACA TAC-3′), were used for amplifying exogenous hE1–2 (exons 1–2 for hMOR-1), hMOR-1S, and hMOR-1Z, respectively. The SP-2 and AP-3, AP-4, and AP-5 primers were used for amplifying endogenous hE1–2, hMOR-1S, and hMOR-1Z, respectively. qPCRs for three reference genes including human succinate dehydrogenase subunit A (hSDHA) using SP-4 (5′-TGC CTG TGA CAA CGT AGA GC-3′) and AP-6 (5′-CCG CAG GAT AGT TAG GCT GAA-3′), human β-2 microglobulin (hB2M) using SP-5 (5′-ACA GCA GTG TCA ACG TAG TAG T-3′), and AP-7 (5′-CGG CAG GCA TAC TCA TCT TTT T-3′) and hG3PDH using SP-3 and AP-2 were used for determining normalization factor (NF) using the following formula, which was described previously ([Pfaffl et al., 2004](#B42)): NF = (C(t)_hSDHA_ × C(t)_hB2M ×_ C(t)_hG3PDH_)^1/3^. All variant C(t) values were normalized with the NFs to obtain ΔC(t): (ΔC(t) = C(t)_variant_ − NF). Fold change was calculated through the 2^−ΔΔC(t)^ format, where ΔΔC(t) value was the difference in the ΔC(t) values between a variant and a reference.

Different concentrations of a Silencer Select siRNA targeting hnRNPH1 (si-H1; 5′-GAA GCA UAC UGG UCC AAA Utt-3′; Ambion, Life Technologies) were transfected, either alone or combined with a BAC minigene construct, into Be(2)C cells in a 24-well plate using NeuroMag reagent (see above). A Ambion Silencer Select negative control siRNA (Life Technologies) was used as a control (si-Ctr). After 48 h of transfection, cells were harvested for protein and RNA extraction. Proteins from whole cells were extracted by directly adding SDS-PAGE loading buffer into wells containing the cells. After heating at 100°C for 10 min and centrifugation at 15,000 × *g* for 10 min, the supernatants were separated on 4–20% SDS-PAGE gels and transferred onto PVDF membranes for Western blot analysis. For antisense vivo-morpholino studies, Be(2)C cells were treated with 1 μm 25mer antisense vivo-morpholino oligo targeting the region containing rs9479757 (antisense: 5′-CAG GCT GTG AAC CCT CCT TCC CTC A-3′) in culture medium for 48 h. A 25mer control vivo-morpholino oligo [control: 5′-CAG *C*CT *C*TG AA*G* CCT C*G*T T*G*C CTC A-3′, in which five bases (shown in italics) in the antisense oligo were switched] was used as a control. RNA from siRNA- and vivo-morpholino-treated cells was extracted as described above.

Membrane were isolated from Be(2)C cells and [^3^H]DAMGO binding as previously described ([Pan et al., 1999](#B38), [2005](#B35)). [^3^H]DAMGO binding was performed at 25°C for 60 min in 50 mm potassium phosphate buffer, pH 7.4, containing 5 mm magnesium sulfate. Specific binding was determined as the difference between total binding and nonspecific binding, defined by levallorphan (1 μm). Protein concentration was determined by Lowry method using BSA as the standard.

[^35^S]GTPγS binding assay was performed as described previously ([Bolan et al., 2004](#B5); [Pasternak et al., 2004](#B39)). Briefly, membranes prepared from Be(2)C cells were incubated in the presence and absence of 1 μm DAMGO for 60 min at 30°C in the assay buffer (50 mm Tris-HCl, pH 7.7, 3 mm MgCl_2_, 0.2 mm EGTA, 10 mm NaCl) containing 0.05 nm [^35^S]GTPγS (PerkinElmer) and 60 μm GDP. The percentage of stimulation was determined by dividing the counts from membrane treated with the DAMGO by those from membrane that had not been treated and multiplying 100.

Five hundred picomoles biotinylated RNA oligos (the same as used in the UV cross-linking studies) were bound to 75 μl of Monomeric Avidin UltraLink Resin (Pierce) following the manufacturer's procedures. The RNA oligo-bound resin was then incubated with 1.8 mg of nuclear extract from Be(2)C cells that was precleared with resin in binding buffer (see EMSA) at 30°C for 26 min and then at 4°C for 30 min. After washing two times with binding buffer, bound proteins were eluted with elution buffer (0.1 m sodium phosphate, 0.15 m NaCl, pH 7.0, 2 mm biotin). The elute was concentrated using an Amicon Ultra concentrator (0.5 ml, 3 kilodaltons; Millipore) and was loaded onto a 10% SDS-PAGE gel and electrophoresed into a 4 mm separation gel. After staining with Coomassie blue, the protein-dense band was cut and digested with trypsin, and was batch purified on reversed-phase micro-tips, and the resulting peptide mixtures were then analyzed by LC-Packings/Dionex Ultimate capillary-HPLC coupled to an Applied Biosystems/MDS Sciex Q-STAR tandem mass spectrometry (MS/MS) system. MS/MS data were then taken for searching a “nonredundant” protein database (3,245,378 entries; National Center for Biotechnology Information) using the MASCOT MS/MS ion search program ([Perkins et al., 1999](#B41)). Scaffold 4.0.5 (Proteome Software) was used to further validate and cross-tabulate the MS/MS-based peptide and protein identifications. Protein and peptide probability was set at 95% with a minimum peptide requirement of 1. Protein–protein interactions among identified proteins were obtained from Ingenuity databases (Ingenuity Systems).

Pearson χ^2^ test was performed by using chisq.test, R version 3.0.2 for SNP analysis. The linkage disequilibrium (LD) plot and the haplotype analysis were performed by using Haploview version 4.2. The test of association independent of rs9479757 was performed by using Plink version 1.07. Two-tailed Student's *t* tests (Prism 5.0) were used in analyzing data from Western blots of the UV cross-linking study, the morpholino oligo study in Be(2) cells and human postmortem PFC samples, and qPCRs from the hnRNPH1 siRNA study, the BAC minigene study in Be(2)C cells treated with morpholino oligos and human postmortem PFC samples, as well as [^3^H]DAMGO and [^35^S]GTPγS binding assays in the morpholino oligo study. One-way ANOVA followed by *post hoc* Newman–Keuls analyses (Prism 5.0) was used in analyzing data from the competing EMSA study, qPCRs of mutated BAC minigene constructs, and Western blots from the hnRNPH1 siRNA study. Data are represented as the mean ± SEM from at least three independent experiments. Statistical significance was set at *p* < 0.05.

## Results

### rs9479757 is associated with heroin addiction severity in Han-Chinese male heroin addicts

To investigate whether rs9479757 is associated with heroin addiction, we genotyped 190 normal male healthy subjects (control) and 332 male heroin addicts (case) that were recruited based on DSM-IV criteria. There were a few subjects in the case group who had the AA genotype, and no subjects in the control group with the AA genotype, which is consistent with the low frequency of AA homozygotes in the Asian population (0.8%, from the dbSNP database). The frequencies of AG and GG genotypes in both control and case groups were in Hardy–Weinberg equilibrium. There were no significant differences in either the genotypic or allelic distribution between control and case groups (for genotype: χ^2^ = 1.38, *p* = 0.501; for allele: χ^2^ = 0.65, *p* = 0.420; [Table 1](#T1)). We then analyzed the association of the genotype and allele frequencies with the severity of heroin addiction, as defined by our standard criteria including route, duration, and self-administrated dosage among the 332 male heroin addicts ([Xie et al., 2013](#B53)). We found that both genotypic and allelic frequencies showed significant differences among the three heroin addiction groups (for genotype: χ^2^ = 13.79, *p* = 0.0080; for allele: χ^2^ = 10.47, *p* = 0.0053; [Table 2](#T2)). The AG genotype and the A allele were clearly associated with the mild group, whereas the GG genotype and the G allele were predominately associated with the moderate and severe groups. These results suggest that rs9479757 is associated with the severity of heroin addiction.

### Table 1.

| SNP | Control (n = 190) | Case (n = 332) | χ2 | p value |
| --- | --- | --- | --- | --- |
| Genotype |   |   |   |   |
| AA | 0 (0)* | 2 (0.6) | 1.38 | 0.501 |
| AG | 22 (11.6) | 43 (13.0) |   |   |
| GG | 168 (88.4) | 287 (86.4) |   |   |
| Allele |   |   |   |   |
| A | 22 (5.8) | 47 (7.1) | 0.65 | 0.420 |
| G | 358 (94.2) | 617 (92.9) |   |   |

Table 1 Caption: Distribution of rs9479757 in normal subjects (control) and heroin addicts (case) groups

### Table 2.

| SNP | Mild (n = 21) | Moderate (n = 162) | Severe (n = 149) | χ2 | p value |
| --- | --- | --- | --- | --- | --- |
| Genotype |   |   |   |   |   |
| AA | 1 (4.8)* | 0 (0.0) | 1 (0.7) | 13.79 | 0.0080 |
| AG | 6 (28.6) | 23 (14.2) | 14 (9.4) |   |   |
| GG | 14 (66.7) | 139 (85.8) | 134 (89.9) |   |   |
| Allele |   |   |   |   |   |
| A | 8 (19.0) | 23 (7.1) | 16 (5.4) | 10.47 | 0.0053 |
| G | 34 (81.0) | 301 (92.9) | 282 (94.6) |   |   |

Table 2 Caption: Association of rs9479757 (IVS2 + 31A/G) with heroin addiction severity in Han-Chinese male heroin addicts

### Haplotype analysis confirms association of rs9479757 with heroin addiction severity

To further investigate this genetic association at the *OPRM1* locus, we genotyped an additional 22 tag SNPs spanning an ∼105 kb *OPRM1* region among 332 heroin addicts for haplotype analysis. We observed a large amount of LD among many of the SNPs at this region ([Fig. 2](#F2)). We next asked whether the genotype frequency or allele frequency for each of the SNPs is associated with the three categorical phenotypes of heroin addiction severity. Using a Pearson χ^2^ test on 3 × 3 or 2 × 3 contingency tables, we observed nominally significant associations with three additional SNPs, rs527434 (*p* = 0.0109 for genotype), rs632499 (*p* = 0.0086 for allele; *p* = 0.0492 for genotype), and rs13195018 (*p* = 0.0365 for allele; *p* = 0.0320 for genotype), besides rs9479757 ([Table 3](#T3)). The observation of additional associations within this haplotype block supports the hypothesis that this genetic variation at *OPRM1* is truly associated with heroin addiction severity as opposed to the possibility that the observed association is artifactual due to genotyping errors. Furthermore, the most significant association was with rs9479757 (*p* = 0.0053 for allelic test and 0.0080 for genotype test; [Table 3](#T3)). As these SNPs are in LD with each other, we further asked whether the haplotype formed by these SNPs is significantly associated with the mild group versus the severe group. In this comparison, the AGCA haplotype was most strongly associated (*p* = 0.0088; [Table 4](#T4)), but this association was weaker than that observed for rs9479757 in the dichotomous test (mild group vs severe group, *p* = 0.001) using Haploview software. We also tested each of these three SNPs, rs527434, rs632499, and rs13195018, for an effect on heroin addiction severity (mild group vs severe group) independent of rs9479757. None of the observed effects was independent ([Table 5](#T5)). In other words, the association signal observed for each of the three other SNPs can be fully explained by their correlation with rs9479757. Together, these data are consistent with the hypothesis that SNP rs9479757 is functionally responsible for the observed association with the severity of heroin addiction.

### Figure 2.

![Figure 2.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c31c/4131016/af08ed89c879/zns9991459970002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4131016_zns9991459970002.jpg)

Linkage disequilibrium (LD) plot of the OPRM1 23 SNPs in heroin addicts. LD was plotted using Haploview version 4.2. The D′ value of each SNP pair is shown in the square. The numbers in the squares are D′ × 100. The D′ value with 1 is indicated in empty squares. Relative locations of exons (boxes) and introns (lines) are shown on the top.

### Table 3.

| SNP | Allele/genotype | Mild (n = 21) | Moderate (n = 162) | Severe (n = 149) | p value |
| --- | --- | --- | --- | --- | --- |
| rs1799971 | A | 0.738 | 0.664 | 0.658 | 0.5825 |
|   | G | 0.262 | 0.336 | 0.342 |   |
|   | AA | 0.524 | 0.426 | 0.396 | 0.7326 |
|   | AG | 0.429 | 0.475 | 0.523 |   |
|   | GG | 0.048 | 0.099 | 0.081 |   |
| rs3798676 | C | 0.929 | 0.935 | 0.902 | 0.3081 |
|   | T | 0.071 | 0.065 | 0.098 |   |
|   | CC | 0.905 | 0.870 | 0.818 | 0.0984 |
|   | CT | 0.048 | 0.130 | 0.169 |   |
|   | TT | 0.048 | 0.000 | 0.014 |   |
| rs4870266 | A | 0.167 | 0.191 | 0.178 | 0.8711 |
|   | G | 0.833 | 0.809 | 0.822 |   |
|   | AA | 0.000 | 0.025 | 0.034 | 0.8247 |
|   | AG | 0.333 | 0.333 | 0.289 |   |
|   | GG | 0.667 | 0.642 | 0.678 |   |
| rs527434 | A | 0.738 | 0.873 | 0.852 | 0.0624 |
|   | G | 0.262 | 0.127 | 0.148 |   |
|   | AA | 0.619 | 0.753 | 0.752 | 0.0109 |
|   | AG | 0.238 | 0.241 | 0.201 |   |
|   | GG | 0.143 | 0.006 | 0.047 |   |
| rs610231 | C | 0.119 | 0.065 | 0.044 | 0.1218 |
|   | T | 0.881 | 0.935 | 0.956 |   |
|   | CC | 0.000 | 0.012 | 0.000 | 0.1613 |
|   | CT | 0.238 | 0.105 | 0.087 |   |
|   | TT | 0.762 | 0.883 | 0.913 |   |
| rs9371773 | A | 0.024 | 0.083 | 0.044 | 0.0693 |
|   | G | 0.976 | 0.917 | 0.956 |   |
|   | AA | 0.000 | 0.000 | 0.007 | 0.0706 |
|   | AG | 0.048 | 0.167 | 0.074 |   |
|   | GG | 0.952 | 0.833 | 0.919 |   |
| rs9384174 | C | 0.667 | 0.598 | 0.602 | 0.6907 |
|   | T | 0.333 | 0.402 | 0.398 |   |
|   | CC | 0.429 | 0.323 | 0.388 | 0.2290 |
|   | CT | 0.476 | 0.551 | 0.429 |   |
|   | TT | 0.095 | 0.127 | 0.184 |   |
| rs9479757 | A | 0.190 | 0.071 | 0.054 | 0.0053 |
|   | G | 0.810 | 0.929 | 0.946 |   |
|   | AA | 0.048 | 0.000 | 0.007 | 0.0080 |
|   | AG | 0.286 | 0.142 | 0.094 |   |
|   | GG | 0.667 | 0.858 | 0.899 |   |
| rs2075572 | C | 0.643 | 0.787 | 0.789 | 0.0909 |
|   | G | 0.357 | 0.213 | 0.211 |   |
|   | CC | 0.429 | 0.623 | 0.658 | 0.1575 |
|   | CG | 0.429 | 0.327 | 0.262 |   |
|   | GG | 0.143 | 0.049 | 0.081 |   |
| rs3798683 | A | 0.310 | 0.342 | 0.326 | 0.8667 |
|   | G | 0.690 | 0.658 | 0.674 |   |
|   | AA | 0.048 | 0.106 | 0.087 | 0.921 |
|   | AG | 0.524 | 0.472 | 0.477 |   |
|   | GG | 0.429 | 0.422 | 0.436 |   |
| rs632499 | A | 0.262 | 0.136 | 0.097 | 0.0086 |
|   | C | 0.738 | 0.864 | 0.903 |   |
|   | AA | 0.048 | 0.025 | 0.013 | 0.0492 |
|   | AC | 0.429 | 0.222 | 0.168 |   |
|   | CC | 0.524 | 0.753 | 0.819 |   |
| rs9397685 | A | 0.786 | 0.799 | 0.768 | 0.6444 |
|   | G | 0.214 | 0.201 | 0.232 |   |
|   | AA | 0.571 | 0.642 | 0.564 | 0.3605 |
|   | AG | 0.429 | 0.315 | 0.409 |   |
|   | GG | 0.000 | 0.043 | 0.027 |   |
| rs3798688 | G | 0.929 | 0.917 | 0.886 | 0.3666 |
|   | T | 0.071 | 0.083 | 0.114 |   |
|   | GG | 0.905 | 0.852 | 0.799 | 0.4501 |
|   | GT | 0.048 | 0.130 | 0.174 |   |
|   | TT | 0.048 | 0.019 | 0.027 |   |
| rs9371776 | A | 0.024 | 0.049 | 0.074 | 0.2636 |
|   | G | 0.976 | 0.951 | 0.926 |   |
|   | AA | 0.000 | 0.006 | 0.000 | 0.3021 |
|   | AG | 0.048 | 0.086 | 0.148 |   |
|   | GG | 0.952 | 0.907 | 0.852 |   |
| rs2010884 | A | 0.286 | 0.392 | 0.403 | 0.3453 |
|   | G | 0.714 | 0.608 | 0.597 |   |
|   | AA | 0.095 | 0.142 | 0.201 | 0.2318 |
|   | AG | 0.381 | 0.500 | 0.403 |   |
|   | GG | 0.524 | 0.358 | 0.396 |   |
| rs13195018 | A | 0.810 | 0.920 | 0.926 | 0.0365 |
|   | C | 0.190 | 0.080 | 0.074 |   |
|   | AA | 0.667 | 0.840 | 0.859 | 0.0320 |
|   | AC | 0.286 | 0.160 | 0.134 |   |
|   | CC | 0.048 | 0.000 | 0.007 |   |
| rs9397687 | C | 0.690 | 0.568 | 0.557 | 0.2597 |
|   | T | 0.310 | 0.432 | 0.443 |   |
|   | CC | 0.476 | 0.315 | 0.342 | 0.3288 |
|   | CT | 0.429 | 0.506 | 0.430 |   |
|   | TT | 0.095 | 0.179 | 0.228 |   |
| rs518596 | C | 0.286 | 0.225 | 0.208 | 0.5106 |
|   | T | 0.714 | 0.775 | 0.792 |   |
|   | CC | 0.095 | 0.049 | 0.047 | 0.8155 |
|   | CT | 0.381 | 0.352 | 0.322 |   |
|   | TT | 0.524 | 0.599 | 0.631 |   |
| rs73576470 | A | 0.952 | 0.926 | 0.916 | 0.6874 |
|   | G | 0.048 | 0.074 | 0.084 |   |
|   | AA | 0.905 | 0.852 | 0.846 | 0.5746 |
|   | AG | 0.095 | 0.148 | 0.141 |   |
|   | GG | 0.000 | 0.000 | 0.013 |   |
| rs7763748 | C | 0.714 | 0.744 | 0.765 | 0.7016 |
|   | T | 0.286 | 0.256 | 0.235 |   |
|   | CC | 0.524 | 0.562 | 0.570 | 0.7116 |
|   | CT | 0.381 | 0.364 | 0.389 |   |
|   | TT | 0.095 | 0.074 | 0.040 |   |
| rs6557337 | C | 0.500 | 0.642 | 0.648 | 0.1671 |
|   | T | 0.500 | 0.358 | 0.352 |   |
|   | CC | 0.286 | 0.407 | 0.416 | 0.3056 |
|   | CT | 0.429 | 0.469 | 0.463 |   |
|   | TT | 0.286 | 0.123 | 0.121 |   |
| rs483481 | C | 0.643 | 0.636 | 0.681 | 0.4826 |
|   | T | 0.357 | 0.364 | 0.319 |   |
|   | CC | 0.381 | 0.383 | 0.456 | 0.7526 |
|   | CT | 0.524 | 0.506 | 0.450 |   |
|   | TT | 0.095 | 0.111 | 0.094 |   |
| rs569284 | G | 0.024 | 0.068 | 0.057 | 0.5031 |
|   | T | 0.976 | 0.932 | 0.943 |   |
|   | GG | 0.000 | 0.000 | 0.000 |   |
|   | GT | 0.048 | 0.136 | 0.114 | 0.4750 |
|   | TT | 0.952 | 0.864 | 0.886 |   |

Table 3 Caption: Comparison of allele and genotype frequencies of 23 OPRM1 SNPs among mild, moderate, and severe groups

### Table 4.

| Haplotype* | Mild (n = 21) | Severe (n = 149) | p value |
| --- | --- | --- | --- |
| AGCA | 0.618 | 0.798 | 0.0088 |
| GGCA | 0.095 | 0.094 | 0.9800 |
| AGAC | 0.095 | 0.037 | 0.8510 |
| GAAC | 0.071 | 0.030 | 0.1758 |
| GAAA | 0.071 | 0.020 | 0.0528 |

Table 4 Caption: Haplotype analysis of four SNPs (rs527434, rs9479757, rs632499, and rs13195018) between mild and severe groups

### Table 5.

| Test SNP | −2 Log likelihood |  | Likelihood ratio test |  |  |
| --- | --- | --- | --- | --- | --- |
| rs9479757 alone | rs9479757 along with test SNP | χ2 | df | p value |  |
| rs527434 | 120 | 120 | 5.6e-06 | 1 | 0.998 |
| rs632499 | 120 | 118 | 2.39 | 1 | 0.122 |
| rs13195018 | 120 | 116 | 4.01 | 2 | 0.134 |

Table 5 Caption: Test of association independent of rs9479757

### rs9479757 alters hnRNPH binding in vitro

Generally, the *cis-*acting elements controlling splicing are located in exons and their adjacent intron regions, although distal intronic *cis-*elements can also be important. The location of SNP rs949757, 31 bases downstream of exon 2, makes the SNP a favorable candidate for modulating regulatory *cis-*elements and thus influencing exon 2 splicing. A bioinformatics analysis of available databases for RNA binding protein motifs [Alternative Splicing Database (previously called Alternative Splicing Rainbow); [Thanaraj et al., 2004](#B48)] revealed that the G allele of this SNP corresponds to a high score as a hnRNPH binding site, which has a highly conserved AGGG consensus core sequence ([Fig. 3](#F3)*A*; [Caputi and Zahler, 2001](#B7)). The G-to-A transition diminished this score, suggesting that SNP rs949757 can alter hnRNPH binding. To test this hypothesis, we performed RNA EMSA with an RNA oligo containing the G allele of the SNP (G31) and nuclear extract from Be(2)C cells, a human neuroblastoma cell line that endogenously expresses the *OPRM1* gene. We observed the formation of a major RNA/protein complex, which was completely abolished by competition with a 50-fold excess of an unlabeled RNA oligo ([Fig. 3](#F3)*B*). To investigate whether hnRNPH participated in this RNA/protein complex, we used an hnRNPH antibody in EMSA supershift experiments. The RNA/protein complex was in fact supershifted by the hnRNPH antibody, but not by a control antibody ([Fig. 3](#F3)*B*), indicating that hnRNPH is involved in the formation of this complex.

### Figure 3.

![Figure 3.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c31c/4131016/4ee35b287518/zns9991459970003.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4131016_zns9991459970003.jpg)

RNA EMSA and UV cross-linking study. A, Comparison of 5′ intron 2 containing SNP rs9479757 among five species. Exon and intron are indicated by uppercase and lowercase letters, respectively. rs9479757 is shown in red letters. Sequences of RNA oligos used in EMSA are underlined. Potential hnRNPH binding site is shown by a blue line. Extra sequences of mouse and rat are shown in the lower box inserted at relative positions by the vertical lines. B, RNA EMSA. EMSA with 32P-labeled RNA oligo containing G allele and Be(2)C NE was performed as described in Materials and Methods. Probe, shifted, and supershifted complex bands are indicated by arrows. Competitor (×50) 220 nm unlabeled RNA oligo; H1 antibody was a gift from Dr. Douglas Black (University of California, Los Angeles); control IgG was from Santa Cruz Biotechnology. C, RNA EMSA with purified (HIS)6-tagged hnRNPH1 (H1-HIS). (HIS)6-tagged hnRNPH1 was synthesized and purified as described in Materials and Methods. Fifteen nanograms (1) or 30 ng (2) of (HIS)6-tagged hnRNPH1 were used in EMSA with 32P-labeled RNA oligo containing the G allele. D, Western blot analysis of UV cross-linking products. Biotinylated G31 or A31 oligo was bound onto strepavidin-magnetic bead, and incubated with Be(2)C nuclear extract. After UV cross-linking, stringent wash, and digestion with RNase A/I, the samples were used in Western blot analysis. Immunoblots with indicated antibodies are shown in the left panel. NE of Be(2)C cells as input controls; G31 and A31, RNA oligos used in UV cross-linking. Quantification of the band intensities on blots measured using ImageLab software (Bio-Rad) is shown in the right panel. All bars are the mean ± SEM. Student's t test was used (n = 3 each). *p < 0.05; **p < 0.01; ***p < 0.001.

Several other hnRNP proteins such as hnRNPF, hnRNPA1, hnRNPA2/B1, and hnRNPU, as well as SRSF1, can also bind to G triplets or G-rich sequences, and these proteins often interact with each other. The supershifting observed in the EMSA by the hnRNPH antibody only shows that the protein is part of the complex, but does not prove a direct interaction of hnRNPH with the RNA oligo since other polyG-binding proteins could be mediating this interaction. To examine which of these proteins in the nuclear extract from Be(2)C cells is the major binding partner to the RNA oligos containing the SNP, we used UV cross-linking coupled with RNA purification and Western blot analysis, which is a common approach for studying RNA–protein interaction. When the biotinylated G31 oligo was used, it was clear that hnRNPH was efficiently cross-linked, while hnRNPF was not ([Fig. 3](#F3)*D*), probably due to its low level in the nuclear extract, as suggested by its absence from the results of the RNA affinity purification coupled to liquid chromatography (LC)-MS/MS experiments, which used the same RNA oligos (see below). Other hnRNP proteins, including hnRNPA1, hnRNPA2/B1, and hnRNPU, as well as SRSF1, also failed to be cross-linked ([Fig. 3](#F3)*D*). These results suggest that hnRNPH is indeed the major binding partner for the G31 oligo in the context of the nuclear extract. On the other hand, when the biotinylated A31 oligo was used, the UV cross-linked hnRNPH was greatly diminished (*t*_(4)_ = 4.26; *p* < 0.05; [Fig. 3](#F3)*D*), suggesting that the G-to-A transition significantly reduced hnRNPH binding. Similarly, the UV cross-linked hnRNPF was also reduced (*t*_(4)_ = 5.04; *p* < 0.01; [Fig. 3](#F3)*D*). A weak UV cross-linking of hnRNPA2/B1 with the A31 oligo, but not the G31 oligo, was detected (*t*_(4)_ = 9.59; *p* < 0.001; [Fig. 3](#F3)*D*).

To further confirm that hnRNPH can directly bind to the sequence with the G allele (G31), we purified a (His)_6_-tagged hnRNPH from *in vitro* TnT. When the purified hnRNPH was used in EMSA, we observed that the purified hnRNPH bound the G31 oligo to form a single complex in a dose-dependent manner ([Fig. 3](#F3)*C*). The complex was totally competed with by a 50-fold excess of an unlabeled G31 oligo, and completely shifted to a low-molecular-weight smear by the hnRNPH antibody, suggesting that hnRNPH can directly bind to the G31 oligo. The different shifting patterns observed when using the hnRNPH antibody compared with nuclear extract from Be(2)C cells may be due to a conformational change of the hnRNPH by the antibody binding.

We next asked whether the A allele can modulate hnRNPH binding in competing EMSA assays. In these assays, RNA/hnRNPH binding was competed with a series of unlabeled RNA oligos with either a G allele or an A allele at different positions and using varied concentrations of unlabeled oligos ([Fig. 4](#F4)*A*). When the unlabeled G31 oligo was used, it competed the complex effectively with IC_50_ values of 26 ± 1 nm ([Fig. 4](#F4)*B*,*C*). However, the unlabeled RNA oligo with the A allele (A31) competed the complex poorly with IC_50_ values of 89 ± 7 nm, 3.5-fold higher than seen with the G31 oligo (*F*_(5,12)_ = 6.12; *p* = 0.0048; *post hoc* analysis of G31 vs A31, *p* < 0.05; [Fig. 4](#F4)*B*,*C*), suggesting that the G-to-A transition of the SNP significantly reduced *in vitro* binding affinity toward hnRNPH.

### Figure 4.

![Figure 4.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c31c/4131016/16df46e0f399/zns9991459970004.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4131016_zns9991459970004.jpg)

Competition RNA EMSA. A, Sequences of unlabeled RNA oligos used in competition EMSA. A letters in red indicated mutations at different positions. B, Competition EMSA. Competition EMSA was performed with 32P-labeled RNA oligo containing G allele and Be(2)C NE in the presence of unlabeled RNA oligos at various concentrations, as described in Materials and Methods. Lane 1: 0 nm; lane 2: 4.4 nm; lane 3: 13.1 nm; lane 4: 39 nm; lane 5: 118 nm; lane 6: 355 nm; lane 7: 709 nm. G31, A31, A32, A24, A25, and A29 were unlabeled RNA oligos in which G was replaced by A at the indicated positions. C, Relative quantification of the competition EMSA. Band intensities in B were quantified using ImageLab, and normalized with those in lane 1 (without competitor, 0 nm) that is always 100%. The data were analyzed using nonlinear regression (Prism 5.0) to obtain IC50 values that are the mean ± SEM. One-way ANOVA was used (n = 3 each). *p < 0.05, compared with G31 or A29.

As we have shown previously, hnRNPH binds to a core sequence containing a G triplet ([Lefave et al., 2011](#B28)). rs9479757 is located at one G triplet. There is another G triplet just five bases upstream of rs949757 ([Fig. 4](#F4)*A*). If hnRNPH is indeed involved in complex formation, mutations in the G positions of both G-triplet sequences should impair hnRNPH binding. We next used four additional mutated RNA oligos with the replacement of A in the indicated positions as A24, A25, A32, and A29 in the competition EMSA. We observed that the A32 oligo that targets the same G triplet as the A31 oligo weakly competed with the complex with IC_50_ values of 80 ± 14 nm (*post hoc* analysis of G31 vs A32, *p* < 0.05), which are similar to those of the A31 oligo ([Fig. 4](#F4)*B*,*C*), suggesting that the G triplet where the SNP is located is important for hnRNPH binding. The A24 or A25 oligo that contained a mutation in the upstream G triplets also showed poor affinities, with IC_50_ values of 107 ± 29 and 87 ± 9 nm (*post hoc* analysis of G31 vs A24 or A25, *p* < 0.05; [Fig. 4](#F4)*B*,*C*), respectively, further suggesting that the upstream triple G mutations contribute to hnRNPH binding. However, the A29 oligo with a double G mutation just 2 nt upstream of the SNP displayed a similar affinity (IC_50_, 28 ± 4) to that of the G31 oligo (*post hoc* analysis of G31 vs A29, *p* > 0.05; [Fig. 4](#F4)*B*,*C*), indicating that only the triple G mutations, and not the double G mutations, have an effect on hnRNPH binding.

### rs949757 modulates exons 2/3 skipping/inclusion of the OPRM1 gene

The position of rs949757 is favorable for its potential role in modulating exon 2 inclusion/skipping. Exon 2 inclusion in exon 1/2/3/4 splicing generates the full-length 7-TM variant hMOR-1, exon 2 skipping in exon 1/3/4 splicing produces the single TM variant hMOR-1Z, and exon 2/3 skipping in exon 1/4 splicing produces the single TM variant hMOR-1S ([Fig. 5](#F5)). Although a minigene approach with a regular plasmid is commonly used for studying alternative splicing, it is not optimal in this case since a large portion of the intron has to be excluded due to the size limitation of regular plasmid constructs. We used the BAC minigene constructs containing the complete genomic regions to overcome this problem ([Lee et al., 2001](#B27)). To investigate the effect of rs949757 on exon 2 inclusion/skipping, we first made two BAC minigene constructs, hBAC-G31 and hBAC-A31, containing identical genomic regions from exon 1 to exon 4 of the human *OPRM1* gene, with the exception of either the G or A allele at the rs949757 locus, respectively ([Fig. 5](#F5)*A*). When transfected into the Be(2)C cell, the hBAC-G31 construct expressed all three splice-variant mRNAs with the following rank of expression levels: hMOR-1 > hMOR-1S > hMOR-1Z ([Fig. 5](#F5)*B*,*C*). The lower level of hMOR-1Z was probably due to nonsense-mediated decay. The hBAC-A31 construct also produced three splice-variant mRNAs. However, the hBAC-A31 construct generated significantly higher levels of hMOR-1S (*F*_(4,19)_ = 16.42; *p* < 0.0001; *post hoc* analysis of hBAC-G31 vs hBAC-A31, *p* < 0.001) and hMOR-1Z mRNA (*F*_(4,19)_ = 8.88; *p* = 0.0003; *post hoc* analysis of hBAC-G31 vs hBAC-A31, *p* < 0.01), and lower levels of hMOR-1 mRNA (*F*_(4,19)_ = 10.91; *p* < 0.0001; *post hoc* analysis of hBAC-G31 vs hBAC-A31, *p* < 0.01), compared with the hBAC-G31 construct ([Fig. 5](#F5)*B*,*C*). The difference in splice-variant expression levels between the two constructs suggests that the A allele either facilitates exon 2 skipping or impedes exon 2 inclusion. To examine the effect of other adjacent Gs on splicing, we then made three additional minigene constructs, hBAC-A24, hBAC-A29, and hBAC-A32, mutated at positions 24, 29, and 32, respectively. Similar to the hBAC-A31, both hBAC-A24 and hBAC-A32 increased exon 2 or exon 2/3 skipping to yield higher levels of hMOR-1S (*post hoc* analysis of hBAC-G31 vs hBAC-A24 or hBAC-A32, *p* < 0.001) and hMOR-1Z mRNA (*post hoc* analysis of hBAC-G31 vs hBAC-A24 or hBAC-A32, *p* < 0.01), and lower levels of hMOR-1 mRNA (*post hoc* analysis of hBAC-G31 vs hBAC-A24 and hBAC-A32, *p* < 0.01 and *p* < 0.001, respectively; [Fig. 5](#F5)*B*,*C*). These findings suggest that both G triplets are necessary for maintaining exon 2 inclusion, and alteration of either one of the G triplets will decrease exon 2 inclusion, most likely by weakening hnRNPH binding. On the other hand, the hBAC-A29 construct showed expression patterns similar to the hBAC-G31 construct (*post hoc* analysis of hBAC-G31 vs hBAC-A29, *p* > 0.05; [Fig. 5](#F5)*B*,*C*), which confirmed the *in vitro* data that the double G mutations were not involved in exon 2 skipping or inclusion. Together, these data suggest that the SNP-contained G-rich region plays an important role in modulating exon 2 inclusion during splicing, likely through its interaction with hnRNPH.

### Figure 5.

![Figure 5.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c31c/4131016/5629c3da3ae6/zns9991459970005.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4131016_zns9991459970005.jpg)

Expression of BAC minigene constructs in Be(2)C cells. A, Schematic of BAC minigene constructs. BAC minigene constructs were made as described in Materials and Methods. A single G-to-A mutation in individual BAC minigene constructs is labeled as a red letter in the top box. Splicing of exons 1/2/3/4 is shown by tilted blue line to generate hMOR-1. Splicing exons 1/4 or 1/3/4 is indicated by tilted red or green lines to produce hMOR-1S and hMOR-1Z, respectively. tdTom, tdTomato fluorescent reporter (ClonTech); neo, pGK-neo cassette; CVM, cytomegalovirus immediate early promoter; poly(A), polyadenylation site in exon 4; SP-1 and AP-1, PCR primers used in regular RT-PCR; pBACGK1.1, vector carrying BAC clones. B, Expression of the OPRM1 splice variants from transfected BAC minigene constructs by regular qRT-PCR. RNA was extracted from transfected Be(2)C cells with indicated BAC constructs after 48 h of transfection, and used for regular RT-PCR with SP-1 and AP-1 primers for the variants, and G3PDH primers, as described in Materials and Methods. C, Expression of the OPRM1 splice variants from transfected BAC minigene constructs by qRT-PCR. The same cDNAs in B were used in SYBR green qPCRs, as described in Materials and Methods. hE1–2 mainly represents hMOR-1. The fold change was calculated through the 2−ΔΔC(t) format, where the ΔΔC(t) value was the difference of ΔC(t) values between A mutation constructs and the G31 construct so that the expression level in the G31 construct is always 1. Bars represent the mean ± SEM. One-way ANOVA was used (n = 6 for G31, A31, and A29; n = 3 for A32 and A24). Comparison to the G31 construct: **p < 0.01; ***p < 0.001.

### Downregulation of hnRNPH facilitates exon skipping

To further investigate the role of hnRNPH in *OPRM1* alternative splicing, we used an siRNA approach to downregulate the expression of hnRNPH protein in Be(2)C cells. Compared with the si-Ctr, treatment of an si-H1 reduced hnRNPH protein levels in a dose-dependent manner in Be(2)C cells [*F*_(3,8)_ = 52.03; *p* < 0.0001; *post hoc* analysis: si-Ctr (10 nm) vs si-H1 (10 or 25 nm), *p* < 0.001; si-Ctr (25 nm) vs si-H1 (10 or 25 nm), *p* < 0.001; si-H1 (10 nm) vs si-H1 (25 nm), *p* < 0.05; [Fig. 6](#F6)*A*,*B*]. Downregulation of hnRNPH significantly enhanced the endogenous exon 2 and exons 2/3 skipping to generate higher levels of hMOR-1S and hMOR-1Z, respectively, in Be(2)C cells (for hMOR-1S: *t*_(6)_ = 3.42, *p* < 0.05; for hMOR-1Z: *t*_(4)_ = 5.83, *p* < 0.01; [Fig. 6](#F6)*C*,*D*). Although the hnRNPH siRNA had a tendency to lower expression of the endogenous hE1–2 that mainly represents hMOR-1, this effect was not statistically significant. We also examined the effect of downregulating hnRNPH on exon 2 inclusion/skipping of the transfected BAC minigene construct hBAC-G31 in Be(2)C cells. When hnRNPH was downregulated by the siRNA, we observed that the exogenous hMOR-1S and hMOR-1Z mRNA levels again increased significantly (for hMOR-1S: *t*_(16)_ = 4.78, *p* < 0.001; for hMOR-1Z: *t*_(10)_ = 3.67, *p* < 0.01), while hE1–2 mRNA levels were considerably reduced (*t*_(10)_ = 3.00, *p* < 0.05; [Fig. 6](#F6)*C*,*D*). A similar scenario was seen when the hBAC-A31 and other triple G mutation constructs were used ([Fig. 5](#F5)*B*,*C*).

### Figure 6.

![Figure 6.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c31c/4131016/187cfdef0953/zns9991459970006.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4131016_zns9991459970006.jpg)

Effect of hnRNPH1 siRNA on exon 2 inclusion/skipping. A, Effect of hnRNPH1 siRNA on the expression of hnRNPH1 protein in Be(2)C cells. An siRNA against hnRNPH1 (si-H1) and control siRNA (si-Ctr; Ambion, Life Technologies) were transfected into Be(2)C cells using NeuroMag reagent as described in Materials and Methods. Lanes 1 and 3: 10 nm siRNA; lanes 2 and 4: 25 nm siRNA. After 48 h of transfection, proteins were extracted from whole cells and used for Western blot analysis with hnRNPH1 or actin antibody, as described in Materials and Methods. B, Quantification of Western blots. Images on the films were analyzed on the ChemiDoc Image System. Band intensities of hnRNPH1 were obtained by adjusting with those of actin, and normalized with that of 10 nm si-Ctr (lane 1). All bars are the mean ± SEM. One-way ANOVA was used (n = 3 each). Red star: compared with lane 1; green star: compared with lane 2; blue star: compared with lane 3. *p < 0.05; ***p < 0.001. C, Effect of hnRNPH1 siRNA on exon 2 inclusion/skipping by regular qRT-PCR. RNA extracted from Be(2)C cells transfected with 25 nm of si-Ctr or si-H1 alone (Endogenous), or combined with BAC minigene construct G31 (hBAC-G31) was used in regular RT-PCR with SP-1 and AP-1 primers for the variants and G3PDH primers. Bands for hMOR-1, hMOR-1Z, and hMOR-1S are shown by boxes labeled as 1/2/3/4, 1/3/4, and 1/4, respectively. D, Effect of hnRNPH1 siRNA on exon 2 inclusion/skipping by qRT-PCR. The same cDNAs in C were used in SYBR green qPCR, as described in Materials and Methods. hE1–2 mainly represents hMOR-1. The fold change was calculated through 2-ΔΔC(t) format, where the ΔΔC(t) value was the difference of ΔC(t) values between the si-H1 and the si-Ctr samples, so the expression level in the si-Ctr samples is always 1. Bars represent the mean ± SEM. Student's t test was used (for endogenous variants: n = 6 each for hE1–2, n = 4 each for hMOR-1S, and n = 3 each for hMOR-1Z; for exogenous variants: n = 6 each for hE1–2, n = 9 each for hMOR-1S, and n = 6 each for hMOR-1Z). *p < 0.05; **p < 0.01; ***p < 0.001.

It was noticed that the genotype of rs9479757 in the Be(2)C cell line is AG (data not shown). Following treatment with the hnRNPH siRNA, the splicing pattern change in the Be(2)C cells with the heterozygous AG genotype was similar to the splicing pattern change in the exogenously transfected BAC minigene construct (hBAC-G31) with the homozygous GG genotype. Both the splicing changes mimicked those seen in the hBAC-A31 and the other triple G mutation constructs, all of which contain the homozygous AA genotype ([Fig. 5](#F5)*B*,*C*). These data suggest that the splicing pattern defined by the heterozygous AG genotype in the Be(2)C cells can be further modified toward that defined by the homozygous AA genotype through the hnRNPH siRNA treatment, and that hnRNPH positively regulates exon 2 inclusion via the G triplets in intron 2.

### An antisense morpholino oligo targeting rs9479757 increases exon 2 skipping

Antisense morpholino oligos are commonly used to modulate alternative splicing since they do not trigger RNase H-mediated degradation. To investigate the effect of blocking hnRNPH binding and other associated splicing factors to the SNP-contained region on exon 2 splicing, we designed an antisense morpholino oligo targeting the region ([Fig. 5](#F5)*A*). When introduced into Be(2)C cells, the antisense oligo significantly increased expression of both the endogenous and exogenous (transfected hBAC-G31) hMOR-1S (for endogenous hMOR-1S: *t*_(16)_ = 2.69, *p* < 0.05; for exogenous hMOR-1S: *t*_(14)_ = 3.11, *p* < 0.01) and hMOR-1Z mRNA (for endogenous hMOR-1Z: *t*_(16)_ = 3.02, *p* < 0.01; for exogenous hMOR-1Z: *t*_(14)_ = 2.96, *p* < 0.05), and reduced the expression of hE1–2 mRNA (for endogenous hE1–2: *t*_(16)_ = 2.61, *p* < 0.05; for exogenous hE1–2: *t*_(14)_ = 2.45, *p* < 0.05; [Fig. 7](#F7)*A–C*). These findings are similar to those from the hnRNPH siRNA studies ([Fig. 6](#F6)), suggesting that the antisense morpholino oligo facilitated exon 2 skipping by presumably blocking hnRNPH binding. Again, the splicing change in the Be(2)C cells with the AG genotype by the antisense morpholino oligo treatment mimicked that seen in the transfected BAC minigene constructs with the homozygous AA genotype.

### Figure 7.

![Figure 7.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c31c/4131016/96c1c0503e61/zns9991459970007.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4131016_zns9991459970007.jpg)

Effect of an antisense vivo-morpholino oligo targeting the G-rich region containing rs9479757 on exon 2 inclusion/skipping and hMOR-1 protein expression. A, Effect of the antisense vivo-morpholino oligo on exon 2 inclusion/skipping by regular RT-PCR. RNA extracted from Be(2)C cells treated with 1 μm antisense vivo-morpholino oligo (Ant) or control morpholino oligo (Ctr) alone, or combined with BAC minigene construct G31 (hBAC-G31) were used in regular RT-PCR with SP-1 and AP-1 primers for the variants and G3PDH primers. Bands for hMOR-1, hMOR-1Z, and hMOR-1S are shown by boxes labeled as 1/2/3/4, 1/3/4, and 1/4, respectively. B, C, Effect of the antisense vivo-morpholino oligo on endogenous exon 2 inclusion/skipping (B) or exogenous (hBAC-G31) exon 2 inclusion/skipping (C) by qRT-PCR. The same cDNAs in A were used in SYBR green qPCR, as described in Materials and Methods. hE1–2 mainly represents hMOR-1. The fold change was calculated through the 2−ΔΔC(t) format, where the ΔΔC(t) value was the difference in ΔC(t) values between the Ant and the Ctr samples so that the expression level in the Ctr samples is always 1. Bars represent the mean ± SEM. Student's t test was used (n = 9 each for endogenous variants; n = 8 each for exogenous variants). *p < 0.05; **p < 0.01. D, Western blot analysis. Proteins from Be(2)C cells treated with the Ctr or the Ant oligo were used for Western blot analysis with a hMOR-1 antibody or a GAPDH antibody. E, Quantification of Western blots. Band intensities of hMOR-1 were adjusted with those of GAPDH. Expression level of hMOR-1 in the Ant samples was normalized with that in the Ctr samples, so that the Ctr is 100%. Bars represent the mean ± SEM. Student's t test was used (n = 4 each). *p < 0.05. F, [3H]DAMGO binding. Membrane proteins from Be(2)C cells treated with the Ctr or the Ant oligo were used in binding with 1 nm [3H]DAMGO. Specific binding in the Ant samples was normalized with that in the Ctr samples, so that the Ctr is 100%. Bars represent the mean ± SEM. Student's t test was used (n = 4 each). *p < 0.05. G, [35S]GTPγS binding. Membrane proteins from Be(2)C cells treated with the Ctr oligo or the Ant oligo were used in [35S]GTPγS binding assay with 1 μm DAMGO. The percentage of the stimulation in the Ant samples was normalized with that in the Ctr samples, so that the Ctr is 100%. Bars represent the mean ± SEM. Student's t test was used (n = 6 each). *p < 0.05.

To examine whether or not there is a parallel change in hMOR-1 protein expression by the antisense morpholino oligo treatment, we performed Western blots using an hMOR-1 exon 4 antibody. We were unable to determine the expression levels of hMOR-1S and hMOR-1Z proteins due to a lack of specific antibodies. However, we observed that the antisense morpholino oligo treatment significantly increased the expression of hMOR-1 protein (*t*_(6)_ = 2.63; *p* < 0.05; [Fig. 7](#F7)*D*,*E*), contradicting its effect on hMOR-1 mRNA expression ([Fig. 7](#F7)*A–C*). This increased hMOR-1 protein expression shown by Western blots was further confirmed by [^3^H]DAMGO binding and [^35^S]GTPγS binding assays. The antisense morpholino oligo treatment led to a >30% increase of functional hMOR-1 receptor in both specific [^3^H]DAMGO binding (*t*_(6)_ = 2.46; *p* < 0.05) and the percentage of stimulation of [^35^S]GTPγS binding by DAMGO (*t*_(10)_ = 2.92; *p* < 0.05; [Fig. 7](#F7)*F*,*G*). Since single TM variants alone do not bind any opioids, the increased [^3^H]DAMGO binding and DAMGO-induced [^35^S]GTPγS binding by the antisense morpholino oligo treatment were indeed contributed by the increased hMOR-1 protein expression. Our previous studies using a Tet-Off CHO cell model indicate that single TM variants such as mMOR-1S can dimerize with the full-length 7-TM mMOR-1 in the endoplasmic reticulum (ER), leading to increased protein expression of mMOR-1 by a chaperone-like function that minimizes ER-associated degradation ([Xu et al., 2013](#B54)). *In vivo* administration of an antisense oligo targeting mMOR-1S significantly reduced expressions of mMOR-1 proteins in the spinal cord and periaqueductal gray with no effect on the expression of mMOR-1 mRNA, and consequently decreased morphine analgesia ([Xu et al., 2013](#B54)). Therefore, it is likely that the increase in hMOR-1 protein expression observed in the antisense morpholino oligo study can be explained by an increase in expression of the single TM variants hMOR-1S and hMOR-1Z, with similar mechanisms seen in the Tet-Off cells and mice. Notably, the effect of the antisense morpholino oligo on hMOR-1 protein expression was stronger than its effect on hMOR-1 mRNA expression.

### rs9479757 genotypes are associated with differential mRNA expression of the OPRM1 splice variants and hMOR-1 protein in human postmortem prefrontal cortex

We next investigated whether or not the results from our *in vitro* and cell culture studies could be recapitulated in human postmortem brain samples. Using qRT-PCR, we analyzed the expression of the *OPRM1* variant mRNAs in 19 PFC samples with the GG genotype (15 samples) and the AG genotype (12 samples). No samples with AA genotype were available for postmortem analysis. We observed that the subjects with the AG genotype expressed significantly higher levels of hMOR-1S (*t*_(25)_ = 3.39; *p* < 0.01) and hMOR-1Z (*t*_(25)_ = 2.58; *p* < 0.05) in the PFC, and lower levels of hE1–2 (mainly representing hMOR-1; *t*_(25)_ = 3.08; *p* < 0.01), compared with individuals with the GG genotype ([Fig. 8](#F8)*A*). Next, we used Western blots to examine the expression of the hMOR-1 protein in these PFC samples. The samples with the AG genotype expressed significantly higher hMOR-1 protein than the samples with the GG genotype (*t*_(13)_ = 2.29; *p* < 0.05; [Fig. 8](#F8)*B*,*C*). These results were very similar to those observed in the Be(2)C cells treated with the antisense morpholino oligo, suggesting that in the human PFC the AG genotype favors exon 2 skipping to increase single TM variant mRNA and reduce hMOR-1 mRNA expression, whereas the GG genotype favors opposite changes. The increased hMOR-1 protein expression in the samples with the AG genotype is also due to the increased single TM variants from the increased exon 2 skipping.

### Figure 8.

![Figure 8.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c31c/4131016/e044b3ff30a5/zns9991459970008.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4131016_zns9991459970008.jpg)

Expression of the OPRM1 splice variants in human postmortem PFC. A, Expression of the variant mRNAs by qRT-PCR. RNAs extracted from the PFC with the GG genotype (n = 15) or with the AG genotype (n = 12) were used for qRT-PCR, as described in Materials and Methods. hE1–2 mainly represents hMOR-1. The fold change was calculated through 2−ΔΔC(t) format, where the ΔΔC(t) value was the difference of ΔC(t) values between the PFC samples with the AG genotype and the GG genotype, so that the expression level in samples with the GG genotype is, as always, 1. Student's t test was used. *p < 0.05; **p < 0.01. B, Expression of hMOR-1 protein by Western blot. Proteins extracted from the PFC with the GG genotype (n = 10) or with the AG genotype (n = 5) were used for Western blots with a hMOR-1 exon 4 antibody or a GAPDH antibody. Left, Immunoblots with indicated antibodies. Right, Quantification of the immunoblots. Band intensities of hMOR-1 were adjusted with those of GAPDH. Expression level in the PFC with the AG genotype was normalized with that with the GG genotype. Bars represent the mean ± SEM. Student's t test was used. *p < 0.05.

### rs9479757 is associated with different sets of RNA processing proteins through hnRNPH

To further explore the roles of rs9479757 in *OPRM1* alternative splicing, we used an RNA affinity purification coupled with the LC-MS/MS approach to investigate how this polymorphism influences its association with splicing factors or *trans*-acting factors through hnRNPH. The same biotinylated RNA bait oligo containing a G or an A allele used in the UV cross-linking studies (see above) was used for purifying its associated proteins with nuclear extract from Be(2)C cells. We adapted an RNA purification strategy using monomeric avidin resin that allows gentle elution with biotin solution, thus reducing nonspecific elution. Analyzing the data from LC-MS/MS revealed that 128 nuclear proteins were associated with the G31 oligo, and 41 nuclear proteins were associated with the A31 oligo ([Table 6](#T6); [Fig. 9](#F9)*A*). Of those identified proteins, 28 proteins were associated with both RNA oligos, 100 proteins were exclusively associated with the G31 oligo, and 13 proteins were exclusively associated with the A31 oligo. Of the 28 shared proteins, a total spectrum count showed greater protein quantity associated with the G31 oligo compared with the A31 oligo ([Table 6](#T6)). These results suggest that the G31 oligo recruited many more proteins than the A31 oligo, possibly through hnRNPH interactions. This suggestion is supported by the observation that hnRNPH was only found in association with the G31 oligo, but not with the A31 oligo ([Table 6](#T6)). Furthermore, the results from the previous UV cross-linking studies identified hnRNPH as the major binding partner for the G31 oligo ([Fig. 3](#F3)*D*). Therefore, it is likely that the G-to-A transition reduces the recruitment of hnRNPH-associated proteins through weakening of hnRNPH binding.

### Table 6.

| Gene ID | Total Spe count |  | Related function |  |  |  | Gene ID | Total Spe count |  | Related function |  |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| G31 oligo | A31 oligo | Splicing | Transcription | Chr binding | RNA binding | G31 oligo | A31 oligo | Splicing | Transcription | Chr binding | RNA binding |  |  |
| ABCA12 | 2 | 0 |   |   | √ |   | H1FX | 8 | 0 |   |   | √ |   |
| ACIN1 | 0 | 1 |   |   |   | √ | HADHA | 4 | 0 |   |   |   |   |
| ARID2 | 2 | 0 |   | √ |   |   | HCFC1 | 1 | 0 | √ |   |   |   |
| BLMH | 0 | 7 |   |   |   |   | HIST1H1B | 18 | 2 |   |   | √ |   |
| BRD3 | 1 | 0 | √ | √ | √ |   | HIST1H1C | 16 | 0 |   |   | √ |   |
| BUB3 | 3 | 0 |   |   | √ |   | HIST1H1D | 16 | 3 |   |   | √ |   |
| BUD13 | 2 | 0 | √ |   |   |   | HNRNPA1 | 21 | 9 | √ |   |   |   |
| CAPRIN1 | 1 | 0 |   |   |   | √ | HNRNPA2B1 | 22 | 1 | √ |   |   |   |
| CASP14 | 2 | 0 |   |   |   |   | HNRNPA3 | 9 | 2 | √ |   |   |   |
| CBX8 | 0 | 1 |   | √ |   |   | HNRNPAB | 3 | 0 | √ |   |   |   |
| CCAR1 | 1 | 0 | √ | √ |   |   | HNRNPC | 5 | 0 | √ |   |   |   |
| CCDC124 | 1 | 0 |   |   |   |   | HNRNPH1 | 2 | 0 | √ |   |   |   |
| CCDC17 | 2 | 0 |   |   |   |   | HNRNPH3 | 1 | 0 | √ |   |   |   |
| CCNK | 2 | 0 |   | √ |   |   | HNRNPK | 6 | 3 | √ | √ | √ |   |
| CDC27 | 3 | 0 |   |   |   |   | HNRNPL | 3 | 1 | √ |   |   |   |
| CDK11A | 1 | 0 |   |   |   |   | HNRNPR | 5 | 0 | √ |   |   |   |
| CHAF1B | 2 | 0 |   | √ | √ |   | HNRNPU | 2 | 0 | √ |   |   |   |
| DDX17 | 13 | 2 | √ | √ |   |   | HP1BP3 | 6 | 0 |   |   | √ |   |
| DDX18 | 2 | 0 |   |   |   | √ | HSPA8 | 3 | 2 |   | √ |   |   |
| DDX21 | 14 | 0 |   |   |   | √ | KHDRBS1 | 6 | 0 |   | √ |   |   |
| DDX23 | 5 | 0 | √ |   |   | √ | KIF4A | 3 | 1 |   |   | √ |   |
| DDX39B | 1 | 0 | √ | √ |   |   | KPNA1 | 0 | 1 |   | √ |   |   |
| DDX47 | 2 | 0 | √ |   |   |   | LUC7L | 1 | 0 | √ |   |   |   |
| DDX5 | 6 | 2 | √ | √ |   |   | MATR3 | 3 | 0 |   |   |   | √ |
| DDX6 | 2 | 1 |   |   |   | √ | MKI67 | 7 | 0 |   |   |   |   |
| DHX9 | 11 | 0 | √ | √ |   |   | MTDH | 0 | 3 |   | √ |   |   |
| DKC1 | 2 | 4 |   |   |   |   | NAT10 | 5 | 1 |   |   | √ |   |
| DNAJC8 | 2 | 0 | √ |   |   |   | NCL | 30 | 8 |   |   |   | √ |
| EEF1A1 | 9 | 0 |   | √ |   |   | NCOA2 | 1 | 0 |   |   |   |   |
| EEF1D | 2 | 0 |   | √ |   |   | NOLC1 | 2 | 3 |   | √ |   |   |
| EFTUD2 | 1 | 0 | √ | √ |   |   | NONO | 15 | 0 |   | √ |   | √ |
| FBL | 1 | 0 |   |   |   | √ | NOP2 | 4 | 3 |   | √ |   | √ |
| G3BP2 | 2 | 0 |   |   |   | √ | NOP56 | 8 | 0 |   |   |   | √ |
| GEMIN4 | 0 | 1 | √ |   |   |   | NPM1 | 5 | 6 | √ | √ |   |   |
| GNB1 | 1 | 0 |   |   |   |   | NTHL1 | 0 | 1 |   |   |   |   |
| GTF3C1 | 1 | 0 |   | √ |   |   | NUMA1 | 3 | 0 |   |   |   |   |
| NUSAP1 | 3 | 0 |   |   | √ |   | SEPT2 | 3 | 0 |   |   |   |   |
| PA2G4 | 5 | 0 |   | √ |   |   | SEPT7 | 2 | 0 |   |   |   |   |
| PAK1IP1 | 3 | 0 |   |   |   |   | SERBP1 | 1 | 0 |   |   |   | √ |
| PARP1 | 2 | 0 |   | √ |   |   | SF3A1 | 3 | 0 | √ |   |   |   |
| PCBP1 | 2 | 0 | √ | √ |   |   | SF3B1 | 2 | 0 | √ |   | √ |   |
| PCBP2 | 3 | 0 | √ |   |   |   | SF3B2 | 3 | 0 | √ |   |   |   |
| PHB2 | 0 | 1 |   | √ |   |   | SF3B3 | 4 | 0 | √ |   |   |   |
| PLRG1 | 2 | 0 | √ | √ |   |   | SFPQ | 19 | 4 | √ | √ |   |   |
| POLDIP3 | 1 | 0 |   |   |   | √ | SMARCA5 | 4 | 0 |   | √ | √ |   |
| POLR1E | 2 | 2 |   | √ |   |   | SNRNP200 | 1 | 0 | √ |   |   |   |
| POP1 | 2 | 0 |   |   |   |   | SNRPA | 2 | 3 | √ | √ |   |   |
| PPP1CA | 3 | 0 |   |   |   |   | SNRPB | 3 | 0 |   | √ |   |   |
| PPP1CB | 4 | 3 |   |   |   |   | SPTBN1 | 1 | 0 |   |   |   |   |
| PPP1CC | 2 | 0 |   |   |   |   | SRRT | 1 | 0 |   |   |   |   |
| PRPF19 | 1 | 0 | √ |   |   |   | SRSF1 | 15 | 6 | √ | √ |   |   |
| PRPF3 | 7 | 2 | √ |   |   |   | SRSF2 | 4 | 0 | √ | √ |   |   |
| PRPH | 2 | 1 |   |   |   |   | SRSF5 | 4 | 0 | √ | √ |   |   |
| PSIP1 | 1 | 0 |   | √ | √ |   | SRSF6 | 4 | 0 | √ | √ |   |   |
| PSPC1 | 0 | 1 |   |   |   |   | SRSF7 | 4 | 0 | √ | √ |   |   |
| RBM15 | 5 | 0 |   |   |   |   | SYNPO2 | 1 | 4 |   |   |   |   |
| RBM25 | 2 | 0 | √ |   |   |   | TCEB3 | 1 | 0 |   | √ |   |   |
| RBM27 | 2 | 0 |   |   |   | √ | TCOF1 | 9 | 4 |   |   |   |   |
| RBM28 | 1 | 0 |   |   |   | √ | THRAP3 | 4 | 0 |   |   |   |   |
| RBM39 | 1 | 0 | √ | √ |   |   | TOP1 | 2 | 0 |   | √ |   |   |
| RBMX | 2 | 0 | √ | √ | √ |   | TOP2A | 1 | 0 |   |   |   |   |
| RFC2 | 1 | 0 |   |   |   |   | TOPORS | 0 | 1 |   | √ |   |   |
| RFC3 | 0 | 1 |   | √ |   |   | TOX3 | 2 | 0 |   | √ | √ |   |
| RPL31 | 3 | 0 |   |   |   | √ | WBP11 | 1 | 0 | √ |   |   |   |
| RPS13 | 2 | 0 |   |   |   | √ | WDR12 | 1 | 0 |   |   |   |   |
| RTF1 | 3 | 0 |   | √ |   |   | WHSC1 | 1 | 0 |   | √ | √ |   |
| RUVBL1 | 6 | 0 | √ | √ |   |   | YBX1 | 2 | 0 | √ | √ |   |   |
| RUVBL2 | 2 | 0 | √ | √ |   |   | YTHDC1 | 1 | 0 | √ |   |   |   |
| S100A9 | 0 | 1 |   |   |   |   | ZC3H14 | 0 | 1 |   |   |   | √ |
| SART1 | 4 | 0 | √ |   |   |   | ZC3H4 | 1 | 0 |   |   | √ |   |
| SEPT11 | 5 | 1 |   |   |   |   |   |   |   |   |   |   |   |

Table 6 Caption: List of 141 nuclear proteins identified by RNA affinity purification coupled with LC-MS/MS

### Figure 9.

![Figure 9.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c31c/4131016/5036b873873c/zns9991459970009.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4131016_zns9991459970009.jpg)

Schematic of proteins identified by RNA affinity purification-coupled LC-MS/MS and the hnRNPH-associated protein network recruited by the G-containing SNP site. A, Schematic of proteins identified by the G31 and A31 oligos. Top, Red and green circles represent proteins identified from the G31 and A31 oligos, respectively. Bottom, Three colored circles indicate proteins that have indicated functions by colored arrows from the G31 oligo (right) or from the A31 oligo (left). Overlapping area shows shared proteins. The number of unique and shared proteins is indicated within the circle and overlapping areas. B, Schematic of hnRNPH-associated protein network recruited by the G-containing SNP site. Protein–protein interactions among the purified proteins with the G31 oligo were obtained using Ingenuity systems. Selected proteins were drawn. Red bar represents G-rich ISE containing rs9479757 SNP. hnRNP proteins and SRSF proteins are shown by oval shapes with light brown and blue colors, respectively. The yellow diamond shape is RBM25, which is associated with U1 snRNP. Light green shapes are proteins linked to U2 snRNP. Light purple shapes are linked to U4/U6 or U5 snRNP. Light red and dark yellow shapes are proteins related to transcription and chromatin binding, respectively.

Of the 128 proteins associated with the G31 oligo, 51 are functionally related to splicing, and 18 other proteins possess RNA binding capabilities ([Fig. 9](#F9)*A*; [Table 6](#T6)). Particularly, several hnRNP proteins, including hnRNPA1, hnRNPA2B1, and hnRNPU, that have been shown to directly interact with hnRNPH (Ingenuity databases), were predominantly associated with the G31 oligo. Also, some proteins related to U1 small nuclear ribonucleoprotein (snRNP; i.e., RBM25), U2 snRNP (hnRNPC, RBM39, SF3B1, SF3B2, and SF3A1), U4/U6 snRNP (PRPF3, RUVBL1, RUVBL2, and SART1), and U5 snRNP (DDX23, SNRNP200, EFTUD2, and SART1), were primarily associated with the G31 oligo (Ingenuity databases). Furthermore, several SR proteins were exclusively (SRSF2, SRSF5, SRSF6, and SRSF7) or predominantly (SRSF1) identified with the G31 oligo ([Fig. 9](#F9)*A*; and [Table 6](#T6)). Interestingly, ESEfinder 3.0 ([Cartegni et al., 2003](#B9)) abundantly predicts the recognition sites for SRSF2 and SRSF6 at the end of exon 2 and the beginning of exon 3 with high scores, raising the possibility that recruiting these SR proteins would facilitate exon 2 and exon 3 inclusions (data not shown). Searching protein–protein interaction databases through Ingenuity revealed that all these G31 oligo-associated proteins directly or indirectly interact with hnRNPH to form a protein complex or network that would be favorable to intron 2 excision ([Fig. 9](#F9)*B*).

Of the 41 proteins associated with the A31 oligo, 13 proteins have functions related to splicing ([Fig. 9](#F9)*A*; [Table 6](#T6)). However, some of these proteins, particularly those associated with hnRNPH, such as hnRNPA2B1, hnRNPA3, and SFPQ, showed lower protein quantities compared with the G31 oligo ([Table 6](#T6)), which is consistent with the lack of hnRNPH associated with the A31 oligo. Although the A31 oligo exclusively associates with 13 additional proteins, few of these proteins are related to splicing. Together, these findings provide a reasonable rationale for how the G-to-A transition led to reduced exon 2 inclusion or intron 2 excision.

We also observed that among 128 proteins associated with the G31 oligo, 45 proteins are known to relate to transcription and 20 proteins have chromatin binding abilities ([Fig. 9](#F9)*A*; [Table 6](#T6)). Some of these proteins have overlapping roles in transcription, chromatin binding, and splicing. Again, fewer proteins associated with the A31 oligo relate to transcription (17 proteins) and chromatin binding (5 proteins; [Fig. 9](#F9); [Table 6](#T6)). These findings not only support the notion that alternative splicing is tightly coupled with transcription and chromatin remodeling, but also provide new clues to how hnRNPH-mediated alternative splicing is coupled to transcription and chromatin remodeling.

## Discussion

The current study describes a significant association of an intronic SNP, rs9479757, with the severity of heroin addiction in Han-Chinese male heroin addicts. Heroin addicts with the AG genotype were more inclined to mild heroin addiction in contrast to heroin addicts with the GG genotype, who were predisposed to severe heroin addiction. This hypothesis was supported by individual SNP analysis and haplotype-based analysis with additional SNPs in the *OPRM1* locus. Importantly, significant associations with four SNPs within the same haplotype block strongly illustrate the influence of genetic variation on the severity of heroin addiction. Conditional haplotype analysis demonstrates that the genetic signal at this locus is fully explained, at least for the SNPs tested, by rs9479757. One theoretical factor influencing our association study is population stratification among the Chinese population. Though this is often a concern in studies conducted in the heterogeneous population of the United States, and adjusted for using ancestry-informative markers, this is less of a concern in this Chinese population as all participants in our study are Han-Chinese recruited from Ningbo City where there is limited population migration. [Shi et al. (2002)](#B46) previously reported that the heroin addicts with the rs9479757 GG genotype in a mixed male and female population were likely to consume a small daily dose of heroin. However, they also showed that the heroin addicts with both the rs9479757 GG and the rs1799971 AG/GG genotypes had a greater daily heroin intake. It is difficult to make a comparison between the study of [Shi et al. (2002)](#B46) and our study because the gender, sample size, and criteria for the severity of heroin addiction in our study were different. We realize that the sample size in our study is relatively small, and it needs to be replicated in a second cohort or larger sample size. Nevertheless, our data suggest that genetic variants of the *OPRM1* gene can be used for predicting the severity of heroin addiction.

Our EMAS and UV cross-linking studies have established that hnRNPH is the major binding partner for the SNP G-containing site, and the G-to-A transition of this site significantly lowered the binding affinity for hnRNPH, suggesting that during splicing, pre-mRNA with the G allele attracts more hnRNPH proteins to the 5′ splice site of intron 2 than pre-mRNA with the A allele, thus modulating exon 2 splicing. This suggestion was validated by our studies using BAC minigene constructs in Be(2)C cells. Our results showed that the G allele of rs9479757 facilitated exon 2 inclusion to decrease the expression of hMOR-1S and hMOR-1Z, and to increase the expression of hMOR-1 mRNA. In contrast, the A allele reduced exon 2 inclusion with opposite effects on mRNA expression of the splice variants. These results were further supported by evidence of splicing changes in the human postmortem PFC with either the AG or GG genotype. Together, these findings demonstrate that the G-contained SNP site functions as an ISE, and that the binding of hnRNPH to this ISE enhances exon 2 inclusion. The G-to-A transition weakens the ISE by decreasing hnRNPH binding, thus leading to the reduction of exon 2 inclusion.

Involvement of hnRNPH in exon 2 inclusion/skipping was further supported by our siRNA studies. Downregulating the expression of hnRNPH by siRNAs led to the reduction of exon 2 inclusion in both the endogenous *OPRM1* gene and the exogenously transfected BAC minigene, a scenario similar to that seen in the G-to-A transition of the SNP. Furthermore, blocking the SNP-contained ISE site to which hnRNPH binds with an antisense vivo-morpholino oligo effectively weakened exon 2 inclusion, not only supporting the role of the ISE site and hnRNPH in exon 2 inclusion/skipping, but also providing potential tools to modulate exon 2 inclusion/skipping and the expression of the full-length MOR-1 and single TM variants *in vivo*.

G-rich regions or repeats have long been known to frequently be present at 5′ introns ([Engelbrecht et al., 1992](#B13)) and involve in exon inclusion and intron excision, acting mainly as an ISE. These G-rich ISE regions have been found in chicken β-tropomycin ([Sirand-Pugnet et al., 1995](#B47)), cardiac troponin T genes ([Carlo et al., 1996](#B8)), minute virus of mice ([Haut and Pintel, 1998](#B15)), human α-globin ([McCullough and Berget, 1997](#B34)), and growth hormone genes ([McCarthy and Phillips, 1998](#B33)). Our results provide another example of how G triplets can promote exon inclusion. U1 snRNPs and hnRNP proteins, especially hnRNPH, have been implicated in mediating the activity of G-rich ISE ([Wang and Cambi, 2009](#B51)).

Despite reduced hMOR-1 mRNA expression, it is intriguing to observe an increase in the levels of hMOR-1 protein in Be(2)C cells treated with the antisense morpholino oligo, as well as in the postmortem PFC samples with the AG genotype. Importantly, along with the above findings, the altered splicing in both these samples showed an increase in the expression of single TM variants hMOR-1S and hMOR-1Z, which have previously been demonstrated to increase the expression of the functional full-length 7-TM MOR-1 at the protein level in both *in vitro* cell models and *in vivo* mouse models, as well as enhancing morphine analgesia, through a chaperon-like action. The single TM variants have also been implicated in disrupting heterodimerization of MOR-1 and DOR-1 to reduce MOR-1 protein degradation ([He et al., 2011](#B16)). Therefore, we speculate that the increase in single TM variants accounts for the observed increase in levels of hMOR-1 protein in these current studies. However, other mechanisms such as translation regulation cannot be ruled out. Previous reports showed that expression levels of MOR-1 protein can be regulated by a nonsynonymous SNP, rs1799971 (A118G), that either altered the secondary mRNA structure to influence MOR-1 translation ([Zhang et al., 2005](#B57)) or modified the *N*-glycosylation of the receptor to affect MOR-1 protein stability ([Huang et al., 2012](#B20)). Our results endorse a new case that the expression level of MOR-1 protein can be modulated by an intronic SNP that acts as a splicing modifier via hnRNPH interactions, providing an important functional link between the regulation of hMOR-1 protein expression and the severity of heroin addiction.

[Huang et al. (2008)](#B19) reported that healthy Han-Chinese women with the AG genotype of this SNP had significantly higher pressure pain threshold (PPT) than those with the GG genotype. One potential explanation for the higher PPT seen in women with the AG genotype is that, similar to the heroin addicts, the women with the AG genotype express more hMOR-1 protein, presumably due to the role of increased levels of single TM variants. Though this interpretation is speculative, the data somewhat support our finding that the genotype of this SNP is associated with the expression level of hMOR-1 protein in heroin addicts.

hnRNPH was originally identified as an RNA binding protein with three quasi-RNA recognition motifs that bind to a G-rich sequence ([Honoré et al., 1995](#B18)). hnRNPH involves a wide range of mRNA processing and has diverse roles in exon or intron definition. When bound to exons, hnRNPH typically inhibits exon inclusion by antagonizing enhancers or spliceosome components ([Chen et al., 1999](#B10); [Mauger et al., 2008](#B32); [Lefave et al., 2011](#B28)). When bound to introns, hnRNPH promotes exon inclusion, as initially demonstrated in regulating the c-*src* N1 exon inclusion via interaction with the G-rich ISE ([Chou et al., 1999](#B11)). Mechanistically, it has been proposed that hnRNPH interacts with hnRNP A/B in intronic G-rich regions to loop out intronic regions ([Martinez-Contreras et al., 2006](#B31)) or to recruit U1 snRNP ([Wang and Cambi, 2009](#B51)). Here, we show an additional example of hnRNPH-mediated exon inclusion in the *OPRM1* gene, which uses a G-rich 5′ intron 2 region containing a heroin addiction severity-associated SNP. Our data from RNA affinity purification-coupled LC-MS/MS demonstrate that the hnRNPH-mediated splicing regulation in the *OPRM1* gene involves multiple hnRNPH-associated proteins. In this context, our data show that hnRNPH has the capacity to organize a large protein complex or network that contributes significantly to splicing and transcription, and that an intronic SNP can modulate the formation and function of this hnRNPH-mediated protein complex.

Although our data suggest that an intronic SNP influences *OPRM1* alternative splicing and can predict the severity of heroin addiction, the exact pathways linking SNP-mediated changes in *OPRM1* alternative splicing, the expression of MOR-1 and single TM variants to the phenotypic severity of heroin addiction remain unknown. Further delineating the mechanisms behind this association using various approaches such as *in vivo* brain imaging with appropriate μ agonist markers and gene targeting mouse models mimicking the human genotypes will greatly advance our understanding of the genetic influence on the severity of heroin addiction.

In conclusion, our current studies illustrate the association of an intronic SNP (rs9479757) with heroin addiction severity among Chinese male heroin addicts. Furthermore, we establish the functional relevance of this SNP in modulating *OPRM1* alternative splicing through hnRNPH-mediated splicing regulation and expression of hMOR-1 protein. Our findings provide new insights into the role of this intronic SNP on heroin addiction severity and *OPRM1* alternative splicing. We suggest a functional link between an SNP-containing splicing modifier and heroin addiction severity, via hnRNPH interactions.

## Footnotes

## References

1. Bart G, Kreek MJ, Ott J, LaForge KS, Proudnikov D, Pollak L, Heilig M. Increased attributable risk related to a functional mu-opioid receptor gene polymorphism in association with alcohol dependence in central Sweden. Neuropsychopharmacology. 2005;30:417–422. doi: 10.1038/sj.npp.1300598.  [DOI](https://doi.org/10.1038/sj.npp.1300598) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15525999/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neuropsychopharmacology&title=Increased%20attributable%20risk%20related%20to%20a%20functional%20mu-opioid%20receptor%20gene%20polymorphism%20in%20association%20with%20alcohol%20dependence%20in%20central%20Sweden&author=G%20Bart&author=MJ%20Kreek&author=J%20Ott&author=KS%20LaForge&author=D%20Proudnikov&volume=30&publication_year=2005&pages=417-422&pmid=15525999&doi=10.1038/sj.npp.1300598&)

2. Bayerer B, Stamer U, Hoeft A, Stüber F. Genomic variations and transcriptional regulation of the human [mu]-opioid receptor gene. Eur J Pain. 2007;11:421–427. doi: 10.1016/j.ejpain.2006.06.004.  [DOI](https://doi.org/10.1016/j.ejpain.2006.06.004) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16843022/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Pain&title=Genomic%20variations%20and%20transcriptional%20regulation%20of%20the%20human%20%5Bmu%5D-opioid%20receptor%20gene&author=B%20Bayerer&author=U%20Stamer&author=A%20Hoeft&author=F%20St%C3%BCber&volume=11&publication_year=2007&pages=421-427&pmid=16843022&doi=10.1016/j.ejpain.2006.06.004&)

3. Befort K, Filliol D, Decaillot FM, Gaveriaux-Ruff C, Hoehe MR, Kieffer BL. A single nucleotide polymorphic mutation in the human mu-opioid receptor severely impairs receptor signaling. J Biol Chem. 2001;276:3130–3137. doi: 10.1074/jbc.M006352200.  [DOI](https://doi.org/10.1074/jbc.M006352200) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11067846/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=A%20single%20nucleotide%20polymorphic%20mutation%20in%20the%20human%20mu-opioid%20receptor%20severely%20impairs%20receptor%20signaling&author=K%20Befort&author=D%20Filliol&author=FM%20Decaillot&author=C%20Gaveriaux-Ruff&author=MR%20Hoehe&volume=276&publication_year=2001&pages=3130-3137&pmid=11067846&doi=10.1074/jbc.M006352200&)

4. Berrettini WH, Lerman CE. Pharmacotherapy and pharmacogenetics of nicotine dependence. Am J Psychiatry. 2005;162:1441–1451. doi: 10.1176/appi.ajp.162.8.1441.  [DOI](https://doi.org/10.1176/appi.ajp.162.8.1441) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16055765/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Psychiatry&title=Pharmacotherapy%20and%20pharmacogenetics%20of%20nicotine%20dependence&author=WH%20Berrettini&author=CE%20Lerman&volume=162&publication_year=2005&pages=1441-1451&pmid=16055765&doi=10.1176/appi.ajp.162.8.1441&)

5. Bolan EA, Pan YX, Pasternak GW. Functional analysis of MOR-1 splice variants of the mouse mu opioid receptor gene Oprm. Synapse. 2004;51:11–18. doi: 10.1002/syn.10277.  [DOI](https://doi.org/10.1002/syn.10277) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14579421/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Synapse&title=Functional%20analysis%20of%20MOR-1%20splice%20variants%20of%20the%20mouse%20mu%20opioid%20receptor%20gene%20Oprm&author=EA%20Bolan&author=YX%20Pan&author=GW%20Pasternak&volume=51&publication_year=2004&pages=11-18&pmid=14579421&doi=10.1002/syn.10277&)

6. Bond C, LaForge KS, Tian M, Melia D, Zhang S, Borg L, Gong J, Schluger J, Strong JA, Leal SM, Tischfield JA, Kreek MJ, Yu L. Single-nucleotide polymorphism in the human mu opioid receptor gene alters beta-endorphin binding and activity: possible implications for opiate addiction. Proc Natl Acad Sci U S A. 1998;95:9608–9613. doi: 10.1073/pnas.95.16.9608.  [DOI](https://doi.org/10.1073/pnas.95.16.9608) | [PMC free article](/articles/PMC21386/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9689128/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&title=Single-nucleotide%20polymorphism%20in%20the%20human%20mu%20opioid%20receptor%20gene%20alters%20beta-endorphin%20binding%20and%20activity:%20possible%20implications%20for%20opiate%20addiction&author=C%20Bond&author=KS%20LaForge&author=M%20Tian&author=D%20Melia&author=S%20Zhang&volume=95&publication_year=1998&pages=9608-9613&pmid=9689128&doi=10.1073/pnas.95.16.9608&)

7. Caputi M, Zahler AM. Determination of the RNA binding specificity of the heterogeneous nuclear ribonucleoprotein (hnRNP) H/H'/F/2H9 family. J Biol Chem. 2001;276:43850–43859. doi: 10.1074/jbc.M102861200.  [DOI](https://doi.org/10.1074/jbc.M102861200) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11571276/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=Determination%20of%20the%20RNA%20binding%20specificity%20of%20the%20heterogeneous%20nuclear%20ribonucleoprotein%20(hnRNP)%20H/H'/F/2H9%20family&author=M%20Caputi&author=AM%20Zahler&volume=276&publication_year=2001&pages=43850-43859&pmid=11571276&doi=10.1074/jbc.M102861200&)

8. Carlo T, Sterner DA, Berget SM. An intron splicing enhancer containing a G-rich repeat facilitates inclusion of a vertebrate micro-exon. RNA. 1996;2:342–353.  [PMC free article](/articles/PMC1369377/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8634915/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=RNA&title=An%20intron%20splicing%20enhancer%20containing%20a%20G-rich%20repeat%20facilitates%20inclusion%20of%20a%20vertebrate%20micro-exon&author=T%20Carlo&author=DA%20Sterner&author=SM%20Berget&volume=2&publication_year=1996&pages=342-353&pmid=8634915&)

9. Cartegni L, Wang J, Zhu Z, Zhang MQ, Krainer AR. ESEfinder: a web resource to identify exonic splicing enhancers. Nucleic Acids Res. 2003;31:3568–3571. doi: 10.1093/nar/gkg616.  [DOI](https://doi.org/10.1093/nar/gkg616) | [PMC free article](/articles/PMC169022/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12824367/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=ESEfinder:%20a%20web%20resource%20to%20identify%20exonic%20splicing%20enhancers&author=L%20Cartegni&author=J%20Wang&author=Z%20Zhu&author=MQ%20Zhang&author=AR%20Krainer&volume=31&publication_year=2003&pages=3568-3571&pmid=12824367&doi=10.1093/nar/gkg616&)

10. Chen CD, Kobayashi R, Helfman DM. Binding of hnRNP H to an exonic splicing silencer is involved in the regulation of alternative splicing of the rat beta-tropomyosin gene. Genes Dev. 1999;13:593–606. doi: 10.1101/gad.13.5.593.  [DOI](https://doi.org/10.1101/gad.13.5.593) | [PMC free article](/articles/PMC316507/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10072387/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genes%20Dev&title=Binding%20of%20hnRNP%20H%20to%20an%20exonic%20splicing%20silencer%20is%20involved%20in%20the%20regulation%20of%20alternative%20splicing%20of%20the%20rat%20beta-tropomyosin%20gene&author=CD%20Chen&author=R%20Kobayashi&author=DM%20Helfman&volume=13&publication_year=1999&pages=593-606&pmid=10072387&doi=10.1101/gad.13.5.593&)

11. Chou MY, Rooke N, Turck CW, Black DL. hnRNP H is a component of a splicing enhancer complex that activates a c-src alternative exon in neuronal cells. Mol Cell Biol. 1999;19:69–77. doi: 10.1128/mcb.19.1.69.  [DOI](https://doi.org/10.1128/mcb.19.1.69) | [PMC free article](/articles/PMC83866/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9858532/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Cell%20Biol&title=hnRNP%20H%20is%20a%20component%20of%20a%20splicing%20enhancer%20complex%20that%20activates%20a%20c-src%20alternative%20exon%20in%20neuronal%20cells&author=MY%20Chou&author=N%20Rooke&author=CW%20Turck&author=DL%20Black&volume=19&publication_year=1999&pages=69-77&pmid=9858532&doi=10.1128/mcb.19.1.69&)

12. Drakenberg K, Nikoshkov A, Horváth MC, Fagergren P, Gharibyan A, Saarelainen K, Rahman S, Nylander I, Bakalkin G, Rajs J, Keller E, Hurd YL. Mu opioid receptor A118G polymorphism in association with striatal opioid neuropeptide gene expression in heroin abusers. Proc Natl Acad Sci U S A. 2006;103:7883–7888. doi: 10.1073/pnas.0600871103.  [DOI](https://doi.org/10.1073/pnas.0600871103) | [PMC free article](/articles/PMC1472539/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16682632/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&title=Mu%20opioid%20receptor%20A118G%20polymorphism%20in%20association%20with%20striatal%20opioid%20neuropeptide%20gene%20expression%20in%20heroin%20abusers&author=K%20Drakenberg&author=A%20Nikoshkov&author=MC%20Horv%C3%A1th&author=P%20Fagergren&author=A%20Gharibyan&volume=103&publication_year=2006&pages=7883-7888&pmid=16682632&doi=10.1073/pnas.0600871103&)

13. Engelbrecht J, Knudsen S, Brunak S. G+C-rich tract in 5′ end of human introns. J Mol Biol. 1992;227:108–113. doi: 10.1016/0022-2836(92)90685-D.  [DOI](https://doi.org/10.1016/0022-2836(92)90685-D) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/1522582/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Mol%20Biol&title=G+C-rich%20tract%20in%205%E2%80%B2%20end%20of%20human%20introns&author=J%20Engelbrecht&author=S%20Knudsen&author=S%20Brunak&volume=227&publication_year=1992&pages=108-113&pmid=1522582&doi=10.1016/0022-2836(92)90685-D&)

14. Fortin JP, Ci L, Schroeder J, Goldstein C, Montefusco MC, Peter I, Reis SE, Huggins GS, Beinborn M, Kopin AS. The mu-opioid receptor variant N190K is unresponsive to peptide agonists yet can be rescued by small-molecule drugs. Mol Pharmacol. 2010;78:837–845. doi: 10.1124/mol.110.064188.  [DOI](https://doi.org/10.1124/mol.110.064188) | [PMC free article](/articles/PMC2981363/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20702761/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Pharmacol&title=The%20mu-opioid%20receptor%20variant%20N190K%20is%20unresponsive%20to%20peptide%20agonists%20yet%20can%20be%20rescued%20by%20small-molecule%20drugs&author=JP%20Fortin&author=L%20Ci&author=J%20Schroeder&author=C%20Goldstein&author=MC%20Montefusco&volume=78&publication_year=2010&pages=837-845&pmid=20702761&doi=10.1124/mol.110.064188&)

15. Haut DD, Pintel DJ. Intron definition is required for excision of the minute virus of mice small intron and definition of the upstream exon. J Virol. 1998;72:1834–1843. doi: 10.1128/jvi.72.3.1834-1843.1998.  [DOI](https://doi.org/10.1128/jvi.72.3.1834-1843.1998) | [PMC free article](/articles/PMC109473/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9499034/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Virol&title=Intron%20definition%20is%20required%20for%20excision%20of%20the%20minute%20virus%20of%20mice%20small%20intron%20and%20definition%20of%20the%20upstream%20exon&author=DD%20Haut&author=DJ%20Pintel&volume=72&publication_year=1998&pages=1834-1843&pmid=9499034&doi=10.1128/jvi.72.3.1834-1843.1998&)

16. He SQ, Zhang ZN, Guan JS, Liu HR, Zhao B, Wang HB, Li Q, Yang H, Luo J, Li ZY, Wang Q, Lu YJ, Bao L, Zhang X. Facilitation of mu-opioid receptor activity by preventing delta-opioid receptor-mediated codegradation. Neuron. 2011;69:120–131. doi: 10.1016/j.neuron.2010.12.001.  [DOI](https://doi.org/10.1016/j.neuron.2010.12.001) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21220103/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neuron&title=Facilitation%20of%20mu-opioid%20receptor%20activity%20by%20preventing%20delta-opioid%20receptor-mediated%20codegradation&author=SQ%20He&author=ZN%20Zhang&author=JS%20Guan&author=HR%20Liu&author=B%20Zhao&volume=69&publication_year=2011&pages=120-131&pmid=21220103&doi=10.1016/j.neuron.2010.12.001&)

17. Hoehe MR, Köpke K, Wendel B, Rohde K, Flachmeier C, Kidd KK, Berrettini WH, Church GM. Sequence variability and candidate gene analysis in complex disease: association of mu opioid receptor gene variation with substance dependence. Hum Mol Genet. 2000;9:2895–2908. doi: 10.1093/hmg/9.19.2895.  [DOI](https://doi.org/10.1093/hmg/9.19.2895) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11092766/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mol%20Genet&title=Sequence%20variability%20and%20candidate%20gene%20analysis%20in%20complex%20disease:%20association%20of%20mu%20opioid%20receptor%20gene%20variation%20with%20substance%20dependence&author=MR%20Hoehe&author=K%20K%C3%B6pke&author=B%20Wendel&author=K%20Rohde&author=C%20Flachmeier&volume=9&publication_year=2000&pages=2895-2908&pmid=11092766&doi=10.1093/hmg/9.19.2895&)

18. Honoré B, Rasmussen HH, Vorum H, Dejgaard K, Liu X, Gromov P, Madsen P, Gesser B, Tommerup N, Celis JE. Heterogeneous nuclear ribonucleoproteins H, H', and F are members of a ubiquitously expressed subfamily of related but distinct proteins encoded by genes mapping to different chromosomes. J Biol Chem. 1995;270:28780–28789. doi: 10.1074/jbc.270.48.28780.  [DOI](https://doi.org/10.1074/jbc.270.48.28780) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7499401/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=Heterogeneous%20nuclear%20ribonucleoproteins%20H,%20H',%20and%20F%20are%20members%20of%20a%20ubiquitously%20expressed%20subfamily%20of%20related%20but%20distinct%20proteins%20encoded%20by%20genes%20mapping%20to%20different%20chromosomes&author=B%20Honor%C3%A9&author=HH%20Rasmussen&author=H%20Vorum&author=K%20Dejgaard&author=X%20Liu&volume=270&publication_year=1995&pages=28780-28789&pmid=7499401&doi=10.1074/jbc.270.48.28780&)

19. Huang CJ, Liu HF, Su NY, Hsu YW, Yang CH, Chen CC, Tsai PS. Association between human opioid receptor genes polymorphisms and pressure pain sensitivity in females*. Anaesthesia. 2008;63:1288–1295. doi: 10.1111/j.1365-2044.2008.05760.x.  [DOI](https://doi.org/10.1111/j.1365-2044.2008.05760.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19032295/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Anaesthesia&title=Association%20between%20human%20opioid%20receptor%20genes%20polymorphisms%20and%20pressure%20pain%20sensitivity%20in%20females*&author=CJ%20Huang&author=HF%20Liu&author=NY%20Su&author=YW%20Hsu&author=CH%20Yang&volume=63&publication_year=2008&pages=1288-1295&pmid=19032295&doi=10.1111/j.1365-2044.2008.05760.x&)

20. Huang P, Chen C, Mague SD, Blendy JA, Liu-Chen LY. A common single nucleotide polymorphism A118G of the mu opioid receptor alters its N-glycosylation and protein stability. Biochem J. 2012;441:379–386. doi: 10.1042/BJ20111050.  [DOI](https://doi.org/10.1042/BJ20111050) | [PMC free article](/articles/PMC3923516/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21864297/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem%20J&title=A%20common%20single%20nucleotide%20polymorphism%20A118G%20of%20the%20mu%20opioid%20receptor%20alters%20its%20N-glycosylation%20and%20protein%20stability&author=P%20Huang&author=C%20Chen&author=SD%20Mague&author=JA%20Blendy&author=LY%20Liu-Chen&volume=441&publication_year=2012&pages=379-386&pmid=21864297&doi=10.1042/BJ20111050&)

21. Ikeda K, Ide S, Han W, Hayashida M, Uhl GR, Sora I. How individual sensitivity to opiates can be predicted by gene analyses. Trends Pharmacol Sci. 2005;26:311–317. doi: 10.1016/j.tips.2005.04.001.  [DOI](https://doi.org/10.1016/j.tips.2005.04.001) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15925706/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Trends%20Pharmacol%20Sci&title=How%20individual%20sensitivity%20to%20opiates%20can%20be%20predicted%20by%20gene%20analyses&author=K%20Ikeda&author=S%20Ide&author=W%20Han&author=M%20Hayashida&author=GR%20Uhl&volume=26&publication_year=2005&pages=311-317&pmid=15925706&doi=10.1016/j.tips.2005.04.001&)

22. Jacobs MM, Ökvist A, Horvath M, Keller E, Bannon MJ, Morgello S, Hurd YL. Dopamine receptor D1 and postsynaptic density gene variants associate with opiate abuse and striatal expression levels. Mol Psychiatry. 2013;18:1205–1210. doi: 10.1038/mp.2012.140.  [DOI](https://doi.org/10.1038/mp.2012.140) | [PMC free article](/articles/PMC3637428/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23044706/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Psychiatry&title=Dopamine%20receptor%20D1%20and%20postsynaptic%20density%20gene%20variants%20associate%20with%20opiate%20abuse%20and%20striatal%20expression%20levels&author=MM%20Jacobs&author=A%20%C3%96kvist&author=M%20Horvath&author=E%20Keller&author=MJ%20Bannon&volume=18&publication_year=2013&pages=1205-1210&pmid=23044706&doi=10.1038/mp.2012.140&)

23. Kim SG, Kim CM, Kang DH, Kim YJ, Byun WT, Kim SY, Park JM, Kim MJ, Oslin DW. Association of functional opioid receptor genotypes with alcohol dependence in Koreans. Alcohol Clin Exp Res. 2004;28:986–990. doi: 10.1097/01.ALC.0000130803.62768.AB.  [DOI](https://doi.org/10.1097/01.ALC.0000130803.62768.AB) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15252283/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Alcohol%20Clin%20Exp%20Res&title=Association%20of%20functional%20opioid%20receptor%20genotypes%20with%20alcohol%20dependence%20in%20Koreans&author=SG%20Kim&author=CM%20Kim&author=DH%20Kang&author=YJ%20Kim&author=WT%20Byun&volume=28&publication_year=2004&pages=986-990&pmid=15252283&doi=10.1097/01.ALC.0000130803.62768.AB&)

24. Kraus J, Borner C, Giannini E, Hickfang K, Braun H, Mayer P, Hoehe MR, Ambrosch A, Konig W, Hollt V. Regulation of Mu opioid receptor gene transcription by interleukin-4 and influence of an allelic variation within a STAT6 transcription factor binding site. J Biol Chem. 2001;276:43901–43908. doi: 10.1074/jbc.M107543200.  [DOI](https://doi.org/10.1074/jbc.M107543200) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11572871/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=Regulation%20of%20Mu%20opioid%20receptor%20gene%20transcription%20by%20interleukin-4%20and%20influence%20of%20an%20allelic%20variation%20within%20a%20STAT6%20transcription%20factor%20binding%20site&author=J%20Kraus&author=C%20Borner&author=E%20Giannini&author=K%20Hickfang&author=H%20Braun&volume=276&publication_year=2001&pages=43901-43908&pmid=11572871&doi=10.1074/jbc.M107543200&)

25. Kreek MJ, Bart G, Lilly C, LaForge KS, Nielsen DA. Pharmacogenetics and human molecular genetics of opiate and cocaine addictions and their treatments. Pharmacol Rev. 2005;57:1–26. doi: 10.1124/pr.57.1.1.  [DOI](https://doi.org/10.1124/pr.57.1.1) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15734726/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacol%20Rev&title=Pharmacogenetics%20and%20human%20molecular%20genetics%20of%20opiate%20and%20cocaine%20addictions%20and%20their%20treatments&author=MJ%20Kreek&author=G%20Bart&author=C%20Lilly&author=KS%20LaForge&author=DA%20Nielsen&volume=57&publication_year=2005&pages=1-26&pmid=15734726&doi=10.1124/pr.57.1.1&)

26. Kroslak T, Laforge KS, Gianotti RJ, Ho A, Nielsen DA, Kreek MJ. The single nucleotide polymorphism A118G alters functional properties of the human mu opioid receptor. J Neurochem. 2007;103:77–87. doi: 10.1111/j.1471-4159.2007.04738.x.  [DOI](https://doi.org/10.1111/j.1471-4159.2007.04738.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17877633/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Neurochem&title=The%20single%20nucleotide%20polymorphism%20A118G%20alters%20functional%20properties%20of%20the%20human%20mu%20opioid%20receptor&author=T%20Kroslak&author=KS%20Laforge&author=RJ%20Gianotti&author=A%20Ho&author=DA%20Nielsen&volume=103&publication_year=2007&pages=77-87&pmid=17877633&doi=10.1111/j.1471-4159.2007.04738.x&)

27. Lee EC, Yu D, Martinez de Velasco J, Tessarollo L, Swing DA, Court DL, Jenkins NA, Copeland NG. A highly efficient Escherichia coli-based chromosome engineering system adapted for recombinogenic targeting and subcloning of BAC DNA. Genomics. 2001;73:56–65. doi: 10.1006/geno.2000.6451.  [DOI](https://doi.org/10.1006/geno.2000.6451) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11352566/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genomics&title=A%20highly%20efficient%20Escherichia%20coli-based%20chromosome%20engineering%20system%20adapted%20for%20recombinogenic%20targeting%20and%20subcloning%20of%20BAC%20DNA&author=EC%20Lee&author=D%20Yu&author=J%20Martinez%20de%20Velasco&author=L%20Tessarollo&author=DA%20Swing&volume=73&publication_year=2001&pages=56-65&pmid=11352566&doi=10.1006/geno.2000.6451&)

28. Lefave CV, Squatrito M, Vorlova S, Rocco GL, Brennan CW, Holland EC, Pan YX, Cartegni L. Splicing factor hnRNPH drives an oncogenic splicing switch in gliomas. EMBO J. 2011;30:4084–4097. doi: 10.1038/emboj.2011.259.  [DOI](https://doi.org/10.1038/emboj.2011.259) | [PMC free article](/articles/PMC3209773/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21915099/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=EMBO%20J&title=Splicing%20factor%20hnRNPH%20drives%20an%20oncogenic%20splicing%20switch%20in%20gliomas&author=CV%20Lefave&author=M%20Squatrito&author=S%20Vorlova&author=GL%20Rocco&author=CW%20Brennan&volume=30&publication_year=2011&pages=4084-4097&pmid=21915099&doi=10.1038/emboj.2011.259&)

29. Lerman C, Wileyto EP, Patterson F, Rukstalis M, Audrain-McGovern J, Restine S, Shields PG, Kaufmann V, Redden D, Benowitz N, Berrettini WH. The functional mu opioid receptor (OPRM1) Asn40Asp variant predicts short-term response to nicotine replacement therapy in a clinical trial. Pharmacogenomics J. 2004;4:184–192. doi: 10.1038/sj.tpj.6500238.  [DOI](https://doi.org/10.1038/sj.tpj.6500238) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15007373/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=The%20functional%20mu%20opioid%20receptor%20(OPRM1)%20Asn40Asp%20variant%20predicts%20short-term%20response%20to%20nicotine%20replacement%20therapy%20in%20a%20clinical%20trial&author=C%20Lerman&author=EP%20Wileyto&author=F%20Patterson&author=M%20Rukstalis&author=J%20Audrain-McGovern&volume=4&publication_year=2004&pages=184-192&pmid=15007373&doi=10.1038/sj.tpj.6500238&)

30. Luo X, Kranzler HR, Zhao H, Gelernter J. Haplotypes at the OPRM1 locus are associated with susceptibility to substance dependence in European-Americans. Am J Med Genet B Neuropsychiatr Genet. 2003;120B:97–108. doi: 10.1002/ajmg.b.20034.  [DOI](https://doi.org/10.1002/ajmg.b.20034) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12815747/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Med%20Genet%20B%20Neuropsychiatr%20Genet&title=Haplotypes%20at%20the%20OPRM1%20locus%20are%20associated%20with%20susceptibility%20to%20substance%20dependence%20in%20European-Americans&author=X%20Luo&author=HR%20Kranzler&author=H%20Zhao&author=J%20Gelernter&volume=120B&publication_year=2003&pages=97-108&pmid=12815747&doi=10.1002/ajmg.b.20034&)

31. Martinez-Contreras R, Fisette JF, Nasim FU, Madden R, Cordeau M, Chabot B. Intronic binding sites for hnRNP A/B and hnRNP F/H proteins stimulate pre-mRNA splicing. PLoS Biol. 2006;4:e21. doi: 10.1371/journal.pbio.0040021.  [DOI](https://doi.org/10.1371/journal.pbio.0040021) | [PMC free article](/articles/PMC1326234/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16396608/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Biol&title=Intronic%20binding%20sites%20for%20hnRNP%20A/B%20and%20hnRNP%20F/H%20proteins%20stimulate%20pre-mRNA%20splicing&author=R%20Martinez-Contreras&author=JF%20Fisette&author=FU%20Nasim&author=R%20Madden&author=M%20Cordeau&volume=4&publication_year=2006&pages=e21&pmid=16396608&doi=10.1371/journal.pbio.0040021&)

32. Mauger DM, Lin C, Garcia-Blanco MA. hnRNP H and hnRNP F complex with Fox2 to silence fibroblast growth factor receptor 2 exon IIIc. Mol Cell Biol. 2008;28:5403–5419. doi: 10.1128/MCB.00739-08.  [DOI](https://doi.org/10.1128/MCB.00739-08) | [PMC free article](/articles/PMC2519734/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18573884/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Cell%20Biol&title=hnRNP%20H%20and%20hnRNP%20F%20complex%20with%20Fox2%20to%20silence%20fibroblast%20growth%20factor%20receptor%202%20exon%20IIIc&author=DM%20Mauger&author=C%20Lin&author=MA%20Garcia-Blanco&volume=28&publication_year=2008&pages=5403-5419&pmid=18573884&doi=10.1128/MCB.00739-08&)

33. McCarthy EM, Phillips JA., 3rd Characterization of an intron splice enhancer that regulates alternative splicing of human GH pre-mRNA. Hum Mol Genet. 1998;7:1491–1496. doi: 10.1093/hmg/7.9.1491.  [DOI](https://doi.org/10.1093/hmg/7.9.1491) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9700205/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mol%20Genet&title=Characterization%20of%20an%20intron%20splice%20enhancer%20that%20regulates%20alternative%20splicing%20of%20human%20GH%20pre-mRNA&author=EM%20McCarthy&author=JA%20Phillips&volume=7&publication_year=1998&pages=1491-1496&pmid=9700205&doi=10.1093/hmg/7.9.1491&)

34. McCullough AJ, Berget SM. G triplets located throughout a class of small vertebrate introns enforce intron borders and regulate splice site selection. Mol Cell Biol. 1997;17:4562–4571. doi: 10.1128/mcb.17.8.4562.  [DOI](https://doi.org/10.1128/mcb.17.8.4562) | [PMC free article](/articles/PMC232310/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9234714/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Cell%20Biol&title=G%20triplets%20located%20throughout%20a%20class%20of%20small%20vertebrate%20introns%20enforce%20intron%20borders%20and%20regulate%20splice%20site%20selection&author=AJ%20McCullough&author=SM%20Berget&volume=17&publication_year=1997&pages=4562-4571&pmid=9234714&doi=10.1128/mcb.17.8.4562&)

35. Pan L, Xu J, Yu R, Xu MM, Pan YX, Pasternak GW. Identification and characterization of six new alternatively spliced variants of the human mu opioid receptor gene, Oprm. Neuroscience. 2005;133:209–220. doi: 10.1016/j.neuroscience.2004.12.033.  [DOI](https://doi.org/10.1016/j.neuroscience.2004.12.033) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15893644/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neuroscience&title=Identification%20and%20characterization%20of%20six%20new%20alternatively%20spliced%20variants%20of%20the%20human%20mu%20opioid%20receptor%20gene,%20Oprm&author=L%20Pan&author=J%20Xu&author=R%20Yu&author=MM%20Xu&author=YX%20Pan&volume=133&publication_year=2005&pages=209-220&pmid=15893644&doi=10.1016/j.neuroscience.2004.12.033&)

36. Pan YX. Identification and characterization of a novel promoter of the mouse mu opioid receptor gene (Oprm) that generates eight splice variants. Gene. 2002;295:97–108. doi: 10.1016/S0378-1119(02)00825-9.  [DOI](https://doi.org/10.1016/S0378-1119(02)00825-9) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12242016/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gene&title=Identification%20and%20characterization%20of%20a%20novel%20promoter%20of%20the%20mouse%20mu%20opioid%20receptor%20gene%20(Oprm)%20that%20generates%20eight%20splice%20variants&author=YX%20Pan&volume=295&publication_year=2002&pages=97-108&pmid=12242016&doi=10.1016/S0378-1119(02)00825-9&)

37. Pan YX. Diversity and complexity of the mu opioid receptor gene: alternative pre-mRNA splicing and promoters. DNA Cell Biol. 2005;24:736–750. doi: 10.1089/dna.2005.24.736.  [DOI](https://doi.org/10.1089/dna.2005.24.736) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16274294/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=DNA%20Cell%20Biol&title=Diversity%20and%20complexity%20of%20the%20mu%20opioid%20receptor%20gene:%20alternative%20pre-mRNA%20splicing%20and%20promoters&author=YX%20Pan&volume=24&publication_year=2005&pages=736-750&pmid=16274294&doi=10.1089/dna.2005.24.736&)

38. Pan YX, Xu J, Bolan E, Abbadie C, Chang A, Zuckerman A, Rossi G, Pasternak GW. Identification and characterization of three new alternatively spliced mu opioid receptor isoforms. Mol Pharmacol. 1999;56:396–403. doi: 10.1124/mol.56.2.396.  [DOI](https://doi.org/10.1124/mol.56.2.396) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10419560/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Pharmacol&title=Identification%20and%20characterization%20of%20three%20new%20alternatively%20spliced%20mu%20opioid%20receptor%20isoforms&author=YX%20Pan&author=J%20Xu&author=E%20Bolan&author=C%20Abbadie&author=A%20Chang&volume=56&publication_year=1999&pages=396-403&pmid=10419560&doi=10.1124/mol.56.2.396&)

39. Pasternak DA, Pan L, Xu J, Yu R, Xu MM, Pasternak GW, Pan YX. Identification of three new alternatively spliced variants of the rat mu opioid receptor gene: dissociation of affinity and efficacy. J Neurochem. 2004;91:881–890. doi: 10.1111/j.1471-4159.2004.02767.x.  [DOI](https://doi.org/10.1111/j.1471-4159.2004.02767.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15525342/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Neurochem&title=Identification%20of%20three%20new%20alternatively%20spliced%20variants%20of%20the%20rat%20mu%20opioid%20receptor%20gene:%20dissociation%20of%20affinity%20and%20efficacy&author=DA%20Pasternak&author=L%20Pan&author=J%20Xu&author=R%20Yu&author=MM%20Xu&volume=91&publication_year=2004&pages=881-890&pmid=15525342&doi=10.1111/j.1471-4159.2004.02767.x&)

40. Pasternak GW, Pan YX. Mu opioids and their receptors: evolution of a concept. Pharmacol Rev. 2013;65:1257–1317. doi: 10.1124/pr.112.007138.  [DOI](https://doi.org/10.1124/pr.112.007138) | [PMC free article](/articles/PMC3799236/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24076545/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacol%20Rev&title=Mu%20opioids%20and%20their%20receptors:%20evolution%20of%20a%20concept&author=GW%20Pasternak&author=YX%20Pan&volume=65&publication_year=2013&pages=1257-1317&pmid=24076545&doi=10.1124/pr.112.007138&)

41. Perkins DN, Pappin DJ, Creasy DM, Cottrell JS. Probability-based protein identification by searching sequence databases using mass spectrometry data. Electrophoresis. 1999;20:3551–3567. doi: 10.1002/(SICI)1522-2683(19991201)20:18<3551::AID-ELPS3551>3.0.CO;2-2.  [DOI](https://doi.org/10.1002/(SICI)1522-2683(19991201)20:18&#x0003c;3551::AID-ELPS3551&#x0003e;3.0.CO;2-2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10612281/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Electrophoresis&title=Probability-based%20protein%20identification%20by%20searching%20sequence%20databases%20using%20mass%20spectrometry%20data&author=DN%20Perkins&author=DJ%20Pappin&author=DM%20Creasy&author=JS%20Cottrell&volume=20&publication_year=1999&pages=3551-3567&pmid=10612281&doi=10.1002/(SICI)1522-2683(19991201)20:18&#x0003c;3551::AID-ELPS3551&#x0003e;3.0.CO;2-2&)

42. Pfaffl MW, Tichopad A, Prgomet C, Neuvians TP. Determination of stable housekeeping genes, differentially regulated target genes and sample integrity: BestKeeper–Excel-based tool using pair-wise correlations. Biotechnol Lett. 2004;26:509–515. doi: 10.1023/B:BILE.0000019559.84305.47.  [DOI](https://doi.org/10.1023/B:BILE.0000019559.84305.47) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15127793/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biotechnol%20Lett&title=Determination%20of%20stable%20housekeeping%20genes,%20differentially%20regulated%20target%20genes%20and%20sample%20integrity:%20BestKeeper%E2%80%93Excel-based%20tool%20using%20pair-wise%20correlations&author=MW%20Pfaffl&author=A%20Tichopad&author=C%20Prgomet&author=TP%20Neuvians&volume=26&publication_year=2004&pages=509-515&pmid=15127793&doi=10.1023/B:BILE.0000019559.84305.47&)

43. Proudnikov D, Randesi M, Levran O, Crystal H, Dorn M, Ott J, Ho A, Kreek MJ. Association of polymorphisms of the mu opioid receptor gene with the severity of HIV infection and response to HIV treatment. J Infect Dis. 2012;205:1745–1756. doi: 10.1093/infdis/jis264.  [DOI](https://doi.org/10.1093/infdis/jis264) | [PMC free article](/articles/PMC3415853/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22457278/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Infect%20Dis&title=Association%20of%20polymorphisms%20of%20the%20mu%20opioid%20receptor%20gene%20with%20the%20severity%20of%20HIV%20infection%20and%20response%20to%20HIV%20treatment&author=D%20Proudnikov&author=M%20Randesi&author=O%20Levran&author=H%20Crystal&author=M%20Dorn&volume=205&publication_year=2012&pages=1745-1756&pmid=22457278&doi=10.1093/infdis/jis264&)

44. Ravindranathan A, Joslyn G, Robertson M, Schuckit MA, Whistler JL, White RL. Functional characterization of human variants of the mu-opioid receptor gene. Proc Natl Acad Sci U S A. 2009;106:10811–10816. doi: 10.1073/pnas.0904509106.  [DOI](https://doi.org/10.1073/pnas.0904509106) | [PMC free article](/articles/PMC2705556/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19528663/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&title=Functional%20characterization%20of%20human%20variants%20of%20the%20mu-opioid%20receptor%20gene&author=A%20Ravindranathan&author=G%20Joslyn&author=M%20Robertson&author=MA%20Schuckit&author=JL%20Whistler&volume=106&publication_year=2009&pages=10811-10816&pmid=19528663&doi=10.1073/pnas.0904509106&)

45. Ray LA, Hutchison KE. A polymorphism of the mu-opioid receptor gene (OPRM1) and sensitivity to the effects of alcohol in humans. Alcohol Clin Exp Res. 2004;28:1789–1795. doi: 10.1097/01.ALC.0000148114.34000.B9.  [DOI](https://doi.org/10.1097/01.ALC.0000148114.34000.B9) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15608594/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Alcohol%20Clin%20Exp%20Res&title=A%20polymorphism%20of%20the%20mu-opioid%20receptor%20gene%20(OPRM1)%20and%20sensitivity%20to%20the%20effects%20of%20alcohol%20in%20humans&author=LA%20Ray&author=KE%20Hutchison&volume=28&publication_year=2004&pages=1789-1795&pmid=15608594&doi=10.1097/01.ALC.0000148114.34000.B9&)

46. Shi J, Hui L, Xu Y, Wang F, Huang W, Hu G. Sequence variations in the mu-opioid receptor gene (OPRM1) associated with human addiction to heroin. Hum Mutat. 2002;19:459–460. doi: 10.1002/humu.9026.  [DOI](https://doi.org/10.1002/humu.9026) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11933204/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mutat&title=Sequence%20variations%20in%20the%20mu-opioid%20receptor%20gene%20(OPRM1)%20associated%20with%20human%20addiction%20to%20heroin&author=J%20Shi&author=L%20Hui&author=Y%20Xu&author=F%20Wang&author=W%20Huang&volume=19&publication_year=2002&pages=459-460&pmid=11933204&doi=10.1002/humu.9026&)

47. Sirand-Pugnet P, Durosay P, Brody E, Marie J. An intronic (A/U)GGG repeat enhances the splicing of an alternative intron of the chicken beta-tropomyosin pre-mRNA. Nucleic Acids Res. 1995;23:3501–3507. doi: 10.1093/nar/23.17.3501.  [DOI](https://doi.org/10.1093/nar/23.17.3501) | [PMC free article](/articles/PMC307230/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7567462/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=An%20intronic%20(A/U)GGG%20repeat%20enhances%20the%20splicing%20of%20an%20alternative%20intron%20of%20the%20chicken%20beta-tropomyosin%20pre-mRNA&author=P%20Sirand-Pugnet&author=P%20Durosay&author=E%20Brody&author=J%20Marie&volume=23&publication_year=1995&pages=3501-3507&pmid=7567462&doi=10.1093/nar/23.17.3501&)

48. Thanaraj TA, Stamm S, Clark F, Riethoven JJ, Le Texier V, Muilu J. ASD: the alternative splicing database. Nucleic Acids Res. 2004;32:D64–D69. doi: 10.1093/nar/gkh030.  [DOI](https://doi.org/10.1093/nar/gkh030) | [PMC free article](/articles/PMC308764/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14681360/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=ASD:%20the%20alternative%20splicing%20database&author=TA%20Thanaraj&author=S%20Stamm&author=F%20Clark&author=JJ%20Riethoven&author=V%20Le%20Texier&volume=32&publication_year=2004&pages=D64-D69&pmid=14681360&doi=10.1093/nar/gkh030&)

49. Uhl GR, Sora I, Wang Z. The m opiate receptor as a candidate gene for pain: polymorphisms, variations in expression, nociception, and opiate responses. Proc Natl Acad Sci U S A. 1999;96:7752–7755. doi: 10.1073/pnas.96.14.7752.  [DOI](https://doi.org/10.1073/pnas.96.14.7752) | [PMC free article](/articles/PMC33614/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10393893/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&title=The%20m%20opiate%20receptor%20as%20a%20candidate%20gene%20for%20pain:%20polymorphisms,%20variations%20in%20expression,%20nociception,%20and%20opiate%20responses&author=GR%20Uhl&author=I%20Sora&author=Z%20Wang&volume=96&publication_year=1999&pages=7752-7755&pmid=10393893&doi=10.1073/pnas.96.14.7752&)

50. Wang D, Quillan JM, Winans K, Lucas JL, Sadée W. Single nucleotide polymorphisms in the human mu opioid receptor gene alter basal G protein coupling and calmodulin binding. J Biol Chem. 2001;276:34624–34630. doi: 10.1074/jbc.M104083200.  [DOI](https://doi.org/10.1074/jbc.M104083200) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11457836/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=Single%20nucleotide%20polymorphisms%20in%20the%20human%20mu%20opioid%20receptor%20gene%20alter%20basal%20G%20protein%20coupling%20and%20calmodulin%20binding&author=D%20Wang&author=JM%20Quillan&author=K%20Winans&author=JL%20Lucas&author=W%20Sad%C3%A9e&volume=276&publication_year=2001&pages=34624-34630&pmid=11457836&doi=10.1074/jbc.M104083200&)

51. Wang E, Cambi F. Heterogeneous nuclear ribonucleoproteins H and F regulate the proteolipid protein/DM20 ratio by recruiting U1 small nuclear ribonucleoprotein through a complex array of G runs. J Biol Chem. 2009;284:11194–11204. doi: 10.1074/jbc.M809373200.  [DOI](https://doi.org/10.1074/jbc.M809373200) | [PMC free article](/articles/PMC2670124/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19244236/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=Heterogeneous%20nuclear%20ribonucleoproteins%20H%20and%20F%20regulate%20the%20proteolipid%20protein/DM20%20ratio%20by%20recruiting%20U1%20small%20nuclear%20ribonucleoprotein%20through%20a%20complex%20array%20of%20G%20runs&author=E%20Wang&author=F%20Cambi&volume=284&publication_year=2009&pages=11194-11204&pmid=19244236&doi=10.1074/jbc.M809373200&)

52. Wang S, Zhao Y, Leiby M, Zhu J. A new positive/negative selection scheme for precise BAC recombineering. Mol Biotechnol. 2009;42:110–116. doi: 10.1007/s12033-009-9142-3.  [DOI](https://doi.org/10.1007/s12033-009-9142-3) | [PMC free article](/articles/PMC2669495/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19160076/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Biotechnol&title=A%20new%20positive/negative%20selection%20scheme%20for%20precise%20BAC%20recombineering&author=S%20Wang&author=Y%20Zhao&author=M%20Leiby&author=J%20Zhu&volume=42&publication_year=2009&pages=110-116&pmid=19160076&doi=10.1007/s12033-009-9142-3&)

53. Xie X, Xu L, Liu H, Chen W, Zhuang D, Zhang J, Duan S, Zhou W. Positive association between–1021TT genotype of dopamine beta hydroxylase gene and progressive behavior of injection heroin users. Neurosci Lett. 2013;541:258–262. doi: 10.1016/j.neulet.2013.02.049.  [DOI](https://doi.org/10.1016/j.neulet.2013.02.049) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23510745/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neurosci%20Lett&title=Positive%20association%20between%E2%80%931021TT%20genotype%20of%20dopamine%20beta%20hydroxylase%20gene%20and%20progressive%20behavior%20of%20injection%20heroin%20users&author=X%20Xie&author=L%20Xu&author=H%20Liu&author=W%20Chen&author=D%20Zhuang&volume=541&publication_year=2013&pages=258-262&pmid=23510745&doi=10.1016/j.neulet.2013.02.049&)

54. Xu J, Xu M, Brown T, Rossi GC, Hurd YL, Inturrisi CE, Pasternak GW, Pan YX. Stabilization of the mu-opioid receptor by truncated single transmembrane splice variants through a chaperone-like action. J Biol Chem. 2013;288:21211–21227. doi: 10.1074/jbc.M113.458687.  [DOI](https://doi.org/10.1074/jbc.M113.458687) | [PMC free article](/articles/PMC3774389/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23760268/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=Stabilization%20of%20the%20mu-opioid%20receptor%20by%20truncated%20single%20transmembrane%20splice%20variants%20through%20a%20chaperone-like%20action&author=J%20Xu&author=M%20Xu&author=T%20Brown&author=GC%20Rossi&author=YL%20Hurd&volume=288&publication_year=2013&pages=21211-21227&pmid=23760268&doi=10.1074/jbc.M113.458687&)

55. Zhang H, Luo X, Kranzler HR, Lappalainen J, Yang BZ, Krupitsky E, Zvartau E, Gelernter J. Association between two mu-opioid receptor gene (OPRM1) haplotype blocks and drug or alcohol dependence. Hum Mol Genet. 2006a;15:807–819. doi: 10.1093/hmg/ddl024.  [DOI](https://doi.org/10.1093/hmg/ddl024) | [PMC free article](/articles/PMC3164878/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16476706/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mol%20Genet&title=Association%20between%20two%20mu-opioid%20receptor%20gene%20(OPRM1)%20haplotype%20blocks%20and%20drug%20or%20alcohol%20dependence&author=H%20Zhang&author=X%20Luo&author=HR%20Kranzler&author=J%20Lappalainen&author=BZ%20Yang&volume=15&publication_year=2006a&pages=807-819&pmid=16476706&doi=10.1093/hmg/ddl024&)

56. Zhang L, Kendler KS, Chen X. The mu-opioid receptor gene and smoking initiation and nicotine dependence. Behav Brain Funct. 2006b;2:28. doi: 10.1186/1744-9081-2-28.  [DOI](https://doi.org/10.1186/1744-9081-2-28) | [PMC free article](/articles/PMC1557520/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16887046/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Behav%20Brain%20Funct&title=The%20mu-opioid%20receptor%20gene%20and%20smoking%20initiation%20and%20nicotine%20dependence&author=L%20Zhang&author=KS%20Kendler&author=X%20Chen&volume=2&publication_year=2006b&pages=28&pmid=16887046&doi=10.1186/1744-9081-2-28&)

57. Zhang Y, Wang D, Johnson AD, Papp AC, Sadée W. Allelic expression imbalance of human mu opioid receptor (OPRM1) caused by variant A118G. J Biol Chem. 2005;280:32618–32624. doi: 10.1074/jbc.M504942200.  [DOI](https://doi.org/10.1074/jbc.M504942200) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16046395/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=Allelic%20expression%20imbalance%20of%20human%20mu%20opioid%20receptor%20(OPRM1)%20caused%20by%20variant%20A118G&author=Y%20Zhang&author=D%20Wang&author=AD%20Johnson&author=AC%20Papp&author=W%20Sad%C3%A9e&volume=280&publication_year=2005&pages=32618-32624&pmid=16046395&doi=10.1074/jbc.M504942200&)
