# In vivo assessment of the metabolic activity of CYP2D6 diplotypes and alleles

## Metadata
**Authors:** Daniel L Hertz, Anna C Snavely, Howard L McLeod, Christine M Walko, Joseph G Ibrahim, Steven Anderson, Karen E Weck, Gustav Magrinat, Oludamilola Olajide, Susan Moore, Rachel Raab, Daniel R Carrizosa, Steven Corso, Garry Schwartz, Jeffrey M Peppercorn, James P Evans, David R Jones, Zeruesenay Desta, David A Flockhart, Lisa A Carey, William J Irvin
**Journal:** British Journal of Clinical Pharmacology
**Date:** 2015 Aug 2
**DOI:** [10.1111/bcp.12665](https://doi.org/10.1111/bcp.12665)
**PMID:** 25907378
**PMCID:** PMC4631184
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4631184/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC4631184/pdf/bcp0080-1122.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC4631184/pdf/bcp0080-1122.pdf)

## Abstract

**Aims:** 
A prospectively enrolled patient cohort was used to assess whether the prediction of CYP2D6 phenotype activity from genotype data could be improved by reclassification of diplotypes or alleles.

**Methods:** 
Three hundred and fifty-five patients receiving tamoxifen 20 mg were genotyped for CYP2D6 and tamoxifen metabolite concentrations were measured. The endoxifen: N-desmethly-tamoxifen metabolic ratio, as a surrogate of CYP2D6 activity, was compared across four diplotypes (EM/IM, EM/PM, IM/IM, IM/PM) that are typically collapsed into an intermediate metabolizer (IM) phenotype. The relative metabolic activity of each allele type (UM, EM, IM, and PM) and each EM and IM allele was estimated for comparison with the activity scores typically assigned, 2, 1, 0.5 and 0, respectively.

**Results:** 
Each of the four IM diplotypes have distinct CYP2D6 activity from each other and from the EM and PM phenotype groups (each P < 0.05). Setting the activity of an EM allele at 1.0, the relative activities of a UM, IM and PM allele were 0.85, 0.67 and 0.52, respectively. The activity of the EM alleles were statistically different (P < 0.0001), with the CYP2D6*2 allele (scaled activity = 0.63) closer in activity to an IM than an EM allele. The activity of the IM alleles were also statistically different (P = 0.014).

**Conclusion:** 
The current systems for translating CYP2D6 genotype into phenotype are not optimally calibrated, particularly in regards to IM diplotypes and the *2 allele. Additional research is needed to improve the prediction of CYP2D6 activity from genetic data for individualized dosing of CYP2D6 dependent drugs.

Keywords: activity score, CYP2D6, endoxifen, N-desmethyl-tamoxifen, pharmacogenetic, tamoxifen

### Aims

A prospectively enrolled patient cohort was used to assess whether the prediction of CYP2D6 phenotype activity from genotype data could be improved by reclassification of diplotypes or alleles.

### Methods

Three hundred and fifty-five patients receiving tamoxifen 20 mg were genotyped for *CYP2D6* and tamoxifen metabolite concentrations were measured. The endoxifen: N-desmethly-tamoxifen metabolic ratio, as a surrogate of CYP2D6 activity, was compared across four diplotypes (*EM/IM*, *EM/PM*, *IM/IM*, *IM/PM*) that are typically collapsed into an intermediate metabolizer (IM) phenotype. The relative metabolic activity of each allele type (*UM*, *EM, IM*, and *PM*) and each *EM* and *IM* allele was estimated for comparison with the activity scores typically assigned, 2, 1, 0.5 and 0, respectively.

### Results

Each of the four IM diplotypes have distinct CYP2D6 activity from each other and from the EM and PM phenotype groups (each *P* < 0.05). Setting the activity of an *EM* allele at 1.0, the relative activities of a *UM*, *IM* and *PM* allele were 0.85, 0.67 and 0.52, respectively. The activity of the EM alleles were statistically different (*P* < 0.0001), with the *CYP2D6**2 allele (scaled activity = 0.63) closer in activity to an *IM* than an *EM* allele. The activity of the *IM* alleles were also statistically different (*P* = 0.014).

### Conclusion

The current systems for translating *CYP2D6* genotype into phenotype are not optimally calibrated, particularly in regards to IM diplotypes and the *2 allele. Additional research is needed to improve the prediction of CYP2D6 activity from genetic data for individualized dosing of CYP2D6 dependent drugs.

## What is Already Known about this Subject

## What this Study Adds

## Introduction

CYP2D6 is one of the primary enzymes responsible for the metabolic activation or inactivation of exogenous compounds. It is estimated that approximately 25% of drugs are metabolized by CYP2D6 [1](#b1), including many analgesics, antidepressants and cytotoxics. There is substantial inter-patient variability in CYP2D6 activity, which is partly caused by genetic variation [2](#b2). The *CYP2D6* gene is highly polymorphic, with over a hundred allelic variants [3](#b3) and gene multiplications, deletions and other structural variants [4](#b4).

CYP2D6 activity, or phenotype, can be used to individualize dosing of certain drugs [5](#b5),[6](#b6). The extensive genetic heterogeneity in *CYP2D6* makes phenotype prediction very complicated. The most common *CYP2D6* alleles have been grouped by functional activity classifications with descending activity: ultra-rapid (*UM*), extensive (*EM*), intermediate (*IM*) or poor (*PM*) metabolism. A given patient has two alleles, giving them 10 possible allelic combinations, or diplotypes (*UM/UM*, *UM/EM*, *EM/EM*, etc.). These diplotypes are collapsed into four phenotypes, UM, EM, IM or PM, using either a translation table or a quantitative activity scoring system [7](#b7),[8](#b8).

Tamoxifen is a selective oestrogen receptor modulator used primarily in the treatment of oestrogen receptor positive breast cancer, and is the first line for use in pre-menopausal patients [9](#b9),[10](#b10). Tamoxifen and its major metabolite N-desmethyl-tamoxifen are weakly anti-oestrogenic, but are metabolically activated via CYP2D6 to the more potent anti-oestrogen endoxifen [11](#b11),[12](#b12). Patients with lower CYP2D6 metabolic activity have lower conversion of N-desmethyl-tamoxifen to endoxifen, which may be associated with decreased treatment effectiveness [13](#b13),[14](#b14). An immense research effort has been dedicated to determining whether patients with low activity *CYP2D6* genotypes receive less benefit from tamoxifen treatment [15](#b15)–[18](#b18). However, the results of these studies are highly controversial [19](#b19)–[21](#b21) and at this time *CYP2D6* genotyping is not recommended prior to tamoxifen initiation.

In a previous manuscript, we confirmed that steady-state endoxifen exposure is dependent on CYP2D6 phenotype and demonstrated that increasing the tamoxifen dose to 40 mg in IM (but not PM) patients normalizes the endoxifen concentrations with that of EM (or UM) patients remaining on 20 mg [22](#b22). Although the clinical usefulness of genotype-guided tamoxifen dosing remains unproven, this large dataset of steady-state metabolite data and comprehensive *CYP2D6* genotyping could be useful for assessing CYP2D6 phenotype prediction systems. The purpose of this analysis was to determine whether any of the diplotypes currently classified as IM phenotype should be reclassified, and to estimate the relative activity of each allele type (*UM*, *EM*, *IM*, *PM*) and each *EM* and *IM* allele to assess the calibration of the activity scoring systems currently in use.

## Methods

### Clinical cohort

Data included in this analysis were collected during the expansion phase of a prospective *CYP2D6*-genotype-guided tamoxifen dose escalation study, Lineberger Comprehensive Cancer Center (LCCC) 0801. The objective of the clinical study was to determine whether increasing the daily tamoxifen dose to 40 mg in patients with *CYP2D6* genotypes that predict reduced activity CYP2D6 phenotype (IM or PM as defined below) would normalize their endoxifen concentrations with that of patients with high predicted CYP2D6 activity phenotypes (EM or UM) treated with the standard 20 mg dose. Patients were treated with the genotype-determined dose for the 4 month duration of the study. Details of the inclusion criteria, study methodology and primary findings for the original 122 patient cohort have been previously published [22](#b22). Briefly, women 18 years or older who had been taking tamoxifen 20 mg for at least 4 months were eligible for enrolment. Relevant exclusion criteria included ECOG performance status >2, impaired kidney or liver function or concurrent use of a moderate or strong CYP2D6 inhibitor. This is the initial analysis of an expansion cohort that brought the total study enrolment to 500 patients. Due to a modification to the endoxifen assay occurring between analysis of the original and expansion cohort samples, only patients from the expansion cohort were eligible for this analysis (Supplementary Figure S[1](#fig01)). This study was approved by the institutional review boards of the University of North Carolina at Chapel Hill and all other participating institutions. All patients signed informed consent prior to participation.

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/42d4/4631184/83fc1350f140/bcp0080-1122-f1.jpg)

A) Baseline endoxifen concentration by CYP2D6 phenotype. Decreasing CYP2D6 activity phenotype was associated with decreased steady-state concentration of endoxifen (Kruskal–Wallis P < 0.0001). CYP2D6 phenotype explained more than 10% of the variability in endoxifen concentration. B) Baseline end: N-DM-tam ratio by CYP2D6 phenotype. Decreasing CYP2D6 activity phenotype was associated with decreased steady-state ratio of endoxifen to N-desmethyl-tamoxifen (Kruskal–Wallis P < 0.0001). CYP2D6 phenotype explained more than 18% of the variability in end: N-DM-tam ratio

### Genotyping and CYP2D6 phenotype activity prediction

Genomic DNA was purified from whole blood collected at study enrolment and genotyped in a CLIA-certified laboratory using the AmpliChip® CYP450 test (Roche Diagnostics, Indianapolis, IN, USA). This chip can identify 20 distinct *CYP2D6* alleles and seven known duplications (denoted by XN). Each of these alleles can be categorized by activity: *UM*, *EM*, *IM* or *PM* (Table[1A](#tbl1)), and the combination of any two alleles represents the patient's diplotype. These diplotypes are then collapsed into four phenotype categories (UM, EM, IM, or PM) (Table[1B](#tbl2)). This particular system for converting diplotype to phenotype is identical to that used in the prospective clinical study to determine which patients remained at 20 mg (UM and EM) and which received a dose increase (IM and PM). The phenotype classification system was based on the most convincing evidence of association between genotype and tamoxifen treatment outcomes from Schroth *et al*. [18](#b18). Because there were few UM patients (*n* = 5) this phenotype was excluded from phenotype comparisons. An additional four patients who carried *UM* alleles, but were not UM phenotype, were excluded from diplotype analyses which include only three phenotypes comprised of six diplotypes (EM: *EM*/*EM,* IM: *EM/IM, EM/PM, IM/IM, IM/PM* and PM: *PM/PM*).

### Table 1A.

| Activity | Genotypes | Allele |
| --- | --- | --- |
| High | *1XN, *2XN, *35XN | UM |
| Normal | *1, *2, and *35 | EM |
| Reduced | *9, *10, *17, *29, *36, *41, *17XN, *41XN | IM |
| None | *3, *4, *5, *6, *7, *8, *11,*15, *19, *20, *40, *4XN | PM |

Table 1 Caption: Transformation of CYP2D6 genotype to allelic activity

### Table 1B.

| Phenotype | Diplotypes |
| --- | --- |
| UM* | UM/UM†, UM/EM† |
| EM | EM/EM, UM/IM†, UM/PM† |
| IM | EM/IM, EM/PM, IM/IM, IM/PM |
| PM | PM/PM |

Table 1 Caption: Transformation of diplotype to activity phenotype

### Tamoxifen and metabolite concentrations

Blood samples were collected from each patient at study enrolment, when all patients had been on 20 mg tamoxifen for at least 4 months. Steady-state plasma concentrations of tamoxifen (Z isomer only), (Z)-4-hydroxy-tamoxifen (4-OH-tam), N-desmethyl-tamoxifen (N-DM-tam, Z isomer only) and (Z)-4-hydroxy-N-desmethyl-tamoxifen (endoxifen; 10% (E), 90% (Z)) were measured using a h.p.l.c.-MS/MS (API 3200) assay method developed using diphenhydramine as the internal standard and liquid–liquid extraction as previously described with minor modification [22](#b22). Tamoxifen, 4-OH-tam, N-DM-tam, endoxifen and diphenhydramine were stable for 28 h and the lower limit of quantification was 0.3 ng ml^–1^ for each analyte using 200 µl of plasma. Variability was minimized in the method by using glass tubes and formic acid as part of the mobile phase. E isomers of each analyte eluted prior to the Z isomer. This analysis focuses primarily on the baseline (at study enrolment, all patients on 20 mg) metabolic ratio of endoxifen to N-DM-tam (end: N-DM-tam). However, endoxifen concentrations at baseline and at 4 months (UM and EM patients on 20 mg, IM and PM on 40 mg) are used in some secondary analyses.

### Statistical methods

Endoxifen and end: N-DM-tam ratio descriptive data are reported using medians and standard deviations. Baseline demographic comparisons across phenotype groups (excluding UM) were carried out via Fisher's exact (categorical data) and Kruskal–Wallis (continuous data) tests. Comparisons of metabolite concentrations across phenotypes and diplotypes (excluding *UM* alleles) and estimation of the percentage of variance explained was carried out via analysis of variance (anova) and Kruskal–Wallis tests. The activity score modeling was performed using linear regression with each allele (*UM*, *EM*, *IM*, or *PM*) its own parameter. After initial modelling the activity of an *EM* allele was fixed to 1.00 and the relative activity of all other alleles were scaled accordingly. The analysis was then rerun with the *EM* (*1, *2, *35) and *IM* (*9, *10, *17, *29, *41/*41XN) alleles included individually to enable independent estimation of the N-DM-tam to endoxifen metabolic activity of each allele. Allele activity was compared within the EM and IM allele types using Kruskal–Wallis tests to determine whether any allele was distinct from the group. Within each allele type the mean activity of each individual allele was compared with that of the most common allele within that group, *1 for *EM* alleles and *41/*41XN for *IM* alleles.

## Results

After exclusion of six patients with inconclusive genotyping, 355 patients with EM (*n* = 119), IM (*n* = 214), PM (*n* = 17), or UM (*n* = 5) predicted CYP2D6 phenotype were eligible for inclusion in these analyses (Supplementary Figure S[1](#fig01)). Demographic data including age, weight, race, time on tamoxifen prior to study consent and concomitant CYP2D6 weak inhibitor use for the entire cohort and by CYP2D6 phenotype can be found in Table[2](#tbl3). As expected, race was unevenly distributed across CYP2D6 phenotypes (*P* = 0.006), as was the use of weak CYP2D6 inhibitors (*P* = 0.01). All other demographic factors were similar (*P* > 0.05). At baseline, with all patients receiving 20 mg tamoxifen, CYP2D6 phenotype was a significant predictor of steady-state endoxifen concentration (*P* < 0.0001, *r*^2^ = 0.11, Figure[1](#fig01)A) and end: N-DM-tam ratio (*P* < 0.0001, *r*^2^ = 0.19, Figure[2](#fig02)B). Because predicted CYP2D6 phenotype explained 80% more variance in the metabolic ratio, compared with endoxifen alone, the end: N-DM-tam ratio was the focus of subsequent analyses.

### Table 2.

|   |   | All patients (n = 355) | UM (n = 5) | EM (n = 119) | IM (n = 214) | PM (n = 17) | P value* |
| --- | --- | --- | --- | --- | --- | --- | --- |
| Age (years) | Median | 52 | 49 | 52 | 52 | 46 |   |
| Range | 24–95 | 43–61 | 30–95 | 24–88 | 38–78 | 0.17 |  |
| Weight (kg) | Median | 71.0 | 69.8 | 71.6 | 71.1 | 70.0 |   |
| Range | 32.1–149.1 | 48.6–78.7 | 44.2–149.1 | 32.1–135.4 | 47.6–140.7 | 0.93 |  |
| Race | White | 83.0% | 80% | 89.0% | 78.8% | 94.1% |   |
| Black | 14.7% | 20% | 6.8% | 19.8% | 5.9% |   |  |
| Other | 2.3% | 0% | 4.2% | 1.4% | 0% | 0.006 |  |
| Months on tamoxifen prior to study | Median | 10.1 | 10.1 | 11.1 | 9.9 | 13.1 |   |
| Range | 3.4–116.3 | 6.2–16.5 | 3.8–116.3 | 3.4–72.1 | 3.7–39.8 | 0.64 |  |
| CYP2D6 weak inhibitor use | Citalopram | 4.0% | 0% | 7.6% | 1.9% | 5.9% | 0.03 |
| Escitalopram | 4.0% | 20% | 5.9% | 2.4% | 5.9% | 0.16 |  |
| Venlafaxine | 9.1% | 0% | 11.8% | 8.0% | 5.9% | 0.53 |  |
| Other | 0.9% | 20% | 0.8% | 0.5% | 0% | 1.00 |  |

Table 2 Caption: Demographic data for all eligible patients (n = 355) and broken down by CYP2D6 phenotype

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/42d4/4631184/314a1f20bdae/bcp0080-1122-f2.jpg)

Baseline end: N-DM-tam ratio by CYP2D6 diplotype. Decreasing CYP2D6 diplotype was associated with decreased steady-state ratio of endoxifen to N-desmethyl-tamoxifen (Kruskal–Wallis P < 0.0001). Within the IM phenotype, the patients with the highest (EM/IM) and lowest (IM/PM) activity diplophenoytpes were distinct from patients with EM (EM/EM) or PM (PM/PM) activity, respectively

### Table 3.

|   |   |   | Baseline end: N-DM-tam ratio |  |  |  |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- |
| Phenotype | Diplotype | n (%) | Median | Standard deviation | P value*vs. EM/EM | 95% CI†vs. EM/EM | P value*vs. PM/PM | 95% CI‡vs. PM/PM |
| EM | EM/EM | 115 (33%) | 0.082 | 0.058 | NA | NA | <0.0001 | (0.058, 0.087) |
| IM | EM/IM | 69 (20%) | 0.061 | 0.030 | 0.0004 | (0.018, 0.044) | <0.0001 | (0.029, 0.054) |
| EM/PM | 89 (26%) | 0.045 | 0.043 | <0.0001 | (0.024, 0.052) | <0.0001 | (0.021, 0.048) |  |
| IM/IM | 19 (5%) | 0.031 | 0.019 | <0.0001 | (0.044, 0.72) | 0.0032 | (0.001, 0.028) |  |
| IM/PM | 37 (11%) | 0.024 | 0.033 | <0.0001 | (0.041, 0.072) | 0.029 | (0.001, 0.030) |  |
| PM | PM/PM | 17 (5%) | 0.017 | 0.020 | <0.0001 | (0.058, 0.087) | NA | NA |

Table 3 Caption: Comparison of baseline endoxifen: N-desmethyl-tamoxifen ratio by CYP2D6 diplotype

### Baseline differences between IM diplotypes

The 214 IM patients were further subdivided into four distinct diplotypes (*EM/IM: n* = 69, *EM/PM: n* = 89, *IM/IM: n* = 19, *IM/PM: n* = 37). The CYP2D6 activity, estimated by the end: N-DM-tam ratio when all patients were treated with 20 mg, varied significantly between these diplotypes (*P* < 0.0001). As expected, the ratio decreased with decreasing activity of the component alleles, from the highest activity *EM/IM* patients (median end: N-DM-tam ratio = 0.061) to the lowest activity *IM/PM* patients (end: N-DM-tam ratio = 0.024). When considering the end: N-DM-tam ratio in the IM phenotype group only, the diplotypes explained approximately 7.6% of the variance. Similarly, the diplotypes were significantly associated with baseline endoxifen concentration, though the amount of variance explained was somewhat lower (*r*^2^ = 0.047, *P* = 0.0027).

### Baseline differences between IM diplotype extremes and EM or PM phenotypes

Across *CYP2D6* diplotypes, the end: N-DM-tam ratio decreased with decreasing diplotype activity, as expected (*P* < 0.0001, Figure[3](#fig03)). Compared with EM phenotype (*EM/EM*) patients, the IM phenotype patients with the highest activity diplotype (*EM/IM*) had significantly lower median baseline end: N-DM-tam ratios (0.061 *vs*. 0.082, *P* = 0.0004, Table[3](#tbl3)) and endoxifen concentration (7.9 ng ml^–1^ *vs*. 8.9 ng ml^–1^, *P* = 0.033, Supplementary Table S1, Figure S[2](#fig02)). Similarly, when compared with PM (*PM/PM*) patients at baseline, the IM diplotype group with the lowest activity (*IM/PM)* had a greater end: N-DM-tam ratio (0.024 *vs*. 0.017, *P* = 0.029) and a trend toward greater endoxifen concentration (4.0 ng ml^–1^ *vs*. 2.4 ng ml^–1^, *P* = 0.068).

### Figure 3.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/42d4/4631184/c0eeb79a42d2/bcp0080-1122-f3.jpg)

Baseline end: N-DM-tam ratio by CYP2D6 allele. The EM and IM alleles were analyzed separately to estimate their individual activity. The EM alleles (*1, *2, *35) generally had greater activity. However, the *2 allele had activity more similar to an IM or PM allele. The IM alleles represent a continuum of activity that spans from activity similar to an EM allele (*9) to activity similar to a PM allele (*17). The UM alleles were combined due to small numbers and the PM alleles were combined because they create non-functional protein or no protein at all, and thus have no activity.

### Differences between IM diplotypes and EM phenotype at 4 months

As previously reported, the endoxifen concentration for the IM phenotype group on 40 mg was similar to that of the EM group on 20 mg at the 4 month time point [22](#b22), but the PM group on 40 mg did not achieve normalized concentration. The 4 month endoxifen concentration for the EM phenotype (*EM/EM*) patients on 20 mg was compared with that of the IM diplotypes on 40 mg. Only the *EM/PM* and *IM/IM* diplotypes had similar endoxifen concentrations after dose increase (*P* > 0.05), the *EM/IM* patients had significantly greater endoxifen concentrations (13.1 ng ml^–1^ *vs*. 8.2 ng ml^–1^, *P* < 0.0001), and the *IM/PM* patients remained significantly lower (5.8 ng ml^–1^ *vs*. 8.2 ng ml^–1^, *P* = 0.0009) (Supplementary Table S[s2](#sd3)).

### Modelling relative activity of allele types and alleles

Using each patient's two *CYP2D6* alleles and their baseline end: N-DM-tam metabolic ratio the relative activity of each allele type (*UM*, *EM*, *IM, PM*) was estimated. Fixing the activity of an *EM* allele to 1.00, the relative activity of a *UM*, *IM,* and *PM* allele was estimated to be 0.85, 0.67 and 0.52, respectively (Table [4](#tbl5)). The activity score modelling analysis was performed a second time with each *EM* (*1, *2, *35) and *IM* (*9, *10, *17, *29, *41/*41XN) allele included as an independent parameter. Individual alleles in the *EM* group had distinct metabolic activity (*P* < 0.0001). Fixing the wild-type (*1) allele activity to 1.00, the ***2 allele had an estimated scaled activity of 0.63, which was significantly different from the *1 allele (*P* = <0.0001, 95% CI 0.021, 0.041) while the *35 allele had activity of 1.03, which was similar to *1 (Figure[3](#fig03)). The individual *IM* alleles also had distinct metabolic activity (*P* = 0.014). Comparing each allele to the activity of the most common *41/*41XN allele (activity = 0.53) only *17 had a significantly different activity estimate (activity = 0.39, 95% CI 0.005, 0.030, *P* = 0.021). The scaled activity of the other alleles ranged from 0.58–0.85 (Table [5](#tbl6)). The scaled activity of the *UM* and *PM* alleles changed marginally to 0.71 and 0.43.

### Table 4.

| Allele type | n | Median | Standard deviation | Scaled activity* |
| --- | --- | --- | --- | --- |
| UM | 9 | 0.056 | 0.053 | 0.85 |
| EM | 390 | 0.066 | 0.053 | 1.00 |
| IM | 149 | 0.044 | 0.034 | 0.67 |
| PM | 162 | 0.034 | 0.039 | 0.52 |

Table 4 Caption: Activity for each allele type relative to an EM allele fixed at 1.00

### Table 5.

| Allele type | Allele | n | Median | Standard deviation | Scaled activity* |
| --- | --- | --- | --- | --- | --- |
| UM | Composite | 9 | 0.056 | 0.053 | 0.71 |
| EM | *1 (wild-type) | 250 | 0.079 | 0.058 | 1.00 |
| *2 | 109 | 0.050 | 0.036 | 0.63 |  |
| *35 | 31 | 0.081 | 0.046 | 1.03 |  |
| IM | *9 | 20 | 0.067 | 0.029 | 0.85 |
| *10 | 11 | 0.051 | 0.044 | 0.65 |  |
| *17 | 21 | 0.031 | 0.022 | 0.39 |  |
| *29 | 10 | 0.046 | 0.031 | 0.58 |  |
| *41/*41XN | 87 | 0.042 | 0.035 | 0.53 |  |
| PM | Composite | 162 | 0.034 | 0.039 | 0.43 |

Table 5 Caption: Activity for each EM and IM allele relative to a wild-type *1 allele fixed at 1.00

## Discussion

The original intent of this prospective genotype-guided tamoxifen escalation study was to determine whether increasing IM or PM patients to 40 mg achieved endoxifen concentrations similar to EM patients maintained on 20 mg. As a composite phenotype group the IM patients achieved similar endoxifen concentrations, but the PM patients remained below EM patients [22](#b22). In this analysis the IM diplotypes were analyzed separately. Doubling the dose successfully achieved concentrations similar to those of EM patients only for the *EM/PM* and *IM/IM* diplotypes. The highest activity *EM/IM* patients may require a smaller dose increase, as their endoxifen concentrations exceeded those of the EM group. In some phenotype classification systems the *EM/IM* and *EM/PM* diplotypes are collapsed with the EM, not the IM, phenotype. These patients had endoxifen concentrations lower than the *EM/EM* patients, suggesting that they may be underdosed if they were considered EM patients and did not receive a dose adjustment. Alternatively, this would prevent potential over-treatment in the *EM/IM* patients. Increasing the lowest activity *IM/PM* patients to the maximum FDA-approved 40 mg dose is insufficient to achieve endoxifen concentrations similar to that of EM patients. Though genotype-guided tamoxifen dose escalation is not clinically warranted at this time, our results indicate that *CYP2D6* genotype-guided treatment algorithms should avoid collapsing the distinct IM diplotypes into a composite IM phenotype group.

Steady-state concentrations of tamoxifen metabolites were used to evaluate CYP2D6 activity scoring systems that categorize patients into four phenotypes (UM, EM, IM, PM). The metabolic ratio of end: N-DM-tam was used as it was nearly twice as reflective of CYP2D6 activity as endoxifen alone, and may be more predictive of tamoxifen treatment efficacy [14](#b14). These results further suggest that the IM phenotype is comprised of four diplotypes (*EM/IM, EM/PM, IM/IM, IM/PM*) that have metabolic activity distinct from each other and from the other CYP2D6 phenotype groups (EM or PM). Several CYP2D6 activity scoring systems that return sometimes discordant phenotypes have been proposed [23](#b23). The Clinical Pharmacogenetics Implementation Consortium (CPIC), which publishes genetics-guided treatment recommendations [24](#b24), has endorsed a system that assigns each allele an activity score of 0 (*PM*), 0.5 (*IM*) or 1 (*EM*), which is multiplied in the event of a gene duplication [5](#b5). In this system *EM/PM* (1 + 0 = 1.0) and *IM/IM* (0.5 + 0.5 = 1.0) patients would have equivalent activity, but *EM/PM* patients had on average 45% greater activity in this analysis (end: N-DM-tam 0.045 *vs*. 0.031, *P* = 0.018). Using a scaled regression model the activity for an *IM* allele (0.67) was similar to that (0.5) used in existing systems. Interestingly, the empirical activity estimate of a *PM* allele (0.52) was much greater than previously assumed. PM alleles have no CYP2D6 enzyme activity as they are gene deletions or non-functional truncated proteins. The activity estimated in this study is likely attributable to alternative pathways for endoxifen formation including CYP3A4/5 and CYP2C9/19 [25](#b25).

The activity of each *EM* and *IM* allele was then estimated and scaled to the activity of the wild-type *1 allele. The relatively common ***2 genotype (allele frequency in this cohort =15.4%) is typically characterized as an *EM* allele (activity = 1.0) but had an activity of only 0.63, which is much closer in activity to an *IM* (0.67) allele. The *in vitro* conversion of N-DM-tam to endoxifen for *CYP2D6**2 has been estimated as only 21% of the wild-type (***1) enzyme [26](#b26) and a dextromethorphan pharmacokinetic study reported that *CYP2D6**2 had approximately 40% of the activity of the wild-type [27](#b27). Recently, the ***2 SNP was reported to alter exonic splicing and reduce gene expression [28](#b28). Our results for tamoxifen bioconversion are consistent with prior results for dextromethorphan metabolism, further supporting the need for reclassification of ***2 as an intermediate function allele. Unlike *2, the *35 *EM* allele had activity very similar to *1 (activity = 1.03) and should continue to be considered a fully functional allele. There was a wide range of activity within the *IM* alleles, with estimates as low as 0.39 (*17) and as high as 0.85 (*9). There is statistical evidence that these alleles have distinct activity but the small number of some alleles makes it difficult to confirm which actually differ. Similar to the previously discussed results for *2, ideally the activity for each allele would be estimated *in vitro* and confirmed in healthy controls in a pharmacokinetic study using targeted pharmacogenetic enrolment.

There are several important limitations of this analysis. Tamoxifen is metabolized by several enzymes and the conversion to endoxifen has an alternate pathway. The estimates in this analysis, such as the ‘activity’ seen in non-functional *PM* alleles, reflect not just CYP2D6 activity but also this alternative pathway. Unfortunately, due to the small number of patients with UM phenotype or *UM* alleles, it was not possible to assess whether the activity of these diplotypes (UM: *UM/UM* and *UM/EM,* EM*: UM/IM* and *UM/PM*) are distinct, should continue to be collapsed into UM and EM phenotypes, or should be collapsed into a single EM phenotype. Another limitation of this analysis is the allele coverage of the AmpliChip CYP450 Test™. This test only reports that a copy number variant was detected, not the actual number of copies of the gene, and is unable to detect certain rare variants [29](#b29). Additionally, some newly discovered functionally consequential variants [30](#b30) and the 2988G > A variant that defines the ***41 haplotype [28](#b28) are missing from this panel. This causes some ***41 alleles to be erroneously called ***2, which would artificially depress the estimate of ***2 activity in this analysis. One final limitation is that the genotype explained a relatively small proportion of the variability in end: N-DM-tam ratio (18.6%), compared with that in prior analyses (53–69%) [14](#b14),[25](#b25). Log-transformation marginally increased the variability explained (24%) but to improve interpretation of results this was not done for the reported analyses. Several other factors including inhibitor co-administration, patient size, and perhaps even the season of sample collection, may influence systemic concentrations of tamoxifen metabolites [31](#b31). Patients taking strong or moderate CYP2D6 inhibitors were excluded from this study but these other factors were not accounted for in drug dosing or in this analysis. Additional clinical and genetic factors, including variants in other enzymes and transporters [32](#b32), will be included in follow-up analyses attempting to validate and extend a previously published endoxifen prediction algorithm [31](#b31).

In conclusion, pharmacogenetic–pharmacokinetic analysis of steady-state tamoxifen metabolite concentrations confirms that predicted CYP2D6 phenotypes are comprised of metabolically distinct diplotypes that should not be collapsed in genotype-guided dosing algorithms. Furthermore, currently used activity scoring systems are not optimally calibrated, specifically in regards to the ***2 genotype which does not have full activity. Due to the inconsistent findings regarding the relationship of *CYP2D6* genotype and tamoxifen efficacy, it is unclear whether these results will be useful for personalizing dosing of tamoxifen. However, these results highlight the need for further research to calibrate optimally CYP2D6 phenotype systems to guide personalized dosing of other CYP2D6-dependent drugs such as codeine [5](#b5) and tricyclic anti-depressants [6](#b6).

## Competing Interests

All authors have completed the Unified Competing Interest form at [http://www.icmje.org/coi_disclosure.pdf](http://www.icmje.org/coi_disclosure.pdf) (available on request from the corresponding author) and declare no support from any organization for the submitted work; SMA is employed by and owns stock in Laboratory Corporation of America in the previous 3 years, JMP has received research funding, was a consultant, and received honoraria from Novartis, GlaxoSmithKline and Genetech, DAF has received research funding from Pfizer and Novartis and was a consultant for Medco Health Solutions and Coriell, HLM has received research funding from Myriad Genetics, reports personal fees and non-financial support from Cancer Genetics inc, and was a consultant for Myriad Genetics, the FDA, Medco Health Solutions and Gentris and KEW was a consultant for Roche Diagnostics.

*This study was supported by Grant No. CA58223 from the National Cancer Institute Specialized Programs of Research Excellence, North Carolina University Cancer Research Fund, University of North Carolina at Chapel Hill Investments for the Future Grant No. 6231, Laboratory Corporation of America, Roche Diagnostics, American Society of Clinical Oncology Foundation and Breast Cancer Research Foundation, National Institute of General Medical Sciences Pharmacogenomics Research Network Award (U-01GM061373) to the Consortium on Breast Cancer Pharmacogenomics*.

## Supporting Information

Additional Supporting Information may be found in the online version of this article at the publisher's web-site:

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Bernard S, Neville KA, Nguyen AT, Flockhart DA. Interethnic differences in genetic polymorphisms of CYP2D6 in the U.S. population: Clinical implications. Oncologist. 2006;11:126–35. doi: 10.1634/theoncologist.11-2-126.  [DOI](https://doi.org/10.1634/theoncologist.11-2-126) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16476833/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncologist&title=Interethnic%20differences%20in%20genetic%20polymorphisms%20of%20CYP2D6%20in%20the%20U.S.%20population:%20Clinical%20implications&author=S%20Bernard&author=KA%20Neville&author=AT%20Nguyen&author=DA%20Flockhart&volume=11&publication_year=2006&pages=126-35&pmid=16476833&doi=10.1634/theoncologist.11-2-126&)

2. Gaedigk A. Complexities of CYP2D6 gene analysis and interpretation. Int Rev Psychiatry. 2013;25:534–53. doi: 10.3109/09540261.2013.825581.  [DOI](https://doi.org/10.3109/09540261.2013.825581) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24151800/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20Rev%20Psychiatry&title=Complexities%20of%20CYP2D6%20gene%20analysis%20and%20interpretation&author=A%20Gaedigk&volume=25&publication_year=2013&pages=534-53&pmid=24151800&doi=10.3109/09540261.2013.825581&)

3. Ingelman-Sundberg M, Daly AK, Nebert DW. 2010. Home Page of the Human Cytochrome P450 (CYP) Allele Nomenclature Database 2008.

4. Meijerman I, Sanderson LM, Smits PH, Beijnen JH, Schellens JH. Pharmacogenetic screening of the gene deletion and duplications of CYP2D6. Drug Metab Rev. 2007;39:45–60. doi: 10.1080/03602530600952206.  [DOI](https://doi.org/10.1080/03602530600952206) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17364880/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Rev&title=Pharmacogenetic%20screening%20of%20the%20gene%20deletion%20and%20duplications%20of%20CYP2D6&author=I%20Meijerman&author=LM%20Sanderson&author=PH%20Smits&author=JH%20Beijnen&author=JH%20Schellens&volume=39&publication_year=2007&pages=45-60&pmid=17364880&doi=10.1080/03602530600952206&)

5. Crews KR, Gaedigk A, Dunnenberger HM, Klein TE, Shen DD, Callaghan JT, Kharasch ED, Skaar TC. Clinical Pharmacogenetics Implementation Consortium. Clinical Pharmacogenetics Implementation Consortium (CPIC) guidelines for codeine therapy in the context of cytochrome P450 2D6 (CYP2D6) genotype. Clin Pharmacol Ther. 2012;91:321–6. doi: 10.1038/clpt.2011.287.  [DOI](https://doi.org/10.1038/clpt.2011.287) | [PMC free article](/articles/PMC3289963/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22205192/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Clinical%20Pharmacogenetics%20Implementation%20Consortium.%20Clinical%20Pharmacogenetics%20Implementation%20Consortium%20(CPIC)%20guidelines%20for%20codeine%20therapy%20in%20the%20context%20of%20cytochrome%20P450%202D6%20(CYP2D6)%20genotype&author=KR%20Crews&author=A%20Gaedigk&author=HM%20Dunnenberger&author=TE%20Klein&author=DD%20Shen&volume=91&publication_year=2012&pages=321-6&pmid=22205192&doi=10.1038/clpt.2011.287&)

6. Hicks JK, Swen JJ, Thorn CF, Sangkuhl K, Kharasch ED, Ellingrod VL, Skaar TC, Muller DJ, Gaedigk A, Stingl JC. Clinical Pharmacogenetics Implementation Consortium Guideline for CYP2D6 and CYP2C19 genotypes and dosing of tricyclic antidepressants. Clin Pharmacol Ther. 2013;93:402–8. doi: 10.1038/clpt.2013.2.  [DOI](https://doi.org/10.1038/clpt.2013.2) | [PMC free article](/articles/PMC3689226/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23486447/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Clinical%20Pharmacogenetics%20Implementation%20Consortium%20Guideline%20for%20CYP2D6%20and%20CYP2C19%20genotypes%20and%20dosing%20of%20tricyclic%20antidepressants&author=JK%20Hicks&author=JJ%20Swen&author=CF%20Thorn&author=K%20Sangkuhl&author=ED%20Kharasch&volume=93&publication_year=2013&pages=402-8&pmid=23486447&doi=10.1038/clpt.2013.2&)

7. Gaedigk A, Simon SD, Pearce RE, Bradford LD, Kennedy MJ, Leeder JS. The CYP2D6 activity score: translating genotype information into a qualitative measure of phenotype. Clin Pharmacol Ther. 2008;83:234–42. doi: 10.1038/sj.clpt.6100406.  [DOI](https://doi.org/10.1038/sj.clpt.6100406) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17971818/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=The%20CYP2D6%20activity%20score:%20translating%20genotype%20information%20into%20a%20qualitative%20measure%20of%20phenotype&author=A%20Gaedigk&author=SD%20Simon&author=RE%20Pearce&author=LD%20Bradford&author=MJ%20Kennedy&volume=83&publication_year=2008&pages=234-42&pmid=17971818&doi=10.1038/sj.clpt.6100406&)

8. Borges S, Desta Z, Jin Y, Faouzi A, Robarge JD, Philip S, Nguyen A, Stearns V, Hayes D, Rae JM, Skaar TC, Flockhart DA, Li L. Composite functional genetic and comedication CYP2D6 activity score in predicting tamoxifen drug exposure among breast cancer patients. J Clin Pharmacol. 2010;50:450–8. doi: 10.1177/0091270009359182.  [DOI](https://doi.org/10.1177/0091270009359182) | [PMC free article](/articles/PMC3816977/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20081063/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Pharmacol&title=Composite%20functional%20genetic%20and%20comedication%20CYP2D6%20activity%20score%20in%20predicting%20tamoxifen%20drug%20exposure%20among%20breast%20cancer%20patients&author=S%20Borges&author=Z%20Desta&author=Y%20Jin&author=A%20Faouzi&author=JD%20Robarge&volume=50&publication_year=2010&pages=450-8&pmid=20081063&doi=10.1177/0091270009359182&)

9. Early Breast Cancer Trialists' Collaborative Group. Tamoxifen for early breast cancer: an overview of the randomised trials. The Lancet. 1998;351:1451–67.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/9605801/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=The%20Lancet&title=Tamoxifen%20for%20early%20breast%20cancer:%20an%20overview%20of%20the%20randomised%20trials&volume=351&publication_year=1998&pages=1451-67&pmid=9605801&)

10. Early Breast Cancer Trialists' Collaborative Group. Relevance of breast cancer hormone receptors and other factors to the efficacy of adjuvant tamoxifen: patient-level meta-analysis of randomised trials. Lancet. 2011;378:771–84. doi: 10.1016/S0140-6736(11)60993-8.  [DOI](https://doi.org/10.1016/S0140-6736(11)60993-8) | [PMC free article](/articles/PMC3163848/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21802721/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Relevance%20of%20breast%20cancer%20hormone%20receptors%20and%20other%20factors%20to%20the%20efficacy%20of%20adjuvant%20tamoxifen:%20patient-level%20meta-analysis%20of%20randomised%20trials&volume=378&publication_year=2011&pages=771-84&pmid=21802721&doi=10.1016/S0140-6736(11)60993-8&)

11. Stearns V, Johnson MD, Rae JM, Morocho A, Novielli A, Bhargava P, Hayes DF, Desta Z, Flockhart DA. Active tamoxifen metabolite plasma concentrations after coadministration of tamoxifen and the selective serotonin reuptake inhibitor paroxetine. J Natl Cancer Inst. 2003;95:1758–64. doi: 10.1093/jnci/djg108.  [DOI](https://doi.org/10.1093/jnci/djg108) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14652237/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Natl%20Cancer%20Inst&title=Active%20tamoxifen%20metabolite%20plasma%20concentrations%20after%20coadministration%20of%20tamoxifen%20and%20the%20selective%20serotonin%20reuptake%20inhibitor%20paroxetine&author=V%20Stearns&author=MD%20Johnson&author=JM%20Rae&author=A%20Morocho&author=A%20Novielli&volume=95&publication_year=2003&pages=1758-64&pmid=14652237&doi=10.1093/jnci/djg108&)

12. Desta Z, Ward BA, Soukhova NV, Flockhart DA. Comprehensive evaluation of tamoxifen sequential biotransformation by the human cytochrome P450 system in vitro: prominent roles for CYP3A and CYP2D6. J Pharmacol Exp Ther. 2004;310:1062–75. doi: 10.1124/jpet.104.065607.  [DOI](https://doi.org/10.1124/jpet.104.065607) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15159443/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharmacol%20Exp%20Ther&title=Comprehensive%20evaluation%20of%20tamoxifen%20sequential%20biotransformation%20by%20the%20human%20cytochrome%20P450%20system%20in%20vitro:%20prominent%20roles%20for%20CYP3A%20and%20CYP2D6&author=Z%20Desta&author=BA%20Ward&author=NV%20Soukhova&author=DA%20Flockhart&volume=310&publication_year=2004&pages=1062-75&pmid=15159443&doi=10.1124/jpet.104.065607&)

13. Madlensky L, Natarajan L, Tchu S, Pu M, Mortimer J, Flatt SW, Nikoloff DM, Hillman G, Fontecha MR, Lawrence HJ, Parker BA, Wu AHB, Pierce JP. Tamoxifen metabolite concentrations, CYP2D6 genotype, and breast cancer outcomes. Clin Pharmacol Ther. 2011;89:718–25. doi: 10.1038/clpt.2011.32.  [DOI](https://doi.org/10.1038/clpt.2011.32) | [PMC free article](/articles/PMC3081375/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21430657/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Tamoxifen%20metabolite%20concentrations,%20CYP2D6%20genotype,%20and%20breast%20cancer%20outcomes&author=L%20Madlensky&author=L%20Natarajan&author=S%20Tchu&author=M%20Pu&author=J%20Mortimer&volume=89&publication_year=2011&pages=718-25&pmid=21430657&doi=10.1038/clpt.2011.32&)

14. Saladores P, Murdter T, Eccles D, Chowbay B, Zgheib NK, Winter S, Ganchev B, Eccles B, Gerty S, Tfayli A, Lim JSL, Yap YS, Ng RCH, Wong NS, Dent R, Habbal MZ, Schaeffeler E, Eichelbaum M, Schroth W, Schwab M, Brauch H. Tamoxifen metabolism predicts drug concentrations and outcome in premenopausal patients with early breast cancer. Pharmacogenomics J. 2015;15:84–94. doi: 10.1038/tpj.2014.34.  [DOI](https://doi.org/10.1038/tpj.2014.34) | [PMC free article](/articles/PMC4308646/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25091503/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=Tamoxifen%20metabolism%20predicts%20drug%20concentrations%20and%20outcome%20in%20premenopausal%20patients%20with%20early%20breast%20cancer&author=P%20Saladores&author=T%20Murdter&author=D%20Eccles&author=B%20Chowbay&author=NK%20Zgheib&volume=15&publication_year=2015&pages=84-94&pmid=25091503&doi=10.1038/tpj.2014.34&)

15. Regan MM, Leyland-Jones B, Bouzyk M, Pagani O, Tang W, Kammler R, Dell'orto P, Biasi MO, Thurlimann B, Lyng MB, Ditzel HJ, Neven P, Debled M, Maibach R, Price KN, Gelber RD, Coates AS, Goldhirsch A, Rae JM, Viale G. Breast International Group (BIG) 1–98 Collaborative Group. CYP2D6 genotype and tamoxifen response in postmenopausal women with endocrine-responsive breast cancer: the Breast International Group 1–98 trial. J Natl Cancer Inst. 2012;104:441–51. doi: 10.1093/jnci/djs125.  [DOI](https://doi.org/10.1093/jnci/djs125) | [PMC free article](/articles/PMC3309132/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22395644/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Natl%20Cancer%20Inst&title=Breast%20International%20Group%20(BIG)%201%E2%80%9398%20Collaborative%20Group.%20CYP2D6%20genotype%20and%20tamoxifen%20response%20in%20postmenopausal%20women%20with%20endocrine-responsive%20breast%20cancer:%20the%20Breast%20International%20Group%201%E2%80%9398%20trial&author=MM%20Regan&author=B%20Leyland-Jones&author=M%20Bouzyk&author=O%20Pagani&author=W%20Tang&volume=104&publication_year=2012&pages=441-51&pmid=22395644&doi=10.1093/jnci/djs125&)

16. Leyland-Jones B, Regan MM, Bouzyk M, Kammler R, Tang W, Pagani O, Maibach R, Dell'Orto P, Thurlimann B, Price KN, Viale G. 2010. Outcome According to CYP2D6 Genotype among Postmenopausal Women with Endocrine-Responsive Early Invasive Breast Cancer Randomized inthe BIG 1–98 Trial. SABCS; General Session 1.

17. Goetz MP, Suman VJ, Hoskin TL, Gnant M, Filipits M, Safgren SL, Kuffel M, Jakesz R, Rudas M, Greil R, Dietze O, Lang A, Offner F, Reynolds CA, Weinshilboum RM, Ames MM, Ingle JN. CYP2D6 metabolism and patient outcome in the Austrian Breast and Colorectal Cancer Study Group trial (ABCSG) 8. Clin Cancer Res. 2013;19:500–7. doi: 10.1158/1078-0432.CCR-12-2153.  [DOI](https://doi.org/10.1158/1078-0432.CCR-12-2153) | [PMC free article](/articles/PMC3548984/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23213055/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&title=CYP2D6%20metabolism%20and%20patient%20outcome%20in%20the%20Austrian%20Breast%20and%20Colorectal%20Cancer%20Study%20Group%20trial%20(ABCSG)%208&author=MP%20Goetz&author=VJ%20Suman&author=TL%20Hoskin&author=M%20Gnant&author=M%20Filipits&volume=19&publication_year=2013&pages=500-7&pmid=23213055&doi=10.1158/1078-0432.CCR-12-2153&)

18. Schroth W, Goetz MP, Hamann U, Fasching PA, Schmidt M, Winter S, Fritz P, Simon W, Suman VJ, Ames MM, Safgren SL, Kuffel MJ, Ulmer HU, Bolander J, Strick R, Beckmann MW, Koelbl H, Weinshilboum RM, Ingle JN, Eichelbaum M, Schwab M, Brauch H. Association between CYP2D6 polymorphisms and outcomes among women with early stage breast cancer treated with tamoxifen. JAMA. 2009;302:1429–36. doi: 10.1001/jama.2009.1420.  [DOI](https://doi.org/10.1001/jama.2009.1420) | [PMC free article](/articles/PMC3909953/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19809024/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA&title=Association%20between%20CYP2D6%20polymorphisms%20and%20outcomes%20among%20women%20with%20early%20stage%20breast%20cancer%20treated%20with%20tamoxifen&author=W%20Schroth&author=MP%20Goetz&author=U%20Hamann&author=PA%20Fasching&author=M%20Schmidt&volume=302&publication_year=2009&pages=1429-36&pmid=19809024&doi=10.1001/jama.2009.1420&)

19. Hertz DL, McLeod HL, Irvin WJ. Tamoxifen and CYP2D6: A contradiction of data. Oncologist. 2012;17:620–30. doi: 10.1634/theoncologist.2011-0418.  [DOI](https://doi.org/10.1634/theoncologist.2011-0418) | [PMC free article](/articles/PMC3360902/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22531359/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncologist&title=Tamoxifen%20and%20CYP2D6:%20A%20contradiction%20of%20data&author=DL%20Hertz&author=HL%20McLeod&author=WJ%20Irvin&volume=17&publication_year=2012&pages=620-30&pmid=22531359&doi=10.1634/theoncologist.2011-0418&)

20. Province MA, Goetz MP, Brauch H, Flockhart DA, Hebert JM, Whaley R, Suman VJ, Schroth W, Winter S, Zembutsu H, Mushiroda T, Newman WG, Lee MM, Ambrosone CB, Beckmann MW, Choi J, Dieudonne A, Fasching PA, Ferraldeschi R, Gong L, Haschke-Becher E, Howell A, Jordan LB, Hamann U, Kiyotani K, Krippl P, Lambrechts D, Latif A, Langsenlehner U, Lorizio W, Neven P, Nguyen AT, Park B, Purdie CA, Quinlan P, Renner W, Schmidt M, Schwab M, Shin J, Stingl JC, Wegman P, Wingren S, Wu AHB, Ziv E, Zirpoli G, Thompson AM, Jordan VC, Nakamura Y, Altman RB, Ames MM, Weinshilboum RM, Eichelbaum M, Ingle JN, Klein TE. CYP2D6 genotype and adjuvant tamoxifen: meta-analysis of heterogeneous study populations. Clin Pharmacol Ther. 2013 doi: 10.1038/clpt.2013.186. ; doi: 10.1038/clpt.2013.186. [Epub ahead of print]  [DOI](https://doi.org/10.1038/clpt.2013.186) | [DOI](https://doi.org/10.1038/clpt.2013.186) | [PMC free article](/articles/PMC3904554/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24060820/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=CYP2D6%20genotype%20and%20adjuvant%20tamoxifen:%20meta-analysis%20of%20heterogeneous%20study%20populations&author=MA%20Province&author=MP%20Goetz&author=H%20Brauch&author=DA%20Flockhart&author=JM%20Hebert&publication_year=2013&pmid=24060820&doi=10.1038/clpt.2013.186&)

21. Kiyotani K, Mushiroda T, Zembutsu H, Nakamura Y. Important and critical scientific aspects in pharmacogenomics analysis: lessons from controversial results of tamoxifen and CYP2D6 studies. J Hum Genet. 2013;58:327–33. doi: 10.1038/jhg.2013.39.  [DOI](https://doi.org/10.1038/jhg.2013.39) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23657426/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Hum%20Genet&title=Important%20and%20critical%20scientific%20aspects%20in%20pharmacogenomics%20analysis:%20lessons%20from%20controversial%20results%20of%20tamoxifen%20and%20CYP2D6%20studies&author=K%20Kiyotani&author=T%20Mushiroda&author=H%20Zembutsu&author=Y%20Nakamura&volume=58&publication_year=2013&pages=327-33&pmid=23657426&doi=10.1038/jhg.2013.39&)

22. Irvin WJ, Walko CM, Weck KE, Ibrahim JG, Chiu WK, Dees EC, Moore SG, Olajide OA, Graham ML, Canale ST, Raab RE, Corso SW, Peppercorn JM, Anderson SM, Friedman KJ, Ogburn ET, Desta Z, Flockhart DA, McLeod HL, Evans JP, Carey LA. Genotype-guided tamoxifen dosing increases active metabolite exposure in women with rReduced CYP2D6 metabolism: a multicenter study. J Clin Oncol. 2011;29:3232–9. doi: 10.1200/JCO.2010.31.4427.  [DOI](https://doi.org/10.1200/JCO.2010.31.4427) | [PMC free article](/articles/PMC3158597/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21768473/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Genotype-guided%20tamoxifen%20dosing%20increases%20active%20metabolite%20exposure%20in%20women%20with%20rReduced%20CYP2D6%20metabolism:%20a%20multicenter%20study&author=WJ%20Irvin&author=CM%20Walko&author=KE%20Weck&author=JG%20Ibrahim&author=WK%20Chiu&volume=29&publication_year=2011&pages=3232-9&pmid=21768473&doi=10.1200/JCO.2010.31.4427&)

23. Hicks JK, Swen JJ, Gaedigk A. Challenges in CYP2D6 phenotype assignment from genotype data: a critical assessment and call for standardization. Curr Drug Metab. 2014;15:218–32. doi: 10.2174/1389200215666140202215316.  [DOI](https://doi.org/10.2174/1389200215666140202215316) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24524666/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Drug%20Metab&title=Challenges%20in%20CYP2D6%20phenotype%20assignment%20from%20genotype%20data:%20a%20critical%20assessment%20and%20call%20for%20standardization&author=JK%20Hicks&author=JJ%20Swen&author=A%20Gaedigk&volume=15&publication_year=2014&pages=218-32&pmid=24524666&doi=10.2174/1389200215666140202215316&)

24. Relling MV, Klein TE. CPIC: Clinical Pharmacogenetics Implementation Consortium of the Pharmacogenomics Research Network. Clin Pharmacol Ther. 2011;89:464–7. doi: 10.1038/clpt.2010.279.  [DOI](https://doi.org/10.1038/clpt.2010.279) | [PMC free article](/articles/PMC3098762/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21270786/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=CPIC:%20Clinical%20Pharmacogenetics%20Implementation%20Consortium%20of%20the%20Pharmacogenomics%20Research%20Network&author=MV%20Relling&author=TE%20Klein&volume=89&publication_year=2011&pages=464-7&pmid=21270786&doi=10.1038/clpt.2010.279&)

25. Murdter TE, Schroth W, Bacchus-Gerybadze L, Winter S, Heinkele G, Simon W, Fasching PA, Fehm T, Eichelbaum M, Schwab M, Brauch H. Activity levels of Tamoxifen metabolites at the estrogen receptor and the impact of genetic polymorphisms of phase I and II enzymes on their concentration levels in plasma. Clin Pharmacol Ther. 2011;89:708–17. doi: 10.1038/clpt.2011.27.  [DOI](https://doi.org/10.1038/clpt.2011.27) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21451508/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Activity%20levels%20of%20Tamoxifen%20metabolites%20at%20the%20estrogen%20receptor%20and%20the%20impact%20of%20genetic%20polymorphisms%20of%20phase%20I%20and%20II%20enzymes%20on%20their%20concentration%20levels%20in%20plasma&author=TE%20Murdter&author=W%20Schroth&author=L%20Bacchus-Gerybadze&author=S%20Winter&author=G%20Heinkele&volume=89&publication_year=2011&pages=708-17&pmid=21451508&doi=10.1038/clpt.2011.27&)

26. Muroi Y, Takahashi M, Saito T, Sakuyama K, Niinuma Y, Ito M, Tsukada C, Ohta K, Endo Y, Oda A, Hirasawa N, Hiratsuka M. Functional characterization of wild-type and 49 CYP2D6 allelic variants for N-desmethyltamoxifen 4-hydroxylation activity. Drug Metab Pharmacokinet. 2014;29:360–6. doi: 10.2133/dmpk.dmpk-14-rg-014.  [DOI](https://doi.org/10.2133/dmpk.dmpk-14-rg-014) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24647041/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Pharmacokinet&title=Functional%20characterization%20of%20wild-type%20and%2049%20CYP2D6%20allelic%20variants%20for%20N-desmethyltamoxifen%204-hydroxylation%20activity&author=Y%20Muroi&author=M%20Takahashi&author=T%20Saito&author=K%20Sakuyama&author=Y%20Niinuma&volume=29&publication_year=2014&pages=360-6&pmid=24647041&doi=10.2133/dmpk.dmpk-14-rg-014&)

27. Abduljalil K, Frank D, Gaedigk A, Klaassen T, Tomalik-Scharte D, Jetter A, Jaehde U, Kirchheiner J, Fuhr U. Assessment of activity levels for CYP2D6*1, CYP2D6*2, and CYP2D6*41 genes by population pharmacokinetics of dextromethorphan. Clin Pharmacol Ther. 2010;88:643–51. doi: 10.1038/clpt.2010.137.  [DOI](https://doi.org/10.1038/clpt.2010.137) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20881950/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Assessment%20of%20activity%20levels%20for%20CYP2D6*1,%20CYP2D6*2,%20and%20CYP2D6*41%20genes%20by%20population%20pharmacokinetics%20of%20dextromethorphan&author=K%20Abduljalil&author=D%20Frank&author=A%20Gaedigk&author=T%20Klaassen&author=D%20Tomalik-Scharte&volume=88&publication_year=2010&pages=643-51&pmid=20881950&doi=10.1038/clpt.2010.137&)

28. Wang D, Poi MJ, Sun X, Gaedigk A, Leeder JS, Sadee W. Common CYP2D6 polymorphisms affecting alternative splicing and transcription: long-range haplotypes with two regulatory variants modulate CYP2D6 activity. Hum Mol Genet. 2014;23:268–78. doi: 10.1093/hmg/ddt417.  [DOI](https://doi.org/10.1093/hmg/ddt417) | [PMC free article](/articles/PMC3857955/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23985325/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mol%20Genet&title=Common%20CYP2D6%20polymorphisms%20affecting%20alternative%20splicing%20and%20transcription:%20long-range%20haplotypes%20with%20two%20regulatory%20variants%20modulate%20CYP2D6%20activity&author=D%20Wang&author=MJ%20Poi&author=X%20Sun&author=A%20Gaedigk&author=JS%20Leeder&volume=23&publication_year=2014&pages=268-78&pmid=23985325&doi=10.1093/hmg/ddt417&)

29. Gaedigk A, Garcia-Ribera C, Jeong HE, Shin JG, Hernandez-Sanchez JT. Resolution of a clinical AmpliChip CYP450 Test no call: discovery and characterization of novel CYP2D6*1 haplotypes. Pharmacogenomics. 2014;15:1175–84. doi: 10.2217/pgs.14.94.  [DOI](https://doi.org/10.2217/pgs.14.94) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25141893/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Resolution%20of%20a%20clinical%20AmpliChip%20CYP450%20Test%20no%20call:%20discovery%20and%20characterization%20of%20novel%20CYP2D6*1%20haplotypes&author=A%20Gaedigk&author=C%20Garcia-Ribera&author=HE%20Jeong&author=JG%20Shin&author=JT%20Hernandez-Sanchez&volume=15&publication_year=2014&pages=1175-84&pmid=25141893&doi=10.2217/pgs.14.94&)

30. Wang D, Papp AC, Sun X. Functional characterization of CYP2D6 enhancer polymorphisms. Hum Mol Genet. 2015;24:1556–62. doi: 10.1093/hmg/ddu566.  [DOI](https://doi.org/10.1093/hmg/ddu566) | [PMC free article](/articles/PMC4381757/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25381333/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mol%20Genet&title=Functional%20characterization%20of%20CYP2D6%20enhancer%20polymorphisms&author=D%20Wang&author=AC%20Papp&author=X%20Sun&volume=24&publication_year=2015&pages=1556-62&pmid=25381333&doi=10.1093/hmg/ddu566&)

31. Teft WA, Gong IY, Dingle B, Potvin K, Younus J, Vandenberg TA, Brackstone M, Perera FE, Choi YH, Zou G, Legan RM, Tirona RG, Kim RB. CYP3A4 and seasonal variation in vitamin D status in addition to CYP2D6 contribute to therapeutic endoxifen level during tamoxifen therapy. Breast Cancer Res Treat. 2013;139:95–105. doi: 10.1007/s10549-013-2511-4.  [DOI](https://doi.org/10.1007/s10549-013-2511-4) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23580071/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Breast%20Cancer%20Res%20Treat&title=CYP3A4%20and%20seasonal%20variation%20in%20vitamin%20D%20status%20in%20addition%20to%20CYP2D6%20contribute%20to%20therapeutic%20endoxifen%20level%20during%20tamoxifen%20therapy&author=WA%20Teft&author=IY%20Gong&author=B%20Dingle&author=K%20Potvin&author=J%20Younus&volume=139&publication_year=2013&pages=95-105&pmid=23580071&doi=10.1007/s10549-013-2511-4&)

32. Klein DJ, Thorn CF, Desta Z, Flockhart DA, Altman RB, Klein TE. PharmGKB summary: tamoxifen pathway, pharmacokinetics. Pharmacogenet Genomics. 2013;23:643–7. doi: 10.1097/FPC.0b013e3283656bc1.  [DOI](https://doi.org/10.1097/FPC.0b013e3283656bc1) | [PMC free article](/articles/PMC4084801/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23962908/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=PharmGKB%20summary:%20tamoxifen%20pathway,%20pharmacokinetics&author=DJ%20Klein&author=CF%20Thorn&author=Z%20Desta&author=DA%20Flockhart&author=RB%20Altman&volume=23&publication_year=2013&pages=643-7&pmid=23962908&doi=10.1097/FPC.0b013e3283656bc1&)
