# Evidence of CNIH3 involvement in opioid dependence

## Metadata
**Authors:** Elliot C Nelson, Arpana Agrawal, Andrew C Heath, Ryan Bogdan, Richard Sherva, Bo Zhang, Ream Al-Hasani, Michael R Bruchas, Yi-Ling Chou, Catherine H Demers, Caitlin E Carey, Emily D Conley, Amanda K Fakira, Lindsay A Farrer, Alison Goate, Scott Gordon, Anjali K Henders, Victor Hesselbrock, Manav Kapoor, Michael T Lynskey, Pamela AF Madden, Jose A Moron, John P Rice, Nancy L Saccone, Sibylle G Schwab, Fiona L Shand, Alexandre A Todorov, Leanne Wallace, Ting Wang, Naomi R Wray, Xin Zhou, Louisa Degenhardt, Nicholas G Martin, Ahmad R Hariri, Henry R Kranzler, Joel Gelernter, Laura J Bierut, David J Clark, Grant W Montgomery
**Journal:** Molecular psychiatry
**Date:** 2015 Aug 4
**DOI:** [10.1038/mp.2015.102](https://doi.org/10.1038/mp.2015.102)
**PMID:** 26239289
**PMCID:** PMC4740268
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4740268/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC4740268/pdf/nihms701291.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC4740268/pdf/nihms701291.pdf)

## Abstract

Opioid dependence, a severe addictive disorder and major societal problem, has been demonstrated to be moderately heritable. We conducted a genome-wide association study in Comorbidity and Trauma Study data comparing opioid dependent daily injectors (N=1167) with opioid misusers who never progressed to daily injection (N=161). The strongest associations, observed for CNIH3 SNPs, were confirmed in two independent samples, the Yale-Penn genetic studies of opioid, cocaine, and alcohol dependence and the Study of Addiction: Genetics and Environment, which both contain non-dependent opioid misusers and opioid dependent individuals. Meta-analyses found 5 genome-wide significant CNIH3 SNPs. The A allele of rs10799590, the most highly associated SNP, was robustly protective [p=4.30E-9; OR 0.64 (95%CI 0.55 – 0.74)]. Epigenetic annotation predicts that this SNP is functional in fetal brain. Neuroimaging data from the Duke Neurogenetics Study (N=312) provide evidence of this SNP’s in vivo functionality; rs10799590 A allele carriers displayed significantly greater right amygdala habituation to threat-related facial expressions, a phenotype associated with resilience to psychopathology. Computational genetic analyses of physical dependence on morphine across 23 mouse strains yielded significant correlations for haplotypes in CNIH3 and functionally-related genes. These convergent findings support CNIH3 involvement in the pathophysiology of opioid dependence complementing prior studies implicating the AMPA glutamate system.

## INTRODUCTION

Twin and family studies provide evidence of a genetic contribution to liability for opioid dependence with heritability estimates ranging from 40 to 60%.^1–4^ However, genetic association studies have produced few consistently replicated findings.^5^ One important factor contributing to this inconsistency is the lack of a definitive control population^6,7^ for these investigations.

Studies ^e.g.,8^ have used unassessed controls based on the premise that, for low prevalence disorders, this approach only modestly reduces power.^9^ However, opioids are among the most highly addictive drugs^10,11^ with high rates of progression from misuse to dependence^12^ and thus the main constraint on the prevalence of opioid dependence may be the prevalence of opioid misuse. The extent to which genetic factors contribute to liability at various stages of opioid addiction (e.g., initiation, regular use, and dependence), and are shared between stages, is not well characterized.^5^ Thus, the use of unassessed, predominately unexposed controls might be problematic for identifying genetic effects expressed after the initiation of opioid misuse. Importantly, significant effects of common SNPs manifesting at intermediate and later stages of addiction would be missed in comparisons to unexposed controls. Similarly, comparisons to assessed, unexposed controls are more useful to examine shared liability for initiation and dependence.

Analyses of candidate gene data in the current report’s discovery sample, the Comorbidity and Trauma Study (CATS), showed that association findings vary substantially depending on the comparison group’s substance exposure status.^6,7^ The current investigation builds on this observation and draws from genetic studies of licit drugs^13–19^ that have yielded well-replicated findings by comparing non-dependent, drug-exposed to substance dependent individuals. Unfortunately, no large samples of non-dependent opioid misusers have been collected. Our discovery^6,7,20^ and confirmation samples^19,21–24^ contain only modest numbers of non-dependent opioid misusers, but are currently the largest such samples with genome-wide association study (GWAS) data. We hypothesize that genetic polymorphisms in opioid misusers influence progression to the population’s opioid dependence endpoint (OD_E_). Our analyses of GWAS data observed the strongest association for cornichon family AMPA receptor auxiliary protein 3 (*CNIH3*) polymorphisms, findings which map nicely onto literature^25–29^ supporting AMPA glutamate system involvement in the pathophysiology of opioid dependence.

## MATERIALS AND METHODS

### GWAS sample subjects and assessment

Detailed descriptions of CATS data collection have been reported.^6,7,20^ Opioid dependent individuals, aged 18 or older, were recruited from opioid substitution therapy (OST) clinics in the greater Sydney region. Neighborhood controls, individuals with little or no lifetime opioid misuse, were recruited from socially disadvantaged neighborhoods in geographic proximity to OST clinics. Written informed consent was obtained from all participants as approved by the institutional review boards (IRBs) of all participating institutions and clinics. Semi-structured psychiatric diagnostic interviews, a modified Semi-Structured Assessment for the Genetics of Alcoholism - Australia (SSAGA-OZ),^30^ were completed in-person.

Since 94.1% of the CATS^6,7,20^ opioid dependent participants reported a period of daily injection, we operationalized having had such a period as the population’s normative opioid dependence endpoint (OD_E_). Comparisons of opioid dependent individuals who differed on daily injection status found substantial phenotypic differences ([Supplementary Table 1](#SD1) and [Supplementary Methods](#SD1)). Comparisons of dependent individuals who never injected daily to non-dependent opioid misusers revealed fewer significant differences. Our GWAS analyses compared the OD_E_ group (N=1167 opioid dependent daily injectors) to a group characterized as having opioid use with impeded progression (OU_IP_) that combined non-dependent opioid misusers (N=88; 69.3% reporting heroin use) and opioid dependent individuals without a history of daily injection (N=73).

The Yale-Penn genetic studies of opioid, cocaine, and alcohol dependence^19,23,24^ were recruited at 5 U.S. sites. All participants gave written informed consent as approved by each site’s IRB. We addressed design and assessment differences ([Supplementary Tables 2 and 3](#SD1), and [Supplementary Methods](#SD1)) that prevented defining phenotypes identical to those in CATS by using an extreme discordant approach. We operationalized the OD_E_ group as opioid dependent individuals whose opioid use had been daily or near daily, included heroin, and injection at least 100 times lifetime and the OU_IP_ group as individuals reporting heroin use who met no lifetime DSM-IV opioid dependence criteria. We limited inclusion to European ancestry (EA) participants to examine confirmation in a sample of comparable ethnicity, retaining 643 OD_E_ and 157 OU_IP_ individuals for analysis.

The Study of Addiction: Genetics and Environment (SAGE)^21,22^ is an alcohol dependence GWAS that selected cases and controls from large investigations targeting non-opioid substance dependence. Each contributing institution’s IRB approved the recruitment protocols. All participants provided written informed consent. Since SAGE did not ascertain participants on the basis of opioid dependence, it included fewer severely dependent individuals and more participants of unclear affection status. We operationalized the OD_E_ group as DSM-IV opioid dependence and the OU_IP_ group as opioid misusers who met at most one dependence and no abuse criterion. Limiting inclusion to EA participants, we retained 190 OD_E_ and 319 OU_IP_ individuals.

### Genotyping and data cleaning

CATS samples were genotyped using the Illumina Human660W-Quad BeadChip at the Johns Hopkins Center for Inherited Disease Research (CIDR). For data cleaning details, see [Supplementary Methods](#SD1). The genotyping rate for the 470,296 SNPs that remained after data cleaning was 99.93%.

The Yale-Penn samples were genotyped on the Illumina HumanOmni1-Quad v1.0 microarray at CIDR and the Yale Center for Genome Analysis. SAGE samples were genotyped at CIDR using Illumina Human 1Mv1_C BeadChips. Genotypic data cleaning and quality control details have been reported for the Yale-Penn studies^19,23,24^ and SAGE.^21,22^

### Data analyses

#### Admixture

PCA was conducted using the SmartPCA program^31^ to provide additional admixture correction. Three PCs were generated via PCA and included as covariates in the regression models. Similar methods were used in the Yale-Penn and SAGE data sets to generate PCs for inclusion in analyses (consistent with their prior publications).^19,21,23,24^

#### SNP-based association

The genomic inflation factor for CATS data was calculated in PLINK^32^ based on the median chi-square value. Logistic regression analyses were performed in PLINK^32^ to examine the association between the log-additive effects of risk allele dosage and group status (OD_E_ versus OU_IP_) controlling for sex, age category, and three PCs. Manhattan and qq plots were constructed for results.

Association analyses of confirmation sample data were conducted consistent with prior reports. The Yale-Penn data were analyzed using logistic regression models embedded in generalized estimating equations to correct for correlations of data from related individuals with age, sex, and three PCs included as covariates. Analyses of SAGE data were conducted in PLINK^32^ with contributing component study, age, sex, and two PCs included as covariates.

Meta analyses were performed using the inverse variance weighting approach of the METAL program.^33^ The phenotypic variance in OD_E_ status explained by rs10799590 was calculated^34^ for the meta-analytic results using odds ratios, risk allele frequencies (RAFs), and prevalence estimates ranging from 0.005 (population prevalence of heroin dependence)^12,35,36^ to 0.25 (approximate prevalence reported for heroin dependence among users).^12^

#### Epigenetic annotation (see Supplementary Methods)

#### Duke Neurogenetics Study (DNS) (see Supplementary Methods)

#### Participants

DNS participants were in good general health and provided informed written consent approved by the Duke University Medical Center IRB. The EA subsample reported here consisted of 312 participants (age = 19.71 ± 1.23 year*s*; 151 males; 65 with DSM-IV diagnoses).

#### Genotyping

DNS participants’ DNA was isolated from saliva and genotyped with Illumina HumanOmniExpress BeadChips. The genotyping rate of rs10799590 was 1.0 and was within HWE χ^2^=0.39, p=0.53.

#### DNS neuroimaging protocol BOLD fMRI paradigm

A widely used and reliable challenge paradigm was employed to elicit amygdala reactivity. The paradigm consists of 4 task blocks requiring face-matching interleaved with 5 control blocks requiring shape-matching (see [Figure S1](#SD1)). All 4 facial expressions convey threat, ambiguity, and/or novelty that robustly recruit corticolimbic circuitry that includes the amygdala.^37^

#### BOLD fMRI data analysis

Following the preprocessing steps, linear contrasts employing canonical hemodynamic response functions were used to estimate amygdala habituation as the linear decrease over successive face matching blocks (i.e., block 1 > block 2 > block 3 > block 4). Follow-up analyses evaluated amygdala response differences in block 1 across genotype groups. Individual contrast images (i.e., weighted sum of beta images) were used in second-level random effects models accounting for scan-to-scan and participant-to-participant variability to determine mean contrast-specific responses using one-sample t-tests. A voxel-level statistical threshold of *P* < 0.05, family wise error corrected for multiple comparisons across the bilateral amygdala regions of interest, and a cluster-level extent threshold of 10 contiguous voxels was applied to these analyses. The bilateral amygdala regions of interest (ROI) were defined using the AAL template. BOLD parameter estimates from maximal voxels in the right and left amygdala ROI exhibiting a main effect for the amygdala habituation contrast were extracted using the VOI tool in SPM8 and exported for regression analyses in SPSS (v.22). Extracting parameter estimates from clusters activated by our fMRI paradigm, rather than those specifically correlated with our independent variables of interest, precludes the possibility of any correlation coefficient inflation that may result when an explanatory covariate is used to select a region of interest.

### Statistical Analyses

Statistical analyses of the imaging data were conducted using linear regression in SPSS to test the association between rs10799590 A allele carrier status and amygdala habituation (i.e., declining amygdala response to repeated stimuli). To maintain variability but constrain the influence of extreme outliers, all imaging variables were winsorized prior to analyses. Gender and psychiatric diagnosis (0,1) were entered as covariates for analyses. To determine whether rs1079950 was most strongly associated with amygdala habituation within this genomic region, SNPs ± 100 kbp that had an LD r^2^ ≥ .50 with rs10799590 within the dataset were identified and binned according to LD (r^2^ ≥ .80). Similar analyses were then conducted to examine the association between carrier status of each SNP and amygdala habituation.

### Genetic analysis of murine interstrain differences

Male mice, aged 7–8 weeks, from 23 inbred strains ([Supplementary Table 5](#SD1)) were housed in Stanford University’s animal care facilities. Experimental protocols were approved by the Institutional Animal Care and Use Committee and complied with the Guide for the Care and Use of Laboratory Animals. Details are provided in [Supplementary Methods](#SD1). ANOVA was used to calculate a *P* value to assess the likelihood that the within-block genetic variation underlies the phenotypic distribution (i.e., the mean number of jumps) observed for the inbred strains examined. Haplotype data were examined for seven genes: *CNIH3, CNIH2, GRIA1, GRIA2, CACNG8, GRIP1, and DLG4. P* values were adjusted to control the false-discovery rate.^38^

## RESULTS

### CATS GWAS

The quantile-quantile (q-q) plot of association results ([Figure S2, Panel A](#SD1)) and the genomic inflation factor value (λ=1.01) indicate an absence of test statistic inflation. The strongest association involves a cluster of chromosome 1 SNPs (Manhattan plot, [Figure S2, Panel B](#SD1)). The six most highly associated, all located in *CNIH3* ([Table 1](#T1)), are in moderate to high LD (r^2^=0.35–0.97); conditional analyses suggest they represent a single association signal ([Figure S2, Panel C](#SD1)). The association reached genome-wide significance (GWS) for rs1436175 [p=2.72 E-8; OR 0.50 (0.39–0.64)] with the risk allele halving the likelihood of progression to OD_E_.

### Table 1.

| Gene | SNP | Chr | Genomic coordinates | RA | Risk Allele Frequency |  | p value | Odds Ratio (95% confidence interval) |
| --- | --- | --- | --- | --- | --- | --- | --- | --- |
| ODEN=1167 | OUIPN=161 |  |  |  |  |  |  |  |
| CNIH3 | rs10799590 | 1 | 224822482 | A | 0.42 | 0.56 | 1.51E-6 | 0.55 (0.43 – 0.70) |
| CNIH3 | rs12130499 | 1 | 224836514 | T | 0.42 | 0.56 | 1.15E-6 | 0.54 (0.43 – 0.70) |
| CNIH3 | rs298733 | 1 | 224842251 | A | 0.42 | 0.57 | 1.53E-6 | 0.55 (0.43 – 0.70) |
| CNIH3 | rs1436171 | 1 | 224881828 | A | 0.44 | 0.59 | 6.26E-7 | 0.54 (0.42 – 0.68) |
| CNIH3 | rs1369846 | 1 | 224894095 | C | 0.38 | 0.54 | 9.42E-8 | 0.52 (0.41 – 0.66) |
| CNIH3 | rs1436175 | 1 | 224908366 | T | 0.37 | 0.53 | 2.72E-8 | 0.50 (0.39 – 0.64) |
| STAB2 | rs10861067 | 12 | 104038974 | G | 0.25 | 0.38 | 2.00E-6 | 0.55 (0.43 – 0.71) |
| STAB2 | rs10778270 | 12 | 104045678 | A | 0.20 | 0.31 | 5.16E-6 | 0.55 (0.42 – 0.71) |
| intergenic | rs9521590 | 13 | 110709426 | A | 0.05 | 0.11 | 4.31E-6 | 0.36 (0.23 – 0.55) |
| FUT8 | rs6573615 | 14 | 66046534 | G | 0.36 | 0.49 | 9.68E-6 | 0.58 (0.46 – 0.74) |

Table 1 Caption: SNPs associated with ODE in CATS participants (p values<1E-5)

### Confirmation of association findings

These analyses focused on the *CNIH3* SNPs because of the substantially stronger association observed for these polymorphisms and the gene’s obvious biological relevance. In the Yale-Penn data ([Table 2](#T2)), trend-level association was observed for three of the six SNPs examined. In the SAGE data, a stronger association was found for five of the six SNPs. In both data sets, all association signals were in the same direction as in the CATS.

### Table 2.

| SNP | CATS (1167 ODE vs 161 OUIP) |  | Yale-Penn (643 ODE vs 157 OUIP) |  | SAGE (190 ODE vs 319 OUIP) |  | Meta-analysis |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| OR (95%CI) | p value | OR (95%CI) | p value | OR (95%CI) | p value | OR (95%CI) | p value | Het X2 (2df) p value |  |
| rs10799590 | 0.55 (0.43 – 0.70) | 1.51E-6 | 0.78 (0.60 – 1.01) | 5.76E-2 | 0.62 (0.47– 0.82) | 6.73E-4 | 0.64 (0.55 – 0.74) | 4.30E-9 | 0.15 |
| rs12130499 | 0.54 (0.43 – 0.70) | 1.15E-6 | 0.78 (0.60– 1.01) | 6.14E-2 | 0.62 (0.47 – 0.82) | 7.49E-4 | 0.64 (0.55 – 0.74) | 4.31E-9 | 0.13 |
| rs298733 | 0.55 (0.43 – 0.70) | 1.53E-6 | 0.79 (0.62 – 1.03) | 7.61E-2 | 0.64 (0.49 – 0.84) | 1.25E-3 | 0.65 (0.56 – 0.75) | 1.25E-8 | 0.12 |
| rs1436171 | 0.54 (0.42 – 0.68) | 6.26E-7 | 0.82 (0.64 – 1.04) | 1.07E-1 | 0.65 (0.49 – 0.85) | 2.04E-3 | 0.66 (0.57 – 0.76) | 2.17E-8 | 0.06 |
| rs1369846 | 0.52 (0.41 – 0.66) | 9.42E-8 | 0.85 (0.67 – 1.09) | 1.94E-1 | 0.66 (0.50 – 0.87) | 2.96E-3 | 0.66 (0.57 – 0.77) | 2.60E-8 | 0.02 |
| rs1436175 | 0.50 (0.39 – 0.64) | 2.72E-8 | 0.93 (0.73 – 1.18) | 5.50E-1 | 0.77 (0.59 – 1.01) | 6.14E-2 | 0.71 (0.61 – 0.82) | 3.09E-6 | 0.002 |

Table 2 Caption: Association of CNIH3 SNPs with ODE in CATS, Yale-Penn, and SAGE data, and in meta-analysis

Meta-analyses performed on these SNPs using data from the three samples found GWS association for five of the six *CNIH3* SNPs ([Table 2](#T2)). The strongest meta-analytic association signal (p=4.30 E-9) was observed for rs10799590; the odds ratio [0.64 (95%CI 0.55 – 0.74)] is indicative of the risk allele’s robust protective effects. Rs1436175, which had the lowest p value in the CATS, failed to reach meta-analytic GWS. This SNP has the lowest LD with the other *CNIH3* SNPs (r^2^=0.35–0.54) and the largest heterogeneity chi-square (p=0.002). The meta-analytic GWS SNPs are in high LD ([Figure S2, Panels C and D](#SD1)).

### Epigenetic annotation

The GWS *CNIH3* SNPs are intronic and not highly conserved. Although the observed associations may be due to high LD with a non-genotyped variant, no exonic SNPs in high LD were identified. Epigenetically-mediated changes in gene expression, which have been reported to occur with opioid use,^39^ are plausible mechanisms for functional associations involving intronic SNPs. Rs10799590 is located within an enhancer that is specific to fetal brain ([Figure 1](#F1)). It is within an H3K4me1peak in fetal brain that DNaseI hypersensitivity data indicate is in an open chromatin state.^40–42^ It is predicted that rs10799590 is within the binding site of transcription factor (TF) TAL1 (which plays important roles in middle brain GABAergic neuron differentiation^68^); the G allele has significantly higher binding potential than the A allele ([Supplementary Table 6](#SD1)).

### Figure 1. Epigenetic Landscape of the Six Intronic CNIH3 SNPs.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/37e1/4740268/6b8caffe257f/nihms701291f1.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4740268_nihms701291f1.jpg)

Rs1369846 and rs298733 are located within retrotransposons.40,41 Evidence of epigenetic functionality for rs10799590 includes the location of this SNP within a fetal brain specific enhancer.46,47 Fetal brain H3K4me1 data indicate that it is within a H3K4me1 peak; DNaseI hypersensitivity data suggest that it is in an open chromatin state in fetal brain. This enhancer mark on rs10799590 was specific to fetal brain, but was not in CD4+ T cells, breast luminal epithelial cells, adult liver, fetal heart, fetal lung, melanocyte, or keratinocytes. The results of motif analyses (Supplementary Table 6) predict that rs10799590 is within the binding site of transcription factor Tal1 with the G allele having significantly higher binding affinity than the A allele.

### Protective allele carrier status predicts greater amygdala habituation

The observation^25^ that similar changes in AMPAR GluA1 subunits occur in the amygdala with opioid addiction^26^ and fear conditioning,^43–45^ coupled with evidence of epigenetically-mediated alterations in gene expression that ensue in both processes after environmental exposures,^43,46^ provided the rationale for examining amygdala habituation to threat-related facial expressions (a reliable intermediate phenotype linked to psychopathology).^46,47^ After accounting for sex and the presence of a DSM-IV disorder, rs10799590 A allele carrier status predicted right (stand Beta = 0.147; ΔF_1,308_=6.93, p < .009, ΔR^2^=0.022; [Figure 2](#F2)), but not left (stand Beta = 0.031; ΔF_1,308_=0.302, p > .582, ΔR^2^<0.001), amygdala habituation. G allele homozygotes (n=102) had blunted right amygdala habituation (0.045±0.374) relative to A allele carriers (n=210; 0.164±0.371). We identified 3 genotyped SNPs (rs1369848, rs12730234, rs1965776) in moderate LD with rs10799590 that tagged SNP blocks; however, none was more strongly associated with amygdala habituation than rs10799590 ([Supplementary Table 7](#SD1)). Follow-up analyses revealed that genotype groups did not differ in initial right amygdala responses to stimuli (stand Beta = 0.067; ΔF_1,308_=1.42, p > .234, ΔR^2^=0.004).

### Figure 2. Amygdala Habituation.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/37e1/4740268/c78413036d16/nihms701291f2.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4740268_nihms701291f2.jpg)

(a) Statistical parametric map illustrating mean bilateral amygdala habituation across all DNS participants (left MNI coordinates: x = −20 y = −8 z = −16, kE = 144, t = 9.03, P < 0.001; right MNI coordinates x = 22 y = −6 z = −14, kE = 83, t = 8.97, P < 0.001) (b) A allele carriers had greater right amygdala habituation (i.e., less persistent activation) relative to G allele homozygotes. The Y axis indicates habituation with greater values indicating a larger decrease in activation over time. See Figure S3 for a depiction of activation across blocks.

### Genetic analysis of interstrain differences in physical dependence on morphine

To link our findings further to existing animal literature, we performed computational haplotype-based genetic mapping analyses^48–50^ of data from 23 inbred mouse strains for a robust measure of opioid physical dependence,^48^ counts of jumps made by morphine dependent mice after naloxone administration. Correlations were calculated for the distribution of the mean number of jumps per strain with known haplotype blocks across strains for *CNIH3* and genes encoding AMPAR subunits and proteins involved in alterations of AMPAR subunit composition in response to opioids. Significant correlation was observed for *CNIH3* haplotype, but not for the more widely expressed *cornichon family AMPA receptor auxiliary protein 2 (CNIH2)* ([Table 3](#T3))*. S*ignificant correlations were also noted for haplotypes in *glutamate receptor, ionotropic, AMPA 1 (GRIA1), glutamate receptor, ionotropic, AMPA 2 (GRIA2), calcium channel, voltage-dependent, gamma subunit 8 (CACNG8),* and *glutamate receptor-interacting protein 1 (GRIP1),* but not *discs, large homolog 4* (*DLG4*)].

### Table 3.

| Gene | P value* |
| --- | --- |
| CNIH3 | 4.0E-4 |
| CACNG8 | 8.3E-4 |
| GRIA1 | 8.1E-6 |
| GRIP1 | 1.7E-5 |
| GRIA2 | 7.4E-3 |
| CNIH2 | .35 |
| DLG4 | .35 |

Table 3 Caption: Computational genetic analysis of interstrain differences in physical dependence on morphine

## DISCUSSION

The current report provides evidence for *CNIH3* involvement in the pathophysiology of opioid dependence. *CNIH3* encodes a small, highly conserved protein. The AMPA receptor core is formed by tetramers of the GluA1-4 subunits and up to four members of three protein groups: transmembrane AMPAR regulatory proteins (TARPs), cornichon homologs (CNIH3 and CNIH2), and the GSG1l protein.^51^ The receptor’s periphery contains transmembrane and other proteins [e.g., post-synaptic density protein 95 (PSD-95)] that bind with core proteins and each other in the postsynaptic density (PSD).^51–54^ CNIH2 and CNIH3 markedly slow AMPAR deactivation and desensitization in heterologous systems.^54–56^ One study^54^ suggested that the actions of CNIH2 and CNIH3 are selective for AMPARs containing GluA1 subunits; however, more recent reports^51,57^ do not support this specificity of binding. An investigation that focused on two hippocampal cell types with markedly different excitatory postsynaptic currents (EPSCs) implicated CNIH2 as largely responsible for the distinction between fast and slow EPSCs. Although this report did not examine whether CNIH3 plays a similar role, prior studies ^e.g., 55^ have found that the two proteins have comparable effects on slowing AMPAR deactivation and desensitization.

Rodent studies^26–29^ have implicated alterations in the subunit composition of brain AMPARs in diverse aspects of opioid addiction. Increased expression of GluA1-containing/GluA2-lacking AMPARs has been observed in the central nucleus of the amygdala^26^ and the hippocampal PSD^28^ in studies of morphine-related context-reward conditioning^26^ and context-dependent behavioral sensitization.^28^ The latter ^28^ found that these changes were mediated via interactions with TARP gamma-8 and GRIP1 proteins. GluA1 knockout mice displayed impaired drug-induced state dependency after operant conditioning with morphine.^29^ Another study^27^ implicated down-regulation of GluA2 expression in the prefrontal cortex in reinstatement of heroin self-administration after prolonged abstinence. Interestingly, *CNIH3* expression is greatest in the frontal cortex, amygdala, and hippocampus in the adult human brain.^58^

Our GWAS analyses found that *CNIH3* SNPs are associated with protection against progression to OD_E_ in OU_IP_ individuals. Since this effect was observed in analyses limited to opioid misusers, it likely represents liability unrelated to that for initiation of opioid misuse. The discovery and confirmation sample OU_IP_ groups have substantially higher RAFs than an Australian (EA) general population sample^59^ ([Supplementary Table 8](#SD1)) to which the OD_E_ groups’ RAFs are more similar. Post-hoc SNP-based association analyses comparing the CATS OU_IP_ group and this general population sample found substantial differences (p values ≤ 6.4E-5); similar comparisons with the CATS OD_E_ group found more modest differences (p≥2.3E-2) in the opposite direction. The phenotypic variance in OD_E_ explained by rs10799590 in our meta-analysis is estimated at 1.17% to 5.85% ([Supplementary Table 9](#SD1)), indicative of a strong effect. Overall, our findings suggest that these *CNIH3* SNPs enable greater, but not complete control in the use of these otherwise highly addictive drugs.

An examination of human post-mortem amygdalae reported a strong positive correlation between GluA1 and PSD-95 mRNA expression in heroin dependent cases, but not in controls (*CNIH3* expression was not reported).^25^ Another study^55^ observed positive correlations for these proteins with CNIH3 (GluA1 0.43; DLG4 0.28) in (unexposed) mammalian brains. Thus, human^25^ and animal^26^ studies provide evidence of altered amygdala GluA1 expression in opioid dependence. The post-mortem report noted^25^ that similar changes in AMPAR GluA1 subunits occur in the amygdala with associative learning of opioid reward^26^ and fear conditioning.^43–45^ Importantly, both processes involve epigenetically-mediated changes in gene expression following an environmental exposure.^39,46^ We thus examined the effects of our most strongly associated SNP using imaging genetics^46,60^ and observed significantly greater right amygdala habituation to threat-related facial expression in rs10799590 A allele carriers. Consistent with the reduced risk we observed in opioid exposed individuals, the polymorphism’s protective neural effects were apparent with subsequent, but not initial, exposures to environmental stimuli. The association of this SNP with OD_E_ likely represents a similar protective process involving greater habituation to the neural effects of opioids that impacts additional opioid use.

Our focus on opioid misusers is a major strength; the modest size of our OU_IP_ groups is an unavoidable limitation. Our assumption that SNPs can offer protection against transitions at different points of the addictive process is supported by the existing animal literature.^26–29^ We only examined confirmation of the *CNIH3* SNP associations; a broader examination might have confirmed other CATS associations. Since both the epigenetic landscape and motif analysis support its potential functionality, our neuroimaging genetics study focused on rs10799590 with post-hoc analyses confirming it as the *CNIH3* SNP most strongly associated with amygdala habituation. Future work should incorporate examination of the other GWS SNPs (e.g., rs298733 also affects TF binding - [Supplementary Table 6](#SD1)) and the possibility of more highly associated, non-genotyped polymorphisms in high LD. Although similarly ascertained samples ^e.g., 61^ have comparable rates of daily opioid injection, reports in other populations have noted lower prevalence. ^e.g.,62^ A more detailed characterization of the OU_IP_ groups’ opioid use would have been useful, but was not obtained. We examined somewhat divergent OD_E_’s in our confirmation samples because of ascertainment and assessment differences ([Supplementary Table 3](#SD1)). While these methodological differences are a limitation, the observed confirmation in two multi-site U.S.-based studies supports the generalizability of our findings. The highly comorbid composition of the OD_E_ and OU_IP_ groups may raise concerns that intergroup differences are better attributable to another phenotype. Post-hoc association analyses conducted in CATS to address this possibility ([Supplementary Table 10](#SD1)) found few associations for comorbid disorders reaching nominal significance. Comorbidity pattern differences across samples ([Supplementary Tables 1 and 2](#SD1)) further argue against this possibility. Finally, our exclusion of participants with non-European ancestry and the lack of genomic inflation (λ=1.01) suggest spurious association due to uncorrected admixture is unlikely.

Our meta-analyses found GWS association with *CNIH3* SNPs conferring robust protective effects against OD_E_, findings that map onto reports of AMPA glutamatergic involvement in opioid dependence.^25–29^ The finding of significantly greater habituation in the right amygdala rs10799590 A allele carriers supports this SNP’s *in vivo* functional effects in humans (complementing evidence of epigenetic functionality). The genetic analyses of mouse strain data support the involvement of *CNIH3,* but not the more highly expressed *CNIH2,* in murine opioid physical dependence. The significant correlations observed for genes encoding AMPAR subunits and related proteins provide additional evidence for genetic risk mediated via this pathway. These convergent findings implicate *CNIH3’s* involvement in opioid dependence and could provide a route to target glutamatergic processes for translational research focusing on improving opioid dependence treatments and developing opioid analgesics with lower dependence risk.

## Supplementary Material

## Acknowledgments

Funding support for the Comorbidity and Trauma Study (CATS) was provided by the National Institute on Drug Abuse (R01 DA17305); GWAS genotyping services at the Center for Inherited Disease Research (CIDR) at The Johns Hopkins University were supported by the National Institutes of Health [contract N01-HG-65403].

Funding for the Yale-Penn genetic studies of opioid, cocaine, or alcohol dependence was provided by the National Institutes of Health [RC2 DA028909, R01 DA12690, R01 DA12849, R01 DA18432, R01 AA11330, R01 AA017535, 5T32GM007205-38, 8UL1TR000142-07]; and the Veterans Affairs Connecticut and Veterans Affairs Philadelphia Mental Illness Research, Education and Clinical Centers. Genotyping services at the Center for Inherited Disease Research (CIDR) at The Johns Hopkins University were supported by the National Institutes of Health [contract N01-HG-65403].

Funding support for the Study of Addiction: Genetics and Environment (SAGE) was provided through the NIH Genes, Environment and Health Initiative [GEI] (U01 HG004422). SAGE is one of the genome-wide association studies funded as part of the Gene Environment Association Studies (GENEVA) under GEI. Assistance with phenotype harmonization and genotype cleaning, as well as with general study coordination, was provided by the GENEVA Coordinating Center (U01 HG004446). Assistance with data cleaning was provided by the National Center for Biotechnology Information. Support for collection of datasets and samples was provided by the Collaborative Study on the Genetics of Alcoholism (COGA; U10 AA008401), the Collaborative Genetic Study of Nicotine Dependence (COGEND; P01 CA089392), and the Family Study of Cocaine Dependence (FSCD; R01 DA013423, R01 DA019963). Funding support for genotyping, which was performed at the Johns Hopkins University Center for Inherited Disease Research, was provided by the NIH GEI (U01HG004438), the National Institute on Alcohol Abuse and Alcoholism, the National Institute on Drug Abuse, and the NIH contract “High throughput genotyping for studying the genetic contributions to human disease” (HHSN268200782096C). The Duke Neurogenetics Study is supported by Duke University and National Institute on Drug Abuse grant DA033369. AA also receives support from R01 DA23668 and K02 DA32573. GWM and NRW are supported by NHMRC Fellowships. TW, BZ, and XZ receive support from DA027995, R01HG007354, R01HG007175, R01ES024992, and ACS grant RSG-14-049-01-DMC. A.R.H. receives support through National Institute on Drug Abuse grants DA033369 and DA031579. RB receives support from the Klingenstein Third Generation Foundation and the National Institutes of Health (NIA R01-AG045231). CHD receives support from T32DA007313. CEC receives support from NSF DGE-1143954.

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Merikangas KR, Stolar M, Stevens DE, Goulet J, Preisig MA, Fenton B, et al. Familial transmission of substance use disorders. Arch Gen Psychiatry. 1998;55:973–979. doi: 10.1001/archpsyc.55.11.973.  [DOI](https://doi.org/10.1001/archpsyc.55.11.973) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9819065/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arch%20Gen%20Psychiatry&title=Familial%20transmission%20of%20substance%20use%20disorders&author=KR%20Merikangas&author=M%20Stolar&author=DE%20Stevens&author=J%20Goulet&author=MA%20Preisig&volume=55&publication_year=1998&pages=973-979&pmid=9819065&doi=10.1001/archpsyc.55.11.973&)

2. Tsuang MT, Lyons MJ, Meyer JM, Doyle T, Eisen SA, Goldberg J, et al. Co-occurrence of abuse of different drugs in men: the role of drug-specific and shared vulnerabilities. Arch Gen Psychiatry. 1998;55:967–972. doi: 10.1001/archpsyc.55.11.967.  [DOI](https://doi.org/10.1001/archpsyc.55.11.967) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9819064/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arch%20Gen%20Psychiatry&title=Co-occurrence%20of%20abuse%20of%20different%20drugs%20in%20men:%20the%20role%20of%20drug-specific%20and%20shared%20vulnerabilities&author=MT%20Tsuang&author=MJ%20Lyons&author=JM%20Meyer&author=T%20Doyle&author=SA%20Eisen&volume=55&publication_year=1998&pages=967-972&pmid=9819064&doi=10.1001/archpsyc.55.11.967&)

3. Kendler KS, Jacobson KC, Prescott CA, Neale MC. Specificity of genetic and environmental risk factors for use and abuse/dependence of cannabis, cocaine, hallucinogens, sedatives, stimulants, and opiates in male twins. Am J Psychiatry. 2003;160:687–695. doi: 10.1176/appi.ajp.160.4.687.  [DOI](https://doi.org/10.1176/appi.ajp.160.4.687) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12668357/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Psychiatry&title=Specificity%20of%20genetic%20and%20environmental%20risk%20factors%20for%20use%20and%20abuse/dependence%20of%20cannabis,%20cocaine,%20hallucinogens,%20sedatives,%20stimulants,%20and%20opiates%20in%20male%20twins&author=KS%20Kendler&author=KC%20Jacobson&author=CA%20Prescott&author=MC%20Neale&volume=160&publication_year=2003&pages=687-695&pmid=12668357&doi=10.1176/appi.ajp.160.4.687&)

4. Sun J, Bi J, Chan G, Oslin D, Farrer L, Gelernter J, et al. Improved methods to identify stable, highly heritable subtypes of opioid use and related behaviors. Addict Behav. 2012;37:1138–1144. doi: 10.1016/j.addbeh.2012.05.010.  [DOI](https://doi.org/10.1016/j.addbeh.2012.05.010) | [PMC free article](/articles/PMC3395719/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22694982/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Addict%20Behav&title=Improved%20methods%20to%20identify%20stable,%20highly%20heritable%20subtypes%20of%20opioid%20use%20and%20related%20behaviors&author=J%20Sun&author=J%20Bi&author=G%20Chan&author=D%20Oslin&author=L%20Farrer&volume=37&publication_year=2012&pages=1138-1144&pmid=22694982&doi=10.1016/j.addbeh.2012.05.010&)

5. Agrawal A, Verweij KJ, Gillespie NA, Heath AC, Lessov-Schlaggar CN, Martin NG, et al. The genetics of addiction-a translational perspective. Transl Psychiatry. 2012;2:e140. doi: 10.1038/tp.2012.54. Erratum in: Transl Psychiatry 2012 (2), e193.  [DOI](https://doi.org/10.1038/tp.2012.54) | [PMC free article](/articles/PMC3410620/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22806211/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Transl%20Psychiatry&title=The%20genetics%20of%20addiction-a%20translational%20perspective&author=A%20Agrawal&author=KJ%20Verweij&author=NA%20Gillespie&author=AC%20Heath&author=CN%20Lessov-Schlaggar&volume=2&publication_year=2012&pages=e140&pmid=22806211&doi=10.1038/tp.2012.54&)

6. Nelson EC, Lynskey MT, Heath AC, Wray N, Agrawal A, Shand FL, et al. Association of OPRD1 polymorphisms with heroin dependence in a large case-control series. Addict Biol. 2014;19:111–121. doi: 10.1111/j.1369-1600.2012.00445.x. Epub 2012 Apr 13.  [DOI](https://doi.org/10.1111/j.1369-1600.2012.00445.x) | [PMC free article](/articles/PMC3867542/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22500942/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Addict%20Biol&title=Association%20of%20OPRD1%20polymorphisms%20with%20heroin%20dependence%20in%20a%20large%20case-control%20series&author=EC%20Nelson&author=MT%20Lynskey&author=AC%20Heath&author=N%20Wray&author=A%20Agrawal&volume=19&publication_year=2014&pages=111-121&pmid=22500942&doi=10.1111/j.1369-1600.2012.00445.x&)

7. Nelson EC, Lynskey MT, Heath AC, Wray N, Agrawal A, Shand F, et al. ANKK1, TTC12, and NCAM1 polymorphisms and heroin dependence: importance of considering drug exposure. JAMA Psychiatry. 2013;70:325–333. doi: 10.1001/jamapsychiatry.2013.282.  [DOI](https://doi.org/10.1001/jamapsychiatry.2013.282) | [PMC free article](/articles/PMC3789525/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23303482/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA%20Psychiatry&title=ANKK1,%20TTC12,%20and%20NCAM1%20polymorphisms%20and%20heroin%20dependence:%20importance%20of%20considering%20drug%20exposure&author=EC%20Nelson&author=MT%20Lynskey&author=AC%20Heath&author=N%20Wray&author=A%20Agrawal&volume=70&publication_year=2013&pages=325-333&pmid=23303482&doi=10.1001/jamapsychiatry.2013.282&)

8. Peng S, Du J, Jiang H, Fu Y, Chen H, Sun H, et al. The dopamine receptor D1 gene is associated with the length of interval between first heroin use and onset of dependence in Chinese Han heroin addicts. J Neural Transm. 2013;120:1591–1598. doi: 10.1007/s00702-013-1029-6.  [DOI](https://doi.org/10.1007/s00702-013-1029-6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23661099/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Neural%20Transm&title=The%20dopamine%20receptor%20D1%20gene%20is%20associated%20with%20the%20length%20of%20interval%20between%20first%20heroin%20use%20and%20onset%20of%20dependence%20in%20Chinese%20Han%20heroin%20addicts&author=S%20Peng&author=J%20Du&author=H%20Jiang&author=Y%20Fu&author=H%20Chen&volume=120&publication_year=2013&pages=1591-1598&pmid=23661099&doi=10.1007/s00702-013-1029-6&)

9. Wellcome Trust Case Control Consortium. Genome-wide association study of 14,000 cases of seven common diseases and 3,000 shared controls. Nature. 2007;447:661–678. doi: 10.1038/nature05911.  [DOI](https://doi.org/10.1038/nature05911) | [PMC free article](/articles/PMC2719288/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17554300/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Genome-wide%20association%20study%20of%2014,000%20cases%20of%20seven%20common%20diseases%20and%203,000%20shared%20controls&volume=447&publication_year=2007&pages=661-678&pmid=17554300&doi=10.1038/nature05911&)

10. Nutt D, King LA, Saulsbury W, Blakemore C. Development of a rational scale to assess the harm of drugs of potential misuse. Lancet. 2007;369:1047–1053. doi: 10.1016/S0140-6736(07)60464-4.  [DOI](https://doi.org/10.1016/S0140-6736(07)60464-4) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17382831/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Development%20of%20a%20rational%20scale%20to%20assess%20the%20harm%20of%20drugs%20of%20potential%20misuse&author=D%20Nutt&author=LA%20King&author=W%20Saulsbury&author=C%20Blakemore&volume=369&publication_year=2007&pages=1047-1053&pmid=17382831&doi=10.1016/S0140-6736(07)60464-4&)

11. Linares OA. The Linares addictive potential model. Journal of Pharmacy and Nutrition Sciences. 2012;2:132–139.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Pharmacy%20and%20Nutrition%20Sciences&title=The%20Linares%20addictive%20potential%20model&author=OA%20Linares&volume=2&publication_year=2012&pages=132-139&)

12. Anthony JC, Warner LA, Kessler RC. Comparative epidemiology of dependence on tobacco, alcohol, controlled substances, and inhalants basic findings From the National Comorbidity Survey. Exp Clin Psychopharmacol. 1994;2:244–268.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Exp%20Clin%20Psychopharmacol&title=Comparative%20epidemiology%20of%20dependence%20on%20tobacco,%20alcohol,%20controlled%20substances,%20and%20inhalants%20basic%20findings%20From%20the%20National%20Comorbidity%20Survey&author=JC%20Anthony&author=LA%20Warner&author=RC%20Kessler&volume=2&publication_year=1994&pages=244-268&)

13. Saccone SF, Hinrichs AL, Saccone NL, Chase GA, Konvicka K, Madden PAF, et al. Cholinergic nicotinic receptor genes implicated in a nicotine dependence association study targeting 348 candidate genes with 3713 SNPs. Hum Mol Genet. 2007;16:36–49. doi: 10.1093/hmg/ddl438.  [DOI](https://doi.org/10.1093/hmg/ddl438) | [PMC free article](/articles/PMC2270437/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17135278/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mol%20Genet&title=Cholinergic%20nicotinic%20receptor%20genes%20implicated%20in%20a%20nicotine%20dependence%20association%20study%20targeting%20348%20candidate%20genes%20with%203713%20SNPs&author=SF%20Saccone&author=AL%20Hinrichs&author=NL%20Saccone&author=GA%20Chase&author=K%20Konvicka&volume=16&publication_year=2007&pages=36-49&pmid=17135278&doi=10.1093/hmg/ddl438&)

14. Thorgeirsson TE, Geller F, Sulem P, Rafnar T, Wiste A, Magnusson KP, et al. A variant associated with nicotine dependence, lung cancer and peripheral arterial disease. Nature. 2008;452:638–642. doi: 10.1038/nature06846.  [DOI](https://doi.org/10.1038/nature06846) | [PMC free article](/articles/PMC4539558/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18385739/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=A%20variant%20associated%20with%20nicotine%20dependence,%20lung%20cancer%20and%20peripheral%20arterial%20disease&author=TE%20Thorgeirsson&author=F%20Geller&author=P%20Sulem&author=T%20Rafnar&author=A%20Wiste&volume=452&publication_year=2008&pages=638-642&pmid=18385739&doi=10.1038/nature06846&)

15. Liu JZ, Tozzi F, Waterworth DM, Pillai SG, Muglia P, Middleton L, et al. Meta-analysis and imputation refines the association of 15q25 with smoking quantity. Nat Genet. 2010;42:436–440. doi: 10.1038/ng.572.  [DOI](https://doi.org/10.1038/ng.572) | [PMC free article](/articles/PMC3612983/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20418889/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Meta-analysis%20and%20imputation%20refines%20the%20association%20of%2015q25%20with%20smoking%20quantity&author=JZ%20Liu&author=F%20Tozzi&author=DM%20Waterworth&author=SG%20Pillai&author=P%20Muglia&volume=42&publication_year=2010&pages=436-440&pmid=20418889&doi=10.1038/ng.572&)

16. Thorgeirsson TE, Gudbjartsson DF, Surakka I, Vink JM, Amin N, Geller F, et al. Sequence variants at CHRNB3-CHRNA6 and CYP2A6 affect smoking behavior. Nat Genet. 2010;42:448–453. doi: 10.1038/ng.573.  [DOI](https://doi.org/10.1038/ng.573) | [PMC free article](/articles/PMC3080600/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20418888/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Sequence%20variants%20at%20CHRNB3-CHRNA6%20and%20CYP2A6%20affect%20smoking%20behavior&author=TE%20Thorgeirsson&author=DF%20Gudbjartsson&author=I%20Surakka&author=JM%20Vink&author=N%20Amin&volume=42&publication_year=2010&pages=448-453&pmid=20418888&doi=10.1038/ng.573&)

17. Tobacco and Genetics Consortium. Genome-wide meta-analyses identify multiple loci associated with smoking behavior. Nat Genet. 2010;42:441–447. doi: 10.1038/ng.571.  [DOI](https://doi.org/10.1038/ng.571) | [PMC free article](/articles/PMC2914600/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20418890/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Genome-wide%20meta-analyses%20identify%20multiple%20loci%20associated%20with%20smoking%20behavior&volume=42&publication_year=2010&pages=441-447&pmid=20418890&doi=10.1038/ng.571&)

18. Bierut LJ, Goate AM, Breslau N, Johnson EO, Bertelsen S, Fox L, et al. ADH1B is associated with alcohol dependence and alcohol consumption in populations of European and African ancestry. Mol Psychiatry. 2012;17:445–450. doi: 10.1038/mp.2011.124. Epub 2011 Oct 4.  [DOI](https://doi.org/10.1038/mp.2011.124) | [PMC free article](/articles/PMC3252425/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21968928/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Psychiatry&title=ADH1B%20is%20associated%20with%20alcohol%20dependence%20and%20alcohol%20consumption%20in%20populations%20of%20European%20and%20African%20ancestry&author=LJ%20Bierut&author=AM%20Goate&author=N%20Breslau&author=EO%20Johnson&author=S%20Bertelsen&volume=17&publication_year=2012&pages=445-450&pmid=21968928&doi=10.1038/mp.2011.124&)

19. Gelernter J, Kranzler HR, Sherva R, Almasy L, Koesterer R, Smith AH, et al. Genome-wide association study of alcohol dependence:significant findings in African- and European-Americans including novel risk loci. Mol Psychiatry. 2014;19:41–49. doi: 10.1038/mp.2013.145. Epub 2013 Oct 29.  [DOI](https://doi.org/10.1038/mp.2013.145) | [PMC free article](/articles/PMC4165335/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24166409/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Psychiatry&title=Genome-wide%20association%20study%20of%20alcohol%20dependence:significant%20findings%20in%20African-%20and%20European-Americans%20including%20novel%20risk%20loci&author=J%20Gelernter&author=HR%20Kranzler&author=R%20Sherva&author=L%20Almasy&author=R%20Koesterer&volume=19&publication_year=2014&pages=41-49&pmid=24166409&doi=10.1038/mp.2013.145&)

20. Shand FL, Degenhardt L, Slade T, Nelson EC. Sex differences amongst dependent heroin users: histories, clinical characteristics and predictors of other substance dependence. Addict Behav. 2011;36:27–36. doi: 10.1016/j.addbeh.2010.08.008.  [DOI](https://doi.org/10.1016/j.addbeh.2010.08.008) | [PMC free article](/articles/PMC2981789/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20833480/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Addict%20Behav&title=Sex%20differences%20amongst%20dependent%20heroin%20users:%20histories,%20clinical%20characteristics%20and%20predictors%20of%20other%20substance%20dependence&author=FL%20Shand&author=L%20Degenhardt&author=T%20Slade&author=EC%20Nelson&volume=36&publication_year=2011&pages=27-36&pmid=20833480&doi=10.1016/j.addbeh.2010.08.008&)

21. Bierut LJ, Agrawal A, Bucholz KK, Doheny KF, Laurie C, Pugh E, et al. A genome-wide association study of alcohol dependence. Proc Natl Acad Sci U S A. 2010;107:5082–5087. doi: 10.1073/pnas.0911109107.  [DOI](https://doi.org/10.1073/pnas.0911109107) | [PMC free article](/articles/PMC2841942/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20202923/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&title=A%20genome-wide%20association%20study%20of%20alcohol%20dependence&author=LJ%20Bierut&author=A%20Agrawal&author=KK%20Bucholz&author=KF%20Doheny&author=C%20Laurie&volume=107&publication_year=2010&pages=5082-5087&pmid=20202923&doi=10.1073/pnas.0911109107&)

22. Bennett SN, Caporaso N, Fitzpatrick AL, Agrawal A, Barnes K, Boyd HA, et al. Phenotype harmonization and cross-study collaboration in GWAS consortia: the GENEVA experience. Genet Epidemiol. 2011;35:159–173. doi: 10.1002/gepi.20564.  [DOI](https://doi.org/10.1002/gepi.20564) | [PMC free article](/articles/PMC3055921/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21284036/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genet%20Epidemiol&title=Phenotype%20harmonization%20and%20cross-study%20collaboration%20in%20GWAS%20consortia:%20the%20GENEVA%20experience&author=SN%20Bennett&author=N%20Caporaso&author=AL%20Fitzpatrick&author=A%20Agrawal&author=K%20Barnes&volume=35&publication_year=2011&pages=159-173&pmid=21284036&doi=10.1002/gepi.20564&)

23. Gelernter J, Kranzler HR, Sherva R, Koesterer R, Almasy L, Zhao H, et al. Genome-wide association study of opioid cependence: multiple associations mapped to calcium and potassium pathways. Biol Psychiatry. 2014;76:66–74. doi: 10.1016/j.biopsych.2013.08.034. Epub 2013 Oct 19.  [DOI](https://doi.org/10.1016/j.biopsych.2013.08.034) | [PMC free article](/articles/PMC3992201/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24143882/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biol%20Psychiatry&title=Genome-wide%20association%20study%20of%20opioid%20cependence:%20multiple%20associations%20mapped%20to%20calcium%20and%20potassium%20pathways&author=J%20Gelernter&author=HR%20Kranzler&author=R%20Sherva&author=R%20Koesterer&author=L%20Almasy&volume=76&publication_year=2014&pages=66-74&pmid=24143882&doi=10.1016/j.biopsych.2013.08.034&)

24. Gelernter J, Sherva R, Koesterer R, Almasy L, Zhao H, Kranzler HR, et al. Genome-wide association study of cocaine dependence and related traits: FAM53B identified as a risk gene. Mol Psychiatry. 2014;19:717–723. doi: 10.1038/mp.2013.99. Epub 2013 Aug 20.  [DOI](https://doi.org/10.1038/mp.2013.99) | [PMC free article](/articles/PMC3865158/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23958962/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Psychiatry&title=Genome-wide%20association%20study%20of%20cocaine%20dependence%20and%20related%20traits:%20FAM53B%20identified%20as%20a%20risk%20gene&author=J%20Gelernter&author=R%20Sherva&author=R%20Koesterer&author=L%20Almasy&author=H%20Zhao&volume=19&publication_year=2014&pages=717-723&pmid=23958962&doi=10.1038/mp.2013.99&)

25. Okvist A, Fagergren P, Whittard J, Garcia-Osta A, Drakenberg K, Horvath MC, et al. Dysregulated postsynaptic density and endocytic zone in the amygdala of human heroin and cocaine abusers. Biol Psychiatry. 2011;69:245–252. doi: 10.1016/j.biopsych.2010.09.037. Epub 2010 Dec 3.  [DOI](https://doi.org/10.1016/j.biopsych.2010.09.037) | [PMC free article](/articles/PMC3017476/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21126734/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biol%20Psychiatry&title=Dysregulated%20postsynaptic%20density%20and%20endocytic%20zone%20in%20the%20amygdala%20of%20human%20heroin%20and%20cocaine%20abusers&author=A%20Okvist&author=P%20Fagergren&author=J%20Whittard&author=A%20Garcia-Osta&author=K%20Drakenberg&volume=69&publication_year=2011&pages=245-252&pmid=21126734&doi=10.1016/j.biopsych.2010.09.037&)

26. Cai YQ, Wang W, Hou YY, Zhang Z, Xie J, Pan ZZ. Central amygdala GluA1 facilitates associative learning of opioid reward. J Neurosci. 2013;33:1577–1588. doi: 10.1523/JNEUROSCI.1749-12.2013.  [DOI](https://doi.org/10.1523/JNEUROSCI.1749-12.2013) | [PMC free article](/articles/PMC3711547/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23345231/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Neurosci&title=Central%20amygdala%20GluA1%20facilitates%20associative%20learning%20of%20opioid%20reward&author=YQ%20Cai&author=W%20Wang&author=YY%20Hou&author=Z%20Zhang&author=J%20Xie&volume=33&publication_year=2013&pages=1577-1588&pmid=23345231&doi=10.1523/JNEUROSCI.1749-12.2013&)

27. Van den Oever MC, Goriounova NA, Li KW, Van der Schors RC, Binnekade R, Schoffelmeer AN, et al. Prefrontal cortex AMPA receptor plasticity is crucial for cue-induced relapse to heroin-seeking. Nat Neurosci. 2008;11:1053–1058. doi: 10.1038/nn.2165.  [DOI](https://doi.org/10.1038/nn.2165) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19160503/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Neurosci&title=Prefrontal%20cortex%20AMPA%20receptor%20plasticity%20is%20crucial%20for%20cue-induced%20relapse%20to%20heroin-seeking&author=MC%20Van%20den%20Oever&author=NA%20Goriounova&author=KW%20Li&author=RC%20Van%20der%20Schors&author=R%20Binnekade&volume=11&publication_year=2008&pages=1053-1058&pmid=19160503&doi=10.1038/nn.2165&)

28. Xia Y, Portugal GS, Fakira AK, Melyan Z, Neve R, Lee HT, et al. Hippocampal GluA1-containing AMPA receptors mediate context-dependent sensitization to morphine. J Neurosci. 2011;31:16279–16291. doi: 10.1523/JNEUROSCI.3835-11.2011.  [DOI](https://doi.org/10.1523/JNEUROSCI.3835-11.2011) | [PMC free article](/articles/PMC3235051/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22072679/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Neurosci&title=Hippocampal%20GluA1-containing%20AMPA%20receptors%20mediate%20context-dependent%20sensitization%20to%20morphine&author=Y%20Xia&author=GS%20Portugal&author=AK%20Fakira&author=Z%20Melyan&author=R%20Neve&volume=31&publication_year=2011&pages=16279-16291&pmid=22072679&doi=10.1523/JNEUROSCI.3835-11.2011&)

29. Aitta-aho T, Möykkynen TP, Panhelainen AE, Vekovischeva OY, Bäckström P, Korpi ER. Importance of GluA1 subunit-containing AMPA glutamate receptors for morphine state-dependency. PLoS One. 2012;7:e38325. doi: 10.1371/journal.pone.0038325.  [DOI](https://doi.org/10.1371/journal.pone.0038325) | [PMC free article](/articles/PMC3365010/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22675452/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=Importance%20of%20GluA1%20subunit-containing%20AMPA%20glutamate%20receptors%20for%20morphine%20state-dependency&author=T%20Aitta-aho&author=TP%20M%C3%B6ykkynen&author=AE%20Panhelainen&author=OY%20Vekovischeva&author=P%20B%C3%A4ckstr%C3%B6m&volume=7&publication_year=2012&pages=e38325&pmid=22675452&doi=10.1371/journal.pone.0038325&)

30. Bucholz KK, Cadoret R, Cloninger CR, Dinwiddie SH, Hesselbrock VM, Nurnberger JI, Jr, et al. A new, semi-structured psychiatric interview for use in genetic linkage studies: a report on the reliability of the SSAGA. J Stud Alcohol. 1994;55:149–158. doi: 10.15288/jsa.1994.55.149.  [DOI](https://doi.org/10.15288/jsa.1994.55.149) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8189735/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Stud%20Alcohol&title=A%20new,%20semi-structured%20psychiatric%20interview%20for%20use%20in%20genetic%20linkage%20studies:%20a%20report%20on%20the%20reliability%20of%20the%20SSAGA&author=KK%20Bucholz&author=R%20Cadoret&author=CR%20Cloninger&author=SH%20Dinwiddie&author=VM%20Hesselbrock&volume=55&publication_year=1994&pages=149-158&pmid=8189735&doi=10.15288/jsa.1994.55.149&)

31. Patterson N, Price AL, Reich D. Population structure and eigenanalysis. PLoS Genet. 2006;2:e190. doi: 10.1371/journal.pgen.0020190.  [DOI](https://doi.org/10.1371/journal.pgen.0020190) | [PMC free article](/articles/PMC1713260/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17194218/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet&title=Population%20structure%20and%20eigenanalysis&author=N%20Patterson&author=AL%20Price&author=D%20Reich&volume=2&publication_year=2006&pages=e190&pmid=17194218&doi=10.1371/journal.pgen.0020190&)

32. Purcell S, Neale B, Todd-Brown K, Thomas L, Ferreira MA, Bender D, et al. PLINK: a tool set for whole-genome association and population-based linkage analyses. Am J Hum Genet. 2007;81:559–575. doi: 10.1086/519795.  [DOI](https://doi.org/10.1086/519795) | [PMC free article](/articles/PMC1950838/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17701901/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=PLINK:%20a%20tool%20set%20for%20whole-genome%20association%20and%20population-based%20linkage%20analyses&author=S%20Purcell&author=B%20Neale&author=K%20Todd-Brown&author=L%20Thomas&author=MA%20Ferreira&volume=81&publication_year=2007&pages=559-575&pmid=17701901&doi=10.1086/519795&)

33. Willer CJ, Li Y, Abecasis GR. METAL: fast and efficient meta-analysis of genomewide association scans. Bioinformatics. 2010;26:2190–2191. doi: 10.1093/bioinformatics/btq340.  [DOI](https://doi.org/10.1093/bioinformatics/btq340) | [PMC free article](/articles/PMC2922887/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20616382/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=METAL:%20fast%20and%20efficient%20meta-analysis%20of%20genomewide%20association%20scans&author=CJ%20Willer&author=Y%20Li&author=GR%20Abecasis&volume=26&publication_year=2010&pages=2190-2191&pmid=20616382&doi=10.1093/bioinformatics/btq340&)

34. So HC, Gui AH, Cherny SS, Sham PC. Evaluating the heritability explained by known susceptibility variants: a survey of ten complex diseases. Genet Epidemiol. 2011;35:310–317. doi: 10.1002/gepi.20579.  [DOI](https://doi.org/10.1002/gepi.20579) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21374718/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genet%20Epidemiol&title=Evaluating%20the%20heritability%20explained%20by%20known%20susceptibility%20variants:%20a%20survey%20of%20ten%20complex%20diseases&author=HC%20So&author=AH%20Gui&author=SS%20Cherny&author=PC%20Sham&volume=35&publication_year=2011&pages=310-317&pmid=21374718&doi=10.1002/gepi.20579&)

35. Hall WD, Ross JE, Lynskey MT, Law MG, Degenhardt LJ. How many dependent heroin users are there in Australia? Med J Aust. 2000;173:528–531. doi: 10.5694/j.1326-5377.2000.tb139321.x.  [DOI](https://doi.org/10.5694/j.1326-5377.2000.tb139321.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11194736/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Med%20J%20Aust&title=How%20many%20dependent%20heroin%20users%20are%20there%20in%20Australia?&author=WD%20Hall&author=JE%20Ross&author=MT%20Lynskey&author=MG%20Law&author=LJ%20Degenhardt&volume=173&publication_year=2000&pages=528-531&pmid=11194736&doi=10.5694/j.1326-5377.2000.tb139321.x&)

36. Degenhardt L, Hall W. The extent of illicit drug use and dependence, and their contribution to the global burden of disease. Lancet. 2012;379:55–70. doi: 10.1016/S0140-6736(11)61138-0.  [DOI](https://doi.org/10.1016/S0140-6736(11)61138-0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22225671/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=The%20extent%20of%20illicit%20drug%20use%20and%20dependence,%20and%20their%20contribution%20to%20the%20global%20burden%20of%20disease&author=L%20Degenhardt&author=W%20Hall&volume=379&publication_year=2012&pages=55-70&pmid=22225671&doi=10.1016/S0140-6736(11)61138-0&)

37. Whalen PJ, Phelps EA, editors. The Human Amygdala. Guilford Press; New York: 2009.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=The%20Human%20Amygdala&publication_year=2009&)

38. Benjamini Y, Hochberg Y. Controlling the false discovery rate: a practical and powerful approach to multiple testing. J R Stat Soc Ser B Stat Methodol. 1995;57:289–300.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20R%20Stat%20Soc%20Ser%20B%20Stat%20Methodol&title=Controlling%20the%20false%20discovery%20rate:%20a%20practical%20and%20powerful%20approach%20to%20multiple%20testing&author=Y%20Benjamini&author=Y%20Hochberg&volume=57&publication_year=1995&pages=289-300&)

39. Sun H, Maze I, Dietz DM, Scobie KN, Kennedy PJ, Damez-Werno D, et al. Morphine epigenomically regulates behavior through alterations in histone H3 lysine 9 dimethylation in the nucleus accumbens. J Neurosci. 2012;32:17454–17464. doi: 10.1523/JNEUROSCI.1357-12.2012.  [DOI](https://doi.org/10.1523/JNEUROSCI.1357-12.2012) | [PMC free article](/articles/PMC3516048/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23197736/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Neurosci&title=Morphine%20epigenomically%20regulates%20behavior%20through%20alterations%20in%20histone%20H3%20lysine%209%20dimethylation%20in%20the%20nucleus%20accumbens&author=H%20Sun&author=I%20Maze&author=DM%20Dietz&author=KN%20Scobie&author=PJ%20Kennedy&volume=32&publication_year=2012&pages=17454-17464&pmid=23197736&doi=10.1523/JNEUROSCI.1357-12.2012&)

40. Zhou X, Maricque B, Xie M, Li D, Sundaram V, Martin EA, et al. The Human Epigenome Browser at Washington University. Nat Methods. 2011;8:989–990. doi: 10.1038/nmeth.1772.  [DOI](https://doi.org/10.1038/nmeth.1772) | [PMC free article](/articles/PMC3552640/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22127213/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Methods&title=The%20Human%20Epigenome%20Browser%20at%20Washington%20University&author=X%20Zhou&author=B%20Maricque&author=M%20Xie&author=D%20Li&author=V%20Sundaram&volume=8&publication_year=2011&pages=989-990&pmid=22127213&doi=10.1038/nmeth.1772&)

41. Zhou X, Lowdon RF, Li D, Lawson HA, Madden PAF, Costello JF, et al. Exploring long-range genome interactions using the WashU Epigenome Browser. Nat Methods. 2013;10:375–376. doi: 10.1038/nmeth.2440.  [DOI](https://doi.org/10.1038/nmeth.2440) | [PMC free article](/articles/PMC3820286/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23629413/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Methods&title=Exploring%20long-range%20genome%20interactions%20using%20the%20WashU%20Epigenome%20Browser&author=X%20Zhou&author=RF%20Lowdon&author=D%20Li&author=HA%20Lawson&author=PAF%20Madden&volume=10&publication_year=2013&pages=375-376&pmid=23629413&doi=10.1038/nmeth.2440&)

42. Zhou X, Li D, Zhang B, Lowdon RF, Rockweiler NB, Sears RL, et al. Epigenomic annotation of genetic variants using the Roadmap Epigenome Browser. Nat Biotechnol. 2015 Feb 18; doi: 10.1038/nbt.3158. [Epub ahead of print]  [DOI](https://doi.org/10.1038/nbt.3158) | [PMC free article](/articles/PMC4467764/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25690851/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Biotechnol&title=Epigenomic%20annotation%20of%20genetic%20variants%20using%20the%20Roadmap%20Epigenome%20Browser&author=X%20Zhou&author=D%20Li&author=B%20Zhang&author=RF%20Lowdon&author=NB%20Rockweiler&publication_year=2015&pmid=25690851&doi=10.1038/nbt.3158&)

43. Yeh SH, Mao SC, Lin HC, Gean PW. Synaptic expression of glutamate receptor after encoding of fear memory in the rat amygdala. Mol Pharmacol. 2006;69:299–308. doi: 10.1124/mol.105.017194. Epub 2005 Oct 11.  [DOI](https://doi.org/10.1124/mol.105.017194) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16219906/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Pharmacol&title=Synaptic%20expression%20of%20glutamate%20receptor%20after%20encoding%20of%20fear%20memory%20in%20the%20rat%20amygdala&author=SH%20Yeh&author=SC%20Mao&author=HC%20Lin&author=PW%20Gean&volume=69&publication_year=2006&pages=299-308&pmid=16219906&doi=10.1124/mol.105.017194&)

44. Dalton GL, Wang YT, Floresco SB, Phillips AG. Disruption of AMPA receptor endocytosis impairs the extinction, but not acquisition of learned fear. Neuropsychopharmacology. 2008;33:2416–2426. doi: 10.1038/sj.npp.1301642. Epub 2007 Nov 28.  [DOI](https://doi.org/10.1038/sj.npp.1301642) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18046303/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neuropsychopharmacology&title=Disruption%20of%20AMPA%20receptor%20endocytosis%20impairs%20the%20extinction,%20but%20not%20acquisition%20of%20learned%20fear&author=GL%20Dalton&author=YT%20Wang&author=SB%20Floresco&author=AG%20Phillips&volume=33&publication_year=2008&pages=2416-2426&pmid=18046303&doi=10.1038/sj.npp.1301642&)

45. Nedelescu H, Kelso CM, Lázaro-Muñoz G, Purpura M, Cain CK, Ledoux JE, et al. Endogenous GluR1-containing AMPA receptors translocate to asymmetric synapses in the lateral amygdala during the early phase of fear memory formation: an electron microscopic immunocytochemical study. J Comp Neurol. 2010;518:4723–4739. doi: 10.1002/cne.22472.  [DOI](https://doi.org/10.1002/cne.22472) | [PMC free article](/articles/PMC3613289/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20963825/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Comp%20Neurol&title=Endogenous%20GluR1-containing%20AMPA%20receptors%20translocate%20to%20asymmetric%20synapses%20in%20the%20lateral%20amygdala%20during%20the%20early%20phase%20of%20fear%20memory%20formation:%20an%20electron%20microscopic%20immunocytochemical%20study&author=H%20Nedelescu&author=CM%20Kelso&author=G%20L%C3%A1zaro-Mu%C3%B1oz&author=M%20Purpura&author=CK%20Cain&volume=518&publication_year=2010&pages=4723-4739&pmid=20963825&doi=10.1002/cne.22472&)

46. Nikolova YS, Koenen KC, Galea S, Wang CM, Seney ML, Sibille E, et al. Beyond genotype: serotonin transporter epigenetic modification predicts human brain function. Nat Neurosci. 2014;17:1153–1155. doi: 10.1038/nn.3778.  [DOI](https://doi.org/10.1038/nn.3778) | [PMC free article](/articles/PMC4146649/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25086606/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Neurosci&title=Beyond%20genotype:%20serotonin%20transporter%20epigenetic%20modification%20predicts%20human%20brain%20function&author=YS%20Nikolova&author=KC%20Koenen&author=S%20Galea&author=CM%20Wang&author=ML%20Seney&volume=17&publication_year=2014&pages=1153-1155&pmid=25086606&doi=10.1038/nn.3778&)

47. Plichta MM, Grimm O, Morgen K, Mier D, Sauer C, Haddad L, et al. Amygdala habituation: A reliable fMRI phenotype. Neuroimage. 2014;103C:383–390. doi: 10.1016/j.neuroimage.2014.09.059.  [DOI](https://doi.org/10.1016/j.neuroimage.2014.09.059) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25284303/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neuroimage&title=Amygdala%20habituation:%20A%20reliable%20fMRI%20phenotype&author=MM%20Plichta&author=O%20Grimm&author=K%20Morgen&author=D%20Mier&author=C%20Sauer&volume=103C&publication_year=2014&pages=383-390&pmid=25284303&doi=10.1016/j.neuroimage.2014.09.059&)

48. Chu LF, Liang DY, Li X, Sahbaie P, D’arcy N, Liao G, et al. From mouse to man: the 5-HT3 receptor modulates physical dependence on opioid narcotics. Pharmacogenet Genomics. 2009;19:193–205. doi: 10.1097/FPC.0b013e328322e73d.  [DOI](https://doi.org/10.1097/FPC.0b013e328322e73d) | [PMC free article](/articles/PMC2730361/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19214139/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=From%20mouse%20to%20man:%20the%205-HT3%20receptor%20modulates%20physical%20dependence%20on%20opioid%20narcotics&author=LF%20Chu&author=DY%20Liang&author=X%20Li&author=P%20Sahbaie&author=N%20D%E2%80%99arcy&volume=19&publication_year=2009&pages=193-205&pmid=19214139&doi=10.1097/FPC.0b013e328322e73d&)

49. Zheng M, Dill D, Peltz G. A better prognosis for genetic association studies in mice. Trends Genet. 2012;28:62–69. doi: 10.1016/j.tig.2011.10.006. Epub 2011 Nov 24.  [DOI](https://doi.org/10.1016/j.tig.2011.10.006) | [PMC free article](/articles/PMC3268904/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22118772/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Trends%20Genet&title=A%20better%20prognosis%20for%20genetic%20association%20studies%20in%20mice&author=M%20Zheng&author=D%20Dill&author=G%20Peltz&volume=28&publication_year=2012&pages=62-69&pmid=22118772&doi=10.1016/j.tig.2011.10.006&)

50. Liang DY, Zheng M, Sun Y, Sahbaie P, Low SA, Peltz G, et al. The Netrin-1 receptor DCC is a regulator of maladaptive responses to chronic morphine administration. BMC Genomics. 2014;15:345. doi: 10.1186/1471-2164-15-345.  [DOI](https://doi.org/10.1186/1471-2164-15-345) | [PMC free article](/articles/PMC4038717/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24884839/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Genomics&title=The%20Netrin-1%20receptor%20DCC%20is%20a%20regulator%20of%20maladaptive%20responses%20to%20chronic%20morphine%20administration&author=DY%20Liang&author=M%20Zheng&author=Y%20Sun&author=P%20Sahbaie&author=SA%20Low&volume=15&publication_year=2014&pages=345&pmid=24884839&doi=10.1186/1471-2164-15-345&)

51. Schwenk J, Baehrens D, Haupt A, Bildl W, Boudkkazi S, Roeper J, et al. Regional diversity and developmental dynamics of the AMPA-receptor proteome in the mammalian brain. Neuron. 2014;84:41–54. doi: 10.1016/j.neuron.2014.08.044.  [DOI](https://doi.org/10.1016/j.neuron.2014.08.044) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25242221/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neuron&title=Regional%20diversity%20and%20developmental%20dynamics%20of%20the%20AMPA-receptor%20proteome%20in%20the%20mammalian%20brain&author=J%20Schwenk&author=D%20Baehrens&author=A%20Haupt&author=W%20Bildl&author=S%20Boudkkazi&volume=84&publication_year=2014&pages=41-54&pmid=25242221&doi=10.1016/j.neuron.2014.08.044&)

52. Sumioka A, Brown TE, Kato AS, Bredt DS, Kauer JA, Tomita S. PDZ binding of TARPγ-8 controls synaptic transmission but not synaptic plasticity. Nat Neurosci. 2011;14:1410–1412. doi: 10.1038/nn.2952.  [DOI](https://doi.org/10.1038/nn.2952) | [PMC free article](/articles/PMC3206644/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22002768/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Neurosci&title=PDZ%20binding%20of%20TARP%CE%B3-8%20controls%20synaptic%20transmission%20but%20not%20synaptic%20plasticity&author=A%20Sumioka&author=TE%20Brown&author=AS%20Kato&author=DS%20Bredt&author=JA%20Kauer&volume=14&publication_year=2011&pages=1410-1412&pmid=22002768&doi=10.1038/nn.2952&)

53. Harmel N, Cokic B, Zolles G, Berkefeld H, Mauric V, Fakler B, et al. AMPA receptors commandeer an ancient cargo exporter for use as an auxiliary subunit for signaling. PLoS One. 2012;7:e30681. doi: 10.1371/journal.pone.0030681.  [DOI](https://doi.org/10.1371/journal.pone.0030681) | [PMC free article](/articles/PMC3265512/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22292017/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=AMPA%20receptors%20commandeer%20an%20ancient%20cargo%20exporter%20for%20use%20as%20an%20auxiliary%20subunit%20for%20signaling&author=N%20Harmel&author=B%20Cokic&author=G%20Zolles&author=H%20Berkefeld&author=V%20Mauric&volume=7&publication_year=2012&pages=e30681&pmid=22292017&doi=10.1371/journal.pone.0030681&)

54. Herring BE, Shi Y, Suh YH, Zheng CY, Blankenship SM, Roche KW, et al. Cornichon proteins determine the subunit composition of synaptic AMPA receptors. Neuron. 2013;77:1083–1096. doi: 10.1016/j.neuron.2013.01.017.  [DOI](https://doi.org/10.1016/j.neuron.2013.01.017) | [PMC free article](/articles/PMC3652566/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23522044/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neuron&title=Cornichon%20proteins%20determine%20the%20subunit%20composition%20of%20synaptic%20AMPA%20receptors&author=BE%20Herring&author=Y%20Shi&author=YH%20Suh&author=CY%20Zheng&author=SM%20Blankenship&volume=77&publication_year=2013&pages=1083-1096&pmid=23522044&doi=10.1016/j.neuron.2013.01.017&)

55. Coombs ID, Soto D, Zonouzi M, Renzi M, Shelley C, Farrant M, et al. Cornichons modify channel properties of recombinant and glial AMPA receptors. J Neurosci. 2012;32:9796–9804. doi: 10.1523/JNEUROSCI.0345-12.2012.  [DOI](https://doi.org/10.1523/JNEUROSCI.0345-12.2012) | [PMC free article](/articles/PMC3428874/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22815494/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Neurosci&title=Cornichons%20modify%20channel%20properties%20of%20recombinant%20and%20glial%20AMPA%20receptors&author=ID%20Coombs&author=D%20Soto&author=M%20Zonouzi&author=M%20Renzi&author=C%20Shelley&volume=32&publication_year=2012&pages=9796-9804&pmid=22815494&doi=10.1523/JNEUROSCI.0345-12.2012&)

56. Boudkkazi S, Brechet AJS, Fakler B. Cornichon2 dictates the time course of excitatory transmission at individual hippocampal synapses. Neuron. 2014;82:848–858. doi: 10.1016/j.neuron.2014.03.031.  [DOI](https://doi.org/10.1016/j.neuron.2014.03.031) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24853943/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neuron&title=Cornichon2%20dictates%20the%20time%20course%20of%20excitatory%20transmission%20at%20individual%20hippocampal%20synapses&author=S%20Boudkkazi&author=AJS%20Brechet&author=B%20Fakler&volume=82&publication_year=2014&pages=848-858&pmid=24853943&doi=10.1016/j.neuron.2014.03.031&)

57. Shanks NF, Cais O, Maruo T, Savas JN, Zaika EI, Azumaya CM, et al. Molecular dissection of the interaction between the AMPA receptor and cornichon homolog-3. J Neurosci. 2014;34:12104–12120. doi: 10.1523/JNEUROSCI.0595-14.2014.  [DOI](https://doi.org/10.1523/JNEUROSCI.0595-14.2014) | [PMC free article](/articles/PMC4152608/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25186755/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Neurosci&title=Molecular%20dissection%20of%20the%20interaction%20between%20the%20AMPA%20receptor%20and%20cornichon%20homolog-3&author=NF%20Shanks&author=O%20Cais&author=T%20Maruo&author=JN%20Savas&author=EI%20Zaika&volume=34&publication_year=2014&pages=12104-12120&pmid=25186755&doi=10.1523/JNEUROSCI.0595-14.2014&)

58. GTEx Consortium. Lonsdale J, Thomas J, Salvatore M, Phillips R, Lo E, et al. The Genotype-Tissue Expression (GTEx) project. Nat Genet. 2013;45:580–585. doi: 10.1038/ng.2653.  [DOI](https://doi.org/10.1038/ng.2653) | [PMC free article](/articles/PMC4010069/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23715323/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=The%20Genotype-Tissue%20Expression%20(GTEx)%20project&author=J%20Lonsdale&author=J%20Thomas&author=M%20Salvatore&author=R%20Phillips&author=E%20Lo&volume=45&publication_year=2013&pages=580-585&pmid=23715323&doi=10.1038/ng.2653&)

59. Duffy DL, Iles MM, Glass D, Zhu G, Barrett JH, Höiom V, et al. IRF4 variants have age-specific effects on nevus count and predispose to melanoma. Am J Hum Genet. 2010;87:6–16. doi: 10.1016/j.ajhg.2010.05.017.  [DOI](https://doi.org/10.1016/j.ajhg.2010.05.017) | [PMC free article](/articles/PMC2896771/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20602913/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=IRF4%20variants%20have%20age-specific%20effects%20on%20nevus%20count%20and%20predispose%20to%20melanoma&author=DL%20Duffy&author=MM%20Iles&author=D%20Glass&author=G%20Zhu&author=JH%20Barrett&volume=87&publication_year=2010&pages=6-16&pmid=20602913&doi=10.1016/j.ajhg.2010.05.017&)

60. Gunduz-Cinar O, MacPherson KP, Cinar R, Gamble-George J, Sugden K, Williams B, et al. Convergent translational evidence of a role for anandamide in amygdala-mediated fear extinction, threat processing and stress-reactivity. Mol Psychiatry. 2013;18:813–823. doi: 10.1038/mp.2012.72. Epub 2012 Jun 12.  [DOI](https://doi.org/10.1038/mp.2012.72) | [PMC free article](/articles/PMC3549323/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22688188/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Psychiatry&title=Convergent%20translational%20evidence%20of%20a%20role%20for%20anandamide%20in%20amygdala-mediated%20fear%20extinction,%20threat%20processing%20and%20stress-reactivity&author=O%20Gunduz-Cinar&author=KP%20MacPherson&author=R%20Cinar&author=J%20Gamble-George&author=K%20Sugden&volume=18&publication_year=2013&pages=813-823&pmid=22688188&doi=10.1038/mp.2012.72&)

61. Day CA, JR, Dietze P, Dolan K. Initiation to heroin injecting among heroin users in Sydney, Australia: cross sectional survey. Harm Reduct J. 2005;2:2. doi: 10.1186/1477-7517-2-2.  [DOI](https://doi.org/10.1186/1477-7517-2-2) | [PMC free article](/articles/PMC550668/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15713226/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Harm%20Reduct%20J&title=Initiation%20to%20heroin%20injecting%20among%20heroin%20users%20in%20Sydney,%20Australia:%20cross%20sectional%20survey&author=CA%20Day&author=P%20Dietze&author=K%20Dolan&volume=2&publication_year=2005&pages=2&pmid=15713226&doi=10.1186/1477-7517-2-2&)

62. Des Jarlais DC, Arasteh K, Perlis T, Hagan H, Heckathorn DD, Mcknight C, et al. The transition from injection to non-injection drug use: long-term outcomes among heroin and cocaine users in New York City. Addiction. 2007;102:778–785. doi: 10.1111/j.1360-0443.2007.01764.x.  [DOI](https://doi.org/10.1111/j.1360-0443.2007.01764.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17506155/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Addiction&title=The%20transition%20from%20injection%20to%20non-injection%20drug%20use:%20long-term%20outcomes%20among%20heroin%20and%20cocaine%20users%20in%20New%20York%20City&author=DC%20Des%20Jarlais&author=K%20Arasteh&author=T%20Perlis&author=H%20Hagan&author=DD%20Heckathorn&volume=102&publication_year=2007&pages=778-785&pmid=17506155&doi=10.1111/j.1360-0443.2007.01764.x&)
