# Genome-wide association analysis of anti-TNF drug response in rheumatoid arthritis patients

## Metadata
**Authors:** Maša Umiċeviċ Mirkov, Jing Cui, Sita H Vermeulen, Eli A Stahl, Erik JM Toonen, Remco R Makkinje, Annette T Lee, Tom WJ Huizinga, Renee Allaart, Anne Barton, Xavier Mariette, Corinne Miceli-Richard, Lindsey A Criswell, Paul P Tak, Niek de Vries, Saedis Saevarsdottir, Leonid Padyukov, S Louis Bridges, Dirk-Jan van Schaardenburg, Tim Jansen, Ellen AJ Dutmer, Mart van de Laar, Pilar Barrera, Timothy RDJ Radstake, Piet LCM van Riel, Hans Scheffer, Barbara Franke, Han G Brunner, Robert M Plenge, Peter K Gregersen, Henk-Jan Guchelaar, Marieke JH Coenen
**Journal:** Annals of the rheumatic diseases
**Date:** 2012 Dec 11
**DOI:** [10.1136/annrheumdis-2012-202405](https://doi.org/10.1136/annrheumdis-2012-202405)
**PMID:** 23233654
**PMCID:** PMC4169706
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4169706/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC4169706/pdf/nihms621909.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC4169706/pdf/nihms621909.pdf)

## Abstract

**Background:** 
Treatment strategies blocking tumor necrosis factor (anti-TNF) have proven very successful in patients with rheumatoid arthritis (RA). However, a significant subset of patients does not respond for unknown reasons. Currently there are no means of identifying these patients prior to treatment. This study was aimed at identifying genetic factors predicting anti-TNF treatment outcome in patient with RA using a genome-wide association approach.

**Methods:** 
We conducted a multi-stage, genome-wide association study with a primary analysis of 2,557,253 single nucleotide polymorphisms (SNPs) in 882 RA patients receiving anti-TNF therapy included through the Dutch Rheumatoid Arthritis Monitoring (DREAM) registry and the database of Apotheekzorg. Linear regression analysis of changes in the Disease Activity Score in 28 joints after 14 weeks of treatment was performed using an additive model. Markers with a p<10−3 were selected for replication in 1,821 RA patients from three independent cohorts. Pathway analysis including all SNPs with a p-value < 10−3 was performed using Ingenuity.

**Results:** 
Seven hundred seventy two markers demonstrated evidence of association with treatment outcome in the initial stage. Eight genetic loci showed improved p-value in the overall meta-analysis compared to the first stage, three of which (rs1568885, rs1813443 and rs4411591) showed directional consistency over all four studied cohorts. We were unable to replicate markers previously reported to be associated with anti-TNF outcome. Network analysis indicated strong involvement of biological processes underlying inflammatory response and cell morphology.

**Conclusion:** 
Using a multi-stage strategy, we have identified 8 genetic loci associated with response to anti-TNF treatment. Further studies are required to validate these findings in additional patient collections.

Keywords: anti-TNF, gene polymorphism, pharmacogenetics, rheumatoid arthritis, genome-wide association study

### Background

Treatment strategies blocking tumor necrosis factor (anti-TNF) have proven very successful in patients with rheumatoid arthritis (RA). However, a significant subset of patients does not respond for unknown reasons. Currently there are no means of identifying these patients prior to treatment. This study was aimed at identifying genetic factors predicting anti-TNF treatment outcome in patient with RA using a genome-wide association approach.

### Methods

We conducted a multi-stage, genome-wide association study with a primary analysis of 2,557,253 single nucleotide polymorphisms (SNPs) in 882 RA patients receiving anti-TNF therapy included through the Dutch Rheumatoid Arthritis Monitoring (DREAM) registry and the database of Apotheekzorg. Linear regression analysis of changes in the Disease Activity Score in 28 joints after 14 weeks of treatment was performed using an additive model. Markers with a p<10^−3^ were selected for replication in 1,821 RA patients from three independent cohorts. Pathway analysis including all SNPs with a p-value < 10^−3^ was performed using Ingenuity.

### Results

Seven hundred seventy two markers demonstrated evidence of association with treatment outcome in the initial stage. Eight genetic loci showed improved p-value in the overall meta-analysis compared to the first stage, three of which (rs1568885, rs1813443 and rs4411591) showed directional consistency over all four studied cohorts. We were unable to replicate markers previously reported to be associated with anti-TNF outcome. Network analysis indicated strong involvement of biological processes underlying inflammatory response and cell morphology.

### Conclusion

Using a multi-stage strategy, we have identified 8 genetic loci associated with response to anti-TNF treatment. Further studies are required to validate these findings in additional patient collections.

## Introduction

Rheumatoid arthritis (RA) is a chronic, systemic inflammatory disease characterized by polyarthritis, joint damage and functional disability[[1](#R1)]. RA cannot be cured and treatment is directed towards reducing the symptoms associated with the disease.

Tumor necrosis factor α (TNF α) is a pleiotropic, pro-inflammatory and immunoregulatory cytokine that plays a crucial role in RA[[2](#R2), [3](#R3)]. The introduction of TNF blocking agents, such as infliximab, etanercept and adalimumab, revolutionized therapy of RA, most notably because of the excellent clinical efficacy and ability of these agents to prevent further structural damage in patients who failed to respond to treatment with conventional disease-modifying anti rheumatic drugs (DMARDs) [[4](#R4), [5](#R5)]. Despite this success, a substantial proportion of RA patients (about 30%) treated with TNF inhibitors does not display any significant clinical improvement [[6](#R6), [7](#R7)]. Given the expensive treatment regime and the potential side effects associated with the treatment, the idea of a priori prediction of response to anti-TNF agents in RA patients is a highly relevant topic[[8](#R8), [9](#R9)].

Studies of clinical parameters and biomarkers have identified several factors that influence anti-TNF treatment outcome, including concurrent use of DMARDs, lower baseline health assessment questionnaire (HAQ) score, gender, smoking, serological status, TNF levels at the site of inflammation, and the synovial microarchitecture. However, these factors explain only a relatively small proportion of the observed variance in response (R^2^ = 17–29%) and are, therefore, not suitable to be used as predictors in clinical setting[[9](#R9)–[12](#R12)]. In addition, effort has been put into the identification of genetic markers predicting anti-TNF treatment outcome. Most of these studies are candidate gene-based studies focusing on polymorphisms in genes known to be involved in RA pathogenesis and genes implicated in TNF-α signaling pathways [[13](#R13)]. The most thoroughly investigated gene is *TNFA*encoding TNFα, the target of anti-TNF treatment. Initial studies suggested a role of a variant in the promoter of the gene (−308G>A) in anti-TNF response, although recent meta-analyses do not support this association[[14](#R14), [15](#R15)]. So far, using the candidate gene approaches, the most convincing evidence of association with response to anti-TNF therapy in RA patients is found for an RA risk allele at the *PTPRC* gene locus[[16](#R16), [17](#R17)].

A number of additional potential candidate loci have been suggested based on the results of three genome-wide association studies (GWAS)[[18](#R18)–[20](#R20)]. In a GWAS of 566 RA patients, Plant *et al.* demonstrated evidence of association at 7 genetic loci with response to TNF blockade, two of which mapped within genes: *PDZ domain-containing protein 2 (PDZD2)* and *eyes absent homolog 4 (EYA4)* [[19](#R19)]. In a small study (n=89) by Liu *et al.* association was reported for markers in the *MAFB* and *PON1* gene regions as well as in a region of chromosome 9 that contains the interferon kappa (*IFN-κ*), *MOBKL2B* and *C9orf72* loci. The most compelling candidate for involvement in anti-TNF treatment response in this study is *IFN-κ*since type I IFNs play a definite role in inflammatory disease and autoimmunity[[21](#R21)]. However these results could not be replicated by others [[20](#R20), [22](#R22)]. Krintel *et al.* reported associations of SNPs within a non-coding region surrounded by the TLR4 gene and the DBC1 gene and a marker within the *FOXP1* gene with treatment outcome in a cohort of 196 Danish patients [[20](#R20)].

To determine whether the reported loci reflect true associations, and to search for novel loci that influence differential response to anti-TNF therapy, we performed a genome-wide association study in a cohort of 882 Dutch RA patients treated with anti-TNF therapy.

## Materials and methods

### Patients and study design

A multistage GWAS was performed including 984 RA patients treated with anti-TNF medication (stage 1) with subsequent follow up of the most significant signals in two replication cohorts (stage 2 (n=954) and 3 (n=867)).

For the initial GWAS analysis, patients were recruited through a collaborative effort in which 669 patients were included as part of the Dutch Rheumatoid Arthritis Monitoring (DREAM) registry ([www.dreamregistry.nl](http://www.dreamregistry.nl)) and 315 patients were enrolled through the database of ApotheekZorg which facilitates the Dutch distribution of adalimumab. All patients were diagnosed with RA according to the 1987 revised American College of Rheumatology (ACR) criteria and were treated with anti-TNF according to the indications in the Netherlands; Disease Activity Score 28 (DAS28) > 3.2 and previous failure on at least two disease-modifying antirheumatic drugs (DMARDs), one of which has to be methotrexate (MTX), all patients were biological naïve [[23](#R23)]. We used the DAS28 change at three months as outcome for our analysis. Patients that stopped treatment within the first three months, were not included in the study. All patients gave written informed consent and the study was approved by the ethical committees of the participating hospitals.

For stage 2, data from 954 RA cases treated with anti-TNF were selected from 9 different cohorts as part of the American College of Rheumatology Research and Education Foundation (REF) “Within Our Reach” project – *Autoimmune Biomarkers Collaborative Network (ABCoN), Academic Medical Center Amsterdam (AMC), Behandelstrategieen voor Rheumatoide Arthritis (BeSt), Biological in Rheumatoid arthritis Genetics and Genomics Study Syndicate (BRAGGSS), Brigham Rheumatoid Arthritis Sequential Study (BRASS), Epidemiological Investigation of Rheumatoid Arthritis (EIRA), Immunex Early Rheumatoid Arthritis (ERA) study, Karolinska Institutet (KI) study, READE,* formerly *Jan van Breemen Institute (READE) study, Treatment of Early Aggressive RA (TEAR)* – this collection has been reported previously in[[16](#R16), [24](#R24)].

Finally, stage 3 included two previously described cohorts; (1) Wellcome Trust Case Control Consortium (WTCCC) comprising 595 RA patients from the UK [[19](#R19)] and (2) 272 RA patients from France ascertained through ReAct [[25](#R25)].

### Genotyping and pre-imputation quality control

For stage 1, genotyping was performed using the Illumina HumanHap550-Duo Bead Chip or the Human660W-Quad BeadChips, according to the instructions of the manufacturer (Illumina, Inc, San Diego, USA).

Pre-imputation quality control procedures were applied using PLINK software [[26](#R26)]. Single nucleotide polymorphisms (SNPs) that had minor allele frequency (MAF) <0.05 and call rates <95% were excluded as well as SNPs with extreme departures from Hardy-Weinberg equilibrium (p<1×10^−5^). Subsequently, quality control filtering was performed at the sample level. Four samples were excluded due to gender mismatch with phenotypic data and 21 samples due to a genotyping rate <95%. Cryptic relatedness between study participants was examined by estimating IBD. Seven DNA samples were excluded based on a PI-HAT>0.125. Lastly, principal components were computed to adjust for population stratification using the EIGENSTRAT package [[27](#R27)]; 59 individuals were removed as outliers, based on the EIGENSTRAT default filter. After quality control (QC) 882 individuals were left for analysis. For the replication cohorts the same QC criteria were used.

### Imputation

To obtain a marker set common to all studies and to increase overall coverage of the genome, imputation was performed using HapMap2 release 21 (downloaded from [http://www.sph.umich.edu/csg/abecasis/MACH/download/HapMap-r21.html](http://www.sph.umich.edu/csg/abecasis/MACH/download/HapMap-r21.html)). Haplotype phasing using MaCH software ([http://www.sph.umich.edu/csg/abecasis/MACH/index.html](http://www.sph.umich.edu/csg/abecasis/MACH/index.html)) [[28](#R28)] was followed by genotype imputation by Minimac ([http://genome.sph.umich.edu/wiki/Minimac](http://genome.sph.umich.edu/wiki/Minimac)).

Post-imputation QC criteria were MAF ≥ 1% and good imputation quality, which was defined as RSQR ≥ 0.3. In total, 2,557,253 SNPs were included in the analysis.

### Stage 1 GWAS

The additive genetic effect of each SNP allele on change in DAS28 at 3 months of treatment was estimated using linear regression analysis with adjustment for baseline DAS28 and the first three principal components (PC) derived using EIGENSTRAT. These analyses were performed using the Mach2qtl software package [[29](#R29)] (downloaded from [http://www.sph.umich.edu/csg/abecasis/MACH/download/HapMap-r21.html](http://www.sph.umich.edu/csg/abecasis/MACH/download/HapMap-r21.html)).

The Dutch samples were not genotyped in one run therefore the results were analyzed using a meta-analysis approach that combines study-specific-β-estimates based on the fixed effect model and using the inverse of the variance of the study-specific-β-estimates to weigh the contribution of each study. Calculations were performed in the METAL package ([www.sph.umich.edu/csg/abecasis/metal](http://www.sph.umich.edu/csg/abecasis/metal)). Within-study genomic control (GC) correction was applied to the variance of β-estimates using lambda factors specific to each study (λ_1_ = 1.013, λ_2_ = 1.016, λ_3_ = 0.996).

### SNP selection for replication in stage 2 and 3

Markers demonstrating association with DAS28 change (p<10^−3^) in stage 1 were selected for replication. Pruning of hits based on LD was performed prior to replication: all SNPs with a HapMap CEU pair-wise correlation coefficient (r^2^) > 0.8 with the most strongly associated SNPs in a locus were eliminated. 772 independent loci were left for replication. Replication analysis in stage 2 was carried out using existing GWA scan data from the REF collection. Those SNPs that passed the the p < 0.05 threshold in stage 2 were further evaluated using GWA data from two collections (WTCCC and ReAct) in stage 3. A meta-analysis using the METAL package ([www.sph.umich.edu/csg/abecasis/metal](http://www.sph.umich.edu/csg/abecasis/metal)) was performed.

### Explorative analysis for functional relation between genes identified in stage 1

All markers showing association with DAS change (p<10^−3^) in stage 1 were investigated for functional interactions by Ingenuity Pathway Analysis (IPA) software (Ingenuity Systems, [www.ingenuity.com](http://www.ingenuity.com)) using an unsupervised analysis. IPA computes a score for each network accordingly to the fit of the user’s set of input genes. The score, representing the −log (p-value), indicates the likelihood of Focus Genes (genes harboring associated SNPs) in a network being found together due to a random chance.

## Results

[Figure 1](#F1) presents an overview of our study approach. The baseline characteristics of the patients included in the study are summarized in [Table 1](#T1).

### Figure 1. Study design of a multi-stage GWAS of response to anti-TNF medication in RA patients.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/679c/4169706/e7d9773a45c3/nihms621909f1.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4169706_nihms621909f1.jpg)

We started out with meta-analysis of GWAS data from Dutch cohort comprised of 882 RA patients treated with anti-TNF medication. We selected 772 SNPs that reached p<0.001 that were further followed up in stage 2 samples (N=954 individuals). 38 SNPs out of 768 investigated in stage 2 passed p<0.05 and were further investigated in two separate cohorts in stage 3 (N=595 and N=272 individuals).

### Table 1.

|   | Stage 1 |  |  | Stage 2 | Stage 3 |  |
| --- | --- | --- | --- | --- | --- | --- |
|   | Combined | DREAM | ApotheekZorg | REF Col* | WTCCC | ReAct |
| number | 984 | 669(69.7) | 315(30.3) | 954 | 595 | 272 |
| Gender female, % | 68.6 | 67.8 | 70.4 | 75.6 | 77.3 | 77.9 |
| AntiTNF agent |   |   |   |   |   |   |
| Infliximab | 225(22.9) | 225(33.7) |   | 415(43.5) | 268(45.0) |   |
| Adalimumab | 638(64.8) | 323(48.2) | 315(100) | 174(18.2) | 68(11.4) | 272(100) |
| Etanercept | 121(12.3) | 121(18.1) |   | 365(38.3) | 259(43.6) |   |
| MTX co-medication, % | 73.4 | 69.2 | 82.2 | 65.6 | 85.6 | 50 |
| DAS28 |   |   |   |   |   |   |
| baseline | 5.5 ± 1.2 | 5.3 ± 1.3 | 5.8 ± 1.0 | 5.5 ± 1.2 | 6.7 ± 0.9 | 5.9 ± 1.0 |
| DeltaDAS, 14 weeks | 3.6 ± 1.3 | 3.9 ± 1.3 | 3.1 ± 1.1 |   |   |   |

Table 1 Caption: Study population characteristics

### Genome-wide association analysis

In *stage 1* 2,448,996 SNPs were tested for association with anti-TNF outcome in the Dutch population. Of these SNPs 2,359 showed evidence of association with treatment response (p-value <10^−3^, [Supplementary Table 1](#SD1)). LD pruning reduced the number of SNPs prioritized for replication analysis to 772.

We aimed to replicate the findings in 954 patients from the REF collection for *stage 2* of our study. 768 SNPs passed the QC in the second stage replication cohort. 39 markers showed nominally significant (p < 0.05) association with treatment outcome under an additive model, 20 of which demonstrated directionally consistent association and a resulting improvement of the association signal in a stage 1 and 2 combined meta-analysis ([Supplementary Table 2](#SD2)).

In *stage 3*the 39 SNP stage 2 markers were further inspected for replication in two independent GWA studies, comprised of 595 patients from the UK and 272 patients from France (ReAct), separately. One SNP (rs11642036) failed QC criteria in both replication cohorts, leaving 38 SNPs for analysis. None of the tested SNPs showed nominal association with treatment outcome in these cohorts ([Table 2](#T2)). However, the meta-analysis including all cohorts showed improved association signals for eight SNPs compared to our stage 1 results, three of which, rs1568885, rs1813443 and rs4411591, demonstrate directional consistency over all four studied cohorts ([Table 2](#T2)).

### Table 2.

|   |   |   |   | Stage1(n=882) |  | Stage2(n=954) |  | Stage3(n=867) |  |  |  | Combined metaanalysis |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
|   |  |  |  |  |  |  |  | WTCCC |  | ReAct |  |   |   |
| SNP | Chr | Position | Allele | PGC | β | PGC | β | PGC | β | PGC | β | P | β |
| rs4411591 | 18 | 6540117 | C | 7.33×10−4 | 0.248 | 0.02206 | 0.208 | 0.483 | 0.097 | 0.461 | 0.114 | 5.14 ×10−5 | 0.202 |
| rs7767069 | 6 | 68827284 | A | 3.41 ×10−4 | −0.188 | 0.02839 | −0.161 | 0.532 | 0.057 | 0.086 | −0.203 | 8.34 ×10−5 | −0.144 |
| rs4651370 | 1 | 185505715 | A | 2.54 ×10−4 | 0.256 | 5.28×10−3 | 0.271 | 0.867 | −0.021 | 0.921 | 0.015 | 1.09 ×10−4 | 0.190 |
| rs1813443 | 11 | 99516221 | C | 3.50 ×10−4 | −0.195 | 0.04021 | −0.155 | 0.605 | −0.048 | 0.403 | −0.097 | 1.37 ×10−4 | −0.148 |
| rs1447722 | 3 | 141037143 | C | 5.46 ×10−4 | 0.193 | 7.63 ×10−4 | 0.185 | 0.881 | −0.014 | 0.573 | −0.064 | 1.62 ×10−4 | 0.134 |
| rs1568885 | 7 | 13604056 | A | 5.93 ×10−4 | 0.255 | 0.0306 | 0.201 | 0.594 | 0.061 | 0.5 | 0.095 | 1.69 ×10−4 | 0.185 |
| rs12142623 | 1 | 185557029 | A | 4.19 ×10−4 | 0.248 | 7.68 ×10−3 | 0.267 | 0.484 | −0.087 | 0.398 | 0.126 | 2.04 ×10−4 | 0.185 |
| rs2378945 | 14 | 31370541 | A | 8.61 ×10−4 | −0.166 | 0.01313 | −0.171 | 0.489 | 0.061 | 0.905 | 0.013 | 6.88 ×10−4 | −0.115 |

Table 2 Caption: Association results for replicated SNPs in all cohorts and final meta-analysis

None of the SNPs previously reported to be associated with treatment outcome showed evidence for association in the Dutch stage 1 cohort ([Table 3](#T3)) [[16](#R16)–[20](#R20)].

### Table 3.

| SNP |  | Chr | MAF | Gene | PGC | β |
| --- | --- | --- | --- | --- | --- | --- |
| SNPs from Liu et al. |  |   |   |   |   |   |
| rs983332 |  | 1 | 0.18 | - | 0.603 | 0.031 |
| rs928655 |  | 1 | 0.23 | GBP6 | 0.051 | 0.109 |
| rs13393173 |  | 2 | 0.19 | LASS6 | 0.71 | −0.006 |
| rs437943 |  | 4 | 0.34 | - | 0.93 | 0.0006 |
| rs10945919 |  | 6 | 0.32 | AKO93144 | 0.44 | 0.03 |
| rs854555 |  | 7 | 0.33 | PON1 | 0.61 | 0.018 |
| rs854548 |  | 7 | 0.25 | PON1 | 0.61 | 0.0102 |
| rs854547 |  | 7 | 0.35 | PON1 | 0.52 | 0.0225 |
| rs868856 |  | 9 | 0.31 | MOBKL2B | 0.65 | −0.0218 |
| rs2814707 |  | 9 | 0.26 | MOBKL2B | 0.95 | −0.002 |
| rs3849942 |  | 9 | 0.26 | C9orf72 | 0.98 | −0.0069 |
| rs774359 |  | 9 | 0.27 | C9orf72 | 0.89 | −0.0159 |
| rs6138150 |  | 20 | 0.18 | - | 0.66 | 0.0329 |
| rs6028945 |  | 20 | 0.14 | - | 0.78 | −0.0635 |
| rs6071980 |  | 20 | 0.12 | - | 0.74 | −0.0026 |
| SNPs from Plant et al. |  |   |   |   |   |   |
| rs12081765 |  | 1 | 0.39 | - | 0.73 | −0.024 |
| rs4694890 |  | 4 | 0.49 | TEC | 0.4 | 0.042 |
| rs1532269 |  | 5 | 0.43 | PDZD2 | 0.92 | 0.051 |
| rs17301249 |  | 6 | 0.18 | EYA4 | 0.36 | −0.05 |
| rs1350948 |  | 11 | 0.14 | - | 0.38 | 0.0642 |
| rs7305646 |  | 12 | 0.50 | - | 0.39 | 0.05 |
| rs7962316 |  | 12 | 0.40 | BC118985 | 0.92 | −0.026 |
| SNPs from Krintel et al. |  |   |   |   |   |   |
| rs10520789 |  | 15 | 0.13 | NR2F2 | 0.47 | 0.071 |
| rs11870477 |  | 17 | 0.13 | MAP2K6 | 0.67 | 0.073 |
| rs16973982 |  | 15 | 0.14 | NR2F2 | 0.60 | 0.072 |
| rs8046065 |  | 16 | 0.10 | CREBBP | 0.12 | 0.146 |
| rs885814 |  | 1 | 0.32 | ALPL | 0.49 | 0.053 |
| rs869179 |  | 1 | 0.34 | ALPL | 0.75 | 0.016 |
| rs2722824 |  | 9 | 0.30 | TLR4 | 0.33 | −0.053 |
| rs885813 |  | 1 | 0.43 | ALPL | 0.30 | 0.049 |
| rs1875620 |  | 9 | 0.46 | C9orf47 | 0.26 | 0.051 |
| rs11525966 |  | 9 | 0.45 | C9orf47 | 0.19 | 0.051 |
| PTPRC |  |   |   |   |   |   |
| rs10919563 |  | 1 | 0.13 | PTPRC | 0.384 | −0.079 |

Table 3 Caption: Initial GWAS association results for loci previously reported to be associated with anti-TNF treatment response

### Explorative pathway analysis of stage 1

We explored the stage 1 dataset for potential functional relationship between genes that showed evidence of association at p<10^−3^ with treatment outcome using the Ingenuity Pathway Analysis (IPA). This resulted in the identification of 8 networks. The highest scoring network (p=10^−41^) included 26 genes identified in the GWAS analysis stage 1 and 9 additional interacting genes ([Figure 2](#F2)). Importantly, this network is predicted to be involved in metabolic disease and biological processes underlying inflammatory response and cell morphology and contains genes implicated in TNF signaling (*NFκB*) and antibody formation (*IgG*).

### Figure 2. Top gene network derived from Ingenuity Pathway Analysis.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/679c/4169706/36a7157542f6/nihms621909f2.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4169706_nihms621909f2.jpg)

Genes/gene products are represented graphically as nodes and the biological relationship between two nodes is represented as an edge (line). Grey colour of the node is indicating genes that were identified in the stage 1 GWAS (p<0.001) and white indicates the molecule was added from the Ingenuity Knowledge Base. Dashed lines indicate indirect interactions; solid lines indicate direct interactions. The style of the arrows indicate specific molecular relationships (A acts on B, A binds to B), dotted lines indicated an indirect interaction. All edges are supported by at least one reference from the literature or from canonical information stored in the Ingenuity Knowledge Base. Nodes are displayed using various shapes that represent the functional classes of the gene product( cytokines, enzyme, complex/group, transporter, transcription regulator, transmembrane receptor, ion channel, ligand-dependent nuclear receptor, kinase, growth factor, other).

## Discussion

In this report, we described the results of the largest GWA study of response to anti-TNF treatment in RA patients conducted to date.

Using a multistage study design, we identified eight genetic loci demonstrating suggestive evidence of association (improved p-values) with treatment outcome in our overall meta-analysis, with three markers (rs4411591, rs1813443 and rs1568885) showing directional consistency over all four cohorts studied. In the combined cohort, eight identified loci together explain 3.8% of the variance in the treatment response. While no single SNP reached a genome-wide level of significance (p<5×10^−8^), these variants represent excellent candidates for further investigation.

Of the eight markers with suggestive evidence of association, two map to an intergenic region in which the nearest gene is interesting in terms of its biological function. The SNPs, rs12142623 and rs4651370, are located ~400 kb downstream from the *phospholipase A2, group IVA (PLA2G4A)* gene. The protein encoded by *PLA2G4A* is a phospholipase enzyme involved in generation of eicosanoids, molecules with regulative function in inflammatory responses. TNFα is one of the first known stimuli for PLA2G4A activation, through action of both TNF receptor subtypes [[30](#R30)]. It might be possible that the identified SNPs influence long range regulatory elements.

In addition, three of the eight identified SNPs map within genes. The marker rs4411591 maps to the *Loc100130480*encoding an hypothetical protein, while rs2378945 is located in the *nucleotide binding protein-like (NUBPL)* gene. *NUBPL* encodes a protein required for the assembly of the respiratory chain NADH dehydrogenase (complex I). Finally, rs1813443 is located in the intronic region of *contactin 5 (CNTN5)*a member of the immunoglobulin superfamily which is thought to have a role in the formation of axon connections in the developing nervous system [[31](#R31)]. Little is known about the possible involvement of these genes in inflammatory disease and there are no apparent functional links to anti-TNF treatment outcome, yet. If association with anti-TNF response can be confirmed in additional replication studies, future functional studies are needed to prove the biological link with anti-TNF response.

We did not find evidence for association for the loci identified from previously published GWA studies on anti-TNF response [[18](#R18)–[20](#R20)] or for the *PTPRC* gene ([Table 3](#T3)) [[16](#R16), [17](#R17)] in the Dutch patients included in our stage 1 GWAS, suggesting that these genes do not play a major role in anti-TNF treatment outcome in our population. However, the power of our study to detect an association with PTPRC loci at p<0.0005 in a set size of 882 patients was 26%.

However, the top network constructed with Ingenuity Pathway analysis indicated involvement of the genes showing suggestive association in stage 1 GWAS in following processes: cell morphology, metabolic disease and inflammatory response. This is in line with the expected biological function of anti-TNF; TNFα is an important mediator of insulin resistance and it also impedes insulin-glucose-mediated uptake [[32](#R32)]. More importantly, studies showed a positive long-term effect of TNFα antagonists on insulin resistance, that was correlated with improvement in disease activity [[33](#R33), [34](#R34)]. The identified network also harbours two interesting interacting molecules: NFκB and IgG. NFκB is a transcription regulator that is preferentially activated by TNF and main downstream target of the TNF signaling pathway [[35](#R35)]. The finding that the genes identified through the GWAS are interacting with IgG is particularly interesting since response failure and side-effects of anti-TNF due to immunogenicity are not rare and it was found that anti-infliximab antibodies are exclusively of IgG isotype [[36](#R36)]. Both molecules might have a central role in determining the outcome of treatment with anti-TNF agents. Besides this network we could map the most prominent associations from stage 1 to the *VAV1* and *SPRED2* genes. *VAV1* has been found to protect T cells from Fas-mediated apoptosis in Jurkat leukemia T cells [[37](#R37)] and it has been confirmed that RA patients are showing differential sensitivity to apoptosis of peripheral blood lymphocytes induced by anti-TNF therapy [[38](#R38)]. *SPRED2* is a known RA risk loci [[39](#R39)]. This network might represent new leads and new additional candidates for future research.

Our study, which included 2703 RA patients treated with anti-TNF agents is the largest GWAS of treatment outcome to date. However, our sample size still remains modest compared to genetic studies of risk of RA [[40](#R40), [41](#R41)] and other complex traits [[42](#R42), [43](#R43)]. Also, there are important aspects that may affect our results. There is considerable disease heterogeneity in RA. In our dataset there is a difference in e.g. the number of women included in the studies but also type of anti-TNF treatment and co-medication use ([Table 1](#T1)). Furthermore, the REF collection used for replication in stage 2 consists of nine smaller cohorts from several populations. Hence, combining results across four different cohorts of RA patients that are rather diverse in subject ascertainment and assessment and previous treatment can lead to different effect estimates amongst studies and false negative results. In addition, the DAS28 score used as measure for treatment outcome in our study, is a composite score including four measures: swollen and tender joint counts, erythrocyte sedimentation rate and self-reported general health. This score is a powerful tool for measuring treatment response in a clinical setting. However, it is likely that this complex measure is influenced by genetic effects that are individually modest and would require large sample sizes to be detected.

In the present study, we have identified eight genetic loci that show suggestive evidence of influencing anti-TNF treatment response based on a multi-stage approach in a population of 2703 Caucasian RA patients. Our findings require further validation in independent cohorts and/or at a functional level.

## Supplementary Material

## Acknowledgements

MJHC is supported by a grant from the Netherlands Genomics Initiative (93511014). RMP is supported by grants from the NIH (R01-AR057108, R01-AR056768, U01-GM092691, R01-AR059648), and holds a Career Award for Medical Scientists from the Burroughs Wellcome Fund. Funding for this project was provided by the American College of Rheumatology Research and Education Foundation. NdV was sponsored by CTMM, the Center for Translational Molecular Medicine ([www.ctmm.nl](http://www.ctmm.nl)), and the Dutch Arthritis Foundation, project TRACER (grant 04I-202). We are in depth to Prof Lars Klareskog and Prof Lars Alfredsson for providing samples from the EIRA study, as well as Doc Johan Askling, Sara Wedrén and the Swedish Rheumatology Register, for their help with the follow-up data. We thank Alejandro Arias-Vasquez for help in the design of the analysis.

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. McInnes IB, Schett G. Cytokines in the pathogenesis of rheumatoid arthritis. Nat Rev Immunol. 2007;7:429–442. doi: 10.1038/nri2094.  [DOI](https://doi.org/10.1038/nri2094) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17525752/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Immunol&title=Cytokines%20in%20the%20pathogenesis%20of%20rheumatoid%20arthritis&author=IB%20McInnes&author=G%20Schett&volume=7&publication_year=2007&pages=429-442&pmid=17525752&doi=10.1038/nri2094&)

2. Tracey D, Klareskog L, Sasso EH, et al. Tumor necrosis factor antagonist mechanisms of action: a comprehensive review. Pharmacol Ther. 2008;117:244–279. doi: 10.1016/j.pharmthera.2007.10.001.  [DOI](https://doi.org/10.1016/j.pharmthera.2007.10.001) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18155297/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacol%20Ther&title=Tumor%20necrosis%20factor%20antagonist%20mechanisms%20of%20action:%20a%20comprehensive%20review&author=D%20Tracey&author=L%20Klareskog&author=EH%20Sasso&volume=117&publication_year=2008&pages=244-279&pmid=18155297&doi=10.1016/j.pharmthera.2007.10.001&)

3. Feldmann M, Maini SR. Role of cytokines in rheumatoid arthritis: an education in pathophysiology and therapeutics. Immunol Rev. 2008;223:7–19. doi: 10.1111/j.1600-065X.2008.00626.x.  [DOI](https://doi.org/10.1111/j.1600-065X.2008.00626.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18613827/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Immunol%20Rev&title=Role%20of%20cytokines%20in%20rheumatoid%20arthritis:%20an%20education%20in%20pathophysiology%20and%20therapeutics&author=M%20Feldmann&author=SR%20Maini&volume=223&publication_year=2008&pages=7-19&pmid=18613827&doi=10.1111/j.1600-065X.2008.00626.x&)

4. Maini R, St Clair EW, Breedveld F, et al. Infliximab (chimeric anti-tumour necrosis factor alpha monoclonal antibody) versus placebo in rheumatoid arthritis patients receiving concomitant methotrexate: a randomised phase III trial. ATTRACT Study Group. Lancet. 1999;354:1932–1939. doi: 10.1016/s0140-6736(99)05246-0.  [DOI](https://doi.org/10.1016/s0140-6736(99)05246-0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10622295/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Infliximab%20(chimeric%20anti-tumour%20necrosis%20factor%20alpha%20monoclonal%20antibody)%20versus%20placebo%20in%20rheumatoid%20arthritis%20patients%20receiving%20concomitant%20methotrexate:%20a%20randomised%20phase%20III%20trial.%20ATTRACT%20Study%20Group&author=R%20Maini&author=EW%20St%20Clair&author=F%20Breedveld&volume=354&publication_year=1999&pages=1932-1939&pmid=10622295&doi=10.1016/s0140-6736(99)05246-0&)

5. Keystone EC, Kavanaugh AF, Sharp JT, et al. Radiographic, clinical, and functional outcomes of treatment with adalimumab (a human anti-tumor necrosis factor monoclonal antibody) in patients with active rheumatoid arthritis receiving concomitant methotrexate therapy: a randomized, placebo-controlled, 52-week trial. Arthritis Rheum. 2004;50:1400–1411. doi: 10.1002/art.20217.  [DOI](https://doi.org/10.1002/art.20217) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15146409/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arthritis%20Rheum&title=Radiographic,%20clinical,%20and%20functional%20outcomes%20of%20treatment%20with%20adalimumab%20(a%20human%20anti-tumor%20necrosis%20factor%20monoclonal%20antibody)%20in%20patients%20with%20active%20rheumatoid%20arthritis%20receiving%20concomitant%20methotrexate%20therapy:%20a%20randomized,%20placebo-controlled,%2052-week%20trial&author=EC%20Keystone&author=AF%20Kavanaugh&author=JT%20Sharp&volume=50&publication_year=2004&pages=1400-1411&pmid=15146409&doi=10.1002/art.20217&)

6. Klareskog L, van der Heijde D, de Jager JP, et al. Therapeutic effect of the combination of etanercept and methotrexate compared with each treatment alone in patients with rheumatoid arthritis: double-blind randomised controlled trial. Lancet. 2004;363:675–681. doi: 10.1016/S0140-6736(04)15640-7.  [DOI](https://doi.org/10.1016/S0140-6736(04)15640-7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15001324/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Therapeutic%20effect%20of%20the%20combination%20of%20etanercept%20and%20methotrexate%20compared%20with%20each%20treatment%20alone%20in%20patients%20with%20rheumatoid%20arthritis:%20double-blind%20randomised%20controlled%20trial&author=L%20Klareskog&author=D%20van%20der%20Heijde&author=JP%20de%20Jager&volume=363&publication_year=2004&pages=675-681&pmid=15001324&doi=10.1016/S0140-6736(04)15640-7&)

7. Maini RN, Breedveld FC, Kalden JR, et al. Therapeutic efficacy of multiple intravenous infusions of anti-tumor necrosis factor alpha monoclonal antibody combined with low-dose weekly methotrexate in rheumatoid arthritis. Arthritis Rheum. 1998;41:1552–1563. doi: 10.1002/1529-0131(199809)41:9<1552::AID-ART5>3.0.CO;2-W.  [DOI](https://doi.org/10.1002/1529-0131(199809)41:9<1552::AID-ART5>3.0.CO;2-W) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9751087/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arthritis%20Rheum&title=Therapeutic%20efficacy%20of%20multiple%20intravenous%20infusions%20of%20anti-tumor%20necrosis%20factor%20alpha%20monoclonal%20antibody%20combined%20with%20low-dose%20weekly%20methotrexate%20in%20rheumatoid%20arthritis&author=RN%20Maini&author=FC%20Breedveld&author=JR%20Kalden&volume=41&publication_year=1998&pages=1552-1563&pmid=9751087&doi=10.1002/1529-0131(199809)41:9<1552::AID-ART5>3.0.CO;2-W&)

8. Tak PP. A personalized medicine approach to biologic treatment of rheumatoid arthritis: a preliminary treatment algorithm. Rheumatology (Oxford) 2012;51:600–609. doi: 10.1093/rheumatology/ker300.  [DOI](https://doi.org/10.1093/rheumatology/ker300) | [PMC free article](/articles/PMC3306168/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21890615/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Rheumatology%20(Oxford)&title=A%20personalized%20medicine%20approach%20to%20biologic%20treatment%20of%20rheumatoid%20arthritis:%20a%20preliminary%20treatment%20algorithm&author=PP%20Tak&volume=51&publication_year=2012&pages=600-609&pmid=21890615&doi=10.1093/rheumatology/ker300&)

9. Hyrich KL, Watson KD, Silman AJ, et al. Predictors of response to anti-TNF-alpha therapy among patients with rheumatoid arthritis: results from the British Society for Rheumatology Biologics Register. Rheumatology (Oxford) 2006;45:1558–1565. doi: 10.1093/rheumatology/kel149.  [DOI](https://doi.org/10.1093/rheumatology/kel149) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16705046/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Rheumatology%20(Oxford)&title=Predictors%20of%20response%20to%20anti-TNF-alpha%20therapy%20among%20patients%20with%20rheumatoid%20arthritis:%20results%20from%20the%20British%20Society%20for%20Rheumatology%20Biologics%20Register&author=KL%20Hyrich&author=KD%20Watson&author=AJ%20Silman&volume=45&publication_year=2006&pages=1558-1565&pmid=16705046&doi=10.1093/rheumatology/kel149&)

10. Atzeni F, Antivalle M, Pallavicini FB, et al. Predicting response to anti-TNF treatment in rheumatoid arthritis patients. Autoimmun Rev. 2009;8:431–437. doi: 10.1016/j.autrev.2009.01.005.  [DOI](https://doi.org/10.1016/j.autrev.2009.01.005) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19211043/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Autoimmun%20Rev&title=Predicting%20response%20to%20anti-TNF%20treatment%20in%20rheumatoid%20arthritis%20patients&author=F%20Atzeni&author=M%20Antivalle&author=FB%20Pallavicini&volume=8&publication_year=2009&pages=431-437&pmid=19211043&doi=10.1016/j.autrev.2009.01.005&)

11. Potter C, Hyrich KL, Tracey A, et al. Association of rheumatoid factor and anti-cyclic citrullinated peptide positivity, but not carriage of shared epitope or PTPN22 susceptibility variants, with anti-tumour necrosis factor response in rheumatoid arthritis. Ann Rheum Dis. 2009;68:69–74. doi: 10.1136/ard.2007.084715.  [DOI](https://doi.org/10.1136/ard.2007.084715) | [PMC free article](/articles/PMC2596303/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18375541/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Rheum%20Dis&title=Association%20of%20rheumatoid%20factor%20and%20anti-cyclic%20citrullinated%20peptide%20positivity,%20but%20not%20carriage%20of%20shared%20epitope%20or%20PTPN22%20susceptibility%20variants,%20with%20anti-tumour%20necrosis%20factor%20response%20in%20rheumatoid%20arthritis&author=C%20Potter&author=KL%20Hyrich&author=A%20Tracey&volume=68&publication_year=2009&pages=69-74&pmid=18375541&doi=10.1136/ard.2007.084715&)

12. Klaasen R, Thurlings RM, Wijbrandts CA, et al. The relationship between synovial lymphocyte aggregates and the clinical response to infliximab in rheumatoid arthritis: a prospective study. Arthritis Rheum. 2009;60:3217–3224. doi: 10.1002/art.24913.  [DOI](https://doi.org/10.1002/art.24913) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19877042/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arthritis%20Rheum&title=The%20relationship%20between%20synovial%20lymphocyte%20aggregates%20and%20the%20clinical%20response%20to%20infliximab%20in%20rheumatoid%20arthritis:%20a%20prospective%20study&author=R%20Klaasen&author=RM%20Thurlings&author=CA%20Wijbrandts&volume=60&publication_year=2009&pages=3217-3224&pmid=19877042&doi=10.1002/art.24913&)

13. Coenen MJ, Toonen EJ, Scheffer H, et al. Pharmacogenetics of anti-TNF treatment in patients with rheumatoid arthritis. Pharmacogenomics. 2007;8:761–773. doi: 10.2217/14622416.8.7.761.  [DOI](https://doi.org/10.2217/14622416.8.7.761) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17638513/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Pharmacogenetics%20of%20anti-TNF%20treatment%20in%20patients%20with%20rheumatoid%20arthritis&author=MJ%20Coenen&author=EJ%20Toonen&author=H%20Scheffer&volume=8&publication_year=2007&pages=761-773&pmid=17638513&doi=10.2217/14622416.8.7.761&)

14. Pavy S, Toonen EJ, Miceli-Richard C, et al. Tumour necrosis factor alpha-308G->A polymorphism is not associated with response to TNFalpha blockers in Caucasian patients with rheumatoid arthritis: systematic review and meta-analysis. Ann Rheum Dis. 2010;69:1022–1028. doi: 10.1136/ard.2009.117622.  [DOI](https://doi.org/10.1136/ard.2009.117622) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19966089/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Rheum%20Dis&title=Tumour%20necrosis%20factor%20alpha-308G->A%20polymorphism%20is%20not%20associated%20with%20response%20to%20TNFalpha%20blockers%20in%20Caucasian%20patients%20with%20rheumatoid%20arthritis:%20systematic%20review%20and%20meta-analysis&author=S%20Pavy&author=EJ%20Toonen&author=C%20Miceli-Richard&volume=69&publication_year=2010&pages=1022-1028&pmid=19966089&doi=10.1136/ard.2009.117622&)

15. Lee YH, Ji JD, Bae SC, et al. Associations between tumor necrosis factor-alpha (TNF-alpha)-308 and-238 G/A polymorphisms and shared epitope status and responsiveness to TNF-alpha blockers in rheumatoid arthritis: a metaanalysis update. J Rheumatol. 2010;37:740–746. doi: 10.3899/jrheum.090707.  [DOI](https://doi.org/10.3899/jrheum.090707) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20194454/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Rheumatol&title=Associations%20between%20tumor%20necrosis%20factor-alpha%20(TNF-alpha)-308%20and-238%20G/A%20polymorphisms%20and%20shared%20epitope%20status%20and%20responsiveness%20to%20TNF-alpha%20blockers%20in%20rheumatoid%20arthritis:%20a%20metaanalysis%20update&author=YH%20Lee&author=JD%20Ji&author=SC%20Bae&volume=37&publication_year=2010&pages=740-746&pmid=20194454&doi=10.3899/jrheum.090707&)

16. Cui J, Saevarsdottir S, Thomson B, et al. Rheumatoid arthritis risk allele PTPRC is also associated with response to anti-tumor necrosis factor alpha therapy. Arthritis Rheum. 2010;62:1849–1861. doi: 10.1002/art.27457.  [DOI](https://doi.org/10.1002/art.27457) | [PMC free article](/articles/PMC3652476/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20309874/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arthritis%20Rheum&title=Rheumatoid%20arthritis%20risk%20allele%20PTPRC%20is%20also%20associated%20with%20response%20to%20anti-tumor%20necrosis%20factor%20alpha%20therapy&author=J%20Cui&author=S%20Saevarsdottir&author=B%20Thomson&volume=62&publication_year=2010&pages=1849-1861&pmid=20309874&doi=10.1002/art.27457&)

17. Plant D, Prajapati R, Hyrich KL, et al. Replication of association of the PTPRC gene with response to anti-tumor necrosis factor therapy in a large UK cohort. Arthritis Rheum. 2012;64:665–670. doi: 10.1002/art.33381.  [DOI](https://doi.org/10.1002/art.33381) | [PMC free article](/articles/PMC3427899/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21952740/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arthritis%20Rheum&title=Replication%20of%20association%20of%20the%20PTPRC%20gene%20with%20response%20to%20anti-tumor%20necrosis%20factor%20therapy%20in%20a%20large%20UK%20cohort&author=D%20Plant&author=R%20Prajapati&author=KL%20Hyrich&volume=64&publication_year=2012&pages=665-670&pmid=21952740&doi=10.1002/art.33381&)

18. Liu C, Batliwalla F, Li W, et al. Genome-wide association scan identifies candidate polymorphisms associated with differential response to anti-TNF treatment in rheumatoid arthritis. Mol Med. 2008;14:575–581. doi: 10.2119/2008-00056.Liu.  [DOI](https://doi.org/10.2119/2008-00056.Liu) | [PMC free article](/articles/PMC2276142/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18615156/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Med&title=Genome-wide%20association%20scan%20identifies%20candidate%20polymorphisms%20associated%20with%20differential%20response%20to%20anti-TNF%20treatment%20in%20rheumatoid%20arthritis&author=C%20Liu&author=F%20Batliwalla&author=W%20Li&volume=14&publication_year=2008&pages=575-581&pmid=18615156&doi=10.2119/2008-00056.Liu&)

19. Plant D, Bowes J, Potter C, et al. Genome-wide association study of genetic predictors of anti-tumor necrosis factor treatment efficacy in rheumatoid arthritis identifies associations with polymorphisms at seven loci. Arthritis Rheum. 2011;63:645–653. doi: 10.1002/art.30130.  [DOI](https://doi.org/10.1002/art.30130) | [PMC free article](/articles/PMC3084508/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21061259/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arthritis%20Rheum&title=Genome-wide%20association%20study%20of%20genetic%20predictors%20of%20anti-tumor%20necrosis%20factor%20treatment%20efficacy%20in%20rheumatoid%20arthritis%20identifies%20associations%20with%20polymorphisms%20at%20seven%20loci&author=D%20Plant&author=J%20Bowes&author=C%20Potter&volume=63&publication_year=2011&pages=645-653&pmid=21061259&doi=10.1002/art.30130&)

20. Krintel SB, Palermo G, Johansen JS, et al. Investigation of single nucleotide polymorphisms and biological pathways associated with response to TNFalpha inhibitors in patients with rheumatoid arthritis. Pharmacogenetics and genomics. 2012;22:577–589. doi: 10.1097/FPC.0b013e3283544043.  [DOI](https://doi.org/10.1097/FPC.0b013e3283544043) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22569225/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics%20and%20genomics&title=Investigation%20of%20single%20nucleotide%20polymorphisms%20and%20biological%20pathways%20associated%20with%20response%20to%20TNFalpha%20inhibitors%20in%20patients%20with%20rheumatoid%20arthritis&author=SB%20Krintel&author=G%20Palermo&author=JS%20Johansen&volume=22&publication_year=2012&pages=577-589&pmid=22569225&doi=10.1097/FPC.0b013e3283544043&)

21. Selmi C, Lleo A, Zuin M, et al. Interferon alpha and its contribution to autoimmunity. Curr Opin Investig Drugs. 2006;7:451–456.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/16729722/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Opin%20Investig%20Drugs&title=Interferon%20alpha%20and%20its%20contribution%20to%20autoimmunity&author=C%20Selmi&author=A%20Lleo&author=M%20Zuin&volume=7&publication_year=2006&pages=451-456&pmid=16729722&)

22. Suarez-Gestal M, Perez-Pampin E, Calaza M, et al. Lack of replication of genetic predictors for the rheumatoid arthritis response to anti-TNF treatments: a prospective case-only study. Arthritis Res Ther. 2010;12:R72. doi: 10.1186/ar2990.  [DOI](https://doi.org/10.1186/ar2990) | [PMC free article](/articles/PMC2888228/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20423481/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arthritis%20Res%20Ther&title=Lack%20of%20replication%20of%20genetic%20predictors%20for%20the%20rheumatoid%20arthritis%20response%20to%20anti-TNF%20treatments:%20a%20prospective%20case-only%20study&author=M%20Suarez-Gestal&author=E%20Perez-Pampin&author=M%20Calaza&volume=12&publication_year=2010&pages=R72&pmid=20423481&doi=10.1186/ar2990&)

23. Kievit W, Fransen J, Adang EM, et al. Long-term effectiveness and safety of TNF-blocking agents in daily clinical practice: results from the Dutch Rheumatoid Arthritis Monitoring register. Rheumatology (Oxford) 2011;50:196–203. doi: 10.1093/rheumatology/keq325.  [DOI](https://doi.org/10.1093/rheumatology/keq325) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21078625/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Rheumatology%20(Oxford)&title=Long-term%20effectiveness%20and%20safety%20of%20TNF-blocking%20agents%20in%20daily%20clinical%20practice:%20results%20from%20the%20Dutch%20Rheumatoid%20Arthritis%20Monitoring%20register&author=W%20Kievit&author=J%20Fransen&author=EM%20Adang&volume=50&publication_year=2011&pages=196-203&pmid=21078625&doi=10.1093/rheumatology/keq325&)

24. Moreland LW, O'Dell JR, Paulus HE, et al. A randomized comparative effectiveness study of oral triple therapy versus etanercept plus methotrexate in early, aggressive rheumatoid arthritis. Arthritis Rheum. 2012 doi: 10.1002/art.34498.  [DOI](https://doi.org/10.1002/art.34498) | [PMC free article](/articles/PMC4036119/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22508468/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arthritis%20Rheum&title=A%20randomized%20comparative%20effectiveness%20study%20of%20oral%20triple%20therapy%20versus%20etanercept%20plus%20methotrexate%20in%20early,%20aggressive%20rheumatoid%20arthritis&author=LW%20Moreland&author=JR%20O'Dell&author=HE%20Paulus&publication_year=2012&pmid=22508468&doi=10.1002/art.34498&)

25. Bombardieri S, Ruiz AA, Fardellone P, et al. Effectiveness of adalimumab for rheumatoid arthritis in patients with a history of TNF-antagonist therapy in clinical practice. Rheumatology (Oxford) 2007;46:1191–1199. doi: 10.1093/rheumatology/kem091.  [DOI](https://doi.org/10.1093/rheumatology/kem091) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17504821/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Rheumatology%20(Oxford)&title=Effectiveness%20of%20adalimumab%20for%20rheumatoid%20arthritis%20in%20patients%20with%20a%20history%20of%20TNF-antagonist%20therapy%20in%20clinical%20practice&author=S%20Bombardieri&author=AA%20Ruiz&author=P%20Fardellone&volume=46&publication_year=2007&pages=1191-1199&pmid=17504821&doi=10.1093/rheumatology/kem091&)

26. Purcell S, Neale B, Todd-Brown K, et al. PLINK: a tool set for whole-genome association and population-based linkage analyses. Am J Hum Genet. 2007;81:559–575. doi: 10.1086/519795.  [DOI](https://doi.org/10.1086/519795) | [PMC free article](/articles/PMC1950838/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17701901/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=PLINK:%20a%20tool%20set%20for%20whole-genome%20association%20and%20population-based%20linkage%20analyses&author=S%20Purcell&author=B%20Neale&author=K%20Todd-Brown&volume=81&publication_year=2007&pages=559-575&pmid=17701901&doi=10.1086/519795&)

27. Price AL, Patterson NJ, Plenge RM, et al. Principal components analysis corrects for stratification in genome-wide association studies. Nat Genet. 2006;38:904–909. doi: 10.1038/ng1847.  [DOI](https://doi.org/10.1038/ng1847) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16862161/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Principal%20components%20analysis%20corrects%20for%20stratification%20in%20genome-wide%20association%20studies&author=AL%20Price&author=NJ%20Patterson&author=RM%20Plenge&volume=38&publication_year=2006&pages=904-909&pmid=16862161&doi=10.1038/ng1847&)

28. Li Y, Willer C, Sanna S, et al. Genotype imputation. Annu Rev Genomics Hum Genet. 2009;10:387–406. doi: 10.1146/annurev.genom.9.081307.164242.  [DOI](https://doi.org/10.1146/annurev.genom.9.081307.164242) | [PMC free article](/articles/PMC2925172/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19715440/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Annu%20Rev%20Genomics%20Hum%20Genet&title=Genotype%20imputation&author=Y%20Li&author=C%20Willer&author=S%20Sanna&volume=10&publication_year=2009&pages=387-406&pmid=19715440&doi=10.1146/annurev.genom.9.081307.164242&)

29. Chen WM, Abecasis GR. Family-based association tests for genomewide association scans. Am J Hum Genet. 2007;81:913–926. doi: 10.1086/521580.  [DOI](https://doi.org/10.1086/521580) | [PMC free article](/articles/PMC2265659/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17924335/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=Family-based%20association%20tests%20for%20genomewide%20association%20scans&author=WM%20Chen&author=GR%20Abecasis&volume=81&publication_year=2007&pages=913-926&pmid=17924335&doi=10.1086/521580&)

30. Jupp OJ, Vandenabeele P, MacEwan DJ. Distinct regulation of cytosolic phospholipase A2 phosphorylation, translocation, proteolysis and activation by tumour necrosis factor-receptor subtypes. Biochem J. 2003;374:453–461. doi: 10.1042/BJ20030705.  [DOI](https://doi.org/10.1042/BJ20030705) | [PMC free article](/articles/PMC1223607/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12786601/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem%20J&title=Distinct%20regulation%20of%20cytosolic%20phospholipase%20A2%20phosphorylation,%20translocation,%20proteolysis%20and%20activation%20by%20tumour%20necrosis%20factor-receptor%20subtypes&author=OJ%20Jupp&author=P%20Vandenabeele&author=DJ%20MacEwan&volume=374&publication_year=2003&pages=453-461&pmid=12786601&doi=10.1042/BJ20030705&)

31. Ogawa J, Kaneko H, Masuda T, et al. Novel neural adhesion molecules in the Contactin/F3 subgroup of the immunoglobulin superfamily: isolation and characterization of cDNAs from rat brain. Neurosci Lett. 1996;218:173–176. doi: 10.1016/s0304-3940(96)13156-6.  [DOI](https://doi.org/10.1016/s0304-3940(96)13156-6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8945756/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neurosci%20Lett&title=Novel%20neural%20adhesion%20molecules%20in%20the%20Contactin/F3%20subgroup%20of%20the%20immunoglobulin%20superfamily:%20isolation%20and%20characterization%20of%20cDNAs%20from%20rat%20brain&author=J%20Ogawa&author=H%20Kaneko&author=T%20Masuda&volume=218&publication_year=1996&pages=173-176&pmid=8945756&doi=10.1016/s0304-3940(96)13156-6&)

32. Peraldi P, Hotamisligil GS, Buurman WA, et al. Tumor necrosis factor (TNF)-alpha inhibits insulin signaling through stimulation of the p55 TNF receptor and activation of sphingomyelinase. J Biol Chem. 1996;271:13018–13022. doi: 10.1074/jbc.271.22.13018.  [DOI](https://doi.org/10.1074/jbc.271.22.13018) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8662983/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=Tumor%20necrosis%20factor%20(TNF)-alpha%20inhibits%20insulin%20signaling%20through%20stimulation%20of%20the%20p55%20TNF%20receptor%20and%20activation%20of%20sphingomyelinase&author=P%20Peraldi&author=GS%20Hotamisligil&author=WA%20Buurman&volume=271&publication_year=1996&pages=13018-13022&pmid=8662983&doi=10.1074/jbc.271.22.13018&)

33. Huvers FC, Popa C, Netea MG, et al. Improved insulin sensitivity by anti-TNFalpha antibody treatment in patients with rheumatic diseases. Ann Rheum Dis. 2007;66:558–559. doi: 10.1136/ard.2006.062323.  [DOI](https://doi.org/10.1136/ard.2006.062323) | [PMC free article](/articles/PMC1856029/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17360784/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Rheum%20Dis&title=Improved%20insulin%20sensitivity%20by%20anti-TNFalpha%20antibody%20treatment%20in%20patients%20with%20rheumatic%20diseases&author=FC%20Huvers&author=C%20Popa&author=MG%20Netea&volume=66&publication_year=2007&pages=558-559&pmid=17360784&doi=10.1136/ard.2006.062323&)

34. Tam LS, Tomlinson B, Chu TT, et al. Impact of TNF inhibition on insulin resistance and lipids levels in patients with rheumatoid arthritis. Clin Rheumatol. 2007;26:1495–1498. doi: 10.1007/s10067-007-0539-8.  [DOI](https://doi.org/10.1007/s10067-007-0539-8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17237906/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Rheumatol&title=Impact%20of%20TNF%20inhibition%20on%20insulin%20resistance%20and%20lipids%20levels%20in%20patients%20with%20rheumatoid%20arthritis&author=LS%20Tam&author=B%20Tomlinson&author=TT%20Chu&volume=26&publication_year=2007&pages=1495-1498&pmid=17237906&doi=10.1007/s10067-007-0539-8&)

35. Nagar M, Jacob-Hirsch J, Vernitsky H, et al. TNF activates a NF-kappaB-regulated cellular program in human CD45RA- regulatory T cells that modulates their suppressive function. J Immunol. 2010;184:3570–3581. doi: 10.4049/jimmunol.0902070.  [DOI](https://doi.org/10.4049/jimmunol.0902070) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20181891/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Immunol&title=TNF%20activates%20a%20NF-kappaB-regulated%20cellular%20program%20in%20human%20CD45RA-%20regulatory%20T%20cells%20that%20modulates%20their%20suppressive%20function&author=M%20Nagar&author=J%20Jacob-Hirsch&author=H%20Vernitsky&volume=184&publication_year=2010&pages=3570-3581&pmid=20181891&doi=10.4049/jimmunol.0902070&)

36. Svenson M, Geborek P, Saxne T, et al. Monitoring patients treated with anti-TNF-alpha biopharmaceuticals: assessing serum infliximab and anti-infliximab antibodies. Rheumatology (Oxford) 2007;46:1828–1834. doi: 10.1093/rheumatology/kem261.  [DOI](https://doi.org/10.1093/rheumatology/kem261) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18032541/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Rheumatology%20(Oxford)&title=Monitoring%20patients%20treated%20with%20anti-TNF-alpha%20biopharmaceuticals:%20assessing%20serum%20infliximab%20and%20anti-infliximab%20antibodies&author=M%20Svenson&author=P%20Geborek&author=T%20Saxne&volume=46&publication_year=2007&pages=1828-1834&pmid=18032541&doi=10.1093/rheumatology/kem261&)

37. Yin J, Wan YJ, Li SY, et al. The distinct role of guanine nucleotide exchange factor Vav1 in Bcl-2 transcription and apoptosis inhibition in Jurkat leukemia T cells. Acta Pharmacol Sin. 2011;32:99–107. doi: 10.1038/aps.2010.185.  [DOI](https://doi.org/10.1038/aps.2010.185) | [PMC free article](/articles/PMC4003318/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21151158/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Acta%20Pharmacol%20Sin&title=The%20distinct%20role%20of%20guanine%20nucleotide%20exchange%20factor%20Vav1%20in%20Bcl-2%20transcription%20and%20apoptosis%20inhibition%20in%20Jurkat%20leukemia%20T%20cells&author=J%20Yin&author=YJ%20Wan&author=SY%20Li&volume=32&publication_year=2011&pages=99-107&pmid=21151158&doi=10.1038/aps.2010.185&)

38. Coury F, Ferraro-Peyret C, Le Cam S, et al. Peripheral blood lymphocytes from patients with rheumatoid arthritis are differentially sensitive to apoptosis induced by anti-tumour necrosis factor-alpha therapy. Clin Exp Rheumatol. 2008;26:234–239.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/18565243/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Exp%20Rheumatol&title=Peripheral%20blood%20lymphocytes%20from%20patients%20with%20rheumatoid%20arthritis%20are%20differentially%20sensitive%20to%20apoptosis%20induced%20by%20anti-tumour%20necrosis%20factor-alpha%20therapy&author=F%20Coury&author=C%20Ferraro-Peyret&author=S%20Le%20Cam&volume=26&publication_year=2008&pages=234-239&pmid=18565243&)

39. Baum L, Ng HK, Wong KS, et al. Associations of apolipoprotein E exon 4 and lipoprotein lipase S447X polymorphisms with acute ischemic stroke and myocardial infarction. Clin Chem Lab Med. 2006;44:274–281. doi: 10.1515/CCLM.2006.047.  [DOI](https://doi.org/10.1515/CCLM.2006.047) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16519597/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Chem%20Lab%20Med&title=Associations%20of%20apolipoprotein%20E%20exon%204%20and%20lipoprotein%20lipase%20S447X%20polymorphisms%20with%20acute%20ischemic%20stroke%20and%20myocardial%20infarction&author=L%20Baum&author=HK%20Ng&author=KS%20Wong&volume=44&publication_year=2006&pages=274-281&pmid=16519597&doi=10.1515/CCLM.2006.047&)

40. Okada Y, Terao C, Ikari K, et al. Meta-analysis identifies nine new loci associated with rheumatoid arthritis in the Japanese population. Nat Genet. 2012;44:511–516. doi: 10.1038/ng.2231.  [DOI](https://doi.org/10.1038/ng.2231) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22446963/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Meta-analysis%20identifies%20nine%20new%20loci%20associated%20with%20rheumatoid%20arthritis%20in%20the%20Japanese%20population&author=Y%20Okada&author=C%20Terao&author=K%20Ikari&volume=44&publication_year=2012&pages=511-516&pmid=22446963&doi=10.1038/ng.2231&)

41. Stahl EA, Raychaudhuri S, Remmers EF, et al. Genome-wide association study meta-analysis identifies seven new rheumatoid arthritis risk loci. Nat Genet. 2010;42:508–514. doi: 10.1038/ng.582.  [DOI](https://doi.org/10.1038/ng.582) | [PMC free article](/articles/PMC4243840/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20453842/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Genome-wide%20association%20study%20meta-analysis%20identifies%20seven%20new%20rheumatoid%20arthritis%20risk%20loci&author=EA%20Stahl&author=S%20Raychaudhuri&author=EF%20Remmers&volume=42&publication_year=2010&pages=508-514&pmid=20453842&doi=10.1038/ng.582&)

42. Lango Allen H, Estrada K, Lettre G, et al. Hundreds of variants clustered in genomic loci and biological pathways affect human height. Nature. 2010;467:832–838. doi: 10.1038/nature09410.  [DOI](https://doi.org/10.1038/nature09410) | [PMC free article](/articles/PMC2955183/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20881960/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Hundreds%20of%20variants%20clustered%20in%20genomic%20loci%20and%20biological%20pathways%20affect%20human%20height&author=H%20Lango%20Allen&author=K%20Estrada&author=G%20Lettre&volume=467&publication_year=2010&pages=832-838&pmid=20881960&doi=10.1038/nature09410&)

43. Chasman DI, Pare G, Zee RY, et al. Genetic loci associated with plasma concentration of low-density lipoprotein cholesterol, high-density lipoprotein cholesterol, triglycerides, apolipoprotein A1, and Apolipoprotein B among 6382 white women in genome-wide analysis with replication. Circ Cardiovasc Genet. 2008;1:21–30. doi: 10.1161/CIRCGENETICS.108.773168.  [DOI](https://doi.org/10.1161/CIRCGENETICS.108.773168) | [PMC free article](/articles/PMC2630707/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19802338/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circ%20Cardiovasc%20Genet&title=Genetic%20loci%20associated%20with%20plasma%20concentration%20of%20low-density%20lipoprotein%20cholesterol,%20high-density%20lipoprotein%20cholesterol,%20triglycerides,%20apolipoprotein%20A1,%20and%20Apolipoprotein%20B%20among%206382%20white%20women%20in%20genome-wide%20analysis%20with%20replication&author=DI%20Chasman&author=G%20Pare&author=RY%20Zee&volume=1&publication_year=2008&pages=21-30&pmid=19802338&doi=10.1161/CIRCGENETICS.108.773168&)
