# SNP discovery, expression and cis-regulatory variation in the UGT2B genes

## Metadata
**Authors:** C Sun, C Southard, D Huo, R D Hernandez, D B Witonsky, O I Olopade, A Di Rienzo
**Journal:** The pharmacogenomics journal
**Date:** 2011 Mar 1
**DOI:** [10.1038/tpj.2011.2](https://doi.org/10.1038/tpj.2011.2)
**PMID:** 21358749
**PMCID:** PMC4643740
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4643740/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC4643740/pdf/nihms736011.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC4643740/pdf/nihms736011.pdf)

## Abstract

UGT2B enzymes metabolize multiple endogenous and exogenous molecules, including steroid hormones and clinical drugs. However, little is known about the inter-individual variation in gene expression and its determinants. We re-sequenced candidate regulatory regions and the partial coding regions (41.1 kb) of UGT2B genes and identified 332 genetic variants. We measured gene expression in normal breast and liver samples and observed different patterns. The expression levels varied greatly across individuals in both tissues and were significantly correlated with each other in liver. Genotyping of tagging single-nucleotide polymorphisms (SNPs) in the same samples and association tests between genotype and transcript levels identified 62 variants that were associated with at least one UGT2B mRNA levels in either tissue. Most of these cis-regulatory SNPs were not shared between tissues, suggesting that this gene family is regulated in a tissue-specific manner. Our results provide insight into studying the role of UGT2B variation in hormone-dependent cancers and drug response.

Keywords: UGT2B, genetic variation, gene expression, cis-regulatory

## Introduction

One of the most important goals of genetics is to identify genetic variation that can explain the observed variation in disease susceptibility and in other phenotypes. However, elucidating the relationship between genetic variation and complex phenotypes at the organismal level is not straightforward. Most polymorphisms that influence complex phenotypes first alter intermediate and molecular phenotypes, including, but not limited to, gene (mRNA, ncRNA, microRNA and so on) expression,[1](#R1) mRNA secondary structure and stability,[2](#R2) protein sequence, RNA splicing,[3](#R3) microRNA interaction[4](#R4) and codon usage,[5](#R5) which in turn contribute to diseases and complex phenotypes. Among these molecular phenotypes, variation in expression levels has been proposed as one of the most important types in the human genome.[1](#R1) To aid in the dissection of the genetic basis of human traits and diseases, numerous association studies of transcript levels have been performed in multiple tissues on a genome-wide scale.[1](#R1)^, ^[6](#R6)^, ^[7](#R7)^, ^[8](#R8)^, ^[9](#R9)^, ^[10](#R10)

Glucuronidation is an important clearance pathway for many endogenous and exogenous molecules, including steroid hormones, bile acid, bilirubin, carcinogens and clinical drugs.[11](#R11)^, ^[12](#R12) UDP-glucuronosyltransferases (UGTs) transfer glucuronic acid from UDP-glucuronic acid to substrates, thus making them more water soluble than their parent compound and more easily excreted through the biliary and renal systems.[11](#R11) In humans, there are two major UGT subfamilies, UGT1A and UGT2B.[13](#R13)^, ^[14](#R14)^, ^[15](#R15) UGT2B includes 7 active members (UGT2B4, UGT2B7, UGT2B10, UGT2B11, UGT2B15, UGT2B17 and UGT2B28) located on chromosome [4](#R4)^,^[13](#R13)^, ^[14](#R14)^, ^[15](#R15) which are mainly expressed in liver, breast, prostate, colon and kidney.[16](#R16)^, ^[17](#R17)^, ^[18](#R18) The UGT2B17 and UGT2B28 genes are not expressed in some individuals due to whole-gene polymorphic deletions (copy number variation, CNV) that are common in human populations.[19](#R19) Because UGT2B enzymes are essential in the metabolism of steroid hormones which in turn have a central role in multiple cancers, it has been proposed that variation in the UGT2B genes is involved in breast[20](#R20) and prostate[21](#R21)^, ^[22](#R22)^, ^[23](#R23)^, ^[24](#R24)^, ^[25](#R25) cancer risk.[26](#R26)^, ^[27](#R27) Therefore, a systematic survey of genetic variants that affect enzyme activity and gene expression will provide a firm basis for elucidating the role of these genes in hormone-dependent cancers.

Although recent genome-wide association studies of transcript levels have identified multiple potential regulatory elements for this family,[6](#R6)^, ^[7](#R7)^, ^[10](#R10) these studies have several shortcomings. First, most studies used lymphoblastoid cell lines (LCLs)[6](#R6)^, ^[7](#R7)^, ^[10](#R10) that express only a few UGT2B genes. Moreover, the regulation of this gene family is likely to differ among tissues as proposed by a recent study.[28](#R28) Therefore, results from LCLs cannot easily be extrapolated to other tissues. Second, the current expression arrays cannot detect modest to small changes in expression level; this problem is further aggravated by the cross-hybridization of probes to the highly similar UGT2B genes. Third, only single-nucleotide polymorphisms (SNPs) from the HapMap project were tested, which has low coverage in regions containing duplicated genes like the UGT2B. Indeed, the average SNP density in the UGT2B cluster is 1 per ~1300 bp in the HapMap data compared with the genome-wide average of 1 per ~750 bp.[29](#R29)

In light of the limitations of previous studies, we performed a more detailed survey of sequence and expression variation of the UGT2B gene cluster. Candidate regulatory regions in the UGT2B cluster were re-sequenced in ethnically diverse individuals and tagging SNPs were selected accordingly. Transcript levels for each UGT2B gene were measured by quantitative real-time PCR (qPCR) in normal breast and liver tissue samples and tagging SNPs were genotyped in the same samples. The subsequent association analysis identified multiple potential cis-regulatory variants, which represent strong candidate susceptibility variants for hormone-dependent cancers and for inter-individual variation in drug response.

## Materials and methods

### Bioinformatics analyses

We first used bioinformatics tools to identify candidate regulatory regions. We used two major approaches: one is the prediction of transcription factor binding sites and the other is the identification of conserved sequences across distantly related species. The binding matrix for transcription factors enriched in the liver or prostate, including HNF1 homeobox A (HNF1A), hepatocyte nuclear factor 4, α (HNF4A), nuclear receptor subfamily 1, group I, member 2 (NR1I2), coxsackie virus and adenovirus receptor (CXADR), POU class 2 homeobox 1 (POU2F1), peroxisome proliferator-activated receptor α (PPARA), retinoid X receptor, γ (RXRG), CCAAT/enhancer binding protein α (CEBPA), forkhead box A2 (FOXA2), nuclear receptor subfamily 2, group F, member 2 (NR2F2) and for nuclear hormone receptors, such as nuclear receptor subfamily 3, group C, member 1 (glucocorticoid receptor, NR3C1) and estrogen receptor 1 (ESR1), were obtained from the TRANSFAC database ([http://www.gene-regulation.com/](http://www.gene-regulation.com/)). Binding site prediction on the human reference sequence was performed by Cluster Buster ([http://zlab.bu.edu/cluster-buster/](http://zlab.bu.edu/cluster-buster/)).[30](#R30) Regions that contain binding site clusters predicted with high probability and that are conserved between humans and at least two other species, as shown on the ECR genome browser ([http://ecrbrowser.dcode.org/](http://ecrbrowser.dcode.org/)), were included in the re-sequencing survey ([Supplementary Figure S1](#SD1)). Moreover, the promoter regions and partial coding regions were also included. In total, 41.1 kb were chosen for re-sequencing.

### Re-sequencing and tagging

SNP selection Fifty-six unrelated HapMap samples (24 YRI, 22 CEU and 10 ASN) were chosen for re-sequencing. cDNA was synthesized using the Super Transcript III kit (Invitrogen, Carlsbad, CA, USA) and utilized for the amplification of UGT2B17 coding regions and UGT2B7 exons 2–6 (that is, the UGT2B7_v4 variant, which is expressed in LCLs, unlike the complete UGT2B7_v1 variant[31](#R31)). All PCR was performed using the primers in [Supplementary Table S1](#SD1). After exonuclease I and Shrimp Alkaline Phosphatase (United States Biochemicals, Cleveland, OH, USA) treatment, sequencing was performed by using PCR and internal primers in [Supplementary Table S1](#SD1) and BigDye Terminator v3.1 (Applied Biosystems, Foster City, CA, USA). Polymorphisms were scored by PolyPhred[32](#R32) and confirmed through visual inspection. Visual genotype plots were drawn by using the Genome Variation Server ([http://gvs.gs.washington.edu/GVS/](http://gvs.gs.washington.edu/GVS/)). The UGT2B15 promoter and exon 1 re-sequencing data in our recent study,[28](#R28) HapMap genotyping data in UGT2B cluster and UGT2B4 re-sequencing data from Environmental Genome Project ([http://egp.gs.washington.edu/welcome.html](http://egp.gs.washington.edu/welcome.html)) were also included in the selection of tagging SNPs. Tagging SNPs were chosen by using ldSelect[33](#R33) with r^2^ greater than or equal to 0.8 and minor allele frequency (MAF) greater than or equal to 0.05. All re-sequencing data will be available in the PharmGKB database ([http://www.pharmgkb.org](http://www.pharmgkb.org)).

### Tissue collection, RNA and DNA isolation

Eighty-one normal breast (4 European American (CA), 8 African American (AA) and 69 of unknown ethnicity; 78 female, 2 male and 1 unknown gender) and 31 normal liver (3 CA, 1 AA and 27 unknown; 14 female, 16 male and 1 unknown) tissue samples were obtained from the University of Chicago Tissue Core Facility. RNA and DNA were isolated by RNeasy Lipid Tissue Mini Kit and QIAamp DNA Mini Kit (Qiagen, Valencia, CA, USA), respectively.

### Genotyping of breast and liver samples

CNVs in UGT2B17 and UGT2B28 were genotyped in breast and liver samples by a previously published qPCR assay[28](#R28) and Taqman assay Hs00852540_s1 (Applied Biosystems) according to manufacturer’s recommendation. Partial UGT2B17 CNV data were reported in our recent study.[28](#R28)

The tagging SNPs were genotyped in all breast and liver samples by iPLEX SNP genotyping assay (Sequenom, San Diego, CA, USA) according to the manufacturer’s protocol. In brief, the PCR and extension primers for each SNP were designed by online tools provided by the manufacturer ([https://mysequenom.com/default.aspx](https://mysequenom.com/default.aspx)) and listed in [Supplementary Table S2](#SD2). Multiplex PCR was performed with HotStarTaq DNA polymerase (Qiagen). After Shrimp Alkaline Phosphatase (Sequenom) treatment, single base extension was performed. The extension product was dispensed onto the SpectroCHIP bioarray (Sequenom) and the mass was determined and distinguished by Matrix-assisted laser desorption/ionization time-of-flight mass spectrometry (Sequenom). The genotype calls were analyzed by Typer 4.0 software (Sequenom) and confirmed through visual inspection. Fifteen HapMap samples with known genotypes at all sites were included as positive controls.

### Quantitative real-time PCR

cDNA for breast and liver samples was synthesized by using the High Capacity Reverse Transcription Kit (Applied Biosystems). Transcript levels for the UGT2B genes in breast and liver tissues were measured by qPCR with power SYBR green (Applied Biosystems) and primers listed in [Supplementary information](#SD1). All qPCR assays had high efficiency (>95%) and all PCR products were sequenced to confirm the primer specificity. The qPCR was performed in triplicate for each gene. β-actin and 18S mRNA level were quantified in breast and liver tissues, respectively, as reference genes due to their relatively conserved expression in these two tissues (data not shown). UGT2B11 and UGT2B28 transcript levels were not quantified because primers specific for these genes could not be designed. All qPCR and Taqman readings in this study were performed on a StepOne Plus Real time PCR System (Applied Biosystems). The details for qPCR are described according to minimum information for publication of qPCR experiment guidelines[34](#R34) in [Supplementary information](#SD1).

### Statistical analyses

First, UGT2B transcript levels in breast or liver were normalized by log_2_ transformation. For individuals with null expression of a specific gene, we assigned a value of one half the minimum value observed for that gene to avoid taking log transformation on zero. Then, we used linear regression models to assess the relationship between genotypes and gene expression. As the true mode of inheritance was unknown, we used a two degree of freedom linear model to jointly test for differences between the three genotype categories. To gain statistical power, we also fitted an additive linear model to the SNP genotype and gene expression data. To identify SNPs that influence UGT2B17 transcript levels in addition to its CNV, we excluded individuals with homozygous deletions and adjusted for CNV genotype (1 copy or 2 copies) by multiple linear regressions. We also used a permutation method to get multiple test-corrected gene cluster-wide P-values. Similarly, we used linear regression models to examine whether UGT2B gene expression was correlated with sex and age. For genes whose expression was correlated with age or sex, we further adjusted for age and/or sex when we assessed the association between genotype and mRNA levels. Similar adjustments for ancestry could not be made due to incomplete information about the ethnicity of our samples. Finally, we asked whether expression levels of different genes in liver or breast are correlated with each other by using Spearman’s rank correlation coefficients. Analysis of gene cluster-wide genotype expression correlations was conducted using plink v1.07 ([http://pngu.mgh.harvard.edu/purcell/plink/](http://pngu.mgh.harvard.edu/purcell/plink/)).[35](#R35) Other statistical analysis and data management were conducted using Stata 11.0 (StataCorp LP, College Station, TX, USA).

## Results

### Re-sequencing survey of the UGT2B gene cluster

Our re-sequencing survey identified 332 genetic variants (see [Supplementary Figure S2](#SD1) and [Supplementary Table S3](#SD1) for details), most (91.8%) of which were not genotyped by the HapMap project. Among all variants, 314 (97.2%) were SNPs while the others were short insertion/deletion polymorphisms. Approximately half of these variants (162 out of 332) showed an MAF >5%. Twenty-seven of them were located in the coding regions and 10 were non-synonymous. We included the HapMap SNP genotype and other available re-sequencing data for the same individuals and used them together with our re-sequencing data (1316 genetic variants total) to select tagging SNPs for the UGT2B cluster. We identified 285 tag bins with MAF greater than or equal to 0.05 for the entire cluster and at least one tagging SNP for each bin was chosen for iPLEX assay design.

### SNP and CNV genotyping in breast and liver DNA samples

Two hundred thirty-eight individual tagging SNPs (83.5%) were clustered in seven genotyping assays; the remaining 47 tagging SNPs were rejected by the assay design program and could not be surveyed. Out of the 238 clustered SNPs, 189 SNPs (79.4%) were successfully genotyped (see [Supplementary Figure S3](#SD1) for detail). Given the high sequence similarity across the UGT2B genes, it is not surprising that many SNPs did not yield a high quality assay. For the successfully genotyped SNPs, the comparison between the iPLEX genotype calls and the known genotypes for 15 controls yielded an error rate <1.9%. We also compared the MAF and linkage disequilibrium pattern between the iPLEX genotypes and our re-sequencing data. As shown in [Supplementary Figure S4](#SD1), the MAF in the iPLEX data are highly correlated with those in our re-sequencing data (r^2^=0.717, P<10^−52^). The linkage disequilibrium pattern is also similar (result not shown). No deviation from Hardy–Weinberg equilibrium was observed (P>0.05) in most SNPs (85.2% for breast and 95.2% for liver samples). It is worth noting that most (64.3%) of the SNPs with Hardy–Weinberg equilibrium departures in the breast samples showed the same pattern in the HapMap and the re-sequencing data when different populations were pooled into a single sample (not shown), suggesting that the Hardy–Weinberg equilibrium departures are due to high differentiation of these SNPs across populations coupled with the mixed ethnicity of the breast samples. When these SNPs were removed from analysis, only 5.3% SNPs in breast showed deviation from Hardy–Weinberg equilibrium.

The genotype frequencies for UGT2B17 and UGT2B28 CNVs in the breast and liver samples are in good agreement with those for the HapMap CEU and YRI population samples ([Supplementary Table S4](#SD1)).[19](#R19)

### UGT2B gene expression in breast and liver

UGT2B transcript levels in breast and liver samples are displayed in [Figures 1a and b](#F1), respectively. In general, UGT2B15 (quantification cycle (Cq)[34](#R34) value, mean±s.d., 32.0±2.6) and UGT2B17 (Cq value, mean±s.d., 33.5±3.0) are expressed at intermediate levels in breast while UGT2B4 (Cq value, mean±s.d., 35.3±3.4) is expressed at lower levels. However, the comparison of expression levels across genes should be interpreted with caution; this is because, even though all our PCR assays had high efficiency (>95%), a subtle difference in efficiency across assays may have a non-trivial effect on the estimate of mRNA levels due to the exponential relationship between number of mRNA molecules and number of cycles. In liver, all five genes show high expression levels (Cq value, mean±s.d., for UGT2B4, 29.3±2.0; for UGT2B10, 29.1±2.4; for UGT2B15, 29.5±1.8; for UGT2B17, 33.1±2.1), especially UGT2B7 (Cq value, mean±s.d., 25.3±2.0). These results are consistent with previous reports about the relative expression levels of UGT2B genes.[18](#R18) UGT2B731 and UGT2B10 transcripts were not detected in breast samples. UGT2B17 transcripts could not be detected in some individuals, which is mostly due to the known common polymorphic deletions of this gene. All genes showed a high degree of inter-individual variation in transcript levels in both breast and liver, with ~133–1400-fold variation between the highest and the lowest non-zero value. This observation also confirmed recent findings in liver.[36](#R36)

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/fb00/4643740/0797ace531ee/nihms736011f1.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4643740_nihms736011f1.jpg)

Boxplots of UGT2B gene expression in breast (a) and liver (b) samples. All genes are normalized to β-actin (a) or 18S (b) and log2 transformed.

A recent study[36](#R36) has suggested a correlation between expression levels of different UGT2B genes in liver. We analyzed published genome-wide expression data in liver samples (not shown)[9](#R9) and observed a similarly strong correlation between UGT2B genes. To test whether this is the case also in our breast and liver samples, we performed correlation analysis on transcript levels for all pairs of genes. As shown in [Table 1a](#T1), a significantly positive correlation between UGT2B15 and UGT2B17 was observed (r=0.469, P<10^−5^) in breast. We also found a negative correlation between UGT2B4 and UGT2B15 (r=−0.308, P=0.005) in breast. In liver, all expressed genes were strongly and positively correlated with each another (see [Table 1b](#T1)). The observation in liver may reflect a shared regulatory mechanism for all UGT2B genes subject to variation in genetic background or environmental exposures.[36](#R36)^, ^[37](#R37)

### Table 1.

|   | UGT2B4 |  | UGT2B15 | UGT2B17 |
| --- | --- | --- | --- | --- |
| (a) |  |  |  |  |
| UGT2B15 | −0.308 (0.005) |  |   |   |
| UGT2B17 | −0.123 (0.27) |  | 0.469 (<10−5) |   |
|   | UGT2B4 | UGT2B7 | UGT2B10 | UGT2B15 |
| (b) |  |  |  |  |
| UGT2B7 | 0.770 (<10−5) |   |   |   |
| UGT2B10 | 0.799 (<10−5) | 0.795 (<10−5) |   |   |
| UGT2B15 | 0.853 (<10−5) | 0.642 (10−5) | 0.646 (10−5) |   |
| UGT2B17 | 0.806 (<10−5) | 0.598 (0.0004) | 0.553 (0.001) | 0.736 (<10−5) |

Table 1 Caption: Spearman’s correlation coefficients (P-values) among UGT2B gene expression in breast (a) and liver (b) samples

Recent studies have proposed that estrogens can upregulate UGT2B15 in breast cancer cell lines[38](#R38) while androgens, especially dihydrotestosterone, can downregulate UGT2B15 and UGT2B17 in prostate cancer cell lines.[39](#R39)^, ^[40](#R40)^, ^[41](#R41)^, ^[42](#R42) These findings raise the possibility that females express some UGT2B members, especially UGT2B15, at higher levels compared with males. In liver, we found that UGT2B10 (P=0.0496; [Figure 2a](#F2)) and UGT2B15 (P=0.039; [Figure 2b](#F2)) had ~3.3-fold higher expression on average in males than females whereas the other UGT2B genes did not show such variation (P>0.13 in other genes, data not shown). This observation is in contrast with the above prediction and it may be due to differences in the regulation of UGT2B genes between liver and breast or prostate, as recently suggested.[28](#R28) It should also be noted that all previous analyses were performed in cancer cell lines[38](#R38)^, ^[39](#R39)^, ^[40](#R40)^, ^[41](#R41)^, ^[42](#R42) rather than in primary cells, as in this study; therefore, this apparent discrepancy could be due, at least in part, to differences in regulation between transformed and primary cells.

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/fb00/4643740/f4d9d5097174/nihms736011f2.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4643740_nihms736011f2.jpg)

Gender difference in UGT2B10 (a, P=0.0496) and UGT2B15 (b, P=0.039) expression in liver. All genes are normalized to 18S and log2 transformed.

To investigate whether UGT2B expression levels are constant during human life, we examined the correlation between age of the breast and liver donors and UGT2B expression levels. UGT2B15 expression levels in liver are significantly correlated with age, with expression increasing in older individuals (r=0.416, P=0.020; [Figure 3](#F3)). After adjusting for gender in a multiple linear model, we still observed a significant correlation between age and UGT2B15 in liver (P=0.049). This finding might be explained by the known changes in steroid hormone levels during human life.[43](#R43) No significant correlation was observed for the other genes in liver (P>0.23 in all cases, data not shown) and for all genes in breast (P>0.08 in all genes, data not shown).

### Figure 3.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/fb00/4643740/d465f93dd90c/nihms736011f3.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4643740_nihms736011f3.jpg)

Correlation between age and UGT2B15 expression in liver (r=0.416, P=0.020). x axis indicates age while y axis UGT2B15 expression (normalized to 18S and log2 transformed). Each point denotes one individual.

### Correlation between SNP and CNV genotype and expression levels

The common deletion of the UGT2B17 gene was previously found to be associated with its expression levels in prostate[25](#R25) and LCL.[44](#R44)^, ^[45](#R45) In LCLs, a significant association was also detected between the UGT2B17 CNV and the expression levels of the UGT2B7, UGT2B10 and UGT2B11 genes.[45](#R45) We tested whether the UGT2B17 CNV is also associated with expression levels in breast and liver; in addition, we tested for an association between CNV and all UGT2B gene expression in both tissues. In breast, UGT2B17 CNV is correlated only with its own expression (r^2^=0.403, P<0.0001; [Supplementary Figure S5](#SD1)) but not that of other UGT2B members (all P>0.22, data not shown). In liver, besides its own expression (r^2^=0.337, P=0.003; [Supplementary Figure S6a](#SD1)), the UGT2B17 CNV was also significantly associated with UGT2B4 (r^2^=0.234, P=0.024; [Supplementary Figure S6b](#SD1)).

We further performed linear regression analysis between tagging SNP genotype and UGT2B expression levels in breast ([Figure 4](#F4)) and liver ([Figure 5](#F5); see [Supplementary Table S5](#SD1) for detail). By this approach, [62](#R62) SNPs were identified to be significantly (P<0.05) correlated with the transcript levels of at least one UGT2B gene in at least one tissue. No SNP passed multiple test correction in liver, which may be due to the low power afforded by the relatively small sample size. Except for rs4860985 in UGT2B15 and rs17671289 in UGT2B4, none of the SNPs was significantly associated with the same gene in both tissues (see [Supplementary Table S5](#SD1)), thus supporting the proposal that the regulation of this gene family is tissue specific.[28](#R28) It was also interesting to note that multiple SNPs are associated with the expression of more than one gene. For example, rs2736483 is associated with UGT2B4 (P=0.000007429), UGT2B7 (P=0.000001492), UGT2B10 (P=0.0001018), UGT2B15 (P=0.0000715) and UGT2B17 (P=0.00352) expression in liver and rs17147073 is associated with UGT2B4 (P=0.04566), UGT2B15 (P=0.03499) and UGT2B17 (P=0.003779) in breast (see [Supplementary Table S5](#SD1)).

### Figure 4.

![Figure 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/fb00/4643740/2dde540946f2/nihms736011f4.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4643740_nihms736011f4.jpg)

Association tests between tagging single-nucleotide polymorphism (SNP) genotype and UGT2B4 (red), UGT2B15 (blue) and UGT2B17 (green) expression in breast samples based on additive linear models. The x axis indicates the SNP position in chromosome 4 (build 36) while the y axis denotes P-value (–log10 transformed) for each SNP with P-value <0.05. The bars across the bottom indicate the location of the UGT2B genes in the order, from left to right: UGT2B17, UGT2B15, UGT2B10, UGT2B7, UGT2B11, UGT2B28 and UGT2B4.

### Figure 5.

![Figure 5](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/fb00/4643740/a0e1631ba1b2/nihms736011f5.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4643740_nihms736011f5.jpg)

Association tests between tagging single-nucleotide polymorphism (SNP) genotype and UGT2B4 (red), UGT2B7 (purple), UGT2B10 (brown) UGT2B15 (blue) and UGT2B17 (green) expression in liver samples based on additive linear models. The x axis indicates the SNP position in chromosome 4 (build 36) while the y axis denotes P-value (–log10 transformed) for each SNP with P-value <0.05. The bars across the bottom of each plot indicate the location of the UGT2B genes in the order, from left to right: UGT2B17, UGT2B15, UGT2B10, UGT2B7, UGT2B11, UGT2B28 and UGT2B4.

The most significant correlations in breast and liver are shown in [Table 2](#T2). As all these SNPs are located far (>380 kb) from UGT2B genes, we hypothesized that they lie within enhancer regions and might alter the binding affinity of transcription factors regulating UGT2B gene expression. To investigate this possibility, we retrieved all the SNPs within the tag bins associated with the expression of at least one UGT2B gene and used the Match program in the TRANSFAC database ([http://www.gene-regulation.com/cgi-bin/pub/programs/match/bin/match.cgi](http://www.gene-regulation.com/cgi-bin/pub/programs/match/bin/match.cgi)) to search for predicted transcription factor binding sites in the regions near the SNPs. By this approach, multiple canonical binding sites for transcription factors were identified ([Supplementary Table S6](#SD1)). As shown in the table, some transcription factors, such as forkhead box D3 (FoxD3), forkhead box A2 (FoxA2 or HNF3B), HNF4A, forkhead box J2 (FoxJ2), paired box 4 (PAX4) and POU class 2 homeobox 1 (POU2F1 or OCT1), are common among those with predicted binding sites near SNPs associated with gene expression levels. This is consistent with the idea that these transcription factors are involved in the regulation of this gene family.[16](#R16)

### Table 2.

| Tissue and gene | SNP positiona | rs ID | Genotype | No. | Mean log2-transformed transcript levels (s.d.) | P-value | R2 |
| --- | --- | --- | --- | --- | --- | --- | --- |
| Breast |  |  |  |  |  |  |  |
| UGT2B4 | 69500987 | rs2168047 | CC | 29 | −7.2 (1.81) | 0.0012 | 0.160 |
|   |   |   | CT | 33 | −10.16 (3.88) |   |   |
|   |   |   | TT | 18 | −10.05 (3.81) |   |   |
| UGT2B15 | 70384595 | rs4557343 | GG | 29 | −8.34 (2.16) | 0.0018 | 0.150 |
|   |   |   | GT | 41 | −6.82 (2.46) |   |   |
|   |   |   | TT | 11 | −5.38 (2.86) |   |   |
| UGT2B17 | 70038552 | rs6822259 | CC | 2 | −13.55 (7.11) | 0.0006 | 0.190b |
|   |   |   | TC | 14 | −12.48 (3.48) |   |   |
|   |   |   | TT | 58 | −9.47 (2.41) |   |   |
|   | CNV and rs6822259 |  | 0 copy | 7 | −18.44 (0.38) | 8.3E-13 | 0.510 |
|   |   |   | 1/2 copies+CC/TC | 16 | −12.61 (3.74) |   |   |
|   |   |   | 1/2 copies+TT | 58 | −9.47 (2.41) |   |   |
| Liver |  |  |  |  |  |  |  |
| UGT2B4 | 70342103 | rs12642263 | AA | 18 | −1.85 (1.96) | 0.013 | 0.269 |
|   |   |   | AG | 12 | 0.23 (1.46) |   |   |
|   |   |   | GG | 1 | −2.31 |   |   |
| UGT2B7 | 70482723 | rs1835823 | GG | 21 | 0.12 (2.01) | 0.047 | 0.196 |
|   |   |   | GC | 9 | 1.36 (2.11) |   |   |
|   |   |   | CC | 1 | 4.90 |   |   |
|   | 70320917 | rs10007427 | CC | 14 | −0.59 (2.07) | 0.019 | 0.263 |
|   |   |   | CT | 14 | 1.62 (1.79) |   |   |
|   |   |   | TT | 1 | −0.29 |   |   |
| UGT2B10 | 70482723 | rs1835823 | GG | 21 | −0.04 (2.37) | 0.06 | 0.182 |
|   |   |   | GC | 9 | 1.19 (1.95) |   |   |
|   |   |   | CC | 1 | 5.12 |   |   |
|   | 70320917 | rs10007427 | CC | 14 | −0.66 (2.32) | 0.056 | 0.199 |
|   |   |   | CT | 14 | 1.52 (2.23) |   |   |
|   |   |   | TT | 1 | 0.36 |   |   |
| UGT2B15 | 69500987 | rs2168047 | CC | 19 | −0.49 (1.87) | 0.0074 | 0.296 |
|   |   |   | CT | 10 | −2.98 (2.14) |   |   |
|   |   |   | TT | 2 | −2.95 (2.00) |   |   |
|   | 70342103 | rs12642263 | AA | 18 | −2.50 (2.18) | 0.0027 | 0.345 |
|   |   |   | AG | 12 | 0.20 (1.35) |   |   |
|   |   |   | GG | 1 | −2.30 |   |   |
| UGT2B17 | 70360475 | rs17147338 | CC | 24 | −1.78 (3.20) | 0.039 | 0.214b |
|   |   |   | CT | 5 | −6.01 (3.13) |   |   |
|   |   |   | TT | 1 | −3.39 |   |   |
|   | CNV and rs17147338 |  | 0 copy or 1 copy+CT | 4 | −7.57 (1.35) | 2.5E-05 | 0.530 |
|   |   |   | 2 copies+CT/TT | 16 | −3.55 (2.46) |   |   |
|   |   |   | 2 copies+CC | 11 | 0.25 (2.88) |   |   |

Table 2 Caption: The most significant association between SNP genotype and UGT2B transcript levels in breast and liver based on two degree of freedom linear models

## Discussion

To identify cis-regulatory variants for the human UGT2B genes, we re-sequenced candidate regulatory elements in ethnically diverse populations. These data, together with publicly available sequence variation information in the same individuals, were used to select tagging SNPs. Then, we quantified UGT2B transcript levels and genotyped the tagging SNPs in normal breast and liver samples to test for associations between genotypes and expression levels. As a result, we found 62 SNPs that are significantly correlated with the expression of at least one UGT2B gene, and 17 SNPs are significantly correlated with the expression of more than one gene. This is the first systematic investigation of sequence variation, breast and liver expression spectrum in unaffected individuals and cis-regulatory variation for the UGT2B family. Our results will undoubtedly provide a firm foundation for studies aimed at investigating the role of UGT2B variation in hormone-dependent diseases and inter-individual variation in drug response.

Our re-sequencing survey identified 162 common (MAF greater than or equal to 5%) SNPs and among them, only 22 (13.6%) were included in HapMap project; this is a substantially lower rate compared with the 25–35% coverage of common SNPs in the genome.[29](#R29) The low SNP coverage in this genomic region in the HapMap is probably due to the difficulties of sequencing and genotyping regions containing highly similar duplicated genes like the UGT2B gene cluster. Therefore, our re-sequencing study significantly extends the information provided by the HapMap project for this particular region of the genome.

Recent genome-wide association studies uncovered many disease susceptibility SNPs located in non-coding regions distant from genes (for example, >100 kb). Some of the disease SNPs has been validated from the functional standpoint. For example, SNPs rs7903146, rs4939827 and rs6983267 are associated with type 2 diabetes,[46](#R46) colorectal cancer[47](#R47) and multiple cancers (as reviewed by reference[48](#R48)), respectively. Further functional in vitro and in vivo assays showed that these SNPs are located within enhancers and that they influence the activity of these enhancers thereby affecting target gene expression.[48](#R48)^, ^[49](#R49)^, ^[50](#R50)^, ^[51](#R51) Many of the SNPs that we find to be associated with variation in UGT2B gene expression levels are far from the corresponding genes; as for SNPs found in genome-wide association studies, further work is necessary to elucidate the mechanism underlying their effect on mRNA levels.

The 1000 Genomes Project ([http://1000genomes.org](http://1000genomes.org)) currently underway seeks to discover essentially all genetic variation (>1% frequency) in each of the major continental populations using next-generation sequencing technologies.[52](#R52) To understand whether such technologies are capable of capturing genetic variation in duplicated gene regions, we analyzed depth of coverage statistics around the UGT2B gene cluster generated during pilot 1 of the 1000 Genomes Project. Specifically, for each gene in the UGT2B gene cluster, we quantified the average depth of coverage per individual for the entire gene region, and the distribution of coverage across chromosome 4 using 1000 randomly chosen regions with matched length ([Supplementary Figure S7](#SD1)). We found that UGT2B4, UGT2B7, UGT2B10 and UGT2B11 have coverage statistics that are consistent with coverage patterns from the rest of chromosome 4. However, UGT2B15, UGT2B17 and UGT2B28 each had much lower coverage than expected for the rest of chromosome 4 (in the 0.1, 0.5 and 2.5% tail, respectively). While the low coverage in UGT2B17 and UGT2B28 may be due to the common polymorphic deletions of these genes, no common CNVs are known for the UGT2B15 gene. Such low coverage statistics suggest that some genetic variants may be missed in the 1000 Genomes Project data for these UGT2B genes. More generally, these results suggest that some duplicated genes can indeed be re-sequenced successfully using next-generation technologies. However, other duplicated gene regions may receive little benefit from next-generation technologies, and will require more robust technologies (such as those used in this study) to discover genetic variation within them.

In addition to SNPs, CNVs may also influence gene expression, as shown in previous studies[25](#R25)^, ^[44](#R44)^, ^[45](#R45) and by the findings reported here. Stranger et al.[45](#R45) proposed that CNV of the UGT2B17 gene influences the expression of other UGT2B members, including UGT2B7, UGT2B10 and UGT2B11. However, we could not detect transcripts for any of these genes in LCL by qPCR (data not shown), suggesting that the previous observation was a false positive possibly due to the cross-hybridization of probes to different UGT2B genes in the array. Indeed, UGT2B17 (data not shown) and UGT2B7_v4,31 a splicing variant of UGT2B7, are highly expressed in LCLs and the sequence similarity between them is high. Probe cross-hybridization may also explain partly the failure to replicate the UGT2B cis-regulatory variants detected in a genome-wide association study in liver[9](#R9) in both our study and a recent one focusing on UGT2B7.[53](#R53) Furthermore, it highlights the value of using qPCR, or other high resolution approaches, to investigate the expression of duplicated genes with high sequence similarity.

There are still three shortcomings in our study. First, our liver sample size is relatively small, which resulted in low power to detect associations between SNP genotypes and expression levels. Association studies on a larger sample size or in vitro functional assays, for example, reporter gene assays, will be necessary to identify regulatory variants in liver. Second, to identify the potential regulatory element most efficiently, we used comparative genomics coupled with computational predictions of transcription factor binding sites. Although this approach has been widely used and proven to be useful,[54](#R54)^, ^[55](#R55)^, ^[56](#R56)^, ^[57](#R57)^, ^[58](#R58)^, ^[59](#R59) recent studies have demonstrated that not all functional regions are conserved across species;[60](#R60) therefore, we may have missed some important regulatory elements. Third, although we obtained genotype data for 189 tagging SNPs in our breast and liver tissues, these SNPs only accounted for 66.3% of the total tag bins; this implies that some regulatory variants may have been missed. Considering these caveats, it is unlikely that our survey has characterized the full repertoire of regulatory variants for this gene family. Nonetheless, our results represent the most extensive and systematic survey of sequence and expression variation in this important family of genes. As a consequence, we identified a large number of strong candidate regulatory variants for the UGT2B genes, especially in breast samples, thus significantly advancing our understanding of the genetic bases of inter-individual variation in UGT2B gene expression.

Breast cancer is the most common cancer in women and cumulative estrogen exposure over a lifetime is a likely risk factor.[61](#R61)^, ^[62](#R62)^, ^[63](#R63) Indeed, despite the decrease of serum estrogen levels in postmenopausal compared with premenopausal women, the estrogen concentration in breast remains relatively constant during aging.[64](#R64) Moreover, estrogen levels in breast tumor tissue are much higher than in normal breast tissue.[65](#R65)^, ^[66](#R66) These observations suggest that variation in steroid hormone metabolism, especially in the target tissues like breast, may have a role in the susceptibility to breast cancer.[64](#R64)^, ^[67](#R67) UGT2B15 and UGT2B17 are good candidates for breast cancer risk due to their high expression in breast (current study).[18](#R18) In addition, UGT2B15 and UGT2B17 have significant activity against estrogen metabolites, especially 2- and 4-hydroxylated ones.[68](#R68) Although UGT2B7 is not expressed in the breast,[31](#R31) considering its high activity against estrogens,[68](#R68) it might also contribute to breast cancer risk by altering the systemic estrogen levels. Previous studies aimed at detecting UGT2B variants associated with breast cancer risk mainly focused on the coding regions,[20](#R20)^, ^[69](#R69) especially on amino-acid changes that alter enzyme activity or specificity, such as D85Y in UGT2B15 and H268Y in UGT2B7. Little attention has been paid to SNPs that alter UGT2B gene expression. Therefore, the SNPs identified in this study as potential cis-regulatory variants affecting the expression of the UGT2B15 and UGT2B17 genes in breast and of the UGT2B7 gene in liver represent strong new candidate variants for breast cancer risk. In this sense, the results of our study provide valuable information to investigate the role of the UGT2B genes in hormone-dependent diseases and in drug response.

## Supplementary Material

## Acknowledgments

We thank Ms Maria Tretiakova and Ying Sun for technical assistance. We also thank three anonymous reviewers for their helpful comments. This research was supported by the University of Chicago Breast SPORE NCI Grant CA125183 and by grant U01 GM61393.

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Rockman MV, Kruglyak L. Genetics of global gene expression. Nat Rev Genet. 2006;7:862–872. doi: 10.1038/nrg1964.  [DOI](https://doi.org/10.1038/nrg1964) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17047685/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Genet&title=Genetics%20of%20global%20gene%20expression&author=MV%20Rockman&author=L%20Kruglyak&volume=7&publication_year=2006&pages=862-872&pmid=17047685&doi=10.1038/nrg1964&)

2. Nackley AG, Shabalina SA, Tchivileva IE, Satterfield K, Korchynskyi O, Makarov SS, et al. Human catechol-O-methyltransferase haplotypes modulate protein expression by altering mRNA secondary structure. Science. 2006;314:1930–1933. doi: 10.1126/science.1131262.  [DOI](https://doi.org/10.1126/science.1131262) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17185601/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=Human%20catechol-O-methyltransferase%20haplotypes%20modulate%20protein%20expression%20by%20altering%20mRNA%20secondary%20structure&author=AG%20Nackley&author=SA%20Shabalina&author=IE%20Tchivileva&author=K%20Satterfield&author=O%20Korchynskyi&volume=314&publication_year=2006&pages=1930-1933&pmid=17185601&doi=10.1126/science.1131262&)

3. Wang GS, Cooper TA. Splicing in disease: disruption of the splicing code and the decoding machinery. Nat Rev Genet. 2007;8:749–761. doi: 10.1038/nrg2164.  [DOI](https://doi.org/10.1038/nrg2164) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17726481/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Genet&title=Splicing%20in%20disease:%20disruption%20of%20the%20splicing%20code%20and%20the%20decoding%20machinery&author=GS%20Wang&author=TA%20Cooper&volume=8&publication_year=2007&pages=749-761&pmid=17726481&doi=10.1038/nrg2164&)

4. Georges M, Coppieters W, Charlier C. Polymorphic miRNA-mediated gene regulation: contribution to phenotypic variation and disease. Curr Opin Genet Dev. 2007;17:166–176. doi: 10.1016/j.gde.2007.04.005.  [DOI](https://doi.org/10.1016/j.gde.2007.04.005) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17467975/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Opin%20Genet%20Dev&title=Polymorphic%20miRNA-mediated%20gene%20regulation:%20contribution%20to%20phenotypic%20variation%20and%20disease&author=M%20Georges&author=W%20Coppieters&author=C%20Charlier&volume=17&publication_year=2007&pages=166-176&pmid=17467975&doi=10.1016/j.gde.2007.04.005&)

5. Kimchi-Sarfaty C, Oh JM, Kim IW, Sauna ZE, Calcagno AM, Ambudkar SV, et al. A ‘silent’ polymorphism in the MDR1 gene changes substrate specificity. Science. 2007;315:525–528. doi: 10.1126/science.1135308.  [DOI](https://doi.org/10.1126/science.1135308) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17185560/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=A%20%E2%80%98silent%E2%80%99%20polymorphism%20in%20the%20MDR1%20gene%20changes%20substrate%20specificity&author=C%20Kimchi-Sarfaty&author=JM%20Oh&author=IW%20Kim&author=ZE%20Sauna&author=AM%20Calcagno&volume=315&publication_year=2007&pages=525-528&pmid=17185560&doi=10.1126/science.1135308&)

6. Cheung VG, Conlin LK, Weber TM, Arcaro M, Jen KY, Morley M, et al. Natural variation in human gene expression assessed in lymphoblastoid cells. Nat Genet. 2003;33:422–425. doi: 10.1038/ng1094.  [DOI](https://doi.org/10.1038/ng1094) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12567189/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Natural%20variation%20in%20human%20gene%20expression%20assessed%20in%20lymphoblastoid%20cells&author=VG%20Cheung&author=LK%20Conlin&author=TM%20Weber&author=M%20Arcaro&author=KY%20Jen&volume=33&publication_year=2003&pages=422-425&pmid=12567189&doi=10.1038/ng1094&)

7. Duan S, Huang RS, Zhang W, Bleibel WK, Roe CA, Clark TA, et al. Genetic architecture of transcript-level variation in humans. Am J Hum Genet. 2008;82:1101–1113. doi: 10.1016/j.ajhg.2008.03.006.  [DOI](https://doi.org/10.1016/j.ajhg.2008.03.006) | [PMC free article](/articles/PMC2651622/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18439551/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=Genetic%20architecture%20of%20transcript-level%20variation%20in%20humans&author=S%20Duan&author=RS%20Huang&author=W%20Zhang&author=WK%20Bleibel&author=CA%20Roe&volume=82&publication_year=2008&pages=1101-1113&pmid=18439551&doi=10.1016/j.ajhg.2008.03.006&)

8. Kristensen VN, Edvardsen H, Tsalenko A, Nordgard SH, Sorlie T, Sharan R, et al. Genetic variation in putative regulatory loci controlling gene expression in breast cancer. Proc Natl Acad Sci USA. 2006;103:7735–7740. doi: 10.1073/pnas.0601893103.  [DOI](https://doi.org/10.1073/pnas.0601893103) | [PMC free article](/articles/PMC1458617/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16684880/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20USA&title=Genetic%20variation%20in%20putative%20regulatory%20loci%20controlling%20gene%20expression%20in%20breast%20cancer&author=VN%20Kristensen&author=H%20Edvardsen&author=A%20Tsalenko&author=SH%20Nordgard&author=T%20Sorlie&volume=103&publication_year=2006&pages=7735-7740&pmid=16684880&doi=10.1073/pnas.0601893103&)

9. Schadt EE, Molony C, Chudin E, Hao K, Yang X, Lum PY, et al. Mapping the genetic architecture of gene expression in human liver. PLoS Biol. 2008;6:e107. doi: 10.1371/journal.pbio.0060107.  [DOI](https://doi.org/10.1371/journal.pbio.0060107) | [PMC free article](/articles/PMC2365981/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18462017/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Biol&title=Mapping%20the%20genetic%20architecture%20of%20gene%20expression%20in%20human%20liver&author=EE%20Schadt&author=C%20Molony&author=E%20Chudin&author=K%20Hao&author=X%20Yang&volume=6&publication_year=2008&pages=e107&pmid=18462017&doi=10.1371/journal.pbio.0060107&)

10. Stranger BE, Nica AC, Forrest MS, Dimas A, Bird CP, Beazley C, et al. Population genomics of human gene expression. Nat Genet. 2007;39:1217–1224. doi: 10.1038/ng2142.  [DOI](https://doi.org/10.1038/ng2142) | [PMC free article](/articles/PMC2683249/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17873874/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Population%20genomics%20of%20human%20gene%20expression&author=BE%20Stranger&author=AC%20Nica&author=MS%20Forrest&author=A%20Dimas&author=CP%20Bird&volume=39&publication_year=2007&pages=1217-1224&pmid=17873874&doi=10.1038/ng2142&)

11. Tukey RH, Strassburg CP. Human UDP-glucuronosyltransferases: metabolism, expression, and disease. Annu Rev Pharmacol Toxicol. 2000;40:581–616. doi: 10.1146/annurev.pharmtox.40.1.581.  [DOI](https://doi.org/10.1146/annurev.pharmtox.40.1.581) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10836148/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Annu%20Rev%20Pharmacol%20Toxicol&title=Human%20UDP-glucuronosyltransferases:%20metabolism,%20expression,%20and%20disease&author=RH%20Tukey&author=CP%20Strassburg&volume=40&publication_year=2000&pages=581-616&pmid=10836148&doi=10.1146/annurev.pharmtox.40.1.581&)

12. Belanger A, Pelletier G, Labrie F, Barbier O, Chouinard S. Inactivation of androgens by UDP-glucuronosyltransferase enzymes in humans. Trends Endocrinol Metab. 2003;14:473–479. doi: 10.1016/j.tem.2003.10.005.  [DOI](https://doi.org/10.1016/j.tem.2003.10.005) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14643063/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Trends%20Endocrinol%20Metab&title=Inactivation%20of%20androgens%20by%20UDP-glucuronosyltransferase%20enzymes%20in%20humans&author=A%20Belanger&author=G%20Pelletier&author=F%20Labrie&author=O%20Barbier&author=S%20Chouinard&volume=14&publication_year=2003&pages=473-479&pmid=14643063&doi=10.1016/j.tem.2003.10.005&)

13. Guillemette C. Pharmacogenomics of human UDP-glucuronosyltransferase enzymes. Pharmacogenomics J. 2003;3:136–158. doi: 10.1038/sj.tpj.6500171.  [DOI](https://doi.org/10.1038/sj.tpj.6500171) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12815363/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=Pharmacogenomics%20of%20human%20UDP-glucuronosyltransferase%20enzymes&author=C%20Guillemette&volume=3&publication_year=2003&pages=136-158&pmid=12815363&doi=10.1038/sj.tpj.6500171&)

14. King CD, Rios GR, Green MD, Tephly TR. UDP-glucuronosyltransferases. Curr Drug Metab. 2000;1:143–161. doi: 10.2174/1389200003339171.  [DOI](https://doi.org/10.2174/1389200003339171) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11465080/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Drug%20Metab&title=UDP-glucuronosyltransferases&author=CD%20King&author=GR%20Rios&author=MD%20Green&author=TR%20Tephly&volume=1&publication_year=2000&pages=143-161&pmid=11465080&doi=10.2174/1389200003339171&)

15. Mackenzie PI, Bock KW, Burchell B, Guillemette C, Ikushiro S, Iyanagi T, et al. Nomenclature update for the mammalian UDP glycosyltransferase (UGT) gene superfamily. Pharmacogenet Genomics. 2005;15:677–685. doi: 10.1097/01.fpc.0000173483.13689.56.  [DOI](https://doi.org/10.1097/01.fpc.0000173483.13689.56) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16141793/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Nomenclature%20update%20for%20the%20mammalian%20UDP%20glycosyltransferase%20(UGT)%20gene%20superfamily&author=PI%20Mackenzie&author=KW%20Bock&author=B%20Burchell&author=C%20Guillemette&author=S%20Ikushiro&volume=15&publication_year=2005&pages=677-685&pmid=16141793&doi=10.1097/01.fpc.0000173483.13689.56&)

16. Gardner-Stephen DA, Mackenzie PI. Liver-enriched transcription factors and their role in regulating UDP glucuronosyltransferase gene expression. Curr Drug Metab. 2008;9:439–452. doi: 10.2174/138920008784746409.  [DOI](https://doi.org/10.2174/138920008784746409) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18537579/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Drug%20Metab&title=Liver-enriched%20transcription%20factors%20and%20their%20role%20in%20regulating%20UDP%20glucuronosyltransferase%20gene%20expression&author=DA%20Gardner-Stephen&author=PI%20Mackenzie&volume=9&publication_year=2008&pages=439-452&pmid=18537579&doi=10.2174/138920008784746409&)

17. Nakamura A, Nakajima M, Yamanaka H, Fujiwara R, Yokoi T. Expression of UGT1A and UGT2B mRNA in human normal tissues and various cell lines. Drug Metab Dispos. 2008;36:1461–1464. doi: 10.1124/dmd.108.021428.  [DOI](https://doi.org/10.1124/dmd.108.021428) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18480185/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Expression%20of%20UGT1A%20and%20UGT2B%20mRNA%20in%20human%20normal%20tissues%20and%20various%20cell%20lines&author=A%20Nakamura&author=M%20Nakajima&author=H%20Yamanaka&author=R%20Fujiwara&author=T%20Yokoi&volume=36&publication_year=2008&pages=1461-1464&pmid=18480185&doi=10.1124/dmd.108.021428&)

18. Ohno S, Nakajin S. Determination of mRNA expression of human UDP-glucuronosyltransferases and application for localization in various human tissues by real-time reverse transcriptase-polymerase chain reaction. Drug Metab Dispos. 2009;37:32–40. doi: 10.1124/dmd.108.023598.  [DOI](https://doi.org/10.1124/dmd.108.023598) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18838504/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Determination%20of%20mRNA%20expression%20of%20human%20UDP-glucuronosyltransferases%20and%20application%20for%20localization%20in%20various%20human%20tissues%20by%20real-time%20reverse%20transcriptase-polymerase%20chain%20reaction&author=S%20Ohno&author=S%20Nakajin&volume=37&publication_year=2009&pages=32-40&pmid=18838504&doi=10.1124/dmd.108.023598&)

19. McCarroll SA, Hadnott TN, Perry GH, Sabeti PC, Zody MC, Barrett JC, et al. Common deletion polymorphisms in the human genome. Nat Genet. 2006;38:86–92. doi: 10.1038/ng1696.  [DOI](https://doi.org/10.1038/ng1696) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16468122/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Common%20deletion%20polymorphisms%20in%20the%20human%20genome&author=SA%20McCarroll&author=TN%20Hadnott&author=GH%20Perry&author=PC%20Sabeti&author=MC%20Zody&volume=38&publication_year=2006&pages=86-92&pmid=16468122&doi=10.1038/ng1696&)

20. Sparks R, Ulrich CM, Bigler J, Tworoger SS, Yasui Y, Rajan KB, et al. UDP-glucuronosyltransferase and sulfotransferase polymorphisms, sex hormone concentrations, and tumor receptor status in breast cancer patients. Breast Cancer Res. 2004;6:R488–R498. doi: 10.1186/bcr818.  [DOI](https://doi.org/10.1186/bcr818) | [PMC free article](/articles/PMC549165/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15318931/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Breast%20Cancer%20Res&title=UDP-glucuronosyltransferase%20and%20sulfotransferase%20polymorphisms,%20sex%20hormone%20concentrations,%20and%20tumor%20receptor%20status%20in%20breast%20cancer%20patients&author=R%20Sparks&author=CM%20Ulrich&author=J%20Bigler&author=SS%20Tworoger&author=Y%20Yasui&volume=6&publication_year=2004&pages=R488-R498&pmid=15318931&doi=10.1186/bcr818&)

21. Hajdinjak T, Zagradisnik B. Prostate cancer and polymorphism D85Y in gene for dihydrotestosterone degrading enzyme UGT2B15: frequency of DD homozygotes increases with Gleason Score. Prostate. 2004;59:436–439. doi: 10.1002/pros.20024.  [DOI](https://doi.org/10.1002/pros.20024) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15065092/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Prostate&title=Prostate%20cancer%20and%20polymorphism%20D85Y%20in%20gene%20for%20dihydrotestosterone%20degrading%20enzyme%20UGT2B15:%20frequency%20of%20DD%20homozygotes%20increases%20with%20Gleason%20Score&author=T%20Hajdinjak&author=B%20Zagradisnik&volume=59&publication_year=2004&pages=436-439&pmid=15065092&doi=10.1002/pros.20024&)

22. MacLeod SL, Nowell S, Plaxco J, Lang NP. An allele-specific polymerase chain reaction method for the determination of the D85Y polymorphism in the human UDP-glucuronosyltransferase 2B15 gene in a case-control study of prostate cancer. Ann Surg Oncol. 2000;7:777–782. doi: 10.1007/s10434-000-0777-3.  [DOI](https://doi.org/10.1007/s10434-000-0777-3) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11129427/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Surg%20Oncol&title=An%20allele-specific%20polymerase%20chain%20reaction%20method%20for%20the%20determination%20of%20the%20D85Y%20polymorphism%20in%20the%20human%20UDP-glucuronosyltransferase%202B15%20gene%20in%20a%20case-control%20study%20of%20prostate%20cancer&author=SL%20MacLeod&author=S%20Nowell&author=J%20Plaxco&author=NP%20Lang&volume=7&publication_year=2000&pages=777-782&pmid=11129427&doi=10.1007/s10434-000-0777-3&)

23. Park J, Chen L, Shade K, Lazarus P, Seigne J, Patterson S, et al. Asp85tyr polymorphism in the udp-glucuronosyltransferase (UGT) 2B15 gene and the risk of prostate cancer. J Urol. 2004;171:2484–2488. doi: 10.1097/01.ju.0000117748.44313.43.  [DOI](https://doi.org/10.1097/01.ju.0000117748.44313.43) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15126881/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Urol&title=Asp85tyr%20polymorphism%20in%20the%20udp-glucuronosyltransferase%20(UGT)%202B15%20gene%20and%20the%20risk%20of%20prostate%20cancer&author=J%20Park&author=L%20Chen&author=K%20Shade&author=P%20Lazarus&author=J%20Seigne&volume=171&publication_year=2004&pages=2484-2488&pmid=15126881&doi=10.1097/01.ju.0000117748.44313.43&)

24. Park J, Chen L, Ratnashinge L, Sellers TA, Tanner JP, Lee JH, et al. Deletion polymorphism of UDP-glucuronosyltransferase 2B17 and risk of prostate cancer in African American and Caucasian men. Cancer Epidemiol Biomarkers Prev. 2006;15:1473–1478. doi: 10.1158/1055-9965.EPI-06-0141.  [DOI](https://doi.org/10.1158/1055-9965.EPI-06-0141) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16896035/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Epidemiol%20Biomarkers%20Prev&title=Deletion%20polymorphism%20of%20UDP-glucuronosyltransferase%202B17%20and%20risk%20of%20prostate%20cancer%20in%20African%20American%20and%20Caucasian%20men&author=J%20Park&author=L%20Chen&author=L%20Ratnashinge&author=TA%20Sellers&author=JP%20Tanner&volume=15&publication_year=2006&pages=1473-1478&pmid=16896035&doi=10.1158/1055-9965.EPI-06-0141&)

25. Karypidis AH, Olsson M, Andersson SO, Rane A, Ekstrom L. Deletion polymorphism of the UGT2B17 gene is associated with increased risk for prostate cancer and correlated to gene expression in the prostate. Pharmacogenomics J. 2008;8:147–151. doi: 10.1038/sj.tpj.6500449.  [DOI](https://doi.org/10.1038/sj.tpj.6500449) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17387331/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=Deletion%20polymorphism%20of%20the%20UGT2B17%20gene%20is%20associated%20with%20increased%20risk%20for%20prostate%20cancer%20and%20correlated%20to%20gene%20expression%20in%20the%20prostate&author=AH%20Karypidis&author=M%20Olsson&author=SO%20Andersson&author=A%20Rane&author=L%20Ekstrom&volume=8&publication_year=2008&pages=147-151&pmid=17387331&doi=10.1038/sj.tpj.6500449&)

26. Nagar S, Remmel RP. Uridine diphosphoglucuronosyltransferase pharmacogenetics and cancer. Oncogene. 2006;25:1659–1672. doi: 10.1038/sj.onc.1209375.  [DOI](https://doi.org/10.1038/sj.onc.1209375) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16550166/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncogene&title=Uridine%20diphosphoglucuronosyltransferase%20pharmacogenetics%20and%20cancer&author=S%20Nagar&author=RP%20Remmel&volume=25&publication_year=2006&pages=1659-1672&pmid=16550166&doi=10.1038/sj.onc.1209375&)

27. Desai AA, Innocenti F, Ratain MJ. UGT pharmacogenomics: implications for cancer risk and cancer therapeutics. Pharmacogenetics. 2003;13:517–523. doi: 10.1097/01.fpc.0000054116.14659.e5.  [DOI](https://doi.org/10.1097/01.fpc.0000054116.14659.e5) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12893990/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=UGT%20pharmacogenomics:%20implications%20for%20cancer%20risk%20and%20cancer%20therapeutics&author=AA%20Desai&author=F%20Innocenti&author=MJ%20Ratain&volume=13&publication_year=2003&pages=517-523&pmid=12893990&doi=10.1097/01.fpc.0000054116.14659.e5&)

28. Sun C, Southard C, Witonsky DB, Olopade O, Di Rienzo A. Allelic imbalance (AI) identifies novel tissue specific cis-regulatory variation for human UGT2B15. Hum Mutat. 2010;31:99–107. doi: 10.1002/humu.21145.  [DOI](https://doi.org/10.1002/humu.21145) | [PMC free article](/articles/PMC2922057/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19847790/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mutat&title=Allelic%20imbalance%20(AI)%20identifies%20novel%20tissue%20specific%20cis-regulatory%20variation%20for%20human%20UGT2B15&author=C%20Sun&author=C%20Southard&author=DB%20Witonsky&author=O%20Olopade&author=A%20Di%20Rienzo&volume=31&publication_year=2010&pages=99-107&pmid=19847790&doi=10.1002/humu.21145&)

29. The International HapMap Consortium. A second generation human haplotype map of over 3.1 million SNPs. Nature. 2007;449:851–861. doi: 10.1038/nature06258.  [DOI](https://doi.org/10.1038/nature06258) | [PMC free article](/articles/PMC2689609/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17943122/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=A%20second%20generation%20human%20haplotype%20map%20of%20over%203.1%20million%20SNPs&volume=449&publication_year=2007&pages=851-861&pmid=17943122&doi=10.1038/nature06258&)

30. Frith MC, Li MC, Weng Z. Cluster-buster: finding dense clusters of motifs in DNA sequences. Nucleic Acids Res. 2003;31:3666–3668. doi: 10.1093/nar/gkg540.  [DOI](https://doi.org/10.1093/nar/gkg540) | [PMC free article](/articles/PMC168947/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12824389/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=Cluster-buster:%20finding%20dense%20clusters%20of%20motifs%20in%20DNA%20sequences&author=MC%20Frith&author=MC%20Li&author=Z%20Weng&volume=31&publication_year=2003&pages=3666-3668&pmid=12824389&doi=10.1093/nar/gkg540&)

31. Sun C, Di Rienzo A. UGT2B7 is not expressed in normal breast. Breast Cancer Res Treat. 2009;117:225–226. doi: 10.1007/s10549-008-0280-2.  [DOI](https://doi.org/10.1007/s10549-008-0280-2) | [PMC free article](/articles/PMC2921443/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19116784/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Breast%20Cancer%20Res%20Treat&title=UGT2B7%20is%20not%20expressed%20in%20normal%20breast&author=C%20Sun&author=A%20Di%20Rienzo&volume=117&publication_year=2009&pages=225-226&pmid=19116784&doi=10.1007/s10549-008-0280-2&)

32. Stephens M, Sloan JS, Robertson PD, Scheet P, Nickerson DA. Automating sequence-based detection and genotyping of SNPs from diploid samples. Nat Genet. 2006;38:375–381. doi: 10.1038/ng1746.  [DOI](https://doi.org/10.1038/ng1746) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16493422/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Automating%20sequence-based%20detection%20and%20genotyping%20of%20SNPs%20from%20diploid%20samples&author=M%20Stephens&author=JS%20Sloan&author=PD%20Robertson&author=P%20Scheet&author=DA%20Nickerson&volume=38&publication_year=2006&pages=375-381&pmid=16493422&doi=10.1038/ng1746&)

33. Carlson CS, Eberle MA, Rieder MJ, Yi Q, Kruglyak L, Nickerson DA. Selecting a maximally informative set of single-nucleotide polymorphisms for association analyses using linkage disequilibrium. Am J Hum Genet. 2004;74:106–120. doi: 10.1086/381000.  [DOI](https://doi.org/10.1086/381000) | [PMC free article](/articles/PMC1181897/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14681826/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=Selecting%20a%20maximally%20informative%20set%20of%20single-nucleotide%20polymorphisms%20for%20association%20analyses%20using%20linkage%20disequilibrium&author=CS%20Carlson&author=MA%20Eberle&author=MJ%20Rieder&author=Q%20Yi&author=L%20Kruglyak&volume=74&publication_year=2004&pages=106-120&pmid=14681826&doi=10.1086/381000&)

34. Bustin SA, Benes V, Garson JA, Hellemans J, Huggett J, Kubista M, et al. The MIQE guidelines: minimum information for publication of quantitative real-time PCR experiments. Clin Chem. 2009;55:611–622. doi: 10.1373/clinchem.2008.112797.  [DOI](https://doi.org/10.1373/clinchem.2008.112797) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19246619/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Chem&title=The%20MIQE%20guidelines:%20minimum%20information%20for%20publication%20of%20quantitative%20real-time%20PCR%20experiments&author=SA%20Bustin&author=V%20Benes&author=JA%20Garson&author=J%20Hellemans&author=J%20Huggett&volume=55&publication_year=2009&pages=611-622&pmid=19246619&doi=10.1373/clinchem.2008.112797&)

35. Purcell S, Neale B, Todd-Brown K, Thomas L, Ferreira MA, Bender D, et al. PLINK: a tool set for whole-genome association and population-based linkage analyses. Am J Hum Genet. 2007;81:559–575. doi: 10.1086/519795.  [DOI](https://doi.org/10.1086/519795) | [PMC free article](/articles/PMC1950838/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17701901/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=PLINK:%20a%20tool%20set%20for%20whole-genome%20association%20and%20population-based%20linkage%20analyses&author=S%20Purcell&author=B%20Neale&author=K%20Todd-Brown&author=L%20Thomas&author=MA%20Ferreira&volume=81&publication_year=2007&pages=559-575&pmid=17701901&doi=10.1086/519795&)

36. Izukawa T, Nakajima M, Fujiwara R, Yamanaka H, Fukami T, Takamiya M, et al. Quantitative analysis of UDP-glucuronosyltransferase (UGT) 1A and UGT2B expression levels in human livers. Drug Metab Dispos. 2009;37:1759–1768. doi: 10.1124/dmd.109.027227.  [DOI](https://doi.org/10.1124/dmd.109.027227) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19439486/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Quantitative%20analysis%20of%20UDP-glucuronosyltransferase%20(UGT)%201A%20and%20UGT2B%20expression%20levels%20in%20human%20livers&author=T%20Izukawa&author=M%20Nakajima&author=R%20Fujiwara&author=H%20Yamanaka&author=T%20Fukami&volume=37&publication_year=2009&pages=1759-1768&pmid=19439486&doi=10.1124/dmd.109.027227&)

37. Ramirez J, Mirkov S, Zhang W, Chen P, Das S, Liu W, et al. Hepatocyte nuclear factor-1 alpha is associated with UGT1A1, UGT1A9 and UGT2B7 mRNA expression in human liver. Pharmacogenomics J. 2008;8:152–161. doi: 10.1038/sj.tpj.6500454.  [DOI](https://doi.org/10.1038/sj.tpj.6500454) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17440429/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=Hepatocyte%20nuclear%20factor-1%20alpha%20is%20associated%20with%20UGT1A1,%20UGT1A9%20and%20UGT2B7%20mRNA%20expression%20in%20human%20liver&author=J%20Ramirez&author=S%20Mirkov&author=W%20Zhang&author=P%20Chen&author=S%20Das&volume=8&publication_year=2008&pages=152-161&pmid=17440429&doi=10.1038/sj.tpj.6500454&)

38. Harrington WR, Sengupta S, Katzenellenbogen BS. Estrogen regulation of the glucuronidation enzyme UGT2B15 in estrogen receptor-positive breast cancer cells. Endocrinology. 2006;147:3843–3850. doi: 10.1210/en.2006-0358.  [DOI](https://doi.org/10.1210/en.2006-0358) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16690804/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Endocrinology&title=Estrogen%20regulation%20of%20the%20glucuronidation%20enzyme%20UGT2B15%20in%20estrogen%20receptor-positive%20breast%20cancer%20cells&author=WR%20Harrington&author=S%20Sengupta&author=BS%20Katzenellenbogen&volume=147&publication_year=2006&pages=3843-3850&pmid=16690804&doi=10.1210/en.2006-0358&)

39. Bao BY, Chuang BF, Wang Q, Sartor O, Balk SP, Brown M, et al. Androgen receptor mediates the expression of UDP-glucuronosyltransferase 2 B15 and B17 genes. Prostate. 2008;68:839–848. doi: 10.1002/pros.20749.  [DOI](https://doi.org/10.1002/pros.20749) | [PMC free article](/articles/PMC2703184/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18302198/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Prostate&title=Androgen%20receptor%20mediates%20the%20expression%20of%20UDP-glucuronosyltransferase%202%20B15%20and%20B17%20genes&author=BY%20Bao&author=BF%20Chuang&author=Q%20Wang&author=O%20Sartor&author=SP%20Balk&volume=68&publication_year=2008&pages=839-848&pmid=18302198&doi=10.1002/pros.20749&)

40. Belanger A, Hum DW, Beaulieu M, Levesque E, Guillemette C, Tchernof A, et al. Characterization and regulation of UDP-glucuronosyltransferases in steroid target tissues. J Steroid Biochem Mol Biol. 1998;65:301–310. doi: 10.1016/s0960-0760(97)00183-0.  [DOI](https://doi.org/10.1016/s0960-0760(97)00183-0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9699884/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Steroid%20Biochem%20Mol%20Biol&title=Characterization%20and%20regulation%20of%20UDP-glucuronosyltransferases%20in%20steroid%20target%20tissues&author=A%20Belanger&author=DW%20Hum&author=M%20Beaulieu&author=E%20Levesque&author=C%20Guillemette&volume=65&publication_year=1998&pages=301-310&pmid=9699884&doi=10.1016/s0960-0760(97)00183-0&)

41. Guillemette C, Levesque E, Beaulieu M, Turgeon D, Hum DW, Belanger A. Differential regulation of two uridine diphospho-glucuronosyltransferases, UGT2B15 and UGT2B17, in human prostate LNCaP cells. Endocrinology. 1997;138:2998–3005. doi: 10.1210/endo.138.7.5226.  [DOI](https://doi.org/10.1210/endo.138.7.5226) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9202245/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Endocrinology&title=Differential%20regulation%20of%20two%20uridine%20diphospho-glucuronosyltransferases,%20UGT2B15%20and%20UGT2B17,%20in%20human%20prostate%20LNCaP%20cells&author=C%20Guillemette&author=E%20Levesque&author=M%20Beaulieu&author=D%20Turgeon&author=DW%20Hum&volume=138&publication_year=1997&pages=2998-3005&pmid=9202245&doi=10.1210/endo.138.7.5226&)

42. Hum DW, Belanger A, Levesque E, Barbier O, Beaulieu M, Albert C, et al. Characterization of UDP-glucuronosyltransferases active on steroid hormones. J Steroid Biochem Mol Biol. 1999;69:413–423. doi: 10.1016/s0960-0760(99)00061-8.  [DOI](https://doi.org/10.1016/s0960-0760(99)00061-8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10419020/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Steroid%20Biochem%20Mol%20Biol&title=Characterization%20of%20UDP-glucuronosyltransferases%20active%20on%20steroid%20hormones&author=DW%20Hum&author=A%20Belanger&author=E%20Levesque&author=O%20Barbier&author=M%20Beaulieu&volume=69&publication_year=1999&pages=413-423&pmid=10419020&doi=10.1016/s0960-0760(99)00061-8&)

43. Straub RH, Miller LE, Scholmerich J, Zietz B. Cytokines and hormones as possible links between endocrinosenescence and immunosenescence. J Neuroimmunol. 2000;109:10–15. doi: 10.1016/s0165-5728(00)00296-4.  [DOI](https://doi.org/10.1016/s0165-5728(00)00296-4) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10969175/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Neuroimmunol&title=Cytokines%20and%20hormones%20as%20possible%20links%20between%20endocrinosenescence%20and%20immunosenescence&author=RH%20Straub&author=LE%20Miller&author=J%20Scholmerich&author=B%20Zietz&volume=109&publication_year=2000&pages=10-15&pmid=10969175&doi=10.1016/s0165-5728(00)00296-4&)

44. Spielman RS, Bastone LA, Burdick JT, Morley M, Ewens WJ, Cheung VG. Common genetic variants account for differences in gene expression among ethnic groups. Nat Genet. 2007;39:226–231. doi: 10.1038/ng1955.  [DOI](https://doi.org/10.1038/ng1955) | [PMC free article](/articles/PMC3005333/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17206142/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Common%20genetic%20variants%20account%20for%20differences%20in%20gene%20expression%20among%20ethnic%20groups&author=RS%20Spielman&author=LA%20Bastone&author=JT%20Burdick&author=M%20Morley&author=WJ%20Ewens&volume=39&publication_year=2007&pages=226-231&pmid=17206142&doi=10.1038/ng1955&)

45. Stranger BE, Forrest MS, Dunning M, Ingle CE, Beazley C, Thorne N, et al. Relative impact of nucleotide and copy number variation on gene expression phenotypes. Science. 2007;315:848–853. doi: 10.1126/science.1136678.  [DOI](https://doi.org/10.1126/science.1136678) | [PMC free article](/articles/PMC2665772/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17289997/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=Relative%20impact%20of%20nucleotide%20and%20copy%20number%20variation%20on%20gene%20expression%20phenotypes&author=BE%20Stranger&author=MS%20Forrest&author=M%20Dunning&author=CE%20Ingle&author=C%20Beazley&volume=315&publication_year=2007&pages=848-853&pmid=17289997&doi=10.1126/science.1136678&)

46. Grant SF, Thorleifsson G, Reynisdottir I, Benediktsson R, Manolescu A, Sainz J, et al. Variant of transcription factor 7-like 2 (TCF7L2) gene confers risk of type 2 diabetes. Nat Genet. 2006;38:320–323. doi: 10.1038/ng1732.  [DOI](https://doi.org/10.1038/ng1732) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16415884/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Variant%20of%20transcription%20factor%207-like%202%20(TCF7L2)%20gene%20confers%20risk%20of%20type%202%20diabetes&author=SF%20Grant&author=G%20Thorleifsson&author=I%20Reynisdottir&author=R%20Benediktsson&author=A%20Manolescu&volume=38&publication_year=2006&pages=320-323&pmid=16415884&doi=10.1038/ng1732&)

47. Broderick P, Carvajal-Carmona L, Pittman AM, Webb E, Howarth K, Rowan A, et al. A genome-wide association study shows that common alleles of SMAD7 influence colorectal cancer risk. Nat Genet. 2007;39:1315–1317. doi: 10.1038/ng.2007.18.  [DOI](https://doi.org/10.1038/ng.2007.18) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17934461/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=A%20genome-wide%20association%20study%20shows%20that%20common%20alleles%20of%20SMAD7%20influence%20colorectal%20cancer%20risk&author=P%20Broderick&author=L%20Carvajal-Carmona&author=AM%20Pittman&author=E%20Webb&author=K%20Howarth&volume=39&publication_year=2007&pages=1315-1317&pmid=17934461&doi=10.1038/ng.2007.18&)

48. Harismendy O, Frazer KA. Elucidating the role of 8q24 in colorectal cancer. Nat Genet. 2009;41:868–869. doi: 10.1038/ng0809-868.  [DOI](https://doi.org/10.1038/ng0809-868) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19639026/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Elucidating%20the%20role%20of%208q24%20in%20colorectal%20cancer&author=O%20Harismendy&author=KA%20Frazer&volume=41&publication_year=2009&pages=868-869&pmid=19639026&doi=10.1038/ng0809-868&)

49. Gaulton KJ, Nammo T, Pasquali L, Simon JM, Giresi PG, Fogarty MP, et al. A map of open chromatin in human pancreatic islets. Nat Genet. 2010;42:255–259. doi: 10.1038/ng.530.  [DOI](https://doi.org/10.1038/ng.530) | [PMC free article](/articles/PMC2828505/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20118932/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=A%20map%20of%20open%20chromatin%20in%20human%20pancreatic%20islets&author=KJ%20Gaulton&author=T%20Nammo&author=L%20Pasquali&author=JM%20Simon&author=PG%20Giresi&volume=42&publication_year=2010&pages=255-259&pmid=20118932&doi=10.1038/ng.530&)

50. Wasserman NF, Aneas I, Nobrega MA. An 8q24 gene desert variant associated with prostate cancer risk confers differential in vivo activity to a MYC enhancer. Genome Res. 2010;20:1191–1197. doi: 10.1101/gr.105361.110.  [DOI](https://doi.org/10.1101/gr.105361.110) | [PMC free article](/articles/PMC2928497/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20627891/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Res&title=An%208q24%20gene%20desert%20variant%20associated%20with%20prostate%20cancer%20risk%20confers%20differential%20in%20vivo%20activity%20to%20a%20MYC%20enhancer&author=NF%20Wasserman&author=I%20Aneas&author=MA%20Nobrega&volume=20&publication_year=2010&pages=1191-1197&pmid=20627891&doi=10.1101/gr.105361.110&)

51. Pittman AM, Naranjo S, Webb E, Broderick P, Lips EH, van Wezel T, et al. The colorectal cancer risk at 18q21 is caused by a novel variant altering SMAD7 expression. Genome Res. 2009;19:987–993. doi: 10.1101/gr.092668.109.  [DOI](https://doi.org/10.1101/gr.092668.109) | [PMC free article](/articles/PMC2694486/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19395656/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Res&title=The%20colorectal%20cancer%20risk%20at%2018q21%20is%20caused%20by%20a%20novel%20variant%20altering%20SMAD7%20expression&author=AM%20Pittman&author=S%20Naranjo&author=E%20Webb&author=P%20Broderick&author=EH%20Lips&volume=19&publication_year=2009&pages=987-993&pmid=19395656&doi=10.1101/gr.092668.109&)

52. The 1000 Genomes Project Consortium. A map of human genome variation from population-scale sequencing. Nature. 2010;467:1061–1073. doi: 10.1038/nature09534.  [DOI](https://doi.org/10.1038/nature09534) | [PMC free article](/articles/PMC3042601/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20981092/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=A%20map%20of%20human%20genome%20variation%20from%20population-scale%20sequencing&volume=467&publication_year=2010&pages=1061-1073&pmid=20981092&doi=10.1038/nature09534&)

53. Innocenti F, Liu W, Fackenthal D, Ramirez J, Chen P, Ye X, et al. Single nucleotide polymorphism discovery and functional assessment of variation in the UDP-glucuronosyltransferase 2B7 gene. Pharmacogenet Genomics. 2008;18:683–697. doi: 10.1097/FPC.0b013e3283037fe4.  [DOI](https://doi.org/10.1097/FPC.0b013e3283037fe4) | [PMC free article](/articles/PMC2680356/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18622261/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Single%20nucleotide%20polymorphism%20discovery%20and%20functional%20assessment%20of%20variation%20in%20the%20UDP-glucuronosyltransferase%202B7%20gene&author=F%20Innocenti&author=W%20Liu&author=D%20Fackenthal&author=J%20Ramirez&author=P%20Chen&volume=18&publication_year=2008&pages=683-697&pmid=18622261&doi=10.1097/FPC.0b013e3283037fe4&)

54. Maitland ML, Grimsley C, Kuttab-Boulos H, Witonsky D, Kasza KE, Yang L, et al. Comparative genomics analysis of human sequence variation in the UGT1A gene cluster. Pharmacogenomics J. 2006;6:52–62. doi: 10.1038/sj.tpj.6500351.  [DOI](https://doi.org/10.1038/sj.tpj.6500351) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16314881/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=Comparative%20genomics%20analysis%20of%20human%20sequence%20variation%20in%20the%20UGT1A%20gene%20cluster&author=ML%20Maitland&author=C%20Grimsley&author=H%20Kuttab-Boulos&author=D%20Witonsky&author=KE%20Kasza&volume=6&publication_year=2006&pages=52-62&pmid=16314881&doi=10.1038/sj.tpj.6500351&)

55. Thompson EE, Kuttab-Boulos H, Witonsky D, Yang L, Roe BA, Di Rienzo A. CYP3A variation and the evolution of salt-sensitivity variants. Am J Hum Genet. 2004;75:1059–1069. doi: 10.1086/426406.  [DOI](https://doi.org/10.1086/426406) | [PMC free article](/articles/PMC1182141/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15492926/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=CYP3A%20variation%20and%20the%20evolution%20of%20salt-sensitivity%20variants&author=EE%20Thompson&author=H%20Kuttab-Boulos&author=D%20Witonsky&author=L%20Yang&author=BA%20Roe&volume=75&publication_year=2004&pages=1059-1069&pmid=15492926&doi=10.1086/426406&)

56. Frazer KA, Tao H, Osoegawa K, de Jong PJ, Chen X, Doherty MF, et al. Noncoding sequences conserved in a limited number of mammals in the SIM2 interval are frequently functional. Genome Res. 2004;14:367–372. doi: 10.1101/gr.1961204.  [DOI](https://doi.org/10.1101/gr.1961204) | [PMC free article](/articles/PMC353216/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14962988/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Res&title=Noncoding%20sequences%20conserved%20in%20a%20limited%20number%20of%20mammals%20in%20the%20SIM2%20interval%20are%20frequently%20functional&author=KA%20Frazer&author=H%20Tao&author=K%20Osoegawa&author=PJ%20de%20Jong&author=X%20Chen&volume=14&publication_year=2004&pages=367-372&pmid=14962988&doi=10.1101/gr.1961204&)

57. Loots GG, Locksley RM, Blankespoor CM, Wang ZE, Miller W, Rubin EM, et al. Identification of a coordinate regulator of interleukins 4, 13, and 5 by cross-species sequence comparisons. Science. 2000;288:136–140. doi: 10.1126/science.288.5463.136.  [DOI](https://doi.org/10.1126/science.288.5463.136) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10753117/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=Identification%20of%20a%20coordinate%20regulator%20of%20interleukins%204,%2013,%20and%205%20by%20cross-species%20sequence%20comparisons&author=GG%20Loots&author=RM%20Locksley&author=CM%20Blankespoor&author=ZE%20Wang&author=W%20Miller&volume=288&publication_year=2000&pages=136-140&pmid=10753117&doi=10.1126/science.288.5463.136&)

58. Schwartz S, Kent WJ, Smit A, Zhang Z, Baertsch R, Hardison RC, et al. Human-mouse alignments with BLASTZ. Genome Res. 2003;13:103–107. doi: 10.1101/gr.809403.  [DOI](https://doi.org/10.1101/gr.809403) | [PMC free article](/articles/PMC430961/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12529312/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Res&title=Human-mouse%20alignments%20with%20BLASTZ&author=S%20Schwartz&author=WJ%20Kent&author=A%20Smit&author=Z%20Zhang&author=R%20Baertsch&volume=13&publication_year=2003&pages=103-107&pmid=12529312&doi=10.1101/gr.809403&)

59. Thomas JW, Touchman JW, Blakesley RW, Bouffard GG, Beckstrom-Sternberg SM, Margulies EH, et al. Comparative analyses of multi-species sequences from targeted genomic regions. Nature. 2003;424:788–793. doi: 10.1038/nature01858.  [DOI](https://doi.org/10.1038/nature01858) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12917688/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Comparative%20analyses%20of%20multi-species%20sequences%20from%20targeted%20genomic%20regions&author=JW%20Thomas&author=JW%20Touchman&author=RW%20Blakesley&author=GG%20Bouffard&author=SM%20Beckstrom-Sternberg&volume=424&publication_year=2003&pages=788-793&pmid=12917688&doi=10.1038/nature01858&)

60. Birney E, Stamatoyannopoulos JA, Dutta A, Guigo R, Gingeras TR, Margulies EH, et al. Identification and analysis of functional elements in 1% of the human genome by the ENCODE pilot project. Nature. 2007;447:799–816. doi: 10.1038/nature05874.  [DOI](https://doi.org/10.1038/nature05874) | [PMC free article](/articles/PMC2212820/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17571346/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Identification%20and%20analysis%20of%20functional%20elements%20in%201%%20of%20the%20human%20genome%20by%20the%20ENCODE%20pilot%20project&author=E%20Birney&author=JA%20Stamatoyannopoulos&author=A%20Dutta&author=R%20Guigo&author=TR%20Gingeras&volume=447&publication_year=2007&pages=799-816&pmid=17571346&doi=10.1038/nature05874&)

61. Key TJ. Serum oestradiol and breast cancer risk. Endocr Relat Cancer. 1999;6:175–180. doi: 10.1677/erc.0.0060175.  [DOI](https://doi.org/10.1677/erc.0.0060175) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10731106/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Endocr%20Relat%20Cancer&title=Serum%20oestradiol%20and%20breast%20cancer%20risk&author=TJ%20Key&volume=6&publication_year=1999&pages=175-180&pmid=10731106&doi=10.1677/erc.0.0060175&)

62. Thomas HV, Reeves GK, Key TJ. Endogenous estrogen and postmenopausal breast cancer: a quantitative review. Cancer Causes Control. 1997;8:922–928. doi: 10.1023/a:1018476631561.  [DOI](https://doi.org/10.1023/a:1018476631561) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9427435/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Causes%20Control&title=Endogenous%20estrogen%20and%20postmenopausal%20breast%20cancer:%20a%20quantitative%20review&author=HV%20Thomas&author=GK%20Reeves&author=TJ%20Key&volume=8&publication_year=1997&pages=922-928&pmid=9427435&doi=10.1023/a:1018476631561&)

63. Cauley JA, Lucas FL, Kuller LH, Stone K, Browner W, Cummings SR. Elevated serum estradiol and testosterone concentrations are associated with a high risk for breast cancer. Study of Osteoporotic Fractures Research Group. Ann Intern Med. 1999;130:270–277. doi: 10.7326/0003-4819-130-4_part_1-199902160-00004.  [DOI](https://doi.org/10.7326/0003-4819-130-4_part_1-199902160-00004) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10068384/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Intern%20Med&title=Elevated%20serum%20estradiol%20and%20testosterone%20concentrations%20are%20associated%20with%20a%20high%20risk%20for%20breast%20cancer.%20Study%20of%20Osteoporotic%20Fractures%20Research%20Group&author=JA%20Cauley&author=FL%20Lucas&author=LH%20Kuller&author=K%20Stone&author=W%20Browner&volume=130&publication_year=1999&pages=270-277&pmid=10068384&doi=10.7326/0003-4819-130-4_part_1-199902160-00004&)

64. Guillemette C, Belanger A, Lepine J. Metabolic inactivation of estrogens in breast tissue by UDP-glucuronosyltransferase enzymes: an overview. Breast Cancer Res. 2004;6:246–254. doi: 10.1186/bcr936.  [DOI](https://doi.org/10.1186/bcr936) | [PMC free article](/articles/PMC1064083/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15535854/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Breast%20Cancer%20Res&title=Metabolic%20inactivation%20of%20estrogens%20in%20breast%20tissue%20by%20UDP-glucuronosyltransferase%20enzymes:%20an%20overview&author=C%20Guillemette&author=A%20Belanger&author=J%20Lepine&volume=6&publication_year=2004&pages=246-254&pmid=15535854&doi=10.1186/bcr936&)

65. Thijssen JH, Blankenstein MA. Endogenous oestrogens and androgens in normal and malignant endometrial and mammary tissues. Eur J Cancer Clin Oncol. 1989;25:1953–1959. doi: 10.1016/0277-5379(89)90377-5.  [DOI](https://doi.org/10.1016/0277-5379(89)90377-5) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/2632276/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Cancer%20Clin%20Oncol&title=Endogenous%20oestrogens%20and%20androgens%20in%20normal%20and%20malignant%20endometrial%20and%20mammary%20tissues&author=JH%20Thijssen&author=MA%20Blankenstein&volume=25&publication_year=1989&pages=1953-1959&pmid=2632276&doi=10.1016/0277-5379(89)90377-5&)

66. Mady EA, Ramadan EE, Ossman AA. Sex steroid hormones in serum and tissue of benign and malignant breast tumor patients. Dis Markers. 2000;16:151–157. doi: 10.1155/2000/305940.  [DOI](https://doi.org/10.1155/2000/305940) | [PMC free article](/articles/PMC3851634/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11381198/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Dis%20Markers&title=Sex%20steroid%20hormones%20in%20serum%20and%20tissue%20of%20benign%20and%20malignant%20breast%20tumor%20patients&author=EA%20Mady&author=EE%20Ramadan&author=AA%20Ossman&volume=16&publication_year=2000&pages=151-157&pmid=11381198&doi=10.1155/2000/305940&)

67. Pharoah PD, Tyrer J, Dunning AM, Easton DF, Ponder BA. Association between common variation in 120 candidate genes and breast cancer risk. PLoS Genet. 2007;3:e42. doi: 10.1371/journal.pgen.0030042.  [DOI](https://doi.org/10.1371/journal.pgen.0030042) | [PMC free article](/articles/PMC1828694/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17367212/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet&title=Association%20between%20common%20variation%20in%20120%20candidate%20genes%20and%20breast%20cancer%20risk&author=PD%20Pharoah&author=J%20Tyrer&author=AM%20Dunning&author=DF%20Easton&author=BA%20Ponder&volume=3&publication_year=2007&pages=e42&pmid=17367212&doi=10.1371/journal.pgen.0030042&)

68. Turgeon D, Carrier JS, Levesque E, Hum DW, Belanger A. Relative enzymatic activity, protein stability, and tissue distribution of human steroid-metabolizing UGT2B subfamily members. Endocrinology. 2001;142:778–787. doi: 10.1210/endo.142.2.7958.  [DOI](https://doi.org/10.1210/endo.142.2.7958) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11159850/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Endocrinology&title=Relative%20enzymatic%20activity,%20protein%20stability,%20and%20tissue%20distribution%20of%20human%20steroid-metabolizing%20UGT2B%20subfamily%20members&author=D%20Turgeon&author=JS%20Carrier&author=E%20Levesque&author=DW%20Hum&author=A%20Belanger&volume=142&publication_year=2001&pages=778-787&pmid=11159850&doi=10.1210/endo.142.2.7958&)

69. Yong M, Schwartz SM, Atkinson C, Makar KW, Thomas SS, Newton KM, et al. Associations between polymorphisms in glucuronidation and sulfation enzymes and mammographic breast density in premenopausal women in the United States. Cancer Epidemiol Biomarkers Prev. 2010;19:537–546. doi: 10.1158/1055-9965.EPI-09-0898.  [DOI](https://doi.org/10.1158/1055-9965.EPI-09-0898) | [PMC free article](/articles/PMC2820123/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20142249/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Epidemiol%20Biomarkers%20Prev&title=Associations%20between%20polymorphisms%20in%20glucuronidation%20and%20sulfation%20enzymes%20and%20mammographic%20breast%20density%20in%20premenopausal%20women%20in%20the%20United%20States&author=M%20Yong&author=SM%20Schwartz&author=C%20Atkinson&author=KW%20Makar&author=SS%20Thomas&volume=19&publication_year=2010&pages=537-546&pmid=20142249&doi=10.1158/1055-9965.EPI-09-0898&)
