# Genetic determinants of antithyroid drug-induced agranulocytosis by human leukocyte antigen genotyping and genome-wide association study

## Metadata
**Authors:** Pei-Lung Chen, Shyang-Rong Shih, Pei-Wen Wang, Ying-Chao Lin, Chen-Chung Chu, Jung-Hsin Lin, Szu-Chi Chen, Ching-Chung Chang, Tien-Shang Huang, Keh Sung Tsai, Fen-Yu Tseng, Chih-Yuan Wang, Jin-Ying Lu, Wei-Yih Chiu, Chien-Ching Chang, Yu-Hsuan Chen, Yuan-Tsong Chen, Cathy Shen-Jang Fann, Wei-Shiung Yang, Tien-Chun Chang
**Journal:** Nature Communications
**Date:** 2015 Jul 7
**DOI:** [10.1038/ncomms8633](https://doi.org/10.1038/ncomms8633)
**PMID:** 26151496
**PMCID:** PMC4506516
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4506516/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC4506516/pdf/ncomms8633.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC4506516/pdf/ncomms8633.pdf)

## Abstract

Graves' disease is the leading cause of hyperthyroidism affecting 1.0–1.6% of the population. Antithyroid drugs are the treatment cornerstone, but may cause life-threatening agranulocytosis. Here we conduct a two-stage association study on two separate subject sets (in total 42 agranulocytosis cases and 1,208 Graves' disease controls), using direct human leukocyte antigen genotyping and SNP-based genome-wide association study. We demonstrate HLA-B*38:02 (Armitage trend Pcombined=6.75 × 10−32) and HLA-DRB1*08:03 (Pcombined=1.83 × 10−9) as independent susceptibility loci. The genome-wide association study identifies the same signals. Estimated odds ratios for these two loci comparing effective allele carriers to non-carriers are 21.48 (95% confidence interval=11.13–41.48) and 6.13 (95% confidence interval=3.28–11.46), respectively. Carrying both HLA-B*38:02 and HLA-DRB1*08:03 increases odds ratio to 48.41 (Pcombined=3.32 × 10−21, 95% confidence interval=21.66–108.22). Our results could be useful for antithyroid-induced agranulocytosis and potentially for agranulocytosis caused by other chemicals.

## Results

### Direct classical HLA loci genotyping and association tests

The structure of the study and the number of participants are summarized in [Supplementary Table 1](#S1). For HLA association study using six directly genotyped classical HLA loci (*HLA-A*, *-B*, *-C*, *-DPB1*, *-DQB1* and *-DRB1*), we observed two significant associations at *HLA-B*38:02* (*P*=1.59 × 10^−9^, note: *P* values are Armitage trend *P* values in this manuscript, if not otherwise specified) and *HLA-DRB1*08:03* (*P*=1.24 × 10^−5^) in the first-stage study (21 TiA cases versus 662 GD controls). The second-stage study (additional 21 TiA cases versus 546 GD controls) replicated the two associations at *HLA-B*38:02* (*P*=1.59 × 10^−27^) and *HLA-DRB1*08:03* (*P*=3.91 × 10^−5^). Combined results from 42 TiA cases and 1,208 GD controls showed even stronger associations: *HLA-B*38:02* (*P*_combined_=6.75 × 10^−32^) and *HLA-DRB1*08:03* (*P*_combined_=1.83 × 10^−9^, [Table 1](#t1)). The allele frequencies of *HLA-B*38:02* were 29.8% in the TiA cases, 3.3% in the GD controls and 3.3% in general population in Taiwan[22](#b22). The allele frequencies of *HLA-DRB1*08:03* were 28.6% in the TiA cases, 8.4% in the GD controls and 8.6% in general population[22](#b22). The genotypic information for 42 TiA cases is shown in [Supplementary Table 2](#S1). The allele frequencies of 6 classical HLA genes in 42 TiA cases and in 1,208 GD controls can be found in [Supplementary Data 1](#S1). Odds ratios of these two HLA alleles for effective allele carriers compared with non-carriers were 21.48 (95% confidence interval, CI=11.13–41.48) and 6.13 (95% CI=3.28–11.46), respectively. These two loci are not in the same linkage disequilibrium (LD) block ([Fig. 1](#f1) and [Supplementary Fig. 1](#S1)). In addition, *HLA-A*02:03*, *HLA-C*07:02* and HLA-*DQB1*06:01* also showed significant association signals (*P*=4.22 × 10^−8^, 4.09 × 10^−6^ and 1.29 × 10^−6^, respectively) but were found to be in LD with either *HLA-B*38:02* or *HLA-DRB1*08:03* ([Fig. 1](#f1) and [Supplementary Fig. 1](#S1)), therefore not considered as independent associations. HLA imputation using single-nucleotide polymorphisms (SNPs) at the HLA region was also carried out. The results were highly similar to those based on direct HLA typing ([Table 2](#t2) and [Supplementary Table 3](#S1)).

### Table 1. Association study in the first stage, second stage and combined stage.

| Chr. | SNP (position*) | Study group | TiA |  |  |  |  | GD controls |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
|   |   |   | n | RAF | OR† | (95% CI) | Pvalue‡ | n | RAF | OR† (95% CI) |
| 6 | HLA-B*38:02 (31323319) | First stage | 21 | 0.238 | 7.42 | (3.47–15.90) | 1.59 × 10−09 | 656 | 0.040 | 1.00 |
|   |   | Second stage | 21 | 0.357 | 21.91 | (10.48–45.83) | 1.59 × 10−27 | 546 | 0.025 | 1.00 |
|   |   | Combined stage | 42 | 0.298 | 12.31 | (7.33–20.67) | 6.75 × 10−32 | 1202 | 0.033 | 1.00 |
| 6 | rs17193122 (31335336) | First stage | 21 | 0.262 | 5.74 | (2.74–12.02) | 8.18 × 10−08 | 474 | 0.058 | 1.00 |
|   |   | Second stage | 21 | 0.381 | 16.77 | (8.20–34.30) | 1.15 × 10−24 | 453 | 0.035 | 1.00 |
|   |   | Combined stage | 42 | 0.321 | 9.59 | (5.78–15.90) | 4.29 × 10−27 | 927 | 0.047 | 1.00 |
| 6 | rs116869525 (32389143) | First stage | 21 | 0.262 | 4.66 | (2.24–9.67) | 3.88 × 10−05 | 474 | 0.071 | 1.00 |
|   |   | Second stage | 21 | 0.286 | 3.77 | (1.86–7362) | 8.24 × 10−05 | 453 | 0.096 | 1.00 |
|   |   | Combined stage | 42 | 0.274 | 4.16 | (2.50–6.90) | 1.27 × 10−08 | 927 | 0.083 | 1.00 |
| 6 | HLA-DRB1*08:03 (32552080) | First stage | 21 | 0.286 | 4.76 | (2.36–9.57) | 1.24 × 10−05 | 651 | 0.078 | 1.00 |
|   |   | Second stage | 21 | 0.286 | 3.96 | (1.97–7.98) | 3.91 × 10−05 | 545 | 0.092 | 1.00 |
|   |   | Combined stage | 42 | 0.286 | 4.36 | (2.66–7.15) | 1.83 × 10−09 | 1196 | 0.084 | 1.00 |

Table 1 Caption: Chr., chromosome; CI, confidence interval; GD, Graves' Disease; HLA, human leukocyte antigen; OR, odds ratio; RAF, risk allele frequency; TiA, thionamide-induced agranulocytosis. *The physical positions were annotated according to NCBI build 37.1. †OR values were calculated using allelic association test. ‡P values were calculated using Cochran–Armitage test for trend.

### Figure 1. Regional plot of association signals and LD patterns at the HLA region.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1369/4506516/501bb0cd5570/ncomms8633-f1.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4506516_ncomms8633-f1.jpg)

(Top panel): The x axis shows chromosomal positions. The left y axis shows –log10P values from the GWAS. Colours of the dots indicate the LD relationship between rs17193122 or rs116869525 and their neighbouring SNPs in the 4-Mb region (30–32 Mb for rs17193122 and 32–34 Mb for rs116869525) based on our own data (969 samples). The red dashed line indicates the significance threshold (9.56 × 10−8). The right y axis shows the recombination rate between SNPs calculated based on Asian-ancestry population data from the 1000 Genomes Project. (Bottom panel): LD map based on r2 in the associated regions using genotype results from our own data (969 samples). We construct the plot using the Haploview software version 4.2, and r2 (× 100) values are depicted in the diamonds.

### Table 2. Results of conditional association analysis using direct HLA typing and imputation.

| HLA | Conditional on | Direct HLA typing; P | Imputed HLA using SNP2HLA; P* |
| --- | --- | --- | --- |
| HLA-B*38:02 | HLA-DRB1*08:03 | 1.89 × 10−16 | 2.07 × 10−14 |
| HLA-DRB1*08:03 | HLA-B*38:02 | 2.24 × 10−6 | 1.68 × 10−6 |

Table 2 Caption: HLA, human leukocyte antigen; SNP, single-nucleotide polymorphism. *SNP2HLA (Jia et al.34,40,41)

### Genome-wide association study

GWAS using Affymetrix Axiom Genome-Wide CHB 1 SNP Array (a total of 522,980 autosomal SNPs after quality-control process) showed an excess of small *P* values at the tail of the quantile–quantile distribution plot ([Supplementary Fig. 2](#S1)) but only limited evidence of inflation caused by population stratification (genomic inflation factor *λ*_GC_=1.01). The multidimensional scaling (MDS) and principal component analyses showed no outliers ([Supplementary Figs 3 and 4](#S1)). We identified two independent association signals at the HLA region ([Table 1](#t1), [Figs 1](#f1) and [2](#f2), [Supplementary Table 4](#S1)). The first signal was represented by rs17193122 (*P*_first stage_=8.18 × 10^−8^, *P*_second stage_=1.15 × 10^−24^, *P*_combined_=4.29 × 10^−27^). The second association signal was represented by rs116869525 (*P*_first stage_=3.88 × 10^−5^, *P*_second stage_=8.24 × 10^−5^, *P*_combined_=1.27 × 10^−8^). In addition to the HLA region at chromosome 6p21, the results also revealed one non-HLA locus showing borderline GWAS association signals, represented by rs56343172 at chromosome 3q13 (*P*_first stage_=1.71 × 10^−5^, *P*_second stage_=7.32 × 10^−4^, *P*_combined_=7.75 × 10^−8^).

### Figure 2. Association results from the genome-wide scan.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1369/4506516/52538640ad3d/ncomms8633-f2.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4506516_ncomms8633-f2.jpg)

Manhattan plot shows the genome-wide P values (−log10P value) of the Cochran–Armitage test for trend from 522,980 polymorphic SNPs in 42 TiA cases and 927 GD controls. SNPs on each chromosome were given the same colour. Red line, P=9.56 × 10−8 (as the genome-wide significance cutoff value of this study).

### HLA-B*38:02 and HLA-DRB1*08:03 have independent effects

The associations at the HLA region derived from two different genotyping techniques (direct classical HLA genotyping and Affymetrix SNP Array) provided convergent evidence of the same signals. The first group of SNPs, represented by rs17193122, were in LD with *HLA-B*38:02* ([Fig. 1](#f1)), and their associations with TiA disappeared when they were tested in the regression model containing *HLA-B*38:02*. Similarly, the second group of SNPs, represented by rs116869525, were in LD with *HLA-DRB1*08:03* ([Fig. 1](#f1)), and did not show significant association as well in the regression model containing *HLA-DRB1*08:03*.

Although *HLA-B*38:02* and *HLA-DRB1*08:03* are sometimes in the same extended HLA haplotype, we could demonstrate their independent effects on the basis of three lines of evidence. First, the LD block patterns of TiA cases and GD controls ([Fig. 1](#f1) and [Supplementary Fig. 1](#S1)) clearly showed that these two alleles resided in two different LD blocks. Second, in our stepwise regression model, these two alleles demonstrated independent effects (*P*=1.89 × 10^−16^ for *HLA-B*38:02* and *P*=2.24 × 10^−6^ for *HLA-DRB1*08:03*, respectively, [Supplementary Table 5](#S1)). Third, in those individuals who carried *HLA-B*38:02* (25 in TiA cases and 77 in GD controls), the percentage of individuals who also carried *HLA-DRB1*08:03* was statistically higher in TiA cases (16/25=64%) than in GD controls (15/76=19.74%, *P*=9.44 × 10^−5^).

### HLA-B*38:02 and HLA-DRB1*08:03 have major effects

We observed none of the 42 TiA cases to be homozygous for *HLA-B*38:02*, and only 2 of them to be homozygous for *HLA-DRB1*08:03*. Following the traditional practice for assessing the risk of HLA-related drug adverse effect[17](#b17),[23](#b23), we calculated disease odds ratio for these loci according to the effective allele carrier status. *HLA-B*38:02* was present in 59.52% of TiA cases, but only in 6.41% of GD controls, which transformed into an odds ratio of 21.48 (*P*=6.28 × 10^−18^, 95% CI=11.13–41.48). *HLA-DRB1*08:03* was present in 52.38% of TiA cases, but only in 15.22% of GD controls, which transformed into an odds ratio of 6.13 (*P*=1.35 × 10^−8^, 95% CI=3.28–11.46). For those who carried both *HLA-B*38:02* and *HLA-DRB1*08:03* (38.10% in TiA cases and 1.26% in GD controls), the odds ratio increased to 48.41 (*P*=3.32 × 10^−21^, 95% CI=21.66–108.22). Using a logistic regression procedure ([Supplementary Table 5](#S1)), the final logistic regression model was:

logit(π)=−4.4936+2.6382 × *HLA-B*38:02*+1.2857 × *HLA-DRB1*08:03* (1)

(both loci under additive model), where π was the probability of TiA.

Under this model, the AUC of the receiver operating characteristic (ROC) curve was 81.22% ([Fig. 3](#f3)). Since ∼74% of the TiA patients carry the risk genotypes and routine follow-up of blood cell count is controversial[4](#b4),[5](#b5),[6](#b6). Different follow-up protocols can be proposed on the basis of genetic risk stratification to possibly prevent 74% of this rare condition from occurrence. Further replication is needed to confirm the estimate. Cost-effectiveness analysis will be carried out in further study.

### Figure 3. The ROC curve to discriminate TiA cases (n=42) from GD controls (n=927).

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1369/4506516/dd1be9749229/ncomms8633-f3.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4506516_ncomms8633-f3.jpg)

Sensitivity, specificity, positive and negative predictive values of the model were 61.09% (95% CI=45.64–76.42), 92.13% (95% CI=90.46–93.60), 21.67% (95% CI=14.67–30.11) and 98.57% (95% CI=97.68–99.18), respectively.

### 3D structure modelling

Despite the strong genetic associations, the mechanisms of actions remain enigmatic. To further probe the direct interactions between the HLA proteins and the thionamide drugs, 3D structure modelling, molecular docking and molecular dynamics simulations were carried out to determine the possible binding modes and to estimate the binding affinities of the thionamide–HLA complexes. For HLA-B*38:02 and HLA-B*38:01 proteins, binding modes in pocket B and pocket F exhibits comparable binding affinities in the range between −4.48 and −6.36 kcal mol^−1^. It should be noted that the binding of the thionamide drugs could be further stabilized by the peptide to be presented. Examples of specific protein–ligand interactions stabilizing the thionamide–HLA complexes of binding modes with relatively strong affinities are illustrated in [Fig. 4](#f4). In the molecular dynamics simulations, it was observed that the thionamide drugs drift and anchor to different sub-pockets of the peptide-binding groove with alternating hydrogen bonds and specific contacts. Cys67 of HLA-B*38:02 and HLA-B*38:01, which signifies the polymorphism of these two HLAs, is found to locate in the neighbourhood of the drug-binding sites. Therefore, it is anticipated that the sulfhydryl groups of the Cys67 and thionamide drugs may form disulfide bonds under the oxidative condition and thereby further stabilize the drug–HLA complex. However, we also note that one of the major binding poses of methimazole at the pocket B of HLA-B*38:01 would cause steric hindrance for the peptide binding. In pocket F of HLA-B*38:02 and HLA-B*38:01 ([Fig. 5](#f5)), and pocket 9 of HLA-DRA/DRB1*08:03 ([Fig. 4e,f](#f4)), one asparagine residue (Asn77 for HLA-B*38:02 and HLA-B*38:01, and Asnα69 for HLA-DRA/DRB1*08:03) was found to play a critical role in stabilizing the drug molecules at the edge of peptide-binding groove, by bonding with the sulfur atom or the nitrogen atom (sometimes both) of the drugs. Besides, Thr80 and Tyr123 in HLA-B*38:02, Tyr74 in HLA-B*38:01, Argα76, Tyrβ37 and Serβ57 in HLA-DRA/DRB1*08:03 provide additional interactions to stabilize the drugs in the F pocket. The peptide-binding groove of HLA-DRA/DRB1*08:03 are rich in aromatic residues and polar residues, which could also trap the thionamide drugs in the binding groove. It should be cautiously noted that here we only depict the most frequently occurring binding modes of many possible ones in the dynamics simulations. Since we did not known which peptides would participate in this process, the binding affinity and the binding modes of the drug in the HLA–peptide complex and the determinants of TiA response were not solved at this moment. Further experiments are needed.

### Figure 4. Specific protein–ligand interactions between thionamide drugs and the HLA proteins identified in this work.

![Figure 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1369/4506516/228963bf444c/ncomms8633-f4.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4506516_ncomms8633-f4.jpg)

(a) Methimazole-HLA-B*38:02. (b) Propylthiouracil-HLA-B*38:02. (c) Methimazole-HLA-B*38:01. (d) Propylthiouracil-HLA-B*38:01. (e) Methimazole-HLA-DRA/DRB1*08:03. (f) Propylthiouracil-HLA-DRA/DRB1*08:03 complexes. The predicted binding affinities are within the range between −4.48 and −6.36 kcal mol−1.

### Figure 5. Surface representation of HLA proteins and their sub-pockets.

![Figure 5](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1369/4506516/f7577abe8014/ncomms8633-f5.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4506516_ncomms8633-f5.jpg)

(a) HLA-B*38:02. The pocket positions are defined on the basis of the crystal structures of peptide-HLA-B*15:01 complex. The sub-pockets of HLA-B*38:01 are located at similar positions. (b) HLA-DRA/DRB1*08:03. The pocket positions are defined on the basis of the crystal structure of peptide-HLA-DR1 complex.

## Discussion

In this study, we performed concurrent direct HLA genotyping and GWAS in a cohort of 42 TiA cases and 1,208 GD controls in a two-stage study, and identified two HLA loci as major genetic determinants of ATD-induced agranulocytosis. This approach had the advantage of exploring the entire genome without bias and also to use two independent genotyping methods to evaluate the effects of classical HLA genes and other non-HLA genes at the HLA region.

For the HLA associations, we demonstrated that *HLA-B*38:02* and *HLA-DRB1*08:03* had independent major effects. Class I and class II HLA genes have different structures, cell-type distributions and functional roles in the immune system[24](#b24). Therefore, the genetic susceptibility from both classes for a phenotype is quite intriguing. Before our finding, Lucena *et al*.[25](#b25) already reported the association of class I and class II HLA genotypes with amoxicillin–clavulanate-induced liver injury. How both class I and class II HLA genes confer genetic susceptibility to the same drug adverse effect deserves further pathophysiological investigation[23](#b23).

Several additional analyses were performed intending to relate the genotypes of the TiA cases to other characteristics ([Supplementary Tables 6–8](#S1)). Some interesting signals were picked up, such as age at agranulocytosis ([Supplementary Table 6](#S1)) and ATDs administered ([Supplementary Table 8](#S1)). Because the sample size was small for each genotype, it may be premature to draw any conclusion from these.

In genetic studies, the HLA region is challenging to get clear-cut results because of high gene density[24](#b24), enormous polymorphism[26](#b26) and extended LD[24](#b24). Although we demonstrated independent associations between TiA and two classical HLA genes/alleles, namely *HLA-B*38:02* and *HLA-DRB1*08:03*, an association study could not exclude the possibility of nearby non-HLA gene(s) to be the genuine causative gene(s).

Previously, Tamai *et al*.[13](#b13) reported an association of TiA with *HLA-DRB1*08032* (the same allele as *HLA-DRB1*08:03*) in 1996. It is noteworthy that Japanese population has very low frequency (<0.5%) of *HLA-B*38:02* (ref. [27](#b27)); therefore, the results from Tamai *et al*.[13](#b13) further provide evidence supporting the independent effects of *HLA-DRB1*08:03* and *HLA-B*38:02*.

Among all genetic studies of drug-induced agranulocytosis, clozapine-induced agranulocytosis is the most studied[15](#b15),[16](#b16),[23](#b23). *HLA-B*38* was associated with clozapine-induced agranulocytosis in Jews, but not in other populations[15](#b15),[16](#b16),[23](#b23),[28](#b28). *HLA-B*38* alleles in Caucasians are mainly *HLA-B*38:01*, not *HLA-B*38:02* popular in Asians[27](#b27). The two alleles differ by one amino acid at the 80th position ([Supplementary Fig. 5](#S1)) located at peptide-binding groove pocket F[29](#b29). It is intriguing that two related HLA alleles were associated with agranulocytosis induced by two structurally unrelated drugs.

A couple of caveats need to be addressed. First, the total number of TiA cases in this report was not big. Second, for any single genetic association study, the possibility of population substructure cannot be completely excluded. Although this report is by far the largest TiA genetic study, with the two-stage study design and extensive quality-control steps supporting the validity of the results, it is still crucial to have independent replications in the same and/or other populations in the future.

The susceptibility HLA alleles we discovered (*HLA-B*38:02* and *HLA-DRB1*08:03*) have higher allele frequencies in Asians (0.036 and 0.048, respectively) than in Caucasians (0.004 and 0.002, respectively)[27](#b27). However, the incidence of TiA, according to available surveys, seems to be similar across the globe[4](#b4),[5](#b5),[6](#b6),[30](#b30),[31](#b31). Therefore, these two alleles cannot be the only reason for TiA susceptibility. There are several possible explanations supported by examples from other diseases. For example, the major susceptibility HLA alleles in different populations might be structurally similar alleles, such as different *HLA-DR4* alleles (*HLA-DRB1*04:01* and *HLA-DRB1*04:05*) for type 1 diabetes susceptibility[32](#b32),[33](#b33). Under this hypothesis, *HLA-B*38:01* and *HLA-DRB1*08:01* might be good TiA candidates in Caucasians. On the other hand, other reasonable hypotheses exist, including different major HLA genes/alleles (such as in GD[22](#b22) and in carbamazepine-induced Stevens–Johnson syndrome[23](#b23)), a spectrum of HLA genes/alleles sharing critical HLA amino-acid polymorphism (such as in rheumatoid arthritis[34](#b34)), or even non-genetic causes.

## Methods

### Study individuals

GD was diagnosed on the basis of the presence of clinical and biochemical hyperthyroidism together with either the presence of thyroid eye disease or a diffuse goitre and a significant titre of autoantibodies[22](#b22). Enrollment of GD individuals has been a continuous process at the National Taiwan University Hospital (NTUH) starting in August 2001. ATD-induced agranulocytosis, TiA, was initially identified according to the questionnaire from the GD participants at NTUH, but has later been actively searched for from the medical records at NTUH and another tertiary medical centre, Kaohsiung Chang Gung Memorial Hospital (KCGMH). Medical records were reviewed and the patients were excluded if they received other medications that could possibly explain their white blood count drops, such as chemotherapy agents, clozapine, dapsone, dipyrone, penicillin G, procainamide, sulfasalazine or ticlopidine, and so on. Although TiA was classically defined as absolute granulocyte counts <500 mm^−3^ in GD patients taking any of the three thionamides (methimazole, carbimazole and propylthiouracil), it should be noted that these cases were identified by clinical infection, not just by laboratory data. All the patients in this study presented with fever and sore throat with or without skin rash were hospitalized for management. Agranulocytosis developed approximately within 3 months after the start of ATDs. Available medical information of those 42 patients might not be sufficient to completely exclude concurrent viral infection. Causality assessment using widely adapted tools, such as the Naranjo algorithm[35](#b35),[36](#b36) and the World Health Organization Collaborating Centre for International Drug Monitoring, the Uppsala Monitoring Centre (WHO-UMC) causality categories[35](#b35), scored ‘probable' adverse drug reaction for all the 42 cases; the causality assessment did not reach the highest definite/certain category mostly because that re-challenge test was not performed and it was not possible to completely exclude all other possible causes.

For TiA ‘cases', we enrolled 21 patients at NTUH between 2001 and March 2012 and genotyped in our first-stage study ([Supplementary Table 1](#S1)). We used GD patients without agranulocytosis as the controls (GD controls). Our first-stage study comprised 497 unrelated GD patients and the 165 probands from unrelated GD families (only one GD individual from each family) enrolled between 2001 and 2006 ([Supplementary Table 1](#S1)), who were essentially the same individuals described in our previous publication[22](#b22). With initial encouraging positive signals in the first-stage study, we then actively recruited additional 21 TiA patients from NTUH and KCGMH for the second-stage study between March 2013 and March 2014 ([Supplementary Table 1](#S1)). Additional 546 unrelated GD patients enrolled between 2001 and 2011 who were not previously genotyped were then included as the GD controls in the second-stage study ([Supplementary Table 1](#S1)). Two of the 21 first-stage TiA cases and all 662 first-stage GD controls overlapped with the individuals reported in our previous paper[22](#b22). All the other TiA cases and the second-stage GD controls were new participants not previously reported. Power calculations were performed with various MAF and OR settings using QUANTO[37](#b37) ([http://biostats.usc.edu/Quanto.html](http://biostats.usc.edu/Quanto.html)). On the basis of the given sample size of the current GWAS study (TiA case number=42, GD control number=927), the power was reasonably adequate to detect major susceptibility gene(s) ([Supplementary Fig. 6](#S1)). The study was approved by the Institutional Review Board of both NTUH and KCGMH. Written informed consent was obtained from all participants.

### Direct HLA genotyping

There were four kinds of HLA genotyping methods used in this study including (1) Dynal RELI SSO typing kits (Dynal biotech Ltd, Bromborough, Wirral, UK, now part of Life Technologies, Carlsbad, CA, USA, [http://www.invitrogen.com/](http://www.invitrogen.com/)), (2) Gold SSP HLA-DPB1 High-resolution Kit (Invitrogen Corp., now part of Life Technologies [http://www.invitrogen.com/](http://www.invitrogen.com/)), (3) LABType SSO kit (One lambda Inc., Canoga Park, CA, USA, [http://www.onelambda.com/](http://www.onelambda.com/)) and (4) SeCore HLA Sequence-Based Typing Kit (Life Technologies Corporation, Brown Deer, WI, USA, [http://www.lifetechnologies.com/](http://www.lifetechnologies.com/)). The first three methods were applied to 2 out of our 21 first-stage TiA cases and all of our 662 first-stage GD controls. The fourth method (SeCore HLA sequence-based typing) was applied to all (including both first-stage and second-stage) of our 42 TiA cases and all of our 546-s-stage GD controls. In addition to the two first-stage TiA cases that were genotyped using more than one genotyping methods, we also randomly selected 24 first-stage GD controls to receive SeCore HLA sequence-based typing, in order to make sure that the genotyping results were consistent across all the genotyping methods.

Dynal RELI SSO genotyping was performed according to the manufacturer's instructions. Briefly, PCR using locus-specific primer sets were applied to amplify both exon 2 and exon 3 of class I (*HLA-A*, *-B* and *-C*) genes or exon 2 of class II *(-DQB1* and *-DRB1*) genes. Subsequently, PCR products were hybridized with sequence-specific oligonucleotide (SSO) probes previously fixed in a linear array on a nylon membrane (*HLA-A*: 48 probes, *-B*: 61 probes, *-C*: 37 probes, *-DQB1*: 41 probes and *-DRB1*: 60 probes). We then interpreted the genotypes using the Pattern Matching programme (Dynal biotech Ltd).

Owing to the lack of the *DPB1* genotyping kit in the Dynal RELI SSO system, we genotyped *HLA-DPB1* based on a sequence-specific primer (SSP) amplification method using ‘Gold SSP *HLA-DPB1* High resolution Kit' according to the manufacturer's protocol. Briefly, 48 PCR reactions were performed for each DNA sample. After PCR amplification and electrophoresis, the patterns of positive amplifications were used to interpret *HLA-DPB1* genotypes with the company's UniMatch software (Invitrogen Corp.).

LABType SSO kit-based genotyping was performed according to the manufacturer's protocol. Briefly, PCR products were hybridized with probes bound to fluorescently coded microspheres (*HLA-A*: 58/61/63 probes, *-B*: 100 probes, *-C*: 56 probes, *-DPB1*: 40 probes, *-DQB1*: 37 probes and *-DRB1*: 70 probes). Subsequently, a flow analyser was used to identify the fluorescent intensity on each microsphere (LABType visual software; One lambda Inc.) and assignment of the HLA genotype was obtained on the basis of the reaction pattern.

SeCore HLA sequence-based typing was performed as follows. *HLA-A*, *HLA-B*, *HLA-C* and *HLA-DPB1* polymorphism on exon 2, exon 3 and exon 4, *HLA-DRB1* polymorphism on exon 2, and *HLA-DQB1* polymorphism on exon 2 and exon 3 were determined using the SeCore HLA Sequence-Based Typing Kit (Life Technologies Corporation) according to the manufacturer's instructions. Briefly, after locus-specific PCR amplification, PCR products were treated with exonuclease I and shrimp alkaline phosphatase to remove excess dNTP and primers. The sequencing reaction for each exon was carried out using the Big Dye terminator v1.1 cycle sequencing kit in both directions. Sequencing was run on an automated ABI 3730 sequencer (Applied Biosystems, Foster City, CA, USA). The allele assignment is obtained by comparing the determined sequence with all combinations of known allele sequence in the IMGT/HLA database using the uTYPE sequence analysis software (Life Technologies Corporation). In case of heterozygous individuals with genotype ambiguities in which several genotype combinations produced same sequencing result, allele designation was assigned according to the most common alleles found in Taiwanese population. Two-field designation with the new naming format was used to assign HLA types.

Ambiguity, which refers to the same reaction patterns produced by several genotype combinations, was dealt with by assigning allele genotypes according to common alleles (allele frequency>0.01) found in Taiwanese population and southern Chinese populations as determined in the population studies of the 13th international histocompatibility workshop.

### Genome-wide SNP genotyping and quality control

Genome-wide genotyping analysis was conducted using the Affymetrix Axiom Genome-Wide CHB 1 Array Plate with 642,832 SNPs (CapitalBio Co., Ltd, Beijing, China). Genome-wide genotypes were called by the Axiom GT1 algorithm. Systemic quality control was then performed for individuals and SNPs. For each individual genotyped in the GWAS data set, quality-control filtering for SNPs was applied to remove SNPs that were missing in >1% of samples, were not autosomal, had MAF of <1% or showed significant deviation from Hardy–Weinberg equilibrium in controls (*P*<1 × 10^−5^). We also removed SNPs for which there was a significant difference in their genotype call rates in case and control data (*P*<1 × 10^−6^). For sample filtering, arrays with generated genotypes for <95% of the loci were excluded. Heterozygosity rates were calculated, deviations of more than 6 s.d. from the mean were excluded. Heterozygosity of X-chromosome SNPs was used to verify the gender of the samples. There was no gender mismatch in our samples. The PLINK version 1.07 software ([http://pngu.mgh.harvard.edu/~purcell/plink/](http://pngu.mgh.harvard.edu/~purcell/plink/)) was used to identify samples with genetic relatedness indicating that they were from the same individual (or monozygotic twins) or from first-, second- or third-degree relatives. These determinations were based on evidence for cryptic relatedness from identity-by-descent status (pi-hat cutoff of 0.125). After filtering, we retained 522,980 autosomal SNPs in 42 TiA cases and 927 GD controls for analysis.

### Statistical analysis

We conducted MDS analysis using the PLINK version 1.07 software and principal components analysis using GCTA[38](#b38) ([http://www.complextraitgenomics.com/software/gcta/](http://www.complextraitgenomics.com/software/gcta/)) to capture population substructure among 969 subjects. For MDS and principal components analysis, all 969 samples (42 TiA cases and 927 GD controls) were analysed together with and without the 281 Asian reference samples from the International HapMap Project ([http://hapmap.ncbi.nlm.nih.gov/index.html.en/](http://hapmap.ncbi.nlm.nih.gov/index.html.en/)), and no outliers were identified ([Supplementary Figs 3 and 4](#S1)). The value of genomic inflation factor (*λ*) was 1.01, which indicated that population stratification effect was negligible in our study sample. After quality control, all GWA analyses were carried out by comparing allele/genotype frequencies between TiA cases and GD controls using five single-point methods: genotype, allele-type and Cochran–Armitage trend test along with tests considering additive effect and effective allele carrier status. The genome-wide significance threshold *P* value was set at 9.56 × 10^−8^ after Bonferroni correction for the number of SNPs (522,980). Given the excess controls in our samples, paired analyses[39](#b39) were carried out by randomly matching 10 GD controls for each TiA case with respect to age and gender ([Supplementary Table 9](#S1)). Adjustments for potential genetic heterogeneity were applied by incorporating the first two or the first ten dimensions/principal components identified by MDS and GCTA[38](#b38) ([Supplementary Tables 10 and 11](#S1)). The association signals at the *HLA-B* and *HLA-DRB1* regions still remained. The multivariate stepwise logistic regression analysis was performed in a stepwise (forward–backward) manner and the Cochran–Mantel–Haenszel test using SAS/STAT version 9.3 ([http://www.sas.com/](http://www.sas.com/)). The entry and removal criteria with *P* values of 0.05 based on the likelihood ratio test were used for the stepwise regression analysis. To evaluate the ability of the logistic regression model for phenotype prediction, a ROC analysis was conducted. A ROC curve to discriminate TiA cases from GD controls was generated using the combined data set, and the AUC was calculated. The LD was assessed using Haploview version 4.2 ([http://www.broadinstitute.org/scientific-community/science/programs/medical-and-population-genetics/haploview/haploview](http://www.broadinstitute.org/scientific-community/science/programs/medical-and-population-genetics/haploview/haploview)). The Manhattan and quantile–quantile plots were created with the R package ([http://www.r-project.org/](http://www.r-project.org/)).

### HLA imputation and association analyses

We used HLA imputation method to independently identify HLA regions from the GWAS study to validate our direct HLA typing. We imputed classical HLA alleles and corresponding amino-acid polymorphisms using Asian reference panel with high-density SNP genotypes and four-digit classical HLA allele genotypes (530 Asian individuals) using the SNP2HLA software[34](#b34),[40](#b40),[41](#b41) ([https://www.broadinstitute.org/mpg/snp2hla/](https://www.broadinstitute.org/mpg/snp2hla/)). We extracted SNP genotypes located in the broad MHC region (chromosome 6: 29–35 Mb) to impute two-digit classical alleles, four-digit classical alleles and amino-acid polymorphisms of the eight class I and class II HLA genes (*HLA-A*, *HLA-B*, *HLA-C*, *HLA-DRB1*, *HLA-DQA1*, *HLA-DQB1*, *HLA-DPA1* and *HLA-DPB1*). PLINK was used to perform the association analyses for the HLA-imputed analysis.

### 3D model construction

The 3D structures of HLA-B*38:02 and HLA-B*38:01 were modelled by I-TASSER 4.2 (ref. [42](#b42)) using the structures with PDB accession IDs of [2BCK](https://www.rcsb.org/structure/2BCK), [3AM8](https://www.rcsb.org/structure/3AM8), [1S7Q](https://www.rcsb.org/structure/1S7Q), [4NT6](https://www.rcsb.org/structure/4NT6) and [1I4F](https://www.rcsb.org/structure/1I4F) as the structural templates. The HLA-DRA/DRB1*08:03 structure was constructed by MODELLER 9v9 (ref. [43](#b43)) based on the structure of HLA-DR1 (PDB accession ID of [1AQD](https://www.rcsb.org/structure/1AQD)). The protonation states of the titratable residues of the proteins were assigned by PROPKA[44](#b44) at pH 7.4 and the hydrogen atoms were added by PDB2PQR 1.9 (ref. [45](#b45)). The initial conformations of the thionamide drug molecules, methimazole and propylthiouracil, were derived from Protein Data Bank with ligand IDs of MMZ and 3CJ, respectively. MarvinSketch 5.1.3 was utilized for the prediction of protonation states at pH 7.4 and the addition of hydrogen atoms to the drug molecules. Quantum chemical calculations using Gaussian03 were applied at the Hartree–Fock level with the 6-31G* basis set[46](#b46), and the RESP (Restrained ElectroStatic Potential)[47](#b47) scheme was employed to determine the atomic charges. Molecular dockings were performed using AutoDock4 (ref. [48](#b48)), with the recalibrated scoring function (AutoDock4^RRP^)[49](#b49). A grid box size of 18.75 × 43.5 × 18.75 Å^3^ was set to encompass the whole binding groove of the protein, and the peptide was not included to allow the drug molecules to explore all possible binding sites. Our docking results reveal that the F pocket is the most favourable pocket for drug binding ([Fig. 5](#f5)), with the estimated binding affinities in the range between −4.48 and −6.36 kcal mol^−1^. To further confirm the binding poses of the drug molecules, 2 ns molecular dynamics simulations were conducted for each complex system by Amberer with the AMBER parm99SB (ref. [50](#b50)) for proteins and the general amber force field[51](#b51) for drugs. The molecular graphics figures were made by PyMOL 1.3 ([Fig. 4](#f4)) and UCSF Chimera 1.6.1 ([Fig. 5](#f5))[52](#b52).

## Additional information

**How to cite this article:** Chen, P-L. *et al*. Genetic determinants of antithyroid drug-induced agranulocytosis by human leukocyte antigen genotyping and genome-wide association study. *Nat. Commun.* 6:7633 doi: 10.1038/ncomms8633 (2015).

## Supplementary Material

## Acknowledgments

We thank D. Valle and C.-H. Chen for helpful discussion and M. Liao and C.-C. Lin for technical support. We thank all the participants who provided DNA and clinical information necessary for this study. This work was supported by the following grants: 100-2314-B-002-118-MY3, 102-2314-B-001-003-MY2, 102-2325-B-002-034-, 102-2314-B-002-020-MY2 and NHRI-EX103-100232PI. We thank the National Center for Genomic Medicine of the National Core Facility Program for Biotechnology, Ministry of Science and Technology, Taiwan, for the technical/bioinformatics support. We thank the Translational Resource Center for Genomic Medicine of National Research Program for Biopharmaceuticals (NRPB), Taiwan. We are grateful to the National Center for High-performance Computing (NCHC), Taiwan, for computer time and facilities.

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Weetman A. P. Graves' disease. N. Engl. J. Med. 343, 1236–1248 (2000).  [DOI](https://doi.org/10.1056/NEJM200010263431707) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11071676/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N.%20Engl.%20J.%20Med.&title=Graves'%20disease&author=A.%20P.%20Weetman&volume=343&publication_year=2000&pages=1236-1248&pmid=11071676&doi=10.1056/NEJM200010263431707&)

2. Tunbridge W. M. et al. The spectrum of thyroid disease in a community: the Whickham survey. Clin. Endocrinol. (Oxf.) 7, 481–493 (1977).  [DOI](https://doi.org/10.1111/j.1365-2265.1977.tb01340.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/598014/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Endocrinol.%20(Oxf.)&title=The%20spectrum%20of%20thyroid%20disease%20in%20a%20community:%20the%20Whickham%20survey&author=W.%20M.%20Tunbridge&volume=7&publication_year=1977&pages=481-493&pmid=598014&doi=10.1111/j.1365-2265.1977.tb01340.x&)

3. Jacobson D. L., Gange S. J., Rose N. R. & Graham N. M. Epidemiology and estimated population burden of selected autoimmune diseases in the United States. Clin. Immunol. Immunopathol. 84, 223–243 (1997).  [DOI](https://doi.org/10.1006/clin.1997.4412) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9281381/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Immunol.%20Immunopathol.&title=Epidemiology%20and%20estimated%20population%20burden%20of%20selected%20autoimmune%20diseases%20in%20the%20United%20States&author=D.%20L.%20Jacobson&author=S.%20J.%20Gange&author=N.%20R.%20Rose&author=N.%20M.%20Graham&volume=84&publication_year=1997&pages=223-243&pmid=9281381&doi=10.1006/clin.1997.4412&)

4. Cooper D. S. Antithyroid drugs. N. Engl. J. Med. 352, 905–917 (2005).  [DOI](https://doi.org/10.1056/NEJMra042972) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15745981/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N.%20Engl.%20J.%20Med.&title=Antithyroid%20drugs&author=D.%20S.%20Cooper&volume=352&publication_year=2005&pages=905-917&pmid=15745981&doi=10.1056/NEJMra042972&)

5. Tajiri J. & Noguchi S. Antithyroid drug-induced agranulocytosis: special reference to normal white blood cell count agranulocytosis. Thyroid 14, 459–462 (2004).  [DOI](https://doi.org/10.1089/105072504323150787) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15242574/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Thyroid&title=Antithyroid%20drug-induced%20agranulocytosis:%20special%20reference%20to%20normal%20white%20blood%20cell%20count%20agranulocytosis&author=J.%20Tajiri&author=S.%20Noguchi&volume=14&publication_year=2004&pages=459-462&pmid=15242574&doi=10.1089/105072504323150787&)

6. Nakamura H., Miyauchi A., Miyawaki N. & Imagawa J. Analysis of 754 cases of antithyroid drug-induced agranulocytosis over 30 years in Japan. J. Clin. Endocrinol. Metab. 98, 4776–4783 (2013).  [DOI](https://doi.org/10.1210/jc.2013-2569) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24057289/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Endocrinol.%20Metab.&title=Analysis%20of%20754%20cases%20of%20antithyroid%20drug-induced%20agranulocytosis%20over%2030%20years%20in%20Japan&author=H.%20Nakamura&author=A.%20Miyauchi&author=N.%20Miyawaki&author=J.%20Imagawa&volume=98&publication_year=2013&pages=4776-4783&pmid=24057289&doi=10.1210/jc.2013-2569&)

7. Andersohn F., Konzen C. & Garbe E. Systematic review: agranulocytosis induced by nonchemotherapy drugs. Ann. Intern. Med. 146, 657–665 (2007).  [DOI](https://doi.org/10.7326/0003-4819-146-9-200705010-00009) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17470834/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann.%20Intern.%20Med.&title=Systematic%20review:%20agranulocytosis%20induced%20by%20nonchemotherapy%20drugs&author=F.%20Andersohn&author=C.%20Konzen&author=E.%20Garbe&volume=146&publication_year=2007&pages=657-665&pmid=17470834&doi=10.7326/0003-4819-146-9-200705010-00009&)

8. Huber M. et al. Drug-induced agranulocytosis in the Berlin case-control surveillance study. Eur. J. Clin. Pharmacol. 70, 339–345 (2014).  [DOI](https://doi.org/10.1007/s00228-013-1618-1) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24297345/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur.%20J.%20Clin.%20Pharmacol.&title=Drug-induced%20agranulocytosis%20in%20the%20Berlin%20case-control%20surveillance%20study&author=M.%20Huber&volume=70&publication_year=2014&pages=339-345&pmid=24297345&doi=10.1007/s00228-013-1618-1&)

9. Edwards I. R. & Aronson J. K. Adverse drug reactions: definitions, diagnosis, and management. Lancet 356, 1255–1259 (2000).  [DOI](https://doi.org/10.1016/S0140-6736(00)02799-9) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11072960/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Adverse%20drug%20reactions:%20definitions,%20diagnosis,%20and%20management&author=I.%20R.%20Edwards&author=J.%20K.%20Aronson&volume=356&publication_year=2000&pages=1255-1259&pmid=11072960&doi=10.1016/S0140-6736(00)02799-9&)

10. Aronson J. K. & Ferner R. E. Joining the DoTS: new approach to classifying adverse drug reactions. Br. Med. J. 327, 1222–1225 (2003).  [DOI](https://doi.org/10.1136/bmj.327.7425.1222) | [PMC free article](/articles/PMC274067/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14630763/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br.%20Med.%20J.&title=Joining%20the%20DoTS:%20new%20approach%20to%20classifying%20adverse%20drug%20reactions&author=J.%20K.%20Aronson&author=R.%20E.%20Ferner&volume=327&publication_year=2003&pages=1222-1225&pmid=14630763&doi=10.1136/bmj.327.7425.1222&)

11. Wang L., McLeod H. L. & Weinshilboum R. M. Genomics and drug response. N. Engl. J. Med. 364, 1144–1153 (2011).  [DOI](https://doi.org/10.1056/NEJMra1010600) | [PMC free article](/articles/PMC3184612/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21428770/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N.%20Engl.%20J.%20Med.&title=Genomics%20and%20drug%20response&author=L.%20Wang&author=H.%20L.%20McLeod&author=R.%20M.%20Weinshilboum&volume=364&publication_year=2011&pages=1144-1153&pmid=21428770&doi=10.1056/NEJMra1010600&)

12. Yunis J. J. et al. HLA associations in clozapine-induced agranulocytosis. Blood 86, 1177–1183 (1995).  [PubMed](https://pubmed.ncbi.nlm.nih.gov/7620171/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=HLA%20associations%20in%20clozapine-induced%20agranulocytosis&author=J.%20J.%20Yunis&volume=86&publication_year=1995&pages=1177-1183&pmid=7620171&)

13. Tamai H. et al. Association between the DRB1*08032 histocompatibility antigen and methimazole-induced agranulocytosis in Japanese patients with Graves disease. Ann. Intern. Med. 124, 490–494 (1996).  [DOI](https://doi.org/10.7326/0003-4819-124-5-199603010-00005) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8602707/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann.%20Intern.%20Med.&title=Association%20between%20the%20DRB1*08032%20histocompatibility%20antigen%20and%20methimazole-induced%20agranulocytosis%20in%20Japanese%20patients%20with%20Graves%20disease&author=H.%20Tamai&volume=124&publication_year=1996&pages=490-494&pmid=8602707&doi=10.7326/0003-4819-124-5-199603010-00005&)

14. Valevski A. et al. HLA-B38 and clozapine-induced agranulocytosis in Israeli Jewish schizophrenic patients. Eur. J. Immunogenet. 25, 11–13 (1998).  [DOI](https://doi.org/10.1046/j.1365-2370.1998.00091.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9587740/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur.%20J.%20Immunogenet.&title=HLA-B38%20and%20clozapine-induced%20agranulocytosis%20in%20Israeli%20Jewish%20schizophrenic%20patients&author=A.%20Valevski&volume=25&publication_year=1998&pages=11-13&pmid=9587740&doi=10.1046/j.1365-2370.1998.00091.x&)

15. Opgen-Rhein C. & Dettling M. Clozapine-induced agranulocytosis and its genetic determinants. Pharmacogenomics 9, 1101–1111 (2008).  [DOI](https://doi.org/10.2217/14622416.9.8.1101) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18681784/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Clozapine-induced%20agranulocytosis%20and%20its%20genetic%20determinants&author=C.%20Opgen-Rhein&author=M.%20Dettling&volume=9&publication_year=2008&pages=1101-1111&pmid=18681784&doi=10.2217/14622416.9.8.1101&)

16. Zhang J. P. & Malhotra A. K. Pharmacogenetics of antipsychotics: recent progress and methodological issues. Expert Opin. Drug Metab. Toxicol. 9, 183–191 (2013).  [DOI](https://doi.org/10.1517/17425255.2013.736964) | [PMC free article](/articles/PMC3547146/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23199282/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Expert%20Opin.%20Drug%20Metab.%20Toxicol.&title=Pharmacogenetics%20of%20antipsychotics:%20recent%20progress%20and%20methodological%20issues&author=J.%20P.%20Zhang&author=A.%20K.%20Malhotra&volume=9&publication_year=2013&pages=183-191&pmid=23199282&doi=10.1517/17425255.2013.736964&)

17. Chung W. H. et al. Medical genetics: a marker for Stevens-Johnson syndrome. Nature 428, 486 (2004).  [DOI](https://doi.org/10.1038/428486a) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15057820/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Medical%20genetics:%20a%20marker%20for%20Stevens-Johnson%20syndrome&author=W.%20H.%20Chung&volume=428&publication_year=2004&pages=486&pmid=15057820&doi=10.1038/428486a&)

18. Ozeki T. et al. Genome-wide association study identifies HLA-A*3101 allele as a genetic risk factor for carbamazepine-induced cutaneous adverse drug reactions in Japanese population. Hum. Mol. Genet. 20, 1034–1041 (2011).  [DOI](https://doi.org/10.1093/hmg/ddq537) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21149285/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum.%20Mol.%20Genet.&title=Genome-wide%20association%20study%20identifies%20HLA-A*3101%20allele%20as%20a%20genetic%20risk%20factor%20for%20carbamazepine-induced%20cutaneous%20adverse%20drug%20reactions%20in%20Japanese%20population&author=T.%20Ozeki&volume=20&publication_year=2011&pages=1034-1041&pmid=21149285&doi=10.1093/hmg/ddq537&)

19. Hetherington S. et al. Genetic variations in HLA-B region and hypersensitivity reactions to abacavir. Lancet 359, 1121–1122 (2002).  [DOI](https://doi.org/10.1016/S0140-6736(02)08158-8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11943262/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Genetic%20variations%20in%20HLA-B%20region%20and%20hypersensitivity%20reactions%20to%20abacavir&author=S.%20Hetherington&volume=359&publication_year=2002&pages=1121-1122&pmid=11943262&doi=10.1016/S0140-6736(02)08158-8&)

20. Mallal S. et al. Association between presence of HLA-B*5701, HLA-DR7, and HLA-DQ3 and hypersensitivity to HIV-1 reverse-transcriptase inhibitor abacavir. Lancet 359, 727–732 (2002).  [DOI](https://doi.org/10.1016/s0140-6736(02)07873-x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11888582/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Association%20between%20presence%20of%20HLA-B*5701,%20HLA-DR7,%20and%20HLA-DQ3%20and%20hypersensitivity%20to%20HIV-1%20reverse-transcriptase%20inhibitor%20abacavir&author=S.%20Mallal&volume=359&publication_year=2002&pages=727-732&pmid=11888582&doi=10.1016/s0140-6736(02)07873-x&)

21. Spraggs C. F. et al. HLA-DQA1*02:01 is a major risk factor for lapatinib-induced hepatotoxicity in women with advanced breast cancer. J. Clin. Oncol. 29, 667–673 (2011).  [DOI](https://doi.org/10.1200/JCO.2010.31.3197) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21245432/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Oncol.&title=HLA-DQA1*02:01%20is%20a%20major%20risk%20factor%20for%20lapatinib-induced%20hepatotoxicity%20in%20women%20with%20advanced%20breast%20cancer&author=C.%20F.%20Spraggs&volume=29&publication_year=2011&pages=667-673&pmid=21245432&doi=10.1200/JCO.2010.31.3197&)

22. Chen P. L. et al. Comprehensive genotyping in two homogeneous Graves' disease samples reveals major and novel HLA association alleles. PLoS ONE 6, e16635 (2011).  [DOI](https://doi.org/10.1371/journal.pone.0016635) | [PMC free article](/articles/PMC3030609/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21307958/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&title=Comprehensive%20genotyping%20in%20two%20homogeneous%20Graves'%20disease%20samples%20reveals%20major%20and%20novel%20HLA%20association%20alleles&author=P.%20L.%20Chen&volume=6&publication_year=2011&pages=e16635&pmid=21307958&doi=10.1371/journal.pone.0016635&)

23. Pavlos R., Mallal S. & Phillips E. HLA and pharmacogenetics of drug hypersensitivity. Pharmacogenomics 13, 1285–1306 (2012).  [DOI](https://doi.org/10.2217/pgs.12.108) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22920398/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=HLA%20and%20pharmacogenetics%20of%20drug%20hypersensitivity&author=R.%20Pavlos&author=S.%20Mallal&author=E.%20Phillips&volume=13&publication_year=2012&pages=1285-1306&pmid=22920398&doi=10.2217/pgs.12.108&)

24. Klein J. & Sato A. The HLA system. First of two parts. N. Engl. J. Med. 343, 702–709 (2000).  [DOI](https://doi.org/10.1056/NEJM200009073431006) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10974135/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N.%20Engl.%20J.%20Med.&title=The%20HLA%20system.%20First%20of%20two%20parts&author=J.%20Klein&author=A.%20Sato&volume=343&publication_year=2000&pages=702-709&pmid=10974135&doi=10.1056/NEJM200009073431006&)

25. Lucena M. I. et al. Susceptibility to amoxicillin-clavulanate-induced liver injury is influenced by multiple HLA class I and II alleles. Gastroenterology 141, 338–347 (2011).  [DOI](https://doi.org/10.1053/j.gastro.2011.04.001) | [PMC free article](/articles/PMC3129430/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21570397/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gastroenterology&title=Susceptibility%20to%20amoxicillin-clavulanate-induced%20liver%20injury%20is%20influenced%20by%20multiple%20HLA%20class%20I%20and%20II%20alleles&author=M.%20I.%20Lucena&volume=141&publication_year=2011&pages=338-347&pmid=21570397&doi=10.1053/j.gastro.2011.04.001&)

26. Erlich H. A., Opelz G. & Hansen J. HLA DNA typing and transplantation. Immunity 14, 347–356 (2001).  [DOI](https://doi.org/10.1016/s1074-7613(01)00115-7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11336680/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Immunity&title=HLA%20DNA%20typing%20and%20transplantation&author=H.%20A.%20Erlich&author=G.%20Opelz&author=J.%20Hansen&volume=14&publication_year=2001&pages=347-356&pmid=11336680&doi=10.1016/s1074-7613(01)00115-7&)

27. Gonzalez-Galarza F. F., Christmas S., Middleton D. & Jones A. R. Allele frequency net: a database and online repository for immune gene frequencies in worldwide populations. Nucleic Acids Res. 39, D913–D919 (2011).  [DOI](https://doi.org/10.1093/nar/gkq1128) | [PMC free article](/articles/PMC3013710/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21062830/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&title=Allele%20frequency%20net:%20a%20database%20and%20online%20repository%20for%20immune%20gene%20frequencies%20in%20worldwide%20populations&author=F.%20F.%20Gonzalez-Galarza&author=S.%20Christmas&author=D.%20Middleton&author=A.%20R.%20Jones&volume=39&publication_year=2011&pages=D913-D919&pmid=21062830&doi=10.1093/nar/gkq1128&)

28. Lieberman J. A. et al. HLA-B38, DR4, DQw3 and clozapine-induced agranulocytosis in Jewish patients with schizophrenia. Arch. Gen. Psychiatry 47, 945–948 (1990).  [DOI](https://doi.org/10.1001/archpsyc.1990.01810220061007) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/2222133/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arch.%20Gen.%20Psychiatry&title=HLA-B38,%20DR4,%20DQw3%20and%20clozapine-induced%20agranulocytosis%20in%20Jewish%20patients%20with%20schizophrenia&author=J.%20A.%20Lieberman&volume=47&publication_year=1990&pages=945-948&pmid=2222133&doi=10.1001/archpsyc.1990.01810220061007&)

29. DeLuca D. S. & Blasczyk R. HistoCheck. Evaluating structural and functional MHC similarities. Methods Mol. Biol. 409, 395–405 (2007).  [DOI](https://doi.org/10.1007/978-1-60327-118-9_30) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18450018/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Methods%20Mol.%20Biol.&title=HistoCheck.%20Evaluating%20structural%20and%20functional%20MHC%20similarities&author=D.%20S.%20DeLuca&author=R.%20Blasczyk&volume=409&publication_year=2007&pages=395-405&pmid=18450018&doi=10.1007/978-1-60327-118-9_30&)

30. Huang C. H., Li K. L., Wu J. H., Wang P. N. & Juang J. H. Antithyroid drug-induced agranulocytosis: report of 13 cases. Chang Gung Med. J. 30, 242–248 (2007).  [PubMed](https://pubmed.ncbi.nlm.nih.gov/17760275/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Chang%20Gung%20Med.%20J.&title=Antithyroid%20drug-induced%20agranulocytosis:%20report%20of%2013%20cases&author=C.%20H.%20Huang&author=K.%20L.%20Li&author=J.%20H.%20Wu&author=P.%20N.%20Wang&author=J.%20H.%20Juang&volume=30&publication_year=2007&pages=242-248&pmid=17760275&)

31. Sheng W. H. et al. Antithyroid-drug-induced agranulocytosis complicated by life-threatening infections. QJM 92, 455–461 (1999).  [DOI](https://doi.org/10.1093/qjmed/92.8.455) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10627862/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=QJM&title=Antithyroid-drug-induced%20agranulocytosis%20complicated%20by%20life-threatening%20infections&author=W.%20H.%20Sheng&volume=92&publication_year=1999&pages=455-461&pmid=10627862&doi=10.1093/qjmed/92.8.455&)

32. Noble J. A. & Valdes A. M. Genetics of the HLA region in the prediction of type 1 diabetes. Curr. Diab. Rep. 11, 533–542 (2011).  [DOI](https://doi.org/10.1007/s11892-011-0223-x) | [PMC free article](/articles/PMC3233362/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21912932/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr.%20Diab.%20Rep.&title=Genetics%20of%20the%20HLA%20region%20in%20the%20prediction%20of%20type%201%20diabetes&author=J.%20A.%20Noble&author=A.%20M.%20Valdes&volume=11&publication_year=2011&pages=533-542&pmid=21912932&doi=10.1007/s11892-011-0223-x&)

33. Pociot F. & McDermott M. F. Genetics of type 1 diabetes mellitus. Genes Immun. 3, 235–249 (2002).  [DOI](https://doi.org/10.1038/sj.gene.6363875) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12140742/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genes%20Immun.&title=Genetics%20of%20type%201%20diabetes%20mellitus&author=F.%20Pociot&author=M.%20F.%20McDermott&volume=3&publication_year=2002&pages=235-249&pmid=12140742&doi=10.1038/sj.gene.6363875&)

34. Okada Y. et al. Risk for ACPA-positive rheumatoid arthritis is driven by shared HLA amino acid polymorphisms in Asian and European populations. Hum. Mol. Genet. 23, 6916–6926 (2014).  [DOI](https://doi.org/10.1093/hmg/ddu387) | [PMC free article](/articles/PMC4245039/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25070946/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum.%20Mol.%20Genet.&title=Risk%20for%20ACPA-positive%20rheumatoid%20arthritis%20is%20driven%20by%20shared%20HLA%20amino%20acid%20polymorphisms%20in%20Asian%20and%20European%20populations&author=Y.%20Okada&volume=23&publication_year=2014&pages=6916-6926&pmid=25070946&doi=10.1093/hmg/ddu387&)

35. Zaki S. A. Adverse drug reaction and causality assessment scales. Lung India 28, 152–153 (2011).  [DOI](https://doi.org/10.4103/0970-2113.80343) | [PMC free article](/articles/PMC3109846/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21712934/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lung%20India&title=Adverse%20drug%20reaction%20and%20causality%20assessment%20scales&author=S.%20A.%20Zaki&volume=28&publication_year=2011&pages=152-153&pmid=21712934&doi=10.4103/0970-2113.80343&)

36. Naranjo C. A. et al. A method for estimating the probability of adverse drug reactions. Clin. Pharmacol. Ther. 30, 239–245 (1981).  [DOI](https://doi.org/10.1038/clpt.1981.154) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7249508/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=A%20method%20for%20estimating%20the%20probability%20of%20adverse%20drug%20reactions&author=C.%20A.%20Naranjo&volume=30&publication_year=1981&pages=239-245&pmid=7249508&doi=10.1038/clpt.1981.154&)

37. Gauderman W. J. Sample size requirements for association studies of gene-gene interaction. Am. J. Epidemiol. 155, 478–484 (2002).  [DOI](https://doi.org/10.1093/aje/155.5.478) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11867360/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Epidemiol.&title=Sample%20size%20requirements%20for%20association%20studies%20of%20gene-gene%20interaction&author=W.%20J.%20Gauderman&volume=155&publication_year=2002&pages=478-484&pmid=11867360&doi=10.1093/aje/155.5.478&)

38. Yang J., Lee S. H., Goddard M. E. & Visscher P. M. GCTA: a tool for genome-wide complex trait analysis. Am. J. Hum. Genet. 88, 76–82 (2011).  [DOI](https://doi.org/10.1016/j.ajhg.2010.11.011) | [PMC free article](/articles/PMC3014363/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21167468/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Hum.%20Genet.&title=GCTA:%20a%20tool%20for%20genome-wide%20complex%20trait%20analysis&author=J.%20Yang&author=S.%20H.%20Lee&author=M.%20E.%20Goddard&author=P.%20M.%20Visscher&volume=88&publication_year=2011&pages=76-82&pmid=21167468&doi=10.1016/j.ajhg.2010.11.011&)

39. Chu X. et al. A genome-wide association study identifies two new risk loci for Graves' disease. Nat. Genet. 43, 897–901 (2011).  [DOI](https://doi.org/10.1038/ng.898) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21841780/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet.&title=A%20genome-wide%20association%20study%20identifies%20two%20new%20risk%20loci%20for%20Graves'%20disease&author=X.%20Chu&volume=43&publication_year=2011&pages=897-901&pmid=21841780&doi=10.1038/ng.898&)

40. Jia X. et al. Imputing amino acid polymorphisms in human leukocyte antigens. PLoS ONE 8, e64683 (2013).  [DOI](https://doi.org/10.1371/journal.pone.0064683) | [PMC free article](/articles/PMC3675122/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23762245/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&title=Imputing%20amino%20acid%20polymorphisms%20in%20human%20leukocyte%20antigens&author=X.%20Jia&volume=8&publication_year=2013&pages=e64683&pmid=23762245&doi=10.1371/journal.pone.0064683&)

41. Pillai N. E. et al. Predicting HLA alleles from high-resolution SNP data in three Southeast Asian populations. Hum. Mol. Genet. 23, 4443–4451 (2014).  [DOI](https://doi.org/10.1093/hmg/ddu149) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24698974/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum.%20Mol.%20Genet.&title=Predicting%20HLA%20alleles%20from%20high-resolution%20SNP%20data%20in%20three%20Southeast%20Asian%20populations&author=N.%20E.%20Pillai&volume=23&publication_year=2014&pages=4443-4451&pmid=24698974&doi=10.1093/hmg/ddu149&)

42. Roy A., Kucukural A. & Zhang Y. I-TASSER: a unified platform for automated protein structure and function prediction. Nat. Protoc. 5, 725–738 (2010).  [DOI](https://doi.org/10.1038/nprot.2010.5) | [PMC free article](/articles/PMC2849174/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20360767/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Protoc.&title=I-TASSER:%20a%20unified%20platform%20for%20automated%20protein%20structure%20and%20function%20prediction&author=A.%20Roy&author=A.%20Kucukural&author=Y.%20Zhang&volume=5&publication_year=2010&pages=725-738&pmid=20360767&doi=10.1038/nprot.2010.5&)

43. Eswar N. et al. Comparative protein structure modeling using Modeller. Curr. Protoc. Bioinformatics Chapter 5, Unit 5–Unit 6 (2006).  [DOI](https://doi.org/10.1002/0471250953.bi0506s15) | [PMC free article](/articles/PMC4186674/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18428767/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr.%20Protoc.%20Bioinformatics&title=Comparative%20protein%20structure%20modeling%20using%20Modeller&author=N.%20Eswar&volume=Chapter%205&publication_year=2006&pages=Unit%205-Unit%206&pmid=18428767&doi=10.1002/0471250953.bi0506s15&)

44. Li H., Robertson A. D. & Jensen J. H. Very fast empirical prediction and rationalization of protein pK(a) values. Proteins 61, 704–721 (2005).  [DOI](https://doi.org/10.1002/prot.20660) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16231289/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proteins&title=Very%20fast%20empirical%20prediction%20and%20rationalization%20of%20protein%20pK(a)%20values&author=H.%20Li&author=A.%20D.%20Robertson&author=J.%20H.%20Jensen&volume=61&publication_year=2005&pages=704-721&pmid=16231289&doi=10.1002/prot.20660&)

45. Dolinsky T. J., Nielsen J. E., McCammon J. A. & Baker N. A. PDB2PQR: an automated pipeline for the setup of Poisson-Boltzmann electrostatics calculations. Nucleic Acids Res. 32, W665–W667 (2004).  [DOI](https://doi.org/10.1093/nar/gkh381) | [PMC free article](/articles/PMC441519/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15215472/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&title=PDB2PQR:%20an%20automated%20pipeline%20for%20the%20setup%20of%20Poisson-Boltzmann%20electrostatics%20calculations&author=T.%20J.%20Dolinsky&author=J.%20E.%20Nielsen&author=J.%20A.%20McCammon&author=N.%20A.%20Baker&volume=32&publication_year=2004&pages=W665-W667&pmid=15215472&doi=10.1093/nar/gkh381&)

46. Hehre W. J., Ditchfie R. & Pople J. A. Self-consistent molecular-orbital methods.12. Further extensions of Gaussian-type basis sets for use in molecular-orbital studies of organic-molecules. J. Chem. Phys. 56, 2257–2261 (1972).  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Chem.%20Phys.&title=Self-consistent%20molecular-orbital%20methods.12.%20Further%20extensions%20of%20Gaussian-type%20basis%20sets%20for%20use%20in%20molecular-orbital%20studies%20of%20organic-molecules&author=W.%20J.%20Hehre&author=R.%20Ditchfie&author=J.%20A.%20Pople&volume=56&publication_year=1972&pages=2257-2261&)

47. Bayly C. I., Cieplak P., Cornell W. D. & Kollman P. A. A well-behaved electrostatic potential based method using charge restraints for deriving atomic charges - the resp model. J. Phys. Chem. 97, 10269–10280 (1993).  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Phys.%20Chem.&title=A%20well-behaved%20electrostatic%20potential%20based%20method%20using%20charge%20restraints%20for%20deriving%20atomic%20charges%20-%20the%20resp%20model&author=C.%20I.%20Bayly&author=P.%20Cieplak&author=W.%20D.%20Cornell&author=P.%20A.%20Kollman&volume=97&publication_year=1993&pages=10269-10280&)

48. Morris G. M. et al. AutoDock4 and AutoDockTools4: automated docking with selective receptor flexibility. J. Comput. Chem. 30, 2785–2791 (2009).  [DOI](https://doi.org/10.1002/jcc.21256) | [PMC free article](/articles/PMC2760638/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19399780/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Comput.%20Chem.&title=AutoDock4%20and%20AutoDockTools4:%20automated%20docking%20with%20selective%20receptor%20flexibility&author=G.%20M.%20Morris&volume=30&publication_year=2009&pages=2785-2791&pmid=19399780&doi=10.1002/jcc.21256&)

49. Wang J. C., Lin J. H., Chen C. M., Perryman A. L. & Olson A. J. Robust scoring functions for protein-ligand interactions with quantum chemical charge models. J. Chem. Inf. Model. 51, 2528–2537 (2011).  [DOI](https://doi.org/10.1021/ci200220v) | [PMC free article](/articles/PMC4639406/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21932857/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Chem.%20Inf.%20Model.&title=Robust%20scoring%20functions%20for%20protein-ligand%20interactions%20with%20quantum%20chemical%20charge%20models&author=J.%20C.%20Wang&author=J.%20H.%20Lin&author=C.%20M.%20Chen&author=A.%20L.%20Perryman&author=A.%20J.%20Olson&volume=51&publication_year=2011&pages=2528-2537&pmid=21932857&doi=10.1021/ci200220v&)

50. Hornak V. et al. Comparison of multiple amber force fields and development of improved protein backbone parameters. Proteins 65, 712–725 (2006).  [DOI](https://doi.org/10.1002/prot.21123) | [PMC free article](/articles/PMC4805110/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16981200/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proteins&title=Comparison%20of%20multiple%20amber%20force%20fields%20and%20development%20of%20improved%20protein%20backbone%20parameters&author=V.%20Hornak&volume=65&publication_year=2006&pages=712-725&pmid=16981200&doi=10.1002/prot.21123&)

51. Wang J. M., Wolf R. M., Caldwell J. W., Kollman P. A. & Case D. A. Development and testing of a general amber force field. J. Comput. Chem. 25, 1157–1174 (2004).  [DOI](https://doi.org/10.1002/jcc.20035) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15116359/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Comput.%20Chem.&title=Development%20and%20testing%20of%20a%20general%20amber%20force%20field&author=J.%20M.%20Wang&author=R.%20M.%20Wolf&author=J.%20W.%20Caldwell&author=P.%20A.%20Kollman&author=D.%20A.%20Case&volume=25&publication_year=2004&pages=1157-1174&pmid=15116359&doi=10.1002/jcc.20035&)

52. Pettersen E. F. et al. UCSF chimera - a visualization system for exploratory research and analysis. J. Comput. Chem. 25, 1605–1612 (2004).  [DOI](https://doi.org/10.1002/jcc.20084) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15264254/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Comput.%20Chem.&title=UCSF%20chimera%20-%20a%20visualization%20system%20for%20exploratory%20research%20and%20analysis&author=E.%20F.%20Pettersen&volume=25&publication_year=2004&pages=1605-1612&pmid=15264254&doi=10.1002/jcc.20084&)
