# Exome Sequencing of Extreme Clopidogrel Response Phenotypes Identifies B4GALT2 as a Determinant of On-treatment Platelet Reactivity

## Metadata
**Authors:** Stuart A Scott, Jean-Philippe Collet, Usman Baber, Yao Yang, Inga Peter, Michael Linderman, Jeffrey Sload, Wanqiong Qiao, Annapoorna S Kini, Samin K Sharma, Robert J Desnick, Valentin Fuster, Roger J Hajjar, Gilles Montalescot, Jean-Sébastien Hulot
**Journal:** Clinical pharmacology and therapeutics
**Date:** 2016 Jun 20
**DOI:** [10.1002/cpt.401](https://doi.org/10.1002/cpt.401)
**PMID:** 27213804
**PMCID:** PMC4982803
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4982803/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC4982803/pdf/nihms-789137.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC4982803/pdf/nihms-789137.pdf)

## Abstract

Interindividual variability in platelet aggregation is common among patients treated with clopidogrel, and both high and low (LTPR) on-treatment platelet reactivity increase risks for adverse clinical outcomes. CYP2C19 influences clopidogrel response but only accounts for ~12% of the variability in platelet reactivity. To identify novel variants implicated in on-treatment platelet reactivity, coronary artery disease (CAD) patients with extreme pharmacodynamic responses to clopidogrel and wild-type CYP2C19 were subjected to exome sequencing. Candidate variants that clustered in the LTPR subgroup subsequently were genotyped across the discovery cohort (n=636). Importantly, carriers of B4GALT2 c.909C>T had lower on-treatment P2Y12 reaction units (PRU; p=0.0077) and residual platelet aggregation (p=0.0008) compared to non-carriers, which remained significant after adjusting for CYP2C19 and other clinical variables in both the discovery (p=0.0298) and replication (n=160; PRU: p=0.0001) cohorts. B4GALT2 is a platelet-expressed galactosyltransferase, indicating that B4GALT2 c.909C>T may influence clopidogrel sensitivity through atypical cell-surface glycoprotein processing and platelet adhesion.

Keywords: Clopidogrel, pharmacogenomics, pharmacodynamics, platelet aggregation, exome sequencing, CYP2C19, B4GALT2

## INTRODUCTION

Dual antiplatelet therapy (DAPT) with clopidogrel and aspirin reduces atherothrombotic events in patients with acute coronary syndrome (ACS) and those undergoing percutaneous coronary intervention (PCI) ([1](#R1)-[3](#R3)). Wide interindividual variability in clopidogrel response and P2Y_12_ receptor inhibition is commonly observed and some patients still experience thrombotic or bleeding events on treatment ([4](#R4)-[6](#R6)). Long term DAPT reduces stent-related events and the risk of non-stent-related ischemic events in patients with ACS but increases the risk of bleeding, which has been strongly linked to mortality ([7](#R7)). An association between bleeding and mortality was also observed in the DAPT Study ([8](#R8)), including more cancer, trauma and direct bleeding-related deaths, which underscores the importance of carefully considering the risks of ischemic versus bleeding events for each patient on antiplatelet therapy.

The major genetic determinant of clopidogrel response variability is the common *CYP2C19*2* loss-of-function allele (c.681G>A; rs4244285), which has been associated with high on-treatment platelet reactivity (HTPR) by both candidate gene ([9](#R9)-[11](#R11)) and genome-wide association studies (GWAS) ([12](#R12)). The CYP2C19 enzyme is directly involved in both steps of the hepatic bioactivation of clopidogrel to its active metabolite, supporting the association between *CYP2C19* loss-of-function alleles (e.g., **2*-**8*) and reduced active metabolites ([13](#R13), [14](#R14)), HTPR ([9](#R9), [10](#R10), [12](#R12), [13](#R13), [15](#R15)), and increased risks for adverse cardiovascular outcomes among ACS/PCI patients ([10](#R10)-[12](#R12), [15](#R15)). In contrast, the *CYP2C19*17* increased activity allele results in enhanced platelet inhibition and has been associated with an increased bleeding risk in some studies ([16](#R16)-[18](#R18)). However, *CYP2C19* only accounts for ~12% of the variability in clopidogrel response ([12](#R12)), which has prompted studies directed at identifying other clinical and/or genetic variables involved in on-treatment platelet reactivity ([19](#R19), [20](#R20)). To identify novel variants that influence on-treatment platelet reactivity, we implemented an extreme phenotype exome sequencing pilot study. This study design was selected based on the success of other previously reported extreme phenotype quantitative trait sequencing studies that identified novel variants associated with electrocardiographic QT intervals ([21](#R21)), therapeutic warfarin doses ([22](#R22), [23](#R23)), *P. aeruginosa* infection ([24](#R24)), and cholesterol levels ([25](#R25)-[27](#R27)).

## RESULTS

### Patient Selection and Characteristics

CAD patients treated with DAPT and with complete evaluation of on-treatment platelet reactivity were selected from the previously reported ONline ASSIstance for Stent Thrombosis (ONASSIST) cohort ([11](#R11)) (n=636) (**Figure 1; Table 1**). The distributions of both the VerifyNow™ P2Y_12_ (VN-P2Y_12_) and light transmission aggregometry (LTA) platelet function results were plotted to identify patients with the most extreme pharmacodynamic responses to clopidogrel (see **Methods** and **Figure 1**). As previously reported, wide interpatient variability in on-treatment platelet reactivity was observed by both P2Y_12_ reaction units (PRU) [median: 167; interquartile range (IQR): 103-230] and residual platelet aggregation (RPA) (median: 12; IQR: 0-88; **Table 1**). A total of 48 and 64 patients were identified with low on-treatment platelet reactivity (LTPR) and HTPR, respectively, 25 patients from each subgroup were wild-type for *CYP2C19* (**1/***1*), and four patients from each extreme subgroup were subjected to exome sequencing (**Table 2**; **Figure 1**). Wild-type *CYP2C19* (**1*/**1*) status was confirmed in these eight patients by genotyping an expanded panel of *CYP2C19* alleles (**2* - **10*, **17*) using an orthogonal platform.

### FIGURE 1. Illustration of the extreme phenotype exome sequencing study design.

![FIGURE 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/784f/4982803/e2e33668e10a/nihms-789137-f0001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4982803_nihms-789137-f0001.jpg)

CAD: coronary artery disease; DAPT: dual antiplatelet therapy; HTPR: high on-treatment platelet reactivity; LTA: light transmission aggregometry; LTPR: low on-treatment platelet reactivity; PCI: percutaneous coronary intervention; PRU: P2Y12 reaction units; RPA: residual platelet aggregation; VN-P2Y12: VerifyNow™ P2Y12.

### TABLE 1.

|   | N=636 |
| --- | --- |
| Demographics |   |
| Age (yr) | 59±14 |
| Male sex [N, (%)] | 535 (84.1%) |
| BMI (kg/m2) | 26.4±4.1 |
| Risk factors [N, (%)] |   |
| Diabetes | 165 (25.9%) |
| Familial CAD | 146 (22.9%) |
| Current smoker | 123 (20.0%) |
| Hypertension | 299 (47.0%) |
| Dyslipidemia | 367 (57.7%) |
| Previous STEMI | 298 (46.8%) |
| CABG | 32 (5.0%) |
| Platelet function testing |   |
| VN-P2Y12 (PRU) |   |
| Min | 0 |
| Max | 419 |
| Median | 167 |
| 25th percentile | 103 |
| 75th percentile | 230 |
| ADP-induced LTA (RPA; %) |   |
| Min | 0 |
| Max | 88 |
| Median | 12 |
| 25th percentile | 0 |
| 75th percentile | 88 |

Table 1 Caption: Patient characteristics from the ONASSIST discovery cohort

### TABLE 2.

| ID | Age | Gender | BMI | PRU | RPA | CYP2C19 Genotype * |
| --- | --- | --- | --- | --- | --- | --- |
| LTPR 1 | 43 | Male | 24 | 7 | 0% | *1/*1 |
| LTPR 2 | 51 | Female | 33 | 7 | 0% | *1/*1 |
| LTPR 3 | 53 | Male | 22 | 8 | 0% | *1/*1 |
| LTPR 4 | 40 | Male | 33 | 9 | 0% | *1/*1 |
|   |  |  |  |  |  |  |
| HTPR 1 | 71 | Female | 23 | 371 | 81% | *1/*1 |
| HTPR 2 | 80 | Male | 27 | 259 | 85% | *1/*1 |
| HTPR 3 | 66 | Male | 34 | 415 | 75% | *1/*1 |
| HTPR 4 | 90 | Male | 30 | 323 | 62% | *1/*1 |

Table 2 Caption: Characteristics of LTPR and HTPR patient subgroups selected for exome sequencing

### Variant Identification and Selection

A total of 129,388 variants were identified by exome sequencing among the eight *CYP2C19*1/*1* patients from the LTPR and HTPR subgroups, 118,634 of which passed the variant calling pipeline quality filters ([28](#R28)) (**Supplementary Table 1**). Among all 118,634 passing variants, 53,993 (45.5%) were located in the targeted consensus coding sequence (CCDS) capture regions and 48,919 (90.6%) of these had genotype calls in all eight samples and were utilized for subsequent ranking and candidate variant selection (see **Methods**).

The focus of our initial analysis was identifying candidate variants implicated in LTPR. A total of 403 variants with available European minor allele frequencies (MAFs) were identified that clustered in the LTPR subgroup with either three or four carriers (heterozygous or homozygous) versus zero carriers in the HTPR subgroup. As detailed in the **Methods**, 28 variants were identified that clustered in the LTPR subgroup with a *P(E)*<0.001 based on Hardy-Weinberg probability and European MAFs, and after manual removal of X chromosome and low quality variants, 18 candidate autosomal variants were selected for genotyping across the entire CAD discovery cohort (**Supplementary Table 2**).

### B4GALT2 and On-treatment Platelet Reactivity

Of the 18 candidate variants potentially implicated in LTPR, 17 variants were successfully genotyped across the discovery cohort of 636 patients and tested for association with on-treatment platelet reactivity. The only variant that was significantly associated with platelet response (PRU or RPA) after Bonferroni correction (0.05/17, p<0.003) was the synonymous c.909C>T (p.Ile303=; rs1061781) variant in the beta-1,4-galactosyltransferase 2 (*B4GALT2*) gene. Carriers of c.909C>T had lower PRU (p=0.0077) and RPA (p=0.0008) compared to non-carriers (**Table 3**; **Figure 2**). Another *B4GALT2* variant, c.366G>C (p.Gln122His; rs1859728), was included in the 18 genotyped candidate variants and although it did not achieve significance after correction for multiple testing, c.366G>C carriers also had lower PRU (p=0.0211) and RPA (p=0.0046) compared to non-carriers (**Table 3**). These two variants did show evidence of linkage disequilibrium (LD) with D’=0.68 and r^2^=0.30, which was consistent with the LD observed in the European (CEU) dataset of the 1000 Genomes Project Phase I (D’=0.78 and r^2^=0.32; **Supplementary Table 3**). The MAFs of *B4GALT2* c.909C>T (0.101) and c.366G>C (0.071) identified in the CAD patient cohort were consistent with the European (non-Finnish) MAFs from the Exome Aggregation Consortium (ExAC) database (0.095 and 0.052, respectively) and their genotype distributions in the discovery cohort were in Hardy-Weinberg equilibrium (p=0.51 and p=0.93, respectively).

### TABLE 3.

| Cohort | B4GALT2 Genotype | PRU | P-Value | RPA | P-Value |  |
| --- | --- | --- | --- | --- | --- | --- |
| Discovery Cohort n=636 | rs1061781 (c.909C>T) |   |   |   |   |  |
| Non-carriers (n=510) | 172.1±87.1 | 0.0077 * | 33.1±23.4 | 0.0008 * |  |  |
| Carriers (n=125) | 148.7±90.0 |   | 25.2±22.6 |   |  |  |
| rs1859728 (c.366G>C) |   |   |   |   |  |  |
| Non-carriers (n=547) | 170.6±86.6 | 0.0211 * | 32.6±23.3 | 0.0046 * |  |  |
| Carriers (n=86) | 147.0±96.1 |   | 24.9±23.0 |   |  |  |
| rs1061781 (c.909C>T) |   |   |   |   |  |  |
| C/C (n=510) | 172.1±87.1 | 0.034 † | 33.1±23.4 | 0.0028 † |  |  |
| C/T (n=120) | 148.2±90.7 |   | 25.2±22.5 |   |  |  |
| T/T (n=5) | 159.4±79.3 |   | 25.2±26.1 |   |  |  |
| rs1859728 (c.366G>C) |   |   |   |   |  |  |
| G/G (n=547) | 170.6±86.6 | 0.0352 † | 32.6±23.3 | 0.0134 † |  |  |
| G/C (n=83) | 148.3±97.1 |   | 25.2±23.2 |   |  |  |
| C/C (n=3) | 110.4±57.2 |   | 17.6±14.7 |   |  |  |
|   |  |  |  |  |  |  |
| Replication Cohort ‡ n=160 | rs1061781 (c.909C>T) |   |   |   |   |  |
| Non-carriers (n=140) | 233.5±17.2 | 0.0001 * | ND | ND |  |  |
| Carriers (n=20) | 142.6±42.2 |   | ND |   |  |  |
| rs1859728 (c.366G>C) |   |   |   |   |  |  |
| Non-carriers (n=147) | 225.7±16.9 | 0.0740 * | ND | ND |  |  |
| Carriers (n=13) | 181.4±76.3 |   | ND |   |  |  |

Table 3 Caption: B4GALT2 genotype and on-treatment platelet reactivity

### FIGURE 2. On-treatment platelet reactivity based on B4GALT2 c.909C>T (rs1061781) carrier status [non-carriers versus carriers (hetero- or homozygous)].

![FIGURE 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/784f/4982803/5f2b44e9ff28/nihms-789137-f0002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4982803_nihms-789137-f0002.jpg)

Platelet reactivity was measured in the discovery cohort by both the (A) VN-P2Y12 and (B) LTA assays. The VN-P2Y12 and LTA results are reported as P2Y12 reaction units (PRU) values and residual platelet aggregation (RPA), respectively.

Multivariable linear regression was performed to determine if *B4GALT2* c.909C>T and c.366G>C were independently associated with on-treatment platelet reactivity among the CAD discovery cohort and if the effect was independent of other variables known to influence platelet reactivity. Importantly, only *B4GALT2* c.909C>T and *CYP2C19* loss-of-function alleles remained independent genetic predictors of on-treatment platelet reactivity (p=0.03 and p=0.01, respectively) after inclusion of relevant clinical variables (adjusted R^2^=0.09; **Table 4**). Interaction terms for age and *B4GALT2* variants were not significant when incorporated into the regression model, indicating that the effect of *B4GALT2* c.909C>T in our cohort was not modified by the independent effect of patient age. Moreover, there was no difference in the mean age between *B4GALT2* carriers and non-carriers in this cohort (58.9±13.9 vs 59.5±13.3, p=0.70).

### TABLE 4.

| Variable | Beta | Standard Error | t | P-value |
| --- | --- | --- | --- | --- |
| Discovery Cohort |   |   |   |   |
| B4GALT2 c.909C>T (rs1061781) | −22.81 | 10.47 | −2.18 | 0.0298 |
| B4GALT2 c.366G>C (rs1859728) | −9.33 | 12.11 | −0.77 | 0.441 |
| CYP2C19 LOF allele carrier | 20.75 | 7.63 | 2.72 | 0.0068 |
| Diabetes | 32.53 | 7.96 | 4.08 | <0.0001 |
| Age | 1.15 | 0.25 | 4.51 | <0.0001 |
| Gender | −14.67 | 9.63 | −1.52 | 0.128 |
|   |  |  |  |  |
| Replication Cohort |   |   |   |   |
| B4GALT2 c.909C>T (rs1061781) | −98.43 | 26.70 | −3.69 | <0.0001 |
| B4GALT2 c.366G>C (rs1859728) | 8.65 | 32.36 | 0.27 | 0.790 |
| CYP2C19 LOF allele carrier | 47.66 | 16.34 | 2.92 | 0.004 |
| Diabetes | 8.56 | 15.85 | 0.54 | 0.59 |
| Age | 2.54 | 0.76 | 3.33 | 0.001 |
| Gender | −25.01 | 17.34 | −1.44 | 0.15 |

Table 4 Caption: Multivariable linear regression of B4GALT2 genotype and on-treatment platelet reactivity measured by PRU

Notably, carriage of *B4GALT2* c.909C>T also imparted a significantly increased odds of presenting with LTPR [PRU<50; odds ratio (OR): 2.46, 95% confidence interval (CI) 1.31-4.61, p<0.005]. The *B4GALT2* c.909C>T association was replicated in an independent cohort of ACS patients following PCI and a 600 mg clopidogrel load (n=160). Similar to the discovery cohort, carriers of c.909C>T had significantly lower PRU compared to non-carriers (142.6±42.2 vs 233.5±17.2; p=0.0001; **Table 3**), which again was independent of *CYP2C19* and other clinical variables by multivariable linear regression (adjusted R^2^=0.20; **Table 4**).

### Chromosome 1p34.1 candidate genes

Interrogation of the haplotype blocks in the chromosome 1p34.1 region from the major populations of the 1000 Genomes Project Phase I (**Supplementary Figure 2**) suggest two potential candidate genes with the most likely biological rationale for a role in platelet aggregation: *B4GALT2* and *ST3GAL3*. Importantly, the PaxDB integrated proteomics dataset ([29](#R29)) indicates that both B4GALT2 and ST3GAL3 are predominantly expressed in plasma and platelets and both enzymes are type II golgi membrane-bound glycoproteins involved in post-translational modification and processing. B4GALT2 transfers galactose from UDP-galactose to similar acceptor sugars ([30](#R30)) and ST3GAL3 catalyzes the transfer of sialic acid from CMP-sialic acid to galactose-containing substrates.

## DISCUSSION

The success of previously reported extreme phenotype studies ([22](#R22)-[24](#R24), [27](#R27)) prompted our exome sequencing pilot study of a subset of carefully phenotyped CAD patients with extreme pharmacodynamic responses to clopidogrel, which identified a novel association between variants in the *B4GALT2* gene and on-treatment platelet reactivity. The two *B4GALT2* variants (c.909C>T and c.366G>C) were identified in the LTPR patients by exome sequencing but not in any of the HTPR patients. The low probabilities of observing the clustering in the LTPR subgroup were driven by relatively low European MAFs (~5-10%), which prompted their inclusion as candidate variants for genotyping across the CAD patient cohort. To our knowledge, this strategy of ranking variants has not previously been utilized in exome sequencing studies; however, CAD patients who carried c.909C>T had lower on-treatment PRU and RPA compared to non-carriers regardless of *CYP2C19* genotype and other clinical variables known to independently influence on-treatment platelet reactivity (i.e., age ([31](#R31)), diabetes ([32](#R32))).

Although our Hardy-Weinberg-based candidate variant ranking method may be a viable option for underpowered and/or pilot studies on exome or candidate gene sequencing, it is not meant to replace formal statistical tests when sample size allows, and should therefore be limited to underpowered and pilot studies when an ethnically-matched MAF database and independent replication cohort are available. Given that both our discovery and replication cohorts were of European descent, it is currently unclear if the identified effect of *B4GALT2* c.909C>T would be consistent in other racial and ethnic groups. Differences in population structure and the borderline European MAF of *B4GALT2* c.909C>T (~10%) would likely contribute to the capacity to have detected an effect from this chromosome 1p34.1 region by the previously reported clopidogrel response GWAS performed in the Amish population ([12](#R12)).

In addition, it is also challenging to compare the proportion of variability in on-treatment platelet reactivity imparted by genetic and clinical variables between studies due to differences in study design (e.g., time and method of platelet reactivity assessment, clinical data collected, etc.), cohorts (e.g., healthy adults, patients, founder populations, outbred populations, ancestry admixture, etc.), antiplatelet indication ([33](#R33)), and the fact that individual patient platelet reactivity measurements can change over time ([34](#R34)). This is further evidenced by the proportions of variability identified by multivariable analysis in our discovery (~9%) and replication (~20%) cohorts, as well as those previously reported in other studies (~6-22%) ([12](#R12), [35](#R35)). Our interpretation of these results is that on-treatment platelet reactivity is a multifactorial trait that is influenced by clinical and genetic factors to differing degrees, and that some of these factors may account for greater interindividual variability depending on patient ethnicity and when (and how) on-treatment reactivity was measured.

The human B4GALT2 is a 372-amino acid polypeptide containing an N-terminal cytoplasmic domain, a single transmembrane domain, a catalytic domain facing the golgi lumen, and three potential glycosylation sites ([36](#R36)). Given its expression in platelets and role in synthesizing glycolipids and glycoproteins, B4GALT2 could influence platelet adhesion through post-translational modification of integrins and/or other cell surface or secreted glycoproteins as they traffic through the endoplasmic reticulum and golgi to the plasma membrane. The platelet surface glycoprotein IIb/IIIa integrin complex (GPIIb/IIIa) is a critical regulator of thrombosis formation through its ability to bind fibrinogen and other blood components following platelet activation, which subsequently facilitates platelet-platelet crosslinks. Notably, both GPIIb and GpIIIa precursors undergo significant processing and modification in the endoplasmic reticulum and golgi, including phosphorylation and glycosylation ([37](#R37)), prior to assembly and stable transport to the platelet surface ([38](#R38)). Although little data is available directly linking B4GALT2 to integrin or other platelet surface glycoprotein processing, galactose is one of the principal sugar moieties involved in protein glycosylation. Moreover, congenital disorders of glycosylation are characterized by platelet count and coagulation abnormalities, and B4GALT1 has recently been implicated in β1 integrin expression and interaction with components of the extracellular matrix, specifically laminin ([39](#R39)).

The *B4GALT2* gene has seven exons and three transcript variants with unique first exons (1A, 1B, 1C) that encode two different isoforms; however, c.909C>T and c.366G>C have the same predicted consequences on both isoforms given that the only difference is a short N-terminal 29 amino-acid sequence. The c.909C>T variant is a synonymous (p.Ile303=) transition polymorphism in exon 6 of *B4GALT2* that is within the catalytic domain of the mature B4GALT2 enzyme (**Supplementary Figure 3**). Although no amino acid is altered, c.909C>T could potentially influence alternative gene splicing; however, NetGene2 ([40](#R40)) and Human Splicing Finder ([41](#R41)) did not predict the creation of any new splicing motifs. Additionally, c.909C>T was not found to be an expression quantitative trait locus (eQTL) or linked to another eQTL, although the tissues included in the currently available eQTL databases do not include platelets. Consequently, it is currently unclear if c.909C>T has a direct functional role on B4GALT2 activity; however, an effect on *B4GALT2* expression mediated by a haplotype with other non-coding variants is still a possible mechanism for this synonymous variant.

The c.366G>C variant is a missense (p.Gln122His) transversion polymorphism in exon 3 of the *B4GALT2* gene that is also within the catalytic domain of the mature B4GALT2 enzyme. PROVEAN and SIFT assessment of the glutamine to histidine amino acid substitution predicted ‘neutral’ and ‘tolerated’, respectively; however, PolyPhen2 predicted this variant to be ‘probably damaging’ with a high score of 0.995. *In silico* algorithms for missense variants are not always consistent in their functional prediction, leaving the interpretation of these discordant results uncertain. Although the c.366G>C variant was not independently significant when included in a model with c.909C>T and other variables, it could still potentially contribute to a functional haplotype that harbors c.909C>T and other sequence variants.

The *ST3GAL3* gene is ~50 kb distal to *B4GALT2* and could also potentially be implicated in platelet aggregation through a similar glycoprotein post-translational modification pathway if *B4GALT2* c.909C>T occurred on a larger haplotype that included functional *ST3GAL3* variants, particularly given that ST3GAL3 is also expressed in the plasma and platelet. Although this hypothesis also requires further investigation, it is notable that mice deficient in the related *St3gal4* have increased platelet clearance rates due to deficiently sialylated platelets ([42](#R42)) and that lower platelet sialic acid has been implicated in human platelet aggregation via cell surface charge modification, which has been postulated to be a contributing factor to CAD ([43](#R43)).

In conclusion, exome sequencing of CAD patients with extreme pharmacodynamic responses to clopidogrel facilitated the identification of a significant association between *B4GALT2* c.909C>T and LTPR, which translated to a significantly increased odds of presenting with LTPR (PRU<50; OR: 2.46, 95% CI 1.31-4.61, p<0.005) among patients who carried *B4GALT2* c.909C>T. Millions of patients are treated with stents annually and exposed to P2Y_12_ inhibitors, mainly clopidogrel, and there are subsets of patients in whom the hemorrhagic risks of prolonged DAPT are outweighed by the benefits of suppressing stent thrombosis and myocardial infarction. As such, if found to be adequately validated by independent studies, *B4GALT2* c.909C>T could potentially help inform whether long-term as opposed to short-term DAPT is warranted prior to initiation of P2Y_12_ inhibitors. Moreover, if the effect of *B4GALT2* c.909C>T is determined to directly influence baseline platelet aggregation, it may also help identify susceptibility to LTPR among patients treated with the more potent P2Y_12_ inhibitors that have increased risks for major bleeding events compared to clopidogrel. However, prior to any discussion of clinical implementation, future studies are warranted to validate these findings, including functional studies of *B4GALT2* and platelet biology, and clinical outcome studies on patients identified as having LTPR and/or bleeding events.

## METHODS

### Discovery Patient Population

The present data correspond to secondary analyses of the ONASSIST project, a French nationwide registry of adult patients presenting with CAD who received stent implantation under antiplatelet therapy (all of European descent). This investigator-initiated observational registry was primarily built to identify clinical, angiographic, and biological determinants of stent thrombosis as previously reported ([11](#R11)). The registry was led and partially supported by the nonprofit Academic Research Organization ACTION (Allies in Cardiovascular Trials, Initiatives and Organized Networks), located at Pitié-Salpêtrière Hospital (University Paris 6). The protocol was approved by the Pitié-Salpêtrière University Hospital Ethics Committee and the study was conducted in accordance with the Helsinki declaration. Written informed consent was obtained from all patients.

### Replication Patient Population

The previously reported replication patient cohort ([32](#R32)) was selected based on its consistency with the discovery cohort in patient population, ethnicity, and antiplatelet treatment. In brief, compliant clopidogrel-naive patients (all self-reported white and non-Hispanic by United States Census Bureau definitions) undergoing PCI at the Mount Sinai Medical Center (New York City, NY) were loaded with a 600 mg dose of clopidogrel either during or on the same day of the procedure (n=160). Blood was drawn for platelet function testing a minimum of 4 hours after the clopidogrel load. Residual platelet reactivity was assessed by the VN-P2Y_12_ assay (Accumetrics Corporation, San Diego, CA, USA) and expressed as P2Y12 reaction units (PRU).

### Study Design and Extreme Phenotype Definitions

ONASSIST compliant patients treated with maintenance DAPT (aspirin 75 mg/d and clopidogrel 75 mg/d) ([11](#R11)) were screened for having both on-treatment platelet reactivity measurements and available DNA. Patients were then selected who had measurements of platelet reactivity using both the point-of-care VN-P2Y_12_ assay and light transmission aggregometry (LTA) (Model 490-4D, Chrono-Log Corporation, Kordia, The Netherlands). For the VN-P2Y_12_ assay, samples were run in the same laboratory according to the manufacturer's instructions and results expressed as PRU. For LTA, the residual platelet aggregation (RPA) corresponds to the level of aggregation curve (%) measured 6 minutes after 20 μmol/L ADP-induced platelet aggregation. The two extreme phenotype subgroups were defined by results from both platelet assays, with LTPR patients having both PRU<50 and RPA<5% and HTPR patients having both PRU>235 and RPA>60%.

### Cytochrome P450-2C19 (CYP2C19) Genotyping

The designations of all *CYP2C19* alleles refer to those defined by the Cytochrome P450 Allele Nomenclature Committee ([44](#R44)). The *CYP2C19*2-*6* loss-of-function and **17* increased-function variants were interrogated among all CAD patients using commercially available and previously validated TaqMan genotyping assays (Applied Biosystems, Foster City, CA, USA) and the 7900HT sequence detection system (Applied Biosystems). In addition, the cases selected for exome sequencing were also genotyped for the *CYP2C19*2 - *10* and **17* variant alleles using the xTAG^®^ CYP2C19 Kit v3 (Luminex Molecular Diagnostics, Inc., Toronto, ON, Canada) as per the manufacturer's instructions. For the replication cohort (see below), 11 *CYP2C19* alleles (**2* - **10*, **13*, **17*) were genotyped using the eSensor® 2C19 Test (GenMark Diagnostics, Carlsbad, CA, USA) as per the manufacturer's instructions. The wild-type *CYP2C19*1* allele was assigned in the absence of other detectable variant alleles.

### Exome Sequencing and Variant Calling

Exome capture was performed using the SeqCap EZ Exome (33.9 Mb) and SeqCap EZ Exome v2.0 (44.0 Mb) capture kits (Roche NimbleGen; Madison, WI, USA) and sequenced using the Illumina GA IIx platform with 75 or 100 bp paired-end sequencing. Sample preparation and sequencing was performed by the Genomics Core Facility, Institute for Genomics and Multiscale Biology, at the Icahn School of Medicine at Mount Sinai. Variant calling and quality filtering was performed using the GATK-based genome analysis pipeline (GAP) as previously described ([28](#R28)). Only variants with genotypes called across all eight samples were considered for candidate variant selection. Despite the use of two exome capture kits and both 76 and 100 bp paired-end sequencing on subjects from each subgroup, the number of quality passing variants per subject was consistent across all eight patients [average: 55,529; standard deviation (sd): 1,941] and within both the LTPR (average: 55,294; sd: 2,652) and HTPR (average: 55,764; sd: 627) subgroups (**Supplementary Table 1**). Similar consistencies were observed across samples with other sequencing and variant metrics (**Supplementary Table 1**).

### Candidate Variant Prioritization and Selection

The pilot study design did not allow for a formal power calculation or statistical test for candidate variant selection. However, to facilitate variant prioritization, passing variants called in all eight LTPR and HTPR patient samples were ranked by probabilities of clustering in either subgroup based on their reported European MAFs. The MAFs of all interrogated variants were extracted from the European (non-Finnish) dataset of the ExAC, which is comprised of 61,486 total unrelated individuals (33,055 non-Finnish Europeans) sequenced as part of various disease-specific and population genetic studies ([45](#R45)). The MAFs of intronic variants without ExAC MAF data were taken from the European Exome Sequencing Project (ESP) dataset ([46](#R46)) when needed. Variants on the X chromosome were excluded from the prioritization analysis due to uncertainty of gender in reference MAFs. Probabilities were calculated for variants that were detected in three or four of the LTPR patients (either heterozygous or homozygous) but not in any of the four HTPR patients. Variants were then ranked by decreasing probability of the observed genotype clustering in the LTPR extreme phenotype subgroup given their European MAFs using the following Hardy-Weinberg-based formula:

| P(E)=(2×(1−f(q))×f(q))N(het)×f(q)2×N(alt_homo) |
| --- |
Where *P(E)* is the probability of the event (i.e., genotypes observed in the LTPR subgroup), *f*(q) is the reported European MAF, N(het) is the number of heterozygous samples identified in the LTPR subgroup, and N(alt_homo) is the number of homozygous minor allele samples identified in the LTPR subgroup. As a pilot study, variants with an empirical *P*(*E*)<0.001 of clustering in the LTPR subgroup and not in any of the HTPR patients were selected as potential candidate variants for genotyping across the entire CAD patient cohort. Prior to genotyping, manual interrogation of all candidate variants for sequencing depth and potential misalignment due to pseudogenes using the Integrative Genomics Viewer (IGV) ([47](#R47)) was performed to remove ambiguous candidate variants.

### Candidate Variant Genotyping and Quality Control

Selected candidate variants were interrogated across the entire CAD patient cohort using multiplexed custom SNPtype™ genotyping assays and 96.96 Dynamic Array™ IFCs (Fluidigm; South San Francisco, CA, USA) ([48](#R48)). In brief, all DNA samples were subjected to specific target amplification (STA), which consisted of an initial denaturation step at 95°C for 15 min followed by 14 amplification cycles (95°C for 15 sec and 60°C for 4 min). SNPtype genotyping was carried out on a 1:100 dilution of STA product using an FC1 Cycler with the SNPtype 96X96 Fast v1 protocol and the Fluidigm Genotyping Analysis software as per the manufacturer's instructions. Quality control was performed by genotyping all variants with independent duplicate SNPtype™ assays across all samples and manual review of genotyping scatter plots. Discordant genotype calls between duplicates were removed, as were variants with ambiguous genotyping scatter plots. In addition, selected variants were confirmed by Sanger sequencing using independent PCR primers and standard protocols. Sanger sequencing results for representative *B4GALT2* c.909C>T and c.366G>C carriers are illustrated in **Supplementary Figure 1**.

### Statistical Analyses

Continuous variables were expressed as mean±standard deviation (SD) and categorical variables as frequencies and percentages. Platelet reactivity was compared between genotype groups using the Student t-test or Kruskal-Wallis one way analysis of variance as appropriate. Association between genotypes and on-treatment platelet reactivity (measured by the VN-P2Y_12_ assay) was performed using linear regression analyses and the association between *B4GALT2* genotype and the risk of presenting with LTPR was evaluated using binary logistic regression analyses. All statistical analyses were performed using either SAS version 9.4 (Cary, NC, USA) or Stata version 12.1 (College Station, TX, USA).

Competency in medical knowledge:

## Supplementary Material

What is the current knowledge on the topic?

Interindividual variability in antiplatelet response is commonly observed among patients treated with clopidogrel, which is influenced, in part, by variant alleles in the *CYP2C19* gene.

What question did this study address?

Although clinical *CYP2C19* genetic testing and pharmacogenetic practice guidelines are available, the identification of novel determinants of on-treatment platelet reactivity, if validated, could ultimately help inform individual antiplatelet therapy.

What this study adds to our knowledge?

Exome sequencing of patients with extreme pharmacodynamic responses to clopidogrel identified *B4GALT2* c.909C>T as a potential genetic determinant of low on-treatment platelet reactivity.

How this might change clinical pharmacology or translational science?

The ability to detect patients with genetic susceptibility to clopidogrel sensitivity could facilitate the identification of patients at high risk of bleeding prior to initiating antiplatelet therapy.

What is the current knowledge on the topic?

Interindividual variability in antiplatelet response is commonly observed among patients treated with clopidogrel, which is influenced, in part, by variant alleles in the *CYP2C19* gene.

What question did this study address?

Although clinical *CYP2C19* genetic testing and pharmacogenetic practice guidelines are available, the identification of novel determinants of on-treatment platelet reactivity, if validated, could ultimately help inform individual antiplatelet therapy.

What this study adds to our knowledge?

Exome sequencing of patients with extreme pharmacodynamic responses to clopidogrel identified *B4GALT2* c.909C>T as a potential genetic determinant of low on-treatment platelet reactivity.

How this might change clinical pharmacology or translational science?

The ability to detect patients with genetic susceptibility to clopidogrel sensitivity could facilitate the identification of patients at high risk of bleeding prior to initiating antiplatelet therapy.

## ACKNOWLEDGEMENTS

This research was supported in part by the National Center for Advancing Translational Sciences (NCATS) and the National Institute of General Medical Sciences (NIGMS) of the National Institutes of Health (NIH), through grants KL2 TR000069 and K23 GM104401, respectively (S.A.S.). In addition, this work was supported in part through the computational resources and staff expertise provided by the Department of Scientific Computing at the Icahn School of Medicine at Mount Sinai. The authors thank Ms. Bernadette Liggayu and the Institute for Personalized Medicine (IPM) at the Icahn School of Medicine at Mount Sinai for technical assistance with the Fluidigm genotyping. S.A.S. would also like to thank Dr. Alan R. Shuldiner from the University of Maryland School of Medicine and Regeneron Pharmaceuticals for helpful discussion and mentorship.

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Mehta SR, et al. Effects of pretreatment with clopidogrel and aspirin followed by long-term therapy in patients undergoing percutaneous coronary intervention: the PCI-CURE study. Lancet. 2001;358:527–33. doi: 10.1016/s0140-6736(01)05701-4.  [DOI](https://doi.org/10.1016/s0140-6736(01)05701-4) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11520521/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Effects%20of%20pretreatment%20with%20clopidogrel%20and%20aspirin%20followed%20by%20long-term%20therapy%20in%20patients%20undergoing%20percutaneous%20coronary%20intervention:%20the%20PCI-CURE%20study.&author=SR%20Mehta&volume=358&publication_year=2001&pages=527-33&pmid=11520521&doi=10.1016/s0140-6736(01)05701-4&)

2. Sabatine MS, et al. Effect of clopidogrel pretreatment before percutaneous coronary intervention in patients with ST-elevation myocardial infarction treated with fibrinolytics: the PCI-CLARITY study. JAMA : the journal of the American Medical Association. 2005;294:1224–32. doi: 10.1001/jama.294.10.1224.  [DOI](https://doi.org/10.1001/jama.294.10.1224) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16143698/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA%20:%20the%20journal%20of%20the%20American%20Medical%20Association&title=Effect%20of%20clopidogrel%20pretreatment%20before%20percutaneous%20coronary%20intervention%20in%20patients%20with%20ST-elevation%20myocardial%20infarction%20treated%20with%20fibrinolytics:%20the%20PCI-CLARITY%20study.&author=MS%20Sabatine&volume=294&publication_year=2005&pages=1224-32&pmid=16143698&doi=10.1001/jama.294.10.1224&)

3. Steinhubl SR, et al. Early and sustained dual oral antiplatelet therapy following percutaneous coronary intervention: a randomized controlled trial. JAMA : the journal of the American Medical Association. 2002;288:2411–20. doi: 10.1001/jama.288.19.2411.  [DOI](https://doi.org/10.1001/jama.288.19.2411) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12435254/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA%20:%20the%20journal%20of%20the%20American%20Medical%20Association&title=Early%20and%20sustained%20dual%20oral%20antiplatelet%20therapy%20following%20percutaneous%20coronary%20intervention:%20a%20randomized%20controlled%20trial.&author=SR%20Steinhubl&volume=288&publication_year=2002&pages=2411-20&pmid=12435254&doi=10.1001/jama.288.19.2411&)

4. Gurbel PA, Bliden KP, Hiatt BL, O'Connor CM. Clopidogrel for coronary stenting: response variability, drug resistance, and the effect of pretreatment platelet reactivity. Circulation. 2003;107:2908–13. doi: 10.1161/01.CIR.0000072771.11429.83.  [DOI](https://doi.org/10.1161/01.CIR.0000072771.11429.83) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12796140/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circulation&title=Clopidogrel%20for%20coronary%20stenting:%20response%20variability,%20drug%20resistance,%20and%20the%20effect%20of%20pretreatment%20platelet%20reactivity.&author=PA%20Gurbel&author=KP%20Bliden&author=BL%20Hiatt&author=CM%20O'Connor&volume=107&publication_year=2003&pages=2908-13&pmid=12796140&doi=10.1161/01.CIR.0000072771.11429.83&)

5. Brar SS, et al. Impact of platelet reactivity on clinical outcomes after percutaneous coronary intervention. A collaborative meta-analysis of individual participant data. J Am Coll Cardiol. 2011;58:1945–54. doi: 10.1016/j.jacc.2011.06.059.  [DOI](https://doi.org/10.1016/j.jacc.2011.06.059) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22032704/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Am%20Coll%20Cardiol&title=Impact%20of%20platelet%20reactivity%20on%20clinical%20outcomes%20after%20percutaneous%20coronary%20intervention.%20A%20collaborative%20meta-analysis%20of%20individual%20participant%20data.&author=SS%20Brar&volume=58&publication_year=2011&pages=1945-54&pmid=22032704&doi=10.1016/j.jacc.2011.06.059&)

6. Cuisset T, et al. Clinical implications of very low on-treatment platelet reactivity in patients treated with thienopyridine: the POBA study (predictor of bleedings with antiplatelet drugs). JACC Cardiovasc Interv. 2013;6:854–63. doi: 10.1016/j.jcin.2013.04.009.  [DOI](https://doi.org/10.1016/j.jcin.2013.04.009) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23968703/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JACC%20Cardiovasc%20Interv&title=Clinical%20implications%20of%20very%20low%20on-treatment%20platelet%20reactivity%20in%20patients%20treated%20with%20thienopyridine:%20the%20POBA%20study%20(predictor%20of%20bleedings%20with%20antiplatelet%20drugs).&author=T%20Cuisset&volume=6&publication_year=2013&pages=854-63&pmid=23968703&doi=10.1016/j.jcin.2013.04.009&)

7. Mehran R, et al. A risk score to predict bleeding in patients with acute coronary syndromes. J Am Coll Cardiol. 2010;55:2556–66. doi: 10.1016/j.jacc.2009.09.076.  [DOI](https://doi.org/10.1016/j.jacc.2009.09.076) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20513595/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Am%20Coll%20Cardiol&title=A%20risk%20score%20to%20predict%20bleeding%20in%20patients%20with%20acute%20coronary%20syndromes.&author=R%20Mehran&volume=55&publication_year=2010&pages=2556-66&pmid=20513595&doi=10.1016/j.jacc.2009.09.076&)

8. Mauri L, et al. Twelve or 30 months of dual antiplatelet therapy after drug-eluting stents. N Engl J Med. 2014;371:2155–66. doi: 10.1056/NEJMoa1409312.  [DOI](https://doi.org/10.1056/NEJMoa1409312) | [PMC free article](/articles/PMC4481318/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25399658/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Twelve%20or%2030%20months%20of%20dual%20antiplatelet%20therapy%20after%20drug-eluting%20stents.&author=L%20Mauri&volume=371&publication_year=2014&pages=2155-66&pmid=25399658&doi=10.1056/NEJMoa1409312&)

9. Hulot JS, et al. Cytochrome P450 2C19 loss-of-function polymorphism is a major determinant of clopidogrel responsiveness in healthy subjects. Blood. 2006;108:2244–7. doi: 10.1182/blood-2006-04-013052.  [DOI](https://doi.org/10.1182/blood-2006-04-013052) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16772608/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Cytochrome%20P450%202C19%20loss-of-function%20polymorphism%20is%20a%20major%20determinant%20of%20clopidogrel%20responsiveness%20in%20healthy%20subjects.&author=JS%20Hulot&volume=108&publication_year=2006&pages=2244-7&pmid=16772608&doi=10.1182/blood-2006-04-013052&)

10. Collet JP, et al. Cytochrome P450 2C19 polymorphism in young patients treated with clopidogrel after myocardial infarction: a cohort study. Lancet. 2009;373:309–17. doi: 10.1016/S0140-6736(08)61845-0.  [DOI](https://doi.org/10.1016/S0140-6736(08)61845-0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19108880/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Cytochrome%20P450%202C19%20polymorphism%20in%20young%20patients%20treated%20with%20clopidogrel%20after%20myocardial%20infarction:%20a%20cohort%20study.&author=JP%20Collet&volume=373&publication_year=2009&pages=309-17&pmid=19108880&doi=10.1016/S0140-6736(08)61845-0&)

11. Cayla G, et al. Clinical, angiographic, and genetic factors associated with early coronary stent thrombosis. JAMA : the journal of the American Medical Association. 2011;306:1765–74. doi: 10.1001/jama.2011.1529.  [DOI](https://doi.org/10.1001/jama.2011.1529) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22028352/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA%20:%20the%20journal%20of%20the%20American%20Medical%20Association&title=Clinical,%20angiographic,%20and%20genetic%20factors%20associated%20with%20early%20coronary%20stent%20thrombosis.&author=G%20Cayla&volume=306&publication_year=2011&pages=1765-74&pmid=22028352&doi=10.1001/jama.2011.1529&)

12. Shuldiner AR, et al. Association of cytochrome P450 2C19 genotype with the antiplatelet effect and clinical efficacy of clopidogrel therapy. JAMA : the journal of the American Medical Association. 2009;302:849–57. doi: 10.1001/jama.2009.1232.  [DOI](https://doi.org/10.1001/jama.2009.1232) | [PMC free article](/articles/PMC3641569/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19706858/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA%20:%20the%20journal%20of%20the%20American%20Medical%20Association&title=Association%20of%20cytochrome%20P450%202C19%20genotype%20with%20the%20antiplatelet%20effect%20and%20clinical%20efficacy%20of%20clopidogrel%20therapy.&author=AR%20Shuldiner&volume=302&publication_year=2009&pages=849-57&pmid=19706858&doi=10.1001/jama.2009.1232&)

13. Brandt JT, et al. Common polymorphisms of CYP2C19 and CYP2C9 affect the pharmacokinetic and pharmacodynamic response to clopidogrel but not prasugrel. J Thromb Haemost. 2007;5:2429–36. doi: 10.1111/j.1538-7836.2007.02775.x.  [DOI](https://doi.org/10.1111/j.1538-7836.2007.02775.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17900275/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Thromb%20Haemost&title=Common%20polymorphisms%20of%20CYP2C19%20and%20CYP2C9%20affect%20the%20pharmacokinetic%20and%20pharmacodynamic%20response%20to%20clopidogrel%20but%20not%20prasugrel.&author=JT%20Brandt&volume=5&publication_year=2007&pages=2429-36&pmid=17900275&doi=10.1111/j.1538-7836.2007.02775.x&)

14. Kim KA, Park PW, Hong SJ, Park JY. The effect of CYP2C19 polymorphism on the pharmacokinetics and pharmacodynamics of clopidogrel: a possible mechanism for clopidogrel resistance. Clin Pharmacol Ther. 2008;84:236–42. doi: 10.1038/clpt.2008.20.  [DOI](https://doi.org/10.1038/clpt.2008.20) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18323861/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=The%20effect%20of%20CYP2C19%20polymorphism%20on%20the%20pharmacokinetics%20and%20pharmacodynamics%20of%20clopidogrel:%20a%20possible%20mechanism%20for%20clopidogrel%20resistance.&author=KA%20Kim&author=PW%20Park&author=SJ%20Hong&author=JY%20Park&volume=84&publication_year=2008&pages=236-42&pmid=18323861&doi=10.1038/clpt.2008.20&)

15. Mega JL, et al. Cytochrome p-450 polymorphisms and response to clopidogrel. N Engl J Med. 2009;360:354–62. doi: 10.1056/NEJMoa0809171.  [DOI](https://doi.org/10.1056/NEJMoa0809171) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19106084/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Cytochrome%20p-450%20polymorphisms%20and%20response%20to%20clopidogrel.&author=JL%20Mega&volume=360&publication_year=2009&pages=354-62&pmid=19106084&doi=10.1056/NEJMoa0809171&)

16. Frere C, Cuisset T, Gaborit B, Alessi MC, Hulot JS. The CYP2C19*17 allele is associated with better platelet response to clopidogrel in patients admitted for non-ST acute coronary syndrome. J Thromb Haemost. 2009;7:1409–11. doi: 10.1111/j.1538-7836.2009.03500.x.  [DOI](https://doi.org/10.1111/j.1538-7836.2009.03500.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19496924/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Thromb%20Haemost&title=The%20CYP2C19*17%20allele%20is%20associated%20with%20better%20platelet%20response%20to%20clopidogrel%20in%20patients%20admitted%20for%20non-ST%20acute%20coronary%20syndrome.&author=C%20Frere&author=T%20Cuisset&author=B%20Gaborit&author=MC%20Alessi&author=JS%20Hulot&volume=7&publication_year=2009&pages=1409-11&pmid=19496924&doi=10.1111/j.1538-7836.2009.03500.x&)

17. Sibbing D, et al. Isolated and interactive impact of common CYP2C19 genetic variants on the antiplatelet effect of chronic clopidogrel therapy. J Thromb Haemost. 2010 doi: 10.1111/j.1538-7836.2010.03921.x.  [DOI](https://doi.org/10.1111/j.1538-7836.2010.03921.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20492469/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Thromb%20Haemost&title=Isolated%20and%20interactive%20impact%20of%20common%20CYP2C19%20genetic%20variants%20on%20the%20antiplatelet%20effect%20of%20chronic%20clopidogrel%20therapy.&author=D%20Sibbing&publication_year=2010&pmid=20492469&doi=10.1111/j.1538-7836.2010.03921.x&)

18. Tiroch KA, et al. Protective effect of the CYP2C19 *17 polymorphism with increased activation of clopidogrel on cardiovascular events. Am Heart J. 2010;160:506–12. doi: 10.1016/j.ahj.2010.06.039.  [DOI](https://doi.org/10.1016/j.ahj.2010.06.039) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20826260/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20Heart%20J&title=Protective%20effect%20of%20the%20CYP2C19%20*17%20polymorphism%20with%20increased%20activation%20of%20clopidogrel%20on%20cardiovascular%20events.&author=KA%20Tiroch&volume=160&publication_year=2010&pages=506-12&pmid=20826260&doi=10.1016/j.ahj.2010.06.039&)

19. Marin F, Gonzalez-Conejero R, Capranzano P, Bass TA, Roldan V, Angiolillo DJ. Pharmacogenetics in cardiovascular antithrombotic therapy. J Am Coll Cardiol. 2009;54:1041–57. doi: 10.1016/j.jacc.2009.04.084.  [DOI](https://doi.org/10.1016/j.jacc.2009.04.084) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19744613/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Am%20Coll%20Cardiol&title=Pharmacogenetics%20in%20cardiovascular%20antithrombotic%20therapy.&author=F%20Marin&author=R%20Gonzalez-Conejero&author=P%20Capranzano&author=TA%20Bass&author=V%20Roldan&volume=54&publication_year=2009&pages=1041-57&pmid=19744613&doi=10.1016/j.jacc.2009.04.084&)

20. Yang Y, Lewis JP, Hulot JS, Scott SA. The pharmacogenetic control of antiplatelet response: candidate genes and CYP2C19. Expert opinion on drug metabolism & toxicology. 2015;11:1599–617. doi: 10.1517/17425255.2015.1068757.  [DOI](https://doi.org/10.1517/17425255.2015.1068757) | [PMC free article](/articles/PMC4829114/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26173871/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Expert%20opinion%20on%20drug%20metabolism%20&%20toxicology&title=The%20pharmacogenetic%20control%20of%20antiplatelet%20response:%20candidate%20genes%20and%20CYP2C19.&author=Y%20Yang&author=JP%20Lewis&author=JS%20Hulot&author=SA%20Scott&volume=11&publication_year=2015&pages=1599-617&pmid=26173871&doi=10.1517/17425255.2015.1068757&)

21. Arking DE, et al. A common genetic variant in the NOS1 regulator NOS1AP modulates cardiac repolarization. Nature genetics. 2006;38:644–51. doi: 10.1038/ng1790.  [DOI](https://doi.org/10.1038/ng1790) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16648850/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature%20genetics&title=A%20common%20genetic%20variant%20in%20the%20NOS1%20regulator%20NOS1AP%20modulates%20cardiac%20repolarization.&author=DE%20Arking&volume=38&publication_year=2006&pages=644-51&pmid=16648850&doi=10.1038/ng1790&)

22. Voora D, et al. A polymorphism in the VKORC1 regulator calumenin predicts higher warfarin dose requirements in African Americans. Clin Pharmacol Ther. 2010;87:445–51. doi: 10.1038/clpt.2009.291.  [DOI](https://doi.org/10.1038/clpt.2009.291) | [PMC free article](/articles/PMC2928561/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20200517/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=A%20polymorphism%20in%20the%20VKORC1%20regulator%20calumenin%20predicts%20higher%20warfarin%20dose%20requirements%20in%20African%20Americans.&author=D%20Voora&volume=87&publication_year=2010&pages=445-51&pmid=20200517&doi=10.1038/clpt.2009.291&)

23. Daneshjou R, et al. Genetic variant in folate homeostasis is associated with lower warfarin dose in African Americans. Blood. 2014;124:2298–305. doi: 10.1182/blood-2014-04-568436.  [DOI](https://doi.org/10.1182/blood-2014-04-568436) | [PMC free article](/articles/PMC4183989/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25079360/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Genetic%20variant%20in%20folate%20homeostasis%20is%20associated%20with%20lower%20warfarin%20dose%20in%20African%20Americans.&author=R%20Daneshjou&volume=124&publication_year=2014&pages=2298-305&pmid=25079360&doi=10.1182/blood-2014-04-568436&)

24. Emond MJ, et al. Exome sequencing of extreme phenotypes identifies DCTN4 as a modifier of chronic Pseudomonas aeruginosa infection in cystic fibrosis. Nature genetics. 2012;44:886–9. doi: 10.1038/ng.2344.  [DOI](https://doi.org/10.1038/ng.2344) | [PMC free article](/articles/PMC3702264/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22772370/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature%20genetics&title=Exome%20sequencing%20of%20extreme%20phenotypes%20identifies%20DCTN4%20as%20a%20modifier%20of%20chronic%20Pseudomonas%20aeruginosa%20infection%20in%20cystic%20fibrosis.&author=MJ%20Emond&volume=44&publication_year=2012&pages=886-9&pmid=22772370&doi=10.1038/ng.2344&)

25. Cohen JC, Kiss RS, Pertsemlidis A, Marcel YL, McPherson R, Hobbs HH. Multiple rare alleles contribute to low plasma levels of HDL cholesterol. Science;2004;305:869–72. doi: 10.1126/science.1099870.  [DOI](https://doi.org/10.1126/science.1099870) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15297675/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Cohen%20JC,%20Kiss%20RS,%20Pertsemlidis%20A,%20Marcel%20YL,%20McPherson%20R,%20Hobbs%20HH.%20Multiple%20rare%20alleles%20contribute%20to%20low%20plasma%20levels%20of%20HDL%20cholesterol.%20Science;2004;305:869%E2%80%9372.%20doi:%2010.1126/science.1099870.)

26. Cohen J, Pertsemlidis A, Kotowski IK, Graham R, Garcia CK, Hobbs HH. Low LDL cholesterol in individuals of African descent resulting from frequent nonsense mutations in PCSK9. Nature genetics. 2005;37:161–5. doi: 10.1038/ng1509.  [DOI](https://doi.org/10.1038/ng1509) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15654334/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature%20genetics&title=Low%20LDL%20cholesterol%20in%20individuals%20of%20African%20descent%20resulting%20from%20frequent%20nonsense%20mutations%20in%20PCSK9.&author=J%20Cohen&author=A%20Pertsemlidis&author=IK%20Kotowski&author=R%20Graham&author=CK%20Garcia&volume=37&publication_year=2005&pages=161-5&pmid=15654334&doi=10.1038/ng1509&)

27. Lange LA, et al. Whole-exome sequencing identifies rare and low-frequency coding variants associated with LDL cholesterol. American journal of human genetics. 2014;94:233–45. doi: 10.1016/j.ajhg.2014.01.010.  [DOI](https://doi.org/10.1016/j.ajhg.2014.01.010) | [PMC free article](/articles/PMC3928660/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24507775/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=American%20journal%20of%20human%20genetics&title=Whole-exome%20sequencing%20identifies%20rare%20and%20low-frequency%20coding%20variants%20associated%20with%20LDL%20cholesterol.&author=LA%20Lange&volume=94&publication_year=2014&pages=233-45&pmid=24507775&doi=10.1016/j.ajhg.2014.01.010&)

28. Linderman MD, et al. Analytical validation of whole exome and whole genome sequencing for clinical applications. BMC medical genomics. 2014;7:20. doi: 10.1186/1755-8794-7-20.  [DOI](https://doi.org/10.1186/1755-8794-7-20) | [PMC free article](/articles/PMC4022392/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24758382/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20medical%20genomics&title=Analytical%20validation%20of%20whole%20exome%20and%20whole%20genome%20sequencing%20for%20clinical%20applications.&author=MD%20Linderman&volume=7&publication_year=2014&pages=20&pmid=24758382&doi=10.1186/1755-8794-7-20&)

29. Wang M, et al. PaxDb, a database of protein abundance averages across all three domains of life. Molecular & cellular proteomics : MCP. 2012;11:492–500. doi: 10.1074/mcp.O111.014704.  [DOI](https://doi.org/10.1074/mcp.O111.014704) | [PMC free article](/articles/PMC3412977/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22535208/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Molecular%20&%20cellular%20proteomics%20:%20MCP&title=PaxDb,%20a%20database%20of%20protein%20abundance%20averages%20across%20all%20three%20domains%20of%20life.&author=M%20Wang&volume=11&publication_year=2012&pages=492-500&pmid=22535208&doi=10.1074/mcp.O111.014704&)

30. Qasba PK, Ramakrishnan B, Boeggeman E. Structure and function of beta -1,4- galactosyltransferase. Current drug targets. 2008;9:292–309. doi: 10.2174/138945008783954943.  [DOI](https://doi.org/10.2174/138945008783954943) | [PMC free article](/articles/PMC2365515/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18393823/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Current%20drug%20targets&title=Structure%20and%20function%20of%20beta%20-1,4-%20galactosyltransferase.&author=PK%20Qasba&author=B%20Ramakrishnan&author=E%20Boeggeman&volume=9&publication_year=2008&pages=292-309&pmid=18393823&doi=10.2174/138945008783954943&)

31. Silvain J, et al. High on-thienopyridine platelet reactivity in elderly coronary patients: the SENIOR-PLATELET study. Eur Heart J. 2012;33:1241–9. doi: 10.1093/eurheartj/ehr407.  [DOI](https://doi.org/10.1093/eurheartj/ehr407) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22067090/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20Heart%20J&title=High%20on-thienopyridine%20platelet%20reactivity%20in%20elderly%20coronary%20patients:%20the%20SENIOR-PLATELET%20study.&author=J%20Silvain&volume=33&publication_year=2012&pages=1241-9&pmid=22067090&doi=10.1093/eurheartj/ehr407&)

32. Baber U, et al. Combined and independent impact of diabetes mellitus and chronic kidney disease on residual platelet reactivity. Thrombosis and haemostasis. 2013;110:118–23. doi: 10.1160/TH13-01-0004.  [DOI](https://doi.org/10.1160/TH13-01-0004) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23677380/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Thrombosis%20and%20haemostasis&title=Combined%20and%20independent%20impact%20of%20diabetes%20mellitus%20and%20chronic%20kidney%20disease%20on%20residual%20platelet%20reactivity.&author=U%20Baber&volume=110&publication_year=2013&pages=118-23&pmid=23677380&doi=10.1160/TH13-01-0004&)

33. Johnson JA, Roden DM, Lesko LJ, Ashley E, Klein TE, Shuldiner AR. Clopidogrel: a case for indication-specific pharmacogenetics. Clin Pharmacol Ther. 2012;91:774–6. doi: 10.1038/clpt.2012.21.  [DOI](https://doi.org/10.1038/clpt.2012.21) | [PMC free article](/articles/PMC3382015/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22513313/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Clopidogrel:%20a%20case%20for%20indication-specific%20pharmacogenetics.&author=JA%20Johnson&author=DM%20Roden&author=LJ%20Lesko&author=E%20Ashley&author=TE%20Klein&volume=91&publication_year=2012&pages=774-6&pmid=22513313&doi=10.1038/clpt.2012.21&)

34. Hochholzer W, et al. Variability of individual platelet reactivity over time in patients treated with clopidogrel: insights from the ELEVATE-TIMI 56 trial. J Am Coll Cardiol. 2014;64:361–8. doi: 10.1016/j.jacc.2014.03.051.  [DOI](https://doi.org/10.1016/j.jacc.2014.03.051) | [PMC free article](/articles/PMC11284982/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25060370/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Am%20Coll%20Cardiol&title=Variability%20of%20individual%20platelet%20reactivity%20over%20time%20in%20patients%20treated%20with%20clopidogrel:%20insights%20from%20the%20ELEVATE-TIMI%2056%20trial.&author=W%20Hochholzer&volume=64&publication_year=2014&pages=361-8&pmid=25060370&doi=10.1016/j.jacc.2014.03.051&)

35. Bouman HJ, et al. Variability in on-treatment platelet reactivity explained by CYP2C19*2 genotype is modest in clopidogrel pretreated patients undergoing coronary stenting. Heart. 2011;97:1239–44. doi: 10.1136/hrt.2010.220509.  [DOI](https://doi.org/10.1136/hrt.2010.220509) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21628721/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Heart&title=Variability%20in%20on-treatment%20platelet%20reactivity%20explained%20by%20CYP2C19*2%20genotype%20is%20modest%20in%20clopidogrel%20pretreated%20patients%20undergoing%20coronary%20stenting.&author=HJ%20Bouman&volume=97&publication_year=2011&pages=1239-44&pmid=21628721&doi=10.1136/hrt.2010.220509&)

36. Almeida R, et al. A family of human beta4-galactosyltransferases. Cloning and expression of two novel UDP-galactose:beta-n-acetylglucosamine beta1, 4-galactosyltransferases, beta4Gal-T2 and beta4Gal-T3. The Journal of biological chemistry. 1997;272:31979–91. doi: 10.1074/jbc.272.51.31979.  [DOI](https://doi.org/10.1074/jbc.272.51.31979) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9405390/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=The%20Journal%20of%20biological%20chemistry&title=A%20family%20of%20human%20beta4-galactosyltransferases.%20Cloning%20and%20expression%20of%20two%20novel%20UDP-galactose:beta-n-acetylglucosamine%20beta1,%204-galactosyltransferases,%20beta4Gal-T2%20and%20beta4Gal-T3.&author=R%20Almeida&volume=272&publication_year=1997&pages=31979-91&pmid=9405390&doi=10.1074/jbc.272.51.31979&)

37. Ahmad I, et al. Phosphorylation and glycosylation interplay: protein modifications at hydroxy amino acids and prediction of signaling functions of the human beta3 integrin family. Journal of cellular biochemistry. 2006;99:706–18. doi: 10.1002/jcb.20814.  [DOI](https://doi.org/10.1002/jcb.20814) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16676352/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20of%20cellular%20biochemistry&title=Phosphorylation%20and%20glycosylation%20interplay:%20protein%20modifications%20at%20hydroxy%20amino%20acids%20and%20prediction%20of%20signaling%20functions%20of%20the%20human%20beta3%20integrin%20family.&author=I%20Ahmad&volume=99&publication_year=2006&pages=706-18&pmid=16676352&doi=10.1002/jcb.20814&)

38. Rosa JP, McEver RP. Processing and assembly of the integrin, glycoprotein IIb-IIIa, in HEL cells. The Journal of biological chemistry. 1989;264:12596–603.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/2501308/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=The%20Journal%20of%20biological%20chemistry&title=Processing%20and%20assembly%20of%20the%20integrin,%20glycoprotein%20IIb-IIIa,%20in%20HEL%20cells.&author=JP%20Rosa&author=RP%20McEver&volume=264&publication_year=1989&pages=12596-603&pmid=2501308&)

39. 56th ASH Annual Meeting and Exposition. 2014. Posttranslational Modification in Megakaryocytes Regulates β1 Integrin Expression, Migration and Platelet Production in Vivo; Abstract 1435.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=56th%20ASH%20Annual%20Meeting%20and%20Exposition&publication_year=2014&)

40. Hebsgaard SM, Korning PG, Tolstrup N, Engelbrecht J, Rouze P, Brunak S. Splice site prediction in Arabidopsis thaliana pre-mRNA by combining local and global sequence information. Nucleic acids research. 1996;24:3439–52. doi: 10.1093/nar/24.17.3439.  [DOI](https://doi.org/10.1093/nar/24.17.3439) | [PMC free article](/articles/PMC146109/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8811101/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20acids%20research&title=Splice%20site%20prediction%20in%20Arabidopsis%20thaliana%20pre-mRNA%20by%20combining%20local%20and%20global%20sequence%20information.&author=SM%20Hebsgaard&author=PG%20Korning&author=N%20Tolstrup&author=J%20Engelbrecht&author=P%20Rouze&volume=24&publication_year=1996&pages=3439-52&pmid=8811101&doi=10.1093/nar/24.17.3439&)

41. Desmet FO, Hamroun D, Lalande M, Collod-Beroud G, Claustres M, Beroud C. Human Splicing Finder: an online bioinformatics tool to predict splicing signals. Nucleic acids research. 2009;37:e67. doi: 10.1093/nar/gkp215.  [DOI](https://doi.org/10.1093/nar/gkp215) | [PMC free article](/articles/PMC2685110/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19339519/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20acids%20research&title=Human%20Splicing%20Finder:%20an%20online%20bioinformatics%20tool%20to%20predict%20splicing%20signals.&author=FO%20Desmet&author=D%20Hamroun&author=M%20Lalande&author=G%20Collod-Beroud&author=M%20Claustres&volume=37&publication_year=2009&pages=e67&pmid=19339519&doi=10.1093/nar/gkp215&)

42. Sorensen AL, et al. Role of sialic acid for platelet life span: exposure of beta-galactose results in the rapid clearance of platelets from the circulation by asialoglycoprotein receptor-expressing liver macrophages and hepatocytes. Blood. 2009;114:1645–54. doi: 10.1182/blood-2009-01-199414.  [DOI](https://doi.org/10.1182/blood-2009-01-199414) | [PMC free article](/articles/PMC2731641/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19520807/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Role%20of%20sialic%20acid%20for%20platelet%20life%20span:%20exposure%20of%20beta-galactose%20results%20in%20the%20rapid%20clearance%20of%20platelets%20from%20the%20circulation%20by%20asialoglycoprotein%20receptor-expressing%20liver%20macrophages%20and%20hepatocytes.&author=AL%20Sorensen&volume=114&publication_year=2009&pages=1645-54&pmid=19520807&doi=10.1182/blood-2009-01-199414&)

43. Mandic R, Opper C, Krappe J, Wesemann W. Platelet sialic acid as a potential pathogenic factor in coronary heart disease. Thromb Res. 2002;106:137–41. doi: 10.1016/s0049-3848(02)00087-7.  [DOI](https://doi.org/10.1016/s0049-3848(02)00087-7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12182913/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Thromb%20Res&title=Platelet%20sialic%20acid%20as%20a%20potential%20pathogenic%20factor%20in%20coronary%20heart%20disease.&author=R%20Mandic&author=C%20Opper&author=J%20Krappe&author=W%20Wesemann&volume=106&publication_year=2002&pages=137-41&pmid=12182913&doi=10.1016/s0049-3848(02)00087-7&)

44. Sim SC, Ingelman-Sundberg M. The Human Cytochrome P450 (CYP) Allele Nomenclature website: a peer-reviewed database of CYP variants and their associated effects. Hum Genomics. 2010;4:278–81. doi: 10.1186/1479-7364-4-4-278.  [DOI](https://doi.org/10.1186/1479-7364-4-4-278) | [PMC free article](/articles/PMC3525213/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20511141/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Genomics&title=The%20Human%20Cytochrome%20P450%20(CYP)%20Allele%20Nomenclature%20website:%20a%20peer-reviewed%20database%20of%20CYP%20variants%20and%20their%20associated%20effects.&author=SC%20Sim&author=M%20Ingelman-Sundberg&volume=4&publication_year=2010&pages=278-81&pmid=20511141&doi=10.1186/1479-7364-4-4-278&)

45. Exome Aggregation Consortium (ExAC) Cambridge, MA: Dec, 2014. URL: http://exac.broadinstitute.org accessed.  [http://exac.broadinstitute.org](http://exac.broadinstitute.org) | [Google Scholar](https://scholar.google.com/scholar_lookup?title=Exome%20Aggregation%20Consortium%20(ExAC)&publication_year=2014&)

46. Exome Variant Server NHLBI. GO Exome Sequencing Project (ESP) Seattle, WA: Dec, 2014. URL: http://evs.gs.washington.edu/EVS/ accessed.  [http://evs.gs.washington.edu/EVS/](http://evs.gs.washington.edu/EVS/) | [Google Scholar](https://scholar.google.com/scholar_lookup?title=GO%20Exome%20Sequencing%20Project%20(ESP)&author=NHLBI%20Exome%20Variant%20Server&publication_year=2014&)

47. Robinson JT, et al. Integrative genomics viewer. Nature biotechnology. 2011;29:24–6. doi: 10.1038/nbt.1754.  [DOI](https://doi.org/10.1038/nbt.1754) | [PMC free article](/articles/PMC3346182/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21221095/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature%20biotechnology&title=Integrative%20genomics%20viewer.&author=JT%20Robinson&volume=29&publication_year=2011&pages=24-6&pmid=21221095&doi=10.1038/nbt.1754&)

48. Wang J, et al. High-throughput single nucleotide polymorphism genotyping using nanofluidic Dynamic Arrays. BMC genomics. 2009;10:561. doi: 10.1186/1471-2164-10-561.  [DOI](https://doi.org/10.1186/1471-2164-10-561) | [PMC free article](/articles/PMC2789104/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19943955/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20genomics&title=High-throughput%20single%20nucleotide%20polymorphism%20genotyping%20using%20nanofluidic%20Dynamic%20Arrays.&author=J%20Wang&volume=10&publication_year=2009&pages=561&pmid=19943955&doi=10.1186/1471-2164-10-561&)
