# Genome-wide analysis identifies novel loci associated with ovarian cancer outcomes: findings from the Ovarian Cancer Association Consortium

## Metadata
**Authors:** Sharon E Johnatty, Jonathan P Tyrer, Siddhartha Kar, Jonathan Beesley, Yi Lu, Bo Gao, Peter A Fasching, Alexander Hein, Arif B Ekici, Matthias W Beckmann, Diether Lambrechts, Els Van Nieuwenhuysen, Ignace Vergote, Sandrina Lambrechts, Mary Anne Rossing, Jennifer A Doherty, Jenny Chang-Claude, Francesmary Modugno, Roberta B Ness, Kirsten B Moysich, Douglas A Levine, Lambertus A Kiemeney, Leon FAG Massuger, Jacek Gronwald, Jan Lubiński, Anna Jakubowska, Cezary Cybulski, Louise Brinton, Jolanta Lissowska, Nicolas Wentzensen, Honglin Song, Valerie Rhenius, Ian Campbell, Diana Eccles, Weiva Sieh, Alice S Whittemore, Valerie McGuire, Joseph H Rothstein, Rebecca Sutphen, Hoda Anton-Culver, Argyrios Ziogas, Simon A Gayther, Aleksandra Gentry-Maharaj, Usha Menon, Susan J Ramus, Celeste L Pearce, Malcolm C Pike, Daniel O Stram, Anna H Wu, Jolanta Kupryjanczyk, Agnieszka Dansonka-Mieszkowska, Iwona K Rzepecka, Beata Spiewankiewicz, Marc T Goodman, Lynne R Wilkens, Michael E Carney, Pamela J Thompson, Florian Heitz, Andreas du Bois, Ira Schwaab, Philipp Harter, Jacobus Pisterer, Peter Hillemanns, Beth Y Karlan, Christine Walsh, Jenny Lester, Sandra Orsulic, Stacey J Winham, Madalene Earp, Melissa C Larson, Zachary C Fogarty, Estrid Høgdall, Allan Jensen, Susanne Kruger Kjaer, Brooke L Fridley, Julie M Cunningham, Robert A Vierkant, Joellen M Schildkraut, Edwin S Iversen, Kathryn L Terry, Daniel W Cramer, Elisa V Bandera, Irene Orlow, Tanja Pejovic, Yukie Bean, Claus Høgdall, Lene Lundvall, Ian McNeish, James Paul, Karen Carty, Nadeem Siddiqui, Rosalind Glasspool, Thomas Sellers, Catherine Kennedy, Yoke-Eng Chiew, Andrew Berchuck, Stuart MacGregor, Anna deFazio, Paul DP Pharoah, Ellen L Goode
**Journal:** Clinical cancer research : an official journal of the American Association for Cancer Research
**Date:** 2015 Jul 7
**DOI:** [10.1158/1078-0432.CCR-15-0632](https://doi.org/10.1158/1078-0432.CCR-15-0632)
**PMID:** 26152742
**PMCID:** PMC4624261
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4624261/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC4624261/pdf/nihms706347.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC4624261/pdf/nihms706347.pdf)

## Abstract

**Purpose:** 
Chemotherapy resistance remains a major challenge in the treatment of ovarian cancer. We hypothesize that germline polymorphisms might be associated with clinical outcome.

**Experimental Design:** 
We analyzed ~2.8 million genotyped and imputed SNPs from the iCOGS experiment for progression-free survival (PFS) and overall survival (OS) in 2,901 European epithelial ovarian cancer (EOC) patients who underwent firstline treatment of cytoreductive surgery and chemotherapy regardless of regimen, and in a subset of 1,098 patients treated with ≥4 cycles of paclitaxel and carboplatin at standard doses. We evaluated the top SNPs in 4,434 EOC patients including patients from The Cancer Genome Atlas. Additionally we conducted pathway analysis of all intragenic SNPs and tested their association with PFS and OS using gene set enrichment analysis.

**Results:** 
Five SNPs were significantly associated (p≤1.0x10−5) with poorer outcomes in at least one of the four analyses, three of which, rs4910232 (11p15.3), rs2549714 (16q23) and rs6674079 (1q22) were located in long non-coding RNAs (lncRNAs) RP11–179A10.1, RP11–314O13.1 and RP11–284F21.8 respectively (p≤7.1x10−6). ENCODE ChIP-seq data at 1q22 for normal ovary shows evidence of histone modification around RP11–284F21.8, and rs6674079 is perfectly correlated with another SNP within the super-enhancer MEF2D, expression levels of which were reportedly associated with prognosis in another solid tumor. YAP1- and WWTR1 (TAZ)-stimulated gene expression, and HDL-mediated lipid transport pathways were associated with PFS and OS, respectively, in the cohort who had standard chemotherapy (pGSEA≤6x10−3).

**Conclusion:** 
We have identified SNPs in three lncRNAs that might be important targets for novel EOC therapies.

Keywords: progression-free survival, overall survival, epithelial ovarian cancer, lncRNA, chemotherapy

### Purpose

Chemotherapy resistance remains a major challenge in the treatment of ovarian cancer. We hypothesize that germline polymorphisms might be associated with clinical outcome.

### Experimental Design

We analyzed ~2.8 million genotyped and imputed SNPs from the iCOGS experiment for progression-free survival (PFS) and overall survival (OS) in 2,901 European epithelial ovarian cancer (EOC) patients who underwent firstline treatment of cytoreductive surgery and chemotherapy regardless of regimen, and in a subset of 1,098 patients treated with ≥4 cycles of paclitaxel and carboplatin at standard doses. We evaluated the top SNPs in 4,434 EOC patients including patients from The Cancer Genome Atlas. Additionally we conducted pathway analysis of all intragenic SNPs and tested their association with PFS and OS using gene set enrichment analysis.

### Results

Five SNPs were significantly associated (p≤1.0x10^−5^) with poorer outcomes in at least one of the four analyses, three of which, rs4910232 (11p15.3), rs2549714 (16q23) and rs6674079 (1q22) were located in long non-coding RNAs (lncRNAs) RP11–179A10.1, RP11–314O13.1 and RP11–284F21.8 respectively (p≤7.1x10^−6^). ENCODE ChIP-seq data at 1q22 for normal ovary shows evidence of histone modification around RP11–284F21.8, and rs6674079 is perfectly correlated with another SNP within the super-enhancer MEF2D, expression levels of which were reportedly associated with prognosis in another solid tumor. YAP1- and WWTR1 (TAZ)-stimulated gene expression, and HDL-mediated lipid transport pathways were associated with PFS and OS, respectively, in the cohort who had standard chemotherapy (p_GSEA_≤6x10^−3^).

### Conclusion

We have identified SNPs in three lncRNAs that might be important targets for novel EOC therapies.

## Introduction

Approximately 238,000 women are diagnosed with ovarian cancer each year. It is the leading cause of death from gynecological cancers and globally approximately 152,000 women will die annually from the disease ([1](#R1)). Over the past three decades, significant advances have been made in chemotherapy for epithelial ovarian cancer (EOC), and the combination of cytoreductive surgery followed by the doublet of a taxane (paclitaxel 135 – 175 mg/m^2^) and platinum (carboplatin AUC > 5) repeated every three weeks has been the most common regimen for primary treatment of this disease, with initial tumor response rates ranging from 70–80% ([2](#R2), [3](#R3)). Although survival rates have improved in the past decade, resistance to chemotherapy remains a major challenge, and the majority of patients with advanced disease succumb to the disease despite initial response to first line treatment ([4](#R4)). The identification of genes relevant to response to chemotherapy and survival of ovarian cancer may contribute to a better understanding of prognosis, and potentially guide the selection of treatment options to help circumvent this obstacle.

It is well recognized that genetic variation can have a direct effect on inter-individual variation in drug responses, although patient response to medication is dependent on multiple factors ranging from patient age, disease type, organ functions, concomitant therapy and drug interactions ([5](#R5)). Comparisons of intra-patient and inter-patient variability in both population-based and twin studies have demonstrated that the smallest differences in drug metabolism and their effects are between monozygotic twins, which is consistent with the hypothesis that genetics may play a significant role in drug responses ([6](#R6), [7](#R7)). While many cancer treatments have been successful in shrinking or eradicating tumor cells, studies of genetic factors related to drug responses are particularly challenging because tumor cell and the non-cancerous host tissue from which they arise share the same genetic background, and failure of treatment may be due to the presence of *de novo* or acquired somatic alterations in tumors rather than germline variation ([8](#R8)).

To date several candidate gene studies have explored germline polymorphisms for an association with response to chemotherapy for ovarian cancer ([9](#R9)). Some obvious candidates are genes that encode drug-metabolizing enzymes and drug transporters that can influence toxicity or treatment response. The most clinically relevant drug metabolising enzymes are member of the cytochrome P450 (CYP) superfamily, of which CYP1, CYP2, and CYP3 contribute to the metabolism of more than 90% of clinically used drugs. There is considerable evidence that polymorphisms in the CYP genes have a significant impact on drug disposition and response, and >60% of Food and Drug Administration (FDA)-approved drug labels regarding genomic biomarkers pertain to polymorphisms in the CYP enzymes ([10](#R10)). Similarly the *ABCB1* gene, the most extensively studied ATP-binding cassette (ABC) transporter involved in transport of a wide range of anti-cancer drugs including paclitaxel ([11](#R11)), was previously shown to be associated with response to first-line paclitaxel-based chemotherapy regimens for ovarian cancer ([12](#R12), [13](#R13)). A systematic review of the most commonly evaluated genes in gynecologic cancers, including *ABCB1*, showed inconsistent findings across studies ([14](#R14)). Other studies including a comprehensive study of *ABCB1* SNPs putatively associated with progression-free survival (PFS) undertaken by the Ovarian Cancer Association Consortium (OCAC) did not replicate the association with PFS, although the possibility of subtle effects from one SNP on overall survival (OS) could not be discounted ([13](#R13)). Recently several ABCA transporters were explored in expression studies using cell-based models and shown to be associated with outcome in serous EOC patients ([15](#R15)), although this finding would need to be replicated in a larger independent study.

However, inter-individual variation in response to chemotherapy and post-treatment outcomes cannot be fully explained by genetic variations in the genes encoding drug metabolizing enzymes, transporters, or drug targets. Recent studies by the OCAC and the Australian Ovarian Cancer Study (AOCS) found that EOC patients carrying *BRCA1* or *BRCA2* germline mutations had better response to treatment and better short-term survival (5 years) than non-carriers ([16](#R16), [17](#R17)). This survival advantage is supported by *in vitro* studies of *BRCA1/2* mutated ovarian cancer cell lines that were shown to be more sensitive to platinum-based chemotherapy ([18](#R18), [19](#R19)). Genome-wide approaches that integrate SNP genotypes, drug-induced cytotoxicity in cell lines and gene expression data have been proposed as models for identifying predictors of treatment outcome ([20](#R20)), although their utility when applied to patient data proved inconclusive ([21](#R21)).

While *in vitro* studies have suggested functional relevance for genes and associated SNPs, the clinical utility of these findings remains in question mainly due to inconsistent results from under-powered and heterogeneous patient studies. In this report we present the findings from a comprehensive large-scale analysis of ~2.8 million genotyped and imputed SNPs from the Collaborative Oncological Gene-environment Study (COGS) project in relation to progression-free and overall survival as surrogate markers of response to chemotherapy in ~3,000 EOC patients with detailed first-line chemotherapy and follow-up data from the OCAC. In a secondary analysis, we also evaluated the association between OS and ~2.8 million SNPs in ~11,000 EOC patients irrespective of treatment regimen.

## Materials and methods

### Study Populations

The main analysis was restricted to invasive EOC patients with detailed chemotherapy and clinical follow-up for disease progression and survival following first-line treatment from thirteen OCAC studies in the initial phase, with an additional four OCAC studies and patients from The Cancer Genome Atlas (TCGA) included in the validation phase ([Supplementary Table 1](#SD1)). Patients were included if they received a minimum of cytoreductive surgery as part of primary treatment, and were of European ancestry, determined using the program LAMP ([22](#R22)) to assign intercontinental ancestry based upon a set of unlinked markers also used to perform principal component (PC) analysis within each major population subgroup ([23](#R23)). A total of 2,901 patients were eligible for the main analysis, a subset of whom (n=1,098) were treated with ≥4 cycles of standard doses of paclitaxel and carboplatin intravenously (IV) at 3-weekly intervals. Clinical definitions and criteria for progression across studies have been previously described ([13](#R13)). Data from TCGA ([http://cancergenome.nih.gov/](http://cancergenome.nih.gov/)) was downloaded through the TCGA data portal and assessed for ancestral outliers to determine those of European descent. A secondary analysis of OS in ~11,000 European EOC patients was also done using patients from 30 OCAC studies ([Supplementary Table 2](#SD1)). All studies received approval from their respective human research ethics committees, and all OCAC participants provided written informed consent.

### Genotyping and imputation

The Collaborative Oncological Gene-environment Study (COGS) and two ovarian cancer GWAS have been described in detail elsewhere ([24](#R24)). Briefly, 211,155 SNPs were genotyped in germline DNA from cases and controls from 43 studies participating in OCAC using a custom Illumina Infinium iSelect array (iCOGS) designed to evaluate genetic variants for association with risk of breast, ovarian and prostate cancers. In addition, two new ovarian cancer GWAS were included which used Illumina 2.5M and Illumina OmniExpress arrays. Genotypes were imputed to the European subset of the phased chromosomes from the 1000 Genome project (version 3). Approximately 8 million SNPs with a minor allele frequency (MAF) of at least 0.02 and an imputation r^2^>0.3 were available for analysis, ~2.8 million of which were well imputed (imputation r^2^ ≥0.9) and were retained in survival analyses. DNA extraction, iPLEX genotyping methods and quality assurance for additional samples genotyped for the validation analysis have also been previously described ([25](#R25)).

### Statistical Analysis

The main analyses were the association between ~2.8 million SNPs and progression-free survival (PFS) and overall survival (OS). Analyses of PFS and OS were conducted separately for all patients known to have had a minimum of cytoreductive surgery for first-line treatment regardless of chemotherapy, hereafter referred to as the ‘all chemo’ analysis, and in a subset of patients known to have received standard of care first-line treatment of cytoreductive surgery and ≥4 cycles of paclitaxel and carboplatin IV at 3-weekly intervals, hereafter referred to as the ‘standard chemo’ subgroup ([Supplementary Table 1](#SD1)). The majority of patients in the ‘standard chemo’ cohort were known to have had paclitaxel at 175 or 135 mg/m^2^ and carboplatin AUC 5 or 6; for the remainder, standard dose was assumed based on treatment schedules. PFS was defined as the interval between the date of histological diagnosis and the first confirmed sign of disease progression or death, as previously described ([13](#R13)); OS was the interval between the date of histological diagnosis and death from any cause. Patients who had an interval of >12 months between the date of histological diagnosis and DNA collection were excluded from the analysis to avoid survival bias. A secondary analysis was OS in the largest available dataset of European invasive EOC patients regardless of treatment (n=11,311), hereafter referred to ‘all OCAC’.

For the main analysis of PFS and OS in ‘all chemo’ and ‘standard chemo’, we obtained the per-allele hazard ratio [log(HR)] and standard error for each SNP using Cox regression models including study, the first two PCs, residual disease (nil vs. any), tumor stage (FIGO stages I-IV), histology (5 subtypes), tumor grade (low vs. high), and age at diagnosis (OS analysis only) as covariates. To avoid inflation for rare SNPs, the likelihood ratio test was used to estimate the standard error for iCOGS SNPs and meta-analyzed with samples included in the US GWAS and U19 studies based on expected imputation accuracy for imputed SNPs. For secondary analysis of OS in the ‘all OCAC’ dataset, Cox regression models included study, age, and the first two PCs and histology as covariates. For the US GWAS and U19 studies, the principal components were estimated separately and the top two and top principal components used respectively. All tests for association were two-tailed and performed using in-house software programmed in C++ and STATA SE v. 11 (Stata Corp., USA). Manhattan and QQ plots were generated using the R project for Statistical Computing version 3.0.1 ([http://www.r-project.org/](http://www.r-project.org/)), and meta-analysis was done using the program Metal ([26](#R26)), and between-study heterogeneity was assessed using the likelihood ratio test to compare regression models with and without a genotype-by-study interaction term.

### SNP selection for validation

Preliminary analyses suggested that dosage scores from imputed SNPs with imputation r^2^ <0.9 were not representative of actual genotypes in this sample (Supplementary Methods & [Supplementary Table 3](#SD1)). We therefore selected SNPs with imputation r^2^ ≥0.9 and adjusted p≤1.0x10^−5^ in at least one of the four main analyses (PFS and OS in ‘all chemo’ and ‘standard chemo’) for genotype validation. SNPs were binned into LD blocks defined by pairwise correlation (r^2^) > 0.8. We used Sequenom Assay Designer 4.0 to design two multiplexes in order to capture at least one SNP representing each block, although some blocks contained SNPs for which an iPLEX assay could not be designed (n=10). All patients for whom we had DNA, clinical follow-up and chemotherapy data were genotyped. We then meta-analyzed estimates from the genotyped samples with non-overlapping iCOGS samples and TCGA data to obtain effect estimates from the largest possible dataset. SNPs that were significant at p≤1.0x10^−5^ in at least one outcome in the final analysis were queried for association with expression of protein-coding genes within 1Mb of the lead SNP using GEO, EGA and TCGA expression array data analyzed in KM-plotter ([27](#R27)).

### Pathway analysis

All intragenic SNPs of the ~8 million (MAF ≥ 0.02 and imputation r^2^>0.3) with p-values for association with PFS and OS in the ‘standard chemo’ cohort were mapped to 25,004 genes annotated with hg19 start and end positions. The boundaries of each gene were extended by 50 kb on both sides for SNP-to-gene mapping to include *cis*-regulatory variation. A total of 23,490 genes were captured by at least one SNP. The negative logarithm (base 10) of the p-value of the most significant SNP in each gene, adjusted for the number of SNPs in the gene (±50 kb) by a modification of the Sidak correction ([28](#R28), [29](#R29)) was used to rank genes based on their association with PFS and OS (‘standard chemo’). A total of 837 known biological pathways (containing between 15 to 500 genes each) from the Kyoto Encyclopedia of Genes and Genomes (KEGG), BioCarta, and Reactome, three standard expert-curated pathway repositories, were accessed via the Molecular Signatures Database (version 4.0; [http://www.broadinstitute.org/gsea/msigdb](http://www.broadinstitute.org/gsea/msigdb)). The pathways were tested for their association with PFS and OS using gene set enrichment analysis (GSEA) run to 1,000 permutations ([30](#R30)). Specifically, we applied the “preranked” GSEA algorithm with default settings and the original GSEA implementation of correction for testing multiple pathways using false discovery (FDR) and familywise error rates (FWER). The genes in each pathway driving the GSEA signal (core genes) were defined as described previously ([30](#R30)).

## Results

### SNP associations

An overview of the analytic approaches in this study is provided in [Supplementary Figure 1](#SD1). There were 158 and 236 SNPs in analysis of OS in ‘all chemo’ and ‘standard chemo’ respectively, and 107 and 252 SNPs in analysis of PFS in ‘all chemo’ and ‘standard chemo’ that were above the minimal p-value threshold for suggestive significance (p=1.0x10^−5^) but none reached the nominal level of genome-wide significance (p= 5x10^−8^; [Figure 1](#F1)). QQ plots and estimates of inflation of the test statistic (λ) revealed some inflation (λ ≤1.15; Supp. [Figure 2](#F2)) which could not be accounted for by SNPs with low MAF (<0.1). Manhattan and QQ plots for the ‘all OCAC’ OS analysis showed similar effects ([Supplementary Figure 3](#SD1)). We selected 130 iCOGS SNPs with imputation r^2^ ≥0.9 and adjusted p≤1.0x10^−5^ in at least one of the four analyses ([Supplementary Table 4](#SD1)), and genotyped 48 SNPs at 22 loci in all patients with chemotherapy and outcome data. To obtain effect estimates from the largest possible sample for PFS and OS in ‘all chemo’ and ‘standard chemo’ for these 48 SNPs, we meta-analyzed estimates from iPLEX genotyped samples (n=3,303), iCOGS imputed data on non-overlapping samples (n=821), and TCGA data (n=310; [Supplementary Table 5](#SD1)).

### Figure 1. Manhattan plots of ~2.8 million SNPs in four analyses of the cohort selected for first-line chemotherapy.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/85c7/4668218/29acd56eee92/nihms706347f1.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4668218_nihms706347f1.jpg)

SNPs with MAF ≥0.02 and imputation r2 ≥0.9 associated with Overall Survival in A. ‘All Chemo’ and B. ‘Standard chemo’, and Progression-free survival in C. ‘All chemo’ and D. ‘Standard chemo’; the blue line represents suggestive significance (p=1x10−5) and the red line represents genome wide significance (p=5x10−8).

### Figure 2. Progression-free survival in ‘all chemo’ analysis for rs4910232.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/85c7/4668218/e4bb91eb74a5/nihms706347f2a.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4668218_nihms706347f2a.jpg)

A. Kaplan Meier curve for PFS in ‘all chemo’ dataset (n=3,177); P-values derived from adjusted Cox PH models of genotyped samples; 0=common homozygotes AA, 1=heterozygotes AG, 2=rare homozygotes GG. B. Forest plot showing site-specific estimates for PFS and rs4910232 in ‘all chemo’ dataset.

Estimates for the most promising SNPs from meta-analysis (p≤1.0x10^−5^ in at least one of the four analyses) are summarized in [Table 1](#T1). The strongest association was for rs4910232 at 11p15.3 and PFS in the ‘all chemo’ analysis (HR=1.17, 95% CI 1.10–1.24; p=4.7x10^−7^). The Kaplan Meier (KM) plot of genotyped samples for rs4910232 showed a significant trend in worse PFS associated with each additional minor allele ([Figure 2A](#F2)) and there was no evidence of between-study heterogeneity (p= 0.7, [Figure 2B](#F2)). This SNP lies within the long non-coding RNA (lncRNA) RP11–179A10.1. Two other SNPs, rs2549714 at 16q23 and rs6674079 at 1q22 were associated with worse OS in ‘standard chemo’ (p=5.0x10^−6^) and ‘all chemo’ analyses (p=7.1x10^−6^) respectively, and are also located in lncRNAs ([Table 1](#T1)). We further explored SNPs within a 1Mb region of rs6674079 at the 1q22 locus using ENCODE ChiP-Seq data and found that rs6674079 is perfectly correlated with rs11264489 which lies within the super-enhancer MEF2D. Histone modification tracks from ENCODE for normal ovarian cancer cell lines suggest a strong regulatory potential for this SNP ([Figure 3](#F3)). The KM plot for rs6674079 clearly showed a significant per-allele trend in worse OS ([Figure 4A](#F4)) and study-specific estimates and heterogeneity tests showed no evidence of between-study heterogeneity (p=0.4, [Figure 4B](#F4)). Forest plots for other significant SNPs (rs7950311, rs2549714 and rs3795247) showed an overall trend in worse survival probabilities per minor allele ([Supplementary Figure 4A–C](#SD1)) and there was no evidence of between-study heterogeneity for any of these SNPs (p≥0.14).

### Table 1.

|   |   |   |   |   |   | OVERALL SURVIVAL |  |  |  | PROGRESSION-FREE SURVIVAL |  |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
|   |   |   |   |   |   |   |  |  |  |  |  |  |  |
|   |   |   |   |   |   | All Chemo (N=4,426) |  | Standard Chemo (N=1,799) |  | All Chemo (N=4,095) |  | Standard Chemo(N=1,598) |  |
|   |   |   |   |   |   |   |  |  |  |  |  |  |  |
| SNP | Chr | Position | Nearest Gene | Effect/RefAllele | aEffectAlleleFreq. | bHR (95% CI) | P | bHR (95% CI) | P | bHR (95% CI) | P | bHR (95% CI) | P |
| rs6674079 | 1q22 | 156486061 | RP11–284F21.8 | G/A | 0.28 | 1.15 (1.08 −1.23) | 7.1x10−6 | 1.07 (0.97–1.18) | 1.9x10−1 | 1.07 (1.01–1.13) | 2.8x10−2 | 0.98 (0.90–1.07) | 6.8x10−1 |
| rs7950311 | 11p15.4 | 5672354 | HBG2 | C/T | 0.48 | 1.10 (1.04–1.07 1.17) | 1.7x10−3 | 1.28 (1.16–1.42) | 6.8x10−7 | 1.03 (0.98–1.09) | 2.5 x10−1 | 1.08 (0.99–1.18) | 7.8x10−2 |
| rs4910232 | 11p15.3 | 11120369 | RP11-179A10.1 | G/T | 0.32 | 1.12 (1.05–1.19) | 9.4x10−4 | 1.20 (1.08–1.33) | 5.3x10−4 | 1.17 (1.10–1.24) | 4.7x10−7 | 1.24 (1.12–1.56) | 1.2x10−5 |
| rs2549714 | 16q23 | 80875263 | RP11–314O13.1 | C/A | 0.06 | 1.20 (1.06–1.36) | 3.4x10−3 | 1.53 (1.28–1.84) | 5.0x10−6 | 1.14 (1.01–1.28) | 2.8 x10−2 | 1.29 (1.08–1.55) | 5.6x10−3 |
| rs3795247 | 19p12 | 21906428 | ZNF100 | C/T | 0.08 | 1.16 (1.04–1.30) | 8.8x10−3 | 1.34 (1.13–1.60) | 9.7x10−4 | 1.26 (1.14–1.40) | 1.05x10−5 | 1.39 (1.18–1.65) | 9.2x10−5 |

Table 1 Caption: Results of meta-analysis of estimates from iPLEX genotyped, non-overlapping iCOGS and TCGA datasets for selected promising SNPs

### Figure 3. ENCODE ChIP-seq data at 1q22 locus.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/85c7/4668218/381efc3d48c3/nihms706347f3.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4668218_nihms706347f3.jpg)

Manhattan plot of all iCOGS imputed/genotyped SNPs at 1q22, black enclosed circles represent genotyped SNPs while open red circles are imputed SNPs. Hash marks indicate location of highly correlated SNPs (r2 >0.9). Colored histograms denote histone modification for H3K4me1 and H3K27ac in normal ovary ChIP-seq data from UCSD and ENCODE.

### Figure 4. Overall survival in ‘all chemo’ for rs6674079.

![Figure 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/85c7/4668218/7a18f2b410ae/nihms706347f4a.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4668218_nihms706347f4a.jpg)

A. Kaplan Meier curve for OS in the ‘all chemo’ dataset. P-value derived from adjusted Cox PH models of genotyped samples (n=4,399): 0=common homozygotes AA, 1=heterozygotes AG, 2=rare homozygotes GG. B. Forest plot showing site-specific estimates for OS and rs6674079 in ‘all chemo’ dataset

We further queried protein-coding genes within a 1Mb region of each of these lead SNPs at 1q22, 11p15.4, 11p15.3, 16q23 and 19p12 ([Table 1](#T1)) using KM-plotter to identify gene expressions that might be associated with PFS and OS using all available data (1,170 and 1,435 patients respectively), and in a subset of cases restricted to optimally debulked serous cases treated with Taxol and platin chemotherapy (330 and 387 patients respectively). Of a total of 55 expression probes for 174 genes queried across the five loci, significant associations that met our Bonferroni-corrected significance threshold of p≤2.3x10^−4^ were observed for 11 probes in at least one analysis ([Supplementary Table 6](#SD1)). The strongest association with outcome was observed for PFS and high (defined as above the median) expression of *SLC25A44* (probe 32091_at) in the unrestricted dataset of 1,170 ovarian cancer patients (HR=1.56, 95% CI 1.33–1.82, log-rank p=1.9x10^−8^; [Supplementary Figure 5A](#SD1)). This association was upheld, although more weakly, in the subset restricted to optimally debulked serous cases treated with Taxol and platin chemotherapy (n=330, HR=1.66, 95% CI 1.24–2.23, log-rank p-value=6.8x10^−4^). High expression of *SEMA4A* (probe 219259_at) was significantly associated with better PFS in the unrestricted dataset (HR 0.71, 95% CI 0.61 – 0.82, log-rank p=4.2x10^−6^; [Supplementary Figure 5B](#SD1)) and marginally with OS (unrestricted dataset log-rank p=3.3x10^−4^ and restricted dataset log-rank p=5.7x10^−4^). Significantly better PFS was also observed for high expression of *SH2D2A* (probe 207351_s_at) in the unrestricted datasets (HR=0.67, 95% CI 0.57 – 0.77, log-rank p=8.4x10^−8^; [Supplementary Figure 5C](#SD1)) with a marginal association for OS in the unrestricted dataset (log-rank p=8.7x10^−4^).

We also evaluated associations between OS and SNPs in the larger ‘all OCAC’ dataset with minimal adjustment. A total of 70 SNPs with imputation r^2^ ≥0.9 at 4 loci achieved a p≤1.0x10^−5^ ([Supplementary Table 7](#SD1)). The top SNP was rs2013459 (HR=1.14, 95% CI 1.08–1.20, p= 9.7x10^−7^ at *PARK2* located at 6q26. Significant SNPs were also identified at *FAR1* (11p15), *ANKLE1*, *BABAM1* and *ABHD8* (all at 19p13) and *SYNE2* (6q25).

### Pathway Analysis

We also explored the polygenic signal in our data using pathway-based analysis. This enrichment analysis of genome-wide single-variant summary statistics from the ‘standard chemo’ subgroup in the context of known biological pathways suggested heterogeneity in the pathways that may be associated with PFS and OS. Eight of the 837 pathways tested were associated with PFS in the ‘standard chemo’ dataset at nominal significance (p_GSEA_<0.05 and FWERGSEA<1), with the “YAP1- and WWTR1 (TAZ)- stimulated gene expression” pathway from the Reactome pathway database emerging as the most significant (p_GSEA_=1x10^−3^, FDR_GSEA_=0.868, FWER_GSEA_=0.575, [Table 2](#T2)). Nine of the 837 pathways were associated with OS in the ‘standard chemo’ data set at the same threshold for nominal significance and the Reactome pathway “HDL-mediated lipid transport” was the top pathway (p_GSEA_=6x10^−3^, FDR_GSEA_=0.303, FWER_GSEA_=0.268, [Table 2](#T2)). Interestingly, the other nominally significant pathways suggested possible involvement of cell cycle genes in determining PFS and of xenobiotic and insulin metabolism genes in determining OS in the ‘standard chemo’ cohort ([Table 2](#T2)).

### Table 2.

| Pathway | aGenes | p-value | bFDR | cFWER | Core genes |
| --- | --- | --- | --- | --- | --- |
| Pathways associated with PFS in ‘standard chemo’ at p<0.05 and FWER<1 |  |  |  |  |  |
| REACTOME_YAP1_AND_WWTR1_TAZ_STIMULATED_GENE_EXPRESSION | 23 | 0.001 | 0.868 | 0.575 | CTGF,TBL1X,NCOA6,TEAD3,MED1,PPARA,TEAD1,NCOA3KAT2B |
| REACTOME_G0_AND_EARLY_G1 | 23 | 0.012 | 1 | 0.991 | RBL2,CDC25A,MYBL2,LIN9,HDAC1,CCNA1,LIN52 |
| REACTOME_AMINE_DERIVED_HORMONES | 15 | 0.025 | 1 | 0.993 | CGA,TPO,SLC5A5,TH |
| REACTOME_FORMATION_OF_INCISION_COMPLEX_IN_GG_NER | 21 | 0.010 | 1 | 0.994 | ERCC2,RAD23B,GTF2H1,GTF2H2,RPA1,ERCC1,DDB2,XPADDB1 |
| REACTOME_G_PROTEIN_ACTIVATION | 27 | 0.007 | 1 | 0.999 | GNB2,GNAT1,GNAI2,GNAI1,POMC,GNB3,GNG4,GNGT2,GNAO1,GNG8,GNG3 |
| REACTOME_LYSOSOME_VESICLE_BIOGENESIS | 22 | 0.013 | 1 | 0.999 | CLTA,AP1B1,AP1S1,DNAJC6,AP1G1,GNS,M6PR,VAMP8,BLOC1S1 |
| REACTOME_INHIBITION_OF_INSULIN_SECRETION_BY_ADRENALINE_NORADRENALINE | 25 | 0.014 | 1 | 0.999 | GNB2,GNAI2,CACNB2,GNAI1,ADRA2A,GNB3,GNG4,GNGT2GNAO1,GNG8,GNG3 |
| REACTOME_CYCLIN_A_B1_ASSOCIATED_EVENTS_DURING_G2_M_TRANSITION | 15 | 0.025 | 1 | 0.999 | CDC25A,PLK1,CCNA1,WEE1,CDC25B,PKMYT1,XPO1 |
| Pathways associated with OS in ‘standard chemo’ at p<0.05 and FWER<1 |  |  |  |  |  |
| REACTOME_HDL_MEDIATED_LIPID_TRANSPORT | 15 | 0.006 | 0.303 | 0.268 | BMP1,CETP,APOA1,APOC3,ABCG1 |
| REACTOME_XENOBIOTICS | 15 | 0.009 | 1 | 0.891 | CYP2A13,CYP2B6,CYP2F1 |
| REACTOME_LIPOPROTEIN_METABOLISM | 28 | 0.005 | 1 | 0.979 | BMP1,CETP,APOA1,APOC3,APOA5,ABCG1 |
| REACTOME_INSULIN_SYNTHESIS_AND_PROCESSING | 20 | 0.005 | 0.915 | 0.980 | SNAP25,INS,EXOC5,ERO1L,PCSK1,EXOC4,PCSK2 |
| BIOCARTA_MTA3_PATHWAY | 19 | 0.013 | 0.772 | 0.982 | TUBA1A,TUBA1C,HDAC1,MBD3,ALDOA,CDH1,MTA1,SNAI2,TUBA3C |
| REACTOME_ACETYLCHOLINE_BINDING_AND_DOWNSTREAM_EVENTS | 15 | 0.022 | 0.781 | 0.994 | CHRNG,CHRND |
| REACTOME_SYNTHESIS_OF_BILE_ACIDS_AND_BILE_SALTS_VIA_7ALPHA_HYDROXYCHOLESTEROL | 15 | 0.032 | 0.716 | 0.996 | SLC27A5, HSD17B4, AKR1D1, SLC27A2, CYP27A1, ACOX2HSD3B7, ABCB11 |
| KEGG_MATURITY_ONSET_DIABETES_OF_THE_YOUNG | 23 | 0.006 | 0.658 | 0.997 | ONECUT1, INS, HNF1A, BHLHA15, NR5A2, FOXA3 |
| REACTOME_IMMUNOREGULATORY_INTERACTIONS_BETWEEN_A_LYMPHOID_AND_A_NON_LYMPHOID_CELL | 56 | 0.001 | 0.621 | 0.998 | CD96, CD8A, CD8B, IFITM1, KIR3DL2, CRTAM, ICAM2KIR3DL1, FCGR3A, LILRB2, CD19, LILRB5, LILRB3CD200R1, RAET1E, FCGR2B, SELL,ULBP2, ULBP1KIR2DL4, B2M, CDH1, CD81 |

Table 2 Caption: Gene set enrichment (pathway-level) analysis results for PFS and OS associations in the ‘standard chemo’ data set

## Discussion

We have evaluated ~2.8 million SNPs across the genome for an association with outcome following first-line chemotherapy in a large cohort of EOC patients and identified SNPs at five loci with p-values that ranged from 1.05x10^−5^ to 4.7x10^−7^. Three SNPs, rs6674079, rs4910232 and rs2549714, were located in long non-coding RNAs (lncRNA) RP11–284F21.8, RP11–179A10.1 and RP11–314O13.1 respectively ([Table 1](#T1)). LncRNAs are RNA transcripts that have been implicated in a wide range of regulatory functions including epigenetic control and regulation of chromatin structure at the cellular level to tumor suppressors and regulators of angiogenesis and metastasis ([31](#R31)). It has been shown that alterations in the function of some lncRNAs, particularly those involved in transcriptional regulation, can play a critical role in cancer progression and exert its effect on genes located on other chromosomes. A well characterized example of this is the lncRNA *HOTAIR* which has been linked to invasiveness and poor prognosis of breast cancer ([32](#R32)). *HOTAIR* is expressed from the *HOXC* gene cluster on chromosome 12, and has been shown to mediate repression of transcription of *HOXD* genes on chromosome 2 via PRC2 ([33](#R33)). While little is known about the specific lncRNAs that we have identified or their target genes, it is likely that associated SNPs in these lncRNAs might exert their effects on chromatin modifying proteins that regulate genes involved in ovarian cancer progression. ENCODE ChIP-seq data for normal ovarian cell lines at the 1q22 locus shows evidence of histone modification in the region of RP11–284F21.8, and rs6674079 at this locus is perfectly correlated with rs11264489 which lies within the super-enhancer MEF2D ([Figure 4](#F4)). Expression studies of MEF2D in hepatocellular carcinoma showed that elevated expression promoted cancer cell growth and was correlated with poor prognosis in patients ([34](#R34)). Further analysis of rs6674079 and other SNPs identified in this study in lncRNAs would be necessary to determine their putative regulatory effects and potential impact on ovarian cancer metastasis and progression.

Several protein-coding genes within 1Mb of rs6674079 at 1q22 were also found to be significantly associated with ovarian cancer progression in unrestricted analyses of KM-plotter data ([Supplementary Table 6](#SD1)). Above-median expression of *SLC25A44* (probe 32091_at), a recently identified member of the SLC25 family of mitochondrial carrier proteins, was significantly associated with worse PFS in analysis in the larger unrestricted dataset of epithelial ovarian cancer (log-rank p≤1.9x10^−8^; [Supplementary Figure 4A](#SD1)). While relatively little is known about specific functions or disease-gene associations with *SLC25A44*, changes in expression of some members of the SLC25 family of transporters have been implicated in resistance to cell death in other cancers ([35](#R35)). Similarly high expression of the signalling protein *SEMA4A* (probe 219259_at; [Supplementary Figure 4B](#SD1)) was significantly associated with better PFS (log-rank p=4.2x10^−6^). SEMA4A is a member of the semaphorin family of soluble and transmembrane proteins which mediate their signal transduction effects through plexins, both of which have been shown to have tumorigenic properties and are aberrantly expressed in human cancers, ([36](#R36), [37](#R37)). Also high expression of *SH2D2A* (probeset 21925_at) which encodes a T-cell-specific adaptor protein (TSAd), was associated with significantly better PFS (log-rank p=8.4x10^−8^; [Supplementary Figure 4C](#SD1)). Chromosmal imbalance at 1q22 was previously identified as a candidate region for response to chemotherapy in human glioma cell lines ([38](#R38)) and it has been shown that alterations on the long arm of chromosome 1, particularly gain of function, are among the most commonly reported chromosomal abnormalities in human cancers ([39](#R39)). Further studies would be necessary to delineate the relevance of these novel findings in EOC outcome.

We found that that PFS-associated SNPs in the ‘standard chemo’ dataset were most significantly enriched in a pathway containing target genes of the transcriptional co-activators YAP1 and WWTR1 and the antisense RNA gene TAZ ([40](#R40), [41](#R41)). *YAP1*, an established ovarian cancer oncogene ([42](#R42)), is known to regulate the cell cycle and epithelial-mesenchymal transition, promoting tumor survival even in the absence of oncogenic KRAS signaling ([43](#R43), [44](#R44)). A gene expression signature representing YAP1 activation in ovarian tumors has also recently been found to be predictive of response to taxane-based adjuvant chemotherapy regimens and is associated with overall survival in ovarian cancer ([45](#R45)). The HDL-mediated lipid transport pathway driven by genes that included *APOA1* was associated with OS in the setting of standard chemotherapy. Higher *APOA1* expression in serous ovarian cancer effusions has previously been associated with improved overall survival in a small cohort ([46](#R46)). Apolipoprotein A-I activity has been shown to reduce viability of platinum-resistant human ovarian cancer cells *in vitro* and inhibit tumor development in a mouse model of ovarian cancer ([47](#R47)).

In our exploratory histology-adjusted analysis of OS in ‘all OCAC’ we observed significant associations with SNPs in *PARK2* and decreased survival. *PARK2*, a component of E3 ubiquitin ligase complexes that drive cyclin D and E degradation, is frequently lost in human cancers, and knock-down in a range of cancer cell lines has been shown to correlate with increased cell proliferation and transcription of genes related to cell cycle control, suggesting a role in disease progression and prognosis ([48](#R48)). *ANKLE1* and *BABAM1* at 19p13.11 (p≤9.5x10^−6^ ; [Supplementary Table 8](#SD1)) were also identified and SNPs at this locus were previously implicated in ovarian cancer risk and survival ([49](#R49)). However in our fully adjusted analysis of ~2900 patients for which we had all covariates, we observed no significant association for any SNP at this locus (p≥0.002). This may be accounted for by the lower power to detect the effects seen in the larger ‘all OCAC’ analysis, or the fact that the lower p-value in the ‘all OCAC’ analysis is an artefact resulting from partial adjustment for confounders of outcome. Further analyses including FIGO stage, grade and residual disease would be necessary to evaluate this locus. We also observed no significant association for candidate SNPs previously identified to be associated with response to chemotherapy using the NHGRI GWAS catalog ([http://www.genome.gov/gwastudies/](http://www.genome.gov/gwastudies/)) with any of our four analyses ([Supplementary Table 9](#SD1)).

Our validation analysis of genotyped data also highlighted the potential for spurious associations using imputed data in smaller samples sets. Although current strategies of ‘pre-phasing’ has improved imputation accuracy for SNPs with MAF 1–3% and prior imputation r^2^ as low as 0.6 in Europeans ([50](#R50)), we observed a high degree of discordance in estimates from imputed data compared to actual genotypes, even for SNPs with reasonable imputation quality (r^2^=0.6–0.9) and particularly for SNPs with MAF<3% (Supplementary Methods and [Supplementary Table 3](#SD1)). We therefore selected SNPs for validation from ~2.8 million SNPs with good imputation quality (r^2^≥0.9) to reduce the risk of false positives.

In conclusion we have identified three SNPs in lncRNAs that have not been previously reported on that were associated with PFS in ovarian cancer regardless of chemotherapy regimens. We also identified two other SNPs, rs7950311 at 11p15.4 associated with OS in the ‘standard chemo’ analysis and rs3795247 at 19p12 associated with PFS in the ‘all chemo’ analysis, both of which reside in genes that have not been previously implicated in solid tumors. To our knowledge this is the largest study that comprehensively analyzes genetic variation across the genome for an association with ovarian cancer outcomes, both with regard to first-line standard-of-care chemotherapy and regardless of treatment. Since residual disease is a strong predictor of overall and progression-free survival, patients were included in our main analyses if they received a minimum of cytoreductive surgery and had available information on level of residual disease. SNPs were prioritized on the basis of good imputation quality (r^2^ ≥0.9) and final estimates were derived from meta-analysis of all available data imputed and genotyped samples from OCAC and publicly available TCGA data. To circumvent methodological flaws we restricted the analysis to European invasive EOC patients participating in the OCAC with standardized definitions of clinical and pathological characteristics. Despite our rigorous analysis approach, there are inherent limitations in the observational design of our study that a randomized clinical trial would circumvent, in that standardized treatment and outcome measurements would be available, and the presence of a control group receiving an alternative treatment would allow assessment of a likely causal relationship between the putative associations and treatment modalities.

Pharmacogenomic studies hold the promise of improving treatment approaches by the identification of genetic markers which may enhance the clinical approaches and cost-effectiveness of these treatment approaches. However, large clinical trials or well-designed prospective cohort studies that take into account differential responses according to EOC tumor types, as well as functional studies that shed light on putative associations are required to succeed in defining the role of genetics in ovarian cancer progression and survival.

## Supplementary Material

Although several genetic loci have been identified for ovarian cancer risk, finding loci associated with outcome remains a challenge primarily because of treatment heterogeneity and small sample sizes. We comprehensively analyzed ~2.8 million variants in the largest collection to date of epithelial ovarian cancer cases with detailed chemotherapy and clinical follow-up data, and identified SNPs in three long non-coding RNAs (lncRNAs) that were associated with progression-free survival, one of which lies within a super-enhancer recently shown to be associated with poor prognosis in another solid tumor. There is a growing body of evidence that lncRNAs are cancer-specific regulators in signalling pathways underlying metastasis and disease progression. While additional work is needed to delineate the role of associated SNPs on lncRNA expression and validate their role in a larger sample, our findings have important implications for the development of diagnostic markers of progression and novel therapeutic targets for epithelial ovarian cancer.

Although several genetic loci have been identified for ovarian cancer risk, finding loci associated with outcome remains a challenge primarily because of treatment heterogeneity and small sample sizes. We comprehensively analyzed ~2.8 million variants in the largest collection to date of epithelial ovarian cancer cases with detailed chemotherapy and clinical follow-up data, and identified SNPs in three long non-coding RNAs (lncRNAs) that were associated with progression-free survival, one of which lies within a super-enhancer recently shown to be associated with poor prognosis in another solid tumor. There is a growing body of evidence that lncRNAs are cancer-specific regulators in signalling pathways underlying metastasis and disease progression. While additional work is needed to delineate the role of associated SNPs on lncRNA expression and validate their role in a larger sample, our findings have important implications for the development of diagnostic markers of progression and novel therapeutic targets for epithelial ovarian cancer.

## Acknowledgements

This study would not have been possible without the contributions of the following: Per Hall (COGS); Douglas F. Easton, Paul Pharoah, Kyriaki Michailidou, Manjeet K. Bolla, Qin Wang (BCAC), Andrew Berchuck (OCAC), Rosalind A. Eeles, Douglas F. Easton, Ali Amin Al Olama, Zsofia Kote-Jarai, Sara Benlloch (PRACTICAL), Georgia Chenevix-Trench, Antonis Antoniou, Lesley McGuffog, Fergus Couch and Ken Offit (CIMBA), Joe Dennis, Alison M. Dunning, Andrew Lee, and Ed Dicks, Craig Luccarini and the staff of the Centre for Genetic Epidemiology Laboratory, Javier Benitez, Anna Gonzalez-Neira and the staff of the CNIO genotyping unit, Jacques Simard and Daniel C. Tessier, Francois Bacot, Daniel Vincent, Sylvie LaBoissière and Frederic Robidoux and the staff of the McGill University and Génome Québec Innovation Centre, Stig E. Bojesen, Sune F. Nielsen, Borge G. Nordestgaard, and the staff of the Copenhagen DNA laboratory, and Julie M. Cunningham, Sharon A. Windebank, Christopher A. Hilker, Jeffrey Meyer and the staff of Mayo Clinic Genotyping Core Facility.

We are grateful to the family and friends of Kathryn Sladek Smith for their generous support for the Ovarian Cancer Association Consortium through their donations to the Ovarian Cancer Research Fund. The authors wish to thank Margie Riggan for her tireless dedication to the Ovarian Cancer Association Consortium through her excellent project and data management. The Australian Ovarian Cancer Study Management Group (D. Bowtell, G. Chenevix-Trench, A. deFazio, D. Gertig, A. Green, P. Webb) and ACS Investigators (A. Green, P. Parsons, N. Hayward, P. Webb, D. Whiteman) thank all the clinical and scientific collaborators (see [http://www.aocstudy.org/](http://www.aocstudy.org/)) and the women for their contribution. GCT & PW are supported by Fellowships from NHMRC. The Belgian study (BEL) would like to thank Gilian Peuteman, Thomas Van Brussel and Dominiek Smeets for technical assistance. The German Ovarian Cancer Study (GER) thank Ursula Eilber and Tanja Koehler for competent technical assistance. The International Collaborative Ovarian Neoplasm study (ICON)7 trial team would like to thank the Medical Research Council (MRC) Clinical Trial Unit (CTU) at the University of London (UCL), the ICON7 Translational Research Sub-group, and the University of Leeds for their work on the coordination of samples and data from the ICON7 trial. The Mayo Clinic Ovarian Cancer Case-Control Study (MAY) thank C. Hilker, S. Windebank, and J. Vollenweider for iSelect genotyping. The Study of Epidemiology and Risk Factors in Cancer Heredity (SEA) would like to acknowledge Craig Luccarini, Caroline Baynes, Don Conroy. The Scottish Randomised Trial in Ovarian Cancer (SRO) thank all members of Scottish Gynaecological Clinical Trails group and SCOTROC1 investigators. The results published here are in part based upon data generated by The Cancer Genome Atlas Pilot Project established by the National Cancer Institute and National Human Genome Research Institute. Information about TCGA can be found at [http://cancergenome.nih.gov/](http://cancergenome.nih.gov/). The United Kingdom Ovarian Cancer Population Study (UKO) particularly thank I. Jacobs, M. Widschwendter, E. Wozniak, A. Ryan, J. Ford and N. Balogun for their contribution to the study. The Westmead Hospital Molecular Biology of Gynaecologic Disease (WMH) thank the Gynaecological Oncology Biobank at Westmead, a member of the Australasian Biospecimen Network-Oncology group, which is funded by the National Health and Medical Research Council Enabling Grants ID 310670 & ID 628903 and the Cancer Institute NSW.

Anna deFazio was funded by the University of Sydney Cancer Research Fund and the Cancer Institute NSW through the Sydney West-Translational Cancer Research Centre. Dr. Beth Y. Karlan is supported by American Cancer Society Early Detection Professorship (SIOP-06–258–01-COUN) and the National Center for Advancing Translational Sciences (NCATS), Grant UL1TR000124. Irene Orlow was supported by NCI CCSG award (P30-CA008748).

Funding for the iCOGS infrastructure came from: the European Community’s Seventh Framework Programme under grant agreement n° 223175 (HEALTH-F2–2009–223175) (COGS), Cancer Research UK (C1287/A10118, C1287/A 10710, C12292/A11174, C1281/A12014, C5047/A8384, C5047/A15007, C5047/A10692, C8197/A16565), the National Institutes of Health (CA128978) and Post-Cancer GWAS initiative (1U19 CA148537, 1U19 CA148065 and 1U19 CA148112 - the GAME-ON initiative), the Department of Defence (W81XWH-10–1–0341), the Canadian Institutes of Health Research (CIHR) for the CIHR Team in Familial Risks of Breast Cancer, Komen Foundation for the Cure, the Breast Cancer Research Foundation, and the Ovarian Cancer Research Fund.

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Ferlay J, Soerjomataram I, Ervik M, Dikshit R, Eser S, Mathers C, et al. [accessed on 13/11/2014];GLOBOCAN 2012 v1.0, Cancer Incidence and Mortality Worldwide: IARC CancerBase No. 11. 2013 http://globocaniarcfr.  [http://globocaniarcfr](http://globocaniarcfr)

2. Marchetti C, Pisano C, Facchini G, Bruni GS, Magazzino FP, Losito S, et al. First-line treatment of advanced ovarian cancer: current research and perspectives. Expert Rev Anticancer Ther. 2010;10:47–60. doi: 10.1586/era.09.167.  [DOI](https://doi.org/10.1586/era.09.167) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20014885/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Expert%20Rev%20Anticancer%20Ther&title=First-line%20treatment%20of%20advanced%20ovarian%20cancer:%20current%20research%20and%20perspectives&author=C%20Marchetti&author=C%20Pisano&author=G%20Facchini&author=GS%20Bruni&author=FP%20Magazzino&volume=10&publication_year=2010&pages=47-60&pmid=20014885&doi=10.1586/era.09.167&)

3. Ozols RF, Bundy BN, Greer BE, Fowler JM, Clarke-Pearson D, Burger RA, et al. Phase III trial of carboplatin and paclitaxel compared with cisplatin and paclitaxel in patients with optimally resected stage III ovarian cancer: a Gynecologic Oncology Group study. J Clin Oncol. 2003;21:3194–3200. doi: 10.1200/JCO.2003.02.153.  [DOI](https://doi.org/10.1200/JCO.2003.02.153) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12860964/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Phase%20III%20trial%20of%20carboplatin%20and%20paclitaxel%20compared%20with%20cisplatin%20and%20paclitaxel%20in%20patients%20with%20optimally%20resected%20stage%20III%20ovarian%20cancer:%20a%20Gynecologic%20Oncology%20Group%20study&author=RF%20Ozols&author=BN%20Bundy&author=BE%20Greer&author=JM%20Fowler&author=D%20Clarke-Pearson&volume=21&publication_year=2003&pages=3194-3200&pmid=12860964&doi=10.1200/JCO.2003.02.153&)

4. Jayson GC, Kohn EC, Kitchener HC, Ledermann JA. Ovarian cancer. Lancet. 2014;384:1376–1388. doi: 10.1016/S0140-6736(13)62146-7.  [DOI](https://doi.org/10.1016/S0140-6736(13)62146-7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24767708/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Ovarian%20cancer&author=GC%20Jayson&author=EC%20Kohn&author=HC%20Kitchener&author=JA%20Ledermann&volume=384&publication_year=2014&pages=1376-1388&pmid=24767708&doi=10.1016/S0140-6736(13)62146-7&)

5. Evans WE, McLeod HL. Pharmacogenomics--drug disposition, drug targets, and side effects. N Engl J Med. 2003;348:538–549. doi: 10.1056/NEJMra020526.  [DOI](https://doi.org/10.1056/NEJMra020526) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12571262/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Pharmacogenomics--drug%20disposition,%20drug%20targets,%20and%20side%20effects&author=WE%20Evans&author=HL%20McLeod&volume=348&publication_year=2003&pages=538-549&pmid=12571262&doi=10.1056/NEJMra020526&)

6. Kalow W, Tang BK, Endrenyi L. Hypothesis: comparisons of inter- and intra-individual variations can substitute for twin studies in drug research. Pharmacogenetics. 1998;8:283–289. doi: 10.1097/00008571-199808000-00001.  [DOI](https://doi.org/10.1097/00008571-199808000-00001) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9731714/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=Hypothesis:%20comparisons%20of%20inter-%20and%20intra-individual%20variations%20can%20substitute%20for%20twin%20studies%20in%20drug%20research&author=W%20Kalow&author=BK%20Tang&author=L%20Endrenyi&volume=8&publication_year=1998&pages=283-289&pmid=9731714&doi=10.1097/00008571-199808000-00001&)

7. Vesell ES. Pharmacogenetic perspectives gained from twin and family studies. Pharmacology & therapeutics. 1989;41:535–552. doi: 10.1016/0163-7258(89)90130-7.  [DOI](https://doi.org/10.1016/0163-7258(89)90130-7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/2654965/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacology%20&%20therapeutics&title=Pharmacogenetic%20perspectives%20gained%20from%20twin%20and%20family%20studies&author=ES%20Vesell&volume=41&publication_year=1989&pages=535-552&pmid=2654965&doi=10.1016/0163-7258(89)90130-7&)

8. Relling MV, Dervieux T. Pharmacogenetics and cancer therapy. Nat Rev Cancer. 2001;1:99–108. doi: 10.1038/35101056.  [DOI](https://doi.org/10.1038/35101056) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11905809/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Cancer&title=Pharmacogenetics%20and%20cancer%20therapy&author=MV%20Relling&author=T%20Dervieux&volume=1&publication_year=2001&pages=99-108&pmid=11905809&doi=10.1038/35101056&)

9. Vella N, Aiello M, Russo AE, Scalisi A, Spandidos DA, Toffoli G, et al. ‘Genetic profiling’ and ovarian cancer therapy (review) Molecular medicine reports. 2011;4:771–777. doi: 10.3892/mmr.2011.512.  [DOI](https://doi.org/10.3892/mmr.2011.512) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21687948/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Molecular%20medicine%20reports&title=%E2%80%98Genetic%20profiling%E2%80%99%20and%20ovarian%20cancer%20therapy%20(review)&author=N%20Vella&author=M%20Aiello&author=AE%20Russo&author=A%20Scalisi&author=DA%20Spandidos&volume=4&publication_year=2011&pages=771-777&pmid=21687948&doi=10.3892/mmr.2011.512&)

10. Li J, Bluth MH. Pharmacogenomics of drug metabolizing enzymes and transporters: implications for cancer therapy. Pharmacogenomics and personalized medicine. 2011;4:11–33. doi: 10.2147/PGPM.S18861.  [DOI](https://doi.org/10.2147/PGPM.S18861) | [PMC free article](/articles/PMC3513217/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23226051/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20and%20personalized%20medicine&title=Pharmacogenomics%20of%20drug%20metabolizing%20enzymes%20and%20transporters:%20implications%20for%20cancer%20therapy&author=J%20Li&author=MH%20Bluth&volume=4&publication_year=2011&pages=11-33&pmid=23226051&doi=10.2147/PGPM.S18861&)

11. Leschziner GD, Andrew T, Pirmohamed M, Johnson MR. ABCB1 genotype and PGP expression, function and therapeutic drug response: a critical review and recommendations for future research. Pharmacogenomics J. 2007;7:154–179. doi: 10.1038/sj.tpj.6500413.  [DOI](https://doi.org/10.1038/sj.tpj.6500413) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16969364/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=ABCB1%20genotype%20and%20PGP%20expression,%20function%20and%20therapeutic%20drug%20response:%20a%20critical%20review%20and%20recommendations%20for%20future%20research&author=GD%20Leschziner&author=T%20Andrew&author=M%20Pirmohamed&author=MR%20Johnson&volume=7&publication_year=2007&pages=154-179&pmid=16969364&doi=10.1038/sj.tpj.6500413&)

12. Green H, Soderkvist P, Rosenberg P, Horvath G, Peterson C. mdr-1 single nucleotide polymorphisms in ovarian cancer tissue: G2677T/A correlates with response to paclitaxel chemotherapy. Clin Cancer Res. 2006;12:854–859. doi: 10.1158/1078-0432.CCR-05-0950.  [DOI](https://doi.org/10.1158/1078-0432.CCR-05-0950) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16467099/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&title=mdr-1%20single%20nucleotide%20polymorphisms%20in%20ovarian%20cancer%20tissue:%20G2677T/A%20correlates%20with%20response%20to%20paclitaxel%20chemotherapy&author=H%20Green&author=P%20Soderkvist&author=P%20Rosenberg&author=G%20Horvath&author=C%20Peterson&volume=12&publication_year=2006&pages=854-859&pmid=16467099&doi=10.1158/1078-0432.CCR-05-0950&)

13. Johnatty SE, Beesley J, Gao B, Chen X, Lu Y, Law MH, et al. ABCB1 (MDR1) polymorphisms and ovarian cancer progression and survival: a comprehensive analysis from the Ovarian Cancer Association Consortium and The Cancer Genome Atlas. Gynecol Oncol. 2013;131:8–14. doi: 10.1016/j.ygyno.2013.07.107.  [DOI](https://doi.org/10.1016/j.ygyno.2013.07.107) | [PMC free article](/articles/PMC3795832/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23917080/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gynecol%20Oncol&title=ABCB1%20(MDR1)%20polymorphisms%20and%20ovarian%20cancer%20progression%20and%20survival:%20a%20comprehensive%20analysis%20from%20the%20Ovarian%20Cancer%20Association%20Consortium%20and%20The%20Cancer%20Genome%20Atlas&author=SE%20Johnatty&author=J%20Beesley&author=B%20Gao&author=X%20Chen&author=Y%20Lu&volume=131&publication_year=2013&pages=8-14&pmid=23917080&doi=10.1016/j.ygyno.2013.07.107&)

14. Diaz-Padilla I, Amir E, Marsh S, Liu G, Mackay H. Genetic polymorphisms as predictive and prognostic biomarkers in gynecological cancers: a systematic review. Gynecol Oncol. 2012;124:354–365. doi: 10.1016/j.ygyno.2011.10.034.  [DOI](https://doi.org/10.1016/j.ygyno.2011.10.034) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22063461/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gynecol%20Oncol&title=Genetic%20polymorphisms%20as%20predictive%20and%20prognostic%20biomarkers%20in%20gynecological%20cancers:%20a%20systematic%20review&author=I%20Diaz-Padilla&author=E%20Amir&author=S%20Marsh&author=G%20Liu&author=H%20Mackay&volume=124&publication_year=2012&pages=354-365&pmid=22063461&doi=10.1016/j.ygyno.2011.10.034&)

15. Hedditch EL, Gao B, Russell AJ, Lu Y, Emmanuel C, Beesley J, et al. ABCA transporter gene expression and poor outcome in epithelial ovarian cancer. J Natl Cancer Inst. 2014:106. doi: 10.1093/jnci/dju149.  [DOI](https://doi.org/10.1093/jnci/dju149) | [PMC free article](/articles/PMC4110473/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24957074/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Natl%20Cancer%20Inst&title=ABCA%20transporter%20gene%20expression%20and%20poor%20outcome%20in%20epithelial%20ovarian%20cancer&author=EL%20Hedditch&author=B%20Gao&author=AJ%20Russell&author=Y%20Lu&author=C%20Emmanuel&publication_year=2014&pages=106&pmid=24957074&doi=10.1093/jnci/dju149&)

16. Candido Dos Reis FJ, Song H, Goode EL, Cunningham JM, Fridley BL, Larson MC, et al. Germline mutation in BRCA1 or BRCA2 and ten-year survival for women diagnosed with epithelial ovarian cancer. Clin Cancer Res. 2014 doi: 10.1158/1078-0432.CCR-14-2497.  [DOI](https://doi.org/10.1158/1078-0432.CCR-14-2497) | [PMC free article](/articles/PMC4338615/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25398451/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&title=Germline%20mutation%20in%20BRCA1%20or%20BRCA2%20and%20ten-year%20survival%20for%20women%20diagnosed%20with%20epithelial%20ovarian%20cancer&author=FJ%20Candido%20Dos%20Reis&author=H%20Song&author=EL%20Goode&author=JM%20Cunningham&author=BL%20Fridley&publication_year=2014&pmid=25398451&doi=10.1158/1078-0432.CCR-14-2497&)

17. Alsop K, Fereday S, Meldrum C, deFazio A, Emmanuel C, George J, et al. BRCA mutation frequency and patterns of treatment response in BRCA mutation-positive women with ovarian cancer: a report from the Australian Ovarian Cancer Study Group. J Clin Oncol. 2012;30:2654–2663. doi: 10.1200/JCO.2011.39.8545.  [DOI](https://doi.org/10.1200/JCO.2011.39.8545) | [PMC free article](/articles/PMC3413277/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22711857/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=BRCA%20mutation%20frequency%20and%20patterns%20of%20treatment%20response%20in%20BRCA%20mutation-positive%20women%20with%20ovarian%20cancer:%20a%20report%20from%20the%20Australian%20Ovarian%20Cancer%20Study%20Group&author=K%20Alsop&author=S%20Fereday&author=C%20Meldrum&author=A%20deFazio&author=C%20Emmanuel&volume=30&publication_year=2012&pages=2654-2663&pmid=22711857&doi=10.1200/JCO.2011.39.8545&)

18. Samouelian V, Maugard CM, Jolicoeur M, Bertrand R, Arcand SL, Tonin PN, et al. Chemosensitivity and radiosensitivity profiles of four new human epithelial ovarian cancer cell lines exhibiting genetic alterations in BRCA2, TGFbeta-RII, KRAS2, TP53 and/or CDNK2A. Cancer Chemother Pharmacol. 2004;54:497–504. doi: 10.1007/s00280-004-0843-9.  [DOI](https://doi.org/10.1007/s00280-004-0843-9) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15258697/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Chemother%20Pharmacol&title=Chemosensitivity%20and%20radiosensitivity%20profiles%20of%20four%20new%20human%20epithelial%20ovarian%20cancer%20cell%20lines%20exhibiting%20genetic%20alterations%20in%20BRCA2,%20TGFbeta-RII,%20KRAS2,%20TP53%20and/or%20CDNK2A&author=V%20Samouelian&author=CM%20Maugard&author=M%20Jolicoeur&author=R%20Bertrand&author=SL%20Arcand&volume=54&publication_year=2004&pages=497-504&pmid=15258697&doi=10.1007/s00280-004-0843-9&)

19. Husain A, He G, Venkatraman ES, Spriggs DR. BRCA1 up-regulation is associated with repair-mediated resistance to cis-diamminedichloroplatinum(II) Cancer Res. 1998;58:1120–1123.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/9515792/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=BRCA1%20up-regulation%20is%20associated%20with%20repair-mediated%20resistance%20to%20cis-diamminedichloroplatinum(II)&author=A%20Husain&author=G%20He&author=ES%20Venkatraman&author=DR%20Spriggs&volume=58&publication_year=1998&pages=1120-1123&pmid=9515792&)

20. Huang RS, Duan S, Shukla SJ, Kistner EO, Clark TA, Chen TX, et al. Identification of genetic variants contributing to cisplatin-induced cytotoxicity by use of a genomewide approach. Am J Hum Genet. 2007;81:427–437. doi: 10.1086/519850.  [DOI](https://doi.org/10.1086/519850) | [PMC free article](/articles/PMC1950832/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17701890/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=Identification%20of%20genetic%20variants%20contributing%20to%20cisplatin-induced%20cytotoxicity%20by%20use%20of%20a%20genomewide%20approach&author=RS%20Huang&author=S%20Duan&author=SJ%20Shukla&author=EO%20Kistner&author=TA%20Clark&volume=81&publication_year=2007&pages=427-437&pmid=17701890&doi=10.1086/519850&)

21. Huang RS, Johnatty SE, Gamazon ER, Im HK, Ziliak D, Duan S, et al. Platinum sensitivity-related germline polymorphism discovered via a cell-based approach and analysis of its association with outcome in ovarian cancer patients. Clin Cancer Res. 2011;17:5490–5500. doi: 10.1158/1078-0432.CCR-11-0724.  [DOI](https://doi.org/10.1158/1078-0432.CCR-11-0724) | [PMC free article](/articles/PMC3160494/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21705454/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&title=Platinum%20sensitivity-related%20germline%20polymorphism%20discovered%20via%20a%20cell-based%20approach%20and%20analysis%20of%20its%20association%20with%20outcome%20in%20ovarian%20cancer%20patients&author=RS%20Huang&author=SE%20Johnatty&author=ER%20Gamazon&author=HK%20Im&author=D%20Ziliak&volume=17&publication_year=2011&pages=5490-5500&pmid=21705454&doi=10.1158/1078-0432.CCR-11-0724&)

22. Sankararaman S, Sridhar S, Kimmel G, Halperin E. Estimating local ancestry in admixed populations. Am J Hum Genet. 2008;82:290–303. doi: 10.1016/j.ajhg.2007.09.022.  [DOI](https://doi.org/10.1016/j.ajhg.2007.09.022) | [PMC free article](/articles/PMC2664993/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18252211/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=Estimating%20local%20ancestry%20in%20admixed%20populations&author=S%20Sankararaman&author=S%20Sridhar&author=G%20Kimmel&author=E%20Halperin&volume=82&publication_year=2008&pages=290-303&pmid=18252211&doi=10.1016/j.ajhg.2007.09.022&)

23. Price AL, Patterson NJ, Plenge RM, Weinblatt ME, Shadick NA, Reich D. Principal components analysis corrects for stratification in genome-wide association studies. Nat Genet. 2006;38:904–909. doi: 10.1038/ng1847.  [DOI](https://doi.org/10.1038/ng1847) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16862161/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Principal%20components%20analysis%20corrects%20for%20stratification%20in%20genome-wide%20association%20studies&author=AL%20Price&author=NJ%20Patterson&author=RM%20Plenge&author=ME%20Weinblatt&author=NA%20Shadick&volume=38&publication_year=2006&pages=904-909&pmid=16862161&doi=10.1038/ng1847&)

24. Pharoah PD, Tsai YY, Ramus SJ, Phelan CM, Goode EL, Lawrenson K, et al. GWAS meta-analysis and replication identifies three new susceptibility loci for ovarian cancer. Nat Genet. 2013;45:362–370. doi: 10.1038/ng.2564.  [DOI](https://doi.org/10.1038/ng.2564) | [PMC free article](/articles/PMC3693183/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23535730/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=GWAS%20meta-analysis%20and%20replication%20identifies%20three%20new%20susceptibility%20loci%20for%20ovarian%20cancer&author=PD%20Pharoah&author=YY%20Tsai&author=SJ%20Ramus&author=CM%20Phelan&author=EL%20Goode&volume=45&publication_year=2013&pages=362-370&pmid=23535730&doi=10.1038/ng.2564&)

25. Johnatty SE, Beesley J, Paul J, Fereday S, Spurdle AB, Webb PM, et al. ABCB1 (MDR 1) polymorphisms and progression-free survival among women with ovarian cancer following paclitaxel/carboplatin chemotherapy. Clin Cancer Res. 2008;14:5594–5601. doi: 10.1158/1078-0432.CCR-08-0606.  [DOI](https://doi.org/10.1158/1078-0432.CCR-08-0606) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18765553/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&title=ABCB1%20(MDR%201)%20polymorphisms%20and%20progression-free%20survival%20among%20women%20with%20ovarian%20cancer%20following%20paclitaxel/carboplatin%20chemotherapy&author=SE%20Johnatty&author=J%20Beesley&author=J%20Paul&author=S%20Fereday&author=AB%20Spurdle&volume=14&publication_year=2008&pages=5594-5601&pmid=18765553&doi=10.1158/1078-0432.CCR-08-0606&)

26. Willer CJ, Li Y, Abecasis GR. METAL: fast and efficient meta-analysis of genomewide association scans. Bioinformatics. 2010;26:2190–2191. doi: 10.1093/bioinformatics/btq340.  [DOI](https://doi.org/10.1093/bioinformatics/btq340) | [PMC free article](/articles/PMC2922887/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20616382/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=METAL:%20fast%20and%20efficient%20meta-analysis%20of%20genomewide%20association%20scans&author=CJ%20Willer&author=Y%20Li&author=GR%20Abecasis&volume=26&publication_year=2010&pages=2190-2191&pmid=20616382&doi=10.1093/bioinformatics/btq340&)

27. Gyorffy B, Lanczky A, Szallasi Z. Implementing an online tool for genome-wide validation of survival-associated biomarkers in ovarian-cancer using microarray data from 1287 patients. Endocrine-related cancer. 2012;19:197–208. doi: 10.1530/ERC-11-0329.  [DOI](https://doi.org/10.1530/ERC-11-0329) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22277193/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Endocrine-related%20cancer&title=Implementing%20an%20online%20tool%20for%20genome-wide%20validation%20of%20survival-associated%20biomarkers%20in%20ovarian-cancer%20using%20microarray%20data%20from%201287%20patients&author=B%20Gyorffy&author=A%20Lanczky&author=Z%20Szallasi&volume=19&publication_year=2012&pages=197-208&pmid=22277193&doi=10.1530/ERC-11-0329&)

28. Christoforou A, Dondrup M, Mattingsdal M, Mattheisen M, Giddaluru S, Nothen MM, et al. Linkage-disequilibrium-based binning affects the interpretation of GWASs. Am J Hum Genet. 2012;90:727–733. doi: 10.1016/j.ajhg.2012.02.025.  [DOI](https://doi.org/10.1016/j.ajhg.2012.02.025) | [PMC free article](/articles/PMC3322240/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22444669/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=Linkage-disequilibrium-based%20binning%20affects%20the%20interpretation%20of%20GWASs&author=A%20Christoforou&author=M%20Dondrup&author=M%20Mattingsdal&author=M%20Mattheisen&author=S%20Giddaluru&volume=90&publication_year=2012&pages=727-733&pmid=22444669&doi=10.1016/j.ajhg.2012.02.025&)

29. Segre AV, Consortium D, investigators M, Groop L, Mootha VK, Daly MJ, et al. Common inherited variation in mitochondrial genes is not enriched for associations with type 2 diabetes or related glycemic traits. PLoS genetics. 2010:6. doi: 10.1371/journal.pgen.1001058.  [DOI](https://doi.org/10.1371/journal.pgen.1001058) | [PMC free article](/articles/PMC2920848/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20714348/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20genetics&title=Common%20inherited%20variation%20in%20mitochondrial%20genes%20is%20not%20enriched%20for%20associations%20with%20type%202%20diabetes%20or%20related%20glycemic%20traits&author=AV%20Segre&author=D%20Consortium&author=M%20investigators&author=L%20Groop&author=VK%20Mootha&publication_year=2010&pages=6&pmid=20714348&doi=10.1371/journal.pgen.1001058&)

30. Subramanian A, Tamayo P, Mootha VK, Mukherjee S, Ebert BL, Gillette MA, et al. Gene set enrichment analysis: a knowledge-based approach for interpreting genome-wide expression profiles. Proc Natl Acad Sci U S A. 2005;102:15545–15550. doi: 10.1073/pnas.0506580102.  [DOI](https://doi.org/10.1073/pnas.0506580102) | [PMC free article](/articles/PMC1239896/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16199517/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&title=Gene%20set%20enrichment%20analysis:%20a%20knowledge-based%20approach%20for%20interpreting%20genome-wide%20expression%20profiles&author=A%20Subramanian&author=P%20Tamayo&author=VK%20Mootha&author=S%20Mukherjee&author=BL%20Ebert&volume=102&publication_year=2005&pages=15545-15550&pmid=16199517&doi=10.1073/pnas.0506580102&)

31. Fritah S, Niclou SP, Azuaje F. Databases for lncRNAs: a comparative evaluation of emerging tools. Rna. 2014;20:1655–1665. doi: 10.1261/rna.044040.113.  [DOI](https://doi.org/10.1261/rna.044040.113) | [PMC free article](/articles/PMC4201818/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25323317/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Rna&title=Databases%20for%20lncRNAs:%20a%20comparative%20evaluation%20of%20emerging%20tools&author=S%20Fritah&author=SP%20Niclou&author=F%20Azuaje&volume=20&publication_year=2014&pages=1655-1665&pmid=25323317&doi=10.1261/rna.044040.113&)

32. Gupta RA, Shah N, Wang KC, Kim J, Horlings HM, Wong DJ, et al. Long non-coding RNA HOTAIR reprograms chromatin state to promote cancer metastasis. Nature. 2010;464:1071–1076. doi: 10.1038/nature08975.  [DOI](https://doi.org/10.1038/nature08975) | [PMC free article](/articles/PMC3049919/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20393566/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Long%20non-coding%20RNA%20HOTAIR%20reprograms%20chromatin%20state%20to%20promote%20cancer%20metastasis&author=RA%20Gupta&author=N%20Shah&author=KC%20Wang&author=J%20Kim&author=HM%20Horlings&volume=464&publication_year=2010&pages=1071-1076&pmid=20393566&doi=10.1038/nature08975&)

33. Mercer TR, Mattick JS. Structure and function of long noncoding RNAs in epigenetic regulation. Nature structural & molecular biology. 2013;20:300–307. doi: 10.1038/nsmb.2480.  [DOI](https://doi.org/10.1038/nsmb.2480) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23463315/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature%20structural%20&%20molecular%20biology&title=Structure%20and%20function%20of%20long%20noncoding%20RNAs%20in%20epigenetic%20regulation&author=TR%20Mercer&author=JS%20Mattick&volume=20&publication_year=2013&pages=300-307&pmid=23463315&doi=10.1038/nsmb.2480&)

34. Ma L, Liu J, Liu L, Duan G, Wang Q, Xu Y, et al. Overexpression of the transcription factor MEF2D in hepatocellular carcinoma sustains malignant character by suppressing G2-M transition genes. Cancer Res. 2014;74:1452–1462. doi: 10.1158/0008-5472.CAN-13-2171.  [DOI](https://doi.org/10.1158/0008-5472.CAN-13-2171) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24390737/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=Overexpression%20of%20the%20transcription%20factor%20MEF2D%20in%20hepatocellular%20carcinoma%20sustains%20malignant%20character%20by%20suppressing%20G2-M%20transition%20genes&author=L%20Ma&author=J%20Liu&author=L%20Liu&author=G%20Duan&author=Q%20Wang&volume=74&publication_year=2014&pages=1452-1462&pmid=24390737&doi=10.1158/0008-5472.CAN-13-2171&)

35. Gutierrez-Aguilar M, Baines CP. Physiological and pathological roles of mitochondrial SLC25 carriers. The Biochemical journal. 2013;454:371–386. doi: 10.1042/BJ20121753.  [DOI](https://doi.org/10.1042/BJ20121753) | [PMC free article](/articles/PMC3806213/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23988125/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=The%20Biochemical%20journal&title=Physiological%20and%20pathological%20roles%20of%20mitochondrial%20SLC25%20carriers&author=M%20Gutierrez-Aguilar&author=CP%20Baines&volume=454&publication_year=2013&pages=371-386&pmid=23988125&doi=10.1042/BJ20121753&)

36. Rehman M, Tamagnone L. Semaphorins in cancer: biological mechanisms and therapeutic approaches. Seminars in cell & developmental biology. 2013;24:179–189. doi: 10.1016/j.semcdb.2012.10.005.  [DOI](https://doi.org/10.1016/j.semcdb.2012.10.005) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23099250/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Seminars%20in%20cell%20&%20developmental%20biology&title=Semaphorins%20in%20cancer:%20biological%20mechanisms%20and%20therapeutic%20approaches&author=M%20Rehman&author=L%20Tamagnone&volume=24&publication_year=2013&pages=179-189&pmid=23099250&doi=10.1016/j.semcdb.2012.10.005&)

37. Malik MF, Ye L, Jiang WG. The Plexin-B family and its role in cancer progression. Histology and histopathology. 2014;29:151–165. doi: 10.14670/HH-29.151.  [DOI](https://doi.org/10.14670/HH-29.151) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24043639/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Histology%20and%20histopathology&title=The%20Plexin-B%20family%20and%20its%20role%20in%20cancer%20progression&author=MF%20Malik&author=L%20Ye&author=WG%20Jiang&volume=29&publication_year=2014&pages=151-165&pmid=24043639&doi=10.14670/HH-29.151&)

38. Weber RG, Rieger J, Naumann U, Lichter P, Weller M. Chromosomal imbalances associated with response to chemotherapy and cytotoxic cytokines in human malignant glioma cell lines. Int J Cancer. 2001;91:213–218. doi: 10.1002/1097-0215(200002)9999:9999<::aid-ijc1036>3.3.co;2-d.  [DOI](https://doi.org/10.1002/1097-0215(200002)9999:9999<::aid-ijc1036>3.3.co;2-d) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11146447/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Cancer&title=Chromosomal%20imbalances%20associated%20with%20response%20to%20chemotherapy%20and%20cytotoxic%20cytokines%20in%20human%20malignant%20glioma%20cell%20lines&author=RG%20Weber&author=J%20Rieger&author=U%20Naumann&author=P%20Lichter&author=M%20Weller&volume=91&publication_year=2001&pages=213-218&pmid=11146447&doi=10.1002/1097-0215(200002)9999:9999<::aid-ijc1036>3.3.co;2-d&)

39. Gregory SG, Barlow KF, McLay KE, Kaul R, Swarbreck D, Dunham A, et al. The DNA sequence and biological annotation of human chromosome 1. Nature. 2006;441:315–321. doi: 10.1038/nature04727.  [DOI](https://doi.org/10.1038/nature04727) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16710414/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=The%20DNA%20sequence%20and%20biological%20annotation%20of%20human%20chromosome%201&author=SG%20Gregory&author=KF%20Barlow&author=KE%20McLay&author=R%20Kaul&author=D%20Swarbreck&volume=441&publication_year=2006&pages=315-321&pmid=16710414&doi=10.1038/nature04727&)

40. Murakami M, Nakagawa M, Olson EN, Nakagawa O. A WW domain protein TAZ is a critical coactivator for TBX5, a transcription factor implicated in Holt-Oram syndrome. Proc Natl Acad Sci U S A. 2005;102:18034–18039. doi: 10.1073/pnas.0509109102.  [DOI](https://doi.org/10.1073/pnas.0509109102) | [PMC free article](/articles/PMC1312418/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16332960/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&title=A%20WW%20domain%20protein%20TAZ%20is%20a%20critical%20coactivator%20for%20TBX5,%20a%20transcription%20factor%20implicated%20in%20Holt-Oram%20syndrome&author=M%20Murakami&author=M%20Nakagawa&author=EN%20Olson&author=O%20Nakagawa&volume=102&publication_year=2005&pages=18034-18039&pmid=16332960&doi=10.1073/pnas.0509109102&)

41. Oh H, Irvine KD. Yorkie: the final destination of Hippo signaling. Trends in cell biology. 2010;20:410–417. doi: 10.1016/j.tcb.2010.04.005.  [DOI](https://doi.org/10.1016/j.tcb.2010.04.005) | [PMC free article](/articles/PMC2919348/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20452772/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Trends%20in%20cell%20biology&title=Yorkie:%20the%20final%20destination%20of%20Hippo%20signaling&author=H%20Oh&author=KD%20Irvine&volume=20&publication_year=2010&pages=410-417&pmid=20452772&doi=10.1016/j.tcb.2010.04.005&)

42. Hall CA, Wang R, Miao J, Oliva E, Shen X, Wheeler T, et al. Hippo pathway effector Yap is an ovarian cancer oncogene. Cancer Res. 2010;70:8517–8525. doi: 10.1158/0008-5472.CAN-10-1242.  [DOI](https://doi.org/10.1158/0008-5472.CAN-10-1242) | [PMC free article](/articles/PMC2970655/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20947521/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=Hippo%20pathway%20effector%20Yap%20is%20an%20ovarian%20cancer%20oncogene&author=CA%20Hall&author=R%20Wang&author=J%20Miao&author=E%20Oliva&author=X%20Shen&volume=70&publication_year=2010&pages=8517-8525&pmid=20947521&doi=10.1158/0008-5472.CAN-10-1242&)

43. Kapoor A, Yao W, Ying H, Hua S, Liewen A, Wang Q, et al. Yap1 activation enables bypass of oncogenic Kras addiction in pancreatic cancer. Cell. 2014;158:185–197. doi: 10.1016/j.cell.2014.06.003.  [DOI](https://doi.org/10.1016/j.cell.2014.06.003) | [PMC free article](/articles/PMC4109295/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24954535/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell&title=Yap1%20activation%20enables%20bypass%20of%20oncogenic%20Kras%20addiction%20in%20pancreatic%20cancer&author=A%20Kapoor&author=W%20Yao&author=H%20Ying&author=S%20Hua&author=A%20Liewen&volume=158&publication_year=2014&pages=185-197&pmid=24954535&doi=10.1016/j.cell.2014.06.003&)

44. Shao DD, Xue W, Krall EB, Bhutkar A, Piccioni F, Wang X, et al. KRAS and YAP1 converge to regulate EMT and tumor survival. Cell. 2014;158:171–184. doi: 10.1016/j.cell.2014.06.004.  [DOI](https://doi.org/10.1016/j.cell.2014.06.004) | [PMC free article](/articles/PMC4110062/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24954536/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell&title=KRAS%20and%20YAP1%20converge%20to%20regulate%20EMT%20and%20tumor%20survival&author=DD%20Shao&author=W%20Xue&author=EB%20Krall&author=A%20Bhutkar&author=F%20Piccioni&volume=158&publication_year=2014&pages=171-184&pmid=24954536&doi=10.1016/j.cell.2014.06.004&)

45. Jeong W, Kim SB, Sohn BH, Park YY, Park ES, Kim SC, et al. Activation of YAP1 is associated with poor prognosis and response to taxanes in ovarian cancer. Anticancer Res. 2014;34:811–817.  [PMC free article](/articles/PMC4082822/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24511017/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Anticancer%20Res&title=Activation%20of%20YAP1%20is%20associated%20with%20poor%20prognosis%20and%20response%20to%20taxanes%20in%20ovarian%20cancer&author=W%20Jeong&author=SB%20Kim&author=BH%20Sohn&author=YY%20Park&author=ES%20Park&volume=34&publication_year=2014&pages=811-817&pmid=24511017&)

46. Tuft Stavnes H, Nymoen DA, Hetland Falkenthal TE, Kaern J, Trope CG, Davidson B. APOA1 mRNA expression in ovarian serous carcinoma effusions is a marker of longer survival. American journal of clinical pathology. 2014;142:51–57. doi: 10.1309/AJCPD8NBSHXRXQL7.  [DOI](https://doi.org/10.1309/AJCPD8NBSHXRXQL7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24926085/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=American%20journal%20of%20clinical%20pathology&title=APOA1%20mRNA%20expression%20in%20ovarian%20serous%20carcinoma%20effusions%20is%20a%20marker%20of%20longer%20survival&author=H%20Tuft%20Stavnes&author=DA%20Nymoen&author=TE%20Hetland%20Falkenthal&author=J%20Kaern&author=CG%20Trope&volume=142&publication_year=2014&pages=51-57&pmid=24926085&doi=10.1309/AJCPD8NBSHXRXQL7&)

47. Su F, Kozak KR, Imaizumi S, Gao F, Amneus MW, Grijalva V, et al. Apolipoprotein A-I (apoA-I) and apoA-I mimetic peptides inhibit tumor development in a mouse model of ovarian cancer. Proc Natl Acad Sci U S A. 2010;107:19997–20002. doi: 10.1073/pnas.1009010107.  [DOI](https://doi.org/10.1073/pnas.1009010107) | [PMC free article](/articles/PMC2993420/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21041624/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&title=Apolipoprotein%20A-I%20(apoA-I)%20and%20apoA-I%20mimetic%20peptides%20inhibit%20tumor%20development%20in%20a%20mouse%20model%20of%20ovarian%20cancer&author=F%20Su&author=KR%20Kozak&author=S%20Imaizumi&author=F%20Gao&author=MW%20Amneus&volume=107&publication_year=2010&pages=19997-20002&pmid=21041624&doi=10.1073/pnas.1009010107&)

48. Bartek J, Hodny Z. PARK2 orchestrates cyclins to avoid cancer. Nat Genet. 2014;46:527–528. doi: 10.1038/ng.2992.  [DOI](https://doi.org/10.1038/ng.2992) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24866187/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=PARK2%20orchestrates%20cyclins%20to%20avoid%20cancer&author=J%20Bartek&author=Z%20Hodny&volume=46&publication_year=2014&pages=527-528&pmid=24866187&doi=10.1038/ng.2992&)

49. Bolton KL, Tyrer J, Song H, Ramus SJ, Notaridou M, Jones C, et al. Common variants at 19p13 are associated with susceptibility to ovarian cancer. Nat Genet. 2010;42:880–884. doi: 10.1038/ng.666.  [DOI](https://doi.org/10.1038/ng.666) | [PMC free article](/articles/PMC3125495/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20852633/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Common%20variants%20at%2019p13%20are%20associated%20with%20susceptibility%20to%20ovarian%20cancer&author=KL%20Bolton&author=J%20Tyrer&author=H%20Song&author=SJ%20Ramus&author=M%20Notaridou&volume=42&publication_year=2010&pages=880-884&pmid=20852633&doi=10.1038/ng.666&)

50. Howie B, Fuchsberger C, Stephens M, Marchini J, Abecasis GR. Fast and accurate genotype imputation in genome-wide association studies through pre-phasing. Nat Genet. 2012;44:955–959. doi: 10.1038/ng.2354.  [DOI](https://doi.org/10.1038/ng.2354) | [PMC free article](/articles/PMC3696580/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22820512/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Fast%20and%20accurate%20genotype%20imputation%20in%20genome-wide%20association%20studies%20through%20pre-phasing&author=B%20Howie&author=C%20Fuchsberger&author=M%20Stephens&author=J%20Marchini&author=GR%20Abecasis&volume=44&publication_year=2012&pages=955-959&pmid=22820512&doi=10.1038/ng.2354&)
