# A Pharmacogenetic Candidate Gene Study of Tenofovir-Associated Fanconi Syndrome

## Metadata
**Authors:** Amber Dahlin, Matthias Wittwer, Melanie de la Cruz, Jonathan M Woo, Rujuta Bam, Valeska Scharen-Guivel, John Flaherty, Adrian S Ray, Tomas Cihlar, Samir K Gupta, Kathleen M Giacomini
**Journal:** Pharmacogenetics and genomics
**Date:** 2015 Feb
**DOI:** [10.1097/FPC.0000000000000110](https://doi.org/10.1097/FPC.0000000000000110)
**PMID:** 25485598
**PMCID:** PMC4331349
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4331349/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC4331349/pdf/nihms639831.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC4331349/pdf/nihms639831.pdf)

## Abstract

**BACKGROUND:** 
Tenofovir disoproxil fumarate (TDF) is a widely used antiretroviral agent with favorable efficacy, safety and tolerability profiles. However, renal adverse events, including rare Fanconi syndrome (FS), may occur in a small subset of HIV-infected treated patients.

**OBJECTIVES:** 
The aim of this study was to identify genetic variants that may associate with TDF-associated FS (TDF-FS).

**METHODS:** 
DNA samples collected from a 19 cases with TDF-FS and 36 matched controls were sequenced and genetic association studies were performed in eight candidate genes: ATP-binding cassette (ABC) transporters ABCC2 (MRP2) and ABCC4 (MRP4), solute carrier family members SLC22A6 (OAT1) and SLC22A8 (OAT3), adenylate kinases 2 (AK2) and 4 (AK4), chloride transporter CIC-5 CLCN5, and Lowe syndrome protein OCRL. Functional effects of a SNP predicted to alter transport of tenofovir were then investigated in cells expressing an identified variant or ABCC4.

**RESULTS:** 
Overall, the case group showed a trend towards a higher proportion of rare alleles. Six SNPs in ABCC2 (3 SNPs), ABCC4 (1 SNP) and OCRL (2 SNPs) were associated with TDF-FS case status, but did not remain significant after correction for multiple testing. Six SNPs, in OCRL (4 SNPs) and ABCC2 (2 SNPs), were significantly associated with increased serum creatinine levels in the cases, and remained significant after multiple test correction (P < 2 × 10−04). One synonymous SNP in ABCC2 (rs8187707; P=2.10 ×10−04; β =−73.3 ml/min/1.73m2)) was also significantly associated with decreased estimated glomerular filtration rate of creatinine in the cases. However, these results were driven by rare SNPs present in a small number of severely affected cases. A previously uncharacterized, non-synonymous SNP, rs11568694, that was predicted to alter MRP4 function, had no significant effect on tenofovir cellular accumulation in vitro.

**CONCLUSIONS:** 
While no single predictive genetic marker for the development of TDF-FS was identified, these findings suggest that multiple rare variants in multiple genes involved in renal handling of tenofovir and/or renal cell homeostasis may be associated with increased susceptibility to TDF-FS.

Keywords: Fanconi Syndrome, tenofovir, HIV, transporter, pharmacogenetics

### BACKGROUND

Tenofovir disoproxil fumarate (TDF) is a widely used antiretroviral agent with favorable efficacy, safety and tolerability profiles. However, renal adverse events, including rare Fanconi syndrome (FS), may occur in a small subset of HIV-infected treated patients.

### OBJECTIVES

The aim of this study was to identify genetic variants that may associate with TDF-associated FS (TDF-FS).

### METHODS

DNA samples collected from a 19 cases with TDF-FS and 36 matched controls were sequenced and genetic association studies were performed in eight candidate genes: ATP-binding cassette (ABC) transporters *ABCC2* (MRP2) and *ABCC4* (MRP4), solute carrier family members *SLC22A6* (OAT1) and *SLC22A8* (OAT3), adenylate kinases 2 (AK2) and 4 (AK4), chloride transporter CIC-5 *CLCN5*, and Lowe syndrome protein *OCRL*. Functional effects of a SNP predicted to alter transport of tenofovir were then investigated in cells expressing an identified variant or *ABCC4*.

### RESULTS

Overall, the case group showed a trend towards a higher proportion of rare alleles. Six SNPs in *ABCC2* (3 SNPs), *ABCC4* (1 SNP) and *OCRL* (2 SNPs) were associated with TDF-FS case status, but did not remain significant after correction for multiple testing. Six SNPs, in *OCRL* (4 SNPs) and *ABCC2* (2 SNPs), were significantly associated with increased serum creatinine levels in the cases, and remained significant after multiple test correction (P < 2 × 10^−04^). One synonymous SNP in *ABCC2* (rs8187707; P=2.10 ×10^−04^; β =−73.3 ml/min/1.73m^2^)) was also significantly associated with decreased estimated glomerular filtration rate of creatinine in the cases. However, these results were driven by rare SNPs present in a small number of severely affected cases. A previously uncharacterized, non-synonymous SNP, rs11568694, that was predicted to alter MRP4 function, had no significant effect on tenofovir cellular accumulation *in vitro*.

### CONCLUSIONS

While no single predictive genetic marker for the development of TDF-FS was identified, these findings suggest that multiple rare variants in multiple genes involved in renal handling of tenofovir and/or renal cell homeostasis may be associated with increased susceptibility to TDF-FS.

## INTRODUCTION

Tenofovir disoproxil fumarate (TDF, Viread^®^) is an orally administered prodrug of tenofovir, a nucleotide inhibitor of the human immunodeficiency virus (HIV) and hepatitis B virus reverse transcriptases. TDF is part of preferred regimens for the treatment of HIV recommended by the Department of Health and Human Services ([1](#R1), [2](#R2)) and the European AIDS Clinical Society ([3](#R3)), and is widely used for the treatment of HIV in both treatment-naïve and treatment-experienced patients. Approved for the treatment of HIV by the U.S. Food and Drug Administration in 2001, TDF has well-established safety and tolerability profiles with more than 9 million patient years of clinical experience ([4](#R4)). Among rare adverse effects, low rates of TDF-associated renal adverse events leading to discontinuation of treatment (0–3%) have been reported in controlled trials ([2](#R2), [5](#R5), [6](#R6)). In the expanded access program for TDF that included > 10,000 patients, graded elevations in serum creatinine (SCr) were observed in 2.2% of patients, and serious renal adverse events of any type occurred in 0.5% of patients, including rare cases (<0.1%) of Fanconi syndrome (FS) ([7](#R7)). FS is characterized by wasting of electrolytes, amino acids and glucose, metabolic acidosis, and occasionally by reductions in renal creatinine clearance ([8](#R8)–[15](#R15)).

The renal toxicity associated with TDF use is multi-factorial, with the mechanism and some of the underlying risk factors not fully understood. Clinical and demographic factors associated with the increased risk of TDF-associated renal adverse events, including rare cases of FS, include increased age, lower body weight, pre-existing renal impairment, diabetes, high blood pressure, and previous or concomitant use of protease inhibitors or nephrotoxic drugs ([7](#R7), [16](#R16)). The severity of renal toxicity associated with TDF shows high inter-individual variability, suggesting that host genetics may play a role in susceptibility to this adverse effect ([17](#R17)). Genetic screens to identify variants in suspected candidate genes have shown that various single nucleotide polymorphisms (SNPs), including those in genes putatively involved in tenofovir transport may be implicated in the risk for TDF-related renal dysfunction. Izzedine et al. presented the first evidence of an association between TDF-related renal toxicity and genetic variants within the renal efflux transporter *ABCC2* (gene product MRP2) ([15](#R15)). Other polymorphisms in *SLC22A6* (gene product OAT1) and *ABCC4* (gene product MRP4) with potential functional consequences for TDF transport have also been identified ([15](#R15), [18](#R18)–[21](#R21)). Non-synonymous variants within *ABCC2* were associated with greater renal tubular dysfunction, while a non-coding variant in *ABCC4* was associated with higher intracellular concentrations of tenofovir ([15](#R15), [17](#R17)). However, these studies centered on cases of mild and moderate TDF-associated renal toxicity, and not upon the rarer and more severe TDF-FS. The genetic studies performed previously were undertaken in patients with evidence of proximal tubulopathy, although none had actual decline in renal functions. As an extension to a prospective epidemiological study of demographic factors involved in TDF-FS, we obtained DNA samples from 19 cases and 36 matching control HIV-infected patients who participated in this study ([22](#R22)).

Here, we report on a genetic analysis of a set of genes encoding transporters in the renal proximal tubules that have been previously suggested or directly implicated in tenofovir renal disposition including *organic anion transporter 1, SLC22A6* (hOAT1); *organic anion transporter 3, SLC22A8* (hOAT3)*; multidrug resistance protein 2*, *ABCC2 (*MRP2); and *multidrug resistance protein 4*, *ABCC4* (MRP4) ([18](#R18), [19](#R19), [23](#R23)–[32](#R32)). While MRP2 efflux pump is not directly involved in renal transport of tenofovir ([33](#R33)), several genetic studies have suggested its association with TDF-mediated renal dysfunction ([15](#R15), [18](#R18), [19](#R19), [27](#R27), [34](#R34)). In addition, we included genes encoding enzymes involved in the intracellular metabolic activation of tenofovir (*adenylate kinase 2*, *AK2; adenylate kinase 4*, *AK4*) ([35](#R35), [36](#R36)) and genes that are associated with Mendelian Fanconi Syndrome (*oculocerebrorenal lowe protein 1*, *OCRL; chloride channel 5*, *CLCN5)* ([37](#R37), [38](#R38)). We used a full sequencing, rather than a genotyping, approach to capture rare as well as common variants in these candidate genes. While no distinct predictive genetic markers for the development of TDF-FS have been identified, this study identified multiple low frequency genetic loci that may play a role in the development of TDF-FS.

## METHODS

### DNA Samples

The DNA samples sequenced in this study were obtained as a part of a recent case-controlled (1:2), cross-sectional cohort study ([22](#R22)). The controls were matched to cases by duration of TDF treatment, race and age. Nineteen FS cases and 36 matched controls were identified at 9 sites in the US and Canada over a 2.5-year period ([22](#R22)). Whole blood samples (30 mL) were obtained from the cohort and genomic DNA was extracted from the blood samples by the use of QIAamp DNA Blood Mini Kit (Qiagen), according to manufacturer’s protocol. This study was approved by the ethics committee of the UCSF Committee on Human Research as well as by those of the individual participating clinical sites. All subjects provided informed consent. Detailed cohort information and clinical outcomes for subjects evaluated in this study are described elsewhere ([22](#R22)). Clinical and demographic information for the subjects evaluated in this study is provided in [Supplemental Table 1](#SD2).

### Selection of Pharmacogenetic Candidate Genes

Eight selected candidate genes were evaluated in this study, of which six are directly or potentially involved in the active renal secretion pathway or intracellular metabolism of tenofovir. We hypothesized that genetic variants in these pathway genes may increase the renal accumulation of tenofovir and/or its metabolites in proximal tubule cells, thereby predisposing TDF-treated patients to the onset of FS. The selected genes included OAT1 (*SLC22A6*), OAT3 (*SLC22A8*) MRP2 (*ABCC2*), MRP4 (*ABCC4*), *AK2* and *AK4*. In addition, two genes known to be involved in the development of Mendelian FS have also been included in the analysis: *OCRL* and *CLCN5*.

### DNA Sequencing and SNP Discovery

Sequencing and SNP analysis were performed by Beckman-Coulter Genomics and the UCSF Genome Center Core (UCSF, San Francisco CA). PCR primers were designed to amplify ~200 base pairs (bp) from intron-exon boundaries for each gene using the Primer3 software. The final concentrations of the PCR components were 2.5 mM MgCl_2_, 0.1 mM dNTPs, 0.025 U of Platinum Taq polymerase (Invitrogen), 2% DMSO, 1X PCR Buffer, 200 μM PCR primers and 10 ng of DNA template. The 2 μL PCR reaction was run with the following conditions: 95°C for 5 min, [94 °C for 20 s, 65 °C for 20 s (0.5 °C decrease per cycle), 72 °C for 45 s; 14 cycles],[94 °C for 20 s, 58 °C for 20 s, 72 °C for 45 s; 35 cycles],72 °C for 10 min. The resulting PCR products were treated with 0.5 U of shrimp alkaline phosphatase (SAP) and 0.5 U of ExoI at 37 °C for 60 min, then 90 °C for 15 min. The PCR products were then used as a template for the sequencing reaction with BigDye Terminator (Applied Biosystems). The sequencing reaction consisted of final concentrations of sequencing buffer, BigDye Terminator mix, 500 μM sequencing primer and PCR product template. The running conditions for the sequencing reaction were: 96 °C for 1 min, [96 °C for 10 s, 55 °C for 5 s, 60 °C for 4 min; 25 cycles]. The sequencing reaction was cleaned up with X-Terminator (Applied Biosystems) and analyzed on the 3730xl DNA Analyzer (Applied Biosystems). The resulting sequencing data were viewed through Sequencher (GeneCodes) in order to identify variations.

### Quality Control (QC) and Data Analysis

The software PLINK v.1.07 ([39](#R39)) was used for data QC. SNPs with a study-wise missing data proportion above 0.05 were removed from the analysis. SNPs failing to meet Hardy-Weinberg equilibrium (HWE) (P < 0.0001), in addition to SNPs with more than 10% missing genotypes were also eliminated from the analysis. After QC, the total remaining SNP counts and minor allele frequencies (MAF) for SNP alleles were determined for each gene and individual. Linkage disequilibrium among SNPs was determined using PLINK and the SNP Annotation and Proxy Search (SNAP) web-server ([http://www.broadinstitute.org/mpg/snap/index.php](http://www.broadinstitute.org/mpg/snap/index.php)) ([40](#R40)).

### Genetic Association Analysis

To identify associations of genetic variants with disease status, logistic and linear regression tests were performed using PLINK v.1.07, and the significance of association was determined using Fisher’s exact (allelic) test with a nominal significance threshold of 0.05. For all tests, the genotype counts, allele frequencies, effect size estimates with 95% confidence intervals (CI) and P-values were computed. To detect associations of SNP alleles with quantitative phenotypic measures (continuous phenotypes), the following clinical measurements were obtained and incorporated into regression models: change in glomerular filtration rate from pre-TDF treatment baseline (ΔeGFR) (as calculated using the Modification of Diet in Renal Disease (MDRD) formula ([41](#R41))) and change in SCr from pre-TDF treatment baseline (ΔSCr). Linear regression models, initially specifying the genotype as an additive covariate and the clinical measurement values or case-control status as a response, and adjusted for covariates (age, sex, race, baseline SCr or eGFR, and bodyweight), were analyzed using PLINK. Values for demographic covariates (age, sex, race and bodyweight) are reported in Gupta et al ([22](#R22)). The Bonferroni correction, defined as 0.05/n where n is the total number of SNPs tested, was applied to nominally significant P values; therefore P values below 2.23 × 10^−04^ were considered significant after correction for multiple testing. For clinical phenotype association analyses, cases and controls were first analyzed separately and the resulting coefficients were compared between the two groups.

### SNP Functional Annotation

Several web-based annotation tools available through the SNP-Nexus web-server ([http://snp-nexus.org](http://snp-nexus.org)) were used for SNP annotation ([42](#R42)–[44](#R44)). Within SNP-Nexus, the SIFT, PolyPhen and FastSNP web servers were applied to the SNP dataset to determine the predicted effects and ranking of functional consequences of SNPs.

### Generation of HEK-cells overexpressing MRP4 reference and MRP4-V854F

The pcDNA5/FRT vector (Invitrogen, Carlsbad, CA) containing the *ABCC4-V854F* sequence was constructed as described elsewhere ([30](#R30)). The full-length *ABCC4* sequence was used as reference cDNA and as template for generation of the variant cDNA. The desired sequence alteration for the variant cDNA was performed using the QuickChange Lightning site-directed mutagenesis kit (Agilent Technologies, Santa Clara, CA) according to the manufacturer’s protocol and as described elsewhere ([45](#R45)). The following primers were used: forward 5′-CAAGTGGTTGGTGTGTTCTCTGTGGCTGTGG-3′ and reverse 5′-CCACAGCCACAGAGAACACACCAACCACTTG-3′. Complete sequencing of the constructs confirmed the presence of the mutation and ensured that no new mutations had been introduced during the mutagenesis. Human embryonic kidney (HEK) cells were transiently transfected with either pcDNA5/FRT-ABCC4-reference or pcDNA5/FRT-ABCC4-V854F expression constructs, or with pcDNA5/FRT (empty vector) as control, as described elsewhere ([45](#R45)). In brief, HEK-Flpin cells (Invitrogen) were seeded in DMEM (Cell Culture Facility, University of California, San Francisco, CA) supplemented with 10% FBS in poly-D-lysine-coated 24-well plates at a density of 600,000 cells/ml. Approximately 24 hours later, the cells were transfected using Lipofectamine 2000 (Invitrogen) in OptiMEM-medium (Cell Culture Facility, University of California, San Francisco, CA) according to the manufacturer’s protocol. The medium was changed back to DMEM containing 10% FBS approximately 12 hours after transfection and transport experiments were performed approximately 48 hours after transfection.

### Cellular Efflux Assay

A previously described method of pre-loading MRP4-expressing cells with TDF under ATP-depleting conditions was used for the MRP4-dependent efflux studies of TFV ([31](#R31)). Unlike TFV, TDF is highly cell permeable and it is converted to parent TFV inside cells ([31](#R31)). The ATP-depleting conditions ([46](#R46)) allow for sufficient accumulation of TFV released from TDF in the presence of functionally active MRP4 protein. To perform the TFV efflux studies, HEK cells transfected with pcDNA5/FRT-ABCC4-reference, pcDNA5/FRT-ABCC4-V854F or pcDNA5/FRT (control) were preincubated at 37 °C for two hours under ATP-depleting conditions with 1 μM [^3^H]TDF (Moravek Radiochemicals, Brea, CA) in a glucose-free Dulbecco’s Modified Eagle’s Medium (DMEM, Invitrogen) supplemented with 10 mM NaN3 and 10 mM 2-deoxy-D-glucose (both from Sigma-Aldrich, St. Louis, MO). After washing the cells twice with ice-cold phosphate-buffered saline (PBS, Cell Culture Facility, University of California, San Francisco, CA), complete pre-warmed DMEM (500 μL/well) was added to 21 wells, the three remaining wells being controls at time 0. The plate was then incubated at 37 °C and the media was removed from the wells after 2, 5, 10, 15, 30, 45, and 60 minutes after washing, and added to 3 mL EcoLite scintillation fluid (MP Bio, Solon, OH). All cells were immediately washed twice with ice-cold PBS after removal of the media. The cells were lysed in 800 μL lysis buffer (0.1 N NaOH and 0.1% SDS in distilled water) by shaking them for 2.5 hours at room temperature. 650 μL of the lysate was then added to 3 mL EcoLite scintillation fluid (MP Bio). Radioactivity of both cell lysates and media was determined on a LS6500 Scintillation Counter (Beckman Coulter, Pasadena, CA) and values were corrected for protein concentration as determined with a BCA assay kit (Thermo Scientific, Rockford, IL). Final values are expressed as % of the maximal values after 60 minutes and are the result of two independent experiments.

## RESULTS

### SNP Identification and Annotation

In this study, we sequenced all coding (exon), flanking (intron/exon junctions) and regulatory (promoter, 5′ untranslated region (5′UTR) and 3′ untranslated region (3′UTR)) regions of eight functionally relevant candidate genes (*ABCC2, ABCC4, SLC22A6, SLC22A8, OCRL, CLCN5, AK1, AK2*), in an effort to identify SNPs associated with TDF-FS. The resulting dataset was comprehensively evaluated by single SNP association and gene set analyses.

After QC of sequencing data, a total of 214 SNPs from all sequenced regions were available for analysis. A descriptive summary of all identified SNPs is provided in [Supplemental Table 2](#SD2). The majority of SNPs found in this analysis were located within intronic regions (N=144; 67.3%), while coding SNPs (N=56) accounted for 26.2% of the total number of SNPs identified ([Figure 1a](#F1)). The remaining SNPs (N=14; 6.5%) were found in the 3′UTR and 5′UTR ([Figure 1a](#F1)). In comparison to the control group, data from the case group suggested a greater proportion of SNPs located within the 3′UTRs of the sequenced genes and greater proportion of coding synonymous SNPs ([Figure 1b](#F1); *P* value not significant).

### Figure 1. Characteristics of SNPs Identified in TDF-FS Cases and Controls.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9923/4331349/016a8bec15b3/nihms639831f1.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4331349_nihms639831f1.jpg)

(A) Overall SNP frequency in the study subject samples by predicted functional category: “3utr” = 3′untranslated region, “5utr” = 5′untranslated region, “coding” = all synonymous and non-synonymous coding SNPs, “intronic” = all SNPs located within the intron/exon flanking regions (< 200 bp from exon termini). (B) Boxplots represent median MAF ± IQR (y-axis) in cases and controls by predicted functional category (colored squares).

Of the 56 coding SNPs, 28 SNPs encoded non-synonymous amino acid changes and 28 SNPs encoded synonymous amino acid changes ([Supplemental Table 2](#SD2)). The predicted effects of the non-synonymous SNPs on protein function were determined using SIFT and PolyPhen algorithms. Five SNPs (101567211A>T (*ABCC2*), 101591491T>G (*ABCC2*), 101594210T>C (*ABCC2*), rs11568658 (*ABCC4*) and rs8187692 (*ABCC2*)) were predicted by both SIFT and PolyPhen as likely to disrupt protein function while another 13 SNPs were predicted by both algorithms to have no effect ([Supplemental Table 2](#SD2)). Analysis of the other SNPs was discordant between the two algorithms.

The mean minor allele frequencies (MAF) and predicted functional effects of SNPs within the whole cohort, and the case and control groups, were determined and compared. Overall, a large proportion of variants identified in this analysis represented rare alleles (MAF<10%) ([Figure 2a](#F2)). While the cumulative SNP distributions were not significantly different overall between the cases and the controls, the case group showed a trend towards a higher proportion of rare alleles ([Figure 2b](#F2)). Consistent with this observation, between the two groups, the mean MAF was significantly higher in the controls (MAF, controls = 0.098; MAF, cases = 0.088; *P*=0.033) ([Figure 2c](#F2)).

### Figure 2. Distribution of Rare Variants.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9923/4331349/287ac8f9c9dc/nihms639831f2.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4331349_nihms639831f2.jpg)

(A) Frequency distribution of alleles in all cohort subjects. (B) Density plot of the cumulative distribution of SNPs in cases (red line) and controls (black line) by number of SNPs (x-axis). (C) Boxplots show median MAF ± IQR (plotted on the y-axis) in the cases and controls.

### Results of Single SNP Association with TDF-FS

We first evaluated the association of the 214 individual genotypes (single SNP analysis) in TDF-FS cases (n=19) and controls (n=36). Complete association results for all SNPs is provided in [Supplemental Table 2](#SD2). Six SNPs were present below a nominal P-value of 0.05, of which four SNPs possessed higher allele frequencies in the cases compared to the controls ([Table 1](#T1)). However, none of the six identified SNPs remained significant after correction for multiple testing (*P* < 2.23 ×10^−04^). Three of these six identified SNPs were present in *ABCC2* and one SNP was found in *ABCC4,* while the remaining two SNPs were found in *OCRL*. One SNP, rs7899457, encoded a synonymous amino acid change in *ABCC2*, while the remaining SNPs were intronic.

### Table 1.

| SNP | Chr. | Chr. Pos. | Gene Symbol | SNP Effect | Amino Acid Change | Minor Allele Counts |  | MAF |  | Odds Ratio | P Value‡ |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
|   |  |  |  |  |  |  |  |  |  |  |  |
| Cases | Controls | Cases | Controls |  |  |  |  |  |  |  |  |
| rs17222519 | 10 | 101554150 | ABCC2 | intronic |   | 0 | 11 | 0.00 | 0.15 | 0 | 0.015 |
| rs7899457 | 10 | 101605503 | ABCC2 | coding | Lys = Lys | 3 | 0 | 0.10 | 0.00 | - | 0.027 |
| rs17216177 | 10 | 101603522 | ABCC2 | intronic |   | 13 | 11 | 0.34 | 0.16 | 2.79 | 0.032 |
| rs2274409 | 13 | 95860214 | ABCC4 | intronic |   | 7 | 29 | 0.18 | 0.43 | 0.30 | 0.018 |
| 128701401T>C | 23 | 128701401 | OCRL | intronic |   | 4 | 0 | 0.17 | 0.00 | - | 0.019 |
| 128699579T>C | 23 | 128699579 | OCRL | intronic |   | 3 | 0 | 0.15 | 0.00 | - | 0.039 |

Table 1 Caption: SNPs associated with TDF-FS case status.

Prior studies have implicated SNPs in *ABCC2* and *ABCC4* in TDF-related renal dysfunction, namely rs2273697, rs8187710, and rs899494, that were also identified in our analysis. However, none of these SNPs had significant associations with TDF-FS ([Supplemental Table 2](#SD2)).

### Association of SNPs with ΔSCr and ΔeGFR in TDF-FS cases

To identify SNPs associated with clinical parameters used to diagnose the cases, patient genotypes from the groups of cases (n=19) and controls (n=36) individually were tested for their association with changes in SCr from baseline (ΔSCr), and changes in estimated glomerular filtration rate from baseline (ΔeGFR). We have intentionally focused on analyzing the genetic associations for ΔSCr and/or ΔeGFR separately in cases and controls rather than across the whole cohort because of the profound differences in these parameters between the two groups of patients. [Supplemental Table 1](#SD2) summarizes the full description of the cohort and the clinical phenotypes that were evaluated.

Thirteen SNPs were nominally associated with increased ΔSCr in the cases, and six SNPs remained significant after multiple test correction ([Table 2](#T2)). The majority of these SNPs (10/13) were found in *OCRL,* and the remainder was located in *ABCC2.* Two intronic SNPs (128718244C>T and 128718318A>G) in *OCRL* were most significantly associated with increased ΔSCr in the cases (P = 5.46 × 10^−06^; β =28.5 mg/dL) ([Table 2](#T2)). However, it was evident that for nearly all of these SNPs, highly significant association results were driven by extreme changes in serum creatinine occurring in only a few cases ([Supplemental Figure 1](#SD1), panels A–F). In particular, the two *OCRL* SNPs were singletons present in a case subject with the highest ΔSCr in the cohort ([Supplemental Figure 1](#SD1), panels E and F). This same case subject, whose ΔSCr was measured at 27.1 mg/dL, also carried at least one variant allele of the other five SNPs ([Supplemental Figure 1](#SD1)). For comparison, the data for the same 13 SNPs in controls were also evaluated; however, none of SNPs were associated with ΔSCr at nominal *P* <0.05 (data not shown).

### Table 2.

| SNP | Chr. | Chr. Pos. | Gene Symbol | SNP Effect | Amino Acid Change | MAF | β (mg/dL) | P Value‡ |
| --- | --- | --- | --- | --- | --- | --- | --- | --- |
| 128718244C>T*1 | 23 | 128718244 | OCRL | intronic |   | 0.06 | 28.5 | 5.46 × 10−06 |
| 128718318A>G*1 | 23 | 128718318 | OCRL | intronic |   | 0.06 | 28.5 | 5.46 × 10−06 |
| rs113165732* | 23 | 128710213 | OCRL | intronic |   | 0.06 | 29.5 | 1.22 × 10−05 |
| rs79174032* | 10 | 101560106 | ABCC2 | intronic |   | 0.05 | 24.5 | 4.86 × 10−05 |
| rs8187707* | 10 | 101610533 | ABCC2 | coding | His = His | 0.12 | 20.3 | 6.80 × 10−05 |
| rs7057639* | 23 | 129418630 | OCRL | intronic |   | 0.05 | 24.8 | 2.19 × 10−04 |
| rs11597282 | 10 | 101590020 | ABCC2 | intronic |   | 0.08 | 12.5 | 1.02 × 10−02 |
| rs3131282 | 23 | 128721316 | OCRL | intronic |   | 0.33 | 12.6 | 1.37 × 10−02 |
| rs1159042 | 23 | 128722284 | OCRL | intronic |   | 0.35 | 12.9 | 1.83 × 10−02 |
| rs2076076 | 23 | 128721193 | OCRL | intronic |   | 0.19 | 10.9 | 1.83 × 10−02 |
| rs5977104 | 23 | 128679030 | OCRL | intronic |   | 0.25 | 11.0 | 2.66 × 10−02 |
| rs72614119 | 23 | 129418440 | OCRL | intronic |   | 0.10 | 12.1 | 2.89 × 10−02 |
| 128701401T>C | 23 | 128701401 | OCRL | intronic |   | 0.17 | 7.7 | 3.83 × 10−02 |

Table 2 Caption: Association of SNPs with Change in Serum Creatinine in 19 TDF-FS Cases.

A similar analysis in cases and controls was performed to assess the association of patient genotype with changes in ΔeGFR from baseline. In the cases, 11 SNPs present in *ABCC2*, *OCRL* and *ABCC4* met criteria for significance at the nominal P value threshold of 0.05, and one SNP, rs8187707, remained significant following multiple test correction ([Table 3](#T3)). The top-ranked associated SNP, rs8187707, encoded a synonymous amino acid change (His = His) in *ABCC2* (P=2.16 × 10^−04^; β=−73.3 ml/min/1.73m^2^) ([Table 3](#T3)). However, this SNP represented a singleton, and was present within a single, severely-affected case subject with a ΔSCr of 27.1 mg/dL and ΔeGFR of −104 mg/min/1.73m^2^ ([Supplemental Figure 1](#SD1), panel G). None of the 11 identified SNPs were associated at nominal significance levels in the controls (data not shown).

### Table 3.

| SNP | Chr. | Chr. Pos. | Gene Symbol | Predicted SNP Effect | Amino Acid Change | MAF | β (ml/min/1.73m2) | P Value‡ |
| --- | --- | --- | --- | --- | --- | --- | --- | --- |
| rs8187707* | 10 | 101610533 | ABCC2 | coding | His = His | 0.12 | −73.3 | 2.16 × 10−04 |
| 128718318A>G | 23 | 128718318 | OCRL | intronic |   | 0.06 | −93.7 | 4.34 × 10−03 |
| 128718244C>T | 23 | 128718244 | OCRL | intronic |   | 0.06 | −93.7 | 4.34 × 10−03 |
| rs11597282 | 10 | 101590020 | ABCC2 | intronic |   | 0.08 | −48.1 | 4.92 × 10−03 |
| rs7057639 | 23 | 129418630 | OCRL | intronic |   | 0.05 | −74.0 | 9.99 × 10−03 |
| rs79174032 | 10 | 101560106 | ABCC2 | intronic |   | 0.05 | −68.4 | 1.18 × 10−02 |
| rs113165732 | 23 | 128710213 | OCRL | intronic |   | 0.06 | −94.9 | 1.26 × 10−02 |
| rs2274405 | 13 | 95858978 | ABCC4 | coding | Ser = Ser | 0.09 | 62.6 | 1.75 × 10−02 |
| 128701401T>C | 23 | 128701401 | OCRL | intronic |   | 0.17 | −29.5 | 2.75 × 10−02 |
| rs17216282 | 10 | 101605550 | ABCC2 | intronic |   | 0.03 | 78.5 | 2.92 × 10−02 |
| 101591491T>G | 10 | 101591491 | ABCC2 | coding | Trp > Arg | 0.13 | −38.2 | 3.22 × 10−02 |

Table 3 Caption: Association of SNPs with Change in Glomerular Filtration Rate in 19 TDF-FS Cases.

Based on these results, *OCRL*, *ABCC2* and *ABCC4* were prioritized as candidate genes for TDF-FS.

### Functional Characterization of MRP4-V854F (rs11568694)

Based on its known importance for TDF transport, ABCC4 has been previously investigated as a candidate gene for TDF-FS. As the majority of known coding variants in *ABCC4* have previously been genotyped and functionally characterized ([30](#R30)), including those that we identified in our analysis as associated with TDF-FS, we first investigated the predicted effects of these variations on protein function and expression *in silico* (summarized in [Supplemental Table 3](#SD2)) and then evaluated the functional effects of an uncharacterized, non-synonymous candidate SNP (rs11568694) on the *ABCC4* protein product MPR4. We generated plasmid constructs of the reference (wild-type) and rs11568694-mutant *ABCC4* protein products (MRP4-ref and MRP4-V854F) and transiently expressed these constructs in HEK-293 cells. A time course of intracellular efflux of [^3^H]tenofovir was evaluated for the MRP4-ref-HEK293, MRP4-V854F-HEK293 and mock-HEK293 cell lines preloaded with [^3^H]TDF. For both MRP4-ref-HEK293 and MRP4-V854F-HEK293, significant time-dependent efflux of tenofovir was observed relative to the mock-transfected control cells ([Figure 3](#F3)). However, there was no significant difference in the intracellular and extracellular concentrations of tenofovir between the MRP4-ref and MRP4-V854F cell lines, indicating that the MRP4-V854F variant is unlikely to affect tenofovir cellular efflux. Although rs11568694 did not appear to alter MRP4-mediated efflux of tenofovir in our assay, this SNP may be in LD with a functional variant(s) that alter *ABCC4* expression or efflux activity. For example, multiple functional variants including rs9524765 and rs11568702 were predicted through sequence analysis of *ABCC4* ([Supplemental Table 2](#SD2) and [Supplemental Table 3](#SD2)) to have deleterious effects on protein expression through disruption of splice sites ([Supplemental Table 3](#SD2)). Additional studies will be required to characterize the effects of these SNPs on *ABCC4* expression and function.

### Figure 3. Functional Characterization of the MRP4-V854F SNP.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9923/4331349/de9b98bc3ebf/nihms639831f3.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4331349_nihms639831f3.jpg)

(A) Time course of tenofovir intracellular accumulation. (B) Time course of tenofovir accumulation in the supernatant (extracellular concentration). Y-axis values represent % of the reference (MRP4-ref) intracellular amount of tenofovir at t=0. Red squares indicate HEK-293-MRP4-reference cell line (reference). Yellow squares indicate HEK-293-MRP4-V854F cell line (variant). Blue squares represent HEK-293-mock cell line (untransfected control). Error bars indicate SEM.

## DISCUSSION

Renal impairment is one of the adverse events occurring among HIV-infected patients receiving antiretroviral therapy. More severe forms of renal toxicity, including FS, can occur rarely among patients receiving TDF as a component of their treatment. The development of TDF-FS shows high inter-individual variation, suggesting that patient genetics may be an important determinant of this rare adverse event. To investigate the genetic component of the TDF-FS development in HIV infected patients, we sequenced eight candidate genes and conducted SNP and gene-based association tests of case-control status and clinical measurements (ΔSCr, ΔeGFR) in an established TDF-FS cohort (add ref [22](#R22)).

Through single-SNP case-control association study, we identified six SNPs present in *OCRL*, *ABCC2* and *ABCC4* that were associated with TDF-FS case status at a nominal significance threshold ([Table 1](#T1)), although none remained significant after correction for multiple testing. We also identified 13 SNPs and 11 SNPs, respectively, that are nominally significantly associated with ΔSCr and ΔeGFR in cases ([Table 2](#T2) and [Table 3](#T3)). The top-ranked SNPs associated with ΔSCr were present in intronic regions of *OCRL* and *ABCC2*, while the top-ranked SNPs associated with ΔeGFR were also found within these genes. Seven SNPs from both analyses remained significant after the correction for multiple testing. Notably, none of these SNPs identified in cases were also significantly associated with these phenotypes in the controls. These SNPs may therefore represent novel candidate loci for development of TDF-FS. While encouraging, these results should be viewed in light of an important limitation, as these SNPs were very rare, occurring in 1–3 individuals from the case group only. Sensitivity analyses to determine the impact of individual genotypes on the association results showed that removal of the few carriers resulted in P values that still met nominal significance but no longer surpassed the multiple test correction threshold (data not shown). Furthermore, highly significant association P values for the SNPs from [Table 2](#T2) and [Table 3](#T3) were driven by singletons, primarily from a single, severely affected case subject with ΔScr of 27.1 mg/dL who expressed all six SNPs associated with clinical phenotypes ([Supplemental Figure 1](#SD1)). Thus, due to our limited sample sizes, and the presence of an “extreme” case outlier, our findings should be interpreted with caution.

Our study design has several novel aspects that collectively serve to enrich for causal variations within TDF-FS candidate genes. First, while previous studies of renal dysfunction related to TDF have evaluated a small number of candidate genes involved in the transport of tenofovir, we have expanded this list to include genes involved in the pharmacological pathway (transport and metabolism) of tenofovir in addition to genes that have important physiological roles in kidney homeostasis and are directly related to the development of heritable renal dysfunction. Second, full sequencing of the exons, intron/exon junctions and regulatory regions of the selected genes enriched for SNPs potentially related to protein expression, function, regulation and/or mRNA splicing, and also helped identify novel as well as rare variants that are likely to be missed through traditional genotyping approaches. Third, in addition to performing a single-SNP association analysis, we also evaluated two of the main clinical phenotypes that were used to diagnose renal dysfunction, ΔSCr and ΔeGFR. Previous genetic studies of TDF-associated nephrotoxicity have not evaluated SNP or gene associations with specific clinical parameters of renal function. Through these analyses, we identified multiple variations within *ABCC2, ABCC4* and *OCRL*, of which the latter was previously associated with the Mendelian form of FS, but has not been implicated in TDF-FS until now. Using this approach, we have successfully enriched for SNP and gene associations of TDF-FS.

It should be noted that ritonavir-boosted protease inhibitors (PIs) have been shown to be associated with small reduction in GFR in TDF-treated patients ([47](#R47), [48](#R48)). PI-based regimens were very frequent in both cases (84% at the time of study enrollment) and controls (51% at the time of study enrollment) ([22](#R22)). However, the SCr and eGFR changes observed in cases were much more profound than the established effects of PIs. While it is possible that there might be a synergy between genetic and pharmacological effects on creatinine secretion, it is unlikely that the profound changes in SCr and eGFR are induced only by the PI-containing antiretroviral regimen.

One of the limitations of this study is that, due to the rarity of TDF-FS, only a small number of cases could be accrued and were available for the genetic analysis. Evaluation of a larger patient cohort, or through meta-analyses of data from prior studies, would be required to confirm our findings. A second limitation of our investigation is that multiple transporters are capable of endogenous creatinine transport (e.g. OCT2, OAT2 and MATE-1) ([49](#R49)) and the genetic polymorphisms in these genes may influence the variability in creatinine clearance independently of TDF exposure. We accounted for the impact of transporter variation by adjusting for baseline creatinine clearance in our genetic association models; however, we identified four SNPs that were significantly associated with baseline serum creatinine values in the cohort. These SNPs were present within *AK2* (33489932G>A; MAF, cohort = 0.0091; β=1.66 mg/dL; P= 1.72 ×10^−08^), *SLC22A6* (rs4149173; MAF, cohort = 0.0091; β=1.66 mg/dL; P= 1.72 ×10^−08^), and *SLC22A8* ((rs45438191; MAF, cohort = 0.0091; β=1.66 mg/dL; P= 1.72 ×10^−08^) and 62763706G>A; MAF, cohort = 0.018; β = 1.00 mg/dL; P = 1.71 ×10^−06^)). However, none of these SNPs were among those we had identified as potential candidates for TDF-FS, and none remained individually significant in the cases. Finally, while additional transporters for TDF exist, e.g. *ABCC10*, we chose to focus our investigation on those genes for which the strongest evidence supports a potential pharmacogenetic effect, which includes the four major organic anion transporters evaluated in this study. Additional studies to evaluate a more comprehensive set of genes are needed in order to clarify the genetic heterogeneity underlying TDF-associated renal damage.

In summary, we have identified several SNPs, many of them novel, in multiple genes that appear to be enriched in cases of TDF-FS in a small cohort of patients treated with TDF. The genes that are potentially associated with increased risk of TDF-FS in HIV-infected patients include those that regulate the renal efflux of tenofovir (i.e. *ABCC4*) or other drugs and xenobiotics (i.e. *ABCC2),* in addition to a key regulator of renal cell homeostasis *(OCRL*). While MRP2 efflux pump (product of *ABCC2*) is unable to directly interact with tenofovir and mediate its cellular efflux ([33](#R33)), it is involved in handling of xenobiotics and other endogenous compounds that could adversely affect the homeostasis of renal proximal tubules. Collectively, these findings suggest that while multiple rare variants in several genes may be associated with increased susceptibility to TDF-FS, no single, distinct predictive genetic marker for the development of this adverse effect was identified.

## Supplementary Material

## Acknowledgments

**Funding Sources:** The authors received financial support from National Institutes of Health Grant GM61390 (support for K.G., M.W., J.W. and M.C.). A.D. acknowledges financial support from National Institutes of Health Fellowship R25 CA112355.

The authors thank Joel Mefford and Dr. John Witte at UCSF for their helpful suggestions regarding the genetic analysis. We also thank the following clinical site investigators for conducting the study and collecting samples for this genetic analysis: Albert Anderson (Atlanta), Indira Brar (Detroit), Claude Fortin (Montreal), Joseph Gathe (Houston), Wilbert Jordan (Los Angeles), Robert Kalayjian (Cleveland), Anita Rachlis (Toronto), and Christina Wyatt (New York). Finally, we thank the participants in this study for their generosity.

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Services DoHaH. Available from: http://aidsinfo.nih.gov/ContentFiles/AdultandAdolescentGL.pdf.  [http://aidsinfo.nih.gov/ContentFiles/AdultandAdolescentGL.pdf](http://aidsinfo.nih.gov/ContentFiles/AdultandAdolescentGL.pdf)

2. Gallant JE, Winston JA, DeJesus E, Pozniak AL, Chen SS, Cheng AK, et al. The 3-year renal safety of a tenofovir disoproxil fumarate vs. a thymidine analogue-containing regimen in antiretroviral-naive patients. AIDS. 2008;22(16):2155–63. doi: 10.1097/QAD.0b013e3283112b8e.  [DOI](https://doi.org/10.1097/QAD.0b013e3283112b8e) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18832879/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=AIDS&title=The%203-year%20renal%20safety%20of%20a%20tenofovir%20disoproxil%20fumarate%20vs.%20a%20thymidine%20analogue-containing%20regimen%20in%20antiretroviral-naive%20patients&author=JE%20Gallant&author=JA%20Winston&author=E%20DeJesus&author=AL%20Pozniak&author=SS%20Chen&volume=22&issue=16&publication_year=2008&pages=2155-63&pmid=18832879&doi=10.1097/QAD.0b013e3283112b8e&)

3. Society EAC. Available from: http://www.europeanaidsclinicalsociety.org/images/stories/EACS-Pdf/EacsGuidelines-v6.1-2edition.pdf.  [http://www.europeanaidsclinicalsociety.org/images/stories/EACS-Pdf/EacsGuidelines-v6.1-2edition.pdf](http://www.europeanaidsclinicalsociety.org/images/stories/EACS-Pdf/EacsGuidelines-v6.1-2edition.pdf)

4. Liver EAFTSOT. EASL clinical practice guidelines: Management of chronic hepatitis B virus infection. J Hepatol. 2012;57(1):167–85. doi: 10.1016/j.jhep.2012.02.010.  [DOI](https://doi.org/10.1016/j.jhep.2012.02.010) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22436845/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Hepatol&title=EASL%20clinical%20practice%20guidelines:%20Management%20of%20chronic%20hepatitis%20B%20virus%20infection&volume=57&issue=1&publication_year=2012&pages=167-85&pmid=22436845&doi=10.1016/j.jhep.2012.02.010&)

5. Smith KY, Weinberg WG, Dejesus E, Fischl MA, Liao Q, Ross LL, et al. Fosamprenavir or atazanavir once daily boosted with ritonavir 100 mg, plus tenofovir/emtricitabine, for the initial treatment of HIV infection: 48-week results of ALERT. AIDS Res Ther. 2008;5:5. doi: 10.1186/1742-6405-5-5.  [DOI](https://doi.org/10.1186/1742-6405-5-5) | [PMC free article](/articles/PMC2365957/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18373851/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=AIDS%20Res%20Ther&title=Fosamprenavir%20or%20atazanavir%20once%20daily%20boosted%20with%20ritonavir%20100%20mg,%20plus%20tenofovir/emtricitabine,%20for%20the%20initial%20treatment%20of%20HIV%20infection:%2048-week%20results%20of%20ALERT&author=KY%20Smith&author=WG%20Weinberg&author=E%20Dejesus&author=MA%20Fischl&author=Q%20Liao&volume=5&publication_year=2008&pages=5&pmid=18373851&doi=10.1186/1742-6405-5-5&)

6. Cooper RD, Wiebe N, Smith N, Keiser P, Naicker S, Tonelli M. Systematic review and meta-analysis: renal safety of tenofovir disoproxil fumarate in HIV-infected patients. Clin Infect Dis. 2010;51(5):496–505. doi: 10.1086/655681.  [DOI](https://doi.org/10.1086/655681) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20673002/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Infect%20Dis&title=Systematic%20review%20and%20meta-analysis:%20renal%20safety%20of%20tenofovir%20disoproxil%20fumarate%20in%20HIV-infected%20patients&author=RD%20Cooper&author=N%20Wiebe&author=N%20Smith&author=P%20Keiser&author=S%20Naicker&volume=51&issue=5&publication_year=2010&pages=496-505&pmid=20673002&doi=10.1086/655681&)

7. Nelson MR, Katlama C, Montaner JS, Cooper DA, Gazzard B, Clotet B, et al. The safety of tenofovir disoproxil fumarate for the treatment of HIV infection in adults: the first 4 years. AIDS. 2007;21(10):1273–81. doi: 10.1097/QAD.0b013e3280b07b33.  [DOI](https://doi.org/10.1097/QAD.0b013e3280b07b33) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17545703/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=AIDS&title=The%20safety%20of%20tenofovir%20disoproxil%20fumarate%20for%20the%20treatment%20of%20HIV%20infection%20in%20adults:%20the%20first%204%20years&author=MR%20Nelson&author=C%20Katlama&author=JS%20Montaner&author=DA%20Cooper&author=B%20Gazzard&volume=21&issue=10&publication_year=2007&pages=1273-81&pmid=17545703&doi=10.1097/QAD.0b013e3280b07b33&)

8. Verhelst D, Monge M, Meynard JL, Fouqueray B, Mougenot B, Girard PM, et al. Fanconi syndrome and renal failure induced by tenofovir: a first case report. American Journal of Kidney Diseases: The Official Journal of the National Kidney Foundation. 2002;40(6):1331. doi: 10.1053/ajkd.2002.36924.  [DOI](https://doi.org/10.1053/ajkd.2002.36924) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12460055/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=American%20Journal%20of%20Kidney%20Diseases:%20The%20Official%20Journal%20of%20the%20National%20Kidney%20Foundation&title=Fanconi%20syndrome%20and%20renal%20failure%20induced%20by%20tenofovir:%20a%20first%20case%20report&author=D%20Verhelst&author=M%20Monge&author=JL%20Meynard&author=B%20Fouqueray&author=B%20Mougenot&volume=40&issue=6&publication_year=2002&pages=1331&pmid=12460055&doi=10.1053/ajkd.2002.36924&)

9. Creput C, Gonzalez-Canali G, Hill G, Piketty C, Kazatchkine M, Nochy D. Renal lesions in HIV-1-positive patient treated with tenofovir. AIDS (London, England) 2003;17(6):935. doi: 10.1097/00002030-200304110-00026.  [DOI](https://doi.org/10.1097/00002030-200304110-00026) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12660548/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=AIDS%20(London,%20England)&title=Renal%20lesions%20in%20HIV-1-positive%20patient%20treated%20with%20tenofovir&author=C%20Creput&author=G%20Gonzalez-Canali&author=G%20Hill&author=C%20Piketty&author=M%20Kazatchkine&volume=17&issue=6&publication_year=2003&pages=935&pmid=12660548&doi=10.1097/00002030-200304110-00026&)

10. Karras A, Lafaurie M, Furco A, Bourgarit A, Droz D, Sereni D, et al. Tenofovir-related nephrotoxicity in human immunodeficiency virus-infected patients: three cases of renal failure, Fanconi syndrome, and nephrogenic diabetes insipidus. Clinical infectious diseases: an official publication of the Infectious Diseases Society of America. 2003;36(8):1070. doi: 10.1086/368314.  [DOI](https://doi.org/10.1086/368314) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12684922/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clinical%20infectious%20diseases:%20an%20official%20publication%20of%20the%20Infectious%20Diseases%20Society%20of%20America&title=Tenofovir-related%20nephrotoxicity%20in%20human%20immunodeficiency%20virus-infected%20patients:%20three%20cases%20of%20renal%20failure,%20Fanconi%20syndrome,%20and%20nephrogenic%20diabetes%20insipidus&author=A%20Karras&author=M%20Lafaurie&author=A%20Furco&author=A%20Bourgarit&author=D%20Droz&volume=36&issue=8&publication_year=2003&pages=1070&pmid=12684922&doi=10.1086/368314&)

11. Reynes J, Peyriere H, de Merle Boever C, Le Moing V, editors. 10th CROI. Feb 9–14, 2002. Renal tubular injury and severe hypophosphotemia (Fanconi’s syndrome) associated with tenofovir therapy.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=10th%20CROI&publication_year=2002&)

12. James CW, Steinhaus MC, Szabo S, Dressier RM. Tenofovir-related nephrotoxicity: case report and review of the literature. Pharmacotherapy. 2004;24(3):415–8. doi: 10.1592/phco.24.4.415.33182.  [DOI](https://doi.org/10.1592/phco.24.4.415.33182) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15040657/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacotherapy&title=Tenofovir-related%20nephrotoxicity:%20case%20report%20and%20review%20of%20the%20literature&author=CW%20James&author=MC%20Steinhaus&author=S%20Szabo&author=RM%20Dressier&volume=24&issue=3&publication_year=2004&pages=415-8&pmid=15040657&doi=10.1592/phco.24.4.415.33182&)

13. Peyriere H, Reynes J, Rouanet I, Daniel N, de Boever CM, Mauboussin JM, et al. Renal tubular dysfunction associated with tenofovir therapy: report of 7 cases. Journal of acquired immune deficiency syndromes (1999) 2004;35(3):269. doi: 10.1097/00126334-200403010-00007.  [DOI](https://doi.org/10.1097/00126334-200403010-00007) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15076241/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20of%20acquired%20immune%20deficiency%20syndromes%20(1999)&title=Renal%20tubular%20dysfunction%20associated%20with%20tenofovir%20therapy:%20report%20of%207%20cases&author=H%20Peyriere&author=J%20Reynes&author=I%20Rouanet&author=N%20Daniel&author=CM%20de%20Boever&volume=35&issue=3&publication_year=2004&pages=269&pmid=15076241&doi=10.1097/00126334-200403010-00007&)

14. Earle KE, Seneviratne T, Shaker J, Shoback D. Fanconi’s syndrome in HIV+ adults: report of three cases and literature review. Journal of bone and mineral research: the official journal of the American Society for Bone and Mineral Research. 2004;19(5):714. doi: 10.1359/jbmr.2004.19.5.714.  [DOI](https://doi.org/10.1359/jbmr.2004.19.5.714) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15068493/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20of%20bone%20and%20mineral%20research:%20the%20official%20journal%20of%20the%20American%20Society%20for%20Bone%20and%20Mineral%20Research&title=Fanconi%E2%80%99s%20syndrome%20in%20HIV+%20adults:%20report%20of%20three%20cases%20and%20literature%20review&author=KE%20Earle&author=T%20Seneviratne&author=J%20Shaker&author=D%20Shoback&volume=19&issue=5&publication_year=2004&pages=714&pmid=15068493&doi=10.1359/jbmr.2004.19.5.714&)

15. Izzedine H, Hulot JS, Villard E, Goyenvalle C, Dominguez S, Ghosn J, et al. Association between ABCC2 gene haplotypes and tenofovir-induced proximal tubulopathy. The Journal of infectious diseases. 2006;194(11):1481. doi: 10.1086/508546.  [DOI](https://doi.org/10.1086/508546) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17083032/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=The%20Journal%20of%20infectious%20diseases&title=Association%20between%20ABCC2%20gene%20haplotypes%20and%20tenofovir-induced%20proximal%20tubulopathy&author=H%20Izzedine&author=JS%20Hulot&author=E%20Villard&author=C%20Goyenvalle&author=S%20Dominguez&volume=194&issue=11&publication_year=2006&pages=1481&pmid=17083032&doi=10.1086/508546&)

16. Antoniou T, Raboud J, Chirhin S, Yoong D, Govan V, Gough K, et al. Incidence of and risk factors for tenofovir-induced nephrotoxicity: a retrospective cohort study. HIV Med. 2005;6(4):284–90. doi: 10.1111/j.1468-1293.2005.00308.x.  [DOI](https://doi.org/10.1111/j.1468-1293.2005.00308.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16011534/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=HIV%20Med&title=Incidence%20of%20and%20risk%20factors%20for%20tenofovir-induced%20nephrotoxicity:%20a%20retrospective%20cohort%20study&author=T%20Antoniou&author=J%20Raboud&author=S%20Chirhin&author=D%20Yoong&author=V%20Govan&volume=6&issue=4&publication_year=2005&pages=284-90&pmid=16011534&doi=10.1111/j.1468-1293.2005.00308.x&)

17. Rodriguez-Novoa S, Labarga P, Soriano V. Pharmacogenetics of tenofovir treatment. Pharmacogenomics. 2009;10(10):1675. doi: 10.2217/pgs.09.115.  [DOI](https://doi.org/10.2217/pgs.09.115) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19842939/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Pharmacogenetics%20of%20tenofovir%20treatment&author=S%20Rodriguez-Novoa&author=P%20Labarga&author=V%20Soriano&volume=10&issue=10&publication_year=2009&pages=1675&pmid=19842939&doi=10.2217/pgs.09.115&)

18. Rodríguez-Nóvoa S, Labarga P, Soriano V, Egan D, Albalater M, Morello J, et al. Predictors of kidney tubular dysfunction in HIV-infected patients treated with tenofovir: a pharmacogenetic study. Clin Infect Dis. 2009;48(11):e108–16. doi: 10.1086/598507.  [DOI](https://doi.org/10.1086/598507) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19400747/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Infect%20Dis&title=Predictors%20of%20kidney%20tubular%20dysfunction%20in%20HIV-infected%20patients%20treated%20with%20tenofovir:%20a%20pharmacogenetic%20study&author=S%20Rodr%C3%ADguez-N%C3%B3voa&author=P%20Labarga&author=V%20Soriano&author=D%20Egan&author=M%20Albalater&volume=48&issue=11&publication_year=2009&pages=e108-16&pmid=19400747&doi=10.1086/598507&)

19. Kiser JJ, Aquilante CL, Anderson PL, King TM, Carten ML, Fletcher CV. Clinical and genetic determinants of intracellular tenofovir diphosphate concentrations in HIV-infected patients. J Acquir Immune Defic Syndr. 2008;47(3):298–303. doi: 10.1097/qai.0b013e31815e7478.  [DOI](https://doi.org/10.1097/qai.0b013e31815e7478) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18398970/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Acquir%20Immune%20Defic%20Syndr&title=Clinical%20and%20genetic%20determinants%20of%20intracellular%20tenofovir%20diphosphate%20concentrations%20in%20HIV-infected%20patients&author=JJ%20Kiser&author=CL%20Aquilante&author=PL%20Anderson&author=TM%20King&author=ML%20Carten&volume=47&issue=3&publication_year=2008&pages=298-303&pmid=18398970&doi=10.1097/qai.0b013e31815e7478&)

20. Kiser JJ, Carten ML, Aquilante CL, Anderson PL, Wolfe P, King TM, et al. The effect of lopinavir/ritonavir on the renal clearance of tenofovir in HIV-infected patients. Clin Pharmacol Ther. 2008;83(2):265–72. doi: 10.1038/sj.clpt.6100269.  [DOI](https://doi.org/10.1038/sj.clpt.6100269) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17597712/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=The%20effect%20of%20lopinavir/ritonavir%20on%20the%20renal%20clearance%20of%20tenofovir%20in%20HIV-infected%20patients&author=JJ%20Kiser&author=ML%20Carten&author=CL%20Aquilante&author=PL%20Anderson&author=P%20Wolfe&volume=83&issue=2&publication_year=2008&pages=265-72&pmid=17597712&doi=10.1038/sj.clpt.6100269&)

21. Bleasby K, Hall LA, Perry JL, Mohrenweiser HW, Pritchard JB. Functional consequences of single nucleotide polymorphisms in the human organic anion transporter hOAT1 (SLC22A6) J Pharmacol Exp Ther. 2005;314(2):923–31. doi: 10.1124/jpet.105.084301.  [DOI](https://doi.org/10.1124/jpet.105.084301) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15914676/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharmacol%20Exp%20Ther&title=Functional%20consequences%20of%20single%20nucleotide%20polymorphisms%20in%20the%20human%20organic%20anion%20transporter%20hOAT1%20(SLC22A6)&author=K%20Bleasby&author=LA%20Hall&author=JL%20Perry&author=HW%20Mohrenweiser&author=JB%20Pritchard&volume=314&issue=2&publication_year=2005&pages=923-31&pmid=15914676&doi=10.1124/jpet.105.084301&)

22. Gupta SK, Anderson AM, Ebrahimi R, Fralich T, Graham H, Scharen-Guivel V, et al. Fanconi syndrome accompanied by renal function decline with tenofovir disoproxil fumarate: a prospective, case-control study of predictors and resolution in HIV-infected patients. PLoS One. 2014;9(3):e92717. doi: 10.1371/journal.pone.0092717.  [DOI](https://doi.org/10.1371/journal.pone.0092717) | [PMC free article](/articles/PMC3961428/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24651857/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=Fanconi%20syndrome%20accompanied%20by%20renal%20function%20decline%20with%20tenofovir%20disoproxil%20fumarate:%20a%20prospective,%20case-control%20study%20of%20predictors%20and%20resolution%20in%20HIV-infected%20patients&author=SK%20Gupta&author=AM%20Anderson&author=R%20Ebrahimi&author=T%20Fralich&author=H%20Graham&volume=9&issue=3&publication_year=2014&pages=e92717&pmid=24651857&doi=10.1371/journal.pone.0092717&)

23. Kohler JJ, Hosseini SH, Green E, Abuin A, Ludaway T, Russ R, et al. Tenofovir renal proximal tubular toxicity is regulated by OAT1 and MRP4 transporters. Lab Invest. 2011;91(6):852–8. doi: 10.1038/labinvest.2011.48.  [DOI](https://doi.org/10.1038/labinvest.2011.48) | [PMC free article](/articles/PMC3103636/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21403643/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lab%20Invest&title=Tenofovir%20renal%20proximal%20tubular%20toxicity%20is%20regulated%20by%20OAT1%20and%20MRP4%20transporters&author=JJ%20Kohler&author=SH%20Hosseini&author=E%20Green&author=A%20Abuin&author=T%20Ludaway&volume=91&issue=6&publication_year=2011&pages=852-8&pmid=21403643&doi=10.1038/labinvest.2011.48&)

24. Nagle MA, Truong DM, Dnyanmote AV, Ahn SY, Eraly SA, Wu W, et al. Analysis of three-dimensional systems for developing and mature kidneys clarifies the role of OAT1 and OAT3 in antiviral handling. J Biol Chem. 2011;286(1):243–51. doi: 10.1074/jbc.M110.139949.  [DOI](https://doi.org/10.1074/jbc.M110.139949) | [PMC free article](/articles/PMC3012981/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20921221/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=Analysis%20of%20three-dimensional%20systems%20for%20developing%20and%20mature%20kidneys%20clarifies%20the%20role%20of%20OAT1%20and%20OAT3%20in%20antiviral%20handling&author=MA%20Nagle&author=DM%20Truong&author=AV%20Dnyanmote&author=SY%20Ahn&author=SA%20Eraly&volume=286&issue=1&publication_year=2011&pages=243-51&pmid=20921221&doi=10.1074/jbc.M110.139949&)

25. Nagle MA, Wu W, Eraly SA, Nigam SK. Organic anion transport pathways in antiviral handling in choroid plexus in Oat1 (Slc22a6) and Oat3 (Slc22a8) deficient tissue. Neurosci Lett. 2013;534:133–8. doi: 10.1016/j.neulet.2012.11.027.  [DOI](https://doi.org/10.1016/j.neulet.2012.11.027) | [PMC free article](/articles/PMC3591490/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23196129/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neurosci%20Lett&title=Organic%20anion%20transport%20pathways%20in%20antiviral%20handling%20in%20choroid%20plexus%20in%20Oat1%20(Slc22a6)%20and%20Oat3%20(Slc22a8)%20deficient%20tissue&author=MA%20Nagle&author=W%20Wu&author=SA%20Eraly&author=SK%20Nigam&volume=534&publication_year=2013&pages=133-8&pmid=23196129&doi=10.1016/j.neulet.2012.11.027&)

26. Uwai Y, Ida H, Tsuji Y, Katsura T, Inui K. Renal transport of adefovir, cidofovir, and tenofovir by SLC22A family members (hOAT1, hOAT3, and hOCT2) Pharm Res. 2007;24(4):811–5. doi: 10.1007/s11095-006-9196-x.  [DOI](https://doi.org/10.1007/s11095-006-9196-x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17372702/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharm%20Res&title=Renal%20transport%20of%20adefovir,%20cidofovir,%20and%20tenofovir%20by%20SLC22A%20family%20members%20(hOAT1,%20hOAT3,%20and%20hOCT2)&author=Y%20Uwai&author=H%20Ida&author=Y%20Tsuji&author=T%20Katsura&author=K%20Inui&volume=24&issue=4&publication_year=2007&pages=811-5&pmid=17372702&doi=10.1007/s11095-006-9196-x&)

27. Nishijima T, Komatsu H, Higasa K, Takano M, Tsuchiya K, Hayashida T, et al. Single nucleotide polymorphisms in ABCC2 associate with tenofovir-induced kidney tubular dysfunction in Japanese patients with HIV-1 infection: a pharmacogenetic study. Clin Infect Dis. 2012;55(11):1558–67. doi: 10.1093/cid/cis772.  [DOI](https://doi.org/10.1093/cid/cis772) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22955427/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Infect%20Dis&title=Single%20nucleotide%20polymorphisms%20in%20ABCC2%20associate%20with%20tenofovir-induced%20kidney%20tubular%20dysfunction%20in%20Japanese%20patients%20with%20HIV-1%20infection:%20a%20pharmacogenetic%20study&author=T%20Nishijima&author=H%20Komatsu&author=K%20Higasa&author=M%20Takano&author=K%20Tsuchiya&volume=55&issue=11&publication_year=2012&pages=1558-67&pmid=22955427&doi=10.1093/cid/cis772&)

28. Elens L, Tyteca D, Panin N, Courtoy P, Lison D, Demoulin JB, et al. Functional defect caused by the 4544G>A SNP in ABCC2: potential impact for drug cellular disposition. Pharmacogenet Genomics. 2011;21(12):884–93. doi: 10.1097/FPC.0b013e32834d672b.  [DOI](https://doi.org/10.1097/FPC.0b013e32834d672b) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22027652/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Functional%20defect%20caused%20by%20the%204544G>A%20SNP%20in%20ABCC2:%20potential%20impact%20for%20drug%20cellular%20disposition&author=L%20Elens&author=D%20Tyteca&author=N%20Panin&author=P%20Courtoy&author=D%20Lison&volume=21&issue=12&publication_year=2011&pages=884-93&pmid=22027652&doi=10.1097/FPC.0b013e32834d672b&)

29. Weiss J, Theile D, Ketabi-Kiyanvash N, Lindenmaier H, Haefeli WE. Inhibition of MRP1/ABCC1, MRP2/ABCC2, and MRP3/ABCC3 by nucleoside, nucleotide, and non-nucleoside reverse transcriptase inhibitors. Drug Metab Dispos. 2007;35(3):340–4. doi: 10.1124/dmd.106.012765.  [DOI](https://doi.org/10.1124/dmd.106.012765) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17172311/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Inhibition%20of%20MRP1/ABCC1,%20MRP2/ABCC2,%20and%20MRP3/ABCC3%20by%20nucleoside,%20nucleotide,%20and%20non-nucleoside%20reverse%20transcriptase%20inhibitors&author=J%20Weiss&author=D%20Theile&author=N%20Ketabi-Kiyanvash&author=H%20Lindenmaier&author=WE%20Haefeli&volume=35&issue=3&publication_year=2007&pages=340-4&pmid=17172311&doi=10.1124/dmd.106.012765&)

30. Abla N, Chinn LW, Nakamura T, Liu L, Huang CC, Johns SJ, et al. The human multidrug resistance protein 4 (MRP4, ABCC4): functional analysis of a highly polymorphic gene. J Pharmacol Exp Ther. 2008;325(3):859–68. doi: 10.1124/jpet.108.136523.  [DOI](https://doi.org/10.1124/jpet.108.136523) | [PMC free article](/articles/PMC2612728/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18364470/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharmacol%20Exp%20Ther&title=The%20human%20multidrug%20resistance%20protein%204%20(MRP4,%20ABCC4):%20functional%20analysis%20of%20a%20highly%20polymorphic%20gene&author=N%20Abla&author=LW%20Chinn&author=T%20Nakamura&author=L%20Liu&author=CC%20Huang&volume=325&issue=3&publication_year=2008&pages=859-68&pmid=18364470&doi=10.1124/jpet.108.136523&)

31. Cihlar T, Ray AS, Laflamme G, Vela JE, Tong L, Fuller MD, et al. Molecular assessment of the potential for renal drug interactions between tenofovir and HIV protease inhibitors. Antivir Ther. 2007;12(2):267–72.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/17503669/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Antivir%20Ther&title=Molecular%20assessment%20of%20the%20potential%20for%20renal%20drug%20interactions%20between%20tenofovir%20and%20HIV%20protease%20inhibitors&author=T%20Cihlar&author=AS%20Ray&author=G%20Laflamme&author=JE%20Vela&author=L%20Tong&volume=12&issue=2&publication_year=2007&pages=267-72&pmid=17503669&)

32. Imaoka T, Kusuhara H, Adachi M, Schuetz JD, Takeuchi K, Sugiyama Y. Functional involvement of multidrug resistance-associated protein 4 (MRP4/ABCC4) in the renal elimination of the antiviral drugs adefovir and tenofovir. Mol Pharmacol. 2007;71(2):619–27. doi: 10.1124/mol.106.028233.  [DOI](https://doi.org/10.1124/mol.106.028233) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17110501/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Pharmacol&title=Functional%20involvement%20of%20multidrug%20resistance-associated%20protein%204%20(MRP4/ABCC4)%20in%20the%20renal%20elimination%20of%20the%20antiviral%20drugs%20adefovir%20and%20tenofovir&author=T%20Imaoka&author=H%20Kusuhara&author=M%20Adachi&author=JD%20Schuetz&author=K%20Takeuchi&volume=71&issue=2&publication_year=2007&pages=619-27&pmid=17110501&doi=10.1124/mol.106.028233&)

33. Ray AS, Cihlar T, Robinson KL, Tong L, Vela JE, Fuller MD, et al. Mechanism of active renal tubular efflux of tenofovir. Antimicrob Agents Chemother. 2006;50(10):3297–304. doi: 10.1128/AAC.00251-06.  [DOI](https://doi.org/10.1128/AAC.00251-06) | [PMC free article](/articles/PMC1610069/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17005808/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Antimicrob%20Agents%20Chemother&title=Mechanism%20of%20active%20renal%20tubular%20efflux%20of%20tenofovir&author=AS%20Ray&author=T%20Cihlar&author=KL%20Robinson&author=L%20Tong&author=JE%20Vela&volume=50&issue=10&publication_year=2006&pages=3297-304&pmid=17005808&doi=10.1128/AAC.00251-06&)

34. Saito S, Iida A, Sekine A, Miura Y, Ogawa C, Kawauchi S, et al. Identification of 779 genetic variations in eight genes encoding members of the ATP-binding cassette, subfamily C (ABCC/MRP/CFTR. J Hum Genet. 2002;47(4):147–71. doi: 10.1007/s100380200018.  [DOI](https://doi.org/10.1007/s100380200018) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12166651/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Hum%20Genet&title=Identification%20of%20779%20genetic%20variations%20in%20eight%20genes%20encoding%20members%20of%20the%20ATP-binding%20cassette,%20subfamily%20C%20(ABCC/MRP/CFTR&author=S%20Saito&author=A%20Iida&author=A%20Sekine&author=Y%20Miura&author=C%20Ogawa&volume=47&issue=4&publication_year=2002&pages=147-71&pmid=12166651&doi=10.1007/s100380200018&)

35. Topalis D, Alvarez K, Barral K, Munier-Lehmann H, Schneider B, Véron M, et al. Acyclic phosphonate nucleotides and human adenylate kinases: impact of a borano group on alpha-P position. Nucleosides Nucleotides Nucleic Acids. 2008;27(4):319–31. doi: 10.1080/15257770801941952.  [DOI](https://doi.org/10.1080/15257770801941952) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18404568/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleosides%20Nucleotides%20Nucleic%20Acids&title=Acyclic%20phosphonate%20nucleotides%20and%20human%20adenylate%20kinases:%20impact%20of%20a%20borano%20group%20on%20alpha-P%20position&author=D%20Topalis&author=K%20Alvarez&author=K%20Barral&author=H%20Munier-Lehmann&author=B%20Schneider&volume=27&issue=4&publication_year=2008&pages=319-31&pmid=18404568&doi=10.1080/15257770801941952&)

36. Panayiotou C, Solaroli N, Johansson M, Karlsson A. Evidence of an intact N-terminal translocation sequence of human mitochondrial adenylate kinase 4. Int J Biochem Cell Biol. 2010;42(1):62–9. doi: 10.1016/j.biocel.2009.09.007.  [DOI](https://doi.org/10.1016/j.biocel.2009.09.007) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19766732/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Biochem%20Cell%20Biol&title=Evidence%20of%20an%20intact%20N-terminal%20translocation%20sequence%20of%20human%20mitochondrial%20adenylate%20kinase%204&author=C%20Panayiotou&author=N%20Solaroli&author=M%20Johansson&author=A%20Karlsson&volume=42&issue=1&publication_year=2010&pages=62-9&pmid=19766732&doi=10.1016/j.biocel.2009.09.007&)

37. Lowe M. Structure and function of the Lowe syndrome protein OCRL1. Traffic. 2005;6(9):711–9. doi: 10.1111/j.1600-0854.2005.00311.x.  [DOI](https://doi.org/10.1111/j.1600-0854.2005.00311.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16101675/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Traffic&title=Structure%20and%20function%20of%20the%20Lowe%20syndrome%20protein%20OCRL1&author=M%20Lowe&volume=6&issue=9&publication_year=2005&pages=711-9&pmid=16101675&doi=10.1111/j.1600-0854.2005.00311.x&)

38. Wellhauser L, D’Antonio C, Bear CE. ClC transporters: discoveries and challenges in defining the mechanisms underlying function and regulation of ClC-5. Pflugers Arch. 2010;460(2):543–57. doi: 10.1007/s00424-009-0769-5.  [DOI](https://doi.org/10.1007/s00424-009-0769-5) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20049483/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pflugers%20Arch&title=ClC%20transporters:%20discoveries%20and%20challenges%20in%20defining%20the%20mechanisms%20underlying%20function%20and%20regulation%20of%20ClC-5&author=L%20Wellhauser&author=C%20D%E2%80%99Antonio&author=CE%20Bear&volume=460&issue=2&publication_year=2010&pages=543-57&pmid=20049483&doi=10.1007/s00424-009-0769-5&)

39. Purcell S, Neale B, Todd-Brown K, Thomas L, Ferreira MA, Bender D, et al. PLINK: a tool set for whole-genome association and population-based linkage analyses. Am J Hum Genet. 2007;81(3):559–75. doi: 10.1086/519795.  [DOI](https://doi.org/10.1086/519795) | [PMC free article](/articles/PMC1950838/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17701901/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=PLINK:%20a%20tool%20set%20for%20whole-genome%20association%20and%20population-based%20linkage%20analyses&author=S%20Purcell&author=B%20Neale&author=K%20Todd-Brown&author=L%20Thomas&author=MA%20Ferreira&volume=81&issue=3&publication_year=2007&pages=559-75&pmid=17701901&doi=10.1086/519795&)

40. Johnson AD, Handsaker RE, Pulit SL, Nizzari MM, O’Donnell CJ, de Bakker PI. SNAP: a web-based tool for identification and annotation of proxy SNPs using HapMap. Bioinformatics. 2008;24(24):2938–9. doi: 10.1093/bioinformatics/btn564.  [DOI](https://doi.org/10.1093/bioinformatics/btn564) | [PMC free article](/articles/PMC2720775/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18974171/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=SNAP:%20a%20web-based%20tool%20for%20identification%20and%20annotation%20of%20proxy%20SNPs%20using%20HapMap&author=AD%20Johnson&author=RE%20Handsaker&author=SL%20Pulit&author=MM%20Nizzari&author=CJ%20O%E2%80%99Donnell&volume=24&issue=24&publication_year=2008&pages=2938-9&pmid=18974171&doi=10.1093/bioinformatics/btn564&)

41. Levey AS, Bosch JP, Lewis JB, Greene T, Rogers N, Roth D. A more accurate method to estimate glomerular filtration rate from serum creatinine: a new prediction equation. Modification of Diet in Renal Disease Study Group. Ann Intern Med. 1999;130(6):461–70. doi: 10.7326/0003-4819-130-6-199903160-00002.  [DOI](https://doi.org/10.7326/0003-4819-130-6-199903160-00002) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10075613/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Intern%20Med&title=A%20more%20accurate%20method%20to%20estimate%20glomerular%20filtration%20rate%20from%20serum%20creatinine:%20a%20new%20prediction%20equation.%20Modification%20of%20Diet%20in%20Renal%20Disease%20Study%20Group&author=AS%20Levey&author=JP%20Bosch&author=JB%20Lewis&author=T%20Greene&author=N%20Rogers&volume=130&issue=6&publication_year=1999&pages=461-70&pmid=10075613&doi=10.7326/0003-4819-130-6-199903160-00002&)

42. Chelala C, Khan A, Lemoine NR. SNPnexus: a web database for functional annotation of newly discovered and public domain single nucleotide polymorphisms. Bioinformatics. 2009;25(5):655–61. doi: 10.1093/bioinformatics/btn653.  [DOI](https://doi.org/10.1093/bioinformatics/btn653) | [PMC free article](/articles/PMC2647830/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19098027/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=SNPnexus:%20a%20web%20database%20for%20functional%20annotation%20of%20newly%20discovered%20and%20public%20domain%20single%20nucleotide%20polymorphisms&author=C%20Chelala&author=A%20Khan&author=NR%20Lemoine&volume=25&issue=5&publication_year=2009&pages=655-61&pmid=19098027&doi=10.1093/bioinformatics/btn653&)

43. Dayem Ullah AZ, Lemoine NR, Chelala C. SNPnexus: a web server for functional annotation of novel and publicly known genetic variants (2012 update) Nucleic Acids Res. 2012;40(Web Server issue):W65–70. doi: 10.1093/nar/gks364.  [DOI](https://doi.org/10.1093/nar/gks364) | [PMC free article](/articles/PMC3394262/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22544707/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=SNPnexus:%20a%20web%20server%20for%20functional%20annotation%20of%20novel%20and%20publicly%20known%20genetic%20variants%20(2012%20update)&author=AZ%20Dayem%20Ullah&author=NR%20Lemoine&author=C%20Chelala&volume=40&issue=Web%20Server%20issue&publication_year=2012&pages=W65-70&pmid=22544707&doi=10.1093/nar/gks364&)

44. Dayem Ullah AZ, Lemoine NR, Chelala C. A practical guide for the functional annotation of genetic variations using SNPnexus. Brief Bioinform. 2013 doi: 10.1093/bib/bbt004.  [DOI](https://doi.org/10.1093/bib/bbt004) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23395730/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Brief%20Bioinform&title=A%20practical%20guide%20for%20the%20functional%20annotation%20of%20genetic%20variations%20using%20SNPnexus&author=AZ%20Dayem%20Ullah&author=NR%20Lemoine&author=C%20Chelala&publication_year=2013&pmid=23395730&doi=10.1093/bib/bbt004&)

45. Schlessinger A, Wittwer MB, Dahlin A, Khuri N, Bonomi M, Fan H, et al. High selectivity of the γ-aminobutyric acid transporter 2 (GAT-2, SLC6A13) revealed by structure-based approach. J Biol Chem. 2012;287(45):37745–56. doi: 10.1074/jbc.M112.388157.  [DOI](https://doi.org/10.1074/jbc.M112.388157) | [PMC free article](/articles/PMC3488050/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22932902/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=High%20selectivity%20of%20the%20%CE%B3-aminobutyric%20acid%20transporter%202%20(GAT-2,%20SLC6A13)%20revealed%20by%20structure-based%20approach&author=A%20Schlessinger&author=MB%20Wittwer&author=A%20Dahlin&author=N%20Khuri&author=M%20Bonomi&volume=287&issue=45&publication_year=2012&pages=37745-56&pmid=22932902&doi=10.1074/jbc.M112.388157&)

46. Reid G, Wielinga P, Zelcer N, De Haas M, Van Deemter L, Wijnholds J, et al. Characterization of the transport of nucleoside analog drugs by the human multidrug resistance proteins MRP4 and MRP5. Mol Pharmacol. 2003;63(5):1094–103. doi: 10.1124/mol.63.5.1094.  [DOI](https://doi.org/10.1124/mol.63.5.1094) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12695538/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Pharmacol&title=Characterization%20of%20the%20transport%20of%20nucleoside%20analog%20drugs%20by%20the%20human%20multidrug%20resistance%20proteins%20MRP4%20and%20MRP5&author=G%20Reid&author=P%20Wielinga&author=N%20Zelcer&author=M%20De%20Haas&author=L%20Van%20Deemter&volume=63&issue=5&publication_year=2003&pages=1094-103&pmid=12695538&doi=10.1124/mol.63.5.1094&)

47. Daar ES, Tierney C, Fischl MA, Sax PE, Mollan K, Budhathoki C, et al. Atazanavir plus ritonavir or efavirenz as part of a 3-drug regimen for initial treatment of HIV-1. Ann Intern Med. 2011;154(7):445–56. doi: 10.1059/0003-4819-154-7-201104050-00316.  [DOI](https://doi.org/10.1059/0003-4819-154-7-201104050-00316) | [PMC free article](/articles/PMC3430716/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21320923/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Intern%20Med&title=Atazanavir%20plus%20ritonavir%20or%20efavirenz%20as%20part%20of%20a%203-drug%20regimen%20for%20initial%20treatment%20of%20HIV-1&author=ES%20Daar&author=C%20Tierney&author=MA%20Fischl&author=PE%20Sax&author=K%20Mollan&volume=154&issue=7&publication_year=2011&pages=445-56&pmid=21320923&doi=10.1059/0003-4819-154-7-201104050-00316&)

48. Sax PE, Tierney C, Collier AC, Daar ES, Mollan K, Budhathoki C, et al. Abacavir/lamivudine versus tenofovir DF/emtricitabine as part of combination regimens for initial treatment of HIV: final results. J Infect Dis. 2011;204(8):1191–201. doi: 10.1093/infdis/jir505.  [DOI](https://doi.org/10.1093/infdis/jir505) | [PMC free article](/articles/PMC3173503/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21917892/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Infect%20Dis&title=Abacavir/lamivudine%20versus%20tenofovir%20DF/emtricitabine%20as%20part%20of%20combination%20regimens%20for%20initial%20treatment%20of%20HIV:%20final%20results&author=PE%20Sax&author=C%20Tierney&author=AC%20Collier&author=ES%20Daar&author=K%20Mollan&volume=204&issue=8&publication_year=2011&pages=1191-201&pmid=21917892&doi=10.1093/infdis/jir505&)

49. Lepist EI, Zhang X, Hao J, Huang J, Kosaka A, Birkus G, et al. Contribution of the organic anion transporter OAT2 to the renal active tubular secretion of creatinine and mechanism for serum creatinine elevations caused by cobicistat. Kidney Int. 2014;86(2):350–7. doi: 10.1038/ki.2014.66.  [DOI](https://doi.org/10.1038/ki.2014.66) | [PMC free article](/articles/PMC4120670/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24646860/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Kidney%20Int&title=Contribution%20of%20the%20organic%20anion%20transporter%20OAT2%20to%20the%20renal%20active%20tubular%20secretion%20of%20creatinine%20and%20mechanism%20for%20serum%20creatinine%20elevations%20caused%20by%20cobicistat&author=EI%20Lepist&author=X%20Zhang&author=J%20Hao&author=J%20Huang&author=A%20Kosaka&volume=86&issue=2&publication_year=2014&pages=350-7&pmid=24646860&doi=10.1038/ki.2014.66&)
