# HLA-DRB1*07:01 is associated with a higher risk of asparaginase allergies

## Metadata
**Authors:** Christian A Fernandez, Colton Smith, Wenjian Yang, Mihir Daté, Donald Bashford, Eric Larsen, W Paul Bowman, Chengcheng Liu, Laura B Ramsey, Tamara Chang, Victoria Turner, Mignon L Loh, Elizabeth A Raetz, Naomi J Winick, Stephen P Hunger, William L Carroll, Suna Onengut-Gumuscu, Wei-Min Chen, Patrick Concannon, Stephen S Rich, Paul Scheet, Sima Jeha, Ching-Hon Pui, William E Evans, Meenakshi Devidas, Mary V Relling
**Journal:** Blood
**Date:** 2014 Jun 26
**DOI:** [10.1182/blood-2014-03-563742](https://doi.org/10.1182/blood-2014-03-563742)
**PMID:** 24970932
**PMCID:** PMC4141516
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4141516/

## Abstract

HLA-DRB1*07:01 is associated with asparaginase hypersensitivity and anti-asparaginase antibodies.

HLA-DRB1 alleles that confer high-affinity binding to asparaginase epitopes lead to a higher frequency of hypersensitivity.

## Introduction

Asparaginase is used to treat leukemias and lymphomas and is one of the essential anticancer drugs for the curative treatment of acute lymphoblastic leukemia (ALL). The drug, a nonhuman enzyme, depletes asparagine in serum and inhibits the proliferation of leukemic cells; however, the precise mechanism of asparaginase action is not completely understood.^1^ Immune responses to asparaginase during treatment are common. About 30% to 75% of patients receiving native *Escherichia coli* asparaginase experience hypersensitivity reactions manifesting typically as urticaria and wheezing, and as many as 70% develop anti-asparaginase antibodies after administration.^2-5^ Furthermore, patients with lower exposure to asparaginase because of hypersensitivity reactions or the development of neutralizing antibodies, have worse outcomes compared with patients with fewer reactions that can tolerate more asparaginase doses.^6^

Two main pathways of allergic reaction have been described for proteins^7^: the classical pathway requires antigen-specific immunoglobulin E (IgE) antibodies, involves the FcεRI receptor, and mediates allergic reactions via the release of histamine. The alternative pathway requires high antigen-specific IgG antibody titers, involves the FcγRIII receptor, and mediates reactions by platelet-activating factor release. There is definite evidence supporting the role of the alternative pathway during asparaginase allergies ^8^; however, contribution by the classical pathway cannot be ruled out. Regardless of the mechanism of hypersensitivity, human leukocyte antigen (HLA) class II alleles play a critical role in these immune responses. Antibody production requires activation of CD4^+^ helper T cells, which involves the engagement of the T-cell receptor (TCR) with the HLA class II complex on antigen-presenting cells.^9^ Moreover, B cells process antigens, bind epitopes on the HLA class II protein, and present the HLA–antigen complex to primed helper T cells, resulting in cytokine secretion by T cells and differentiation of B cells into antibody-secreting plasma cells.^10^

Given the critical role HLA class II proteins play during adaptive immune responses and to the involvement of antigen-specific antibodies during hypersensitivity reactions, our primary goal was to investigate associations of HLA class II alleles with hypersensitivity reactions to asparaginase. An additional objective was to elucidate the mechanism of asparaginase-induced allergies by determining how polymorphic HLA amino acid variants alter the binding pocket of the HLA protein and affect asparaginase antigenicity.

## Materials and methods

### Patients

Five cohorts of patients were available to investigate the association of HLA class II alleles with asparaginase hypersensitivity in pediatric ALL patients (n = 3547; supplemental Figure 1). Informed consent from the parents or guardians and assent from the patients as appropriate were obtained according to institutional review board guidelines for treatment and for genomic research. Three cohorts were treated with St. Jude Children’s Research Hospital (SJCRH) protocols: Total XIIIA (n = 154, accrued patients from 1991 to 1994), Total XV (n = 498), and Total XVI (n = 271). Two cohorts were treated with Children’s Oncology Group (COG) protocols: POG 9906 (n = 234) and COG AALL0232 (n = 2390). Patients enrolled on SJCRH Total XIIIA, Total XV, and COG protocol POG 9906 received native *E coli* asparaginase (Elspar), whereas those enrolled on SJCRH Total XVI and COG AALL0232 protocols received PEGylated *E coli* asparaginase (Oncaspar) ([Table 1](#T1)). The combined cohort containing SJCRH and COG patients with imputed HLA class II alleles had a median age of 9.3 years at diagnosis, with a range from 0.71 to 30 years. The dose and frequency of asparaginase used varied by protocol and treatment arm ([Table 1](#T1)). Asparaginase hypersensitivities to Elspar or Oncaspar were graded according to the standard National Cancer Institute common toxicity criteria,^11^ with grade 2 hypersensitivity and higher considered as allergy. An enzyme-linked immunosorbent assay–based method was used to detect anti-*E coli* asparaginase IgG antibodies, as described previously.^8,12^ Patients were assigned a positive or negative status based on their titer value relative to the negative control process mean.^8^

### Table 1.

| Data set | Sample size | % Allergy* | Asparaginase formulation | Asparaginase dose | Genotyping platform | SNPs used for HLA imputation | Imputed SNPs |
| --- | --- | --- | --- | --- | --- | --- | --- |
| T1DGC | 4 871 | NA | NA | NA | Illumina Immunochip | NA† | NA |
| Total XIIIA | 91 | 30.8 | Elspar (native E coli asparaginase) | 10 000 IU/m2, IM 3 times weekly (induction and reinduction phases; 18 doses) and on weeks 4, 8, 12, 16, 20, 24, and 28 of continuation | Illumina Exome Beadchip | 1 814 | 4 753 |
| Total XV | 320 | 45.6 | Elspar (native E coli asparaginase) | 10 000-25 000 IU/m2, IM 3 times weekly or once weekly (induction, continuation/reinduction; 24-28 doses) | Affymetrix 500K | 578 | 5 989 |
| Total XVI | 130 | 18.5 | Oncaspar (PEGylated E coli asparaginase) | 2 500-3 500 IU/m2, IV weekly (induction, continuation/reinduction; 5-17 doses) | Affymetrix 6.0 | 964 | 5 603 |
| POG 9906 | 125 | 28.0 | Elspar (native E coli asparaginase) | 6 000-15 000 IU/m2, IM 3 times weekly or once weekly (induction, consolidation, interim maintenance, delayed intensification; 52 doses) | Affymetrix 500K | 578 | 5 989 |
| AALL0232 | 1 204 | 11.4 | Oncaspar (PEGylated E coli asparaginase) | 2 500 IU/m2, IM weekly (induction, consolidation, interim maintenance, delayed intensification; 6-12 doses) | Affymetrix 6.0 | 964 | 5 603 |

Table 1 Caption: Data sets used for HLA association study and for HLA class II allele imputation

### Genotyping

Germline DNA collected after remission was interrogated using the Affymetrix Human Mapping 500K Array Set, the Affymetrix Genome-Wide Human single nucleotide polymorphism (SNP) Array 6.0, or the Illumina Exome Beadchip array (n = 3146; supplemental Figure 1).^13^ Ancestry was estimated using STRUCTURE on the basis of the ancestry-informative SNPs interrogated by each genotyping platform, as described previously.^14^

### Imputation of HLA class II alleles and amino acid variants

To impute the HLA alleles of pediatric ALL patients using genome-wide SNP data, the Type 1 Diabetes Genetics Consortium (T1DGC) reference panel was used, as previously described.^15^ The T1DGC reference panel includes 4871 unrelated individuals of European ancestry, each with experimentally determined *HLA-DRB1* and *HLA-DQB1* alleles as well as genotypes interrogated by the Illumina Immunochip array. The SNP data required for HLA allele imputation included 7792 SNPs spanning hg18 chr6: 29 600 542 to 33 268 403 from the Immunochip array and single nucleotide variants from *HLA-DRB1* and *HLA-DQB1* genes inferred with the EMBL-EBI Immunogenetics HLA database ([http://www.ebi.ac.uk/ipd/imgt/hla/](http://www.ebi.ac.uk/ipd/imgt/hla/)).^16^ HLA allele imputations in ALL patients were restricted to those with ≥90% European ancestry because of the poor imputation accuracies attained in non-European individuals when imputing HLA alleles using a European reference panel.^17-20^ No SNP data within *HLA-DPB1* were available in the T1DGC reference panel. The overlap of Illumina Immunochip SNPs from the T1DGC reference panel and SNPs typed in patient samples was determined ([Table 1](#T1)), and Immunochip SNPs that were not genotyped in patients were imputed using the BEAGLE software package^21^ with the T1DGC reference panel. The T1DGC data set and BEAGLE were used to impute the *HLA-DRB1* and *HLA-DQB1* alleles in ALL patients. HLA-DRB1 amino acid sequences were inferred using the imputed *HLA-DRB1* alleles and the EMBL-EBI Immunogenetics HLA database.

### HLA typing and determination of HLA-DRB1 and HLA-DQB1 imputation concordance

HLA testing was performed by molecular methods in a subset of patients (61 SJCRH patients). All but the earliest 3 samples were tested by a minimum of 2 of 3 methods: sequence-specific primers (Life Technologies), Luminex sequence-specific oligoprobes (LabType, One Lambda), and DNA sequencing (Abbott Molecular). The 3 earliest samples included in this study were tested only by sequence-specific primers for exon 2 of the *HLA-DRB1* locus (Life Technologies, Carlsbad, CA).^22^ Extracted DNA was amplified using Taq polymerase and 95 primer pairs specific for areas of known variation in exon 2 of the *HLA-DRB1* or *HLA-DQB1* loci. Samples tested by Luminex sequence-specific oligoprobes were amplified by locus-specific biotinylated primers for exon 2 of the *HLA-DRB1* or *HLA-DQB1* loci. Oligonucleotide probes specific for known areas of variation were bound to specific beads of a given fluorescent dye mixture and analyzed in a Luminex flow cytometer. DNA sequencing was performed for *HLA-DRB1* and *HLA-DQB1* as described, with amplification performed using forward and reverse Big Dye–labeled primers for exon 2 of *HLA-DRB1* and *HLA-DQB1*. Sequencing was performed on an ABI 3130xL or 3500xL genetic analyzer and sequences were evaluated with HLA allele analysis software from Conexio Genomics.^23^ The concordance between the experimentally determined HLA alleles and the imputed HLA alleles was assessed by determining whether the methods gave identical results for each patient within the subset.

### Binding affinity

To estimate the binding affinity of unique *HLA-DRB1* alleles for various asparaginase epitopes, the Immune Epitope Database (IEDB, [www.iedb.org](http://www.iedb.org)) was used.^24^ The protein sequence of native *E coli* asparaginase (Entrez Protein database, accession number [P00805](https://ncbi.nlm.nih.gov/nuccore/P00805)) without the signal sequence was parsed into overlapping 15-mer peptide fragments and each epitope was scored for binding to HLA-DRB1 (percentile rank) using a consensus, sequence-based method.^25,26^ A lower score indicates high-affinity binding; higher scores indicate low-affinity binding. The lowest score for each allele was determined, and the 50th percentile binding score of all *HLA-DRB1* alleles was estimated. Alleles above the 50th percentile score were dichotomized into a “low-binding” category, and alleles below the 50th percentile score were dichotomized into a “high-binding” category.

### Structural modeling and simulations

Homology models of HLA-DRB1*07:01, *14:01, *15:01, and *01:01 protein structures were made using the SwissModel server^27-29^ and 3PDO from the Research Collaboratory for Structural Bioinformatics protein data bank^30^ as the template. Electrostatic and polar interactions of key residues with their neighboring residues were calculated with 6Å cutoff using Schrodinger’s PyMol molecular visualization system. The peptide selected for docking into all 4 proteins was ^212^PKVGIVYNYANASDLPAKA^231^, which was predicted by the IEDB server to contain the region of highest binding affinity for the *07:01 and *14:01 alleles. ZDOCK^31^ was used to find initial dockings for each protein, and the docking with the lowest estimated binding free energy across the different registers was chosen. The dockings were refined using Rosetta FlexPepDock^32^ to place side chains and adjust the backbone pose within the starting register. A 20-ns molecular dynamics run with explicit water was performed using Amber12 with the ff99SB force field^33,34^ after minimization and equilibration. Binding free energy was estimated from snapshots of the simulation using the Molecular Mechanics/Generalized Born Surface Area method.^35^

### Data analysis

To evaluate the association between *HLA-DRB1* and *HLA-DQB1* alleles and asparaginase hypersensitivity, patients treated with SJCRH Total protocols (n = 541, Total XIIIA, Total XV, and Total XVI) were assigned to the discovery cohort and patients treated with COG protocols (n = 1329; POG 9906 and COG AALL0232) were assigned to the validation cohort. To determine the association between polymorphic HLA-DRB1 amino acid position or predicted binding-affinity categories with asparaginase hypersensitivity, the combined cohort was used (n = 1870). A general linear model was used to analyze the association of HLA alleles, polymorphic amino acid positions, and predicted binding-affinity categories with asparaginase hypersensitivity. Gender, age (<10 or ≥10 years), treatment arm, and ALL immunophenotype were included in the model as categorical covariates. Homozygous variant *HLA-DRB1**07:01 genotypes and homozygous variant HLA-DRB1 amino acid residue groups with fewer than 5 observed patients were combined with the heterozygous variant groups where indicated. To account for multiple testing, *P* values were determined by randomly permuting HLA genotypes or amino acids 10 000 times, and *P* values were computed that reflected the probability of observing each result by chance (indicated as P_adjusted_). Differences between the binding free energy of the HLA-DRB1*07:01 protein and the free energy of HLA-DRB1*01:01, HLA-DRB1*14:01, and HLA-DRB1*15:01 were determined using a pairwise Wilcoxon rank sum test on the sets of snapshot binding energies collected from the Molecular Mechanics/Generalized Born Surface Area procedure. Permuted *P* values (P_adjusted_) < .05 were considered significant, and within our validation cohort, *P* values < .05 were considered significant. R statistical software (version 2.13.2) was used for analysis.

## Results

Class II HLA alleles were imputed in 1870 patients of European descent that had data available on asparaginase hypersensitivity ([Figure 1A](#F1); supplemental Figure 1; supplemental Table 1). Univariate analysis of potential risk factors for asparaginase hypersensitivity identified a higher risk for male patients (22% vs 17%; supplemental Table 2), age <10 years (23% vs 17%; supplemental Table 2), and for treatment with native *E coli* asparaginase rather than PEGylated asparaginase (39% vs 12%; supplemental Table 2). The incidence of asparaginase allergy varied by treatment protocol (*P* < 1.0 × 10^−6^; supplemental Table 2), ranging from 11% for COG ALL0232 to 46% for the SJCRH Total XV protocol. A multivariate analysis containing all covariates showed that gender, ALL immunophenotype, and treatment protocol were associated with asparaginase hypersensitivity (supplemental Table 2). T-lineage leukemia patients were only enrolled on SJCRH protocols Total XIIIA, Total XV, and Total XVI; within these protocols, we observed a higher risk of hypersensitivity among B-lineage leukemia patients compared with T-lineage leukemia patients (*P* = .01; supplemental Table 2). As a result, immunophenotype was included in the multivariate analysis even though it was not significant in all trials. Data on the phase of therapy in which the reaction occurred were available for 314 of the 370 cases. Within these patients, 85% of the reactions occurred during reexposure to asparaginase given in a phase after induction, suggesting that sensitization occurs during induction therapy in most cases.

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8867/4141516/0113e2b656d6/1266f1.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4141516_1266f1.jpg)

HLA-DRB1 and HLA-DQB1 alleles were imputed for asparaginase hypersensitivity studies. The HLA-DRB1 and HLA-DQB1 alleles of patients enrolled on either SJCRH or COG protocols were imputed to 4-digit resolution using SNP data from the patients and the T1DGC reference panel with both SNP data and 4-digit resolution HLA typing. (A) The discovery cohort consisting of SJCRH patients was used to identify associations between asparaginase hypersensitivity reactions and imputed HLA-DRB1 and/or HLA-DQB1 alleles. The validation cohort consisted of COG patients. (B) Using the imputed HLA-DRB1 alleles, the HLA-DRB1 amino acid sequence was inferred for every patient, and the association between polymorphic amino acids and asparaginase hypersensitivity reactions was determined. (C) Using the IEDB, imputed HLA alleles were scored for their binding affinity to E coli asparaginase peptide fragments. The imputed HLA alleles of patients were categorized into “high” or “low” binding categories for asparaginase, and the categories were tested for association with asparaginase hypersensitivity.

Among the 1870 patients with leukemia, 39 unique *HLA-DRB1* and 15 unique *HLA-DQB1* alleles were imputed ([Figure 2A-B](#F2)). The overall concordance of HLA class II imputations was 91% for *HLA-DRB1* and 97% for *HLA-DQB1* in the subset of 61 patients with experimentally determined HLA types (supplemental Tables 3 and 4). For *HLA-DRB1*, 17 of 24 alleles had an imputation accuracy of 100% (supplemental Table 3), and for *HLA-DQB1*, 9 of 12 alleles had an imputation accuracy of 100% (supplemental Table 4).

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8867/4141516/acdc2eec5e1f/1266f2.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4141516_1266f2.jpg)

HLA class II allele imputations resulted in 54 unique class II HLA alleles in patients. The imputation of HLA-DRB1 alleles within the combined data sets (n = 1870) resulted in (A) 39 unique HLA-DRB1 alleles and (B) 15 unique HLA-DQB1 alleles of varying allele frequency.

In the discovery cohort, we identified an association between *HLA-DRB1**07:01 and asparaginase hypersensitivity (*P* = .001 [P_adjusted_ = .023], odds ratio [OR] = 1.92, 95% confidence interval [CI] = 1.29-2.87; [Figure 3A-B](#F3)). The association of *HLA-DRB1**07:01 with asparaginase hypersensitivity was replicated in the validation cohort ([Figure 3C](#F3); *P* = .014, OR = 1.49, CI = 1.08-2.03), and a higher frequency of hypersensitivity was detected in patients with the *HLA-DRB1**07:01 allele across all 5 cohorts (supplemental Figure 2). Combining all SJCRH and COG patients, those carrying the risk allele had a 45% higher incidence of hypersensitivity compared with patients who did not (25.8% incidence of allergy for those carrying the risk allele vs a 17.8% incidence for those without the allele, *P* = 7.5 × 10^−5^ [P_adjusted_ = .002], OR = 1.64, CI = 1.28-2.09; [Figure 3D](#F3), supplemental Figure 3, supplemental Table 5). Furthermore, when patients were divided into the 536 patients who received native *E coli* asparaginase and the 1334 patients who received PEGylated *E coli* asparaginase ([Table 1](#T1)), *HLA-DRB1**07:01 remained the strongest HLA allele associated with hypersensitivity in each group (*P* = .007 and *P* = .005, respectively).

### Figure 3.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8867/4141516/ccef965af265/1266f3.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4141516_1266f3.jpg)

Association of HLA-DRB1 and HLA-DQB1 alleles with asparaginase hypersensitivity. Using a general linear model adjusted for gender, age, treatment arm, and ALL immunophenotype, (A) HLA-DRB1*07:01 (P = .001 [Padjusted = .023]) was associated with asparaginase hypersensitivities within the discovery cohort (n = 541). The dashed horizontal red line identifies a permuted significance threshold of P < .05 that is adjusted for multiple testing. Patients with the HLA-DRB1*07:01 allele had a higher incidence of hypersensitivity compared with patients that did not have the allele within the (B) discovery cohort (n = 541), (C) validation cohort (n = 1329), and (D) the combined cohort (n = 1870).

To determine the protein sequence features of *HLA-DRB1**07:01 that confer a greater risk of asparaginase hypersensitivity ([Figure 1B](#F1)), polymorphic amino acid variants of the HLA-DRB1 protein were inferred. A total of 30 polymorphic amino acid positions were identified within our patient population, with 16 positions having 3 or more amino acid residues possible (90 total residues within the 30 polymorphic amino acid positions).

There were 10 risk amino acid positions that were associated with asparaginase hypersensitivity: a glycine at position 73 had the strongest association with asparaginase hypersensitivity (*P* = 6.45 × 10^−6^ [P_adjusted_ < 1.00 × 10^−4^], OR = 1.76, CI = 1.38-2.26; [Figure 4A-B](#F4), [Table 2](#T2)), 6 positions (11, 13, 14, 25, 30, and 74) were uniquely coded by *HLA-DRB1**07:01 (*P* = 1.34 × 10^−4^ [P_adjusted_ = .003], OR = 1.69, CI = 1.29-2.21; [Figure 4A](#F4), [Table 2](#T2)), and 3 positions (57, 60, and 78) were associated with hypersensitivity to a lesser extent (*P* < 2.2 × 10^−3^ [P_adjusted_ < .05]; [Figure 4A](#F4), [Table 2](#T2)). Examination of the published crystal structure (PDB code 3PDO) of HLA-DR1^30^ shows that positions 11, 13, 14, 25, and 30 of the HLA-DRB1 chain are all located within the β-sheet floor of the protein and positions 57, 60, 73, 74, and 78 are on the adjacent α helix in close proximity to positions 11 and 13 ([Figure 5](#F5)). The side chains of floor residues 11, 13, and 30 point toward the binding pocket, whereas those of residues 14 and 25 point toward the β-sheet floor. The amino acid residues at these positions associated with a higher risk of hypersensitivity ([Table 2](#T2)) were all present in the *HLA-DRB1**07:01 allele. These results suggest that *HLA-DRB1**07:01 may predispose to a higher incidence of allergies because structural features of the binding pocket may affect the interaction of asparaginase epitopes with the HLA-DRB1 protein. Two other *HLA-DRB1* alleles within our data set contain a glycine at position 73 ([Table 2](#T2), *HLA-DRB1**03:01 and *HLA-DRB1**03:02); both had an OR > 1.0, although neither allele reached statistical significance for association with allergy (supplemental Table 5). After adjusting for position 73 in the general linear model, only a tyrosine at position 37 was associated with hypersensitivity reactions (*P* = .002 [P_adjusted_ = .036], OR = 2.02, CI = 1.56-2.63). This risk residue in position 37 was coded by 17 different *HLA-DRB1* alleles in our data set, and the position was not associated with hypersensitivity without adjusting for the strongest association.

### Figure 4.

![Figure 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8867/4141516/e6e1e9b8efac/1266f4.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4141516_1266f4.jpg)

Polymorphic amino acid positions of HLA-DRB1 are associated with asparaginase hypersensitivity. (A) Amino acid position 73 had the strongest association with asparaginase hypersensitivity, and positions 11, 13, 14, 25, 30, 57, 60, 74, and 78 were associated with hypersensitivity to a lesser extent. The y-axis shows the permuted P value (Padjusted) of each association tested and the x-axis represents the amino acid positions of the HLA-DRB1 protein, excluding the leader signal sequence. The dashed horizontal red line identifies a permuted significance threshold of P < .05 that is adjusted for multiple testing. (B) Patients with a glycine (Gly) residue at HLA-DRB1 amino acid position 73 had a higher incidence of hypersensitivity compared with patients with an alanine (Ala/Ala) (n = 1,870, P = 6.45 × 10−6 [Padjusted < 1.00 × 10−4]).

### Table 2.

| AA position (no leader)* | Risk residue† | P value (Padjusted)‡ | OR‡ | 95% CI | Other possible amino acids¶ | HLA-DRB1 alleles with risk residue |
| --- | --- | --- | --- | --- | --- | --- |
| 11 | G | 1.34 × 10−4 (.003) | 1.69 | 1.29-2.21 | D, G, L, P, S, V | *07:01 |
| 13 | Y | 1.34 × 10−4 (.003) | 1.69 | 1.29-2.21 | F, G, H, R, S, Y | *07:01 |
| 14 | K | 1.34 × 10−4 (.003) | 1.69 | 1.29-2.21 | E, K | *07:01 |
| 25 | Q | 1.34 × 10−4 (.003) | 1.69 | 1.29-2.21 | Q,R | *07:01 |
| 30 | L | 1.34 × 10−4 (.003) | 1.69 | 1.29-2.21 | C, G, H, L, R, Y | *07:01 |
| 57 | V | 1.19 × 10−3 (.023) | 1.53 | 1.18-1.98 | A, D, S | *07:01, *09:01, *12:01 |
| 60 | S | 1.19 × 10−3 (.023) | 1.53 | 1.18-1.98 | H, Y | *07:01, *09:01, *12:01 |
| 73 | G | 6.45 × 10−6 (<1.00 × 10−4) | 1.76 | 1.38-2.26 | A, G | *07:01,*03:01, *03:02 |
| 74 | Q | 1.34 × 10−4 (.003) | 1.69 | 1.29-2.21 | A, E, L, Q, R | *07:01 |
| 78 | V | 2.18 × 10−3 (.042) | 1.51 | 1.16-1.97 | V, Y | *07:01, *09:01 |

Table 2 Caption: HLA-DRB1 polymorphic amino acid positions associated with asparaginase hypersensitivity

### Figure 5.

![Figure 5](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8867/4141516/64214b49e682/1266f5.jpg)

The 3-dimensional structure of HLA-DR. The 3-dimensional ribbon model of HLA-DR is based on Protein Data Bank entry 3PDO with the DR α chain shown in violet and the DR β chain in gray. Amino acid positions identified within the HLA-DRB1 protein by the association analysis are shown for positions 11 (black), 13 (blue), 14 (cyan), 25 (orange), 30 (brown), 57 (yellow), 60 (pink), 73 (red), 74 (green), and 78 (purple).

The binding affinities of *HLA-DRB1* alleles for asparaginase epitopes were estimated using the IEDB, which uses a consensus sequence-based method to estimate binding affinities^25,26^ ([Figure 1C](#F1); supplemental Figure 4). Patients with 1 or 2 high-binding alleles had a higher incidence of asparaginase hypersensitivities compared with patients with 2 low-binding alleles (*P* = 3.3 × 10^−4^, OR = 1.38, CI = 1.16-1.64; [Figure 6A](#F6)). *HLA-DRB1**07:01 was a predicted high-binding allele (supplemental Table 5), consistent with its association with asparaginase hypersensitivity reactions, and all *HLA-DRB1* alleles with significant (*P* < .1) association with allergies (supplemental Table 5) had predicted binding affinities consistent with their estimated OR ([Figure 6B](#F6)), including 3 alleles coding for a tyrosine at position 37 (*HLA-DRB1**04:05, *HLA-DRB1**04:08, and *HLA-DRB1**04:02).

### Figure 6.

![Figure 6](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8867/4141516/3eb697f782a1/1266f6.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4141516_1266f6.jpg)

High-affinity binding of HLA-DRB1 alleles was associated with asparaginase hypersensitivity reactions. (A) Patients with high-affinity binding alleles had a higher incidence of asparaginase hypersensitivity compared with patients with low-affinity binding alleles (n = 1870, P = 3.3 × 10−4). (B) Estimated ORs for association of HLA-DRB1*01:01, HLA-DRB1*04:02, HLA-DRB1*04:05, HLA-DRB1*04:08, HLA-DRB1*07:01, HLA-DRB1*14:01, and HLA-DRB1*15:01 alleles with asparaginase hypersensitivity, all at P < .1, are shown vs the predicted binding category (predicted low-binding alleles are shown in blue; predicted high-binding alleles are shown in red). All HLA-DRB1 alleles had predicted binding affinities consistent with the estimated odds ratios.

Homology models were made of the high-affinity HLA-DRB1*07:01 protein and of 3 proteins predicted to have low-binding affinity (HLA-DRB1*14:01, *15:01, and *01:01). All predicted low-binding alleles had OR < 1 for their association with allergy, and they showed the strongest association to protect against asparaginase hypersensitivity (supplemental Table 5; [Figure 6B](#F6)). The amino acid residues at positions 11, 13, 14, 25, 30, 57, 60, 73, 74, and 78 for these 3 low-affinity binding alleles differ from the corresponding amino acid sequence of *HLA-DRB1**07:01 (supplemental Table 6). Structural comparison of these 4 alleles shows that several salt bridges and/or polar contacts exist between amino acids in risk positions and neighboring residues for the 3 low-affinity binding alleles, but not for the HLA-DRB1*07:01 structure (supplemental Table 6). This, together with the occurrence of glycine at positions 11 and 73 in the HLA-DRB1*07:01 protein, suggests that the HLA-DRB1*07:01 protein may have more flexibility in the peptide-binding groove compared with the HLA-DRB1*14:01, *15:01, and *01:01 proteins, which may result in higher affinity binding between asparaginase epitopes and the HLA-DR structure.

To complement the results generated using the IEDB, we used a molecular dynamics–based method to provide a structure-based estimate of the binding free energy between the HLA protein and the putatively most immunogenic peptide region of asparaginase. For each of the 4 HLA alleles for which homology models were built, we first applied a docking procedure that aligned the peptide in several different registers within the binding groove of the HLA model. The molecular dynamics procedure was then applied for each register of each allele, and the register with the lowest binding free energy for each allele was compared. The *HLA-DRB1**07:01 allele was estimated to bind the asparaginase peptide with significantly lower binding free energy (higher affinity) than the other 3 alleles (supplemental Figure 5, *P* < 1.0 × 10^−6^).

Anti-asparaginase IgG antibody data were available in 502 SJCRH patients (89 treated in Total XIIIA, 283 in Total XV, and 130 in Total XVI). A higher incidence of anti-asparaginase IgG antibody positive patients was detected in carriers of the *HLA-DRB1**07:01 allele ([Figure 7A](#F7), *P* = 1.4 × 10^−5^, OR = 2.92, CI = 1.82-4.80), in patients with a glycine risk residue at amino acid position 73 ([Figure 7B](#F7), *P* = 5.2 × 10^−5^, OR = 2.39, CI = 1.57-3.67), or in patients with 2 high-affinity binding alleles ([Figure 7C](#F7), *P* = 1.4 × 10^−3^, OR = 1.59, CI = 1.20-2.12) compared with patients with one or 2 low-affinity binding alleles, consistent with the allergy data. The proportion of *HLA-DRB1**07:01 allele carriers among allergy cases that tested positive for anti-asparaginase IgG antibodies (n = 141) vs those who were negative (n = 81) was 35% vs 28% (*P* = .37), although power to detect any difference was low because of the low sample size.

### Figure 7.

![Figure 7](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8867/4141516/a54dd8c1894d/1266f7.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4141516_1266f7.jpg)

Asparaginase antibodies were associated with HLA-DRB1*07:01 genotype, amino acid composition, and binding affinity. Anti-asparaginase IgG antibody status was available for 502 ALL patients. A higher incidence of asparaginase antibodies was detected for patients with (A) HLA-DRB1*07:01 (P = 1.4 × 10−5), (B) a glycine amino acid within HLA-DRB1 amino acid position 73 (P = 5.2 × 10−5), or (C) HLA-DRB1 alleles with predicted high-affinity binding for asparaginase (P = 1.4 × 10−3). Ala, alanine; Gly, glycine; het, heterozygous; homo, homozygous.

## Discussion

Patients with low asparaginase exposure during treatment of malignancy have been reported to have increased risk of relapse.^6^ Patients who develop hypersensitivity reactions or neutralizing antibodies to asparaginase have been shown to have increased asparaginase clearance leading to subtherapeutic serum concentrations of the drug.^2,36-44^ The frequency of asparaginase hypersensitivity has been associated with the pharmaceutical preparation of asparaginase,^6,45^ tumor immunophenotype,^8,46^ the treatment schedule of asparaginase,^47^ and racial ancestry^48^; importantly, we controlled for these covariates in our analyses of whether HLA genotypes were related to allergy in these prospective clinical trials. The influence of HLA genes on asparaginase hypersensitivity has not been studied previously. Herein, we identified that there is an association of the *HLA-DRB1**07:01 allele with asparaginase hypersensitivity and anti-asparaginase antibodies ([Figure 3A-D](#F3); [Figure 7A](#F7)). Nevertheless, a limitation of our study could be heterogeneity in reporting clinical allergy to asparaginase. Although it is likely that the most important cases of first instance of clinical allergy to asparaginase are captured by allergies of grade 2 and higher because of severity of those reactions (eg, rash, flushing, urticaria, dyspnea, drug fever ≥38°C), and these cases often precipitate therapeutic intervention for future doses, it is also possible that clinicians may have used antihistamines or glucocorticoids that attenuated or minimized reactions to grade 1 or lower, and thus some reactions may have been missed in our study.

Variants in HLA genes on chromosome 6 is linked with more autoimmune diseases than any other genomic region.^49,50^ Genetic variants within HLA genes have been associated with several diseases including celiac disease,^51^ rheumatoid arthritis,^52,53^ type I diabetes,^54,55^ and multiple sclerosis.^56^ Several *HLA-B* alleles have been associated with small molecule drug allergy, including abacavir,^57,58^ and adverse reactions to carbamazepine^59^ and allopurinol.^60-62^ HLA allele associations with immune responses to protein-based pharmaceuticals have also been reported for interferon-β (*HLA-DRB1**04:01 and *04:08)^63,64^ and for recombinant human erythropoietin (*HLA-DRB1**09).^65,66^ Furthermore, *HLA-DRB1**07:01 has previously been linked to elevated levels of serum alanine aminotransferase during ximelagatran and lapatinib treatment through a drug-induced adaptive immune response.^9,22^

Although direct experimental HLA typing is costly and laborious, imputation of HLA types using SNP genotyping arrays has been shown to provide a reasonably accurate and cost-effective way of investigating drug hypersensitivity or disease association with classical HLA alleles.^15,52,67,68^ The *HLA-DRB1* and *HLA-DQB1* allele imputation accuracies for individuals of European descent have ranged from 72% to 98%.^17,18,20,52^ We achieved similar imputation accuracies of 91% and 97% for *HLA-DRB1* and *HLA-DQB1*, respectively; furthermore, the imputation concordance for *HLA-DRB1**07:01 was 100% (supplemental Table 3). We took advantage of imputation and found that the *HLA-DRB1**07:01 allele was associated with allergy to asparaginase.

Inferring polymorphic HLA amino acid positions from HLA alleles^69^ allows for interrogation of specific residues within the HLA binding pocket for association with hypersensitivity. We identified HLA-DRB1 amino acid positions 11, 13, 14, 25, 30, 37, 57, 60, 73, 74, and 78 associated with asparaginase hypersensitivity. Interestingly, several of these have been associated with autoimmune disease: position 11 with sarcoidosis,^70^ positions 13 and 60 with multiple sclerosis,^71,72^ position 37 with psoriasis vulgaris,^73^ position 57 with Vogt-Koyanagi-Harada disease,^74^ position 78 with primary biliary cirrhosis,^75^ and position 74 with Graves’ disease^76,77^ and autoimmune polyglandular syndrome.^78^ Moreover, association of HLA-DRB1 amino acid positions 11, 13, and 74 with rheumatoid arthritis susceptibility was recently reported,^52^ suggesting that the HLA-DRB1 amino acid positions that predispose to asparaginase allergy may also be important for other immune responses.

Binding of epitopes to the HLA class II protein is required for antigen presentation to CD4^+^ T cells and to mediate an immune response.^79^ Epitope binding prediction tools have been used for vaccine development strategies,^80^ and for designing nonimmunogenic protein-based pharmaceuticals, such as factor VIII^81^ and asparaginase.^82^ Interestingly, the most immunogenic asparaginase region identified in our study using IEDB (^212^PKVGIVYNYANASDLPAKA^231^) was shown by Cantor et al^82^ to be the most immunogenic fragment of asparaginase using a T-cell activation assay and transgenic *HLA-DRB1**04:01 mice. Whether less immunogenic formulations of asparaginase could be developed for specific *HLA-DRB1* alleles that maintain undiminished asparaginase activity is unknown. No other studies have used human HLA class II loci to identify immunodominant epitopes of native *E coli* asparaginase. Using the IEDB, we found that patients harboring the *HLA-DRB1* alleles with high-affinity binding to asparaginase epitopes had a higher incidence of asparaginase hypersensitivity compared with those with only low-affinity binding alleles ([Figure 6A](#F6)). Furthermore, *HLA-DRB1**07:01 was predicted to have high-affinity binding for asparaginase epitopes ([Figure 6B](#F6)). Comparing HLA-DR structures with low-affinity binding alleles to the HLA-DR structure with the *HLA-DRB1**07:01 allele (supplemental Table 6) suggests that inherited variation in *HLA-DRB1* results in variation in amino acids within the binding pocket of the protein that can influence the interaction of epitopes with the HLA-DR protein and affect the immunogenicity of asparaginase. This is the first known report to use IEDB to computationally predict the immunogenicity of a drug based on patient *HLA-DRB1* genotypes; such an approach might be useful for other protein-based drugs in the future.

In conclusion, we identified an association between the *HLA-DRB1**07:01 allele and hypersensitivity reactions to asparaginase. Taken together with its association with anti-asparaginase antibodies, the polymorphic amino acids encoded, and structural analysis of the HLA-DR proteins, our results suggest a structure–activity relationship whereby *HLA-DRB1* alleles that encode for polymorphic amino acid variants within the HLA-DRB1 protein that confer high-affinity binding to asparaginase epitopes can lead to a higher frequency of asparaginase hypersensitivity compared with HLA alleles that confer low-affinity binding. Using the IEDB and the protein sequence of the biologic drug, it is possible to a priori identify patients that are predisposed to developing an immune response. Strategies that can identify patients that are predisposed to developing allergic reactions to protein therapeutics may help lower frequency of immunogenicity during clinical trials by excluding high-risk patients from the study, selectively directing the use of premedications, or perhaps even by developing “designer therapeutics” as alternative forms of the protein to permit the use of HLA-specific preparations in predisposed individuals. Our results emphasize the importance of variation at the HLA locus for predicting the immunogenicity of protein therapeutics.

## Acknowledgments

The study was supported by the National Cancer Institute (grants GM 92666, CA 21765, CA 142665, CA 36401, CA98543 [COG Chair's grant], CA98413 [COG Statistical Center], and CA114766 [COG Specimen Banking]) and the American Lebanese Syrian Associated Charities.

S.P.H. is the Ergen Family Chair in Pediatric Cancer.

## Footnotes

## Authorship

Contribution: C.A.F. and M.V.R. designed the project, analyzed the data, interpreted the data, and drafted the manuscript; C.A.F., C.S., and W.Y. performed statistical analysis and HLA imputations; M.D. and D.B. performed the structural analysis; V.T. performed the HLA typing; S.O.G., W.M.C., P.C., and S.S.R. provided the T1DGC data; E.L., W.P.B., M.L.L., E.A.R., N.J.W., S.P.H., W.L.C., S.J., C.H.P., W.E.E., and M.D. were investigators for the clinical protocols; C.L., L.B.R., T.C., and P.S. interpreted the data; and all authors contributed to the writing of the manuscript.

Conflict-of-interest disclosure: The authors declare no competing financial interests.

Correspondence: Mary V. Relling, St. Jude Children’s Research Hospital, 262 Danny Thomas Place, Memphis, TN 38105; e-mail: mary.relling@stjude.org.

## References

1. Asselin BL, Ryan D, Frantz CN, et al. In vitro and in vivo killing of acute lymphoblastic leukemia cells by L-asparaginase. Cancer Res. 1989;49(15):4363–4368.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/2743326/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=In%20vitro%20and%20in%20vivo%20killing%20of%20acute%20lymphoblastic%20leukemia%20cells%20by%20L-asparaginase.&author=BL%20Asselin&author=D%20Ryan&author=CN%20Frantz&volume=49&issue=15&publication_year=1989&pages=4363-4368&pmid=2743326&)

2. Asselin BL. The three asparaginases. Comparative pharmacology and optimal use in childhood leukemia. Adv Exp Med Biol. 1999;457:621–629.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/10500842/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Adv%20Exp%20Med%20Biol&title=The%20three%20asparaginases.%20Comparative%20pharmacology%20and%20optimal%20use%20in%20childhood%20leukemia.&author=BL%20Asselin&volume=457&publication_year=1999&pages=621-629&pmid=10500842&)

3. Oettgen HF, Stephenson PA, Schwartz MK, et al. Toxicity of E. coli L-asparaginase in man. Cancer. 1970;25(2):253–278. doi: 10.1002/1097-0142(197002)25:2<253::aid-cncr2820250204>3.0.co;2-u.  [DOI](https://doi.org/10.1002/1097-0142(197002)25:2<253::aid-cncr2820250204>3.0.co;2-u) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/4905153/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer&title=Toxicity%20of%20E.%20coli%20L-asparaginase%20in%20man.&author=HF%20Oettgen&author=PA%20Stephenson&author=MK%20Schwartz&volume=25&issue=2&publication_year=1970&pages=253-278&pmid=4905153&doi=10.1002/1097-0142(197002)25:2<253::aid-cncr2820250204>3.0.co;2-u&)

4. Appel IM, Kazemier KM, Boos J, et al. Pharmacokinetic, pharmacodynamic and intracellular effects of PEG-asparaginase in newly diagnosed childhood acute lymphoblastic leukemia: results from a single agent window study. Leukemia. 2008;22(9):1665–1679. doi: 10.1038/leu.2008.165.  [DOI](https://doi.org/10.1038/leu.2008.165) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18580955/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Leukemia&title=Pharmacokinetic,%20pharmacodynamic%20and%20intracellular%20effects%20of%20PEG-asparaginase%20in%20newly%20diagnosed%20childhood%20acute%20lymphoblastic%20leukemia:%20results%20from%20a%20single%20agent%20window%20study.&author=IM%20Appel&author=KM%20Kazemier&author=J%20Boos&volume=22&issue=9&publication_year=2008&pages=1665-1679&pmid=18580955&doi=10.1038/leu.2008.165&)

5. Bowman WP, Larsen EL, Devidas M, et al. Augmented therapy improves outcome for pediatric high risk acute lymphocytic leukemia: results of Children’s Oncology Group trial P9906. Pediatr Blood Cancer. 2011;57(4):569–577. doi: 10.1002/pbc.22944.  [DOI](https://doi.org/10.1002/pbc.22944) | [PMC free article](/articles/PMC3136564/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21360654/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pediatr%20Blood%20Cancer&title=Augmented%20therapy%20improves%20outcome%20for%20pediatric%20high%20risk%20acute%20lymphocytic%20leukemia:%20results%20of%20Children%E2%80%99s%20Oncology%20Group%20trial%20P9906.&author=WP%20Bowman&author=EL%20Larsen&author=M%20Devidas&volume=57&issue=4&publication_year=2011&pages=569-577&pmid=21360654&doi=10.1002/pbc.22944&)

6. Silverman LB, Gelber RD, Dalton VK, et al. Improved outcome for children with acute lymphoblastic leukemia: results of Dana-Farber Consortium Protocol 91-01. Blood. 2001;97(5):1211–1218. doi: 10.1182/blood.v97.5.1211.  [DOI](https://doi.org/10.1182/blood.v97.5.1211) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11222362/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Improved%20outcome%20for%20children%20with%20acute%20lymphoblastic%20leukemia:%20results%20of%20Dana-Farber%20Consortium%20Protocol%2091-01.&author=LB%20Silverman&author=RD%20Gelber&author=VK%20Dalton&volume=97&issue=5&publication_year=2001&pages=1211-1218&pmid=11222362&doi=10.1182/blood.v97.5.1211&)

7. Finkelman FD. Anaphylaxis: lessons from mouse models. J Allergy Clin Immunol. 2007;120(3):506-517.  [DOI](https://doi.org/10.1016/j.jaci.2007.07.033) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17765751/)

8. Liu C, Kawedia JD, Cheng C, et al. Clinical utility and implications of asparaginase antibodies in acute lymphoblastic leukemia. Leukemia. 2012;26(11):2303–2309. doi: 10.1038/leu.2012.102.  [DOI](https://doi.org/10.1038/leu.2012.102) | [PMC free article](/articles/PMC3516853/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22484422/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Leukemia&title=Clinical%20utility%20and%20implications%20of%20asparaginase%20antibodies%20in%20acute%20lymphoblastic%20leukemia.&author=C%20Liu&author=JD%20Kawedia&author=C%20Cheng&volume=26&issue=11&publication_year=2012&pages=2303-2309&pmid=22484422&doi=10.1038/leu.2012.102&)

9. Kindmark A, Jawaid A, Harbron CG, et al. Genome-wide pharmacogenetic investigation of a hepatic adverse event without clinical signs of immunopathology suggests an underlying immune pathogenesis. Pharmacogenomics J. 2008;8(3):186–195. doi: 10.1038/sj.tpj.6500458.  [DOI](https://doi.org/10.1038/sj.tpj.6500458) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17505501/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=Genome-wide%20pharmacogenetic%20investigation%20of%20a%20hepatic%20adverse%20event%20without%20clinical%20signs%20of%20immunopathology%20suggests%20an%20underlying%20immune%20pathogenesis.&author=A%20Kindmark&author=A%20Jawaid&author=CG%20Harbron&volume=8&issue=3&publication_year=2008&pages=186-195&pmid=17505501&doi=10.1038/sj.tpj.6500458&)

10. Reveille JD. The genetic basis of autoantibody production. Autoimmun Rev. 2006;5(6):389–398. doi: 10.1016/j.autrev.2005.10.012.  [DOI](https://doi.org/10.1016/j.autrev.2005.10.012) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16890892/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Autoimmun%20Rev&title=The%20genetic%20basis%20of%20autoantibody%20production.&author=JD%20Reveille&volume=5&issue=6&publication_year=2006&pages=389-398&pmid=16890892&doi=10.1016/j.autrev.2005.10.012&)

11. National Cancer Institute. Cancer Therapy Evaluation Program. Common Toxicity Criteria, Version 2.0. June 1, 1999. http://ctep.cancer.gov/reporting/ctc.html; 1999.  [http://ctep.cancer.gov/reporting/ctc.html](http://ctep.cancer.gov/reporting/ctc.html)

12. Wang B, Hak LJ, Relling MV, Pui CH, Woo MH, Storm MC. ELISA to evaluate plasma anti-asparaginase IgG concentrations in patients with acute lymphoblastic leukemia. J Immunol Methods. 2000;239(1-2):75–83. doi: 10.1016/s0022-1759(00)00182-4.  [DOI](https://doi.org/10.1016/s0022-1759(00)00182-4) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10821949/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Immunol%20Methods&title=ELISA%20to%20evaluate%20plasma%20anti-asparaginase%20IgG%20concentrations%20in%20patients%20with%20acute%20lymphoblastic%20leukemia.&author=B%20Wang&author=LJ%20Hak&author=MV%20Relling&author=CH%20Pui&author=MH%20Woo&volume=239&issue=1-2&publication_year=2000&pages=75-83&pmid=10821949&doi=10.1016/s0022-1759(00)00182-4&)

13. Treviño LR, Yang W, French D, et al. Germline genomic variants associated with childhood acute lymphoblastic leukemia. Nat Genet. 2009;41(9):1001–1005. doi: 10.1038/ng.432.  [DOI](https://doi.org/10.1038/ng.432) | [PMC free article](/articles/PMC2762391/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19684603/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Germline%20genomic%20variants%20associated%20with%20childhood%20acute%20lymphoblastic%20leukemia.&author=LR%20Trevi%C3%B1o&author=W%20Yang&author=D%20French&volume=41&issue=9&publication_year=2009&pages=1001-1005&pmid=19684603&doi=10.1038/ng.432&)

14. Yang JJ, Cheng C, Devidas M, et al. Ancestry and pharmacogenomics of relapse in acute lymphoblastic leukemia. Nat Genet. 2011;43(3):237–241. doi: 10.1038/ng.763.  [DOI](https://doi.org/10.1038/ng.763) | [PMC free article](/articles/PMC3104508/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21297632/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Ancestry%20and%20pharmacogenomics%20of%20relapse%20in%20acute%20lymphoblastic%20leukemia.&author=JJ%20Yang&author=C%20Cheng&author=M%20Devidas&volume=43&issue=3&publication_year=2011&pages=237-241&pmid=21297632&doi=10.1038/ng.763&)

15. Pereyra F, Jia X, McLaren PJ, et al. International HIV Controllers Study. The major genetic determinants of HIV-1 control affect HLA class I peptide presentation. Science. 2010;330(6010):1551–1557. doi: 10.1126/science.1195271.  [DOI](https://doi.org/10.1126/science.1195271) | [PMC free article](/articles/PMC3235490/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21051598/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=The%20major%20genetic%20determinants%20of%20HIV-1%20control%20affect%20HLA%20class%20I%20peptide%20presentation.&author=F%20Pereyra&author=X%20Jia&author=PJ%20McLaren&volume=330&issue=6010&publication_year=2010&pages=1551-1557&pmid=21051598&doi=10.1126/science.1195271&)

16. Robinson J, Waller MJ, Fail SC, et al. The IMGT/HLA database. Nucleic Acids Res. 2009;37(Database issue):D1013–D1017. doi: 10.1093/nar/gkn662.  [DOI](https://doi.org/10.1093/nar/gkn662) | [PMC free article](/articles/PMC2686596/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18838392/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=The%20IMGT/HLA%20database.&author=J%20Robinson&author=MJ%20Waller&author=SC%20Fail&volume=37&issue=Database%20issue&publication_year=2009&pages=D1013-D1017&pmid=18838392&doi=10.1093/nar/gkn662&)

17. Zhang XC, Li SS, Wang H, Hansen JA, Zhao LP. Empirical evaluations of analytical issues arising from predicting HLA alleles using multiple SNPs. BMC Genet. 2011;12:39. doi: 10.1186/1471-2156-12-39.  [DOI](https://doi.org/10.1186/1471-2156-12-39) | [PMC free article](/articles/PMC3111398/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21518453/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Genet&title=Empirical%20evaluations%20of%20analytical%20issues%20arising%20from%20predicting%20HLA%20alleles%20using%20multiple%20SNPs.&author=XC%20Zhang&author=SS%20Li&author=H%20Wang&author=JA%20Hansen&author=LP%20Zhao&volume=12&publication_year=2011&pages=39&pmid=21518453&doi=10.1186/1471-2156-12-39&)

18. Dilthey A, Leslie S, Moutsianas L, et al. Multi-population classical HLA type imputation. PLOS Comput Biol. 2013;9(2):e1002877. doi: 10.1371/journal.pcbi.1002877.  [DOI](https://doi.org/10.1371/journal.pcbi.1002877) | [PMC free article](/articles/PMC3572961/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23459081/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLOS%20Comput%20Biol&title=Multi-population%20classical%20HLA%20type%20imputation.&author=A%20Dilthey&author=S%20Leslie&author=L%20Moutsianas&volume=9&issue=2&publication_year=2013&pages=e1002877&pmid=23459081&doi=10.1371/journal.pcbi.1002877&)

19. Tekola Ayele F, Hailu E, Finan C, et al. doi: 10.1371/journal.pone.0040206. Prediction of HLA class II alleles using SNPs in an African population [published correction appears in PLos ONE. 2012;7(7) doi/10.1371/annotation/3529a6a2-4ba2-47dc-929e-399d441b0afa]. PLoS ONE. 2012;7(6):e40206.  [DOI](https://doi.org/10.1371/journal.pone.0040206) | [PMC free article](/articles/PMC3386230/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22761960/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Tekola%20Ayele%20F,%20Hailu%20E,%20Finan%20C,%20et%20al.%20doi:%2010.1371/journal.pone.0040206.%20Prediction%20of%20HLA%20class%20II%20alleles%20using%20SNPs%20in%20an%20African%20population%20%5Bpublished%20correction%20appears%20in%20PLos%20ONE.%202012;7(7)%20doi/10.1371/annotation/3529a6a2-4ba2-47dc-929e-399d441b0afa%5D.%20PLoS%20ONE.%202012;7(6):e40206.)

20. Jia X, Han B, Onengut-Gumuscu S, et al. Imputing amino acid polymorphisms in human leukocyte antigens. PLoS ONE. 2013;8(6):e64683. doi: 10.1371/journal.pone.0064683.  [DOI](https://doi.org/10.1371/journal.pone.0064683) | [PMC free article](/articles/PMC3675122/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23762245/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&title=Imputing%20amino%20acid%20polymorphisms%20in%20human%20leukocyte%20antigens.&author=X%20Jia&author=B%20Han&author=S%20Onengut-Gumuscu&volume=8&issue=6&publication_year=2013&pages=e64683&pmid=23762245&doi=10.1371/journal.pone.0064683&)

21. Browning BL, Browning SR. A unified approach to genotype imputation and haplotype-phase inference for large data sets of trios and unrelated individuals. Am J Hum Genet. 2009;84(2):210–223. doi: 10.1016/j.ajhg.2009.01.005.  [DOI](https://doi.org/10.1016/j.ajhg.2009.01.005) | [PMC free article](/articles/PMC2668004/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19200528/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=A%20unified%20approach%20to%20genotype%20imputation%20and%20haplotype-phase%20inference%20for%20large%20data%20sets%20of%20trios%20and%20unrelated%20individuals.&author=BL%20Browning&author=SR%20Browning&volume=84&issue=2&publication_year=2009&pages=210-223&pmid=19200528&doi=10.1016/j.ajhg.2009.01.005&)

22. Spraggs CF, Budde LR, Briley LP, et al. HLA-DQA1*02:01 is a major risk factor for lapatinib-induced hepatotoxicity in women with advanced breast cancer. J Clin Oncol. 2011;29(6):667–673. doi: 10.1200/JCO.2010.31.3197.  [DOI](https://doi.org/10.1200/JCO.2010.31.3197) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21245432/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=HLA-DQA1*02:01%20is%20a%20major%20risk%20factor%20for%20lapatinib-induced%20hepatotoxicity%20in%20women%20with%20advanced%20breast%20cancer.&author=CF%20Spraggs&author=LR%20Budde&author=LP%20Briley&volume=29&issue=6&publication_year=2011&pages=667-673&pmid=21245432&doi=10.1200/JCO.2010.31.3197&)

23. Simon MR, Jan M, Yee J, et al. Tryptase is not cleared by the kidneys into the urine. Int Arch Allergy Immunol. 2010;152(1):28–31. doi: 10.1159/000260080.  [DOI](https://doi.org/10.1159/000260080) | [PMC free article](/articles/PMC2956006/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19940502/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20Arch%20Allergy%20Immunol&title=Tryptase%20is%20not%20cleared%20by%20the%20kidneys%20into%20the%20urine.&author=MR%20Simon&author=M%20Jan&author=J%20Yee&volume=152&issue=1&publication_year=2010&pages=28-31&pmid=19940502&doi=10.1159/000260080&)

24. Vita R, Zarebski L, Greenbaum JA, et al. The immune epitope database 2.0. Nucleic Acids Res. 2010;38(Database issue):D854–D862. doi: 10.1093/nar/gkp1004.  [DOI](https://doi.org/10.1093/nar/gkp1004) | [PMC free article](/articles/PMC2808938/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19906713/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=The%20immune%20epitope%20database%202.0.&author=R%20Vita&author=L%20Zarebski&author=JA%20Greenbaum&volume=38&issue=Database%20issue&publication_year=2010&pages=D854-D862&pmid=19906713&doi=10.1093/nar/gkp1004&)

25. Wang P, Sidney J, Dow C, Mothé B, Sette A, Peters B. A systematic assessment of MHC class II peptide binding predictions and evaluation of a consensus approach. PLOS Comput Biol. 2008;4(4):e1000048. doi: 10.1371/journal.pcbi.1000048.  [DOI](https://doi.org/10.1371/journal.pcbi.1000048) | [PMC free article](/articles/PMC2267221/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18389056/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLOS%20Comput%20Biol&title=A%20systematic%20assessment%20of%20MHC%20class%20II%20peptide%20binding%20predictions%20and%20evaluation%20of%20a%20consensus%20approach.&author=P%20Wang&author=J%20Sidney&author=C%20Dow&author=B%20Moth%C3%A9&author=A%20Sette&volume=4&issue=4&publication_year=2008&pages=e1000048&pmid=18389056&doi=10.1371/journal.pcbi.1000048&)

26. Wang P, Sidney J, Kim Y, et al. Peptide binding predictions for HLA DR, DP and DQ molecules. BMC Bioinformatics. 2010;11:568. doi: 10.1186/1471-2105-11-568.  [DOI](https://doi.org/10.1186/1471-2105-11-568) | [PMC free article](/articles/PMC2998531/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21092157/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Bioinformatics&title=Peptide%20binding%20predictions%20for%20HLA%20DR,%20DP%20and%20DQ%20molecules.&author=P%20Wang&author=J%20Sidney&author=Y%20Kim&volume=11&publication_year=2010&pages=568&pmid=21092157&doi=10.1186/1471-2105-11-568&)

27. Laroche D, Vergnaud MC, Sillard B, Soufarapis H, Bricard H. Biochemical markers of anaphylactoid reactions to drugs. Comparison of plasma histamine and tryptase. Anesthesiology. 1991;75(6):945–949. doi: 10.1097/00000542-199112000-00004.  [DOI](https://doi.org/10.1097/00000542-199112000-00004) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/1741515/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Anesthesiology&title=Biochemical%20markers%20of%20anaphylactoid%20reactions%20to%20drugs.%20Comparison%20of%20plasma%20histamine%20and%20tryptase.&author=D%20Laroche&author=MC%20Vergnaud&author=B%20Sillard&author=H%20Soufarapis&author=H%20Bricard&volume=75&issue=6&publication_year=1991&pages=945-949&pmid=1741515&doi=10.1097/00000542-199112000-00004&)

28. Schleimer RP, Gillespie E, Daiuta R, Lichtenstein LM. Release of histamine from human leukocytes stimulated with the tumor-promoting phorbol diesters. II. Interaction with other stimuli. J Immunol. 1982;128(1):136–140.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/6172475/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Immunol&title=Release%20of%20histamine%20from%20human%20leukocytes%20stimulated%20with%20the%20tumor-promoting%20phorbol%20diesters.%20II.%20Interaction%20with%20other%20stimuli.&author=RP%20Schleimer&author=E%20Gillespie&author=R%20Daiuta&author=LM%20Lichtenstein&volume=128&issue=1&publication_year=1982&pages=136-140&pmid=6172475&)

29. Peitsch MC. Protein modeling by e-mail. Nat Bio-Technol. 1995;13(7):658–660.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Bio-Technol&title=Protein%20modeling%20by%20e-mail.&author=MC%20Peitsch&volume=13&issue=7&publication_year=1995&pages=658-660&)

30. Günther S, Schlundt A, Sticht J, et al. Bidirectional binding of invariant chain peptides to an MHC class II molecule. Proc Natl Acad Sci USA. 2010;107(51):22219–22224. doi: 10.1073/pnas.1014708107.  [DOI](https://doi.org/10.1073/pnas.1014708107) | [PMC free article](/articles/PMC3009805/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21115828/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20USA&title=Bidirectional%20binding%20of%20invariant%20chain%20peptides%20to%20an%20MHC%20class%20II%20molecule.&author=S%20G%C3%BCnther&author=A%20Schlundt&author=J%20Sticht&volume=107&issue=51&publication_year=2010&pages=22219-22224&pmid=21115828&doi=10.1073/pnas.1014708107&)

31. Pierce BG, Hourai Y, Weng Z. Accelerating protein docking in ZDOCK using an advanced 3D convolution library. PLoS ONE. 2011;6(9):e24657. doi: 10.1371/journal.pone.0024657.  [DOI](https://doi.org/10.1371/journal.pone.0024657) | [PMC free article](/articles/PMC3176283/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21949741/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&title=Accelerating%20protein%20docking%20in%20ZDOCK%20using%20an%20advanced%203D%20convolution%20library.&author=BG%20Pierce&author=Y%20Hourai&author=Z%20Weng&volume=6&issue=9&publication_year=2011&pages=e24657&pmid=21949741&doi=10.1371/journal.pone.0024657&)

32. Raveh B, London N, Schueler-Furman O. Sub-angstrom modeling of complexes between flexible peptides and globular proteins. Proteins. 2010;78(9):2029–2040. doi: 10.1002/prot.22716.  [DOI](https://doi.org/10.1002/prot.22716) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20455260/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proteins&title=Sub-angstrom%20modeling%20of%20complexes%20between%20flexible%20peptides%20and%20globular%20proteins.&author=B%20Raveh&author=N%20London&author=O%20Schueler-Furman&volume=78&issue=9&publication_year=2010&pages=2029-2040&pmid=20455260&doi=10.1002/prot.22716&)

33. Case D, Darden TA, Cheatham TE, et al. Amber 12. San Francisco: University of California; 2012.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Amber%2012&author=D%20Case&author=TA%20Darden&author=TE%20Cheatham&publication_year=2012&)

34. Hornak V, Abel R, Okur A, Strockbine B, Roitberg A, Simmerling C. Comparison of multiple Amber force fields and development of improved protein backbone parameters. Proteins. 2006;65(3):712–725. doi: 10.1002/prot.21123.  [DOI](https://doi.org/10.1002/prot.21123) | [PMC free article](/articles/PMC4805110/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16981200/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proteins&title=Comparison%20of%20multiple%20Amber%20force%20fields%20and%20development%20of%20improved%20protein%20backbone%20parameters.&author=V%20Hornak&author=R%20Abel&author=A%20Okur&author=B%20Strockbine&author=A%20Roitberg&volume=65&issue=3&publication_year=2006&pages=712-725&pmid=16981200&doi=10.1002/prot.21123&)

35. Wang W, Donini O, Reyes CM, Kollman PA. Biomolecular simulations: recent developments in force fields, simulations of enzyme catalysis, protein-ligand, protein-protein, and protein-nucleic acid noncovalent interactions. Annu Rev Biophys Biomol Struct. 2001;30:211–243. doi: 10.1146/annurev.biophys.30.1.211.  [DOI](https://doi.org/10.1146/annurev.biophys.30.1.211) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11340059/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Annu%20Rev%20Biophys%20Biomol%20Struct&title=Biomolecular%20simulations:%20recent%20developments%20in%20force%20fields,%20simulations%20of%20enzyme%20catalysis,%20protein-ligand,%20protein-protein,%20and%20protein-nucleic%20acid%20noncovalent%20interactions.&author=W%20Wang&author=O%20Donini&author=CM%20Reyes&author=PA%20Kollman&volume=30&publication_year=2001&pages=211-243&pmid=11340059&doi=10.1146/annurev.biophys.30.1.211&)

36. Asselin BL, Whitin JC, Coppola DJ, Rupp IP, Sallan SE, Cohen HJ. Comparative pharmacokinetic studies of three asparaginase preparations. J Clin Oncol. 1993;11(9):1780–1786. doi: 10.1200/JCO.1993.11.9.1780.  [DOI](https://doi.org/10.1200/JCO.1993.11.9.1780) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8355045/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Comparative%20pharmacokinetic%20studies%20of%20three%20asparaginase%20preparations.&author=BL%20Asselin&author=JC%20Whitin&author=DJ%20Coppola&author=IP%20Rupp&author=SE%20Sallan&volume=11&issue=9&publication_year=1993&pages=1780-1786&pmid=8355045&doi=10.1200/JCO.1993.11.9.1780&)

37. Raetz EA, Salzer WL. Tolerability and efficacy of L-asparaginase therapy in pediatric patients with acute lymphoblastic leukemia. J Pediatr Hematol Oncol. 2010;32(7):554–563. doi: 10.1097/MPH.0b013e3181e6f003.  [DOI](https://doi.org/10.1097/MPH.0b013e3181e6f003) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20724951/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pediatr%20Hematol%20Oncol&title=Tolerability%20and%20efficacy%20of%20L-asparaginase%20therapy%20in%20pediatric%20patients%20with%20acute%20lymphoblastic%20leukemia.&author=EA%20Raetz&author=WL%20Salzer&volume=32&issue=7&publication_year=2010&pages=554-563&pmid=20724951&doi=10.1097/MPH.0b013e3181e6f003&)

38. Avramis VI, Avramis EV, Hunter W, Long MC. Immunogenicity of native or pegylated E. coli and Erwinia asparaginases assessed by ELISA and surface plasmon resonance (SPR-biacore) assays of IgG antibodies (Ab) in sera from patients with acute lymphoblastic leukemia (ALL). Anticancer Res. 2009;29(1):299–302.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/19331165/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Anticancer%20Res&title=Immunogenicity%20of%20native%20or%20pegylated%20E.%20coli%20and%20Erwinia%20asparaginases%20assessed%20by%20ELISA%20and%20surface%20plasmon%20resonance%20(SPR-biacore)%20assays%20of%20IgG%20antibodies%20(Ab)%20in%20sera%20from%20patients%20with%20acute%20lymphoblastic%20leukemia%20(ALL).&author=VI%20Avramis&author=EV%20Avramis&author=W%20Hunter&author=MC%20Long&volume=29&issue=1&publication_year=2009&pages=299-302&pmid=19331165&)

39. Ebeid EN, Kamel MM, Ali BA. Detection of anti-asparaginase antibodies during therapy with E.coli asparaginase in children with newly diagnosed acute lymphoblastic leukemia and lymphoma. J Egypt Natl Canc Inst. 2008;20(2):127–133.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/20029468/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Egypt%20Natl%20Canc%20Inst&title=Detection%20of%20anti-asparaginase%20antibodies%20during%20therapy%20with%20E.coli%20asparaginase%20in%20children%20with%20newly%20diagnosed%20acute%20lymphoblastic%20leukemia%20and%20lymphoma.&author=EN%20Ebeid&author=MM%20Kamel&author=BA%20Ali&volume=20&issue=2&publication_year=2008&pages=127-133&pmid=20029468&)

40. Vrooman LM, Supko JG, Neuberg DS, et al. Erwinia asparaginase after allergy to E. coli asparaginase in children with acute lymphoblastic leukemia. Pediatr Blood Cancer. 2010;54(2):199–205. doi: 10.1002/pbc.22225.  [DOI](https://doi.org/10.1002/pbc.22225) | [PMC free article](/articles/PMC3706086/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19672973/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pediatr%20Blood%20Cancer&title=Erwinia%20asparaginase%20after%20allergy%20to%20E.%20coli%20asparaginase%20in%20children%20with%20acute%20lymphoblastic%20leukemia.&author=LM%20Vrooman&author=JG%20Supko&author=DS%20Neuberg&volume=54&issue=2&publication_year=2010&pages=199-205&pmid=19672973&doi=10.1002/pbc.22225&)

41. Woo MH, Hak LJ, Storm MC, et al. Anti-asparaginase antibodies following E. coli asparaginase therapy in pediatric acute lymphoblastic leukemia. Leukemia. 1998;12(10):1527–1533. doi: 10.1038/sj.leu.2401162.  [DOI](https://doi.org/10.1038/sj.leu.2401162) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9766495/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Leukemia&title=Anti-asparaginase%20antibodies%20following%20E.%20coli%20asparaginase%20therapy%20in%20pediatric%20acute%20lymphoblastic%20leukemia.&author=MH%20Woo&author=LJ%20Hak&author=MC%20Storm&volume=12&issue=10&publication_year=1998&pages=1527-1533&pmid=9766495&doi=10.1038/sj.leu.2401162&)

42. Willer A, Gerss J, König T, et al. Anti-Escherichia coli asparaginase antibody levels determine the activity of second-line treatment with pegylated E coli asparaginase: a retrospective analysis within the ALL-BFM trials. Blood. 2011;118(22):5774–5782. doi: 10.1182/blood-2011-07-367904.  [DOI](https://doi.org/10.1182/blood-2011-07-367904) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21940824/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Anti-Escherichia%20coli%20asparaginase%20antibody%20levels%20determine%20the%20activity%20of%20second-line%20treatment%20with%20pegylated%20E%20coli%20asparaginase:%20a%20retrospective%20analysis%20within%20the%20ALL-BFM%20trials.&author=A%20Willer&author=J%20Gerss&author=T%20K%C3%B6nig&volume=118&issue=22&publication_year=2011&pages=5774-5782&pmid=21940824&doi=10.1182/blood-2011-07-367904&)

43. Panosyan EH, Seibel NL, Martin-Aragon S, et al. Children’s Cancer Group Study CCG-1961. Asparaginase antibody and asparaginase activity in children with higher-risk acute lymphoblastic leukemia: Children’s Cancer Group Study CCG-1961. J Pediatr Hematol Oncol. 2004;26(4):217–226. doi: 10.1097/00043426-200404000-00002.  [DOI](https://doi.org/10.1097/00043426-200404000-00002) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15087948/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pediatr%20Hematol%20Oncol&title=Asparaginase%20antibody%20and%20asparaginase%20activity%20in%20children%20with%20higher-risk%20acute%20lymphoblastic%20leukemia:%20Children%E2%80%99s%20Cancer%20Group%20Study%20CCG-1961.&author=EH%20Panosyan&author=NL%20Seibel&author=S%20Martin-Aragon&volume=26&issue=4&publication_year=2004&pages=217-226&pmid=15087948&doi=10.1097/00043426-200404000-00002&)

44. Albertsen BK, Schrøder H, Jakobsen P, et al. Antibody formation during intravenous and intramuscular therapy with Erwinia asparaginase. Med Pediatr Oncol. 2002;38(5):310–316. doi: 10.1002/mpo.10096.  [DOI](https://doi.org/10.1002/mpo.10096) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11979454/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Med%20Pediatr%20Oncol&title=Antibody%20formation%20during%20intravenous%20and%20intramuscular%20therapy%20with%20Erwinia%20asparaginase.&author=BK%20Albertsen&author=H%20Schr%C3%B8der&author=P%20Jakobsen&volume=38&issue=5&publication_year=2002&pages=310-316&pmid=11979454&doi=10.1002/mpo.10096&)

45. Eden OB, Shaw MP, Lilleyman JS, Richards S. Non-randomised study comparing toxicity of Escherichia coli and Erwinia asparaginase in children with leukaemia. Med Pediatr Oncol. 1990;18(6):497–502. doi: 10.1002/mpo.2950180612.  [DOI](https://doi.org/10.1002/mpo.2950180612) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/2233523/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Med%20Pediatr%20Oncol&title=Non-randomised%20study%20comparing%20toxicity%20of%20Escherichia%20coli%20and%20Erwinia%20asparaginase%20in%20children%20with%20leukaemia.&author=OB%20Eden&author=MP%20Shaw&author=JS%20Lilleyman&author=S%20Richards&volume=18&issue=6&publication_year=1990&pages=497-502&pmid=2233523&doi=10.1002/mpo.2950180612&)

46. Woo MH, Hak LJ, Storm MC, et al. Hypersensitivity or development of antibodies to asparaginase does not impact treatment outcome of childhood acute lymphoblastic leukemia. J Clin Oncol. 2000;18(7):1525–1532. doi: 10.1200/JCO.2000.18.7.1525.  [DOI](https://doi.org/10.1200/JCO.2000.18.7.1525) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10735901/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Hypersensitivity%20or%20development%20of%20antibodies%20to%20asparaginase%20does%20not%20impact%20treatment%20outcome%20of%20childhood%20acute%20lymphoblastic%20leukemia.&author=MH%20Woo&author=LJ%20Hak&author=MC%20Storm&volume=18&issue=7&publication_year=2000&pages=1525-1532&pmid=10735901&doi=10.1200/JCO.2000.18.7.1525&)

47. Evans WE, Tsiatis A, Rivera G, et al. Anaphylactoid reactions to Escherichia coli and Erwinia asparaginase in children with leukemia and lymphoma. Cancer. 1982;49(7):1378–1383. doi: 10.1002/1097-0142(19820401)49:7<1378::aid-cncr2820490713>3.0.co;2-z.  [DOI](https://doi.org/10.1002/1097-0142(19820401)49:7<1378::aid-cncr2820490713>3.0.co;2-z) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7037164/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer&title=Anaphylactoid%20reactions%20to%20Escherichia%20coli%20and%20Erwinia%20asparaginase%20in%20children%20with%20leukemia%20and%20lymphoma.&author=WE%20Evans&author=A%20Tsiatis&author=G%20Rivera&volume=49&issue=7&publication_year=1982&pages=1378-1383&pmid=7037164&doi=10.1002/1097-0142(19820401)49:7<1378::aid-cncr2820490713>3.0.co;2-z&)

48. Chen SH, Pei D, Yang W, et al. Genetic variations in GRIA1 on chromosome 5q33 related to asparaginase hypersensitivity. Clin Pharmacol Ther. 2010;88(2):191–196. doi: 10.1038/clpt.2010.94.  [DOI](https://doi.org/10.1038/clpt.2010.94) | [PMC free article](/articles/PMC3000799/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20592726/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Genetic%20variations%20in%20GRIA1%20on%20chromosome%205q33%20related%20to%20asparaginase%20hypersensitivity.&author=SH%20Chen&author=D%20Pei&author=W%20Yang&volume=88&issue=2&publication_year=2010&pages=191-196&pmid=20592726&doi=10.1038/clpt.2010.94&)

49. Trowsdale J. The MHC, disease and selection. Immunol Lett. 2011;137(1-2):1–8. doi: 10.1016/j.imlet.2011.01.002.  [DOI](https://doi.org/10.1016/j.imlet.2011.01.002) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21262263/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Immunol%20Lett&title=The%20MHC,%20disease%20and%20selection.&author=J%20Trowsdale&volume=137&issue=1-2&publication_year=2011&pages=1-8&pmid=21262263&doi=10.1016/j.imlet.2011.01.002&)

50. Price P, Witt C, Allcock R, et al. The genetic basis for the association of the 8.1 ancestral haplotype (A1, B8, DR3) with multiple immunopathological diseases. Immunol Rev. 1999;167:257–274. doi: 10.1111/j.1600-065x.1999.tb01398.x.  [DOI](https://doi.org/10.1111/j.1600-065x.1999.tb01398.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10319267/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Immunol%20Rev&title=The%20genetic%20basis%20for%20the%20association%20of%20the%208.1%20ancestral%20haplotype%20(A1,%20B8,%20DR3)%20with%20multiple%20immunopathological%20diseases.&author=P%20Price&author=C%20Witt&author=R%20Allcock&volume=167&publication_year=1999&pages=257-274&pmid=10319267&doi=10.1111/j.1600-065x.1999.tb01398.x&)

51. Sollid LM, Markussen G, Ek J, Gjerde H, Vartdal F, Thorsby E. Evidence for a primary association of celiac disease to a particular HLA-DQ alpha/beta heterodimer. J Exp Med. 1989;169(1):345–350. doi: 10.1084/jem.169.1.345.  [DOI](https://doi.org/10.1084/jem.169.1.345) | [PMC free article](/articles/PMC2189170/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/2909659/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Exp%20Med&title=Evidence%20for%20a%20primary%20association%20of%20celiac%20disease%20to%20a%20particular%20HLA-DQ%20alpha/beta%20heterodimer.&author=LM%20Sollid&author=G%20Markussen&author=J%20Ek&author=H%20Gjerde&author=F%20Vartdal&volume=169&issue=1&publication_year=1989&pages=345-350&pmid=2909659&doi=10.1084/jem.169.1.345&)

52. Raychaudhuri S, Sandor C, Stahl EA, et al. Five amino acids in three HLA proteins explain most of the association between MHC and seropositive rheumatoid arthritis. Nat Genet. 2012;44(3):291–296. doi: 10.1038/ng.1076.  [DOI](https://doi.org/10.1038/ng.1076) | [PMC free article](/articles/PMC3288335/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22286218/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Five%20amino%20acids%20in%20three%20HLA%20proteins%20explain%20most%20of%20the%20association%20between%20MHC%20and%20seropositive%20rheumatoid%20arthritis.&author=S%20Raychaudhuri&author=C%20Sandor&author=EA%20Stahl&volume=44&issue=3&publication_year=2012&pages=291-296&pmid=22286218&doi=10.1038/ng.1076&)

53. Stastny P. Association of the B-cell alloantigen DRw4 with rheumatoid arthritis. N Engl J Med. 1978;298(16):869–871. doi: 10.1056/NEJM197804202981602.  [DOI](https://doi.org/10.1056/NEJM197804202981602) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/147420/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Association%20of%20the%20B-cell%20alloantigen%20DRw4%20with%20rheumatoid%20arthritis.&author=P%20Stastny&volume=298&issue=16&publication_year=1978&pages=869-871&pmid=147420&doi=10.1056/NEJM197804202981602&)

54. Erlich HA, Valdes AM, McDevitt SL, et al. Type 1 Diabetes Genetics Consortium (T1DGC) Next generation sequencing reveals the association of DRB3*02:02 with type 1 diabetes. Diabetes. 2013;62(7):2618–2622. doi: 10.2337/db12-1387.  [DOI](https://doi.org/10.2337/db12-1387) | [PMC free article](/articles/PMC3712046/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23462545/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Diabetes&title=Next%20generation%20sequencing%20reveals%20the%20association%20of%20DRB3*02:02%20with%20type%201%20diabetes.&author=HA%20Erlich&author=AM%20Valdes&author=SL%20McDevitt&volume=62&issue=7&publication_year=2013&pages=2618-2622&pmid=23462545&doi=10.2337/db12-1387&)

55. Davies JL, Kawaguchi Y, Bennett ST, et al. A genome-wide search for human type 1 diabetes susceptibility genes. Nature. 1994;371(6493):130–136. doi: 10.1038/371130a0.  [DOI](https://doi.org/10.1038/371130a0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8072542/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=A%20genome-wide%20search%20for%20human%20type%201%20diabetes%20susceptibility%20genes.&author=JL%20Davies&author=Y%20Kawaguchi&author=ST%20Bennett&volume=371&issue=6493&publication_year=1994&pages=130-136&pmid=8072542&doi=10.1038/371130a0&)

56. Oksenberg JR, Barcellos LF. Multiple sclerosis genetics: leaving no stone unturned. Genes Immun. 2005;6(5):375–387. doi: 10.1038/sj.gene.6364237.  [DOI](https://doi.org/10.1038/sj.gene.6364237) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15973459/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genes%20Immun&title=Multiple%20sclerosis%20genetics:%20leaving%20no%20stone%20unturned.&author=JR%20Oksenberg&author=LF%20Barcellos&volume=6&issue=5&publication_year=2005&pages=375-387&pmid=15973459&doi=10.1038/sj.gene.6364237&)

57. Mallal S, Nolan D, Witt C, et al. Association between presence of HLA-B*5701, HLA-DR7, and HLA-DQ3 and hypersensitivity to HIV-1 reverse-transcriptase inhibitor abacavir. Lancet. 2002;359(9308):727–732. doi: 10.1016/s0140-6736(02)07873-x.  [DOI](https://doi.org/10.1016/s0140-6736(02)07873-x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11888582/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Association%20between%20presence%20of%20HLA-B*5701,%20HLA-DR7,%20and%20HLA-DQ3%20and%20hypersensitivity%20to%20HIV-1%20reverse-transcriptase%20inhibitor%20abacavir.&author=S%20Mallal&author=D%20Nolan&author=C%20Witt&volume=359&issue=9308&publication_year=2002&pages=727-732&pmid=11888582&doi=10.1016/s0140-6736(02)07873-x&)

58. Hetherington S, Hughes AR, Mosteller M, et al. Genetic variations in HLA-B region and hypersensitivity reactions to abacavir. Lancet. 2002;359(9312):1121–1122. doi: 10.1016/S0140-6736(02)08158-8.  [DOI](https://doi.org/10.1016/S0140-6736(02)08158-8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11943262/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Genetic%20variations%20in%20HLA-B%20region%20and%20hypersensitivity%20reactions%20to%20abacavir.&author=S%20Hetherington&author=AR%20Hughes&author=M%20Mosteller&volume=359&issue=9312&publication_year=2002&pages=1121-1122&pmid=11943262&doi=10.1016/S0140-6736(02)08158-8&)

59. Chung WH, Hung SI, Hong HS, et al. Medical genetics: a marker for Stevens-Johnson syndrome. Nature. 2004;428(6982):486. doi: 10.1038/428486a.  [DOI](https://doi.org/10.1038/428486a) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15057820/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Medical%20genetics:%20a%20marker%20for%20Stevens-Johnson%20syndrome.&author=WH%20Chung&author=SI%20Hung&author=HS%20Hong&volume=428&issue=6982&publication_year=2004&pages=486&pmid=15057820&doi=10.1038/428486a&)

60. Hung SI, Chung WH, Liou LB, et al. HLA-B*5801 allele as a genetic marker for severe cutaneous adverse reactions caused by allopurinol. Proc Natl Acad Sci USA. 2005;102(11):4134–4139. doi: 10.1073/pnas.0409500102.  [DOI](https://doi.org/10.1073/pnas.0409500102) | [PMC free article](/articles/PMC554812/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15743917/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20USA&title=HLA-B*5801%20allele%20as%20a%20genetic%20marker%20for%20severe%20cutaneous%20adverse%20reactions%20caused%20by%20allopurinol.&author=SI%20Hung&author=WH%20Chung&author=LB%20Liou&volume=102&issue=11&publication_year=2005&pages=4134-4139&pmid=15743917&doi=10.1073/pnas.0409500102&)

61. Tassaneeyakul W, Jantararoungtong T, Chen P, et al. Strong association between HLA-B*5801 and allopurinol-induced Stevens-Johnson syndrome and toxic epidermal necrolysis in a Thai population. Pharmacogenet Genomics. 2009;19(9):704–709. doi: 10.1097/FPC.0b013e328330a3b8.  [DOI](https://doi.org/10.1097/FPC.0b013e328330a3b8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19696695/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Strong%20association%20between%20HLA-B*5801%20and%20allopurinol-induced%20Stevens-Johnson%20syndrome%20and%20toxic%20epidermal%20necrolysis%20in%20a%20Thai%20population.&author=W%20Tassaneeyakul&author=T%20Jantararoungtong&author=P%20Chen&volume=19&issue=9&publication_year=2009&pages=704-709&pmid=19696695&doi=10.1097/FPC.0b013e328330a3b8&)

62. Kaniwa N, Saito Y, Aihara M, et al. JSAR research group. HLA-B locus in Japanese patients with anti-epileptics and allopurinol-related Stevens-Johnson syndrome and toxic epidermal necrolysis. Pharmacogenomics. 2008;9(11):1617–1622. doi: 10.2217/14622416.9.11.1617.  [DOI](https://doi.org/10.2217/14622416.9.11.1617) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19018717/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=HLA-B%20locus%20in%20Japanese%20patients%20with%20anti-epileptics%20and%20allopurinol-related%20Stevens-Johnson%20syndrome%20and%20toxic%20epidermal%20necrolysis.&author=N%20Kaniwa&author=Y%20Saito&author=M%20Aihara&volume=9&issue=11&publication_year=2008&pages=1617-1622&pmid=19018717&doi=10.2217/14622416.9.11.1617&)

63. Hoffmann S, Cepok S, Grummel V, et al. doi: 10.1016/j.ajhg.2008.07.006. HLA-DRB1*0401 and HLA-DRB1*0408 are strongly associated with the development of antibodies against interferon-beta therapy in multiple sclerosis [published correction appears in Am J Hum Genet. 2008;83(4):541]. Am J Hum Genet. 2008;83(2):219-227.  [DOI](https://doi.org/10.1016/j.ajhg.2008.07.006) | [PMC free article](/articles/PMC2495071/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18656179/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Hoffmann%20S,%20Cepok%20S,%20Grummel%20V,%20et%20al.%20doi:%2010.1016/j.ajhg.2008.07.006.%20HLA-DRB1*0401%20and%20HLA-DRB1*0408%20are%20strongly%20associated%20with%20the%20development%20of%20antibodies%20against%20interferon-beta%20therapy%20in%20multiple%20sclerosis%20%5Bpublished%20correction%20appears%20in%20Am%20J%20Hum%20Genet.%202008;83(4):541%5D.%20Am%20J%20Hum%20Genet.%202008;83(2):219-227.)

64. Weber F, Cepok S, Wolf C, et al. Single-nucleotide polymorphisms in HLA- and non-HLA genes associated with the development of antibodies to interferon-β therapy in multiple sclerosis patients. Pharmacogenomics J. 2012;12(3):238–245. doi: 10.1038/tpj.2011.14.  [DOI](https://doi.org/10.1038/tpj.2011.14) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21502966/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=Single-nucleotide%20polymorphisms%20in%20HLA-%20and%20non-HLA%20genes%20associated%20with%20the%20development%20of%20antibodies%20to%20interferon-%CE%B2%20therapy%20in%20multiple%20sclerosis%20patients.&author=F%20Weber&author=S%20Cepok&author=C%20Wolf&volume=12&issue=3&publication_year=2012&pages=238-245&pmid=21502966&doi=10.1038/tpj.2011.14&)

65. Vigneault F, Woods M, Buzon MJ, et al. Transcriptional profiling of CD4 T cells identifies distinct subgroups of HIV-1 elite controllers. J Virol. 2011;85(6):3015–3019. doi: 10.1128/JVI.01846-10.  [DOI](https://doi.org/10.1128/JVI.01846-10) | [PMC free article](/articles/PMC3067917/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21177805/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Virol&title=Transcriptional%20profiling%20of%20CD4%20T%20cells%20identifies%20distinct%20subgroups%20of%20HIV-1%20elite%20controllers.&author=F%20Vigneault&author=M%20Woods&author=MJ%20Buzon&volume=85&issue=6&publication_year=2011&pages=3015-3019&pmid=21177805&doi=10.1128/JVI.01846-10&)

66. Pereyra F, Jia X, McLaren PJ, et al. International HIV Controllers Study. The major genetic determinants of HIV-1 control affect HLA class I peptide presentation. Science. 2010;330(6010):1551–1557. doi: 10.1126/science.1195271.  [DOI](https://doi.org/10.1126/science.1195271) | [PMC free article](/articles/PMC3235490/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21051598/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=The%20major%20genetic%20determinants%20of%20HIV-1%20control%20affect%20HLA%20class%20I%20peptide%20presentation.&author=F%20Pereyra&author=X%20Jia&author=PJ%20McLaren&volume=330&issue=6010&publication_year=2010&pages=1551-1557&pmid=21051598&doi=10.1126/science.1195271&)

67. Foo JN, Smedby KE, Akers NK, et al. Coding variants at hexa-allelic amino acid 13 of HLA-DRB1 explain independent SNP associations with follicular lymphoma risk. Am J Hum Genet. 2013;93(1):167–172. doi: 10.1016/j.ajhg.2013.05.020.  [DOI](https://doi.org/10.1016/j.ajhg.2013.05.020) | [PMC free article](/articles/PMC3710749/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23791106/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=Coding%20variants%20at%20hexa-allelic%20amino%20acid%2013%20of%20HLA-DRB1%20explain%20independent%20SNP%20associations%20with%20follicular%20lymphoma%20risk.&author=JN%20Foo&author=KE%20Smedby&author=NK%20Akers&volume=93&issue=1&publication_year=2013&pages=167-172&pmid=23791106&doi=10.1016/j.ajhg.2013.05.020&)

68. McCormack M, Alfirevic A, Bourgeois S, et al. HLA-A*3101 and carbamazepine-induced hypersensitivity reactions in Europeans. N Engl J Med. 2011;364(12):1134–1143. doi: 10.1056/NEJMoa1013297.  [DOI](https://doi.org/10.1056/NEJMoa1013297) | [PMC free article](/articles/PMC3113609/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21428769/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=HLA-A*3101%20and%20carbamazepine-induced%20hypersensitivity%20reactions%20in%20Europeans.&author=M%20McCormack&author=A%20Alfirevic&author=S%20Bourgeois&volume=364&issue=12&publication_year=2011&pages=1134-1143&pmid=21428769&doi=10.1056/NEJMoa1013297&)

69. de Bakker PI, Raychaudhuri S. Interrogating the major histocompatibility complex with high-throughput genomics. Hum Mol Genet. 2012;21(R1):R29–R36. doi: 10.1093/hmg/dds384.  [DOI](https://doi.org/10.1093/hmg/dds384) | [PMC free article](/articles/PMC3459647/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22976473/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mol%20Genet&title=Interrogating%20the%20major%20histocompatibility%20complex%20with%20high-throughput%20genomics.&author=PI%20de%20Bakker&author=S%20Raychaudhuri&volume=21&issue=R1&publication_year=2012&pages=R29-R36&pmid=22976473&doi=10.1093/hmg/dds384&)

70. Foley PJ, McGrath DS, Puscinska E, et al. Human leukocyte antigen-DRB1 position 11 residues are a common protective marker for sarcoidosis. Am J Respir Cell Mol Biol. 2001;25(3):272–277. doi: 10.1165/ajrcmb.25.3.4261.  [DOI](https://doi.org/10.1165/ajrcmb.25.3.4261) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11588003/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Respir%20Cell%20Mol%20Biol&title=Human%20leukocyte%20antigen-DRB1%20position%2011%20residues%20are%20a%20common%20protective%20marker%20for%20sarcoidosis.&author=PJ%20Foley&author=DS%20McGrath&author=E%20Puscinska&volume=25&issue=3&publication_year=2001&pages=272-277&pmid=11588003&doi=10.1165/ajrcmb.25.3.4261&)

71. Ramagopalan SV, McMahon R, Dyment DA, Sadovnick AD, Ebers GC, Wittkowski KM. An extension to a statistical approach for family based association studies provides insights into genetic risk factors for multiple sclerosis in the HLA-DRB1 gene. BMC Med Genet. 2009;10:10. doi: 10.1186/1471-2350-10-10.  [DOI](https://doi.org/10.1186/1471-2350-10-10) | [PMC free article](/articles/PMC2669470/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19193207/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Med%20Genet&title=An%20extension%20to%20a%20statistical%20approach%20for%20family%20based%20association%20studies%20provides%20insights%20into%20genetic%20risk%20factors%20for%20multiple%20sclerosis%20in%20the%20HLA-DRB1%20gene.&author=SV%20Ramagopalan&author=R%20McMahon&author=DA%20Dyment&author=AD%20Sadovnick&author=GC%20Ebers&volume=10&publication_year=2009&pages=10&pmid=19193207&doi=10.1186/1471-2350-10-10&)

72. Barcellos LF, Sawcer S, Ramsay PP, et al. Heterogeneity at the HLA-DRB1 locus and risk for multiple sclerosis. Hum Mol Genet. 2006;15(18):2813–2824. doi: 10.1093/hmg/ddl223.  [DOI](https://doi.org/10.1093/hmg/ddl223) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16905561/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mol%20Genet&title=Heterogeneity%20at%20the%20HLA-DRB1%20locus%20and%20risk%20for%20multiple%20sclerosis.&author=LF%20Barcellos&author=S%20Sawcer&author=PP%20Ramsay&volume=15&issue=18&publication_year=2006&pages=2813-2824&pmid=16905561&doi=10.1093/hmg/ddl223&)

73. Jee SH, Tsai TF, Tsai WL, Liaw SH, Chang CH, Hu CY. HLA-DRB1*0701 and DRB1*1401 are associated with genetic susceptibility to psoriasis vulgaris in a Taiwanese population. Br J Dermatol. 1998;139(6):978–983. doi: 10.1046/j.1365-2133.1998.02552.x.  [DOI](https://doi.org/10.1046/j.1365-2133.1998.02552.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9990359/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Dermatol&title=HLA-DRB1*0701%20and%20DRB1*1401%20are%20associated%20with%20genetic%20susceptibility%20to%20psoriasis%20vulgaris%20in%20a%20Taiwanese%20population.&author=SH%20Jee&author=TF%20Tsai&author=WL%20Tsai&author=SH%20Liaw&author=CH%20Chang&volume=139&issue=6&publication_year=1998&pages=978-983&pmid=9990359&doi=10.1046/j.1365-2133.1998.02552.x&)

74. Shindo Y, Inoko H, Yamamoto T, Ohno S. HLA-DRB1 typing of Vogt-Koyanagi-Harada’s disease by PCR-RFLP and the strong association with DRB1*0405 and DRB1*0410. Br J Ophthalmol. 1994;78(3):223–226. doi: 10.1136/bjo.78.3.223.  [DOI](https://doi.org/10.1136/bjo.78.3.223) | [PMC free article](/articles/PMC504742/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7908535/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Ophthalmol&title=HLA-DRB1%20typing%20of%20Vogt-Koyanagi-Harada%E2%80%99s%20disease%20by%20PCR-RFLP%20and%20the%20strong%20association%20with%20DRB1*0405%20and%20DRB1*0410.&author=Y%20Shindo&author=H%20Inoko&author=T%20Yamamoto&author=S%20Ohno&volume=78&issue=3&publication_year=1994&pages=223-226&pmid=7908535&doi=10.1136/bjo.78.3.223&)

75. Umemura T, Joshita S, Ichijo T, et al. Shinshu PBC Study Group. Human leukocyte antigen class II molecules confer both susceptibility and progression in Japanese patients with primary biliary cirrhosis. Hepatology. 2012;55(2):506–511. doi: 10.1002/hep.24705.  [DOI](https://doi.org/10.1002/hep.24705) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21953406/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hepatology&title=Human%20leukocyte%20antigen%20class%20II%20molecules%20confer%20both%20susceptibility%20and%20progression%20in%20Japanese%20patients%20with%20primary%20biliary%20cirrhosis.&author=T%20Umemura&author=S%20Joshita&author=T%20Ichijo&volume=55&issue=2&publication_year=2012&pages=506-511&pmid=21953406&doi=10.1002/hep.24705&)

76. Ban Y, Davies TF, Greenberg DA, et al. Arginine at position 74 of the HLA-DR beta1 chain is associated with Graves’ disease. Genes Immun. 2004;5(3):203–208. doi: 10.1038/sj.gene.6364059.  [DOI](https://doi.org/10.1038/sj.gene.6364059) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15029234/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genes%20Immun&title=Arginine%20at%20position%2074%20of%20the%20HLA-DR%20beta1%20chain%20is%20associated%20with%20Graves%E2%80%99%20disease.&author=Y%20Ban&author=TF%20Davies&author=DA%20Greenberg&volume=5&issue=3&publication_year=2004&pages=203-208&pmid=15029234&doi=10.1038/sj.gene.6364059&)

77. Simmonds MJ, Howson JM, Heward JM, et al. Regression mapping of association between the human leukocyte antigen region and Graves disease. Am J Hum Genet. 2005;76(1):157–163. doi: 10.1086/426947.  [DOI](https://doi.org/10.1086/426947) | [PMC free article](/articles/PMC1196419/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15558498/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=Regression%20mapping%20of%20association%20between%20the%20human%20leukocyte%20antigen%20region%20and%20Graves%20disease.&author=MJ%20Simmonds&author=JM%20Howson&author=JM%20Heward&volume=76&issue=1&publication_year=2005&pages=157-163&pmid=15558498&doi=10.1086/426947&)

78. Menconi F, Osman R, Monti MC, Greenberg DA, Concepcion ES, Tomer Y. Shared molecular amino acid signature in the HLA-DR peptide binding pocket predisposes to both autoimmune diabetes and thyroiditis. Proc Natl Acad Sci USA. 2010;107(39):16899–16903. doi: 10.1073/pnas.1009511107.  [DOI](https://doi.org/10.1073/pnas.1009511107) | [PMC free article](/articles/PMC2947869/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20837527/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20USA&title=Shared%20molecular%20amino%20acid%20signature%20in%20the%20HLA-DR%20peptide%20binding%20pocket%20predisposes%20to%20both%20autoimmune%20diabetes%20and%20thyroiditis.&author=F%20Menconi&author=R%20Osman&author=MC%20Monti&author=DA%20Greenberg&author=ES%20Concepcion&volume=107&issue=39&publication_year=2010&pages=16899-16903&pmid=20837527&doi=10.1073/pnas.1009511107&)

79. Chevalier MF, Jülg B, Pyo A, et al. HIV-1-specific interleukin-21+ CD4+ T cell responses contribute to durable viral control through the modulation of HIV-specific CD8+ T cell function. J Virol. 2011;85(2):733–741. doi: 10.1128/JVI.02030-10.  [DOI](https://doi.org/10.1128/JVI.02030-10) | [PMC free article](/articles/PMC3020027/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21047960/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Virol&title=HIV-1-specific%20interleukin-21+%20CD4+%20T%20cell%20responses%20contribute%20to%20durable%20viral%20control%20through%20the%20modulation%20of%20HIV-specific%20CD8+%20T%20cell%20function.&author=MF%20Chevalier&author=B%20J%C3%BClg&author=A%20Pyo&volume=85&issue=2&publication_year=2011&pages=733-741&pmid=21047960&doi=10.1128/JVI.02030-10&)

80. Sette A, Peters B. Immune epitope mapping in the post-genomic era: lessons for vaccine development. Curr Opin Immunol. 2007;19(1):106–110. doi: 10.1016/j.coi.2006.11.002.  [DOI](https://doi.org/10.1016/j.coi.2006.11.002) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17113275/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Opin%20Immunol&title=Immune%20epitope%20mapping%20in%20the%20post-genomic%20era:%20lessons%20for%20vaccine%20development.&author=A%20Sette&author=B%20Peters&volume=19&issue=1&publication_year=2007&pages=106-110&pmid=17113275&doi=10.1016/j.coi.2006.11.002&)

81. Yanover C, Jain N, Pierce G, Howard TE, Sauna ZE. Pharmacogenetics and the immunogenicity of protein therapeutics. Nat Biotechnol. 2011;29(10):870–873. doi: 10.1038/nbt.2002.  [DOI](https://doi.org/10.1038/nbt.2002) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21997623/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Biotechnol&title=Pharmacogenetics%20and%20the%20immunogenicity%20of%20protein%20therapeutics.&author=C%20Yanover&author=N%20Jain&author=G%20Pierce&author=TE%20Howard&author=ZE%20Sauna&volume=29&issue=10&publication_year=2011&pages=870-873&pmid=21997623&doi=10.1038/nbt.2002&)

82. Cantor JR, Yoo TH, Dixit A, Iverson BL, Forsthuber TG, Georgiou G. Therapeutic enzyme deimmunization by combinatorial T-cell epitope removal using neutral drift. Proc Natl Acad Sci USA. 2011;108(4):1272–1277. doi: 10.1073/pnas.1014739108.  [DOI](https://doi.org/10.1073/pnas.1014739108) | [PMC free article](/articles/PMC3029727/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21209329/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20USA&title=Therapeutic%20enzyme%20deimmunization%20by%20combinatorial%20T-cell%20epitope%20removal%20using%20neutral%20drift.&author=JR%20Cantor&author=TH%20Yoo&author=A%20Dixit&author=BL%20Iverson&author=TG%20Forsthuber&volume=108&issue=4&publication_year=2011&pages=1272-1277&pmid=21209329&doi=10.1073/pnas.1014739108&)
