# The involvement of Kras gene 3′-UTR polymorphisms in risk of cancer and influence on patient response to anti-EGFR therapy in metastatic colorectal cancer: a meta-analysis

## Metadata
**Authors:** Hou-Qun Ying, Feng Wang, Bang-Shun He, Yu-Qin Pan, Tian-Yi Gao, Ye-Qiong Xu, Rui Li, Qi-Wen Deng, Hui-Lin Sun, Shu-Kui Wang
**Journal:** OncoTargets and therapy
**Date:** 2014 Aug 25
**DOI:** [10.2147/OTT.S65496](https://doi.org/10.2147/OTT.S65496)
**PMID:** 25210463
**PMCID:** PMC4154892
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4154892/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC4154892/pdf/ott-7-1487.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC4154892/pdf/ott-7-1487.pdf)

## Abstract

**Background:** 
Genetic variation of the Kras oncogene is a candidate factor for increasing susceptibility to carcinoma and modulating response of metastatic colorectal cancer (mCRC) patients treated with anti-epidermal growth factor receptor monoclonal antibody (anti-EGFR). However, results from an increasing number of studies concerning the association of Kras gene rs712 and rs61764370 polymorphisms with risk of cancer and treatment of mCRC using anti-EGFR remain equivocal.

**Methods:** 
Risk associations were evaluated in 1,661 cases and 2,139 controls from six studies concerning rs712 and 14,796 cases and 14,985 controls from 29 studies concerning rs61764370. Response association was also examined in a subset of four studies pertaining to rs61764370 and anti-EGFR treatment in mCRC.

**Results:** 
Results of a meta-analysis showed that allele T (P-value of heterogeneity test [PH] =0.08, odds ratio [OR] =1.33, 95% confidence interval [CI]: 1.08–1.64) and genotype GT/TT (PH=0.14, OR =1.30, 95% CI: 1.10–1.55) in rs712 were strongly associated with cancer in Chinese subjects. No evidence of association was observed between rs712 and risk of cancer in the overall population or between rs61764370 and ovarian, breast, colorectal, or non-small-cell lung cancer risk in the Caucasian population. No significant association was found between rs61764370 and patient response to anti-EGFR therapy in mCRC.

**Conclusion:** 
The findings not only provide further evidence that allele T of rs712 increases genetic predisposition to cancer in Chinese population, but also no significant association between rs61764370 and cancer risk in Caucasian population, and suggest that genotype GT/TT of rs61764370 may not be a biomarker for predicting clinical outcome of anti-EGFR therapy in mCRC.

Keywords: rs712, rs61764370, single nuclear polymorphism

### Background

Genetic variation of the *Kras* oncogene is a candidate factor for increasing susceptibility to carcinoma and modulating response of metastatic colorectal cancer (mCRC) patients treated with anti-epidermal growth factor receptor monoclonal antibody (anti-EGFR). However, results from an increasing number of studies concerning the association of Kras gene rs712 and rs61764370 polymorphisms with risk of cancer and treatment of mCRC using anti-EGFR remain equivocal.

### Methods

Risk associations were evaluated in 1,661 cases and 2,139 controls from six studies concerning rs712 and 14,796 cases and 14,985 controls from 29 studies concerning rs61764370. Response association was also examined in a subset of four studies pertaining to rs61764370 and anti-EGFR treatment in mCRC.

### Results

Results of a meta-analysis showed that allele T (*P*-value of heterogeneity test [*P*_H_] =0.08, odds ratio [OR] =1.33, 95% confidence interval [CI]: 1.08–1.64) and genotype GT/TT (*P*_H_=0.14, OR =1.30, 95% CI: 1.10–1.55) in rs712 were strongly associated with cancer in Chinese subjects. No evidence of association was observed between rs712 and risk of cancer in the overall population or between rs61764370 and ovarian, breast, colorectal, or non-small-cell lung cancer risk in the Caucasian population. No significant association was found between rs61764370 and patient response to anti-EGFR therapy in mCRC.

### Conclusion

The findings not only provide further evidence that allele T of rs712 increases genetic predisposition to cancer in Chinese population, but also no significant association between rs61764370 and cancer risk in Caucasian population, and suggest that genotype GT/TT of rs61764370 may not be a biomarker for predicting clinical outcome of anti-EGFR therapy in mCRC.

## Introduction

In spite of abundant emerging data contributing to understanding of the molecular mechanisms of carcinogenesis and cancer prevention, the number of new diagnoses and death rates, especially in developing countries, continue to rise. In the People’s Republic of China, cancer morbidity and mortality rates in 2009 were 285.91/100,000 and 180.54/100,000, respectively, which were higher than the rates of 250.03/100,000 and 166.22/100,000, respectively, in 2004.[1](#b1-ott-7-1487)^–^[3](#b3-ott-7-1487) Further, a 2012 US cancer report showed that approximately 1.6 million new cancer cases and 0.58 million cancer deaths were projected to occur in 2013.[4](#b4-ott-7-1487) Many factors, such as mutation, single nucleotide polymorphism (SNP), and epigenetic dysregulation of oncogene or tumor suppressor gene, have been found to lead to activation of oncogene or expressed silence of tumor suppressor gene and eventually give rise to carcinogenesis.[5](#b5-ott-7-1487)

Kras gene, a member of the Ras gene family, is one of the most important oncogenes in carcinogenesis and acts as an intracellular signal transducer.[6](#b6-ott-7-1487) It encodes a guanosine diphosphate (GDP)/GTP guanosine triphosphate (GTP)-binding protein that belongs to the small GTPase superfamily, regulates signal transduction, and is involved in cell proliferation and differentiation through Kras-related RAF/MEK/MAPK, AKT, and ERK pathways.[6](#b6-ott-7-1487)^–^[8](#b8-ott-7-1487) Mutation of the *Kras* oncogene plays a pivotal role in the pathogenesis of various solid tumors in humans,[9](#b9-ott-7-1487) with a 30%–60% mutation frequency detected in colorectal adenocarcinomas.[10](#b10-ott-7-1487) On the other hand, repression of Kras expression could inhibit tumor growth and invasion by small interfering RNA (siRNA) or microRNA (miRNA).[11](#b11-ott-7-1487) Let-7 miRNA posttranscriptionally regulates Kras oncogene expression by targeting the 3′-untranslated region (3′-UTR) of messenger RNA (mRNA) for degradation or translation repression.[12](#b12-ott-7-1487) Let-7 complementary binding site (LCS) SNPs, located in Kras gene 3′-URT, have been found to modulate the binding ability with let-7,[12](#b12-ott-7-1487) consequently resulting in aberrant expression of Kras gene. Thus, these loci are considered candidate genetic susceptibility factors for carcinogenesis.

Recently, emerging studies concerning let-7 LCS polymorphisms in *Kras* 3′-UTR, rs712 and rs61764370, reported that these SNPs increased risk of cancer and affected the survival of patients with malignant cancer using anti-epidermal growth factor receptor monoclonal antibody (EGFR) therapy in metastatic colorectal cancer (mCRC).[13](#b13-ott-7-1487)^,^[14](#b14-ott-7-1487) However, other studies pertaining to these loci had conflicting conclusions.[15](#b15-ott-7-1487)^,^[16](#b16-ott-7-1487)

On the basis of accumulating evidence, a comprehensive meta-analysis of retrospective and prospective studies was conducted for the following purposes: 1) to evaluate the association of rs712 and rs61764370 with risk of cancer; and 2) to estimate the influence of rs61764370 genotypes on anti-EGFR treatment in mCRC.

## Materials and methods

### Study identification and selection

In this meta-analysis, relevant studies dating to November 2013 were searched for in the PubMed, Google Scholar, Embase, and Wanfang Data in accordance with the Meta-analysis Of Observational Studies in Epidemiology (MOOSE) guidelines.[17](#b17-ott-7-1487) Additional studies were identified by manual retrieval in order to obtain substantial articles. The following search terms were used: 1) “rs712, rs61764370 or LCS6 and tumor, cancer or carcinoma”; 2) “Kras polymorphism and tumor, cancer or carcinoma”; 3) “Let-7, Kras and tumor, cancer or carcinoma”; 4) “Let-7, Kras, LCS6 and cancer, EGFR”. Relevant studies were first identified through review of each retrieved title and abstract. Then, relevant full-text studies were identified as eligible for meta-analysis according to the following inclusion criteria: 1) case control study concerning rs712, rs61764370, and cancer risk, or anti-EGFR therapy in mCRC, in English or Chinese; 2) cases were solid cancer patients and controls were cancer-free healthy individuals; 3) sufficient genotype frequency data were provided for calculating odds ratio (OR) and 95% confidence interval (CI); and 4) genotype distribution of the control group was consistent with Hardy–Weinberg equilibrium. Non-case control studies, reviews, comments, communications, meta-analyses, single-group design studies, and case control studies with duplicated data were excluded from this study.

### Data extraction

Two investigators (Hou-Qun Ying and Feng Wang) independently extracted data from each study identified as eligible per the inclusion and exclusion criteria. A consensus was required for the inclusion of studies. From each eligible study, baseline characteristic data were extracted, which comprised the following: author name or abbreviated study name; year of publication; country; ethnicity; cases and controls; detection method; genotype data; number of total and part responses as well as nonresponses; ORs; and 95% CIs.

### Statistical analysis

Crude ORs and 95% CIs were used as common measurements for assessing the strength between Kras polymorphism and cancer risk as well as response to anti-EGFR therapy in mCRC patients. Heterogeneity was assessed by Cochran’s Q test and *I*^2^,[18](#b18-ott-7-1487)^,^[19](#b19-ott-7-1487) and a *P*-value of heterogeneity test (*P*_H_) <0.10 was considered significant heterogeneity. The fixed model was chosen to evaluate the combined data when the heterogeneity test was assumed to be homogenous; otherwise, the random model was used to estimate the overall effect.[20](#b20-ott-7-1487)^,^[21](#b21-ott-7-1487) Stability of meta-analysis was estimated using sensitivity analysis by omitting each eligible study successively. Both Begg’s funnel plot and Egger’s test were used to establish possible publication bias,[21](#b21-ott-7-1487)^,^[22](#b22-ott-7-1487) and asymmetry of funnel plot and *P*-value of Egger’s test <0.05 were considered to indicate the existence of publication bias. All calculations were performed using Stata (v 11.0; StataCorp LP, College Station, TX, USA) and RevMan (v 5.2; The Nordic Cochrane Centre, The Cochrane Collaboration, Copenhagen, Denmark) software.

## Results

### Eligible studies

The flowchart of the eligible study search is shown in [Figure 1](#f1-ott-7-1487). In total, 364 articles were obtained from the databases and by manual retrieval. According to the inclusion and exclusion criteria, 270 unrelated articles, 61 reviews and meta-analyses, 14 comments or communications, and one study with insufficient genotype data were excluded from the present study. As a result, a total of six case control studies[13](#b13-ott-7-1487)^,^[15](#b15-ott-7-1487)^,^[23](#b23-ott-7-1487)^–^[26](#b26-ott-7-1487) concerning rs712 and cancer risk, 29 case control studies[27](#b27-ott-7-1487)^–^[35](#b35-ott-7-1487) relating to rs61764370 and cancer, and four studies[14](#b14-ott-7-1487)^,^[16](#b16-ott-7-1487)^,^[35](#b35-ott-7-1487)^,^[36](#b36-ott-7-1487) concerning rs61764370 and anti-EGFR treatment in mCRC were enrolled as eligible studies. The baseline characteristics of eligible studies are listed in [Tables 1](#t1-ott-7-1487) and [2](#t2-ott-7-1487).

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/956e/4154892/a062ae53b599/ott-7-1487Fig1.jpg)

Flowchart of retrieval and identification of eligible studies. Abbreviation: EGFR, epidermal growth factor receptor monoclonal antibody.

### Table 1.

| Study and year | Country | Ethnicity | Cases | Controls | Analysis assay |
| --- | --- | --- | --- | --- | --- |
| BEL 201128 | Belgium | Caucasian | 173 invasive epithelial ovarian cancer patients | 253 healthy controls | Fluidigm |
| BWH 201128 | USA | Caucasian | 137 invasive epithelial ovarian cancer patients | 142 healthy controls | Illumina Hap317 |
| Chin et al, 200830 | USA | Caucasian | 325 non-small-cell lung cancer patients | 325 healthy controls | TaqMan®-PCR |
| Chin et al, 2008 (2)30 | USA | Caucasian | 2,205 non-small-cell lung cancer patients | 1,497 healthy controls | TaqMan®-PCR |
| Christensen et al, 200933 | USA | Caucasian | 513 head and neck squamous cell cancer patients | 597 healthy controls | TaqMan®-PCR |
| Cerne et al, 201229 | Slovenia | Caucasian | 530 sporadic and 165 familial breast cancer cases | 270 cancer-free controls | TaqMan®n-PCR |
| DOV 201128 | USA | Caucasian | 698 invasive epithelial ovarian cancer patients | 721 healthy controls | TaqMan®-PCR |
| GER 201128 | Germany | Caucasian | 213 invasive epithelial ovarian cancer patients | 265 healthy controls | Fluidigm |
| HJO 201128 | Germany | Caucasian | 195 invasive epithelial ovarian cancer patients | 151 healthy controls | Fluidigm |
| HMO 201128 | Belarus | Caucasian | 259 invasive epithelial ovarian cancer patients | 426 healthy controls | Fluidigm |
| HOC 201128 | Finland | Caucasian | 350 invasive epithelial ovarian cancer patients | 434 healthy controls | Fluidigm |
| Hollestelle et al, 201127 | the Netherlands | Caucasian | 1,042 breast cancer | 797 cancer-free controls | TaqMan®-PCR |
| HOP 201128 | USA | Caucasian | 365 invasive epithelial ovarian cancer patients | 368 healthy controls | TaqMan®-PCR |
| Kjersem et al, 201235 | Norway | Caucasian | 197 colorectal cancer patients | 358 healthy controls | TaqMan®-PCR |
| Landi et al, 201215 | Czech Republic | Caucasian | 717 colorectal cancer patients | 1,171 healthy volunteers | AS-PCR |
| Li et al, 201323 | People’s Republic of China | Chinese | 181 gastric cancer patients | 674 cancer free controls | PCR-RFLP |
| MAY 201128 | USA | Caucasian | 358 invasive epithelial ovarian cancer patients | 520 healthy controls | Illumina 610 Quad |
| NCO 201128 | USA | Caucasian | 494 invasive epithelial ovarian cancer patients | 655 healthy controls | Illumina 610 Quad |
| NTH 201128 | the Netherlands | Caucasian | 296 invasive epithelial ovarian cancer patients | 327 healthy controls | Fluidigm |
| OVA 201128 | Canada | Caucasian | 494 invasive epithelial ovarian cancer patients | 416 healthy controls | Fluidigm |
| Paranjape et al, 201131 | USA | Caucasian | 415 breast cancer patients | 457 healthy controls | TaqMan® PCR |
| Pan et al, 201413 | People’s Republic of China | Chinese | 339 colorectal cancer patients | 313 healthy controls | PCR-RFLP |
| Pan et al, 201425 | People’s Republic of China | Chinese | 188 nasopharyngeal carcinoma patients | 356 healthy controls | PCR-RFLP |
| Peng et al, 201026 | People’s Republic of China | Chinese | 83 non-small-cell lung cancer patients | 80 healthy volunteers | PCR-RFLP |
| PVM 201128 | Denmark | Caucasian | 201 invasive epithelial ovarian cancer patients | 215 healthy controls | Fluidigm |
| Ratner et al, 201032 | USA | Caucasian | 100 ovarian cancer patients | 101 healthy controls | TaqMan®-PCR |
| Ratner et al, 2010 (2)32 | USA | Caucasian | 320 ovarian cancer patients | 322 healthy controls | TaqMan®-PCR |
| Ryan et al, 201234 | USA | Caucasian | 375 colorectal cancer patients | 202 healthy controls | No data |
| TBO 201128 | USA | Caucasian | 227 invasive epithelial ovarian cancer patients | 168 healthy controls | Illumina 610 Quad |
| TOR 201128 | Canada | Caucasian | 734 invasive epithelial ovarian cancer patients | 556 healthy controls | Illumina 610 Quad |
| UC1 201128 | USA | Caucasian | 192 invasive epithelial ovarian cancer patients | 372 healthy controls | Fluidigm |
| UK-GWAS 201128 | UK | Caucasian | 1,325 invasive epithelial ovarian cancer patients | 1,325 healthy controls | Fluidigm |
| UK2 201128 | UK | Caucasian | 1,778 invasive epithelial ovarian cancer patients | 2,355 healthy controls | Illumina 610 Quad |
| USC 201128 | USA | Caucasian | 260 invasive epithelial ovarian cancer patients | 343 healthy controls | TaqMan®-PCR |
| Yan et al, 201324 | People’s Republic of China | Chinese | 153 glioma patients | 204 healthy controls | PCR-RFLP |

Table 1 Caption: Baseline characteristics of each eligible study concerning Kras polymorphisms and risk of cancer

### Table 2.

| Study and year | Country | Ethnicity | Cases | Anti-EGFR antibody | CR + PR |  | SD + PD |  | P-value |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| TT genotype | TG/GG genotype | TT genotype | TG/GG genotype |  |  |  |  |  |  |
| Graziano et al, 201036 | Italy | Caucasian | 121 metastatic colorectal cancer patients | Cetuximab | 20 | 6 | 67 | 28 | >0.05 |
| Sebio et al, 201316 | Spain | Caucasian | 92 metastatic colorectal cancer patients | Cetuximab and panitumumab | 23 | 0 | 49 | 20 | <0.01 |
| Kjersem et al, 201235 | Norway | Caucasian | 355 metastatic colorectal cancer patients | Cetuximab | 140 | 33 | 157 | 25 | >0.05 |
| Zhang et al, 201114 | USA | Caucasian | 98 metastatic colorectal cancer patients | Cetuximab | 5 | 5 | 78 | 10 | <0.01 |

Table 2 Caption: Baseline characteristics of each eligible study of rs61764370 and clinical outcome of metastatic colorectal cancer patients treated with anti-EGFR

### rs712 and cancer risk

The results of heterogeneity testing and overall effects of meta-analysis and Egger’s test are listed in [Table 3](#t3-ott-7-1487). As shown in [Table 3](#t3-ott-7-1487) and [Figure 2](#f2-ott-7-1487), no significant association was found between rs712 and risk of cancer in the overall population (*P*_H_=0.27, OR =1.10, 95% CI: 0.95–1.28 for genotype GT versus genotype GG; *P*_H_=0.04, OR =1.21, 95% CI: 0.90–1.50 for genotype GT/TT versus genotype GG; *P*_H_=0.002, OR =1.23, 95% CI: 0.98–1.54 for T versus G). After stratifying the population into Chinese and Caucasian subgroups, significant associations were observed in comparisons of GT/TT and GG (*P*_H_=0.14, OR =1.30, 95% CI: 1.10–1.55) and T and G (*P*_H_=0.08, OR =1.33, 95% CI: 1.08–1.64) in the Chinese population.

### Table 3.

| Locus | Comparison | Population/Subgroup | PH | I2 | PZ | PE | OR and 95% CI |
| --- | --- | --- | --- | --- | --- | --- | --- |
| rs712 | Genotype GT vs genotype GG | Overall | 0.23 | 27% | 0.19 | 0.39 | 1.10 (0.95–1.28) |
| Chinese | 0.27 | 23% | 0.07 | NA | 1.18 (0.98–1.41) |  |  |
| Caucasian | NA | NA | 0.75 | NA | 0.96 (0.74–1.24) |  |  |
| Genotype GT/TT vs genotype GG | Overall | 0.04 | 58% | 0.10 | 0.41 | 1.21 (0.90–1.50) |  |
| Chinese | 0.14 | 43% | 0.002 | NA | 1.30 (1.10–1.55) |  |  |
| Caucasian | NA | NA | 0.59 | NA | 0.94 (0.73–1.19) |  |  |
| T vs G | Overall | 0.002 | 73% | 0.07 | 0.27 | 1.23 (0.98–1.54) |  |
| Chinese | 0.08 | 52% | 0.008 | NA | 1.33 (1.08–1.64) |  |  |
| Caucasian | NA | NA | 0.45 | NA | 0.94 (0.80–1.11) |  |  |
| rs61764370 | Genotype GT/GG vs genotype TT | Overall | 0.03 | 37% | 0.20 | 0.32 | 1.06 (0.97–1.15) |
| Ovarian cancer | 0.008 | 48% | 0.28 | NA | 1.06 (0.95–1.19) |  |  |
| Breast cancer | 0.97 | 0% | 0.95 | NA | 0.99 (0.83–1.19) |  |  |
| Colorectal cancer | 0.50 | 0% | 0.42 | NA | 1.13 (0.83–1.54) |  |  |
| Non-small-cell lung cancer | 0.05 | 73% | 0.73 | NA | 0.93 (0.60–1.43) |  |  |
| rs61764370a | Genotype GT/GG vs genotype TT | Overall | 0.003 | 78% | 0.79 | NA | 1.18 (0.34–4.17) |

Table 3 Caption: Meta-analysis results of rs712, rs61764370, and cancer risks as well as response of anti-EGFR therapy in metastatic colorectal cancer patients

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/956e/4154892/5402f504f336/ott-7-1487Fig2.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4154892_ott-7-1487Fig2.jpg)

Results of meta-analysis of rs712 and rs61764370 polymorphism loci and cancer risk. Notes: (A) T versus G of rs712. (B) Genotype GT versus genotype GG of rs712. (C) Genotype GT/TT versus genotype GG of rs712. (D) Genotype GT/GG versus genotype TT of rs61764370. Abbreviations: CI, confidence interval; M–H, Mantel–Haenszel; BEL, Belgium Ovarian Cancer Study; BWH, Brigham Women’s Hospital Study; DOV:, Diseases of the Ovary and their Evaluation Study; GER, German Ovarian Cancer Study; HJO, Hannover–Jena Ovarian Cancer Study; HMO, Hannover–Minsk Ovarian Cancer Study; HOC, Helsinki Ovarian Cancer Study; HOP, Hormones and Ovarian Cancer Prediction Study; MAY, Mayo Clinic Ovarian Cancer Study; NCO, North Carolina Ovarian Cancer Study; NTH, Nijmegen Ovarian Cancer Study; OVA, Ovarian Cancer Study; PVM, Pelvic Mass Study and Malignant Ovarian Cancer Study; TBO, Tampa Bay Ovarian Cancer Study; TOR, Familial Ovarian Tumour Study; UCI, UC Irvine Ovarian Cancer Study; UK2, SEARCH, Southampton Ovarian Cancer Study, Scottish Randomized Trial in Ovarian Cancer, United Kingdom Ovarian Cancer Population Study; USC; Los Angeles County Case–Control Studies of Ovarian Cancer; UK-GWAS, SEARCH, United Kingdom Ovarian Cancer Population Study, Cancer Research UK Familial Ovarian Cancer Register, Royal Marsden Hospital Study, UK 1958 Birth cohort, UK Colorectal control.

### rs61764370 and cancer risk

Because of the low frequency of genotype GG in rs61764370, the majority of studies did not provide data for genotype GG, but, combining GG and GT, one single comparison (GT/GG versus TT) was evaluated in this locus. The comparison was analyzed in 29 studies, which included 14,796 cases and 147,985 controls. As shown in [Table 3](#t3-ott-7-1487) and [Figure 2](#f2-ott-7-1487), the GT/GG genotype of rs61764370 was not significantly associated with cancer risk in the overall population (*P*_H_=0.03, OR =1.06, 95% CI: 0.97–1.15). After stratification analyses in accordance with cancer type, the GT/GG genotype was not observed to be associated with ovarian cancer (*P*_H_=0.008, OR =1.06, 95% CI: 0.95–1.19), breast cancer (*P*_H_=0.97, OR =0.99, 95% CI: 0.83–1.19), colorectal cancer (*P*_H_=0.50, OR =1.13, 95% CI: 0.83–1.54), or non-small-cell lung cancer (*P*_H_=0.05, OR =0.93, 95% CI: 0.60–1.43).

### rs61764370 and response of anti-EGFR treatment in mCRC

The association of rs61764370 and influence of anti-EGFR treatment in mCRC patients were estimated in combining with four original studies. Result in overall population showed that no statistically significant association was found between GT/GG genotype and response of mCRC treated with anti-EGFR (*P*_H_=0.003, OR =1.18, 95% CI =0.34–4.71) ([Figure 3](#f3-ott-7-1487)).

### Figure 3.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/956e/4154892/50b18a460bf4/ott-7-1487Fig3.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4154892_ott-7-1487Fig3.jpg)

Begg’s funnel plots of rs712, rs61764370, and cancer risk. Notes: (A) T versus G of rs712. (B) Genotype GT versus genotype GG of rs712. (C) Genotype GT/TT versus genotype GG of rs712. (D) Genotype GT/GG versus genotype TT of rs61764370. Abbreviations: Log, logarithm; OR, odds ratio; SE, standard error.

### Sensitivity analysis

The stability of this meta-analysis was examined to establish the influence of each eligible study on the pooled ORs by omitting a single study successively each time, and the corresponding pooled ORs were not materially changed in any comparison.

### Publication bias

Possible publication bias was assessed using Begg’s funnel plot and Egger’s test. As shown in [Table 3](#t3-ott-7-1487) and [Figure 3](#f3-ott-7-1487), the shapes of the funnel plots were symmetrical, and the *P*-values from the Egger’s test indicated that no publication bias was found in any comparison.

## Discussion

miRNA is an endogenous small non-coding RNA of 17–24 nucleotides that negatively regulates gene expression at the posttranscriptional level, predominantly by binding to the 3′-UTR of target mRNAs through nucleotide pairing.[37](#b37-ott-7-1487) It provides a wide range of functions in various physiological and pathological processes, including organ growth and development, cell proliferation and differentiation, and carcinogenesis and metastasis.[38](#b38-ott-7-1487)^–^[41](#b41-ott-7-1487) Let-7, the first discovered miRNA family, which includes let-7a–g and i, has been verified as a tumor suppressor factor in various kinds of cancer.[12](#b12-ott-7-1487)^,^[42](#b42-ott-7-1487)^,^[43](#b43-ott-7-1487) Expression of Kras was downregulated through ten let-7 LCSs, which were found in *Kras* 3′-UTR.[30](#b30-ott-7-1487) SNPs of rs712 in LCS1 and rs61764370 in LCS6 can disrupt the let-7 binding site and decrease the combining capacity between them, contributing to aberrant Kras expression.[30](#b30-ott-7-1487) Increasing evidence shows two SNPs (rs712 and rs61764370) not only are associated with cancer, but also rs61764370 can modulate the anti-EGFR treatment response in mCRC. Meanwhile, contradictory results have been observed in other studies.[13](#b13-ott-7-1487)^,^[14](#b14-ott-7-1487)^,^[16](#b16-ott-7-1487)

In the current study, the possible associations of rs712 and rs61764370 with risk of cancer and anti-EGFR therapy efficacy in mCRC were investigated by meta-analysis. The results showed that genotypes GT and GT/TT and allele T of rs712, and genotype GT/GG of rs61764370, were not associated with cancer, revealing that appearance of genotypes GT and GT/TT and the T allele of rs712 might not increase predisposition to cancer in the overall population and that genotype GT/GG of rs61764370 was not a genetic susceptibility factor for cancer in the Caucasian population. Significant associations were observed between genotype GT/TT and allele T of rs712 and risk of cancer in Chinese populations. The findings suggest that genotype GT/TT and allele T of rs712 could increase cancer risk and might be genetic susceptibility factors for cancer, only in the Chinese population. The following possible reasons might account for our findings.

Due to differences in ethnic genetic backgrounds in Caucasian and Chinese populations, frequency of the G allele of rs61764370 in the Chinese population is less than 1%, and no study reported an association of this locus with cancer risk in the Chinese population. Although rs712 allele frequency in the Caucasian population is higher than 5%, only one eligible study[15](#b15-ott-7-1487) reported the association between rs712 and cancer risk in this population; therefore, small sample sizes of cases and controls in eligible studies may limit the power to reach a more precise result in Caucasian populations, for only one eligible study with sample size of cases and controls were less than 1000 concerning rs712 and cancer risk in Caucasian population. Moreover, on the basis of capability of let-7 regulating *Kras* expression, we deduced that the allele T of rs712 might disrupt and interfere with the combining capacity between let-7 and the 3′-URT of *Kras* mRNA and somehow lower the level of cellular let-7 concentration or reduce its activity.[30](#b30-ott-7-1487)^,^[44](#b44-ott-7-1487) Due to loss of inhibition, expression of Kras is upregulated. Consequently, lower concentration or activity of let-7 and higher Kras-expressed p21 protein are involved in promoting cell proliferation and division, leading to carcinogenesis and metastases.[45](#b45-ott-7-1487)^,^[46](#b46-ott-7-1487)

Biological target treatment is an effective measure for malignant cancer therapy. Anti-EGFR monoclonal antibodies, cetuximab and panitumumab, are extensively used in mCRC therapy until now. Both mutation and SNP of Kras gene has been reported to affect response rates of mCRC treated with anti-EGFR.[47](#b47-ott-7-1487) Combining each including study, our meta-analysis results showed no statistically significant effect of genotype GT/GG of rs61764370 on response rates of mCRC patients treated with anti-EGFR, suggesting that genotype GT/GG does not influence the anti-EGFR therapy response in mCRC, thus should not be considered a predictor of the efficacy of anti-EGFR therapy in mCRC.

The current meta-analysis is, to our knowledge, the first assessment of the relationship between Kras polymorphism and risk of cancer, as well as the first assessment of treatment of anti-EGFR in mCRC, and provides a more reliable estimation of the association between rs712, rs61764370 and cancer risk as well as response to anti-EGFR therapy in mCRC patients when compared with any single study with small samples. However, there are several limitations of the meta-analysis, which should be addressed. First, retrieval of eligible studies was only performed in PubMed, Google Scholar, Embase, and Wanfang databases in English and Chinese, which means eligible studies published in other languages may have been overlooked, which could have led to selection bias. Second, small numbers of cases (<1,000) in the majority of eligible studies decreased the statistical power. Third, the sample size of this meta-analysis is the largest of sample size in the Meta-analysis so far, but it was neither large nor comprehensive enough to allow for a precise conclusion to be reached, especially in Chinese or Caucasian population. Finally, due to unavailable data in some included studies, we could not perform a meta-analysis based on adjustments for age, diet, smoking, or other environmental factors.

## Conclusion

Genotype GT/TT and allele T of rs712 may be potential risk factors for developing cancer in the Chinese population, while GT/GG of rs61764370 neither increases predisposition to cancer in Caucasian people nor predicts clinical outcome of anti-EGFR therapy in mCRC. Given the limitations of the current study, a larger sample size and functional analysis are warranted to further validate the results.

## Acknowledgments

This study was supported by the National Natural Science Foundation of China (81172141); Nanjing Science and Technology Committee project (201108025); Nanjing Medical Technology Development Project (ZKX11025). Nanjing Health Young Talent Project; Jiangsu Provincial Key Medical Talents to SKW; and Nanjing Medical Science and Technique Development Foundation to YQP (QRX11255) and BSH (QRX11254). Dr Matthew B Scott contributed to language revision of this study.

## Footnotes

## References

1. Chen WQ, Zhang SW, Zheng RS, et al. Report of cancer incidence and mortality in China, 2009. China Cancer. 2013;22:2–12. doi: 10.5732/cjc.013.10027. Chinese.  [DOI](https://doi.org/10.5732/cjc.013.10027) | [PMC free article](/articles/PMC3845572/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23489585/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=China%20Cancer&title=Report%20of%20cancer%20incidence%20and%20mortality%20in%20China,%202009&author=WQ%20Chen&author=SW%20Zhang&author=RS%20Zheng&volume=22&publication_year=2013&pages=2-12&pmid=23489585&doi=10.5732/cjc.013.10027&)

2. Zhang SW, Chen WQ, Lei ZL, Zou XN, Zhao P. A report of cancer incidence from 37 cancer registries in China, 2004. China Cancer. 2008;17:909–912. Chinese.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=China%20Cancer&title=A%20report%20of%20cancer%20incidence%20from%2037%20cancer%20registries%20in%20China,%202004&author=SW%20Zhang&author=WQ%20Chen&author=ZL%20Lei&author=XN%20Zou&author=P%20Zhao&volume=17&publication_year=2008&pages=909-912&)

3. Chen WQ, Zhang SW, Kong LZ, Lei ZL, Zhao P. Cancer mortality report of 34 cancer registries in China, 2004. China Cancer. 2008;17:913–916. Chinese.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=China%20Cancer&title=Cancer%20mortality%20report%20of%2034%20cancer%20registries%20in%20China,%202004&author=WQ%20Chen&author=SW%20Zhang&author=LZ%20Kong&author=ZL%20Lei&author=P%20Zhao&volume=17&publication_year=2008&pages=913-916&)

4. Siegel R, Naishadham D, Jemal A. Cancer statistics, 2013. CA Cancer J Clin. 2013;63:11–30. doi: 10.3322/caac.21166.  [DOI](https://doi.org/10.3322/caac.21166) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23335087/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=CA%20Cancer%20J%20Clin&title=Cancer%20statistics,%202013&author=R%20Siegel&author=D%20Naishadham&author=A%20Jemal&volume=63&publication_year=2013&pages=11-30&pmid=23335087&doi=10.3322/caac.21166&)

5. Garraway LA, Lander ES. Lessons from the cancer genome. Cell. 2013;153:17–37. doi: 10.1016/j.cell.2013.03.002.  [DOI](https://doi.org/10.1016/j.cell.2013.03.002) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23540688/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell&title=Lessons%20from%20the%20cancer%20genome&author=LA%20Garraway&author=ES%20Lander&volume=153&publication_year=2013&pages=17-37&pmid=23540688&doi=10.1016/j.cell.2013.03.002&)

6. Kranenburg O. The KRAS oncogene: past, present, and future. Biochim Biophys Acta. 2005;1756:81–82. doi: 10.1016/j.bbcan.2005.10.001.  [DOI](https://doi.org/10.1016/j.bbcan.2005.10.001) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16269215/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochim%20Biophys%20Acta&title=The%20KRAS%20oncogene:%20past,%20present,%20and%20future&author=O%20Kranenburg&volume=1756&publication_year=2005&pages=81-82&pmid=16269215&doi=10.1016/j.bbcan.2005.10.001&)

7. Kent OA, Fox-Talbot K, Halushka MK. RREB1 repressed miR-143/145 modulates KRAS signaling through downregulation of multiple targets. Oncogene. 2013;32:2576–2585. doi: 10.1038/onc.2012.266.  [DOI](https://doi.org/10.1038/onc.2012.266) | [PMC free article](/articles/PMC8177721/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22751122/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncogene&title=RREB1%20repressed%20miR-143/145%20modulates%20KRAS%20signaling%20through%20downregulation%20of%20multiple%20targets&author=OA%20Kent&author=K%20Fox-Talbot&author=MK%20Halushka&volume=32&publication_year=2013&pages=2576-2585&pmid=22751122&doi=10.1038/onc.2012.266&)

8. Deng M, Tang H, Zhou Y, et al. miR-216b suppresses tumor growth and invasion by targeting KRAS in nasopharyngeal carcinoma. J Cell Sci. 2011;124:2997–3005. doi: 10.1242/jcs.085050.  [DOI](https://doi.org/10.1242/jcs.085050) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21878506/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Cell%20Sci&title=miR-216b%20suppresses%20tumor%20growth%20and%20invasion%20by%20targeting%20KRAS%20in%20nasopharyngeal%20carcinoma&author=M%20Deng&author=H%20Tang&author=Y%20Zhou&volume=124&publication_year=2011&pages=2997-3005&pmid=21878506&doi=10.1242/jcs.085050&)

9. Rodenhuis S, van de Wetering ML, Mooi WJ, Evers SG, van Zandwijk N, Bos JL. Mutational activation of the K-ras oncogene. A possible pathogenetic factor in adenocarcinoma of the lung. N Engl J Med. 1987;317:929–935. doi: 10.1056/NEJM198710083171504.  [DOI](https://doi.org/10.1056/NEJM198710083171504) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/3041218/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Mutational%20activation%20of%20the%20K-ras%20oncogene.%20A%20possible%20pathogenetic%20factor%20in%20adenocarcinoma%20of%20the%20lung&author=S%20Rodenhuis&author=ML%20van%20de%20Wetering&author=WJ%20Mooi&author=SG%20Evers&author=N%20van%20Zandwijk&volume=317&publication_year=1987&pages=929-935&pmid=3041218&doi=10.1056/NEJM198710083171504&)

10. Brink M, de Goeij AF, Weijenberg MP, et al. K-ras oncogene mutations in sporadic colorectal cancer in The Netherlands Cohort Study. Carcinogenesis. 2003;24:703–710. doi: 10.1093/carcin/bgg009.  [DOI](https://doi.org/10.1093/carcin/bgg009) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12727799/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Carcinogenesis&title=K-ras%20oncogene%20mutations%20in%20sporadic%20colorectal%20cancer%20in%20The%20Netherlands%20Cohort%20Study&author=M%20Brink&author=AF%20de%20Goeij&author=MP%20Weijenberg&volume=24&publication_year=2003&pages=703-710&pmid=12727799&doi=10.1093/carcin/bgg009&)

11. Dassow H, Aigner A. MicroRNAs (miRNAs) in colorectal cancer: from aberrant expression towards therapy. Curr Pharm Des. 2013;19:1242–1252. doi: 10.2174/138161213804805739.  [DOI](https://doi.org/10.2174/138161213804805739) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23570052/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Pharm%20Des&title=MicroRNAs%20(miRNAs)%20in%20colorectal%20cancer:%20from%20aberrant%20expression%20towards%20therapy&author=H%20Dassow&author=A%20Aigner&volume=19&publication_year=2013&pages=1242-1252&pmid=23570052&doi=10.2174/138161213804805739&)

12. Johnson SM, Grosshans H, Shingara J, et al. RAS is regulated by the let-7 microRNA family. Cell. 2005;120:635–647. doi: 10.1016/j.cell.2005.01.014.  [DOI](https://doi.org/10.1016/j.cell.2005.01.014) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15766527/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell&title=RAS%20is%20regulated%20by%20the%20let-7%20microRNA%20family&author=SM%20Johnson&author=H%20Grosshans&author=J%20Shingara&volume=120&publication_year=2005&pages=635-647&pmid=15766527&doi=10.1016/j.cell.2005.01.014&)

13. Pan XM, Sun RF, Li ZH, et al. A let-7 KRAS rs712 polymorphism increases colorectal cancer risk. Tumour Biol. 2014;35:831–835. doi: 10.1007/s13277-013-1114-3.  [DOI](https://doi.org/10.1007/s13277-013-1114-3) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23975373/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Tumour%20Biol&title=A%20let-7%20KRAS%20rs712%20polymorphism%20increases%20colorectal%20cancer%20risk&author=XM%20Pan&author=RF%20Sun&author=ZH%20Li&volume=35&publication_year=2014&pages=831-835&pmid=23975373&doi=10.1007/s13277-013-1114-3&)

14. Zhang W, Winder T, Ning Y, et al. A let-7 microRNA-binding site polymorphism in 3′-untranslated region of KRAS gene predicts response in wild-type KRAS patients with metastatic colorectal cancer treated with cetuximab monotherapy. Ann Oncol. 2011;22:104–109. doi: 10.1093/annonc/mdq315.  [DOI](https://doi.org/10.1093/annonc/mdq315) | [PMC free article](/articles/PMC8890483/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20603437/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Oncol&title=A%20let-7%20microRNA-binding%20site%20polymorphism%20in%203%E2%80%B2-untranslated%20region%20of%20KRAS%20gene%20predicts%20response%20in%20wild-type%20KRAS%20patients%20with%20metastatic%20colorectal%20cancer%20treated%20with%20cetuximab%20monotherapy&author=W%20Zhang&author=T%20Winder&author=Y%20Ning&volume=22&publication_year=2011&pages=104-109&pmid=20603437&doi=10.1093/annonc/mdq315&)

15. Landi D, Gemignani F, Landi S. Role of variations within microRNA-binding sites in cancer. Mutagenesis. 2012;27:205–210. doi: 10.1093/mutage/ger055.  [DOI](https://doi.org/10.1093/mutage/ger055) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22294768/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mutagenesis&title=Role%20of%20variations%20within%20microRNA-binding%20sites%20in%20cancer&author=D%20Landi&author=F%20Gemignani&author=S%20Landi&volume=27&publication_year=2012&pages=205-210&pmid=22294768&doi=10.1093/mutage/ger055&)

16. Sebio A, Paré L, Páez D, et al. The LCS6 polymorphism in the binding site of let-7 microRNA to the KRAS 3′-untranslated region: its role in the efficacy of anti-EGFR-based therapy in metastatic colorectal cancer patients. Pharmacogenet Genomics. 2013;23:142–147. doi: 10.1097/FPC.0b013e32835d9b0b.  [DOI](https://doi.org/10.1097/FPC.0b013e32835d9b0b) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23324806/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=The%20LCS6%20polymorphism%20in%20the%20binding%20site%20of%20let-7%20microRNA%20to%20the%20KRAS%203%E2%80%B2-untranslated%20region:%20its%20role%20in%20the%20efficacy%20of%20anti-EGFR-based%20therapy%20in%20metastatic%20colorectal%20cancer%20patients&author=A%20Sebio&author=L%20Par%C3%A9&author=D%20P%C3%A1ez&volume=23&publication_year=2013&pages=142-147&pmid=23324806&doi=10.1097/FPC.0b013e32835d9b0b&)

17. Stroup DF, Berlin JA, Morton SC, et al. Meta-analysis of observational studies in epidemiology: a proposal for reporting. Meta-analysis Of Observational Studies in Epidemiology (MOOSE) group. JAMA. 2000;283:2008–2012. doi: 10.1001/jama.283.15.2008.  [DOI](https://doi.org/10.1001/jama.283.15.2008) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10789670/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA&title=Meta-analysis%20of%20observational%20studies%20in%20epidemiology:%20a%20proposal%20for%20reporting.%20Meta-analysis%20Of%20Observational%20Studies%20in%20Epidemiology%20(MOOSE)%20group&author=DF%20Stroup&author=JA%20Berlin&author=SC%20Morton&volume=283&publication_year=2000&pages=2008-2012&pmid=10789670&doi=10.1001/jama.283.15.2008&)

18. Biggerstaff BJ, Jackson D. The exact distribution of Cochran’s heterogeneity statistic in one-way random effects meta-analysis. Stat Med. 2008;27:6093–6110. doi: 10.1002/sim.3428.  [DOI](https://doi.org/10.1002/sim.3428) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18781561/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Stat%20Med&title=The%20exact%20distribution%20of%20Cochran%E2%80%99s%20heterogeneity%20statistic%20in%20one-way%20random%20effects%20meta-analysis&author=BJ%20Biggerstaff&author=D%20Jackson&volume=27&publication_year=2008&pages=6093-6110&pmid=18781561&doi=10.1002/sim.3428&)

19. Higgins JP, Thompson SG. Quantifying heterogeneity in a meta-analysis. Stat Med. 2002;21:1539–1558. doi: 10.1002/sim.1186.  [DOI](https://doi.org/10.1002/sim.1186) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12111919/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Stat%20Med&title=Quantifying%20heterogeneity%20in%20a%20meta-analysis&author=JP%20Higgins&author=SG%20Thompson&volume=21&publication_year=2002&pages=1539-1558&pmid=12111919&doi=10.1002/sim.1186&)

20. Hedges LV, Vevea JL. Fixed- and random-effects models in meta-analysis. Psychol Methods. 1998;3:486–504.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Psychol%20Methods&title=Fixed-%20and%20random-effects%20models%20in%20meta-analysis&author=LV%20Hedges&author=JL%20Vevea&volume=3&publication_year=1998&pages=486-504&)

21. Egger M, Davey Smith G, Schneider M, Minder C. Bias in meta-analysis detected by a simple, graphical test. BMJ. 1997;315:629–634. doi: 10.1136/bmj.315.7109.629.  [DOI](https://doi.org/10.1136/bmj.315.7109.629) | [PMC free article](/articles/PMC2127453/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9310563/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMJ&title=Bias%20in%20meta-analysis%20detected%20by%20a%20simple,%20graphical%20test&author=M%20Egger&author=G%20Davey%20Smith&author=M%20Schneider&author=C%20Minder&volume=315&publication_year=1997&pages=629-634&pmid=9310563&doi=10.1136/bmj.315.7109.629&)

22. Begg CB, Mazumdar M. Operating characteristics of a rank correlation test for publication bias. Biometrics. 1994;50:1088–1101.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/7786990/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biometrics&title=Operating%20characteristics%20of%20a%20rank%20correlation%20test%20for%20publication%20bias&author=CB%20Begg&author=M%20Mazumdar&volume=50&publication_year=1994&pages=1088-1101&pmid=7786990&)

23. Li ZH, Pan XM, Han BW, et al. A let-7 binding site polymorphism rs712 in the KRAS 3′ UTR is associated with an increased risk of gastric cancer. Tumour Biol. 2013;34:3159–3163. doi: 10.1007/s13277-013-0885-x.  [DOI](https://doi.org/10.1007/s13277-013-0885-x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23729275/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Tumour%20Biol&title=A%20let-7%20binding%20site%20polymorphism%20rs712%20in%20the%20KRAS%203%E2%80%B2%20UTR%20is%20associated%20with%20an%20increased%20risk%20of%20gastric%20cancer&author=ZH%20Li&author=XM%20Pan&author=BW%20Han&volume=34&publication_year=2013&pages=3159-3163&pmid=23729275&doi=10.1007/s13277-013-0885-x&)

24. Yan L, Wang QW, Tian KL. Association between the single nucleotide polymorphism of let-7 target gene KRAS-binding site rs712 and risk of glioma. Chin J Cancer Prev Treat. 2013;20:811–814. Chinese.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Chin%20J%20Cancer%20Prev%20Treat&title=Association%20between%20the%20single%20nucleotide%20polymorphism%20of%20let-7%20target%20gene%20KRAS-binding%20site%20rs712%20and%20risk%20of%20glioma&author=L%20Yan&author=QW%20Wang&author=KL%20Tian&volume=20&publication_year=2013&pages=811-814&)

25. Pan XM, Jia J, Guo XM, et al. Lack of association between let-7 binding site polymorphism rs712 and risk of nasopharyngeal carcinoma. Fam Cancer. 2014;13:93–97. doi: 10.1007/s10689-013-9681-4.  [DOI](https://doi.org/10.1007/s10689-013-9681-4) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23996697/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Fam%20Cancer&title=Lack%20of%20association%20between%20let-7%20binding%20site%20polymorphism%20rs712%20and%20risk%20of%20nasopharyngeal%20carcinoma&author=XM%20Pan&author=J%20Jia&author=XM%20Guo&volume=13&publication_year=2014&pages=93-97&pmid=23996697&doi=10.1007/s10689-013-9681-4&)

26. Peng XB, Zhao J, Lei Z, Liu RY, Liu ZY, Jiang XF, Zhang HT. Association of a SNP in microRNA let-7 complementary region in KRAS 3′UTR with non-small cell lung cancer. Soochow Univ J Med Sci. 2010;4:786–790. Chinese.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Soochow%20Univ%20J%20Med%20Sci&title=Association%20of%20a%20SNP%20in%20microRNA%20let-7%20complementary%20region%20in%20KRAS%203%E2%80%B2UTR%20with%20non-small%20cell%20lung%20cancer&author=XB%20Peng&author=J%20Zhao&author=Z%20Lei&author=RY%20Liu&author=ZY%20Liu&volume=4&publication_year=2010&pages=786-790&)

27. Hollestelle A, Pelletier C, Hooning M, et al. Prevalence of the variant allele rs61764370 T>G in the 3′UTR of KRAS among Dutch BRCA1, BRCA2 and non-BRCA1/BRCA2 breast cancer families. Breast Cancer Res Treat. 2011;128:79–84. doi: 10.1007/s10549-010-1080-z.  [DOI](https://doi.org/10.1007/s10549-010-1080-z) | [PMC free article](/articles/PMC3735357/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20676756/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Breast%20Cancer%20Res%20Treat&title=Prevalence%20of%20the%20variant%20allele%20rs61764370%20T>G%20in%20the%203%E2%80%B2UTR%20of%20KRAS%20among%20Dutch%20BRCA1,%20BRCA2%20and%20non-BRCA1/BRCA2%20breast%20cancer%20families&author=A%20Hollestelle&author=C%20Pelletier&author=M%20Hooning&volume=128&publication_year=2011&pages=79-84&pmid=20676756&doi=10.1007/s10549-010-1080-z&)

28. Pharoah PD, Palmieri RT, Ramus SJ, et al. The role of KRAS rs61764370 in invasive epithelial ovarian cancer: implications for clinical testing. Clin Cancer Res. 2011;17:3742–3750. doi: 10.1158/1078-0432.CCR-10-3405.  [DOI](https://doi.org/10.1158/1078-0432.CCR-10-3405) | [PMC free article](/articles/PMC3107901/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21385923/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&title=The%20role%20of%20KRAS%20rs61764370%20in%20invasive%20epithelial%20ovarian%20cancer:%20implications%20for%20clinical%20testing&author=PD%20Pharoah&author=RT%20Palmieri&author=SJ%20Ramus&volume=17&publication_year=2011&pages=3742-3750&pmid=21385923&doi=10.1158/1078-0432.CCR-10-3405&)

29. Cerne JZ, Stegel V, Gersak K, Novakovic S. KRAS rs61764370 is associated with HER2-overexpressed and poorly-differentiated breast cancer in hormone replacement therapy users: a case control study. BMC Cancer. 2012;12:105. doi: 10.1186/1471-2407-12-105.  [DOI](https://doi.org/10.1186/1471-2407-12-105) | [PMC free article](/articles/PMC3342891/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22436609/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Cancer&title=KRAS%20rs61764370%20is%20associated%20with%20HER2-overexpressed%20and%20poorly-differentiated%20breast%20cancer%20in%20hormone%20replacement%20therapy%20users:%20a%20case%20control%20study&author=JZ%20Cerne&author=V%20Stegel&author=K%20Gersak&author=S%20Novakovic&volume=12&publication_year=2012&pages=105&pmid=22436609&doi=10.1186/1471-2407-12-105&)

30. Chin LJ, Ratner E, Leng S, et al. A SNP in a let-7 microRNA complementary site in the KRAS 3′ untranslated region increases non-small cell lung cancer risk. Cancer Res. 2008;68:8535–8540. doi: 10.1158/0008-5472.CAN-08-2129.  [DOI](https://doi.org/10.1158/0008-5472.CAN-08-2129) | [PMC free article](/articles/PMC2672193/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18922928/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=A%20SNP%20in%20a%20let-7%20microRNA%20complementary%20site%20in%20the%20KRAS%203%E2%80%B2%20untranslated%20region%20increases%20non-small%20cell%20lung%20cancer%20risk&author=LJ%20Chin&author=E%20Ratner&author=S%20Leng&volume=68&publication_year=2008&pages=8535-8540&pmid=18922928&doi=10.1158/0008-5472.CAN-08-2129&)

31. Paranjape T, Heneghan H, Lindner R, et al. A 3′-untranslated region KRAS variant and triple-negative breast cancer: a case-control and genetic analysis. Lancet Oncol. 2011;12:377–386. doi: 10.1016/S1470-2045(11)70044-4.  [DOI](https://doi.org/10.1016/S1470-2045(11)70044-4) | [PMC free article](/articles/PMC3488438/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21435948/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet%20Oncol&title=A%203%E2%80%B2-untranslated%20region%20KRAS%20variant%20and%20triple-negative%20breast%20cancer:%20a%20case-control%20and%20genetic%20analysis&author=T%20Paranjape&author=H%20Heneghan&author=R%20Lindner&volume=12&publication_year=2011&pages=377-386&pmid=21435948&doi=10.1016/S1470-2045(11)70044-4&)

32. Ratner E, Lu L, Boeke M, et al. A KRAS-variant in ovarian cancer acts as a genetic marker of cancer risk. Cancer Res. 2010;70:6509–6515. doi: 10.1158/0008-5472.CAN-10-0689.  [DOI](https://doi.org/10.1158/0008-5472.CAN-10-0689) | [PMC free article](/articles/PMC2923587/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20647319/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=A%20KRAS-variant%20in%20ovarian%20cancer%20acts%20as%20a%20genetic%20marker%20of%20cancer%20risk&author=E%20Ratner&author=L%20Lu&author=M%20Boeke&volume=70&publication_year=2010&pages=6509-6515&pmid=20647319&doi=10.1158/0008-5472.CAN-10-0689&)

33. Christensen BC, Moyer BJ, Avissar M, et al. A let-7 microRNA-binding site polymorphism in the KRAS 3′ UTR is associated with reduced survival in oral cancers. Carcinogenesis. 2009;30:1003–1007. doi: 10.1093/carcin/bgp099.  [DOI](https://doi.org/10.1093/carcin/bgp099) | [PMC free article](/articles/PMC2691138/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19380522/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Carcinogenesis&title=A%20let-7%20microRNA-binding%20site%20polymorphism%20in%20the%20KRAS%203%E2%80%B2%20UTR%20is%20associated%20with%20reduced%20survival%20in%20oral%20cancers&author=BC%20Christensen&author=BJ%20Moyer&author=M%20Avissar&volume=30&publication_year=2009&pages=1003-1007&pmid=19380522&doi=10.1093/carcin/bgp099&)

34. Ryan BM, Robles AI, Harris CC. KRAS-LCS6 genotype as a prognostic marker in early-stage CRC – letter. Clin Cancer Res. 2012;18:3487–3488. doi: 10.1158/1078-0432.CCR-12-0250. author reply 3489.  [DOI](https://doi.org/10.1158/1078-0432.CCR-12-0250) | [PMC free article](/articles/PMC4132878/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22669132/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&title=KRAS-LCS6%20genotype%20as%20a%20prognostic%20marker%20in%20early-stage%20CRC%20%E2%80%93%20letter&author=BM%20Ryan&author=AI%20Robles&author=CC%20Harris&volume=18&publication_year=2012&pages=3487-3488&pmid=22669132&doi=10.1158/1078-0432.CCR-12-0250&)

35. Kjersem JB, Ikdahl T, Guren T, et al. Let-7 miRNA-binding site polymorphism in the KRAS 3′UTR; colorectal cancer screening population prevalence and influence on clinical outcome in patients with metastatic colorectal cancer treated with 5-fluorouracil and oxaliplatin ± cetuximab. BMC Cancer. 2012;12:534. doi: 10.1186/1471-2407-12-534.  [DOI](https://doi.org/10.1186/1471-2407-12-534) | [PMC free article](/articles/PMC3526507/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23167843/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Cancer&title=Let-7%20miRNA-binding%20site%20polymorphism%20in%20the%20KRAS%203%E2%80%B2UTR;%20colorectal%20cancer%20screening%20population%20prevalence%20and%20influence%20on%20clinical%20outcome%20in%20patients%20with%20metastatic%20colorectal%20cancer%20treated%20with%205-fluorouracil%20and%20oxaliplatin%20%C2%B1%20cetuximab&author=JB%20Kjersem&author=T%20Ikdahl&author=T%20Guren&volume=12&publication_year=2012&pages=534&pmid=23167843&doi=10.1186/1471-2407-12-534&)

36. Graziano F, Canestrari E, Loupakis F, et al. Genetic modulation of the Let-7 microRNA binding to KRAS 3′-untranslated region and survival of metastatic colorectal cancer patients treated with salvage cetuximab-irinotecan. Pharmacogenomics J. 2010;10:458–464. doi: 10.1038/tpj.2010.9.  [DOI](https://doi.org/10.1038/tpj.2010.9) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20177422/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=Genetic%20modulation%20of%20the%20Let-7%20microRNA%20binding%20to%20KRAS%203%E2%80%B2-untranslated%20region%20and%20survival%20of%20metastatic%20colorectal%20cancer%20patients%20treated%20with%20salvage%20cetuximab-irinotecan&author=F%20Graziano&author=E%20Canestrari&author=F%20Loupakis&volume=10&publication_year=2010&pages=458-464&pmid=20177422&doi=10.1038/tpj.2010.9&)

37. Bartel DP. MicroRNAs: genomics, biogenesis, mechanism, and function. Cell. 2004;116:281–297. doi: 10.1016/s0092-8674(04)00045-5.  [DOI](https://doi.org/10.1016/s0092-8674(04)00045-5) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14744438/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell&title=MicroRNAs:%20genomics,%20biogenesis,%20mechanism,%20and%20function&author=DP%20Bartel&volume=116&publication_year=2004&pages=281-297&pmid=14744438&doi=10.1016/s0092-8674(04)00045-5&)

38. Alvarez-Garcia I, Miska EA. MicroRNA functions in animal development and human disease. Development. 2005;132:4653–4662. doi: 10.1242/dev.02073.  [DOI](https://doi.org/10.1242/dev.02073) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16224045/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Development&title=MicroRNA%20functions%20in%20animal%20development%20and%20human%20disease&author=I%20Alvarez-Garcia&author=EA%20Miska&volume=132&publication_year=2005&pages=4653-4662&pmid=16224045&doi=10.1242/dev.02073&)

39. Zhao C, Sun G, Li S, et al. MicroRNA let-7b regulates neural stem cell proliferation and differentiation by targeting nuclear receptor TLX signaling. Proc Natl Acad Sci U S A. 2010;107:1876–1881. doi: 10.1073/pnas.0908750107.  [DOI](https://doi.org/10.1073/pnas.0908750107) | [PMC free article](/articles/PMC2836616/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20133835/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&title=MicroRNA%20let-7b%20regulates%20neural%20stem%20cell%20proliferation%20and%20differentiation%20by%20targeting%20nuclear%20receptor%20TLX%20signaling&author=C%20Zhao&author=G%20Sun&author=S%20Li&volume=107&publication_year=2010&pages=1876-1881&pmid=20133835&doi=10.1073/pnas.0908750107&)

40. Mirnezami AH, Pickard K, Zhang L, Primrose JN, Packham G. MicroRNAs: key players in carcinogenesis and novel therapeutic targets. Eur J Surg Oncol. 2009;35:339–347. doi: 10.1016/j.ejso.2008.06.006.  [DOI](https://doi.org/10.1016/j.ejso.2008.06.006) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18644693/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Surg%20Oncol&title=MicroRNAs:%20key%20players%20in%20carcinogenesis%20and%20novel%20therapeutic%20targets&author=AH%20Mirnezami&author=K%20Pickard&author=L%20Zhang&author=JN%20Primrose&author=G%20Packham&volume=35&publication_year=2009&pages=339-347&pmid=18644693&doi=10.1016/j.ejso.2008.06.006&)

41. Asangani IA, Rasheed SA, Nikolova DA, et al. MicroRNA-21 (miR-21) post-transcriptionally downregulates tumor suppressor Pdcd4 and stimulates invasion, intravasation and metastasis in colorectal cancer. Oncogene. 2008;27:2128–2136. doi: 10.1038/sj.onc.1210856.  [DOI](https://doi.org/10.1038/sj.onc.1210856) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17968323/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncogene&title=MicroRNA-21%20(miR-21)%20post-transcriptionally%20downregulates%20tumor%20suppressor%20Pdcd4%20and%20stimulates%20invasion,%20intravasation%20and%20metastasis%20in%20colorectal%20cancer&author=IA%20Asangani&author=SA%20Rasheed&author=DA%20Nikolova&volume=27&publication_year=2008&pages=2128-2136&pmid=17968323&doi=10.1038/sj.onc.1210856&)

42. Kumar MS, Erkeland SJ, Pester RE, et al. Suppression of non-small cell lung tumor development by the let-7 microRNA family. Proc Natl Acad Sci U S A. 2008;105:3903–3908. doi: 10.1073/pnas.0712321105.  [DOI](https://doi.org/10.1073/pnas.0712321105) | [PMC free article](/articles/PMC2268826/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18308936/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&title=Suppression%20of%20non-small%20cell%20lung%20tumor%20development%20by%20the%20let-7%20microRNA%20family&author=MS%20Kumar&author=SJ%20Erkeland&author=RE%20Pester&volume=105&publication_year=2008&pages=3903-3908&pmid=18308936&doi=10.1073/pnas.0712321105&)

43. Park SM, Shell S, Radjabi AR, et al. Let-7 prevents early cancer progression by suppressing expression of the embryonic gene HMGA2. Cell Cycle. 2007;6:2585–2590. doi: 10.4161/cc.6.21.4845.  [DOI](https://doi.org/10.4161/cc.6.21.4845) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17957144/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell%20Cycle&title=Let-7%20prevents%20early%20cancer%20progression%20by%20suppressing%20expression%20of%20the%20embryonic%20gene%20HMGA2&author=SM%20Park&author=S%20Shell&author=AR%20Radjabi&volume=6&publication_year=2007&pages=2585-2590&pmid=17957144&doi=10.4161/cc.6.21.4845&)

44. Landi D, Gemignani F, Naccarati A, et al. Polymorphisms within micro-RNA-binding sites and risk of sporadic colorectal cancer. Carcinogenesis. 2008;29:579–584. doi: 10.1093/carcin/bgm304.  [DOI](https://doi.org/10.1093/carcin/bgm304) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18192692/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Carcinogenesis&title=Polymorphisms%20within%20micro-RNA-binding%20sites%20and%20risk%20of%20sporadic%20colorectal%20cancer&author=D%20Landi&author=F%20Gemignani&author=A%20Naccarati&volume=29&publication_year=2008&pages=579-584&pmid=18192692&doi=10.1093/carcin/bgm304&)

45. Ruzzo A, Graziano F, Vincenzi B, et al. High let-7a microRNA levels in KRAS-mutated colorectal carcinomas may rescue anti-EGFR therapy effects in patients with chemotherapy-refractory metastatic disease. Oncologist. 2012;17:823–829. doi: 10.1634/theoncologist.2012-0081.  [DOI](https://doi.org/10.1634/theoncologist.2012-0081) | [PMC free article](/articles/PMC3380881/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22584434/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncologist&title=High%20let-7a%20microRNA%20levels%20in%20KRAS-mutated%20colorectal%20carcinomas%20may%20rescue%20anti-EGFR%20therapy%20effects%20in%20patients%20with%20chemotherapy-refractory%20metastatic%20disease&author=A%20Ruzzo&author=F%20Graziano&author=B%20Vincenzi&volume=17&publication_year=2012&pages=823-829&pmid=22584434&doi=10.1634/theoncologist.2012-0081&)

46. Scharovsky OG, Rozados VR, Gervasoni SI, Matar P. Inhibition of ras oncogene: a novel approach to antineoplastic therapy. J Biomed Sci. 2000;7:292–298. doi: 10.1007/BF02253247.  [DOI](https://doi.org/10.1007/BF02253247) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10895051/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biomed%20Sci&title=Inhibition%20of%20ras%20oncogene:%20a%20novel%20approach%20to%20antineoplastic%20therapy&author=OG%20Scharovsky&author=VR%20Rozados&author=SI%20Gervasoni&author=P%20Matar&volume=7&publication_year=2000&pages=292-298&pmid=10895051&doi=10.1007/BF02253247&)

47. Di Fiore F, Blanchard F, Charbonnier F, et al. Clinical relevance of KRAS mutation detection in metastatic colorectal cancer treated by Cetuximab plus chemotherapy. Br J Cancer. 2007;96:1166–1169. doi: 10.1038/sj.bjc.6603685.  [DOI](https://doi.org/10.1038/sj.bjc.6603685) | [PMC free article](/articles/PMC2360149/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17375050/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Cancer&title=Clinical%20relevance%20of%20KRAS%20mutation%20detection%20in%20metastatic%20colorectal%20cancer%20treated%20by%20Cetuximab%20plus%20chemotherapy&author=F%20Di%20Fiore&author=F%20Blanchard&author=F%20Charbonnier&volume=96&publication_year=2007&pages=1166-1169&pmid=17375050&doi=10.1038/sj.bjc.6603685&)
