# Impact of polymorphisms in drug pathway genes on disease-free survival in adults with acute myeloid leukemia

## Metadata
**Authors:** Sook Wah Yee, Joel A Mefford, Natasha Singh, Mary-Elizabeth Percival, Adrian Stecula, Kuo Yang, John S Witte, Atsushi Takahashi, Michiaki Kubo, Koichi Matsuda, Kathleen M Giacomini, Charalambos Andreadis
**Journal:** Journal of human genetics
**Date:** 2013 May 16
**DOI:** [10.1038/jhg.2013.38](https://doi.org/10.1038/jhg.2013.38)
**PMID:** 23677058
**PMCID:** PMC4068832
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4068832/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC4068832/pdf/nihms467280.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC4068832/pdf/nihms467280.pdf)

## Abstract

Acute myeloid leukemia (AML) is a clinically heterogeneous disease, with 5-year disease-free survival (DFS) ranging from under 10% to over 70% for distinct groups of patients. At our institution, cytarabine, etoposide and busulfan are used in first or second remission patients treated with a 2-step approach to autologous stem cell transplantation (ASCT). In this study, we tested the hypothesis that polymorphisms in the pharmacokinetic and pharmacodynamic pathway genes of these drugs are associated with DFS in AML patients. A total of 1659 variants in 42 genes were analyzed for their association with DFS using a Cox proportional hazards model. 154 genetically European patients were used for the primary analysis. An intronic SNP in ABCC3 (rs4148405) was associated with a significantly shorter DFS (HR=3.2, p=5.6 x 10(-6)) in our primary cohort. In addition a SNP in the GSTM1-GSTM5 locus, rs3754446, was significantly associated with a shorter DFS in all patients (HR=1.8, p=0.001 for 154 European ancestry; HR=1.7, p=0.028 for 125 non-European patients). Thus for the first time, genetic variants in drug pathway genes are shown to be associated with DFS in AML patients treated with chemotherapy-based autologous ASCT.

## Introduction

Adult acute myeloid leukemia (AML) is a hematologic malignancy with widely heterogeneous clinical outcomes. New treatments for AML are increasingly being tested in clinical trials of patients with specific tumor cell mutations^1,2^. Although there have been substantial improvements in the number of patients who achieve complete remission, the choice of induction and post-remission therapy for adult AML is still based on the “one size fits all” principle. Most regimens incorporate antimetabolites (e.g. cytarabine, fludarabine), topoisomerase II inhibitors (e.g. etoposide, daunorubicin, idarubicin, mitoxantrone) and alkylating agents (e.g. busulfan, cyclophosphamide) for the treatment of AML. Prognostic factors for treatment response include age, prior exposure to chemotherapy, cytogenetic markers and expression profiles, and appearance of specific genetic mutations in tumor tissue, such as mutation and translocation of particular genes (e.g. *FLT3*, *NPM1*)^3,4^. However, these prognostic factors do not adequately capture the wide diversity of clinical outcomes in this disease. The percent of adults with AML who can survive 3 or more years and may be cured is approximately 5-70%^5,6^.

One possible explanation for the difference in response to AML treatment is germline genetic variation. While the pharmacogenomics of AML drug response is an active area of research, there remain large challenges. These include the: (i) poor availability of uniform, well-collected and well-defined drug response phenotype information; (ii) lack of widely available germline DNA not contaminated with tumor cells (myeloblasts); (iii) limited availability of panels of genotype data in large patient cohorts; and (iv) inability to validate findings in replication studies. In this study, we overcome many of these challenges. In particular, to our knowledge, this is the first study that involved the analysis of large numbers of genetic polymorphisms in a cohort of AML patients treated with high-dose chemotherapy followed by autologous stem cell transplantation (ASCT). We present results from testing the association between germline variants in drug pathway and other genes with disease-free survival (DFS) in adult AML patients. We identified new associations between AML DFS and polymorphisms in several drug pathway genes for cytarabine, etoposide and busulfan, and also replicated SNPs previously reported to be associated with the AML response phenotypes.

## Materials and Methods

### Clinical Protocol, Study Criteria and Patient Cohorts

The population for the current study is AML patients who were enrolled in UCSF study protocols 9203 or 9303 between 1988 and 2010. The study protocols and patient selection criteria have been previously described^7-9^. This treatment protocol was used in patients with low and standard risk *de novo* AML, including APL (acute promyelocytic leukemia) in first or second complete remission. It was also used in a small number of patients with high risk AML (i.e. with secondary AML) if allogeneic SCT was not an option for the patient (e.g. unavailable donor)^7^. In step 1, patients were treated with consolidation chemotherapy including cytarabine 2000/mg/m^2^ (i.v.) twice daily for 4 days concurrently with etoposide 40 mg/kg by i.v. infusion over the 4 days. During the recovery period from chemotherapy, peripheral blood stem cells were collected under granulocyte colony-stimulating factor stimulation. In step 2, patients underwent ASCT, which involved the preparative regimen of busulfan (total dose 16 mg/kg orally or 12.8 mg/kg intravenously, over 16 doses in 4 days) followed by etoposide 60 mg/kg (i.v. bolus) and reinfusion of blood or marrow stem cells. Patients had to be in complete remission for at least 30 days prior to step 2 ([Figure 1](#F1)). Complete remission was defined as normal bone marrow morphology with <5% blasts, resolution of previously abnormal cytogenetics and no evidence of extramedullary leukemia. In addition, patients must meet criteria for neutrophil and platelet counts, liver and kidney function^7-9^. Detailed procedures of patient enrollment, diagnosis, data collection and follow-up have been previously described^7-9^. Briefly, patients were actively followed up in the beginning within 6 months of diagnosis, with subsequent annual followed up by clinic visits. UCSF electronic medical records, the UCSF Blood and Bone Marrow Transplant Clinic database and patients' medical charts were abstracted to determine patients' remission status. The UCSF Committee on Human Research approved the research protocol (IRB no. 10-00649).

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3a76/4068832/5f2a1c332105/nihms467280f1.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4068832_nihms467280f1.jpg)

Schematic of workflow applied to determine the association of genetic variations in 42 candidate genes with disease free survival (DFS) in AML patients treated with a two-step treatment protocol prior to autologous stem cell transplantation. The workflow begins with phenotype data collection and genotyping of DNA samples using a genomewide SNP array. Candidate genes were selected based on their roles in the: drugs' pharmacokinetic/pharmacodynamics pathway; DNA mismatch repair mechanism; association with the drug cytotoxicity in lymphoblastoic cell lines previously identified in literatures; and pharmacogenomics studies of AML drug response. After association of each SNPs with disease-free survival in the 154 AML patients of European ancestry, the SNPs with p<0.01 and their tag-SNPs were examined for their potential functional roles using databases to identify eQTL SNPs and predicted regulatory elements such as binding sites of transcription factors and for their replications of previous studies. The SNPs with p<0.01 were also examined for their associations with DFS in 125 non-European ancestry. Finally, imputation was performed to identify other SNPs with stronger associations with DFS in European ancestry.

### DNA Isolation and Genotyping

DNA was isolated from peripheral blood stem cells, which were collected during the recovery from step 1 consolidation chemotherapy. As noted in the above section, patients were in complete remission prior to consolidation chemotherapy and hence the samples utilized in this step contained less than 5% leukemic cells. DNA was isolated at the UCSF DNA Banking and Extraction Services Lab. The lab followed standard DNA extraction protocol described in the Wizard^®^ Genomic DNA Purification Kit (Promega). The DNA was then quantified using Picogreen and normalized to 50 ng/μL. For each sample, we genotyped 250 ng of DNA. The Illumina HumanOmniExpress v1.0 Beadchip was used, following the manufacturer's protocols, at the Center for Genomic Medicine, RIKEN, Yokohama, Japan. For quality control of the genotyping, we included one HapMap trio and three duplicates of the DNA samples from the AML patients. A total of 328 distinct DNA samples from patients were genotyped, along with 3 duplicates and 3 HapMap samples (trio).

### Patients' Ancestral Origin

The genetic ancestral origin of patients was determined using principal components analysis implemented in Eigenstrat^10^. Genotype information on our 328 AML patients was analyzed in conjunction with SNP data from the HapMap project, which consist of Europeans (CEU and TSI), Asians (JPT and CHB), Africans (TSI and ASW) and Mexican (MXL). From these analyses we were able to distinguish 154 patients of European ancestry based on their close clustering with the European HapMap samples.

### Pathway, Gene and SNP Selection

A total of 42 genes were selected for analysis based on the following criteria: (i) genes in the pharmacokinetic and/or pharmacodynamics pathway of the drugs administered (cytarabine, etoposide and busulfan)^11-15^; (ii) genes described in literature as having significant associations with the drug cytotoxicity in lymphoblastoid cell lines^16,17^; (iii) genes involved in DNA mismatch repair^18^; and (iv) genes found previously to be associated with the AML response phenotype^19^ (see [Supplementary Table 1, Figure 1](#SD1)). After filtering the SNPs with low call rates (< 90%) and SNPs with low minor allele frequencies (MAF) (MAF < 1%) in the 154 European ancestry patients, we selected the SNPs in the candidate genes and within 25000 bp upstream and downstream flanking regions.

### Statistical Analysis of the Associations

The primary analysis was to estimate the association between SNPs in the selected candidate genes and disease free survival (DFS) in 154 patients of European ancestry. A Cox proportional hazard model was used to estimate the hazard ratios (HR) and 95% confidence limits for the effect of genotype on DFS. An additive coding of genotypes was used in all analyses. The genetic effect estimates were adjusted for levels of a clinical risk score (see [Table 1](#T1)). The SNP associations with p<0.01 were also tested in the 125 non-European patients using a Cox proportional hazard model. In light of the heterogeneous ethnicity of the samples, the genetic effect estimates were adjusted for the first 10 principal components calculated from the GWAS data, as well as the clinical risk score. The Cox proportional hazards function from the R-project (version 2.15.1) was used. We used 3 x 10^-5^ (= 0.05 / 1659) as the significance level after Bonferroni correction for multiple testing.

### Table 1.

| Variables | Patients of European ancestry (N=154) | Patients of non-European ancestry (N=125) |
| --- | --- | --- |
| Sex, N (%) |   |   |
| Male | 76 (49.3%) | 67 (53.6%) |
| Female | 78 (50.6%) | 58 (46.4%) |
| Age (years) |   |   |
| Median (SD, range) |   |   |
| Age at Diagnosis (N=152) | 47.0 (13.0, 18 - 72) (N=152) | 40.5 (12.8, 17 - 68) (N=124) |
| Age at autologous bone marrow transplantation | 47.0 (13.0, 19 - 72) | 41.0 (12.8, 19 - 69) |
| Year of Transplantation |   |   |
| 1988 to 1995 | 36 (23.4%) | 38 (30.4%) |
| 1996 to 2000 | 41 (26.6%) | 31 (24.8%) |
| 2001 to 2010 | 77 (50.0%) | 56 (44.8%) |
| De novo/Secondary AML, N (%) |   |   |
| De novo | 147 (95.5%) | 110 (88.0%) |
| Secondary | 7 (4.5%) | 15 (12.0%) |
| Risk, N (%) |   |   |
| Acute promyeloid leukemia | 15 (9.7%) | 12 (9.6%) |
| Low | 18 (11.7%) | 18 (14.4%) |
| Standard | 114 (74.0%) | 80 (64.0%) |
| High | 7 (4.5%) | 15 (12.0%) |
| Disease Free Survival Time, mo |   |   |
| Median (SD) | 21.4 (43.9) | 16.7 (44.8) |
| Busulfan Area Under the Curve (AUC) (0-6 hr) (μmol*min) |   |   |
| Median (SD, range) | 1164 (113, 973-1353) (N=12) | 1173 (391, 777-1926) (N=9) |

Table 1 Caption: Demographic and characteristics of patients undergone autologous stem cells transplantation for acute myeloid leukemia from 1988 to 2010.

### Fine Mapping of Associations via Imputation

In order to further clarify the association signals, we performed imputation on genes with p<0.005 in the primary analysis. For eight genes (*ABCC3*, *DCK*, *GSTM1*, *GSTT1*, *MSH3*, *RRM1*, *SLC22A12*, and *SLC28A3*) the genotypes at polymorphic sites known from the 1000 Genomes Project but not observed on the Illumina HumanOmniExpress v1.0 Beadchip were imputed using IMPUTE version 2 (version 2.3.0 for Mac OS X, [http://http://mathgen.stats.ox.ac.uk/impute/impute_v2.html](http://http://mathgen.stats.ox.ac.uk/impute/impute_v2.html)). The reference panel for the imputation was the 1000 Genomes Phase I integrated variant set referenced to NCBI build b37 (March 2012 relase, retrieved Jan. 20 2013 from [http://http://mathgen.stats.ox.ac.uk/impute/impute_v2.html#reference](http://http://mathgen.stats.ox.ac.uk/impute/impute_v2.html#reference)). Imputed variants with imputation quality scores < 0.3 or MAF < 0.01 were excluded. The remaining imputed SNPs were each used in Cox proportional hazards models to predict disease free survival, as were the genotyped SNPs.

### Functional Studies

The potential functional effects of the SNPs with p<0.01 associated with DFS in European population were examined using the following steps:

## Results

This study investigates the potential associations between variants in 42 candidate genes and disease-free survival (DFS) in adult acute myeloid leukemia (AML) patients treated with a 2-step approach to autologous stem cell transplantation (ASCT). High dose cytarabine, etoposide and busulfan were used in this treatment approach. [Table 1](#T1) describes the demographic characteristics of this cohort, which consisted of 154 patients of European ancestry and 125 patients of non-European ancestry (African, Asian and Mexican). In the DFS analysis, 55 patients of European ancestry (35.7%) and 47 of non-European ancestry (37.6%) relapsed during the observation period, which extended from 1988 to 2010.

We applied quality control criteria to the SNP data, with a genotype call rate of 0.99 and minor allele frequency ≥ 0.01. After filtering, a total of 42 genes covered by 1659 SNPs were included in this association analysis. Furthermore, none of the DNA samples showed chromosomal abnormality by GenomeStudio (Illumina). Results in [Figure 2](#F2) and [Table 2](#T2) showed that among the 40 SNPs with p<0.01, the SNPs in the pharmacokinetic/ pharmacodynamic pathway genes have stronger associations compared to SNPs not in the drug pathway. After Bonferroni correction for multiple testing (1659 tests), a SNP in the first intron of *ABCC3* (rs4148405) was significantly associated with DFS, with the minor allele (G) associated with shorter time to relapse (p-unadjusted=9.5 x 10^-6^, [Figure 3a](#F3)). Although other SNPs did not reach significance after Bonferroni-correction, overall there were 23 SNPs associated with DFS at p<0.005 ([Table 2](#T2)). These 23 SNPs are in or within 25000bp of 8 genes: *SLC28A3*, *DCK*, *RRM1*, *GSTM1*, *ABCC3*, *MSH3*, *GSTT1*, or *SLC22A12*. The majority of the minor alleles were associated with poor outcome (shorter DFS). Kaplan-Meier estimate plots of DFS are shown in [Figure 3a-d](#F3) for four of the top SNPs with MAF ≥ 3%. Other SNPs in the genes of the cytarabine pathway including *NT5C2* and *RRM2B* were also associated with DFS but with weaker p-values (p<0.01, [Table 2](#T2)). Interestingly, SNPs in 3 out of 8 selected genes (*SLC25A37*, *WNT5N* and *P2RX1*), for which expression levels have previously been correlated with either etoposide or cytarabine IC_50_ values in lymphoblastoid cell lines (LCL), showed significant but weaker association (p<0.01) compared to genes in the drug pathways. Next, we examined the 40 SNPs in patients of non-European Ancestry. Only one SNP, in *GSTM1*-*GSTM5* locus (rs3754446), was significantly associated with DFS in patients of non-European Ancestry (HR=1.7, p=0.028). Overall, in the entire cohort, the minor allele of the SNP (rs3754446) was significantly associated with shorter DFS (HR=1.7, p=0.00027).

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3a76/4068832/9e85acb89ac7/nihms467280f2.jpg)

Plot showing the significance (-log10 of the P-value) of associations of the 1659 SNPs with DFS in 154 AML patients of European ancestries. Only SNPs with minor allele frequencies of ≥1% in the selected 42 candidate genes are shown. Each dot represents a SNP. SNPs above the black dotted line are SNPs with p<0.01, and the SNP above the red dotted line reached a p-value,which was significant after correction for multiple testing (p<3x10-5).

### Table 2.

| SNP | Chromosome | Gene | Feature | 5′ Flanking Gene/3′ Flanking Gene | Risk allele | MAF | P (unadjusted) | HR | (95% CI) of the HR | Genotype Counts | Minor Allele | Major Allele | Reason for gene selection |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| rs4148405 | 17 | ABCC3 | intron | CACNA1G/ANKRD40 | G | 0.12 | 9.45E-06 | 3.12 | (1.88-5.15) | 2/34/117 | G | T | Drug pathway |
| rs11140500 | 9 | SLC28A3 | intron | RMI1/NTRK2 | T | 0.01 | 0.00018 | 9.94 | (2.99-33.09 | 0/3/151 | T | C | Drug pathway |
| rs10805074 | 4 | DCK | intron | MOBKL1A/LOC100128311 | A | 0.03 | 0.00039 | 4.59 | (1.98-10.67 | 0/9/145 | A | G | Drug pathway |
| rs7684954 | 4 | DCK | intron | LOC100128311/LOC727995 | A | 0.03 | 0.00039 | 4.59 | (1.98-10.67 | 0/9/145 | A | G | Drug pathway |
| rs4593998 | 11 | RRM1 | intron | RRM1/LOC643244 | A | 0.14 | 0.00093 | 2.28 | (1.40-3.71 | 4/34/116 | A | G | Drug pathway |
| rs6842838 | 4 | DCK | 3′UTR of MOB1B | MOB1B/DCK | G | 0.05 | 0.0010 | 3.21 | (1.60-6.45) | 0/15/139 | G | T | Drug pathway |
| rs1385985 | 4 | DCK | 3′UTR of MOB1B | GRSF1/DCK | C | 0.05 | 0.0010 | 3.21 | (1.60-6.45) | 0/15/139 | C | T | Drug pathway |
| rs3754446 | 1 | GSTM1 | near-gene-5 [GSTM5] | GSTM1/GSTM5 | G | 0.38 | 0.0012 | 1.81 | (1.26-2.59) | 26/66/62 | G | T | Drug pathway |
| rs7689093 | 4 | DCK | near-gene-5 [DCK] | MOBKL1A/DCK | G | 0.01 | 0.0016 | 5.34 | 1.89-15.13) | 0/4/150 | G | A | Drug pathway |
| rs1989983 | 17 | ABCC3 | near-gene-5 [ABCC3] | CACNA1G/ABCC3 | A | 0.11 | 0.0017 | 2.33 | (1.37-3.96) | 2/30/122 | A | G | Drug pathway |
| rs2301835 | 17 | ABCC3 | synonymous variant in codi | CACNA1G/ABCC3 | T | 0.06 | 0.0029 | 2.60 | (1.39-4.89) | 1/16/137 | T | C | Drug pathway |
| rs12515548 | 5 | MSH3 | intron | LOC100128458/RASGRF2 | T | 0.13 | 0.0029 | 2.07 | (1.28-3.35) | 6/27/121 | T | C | DNA mismatch repair genes |
| rs2277624 | 17 | ABCC3 | synonymous variant | CACNA1G/ANKRD40 | A | 0.25 | 0.0040 | 1.75 | (1.20-2.55) | 12/54/87 | A | G | Drug pathway |
| rs11090305 | 22 | GSTT1 | near-gene-5 [CABIN1] | GSTTP2/CABIN1 | C | 0.18 | 0.0044 | 1.98 | (1.24-3.17) | 3/50/101 | C | T | Drug pathway |
| rs7130539 | 11 | RRM1 | intron | STIM1/OR55B1P | C | 0.06 | 0.0047 | 2.56 | (1.33-4.91) | 1/16/137 | C | T | Drug pathway |
| rs11031136 | 11 | RRM1 | intergenic | OR55B1P/LOC643244 | G | 0.06 | 0.0047 | 2.56 | (1.33-4.91) | 1/16/137 | G | T | Drug pathway |
| rs528211 | 11 | SLC22A12 | intergenic | SLC22A11/SLC22A12 | G | 0.29 | 0.0048 | 0.51 | (0.31-0.81) | 13/62/79 | A | G | Other AML study |
| rs2360872 | 11 | SLC22A12 | intergenic | SLC22A11/SLC22A12 | C | 0.29 | 0.0048 | 0.51 | (0.31-0.81) | 13/62/79 | T | C | Other AML study |
| rs505802 | 11 | SLC22A12 | near-gene-5 [SLC22A12] | SLC22A11/SLC22A12 | A | 0.29 | 0.0048 | 0.51 | (0.31-0.81) | 13/62/79 | G | A | Other AML study |
| rs524023 | 11 | SLC22A12 | near-gene-5 [SLC22A12] | SLC22A11/SLC22A12 | G | 0.29 | 0.0048 | 0.51 | (0.31-0.81) | 13/62/79 | A | G | Other AML study |
| rs9734313 | 11 | SLC22A12 | 5′UTR | SLC22A11/SLC22A12 | T | 0.29 | 0.0048 | 0.51 | (0.31-0.81) | 13/62/79 | C | T | Other AML study |
| rs11231825 | 11 | SLC22A12 | synonymous variant | SLC22A11/SLC22A12 | C | 0.29 | 0.0048 | 0.51 | (0.31-0.81) | 13/62/79 | T | C | Other AML study |
| rs2268166 | 11 | RRM1 | intron | STIM1/OR55B1P | G | 0.06 | 0.0049 | 2.55 | (0.20-0.75) | 1/16/136 | G | T | Drug pathway |
| rs11606370 | 11 | SLC22A12 | intron of NRXN2 | SLC22A12/RASGRP2 | A | 0.30 | 0.0050 | 0.51 | (0.32-0.82) | 14/64/76 | C | A | Other AML study |
| rs11191547 | 10 | NT5C2 | intergenic | CNNM2/NT5C2 | T | 0.31 | 0.0051 | 1.71 | (1.17-2.49) | 15/66/73 | T | C | Drug pathway |
| rs11191549 | 10 | NT5C2 | near-gene-3 [NT5C2] | CNNM2/NT5C2 | T | 0.31 | 0.0051 | 1.71 | (1.17-2.49) | 15/66/73 | T | C | Drug pathway |
| rs11191553 | 10 | NT5C2 | intron | CNNM2/LOC100128863 | T | 0.31 | 0.0051 | 1.71 | (1.17-2.49) | 15/66/73 | T | G | Drug pathway |
| rs10883836 | 10 | NT5C2 | intron | LOC100128863/LOC729081 | C | 0.31 | 0.0051 | 1.71 | (1.17-2.49) | 15/66/73 | C | T | Drug pathway |
| rs7095304 | 10 | NT5C2 | intergenic | NT5C2/LOC401648 | A | 0.31 | 0.0051 | 1.71 | (1.17-2.49) | 15/66/73 | A | G | Drug pathway |
| rs6151816 | 5 | MSH3 | intron | LOC100128458/RASGRF2 | T | 0.12 | 0.0055 | 2.05 | (1.24-3.41) | 5/27/122 | T | C | DNA mismatch repair genes |
| rs893006 | 11 | SLC22A12 | intron | SLC22A11/NRXN2 | T | 0.28 | 0.0055 | 0.51 | (0.32-0.82) | 13/61/80 | G | T | Other AML study |
| rs7818607 | 8 | SLC25A37 | intergenic | SLC25A37/LOC646721 | A | 0.30 | 0.0057 | 1.75 | (1.18-2.59) | 16/59/79 | A | C | Associated with drug cytotoxicy in LBL |
| rs2853229 | 8 | RRM2B | intron | NCALD/UBR5 | A | 0.49 | 0.0062 | 1.68 | (1.16-2.44) | 38/74/42 | A | C | Drug pathway |
| rs8534 | 8 | SLC25A37 | intergenic | SLC25A37/LOC646721 | T | 0.38 | 0.0067 | 1.68 | (1.15-2.43) | 24/69/61 | T | C | Associated with drug cytotoxicy in LBL |
| rs8079740 | 17 | ABCC3 | intron of CACNA1G | SPATA20ANCC3 | G | 0.31 | 0.0078 | 0.56 | (0.37-0.86) | 20/56/78 | A | G | Drug pathway |
| rs757420 | 17 | ABCC3 | intergenic | CACNA1G/ABCC3 | T | 0.30 | 0.0079 | 0.55 | (0.36-0.86) | 18/56/80 | T | C | Drug pathway |
| rs2010851 | 12 | WNT5B | near-gene-3 [WNT5B] | WNT5B/LOC100132548 | A | 0.30 | 0.0082 | 0.52 | (0.32-0.84) | 13/67/74 | C | A | Associated with drug cytotoxicy in LBL |
| rs4995289 | 17 | P2RX1 | intergenic | P2RX1/ATP2A3 | T | 0.28 | 0.0088 | 0.51 | (0.31-0.85) | 14/59/81 | C | T | Associated with drug cytotoxicy in LBL |
| rs1516801 | 17 | P2RX1 | intergenic | P2RX1/ATP2A3 | G | 0.28 | 0.0088 | 0.51 | (0.31-0.85) | 14/59/81 | T | G | Associated with drug cytotoxicy in LBL |
| rs2607662 | 8 | RRM2B | intron of UBR5 | NCALD/UBR5 | T | 0.46 | 0.0095 | 1.64 | (1.13-2.37) | 31/79/44 | T | C | Drug pathway |

Table 2 Caption: Significant SNPs (P<0.01) associated with disease free survival in 154 AML patients of European ancestry.

### Figure 3a – d.

![Figure 3a – d](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3a76/4068832/7ed83da4719e/nihms467280f3a.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4068832_nihms467280f3a.jpg)

Kaplan-Meier estimate of disease-free survival (DFS) stratified by the top SNPs (a) rs4148405 ABCC3 (b) rs10805074 DCK (c) rs3754446 GSTM1 and (d) rs505802 SLC22A12 genotypes in patients of European ancestry.

Imputations of the eight candidate genes were performed to determine whether other SNPs in the regions have stronger association with DFS. The results showed that there are 234 imputed SNPs with p<0.01 (MAF ≥ 1%), and among these, there are 93 SNPs with improved p-values compare to the genotyped SNPs ([Figure 4](#F4)). Several SNPs in *DCK* have significant p-values <1.0 x 10^-4^ ([Figure 4](#F4)), with MAF 5-10%. Although none of these 93 SNPs are in exonic regions, on examination in the eQTL Browser, Regulome Database and GTEx eQTL Browser, we determined that SNPs in GSTM1 (rs929166, rs11101989) and MSH3 (6151896), are associated with their respective gene expression levels in liver^21^ or lymphoblastoid cells^22^ (data not shown). In the *ABCC3* and *SLC28A3* regions, imputation analysis did not identify other more significant SNPs in addition to the most significant genotyped SNPs, rs4148405 and rs11140500, respectively ([Figure 4](#F4)).

### Figure 4.

![Figure 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3a76/4068832/a5e8f7b52adf/nihms467280f4.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4068832_nihms467280f4.jpg)

Plot showing the association of the genotyped and imputed SNPs with MAF ≥1% in the selected top 8 candidate genes (ABCC3, DCK, SLC28A3, SLC22AA12, MSH3, RRM1, GSTM1, GSTT1). Each colored diamond (not gray or black) represents a genotyped SNP and the grey/black dots represent imputed SNPs.

Using *in silico* analysis, we determined whether the 40 SNPs and their tag-SNPs were in known or predicted functional regions of the genome. Several of the SNPs were in DNA regions predicted to have binding sites for transcription factors. Some of these regions appear in the ENCODE Chip-Seq and DNase I peaks ([Supplementary Table 2](#SD1)), suggesting that they could have regulatory functions. Interestingly, the GTEx (Genotype-Tissue Expression) eQTL Browser ([http://www.ncbi.nlm.nih.gov/gtex/GTEX2/gtex.cgi](http://www.ncbi.nlm.nih.gov/gtex/GTEX2/gtex.cgi)) and eQTL Browser ([http://eqtl.uchicago.edu/cgi-bin/gbrowse/eqtl/](http://eqtl.uchicago.edu/cgi-bin/gbrowse/eqtl/)) showed that the SNP in *GSTM1*, rs3754446, is associated with *GSTM1* and GSTM5 expression levels in the liver^21^, brain and lymphoblastoid cell lines^22-24^ ([Supplementary Table 2](#SD1)).

## Discussion

Previous pharmacogenomic studies of AML response to chemotherapy have been limited to a small number of candidate genes^2,25-27^. Though studies related to genes in drug pathways have been performed, many have not been replicated. In addition, some AML pharmacogenomics studies have been conducted using DNA from blast cells, which may have included somatic cell mutations in addition to germline polymorphisms^28-30^.

To determine whether germline genetic variations are associated with AML response, we designed our own pharmacogenomic study in AML patients treated with a chemotherapeutic regimen consisting of cytarabine, etoposide and busulfan followed by autologous stem cell transplantation. Our study was focused primarily on 154 AML patients of European ancestry. The analysis was centered on 42 genes related to the pharmacokinetic and pharmacodynamic pathways of the chemotherapy. A few other genes that had previously been associated with drug response in AML were also included ([Supplementary Table 1, Figure 1 and Figure 2](#SD1)). The goals of this study were to identify new associations with DFS in AML patients and to determine whether SNPs previously reported to be associated with AML response could be replicated.

Overall, the most significant SNP in our analysis was the intronic variant in *ABCC3*, rs4148405 (HR=3.1, p=9.5 x 10(-6)). ABCC3 is a multidrug resistance-associated protein (MRP3), which is known to transport the etoposide metabolite, etoposide glucuronide^31^. Several lines of evidence support a role for *ABCC3* in DFS in AML patients. First in a previous study, a promoter variant in *ABCC3*, rs4793665 was associated with a shorter survival time in Israeli AML patients^29^. Though this variant was not associated with AML response in the current study, the data support a role of the transporter in response to chemotherapy in AML. Second, following etoposide administration to *Abcc2* -/-, *Abcc3* -/- mice, higher etoposide glucuronide levels were observed in the liver^32^, consistent with a potentially important role of *ABCC3* in etoposide pharmacokinetics. Third, cell lines transfected with *ABCC3* show greater resistance to etoposide,^33^ Finally, higher *ABCC3* expression levels in leukemia cells are associated with poor outcome in children with leukemia^34,35^.

Data from Regulome Database suggest that the *ABCC3* variant, rs4148405, is in a functional location in the genome, as this region demonstrates direct evidence of binding through ChIP-seq studies. Transcription factors that play a role in hepatic gene regulation (e.g. *CEBPB*, *USF1*, *FOXA1*) have DNA response elements within this gene region (see [http://regulome.stanford.edu/snp/chr17/48713567](http://regulome.stanford.edu/snp/chr17/48713567)) ^20,36^. Based on the results of our primary analysis with DFS, we speculate that the minor allele of rs4148405 is associated with higher expression levels of *ABCC3* in the liver and/or leukemia cells, and thus reduced levels of etoposide in the tumor cells. Although the SNP rs4148405 is found at a MAF of greater than 10% in non-European populations, this SNP was not significantly associated with DFS in the AML patients of non-European ancestry. It is possible that different linkage disequilibrium patterns between rs4148405 and potential causative SNPs may have confounded the analysis.

In addition to a SNP in *ABCC3*, we also identified 23 SNPs in seven other genes (*SLC28A3*, *DCK*, *RRM1*, *GSTM1*, *GSTT1*, *MSH3* and *SLC22A12*) that were associated with DFS in the European AML patients (with p< 0.005, [Table 2](#T2)). Interestingly, expression levels of these genes or other SNPs in these genes have been previously associated with response to chemotherapy in AML or other cancers^19,29,37-39^. Expression levels or SNPs in these genes have also been associated with IC_50_ values of various chemotherapy agents in cell lines^40,41^. Genetic polymorphisms in glutathione-S-transferases, such as *GSTT1* and *GSTM1*, have been widely studied for their associations with drug toxicity, drug response and disease risk in leukemia patients^29,39,42^. Notably, *GSTM1* and *GSTT1* deletions have been implicated in various phenotypes associated with leukemia including drug response^43^, busulfan pharmacokinetics^28^ and disease risk^44^. Although we did not examine the effect of the *GSTM1* deletion in this study, we identified several SNPs (genotyped and imputed) in the *GSTM1*-*GSTM5* locus (rs3754446, rs929166 and rs11101989) associated with DFS in individuals of European ancestry that have not been previously reported ([Table 2](#T2), [Figure 2b](#F2)). One of the SNPs in this locus, rs3754446, was also associated with AML response in the individuals of non-European ancestry (HR=1.7, p=0.028). Overall, in the entire cohort, this minor allele SNP, rs3754446, was significantly associated with shorter DFS (HR=1.7, p=0.00027). Perusal of eQTL databases (Regulome Database, GTEx Browser and eQTL Browser) suggests that the SNPs in the *GSTM1*-*GSTM5* locus are associated with expression levels of *GSTM1* and/or *GSTM5* in liver^21^, brain and lymphoblastoid cell lines^22-24^. Thus, the SNPs in these genes, which are involved in drug metabolism, could affect AML response by affecting the pharmacokinetics of the drugs used in the treatment of AML. We examined 21 AML patients from our overall cohort, where we have their first-dose busulfan AUC (area under the curve). Interestingly, we observed a significant association between rs3754446 and reduced busulfan AUC in these 21 AML patients (p = 0.03, [Supplementary Figure 1](#SD1)). The minor allele, G, in rs3754446, was associated with lower busulfan plasma levels (AUCs), which was consistent with our observation that patients with the G allele had shorter DFS ([Figure 3c](#F3)). Because higher busulfan plasma levels have been associated with busulfan liver toxicity,^14,45^ future studies are needed to determine whether the SNPs in *GSTM1*-*GSTM5* are associated with liver toxicity. Collectively these data suggest that *GSTM1* could play an important role in determining busulfan drug levels, drug response and/or drug toxicity.

Recently, using a drug-metabolizing enzyme/transporter genes (DMET) SNP array a synonymous variant in *SLC22A12* (rs11231825) was found to be associated with response in 94 AML patients treated with a combination drug regimen of Gemtuzumab-Ozogamicin with Fludarabin-Cytarabine-Idarubicin^19^. Among the SNPs in the DMET genes that were found to be significantly associated with AML response, we were able to replicate the SNP (rs11231825) in the uric acid transporter, *SLC22A12*. Notably, this synonymous variant, which is in LD with a SNP upstream of *SLC22A12*, rs505802, has been found in various genomewide association studies to be associated with uric acid levels^46,47^. In these genomewide association studies ^46,47^, the minor allele T, has been associated with higher uric acid levels. Though speculative, our study and the previous study^19^ which demonstrated that patients with the T allele have a better response to chemotherapy, suggest that higher uric acid levels may be associated with longer DFS time. In our study, the T allele, which is associated with higher uric acid levels^51,52^, was associated with longer DFS time (see [Figure 3d](#F3)). Uric acid is a potent antioxidant, and it is possible that higher levels are beneficial for survival in AML patients.

Previous studies have shown that the nucleoside transporter, *SLC28A3* (CNT3) plays a role in cytarabine cytotoxicity and resistance^40,48,49^. In this study, we observed several low allele frequency variants (MAF 1%) in *SLC28A3* are associated with DFS in AML patients on cytarabine and other chemotherapy. Therefore, we hypothesized that CNT3 may transport cytarabine. [Supplementary Figure 2a](#SD1) shows that radiolabeled cytarabine was taken up into CNT3 stably expressing cells and the uptake (over empty vector cells) was significantly enhanced in cell lines exposed to the equilibrative nucleoside transporter inhibitor *S*-(4-nitrobenzyl)-6-thioinosine (NBTI), which reduced background uptake of cytarabine in the cells. Notably, cytarabine uptake decreased significantly in CNT3 stable cells treated with the *SLC28A3* inhibitor (Phloridzin) or with both inhibitors ([Supplementary Figure 2a](#SD1)). Fludarabine, a known substrate of CNT3 was used as a positive control in this study ([Supplementary Figure 2b](#SD1))^50^. Because low allele frequency variant in CNT3 was found to be significantly associated with DFS ([Table 2](#T2)), we interpret the results with caution. Though our finding that *SLC28A3* transports cytarabine supports the association, functional studies of variants in this region and/or a larger sample size are required to determine whether these uncommon variants are associated with cytarabine response.

Deoxycytidine kinase (*DCK*) plays an important role in activating cytarabine to its active metabolite, cytarabine triphosphate. Two tag-SNPs, rs4308342 and rs3775289, in *DCK* that were associated with DFS in our AML patients of European ancestry have been previously associated with the IC_50_ of another nucleoside analog, gemcitabine, in lymphoblastoid cell lines (LCL)^40^. Further, a previous study demonstrated that the level of cytarabine triphosphate in AML blast cells correlates with the ratio in expression levels of the cytarabine activating enzyme, *DCK* and the inactivating enzyme, 5′-nucleotidase, cytosolic II (*NT5C2*)^51^. In our study, in addition to the two tag SNPs in *DCK* associated with DFS in AML patients, five SNPs in the *NT5C2* region were associated with DFS ([Table 2](#T2)). Though several of these SNPs are eQTLs (see [Supplementary Table 2](#SD1)), further studies are required to determine whether these SNPs play important roles in determining the levels of cytarabine triphosphate in AML blast cells. Other genes, which play important roles in the cytarabine pharmacodynamics pathway are ribonucleotide reductase, *RRM1* and *RRM2*, which are considered targets of nucleoside drugs such as cytarabine. The role of this enzyme is to regulate intracellular pools of ribonucleotides, such as deoxycytidine triphosphate (dCTP), which is important in building blocks for DNA replication. Studies have shown that AML blasts cells with high levels of dCTP are resistant to cytarabine, and that there is a significant correlation between *RRM1* and *RRM2* gene expression levels and dCTP levels after cytarabine treatment in AML blast cells^15^. Consistent with previous studies showing that SNPs in *RRM1* are associated with response or toxicity to gemcitabine-based chemotherapy in lung and breast cancer patients^52,53^, our findings suggest that the SNPs in *RRM1* are associated with AML response to chemotherapy that include cytarabine.

Though our current association analysis supports the important roles of drug pathway genes, mainly transporters and enzymes, in AML response, we also selected 8 genes that have been associated with cytarabine or etoposide IC_50_ values in LCL^16,17^. A few SNPs in the 8 genes were significantly associated with response in AML patients suggesting that genes identified in in vitro assays in LCLs may also be important predictors of *in vivo* drug response in AML patients^16,17^

In summary, in this genetic association study of DFS in AML patients, we identified polymorphisms that have not been previously associated with AML response, including SNPs in *ABCC3*, *DCK*, *GSTM1*, *MSH3*, *NT5C2*, *RRM1* and *SLC28A3*. A SNP in *ABCC3*, rs4148405, which remained significant after multiple testing, was associated with DFS, suggesting an important role of *ABCC3* in determining etoposide levels in the liver and other tissues and hence AML response. Many of the significant SNPs or their tag-SNPs were eQTLs or located in functional regions in the genome. Finally, we determined for the first time that *SLC28A3* (CNT3) transported cytarabine into cells suggesting an important role of this transporter in cytarabine cytotoxicity.

## Supplementary Material

## Acknowledgments

This research was supported in part by a grant from NIH NIGMS (GM61390). This study was supported by the NIH Pharmacogenomics Research Network (PGRN)-RIKEN Center for Genomic Medicine (CGM) Strategic Alliance. We are grateful to the UCSF AML Tissue Bank staff (Ms. Joy Cruz, Ms. Christine Cheng). SWY would like to thank Chris Gignoux and Pär Mattson for their help in creating plots in R-project.

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Daver N, Cortes J. Molecular targeted therapy in acute myeloid leukemia. Hematology. 2012;17 Suppl 1:S59–62. doi: 10.1179/102453312X13336169155619.  [DOI](https://doi.org/10.1179/102453312X13336169155619) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22507781/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hematology&title=Molecular%20targeted%20therapy%20in%20acute%20myeloid%20leukemia&author=N%20Daver&author=J%20Cortes&volume=17%20Suppl%201&publication_year=2012&pages=S59-62&pmid=22507781&doi=10.1179/102453312X13336169155619&)

2. Patel JP, et al. Prognostic relevance of integrated genetic profiling in acute myeloid leukemia. N Engl J Med. 2012;366:1079–89. doi: 10.1056/NEJMoa1112304.  [DOI](https://doi.org/10.1056/NEJMoa1112304) | [PMC free article](/articles/PMC3545649/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22417203/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Prognostic%20relevance%20of%20integrated%20genetic%20profiling%20in%20acute%20myeloid%20leukemia&author=JP%20Patel&volume=366&publication_year=2012&pages=1079-89&pmid=22417203&doi=10.1056/NEJMoa1112304&)

3. Buccisano F, et al. Prognostic and therapeutic implications of minimal residual disease detection in acute myeloid leukemia. Blood. 2012;119:332–41. doi: 10.1182/blood-2011-08-363291.  [DOI](https://doi.org/10.1182/blood-2011-08-363291) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22039260/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Prognostic%20and%20therapeutic%20implications%20of%20minimal%20residual%20disease%20detection%20in%20acute%20myeloid%20leukemia&author=F%20Buccisano&volume=119&publication_year=2012&pages=332-41&pmid=22039260&doi=10.1182/blood-2011-08-363291&)

4. Martelli MP, Sportoletti P, Tiacci E, Martelli MF, Falini B. Mutational landscape of AML with normal cytogenetics: Biological and clinical implications. Blood Rev. 2013;27:13–22. doi: 10.1016/j.blre.2012.11.001.  [DOI](https://doi.org/10.1016/j.blre.2012.11.001) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23261068/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood%20Rev&title=Mutational%20landscape%20of%20AML%20with%20normal%20cytogenetics:%20Biological%20and%20clinical%20implications&author=MP%20Martelli&author=P%20Sportoletti&author=E%20Tiacci&author=MF%20Martelli&author=B%20Falini&volume=27&publication_year=2013&pages=13-22&pmid=23261068&doi=10.1016/j.blre.2012.11.001&)

5. Kroger N, et al. Autologous stem cell transplantation for therapy-related acute myeloid leukemia and myelodysplastic syndrome. Bone Marrow Transplant. 2006;37:183–9. doi: 10.1038/sj.bmt.1705226.  [DOI](https://doi.org/10.1038/sj.bmt.1705226) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16299545/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bone%20Marrow%20Transplant&title=Autologous%20stem%20cell%20transplantation%20for%20therapy-related%20acute%20myeloid%20leukemia%20and%20myelodysplastic%20syndrome&author=N%20Kroger&volume=37&publication_year=2006&pages=183-9&pmid=16299545&doi=10.1038/sj.bmt.1705226&)

6. Novitzky N, Thomas V, du Toit C, McDonald A. Is there a role for autologous stem cell transplantation for patients with acute myelogenous leukemia? A retrospective analysis. Biol Blood Marrow Transplant. 2011;17:875–84. doi: 10.1016/j.bbmt.2010.09.016.  [DOI](https://doi.org/10.1016/j.bbmt.2010.09.016) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20870030/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biol%20Blood%20Marrow%20Transplant&title=Is%20there%20a%20role%20for%20autologous%20stem%20cell%20transplantation%20for%20patients%20with%20acute%20myelogenous%20leukemia?%20A%20retrospective%20analysis&author=N%20Novitzky&author=V%20Thomas&author=C%20du%20Toit&author=A%20McDonald&volume=17&publication_year=2011&pages=875-84&pmid=20870030&doi=10.1016/j.bbmt.2010.09.016&)

7. Linker CA, et al. Autologous stem cell transplantation for advanced acute myeloid leukemia. Bone Marrow Transplant. 2002;29:297–301. doi: 10.1038/sj.bmt.1703361.  [DOI](https://doi.org/10.1038/sj.bmt.1703361) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11896426/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bone%20Marrow%20Transplant&title=Autologous%20stem%20cell%20transplantation%20for%20advanced%20acute%20myeloid%20leukemia&author=CA%20Linker&volume=29&publication_year=2002&pages=297-301&pmid=11896426&doi=10.1038/sj.bmt.1703361&)

8. Linker CA, et al. Auto-SCT for AML in second remission: CALGB study 9620. Bone Marrow Transplant. 2009;44:353–9. doi: 10.1038/bmt.2009.36.  [DOI](https://doi.org/10.1038/bmt.2009.36) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19289999/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bone%20Marrow%20Transplant&title=Auto-SCT%20for%20AML%20in%20second%20remission:%20CALGB%20study%209620&author=CA%20Linker&volume=44&publication_year=2009&pages=353-9&pmid=19289999&doi=10.1038/bmt.2009.36&)

9. Linker CA, et al. Autologous stem cell transplantation for acute myeloid leukemia in first remission. Biol Blood Marrow Transplant. 2000;6:50–7. doi: 10.1016/s1083-8791(00)70052-8.  [DOI](https://doi.org/10.1016/s1083-8791(00)70052-8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10707999/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biol%20Blood%20Marrow%20Transplant&title=Autologous%20stem%20cell%20transplantation%20for%20acute%20myeloid%20leukemia%20in%20first%20remission&author=CA%20Linker&volume=6&publication_year=2000&pages=50-7&pmid=10707999&doi=10.1016/s1083-8791(00)70052-8&)

10. Price AL, et al. Principal components analysis corrects for stratification in genome-wide association studies. Nat Genet. 2006;38:904–9. doi: 10.1038/ng1847.  [DOI](https://doi.org/10.1038/ng1847) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16862161/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Principal%20components%20analysis%20corrects%20for%20stratification%20in%20genome-wide%20association%20studies&author=AL%20Price&volume=38&publication_year=2006&pages=904-9&pmid=16862161&doi=10.1038/ng1847&)

11. Yang J, et al. Etoposide pathway. Pharmacogenet Genomics. 2009;19:552–3. doi: 10.1097/FPC.0b013e32832e0e7f.  [DOI](https://doi.org/10.1097/FPC.0b013e32832e0e7f) | [PMC free article](/articles/PMC4164627/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19512958/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Etoposide%20pathway&author=J%20Yang&volume=19&publication_year=2009&pages=552-3&pmid=19512958&doi=10.1097/FPC.0b013e32832e0e7f&)

12. Li L, et al. Gemcitabine metabolic pathway genetic polymorphisms and response in patients with non-small cell lung cancer. Pharmacogenet Genomics. 2012;22:105–16. doi: 10.1097/FPC.0b013e32834dd7e2.  [DOI](https://doi.org/10.1097/FPC.0b013e32834dd7e2) | [PMC free article](/articles/PMC3259218/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22173087/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Gemcitabine%20metabolic%20pathway%20genetic%20polymorphisms%20and%20response%20in%20patients%20with%20non-small%20cell%20lung%20cancer&author=L%20Li&volume=22&publication_year=2012&pages=105-16&pmid=22173087&doi=10.1097/FPC.0b013e32834dd7e2&)

13. Hassan M, Andersson BS. Role of pharmacogenetics in busulfan/cyclophosphamide conditioning therapy prior to hematopoietic stem cell transplantation. Pharmacogenomics. 2013;14:75–87. doi: 10.2217/pgs.12.185.  [DOI](https://doi.org/10.2217/pgs.12.185) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23252950/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Role%20of%20pharmacogenetics%20in%20busulfan/cyclophosphamide%20conditioning%20therapy%20prior%20to%20hematopoietic%20stem%20cell%20transplantation&author=M%20Hassan&author=BS%20Andersson&volume=14&publication_year=2013&pages=75-87&pmid=23252950&doi=10.2217/pgs.12.185&)

14. Abbasi N, et al. Pharmacogenetics of intravenous and oral busulfan in hematopoietic cell transplant recipients. J Clin Pharmacol. 2011;51:1429–38. doi: 10.1177/0091270010382915.  [DOI](https://doi.org/10.1177/0091270010382915) | [PMC free article](/articles/PMC3117932/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21135089/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Pharmacol&title=Pharmacogenetics%20of%20intravenous%20and%20oral%20busulfan%20in%20hematopoietic%20cell%20transplant%20recipients&author=N%20Abbasi&volume=51&publication_year=2011&pages=1429-38&pmid=21135089&doi=10.1177/0091270010382915&)

15. Lamba JK. Genetic factors influencing cytarabine therapy. Pharmacogenomics. 2009;10:1657–74. doi: 10.2217/pgs.09.118.  [DOI](https://doi.org/10.2217/pgs.09.118) | [PMC free article](/articles/PMC2828057/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19842938/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Genetic%20factors%20influencing%20cytarabine%20therapy&author=JK%20Lamba&volume=10&publication_year=2009&pages=1657-74&pmid=19842938&doi=10.2217/pgs.09.118&)

16. Huang RS, et al. A genome-wide approach to identify genetic variants that contribute to etoposide-induced cytotoxicity. Proc Natl Acad Sci U S A. 2007;104:9758–63. doi: 10.1073/pnas.0703736104.  [DOI](https://doi.org/10.1073/pnas.0703736104) | [PMC free article](/articles/PMC1887589/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17537913/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&title=A%20genome-wide%20approach%20to%20identify%20genetic%20variants%20that%20contribute%20to%20etoposide-induced%20cytotoxicity&author=RS%20Huang&volume=104&publication_year=2007&pages=9758-63&pmid=17537913&doi=10.1073/pnas.0703736104&)

17. Hartford CM, et al. Population-specific genetic variants important in susceptibility to cytarabine arabinoside cytotoxicity. Blood. 2009;113:2145–53. doi: 10.1182/blood-2008-05-154302.  [DOI](https://doi.org/10.1182/blood-2008-05-154302) | [PMC free article](/articles/PMC2652364/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19109566/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Population-specific%20genetic%20variants%20important%20in%20susceptibility%20to%20cytarabine%20arabinoside%20cytotoxicity&author=CM%20Hartford&volume=113&publication_year=2009&pages=2145-53&pmid=19109566&doi=10.1182/blood-2008-05-154302&)

18. Hewish M, et al. Cytosine-based nucleoside analogs are selectively lethal to DNA mismatch repair-deficient tumour cells by enhancing levels of intracellular oxidative stress. Br J Cancer. 2013 doi: 10.1038/bjc.2013.3.  [DOI](https://doi.org/10.1038/bjc.2013.3) | [PMC free article](/articles/PMC3590674/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23361057/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Cancer&title=Cytosine-based%20nucleoside%20analogs%20are%20selectively%20lethal%20to%20DNA%20mismatch%20repair-deficient%20tumour%20cells%20by%20enhancing%20levels%20of%20intracellular%20oxidative%20stress&author=M%20Hewish&publication_year=2013&pmid=23361057&doi=10.1038/bjc.2013.3&)

19. Iacobucci I, et al. Profiling of drug-metabolizing enzymes/transporters in CD33+ acute myeloid leukemia patients treated with Gemtuzumab-Ozogamicin and Fludarabine, Cytarabine and Idarubicin. Pharmacogenomics J. 2012 doi: 10.1038/tpj.2012.13.  [DOI](https://doi.org/10.1038/tpj.2012.13) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22584460/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=Profiling%20of%20drug-metabolizing%20enzymes/transporters%20in%20CD33+%20acute%20myeloid%20leukemia%20patients%20treated%20with%20Gemtuzumab-Ozogamicin%20and%20Fludarabine,%20Cytarabine%20and%20Idarubicin&author=I%20Iacobucci&publication_year=2012&pmid=22584460&doi=10.1038/tpj.2012.13&)

20. Boyle AP, et al. Annotation of functional variation in personal genomes using RegulomeDB. Genome Res. 2012;22:1790–7. doi: 10.1101/gr.137323.112.  [DOI](https://doi.org/10.1101/gr.137323.112) | [PMC free article](/articles/PMC3431494/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22955989/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Res&title=Annotation%20of%20functional%20variation%20in%20personal%20genomes%20using%20RegulomeDB&author=AP%20Boyle&volume=22&publication_year=2012&pages=1790-7&pmid=22955989&doi=10.1101/gr.137323.112&)

21. Schadt EE, et al. Mapping the genetic architecture of gene expression in human liver. PLoS Biol. 2008;6:e107. doi: 10.1371/journal.pbio.0060107.  [DOI](https://doi.org/10.1371/journal.pbio.0060107) | [PMC free article](/articles/PMC2365981/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18462017/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Biol&title=Mapping%20the%20genetic%20architecture%20of%20gene%20expression%20in%20human%20liver&author=EE%20Schadt&volume=6&publication_year=2008&pages=e107&pmid=18462017&doi=10.1371/journal.pbio.0060107&)

22. Stranger BE, et al. Population genomics of human gene expression. Nat Genet. 2007;39:1217–24. doi: 10.1038/ng2142.  [DOI](https://doi.org/10.1038/ng2142) | [PMC free article](/articles/PMC2683249/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17873874/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Population%20genomics%20of%20human%20gene%20expression&author=BE%20Stranger&volume=39&publication_year=2007&pages=1217-24&pmid=17873874&doi=10.1038/ng2142&)

23. Veyrieras JB, et al. High-resolution mapping of expression-QTLs yields insight into human gene regulation. PLoS Genet. 2008;4:e1000214. doi: 10.1371/journal.pgen.1000214.  [DOI](https://doi.org/10.1371/journal.pgen.1000214) | [PMC free article](/articles/PMC2556086/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18846210/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet&title=High-resolution%20mapping%20of%20expression-QTLs%20yields%20insight%20into%20human%20gene%20regulation&author=JB%20Veyrieras&volume=4&publication_year=2008&pages=e1000214&pmid=18846210&doi=10.1371/journal.pgen.1000214&)

24. Gibbs JR, et al. Abundant quantitative trait loci exist for DNA methylation and gene expression in human brain. PLoS Genet. 2010;6:e1000952. doi: 10.1371/journal.pgen.1000952.  [DOI](https://doi.org/10.1371/journal.pgen.1000952) | [PMC free article](/articles/PMC2869317/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20485568/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet&title=Abundant%20quantitative%20trait%20loci%20exist%20for%20DNA%20methylation%20and%20gene%20expression%20in%20human%20brain&author=JR%20Gibbs&volume=6&publication_year=2010&pages=e1000952&pmid=20485568&doi=10.1371/journal.pgen.1000952&)

25. Golub TR, et al. Molecular classification of cancer: class discovery and class prediction by gene expression monitoring. Science. 1999;286:531–7. doi: 10.1126/science.286.5439.531.  [DOI](https://doi.org/10.1126/science.286.5439.531) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10521349/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=Molecular%20classification%20of%20cancer:%20class%20discovery%20and%20class%20prediction%20by%20gene%20expression%20monitoring&author=TR%20Golub&volume=286&publication_year=1999&pages=531-7&pmid=10521349&doi=10.1126/science.286.5439.531&)

26. Gaidzik VI, et al. TET2 mutations in acute myeloid leukemia (AML): results from a comprehensive genetic and clinical analysis of the AML study group. J Clin Oncol. 2012;30:1350–7. doi: 10.1200/JCO.2011.39.2886.  [DOI](https://doi.org/10.1200/JCO.2011.39.2886) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22430270/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=TET2%20mutations%20in%20acute%20myeloid%20leukemia%20(AML):%20results%20from%20a%20comprehensive%20genetic%20and%20clinical%20analysis%20of%20the%20AML%20study%20group&author=VI%20Gaidzik&volume=30&publication_year=2012&pages=1350-7&pmid=22430270&doi=10.1200/JCO.2011.39.2886&)

27. Paschka P, et al. Wilms' tumor 1 gene mutations independently predict poor outcome in adults with cytogenetically normal acute myeloid leukemia: a cancer and leukemia group B study. J Clin Oncol. 2008;26:4595–602. doi: 10.1200/JCO.2007.15.2058.  [DOI](https://doi.org/10.1200/JCO.2007.15.2058) | [PMC free article](/articles/PMC2653131/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18559874/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Wilms'%20tumor%201%20gene%20mutations%20independently%20predict%20poor%20outcome%20in%20adults%20with%20cytogenetically%20normal%20acute%20myeloid%20leukemia:%20a%20cancer%20and%20leukemia%20group%20B%20study&author=P%20Paschka&volume=26&publication_year=2008&pages=4595-602&pmid=18559874&doi=10.1200/JCO.2007.15.2058&)

28. Kim SD, et al. Influence of GST gene polymorphisms on the clearance of intravenous busulfan in adult patients undergoing hematopoietic cell transplantation. Biol Blood Marrow Transplant. 2011;17:1222–30. doi: 10.1016/j.bbmt.2010.12.708.  [DOI](https://doi.org/10.1016/j.bbmt.2010.12.708) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21215809/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biol%20Blood%20Marrow%20Transplant&title=Influence%20of%20GST%20gene%20polymorphisms%20on%20the%20clearance%20of%20intravenous%20busulfan%20in%20adult%20patients%20undergoing%20hematopoietic%20cell%20transplantation&author=SD%20Kim&volume=17&publication_year=2011&pages=1222-30&pmid=21215809&doi=10.1016/j.bbmt.2010.12.708&)

29. Muller P, et al. Polymorphisms in transporter and phase II metabolism genes as potential modifiers of the predisposition to and treatment outcome of de novo acute myeloid leukemia in Israeli ethnic groups. Leuk Res. 2008;32:919–29. doi: 10.1016/j.leukres.2007.10.011.  [DOI](https://doi.org/10.1016/j.leukres.2007.10.011) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18207572/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Leuk%20Res&title=Polymorphisms%20in%20transporter%20and%20phase%20II%20metabolism%20genes%20as%20potential%20modifiers%20of%20the%20predisposition%20to%20and%20treatment%20outcome%20of%20de%20novo%20acute%20myeloid%20leukemia%20in%20Israeli%20ethnic%20groups&author=P%20Muller&volume=32&publication_year=2008&pages=919-29&pmid=18207572&doi=10.1016/j.leukres.2007.10.011&)

30. Emadi A, Karp JE. The clinically relevant pharmacogenomic changes in acute myelogenous leukemia. Pharmacogenomics. 2012;13:1257–69. doi: 10.2217/pgs.12.102.  [DOI](https://doi.org/10.2217/pgs.12.102) | [PMC free article](/articles/PMC3544213/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22920396/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=The%20clinically%20relevant%20pharmacogenomic%20changes%20in%20acute%20myelogenous%20leukemia&author=A%20Emadi&author=JE%20Karp&volume=13&publication_year=2012&pages=1257-69&pmid=22920396&doi=10.2217/pgs.12.102&)

31. Zelcer N, Saeki T, Reid G, Beijnen JH, Borst P. Characterization of drug transport by the human multidrug resistance protein 3 (ABCC3) J Biol Chem. 2001;276:46400–7. doi: 10.1074/jbc.M107041200.  [DOI](https://doi.org/10.1074/jbc.M107041200) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11581266/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=Characterization%20of%20drug%20transport%20by%20the%20human%20multidrug%20resistance%20protein%203%20(ABCC3)&author=N%20Zelcer&author=T%20Saeki&author=G%20Reid&author=JH%20Beijnen&author=P%20Borst&volume=276&publication_year=2001&pages=46400-7&pmid=11581266&doi=10.1074/jbc.M107041200&)

32. Lagas JS, et al. P-glycoprotein (P-gp/Abcb1), Abcc2, and Abcc3 determine the pharmacokinetics of etoposide. Clin Cancer Res. 2010;16:130–40. doi: 10.1158/1078-0432.CCR-09-1321.  [DOI](https://doi.org/10.1158/1078-0432.CCR-09-1321) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20028753/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&title=P-glycoprotein%20(P-gp/Abcb1),%20Abcc2,%20and%20Abcc3%20determine%20the%20pharmacokinetics%20of%20etoposide&author=JS%20Lagas&volume=16&publication_year=2010&pages=130-40&pmid=20028753&doi=10.1158/1078-0432.CCR-09-1321&)

33. Belinsky MG, et al. Analysis of the in vivo functions of Mrp3. Mol Pharmacol. 2005;68:160–8. doi: 10.1124/mol.104.010587.  [DOI](https://doi.org/10.1124/mol.104.010587) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15814571/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Pharmacol&title=Analysis%20of%20the%20in%20vivo%20functions%20of%20Mrp3&author=MG%20Belinsky&volume=68&publication_year=2005&pages=160-8&pmid=15814571&doi=10.1124/mol.104.010587&)

34. Steinbach D, et al. The multidrug resistance-associated protein 3 (MRP3) is associated with a poor outcome in childhood ALL and may account for the worse prognosis in male patients and T-cell immunophenotype. Blood. 2003;102:4493–8. doi: 10.1182/blood-2002-11-3461.  [DOI](https://doi.org/10.1182/blood-2002-11-3461) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12816874/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=The%20multidrug%20resistance-associated%20protein%203%20(MRP3)%20is%20associated%20with%20a%20poor%20outcome%20in%20childhood%20ALL%20and%20may%20account%20for%20the%20worse%20prognosis%20in%20male%20patients%20and%20T-cell%20immunophenotype&author=D%20Steinbach&volume=102&publication_year=2003&pages=4493-8&pmid=12816874&doi=10.1182/blood-2002-11-3461&)

35. Steinbach D, et al. Response to chemotherapy and expression of the genes encoding the multidrug resistance-associated proteins MRP2, MRP3, MRP4, MRP5, and SMRP in childhood acute myeloid leukemia. Clin Cancer Res. 2003;9:1083–6.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/12631611/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&title=Response%20to%20chemotherapy%20and%20expression%20of%20the%20genes%20encoding%20the%20multidrug%20resistance-associated%20proteins%20MRP2,%20MRP3,%20MRP4,%20MRP5,%20and%20SMRP%20in%20childhood%20acute%20myeloid%20leukemia&author=D%20Steinbach&volume=9&publication_year=2003&pages=1083-6&pmid=12631611&)

36. Dunham I, et al. An integrated encyclopedia of DNA elements in the human genome. Nature. 2012;489:57–74. doi: 10.1038/nature11247.  [DOI](https://doi.org/10.1038/nature11247) | [PMC free article](/articles/PMC3439153/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22955616/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=An%20integrated%20encyclopedia%20of%20DNA%20elements%20in%20the%20human%20genome&author=I%20Dunham&volume=489&publication_year=2012&pages=57-74&pmid=22955616&doi=10.1038/nature11247&)

37. Kim KI, et al. Combined interaction of multi-locus genetic polymorphisms in cytarabine arabinoside metabolic pathway on clinical outcomes in adult acute myeloid leukaemia (AML) patients. Eur J Cancer. 2013;49:403–10. doi: 10.1016/j.ejca.2012.07.022.  [DOI](https://doi.org/10.1016/j.ejca.2012.07.022) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22964418/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Cancer&title=Combined%20interaction%20of%20multi-locus%20genetic%20polymorphisms%20in%20cytarabine%20arabinoside%20metabolic%20pathway%20on%20clinical%20outcomes%20in%20adult%20acute%20myeloid%20leukaemia%20(AML)%20patients&author=KI%20Kim&volume=49&publication_year=2013&pages=403-10&pmid=22964418&doi=10.1016/j.ejca.2012.07.022&)

38. Shi JY, et al. Association between single nucleotide polymorphisms in deoxycytidine kinase and treatment response among acute myeloid leukaemia patients. Pharmacogenetics. 2004;14:759–68. doi: 10.1097/00008571-200411000-00007.  [DOI](https://doi.org/10.1097/00008571-200411000-00007) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15564883/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=Association%20between%20single%20nucleotide%20polymorphisms%20in%20deoxycytidine%20kinase%20and%20treatment%20response%20among%20acute%20myeloid%20leukaemia%20patients&author=JY%20Shi&volume=14&publication_year=2004&pages=759-68&pmid=15564883&doi=10.1097/00008571-200411000-00007&)

39. Xiao Z, et al. Glutathione S-transferases (GSTT1 and GSTM1) genes polymorphisms and the treatment response and prognosis in Chinese patients with de novo acute myeloid leukemia. Leuk Res. 2008;32:1288–91. doi: 10.1016/j.leukres.2007.10.010.  [DOI](https://doi.org/10.1016/j.leukres.2007.10.010) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18035413/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Leuk%20Res&title=Glutathione%20S-transferases%20(GSTT1%20and%20GSTM1)%20genes%20polymorphisms%20and%20the%20treatment%20response%20and%20prognosis%20in%20Chinese%20patients%20with%20de%20novo%20acute%20myeloid%20leukemia&author=Z%20Xiao&volume=32&publication_year=2008&pages=1288-91&pmid=18035413&doi=10.1016/j.leukres.2007.10.010&)

40. Li L, et al. Gemcitabine and arabinosylcytosin pharmacogenomics: genome-wide association and drug response biomarkers. PLoS One. 2009;4:e7765. doi: 10.1371/journal.pone.0007765.  [DOI](https://doi.org/10.1371/journal.pone.0007765) | [PMC free article](/articles/PMC2770319/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19898621/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=Gemcitabine%20and%20arabinosylcytosin%20pharmacogenomics:%20genome-wide%20association%20and%20drug%20response%20biomarkers&author=L%20Li&volume=4&publication_year=2009&pages=e7765&pmid=19898621&doi=10.1371/journal.pone.0007765&)

41. Mitra AK, et al. Genetic variants in cytosolic 5′-nucleotidase II are associated with its expression and cytarabine sensitivity in HapMap cell lines and in patients with acute myeloid leukemia. J Pharmacol Exp Ther. 2011;339:9–23. doi: 10.1124/jpet.111.182873.  [DOI](https://doi.org/10.1124/jpet.111.182873) | [PMC free article](/articles/PMC3186292/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21712425/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharmacol%20Exp%20Ther&title=Genetic%20variants%20in%20cytosolic%205%E2%80%B2-nucleotidase%20II%20are%20associated%20with%20its%20expression%20and%20cytarabine%20sensitivity%20in%20HapMap%20cell%20lines%20and%20in%20patients%20with%20acute%20myeloid%20leukemia&author=AK%20Mitra&volume=339&publication_year=2011&pages=9-23&pmid=21712425&doi=10.1124/jpet.111.182873&)

42. Mossallam GI, Abdel Hamid TM, Samra MA. Glutathione S-transferase GSTM1 and GSTT1 polymorphisms in adult acute myeloid leukemia; its impact on toxicity and response to chemotherapy. J Egypt Natl Canc Inst. 2006;18:264–73.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/17671537/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Egypt%20Natl%20Canc%20Inst&title=Glutathione%20S-transferase%20GSTM1%20and%20GSTT1%20polymorphisms%20in%20adult%20acute%20myeloid%20leukemia;%20its%20impact%20on%20toxicity%20and%20response%20to%20chemotherapy&author=GI%20Mossallam&author=TM%20Abdel%20Hamid&author=MA%20Samra&volume=18&publication_year=2006&pages=264-73&pmid=17671537&)

43. Voso MT, et al. Prognostic role of glutathione S-transferase polymorphisms in acute myeloid leukemia. Leukemia. 2008;22:1685–91. doi: 10.1038/leu.2008.169.  [DOI](https://doi.org/10.1038/leu.2008.169) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18580952/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Leukemia&title=Prognostic%20role%20of%20glutathione%20S-transferase%20polymorphisms%20in%20acute%20myeloid%20leukemia&author=MT%20Voso&volume=22&publication_year=2008&pages=1685-91&pmid=18580952&doi=10.1038/leu.2008.169&)

44. Das P, Shaik AP, Bammidi VK. Meta-analysis study of glutathione-S-transferases (GSTM1, GSTP1, and GSTT1) gene polymorphisms and risk of acute myeloid leukemia. Leuk Lymphoma. 2009;50:1345–51. doi: 10.1080/10428190903003236.  [DOI](https://doi.org/10.1080/10428190903003236) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19811334/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Leuk%20Lymphoma&title=Meta-analysis%20study%20of%20glutathione-S-transferases%20(GSTM1,%20GSTP1,%20and%20GSTT1)%20gene%20polymorphisms%20and%20risk%20of%20acute%20myeloid%20leukemia&author=P%20Das&author=AP%20Shaik&author=VK%20Bammidi&volume=50&publication_year=2009&pages=1345-51&pmid=19811334&doi=10.1080/10428190903003236&)

45. Zhang H, et al. Pharmacokinetic-directed high-dose busulfan combined with cyclophosphamide and etoposide results in predictable drug levels and durable long-term survival in lymphoma patients undergoing autologous stem cell transplantation. Biol Blood Marrow Transplant. 2012;18:1287–94. doi: 10.1016/j.bbmt.2012.02.006.  [DOI](https://doi.org/10.1016/j.bbmt.2012.02.006) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22370160/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biol%20Blood%20Marrow%20Transplant&title=Pharmacokinetic-directed%20high-dose%20busulfan%20combined%20with%20cyclophosphamide%20and%20etoposide%20results%20in%20predictable%20drug%20levels%20and%20durable%20long-term%20survival%20in%20lymphoma%20patients%20undergoing%20autologous%20stem%20cell%20transplantation&author=H%20Zhang&volume=18&publication_year=2012&pages=1287-94&pmid=22370160&doi=10.1016/j.bbmt.2012.02.006&)

46. Takeuchi F, et al. Genetic Impact on Uric Acid Concentration and Hyperuricemia in the Japanese Population. J Atheroscler Thromb. 2012 doi: 10.5551/jat.15727.  [DOI](https://doi.org/10.5551/jat.15727) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23238572/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Atheroscler%20Thromb&title=Genetic%20Impact%20on%20Uric%20Acid%20Concentration%20and%20Hyperuricemia%20in%20the%20Japanese%20Population&author=F%20Takeuchi&publication_year=2012&pmid=23238572&doi=10.5551/jat.15727&)

47. Kolz M, et al. Meta-analysis of 28,141 individuals identifies common variants within five new loci that influence uric acid concentrations. PLoS Genet. 2009;5:e1000504. doi: 10.1371/journal.pgen.1000504.  [DOI](https://doi.org/10.1371/journal.pgen.1000504) | [PMC free article](/articles/PMC2683940/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19503597/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet&title=Meta-analysis%20of%2028,141%20individuals%20identifies%20common%20variants%20within%20five%20new%20loci%20that%20influence%20uric%20acid%20concentrations&author=M%20Kolz&volume=5&publication_year=2009&pages=e1000504&pmid=19503597&doi=10.1371/journal.pgen.1000504&)

48. Errasti-Murugarren E, Pastor-Anglada M, Casado FJ. Role of CNT3 in the transepithelial flux of nucleosides and nucleoside-derived drugs. J Physiol. 2007;582:1249–60. doi: 10.1113/jphysiol.2007.130138.  [DOI](https://doi.org/10.1113/jphysiol.2007.130138) | [PMC free article](/articles/PMC2075247/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17412768/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Physiol&title=Role%20of%20CNT3%20in%20the%20transepithelial%20flux%20of%20nucleosides%20and%20nucleoside-derived%20drugs&author=E%20Errasti-Murugarren&author=M%20Pastor-Anglada&author=FJ%20Casado&volume=582&publication_year=2007&pages=1249-60&pmid=17412768&doi=10.1113/jphysiol.2007.130138&)

49. Sarkar M, et al. Cytosine arabinoside affects multiple cellular factors and induces drug resistance in human lymphoid cells. Biochem Pharmacol. 2005;70:426–32. doi: 10.1016/j.bcp.2005.05.014.  [DOI](https://doi.org/10.1016/j.bcp.2005.05.014) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15950950/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem%20Pharmacol&title=Cytosine%20arabinoside%20affects%20multiple%20cellular%20factors%20and%20induces%20drug%20resistance%20in%20human%20lymphoid%20cells&author=M%20Sarkar&volume=70&publication_year=2005&pages=426-32&pmid=15950950&doi=10.1016/j.bcp.2005.05.014&)

50. Badagnani I, et al. Functional analysis of genetic variants in the human concentrative nucleoside transporter 3 (CNT3; SLC28A3) Pharmacogenomics J. 2005;5:157–65. doi: 10.1038/sj.tpj.6500303.  [DOI](https://doi.org/10.1038/sj.tpj.6500303) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15738947/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=Functional%20analysis%20of%20genetic%20variants%20in%20the%20human%20concentrative%20nucleoside%20transporter%203%20(CNT3;%20SLC28A3)&author=I%20Badagnani&volume=5&publication_year=2005&pages=157-65&pmid=15738947&doi=10.1038/sj.tpj.6500303&)

51. Yamauchi T, et al. Intracellular cytarabine triphosphate production correlates to deoxycytidine kinase/cytosolic 5′-nucleotidase II expression ratio in primary acute myeloid leukemia cells. Biochem Pharmacol. 2009;77:1780–6. doi: 10.1016/j.bcp.2009.03.011.  [DOI](https://doi.org/10.1016/j.bcp.2009.03.011) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19428333/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem%20Pharmacol&title=Intracellular%20cytarabine%20triphosphate%20production%20correlates%20to%20deoxycytidine%20kinase/cytosolic%205%E2%80%B2-nucleotidase%20II%20expression%20ratio%20in%20primary%20acute%20myeloid%20leukemia%20cells&author=T%20Yamauchi&volume=77&publication_year=2009&pages=1780-6&pmid=19428333&doi=10.1016/j.bcp.2009.03.011&)

52. Kim SO, et al. Efficacy of gemcitabine in patients with non-small cell lung cancer according to promoter polymorphisms of the ribonucleotide reductase M1 gene. Clin Cancer Res. 2008;14:3083–8. doi: 10.1158/1078-0432.CCR-07-4591.  [DOI](https://doi.org/10.1158/1078-0432.CCR-07-4591) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18483375/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&title=Efficacy%20of%20gemcitabine%20in%20patients%20with%20non-small%20cell%20lung%20cancer%20according%20to%20promoter%20polymorphisms%20of%20the%20ribonucleotide%20reductase%20M1%20gene&author=SO%20Kim&volume=14&publication_year=2008&pages=3083-8&pmid=18483375&doi=10.1158/1078-0432.CCR-07-4591&)

53. Rha SY, et al. An association between RRM1 haplotype and gemcitabine-induced neutropenia in breast cancer patients. Oncologist. 2007;12:622–30. doi: 10.1634/theoncologist.12-6-622.  [DOI](https://doi.org/10.1634/theoncologist.12-6-622) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17602053/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncologist&title=An%20association%20between%20RRM1%20haplotype%20and%20gemcitabine-induced%20neutropenia%20in%20breast%20cancer%20patients&author=SY%20Rha&volume=12&publication_year=2007&pages=622-30&pmid=17602053&doi=10.1634/theoncologist.12-6-622&)
