# Identification of Candidate Single-Nucleotide Polymorphisms in NRXN1 Related to Antipsychotic Treatment Response in Patients with Schizophrenia

## Metadata
**Authors:** Aaron Jenkins, José A Apud, Fengyu Zhang, Heather Decot, Daniel R Weinberger, Amanda J Law
**Journal:** Neuropsychopharmacology
**Date:** 2014 Apr 16
**DOI:** [10.1038/npp.2014.65](https://doi.org/10.1038/npp.2014.65)
**PMID:** 24633560
**PMCID:** PMC4104334
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4104334/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC4104334/pdf/npp201465a.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC4104334/pdf/npp201465a.pdf)

## Abstract

Neurexins are presynaptic neuronal adhesion molecules that interact with postsynaptic neuroligins to form an inter-synaptic complex required for synaptic specification and efficient neurotransmission. Deletions and point mutations in the neurexin 1 (NRXN1) gene are associated with a broad spectrum of neuropsychiatric and neurodevelopmental disorders, including autism, intellectual disability, epilepsy, developmental delay, and schizophrenia. Recently, small nucleotide polymorphisms in NRXN1 have been associated with antipsychotic drug response in patients with schizophrenia. Based on previous suggestive evidence of an impact on clozapine response in patients with schizophrenia, we conducted an association study of NRXN1 polymorphisms (rs12467557 and rs10490162) with antipsychotic treatment response in 54 patients with schizophrenia in a double blind, placebo-controlled NIMH inpatient crossover trial and examined for association with risk for schizophrenia in independent case-control and family-based clinical cohorts. Pharmacogenetic analysis in the placebo controlled trial revealed significant association of rs12467557and rs10490162 with drug response, whereby individuals homozygous for the A allele, at either SNP, showed significant improvement in positive symptoms, general psychopathology, thought disturbance, and negative symptoms, whereas patients carrying the G allele showed no overall response. Although we did not find evidence of the same NRXN1 SNPs being associated with results of the NIMH sponsored CATIE trial, other SNPs showed weakly positive signals. The family and case-control analyses for schizophrenia risk were negative. Our results provide confirmatory evidence of genetically determined differences in drug response in patients with schizophrenia related to NRXN1 variation. Furthermore, these findings potentially implicate NRXN1 in the therapeutic actions of antipsychotic drugs.

## INTRODUCTION

Schizophrenia is a chronic psychiatric illness, with complex neurodevelopmental and genetic origins ([Weinberger, 1987](#bib50); [Prathikanti and Weinberger, 2005](#bib36); [Sullivan et al, 2012](#bib46); [Owen, 2012](#bib35)). Positive symptoms (including delusions and hallucinations), negative symptoms (including lack of motivation and social withdrawal), and deficits in neurocognition represent core phenotypic features of the illness. Antipsychotic drugs, both typical and atypical, represent the mainstay of treatment for schizophrenia, but individual variability in drug efficacy, tolerability, and pharmacokinetics are major treatment challenges ([Lieberman et al, 2005](#bib25)). Pharmacogenetic studies of inter-individual variation in drug response have emerged as potentially powerful tools for improving therapeutic outcomes in psychiatry ([Malhotra et al, 2012a](#bib27); [Lotrich, 2012](#bib26)), through maximizing efficacy and minimizing antipsychotic side effects ([Malhotra et al, 2012b](#bib28)). Although dopamine-serotonin receptor antagonism is the primary pharmacological target of antipsychotic drug action ([Snyder, 1981](#bib41); [Kapur and Mamo, 2003](#bib19)), and polymorphisms in the dopamine D2 receptor and serotonin 5HT2C receptor have been associated with clinical response and adverse side effects ([Giegling et al, 2013](#bib13); [Zhang et al, 2010](#bib51); [Ikeda et al, 2008](#bib16)), recent pharmacogenetic studies highlight targets outside of the dopaminergic/serotonergic system, including NRXN1 ([Souza et al, 2010](#bib43); [Lett et al, 2011](#bib24)), KCNH2, GSK3, AKT1, and GRM8, as potential predictors of treatment response ([Apud et al, 2012](#bib1); [Souza et al, 2008](#bib42); [Need et al, 2009](#bib33)). Pharmacogenetic identification of drug response predictors has significant utility for the identification of novel treatment targets and potential for uncovering the molecular mechanisms of antipsychotic drug action.

Presynaptic neurexins (NRXN1–3) and partners, postsynaptic neuroligins (NLGN1–5), are trans-synaptic cell-adhesion, type I membrane proteins that connect presynaptic and postsynaptic synaptic specializations and influence neural network activity via physical regulation of synaptic transmission ([Sudhof, 2008](#bib45); [Varoqueaux et al, 2006](#bib49); [Zhang et al, 2010](#bib52)). Specifically, NRXNs are critical modulators of various neuronal processes, including the differentiation, maturation, stabilization, and plasticity of both excitatory and inhibitory synapses in the mammalian brain ([Sudhof, 2008](#bib45); [Varoqueaux et al, 2006](#bib49); [Zhang et al, 2010](#bib52)). Copy number variation, frameshift, and missense mutations in the NRXN1 gene [2p16.3] have been consistently linked to a broad spectrum of neurocognitive disorders, including autism spectrum disorders, Alzheimer's disease, intellectual disability, and schizophrenia ([Moller et al, 2013](#bib31); [Swaminathan et al, 2012](#bib48); [Duong et al, 2012](#bib11); [Bena et al, 2013](#bib3); [Dabell et al, 2013](#bib8); [Kirov et al, 2008](#bib21); [Rujescu et al, 2009](#bib39); [Kirov et al, 2009](#bib22); [Ikeda et al, 2010](#bib17)), highlighting a crucial role for NRXN1 in normal human neurodevelopment and neurocognitive disease. NRXN1 spans 1.12 Mb and encodes 23 canonical exons ([Figure 1](#fig1); [Missler et al, 1998](#bib29); [Rowen et al, 2002](#bib38)). Two alternative promoters encode a longer *α*-NRXN1 transcript and a shorter *β*-NRXN1 transcript, respectively, and extensive alternative splicing of the primary transcripts results in potentially thousands of evolutionarily conserved isoforms that are differentially expressed in the brain ([Figure 1](#fig1); [Sudhof, 2008](#bib45); [Missler et al, 1998](#bib29); [Rowen et al, 2002](#bib38)). Interestingly, the majority of CNVs reported in schizophrenia are nonrecurrent deletions clustering in the 5′ exonic promoter region encoding the *α*-NRXN1 transcript ([Kirov et al, 2008](#bib21); [Rujescu et al, 2009](#bib39); [Kirov et al, 2009](#bib22); [Ikeda et al, 2010](#bib17); [Grayton et al, 2012](#bib14)).

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9521/4104334/1b9206d52ab7/npp201465f1.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4104334_npp201465f1.jpg)

Organization of the human NRXN1 gene as described by Rowen et al (2002). The exon-intron structure is depicted (not to scale). The α-NRNX1 transcript is transcribed from the α promoter at the 5′end of the gene. Five alternative spicing sites (sites 1–5) are utilized in the gene. The approximate location of rs12467557 and rs10490162 are shown. Grey italics indicate alternative exons.

Recently, polymorphisms in *α*-NRXN1 have been associated with clozapine response in schizophrenia patients ([Souza et al, 2010](#bib43); [Lett et al, 2011](#bib24)) and nicotine dependence in healthy individuals ([Bierut et al, 2007](#bib4); [Nussbaum et al, 2008](#bib34)). Specifically, [Souza et al (2010)](#bib43) reported nominal evidence of association of polymorphisms rs12467557 and rs10490162 in intron 2 of *α*-NRXN1 ([Figure 1](#fig1)) with treatment response in patients with schizophrenia, defined as a 20% reduction on the overall Brief Psychiatric Rating Scale from baseline score at enrolment after 6 months of clozapine treatment. Whereby, subjects carrying major alleles (A) at both variants were more likely to respond to clozapine treatment. Unfortunately, trend significance, lack of placebo control, and lack of examination of effects of treatment on symptom subcategories limits interpretation of these data and warrants independent replication.

In this study, we conducted a pharmacogenetic analysis of antipsychotic drug response in patients with schizophrenia (*N*=54) in the Clinical Brain Disorders Branch/National Institute of Mental Health (NIMH) double-blind, placebo-controlled inpatient crossover trial and examined for association of the same two SNPs in NRXN1 (rs12467557 and rs10490162) with treatment response as measured by the Positive and Negative Syndrome Scale (PANSS; [Kay et al, 1991](#bib20); [Apud et al, 2012](#bib1)). In a secondary analysis, we evaluated association in the NIMH-sponsored multicenter Clinical Antipsychotic Trials in Intervention Effectiveness (CATIE) study, though the CATIE trial is a comparative efficacy trial and not a placebo-controlled therapeutic trial. Furthermore, we investigated NRXN1, rs12467557, and rs10490162 for clinical genetic association to schizophrenia. Our findings demonstrate significant association of NRXN1, rs12467557, (and rs10490162) with antipsychotic drug response in the NIMH cohort. Neither rs12467557 nor rs10490162 were significantly associated with risk for schizophrenia in the NIMH cohort (*N*=356 families; 445 patients and 488 healthy controls) or outcome measures in the CATIE trial.

## MATERIALS AND METHODS

### Antipsychotic Treatment Response Study Design and Subjects

#### NIMH study cohort

Fifty-four patients with schizophrenia, admitted to the Clinical Brain Disorders Branch schizophrenia research inpatient unit at the NIH Clinical Center between 1998 and 2010, were included in this study (DR Weinberger, PI; see [Apud et al, 2012](#bib1)). All subjects were Caucasian of European ancestry and were diagnosed with chronic schizophrenia using DSM-IV criteria. Subjects volunteered to participate in a double-blind, placebo-controlled, crossover study with atypical antipsychotics, including olanzapine, risperidone, quetiapine, ziprasidone, or aripiprazole. All patients, and a family member if necessary, provided informed consent for participation in the study, which had been approved by the NIH Institutional Review Board. The program typically enrolled patients who had already had adequate trials of standard antipsychotic medication interventions but were only partially effective in decreasing their symptoms. Upon admission, all patients underwent medical, neurological, and psychiatric evaluation usually lasting 4–12 weeks. If the patient had significant medical problems, a history of violence, or suicidal behavior not identified during the prehospitalization screening, or if that patient decided to withdraw his/her consent, he/she could leave the hospital or participate in other protocols not involving placebo studies.

All patients included in the study were on atypical antipsychotic medications before their admission. After the initial evaluation period, patients were maintained on a standard dose of one of various atypical antipsychotics (noted above) and all other medications were discontinued. The decision about which antipsychotic medication would be used in the blinded study was based on best previous response, patient preference, and side effect profile. The patients remained on the single antipsychotic medication in an open-label fashion for several weeks before transitioning to coded medication of the same specific drug and were tapered from their medication over a period of 4-7 days. Subgroup 1 (*N*=35) underwent a sequence of 4 weeks of coded placebo followed by 4 weeks of a coded standard atypical antipsychotic. Subgroup 2 (*N*=19) underwent the inverse sequence (ie, after a similar open medication and taper period, they received standard atypical antipsychotic treatment for 4 weeks followed by placebo for 4 weeks; (as described previously, [Apud et al, 2012](#bib1))). Because of the highly supervised and structured nature of the NIMH inpatient environment, most patients tolerated the 4 weeks of placebo well. At the end of the coded double-blind protocol, the patients were restarted on an active medication for 6 more weeks before starting another protocol, or until they were well enough to be discharged.

Upon admission, each patient was rated once on the 16-item PANSS scale ([Kay et al, 1991](#bib20)). Thereafter, weekly PANSS rating were performed independently (by one of the four research nurses). Two weeks before starting the protocol and up to 4 weeks after completing the protocol, ratings were performed twice a week. The mean PANSS subscores before the patients entered the placebo-controlled protocol were as follows: 14.25 positive Syndrome (SD=4.09); 15.69 for negative Syndrome (SD=4.6); 29.58 for general psychopathology (SD=5.81); 8.48 for anergia (SD=2.23); 8.36 for thought disturbance (SD=3.61); 5.08 for activation (SD=1.31); 4.89 for paranoid/belligerence (SD=0.85); and 8.40 for depression (SD=2.18).

#### CATIE study cohort

Data from CATIE study was used to test for association of SNPs in NRXN1, although the designs of two studies are notably different as the CATIE trial is a comparative effectiveness trial, not a therapeutic or placebo-controlled trial. Only subjects involved with phase 1/1A of the CATIE study, in which patients were randomly assigned to one of the five medications (the first drug assigned), were included. To gain adequate power and to control for the genetic heterogeneity, only individuals of European ancestry were included in analysis (*N*=418). The detailed sample and approaches to analysis of these data have been described elsewhere ([Apud et al, 2012](#bib1)) and more details on the CATIE study have been published ([Lieberman et al, 2005](#bib25)). In brief, the primary outcome was time to discontinued use of antipsychotic medication and the secondary outcome measure was change in PANSS ratings (positive syndrome, negative syndrome, and general psychopathology and combined score) before and after initiation of the trial.

#### Clinical cohorts

Family-based and case-control samples were used for clinical genetic investigation of NRXN1 polymorphisms. The samples were ascertained as part of the Clinical Brain Disorders/National Institute of Mental Health (NIMH) Sibling Study (CBDB/SS, DR Weinberger, PI). DNA was available from 445 probands, 400 siblings of probands, 612 parents, and 488 unrelated controls (as previously described in detail, [Law et al, 2012](#bib23)). All probands met DSM-IV criteria for a broad diagnosis category consisting of schizophrenia, schizoaffective disorder, simple schizophrenia, psychosis NOS, delusional disorder, schizotypal, schizoid, or paranoid personality disorder. Control subjects were ascertained from the NIMH normal volunteer office and required absence of diagnosis of a psychiatric disorder, extended to include first-degree relatives. All subjects were interviewed with the structured diagnostic evaluation for DSM-IV diagnoses (SCID interview) and were examined for medical and psychiatric exclusions as described elsewhere ([Law et al, 2012](#bib23)). For family-based association analysis, 356 families with a single affected proband were available. A partially independent case-control analysis was used comprising 445 unrelated probands and 488 unrelated healthy controls. Inclusion criteria for all participants were: self-identification as Caucasian (mostly European ancestry), aged between 18 and 60 years and IQ scores >70 (for probands, premorbid IQ). All subjects gave written informed consent. Patients were also interviewed by a clinical psychiatrist or psychologist for evaluation of ongoing symptoms using the PANSS scale.

### Genotyping and Statistical Analysis

In the NIMH inpatient cohort, we genotyped two polymorphisms in NRXN1 (rs10490162 and rs12467557), which are nominally associated with clozapine response in patients with schizophrenia ([Souza et al, 2010](#bib43)) and nicotine dependence ([Bierut et al, 2007](#bib4); [Nussbaum et al, 2008](#bib34)). rs10490162 and rs12467557 are located in intron 2 of the NRXN1 gene and are 5.3 kb apart and in weak-to-moderate LD (*r*^2^=0.52; *D*'=1). Genotypes were determined using the Taqman 5′-exonuclease fluorescent assay (details available upon request) and the allelic discrimination was read on ABI 7900 SDS systems (Applied Biosystems, Foster City, CA). SNP genotypes from the CATIE study that mapped to the NRXN1 genic region (*N*=269) were acquired from a genome-wide association study of schizophrenia ([Sullivan et al, 2008](#bib47)). Because rs124677557 was not genotyped in the original study, we used a proxy SNP rs17041184, which was in complete linkage disequilibrium (*D*'=1 and *r*^2^=1). Of the 269 SNPs in the GWAS data set mapping to the NRXN1 locus, 73 can be considered independent variants (*r*^2^<0.2).

Statistical analyses were performed based on the specific study and measurement outcomes. In the NIMH inpatient study, which is a within-subject crossover design with repeated measures, we used a general linear mixed model to examine the effects of treatment, genotype and their interaction on PANSS rating. CATIE data were analyzed using the Cox proportional hazard model and general linear mixed model, respectively, to examine the association of NRXN1 genotype with time to discontinued use of medication and PANSS ratings. Analysis was performed while controlling for age, years of treatment, and CPZEQ. Drug clearance was important in evaluating the antipsychotic drug response, and clearance data were obtained through an ancillary project to the CATIE study ([Bigos et al, 2008](#bib6), [2011](#bib5)). For clinical genetic association, main effect analyses of single SNPs were conducted using unconditional logistic regression models controlling for sex and age in the case-control sample and using the family-based association test (FBAT, [http://www.biostat.harvard.edu/fbat/fbat.htm?](http://www.biostat.harvard.edu/fbat/fbat.htm?)) in families with permutation testing for significance assessment.

## RESULTS

### Antipsychotic Treatment Response in the CBDB/NIMH Cohort

Treatment effects were observed on several neuropsychiatric symptoms and syndromal clusters ascertained from the PANSS ratings ([Supplementary Table S1](#sup1) and [Apud et al, 2012](#bib1)). Overall, medications significantly improved symptoms, including those comprising the positive syndrome (Estimate=−1.3565, *p*=0.0007), general psychopathology (Estimate=−2.2851, *p*=0.0093), thought disturbance (Estimate=−0.8928, *p*=0.0004), and activation (Estimate=−0.6841, *p*=0.0018). A suggestive effect on the negative syndrome was also observed (Estimate=−1.0836, *p*=0.0124) but was not significant after correction for multiple testing.

### NRXN1 Polymorphisms and Antipsychotic Treatment Response in the CBDB/NIMH Cohort

We observed a significant interaction between rs12467557 genotypes and antipsychotic treatment response on improvement of PANSS ratings of the negative syndrome (F=4.27; *p*=0.044), positive syndrome (F=4.89; *p*=0.032); general psychopathology (F=9.96; *p*=0.0029); anergia (F=5.6; *p*=0.022), and thought disturbance (F=7.74; *p*=0.008). These effects are particularly noteworthy considering the relatively small sample size in our study. Similar but less-significant interactions were observed for rs10490162 ([Table 1](#tbl1)). Genotypic groups did not significantly differ on any demographic variable, including sex, age, IQ, and antipsychotic drug dose, in the treatment arm of the trial (Table 3). In addition, the genotypic groups did not differ with regards to smoking status ([Table 2](#tbl2)), importantly controlling for previously identified associations of these polymorphisms to smoking dependence in normal individuals ([Bierut et al, 2007](#bib4); [Nussbaum et al, 2008](#bib34)).

### Table 1. Antipsychotic Treatment Response by Genotype as Assessed by the Positive and Negative Syndrome Scale (PANSS).

|   | NRXN1_rs12467557 |  |  |  | NRXN1_rs10490162 |  |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- |
|   | Estimatea | t | p | p* | Estimatea | t | p | p* |
| Negative syndrome |   |   |   | 0.0447 |   |   |   | 0.0863 |
| AA | −1.45 | −2.94 | 0.0051 |   | −1.57 | −2.98 | 0.0047 |   |
| AG+GG | 1.80 | 1.19 | 0.2399 |   | 0.54 | 0.5 | 0.6219 |   |
| Positive syndrome |   |   |   | 0.0324 |   |   |   | 0.2319 |
| AA | −1.35 | −3.29 | 0.002 |   | −1.34 | −2.99 | 0.0045 |   |
| AG+GG | 1.55 | 1.23 | 0.2265 |   | −0.10 | −0.11 | 0.9139 |   |
| General psychopathology |   |   |   | 0.0029 |   |   |   | 0.1397 |
| AA | −3.01 | −2.96 | 0.005 |   | −2.94 | −2.54 | 0.0146 |   |
| AG+GG | 7.30 | 2.32 | 0.025 |   | 1.01 | 0.43 | 0.6728 |   |
| Anergia |   |   |   | 0.0223 |   |   |   | 0.0225 |
| AA | −0.58 | −2.01 | 0.0505 |   | −0.70 | −2.29 | 0.0266 |   |
| AG+GG | 1.60 | 1.81 | 0.0772 |   | 0.95 | 1.5 | 0.1409 |   |
| Thought disturbance |   |   |   | 0.0080 |   |   |   | 0.1187 |
| AA | −0.95 | −3.51 | 0.001 |   | −0.95 | −3.16 | 0.0029 |   |
| AG+GG | 1.45 | 1.75 | 0.0873 |   | 0.14 | 0.23 | 0.8186 |   |
| Activation |   |   |   | 0.1600 |   |   |   | 0.4347 |
| AA | −0.73 | −2.98 | 0.0046 |   | −0.73 | −2.78 | 0.008 |   |
| AG+GG | 0.39 | 0.52 | 0.6082 |   | −0.26 | −0.47 | 0.6372 |   |
| Paranoid belligerence |   |   |   | 0.0459 |   |   |   | 0.2725 |
| AA | −0.26 | −1.58 | 0.1212 |   | −0.25 | −1.43 | 0.1598 |   |
| AG+GG | 0.81 | 1.63 | 0.1115 |   | 0.19 | 0.53 | 0.596 |   |
| Depression |   |   |   | 0.1302 |   |   |   | 0.6567 |
| AA | −0.62 | −1.59 | 0.1183 |   | −0.56 | −1.34 | 0.1869 |   |
| AG+GG | 1.30 | 1.09 | 0.2832 |   | −0.14 | −0.16 | 0.8747 |   |

Table 1 Caption: p*, p values for interaction between SNPs and treatment on PANSS ratings adjusted for duration of illness. A=common allele, G=minor allele. aEstimates reflects mean difference in PANSS ratings between placebo and treatment arms by genotype based on post-hoc least-square estimation from the general linear mixed model.

### Table 2. Demographic Characteristics of Schizophrenia Patients by NRXN1 Genotype rs10490162.

| Characteristic | AA |  |  | AG/GG |  |  |
| --- | --- | --- | --- | --- | --- | --- |
|   | N | Mean | SD | N | Mean | SD |
| Sex (female %) | 42 | 24% | 0.43% | 12 | 0.33% | 0.49% |
| Age (years) | 42 | 28.07 | 7 | 12 | 27.75 | 6.426 |
| Age at onset (years) | 42 | 20.64 | 4.51 | 12 | 21.75 | 4.957 |
| Education (years) | 42 | 13.64 | 2.03 | 12 | 13.83 | 1.85 |
| Antipsychotic dose (CPZ equivalents; treatment arm) | 37 | 605.67 | 296.02 | 12 | 558.64 | 276.78 |
| Smoker (%) | 40 | 0.48 | 0.51 | 12 | 0.58 | 0.515 |
| Past smoker (%) | 40 | 0.63 | 0.49 | 12 | 0.67 | 0.492 |
| Duration of illness (years) | 42 | 7.9 | 6.44 | 12 | 6.5 | 6.882 |
| Full-scale IQ | 42 | 90.45 | 13.6 | 12 | 99 | 11.747 |

Table 2 Caption: Note: No statistical significant difference in demographic characteristics were observed (p>0.05).

*Post-hoc* analyses revealed that individuals homozygous for the major allele (A) at rs12467557 (or rs10490162) showed a superior response to antipsychotic treatment ([Table 1](#tbl1)). Specifically, AA genotype individuals had significant reductions in ratings of positive syndrome (Estimate=−1.45, *p*=0.002), negative syndrome (Estimate=−1.45, *p*=0.005), general psychopathology (Estimate=−3.01, *p*=0.005), thought disturbance (Estimate=−0.95, *p*=0.001), and activation (Estimate=−0.73, *p*=0.005) when in the medicated phase of the trial. It is noteworthy that the magnitude of the genotype effect on the change in each of these syndrome scores was greater than the overall effect of medication. Subjects carrying minor alleles (GG+AG) at the same variants did not exhibit any significant improvement based on measures of treatment response ([Table 1](#tbl1)). Importantly, PANSS ratings were not different for the AA *vs* AG+GG genotypic groups in the placebo *vs* placebo phase, demonstrating that the AA individuals were not significantly differentially more symptomatic compared with the other genotypes. It is important to note that because in these analyses we treat genotype as categorical variables, the regression coefficient estimate actually denotes the mean chance in PANSS rating after adjusting for covariates. Finally, we also performed haplotype analyses of two SNPs, which did not increase the strength of the association (data not shown), likely because these SNPs are in weak LD in this sample (*r*^2^=0.4).

### NRXN1 Polymorphisms and Antipsychotic Treatment Response in the CATIE Study

We failed to replicate association of either rs17041184 or rs10490162 with PANSS ratings (general psychopathology, negative, and positive) in the CATIE data set. In the analysis of three PANSS ratings using the same coding as the placebo controlled data ([Table 3](#tbl3)), we did not found any significant difference in the PANSS scores before and after treatment by genotype group, except that rs10490162 was trend associated with negative PANSS rating (*p*=0.088). Analysis of time to discontinued use of medication showed that minor allele carriers of rs17041184 (OR=0.526, *P*=0.0418) were less likely to discontinue before the end of the phase 1/1A trial ([Supplementary Table S2](#sup1)).

### Table 3. Least Square Mean Estimates of Three PANSS Ratings by Genotype Groups in the European Ancestry Sample of CATIE Study.

| Genotype | General psychopathology |  |  |  | Negative PANSS |  |  |  | Positive |  |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
|   | Estimate | SE | p | p* | Estimate | SE | p | p* | Estimate | SE | p | p* |
| rs17041184a |  |  |  |  |  |  |  |  |  |  |  |  |
| TT | 2.5426 | 0.477 | <0.0001 | 0.6116 | 0.944 | 0.3135 | 0.0028 | 0.3621 | 1.4958 | 0.3056 | <0.0001 | 0.5685 |
| CT+CC | 1.8205 | 1.3385 | 0.1745 |   | 1.797 | 0.8808 | 0.042 |   | 0.9758 | 0.8582 | 0.2562 |   |
|   |   |   |   |   |   |   |   |   |   |   |   |   |
| rs10490162 |  |  |  |  |  |  |  |  |  |  |  |  |
| AA | 2.1518 | 0.501 | <0.0001 | 0.1835 | 0.8123 | 0.3276 | 0.0136 | 0.0888 | 1.2985 | 0.32 | <0.0001 | 0.3871 |
| AG+GG | 3.6357 | 0.9947 | 0.0003 |   | 2.0560 | 0.6513 | 0.0017 |   | 1.9148 | 0.636 | 0.0028 |   |

Table 3 Caption: Note: p, p value for treatment effect in each genotype group from the post-hoc analysis based on general linear mixed model; p*, p value for genotype and treatment interaction to test whether or not antipsychotic response was different by genotype group. ars17041184 is a proxy SNP of rs12467557 (r2=1 and D'=1), which was not genotyped in CATIE data.

In a subsidiary analysis, we explored additional SNPs in the genic region of NRXN1 and found nominal association with both PANSS ratings and time to discontinued use of medication for several SNPs. Eleven SNPs were associated with time to discontinuation (*p*<0.1; [Table 4](#tbl4)), which also showed association signals in the combined *p* values of three PANSS syndromes (*p*<0.05), and association with time to discontinued use of medication was observed at three intronic SNPs rs7568888 (*p*=0.00574), rs10176824 (*p*=0.00173), and rs17496470 (*p*=0.01395); the association of these three SNPs with the change in PANSS ratings between before and after treatment were also significant. Two other SNPs, rs6545111 (*p*=0.005209) and rs1718049 (*p*=0.00848), also showed association with negative symptoms ([Table 4](#tbl4)). We note, however, that none of these associations would survive correction for multiple testing. The detailed estimates can be found in [Supplementary Table S3](#sup1). Among these 11 SNPs, 2 were in strong LD and 2 were in moderate LD (see [Table 4](#tbl4)). The other seven SNPs, including those showing the most consistent association, were relatively independent (*r*^2^<0.2).

### Table 4. Association of NRXN1 SNPs with Time to Discontinuation (p<0.1) and PANSS Rating (Comb p<0.05) in the CATIE Study.

| Chr | Gene | Location | BP(HG18) | SNP | distance | Discontinuation | PANSS ratings |  |  | comb3_p |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
|   |   |   |   |   |   | P* | p_gen | p_neg | p_pos |   |
| 2 |   |   | 49553295 | rs6545111 |   | 0.07746 | 0.194454 | 0.005209 | 0.187465 | 0.001789 |
| 2 | — | Upstream | 49768414 | rs17180439 | 453872 | 0.06720 | 0.102235 | 0.008480 | 0.118872 | 0.000769 |
| 2 | — | Upstream | 49780074 | rs1518826 | 465532 | 0.02267 | 0.159074 | 0.045965 | 0.098444 | 0.003508 |
| 2 |   |   | 49845664 | rs1914791 |   | 0.04683 | 0.163218 | 0.047506 | 0.405138 | 0.015106 |
| 2 | — | Upstream | 49810789 | rs10211040 | 496247 | 0.04346 | 0.126967 | 0.036632 | 0.011133 | 0.000380 |
| 2 | NRXN1 | Intron | 50099333 | rs7568888 | 0 | 0.00574 | 0.118042 | 0.188359 | 0.105221 | 0.009398 |
| 2 | NRXN1 | Intron | 50115890 | rs10176824 | 0 | 0.00173 | 0.164830 | 0.092664 | 0.014243 | 0.001433 |
| 2 | NRXN1 | Intron | 50150244 | rs17496470 | 0 | 0.01395 | 0.102509 | 0.489385 | 0.180992 | 0.034503 |
| 2 | NRXN1 | Intron | 50547858 | rs6726487 | 0 | 0.02236 | 0.127715 | 0.562803 | 0.187808 | 0.048390 |
| 2 | — | Upstream | 51420208 | rs17524147 | 311299 | 0.03482 | 0.253534 | 0.841519 | 0.029657 | 0.042337 |
| 2 | — | Upstream | 51435011 | rs17590125 | 326102 | 0.04651 | 0.126366 | 0.576434 | 0.049158 | 0.019242 |

Table 4 Caption: Note: SNPs rs1518826 and rs10211040 are in moderate LD (r2=0.79) and rs17524147 and rs17590125 are in strong LD (r2=0.93). Thus, there are nine relatively independent signals in the analysis. p*, p value for testing for genetic variation with time to discontinuation; p_gen, p_neg and p_pos are p values for genotype × treatment interactions on PANSS general psychopathology, negative and positive symptom, respectively, using general linear mixed model; comb3_p, combined p values of three PANSS symptoms using Stouffer's Z-score method.

### Association of NRXN1 Polymorphisms with Schizophrenia

Single marker analysis in the family-based sample revealed no evidence of association with rs12467557 or rs10490162 (*p*=0.33; *p*=0.54, respectively). Likewise, we failed to observe evidence of association of these SNPs with schizophrenia in our case-control sample (rs12467557, *p*=0.74; rs10490162, *p*=0.959). Both SNPS were in Hardy-Weinberg equilibrium in both cases and controls (*p*>0.7) and were in moderate linkage disequilibrium (*r*^2^=0.522), consistent with that observed in the NIMH treatment response cohort.

## DISCUSSION

Our study provides statistically significant evidence of association of NRXN1 rs12467557 and rs10490162 with atypical antipsychotic treatment response in a placebo-controlled, in-patient treatment response study. Schizophrenia patients homozygous for major alleles at either variant (rs10490162 and rs12467557) were more likely to show favorable treatment response as opposed to those carrying minor alleles, including improvement of negative symptoms. Notably, the effect size of the pharmacogenetic association is greater than the effect of antipsychotic treatment, *per se*. These results are of particular interest given the relatively small sample size and tightly controlled nature of this study and are consistent with previous data suggestive of an impact of these SNPs on clozapine response in an uncontrolled trial ([Souza et al, 2010](#bib43)).

Our failure to find association with risk for schizophrenia in our family and case-control data sets is consistent with the negative signals for common variant associations in much larger studies ([Ripke et al, 2013](#bib37)). Given the recent findings of association of rare, highly penetrant NRXN1 CNVs with schizophrenia ([Kirov et al, 2008](#bib21); [Rujescu et al, 2009](#bib39); [Kirov et al, 2009](#bib22); [Ikeda et al, 2010](#bib17)), it is perhaps not surprising that we fail to find significant association of either common variant with risk for illness. Most of the recurrent CNVs that have been associated with schizophrenia appear to represent biological risk that is not represented by common variants at the same loci.

Over the past several years, evidence has emerged suggesting that CNVs have a role in the etiology of neurodevelopmental disorders, such as schizophrenia, autism, and intellectual disability ([Kirov et al, 2008](#bib21); [Rujescu et al, 2009](#bib39); [Kirov et al, 2009](#bib22); [Ikeda et al, 2010](#bib17); [Muhleisen et al, 2011](#bib32); [Stefansson et al, 2008](#bib44); [Bassett et al, 2008](#bib2); [International Schizophrenia Consortium, 2008](#bib18)). Deletions within NRXN1 were first reported in schizophrenia by Kirov *et al* in 2008, whereby a deletion spanning the proximal promoter and first exon of *α*-NRXN1 was observed in a single proband and an affected sibling with schizophrenia. Subsequently, chromosomal deletions disrupting NRXN1 have been replicated by a number of groups ([Rujescu et al, 2009](#bib39); [Kirov et al, 2009](#bib22); [Ikeda et al, 2010](#bib17); [Muhleisen et al, 2011](#bib32); [Stefansson et al, 2008](#bib44); [Bassett et al, 2008](#bib2); [International Schizophrenia Consortium, 2008](#bib18)). A meta-analysis of these studies revealed strong evidence that deletions of the NRXN1 gene that impact the proximal region of *α*-NRXN1 confer a substantial increase in risk of schizophrenia, which further increases when restricted to functional deletions>100 kb ([Kirov et al, 2009](#bib22)).

Intron 2 of NRXN1-*α*, which contains rs12467557 and rs10490162 ([Figure 1](#fig1)), is in the region most frequently disrupted by CNVs in schizophrenia ([Kirov et al, 2009](#bib22); [Muhleisen et al, 2011](#bib32); [Stefansson et al, 2008](#bib44); [Chen et al, 2013](#bib7)). The intron spans 101 kb, making it one of the largest segments within the gene ([Rowen et al, 2002](#bib38)). Its large size, taken together with its high level of sequence conservation throughout vertebrates, suggests that it has played an important role in the evolution of the NRXN1 gene. Introns are ubiquitous throughout the human genome and have been postulated to fulfill a variety of cellular functions, including carriers of transcriptional regulatory elements as well as key players in alternative splicing ([Fedorova and Fedorov, 2003](#bib12)). Large introns (>50 kb) show an abundance of interspersed repetitive elements (SINEs and LINEs) that form stable structures, which have been hypothesized to bring donor and splicing junctions closer together, thus enhancing splicing efficiency ([Shepard et al, 2009](#bib40)). Consistent with this, examination of the UCSC genome bioinformatics suite incorporating the ENCODE Regulation track reveals that rs10490162 lies 226 bp from a LINE (L1ME3) element (chr2:51247202-51247431; GRCh37/hg19). Recent evidence also demonstrates that small-size inverted repeats are overrepresented in the 5′ portion of NRXN1 and its immediate upstream region, indicative of a region of genomic instability and a mechanism underlying the region as a deletion hotspot in neurocognitive and psychiatric disease ([Chen et al, 2013](#bib7)).

At present the mechanism of association of NRXN1 polymorphisms with antipsychotic drug response is unclear. *In-silico* evidence suggests that the region monitored by these SNPs has a role in splicing or regulation of expression of the gene. Indeed, primary NRXN transcripts undergo complex alternative splicing that is predicted to give rise to>1000 distinct isoforms ([Missler et al, 1998](#bib29); [Rowen et al, 2002](#bib38)). *α*-NRXNs contain five canonical splice sites, referred to as 1–5, with only 4 and 5 shared by *β*-NRXNs ([Figure 1](#fig1); [Rowen et al, 2002](#bib38)). Splice site 1 in NRXN1-*α* involves splicing of exons 3, 4, and 5, which reside in intron 2, placing the splice site in direct proximity to rs12467557 and rs10490162 ([Figure 1](#fig1)). Interestingly, rs10490162 lies within a spliced EST cloned from cerebellum, [DA128541](https://www.ncbi.nlm.nih.gov/nuccore/DA128541) (chr2:51247164-51247752). The general significance of these observations is at present unclear given that association was suggested to polymorphisms in the 3′ intronic region of NRXN1 in the CATIE data set, a region common to both NRXN1-*α* and NRXN1-*β*. Deep re-sequencing of the NRXN1 association regions, combined with experimental verification of functionality, is required to determine whether genotyped SNPs are in linkage disequilibrium with unknown functional/causal variants or are themselves functional.

Although there is no direct evidence that NRXN signaling is a molecular regulator of antipsychotic drug activity, NRXNs are expressed in brain areas that control the activity of dopaminergic signaling ([Hishimoto et al, 2007](#bib15)), the primary mechanism of antipsychotic drug action and a signaling mechanism, which may explain association of NRXN1 and NRXN3 with nicotine and substance dependence ([Bierut et al, 2007](#bib4); [Nussbaum et al, 2008](#bib34); [Hishimoto et al, 2007](#bib15); [Docampo et al, 2012](#bib9)). Furthermore, NRXN1-*α* has been shown to be critical for calcium-mediated neurotransmitter release and the regulation of voltage-dependent Ca^2+^ channel activity, via coupling of Ca^2+^ channels to synaptic vesicle release ([Zhang et al, 2010](#bib52); [Missler et al, 2003](#bib30); [Dudanova et al, 2006](#bib10)); thus it is plausible that NRXN1 may directly impact dopaminergic signaling in brain.

Finally, although we failed to replicate direct association of rs10490162 and rs12467557 with outcome measures in the CATIE data set, we present nominal evidence of association with other common variants in the NRXN1 gene with both time to discontinuation of medication and PANSS scores, providing additional support for association of the NRXN1 gene with antipsychotic treatment response in patients with schizophrenia. We note that failure to replicate the exact SNP association (observed herein in the CBDB/NIMH study and in the study of [Souza et al, 2010](#bib43)) in the CATIE data set should not be overinterpreted. The two trials used in this study were performed using vastly different clinical and experimental models and with different outcome measures, ie, CATIE, an outpatient parallel, comparative cohort, drug-only design and the other a double-blind, placebo-controlled inpatient crossover study, which ultimately makes direct comparison between the studies difficult.

Future studies examining the association of NRXN1 polymorphisms with antipsychotic treatment response are warranted by our results. The elucidation of the functional significance of variants within the NRXN1 gene may reveal new pharmacological targets for the development of novel therapeutic strategies for schizophrenia. Such discoveries may be vitally important given the limited efficacy of current antipsychotics to ameliorate symptoms associated with the disease and the need for a new generation of psychiatric drugs.

## FUNDING AND DISCLOSURE

The authors declare no conflict of interest.

## Acknowledgments

This work was supported by funds from the Intramural Research Program of the National Institutes of Mental Health, National Institutes of Health. We thank Bhaskar Kolachana for genotyping of the NIMH samples.

## Footnotes

## Supplementary Material

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Apud JA, Zhang F, Decot H, Bigos KL, Weinberger DR. Genetic variation in KCNH2 associated with expression in the brain of a unique hERG isoform modulates treatment response in patients with schizophrenia. Am J Psychiatry. 2012;169:725–734. doi: 10.1176/appi.ajp.2012.11081214.  [DOI](https://doi.org/10.1176/appi.ajp.2012.11081214) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22706279/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Psychiatry&title=Genetic%20variation%20in%20KCNH2%20associated%20with%20expression%20in%20the%20brain%20of%20a%20unique%20hERG%20isoform%20modulates%20treatment%20response%20in%20patients%20with%20schizophrenia&author=JA%20Apud&author=F%20Zhang&author=H%20Decot&author=KL%20Bigos&author=DR%20Weinberger&volume=169&publication_year=2012&pages=725-734&pmid=22706279&doi=10.1176/appi.ajp.2012.11081214&)

2. Bassett AS, Marshall CR, Lionel AC, Chow EW, Scherer SW. Copy number variations and risk for schizophrenia in 22q11.2 deletion syndrome. Hum Mol Genet. 2008;17:4045–4053. doi: 10.1093/hmg/ddn307.  [DOI](https://doi.org/10.1093/hmg/ddn307) | [PMC free article](/articles/PMC2638574/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18806272/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mol%20Genet&title=Copy%20number%20variations%20and%20risk%20for%20schizophrenia%20in%2022q11.2%20deletion%20syndrome&author=AS%20Bassett&author=CR%20Marshall&author=AC%20Lionel&author=EW%20Chow&author=SW%20Scherer&volume=17&publication_year=2008&pages=4045-4053&pmid=18806272&doi=10.1093/hmg/ddn307&)

3. Bena F, Bruno DL, Eriksson M, van Ravenswaaij-Arts C, Stark Z, Dijkhuizen T, et al. Molecular and clinical characterization of 25 individuals with exonic deletions of NRXN1 and comprehensive review of the literature. Am J Med Genet B Neuropsychiatr Genet. 2013;162B:388–403. doi: 10.1002/ajmg.b.32148.  [DOI](https://doi.org/10.1002/ajmg.b.32148) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23533028/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Med%20Genet%20B%20Neuropsychiatr%20Genet&title=Molecular%20and%20clinical%20characterization%20of%2025%20individuals%20with%20exonic%20deletions%20of%20NRXN1%20and%20comprehensive%20review%20of%20the%20literature&author=F%20Bena&author=DL%20Bruno&author=M%20Eriksson&author=C%20van%20Ravenswaaij-Arts&author=Z%20Stark&volume=162B&publication_year=2013&pages=388-403&pmid=23533028&doi=10.1002/ajmg.b.32148&)

4. Bierut LJ, Madden PA, Breslau N, Johnson EO, Hatsukami D, Pomerleau OF, et al. Novel genes identified in a high-density genome wide association study for nicotine dependence. Hum Mol Genet. 2007;16:24–35. doi: 10.1093/hmg/ddl441.  [DOI](https://doi.org/10.1093/hmg/ddl441) | [PMC free article](/articles/PMC2278047/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17158188/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mol%20Genet&title=Novel%20genes%20identified%20in%20a%20high-density%20genome%20wide%20association%20study%20for%20nicotine%20dependence&author=LJ%20Bierut&author=PA%20Madden&author=N%20Breslau&author=EO%20Johnson&author=D%20Hatsukami&volume=16&publication_year=2007&pages=24-35&pmid=17158188&doi=10.1093/hmg/ddl441&)

5. Bigos KL1, Bies RR, Pollock BG, Lowy JJ, Zhang F, Weinberger DR. Genetic variation in CYP3A43 explains racial difference in olanzapine clearance. Mol Psychiatry. 2011;16:620–625. doi: 10.1038/mp.2011.38.  [DOI](https://doi.org/10.1038/mp.2011.38) | [PMC free article](/articles/PMC3100476/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21519338/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Psychiatry&title=Genetic%20variation%20in%20CYP3A43%20explains%20racial%20difference%20in%20olanzapine%20clearance&author=KL1%20Bigos&author=RR%20Bies&author=BG%20Pollock&author=JJ%20Lowy&author=F%20Zhang&volume=16&publication_year=2011&pages=620-625&pmid=21519338&doi=10.1038/mp.2011.38&)

6. Bigos KL1, Pollock BG, Coley KC, Miller DD, Marder SR, Aravagiri M, et al. Sex, race, and smoking impact olanzapine exposure. J Clin Pharmacol. 2008;48:157–165. doi: 10.1177/0091270007310385.  [DOI](https://doi.org/10.1177/0091270007310385) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18199892/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Pharmacol&title=Sex,%20race,%20and%20smoking%20impact%20olanzapine%20exposure&author=KL1%20Bigos&author=BG%20Pollock&author=KC%20Coley&author=DD%20Miller&author=SR%20Marder&volume=48&publication_year=2008&pages=157-165&pmid=18199892&doi=10.1177/0091270007310385&)

7. Chen X, Shen Y, Zhang F, Chiang C, Pillalamarri V, Blumenthal I, et al. Molecular analysis of a deletion hotspot in the NRXN1 region reveals the involvement of short inverted repeats in deletion CNVs. Am J Hum Genet. 2013;92:375–386. doi: 10.1016/j.ajhg.2013.02.006.  [DOI](https://doi.org/10.1016/j.ajhg.2013.02.006) | [PMC free article](/articles/PMC3591860/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23472757/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=Molecular%20analysis%20of%20a%20deletion%20hotspot%20in%20the%20NRXN1%20region%20reveals%20the%20involvement%20of%20short%20inverted%20repeats%20in%20deletion%20CNVs&author=X%20Chen&author=Y%20Shen&author=F%20Zhang&author=C%20Chiang&author=V%20Pillalamarri&volume=92&publication_year=2013&pages=375-386&pmid=23472757&doi=10.1016/j.ajhg.2013.02.006&)

8. Dabell MP, Rosenfeld JA, Bader P, Escobar LF, El-Khechen D, Vallee SE, et al. Investigation of NRXN1 deletions: clinical and molecular characterization. Am J Med Genet. 2013;161:717–731. doi: 10.1002/ajmg.a.35780.  [DOI](https://doi.org/10.1002/ajmg.a.35780) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23495017/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Med%20Genet&title=Investigation%20of%20NRXN1%20deletions:%20clinical%20and%20molecular%20characterization&author=MP%20Dabell&author=JA%20Rosenfeld&author=P%20Bader&author=LF%20Escobar&author=D%20El-Khechen&volume=161&publication_year=2013&pages=717-731&pmid=23495017&doi=10.1002/ajmg.a.35780&)

9. Docampo E, Ribases M, Gratacos M, Bruguera E, Cabezas C, Sanchez-Mora C, et al. Association of neurexin 3 polymorphisms with smoking behavior. Genes Brain Behav. 2012;11:704–711. doi: 10.1111/j.1601-183X.2012.00815.x.  [DOI](https://doi.org/10.1111/j.1601-183X.2012.00815.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22716474/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genes%20Brain%20Behav&title=Association%20of%20neurexin%203%20polymorphisms%20with%20smoking%20behavior&author=E%20Docampo&author=M%20Ribases&author=M%20Gratacos&author=E%20Bruguera&author=C%20Cabezas&volume=11&publication_year=2012&pages=704-711&pmid=22716474&doi=10.1111/j.1601-183X.2012.00815.x&)

10. Dudanova I, Sedej S, Ahmad M, Masius H, Sargsyan V, Zhang W, et al. Important contribution of alpha-neurexins to Ca2+-triggered exocytosis of secretory granules. J Neurosci. 2006;26:10599–10613. doi: 10.1523/JNEUROSCI.1913-06.2006.  [DOI](https://doi.org/10.1523/JNEUROSCI.1913-06.2006) | [PMC free article](/articles/PMC6674690/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17035546/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Neurosci&title=Important%20contribution%20of%20alpha-neurexins%20to%20Ca2+-triggered%20exocytosis%20of%20secretory%20granules&author=I%20Dudanova&author=S%20Sedej&author=M%20Ahmad&author=H%20Masius&author=V%20Sargsyan&volume=26&publication_year=2006&pages=10599-10613&pmid=17035546&doi=10.1523/JNEUROSCI.1913-06.2006&)

11. Duong L, Klitten LL, Moller RS, Ingason A, Jakobsen KD, Skjodt C, et al. Mutations in NRXN1 in a family multiply affected with brain disorders: NRXN1 mutations and brain disorders. Am J Med Genet B Neuropsychiatr Genet. 2012;159B:354–358. doi: 10.1002/ajmg.b.32036.  [DOI](https://doi.org/10.1002/ajmg.b.32036) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22337556/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Med%20Genet%20B%20Neuropsychiatr%20Genet&title=Mutations%20in%20NRXN1%20in%20a%20family%20multiply%20affected%20with%20brain%20disorders:%20NRXN1%20mutations%20and%20brain%20disorders&author=L%20Duong&author=LL%20Klitten&author=RS%20Moller&author=A%20Ingason&author=KD%20Jakobsen&volume=159B&publication_year=2012&pages=354-358&pmid=22337556&doi=10.1002/ajmg.b.32036&)

12. Fedorova L, Fedorov A. Introns in gene evolution. Genetica. 2003;118:123–131.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/12868603/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genetica&title=Introns%20in%20gene%20evolution&author=L%20Fedorova&author=A%20Fedorov&volume=118&publication_year=2003&pages=123-131&pmid=12868603&)

13. Giegling I, Balzarro B, Porcelli S, Schafer M, Hartmann AM, Friedl M, et al. Influence of ANKK1 and DRD2 polymorphisms in response to haloperidol. Eur Arch Psychiatry Clin Neurosci. 2013;263:65–74. doi: 10.1007/s00406-012-0348-1.  [DOI](https://doi.org/10.1007/s00406-012-0348-1) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22893251/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20Arch%20Psychiatry%20Clin%20Neurosci&title=Influence%20of%20ANKK1%20and%20DRD2%20polymorphisms%20in%20response%20to%20haloperidol&author=I%20Giegling&author=B%20Balzarro&author=S%20Porcelli&author=M%20Schafer&author=AM%20Hartmann&volume=263&publication_year=2013&pages=65-74&pmid=22893251&doi=10.1007/s00406-012-0348-1&)

14. Grayton HM, Fernandes C, Rujescu D, Collier DA. Copy number variations in neurodevelopmental disorders. Prog Neurobiol. 2012;99:81–91. doi: 10.1016/j.pneurobio.2012.07.005.  [DOI](https://doi.org/10.1016/j.pneurobio.2012.07.005) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22813947/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Prog%20Neurobiol&title=Copy%20number%20variations%20in%20neurodevelopmental%20disorders&author=HM%20Grayton&author=C%20Fernandes&author=D%20Rujescu&author=DA%20Collier&volume=99&publication_year=2012&pages=81-91&pmid=22813947&doi=10.1016/j.pneurobio.2012.07.005&)

15. Hishimoto A, Liu QR, Drgon T, Pletnikova O, Walther D, Zhu XG, et al. Neurexin 3 polymorphisms are associated with alcohol dependence and altered expression of specific isoforms. Hum Mol Genet. 2007;16:2880–2891. doi: 10.1093/hmg/ddm247.  [DOI](https://doi.org/10.1093/hmg/ddm247) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17804423/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mol%20Genet&title=Neurexin%203%20polymorphisms%20are%20associated%20with%20alcohol%20dependence%20and%20altered%20expression%20of%20specific%20isoforms&author=A%20Hishimoto&author=QR%20Liu&author=T%20Drgon&author=O%20Pletnikova&author=D%20Walther&volume=16&publication_year=2007&pages=2880-2891&pmid=17804423&doi=10.1093/hmg/ddm247&)

16. Ikeda M, Yamanouchi Y, Kinoshita Y, Kitajima T, Yoshimura R, Hashimoto S, et al. Variants of dopamine and serotonin candidate genes as predictors of response to risperidone treatment in first-episode schizophrenia. Pharmacogenomics. 2008;9:1437–1443. doi: 10.2217/14622416.9.10.1437.  [DOI](https://doi.org/10.2217/14622416.9.10.1437) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18855532/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Variants%20of%20dopamine%20and%20serotonin%20candidate%20genes%20as%20predictors%20of%20response%20to%20risperidone%20treatment%20in%20first-episode%20schizophrenia&author=M%20Ikeda&author=Y%20Yamanouchi&author=Y%20Kinoshita&author=T%20Kitajima&author=R%20Yoshimura&volume=9&publication_year=2008&pages=1437-1443&pmid=18855532&doi=10.2217/14622416.9.10.1437&)

17. Ikeda M, Aleksic B, Kirov G, Kinoshita Y, Yamanouchi Y, Kitajima T, et al. Copy number variation in schizophrenia in the Japanese population. Biol Psychiatry. 2010;67:283–286. doi: 10.1016/j.biopsych.2009.08.034.  [DOI](https://doi.org/10.1016/j.biopsych.2009.08.034) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19880096/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biol%20Psychiatry&title=Copy%20number%20variation%20in%20schizophrenia%20in%20the%20Japanese%20population&author=M%20Ikeda&author=B%20Aleksic&author=G%20Kirov&author=Y%20Kinoshita&author=Y%20Yamanouchi&volume=67&publication_year=2010&pages=283-286&pmid=19880096&doi=10.1016/j.biopsych.2009.08.034&)

18. International Schizophrenia Consortium Rare chromosomal deletions and duplications increase risk of schizophrenia. Nature. 2008;455:237–241. doi: 10.1038/nature07239.  [DOI](https://doi.org/10.1038/nature07239) | [PMC free article](/articles/PMC3912847/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18668038/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Rare%20chromosomal%20deletions%20and%20duplications%20increase%20risk%20of%20schizophrenia&volume=455&publication_year=2008&pages=237-241&pmid=18668038&doi=10.1038/nature07239&)

19. Kapur S, Mamo D. Half a century of antipsychotics and still a central role for dopamine D2 receptors. Prog Neuropsychopharmacol Biol Psychiatry. 2003;27:1081–1090. doi: 10.1016/j.pnpbp.2003.09.004.  [DOI](https://doi.org/10.1016/j.pnpbp.2003.09.004) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14642968/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Prog%20Neuropsychopharmacol%20Biol%20Psychiatry&title=Half%20a%20century%20of%20antipsychotics%20and%20still%20a%20central%20role%20for%20dopamine%20D2%20receptors&author=S%20Kapur&author=D%20Mamo&volume=27&publication_year=2003&pages=1081-1090&pmid=14642968&doi=10.1016/j.pnpbp.2003.09.004&)

20. Kay SR, Opler LA, Spitzer RL, Williams JB, Fiszbein A, Gorelick A. SCID-PANSS: two-tier diagnostic system for psychotic disorders. Compr Psychiatry. 1991;32:355–361. doi: 10.1016/0010-440x(91)90085-q.  [DOI](https://doi.org/10.1016/0010-440x(91)90085-q) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/1935026/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Compr%20Psychiatry&title=SCID-PANSS:%20two-tier%20diagnostic%20system%20for%20psychotic%20disorders&author=SR%20Kay&author=LA%20Opler&author=RL%20Spitzer&author=JB%20Williams&author=A%20Fiszbein&volume=32&publication_year=1991&pages=355-361&pmid=1935026&doi=10.1016/0010-440x(91)90085-q&)

21. Kirov G, Gumus D, Chen W, Norton N, Georgieva L, Sari M, et al. Comparative genome hybridization suggests a role for NRXN1 and APBA2 in schizophrenia. Hum Mol Genet. 2008;17:458–465. doi: 10.1093/hmg/ddm323.  [DOI](https://doi.org/10.1093/hmg/ddm323) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17989066/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mol%20Genet&title=Comparative%20genome%20hybridization%20suggests%20a%20role%20for%20NRXN1%20and%20APBA2%20in%20schizophrenia&author=G%20Kirov&author=D%20Gumus&author=W%20Chen&author=N%20Norton&author=L%20Georgieva&volume=17&publication_year=2008&pages=458-465&pmid=17989066&doi=10.1093/hmg/ddm323&)

22. Kirov G, Rujescu D, Ingason A, Collier DA, O'Donovan MC, Owen MJ. Neurexin 1 (NRXN1) deletions in schizophrenia. Schizophr Bull. 2009;35:851–854. doi: 10.1093/schbul/sbp079.  [DOI](https://doi.org/10.1093/schbul/sbp079) | [PMC free article](/articles/PMC2728827/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19675094/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Schizophr%20Bull&title=Neurexin%201%20(NRXN1)%20deletions%20in%20schizophrenia&author=G%20Kirov&author=D%20Rujescu&author=A%20Ingason&author=DA%20Collier&author=MC%20O'Donovan&volume=35&publication_year=2009&pages=851-854&pmid=19675094&doi=10.1093/schbul/sbp079&)

23. Law AJ, Wang Y, Sei Y, O'Donnell P, Piantadosi P, Papaleo F, et al. Neuregulin 1-ErbB4-PI3K signaling in schizophrenia and phosphoinositide 3-kinase-p110delta inhibition as a potential therapeutic strategy. Proc Natl Acad Sci USA. 2012;109:12165–12170. doi: 10.1073/pnas.1206118109.  [DOI](https://doi.org/10.1073/pnas.1206118109) | [PMC free article](/articles/PMC3409795/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22689948/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20USA&title=Neuregulin%201-ErbB4-PI3K%20signaling%20in%20schizophrenia%20and%20phosphoinositide%203-kinase-p110delta%20inhibition%20as%20a%20potential%20therapeutic%20strategy&author=AJ%20Law&author=Y%20Wang&author=Y%20Sei&author=P%20O'Donnell&author=P%20Piantadosi&volume=109&publication_year=2012&pages=12165-12170&pmid=22689948&doi=10.1073/pnas.1206118109&)

24. Lett TA, Tiwari AK, Meltzer HY, Lieberman JA, Potkin SG, Voineskos AN, et al. The putative functional rs1045881 marker of neurexin-1 in schizophrenia and clozapine response. Schizophr Res. 2011;132:121–124. doi: 10.1016/j.schres.2011.08.007.  [DOI](https://doi.org/10.1016/j.schres.2011.08.007) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21890328/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Schizophr%20Res&title=The%20putative%20functional%20rs1045881%20marker%20of%20neurexin-1%20in%20schizophrenia%20and%20clozapine%20response&author=TA%20Lett&author=AK%20Tiwari&author=HY%20Meltzer&author=JA%20Lieberman&author=SG%20Potkin&volume=132&publication_year=2011&pages=121-124&pmid=21890328&doi=10.1016/j.schres.2011.08.007&)

25. Lieberman JA, Stroup TS, McEvoy JP, Swartz MS, Rosenheck RA, Perkins DO, et al. Effectiveness of antipsychotic drugs in patients with chronic schizophrenia. N Engl J Med. 2005;353:1209–1223. doi: 10.1056/NEJMoa051688.  [DOI](https://doi.org/10.1056/NEJMoa051688) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16172203/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Effectiveness%20of%20antipsychotic%20drugs%20in%20patients%20with%20chronic%20schizophrenia&author=JA%20Lieberman&author=TS%20Stroup&author=JP%20McEvoy&author=MS%20Swartz&author=RA%20Rosenheck&volume=353&publication_year=2005&pages=1209-1223&pmid=16172203&doi=10.1056/NEJMoa051688&)

26. Lotrich FE. The emerging potential of pharmacogenetics in psychiatry. Am J Psychiatry. 2012;169:681–683. doi: 10.1176/appi.ajp.2012.12040457.  [DOI](https://doi.org/10.1176/appi.ajp.2012.12040457) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22760186/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Psychiatry&title=The%20emerging%20potential%20of%20pharmacogenetics%20in%20psychiatry&author=FE%20Lotrich&volume=169&publication_year=2012&pages=681-683&pmid=22760186&doi=10.1176/appi.ajp.2012.12040457&)

27. Malhotra AK, Zhang JP, Lencz T. Pharmacogenetics in psychiatry: translating research into clinical practice. Mol Psychiatry. 2012;17:760–769. doi: 10.1038/mp.2011.146.  [DOI](https://doi.org/10.1038/mp.2011.146) | [PMC free article](/articles/PMC3928114/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22083729/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Psychiatry&title=Pharmacogenetics%20in%20psychiatry:%20translating%20research%20into%20clinical%20practice&author=AK%20Malhotra&author=JP%20Zhang&author=T%20Lencz&volume=17&publication_year=2012&pages=760-769&pmid=22083729&doi=10.1038/mp.2011.146&)

28. Malhotra AK, Correll CU, Chowdhury NI, Müller DJ, Gregersen PK, Lee AT, et al. Association between common variants near the melanocortin 4 receptor gene and severe antipsychotic drug-induced weight gain. Arch Gen Psychiatry. 2012;69:904–912. doi: 10.1001/archgenpsychiatry.2012.191.  [DOI](https://doi.org/10.1001/archgenpsychiatry.2012.191) | [PMC free article](/articles/PMC4166499/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22566560/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arch%20Gen%20Psychiatry&title=Association%20between%20common%20variants%20near%20the%20melanocortin%204%20receptor%20gene%20and%20severe%20antipsychotic%20drug-induced%20weight%20gain&author=AK%20Malhotra&author=CU%20Correll&author=NI%20Chowdhury&author=DJ%20M%C3%BCller&author=PK%20Gregersen&volume=69&publication_year=2012&pages=904-912&pmid=22566560&doi=10.1001/archgenpsychiatry.2012.191&)

29. Missler M, Fernandez-Chacon R, Sudhof TC. The making of neurexins. J Neurochem. 1998;71:1339–1347. doi: 10.1046/j.1471-4159.1998.71041339.x.  [DOI](https://doi.org/10.1046/j.1471-4159.1998.71041339.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9751164/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Neurochem&title=The%20making%20of%20neurexins&author=M%20Missler&author=R%20Fernandez-Chacon&author=TC%20Sudhof&volume=71&publication_year=1998&pages=1339-1347&pmid=9751164&doi=10.1046/j.1471-4159.1998.71041339.x&)

30. Missler M, Zhang W, Rohlmann A, Kattenstroth G, Hammer RE, Gottmann K, et al. Alpha-neurexins couple Ca2+ channels to synaptic vesicle exocytosis. Nature. 2003;423:939–948. doi: 10.1038/nature01755.  [DOI](https://doi.org/10.1038/nature01755) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12827191/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Alpha-neurexins%20couple%20Ca2+%20channels%20to%20synaptic%20vesicle%20exocytosis&author=M%20Missler&author=W%20Zhang&author=A%20Rohlmann&author=G%20Kattenstroth&author=RE%20Hammer&volume=423&publication_year=2003&pages=939-948&pmid=12827191&doi=10.1038/nature01755&)

31. Moller RS, Weber YG, Klitten LL, Trucks H, Muhle H, Kunz WS, et al. Exon-disrupting deletions of NRXN1 in idiopathic generalized epilepsy. Epilepsia. 2013;54:256–264. doi: 10.1111/epi.12078.  [DOI](https://doi.org/10.1111/epi.12078) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23294455/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Epilepsia&title=Exon-disrupting%20deletions%20of%20NRXN1%20in%20idiopathic%20generalized%20epilepsy&author=RS%20Moller&author=YG%20Weber&author=LL%20Klitten&author=H%20Trucks&author=H%20Muhle&volume=54&publication_year=2013&pages=256-264&pmid=23294455&doi=10.1111/epi.12078&)

32. Muhleisen TW, Basmanav FB, Forstner AJ, Mattheisen M, Priebe L, Herms S, et al. Resequencing and follow-up of neurexin 1 (NRXN1) in schizophrenia patients. Schizophr Res. 2011;127:35–40. doi: 10.1016/j.schres.2011.01.001.  [DOI](https://doi.org/10.1016/j.schres.2011.01.001) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21288692/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Schizophr%20Res&title=Resequencing%20and%20follow-up%20of%20neurexin%201%20(NRXN1)%20in%20schizophrenia%20patients&author=TW%20Muhleisen&author=FB%20Basmanav&author=AJ%20Forstner&author=M%20Mattheisen&author=L%20Priebe&volume=127&publication_year=2011&pages=35-40&pmid=21288692&doi=10.1016/j.schres.2011.01.001&)

33. Need AC, Keefe RS, Ge D, Grossman I, Dickson S, McEvoy JP, et al. Pharmacogenetics of antipsychotic response in the CATIE trial: a candidate gene analysis. Eur J Hum Genet. 2009;17:946–957. doi: 10.1038/ejhg.2008.264.  [DOI](https://doi.org/10.1038/ejhg.2008.264) | [PMC free article](/articles/PMC2986499/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19156168/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Hum%20Genet&title=Pharmacogenetics%20of%20antipsychotic%20response%20in%20the%20CATIE%20trial:%20a%20candidate%20gene%20analysis&author=AC%20Need&author=RS%20Keefe&author=D%20Ge&author=I%20Grossman&author=S%20Dickson&volume=17&publication_year=2009&pages=946-957&pmid=19156168&doi=10.1038/ejhg.2008.264&)

34. Nussbaum J, Xu Q, Payne TJ, Ma JZ, Huang W, Gelernter J, et al. Significant association of the neurexin-1 gene (NRXN1) with nicotine dependence in European- and African-American smokers. Hum Mol Genet. 2008;17:1569–1577. doi: 10.1093/hmg/ddn044.  [DOI](https://doi.org/10.1093/hmg/ddn044) | [PMC free article](/articles/PMC2902291/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18270208/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mol%20Genet&title=Significant%20association%20of%20the%20neurexin-1%20gene%20(NRXN1)%20with%20nicotine%20dependence%20in%20European-%20and%20African-American%20smokers&author=J%20Nussbaum&author=Q%20Xu&author=TJ%20Payne&author=JZ%20Ma&author=W%20Huang&volume=17&publication_year=2008&pages=1569-1577&pmid=18270208&doi=10.1093/hmg/ddn044&)

35. Owen MJ. Implications of genetic findings for understanding schizophrenia. Schizophr Bull. 2012;38:904–907. doi: 10.1093/schbul/sbs103.  [DOI](https://doi.org/10.1093/schbul/sbs103) | [PMC free article](/articles/PMC3446223/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22987847/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Schizophr%20Bull&title=Implications%20of%20genetic%20findings%20for%20understanding%20schizophrenia&author=MJ%20Owen&volume=38&publication_year=2012&pages=904-907&pmid=22987847&doi=10.1093/schbul/sbs103&)

36. Prathikanti S, Weinberger DR. Psychiatric genetics—the new era: genetic research and some clinical implications. Br Med Bull. 2005;73-74:107–122. doi: 10.1093/bmb/ldh055.  [DOI](https://doi.org/10.1093/bmb/ldh055) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16365481/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20Med%20Bull&title=Psychiatric%20genetics%E2%80%94the%20new%20era:%20genetic%20research%20and%20some%20clinical%20implications&author=S%20Prathikanti&author=DR%20Weinberger&volume=73-74&publication_year=2005&pages=107-122&pmid=16365481&doi=10.1093/bmb/ldh055&)

37. Ripke, Chambert K, Moran JL, Kähler AK, Akterin S, Bergen SE, et al. Genome-wide association analysis identifies 13 new risk loci for schizophrenia. Nat Genet. 2013;45:1150–1159. doi: 10.1038/ng.2742.  [DOI](https://doi.org/10.1038/ng.2742) | [PMC free article](/articles/PMC3827979/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23974872/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Genome-wide%20association%20analysis%20identifies%2013%20new%20risk%20loci%20for%20schizophrenia&author=%20Ripke&author=K%20Chambert&author=JL%20Moran&author=AK%20K%C3%A4hler&author=S%20Akterin&volume=45&publication_year=2013&pages=1150-1159&pmid=23974872&doi=10.1038/ng.2742&)

38. Rowen L, Young J, Birditt B, Kaur A, Madan A, Philipps DL, et al. Analysis of the human neurexin genes: alternative splicing and the generation of protein diversity. Genomics. 2002;79:587–597. doi: 10.1006/geno.2002.6734.  [DOI](https://doi.org/10.1006/geno.2002.6734) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11944992/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genomics&title=Analysis%20of%20the%20human%20neurexin%20genes:%20alternative%20splicing%20and%20the%20generation%20of%20protein%20diversity&author=L%20Rowen&author=J%20Young&author=B%20Birditt&author=A%20Kaur&author=A%20Madan&volume=79&publication_year=2002&pages=587-597&pmid=11944992&doi=10.1006/geno.2002.6734&)

39. Rujescu D, Ingason A, Cichon S, Pietilainen OP, Barnes MR, Toulopoulou T, et al. Disruption of the neurexin 1 gene is associated with schizophrenia. Hum Mol Genet. 2009;18:988–996. doi: 10.1093/hmg/ddn351.  [DOI](https://doi.org/10.1093/hmg/ddn351) | [PMC free article](/articles/PMC2695245/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18945720/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mol%20Genet&title=Disruption%20of%20the%20neurexin%201%20gene%20is%20associated%20with%20schizophrenia&author=D%20Rujescu&author=A%20Ingason&author=S%20Cichon&author=OP%20Pietilainen&author=MR%20Barnes&volume=18&publication_year=2009&pages=988-996&pmid=18945720&doi=10.1093/hmg/ddn351&)

40. Shepard S, McCreary M, Fedorov A. The peculiarities of large intron splicing in animals. PLoS One. 2009;4:e7853. doi: 10.1371/journal.pone.0007853.  [DOI](https://doi.org/10.1371/journal.pone.0007853) | [PMC free article](/articles/PMC2773006/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19924226/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=The%20peculiarities%20of%20large%20intron%20splicing%20in%20animals&author=S%20Shepard&author=M%20McCreary&author=A%20Fedorov&volume=4&publication_year=2009&pages=e7853&pmid=19924226&doi=10.1371/journal.pone.0007853&)

41. Snyder SH. Dopamine receptors, neuroleptics, and schizophrenia. Am J Psychiatry. 1981;138:460–464. doi: 10.1176/ajp.138.4.460.  [DOI](https://doi.org/10.1176/ajp.138.4.460) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/6111227/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Psychiatry&title=Dopamine%20receptors,%20neuroleptics,%20and%20schizophrenia&author=SH%20Snyder&volume=138&publication_year=1981&pages=460-464&pmid=6111227&doi=10.1176/ajp.138.4.460&)

42. Souza RP, Romano-Silva MA, Lieberman JA, Meltzer HY, Wong AH, Kennedy JL. Association study of GSK3 gene polymorphisms with schizophrenia and clozapine response. Psychopharmacology (Berl) 2008;200:177–186. doi: 10.1007/s00213-008-1193-9.  [DOI](https://doi.org/10.1007/s00213-008-1193-9) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18500637/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Psychopharmacology%20(Berl)&title=Association%20study%20of%20GSK3%20gene%20polymorphisms%20with%20schizophrenia%20and%20clozapine%20response&author=RP%20Souza&author=MA%20Romano-Silva&author=JA%20Lieberman&author=HY%20Meltzer&author=AH%20Wong&volume=200&publication_year=2008&pages=177-186&pmid=18500637&doi=10.1007/s00213-008-1193-9&)

43. Souza RP, Meltzer HY, Lieberman JA, Le FB, Kennedy JL. Influence of neurexin 1 (NRXN1) polymorphisms in clozapine response. Hum Psychopharmacol. 2010;25:582–585. doi: 10.1002/hup.1146.  [DOI](https://doi.org/10.1002/hup.1146) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20860064/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Psychopharmacol&title=Influence%20of%20neurexin%201%20(NRXN1)%20polymorphisms%20in%20clozapine%20response&author=RP%20Souza&author=HY%20Meltzer&author=JA%20Lieberman&author=FB%20Le&author=JL%20Kennedy&volume=25&publication_year=2010&pages=582-585&pmid=20860064&doi=10.1002/hup.1146&)

44. Stefansson H, Rujescu D, Cichon S, Pietilainen OP, Ingason A, Steinberg S, et al. Large recurrent microdeletions associated with schizophrenia. Nature. 2008;455:232–236. doi: 10.1038/nature07229.  [DOI](https://doi.org/10.1038/nature07229) | [PMC free article](/articles/PMC2687075/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18668039/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Large%20recurrent%20microdeletions%20associated%20with%20schizophrenia&author=H%20Stefansson&author=D%20Rujescu&author=S%20Cichon&author=OP%20Pietilainen&author=A%20Ingason&volume=455&publication_year=2008&pages=232-236&pmid=18668039&doi=10.1038/nature07229&)

45. Sudhof TC. Neuroligins and neurexins link synaptic function to cognitive disease. Nature. 2008;455:903–911. doi: 10.1038/nature07456.  [DOI](https://doi.org/10.1038/nature07456) | [PMC free article](/articles/PMC2673233/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18923512/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Neuroligins%20and%20neurexins%20link%20synaptic%20function%20to%20cognitive%20disease&author=TC%20Sudhof&volume=455&publication_year=2008&pages=903-911&pmid=18923512&doi=10.1038/nature07456&)

46. Sullivan PF, Daly MJ, O'Donovan M. Genetic architectures of psychiatric disorders: the emerging picture and its implications. Nat Rev Genet. 2012;13:537–551. doi: 10.1038/nrg3240.  [DOI](https://doi.org/10.1038/nrg3240) | [PMC free article](/articles/PMC4110909/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22777127/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Genet&title=Genetic%20architectures%20of%20psychiatric%20disorders:%20the%20emerging%20picture%20and%20its%20implications&author=PF%20Sullivan&author=MJ%20Daly&author=M%20O'Donovan&volume=13&publication_year=2012&pages=537-551&pmid=22777127&doi=10.1038/nrg3240&)

47. Sullivan PF, Lin D, Tzeng JY, van den Oord E, Perkins D, Stroup TS, et al. Genomewide association for schizophrenia in the CATIE study: results of stage 1. Mol Psychiatry. 2008;14:1144–1144. doi: 10.1038/mp.2008.25.  [DOI](https://doi.org/10.1038/mp.2008.25) | [PMC free article](/articles/PMC3910086/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18347602/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Psychiatry&title=Genomewide%20association%20for%20schizophrenia%20in%20the%20CATIE%20study:%20results%20of%20stage%201&author=PF%20Sullivan&author=D%20Lin&author=JY%20Tzeng&author=E%20van%20den%20Oord&author=D%20Perkins&volume=14&publication_year=2008&pages=1144-1144&pmid=18347602&doi=10.1038/mp.2008.25&)

48. Swaminathan S, Shen L, Kim S, Inlow M, West JD, Faber KM, et al. Analysis of copy number variation in Alzheimer's disease: the NIALOAD/ NCRAD Family Study. Curr Alzheimer Res. 2012;9:801–814. doi: 10.2174/156720512802455331.  [DOI](https://doi.org/10.2174/156720512802455331) | [PMC free article](/articles/PMC3500615/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22486522/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Alzheimer%20Res&title=Analysis%20of%20copy%20number%20variation%20in%20Alzheimer's%20disease:%20the%20NIALOAD/%20NCRAD%20Family%20Study&author=S%20Swaminathan&author=L%20Shen&author=S%20Kim&author=M%20Inlow&author=JD%20West&volume=9&publication_year=2012&pages=801-814&pmid=22486522&doi=10.2174/156720512802455331&)

49. Varoqueaux F, Aramuni G, Rawson RL, Mohrmann R, Missler M, Gottmann K, et al. Neuroligins determine synapse maturation and function. Neuron. 2006;51:741–754. doi: 10.1016/j.neuron.2006.09.003.  [DOI](https://doi.org/10.1016/j.neuron.2006.09.003) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16982420/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neuron&title=Neuroligins%20determine%20synapse%20maturation%20and%20function&author=F%20Varoqueaux&author=G%20Aramuni&author=RL%20Rawson&author=R%20Mohrmann&author=M%20Missler&volume=51&publication_year=2006&pages=741-754&pmid=16982420&doi=10.1016/j.neuron.2006.09.003&)

50. Weinberger DR. Implications of normal brain development for the pathogenesis of schizophrenia. Arch Gen Psychiatry. 1987;44:660–669. doi: 10.1001/archpsyc.1987.01800190080012.  [DOI](https://doi.org/10.1001/archpsyc.1987.01800190080012) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/3606332/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arch%20Gen%20Psychiatry&title=Implications%20of%20normal%20brain%20development%20for%20the%20pathogenesis%20of%20schizophrenia&author=DR%20Weinberger&volume=44&publication_year=1987&pages=660-669&pmid=3606332&doi=10.1001/archpsyc.1987.01800190080012&)

51. Zhang C, Atasoy D, Arac D, Yang X, Fucillo MV, Robison AJ, et al. Neurexins physically and functionally interact with GABA(A) receptors. Neuron. 2010;66:403–416. doi: 10.1016/j.neuron.2010.04.008.  [DOI](https://doi.org/10.1016/j.neuron.2010.04.008) | [PMC free article](/articles/PMC3243752/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20471353/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neuron&title=Neurexins%20physically%20and%20functionally%20interact%20with%20GABA(A)%20receptors&author=C%20Zhang&author=D%20Atasoy&author=D%20Arac&author=X%20Yang&author=MV%20Fucillo&volume=66&publication_year=2010&pages=403-416&pmid=20471353&doi=10.1016/j.neuron.2010.04.008&)

52. Zhang JP, Lencz T, Malhotra AK. D2 receptor genetic variation and clinical response to antipsychotic drug treatment: a meta-analysis. Am J Psychiatry. 2010;167:763–772. doi: 10.1176/appi.ajp.2009.09040598.  [DOI](https://doi.org/10.1176/appi.ajp.2009.09040598) | [PMC free article](/articles/PMC2896457/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20194480/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Psychiatry&title=D2%20receptor%20genetic%20variation%20and%20clinical%20response%20to%20antipsychotic%20drug%20treatment:%20a%20meta-analysis&author=JP%20Zhang&author=T%20Lencz&author=AK%20Malhotra&volume=167&publication_year=2010&pages=763-772&pmid=20194480&doi=10.1176/appi.ajp.2009.09040598&)
