# HLA-G 3’UTR Polymorphisms Impact the Prognosis of Stage II-III CRC Patients in Fluoropyrimidine-Based Treatment

## Metadata
**Authors:** Marica Garziera, Ettore Bidoli, Erika Cecchin, Enrico Mini, Stefania Nobili, Sara Lonardi, Angela Buonadonna, Domenico Errante, Nicoletta Pella, Mario D’Andrea, Francesco De Marchi, Antonino De Paoli, Chiara Zanusso, Elena De Mattia, Renato Tassi, Giuseppe Toffoli
**Journal:** PLoS ONE
**Date:** 2015 Dec 3
**DOI:** [10.1371/journal.pone.0144000](https://doi.org/10.1371/journal.pone.0144000)
**PMID:** 26633805
**PMCID:** PMC4669157
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4669157/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC4669157/pdf/pone.0144000.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC4669157/pdf/pone.0144000.pdf)

## Abstract

An important hallmark of CRC is the evasion of immune surveillance. HLA-G is a negative regulator of host’s immune response. Overexpression of HLA-G protein in primary tumour CRC tissues has already been associated to worse prognosis; however a definition of the role of immunogenetic host background is still lacking. Germline polymorphisms in the 3’UTR region of HLA-G influence the magnitude of the protein by modulating HLA-G mRNA stability. Soluble HLA-G has been associated to 3’UTR +2960 Ins/Ins and +3035 C/T (lower levels) and +3187 G/G (high levels) genotypes. HLA-G 3’UTR SNPs have never been explored in CRC outcome. The purpose of this study was to investigate if common HLA-G 3’UTR polymorphisms have an impact on DFS and OS of 253 stage II-III CRC patients, after primary surgery and ADJ-CT based on FL. The 3’UTR was sequenced and SNPs were analyzed for their association with survival by Kaplan-Meier and multivariate Cox models; results underwent internal validation using a resampling method (bootstrap analysis). In a multivariate analysis, we estimated an association with improved DFS in Ins allele (Ins/Del +Ins/Ins) carriers (HR 0.60, 95% CI 0.38–0.93, P = 0.023) and in patients with +3035 C/T genotype (HR 0.51, 95% CI 0.26–0.99, P = 0.045). The +3187 G/G mutated carriers (G/G vs A/A+A/G) were associated to a worst prognosis in both DFS (HR 2.46, 95% CI 1.19–5.05, P = 0.015) and OS (HR 2.71, 95% CI 1.16–6.63, P = 0.022). Our study shows a prognostic and independent role of 3 HLA-G 3’UTR SNPs, +2960 14-bp INDEL, +3035 C>T, and +3187 A>G.

## Introduction

Colorectal cancer (CRC) is still a clinical burden being the third most common cancer in the United States [[1](#pone.0144000.ref001)] and the second leading cause of cancer death in Europe, in both women and men [[2](#pone.0144000.ref002)]. Recent advances in protein- and genomic-based technologies, validated predictive and prognostic biomarkers, have demonstrated that CRC should be considered as a heterogeneous disease [[3](#pone.0144000.ref003)–[5](#pone.0144000.ref005)]. Adjuvant chemotherapy (ADJ-CT) based on fluoropyrimidine (FL) is generally administered in stage II-III patients after surgical resection of the primary tumour. Despite optimal surgery and adjuvant therapies, the risk of recurrence for stages II or III disease is about 40% [[6](#pone.0144000.ref006)] and ~ 80% of stage II CRC patients will be disease-free even without ADJ-CT [[7](#pone.0144000.ref007)]. Recently, gene expression profile signatures of immune-system related genes and presence of the immune infiltrates in tumour microenvironment were shown to have an independent prognostic significance in CRC compared to classical clinical factors [[8](#pone.0144000.ref008)–[11](#pone.0144000.ref011)].

Tumour immunogenicity, cancer cell capability to escape from the host’s immune system surveillance, and immunogenetic background of the patient, represent a future challenge of current research [[12](#pone.0144000.ref012)]. Two main mechanisms contribute to the cancer immunoediting process [[13](#pone.0144000.ref013)] leading to poorly immunogenic tumour cell variants invisible to the immune system. The first is the minimization of the level of tumour associated antigens (TAA) presentation through the downregulation or loss of the human leukocyte antigen (HLA) class I expression by tumour cells [[14](#pone.0144000.ref014)]. The second is the competence of cancer cells in regulating the expression of the non-classical HLA class I molecules such as HLA-G [[15](#pone.0144000.ref015)]. The *HLA-G* gene codifies for a tolerogenic molecule with well recognized immune-inhibitory properties on both innate and adaptive immune responses [[16](#pone.0144000.ref016)–[18](#pone.0144000.ref018)]. HLA-G is highly expressed in physiological conditions in trophoblast at fetal-maternal interface and has a restricted distribution in normal tissues [[19](#pone.0144000.ref019)]; however, an increased expression can be induced in pathological conditions such as cancer [[20](#pone.0144000.ref020),[21](#pone.0144000.ref021)]. Several genetic variations involved in *HLA-G* regulation have been so far described in the 5’ upstream regulatory (or promoter) region (5’URR) as well as in the 3’ untranslated region (3’UTR), while in contrast to the classical *HLA* class I loci, a lower variability in the coding regions is observed [[22](#pone.0144000.ref022)–[24](#pone.0144000.ref024)]. Increased soluble HLA-G levels in biological fluids are associated with down-modulation of the immune response in the host [[15](#pone.0144000.ref015)]. The *HLA-G* 3’UTR is the most studied segment of the gene due to the presence of multiple regulatory elements implicated in the modulation of HLA-G expression.

Nine (+2960 14-base pair (bp) INDEL, +3003 T>C, +3010 C>G, +3027 C>A, +3035 C>T, +3142 G>C, +3187 A>G, +3196 C>G and +3227 G>A) single nucleotide polymorphisms (SNPs) are known in this region, which can potentially alter the set of microRNAs (miRNAs) capable of binding the 3’UTR, thus influencing *HLA-G* RNA turnover, stability and splicing [[25](#pone.0144000.ref025),[26](#pone.0144000.ref026)]. At least three of these genetic variants have been associated to the transcriptional and post transcriptional control of *HLA-G* regulation [[27](#pone.0144000.ref027),[28](#pone.0144000.ref028)]. In particular, the presence (Ins) or absence (Del) of a 14-bp fragment (5’-ATTTGTTCATGCCT-3’) in position+2960 (14-bp INDEL, 14-bp Ins/Del) influences transcripts stability and is the most studied SNP. Presence of 14-bp *Ins* allele produces a more unstable *HLA-G* mRNA causing lower levels of the protein [[25](#pone.0144000.ref025)]. The G nucleotide in position +3142 favours the targeting of three miRNAs (miR-148a, -148b, and -152) leading to an increase in mRNA degradation [[22](#pone.0144000.ref022)]. Four-bp upstream to +3187 A>G and 9-bp downstream to +3196 C>G SNPs, two AU-rich motifs are present. The wild type +3187*A* allele is associated to decreased mRNA stability modifying an AU-rich motif [[29](#pone.0144000.ref029)]. Significant differences in soluble HLA-G protein levels have been observed in carriers for +2960 Ins/Ins and +3035 C/T (lower levels) [[29](#pone.0144000.ref029)–[32](#pone.0144000.ref032)], and in +3027 C/C, +3142 G/C, +3187 A/G (higher levels) [[29](#pone.0144000.ref029),[33](#pone.0144000.ref033)] genotypes. Some 3’UTR SNPs, 14-bp INDEL in particular, have been related to the susceptibility to certain diseases such as autoimmune diseases [[33](#pone.0144000.ref033)], preeclampsia, transplantation, chronic inflammatory diseases and several types of cancer [[30](#pone.0144000.ref030),[31](#pone.0144000.ref031),[34](#pone.0144000.ref034)]. Notably, a recent study in human cancer (chronic lymphocytic leukemia) has reported an association with the +2960 14-bp INDEL polymorphism and plasmatic HLA-G protein levels and survival [[31](#pone.0144000.ref031)].

HLA-G is overexpressed in primary CRC lesions [[35](#pone.0144000.ref035)–[37](#pone.0144000.ref037)], and higher levels of the soluble protein have been detected in plasma of CRC patients compared to that of patients with benign diseases or healthy donors [[38](#pone.0144000.ref038),[39](#pone.0144000.ref039)]. Expression of HLA-G in tumour tissues has been associated with the clinical outcome of CRC as an independent and unfavourable prognostic factor of reduced OS [[40](#pone.0144000.ref040),[41](#pone.0144000.ref041)].

To date, the characterization of *HLA-G* genotypes, alleles and haplotypes in CRC patients has not been explored as well as their role in the prediction of CRC prognosis [[42](#pone.0144000.ref042)].

Considering the relevance of the 3’UTR region in the control and regulation of the HLA-G transcripts and the lack of data in literature, our purpose was searching for associations between *HLA-*G 3’UTR polymorphisms detected at the germinal level, and the disease free survival (DFS) and OS of stage II-III CRC patients in adjuvant regimen. We analyzed a cohort of 253 CRC patients to investigate if functional SNPs in the 3’UTR of *HLA-G* gene, alone or in combination in the 3’UTR haplotypes, are associated to advantage or disadvantage for DFS and OS.

## Materials and Methods

### Ethics Statement

A written Informed Consent was obtained before surgery from all the participants to the use of their blood samples and clinical data for research purpose. The study was approved by the ethical committees of the participating institutes, the Centro di Riferimento Oncologico (CRO)-Aviano National Cancer Institute, Aviano, Italy, University Hospital, Florence, Italy, Istituto Oncologico Veneto, Padua, Italy, Ospedale Civile di Vittorio Veneto, Vittorio Veneto, Italy, University Hospital “S. Maria Della Misericordia”, Udine, Italy, and “San Filippo Neri” Hospital, Rome, Italy.

### Patients and treatment

A total of 253 CRC patients with newly diagnosed, untreated, histopathologically confirmed CRC, were included from an existing prospective collection of only blood samples stored at the Experimental and Clinical Pharmacology Unit of Centro di Riferimento Oncologico (CRO)-Aviano, based on previous multicenter pharmacogenomic studies [[43](#pone.0144000.ref043),[44](#pone.0144000.ref044)]. Eligible criteria were: stage II-III CRC, radiologically-confirmed absence of distant metastasis, age >18 years, performance status (WHO) 0–2, normal bone marrow, renal and liver function, and Caucasian ethnicity. Overall patients after diagnosis underwent primary surgery and received ADJ-CT based on fluoropyrimidine (FL) (i.e., 5-fluorouracil/folinic acid or capecitabine) [[44](#pone.0144000.ref044)], or FL plus oxaliplatin (FL+OXA) [[43](#pone.0144000.ref043)].

ADJ-CT was continued until completion of the planned cycles, recurrence, toxicity or patient refusal. Patients follow-up was measured from the time of surgery to the last contact or disease recurrence. Biological tests, pulmonary X-ray, positron emission tomography (PET) and/or computed tomography (CT) imaging alternatively with abdominal ultra-sonography were carried out every 3 months during the first 3 years after surgery. In the next 2 years PET/CT were performed every 6 months and then annually. Overall evaluations were conducted independently by the type of ADJ-CT. Recurrence was defined based on PET/CT scans in the case of metastasis presence, with pathologic confirmation made by the oncologist when necessary.

### HLA-G 3’UTR genotyping

A peripheral blood sample was collected in acid citrate dextrose (ACD) tubes from nearly all CRC patients. Genomic DNA was extracted from whole blood or from normal colon mucosa tissue by using the EZ1 DNA Blood or Tissue kit and the BioRobot EZ1 Workstation (QIAGEN Inc., Valencia, CA, USA). The 3’UTR of the *HLA-G* gene was amplified by polymerase chain reaction (PCR) using the already published [[23](#pone.0144000.ref023)] primers HLAG8F: 5’- TGTGAAACAGCTGCCCTGTGT-3’ and HLAG8R: 5’- GTCTTCCATTTATTTTGTCTCT-3’. PCR reactions were carried out in a final volume of 30 μl containing 1.25 mM MgCl2, 0.25 mM of each dNTPs, 5 pmol of each primer, about 50–200 ng of genomic DNA template, 1X PCR Buffer and 0.5 units of AmpliTaq Gold DNA polymerase (Applied Biosystems, Foster City, CA, USA). The PCR cycles were as follows: 5 mins. of initial denaturation at 94°C, 30 cycles of 45 secs. at 95°C, 45 secs. at 56°C, 60 secs. at 72°C, and the final extension step at 72°C for 7 mins. Five microliters of PCR products (344 bp in presence of deletion and 358 bp for insertion) were first analyzed by electrophoresis on 4% agarose gel stained with ethidium bromide. The remaining 25 μl of PCR reactions were purified using Diffinity RapidTip 2 tips (Sigma-Aldrich, St. Louis, MO, USA). Purified reactions (1–2 μl) were sequenced (Sanger method) by the use of the Big Dye Terminator kit (Applied Biosystems, Foster City, CA, USA) and an ABI PRISM capillary sequencer with the reverse HLAG8R primer to prevent sequence overlaps in heterozygous 14-bp samples [[23](#pone.0144000.ref023)]. Chromatograms were visualized with Chromas software version 2.01 and all single nucleotide polymorphisms (SNPs), and single nucleotide variants (SNVs) detected were recorded for each study participant.

### Statistical analysis

The aim of this study was to assess associations between *HLA-G* polymorphisms in the 3’UTR regulatory region and DFS and OS respectively of stage II-III CRC patients treated with ADJ-CT after primary surgery. DFS was defined as the time from date of surgery to date of clinically detectable recurrence (local, regional or distant), death from any cause, or last follow-up evaluation. OS was defined as the time from date of surgery to date of death from any cause, or last follow-up time. Longitudinal analyses were determined by means of Kaplan-Meier method (log-rank test) and Cox models. Cox proportional hazard models were used to estimate adjusted hazard ratios (HRs) and corresponding 95% confidence intervals (CIs). Associations were firstly evaluated by means of univariate models and only those that resulted statistically significant (two-sided *P*≤0.05) were included in multivariate models. Adjustment for age (continuous variable), sex (male *vs* female), stadium (II *vs* III); first tumour location (colon *vs* rectum) and type of ADJ-CT (FL-alone *vs* FL+OXA) were computed. After performing a Cox regression using the common genomic model, the associations of SNPs with clinical outcomes were also evaluated for genomic models of transmission (dominant and recessive). In a dominant model for a SNP with a major allele “A” and a minor allele “b”, the collective genotypes (“Ab”+”bb”) are compared to a reference genotype “AA”. For a recessive model, “bb” is compared to a collective (“AA”+”Ab”) reference group. The HR of a reference genotype group is arbitrarily fixed at 1.00. Survival analyses were not computed when a genotype or a haplotype was detected in only one patient. Only haplotypes with frequency >1% were included in the survival analyses. An internal validation of the study results was carried out by a bootstrap resampling technique. We ran 1000 bootstrapped Cox models adjusted for the aforementioned variables. SAS software, version 9.2 (SAS Institute Inc., Cary, NC, 1999–2001) was adopted for the estimations. Adherences of genotypic proportions to expectations under Hardy-Weinberg (HW) equilibrium and two-locus linkage disequilibrium (LD) were evaluated by means of the Haploview program v4.2. The most probable haplotype of each sample at the unknown gametic phase, was reconstructed by the use of the PHASE method (program v2.1.1) [[45](#pone.0144000.ref045)].

## Results

### Patient characteristics and survival analysis

The main demographic and clinical characteristics of CRC patients (N = 253) together with log-rank tests are summarized in [Table 1](#pone.0144000.t001).

### Table 1. Clinical and demographic characteristics of patients.

| Variables | N (%) | 5-years | Log-rank | 5-years | Log-rank |
| --- | --- | --- | --- | --- | --- |
|   |   | DFS % | P-value 1 | OS % | P-value 1 |
| Age, median |   |   |   |   |   |
| < 62.5 years | 122 (48) | 67 | 0.698 | 80 | 0.297 |
| > 62.5 years | 131 (52) | 63 |   | 76 |   |
| Gender |   |   |   |   |   |
| Male | 140 (55) | 60 | 0.076 | 73 | 0.019 |
| Female | 113 (45) | 71 |   | 84 |   |
| UICC (TNM) stage |   |   |   |   |   |
| II | 72 (28) | 79 | 0.010 | 89 | 0.010 |
| III | 181 (72) | 60 |   | 73 |   |
| First tumour location |   |   |   |   |   |
| Colon | 191 (76) | 67 | 0.435 | 81 | 0.050 |
| Cecum | 10 (5) |   |   |   |   |
| Right | 50 (26) |   |   |   |   |
| Transverse | 13 (7) |   |   |   |   |
| Left | 83 (44) |   |   |   |   |
| Sigma | 35 (18) |   |   |   |   |
| Rectum | 62 (24) | 61 | 0.522 | 68 |   |
| Adjuvant Chemotherapy |   |   |   |   |   |
| FL+OXA | 143 (56) | 66 | 0.512 | 78 | 0.629 |
| FL | 110 (44) | 64 |   | 77 |   |
| HLA-G 3’UTR SNPs |   |   |   |   |   |
| +2960 14 bp INDEL |   |   |   |   |   |
| Del/Del | 78 (31) | 54 | 0.036 | 70 | 0.231 |
| Ins/Del | 115 (45) | 70 |   | 79 |   |
| Ins/Ins | 60 (24) | 70 |   | 84 |   |
| Dominant Model | 175 (69) | 70 | 0.010 | 81 | 0.095 |
| Recessive Model | 60 (24) | 70 | 0.411 | 84 | 0.329 |
| +3003 T>C |   |   |   |   |   |
| T/T | 201 (79) | 66 | 0.725 | 77 | 0.996 |
| T/C | 47 (19) | 60 |   | 79 |   |
| C/C | 5 (2) | 80 |   | 80 |   |
| Dominant Model | 52 (21) | 63 | 0.667 | 80 | 0.935 |
| Recessive Model | 5 (2) | 80 | 0.600 | 80 | 0.985 |
| +3010 C>G |   |   |   |   |   |
| C/C | 94 (37) | 68 | 0.162 | 80 | 0.213 |
| C/G | 119 (47) | 67 |   | 80 |   |
| G/G | 40 (16) | 54 |   | 67 |   |
| Dominant Model | 159 (63) | 64 | 0.318 | 77 | 0.577 |
| Recessive Model | 40 (16) | 54 | 0.063 | 67 | 0.079 |
| +3027 C>A |   |   |   |   |   |
| C/C | 225 (89) | 63 | 0.176 | 78 | 0.684 |
| C/A | 27 (11) | 81 |   | 80 |   |
| +3035 C>T |   |   |   |   |   |
| C/C | 205 (81) | 63 | 0.051 | 77 | 0.367 |
| C/T | 47 (19) | 77 |   | 81 |   |
| +3142 G>C |   |   |   |   |   |
| G/G | 93 (37) | 68 | 0.113 | 79 | 0.226 |
| G/C | 119 (47) | 68 |   | 80 |   |
| C/C | 41 (16) | 52 |   | 67 |   |
| Dominant Model | 160 (63) | 64 | 0.366 | 77 | 0.627 |
| Recessive Model | 41 (16) | 52 | 0.038 | 67 | 0.085 |
| +3187 A>G |   |   |   |   |   |
| A/A | 145 (57) | 68 | 0.019 | 82 | 0.036 |
| A/G | 93 (37) | 65 |   | 76 |   |
| G/G | 15 (6) | 33 |   | 51 |   |
| Dominant Model | 108 (43) | 61 | 0.130 | 73 | 0.063 |
| Recessive Model | 15 (6) | 33 | 0.007 | 51 | 0.025 |
| +3196 C>G |   |   |   |   |   |
| C/C | 106 (42) | 60 | 0.062 | 72 | 0.191 |
| C/G | 107 (42) | 73 |   | 84 |   |
| G/G | 40 (16) | 59 |   | 79 |   |
| Dominant Model | 147 (58) | 69 | 0.074 | 82 | 0.076 |
| Recessive Model | 40 (16) | 59 | 0.417 | 79 | 0.773 |

Table 1 Caption: DFS, Disease Free Survival; OS, Overall Survival; SNPs, Single nucleotide polymorphisms; FL, Fluoropirymidine; OXA, Oxaliplatin; significant values (≤0.05) are shown in bold. 1 P values (log-rank test) given for the genotypes, the dominant and/or the recessive models.

Mean age was 60.5 +/-10.9 (IQ range 54–68 [25%-75%]) years, median age was 62.5 (range 24–82) years at onset. Most tumors (181/253 = 72%) presented stage III at the time of onset and were preferentially located at the left colon portion (191/253 = 76%). At diagnosis, stage II patients (N = 72) had pT3N0M0 (94%) of whom 6% had pT4aN0MO stage; most of them had >12 lymph nodes excised (72%), while 16% of patients had <12 lymph nodes excised. In 12% of cases the number of the analysed lymph nodes was not reported, although there was confirmation of no nodal involvement (N0). Mean follow-up time was 44.4 months for DFS (95% CI 42.1–46.7) and 74.8 months for OS (95% CI 71.9–77.8). Median follow-up time was 56.3 (range 1.2–186.3) months for DFS and 62.8 (range 4.6–186.3) months for OS. Five-year DFS was 65% ([Fig 1A](#pone.0144000.g001)) and 5-year OS was 78% ([Fig 1B](#pone.0144000.g001)). Total relapses were 82 (82/253 = 32%), 37% of staged III (67/181) and in 21% of stage II. Fifty six patients (56/253 = 22%) died during follow-up. Adjuvant therapy was administered as FL in 44% or as FL plus platinum (FL+OXA) in 56% of patients. One hundred and forty five patients (145/253 = 57%) completed the planned cycles for ADJ-CT. ADJ-CT treatment, FL alone or FL+OXA, was not significantly associated with both DFS (*P* = 0.512) and OS (*P* = 0.629) ([Table 1](#pone.0144000.t001)). A significantly shorter OS was associated with males (*P* = 0.019), more advanced tumour stage (III) and rectal tumour location (*P* = 0.050) ([Table 1](#pone.0144000.t001)). Moreover, advanced tumour stage was also associated with a shorter DFS (*P* = 0.010).

### Fig 1. Kaplan-Meier survival curves for Disease Free Survival and Overall Survival according to the genotypes.

![Fig 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8ada/4669157/e7a4e06d8887/pone.0144000.g001.jpg)

(A) Disease Free Survival curve for the total of CRC patients. (B) Overall Survival curve for the total of CRC patients. (C) Disease Free Survival curve for the +2960 14-bp INDEL SNP according to the dominant model. (D) Disease Free Survival curve for the +3035 C>T SNP. (E) Disease Free Survival curve for the +3187 A>G SNP according to the recessive model. (F) Overall Survival curve for the +3187 A>G according to the recessive model.

### HLA-G 3’UTR germinal screening

For each CRC patient the *HLA-G* 3’UTR segment was analyzed by direct sequencing. We detected 9 common SNPs at *HLA-G* 3’UTR: +2960 14-bp INDEL (rs371194629), +3003 T>C (rs1707), +3010 C>G (rs1710), +3027 C>A (rs17179101), +3035 C>T (rs17179108), +3142 G>C (rs1063320), +3187 A>G (rs9380142), +3196 C>G (rs1610696) and +3227 C>A (rs1233331). The germinal allele frequencies for *HLA-G* 3’UTR polymorphisms detected in this set of CRC patients were reported in [S1 Table](#pone.0144000.s003). Only those polymorphisms with ≥5% variant allelic frequency (+2960 14-bp INDEL, +3003 T>C, +3010 C>G, +3027 C>A, +3035 C>T, +3142 G>C, +3187 A>G, and +3196 C>G) were considered for survival (DFS and OS) analysis ([Table 1](#pone.0144000.t001)). The distribution of genotypes of all selected SNPs were in agreement with HW equilibrium ([S1 Table](#pone.0144000.s003)). Strong LD (r^2^ = 0.98) was found between +3010 C>G and +3142 G>C polymorphisms ([S1 Fig](#pone.0144000.s001)). Finally, *HLA-G* 3’UTR haplotypes were reconstructed from the unphased gametic genotype data at the 16 variation sites for each of the 253 individuals affected by CRC by PHASE method. A total of 20 different haplotypes were defined; of these, 14 have been already described [[46](#pone.0144000.ref046)] and 7 (30%) were novel (for more details see [S2 Table](#pone.0144000.s004)). The most represented haplotype was UTR-2 (36%), then UTR-1 (24%), UTR-3 (13%), UTR-4 (11%), UTR-7 (5%), UTR-5 (3%), UTR-18 (3%), UTR-15 (1%) and UTR-6 (1%). These 9 haplotypes represented more than 97% of total, but UTRs 1–4 were predominant (84%).

### Associations between HLA-G 3’UTR SNPs and Disease Free Survival

Among the 8 eligible polymorphisms identified, 4 SNPs (+2960 14-bp INDEL, +3035 C>T, +3142 G>C, and +3187 A>G) in the *HLA-G* 3’UTR region were associated to DFS in univariate analysis ([Table 1](#pone.0144000.t001)). Statistically significant associations between DFS and the polymorphisms +2960 14-bp INDEL (log-rank *P* = 0.036), +3035 C>T (log-rank *P* = 0.051, [Fig 1D](#pone.0144000.g001)), +3142 G>C in the recessive model (log-rank *P* = 0.038), and +3187 A>G (log-rank *P* = 0.019) were found ([Table 1](#pone.0144000.t001)). The polymorphism +2960 14-bp INDEL was also associated to DFS in the dominant (Ins/Del + Ins/Ins *vs* Del/Del) model (log-rank *P* = 0.010, [Fig 1C](#pone.0144000.g001)) and the +3187 A>G SNP in the corresponding recessive (G/G *vs* A/G+A/A) model (log-rank *P* = 0.007, [Fig 1E](#pone.0144000.g001)). In particular, the Del/Del genotype was associated to a reduced 5-year DFS (54%) compared to both Ins/Del (70%) and Ins/Ins genotypes (70%) ([Table 1](#pone.0144000.t001)). The +3035 C/T genotype was borderline significantly associated to prolonged 5-year DFS (77%) than the wild type +3035 C/C (63%) combination. Patients with the +3142 C/C mutated genotype had a reduced 5-year DFS (52% with respect to 68% in subjects carrying the wild type-G/G and heterozygous-G/C genotypes. The presence of the +3187 GG homozygous mutated genotype was associated with a reduced 5-year DFS (33%) compared to wild type +3187 A/A (68%) and the heterozygous +3187 A/G (65%) genotypes. UTR-1 haplotype, the second most represented (24%) in our CRC cohort, was associated to reduced DFS (log-rank *P* = 0.007, [S2A Fig](#pone.0144000.s002)).

We further evaluated the association between the *HLA-G* 3’UTR SNPs and related reconstructed most abundant haplotypes (UTR2-, UTR-1, UTR-3, UTR-4, UTR-7, UTR-5, UTR-18, UTR-15 and UTR-6) and DFS of CRC patients by means of multivariate Cox models ([Table 2](#pone.0144000.t002) and [Table 3](#pone.0144000.t003) for haplotypes).

### Table 2. Associations of HLA-G 3’UTR SNPs with disease free survival and overall survival in 253 stage II-III CRC patients.

|   | Disease Free Survival |  |  |  | Overall Survival |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- |
| HLA-G 3’UTR Genotypes and genetic models | N | Relapses, N | Adjusted HR (95% CI) 1 | P-value | Dead, N | Adjusted HR (95% CI) 1 | P-value |
| +2960 14 bp INDEL (rs371194629) |   |   |   |   |   |   |   |
| Del/Del | 78 | 33 | 1 | - | 22 | 1 | - |
| Ins/Del | 115 | 32 | 0.59 (0.36–0.96) | 0.035 | 23 | 0.72 (0.40–1.30) | 0.269 |
| Ins/Ins | 60 | 17 | 0.61 (0.34–1.10) | 0.097 | 11 | 0.63 (0.30–1.30) | 0.212 |
| Dominant model (Ins/Del+Ins/Ins) | 175 | 49 | 0.60 (0.38–0.93) | 0.023 | 34 | 0.68 (0.40–1.18) | 0.171 |
| Recessive model (Ins/Ins) | 60 | 17 | 0.82 (0.48–1.39) | 0.457 | 11 | 0.76 (0.39–1.46) | 0.406 |
| +3003 T>C (rs1707) |   |   |   |   |   |   |   |
| T/T | 201 | 64 | 1 | - | 45 | 1 | - |
| T/C | 47 | 17 | 1.26 (0.73–2.17) | 0.401 | 10 | 1.01 (0.51–2.03) | 0.968 |
| C/C | 5 | 1 | 0.72 (0.10–5.22) | 0.744 | 1 | 1.15 (0.16–8.42) | 0.895 |
| Dominant model (T/C+C/C) | 52 | 18 | 1.21 (0.71–2.06) | 0.480 | 11 | 1.03 (0.53–2.00) | 0.942 |
| Recessive model (C/C) | 5 | 1 | 0.69 (0.10–4.96) | 0.709 | 1 | 1.14 (0.16–8.36) | 0.896 |
| +3010 C>G (rs1710) |   |   |   |   |   |   |   |
| C/C | 94 | 27 | 1 | - | 19 | 1 | - |
| C/G | 119 | 38 | 1.11 (0.68–1.83) | 0.670 | 24 | 0.99 (0.54–1.82) | 0.973 |
| G/G | 40 | 17 | 1.63 (0.89–3.00) | 0.115 | 13 | 1.60 (0.78–3.26) | 0.197 |
| Dominant model (C/G+G/G) | 159 | 55 | 1.24 (0.78–1.96) | 0.369 | 37 | 1.15 (0.66–2.00) | 0.632 |
| Recessive model (G/G) | 40 | 17 | 1.54 (0.90–2.63) | 0.117 | 13 | 1.61 (0.86–3.02) | 0.140 |
| +3027 C>A (rs17179101) |   |   |   |   |   |   |   |
| C/C | 225 | 77 | 1 | - | 50 | 1 | - |
| C/A | 27 | 5 | 0.54 (0.22–1.35) | 0.186 | 5 | 0.82 (0.33–2.05) | 0.666 |
| +3035 C>T (rs17179108) |   |   |   |   |   |   |   |
| C/C | 205 | 72 | 1 | - | 47 | 1 | - |
| C/T | 47 | 10 | 0.51 (0.26–0.99) | 0.045 | 8 | 0.66 (0.31–1.41) | 0.287 |
| +3142 G>C (rs1063320) |   |   |   |   |   |   |   |
| G/G | 93 | 27 | 1 | - | 19 | 1 | - |
| G/C | 119 | 37 | 1.05 (0.64–1.72) | 0.860 | 24 | 0.96 (0.52–1.75) | 0.882 |
| C/C | 41 | 18 | 1.65 (0.90–3.00) | 0.103 | 13 | 1.54 (0.76–3.15) | 0.233 |
| Dominant model (G/C+C/C) | 160 | 55 | 1.19 (0.75–1.89) | 0.465 | 37 | 1.10 (0.63–1.93) | 0.728 |
| Recessive model (C/C) | 41 | 18 | 1.61 (0.95–2.71) | 0.078 | 13 | 1.59 (0.84–2.98) | 0.153 |
| +3187 A>G (rs9380142) |   |   |   |   |   |   |   |
| A/A | 145 | 42 | 1 | - | 26 | 1 | - |
| A/G | 93 | 31 | 1.16 (0.73–1.86) | 0.529 | 23 | 1.26 (0.71–2.22) | 0.431 |
| G/G | 15 | 9 | 2.61 (1.24–5.50) | 0.012 | 7 | 2.96 (1.22–7.15) | 0.016 |
| Dominant model (A/G+G/G) | 108 | 40 | 1.33 (0.86–2.05) | 0.205 | 30 | 1.46 (0.86–2.48) | 0.162 |
| Recessive model (G/G) | 15 | 9 | 2.46 (1.19–5.05) | 0.015 | 7 | 2.71 (1.16–6.33) | 0.022 |
| +3196 C>G (rs1610696) |   |   |   |   |   |   |   |
| C/C | 106 | 40 | 1 | - | 29 | 1 | - |
| C/G | 107 | 26 | 0.61 (0.37–1.02) | 0.059 | 18 | 0.68 (0.37–1.24) | 0.206 |
| G/G | 40 | 16 | 0.96 (0.54–1.72) | 0.899 | 9 | 0.72 (0.34–1.52) | 0.388 |
| Dominant model (C/G+G/G) | 147 | 42 | 0.72 (0.46–1.12) | 0.142 | 27 | 0.69 (0.41–1.18) | 0.173 |
| Recessive model (G/G) | 40 | 16 | 1.18 (0.68–2.05) | 0.559 | 9 | 0.84 (0.41–1.72) | 0.630 |

Table 2 Caption: 1HRs (Hazard Ratios) adjusted for age, sex, tumour stage, first tumour site and type of adjuvant chemotherapy (FL plus or without OXA). Significant values (≤0.05) are shown in bold.

### Table 3. Associations between HLA-G 3’UTR haplotypes with disease free survival and overall survival in 253 stage II-III CRC patients (N = 506).

|   |   | Disease Free Survival |  |  | Overall Survival |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- |
| HLA-G 3’UTR Haplotypes 1 | N | Relapses, N | Adjusted HR (95% CI) 2 | P-value | Dead, N | Adjusted HR (95% CI) 2 | P-value |
| UTR-2 |   |   |   |   |   |   |   |
| InsTCCCGAGG | 180 | 42 | 1 | - | 26 | 1 | - |
| Het | 108 | 29 | 0.72 (0.44–1.18) | 0.197 | 19 | 0.76 (0.43–1.37) | 0.365 |
| Hom | 72 | 13 | 0.92 (0.49–1.73) | 0.803 | 7 | 0.68 (0.29–1.55) | 0.354 |
| UTR-1 |   |   |   |   |   |   |   |
| DelTGCCCGCG | 120 | 38 | 1 | - | 28 | 1 | - |
| Het | 90 | 29 | 1.08 (0.67–1.74) | 0.745 | 21 | 1.12 (0.63–1.99) | 0.699 |
| Hom | 30 | 9 | 2.53 (1.20–5.32) | 0.014 | 7 | 2.82 (1.17–6.77) | 0.021 |
| UTR-3 |   |   |   |   |   |   |   |
| DelTCCCGACG | 66 | 21 | 1 | - | 14 | 1 | - |
| Het | 56 | 19 | 1.09 (0.65–1.82) | 0.753 | 13 | 1.11 (0.59–2.09) | 0.745 |
| Hom | 10 | 2 | 1.98 (0.48–8.19) | 0.347 | 1 | 1.15 (0.16–8.53) | 0.890 |
| UTR-4 |   |   |   |   |   |   |   |
| DelCGCCCACG | 55 | 17 | 1 | - | 11 | 1 | - |
| Het | 47 | 17 | 1.28 (0.74–2.19) | 0.375 | 11 | 1.18 (0.61–2.31) | 0.623 |
| Hom | 8 | 0 | - | - | 0 | - | - |
| UTR-7 |   |   |   |   |   |   |   |
| InsTCATGACG | 27 | 4 | 1 | - | 5 | 1 | - |
| Het | 25 | 4 | 0.47 (0.17–1.29) | 0.144 | 4 | 0.76 (0.27–2.13) | 0.760 |
| UTR-5 |   |   |   |   |   |   |   |
| InsTCCTGACG | 17 | 4 | 1 | - | 3 | 1 | - |
| Het | 17 | 4 | 0.56 (0.21–1.54) | 0.263 | 3 | 0.74 (0.23–2.39) | 0.617 |
| UTR-18 |   |   |   |   |   |   |   |
| DelTGCCCACA | 15 | 3 | 1 | - | 2 | 1 | - |
| Het | 15 | 3 | 0.55 (0.17–1.74) | 0.307 | 2 | 0.57 (0.14–2.37) | 0.441 |
| UTR-15 |   |   |   |   |   |   |   |
| InsTCCCGACG | 6 | 2 | 1 | - | 1 | 1 | - |
| Het | 4 | 2 | 1.91 (0.46–7.94) | 0.371 | 1 | 0.87 (0.12–6.37) | 0.893 |
| UTR-6 |   |   |   |   |   |   |   |
| DelTGCCCACG | 6 | 3 | 1 | - | 1 | 1 | - |
| Het | 6 | 3 | 1.25 (0.38–4.16) | 0.715 | 1 | 0.58 (0.08–4.33) | 0.596 |

Table 3 Caption: Het, haplotype in heterozygous state; Hom, haplotype in homozygous state; significant values (≤0.05) are shown in bold. 1 HLA-G 3’UTR haplotypes were reconstructed by PHASE method according to worldwide distributions46 2HRs (Hazard Ratios) adjusted for age, sex, tumour stage, first tumour site and type of adjuvant chemotherapy (FL plus or without OXA).

The Ins/Del heterozygous carriers had a significantly reduced risk or recurrence (HR 0.59, 95% CI 0.36–0.96, *P* = 0.035) as well *Ins* allele carriers of (low-HLA-G secretor) according to the dominant model (HR 0.60, 95% CI 0.38–0.93, *P* = 0.023). Forty-two percent of patients with the Del/Del genotype (N = 78) had a recurrence (33/78), while a lower incidence (28%) was observed in patients with Ins/Del (N = 115) and Ins/Ins (N = 60) genotypes ([Table 2](#pone.0144000.t002)).

A borderline statistically significant association with reduced risk of disease recurrence was determined in patients carrying the +3035 C/T (low-HLA-G secretor) genotype (HR 0.51, 95% CI 0.26–0.99, *P* = 0.045). The 21% of heterozygous (C/T) patients for +3035 C>T SNP had a relapse (10/47) with respect to the 35% found in the wild type +3035 C/C combination (72/205). CRC patients carrying the +3187 G/G mutated genotype (high-HLA-G secretor) were associated with an increased risk or relapse (HR 2.61, 95% CI 1.24–5.50, *P* = 0.012 and consistently according to the recessive model (HR 2.46, 95% CI 1.19–5.05, *P* = 0.015). A detectable relapse was observed in the 60% (9/15) of CRC patients with the +3187 G/G mutated genotype (N = 15), in 33% (31/93) of heterozygous A/G (N = 93), and in 29% (42/145) of wild-type A/A (N = 145) patients.

UTR-1 haplotype, containing the +3187*G* mutated allele, was associated with poor prognosis in DFS when present in double (Hom) dose (HR 2.53, 95% CI 1.20–5.32, *P* = 0.014) ([Table 3](#pone.0144000.t003)). The HR of the +3035 C/T and +2960 14-bp INDEL combined effect confirmed the protective role of both SNPs in DFS (HR 0.40, 95% CI 0.20–0.82, *P* = 0.013).

### Associations between HLA-G 3’UTR SNPs and Overall Survival

The +3187 A>G SNP was the only one to be associated with OS (log-rank *P* = 0.036), and in the corresponding recessive model (log-rank *P* = 0.025, [Fig 1F](#pone.0144000.g001)) in univariate analysis ([Table 1](#pone.0144000.t001)). Similarly, UTR-1 haplotype was associated to OS (log-rank *P* = 0.025, [S2B Fig](#pone.0144000.s002)). In multivariate analysis carriers for the +3187 G/G genotype (high-HLA-G secretor) were associated with reduced OS (HR 2.96, 95% CI 1.22–7.15, *P* = 0.016) and consistently in the recessive model (HR 2.71, 95% CI 1.16–6.33, *P* = 0.022). Mortality rate was higher (47%) in homozygous mutated +3187 GG (7/15) carriers than in the heterozygous +3187 A/G (25%, 23/93) and in the wild type +3187 A/A (18%, 26/145) carriers ([Table 2](#pone.0144000.t002)). Multivariate analysis for most represented haplotypes showed an association between UTR-1 haplotype in double (Hom) dose and diminished OS (HR 2.82, 95% CI 1.17–6.77, *P* = 0.021) ([Table 3](#pone.0144000.t003)). The observed associations in both DFS and OS were unmodified in a bootstrap model confirming the internal validity of the associations observed (not shown).

The relation between types of polymorphisms and DFS or OS risk, was further examined stratifying by stage (II and III) (not shown). Although some differences in the hazard ratios were detected across strata, the observed associations were still confirmed. Moreover, these associations were compatible with the effect of random variation since heterogeneity tests were not significant.

## Discussion

Emerging data demonstrates a key role of genes involved in immune response checkpoints and their associations with the CRC clinical outcomes [[47](#pone.0144000.ref047)]. In order to progress, malignant tumours must elude or evade the host’s immune system. In the quest to develop personalized cancer therapies, researchers are increasingly examining the patient’s immune response to cancer. SNPs within genes involved in immune response should be helpful to define the immunogenetic profile of the patients and to improve treatment strategies modulating anti-tumor immune response by targeting novel immune checkpoints. Improvement in immunosurveillance mechanisms may be achieved by means of immunotherapies with monoclonal antibodies and through chemotherapies and radiotherapies [[48](#pone.0144000.ref048)]. The purpose of personalized medicine is to identify the optimal treatment for each individual patient to maximize benefits and minimize adverse effects. To achieve this goal, novel informative biomarkers and new approaches to optimize clinical outcomes are needed in order to better stratify patients for cancer care.

The potential clinical relevance of HLA-G in cancer as a negative regulator due to its direct or indirect tolerogenic properties to avoid immune cells response, was previously highlighted in several studies [[17](#pone.0144000.ref017),[24](#pone.0144000.ref024)]. However, HLA-G molecule may counteract or elicit the progression of cancer as a consequence of its immune-modulatory properties regulated by SNPs present in the untranslated regions [[15](#pone.0144000.ref015)]. Previous studies have reported significant associations between *HLAG* polimorphisms (in particular the +2960 14-bp INDEL) and cancer risk [[30](#pone.0144000.ref030),[34](#pone.0144000.ref034)], but to the best of our knowledge, this is the first study indicating a role for *HLA-G* 3’UTR regulatory SNPs in DFS and OS after adjuvant treatment of CRC. Our results emphasize the role of the host’s immunogenetic background in the CRC prognosis as well as report the molecular characterization of the 3’UTR region at the germinal level in subjects affected by colorectal cancer [[42](#pone.0144000.ref042)].

The common *HLA-G* 3’UTR polymorphisms were investigated and after multivariate survival analyses using Cox’s regression models we found that +2960 14-bp INDEL, +3035 C>T and +3187 A>G SNPs had a significant and independent prognostic role with high internal validity by bootstrap modeling. *HLAG* 3’UTR SNPs previously reported to be associated with a reduced protein production such as +2960 14-bp INDEL and +3035 C>T were linked to a better prognosis, whereas the +3187 A>G SNP (increased HLAG production), was associated to worse DFS and OS.

This study shows that the +2960 14-bp INDEL SNP, already described as a disease risk-marker, is also a prognostic marker for DFS in CRC patients treated with standard ADJ-CT. Intriguingly, our results on survival for +2960 14-bp INDEL SNP are in agreement with those observed in a non-solid tumour [[31](#pone.0144000.ref031)] and also in patients infected by the human immunodeficiency virus (HIV) [[49](#pone.0144000.ref049)], further highlighting the prognostic relevance for the 14-bp Ins/Del polymorphism. The +2960 14-bp INDEL (Ins/Del) SNP (rs371194629) was reported to modulate the magnitude of HLA-G production by regulating HLA-G mRNA stability [[25](#pone.0144000.ref025)]. In particular, the Del/Del genotype has been associated with high and stable *HLA-G* mRNA expression and higher levels of the soluble HLA-G, whereas the Ins/Ins genotype displays a lower production of mRNA and soluble or membrane bound molecules [[29](#pone.0144000.ref029)–[32](#pone.0144000.ref032)]. At the multivariate analysis, we estimated an association with a reduced DFS in patients who are carriers of *Ins* allele in the heterozygous Ins/Del patients and in (Ins/Del+Ins/Ins) agreement with the dominant model (HR 0.60, 95% 0.38–0.93, *P* = 0.023). CRC patients with the Del/Del genotype showed an increased relapse rate and reduced 5-year DFS %.

Concerning OS, we found a similar trend of improved prognosis for *Ins* allele even if not statistically significant probably due to the small sample size. In reference to SNP +3035 C>T (rs17179108), it has been reported, in a recent published study [[29](#pone.0144000.ref029)], that subjects presenting the +3035 C/T genotype had significantly lower levels of the soluble HLA-G compared to +3035 C/C (wild type) subjects. We observed a protective role of +3035 C/T genotype in the outcome (DFS) of CRC patients, though with a borderline (HR 0.51, 95% 0.26–0.99, *P* = 0.045) statistically significant association in the multivariate Cox’s model. The +3035 C/T genotype was associated to prolonged 5-year DFS (77%) and a lower (21%) recurrence incidence (10/47) with respect to the wild type +3035 C/C combination that presented a decreased 5-year DFS (63%) and a 35% of relapse (72/205). CRC carriers for +3035 C/C had a decreased 5-year DFS % and an increased relapse incidence. It should be pointed out that the +3035 C/T heterozygous genotype, detected in 47 patients, was always associated to the *Ins* allele, in heterozygous Ins/Del (N = 27) and in homozygous Ins/Ins (N = 20) patients. Even if a protective HR in DFS resulted in the combined analysis of +3035 C/T and +2960 14-bp INDEL polymorphisms, no firm conclusion about a multiplicative or additive effect of the 2 SNPs cannot be inferred from this study. Regarding the +3187 A>G polymorphism (rs9380142), the +3187*A* allele has been associated to decreased HLA-G expression and the presence of +3187 G/G genotype to significantly increased soluble levels of HLA-G [[29](#pone.0144000.ref029),[50](#pone.0144000.ref050)]. To date, no association with survival and this *HLA-*G SNP was reported. We found an association between +3187 G/G carriers (HR 2.61, 95% CI 1.24–5.50, *P* = 0.012), and according to the recessive (G/G *vs* A/A+A/G) model (HR 2.46, 95% CI 1.19–5.05, *P* = 0.015) with reduced DFS. Similarly, the +3187 G/G carriers were associated to a reduced OS (HR 2.96, 95% CI 1.22–7.15, *P* = 0.016), also in the recessive model (HR 2.71, 95% CI 1.16–6.63, *P* = 0.022). These results may highlight that the modulation of the clinical outcome in CRC patients harbouring the +3187 A>G change is due to the contribution of the *G* allele in double dose. CRC patients carriers for +3187 G/G mutated genotype (N = 15) had increased recurrence rate and a reduced 5-year DFS %. Carriers of +3187 G/G mutated combination were also associated to a reduced 5-year OS percentage and increased mortality rate. The opposite prognostic associations found in CRC patients for SNPs +2960 14-bp INDEL and +3187 A>G, are corroborated by the evidence that these two polymorphisms are not in LD (r^2^ = 0.27). Furthermore, presence of +3187*G* allele is always associated to the *Del* allele, which are represented in the reconstructed UTR-1 haplotype. The latter, when present in double dose (UTR-1/UTR-1), has been associated to an unfavourable prognosis such as +3187 G/G genotype.

The variation sites described in the 3’UTR are mainly arranged in haplotypes (known as UTR-1 to UTR-44) with the UTR-1 and UTR-2 as the most frequent in the worldwide population [[46](#pone.0144000.ref046)]. Therefore, we performed a haplotype analysis on *HLA-G* 3’UTR variants to test whether haplotypes are more predictive than single variants.

UTR-1 (**Del**C**TGC**G**C**CGCGT**CGC**G) haplotype carrying the 14-bp *Del*, +3003*T*, +3010*G*, +3027*C*, +3035*C*, +3142*C*, +3187*G* and +3196*C* alleles, has been considered as a high expressing haplotype. In particular, individuals with the +3187 G/G genotype and thus the UTR-1/UTR-1 combination in double dose, exhibit significant higher levels of the soluble HLA-G [[29](#pone.0144000.ref029),[51](#pone.0144000.ref051)]. UTR-1/UTR-1 haplotype combination shares characteristics of high HLA-G producer presenting the 14-bp Del/Del, the 3142 C/C and the 3187 G/G genotypes. UTR-1 was the only *HLA-G* 3’UTR haplotype associated with prognosis in CRC patients in multivariate regression Cox’s analysis. We found an association of reduced DFS (HR 2.53, 95% CI 1.20–5.32, *P* = 0.014) and OS (HR 2.82, 95% CI 1.17–6.77, *P* = 0.021) in CRC patients carrying the UTR-1 haplotype in double (Hom) dose. Estimations found are not surprising since patients homozygous for UTR-1 haplotype (N = 15) are the same carriers for +3187 G/G change and share the same survival pattern.

Moreover, we observed a strong LD ([S1 Fig](#pone.0144000.s001), available online) between the +3010 C>G (rs1710) and +3142 G>C (rs1063320) SNPs, consistently with the data reported for the worldwide population [[46](#pone.0144000.ref046)]. Both polymorphisms were not associated to DFS or OS in multivariate analysis ([Table 2](#pone.0144000.t002)). In summary, our results demonstrates an independent potential prognostic value after multivariate analysis for three *HLA-G* 3’UTR polymorphisms, the +2960 14-bp INDEL, the +3035 C>T, and +3187 A>G. CRC patients carrying the *Ins* allele (lower HLA-G producer) were associated with a favourable DFS with a reduced risk of relapse (protective prognostic role). CRC patients carriers for the +3187 G/G genotype and UTR-1/UTR-1 haplotype (higher HLA-G producer) were associated with an unfavourable prognosis in both DFS and OS. Furthermore, patients carrying the +3035 C/T genotype (lower HLA-G producer) and therefore the *Ins* allele (in heterozygous or homozygous state) were associated with an improved prognosis though with a borderline significant association.

Finally, this study supports associations between the non-metastatic colorectal cancer outcome after CT treatment and polymorphisms of a gene involved in immune tumour escape. Our preliminary findings share a functional rationale considering that patients with higher levels of HLA-G would be more immunosuppressed and have a worse clinical prognosis as postulated by Rizzo *et al* [[31](#pone.0144000.ref031)]. The effect of immune surveillance on the outcome of patients after chemotherapeutic treatments is one of the currently attracting issues in cancer therapy. It is well recognized that the individual variability of drug response depends also on the genetic variations in human genome, thus enforcing the concept of personalized medicine [[52](#pone.0144000.ref052)]. Due to the functional impact of the HLA-G protein in cancer immune contexture, and the known correlations of functional-regulatory SNPs in the 3’UTR with the HLA-G protein level, the concept of germline genomic variation is very attractive. Only a small sample of blood is required for the genetic test, and the genotyping process is a standard and common method used in clinical practice. In addition, it should be especially considered for inoperable patients. Future perspectives will be analyze *HLA-G* 3’UTR polymorphisms in the outcome of metastatic CRC patients. These results could provide new insights to better stratify patients and also for combination therapy between CT and monoclonal antibodies (i.e. cetuximab and bevacizumab) currently adopted in clinical practice.

Some limitations of this work should be highlighted. Despite an internal validation (bootstrap analysis) confirming the results, *HLA-G* 3’UTR SNPs should be analyzed for their prognostic role in an independent CRC cohort treated with surgical resection and without any chemotherapy. Moreover, our data should be explored considering also the contribution of validated prognostic and predictive biomarkers for CRC. Another limit is the lack of biological samples such as tumour or plasma samples to correlate genetic data with tissue and/or soluble levels of HLA-G to sustain functional hypothesis about the regulatory role of these SNPs, even if previous published studies support consistent data [[29](#pone.0144000.ref029)–[32](#pone.0144000.ref032),[51](#pone.0144000.ref051)]. Nonetheless, these novel findings presented here for the role of *HLA-G* 3’UTR region in prognosis of colorectal cancer provide the basis for implementation of personalized cancer treatments. Identification of high risk patients with well-recognized prognostic, predictive and novel immune-related genomic biomarkers may represent a new frontier in the management of CRC patients. In conclusion, +3035 C>T and in particular, +2960 14-bp INDEL and +3187 A>G polymorphisms in the regulatory 3’UTR of the *HLA-G* gene, have emerged as novel prognostic biomarkers in determining survival outcome in colorectal cancer. Therefore, our exploratory findings should be verified in independent larger CRC cohorts as well as further relevant functional studies are required.

## Supporting Information

## Acknowledgments

We would like to thank Mrs Luigina Mei for her invaluable English language assistance.

## Data Availability

All relevant data are within the paper and its Supporting Information files.

## Funding Statement

This work was supported by grant «Application of advanced nanotechnology in the development of innovative cancer diagnostic tools», AIRCx1000 Special Program Molecular Oncology (Grant number 12214).

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Data Availability Statement

All relevant data are within the paper and its Supporting Information files.

### Supplementary Materials

### Data Availability Statement

All relevant data are within the paper and its Supporting Information files.

## References

1. Siegel R, Desantis C, Jemal A. Colorectal cancer statistics. CA Cancer J Clin 2014;64:104–17. 10.3322/caac.21220  [DOI](https://doi.org/10.3322/caac.21220) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24639052/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=CA%20Cancer%20J%20Clin&title=Colorectal%20cancer%20statistics&author=R%20Siegel&author=C%20Desantis&author=A%20Jemal&volume=64&publication_year=2014&pages=104-17&pmid=24639052&doi=10.3322/caac.21220&)

2. Ferlay J, Steliarova-Foucher E, Lortet-Tieulent J, Rosso S, Coebergh JW, Comber H, et al. Cancer incidence and mortality patterns in Europe: Estimates for 40 countries in 2012. Eur J Cancer 2013;49:1374–403. 10.1016/j.ejca.2012.12.027  [DOI](https://doi.org/10.1016/j.ejca.2012.12.027) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23485231/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Cancer&title=Cancer%20incidence%20and%20mortality%20patterns%20in%20Europe:%20Estimates%20for%2040%20countries%20in%202012&author=J%20Ferlay&author=E%20Steliarova-Foucher&author=J%20Lortet-Tieulent&author=S%20Rosso&author=JW%20Coebergh&volume=49&publication_year=2013&pages=1374-403&pmid=23485231&doi=10.1016/j.ejca.2012.12.027&)

3. Johnston PG. Identification of clinically relevant molecular subtypes in colorectal cancer: the dawning of a new era. Oncologist 2014;19:568–73. 10.1634/theoncologist.2014-038  [DOI](https://doi.org/10.1634/theoncologist.2014-038) | [PMC free article](/articles/PMC4012975/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24718513/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncologist&title=Identification%20of%20clinically%20relevant%20molecular%20subtypes%20in%20colorectal%20cancer:%20the%20dawning%20of%20a%20new%20era&author=PG%20Johnston&volume=19&publication_year=2014&pages=568-73&pmid=24718513&doi=10.1634/theoncologist.2014-038&)

4. Duffy MJ. Personalized treatment for patients with colorectal cancer: role of biomarkers. Biomark Med 2015;9:337–47. 10.2217/bmm.15.3  [DOI](https://doi.org/10.2217/bmm.15.3) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25808438/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biomark%20Med&title=Personalized%20treatment%20for%20patients%20with%20colorectal%20cancer:%20role%20of%20biomarkers&author=MJ%20Duffy&volume=9&publication_year=2015&pages=337-47&pmid=25808438&doi=10.2217/bmm.15.3&)

5. Goel G. Evolving role of gene expression signatures as biomarkers in early-stage colon cancer. J Gastrointest Cancer 2014;45:399–404. 10.1007/s12029-014-9634-7  [DOI](https://doi.org/10.1007/s12029-014-9634-7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24989938/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Gastrointest%20Cancer&title=Evolving%20role%20of%20gene%20expression%20signatures%20as%20biomarkers%20in%20early-stage%20colon%20cancer&author=G%20Goel&volume=45&publication_year=2014&pages=399-404&pmid=24989938&doi=10.1007/s12029-014-9634-7&)

6. Broadbridge VT, Karapetis CS, Beeke C, Woodman RJ, Padbury R, Maddern G, et al. Do metastatic colorectal cancer patients who present with late relapse after curative surgery have a better survival? Br J Cancer 2013;109:1338–343. 10.1038/bjc.2013.388  [DOI](https://doi.org/10.1038/bjc.2013.388) | [PMC free article](/articles/PMC3778277/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23860523/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Cancer&title=Do%20metastatic%20colorectal%20cancer%20patients%20who%20present%20with%20late%20relapse%20after%20curative%20surgery%20have%20a%20better%20survival?&author=VT%20Broadbridge&author=CS%20Karapetis&author=C%20Beeke&author=RJ%20Woodman&author=R%20Padbury&volume=109&publication_year=2013&pages=1338-343&pmid=23860523&doi=10.1038/bjc.2013.388&)

7. Kopetz S, Tabernero J, Rosenberg R, Jiang ZQ, Moreno V, Bachleitner-Hofmann T, et al. Genomic classifier ColoPrint predicts recurrence in stage II colorectal cancer patients more accurately than clinical factors. Oncologist 2015;20:127–33. 10.1634/theoncologist.2014-0325  [DOI](https://doi.org/10.1634/theoncologist.2014-0325) | [PMC free article](/articles/PMC4319631/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25561511/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncologist&title=Genomic%20classifier%20ColoPrint%20predicts%20recurrence%20in%20stage%20II%20colorectal%20cancer%20patients%20more%20accurately%20than%20clinical%20factors&author=S%20Kopetz&author=J%20Tabernero&author=R%20Rosenberg&author=ZQ%20Jiang&author=V%20Moreno&volume=20&publication_year=2015&pages=127-33&pmid=25561511&doi=10.1634/theoncologist.2014-0325&)

8. Fehlker M, Huska MR, Jöns T, Andrade-Navarro MA, Kemmner W. Concerted down-regulation of immune-system related genes predicts metastasis in colorectal carcinoma. BMC Cancer 2014;14:64 10.1186/1471-2407-14-64  [DOI](https://doi.org/10.1186/1471-2407-14-64) | [PMC free article](/articles/PMC3922093/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24495478/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Cancer&title=Concerted%20down-regulation%20of%20immune-system%20related%20genes%20predicts%20metastasis%20in%20colorectal%20carcinoma&author=M%20Fehlker&author=MR%20Huska&author=T%20J%C3%B6ns&author=MA%20Andrade-Navarro&author=W%20Kemmner&volume=14&publication_year=2014&pages=64&pmid=24495478&doi=10.1186/1471-2407-14-64&)

9. Ong CW, Chong PY, McArt DG, Chan JY, Tan HT, Kumar AP, et al. The prognostic value of the stem-like group in colorectal cancer using a panel of immunohistochemistry markers. Oncotarget 2015. 6:12763–73.  [DOI](https://doi.org/10.18632/oncotarget.3497) | [PMC free article](/articles/PMC4494972/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25906747/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncotarget&title=The%20prognostic%20value%20of%20the%20stem-like%20group%20in%20colorectal%20cancer%20using%20a%20panel%20of%20immunohistochemistry%20markers&author=CW%20Ong&author=PY%20Chong&author=DG%20McArt&author=JY%20Chan&author=HT%20Tan&volume=6&publication_year=2015&pages=12763-73&pmid=25906747&doi=10.18632/oncotarget.3497&)

10. Roxburgh CS, Richards CH, Macdonald AI, Powell AG, McGlynn LM, McMillan DC, et al. The in situ local immune response, tumour senescence and proliferation in colorectal cancer. Br J Cancer 2013;109:2207–16. 10.1038/bjc.2013.556  [DOI](https://doi.org/10.1038/bjc.2013.556) | [PMC free article](/articles/PMC3798960/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24022192/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Cancer&title=The%20in%20situ%20local%20immune%20response,%20tumour%20senescence%20and%20proliferation%20in%20colorectal%20cancer&author=CS%20Roxburgh&author=CH%20Richards&author=AI%20Macdonald&author=AG%20Powell&author=LM%20McGlynn&volume=109&publication_year=2013&pages=2207-16&pmid=24022192&doi=10.1038/bjc.2013.556&)

11. Reissfelder C, Stamova S, Gossmann C, Braun M, Bonertz A, Walliczek U, et al. Tumor-specific cytotoxic T lymphocyte activity determines colorectal cancer patient prognosis. J Clin Invest 2015;125:739–51. 10.1172/JCI74894  [DOI](https://doi.org/10.1172/JCI74894) | [PMC free article](/articles/PMC4319435/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25562322/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Invest&title=Tumor-specific%20cytotoxic%20T%20lymphocyte%20activity%20determines%20colorectal%20cancer%20patient%20prognosis&author=C%20Reissfelder&author=S%20Stamova&author=C%20Gossmann&author=M%20Braun&author=A%20Bonertz&volume=125&publication_year=2015&pages=739-51&pmid=25562322&doi=10.1172/JCI74894&)

12. Angelova M, Charoentong P, Hackl H, Fischer ML, Snajder R, Krogsdam AM, et al. Characterization of the immunophenotypes and antigenomes of colorectal cancers reveals distinct tumor escape mechanisms and novel targets for immunotherapy. Genome Biol 2015;16:64 10.1186/s13059-015-0620-6  [DOI](https://doi.org/10.1186/s13059-015-0620-6) | [PMC free article](/articles/PMC4377852/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25853550/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Biol&title=Characterization%20of%20the%20immunophenotypes%20and%20antigenomes%20of%20colorectal%20cancers%20reveals%20distinct%20tumor%20escape%20mechanisms%20and%20novel%20targets%20for%20immunotherapy&author=M%20Angelova&author=P%20Charoentong&author=H%20Hackl&author=ML%20Fischer&author=R%20Snajder&volume=16&publication_year=2015&pages=64&pmid=25853550&doi=10.1186/s13059-015-0620-6&)

13. Schreiber RD, Old LJ, Smyth MJ. Cancer immunoediting: integrating immunity’s roles in cancer suppression and promotion. Science 2011;331:1565–70. 10.1126/science.1203486  [DOI](https://doi.org/10.1126/science.1203486) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21436444/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=Cancer%20immunoediting:%20integrating%20immunity%E2%80%99s%20roles%20in%20cancer%20suppression%20and%20promotion&author=RD%20Schreiber&author=LJ%20Old&author=MJ%20Smyth&volume=331&publication_year=2011&pages=1565-70&pmid=21436444&doi=10.1126/science.1203486&)

14. Bukur J, Jasinski S, Seliger B. The role of classical and nonclassical HLA class I antigens in human tumors. Semin Cancer Biol 2012;22:350–58. 10.1016/j.semcancer.2012.03.003  [DOI](https://doi.org/10.1016/j.semcancer.2012.03.003) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22465194/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Semin%20Cancer%20Biol&title=The%20role%20of%20classical%20and%20nonclassical%20HLA%20class%20I%20antigens%20in%20human%20tumors&author=J%20Bukur&author=S%20Jasinski&author=B%20Seliger&volume=22&publication_year=2012&pages=350-58&pmid=22465194&doi=10.1016/j.semcancer.2012.03.003&)

15. Rouas-Freiss N, Moreau P, LeMaoult J, Carosella ED. The dual role of HLA-G in cancer. J Immunol Res 2014;2014:359748 10.1155/2014/359748  [DOI](https://doi.org/10.1155/2014/359748) | [PMC free article](/articles/PMC3995100/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24800261/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Immunol%20Res&title=The%20dual%20role%20of%20HLA-G%20in%20cancer&author=N%20Rouas-Freiss&author=P%20Moreau&author=J%20LeMaoult&author=ED%20Carosella&volume=2014&publication_year=2014&pages=359748&pmid=24800261&doi=10.1155/2014/359748&)

16. Curigliano G, Criscitiello C, Gelao L, Goldhirsch A. Molecular pathways: human leukocyte antigen G (HLA-G). Clin Cancer Res 2013;19:5564–71. 10.1158/1078-0432.CCR-12-3697  [DOI](https://doi.org/10.1158/1078-0432.CCR-12-3697) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23897901/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&title=Molecular%20pathways:%20human%20leukocyte%20antigen%20G%20(HLA-G)&author=G%20Curigliano&author=C%20Criscitiello&author=L%20Gelao&author=A%20Goldhirsch&volume=19&publication_year=2013&pages=5564-71&pmid=23897901&doi=10.1158/1078-0432.CCR-12-3697&)

17. Carosella ED, Favier B, Rouas-Freiss N, Moreau P, Lemaoult J. Beyond the increasing complexity of the immunomodulatory HLA-G molecule. Blood 2008;111:4862–70. 10.1182/blood-2007-12-127662  [DOI](https://doi.org/10.1182/blood-2007-12-127662) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18334671/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Beyond%20the%20increasing%20complexity%20of%20the%20immunomodulatory%20HLA-G%20molecule&author=ED%20Carosella&author=B%20Favier&author=N%20Rouas-Freiss&author=P%20Moreau&author=J%20Lemaoult&volume=111&publication_year=2008&pages=4862-70&pmid=18334671&doi=10.1182/blood-2007-12-127662&)

18. Amodio G, Sales de Albuquerque R, Gregori S. New insights into HLA-G mediated tolerance. Tissue Antigens 2014;84:255–63. 10.1111/tan.12427  [DOI](https://doi.org/10.1111/tan.12427) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25132109/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Tissue%20Antigens&title=New%20insights%20into%20HLA-G%20mediated%20tolerance&author=G%20Amodio&author=R%20Sales%20de%20Albuquerque&author=S%20Gregori&volume=84&publication_year=2014&pages=255-63&pmid=25132109&doi=10.1111/tan.12427&)

19. Donadi EA, Castelli EC, Arnaiz-Villena A, Roger M, Rey D, Moreau P. Implications of the polymorphism of HLA-G on its function, regulation, evolution and disease association. Cell Mol Life Sci 2011;68:369–95. 10.1007/s00018-010-0580-7  [DOI](https://doi.org/10.1007/s00018-010-0580-7) | [PMC free article](/articles/PMC3021195/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21107637/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell%20Mol%20Life%20Sci&title=Implications%20of%20the%20polymorphism%20of%20HLA-G%20on%20its%20function,%20regulation,%20evolution%20and%20disease%20association&author=EA%20Donadi&author=EC%20Castelli&author=A%20Arnaiz-Villena&author=M%20Roger&author=D%20Rey&volume=68&publication_year=2011&pages=369-95&pmid=21107637&doi=10.1007/s00018-010-0580-7&)

20. Gimenes F, Teixeira JJ, de Abreu AL, Souza RP, Pereira MW, da Silva VR, et al. Human leukocyte antigen (HLA)-G and cervical cancer immunoediting: A candidate molecule for therapeutic intervention and prognostic biomarker? Biochim Biophys Acta 2014;1846:576–89. 10.1016/j.bbcan.2014.10.004  [DOI](https://doi.org/10.1016/j.bbcan.2014.10.004) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25453366/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochim%20Biophys%20Acta&title=Human%20leukocyte%20antigen%20(HLA)-G%20and%20cervical%20cancer%20immunoediting:%20A%20candidate%20molecule%20for%20therapeutic%20intervention%20and%20prognostic%20biomarker?&author=F%20Gimenes&author=JJ%20Teixeira&author=AL%20de%20Abreu&author=RP%20Souza&author=MW%20Pereira&volume=1846&publication_year=2014&pages=576-89&pmid=25453366&doi=10.1016/j.bbcan.2014.10.004&)

21. Cai MY, Xu YF, Qiu SJ, Ju MJ, Gao Q, Li YW, et al. Human Leukocyte Antigen-G Protein Expression Is an Unfavorable Prognostic Predictor of Hepatocellular Carcinoma following Curative Resection. Clin Cancer Res 2009;15:4686–93. 10.1158/1078-0432.CCR-09-0463  [DOI](https://doi.org/10.1158/1078-0432.CCR-09-0463) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19584149/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&title=Human%20Leukocyte%20Antigen-G%20Protein%20Expression%20Is%20an%20Unfavorable%20Prognostic%20Predictor%20of%20Hepatocellular%20Carcinoma%20following%20Curative%20Resection&author=MY%20Cai&author=YF%20Xu&author=SJ%20Qiu&author=MJ%20Ju&author=Q%20Gao&volume=15&publication_year=2009&pages=4686-93&pmid=19584149&doi=10.1158/1078-0432.CCR-09-0463&)

22. Castelli EC, Mendes-Junior CT, Veiga-Castelli LC, Roger M, Moreau P, Donadi EA. A comprehensive study of polymorphic sites along the HLA-G gene: implication for gene regulation and evolution. Mol Biol Evol 2011;28:3069–86. 10.1093/molbev/msr138  [DOI](https://doi.org/10.1093/molbev/msr138) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21622995/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Biol%20Evol&title=A%20comprehensive%20study%20of%20polymorphic%20sites%20along%20the%20HLA-G%20gene:%20implication%20for%20gene%20regulation%20and%20evolution&author=EC%20Castelli&author=CT%20Mendes-Junior&author=LC%20Veiga-Castelli&author=M%20Roger&author=P%20Moreau&volume=28&publication_year=2011&pages=3069-86&pmid=21622995&doi=10.1093/molbev/msr138&)

23. Castelli EC, Mendes-Junior CT, Deghaide NH, de Albuquerque RS, Muniz YC, Simões RT, et al. The genetic structure of 3’untranslated region of the HLA-G gene: polymorphisms and haplotypes. Genes Immun 2010;11:134–41. 10.1038/gene.2009.74  [DOI](https://doi.org/10.1038/gene.2009.74) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19798077/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genes%20Immun&title=The%20genetic%20structure%20of%203%E2%80%99untranslated%20region%20of%20the%20HLA-G%20gene:%20polymorphisms%20and%20haplotypes&author=EC%20Castelli&author=CT%20Mendes-Junior&author=NH%20Deghaide&author=RS%20de%20Albuquerque&author=YC%20Muniz&volume=11&publication_year=2010&pages=134-41&pmid=19798077&doi=10.1038/gene.2009.74&)

24. Carosella ED, Moreau P, Lemaoult J, Rouas-Freiss N. HLA-G: from biology to clinical benefits. Trends Immunol 2008;29:125–32. 10.1016/j.it.2007.11.005  [DOI](https://doi.org/10.1016/j.it.2007.11.005) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18249584/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Trends%20Immunol&title=HLA-G:%20from%20biology%20to%20clinical%20benefits&author=ED%20Carosella&author=P%20Moreau&author=J%20Lemaoult&author=N%20Rouas-Freiss&volume=29&publication_year=2008&pages=125-32&pmid=18249584&doi=10.1016/j.it.2007.11.005&)

25. Hviid TV, Hylenius S, Rørbye C, Nielsen LG. HLA-G allelic variants are associated with differences in the HLA-G mRNA isoform profile and HLA-G mRNA levels. Immunogenetics 2003;55:63–79.  [DOI](https://doi.org/10.1007/s00251-003-0547-z) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12712263/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Immunogenetics&title=HLA-G%20allelic%20variants%20are%20associated%20with%20differences%20in%20the%20HLA-G%20mRNA%20isoform%20profile%20and%20HLA-G%20mRNA%20levels&author=TV%20Hviid&author=S%20Hylenius&author=C%20R%C3%B8rbye&author=LG%20Nielsen&volume=55&publication_year=2003&pages=63-79&pmid=12712263&doi=10.1007/s00251-003-0547-z&)

26. Larsen MH, Hviid TV. Human leukocyte antigen-G polymorphism in relation to expression, function, and disease. Hum Immunol 2009;70:1026–34. 10.1016/j.humimm.2009.07.015  [DOI](https://doi.org/10.1016/j.humimm.2009.07.015) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19651180/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Immunol&title=Human%20leukocyte%20antigen-G%20polymorphism%20in%20relation%20to%20expression,%20function,%20and%20disease&author=MH%20Larsen&author=TV%20Hviid&volume=70&publication_year=2009&pages=1026-34&pmid=19651180&doi=10.1016/j.humimm.2009.07.015&)

27. Castelli EC, Veiga-Castelli LC, Yaghi L, Moreau P, Donadi EA. Transcriptional and posttranscriptional regulations of the HLA-G gene. J Immunol Res 2014;2014:734068 10.1155/2014/734068  [DOI](https://doi.org/10.1155/2014/734068) | [PMC free article](/articles/PMC3987962/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24741620/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Immunol%20Res&title=Transcriptional%20and%20posttranscriptional%20regulations%20of%20the%20HLA-G%20gene&author=EC%20Castelli&author=LC%20Veiga-Castelli&author=L%20Yaghi&author=P%20Moreau&author=EA%20Donadi&volume=2014&publication_year=2014&pages=734068&pmid=24741620&doi=10.1155/2014/734068&)

28. Porto IO, Mendes-Junior CT, Felício LP, Georg RC, Moreau P, et al. MicroRNAs targeting the immunomodulatory HLA-G gene: a new survey searching for microRNAs with potential to regulate HLA-G. Mol Immunol 2015;65:230–41. 10.1016/j.molimm.2015.01.030  [DOI](https://doi.org/10.1016/j.molimm.2015.01.030) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25700346/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Immunol&title=MicroRNAs%20targeting%20the%20immunomodulatory%20HLA-G%20gene:%20a%20new%20survey%20searching%20for%20microRNAs%20with%20potential%20to%20regulate%20HLA-G&author=IO%20Porto&author=CT%20Mendes-Junior&author=LP%20Fel%C3%ADcio&author=RC%20Georg&author=P%20Moreau&volume=65&publication_year=2015&pages=230-41&pmid=25700346&doi=10.1016/j.molimm.2015.01.030&)

29. Martelli-Palomino G, Pancotto JA, Muniz YC, Mendes-Junior CT, Castelli EC, Massaro JD, et al. Polymorphic sites at the 3' untranslated region of the HLA-G gene are associated with differential hla-g soluble levels in the Brazilian and French population. PLoS One 2013;8:e71742 10.1371/journal.pone.0071742  [DOI](https://doi.org/10.1371/journal.pone.0071742) | [PMC free article](/articles/PMC3808361/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24204558/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=Polymorphic%20sites%20at%20the%203'%20untranslated%20region%20of%20the%20HLA-G%20gene%20are%20associated%20with%20differential%20hla-g%20soluble%20levels%20in%20the%20Brazilian%20and%20French%20population&author=G%20Martelli-Palomino&author=JA%20Pancotto&author=YC%20Muniz&author=CT%20Mendes-Junior&author=EC%20Castelli&volume=8&publication_year=2013&pages=e71742&pmid=24204558&doi=10.1371/journal.pone.0071742&)

30. Jeong S, Park S, Park BW, Park Y, Kwon OJ, Kim HS. Human leukocyte antigen-G (HLA-G) polymorphism and expression in breast cancer patients. PLoS One 2014;9:e98284 10.1371/journal.pone.0098284  [DOI](https://doi.org/10.1371/journal.pone.0098284) | [PMC free article](/articles/PMC4037222/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24870375/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=Human%20leukocyte%20antigen-G%20(HLA-G)%20polymorphism%20and%20expression%20in%20breast%20cancer%20patients&author=S%20Jeong&author=S%20Park&author=BW%20Park&author=Y%20Park&author=OJ%20Kwon&volume=9&publication_year=2014&pages=e98284&pmid=24870375&doi=10.1371/journal.pone.0098284&)

31. Rizzo R, Audrito V, Vacca P, Rossi D, Brusa D, Stignani M, et al. HLA-G is a component of the chronic lymphocytic leukemia escape repertoire to generate immune suppression: impact of the HLA-G 14 base pair (rs66554220) polymorphism. Haematologica 2014;99:888–96. 10.3324/haematol.2013.095281  [DOI](https://doi.org/10.3324/haematol.2013.095281) | [PMC free article](/articles/PMC4008099/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24362551/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Haematologica&title=HLA-G%20is%20a%20component%20of%20the%20chronic%20lymphocytic%20leukemia%20escape%20repertoire%20to%20generate%20immune%20suppression:%20impact%20of%20the%20HLA-G%2014%20base%20pair%20(rs66554220)%20polymorphism&author=R%20Rizzo&author=V%20Audrito&author=P%20Vacca&author=D%20Rossi&author=D%20Brusa&volume=99&publication_year=2014&pages=888-96&pmid=24362551&doi=10.3324/haematol.2013.095281&)

32. Dahl M, Perin TL, Djurisic S, Rasmussen M, Ohlsson J, Buus S, et al. Soluble human leukocyte antigen-G in seminal plasma is associated with HLA-G genotype: possible implications for fertility success. Am J Reprod Immunol 2014;72:89–105. 10.1111/aji.12251  [DOI](https://doi.org/10.1111/aji.12251) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24735563/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Reprod%20Immunol&title=Soluble%20human%20leukocyte%20antigen-G%20in%20seminal%20plasma%20is%20associated%20with%20HLA-G%20genotype:%20possible%20implications%20for%20fertility%20success&author=M%20Dahl&author=TL%20Perin&author=S%20Djurisic&author=M%20Rasmussen&author=J%20Ohlsson&volume=72&publication_year=2014&pages=89-105&pmid=24735563&doi=10.1111/aji.12251&)

33. Mariaselvam CM, Chaaben AB, Salah S, Charron D, Krishnamoorthy R, Tamouza R, et al. Human leukocyte antigen-G polymorphism influences the age of onset and autoantibody status in rheumatoid arthritis. Tissue Antigens 2015;85:182–9. 10.1111/tan.12521  [DOI](https://doi.org/10.1111/tan.12521) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25656292/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Tissue%20Antigens&title=Human%20leukocyte%20antigen-G%20polymorphism%20influences%20the%20age%20of%20onset%20and%20autoantibody%20status%20in%20rheumatoid%20arthritis&author=CM%20Mariaselvam&author=AB%20Chaaben&author=S%20Salah&author=D%20Charron&author=R%20Krishnamoorthy&volume=85&publication_year=2015&pages=182-9&pmid=25656292&doi=10.1111/tan.12521&)

34. Ferguson R, Ramanakumar AV, Koushik A, Coutlée F, Franco E, Roger M, et al. Human leukocyte antigen G polymorphism is associated with an increased risk of invasive cancer of the uterine cervix. Int J Cancer 2012;131:E312–9. 10.1002/ijc.27356  [DOI](https://doi.org/10.1002/ijc.27356) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22095460/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Cancer&title=Human%20leukocyte%20antigen%20G%20polymorphism%20is%20associated%20with%20an%20increased%20risk%20of%20invasive%20cancer%20of%20the%20uterine%20cervix&author=R%20Ferguson&author=AV%20Ramanakumar&author=A%20Koushik&author=F%20Coutl%C3%A9e&author=E%20Franco&volume=131&publication_year=2012&pages=E312-9&pmid=22095460&doi=10.1002/ijc.27356&)

35. Ye SR, Yang H, Li K, Dong DD, Lin XM, Yie SM. Human leukocyte antigen G expression: as a significant prognostic indicator for patients with colorectal cancer. Mod Pathol 2007;20:375–83.  [DOI](https://doi.org/10.1038/modpathol.3800751) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17277760/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mod%20Pathol&title=Human%20leukocyte%20antigen%20G%20expression:%20as%20a%20significant%20prognostic%20indicator%20for%20patients%20with%20colorectal%20cancer&author=SR%20Ye&author=H%20Yang&author=K%20Li&author=DD%20Dong&author=XM%20Lin&volume=20&publication_year=2007&pages=375-83&pmid=17277760&doi=10.1038/modpathol.3800751&)

36. Hansel DE, Rahman A, Wilentz RE, Shih IeM, McMaster MT, Yeo CJ, et al. HLA-G upregulation in pre-malignant and malignant lesions of the gastrointestinal tract. Int J Gastrointest Cancer 2005;35:15–23.  [DOI](https://doi.org/10.1385/ijgc:35:1:015) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15722570/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Gastrointest%20Cancer&title=HLA-G%20upregulation%20in%20pre-malignant%20and%20malignant%20lesions%20of%20the%20gastrointestinal%20tract&author=DE%20Hansel&author=A%20Rahman&author=RE%20Wilentz&author=IeM%20Shih&author=MT%20McMaster&volume=35&publication_year=2005&pages=15-23&pmid=15722570&doi=10.1385/ijgc:35:1:015&)

37. Fukushima Y, Oshika Y, Nakamura M, Tokunaga T, Hatanaka H, Abe Y, et al. Increased expression of human histocompatibility leukocyte antigen-G in colorectal cancer cells. Int J Mol Med 1998;2:349–51.  [DOI](https://doi.org/10.3892/ijmm.2.3.349) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9855710/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Mol%20Med&title=Increased%20expression%20of%20human%20histocompatibility%20leukocyte%20antigen-G%20in%20colorectal%20cancer%20cells&author=Y%20Fukushima&author=Y%20Oshika&author=M%20Nakamura&author=T%20Tokunaga&author=H%20Hatanaka&volume=2&publication_year=1998&pages=349-51&pmid=9855710&doi=10.3892/ijmm.2.3.349&)

38. Zhu CB, Wang CX, Zhang X, Zhang J, Li W. Serum sHLA-G levels: a useful indicator in distinguishing colorectal cancer from benign colorectal diseases. Int J Cancer 2011;128:617–22. 10.1002/ijc.25372  [DOI](https://doi.org/10.1002/ijc.25372) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20473865/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Cancer&title=Serum%20sHLA-G%20levels:%20a%20useful%20indicator%20in%20distinguishing%20colorectal%20cancer%20from%20benign%20colorectal%20diseases&author=CB%20Zhu&author=CX%20Wang&author=X%20Zhang&author=J%20Zhang&author=W%20Li&volume=128&publication_year=2011&pages=617-22&pmid=20473865&doi=10.1002/ijc.25372&)

39. Cao M, Yie SM, Liu J, Ye SR, Xia D, Gao E. Plasma soluble HLA-G is a potential biomarker for diagnosis of colorectal, gastric, esophageal and lung cancer. Tissue Antigens 2011;78:120–28. 10.1111/j.1399-0039.2011.01716.x  [DOI](https://doi.org/10.1111/j.1399-0039.2011.01716.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21726203/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Tissue%20Antigens&title=Plasma%20soluble%20HLA-G%20is%20a%20potential%20biomarker%20for%20diagnosis%20of%20colorectal,%20gastric,%20esophageal%20and%20lung%20cancer&author=M%20Cao&author=SM%20Yie&author=J%20Liu&author=SR%20Ye&author=D%20Xia&volume=78&publication_year=2011&pages=120-28&pmid=21726203&doi=10.1111/j.1399-0039.2011.01716.x&)

40. Guo ZY, Lv YG, Wang L, Shi SJ, Yang F, Zheng GX, et al. Predictive value of HLA-G and HLA-E in the prognosis of colorectal cancer patients. Cell Immunol 2015;293:10–16. 10.1016/j.cellimm.2014.10.003  [DOI](https://doi.org/10.1016/j.cellimm.2014.10.003) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25461612/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell%20Immunol&title=Predictive%20value%20of%20HLA-G%20and%20HLA-E%20in%20the%20prognosis%20of%20colorectal%20cancer%20patients&author=ZY%20Guo&author=YG%20Lv&author=L%20Wang&author=SJ%20Shi&author=F%20Yang&volume=293&publication_year=2015&pages=10-16&pmid=25461612&doi=10.1016/j.cellimm.2014.10.003&)

41. Zeestraten ECM, Reimers MS, Saadatmand S, Goossens-Beumer IJ, Dekker JW, Liefers GJ, et al. Combined analysis of HLA class I, HLA-E and HLA-G predicts prognosis in colon cancer patients. Br J Cancer 2014;111:459–68. 10.1038/bjc.2013.696  [DOI](https://doi.org/10.1038/bjc.2013.696) | [PMC free article](/articles/PMC3899753/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24196788/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Cancer&title=Combined%20analysis%20of%20HLA%20class%20I,%20HLA-E%20and%20HLA-G%20predicts%20prognosis%20in%20colon%20cancer%20patients&author=ECM%20Zeestraten&author=MS%20Reimers&author=S%20Saadatmand&author=IJ%20Goossens-Beumer&author=JW%20Dekker&volume=111&publication_year=2014&pages=459-68&pmid=24196788&doi=10.1038/bjc.2013.696&)

42. Garziera M, Toffoli G. Inhibition of host immune response in colorectal cancer: human leukocyte antigen-G and beyond. World J Gastroenterol 2014;20:3778–94. 10.3748/wjg.v20.i14.3778  [DOI](https://doi.org/10.3748/wjg.v20.i14.3778) | [PMC free article](/articles/PMC3983436/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24744572/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=World%20J%20Gastroenterol&title=Inhibition%20of%20host%20immune%20response%20in%20colorectal%20cancer:%20human%20leukocyte%20antigen-G%20and%20beyond&author=M%20Garziera&author=G%20Toffoli&volume=20&publication_year=2014&pages=3778-94&pmid=24744572&doi=10.3748/wjg.v20.i14.3778&)

43. Cecchin E, D'Andrea M, Lonardi S, Zanusso C, Pella N, Errante D, et al. A prospective validation pharmacogenomic study in the adjuvant setting of colorectal cancer patients treated with the 5-fluorouracil/leucovorin/oxaliplatin (FOLFOX4) regimen. Pharmacogenomics J 2013;13:403–409 10.1038/tpj.2012.31  [DOI](https://doi.org/10.1038/tpj.2012.31) | [PMC free article](/articles/PMC3859145/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22868256/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=A%20prospective%20validation%20pharmacogenomic%20study%20in%20the%20adjuvant%20setting%20of%20colorectal%20cancer%20patients%20treated%20with%20the%205-fluorouracil/leucovorin/oxaliplatin%20(FOLFOX4)%20regimen&author=E%20Cecchin&author=M%20D'Andrea&author=S%20Lonardi&author=C%20Zanusso&author=N%20Pella&volume=13&publication_year=2013&pages=403-409&pmid=22868256&doi=10.1038/tpj.2012.31&)

44. Cecchin E, Perrone G, Nobili S, Polesel J, De Mattia E, Zanusso C, et al. MTHFR-1298 A>C (rs1801131) is a predictor of survival in two cohorts of stage II/III colorectal cancer patients treated with adjuvant fluoropyrimidine chemotherapy with or without oxaliplatin. Pharmacogenomics J 2015;15: 219–25. 10.1038/tpj.2014.64  [DOI](https://doi.org/10.1038/tpj.2014.64) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25331073/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=MTHFR-1298%20A>C%20(rs1801131)%20is%20a%20predictor%20of%20survival%20in%20two%20cohorts%20of%20stage%20II/III%20colorectal%20cancer%20patients%20treated%20with%20adjuvant%20fluoropyrimidine%20chemotherapy%20with%20or%20without%20oxaliplatin&author=E%20Cecchin&author=G%20Perrone&author=S%20Nobili&author=J%20Polesel&author=E%20De%20Mattia&volume=15&publication_year=2015&pages=219-25&pmid=25331073&doi=10.1038/tpj.2014.64&)

45. Stephens M, Smith NJ, Donnelly P. A new statistical method for haplotype reconstruction from population data. Am J Hum Genet 2001;68:978–989.  [DOI](https://doi.org/10.1086/319501) | [PMC free article](/articles/PMC1275651/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11254454/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=A%20new%20statistical%20method%20for%20haplotype%20reconstruction%20from%20population%20data&author=M%20Stephens&author=NJ%20Smith&author=P%20Donnelly&volume=68&publication_year=2001&pages=978-989&pmid=11254454&doi=10.1086/319501&)

46. Sabbagh A, Luisi P, Castelli EC, Gineau L, Courtin D, Milet J, et al. Worldwide genetic variation at the 3' untranslated region of the HLA-G gene: balancing selection influencing genetic diversity. Genes Immun 2014;15:95–106. 10.1038/gene.2013.67  [DOI](https://doi.org/10.1038/gene.2013.67) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24352166/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genes%20Immun&title=Worldwide%20genetic%20variation%20at%20the%203'%20untranslated%20region%20of%20the%20HLA-G%20gene:%20balancing%20selection%20influencing%20genetic%20diversity&author=A%20Sabbagh&author=P%20Luisi&author=EC%20Castelli&author=L%20Gineau&author=D%20Courtin&volume=15&publication_year=2014&pages=95-106&pmid=24352166&doi=10.1038/gene.2013.67&)

47. Stremitzer S, Sunakawa Y, Zhang W, Yang D, Ning Y, Stintzing S, et al. Variations in genes involved in immune response checkpoints and association with outcomes in patients with resected colorectal liver metastases. Pharmacogenomics J. 2015. March 10 10.1038/tpj.2015.14  [DOI](https://doi.org/10.1038/tpj.2015.14) | [PMC free article](/articles/PMC7505123/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25752522/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=Variations%20in%20genes%20involved%20in%20immune%20response%20checkpoints%20and%20association%20with%20outcomes%20in%20patients%20with%20resected%20colorectal%20liver%20metastases&author=S%20Stremitzer&author=Y%20Sunakawa&author=W%20Zhang&author=D%20Yang&author=Y%20Ning&publication_year=2015&pmid=25752522&doi=10.1038/tpj.2015.14&)

48. Zitvogel L, Kroemer G. Zitvogel L, Kroemer G. Cancer: Antibodies regulate antitumour immunity. Nature. 2015;521:35–7. 10.1038/nature14388  [DOI](https://doi.org/10.1038/nature14388) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25924066/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Cancer:%20Antibodies%20regulate%20antitumour%20immunity&author=L%20Zitvogel&author=G%20Kroemer&author=L%20Zitvogel&author=G%20Kroemer&volume=521&publication_year=2015&pages=35-7&pmid=25924066&doi=10.1038/nature14388&)

49. Larsen MH, Zinyama R, Kallestrup P, Gerstoft J, Gomo E, Thørner LW, et al. HLA-G 3' untranslated region 14-base pair deletion: association with poor survival in an HIV-1-infected Zimbabwean population. J Infect Dis 2013;207:903–6. 10.1093/infdis/jis924  [DOI](https://doi.org/10.1093/infdis/jis924) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23264673/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Infect%20Dis&title=HLA-G%203'%20untranslated%20region%2014-base%20pair%20deletion:%20association%20with%20poor%20survival%20in%20an%20HIV-1-infected%20Zimbabwean%20population&author=MH%20Larsen&author=R%20Zinyama&author=P%20Kallestrup&author=J%20Gerstoft&author=E%20Gomo&volume=207&publication_year=2013&pages=903-6&pmid=23264673&doi=10.1093/infdis/jis924&)

50. Yie SM, Li LH, Xiao R, Librach CL. A single base-pair mutation in the 3'-untranslated region of HLA-G mRNA is associated with pre-eclampsia. Mol Hum Reprod 2008;14:649–53. 10.1093/molehr/gan059  [DOI](https://doi.org/10.1093/molehr/gan059) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18952696/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Hum%20Reprod&title=A%20single%20base-pair%20mutation%20in%20the%203'-untranslated%20region%20of%20HLA-G%20mRNA%20is%20associated%20with%20pre-eclampsia&author=SM%20Yie&author=LH%20Li&author=R%20Xiao&author=CL%20Librach&volume=14&publication_year=2008&pages=649-53&pmid=18952696&doi=10.1093/molehr/gan059&)

51. Carlini F, Traore K, Cherouat N, Roubertoux P, Buhler S, Cortey M, et al. HLA-G UTR haplotype conservation in the Malian population: association with soluble HLA-G. PLoS One 2013;8:e82517 10.1371/journal.pone.0082517  [DOI](https://doi.org/10.1371/journal.pone.0082517) | [PMC free article](/articles/PMC3871591/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24376542/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=HLA-G%20UTR%20haplotype%20conservation%20in%20the%20Malian%20population:%20association%20with%20soluble%20HLA-G&author=F%20Carlini&author=K%20Traore&author=N%20Cherouat&author=P%20Roubertoux&author=S%20Buhler&volume=8&publication_year=2013&pages=e82517&pmid=24376542&doi=10.1371/journal.pone.0082517&)

52. Toffoli G, Giodini L, Buonadonna A, Berretta M, De Paoli A, Scalone S, et al. Clinical validity of a DPYD-based pharmacogenetic test to predict severe toxicity to fluoropyrimidines. Int J Cancer 2015;June 23 10.1002/ijc.29654  [DOI](https://doi.org/10.1002/ijc.29654) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26099996/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Cancer&title=Clinical%20validity%20of%20a%20DPYD-based%20pharmacogenetic%20test%20to%20predict%20severe%20toxicity%20to%20fluoropyrimidines&author=G%20Toffoli&author=L%20Giodini&author=A%20Buonadonna&author=M%20Berretta&author=A%20De%20Paoli&publication_year=2015&pmid=26099996&doi=10.1002/ijc.29654&)
