# Polymorphism at 19q13.41 predicts breast cancer survival specifically after endocrine therapy

## Metadata
**Authors:** Sofia Khan, Rainer Fagerholm, Sajjad Rafiq, William Tapper, Kristiina Aittomäki, Jianjun Liu, Carl Blomqvist, Diana Eccles, Heli Nevanlinna
**Journal:** Clinical cancer research : an official journal of the American Association for Cancer Research
**Date:** 2015 May 11
**DOI:** [10.1158/1078-0432.CCR-15-0296](https://doi.org/10.1158/1078-0432.CCR-15-0296)
**PMID:** 25964295
**PMCID:** PMC4574404
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4574404/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC4574404/pdf/emss-63442.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC4574404/pdf/emss-63442.pdf)

## Abstract

**Purpose:** 
Although most estrogen receptor (ER)-positive breast cancer patients benefit from endocrine therapies, a significant proportion do not. Our aim was to identify inherited genetic variations that might predict survival among patients receiving adjuvant endocrine therapies.

**Experimental Design:** 
We performed a meta-analysis of two genome-wide studies; Helsinki Breast Cancer Study, 805 patients, with 240 receiving endocrine therapy and Prospective study of Outcomes in Sporadic versus Hereditary breast cancer, 536 patients, with 155 endocrine therapy-patients, evaluating 486,478 single nucleotide polymorphisms (SNPs). The top four associations from the endocrine treatment subgroup were further investigated in two independent datasets totalling 5011 patients, with 3485 receiving endocrine therapy.

**Results:** 
A meta-analysis identified a common SNP rs8113308, mapped to 19q13.41, associating with reduced survival among endocrine treated patients (hazard ratio (HR) 1.69, 95% confidence interval (CI) 1.37-2.07, P = 6.34 ×10−7) and improved survival among ER-negative patients, with a similar trend in ER-positive cases not receiving endocrine therapy. In a multivariate analysis adjusted for conventional prognostic factors, we found a significant interaction between the rs8113308 and endocrine treatment indicating a predictive, treatment-specific effect of the SNP rs8113308 on breast cancer survival, with the per-allele HR for interaction 2.16 (95% CI 1.30 – 3.60, Pinteraction = 0.003) and HR=7.77 (95% CI 0.93 – 64.71) for the homozygous genotype carriers. A biological rationale is suggested by in silico functional analyses.

**Conclusions:** 
Our findings suggest carrying the rs8113308 rare allele may identify patients who will not benefit from adjuvant endocrine treatment.

Keywords: breast cancer, survival, endocrine therapy, GWAS, genotype-phenotype correlations

### Purpose

Although most estrogen receptor (ER)-positive breast cancer patients benefit from endocrine therapies, a significant proportion do not. Our aim was to identify inherited genetic variations that might predict survival among patients receiving adjuvant endocrine therapies.

### Experimental Design

We performed a meta-analysis of two genome-wide studies; Helsinki Breast Cancer Study, 805 patients, with 240 receiving endocrine therapy and Prospective study of Outcomes in Sporadic versus Hereditary breast cancer, 536 patients, with 155 endocrine therapy-patients, evaluating 486,478 single nucleotide polymorphisms (SNPs). The top four associations from the endocrine treatment subgroup were further investigated in two independent datasets totalling 5011 patients, with 3485 receiving endocrine therapy.

### Results

A meta-analysis identified a common SNP rs8113308, mapped to 19q13.41, associating with reduced survival among endocrine treated patients (hazard ratio (HR) 1.69, 95% confidence interval (CI) 1.37-2.07, P = 6.34 ×10^−7^) and improved survival among ER-negative patients, with a similar trend in ER-positive cases not receiving endocrine therapy. In a multivariate analysis adjusted for conventional prognostic factors, we found a significant interaction between the rs8113308 and endocrine treatment indicating a predictive, treatment-specific effect of the SNP rs8113308 on breast cancer survival, with the per-allele HR for interaction 2.16 (95% CI 1.30 – 3.60, *P*_interaction_ = 0.003) and HR=7.77 (95% CI 0.93 – 64.71) for the homozygous genotype carriers. A biological rationale is suggested by *in silico* functional analyses.

### Conclusions

Our findings suggest carrying the rs8113308 rare allele may identify patients who will not benefit from adjuvant endocrine treatment.

## Introduction

Breast cancer is the most common cancer among women worldwide and a leading cause of cancer-related deaths([1](#R1)). Breast cancer can be divided into two major types by the estrogen receptor alpha (ER) status; ER-positive breast cancer is driven by the female hormone estrogen whereas ER-negative breast cancer does not depend on estrogen. Endocrine therapies target the ER-positive type, which accounts for about 70% of all breast cancer ([2](#R2)). Currently available endocrine therapies aim to either selectively block the estrogen receptor by binding ER (tamoxifen), decrease ovarian estrogen production (ovarian ablation, luteinising hormone releasing hormone (LHRH) agonists) or, in post-menopausal women, blocking the conversion of androgen to estrogen in peripheral fat (aromatase inhibitors) or selectively down-regulating ER (e.g. fulvestrant) ([3](#R3)). Randomized controlled trials have demonstrated that breast cancer recurrence and death may be reduced by approximately one third by endocrine adjuvant treatments in patients with ER-positive breast cancer ([4](#R4), [5](#R5)). However, approximately 30% of ERα-positive breast cancers do not respond to endocrine therapies (*de novo* resistance) ([6](#R6)) and in addition, the majority of tumors that initially respond to treatment develop resistance over time (acquired resistance) ([7](#R7)).

There are several potential mechanisms of resistance to endocrine therapy (reviewed in ([8](#R8))) including e.g., as the most important mechanism, loss of expression of ER alpha (due to an emerging subclone of ER-negative cancer). Beside mechanisms related to ER, resistance to endocrine therapy may also occur due to increased growth factor signalling and dysfunctional metabolism of hormonal agents. As an example, patients carrying inactive alleles of *CYP2D6* (approximately 7-10% of Caucasian women) fail to convert tamoxifen to its primary active metabolite, endoxifen, and may consequently be less responsive to tamoxifen ([9](#R9)-[12](#R12)). However, the association with endocrine treatment outcome remains currently controversial. Presently, aside from ER status, no unequivocal biomarkers have been identified to determine whether a patient will benefit from adjuvant endocrine therapy.

Germline genetic variations, such as single nucleotide polymorphisms (SNPs), have been assessed as potential predictors of survival for breast cancer patients in general ([13](#R13)-[15](#R15)) and in different subgroups including those defined by endocrine therapies ([16](#R16), [17](#R17)). A majority of these studies have applied a candidate gene approach focusing on SNPs within pre-specified genes of interest. These studies have provided some indicative results, but further studies will be needed to validate the findings. Recently, genome-wide association studies (GWASs) have been performed with the aim of identifying genetic variants influencing the outcome of breast cancer ([18](#R18)-[20](#R20)), including our previous studies that identified ARRDC3 locus influencing prognosis in especially early-onset breast cancer ([21](#R21), [22](#R22)). Further, a GWAS conducted in subgroup of patients receiving adjuvant tamoxifen therapy in a Japanese population detected significant associations with recurrence-free survival at 10q22 ([23](#R23)).

The availability of GWA data from the patient population in our Helsinki Breast Cancer Study (HEBCS) study together with GWA data from Prospective study of Outcomes in Sporadic versus Hereditary breast cancer (POSH) study enables an agnostic genome-wide approach to identify common genetic variants associated with survival for breast cancer. In the present study, we focus on ER-positive breast cancer treated with adjuvant endocrine therapy in order to identify putative genetic markers for endocrine treatment outcome. We implemented a two-stage study design with 1341 breast cancer patients from the two above mentioned GWAS in stage I and then a further 5011 patients from the two in stage II validation datasets.

## Materials and Methods

### Study population

#### stage I discovery datasets

Samples included in stage I discovery dataset came from participants of the HEBCS and POSH studies. For HEBCS GWS, 805 cases were included. Of these, 563 cases originated from a prospective patient series of unselected, incident breast cancer patients while 242 cases were obtained from additional familial patient series ([24](#R24)-[26](#R26)). All cases were ascertained at the Helsinki University Hospital; see [Supplementary Methods](#SD1) for the details of the collection. Of the 805 samples, 240 samples were recorded to have received endocrine therapy ([Table 1](#T1)).

| Characteristics | HEBCS GWS | POSH GWS | POSH Validation | SUCCESS-A |
| --- | --- | --- | --- | --- |
| No. of cases | 805 | 536 | 1415 | 3596 |
| Vital status |   |   |   |   |
| Alive | 466 (58%) | 300 (56%) | 1194 (84%) | 3389 (94%) |
| Deceased: all-cause | 339 (42%) | 236 (44%) | 221 (16%) | 207 (6%) |
| Deceased: BC-specific | 312 (39%) | 235 (44%) | 208 (15%) | NA |
| Follow-up mean ±SD | 10.6 ± 6.6 | 4.1± 2.0 | 5.0± 1.9 | 3.9± 1.7 |
| Age, mean [range],y | 54.1 [22 - 87] | 35.8 [18 - 41] | 35.8 [18 - 40] | 53.6 [19 - 85] |
| ER |   |   |   |   |
| Negative | 230 (29%) | 370 (69%) | 318 (22%) | 1106 (31%) |
| Positive | 513 (64%) | 165 (31%) | 1089 (77%) | 2458 (68%) |
| Missing, No. | 62 (8%) | 1 (0.2%) | 8 (1%) | 32 (1%) |
| Grade |   |   |   |   |
| 1 | 144 (18%) | 13 (2%) | 106 (7%) | 165 (5%) |
| 2 | 312 (39%) | 84 (16%) | 549 (39%) | 1698 (47%) |
| 3 | 280 (35%) | 422 (79%) | 726 (51%) | 1698 (47%) |
| Missing, No. | 69 (9%) | 17 (3%) | 34 (2%) | 35 (1%) |
| T |   |   |   |   |
| 1 | 390 (48%) | 232 (43%) | 692 (49%) | 1464 (41%) |
| 2 | 304 (38%) | 236 (44%) | 493 (35%) | 1856 (52%) |
| 3 | 50 (6%) | 49 (9%) | 49 (3%) | 192 (5%) |
| 4 | 47 (6%) | 12 (2%) | 34 (2%) | 50 (1%) |
| Missing, No. | 14 (2%) | 7 (1%) | 147 (10%) | 34* (1%) |
| N |   |   |   |   |
| Negative | 338 (42%) | 248 (46%) | 654 (46%) | 1248 (35%) |
| Positive | 446 (55%) | 262 (49%) | 742 (52%) | 2311 (64%) |
| Missing, No. | 21 (3%) | 26 (5%) | 19 (1%) | 37 (1%) |
| M |   |   |   |   |
| Negative | 740 (92%) | 481 (90%) | 1398 (99%) | 3487 (97%) |
| Positive | 57 (7%) | 50 (9%) | 10 (0.7%) | 4 (0.1%) |
| Missing, No. | 8 (1%) | 5 (1%) | 7 (0.3%) | 105 (2.9%) |
| Adjuvant chemotherapy treatmenta | 364 (45%) | 518 (96.6%) | 1018 (72%) | 3596 (100%) |
| A&T | 14 (2%) | 129 (24%) | 187 (18%) | - |
| Antracyclines | 191 (24%) | 376 (70%) | 817 (80%) | 3596 (100%) |
| Taxanes | 2 (0.2%) | 8 (1.5%) | 5 (0.5%) | - |
| CMF | 153 (19%) | 4 (1%) | 9 (1%) | - |
| Adjuvant Endocrine treatmentab | 240 (29.8%) | 155 (29%) | 1027 (72.6%) | 2458 (68%) |
| Anti-estrogen (Tamoxifen) | 234 (29%) | 145 (27%) | 966 (68%) | 2458 (68%) |
| Aromatase inhibitor | 6 (20.7%) | 9 (1.7%) | 33 (2%) | 223 (6%) |
| LHRH agonist | 0 | 49 (9%) | 250 (17.7%) | 29 (1%) |
| No endocrine treatment (tamoxifen/AI/LHRH agonist) | 272 (34%) | 10 (1.8%) | 57 (4%) | 0 |

Table Caption: Abbreviations: NA, not available; T, tumor size according to TNM classification; N, metastasis to lymph node; M, distant metastasis aThe total numbers may not add up, since a patient may have received several types of adjuvant chemotherapy/endocrine treatment. bAmong ER-positive patients.

The POSH GWS consisted of 574 participants from the POSH study ([27](#R27)). Prospective early onset breast cancer cases were included in the POSH study, with participants diagnosed with invasive breast cancer aged 40 years or younger. Details of the patient selection are provided in the [Supplementary Methods](#SD1). POSH GWS included 155 patients that had received endocrine therapy ([Table 1](#T1)). All participants of both studies provided written informed consent before participating in the study.

#### stage II validation datasets

A further 1415 breast cancer patients from the POSH study ([27](#R27)) unselected for any differential survival were included in the stage II validation dataset. POSH validation included 1027 patients that had received endocrine therapy ([Table 1](#T1)).

As an additional independent validation dataset in stage II we used a series of 3596 patients from the prospectively randomized SUCCESS-A trial. Details of the collection are provided in the [Supplementary Methods](#SD1). A total of 2458 cases of the 3596 cases had received endocrine treatment.

The age and tumor characteristics of study participants from HEBCS GWS, POSH GWS, POSH validation and SUCCESS-A are presented in [Table 1](#T1). The flow of samples through the various stages of the study has been summarized in [Supplementary Fig. S1](#SD2).

### Genome wide genotyping and harmonized quality control of HEBCS and POSH GWS

Genotyping of the Helsinki samples was conducted using the Illumina 550 platform and POSH GWS using the Illumina 660-Quad SNP array as previously described([21](#R21), [28](#R28)). To ensure the harmonisation of genotype calling between HEBCS and POSH GWS, the HEBCS GWS intensity files were processed with Illumina’s Genome Studio software to call genotypes consistently with the POSH genotypes using a GenCall threshold of 0.15. Rare SNPs were excluded from analysis based on a MAF cut-off of 0.01, a genotyping call rate <95% and Hardy-Weinberg equilibrium P value <0.0001. The detailed description of harmonized quality control is in ([21](#R21)).

### Replication genotyping

For replication genotyping we selected the top four associations from the genome wide meta-analysis of HEBCS and POSH GWS in the ER-positive endocrine treatment subgroup that fulfilled the following criteria: most significant independent associations with meta-*P* < 1.0 × 10^−4^ within the ER-positive endocrine treatment subgroup, showing significant (*P*_heterogeneity_ < 0.01) heterogeneity by endocrine treatment evaluating ER-positive endocrine treated subgroup and ER-positive subgroup not treated with endocrine therapy, and having significant interaction with endocrine treatment (likelihood ratio test *P*-value per allele < 1.0 × 10^−3^) in a pooled dataset of HEBCS and POSH GWS ER-positive cases. These four SNPs were genotyped in the 1415 additional young onset cases from the POSH stage II validation study. SNPs were genotyped by KBiosciences using the KASPar chemistry, which is a competitive allele-specific PCR SNP genotyping system. The SUCCESS-A GWAS was genotyped on the Illumina HumanOmniExpress-12v1 G FFPE array.

### Imputation of HEBCS and POSH GWS

The imputation of genome wide SNP information in HEBCS and POSH GWS was performed based on 1000 Genomes Project phase 1 and release version 3 European reference haplotypes. Quality control measures applied to imputed data included excluding SNPs with HWE *P* value < 1 × 10^−6^, MAF < 5%, and imputed genotype call rate ≤90% and individuals call rate ≤90%. The detailed description of imputation is in ([22](#R22)).

### Statistical Analysis

See [Supplementary Methods](#SD1) for a detailed description of the statistical analyses. In stage I, Cox’s proportional hazards models were used to derive hazard ratios (HR) for breast cancer specific mortality in association with each SNP. Follow-up time was calculated from the date of diagnosis to the date of last follow-up or breast cancer related death and right-censored at 10 years.

In stage II, the follow-up time was calculated from the date of diagnosis to the date of last follow-up or breast cancer related death for POSH validation dataset. For SUCCESS-A the follow-up time was calculated from the date of diagnosis to the date of last follow-up or death from any cause, due to lack of cause-of-death information. The meta-analysis in stage I as well as the meta-analysis of stage I and stage II was performed with R package MetABEL ([29](#R29)). The Cox’s proportional hazard models were performed with R package GenABEL ([29](#R29)).

For HEBCS GWS, POSH GWS and POSH validation we had cause-of-death information that enabled us to evaluate the breast cancer specific survival. For SUCCESS-A the only outcome information was overall survival (endpoint: all-cause mortality) and progression free survival (endpoints: local or metastatic recurrence or death). In order to assess differences in survival when using different endpoints, we further conducted a sensitivity analysis. In the sensitivity analysis we analysed the survival across all the four studies using a common endpoint; either 10 year overall survival or 5 year progression free survival.

In order to test for interaction between endocrine treatment and a given SNP of interest, SNP genotype data was fitted into two multivariate Cox’s proportional hazards models including also clinically relevant covariates: one with both endocrine treatment and the SNP represented as individual covariates, and one that included an interaction term between the two. A likelihood ratio test between models was then conducted to examine whether the interaction model is a better fit for the prognostic data. The interaction tests, specifying breast cancer related death as the endpoint, were conducted in a pooled dataset of ER-positive cases only and were stratified by study.

### Expression quantitative trait loci (eQTL) analysis

In order to analyse the correlation between the loci of interest and gene expression we utilized the breast cancer sample data generated by the METABRIC project ([30](#R30), [31](#R31)). The expression data was obtained from the European Genome-Phenome Archive, which is hosted by the European Bioinformatics Institute, under accession number EGAS00000000083. See [Supplementary Methods](#SD1) for the details of the data preparation for eQTL analysis. The analysis was conducted with R-package Matrix eQTL ([32](#R32)) using linear regression and ANOVA models. In addition we utilized online results of the peripheral blood eQTL meta-analysis ([33](#R33)) and lymphoblastoid exon expression QTL in Geuvadis project ([34](#R34)).

### In silico tools

In order to investigate whether the loci of interest harbour known or predicted regulatory elements, we explored the ENCODE data using HaploReg2 ([35](#R35)) and RegulomeDB ([36](#R36)). To assess gene expression-based survival we utilized BreastMark that integrates gene expression and survival data from 26 datasets on 12 different microarray platforms corresponding to ~17,000 genes in up to 4,738 samples ([37](#R37)). The genes that were identified by eQTL analysis were analysed at the protein level by exploring the protein-protein interaction network with STRING program ([38](#R38)).

## Results

### stage I: HEBCS and POSH GWS meta-analysis

We performed a fixed-effects meta-analysis to combine HR estimates from HEBCS and POSH stage I GWS studies including 805 and 536 study subjects. In the two datasets altogether 486,478 SNPs were common and passed the QC process. The meta-analysis was performed including all cases and in the subgroup of endocrine treated patients; combining anti-estrogen, aromatase inhibitor and LHRH agonist treatments totalling 240 endocrine treated patients in HEBCS GWS and 155 in POSH GWS. After LD-pruning, the top four associations (rs8113308, rs4082843, rs4767413, and rs11085098 in chromosomes 19, 4, 12, and 19, respectively; [Supplementary Table S1](#SD1)) were selected from the genome-wide meta-analysis of HEBCS and POSH GWS in the ER-positive endocrine treatment subgroup for genotyping in the stage II POSH validation samples. The selected SNPs fulfilled the following criteria: the most significant independent associations with meta-P < 1.0 × 10^−4^ within the ER-positive endocrine treatment subgroup, showing significant (P_heterogeneity_ < 0.01) survival heterogeneity by endocrine treatment and having significant interaction with endocrine treatment (likelihood ratio test P-value per allele < 1.0 × 10^−3^) in a pooled dataset of HEBCS and POSH GWS ER-positive cases. All the SNPs were identified under an additive inheritance model.

### stage II: POSH validation and SUCCESS-A

Of the four SNPs which were formally tested for replication, all were successfully genotyped and two SNPs demonstrated nominal replication signals in the same direction as in the stage I (HEBCS and POSH GWS) patients. As an additional independent validation dataset in stage II we used SUCCESS-A. Since SUCCESS-A was genotyped in a different version of Illumina genotyping chip, there were no exact SNP matches for two of the SNPs. For SNP rs8113308 we utilized the genotype information of a tag SNP rs8108525 (r^2^ = 0.81). For SNP rs4082843 no tag SNP could be found with r^2^ > 0.80. For remaining SNPs an exact SNP match was present in SUCCESS-A genotyping data.

### stage I and stage II meta-analysis

In the meta-analysis of stage I and stage II, the strongest replication and meta-analysis signal was observed at rs8113308 under the additive inheritance model. The minor allele was found to consistently associate with poor survival specifically after adjuvant endocrine therapy among ER-positive patients (HR = 1.69; 95% confidence interval (CI), 1.37-2.07, P = 6.34 × 10^−7^). The most common endocrine treatment regimen in all the four datasets was tamoxifen ([Table 1](#T1)), and a similar effect was found within the tamoxifen-treated subgroup (HR = 1.65; 95% CI, 1.35 – 2.03; P = 1.44 × 10^−6^). However, the minor allele associated with improved breast cancer outcome in ER-negative patients (HR = 0.71; 95% CI, 0.56 – 0.91; P= 6 × 10^−3^), with a similar trend in ER-positive patients not receiving endocrine therapy (HR = 0.66; 95% CI, 0.40 – 1.07; P= 9.32 × 10^−2^), ([Table 2](#T2)), suggesting a treatment specific effect. The Kaplan-Meier plots of cumulative 10-year survival of rs8113308 genotypes among ER-positive endocrine treated patients in pooled stage I (HEBCS and POSH GWS), ER-positive non-treated (available only for HEBCS) and ER-negative patients in pooled stage I (HEBCS and POSH GWS) are presented in [Fig. 1](#F1). The Kaplan-Meier plots separately for all the four studies are presented in [Supplementary Fig. S2](#SD3) and [S3](#SD4).

### Table 2. stage I and stage II meta-analysis of univariate Cox’s regression analysis results for the four associations in stageI and stage II. The table presents per study as well as the meta-analysis results in ER-positive patients receiving endocrine treatment, ER-positive patients not receiving endocrine treatment and ER-negative patients. The per study results in ER-positive patients not receiving endocrine treatment are only presented for HEBCS, because very few ER-positive patients did not receive endocrine treatment in POSH GWS, POSH validation and SUCCESS-A. For SNPs rs4082843, there was no exact SNP match or a tag SNP with r2>0.8 available in SUCCESS-A data.

| ER-positive patients receiving endocrine treatment |  |  |  |  |  |  |  |  |  |  |  |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| SNP | Chr:positiona | MAFb | HEBCS GWS HR (95% CI) | HEBCS GWS P | POSH GWS HR (95% CI) | POSH GWS P | POSH val. HR (95% CI) | POSH val. P | Success-A HR (95% CI) | Success-A P | Meta analysis HR (95% CI) | Meta-analysis P | Location |
| rs8113308 | 19:52445386 | 0.152 | 1.72 (1.08-2.72) | 0.022 | 2.17 (1.37-3.45) | 9.81 × 10−4 | 1.45 (1.04-2.02) | 0.030 | 1.72 (1.11-2.68) | 0.015 | 1.69 (1.37-2.07) | 6.34 × 10−7 | ZNF613 |
| rs4082843 | 4:7109083 | 0.164 | 0,40 (0.23-0.67) | 6.50 × 10−4 | 0,36 (0.15-0.82) | 0.015 | 1,07 (0.75-1.51) | 0,484 | - | - | 0.72 (0.55-0.95) | 2.18 × 10−2 | GRPEL1 \| SORCS2 |
| rs4767413 | 12:116951069 | 0.178 | 2,06 (1.41-3.01) | 1.90 × 10−4 | 1,67 (1.08-2.57) | 0.021 | 1,13 (0.82-1.56) | 0,328 | 1.07 (0.73-1.58) | 0.721 | 1.39 (1.15-1.67) | 5.86 × 10−4 | MED13L \| LINC00173 |
| rs11085098 | 19:4784553 | 0.307 | 1,76 (1.25-2.47) | 0.001 | 1,61 (1.11-2.33) | 0.012 | 0,92 (0.69-1.22) | 0,242 | 0.83 (0.59-1.17) | 0.277 | 1.16 (0.99-1.37) | 7.02 × 10−2 | MIR7-3HG \| FEM1A |

Table 2 Caption: aAccording to the human genome build 36. bMinor allele frequency in Caucasian of European descent.

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a877/4574404/c3f650855dd1/emss-63442-f0001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4574404_emss-63442-f0001.jpg)

Kaplan-Meier plots of cumulative breast cancer specific 10-year survival of rs8113308 genotypes A) in a pooled stage I (HEBCS + POSH GWS) data among ER-positive patients receiving endocrine therapy, B) in HEBCS GWS data among ER-positive patients not receiving endocrine therapy, C) in a pooled stage I (HEBCS + POSH GWS) data among ER-negative patients. Number of patients at risk is presented under each Kaplan-Meier –plot.

We further investigated the survival association of rs8113308 in all patients and in phenotype- and treatment-based subgroups separately in each of the four studies ([Fig. 2](#F2)). The association of the SNP in ER-positive patients receiving endocrine therapy was found consistent throughout the four studies ([Fig. 2](#F2) and [Table 2](#T2))

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a877/4574404/37887a22b301/emss-63442-f0002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4574404_emss-63442-f0002.jpg)

Forest plots of HRs and their CIs for the SNP rs8113308 in the entire sample set and within phenotype- and treatment-based subgroups separately in each of the four studies. Cox’s proportional hazards models were used to derive hazard ratios (HR) for breast cancer specific mortality in HEBCS GWS, POSH GWS and POSH validation and for all-cause mortality for SUCCESS-A.

Based on the sensitivity analysis where we analysed the survival across all the four studies using also 10 year overall survival or 5 year progression free survival, very similar association was seen as in the main meta-analysis, regardless of the used endpoint ([Supplementary Fig. S4](#SD5)).

In addition to the SNP rs8113308, one further SNP, rs4767413, showed a consistent association across all four studies among ER-positive endocrine treated patients, however, the result was not significant in stage II studies (POSH validation and SUCCESS-A). SNP rs4767413, located in an intergenic region in chromosome 12, was found to associate with poor survival among ER-positive patients receiving endocrine treatment with HR = 1.39; 95% CI, 1.15 – 1.67; *P* = 5.86 × 10^−4^, but not among ER-positive patients not treated with endocrine therapy (HR = 0.91; 95% CI, 0.56 – 1.48). No consistent association was seen among the four studies in ER-negative subgroup. The remaining two SNPs from stage I included in the meta-analysis of stage I and stage II did not show concordant association in stage II ([Table 2](#T2)).

### SNP rs8113308 interaction with endocrine therapy

Given that endocrine treatment is predominantly administered to ER-positive cases, it is possible that an apparent interaction between rs8113308 and endocrine treatment actually indicates an interaction between the SNP and ER status instead of a predictive, treatment-specific effect. In order to address this possibility, the interaction test, specifying breast cancer related death as the endpoint, was conducted in a pooled dataset of ER-positive cases only, including HEBCS GWS, POSH GWS and POSH validation datasets – due to lack of cause-of-death information, SUCCESS-A study was not included. In the multivariate Cox’s proportional hazards model including the SNP rs8113308 and endocrine treatment separately along with the following covariates; progesterone receptor status, tumor size, lymph node metastasis, distant metastasis at diagnosis, age at diagnosis, and tumor histologic grade, we found both the SNP and the endocrine treatment to be independently prognostic; per allele HR = 1.27, 95% CI 1.02 – 1.59; *P* = 0.036 and endocrine treatment HR = 0.59; 95% CI, 0.44 – 0.80; *P* = 6.76 × 10^−4^. In contrast, when an interaction term (SNP * endocrine treatment) was added to the model, the SNP lost its independent prognostic value and the interaction between SNP rs8113308 and endocrine treatment associated significantly with poor breast cancer survival, HR for per-allele rs8113308:endocrine treatment = 2.16; 95% CI, 1.30 – 3.60; *P* = 3.13 × 10^−3^ ([Table 3](#T3)). The interaction model is a better fit for the prognostic data than the model without an interaction term (likelihood ratio test *P* value = 0.0021). When using a co-dominant model, the interaction remained statistically significant despite the loss of power introduced by the additional genotype covariate (likelihood ratio test *P* value = 0.0078) ([Table 3](#T3)), while the effect size depended on allele dose: the hazard ratio for the interaction between endocrine treatment and the heterozygous AG genotype is HR = 1.95; 95% CI, 1.08 – 3.49, and HR = 7.77; 95% CI, 0.93 – 64.71 for the interaction between endocrine treatment and rare homozygous GG genotype.

### Table 3.

| Per allele model assuming no interaction |  |  |
| --- | --- | --- |
| Covariate | HR (95% CI) | p-value |
| per-allele rs8113308 | 1.27 (1.02-1.59) | 3.61 × 10−2 |
| endocrine | 0.59 (0.44-0.80) | 6.76 × 10−4 |
| PR | 0.60 (0.46-0.78) | 1.68 × 10−4 |
| T | 1.45 (1.27-1.67) | 8.89 × 10−8 |
| N | 2.14 (1.61-2.85) | 1.92 × 10−7 |
| M | 1.60 (1.02-2.51) | 3.91 × 10−2 |
| Grade | 1.52 (1.25-1.84) | 2.32 × 10−5 |

Table 3 Caption: Cox’s proportional hazards models to test for interaction between endocrine treatment and rs8113308 in the pooled dataset of HEBCS and POSH GWS and POSH validation. The model was stratified by study and adjusted by age and used 10y breast cancer specific survival and included ER-positive cases only; per allele model assuming no interaction and per allele model including per allele SNP:endocrine interaction term. Likelihood ratio test P= 0.0021. Codominant model assuming no interaction and codominant model including per genotype SNP:endocrine interaction term. Likelihood ratio test P = 0.0078.

### Association with clinical predictors

We assessed the association of SNP rs8113308 and rs4767413 with clinical predictors of breast cancer prognosis in a pooled set of HEBCS and POSH GWS and POSH validation ([Supplementary Tables S2 and S3](#SD1)). There were no significant associations between SNP rs8113308 or rs4767413 and clinical features.

### Investigation of imputed SNPs

We next examined the rs8113308 LD region (r^2^≥0.2) for stronger associations conducting a meta-analysis of HEBCS and POSH imputed data of 869 SNPs. We identified an association with HR= 3.40; 95% CI, 2.04 – 5.66 (*P* = 2.64 × 10^−6^) for one imputed SNP, rs10410393 (r^2^ = 0.2). However, it did not show concordant direction of association in the SUCCESS-A data. Additionally, the minor allele frequency for this SNP in European population is 0.036, being very rare. Subsequently, the rs8113308 remained the strongest associated variant in the region ([Supplementary Fig. S5](#SD6)). Similarly, we investigated the LD region (r^2^≥0.2) of SNP rs4767413 and found stronger associations for four imputed SNPs in the HEBCS and POSH data sets, with rs11611797 (r^2^ = 0.87) being strongest (HR= 2.05; 95% CI, 1.76 – 2.57 (P = 1.76 × 10^−6^). However, none of the four imputed SNPs were significant in SUCCESS-A data.

### eQTL analysis

In order to test for correlation of the SNP rs8113308 genotypes and RNA expression, we performed an eQTL analysis in 821 ER-positive and 321 ER-negative breast tumors in the METABRIC project ([30](#R30), [31](#R31)). We conducted the analysis both in cis and in trans. For cis-eQTL analysis we included all the genes within 100kb of the SNP rs8113308; *ZNF613, ZNF350, ZNF615* and *ZNF649*. The re-annotation of Illumina probes by Barbosa-Morais and colleagues classified the probes for *ZNF615* and *ZNF649* as unreliable and probes for *ZNF613* and *ZNF350* as reliable ([39](#R39)); the eQTL was only considered for reliable probes. The appliance of a linear regression model did not reveal any significant cis or trans eQTLs. The cis-eQTL analysis utilizing the ANOVA model, allowing genotype to have both additive and dominant effects, indicated an association between rs11881650 (a tag SNP for rs8113308; r^2^ = 0.81) and expression of *ZNF350* (*P* = 3.85 × 10^−3^) in ER-positive breast tumors whereas no association was seen in ER-negative breast tumors ([Supplementary Table S4](#SD1)). The cis eQTL for *ZNF350* remains significant also after Bonferroni adjustment (adjusted *P* = 9.22 × 10^−3^). The high *ZNF350* mRNA expression level was linked to SNP rs11881650 rare homozygote genotype compared to the common homozygote and heterozygote genotypes, *P* = 0.018 and *P* = 0.015, respectively ([Fig. 3A](#F3)) in ER-positive breast tumors; no difference was seen among ER-negative tumors. Concordant supportive evidence was seen in peripheral blood tissue where rs7246064 (a tag SNP for rs8113308; r^2^ = 1) and *ZNF350* mRNA expression were correlated (*P* = 3.27 × 10^−3^) ([Supplementary Table S5](#SD1)), no significant correlation between SNP rs8113308 and *ZNF350* mRNA expression was seen in lymphoblastoid cell lines. The trans-eQTL analysis of the METABRIC data utilizing ANOVA model with Bonferroni adjusted *P* of < 0.05 and applied to the reliable probes revealed association with expression of the *EPS8L1* and *ZNF347* genes in 19q13 locus and *CYP26A1* in 10q23 in ER-positive breast tumors, with no significant eQTL for these genes among ER-negative tumors ([Supplementary Table S6](#SD1)). The eQTL analysis for SNP rs4767413 did not reveal any statistically significant cis- or trans-eQTL correlations.

### Figure 3.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a877/4574404/1971feda62b0/emss-63442-f0003.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4574404_emss-63442-f0003.jpg)

ZNF350 mRNA levels by genotype using Metabric data (A) and gene expression-based disease-free survival of ZNF350 using online web-based service BreastMark (B-D). A) Boxplot of ZNF350 mRNA levels by SNP rs11881650 (a tag SNP for rs8113308; r2 = 0.81) genotype (0=common homozygote, 1=heterozygote, 2=rare homozygote). * Wilcoxon rank sum test for common homozygote vs. rare homozygote, P = 0.018 in ER-positive tumors. Survival in B) ER-positive patients receiving tamoxifen treatment (N = 614, events = 149), C) ER-positive patients not receiving tamoxifen treatment (N = 1376, events = 451) and D) ER-negative patients (N = 423, events = 197). The cut-off for expression level was set to high, i.e. the top 25% expression level based on the inter quartile range. The follow-up time was not adjustable.

### Gene expression survival

We searched BreastMark database to analyse and visualize survival differences based on mRNA expression differences in public mRNA expression data. In the BreastMark the only endocrine treatment group available is tamoxifen, and no other information for endocrine treatment is given. *ZNF350* showed gene expression-based survival difference in ER-positive tamoxifen treated patients with high *ZNF350* expression associating with poor survival with HR 1.61 (1.14 – 2.27), logrank *P* = 0.006, whereas no survival difference by different *ZNF350* expression levels was seen in ER-positive tamoxifen non-treated patients or in ER-negative tamoxifen non-treated patients ([Fig. 3B-D](#F3)). No survival difference by gene expression level was seen for *ZNF615*, nor for *EPS8L1, ZNF347* or *CYP26A1*. For genes within 100kb of the SNP rs4767413 (*MAP1LC3B2* and *MIR4472-2*) survival difference by gene expression could not be analysed due to lack of the probe data for these genes in BreastMark.

### In silico functional studies

In order to assess the functional role of the rs8113308 locus we explored ENCODE data with designated tools HaploReg2 and RegulomeDB for regulatory elements and protein binding sites residing in the region. The Encode data indicates that the rs11879758, a tag SNP for rs8113308 (r^2^ = 0.85) locates in the site where there are promoter histone marks in nine cell lines and, based on the regulatory chromatin states, one active promoter in mammary epithelial cells with the closest annotated gene being *ZNF350*. Protein-protein interaction networks were searched by STRING program ([Supplementary Fig. S6](#SD7)) demanding a high confidence score for interactions (0.700) derived from experimental studies, databases and text mining and it showed *RNF11, ATXN2, BRCA1* and *GADD45A* proteins interacting with *ZNF350*.

## Discussion

ER-positive breast cancer is commonly treated with adjuvant endocrine therapies. Adjuvant endocrine treatment has been shown to increase overall survival and in light of recent studies is likely to be recommended for even longer duration in the adjuvant setting ([40](#R40), [41](#R41)). However many patients do not benefit from these therapies and predictors for response or resistance to endocrine treatment are urgently needed. In this study we report a meta-analysis of two genome wide studies and two validation datasets for identifying genetic variants associated with breast cancer related mortality specifically after adjuvant endocrine treatment. In a meta-analysis involving individuals treated with adjuvant endocrine therapy we identified SNP rs8113308 specifically and significantly predicting outcome after endocrine treatment. We were further able to show that among patients with ER-positive tumors, there is a significant interaction between the rs8113308 and endocrine treatment indicating a predictive, treatment-specific effect on survival, independent of conventional prognostic markers. In addition, SNP rs4767413 showed a consistent association across all four studies among ER-positive endocrine treated patients and a significant interaction result with endocrine treatment in stage I. However, the survival association was not significant in stage II studies (POSH validation and SUCCESS-A), and no further supportive evidence could be obtained from the eQTL and gene expression survival analyses. These results thus remain inconclusive and warrant further studies.

Our primary interest in this study was to evaluate the breast cancer specific survival. In stage I studies (HEBCS GWS and POSH GWS) and stage II POSH validation the analyses were performed using breast cancer specific mortality as the endpoint. A randomized clinical trial, SUCCESS-A, with data available via dbGAP was added to further validate our stage I findings. Since the only outcome data available for SUCCESS-A was overall or progression free survival, we used overall survival as the endpoint for SUCCESS-A in the stage II main meta-analysis but further performed also a sensitivity analysis assessing alternative endpoints throughout all four studies which showed very similar association, regardless of endpoint.

Various endocrine therapies work by different mechanisms to antagonize the growth-promoting activity of estrogen and different endocrine therapies are administered to pre-menopausal and post-menopausal women. In our study we combined anti-estrogen, aromatase inhibitor and LHRH agonist treatments in order to gain more statistical power. However, the most common endocrine treatment regimen in all the three datasets was tamoxifen and a similar result was found specifically within the tamoxifen-treated subgroup.

The direction of the association was consistent across the studies and remained statistically significant in the study-stratified pooled analyses even though patients in HEBCS GWS had relatively later onset breast cancer and POSH GWS contains only early-onset breast cancer patients. Furthermore, the age-adjusted multivariate analysis suggests that the treatment interaction was not affected by the patients’ age. In stage I we could specifically assess also the ER-positive patients not treated with endocrine therapy which allowed us to elucidate whether the association is linked to ER positivity or endocrine treatment. The significant interaction between endocrine treatment and rs8113308 among ER-positive patients provided strong evidence that the identified association links specifically and significantly to endocrine treatment subgroup.

Based on HapMap, 27% of the population carries at least one allele of this SNP and 3% carry the homozygous genotype. In HEBCS GWS, POSH GWS and POSH validation, 26%, 22% and 25% of the cases carried at least one allele (HR = 2.16) and 2.11%, 2.06%, 2,35% were homozygous carriers, respectively, (HR = 7.17). Previously, Kiyotani and colleagues performed a GWAS to assess the genetic factors influencing survival among patients receiving adjuvant tamoxifen therapy in Japanese population. They reported significant association with recurrence-free survival for SNP rs10509373 at 10q22 utilizing 240 patients in GWAS and 105 and 117 patients in the replication phase ([23](#R23)). We did not find any significantly associated SNPs on chromosome 10, but because of the differences in allele frequencies comparison between the two studies is difficult.

The SNP rs8113308 is located on the long arm of the chromosome 19, in intron five of a gene *ZNF613* encoding for Zinc Finger Protein 613. The 100kb flanking region harbours a multitude of zing finger genes. The cis-eQTL analysis indicated an association between rs11881650 (a tag SNP for rs8113308; r^2^ = 0.81) and mRNA levels of *ZNF350* (also known as *ZBRK1*) as well as trans eQTL associations to *EPS8L1, ZNF347* and *CYP26A1*. However, a tag SNP (r^2^ = 0.85) for rs8113308 locates in an active promoter site which is specifically active in mammary epithelial cells with the closest annotated gene being *ZNF350*; only *ZNF350* showed gene expression-based survival difference among ER-positive tamoxifen-treated patients, with the high *ZNF350* expression associating with poor survival. When investigating the correlation of the tag SNP rs11881650 genotypes and RNA expression in ER-positive tumorous breast tissue we found that the rare allele correlated with increased *ZNF350* mRNA expression levels. Together these data indicate that the rs8113308 rare allele associates with increased expression of *ZNF350* and poor survival of breast cancer patients after endocrine treatment.

In a network analysis with a high confidence score we saw an interplay with ZNF350, RNF11, GADD45A and BRCA1, with the two latter interacting with ER ([Supplementary Fig. S6](#SD7)). *ZNF350* expression is altered in different human carcinomas including breast cancer ([42](#R42)). Zinc Finger Protein 350 can bind *GADD45A* in a BRCA1-dependent manner ([43](#R43)) as well as via binding sites located in the *GADD45A* promoter region, suggesting that ZNF350 represses *GADD45A* expression via multiple binding sites ([44](#R44)). The overexpression of *ZNF350* has been shown to cause a decrease in both GADD45A transcripts and protein ([44](#R44)). GADD45A takes part in several cellular processes, including cell cycle arrest and apoptosis ([45](#R45)). Functional assays on GADD45 family proteins, including GADD45A, have shown that they bind to nuclear hormone receptors including ER alpha and act as nuclear co-activators ([46](#R46)). Recently it has also been found that high level of GADD45A protein expression correlates with ER-positivity and low level with ER-negativity in breast cancer ([47](#R47)). Since aberrant expression of cell cycle regulators has been suggested to contribute to tamoxifen resistance ([48](#R48)) it could be hypothesised that altered expression of *GADD45A* might have an effect on endocrine treatment response. Moreover, the role of GADD45A as co-activator of ER alpha could affect the antagonist effect of endocrine treatments that block the estrogen receptor binding, e.g. tamoxifen. The biological rationale behind the identified association of rs8113308 and breast cancer outcome after endocrine treatment might thus be the altered *ZNF350* expression levels and subsequent alterations in the expression and activity of GADD45A and its interacting partners, including ER alpha, but further studies are required to elucidate the actual mechanism.

To our knowledge, this is the first meta-analysis of two genome-wide studies and two validation sets to assess the genetic factors influencing survival for breast cancer patients receiving adjuvant endocrine treatment among women of European descent. It should be noted that our findings do not reach genome-wide significance as such, despite our use of GWAS as a starting point. However, the results are supported by the significant statistical interaction between the rs8113308 and endocrine treatment among patients with ER-positive tumors, indicating a predictive, treatment-specific effect on survival, independent of conventional prognostic markers, as well as by consistent *in silico* functional findings and a biological rationale. Pending further validation in other large datasets, our findings may potentially influence personalized treatment by identifying patients who would not benefit from endocrine treatment. Further fine mapping studies will help to identify the causative and most significant variants responsible for the observed associations, while functional studies will be necessary to fully elucidate the underlying biological mechanism.

## Supplementary Material

Endocrine therapies target ER-positive breast cancer, which accounts for the majority of all breast cancers. However, approximately 30% of ER-positive breast cancers do not respond to endocrine therapies. We identified a common rs8113308 SNP variation that was found to associate with poor breast cancer outcome after adjuvant endocrine therapy and improved breast cancer outcome in ER-negative patients, with a similar trend in ER-positive patients not treated with endocrine therapy. In addition we found a significant interaction between the rs8113308 and endocrine treatment among patients with ER-positive tumors indicating a predictive, treatment-specific effect on survival, independent of conventional prognostic markers. A biological rationale is suggested by *in silico* functional analyses. Pending further validation in additional datasets, this may have significant impact on personalized breast cancer treatment for identification of patients for whom adjuvant endocrine treatment would be ineffective and who could therefore be selected for clinical trials of alternative therapies.

Endocrine therapies target ER-positive breast cancer, which accounts for the majority of all breast cancers. However, approximately 30% of ER-positive breast cancers do not respond to endocrine therapies. We identified a common rs8113308 SNP variation that was found to associate with poor breast cancer outcome after adjuvant endocrine therapy and improved breast cancer outcome in ER-negative patients, with a similar trend in ER-positive patients not treated with endocrine therapy. In addition we found a significant interaction between the rs8113308 and endocrine treatment among patients with ER-positive tumors indicating a predictive, treatment-specific effect on survival, independent of conventional prognostic markers. A biological rationale is suggested by *in silico* functional analyses. Pending further validation in additional datasets, this may have significant impact on personalized breast cancer treatment for identification of patients for whom adjuvant endocrine treatment would be ineffective and who could therefore be selected for clinical trials of alternative therapies.

## Acknowledgments

We would like to thank Drs. Kirsimari Aaltonen, Dario Greco, Xiaofeng Dai, Päivi Heikkilä and Karl von Smitten, as well as Tuomas Heikkinen for their help with the HEBCS patient samples and data, and research nurse Irja Erkkilä for the assistance in the HEBCS data collection and management. For the POSH study we wish to thank Nikki Graham, Sylvia Diaper (University of Southampton DNA bank) and Kathy Potter (Southampton CRUK Centre Tissue Bank). Assistance with phenotype harmonization and genotype cleaning, as well as with general study coordination, was provided by the GARNET Coordinating Center (U01 HG005157). For the Genome-wide association study in breast cancer patients from the prospectively randomized SUCCESS-A trial assistance with data cleaning was provided by the National Center for Biotechnology Information.

**Financial support**

This work was supported by the Helsinki University Hospital Research Fund, the Finnish Cancer Society, the Nordic Cancer Union, the Academy of Finland [266528] and the Sigrid Juselius Foundation as well as the Emil Aaltonen Foundation and the Maud Kuistila Foundation (S. Khan). Funding for the POSH GWAS was provided by Breast Cancer Campaign grants 2010NovPR62 and 2013MayPR044 and the POSH study is supported by Cancer Research Grants C1275/A15956 and C1275/A19187. Genotyping at the National Institute of Singapore was financially supported by the Agency for Science, Technology and Research (A*STAR), Singapore. Funding support for the Genome-wide association study in breast cancer patients from the prospectively randomized SUCCESS-A trial was provided through the NHGRI Genomics and Randomized Trials Network [GARNET] (U01 HG005152). Funding support for genotyping, which was performed at the Center for Inherited Disease Research (CIDR) at Johns Hopkins University, was provided by the NIH Genes, Environment and Health Initiative [GEI] (U01 HG005137).

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Jemal A, Bray F, Center MM, Ferlay J, Ward E, Forman D. Global cancer statistics. CA: a cancer journal for clinicians. 2011;61:69–90. doi: 10.3322/caac.20107.  [DOI](https://doi.org/10.3322/caac.20107) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21296855/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=CA:%20a%20cancer%20journal%20for%20clinicians&title=Global%20cancer%20statistics&author=A%20Jemal&author=F%20Bray&author=MM%20Center&author=J%20Ferlay&author=E%20Ward&volume=61&publication_year=2011&pages=69-90&pmid=21296855&doi=10.3322/caac.20107&)

2. Harvey JM, Clark GM, Osborne CK, Allred DC. Estrogen receptor status by immunohistochemistry is superior to the ligand-binding assay for predicting response to adjuvant endocrine therapy in breast cancer. Journal of clinical oncology: official journal of the American Society of Clinical Oncology. 1999;17:1474–81. doi: 10.1200/JCO.1999.17.5.1474.  [DOI](https://doi.org/10.1200/JCO.1999.17.5.1474) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10334533/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20of%20clinical%20oncology:%20official%20journal%20of%20the%20American%20Society%20of%20Clinical%20Oncology&title=Estrogen%20receptor%20status%20by%20immunohistochemistry%20is%20superior%20to%20the%20ligand-binding%20assay%20for%20predicting%20response%20to%20adjuvant%20endocrine%20therapy%20in%20breast%20cancer&author=JM%20Harvey&author=GM%20Clark&author=CK%20Osborne&author=DC%20Allred&volume=17&publication_year=1999&pages=1474-81&pmid=10334533&doi=10.1200/JCO.1999.17.5.1474&)

3. Ali S, Buluwela L, Coombes RC. Antiestrogens and their therapeutic applications in breast cancer and other diseases. Annual review of medicine. 2011;62:217–32. doi: 10.1146/annurev-med-052209-100305.  [DOI](https://doi.org/10.1146/annurev-med-052209-100305) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21054173/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Annual%20review%20of%20medicine&title=Antiestrogens%20and%20their%20therapeutic%20applications%20in%20breast%20cancer%20and%20other%20diseases&author=S%20Ali&author=L%20Buluwela&author=RC%20Coombes&volume=62&publication_year=2011&pages=217-32&pmid=21054173&doi=10.1146/annurev-med-052209-100305&)

4. Early Breast Cancer Trialists’ Collaborative G. Davies C, Godwin J, Gray R, Clarke M, Cutter D, et al. Relevance of breast cancer hormone receptors and other factors to the efficacy of adjuvant tamoxifen: patient-level meta-analysis of randomised trials. Lancet. 2011;378:771–84. doi: 10.1016/S0140-6736(11)60993-8.  [DOI](https://doi.org/10.1016/S0140-6736(11)60993-8) | [PMC free article](/articles/PMC3163848/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21802721/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Relevance%20of%20breast%20cancer%20hormone%20receptors%20and%20other%20factors%20to%20the%20efficacy%20of%20adjuvant%20tamoxifen:%20patient-level%20meta-analysis%20of%20randomised%20trials&author=C%20Davies&author=J%20Godwin&author=R%20Gray&author=M%20Clarke&author=D%20Cutter&volume=378&publication_year=2011&pages=771-84&pmid=21802721&doi=10.1016/S0140-6736(11)60993-8&)

5. Dowsett M, Cuzick J, Ingle J, Coates A, Forbes J, Bliss J, et al. Meta-analysis of breast cancer outcomes in adjuvant trials of aromatase inhibitors versus tamoxifen. Journal of clinical oncology: official journal of the American Society of Clinical Oncology. 2010;28:509–18. doi: 10.1200/JCO.2009.23.1274.  [DOI](https://doi.org/10.1200/JCO.2009.23.1274) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19949017/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20of%20clinical%20oncology:%20official%20journal%20of%20the%20American%20Society%20of%20Clinical%20Oncology&title=Meta-analysis%20of%20breast%20cancer%20outcomes%20in%20adjuvant%20trials%20of%20aromatase%20inhibitors%20versus%20tamoxifen&author=M%20Dowsett&author=J%20Cuzick&author=J%20Ingle&author=A%20Coates&author=J%20Forbes&volume=28&publication_year=2010&pages=509-18&pmid=19949017&doi=10.1200/JCO.2009.23.1274&)

6. Riggins RB, Bouton AH, Liu MC, Clarke R. Antiestrogens, aromatase inhibitors, and apoptosis in breast cancer. Vitamins and hormones. 2005;71:201–37. doi: 10.1016/S0083-6729(05)71007-4.  [DOI](https://doi.org/10.1016/S0083-6729(05)71007-4) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16112269/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Vitamins%20and%20hormones&title=Antiestrogens,%20aromatase%20inhibitors,%20and%20apoptosis%20in%20breast%20cancer&author=RB%20Riggins&author=AH%20Bouton&author=MC%20Liu&author=R%20Clarke&volume=71&publication_year=2005&pages=201-37&pmid=16112269&doi=10.1016/S0083-6729(05)71007-4&)

7. Osborne CK, Schiff R. Mechanisms of endocrine resistance in breast cancer. Annual review of medicine. 2011;62:233–47. doi: 10.1146/annurev-med-070909-182917.  [DOI](https://doi.org/10.1146/annurev-med-070909-182917) | [PMC free article](/articles/PMC3656649/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20887199/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Annual%20review%20of%20medicine&title=Mechanisms%20of%20endocrine%20resistance%20in%20breast%20cancer&author=CK%20Osborne&author=R%20Schiff&volume=62&publication_year=2011&pages=233-47&pmid=20887199&doi=10.1146/annurev-med-070909-182917&)

8. Normanno N, Di Maio M, De Maio E, De Luca A, de Matteis A, Giordano A, et al. Mechanisms of endocrine resistance and novel therapeutic strategies in breast cancer. Endocrine-related cancer. 2005;12:721–47. doi: 10.1677/erc.1.00857.  [DOI](https://doi.org/10.1677/erc.1.00857) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16322319/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Endocrine-related%20cancer&title=Mechanisms%20of%20endocrine%20resistance%20and%20novel%20therapeutic%20strategies%20in%20breast%20cancer&author=N%20Normanno&author=M%20Di%20Maio&author=E%20De%20Maio&author=A%20De%20Luca&author=A%20de%20Matteis&volume=12&publication_year=2005&pages=721-47&pmid=16322319&doi=10.1677/erc.1.00857&)

9. Hoskins JM, Carey LA, McLeod HL. CYP2D6 and tamoxifen: DNA matters in breast cancer. Nature reviews Cancer. 2009;9:576–86. doi: 10.1038/nrc2683.  [DOI](https://doi.org/10.1038/nrc2683) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19629072/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature%20reviews%20Cancer&title=CYP2D6%20and%20tamoxifen:%20DNA%20matters%20in%20breast%20cancer&author=JM%20Hoskins&author=LA%20Carey&author=HL%20McLeod&volume=9&publication_year=2009&pages=576-86&pmid=19629072&doi=10.1038/nrc2683&)

10. Teh LK, Bertilsson L. Pharmacogenomics of CYP2D6: molecular genetics, interethnic differences and clinical importance. Drug metabolism and pharmacokinetics. 2012;27:55–67. doi: 10.2133/dmpk.dmpk-11-rv-121.  [DOI](https://doi.org/10.2133/dmpk.dmpk-11-rv-121) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22185816/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20metabolism%20and%20pharmacokinetics&title=Pharmacogenomics%20of%20CYP2D6:%20molecular%20genetics,%20interethnic%20differences%20and%20clinical%20importance&author=LK%20Teh&author=L%20Bertilsson&volume=27&publication_year=2012&pages=55-67&pmid=22185816&doi=10.2133/dmpk.dmpk-11-rv-121&)

11. Province MA, Goetz MP, Brauch H, Flockhart DA, Hebert JM, Whaley R, et al. CYP2D6 genotype and adjuvant tamoxifen: meta-analysis of heterogeneous study populations. Clinical pharmacology and therapeutics. 2014;95:216–27. doi: 10.1038/clpt.2013.186.  [DOI](https://doi.org/10.1038/clpt.2013.186) | [PMC free article](/articles/PMC3904554/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24060820/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clinical%20pharmacology%20and%20therapeutics&title=CYP2D6%20genotype%20and%20adjuvant%20tamoxifen:%20meta-analysis%20of%20heterogeneous%20study%20populations&author=MA%20Province&author=MP%20Goetz&author=H%20Brauch&author=DA%20Flockhart&author=JM%20Hebert&volume=95&publication_year=2014&pages=216-27&pmid=24060820&doi=10.1038/clpt.2013.186&)

12. Saladores P, Murdter T, Eccles D, Chowbay B, Zgheib NK, Winter S, et al. Tamoxifen metabolism predicts drug concentrations and outcome in premenopausal patients with early breast cancer. The pharmacogenomics journal. 2015;15:84–94. doi: 10.1038/tpj.2014.34.  [DOI](https://doi.org/10.1038/tpj.2014.34) | [PMC free article](/articles/PMC4308646/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25091503/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=The%20pharmacogenomics%20journal&title=Tamoxifen%20metabolism%20predicts%20drug%20concentrations%20and%20outcome%20in%20premenopausal%20patients%20with%20early%20breast%20cancer&author=P%20Saladores&author=T%20Murdter&author=D%20Eccles&author=B%20Chowbay&author=NK%20Zgheib&volume=15&publication_year=2015&pages=84-94&pmid=25091503&doi=10.1038/tpj.2014.34&)

13. Yu CP, Yu JC, Sun CA, Tzao C, Ho JY, Yen AM. Tumor susceptibility and prognosis of breast cancer associated with the G870A polymorphism of CCND1. Breast Cancer Res Treat. 2008;107:95–102. doi: 10.1007/s10549-007-9522-y.  [DOI](https://doi.org/10.1007/s10549-007-9522-y) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18043898/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Breast%20Cancer%20Res%20Treat&title=Tumor%20susceptibility%20and%20prognosis%20of%20breast%20cancer%20associated%20with%20the%20G870A%20polymorphism%20of%20CCND1&author=CP%20Yu&author=JC%20Yu&author=CA%20Sun&author=C%20Tzao&author=JY%20Ho&volume=107&publication_year=2008&pages=95-102&pmid=18043898&doi=10.1007/s10549-007-9522-y&)

14. Udler MS, Azzato EM, Healey CS, Ahmed S, Pooley KA, Greenberg D, et al. Common germline polymorphisms in COMT, CYP19A1, ESR1, PGR, SULT1E1 and STS and survival after a diagnosis of breast cancer. International journal of cancer Journal international du cancer. 2009;125:2687–96. doi: 10.1002/ijc.24678.  [DOI](https://doi.org/10.1002/ijc.24678) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19551860/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=International%20journal%20of%20cancer%20Journal%20international%20du%20cancer&title=Common%20germline%20polymorphisms%20in%20COMT,%20CYP19A1,%20ESR1,%20PGR,%20SULT1E1%20and%20STS%20and%20survival%20after%20a%20diagnosis%20of%20breast%20cancer&author=MS%20Udler&author=EM%20Azzato&author=CS%20Healey&author=S%20Ahmed&author=KA%20Pooley&volume=125&publication_year=2009&pages=2687-96&pmid=19551860&doi=10.1002/ijc.24678&)

15. Abraham JE, Harrington P, Driver KE, Tyrer J, Easton DF, Dunning AM, et al. Common polymorphisms in the prostaglandin pathway genes and their association with breast cancer susceptibility and survival. Clin Cancer Res. 2009;15:2181–91. doi: 10.1158/1078-0432.CCR-08-0716.  [DOI](https://doi.org/10.1158/1078-0432.CCR-08-0716) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19276290/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&title=Common%20polymorphisms%20in%20the%20prostaglandin%20pathway%20genes%20and%20their%20association%20with%20breast%20cancer%20susceptibility%20and%20survival&author=JE%20Abraham&author=P%20Harrington&author=KE%20Driver&author=J%20Tyrer&author=DF%20Easton&volume=15&publication_year=2009&pages=2181-91&pmid=19276290&doi=10.1158/1078-0432.CCR-08-0716&)

16. Wegman P, Elingarami S, Carstensen J, Stal O, Nordenskjold B, Wingren S. Genetic variants of CYP3A5, CYP2D6, SULT1A1, UGT2B15 and tamoxifen response in postmenopausal patients with breast cancer. Breast cancer research: BCR. 2007;9:R7. doi: 10.1186/bcr1640.  [DOI](https://doi.org/10.1186/bcr1640) | [PMC free article](/articles/PMC1851378/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17244352/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Breast%20cancer%20research:%20BCR&title=Genetic%20variants%20of%20CYP3A5,%20CYP2D6,%20SULT1A1,%20UGT2B15%20and%20tamoxifen%20response%20in%20postmenopausal%20patients%20with%20breast%20cancer&author=P%20Wegman&author=S%20Elingarami&author=J%20Carstensen&author=O%20Stal&author=B%20Nordenskjold&volume=9&publication_year=2007&pages=R7&pmid=17244352&doi=10.1186/bcr1640&)

17. Kiyotani K, Mushiroda T, Imamura CK, Hosono N, Tsunoda T, Kubo M, et al. Significant effect of polymorphisms in CYP2D6 and ABCC2 on clinical outcomes of adjuvant tamoxifen therapy for breast cancer patients. Journal of clinical oncology: official journal of the American Society of Clinical Oncology. 2010;28:1287–93. doi: 10.1200/JCO.2009.25.7246.  [DOI](https://doi.org/10.1200/JCO.2009.25.7246) | [PMC free article](/articles/PMC4872305/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20124171/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20of%20clinical%20oncology:%20official%20journal%20of%20the%20American%20Society%20of%20Clinical%20Oncology&title=Significant%20effect%20of%20polymorphisms%20in%20CYP2D6%20and%20ABCC2%20on%20clinical%20outcomes%20of%20adjuvant%20tamoxifen%20therapy%20for%20breast%20cancer%20patients&author=K%20Kiyotani&author=T%20Mushiroda&author=CK%20Imamura&author=N%20Hosono&author=T%20Tsunoda&volume=28&publication_year=2010&pages=1287-93&pmid=20124171&doi=10.1200/JCO.2009.25.7246&)

18. Azzato EM, Tyrer J, Fasching PA, Beckmann MW, Ekici AB, Schulz-Wendtland R, et al. Association between a germline OCA2 polymorphism at chromosome 15q13.1 and estrogen receptor-negative breast cancer survival. J Natl Cancer Inst. 2010;102:650–62. doi: 10.1093/jnci/djq057.  [DOI](https://doi.org/10.1093/jnci/djq057) | [PMC free article](/articles/PMC2864289/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20308648/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Natl%20Cancer%20Inst&title=Association%20between%20a%20germline%20OCA2%20polymorphism%20at%20chromosome%2015q13.1%20and%20estrogen%20receptor-negative%20breast%20cancer%20survival&author=EM%20Azzato&author=J%20Tyrer&author=PA%20Fasching&author=MW%20Beckmann&author=AB%20Ekici&volume=102&publication_year=2010&pages=650-62&pmid=20308648&doi=10.1093/jnci/djq057&)

19. Azzato EM, Pharoah PD, Harrington P, Easton DF, Greenberg D, Caporaso NE, et al. A genome-wide association study of prognosis in breast cancer. Cancer Epidemiol Biomarkers Prev. 2010;19:1140–3. doi: 10.1158/1055-9965.EPI-10-0085.  [DOI](https://doi.org/10.1158/1055-9965.EPI-10-0085) | [PMC free article](/articles/PMC2852476/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20332263/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Epidemiol%20Biomarkers%20Prev&title=A%20genome-wide%20association%20study%20of%20prognosis%20in%20breast%20cancer&author=EM%20Azzato&author=PD%20Pharoah&author=P%20Harrington&author=DF%20Easton&author=D%20Greenberg&volume=19&publication_year=2010&pages=1140-3&pmid=20332263&doi=10.1158/1055-9965.EPI-10-0085&)

20. Shu XO, Long J, Lu W, Li C, Chen WY, Delahanty R, et al. Novel genetic markers of breast cancer survival identified by a genome-wide association study. Cancer Res. 2012;72:1182–9. doi: 10.1158/0008-5472.CAN-11-2561.  [DOI](https://doi.org/10.1158/0008-5472.CAN-11-2561) | [PMC free article](/articles/PMC3294129/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22232737/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=Novel%20genetic%20markers%20of%20breast%20cancer%20survival%20identified%20by%20a%20genome-wide%20association%20study&author=XO%20Shu&author=J%20Long&author=W%20Lu&author=C%20Li&author=WY%20Chen&volume=72&publication_year=2012&pages=1182-9&pmid=22232737&doi=10.1158/0008-5472.CAN-11-2561&)

21. Rafiq S, Tapper W, Collins A, Khan S, Politopoulos I, Gerty S, et al. Identification of inherited genetic variations influencing prognosis in early-onset breast cancer. Cancer Res. 2013;73:1883–91. doi: 10.1158/0008-5472.CAN-12-3377.  [DOI](https://doi.org/10.1158/0008-5472.CAN-12-3377) | [PMC free article](/articles/PMC3601979/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23319801/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=Identification%20of%20inherited%20genetic%20variations%20influencing%20prognosis%20in%20early-onset%20breast%20cancer&author=S%20Rafiq&author=W%20Tapper&author=A%20Collins&author=S%20Khan&author=I%20Politopoulos&volume=73&publication_year=2013&pages=1883-91&pmid=23319801&doi=10.1158/0008-5472.CAN-12-3377&)

22. Rafiq S, Khan S, Tapper W, Collins A, Upstill-Goddard R, Gerty S, et al. A genome wide meta-analysis study for identification of common variation associated with breast cancer prognosis. PloS one. 2014;9:e101488. doi: 10.1371/journal.pone.0101488.  [DOI](https://doi.org/10.1371/journal.pone.0101488) | [PMC free article](/articles/PMC4272267/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25526632/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PloS%20one&title=A%20genome%20wide%20meta-analysis%20study%20for%20identification%20of%20common%20variation%20associated%20with%20breast%20cancer%20prognosis&author=S%20Rafiq&author=S%20Khan&author=W%20Tapper&author=A%20Collins&author=R%20Upstill-Goddard&volume=9&publication_year=2014&pages=e101488&pmid=25526632&doi=10.1371/journal.pone.0101488&)

23. Kiyotani K, Mushiroda T, Tsunoda T, Morizono T, Hosono N, Kubo M, et al. A genome-wide association study identifies locus at 10q22 associated with clinical outcomes of adjuvant tamoxifen therapy for breast cancer patients in Japanese. Hum Mol Genet. 2012;21:1665–72. doi: 10.1093/hmg/ddr597.  [DOI](https://doi.org/10.1093/hmg/ddr597) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22180457/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mol%20Genet&title=A%20genome-wide%20association%20study%20identifies%20locus%20at%2010q22%20associated%20with%20clinical%20outcomes%20of%20adjuvant%20tamoxifen%20therapy%20for%20breast%20cancer%20patients%20in%20Japanese&author=K%20Kiyotani&author=T%20Mushiroda&author=T%20Tsunoda&author=T%20Morizono&author=N%20Hosono&volume=21&publication_year=2012&pages=1665-72&pmid=22180457&doi=10.1093/hmg/ddr597&)

24. Syrjäkoski K, Vahteristo P, Eerola H, Tamminen A, Kivinummi K, Sarantaus L, et al. Population-based study of BRCA1 and BRCA2 mutations in 1035 unselected Finnish breast cancer patients. J Natl Cancer Inst. 2000;92:1529–31. doi: 10.1093/jnci/92.18.1529.  [DOI](https://doi.org/10.1093/jnci/92.18.1529) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10995809/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Natl%20Cancer%20Inst&title=Population-based%20study%20of%20BRCA1%20and%20BRCA2%20mutations%20in%201035%20unselected%20Finnish%20breast%20cancer%20patients&author=K%20Syrj%C3%A4koski&author=P%20Vahteristo&author=H%20Eerola&author=A%20Tamminen&author=K%20Kivinummi&volume=92&publication_year=2000&pages=1529-31&pmid=10995809&doi=10.1093/jnci/92.18.1529&)

25. Eerola H, Blomqvist C, Pukkala E, Pyrhonen S, Nevanlinna H. Familial breast cancer in southern Finland: how prevalent are breast cancer families and can we trust the family history reported by patients? European journal of cancer. 2000;36:1143–8. doi: 10.1016/s0959-8049(00)00093-9.  [DOI](https://doi.org/10.1016/s0959-8049(00)00093-9) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10854948/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=European%20journal%20of%20cancer&title=Familial%20breast%20cancer%20in%20southern%20Finland:%20how%20prevalent%20are%20breast%20cancer%20families%20and%20can%20we%20trust%20the%20family%20history%20reported%20by%20patients?&author=H%20Eerola&author=C%20Blomqvist&author=E%20Pukkala&author=S%20Pyrhonen&author=H%20Nevanlinna&volume=36&publication_year=2000&pages=1143-8&pmid=10854948&doi=10.1016/s0959-8049(00)00093-9&)

26. Kilpivaara O, Bartkova J, Eerola H, Syrjäkoski K, Vahteristo P, Lukas J, et al. Correlation of CHEK2 protein expression and c.1100delC mutation status with tumor characteristics among unselected breast cancer patients. International journal of cancer Journal international du cancer. 2005;113:575–80. doi: 10.1002/ijc.20638.  [DOI](https://doi.org/10.1002/ijc.20638) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15472904/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=International%20journal%20of%20cancer%20Journal%20international%20du%20cancer&title=Correlation%20of%20CHEK2%20protein%20expression%20and%20c.1100delC%20mutation%20status%20with%20tumor%20characteristics%20among%20unselected%20breast%20cancer%20patients&author=O%20Kilpivaara&author=J%20Bartkova&author=H%20Eerola&author=K%20Syrj%C3%A4koski&author=P%20Vahteristo&volume=113&publication_year=2005&pages=575-80&pmid=15472904&doi=10.1002/ijc.20638&)

27. Eccles D, Gerty S, Simmonds P, Hammond V, Ennis S, Altman DG. Prospective study of Outcomes in Sporadic versus Hereditary breast cancer (POSH): study protocol. BMC Cancer. 2007;7:160. doi: 10.1186/1471-2407-7-160.  [DOI](https://doi.org/10.1186/1471-2407-7-160) | [PMC free article](/articles/PMC1995215/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17697367/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Cancer&title=Prospective%20study%20of%20Outcomes%20in%20Sporadic%20versus%20Hereditary%20breast%20cancer%20(POSH):%20study%20protocol&author=D%20Eccles&author=S%20Gerty&author=P%20Simmonds&author=V%20Hammond&author=S%20Ennis&volume=7&publication_year=2007&pages=160&pmid=17697367&doi=10.1186/1471-2407-7-160&)

28. Li J, Humphreys K, Heikkinen T, Aittomaki K, Blomqvist C, Pharoah PD, et al. A combined analysis of genome-wide association studies in breast cancer. Breast Cancer Res Treat. 2011;126:717–27. doi: 10.1007/s10549-010-1172-9.  [DOI](https://doi.org/10.1007/s10549-010-1172-9) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20872241/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Breast%20Cancer%20Res%20Treat&title=A%20combined%20analysis%20of%20genome-wide%20association%20studies%20in%20breast%20cancer&author=J%20Li&author=K%20Humphreys&author=T%20Heikkinen&author=K%20Aittomaki&author=C%20Blomqvist&volume=126&publication_year=2011&pages=717-27&pmid=20872241&doi=10.1007/s10549-010-1172-9&)

29. Aulchenko YS, Ripke S, Isaacs A, van Duijn CM. GenABEL: an R library for genome-wide association analysis. Bioinformatics. 2007;23:1294–6. doi: 10.1093/bioinformatics/btm108.  [DOI](https://doi.org/10.1093/bioinformatics/btm108) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17384015/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=GenABEL:%20an%20R%20library%20for%20genome-wide%20association%20analysis&author=YS%20Aulchenko&author=S%20Ripke&author=A%20Isaacs&author=CM%20van%20Duijn&volume=23&publication_year=2007&pages=1294-6&pmid=17384015&doi=10.1093/bioinformatics/btm108&)

30. Dvinge H, Git A, Graf S, Salmon-Divon M, Curtis C, Sottoriva A, et al. The shaping and functional consequences of the microRNA landscape in breast cancer. Nature. 2013;497:378–82. doi: 10.1038/nature12108.  [DOI](https://doi.org/10.1038/nature12108) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23644459/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=The%20shaping%20and%20functional%20consequences%20of%20the%20microRNA%20landscape%20in%20breast%20cancer&author=H%20Dvinge&author=A%20Git&author=S%20Graf&author=M%20Salmon-Divon&author=C%20Curtis&volume=497&publication_year=2013&pages=378-82&pmid=23644459&doi=10.1038/nature12108&)

31. Curtis C, Shah SP, Chin SF, Turashvili G, Rueda OM, Dunning MJ, et al. The genomic and transcriptomic architecture of 2,000 breast tumours reveals novel subgroups. Nature. 2012;486:346–52. doi: 10.1038/nature10983.  [DOI](https://doi.org/10.1038/nature10983) | [PMC free article](/articles/PMC3440846/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22522925/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=The%20genomic%20and%20transcriptomic%20architecture%20of%202,000%20breast%20tumours%20reveals%20novel%20subgroups&author=C%20Curtis&author=SP%20Shah&author=SF%20Chin&author=G%20Turashvili&author=OM%20Rueda&volume=486&publication_year=2012&pages=346-52&pmid=22522925&doi=10.1038/nature10983&)

32. Shabalin AA. Matrix eQTL: ultra fast eQTL analysis via large matrix operations. Bioinformatics. 2012;28:1353–8. doi: 10.1093/bioinformatics/bts163.  [DOI](https://doi.org/10.1093/bioinformatics/bts163) | [PMC free article](/articles/PMC3348564/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22492648/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=Matrix%20eQTL:%20ultra%20fast%20eQTL%20analysis%20via%20large%20matrix%20operations&author=AA%20Shabalin&volume=28&publication_year=2012&pages=1353-8&pmid=22492648&doi=10.1093/bioinformatics/bts163&)

33. Westra HJ, Peters MJ, Esko T, Yaghootkar H, Schurmann C, Kettunen J, et al. Systematic identification of trans eQTLs as putative drivers of known disease associations. Nat Genet. 2013;45:1238–43. doi: 10.1038/ng.2756.  [DOI](https://doi.org/10.1038/ng.2756) | [PMC free article](/articles/PMC3991562/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24013639/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Systematic%20identification%20of%20trans%20eQTLs%20as%20putative%20drivers%20of%20known%20disease%20associations&author=HJ%20Westra&author=MJ%20Peters&author=T%20Esko&author=H%20Yaghootkar&author=C%20Schurmann&volume=45&publication_year=2013&pages=1238-43&pmid=24013639&doi=10.1038/ng.2756&)

34. Lappalainen T, Sammeth M, Friedlander MR, t Hoen PA, Monlong J, Rivas MA, et al. Transcriptome and genome sequencing uncovers functional variation in humans. Nature. 2013;501:506–11. doi: 10.1038/nature12531.  [DOI](https://doi.org/10.1038/nature12531) | [PMC free article](/articles/PMC3918453/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24037378/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Transcriptome%20and%20genome%20sequencing%20uncovers%20functional%20variation%20in%20humans&author=T%20Lappalainen&author=M%20Sammeth&author=MR%20Friedlander&author=PA%20t%20Hoen&author=J%20Monlong&volume=501&publication_year=2013&pages=506-11&pmid=24037378&doi=10.1038/nature12531&)

35. Ward LD, Kellis M. HaploReg: a resource for exploring chromatin states, conservation, and regulatory motif alterations within sets of genetically linked variants. Nucleic acids research. 2012;40:D930–4. doi: 10.1093/nar/gkr917.  [DOI](https://doi.org/10.1093/nar/gkr917) | [PMC free article](/articles/PMC3245002/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22064851/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20acids%20research&title=HaploReg:%20a%20resource%20for%20exploring%20chromatin%20states,%20conservation,%20and%20regulatory%20motif%20alterations%20within%20sets%20of%20genetically%20linked%20variants&author=LD%20Ward&author=M%20Kellis&volume=40&publication_year=2012&pages=D930-4&pmid=22064851&doi=10.1093/nar/gkr917&)

36. Boyle AP, Hong EL, Hariharan M, Cheng Y, Schaub MA, Kasowski M, et al. Annotation of functional variation in personal genomes using RegulomeDB. Genome research. 2012;22:1790–7. doi: 10.1101/gr.137323.112.  [DOI](https://doi.org/10.1101/gr.137323.112) | [PMC free article](/articles/PMC3431494/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22955989/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20research&title=Annotation%20of%20functional%20variation%20in%20personal%20genomes%20using%20RegulomeDB&author=AP%20Boyle&author=EL%20Hong&author=M%20Hariharan&author=Y%20Cheng&author=MA%20Schaub&volume=22&publication_year=2012&pages=1790-7&pmid=22955989&doi=10.1101/gr.137323.112&)

37. Madden SF, Clarke C, Gaule P, Aherne ST, O’Donovan N, Clynes M, et al. BreastMark: An Integrated Approach to Mining Publicly Available Transcriptomic Datasets Relating to Breast Cancer Outcome. Breast cancer research: BCR. 2013;15:R52. doi: 10.1186/bcr3444.  [DOI](https://doi.org/10.1186/bcr3444) | [PMC free article](/articles/PMC3978487/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23820017/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Breast%20cancer%20research:%20BCR&title=BreastMark:%20An%20Integrated%20Approach%20to%20Mining%20Publicly%20Available%20Transcriptomic%20Datasets%20Relating%20to%20Breast%20Cancer%20Outcome&author=SF%20Madden&author=C%20Clarke&author=P%20Gaule&author=ST%20Aherne&author=N%20O%E2%80%99Donovan&volume=15&publication_year=2013&pages=R52&pmid=23820017&doi=10.1186/bcr3444&)

38. Jensen LJ, Kuhn M, Stark M, Chaffron S, Creevey C, Muller J, et al. STRING 8--a global view on proteins and their functional interactions in 630 organisms. Nucleic acids research. 2009;37:D412–6. doi: 10.1093/nar/gkn760.  [DOI](https://doi.org/10.1093/nar/gkn760) | [PMC free article](/articles/PMC2686466/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18940858/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20acids%20research&title=STRING%208--a%20global%20view%20on%20proteins%20and%20their%20functional%20interactions%20in%20630%20organisms&author=LJ%20Jensen&author=M%20Kuhn&author=M%20Stark&author=S%20Chaffron&author=C%20Creevey&volume=37&publication_year=2009&pages=D412-6&pmid=18940858&doi=10.1093/nar/gkn760&)

39. Barbosa-Morais NL, Dunning MJ, Samarajiwa SA, Darot JF, Ritchie ME, Lynch AG, et al. A re-annotation pipeline for Illumina BeadArrays: improving the interpretation of gene expression data. Nucleic acids research. 2010;38:e17. doi: 10.1093/nar/gkp942.  [DOI](https://doi.org/10.1093/nar/gkp942) | [PMC free article](/articles/PMC2817484/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19923232/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20acids%20research&title=A%20re-annotation%20pipeline%20for%20Illumina%20BeadArrays:%20improving%20the%20interpretation%20of%20gene%20expression%20data&author=NL%20Barbosa-Morais&author=MJ%20Dunning&author=SA%20Samarajiwa&author=JF%20Darot&author=ME%20Ritchie&volume=38&publication_year=2010&pages=e17&pmid=19923232&doi=10.1093/nar/gkp942&)

40. Davies C, Pan H, Godwin J, Gray R, Arriagada R, Raina V, et al. Long-term effects of continuing adjuvant tamoxifen to 10 years versus stopping at 5 years after diagnosis of oestrogen receptor-positive breast cancer: ATLAS, a randomised trial. Lancet. 2013;381:805–16. doi: 10.1016/S0140-6736(12)61963-1.  [DOI](https://doi.org/10.1016/S0140-6736(12)61963-1) | [PMC free article](/articles/PMC3596060/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23219286/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Long-term%20effects%20of%20continuing%20adjuvant%20tamoxifen%20to%2010%20years%20versus%20stopping%20at%205%20years%20after%20diagnosis%20of%20oestrogen%20receptor-positive%20breast%20cancer:%20ATLAS,%20a%20randomised%20trial&author=C%20Davies&author=H%20Pan&author=J%20Godwin&author=R%20Gray&author=R%20Arriagada&volume=381&publication_year=2013&pages=805-16&pmid=23219286&doi=10.1016/S0140-6736(12)61963-1&)

41. Jin H, Tu D, Zhao N, Shepherd LE, Goss PE. Longer-term outcomes of letrozole versus placebo after 5 years of tamoxifen in the NCIC CTG MA.17 trial: analyses adjusting for treatment crossover. Journal of clinical oncology: official journal of the American Society of Clinical Oncology. 2012;30:718–21. doi: 10.1200/JCO.2010.34.4010.  [DOI](https://doi.org/10.1200/JCO.2010.34.4010) | [PMC free article](/articles/PMC3295549/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22042967/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20of%20clinical%20oncology:%20official%20journal%20of%20the%20American%20Society%20of%20Clinical%20Oncology&title=Longer-term%20outcomes%20of%20letrozole%20versus%20placebo%20after%205%20years%20of%20tamoxifen%20in%20the%20NCIC%20CTG%20MA.17%20trial:%20analyses%20adjusting%20for%20treatment%20crossover&author=H%20Jin&author=D%20Tu&author=N%20Zhao&author=LE%20Shepherd&author=PE%20Goss&volume=30&publication_year=2012&pages=718-21&pmid=22042967&doi=10.1200/JCO.2010.34.4010&)

42. Garcia V, Dominguez G, Garcia JM, Silva J, Pena C, Silva JM, et al. Altered expression of the ZBRK1 gene in human breast carcinomas. The Journal of pathology. 2004;202:224–32. doi: 10.1002/path.1513.  [DOI](https://doi.org/10.1002/path.1513) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14743505/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=The%20Journal%20of%20pathology&title=Altered%20expression%20of%20the%20ZBRK1%20gene%20in%20human%20breast%20carcinomas&author=V%20Garcia&author=G%20Dominguez&author=JM%20Garcia&author=J%20Silva&author=C%20Pena&volume=202&publication_year=2004&pages=224-32&pmid=14743505&doi=10.1002/path.1513&)

43. Zheng L, Pan H, Li S, Flesken-Nikitin A, Chen PL, Boyer TG, et al. Sequence-specific transcriptional corepressor function for BRCA1 through a novel zinc finger protein, ZBRK1. Molecular cell. 2000;6:757–68. doi: 10.1016/s1097-2765(00)00075-7.  [DOI](https://doi.org/10.1016/s1097-2765(00)00075-7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11090615/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Molecular%20cell&title=Sequence-specific%20transcriptional%20corepressor%20function%20for%20BRCA1%20through%20a%20novel%20zinc%20finger%20protein,%20ZBRK1&author=L%20Zheng&author=H%20Pan&author=S%20Li&author=A%20Flesken-Nikitin&author=PL%20Chen&volume=6&publication_year=2000&pages=757-68&pmid=11090615&doi=10.1016/s1097-2765(00)00075-7&)

44. Yun J, Lee WH. Degradation of transcription repressor ZBRK1 through the ubiquitin-proteasome pathway relieves repression of Gadd45a upon DNA damage. Molecular and cellular biology. 2003;23:7305–14. doi: 10.1128/MCB.23.20.7305-7314.2003.  [DOI](https://doi.org/10.1128/MCB.23.20.7305-7314.2003) | [PMC free article](/articles/PMC230312/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14517299/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Molecular%20and%20cellular%20biology&title=Degradation%20of%20transcription%20repressor%20ZBRK1%20through%20the%20ubiquitin-proteasome%20pathway%20relieves%20repression%20of%20Gadd45a%20upon%20DNA%20damage&author=J%20Yun&author=WH%20Lee&volume=23&publication_year=2003&pages=7305-14&pmid=14517299&doi=10.1128/MCB.23.20.7305-7314.2003&)

45. Salvador JM, Brown-Clay JD, Fornace AJ., Jr. Gadd45 in stress signaling, cell cycle control, and apoptosis. Advances in experimental medicine and biology. 2013;793:1–19. doi: 10.1007/978-1-4614-8289-5_1.  [DOI](https://doi.org/10.1007/978-1-4614-8289-5_1) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24104470/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Advances%20in%20experimental%20medicine%20and%20biology&title=Gadd45%20in%20stress%20signaling,%20cell%20cycle%20control,%20and%20apoptosis&author=JM%20Salvador&author=JD%20Brown-Clay&author=AJ%20Fornace&volume=793&publication_year=2013&pages=1-19&pmid=24104470&doi=10.1007/978-1-4614-8289-5_1&)

46. Yi YW, Kim D, Jung N, Hong SS, Lee HS, Bae I. Gadd45 family proteins are coactivators of nuclear hormone receptors. Biochemical and biophysical research communications. 2000;272:193–8. doi: 10.1006/bbrc.2000.2760.  [DOI](https://doi.org/10.1006/bbrc.2000.2760) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10872826/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochemical%20and%20biophysical%20research%20communications&title=Gadd45%20family%20proteins%20are%20coactivators%20of%20nuclear%20hormone%20receptors&author=YW%20Yi&author=D%20Kim&author=N%20Jung&author=SS%20Hong&author=HS%20Lee&volume=272&publication_year=2000&pages=193-8&pmid=10872826&doi=10.1006/bbrc.2000.2760&)

47. Tront JS, Willis A, Huang Y, Hoffman B, Liebermann DA. Gadd45a levels in human breast cancer are hormone receptor dependent. Journal of translational medicine. 2013;11:131. doi: 10.1186/1479-5876-11-131.  [DOI](https://doi.org/10.1186/1479-5876-11-131) | [PMC free article](/articles/PMC3665690/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23706118/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20of%20translational%20medicine&title=Gadd45a%20levels%20in%20human%20breast%20cancer%20are%20hormone%20receptor%20dependent&author=JS%20Tront&author=A%20Willis&author=Y%20Huang&author=B%20Hoffman&author=DA%20Liebermann&volume=11&publication_year=2013&pages=131&pmid=23706118&doi=10.1186/1479-5876-11-131&)

48. Musgrove EA, Sutherland RL. Biological determinants of endocrine resistance in breast cancer. Nature reviews Cancer. 2009;9:631–43. doi: 10.1038/nrc2713.  [DOI](https://doi.org/10.1038/nrc2713) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19701242/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature%20reviews%20Cancer&title=Biological%20determinants%20of%20endocrine%20resistance%20in%20breast%20cancer&author=EA%20Musgrove&author=RL%20Sutherland&volume=9&publication_year=2009&pages=631-43&pmid=19701242&doi=10.1038/nrc2713&)
