# Hypertrophy‐Associated Polymorphisms Ascertained in a Founder Cohort Applied to Heart Failure Risk and Mortality

## Metadata
**Authors:** Afshin Parsa, Yen‐Pei C Chang, Reagan J Kelly, Mary C Corretti, Kathleen A Ryan, Shawn W Robinson, Stephen S Gottlieb, Sharon LR Kardia, Alan R Shuldiner, Stephen B Liggett
**Journal:** Clinical and Translational Science
**Date:** 2011 Feb 23
**DOI:** [10.1111/j.1752-8062.2010.00251.x](https://doi.org/10.1111/j.1752-8062.2010.00251.x)
**PMID:** 21348951
**PMCID:** PMC4373555
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4373555/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC4373555/pdf/CTS-4-17.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC4373555/pdf/CTS-4-17.pdf)

## Abstract

A three‐stage approach was undertaken using genome‐wide, case‐control, and case‐only association studies to identify genetic variants associated with heart failure mortality. In an Amish founder population (n = 851), cardiac hypertrophy, a trait integral to the adaptive response to failure, was found to be heritable (h 2= 0.28, p = 0.0002) and GWAS revealed 21 candidate hypertrophy SNPs. In a case (n = 1,610)‐control (n = 463) study in unrelated Caucasians, one of the SNPs associated with hypertrophy (rs2207418, p = 8 × 10−6), was associated with heart failure, RR = 1.85(1.25–2.73, p = 0.0019). In heart failure cases rs2207418 was associated with increased mortality, HR = 1.51(1.20–1.97, p = 0.0004). There was consistency between studies, with the GG allele being associated with increased ventricular mass (˜13 g/m2) in the Amish, heart failure risk, and heart failure mortality. This SNP is in a gene desert of chromosome 20p12. Five genes are within 2.0 mbp of rs2207418 but with low LD between their SNPs and rs2207418. A region near this SNP is highly conserved in multiple vertebrates (lod score = 1,208). This conservation and the internal consistency across studies suggests that this region has biologic importance in heart failure, potentially acting as an enhancer or repressor element. rs2207418 may be useful for predicting a more progressive form of heart failure that may require aggressive therapy. Clin Trans Sci 2011; Volume 4: 17–23

Keywords: genetics, heart failure, hypertrophy, mortality, signal transduction

## Introduction

The progression of symptoms, ventricular dysfunction, arrhythmias, and mortality remains highly variable in chronic heart failure regardless of etiology or treatment.[1](#b1), [2](#b2) This interindividual variability, the variable penetrance of inherited cardiomyopathies,[3](#b3) and epidemiologic evidence,[4](#b4) suggest that common polymorphisms might have modifying effects in heart failure, predisposing to certain phenotypes including the rate of progression. An integral nodal point during the development of heart failure is cardiac hypertrophy.[5](#b5), [6](#b6) Cardiac hypertrophy is heritable[7](#b7) and is a recognized risk factor for heart failure.[8](#b8), [9](#b9) In this study, we hypothesized that interindividual variability in the genetic risk factors for the hypertrophy trait may also influence the risk for developing heart failure or its prognosis over time. While genome‐wide association studies have identified potential hypertrophy‐associated SNPs in various heterogeneous populations,[10](#b10), [11](#b11) these have not been associated with heart failure mortality.[12](#b12) We considered a different approach, by which potential hypertrophy‐associated SNPs were identified by GWAS in a homogeneous founder population of individuals in the absence of confounding disease that have highly similar environmental influences and genetic backgrounds and thus decreased noise. We hypothesized that SNPs associated with hypertrophy in this population altered the risk of heart failure and/or heart failure mortality. While these SNPs from the founder population might be considered risk factors for so called “physiological hypertrophy,” it has been proposed that the structural, metabolic and genomic events associated with physiologic and pathologic hypertrophy represent a continuum that includes commonalities.[13](#b13), [14](#b14) Thus, variability in the hypertrophic response may lead to subsequent variability in the adaptive and maladaptive events of progressive heart failure. For example, subjects genetically predisposed to excessive pathological hypertrophy in the face of a uniform stress might progress more readily to heart failure, or deteriorate more rapidly once heart failure has developed. On the other hand, a genetic predisposition to protective hypertrophy as develops in trained athletes might be resistant to heart failure or its adverse sequelae. In considering these possibilities we designed a study that would identify potential genetic determinants of cardiac hypertrophy in a normal founder population, and then specifically explored which of these impacted heart failure risk and progression.

## Methods

### Patient populations and echocardiography

The founder population consisted of 851 Old Order Amish individuals ages 20–80 years recruited from Lancaster, PA as part of the Heredity and Phenotype Intervention Heart (HAPI Heart) Study. This cohort of apparently healthy individuals has been previously described in detail.[15](#b15) The genealogy of all participants was known based on a formalized record system, which has been confirmed by genetic studies.[16](#b16) The heart failure cohort was established by the NHLBI‐sponsored Specialized Center of Clinically Oriented Research (SCCOR) and consisted of unrelated prospectively enrolled patients from the University of Cincinnati College of Medicine (*n* = 1,302) and the University of Maryland School of Medicine (*n* = 308). This group consisted of white patients between the ages of 18 and 90 years with left ventricular ejection fraction (LVEF) <35%, and a diagnosis of ischemic or nonischemic cardiomyopathy. Patients with valvular lesions or myocarditis, or those with hypertrophic cardiomyopathy, were not included. The control group has been previously described and consisted of 463 asymptomatic Caucasians with normal electrocardiograms and echocardiograms.[17](#b17) Left ventricular mass (LVM) was calculated from 2D echocardiography according to the American Society of Echocardiography recommendations.[18](#b18) LVM index (LVMI) was determined from the LVM divided by the calculated body surface area.

### GWAS in the Amish HAPI Heart Study

Genomic DNA from the Amish cohort was genotyped on the Affymetrix GeneChip Human Mapping 500K Array set (Affymetrix, Santa Clara, CA, USA). GeneChip Genotyping Analysis Software (GTYPE 4.0) was used for automated genotype calling as part of the GeneChip Operating Software platform. The GTYPE‐generated chip files were subsequently analyzed using the BRLMM genotype calling algorithm for greater accuracy.[19](#b19) Only samples with call rates >93% on both microarrays (*Nsp* I and *Stly* I digestions) were used for analysis. The mean call rate of the 851 resulting samples was 97.5%. Marker call rate was then assessed across the acceptable samples, and markers with a call rate >90% across samples and minor allele frequency >5% were considered for analysis (*n* = 361,034). Quantitative trait association analysis of LVMI using a variance component approach was then performed as implemented in SOLAR software (Southwest Foundation for Biomedical Reserch, San Antonio, TX, USA).[20](#b20) We used the measured genotype approach, in which we estimated the likelihood of additive, recessive and dominant genetic models given the pedigree structure.[21](#b21) Within each model, we simultaneously estimated the effects of age and sex. Parameter estimates were obtained by maximum likelihood methods, and the significance of association was tested by the likelihood ratio test. Selection of candidate SNPs from the Amish GWAS was performed in two steps. First we selected our top 13 loci based solely on a *p*‐value of ≤10^−6^ in our additive model. We then looked at our association results based on dominant and recessive models and selected another 10 candidate loci based on SNPs with a combination of lowest *p*‐value and consistency across all three genotypes.

### Follow‐up genotyping and association analysis in the normal and heart failure cohorts

Because the identified LVMI candidate SNPs were derived in the Amish, who are Caucasian, only patients who were self‐identified as “white” or “Caucasian” were included in follow‐up genotyping in the normal and heart failure cohorts from the general population. Of the SNPs found to be associated with LVMI in the Amish by the aforementioned criteria, several were in the same gene and were in 100% linkage disequilibrium (LD). In these situations one SNP from the group was chosen for further genotyping. Thus 21 SNP positions were genotyped for each subject in the normal and heart failure cohorts. TaqMan SNP Genotyping Assays (Applied Biosystems, Foster City, CA, USA) were utilized to detect the allele at each of the SNP sites identified in the Amish hypertrophy GWAS from DNA from the normal and heart failure cohorts. These assays were performed on an Applied Biosystems Model 7900HT Fast system, using the probes and primers supplied by the manufacturer for allelic discrimination, with a call rate of >95%. Hardy–Weinberg Equilibrium (HWE) was assessed using a chi‐square test for all SNPs with allele frequencies >0.01. Differences in the distribution of alleles between control and case cohorts were determined by Chi‐square tests. We utilized a prespecified mortality outcome which consisted of all‐cause mortality, cardiac transplant, and implanted cardiac defibrillator shock for ventricular tachycardia. Relative risks were obtained by Cox Proportional Hazards modeling using a genotypic model both before and after adjustment for age at diagnosis and sex. All analyses were carried out using the R Statistical Language ([http://www.R‐project.org](http://www.R-project.org)). Two‐tailed tests and an alpha level = 0.05 were used to assess statistical significance.

### Estimation of linkage disequilibrium and conserved sequence analysis

LD between the genotyped SNPs was calculated as *r* ^2^.[22](#b22) LD (*r* ^2^) relationships between the genotyped SNPs and those of CEU subjects from the HapMap (phase 1 and 2 datasets, [http://www.hapmap.org](http://www.hapmap.org)), were calculated using Haploview.[23](#b23) Estimates of sequence similarity and conservation was based on lod scores calculated by the PhastCons program (PhastCons Conserved Elements, 28‐way Vertebrate Multiz Alignment, University of California Santa Cruz, Santa Cruz, CA, USA) provided by UCSC genome browser.[24](#b24)

## Results

The demographic characteristics and echocardiography results for the Amish cohort (*n* = 851) are shown in [Table 1](#t1). The mean LVM was 152 g (interquartile range 45.0 g) and the mean LVMI was 82 g/cm^2^ (interquartile range 23.3 g/cm^2^). In this founder population, the heritability (*h* ^2^) for LVM was calculated to be 0.32, *p* < 0.0001, and for LVMI = 0.28, *p* = 0.0002. GWAS revealed 21 nonredundant SNPs associated with LVMI under an additive, dominant or recessive model using the specified criteria ([Table 2](#t2)). Typically, the difference in LVMI between the two homozygous states was ~10 g/m^2^. In some cases, there was a gene‐dose dependency for the three possible genotypes. For example, the HTR2A SNP had LVMI values of 94.4, 89.0, and 84.3 g/m^2^ for subjects who were homozygous for the minor allele, heterozygous, and homozygous for the major allele, respectively ([Table 2](#t2)). In other cases, the LVMI values for the homozygous minor (or major) allele and heterozygosity were similar, but clearly differed from the other homozygous allele. For example, for the FGF1 SNP the minor allelic state and heterozygosity had LVMI values of 86.6 and 87.3 g/m^2^, while the homozygous major allelic state had an LVMI value of 106.3 g/m^2^ ([Table 2](#t2)).

### Table 1.

| Age (y) | 43 ± 14.5 | LVID (D) (cm) | 4.8 ± 0.53 |
| --- | --- | --- | --- |
| Sex% female | 47 | LVID (S) (cm) | 2.8 ± 0.45 |
| SBP (mmHg) | 121 ± 14 | LVPWD (cm) | 0.89 ± 0.14 |
| DBP (mmHg) | 76 ± 8.6 | LV (%) | 65 ± 5.5 |
| Weight (kg) | 74 ± 12 | LVM (g) | 152 ± 42 |
| BMI (kg/m2) | 26.6 ± 4.6 | LVMI (g/m2) | 82 ± 20 |

Table 1 Caption: Demographic and echocardiographic parameters of the Old Order Amish cohort (n = 851, mean ± SD).

### Table 2.

| Chr | Position | Gene | Affy SNP | dbSNP ID | n (11) | n (12) | n (22) | LVMI (11) | LVMI (12) | LVMI (22) | p (add) | p (dom) | p (rec) |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| 10 | 105836064 | COL17A1 | SNP_A‐2243231 | rs1320448 | 0 | 80 | 672 |   | 97.1 | 84.9 | 1.80E‐08 |   | 1.80E‐08 |
| 1 | 214783160 | ESRRG | SNP_A‐1890136 | rs12757165 | 231 | 431 | 166 | 82.3 | 87.8 | 92.3 | 1.29E‐07 | 4.58E‐06 | 1.42E‐04 |
| 1 | 43368971 | SLC2A1 | SNP_A‐2033216 | rs16830359 | 0 | 79 | 723 |   | 97.2 | 85.8 | 1.43E‐07 |   | 1.43E‐07 |
| 6 | 30201343 | TRIM38 | SNP_A‐1985591 | rs10947055 | 0 | 109 | 679 |   | 94.9 | 85.1 | 2.24E‐07 |   | 2.24E‐07 |
| 10 | 57086967 | ZWINT | SNP_A‐2144007 | rs1916521 | 107 | 409 | 311 | 82.3 | 84.9 | 91.2 | 5.74E‐07 | 1.16E‐02 | 4.96E‐07 |
| 10 | 82689471 | NRG3 | SNP_A‐1876715 | rs1484170 | 41 | 271 | 514 | 95.6 | 90.3 | 84.8 | 1.33E‐06 | 4.63E‐03 | 6.33E‐06 |
| 8 | 61166451 | CA8 | SNP_A‐4288127 | rs6995588 | 673 | 104 | 0 | 85.8 | 95.0 |   | 2.47E‐06 | 2.47E‐06 |   |
| 6 | 104538858 | GRIK2 | SNP_A‐1780800 | rs4520040 | 1 | 155 | 600 | 139.8 | 92.9 | 85.8 | 3.03E‐06 | 4.45E‐03 | 8.48E‐06 |
| 3 | 110687103 | DPPA4 | SNP_A‐4271362 | rs769554 | 645 | 102 | 16 | 85.7 | 92.9 | 100.1 | 3.42E‐06 | 1.09E‐05 | 4.64E‐03 |
| 6 | 22354583 | SOX4 | SNP_A‐1903803 | rs4236016 | 0 | 91 | 668 |   | 94.7 | 85.3 | 4.38E‐06 |   | 4.38E‐06 |
| 15 | 23463432 | UBE3A | SNP_A‐2106127 | rs17636733 | 213 | 434 | 153 | 84.7 | 86.0 | 94.4 | 5.03E‐06 | 2.59E‐02 | 1.80E‐07 |
| 13 | 47096717 | HTR2A | SNP_A‐2046805 | rs1575891 | 53 | 283 | 439 | 94.3 | 89.0 | 84.2 | 6.32E‐06 | 3.09E‐03 | 4.54E‐05 |
| 3 | 12601516 | RAF1 | SNP_A‐1888231 | rs3729931 | 268 | 402 | 156 | 91.7 | 84.8 | 84.3 | 1.62E‐05 | 6.99E‐07 | 9.25E‐02 |
| 1 | 206824209 | PLXNA2 | SNP_A‐4235615 | rs17259784 | 49 | 339 | 421 | 90.2 | 90.3 | 84.2 | 3.42E‐05 | 2.84E‐01 | 6.40E‐06 |
| 5 | 141998044 | FGF1 | SNP_A‐1951378 | rs152528 | 256 | 508 | 23 | 86.6 | 87.3 | 106.3 | 1.21E‐02 | 3.36E‐01 | 7.72E‐07 |
| 3 | 7663662 | GRM7 | SNP_A‐2212718 | rs4686148 | 164 | 419 | 242 | 92.8 | 86.6 | 84.7 | 4.90E‐05 | 2.71E‐05 | 1.61E‐02 |
| 12 | 59083970 | FAM19A2 | SNP_A‐4219954 | rs10506410 | 65 | 318 | 436 | 97.3 | 85.9 | 85.8 | 1.34E‐03 | 1.74E‐06 | 1.49E‐01 |
| 4 | 20694428 | KCNIP4 | SNP_A‐2009635 | rs6817687 | 716 | 93 | 10 | 88.2 | 80.3 | 77.7 | 4.61E‐05 | 3.70E‐05 | 1.14E‐01 |
| 15 | 37102650 | C15orf54 | SNP_A‐2272171 | rs12907914 | 190 | 420 | 217 | 87.0 | 84.5 | 92.4 | 2.86E‐03 | 9.17E‐01 | 1.35E‐06 |
| 20 | 11124903 | 20p12 | SNP_A‐2007354 | rs2207418 | 41 | 320 | 445 | 99.5 | 86.5 | 86.3 | 9.16E‐03 | 8.86E‐06 | 2.90E‐01 |
| 13 | 48978848 | PHF11 | SNP_A‐2242054 | rs2031532 | 219 | 405 | 193 | 89.3 | 88.5 | 81.7 | 7.60E‐05 | 5.54E‐02 | 5.23E‐06 |

Table 2 Caption: SNPs associated with LVMI in the Old Order Amish cohort+.

Assays were developed for the list of 21 candidate hypertrophy SNPs to ascertain potential associations with heart failure risk and mortality in the case‐control, and case‐only studies, respectively. The healthy control population consisted of 463 Caucasian volunteers who were 60% female, with a mean (±SD) age of 49 ± 12.2 years and LVEF of 59 ± 9.9%. The demographic characteristics for the 1,610 patients in the heart failure cohort are shown in [Table 3](#t3). The mean age was 52.6 years at enrollment, the average follow‐up time was 5.8 years, and there were 711 events. The genotypes of the control subjects and the heart failure patients are shown in [Tables 4](#t4) and [5](#t5), respectively. Genotypes for one SNP in the control cohort (rs3729931) and two in the heart failure cohort (rs2207418 and rs4686148) showed some deviation from HWE (Supplementary *Table 1*).

### Table 3.

| N | 1,610 |
| --- | --- |
| Age (Y) | 52.6 ± 15.3 |
| Sex (% female) | 37.8 |
| EF% < 25 (%) | 42.6 |
| Follow‐up time (Y) | 5.8 ± 5.5 |
| Had event (N) | 711 |

Table 3 Caption: Demographic characteristics of the heart failure cohort (mean ± SD).

### Table 4.

| GENE/location | SNP | Alleles |  | Genotype counts |  |  | Allele frequencies |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- |
| 1 | 2 | 11 | 12 | 22 | Allele 1 | Allele 2 |  |  |
| FAM19A | rs10506410 | T | A | 22 | 155 | 194 | 0.268 | 0.732 |
| TRIM38 | rs10947055 | C | T | 5 | 68 | 375 | 0.087 | 0.913 |
| ESRRG | rs12757165 | A | G | 195 | 204 | 47 | 0.666 | 0.334 |
| C15orf54 | rs12907914 | C | G | 76 | 219 | 154 | 0.413 | 0.587 |
| COL17A1 | rs1320448 | A | G | 0 | 6 | 443 | 0.007 | 0.993 |
| NRG3 | rs1484170 | C | T | 21 | 129 | 298 | 0.191 | 0.809 |
| FGF1 | rs152528 | C | T | 170 | 199 | 70 | 0.614 | 0.386 |
| HTR2A | rs1575891 | T | C | 24 | 156 | 249 | 0.238 | 0.762 |
| SLCA1 | rs16830359 | A | G | 0 | 5 | 446 | 0.006 | 0.994 |
| PLXNA2 | rs17259784 | C | G | 17 | 123 | 306 | 0.176 | 0.824 |
| UBE3A | rs17636733 | T | C | 114 | 220 | 109 | 0.506 | 0.494 |
| ZWINT | rs1916521 | T | C | 63 | 213 | 167 | 0.383 | 0.617 |
| PHF11 | rs2031532 | G | A | 194 | 196 | 58 | 0.652 | 0.348 |
| 20p12 | rs2207418 | G | A | 36 | 174 | 242 | 0.272 | 0.728 |
| RAF1 | rs3729931 | G | A | 167 | 235 | 45 | 0.636 | 0.364 |
| SOX4 | rs4236016 | C | T | 0 | 2 | 446 | 0.002 | 0.998 |
| GRIK2 | rs4520040 | A | G | 1 | 29 | 426 | 0.034 | 0.966 |
| GRM7 | rs4686148 | T | G | 43 | 190 | 199 | 0.319 | 0.681 |
| KCNIP4 | rs6817687 | T | C | 293 | 128 | 13 | 0.823 | 0.177 |
| CA8 | rs6995588 | C | T | 447 | 10 | 1 | 0.987 | 0.013 |
| DPPA4 | rs769554 | C | T | 438 | 3 | 0 | 0.997 | 0.003 |
| +Alleles are based on the (+) strand relative to the human reference sequence from HapMap release #24 NCBI B36 assembly. |  |  |  |  |  |  |  |  |

Table 4 Caption: Genotypes of the normal control cohort+.

### Table 5.

|   | SNP | Alleles |  | Genotype counts |  |  | Allele frequencies |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- |
| GENE/location |   | 1 | 2 | 11 | 12 | 22 | Allele 1 | Allele 2 |
| FAM19A | rs10506410 | T | A | 76 | 532 | 847 | 0.235 | 0.765 |
| TRIM38 | rs10947055 | C | T | 16 | 248 | 1286 | 0.09 | 0.91 |
| ESRRG | rs12757165 | A | G | 621 | 563 | 121 | 0.692 | 0.308 |
| C15orf54 | rs12907914 | C | G | 236 | 715 | 576 | 0.389 | 0.611 |
| COL17A1 | rs1320448 | A | G | 0 | 25 | 1508 | 0.008 | 0.992 |
| NRG3 | rs1484170 | C | T | 46 | 448 | 1004 | 0.18 | 0.82 |
| FGF1 | rs152528 | C | T | 549 | 680 | 251 | 0.601 | 0.399 |
| HTR2A | rs1575891 | T | C | 87 | 576 | 827 | 0.252 | 0.748 |
| SLCA1 | rs16830359 | A | G | 1 | 18 | 1518 | 0.007 | 0.993 |
| PLXNA2 | rs17259784 | C | G | 48 | 443 | 1025 | 0.178 | 0.822 |
| UBE3A | rs17636733 | T | C | 461 | 734 | 341 | 0.539 | 0.461 |
| ZWINT | rs1916521 | T | C | 192 | 729 | 600 | 0.366 | 0.634 |
| PHF11 | rs2031532 | G | A | 580 | 566 | 167 | 0.657 | 0.343 |
| 20p12 | rs2207418 | G | A | 221 | 576 | 750 | 0.329 | 0.671 |
| RAF1 | rs3729931 | G | A | 612 | 738 | 169 | 0.646 | 0.354 |
| SOX4 | rs4236016 | C | T | 1 | 5 | 1530 | 0.002 | 0.998 |
| GRIK2 | rs4520040 | A | G | 5 | 120 | 1412 | 0.042 | 0.958 |
| GRM7 | rs4686148 | T | G | 115 | 680 | 705 | 0.303 | 0.697 |
| KCNIP4 | rs6817687 | T | C | 967 | 469 | 49 | 0.809 | 0.191 |
| CA8 | rs6995588 | C | T | 1527 | 29 | 2 | 0.989 | 0.011 |
| DPPA4 | rs769554 | C | T | 1460 | 15 | 0 | 0.995 | 0.005 |

Table 5 Caption: Genotypes of the heart failure cohort+

In comparing the distribution of alleles between normal controls and heart failure patients (Supplementary Table 2), one SNP, rs2207418, had a significant difference in the distribution of the three genotypes (*p* = 0.0016). In the unadjusted analysis, the relative risk of heart failure for the GG genotype is 1.98 (95% CI = 1.35 to 2.90, *p* = 0.0004). Adjusted for age and sex, the association was maintained with RR = 1.85 (95% CI = 1.25 to 2.73, *p* = 0.0019) for the homozygous minor allele. For the other Amish hypertrophy‐associated SNPs, no *p*‐value was less than 0.12 for the genotype distribution analysis between cases and controls (Supplementary Table 2). There was no association between the Amish hypertrophy‐associated SNPs and LVMI in the heart failure cohort (data not shown).

For the mortality association analysis in the heart failure cohort, we found that three of the LVM candidate SNPs were associated in the unadjusted analysis, and two in the age and sex adjusted analysis ([Table 6](#t6)). The most robust association with mortality was again with rs2207418. In heart failure, the homozygous minor allele for this SNP was associated with mortality with a hazard ratio of 1.57 (95% CI = 1.25 to 1.97, *p* = 0.00013) in the unadjusted model, and when adjusted for age and sex this ratio for the homozygous minor allele was virtually the same, being 1.51 (95% CI = 1.2 to 1.9, *p* = 0.0004). Interestingly, in the Amish this SNP was associated with an increased LVMI of ~13 g/m^2^, when compared to the heterozygotes or those homozygous for the major allele ([Table 2](#t2)). Thus, these findings are consistent across three studies in that the trait, risk, and hazard are found only with minor allele homozygosity. The other SNP associated with increased risk of mortality was in COL17A1. In the unadjusted analyses the G allele is associated with increased risk (HR = 1.71, *p* = 0.0051); in the adjusted analysis the effect size was similar (HR = 1.62, 95% CI = 1.10 to 2.37) with a modest level of significance (*p* = 0.013). There were no homozygous COL17A1 GG subjects. The rs152528 SNP in FGF1 was marginally significant for protection (HR = 0.852) in the unadjusted analysis but this was only observed with the heterozygous state. After adjustment, the association with the heterozygous state approached significance (HR = 0.855, 95% CI = 0.728 to 1.00, *p* = 0.056). No significant association was found for “carrier” status for the FGF1 SNP (data not shown).

### Table 6.

| Gene/location | rs | Genotype | Unadjusted |  | Age and sex adjusted |  |
| --- | --- | --- | --- | --- | --- | --- |
| Haz ratio (95% CI) | p | Haz ratio (95% CI) | p |  |  |  |
| 20p12 | 2207418 | AG | 1.10 (0.94 to 1.28) | 0.22 | 1.093 (0.93 to 1.28) | 0.27 |
| GG | 1.57 (1.25 to 1.97) | 0.00013 | 1.51 (1.20 to 1.97) | 0.0004 |  |  |
| COL17A1 | 1320448 | AG | 1.71 (11.17 to 2.48) | 0.0051 | 1.62 (1.10 to 2.37) | 0.013 |
| N/A |   |   |   |   |  |  |
| FGF1 | 152528 | CT | 0.852 (0.726 to 1.00) | 0.05 | 0.855 (0.728 to 1.00) | 0.056 |
| CC | 0.876 (0.701 to 1.093) | 0.24 | 0.894 (0.715 to 1.11) | 0.32 |  |  |

Table 6 Caption: SNPs associated with mortality in the heart failure cohort.

rs2207418 is located on chromosome 20p12 and is within a repeat element that belongs to MIR family of short interspersed nuclear elements (SINE), and is >500 kb away from the nearest known genes (see HapMap release 24 NCBI B36 assembly). There are five annotated genes, however, within ~2 Mb of this SNP: *C20orf94*, *BTBD3*, *SNAP25*, *MKK5*, and *JAG1*. Analysis of LD between rs2207418 and HapMap SNPs within and near these five genes (20 kb 59 and 39 to the first and last known exons) did not reveal any SNP with strong LD to rs2207418 (all pairwise *r* ^2^ < 0.10, Supplementary *Figures 1–5*). A further analysis of this region was performed to examine elements highly conserved between homologous sequences from 28 vertebrate species ([Figure 1](#f1)). We found that this region encompassing rs2207418 has multiple conserved elements across these species. One region (~15 kb 39) is also found in fish, with a lod score = 1,208 ([Figure 1](#f1)). This degree of conservation suggests that this region may serve an important function that has not been previously recognized, such as an enhancer or repressor element for one of the five aforementioned genes, or other genes that are more remote.

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/18be/5439848/1e48fada180a/CTS-4-17-g001.jpg)

Sequence comparisons across species in the SINE region surrounding rs2207418. The lod score was calculated from the PhastCons conserved elements 28‐way Multiz alignment of vertebrate sequences.

## Discussion

In an effort to identify variants that associate with heart failure mortality, we took a three‐stage approach that began with an unbiased GWAS for a relevant trait (cardiac hypertrophy) in a founder population. In this group of apparently healthy subjects LVM and LVMI values were normally distributed with estimated heritable components of 32% and 28%, respectively. The list of associated SNPs generated from this analysis was then utilized in a case‐control study for heart failure risk, and a case study with mortality as the end point, in cohorts of unrelated individuals. Because of the *a priori* evidence from the founder study for the 21 SNPs having association with a relevant trait (and the small number of SNPs being considered) we did not perform any corrections for multiple comparisons in the heart failure trials. Of these SNPs, the homozygous rs2207418 genotype was associated with an 85% increased risk for heart failure in the case‐control study, which was highly significant (*p* = 0.0019 in the adjusted analysis). In the mortality analysis this same SNP and two others showed some evidence of association, with rs2204718 having the most robust findings with a 51% increase in mortality at a significance level of *p* = 0.0004 for the homozygous GG genotype. This SNP is in a region of chr 20 (position 11124903) which consists of a SINE. While the SNP lies within a haplotype block (covering ~100 kbp, data not shown) this region still remains in the SINE, thus the LD has no readily inferred biologic implications. Five genes lie within 2 mbp of rs2204718: *JAG1*, *C20orf94*, *BTBD3*, *SNAP25*, and *MKK5.* However, the LD between rs2204718 and those within these five genes is low, with the most intriguing being the intronic SNP rs6108653 in *JAG1*, whose protein product JAG1 is a ligand for the Notch1 receptor. *JAG1* and Notch1 are critical for normal cardiac and great vessel development.[25](#b25) Mutations of *JAG1* are associated with severe congenital cardiac defects including Alagille syndrome,[26](#b26), [27](#b27) and tetralogy of Fallot,[28](#b28) however these represent developmental defects of the great vessels. Furthermore, LD between rs2204718 and rs6108653 is low (r^2^ < 0.01). We have considered that rs2204718 may lie within (or is in LD with another variant) in a previously unrecognized regulatory element of the aforementioned five genes, or more distant genes. In support of this notion, the SINE where rs2204718 is located represents a highly conserved region amongst primates and mammals, and a smaller segment is also conserved in other vertebrates such as fish ([Figure 1](#f1)). These findings are consistent with the concept that this region serves an important role in gene regulation, likely as an enhancer or repressor element.

Although the case‐control and case‐only trials are related (the same heart failure cases) it is important to recognize the consistency of the findings with rs2204718 across all three studies. In the Amish hypertrophy study, the SNP was associated with an increased LVMI (*p* = 8.8 × 10^−6^) in the homozygous (minor allele) state. In the case‐control study, this same genotype was associated with a significant risk of heart failure (*p* = 0.0019), and in the case mortality study it was also associated with a highly significant risk of death (*p* = 0.0004). This consistency lends credence to our assertion that rs2204718 represents a marker for heart failure risk and mortality in Caucasians. Of note, the minor allele is also present in those of African and Asian descent at frequencies comparable to those of Europeans (see HapMap phase III/release 2), so these results may also be applicable for those populations as well. The association with the *COL17A1* SNP rs1320448 was significant in the Amish hypertrophy cohort (*p* = 1 × 10^−8^), but the minor allele frequency in the Amish (~0.06) was ~10‐fold higher than in either of the unrelated Caucasian cohorts. There was no association with risk in the case‐control study, and the mortality association was restricted to 19 subjects who carried a minor allele and had somewhat modest statistical support with an adjusted *p*‐value of 0.013. Taken together, we would consider this SNP to be of minor, if any, importance in heart failure mortality in those of European descent. Interestingly, both Asian and African populations show higher minor allele frequencies of rs1320448 compared to Europeans, so this SNP may be more relevant in these populations. *COL17A1* encodes type XVII collagen, which is not expressed in the heart or great vessels. Mutations of *COL17A1* have been associated with epidermolysis and pemphigoid lesions,[29](#b29) but to our knowledge no associations between *COL17A1* variants and cardiac disease have been reported.

## Conclusion

In summary, we have identified candidate SNPs associated with cardiac hypertrophy in an Amish founder population. These were genotyped in case‐control and case‐only heart failure cohorts with the latter focusing on mortality. In a highly conserved SINE region on chromosome 20p12, a SNP was associated with hypertrophy in the Amish, heart failure risk in the case‐control study, and mortality in patients with heart failure. Patients with this SNP who are at high risk for progression/death might be considered for more aggressive therapy, but this will need to be addressed with additional clinical studies.

## Acknowledgments and Funding

We thank Patrick E. Donnelly and Wendy Post, MD, for performing echocardiograms and their interpretation in the Amish, Stacey L. McCulle for technical assistance and database management, Joanne P. Marshall, RN, for assistance in patient recruitment, and Esther Moses for manuscript preparation.

## Supporting information

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Givertz MM. Underlying causes and survival in patients with heart failure. N Engl J Med. 2000; 342: 1120–1122.  [DOI](https://doi.org/10.1056/NEJM200004133421509) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10760314/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Underlying%20causes%20and%20survival%20in%20patients%20with%20heart%20failure&author=MM%20Givertz&volume=342&publication_year=2000&pages=1120-1122&pmid=10760314&doi=10.1056/NEJM200004133421509&)

2. van Campen LC, Visser FC, Visser CA. Ejection fraction improvement by beta‐blocker treatment in patients with heart failure: an analysis of studies published in the literature. J Cardiovasc Pharmacol. 1998; 32(Suppl 1): S31–S35.  [DOI](https://doi.org/10.1097/00005344-199800003-00006) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9731693/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Cardiovasc%20Pharmacol&title=Ejection%20fraction%20improvement%20by%20beta%E2%80%90blocker%20treatment%20in%20patients%20with%20heart%20failure:%20an%20analysis%20of%20studies%20published%20in%20the%20literature&author=LC%20van%20Campen&author=FC%20Visser&author=CA%20Visser&volume=32&issue=Suppl%201&publication_year=1998&pages=S31-S35&pmid=9731693&doi=10.1097/00005344-199800003-00006&)

3. Franz W‐M, Müller OJ, Katus HA. Cardiomyopathies: from genetics to the prospect of treatment. Lancet. 2001; 358: 1627–1637.  [DOI](https://doi.org/10.1016/S0140-6736(01)06657-0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11716909/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Cardiomyopathies:%20from%20genetics%20to%20the%20prospect%20of%20treatment&author=W%E2%80%90M%20Franz&author=OJ%20M%C3%BCller&author=HA%20Katus&volume=358&publication_year=2001&pages=1627-1637&pmid=11716909&doi=10.1016/S0140-6736(01)06657-0&)

4. Kannel WB. Incidence and epidemiology of heart failure. Heart Fail Rev. 2000; 5: 167–173.  [DOI](https://doi.org/10.1023/A:1009884820941) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16228142/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Heart%20Fail%20Rev&title=Incidence%20and%20epidemiology%20of%20heart%20failure&author=WB%20Kannel&volume=5&publication_year=2000&pages=167-173&pmid=16228142&doi=10.1023/A:1009884820941&)

5. Dorn GWI. Having a change of heart: reversing the suicidal proclivities of cardiac myocytes. Trans Am Clin Climatol Assoc. 2009; 120: 189–198.  [PMC free article](/articles/PMC2744538/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19768177/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Trans%20Am%20Clin%20Climatol%20Assoc&title=Having%20a%20change%20of%20heart:%20reversing%20the%20suicidal%20proclivities%20of%20cardiac%20myocytes&author=GWI%20Dorn&volume=120&publication_year=2009&pages=189-198&pmid=19768177&)

6. Swynghedauw B, Delcayre C, Samuel JL, Mebazaa A, Cohen‐Solal A. Molecular mechanisms in evolutionary cardiology failure. Ann N Y Acad Sci. 2010; 1188: 58–67.  [DOI](https://doi.org/10.1111/j.1749-6632.2009.05084.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20201887/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20N%20Y%20Acad%20Sci&title=Molecular%20mechanisms%20in%20evolutionary%20cardiology%20failure&author=B%20Swynghedauw&author=C%20Delcayre&author=JL%20Samuel&author=A%20Mebazaa&author=A%20Cohen%E2%80%90Solal&volume=1188&publication_year=2010&pages=58-67&pmid=20201887&doi=10.1111/j.1749-6632.2009.05084.x&)

7. Arnett DK, Hong Y, Bella JN, Oberman A, Kitzman DW, Hopkins PN, Rao DC, Devereux RB. Sibling correlation of left ventricular mass and geometry in hypertensive African Americans and whites: the HyperGEN study. Hypertension Genetic Epidemiology Network. Am J Hypertens. 2001; 14: 1226–1230.  [DOI](https://doi.org/10.1016/s0895-7061(01)02200-2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11775131/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hypertension%20Genetic%20Epidemiology%20Network.%20Am%20J%20Hypertens.&title=Sibling%20correlation%20of%20left%20ventricular%20mass%20and%20geometry%20in%20hypertensive%20African%20Americans%20and%20whites:%20the%20HyperGEN%20study&author=DK%20Arnett&author=Y%20Hong&author=JN%20Bella&author=A%20Oberman&author=DW%20Kitzman&volume=14&publication_year=2001&pages=1226-1230&pmid=11775131&doi=10.1016/s0895-7061(01)02200-2&)

8. Gardin JM, McClelland R, Kitzman D, Lima JA, Bommer W, Klopfenstein HS, Wong ND, Smith VE, Gottdiener J. M‐mode echocardiographic predictors of six‐ to seven‐year incidence of coronary heart disease, stroke, congestive heart failure, and mortality in an elderly cohort (the Cardiovascular Health Study). Am J Cardiol. 2001; 87: 1051–1057.  [DOI](https://doi.org/10.1016/s0002-9149(01)01460-6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11348601/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Cardiol&title=M%E2%80%90mode%20echocardiographic%20predictors%20of%20six%E2%80%90%20to%20seven%E2%80%90year%20incidence%20of%20coronary%20heart%20disease,%20stroke,%20congestive%20heart%20failure,%20and%20mortality%20in%20an%20elderly%20cohort%20(the%20Cardiovascular%20Health%20Study)&author=JM%20Gardin&author=R%20McClelland&author=D%20Kitzman&author=JA%20Lima&author=W%20Bommer&volume=87&publication_year=2001&pages=1051-1057&pmid=11348601&doi=10.1016/s0002-9149(01)01460-6&)

9. Levy D, Garrison RJ, Savage DD, Kannel WB, Castelli WP. Prognostic implications of echocardiographically determined left ventricular mass in the Framingham Heart Study. N Engl J Med. 1990; 322: 1561–1566.  [DOI](https://doi.org/10.1056/NEJM199005313222203) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/2139921/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Prognostic%20implications%20of%20echocardiographically%20determined%20left%20ventricular%20mass%20in%20the%20Framingham%20Heart%20Study&author=D%20Levy&author=RJ%20Garrison&author=DD%20Savage&author=WB%20Kannel&author=WP%20Castelli&volume=322&publication_year=1990&pages=1561-1566&pmid=2139921&doi=10.1056/NEJM199005313222203&)

10. Vasan RS, Glazer NL, Felix JF, Lieb W, Wild PS, Felix SB, Watzinger N, Larson MG, Smith NL, Dehghan A, Grosshennig A, Schillert A, Teumer A, Schmidt R, Kathiresan S, Lumley T, Aulchenko YS, Konig IR, Zeller T, Homuth G, Struchalin M, Aragam J, Bis JC, Rivadeneira F, Erdmann J, Schnabel RB, Dorr M, Zweiker R, Lind L, Rodeheffer RJ, Greiser KH, Levy D, Haritunians T, Deckers JW, Stritzke J, Lackner KJ, Volker U, Ingelsson E, Kullo I, Haerting J, O’Donnell CJ, Heckbert SR, Stricker BH, Ziegler A, Reffelmann T, Redfield MM, Werdan K, Mitchell GF, Rice K, Arnett DK, Hofman A, Gottdiener JS, Uitterlinden AG, Meitinger T, Blettner M, Friedrich N, Wang TJ, Psaty BM, van Duijn CM, Wichmann HE, Munzel TF, Kroemer HK, Benjamin EJ, Rotter JI, Witteman JC, Schunkert H, Schmidt H, Volzke H, Blankenberg S. Genetic variants associated with cardiac structure and function: a meta‐analysis and replication of genome‐wide association data. JAMA. 2009; 302: 168–178.  [DOI](https://doi.org/10.1001/jama.2009.978-a) | [PMC free article](/articles/PMC2975567/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19584346/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA&title=Genetic%20variants%20associated%20with%20cardiac%20structure%20and%20function:%20a%20meta%E2%80%90analysis%20and%20replication%20of%20genome%E2%80%90wide%20association%20data&author=RS%20Vasan&author=NL%20Glazer&author=JF%20Felix&author=W%20Lieb&author=PS%20Wild&volume=302&publication_year=2009&pages=168-178&pmid=19584346&doi=10.1001/jama.2009.978-a&)

11. Smith NL, Felix JF, Morrison AC, Demissie S, Glazer NL, Loehr LR, Cupples LA, Dehghan A, Lumley T, Rosamond WD, Lieb W, Rivadeneira F, Bis JC, Folsom AR, Benjamin E, Aulchenko YS, Haritunians T, Couper D, Murabito J, Wang YA, Stricker BH, Gottdiener JS, Chang PP, Wang TJ, Rice KM, Hofman A, Heckbert SR, Fox ER, O’Donnell CJ, Uitterlinden AG, Rotter JI, Willerson JT, Levy D, van Duijn CM, Psaty BM, Witteman JC, Boerwinkle E, Vasan RS. Association of genome‐wide variation with the risk of incident heart failure in adults of European and African ancestry: a prospective meta‐analysis from the cohorts for heart and aging research in genomic epidemiology (CHARGE) consortium. Circ Cardiovasc Genet. 2010; 3: 256–266.  [DOI](https://doi.org/10.1161/CIRCGENETICS.109.895763) | [PMC free article](/articles/PMC3025695/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20445134/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circ%20Cardiovasc%20Genet&title=Association%20of%20genome%E2%80%90wide%20variation%20with%20the%20risk%20of%20incident%20heart%20failure%20in%20adults%20of%20European%20and%20African%20ancestry:%20a%20prospective%20meta%E2%80%90analysis%20from%20the%20cohorts%20for%20heart%20and%20aging%20research%20in%20genomic%20epidemiology%20(CHARGE)%20consortium&author=NL%20Smith&author=JF%20Felix&author=AC%20Morrison&author=S%20Demissie&author=NL%20Glazer&volume=3&publication_year=2010&pages=256-266&pmid=20445134&doi=10.1161/CIRCGENETICS.109.895763&)

12. Morrison AC, Felix JF, Cupples LA, Glazer NL, Loehr LR, Dehghan A, Demissie S, Bis JC, Rosamond WD, Aulchenko YS, Wang YA, Haritunians T, Folsom AR, Rivadeneira F, Benjamin EJ, Lumley T, Couper D, Stricker BH, O’Donnell CJ, Rice KM, Chang PP, Hofman A, Levy D, Rotter JI, Fox ER, Uitterlinden AG, Wang TJ, Psaty BM, Willerson JT, van Duijn CM, Boerwinkle E, Witteman JC, Vasan RS, Smith NL. Genomic variation associated with mortality among adults of European and african ancestry with heart failure: the cohorts for heart and aging research in genomic epidemiology consortium. Circ Cardiovasc Genet. 2010; 3: 248–255.  [DOI](https://doi.org/10.1161/CIRCGENETICS.109.895995) | [PMC free article](/articles/PMC3033765/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20400778/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circ%20Cardiovasc%20Genet&title=Genomic%20variation%20associated%20with%20mortality%20among%20adults%20of%20European%20and%20african%20ancestry%20with%20heart%20failure:%20the%20cohorts%20for%20heart%20and%20aging%20research%20in%20genomic%20epidemiology%20consortium&author=AC%20Morrison&author=JF%20Felix&author=LA%20Cupples&author=NL%20Glazer&author=LR%20Loehr&volume=3&publication_year=2010&pages=248-255&pmid=20400778&doi=10.1161/CIRCGENETICS.109.895995&)

13. Dorn GWI. The fuzzy logic of physiological cardiac hypertrophy. Hypertension. 2007; 49: 962–970.  [DOI](https://doi.org/10.1161/HYPERTENSIONAHA.106.079426) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17389260/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hypertension&title=The%20fuzzy%20logic%20of%20physiological%20cardiac%20hypertrophy&author=GWI%20Dorn&volume=49&publication_year=2007&pages=962-970&pmid=17389260&doi=10.1161/HYPERTENSIONAHA.106.079426&)

14. Bernardo BC, Weeks KL, Pretorius L, McMullen JR. Molecular distinction between physiological and pathological cardiac hypertrophy: experimental findings and therapeutic strategies. Pharmacol Ther. 2010; 128: 191–227.  [DOI](https://doi.org/10.1016/j.pharmthera.2010.04.005) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20438756/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacol%20Ther&title=Molecular%20distinction%20between%20physiological%20and%20pathological%20cardiac%20hypertrophy:%20experimental%20findings%20and%20therapeutic%20strategies&author=BC%20Bernardo&author=KL%20Weeks&author=L%20Pretorius&author=JR%20McMullen&volume=128&publication_year=2010&pages=191-227&pmid=20438756&doi=10.1016/j.pharmthera.2010.04.005&)

15. Sorkin J, Post W, Pollin TI, O’Connell JR, Mitchell BD, Shuldiner AR. Exploring the genetics of longevity in the Old Order Amish. Mech Ageing Dev. 2005; 126: 347–350.  [DOI](https://doi.org/10.1016/j.mad.2004.08.027) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15621217/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mech%20Ageing%20Dev&title=Exploring%20the%20genetics%20of%20longevity%20in%20the%20Old%20Order%20Amish&author=J%20Sorkin&author=W%20Post&author=TI%20Pollin&author=JR%20O%E2%80%99Connell&author=BD%20Mitchell&volume=126&publication_year=2005&pages=347-350&pmid=15621217&doi=10.1016/j.mad.2004.08.027&)

16. Pollin TI, McBride DJ, Agarwala R, Schaffer AA, Shuldiner AR, Mitchell BD, O’Connell JR. Investigations of the Y chromosome, male founder structure and YSTR mutation rates in the Old Order Amish. Hum Hered. 2008; 65: 91–104.  [DOI](https://doi.org/10.1159/000108941) | [PMC free article](/articles/PMC2857628/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17898540/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Hered&title=Investigations%20of%20the%20Y%20chromosome,%20male%20founder%20structure%20and%20YSTR%20mutation%20rates%20in%20the%20Old%20Order%20Amish&author=TI%20Pollin&author=DJ%20McBride&author=R%20Agarwala&author=AA%20Schaffer&author=AR%20Shuldiner&volume=65&publication_year=2008&pages=91-104&pmid=17898540&doi=10.1159/000108941&)

17. Liggett SB, Cresci S, Kelly RJ, Syed FM, Matkovich SJ, Hahn HS, Diwan A, Martini JS, Sparks L, Parekh RR, Spertus JA, Koch WJ, Kardia SL, Dorn II GW. A GRK5 polymorphism that inhibits b‐adrenergic receptor signaling is protective in heart failure. Nat Med. 2008; 14: 510–517.  [DOI](https://doi.org/10.1038/nm1750) | [PMC free article](/articles/PMC2596476/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18425130/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Med&title=A%20GRK5%20polymorphism%20that%20inhibits%20b%E2%80%90adrenergic%20receptor%20signaling%20is%20protective%20in%20heart%20failure&author=SB%20Liggett&author=S%20Cresci&author=RJ%20Kelly&author=FM%20Syed&author=SJ%20Matkovich&volume=14&publication_year=2008&pages=510-517&pmid=18425130&doi=10.1038/nm1750&)

18. Devereux R, Alonso D, Lutas E, Gottlieb G, Campo E, Sachs I, Reichek N. Echocardiographic assessment of left ventricular hypertrophy: comparison to necropsy findings. Am J Cardiol. 1986; 57: 450–458.  [DOI](https://doi.org/10.1016/0002-9149(86)90771-x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/2936235/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Cardiol&title=Echocardiographic%20assessment%20of%20left%20ventricular%20hypertrophy:%20comparison%20to%20necropsy%20findings&author=R%20Devereux&author=D%20Alonso&author=E%20Lutas&author=G%20Gottlieb&author=E%20Campo&volume=57&publication_year=1986&pages=450-458&pmid=2936235&doi=10.1016/0002-9149(86)90771-x&)

19. BRLMM: an improved genotype calling method for the GeneChip Human Mapping 500K Array Set . http://www.affymetrix.com/support/technical/whitepapers/brlmm_whitepaper.pdf. 2006. Ref Type: Electronic Citation  [http://www.affymetrix.com/support/technical/whitepapers/brlmm_whitepaper.pdf](http://www.affymetrix.com/support/technical/whitepapers/brlmm_whitepaper.pdf)

20. Almasy L, Blangero J. Multipoint quantitative‐trait linkage analysis in general pedigrees. Am J Hum Genet. 1998; 62: 1198–1211.  [DOI](https://doi.org/10.1086/301844) | [PMC free article](/articles/PMC1377101/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9545414/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=Multipoint%20quantitative%E2%80%90trait%20linkage%20analysis%20in%20general%20pedigrees&author=L%20Almasy&author=J%20Blangero&volume=62&publication_year=1998&pages=1198-1211&pmid=9545414&doi=10.1086/301844&)

21. Boerwinkle E, Chakraborty R, Sing CF. The use of measured genotype information in the analysis of quantitative phenotypes in man. I. Models and analytical methods. Ann Intern Med. 1986; 50: 181–194.  [DOI](https://doi.org/10.1111/j.1469-1809.1986.tb01037.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/3435047/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Intern%20Med&title=The%20use%20of%20measured%20genotype%20information%20in%20the%20analysis%20of%20quantitative%20phenotypes%20in%20man.%20I.%20Models%20and%20analytical%20methods&author=E%20Boerwinkle&author=R%20Chakraborty&author=CF%20Sing&volume=50&publication_year=1986&pages=181-194&pmid=3435047&doi=10.1111/j.1469-1809.1986.tb01037.x&)

22. Hill WG. Estimation of linkage disequilibrium in randomly mating populations. Heredity. 1974; 33: 229–239.  [DOI](https://doi.org/10.1038/hdy.1974.89) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/4531429/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Heredity&title=Estimation%20of%20linkage%20disequilibrium%20in%20randomly%20mating%20populations&author=WG%20Hill&volume=33&publication_year=1974&pages=229-239&pmid=4531429&doi=10.1038/hdy.1974.89&)

23. Barrett JC, Fry B, Maller J, Daly MJ. Haploview: analysis and visualization of LD and haplotype maps. Bioinformatics. 2005; 21: 263–265.  [DOI](https://doi.org/10.1093/bioinformatics/bth457) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15297300/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=Haploview:%20analysis%20and%20visualization%20of%20LD%20and%20haplotype%20maps&author=JC%20Barrett&author=B%20Fry&author=J%20Maller&author=MJ%20Daly&volume=21&publication_year=2005&pages=263-265&pmid=15297300&doi=10.1093/bioinformatics/bth457&)

24. Siepel A, Bejerano G, Pedersen JS, Hinrichs AS, Hou M, Rosenbloom K, Clawson H, Spieth J, Hillier LW, Richards S, Weinstock GM, Wilson RK, Gibbs RA, Kent WJ, Miller W, Haussler D. Evolutionarily conserved elements in vertebrate, insect, worm, and yeast genomes. Genome Res. 2005; 15: 1034–1050.  [DOI](https://doi.org/10.1101/gr.3715005) | [PMC free article](/articles/PMC1182216/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16024819/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Res&title=Evolutionarily%20conserved%20elements%20in%20vertebrate,%20insect,%20worm,%20and%20yeast%20genomes&author=A%20Siepel&author=G%20Bejerano&author=JS%20Pedersen&author=AS%20Hinrichs&author=M%20Hou&volume=15&publication_year=2005&pages=1034-1050&pmid=16024819&doi=10.1101/gr.3715005&)

25. Gridley T. Notch signaling and inherited disease syndromes. Hum Mol Genet. 2003; 12(Spec No1): R9–R13.  [DOI](https://doi.org/10.1093/hmg/ddg052) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12668592/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mol%20Genet&title=Notch%20signaling%20and%20inherited%20disease%20syndromes&author=T%20Gridley&volume=12&publication_year=2003&pages=R9-R13&pmid=12668592&doi=10.1093/hmg/ddg052&)

26. Boyer‐Di Ponio J, Wright‐Crosnier C, Groyer‐Picard MT, Driancourt C, Beau I, Hadchouel M, Meunier‐Rotival M. Biological function of mutant forms of JAGGED1 proteins in Alagille syndrome: inhibitory effect on Notch signaling. Hum Mol Genet. 2007; 16: 2683–2692.  [DOI](https://doi.org/10.1093/hmg/ddm222) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17720887/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mol%20Genet&title=Biological%20function%20of%20mutant%20forms%20of%20JAGGED1%20proteins%20in%20Alagille%20syndrome:%20inhibitory%20effect%20on%20Notch%20signaling&author=J%20Boyer%E2%80%90Di%20Ponio&author=C%20Wright%E2%80%90Crosnier&author=MT%20Groyer%E2%80%90Picard&author=C%20Driancourt&author=I%20Beau&volume=16&publication_year=2007&pages=2683-2692&pmid=17720887&doi=10.1093/hmg/ddm222&)

27. Crosnier C, Driancourt C, Raynaud N, Dhorne‐Pollet S, Pollet N, Bernard O, Hadchouel M, Meunier‐Rotival M. Mutations in JAGGED1 gene are predominantly sporadic in Alagille syndrome. Gastroenterology. 1999; 116: 1141–1148.  [DOI](https://doi.org/10.1016/s0016-5085(99)70017-x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10220506/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gastroenterology&title=Mutations%20in%20JAGGED1%20gene%20are%20predominantly%20sporadic%20in%20Alagille%20syndrome&author=C%20Crosnier&author=C%20Driancourt&author=N%20Raynaud&author=S%20Dhorne%E2%80%90Pollet&author=N%20Pollet&volume=116&publication_year=1999&pages=1141-1148&pmid=10220506&doi=10.1016/s0016-5085(99)70017-x&)

28. Eldadah ZA, Hamosh A, Biery NJ, Montgomery RA, Duke M, Elkins R, Dietz HC. Familial Tetralogy of Fallot caused by mutation in the jagged1 gene. Hum Mol Genet. 2001; 10: 163–169.  [DOI](https://doi.org/10.1093/hmg/10.2.163) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11152664/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mol%20Genet&title=Familial%20Tetralogy%20of%20Fallot%20caused%20by%20mutation%20in%20the%20jagged1%20gene&author=ZA%20Eldadah&author=A%20Hamosh&author=NJ%20Biery&author=RA%20Montgomery&author=M%20Duke&volume=10&publication_year=2001&pages=163-169&pmid=11152664&doi=10.1093/hmg/10.2.163&)

29. Pasmooij AM, Pas HH, Jansen GH, Lemmink HH, Jonkman MF. Localized and generalized forms of blistering in junctional epidermolysis bullosa due to COL17A1 mutations in the Netherlands. Br J Dermatol. 2007; 156: 861–870.  [DOI](https://doi.org/10.1111/j.1365-2133.2006.07730.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17263807/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Dermatol&title=Localized%20and%20generalized%20forms%20of%20blistering%20in%20junctional%20epidermolysis%20bullosa%20due%20to%20COL17A1%20mutations%20in%20the%20Netherlands&author=AM%20Pasmooij&author=HH%20Pas&author=GH%20Jansen&author=HH%20Lemmink&author=MF%20Jonkman&volume=156&publication_year=2007&pages=861-870&pmid=17263807&doi=10.1111/j.1365-2133.2006.07730.x&)
