# Inherited variation in OATP1B1 is associated with treatment outcome in acute myeloid leukemia

## Metadata
**Authors:** Christina D Drenberg, Steven W Paugh, Stanley B Pounds, Lei Shi, Shelley J Orwick, Lie Li, Shuiying Hu, Alice A Gibson, Raul C Ribeiro, Jeffrey E Rubnitz, William E Evans, Alex Sparreboom, Sharyn D Baker
**Journal:** Clinical pharmacology and therapeutics
**Date:** 2016 Feb 20
**DOI:** [10.1002/cpt.315](https://doi.org/10.1002/cpt.315)
**PMID:** 26663398
**PMCID:** PMC4898266
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4898266/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC4898266/pdf/nihms-744746.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC4898266/pdf/nihms-744746.pdf)

## Abstract

Using broad interrogation of clinically relevant ADME genes on the DMET platform, we identified a genetic variant in SLCO1B1 (rs2291075; c.597C>T), encoding the transporter OATP1B1, associated with event free (P=0.006, hazard ratio=1.74) and overall survival (P=0.012, hazard ratio=1.85) in children with de novo acute myeloid leukemia (AML). Lack of SLCO1B1 expression in leukemic blasts suggested the association might be due to inherent rather than somatic effect. rs2291075 was in strong linkage with known functional variants rs2306283 (c.388A>G) and rs4149056 (c.521T>C). Functional studies in vitro determined that four AML-directed chemotherapeutics (cytarabine, daunorubicin, etoposide, and mitoxantrone) are substrates for OATP1B1 and the mouse ortholog Oatp1b2. In vivo pharmacokinetic studies using Oatp1b2-deficient mice further confirmed our results. Collectively, these findings demonstrate an important role for OATP1B1 in the systemic pharmacokinetics of multiple drugs used in the treatment of AML and suggest that inherited variability in host transporter function influences the effectiveness of therapy.

Keywords: acute myeloid leukemia, SLCO1B1, pharmacogenomics, DMET

## Introduction

Improvements in survival have been achieved for children and adolescents with acute myeloid leukemia (AML), with the 5-year survival rates increasing between 1975 and 2010 from less than 20% to more than 70%.^1^ However, in the past decade, outcome has not improved. The mainstay therapeutic approach in AML is systemically administered intensive combination chemotherapy for all patients and hematopoietic stem cell transplantation for those at very high risk of relapse. Although several prognostic factors in childhood AML have been identified, current predictive survival analysis remains inadequate. A number of studies have evaluated mechanisms of drug resistance in AML; however, rather than taking a broad approach, these studies have focused on genes involved in the cellular metabolism of cytarabine (Ara-C), the backbone of nearly all AML treatment strategies^2-6^, or have used a candidate gene approach of drug metabolizing enzymes or drug transporters with inconclusive results.^7-10^

Using broad interrogation of clinically relevant genes involved in drug metabolism and disposition, as represented on the DMET Plus platform, we performed a pharmacogenetic-association study between inherited genetic variants and clinical outcomes in children with *de novo* AML. Confirmatory functional studies were performed utilizing *in vitro* and *in vivo* preclinical models.

## Results

### Genetic variation in SLCO1B1 is associated with survival outcome in AML

We performed a pharmacogenetic-association study on leukemic and/or germline DNA of 164 pediatric patients with *de novo* AML treated on the AML02 multicenter trial^11^ receiving combination chemotherapy with Ara-C, daunorubicin (Dauno), etoposide (VP16) and mitoxantrone (MTZ). Using the Affymetrix DMET Plus platform, we interrogated 1,936 variants in 225 genes that are involved in drug absorption, distribution, metabolism, and excretion (ADME). The workflow of analyses, demographic characteristics of this patient cohort, and treatment schema are illustrated in **Figure 1, Supplementary Table S1**, and **Supplementary Figure S1. Supplementary Table S2** lists the single-nucleotide polymorphisms (SNPs) that were significantly associated (*P*<0.05) with event free survival (EFS) and overall survival (OS). After adjusting for known prognostic factors including age, risk group, and treatment arm, and eliminating SNPs with minimal genotypic variation as described in the methods (e.g., minor allele frequency [MAF] > 0.01), the variant most significantly associated with both EFS (*P*=0.006, hazard ratio 1.74, 95% CI=1.17–2.61) and OS (*P*=0.012, hazard ratio 1.85, 95% CI=1.13–3.01) was a synonymous SNP rs2291075 in *SLCO1B1* (c.597C>T, F199F), a gene encoding the hepatocellular uptake transporter OATP1B1 (**Figure 2a-b, Table 1, Supplementary Table S2**). More specifically, patients homozygous (T/T) for rs2291075 had the most favorable outcome, heterozygous (C/T) patients had intermediate outcome, and non-carriers (C/C) had the worst EFS (**Figure 2a**) and OS (**Figure 2b**).

### Figure 1. Schematic of work flow to determine the association of genetic variations in DMET genes with treatment outcomes in pediatric AML.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c49b/4898266/1a5a833ec962/nihms-744746-f0001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4898266_nihms-744746-f0001.jpg)

A total of 232 patients with de novo AML were enrolled on the St. Jude Children's Research Hospital AML02 multicenter trial. There was a total of 224 evaluable DMET arrays (N = 165) comprised of 99 somatic only, 7 germline only, and 59 matched somatic/germline DNA samples eligible for analysis. All arrays had > 97% call rate. *Samples from the same patient with > 10% discordance were excluded from analysis (N = 1). Somatic and germline DMET genotyping results were combined (somatic data was used if both available, N = 165); a total of 793 probe sets with monotonic alleles were removed; leaving 1143 probe sets for analysis.

### Figure 2. Variation in SLCO1B1 significantly associates with treatment outcome.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c49b/4898266/9ce545e62135/nihms-744746-f0002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4898266_nihms-744746-f0002.jpg)

Kaplan-Meier plots showing a significant association between pediatric AML patients (N = 165) homozygous for the rs2291075 variant (T/T, red) with (a) event free survival (EFS) and (b) overall survival (OS). Heterozygous (C/T, blue) and non-carriers (C/C, black) are also shown in plots. Kaplan-Meier plots showing the association remained significant when patients were grouped based on common SLCO1B1 haplotypes (*1A, *1B, and other [*5, *15 = *1B + *5, others]) with (c) EFS and (d) OS (N = 164 due to lack of genotype call for one patient). Association of SNP and survival outcome was tested by Cox regression after adjusting for prognostic factors including age, risk group, and treatment arm. P-value represents the genotype and outcome association using an additive model.

### Table 1.

| Probe SetID | Gene | dbSNP ID | Common name | Genotype frequency | Testallele | Major/Minorallele | MAF | Event Free Survival |  |  | Overall Survival |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| p-value | HR | 95% CI of HR | p-value | HR | 95% CI of HR |  |  |  |  |  |  |  |  |
| AM_10503 | SLCO1B1 | rs2291075 | SLCO1B1_c.597C>T(F199F) | 68 C/C; 73 C/T; 23 T/T | C | C/T | 0.36 | 0.006 | 1.74 | 1.17 - 2.61 | 0.012 | 1.85 | 1.13 - 3.01 |
| AM_13021 | UGT1A1 | rs3755319 | UGT1A1*112_c.-1353A>C | 54 A/A; 66 A/C; 44 C/C | A | A/C | 0.47 | 0.03 | 0.70 | 0.50 - 0.98 | 0.02 | 0.63 | 0.42 - 0.93 |
| AM_12783 | ABCB11 | rs4668115 | ABCB11_c.-10013G>A | 19 A/A; 66 A/G; 79 G/G | G | G/A | 0.32 | 0.05 | 0.70 | 0.49 - 1.00 | 0.03 | 0.64 | 0.42 - 0.97 |
| AM_13018 | UGT1A1 | rs4124874 | UGT1A1*60_c.-3279T>G(Promoter) | 48 G/G; 64 G/T; 51 T/T; 1 NA | T | T/G | 0.49 | 0.009 | 0.64 | 0.46 - 0.90 | 0.01 | 0.59 | 0.39 - 0.88 |
| AM_13011 | UGT1A3 | rs7574296 | UGT1A3_c.477A>G(A159A) | 46 A/A; 63 A/G; 45 G/G; 10 NA | A | A/G | 0.50 | 0.006 | 0.62 | 0.44 - 0.88 | 0.02 | 0.62 | 0.41 - 0.93 |
| AM_13007 | UGT1A3 | rs6706232 | UGT1A3_c.81G>A(E27E) | 43 A/A; 66 A/G; 51 G/G; 4 NA | G | G/A | 0.48 | 0.004 | 0.61 | 0.43 - 0.86 | 0.01 | 0.56 | 0.37 - 0.85 |
| AM_10733 | SLC28A2 | rs10519020 | SLC28A2_c.734G>C(S245T) | 10 C/G; 154 G/G | G | G/C | 0.03 | 0.002 | 0.31 | 0.14 - 0.69 | 0.002 | 0.29 | 0.12 - 0.68 |

Table 1 Caption: Significant SNPs (P < 0.05) associated with EFS and OS in 164 pediatric AML patients meeting inclusion criteria and sorted by hazards ratio.

To investigate the predictive value of rs2291075 compared with established risk factors, we fit a multi-predictor model to the outcome data (**Supplementary Table S3**), similar to that previously reported by Rubnitz et al.^11^ Each copy of the T allele reduced the rate of events by a factor of 0.492 (95% CI = 0.321-0.754; p = 0.001) and the rate of death by a factor of 0.419 (95% CI = 0.240–0.734; p = 0.002). We found that the association of rs2291075 genotype was maintained among both low and high risk patients where patients with homozygous (T/T) genotype had the most favorable outcome and non-carriers (C/C) had the worst EFS and OS (**Supplementary Figure S2**). While the effect of rs2291075 genotype was small among patients with low risk factors (EFS, *P*=0.021; OS, *P*=0.544; **Supplementary Figure S2c-d**), among patients with high risk factors, the homozygous (T/T) genotype conferred an outcome comparable to that of patients with low risk factors for EFS (*P*<0.0001) and OS (*P*<0.0001) (**Supplementary Figure S2e-f**). We found no evidence that the genotype effect differs by treatment arm for EFS (*P*=0.45) or OS (*P*=0.42); regardless of treatment arm, the same trend was observed where patients harboring homozygous (T/T) genotype had improved survival and non-carriers (C/C) had a worse outcome, although the association was only statistically significant for EFS in patients treated with high-dose Ara-C (**Supplementary Figure S3**). Since SNPs in *SLCO1B1* have previously been associated with race^12^, we assessed our patient population based on three racial groups (white, black, and other). We did not observe an association between outcome and race for either EFS (*P*=0.106) or OS (*P*=0.238) (**Supplemental Figure S4a-b**). While the same trend associating rs2291075 genotype with survival was observed within each racial group, the data did not achieve statistical significance (**Supplemental Figure S4c-h**). In addition to survival outcomes, we evaluated the association of rs2291075 with toxicity and found no significant correlations (**Supplemental Table S4**).

We found the MAF of rs2291075 (MAF=0.36) in our patient cohort was identical to the MAF in the HapMap CEU (Utah residents with ancestry from northern and western Europe) dataset (MAF=0.36). Proxy SNPs were then determined using SNAP version 2.2^13^ based on phased genotype data for people of European ancestry available from 1000 genomes 1 data in the International HapMap Project^14^ and implementing a stringent r^2^ cutoff (r^2^>0.8). While the majority of proxy SNPs were intronic, interestingly, a missense SNP rs2306283 (c.388A>G, N130D; found in haplotype *1B, among others) was found to be in complete linkage disequilibrium ([LD], D’=1, r^2^=0.932) in the HapMap dataset (**Figure 3**).

### Figure 3. Identification of SNPs in SLCO1B1 located in close proximity to rs2291075.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c49b/4898266/2cead755db3f/nihms-744746-f0003.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4898266_nihms-744746-f0003.jpg)

SNAP analysis version 2.2 developed by the Broad Institute13 was performed using data from the CEU population of the 1000 Genomes pilot 1 dataset and based on phased genotype data from the International HapMap Project14. The linkage disequilibrium (LD) plot shows proxy SNPs based on physical distance on chromosome 12 and r2. The horizontal dashed line indicates r2 > 0.8. The 10 SNPs in SLCO1B1 meeting these criteria are listed in the table, their distance, r2 and D’ relative to rs2291075 are indicated.

We then assessed all SNPs in *SLCO1B1* on the DMET Plus array (N=18) for LD using the HaploView 4.2 software.^15^ All SNPs were found to be in Hardy-Weinberg equilibrium with the exception of rs11045819 (*P*<0.05). rs4149056 (c.521T>C, V174A; variant allele present in *5, *15 and *17 haplotypes), which has been shown to have reduced function^16^ and is associated with adverse clinical effects such as statin induced myopathy^17^ and methotrexate pharmacokinetics in children with acute lymphoblastic leukemia,^18, 19^ was found to be in perfect LD with rs2291075 (D’=1; r^2^=0.177) (**Supplementary Figure S5**). While rs4149056 did not significantly associate with EFS (*P*=0.069; hazard ratio 2.08, 95% CI=0.93–4.68), it was significantly associated with OS (*P*=0.05; hazard ratio 2.97, 95% CI= 0.95–9.31) (**Supplementary Table S5**). Additionally, rs2306283 (variant allele present in *1B haplotype), which has been shown to have increased activity for some substrates^20^, was found to be in strong LD with rs2291075 (among others; D’=0.866; r^2^=0.545) (**Supplementary Figure S5**) consistent with the HapMap proxy SNP analysis. Given the level of LD between rs2291075 and known functional variants rs2306283 and rs4149056, we assigned haplotypes involving these 3 SNPs (*1A [wild-type]; *1B; other^21^) to all patients (**Supplementary Table S6**) and found that patients designated homozygous *1A/*1A or *1A/*1B (N=68) had the worst EFS and OS, those heterozygous for *1A and *1B (*1A/other; *1B/other; N=72) had an intermediate outcome, and patients that were not carriers of *1A or *1B (other/other; N=23) had the most favorable outcome (EFS, *P*=0.01; OS, *P*=0.01) (**Figure 2c-d**). These findings suggest that OATP1B1 has an important role in the pharmacokinetics and/or pharmacodynamics of AML-directed chemotherapy and may be an important predictor of drug response in AML patients.

To decipher whether variation in *SLCO1B1* was associated with the leukemic blasts or was inherent to the host, we evaluated gene expression of *SLCO1B1* in pediatric AML primary blast samples and a panel of human AML cell lines considered to be either cytogenetically normal or harboring a variety of clinically relevant genetic lesions and representing a variety of subtypes. In comparison to expression of *SLC29A1* (**Figure 4**), a gene encoding the equilibrative nucleoside transporter 1 (ENT-1) and known to be expressed on AML blasts^2^, expression of *SLCO1B1* in AML cells from patients or cell lines was negligible, suggesting that germline variation might underlie the associations we observed between *SLCO1B1* variants and outcome rather than somatic changes (**Figure 4**; portion of data publically available^22^). This is not surprising, since OATP1B1 is liver specific.^23^ Given these findings, and the strong LD of SNP rs2291075 with two functional variants, we hypothesized that one or more of the anti-leukemic agents used to treat AML may be a substrate of OATP1B1, and performed further functional studies *in vitro* and *in vivo*.

### Figure 4. Gene expression of SLCO1B1 in pediatric AML primary blast samples.

![Figure 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c49b/4898266/706ecc2a6cc9/nihms-744746-f0004.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4898266_nihms-744746-f0004.jpg)

Heatmap representation of SLCO1B1 (probe ID 210366_at) and SLC29A1 (probe IDs: 201801_s_at and 201802_at) gene expression in pediatric AML primary blast22 samples and human AML cell lines, representing multiple subtypes of AML and a variety of clinically relevant mutations. R program was used to visualize log2 gene expression values from an Affymetrix Human U133A and U133A Plus 2.0 arrays. Probe set signal values were set to lowest value across the entire data set and highest value. Each case is represented by a color with red representing high expression and blue representing low expression (scale shown below heatmap). Genetic subtype of each case is represented by colored bars across the top. Normal, cytogenetically normal; Inv(16), inversion 16; t(15;17), translocation of chromosomes 15 and 17; t(8;21), translocation of chromosomes 8 and 21; MLL, MLL-rearranged AML; M7, French-American-British (FAB) subtype 7 designation; CL, cell line.

### Multiple AML-directed chemotherapies are substrates for OATP1B1

Cellular transport studies of Ara-C, Dauno, VP16, and MTZ were performed *in vitro* with cells stably expressing OATP1B1 *1A (wild-type). Interestingly, we found that the intracellular uptake of all four drugs was significantly enhanced compared with vector control cells in a time- and concentration-dependent manner, with the exception of VP16 (**Supplementary Figure S6**). Moreover, the efficiency of drug transport of all four drugs was significantly increased in vector control cells compared with cells expressing OATP1B1*1A (transport increased 1.5-2-fold); whereas we observed significant impaired transport in cells expressing either OATP1B1*5 (transport reduced 46.3–77.1%; *P*<0.0001) or OATP1B1*15 (transport reduced 35.2–80.5%; *P*<0.0001) compared with OATP1B1*1A; and with Ara-C in OATP1B1*1B expressing cells (transport reduced 68.4%; *P*<0.0001; **Supplementary Figure S7, Figure 5**).

### Figure 5. Influence of SLCO1B1 haplotypes on transport of AML-directed chemotherapy.

![Figure 5](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c49b/4898266/3c3afeac5ff8/nihms-744746-f0005.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4898266_nihms-744746-f0005.jpg)

[3H]-Labeled AML-directed chemotherapy were incubated with HEK293 cells expressing the OATP1B1*1A, *1B, *5, and *15 variants for 15 min. Intracellular concentrations of (a) cytarabine (Ara-C; 10 μM), (b) daunorubicin (Dauno; 10 μM), (c) etoposide (VP16; 10 μM), and (d) mitoxantrone (MTZ; 1 μM) were determined by liquid scintillation counting. Data represent the mean of 2-3 independent experiments performed in triplicate and are expressed as % wild-type (i.e., OATP1B1*1A). Data represent the mean ± s.d. *, P< 0.0001, two-tailed t-test.

Next, we determined whether these drugs were substrates for the mouse ortholog transporter Oatp1b2 *in vitro*. Intracellular uptake of all four drugs was enhanced compared with vector control cells; this was statistically significant for Ara-C, Dauno, and VP16 (**Supplementary Figure S8**). To test whether Ara-C, Dauno, VP16, and/or MTZ are transported by OATP1B-type carriers *in vivo*, we determined the liver uptake and plasma pharmacokinetic profile of these agents (5-10 mg/kg; i.v. or i.p.) in mice deficient in the ortholog transporter Oatp1b2 [Oatp1b2(−/−)]. We found that liver accumulation, expressed as the liver-to-plasma concentration ratio, of Dauno, VP16 and MTZ in Oatp1b2(−/−) mice were reduced (1.2-2.4 fold) compared to wild-type mice, and this phenomenon was accompanied by increases (1.3-1.5 fold) in plasma area under the concentration-time curve, which was significant for Dauno, VP16, and MTZ ([Figure 6](#F6), [Supplementary Figure S9](#SD10), [Supplementary Table S7](#SD17)). These phenotypes are consistent with those reported for other chemotherapeutic agents, including doxorubicin, methotrexate, and taxanes in similar *in vivo* models.^24-26^ Plasma concentrations of Ara-C were not affected by Oatp1b2 deficiency, which may reflect a differential contribution of this transporter in elimination of Ara-C between different species.

### Figure 6. Influence of Oatp1b2 deficiency on liver accumulation and the plasma pharmacokinetic profile of AML-directed therapy and liver accumulation.

![Figure 6](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c49b/4898266/91011d6223ac/nihms-744746-f0006.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4898266_nihms-744746-f0006.jpg)

Wild-type (WT) or Oatp1b2(−/−) mice (N = 3 - 4 per group) were given a single i.v. dose of cytarabine (Ara-C; 10 mg/kg), daunorubicin (Dauno; 5 mg/kg), etoposide (VP16; 5 mg/kg) or mitoxantrone (MTZ; 5 mg/kg). Results represent the mean ± s.d. liver accumulation corrected for plasma in the top panels (a-d) or plasma area under the concentration-time curve (AUC) in the bottom panels (e-h), and are expressed as % WT. To prevent in vivo deamination of Ara-C, mice were pre-treated with tetrahydrouridine (THU; 25 mg/kg; i.p.) 1 h before Ara-C administration. The corresponding concentration-time profiles and kinetic parameter estimates are provided in Supplementary Figure S6 and Supplemental Table S6, respectively. *, P < 0.03; **, P < 0.008; *** P < 0.002, two-tailed t-test.

## Discussion

We evaluated the association of a broad range of ADME gene variants and pediatric AML clinical outcomes. Using the DMET array, we identified a significant association between an inherited genetic variant in *SLCO1B1* and survival. We observed the synonymous SNP, rs2291075, to have the most significant association with both EFS and OS. We evaluated the predictive value of rs2291075 in patients categorized as low or high risk by implementing a multi-predictor model; we found high risk patients that are homozygous (T/T) for rs2291075 had favorable outcomes which were comparable to low risk patients. In linkage analysis, rs2291075 was determined to be in strong LD with two known functional variants rs4149056 and rs2306283. Gene expression of *SLCO1B1* revealed negligible expression of this transporter in primary leukemic blasts and a panel of AML cell lines suggesting that germline variation might underlie the associations we observed between *SLCO1B1* variants and outcome rather than somatic changes. *In vitro* transport assays confirmed that four AML-directed chemotherapies Ara-C, Dauno, VP16, and MTZ are substrates for OATP1B1 in a time- and concentration-dependent manner. Our *in vitro* results are consistent with previous reports demonstrating VP16 is a substrate for OATP1B1.^27^ Furthermore, expression of the functional variants rs4149056 (variant allele present in *5 and *15 haplotypes) and rs2306283 (variant allele present in *1B and *15) resulted in reduced accumulation of all AML-directed chemotherapies, with the exception of Dauno and the *1B variant which resulted in increased uptake. Finally, we found Ara-C, Dauno, VP16, and MTZ to be substrates for the mouse ortholog Oatp1b2 *in vitro*. A series of pharmacokinetic studies using a mouse deficient for the Oatp1b2 transporter revealed a decrease in liver accumulation and increase in plasma exposure of Dauno, VP16, and MTZ compared to wild-type mice. Overall, our preclinical findings support a direct contribution of *SLCO1B1* to the pharmacokinetics of AML-directed chemotherapy and indicate that patients inheriting *SLCO1B1* reduced function alleles have altered drug disposition.

Previously, rs2291075 has been reported as a tag SNP to distinguish the three major *SLCO1B1* haplotypes (*1A, *1B, and *15).^28^ Assignment of these haplotypes to our patient cohort generated survival plots similar to those based on genotype for rs2291075, supporting the role of rs2291075 as a tag SNP. We observed reduced transport function of multiple AML-chemotherapy agents *in vitro* with the OATP1B1*5 and *15 haplotypes similar to what has been reported for these haplotypes and transport of multiple substrates including 17-beta-estradiol-17-beta-d-glucuronide and pravastatin.^21, 29-31^ Collectively, these findings suggest that the presence of at least one rs4149056 C allele may result in decreased liver accumulation, lower clearance, and higher systemic exposure to multiple AML-directed chemotherapy agents, and ultimately a survival benefit. Establishment of exposure-response relationships will be critical to the further improvement of AML-directed chemotherapy, especially in populations with inherent wide inter- and intra-individual variability in pharmacokinetics.^32^ Given that some of the agents were administered daily together, future studies should include pharmacokinetic studies to evaluate the potential for genotype-dependent OATP1B1-mediated drug-drug interactions that could influence exposure-response relationships.

While genetic polymorphisms significantly contributed to the observed survival outcomes we report in this study, given that pharmacokinetics in children are affected by developmental changes with age^33^, we speculate that the ontogeny of OATP1B1 may also influence the systemic exposure of the AML-directed chemotherapy agents. Of note, hepatic mRNA expression of OATP1B1 was found to be significantly lower in all pediatric age groups compared with adults.^33^ Additionally it has recently been reported that rs2291075, among several other variants in the *SLCO1B3-SLCO1B1* genomic region, significantly associated with OATP1B1 protein expression^34^. Therefore, the disposition of AML-directed chemotherapy agents may be subject to age-related changes compounded by variant specific alterations. Better characterization of OATP1B1 hepatic mRNA and protein expression and activity in children and adolescents will be necessary in order to determine the potential influence of ontogeny in this population.

A more comprehensive understanding of the role of transporters in clinical outcomes might reveal novel interactions that could be exploited to improve response. Future studies will prospectively evaluate associations between *SLCO1B1* genotype, pharmacokinetics of AML-directed chemotherapy, and treatment outcomes to better understand the functional consequences of the *SLCO1B1* variants and further elucidate the clinical impact of common and rare variants in this transporter. Collectively, these findings signify an important role for OATP1B1 in the systemic pharmacokinetics of multiple drugs used in the treatment of AML and suggest that inherited variability in host transporter function may influence the effectiveness of therapy.

## Methods

### Patient cohort

This study was approved by the institutional review boards of all participating institutions and written informed consent was obtained from all patients or their guardians or parents (Approved Project #: BIMS20130104). The population for the current study consists of a cohort of children with *de novo* AML who were enrolled in the AML02 trial (ClinicalTrials.gov identifier: [NCT00136084](https://clinicaltrials.gov/ct2/show/NCT00136084)). Patient demographics are described in **Supplemental Table 1**. Outcome results of the AML02 study have previously been reported.^11^

### DNA isolation and genetic variant analysis using the DMET platform

Somatic and germline DNA was obtained from St. Jude Tissue Resources and was quantified using Quant-iT picogreen dsDNA kit (Invitrogen, Grand Island, NY) and normalized to 60 ng/μL for each sample. A total of 224 distinct DNA samples were interrogated by the DMET Plus array (Affymetrix, Cleveland, OH) following manufacturer's protocols, at the St. Jude Hartwell Center. Genotyping calls were generated using DMET Console 1.3 (Affymetrix). Genotypes of all single-nucleotide polymorphisms (SNPs) on the DMET array were reported either as “call” or as “no call”. All arrays had greater than 97% call rate and markers on the DMET Plus array with call rates less than 95% were excluded from analysis. All arrays were assessed for quality control and the concordance between matched germline and somatic DNA (N = 59) was determined (**Figure 1**). We found one paired sample with > 10% discordance, all other samples (58/59, 93.3%) were found to be in complete concordance. For paired samples, genotype calls for somatic DNA were used for statistical association analysis.

### Statistical analysis of clinical data associations

We screened the association of SNPs with clinical outcome. To focus attention on SNPs with statistically stable association analysis results, we limited consideration to the SNPs with genotype distribution such that the smallest group with a classical Mendellian genotype (“AA”, “AC”, etc) had more than 10 subjects and having a minor allele frequency (MAF) > 0.01. Event-free survival (EFS) was defined as the time elapsed from the date of study enrollment to the earliest of disease resistance, relapse, death, or study withdrawal with subjects having not experienced any of those events censored at the date of last follow up. Overall survival (OS) was defined as the time elapsed from date of study enrollment to death with living subjects censored at the date of last follow up. For each SNP, we used a univariate Cox^35^ proportional hazards regression model to test the association of genotype model with EFS and OS. For our top ranked SNP, we also used the Cox^35^ regression model to evaluate whether the genotypic effect on OS and EFS differed across arms (statistical interaction test) and to evaluate the association of genotype of the top ranked SNP with OS and EFS while adjusting for previously identified prognostic factors including age (> 10 years, < 10 years), risk (low, standard, high), and treatment arm (low dose Ara-C, high dose Ara-C).^11^ To assess predictive value of our top ranking SNP, we fit a multi-predictor model as previously reported^11^ to the data. Molecular and prognostic variables included in this model included core-binding factor abnormality t(8;21) or inv(16), presence of MLL-rearrangement other than t(9;11), presence of FLT3-ITD, M7 FAB type, presence of detectable minimal residual disease on day 22, and age (dichotomized as < 10 years or > 10 years). Statistical analyses were performed using R software ([www.r-project.org](http://www.r-project.org)).

Fine and Gray's model^36^ was used to test the association of each SNP's genotype with time to first toxicity, with transplant, relapse, death and any off-study events as competing event. Patients were censored at last follow-up date. Only grade 3 or higher toxicities were included for association analysis and the time to event was defined as the time to first grade 3 or higher toxicity. All grade 3 and above toxicities were categorized into 8 classes (CNS, coagulation, febrile neutropenia, GI, hypo-chemistry, infectious, pulmonary and other).

### RNA expression analysis

RNA was isolated from primary blast samples, as previously described^22^, and from cell lines using Trizol (Invitrogen). Gene expression was determined using Human Genome U133A Array for primary samples and downloaded from [http://www.stjuderesearch.org/site/data/AML1/](http://www.stjuderesearch.org/site/data/AML1/), gene expression for the cells lines was determined using Human Genome U133 Plus 2.0 Array (Affymetrix) in accordance with the manufacture's protocol. All gene expression microarrays were performed by the St. Jude Children's Research Hospital Hartwell Center for Bioinformatics and Biotechnology. Gene expression data were MAS5^37^ processed using the Affy^38^ Bioconductor^39^ R-project package. Overlapping probesets between the HG-U133A and HGU133 Plus 2.0 arrays were selected, the data combined, log2 transformed and then quantile normalized using the preprocess Core R package.^40^ The heatmap represents log2 transformed unscaled uncentered data (e.g., non-row and non-column scaled).

### In vitro transport studies

Ara-C, daunorubicin (Dauno), etoposide (VP-16), and mitoxantrone (MTZ) were obtained from Sigma-Aldrich (St. Louis, MO). [^3^H]Ara-C, [^3^H]VP16, and [^3^H]MTZ were obtained from Moravek Biochemicals (Brea, CA); and [^3^H]Dauno was obtained from PerkinElmer (Akron, OH). The uptake of [^3^H]Ara-C, [^3^H]Dauno, [^3^H]VP16, and [^3^H]MTZ were evaluated in HEK293 cells stably transfected containing control vector (VC), inducible *SLCO1B1* coding variants *1A, *1B, *5, and *15, previously described^16^, or the mouse ortholog transporter Oatp1b2, previously described.^41^ The results from the *in vitro* transport studies were normalized to total protein content as measured by a Pierce BCA Protein Assay Kit (Thermo Scientific, Rockford, IL) and normalized to uptake values in cells transfected with VC or wild-type (*1A) and reported as % control.

### In vivo pharmacokinetic studies

Adult male Oatp1b2 knockout [Oatp1b2(−/−)] mice and age-matched wild-type mice, both on a DBA/1LacJ background, were bred in-house.^42^ Mice were housed in a temperature-controlled environment with 12-hour light cycle and given a standard diet and water *ad libitum*. All *in vivo* studies were approved by the Institutional Animal Care and Use Committee at St. Jude Children's Research Hospital (Memphis, TN).

Ara-C and MTZ were formulated in saline (2 mg/mL and 1 mg/mL, respectively). Injectable Dauno (Teva, Irvine, CA) and VP16 (Teva) were obtained from the St. Jude Children's Research Hospital Pharmacy and diluted in saline (0.2 mg/mL and 0.4 mg/mL, respectively). Single doses of Ara-C (10 mg/kg), Dauno (5 mg/kg), VP16 (5 mg/kg), and MTZ (5 mg/kg) were administered to Oatp1b2(−/−) and age-matched wild-type mice intravenously via tail vein injection. Dose selection was based on previous reports indicating a similar systemic exposure to those observed in humans.^43,44,45, 46^ To prevent *in vivo* deamination of Ara-C, mice were pre-treated with tetrahydrouridine (THU; 25 mg/kg; i.p.) 1 h before Ara-C administration.^43^ Plasma from each mouse was collected at 3.5, 7.5, and 15 minutes from the submandibular vein using a lancet and at 30 and 60 minutes from the retro-orbital venous plexus using a heparinized capillary tube (Fisher Scientific). A final blood draw of 150 μL was obtained at 2 hours by a cardiac puncture and livers were harvested. All blood samples were centrifuged at 3000 x g for 5 minutes, and plasma was separated and stored at −80°C until analysis. To prevent oxidative degradation of MTZ, 1 μL of 100 mg/mL ascorbic acid (Sigma Aldrich) in 0.1 M citrate buffer (Teknova, Hollister, CA) was added to plasma samples.^45^

Plasma and liver concentrations of Ara-C, Dauno, VP16, MTZ were determined by high-performance liquid chromatography with tandem mass spectrometric detection (LC-MS/MS) using a Waters ACQUITY separation system coupled to a TQD detector and according to previously validated methods^45, 47-49^, with modifications (described below for Ara-C). Briefly, 10 μL of plasma was extracted with 60 μL acetonitrile containing internal standards. Liver was homogenized with 10 times volume of water and 20 μL homogenate was extracted with 80 μL acetonitrile containing internal standards. Samples were centrifuged as above, and 2 μL supernatant was injected for analysis. Calibrators and QCs were made using blank plasma or liver homogenate from the same mouse strains the pharmacokinetic studies were conducted in. Pharmacokinetic parameters were calculated using non-compartmental methods in WinNonlin 6.2 software (Pharsight, St. Louis, MO). All data are presented as a mean ± SE.

### Ara-C analytical method

To quantify Ara-C, a LC-MSMS method was validated based on a previously published method^47^ with the following modifications. Clofazimine (Sigma Aldrich) was chosen as internal standard. Quantitation was carried out with a Waters UPLC H-class separation system (Milford, MA,) and Acquity TQD triple-quadruple system (Beverly, MA). Separation was achieved on an ACQUITY UPLC HSS T3 Column (1.7 μm, 50 × 2.1 mm) using a column heater operating at 35°C with an ACQUITY in-line filter. The gradient mobile phase was composed of 0.1% formic acid in H2O (A)-0.1% formic acid in acetonitrile (B). The flow rate was 0.6 ml/min. B increased from 5 to 95% in 1 min and kept at 95% for 0.9 min and was back to 5% at 1.91 min. The separation was completed within 3 min including equilibration. The instrument was equipped with an electrospray interface, and was controlled by Masslynx 4.1 software (Waters, MA). The analysis was performed in MRM mode: m/z 244.2>111.9 for Ara-C; m/z 473.3>430.9 for internal standard. The MS/MS conditions were as follows: capillary voltage: 1.5kV; cone voltage: 20 v; source temperature: 150°C; desolvation temperature: 400 °C; desolvation gas flow: 850 l/h; and collision energy: 10 for Ara-C and 38 for the internal standard.

### Statistical analysis

Statistical analyses for *in vitro* experiments were based on a two-tailed *t*-test (Graphpad Prism v5.0, La Jolla, CA), and *P* < 0.05 was considered statistically significant.

## Supplementary Material

What is the current knowledge on the topic?

Genetic variation in DMET genes influence the effectiveness and toxicity profile of multiple therapeutics. Limited candidate gene approaches have identified variants involved in cytarabine drug metabolism that associated with survival outcome in acute myeloid leukemia (AML).

What question did this study address?

This pharmacogenomics study evaluated the association of single nucleotide polymorphisms (SNPs) in DMET genes on survival outcomes of patients with AML.

What this study adds to our knowledge

AML patients harboring two copies of the rs2291075 variant have a significantly more favorable survival outcome compared to individuals that are heterozygous or non-carriers. The hepatic transporter OATP1B1 has an important role in the system pharmacokinetics of multiple drugs used in the treatment of AML.

How this might change clinical pharmacology and therapeutics

These data add to the increasing body of evidence that inherited variability in host transporter function influences effectiveness of therapy and there is an urgent need to establish exposure-response relationships, especially in patients with inherent wide inter- and intra-individual variability to the further improve AML-directed chemotherapy.

What is the current knowledge on the topic?

Genetic variation in DMET genes influence the effectiveness and toxicity profile of multiple therapeutics. Limited candidate gene approaches have identified variants involved in cytarabine drug metabolism that associated with survival outcome in acute myeloid leukemia (AML).

What question did this study address?

This pharmacogenomics study evaluated the association of single nucleotide polymorphisms (SNPs) in DMET genes on survival outcomes of patients with AML.

What this study adds to our knowledge

AML patients harboring two copies of the rs2291075 variant have a significantly more favorable survival outcome compared to individuals that are heterozygous or non-carriers. The hepatic transporter OATP1B1 has an important role in the system pharmacokinetics of multiple drugs used in the treatment of AML.

How this might change clinical pharmacology and therapeutics

These data add to the increasing body of evidence that inherited variability in host transporter function influences effectiveness of therapy and there is an urgent need to establish exposure-response relationships, especially in patients with inherent wide inter- and intra-individual variability to the further improve AML-directed chemotherapy.

## Acknowledgments

We thank Richard Kim and Jeffrey Stock for providing the Oatp1b2(−/−) mice. This study was supported by R01 CA138744 (to S.D.B.), R01 CA151633 (to A.S.), the National Institutes of Health Cancer Center Support Grant P30 CA021765, and by the American Lebanese Syrian Associated Charities (ALSAC).

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Ribeiro RC. Advances in treatment of de-novo pediatric acute myeloid leukemia. Curr Opin Oncol. 2014;26:656–62. doi: 10.1097/CCO.0000000000000136.  [DOI](https://doi.org/10.1097/CCO.0000000000000136) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25226383/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Opin%20Oncol&title=Advances%20in%20treatment%20of%20de-novo%20pediatric%20acute%20myeloid%20leukemia.&author=RC%20Ribeiro&volume=26&publication_year=2014&pages=656-62&pmid=25226383&doi=10.1097/CCO.0000000000000136&)

2. Galmarini CM, et al. Potential mechanisms of resistance to cytarabine in AML patients. Leuk Res. 2002;26:621–9. doi: 10.1016/s0145-2126(01)00184-9.  [DOI](https://doi.org/10.1016/s0145-2126(01)00184-9) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12008078/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Leuk%20Res&title=Potential%20mechanisms%20of%20resistance%20to%20cytarabine%20in%20AML%20patients.&author=CM%20Galmarini&volume=26&publication_year=2002&pages=621-9&pmid=12008078&doi=10.1016/s0145-2126(01)00184-9&)

3. Lamba JK. Genetic factors influencing cytarabine therapy. Pharmacogenomics. 2009;10:1657–74. doi: 10.2217/pgs.09.118.  [DOI](https://doi.org/10.2217/pgs.09.118) | [PMC free article](/articles/PMC2828057/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19842938/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Genetic%20factors%20influencing%20cytarabine%20therapy.&author=JK%20Lamba&volume=10&publication_year=2009&pages=1657-74&pmid=19842938&doi=10.2217/pgs.09.118&)

4. Mahlknecht U, et al. SNP analyses in cytarabine metabolizing enzymes in AML patients and their impact on treatment response and patient survival: identification of CDA SNP C-451T as an independent prognostic parameter for survival. Leukemia. 2009;23:1929–32. doi: 10.1038/leu.2009.113.  [DOI](https://doi.org/10.1038/leu.2009.113) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19458626/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Leukemia&title=SNP%20analyses%20in%20cytarabine%20metabolizing%20enzymes%20in%20AML%20patients%20and%20their%20impact%20on%20treatment%20response%20and%20patient%20survival:%20identification%20of%20CDA%20SNP%20C-451T%20as%20an%20independent%20prognostic%20parameter%20for%20survival.&author=U%20Mahlknecht&volume=23&publication_year=2009&pages=1929-32&pmid=19458626&doi=10.1038/leu.2009.113&)

5. Mitra AK, et al. Genetic variants in cytosolic 5′-nucleotidase II are associated with its expression and cytarabine sensitivity in HapMap cell lines and in patients with acute myeloid leukemia. J Pharmacol Exp Ther. 2011;339:9–23. doi: 10.1124/jpet.111.182873.  [DOI](https://doi.org/10.1124/jpet.111.182873) | [PMC free article](/articles/PMC3186292/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21712425/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharmacol%20Exp%20Ther&title=Genetic%20variants%20in%20cytosolic%205%E2%80%B2-nucleotidase%20II%20are%20associated%20with%20its%20expression%20and%20cytarabine%20sensitivity%20in%20HapMap%20cell%20lines%20and%20in%20patients%20with%20acute%20myeloid%20leukemia.&author=AK%20Mitra&volume=339&publication_year=2011&pages=9-23&pmid=21712425&doi=10.1124/jpet.111.182873&)

6. Shi JY, et al. Association between single nucleotide polymorphisms in deoxycytidine kinase and treatment response among acute myeloid leukaemia patients. Pharmacogenetics. 2004;14:759–68. doi: 10.1097/00008571-200411000-00007.  [DOI](https://doi.org/10.1097/00008571-200411000-00007) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15564883/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=Association%20between%20single%20nucleotide%20polymorphisms%20in%20deoxycytidine%20kinase%20and%20treatment%20response%20among%20acute%20myeloid%20leukaemia%20patients.&author=JY%20Shi&volume=14&publication_year=2004&pages=759-68&pmid=15564883&doi=10.1097/00008571-200411000-00007&)

7. Crump C, et al. Glutathione S-transferase theta 1 gene deletion and risk of acute myeloid leukemia. Cancer Epidemiol Biomarkers Prev. 2000;9:457–60.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/10815689/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Epidemiol%20Biomarkers%20Prev&title=Glutathione%20S-transferase%20theta%201%20gene%20deletion%20and%20risk%20of%20acute%20myeloid%20leukemia.&author=C%20Crump&volume=9&publication_year=2000&pages=457-60&pmid=10815689&)

8. Seedhouse CH, et al. Sequential influences of leukemia-specific and genetic factors on p-glycoprotein expression in blasts from 817 patients entered into the National Cancer Research Network acute myeloid leukemia 14 and 15 trials. Clin Cancer Res. 2007;13:7059–66. doi: 10.1158/1078-0432.CCR-07-1484.  [DOI](https://doi.org/10.1158/1078-0432.CCR-07-1484) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18056183/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&title=Sequential%20influences%20of%20leukemia-specific%20and%20genetic%20factors%20on%20p-glycoprotein%20expression%20in%20blasts%20from%20817%20patients%20entered%20into%20the%20National%20Cancer%20Research%20Network%20acute%20myeloid%20leukemia%2014%20and%2015%20trials.&author=CH%20Seedhouse&volume=13&publication_year=2007&pages=7059-66&pmid=18056183&doi=10.1158/1078-0432.CCR-07-1484&)

9. Yee SW, et al. Impact of polymorphisms in drug pathway genes on disease-free survival in adults with acute myeloid leukemia. J Hum Genet. 2013;58:353–61. doi: 10.1038/jhg.2013.38.  [DOI](https://doi.org/10.1038/jhg.2013.38) | [PMC free article](/articles/PMC4068832/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23677058/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Hum%20Genet&title=Impact%20of%20polymorphisms%20in%20drug%20pathway%20genes%20on%20disease-free%20survival%20in%20adults%20with%20acute%20myeloid%20leukemia.&author=SW%20Yee&volume=58&publication_year=2013&pages=353-61&pmid=23677058&doi=10.1038/jhg.2013.38&)

10. Steinbach D, Legrand O. ABC transporters and drug resistance in leukemia: was P-gp nothing but the first head of the Hydra? Leukemia. 2007;21:1172–6. doi: 10.1038/sj.leu.2404692.  [DOI](https://doi.org/10.1038/sj.leu.2404692) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17429427/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Leukemia&title=ABC%20transporters%20and%20drug%20resistance%20in%20leukemia:%20was%20P-gp%20nothing%20but%20the%20first%20head%20of%20the%20Hydra?&author=D%20Steinbach&author=O%20Legrand&volume=21&publication_year=2007&pages=1172-6&pmid=17429427&doi=10.1038/sj.leu.2404692&)

11. Rubnitz JE, et al. Minimal residual disease-directed therapy for childhood acute myeloid leukaemia: results of the AML02 multicentre trial. Lancet Oncol. 2010;11:543–52. doi: 10.1016/S1470-2045(10)70090-5.  [DOI](https://doi.org/10.1016/S1470-2045(10)70090-5) | [PMC free article](/articles/PMC3171799/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20451454/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet%20Oncol&title=Minimal%20residual%20disease-directed%20therapy%20for%20childhood%20acute%20myeloid%20leukaemia:%20results%20of%20the%20AML02%20multicentre%20trial.&author=JE%20Rubnitz&volume=11&publication_year=2010&pages=543-52&pmid=20451454&doi=10.1016/S1470-2045(10)70090-5&)

12. Weiner M, et al. Effects of tuberculosis, race, and human gene SLCO1B1 polymorphisms on rifampin concentrations. Antimicrob Agents Chemother. 2010;54:4192–200. doi: 10.1128/AAC.00353-10.  [DOI](https://doi.org/10.1128/AAC.00353-10) | [PMC free article](/articles/PMC2944564/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20660695/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Antimicrob%20Agents%20Chemother&title=Effects%20of%20tuberculosis,%20race,%20and%20human%20gene%20SLCO1B1%20polymorphisms%20on%20rifampin%20concentrations.&author=M%20Weiner&volume=54&publication_year=2010&pages=4192-200&pmid=20660695&doi=10.1128/AAC.00353-10&)

13. Johnson AD, Handsaker RE, Pulit SL, Nizzari MM, O'Donnell CJ, de Bakker PI. SNAP: a web-based tool for identification and annotation of proxy SNPs using HapMap. Bioinformatics. 2008;24:2938–9. doi: 10.1093/bioinformatics/btn564.  [DOI](https://doi.org/10.1093/bioinformatics/btn564) | [PMC free article](/articles/PMC2720775/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18974171/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=SNAP:%20a%20web-based%20tool%20for%20identification%20and%20annotation%20of%20proxy%20SNPs%20using%20HapMap.&author=AD%20Johnson&author=RE%20Handsaker&author=SL%20Pulit&author=MM%20Nizzari&author=CJ%20O'Donnell&volume=24&publication_year=2008&pages=2938-9&pmid=18974171&doi=10.1093/bioinformatics/btn564&)

14. The International HapMap Project Nature. 2003;426:789–96. doi: 10.1038/nature02168.  [DOI](https://doi.org/10.1038/nature02168) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14685227/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&volume=426&publication_year=2003&pages=789-96&pmid=14685227&doi=10.1038/nature02168&)

15. Barrett JC, Fry B, Maller J, Daly MJ. Haploview: analysis and visualization of LD and haplotype maps. Bioinformatics. 2005;21:263–5. doi: 10.1093/bioinformatics/bth457.  [DOI](https://doi.org/10.1093/bioinformatics/bth457) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15297300/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=Haploview:%20analysis%20and%20visualization%20of%20LD%20and%20haplotype%20maps.&author=JC%20Barrett&author=B%20Fry&author=J%20Maller&author=MJ%20Daly&volume=21&publication_year=2005&pages=263-5&pmid=15297300&doi=10.1093/bioinformatics/bth457&)

16. Ramsey LB, et al. Rare versus common variants in pharmacogenetics: SLCO1B1 variation and methotrexate disposition. Genome Res. 2012;22:1–8. doi: 10.1101/gr.129668.111.  [DOI](https://doi.org/10.1101/gr.129668.111) | [PMC free article](/articles/PMC3246196/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22147369/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Res&title=Rare%20versus%20common%20variants%20in%20pharmacogenetics:%20SLCO1B1%20variation%20and%20methotrexate%20disposition.&author=LB%20Ramsey&volume=22&publication_year=2012&pages=1-8&pmid=22147369&doi=10.1101/gr.129668.111&)

17. Link E, et al. SLCO1B1 variants and statin-induced myopathy--a genomewide study. N Engl J Med. 2008;359:789–99. doi: 10.1056/NEJMoa0801936.  [DOI](https://doi.org/10.1056/NEJMoa0801936) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18650507/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=SLCO1B1%20variants%20and%20statin-induced%20myopathy--a%20genomewide%20study.&author=E%20Link&volume=359&publication_year=2008&pages=789-99&pmid=18650507&doi=10.1056/NEJMoa0801936&)

18. Ramsey LB, et al. Genome-wide study of methotrexate clearance replicates SLCO1B1. Blood. 2013;121:898–904. doi: 10.1182/blood-2012-08-452839.  [DOI](https://doi.org/10.1182/blood-2012-08-452839) | [PMC free article](/articles/PMC3567337/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23233662/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Genome-wide%20study%20of%20methotrexate%20clearance%20replicates%20SLCO1B1.&author=LB%20Ramsey&volume=121&publication_year=2013&pages=898-904&pmid=23233662&doi=10.1182/blood-2012-08-452839&)

19. Trevino LR, et al. Germline genetic variation in an organic anion transporter polypeptide associated with methotrexate pharmacokinetics and clinical effects. J Clin Oncol. 2009;27:5972–8. doi: 10.1200/JCO.2008.20.4156.  [DOI](https://doi.org/10.1200/JCO.2008.20.4156) | [PMC free article](/articles/PMC2793040/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19901119/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Germline%20genetic%20variation%20in%20an%20organic%20anion%20transporter%20polypeptide%20associated%20with%20methotrexate%20pharmacokinetics%20and%20clinical%20effects.&author=LR%20Trevino&volume=27&publication_year=2009&pages=5972-8&pmid=19901119&doi=10.1200/JCO.2008.20.4156&)

20. Niemi M, Pasanen MK, Neuvonen PJ. Organic anion transporting polypeptide 1B1: a genetically polymorphic transporter of major importance for hepatic drug uptake. Pharmacol Rev. 2011;63:157–81. doi: 10.1124/pr.110.002857.  [DOI](https://doi.org/10.1124/pr.110.002857) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21245207/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacol%20Rev&title=Organic%20anion%20transporting%20polypeptide%201B1:%20a%20genetically%20polymorphic%20transporter%20of%20major%20importance%20for%20hepatic%20drug%20uptake.&author=M%20Niemi&author=MK%20Pasanen&author=PJ%20Neuvonen&volume=63&publication_year=2011&pages=157-81&pmid=21245207&doi=10.1124/pr.110.002857&)

21. Nozawa T, et al. Genetic polymorphisms of human organic anion transporters OATP-C (SLC21A6) and OATP-B (SLC21A9): allele frequencies in the Japanese population and functional analysis. J Pharmacol Exp Ther. 2002;302:804–13. doi: 10.1124/jpet.302.2.804.  [DOI](https://doi.org/10.1124/jpet.302.2.804) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12130747/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharmacol%20Exp%20Ther&title=Genetic%20polymorphisms%20of%20human%20organic%20anion%20transporters%20OATP-C%20(SLC21A6)%20and%20OATP-B%20(SLC21A9):%20allele%20frequencies%20in%20the%20Japanese%20population%20and%20functional%20analysis.&author=T%20Nozawa&volume=302&publication_year=2002&pages=804-13&pmid=12130747&doi=10.1124/jpet.302.2.804&)

22. Ross ME, et al. Gene expression profiling of pediatric acute myelogenous leukemia. Blood. 2004;104:3679–87. doi: 10.1182/blood-2004-03-1154.  [DOI](https://doi.org/10.1182/blood-2004-03-1154) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15226186/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Gene%20expression%20profiling%20of%20pediatric%20acute%20myelogenous%20leukemia.&author=ME%20Ross&volume=104&publication_year=2004&pages=3679-87&pmid=15226186&doi=10.1182/blood-2004-03-1154&)

23. Hsiang B, et al. A novel human hepatic organic anion transporting polypeptide (OATP2). Identification of a liver-specific human organic anion transporting polypeptide and identification of rat and human hydroxymethylglutaryl-CoA reductase inhibitor transporters. J Biol Chem. 1999;274:37161–8. doi: 10.1074/jbc.274.52.37161.  [DOI](https://doi.org/10.1074/jbc.274.52.37161) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10601278/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=A%20novel%20human%20hepatic%20organic%20anion%20transporting%20polypeptide%20(OATP2).%20Identification%20of%20a%20liver-specific%20human%20organic%20anion%20transporting%20polypeptide%20and%20identification%20of%20rat%20and%20human%20hydroxymethylglutaryl-CoA%20reductase%20inhibitor%20transporters.&author=B%20Hsiang&volume=274&publication_year=1999&pages=37161-8&pmid=10601278&doi=10.1074/jbc.274.52.37161&)

24. Durmus S, Naik J, Buil L, Wagenaar E, van Tellingen O, Schinkel AH. In vivo disposition of doxorubicin is affected by mouse Oatp1a/1b and human OATP1A/1B transporters. Int J Cancer. 2014;135:1700–10. doi: 10.1002/ijc.28797.  [DOI](https://doi.org/10.1002/ijc.28797) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24554572/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Cancer&title=In%20vivo%20disposition%20of%20doxorubicin%20is%20affected%20by%20mouse%20Oatp1a/1b%20and%20human%20OATP1A/1B%20transporters.&author=S%20Durmus&author=J%20Naik&author=L%20Buil&author=E%20Wagenaar&author=O%20van%20Tellingen&volume=135&publication_year=2014&pages=1700-10&pmid=24554572&doi=10.1002/ijc.28797&)

25. Iusuf D, et al. Human OATP1B1, OATP1B3 and OATP1A2 can mediate the in vivo uptake and clearance of docetaxel. Int J Cancer. 2015;136:225–33. doi: 10.1002/ijc.28970.  [DOI](https://doi.org/10.1002/ijc.28970) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24825069/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Cancer&title=Human%20OATP1B1,%20OATP1B3%20and%20OATP1A2%20can%20mediate%20the%20in%20vivo%20uptake%20and%20clearance%20of%20docetaxel.&author=D%20Iusuf&volume=136&publication_year=2015&pages=225-33&pmid=24825069&doi=10.1002/ijc.28970&)

26. van de Steeg E, van Esch A, Wagenaar E, Kenworthy KE, Schinkel AH. Influence of human OATP1B1, OATP1B3, and OATP1A2 on the pharmacokinetics of methotrexate and paclitaxel in humanized transgenic mice. Clin Cancer Res. 2013;19:821–32. doi: 10.1158/1078-0432.CCR-12-2080.  [DOI](https://doi.org/10.1158/1078-0432.CCR-12-2080) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23243220/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&title=Influence%20of%20human%20OATP1B1,%20OATP1B3,%20and%20OATP1A2%20on%20the%20pharmacokinetics%20of%20methotrexate%20and%20paclitaxel%20in%20humanized%20transgenic%20mice.&author=E%20van%20de%20Steeg&author=A%20van%20Esch&author=E%20Wagenaar&author=KE%20Kenworthy&author=AH%20Schinkel&volume=19&publication_year=2013&pages=821-32&pmid=23243220&doi=10.1158/1078-0432.CCR-12-2080&)

27. Fahrmayr C, Konig J, Auge D, Mieth M, Fromm MF. Identification of drugs and drug metabolites as substrates of multidrug resistance protein 2 (MRP2) using triple-transfected MDCK-OATP1B1-UGT1A1-MRP2 cells. Br J Pharmacol. 2012;165:1836–47. doi: 10.1111/j.1476-5381.2011.01672.x.  [DOI](https://doi.org/10.1111/j.1476-5381.2011.01672.x) | [PMC free article](/articles/PMC3372834/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21923755/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Pharmacol&title=Identification%20of%20drugs%20and%20drug%20metabolites%20as%20substrates%20of%20multidrug%20resistance%20protein%202%20(MRP2)%20using%20triple-transfected%20MDCK-OATP1B1-UGT1A1-MRP2%20cells.&author=C%20Fahrmayr&author=J%20Konig&author=D%20Auge&author=M%20Mieth&author=MF%20Fromm&volume=165&publication_year=2012&pages=1836-47&pmid=21923755&doi=10.1111/j.1476-5381.2011.01672.x&)

28. Jada SR, et al. Pharmacogenetics of SLCO1B1: haplotypes, htSNPs and hepatic expression in three distinct Asian populations. Eur J Clin Pharmacol. 2007;63:555–63. doi: 10.1007/s00228-007-0285-5.  [DOI](https://doi.org/10.1007/s00228-007-0285-5) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17415554/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Clin%20Pharmacol&title=Pharmacogenetics%20of%20SLCO1B1:%20haplotypes,%20htSNPs%20and%20hepatic%20expression%20in%20three%20distinct%20Asian%20populations.&author=SR%20Jada&volume=63&publication_year=2007&pages=555-63&pmid=17415554&doi=10.1007/s00228-007-0285-5&)

29. Iwai M, Suzuki H, Ieiri I, Otsubo K, Sugiyama Y. Functional analysis of single nucleotide polymorphisms of hepatic organic anion transporter OATP1B1 (OATP-C). Pharmacogenetics. 2004;14:749–57. doi: 10.1097/00008571-200411000-00006.  [DOI](https://doi.org/10.1097/00008571-200411000-00006) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15564882/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=Functional%20analysis%20of%20single%20nucleotide%20polymorphisms%20of%20hepatic%20organic%20anion%20transporter%20OATP1B1%20(OATP-C).&author=M%20Iwai&author=H%20Suzuki&author=I%20Ieiri&author=K%20Otsubo&author=Y%20Sugiyama&volume=14&publication_year=2004&pages=749-57&pmid=15564882&doi=10.1097/00008571-200411000-00006&)

30. Nishizato Y, et al. Polymorphisms of OATP-C (SLC21A6) and OAT3 (SLC22A8) genes: consequences for pravastatin pharmacokinetics. Clin Pharmacol Ther. 2003;73:554–65. doi: 10.1016/S0009-9236(03)00060-2.  [DOI](https://doi.org/10.1016/S0009-9236(03)00060-2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12811365/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Polymorphisms%20of%20OATP-C%20(SLC21A6)%20and%20OAT3%20(SLC22A8)%20genes:%20consequences%20for%20pravastatin%20pharmacokinetics.&author=Y%20Nishizato&volume=73&publication_year=2003&pages=554-65&pmid=12811365&doi=10.1016/S0009-9236(03)00060-2&)

31. Tirona RG, Leake BF, Merino G, Kim RB. Polymorphisms in OATP-C: identification of multiple allelic variants associated with altered transport activity among European- and African-Americans. J Biol Chem. 2001;276:35669–75. doi: 10.1074/jbc.M103792200.  [DOI](https://doi.org/10.1074/jbc.M103792200) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11477075/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=Polymorphisms%20in%20OATP-C:%20identification%20of%20multiple%20allelic%20variants%20associated%20with%20altered%20transport%20activity%20among%20European-%20and%20African-Americans.&author=RG%20Tirona&author=BF%20Leake&author=G%20Merino&author=RB%20Kim&volume=276&publication_year=2001&pages=35669-75&pmid=11477075&doi=10.1074/jbc.M103792200&)

32. Lesko LJ, Rowland M, Peck CC, Blaschke TF. Optimizing the science of drug development: opportunities for better candidate selection and accelerated evaluation in humans. Pharm Res. 2000;17:1335–44. doi: 10.1023/a:1007574217260.  [DOI](https://doi.org/10.1023/a:1007574217260) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11205725/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharm%20Res&title=Optimizing%20the%20science%20of%20drug%20development:%20opportunities%20for%20better%20candidate%20selection%20and%20accelerated%20evaluation%20in%20humans.&author=LJ%20Lesko&author=M%20Rowland&author=CC%20Peck&author=TF%20Blaschke&volume=17&publication_year=2000&pages=1335-44&pmid=11205725&doi=10.1023/a:1007574217260&)

33. Brouwer K, et al. Human Ontogeny of Drug Transporters: Review and Recommendations of the Pediatric Transporter Working Group. Clin Pharmacol Ther. 2015;98:266–87. doi: 10.1002/cpt.176.  [DOI](https://doi.org/10.1002/cpt.176) | [PMC free article](/articles/PMC4731327/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26088472/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Human%20Ontogeny%20of%20Drug%20Transporters:%20Review%20and%20Recommendations%20of%20the%20Pediatric%20Transporter%20Working%20Group.&author=K%20Brouwer&volume=98&publication_year=2015&pages=266-87&pmid=26088472&doi=10.1002/cpt.176&)

34. Nies AT, et al. Genetics is a major determinant of expression of the human hepatic uptake transporter OATP1B1, but not of OATP1B3 and OATP2B1. Genome Med. 2013;5:1. doi: 10.1186/gm405.  [DOI](https://doi.org/10.1186/gm405) | [PMC free article](/articles/PMC3706890/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23311897/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Med&title=Genetics%20is%20a%20major%20determinant%20of%20expression%20of%20the%20human%20hepatic%20uptake%20transporter%20OATP1B1,%20but%20not%20of%20OATP1B3%20and%20OATP2B1.&author=AT%20Nies&volume=5&publication_year=2013&pages=1&pmid=23311897&doi=10.1186/gm405&)

35. Cox DR. Regression models and life tables. Journal of the Royal Statistical Society Series B. 1972;34:187–220.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20of%20the%20Royal%20Statistical%20Society%20Series%20B&title=Regression%20models%20and%20life%20tables.&author=DR%20Cox&volume=34&publication_year=1972&pages=187-220&)

36. Fine JP GR. A proportional hazards model for the subdistribution of a competing risk. J Am Stat Assoc. 1999;94:496–509.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Am%20Stat%20Assoc&title=A%20proportional%20hazards%20model%20for%20the%20subdistribution%20of%20a%20competing%20risk.&author=GR%20Fine%20JP&volume=94&publication_year=1999&pages=496-509&)

37. Hubbell E, Liu WM, Mei R. Robust estimators for expression analysis. Bioinformatics. 2002;18:1585–92. doi: 10.1093/bioinformatics/18.12.1585.  [DOI](https://doi.org/10.1093/bioinformatics/18.12.1585) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12490442/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=Robust%20estimators%20for%20expression%20analysis.&author=E%20Hubbell&author=WM%20Liu&author=R%20Mei&volume=18&publication_year=2002&pages=1585-92&pmid=12490442&doi=10.1093/bioinformatics/18.12.1585&)

38. Gautier L, Cope L, Bolstad BM, Irizarry RA. affy--analysis of Affymetrix GeneChip data at the probe level. Bioinformatics. 2004;20:307–15. doi: 10.1093/bioinformatics/btg405.  [DOI](https://doi.org/10.1093/bioinformatics/btg405) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14960456/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=affy--analysis%20of%20Affymetrix%20GeneChip%20data%20at%20the%20probe%20level.&author=L%20Gautier&author=L%20Cope&author=BM%20Bolstad&author=RA%20Irizarry&volume=20&publication_year=2004&pages=307-15&pmid=14960456&doi=10.1093/bioinformatics/btg405&)

39. Gentleman RC, et al. Bioconductor: open software development for computational biology and bioinformatics. Genome Biol. 2004;5:R80. doi: 10.1186/gb-2004-5-10-r80.  [DOI](https://doi.org/10.1186/gb-2004-5-10-r80) | [PMC free article](/articles/PMC545600/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15461798/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Biol&title=Bioconductor:%20open%20software%20development%20for%20computational%20biology%20and%20bioinformatics.&author=RC%20Gentleman&volume=5&publication_year=2004&pages=R80&pmid=15461798&doi=10.1186/gb-2004-5-10-r80&)

40. Bolstad BM, Irizarry RA, Astrand M, Speed TP. A comparison of normalization methods for high density oligonucleotide array data based on variance and bias. Bioinformatics. 2003;19:185–93. doi: 10.1093/bioinformatics/19.2.185.  [DOI](https://doi.org/10.1093/bioinformatics/19.2.185) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12538238/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=A%20comparison%20of%20normalization%20methods%20for%20high%20density%20oligonucleotide%20array%20data%20based%20on%20variance%20and%20bias.&author=BM%20Bolstad&author=RA%20Irizarry&author=M%20Astrand&author=TP%20Speed&volume=19&publication_year=2003&pages=185-93&pmid=12538238&doi=10.1093/bioinformatics/19.2.185&)

41. de Graan AJ, et al. Influence of polymorphic OATP1B-type carriers on the disposition of docetaxel. Clin Cancer Res. 2012;18:4433–40. doi: 10.1158/1078-0432.CCR-12-0761.  [DOI](https://doi.org/10.1158/1078-0432.CCR-12-0761) | [PMC free article](/articles/PMC3464009/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22711709/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&title=Influence%20of%20polymorphic%20OATP1B-type%20carriers%20on%20the%20disposition%20of%20docetaxel.&author=AJ%20de%20Graan&volume=18&publication_year=2012&pages=4433-40&pmid=22711709&doi=10.1158/1078-0432.CCR-12-0761&)

42. Zaher H, et al. Targeted disruption of murine organic anion-transporting polypeptide 1b2 (Oatp1b2/Slco1b2) significantly alters disposition of prototypical drug substrates pravastatin and rifampin. Mol Pharmacol. 2008;74:320–9. doi: 10.1124/mol.108.046458.  [DOI](https://doi.org/10.1124/mol.108.046458) | [PMC free article](/articles/PMC2562886/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18413659/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Pharmacol&title=Targeted%20disruption%20of%20murine%20organic%20anion-transporting%20polypeptide%201b2%20(Oatp1b2/Slco1b2)%20significantly%20alters%20disposition%20of%20prototypical%20drug%20substrates%20pravastatin%20and%20rifampin.&author=H%20Zaher&volume=74&publication_year=2008&pages=320-9&pmid=18413659&doi=10.1124/mol.108.046458&)

43. Avramis VI, Powell W. Pharmacology of combination chemotherapy of cytosine arabinoside (ara-C) and uracil arabinoside (ara-U) or tetrahydrouridine (THU) against murine leukemia L1210/0 in tumor-bearing mice. Cancer Invest. 1987;5:293–9.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/3664332/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Invest&title=Pharmacology%20of%20combination%20chemotherapy%20of%20cytosine%20arabinoside%20(ara-C)%20and%20uracil%20arabinoside%20(ara-U)%20or%20tetrahydrouridine%20(THU)%20against%20murine%20leukemia%20L1210/0%20in%20tumor-bearing%20mice.&author=VI%20Avramis&author=W%20Powell&volume=5&publication_year=1987&pages=293-9&pmid=3664332&)

44. Galettis P, Boutagy J, Ma DD. Daunorubicin pharmacokinetics and the correlation with P-glycoprotein and response in patients with acute leukaemia. Br J Cancer. 1994;70:324–9. doi: 10.1038/bjc.1994.301.  [DOI](https://doi.org/10.1038/bjc.1994.301) | [PMC free article](/articles/PMC2033508/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7914424/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Cancer&title=Daunorubicin%20pharmacokinetics%20and%20the%20correlation%20with%20P-glycoprotein%20and%20response%20in%20patients%20with%20acute%20leukaemia.&author=P%20Galettis&author=J%20Boutagy&author=DD%20Ma&volume=70&publication_year=1994&pages=324-9&pmid=7914424&doi=10.1038/bjc.1994.301&)

45. An G, Morris ME. A physiologically based pharmacokinetic model of mitoxantrone in mice and scale-up to humans: a semi-mechanistic model incorporating DNA and protein binding. AAPS J. 2012;14:352–64. doi: 10.1208/s12248-012-9344-7.  [DOI](https://doi.org/10.1208/s12248-012-9344-7) | [PMC free article](/articles/PMC3326165/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22451016/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=AAPS%20J&title=A%20physiologically%20based%20pharmacokinetic%20model%20of%20mitoxantrone%20in%20mice%20and%20scale-up%20to%20humans:%20a%20semi-mechanistic%20model%20incorporating%20DNA%20and%20protein%20binding.&author=G%20An&author=ME%20Morris&volume=14&publication_year=2012&pages=352-64&pmid=22451016&doi=10.1208/s12248-012-9344-7&)

46. Lagas JS, et al. P-glycoprotein (P-gp/Abcb1), Abcc2, and Abcc3 determine the pharmacokinetics of etoposide. Clin Cancer Res. 2010;16:130–40. doi: 10.1158/1078-0432.CCR-09-1321.  [DOI](https://doi.org/10.1158/1078-0432.CCR-09-1321) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20028753/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&title=P-glycoprotein%20(P-gp/Abcb1),%20Abcc2,%20and%20Abcc3%20determine%20the%20pharmacokinetics%20of%20etoposide.&author=JS%20Lagas&volume=16&publication_year=2010&pages=130-40&pmid=20028753&doi=10.1158/1078-0432.CCR-09-1321&)

47. Hsieh Y, Duncan CJ, Liu M. A mixed-mode liquid chromatography-tandem mass spectrometric method for the determination of cytarabine in mouse plasma. J Chromatogr B Analyt Technol Biomed Life Sci. 2007;854:8–12. doi: 10.1016/j.jchromb.2007.03.034.  [DOI](https://doi.org/10.1016/j.jchromb.2007.03.034) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17448737/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Chromatogr%20B%20Analyt%20Technol%20Biomed%20Life%20Sci&title=A%20mixed-mode%20liquid%20chromatography-tandem%20mass%20spectrometric%20method%20for%20the%20determination%20of%20cytarabine%20in%20mouse%20plasma.&author=Y%20Hsieh&author=CJ%20Duncan&author=M%20Liu&volume=854&publication_year=2007&pages=8-12&pmid=17448737&doi=10.1016/j.jchromb.2007.03.034&)

48. Yang Y. Development and validation of a high-performance liquid chromatography-tandem mass spectrometric method for quantification of daunorubicin in rat plasma. Talanta. 2007;71:596–604. doi: 10.1016/j.talanta.2006.05.003.  [DOI](https://doi.org/10.1016/j.talanta.2006.05.003) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19071346/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Talanta&title=Development%20and%20validation%20of%20a%20high-performance%20liquid%20chromatography-tandem%20mass%20spectrometric%20method%20for%20quantification%20of%20daunorubicin%20in%20rat%20plasma.&author=Y%20Yang&volume=71&publication_year=2007&pages=596-604&pmid=19071346&doi=10.1016/j.talanta.2006.05.003&)

49. Raza A AA, Bashir S, Ahmad B, Irfan J, Mehta JD, Gill HS, Khan A, Alam M, Schätzlein AG. Rapid and sensitive liquid chromatographic method for determination of etoposide in plasma and biological samples. Journal of Liquid Chromatography & Related Technologies. 2013;36:2796–813.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Liquid%20Chromatography%20&%20Related%20Technologies&title=Rapid%20and%20sensitive%20liquid%20chromatographic%20method%20for%20determination%20of%20etoposide%20in%20plasma%20and%20biological%20samples.&author=AA%20Raza%20A&author=S%20Bashir&author=B%20Ahmad&author=J%20Irfan&author=JD%20Mehta&volume=36&publication_year=2013&pages=2796-813&)
