# Detection of allele specific differences in IFNL3 (IL28B) mRNA expression

## Metadata
**Authors:** Susanne Knapp, Naeem Meghjee, Sorcha Cassidy, Khaleel Jamil, Mark Thursz
**Journal:** BMC Medical Genetics
**Date:** 2014 Oct 5
**DOI:** [10.1186/s12881-014-0104-7](https://doi.org/10.1186/s12881-014-0104-7)
**PMID:** 25287681
**PMCID:** PMC4411934
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4411934/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC4411934/pdf/12881_2014_Article_104.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC4411934/pdf/12881_2014_Article_104.pdf)

## Abstract

**Background:** 
Variants of the interferon-lambda3 (IFNL3) gene have been associated with both spontaneous and treatment induced clearance of HCV infection. Attempts to link polymorphisms of the IFNL3 gene with variation in the level of IFNL3 expression have been inconclusive. This is partially due to the difficulty to design assays distinguishing IFNL3 from IFNL2.

**Methods:** 
In this study an allele specific real-time PCR (RT-PCR) assay was developed which allows the relative quantification of the two IFNL3 transcripts in cells heterozygous for SNP IFNL3.rs4803217 in the 3'UTR of the IFNL3 gene. This SNP is in strong linkage disequilibrium (LD) with the predictive marker rs12979860.

**Results:** 
Raji cells showed two-fold increased levels of IFNL3.rs4803217 C-allele expression. In peripheral blood mononuclear cells (PBMCs) of eight uninfected donors, two donors showed increased IFNL3.rs4803217 C-allele expression.

**Conclusion:** 
This indicates that allele specific differences in IFNL3 expression vary between individuals and might contribute to the variety of outcomes in HCV infected patients.

**Electronic supplementary material:** 
The online version of this article (doi:10.1186/s12881-014-0104-7) contains supplementary material, which is available to authorized users.

Keywords: Variation of allele specific expression, Quantitative RT-PCR, IFNL3 gene, Hepatitis C Virus (HCV) infection, in vitro assay

### Background

Variants of the interferon-lambda3 (*IFNL3*) gene have been associated with both spontaneous and treatment induced clearance of HCV infection. Attempts to link polymorphisms of the *IFNL3* gene with variation in the level of *IFNL3* expression have been inconclusive. This is partially due to the difficulty to design assays distinguishing *IFNL3* from *IFNL2*.

### Methods

In this study an allele specific real-time PCR (RT-PCR) assay was developed which allows the relative quantification of the two *IFNL3* transcripts in cells heterozygous for SNP *IFNL3.rs4803217* in the 3'UTR of the *IFNL3* gene. This SNP is in strong linkage disequilibrium (LD) with the predictive marker *rs12979860*.

### Results

Raji cells showed two-fold increased levels of *IFNL3.rs4803217* C-allele expression. In peripheral blood mononuclear cells (PBMCs) of eight uninfected donors, two donors showed increased *IFNL3.rs4803217* C-allele expression.

### Conclusion

This indicates that allele specific differences in *IFNL3* expression vary between individuals and might contribute to the variety of outcomes in HCV infected patients.

### Electronic supplementary material

The online version of this article (doi:10.1186/s12881-014-0104-7) contains supplementary material, which is available to authorized users.

## Background

Recently genetic polymorphisms in the interferon-lambda3 gene (*IFNL3*), also known as interleukin 28B (*IL-28B*) have been shown to be highly associated with the outcome and treatment response of Hepatitis C infection in ethnically diverse cohorts [[1](#CR1)–[6](#CR6)]. Several single nucleotide polymorphisms (SNPs) are in linkage disequilibrium (LD), which makes the identification of the functional polymorphism challenging. Attempts have been made to link the effects of the polymorphism to a difference in the level of expression of *IFNL3* transcript *in vitro* and *in vivo* [[4](#CR4),[7](#CR7)] or differences in potency of the corresponding gene products [[8](#CR8)], but no consistent difference could be detected.

The challenge with designing genotyping assays for *IFNL3* is the close homology with *IFNL2 (IL28A)*, with which it shares 96% homology on the DNA level [[4](#CR4),[9](#CR9)]. Several *IFNL3* specific TaqMan assays have been designed which discriminate *IFNL3* from *IFNL2* [[8](#CR8),[10](#CR10)], but are not able to discriminate between the two alleles within the gene. In order to study the allele specific expression of *IFNL3*, we developed a SYBRGreen based RT-PCR assay which is able to quantify the two allele specific transcripts of *IFNL3* (whilst excluding a signal from *IFNL2*) based around the two expressed alleles of the *rs4803217* SNP in the 3’UTR region of *IFNL3.* In the Asian and Caucasian population *SNP rs4803217* is in close LD with *rs12979860* (r^2^ = 0.98 [[11](#CR11),[12](#CR12)])*,* which predicts outcome of HCV infection and treatment response. The relative amount of allele specific transcript was measured after interferon stimulation of Huh7, Raji and Jurkat cells, and in peripheral blood mononuclear cells (PBMCs) of eight uninfected healthy donors, which were heterozygous for *IFNL3.rs4803217*. The assay is a modification of allele specific quantitative RT-PCR, which has been developed by Germer *et al.* [[13](#CR13)] to accurately measure varying allele frequencies between 5% and 95% for SNPs in pooled DNA samples. We report that the presented *IFNL3* specific assay is able to accurately measure variation of allele specific expression between individuals.

## Methods

### Cell lines

The cell lines used in our experiments were Huh7, Raji and Jurkat cell lines. The Huh7 cell line is derived from hepatocellular carcinoma [[14](#CR14)]. The Raji cell line is a suspension cell line derived from B-lymphocytes [[15](#CR15)]. The Jurkat suspension cell line is derived from T-cells [[16](#CR16)]. Huh7, Jurkat, Raji cells were cultivated under standard conditions in DMEM medium containing 5% Pen/Strep and 10% heat inactivated Foetal Calf Serum (FCS) and incubated at 37°C under 5% CO2/air. Confluent Huh7 and stationary phase Raji and Jurkat cells were diluted to 2 × 10^5^ cells/ml into 1 ml of medium in 12-well plates. On the following day, when the number of cells reached 4 × 10^5^ cells/ml, cells were stimulated by adding interferon alpha (IFNα) (30–2000 Units/ml; Roferon, Roche) [[17](#CR17)], interferon beta (IFNβ) (50–1000 Units/ml; human interferon β1a, Sigma)[[18](#CR18)], interferon gamma (IFNγ) (50–500 ng/ul; R&D systems)[[19](#CR19)], interferon lambda 3 (IFNλ3) (IL-28B 500 ng/ml; R&D systems) [[20](#CR20)], toll like receptor 7 (TLR7) agonist [RWJ21757](https://www.ncbi.nlm.nih.gov/protein/RWJ21757) (10 μmol; R&D systems) [[21](#CR21)] or tumour necrosis factor alpha (TNFα) (40 ng/μl; R&D systems) [[22](#CR22)].

Huh7, Raji and Jurkat cells were tested for the expression of interleukin 10 receptor beta (*IL10RB)*, interferon alpha receptor (*IFNαR), IFNL2/3 IL28*, 2'-5'-oligoadenylate synthetase (*OAS1)* and myxovirus (influenza virus) resistance 1 (*Mx1)* (with *GAPDH* as a housekeeping gene) using primers published by Diegelmann et al. [[23](#CR23)].

#### Peripheral Blood Mononuclear cells (PBMCs)

Peripheral blood mononuclear cells (PBMCs) from eight consented uninfected donors heterozygous for the *IFNL3.rs4803217* polymorphism were isolated using Hypaque-Ficoll (Amersham Biosciences) density centrifugation. 1–5 × 10^6^ cells in 1 ml of medium were stimulated with 800 Units /mls IFNα for 6 hours.

#### Genotyping of cell lines and healthy donors

SYBRGreen based RT-PCR was used to characterize the genomic DNA of cell lines and uninfected donors for the *IFNL3 polymorphisms rs12979860, rs8103142 and rs4803217.* The schematic location of these SNPs and their pairwise linkage disequilibrium is shown in Figure [1](#Fig1)A. Primers were designed by the author to be specific for *IFNL3* and ordered from Invitrogen. Primer sequences are listed in Table [1](#Tab1). The position of the three primers for the *rs4803217* RT-PCR assay within the sequence of *IFNL2/3* is illustrated in Figure [1](#Fig1)B.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/618d/4411934/1fd077721a83/12881_2014_104_Fig1_HTML.jpg)

Schematic location of the rs4803217 SNP in relationship to other SNPs in and near the IFNL2/3 gene. All four variants are in high linkage disequilibrium in Caucasians (r2 ≥ 0.92) [12]. The linkage disequilibrium between the four markers is represented in the table below the diagram and displayed as r2 (10;32), with a lower linkage disequilibrium reported between rs12979860 and rs8099917 [11]. (A) The nucleotide sequence around rs4803217 is shown for IFNL3 and IFNL2 with primers designed to be specific for IFNL3 (B). The IFNL3 specific forward primer contains a mismatch to IFNL2 at position 11 (from the 5’end) for IFNL3 specificity. To distinguish A and C allele of the rs4803217 SNP, the A specific primer ends with A at the 3’ end, and the C allele specific primer ends with C at the 3’ end. The reverse common primer contains one IFNL3 specific mismatch to IFNL2 at the 3rd base from the 3’ end. Mismatches between IFNL3 and IFNL2 are highlighted by asterix*, and the IFNL3 specific base within each primer is underlined.

| SNP | Location | Allele specific primer (5’- > 3’) | Common primer (5’- > 3’) |
| --- | --- | --- | --- |
| rs12979860 | 3 kb upstream | gcattcaaccctggttcA | gcttatcgcatacggctaggc |
| gcattcaaccctggttcG |  |  |  |
| rs8103142 | exon 2 (K70R) | cttctgctgaaggactgcaA | tgagcagggctgggagg |
| cttctgctgaaggactgcaG |  |  |  |
| rs4803217 | 3’UTR | cgactgggtgacaataaattaagA | cctgtgtgtctgacccttcc |
| cgactgggtgacaataaattaagC |  |  |  |

Table Caption: Primer sequences for IFNL3 genotyping and expression studies

Each DNA sample was interrogated twice, once with the primers specific for allele 1 (A) in combination with the common primer, then with the primer specific for allele 2 (C) in combination with the same common primer. The genotyping method using the primer combination for *IFNL3.rs12979860* has been described earlier [[24](#CR24)].

Reactions were performed in 96 well plates with 10 ng/μl genomic DNA and 0.5 μM of each primer (one allele specific and one common primer per reaction) in the presence of a SYBRGreen containing reagent mix (QuantiTect, Qiagen) using a StepOnePlus instrument (Invitrogen). The cycling conditions were initial denaturation of 10 min 95°C, 40 cycles of 15 seconds 95°C and 60 seconds 60°C, followed by a melt curve.

Genomic DNA for the validation of the method was used as 10 ng/ul after quantification with a NanoDrop spectrophotometer (Thermo Fisher).

#### cDNA synthesis

RNA was extracted from cells following the RNeasy protocol (Qiagen) and from frozen PBMCs of uninfected donors using Trizol reagent (Invitrogen) in a standard protocol including DNAse treatment. cDNA was synthesized using the Retroscript kit (Ambion) according to manufacturer’s instructions, but using oligo dTs and random decamers (2.5 μM each) in combination to optimize the efficiency of the cDNA synthesis. 1ug of RNA was transcribed into cDNA.

#### Relative quantification of gene expression by real time PCR

The same assay, including conditions and primers, which was used for genotyping was also used for relative quantification of allele specific gene expression. But in contrast to the genotyping assay, samples were run in triplicates for each allele specific reaction. 1 μl of a 20 μl of cDNA (cDNA from 50 ng of RNA) volume was put in each reaction. In order to use the assay for quantification of allele specific expression, the bias induced by the difference in amplification efficiency between the two allele specific reactions was corrected for by the use of a heterozygous genomic control in each experiment.

In order to validate the genotyping assay for its use in detecting potential 1.5 to 2 fold differences between the expression of the two alleles of *IFNL3.rs4803217*, different ratios of the two alleles from two DNAs homozygous for the respective alleles were combined and their respective dCt (difference between Ct value of allele 1 and 2) measured using the *IFNL3.rs4803217* genotyping assay. Assuming both allele specific primers have the same amplification efficiency of the assay, the reaction specific for allele1 should get amplified after the same number of amplification cycles than allele 2, and dCt should be zero for a heterozygous sample carrying the same amount of allele 1 and 2. The same results (with exception of DNA measurement and pipetting errors) should be approximated if allele 1 and allele 2 are combined in a 1:1 ratio from the two homozygous DNA samples. If the alleles are present in a ratio 1:2, with allele 2 having twice the frequency of allele 1), then allele 2 should get amplified one cycle earlier (as each cycle leads to duplication of the number of amplified copies) and the dCt should be 1. Therefore the relationship between ratio and dCt is represented by equation [[1](#CR1)]: Ratio = 1:2^dCt^. Allele 1 (A) and 2 (C) were combined in ratios 4:1, 3.1, 2.5:1, 2:1, 1.5:1, 1:1, 1:1.5, 1:2, 1:2.5, 1:3, 1:4.

The results were corrected with the dCT of the heterozygote genomic DNA control (HH23), which compensates for the difference in amplification efficiency of the two alleles. The relationship between known ratios of the two alleles and measured dCt values is shown in Figure [2](#Fig2).

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/618d/4411934/f126597395dc/12881_2014_104_Fig2_HTML.jpg)

Relationship between dCt and varying ratios of allele 1 (allele A) and 2 (allele C) measured by the IFNL3.rs4803217 assay. The assay shows the differences in the Ct value (dCT) of the rs4803217 assay for different ratios of C: A allele. The differences (dCt) in the Ct value for 11 different ratios between A and C allele (4:1, 3:1, 2.5:1, 2:1, 1.5:1, 1:1, 1:1.5, 1:2, 1:2.5, 1:3, 1:4) is plotted after correction with the difference observed in the heterozygous genomic control (HH23) , where the ratio A:C is exactly 1:1. Sensitivity of IFNL3.rs4803217 assay to detect 1.5-4 fold changes in allele specific expression. Data points are the mean of triplicate reactions and error bars indicate +/− SEM.

The inability of the assay to discriminate against genomic DNA present in the sample is off-set by the advantage of internal validation using three sequenced genotyping controls. As the genomic DNAs of the cells tested for variation of allele specific PCR are heterozygous for the SNP which the assay is detecting, they cannot contribute to variation in allele specific expression. Therefore any observed variation (after correction for the bias introduced through unequal amplification efficiency of the two allele specific reactions which is calculated by including a genomic heterozygous control in each assay) can be attributed to a variation in allele specific expression. RNA (without reverse transcription) controls were also included in each assay in order to detect amplification resulting from genomic DNA in the cDNA.

#### Statistical analysis

Data were analysed using Graph Pad Prism. The Mann–Whitney U test was applied to compare the means of the difference in allele expression between heterozygote genomic controls and cDNA in stimulated cells from four experiments in Raji cells, using three independently synthesized batches of cDNA from two independent stimulation events. Data obtained from the eight uninfected donors were analyzed in the same way. Two tailed p–values are calculated. Data are expressed as mean +/− SEM.

## Results

### Genotyping of cell lines and healthy donors

The *rs4803217* RT-PCR assay is specific for *IFNL3* (Figure [1](#Fig1)B), as it distinguishes three genotypes (Figure [3](#Fig3)). It excludes the amplification of *IFNL2*, which has the A allele “fixed” in the position of the *rs4803217* SNP ( 3’ end of the specific primer) . Cross reactivity with *IFNL2* would not allow the detection of CC homozygocity if this primer also recognized the A allele of *IFNL2*. The differences between the crossing points (dCt values) for the two alleles of *IFNL3.rs4803217* is robustly greater than dCT = 6 for homozygotes AA or CC in the absence of the other allele, whereas in heterozygotes the average dCt between the two allele specific reactions varies between 0.06 and 1 (assay variability) which is the result of the difference in amplification efficiency for the two alleles. To further confirm the *IFNL3* and allele specificity of the *rs4803217* assay, we sequenced amplicons which suggested AA and CC homozygocity respectively from samples which had been used as controls in each assay. We confirmed that the products were *IFNL3* and allele specific (Additional file [1](#MOESM1): Figure S1: Sequence of amplification products).

### Figure 3.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/618d/4411934/0f9d78c174e5/12881_2014_104_Fig3_HTML.jpg)

The IFNL3.rs4803217 assay distinguishes three IFNL3 specific genotypes. The Ct value is plotted for te amplification of the A and C allele specific primers respectively in each of the three genotypes.. In a sample with AA homozygocity (HH3), the A allele is getting amplified much earlier (dCt ≥ 6) in comparison to the C allele; in a sample with CC homozygocity (HH16), the C allele is getting amplified much earlier than allele A (dCT = −6). In the case of AC heterozygocity (HH11), both alleles are amplified after almost the same number of cycles (dCt = 0.06-1).

Huh7, Jurkat and Raji cells are heterozygous for the *IFNL3.rs12979860*, *IFNL3.rs8103142*, and *IFNL3.rs4803217* polymorphisms (Figure [1](#Fig1)A), thus making it an ideal system for the relative comparison of the expression levels of the two alleles. RNA controls revealed that only negligible amounts of genomic DNA were present in the cDNA samples used. Eight uninfected donors heterozygous for *rs4803217* were also heterozygous for *rs12979860* and *rs8103142*, which confirms the strong linkage disequilibrium between the three markers.

### Relative quantification of gene expression by real time PCR

Huh7, Raji and Jurkat cells showed constitutive expression of *IL10RB*, *IFNαR* and inducible expression of *IFNL2/3, OAS1* and *Mx1* (with *GAPDH* as a reference) after stimulation with 800 iU/ml IFNα, indicating that the receptors for the recognition of the IFN stimulation are intact and *IFNL2/3* is induced. Additional file [2](#MOESM2): Figure S2 (induction of OAS etc after IFNa stimulation) shows the induction of *OAS* and *IFNL2/3* in Huh7 cells after 3, 6, 8, 12 and 24 hours of stimulation with 800 iU IFNα.

Initial dose response experiments demonstrated that induction of *IFNL3* expression was optimal using 800 iU/mls IFNα and 6 hour incubation (see also Additional file [2](#MOESM2): Figure S2 (gene induction)). Therefore *IFNL3* expression for this manuscript was measured following this protocol. IFNβ showed the best induction after 6 hours of 500 iU/mls IFNβ, and IFNλ at 500 ng/ml. Both were less efficient in the induction of *IFNL3* expression and did not lead to reproducible comparisons of allele specific *IFNL3* expression. Toll like receptor 7 (TLR7) agonist [RWJ21757](https://www.ncbi.nlm.nih.gov/protein/RWJ21757) and tumor necrosis factor alpha (TNFα) were unable to induce detectable levels of *IFNL3* expression in Huh7 cells (data not shown) and were not tested in Raji or Jurkat cells.

As the validation experiment shows, the *IFNL3.rs4803217* allele specific PCR assay is able to measure the relationship between dCt and 1.5-2 fold changes in the ratio of the two specific alleles (Figure [2](#Fig2)). We therefore concluded that the assay for *IFNL3.rs4803217* is suitable to measure a difference in the expression of the two alleles in cells heterozygous for the *IFNL3.rs4803217 SNP*.

Using the *IFNL3.rs4803217* genotyping based quantitative RT-PCR assay, we detected a two-fold increased expression of *IFNL3.rs4803217* C specific transcript in Raji cells in four independent experiments (fold-change 2.09 (95% CI = 0.94-1.07) versus 1.01 (95% CI = 1.50-2.70), p = 0.029). The combined result of the four experiments is represented in Figure [4](#Fig4). In order to validate the results in primary cells we looked at allele specific *IFNL3* expression in eight healthy donors heterozygous for the *IFNL3.rs4803217* polymorphism. The relative expression levels of the two alleles showed variation between individuals, with ratios of C to A allele ranging between 1.4 and 0.7. Two of the eight donors showed an increased *IFNL3.rs4803217* C allele expression. The results were reproducible for each individual. The differences in the ratios did not reach statistical significance (Figure [5](#Fig5)). Due to the relatively low expression of the *IFNL3* gene in the B cells of uninfected subjects [[23](#CR23)], we were unable to measure allele specific *IFNL3.rs4803217* expression in their B cells (data not shown)

### Figure 4.

![Figure 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/618d/4411934/6f8d877d891f/12881_2014_104_Fig4_HTML.jpg)

Relative level of IFNL3.rs4803217 C-allele specific transcript detected in Raji cells heterozygous for IFNL3.rs4803217 after stimulation with 800 iU IFNa, compared to a genomic heterozygote control. The ratio of C:A allele is expressed as 2^dCt after correction with the differences observed in the genomic heterozygote control. The results of 4 independent experiments are shown. The p-value 0.029 was calculated using the Mann–Whitney U test.

### Figure 5.

![Figure 5](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/618d/4411934/01ccc9dc1262/12881_2014_104_Fig5_HTML.jpg)

Relative levels of IFNL3.rs4803217 C-allele specific transcript detected in frozen PBMCs from eight uninfected donors heterozygous for IFNL3.rs4803217 after stimulation with 800 iU IFNα, compared to a genomic heterozygote control. The ratio of C:A allele is expressed as 2^dCt value after correction with the differences observed in the genomic heterozygous control.

## Discussion

In order to develop an assay that is able to measure the expressed products of *IFNL3* in an allele specific manner, the 3’UTR SNP *rs4803217* was chosen, as it is in strong linkage disequilibrium with *rs12979860,* a predictive marker for the clinical outcome of HCV infection and treatment response*.* The assay clearly excludes the amplification of *IFNL2* specific sequence as it shows the distinction of three genotypes including AA homozygocity, which would not be possible if the C allele of *IFNL2* was present. In addition our assay is also able to discriminate between the two variants of the *IFNL3.rs4803217* polymorphism. Using this assay we observe increased expression of the *IFNL3.rs4803217 C* allele in Raji cells at 2 fold level. No difference in the relative level of allele specific expression was observed for Huh7 and Jurkat cells. Raji cells are derived from a B-cell line, where high levels of IFN-λR have been detected, suggesting that B-cells are actively involved in *IFNL3* signalling [[25](#CR25)]. Raj cells are reported to produce higher levels of *IFNL3* compared to Huh7 or Jurkat cell lines [[10](#CR10)], which supports the finding that allele specific differences of expression can be context and tissue specific [[26](#CR26)]. Unfortunately expression of *IFNL3* in the B-cell population of the healthy donors resulted in undetectable levels of *IFNL3* due to the low expression of *IFNL3* and smaller number of cells available (data not shown).

In order to validate our assay in primary cells, we selected eight uninfected donors heterozygous for *rs4803217* and *rs12979860.* The C allele was up-regulated in two of the eight donors. We detected marked variation in the relative ratio between the C and A allele, ranging from 1.4 to 0.7. Although this variation is not as high as the 2 fold increase as the expression of the C allele in Raji cells, and not as clear cut as a 1.5 ratio from the standard curve in Figure [2](#Fig2), the ratio was clearly reproducible for each donor.

The C allele of *IFNL3.rs4803217* is in LD with the *C* allele of *IFNL3.rs12979860*, which is associated with a high frequency of clearance and treatment response. Our results indicate that in certain cells or individuals the *rs12979860 CC* and also *CT* genotype are possibly producing more *IFNL3* transcript than the TT genotype, an observation which had been reported earlier using whole blood samples [[4](#CR4)], but was not confirmed in liver tissue [[8](#CR8)]. In contrast to Suppiah’s approach [[4](#CR4)] where allele specific variation is measured by comparing *IFNL2/3* expression in individuals homozygous for the two alleles of *rs12979860*, our assay is specific for *IFNL3* and is carried out in heterozygotes, so that a comparison of expression between the two alleles within the same cell is less likely to be subject to confounding variables. Our results obtained in eight uninfected healthy donors heterozygous for the *IFNL3* SNP *rs4803217* (and consequently *rs12979860*) suggests that variation exists between individuals with regards to allele specific expression. As we only study the differences observed in the allele specific expression of heterozygotes, we cannot exclude that homozygosity of each of the alleles could result in a greater effect of the genotype variation.

Variation in allelic gene expression affects 20-50% of human genes and may account for variation in the transmission of diseases and disease outcomes [[27](#CR27)–[29](#CR29)].

It has been reported that differences in allele specific expression are inherited, but that only 3-30% of individuals exhibit the allele specific expression [[27](#CR27)]. It was shown that allelic variation in the adenomatous polyposis coli (APC) gene expression plays a critical role in colon cancer [[30](#CR30)]. Variation in allelic expression was also measured for the *rs2834167* SNP in *IL10RB* in Epstein Bar virus (EBV) transformed B-cell line heterozygous for this SNP and might be the underlying functional cause for the association of this SNP with the outcome of HBV infection [[31](#CR31)].

In the literature conflicting results are reported with regards to *IFNL3* expression in PBMCs versus liver, and differences seem to exist between healthy and chronic donors. In our technical manuscript we use PBMCs of healthy donors to develop the assay as a tool which is able to measure *IFNL3* and allele specific expression of *IFNL3* transcripts in patient cohorts. A few publications have shown a potential role of blood and immune cells in the recruitment process to the liver during inflammation. Although the expression status of cells might differ between healthy donors and during the process of inflammation, PBMCs are a valid starting point for measuring allele specific expression of *IFNL3* [[32](#CR32)].

As we did not find consistent up-regulation of the C allele in eight donors heterozygous for the C allele of *IFNL3.rs4803217* and *rs12979860*, we investigated the possibility that the observed variation in expression of the *rs4803217 C* transcript is influenced by other SNPs in the *IFNL3* region. It has been reported that a new variant upstream of the *IFNL3 gene*, which is in high linkage disequilibrium with *rs12979860,* and leads to a frameshift mutation creating a new interferon gene *IFNL4*, might explain the association with the clinical outcomes of HCV infection [[33](#CR33)]. It has been noted that the expression of the *IFNL3* gene is dependent on the genotype of the novel *ss469415590 SNP* [[34](#CR34)]. Genotyping all donors in our study for *ss469415590* revealed that they were all heterozygous for *ss469415590*, thus excluding the possibility of an influence of this variant.

In the three cell lines and eight donors heterozygous for *IFNL3.rs4803217* we find 100% linkage with the *IFNL3.rs12979860* marker. High LD between both markers is characteristic for Asian and Caucasian populations [[12](#CR12)] (HapMap database r2 = 0.9 and 0.7-0.8 respectively), but reported to be at r2 = 0.4 for Sub-Saharan samples. More recently it was shown that the *IFNL3.rs4803217* variation seems to affect the stability of the transcript, which could be another way by which the *IFNL3* polymorphism exerts its function [[35](#CR35)]. Our finding of varying C:A allele expression in certain cells and individuals could be a result of differential stability of the transcript in different individuals. Further studies need to be undertaken to explore the function of the *rs4803217* polymorphism in the outcome of HCV infection and treatment response, using an *IFNL3* and allele specific assay for *rs4803217*.

## Conclusion

In conclusion, we show that the allele specific assay for *IFNL3* which we developed is a useful tool to determine the *IFNL3.rs4803217* genotype and at the same time accurately and directly compare the expression of two alleles of the *IFNL3* gene within heterozygous cells. As the *rs4803217* SNP has been reported to have an effect on the stability of the transcript [[35](#CR35)] the developed assay could be used for the measurement of allele specific *IFNL3* expression in patient groups with different HCV outcomes and treatment response, to determine the clinical relevance of the observed variation.

## Acknowledgements

The authors acknowledge funding by the EU framework seven (FP7) grant 260844 awarded to MT. SK was funded by the EU framework seven (FP7) grant 260844. NM was a BSc student at the time of the conception of the project. SC and KJ were supported by the Wellcome Trust. All authors acknowledge support from the Imperial College Biomedical Research Centre. The authors thank Dr Marco Purbhoo for making available the Huh7, Raji and Jurkat cell lines.

### Additional files

### Additional files

## Footnotes

## Contributor Information

Susanne Knapp, Email: s.knapp@imperial.ac.uk.

Naeem Meghjee, Email: naeem.meghjee08@imperial.ac.uk.

Sorcha Cassidy, Email: sorcha.cassidy09@imperial.ac.uk.

Khaleel Jamil, Email: k.jamil@imperial.ac.uk.

Mark Thursz, Email: m.thursz@imperial.ac.uk.

## References

1. Ge D, Fellay J, Thompson AJ, Simon JS, Shianna KV, Urban TJ, Heinzen EL, Qiu P, Bertelsen AH, Muir AJ, Sulkowski M, McHutchison JG, Goldstein DB. Genetic variation in IL28B predicts hepatitis C treatment-induced viral clearance. Nature. 2009;461(7262):399–401. doi: 10.1038/nature08309.  [DOI](https://doi.org/10.1038/nature08309) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19684573/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Genetic%20variation%20in%20IL28B%20predicts%20hepatitis%20C%20treatment-induced%20viral%20clearance&author=D%20Ge&author=J%20Fellay&author=AJ%20Thompson&author=JS%20Simon&author=KV%20Shianna&volume=461&issue=7262&publication_year=2009&pages=399-401&pmid=19684573&doi=10.1038/nature08309&)

2. Thomas DL, Thio CL, Martin MP, Qi Y, Ge D, O'Huigin C, Kidd J, Kidd K, Khakoo SI, Alexander G, Goedert JJ, Kirk GD, Donfield SM, Rosen HR, Tobler LH, Busch MP, McHutchison JG, Goldstein DB, Carrington M. Genetic variation in IL28B and spontaneous clearance of hepatitis C virus. Nature. 2009;461(7265):798–801. doi: 10.1038/nature08463.  [DOI](https://doi.org/10.1038/nature08463) | [PMC free article](/articles/PMC3172006/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19759533/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Genetic%20variation%20in%20IL28B%20and%20spontaneous%20clearance%20of%20hepatitis%20C%20virus&author=DL%20Thomas&author=CL%20Thio&author=MP%20Martin&author=Y%20Qi&author=D%20Ge&volume=461&issue=7265&publication_year=2009&pages=798-801&pmid=19759533&doi=10.1038/nature08463&)

3. Tanaka Y, Nishida N, Sugiyama M, Kurosaki M, Matsuura K, Sakamoto N, Nakagawa M, Korenaga M, Hino K, Hige S, Ito Y, Mita E, Tanaka E, Mochida S, Murawaki Y, Honda M, Sakai A, Hiasa Y, Nishiguchi S, Koike A, Sakaida I, Imamura M, Ito K, Yano K, Masaki N, Sugauchi F, Izumi N, Tokunaga K, Mizokami M. Genome-wide association of IL28B with response to pegylated interferon-alpha and ribavirin therapy for chronic hepatitis C. Nat Genet. 2009;41(10):1105–9. doi: 10.1038/ng.449.  [DOI](https://doi.org/10.1038/ng.449) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19749757/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Genome-wide%20association%20of%20IL28B%20with%20response%20to%20pegylated%20interferon-alpha%20and%20ribavirin%20therapy%20for%20chronic%20hepatitis%20C&author=Y%20Tanaka&author=N%20Nishida&author=M%20Sugiyama&author=M%20Kurosaki&author=K%20Matsuura&volume=41&issue=10&publication_year=2009&pages=1105-9&pmid=19749757&doi=10.1038/ng.449&)

4. Suppiah V, Moldovan M, Ahlenstiel G, Berg T, Weltman M, Abate ML, Bassendine M, Spengler U, Dore GJ, Powell E, Riordan S, Sheridan D, Smedile A, Fragomeli V, Müller T, Bahlo M, Stewart GJ, Booth DR, George J. IL28B is associated with response to chronic hepatitis C interferon-alpha and ribavirin therapy. Nat Genet. 2009;41(10):1100–4. doi: 10.1038/ng.447.  [DOI](https://doi.org/10.1038/ng.447) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19749758/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=IL28B%20is%20associated%20with%20response%20to%20chronic%20hepatitis%20C%20interferon-alpha%20and%20ribavirin%20therapy&author=V%20Suppiah&author=M%20Moldovan&author=G%20Ahlenstiel&author=T%20Berg&author=M%20Weltman&volume=41&issue=10&publication_year=2009&pages=1100-4&pmid=19749758&doi=10.1038/ng.447&)

5. Rauch A, Kutalik Z, Descombes P, Cai T, Di IJ, Mueller T, Bochud M, Battegay M, Bernasconi E, Borovicka J, Colombo S, Cerny A, Dufour JF, Furrer H, Günthard HF, Heim M, Hirschel B, Malinverni R, Moradpour D, Müllhaupt B, Witteck A, Beckmann JS, Berg T, Bergmann S, Negro F, Telenti A, Bochud PY, Swiss Hepatitis C Cohort Study. Swiss HIV Cohort Study Genetic variation in IL28B is associated with chronic hepatitis C and treatment failure: a genome-wide association study. Gastroenterology. 2010;138(4):1338–45. doi: 10.1053/j.gastro.2009.12.056.  [DOI](https://doi.org/10.1053/j.gastro.2009.12.056) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20060832/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gastroenterology&title=Genetic%20variation%20in%20IL28B%20is%20associated%20with%20chronic%20hepatitis%20C%20and%20treatment%20failure:%20a%20genome-wide%20association%20study&author=A%20Rauch&author=Z%20Kutalik&author=P%20Descombes&author=T%20Cai&author=IJ%20Di&volume=138&issue=4&publication_year=2010&pages=1338-45&pmid=20060832&doi=10.1053/j.gastro.2009.12.056&)

6. Pedergnana V, Abdel-Hamid M, Guergnon J, Mohsen A, Le Fouler L, Theodorou I, Mohamed MK, Fontanet A, Plancoulaine S, Abel L. Analysis of IL28B variants in an Egyptian population defines the 20 kilobases minimal region involved in spontaneous clearance of hepatitis C virus. PLoS One. 2012;7(6):e38578. doi: 10.1371/journal.pone.0038578.  [DOI](https://doi.org/10.1371/journal.pone.0038578) | [PMC free article](/articles/PMC3375300/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22719902/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=Analysis%20of%20IL28B%20variants%20in%20an%20Egyptian%20population%20defines%20the%2020%20kilobases%20minimal%20region%20involved%20in%20spontaneous%20clearance%20of%20hepatitis%20C%20virus&author=V%20Pedergnana&author=M%20Abdel-Hamid&author=J%20Guergnon&author=A%20Mohsen&author=L%20Le%20Fouler&volume=7&issue=6&publication_year=2012&pages=e38578&pmid=22719902&doi=10.1371/journal.pone.0038578&)

7. Abe H, Hayes CN, Ochi H, Maekawa T, Tsuge M, Miki D, Mitsui F, Hiraga N, Imamura M, Takahashi S, Kubo M, Nakamura Y, Chayama K, Mitsui F, Hiraga N, Imamura M, Takahashi S, Kubo M, Nakamura Y, Chayama K. IL28 variation affects expression of interferon stimulated genes and peg-interferon and ribavirin therapy. J Hepatol. 2011;54(6):1094–101. doi: 10.1016/j.jhep.2010.09.019.  [DOI](https://doi.org/10.1016/j.jhep.2010.09.019) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21145800/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Hepatol&title=IL28%20variation%20affects%20expression%20of%20interferon%20stimulated%20genes%20and%20peg-interferon%20and%20ribavirin%20therapy&author=H%20Abe&author=CN%20Hayes&author=H%20Ochi&author=T%20Maekawa&author=M%20Tsuge&volume=54&issue=6&publication_year=2011&pages=1094-101&pmid=21145800&doi=10.1016/j.jhep.2010.09.019&)

8. Urban TJ, Thompson AJ, Bradrick SS, Fellay J, Schuppan D, Cronin KD, Hong L, McKenzie A, Patel K, Shianna KV, McHutchison JG, Goldstein DB, Afdhal N. IL28B genotype is associated with differential expression of intrahepatic interferon-stimulated genes in patients with chronic hepatitis C. Hepatology. 2010;52(6):1888–96. doi: 10.1002/hep.23912.  [DOI](https://doi.org/10.1002/hep.23912) | [PMC free article](/articles/PMC3653303/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20931559/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hepatology&title=IL28B%20genotype%20is%20associated%20with%20differential%20expression%20of%20intrahepatic%20interferon-stimulated%20genes%20in%20patients%20with%20chronic%20hepatitis%20C&author=TJ%20Urban&author=AJ%20Thompson&author=SS%20Bradrick&author=J%20Fellay&author=D%20Schuppan&volume=52&issue=6&publication_year=2010&pages=1888-96&pmid=20931559&doi=10.1002/hep.23912&)

9. Ito K, Higami K, Masaki N, Sugiyama M, Mukaide M, Saito H, Aoki Y, Sato Y, Imamura M, Murata K, Nomura H, Hige S, Adachi H, Hino K, Yatsuhashi H, Orito E, Kani S, Tanaka Y, Mizokami M. The rs8099917 polymorphism, when determined by a suitable genotyping method, is a better predictor for response to pegylated alpha interferon/ribavirin therapy in Japanese patients than other single nucleotide polymorphisms associated with interleukin-28B. J Clin Microbiol. 2011;49(5):1853–60. doi: 10.1128/JCM.02139-10.  [DOI](https://doi.org/10.1128/JCM.02139-10) | [PMC free article](/articles/PMC3122695/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21389156/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Microbiol&title=The%20rs8099917%20polymorphism,%20when%20determined%20by%20a%20suitable%20genotyping%20method,%20is%20a%20better%20predictor%20for%20response%20to%20pegylated%20alpha%20interferon/ribavirin%20therapy%20in%20Japanese%20patients%20than%20other%20single%20nucleotide%20polymorphisms%20associated%20with%20interleukin-28B&author=K%20Ito&author=K%20Higami&author=N%20Masaki&author=M%20Sugiyama&author=M%20Mukaide&volume=49&issue=5&publication_year=2011&pages=1853-60&pmid=21389156&doi=10.1128/JCM.02139-10&)

10. Sugiyama M, Kimura T, Naito S, Mukaide M, Shinauchi T, Ueno M, Ito K, Murata K, Mizokami M. Development of specific and quantitative real-time detection PCR and immunoassays for λ3-interferon. Hepatol Res. 2012;42(11):1089–99. doi: 10.1111/j.1872-034X.2012.01032.x.  [DOI](https://doi.org/10.1111/j.1872-034X.2012.01032.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22672583/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hepatol%20Res&title=Development%20of%20specific%20and%20quantitative%20real-time%20detection%20PCR%20and%20immunoassays%20for%20%CE%BB3-interferon&author=M%20Sugiyama&author=T%20Kimura&author=S%20Naito&author=M%20Mukaide&author=T%20Shinauchi&volume=42&issue=11&publication_year=2012&pages=1089-99&pmid=22672583&doi=10.1111/j.1872-034X.2012.01032.x&)

11. De Castellarnau M, Aparicio E, Parera M, Franco S, Tural C, Clotet B, Martinez MA. Deciphering the interleukin 28B variants that better predict response to pegylated interferon-alpha and ribavirin therapy in HCV/HIV-1 coinfected patients. PLoS One. 2012;7(2):e31016. doi: 10.1371/journal.pone.0031016.  [DOI](https://doi.org/10.1371/journal.pone.0031016) | [PMC free article](/articles/PMC3273458/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22328925/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=Deciphering%20the%20interleukin%2028B%20variants%20that%20better%20predict%20response%20to%20pegylated%20interferon-alpha%20and%20ribavirin%20therapy%20in%20HCV/HIV-1%20coinfected%20patients&author=M%20De%20Castellarnau&author=E%20Aparicio&author=M%20Parera&author=S%20Franco&author=C%20Tural&volume=7&issue=2&publication_year=2012&pages=e31016&pmid=22328925&doi=10.1371/journal.pone.0031016&)

12. Smith KR, Suppiah V, O'Connor K, Berg T, Weltman M, Abate ML, Spengler U, Bassendine M, Matthews G, Irving WL, Powell E, Riordan S, Ahlenstiel G, Stewart GJ, Bahlo M, George J, Booth DR. International Hepatitis C Genetics Consortium (IHCGC): Identification of improved IL28B SNPs and haplotypes for prediction of drug response in treatment of hepatitis C using massively parallel sequencing in a cross-sectional European cohort. Genome Med. 2011;3(8):57. doi: 10.1186/gm273.  [DOI](https://doi.org/10.1186/gm273) | [PMC free article](/articles/PMC3238183/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21884576/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Med&title=International%20Hepatitis%20C%20Genetics%20Consortium%20(IHCGC):%20Identification%20of%20improved%20IL28B%20SNPs%20and%20haplotypes%20for%20prediction%20of%20drug%20response%20in%20treatment%20of%20hepatitis%20C%20using%20massively%20parallel%20sequencing%20in%20a%20cross-sectional%20European%20cohort&author=KR%20Smith&author=V%20Suppiah&author=K%20O'Connor&author=T%20Berg&author=M%20Weltman&volume=3&issue=8&publication_year=2011&pages=57&pmid=21884576&doi=10.1186/gm273&)

13. Germer S, Holland MJ, Higuchi R. High-throughput SNP allele-frequency determination in pooled DNA samples by kinetic PCR. Genome Res. 2000;10(2):258–66. doi: 10.1101/gr.10.2.258.  [DOI](https://doi.org/10.1101/gr.10.2.258) | [PMC free article](/articles/PMC310828/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10673283/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Res&title=High-throughput%20SNP%20allele-frequency%20determination%20in%20pooled%20DNA%20samples%20by%20kinetic%20PCR&author=S%20Germer&author=MJ%20Holland&author=R%20Higuchi&volume=10&issue=2&publication_year=2000&pages=258-66&pmid=10673283&doi=10.1101/gr.10.2.258&)

14. Nakabayashi H, Taketa K, Miyano K, Yamane T, Sato J. Growth of human hepatoma cells lines with differentiated functions in chemically defined medium. Cancer Res. 1982;42(9):3858–63.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/6286115/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=Growth%20of%20human%20hepatoma%20cells%20lines%20with%20differentiated%20functions%20in%20chemically%20defined%20medium&author=H%20Nakabayashi&author=K%20Taketa&author=K%20Miyano&author=T%20Yamane&author=J%20Sato&volume=42&issue=9&publication_year=1982&pages=3858-63&pmid=6286115&)

15. Drexler HG, Minowada J. History and classification of human leukemia-lymphoma cell lines. Leuk Lymphoma. 1998;31(3–4):305–16. doi: 10.3109/10428199809059223.  [DOI](https://doi.org/10.3109/10428199809059223) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9869194/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Leuk%20Lymphoma&title=History%20and%20classification%20of%20human%20leukemia-lymphoma%20cell%20lines&author=HG%20Drexler&author=J%20Minowada&volume=31&issue=3%E2%80%934&publication_year=1998&pages=305-16&pmid=9869194&doi=10.3109/10428199809059223&)

16. Schneider U, Schwenk HU, Bornkamm G. Characterization of EBV-genome negative "null" and "T" cell lines derived from children with acute lymphoblastic leukemia and leukemic transformed non-Hodgkin lymphoma. Int J Cancer. 1977;19(5):621–6. doi: 10.1002/ijc.2910190505.  [DOI](https://doi.org/10.1002/ijc.2910190505) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/68013/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Cancer&title=Characterization%20of%20EBV-genome%20negative%20%22null%22%20and%20%22T%22%20cell%20lines%20derived%20from%20children%20with%20acute%20lymphoblastic%20leukemia%20and%20leukemic%20transformed%20non-Hodgkin%20lymphoma&author=U%20Schneider&author=HU%20Schwenk&author=G%20Bornkamm&volume=19&issue=5&publication_year=1977&pages=621-6&pmid=68013&doi=10.1002/ijc.2910190505&)

17. Zhao LJ, Hua X, He SF, Ren H, Qi ZT. Interferon alpha regulates MAPK and STAT1 pathways in human hepatoma cells. Virol J. 2011;8:157. doi: 10.1186/1743-422X-8-157.  [DOI](https://doi.org/10.1186/1743-422X-8-157) | [PMC free article](/articles/PMC3080318/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21466707/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Virol%20J&title=Interferon%20alpha%20regulates%20MAPK%20and%20STAT1%20pathways%20in%20human%20hepatoma%20cells&author=LJ%20Zhao&author=X%20Hua&author=SF%20He&author=H%20Ren&author=ZT%20Qi&volume=8&publication_year=2011&pages=157&pmid=21466707&doi=10.1186/1743-422X-8-157&)

18. Pedersen IM, Cheng G, Wieland S, Volinia S, Croce CM, Chisari FV, David M. Interferon modulation of cellular microRNAs as an antiviral mechanism. Nature. 2007;449(7164):919–22. doi: 10.1038/nature06205.  [DOI](https://doi.org/10.1038/nature06205) | [PMC free article](/articles/PMC2748825/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17943132/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Interferon%20modulation%20of%20cellular%20microRNAs%20as%20an%20antiviral%20mechanism&author=IM%20Pedersen&author=G%20Cheng&author=S%20Wieland&author=S%20Volinia&author=CM%20Croce&volume=449&issue=7164&publication_year=2007&pages=919-22&pmid=17943132&doi=10.1038/nature06205&)

19. Shin EC, Seifert U, Kato T, Rice CM, Feinstone SM, Kloetzel PM, Rehermann B. Virus-induced type I IFN stimulates generation of immunoproteasomes at the site of infection. J Clin Invest. 2006;116(11):3006–14. doi: 10.1172/JCI29832.  [DOI](https://doi.org/10.1172/JCI29832) | [PMC free article](/articles/PMC1592549/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17039255/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Invest&title=Virus-induced%20type%20I%20IFN%20stimulates%20generation%20of%20immunoproteasomes%20at%20the%20site%20of%20infection&author=EC%20Shin&author=U%20Seifert&author=T%20Kato&author=CM%20Rice&author=SM%20Feinstone&volume=116&issue=11&publication_year=2006&pages=3006-14&pmid=17039255&doi=10.1172/JCI29832&)

20. Zhang L, Jilg N, Shao RX, Lin W, Fusco DN, Zhao H, Goto K, Peng LF, Chen WC, Chung RT. IL28B inhibits hepatitis C virus replication through the JAK-STAT pathway. J Hepatol. 2011;55(2):289–98. doi: 10.1016/j.jhep.2010.11.019.  [DOI](https://doi.org/10.1016/j.jhep.2010.11.019) | [PMC free article](/articles/PMC3068235/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21147189/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Hepatol&title=IL28B%20inhibits%20hepatitis%20C%20virus%20replication%20through%20the%20JAK-STAT%20pathway&author=L%20Zhang&author=N%20Jilg&author=RX%20Shao&author=W%20Lin&author=DN%20Fusco&volume=55&issue=2&publication_year=2011&pages=289-98&pmid=21147189&doi=10.1016/j.jhep.2010.11.019&)

21. Gondois-Rey F, Dental C, Halfon P, Baumert TF, Olive D, Hirsch I. Hepatitis C virus is a weak inducer of interferon alpha in plasmacytoid dendritic cells in comparison with influenza and human herpesvirus type-1. PLoS One. 2009;4(2):e4319. doi: 10.1371/journal.pone.0004319.  [DOI](https://doi.org/10.1371/journal.pone.0004319) | [PMC free article](/articles/PMC2629532/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19183807/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=Hepatitis%20C%20virus%20is%20a%20weak%20inducer%20of%20interferon%20alpha%20in%20plasmacytoid%20dendritic%20cells%20in%20comparison%20with%20influenza%20and%20human%20herpesvirus%20type-1&author=F%20Gondois-Rey&author=C%20Dental&author=P%20Halfon&author=TF%20Baumert&author=D%20Olive&volume=4&issue=2&publication_year=2009&pages=e4319&pmid=19183807&doi=10.1371/journal.pone.0004319&)

22. Helbig KJ, Ruszkiewicz A, Semendric L, Harley HA, McColl SR, Beard MR. Expression of the CXCR3 ligand I-TAC by hepatocytes in chronic hepatitis C and its correlation with hepatic inflammation. Hepatology. 2004;39(5):1220–9. doi: 10.1002/hep.20167.  [DOI](https://doi.org/10.1002/hep.20167) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15122750/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hepatology&title=Expression%20of%20the%20CXCR3%20ligand%20I-TAC%20by%20hepatocytes%20in%20chronic%20hepatitis%20C%20and%20its%20correlation%20with%20hepatic%20inflammation&author=KJ%20Helbig&author=A%20Ruszkiewicz&author=L%20Semendric&author=HA%20Harley&author=SR%20McColl&volume=39&issue=5&publication_year=2004&pages=1220-9&pmid=15122750&doi=10.1002/hep.20167&)

23. Diegelmann J, Beigel F, Zitzmann K, Kaul A, Goke B, Auernhammer CJ, Bartenschlager R, Diepolder HM, Brand S. Comparative analysis of the lambda-interferons IL-28A and IL-29 regarding their transcriptome and their antiviral properties against hepatitis C virus. PLoS One. 2010;5(12):e15200. doi: 10.1371/journal.pone.0015200.  [DOI](https://doi.org/10.1371/journal.pone.0015200) | [PMC free article](/articles/PMC2999541/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21170333/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=Comparative%20analysis%20of%20the%20lambda-interferons%20IL-28A%20and%20IL-29%20regarding%20their%20transcriptome%20and%20their%20antiviral%20properties%20against%20hepatitis%20C%20virus&author=J%20Diegelmann&author=F%20Beigel&author=K%20Zitzmann&author=A%20Kaul&author=B%20Goke&volume=5&issue=12&publication_year=2010&pages=e15200&pmid=21170333&doi=10.1371/journal.pone.0015200&)

24. Knapp S, Warshow U, Ho KM, Hegazy D, Little AM, Fowell A, Alexander G, Thursz M, Cramp M, Khakoo SI. A polymorphism in IL28B distinguishes exposed, uninfected individuals from spontaneous resolvers of HCV infection. Gastroenterology. 2011;141(1):320–5. doi: 10.1053/j.gastro.2011.04.005.  [DOI](https://doi.org/10.1053/j.gastro.2011.04.005) | [PMC free article](/articles/PMC3194089/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21600205/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gastroenterology&title=A%20polymorphism%20in%20IL28B%20distinguishes%20exposed,%20uninfected%20individuals%20from%20spontaneous%20resolvers%20of%20HCV%20infection&author=S%20Knapp&author=U%20Warshow&author=KM%20Ho&author=D%20Hegazy&author=AM%20Little&volume=141&issue=1&publication_year=2011&pages=320-5&pmid=21600205&doi=10.1053/j.gastro.2011.04.005&)

25. Witte K, Gruetz G, Volk HD, Looman AC, Asadullah K, Sterry W, Sabat R, Wolk K. Despite IFN-lambda receptor expression, blood immune cells, but not keratinocytes or melanocytes, have an impaired response to type III interferons: implications for therapeutic applications of these cytokines. Genes Immun. 2009;10(8):702–14. doi: 10.1038/gene.2009.72.  [DOI](https://doi.org/10.1038/gene.2009.72) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19798076/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genes%20Immun&title=Despite%20IFN-lambda%20receptor%20expression,%20blood%20immune%20cells,%20but%20not%20keratinocytes%20or%20melanocytes,%20have%20an%20impaired%20response%20to%20type%20III%20interferons:%20implications%20for%20therapeutic%20applications%20of%20these%20cytokines&author=K%20Witte&author=G%20Gruetz&author=HD%20Volk&author=AC%20Looman&author=K%20Asadullah&volume=10&issue=8&publication_year=2009&pages=702-14&pmid=19798076&doi=10.1038/gene.2009.72&)

26. Knight JC, Keating BJ, Rockett KA, Kwiatkowski DP. In vivo characterization of regulatory polymorphisms by allele-specific quantification of RNA polymerase loading. Nat Genet. 2003;33(4):469–75. doi: 10.1038/ng1124.  [DOI](https://doi.org/10.1038/ng1124) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12627232/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=In%20vivo%20characterization%20of%20regulatory%20polymorphisms%20by%20allele-specific%20quantification%20of%20RNA%20polymerase%20loading&author=JC%20Knight&author=BJ%20Keating&author=KA%20Rockett&author=DP%20Kwiatkowski&volume=33&issue=4&publication_year=2003&pages=469-75&pmid=12627232&doi=10.1038/ng1124&)

27. Yan H, Yuan W, Velculescu VE, Vogelstein B, Kinzler KW. Allelic variation in human gene expression. Science. 2002;297(5584):1143. doi: 10.1126/science.1072545.  [DOI](https://doi.org/10.1126/science.1072545) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12183620/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=Allelic%20variation%20in%20human%20gene%20expression&author=H%20Yan&author=W%20Yuan&author=VE%20Velculescu&author=B%20Vogelstein&author=KW%20Kinzler&volume=297&issue=5584&publication_year=2002&pages=1143&pmid=12183620&doi=10.1126/science.1072545&)

28. Lo HS, Wang Z, Hu Y, Yang HH, Gere S, Buetow KH, Lee MP. Allelic variation in gene expression is common in the human genome. Genome Res. 2003;13(8):1855–62. doi: 10.1101/gr.1006603.  [DOI](https://doi.org/10.1101/gr.1006603) | [PMC free article](/articles/PMC403776/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12902379/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Res&title=Allelic%20variation%20in%20gene%20expression%20is%20common%20in%20the%20human%20genome&author=HS%20Lo&author=Z%20Wang&author=Y%20Hu&author=HH%20Yang&author=S%20Gere&volume=13&issue=8&publication_year=2003&pages=1855-62&pmid=12902379&doi=10.1101/gr.1006603&)

29. Grievink H, Stowell KM. Allele-specific differences in ryanodine receptor 1 mRNA expression levels may contribute to phenotypic variability in malignant hyperthermia. Orphanet J Rare Dis. 2010;5:10. doi: 10.1186/1750-1172-5-10.  [DOI](https://doi.org/10.1186/1750-1172-5-10) | [PMC free article](/articles/PMC2895584/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20482855/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Orphanet%20J%20Rare%20Dis&title=Allele-specific%20differences%20in%20ryanodine%20receptor%201%20mRNA%20expression%20levels%20may%20contribute%20to%20phenotypic%20variability%20in%20malignant%20hyperthermia&author=H%20Grievink&author=KM%20Stowell&volume=5&publication_year=2010&pages=10&pmid=20482855&doi=10.1186/1750-1172-5-10&)

30. Yan H, Dobbie Z, Gruber SB, Markowitz S, Romans K, Giardiello FM, Kinzler KW, Vogelstein B. Small changes in expression affect predisposition to tumorigenesis. Nat Genet. 2002;30(1):25–6. doi: 10.1038/ng799.  [DOI](https://doi.org/10.1038/ng799) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11743581/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Small%20changes%20in%20expression%20affect%20predisposition%20to%20tumorigenesis&author=H%20Yan&author=Z%20Dobbie&author=SB%20Gruber&author=S%20Markowitz&author=K%20Romans&volume=30&issue=1&publication_year=2002&pages=25-6&pmid=11743581&doi=10.1038/ng799&)

31. Frodsham AJ, Zhang L, Dumpis U, Taib NA, Best S, Durham A, Hennig BJ, Hellier S, Knapp S, Wright M, Chiaramonte M, Bell JI, Graves M, Whittle HC, Thomas HC, Thursz MR, Hill AV, Hennig BJ, Hellier S, Knapp S, Wright M, Chiaramonte M, Bell JI, Graves M, Whittle HC, Thomas HC, Thursz MR, Hill AV. Class II cytokine receptor gene cluster is a major locus for hepatitis B persistence. Proc Natl Acad Sci U S A. 2006;103(24):9148–53. doi: 10.1073/pnas.0602800103.  [DOI](https://doi.org/10.1073/pnas.0602800103) | [PMC free article](/articles/PMC1482581/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16757563/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&title=Class%20II%20cytokine%20receptor%20gene%20cluster%20is%20a%20major%20locus%20for%20hepatitis%20B%20persistence&author=AJ%20Frodsham&author=L%20Zhang&author=U%20Dumpis&author=NA%20Taib&author=S%20Best&volume=103&issue=24&publication_year=2006&pages=9148-53&pmid=16757563&doi=10.1073/pnas.0602800103&)

32. Accapezzato D, Francavilla V, Paroli M, Casciaro M, Chircu LV, Cividini A, Abrignani S, Mondelli MU, Barnaba V. Hepatic expansion of a virus-specific regulatory CD8(+) T cell population in chronic hepatitis C virus infection. J Clin Invest. 2004;113(7):963–72. doi: 10.1172/JCI200420515.  [DOI](https://doi.org/10.1172/JCI200420515) | [PMC free article](/articles/PMC379326/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15057302/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Invest&title=Hepatic%20expansion%20of%20a%20virus-specific%20regulatory%20CD8(+)%20T%20cell%20population%20in%20chronic%20hepatitis%20C%20virus%20infection&author=D%20Accapezzato&author=V%20Francavilla&author=M%20Paroli&author=M%20Casciaro&author=LV%20Chircu&volume=113&issue=7&publication_year=2004&pages=963-72&pmid=15057302&doi=10.1172/JCI200420515&)

33. Prokunina-Olsson L, Muchmore B, Tang W, Pfeiffer RM, Park H, Dickensheets H, Hergott D, Porter-Gill P, Mumy A, Kohaar I, Chen S, Brand N, Tarway M, Liu L, Sheikh F, Astemborski J, Bonkovsky HL, Edlin BR, Howell CD, Morgan TR, Thomas DL, Rehermann B, Donnelly RP, O'Brien TR. A variant upstream of IFNL3 (IL28B) creating a new interferon gene IFNL4 is associated with impaired clearance of hepatitis C virus. Nat Genet. 2013;45(2):164–71. doi: 10.1038/ng.2521.  [DOI](https://doi.org/10.1038/ng.2521) | [PMC free article](/articles/PMC3793390/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23291588/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=A%20variant%20upstream%20of%20IFNL3%20(IL28B)%20creating%20a%20new%20interferon%20gene%20IFNL4%20is%20associated%20with%20impaired%20clearance%20of%20hepatitis%20C%20virus&author=L%20Prokunina-Olsson&author=B%20Muchmore&author=W%20Tang&author=RM%20Pfeiffer&author=H%20Park&volume=45&issue=2&publication_year=2013&pages=164-71&pmid=23291588&doi=10.1038/ng.2521&)

34. Bibert S, Roger T, Calandra T, Bochud M, Cerny A, Semmo N, Duong FH, Gerlach T, Malinverni R, Moradpour D, Negro F, Müllhaupt B, Bochud PY, Swiss Hepatitis C Cohort Study IL28B expression depends on a novel TT/-G polymorphism which improves HCV clearance prediction. J Exp Med. 2013;210(6):1109–16. doi: 10.1084/jem.20130012.  [DOI](https://doi.org/10.1084/jem.20130012) | [PMC free article](/articles/PMC3674704/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23712427/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Exp%20Med&title=IL28B%20expression%20depends%20on%20a%20novel%20TT/-G%20polymorphism%20which%20improves%20HCV%20clearance%20prediction&author=S%20Bibert&author=T%20Roger&author=T%20Calandra&author=M%20Bochud&author=A%20Cerny&volume=210&issue=6&publication_year=2013&pages=1109-16&pmid=23712427&doi=10.1084/jem.20130012&)

35. McFarland AP, Horner SM, Jarret A, Joslyn RC, Bindewald E, Shapiro BA, Delker DA, Hagedorn CH, Carrington M, Gale M, Jr, Savan R. The favorable IFNL3 genotype escapes mRNA decay mediated by AU-rich elements and hepatitis C virus-induced microRNAs. Nat Immunol. 2014;15(1):72–9. doi: 10.1038/ni.2758.  [DOI](https://doi.org/10.1038/ni.2758) | [PMC free article](/articles/PMC4183367/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24241692/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Immunol.&title=The%20favorable%20IFNL3%20genotype%20escapes%20mRNA%20decay%20mediated%20by%20AU-rich%20elements%20and%20hepatitis%20C%20virus-induced%20microRNAs&author=AP%20McFarland&author=SM%20Horner&author=A%20Jarret&author=RC%20Joslyn&author=E%20Bindewald&volume=15&issue=1&publication_year=2014&pages=72-9&pmid=24241692&doi=10.1038/ni.2758&)
