# A pharmacogenetic pilot study reveals MTHFR, DRD3, and MDR1 polymorphisms as biomarker candidates for slow atorvastatin metabolizers

## Metadata
**Authors:** Rafael B R León-Cachón, Jorge A Ascacio-Martínez, María E Gamino-Peña, Ricardo M Cerda-Flores, Irene Meester, Hugo L Gallardo-Blanco, Magdalena Gómez-Silva, Everardo Piñeyro-Garza, Hugo A Barrera-Saldaña
**Journal:** BMC Cancer
**Date:** 2016 Feb 8
**DOI:** [10.1186/s12885-016-2062-2](https://doi.org/10.1186/s12885-016-2062-2)
**PMID:** 26857559
**PMCID:** PMC4746878
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4746878/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC4746878/pdf/12885_2016_Article_2062.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC4746878/pdf/12885_2016_Article_2062.pdf)

## Abstract

**Background:** 
The genetic variation underlying atorvastatin (ATV) pharmacokinetics was evaluated in a Mexican population. Aims of this study were: 1) to reveal the frequency of 87 polymorphisms in 36 genes related to drug metabolism in healthy Mexican volunteers, 2) to evaluate the impact of these polymorphisms on ATV pharmacokinetics, 3) to classify the ATV metabolic phenotypes of healthy volunteers, and 4) to investigate a possible association between genotypes and metabolizer phenotypes.

**Methods:** 
A pharmacokinetic study of ATV (single 80-mg dose) was conducted in 60 healthy male volunteers. ATV plasma concentrations were measured by high-performance liquid chromatography mass spectrometry. Pharmacokinetic parameters were calculated by the non-compartmental method. The polymorphisms were determined with the PHARMAchip® microarray and the TaqMan® probes genotyping assay.

**Results:** 
Three metabolic phenotypes were found in our population: slow, normal, and rapid. Six gene polymorphisms were found to have a significant effect on ATV pharmacokinetics: MTHFR (rs1801133), DRD3 (rs6280), GSTM3 (rs1799735), TNFα (rs1800629), MDR1 (rs1045642), and SLCO1B1 (rs4149056). The combination of MTHFR, DRD3 and MDR1 polymorphisms associated with a slow ATV metabolizer phenotype.

**Conclusion:** 
Further studies using a genetic preselection method and a larger population are needed to confirm these polymorphisms as predictive biomarkers for ATV slow metabolizers.

**Trial registration:** 
Australian New Zealand Clinical Trials Registry: ACTRN12614000851662, date registered: August 8, 2014.

**Electronic supplementary material:** 
The online version of this article (doi:10.1186/s12885-016-2062-2) contains supplementary material, which is available to authorized users.

Keywords: Atorvastatin, Genotype phenotype association, Predictive genetic testing

### Background

The genetic variation underlying atorvastatin (ATV) pharmacokinetics was evaluated in a Mexican population. Aims of this study were: 1) to reveal the frequency of 87 polymorphisms in 36 genes related to drug metabolism in healthy Mexican volunteers, 2) to evaluate the impact of these polymorphisms on ATV pharmacokinetics, 3) to classify the ATV metabolic phenotypes of healthy volunteers, and 4) to investigate a possible association between genotypes and metabolizer phenotypes.

### Methods

A pharmacokinetic study of ATV (single 80-mg dose) was conducted in 60 healthy male volunteers. ATV plasma concentrations were measured by high-performance liquid chromatography mass spectrometry. Pharmacokinetic parameters were calculated by the non-compartmental method. The polymorphisms were determined with the PHARMAchip® microarray and the TaqMan® probes genotyping assay.

### Results

Three metabolic phenotypes were found in our population: slow, normal, and rapid. Six gene polymorphisms were found to have a significant effect on ATV pharmacokinetics: *MTHFR* (rs1801133), *DRD3* (rs6280), *GSTM3* (rs1799735), *TNFα* (rs1800629), *MDR1* (rs1045642), and *SLCO1B1* (rs4149056). The combination of *MTHFR*, *DRD3* and *MDR1* polymorphisms associated with a slow ATV metabolizer phenotype.

### Conclusion

Further studies using a genetic preselection method and a larger population are needed to confirm these polymorphisms as predictive biomarkers for ATV slow metabolizers.

### Trial registration

Australian New Zealand Clinical Trials Registry: [ACTRN12614000851662](http://www.ANZCTR.org.au/ACTRN12614000851662.aspx), date registered: August 8, 2014.

### Electronic supplementary material

The online version of this article (doi:10.1186/s12885-016-2062-2) contains supplementary material, which is available to authorized users.

## Background

The drug atorvastatin (ATV) is widely prescribed to treat hypercholesterolemia, which is a predisposing factor for developing atherosclerosis. ATV, like all statins, acts by inhibiting 3-hydroxy-3-methylglutaryl-coenzyme A reductase, an essential enzyme in cholesterol biosynthesis. ATV reduces the risk of atherosclerosis by lowering the levels of low-density lipoprotein-bound cholesterol [[1](#CR1)]. ATV is administered orally, as a calcium salt (acid form), at a dose that ranges between 10 and 80 mg/day. Once ATV is ingested, several enzymes participate in its metabolism. The enzymes encoded by *CYP3A4* and *CYP3A5* are the most important ones [[2](#CR2)]. These enzymes transform ATV first to its lactone form and subsequently into 2 pharmacologically active metabolites (2-hydroxy-ATV and 4-hydroxy-ATV) [[3](#CR3), [4](#CR4)]. Next, ATV and its secondary metabolites are glucuronidated by uridine diphosphoglucuronosyltransferases (encoded by *UGT1A1* and *UGT1A3*) [[5](#CR5)]. Apart from metabolic enzymes, carrier proteins are involved in ATV metabolism, such as: 1) P-glycoprotein, also known as multidrug resistance protein 1 (MDR1), product of *ABCB1*, 2) the organic anion-transporting polypeptides (OATP1B1 and OATP1B3) encoded by *SCLO1B1* and *SCLO1B3*, respectively [[6](#CR6)–[8](#CR8)], and 3) the breast cancer resistance protein (BCRP2), product of *ABCG2* [[9](#CR9), [10](#CR10)].

Drug performance is evaluated on two main areas: pharmacokinetics and pharmacodynamics. The former focuses primarily on the evaluation of absorption, distribution, metabolism, and excretion processes (ADME) of a drug; consequently, the variation in genes responsible for this process contributes to interindividual variability [[11](#CR11)]. On the other hand, pharmacodynamics evaluates biochemical and physiological effects, as well as the mechanisms of action of a drug, *i.e*. it focuses on the drug response [[12](#CR12)]. The pharmacokinetics of ATV displays high interindividual variability of up to 30 % [[13](#CR13), [14](#CR14)]. This pharmacokinetic discrepancy reveals variations in the ADME processes as the drug passes through the human body [[12](#CR12)]. Factors such as age, gender, ethnicity, and genetic variability are involved in such interindividual differences [[15](#CR15), [16](#CR16)]. Genetic factors are responsible for about 15 to 30 % of the interindividual variation in metabolism, and thus in the response of each patient to certain classes of drugs. However, for some drugs this percentage can increase to up to 95 % [[17](#CR17)]. These genetic factors are mainly due to variations in short tandem repeats, copy number variations, insertions and deletions, and single-nucleotide polymorphisms (SNPs). The latter is the most common source of variation [[18](#CR18)–[20](#CR20)]. The presence of these polymorphisms in genes involved in the metabolism of ATV may explain its pharmacokinetic variability [[17](#CR17)], since the frequency and consequences also vary between different populations [[21](#CR21)]. The aims of this study were: 1) to reveal the frequency of 87 polymorphisms in 36 genes related to drug metabolism in healthy volunteers, 2) to evaluate the impact of these polymorphisms on ATV pharmacokinetics, 3) to classify the metabolic phenotypes for ATV, and 4) to investigate a possible association between genotypes and phenotypes.

## Methods

### Design

A randomized pilot study was carried out in 60 healthy Mexican volunteers to determine ATV pharmacokinetic parameters. A single dose of 80 mg ATV was administered. The clinical study complied with Good Clinical Practice standards, the guidelines of the Declarations of Helsinki and Tokyo, and the Mexican regulations on Bioavailability and Bioequivalence Studies (NOM-177-SSA1-1998) [[22](#CR22)]. Furthermore, the protocol was approved by the Research and Ethics Committee of the pharmacokinetic study center, Ipharma S.A. (Monterrey, Mexico). The clinical study has been registered at the Australian New Zealand Clinical Trials Registry (registration number: ACTRN12614000851662).

### Study population

Sixty four healthy male candidates from northeastern Mexico were recruited and a written informed consent was obtained. Inclusion criteria were: non-smoker, 18-to-45-year old, weight ≥ 50 kg, body mass index (BMI) of 20–26 kg/m^2^, availability for completing the study, being healthy. Since ATV is classified as a pregnancy category X drug, only males were considered for the study. Candidates were excluded for: any abnormal lab result, significant personal or family medical history of angioedema o allergies, the existence of concurrent disease, use of prescription or over-the-counter medication or alcohol before enrollment, history of smoking, alcohol or drug abuse, and incompliance or non-willingness to complete the study. Four candidates were excluded because of the consumption of alcohol and/or substances or an altered blood pressure. The health status of the volunteers was confirmed by a medical history, a physical examination, an electrocardiogram (ECG), laboratory tests (blood count, blood chemistry, liver function tests, and urinalysis), and seronegativity for human immunodeficiency virus and hepatitis B and C viruses.

### Drug administration and sampling

After an overnight fast at the study center (Ipharma, S. A.), each subject was given a single dose of 80 mg of ATV-coated tablets (Pfizer Pharmaceuticals LLC, Caguas Site, Caguas, PR). The volunteers were under direct medical supervision at the study site. Venous blood (4 mL) was collected in K_2_EDTA-coated Vacutainers^TM^ (BD Diagnostics, Franklin Lakes, NJ, US), before ATV administration (time 0), and at the following time points after drug administration: 0.25, 0.5, 0.75, 1, 1.5, 2, 2.5, 3, 3.5, 4, 5, 6, 8, 12, 24, 36 and 48 h. Plasma was separated by centrifugation (15 min at 1600 g at 4 °C) and stored in cryovials at −80 °C until analysis, using a method validated by Ipharma S. A. [[3](#CR3), [23](#CR23), [24](#CR24)].

### Pharmacogenetic tests

Leukocytes were obtained from the buffy coat, and genomic DNA was extracted by the alkaline lysis method [[25](#CR25)]. Seven multiplex polymerase chain reactions (PCRs) amplified the desired gene regions, following a validated protocol [[26](#CR26)]. Screening for gene polymorphisms was performed using the PHARMAchip® microarray (Progenika, Derio, ES). This pharmacogenetic genotyping device detects, with a 99.9 % specificity and sensitivity, 85 gene polymorphisms in 34 genes involved in drug metabolism and response, including those encoding cytochrome P450 enzymes, phase II metabolism enzymes, receptors, and transporters. Amplified products were fractionated with DNAse according to a validated protocol [[26](#CR26)], followed by fluorescent labeling and hybridization of the microarray, in an automated TECAN HS4800PRO platform (Ventana Medical Systems Inc., Tucson, AZ, US). The hybridization pattern was revealed using the Innoscan 710 scanner (Innopsys S.A., Carbonne, FR). Polymorphic variants were determined using PHARMAchip software V.3.2.9 [[26](#CR26)]. Two additional polymorphisms not included in the PHARMAchip, rs2231142 (C__15854163_70) and rs4149056 (C__30633906_10), in *ABCG2* and *SLCO1B1* respectively, were included in the study and analyzed by Real-Time PCR system using validated Genotyping Assays (Applied Biosystems, Foster City, CA, US) according to the manufacturer’s instructions. Typed polymorphisms were only includedin subsequent association studies after having passed three quality control tests: the genotype call rate (>0.90 completeness to obtain 99.8 % accuracy), the Hardy-Weinberg equilibrium (HWE) test (*P*-value > 0.05), and the minor allele frequency (MAF) criterion (>0.01).

### Determination of ATV calcium in plasma

Proteins were eliminated from the plasma samples by adding 4 volumes of acetonitrile to 100-μL samples, vortexing (70 rpm, 4 min.), and precipitating by centrifugation (9600 g, 10 min., 10 °C). Protein-free supernatant (300 μL) was recovered and 5-μL samples were injected into an Agilent 1100 high-performance liquid chromatographer (HPLC; equipped with an autosampler and a binary pump), which was connected to an Agilent 6410 tandem mass spectrometer (MS/MS) with a triple quadrupole detector (Agilent Technologies, Santa Clara, CA, US) to measure ATV calcium levels. A C_18_ pre-column and a Synergi^TM^ Fusion-RP column (4 μm, 80 Å, 50 × 2 mm; Phenomenex, Torrance, CA, US) formed the solid phase, whereas the mobile phase consisted of 0.03 % formic acid/70 % acetonitrile in analytical grade water. The column temperature was 40 °C, the flow rate 0.4 mL/min, and the auto-sampler temperature 20 °C. The detection system used an ESI MS/MS precursor ion (+) 559.3 m/z and a product ion (+) 440.3 m/z. Under these conditions, interday linearity was assessed by performing calibration curves from 0.5 to 100 ng/mL (0.5, 2.5, 5, 10, 25, 50 and 100); intraday quality control was evaluated by using eight ATV control samples of 1.7, 7.5, 35, and 75 ng/mL each.

### Pharmacokinetic analysis

WinNonlin® professional software V.5.3 (Pharsight Corp., Mountain View, CA, US) was used for pharmacokinetic analysis. The maximum plasma concentration (C_max_) and the time to reach C_max_ (T_max_) were calculated from the observed concentration-time data in plasma. Pharmacokinetic parameters were estimated with the non-compartmental method after oral administration of a single dose of ATV and were as follows: 1) the area under the plasma concentration-time curve from time 0 to the time of the last measurement (AUC_0-t_), calculated using the logarithmic-linear trapezoidal rule, 2) the area under the curve from time 0 to the time extrapolated to infinity (AUC_0-∞_), 3) the apparent clearance of the fraction dose absorbed (Cl/F), 4) the elimination rate constant in the terminal phase (Ke), and 5) the half-life in the terminal phase of the drug (T_1/2_).

### Statistical analyses

For sample size calculation, it was assumed that the coefficient of variation (CV) was 45 % for the C_max_ and AUC of ATV. Considering a significance level of 5 %, a minimum power of 80 %, an Ω of 0.25, and a confidence interval of 90 %, a sample size of 58 would suffice. The metabolizer phenotypes classification was made using a multivariate analysis of the combined pharmacokinetics parameters C_max_ and AUC_0-t_. To minimize the effect of scale differences, before calculating the distance matrix, these variables were standardized. Next, the individual values of C_max_ and AUC_0-t_ were subjected to hierarchical cluster analysis (HCA) using the Ward linkage method and the interindividual Manhattan distances were computed. The standardization, HCA, and the hierarchical clustering dendogram were made using Minitab 16 demo software (Minitab Inc., State College, PA, US) [[27](#CR27)]. We identified the participants of each cluster and calculated the geometric means of all pharmacokinetic parameters of each cluster. According to the geometric means of the pharmacokinetic parameters of the clusters they were classified into metabolizer phenotypes. Next, one-way ANOVA and the Kruskal-Wallis H test were used to validate the classification model. The HWE was determined by comparing the genotype frequencies with the expected values using the maximum likelihood method [[28](#CR28)]. To detect significant differences between 2 groups, Student’s *t*-test or the Mann Whitney *U* test were used for parametric and non-parametric distributions, respectively. Differences between more than 2 groups were assessed by one-way ANOVA or the Kruskal-Wallis H test for parametric and non-parametric distributions, respectively. Post hoc tests (LSD and Tamhane’s T2) were used for pairwise comparisons. To evaluate the contribution of genetic factors to the variability of the pharmacokinetic parameters linear regression analysis was done. Possible associations between genotypes or genotype combinations and phenotypes were assesed using contingency tables *Χ*^2^ statistics and Fisher’s exact tests. The linear regression analysis and association studies were performed under three different models (dominant, recessive, and additive). Odds ratios were estimated with 95 % confidence intervals. The model for prediction was confirmed using stepwise multiple linear regression analysis. Aforementioned analyses were performed with SPSS for Windows, V.20 (IBM Corp., Armonk, NY, US). All *P*-values were two-tailed. The corrected *P* (Pc) values were adjusted according to Bonferroni’s correction for multiple comparisons and the Benajmini-Hochberg procedure was applied to exclude spurious associations [[29](#CR29)]. A *P*-value ≤ 0.05 was considered statistically significant.

## Results

### Study population

Sixty male subjects completed the study. Volunteers were of mestizo descent, most of them students (73 %) from the state of Nuevo Leon (83 %). Other demographic characteristics did not display significant variability (Table [1](#Tab1)).

### Table 1.

|   | Gender | BMI (kg/m2) | BS (m2) | Age (years) | Weight (kg) | Height (m) | n |
| --- | --- | --- | --- | --- | --- | --- | --- |
| Mean | M | 23.43 | 1.84 | 24.01 | 70.58 | 1.73 | 60 |
| SD |   | 1.64 | 0.131 | 4.35 | 8.24 | 0.065 |   |

Table 1 Caption: Demographic data of volunteers

### ATV pharmacokinetics

Despite controlling physiological and environmental conditions the pharmacokinetic parameters were highly variable (Table [2](#Tab2)). The geometric mean ± SD for the pharmacokinetic parameters obtained were: C_max_ = 41.44 ± 23.35 ng/mL, AUC_0-t_ = 141.88 ± 86.78 ng/mL*h, AUC_0-∞_ = 157.12 ± 87.24 ng/mL*hr, Cl/F = 509.20 ± 265.57 L/h, T_1/2_ = 9.81 ± 6.58 h, and the Ke = 0.071 ± 0.035.

### Table 2.

|   | All subjects | Phenotype |  |  |
| --- | --- | --- | --- | --- |
| Parameters |   | Slow | Normal | Rapid |
| N | 60 | 18 | 25 | 17 |
| Cmax (ng/mL) | 41.44 ± 23.35 | 75.39 ± 15.74* | 40.48 ± 6.37* | 22.11 ± 8.15* |
| AUC0-t (ng/mL*h) | 141.88 ± 86.78 | 218.14 ± 101.90* | 152.69 ± 48.83* | 80.78 ± 27.24* |
| AUC0-∞ (ng/mL*h) | 157.12 ± 87.24 | 231.55 ± 100.02* | 166.35 ± 55.59* | 95.81 ± 28.38* |
| Cl/F (L/h) | 509.20 ± 265.57 | 345.49 ± 119.94* | 480.93 ± 152.07* | 834.97 ± 249.52* |
| T1/2 (h) | 9.81 ± 6.58 | 9.64 ± 6.18 | 10.76 ± 7.54 | 8.71 ± 5.43 |
| Ke | 0.071 ± 0.035 | 0.072 ± 0.041* | 0.064 ± 0.030* | *0.080 ± 0.035 |

Table 2 Caption: Pharmacokinetic parameters according to metabolizer phenotype

### Classification of metabolizer phenotypes

C_max_ and AUC_0-t_ were used for HCA classification of pharmacokinetic profiles, because C_max_ tends to best reveal differences in pharmacokinetic profiles and AUC_0-t_ is considered to be the best parameter to evaluate a drug’s interindividual pharmacokinetic variation [[30](#CR30)]. The HCA, based on centroid distance, revealed three main clusters. Which we identified as slow metabolizers (30.00 %), normal metabolizers (41.66 %), and rapid metabolizers (28.33 %), as shown in Fig. [1](#Fig1). The geometric means of the pharmacokinetics parameters were significantly different among the three clusters (*P* ≤ 0.016), except for T_1/2_ (Table [2](#Tab2)). The mean concentration-time profile and the geometric mean pharmacokinetic parameters of ATV obtained for each metabolizer phenotype are shown in Fig. [2a](#Fig2) and Table [2](#Tab2), respectively. We observed a > 9-fold difference in ATV pharmacokinetic paratmeters between the fastest metabolizer indivual (C_max_ = 10.94 ng/mL and AUC_0-t_ = 55.23 ng/mL*h) and the slowest metabolizer individual (C_max_ = 101.85 ng/mL and AUC_0-t_ = 454.41 ng/mL*h). The distribution of phenotypes regarding C_max_-AUC_0-t_ values, are presented in Fig. [2b](#Fig2).

### Fig. 1.

![Fig. 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/340c/4746878/1398c7049f1c/12885_2016_2062_Fig1_HTML.jpg)

Classification of ATV metabolic phenotypes. A dendrogram generated with the Manhattan distance and Ward’s linkage method illustrates rapid metabolizers (red), normal metabolizers (green), and slow metabolizers (blue)

### Fig. 2.

![Fig. 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/340c/4746878/6fb3b54dacc3/12885_2016_2062_Fig2_HTML.jpg)

Distinctive pharmacokinetic profiles of rapid, normal and slow metabolizers. a Mean peak plasma ATV concentration-time curves after a single 80-mg dose of ATV of the three metabolizer phenotypes. Data shown are mean ± standard error (SE) concentrations. b Distributions of metabolizer phenotypes with regard to Cmax-AUC0>-t values. For both a xand b: rapid metabolizers in red, normal in green, and slow ones in blue

### Pharmacogenetic tests

The allele and genotype frequencies of the gene polymorphisms with a potential impact on drug metabolism are presented in Additional file [1](#MOESM1). The HWE applied to most of the gene polymorphisms, with the exception of the gene deletions of *GSTM1* and *GSTT1*, because the heterozygous variants were not detected [[31](#CR31)]. The polymorphism rs1800896 in the *IL10* was not in HWE equilibrium. The polymorphisms in *CYP2D6*, *NAT2*, *TPMT*, and *TYMS* were below the call rate threshold of 0.9. The SNPs in *DPYD*, rs1799807 in *BCHE*, and rs28399504 and rs41291556 in *CYP2C19* had a MAF < 0.01. The aforementioned polymorphisms were excluded from subsequent analyses; a total of 30 SNPs remained for statistical analysis.

### Association between gene polymorphisms and ATV pharmacokinetics

The various pharmacokinetic parameters were affected differentially by the different genetic *loci*;*i.e*. a certain polymorphism had an effect on C_max_ but not on AUC_0-t_ or the other way round, while there were also polymorphisms that affected both and/or other parameters (Table [3](#Tab3)). The effect of *MTHFR*-rs1801133 on C_max_ was statistically significant. Heterozygous variant (C/T) carriers and homozygous variant (T/T) carriers had lower C_max_ values compared to homozygous wild-type (C/C) carriers (*P* = 0.018 and 0.004, respectively). Carriers of the variant genotype (C/T or T/T) showed significant lower values of C_max_ (*P* = 0.006), AUC_0-t_ (*P* = 0.050) and AUC_0-∞_ (*P* = 0.044) but statistically significant higher values of Cl/F (*P* = 0.044) as compared to homozygous wild-type subjects (C/C). The genotypes resulting from the *DRD3*-rs6280 (Ser9Gly) polymorphism had a significant impact on ATV pharmacokinetics. First, the homozygous wild-type (C/C) carriers had lower T_1/2_ values (*P* = 0.003) and higher Ke values (*P* = 0.008) as compared to homozygous variant (T/T) carriers. Second, when comparing T/T with (C/T) genotypes, there were significant differences for AUC_0-t_, AUC_0-∞_, Cl/F, T_1/2_ and Ke values (*P* were 0.027, 0.024, 0.024, 0.027, and 0.041, respectively). Third, the presence of the wild-type allele, combination (C/C + C/T), had a significant influence on all pharmacokinetics parameters (*P* values: C_max_ = 0.050, AUC_0-t_ = 0.026, AUC_0-∞_ = 0.016, Cl/F = 0.016, T_1/2_ = 0.004, and Ke = 0.007). *GSTM3*- rs1799735 had a significant effect on AUC_0-∞_ and Cl/F when *A/*A and *A/*B were compared (*P* = 0.041 for both parameters). Regarding *TNF*-rs1800629, the A allele carriers had significantly higher AUC_0-t_, AUC_0-∞_, and T_1/2_ values (*P* = 0.035, 0.030, and 0.025, respectively) and lower Cl/F and Ke values (*P* = 0.030 and 0.025). The homozygous variant (A/A) carrier was not found. MDR1 (*ABCB1*) and OATP1B1 (*SLCO1B1*) were the only ATV transport-related genes with an effect on ATV pharmacokinetics. The variant allele (T) of *MDR1*-rs1045642 produced a significant increase of C_max_ (*P* = 0.037) when the combination of the heterozygous and homozygous variant (C/T + T/T) is compared with the homozygous wild-type (C/C). The C allele of the *SLCO1B1*-rs4149056 polymorphism significantly affected AUC_0-t_, AUC_0-∞_, and Cl/F values (*P* = 0.004 for all three parameters) in homozygous wild-type allele and heterozygous (C/C + C/T) carriers. None of the other 24 polymorphisms tested had a significant impact on ATV pharmacokinetics. The influences of polymorphisms on the ATV pharmacokinetics are shown in Table [3](#Tab3).

### Table 3.

|   |   | Pharmacokinetics parameters |  |  |  |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- |
| Genotypes | N | Cmax (ng/mL) | AUC0-t (ng/mL*h) | AUC0-∞ (ng/mL*h) | Cl/F (L/h) | T1/2 (h) | Ke |
| MTHFR rs1801133 |  |  |  |  |  |  |  |
| C/C | 14 | 60.46 ± 20.60 | 195.77 ± 91.84 | 213.25 ± 94.16 | 448.86 ± 202.58 | 12.95 ± 9.30 | 0.07 ± 0.04 |
| C/T | 37 | 44.81 ± 24.32* | 152.83 ± 87.67 | 166.43 ± 86.52 | 595.20 ± 274.47 | 10.78 ± 5.84 | 0.08 ± 0.04 |
| T/T | 9 | 34.89 ± 12.90** | 141.64 ± 66.29 | 151.57 ± 67.28 | 623.70 ± 287.68 | 9.94 ± 4.19 | 0.08 ± 0.03 |
| C/T + T/T | 46 | 42.87 ± 22.77§ | 150.64 ± 83.37§ | 163.52 ± 82.64§ | 600.77 ± 274.06§ | 10.62 ± 5.52 | 0.08 ± 0.03 |
| DRD3 rs6280 |  |  |  |  |  |  |  |
| C/C | 17 | 39.90 ± 14.47 | 149.50 ± 59.86 | 159.43 ± 57.99 | 562.72 ± 200.54 | 8.63 ± 2.33¢ | 0.08 ± 0.02¢ |
| C/T | 26 | 44.12 ± 23.77 | 143.60 ± 85.50 | 156.77 ± 86.61 | 654.34 ± 316.38 | 10.67 ± 6.65 | 0.09 ± 0.04 |
| T/T | 17 | 58.41 ± 26.70# | 199.72 ± 102.65§§, # | 218.89 ± 100.70§§, # | 431.78 ± 177.83§§, # | 14.44 ± 8.15§§, # | 0.06 ± 0.03§§, # |
| C/C + C/T | 43 | 42.45 ± 20.51 | 145.93 ± 75.66 | 157.83 ± 75.82 | 618.12 ± 277.41 | 9.86 ± 5.43 | 0.09 ± 0.04 |
| GSTM3 rs1799735 |  |  |  |  |  |  |  |
| *A/*A | 55 | 44.71 ± 21.36 | 158.00 ± 89.57 | 171.60 ± 90.03 | 582.27 ± 270.90 | 10.92 ± 6.24 | 0.08 ± 0.03 |
| *A/*B | 5 | 71.92 ± 32.15 | 196.08 ± 33.24 | 213.93 ± 28.26¥ | 378.87 ± 46.77¥ | 13.82 ± 10.17 | 0.08 ± 0.07 |
| TNF rs1800629 |  |  |  |  |  |  |  |
| G/G | 51 | 45.54 ± 22.80 | 153.18 ± 82.16 | 167.14 ± 83.12 | 586.66 ± 266.42 | 10.59 ± 6.14 | 0.08 ± 0.04 |
| G/A | 9 | 55.88 ± 26.94 | 215.85 ± 96.68€ | 230.57 ± 93.37€ | 391.55 ± 132.91€ | 14.66 ± 8.45€ | 0.06 ± 0.02€ |
| MDR1 rs1045642 |  |  |  |  |  |  |  |
| C/C | 13 | 33.43 ± 13.40 | 135.68 ± 78.72 | 150.53 ± 79.10 | 699.77 ± 376.59 | 10.87 ± 4.99 | 0.08 ± 0.04 |
| C/T + T/T | 47 | 50.78 ± 24.21¤ | 168.22 ± 88.36 | 181.93 ± 88.95 | 528.14 ± 216.66 | 11.24 ± 7.00 | 0.08 ± 0.03 |
| SLCO1B1 rs4149056 |  |  |  |  |  |  |  |
| C/C + C/T | 11 | 53.92 ± 24.72 | 222.27 ± 91.48 | 241.23 ± 92.98 | 390.51 ± 195.46 | 12.42 ± 8.59 | 0.07 ± 0.03 |
| T/T | 49 | 45.41 ± 23.01 | 147.45 ± 80.40≠ | 160.29 ± 79.55≠ | 604.57 ± 264.87≠ | 10.88 ± 6.12 | 0.08 ± 0.04 |
| P-values of genotype combinations on ATV pharmacokinetics |  |  |  |  |  |  |  |
| Clusters |   | Cmax | AUC0-t | AUC0-∞ | Cl/F | T1/2 | Ke |
| A vs. B |   | 0.163 | 0.060 | 0.041£ | 0.041£ | 0.519 | 0.519 |
| B vs. C |   | 0.007£ | 4 × 10−4£ | 0.001£ | 0.001£ | 0.177 | 0.177 |
| A vs. C |   | 0.016£ | 0.011£ | 0.011£ | 0.011£ | 0.181 | 0.181 |
| C vs. A + B |   | 0.001£ | 9.1 × 10−5£ | 1.31 × 10−4£ | 9.5 × 10−5£ | 0.112 | 0.399 |
| D vs. E |   | 0.187 | 0.060 | 0.041£ | 0.002£ | 0.610 | 0.486 |
| E vs. F |   | 2.3 × 10−4£ | 0.001£ | 0.001£ | 0.004£ | 0.241 | 0.260 |
| D vs. F |   | 1 × 10−3£ | 0.014£ | 0.011£ | 6 × 10−6£ | 0.230 | 0.177 |
| F vs. D + E |   | 5.3 × 10−5£ | 3.61 × 10−4£ | 4.22 × 10−4£ | 1.35 × 10−4£ | 0.115 | 0.176 |

Table 3 Caption: Polymorphisms and genotype clusters with significant effect on ATV pharmacokinetics

### Association between genotypes and metabolizer phenotypes

No individual genotype correlated with any metabolizer phenotype after Bonferroni’s correction for multiple testing. However, MDR1-rs1045642 behavior was remarkable in this aspect, as no homozygous wild-type (C/C) was a slow metabolizer.

Of the six polymorphisms with an effect on ATV pharmacokinetics, three polymorphisms associated with the slow metabolizer phenotype considering genetic models. The C/T or T/T genotype of *MTHFR*-rs1801133, the T/T genotype of *DRD3*-rs6280, and the C/T or T/T genotype of *MDR1*-rs1045642 were significantly associated with slow metabolizer phenotype using dominant, recessive, and dominant models, respectively. This association remained statistically significant after adjusting for multiple testing using Bonferroni’s correction (*P* < 0.05; Table [4](#Tab4)).

### Table 4.

| Association values between genotypes and the slow metabolizer phenotype using dominant, recessive and additive models |  |  |  |  |  |
| --- | --- | --- | --- | --- | --- |
| Gene | Polymorphism | Model | OR (95 % CI) | P-Value | Pc Value |
| MTHFR | rs1801133 | Dominant (C/C vs. C/T + T/T) | C/T + T/T = 0.64 (0.42–0.99)* | 0.011 | 0.028** |
|   |   |   | C/C = 3.11 (1.26–7.68)* |   |   |
| DRD3 | rs6280 | Recessive (C/C + C/T vs. T/T) | C/C + C/T = 0.62 (0.38–1.00) | 0.015 | 0.034** |
|   |   |   | T /T = 2.63 (1.21–5.70)* |   |   |
| GSTM3 | rs1799735 | Additive (*A/*A vs. *A/*B) | *A/*A = 0.88 (0.70–1.09) | 0.126 | 0.308 |
|   |   |   | *A/*B = 3.50 (0.64–19.20) |   |   |
| TNF | rs1800629 | Additive (G/G vs. G/A) | G/G = 0.89 (0.68–1.16) | 0.324 | 0.553 |
|   |   |   | G/A = 1.82 (0.55–6.00) |   |   |
| MDR1 | rs1045642 | Dominant (C/C vs. C/T + T/T) | C/T + T/T = 1.45 (1.18–1.77)* | 0.008 | 0.020** |
|   |   |   | C/C = 0.06 (0.003–1.00) |   |   |
| SLCO1B1 | rs4149056 | Recessive (C/C + C/T vs. T/T) | C/C + C/T = 1.33 (0.45–3.99) | 0.610 | 0.884 |
|   |   |   | T/T = 0.93 (0.70–1.23) |   |   |

Table 4 Caption: Association values between genotypes and metabolizer phenotypes

Linear regression analysis using aforementioned genetic models confirmed that these five polymorphisms affected the variability of pharmacokinetic parameters of ATV, except for the *TNF*-rs1800629 polymorphism (Additional file [2](#MOESM2)).

Next, we analyzed genotype combinations of the six polymorphisms that individually had a significant effect on ATV pharmacokinetic parameters (Table [3](#Tab3) and Additional file [3](#MOESM3)): cluster A (subjects with genotypes related to normal metabolism), cluster B (subjects with only 1 genotype related to decreased metabolism), and cluster C (subjects with 2 or more genotypes related to decreased metabolism). As the *MTHFR*, *MDR1*, and *DRD3* genotypes effected pharmacokinetics most, clusters that only considered these genes were formed: Cluster D (subjects with *MTHFR*, *MDR1* and *DRD3* genotypes related to normal metabolism), cluster E (subjects in which either *MTHFR*, *MDR1* or *DRD3* gentoype related to decreased metabolism), and cluster F (subjects in which all *MTHFR*, *MDR1* and *DRD3* genotypes were related to decreased metabolism). The analysis of genotype combinations revealed that cluster C, *i.e*. subjects with 2 or more genotypes related to decreased metabolism, had a significant higher C_max_ (*P* ≤ 0.016), AUC_0-t_ (*P* ≤ 0.011) and AUC_0-∞_ (*P* = 0.011), but significantly lower Cl/F values (*P* = 0.011) when compared with cluster A and B. The genotype combination analysis limited to *MTHFR*, *MDR1* and *DRD3* showed that cluster F was significantly different from clusters D and E; a higher C_max_ (*P* = 5.3 × 10^−5^), AUC_0-t_ (*P* = 3.61 × 10^−4^), and AUC_0-∞_ = (4.22 × 10^−4^), but lower Cl/F (*P* = 1.35 × 10^−4^). The influences of the clusters on ATV pharmacokinetics are shown in Table [3](#Tab3).

The association analysis between clusters and phenotypes displayed a mutual dependency and association (*P* ≤ 0.05). The C and F cluster were associated with slow metabolizers as shown in Table [5](#Tab5). The stepwise multiple regression analysis showed that the combination of *MTHFR*, *DRD3* and *MDR1* polymorphisms are related to ATV slow metabolizers. The combination of these three polymorphisms contributed to the pharmacokinetic variability prediction with an R^2^ = 0.295, and adjusted R^2^ = 0.257 with a *P* = 2.26 × 10^−4^.

### Table 5.

| Association between genotype combinations and metabolizer phenotypes |  |  |  |  |
| --- | --- | --- | --- | --- |
| Clusters | Phenotypes | OR (95 % CI) | P-Value | Pc Value |
| A, B, C | Rapid, Normal, Slow | - | 1 × 10−3** | 3.1 × 10−3** |
| C, A + B | Slow, Normal + Rapid | 4.00 (1.39–11.49)* | 3.83 × 10−4** | 1 × 10−3** |
| D, E, F | Rapid, Normal, Slow | - | 6.6 × 10−5** | 1.6 × 10−3** |
| F, D + E | Slow, Normal + Rapid | 4.53 (1.62–12.68)* | 7 × 10−6** | 2.9 × 10−5** |

Table 5 Caption: Association values between genotypes and metabolizer phenotypes

### Adverse effects

ATV was well tolerated by all subjects, since no volunteer showed any adverse effects during and at the end of the pharmacokinetic study. No clinically significant changes from baseline were observed in the physical examination or the ECG during the study, and no clinically significant mean changes from baseline were observed for any laboratory parameters.

## Discussion

Numerous studies have been performed to better characterize the high variability in ATV pharmacokinetic parameters. In this study, the maximum and minimum C_max_ and AUC_0-t_ values differed by approximately 10-folds in 60 healthy volunteers. Other studies even observed 15-folds for the C_max_ and 12-folds for AUC, after a single dosis of ATV [[32](#CR32)]. The interindividual variability exists even though the study subjects are under controlled conditions. Actually, the search of prediction biomarkers for disease risk and the response to treatment is an area of research with great activity. However, so far no pharmacogenetic testing in clinical studies have been carried out in Mexicans. To contribute to the identification of the genetic architecture underlying the drug metabolism and response in the Mexican population, we examined the impact of 30 polymorphisms in genes related to drug metabolism and response on ATV pharmacokinetics.

In this study, we propose a novel and simple approach to classify the metabolizer phenotypes from the analysis of pharmacokinetic profiles. This approach uses C_max_, which reflects the absorption rate, and the AUC_0-t_) that reflects the extent of absorption and clearance. We distinguished three different metabolic phenotypes (slow, normal, and rapid) with significant differences for pharmacokinetic parameters (Table [2](#Tab2)). The slow phenotype displayed the highest variability in pharmacokinetic parametes, clearly illustrated by the greater dispersion of C_max_-AUC_0-t_ valuesof the slow metabolizers as compared to the normal and rapid ones. T_1/2_ behaves stochastically, and its variance increased with the time, which may be the reason that there is no significant T_1/2_ difference among metabolizer phenotypes. Our phenotype classification is consistent with the one reported by Quing Huang *et al*., who used a pharmacometabonomic approach to classify 48 healthy volunteers as low, medium, and high ATV metabolizers [[32](#CR32)]. Slow metabolizers tend to have higher plasma drug levels and to be more susceptible to adverse side effects. On the other hand, rapid metabolizers tend to have lower plasma drug levels, which may explain a poor drug response. Our results support the use of this classification method for *in vivo* studies.

There was a significant effect of six gene polymorphisms on different pharmacokinetics parameters (*P* ≤ 0.05). T allele carriers of *MTHFR*-rs1801133 had a lower C_max_ and AUC but increased Cl/F. This pharmacokinetic profile is consistent with increased clearance activity and a lower ATV concentration in the body, which may lead to a weaker response to ATV.

To our knowledge, our study is the first that reports an effect of the *MTHFR*-rs1801133 polymorphism on statin pharmacokinetics, and is consistent with previous studies on drug response and cardiovascular disease susceptibility. In 2008, Maitland-van der Zee *et al*. found that the *MTHFR*-rs1801133 C/C genotype protects against coronary heart disease in different populations [[33](#CR33)]. Another study reported that the C/C genotype protected against cardiovascular disease in a Turkish population under statin therapy [[34](#CR34)]. The C allele frequencies reported in both studies [[33](#CR33), [34](#CR34)] are different from the allele frequencies found in the Mexican population. The *MTHFR*-rs1801133 polymorphism (C677T) causes an Ala → Val substitution which decreases enzyme activity leading to increased homocysteine levels. Hyperhomocysteinemia is a known risk factor for cardiovascular disease [[34](#CR34)]. How the T variant of *MTHFR*-rs1801133 augments ATV clearance, so that the drug response is diminished, remains to be elucidated.

With respect to the *DRD3*-rs6280 (Ser9Gly) polymorphism, the homozygous variant genotype (T/T) affected the pharmacokinetic parameters consistent with a slow metabolizer phenotype. So far, the influence of *DRD3*-rs6280 on ATV pharmacokinetics has not been reported. *DRD3* encodes a dopamine receptor and is functionally related to reward stimuli and control of movement [[35](#CR35)]. *DRD3*-rs6280 polymorphisms have been mainly related to addictive behavior[[36](#CR36), [37](#CR37)] and involuntary movements [[38](#CR38)]. The C allele is related to a stronger intracellular response to dopamine [[35](#CR35)]. However, another *DRD3* polymorphism (rs1486012) has been associated with a decrease in lopinavir/ritonavir elimination [[39](#CR39)].

Subjects that carry the *B allele of *GSTM3*-rs1799735 had a decreased clearance of ATV and therefore an increased AUC_0-∞_. These results are not consistent with the higher detoxification activity associated with the *B allele [[40](#CR40)] of this gene which encodes a glutathione S-transferase M3 that conjugates glutathione with substrates like drugs, toxins, and carcinogens. The *B allele represents a 3-bp deletion in intron 6, which generates a recognition sequence for the Ying Yang transcription factor (YY1), and thus alters the gene expression of *GSTM3*. We found no previous report relating this polymorphism to statin metabolism. The reason for the lack of association of the *GSTM3* polymorphism with the metabolic phenotypes may be due to the low frequency of the *B allele in our relatively small study group. *GSTM3*-rs1799735 has been associated with various types of cancers with different effects [[40](#CR40)–[42](#CR42)]. Allele frequencies found for rs1799735 are similar to those reported by Jain *et al*. [[42](#CR42)]. Nevertheless, the homozygous variant (*B/*B) carrier was absent in our sample.

Heterozygous A allele carriers of *TNF*-rs1800629 had pharmacokinetic parameters that were consistent with a diminished clearance of ATV. Nevertheless, their influence on ATV pharmacokinetic variability was not confirmed by linear regression analysis. The rs1800629 (G/A) polymorphism is located in the promoter region of *TNF. TNF* encodes the pro-inflammatory cytokine tumor necrosis factor alpha. The variant A allele increases protein expression; therefore the wild-type G allele has a protective effect by reducing the risk of thrombosis in patients with hemodialysis [[43](#CR43)]. It is unknown how *TNF* is involved in ATV metabolism, but there is evidence that ATV may attenuate *TNF* expression [[44](#CR44)], which results in a reduction of the inflammatory process. The allele and genotype frequencies in our study group are similar to those of a Turkish population [[43](#CR43)].

Previous studies have reported that the *MDR1*-rs1045642 polymorphism affects the response to ATV treatment. However, different studies disagree with respect to which homozygous genotype results in a better response to therapy [[45](#CR45)–[47](#CR47)]. *MDR1* encodes a P-glycoprotein transporter, that functions as an ATP-dependent efflux pump and thus protects against harmful substances. *MDR1* is widely expressed, for example in the small intestine, the blood–brain barrier, hepatocytes, and kidney proximal tubules [[48](#CR48), [49](#CR49)]. The *MDR1*-rs1045642 (C3435T) polymorphism is located in exon 26 of the *MDR1*/*ABCB1* at a wobble position that does not produce an amino acid change. However, Hoffmeyer *et al*. [[48](#CR48)] have reported that subjects homozygous for the variant (T/T) not only had reduced expression of *MDR1* but also higher drug plasma levels as compared to subjects homozygous for the wild-type (C/C) [[48](#CR48)]. In our study, carriers of the variant T allele, either homozygous or heterozygous, had a higher C_max_. Similar results were found by Zhou *et al*. [[50](#CR50)] and Gonzalez-Vacarezza *et al*. [[51](#CR51)] on fluvastatin and quetiapine pharmacokinetics, respectively. The T allele and genotype frequencies found in our study group were similar to those reported in the Lahu population (0.54) [[52](#CR52)], the Dutch population (0.52) [[53](#CR53)], and in Caucasians (0.53) [[54](#CR54)]; however, they differ from those found in the Chilean population (0.34) [[16](#CR16)], the African population (0.17), and in African-Americans (0.39) [[55](#CR55)].

Similar to *MDR1* polymorphisms, there are conflicting reports about the response and adverse drug reactions to statins in subjects with the *SLCO1B1*-rs4149056 (SLCO1B1*5) polymorphism [[47](#CR47), [56](#CR56)]. *SLCO1B1*, which is highly expressed in the liver, encodes an organic anion influx pump for numerous compounds. The *SLCO1B1*-rs4149056 (c.T521C) polymorphism results in the substitution of alanine for valine at amino acid residue 174, which reduces transport activity and leads to higher circulating statin concentrations that could be responsible for the reported adverse effects [[57](#CR57)]. Our results support that rs4149056 affects ATV pharmacokinetics. However, the polymorphism did not associate with metabolic phenotypes. This could be due to the low frequency of C allele carriers in our sample. The variant T allele is the most frequent in Mexican population.

In summary, in our study we found six polymorphisms in different genes that have a significant effect on the pharmacokinetics of ATV. GSTM3 is a phase II metabolizing enzyme, which explains its impact on clearing parameters (Cl/F), and as a consequence AUC. *MDR1* and *SLCO1B1* are transporters, and this function may explain their impacton ATV pharmacokinetic parameters (C_max_, AUC, and Cl/F). However, the significant impact of the *DRD3*, *MTHFR*, and *TNF* polymorphisms on ATV C_max_, AUC’s, Cl/F T_1/2,_ and Ke is achieved by so far unknown mechanisms.

The value of a pharmacokinetic parameter is the end result of a complex ADME process that involves many proteins. Polymorphisms in the different encoding genes may neutralize each other, which makes it highly unlikely that a single polymorphism determines a metabolizer phenotype. However, a slow metabolizer phenotype may be the result of various polymorphisms that reinforce a certain impact. Indeed the accumulation of polymorphisms of the six genes (*MTHFR*, *GSTM3*, *DRD3*, *TNF*, *MDR1*, and *SLCO1B1*) coincided with a shift from rapid metabolizers (no genotypes related to decreased metabolism in cluster A) to slow metabolizers (cluster C, up to six polymorphisms). *MTHFR*, *MDR1* and *DRD3* polymorphisms seem to have a leading impact on metabolizer phenotype, and were sufficient to identify slow metabolizers in our study group. For example, cluster D, comprised of *MTHFR*, *MDR1* and *DRD3* wild-type genotypes, were rapid metabolizers; cluster E is a mixture of the rapid and normal metabolizers, whereas the accumulation of these three polymorphisms (cluster F) is sufficient to identify slow metabolizers. These results were confirmed by association tests with different models, and stepwise multiple regression analysis. The interference of other genes in the A and B clusters may explain the lack of association with the rapid metabolizer phenotype. This can be explained by the complexity of the process that underlies the pharmacokinetic parameters, which involves many genes and other environmental factors. Thus, if a gene variant has a relatively small impact it will not be detected. The distribution plot (Fig. [2b](#Fig2)) illustrates that slow metabolizer phenotype accumulates the widest variability of polymorphisms. Although we found six polymorphism with effect on the ATV pharmacokinetics, we know that the variability is not limited to the presence of these six markers, since other polymorphisms with little or moderate influence and not analyzed in this study, could contribute to the observed variability in each metabolic phenotype.

To our knowledge, this is the first report on the use of a massive genotyping tool (microarrays) to associate gene polymorphisms with pharmacokinetic variability of a drug commonly used by the Mexican population. The pharmacokinetic variability of ATV depends on several factors, including genetic factors. We identified six polymorphisms on six different genes that, individually, had an impact on some or all pharmacokinetic parameters. Absence of all polymorphism corresponded to rapid metabolizers, whereas the accumulation of polymorphisms caused a shift to slow metabolizers. However, some shortcomings existed in our study. First, there were insufficient data to correlate the metabolic phenotypes with the ATV response. Second, the number of participants was not large enough to validate our findings. Hence, our findings need to be validated in a larger population with genotype preselection. In order to identify (a set of) candidate predictors for ATV metabolizer phenotype, confirmative studies should be performed that focuse on the identified six polymorphisms. Our results may also be considered for future meta-analysis. The ultimate aim is that a pharmacogenetic analysis of a set of genes can be used to guide a personalized dosage that ensures drug response and prevents adverse drug effects.

## Conclusions

In summary, this pilot study offers a novel, comprehensive approach to understand the genetic contribution to the variability of ATV metabolism in a Mexican population. It enabled the identification of candidate predictive biomarkers for slow ATV metabolizers. The future confirmation of the predictive potential of these candidate genetic biomarkers and their incorporation in routine genotyping tests may optimize ATV efficacy in the clinical practice.

## Acknowledgements

Research was mainly supported by Ipharma S.A., Monterrey, Mexico. We thank the technical staff of Ipharma, S.A. and Vitagenesis, S.A. for facilities, advice and technical support. We also thank Lourdes Garza-Rodríguez and Herminia G. Martínez-Rodríguez for valuable suggestions.

## Abbreviations

### Additional files

### Additional files

## Footnotes

## Contributor Information

Rafael B. R. León-Cachón, Email: rafael.reyesleon@udem.edu

Jorge A. Ascacio-Martínez, Email: ascacio@hotmail.com

María E. Gamino-Peña, Email: gaminelen@gmail.com

Ricardo M. Cerda-Flores, Email: ricardocerda_mx@yahoo.com.mx

Irene Meester, Email: meesterirene@hotmail.com.

Hugo L. Gallardo-Blanco, Email: hugoleonid2011@icloud.com

Magdalena Gómez-Silva, Email: magagomez2000@hotmail.com.

Everardo Piñeyro-Garza, Email: epineyro@i-pharma.com.mx.

Hugo A. Barrera-Saldaña, Email: habarrera@gmail.com

## References

1. Lennernas H. Clinical pharmacokinetics of atorvastatin. Clin Pharmacokinet. 2003;42(13):1141–60. doi: 10.2165/00003088-200342130-00005.  [DOI](https://doi.org/10.2165/00003088-200342130-00005) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14531725/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacokinet&title=Clinical%20pharmacokinetics%20of%20atorvastatin&author=H%20Lennernas&volume=42&issue=13&publication_year=2003&pages=1141-60&pmid=14531725&doi=10.2165/00003088-200342130-00005&)

2. Kivisto KT, Niemi M, Schaeffeler E, Pitkala K, Tilvis R, Fromm MF, et al. Lipid-lowering response to statins is affected by CYP3A5 polymorphism. Pharmacogenetics. 2004;14(8):523–5. doi:10.1186/00008571-200408000-00005.  [DOI](https://doi.org/10.1097/01.fpc.0000114762.78957.a5) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15284534/)

3. Ahmed T, Kollipara S, Gautam A, Gigras R, Kothari M, Saha N, et al. Bioavailability and Interaction Potential of Atorvastatin and Losartan on Coadministration in Healthy Human Subjects. J Bioequivalence Bioavailability. 2009;1(1):018–27.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Bioequivalence%20Bioavailability&title=Bioavailability%20and%20Interaction%20Potential%20of%20Atorvastatin%20and%20Losartan%20on%20Coadministration%20in%20Healthy%20Human%20Subjects&author=T%20Ahmed&author=S%20Kollipara&author=A%20Gautam&author=R%20Gigras&author=M%20Kothari&volume=1&issue=1&publication_year=2009&pages=018-27&)

4. Jacobsen W, Kuhn B, Soldner A, Kirchner G, Sewing KF, Kollman PA, et al. Lactonization is the critical first step in the disposition of the 3-hydroxy-3-methylglutaryl-CoA reductase inhibitor atorvastatin. Drug Metab Dispos. 2000;28(11):1369–78.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/11038166/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Lactonization%20is%20the%20critical%20first%20step%20in%20the%20disposition%20of%20the%203-hydroxy-3-methylglutaryl-CoA%20reductase%20inhibitor%20atorvastatin&author=W%20Jacobsen&author=B%20Kuhn&author=A%20Soldner&author=G%20Kirchner&author=KF%20Sewing&volume=28&issue=11&publication_year=2000&pages=1369-78&pmid=11038166&)

5. Riedmaier S, Klein K, Hofmann U, Keskitalo JE, Neuvonen PJ, Schwab M, et al. UDP-glucuronosyltransferase (UGT) polymorphisms affect atorvastatin lactonization in vitro and in vivo. Clin Pharmacol Ther. 2010;87(1):65–73. doi: 10.1038/clpt.2009.181.  [DOI](https://doi.org/10.1038/clpt.2009.181) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19794410/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=UDP-glucuronosyltransferase%20(UGT)%20polymorphisms%20affect%20atorvastatin%20lactonization%20in%20vitro%20and%20in%20vivo&author=S%20Riedmaier&author=K%20Klein&author=U%20Hofmann&author=JE%20Keskitalo&author=PJ%20Neuvonen&volume=87&issue=1&publication_year=2010&pages=65-73&pmid=19794410&doi=10.1038/clpt.2009.181&)

6. Chen C, Mireles RJ, Campbell SD, Lin J, Mills JB, Xu JJ, et al. Differential interaction of 3-hydroxy-3-methylglutaryl-coa reductase inhibitors with ABCB1, ABCC2, and OATP1B1. Drug Metab Dispos. 2005;33(4):537–46. doi: 10.1124/dmd.104.002477.  [DOI](https://doi.org/10.1124/dmd.104.002477) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15616150/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Differential%20interaction%20of%203-hydroxy-3-methylglutaryl-coa%20reductase%20inhibitors%20with%20ABCB1,%20ABCC2,%20and%20OATP1B1&author=C%20Chen&author=RJ%20Mireles&author=SD%20Campbell&author=J%20Lin&author=JB%20Mills&volume=33&issue=4&publication_year=2005&pages=537-46&pmid=15616150&doi=10.1124/dmd.104.002477&)

7. Romaine SP, Bailey KM, Hall AS, Balmforth AJ. The influence of SLCO1B1 (OATP1B1) gene polymorphisms on response to statin therapy. Pharmacogenomics J. 2010;10(1):1–11. doi:10.1038/tpj.2009.54tpj200954.  [DOI](https://doi.org/10.1038/tpj.2009.54) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19884908/)

8. Schwarz UI. Meyer zu Schwabedissen HE, Tirona RG, Suzuki A, Leake BF, Mokrab Y et al. Identification of novel functional organic anion-transporting polypeptide 1B3 polymorphisms and assessment of substrate specificity. Pharmacogenet Genomics. 2011;21(3):103–14. doi: 10.1097/FPC.0b013e328342f5b1.  [DOI](https://doi.org/10.1097/FPC.0b013e328342f5b1) | [PMC free article](/articles/PMC3044558/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21278621/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Meyer%20zu%20Schwabedissen%20HE,%20Tirona%20RG,%20Suzuki%20A,%20Leake%20BF,%20Mokrab%20Y%20et%20al.%20Identification%20of%20novel%20functional%20organic%20anion-transporting%20polypeptide%201B3%20polymorphisms%20and%20assessment%20of%20substrate%20specificity&author=UI%20Schwarz&volume=21&issue=3&publication_year=2011&pages=103-14&pmid=21278621&doi=10.1097/FPC.0b013e328342f5b1&)

9. Keskitalo JE, Zolk O, Fromm MF, Kurkinen KJ, Neuvonen PJ, Niemi M. ABCG2 polymorphism markedly affects the pharmacokinetics of atorvastatin and rosuvastatin. Clin Pharmacol Ther. 2009;86(2):197–203. doi: 10.1038/clpt.2009.79.  [DOI](https://doi.org/10.1038/clpt.2009.79) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19474787/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=ABCG2%20polymorphism%20markedly%20affects%20the%20pharmacokinetics%20of%20atorvastatin%20and%20rosuvastatin&author=JE%20Keskitalo&author=O%20Zolk&author=MF%20Fromm&author=KJ%20Kurkinen&author=PJ%20Neuvonen&volume=86&issue=2&publication_year=2009&pages=197-203&pmid=19474787&doi=10.1038/clpt.2009.79&)

10. Mangravite LM, Thorn CF, Krauss RM. Clinical implications of pharmacogenomics of statin treatment. Pharmacogenomics J. 2006;6(6):360–74. doi:10.1038/sj.tpj.6500384.6500384.  [DOI](https://doi.org/10.1038/sj.tpj.6500384) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16550210/)

11. U.S. Food and Drug Administration. Guidance for Industry Clinical Pharmacogenomics: Premarket Evaluation in Early-Phase Clinical Studies and Recommendations for Labeling.Silver Spring: U.S. Department of Health and Human Services; 2013.

12. Leon-Cachon RB, Ascacio-Martinez JA, Barrera-Saldana HA. Individual response to drug therapy: bases and study approaches. Rev Invest Clin. 2012;64(4):364–76.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/23227587/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Rev%20Invest%20Clin&title=Individual%20response%20to%20drug%20therapy:%20bases%20and%20study%20approaches&author=RB%20Leon-Cachon&author=JA%20Ascacio-Martinez&author=HA%20Barrera-Saldana&volume=64&issue=4&publication_year=2012&pages=364-76&pmid=23227587&)

13. U.S. Food and Drug Administration . Draft Guidance on Atorvastatin Calcium and Ezetimibe. Silver Spring: U.S. Department of Health and Human Services; 2014.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Draft%20Guidance%20on%20Atorvastatin%20Calcium%20and%20Ezetimibe&publication_year=2014&)

14. Gandelman K, Malhotra B, LaBadle R, Crownover P, Bergstrom T. Analytes of Interest and Choice of Dose: Two Important Considerations in the Design of Bioequivalence Studies with Atorvastatin. J Bioequiv Availab. 2011;3(4):062–8. doi: 10.4172/jbb.1000060.  [DOI](https://doi.org/10.4172/jbb.1000060) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Bioequiv%20Availab&title=Analytes%20of%20Interest%20and%20Choice%20of%20Dose:%20Two%20Important%20Considerations%20in%20the%20Design%20of%20Bioequivalence%20Studies%20with%20Atorvastatin&author=K%20Gandelman&author=B%20Malhotra&author=R%20LaBadle&author=P%20Crownover&author=T%20Bergstrom&volume=3&issue=4&publication_year=2011&pages=062-8&doi=10.4172/jbb.1000060&)

15. Cavallari L, Lam Y. Pharmacogenetics. In: DiPiro J, Talbert R, Yee G, Matzke G, Wells B, Posey L, eds. Pharmacotherapy: A Pathophysiologic Approach. 9a ed. New York, NY: McGraw-Hill; 2014. p. 2559.

16. Rosales A, Alvear M, Cuevas A, Saavedra N, Zambrano T, Salazar LA. Identification of pharmacogenetic predictors of lipid-lowering response to atorvastatin in Chilean subjects with hypercholesterolemia. Clin Chim Acta Intl J Clin Chem. 2012;413(3–4):495–501. doi: 10.1016/j.cca.2011.11.003.  [DOI](https://doi.org/10.1016/j.cca.2011.11.003) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22120734/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Chim%20Acta%20Intl%20J%20Clin%20Chem&title=Identification%20of%20pharmacogenetic%20predictors%20of%20lipid-lowering%20response%20to%20atorvastatin%20in%20Chilean%20subjects%20with%20hypercholesterolemia&author=A%20Rosales&author=M%20Alvear&author=A%20Cuevas&author=N%20Saavedra&author=T%20Zambrano&volume=413&issue=3%E2%80%934&publication_year=2012&pages=495-501&pmid=22120734&doi=10.1016/j.cca.2011.11.003&)

17. Eichelbaum M, Ingelman-Sundberg M, Evans WE. Pharmacogenomics and individualized drug therapy. Annu Rev Med. 2006;57:119–37. doi: 10.1146/annurev.med.56.082103.104724.  [DOI](https://doi.org/10.1146/annurev.med.56.082103.104724) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16409140/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Annu%20Rev%20Med&title=Pharmacogenomics%20and%20individualized%20drug%20therapy&author=M%20Eichelbaum&author=M%20Ingelman-Sundberg&author=WE%20Evans&volume=57&publication_year=2006&pages=119-37&pmid=16409140&doi=10.1146/annurev.med.56.082103.104724&)

18. Abecasis GR, Altshuler D, Auton A, Brooks LD, Durbin RM, Gibbs RA, et al. A map of human genome variation from population-scale sequencing. Nature. 2010;467(7319):1061–73. doi:10.1038/nature09534. nature09534.  [DOI](https://doi.org/10.1038/nature09534) | [PMC free article](/articles/PMC3042601/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20981092/)

19. Madsen BE, Villesen P, Wiuf C. Short tandem repeats and genetic variation. Methods Mol Biol. 2010;628:297–306. doi: 10.1007/978-1-60327-367-1_16.  [DOI](https://doi.org/10.1007/978-1-60327-367-1_16) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20238088/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Methods%20Mol%20Biol&title=Short%20tandem%20repeats%20and%20genetic%20variation&author=BE%20Madsen&author=P%20Villesen&author=C%20Wiuf&volume=628&publication_year=2010&pages=297-306&pmid=20238088&doi=10.1007/978-1-60327-367-1_16&)

20. Pang AW, MacDonald JR, Pinto D, Wei J, Rafiq MA, Conrad DF, et al. Towards a comprehensive structural variation map of an individual human genome. Genome Biol. 2010;11(5):R52. doi:10.1186/gb-2010-11-5-r52.gb-2010-11-5-r52.  [DOI](https://doi.org/10.1186/gb-2010-11-5-r52) | [PMC free article](/articles/PMC2898065/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20482838/)

21. Hirschhorn JN, Daly MJ. Genome-wide association studies for common diseases and complex traits. Nat Rev Genet. 2005;6(2):95–108. doi:10.1038/nrg1521. nrg1521.  [DOI](https://doi.org/10.1038/nrg1521) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15716906/)

22. Solorzano-Flores LI. NORMA Oficial Mexicana NOM-177-SSA1-1998, Que establece las pruebas y procedimientos para demostrar que un medicamento es intercambiable. Requisitos a que deben sujetarse los terceros autorizados que realicen las pruebas. Secretaría de Salud, México, DF. 1999. http://www.salud.gob.mx/unidades/cdi/nom/177ssa18.html. 2010.  [http://www.salud.gob.mx/unidades/cdi/nom/177ssa18.html](http://www.salud.gob.mx/unidades/cdi/nom/177ssa18.html)

23. Ma L, Dong J, Chen X, Wang G. Development and validation of atorvastatin by LC–ESI–MS and application in bioequivalence research in healthy Chinese volunteers. Chromatographia. 2007;65(11):737–41. doi: 10.1365/s10337-007-0236-4.  [DOI](https://doi.org/10.1365/s10337-007-0236-4) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Chromatographia&title=Development%20and%20validation%20of%20atorvastatin%20by%20LC%E2%80%93ESI%E2%80%93MS%20and%20application%20in%20bioequivalence%20research%20in%20healthy%20Chinese%20volunteers&author=L%20Ma&author=J%20Dong&author=X%20Chen&author=G%20Wang&volume=65&issue=11&publication_year=2007&pages=737-41&doi=10.1365/s10337-007-0236-4&)

24. Stanisz B, Kania L. Validation of HPLC method for determination of atorvastatin in tablets and for monitoring stability in solid phase. Acta Pol Pharm. 2006;63(6):471–6.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/17438862/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Acta%20Pol%20Pharm&title=Validation%20of%20HPLC%20method%20for%20determination%20of%20atorvastatin%20in%20tablets%20and%20for%20monitoring%20stability%20in%20solid%20phase&author=B%20Stanisz&author=L%20Kania&volume=63&issue=6&publication_year=2006&pages=471-6&pmid=17438862&)

25. Sambrook J. Russell D W. Preparation and Analysis of Eukaryotic Genomic DNA. Molecular cloning: a laboratory manual. Third ed. Cold Spring Harbor: Cold Spring Harbor Laboratory Press; 2001.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Russell%20D%20W.%20Preparation%20and%20Analysis%20of%20Eukaryotic%20Genomic%20DNA.%20Molecular%20cloning:%20a%20laboratory%20manual.%20Third%20ed&author=J%20Sambrook&publication_year=2001&)

26. Tejedor D, Castillo S, Mozas P, Jimenez E, Lopez M, Tejedor MT, et al. Reliable low-density DNA array based on allele-specific probes for detection of 118 mutations causing familial hypercholesterolemia. Clin Chem. 2005;51(7):1137–44. doi: 10.1373/clinchem.2004.045203.  [DOI](https://doi.org/10.1373/clinchem.2004.045203) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15890894/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Chem&title=Reliable%20low-density%20DNA%20array%20based%20on%20allele-specific%20probes%20for%20detection%20of%20118%20mutations%20causing%20familial%20hypercholesterolemia&author=D%20Tejedor&author=S%20Castillo&author=P%20Mozas&author=E%20Jimenez&author=M%20Lopez&volume=51&issue=7&publication_year=2005&pages=1137-44&pmid=15890894&doi=10.1373/clinchem.2004.045203&)

27. Miyaji K, Nagao K, Bannai M, Asakawa H, Kohyama K, Ohtsu D, et al. Characteristic metabolism of free amino acids in cetacean plasma: cluster analysis and comparison with mice. PLoS One. 2010;5(11):e13808. doi: 10.1371/journal.pone.0013808.  [DOI](https://doi.org/10.1371/journal.pone.0013808) | [PMC free article](/articles/PMC2970564/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21072195/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=Characteristic%20metabolism%20of%20free%20amino%20acids%20in%20cetacean%20plasma:%20cluster%20analysis%20and%20comparison%20with%20mice&author=K%20Miyaji&author=K%20Nagao&author=M%20Bannai&author=H%20Asakawa&author=K%20Kohyama&volume=5&issue=11&publication_year=2010&pages=e13808&pmid=21072195&doi=10.1371/journal.pone.0013808&)

28. Reed TE, Schull WJ. A general maximum likelihood estimation program. Am J Hum Genet. 1968;20(6):579–80.  [PMC free article](/articles/PMC1706381/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/5703692/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=A%20general%20maximum%20likelihood%20estimation%20program&author=TE%20Reed&author=WJ%20Schull&volume=20&issue=6&publication_year=1968&pages=579-80&pmid=5703692&)

29. Benjamini Y, Hochberg Y. Controlling the false discovery rate: a practical and powerful approach to multiple testing. J R Stat Soc Ser B Methodol. 1995;57(1):289–300.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20R%20Stat%20Soc%20Ser%20B%20Methodol&title=Controlling%20the%20false%20discovery%20rate:%20a%20practical%20and%20powerful%20approach%20to%20multiple%20testing&author=Y%20Benjamini&author=Y%20Hochberg&volume=57&issue=1&publication_year=1995&pages=289-300&)

30. Endrenyi L, Fritsch S, Yan W. Cmax/AUC is a clearer measure than Cmax for absorption rates in investigations of bioequivalence. Int J Clin Pharmacol Ther Toxicol. 1991;29(10):394–9.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/1748540/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Clin%20Pharmacol%20Ther%20Toxicol&title=Cmax/AUC%20is%20a%20clearer%20measure%20than%20Cmax%20for%20absorption%20rates%20in%20investigations%20of%20bioequivalence&author=L%20Endrenyi&author=S%20Fritsch&author=W%20Yan&volume=29&issue=10&publication_year=1991&pages=394-9&pmid=1748540&)

31. Burim RV, Canalle R, Martinelli Ade L, Takahashi CS. Polymorphisms in glutathione S-transferases GSTM1, GSTT1 and GSTP1 and cytochromes P450 CYP2E1 and CYP1A1 and susceptibility to cirrhosis or pancreatitis in alcoholics. Mutagenesis. 2004;19(4):291–8. doi: 10.1093/mutage/geh034.  [DOI](https://doi.org/10.1093/mutage/geh034) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15215328/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mutagenesis&title=Polymorphisms%20in%20glutathione%20S-transferases%20GSTM1,%20GSTT1%20and%20GSTP1%20and%20cytochromes%20P450%20CYP2E1%20and%20CYP1A1%20and%20susceptibility%20to%20cirrhosis%20or%20pancreatitis%20in%20alcoholics&author=RV%20Burim&author=R%20Canalle&author=L%20Martinelli%20Ade&author=CS%20Takahashi&volume=19&issue=4&publication_year=2004&pages=291-8&pmid=15215328&doi=10.1093/mutage/geh034&)

32. Huang Q, Aa J, Jia H, Xin X, Tao C, Liu L et al. A Pharmacometabonomic Approach to Predicting Metabolic Phenotypes and Pharmacokinetic Parameters of Atorvastatin in Healthy Volunteers. Journal of proteome research. 2015. doi:10.1021/acs.jproteome.5b00440.  [DOI](https://doi.org/10.1021/acs.jproteome.5b00440) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26216528/)

33. Maitland-van der Zee A-H, Lynch A, Boerwinkle E, Arnett DK, Davis BR, Leiendecker-Foster C, et al. Interactions between the SNPs in the homocysteine pathway (MTHFR 677C > T, MTHFR 1298 A > C and CBSins) and the efficacy of HMG-CoA reductase inhibitors in preventing cardiovascular disease in high-risk hypertensives: The GenHAT Study. Pharmacogenet Genomics. 2008;18(8):651–6. doi: 10.1097/FPC.0b013e3282fe1759.  [DOI](https://doi.org/10.1097/FPC.0b013e3282fe1759) | [PMC free article](/articles/PMC2729516/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18622257/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Interactions%20between%20the%20SNPs%20in%20the%20homocysteine%20pathway%20(MTHFR%20677C%E2%80%89>%E2%80%89T,%20MTHFR%201298%20A%E2%80%89>%E2%80%89C%20and%20CBSins)%20and%20the%20efficacy%20of%20HMG-CoA%20reductase%20inhibitors%20in%20preventing%20cardiovascular%20disease%20in%20high-risk%20hypertensives:%20The%20GenHAT%20Study&author=A-H%20Maitland-van%20der%20Zee&author=A%20Lynch&author=E%20Boerwinkle&author=DK%20Arnett&author=BR%20Davis&volume=18&issue=8&publication_year=2008&pages=651-6&pmid=18622257&doi=10.1097/FPC.0b013e3282fe1759&)

34. Izmirli M, Bacaksiz A, Alptekin D, Uysal O, Kilic U. Statin and MTHFR C677T Polymorphism in Patients with Cardiovascular Diseases. Bezmialem Sci. 2014;2014(1):38–41. doi: 10.14235/bs.2014.311.  [DOI](https://doi.org/10.14235/bs.2014.311) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bezmialem%20Sci&title=Statin%20and%20MTHFR%20C677T%20Polymorphism%20in%20Patients%20with%20Cardiovascular%20Diseases&author=M%20Izmirli&author=A%20Bacaksiz&author=D%20Alptekin&author=O%20Uysal&author=U%20Kilic&volume=2014&issue=1&publication_year=2014&pages=38-41&doi=10.14235/bs.2014.311&)

35. Savitz J, Hodgkinson CA, Martin-Soelch C, Shen PH, Szczepanik J, Nugent A, et al. The functional DRD3 Ser9Gly polymorphism (rs6280) is pleiotropic, affecting reward as well as movement. PLoS One. 2013;8(1):e54108. doi: 10.1371/journal.pone.0054108.  [DOI](https://doi.org/10.1371/journal.pone.0054108) | [PMC free article](/articles/PMC3554713/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23365649/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=The%20functional%20DRD3%20Ser9Gly%20polymorphism%20(rs6280)%20is%20pleiotropic,%20affecting%20reward%20as%20well%20as%20movement&author=J%20Savitz&author=CA%20Hodgkinson&author=C%20Martin-Soelch&author=PH%20Shen&author=J%20Szczepanik&volume=8&issue=1&publication_year=2013&pages=e54108&pmid=23365649&doi=10.1371/journal.pone.0054108&)

36. Kang SG, Lee BH, Lee JS, Chai YG, Ko KP, Lee HJ, et al. DRD3 gene rs6280 polymorphism may be associated with alcohol dependence overall and with Lesch type I alcohol dependence in Koreans. Neuropsychobiology. 2014;69(3):140–6. doi: 10.1159/000358062.  [DOI](https://doi.org/10.1159/000358062) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24776816/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neuropsychobiology&title=DRD3%20gene%20rs6280%20polymorphism%20may%20be%20associated%20with%20alcohol%20dependence%20overall%20and%20with%20Lesch%20type%20I%20alcohol%20dependence%20in%20Koreans&author=SG%20Kang&author=BH%20Lee&author=JS%20Lee&author=YG%20Chai&author=KP%20Ko&volume=69&issue=3&publication_year=2014&pages=140-6&pmid=24776816&doi=10.1159/000358062&)

37. Kuo SC, Yeh YW, Chen CY, Huang CC, Chang HA, Yen CH, et al. DRD3 variation associates with early-onset heroin dependence, but not specific personality traits. Prog Neuropsychopharmacol Biol Psychiatry. 2014;51:1–8. doi: 10.1016/j.pnpbp.2013.12.018.  [DOI](https://doi.org/10.1016/j.pnpbp.2013.12.018) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24398431/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Prog%20Neuropsychopharmacol%20Biol%20Psychiatry&title=DRD3%20variation%20associates%20with%20early-onset%20heroin%20dependence,%20but%20not%20specific%20personality%20traits&author=SC%20Kuo&author=YW%20Yeh&author=CY%20Chen&author=CC%20Huang&author=HA%20Chang&volume=51&publication_year=2014&pages=1-8&pmid=24398431&doi=10.1016/j.pnpbp.2013.12.018&)

38. Jimenez-Jimenez FJ, Alonso-Navarro H, Martinez C, Zurdo M, Turpin-Fenoll L, Millan J, et al. Dopamine receptor D3 (DRD3) gene rs6280 variant and risk for restless legs syndrome. Sleep Med. 2013;14(4):382–4. doi: 10.1016/j.sleep.2012.11.009.  [DOI](https://doi.org/10.1016/j.sleep.2012.11.009) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23312624/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Sleep%20Med&title=Dopamine%20receptor%20D3%20(DRD3)%20gene%20rs6280%20variant%20and%20risk%20for%20restless%20legs%20syndrome&author=FJ%20Jimenez-Jimenez&author=H%20Alonso-Navarro&author=C%20Martinez&author=M%20Zurdo&author=L%20Turpin-Fenoll&volume=14&issue=4&publication_year=2013&pages=382-4&pmid=23312624&doi=10.1016/j.sleep.2012.11.009&)

39. Lopez Aspiroz E, Cabrera Figueroa SE, Porras Hurtado GL, Cruz Guerrero R, Dominguez-Gil Hurle A, Carracedo A, et al. Pharmacogenetic analysis of SNPs in genes involved in the pharmacokinetics and response to lopinavir/ritonavir therapy. Curr Drug Metab. 2013;14(7):729–37. doi: 10.2174/1389200211314070001.  [DOI](https://doi.org/10.2174/1389200211314070001) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24001122/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Drug%20Metab&title=Pharmacogenetic%20analysis%20of%20SNPs%20in%20genes%20involved%20in%20the%20pharmacokinetics%20and%20response%20to%20lopinavir/ritonavir%20therapy&author=E%20Lopez%20Aspiroz&author=SE%20Cabrera%20Figueroa&author=GL%20Porras%20Hurtado&author=R%20Cruz%20Guerrero&author=A%20Dominguez-Gil%20Hurle&volume=14&issue=7&publication_year=2013&pages=729-37&pmid=24001122&doi=10.2174/1389200211314070001&)

40. Xu Y, Wang J, Dong W. GSTM3 A/B polymorphism and risk for head and neck cancer: a meta-analysis. PLoS One. 2014;9(1):e83851. doi: 10.1371/journal.pone.0083851.  [DOI](https://doi.org/10.1371/journal.pone.0083851) | [PMC free article](/articles/PMC3885523/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24416175/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=GSTM3%20A/B%20polymorphism%20and%20risk%20for%20head%20and%20neck%20cancer:%20a%20meta-analysis&author=Y%20Xu&author=J%20Wang&author=W%20Dong&volume=9&issue=1&publication_year=2014&pages=e83851&pmid=24416175&doi=10.1371/journal.pone.0083851&)

41. Salinas-Souza C, Petrilli AS, de Toledo SR. Glutathione S-transferase polymorphisms in osteosarcoma patients. Pharmacogenet Genomics. 2010;20(8):507–15. doi: 10.1097/FPC.0b013e32833caa45.  [DOI](https://doi.org/10.1097/FPC.0b013e32833caa45) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20577141/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Glutathione%20S-transferase%20polymorphisms%20in%20osteosarcoma%20patients&author=C%20Salinas-Souza&author=AS%20Petrilli&author=SR%20de%20Toledo&volume=20&issue=8&publication_year=2010&pages=507-15&pmid=20577141&doi=10.1097/FPC.0b013e32833caa45&)

42. Jain M, Kumar S, Lal P, Tiwari A, Ghoshal UC, Mittal B. Role of GSTM3 polymorphism in the risk of developing esophageal cancer. Cancer Epidemiol Biomarkers Prev. 2007;16(1):178–81. doi: 10.1158/1055-9965.EPI-06-0542.  [DOI](https://doi.org/10.1158/1055-9965.EPI-06-0542) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17220350/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Epidemiol%20Biomarkers%20Prev&title=Role%20of%20GSTM3%20polymorphism%20in%20the%20risk%20of%20developing%20esophageal%20cancer&author=M%20Jain&author=S%20Kumar&author=P%20Lal&author=A%20Tiwari&author=UC%20Ghoshal&volume=16&issue=1&publication_year=2007&pages=178-81&pmid=17220350&doi=10.1158/1055-9965.EPI-06-0542&)

43. Sener EF, Taheri S, Korkmaz K, Zararsiz G, Serhatlioglu F, Unal A, et al. Association of TNF-alpha −308 G > A and ACE I/D gene polymorphisms in hemodialysis patients with arteriovenous fistula thrombosis. Int Urol Nephrol. 2014;46(7):1419–25. doi: 10.1007/s11255-013-0580-2.  [DOI](https://doi.org/10.1007/s11255-013-0580-2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24126814/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20Urol%20Nephrol&title=Association%20of%20TNF-alpha%20%E2%88%92308%C2%A0G%E2%80%89>%E2%80%89A%20and%20ACE%20I/D%20gene%20polymorphisms%20in%20hemodialysis%20patients%20with%20arteriovenous%20fistula%20thrombosis&author=EF%20Sener&author=S%20Taheri&author=K%20Korkmaz&author=G%20Zararsiz&author=F%20Serhatlioglu&volume=46&issue=7&publication_year=2014&pages=1419-25&pmid=24126814&doi=10.1007/s11255-013-0580-2&)

44. Sun YM, Tian Y, Li X, Liu YY, Wang LF, Li J, et al. Effect of atorvastatin on expression of IL-10 and TNF-alpha mRNA in myocardial ischemia-reperfusion injury in rats. Biochem Biophys Res Commun. 2009;382(2):336–40. doi: 10.1016/j.bbrc.2009.03.019.  [DOI](https://doi.org/10.1016/j.bbrc.2009.03.019) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19275881/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem%20Biophys%20Res%20Commun&title=Effect%20of%20atorvastatin%20on%20expression%20of%20IL-10%20and%20TNF-alpha%20mRNA%20in%20myocardial%20ischemia-reperfusion%20injury%20in%20rats&author=YM%20Sun&author=Y%20Tian&author=X%20Li&author=YY%20Liu&author=LF%20Wang&volume=382&issue=2&publication_year=2009&pages=336-40&pmid=19275881&doi=10.1016/j.bbrc.2009.03.019&)

45. Kajinami K, Brousseau ME, Ordovas JM, Schaefer EJ. Polymorphisms in the multidrug resistance-1 (MDR1) gene influence the response to atorvastatin treatment in a gender-specific manner. Am J Cardiol. 2004;93(8):1046–50. doi: 10.1016/j.amjcard.2004.01.014.  [DOI](https://doi.org/10.1016/j.amjcard.2004.01.014) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15081455/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Cardiol&title=Polymorphisms%20in%20the%20multidrug%20resistance-1%20(MDR1)%20gene%20influence%20the%20response%20to%20atorvastatin%20treatment%20in%20a%20gender-specific%20manner&author=K%20Kajinami&author=ME%20Brousseau&author=JM%20Ordovas&author=EJ%20Schaefer&volume=93&issue=8&publication_year=2004&pages=1046-50&pmid=15081455&doi=10.1016/j.amjcard.2004.01.014&)

46. Hoenig MR, Walker PJ, Gurnsey C, Beadle K, Johnson L. The C3435T polymorphism in ABCB1 influences atorvastatin efficacy and muscle symptoms in a high-risk vascular cohort. J Clin Lipidol. 2011;5(2):91–6. doi: 10.1016/j.jacl.2011.01.001.  [DOI](https://doi.org/10.1016/j.jacl.2011.01.001) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21392722/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Lipidol&title=The%20C3435T%20polymorphism%20in%20ABCB1%20influences%20atorvastatin%20efficacy%20and%20muscle%20symptoms%20in%20a%20high-risk%20vascular%20cohort&author=MR%20Hoenig&author=PJ%20Walker&author=C%20Gurnsey&author=K%20Beadle&author=L%20Johnson&volume=5&issue=2&publication_year=2011&pages=91-6&pmid=21392722&doi=10.1016/j.jacl.2011.01.001&)

47. Shabana MF, Mishriki AA, Issac MS, Bakhoum SW. Do MDR1 and SLCO1B1 polymorphisms influence the therapeutic response to atorvastatin? A study on a cohort of Egyptian patients with hypercholesterolemia. Mol Diagn Ther. 2013;17(5):299–309. doi: 10.1007/s40291-013-0038-3.  [DOI](https://doi.org/10.1007/s40291-013-0038-3) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23677857/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Diagn%20Ther&title=Do%20MDR1%20and%20SLCO1B1%20polymorphisms%20influence%20the%20therapeutic%20response%20to%20atorvastatin?%20A%20study%20on%20a%20cohort%20of%20Egyptian%20patients%20with%20hypercholesterolemia&author=MF%20Shabana&author=AA%20Mishriki&author=MS%20Issac&author=SW%20Bakhoum&volume=17&issue=5&publication_year=2013&pages=299-309&pmid=23677857&doi=10.1007/s40291-013-0038-3&)

48. Hoffmeyer S, Burk O, von Richter O, Arnold HP, Brockmoller J, Johne A, et al. Functional polymorphisms of the human multidrug-resistance gene: multiple sequence variations and correlation of one allele with P-glycoprotein expression and activity in vivo. Proc Natl Acad Sci U S A. 2000;97(7):3473–8. doi: 10.1073/pnas.97.7.3473.  [DOI](https://doi.org/10.1073/pnas.97.7.3473) | [PMC free article](/articles/PMC16264/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10716719/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&title=Functional%20polymorphisms%20of%20the%20human%20multidrug-resistance%20gene:%20multiple%20sequence%20variations%20and%20correlation%20of%20one%20allele%20with%20P-glycoprotein%20expression%20and%20activity%20in%20vivo&author=S%20Hoffmeyer&author=O%20Burk&author=O%20von%20Richter&author=HP%20Arnold&author=J%20Brockmoller&volume=97&issue=7&publication_year=2000&pages=3473-8&pmid=10716719&doi=10.1073/pnas.97.7.3473&)

49. Wessler JD, Grip LT, Mendell J, Giugliano RP. The P-glycoprotein transport system and cardiovascular drugs. J Am Coll Cardiol. 2013;61(25):2495–502. doi: 10.1016/j.jacc.2013.02.058.  [DOI](https://doi.org/10.1016/j.jacc.2013.02.058) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23563132/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Am%20Coll%20Cardiol&title=The%20P-glycoprotein%20transport%20system%20and%20cardiovascular%20drugs&author=JD%20Wessler&author=LT%20Grip&author=J%20Mendell&author=RP%20Giugliano&volume=61&issue=25&publication_year=2013&pages=2495-502&pmid=23563132&doi=10.1016/j.jacc.2013.02.058&)

50. Zhou Q, Ruan ZR, Yuan H, Zeng S. CYP2C9*3(1075A > C), MDR1 G2677T/A and MDR1 C3435T are determinants of inter-subject variability in fluvastatin pharmacokinetics in healthy Chinese volunteers. Arzneimittelforschung. 2012;62(11):519–24. doi: 10.1055/s-0032-1323696.  [DOI](https://doi.org/10.1055/s-0032-1323696) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22941809/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arzneimittelforschung&title=CYP2C9*3(1075A%E2%80%89>%E2%80%89C),%20MDR1%20G2677T/A%20and%20MDR1%20C3435T%20are%20determinants%20of%20inter-subject%20variability%20in%20fluvastatin%20pharmacokinetics%20in%20healthy%20Chinese%20volunteers&author=Q%20Zhou&author=ZR%20Ruan&author=H%20Yuan&author=S%20Zeng&volume=62&issue=11&publication_year=2012&pages=519-24&pmid=22941809&doi=10.1055/s-0032-1323696&)

51. Gonzalez-Vacarezza N, Dorado P, Penas-Lledo EM, Farinas H, Estevez-Carrizo FE, Llerena A. MDR-1 genotypes and quetiapine pharmacokinetics in healthy volunteers. Drug Metabol Drug Interact. 2013;28(3):163–6. doi: 10.1515/dmdi-2013-0008.  [DOI](https://doi.org/10.1515/dmdi-2013-0008) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23740681/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metabol%20Drug%20Interact&title=MDR-1%20genotypes%20and%20quetiapine%20pharmacokinetics%20in%20healthy%20volunteers&author=N%20Gonzalez-Vacarezza&author=P%20Dorado&author=EM%20Penas-Lledo&author=H%20Farinas&author=FE%20Estevez-Carrizo&volume=28&issue=3&publication_year=2013&pages=163-6&pmid=23740681&doi=10.1515/dmdi-2013-0008&)

52. Dong Q, Xu B, Tan Y, Liu Z, Tian L, Zhang B, et al. The genetic variability of MDR1 C3435T polymorphisms in four Southern Chinese populations. Biomed Pharmacother. 2009;63(9):658–62. doi: 10.1016/j.biopha.2008.09.003.  [DOI](https://doi.org/10.1016/j.biopha.2008.09.003) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18926661/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biomed%20Pharmacother&title=The%20genetic%20variability%20of%20MDR1%20C3435T%20polymorphisms%20in%20four%20Southern%20Chinese%20populations&author=Q%20Dong&author=B%20Xu&author=Y%20Tan&author=Z%20Liu&author=L%20Tian&volume=63&issue=9&publication_year=2009&pages=658-62&pmid=18926661&doi=10.1016/j.biopha.2008.09.003&)

53. Becker ML, Visser LE, van Schaik RH, Hofman A, Uitterlinden AG, Stricker BH. Influence of genetic variation in CYP3A4 and ABCB1 on dose decrease or switching during simvastatin and atorvastatin therapy. Pharmacoepidemiol Drug Saf. 2010;19(1):75–81. doi: 10.1002/pds.1866.  [DOI](https://doi.org/10.1002/pds.1866) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19802823/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacoepidemiol%20Drug%20Saf&title=Influence%20of%20genetic%20variation%20in%20CYP3A4%20and%20ABCB1%20on%20dose%20decrease%20or%20switching%20during%20simvastatin%20and%20atorvastatin%20therapy&author=ML%20Becker&author=LE%20Visser&author=RH%20van%20Schaik&author=A%20Hofman&author=AG%20Uitterlinden&volume=19&issue=1&publication_year=2010&pages=75-81&pmid=19802823&doi=10.1002/pds.1866&)

54. Henningsson A, Marsh S, Loos WJ, Karlsson MO, Garsa A, Mross K, et al. Association of CYP2C8, CYP3A4, CYP3A5, and ABCB1 polymorphisms with the pharmacokinetics of paclitaxel. Clin Cancer Res. 2005;11(22):8097–104. doi: 10.1158/1078-0432.CCR-05-1152.  [DOI](https://doi.org/10.1158/1078-0432.CCR-05-1152) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16299241/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&title=Association%20of%20CYP2C8,%20CYP3A4,%20CYP3A5,%20and%20ABCB1%20polymorphisms%20with%20the%20pharmacokinetics%20of%20paclitaxel&author=A%20Henningsson&author=S%20Marsh&author=WJ%20Loos&author=MO%20Karlsson&author=A%20Garsa&volume=11&issue=22&publication_year=2005&pages=8097-104&pmid=16299241&doi=10.1158/1078-0432.CCR-05-1152&)

55. Schaeffeler E, Eichelbaum M, Brinkmann U, Penger A, Asante-Poku S, Zanger UM, et al. Frequency of C3435T polymorphism of MDR1 gene in African people. Lancet. 2001;358(9279):383–4. doi: 10.1016/S0140-6736(01)05579-9.  [DOI](https://doi.org/10.1016/S0140-6736(01)05579-9) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11502320/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Frequency%20of%20C3435T%20polymorphism%20of%20MDR1%20gene%20in%20African%20people&author=E%20Schaeffeler&author=M%20Eichelbaum&author=U%20Brinkmann&author=A%20Penger&author=S%20Asante-Poku&volume=358&issue=9279&publication_year=2001&pages=383-4&pmid=11502320&doi=10.1016/S0140-6736(01)05579-9&)

56. Santos PC, Gagliardi AC, Miname MH, Chacra AP, Santos RD, Krieger JE, et al. SLCO1B1 haplotypes are not associated with atorvastatin-induced myalgia in Brazilian patients with familial hypercholesterolemia. Eur J Clin Pharmacol. 2012;68(3):273–9. doi: 10.1007/s00228-011-1125-1.  [DOI](https://doi.org/10.1007/s00228-011-1125-1) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21928084/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Clin%20Pharmacol&title=SLCO1B1%20haplotypes%20are%20not%20associated%20with%20atorvastatin-induced%20myalgia%20in%20Brazilian%20patients%20with%20familial%20hypercholesterolemia&author=PC%20Santos&author=AC%20Gagliardi&author=MH%20Miname&author=AP%20Chacra&author=RD%20Santos&volume=68&issue=3&publication_year=2012&pages=273-9&pmid=21928084&doi=10.1007/s00228-011-1125-1&)

57. Pasanen MK, Fredrikson H, Neuvonen PJ, Niemi M. Different effects of SLCO1B1 polymorphism on the pharmacokinetics of atorvastatin and rosuvastatin. Clin Pharmacol Ther. 2007;82(6):726–33. doi: 10.1038/sj.clpt.6100220.  [DOI](https://doi.org/10.1038/sj.clpt.6100220) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17473846/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Different%20effects%20of%20SLCO1B1%20polymorphism%20on%20the%20pharmacokinetics%20of%20atorvastatin%20and%20rosuvastatin&author=MK%20Pasanen&author=H%20Fredrikson&author=PJ%20Neuvonen&author=M%20Niemi&volume=82&issue=6&publication_year=2007&pages=726-33&pmid=17473846&doi=10.1038/sj.clpt.6100220&)
