# IFNL4 and IFNL3 Associated Polymorphisms Strongly Influence the Spontaneous IFN-Alpha Receptor-1 Expression in HCV-Infected Patients

## Metadata
**Authors:** Licia Bordi, Claudia Caglioti, Anna Rosa Garbuglia, Daniele Lapa, Concetta Castilletti, Chiara Taibi, Maria Rosaria Capobianchi, Eleonora Lalle
**Journal:** PLoS ONE
**Date:** 2015 Feb 12
**DOI:** [10.1371/journal.pone.0117397](https://doi.org/10.1371/journal.pone.0117397)
**PMID:** 25675103
**PMCID:** PMC4326132
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4326132/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC4326132/pdf/pone.0117397.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC4326132/pdf/pone.0117397.pdf)

## Abstract

Single-nucleotide polymorphism in IFNL3 gene (rs12979860) predicts spontaneous and therapy-induced HCV clearance. In a previous study from our group PBMC from patients with favourable rs12979860 genotype showed higher levels of IFNAR-1 mRNA. Recently, a dinucleotide polymorphism, ss469415590 (TT or ΔG), has been discovered in the region upstream IFNL3 gene, which is in high linkage disequilibrium with rs12979860. ss469415590[ΔG] is a frameshift variant that creates a novel gene, designed IFNL4, encoding the interferon-lambda 4 protein (IFNL4). The aim of the present study was to extend the analysis of IFNAR-1 mRNA levels to the ss469415590 variants. Our results highlight that the difference of IFNAR-1 mRNA levels between favourable and unfavourable genotype combinations, at both rs12979860 and ss469415590 loci, is stronger than that observed for single polymorphisms at each locus. These findings suggest may represent the biological basis for the observed association between IFNL3 CC and IFNL4 TT/TT genotypes and favourable outcome of either natural HCV infection (clearance vs chronic evolution) or IFN-based therapy.

## Introduction

The global prevalence of Hepatitis C virus (HCV) infection is estimated to be 2%, accounting for up to 180 million people infected worldwide [[1](#pone.0117397.ref001)]. The ability of HCV to inhibit the activation of endogenous type I interferon (IFN) system could underlie its success in establishing a chronic infection [[2](#pone.0117397.ref002)]. Type I IFN comprises a family of about 20 members that bind a common receptor complex, the type I IFN receptor (IFNAR) [[3](#pone.0117397.ref003)]. IFNs exhibit direct antiviral activity by inducing an antiviral state through expression of various intracellular genes (IFN-stimulated genes, ISGs), including IP10, whose levels are known to be prognostic factors of viral response to standard of care therapy (SOC) [[4](#pone.0117397.ref004)–[6](#pone.0117397.ref006)]. A group of recently discovered cytokines (IFN-lambda1/interleukin-29 [IL-29], IFN-lambda2/IL-28A, and IFN-lambda3/IL-28B, also designated IFNL1 to 3, respectively), assigned to a new type of IFN (type III IFN), gained increased attention in the HCV field [[7](#pone.0117397.ref007)]. The biological activity of type III IFN overlaps to some extent with that of type I IFN, and similar subsets of ISGs are induced as well [[8](#pone.0117397.ref008)]. However, there are important kinetic differences in the induction of ISGs by type I and type III IFN, and these differences may indeed produce distinct functional activities [[9](#pone.0117397.ref009)].

Genome-wide association studies (GWAS) identified several single-nucleotide polymorphisms (SNPs) in *IFNL3* genomic region that are strongly related to spontaneous and therapy-induced HCV clearance rate [[10](#pone.0117397.ref010)–[13](#pone.0117397.ref013)]. Numerous investigators confirmed the importance of *IFNL3* SNPs in the therapeutic response of HCV patients to SOC [[14](#pone.0117397.ref014),[15](#pone.0117397.ref015)]. Among the identified SNPs, rs12979860 appeared the most relevant, being the rs12979860-favorable CC genotype associated with a more than two-fold increased rate of sustained virologic response (SVR) with respect to hapless (CT or TT) genotypes [[16](#pone.0117397.ref016)]. More recently, a new transiently induced gene near the *IFNL3* gene has been discovered, harbouring a dinucleotide variant ss469415590 (TT or ΔG), which is in high linkage disequilibrium with rs12979860. The ss469415590 [ΔG] allele is a frameshift variant that creates a novel gene, *IFNL4*, encoding a fully functional protein designated interferon-lambda 4 (IFNL4) [[17](#pone.0117397.ref017),[18](#pone.0117397.ref018)]. The ss469415590 [TT] allele, which abrogates the production of the IFNL4 protein, is considered the favourable haplotype towards response to SOC, while the ss469415590 [ΔG] is considered the unfavourable one [[18](#pone.0117397.ref018),[19](#pone.0117397.ref019)].

The biological basis for the influence of *IFNL3* polymorphisms on HCV infection is not clear so far, and is probably complex, involving several host functions, exerted both at tissue (liver) and systemic (i.e. blood) level [[20](#pone.0117397.ref020)]. To this respect, peripheral blood mononuclear cells (PBMC) are important players in the struggle against HCV [[21](#pone.0117397.ref021)–[23](#pone.0117397.ref023)], and, although not considered as main target of HCV replication, their functions are impaired by indirect mechanisms involving HCV-induced cytokines or viral products [[24](#pone.0117397.ref024)–[27](#pone.0117397.ref027)]. It has been shown that enhanced expression of ISGs, not only in the liver, but also in PBMC, is a negative prognostic factor of virological response to IFN therapy [[5](#pone.0117397.ref005), [28](#pone.0117397.ref028)–[30](#pone.0117397.ref030)].

Previous findings from our group showed that IFNAR-1 mRNA levels are strongly reduced in HCV-infected subjects, and the reduction is much stronger in individuals with rs12979860 CT/TT alleles as compared to CC [[31](#pone.0117397.ref031)]. We hypothesized that endogenously produced IFN-lambda may be responsible for the partial reversal of the impairment IFNAR-1 expression in CC individuals, possibly conferring to these individuals a response advantage to either endogenous or exogenous IFN-alpha [[31](#pone.0117397.ref031)].

In the present study the analysis of IFNAR-1 mRNA levels was extended to the ss469415590 variants, either alone or in conjunction with the rs12979860 polymorphisms.

## Materials and Methods

The project was approved by the Local Ethical Committee, and patients agreed to participate to the study by signing informed consent.

PBMC samples from 32 treatment-naїve patients, chronically infected with HCV (genotype 1 or 4), collected at the National Institute for Infectious Diseases and stored in the Institutional Biorepository, for whom the rs12979860 (here designated *IFNL3*) and ss469415590 (here designated *IFNL4*) genotypes were known, were retrospectively selected to perform a comparative analysis.

*IFNL3* rs12979860 genotype was established on genomic DNA, using a custom made TaqMan assay previously described [[31](#pone.0117397.ref031)]. *IFNL4* ss469415590 genotype was established using a method established by Prokunina-Olsson and collaborators [[18](#pone.0117397.ref018)]. Allelic discrimination was achieved using the SDS 1.3 software [[18](#pone.0117397.ref018)].

Total cellular RNA was extracted from PBMC using Trizol (Gibco BRL, Grand Island, NY, USA) and reverse-transcribed by TaqMan Reverse Transcription Reagent kit (Applied Biosystems, Foster City, CA, USA). The quantification of IFNAR-1 and IP10 mRNA was performed by Taqman real-time RT-PCR methods previously established in our laboratory [[5](#pone.0117397.ref005),[32](#pone.0117397.ref032)]. Results were expressed as ratio to beta-actin.

In a set of experiments the PBMC were exposed for 3h to either medium or 10^3^ IU/ml recombinant human IFN-alpha2b (Intron; Schering Corp., Kenilworth, NJ, USA; specific activity: 400MIU/mg,1IU corresponding to 2.5pg). The timing and dose of IFN–alpha experiment was selected on the basis of a preliminary experiment performed on healthy donor PBMC. In fact, the 3h time point corresponds to peak of IP10 mRNA, and at this time point the enhancement is at plateau from 10^2^ IU/ml onward. However, we selected 10^3^ IU/ml to avoid the individual variability due to the possible suboptimal induction at lower doses.

Statistical analyses were performed by Prism 4 software (GraphPad, San Diego, CA). Differences were evaluated by the non parametric Mann-Whitney U test or by Student’s t test, as appropriate. Correlations were analyzed by Pearson r test. Differences with p<0.05 were considered statistically significant.

## Results

The distribution of *IFNL3* genotypes was: 8 CC (25%), 13 CT (40.5%), 11 TT (34.5%); the distribution of *IFNL4* genotypes was: 18 TT/TT (56.2%), 10 TT/ΔG (31.3%), 4 ΔG/ΔG (12.5%) ([Table 1](#pone.0117397.t001)). As shown in [Fig. 1](#pone.0117397.g001), median levels of IFNAR-1 mRNA in PBMC from patients carrying *IFNL4* TT/TT were 1,82 fold higher than those from patients carrying the ΔG allele (either TT/ΔG or ΔG/ΔG) [median: 1.026 (IQR: 0.5710–1.420) vs 0.5640 (0.4585–0.9135) p = 0.0053]. No correlation between IFNAR-1 expression and HCV viral load was observed (r = -0.1152; p = 0.6598).

### Table 1. Characteristics of 32 treatment naive HCV-infected patients included in the study.

| Age [median (range)] years | 53.5 (30–81) |
| --- | --- |
| Sex (Male, Female) | 25, 7 |
| AST [median (range)] IU/L | 41.5 (17–162) |
| ALT [median (range)] IU/L | 63 (13–310) |
| γ-GT [median (range)] IU/L | 38 (9–599) |
| HCV load [median (range)] Log10 IU/ml | 6.12 (4.22–6.84) |
| HCV genotypes (gt1, gt4) | 22, 10 |
| IFNL3 rs 12979860 genotypes (CC, CT, TT) | 8, 13, 11 |
| IFNL4 ss469415590 genotypes (TT/TT, TT/ΔG, ΔG/ΔG) | 18,10, 4 |
### Fig 1. Levels of IFNAR-1 mRNA in PBMC from HCV-infected naїve subjects carrying IFNL4 ss469415590 TT/TT genotype vs patients carrying the ΔG allele.

![Fig 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/200c/4326132/abb31b8b683c/pone.0117397.g001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4326132_pone.0117397.g001.jpg)

Results are expressed as ratio to beta-actin. The horizontal bar indicates median.

When considering the patients grouped according to the *IFNL3* and *IFNL4* genotype combinations, 8 patients were *IFNL3* CC and *IFNL4* TT/TT (*IFNL3* and *IFNL4* favourable), Group 1; 11 patients were *IFNL3* CT or TT and *IFNL4* TT/TT (*IFNL3* unfavourable and *IFNL4* favourable), Group 2; 14 patients were *IFNL3* CT or TT and *IFNL4* TT/ΔG or ΔG/ΔG (*IFNL3* and *IFNL4* unfavourable), Group 3.

The median levels of IFNAR-1 mRNA in PBMC from patients with the various *IFNL3* and *IFNL4* genotype combinations are shown in [Fig. 2](#pone.0117397.g002) Panel A. As can be seen ([Fig. 2](#pone.0117397.g002) Panel A) PBMC from patients with favourable combination at both *IFNL3* and *IFNL4* loci contain higher levels of IFNAR-1 mRNA as compared to the other groups. In particular, the most prominent difference (2.6 fold) was observed for Group 1 vs Group 3 [median: 1.466 (IQR: 1.025–1.655) vs 0.564 (IQR: 0.458–0.913); p = 0.0006]; an 1.51 fold difference was observed vs Group 2 [median:0.969 (IQR: 0.540–1.137); p = 0.0149]; the difference between Group 2 and Group 3 was not statistically significant [p = 0.1628].

### Fig 2. Levels of IFNAR-1 mRNA in PBMC from the HCV-infected naïve subjects grouped according to their IFNL3 and IFNL4 genotype combinations.

![Fig 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/200c/4326132/814a56600e13/pone.0117397.g002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4326132_pone.0117397.g002.jpg)

Panel A. Group 1: IFNL3 CC and IFNL4 TT/TT (IFNL3 favourable, IFNL4 favourable), n = 8; Group 2: IFNL3 CT or TT and IFNL4 TT/TT (IFNL3 unfavourable and IFNL4 favourable) n = 10; Group 3: IFNL3 CT or TT and IFNL4 TT/ΔG or ΔG/ΔG (IFNL3 and IFNL4 unfavourable) n = 14. The results are expressed as ratio to beta-actin (median, IQR). Levels of IP10 mRNA in PBMC from the various groups after 3h of exposure to 103 IU/ml IFN-alpha2b in vitro, according to their IFNL3 and IFNL4 genotype combinations. Panel B. Group 1: IFNL3 CC and IFNL4 TT/TT (IFNL3 favourable, IFNL4 favourable) n = 6; Group 2: IFNL3 CT or TT and IFNL4 TT/TT (IFNL3 unfavourable and IFNL4 favourable) n = 11; Group 3: IFNL3 CT or TT and IFNL4 TT/ΔG or ΔG/ΔG (IFNL3 and IFNL4 unfavourable) n = 11. The results are expressed as ratio to beta-actin, after subtraction of values from unexposed cultures (median, IQR). The range of IP10 mRNA levels in unexposed PBMC cultures was 0,375 to 0,967.

To explore whether the differences in the levels of IFNAR-1 mRNA could lead to an increased response to exogenously administered IFN-alpha, PBMC from patients with the various genotype combinations were exposed to IFN-alpha and the induction of mRNA for IP10, as biomarker of IFN activity, was measured. The results, shown in [Fig. 2](#pone.0117397.g002) Panel B, indicate a gradient of IFN response among the groups in term of IP10 mRNA induction, that paralleled the levels of IFNAR-1 expressed before IFN-alpha treatment ([Fig. 2](#pone.0117397.g002) Panel A). In particular, the difference between Group 1 and Group 3 was highly significant [median: 37.28 (IQR: 33.53–38.37) vs 16.06 (IQR: 9.979–29.27); p = 0.0193]. IFNAR-1 mRNA levels were not significantly modified by IFN-alpha treatment ([Fig. 3](#pone.0117397.g003)), reflecting the differences among groups observed at baseline.

### Fig 3. Levels of IFNAR-1 mRNA in PBMC from the HCV-infected naïve subjects after 3h of exposure to 103 IU/ml IFN-alpha2b in vitro, according to their IFNL3 and IFNL4 genotype combinations.

![Fig 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/200c/4326132/8ccbadb39e5f/pone.0117397.g003.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4326132_pone.0117397.g003.jpg)

Group 1: IFNL3 CC and IFNL4 TT/TT (IFNL3 favourable, IFNL4 favourable) n = 6; Group 2: IFNL3 CT or TT and IFNL4 TT/TT (IFNL3 unfavourable and IFNL4 favourable) n = 11; Group 3: IFNL3 CT or TT and IFNL4 TT/ΔG or ΔG/ΔG (IFNL3 and IFNL4 unfavourable) n = 11. The results are expressed as ratio to beta-actin, after subtraction of values from unexposed cultures (median, IQR).

## Discussion

Several studies confirmed the importance of *IFNL3* SNPs in the kinetics of HCV RNA decay and, ultimately, in the response to PEG-IFN plus RBV treatment [[14](#pone.0117397.ref014),[15](#pone.0117397.ref015)]. In addition, *IFNL3* genotype has been correlated with the expression of IFN-lambda receptor-1 and with non-responsiveness to IFN-alpha therapy [[33](#pone.0117397.ref033)].

Recently, a polymorphism within the gene encoding IFNL4 protein (ss469415590 TT or ΔG), in strong linkage disequilibrium with rs12979860 polymorphism, has been described, being more strongly associated with HCV clearance and treatment outcome than the previous one [[18](#pone.0117397.ref018),[19](#pone.0117397.ref019)].

More recently, *IFNL4* genotype has also been shown to be correlated with the probability of virus eradication in patients receiving IFN-free therapy [[34](#pone.0117397.ref034)]. Based on these evidences and considering a previous study in which our group observed higher levels of IFNAR-1 mRNA in PBMC from patients with favourable *IFNL3* genotype [[31](#pone.0117397.ref031)], we extended the analysis to *IFNL4* polymorphisms, considering also the combination of alleles at both *IFNL3* and *IFNL4* loci.

Our results highlighted a significantly higher expression (1.82 fold) of IFNAR-1 mRNA in PBMC from patients carrying *IFNL4* TT/TT genotype vs patients carrying the ΔG allele. More interestingly, the most prominent difference of IFNAR-1 mRNA levels was observed between patients with the most favourable combination at both *IFNL3* and *IFNL4* loci with respect to patients with combinations involving any unfavourable allele. Overall, the difference between the favourable and unfavourable combinations at both loci was much stronger than that observed for the single polymorphisms at each locus, i.e. 2.6 fold for the combination vs 1.82 fold for *IFNL4* (present study) and 2.3 fold for *IFNL3* [[31](#pone.0117397.ref031)].

The higher expression of IFNAR-1 in the favourable combination is biologically relevant, since it is connected with a stronger response to IFN-alpha exposure *in vitro*, in terms of IP10 mRNA induction.

One limitation of this study is that we could not evaluate any correlation between *IFNL3* and *IFNL4* genotypes, baseline levels of IFNAR-1 and therapeutic response, since only few patients included in this study underwent IFN-based treatment so far (12 overall, 4 SVR and 8 non SVR).

The findings from the present study, if confirmed in larger studies and supported by treatment outcome data, may offer a key for elucidating the biological basis of the advantage represented by *IFNL3* and *IFNL4* favourable genotypes towards natural or therapy-induced HCV clearance.

## Acknowledgments

The contribution of Laboratory of Microbiology and Infectious Disease Biorepository from National Institute for Infectious Diseases “L. Spallanzani,” Rome, Italy, is warmly acknowledged.

## Data Availability

All relevant data are within the paper.

## Funding Statement

This work was partly supported by grants from Italian Ministry of Health, Ricerca Corrente and Ricerca Finalizzata. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Data Availability Statement

All relevant data are within the paper.

### Data Availability Statement

All relevant data are within the paper.

## References

1. El-Serag HB (2012) Epidemiology of viral hepatitis and hepatocellular carcinoma. Gastroenterology 142: 1264–1273. 10.1053/j.gastro.2011.12.061  [DOI](https://doi.org/10.1053/j.gastro.2011.12.061) | [PMC free article](/articles/PMC3338949/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22537432/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gastroenterology&title=Epidemiology%20of%20viral%20hepatitis%20and%20hepatocellular%20carcinoma&author=HB%20El-Serag&volume=142&publication_year=2012&pages=1264-1273&pmid=22537432&doi=10.1053/j.gastro.2011.12.061&)

2. Strader DB, Wright T, Thomas DL, Seeff LB, American Association for the Study of Liver Diseases (2004) Diagnosis, management, and treatment of hepatitis C. Hepatology 39: 1147–1171.  [DOI](https://doi.org/10.1002/hep.20119) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15057920/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hepatology&title=Diagnosis,%20management,%20and%20treatment%20of%20hepatitis%20C&author=DB%20Strader&author=T%20Wright&author=DL%20Thomas&author=LB%20Seeff&volume=39&publication_year=2004&pages=1147-1171&pmid=15057920&doi=10.1002/hep.20119&)

3. Pestka S, Krause CD, Walter MR (2004) Interferons, interferon-like cytokines, and their receptors. Immunol Rev 202:8–32.  [DOI](https://doi.org/10.1111/j.0105-2896.2004.00204.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15546383/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Immunol%20Rev&title=Interferons,%20interferon-like%20cytokines,%20and%20their%20receptors&author=S%20Pestka&author=CD%20Krause&author=MR%20Walter&volume=202&publication_year=2004&pages=8-32&pmid=15546383&doi=10.1111/j.0105-2896.2004.00204.x&)

4. Lagging M, Romero AI, Westin J, Norkrans G, Dhillon AP, et al. (2006) IP-10 predicts viral response and therapeutic outcome in difficult-to-treat patients with HCV genotype 1 infection. Hepatology 44: 1617–25.  [DOI](https://doi.org/10.1002/hep.21407) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17133471/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hepatology&title=IP-10%20predicts%20viral%20response%20and%20therapeutic%20outcome%20in%20difficult-to-treat%20patients%20with%20HCV%20genotype%201%20infection&author=M%20Lagging&author=AI%20Romero&author=J%20Westin&author=G%20Norkrans&author=AP%20Dhillon&volume=44&publication_year=2006&pages=1617-25&pmid=17133471&doi=10.1002/hep.21407&)

5. Lalle E, Calcaterra S, Horejsh D, Abbate I, D’Offizi G, et al. (2008) Ability of peripheral blood mononuclear cells to activate interferon response in vitro is predictive of virological response in HCV patients. J Biol Regul Homeost Agents 22: 153–160.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/18842168/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Regul%20Homeost%20Agents&title=Ability%20of%20peripheral%20blood%20mononuclear%20cells%20to%20activate%20interferon%20response%20in%20vitro%20is%20predictive%20of%20virological%20response%20in%20HCV%20patients&author=E%20Lalle&author=S%20Calcaterra&author=D%20Horejsh&author=I%20Abbate&author=G%20D%E2%80%99Offizi&volume=22&publication_year=2008&pages=153-160&pmid=18842168&)

6. Petry H, Cashion L, Szymanski P, Ast O, Orme A, et al. (2006) Mx1 and IP-10: biomarkers to measure IFN-beta activity in mice following gene-based delivery. J Interferon Cytokine Res 26: 699–705.  [DOI](https://doi.org/10.1089/jir.2006.26.699) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17032164/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Interferon%20Cytokine%20Res&title=Mx1%20and%20IP-10:%20biomarkers%20to%20measure%20IFN-beta%20activity%20in%20mice%20following%20gene-based%20delivery&author=H%20Petry&author=L%20Cashion&author=P%20Szymanski&author=O%20Ast&author=A%20Orme&volume=26&publication_year=2006&pages=699-705&pmid=17032164&doi=10.1089/jir.2006.26.699&)

7. Donnelly RP, Kotenko SV (2010) Interferon-lambda: a new addition to an old family. J Interferon Cytokine Res 30: 555–564. 10.1089/jir.2010.0078  [DOI](https://doi.org/10.1089/jir.2010.0078) | [PMC free article](/articles/PMC2925029/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20712453/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Interferon%20Cytokine%20Res&title=Interferon-lambda:%20a%20new%20addition%20to%20an%20old%20family&author=RP%20Donnelly&author=SV%20Kotenko&volume=30&publication_year=2010&pages=555-564&pmid=20712453&doi=10.1089/jir.2010.0078&)

8. Sheppard P, Kindsvogel W, Xu W, Henderson K, Schlutsmeyer S, et al. (2003) IL-28, IL-29 and their class II cytokine receptor IL-28R. Nat Immunol 4: 63–68.  [DOI](https://doi.org/10.1038/ni873) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12469119/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Immunol&title=IL-28,%20IL-29%20and%20their%20class%20II%20cytokine%20receptor%20IL-28R&author=P%20Sheppard&author=W%20Kindsvogel&author=W%20Xu&author=K%20Henderson&author=S%20Schlutsmeyer&volume=4&publication_year=2003&pages=63-68&pmid=12469119&doi=10.1038/ni873&)

9. Bolen CR, Ding S, Robek MD, Kleinstein SH (2014) Dynamic expression profiling of type I and type III interferon-stimulated hepatocytes reveals a stable hierarchy of gene expression. Hepatology 59: 1262–72. 10.1002/hep.26657  [DOI](https://doi.org/10.1002/hep.26657) | [PMC free article](/articles/PMC3938553/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23929627/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hepatology&title=Dynamic%20expression%20profiling%20of%20type%20I%20and%20type%20III%20interferon-stimulated%20hepatocytes%20reveals%20a%20stable%20hierarchy%20of%20gene%20expression&author=CR%20Bolen&author=S%20Ding&author=MD%20Robek&author=SH%20Kleinstein&volume=59&publication_year=2014&pages=1262-72&pmid=23929627&doi=10.1002/hep.26657&)

10. Ge D, Fellay J, Thompson AJ, Simon JS, Shianna KV, et al. (2009) Genetic variation in IL-28B predicts hepatitis C treatment-induced viral clearance. Nature 461: 399–401. 10.1038/nature08309  [DOI](https://doi.org/10.1038/nature08309) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19684573/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Genetic%20variation%20in%20IL-28B%20predicts%20hepatitis%20C%20treatment-induced%20viral%20clearance&author=D%20Ge&author=J%20Fellay&author=AJ%20Thompson&author=JS%20Simon&author=KV%20Shianna&volume=461&publication_year=2009&pages=399-401&pmid=19684573&doi=10.1038/nature08309&)

11. Suppiah V, Moldovan M, Ahlenstiel G, Berg T, Weltman M, et al. (2009) IL28B is associated with response to chronic hepatitis C interferon-alpha and ribavirin therapy. Nat Genet 41: 1100–4. 10.1038/ng.447  [DOI](https://doi.org/10.1038/ng.447) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19749758/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=IL28B%20is%20associated%20with%20response%20to%20chronic%20hepatitis%20C%20interferon-alpha%20and%20ribavirin%20therapy&author=V%20Suppiah&author=M%20Moldovan&author=G%20Ahlenstiel&author=T%20Berg&author=M%20Weltman&volume=41&publication_year=2009&pages=1100-4&pmid=19749758&doi=10.1038/ng.447&)

12. Tanaka Y, Nishida N, Sugiyama M, Kurosaki M, Matsuura K, et al. (2009) Genome-wide association of IL-28B with response to pegylated interferon-alpha and ribavirin therapy for chronic hepatitis C. Nat Genet 41: 1105–1109. 10.1038/ng.449  [DOI](https://doi.org/10.1038/ng.449) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19749757/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Genome-wide%20association%20of%20IL-28B%20with%20response%20to%20pegylated%20interferon-alpha%20and%20ribavirin%20therapy%20for%20chronic%20hepatitis%20C&author=Y%20Tanaka&author=N%20Nishida&author=M%20Sugiyama&author=M%20Kurosaki&author=K%20Matsuura&volume=41&publication_year=2009&pages=1105-1109&pmid=19749757&doi=10.1038/ng.449&)

13. Thomas DL, Thio CL, Martin MP, Qi Y, Ge D, et al. (2009) Genetic variation in IL28B and spontaneous clearance of hepatitis C virus. Nature 461: 798–801. 10.1038/nature08463  [DOI](https://doi.org/10.1038/nature08463) | [PMC free article](/articles/PMC3172006/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19759533/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Genetic%20variation%20in%20IL28B%20and%20spontaneous%20clearance%20of%20hepatitis%20C%20virus&author=DL%20Thomas&author=CL%20Thio&author=MP%20Martin&author=Y%20Qi&author=D%20Ge&volume=461&publication_year=2009&pages=798-801&pmid=19759533&doi=10.1038/nature08463&)

14. Mangia A, Thompson AJ, Santoro R, Piazzolla V, Tillmann HL, et al. (2010) An IL-28B polymorphism determines treatment response of hepatitis C virus genotype 2 or 3 patients who do not achieve a rapid virologic response. Gastroenterology 139: 821–827. 10.1053/j.gastro.2010.05.079  [DOI](https://doi.org/10.1053/j.gastro.2010.05.079) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20621700/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gastroenterology&title=An%20IL-28B%20polymorphism%20determines%20treatment%20response%20of%20hepatitis%20C%20virus%20genotype%202%20or%203%20patients%20who%20do%20not%20achieve%20a%20rapid%20virologic%20response&author=A%20Mangia&author=AJ%20Thompson&author=R%20Santoro&author=V%20Piazzolla&author=HL%20Tillmann&volume=139&publication_year=2010&pages=821-827&pmid=20621700&doi=10.1053/j.gastro.2010.05.079&)

15. McCarthy JJ, Li JH, Thompson A, Suchindran S, Lao XQ, et al. (2010) Replicated association between an IL-28B gene variant and a sustained response to pegylated interferon and ribavirin. Gastroenterology 138: 2307–2314. 10.1053/j.gastro.2010.02.009  [DOI](https://doi.org/10.1053/j.gastro.2010.02.009) | [PMC free article](/articles/PMC2883666/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20176026/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gastroenterology&title=Replicated%20association%20between%20an%20IL-28B%20gene%20variant%20and%20a%20sustained%20response%20to%20pegylated%20interferon%20and%20ribavirin&author=JJ%20McCarthy&author=JH%20Li&author=A%20Thompson&author=S%20Suchindran&author=XQ%20Lao&volume=138&publication_year=2010&pages=2307-2314&pmid=20176026&doi=10.1053/j.gastro.2010.02.009&)

16. Lange CM, Zeuzem S (2011) IL-28B single nucleotide polymorphisms in the treatment of hepatitis C. J Hepatol 55: 692–701. 10.1016/j.jhep.2011.03.006  [DOI](https://doi.org/10.1016/j.jhep.2011.03.006) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21440591/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Hepatol&title=IL-28B%20single%20nucleotide%20polymorphisms%20in%20the%20treatment%20of%20hepatitis%20C&author=CM%20Lange&author=S%20Zeuzem&volume=55&publication_year=2011&pages=692-701&pmid=21440591&doi=10.1016/j.jhep.2011.03.006&)

17. Hamming OJ, Terczyńska-Dyla E, Vieyres G, Dijkman R, Jųrgensen SE, et al. (2013) Interferon lambda 4 signals via the IFNλ receptor to regulate antiviral activity against HCV and coronaviruses. EMBO J 32: 3055–65. 10.1038/emboj.2013.232  [DOI](https://doi.org/10.1038/emboj.2013.232) | [PMC free article](/articles/PMC3844954/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24169568/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=EMBO%20J&title=Interferon%20lambda%204%20signals%20via%20the%20IFN%CE%BB%20receptor%20to%20regulate%20antiviral%20activity%20against%20HCV%20and%20coronaviruses&author=OJ%20Hamming&author=E%20Terczy%C5%84ska-Dyla&author=G%20Vieyres&author=R%20Dijkman&author=SE%20J%C5%B3rgensen&volume=32&publication_year=2013&pages=3055-65&pmid=24169568&doi=10.1038/emboj.2013.232&)

18. Prokunina-Olsson L, Muchmore B, Tang W, Pfeiffer RM, Park H, et al. (2013) A variant upstream of IFNL3 (IL28B) creating a new interferon gene IFNL4 is associated with impaired clearance of hepatitis C virus. Nat Genet. 45: 164–171. 10.1038/ng.2521  [DOI](https://doi.org/10.1038/ng.2521) | [PMC free article](/articles/PMC3793390/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23291588/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=A%20variant%20upstream%20of%20IFNL3%20(IL28B)%20creating%20a%20new%20interferon%20gene%20IFNL4%20is%20associated%20with%20impaired%20clearance%20of%20hepatitis%20C%20virus&author=L%20Prokunina-Olsson&author=B%20Muchmore&author=W%20Tang&author=RM%20Pfeiffer&author=H%20Park&volume=45&publication_year=2013&pages=164-171&pmid=23291588&doi=10.1038/ng.2521&)

19. O’Brien TR, Prokunina-Olsson L, Donnelly RP (2014) IFN-λ4: The Paradoxical New Member of the Interferon Lambda Family. J Interferon Cytokine Res. 34: 829–38. 10.1089/jir.2013.0136  [DOI](https://doi.org/10.1089/jir.2013.0136) | [PMC free article](/articles/PMC4217005/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24786669/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Interferon%20Cytokine%20Res&title=IFN-%CE%BB4:%20The%20Paradoxical%20New%20Member%20of%20the%20Interferon%20Lambda%20Family&author=TR%20O%E2%80%99Brien&author=L%20Prokunina-Olsson&author=RP%20Donnelly&volume=34&publication_year=2014&pages=829-38&pmid=24786669&doi=10.1089/jir.2013.0136&)

20. Chinnaswamy S (2014) Genetic Variants at the IFNL3 Locus and Their Association with Hepatitis C Virus Infections Reveal Novel Insights into Host-Virus Interactions. J Interferon Cytokine Res 34: 479–97. 10.1089/jir.2013.0113  [DOI](https://doi.org/10.1089/jir.2013.0113) | [PMC free article](/articles/PMC4080901/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24555572/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Interferon%20Cytokine%20Res&title=Genetic%20Variants%20at%20the%20IFNL3%20Locus%20and%20Their%20Association%20with%20Hepatitis%20C%20Virus%20Infections%20Reveal%20Novel%20Insights%20into%20Host-Virus%20Interactions&author=S%20Chinnaswamy&volume=34&publication_year=2014&pages=479-97&pmid=24555572&doi=10.1089/jir.2013.0113&)

21. Chen AY, Zeremski M, Chauhan R, Jacobson IM, Talal AH, et al. (2013) Persistence of hepatitis C virus during and after otherwise clinically successful treatment of chronic hepatitis C with standard pegylated interferon α-2b and ribavirin therapy. PLoS One 8: e80078 10.1371/journal.pone.0080078  [DOI](https://doi.org/10.1371/journal.pone.0080078) | [PMC free article](/articles/PMC3836963/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24278242/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=Persistence%20of%20hepatitis%20C%20virus%20during%20and%20after%20otherwise%20clinically%20successful%20treatment%20of%20chronic%20hepatitis%20C%20with%20standard%20pegylated%20interferon%20%CE%B1-2b%20and%20ribavirin%20therapy&author=AY%20Chen&author=M%20Zeremski&author=R%20Chauhan&author=IM%20Jacobson&author=AH%20Talal&volume=8&publication_year=2013&pages=e80078&pmid=24278242&doi=10.1371/journal.pone.0080078&)

22. Inglot M, Pawlowski T, Szymczak A, Malyszczak K, Zalewska M, et al. (2013) Replication of hepatitis C virus in peripheral blood mononuclear cells in patients with chronic hepatitis C treated with pegylated interferon alpha and ribavirin. Postepy Hig Med Dosw (Online) 67: 186–91.  [DOI](https://doi.org/10.5604/17322693.1038785) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23619217/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Postepy%20Hig%20Med%20Dosw%20(Online)&title=Replication%20of%20hepatitis%20C%20virus%20in%20peripheral%20blood%20mononuclear%20cells%20in%20patients%20with%20chronic%20hepatitis%20C%20treated%20with%20pegylated%20interferon%20alpha%20and%20ribavirin&author=M%20Inglot&author=T%20Pawlowski&author=A%20Szymczak&author=K%20Malyszczak&author=M%20Zalewska&volume=67&publication_year=2013&pages=186-91&pmid=23619217&doi=10.5604/17322693.1038785&)

23. Pawełczyk A, Kubisa N, Jabłońska J, Bukowska-Ośko I, Caraballo Cortes K, et al. (2013) Detection of hepatitis C virus (HCV) negative strand RNA and NS3 protein in peripheral blood mononuclear cells (PBMC): CD3+, CD14+ and CD19+. Virol J 10: 346 10.1186/1743-422X-10-346  [DOI](https://doi.org/10.1186/1743-422X-10-346) | [PMC free article](/articles/PMC4222874/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24279719/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Virol%20J&title=Detection%20of%20hepatitis%20C%20virus%20(HCV)%20negative%20strand%20RNA%20and%20NS3%20protein%20in%20peripheral%20blood%20mononuclear%20cells%20(PBMC):%20CD3+,%20CD14+%20and%20CD19+&author=A%20Pawe%C5%82czyk&author=N%20Kubisa&author=J%20Jab%C5%82o%C5%84ska&author=I%20Bukowska-O%C5%9Bko&author=K%20Caraballo%20Cortes&volume=10&publication_year=2013&pages=346&pmid=24279719&doi=10.1186/1743-422X-10-346&)

24. Chattergoon MA, Latanich R, Quinn J, Winter ME, Buckheit RW 3rd, et al. (2014) HIV and HCV activate the inflammasome in monocytes and macrophages via endosomal Toll-like receptors without induction of type 1 interferon. PLoS Pathog 10: e1004082 10.1371/journal.ppat.1004082  [DOI](https://doi.org/10.1371/journal.ppat.1004082) | [PMC free article](/articles/PMC4006909/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24788318/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Pathog&title=HIV%20and%20HCV%20activate%20the%20inflammasome%20in%20monocytes%20and%20macrophages%20via%20endosomal%20Toll-like%20receptors%20without%20induction%20of%20type%201%20interferon&author=MA%20Chattergoon&author=R%20Latanich&author=J%20Quinn&author=ME%20Winter&author=RW%20Buckheit&volume=10&publication_year=2014&pages=e1004082&pmid=24788318&doi=10.1371/journal.ppat.1004082&)

25. Cimini E, Bonnafous C, Bordoni V, Lalle E, Sicard H, et al. (2012) Interferon-α improves phosphoantigen-induced Vγ9Vδ2 T-cells interferon-γ production during chronic HCV infection. PLoS One 7: e37014 10.1371/journal.pone.0037014  [DOI](https://doi.org/10.1371/journal.pone.0037014) | [PMC free article](/articles/PMC3358305/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22629350/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=Interferon-%CE%B1%20improves%20phosphoantigen-induced%20V%CE%B39V%CE%B42%20T-cells%20interferon-%CE%B3%20production%20during%20chronic%20HCV%20infection&author=E%20Cimini&author=C%20Bonnafous&author=V%20Bordoni&author=E%20Lalle&author=H%20Sicard&volume=7&publication_year=2012&pages=e37014&pmid=22629350&doi=10.1371/journal.pone.0037014&)

26. Holder KA, Russell RS, Grant MD (2014) Natural Killer Cell Function and Dysfunction in Hepatitis C Virus Infection. Biomed Res Int. 2014: 903764 10.1155/2014/903764  [DOI](https://doi.org/10.1155/2014/903764) | [PMC free article](/articles/PMC4095668/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25057504/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biomed%20Res%20Int&title=Natural%20Killer%20Cell%20Function%20and%20Dysfunction%20in%20Hepatitis%20C%20Virus%20Infection&author=KA%20Holder&author=RS%20Russell&author=MD%20Grant&volume=2014&publication_year=2014&pages=903764&pmid=25057504&doi=10.1155/2014/903764&)

27. Piazzolla G, Tortorella C, Schiraldi O, Antonaci S (2000) Relationship between interferon-gamma, interleukin-10, and interleukin-12 production in chronic hepatitis C and in vitro effects of interferon-alpha. J Clin Immunol 20: 54–61.  [DOI](https://doi.org/10.1023/a:1006694627907) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10798608/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Immunol&title=Relationship%20between%20interferon-gamma,%20interleukin-10,%20and%20interleukin-12%20production%20in%20chronic%20hepatitis%20C%20and%20in%20vitro%20effects%20of%20interferon-alpha&author=G%20Piazzolla&author=C%20Tortorella&author=O%20Schiraldi&author=S%20Antonaci&volume=20&publication_year=2000&pages=54-61&pmid=10798608&doi=10.1023/a:1006694627907&)

28. Abbate I, Romano M, Longo R, Cappiello G, Lo Iacono O, et al. (2003) Endogenous levels of mRNA for IFNs and IFN-related genes in hepatic biopsies of chronic HCV-infected and non-alcoholic steatohepatitis patients. J Med Virol 70: 581–7.  [DOI](https://doi.org/10.1002/jmv.10433) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12794720/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Med%20Virol&title=Endogenous%20levels%20of%20mRNA%20for%20IFNs%20and%20IFN-related%20genes%20in%20hepatic%20biopsies%20of%20chronic%20HCV-infected%20and%20non-alcoholic%20steatohepatitis%20patients&author=I%20Abbate&author=M%20Romano&author=R%20Longo&author=G%20Cappiello&author=O%20Lo%20Iacono&volume=70&publication_year=2003&pages=581-7&pmid=12794720&doi=10.1002/jmv.10433&)

29. Lempicki RA, Polis MA, Yang J, McLaughlin M, Koratich C, et al. (2006) Gene expression profiles in hepatitis C virus (HCV) and HIV coinfection: class prediction analyses before treatment predict the outcome of anti-HCV therapy among HIV-coinfected persons. J Infect Dis 193: 1172–7.  [DOI](https://doi.org/10.1086/501365) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16544259/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Infect%20Dis&title=Gene%20expression%20profiles%20in%20hepatitis%20C%20virus%20(HCV)%20and%20HIV%20coinfection:%20class%20prediction%20analyses%20before%20treatment%20predict%20the%20outcome%20of%20anti-HCV%20therapy%20among%20HIV-coinfected%20persons&author=RA%20Lempicki&author=MA%20Polis&author=J%20Yang&author=M%20McLaughlin&author=C%20Koratich&volume=193&publication_year=2006&pages=1172-7&pmid=16544259&doi=10.1086/501365&)

30. Pawlotsky JM, Hovanessian AG, Roudot-Thoraval F, Robert N, Bouvier M, et al. (1996) Effect of alpha interferon (IFN-alpha) on 2′-5′ oligoadenylate synthetase activity in peripheral blood mononuclear cells of patients with chronic hepatitis C: relationship to the antiviral effect of IFN-alpha. Antimicrob Agents Chemother 40: 320–4.  [DOI](https://doi.org/10.1128/aac.40.2.320) | [PMC free article](/articles/PMC163109/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8834873/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Antimicrob%20Agents%20Chemother&title=Effect%20of%20alpha%20interferon%20(IFN-alpha)%20on%202%E2%80%B2-5%E2%80%B2%20oligoadenylate%20synthetase%20activity%20in%20peripheral%20blood%20mononuclear%20cells%20of%20patients%20with%20chronic%20hepatitis%20C:%20relationship%20to%20the%20antiviral%20effect%20of%20IFN-alpha&author=JM%20Pawlotsky&author=AG%20Hovanessian&author=F%20Roudot-Thoraval&author=N%20Robert&author=M%20Bouvier&volume=40&publication_year=1996&pages=320-4&pmid=8834873&doi=10.1128/aac.40.2.320&)

31. Lalle E, Bordi L, Caglioti C, Garbuglia AR, Castilletti C, et al. (2014) IFN-alpha Receptor-1 Upregulation in PBMC from HCV Naïve Patients carrying CC genotype. Possible role of IFN-lambda. PlosOne 9: e93434 10.1371/journal.pone.0093434  [DOI](https://doi.org/10.1371/journal.pone.0093434) | [PMC free article](/articles/PMC3972101/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24691098/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PlosOne&title=IFN-alpha%20Receptor-1%20Upregulation%20in%20PBMC%20from%20HCV%20Na%C3%AFve%20Patients%20carrying%20CC%20genotype.%20Possible%20role%20of%20IFN-lambda&author=E%20Lalle&author=L%20Bordi&author=C%20Caglioti&author=AR%20Garbuglia&author=C%20Castilletti&volume=9&publication_year=2014&pages=e93434&pmid=24691098&doi=10.1371/journal.pone.0093434&)

32. Capobianchi MR, Lalle E, Martini F, Poccia F, D’Offizi G, et al. (2006) Influence of GBV-C infection on the endogenous activation of the IFN system in HIV-coinfected patients. Cell Mol Biol 52: 3–8.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/16914092/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell%20Mol%20Biol&title=Influence%20of%20GBV-C%20infection%20on%20the%20endogenous%20activation%20of%20the%20IFN%20system%20in%20HIV-coinfected%20patients&author=MR%20Capobianchi&author=E%20Lalle&author=F%20Martini&author=F%20Poccia&author=G%20D%E2%80%99Offizi&volume=52&publication_year=2006&pages=3-8&pmid=16914092&)

33. Duong FH, Trincucci G, Boldanova T, Calabrese D, Campana B, et al. (2014) IFN-λ receptor 1 expression is induced in chronic hepatitis C and correlates with the IFN-λ3 genotype and with non-responsiveness to IFN-α therapies. J Exp Med 211: 857–68. 10.1084/jem.20131557  [DOI](https://doi.org/10.1084/jem.20131557) | [PMC free article](/articles/PMC4010915/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24752298/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Exp%20Med&title=IFN-%CE%BB%20receptor%201%20expression%20is%20induced%20in%20chronic%20hepatitis%20C%20and%20correlates%20with%20the%20IFN-%CE%BB3%20genotype%20and%20with%20non-responsiveness%20to%20IFN-%CE%B1%20therapies&author=FH%20Duong&author=G%20Trincucci&author=T%20Boldanova&author=D%20Calabrese&author=B%20Campana&volume=211&publication_year=2014&pages=857-68&pmid=24752298&doi=10.1084/jem.20131557&)

34. Meissner EG, Bon D, Prokunina-Olsson L, Tang W, Masur H, et al. (2014) IFNL4-ΔG Genotype Is Associated With Slower Viral Clearance in Hepatitis C, Genotype-1 Patients Treated With Sofosbuvir and Ribavirin. J Infect Dis 209: 1700–4. 10.1093/infdis/jit827  [DOI](https://doi.org/10.1093/infdis/jit827) | [PMC free article](/articles/PMC4017364/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24367041/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Infect%20Dis&title=IFNL4-%CE%94G%20Genotype%20Is%20Associated%20With%20Slower%20Viral%20Clearance%20in%20Hepatitis%20C,%20Genotype-1%20Patients%20Treated%20With%20Sofosbuvir%20and%20Ribavirin&author=EG%20Meissner&author=D%20Bon&author=L%20Prokunina-Olsson&author=W%20Tang&author=H%20Masur&volume=209&publication_year=2014&pages=1700-4&pmid=24367041&doi=10.1093/infdis/jit827&)
