# The FNTB promoter polymorphism rs11623866 as a potential predictive biomarker for lonafarnib treatment of ovarian cancer patients

## Metadata
**Authors:** Hagen Sjard Bachmann, Werner Meier, Andreas du Bois, Rainer Kimmig, Jan Dominik Kuhlmann, Winfried Siffert, Jalid Sehouli, Kerstin Wollschlaeger, Jens Huober, Peter Hillemanns, Alexander Burges, Barbara Schmalfeldt, Behnaz Aminossadati, Pauline Wimberger
**Journal:** British Journal of Clinical Pharmacology
**Date:** 2015 Jul 22
**DOI:** [10.1111/bcp.12688](https://doi.org/10.1111/bcp.12688)
**PMID:** 26033044
**PMCID:** PMC4631186
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4631186/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC4631186/pdf/bcp0080-1139.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC4631186/pdf/bcp0080-1139.pdf)

## Abstract

**Aim:** 
Despite promising preclinical findings regarding clinical utility of farnesyltransferase inhibitors (FTI), such as lonafarnib, success of clinical trials is limited. A multicentre AGO-OVAR-15 phase II trial reported an unfavourable effect of lonafarnib on the outcome of patients with advanced ovarian cancer. This study was performed as a genetic subgroup analysis of the AGO-OVAR-15 trial, and investigated the utility of the promoter polymorphism rs11623866 of the farnesyltransferase ß-subunit gene (FNTB) in predicting the clinical effectiveness of lonafarnib.

**Methods:** 
The influence of rs11623866 (c.-609G > C) on FNTB promoter activity was investigated by electrophoretic-mobility-shift assay, luciferase-reporter assay and RT-qPCR. A total of 57 out of 105 patients from the AGO-OVAR-15 trial, treated with carboplatin and paclitaxel ± lonafarnib, was genotyped for rs11623866 by restriction fragment length polymorphism analysis. Genotype-dependent survival analysis was performed by Kaplan–Meier analysis.

**Results:** 
The presence of the G allele was associated with increased FNTB promoter activity compared with the C allele. An unfavourable effect of lonafarnib was limited to patients carrying a GG genotype (HRPFS 6.2, 95%CI = 2.01, 19.41, P = 0.002; HROS 9.6, 95%CI = 1.89, 48.54, P = 0.006). Median progression free survival (PFS) for patients with the GG genotype in the lonafarnib treated arm was 10 months, whereas median PFS without FTI-treatment was 40 months. Median overall survival (OS) in the lonafarnib-treated group was 19 months, whereas median OS was not reached in the untreated group.

**Conclusions:** 
Discrepancies between preclinical success and clinical failure may be due to the patients' genetic variability of FNTB. Therefore, our results may encourage retrospective evaluation of FNTB polymorphisms in previous FTI studies, especially those reporting positive FTI response.

Keywords: farnesyltransferase inhibitor, lonafarnib, ovarian cancer, rs11623866, single nucleotide polymorphism

### Aim

Despite promising preclinical findings regarding clinical utility of farnesyltransferase inhibitors (FTI), such as lonafarnib, success of clinical trials is limited. A multicentre AGO-OVAR-15 phase II trial reported an unfavourable effect of lonafarnib on the outcome of patients with advanced ovarian cancer. This study was performed as a genetic subgroup analysis of the AGO-OVAR-15 trial, and investigated the utility of the promoter polymorphism rs11623866 of the farnesyltransferase ß-subunit gene (*FNTB*) in predicting the clinical effectiveness of lonafarnib.

### Methods

The influence of rs11623866 (c.-609G > C) on *FNTB* promoter activity was investigated by electrophoretic-mobility-shift assay, luciferase-reporter assay and RT-qPCR. A total of 57 out of 105 patients from the AGO-OVAR-15 trial, treated with carboplatin and paclitaxel ± lonafarnib, was genotyped for rs11623866 by restriction fragment length polymorphism analysis. Genotype-dependent survival analysis was performed by Kaplan–Meier analysis.

### Results

The presence of the G allele was associated with increased *FNTB* promoter activity compared with the C allele. An unfavourable effect of lonafarnib was limited to patients carrying a GG genotype (HR_PFS_ 6.2, 95%CI = 2.01, 19.41, *P* = 0.002; HR_OS_ 9.6, 95%CI = 1.89, 48.54, *P* = 0.006). Median progression free survival (PFS) for patients with the GG genotype in the lonafarnib treated arm was 10 months, whereas median PFS without FTI-treatment was 40 months. Median overall survival (OS) in the lonafarnib-treated group was 19 months, whereas median OS was not reached in the untreated group.

### Conclusions

Discrepancies between preclinical success and clinical failure may be due to the patients' genetic variability of *FNTB*. Therefore, our results may encourage retrospective evaluation of *FNTB* polymorphisms in previous FTI studies, especially those reporting positive FTI response.

## What is Already Known about this Subject

## What this Study Adds

## Introduction

Ovarian cancer is the leading cause of death among women with gynaecologic malignancies [1](#b1). Standard treatment of ovarian cancer constitutes primary radical surgery, aiming at macroscopically complete tumour resection and subsequent platinum- and paclitaxel-based chemotherapy [2](#b2). Residual tumour burden after primary surgery is believed to be one of the most relevant prognostic factors for ovarian malignancies [3](#b3),[4](#b4). Advanced ovarian cancer is usually chemotherapy sensitive with an overall clinical response rate of 70–80% [5](#b5). However, despite this profound sensitivity to platinum-based chemotherapy and despite continuous attempts to implement maintenance therapies, more than 50% of all patients experience recurrence, resulting in a poor overall prognosis [5](#b5),[6](#b6). Therefore, the development of targeted therapy strategies is highly desirable.

In this context, there are recent advances in the administration of anti-angiogenetic monoclonal antibodies (e.g. bevacizumab) or tyrosine kinase inhibitors (e.g. pazopanib) for targeted ovarian cancer therapy [7](#b7),[8](#b8). However, presently, no predictive biomarkers are available for these kinds of therapies. Apart from this, farnesyltransferase inhibitors (FTI), such as lonafarnib, have also been of significant clinical interest. The FTI lonafarnib abrogates lipid modification of H-Ras and other farnesylation-dependent proteins, such as Rheb, RhoB or centromer-associated motor proteins, thereby interfering with tumourigenic signalling [9](#b9),[10](#b10). Preclinical results showed that lonafarnib, either as single agent or in combination with taxanes, is active not only in a broad spectrum of tumour cell lines *in vitro* but also in human ovarian cancer and breast cancer animal models [11](#b11)–[13](#b13). Due to these encouraging results, a variety of clinical studies investigated the effect of lonafarnib in different cancer entities. However, the majority of trials failed to demonstrate any substantial clinical benefit of lonafarnib. Consequently, the concept of targeting farnesyltransferase activity has not entered clinical practice [14](#b14)–[17](#b17). In a randomized clinical trial (AGO-OVAR-15, phase II), we recently analyzed the potential clinical benefit of carboplatin and paclitaxel with or without lonafarnib in first line treatment of epithelial ovarian cancer International Federation of Gynaecology and Obstetrics (FIGO) stages IIB-IV. This clinical trial comprised 105 patients and did not resolve any significant difference in the non-lonafarnib-treated *vs*. the lonafarnib-treated arm, with regard to progression free survival (PFS) and overall survival (OS). In contrast, lonafarnib treatment appeared to exert a negative effect upon survival, since significant inferiority of the lonafarnib-treated arm was reported in a stratum with residual tumour of more than 1 cm or the presence of distant metastasis (FIGO IV) [18](#b18).

So far, farnesyltransferase expression level has been discussed as a potential predictive biomarker for lonafarnib response [19](#b19). However, little is known about transcriptional regulation of farnesyltransferases and how this relates to lonafarnib effectiveness [20](#b20). We previously demonstrated prognostic relevance of *FNTB*-promoter core polymorphisms in several cancer entities, such as breast or renal cell cancer [21](#b21). Of those, rs11623866 (c.-609G > C) was the only SNP with a sufficient minor allele frequency for analysis in a limited patient cohort and was in linkage disequilibrium to the other *FNTB* promoter SNPs with lower allele frequency and *in silico* analysis predicted its potential functionality.

Therefore, we genotyped a subgroup of patients from the AGO-OVAR-15 clinical trial and investigated, in terms of an exploratory genetic study, whether the candidate *FNTB* promoter polymorphism rs11623866 i) influences farnesyltransferase expression and ii) may be a predictive biomarker for the effect of lonafarnib in ovarian cancer patients.

## Methods

### Patient characteristics

The current study was based on the recent AGO-OVAR-15, phase II clinical trial (EudraCT number: 2004-004515-26), comprising 105 patients. This trial compared standard chemotherapy (carboplatin and paclitaxel) with or without lonafarnib in primary advanced ovarian cancer. Patients above 18 years with histologically confirmed FIGO stages IIB to IV ovarian cancer were included. They had undergone previous debulking surgery (with the aim of macroscopic complete tumour resection) within 6 weeks before random assignment had been eligible. Lonafarnib was administered at a dose of 100 mg orally twice a day during chemotherapy and was increased thereafter to 200 mg twice a day, up to 6 months as a maintenance therapy. Maintenance therapy was administered for a maximum of 6 months. Patients were stratified according to residual tumour size and FIGO stage [18](#b18). Stratum 1 consisted of patients with FIGO IIB to IIIC and a residual tumour up to 1 cm and stratum 2 consisted of patients with FIGO stage IV and/or a residual tumour of more than 1 cm.

In an amendment of the existing approval for the AGO-OVAR-15 trial, we investigated whether rs11623866 could be a predictive biomarker for the effect of lonafarnib. This amendment was approved by the ethics committee, when the AGO-OVAR-15 trial had already started (Ethikkommission der Ärztekammer Nordrhein, Düsseldorf, reference number: 2004-004515-26 / 2005276 / 10-066) and was performed in accordance with good clinical practice guidelines, national laws and the Declaration of Helsinki. To prevent a systematic bias due to selection of long-living patients, we recruited also those patients with available DNA that already died (as suggested by the local ethics committee and in accordance with the declaration of the Central Ethics Committee of Germany [22](#b22). In total, DNA samples for genotyping could be obtained from 57 (54.1%) out of 105 participants in the AGO-OVAR-15 (subgroup genetics) trial. Clinicopathological data of this subgroup were comparable with those of the complete study group of the AGO-OVAR-15 trial and are summarized in Table[1](#tbl1). Genotype distributions of both treatment arms were not significantly different (*P* = 0.132).

### Table 1.

|   | All |   | TC arm |   | LTC arm |   |
| --- | --- | --- | --- | --- | --- | --- |
| Characteristic | n | % | n | % | n | % |
| Number of patients | 57 | 100 | 27 | 47.4 | 30 | 52.6 |
| Age (years), median (range) | 56 | (21–74) | 56 | (41–74) | 59 | (21–71) |
| Performance status |   |   |   |   |   |   |
| ECOG 0 | 28 | 49.1 | 13 | 48.1 | 15 | 50.0 |
| ECOG 1 | 27 | 47.4 | 13 | 48.1 | 14 | 46.7 |
| ECOG 2 | 2 | 3.5 | 1 | 3.7 | 1 | 3.3 |
| FIGO stage |   |   |   |   |   |   |
| IIB-III | 47 | 82.5 | 23 | 85.2 | 24 | 80.0 |
| IV | 10 | 17.5 | 4 | 14.8 | 6 | 20.0 |
| Histology |   |   |   |   |   |   |
| Serous | 42 | 73.7 | 20 | 74.1 | 22 | 73.3 |
| Endometroid | 4 | 7.0 | 3 | 11.1 | 1 | 3.3 |
| Mucinous | 3 | 5.3 | 2 | 7.4 | 1 | 3.3 |
| Other | 8 | 14.0 | 2 | 7.4 | 6 | 20.0 |
| Primary tumour |   |   |   |   |   |   |
| Ovary | 52 | 91.2 | 23 | 85.2 | 29 | 96.7 |
| Fallopian tube | 0 |   | 0 |   | 0 |   |
| Peritoneum | 5 | 8.8 | 4 | 14.8 | 1 | 3.3 |
| Histologic grading |   |   |   |   |   |   |
| G1 | 2 | 3.5 | 0 |   | 2 | 6.7 |
| G2 | 20 | 35.1 | 12 | 44.4 | 8 | 26.7 |
| G3 | 33 | 57.9 | 14 | 51.9 | 19 | 63.3 |
| Unknown | 2 | 3.5 | 1 | 3.7 | 1 | 3.3 |
| rs11623866 G > C |   |   |   |   |   |   |
| GG | 20 | 35.1 | 13 | 48.2 | 7 | 23.3 |
| GC | 28 | 49.1 | 10 | 37.0 | 18 | 60.0 |
| CC | 9 | 15.8 | 4 | 14.8 | 5 | 16.7 |

Table 1 Caption: Baseline characteristics. The table summarizes the patients' clinicopathological data and the rs11623866 genotype distribution in the AGO-OVAR-15 subgroup genetics trial, with regard to the lonafarnib treated arm and the non-treated arm

### Polymorphism retrieval and bioinformatic analyses

We retrieved and analyzed data of about 250 putatively functional genetic variations of the *FNTB* gene region (5' upstream region, 5'UTR, coding region, exon-intron boundaries and 3'UTR) from NCBI dbSNP [23](#b23) and the 1000 Genome browser [24](#b24). Variations with a minor allele frequency (MAF) of at least 0.01 (1%) in Caucasians were considered to be polymorphisms according to the common definition [25](#b25). To locate the core promoter region we performed promoter analysis using PromoterInspector and Gene2Promoter [26](#b26). Analysis of putative rs11623866 allele-dependent binding sites of transcription factors was performed with MatInspector [27](#b27), Consite [28](#b28) and Alibaba2.1 [29](#b29).

### DNA preparation and determination of FNTB rs11623866 genotypes

For rs11623866 genotyping, DNA was extracted from whole blood or routinely processed paraffin-embedded tissue. Dewaxing and isolation of genomic DNA were done following the manufacturer's instructions using the QIAamp DNA Mini Kit (Qiagen, Hilden, Germany). For polymerase chain reaction (PCR) the Taq DNA Polymerase Master Mix RED (Ampliqon, Herlev, Denmark) was used. Genotypes of the rs11623866 polymorphism were determined by restriction fragment length polymorphism analyses using the ‘slowdown’ PCR [30](#b30). The PCR was performed with the following primers: forward: 5'-GCGGACTGACTGTCTATTT-3', reverse: 5'-GACGCCGTCTCAGTATCA -3' resulting in a PCR product of 140 bp. Amplified fragments were digested with restriction enzyme RsaI (New England Biolabs, Beverly, MA, USA) by incubating for 4 h at 37 °C. RsaI specifically cuts PCR products that carry the G allele (101 + 39 bp). Adequate negative and positive controls were routinely used to ascertain correct genotyping. Accuracy of genotyping was additionally validated by direct sequencing of 20 randomly selected samples. This revealed complete concordance with the previous results obtained by PCR product digestion.

### Electrophoretic mobility shift assay (EMSA)

Nuclear extracts of the SKOV-3 ovarian cancer cell line were prepared using the NuCLEAR Extraction Kit (Sigma, Deisenhofen, Germany), according to the manufacturer's instructions. Sequences of unlabelled and DY-682 fluorescence-labeled oligonucleotides (Eurofins MWG Operon, Ebersberg, Germany) of the double-stranded DNA probes used were as follows: 5'-ATCCTTGTTCCTT(G/C)TACTGCATTTCAG-3' and 5'-CTGAAATGCAGTA(C/G)AAGGAACAAGGAT-3'. EMSAs were performed in triplicate and were carried out with the EMSA buffer kit (Li-COR Bioscience, Lincoln, NE, USA) according to the manufacturer's instructions. Double-stranded DNA probes and SKOV3 nuclear extracts (5 µg) were incubated with 2 µl of 10% binding buffer, 2.5 mm DTT, 0.25% Tween-20, 1 µg of poly (dI–dC) and 0.05% NP-40 in a total volume of 20 µl for 30 min at room temperature and separated by electrophoresis on a 6% polyacrylamide gel in 0.5% Tris-borate-EDTA running buffer. The gels were scanned by direct infrared fluorescence detection on the Odyssey imaging system (Li-COR Bioscience, Lincoln, NE, USA).

### Luciferase reporter assay

The promoter region of *FNTB*, comprising either the G allele or the C allele, was PCR-amplified from genomic DNA and cloned into the pGEM-T Easy vector (Promega, Madison, WI, USA). The amplified promoter region was restricted from the pGEM-T Easy vector and cloned upstream of the firefly luciferase coding region into the pGL4.10 vector (Promega, Madison, WI, USA). SKOV3 and A2780 cells were plated into 96-well plates at a density of 1,5 × 10^4^ cells/well in 100 µl of Dulbecco's modified eagle medium and Roswell Park Memorial Institute medium, respectively, with 10% fetal bovine serum. After 24 h, co-transfections were carried out in 50 µl of medium without serum using 150 ng of the respective pGL4 reporter construct and 50 ng of Renilla luciferase control vector (pGL4.74, Promega, Madison, WI, USA) containing 0.5 µl of Lipofectamine 2000 (Invitrogen, Karlsruhe, Germany) per transfection according to the manufacturer's instructions. After 6 h, the transfection mix was removed and cells were incubated with new medium. Twenty-four h after treatment, cells were harvested and assayed for Firefly and Renilla luciferase activities using the Dual-Glo Luciferase Assay System (Promega, Madison, WI, USA) on a Lumat LB 9501 Luminometer (Berthold, Bad Wildbad, Germany). To correct for variable transfection efficiency Firefly luciferase activity was normalized to Renilla luciferase activity.

### RT-qPCR

Total RNA was extracted from snap-frozen samples retrieved from the University Hospital of Essen, Germany, utilizing the Qiagen RNeasy kit according to the manufacturer's instructions. One µg of total RNA was applied for cDNA synthesis with oligo dT primers (Roche, Mannheim, Germany) and Superscript II reverse transcriptase (Invitrogen, Karlsruhe, Germany). In order to quantify *FNTB* mRNA levels, we utilized the QuantiTect SYBR Green PCR kit (Qiagen, Hilden, Germany). Primer sequences for *FNTB* transcript detection were designed in order to amplify a region near the translation initiation site (160 bp), spanning exon 1 and exon 2 (forward: 5' CTT CTC CGA GTT CTT TCA, reverse: 5' ATC TTT TCT TCT ACT TTT GC). All experimental steps were performed according to the manufacturer's instructions. Quantitative RT-qPCR was performed using the ABI-7500 system (Applied Biosystems, Darmstadt, Germany). The Cq-threshold was adjusted to a fluorescent level above the background signal and within the linear range of each amplification plot. Melting curves were drawn after each PCR run in order to ensure that a single and specific PCR-product was generated. All samples, including non-RT (without reverse transcriptase) and no-template controls were assayed in triplicate. Mean Cq values and deviations between the triplicates were calculated. Samples with a deviation > 0.5 or with any evidence for melting curve abnormality were repeated. In order to allow across sample comparison, *FTNB* expression was normalized to human beta actin (ACTB) expression as a reference gene [31](#b31). Relative expression values were calculated as following: 2 ^-[Cq(FNTB) - Cq(ACTB^^)]^.

### Statistical analysis

Clinical variables and genotypes were compared using either Student's *t*-test, linear anova for continuous variables or Pearson's χ^2^ test for categorical data. Control for deviation from the Hardy–Weinberg equilibrium was conducted using a publically available Hardy–Weinberg equilibrium calculator [32](#b32). Differences with *P* values <0.05 were regarded as statistically significant. All statistical analyses were performed using GraphPad Prism 6.0 (GraphPad Software, LaJolla, CA, USA) and SPSS software version 21.0 (IBM, Armonk, NY, USA).

## Results

### Effect of lonafarnib in the studied subpopulation

From 57 out of 105 study patients from the AGO-OVAR-15, we could obtain DNA samples for genotyping. Kaplan–Meier analysis, comparing PFS and OS of the lonafarnib-treated arm *vs*. the untreated arm in our genetic analysis is shown in Figure[1](#fig01). Curve progression compares with that of the complete study group, as already published [18](#b18). The tendency for a negative effect of lonafarnib treatment on PFS is detectable in the subgroup of the current study as well. Furthermore, the lonafarnib-treated arm showed significantly decreased OS (HR 2.178, 95% CI 1.04, 4.46, *P* = 0.041).

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9c6e/4631186/04f60a23c0d5/bcp0080-1139-f1.jpg)

Survival dependent on treatment arm. Kaplan–Meier analysis and the log-rank test were used to calculate survival curves representing progression free and overall survival of 57 patients of the AGO-OVAR-15 study dependent on treatment arm. TC taxane (paclitaxel) + carboplatin (paclitaxel) (). LTC lonafarnib + TC ()

### Genetic variability of the FNTB gene locus and in silico analysis of rs11623866

Because the *FNTB* promoter region is not characterized, we used *in silico* prediction to determine size and localization of the core promoter region. According to this analysis, the core promoter region is located within the first 834 bp 5'upstream of the translation start point. Therefore, we analyzed all genetic variants within 1000 bp 5'upstream of the translation start point as well as all variants located in other putatively functional regions of the *FNTB* gene, i.e. 5'UTR, exons, exon intron boundaries and 3'UTR. Although databases list about 250 genetic variants within these regions, only eight of them reach a MAF of at least 1% in Caucasians (Figure[2](#fig02)A). Three polymorphisms are located within the putative core promoter region. One polymorphism is a synonymous single nucleotide polymorphism in exon 10 and two polymorphisms are part of the 3'UTR of *FNTB*. Of those, rs11623866 is the only polymorphism with a sufficient minor allele frequency (38%) for analysis in our limited patient cohort. Furthermore, it is in linkage disequilibrium to the other *FNTB* promoter polymorphisms with lower allele frequencies (data not shown) and *in silico* analysis predicted its potential functionality due to allele-dependent binding sites of different transcription factors (Figure[2](#fig02)B).

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9c6e/4631186/88d63a802e3f/bcp0080-1139-f2.jpg)

Functional properties of the FNTB G > C polymorphism rs116238662. A) Structure of the FNTB gene locus. Black boxes represent exons, they are not drawn to scale. The 5' and 3'UTR are highlighted in white. Positions of FNTB polymorphisms are indicated together with minor allele frequencies given in brackets. B) In silico transcription factor binding sites, rs11623866 alleles and the surrounding sequence are shown. Transcription factor names and their putative binding sites are shown above and below the corresponding FNTB alleles, respectively. C) Genotype-dependent binding of nuclear proteins to the human FNTB promoter as analyzed by EMSA. Arrow indicates specifically retarded bands observed upon addition of nuclear extracts. A nuclear protein bound the C allele exclusively. D) Genotype-dependent activity of FNTB 5'-regulatory region reporter constructs expressed in A2780 and SKOV3 ovarian carcinoma cells. E) Relative expression of FNTB mRNA in leukocytes of healthy donors, renal cancer and ovarian cancer tissues dependent on rs11623866 genotypes

### Genotype-specific binding of SKOV3 nuclear protein to the rs11623866 C allele

We evaluated, whether the rs11623866 polymorphism influences binding of nuclear proteins and may induce differential *FNTB* promoter activity. An EMSA was performed, using SKOV-3 ovarian cancer cell line nuclear extracts (Figure[2](#fig02)C). After incubation with an oligonucleotide probe harbouring the rs11623866 C allele, we observed a distinct band, which was specifically retarded in the presence of a molar excess of a competitor probe but not in the presence of a cross-competitor probe. After incubation with a rs11623866 G allele comprising probe, we did not notice any size corresponding band, independently from the presence of any competitor type. These findings suggest allele-dependent binding of SKOV-3 nuclear protein, preferentially to the C allele.

### Presence of the rs11623866 C allele is associated with impaired FNTB promoter activity in vitro

We subsequently investigated, whether the *FNTB* rs11623866 G > C polymorphism is associated with differential *FNTB* promoter activity *in vitro*. Normalized firefly luciferase read-outs of both cell lines revealed reporter activity to be significantly reduced in samples with C allele promoter sequence, compared to G-harbouring promoters (*P* = 0.0316, *P* = 0.0404, respectively, Figure[2](#fig02)D). This suggests that the C allele is associated with reduced *FNTB* promoter activity *in vitro*.

### Presence of the rs11623866 C allele parallels impaired FNTB transcription in vivo

Differential promoter activity, associated with rs11623866 G > C polymorphism, may translate into differential rs11623866 G > C dependent *FNTB* transcript abundance *in vivo*. To test this assumption, we quantified *FNTB* expression in leucocytes from healthy subjects and different rs11623866 G > C genotypes (*n* = 22, Figure[2](#fig02)E). The GC genotype was significantly associated with decreased *FNTB* expression, compared with the GG genotype (*P* = 0.0149). In snap-frozen specimens from primary human renal cell or ovarian carcinoma (*n* = 19, *n* = 26 respectively), we likewise revealed the presence of the GC as well as the CC genotype to parallel decreased *FNTB* expression, compared with the presence of the GG genotype. However, the difference observed in the ovarian cancer tissues missed statistical significance (*P* = 0.0421, *P* = 0.1263, respectively). According to these findings we concluded that differential promoter activity, associated with the rs11623866 G > C polymorphism, is accompanied by impaired *FNTB* transcription in individuals with a GC/CC genotype in normal and malignant tissue.

### Association of the FNTB rs11623866 G > C polymorphism with the clinical effect of lonafarnib

As reported earlier, the whole study population showed a trend for inferiority of the lonafarnib-treated arm. However, for lonafarnib-treated patients with a GG genotype, highly significantly decreased PFS and OS were observed (*P* < 0.001, *P* = 0.001, respectively, Figure[3](#fig03)A). Concomitantly, hazard ratios were significantly increased for lonafarnib-treated GG genotype carriers (HR_PFS_ 6.2, *P* = 0.002; HR_OS_ 9.6, *P* = 0.006, Table[2](#tbl2)). Contrarily, for patients with a GC or CC genotype, no significant difference in the survival curves and hazard ratios could be observed between the two study arms (*P* = 0.919, *P* = 0.881, *P* = 0.625, *P* = 0.538, Table[2](#tbl2), Figure[3](#fig03)B + C). To evaluate further the effect of lonafarnib, we analyzed the toxicities by treatment arm for the subgroup genetics and for GG genotype carriers. Distribution and frequency of toxic side effects was comparable with those reported for the complete study population (Supplementary [Table 1](#sd1)). Especially for GG genotype carriers, the toxicity profile was comparable between the two study arms (Supplementary [Table 2](#sd2)).

### Figure 3.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9c6e/4631186/38f4cdde0349/bcp0080-1139-f3.jpg)

Survival dependent on rs11623866 genotype and treatment arm. Kaplan–Meier analysis and the log-rank test were used to calculate survival curves representing progression free (PFS) and overall survival (OS) of 57 patients of the AGO-OVAR-15 study dependent on rs11623866 genotype and treatment arm. TC taxane (paclitaxel) + carboplatin (); LTC lonafarnib + TC ()

### Table 2.

|   | FNTB rs11623866 G > C Genotype |  |  |  |  |  |
| --- | --- | --- | --- | --- | --- | --- |
|   | GG |  | GC |  | CC |  |
|   | HR (95% CI) | P value | HR (95% CI) | P value | HR (95% CI) | P value |
| Progression-free survival |   |   |   |   |   |   |
| Treatment |   |   |   |   |   |   |
| TC | 1 |   | 1 |   | 1 |   |
| LTC | 6.242 (2.01, 19.41) | 0.002 | 1.046 (0.44, 2.50) | 0.919 | 1.456 (0.32, 6.62) | 0.627 |
| Overall survival |   |   |   |   |   |   |
| Treatment |   |   |   |   |   |   |
| TC | 1 |   | 1 |   | 1 |   |
| LTC | 9.589 (1.89, 48.54) | 0.006 | 1.080 (0.40, 2.91) | 0.881 | 0.474 (0.04, 5.35) | 0.546 |

Table 2 Caption: Cox regression analyses. The table summarizes results of Cox regression analyses in the AGO-OVAR-15 subgroup genetics trial with regard to rs11623866 genotype, the respective treatment arms and survival (PFS and OS)

## Discussion

The present investigation was performed in association to the AGO-OVAR-15 phase II trial and molecular *in vitro* and *in vivo* analysis revealed that the C allele of the *FNTB* rs11623866 polymorphism was associated with reduced *FNTB* promoter activity and *FNTB* mRNA expression compared with the G allele. Moreover, the adverse effect of lonafarnib was restricted to patients carrying the homozygous GG genotype.

On the basis of promising preclinical data, a variety of clinical trials have been performed to investigate the potential clinical utility of lonafarnib as a targeted therapeutic drug, either as a single agent or in combination with chemotherapy. A recent phase II study evaluated the effect of lonafarnib in combination with paclitaxel in a small cohort of patients (*n* = 33) with resistant non-small cell lung cancer and reported partial response or stable disease in 48% of patients [33](#b33). In contrast, the majority of clinical trials failed to unravel any substantial objective response to lonafarnib administration in, for example, haematological malignancies, head and neck, colorectal or urothelial cancers [14](#b14)–[17](#b17). Due to these discouraging findings, pharmacological inhibition of farnesyltransferase has not yet been translated into clinical practice.

The present genotyping study was performed in association with the AGO-OVAR-15 phase II trial and evaluated the clinical utility of the *FNTB* rs11623866 G > C polymorphism as a potential pharmacogenetic predictive biomarker for lonafarnib treatment in ovarian cancer patients. Notably, analysis was conducted at a time when it was yet unknown whether or not the enrolled ovarian cancer patients would benefit from the addition of lonafarnib to standard platinum-based chemotherapy. As its central finding, AGO-OVAR-15 finally revealed no significant difference in the non-lonafarnib-treated *vs*. lonafarnib-treated arm. Surprisingly, even inferiority in the lonafarnib-treated arm was observed in the stratum with residual tumour of more than 1 cm or the presence of distant metastasis (FIGO IV) [18](#b18). A key finding of the present study was that the adverse effect of lonafarnib was restricted to patients carrying the homozygous GG genotype. However, due to the limited number of patients, a further stratification of the subgroup genetics with regard to FIGO stage and residual tumour burden, as conducted in AGO-OVAR-15 [18](#b18), could not be done. At first glance, these findings may appear counterintuitive. A straightforward hypothesis would imply that increased expression of a molecular target may favour the success of a related targeted therapy. This is demonstrated in, for example, breast cancer patients, in which a correlation between *HER2*-amplification/overexpression and clinical benefit of trastuzumab (a monoclonal antibody, targeting HER2) is well established [34](#b34),[35](#b35). However, interestingly, our findings are in line with a recent study, correlating farnesyltransferase mRNA expression level with clinical response to a combination therapy of docetaxel and lonafarnib in patients with therapy refractory solid malignancies. In this context, it was reported that patients with lower FNTB baseline expression were more likely to respond to this combination therapy [19](#b19). However, the functional background of this effect remains to be addressed in future studies. In this context, we also have to take into consideration that we focused on only one out of a variety of existing SNPs in the *FNTB* gene. Moreover, control of *FNTB* expression by its promoter sequence may be dependent on the cellular background as well as on the interaction between transcriptional activators and repressors. Thus, given our data from EMSA analysis, it may be possible that the nuclear protein fraction, specifically binding the C allele, comprises transcriptional repressor protein(s), which may contribute to negative regulation of *FNTB* transcription. However, in spite of these limitations, the molecular results in our study are indicative that the alleles of the *FNTB* rs11623866 G > C polymorphism are associated with differential *FNTB* promoter activity.

Conclusively, given the toxicity of lonafarnib [18](#b18) and its limited response among several clinical trials and different cancer entities [14](#b14)–[17](#b17), the identification of a predictive biomarker for lonafarnib treatment is highly desirable, since there may still exist a subgroup of patients who benefits from this kind of targeted therapy.

Notably, our genetic substudy was performed in parallel with the AGO-OVAR-15 main trial, i.e. to a time, when it was not clear yet, whether patients will have clinical benefit from lonafarnib, added to platinum-based chemotherapy. Nevertheless, given that rs11623866 enabled a stratification of patients with an adverse effect of lonafarnib, we suppose our study to be a relevant cornerstone, as it implies the general clinical relevance of concomitant and auxiliary SNP analysis among FTI studies as a potential diagnostic tool. However, due to the limited number of patients, the present study should be considered exploratory and our results need to be validated in larger patient cohorts. Nevertheless, our data provide the rationale to evaluate *FNTB* polymorphisms systematically in other FTI-studies in a retrospective manner, particularly in those trials where at least partial clinical response has been reported. Moreover, apart from the oncological background, it could also be interesting to transfer *FNTB* polymorphisms analysis, particularly for rs11623866, to other non-oncological diseases, such as chronic hepatitis D or Hutchinson–Gilford progeria syndrome, in which FTIs are candidate drugs of interest [36](#b36)–[38](#b38). This may help to dissect the mode of action of FTIs, with regard to *FNTB* promoter activity and possibly to refine a small proportion of patients, in which the presence of a SNP may be indicative for clinical response to FTIs. Moreover, since lonafarnib confers an adverse outcome in patients with a GG genotype, our study implies a further rationale to investigate whether those patients may possibly benefit from a therapeutic strategy of farnesyltransferase activation.

## Competing Interests

All authors have completed the Unified Competing Interest form at [http://www.icmje.org/coi_disclosure.pdf](http://www.icmje.org/coi_disclosure.pdf) (available on request from the corresponding author) and declare no support from any organization for the submitted work.

The authors want to thank the AGO study office staff, especially Mrs Gabriele Elser, for administrating our genetic substudy and for coordinating the obtainment of the patients' consent.

## Author Contributions

The principal investigator is Pauline Wimberger, MD, PhD.

HSB, WM, AB, RK, JDK, WS, JS, KW, JH, PH, AB, BS, BA and PW made substantial contributions to the conception and design of the study, to the acquisition data and to analysis/interpretation of the results. HSB, PW, WS and JDK designed and performed *in vitro* analyses and functional assays. HSB, PW, WS and JDK were involved in drafting the manuscript or revising it. All authors read and approved the manuscript in its final version.

## Supporting Information

Additional Supporting Information may be found in the online version of this article at the publisher's web-site:

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Black RJ, Bray F, Ferlay J, Parkin DM. Cancer incidence and mortality in the European Union: cancer registry data and estimates of national incidence for 1990. Eur J Cancer. 1997;33:1075–107. doi: 10.1016/s0959-8049(96)00492-3.  [DOI](https://doi.org/10.1016/s0959-8049(96)00492-3) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9376190/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Cancer&title=Cancer%20incidence%20and%20mortality%20in%20the%20European%20Union:%20cancer%20registry%20data%20and%20estimates%20of%20national%20incidence%20for%201990&author=RJ%20Black&author=F%20Bray&author=J%20Ferlay&author=DM%20Parkin&volume=33&publication_year=1997&pages=1075-107&pmid=9376190&doi=10.1016/s0959-8049(96)00492-3&)

2. du Bois A, Quinn M, Thigpen T, Vermorken J, Avall-Lundqvist E, Bookman M, Bowtell D, Brady M, Casado A, Cervantes A, Eisenhauer E, Friedlaender M, Fujiwara K, Grenman S, Guastalla JP, Harper P, Hogberg T, Kaye S, Kitchener H, Kristensen G, Mannel R, Meier W, Miller B, Neijt JP, Oza A, Ozols R, Parmar M, Pecorelli S, Pfisterer J, Poveda A, Provencher D, Pujade-Lauraine E, Randall M, Rochon J, Rustin G, Sagae S, Stehman F, Stuart G, Trimble E, Vasey P, Vergote I, Verheijen R, Wagner U. 2004 consensus statements on the management of ovarian cancer: final document of the 3rd International Gynecologic Cancer Intergroup Ovarian Cancer Consensus Conference (GCIG OCCC 2004) Ann Oncol. 2005;16:viii7–12. doi: 10.1093/annonc/mdi961.  [DOI](https://doi.org/10.1093/annonc/mdi961) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16239238/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Oncol&title=2004%20consensus%20statements%20on%20the%20management%20of%20ovarian%20cancer:%20final%20document%20of%20the%203rd%20International%20Gynecologic%20Cancer%20Intergroup%20Ovarian%20Cancer%20Consensus%20Conference%20(GCIG%20OCCC%202004)&author=A%20du%20Bois&author=M%20Quinn&author=T%20Thigpen&author=J%20Vermorken&author=E%20Avall-Lundqvist&volume=16&publication_year=2005&pages=viii7-12&pmid=16239238&doi=10.1093/annonc/mdi961&)

3. du Bois A, Reuss A, Pujade-Lauraine E, Harter P, Ray-Coquard I, Pfisterer J. Role of surgical outcome as prognostic factor in advanced epithelial ovarian cancer: a combined exploratory analysis of 3 prospectively randomized phase 3 multicenter trials: by the Arbeitsgemeinschaft Gynaekologische Onkologie Studiengruppe Ovarialkarzinom (AGO-OVAR) and the Groupe d'Investigateurs Nationaux Pour les Etudes des Cancers de l'Ovaire (GINECO) Cancer. 2009;115:1234–44. doi: 10.1002/cncr.24149.  [DOI](https://doi.org/10.1002/cncr.24149) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19189349/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer&title=Role%20of%20surgical%20outcome%20as%20prognostic%20factor%20in%20advanced%20epithelial%20ovarian%20cancer:%20a%20combined%20exploratory%20analysis%20of%203%20prospectively%20randomized%20phase%203%20multicenter%20trials:%20by%20the%20Arbeitsgemeinschaft%20Gynaekologische%20Onkologie%20Studiengruppe%20Ovarialkarzinom%20(AGO-OVAR)%20and%20the%20Groupe%20d'Investigateurs%20Nationaux%20Pour%20les%20Etudes%20des%20Cancers%20de%20l'Ovaire%20(GINECO)&author=A%20du%20Bois&author=A%20Reuss&author=E%20Pujade-Lauraine&author=P%20Harter&author=I%20Ray-Coquard&volume=115&publication_year=2009&pages=1234-44&pmid=19189349&doi=10.1002/cncr.24149&)

4. Wimberger P, Wehling M, Lehmann N, Kimmig R, Schmalfeldt B, Burges A, Harter P, Pfisterer J, du Bois A. Influence of residual tumor on outcome in ovarian cancer patients with FIGO stage IV disease: an exploratory analysis of the AGO-OVAR (Arbeitsgemeinschaft Gynaekologische Onkologie Ovarian Cancer Study Group) Ann Surg Oncol. 2010;17:1642–8. doi: 10.1245/s10434-010-0964-9.  [DOI](https://doi.org/10.1245/s10434-010-0964-9) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20165986/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Surg%20Oncol&title=Influence%20of%20residual%20tumor%20on%20outcome%20in%20ovarian%20cancer%20patients%20with%20FIGO%20stage%20IV%20disease:%20an%20exploratory%20analysis%20of%20the%20AGO-OVAR%20(Arbeitsgemeinschaft%20Gynaekologische%20Onkologie%20Ovarian%20Cancer%20Study%20Group)&author=P%20Wimberger&author=M%20Wehling&author=N%20Lehmann&author=R%20Kimmig&author=B%20Schmalfeldt&volume=17&publication_year=2010&pages=1642-8&pmid=20165986&doi=10.1245/s10434-010-0964-9&)

5. Bookman MA. Extending the platinum-free interval in recurrent ovarian cancer: the role of topotecan in second-line chemotherapy. Oncologist. 1999;4:87–94.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/10337378/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncologist&title=Extending%20the%20platinum-free%20interval%20in%20recurrent%20ovarian%20cancer:%20the%20role%20of%20topotecan%20in%20second-line%20chemotherapy&author=MA%20Bookman&volume=4&publication_year=1999&pages=87-94&pmid=10337378&)

6. Martin LP, Schilder RJ. Management of recurrent ovarian carcinoma: current status and future directions. Semin Oncol. 2009;36:112–25. doi: 10.1053/j.seminoncol.2008.12.003.  [DOI](https://doi.org/10.1053/j.seminoncol.2008.12.003) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19332246/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Semin%20Oncol&title=Management%20of%20recurrent%20ovarian%20carcinoma:%20current%20status%20and%20future%20directions&author=LP%20Martin&author=RJ%20Schilder&volume=36&publication_year=2009&pages=112-25&pmid=19332246&doi=10.1053/j.seminoncol.2008.12.003&)

7. Perren TJ, Swart AM, Pfisterer J, Ledermann JA, Pujade-Lauraine E, Kristensen G, Carey MS, Beale P, Cervantes A, Kurzeder C, du Bois A, Sehouli J, Kimmig R, Stahle A, Collinson F, Essapen S, Gourley C, Lortholary A, Selle F, Mirza MR, Leminen A, Plante M, Stark D, Qian W, Parmar MK, Oza AM. A phase 3 trial of bevacizumab in ovarian cancer. N Engl J Med. 2011;365:2484–96. doi: 10.1056/NEJMoa1103799.  [DOI](https://doi.org/10.1056/NEJMoa1103799) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22204725/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=A%20phase%203%20trial%20of%20bevacizumab%20in%20ovarian%20cancer&author=TJ%20Perren&author=AM%20Swart&author=J%20Pfisterer&author=JA%20Ledermann&author=E%20Pujade-Lauraine&volume=365&publication_year=2011&pages=2484-96&pmid=22204725&doi=10.1056/NEJMoa1103799&)

8. Friedlander M, Hancock KC, Rischin D, Messing MJ, Stringer CA, Matthys GM, Ma B, Hodge JP, Lager JJ. A Phase II, open-label study evaluating pazopanib in patients with recurrent ovarian cancer. Gynecol Oncol. 2010;119:32–7. doi: 10.1016/j.ygyno.2010.05.033.  [DOI](https://doi.org/10.1016/j.ygyno.2010.05.033) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20584542/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gynecol%20Oncol&title=A%20Phase%20II,%20open-label%20study%20evaluating%20pazopanib%20in%20patients%20with%20recurrent%20ovarian%20cancer&author=M%20Friedlander&author=KC%20Hancock&author=D%20Rischin&author=MJ%20Messing&author=CA%20Stringer&volume=119&publication_year=2010&pages=32-7&pmid=20584542&doi=10.1016/j.ygyno.2010.05.033&)

9. Basso AD, Mirza A, Liu G, Long BJ, Bishop WR, Kirschmeier P. The farnesyl transferase inhibitor (FTI) SCH66336 (lonafarnib) inhibits Rheb farnesylation and mTOR signaling. Role in FTI enhancement of taxane and tamoxifen anti-tumor activity. J Biol Chem. 2005;280:31101–8. doi: 10.1074/jbc.M503763200.  [DOI](https://doi.org/10.1074/jbc.M503763200) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16006564/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=The%20farnesyl%20transferase%20inhibitor%20(FTI)%20SCH66336%20(lonafarnib)%20inhibits%20Rheb%20farnesylation%20and%20mTOR%20signaling.%20Role%20in%20FTI%20enhancement%20of%20taxane%20and%20tamoxifen%20anti-tumor%20activity&author=AD%20Basso&author=A%20Mirza&author=G%20Liu&author=BJ%20Long&author=WR%20Bishop&volume=280&publication_year=2005&pages=31101-8&pmid=16006564&doi=10.1074/jbc.M503763200&)

10. Basso AD, Kirschmeier P, Bishop WR. Lipid posttranslational modifications. Farnesyl transferase inhibitors. J Lipid Res. 2006;47:15–31. doi: 10.1194/jlr.R500012-JLR200.  [DOI](https://doi.org/10.1194/jlr.R500012-JLR200) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16278491/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Lipid%20Res&title=Lipid%20posttranslational%20modifications.%20Farnesyl%20transferase%20inhibitors&author=AD%20Basso&author=P%20Kirschmeier&author=WR%20Bishop&volume=47&publication_year=2006&pages=15-31&pmid=16278491&doi=10.1194/jlr.R500012-JLR200&)

11. Nielsen LL, Shi B, Hajian G, Yaremko B, Lipari P, Ferrari E, Gurnani M, Malkowski M, Chen J, Bishop WR, Liu M. Combination therapy with the farnesyl protein transferase inhibitor SCH66336 and SCH58500 (p53 adenovirus) in preclinical cancer models. Cancer Res. 1999;59:5896–901.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/10606231/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=Combination%20therapy%20with%20the%20farnesyl%20protein%20transferase%20inhibitor%20SCH66336%20and%20SCH58500%20(p53%20adenovirus)%20in%20preclinical%20cancer%20models&author=LL%20Nielsen&author=B%20Shi&author=G%20Hajian&author=B%20Yaremko&author=P%20Lipari&volume=59&publication_year=1999&pages=5896-901&pmid=10606231&)

12. Niessner H, Beck D, Sinnberg T, Lasithiotakis K, Maczey E, Gogel J, Venturelli S, Berger A, Mauthe M, Toulany M, Flaherty K, Schaller M, Schadendorf D, Proikas-Cezanne T, Schittek B, Garbe C, Kulms D, Meier F. The farnesyl transferase inhibitor lonafarnib inhibits mTOR signaling and enforces sorafenib-induced apoptosis in melanoma cells. J Invest Dermatol. 2011;131:468–79. doi: 10.1038/jid.2010.297.  [DOI](https://doi.org/10.1038/jid.2010.297) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20944654/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Invest%20Dermatol&title=The%20farnesyl%20transferase%20inhibitor%20lonafarnib%20inhibits%20mTOR%20signaling%20and%20enforces%20sorafenib-induced%20apoptosis%20in%20melanoma%20cells&author=H%20Niessner&author=D%20Beck&author=T%20Sinnberg&author=K%20Lasithiotakis&author=E%20Maczey&volume=131&publication_year=2011&pages=468-79&pmid=20944654&doi=10.1038/jid.2010.297&)

13. Taylor SA, Marrinan CH, Liu G, Nale L, Bishop WR, Kirschmeier P, Liu M, Long BJ. Combining the farnesyltransferase inhibitor lonafarnib with paclitaxel results in enhanced growth inhibitory effects on human ovarian cancer models in vitro and in vivo. Gynecol Oncol. 2008;109:97–106. doi: 10.1016/j.ygyno.2007.12.013.  [DOI](https://doi.org/10.1016/j.ygyno.2007.12.013) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18237771/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gynecol%20Oncol&title=Combining%20the%20farnesyltransferase%20inhibitor%20lonafarnib%20with%20paclitaxel%20results%20in%20enhanced%20growth%20inhibitory%20effects%20on%20human%20ovarian%20cancer%20models%20in%20vitro%20and%20in%20vivo&author=SA%20Taylor&author=CH%20Marrinan&author=G%20Liu&author=L%20Nale&author=WR%20Bishop&volume=109&publication_year=2008&pages=97-106&pmid=18237771&doi=10.1016/j.ygyno.2007.12.013&)

14. Hanrahan EO, Kies MS, Glisson BS, Khuri FR, Feng L, Tran HT, Ginsberg LE, Truong MT, Hong WK, Kim ES. A phase II study of lonafarnib (SCH66336) in patients with chemorefractory, advanced squamous cell carcinoma of the head and neck. Am J Clin Oncol. 2009;32:274–9. doi: 10.1097/COC.0b013e318187dd57.  [DOI](https://doi.org/10.1097/COC.0b013e318187dd57) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19433965/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Clin%20Oncol&title=A%20phase%20II%20study%20of%20lonafarnib%20(SCH66336)%20in%20patients%20with%20chemorefractory,%20advanced%20squamous%20cell%20carcinoma%20of%20the%20head%20and%20neck&author=EO%20Hanrahan&author=MS%20Kies&author=BS%20Glisson&author=FR%20Khuri&author=L%20Feng&volume=32&publication_year=2009&pages=274-9&pmid=19433965&doi=10.1097/COC.0b013e318187dd57&)

15. Ravoet C, Mineur P, Robin V, Debusscher L, Bosly A, Andre M, El Housni H, Soree A, Bron D, Martiat P. Farnesyl transferase inhibitor (lonafarnib) in patients with myelodysplastic syndrome or secondary acute myeloid leukaemia: a phase II study. Ann Hematol. 2008;87:881–5. doi: 10.1007/s00277-008-0536-2.  [DOI](https://doi.org/10.1007/s00277-008-0536-2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18641985/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Hematol&title=Farnesyl%20transferase%20inhibitor%20(lonafarnib)%20in%20patients%20with%20myelodysplastic%20syndrome%20or%20secondary%20acute%20myeloid%20leukaemia:%20a%20phase%20II%20study&author=C%20Ravoet&author=P%20Mineur&author=V%20Robin&author=L%20Debusscher&author=A%20Bosly&volume=87&publication_year=2008&pages=881-5&pmid=18641985&doi=10.1007/s00277-008-0536-2&)

16. Winquist E, Moore MJ, Chi KN, Ernst DS, Hirte H, North S, Powers J, Walsh W, Boucher T, Patton R, Seymour L. A multinomial phase II study of lonafarnib (SCH 66336) in patients with refractory urothelial cancer. Urol Oncol. 2005;23:143–9. doi: 10.1016/j.urolonc.2004.12.012.  [DOI](https://doi.org/10.1016/j.urolonc.2004.12.012) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15907712/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Urol%20Oncol&title=A%20multinomial%20phase%20II%20study%20of%20lonafarnib%20(SCH%2066336)%20in%20patients%20with%20refractory%20urothelial%20cancer&author=E%20Winquist&author=MJ%20Moore&author=KN%20Chi&author=DS%20Ernst&author=H%20Hirte&volume=23&publication_year=2005&pages=143-9&pmid=15907712&doi=10.1016/j.urolonc.2004.12.012&)

17. Sharma S, Kemeny N, Kelsen DP, Ilson D, O'Reilly E, Zaknoen S, Baum C, Statkevich P, Hollywood E, Zhu Y, Saltz LB. A phase II trial of farnesyl protein transferase inhibitor SCH 66336, given by twice-daily oral administration, in patients with metastatic colorectal cancer refractory to 5-fluorouracil and irinotecan. Ann Oncol. 2002;13:1067–71. doi: 10.1093/annonc/mdf173.  [DOI](https://doi.org/10.1093/annonc/mdf173) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12176785/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Oncol&title=A%20phase%20II%20trial%20of%20farnesyl%20protein%20transferase%20inhibitor%20SCH%2066336,%20given%20by%20twice-daily%20oral%20administration,%20in%20patients%20with%20metastatic%20colorectal%20cancer%20refractory%20to%205-fluorouracil%20and%20irinotecan&author=S%20Sharma&author=N%20Kemeny&author=DP%20Kelsen&author=D%20Ilson&author=E%20O'Reilly&volume=13&publication_year=2002&pages=1067-71&pmid=12176785&doi=10.1093/annonc/mdf173&)

18. Meier W, du Bois A, Rau J, Gropp-Meier M, Baumann K, Huober J, Wollschlaeger K, Kreienberg R, Canzler U, Schmalfeldt B, Wimberger P, Richter B, Schroder W, Belau A, Stahle A, Burges A, Sehouli J. Randomized phase II trial of carboplatin and paclitaxel with or without lonafarnib in first-line treatment of epithelial ovarian cancer stage IIB-IV. Gynecol Oncol. 2012;126:236–40. doi: 10.1016/j.ygyno.2012.04.050.  [DOI](https://doi.org/10.1016/j.ygyno.2012.04.050) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22564713/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gynecol%20Oncol&title=Randomized%20phase%20II%20trial%20of%20carboplatin%20and%20paclitaxel%20with%20or%20without%20lonafarnib%20in%20first-line%20treatment%20of%20epithelial%20ovarian%20cancer%20stage%20IIB-IV&author=W%20Meier&author=A%20du%20Bois&author=J%20Rau&author=M%20Gropp-Meier&author=K%20Baumann&volume=126&publication_year=2012&pages=236-40&pmid=22564713&doi=10.1016/j.ygyno.2012.04.050&)

19. Kauh J, Chanel-Vos C, Escuin D, Fanucchi MP, Harvey RD, Saba N, Shin DM, Gal A, Pan L, Kutner M, Ramalingam SS, Bender L, Marcus A, Giannakakou P, Khuri FR. Farnesyl transferase expression determines clinical response to the docetaxel-lonafarnib combination in patients with advanced malignancies. Cancer. 2011;117:4049–59. doi: 10.1002/cncr.26004.  [DOI](https://doi.org/10.1002/cncr.26004) | [PMC free article](/articles/PMC3131496/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21365629/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer&title=Farnesyl%20transferase%20expression%20determines%20clinical%20response%20to%20the%20docetaxel-lonafarnib%20combination%20in%20patients%20with%20advanced%20malignancies&author=J%20Kauh&author=C%20Chanel-Vos&author=D%20Escuin&author=MP%20Fanucchi&author=RD%20Harvey&volume=117&publication_year=2011&pages=4049-59&pmid=21365629&doi=10.1002/cncr.26004&)

20. Holstein SA, Hohl RJ. Is there a future for prenyltransferase inhibitors in cancer therapy? Curr Opin Pharmacol. 2012;12:704–9. doi: 10.1016/j.coph.2012.06.013.  [DOI](https://doi.org/10.1016/j.coph.2012.06.013) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22817869/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Opin%20Pharmacol&title=Is%20there%20a%20future%20for%20prenyltransferase%20inhibitors%20in%20cancer%20therapy?&author=SA%20Holstein&author=RJ%20Hohl&volume=12&publication_year=2012&pages=704-9&pmid=22817869&doi=10.1016/j.coph.2012.06.013&)

21. Bachmann HSLA, Bau M, Schmid KW, Siffert W. FNTB promoter polymorphisms are associated with survival in different cancers. Naunyn-Schmiedeebergs Arch Pharmacol. 2009;379:93–93. , Meeting Abstract: 468.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Naunyn-Schmiedeebergs%20Arch%20Pharmacol&title=FNTB%20promoter%20polymorphisms%20are%20associated%20with%20survival%20in%20different%20cancers&author=HSLA%20Bachmann&author=M%20Bau&author=KW%20Schmid&author=W%20Siffert&volume=379&publication_year=2009&pages=93-93&)

22. Zentrale Ethikkommission bei der Bundesärtzekammer. 2003. Die (Weiter-)Verwendung von menschlichen Körpermaterialien von Verstorbenen für Zwecke medizinischer Forschung. Dtsch Arztebl; 100: A 2251.

23. Sherry ST, Ward MH, Kholodov M, Baker J, Phan L, Smigielski EM, Sirotkin K. dbSNP: the NCBI database of genetic variation. Nucleic Acids Res. 2001;29:308–11. doi: 10.1093/nar/29.1.308.  [DOI](https://doi.org/10.1093/nar/29.1.308) | [PMC free article](/articles/PMC29783/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11125122/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=dbSNP:%20the%20NCBI%20database%20of%20genetic%20variation&author=ST%20Sherry&author=MH%20Ward&author=M%20Kholodov&author=J%20Baker&author=L%20Phan&volume=29&publication_year=2001&pages=308-11&pmid=11125122&doi=10.1093/nar/29.1.308&)

24. Genomes Project C, Abecasis GR, Altshuler D, Auton A, Brooks LD, Durbin RM, Gibbs RA, Hurles ME, McVean GA. A map of human genome variation from population-scale sequencing. Nature. 2010;467:1061–73. doi: 10.1038/nature09534.  [DOI](https://doi.org/10.1038/nature09534) | [PMC free article](/articles/PMC3042601/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20981092/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=A%20map%20of%20human%20genome%20variation%20from%20population-scale%20sequencing&author=C%20Genomes%20Project&author=GR%20Abecasis&author=D%20Altshuler&author=A%20Auton&author=LD%20Brooks&volume=467&publication_year=2010&pages=1061-73&pmid=20981092&doi=10.1038/nature09534&)

25. Brookes AJ. The essence of SNPs. Gene. 1999;234:177–86. doi: 10.1016/s0378-1119(99)00219-x.  [DOI](https://doi.org/10.1016/s0378-1119(99)00219-x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10395891/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gene&title=The%20essence%20of%20SNPs&author=AJ%20Brookes&volume=234&publication_year=1999&pages=177-86&pmid=10395891&doi=10.1016/s0378-1119(99)00219-x&)

26. Scherf M, Klingenhoff A, Werner T. Highly specific localization of promoter regions in large genomic sequences by PromoterInspector: a novel context analysis approach. J Mol Biol. 2000;297:599–606. doi: 10.1006/jmbi.2000.3589.  [DOI](https://doi.org/10.1006/jmbi.2000.3589) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10731414/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Mol%20Biol&title=Highly%20specific%20localization%20of%20promoter%20regions%20in%20large%20genomic%20sequences%20by%20PromoterInspector:%20a%20novel%20context%20analysis%20approach&author=M%20Scherf&author=A%20Klingenhoff&author=T%20Werner&volume=297&publication_year=2000&pages=599-606&pmid=10731414&doi=10.1006/jmbi.2000.3589&)

27. Quandt K, Frech K, Karas H, Wingender E, Werner T. MatInd and MatInspector: new fast and versatile tools for detection of consensus matches in nucleotide sequence data. Nucleic Acids Res. 1995;23:4878–84. doi: 10.1093/nar/23.23.4878.  [DOI](https://doi.org/10.1093/nar/23.23.4878) | [PMC free article](/articles/PMC307478/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8532532/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=MatInd%20and%20MatInspector:%20new%20fast%20and%20versatile%20tools%20for%20detection%20of%20consensus%20matches%20in%20nucleotide%20sequence%20data&author=K%20Quandt&author=K%20Frech&author=H%20Karas&author=E%20Wingender&author=T%20Werner&volume=23&publication_year=1995&pages=4878-84&pmid=8532532&doi=10.1093/nar/23.23.4878&)

28. Sandelin A, Wasserman WW, Lenhard B. ConSite: web-based prediction of regulatory elements using cross-species comparison. Nucleic Acids Res. 2004;32:W249–52. doi: 10.1093/nar/gkh372.  [DOI](https://doi.org/10.1093/nar/gkh372) | [PMC free article](/articles/PMC441510/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15215389/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=ConSite:%20web-based%20prediction%20of%20regulatory%20elements%20using%20cross-species%20comparison&author=A%20Sandelin&author=WW%20Wasserman&author=B%20Lenhard&volume=32&publication_year=2004&pages=W249-52&pmid=15215389&doi=10.1093/nar/gkh372&)

29. Grabe N. AliBaba2: context specific identification of transcription factor binding sites. In Silico Biol. 2002;2:S1–15.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/11808873/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=In%20Silico%20Biol&title=AliBaba2:%20context%20specific%20identification%20of%20transcription%20factor%20binding%20sites&author=N%20Grabe&volume=2&publication_year=2002&pages=S1-15&pmid=11808873&)

30. Bachmann HS, Siffert W, Frey UH. Successful amplification of extremely GC-rich promoter regions using a novel 'slowdown PCR' technique. Pharmacogenetics. 2003;13:759–66. doi: 10.1097/00008571-200312000-00006.  [DOI](https://doi.org/10.1097/00008571-200312000-00006) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14646694/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=Successful%20amplification%20of%20extremely%20GC-rich%20promoter%20regions%20using%20a%20novel%20'slowdown%20PCR'%20technique&author=HS%20Bachmann&author=W%20Siffert&author=UH%20Frey&volume=13&publication_year=2003&pages=759-66&pmid=14646694&doi=10.1097/00008571-200312000-00006&)

31. Vandesompele J, De Preter K, Pattyn F, Poppe B, Van Roy N, De Paepe A, Speleman F. Accurate normalization of real-time quantitative RT-PCR data by geometric averaging of multiple internal control genes. Genome Biol. 2002 doi: 10.1186/gb-2002-3-7-research0034. ; 3: RESEARCH0034.  [DOI](https://doi.org/10.1186/gb-2002-3-7-research0034) | [PMC free article](/articles/PMC126239/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12184808/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Biol&title=Accurate%20normalization%20of%20real-time%20quantitative%20RT-PCR%20data%20by%20geometric%20averaging%20of%20multiple%20internal%20control%20genes&author=J%20Vandesompele&author=K%20De%20Preter&author=F%20Pattyn&author=B%20Poppe&author=N%20Van%20Roy&publication_year=2002&pmid=12184808&doi=10.1186/gb-2002-3-7-research0034&)

32. Rodriguez S, Gaunt TR, Day IN. Hardy-Weinberg equilibrium testing of biological ascertainment for Mendelian randomization studies. Am J Epidemiol. 2009;169:505–14. doi: 10.1093/aje/kwn359.  [DOI](https://doi.org/10.1093/aje/kwn359) | [PMC free article](/articles/PMC2640163/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19126586/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Epidemiol&title=Hardy-Weinberg%20equilibrium%20testing%20of%20biological%20ascertainment%20for%20Mendelian%20randomization%20studies&author=S%20Rodriguez&author=TR%20Gaunt&author=IN%20Day&volume=169&publication_year=2009&pages=505-14&pmid=19126586&doi=10.1093/aje/kwn359&)

33. Kim ES, Kies MS, Fossella FV, Glisson BS, Zaknoen S, Statkevich P, Munden RF, Summey C, Pisters KM, Papadimitrakopoulou V, Tighiouart M, Rogatko A, Khuri FR. Phase II study of the farnesyltransferase inhibitor lonafarnib with paclitaxel in patients with taxane-refractory/resistant non-small cell lung carcinoma. Cancer. 2005;104:561–9. doi: 10.1002/cncr.21188.  [DOI](https://doi.org/10.1002/cncr.21188) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16028213/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer&title=Phase%20II%20study%20of%20the%20farnesyltransferase%20inhibitor%20lonafarnib%20with%20paclitaxel%20in%20patients%20with%20taxane-refractory/resistant%20non-small%20cell%20lung%20carcinoma&author=ES%20Kim&author=MS%20Kies&author=FV%20Fossella&author=BS%20Glisson&author=S%20Zaknoen&volume=104&publication_year=2005&pages=561-9&pmid=16028213&doi=10.1002/cncr.21188&)

34. Saracchini S, Foltran L, Tuccia F, Bassini A, Sulfaro S, Micheli E, Del Conte A, Bertola M, Gion M, Lorenzon M, Tumolo S. Phase II study of liposome-encapsulated doxorubicin plus cyclophosphamide, followed by sequential trastuzumab plus docetaxel as primary systemic therapy for breast cancer patients with HER2 overexpression or amplification. Breast. 2013;22:1101–7. doi: 10.1016/j.breast.2013.09.001.  [DOI](https://doi.org/10.1016/j.breast.2013.09.001) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24074879/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Breast&title=Phase%20II%20study%20of%20liposome-encapsulated%20doxorubicin%20plus%20cyclophosphamide,%20followed%20by%20sequential%20trastuzumab%20plus%20docetaxel%20as%20primary%20systemic%20therapy%20for%20breast%20cancer%20patients%20with%20HER2%20overexpression%20or%20amplification&author=S%20Saracchini&author=L%20Foltran&author=F%20Tuccia&author=A%20Bassini&author=S%20Sulfaro&volume=22&publication_year=2013&pages=1101-7&pmid=24074879&doi=10.1016/j.breast.2013.09.001&)

35. Fuchs EM, Kostler WJ, Horvat R, Hudelist G, Kubista E, Attems J, Zielinski CC, Singer CF. High-level ERBB2 gene amplification is associated with a particularly short time-to-metastasis, but results in a high rate of complete response once trastuzumab-based therapy is offered in the metastatic setting. Int J Cancer. 2014 doi: 10.1002/ijc.28660. ; 135: 224-31. DOI: 10.1002/ijc.28660.  [DOI](https://doi.org/10.1002/ijc.28660) | [DOI](https://doi.org/10.1002/ijc.28660) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24311197/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Cancer&title=High-level%20ERBB2%20gene%20amplification%20is%20associated%20with%20a%20particularly%20short%20time-to-metastasis,%20but%20results%20in%20a%20high%20rate%20of%20complete%20response%20once%20trastuzumab-based%20therapy%20is%20offered%20in%20the%20metastatic%20setting&author=EM%20Fuchs&author=WJ%20Kostler&author=R%20Horvat&author=G%20Hudelist&author=E%20Kubista&publication_year=2014&pmid=24311197&doi=10.1002/ijc.28660&)

36. Gunsar F. Treatment of delta hepatitis. Expert Rev Anti Infect Ther. 2013;11:489–98. doi: 10.1586/eri.13.35.  [DOI](https://doi.org/10.1586/eri.13.35) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23627855/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Expert%20Rev%20Anti%20Infect%20Ther&title=Treatment%20of%20delta%20hepatitis&author=F%20Gunsar&volume=11&publication_year=2013&pages=489-98&pmid=23627855&doi=10.1586/eri.13.35&)

37. Gordon LB, Massaro J, D'Agostino RB, Sr, Campbell SE, Brazier J, Brown WT, Kleinman ME, Kieran MW. Progeria Clinical Trials C. Impact of farnesylation inhibitors on survival in Hutchinson-Gilford progeria syndrome. Circulation. 2014;130:27–34. doi: 10.1161/CIRCULATIONAHA.113.008285.  [DOI](https://doi.org/10.1161/CIRCULATIONAHA.113.008285) | [PMC free article](/articles/PMC4082404/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24795390/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circulation&title=Progeria%20Clinical%20Trials%20C.%20Impact%20of%20farnesylation%20inhibitors%20on%20survival%20in%20Hutchinson-Gilford%20progeria%20syndrome&author=LB%20Gordon&author=J%20Massaro&author=RB%20D'Agostino&author=SE%20Campbell&author=J%20Brazier&volume=130&publication_year=2014&pages=27-34&pmid=24795390&doi=10.1161/CIRCULATIONAHA.113.008285&)

38. Oshima J, Hisama FM, Martin GM. An encouraging progress report on the treatment of progeria and its implications for atherogenesis. Circulation. 2014;130:4–6. doi: 10.1161/CIRCULATIONAHA.114.010648.  [DOI](https://doi.org/10.1161/CIRCULATIONAHA.114.010648) | [PMC free article](/articles/PMC4322865/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24795391/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circulation&title=An%20encouraging%20progress%20report%20on%20the%20treatment%20of%20progeria%20and%20its%20implications%20for%20atherogenesis&author=J%20Oshima&author=FM%20Hisama&author=GM%20Martin&volume=130&publication_year=2014&pages=4-6&pmid=24795391&doi=10.1161/CIRCULATIONAHA.114.010648&)
