# Potentially Functional SNPs (pfSNPs) as Novel Genomic Predictors of 5-FU Response in Metastatic Colorectal Cancer Patients

## Metadata
**Authors:** Jingbo Wang, Xu Wang, Mingjue Zhao, Su Pin Choo, Sin Jen Ong, Simon Y K Ong, Samuel S Chong, Yik Ying Teo, Caroline G L Lee
**Journal:** PLoS ONE
**Date:** 2014 Nov 5
**DOI:** [10.1371/journal.pone.0111694](https://doi.org/10.1371/journal.pone.0111694)
**PMID:** 25372392
**PMCID:** PMC4221105
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4221105/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC4221105/pdf/pone.0111694.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC4221105/pdf/pone.0111694.pdf)

## Abstract

5-Fluorouracil (5-FU) and its pro-drug Capecitabine have been widely used in treating colorectal cancer. However, not all patients will respond to the drug, hence there is a need to develop reliable early predictive biomarkers for 5-FU response. Here, we report a novel potentially functional Single Nucleotide Polymorphism (pfSNP) approach to identify SNPs that may serve as predictive biomarkers of response to 5-FU in Chinese metastatic colorectal cancer (CRC) patients. 1547 pfSNPs and one variable number tandem repeat (VNTR) in 139 genes in 5-FU drug (both PK and PD pathway) and colorectal cancer disease pathways were examined in 2 groups of CRC patients. Shrinkage of liver metastasis measured by RECIST criteria was used as the clinical end point. Four non-responder-specific pfSNPs were found to account for 37.5% of all non-responders (P<0.0003). Five additional pfSNPs were identified from a multivariate model (AUC under ROC = 0.875) that was applied for all other pfSNPs, excluding the non-responder-specific pfSNPs. These pfSNPs, which can differentiate the other non-responders from responders, mainly reside in tumor suppressor genes or genes implicated in colorectal cancer risk. Hence, a total of 9 novel SNPs with potential functional significance may be able to distinguish non-responders from responders to 5-FU. These pfSNPs may be useful biomarkers for predicting response to 5-FU.

## Introduction

Every year, more than one million individuals worldwide will develop colorectal cancer [[1]](#pone.0111694-Cunningham1), accounting for 10% of the global cancer burden. Colorectal cancer (CRC) is the most frequently diagnosed cancer in Singapore (7,909 new cases between 2005–2009) [[2]](#pone.0111694-SC1). More than half of CRC patients develop metastatic disease (stage 4) either at diagnosis or at relapse following initial curative intent therapy. This translates to a substantial proportion of patients who may need treatment for the metastasis or relapse of colorectal cancer.

5-fluorouracil (5-FU) and its pro-drug, Capecitabine, are widely used in treating CRC. It has been proposed that there are two distinct modes of action for 5-FU. First, it acts as anti-metabolite whereby its active form, FdUMP, produced by Thymidine Phosphorylase (TYMP), inhibits Thymidylate Synthase (TYMS). Second, it can induce cell death, whereby incorporation of its active products FUTP and FdUTP into RNA and DNA, respectively, leads to subsequent cell apoptosis [[3]](#pone.0111694-Sobrero1). Uridine Monophosphate Synthetase (UMPS, also known as OPRT) is responsible for converting 5-FU to FUMP, which is the first step of producing FUTP and FdUTP. However, the two pathways may overlap, because the intermediate product in the “cell toxicity” pathway, FUDP, may also be converted to FdUDP and subsequently FdUMP and participate in the “anti-metabolite” pathway. 5-FU is catabolized into the inactive form of DHFU by Dihydropyrimidine Dehydrogenase (DPYD), and DPYD is the rate-limiting enzyme in degrading 5-FU.

At present, there are no reliable tests for early prediction of response to 5-FU. Developing a reliable early predictive biomarker of response to common chemotherapy, like 5-FU, in metastatic colorectal cancer has the potential to lead to appropriate tailoring of treatment for individual patients and help move us closer to a truly personalized care. Overall economic cost benefits are realized in both predicted responders and non-responders. Responders get appropriate treatment with confidence of anticipated response. Predicted non-responders to conventional treatment avoid wasted expense of 3 cycles of futile treatment, unnecessary toxicities that themselves require remedies and time loss in terms of futile treatment and loss of a window of opportunity for effective treatment. These patients may be selected as candidates for novel therapies and combination chemotherapy.

Thus far, only variants in the “5-FU PD” genes were reported to be significantly associated with 5-FU efficacy measured by tumour shrinkage in some studies [[4]](#pone.0111694-Graziano1)–[[7]](#pone.0111694-Jakobsen1). Unfortunately, replication of such reported association between “5-FU PD” gene variants and tumour shrinkage remains challenging [[4]](#pone.0111694-Graziano1), [[6]](#pone.0111694-Etienne1), [[8]](#pone.0111694-FarinaSarasqueta1)–[[13]](#pone.0111694-Cohen1), suggesting the possible presence of other loci in determining 5-FU efficacy.

It was interesting to note that the variants in the “5-FU PK” genes were mainly investigated for their association with 5-FU toxicity, not efficacy [[14]](#pone.0111694-Ichikawa1)–[[17]](#pone.0111694-Schwab1), despite the expression levels of these genes having been previously associated with 5-FU efficacy [[18]](#pone.0111694-Koopman1). Limited efforts [[8]](#pone.0111694-FarinaSarasqueta1), [[19]](#pone.0111694-Lund1) attempted to explore the possible association for efficacy, but failed.

Furthermore, most studies on response to 5-FU treatment focused primarily on SNPs in a few candidate genes. Only one study examined 21 variants primarily in coding region of 11 genes involved in metabolism/action of 5-FU and other related pharmacological pathways [[19]](#pone.0111694-Lund1). However, this study still does not comprehensively interrogate all possible variants that may be involved in 5-FU response.

Another limitation of current studies is that only univariate analyses have, thus far, been employed and this may not have sufficient power for detection of association of drug response with less common/rare SNPs with small sample size. Multivariate model was successfully employed to estimate the appropriate dose of warfarin based on clinical and genetic data [[20]](#pone.0111694-Deng1).

Hence, in this study, we employ a novel approach interrogating potentially functional SNPs (pfSNPs) in relevant drug and disease pathway to identify association with drug response. 1,547 potentially functional SNPs+1 VNTR (Variable Number Tandem Repeat) from 139 genes in the drug (both PK and PD pathway) and disease pathways were examined. Potentially functional SNPs were identified using the pfSNP Web Resource ([http://pfs.nus.edu.sg/](http://pfs.nus.edu.sg/)) [[21]](#pone.0111694-Wang1) which included SNPs that were previously reported to be functional or associated with disease/drug response; SNPs that were inferred to be potentially functional from genetic approaches as well as those predicted to be potentially functional from sequence motifs.

As the number of samples was limited, a two-step study design was employed. In the first stage, we examined 62 patients who were only on Capecitabine, a pro-drug of 5-FU, to identify interesting SNPs that are marginally associated with drug response as measured by tumor shrinkage. These SNPs were then examined in another group of 27 patients who were treated with 5-FU and oxaliplatin. Combined Multivariate and Collapsing (CMC) analysis was employed to evaluate the less common (≤5%), non-responder-specific pfSNPs for their association with 5-FU drug response while a logistic regression based multivariate model using stepwise Akaike Information Criterion (stepAIC) procedure was used to interrogate the other pfSNPs for their association with 5-FU drug response in patients that do not carry the non-responder-specific pfSNPs.

## Materials and Methods

### Patient samples and clinical parameters

The shrinkage of liver metastatic CRC tumour was used as the clinical endpoint for measuring treatment efficacy. Response Evaluation Criteria In Solid Tumours (RECIST) [[22]](#pone.0111694-Sohaib1) is used to determine tumour response. In the patients recruited, there was no patient belonging to the ‘Complete Response’ category. Patients with ‘Partial Response’ were deemed as ‘Responders’ and patients with ‘Progressive Disease’ or ‘Stable Disease’ were classified as ‘Non-Responders’ in the association analysis.

A total of 89 unrelated Chinese metastatic CRC patients were recruited. All patients had liver metastasis and were given neo-adjuvant chemotherapy prior to operation for the liver lesion. [Table 1](#pone-0111694-t001) shows the characteristics of the patients in each study.

### Table 1. The demographic characteristics of the patients recruited for each study.

|   | Group 1 | Group 2 |
| --- | --- | --- |
| Number of Patients | 62 | 27 |
| Ages (Median) | 36–78 (60) | 42–86 (62) |
| Males (Females) | 49(13) | 18(9) |
| Prior Drug Exposure |   |   |
| 5-FU alone | 0 | 2 |
| Capecitabine alone | 0 | 3 |
| 5-FU + oxaliplatin | 0 | 2 |
| Capecitabine + oxaliplatin | 0 | 0 |
| 5-FU + Radio Therapy | 0 | 2 |
| Drugs Treated |   |   |
| 5-FU alone | 0 | 2 |
| Capecitabine alone | 62 | 1 |
| 5-FU + oxaliplatin | 0 | 15 |
| Capecitabine + oxaliplatin | 0 | 8 |
| 5-FU + Irinotecan | 0 | 1 |
| Response |   |   |
| Partial response | 13 | 12 |
| Stable disease | 31 | 7 |
| Progressive disease | 18 | 8 |
In group 1, 62 unrelated Stage IV CRC Chinese patients were recruited. These patients were treated with only Capecitabine and they have never been previously exposed to this drug. Of these 62 patients, 13 had partial response, 31 had stable disease, and 18 had progressive disease. Hence, the response rate of this group of patients is only ∼21%, which is typical for single-agent treatment [[23]](#pone.0111694-VanCutsem1). ∼79% of the patients were male, and the median age of the cohort was 60 years.

In group 2, 27 unrelated Chinese liver metastatic CRC patients who were treated with 5-FU (a few had Capecitabine) alone (a few) or with oxaliplatin regime (most) as their neo-adjuvant chemotherapy were examined. Some of these patients had also been previously exposed to these drugs. Of these 27 patients, 12 had partial response, seven had stable disease, and eight had progressive disease. Hence, the response rate was ∼45%, which is typical for patients undergoing 2-drug combination therapy [[24]](#pone.0111694-deGramont1). Two-thirds of these patients are males, and had a median age of 62 years.

### Ethics statement

This study has been approved by Singhealth Centralized Institutional Review Board (CIRB) (Reference No: NC05–22 and 2005/421/B).

### Selection of potentially functional SNPs (pfSNPs) for association study

The pfSNP resource ([http://pfs.nus.edu.sg/](http://pfs.nus.edu.sg/)) [[21]](#pone.0111694-Wang1) was employed to identify SNPs in genes associated with 5-FU/Capecitabine, oxaliplatin as well as colorectal cancer. Approximately 2800 pfSNPs in 214 genes were found to be associated with keywords including “fluorouracil”, “5-fluorouracil”, “capecitabine”, “platinum”, “oxaliplatin” as well as “colorectal cancer”. As only 1,536 SNPs can be genotyped within a single customized GoldenGate Genotyping Array (Illumina, Inc), the following criteria were employed to select a subset of these 2800 pfSNPs: all SNPs within the promoter, coding, 5′/3′ un-translated regions which has a GoldenGate Score (GGS: measure of assay quality by their platform) of greater than 0.5 were selected. For the introns, pfSNP with a GGS>0.7 were selected and monomorphic ones reported in HapMap CHB population were excluded, except for those previously reported as functional. For any adjacent SNPs that may interfere with each other in the assay, we selected the SNP according to the following order: “Previously reported → Non-Synonymous → Synonymous → UTR → Intron”. A list of all of the SNPs included on the GoldenGate array is available as **Table S1**.

Among the markers that were unsuitable to be genotyped by GoldenGate assay, 14 important markers in 5-FU response prediction were selected to be genotyped by other methods (listed in **Table S2**). The 14 markers include the previously well studied VNTR with embedded SNP [[25]](#pone.0111694-Kawakami1) as well as the 6bps 3′ UTR indel in the TYMS gene because GoldenGate technology could only genotype SNPs. Other markers are SNPs with low GoldenGate scores in the TYMS, TYMP and DPYD genes. A customized Sequenom’s MassARRAY panel was used to genotype 11 of the 14 markers and 1 SNP was genotyped by ABI TaqMan. In addition, we developed a novel method to genotype the VNTR (rs2853542) and embedded SNP (rs34743033) in the TYMS gene. This VNTR can have either 2 or 3 repeats, and the SNP can occur in both the second and third repeat [[26]](#pone.0111694-Kawakami2). We developed a robust method using Sanger Sequencing for this purpose. A fragment of 486 bp was amplified using the primers (F- *CTGCTGGCTTAGAGAAGGCG* and R- *AGCGGAGGATGTGTTGGATC*) and the amplicon was sequenced in both directions using the forward and reverse primers. Different genotypes would yield distinct patterns on the forward and reverse sequencing reads (As shown in **Figure S1**), allowing the genotype to be easily deduced.

In summary, there were 1,536 markers (Listed in **Table S1**) genotyped with a single customized Illumina GoldenGate SNP genotyping array, 11 (Listed in **Table S2**) by Sequenom’s MassARRAY, 1 (rs11479) by ABI TaqMan and the VNTR (rs2853542), with the embedded SNP (rs34743033) was genotyped using Sanger Sequencing.

The distribution of the SNPs and genes selected in the three categories (CRC, Fluorouracil and Platinum related) is depicted in **Figure S2**. More than half of the SNPs (863) are from fluorouracil-related genes and 702 SNPs are from platinum-related genes. A considerable portion of the SNPs (656) are from CRC-related genes, although 40% (32 out of 80) of these genes and 88% (579 out of 656) of the SNPs are related to fluorouracil and/or platinum as well.

The numbers of SNPs in each gene region and function category covered in this study are shown in **Figure S3**. Each of the four gene regions, namely promoter, coding, intron and 3′ UTR, are adequately covered in general. For promoter and 3′UTR, most of the SNPs genotyped are those that change TF binding sites. In the coding region, SNPs that change ESE/ESS sites are the most abundant, and non-synonymous SNPs that cause deleterious effects are the second most abundant. The intron region SNPs are enriched with those with a signature of recent positive selection.

The distribution of SNP minor allele frequency for the intronic versus non-intronic region is shown in **Table S3**. For coding, 3′UTR, and promoter regions, a number of SNPs not previously genotyped by HapMap has been genotyped in this study. As the study aims to also explore rare variants, a number of SNPs reported by HapMap to be monomorphic was also genotyped. For the intron region, since most of the SNPs are those with a signature of recent positive selection, they have MAF more than 5%. We did not genotype many monomorphic ones within introns.

### Single marker association analysis

Hardy-Weinberg equilibrium and minor allele frequency were analysed using the Microsoft Excel-based SNP Statistics Calculator developed in-house. Single marker association analysis was performed using PLINK [[27]](#pone.0111694-Purcell1). The P-value was calculated by the permutation-based method, and the Odds Ratio (OR) and corresponding 95% confidence interval were determined using regular allele-based association analyses, because the permutation-based method does not provide such information. The genotype of the VNTR (rs2853542) and embedded SNP (rs34743033) in the TYMS gene is re-coded as a bi-allelic SNP comprising the high-expression allele and the low-expression allele according to Kawakami et al [[26]](#pone.0111694-Kawakami2).

### Combined Multivariate and Collapsing (CMC) analyses for non-responder-specific SNPs

Combined Multivariate and Collapsing (CMC) method was first proposed as a method for analysing rare SNPs [[28]](#pone.0111694-Li1). CMC utilizes Hotelling’s T^2^ test to analyse more than 2 groups of collapsed variants. When only 2 groups of such variants are analysed, Fisher’s exact test can be used. In this study, we used the Fisher’s exact test to evaluate if the collapsed minor allele of non-responder specific SNPs which are less common (≤5% minor allele frequency) would be a good indicator of responsiveness to 5-FU.

### Logistic regression based multivariate model using Akaike Information Criterion (stepAIC) procedure to predict drug response in patients who do not have non-responder-specific SNPs

Patients, who do not have any of the non-responder specific SNPs, were divided into 2 groups. Data from the first group comprising 80% of the patients was used to train a logistic regression based multivariate model by using the Akaike’s Information Criterion (AIC) in a stepwise algorithm (using R package “stepAIC”) while the data from the other 20% of patients were used to validate the model. The selected model was evaluated by the Area Under the Curve (AUC) of the Receiver Operating Characteristic (ROC) curves. The optimal cut-off point for the logistic regression was chosen where the maximum sum of sensitivity and specificity is obtained [[29]](#pone.0111694-Zou1), [[30]](#pone.0111694-Patti1).

## Results and Discussion

A high concordance (R^2^ = 0.9494) was observed between allele frequencies of SNPs in our study and those from the CHB (Chinese in Beijing) population in HapMap (Release 27) (**Figure S4**) affirming the quality of our genotyping. A large proportion of the SNPs examined in this study were either monomorphic (36%) or had a high minor allele frequency (MAF≥0.1, 46%) (**Figure S5**). Seventy-two and 66 SNPs in Groups 1 and 2, respectively, were found to significantly deviate from Hardy-Weinberg Equilibrium and were excluded from further analysis.

Genotype was successfully (97–100%) assigned in 9 out of 11 SNPs genotyped using the Sequenom’s MassARRAY. Sanger sequencing successfully assigned genotype of the 2 SNPs to all the samples while TaqMan assay successfully assigned genotypes to 96% of the samples. All the SNPs successfully genotyped by these methods were found to be in Hardy-Weinberg equilibrium.

### Single SNP association analysis identified three non-responder specific SNPs in the UMPS gene that may represent potential predictive biomarker for non-response to 5-FU

As the number of samples in this study was small, a cross validation approach was employed where the samples were segregated into 2 distinct groups for discovery and validation to enhance the robustness of our findings.

A total of 36 SNPs in 12 genes were found to be associated with drug response before multiple test correction in Group 1 patients ([Table 2](#pone-0111694-t002)). As the sample size was small (n = 62), none of these markers were statistically significant after Bonferroni correction.

### Table 2. List of SNPs showing P<0.05 in Group 1 ranked by P value.

|   |   |   |   |   |   |   | Allele Count |  |   |   |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| SN | Gene | mRNA Location | AA Change | rs No. | Function Summary | P | NR (98) | Rsp (26) | OR | OR 95 CI |  |
| 1 | ATP7A | E23/3UTR/T1960C | – | rs1062472 | RPS | 0.003 | 22 | 14 | 4.03 | 1.55 | 9.29 |
| 2 | RRM1 | 5UR//A-2723G | – | rs3750996 | TF | 0.004 | 30 | 1 | 11.36 | 1.46 | 86.66 |
| 3 | DLG5 | E/23/G4442T | P1481Q | rs2289310 | Reported,Non-Syn,ProteinDomain,Del AA, ESE/ESS | 0.005 | 15 | 11 | 4.06 | 1.53 | 10.13 |
| 4 | RRM1 | 5UR//T-265G | – | rs1735068 | TF | 0.010 | 19 | 12 | 3.56 | 1.26 | 7.70 |
| 5 | RRM1 | 5UR//A-4023G | – | rs3794050 | TF | 0.012 | 20 | 12 | 3.34 | 1.26 | 7.70 |
| 6 | RRM1 | 5UR//T-659C | – | rs1662162 | TF | 0.012 | 20 | 12 | 3.34 | 1.20 | 7.24 |
| 7 | UMPS | 5UR//T-1256A | – | rs12492095 | TF | 0.013 | 11 | 9 | 4.19 | 1.45 | 10.88 |
| 8 | UMPS | E/3/G638C | – | rs1801019 | Reported,Non-Syn,ESE/ESS | 0.013 | 11 | 9 | 4.19 | 1.45 | 10.88 |
| 9 | APC | I/2/C-230A | – | rs2464805 | RPS, ISRE | 0.018 | 6 | 6 | 4.60 | 1.40 | 16.42 |
| 10 | SMARCA2 | I/28/G-8275T\|I/29/G2824T | –\|– | rs7048976 | RPS | 0.021 | 39 | 4 | 3.64 | 1.09 | 10.62 |
| 11 | RRM1 | 5UR//G-2528A | – | rs1561876 | TF | 0.023 | 21 | 12 | 3.14 | 1.20 | 7.24 |
| 12 | RRM1 | E/19/G2232A | A744A | rs1042858 | ESE/ESS | 0.023 | 21 | 12 | 3.14 | 1.14 | 6.83 |
| 13 | RRM1 | E19/3UTR/C316A | – | rs1042927 | miRNA | 0.023 | 21 | 12 | 3.14 | 1.14 | 6.83 |
| 14 | WDR7 | I/13/A323G | – | rs11664579 | Reported | 0.024 | 10 | 8 | 3.91 | 1.30 | 10.42 |
| 15 | TFRC | E/4/C424T | S142G | rs3817672 | Reported,Non-Syn,ProteinDomain | 0.025 | 14 | 9 | 3.18 | 1.24 | 8.91 |
| 16 | ATP7A | E/10/G2299C | V767L | rs2227291 | Non-Syn,ProteinDomain,Del AA, ESE/ESS | 0.025 | 22 | 12 | 2.96 | 1.14 | 6.83 |
| 17 | ATP7A | I/12/C-882A | – | rs17139617 | RPS | 0.025 | 22 | 12 | 2.96 | 1.14 | 6.83 |
| 18 | ABCC4 | 3DR//A75075G\|E31/3UTR/A879G | –\|– | rs1059751 | Reported | 0.025 | 38 | 17 | 2.98 | 1.19 | 7.20 |
| 19 | ABCC4 | 3DR//A74234C\|E31/3UTR/A38C | –\|– | rs3742106 | miRNA | 0.029 | 40 | 17 | 2.74 | 1.10 | 6.63 |
| 20 | ABCC4 | 3DR//T74507C\|E31/3UTR/T311C | –\|– | rs4148551 | Reported | 0.029 | 40 | 17 | 2.74 | 1.10 | 6.63 |
| 21 | APC | E/16/T5465A | V1822D | rs459552 | Reported,Non-Syn,ProteinDomain, ESE/ESS | 0.033 | 5 | 5 | 4.43 | 1.23 | 17.40 |
| 22 | APC | I/6/T-3774C | – | rs2431238 | RPS | 0.033 | 5 | 5 | 4.43 | 1.23 | 17.40 |
| 23 | CDC2 | 5UR//C-3953T\|I/1/C263T | –\|– | rs2448341 | TF, ISRE | 0.039 | 42 | 5 | 3.15 | 1.07 | 8.76 |
| 24 | RRM1 | 5UR//C-3890T | – | rs7934581 | TF | 0.043 | 13 | 8 | 2.91 | 0.95 | 6.98 |
| 25 | ERCC6 | I/6/T871G | – | rs4253101 | RPS | 0.045 | 50 | 7 | 2.83 | 1.14 | 7.59 |
| 26 | SLCO6A1 | I/10/A-2493G | – | rs1562961 | RPS | 0.046 | 38 | 16 | 2.53 | 1.02 | 6.01 |
| 27 | SLCO6A1 | I/11/C702T | – | rs10062613 | RPS | 0.046 | 38 | 16 | 2.53 | 1.02 | 6.01 |
| 28 | SLCO6A1 | I/12/A7020C | – | rs6873738 | RPS | 0.046 | 38 | 16 | 2.53 | 1.02 | 6.01 |
| 29 | SLCO6A1 | I/12/G-738T | – | rs6877722 | RPS | 0.046 | 38 | 16 | 2.53 | 1.02 | 6.01 |
| 30 | SLCO6A1 | I/12/T517C | – | rs1901512 | RPS | 0.046 | 38 | 16 | 2.53 | 1.02 | 6.01 |
| 31 | SLCO6A1 | I/3/T142C | – | rs10041525 | RPS, ISRE | 0.046 | 38 | 16 | 2.53 | 1.02 | 6.01 |
| 32 | SLCO6A1 | I/3/T220C | – | rs10041507 | RPS, ISRE | 0.046 | 38 | 16 | 2.53 | 1.02 | 6.01 |
| 33 | SLCO6A1 | I/4/A-9G | – | rs11746217 | RPS, ISRE | 0.046 | 38 | 16 | 2.53 | 1.02 | 6.01 |
| 34 | SLCO6A1 | I/6/A4781G | – | rs1452057 | RPS | 0.046 | 38 | 16 | 2.53 | 1.02 | 6.01 |
| 35 | SLCO6A1 | I/9/G1986T | – | rs1901521 | RPS | 0.046 | 38 | 16 | 2.53 | 1.02 | 6.01 |
| 36 | SLCO6A1 | I/9/T5068G | – | rs1901522 | RPS | 0.046 | 38 | 16 | 2.53 | 1.02 | 6.01 |

Table 2 Caption: Abbreviations: Reported: previously reported in the literature to be associated with disease/function; ESE/ESS: change Exon splice enhancer/silencer; NMD: mRNA nonsense mediated decay; Non-syn: non-synonymous SNP; ProteinDomain: residing in important protein domains; miRNA: change miRNA binding site; RPS: show signature of recent positive selection; TF: change transcription factor binding site; ISRE: change intron splice regulatory element.

Nonetheless, there were 68 low frequency pfSNPs in Group 1 that were uniquely found only in non-responders (non-responder-specific pfSNP). To evaluate if any of these non-responder-specific pfSNPs may represent potential predictive biomarker for non-response to 5-FU, we examined a second group of 27 patients. Of these 68 non-responder specific pfSNPs, 24 remained non-responder-specific even in Group 2 and these are presented in [Table 3](#pone-0111694-t003). However, only 3 of the non-responder-specific SNPs in the Uridine Monophosphate Synthetase (UMPS) gene, namely rs2291078 (E/4/T1050A C350*), rs3772809 (E/6/A1336G H446Y) and rs3772810 (E6/3UTR/A28G) ([Table 2](#pone-0111694-t002) **,** SNPs 42–44) ([Table 3](#pone-0111694-t003), SNPs 1–3) were found to be statistically significant (p = 0.036 before multiple test correction) in Group 2. When the 2 groups of patients were combined and analysed, these 3 SNPs remained statistically significant (p = 0.032 before multiple test correction). The observation that out of a total of 89 patients in the combined group, no responders were found to carry this allele suggests that this allele may be a “causal” allele for determining the response to 5-FU. The non-statistical significant data obtained (after multiple test correction) suggests that this may not be the only “causal” alleles for 5-FU response and there are likely other alleles that also play a role in 5-FU response suggesting “locus heterogeneity” of response to 5-FU.

### Table 3. List of SNPs with non-responder specific allele in both groups.

|   |   |   |   |   | MAF |  |   | Group 1 |  |  | Group 2 |  |  | Combined |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
|   |   |   |   |   |   |   |   |   | Allele Count |  |   | Allele Count |  |   | Allele Count |  |
| SN | Gene | mRNA Location | AA Change | rs No. | HapMap | This study | Function Summary | P | NR (98) | Rsp (26) | P | NR (30) | Rsp (24) | P | NR (128) | Rsp (50) |
| 1 | UMPS | E/4/T1050A | C350* | rs2291078 | – | 7.3% | NMD, Non-Syn,ProteinDomain,ESE/ESS | 0.140 | 8 | 0 | 0.036 | 5 | 0 | 0.032 | 13 | 0 |
| 2 | UMPS | E/6/A1336G | H446Y | rs3772809 | 5.4% | 7.3% | Non-Syn,ProteinDomain,ESE/ESS | 0.140 | 8 | 0 | 0.036 | 5 | 0 | 0.032 | 13 | 0 |
| 3 | UMPS | E6/3UTR/A28G | – | rs3772810 | 5.4% | 7.3% | miRNA, TF | 0.140 | 8 | 0 | 0.036 | 5 | 0 | 0.032 | 13 | 0 |
| 4 | TK1 | 5UR//G-1181A | – | rs8071253 | – | 3.4% | TF | 0.250 | 5 | 0 | 0.367 | 1 | 0 | 0.119 | 6 | 0 |
| 5 | DPYD | I/3/A22444C | – | rs10493895 | 1.8% | 2.8% | Reported | 0.305 | 4 | 0 | 0.367 | 1 | 0 | 0.156 | 5 | 0 |
| 6 | DPYD | I/3/A39932G | – | rs10747486 | 1.9% | 2.8% | RPS | 0.305 | 4 | 0 | 0.367 | 1 | 0 | 0.156 | 5 | 0 |
| 7 | DPYD | I/3/C30060T | – | rs1931063 | 2.2% | 2.8% | RPS | 0.305 | 4 | 0 | 0.367 | 1 | 0 | 0.156 | 5 | 0 |
| 8 | DPYD | I/3/G-27495C | – | rs4537601 | 1.8% | 2.8% | RPS | 0.305 | 4 | 0 | 0.367 | 1 | 0 | 0.156 | 5 | 0 |
| 9 | DPYD | I/3/G29092A | – | rs1333717 | 1.8% | 2.8% | RPS | 0.305 | 4 | 0 | 0.367 | 1 | 0 | 0.156 | 5 | 0 |
| 10 | WDR7 | I/1/A35G | – | rs501415 | 2.3% | 2.8% | TF, ISRE | 0.305 | 4 | 0 | 0.367 | 1 | 0 | 0.156 | 5 | 0 |
| 11 | DPYD | I/4/A5229G | – | rs6683957 | 0.7% | 2.2% | RPS | 0.376 | 3 | 0 | 0.367 | 1 | 0 | 0.206 | 4 | 0 |
| 12 | DPYD | I/4/G3787C | – | rs4970728 | 2.2% | 2.2% | RPS | 0.376 | 3 | 0 | 0.367 | 1 | 0 | 0.206 | 4 | 0 |
| 13 | REV3L | E/26/C8285T | R2762Q | rs3218592 | 2.9% | 2.2% | Non-Syn,ProteinDomain,DelAA, ESE/ESS | 0.376 | 3 | 0 | 0.367 | 1 | 0 | 0.206 | 4 | 0 |
| 14 | WDR7 | E27/3UTR/C112T\|E28/3UTR/C112T | –\|– | rs3745032 | – | 2.2% | TF | 0.376 | 3 | 0 | 0.367 | 1 | 0 | 0.206 | 4 | 0 |
| 15 | WDR7 | E27/3UTR/C2176G\|E28/3UTR/C2176G | –\|– | rs3745030 | 3.3% | 2.2% | TF | 0.376 | 3 | 0 | 0.367 | 1 | 0 | 0.206 | 4 | 0 |
| 16 | WDR7 | I/14/T9751A | – | rs11876256 | 2.2% | 2.2% | RPS | 0.376 | 3 | 0 | 0.367 | 1 | 0 | 0.206 | 4 | 0 |
| 17 | WDR7 | I/19/T-30795A\|I/20/T-30795A | –\|– | rs2576415 | 2.2% | 2.8% | RPS | 0.376 | 3 | 0 | 0.197 | 2 | 0 | 0.156 | 5 | 0 |
| 18 | WDR7 | I/12/C-2229T | – | rs11877604 | 2.2% | 1.7% | RPS | 0.467 | 2 | 0 | 0.367 | 1 | 0 | 0.271 | 3 | 0 |
| 19 | SMARCD1 | E/4/A423G | V141V | rs2307083 | 0.0% | 1.1% | CodonDiff, TF | 0.612 | 1 | 0 | 0.367 | 1 | 0 | 0.374 | 2 | 0 |
| 20 | UPB1 | 5UR//G-96A | – | rs2232861 | – | 1.1% | TF | 0.612 | 1 | 0 | 0.367 | 1 | 0 | 0.374 | 2 | 0 |
| 21 | WDR7 | I/18/A-2783C\|I/19/A-2783C | –\|– | rs9946253 | 0.4% | 1.1% | Reported | 0.612 | 1 | 0 | 0.367 | 1 | 0 | 0.374 | 2 | 0 |
| 22 | WDR7 | I/20/A20363G\|I/21/A20363G | –\|– | rs2083020 | 0.4% | 1.1% | Reported | 0.612 | 1 | 0 | 0.367 | 1 | 0 | 0.374 | 2 | 0 |
| 23 | WDR7 | I/21/A532G\|I/22/A532G | –\|– | rs8094838 | 0.4% | 1.1% | Reported | 0.612 | 1 | 0 | 0.367 | 1 | 0 | 0.374 | 2 | 0 |
| 24 | WDR7 | I/21/T-2912G\|I/22/T-2912G | –\|– | rs6566846 | 0.0% | 1.1% | Reported, RPS | 0.612 | 1 | 0 | 0.367 | 1 | 0 | 0.374 | 2 | 0 |

Table 3 Caption: Abbreviations: Reported: previously reported in the literature to be associated with disease/function; ESE/ESS: change Exon splice enhancer/silencer; NMD: mRNA nonsense mediated decay; Non-syn: non-synonymous SNP; ProteinDomain: residing in important protein domains; miRNA: change miRNA binding site; RPS: show signature of recent positive selection; TF: change transcription factor binding site; ISRE: change intron splice regulatory element; DelAA: Deleterious amino acid change; CodonDiff: High codon usage difference.

The UMPS gene is important in determining 5-FU response, as it converts 5-FU into its active metabolite, FUMP, which can participate in both the cell toxicity pathway as well as in the “anti-metabolite” pathway. In the cell toxicity pathway, FUMP can be further converted into FUTP and FdUTP which is then incorporated into RNA and DNA respectively. In the “anti-metabolite” pathway, FUMP can be converted into FdUMP and inhibits TYMS.

These 3 alleles in the UMPS gene are in perfect linkage disequilibrium (LD) and hence will occur together all the time. The predicted molecular functions of the three alleles unique to the non-responders are all associated with the disruption of the UMPS gene function and support their unique presence in the non-responders ([Figure 1](#pone-0111694-g001)).

### Figure 1. The molecular functions of the three SNPs in UMPS gene with minor allele uniquely found in non-responders are all linked to disabling UMPS.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e040/4221105/edaa874c22e7/pone.0111694.g001.jpg)

The A allele of rs2291078 (UMPS E/4/T1050A C350*) was predicted to create a stop codon in exon 4 of the UMPS mRNA. UMPS mRNA containing this stop codon may be quickly degraded since stop codon appearing more than 50 bps from the last exon-exon junction would induce non-sense mediated decay of mRNA [[31]](#pone.0111694-Nagy1). Therefore, patients carrying this A allele may have lower UMPS mRNA and protein abundance.

The co-occurrence of 3′UTR pfSNP rs3772810 (UMPS E6/3UTR/A28G) with this pfSNP rs2291078 suggests that the expression of this gene may be further attenuated. The G allele of the 3′UTR SNP rs3772810 (UMPS E6/3UTR/A28G) is predicted to create binding sites for miRNA 23a, 23b and 130a*. The miRNA 23a is shown to be up-regulated under hypoxic condition commonly found in tumors [[32]](#pone.0111694-Kulshreshtha1). Notably, a recent publication reported that miRNA 23a is up-regulated in metastatic colorectal cancer [[33]](#pone.0111694-Jahid1) suggesting that patients with the G allele may be non-responsive to 5-FU since miRNA 23a may suppress the expression of UMPS mRNA containing the G allele.

Also co-occurring with these 2 pfSNPs, is the non-synonymous pfSNP rs3772809 (UMPS E/6/A1336G H446Y) which have the potential to alter the function of the UMPS gene. Hence, these 3 co-occurring pfSNPs, which accounted for 17.2% of all the non-responders have the potential to be the causal variants affecting the function of UMPS and thus response to 5-FU although further experiments are required to validate the potential functionality of these pfSNPs.

### Combined Multivariate and Collapsing (CMC) analyses revealed that a minimum of four non-responder specific pfSNPs that are not in linkage equilibrium can significantly distinguish non-responders from responders

We proceeded to determine if combination of non-responder specific pfSNPs ([Table 3](#pone-0111694-t003)) can account for a greater percentage of 5-FU non-responders than the abovementioned 3 non-responder specific UMPS pfSNPs in perfect LD that show statistical significance (before multiple test corrections). Since the 3 non-responder specific UMPS pfSNPs are in perfect LD, only one was selected for further analyses. We then identify the minimum number of additional non-responder specific pfSNPs from [Table 3](#pone-0111694-t003) that can account for the maximum percentage of 5-FU non-responders and employed Combined Multivariate and Collapsing (CMC) analyses [[28]](#pone.0111694-Li1) to determine its statistical significance.

Notably, three other non-responder specific pfSNPs together with any one of the UMPS non-responder specific pfSNPs were found to account for 37.5% of all non-responders from the 2 groups of patients ([Table 4](#pone-0111694-t004)). CMC analyses revealed statistical significance (P = 0.0003) of these 4 non-responder specific pfSNPs (3 non UMPS plus any one of the 3 UMPS pfSNPs) suggesting significant association of these pfSNPs with non-responsiveness.

### Table 4. The list of less-common SNPs with non-responder specific allele and their predicted molecular functions.

| S.N | rsNo | Gene Name | mRNA Location | AA Change | Non-responder Specific Allele and Frequency (HapMap/This Study) | Allele Count |  | P | Potential Molecular Function |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
|   |   |   |   |   |   | NR | Rsp |   |   |
| 1 | rs3772810 | UMPS | E6/3UTR/A28G | – | G (5.4%/7.3%) | 13 | 0 | 0.02 | Creates miR23a binding site |
| 2 | rs3218592 | REV3L | E/26/C8285T | R2762Q | T (1.8%/2.2%) | 4 | 0 | 0.45 | Predicted deleterious by Polyphen and SIFT |
| 3 | rs8071253 | TK1 | 5UR//G-1181A | – | A (Unknown/3.4%) | 6 | 0 | 0.26 | Creates TGA1a binding site |
| 4 | rs501415 | WDR7 | I/1/A35G | – | G (2.3%/2.8%) | 5 | 0 | 0.41 | Disrupts a binding site of RUNX1 and possbily RUNX3 (Since all RUNX family proteins are sharing same motif) |
SNP rs3218592 causes a non-conservative amino acid change in the REV3L gene which encodes the catalytic subunit of DNA Polymerase Zeta. DNA Polymerase Zeta was reported to be significantly down regulated in human colorectal cancer [[34]](#pone.0111694-Pan1) and was suggested to be a tumour suppressor [[35]](#pone.0111694-Zhang2). The T allele of the rs3218592 (E/26/C8285T, R2762Q) which is uniquely found in non-responder in our study is predicted to be damaging to the protein function by both Polyphen [[36]](#pone.0111694-Adzhubei1) and SIFT [[37]](#pone.0111694-Sim1). We thus hypothesize that the patients carrying this deleterious allele would have more aggressive disease and hence are more likely to be non-responsive to treatment.

pfSNP (rs8071253) resides in the promoter region of TK1 and is predicted to create a xenobiotic-stress activated TGA1a transcription binding site.

The third SNP (rs501415) resides within the first intron of WDR7 and is predicted to disrupt AML1 (RUNX) transcription factor bind site. Since the entire RUNX family share the same binding site (TGt/cGGT) [[19]](#pone.0111694-Lund1), we postulate that this SNP may also affect the binding of RUNX3. RUNX3 had been hailed as a tumour suppressor gene and reduced expression of RUNX3 has been previously associated with poorer survival in colorectal cancer patients [[38]](#pone.0111694-Soong1). Nonetheless, the role of WDR7 in 5-FU resistance remained unclear. It was reported to be associated with 5-FU by PharmGKB [[39]](#pone.0111694-Potti1) but the publication [[39]](#pone.0111694-Potti1) was recently retracted [[40]](#pone.0111694-Potti2).

### The logistic regression-based multivariate model identified an additional 5 pfSNPs which are not non-responder-specific that may distinguish responders from non-responders

In addition to the non-responder specific pfSNPs that are associated with patients who do not respond to 5-FU, we proceeded to identify additional pfSNPs that may be associated with 5-FU drug response by training a logistic regression-based multivariate model with the stepAIC method using data from the other pfSNPs. The multivariate model identified 5 additional pfSNPs, namely, rs2289310 (DLG5, E/23/G4442T, P1481Q), rs1047840 (EXO1, E/12/G1765A, E589K), rs17431184 (PTEN, I/7/T-400C), rs2236722 (CYP19A1, E/2/A115G, W39R) and rs17160359 (ABCB1, 5UR//G-4254T) ([Table 5](#pone-0111694-t005)) that may distinguish responders from non-responders. The AUC (Area Under Curve) for ROC (Receiver Operating Characteristic) curve of these 5 SNPs is 0.875 ([Figure 2](#pone-0111694-g002)). A predicted value of 0.794 was identified as the optimal cut-off point to predict drug response, with sensitivity of 62.5% and specificity of 100%. With this threshold, the logistic-based multivariate model can correctly identify 39.1% (25/64) of non-responders. Together with the 37.5% of non-responders predicted by the non-responder-specific SNPs, a total of 76.6% (49/64) of non-responders can be correctly identified by both models.

### Table 5. The list of common SNPs included into the multi-variate model and their predicted molecular functions.

| S.N | rsNo | Gene Name | mRNALocation | AA Change | Allele | Allele Count (Frequency) |  | Potential Molecular Function | Discussion |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
|   |   |   |   |   |   | NR (128) | Rsp (50) |   |   |
| 1 | rs2289310 | DLG5 | E/23/G4442T | P1481Q | T | 22 (17%) | 17 (34%) | Causes a non-conservedAA change in PDZ-likedomain. The PDZ-likedomain is important incell-junction according toSuperFamily database. | DLG5 is involved in maintaining the epithelial integrity. The T allele causes a non-conserved AA change in the PDZ-like domain which is important in cell-junction. The T allele has been associated with increased risk for Inflammatory Bowel Disease (IBD) and Crohn Disease (CD). The authors suggest it may have a role in altering tight-junction mediated permeability. It has been shown that enhanced intestinal permeability would increase 5-FU absorption. Therefore, patients with T allele may have higher 5-FU bioavailability and more likely to respond. |
| 2 | rs1047840 | EXO1 | E/12/G1765AE/13/G1765A | E589K E589K | A | 20 (16%) | 15 (30%) | Causes a non-conservedAA change in theExonuclease domain. | EXO1 is a gene controlling DNA repair (OMIM 606063). The A allele causes a non-conserved AA change in the exo-nuclease domain. The A allele has been associated with higher colorectal cancer risk in a UK population. We hypothesize that the increased risk is due to the reduced DNA repairing function of EXO1 caused by the A allele. Therefore, patients having A allele may be more responsive to DNA damaging agents like 5-FU. |
| 3 | rs17431184 | PTEN | I/7/T-400C | – | C | 15 (12%) | 13 (26%) | Creates intronic splicingregulatory elements andunder recent positiveselection | PTEN is a well-known tumor suppressor and part of the apoptosis signalling pathway. The C allele is under recent positive selection. Therefore, patients having this allele may have stronger PTEN activity and respond better to 5-FU. |
| 4 | rs2236722 | CYP19A1 | E/2/A115GE/3/A115G | W39R W39R | G | 16 (13%) | 1 (2%) | Causes a non-conservedAA change in theCYP_P450 family domain.Predicted to be deleteriousby Polyphen. Also altersexonic splicing element. | A number of SNPs in the CYP19A1 gene have been associated with colorectal cancer risk. In the particular paper, this SNP has not been tested in this study because the study was conducted in an US white population and the SNP is likely to be mono-morphic according to HapMap. The G allele is detrimental to the protein function therefore patients having the G allele should have lower oestrogen level. This may translates to a higher colorectal cancer risk and more progressive disease therefore less responsive to 5-FU. |
| 5 | rs17160359 | ABCB1 | 5UR/G-4254T | – | T | 3 (2.3%) | 5 (10%) | Creates binding sitefor HMGA1. | ABCB1 does not transport 5-FU but a few SNPs in it are implicated in CRC risk. The T allele creates a binding site of HMGA1 which is upregulated in various cancers. |
### Figure 2. The ROC curve for the logistic regression based multivariate model trained to differentiate non-responders who do not have the non-responder.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e040/4221105/ba30ad57095d/pone.0111694.g002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4221105_pone.0111694.g002.jpg)

The AUC of the ROC curve is 0.875. The point of maximum sum of sensitivity and specificity is highlighted by the green circle on the ROC curve. The corresponding sensitivity and specificity is 62.5% and 100% respectively.

It is noteworthy that the SNPs in the multivariate model are primarily localized within tumor suppressor gene (PTEN) or in genes which are mainly associated with colorectal cancer (DLG5, EXO1, CYP19A1 and ABCB1) ([Table 5](#pone-0111694-t005)). Notably, one of the non-responder-specific SNP (rs3218592, REV3L E/26/C8285T, R2762Q) ([Table 4](#pone-0111694-t004)) residing in the gene, Rev3L, has also been implicated to be a tumor suppressor gene [[41]](#pone.0111694-Brondello1).

The PTEN (phosphatase and tensin homolog) gene is a well-known tumor suppressor gene that was recently reported to control DNA repair and sensitivity to genotoxic stress [[42]](#pone.0111694-Bassi1). The higher frequency of the C allele in the responders (26% in the responders vs. 12% in the non-responders) suggests that patients with this allele may exhibit lower tolerance to genotoxic stress caused by DNA damaging agents like 5-FU.

The DLG5 (Disks large homolog 5) gene encodes a member of the membrane-associated guanylate kinase (MAGUK) family of scaffolding proteins which is involved in maintaining the epithelial integrity [[43]](#pone.0111694-Festen1). The T allele of rs2289310 (DLG5, E/23/G4442T, P1481Q) causes a non-conserved amino acid change in the vicinity of one of the PDZ domains in this gene (aa 1391–1472; Prosite score 13.531). This variant was postulated to impair the scaffolding functions of DLG5 [[44]](#pone.0111694-Stoll1) and enhance the tight junction-mediated gut permeability [[45]](#pone.0111694-Newman1). This polymorphism has also been associated with increased risk of Inflammatory Bowel Disease [[45]](#pone.0111694-Newman1) which may lead to increased risk for colorectal cancer [[46]](#pone.0111694-Triantafillidis1). Since enhanced gut permeability was reported to lead to higher 5-FU absorption in rats [[47]](#pone.0111694-Hirata1), we hypothesize that the T allele would lead to better drug absorption and thus better response. Consistent with our hypothesis, more responders have the T allele (MAF of 34% in responder vs 17% in non-responder) in this study.

The EXO1 (exonuclease 1) gene has been implicated to play roles in DNA replication, recombination, repair, telomere integrity [[48]](#pone.0111694-Tran1) as well as damage signalling decisions [[49]](#pone.0111694-Tomimatsu1). The A allele of rs1047840 (EXO1, E/12/G1765A or E/13/G1765A, E589K) causes a non-conserved amino acid change and is predicted to reside within a region that is highly conserved amongst the XP-G/RAD2 DNA Repair Endonuclease Family (HMMPanther PTHR11081). In the Kin-cohort analyses, the A allele has been associated with higher risk of colorectal cancer in a UK population [[50]](#pone.0111694-Webb1). This SNP has also been associated with higher risk for various other cancers in the Chinese population [[51]](#pone.0111694-Luo1)–[[57]](#pone.0111694-Bau1). We hypothesize that the increased cancer risk associated with this SNP could be due to less efficient repair of DNA damage in individuals carrying the A-allele. As the metabolites of 5-FU gets incorporated into DNA damaging the host DNA, the inefficient repair mechanisms of individuals carrying the A-allele of this SNP may result in greater cell death hence enhancing the effectiveness of 5-FU treatment. This is consistent with our observations that greater percentage of responders carries the A-allele of this SNP compared to non-responders (30% versus 16%).

CYP19A1 (cytochrome P450, family 19, subfamily A, polypeptide 1) or Aromatase is a member of the cytochrome P450 superfamily of enzymes and plays an important role in the metabolism of oestrogens. Oestrogen has been associated with lower risk of colorectal cancer [[58]](#pone.0111694-Rennert1)–[[60]](#pone.0111694-Nanda1). Although several SNPs in the CYP19A1 gene had been reported to be associated with risk for colorectal cancer in a Caucasian population [[61]](#pone.0111694-Slattery1), SNP rs2236722 (E/2/A115G or E/3/A115G, W39R) was not examined in that study as it is monomorphic in HapMap CEU population. Nonetheless, this SNP (rs2236722), which occurs at a frequency of 3.3% in HapMap CHB population and 9.5% in our study, causes a non-conserved amino acid change from hydrophobic tryptophan to charged arginine in the CYP_P450 family domain and is predicted by Polyphen [[36]](#pone.0111694-Adzhubei1) to be a deleterious alteration. Hence, it is possible that this deleterious change in CYP19A1 may lead to lower oestrogen levels leading to higher colorectal cancer risk. We thus hypothesize that patients with the minor G-allele may have more progressive disease and hence are less responsive to 5-FU treatment.

The final gene implicated by the multivariate model to be associated with 5-FU response is the ABCB1 (ATP-Binding Cassette, Sub-Family B (MDR/TAP), Member 1) or the MDR1 (multidrug resistance protein 1). Although 5-FU is not a substrate of MDR1 protein, there’s some, albeit controversial evidence that SNPs within the ABCB1 gene, may be associated with CRC risk [[62]](#pone.0111694-He1), [[63]](#pone.0111694-Potocnik1). Nonetheless, the SNP rs17160359 (ABCB1, 5UR/G-4254T) which is implicated in the multivariate model to be associated with drug response in CRC patient, resides in the promoter region and the T allele of the SNP creates a binding site of a transcription factor called HMGA1 which is expressed at very low level in adult human tissues but highly expressed in various tumours [[64]](#pone.0111694-Fusco1).

In summary, three perfect LD, non-responder-specific pfSNPs within the UMPS gene which plays a role in 5-FU metabolism together with 3 other non-responder-specific pfSNPs and 5 other pfSNPs in genes that may play roles in modulating tumor risks may collaborate to influence the patient’s response of CRC drugs. This study thus provides one of the building blocks for subsequent meta-analysis in larger cohort of patients.

## Supporting Information

## Acknowledgments

We thank Mah Way-Champ and Teo Wei Bing for their assistance in the project.

## Data Availability

The authors confirm that all data underlying the findings are fully available without restriction. All relevant data are within the paper and its Supporting Information files.

## Funding Statement

This work was supported by a grant (SERC 112 148 0008) from Singapore BioMedical Research Council - Science and Engineering Research Council (BMRC-SERC) to A/Professors Caroline Lee, Simon Ong, Yik Ying Teo and Samuel Chong. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Data Availability Statement

The authors confirm that all data underlying the findings are fully available without restriction. All relevant data are within the paper and its Supporting Information files.

### Supplementary Materials

### Data Availability Statement

The authors confirm that all data underlying the findings are fully available without restriction. All relevant data are within the paper and its Supporting Information files.

## References

1. Cunningham D, Atkin W, Lenz HJ, Lynch HT, Minsky B, et al. (2010) Colorectal cancer. Lancet 375: 1030–1047.  [DOI](https://doi.org/10.1016/S0140-6736(10)60353-4) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20304247/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Colorectal%20cancer&author=D%20Cunningham&author=W%20Atkin&author=HJ%20Lenz&author=HT%20Lynch&author=B%20Minsky&volume=375&publication_year=2010&pages=1030-1047&pmid=20304247&doi=10.1016/S0140-6736(10)60353-4&)

2. SC R (2011) Singapore Cancer Registry Interim Annual Registry Report. Trends in Cancer Incidence in Singapore 2005–2009.

3. Sobrero AF, Aschele C, Bertino JR (1997) Fluorouracil in colorectal cancer–a tale of two drugs: implications for biochemical modulation. Journal of clinical oncology: official journal of the American Society of Clinical Oncology 15: 368–381.  [DOI](https://doi.org/10.1200/JCO.1997.15.1.368) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8996164/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20of%20clinical%20oncology:%20official%20journal%20of%20the%20American%20Society%20of%20Clinical%20Oncology&title=Fluorouracil%20in%20colorectal%20cancer%E2%80%93a%20tale%20of%20two%20drugs:%20implications%20for%20biochemical%20modulation&author=AF%20Sobrero&author=C%20Aschele&author=JR%20Bertino&volume=15&publication_year=1997&pages=368-381&pmid=8996164&doi=10.1200/JCO.1997.15.1.368&)

4. Graziano F, Ruzzo A, Loupakis F, Santini D, Catalano V, et al. (2008) Liver-only metastatic colorectal cancer patients and thymidylate synthase polymorphisms for predicting response to 5-fluorouracil-based chemotherapy. British journal of cancer 99: 716–721.  [DOI](https://doi.org/10.1038/sj.bjc.6604555) | [PMC free article](/articles/PMC2528158/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18728661/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=British%20journal%20of%20cancer&title=Liver-only%20metastatic%20colorectal%20cancer%20patients%20and%20thymidylate%20synthase%20polymorphisms%20for%20predicting%20response%20to%205-fluorouracil-based%20chemotherapy&author=F%20Graziano&author=A%20Ruzzo&author=F%20Loupakis&author=D%20Santini&author=V%20Catalano&volume=99&publication_year=2008&pages=716-721&pmid=18728661&doi=10.1038/sj.bjc.6604555&)

5. Salgado J, Zabalegui N, Gil C, Monreal I, Rodriguez J, et al. (2007) Polymorphisms in the thymidylate synthase and dihydropyrimidine dehydrogenase genes predict response and toxicity to capecitabine-raltitrexed in colorectal cancer. Oncology reports 17: 325–328.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/17203168/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncology%20reports&title=Polymorphisms%20in%20the%20thymidylate%20synthase%20and%20dihydropyrimidine%20dehydrogenase%20genes%20predict%20response%20and%20toxicity%20to%20capecitabine-raltitrexed%20in%20colorectal%20cancer&author=J%20Salgado&author=N%20Zabalegui&author=C%20Gil&author=I%20Monreal&author=J%20Rodriguez&volume=17&publication_year=2007&pages=325-328&pmid=17203168&)

6. Etienne MC, Formento JL, Chazal M, Francoual M, Magne N, et al. (2004) Methylenetetrahydrofolate reductase gene polymorphisms and response to fluorouracil-based treatment in advanced colorectal cancer patients. Pharmacogenetics 14: 785–792.  [DOI](https://doi.org/10.1097/00008571-200412000-00001) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15608557/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=Methylenetetrahydrofolate%20reductase%20gene%20polymorphisms%20and%20response%20to%20fluorouracil-based%20treatment%20in%20advanced%20colorectal%20cancer%20patients&author=MC%20Etienne&author=JL%20Formento&author=M%20Chazal&author=M%20Francoual&author=N%20Magne&volume=14&publication_year=2004&pages=785-792&pmid=15608557&doi=10.1097/00008571-200412000-00001&)

7. Jakobsen A, Nielsen JN, Gyldenkerne N, Lindeberg J (2005) Thymidylate synthase and methylenetetrahydrofolate reductase gene polymorphism in normal tissue as predictors of fluorouracil sensitivity. Journal of clinical oncology: official journal of the American Society of Clinical Oncology 23: 1365–1369.  [DOI](https://doi.org/10.1200/JCO.2005.06.219) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15735113/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20of%20clinical%20oncology:%20official%20journal%20of%20the%20American%20Society%20of%20Clinical%20Oncology&title=Thymidylate%20synthase%20and%20methylenetetrahydrofolate%20reductase%20gene%20polymorphism%20in%20normal%20tissue%20as%20predictors%20of%20fluorouracil%20sensitivity&author=A%20Jakobsen&author=JN%20Nielsen&author=N%20Gyldenkerne&author=J%20Lindeberg&volume=23&publication_year=2005&pages=1365-1369&pmid=15735113&doi=10.1200/JCO.2005.06.219&)

8. Farina-Sarasqueta A, van Lijnschoten G, Rutten HJ, van den Brule AJ (2010) Value of gene polymorphisms as markers of 5-FU therapy response in stage III colon carcinoma: a pilot study. Cancer Chemother Pharmacol 66: 1167–1171.  [DOI](https://doi.org/10.1007/s00280-010-1403-0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20665215/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Chemother%20Pharmacol&title=Value%20of%20gene%20polymorphisms%20as%20markers%20of%205-FU%20therapy%20response%20in%20stage%20III%20colon%20carcinoma:%20a%20pilot%20study&author=A%20Farina-Sarasqueta&author=G%20van%20Lijnschoten&author=HJ%20Rutten&author=AJ%20van%20den%20Brule&volume=66&publication_year=2010&pages=1167-1171&pmid=20665215&doi=10.1007/s00280-010-1403-0&)

9. Tsuji T, Hidaka S, Sawai T, Nakagoe T, Yano H, et al. (2003) Polymorphism in the thymidylate synthase promoter enhancer region is not an efficacious marker for tumor sensitivity to 5-fluorouracil-based oral adjuvant chemotherapy in colorectal cancer. Clinical cancer research: an official journal of the American Association for Cancer Research 9: 3700–3704.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/14506161/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clinical%20cancer%20research:%20an%20official%20journal%20of%20the%20American%20Association%20for%20Cancer%20Research&title=Polymorphism%20in%20the%20thymidylate%20synthase%20promoter%20enhancer%20region%20is%20not%20an%20efficacious%20marker%20for%20tumor%20sensitivity%20to%205-fluorouracil-based%20oral%20adjuvant%20chemotherapy%20in%20colorectal%20cancer&author=T%20Tsuji&author=S%20Hidaka&author=T%20Sawai&author=T%20Nakagoe&author=H%20Yano&volume=9&publication_year=2003&pages=3700-3704&pmid=14506161&)

10. Vignoli M, Nobili S, Napoli C, Putignano AL, Morganti M, et al. (2011) Thymidylate synthase expression and genotype have no major impact on the clinical outcome of colorectal cancer patients treated with 5-fluorouracil. Pharmacological research: the official journal of the Italian Pharmacological Society 64: 242–248.  [DOI](https://doi.org/10.1016/j.phrs.2011.04.006) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21536130/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacological%20research:%20the%20official%20journal%20of%20the%20Italian%20Pharmacological%20Society&title=Thymidylate%20synthase%20expression%20and%20genotype%20have%20no%20major%20impact%20on%20the%20clinical%20outcome%20of%20colorectal%20cancer%20patients%20treated%20with%205-fluorouracil&author=M%20Vignoli&author=S%20Nobili&author=C%20Napoli&author=AL%20Putignano&author=M%20Morganti&volume=64&publication_year=2011&pages=242-248&pmid=21536130&doi=10.1016/j.phrs.2011.04.006&)

11. Marcuello E, Altes A, Menoyo A, Rio ED, Baiget M (2006) Methylenetetrahydrofolate reductase gene polymorphisms: genomic predictors of clinical response to fluoropyrimidine-based chemotherapy? Cancer chemotherapy and pharmacology 57: 835–840.  [DOI](https://doi.org/10.1007/s00280-005-0089-1) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16187112/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20chemotherapy%20and%20pharmacology&title=Methylenetetrahydrofolate%20reductase%20gene%20polymorphisms:%20genomic%20predictors%20of%20clinical%20response%20to%20fluoropyrimidine-based%20chemotherapy?&author=E%20Marcuello&author=A%20Altes&author=A%20Menoyo&author=ED%20Rio&author=M%20Baiget&volume=57&publication_year=2006&pages=835-840&pmid=16187112&doi=10.1007/s00280-005-0089-1&)

12. Ruzzo A, Graziano F, Loupakis F, Rulli E, Canestrari E, et al. (2007) Pharmacogenetic profiling in patients with advanced colorectal cancer treated with first-line FOLFOX-4 chemotherapy. Journal of clinical oncology: official journal of the American Society of Clinical Oncology 25: 1247–1254.  [DOI](https://doi.org/10.1200/JCO.2006.08.1844) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17401013/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20of%20clinical%20oncology:%20official%20journal%20of%20the%20American%20Society%20of%20Clinical%20Oncology&title=Pharmacogenetic%20profiling%20in%20patients%20with%20advanced%20colorectal%20cancer%20treated%20with%20first-line%20FOLFOX-4%20chemotherapy&author=A%20Ruzzo&author=F%20Graziano&author=F%20Loupakis&author=E%20Rulli&author=E%20Canestrari&volume=25&publication_year=2007&pages=1247-1254&pmid=17401013&doi=10.1200/JCO.2006.08.1844&)

13. Cohen V, Panet-Raymond V, Sabbaghian N, Morin I, Batist G, et al. (2003) Methylenetetrahydrofolate reductase polymorphism in advanced colorectal cancer: a novel genomic predictor of clinical response to fluoropyrimidine-based chemotherapy. Clinical cancer research: an official journal of the American Association for Cancer Research 9: 1611–1615.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/12738713/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clinical%20cancer%20research:%20an%20official%20journal%20of%20the%20American%20Association%20for%20Cancer%20Research&title=Methylenetetrahydrofolate%20reductase%20polymorphism%20in%20advanced%20colorectal%20cancer:%20a%20novel%20genomic%20predictor%20of%20clinical%20response%20to%20fluoropyrimidine-based%20chemotherapy&author=V%20Cohen&author=V%20Panet-Raymond&author=N%20Sabbaghian&author=I%20Morin&author=G%20Batist&volume=9&publication_year=2003&pages=1611-1615&pmid=12738713&)

14. Ichikawa W, Takahashi T, Suto K, Sasaki Y, Hirayama R (2006) Orotate phosphoribosyltransferase gene polymorphism predicts toxicity in patients treated with bolus 5-fluorouracil regimen. Clinical cancer research: an official journal of the American Association for Cancer Research 12: 3928–3934.  [DOI](https://doi.org/10.1158/1078-0432.CCR-05-2665) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16818689/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clinical%20cancer%20research:%20an%20official%20journal%20of%20the%20American%20Association%20for%20Cancer%20Research&title=Orotate%20phosphoribosyltransferase%20gene%20polymorphism%20predicts%20toxicity%20in%20patients%20treated%20with%20bolus%205-fluorouracil%20regimen&author=W%20Ichikawa&author=T%20Takahashi&author=K%20Suto&author=Y%20Sasaki&author=R%20Hirayama&volume=12&publication_year=2006&pages=3928-3934&pmid=16818689&doi=10.1158/1078-0432.CCR-05-2665&)

15. Zhang H, Li YM, Jin X (2007) DPYD*5 gene mutation contributes to the reduced DPYD enzyme activity and chemotherapeutic toxicity of 5-FU: results from genotyping study on 75 gastric carcinoma and colon carcinoma patients. Med Oncol 24: 251–258.  [DOI](https://doi.org/10.1007/BF02698048) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17848752/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Med%20Oncol&title=DPYD*5%20gene%20mutation%20contributes%20to%20the%20reduced%20DPYD%20enzyme%20activity%20and%20chemotherapeutic%20toxicity%20of%205-FU:%20results%20from%20genotyping%20study%20on%2075%20gastric%20carcinoma%20and%20colon%20carcinoma%20patients&author=H%20Zhang&author=YM%20Li&author=X%20Jin&volume=24&publication_year=2007&pages=251-258&pmid=17848752&doi=10.1007/BF02698048&)

16. Gross E, Busse B, Riemenschneider M, Neubauer S, Seck K, et al. (2008) Strong association of a common dihydropyrimidine dehydrogenase gene polymorphism with fluoropyrimidine-related toxicity in cancer patients. PloS one 3: e4003.  [DOI](https://doi.org/10.1371/journal.pone.0004003) | [PMC free article](/articles/PMC2602733/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19104657/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PloS%20one&title=Strong%20association%20of%20a%20common%20dihydropyrimidine%20dehydrogenase%20gene%20polymorphism%20with%20fluoropyrimidine-related%20toxicity%20in%20cancer%20patients&author=E%20Gross&author=B%20Busse&author=M%20Riemenschneider&author=S%20Neubauer&author=K%20Seck&volume=3&publication_year=2008&pages=e4003&pmid=19104657&doi=10.1371/journal.pone.0004003&)

17. Schwab M, Zanger UM, Marx C, Schaeffeler E, Klein K, et al. (2008) Role of genetic and nongenetic factors for fluorouracil treatment-related severe toxicity: a prospective clinical trial by the German 5-FU Toxicity Study Group. Journal of clinical oncology: official journal of the American Society of Clinical Oncology 26: 2131–2138.  [DOI](https://doi.org/10.1200/JCO.2006.10.4182) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18299612/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20of%20clinical%20oncology:%20official%20journal%20of%20the%20American%20Society%20of%20Clinical%20Oncology&title=Role%20of%20genetic%20and%20nongenetic%20factors%20for%20fluorouracil%20treatment-related%20severe%20toxicity:%20a%20prospective%20clinical%20trial%20by%20the%20German%205-FU%20Toxicity%20Study%20Group&author=M%20Schwab&author=UM%20Zanger&author=C%20Marx&author=E%20Schaeffeler&author=K%20Klein&volume=26&publication_year=2008&pages=2131-2138&pmid=18299612&doi=10.1200/JCO.2006.10.4182&)

18. Koopman M, Venderbosch S, Nagtegaal ID, van Krieken JH, Punt CJ (2009) A review on the use of molecular markers of cytotoxic therapy for colorectal cancer, what have we learned? Eur J Cancer 45: 1935–1949.  [DOI](https://doi.org/10.1016/j.ejca.2009.04.023) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19473832/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Cancer&title=A%20review%20on%20the%20use%20of%20molecular%20markers%20of%20cytotoxic%20therapy%20for%20colorectal%20cancer,%20what%20have%20we%20learned?&author=M%20Koopman&author=S%20Venderbosch&author=ID%20Nagtegaal&author=JH%20van%20Krieken&author=CJ%20Punt&volume=45&publication_year=2009&pages=1935-1949&pmid=19473832&doi=10.1016/j.ejca.2009.04.023&)

19. Lund AH, van Lohuizen M (2002) RUNX: a trilogy of cancer genes. Cancer Cell 1: 213–215.  [DOI](https://doi.org/10.1016/s1535-6108(02)00049-1) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12086855/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Cell&title=RUNX:%20a%20trilogy%20of%20cancer%20genes&author=AH%20Lund&author=M%20van%20Lohuizen&volume=1&publication_year=2002&pages=213-215&pmid=12086855&doi=10.1016/s1535-6108(02)00049-1&)

20. Deng T, Zhang Y (2009) 5-Aza-2′-deoxycytidine reactivates expression of RUNX3 by deletion of DNA methyltransferases leading to caspase independent apoptosis in colorectal cancer Lovo cells. Biomedicine & pharmacotherapy = Biomedecine & pharmacotherapie 63: 492–500.  [DOI](https://doi.org/10.1016/j.biopha.2008.08.013) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18848767/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biomedicine%20&%20pharmacotherapy%E2%80%8A=%E2%80%8ABiomedecine%20&%20pharmacotherapie&title=5-Aza-2%E2%80%B2-deoxycytidine%20reactivates%20expression%20of%20RUNX3%20by%20deletion%20of%20DNA%20methyltransferases%20leading%20to%20caspase%20independent%20apoptosis%20in%20colorectal%20cancer%20Lovo%20cells&author=T%20Deng&author=Y%20Zhang&volume=63&publication_year=2009&pages=492-500&pmid=18848767&doi=10.1016/j.biopha.2008.08.013&)

21. Wang J, Ronaghi M, Chong SS, Lee CG (2011) pfSNP: An integrated potentially functional SNP resource that facilitates hypotheses generation through knowledge syntheses. Human Mutation 32: 19–24.  [DOI](https://doi.org/10.1002/humu.21331) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20672376/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Human%20Mutation&title=pfSNP:%20An%20integrated%20potentially%20functional%20SNP%20resource%20that%20facilitates%20hypotheses%20generation%20through%20knowledge%20syntheses&author=J%20Wang&author=M%20Ronaghi&author=SS%20Chong&author=CG%20Lee&volume=32&publication_year=2011&pages=19-24&pmid=20672376&doi=10.1002/humu.21331&)

22. Sohaib A (2012) RECIST rules. Cancer imaging: the official publication of the International Cancer Imaging Society 12: 345–346.  [DOI](https://doi.org/10.1102/1470-7330.2012.9011) | [PMC free article](/articles/PMC3460554/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23023096/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20imaging:%20the%20official%20publication%20of%20the%20International%20Cancer%20Imaging%20Society&title=RECIST%20rules&author=A%20Sohaib&volume=12&publication_year=2012&pages=345-346&pmid=23023096&doi=10.1102/1470-7330.2012.9011&)

23. Van Cutsem E, Hoff PM, Harper P, Bukowski RM, Cunningham D, et al. (2004) Oral capecitabine vs intravenous 5-fluorouracil and leucovorin: integrated efficacy data and novel analyses from two large, randomised, phase III trials. British journal of cancer 90: 1190–1197.  [DOI](https://doi.org/10.1038/sj.bjc.6601676) | [PMC free article](/articles/PMC2409640/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15026800/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=British%20journal%20of%20cancer&title=Oral%20capecitabine%20vs%20intravenous%205-fluorouracil%20and%20leucovorin:%20integrated%20efficacy%20data%20and%20novel%20analyses%20from%20two%20large,%20randomised,%20phase%20III%20trials&author=E%20Van%20Cutsem&author=PM%20Hoff&author=P%20Harper&author=RM%20Bukowski&author=D%20Cunningham&volume=90&publication_year=2004&pages=1190-1197&pmid=15026800&doi=10.1038/sj.bjc.6601676&)

24. de Gramont A, Figer A, Seymour M, Homerin M, Hmissi A, et al. (2000) Leucovorin and fluorouracil with or without oxaliplatin as first-line treatment in advanced colorectal cancer. Journal of clinical oncology: official journal of the American Society of Clinical Oncology 18: 2938–2947.  [DOI](https://doi.org/10.1200/JCO.2000.18.16.2938) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10944126/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20of%20clinical%20oncology:%20official%20journal%20of%20the%20American%20Society%20of%20Clinical%20Oncology&title=Leucovorin%20and%20fluorouracil%20with%20or%20without%20oxaliplatin%20as%20first-line%20treatment%20in%20advanced%20colorectal%20cancer&author=A%20de%20Gramont&author=A%20Figer&author=M%20Seymour&author=M%20Homerin&author=A%20Hmissi&volume=18&publication_year=2000&pages=2938-2947&pmid=10944126&doi=10.1200/JCO.2000.18.16.2938&)

25. Kawakami K, Watanabe G (2003) Identification and functional analysis of single nucleotide polymorphism in the tandem repeat sequence of thymidylate synthase gene. Cancer Res 63: 6004–6007.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/14522928/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=Identification%20and%20functional%20analysis%20of%20single%20nucleotide%20polymorphism%20in%20the%20tandem%20repeat%20sequence%20of%20thymidylate%20synthase%20gene&author=K%20Kawakami&author=G%20Watanabe&volume=63&publication_year=2003&pages=6004-6007&pmid=14522928&)

26. Kawakami K, Watanabe G (2003) Identification and functional analysis of single nucleotide polymorphism in the tandem repeat sequence of thymidylate synthase gene. Cancer research 63: 6004–6007.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/14522928/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20research&title=Identification%20and%20functional%20analysis%20of%20single%20nucleotide%20polymorphism%20in%20the%20tandem%20repeat%20sequence%20of%20thymidylate%20synthase%20gene&author=K%20Kawakami&author=G%20Watanabe&volume=63&publication_year=2003&pages=6004-6007&pmid=14522928&)

27. Purcell S, Neale B, Todd-Brown K, Thomas L, Ferreira MA, et al. (2007) PLINK: a tool set for whole-genome association and population-based linkage analyses. American journal of human genetics 81: 559–575.  [DOI](https://doi.org/10.1086/519795) | [PMC free article](/articles/PMC1950838/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17701901/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=American%20journal%20of%20human%20genetics&title=PLINK:%20a%20tool%20set%20for%20whole-genome%20association%20and%20population-based%20linkage%20analyses&author=S%20Purcell&author=B%20Neale&author=K%20Todd-Brown&author=L%20Thomas&author=MA%20Ferreira&volume=81&publication_year=2007&pages=559-575&pmid=17701901&doi=10.1086/519795&)

28. Li B, Leal SM (2008) Methods for detecting associations with rare variants for common diseases: application to analysis of sequence data. American journal of human genetics 83: 311–321.  [DOI](https://doi.org/10.1016/j.ajhg.2008.06.024) | [PMC free article](/articles/PMC2842185/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18691683/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=American%20journal%20of%20human%20genetics&title=Methods%20for%20detecting%20associations%20with%20rare%20variants%20for%20common%20diseases:%20application%20to%20analysis%20of%20sequence%20data&author=B%20Li&author=SM%20Leal&volume=83&publication_year=2008&pages=311-321&pmid=18691683&doi=10.1016/j.ajhg.2008.06.024&)

29. Zou KH, O'Malley AJ, Mauri L (2007) Receiver-operating characteristic analysis for evaluating diagnostic tests and predictive models. Circulation 115: 654–657.  [DOI](https://doi.org/10.1161/CIRCULATIONAHA.105.594929) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17283280/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circulation&title=Receiver-operating%20characteristic%20analysis%20for%20evaluating%20diagnostic%20tests%20and%20predictive%20models&author=KH%20Zou&author=AJ%20O'Malley&author=L%20Mauri&volume=115&publication_year=2007&pages=654-657&pmid=17283280&doi=10.1161/CIRCULATIONAHA.105.594929&)

30. Patti G, Nusca A, Mangiacapra F, Gatto L, D'Ambrosio A, et al. (2008) Point-of-care measurement of clopidogrel responsiveness predicts clinical outcome in patients undergoing percutaneous coronary intervention results of the ARMYDA-PRO (Antiplatelet therapy for Reduction of MYocardial Damage during Angioplasty-Platelet Reactivity Predicts Outcome) study. Journal of the American College of Cardiology 52: 1128–1133.  [DOI](https://doi.org/10.1016/j.jacc.2008.06.038) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18804738/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20of%20the%20American%20College%20of%20Cardiology&title=Point-of-care%20measurement%20of%20clopidogrel%20responsiveness%20predicts%20clinical%20outcome%20in%20patients%20undergoing%20percutaneous%20coronary%20intervention%20results%20of%20the%20ARMYDA-PRO%20(Antiplatelet%20therapy%20for%20Reduction%20of%20MYocardial%20Damage%20during%20Angioplasty-Platelet%20Reactivity%20Predicts%20Outcome)%20study&author=G%20Patti&author=A%20Nusca&author=F%20Mangiacapra&author=L%20Gatto&author=A%20D'Ambrosio&volume=52&publication_year=2008&pages=1128-1133&pmid=18804738&doi=10.1016/j.jacc.2008.06.038&)

31. Nagy E, Maquat LE (1998) A rule for termination-codon position within intron-containing genes: when nonsense affects RNA abundance. Trends in biochemical sciences 23: 198–199.  [DOI](https://doi.org/10.1016/s0968-0004(98)01208-0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9644970/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Trends%20in%20biochemical%20sciences&title=A%20rule%20for%20termination-codon%20position%20within%20intron-containing%20genes:%20when%20nonsense%20affects%20RNA%20abundance&author=E%20Nagy&author=LE%20Maquat&volume=23&publication_year=1998&pages=198-199&pmid=9644970&doi=10.1016/s0968-0004(98)01208-0&)

32. Kulshreshtha R, Ferracin M, Wojcik SE, Garzon R, Alder H, et al. (2007) A microRNA signature of hypoxia. Molecular and cellular biology 27: 1859–1867.  [DOI](https://doi.org/10.1128/MCB.01395-06) | [PMC free article](/articles/PMC1820461/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17194750/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Molecular%20and%20cellular%20biology&title=A%20microRNA%20signature%20of%20hypoxia&author=R%20Kulshreshtha&author=M%20Ferracin&author=SE%20Wojcik&author=R%20Garzon&author=H%20Alder&volume=27&publication_year=2007&pages=1859-1867&pmid=17194750&doi=10.1128/MCB.01395-06&)

33. Jahid S, Sun J, Edwards RA, Dizon D, Panarelli NC, et al. (2012) miR-23a promotes the transition from indolent to invasive colorectal cancer. Cancer discovery 2: 540–553.  [DOI](https://doi.org/10.1158/2159-8290.CD-11-0267) | [PMC free article](/articles/PMC3651884/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22628407/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20discovery&title=miR-23a%20promotes%20the%20transition%20from%20indolent%20to%20invasive%20colorectal%20cancer&author=S%20Jahid&author=J%20Sun&author=RA%20Edwards&author=D%20Dizon&author=NC%20Panarelli&volume=2&publication_year=2012&pages=540-553&pmid=22628407&doi=10.1158/2159-8290.CD-11-0267&)

34. Pan Q, Fang Y, Xu Y, Zhang K, Hu X (2005) Down-regulation of DNA polymerases kappa, eta, iota, and zeta in human lung, stomach, and colorectal cancers. Cancer Letters 217: 139–147.  [DOI](https://doi.org/10.1016/j.canlet.2004.07.021) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15617831/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Letters&title=Down-regulation%20of%20DNA%20polymerases%20kappa,%20eta,%20iota,%20and%20zeta%20in%20human%20lung,%20stomach,%20and%20colorectal%20cancers&author=Q%20Pan&author=Y%20Fang&author=Y%20Xu&author=K%20Zhang&author=X%20Hu&volume=217&publication_year=2005&pages=139-147&pmid=15617831&doi=10.1016/j.canlet.2004.07.021&)

35. Zhang S, Chen H, Zhao X, Cao J, Tong J, et al. (2013) REV3L 3′UTR 460 T>C polymorphism in microRNA target sites contributes to lung cancer susceptibility. Oncogene 32: 242–250.  [DOI](https://doi.org/10.1038/onc.2012.32) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22349819/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncogene&title=REV3L%203%E2%80%B2UTR%20460%20T>C%20polymorphism%20in%20microRNA%20target%20sites%20contributes%20to%20lung%20cancer%20susceptibility&author=S%20Zhang&author=H%20Chen&author=X%20Zhao&author=J%20Cao&author=J%20Tong&volume=32&publication_year=2013&pages=242-250&pmid=22349819&doi=10.1038/onc.2012.32&)

36. Adzhubei IA, Schmidt S, Peshkin L, Ramensky VE, Gerasimova A, et al. (2010) A method and server for predicting damaging missense mutations. Nature Methods 7: 248–249.  [DOI](https://doi.org/10.1038/nmeth0410-248) | [PMC free article](/articles/PMC2855889/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20354512/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature%20Methods&title=A%20method%20and%20server%20for%20predicting%20damaging%20missense%20mutations&author=IA%20Adzhubei&author=S%20Schmidt&author=L%20Peshkin&author=VE%20Ramensky&author=A%20Gerasimova&volume=7&publication_year=2010&pages=248-249&pmid=20354512&doi=10.1038/nmeth0410-248&)

37. Sim NL, Kumar P, Hu J, Henikoff S, Schneider G, et al. (2012) SIFT web server: predicting effects of amino acid substitutions on proteins. Nucleic Acids Research 40: W452–457.  [DOI](https://doi.org/10.1093/nar/gks539) | [PMC free article](/articles/PMC3394338/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22689647/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Research&title=SIFT%20web%20server:%20predicting%20effects%20of%20amino%20acid%20substitutions%20on%20proteins&author=NL%20Sim&author=P%20Kumar&author=J%20Hu&author=S%20Henikoff&author=G%20Schneider&volume=40&publication_year=2012&pages=W452-457&pmid=22689647&doi=10.1093/nar/gks539&)

38. Soong R, Shah N, Peh BK, Chong PY, Ng SS, et al. (2009) The expression of RUNX3 in colorectal cancer is associated with disease stage and patient outcome. British Journal of Cancer 100: 676–679.  [DOI](https://doi.org/10.1038/sj.bjc.6604899) | [PMC free article](/articles/PMC2653772/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19223906/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=British%20Journal%20of%20Cancer&title=The%20expression%20of%20RUNX3%20in%20colorectal%20cancer%20is%20associated%20with%20disease%20stage%20and%20patient%20outcome&author=R%20Soong&author=N%20Shah&author=BK%20Peh&author=PY%20Chong&author=SS%20Ng&volume=100&publication_year=2009&pages=676-679&pmid=19223906&doi=10.1038/sj.bjc.6604899&)

39. Potti A, Dressman HK, Bild A, Riedel RF, Chan G, et al. (2006) Genomic signatures to guide the use of chemotherapeutics. Nature Medicine 12: 1294–1300.  [DOI](https://doi.org/10.1038/nm1491) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17057710/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature%20Medicine&title=Genomic%20signatures%20to%20guide%20the%20use%20of%20chemotherapeutics&author=A%20Potti&author=HK%20Dressman&author=A%20Bild&author=RF%20Riedel&author=G%20Chan&volume=12&publication_year=2006&pages=1294-1300&pmid=17057710&doi=10.1038/nm1491&)

40. Potti A, Dressman HK, Bild A, Riedel RF, Chan G, et al. (2011) Retraction: Genomic signatures to guide the use of chemotherapeutics. Nature Medicine 17: 135.  [DOI](https://doi.org/10.1038/nm0111-135) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21217686/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature%20Medicine&title=Retraction:%20Genomic%20signatures%20to%20guide%20the%20use%20of%20chemotherapeutics&author=A%20Potti&author=HK%20Dressman&author=A%20Bild&author=RF%20Riedel&author=G%20Chan&volume=17&publication_year=2011&pages=135&pmid=21217686&doi=10.1038/nm0111-135&)

41. Brondello JM, Pillaire MJ, Rodriguez C, Gourraud PA, Selves J, et al. (2008) Novel evidences for a tumor suppressor role of Rev3, the catalytic subunit of Pol zeta. Oncogene 27: 6093–6101.  [DOI](https://doi.org/10.1038/onc.2008.212) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18622427/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncogene&title=Novel%20evidences%20for%20a%20tumor%20suppressor%20role%20of%20Rev3,%20the%20catalytic%20subunit%20of%20Pol%20zeta&author=JM%20Brondello&author=MJ%20Pillaire&author=C%20Rodriguez&author=PA%20Gourraud&author=J%20Selves&volume=27&publication_year=2008&pages=6093-6101&pmid=18622427&doi=10.1038/onc.2008.212&)

42. Bassi C, Ho J, Srikumar T, Dowling RJ, Gorrini C, et al. (2013) Nuclear PTEN controls DNA repair and sensitivity to genotoxic stress. Science 341: 395–399.  [DOI](https://doi.org/10.1126/science.1236188) | [PMC free article](/articles/PMC5087104/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23888040/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=Nuclear%20PTEN%20controls%20DNA%20repair%20and%20sensitivity%20to%20genotoxic%20stress&author=C%20Bassi&author=J%20Ho&author=T%20Srikumar&author=RJ%20Dowling&author=C%20Gorrini&volume=341&publication_year=2013&pages=395-399&pmid=23888040&doi=10.1126/science.1236188&)

43. Festen EA, Zhernakova A, Wijmenga C, Weersma RK (2008) Association of DLG5 variants with gluten-sensitive enteropathy. Gut 57: 1027–1028.  [DOI](https://doi.org/10.1136/gut.2007.144857) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18559397/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gut&title=Association%20of%20DLG5%20variants%20with%20gluten-sensitive%20enteropathy&author=EA%20Festen&author=A%20Zhernakova&author=C%20Wijmenga&author=RK%20Weersma&volume=57&publication_year=2008&pages=1027-1028&pmid=18559397&doi=10.1136/gut.2007.144857&)

44. Stoll M, Corneliussen B, Costello CM, Waetzig GH, Mellgard B, et al. (2004) Genetic variation in DLG5 is associated with inflammatory bowel disease. Nature Genetics 36: 476–480.  [DOI](https://doi.org/10.1038/ng1345) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15107852/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature%20Genetics&title=Genetic%20variation%20in%20DLG5%20is%20associated%20with%20inflammatory%20bowel%20disease&author=M%20Stoll&author=B%20Corneliussen&author=CM%20Costello&author=GH%20Waetzig&author=B%20Mellgard&volume=36&publication_year=2004&pages=476-480&pmid=15107852&doi=10.1038/ng1345&)

45. Newman WG, Gu X, Wintle RF, Liu X, van Oene M, et al. (2006) DLG5 variants contribute to Crohn disease risk in a Canadian population. Human Mutation 27: 353–358.  [DOI](https://doi.org/10.1002/humu.20301) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16450402/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Human%20Mutation&title=DLG5%20variants%20contribute%20to%20Crohn%20disease%20risk%20in%20a%20Canadian%20population&author=WG%20Newman&author=X%20Gu&author=RF%20Wintle&author=X%20Liu&author=M%20van%20Oene&volume=27&publication_year=2006&pages=353-358&pmid=16450402&doi=10.1002/humu.20301&)

46. Triantafillidis JK, Nasioulas G, Kosmidis PA (2009) Colorectal cancer and inflammatory bowel disease: epidemiology, risk factors, mechanisms of carcinogenesis and prevention strategies. Anticancer Research 29: 2727–2737.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/19596953/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Anticancer%20Research&title=Colorectal%20cancer%20and%20inflammatory%20bowel%20disease:%20epidemiology,%20risk%20factors,%20mechanisms%20of%20carcinogenesis%20and%20prevention%20strategies&author=JK%20Triantafillidis&author=G%20Nasioulas&author=PA%20Kosmidis&volume=29&publication_year=2009&pages=2727-2737&pmid=19596953&)

47. Hirata K, Horie T (1999) Changes in intestinal absorption of 5-fluorouracil-treated rats. Pharmacology & Toxicology 85: 33–36.  [DOI](https://doi.org/10.1111/j.1600-0773.1999.tb01060.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10426161/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacology%20&%20Toxicology&title=Changes%20in%20intestinal%20absorption%20of%205-fluorouracil-treated%20rats&author=K%20Hirata&author=T%20Horie&volume=85&publication_year=1999&pages=33-36&pmid=10426161&doi=10.1111/j.1600-0773.1999.tb01060.x&)

48. Tran PT, Erdeniz N, Symington LS, Liskay RM (2004) EXO1-A multi-tasking eukaryotic nuclease. DNA Repair 3: 1549–1559.  [DOI](https://doi.org/10.1016/j.dnarep.2004.05.015) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15474417/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=DNA%20Repair&title=EXO1-A%20multi-tasking%20eukaryotic%20nuclease&author=PT%20Tran&author=N%20Erdeniz&author=LS%20Symington&author=RM%20Liskay&volume=3&publication_year=2004&pages=1549-1559&pmid=15474417&doi=10.1016/j.dnarep.2004.05.015&)

49. Tomimatsu N, Mukherjee B, Deland K, Kurimasa A, Bolderson E, et al. (2012) Exo1 plays a major role in DNA end resection in humans and influences double-strand break repair and damage signaling decisions. DNA Repair 11: 441–448.  [DOI](https://doi.org/10.1016/j.dnarep.2012.01.006) | [PMC free article](/articles/PMC3319278/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22326273/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=DNA%20Repair&title=Exo1%20plays%20a%20major%20role%20in%20DNA%20end%20resection%20in%20humans%20and%20influences%20double-strand%20break%20repair%20and%20damage%20signaling%20decisions&author=N%20Tomimatsu&author=B%20Mukherjee&author=K%20Deland&author=A%20Kurimasa&author=E%20Bolderson&volume=11&publication_year=2012&pages=441-448&pmid=22326273&doi=10.1016/j.dnarep.2012.01.006&)

50. Webb EL, Rudd MF, Sellick GS, El Galta R, Bethke L, et al. (2006) Search for low penetrance alleles for colorectal cancer through a scan of 1467 non-synonymous SNPs in 2575 cases and 2707 controls with validation by kin-cohort analysis of 14 704 first-degree relatives. Human Molecular Genetics 15: 3263–3271.  [DOI](https://doi.org/10.1093/hmg/ddl401) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17000706/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Human%20Molecular%20Genetics&title=Search%20for%20low%20penetrance%20alleles%20for%20colorectal%20cancer%20through%20a%20scan%20of%201467%20non-synonymous%20SNPs%20in%202575%20cases%20and%202707%20controls%20with%20validation%20by%20kin-cohort%20analysis%20of%2014%20704%20first-degree%20relatives&author=EL%20Webb&author=MF%20Rudd&author=GS%20Sellick&author=R%20El%20Galta&author=L%20Bethke&volume=15&publication_year=2006&pages=3263-3271&pmid=17000706&doi=10.1093/hmg/ddl401&)

51. Luo X, Hong XS, Xiong XD, Zeng LQ, Lim CE (2012) A single nucleotide polymorphism in EXO1 gene is associated with cervical cancer susceptibility in Chinese patients. International journal of gynecological cancer: official journal of the International Gynecological Cancer Society 22: 220–225.  [DOI](https://doi.org/10.1097/IGC.0b013e318234fd8a) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22146767/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=International%20journal%20of%20gynecological%20cancer:%20official%20journal%20of%20the%20International%20Gynecological%20Cancer%20Society&title=A%20single%20nucleotide%20polymorphism%20in%20EXO1%20gene%20is%20associated%20with%20cervical%20cancer%20susceptibility%20in%20Chinese%20patients&author=X%20Luo&author=XS%20Hong&author=XD%20Xiong&author=LQ%20Zeng&author=CE%20Lim&volume=22&publication_year=2012&pages=220-225&pmid=22146767&doi=10.1097/IGC.0b013e318234fd8a&)

52. Jin G, Wang H, Hu Z, Liu H, Sun W, et al. (2008) Potentially functional polymorphisms of EXO1 and risk of lung cancer in a Chinese population: A case-control analysis. Lung cancer 60: 340–346.  [DOI](https://doi.org/10.1016/j.lungcan.2007.11.003) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18079015/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lung%20cancer&title=Potentially%20functional%20polymorphisms%20of%20EXO1%20and%20risk%20of%20lung%20cancer%20in%20a%20Chinese%20population:%20A%20case-control%20analysis&author=G%20Jin&author=H%20Wang&author=Z%20Hu&author=H%20Liu&author=W%20Sun&volume=60&publication_year=2008&pages=340-346&pmid=18079015&doi=10.1016/j.lungcan.2007.11.003&)

53. Chang JS, Yeh RF, Wiencke JK, Wiemels JL, Smirnov I, et al. (2008) Pathway analysis of single-nucleotide polymorphisms potentially associated with glioblastoma multiforme susceptibility using random forests. Cancer epidemiology, biomarkers & prevention: a publication of the American Association for Cancer Research, cosponsored by the American Society of Preventive Oncology 17: 1368–1373.  [DOI](https://doi.org/10.1158/1055-9965.EPI-07-2830) | [PMC free article](/articles/PMC6986563/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18559551/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20epidemiology,%20biomarkers%20&%20prevention:%20a%20publication%20of%20the%20American%20Association%20for%20Cancer%20Research,%20cosponsored%20by%20the%20American%20Society%20of%20Preventive%20Oncology&title=Pathway%20analysis%20of%20single-nucleotide%20polymorphisms%20potentially%20associated%20with%20glioblastoma%20multiforme%20susceptibility%20using%20random%20forests&author=JS%20Chang&author=RF%20Yeh&author=JK%20Wiencke&author=JL%20Wiemels&author=I%20Smirnov&volume=17&publication_year=2008&pages=1368-1373&pmid=18559551&doi=10.1158/1055-9965.EPI-07-2830&)

54. Hsu NY, Wang HC, Wang CH, Chiu CF, Tseng HC, et al. (2009) Lung cancer susceptibility and genetic polymorphisms of Exo1 gene in Taiwan. Anticancer Research 29: 725–730.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/19331228/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Anticancer%20Research&title=Lung%20cancer%20susceptibility%20and%20genetic%20polymorphisms%20of%20Exo1%20gene%20in%20Taiwan&author=NY%20Hsu&author=HC%20Wang&author=CH%20Wang&author=CF%20Chiu&author=HC%20Tseng&volume=29&publication_year=2009&pages=725-730&pmid=19331228&)

55. Tsai MH, Tseng HC, Liu CS, Chang CL, Tsai CW, et al. (2009) Interaction of Exo1 genotypes and smoking habit in oral cancer in Taiwan. Oral Oncology 45: e90–94.  [DOI](https://doi.org/10.1016/j.oraloncology.2009.03.011) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19515603/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oral%20Oncology&title=Interaction%20of%20Exo1%20genotypes%20and%20smoking%20habit%20in%20oral%20cancer%20in%20Taiwan&author=MH%20Tsai&author=HC%20Tseng&author=CS%20Liu&author=CL%20Chang&author=CW%20Tsai&volume=45&publication_year=2009&pages=e90-94&pmid=19515603&doi=10.1016/j.oraloncology.2009.03.011&)

56. Wang HC, Chiu CF, Tsai RY, Kuo YS, Chen HS, et al. (2009) Association of genetic polymorphisms of EXO1 gene with risk of breast cancer in Taiwan. Anticancer Research 29: 3897–3901.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/19846925/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Anticancer%20Research&title=Association%20of%20genetic%20polymorphisms%20of%20EXO1%20gene%20with%20risk%20of%20breast%20cancer%20in%20Taiwan&author=HC%20Wang&author=CF%20Chiu&author=RY%20Tsai&author=YS%20Kuo&author=HS%20Chen&volume=29&publication_year=2009&pages=3897-3901&pmid=19846925&)

57. Bau DT, Wang HC, Liu CS, Chang CL, Chiang SY, et al. (2009) Single-nucleotide polymorphism of the Exo1 gene: association with gastric cancer susceptibility and interaction with smoking in Taiwan. The Chinese journal of physiology 52: 411–418.  [DOI](https://doi.org/10.4077/cjp.2009.amh076) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20337148/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=The%20Chinese%20journal%20of%20physiology&title=Single-nucleotide%20polymorphism%20of%20the%20Exo1%20gene:%20association%20with%20gastric%20cancer%20susceptibility%20and%20interaction%20with%20smoking%20in%20Taiwan&author=DT%20Bau&author=HC%20Wang&author=CS%20Liu&author=CL%20Chang&author=SY%20Chiang&volume=52&publication_year=2009&pages=411-418&pmid=20337148&doi=10.4077/cjp.2009.amh076&)

58. Rennert G, Rennert HS, Pinchev M, Lavie O, Gruber SB (2009) Use of hormone replacement therapy and the risk of colorectal cancer. Journal of clinical oncology: official journal of the American Society of Clinical Oncology 27: 4542–4547.  [DOI](https://doi.org/10.1200/JCO.2009.22.0764) | [PMC free article](/articles/PMC2754905/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19704062/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20of%20clinical%20oncology:%20official%20journal%20of%20the%20American%20Society%20of%20Clinical%20Oncology&title=Use%20of%20hormone%20replacement%20therapy%20and%20the%20risk%20of%20colorectal%20cancer&author=G%20Rennert&author=HS%20Rennert&author=M%20Pinchev&author=O%20Lavie&author=SB%20Gruber&volume=27&publication_year=2009&pages=4542-4547&pmid=19704062&doi=10.1200/JCO.2009.22.0764&)

59. Martinez ME (2001) Hormone replacement therapy and adenoma recurrence: implications for its role in colorectal cancer risk. Journal of the National Cancer Institute 93: 1764–1765.  [DOI](https://doi.org/10.1093/jnci/93.23.1764) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11734585/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20of%20the%20National%20Cancer%20Institute&title=Hormone%20replacement%20therapy%20and%20adenoma%20recurrence:%20implications%20for%20its%20role%20in%20colorectal%20cancer%20risk&author=ME%20Martinez&volume=93&publication_year=2001&pages=1764-1765&pmid=11734585&doi=10.1093/jnci/93.23.1764&)

60. Nanda K, Bastian LA, Hasselblad V, Simel DL (1999) Hormone replacement therapy and the risk of colorectal cancer: a meta-analysis. Obstetrics and gynecology 93: 880–888.  [DOI](https://doi.org/10.1016/s0029-7844(98)00424-4) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10912438/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Obstetrics%20and%20gynecology&title=Hormone%20replacement%20therapy%20and%20the%20risk%20of%20colorectal%20cancer:%20a%20meta-analysis&author=K%20Nanda&author=LA%20Bastian&author=V%20Hasselblad&author=DL%20Simel&volume=93&publication_year=1999&pages=880-888&pmid=10912438&doi=10.1016/s0029-7844(98)00424-4&)

61. Slattery ML, Lundgreen A, Herrick JS, Kadlubar S, Caan BJ, et al. (2011) Variation in the CYP19A1 gene and risk of colon and rectal cancer. Cancer causes & control: CCC 22: 955–963.  [DOI](https://doi.org/10.1007/s10552-011-9768-x) | [PMC free article](/articles/PMC3225228/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21479914/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20causes%20&%20control:%20CCC&title=Variation%20in%20the%20CYP19A1%20gene%20and%20risk%20of%20colon%20and%20rectal%20cancer&author=ML%20Slattery&author=A%20Lundgreen&author=JS%20Herrick&author=S%20Kadlubar&author=BJ%20Caan&volume=22&publication_year=2011&pages=955-963&pmid=21479914&doi=10.1007/s10552-011-9768-x&)

62. He T, Mo A, Zhang K, Liu L (2013) ABCB1/MDR1 gene polymorphism and colorectal cancer risk: a meta-analysis of case-control studies. Colorectal disease: the official journal of the Association of Coloproctology of Great Britain and Ireland 15: 12–18.  [DOI](https://doi.org/10.1111/j.1463-1318.2012.02919.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23279665/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Colorectal%20disease:%20the%20official%20journal%20of%20the%20Association%20of%20Coloproctology%20of%20Great%20Britain%20and%20Ireland&title=ABCB1/MDR1%20gene%20polymorphism%20and%20colorectal%20cancer%20risk:%20a%20meta-analysis%20of%20case-control%20studies&author=T%20He&author=A%20Mo&author=K%20Zhang&author=L%20Liu&volume=15&publication_year=2013&pages=12-18&pmid=23279665&doi=10.1111/j.1463-1318.2012.02919.x&)

63. Potocnik U, Glavac D, Dean M (2008) Common germline MDR1/ABCB1 functional polymorphisms and haplotypes modify susceptibility to colorectal cancers with high microsatellite instability. Cancer Genet Cytogenet 183: 28–34.  [DOI](https://doi.org/10.1016/j.cancergencyto.2008.01.023) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18474294/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Genet%20Cytogenet&title=Common%20germline%20MDR1/ABCB1%20functional%20polymorphisms%20and%20haplotypes%20modify%20susceptibility%20to%20colorectal%20cancers%20with%20high%20microsatellite%20instability&author=U%20Potocnik&author=D%20Glavac&author=M%20Dean&volume=183&publication_year=2008&pages=28-34&pmid=18474294&doi=10.1016/j.cancergencyto.2008.01.023&)

64. Fusco A, Fedele M (2007) Roles of HMGA proteins in cancer. Nature reviews Cancer 7: 899–910.  [DOI](https://doi.org/10.1038/nrc2271) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18004397/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature%20reviews%20Cancer&title=Roles%20of%20HMGA%20proteins%20in%20cancer&author=A%20Fusco&author=M%20Fedele&volume=7&publication_year=2007&pages=899-910&pmid=18004397&doi=10.1038/nrc2271&)
