# Genome-wide Association Study of Atazanavir Pharmacokinetics and Hyperbilirubinemia in AIDS Clinical Trials Group Protocol A5202

## Metadata
**Authors:** Daniel H Johnson, Charles Venuto, Marylyn D Ritchie, Gene D Morse, Eric S Daar, Paul J McLaren, David W Haas
**Journal:** Pharmacogenetics and genomics
**Date:** 2014 Apr
**DOI:** [10.1097/FPC.0000000000000034](https://doi.org/10.1097/FPC.0000000000000034)
**PMID:** 24557078
**PMCID:** PMC4059003
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4059003/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC4059003/pdf/nihms-558198.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC4059003/pdf/nihms-558198.pdf)

## Abstract

**Background:** 
Atazanavir-associated hyperbilirubinemia can cause premature discontinuation of atazanavir and avoidance of its initial prescribing. We used genome-wide genotyping and clinical data to characterize determinants of atazanavir pharmacokinetics and hyperbilirubinemia in AIDS Clinical Trials Group protocol A5202.

**Methods:** 
Plasma atazanavir pharmacokinetics and indirect bilirubin concentrations were characterized in HIV-1-infected subjects randomized to atazanavir/ritonavir-containing regimens. A subset had genome-wide genotype data available.

**Results:** 
Genome-wide assay data were available from 542 subjects, of who 475 also had estimated atazanavir clearance and relevant covariate data available. Peak bilirubin concentration and relevant covariates were available for 443 participants. By multivariate analysis, higher peak on-treatment bilirubin was associated with UGT1A1 rs887829 T allele (P=6.4×10−12), higher baseline hemoglobin (P=4.9×10−13), higher baseline bilirubin (P=6.7×10−12), and slower plasma atazanavir clearance (P=8.6×10−11). For peak bilirubin >3.0 mg/dL, the positive predictive value of baseline bilirubin ≥0.5 mg/dL with hemoglobin ≥14g/dL was 0.51, which increased to 0.85 with rs887829 TT homozygosity. For peak bilirubin ≤3.0 mg/dL, the positive predictive value of baseline bilirubin <0.5 mg/dL with hemoglobin <14 g/dL was 0.91, which increased to 0.96 with rs887829 CC homozygosity. No polymorphism predicted atazanavir pharmacokinetics at genome-wide significance.

**Conclusions:** 
Atazanavir-associated hyperbilirubinemia is best predicted by considering UGT1A1 genotype, baseline bilirubin, and baseline hemoglobin values in combination. Use of ritonavir as a pharmacokinetic enhancer may have abrogated genetic associations with atazanavir pharmacokinetics.

Keywords: HIV, atazanavir, pharmacogenomics, pharmacokinetics, UGT1A1

### Background

Atazanavir-associated hyperbilirubinemia can cause premature discontinuation of atazanavir and avoidance of its initial prescribing. We used genome-wide genotyping and clinical data to characterize determinants of atazanavir pharmacokinetics and hyperbilirubinemia in AIDS Clinical Trials Group protocol A5202.

### Methods

Plasma atazanavir pharmacokinetics and indirect bilirubin concentrations were characterized in HIV-1-infected subjects randomized to atazanavir/ritonavir-containing regimens. A subset had genome-wide genotype data available.

### Results

Genome-wide assay data were available from 542 subjects, of who 475 also had estimated atazanavir clearance and relevant covariate data available. Peak bilirubin concentration and relevant covariates were available for 443 participants. By multivariate analysis, higher peak on-treatment bilirubin was associated with *UGT1A1* rs887829 T allele (P=6.4×10^−12^), higher baseline hemoglobin (P=4.9×10^−13^), higher baseline bilirubin (P=6.7×10^−12^), and slower plasma atazanavir clearance (P=8.6×10^−11^). For peak bilirubin >3.0 mg/dL, the positive predictive value of baseline bilirubin ≥0.5 mg/dL with hemoglobin ≥14g/dL was 0.51, which increased to 0.85 with rs887829 TT homozygosity. For peak bilirubin ≤3.0 mg/dL, the positive predictive value of baseline bilirubin <0.5 mg/dL with hemoglobin <14 g/dL was 0.91, which increased to 0.96 with rs887829 CC homozygosity. No polymorphism predicted atazanavir pharmacokinetics at genome-wide significance.

### Conclusions

Atazanavir-associated hyperbilirubinemia is best predicted by considering *UGT1A1* genotype, baseline bilirubin, and baseline hemoglobin values in combination. Use of ritonavir as a pharmacokinetic enhancer may have abrogated genetic associations with atazanavir pharmacokinetics.

## Introduction

The once-daily HIV-1 protease inhibitor atazanavir with low-dose ritonavir as a pharmacokinetic enhancer (atazanavir/r) is generally safe, effective, and well tolerated [[1](#R1)-[3](#R3)] in first-line regimen for HIV-1 infection [[4](#R4)]. Atazanavir increases plasma indirect bilirubin concentrations by inhibiting uridine diphosphate glucuronosyltransferase (UGT) 1A1-mediated bilirubin glucuronidation [[5](#R5)]. Approximately 40% of patients prescribed atazanavir/r experience at least one indirect bilirubin elevation greater than 2.5 times the upper limit of normal (i.e. grade 3) [[1](#R1), [6](#R6)-[8](#R8)], and approximately 5% at least 5 times the upper limit of normal (i.e. grade 4) [[1](#R1), [8](#R8)]. While atazanavir-associated hyperbilirubinemia does not indicate hepatic injury, some patients discontinue atazanavir due to icterus [[3](#R3), [9](#R9), [10](#R10)], or are not prescribed atazanavir to avoid this possibility. Reliable baseline predictors of subsequent hyperbilirubinemia could inform atazanavir/r prescribing.

Greater plasma atazanavir concentrations are associated with greater plasma bilirubin elevations [[7](#R7), [11](#R11)]. Atazanavir undergoes phase I metabolism by hepatic cytochrome P450 (CYP) 3A isoforms [[12](#R12)] and is a substrate for membrane transporters including P-glycoprotein (encoded by *ABCB1*), multidrug resistance-associated proteins (MRPs, encoded by *ABCC* genes), and organic anion transporting polypeptides (OATPs, encoded by *SLCO* genes) [[13](#R13)]. Candidate gene studies, most involving unboosted atazanavir, have suggested associations between atazanavir pharmacokinetics and genetic polymorphisms in *ABCB1* [[14](#R14)-[16](#R16)], *CYP3A5* [[16](#R16)-[18](#R18)], and *NR1I2* (encoding pregnane X receptor) [[16](#R16), [19](#R19)]. Plasma atazanavir exposure is also influenced by non-genetic factors including concomitant antiretrovirals [[20](#R20)-[22](#R22)], other medications such as rifampin {Acosta, 2007 #2192}, food [[23](#R23)], and gastric acid blocking medications [[12](#R12), [24](#R24)].

Interindividual differences in plasma indirect bilirubin concentrations have been associated with a *UGT1A1* promoter tandem TA repeat. The *UGT1A1*28* (TA) _7_ allele is associated with reduced *UGT1A1* transcription as compared to *UGT1A1*1* (TA)_6_ [[25](#R25), [26](#R26)]. Among atazanavir recipients, *UGT1A1*28* has been strongly associated with unconjugated hyperbilirubinemia [[27](#R27), [28](#R28)], but inconsistently with atazanavir discontinuation[[28](#R28), [29](#R29)].

The present study used a genome-wide approach to investigate genetic and non-genetic associations with hyperbilirubinemia, and genetic predictors of plasma atazanavir pharmacokinetics among subjects randomized to atazanavir/r-containing regimens in a prospective clinical trial.

## Methods

### Study Participants

AIDS Clinical Trials Group (Protocol) A5202 (ClinTrials.gov [NCT00118898](https://clinicaltrials.gov/ct2/show/NCT00118898)) was a phase IIIb equivalence study of four once-daily regimens for initial treatment of HIV-1 infection. Primary results of A5202 have been previously reported [[3](#R3), [30](#R30)]. Briefly, A5202 subjects enrolled from 2005 to 2007 were randomized to open-label atazanavir (300 mg) plus ritonavir (100 mg), or efavirenz (600 mg), with either placebo-controlled abacavir/lamivudine (600 mg/300 mg) or tenofovir DF/emtricitabine (300 mg/200 mg). Study evaluations including indirect bilirubin and hemoglobin determinations were performed before entry, at entry, at weeks 4, 8, 16 and 24, and every 12 weeks thereafter until the last enrolled subject was followed 96 weeks. Bilirubin and hemoglobin were assayed at research site clinical laboratories. Subjects in the present study were also the basis of a previous analysis focused on *UGT1A1*28* and premature discontinuation of atazanavir[[28](#R28)].

### Atazanavir assays and plasma sampling

In A5202, plasma samples for atazanavir assays were obtained during any two scheduled visits during the first 24 weeks of study. At one visit a sample was to be drawn immediately before an observed dose and again 3-4 hours later. During the second visit a sample was drawn between 5 and 12-15 hours post-dose (12 versus 15 hours depending on whether the dose was in the morning or evening). Additional samples for atazanavir assay were collected at week 48 and every 48 weeks thereafter, at final study visit, with first documented virologic failure, and with medication change due to virologic failure.

Atazanavir was quantified using a previously reported reverse phase high performance liquid chromatography (HPLC) method utilized at the University at Buffalo. Atazanavir was separated on a Waters 5μm Symmetry™ shield RP C8, 3.0 × 150 mm column, with a chromatographic system consisting of Waters 2695 Alliance Separations Module, and a 996 Photodiode Array Detector. The system was controlled by Waters Empower 2 software Version 6.20.00.00 that collected all chromatographic data for analysis, generating a calibration standard curve that was linear from 100-16,000 ng/mL.

### Derivation of pharmacokinetic parameters

A model-based population pharmacokinetic analysis was performed using pharmacokinetic samples collected at steady-state during the first 24 weeks of therapy. Concentration-time data were analyzed using the first-order conditional estimation with interaction (FOCE-I) method of the non-linear mixed effects modeling program NONMEM (version VII; ICON, Ellicott City, MD). One- and two-compartment models were tested to identify an appropriate structural model. A single compartment model with first-order absorption and elimination was ultimately selected based on successful convergence (which two-compartment models did not) and goodness-of-fit plots ([Supplementary Figure, Supplemental Digital Content 1](#SD1), [http://links.lww.com/FPC/A688](http://links.lww.com/FPC/A688)). Estimation of population pharmacokinetic parameters included clearance, volume of distribution, and absorption rate constant. Interindividual random effects were described on an exponential scale, while a proportional error model was used to describe residual variability. Individual Bayesian estimates of oral clearance values of atazanavir were estimated from the structural model.

### Identifying genetic polymorphisms

Consent for genetic analyses was obtained under ACTG protocol A5128 [[31](#R31)]. The Vanderbilt Institutional Review Boards and the ACTG approved this use of DNA. Genome-wide data from the Illumina Human-1M-Duo platform were available from a separate immunogenomics project [[32](#R32)]. Genetic data management and association analyses were performed with PLINK version 1.07 [[33](#R33)]. Genetic ancestry was assigned using principal components as described elsewhere [[32](#R32)]. Samples that did not group with African American, European, or Hispanic populations were excluded. Analyses were performed separately for African American, European, and Hispanic ancestry populations, and for all three populations combined. Within each population, only polymorphisms with >98% genotyping efficiency and >2% minor allele frequency were included in analyses. Samples were censored for >2% missingness, excessive heterozygosity (|F| >0.1), or possible sex misclassification. For sample pairs with high estimated relatedness as determined by identity by descent (pi-hat > 0.125), only one sample was included in analyses. Final analyses included as many as 925,738 assayed polymorphisms.

Targeted genotyping of *UGT1A1* rs8175347 (i.e. (TA)_5 _**36*, (TA)_6 _**1*, (TA)_7 _**28*, and (TA)_8 _**37*) was by fragment analysis as described elsewhere [[34](#R34)]. Laboratory personnel with no knowledge of clinical data performed genotyping. Genotype assignment was confirmed by visual inspection for electrophoresis peaks representing promoter TA tandem repeats **36* (TA)_5_, **1* (TA)_6_, **28* (TA)_7_, and **37* (TA)_8_.

### Association testing

Genetic associations were assessed using an uncorrected Wald test, as well as a linear regression model that adjusted for covariates that by univariate analysis had P<0.05. For atazanavir clearance, covariates included were body mass index, abacavir/lamivudine versus tenofovir DF/emtricitabine assignment at randomization, sex, and occurrence of an atazanavir concentration below limit of quantification. Peak bilirubin was defined as the highest concentration during the first 24 weeks of atazanavir/r prescribing in A5202. For peak bilirubin, univariate analyses showed associations with age, baseline bilirubin concentration, baseline hemoglobin concentration, atazanavir clearance, and sex; after correcting for hemoglobin, sex was no longer associated so was not included as a covariate.

To correct for population stratification, principal components were determined separately for each population by EIGENSTRAT [[35](#R35)]. The first two principal components were included as covariates for African American, European, and Hispanic groups. The first four principal components generated with the combined population were included as covariates in the combined group analyses. P values were adjusted for genomic inflation factor based on median chi-squared value.

From our genome-wide dataset, 13 functional polymorphisms in genes suggested to affect atazanavir disposition were chosen a priori from PharmGKB[[36](#R36)] (*CYP3A4* [[12](#R12)], *CYP3A5* [[12](#R12), [17](#R17), [18](#R18)], *ABCB1* [[13](#R13)], *SLCO1B1*[[13](#R13)], *ABCC1* [[13](#R13)], *ABCC2* [[13](#R13)], and *NR1I2* [[16](#R16), [37](#R37)]) and previous atazanavir pharmacogenetic studies [[14](#R14)-[19](#R19)]. Similarly, a targeted set of 27 polymorphisms previously associated with bilirubin metabolism was assembled a priori.

Non-genetic statistical analyses were performed using R version 2.15.0 [[38](#R38)], using two-sided tests for P values and confidence intervals, without adjusting for multiple comparisons. Bonferroni-corrected genome-wide significance was defined as 5.0×10^−8^ giving a genome-wide family error rate of 0.05 for 925,738 comparisons. For the 13 polymorphisms in atazanavir disposition genes and 27 in bilirubin associated genes, Bonferroni corrected P value thresholds for the single combined group were 0.0038 and 0.0019, respectively. For analyses of these a priori designated polymorphisms in all three ethnic groups, Bonferroni corrected P values were 0.0013 and 0.00062 for pharmacokinetic and bilirubin associated genes respectively.

## Results

### Study Participants

Among 928 subjects randomized to receive atazanavir/r-containing regimens in A5202, atazanavir plasma concentration data were available from 815 subjects for population pharmacokinetic modeling. Forty-eight out of 2195 atazanavir concentrations were below the lower limit of quantitation (< 3%), and were substituted with concentration values of 50 ng/mL. Genome-wide assay data were available from 542 subjects [[32](#R32)], of who 475 (88%) also had estimated atazanavir clearance and relevant covariate data available. Peak bilirubin concentration and relevant covariates were available for 443 participants. Derivation of the study population is shown in **Figure 1**. Baseline characteristics of study participants are shown in **Table 1**.

### Figure 1. Derivation of the study population.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f1ee/4059003/c1fc921b8861/nihms-558198-f0001.jpg)

Of 928 individuals randomized to atazanavir-containing regimens in A5202, a total of 464 were ultimately included in analyses for genetic associations with baseline indirect bilirubin, 444 in analyses for genetic associations with peak indirect bilirubin, and 474 in analyses for genetic associations with atazanavir clearance.

### Table 1.

|   | Pharmacokinetic association analyses (N = 475) | Bilirubin association analyses (N = 443) |
| --- | --- | --- |
| Race/ethnicity |   |   |
| Black, n (%) | 145 (31%) | 135 (30%) |
| White, n (%) | 212 (45%) | 204 (46%) |
| Hispanic, n (%) | 118 (25%) | 104 (23%) |
| Female | 13.5% | 14.2% |
| Randomized to TDF/FTCa | 51% | 51% |
| Median BMIb in kg/m2 (SDc) | 25.7 (5.15) | 25.8 (5.19) |
| Median age in years (SD) | 38.5 (10.3) | 38.8 (10.3) |
| Median baseline bilirubin in mg/dL (IQRd) | - | 0.5 (0.3 - 0.8) |
| Median baseline hemoglobin in g/dL (IQR) | - | 13.8 (12.4 - 14.9) |

Table 1 Caption: Baseline characteristics of study subjects

### Genetic associations with atazanavir pharmacokinetics

The parameter estimates (and relative standard error as percentage) for the one-compartment pharmacokinetic model were 0.47 h^−1^ (11%) for the first-order absorption rate constant, 86.7 L (5%) for the volume of distribution, and 7.86 L/h (2%) for apparent oral clearance with an interindividual variability of 36% (8%). The median estimated atazanavir clearance for subjects with genome-wide assay data was 7.75 L/hr (95% C.I. 7.54 to 7.96 L/hr). No polymorphism was associated with clearance at genome-wide significance, whether considering each population separately, or in a combined analysis including all three populations. Of the 10 polymorphisms with the lowest P values in the combined population, none were in genes with apparent relevance to drug disposition (**Table 2, top)**. Data for all polymorphisms associated with estimated atazanavir clearance at P <5.0×10^–5^ are listed in [Supplemental Table 1, Supplemental Digital Content 2](#SD2), [http://links.lww.com/FPC/A689](http://links.lww.com/FPC/A689). Of the 13 polymorphisms designated a priori for possible associations with atazanavir clearance, none were significant after correcting for multiple comparisons in the combined population. There were trends toward nominal associations with rs4149056 in *SLCO1B1* (β= −0.089, P=0.049) and rs776746 in *CYP3A5* (β= 0.076, P=0.097), as shown in **Table 3**.

### Table 2.

| SNP | Overall | Overall β | Gene | Distance | AfAm MAF | AfAm β | Hisp MAF | Hisp β | Euro MAF | Euro β |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| Associations with clearance |  |   |   |   |   |   |   |   |   |   |
| rs1899979 | 2.38×10−6 | 0.21 | BICC1 | 0 | 0.34 | 0.18 | 0.26 | 0.23 | 0.24 | 0.22 |
| rs2335462 | 3.81×10−6 | 0.21 | FLYWCH2 | 0 | 0.25 | 0.2 | 0.3 | 0.16 | 0.27 | 0.2 |
| rs17156916 | 5.05×10−6 | −0.2 | NUDT12 | 1158305 | 0.1 | −0.25 | NA | NA | 0.15 | −0.26 |
| rs4719772 | 6.07×10−6 | 0.2 | MPP6 | −184933 | 0.44 | 0.11 | 0.48 | 0.3 | 0.41 | 0.19 |
| rs8041911 | 1.26×10−5 | 0.19 | DUOX1 | 0 | 0.17 | 0.26 | 0.22 | 0.3 | 0.22 | 0.1 |
| rs540171 | 1.28×10−5 | 0.19 | LPAR1 | −53662 | 0.2 | 0.3 | 0.1 | 0.25 | 0.1 | 0.1 |
| rs575107 | 1.28×10−5 | 0.19 | LPAR1 | −52145 | 0.2 | 0.3 | 0.1 | 0.25 | 0.1 | 0.1 |
| rs211069 | 1.33×10−5 | 0.19 | PRKG1 | 0 | 0.46 | 0.14 | 0.5 | 0.21 | 0.44 | 0.2 |
| rs6929400 | 1.37×10−5 | −0.19 | C6orf105 | 0 | 0.18 | −0.17 | 0.28 | −0.15 | 0.42 | −0.22 |
| rs10902646 | 1.42×10−5 | −0.19 | SFN | −6648 | 0.19 | −0.25 | 0.24 | −0.1 | 0.42 | −0.21 |

Table 2 Caption: Polymorphisms with the lowest P-values for association with estimated plasma atazanavir clearance in the combined population analysis.

### Table 3.

| SNP | Pclearance | β clearance | CHR | Position (bp) | Gene | AfAm MAF | Hisp MAF | Euro MAF | AfAm βClearance | Hisp βClearance | Euro βClearance |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| rs10264272 | 0.73 | −0.016 | 7 | 99100771 | CYP3A5 | 0.13 | - | - | −0.026 | - | - |
| rs1045642 | 0.23 | −0.054 | 7 | 86976581 | ABCB1 | 0.23 | 0.41 | 0.43 | −0.015 | −0.082 | 0.038 |
| rs1128503 | 1 | −0.00018 | 7 | 87017537 | ABCB1 | 0.23 | 0.41 | 0.45 | −0.037 | −0.046 | 0.047 |
| rs17222723 | 0.36 | −0.041 | 10 | 101585986 | ABCC2 | 0.061 | 0.044 | 0.05 | −0.18 | 0.078 | 0.0023 |
| rs2229109 | 0.76 | −0.013 | 7 | 87017745 | ABCB1 | - | 0.042 | 0.039 | - | −0.13 | 0.11 |
| rs2235015 | 0.49 | 0.031 | 7 | 87037500 | ABCB1 | 0.33 | 0.19 | 0.18 | −0.083 | 0.22 | 0.03 |
| rs2238476 | 0.39 | −0.038 | 16 | 16121373 | ABCC1 | 0.045 | 0.11 | 0.048 | −0.059 | −0.11 | −0.00078 |
| rs2273697 | 0.61 | 0.023 | 10 | 101553805 | ABCC2 | 0.15 | 0.16 | 0.2 | 0.044 | 0.056 | 0.0024 |
| rs246240 | 0.48 | 0.032 | 16 | 16026525 | ABCC1 | 0.1 | 0.2 | 0.17 | 0.013 | 0.04 | 0.05 |
| rs35592 | 0.62 | −0.022 | 16 | 16049324 | ABCC1 | 0.5 | 0.3 | 0.25 | −0.059 | 0.0068 | −0.045 |
| rs4149056 | 0.049 | −0.089 | 12 | 21222816 | SLCO1B1 | 0.031 | 0.15 | 0.17 | −0.093 | −0.082 | −0.11 |
| rs776746 | 0.097 | 0.076 | 7 | 99108475 | CYP3A5 | 0.31 | 0.23 | 0.071 | −0.11 | 0.18 | −0.014 |
| rs8187710 | 0.98 | 0.00085 | 10 | 101601284 | ABCC2 | 0.16 | 0.054 | 0.051 | −0.077 | 0.07 | 0.028 |

Table 3 Caption: Associations between targeted drug metabolism and transporter gene polymorphisms and plasma atazanavir clearance.

When each population was analyzed separately, of the 13 targeted polymorphisms, rs2235015 in *ABCB1* was nominally associated with clearance among Hispanics (P_Clearance_=0.013). Additionally, clearance was nominally associated in African Americans with rs1722273 in *ABCC2* (P=0.033) and in Hispanics with rs776746 in *CYP3A5* (P=0.046). None were significant after Bonferroni correction.

### Associations with indirect bilirubin concentrations

Among the 443 participants included in analyses for genomic associations with peak bilirubin concentrations, median baseline indirect bilirubin was 0.5 mg/dL (IQR 0.3 - 0.8 mg/dL), and median peak indirect bilirubin during the first 24 weeks of therapy was 2.2 mg/dL (IQR 1.6 - 3.4 mg/dL). By univariate analysis, greater baseline (i.e. pre-atazanavir) bilirubin concentrations were significantly associated with male sex (P=1.4×10^−5^) and greater baseline hemoglobin concentration (P=7.1×10^−6^, β=0.094 bilirubin mg/dL per hemoglobin g/dL). In genome-wide analysis of baseline indirect bilirubin in the combined population, 10 polymorphisms were genome-wide significant, each in or near *UGT1A1*. The smallest P value was for rs6742078 (P=1.8×10^−9^, β=0.30 mg/dL per T allele).

Clinical variables associated with greater on-treatment peak indirect bilirubin concentration by univariate analysis included greater baseline indirect bilirubin concentration (P=2.61×10^−25^, β=0.36), greater baseline hemoglobin concentration (P=1.37x10^−13^, β=0.12 (mg/dL)/(g/dL)), slower estimated atazanavir clearance (P=8.98x10^−9^, β= −0.55 (mg/dL) per (L/h)), and younger age (P=4.93x10^−4^, β= −0.0097 mg/dL per year). There was no association between on-treatment peak indirect bilirubin concentration and either baseline alanine aminotransferase (ALT, P=0.72) or baseline hepatitis C seropositivity (P= 0.16). In multivariate genome-wide analysis of peak on-treatment indirect bilirubin concentration in the combined population (adjusting for baseline bilirubin concentration, baseline hemoglobin concentration, age, atazanavir clearance, and the top 4 principal components), 10 polymorphisms in or near *UGT1A1* were genome-wide significant, the smallest P value being for rs887829. In this analysis, greater peak bilirubin was associated with rs887829 T allele (P=6.4×10^−12^, β=0.30 mg/dL per T allele), greater baseline bilirubin concentration (P=6.7×10^−12^, β=0.22), slower estimated atazanavir clearance (P=8.6×10^−11^, β= −0.51 (mg/dL)/(L/hr)), and greater baseline hemoglobin concentration (P=4.9×10^−13^, β=0.098 (mg/dL)/(g/dL)). After controlling for rs887829, no additional polymorphisms were genome-wide significant. Of 27 targeted polymorphisms selected a priori for possible association with bilirubin, only those in or near *UGT1A1* were significant after Bonferroni correction ([Supplemental Table 2, Supplemental Digital Content 3](#SD3), [http://links.lww.com/FPC/A690](http://links.lww.com/FPC/A690)).

In addition to genome-wide assay data, the *UGT1A1* tandem TA repeat (rs8175347 which defines *UGT1A1 *28*, **36* and **37* alleles) was directly assayed. Among A5202 subjects, rs8175347 was in almost complete linkage disequilibrium with rs887829 (R^2^=0.99), so was not informative beyond rs887829.

### Prediction of peak on-treatment bilirubin

Associations between peak plasma bilirubin >3.0 mg/dL (an approximate concentration above which jaundice icterus tends to be noticeable) and rs887829 genotype were considered in relation to a baseline bilirubin threshold of 0.5 mg/dL and a baseline hemoglobin threshold of 14 g/dL. These baseline thresholds approximate median values. As shown in **Figure 2**, the positive predictive value of baseline bilirubin ≥0.5 mg/dL and baseline hemoglobin ≥14 g/dL for peak bilirubin >3.0 mg/dL was 0.51 (95% C.I 0.40 to 0.57). This increased to 0.85 (95% C.I., 0.65 to 0.96) by also including rs887829 TT homozygosity. Conversely, the positive predictive value of baseline bilirubin <0.5 mg/dL and baseline hemoglobin <14 g/dL for peak bilirubin ≤3.0 mg/dL was 0.91 (95% C.I., 0.85 to 0.95). This increased to 0.96 (95% C.I., 0.88 to 0.99) by also including rs887829 CC homozygosity. Among individual with rs887829 CT heterozygosity and/or representing discordant baseline bilirubin and hemoglobin strata, the likelihood of peak bilirubin greater or less than 3.0 mg/dL reflected ordinal effects of each of the three variables (**Figure 2**). Additional analyses that considered several alternative peak plasma bilirubin thresholds greater than 3.0 mg/dL did not substantially improve prediction (data not shown).

### Figure 2. Predictive values of baseline bilirubin, baseline hemoglobin, and rs887829 genotype for peak on-treatment bilirubin.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f1ee/4059003/daa7d84e1f72/nihms-558198-f0002.jpg)

For these analyses a cut-off of 3.0 mg/dL was used. Each dot represents a different individual. Positive predictive values and 95% confidence intervals are provided. Positive predictive value = (number of participants with peak bilirubin >3.0 mg/dL) / (all individuals with the predictor variables indicated). BBR, baseline bilirubin; HGB, baseline hemoglobin.

Multivariable models assessed the extent to which interindividual variability in peak on-treatment bilirubin concentrations could be explained by genetic and non-genetic factors, based on adjusted-R^2^ values (**Table 4**). A model that included all four variables (baseline bilirubin concentration, baseline hemoglobin concentration, estimated plasma atazanavir clearance, and rs887829 genotype) explained 41.3% of variability in peak on-treatment bilirubin concentrations (Model 6). In models that included baseline hemoglobin concentration and estimated plasma atazanavir clearance, adding either baseline bilirubin concentration or rs887829 genotype contributed comparably to peak on-treatment bilirubin concentrations (33.4% in Model 1 vs. 33.6% in Model 2). In models that included baseline hemoglobin concentration, adding either baseline bilirubin concentration or rs887829 genotype contributed comparably to peak on-treatment bilirubin concentrations (30.3% in Model 3 vs. 26.4% in Model 4).

### Table 4.

| Model # |  |  |  |  |  |  |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- |
|   | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 |
| HGB | + | + | + | + | + | + | + | + |
| ATV clearance | + | + | 0 | 0 | 0 | + | + | 0 |
| Baseline Bilirubin | + | 0 | + | 0 | + | + | 0 | 0 |
| rs887829 | 0 | + | 0 | + | + | + | 0 | 0 |
| R2 Adjusted* | 0.334 | 0.336 | 0.303 | 0.264 | 0.368 | 0.413 | 0.181 | 0.125 |

Table 4 Caption: Adjusted R2 for prediction of peak bilirubin of various models

## Discussion

We used a genome-wide approach to investigate associations between human genetic variants, pharmacokinetics, and indirect bilirubin concentrations among subjects randomized to receive atazanavir/r in ACTG protocol A5202. In a multivariate model, variables independently associated with peak on-treatment indirect bilirubin concentrations were *UGT1A1* rs887829 genotype, baseline bilirubin concentration, baseline hemoglobin concentration, and atazanavir clearance, which together explained 41.3% of interindividual variability. The best pre-treatment predictor of peak on-treatment indirect bilirubin ≤3.0 mg/dL (i.e. excluding atazanavir pharmacokinetic data) was the combination of rs887829 CC homozygosity, lower bilirubin concentration, and lower hemoglobin concentration. Conversely, the best pre-treatment predictor of peak on-treatment indirect bilirubin >3.0 mg/dL was the combination of rs887829 TT homozygosity, greater bilirubin concentration, and greater hemoglobin concentration.

Previous candidate gene studies have suggested associations between atazanavir pharmacokinetics and polymorphisms in drug metabolism and/or transporter genes. The present study did not replicate such associations, nor did we identify new genetic associations with atazanavir pharmacokinetics among targeted polymorphisms in genes relevant to atazanavir disposition or in genome-wide analyses. We hypothesize that ritonavir, by inhibiting the activity of CYP isoforms including CYP3A5, may have masked potential genetic associations with atazanavir pharmacokinetics. Drug metabolism and transporter gene polymorphisms were therefore not included in multivariate models for on-treatment bilirubin concentrations.

In the combined population analysis of atazanavir clearance there were trends toward nominally significant associations with *SLCO1B1* 521T→C (rs4149056) and the *CYP3A5*3* loss-of-function polymorphism (rs776746). Among Hispanics, rs776746 was also nominally associated with clearance. However, neither polymorphism was significant after Bonferroni correction among Blacks, Whites, or Hispanics analyzed separately. Multiple prior studies have associated *SLCO1B1* rs4149056 with increased plasma lopinavir concentrations among patients prescribed lopinavir/ritonavir[[39](#R39)-[42](#R42)][[43](#R43)] and one study associated this polymorphisms with decreased plasma amprenavir concentrations among patients prescribed fos-amprenavir/ritonavir [[41](#R41)]. A prior study of 31 HIV-negative individuals suggested an association between *CYP3A5* genotype and plasma clearance of atazanavir (without ritonavir), which lessened when combined with ritonavir [[17](#R17)]. The present study did not replicate *ABCB1* 3435C→T (rs1045642), which has been reported to be associated with pharmacokinetics for atazanavir without ritonavir [[14](#R14)] and with ritonavir [[15](#R15)]. While we cannot explain the failure of *ABCB1* 3435C→T to replicate in our study, reported associations of this polymorphism with pharmacokinetics and other phenotypes for many drugs have failed to replicate. The *NR1I2* polymorphism 63396C→T (rs2472677) reported to be associated with atazanavir pharmacokinetics [[19](#R19), [37](#R37)] was not assayed in the present study.

The *UGT1A1* rs887829 polymorphism is a well-established predictor of increased plasma indirect bilirubin concentrations in the general population (i.e. Gilbert trait) [[44](#R44)-[46](#R46)], and accounts for approximately 15% of variation in African Americans [[44](#R44)] and Europeans [[46](#R46)]. In the present study rs887829 was in almost complete linkage disequilibrium with *UGT1A1*28* (rs8175347) genotype. Previous studies have associated *UGT1A1*28* with greater indirect bilirubin elevations among HIV-infected individuals prescribed indinavir [[27](#R27)] or atazanavir [[27](#R27), [28](#R28)]. In a study of patients HIV-infected patients prescribed indinavir and HIV-negative individuals, increased likelihood of hyperbilirubinemia was associated with multiple polymorphisms in *UGT1A* genes including *UGT1A1**28, *UGT1A7* (rs7586110), *UGT1A7* (rs17868323), and *UGT1A3* (rs3806596). These associations were also seen in the present study, but disappeared after correcting for rs887829.

Polymorphisms in genes beyond *UGT1A1* have been reported to be associated with serum bilirubin concentrations in the general population, including *ABCC2, ABCB4, ABCB11, ATP8B1*, *SLCO1B1* [[46](#R46)], *SLCO1B3* and *G6PD* [[45](#R45)]. In addition, bilirubin concentrations have been associated with *ABCB1* 3435C→T among patients prescribed atazanavir without ritonavir but not with ritonavir [[17](#R17)], although results have been inconsistent [[14](#R14)]. The present study did not replicate genetic predictors of hyperbilirubinemia beyond rs887829 ([Supplemental Table 2, Supplemental Digital Content 3](#SD3), [http://links.lww.com/FPC/A690](http://links.lww.com/FPC/A690)).

It is not unexpected that baseline bilirubin is associated was peak on-treatment indirect bilirubin concentrations, independent of *UGT1A1* rs887829 genotype. In population-based studies, genetic polymorphisms have explained less than 50% of interindividual variability in bilirubin concentrations. Non-genetic factors such as increased rates of red blood cell turnover (i.e. increased bilirubin production) are certain to affect baseline bilirubin concentrations. The association between peak on-treatment indirect bilirubin concentrations and higher hemoglobin concentrations may be the consequence of a greater mass of red blood cells from which to produce bilirubin.

In genetic association studies, unrecognized population stratification can lead to false discovery. This was addressed in the present study by correcting for genetic ancestry using principal components in the total population analysis as well as in each race/ethnicity group analyzed separately.

There were limitations to the present study. Some previously reported pharmacogenetic associations have involved atazanavir without ritonavir [[14](#R14), [16](#R16), [19](#R19), [47](#R47)], and a decreased effect of genetic variants has been suggested for atazanavir with ritonavir [[15](#R15), [17](#R17)]. In the present study, genetic associations with atazanavir pharmacokinetics may have been abrogated by the use of atazanavir with ritonavir. Clearance was estimated from sparse pharmacokinetic data applied to a population pharmacokinetic model, and errors in the model would have misestimated pharmacokinetic parameters. However, Colombo et al compared population pharmacokinetic modeling for a rich versus a sparse data set and found similar individual variances between the two groups [[24](#R24)]. Although we used a genome-wide approach, it is possible that polymorphisms not assayed herein, such as rare variants and copy number polymorphisms, or with weaker effect sizes than can be detected with this sample size. Our sample size of 443 participants for bilirubin analyses and 475 participants for pharmacokinetic analyses may have limited our ability to identify polymorphisms with small effects. However, unlike GWAS of complex traits such as diabetes, large effect sizes with drug metabolism/transport genes and off-target genes often reveal genome-wide significant associations with modest sample sizes. For example, genetic prediction of abacavir hypersensitivity is genome-wide significant (P<5.0×10^−8^) with only 15 cases and 200 controls [[48](#R48)], statin response with only 85 cases and 90 controls [[49](#R49)], and nevirapine rash with only 71 cases and 77 controls [[50](#R50)].

In summary, peak plasma indirect bilirubin concentrations during the first 24 weeks of therapy are associated with *UTG1A1* rs887829, baseline bilirubin concentrations, baseline hemoglobin concentrations, and atazanavir clearance. It is possible that therapeutic drug monitoring with atazanavir dose reduction in selected individuals could reduce hyperbilirubinemia, but this question was not addressed by the present study. Consideration of genetic and clinical laboratory data at baseline best stratifies risk for subsequent development of on-treatment hyperbilirubinemia.

## Supplementary Material

## ACKNOWLEDGMENTS

The authors are grateful to the many persons with HIV infection who volunteered for A5202 and A5128. In addition, they acknowledge the contributions of study teams and site staff for protocols A5202 and A5128. The project described was supported by Award Number U01AI068636 from the National Institute of Allergy and Infectious Diseases (NIAID) and supported by National Institute of Mental Health (NIMH), National Institute of Dental and Craniofacial Research (NIDCR). The content is solely the responsibility of the authors and does not necessarily represent the official views of the National Institute of Allergy and Infectious Diseases or the National Institutes of Health. This work was supported by the AIDS Clinical Trials Group funded by the National Institute of Allergy and Infectious Diseases (AI-068636, AI-038858, AI-068634, AI-038855). Grant support included AI-069439, RR-024975, AI-054999 (DWH), AI-077505 (HJR), BRS-ACURE-06-00140-T001 (GDM), and AI-069434 (ACC).

Clinical Research Sites that participated in ACTG protocol A5202, and collected DNA under protocol A5128, were supported by the following grants from NIAID: AI-069477, AI-027675, AI-073961, AI-069474, AI-069432, AI-069513, AI-069423, AI-050410, AI-069452, AI-69450, AI-054907, AI-069428, AI-069439, AI-069467, AI-045008, AI-069495, AI-069415, AI-069556, AI-069484, AI-069424, AI-069532, AI-069419, AI-069471, AI-025859, AI-069418, AI-050409, AI-069423, AI-069501, AI-069502, AI-069511, AI-069434, AI-069465, AI-069471, AI-069494, AI-069424, AI-069472, AI-069501, AI-069470, AI-046376, AI-069511, AI-072626, AI-069472, AI-038858, AI-069472, AI-069472, AI-069471, AI-027661, AI-027661, AI-034853, AI-069472, AI-069447, AI-032782, AI-027658, AI-27666, AI-027661, AI-058740, AI-046370, AI-069423, AI-069470, AI-069511, AI-069470, AI-069494, and by the following grants from the National Center for Research Resources (NCRR): RR-00051, RR-00046, RR-025747, RR-025777, RR-024160, RR-024996, RR-024156, RR024160, and RR-024160. Study drugs were provided by Bristol-Myers Squibb Co. and GlaxoSmithKline, Inc.

Grant support included:

The project described was supported by Award Number U01AI068636 from the National Institute of Allergy and Infectious Diseases (NIAID) and supported by National Institute of Mental Health (NIMH), National Institute of Dental and Craniofacial Research (NIDCR). The content is solely the responsibility of the authors and does not necessarily represent the official views of the National Institute of Allergy and Infectious Diseases or the National Institutes of Health. This work was supported by the AIDS Clinical Trials Group funded by the National Institute of Allergy and Infectious Diseases (AI-068636, AI-038858, AI-068634, AI-038855).

Grant support included AI-069439, RR-024975, AI-054999 (DWH), AI-077505 (HJR), BRSACURE-06-00140-T001 (GDM), and AI-069434 (ACC).

Clinical Research Sites that participated in ACTG protocol A5202, and collected DNA under protocol A5128, were supported by the following grants from NIAID: AI-069477, AI-027675, AI-073961, AI-069474, AI-069432, AI-069513, AI-069423, AI-050410, AI-069452, AI-69450, AI-054907, AI-069428, AI-069439, AI-069467, AI-045008, AI-069495, AI-069415, AI-069556, AI-069484, AI-069424, AI-069532, AI-069419, AI-069471, AI-025859, AI-069418, AI-050409, AI-069423, AI-069501, AI-069502, AI-069511, AI-069434, AI-069465, AI-069471, AI-069494, AI-069424, AI-069472, AI-069501, AI-069470, AI-046376, AI-069511, AI-072626, AI-069472, AI-038858, AI-069472, AI-069472, AI-069471, AI-027661, AI-027661, AI-034853, AI-069472, AI-069447, AI-032782, AI-027658, AI-27666, AI-027661, AI-058740, AI-046370, AI-069423, AI-069470, AI-069511, AI-069470, AI-069494, and by the following grants from the National Center for Research Resources (NCRR): RR-00051, RR-00046, RR-025747, RR-025777, RR-024160, RR-024996, RR-024156, RR024160, and RR-024160. Study drugs were provided by Abbott Laboratories, Bristol-Myers Squibb Co. and GlaxoSmithKline, Inc.

David W. Haas has been principal investigator on a research grant to Vanderbilt University from Merck, and has been a consultant to Merck.

Eric S. Daar has received research grants from Abbott, Bristol Myers Squibb, Merck, Pfizer, ViiV, and Gilead, and has been a consultant/advisor to Bristol Myers Squibb, Gilead, Janssen, Merck, and ViiV.

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Molina JM, Andrade-Villanueva J, Echevarria J, et al. Once-daily atazanavir/ritonavir versus twice-daily lopinavir/ritonavir, each in combination with tenofovir and emtricitabine, for management of antiretroviral-naive HIV-1-infected patients: 48 week efficacy and safety results of the CASTLE study. Lancet. 2008;372:646–55. doi: 10.1016/S0140-6736(08)61081-8.  [DOI](https://doi.org/10.1016/S0140-6736(08)61081-8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18722869/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Once-daily%20atazanavir/ritonavir%20versus%20twice-daily%20lopinavir/ritonavir,%20each%20in%20combination%20with%20tenofovir%20and%20emtricitabine,%20for%20management%20of%20antiretroviral-naive%20HIV-1-infected%20patients:%2048%20week%20efficacy%20and%20safety%20results%20of%20the%20CASTLE%20study.&author=JM%20Molina&author=J%20Andrade-Villanueva&author=J%20Echevarria&volume=372&publication_year=2008&pages=646-55&pmid=18722869&doi=10.1016/S0140-6736(08)61081-8&)

2. Gatell J, Salmon-Ceron D, Lazzarin A, et al. Efficacy and safety of atazanavir-based highly active antiretroviral therapy in patients with virologic suppression switched from a stable, boosted or unboosted protease inhibitor treatment regimen: the SWAN Study (AI424-097) 48-week results. Clin Infect Dis. 2007;44:1484–92. doi: 10.1086/517497.  [DOI](https://doi.org/10.1086/517497) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17479947/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Infect%20Dis&title=Efficacy%20and%20safety%20of%20atazanavir-based%20highly%20active%20antiretroviral%20therapy%20in%20patients%20with%20virologic%20suppression%20switched%20from%20a%20stable,%20boosted%20or%20unboosted%20protease%20inhibitor%20treatment%20regimen:%20the%20SWAN%20Study%20(AI424-097)%2048-week%20results.&author=J%20Gatell&author=D%20Salmon-Ceron&author=A%20Lazzarin&volume=44&publication_year=2007&pages=1484-92&pmid=17479947&doi=10.1086/517497&)

3. Daar ES, Tierney C, Fischl MA, et al. Atazanavir plus ritonavir or efavirenz as part of a 3-drug regimen for initial treatment of HIV-1. Ann Intern Med. 2011;154:445–56. doi: 10.1059/0003-4819-154-7-201104050-00316.  [DOI](https://doi.org/10.1059/0003-4819-154-7-201104050-00316) | [PMC free article](/articles/PMC3430716/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21320923/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Intern%20Med&title=Atazanavir%20plus%20ritonavir%20or%20efavirenz%20as%20part%20of%20a%203-drug%20regimen%20for%20initial%20treatment%20of%20HIV-1.&author=ES%20Daar&author=C%20Tierney&author=MA%20Fischl&volume=154&publication_year=2011&pages=445-56&pmid=21320923&doi=10.1059/0003-4819-154-7-201104050-00316&)

4. Panel on Antiretroviral Guidelines for Adults and Adolescents . Guidelines for the use of antiretroviral agents in HIV-1-infected adults and adolescents. Department of Health and Human Services; [August 15, 2013]. Available at: http://aidsinfo.nih.gov/contentfiles/lvguidelines/adultandadolescentgl.pdf. Last updated February 12, 2013.  [http://aidsinfo.nih.gov/contentfiles/lvguidelines/adultandadolescentgl.pdf](http://aidsinfo.nih.gov/contentfiles/lvguidelines/adultandadolescentgl.pdf) | [Google Scholar](https://scholar.google.com/scholar_lookup?title=Guidelines%20for%20the%20use%20of%20antiretroviral%20agents%20in%20HIV-1-infected%20adults%20and%20adolescents&)

5. Zhang D, Chando TJ, Everett DW, Patten CJ, Dehal SS, Humphreys WG. In vitro inhibition of UDP glucuronosyltransferases by atazanavir and other HIV protease inhibitors and the relationship of this property to in vivo bilirubin glucuronidation. Drug Metab Dispos. 2005;33:1729–39. doi: 10.1124/dmd.105.005447.  [DOI](https://doi.org/10.1124/dmd.105.005447) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16118329/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=In%20vitro%20inhibition%20of%20UDP%20glucuronosyltransferases%20by%20atazanavir%20and%20other%20HIV%20protease%20inhibitors%20and%20the%20relationship%20of%20this%20property%20to%20in%20vivo%20bilirubin%20glucuronidation.&author=D%20Zhang&author=TJ%20Chando&author=DW%20Everett&author=CJ%20Patten&author=SS%20Dehal&volume=33&publication_year=2005&pages=1729-39&pmid=16118329&doi=10.1124/dmd.105.005447&)

6. Malan DR, Krantz E, David N, Wirtz V, Hammond J, McGrath D. Efficacy and safety of atazanavir, with or without ritonavir, as part of once-daily highly active antiretroviral therapy regimens in antiretroviral-naive patients. JAIDS. 2008;47:161–7. doi: 10.1097/QAI.0b013e31815ace6a.  [DOI](https://doi.org/10.1097/QAI.0b013e31815ace6a) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17971713/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAIDS&title=Efficacy%20and%20safety%20of%20atazanavir,%20with%20or%20without%20ritonavir,%20as%20part%20of%20once-daily%20highly%20active%20antiretroviral%20therapy%20regimens%20in%20antiretroviral-naive%20patients.&author=DR%20Malan&author=E%20Krantz&author=N%20David&author=V%20Wirtz&author=J%20Hammond&volume=47&publication_year=2008&pages=161-7&pmid=17971713&doi=10.1097/QAI.0b013e31815ace6a&)

7. Cleijsen RM, van de Ende ME, Kroon FP, et al. Therapeutic drug monitoring of the HIV protease inhibitor atazanavir in clinical practice. J Antimicrob Chemother. 2007;60:897–900. doi: 10.1093/jac/dkm298.  [DOI](https://doi.org/10.1093/jac/dkm298) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17704117/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Antimicrob%20Chemother&title=Therapeutic%20drug%20monitoring%20of%20the%20HIV%20protease%20inhibitor%20atazanavir%20in%20clinical%20practice.&author=RM%20Cleijsen&author=ME%20van%20de%20Ende&author=FP%20Kroon&volume=60&publication_year=2007&pages=897-900&pmid=17704117&doi=10.1093/jac/dkm298&)

8. Torti C, Lapadula G, Antinori A, et al. Hyperbilirubinemia during atazanavir treatment in 2,404 patients in the Italian atazanavir expanded access program and MASTER Cohorts. Infection. 2009;37:244–9. doi: 10.1007/s15010-008-8010-6.  [DOI](https://doi.org/10.1007/s15010-008-8010-6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19471856/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Infection&title=Hyperbilirubinemia%20during%20atazanavir%20treatment%20in%202,404%20patients%20in%20the%20Italian%20atazanavir%20expanded%20access%20program%20and%20MASTER%20Cohorts.&author=C%20Torti&author=G%20Lapadula&author=A%20Antinori&volume=37&publication_year=2009&pages=244-9&pmid=19471856&doi=10.1007/s15010-008-8010-6&)

9. Puls RL, Srasuebkul P, Petoumenos K, et al. Efavirenz versus boosted atazanavir or zidovudine and abacavir in antiretroviral treatment-naive, HIV-infected subjects: week 48 data from the Altair study. Clin Infect Dis. 2010;51:855–64. doi: 10.1086/656363.  [DOI](https://doi.org/10.1086/656363) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20735258/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Infect%20Dis&title=Efavirenz%20versus%20boosted%20atazanavir%20or%20zidovudine%20and%20abacavir%20in%20antiretroviral%20treatment-naive,%20HIV-infected%20subjects:%20week%2048%20data%20from%20the%20Altair%20study.&author=RL%20Puls&author=P%20Srasuebkul&author=K%20Petoumenos&volume=51&publication_year=2010&pages=855-64&pmid=20735258&doi=10.1086/656363&)

10. McDonald C, Uy J, Hu W, et al. Clinical significance of hyperbilirubinemia among HIV-1-infected patients treated with atazanavir/ritonavir through 96 weeks in the CASTLE study. AIDS Patient Care STDS. 2012;26:259–64. doi: 10.1089/apc.2011.0092.  [DOI](https://doi.org/10.1089/apc.2011.0092) | [PMC free article](/articles/PMC3335103/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22404426/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=AIDS%20Patient%20Care%20STDS&title=Clinical%20significance%20of%20hyperbilirubinemia%20among%20HIV-1-infected%20patients%20treated%20with%20atazanavir/ritonavir%20through%2096%20weeks%20in%20the%20CASTLE%20study.&author=C%20McDonald&author=J%20Uy&author=W%20Hu&volume=26&publication_year=2012&pages=259-64&pmid=22404426&doi=10.1089/apc.2011.0092&)

11. Gandhi M, Ameli N, Bacchetti P, et al. Atazanavir concentration in hair is the strongest predictor of outcomes on antiretroviral therapy. Clin Infect Dis. 2011;52:1267–75. doi: 10.1093/cid/cir131.  [DOI](https://doi.org/10.1093/cid/cir131) | [PMC free article](/articles/PMC3079399/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21507924/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Infect%20Dis&title=Atazanavir%20concentration%20in%20hair%20is%20the%20strongest%20predictor%20of%20outcomes%20on%20antiretroviral%20therapy.&author=M%20Gandhi&author=N%20Ameli&author=P%20Bacchetti&volume=52&publication_year=2011&pages=1267-75&pmid=21507924&doi=10.1093/cid/cir131&)

12. Reyataz Package Insert. Revised March 2012 Bristol-Myers Squibb.; Princeton, NJ:  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Reyataz%20Package%20Insert&)

13. Janneh O, Jones E, Chandler B, Owen A, Khoo SH. Inhibition of P-glycoprotein and multidrug resistance-associated proteins modulates the intracellular concentration of lopinavir in cultured CD4 T cells and primary human lymphocytes. J Antimicrob Chemother. 2007;60:987–93. doi: 10.1093/jac/dkm353.  [DOI](https://doi.org/10.1093/jac/dkm353) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17890284/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Antimicrob%20Chemother&title=Inhibition%20of%20P-glycoprotein%20and%20multidrug%20resistance-associated%20proteins%20modulates%20the%20intracellular%20concentration%20of%20lopinavir%20in%20cultured%20CD4%20T%20cells%20and%20primary%20human%20lymphocytes.&author=O%20Janneh&author=E%20Jones&author=B%20Chandler&author=A%20Owen&author=SH%20Khoo&volume=60&publication_year=2007&pages=987-93&pmid=17890284&doi=10.1093/jac/dkm353&)

14. Rodriguez Novoa S, Barreiro P, Rendon A, et al. Plasma levels of atazanavir and the risk of hyperbilirubinemia are predicted by the 3435C-->T polymorphism at the multidrug resistance gene 1. Clin Infect Dis. 2006;42:291–5. doi: 10.1086/499056.  [DOI](https://doi.org/10.1086/499056) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16355344/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Infect%20Dis&title=Plasma%20levels%20of%20atazanavir%20and%20the%20risk%20of%20hyperbilirubinemia%20are%20predicted%20by%20the%203435C-->T%20polymorphism%20at%20the%20multidrug%20resistance%20gene%201.&author=S%20Rodriguez%20Novoa&author=P%20Barreiro&author=A%20Rendon&volume=42&publication_year=2006&pages=291-5&pmid=16355344&doi=10.1086/499056&)

15. Rodriguez-Novoa S, Martin-Carbonero L, Barreiro P, et al. Genetic factors influencing atazanavir plasma concentrations and the risk of severe hyperbilirubinemia. AIDS. 2007;21:41–6. doi: 10.1097/QAD.0b013e328011d7c1.  [DOI](https://doi.org/10.1097/QAD.0b013e328011d7c1) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17148966/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=AIDS&title=Genetic%20factors%20influencing%20atazanavir%20plasma%20concentrations%20and%20the%20risk%20of%20severe%20hyperbilirubinemia.&author=S%20Rodriguez-Novoa&author=L%20Martin-Carbonero&author=P%20Barreiro&volume=21&publication_year=2007&pages=41-6&pmid=17148966&doi=10.1097/QAD.0b013e328011d7c1&)

16. Kile DA, MaWhinney S, Aquilante CL, Rower JE, Castillo-Mancilla JR, Anderson PL. A population pharmacokinetic-pharmacogenetic analysis of atazanavir. AIDS Res Human Retrovir. 2012;28:1227–34. doi: 10.1089/aid.2011.0378.  [DOI](https://doi.org/10.1089/aid.2011.0378) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22394315/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=AIDS%20Res%20Human%20Retrovir&title=A%20population%20pharmacokinetic-pharmacogenetic%20analysis%20of%20atazanavir.&author=DA%20Kile&author=S%20MaWhinney&author=CL%20Aquilante&author=JE%20Rower&author=JR%20Castillo-Mancilla&volume=28&publication_year=2012&pages=1227-34&pmid=22394315&doi=10.1089/aid.2011.0378&)

17. Anderson PL, Aquilante CL, Gardner EM, et al. Atazanavir pharmacokinetics in genetically determined CYP3A5 expressors versus non-expressors. J Antimicrob Chemother. 2009;64:1071–9. doi: 10.1093/jac/dkp317.  [DOI](https://doi.org/10.1093/jac/dkp317) | [PMC free article](/articles/PMC2760462/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19710077/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Antimicrob%20Chemother&title=Atazanavir%20pharmacokinetics%20in%20genetically%20determined%20CYP3A5%20expressors%20versus%20non-expressors.&author=PL%20Anderson&author=CL%20Aquilante&author=EM%20Gardner&volume=64&publication_year=2009&pages=1071-9&pmid=19710077&doi=10.1093/jac/dkp317&)

18. Wempe MF, Anderson PL. Atazanavir metabolism according to CYP3A5 status: an in vitroin vivo assessment. Drug Metab Dispos. 2011;39:522–7. doi: 10.1124/dmd.110.036178.  [DOI](https://doi.org/10.1124/dmd.110.036178) | [PMC free article](/articles/PMC3061561/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21148251/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Atazanavir%20metabolism%20according%20to%20CYP3A5%20status:%20an%20in%20vitroin%20vivo%20assessment.&author=MF%20Wempe&author=PL%20Anderson&volume=39&publication_year=2011&pages=522-7&pmid=21148251&doi=10.1124/dmd.110.036178&)

19. Siccardi M, D'Avolio A, Baietto L, et al. Association of a single-nucleotide polymorphism in the pregnane X receptor (PXR 63396C-->T) with reduced concentrations of unboosted atazanavir. Clin Infect Dis. 2008;47:1222–5. doi: 10.1086/592304.  [DOI](https://doi.org/10.1086/592304) | [PMC free article](/articles/PMC3672984/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18831695/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Infect%20Dis&title=Association%20of%20a%20single-nucleotide%20polymorphism%20in%20the%20pregnane%20X%20receptor%20(PXR%2063396C-->T)%20with%20reduced%20concentrations%20of%20unboosted%20atazanavir.&author=M%20Siccardi&author=A%20D'Avolio&author=L%20Baietto&volume=47&publication_year=2008&pages=1222-5&pmid=18831695&doi=10.1086/592304&)

20. Mirochnick M, Best BM, Stek AM, et al. Atazanavir pharmacokinetics with and without tenofovir during pregnancy. JAIDS. 2011;56:412–9. doi: 10.1097/QAI.0b013e31820fd093.  [DOI](https://doi.org/10.1097/QAI.0b013e31820fd093) | [PMC free article](/articles/PMC3125419/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21283017/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAIDS&title=Atazanavir%20pharmacokinetics%20with%20and%20without%20tenofovir%20during%20pregnancy.&author=M%20Mirochnick&author=BM%20Best&author=AM%20Stek&volume=56&publication_year=2011&pages=412-9&pmid=21283017&doi=10.1097/QAI.0b013e31820fd093&)

21. Taburet AM, Piketty C, Chazallon C, et al. Interactions between atazanavir-ritonavir and tenofovir in heavily pretreated human immunodeficiency virus-infected patients. Antimicrob Agents Chemother. 2004;48:2091–6. doi: 10.1128/AAC.48.6.2091-2096.2004.  [DOI](https://doi.org/10.1128/AAC.48.6.2091-2096.2004) | [PMC free article](/articles/PMC415627/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15155205/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Antimicrob%20Agents%20Chemother&title=Interactions%20between%20atazanavir-ritonavir%20and%20tenofovir%20in%20heavily%20pretreated%20human%20immunodeficiency%20virus-infected%20patients.&author=AM%20Taburet&author=C%20Piketty&author=C%20Chazallon&volume=48&publication_year=2004&pages=2091-6&pmid=15155205&doi=10.1128/AAC.48.6.2091-2096.2004&)

22. Dailly E, Tribut O, Tattevin P, et al. Influence of tenofovir, nevirapine and efavirenz on ritonavir-boosted atazanavir pharmacokinetics in HIV-infected patients. Eur J Clin Pharmacol. 2006;62:523–6. doi: 10.1007/s00228-006-0122-2.  [DOI](https://doi.org/10.1007/s00228-006-0122-2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16763827/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Clin%20Pharmacol&title=Influence%20of%20tenofovir,%20nevirapine%20and%20efavirenz%20on%20ritonavir-boosted%20atazanavir%20pharmacokinetics%20in%20HIV-infected%20patients.&author=E%20Dailly&author=O%20Tribut&author=P%20Tattevin&volume=62&publication_year=2006&pages=523-6&pmid=16763827&doi=10.1007/s00228-006-0122-2&)

23. Le Tiec C, Barrail A, Goujard C, Taburet AM. Clinical pharmacokinetics and summary of efficacy and tolerability of atazanavir. Clin Pharmacokinet. 2005;44:1035–50. doi: 10.2165/00003088-200544100-00003.  [DOI](https://doi.org/10.2165/00003088-200544100-00003) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16176117/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacokinet&title=Clinical%20pharmacokinetics%20and%20summary%20of%20efficacy%20and%20tolerability%20of%20atazanavir.&author=C%20Le%20Tiec&author=A%20Barrail&author=C%20Goujard&author=AM%20Taburet&volume=44&publication_year=2005&pages=1035-50&pmid=16176117&doi=10.2165/00003088-200544100-00003&)

24. Colombo S, Buclin T, Cavassini M, et al. Population pharmacokinetics of atazanavir in patients with human immunodeficiency virus infection. Antimicrob Agents Chemother. 2006;50:3801–8. doi: 10.1128/AAC.00098-06.  [DOI](https://doi.org/10.1128/AAC.00098-06) | [PMC free article](/articles/PMC1635184/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16940065/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Antimicrob%20Agents%20Chemother&title=Population%20pharmacokinetics%20of%20atazanavir%20in%20patients%20with%20human%20immunodeficiency%20virus%20infection.&author=S%20Colombo&author=T%20Buclin&author=M%20Cavassini&volume=50&publication_year=2006&pages=3801-8&pmid=16940065&doi=10.1128/AAC.00098-06&)

25. Bosma PJ, Chowdhury JR, Bakker C, et al. The genetic basis of the reduced expression of bilirubin UDP-glucuronosyltransferase 1 in Gilbert's syndrome. New Engl J Med. 1995;333:1171–5. doi: 10.1056/NEJM199511023331802.  [DOI](https://doi.org/10.1056/NEJM199511023331802) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7565971/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=New%20Engl%20J%20Med&title=The%20genetic%20basis%20of%20the%20reduced%20expression%20of%20bilirubin%20UDP-glucuronosyltransferase%201%20in%20Gilbert's%20syndrome.&author=PJ%20Bosma&author=JR%20Chowdhury&author=C%20Bakker&volume=333&publication_year=1995&pages=1171-5&pmid=7565971&doi=10.1056/NEJM199511023331802&)

26. Monaghan G, Ryan M, Seddon R, Hume R, Burchell B. Genetic variation in bilirubin UPD-glucuronosyltransferase gene promoter and Gilbert's syndrome. Lancet. 1996;347:578–81. doi: 10.1016/s0140-6736(96)91273-8.  [DOI](https://doi.org/10.1016/s0140-6736(96)91273-8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8596320/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Genetic%20variation%20in%20bilirubin%20UPD-glucuronosyltransferase%20gene%20promoter%20and%20Gilbert's%20syndrome.&author=G%20Monaghan&author=M%20Ryan&author=R%20Seddon&author=R%20Hume&author=B%20Burchell&volume=347&publication_year=1996&pages=578-81&pmid=8596320&doi=10.1016/s0140-6736(96)91273-8&)

27. Rotger M, Taffe P, Bleiber G, et al. Gilbert syndrome and the development of antiretroviral therapy-associated hyperbilirubinemia. J Infect Dis. 2005;192:1381–6. doi: 10.1086/466531.  [DOI](https://doi.org/10.1086/466531) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16170755/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Infect%20Dis&title=Gilbert%20syndrome%20and%20the%20development%20of%20antiretroviral%20therapy-associated%20hyperbilirubinemia.&author=M%20Rotger&author=P%20Taffe&author=G%20Bleiber&volume=192&publication_year=2005&pages=1381-6&pmid=16170755&doi=10.1086/466531&)

28. Ribaudo HJ, Daar ES, Tierney C, et al. Impact of UGT1A1 Gilbert variant on discontinuation of ritonavir-boosted atazanavir in AIDS Clinical Trials Group Study A5202. J Infect Dis. 2013;207:420–5. doi: 10.1093/infdis/jis690.  [DOI](https://doi.org/10.1093/infdis/jis690) | [PMC free article](/articles/PMC3537445/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23148286/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Infect%20Dis&title=Impact%20of%20UGT1A1%20Gilbert%20variant%20on%20discontinuation%20of%20ritonavir-boosted%20atazanavir%20in%20AIDS%20Clinical%20Trials%20Group%20Study%20A5202.&author=HJ%20Ribaudo&author=ES%20Daar&author=C%20Tierney&volume=207&publication_year=2013&pages=420-5&pmid=23148286&doi=10.1093/infdis/jis690&)

29. Lubomirov R, Colombo S, di Iulio J, et al. Association of pharmacogenetic markers with premature discontinuation of first-line anti-HIV therapy: an observational cohort study. J Infect Dis. 2011;203:246–57. doi: 10.1093/infdis/jiq043.  [DOI](https://doi.org/10.1093/infdis/jiq043) | [PMC free article](/articles/PMC3071070/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21288825/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Infect%20Dis&title=Association%20of%20pharmacogenetic%20markers%20with%20premature%20discontinuation%20of%20first-line%20anti-HIV%20therapy:%20an%20observational%20cohort%20study.&author=R%20Lubomirov&author=S%20Colombo&author=J%20di%20Iulio&volume=203&publication_year=2011&pages=246-57&pmid=21288825&doi=10.1093/infdis/jiq043&)

30. Sax PE, Tierney C, Collier AC, et al. Abacavir-lamivudine versus tenofovir-emtricitabine for initial HIV-1 therapy. N Engl J Med. 2009;361:2230–40. doi: 10.1056/NEJMoa0906768.  [DOI](https://doi.org/10.1056/NEJMoa0906768) | [PMC free article](/articles/PMC2800041/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19952143/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Abacavir-lamivudine%20versus%20tenofovir-emtricitabine%20for%20initial%20HIV-1%20therapy.&author=PE%20Sax&author=C%20Tierney&author=AC%20Collier&volume=361&publication_year=2009&pages=2230-40&pmid=19952143&doi=10.1056/NEJMoa0906768&)

31. Haas DW, Wilkinson GR, Kuritzkes DR, et al. A multi-investigator/institutional DNA bank for AIDS-related human genetic studies: AACTG Protocol A5128. HIV Clin Trials. 2003;4:287–300. doi: 10.1310/MUQC-QXBC-8118-BPM5.  [DOI](https://doi.org/10.1310/MUQC-QXBC-8118-BPM5) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14583845/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=HIV%20Clin%20Trials&title=A%20multi-investigator/institutional%20DNA%20bank%20for%20AIDS-related%20human%20genetic%20studies:%20AACTG%20Protocol%20A5128.&author=DW%20Haas&author=GR%20Wilkinson&author=DR%20Kuritzkes&volume=4&publication_year=2003&pages=287-300&pmid=14583845&doi=10.1310/MUQC-QXBC-8118-BPM5&)

32. Pereyra F, Jia X, McLaren PJ, et al. The major genetic determinants of HIV-1 control affect HLA class I peptide presentation. Science. 2010;330:1551–7. doi: 10.1126/science.1195271.  [DOI](https://doi.org/10.1126/science.1195271) | [PMC free article](/articles/PMC3235490/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21051598/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=The%20major%20genetic%20determinants%20of%20HIV-1%20control%20affect%20HLA%20class%20I%20peptide%20presentation.&author=F%20Pereyra&author=X%20Jia&author=PJ%20McLaren&volume=330&publication_year=2010&pages=1551-7&pmid=21051598&doi=10.1126/science.1195271&)

33. Purcell S, Neale B, Todd-Brown K, et al. PLINK: a tool set for whole-genome association and population-based linkage analyses. Am J Hum Genet. 2007;81:559–75. doi: 10.1086/519795.  [DOI](https://doi.org/10.1086/519795) | [PMC free article](/articles/PMC1950838/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17701901/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=PLINK:%20a%20tool%20set%20for%20whole-genome%20association%20and%20population-based%20linkage%20analyses.&author=S%20Purcell&author=B%20Neale&author=K%20Todd-Brown&volume=81&publication_year=2007&pages=559-75&pmid=17701901&doi=10.1086/519795&)

34. Baudhuin LM, Highsmith WE, Skierka J, Holtegaard L, Moore BE, O'Kane DJ. Comparison of three methods for genotyping the UGT1A1 (TA)n repeat polymorphism. Clin Biochem. 2007;40:710–7. doi: 10.1016/j.clinbiochem.2007.03.007.  [DOI](https://doi.org/10.1016/j.clinbiochem.2007.03.007) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17459361/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Biochem&title=Comparison%20of%20three%20methods%20for%20genotyping%20the%20UGT1A1%20(TA)n%20repeat%20polymorphism.&author=LM%20Baudhuin&author=WE%20Highsmith&author=J%20Skierka&author=L%20Holtegaard&author=BE%20Moore&volume=40&publication_year=2007&pages=710-7&pmid=17459361&doi=10.1016/j.clinbiochem.2007.03.007&)

35. Price AL, Patterson NJ, Plenge RM, Weinblatt ME, Shadick NA, Reich D. Principal components analysis corrects for stratification in genome-wide association studies. Nat Genet. 2006;38:904–9. doi: 10.1038/ng1847.  [DOI](https://doi.org/10.1038/ng1847) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16862161/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Principal%20components%20analysis%20corrects%20for%20stratification%20in%20genome-wide%20association%20studies.&author=AL%20Price&author=NJ%20Patterson&author=RM%20Plenge&author=ME%20Weinblatt&author=NA%20Shadick&volume=38&publication_year=2006&pages=904-9&pmid=16862161&doi=10.1038/ng1847&)

36. McDonagh EM, Whirl-Carrillo M, Garten Y, Altman RB, Klein TE. From pharmacogenomic knowledge acquisition to clinical applications: the PharmGKB as a clinical pharmacogenomic biomarker resource. Biomark Med. 2011;5:795–806. doi: 10.2217/bmm.11.94.  [DOI](https://doi.org/10.2217/bmm.11.94) | [PMC free article](/articles/PMC3339046/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22103613/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biomark%20Med&title=From%20pharmacogenomic%20knowledge%20acquisition%20to%20clinical%20applications:%20the%20PharmGKB%20as%20a%20clinical%20pharmacogenomic%20biomarker%20resource.&author=EM%20McDonagh&author=M%20Whirl-Carrillo&author=Y%20Garten&author=RB%20Altman&author=TE%20Klein&volume=5&publication_year=2011&pages=795-806&pmid=22103613&doi=10.2217/bmm.11.94&)

37. Schipani A, Siccardi M, D'Avolio A, et al. Population pharmacokinetic modeling of the association between 63396C->T pregnane X receptor polymorphism and unboosted atazanavir clearance. Antimicrob Agents Chemother. 2010;54:5242–50. doi: 10.1128/AAC.00781-10.  [DOI](https://doi.org/10.1128/AAC.00781-10) | [PMC free article](/articles/PMC2981241/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20921307/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Antimicrob%20Agents%20Chemother&title=Population%20pharmacokinetic%20modeling%20of%20the%20association%20between%2063396C->T%20pregnane%20X%20receptor%20polymorphism%20and%20unboosted%20atazanavir%20clearance.&author=A%20Schipani&author=M%20Siccardi&author=A%20D'Avolio&volume=54&publication_year=2010&pages=5242-50&pmid=20921307&doi=10.1128/AAC.00781-10&)

38. [August 15 2013];R: A language and environment for statistical computing. Available at: http://www.R-project.org.  [http://www.R-project.org](http://www.R-project.org)

39. Hartkoorn RC, Kwan WS, Shallcross V, et al. HIV protease inhibitors are substrates for OATP1A2, OATP1B1 and OATP1B3 and lopinavir plasma concentrations are influenced by SLCO1B1 polymorphisms. Pharmacogenet Genomics. 2010;20:112–20. doi: 10.1097/FPC.0b013e328335b02d.  [DOI](https://doi.org/10.1097/FPC.0b013e328335b02d) | [PMC free article](/articles/PMC4859410/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20051929/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=HIV%20protease%20inhibitors%20are%20substrates%20for%20OATP1A2,%20OATP1B1%20and%20OATP1B3%20and%20lopinavir%20plasma%20concentrations%20are%20influenced%20by%20SLCO1B1%20polymorphisms.&author=RC%20Hartkoorn&author=WS%20Kwan&author=V%20Shallcross&volume=20&publication_year=2010&pages=112-20&pmid=20051929&doi=10.1097/FPC.0b013e328335b02d&)

40. Lubomirov R, di Iulio J, Fayet A, et al. ADME pharmacogenetics: investigation of the pharmacokinetics of the antiretroviral agent lopinavir coformulated with ritonavir. Pharmacogenet Genomics. 2010;20:217–30. doi: 10.1097/FPC.0b013e328336eee4.  [DOI](https://doi.org/10.1097/FPC.0b013e328336eee4) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20139798/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=ADME%20pharmacogenetics:%20investigation%20of%20the%20pharmacokinetics%20of%20the%20antiretroviral%20agent%20lopinavir%20coformulated%20with%20ritonavir.&author=R%20Lubomirov&author=J%20di%20Iulio&author=A%20Fayet&volume=20&publication_year=2010&pages=217-30&pmid=20139798&doi=10.1097/FPC.0b013e328336eee4&)

41. Zhang X, Tierney C, Albrecht M, et al. Discordant associations between SLCO1B1 521T-->C and plasma levels of ritonavir-boosted protease inhibitors in AIDS clinical trials group study A5146. Ther Drug Monit. 2013;35:209–16. doi: 10.1097/FTD.0b013e318280d0ad.  [DOI](https://doi.org/10.1097/FTD.0b013e318280d0ad) | [PMC free article](/articles/PMC3603284/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23503447/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ther%20Drug%20Monit&title=Discordant%20associations%20between%20SLCO1B1%20521T-->C%20and%20plasma%20levels%20of%20ritonavir-boosted%20protease%20inhibitors%20in%20AIDS%20clinical%20trials%20group%20study%20A5146.&author=X%20Zhang&author=C%20Tierney&author=M%20Albrecht&volume=35&publication_year=2013&pages=209-16&pmid=23503447&doi=10.1097/FTD.0b013e318280d0ad&)

42. Kohlrausch FB, de Cassia Estrela R, Barroso PF, Suarez-Kurtz G. The impact of SLCO1B1 polymorphisms on the plasma concentration of lopinavir and ritonavir in HIV-infected men. Br J Clin Pharmacol. 2010;69:95–8. doi: 10.1111/j.1365-2125.2009.03551.x.  [DOI](https://doi.org/10.1111/j.1365-2125.2009.03551.x) | [PMC free article](/articles/PMC2830602/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20078617/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Clin%20Pharmacol&title=The%20impact%20of%20SLCO1B1%20polymorphisms%20on%20the%20plasma%20concentration%20of%20lopinavir%20and%20ritonavir%20in%20HIV-infected%20men.&author=FB%20Kohlrausch&author=R%20de%20Cassia%20Estrela&author=PF%20Barroso&author=G%20Suarez-Kurtz&volume=69&publication_year=2010&pages=95-8&pmid=20078617&doi=10.1111/j.1365-2125.2009.03551.x&)

43. Rakhmanina NY, Neely MN, Van Schaik RH, et al. CYP3A5, ABCB1, and SLCO1B1 polymorphisms and pharmacokinetics and virologic outcome of lopinavir/ritonavir in HIV-infected children. Ther Drug Monit. 2011;33:417–24. doi: 10.1097/FTD.0b013e318225384f.  [DOI](https://doi.org/10.1097/FTD.0b013e318225384f) | [PMC free article](/articles/PMC3139013/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21743379/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ther%20Drug%20Monit&title=CYP3A5,%20ABCB1,%20and%20SLCO1B1%20polymorphisms%20and%20pharmacokinetics%20and%20virologic%20outcome%20of%20lopinavir/ritonavir%20in%20HIV-infected%20children.&author=NY%20Rakhmanina&author=MN%20Neely&author=RH%20Van%20Schaik&volume=33&publication_year=2011&pages=417-24&pmid=21743379&doi=10.1097/FTD.0b013e318225384f&)

44. Chen G, Ramos E, Adeyemo A, et al. UGT1A1 is a major locus influencing bilirubin levels in African Americans. Eur J Hum Genet. 2012;20:463–8. doi: 10.1038/ejhg.2011.206.  [DOI](https://doi.org/10.1038/ejhg.2011.206) | [PMC free article](/articles/PMC3306855/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22085899/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Hum%20Genet&title=UGT1A1%20is%20a%20major%20locus%20influencing%20bilirubin%20levels%20in%20African%20Americans.&author=G%20Chen&author=E%20Ramos&author=A%20Adeyemo&volume=20&publication_year=2012&pages=463-8&pmid=22085899&doi=10.1038/ejhg.2011.206&)

45. Sanna S, Busonero F, Maschio A, et al. Common variants in the SLCO1B3 locus are associated with bilirubin levels and unconjugated hyperbilirubinemia. Hum Mol Genet. 2009;18:2711–8. doi: 10.1093/hmg/ddp203.  [DOI](https://doi.org/10.1093/hmg/ddp203) | [PMC free article](/articles/PMC2701337/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19419973/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mol%20Genet&title=Common%20variants%20in%20the%20SLCO1B3%20locus%20are%20associated%20with%20bilirubin%20levels%20and%20unconjugated%20hyperbilirubinemia.&author=S%20Sanna&author=F%20Busonero&author=A%20Maschio&volume=18&publication_year=2009&pages=2711-8&pmid=19419973&doi=10.1093/hmg/ddp203&)

46. Johnson AD, Kavousi M, Smith AV, et al. Genome-wide association meta-analysis for total serum bilirubin levels. Hum Mol Genet. 2009;18:2700–10. doi: 10.1093/hmg/ddp202.  [DOI](https://doi.org/10.1093/hmg/ddp202) | [PMC free article](/articles/PMC2701336/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19414484/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mol%20Genet&title=Genome-wide%20association%20meta-analysis%20for%20total%20serum%20bilirubin%20levels.&author=AD%20Johnson&author=M%20Kavousi&author=AV%20Smith&volume=18&publication_year=2009&pages=2700-10&pmid=19414484&doi=10.1093/hmg/ddp202&)

47. Castillo-Mancilla JAC, Campbell T, Haas DW, Smeaton L, Wempe MF, Firnhaber C, LaRosa A, Andrade A, Baheti G, Fletcher CV, Anderson PL. Effect of CYP3A5 polymorphisms on the metabolism of atazanavir in HIV-infected patients: a sub-study of AIDS Clinical Trials Groups A5175 (NWCS 342). Presented at 13th International Workshop on Clinical Pharmacology of HIV Therapy (Barcelona) 2012 Abstract number 22.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Presented%20at%2013th%20International%20Workshop%20on%20Clinical%20Pharmacology%20of%20HIV%20Therapy%20(Barcelona)&title=Effect%20of%20CYP3A5%20polymorphisms%20on%20the%20metabolism%20of%20atazanavir%20in%20HIV-infected%20patients:%20a%20sub-study%20of%20AIDS%20Clinical%20Trials%20Groups%20A5175%20(NWCS%20342).&author=JAC%20Castillo-Mancilla&author=T%20Campbell&author=DW%20Haas&author=L%20Smeaton&author=MF%20Wempe&publication_year=2012&)

48. Nelson MR, Bacanu SA, Mosteller M, et al. Genome-wide approaches to identify pharmacogenetic contributions to adverse drug reactions. Pharmacogenomics J. 2009;9:23–33. doi: 10.1038/tpj.2008.4.  [DOI](https://doi.org/10.1038/tpj.2008.4) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18301416/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=Genome-wide%20approaches%20to%20identify%20pharmacogenetic%20contributions%20to%20adverse%20drug%20reactions.&author=MR%20Nelson&author=SA%20Bacanu&author=M%20Mosteller&volume=9&publication_year=2009&pages=23-33&pmid=18301416&doi=10.1038/tpj.2008.4&)

49. Link E, Parish S, Armitage J, et al. SLCO1B1 variants and statin-induced myopathy--a genomewide study. N Engl J Med. 2008;359:789–99. doi: 10.1056/NEJMoa0801936.  [DOI](https://doi.org/10.1056/NEJMoa0801936) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18650507/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=SLCO1B1%20variants%20and%20statin-induced%20myopathy--a%20genomewide%20study.&author=E%20Link&author=S%20Parish&author=J%20Armitage&volume=359&publication_year=2008&pages=789-99&pmid=18650507&doi=10.1056/NEJMoa0801936&)

50. Chantarangsu S, Mushiroda T, Mahasirimongkol S, et al. Genome-wide association study identifies variations in 6p21.3 associated with nevirapine-induced rash. Clin Infect Dis. 2011;53:341–8. doi: 10.1093/cid/cir403.  [DOI](https://doi.org/10.1093/cid/cir403) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21810746/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Infect%20Dis&title=Genome-wide%20association%20study%20identifies%20variations%20in%206p21.3%20associated%20with%20nevirapine-induced%20rash.&author=S%20Chantarangsu&author=T%20Mushiroda&author=S%20Mahasirimongkol&volume=53&publication_year=2011&pages=341-8&pmid=21810746&doi=10.1093/cid/cir403&)
