# Pharmacogenetic meta-analysis of genome-wide association studies of LDL cholesterol response to statins

## Metadata
**Authors:** Iris Postmus, Stella Trompet, Harshal A Deshmukh, Michael R Barnes, Xiaohui Li, Helen R Warren, Daniel I Chasman, Kaixin Zhou, Benoit J Arsenault, Louise A Donnelly, Kerri L Wiggins, Christy L Avery, Paula Griffin, QiPing Feng, Kent D Taylor, Guo Li, Daniel S Evans, Albert V Smith, Catherine E de Keyser, Andrew D Johnson, Anton J M de Craen, David J Stott, Brendan M Buckley, Ian Ford, Rudi G J Westendorp, P Eline Slagboom, Naveed Sattar, Patricia B Munroe, Peter Sever, Neil Poulter, Alice Stanton, Denis C Shields, Eoin O’Brien, Sue Shaw-Hawkins, Y-D Ida Chen, Deborah A Nickerson, Joshua D Smith, Marie Pierre Dubé, S Matthijs Boekholdt, G Kees Hovingh, John J P Kastelein, Paul M McKeigue, John Betteridge, Andrew Neil, Paul N Durrington, Alex Doney, Fiona Carr, Andrew Morris, Mark I McCarthy, Leif Groop, Emma Ahlqvist, Welcome Trust Case Control Consortium, Joshua C Bis, Kenneth Rice, Nicholas L Smith, Thomas Lumley, Eric A Whitsel, Til Stürmer, Eric Boerwinkle, Julius S Ngwa, Christopher J O’Donnell, Ramachandran S Vasan, Wei-Qi Wei, Russell A Wilke, Ching-Ti Liu, Fangui Sun, Xiuqing Guo, Susan R Heckbert, Wendy Post, Nona Sotoodehnia, Alice M Arnold, Jeanette M Stafford, Jingzhong Ding, David M Herrington, Stephen B Kritchevsky, Gudny Eiriksdottir, Leonore J Launer, Tamara B Harris, Audrey Y Chu, Franco Giulianini, Jean G MacFadyen, Bryan J Barratt, Fredrik Nyberg, Bruno H Stricker, André G Uitterlinden, Albert Hofman, Fernando Rivadeneira, Valur Emilsson, Oscar H Franco, Paul M Ridker, Vilmundur Gudnason, Yongmei Liu, Joshua C Denny, Christie M Ballantyne, Jerome I Rotter, L Adrienne Cupples, Bruce M Psaty, Colin N A Palmer, Jean-Claude Tardif, Helen M Colhoun
**Journal:** Nature Communications
**Date:** 2014 Oct 28
**DOI:** [10.1038/ncomms6068](https://doi.org/10.1038/ncomms6068)
**PMID:** 25350695
**PMCID:** PMC4220464
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4220464/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC4220464/pdf/ncomms6068.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC4220464/pdf/ncomms6068.pdf)

## Abstract

Statins effectively lower LDL cholesterol levels in large studies and the observed interindividual response variability may be partially explained by genetic variation. Here we perform a pharmacogenetic meta-analysis of genome-wide association studies (GWAS) in studies addressing the LDL cholesterol response to statins, including up to 18,596 statin-treated subjects. We validate the most promising signals in a further 22,318 statin recipients and identify two loci, SORT1/CELSR2/PSRC1 and SLCO1B1, not previously identified in GWAS. Moreover, we confirm the previously described associations with APOE and LPA. Our findings advance the understanding of the pharmacogenetic architecture of statin response.

## Results

### First-stage meta-analysis

The GIST consortium includes 6 RCTs (*n*=8,421 statin recipients) and 10 observational studies (*n*=10,175 statin recipients) that participated in the first stage (see Methods; [Supplementary Tables 1 and 2](#S1); [Supplementary Notes 1 and 2](#S1)). To search for genetic variants associated with differential LDL-C response to statin therapy, each study independently performed a GWAS among statin users, using the difference between the natural log-transformed LDL-C levels on- and off-treatment as the response variable (see Methods).

The first-stage meta-analysis identified three loci, including 13 SNPs, that attained genome-wide significance (*P*<5 × 10^−8^) for association with LDL-C response to statin treatment ([Fig. 1](#f1); [Table 1](#t1)). The most significant association was for a SNP on chromosome 19, at *APOE* (rs445925, minor allele frequency (MAF)=0.098, *β*=−0.043, s.e.=0.005, *P*=1.58 × 10^−18^; [Fig. 2a](#f2)), indicating that carriers of the rs445925 SNP respond to statins with an additional 4.3% increase per allele in LDL-C lowering effect compared with non-carriers. The second strongest association was with a SNP at *LPA* on chromosome 6 (rs10455872, MAF=0.069, *β*=0.041, s.e.=0.006, *P*=1.95 × 10^−11^; [Fig. 2b](#f2)), indicating a 5.9% smaller LDL-C lowering per minor allele for carriers of the SNP compared with non-carriers. Associations at both loci have previously been described[7](#b7),[8](#b8). A third genome-wide significant association was found with a SNP at *RICTOR* on chromosome 5 (rs13166647, MAF=0.230, *β*=−0.253, s.e.=0.046, *P*=4.50 × 10^−8^), although genotypes for this SNP were only available in two studies within the first stage (*n*=2,144).

### Figure 1. Results of the GWAS meta-analysis.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/81ff/4220464/e9cc10745a6b/ncomms6068-f1.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4220464_ncomms6068-f1.jpg)

Manhattan plot presenting the −log10 P values from the combined meta-analysis (n=40,914) on LDL-C response after statin treatment. P values were generated using linear regression analysis.

### Table 1. Genome-wide significant associations in stage 1, stage 2 and combined meta-analysis.

| Chr | Position | Lead SNP | Gene | Coding allele | Noncoding allele | Phase | N | Frequency-coding allele | Beta* | s.e. | % Extra reduction† | P value |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| 1 | 109620053 | rs646776 | SORT1/CELSR2/PSRC1 | C | T | Stage 1 | 16,697 | 0.230 | −0.015 | 0.003 | 1.5 | 6.70 × 10−7 |
|   |   |   |   |   |   | Stage 2 | 21,902 | 0.216 | −0.010 | 0.003 | 1.0 | 2.43 × 10−4 |
|   |   |   |   |   |   | Combined | 38,599 |   | −0.013 | 0.002 | 1.3 | 1.05 × 10−9 |
| 6 | 160930108 | rs10455872 | LPA | G | A | Stage 1 | 12,981 | 0.069 | 0.041 | 0.006 | −4.1 | 1.95 × 10−11 |
|   |   |   |   |   |   | Stage 2 | 18,075 | 0.087 | 0.059 | 0.005 | −5.9 | 7.14 × 10−35 |
|   |   |   |   |   |   | Combined | 31,056 |   | 0.052 | 0.004 | −5.2 | 7.41 × 10−44 |
| 12 | 21260064 | rs2900478 | SLCO1B1 | A | T | Stage 1 | 16,749 | 0.165 | 0.016 | 0.003 | −1.6 | 2.26 × 10−6 |
|   |   |   |   |   |   | Stage 2 | 7,504 | 0.164 | 0.017 | 0.006 | −1.7 | 3.54 × 10−3 |
|   |   |   |   |   |   | Combined | 24,253 |   | 0.016 | 0.003 | −1.6 | 1.22 × 10−9 |
| 19 | 50107480 | rs445925 | APOE | A | G | Stage 1 | 13,909 | 0.098 | −0.043 | 0.005 | 4.3 | 1.58 × 10−18 |
|   |   |   |   |   |   | Stage 2 | 3,613 | 0.157 | −0.088 | 0.011 | 8.8 | 1.41 × 10−15 |
|   |   |   |   |   |   | Combined | 17,522 |   | −0.051 | 0.005 | 5.1 | 8.52 × 10−29 |

Table 1 Caption: Chr, chromosome; SNP, single nucleotide polymorphism. *Beta for difference between the natural log-transformed on- and off-treatment low-density lipoprotein cholesterol (LDL-C) levels adjusted for natural log-transformed off-treatment LDL-C-, age-, sex- and study-specific covariates. The beta reflects the fraction of differential LDL-C lowering in carriers versus non-carriers of the SNP; a negative beta indicates a better statin response (stronger LDL-C reduction), a positive beta a worse statin response. Betas and P values were generated using linear regression analysis. †This percentage reflects the % extra LDL-C lowering in carriers versus non-carriers of the SNP.

### Figure 2. Regional association plots of the genome-wide significant associations with LDL-C response after statin treatment.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/81ff/4220464/784775b9a440/ncomms6068-f2.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4220464_ncomms6068-f2.jpg)

The plots show the genome-wide significant associated loci in the combined meta-analysis (n=40,914), the APOE locus (a), the LPA locus (b), the SORT1/CELSR2/PSRC1 locus (c) and the SLCO1B1 locus (d) (generated using LocusZoom ( http://genome.sph.umich.edu/wiki/LocusZoom)). The colour of the SNPs is based on the LD with the lead SNP (shown in purple). The RefSeq genes in the region are shown in the lower panel. P values were generated using linear regression analysis.

### Second-stage meta-analysis

We selected 246 SNPs with *P* <5 × 10^−4^ from 158 loci for further investigation in three additional studies comprising up to 22,318 statin-treated subjects (see Methods; [Supplementary Tables 1 and 5](#S1); [Supplementary Note 3](#S1)). This second stage confirmed the genome-wide significant associations between variations within the *APOE* and *LPA* loci and LDL-C response, as observed in the first stage ([Table 1](#t1); [Supplementary Fig. 2](#S1); [Supplementary Table 5](#S1)). In addition, SNPs at two new loci with *P* values between 6.70 × 10^−7^ and 2.26 × 10^−6^ in the first phase were shown to be significantly associated with statin-induced LDL-C lowering after statin treatment in the total combined meta-analysis at a genome-wide level: *SORT1/CELSR2/PSRC1* (rs646776, *β*=−0.013, s.e.=0.002, *P*=1.05 × 10^−9^ and rs12740374, *β*=−0.013, s.e.=0.002, *P*=1.05 × 10^−9^; [Fig 2c](#f2)) and *SLCO1B1* (rs2900478, *β*=0.016, s.e.=0.003, *P*=1.22 × 10^−9^; [Fig 2d](#f2)), indicating an additional 1.5% increase per allele in LDL-C lowering effect for carriers of the *SORT1/CELSR2/PSRC1* SNP and a 1.6% smaller LDL-C lowering per minor allele for carriers of the *SLCO1B1* SNP.

The six next-ranked SNPs with *P* values just below 5 × 10^−8^ in the combined meta-analysis, including the two SNPs at *RICTOR* (rs13166647 and rs13172966), were selected for additional genotyping in the Scandinavian ASCOT participants (see Methods). None of these six SNPs reached genome-wide significance after this additional genotyping ([Supplementary Table 6](#S1)). Therefore, our overall genome-wide significant findings were the SNPs at *APOE*, *LPA*, *SORT1/CELSR2/PSRC1* and *SLCO1B1*.

### Subfraction analyses

To extend our results for the novel GWAS finding *SORT1/CELSR2/PSRC1*, we performed additional association analyses, using measurements of cholesterol levels in four LDL subfractions (large, medium, small and very small) from two of the trials in GIST, CAP and PRINCE ([Table 2](#t2); see Methods). The minor allele of *SORT1* rs646776 was associated with greater statin-induced reductions in levels of all LDL subfractions, and there was a nonsignificant trend for larger effect sizes and greater statistical significance for lowering of small and very small LDL ([Table 2](#t2)). In contrast, the *APOE* SNP associated with greater LDL-C response to statins (rs445925) showed a small and nonsignificant association with change in very small LDL ([Table 2](#t2)). For the minor allele of rs2900478 (*SLCO1B1*), the borderline significant association with smaller magnitude of LDL-C reduction showed a trend for preferential association with larger versus smaller LDL subfractions. The lack of association of rs10455872 (*LPA*) with changes in LDL subfractions is consistent with evidence discussed below that this locus affects levels of lipoprotein(a) (Lp(a)) and not LDL particles. Using generalized estimating equations, we tested the association of log change in each of the LDL subfractions with interactions of the four SNPs. For very small LDL, the association with the rs646776 minor allele was significantly different from that of the other minor alleles (*P*=0.03 after adjustment for multiple testing).

### Table 2. Associations of the minor alleles of rs646776, rs445925, rs2900478 and rs10455872 with changes in LDL-C and LDL subfractions in response to statin in the combined CAP and PRINCE studies.

| Change* | SORT1/CELSR2/PSRC1rs646776 (MAF 0.2) |  |  | APOE rs445925 (MAF 0.086) |  |  | SLCO1B1 rs2900478 (MAF 0.16) |  |  | LPA rs10455872 (MAF 0.056) |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
|   | Beta | s.e. | P value | Beta | s.e. | P value | Beta | s.e. | P value | Beta | s.e. | P value |
| LDL-C total | −0.023 | 0.008 | 0.003 | −0.046 | 0.018 | 0.008 | 0.010 | 0.005 | 0.04 | 0.032 | 0.019 | 0.09 |
| Large LDL-C | −0.028 | 0.014 | 0.042 | −0.075 | 0.029 | 0.009 | 0.02 | 0.008 | 0.01 | 0.036 | 0.031 | 0.23 |
| Medium LDL-C | −0.027 | 0.015 | 0.075 | −0.079 | 0.032 | 0.012 | 0.016 | 0.009 | 0.07 | 0.010 | 0.034 | 0.77 |
| Small LDL-C | −0.047 | 0.018 | 0.009 | −0.071 | 0.037 | 0.050 | 0.002 | 0.010 | 0.83 | −0.024 | 0.039 | 0.54 |
| Very small LDL-C | −0.034 | 0.009 | 0.00006 | −0.022 | 0.017 | 0.202 | 0.001 | 0.005 | 0.90 | 0.008 | 0.019 | 0.67 |

Table 2 Caption: LDL-C, low-density lipoprotein cholesterol; MAF, minor allele frequency. *Change: ln (on treatment)−ln (baseline) models adjusted for log (baseline variable), age, sex, body mass index, smoking(y/n) and study (CAP versus PRINCE). Betas and P values were assessed using a generalized estimating equation method.

### Effects of off-treatment LDL-C

To demonstrate that our findings for LDL-C response to statin treatment are unlikely to be explained through associations with baseline LDL-C levels, we performed a number of additional analyses (see Methods). First, [Supplementary Table 7](#S1) shows regression coefficients for baseline-adjusted and measurement noise-corrected estimates of the direct effect of genotype on on-treatment LDL-C at the strongest SNPs in the GIST meta-analysis (*P*<1 × 10^−8^), which were available in the CARDS data set. Correcting our effect size estimate further and modelling measurement noise at baseline reduced the apparent effect only slightly for all the markers, suggesting that there is little effect of measurement noise. Next, within the JUPITER trial, additional analyses were performed to determine whether there was an interaction between LDL-C change and statin or placebo allocation. [Supplementary Table 8](#S1) shows significant *P* values for interaction (all <5 × 10^−2^) for SNPs at the four genome-wide significant loci in the GIST meta-analysis, also suggesting that genetic effects on baseline LDL-C as manifested in the placebo group contribute at most only in part to genetic effects on LDL-C response in the statin group.

### Genome-Wide Conditional Analysis

To investigate whether there were multiple SNPs within any gene and multiple loci associated with differential LDL-C lowering to statin therapy, we performed a conditional analysis across the genome using the summary statistics of the combined meta-analysis. The results of the Genome-Wide Conditional Analysis (GWCA; see Methods; [Supplementary Table 9](#S1)) showed 14 SNPs independently associated with statin response and these explained ~5% of the variation in LDL-C response to statin treatment. Of the 14 independent SNPs, 6 were genome-wide significant in the combined GWAS meta-analysis ([Supplementary Table 5](#S1)).

### Previous findings

In [Supplementary Table 10](#S1), we performed a look-up in our GWAS meta-analysis for SNPs previously described in the literature (NHGRI Catalogue[11](#b11) of Published GWAS and Candidate gene studies) to be associated with statin response, besides the loci associated at a genome-wide level in the current study. None of these SNPs was associated with statin response in our GWAS after correcting for multiple testing.

### Functional analyses

Functional characterization of the 246 SNPs selected for the second stage was performed using a range of bioinformatics tools (see Methods). A total of 420 expression quantitative trait loci (eQTL) associations were identified across a wide range of tissues ([Supplementary Data 1](#S1)), which comprised 67 independent gene eQTL associations. Eleven genes, including *APOE*, *SORT1*, *CELSR2* and *PSRC1*, showed eQTLs in liver, which considering its primary role in mediating statin-induced LDL reduction may be particularly relevant to statin response. Putative gene eQTLs were combined with genes annotated to variants in linkage disequilibrium (LD) with LDL-C response-associated variants, resulting in a list of 185 candidate gene loci, defined by 2,681 SNPs ([Supplementary Data 2 and 3](#S1)). To identify statin responsive genes among the candidate loci, gene expression data measured in response to statin treatment in a range of cell lines was retrieved from the Connectivity Map resource[12](#b12) (see Methods). Five genes (*APOE, BRCA1, GRPEL1, ADRB2* and *ETV1*) showed convincing evidence of statin responsiveness on the basis of greater than twofold differential expression in response to statin treatment. Eight genes showed suggestive evidence (1.5- to 2-fold change; *TOMM40*, *SREBP1*, *PSRC1*, *BCL3*, *BCAM*, *ANK3*, *SIVA1* and *RANBP9*; [Supplementary Data 3](#S1)).

Finally, involvement in statin response was investigated at a pathway level using GeneGo Metacore (Thomson Reuters[13](#b13)). Briefly, 87 literature-reported genes linked to statin response were combined with the 185 candidate gene loci reported here ([Supplementary Data 3](#S1)). A conservative network of direct interactions was constructed between query genes ([Supplementary Data 4](#S1)). The network included 24 genes located in the LDL-C-associated loci ([Supplementary Fig. 4](#S1)). Collectively, our functional and pathway analysis confirms a strong biological and functional role in statin response for several strongly associated gene loci, including *APOE/TOMM40/PVRL2* and *SORT1/CELSR2/PSRC2.*

## Discussion

We have performed a meta-analysis of GWAS including more than 40,000 subjects, investigating genetic variants associated with variation in LDL-C lowering on statin treatment independent from associations with baseline LDL-C. We identified four loci at genome-wide significance, including the previously identified *APOE* and *LPA,* and the novel GWAS loci *SORT1/CELSR2/PSRC1* and *SLCO1B1*.

Nine SNPs in the *APOE* gene region reached genome-wide significance for LDL-C response. The minor allele of the lead SNP rs445925, which is a proxy for the apoE ε2 protein variant defining SNP rs7412 (ref. 14)[14](#b14), was associated with a larger LDL-C-lowering response to statins compared with carriers of the major allele. The magnitude and direction of the effect size was similar to previously reported findings for the rs445925 variant in the GWAS study performed in CARDS and ASCOT[8](#b8) and of the SNP rs7412 in JUPITER[7](#b7). Since the apoE ε2 protein results in increased hepatic cholesterol synthesis, it may also predispose to stronger inhibition of cholesterol synthesis by statin treatment[8](#b8),[10](#b10).

Three independent SNPs at *LPA* were significantly associated with LDL-C response to statins. The minor G allele of the lead SNP rs10455872 was associated with smaller LDL-C reduction than the major allele. This result was similar to the previous GWAS findings for this SNP in the JUPITER trial and the combined ASCOT and CARDS study[7](#b7),[8](#b8). The rs10455872 SNP was strongly associated with the KIV-2 copy number variant in Lp(a), which encodes variability in apo(a) size and is responsible for ~30% of variance in Lp(a) levels[8](#b8),[15](#b15). Furthermore, rs10455872 was shown to be strongly associated with plasma Lp(a) levels[16](#b16). Standard assays of LDL-C, as well as the Friedewald formula, include cholesterol that resides in Lp(a)[6](#b6),[8](#b8). Carriers of this *LPA* variant are characterized by higher Lp(a) levels and a larger proportion of their measured LDL-C resides in Lp(a) particles[8](#b8),[10](#b10). Since statin therapy does not reduce the number of Lp(a) particles[17](#b17), their presence attenuates the measured LDL-C response to statins.

Two SNPs at *SORT1/CELSR2/PSRC1* (rs646776 and rs12740374) on chromosome 1p were associated with an enhanced statin LDL-C response. A similar association was previously observed in a large candidate gene study in HPS[6](#b6); however, we demonstrate this finding now first at a genome-wide significance level. The minor allele of rs12740374 has been shown to generate a binding site for the transcription factor C/EBPa[18](#b18). Transcription results in upregulation of hepatic expression of three genes at this locus, *SORT1*, *CELSR2* and *PSRC1* (ref. [18](#b18)), which we also showed in our eQTL analysis ([Supplementary Data 1](#S1)). Of these, *SORT1* is most notable, in that it encodes the multifunctional intracellular trafficking protein sortilin, which has been shown to bind tightly to apoB[19](#b19). Sortilin-induced lowering of plasma LDL-C results from two mechanisms: reduced secretion of apoB-containing precursors, and, perhaps of greater importance, increased hepatic LDL uptake via binding to sortilin at the cell surface, with subsequent internalization and lysosomal degradation[19](#b19). Notably, the minor allele of rs646776 is preferentially associated with lower levels of small and very small LDL ([Table 2](#t2)), suggesting that sortilin is of particular importance for regulating levels of these particles[18](#b18). Smaller LDL subfractions have been shown to be relatively enriched in particles with reduced LDL receptor binding affinity and cellular uptake[20](#b20), a property that may contribute to their associations with increased risk for cardiovascular disease[21](#b21). This property may also underlie the diminished efficacy of statins for reduction of these particles ([Supplementary Fig. 3](#S1))[22](#b22), since statins act to reduce LDL-C levels to a large extent by increasing LDL receptor expression as a result of upregulation of the transcription factor SREBP2, whereas *SORT1* is not regulated by this mechanism. Hence, the greater statin-mediated reduction of LDL-C among carriers of the rs646776 minor allele could be attributed to relative depletion of LDL particles dependent on sortilin for clearance and hence a residually greater proportion of those LDL particles whose uptake is more dependent on the LDL receptor than on sortilin.

Notably, the strong association of rs646776 with statin-induced reductions in small and very small LDL particles contrasts to the weaker associations of changes in these particles with rs445925, likely the result of differing mechanisms underlying the effects of these SNPs on statin response. As noted above, rs445925 is a proxy for the SNP defining the apoE ε2 protein variant that is thought to predispose to heightened statin response as a result of greater statin inhibition of cholesterol synthesis and hence upregulation of SREBP and LDL receptor activity.

The *SLCO1B1* rs2900478 minor allele was associated with a smaller LDL-C reduction in response to statin treatment. *SLCO1B1* encodes the organic anion-transporting polypeptide OATP1B1 and facilitates the hepatic uptake of statins[23](#b23). SNP rs2900478 is in strong LD (*r*^2^=0.89) with rs4149056, which represents the Val174Ala substitution resulting in complete loss of function. In the HPS trial, which used simvastatin, this candidate gene SNP was associated with a 1% lower LDL-C reduction per allele[6](#b6). Single-dose studies have shown that the observed area under the curve of plasma level of active simvastatin after a dose of 40 mg was 221% higher in rs4149056 CC homozygotes compared with rs4149056 TT homozygotes, as compared with atorvastatin 20 mg (144% higher for CC versus TT) and rosuvastatin 40 mg (117% higher for CC versus TT)[24](#b24). This finding results from the slower hepatic uptake of statins caused by the genetic variant, which would also be expected to result in a reduction in the cholesterol-lowering effect[25](#b25). In a GWAS of the genetic risk factors for simvastatin-induced myopathy, *SLCO1B1* showed the strongest association[25](#b25). Homozygous carriers of the *SLCO1B1* variant had a 16.9 times higher risk for myopathy compared with non-carriers. This might have led to a decrease in study medication adherence, and consequently a decreased effect on LDL-C in carriers of this SNP. In addition, previous analysis in the GoDARTS study showed that the effect of the *SLCO1B1* gene on statin efficacy was abolished after removal of individuals who showed signs of intolerance[26](#b26).

GWCA identified three independent loci in the *APOE* gene region and two loci in the *LPA* gene region ([Supplementary Table 9](#S1)). GWCA also showed several other loci with *P* <5 × 10^−8^ that were not GWAS significant on single-SNP analysis (*HGD*, *RNF175*, *ISCA1L-HTR1A*, *GLIS3-SLC1A1*, *LOC100128657*, *NKX2-3-SLC25A28* and *PELI2*). These findings will require replication in independent, larger data sets. The significant SNPs in the GWCA analysis explained ~5% of the variation in LDL-C response to statin treatment. Whether this 5% is clinically relevant should be investigated by other studies. For example, it would be of interest to investigate whether this differential LDL-C lowering is also associated with differential event reduction by statin treatment.

In the current study, we combined the results of 6 randomized clinical trials and 10 observational studies in the first stage. This approach resulted also in combining several types of statins, since different statins were studied in the trials and within the observational studies ([Supplementary Table 2](#S1)). This, and the variation in statin dosage during follow-up for an individual, is a limitation of the current study, since, for example, the impact of the *SLCO1B1* variant on statin pharmacogenetics is known to be highly dependent on statin type and dose[24](#b24),[27](#b27). To overcome this limitation, the individual study analyses were adjusted for statin dose. Dividing the actual statin dose given by the statin-specific dose equivalent ([Supplementary Table 3](#S1)) gives the statin-adjusted equivalent based on the daily dosages required to achieve a mean 30% LDL-C reduction. Using this table, we made the different statin dosages and types comparable within the studies. To correct for between-study variance, we used a fixed effect meta-analysis with inverse variance weighting. Since we observed that the *SLCO1B1* gene was genome-wide significantly associated with LDL lowering, this highlights the thoroughness of our analytical approach, in which the analyses were correctly adjusted for the type and dose of statins used ([Supplementary Table 3](#S1)). Moreover, a comparison of the estimates of the SNPs between the RCTs (where there are no intra-individual differences in dosages) with the estimates of the SNPs in the observational studies showed large homogeneity between the estimates in the various study designs ([Supplementary Fig. 2](#S1)), indicating that our adjustment for dosage seems to be sufficient within this study.

Another possible limitation of the current study is the influence of the identified genetic variants on baseline LDL-C levels. In pharmacogenetic studies investigating the LDL-C-lowering response to statins, it is important to eliminate the effect of association between the genetic variant and baseline LDL-C levels, since those findings may confound the response to treatment associations. Previous large GWAS studies have shown strong associations between baseline LDL-C levels and genetic variants in *SORT1/CELSR2/PSRC1*, *APOE* and *LPA*[28](#b28). To eliminate those possible confounding effects, our response to treatment analyses were adjusted for baseline LDL-C levels. In addition, additional analysis in CARDS and JUPITER suggests no or little influence of genetic associations with baseline LDL-C on the genetic effects on LDL-C-lowering response.

In conclusion, this study is the largest meta-analysis of GWAS for LDL-C response to statin therapy conducted to date. Our results demonstrate that apart from the previously identified *APOE* and *LPA* loci, two new loci, *SORT1/CELSR2/PSRC1* and *SLCO1B1*, also have a modest but genome-wide significant effect on LDL-C response. The minor alleles of the *APOE* rs445925 and *SORT1*/*CELSR2/PSRC1* rs646776 SNPs were associated with a larger statin response, whereas the minor alleles of the *LPA* rs10455872 and *SLCO1B1* rs2900478 SNPs were associated with a smaller statin response. Our findings advance the understanding of the pharmacogenetic architecture of statin response.

## Methods

### Study populations

The meta-analysis was conducted in the GIST consortium, which includes data from 8 randomized controlled statin trials (RCTs) and 11 prospective, population-based studies. The initial analysis (first stage) was performed in 8,421 statin-treated subjects from 6 RCTs (ASCOT, CARDS, CAP, PRINCE, PROSPER and TNT) and 10,175 statin-treated subjects from 10 observational studies (AGES, ARIC, BioVU, CHS, FHS, GoDARTS I, GoDARTS II, Health ABC, HVH and MESA). Further investigation (second stage) was performed in 21,975 statin-treated subjects from two randomized trials (HPS and JUPITER) and one observational study (Rotterdam Study). Six SNPs were additionally genotyped in the Scandinavian participants of the ASCOT study. The details of the first- and second-stage studies can be found in the [Supplementary Tables 1 and 2](#S1) and [Supplementary Notes 1 and 2](#S1).

### Subjects

Response to statin treatment was studied in statin-treated subjects only and not in those treated with placebo. Subjects included in the observational studies’ analysis should be treated with statins and have LDL-C measurements before and after start of statin treatment. Subjects of reported or suspected non-European ancestry were excluded. All participants gave written informed consent and the study was approved by all institutional ethics committees.

### Outcome measurements

The response to statin treatment was defined as the difference between the natural log-transformed on- and off-treatment LDL-C levels. The beta of the corresponding regression thus reflects the fraction of differential LDL lowering in carriers versus non-carriers of the SNP. For observational studies, the on-treatment LDL-C levels were taken into account for all kinds of prescribed statins, at any dosage, for any indication and for at least 4 weeks before measurement. Characteristics of on- and off-treatment LDL-C levels and statins used in each study are shown in [Supplementary Table 2](#S1). For each individual, at least one off-treatment LDL-C measurement and at least one on-treatment LDL-C measurement were required. When multiple on- or off-treatment measurements were available, the mean of the cholesterol measurements was used. Subjects with missing on- or off-treatment measurements were excluded, with the exception of the GoDARTS cohorts for which missing off-treatment LDL-C levels were estimated using imputation methods ([Supplementary Note 2](#S1)). In the HPS, proportional LDL-C response was defined by the changes in natural log lipid levels from the screening visit before starting statin therapy to the randomization visit[6](#b6).

### Genotyping and imputation

Genotyping, quality control, data cleaning and imputation were performed independently in each study using different genetic platforms and software as outlined in [Supplementary Table 4](#S1). In all studies, genotyping was performed using Illumina, Affymetrix or Perlegen genotyping arrays, and MACH, Impute or BIMBAM software was used for imputation.

### GWAS analysis

Each study independently performed the GWAS on the difference between natural log-transformed on- and off-treatment LDL-C levels. To control for possible associations with off-treatment LDL-C levels, analyses were adjusted for the natural log-transformed off-treatment LDL-C level. An additive genetic model was assumed and tested using a linear regression model. For imputed SNPs, regression analysis was performed onto expected allele dosage. Analyses were additionally adjusted for age-, sex- and study-specific covariates (for example, ancestry principal components or country). Analyses in the observational studies were, if available, additionally adjusted for the statin dose by the natural logarithm of the dose equivalent as defined in [Supplementary Table 3](#S1). This table shows the dose equivalent per statin type; dividing the statin dosage of an individual by the dose equivalent shown in [Supplementary Table 3](#S1) will give the adjusted statin dosage.

### Quality control and meta-analysis

Centrally, within each study, SNPs with MAF <1% or imputation quality <0.3 were excluded from the analysis. QQ-plots were assessed for each study to identify between-study differences ([Supplementary Fig. 1](#S1)). The software package METAL was used for performing the meta-analysis ( [http://www.sph.umich.edu/csg/abecasis/Metal/index.html](http://www.sph.umich.edu/csg/abecasis/Metal/index.html)). A fixed effects, inverse variance weighted approach was used. Using an inverse variance weighted meta-analysis will give smaller weights to studies with large s.e.. To correct for possible population stratification, genomic control was performed by adjusting the within-study findings and the meta-analysis results for the genomic inflation factor.

### Second stage

SNPs with *P* values <5 × 10^−4^ in the first-stage meta-analysis were selected for further investigation in a second stage. A maximum of two SNPs per locus were selected, based on statistical significance, except for the *APOE* locus, for which all genome-wide significant associated SNPs were selected for validation. A total of 246 SNPs, within 158 independent loci, were selected for the second stage, which was performed in the JUPITER trial, HPS study and the Rotterdam Study, which all had GWAS data and response to statin treatment available. For 2 of the 246 SNPs, a proxy was used in the JUPITER trial, and 31 SNPs were not available, nor was a proxy SNP. HPS provided data on 151 directly genotyped SNPs from GWAS and IPLEX experiments, including 48 of the requested SNPs and 103 proxy SNPs (*r*^2^>0.8). Analysis in HPS was not adjusted for ln baseline LDL-C levels. In addition, the number of subjects with data varied from SNP-to-SNP and ranges from ~4,000 for variants with GWAS data to ~18,000 for some candidate genes. Results of the first and second stage were combined using fixed effects, inverse variance weighted meta-analysis and analysed by METAL. As a third stage, six SNPs with *P* values 5 × 10^−8^<*P*<5 × 10^−7^ in the combined meta-analysis were selected for additional genotyping in the Scandinavian participants of the ASCOT study. Kaspar assays were designed for four of the SNPs using the KBioscience Primerpicker software, and oligos were provided by Intergrated DNA technologies ( [http://eu.idtdna.com/site](http://eu.idtdna.com/site)). Full Kaspar methodology is available from LGC SNP genotyping ( [http://www.lgcgenomics.com/genotyping/kasp-genotyping-reagents/](http://www.lgcgenomics.com/genotyping/kasp-genotyping-reagents/)). Two SNPs (rs981844 and rs13166647) were genotyped using Taqman assays supplied by Life Technologies ( [http://www.lifetechnologies.com/uk/en/home.html](http://www.lifetechnologies.com/uk/en/home.html)) using the standard Taqman protocol. Results of the additional genotyping were combined with results from the first and second stages using a fixed effects, inverse variance weighted meta-analysis and analysed by METAL.

### Determination of changes in LDL subfractions

LDL subclasses were analysed as described previously[29](#b29) using non-denaturing gradient gel electrophoresis of fasting plasma samples taken at baseline and after 6 weeks of simvastatin 40 mg per day (CAP study, *n*=579) or 12 weeks of pravastatin 40 mg per day (PRINCE study, *n*=1,284). Aliquots of 3.0 ml of whole plasma were mixed 1:1 with a sampling buffer of 20% sucrose and 0.25% bromophenol blue. Electrophoresis of samples and size calibration standards was performed using 2–14% polyacrylamide gradients at 150 V for 3 h following a 15-min pre-run at 75 V. Gels were stained with 0.07% Sudan black for 1 h and stored in a 0.81% acetic acid, 4% methanol solution until they were scanned by computer-assisted densitometry for determination of areas of LDL IVb (22.0–23.2 nm), LDL IVa (23.3–24.1 nm), LDL IIIb (24.2–24.6 nm), LDL IIIa (24.7–25.5 nm), LDL IIb (25.6–26.4 nm), LDL IIa (26.5–27.1 nm) and LDL I (27.2–28.5 nm). The cholesterol concentrations of the subfractions (mg dl^−1^ plasma) were determined by multiplying percent of the total stained LDL area for each subfraction by the LDL-C for that sample. For genetic association analyses, subfractions were grouped into large LDL (LDL I+IIa), medium LDL (LDL IIb), small LDL (LDL IIIa) and very small LDL (LDL IIIb+IVa+IVb) as described previously[18](#b18). A generalized estimating equation method was used to test the association of log change with the interaction of the four SNPs by LDL subfraction.

### Effect of off-treatment LDL-C

Effects of genetic variation on treatment response as measured by on-treatment LDL-C could be mediated through effects on the off-treatment LDL-C. To evaluate whether genetic on-treatment LDL-C likely reflects residual effect on off-treatment LDL-C, it is necessary to adjust for the off-treatment LDL-C levels and to correct the maximum likelihood estimate of the adjusted effect of genotype on on-treatment value for the noise in off-treatment values (the noise is both random measurement error and intra-individual variation in usual LDL-C). This analysis was only carried out in CARDS in which multiple baseline measurements were available. From the rules of path analysis, we calculated the direct effect *γ* of genotype on an on-treatment trait value as *β*−*αδ* (1−*ρ*)/*ρ*, where *β* is the coefficient of regression for on-treatment trait value on genotype adjusted for measured off-treatment value, *α* is the coefficient of regression of baseline LDL on genotype, *ρ* is the intraclass correlation between replicate measurements of off-treatment values and *δ* is the coefficient of regression for on-treatment value on observed off-treatment value[8](#b8). For these calculations, we used *ρ*=0.8 as a plausible value for the intraclass correlation based on the within-person correlation in LDL-C values taken over two off-treatment visits in CARDS. The interaction of candidate SNPs with statin versus placebo allocation was assessed in the JUPITER trial, since this study was not involved in the first-stage meta-analysis. Regression models were applied to the combined population of statin- and placebo-treated subjects by including extra terms encoding placebo allocation and the product of placebo allocation with SNP minor allele dose[7](#b7).

### GWCA using Genome-Complex Trait Analysis

There may be multiple causal variants in a gene and the total variation that could be explained at a locus may be underestimated if only the most significant SNP in the region is selected. To identify independent SNPs, we ideally can perform a conditional analysis, starting with the top associated SNP, across the whole genome followed by a stepwise procedure of selecting additional SNPs, one by one, according to their conditional *P* values. Such a strategy would allow the discovery of more than two associated SNPs at a locus. To identify independent SNPs across the genome-wide data, we used an approximate conditional and joint analysis approach implemented in Genome-Complex Trait Analysis (GCTA) software ( [http://www.complextraitgenomics.com/software/gcta/](http://www.complextraitgenomics.com/software/gcta/)). We used summary-level statistics from the first- and second-stage-combined meta-analysis and LD corrections between SNPs estimated from CARDS GWAS data. SNPs on different chromosomes or more than 10 Mb distant are assumed to be in linkage equilibrium. The model selection process in GCTA starts with the most significant SNP in the single-SNP meta-analysis across the whole genome with *P* value <5 × 10^−7^. In the next step, it calculates the *P* values of all the remaining SNPs conditional on the top SNP that have already been selected in the model. To avoid problems due to colinearity, if the squared multiple correlations between a SNP to be tested and the selected SNP(s) is larger than a cut-off value, such as 0.9, the conditional *P* value for that SNP will be set to 1. Select the SNPs with minimum conditional *P* value that is lower than the cut-off *P* value. Fit all the selected SNPs jointly in a model and drop the SNPs with the *P* value that is greater than the cut-off *P* value. This process is repeated until no SNPs can be added or removed from the model.

### Pathway analysis and construction of a statin response network

Genes showing evidence of association (based on direct association or LD (HapMap CEU *r*^2^>0.8)) were reviewed for evidence of involvement in statin response at a pathway level using GeneGo Metacore (Thomson Reuters (portal.genego.com)). A statin response network was constructed in two stages. First, all genes with a literature-reported involvement in statin response (based on Medical Subject Headings (MeSH)) were identified using GeneGo MetaCore ([Supplementary Data 3](#S1)). Second, these genes were combined with all genes in associated loci (including genes in LD) and a network was constructed based on direct interactions only. By including direct interactions only, we created a conservative network of direct gene interactions that have been consistently linked to statin response in the literature.

### eQTL analysis

LDL-C-associated index SNPs (246 SNPs) were used to identify 1,443 LD proxy SNPs displaying complete LD (*r*^2^=1) across four HapMap builds in European ancestry samples (CEU) using the SNAP tool ( [http://www.broadinstitute.org/mpg/snap/](http://www.broadinstitute.org/mpg/snap/)). The primary index SNPs and LD proxies were searched against a collected database of expression SNP (eSNP) results, including the following tissues: fresh lymphocytes[30](#b30), fresh leukocytes[31](#b31), leukocyte samples in individuals with Celiac disease[32](#b32), whole-blood samples[33](#b33),[34](#b34),[35](#b35),[36](#b36), lymphoblastoid cell lines (LCL) derived from asthmatic children[37](#b37),[38](#b38), HapMap LCL from three populations[39](#b39), a separate study on HapMap CEU LCL[40](#b40), additional LCL population samples[41](#b41),[42](#b42),[43](#b43) (Mangravite *et al.*, unpublished), CD19+ B cells[44](#b44), primary phytohaemagglutinin-stimulated T cells[41](#b41), CD4+ T cells[45](#b45), peripheral blood monocytes[44](#b44),[46](#b46),[47](#b47), CD11+ dendritic cells before and after *Mycobacterium* tuberculosis infection[48](#b48), omental and subcutaneous adipose[33](#b33),[43](#b43),[49](#b49), stomach[49](#b49), endometrial carcinomas[50](#b50), ER+ and ER− breast cancer tumour cells[51](#b51), brain cortex[46](#b46),[52](#b52),[53](#b53), prefrontal cortex[54](#b54),[55](#b55), frontal cortex[56](#b56), temporal cortex[53](#b53),[56](#b56), pons[56](#b56), cerebellum[53](#b53),[56](#b56), three additional large studies of brain regions including prefrontal cortex, visual cortex and cerebellum, respectively[57](#b57), liver[49](#b49),[58](#b58),[59](#b59), osteoblasts[60](#b60), ileum[49](#b49),[61](#b61), lung[62](#b62), skin[43](#b43),[63](#b63) and primary fibroblasts[41](#b41). Micro-RNA QTLs were also queried for LCL[64](#b64) and gluteal and abdominal adipose[65](#b65). The collected eSNP results met the criteria for association with gene expression levels as defined in the original papers. In each case where a LDL-C-associated SNP or proxy was associated with a transcript, we further examined the strongest eSNP for that transcript within that data set (best eSNP), and the LD between the best eSNP and GIST-selected eSNPs to estimate the concordance of the LDL-C and expression signals.

### Statin response connectivity map analysis

The Connectivity Map (Cmap) data set is available at the Broad Institute ( [www.broadinstitute.org/cmap](http://www.broadinstitute.org/cmap)) and contains more than 7,000 expression profiles representing 1,309 compounds used on five different cultured human cancer cell lines (MCF7, ssMCF7, HL60, PC3 and SKMEL5). We selected (prostate tumour-derived) PC3 cells as they showed the most responsiveness to statins at a genome-wide level. Four statins were included in our analysis, including pravastatin, atorvastatin, simvastatin and rosuvastatin. PC3 Instance reference files for each statin treatment were extracted (as defined by Lamb *et al.*[12](#b12)), that is, a treatment associated to its control pair. Transcripts were considered to show evidence of differential expression with a fold change >2. A fold change >1.5 was considered to be suggestive of differential expression only.

### Exploration of functional impact among directly and indirectly associated variants

Genes and variants across all LDL-C-associated loci were investigated for evidence of functional perturbation using a range of bioinformatics tools and databases. Variants showing LD (CEU *r*^2^>0.8) with associated variants were explored for impact on coding gene function using Annovar[66](#b66) and regulatory function using a combination of HaploReg[67](#b67) and Regulomedb[68](#b68), which both draw on comprehensive data from the Encyclopedia of DNA Elements (ENCODE)[69](#b69) and the NIH Roadmap Epigenomics consortium[70](#b70). Building on the functional annotation, we also identified variants that were shown to mediate eQTLs. Genes in associated loci were also used to query the NIH connectivity map for evidence of differential expression in PC3 cell lines treated with pravastatin, simvastatin and rosuvastatin. By combining a wide range of functional data and pathway support, we were able to build up a view of genes with the highest level of support in statin response.

## Author contributions

I.P., S.T., H.A.D., M.R.B., X.L., H.R.W., D.I.C., K.Z., B.J.A., B.M.P., G.H., R.M.K., J.W.J. and M.J.C. constituted the writing and analysis group. I.P., S.T., H.A.D. and K.Z. performed quality control on the individual study summary results. I.P. and S.T. performed meta-analysis. I.P., H.A.D., M.R.B., X.L., H.R.W., D.I.C. and R.M.K. performed additional analyses. All analysis and writing group authors extensively discussed the analysis, results, interpretation and presentation of results. All authors contributed to the research and reviewed the manuscript.

Study concept and design of contributing studies by (PROSPER) J.W.J., D.J.S., B.M.B., I.F., N.S. and R.G.J.W.; (ASCOT) M.J.C., P.S., N.P., A.S., D.C.S. and E.O.; (CARDS) H.A.D., H.M.C., P.M.M., J.B., P.N.D., A.D. and G.H.; (PARC) X.L., Y.-D.I.C., J.I.R. and R.M.K.; (TNT) J.J.P.K.; (AGES) L.J.L., T.B.H. and V.G.; (ARIC) C.L.A., E.A.W., T.S., E.B. and C.M.B.; (BioVU) Q.F., W.-Q.W., R.A.W. and J.C.D.; (CHS, HVH) N.S., K.R., T.L., J.I.R., B.M.P. and S.R.H.; (FHS) L.A.C. and V.R.; (GoDARTS) C.N.A.P. and H.M.C.; (HABC) Y.L.; (MESA) X.G., S.R.H., W.P. and J.I.R.; (Rotterdam Study) C.E.d.K., B.H.S., A.G.U., A.H. and F.R.; and (JUPITER) D.I.C., B.J.B., F.N. and P.M.R.

Phenotype data acquisition of contributing studies by (PROSPER) J.W.J., D.J.S., B.M.B., I.F., A.J.M.d.C., N.S. and R.G.J.W.; (ASCOT) M.J.C., P.B.M., P.S., N.P., A.S., D.C.S., E.O. and S.S.H.; (CARDS) H.A.D., H.M.C., P.M.M., J.B., P.N.D., A.D. and G.H.; (PARC) X.L., Y.-D.I.C., J.I.R. and R.M.K.; (TNT) J.J.P.K.; (AGES) G.E.; (ARIC) C.M.B.; (BioVU) W.W.; (CHS, HVH) K.L.W., J.C.B., A.M.A., N.L.S., B.M.P. and S.R.H.; (FHS) L.A.C., C.J.O., V.R.; (GoDARTS) C.N.A.P., L.A.D., K.Z., A.D., A.M. and H.M.C.; (HABC) D.M.H. and S.B.K.; (MESA) W.P. and J.I.R.; (Rotterdam Study) C.E.K., B.H.S., A.H. and O.H.F.; (JUPITER) D.I.C., F.G., J.G.M. and P.M.R.

Genotype data acquisition of contributing studies by (PROSPER) S.T., J.W.J., A.J.M.C. and P.E.S.; (ASCOT) M.J.C., P.B.M., P.S., A.S. and S.S.H.; (CARDS) H.A.D., H.M.C., P.M.M., P.N.D., A.D. and G.H.; (PARC) Y.-D.I.C., J.I.R., D.A.N. and J.D.S.; (TNT) B.J.A., M.P.D., S.M.B., G.K.H. and J.-C.T.; (AGES) A.V.S.; (ARIC) E.B.; (BioVU) Q.F., J.C.D., C.T.L. and F.S.; (CHS, HVH) G.L., J.C.B., K.D.T., J.I.R., K.R., T.L. and S.R.H.; (FHS) C.J.O.; (GoDARTS) C.N.A.P., K.Z., A.D., F.C., H.M.C., M.I.M., L.G., E.A. and WTCCC2; (HABC) Y.L.; (MESA) K.D.T. and J.I.R.; (Rotterdam Study) A.G.U. and F.R.; and (JUPITER) D.I.C., F.G., B.J.B., F.N. and P.M.R.

Primary analysis from contributing studies by (PROSPER) I.P., S.T., A.J.M.C. and P.E.S.; (ASCOT) M.J.C., M.R.B. and H.R.W.; (CARDS) H.A.D., H.M.C. and P.M.M.; (PARC) X.L., Y.-D.I.C. and J.I.R.; (TNT) B.J.A., M.P.D., S.M.B., G.K.H. and J.-C.T.; (AGES) A.V.S.; (ARIC) C.L.A., E.A.W. and T.S.; (BioVU) Q.F., W.W., C.T.L. and F.S.; (CHS, HVH) K.L.W. and G.L.; (FHS) L.A.C., P.G. and J.S.N.; (GoDARTS) C.N.A.P., L.A.D., K.Z. and H.M.C.; (HABC) D.S.E., J.M.S. and J.D.; (MESA) K.D.T., X.G., X.L. and J.I.R.; (Rotterdam Study) C.E.K. and B.H.S.; and (JUPITER) D.I.C., A.Y.C., F.G., J.G.M. and P.M.R.

## Additional information

**How to cite this article:** Postmus, I. *et al.* Pharmacogenetic meta-analysis of genome-wide association studies of LDL cholesterol response to statins. *Nat. Commun.* 5:5068 doi: 10.1038/ncomms6068 (2014).

## Supplementary Material

## Acknowledgments

Data from the Heart Protection Study have been contributed by Heart Protection Study Collaborative Group[6](#b6).

*PROSPER/PHASE*: The Prospective Study of Pravastatin in the Elderly at Risk (PROSPER) trial was supported by an investigator-initiated grant from Bristol-Myers Squibb, USA. The study was conducted, analysed and reported independently of the company. The GWAS project PHASE has received funding from the European Union’s Seventh Framework Programme (FP7/2007–2013) under grant agreement HEALTH-F2-2009-223004. A part of the genotyping was funded by The Netherlands Consortium for Healthy Ageing (NGI: 05060810). Professor Dr J.W.J. is an established clinical investigator of The Netherlands Heart Foundation (2001 D 032).

*ASCOT*: The Anglo-Scandinavian Cardiac Outcomes Trial (ASCOT) trial was funded by an investigator-initiated grant from Pfizer, USA. The study was investigator led and was conducted, analysed and reported independently of the company. The Genomewide Association Scan was funded by the National Institutes for Health Research (NIHR) as part of the portfolio of translational research of the NIHR Biomedical Research Unit at Barts and the NIHR Biomedical Research Centre at Imperial College, the International Centre for Circulatory Health Charity and the Medical Research Council through G952010. On behalf of the ASCOT investigators, we thank all ASCOT trial participants, physicians, nurses, and practices in the participating countries for their important contribution to the study.

*CARDS*: We thank the other investigators, the staff and the participants of the CARDS study. A full list of CARDS investigators can be found in original CARDS paper[71](#b71). CARDS was funded by grants to the Universities of London and Manchester by Pfizer, Diabetes UK and the Department of Health.

*PARC*: This research was supported by the National Institutes of Health: grant U19 HL069757 from the National Heart, Lung, and Blood Institute, and grant UL1TR000124 from the National Center for Advancing Translational Sciences.

*TNT*: The TNT study was funded by Pfizer, who also provided support for genotyping.

*AGES*: This study has been funded by NIH contract N01-AG-1-2100, the NIA Intramural Research Program, Hjartavernd (the Icelandic Heart Association) and the Althingi (the Icelandic Parliament). The study is approved by the Icelandic National Bioethics Committee, VSN: 00-063. The researchers are indebted to the participants for their willingness to participate in the study.

*ARIC*: The Atherosclerosis Risk in Communities Study is carried out as a collaborative study supported by National Heart, Lung, and Blood Institute contracts (HHSN268201100005C, HHSN268201100006C, HHSN268201100007C, HHSN268201100008C, HHSN268201100009C, HHSN268201100010C, HHSN268201100011C and HHSN268201100012C), R01HL087641, R01HL59367 and R01HL086694; National Human Genome Research Institute contract U01HG004402; and National Institutes of Health contract HHSN268200625226C. We thank the staff and participants of the ARIC study for their important contributions. Infrastructure was partly supported by Grant Number UL1RR025005, a component of the National Institutes of Health and NIH Roadmap for Medical Research.

*BioVU*: BioVU receives support through the National Center for Research Resources UL1 RR024975, which is now the National Center for Advancing Translational Sciences, 2 UL1 TR000445. Genotyping was supported via grant U01-HG04603 from the National Human Genome Research Institute and RC2-GM092318 from the National Insitute of General Medical Sciences.

*CHS*: This CHS research was supported by NHLBI contracts HHSN268201200036C, HHSN268200800007C, N01HC55222, N01HC85079, N01HC85080, N01HC85081, N01HC85082, N01HC85083 and N01HC85086; and NHLBI grants HL080295, HL087652, HL105756 and HL103612 with additional contribution from the National Institute of Neurological Disorders and Stroke (NINDS). Additional support was provided through AG023629 from the National Institute on Aging (NIA). A full list of CHS investigators and institutions can be found at [http://www.chs-nhlbi.org/pi.htm](http://www.chs-nhlbi.org/pi.htm). The provision of genotyping data was supported in part by the National Center for Advancing Translational Sciences, CTSI grant UL1TR000124, and the National Institute of Diabetes and Digestive and Kidney Disease Diabetes Research Center (DRC) grant DK063491 to the Southern California Diabetes Endocrinology Research Center.

*Framingham HS*: The Framingham Heart Study work was supported by the National Heart Lung and Blood Institute of the National Institutes of Health and Boston University School of Medicine (Contract No. N01-HC-25195), its contract with Affymetrix, Inc. for genotyping services (Contract No. N02-HL-6-4278) and based on analyses by Framingham Heart Study investigators participating in the SNP Health Association Resource (SHARe) project. A portion of this research was conducted using the Linux Cluster for Genetic Analysis (LinGA-II) funded by the Robert Dawson Evans Endowment of the Department of Medicine at Boston University School of Medicine and Boston Medical Center, also supported by R01HL103612 (PI psaty, subcontract PI, Vasan).

*GoDARTS*: We are grateful to all the participants who took part in this study, to the general practitioners, to the Scottish School of Primary Care for their help in recruiting the participants and to the whole team, which includes interviewers, computer and laboratory technicians, clerical workers, research scientists, volunteers, managers, receptionists and nurses. The Wellcome Trust provides support for Wellcome Trust United Kingdom Type 2 Diabetes Case Control Collection (GoDARTS) and informatics support is provided by the Chief Scientist Office. The Wellcome Trust funds the Scottish Health Informatics Programme, provides core support for the Wellcome Trust Centre for Human Genetics in Oxford and funds the Wellcome Trust Case Control Consortium. This research was specifically funded by Diabetes UK (07/0003525) and the Wellcome Trust (084727/Z/08/Z, 085475/Z/08/Z and 085475/B/08/Z). Genotyping of the GoDARTS samples (sample 2) was also funded as part of the EU IMI-SUMMIT programme. K.Z. holds a Henry Wellcome Post-Doctoral Fellowship.

*Health ABC*: The Health ABC study was supported by NIA contracts N01AG62101, N01AG62103 and N01AG62106. The genome-wide association study was funded by NIA grant 1R01AG032098-01A1 to Wake Forest University Health Sciences and genotyping services were provided by the Center for Inherited Disease Research (CIDR). CIDR is fully funded through a federal contract from the National Institutes of Health to The Johns Hopkins University, contract number HHSN268200782096C. This research was supported in part by the Intramural Research Program of the NIH, National Institute on Aging.

*HVH*: This Heart and Vascular Health Study research was supported by NHLBI grants HL085251, HL073410, HL085251 and HL068986.

*MESA*: The Multi-Ethnic Study of Atherosclerosis (MESA) and MESA SNP Health Association Resource (SHARe) are conducted and supported by the National Heart, Lung, and Blood Institute (NHLBI) in collaboration with MESA investigators. Support is provided by grants and contracts N01 HC-95159 through N01-HC-95169 and RR-024156. Additional funding was supported in part by the Clinical Translational Science Institute grant UL1RR033176 and is now at the National Center for Advancing Translational Sciences, CTSI grant UL1TR000124. We thank the other investigators in the Pharmarcogenetics Working Group, the staff and the participants of the MESA study for their valuable contributions. A full list of participating MESA investigators and institutions can be found at [http://www.mesa-nhlbi.org](http://www.mesa-nhlbi.org).

*JUPITER*: Genetic analysis in JUPITER was supported by a research grant from AstraZeneca to D.C. and P.M.

*Rotterdam study*: The Rotterdam Study is supported by the Erasmus Medical Center and Erasmus University Rotterdam; the Netherlands Organization for Health Research and Development (ZonMw); the Research Institute for Diseases in the Elderly; the Ministry of Education, Culture and Science; the Ministry of Health Welfare and Sports; and the European Commission and Municipality of Rotterdam. This work was supported by the Netherlands Genomics Initiative (NGI) Netherlands Organization for Scietific Research (NOW; 050-060-810).

## Footnotes

## Contributor Information

Membership of Wellcome Trust Case Control Consortium:

[Peter Donnelly](https://pubmed.ncbi.nlm.nih.gov/?term=%22Donnelly%20P%22%5BAuthor%5D), [Ines Barroso](https://pubmed.ncbi.nlm.nih.gov/?term=%22Barroso%20I%22%5BAuthor%5D), [Jenefer M. Blackwell](https://pubmed.ncbi.nlm.nih.gov/?term=%22Blackwell%20JM%22%5BAuthor%5D), [Elvira Bramon](https://pubmed.ncbi.nlm.nih.gov/?term=%22Bramon%20E%22%5BAuthor%5D), [Matthew A. Brown](https://pubmed.ncbi.nlm.nih.gov/?term=%22Brown%20MA%22%5BAuthor%5D), [Juan P. Casas](https://pubmed.ncbi.nlm.nih.gov/?term=%22Casas%20JP%22%5BAuthor%5D), [Aiden Corvin](https://pubmed.ncbi.nlm.nih.gov/?term=%22Corvin%20A%22%5BAuthor%5D), [Panos Deloukas](https://pubmed.ncbi.nlm.nih.gov/?term=%22Deloukas%20P%22%5BAuthor%5D), [Audrey Duncanson](https://pubmed.ncbi.nlm.nih.gov/?term=%22Duncanson%20A%22%5BAuthor%5D), [Janusz Jankowski](https://pubmed.ncbi.nlm.nih.gov/?term=%22Jankowski%20J%22%5BAuthor%5D), [Hugh S. Markus](https://pubmed.ncbi.nlm.nih.gov/?term=%22Markus%20HS%22%5BAuthor%5D), [Christopher G. Mathew](https://pubmed.ncbi.nlm.nih.gov/?term=%22Mathew%20CG%22%5BAuthor%5D), [Colin N. A. Palmer](https://pubmed.ncbi.nlm.nih.gov/?term=%22Palmer%20CNA%22%5BAuthor%5D), [Robert Plomin](https://pubmed.ncbi.nlm.nih.gov/?term=%22Plomin%20R%22%5BAuthor%5D), [Anna Rautanen](https://pubmed.ncbi.nlm.nih.gov/?term=%22Rautanen%20A%22%5BAuthor%5D), [Stephen J. Sawcer](https://pubmed.ncbi.nlm.nih.gov/?term=%22Sawcer%20SJ%22%5BAuthor%5D), [Richard C. Trembath](https://pubmed.ncbi.nlm.nih.gov/?term=%22Trembath%20RC%22%5BAuthor%5D), [Ananth C. Viswanathan](https://pubmed.ncbi.nlm.nih.gov/?term=%22Viswanathan%20AC%22%5BAuthor%5D), and [Nicholas W. Wood](https://pubmed.ncbi.nlm.nih.gov/?term=%22Wood%20NW%22%5BAuthor%5D)

Data and Analysis Group:

[Chris C. A. Spencer](https://pubmed.ncbi.nlm.nih.gov/?term=%22Spencer%20CCA%22%5BAuthor%5D), [Gavin Band](https://pubmed.ncbi.nlm.nih.gov/?term=%22Band%20G%22%5BAuthor%5D), [Céline Bellenguez](https://pubmed.ncbi.nlm.nih.gov/?term=%22Bellenguez%20C%22%5BAuthor%5D), [Colin Freeman](https://pubmed.ncbi.nlm.nih.gov/?term=%22Freeman%20C%22%5BAuthor%5D), [Garrett Hellenthal](https://pubmed.ncbi.nlm.nih.gov/?term=%22Hellenthal%20G%22%5BAuthor%5D), [Eleni Giannoulatou](https://pubmed.ncbi.nlm.nih.gov/?term=%22Giannoulatou%20E%22%5BAuthor%5D), [Matti Pirinen](https://pubmed.ncbi.nlm.nih.gov/?term=%22Pirinen%20M%22%5BAuthor%5D), [Richard Pearson](https://pubmed.ncbi.nlm.nih.gov/?term=%22Pearson%20R%22%5BAuthor%5D), [Amy Strange](https://pubmed.ncbi.nlm.nih.gov/?term=%22Strange%20A%22%5BAuthor%5D), [Zhan Su](https://pubmed.ncbi.nlm.nih.gov/?term=%22Su%20Z%22%5BAuthor%5D), [Damjan Vukcevic](https://pubmed.ncbi.nlm.nih.gov/?term=%22Vukcevic%20D%22%5BAuthor%5D), and [Peter Donnelly](https://pubmed.ncbi.nlm.nih.gov/?term=%22Donnelly%20P%22%5BAuthor%5D)

DNA, Genotyping, Data QC and Informatics Group:

[Cordelia Langford](https://pubmed.ncbi.nlm.nih.gov/?term=%22Langford%20C%22%5BAuthor%5D), [Sarah E. Hunt](https://pubmed.ncbi.nlm.nih.gov/?term=%22Hunt%20SE%22%5BAuthor%5D), [Sarah Edkins](https://pubmed.ncbi.nlm.nih.gov/?term=%22Edkins%20S%22%5BAuthor%5D), [Rhian Gwilliam](https://pubmed.ncbi.nlm.nih.gov/?term=%22Gwilliam%20R%22%5BAuthor%5D), [Hannah Blackburn](https://pubmed.ncbi.nlm.nih.gov/?term=%22Blackburn%20H%22%5BAuthor%5D), [Suzannah J. Bumpstead](https://pubmed.ncbi.nlm.nih.gov/?term=%22Bumpstead%20SJ%22%5BAuthor%5D), [Serge Dronov](https://pubmed.ncbi.nlm.nih.gov/?term=%22Dronov%20S%22%5BAuthor%5D), [Matthew Gillman](https://pubmed.ncbi.nlm.nih.gov/?term=%22Gillman%20M%22%5BAuthor%5D), [Emma Gray](https://pubmed.ncbi.nlm.nih.gov/?term=%22Gray%20E%22%5BAuthor%5D), [Naomi Hammond](https://pubmed.ncbi.nlm.nih.gov/?term=%22Hammond%20N%22%5BAuthor%5D), [Alagurevathi Jayakumar](https://pubmed.ncbi.nlm.nih.gov/?term=%22Jayakumar%20A%22%5BAuthor%5D), [Owen T. McCann](https://pubmed.ncbi.nlm.nih.gov/?term=%22McCann%20OT%22%5BAuthor%5D), [Jennifer Liddle](https://pubmed.ncbi.nlm.nih.gov/?term=%22Liddle%20J%22%5BAuthor%5D), [Simon C. Potter](https://pubmed.ncbi.nlm.nih.gov/?term=%22Potter%20SC%22%5BAuthor%5D), [Radhi Ravindrarajah](https://pubmed.ncbi.nlm.nih.gov/?term=%22Ravindrarajah%20R%22%5BAuthor%5D), [Michelle Ricketts](https://pubmed.ncbi.nlm.nih.gov/?term=%22Ricketts%20M%22%5BAuthor%5D), [Matthew Waller](https://pubmed.ncbi.nlm.nih.gov/?term=%22Waller%20M%22%5BAuthor%5D), [Paul Weston](https://pubmed.ncbi.nlm.nih.gov/?term=%22Weston%20P%22%5BAuthor%5D), [Sara Widaa](https://pubmed.ncbi.nlm.nih.gov/?term=%22Widaa%20S%22%5BAuthor%5D), [Pamela Whittaker](https://pubmed.ncbi.nlm.nih.gov/?term=%22Whittaker%20P%22%5BAuthor%5D), [Ines Barroso](https://pubmed.ncbi.nlm.nih.gov/?term=%22Barroso%20I%22%5BAuthor%5D), and [Panos Deloukas](https://pubmed.ncbi.nlm.nih.gov/?term=%22Deloukas%20P%22%5BAuthor%5D)

Publications Committee:

[Christopher G. Mathew](https://pubmed.ncbi.nlm.nih.gov/?term=%22Mathew%20CG%22%5BAuthor%5D), [Jenefer M. Blackwell](https://pubmed.ncbi.nlm.nih.gov/?term=%22Blackwell%20JM%22%5BAuthor%5D), [Matthew A. Brown](https://pubmed.ncbi.nlm.nih.gov/?term=%22Brown%20MA%22%5BAuthor%5D), [Aiden Corvin](https://pubmed.ncbi.nlm.nih.gov/?term=%22Corvin%20A%22%5BAuthor%5D), [Mark I. McCarthy](https://pubmed.ncbi.nlm.nih.gov/?term=%22McCarthy%20MI%22%5BAuthor%5D), and [Chris C. A. Spencer](https://pubmed.ncbi.nlm.nih.gov/?term=%22Spencer%20CCA%22%5BAuthor%5D)

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Davidson M. H. & Toth P. P. Comparative effects of lipid-lowering therapies. Prog. Cardiovasc. Dis. 47, 73–104 (2004).  [DOI](https://doi.org/10.1016/j.pcad.2004.04.007) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15586350/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Prog.%20Cardiovasc.%20Dis.&title=Comparative%20effects%20of%20lipid-lowering%20therapies&author=M.%20H.%20Davidson&author=P.%20P.%20Toth&volume=47&publication_year=2004&pages=73-104&pmid=15586350&doi=10.1016/j.pcad.2004.04.007&)

2. Baigent C. et al. Efficacy and safety of more intensive lowering of LDL cholesterol: a meta-analysis of data from 170,000 participants in 26 randomised trials. Lancet 376, 1670–1681 (2010).  [DOI](https://doi.org/10.1016/S0140-6736(10)61350-5) | [PMC free article](/articles/PMC2988224/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21067804/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Efficacy%20and%20safety%20of%20more%20intensive%20lowering%20of%20LDL%20cholesterol:%20a%20meta-analysis%20of%20data%20from%20170,000%20participants%20in%2026%20randomised%20trials&author=C.%20Baigent&volume=376&publication_year=2010&pages=1670-1681&pmid=21067804&doi=10.1016/S0140-6736(10)61350-5&)

3. Mangravite L. M., Thorn C. F. & Krauss R. M. Clinical implications of pharmacogenomics of statin treatment. Pharmacogenomics J. 6, 360–374 (2006).  [DOI](https://doi.org/10.1038/sj.tpj.6500384) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16550210/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J.&title=Clinical%20implications%20of%20pharmacogenomics%20of%20statin%20treatment&author=L.%20M.%20Mangravite&author=C.%20F.%20Thorn&author=R.%20M.%20Krauss&volume=6&publication_year=2006&pages=360-374&pmid=16550210&doi=10.1038/sj.tpj.6500384&)

4. Postmus I. et al. Pharmacogenetics of statins: achievements, whole-genome analyses and future perspectives. Pharmacogenomics 13, 831–840 (2012).  [DOI](https://doi.org/10.2217/pgs.12.25) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22594514/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Pharmacogenetics%20of%20statins:%20achievements,%20whole-genome%20analyses%20and%20future%20perspectives&author=I.%20Postmus&volume=13&publication_year=2012&pages=831-840&pmid=22594514&doi=10.2217/pgs.12.25&)

5. Chasman D. I. et al. Pharmacogenetic study of statin therapy and cholesterol reduction. JAMA 291, 2821–2827 (2004).  [DOI](https://doi.org/10.1001/jama.291.23.2821) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15199031/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA&title=Pharmacogenetic%20study%20of%20statin%20therapy%20and%20cholesterol%20reduction&author=D.%20I.%20Chasman&volume=291&publication_year=2004&pages=2821-2827&pmid=15199031&doi=10.1001/jama.291.23.2821&)

6. Hopewell J. C. et al. Impact of common genetic variation on response to simvastatin therapy among 18 705 participants in the Heart Protection Study. Eur. Heart J. 34, 982–992 (2013).  [DOI](https://doi.org/10.1093/eurheartj/ehs344) | [PMC free article](/articles/PMC3612775/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23100282/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur.%20Heart%20J.&title=Impact%20of%20common%20genetic%20variation%20on%20response%20to%20simvastatin%20therapy%20among%2018%20705%20participants%20in%20the%20Heart%20Protection%20Study&author=J.%20C.%20Hopewell&volume=34&publication_year=2013&pages=982-992&pmid=23100282&doi=10.1093/eurheartj/ehs344&)

7. Chasman D. I. et al. Genetic determinants of statin-induced low-density lipoprotein cholesterol reduction: the Justification for the Use of Statins in Prevention: an Intervention Trial Evaluating Rosuvastatin (JUPITER) trial. Circ. Cardiovasc. Genet. 5, 257–264 (2012).  [DOI](https://doi.org/10.1161/CIRCGENETICS.111.961144) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22331829/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circ.%20Cardiovasc.%20Genet.&title=Genetic%20determinants%20of%20statin-induced%20low-density%20lipoprotein%20cholesterol%20reduction:%20the%20Justification%20for%20the%20Use%20of%20Statins%20in%20Prevention:%20an%20Intervention%20Trial%20Evaluating%20Rosuvastatin%20(JUPITER)%20trial&author=D.%20I.%20Chasman&volume=5&publication_year=2012&pages=257-264&pmid=22331829&doi=10.1161/CIRCGENETICS.111.961144&)

8. Deshmukh H. A. et al. Genome-wide association study of genetic determinants of LDL-c response to atorvastatin therapy: importance of Lp(a). J. Lipid Res. 53, 1000–1011 (2012).  [DOI](https://doi.org/10.1194/jlr.P021113) | [PMC free article](/articles/PMC3329377/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22368281/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Lipid%20Res.&title=Genome-wide%20association%20study%20of%20genetic%20determinants%20of%20LDL-c%20response%20to%20atorvastatin%20therapy:%20importance%20of%20Lp(a)&author=H.%20A.%20Deshmukh&volume=53&publication_year=2012&pages=1000-1011&pmid=22368281&doi=10.1194/jlr.P021113&)

9. Barber M. J. et al. Genome-wide association of lipid-lowering response to statins in combined study populations. PLoS ONE 5, e9763 (2010).  [DOI](https://doi.org/10.1371/journal.pone.0009763) | [PMC free article](/articles/PMC2842298/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20339536/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&title=Genome-wide%20association%20of%20lipid-lowering%20response%20to%20statins%20in%20combined%20study%20populations&author=M.%20J.%20Barber&volume=5&publication_year=2010&pages=e9763&pmid=20339536&doi=10.1371/journal.pone.0009763&)

10. Thompson J. F. et al. Comprehensive whole-genome and candidate gene analysis for response to statin therapy in the Treating to New Targets (TNT) cohort. Circ. Cardiovasc. Genet. 2, 173–181 (2009).  [DOI](https://doi.org/10.1161/CIRCGENETICS.108.818062) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20031582/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circ.%20Cardiovasc.%20Genet.&title=Comprehensive%20whole-genome%20and%20candidate%20gene%20analysis%20for%20response%20to%20statin%20therapy%20in%20the%20Treating%20to%20New%20Targets%20(TNT)%20cohort&author=J.%20F.%20Thompson&volume=2&publication_year=2009&pages=173-181&pmid=20031582&doi=10.1161/CIRCGENETICS.108.818062&)

11. Hindorff L. A. et al. Potential etiologic and functional implications of genome-wide association loci for human diseases and traits. Proc. Natl Acad. Sci. USA 106, 9362–9367 (2009).  [DOI](https://doi.org/10.1073/pnas.0903103106) | [PMC free article](/articles/PMC2687147/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19474294/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl%20Acad.%20Sci.%20USA&title=Potential%20etiologic%20and%20functional%20implications%20of%20genome-wide%20association%20loci%20for%20human%20diseases%20and%20traits&author=L.%20A.%20Hindorff&volume=106&publication_year=2009&pages=9362-9367&pmid=19474294&doi=10.1073/pnas.0903103106&)

12. Lamb J. et al. The Connectivity Map: using gene-expression signatures to connect small molecules, genes, and disease. Science 313, 1929–1935 (2006).  [DOI](https://doi.org/10.1126/science.1132939) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17008526/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=The%20Connectivity%20Map:%20using%20gene-expression%20signatures%20to%20connect%20small%20molecules,%20genes,%20and%20disease&author=J.%20Lamb&volume=313&publication_year=2006&pages=1929-1935&pmid=17008526&doi=10.1126/science.1132939&)

13. Brennan R. J., Nikolskya T. & Bureeva S. Network and pathway analysis of compound-protein interactions. Methods Mol. Biol. 575, 225–247 (2009).  [DOI](https://doi.org/10.1007/978-1-60761-274-2_10) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19727618/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Methods%20Mol.%20Biol.&title=Network%20and%20pathway%20analysis%20of%20compound-protein%20interactions&author=R.%20J.%20Brennan&author=T.%20Nikolskya&author=S.%20Bureeva&volume=575&publication_year=2009&pages=225-247&pmid=19727618&doi=10.1007/978-1-60761-274-2_10&)

14. Ken-Dror G., Talmud P. J., Humphries S. E. & Drenos F. APOE/C1/C4/C2 gene cluster genotypes, haplotypes and lipid levels in prospective coronary heart disease risk among UK healthy men. Mol. Med. 16, 389–399 (2010).  [DOI](https://doi.org/10.2119/molmed.2010.00044) | [PMC free article](/articles/PMC2935949/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20498921/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol.%20Med.&title=APOE/C1/C4/C2%20gene%20cluster%20genotypes,%20haplotypes%20and%20lipid%20levels%20in%20prospective%20coronary%20heart%20disease%20risk%20among%20UK%20healthy%20men&author=G.%20Ken-Dror&author=P.%20J.%20Talmud&author=S.%20E.%20Humphries&author=F.%20Drenos&volume=16&publication_year=2010&pages=389-399&pmid=20498921&doi=10.2119/molmed.2010.00044&)

15. Lanktree M. B., Anand S. S., Yusuf S. & Hegele R. A. Comprehensive analysis of genomic variation in the LPA locus and its relationship to plasma lipoprotein(a) in South Asians, Chinese, and European Caucasians. Circ. Cardiovasc. Genet. 3, 39–46 (2010).  [DOI](https://doi.org/10.1161/CIRCGENETICS.109.907642) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20160194/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circ.%20Cardiovasc.%20Genet.&title=Comprehensive%20analysis%20of%20genomic%20variation%20in%20the%20LPA%20locus%20and%20its%20relationship%20to%20plasma%20lipoprotein(a)%20in%20South%20Asians,%20Chinese,%20and%20European%20Caucasians&author=M.%20B.%20Lanktree&author=S.%20S.%20Anand&author=S.%20Yusuf&author=R.%20A.%20Hegele&volume=3&publication_year=2010&pages=39-46&pmid=20160194&doi=10.1161/CIRCGENETICS.109.907642&)

16. Qi Q., Workalemahu T., Zhang C., Hu F. B. & Qi L. Genetic variants, plasma lipoprotein(a) levels, and risk of cardiovascular morbidity and mortality among two prospective cohorts of type 2 diabetes. Eur. Heart J. 33, 325–334 (2012).  [DOI](https://doi.org/10.1093/eurheartj/ehr350) | [PMC free article](/articles/PMC3270044/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21900290/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur.%20Heart%20J.&title=Genetic%20variants,%20plasma%20lipoprotein(a)%20levels,%20and%20risk%20of%20cardiovascular%20morbidity%20and%20mortality%20among%20two%20prospective%20cohorts%20of%20type%202%20diabetes&author=Q.%20Qi&author=T.%20Workalemahu&author=C.%20Zhang&author=F.%20B.%20Hu&author=L.%20Qi&volume=33&publication_year=2012&pages=325-334&pmid=21900290&doi=10.1093/eurheartj/ehr350&)

17. Scanu A. M. & Hinman J. Issues concerning the monitoring of statin therapy in hypercholesterolemic subjects with high plasma lipoprotein(a) levels. Lipids 37, 439–444 (2002).  [DOI](https://doi.org/10.1007/s11745-002-0915-1) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12056584/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lipids&title=Issues%20concerning%20the%20monitoring%20of%20statin%20therapy%20in%20hypercholesterolemic%20subjects%20with%20high%20plasma%20lipoprotein(a)%20levels&author=A.%20M.%20Scanu&author=J.%20Hinman&volume=37&publication_year=2002&pages=439-444&pmid=12056584&doi=10.1007/s11745-002-0915-1&)

18. Musunuru K. et al. From noncoding variant to phenotype via SORT1 at the 1p13 cholesterol locus. Nature 466, 714–719 (2010).  [DOI](https://doi.org/10.1038/nature09266) | [PMC free article](/articles/PMC3062476/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20686566/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=From%20noncoding%20variant%20to%20phenotype%20via%20SORT1%20at%20the%201p13%20cholesterol%20locus&author=K.%20Musunuru&volume=466&publication_year=2010&pages=714-719&pmid=20686566&doi=10.1038/nature09266&)

19. Strong A. et al. Hepatic sortilin regulates both apolipoprotein B secretion and LDL catabolism. J. Clin. Invest. 122, 2807–2816 (2012).  [DOI](https://doi.org/10.1172/JCI63563) | [PMC free article](/articles/PMC3408750/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22751103/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Invest.&title=Hepatic%20sortilin%20regulates%20both%20apolipoprotein%20B%20secretion%20and%20LDL%20catabolism&author=A.%20Strong&volume=122&publication_year=2012&pages=2807-2816&pmid=22751103&doi=10.1172/JCI63563&)

20. Campos H., Arnold K. S., Balestra M. E., Innerarity T. L. & Krauss R. M. Differences in receptor binding of LDL subfractions. Arterioscler. Thromb. Vasc. Biol. 16, 794–801 (1996).  [DOI](https://doi.org/10.1161/01.atv.16.6.794) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8640407/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arterioscler.%20Thromb.%20Vasc.%20Biol.&title=Differences%20in%20receptor%20binding%20of%20LDL%20subfractions&author=H.%20Campos&author=K.%20S.%20Arnold&author=M.%20E.%20Balestra&author=T.%20L.%20Innerarity&author=R.%20M.%20Krauss&volume=16&publication_year=1996&pages=794-801&pmid=8640407&doi=10.1161/01.atv.16.6.794&)

21. Williams P. T., Zhao X. Q., Marcovina S. M., Brown B. G. & Krauss R. M. Levels of cholesterol in small LDL particles predict atherosclerosis progression and incident CHD in the HDL-Atherosclerosis Treatment Study (HATS). PLoS ONE 8, e56782 (2013).  [DOI](https://doi.org/10.1371/journal.pone.0056782) | [PMC free article](/articles/PMC3584097/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23460815/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&title=Levels%20of%20cholesterol%20in%20small%20LDL%20particles%20predict%20atherosclerosis%20progression%20and%20incident%20CHD%20in%20the%20HDL-Atherosclerosis%20Treatment%20Study%20(HATS)&author=P.%20T.%20Williams&author=X.%20Q.%20Zhao&author=S.%20M.%20Marcovina&author=B.%20G.%20Brown&author=R.%20M.%20Krauss&volume=8&publication_year=2013&pages=e56782&pmid=23460815&doi=10.1371/journal.pone.0056782&)

22. Choi Y. J. et al. Effects of the PPAR-delta agonist MBX-8025 on atherogenic dyslipidemia. Atherosclerosis 220, 470–476 (2012).  [DOI](https://doi.org/10.1016/j.atherosclerosis.2011.10.029) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22169113/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Atherosclerosis&title=Effects%20of%20the%20PPAR-delta%20agonist%20MBX-8025%20on%20atherogenic%20dyslipidemia&author=Y.%20J.%20Choi&volume=220&publication_year=2012&pages=470-476&pmid=22169113&doi=10.1016/j.atherosclerosis.2011.10.029&)

23. Konig J., Seithel A., Gradhand U. & Fromm M. F. Pharmacogenomics of human OATP transporters. Naunyn Schmiedebergs Arch. Pharmacol. 372, 432–443 (2006).  [DOI](https://doi.org/10.1007/s00210-006-0040-y) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16525793/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Naunyn%20Schmiedebergs%20Arch.%20Pharmacol.&title=Pharmacogenomics%20of%20human%20OATP%20transporters&author=J.%20Konig&author=A.%20Seithel&author=U.%20Gradhand&author=M.%20F.%20Fromm&volume=372&publication_year=2006&pages=432-443&pmid=16525793&doi=10.1007/s00210-006-0040-y&)

24. Wilke R. A. et al. The clinical pharmacogenomics implementation consortium: CPIC guideline for SLCO1B1 and simvastatin-induced myopathy. Clin. Pharmacol. Ther. 92, 112–117 (2012).  [DOI](https://doi.org/10.1038/clpt.2012.57) | [PMC free article](/articles/PMC3384438/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22617227/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=The%20clinical%20pharmacogenomics%20implementation%20consortium:%20CPIC%20guideline%20for%20SLCO1B1%20and%20simvastatin-induced%20myopathy&author=R.%20A.%20Wilke&volume=92&publication_year=2012&pages=112-117&pmid=22617227&doi=10.1038/clpt.2012.57&)

25. Link E. et al. SLCO1B1 variants and statin-induced myopathy-a genomewide study. New Engl. J. Med. 359, 789–799 (2008).  [DOI](https://doi.org/10.1056/NEJMoa0801936) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18650507/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=New%20Engl.%20J.%20Med.&title=SLCO1B1%20variants%20and%20statin-induced%20myopathy-a%20genomewide%20study&author=E.%20Link&volume=359&publication_year=2008&pages=789-799&pmid=18650507&doi=10.1056/NEJMoa0801936&)

26. Donnelly L. A. et al. Common nonsynonymous substitutions in SLCO1B1 predispose to statin intolerance in routinely treated individuals with type 2 diabetes: a go-DARTS study. Clin. Pharmacol. Ther. 89, 210–216 (2011).  [DOI](https://doi.org/10.1038/clpt.2010.255) | [PMC free article](/articles/PMC3353487/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21178985/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Common%20nonsynonymous%20substitutions%20in%20SLCO1B1%20predispose%20to%20statin%20intolerance%20in%20routinely%20treated%20individuals%20with%20type%202%20diabetes:%20a%20go-DARTS%20study&author=L.%20A.%20Donnelly&volume=89&publication_year=2011&pages=210-216&pmid=21178985&doi=10.1038/clpt.2010.255&)

27. Niemi M., Pasanen M. K. & Neuvonen P. J. Organic anion transporting polypeptide 1B1: a genetically polymorphic transporter of major importance for hepatic drug uptake. Pharmacol. Rev. 63, 157–181 (2011).  [DOI](https://doi.org/10.1124/pr.110.002857) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21245207/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacol.%20Rev.&title=Organic%20anion%20transporting%20polypeptide%201B1:%20a%20genetically%20polymorphic%20transporter%20of%20major%20importance%20for%20hepatic%20drug%20uptake&author=M.%20Niemi&author=M.%20K.%20Pasanen&author=P.%20J.%20Neuvonen&volume=63&publication_year=2011&pages=157-181&pmid=21245207&doi=10.1124/pr.110.002857&)

28. Teslovich T. M. et al. Biological, clinical and population relevance of 95 loci for blood lipids. Nature 466, 707–713 (2010).  [DOI](https://doi.org/10.1038/nature09270) | [PMC free article](/articles/PMC3039276/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20686565/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Biological,%20clinical%20and%20population%20relevance%20of%2095%20loci%20for%20blood%20lipids&author=T.%20M.%20Teslovich&volume=466&publication_year=2010&pages=707-713&pmid=20686565&doi=10.1038/nature09270&)

29. Dreon D. M., Fernstrom H. A., Williams P. T. & Krauss R. M. Reduced LDL particle size in children consuming a very-low-fat diet is related to parental LDL-subclass patterns. Am. J. Clin. Nutr. 71, 1611–1616 (2000).  [DOI](https://doi.org/10.1093/ajcn/71.6.1611) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10837306/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Clin.%20Nutr.&title=Reduced%20LDL%20particle%20size%20in%20children%20consuming%20a%20very-low-fat%20diet%20is%20related%20to%20parental%20LDL-subclass%20patterns&author=D.%20M.%20Dreon&author=H.%20A.%20Fernstrom&author=P.%20T.%20Williams&author=R.%20M.%20Krauss&volume=71&publication_year=2000&pages=1611-1616&pmid=10837306&doi=10.1093/ajcn/71.6.1611&)

30. Goring H. H. et al. Discovery of expression QTLs using large-scale transcriptional profiling in human lymphocytes. Nat. Genet. 39, 1208–1216 (2007).  [DOI](https://doi.org/10.1038/ng2119) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17873875/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet.&title=Discovery%20of%20expression%20QTLs%20using%20large-scale%20transcriptional%20profiling%20in%20human%20lymphocytes&author=H.%20H.%20Goring&volume=39&publication_year=2007&pages=1208-1216&pmid=17873875&doi=10.1038/ng2119&)

31. Idaghdour Y. et al. Geographical genomics of human leukocyte gene expression variation in southern Morocco. Nat. Genet. 42, 62–67 (2010).  [DOI](https://doi.org/10.1038/ng.495) | [PMC free article](/articles/PMC2798927/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19966804/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet.&title=Geographical%20genomics%20of%20human%20leukocyte%20gene%20expression%20variation%20in%20southern%20Morocco&author=Y.%20Idaghdour&volume=42&publication_year=2010&pages=62-67&pmid=19966804&doi=10.1038/ng.495&)

32. Heap G. A. et al. Complex nature of SNP genotype effects on gene expression in primary human leucocytes. BMC Med. Genomics 2, 1 (2009).  [DOI](https://doi.org/10.1186/1755-8794-2-1) | [PMC free article](/articles/PMC2628677/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19128478/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Med.%20Genomics&title=Complex%20nature%20of%20SNP%20genotype%20effects%20on%20gene%20expression%20in%20primary%20human%20leucocytes&author=G.%20A.%20Heap&volume=2&publication_year=2009&pages=1&pmid=19128478&doi=10.1186/1755-8794-2-1&)

33. Emilsson V. et al. Genetics of gene expression and its effect on disease. Nature 452, 423–428 (2008).  [DOI](https://doi.org/10.1038/nature06758) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18344981/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Genetics%20of%20gene%20expression%20and%20its%20effect%20on%20disease&author=V.%20Emilsson&volume=452&publication_year=2008&pages=423-428&pmid=18344981&doi=10.1038/nature06758&)

34. Fehrmann R. S. et al. Trans-eQTLs reveal that independent genetic variants associated with a complex phenotype converge on intermediate genes, with a major role for the HLA. PLoS Genet. 7, e1002197 (2011).  [DOI](https://doi.org/10.1371/journal.pgen.1002197) | [PMC free article](/articles/PMC3150446/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21829388/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet.&title=Trans-eQTLs%20reveal%20that%20independent%20genetic%20variants%20associated%20with%20a%20complex%20phenotype%20converge%20on%20intermediate%20genes,%20with%20a%20major%20role%20for%20the%20HLA&author=R.%20S.%20Fehrmann&volume=7&publication_year=2011&pages=e1002197&pmid=21829388&doi=10.1371/journal.pgen.1002197&)

35. Mehta D. et al. Impact of common regulatory single-nucleotide variants on gene expression profiles in whole blood. Eur. J. Hum. Genet. 21, 48–54 (2013).  [DOI](https://doi.org/10.1038/ejhg.2012.106) | [PMC free article](/articles/PMC3522194/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22692066/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur.%20J.%20Hum.%20Genet.&title=Impact%20of%20common%20regulatory%20single-nucleotide%20variants%20on%20gene%20expression%20profiles%20in%20whole%20blood&author=D.%20Mehta&volume=21&publication_year=2013&pages=48-54&pmid=22692066&doi=10.1038/ejhg.2012.106&)

36. Sasayama D. et al. Identification of single nucleotide polymorphisms regulating peripheral blood mRNA expression with genome-wide significance: an eQTL study in the Japanese population. PLoS ONE 8, e54967 (2013).  [DOI](https://doi.org/10.1371/journal.pone.0054967) | [PMC free article](/articles/PMC3554677/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23359819/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&title=Identification%20of%20single%20nucleotide%20polymorphisms%20regulating%20peripheral%20blood%20mRNA%20expression%20with%20genome-wide%20significance:%20an%20eQTL%20study%20in%20the%20Japanese%20population&author=D.%20Sasayama&volume=8&publication_year=2013&pages=e54967&pmid=23359819&doi=10.1371/journal.pone.0054967&)

37. Dixon A. L. et al. A genome-wide association study of global gene expression. Nat. Genet. 39, 1202–1207 (2007).  [DOI](https://doi.org/10.1038/ng2109) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17873877/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet.&title=A%20genome-wide%20association%20study%20of%20global%20gene%20expression&author=A.%20L.%20Dixon&volume=39&publication_year=2007&pages=1202-1207&pmid=17873877&doi=10.1038/ng2109&)

38. Liang L. et al. A cross-platform analysis of 14,177 expression quantitative trait loci derived from lymphoblastoid cell lines. Genome Res. 23, 716–726 (2013).  [DOI](https://doi.org/10.1101/gr.142521.112) | [PMC free article](/articles/PMC3613588/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23345460/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Res.&title=A%20cross-platform%20analysis%20of%2014,177%20expression%20quantitative%20trait%20loci%20derived%20from%20lymphoblastoid%20cell%20lines&author=L.%20Liang&volume=23&publication_year=2013&pages=716-726&pmid=23345460&doi=10.1101/gr.142521.112&)

39. Stranger B. E. et al. Population genomics of human gene expression. Nat. Genet. 39, 1217–1224 (2007).  [DOI](https://doi.org/10.1038/ng2142) | [PMC free article](/articles/PMC2683249/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17873874/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet.&title=Population%20genomics%20of%20human%20gene%20expression&author=B.%20E.%20Stranger&volume=39&publication_year=2007&pages=1217-1224&pmid=17873874&doi=10.1038/ng2142&)

40. Kwan T. et al. Genome-wide analysis of transcript isoform variation in humans. Nat. Genet. 40, 225–231 (2008).  [DOI](https://doi.org/10.1038/ng.2007.57) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18193047/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet.&title=Genome-wide%20analysis%20of%20transcript%20isoform%20variation%20in%20humans&author=T.%20Kwan&volume=40&publication_year=2008&pages=225-231&pmid=18193047&doi=10.1038/ng.2007.57&)

41. Dimas A. S. et al. Common regulatory variation impacts gene expression in a cell type-dependent manner. Science 325, 1246–1250 (2009).  [DOI](https://doi.org/10.1126/science.1174148) | [PMC free article](/articles/PMC2867218/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19644074/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=Common%20regulatory%20variation%20impacts%20gene%20expression%20in%20a%20cell%20type-dependent%20manner&author=A.%20S.%20Dimas&volume=325&publication_year=2009&pages=1246-1250&pmid=19644074&doi=10.1126/science.1174148&)

42. Cusanovich D. A. et al. The combination of a genome-wide association study of lymphocyte count and analysis of gene expression data reveals novel asthma candidate genes. Hum. Mol. Genet. 21, 2111–2123 (2012).  [DOI](https://doi.org/10.1093/hmg/dds021) | [PMC free article](/articles/PMC3315207/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22286170/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum.%20Mol.%20Genet.&title=The%20combination%20of%20a%20genome-wide%20association%20study%20of%20lymphocyte%20count%20and%20analysis%20of%20gene%20expression%20data%20reveals%20novel%20asthma%20candidate%20genes&author=D.%20A.%20Cusanovich&volume=21&publication_year=2012&pages=2111-2123&pmid=22286170&doi=10.1093/hmg/dds021&)

43. Grundberg E. et al. Mapping cis- and trans-regulatory effects across multiple tissues in twins. Nat. Genet. 44, 1084–1089 (2012).  [DOI](https://doi.org/10.1038/ng.2394) | [PMC free article](/articles/PMC3784328/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22941192/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet.&title=Mapping%20cis-%20and%20trans-regulatory%20effects%20across%20multiple%20tissues%20in%20twins&author=E.%20Grundberg&volume=44&publication_year=2012&pages=1084-1089&pmid=22941192&doi=10.1038/ng.2394&)

44. Fairfax B. P. et al. Genetics of gene expression in primary immune cells identifies cell type-specific master regulators and roles of HLA alleles. Nat. Genet. 44, 502–510 (2012).  [DOI](https://doi.org/10.1038/ng.2205) | [PMC free article](/articles/PMC3437404/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22446964/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet.&title=Genetics%20of%20gene%20expression%20in%20primary%20immune%20cells%20identifies%20cell%20type-specific%20master%20regulators%20and%20roles%20of%20HLA%20alleles&author=B.%20P.%20Fairfax&volume=44&publication_year=2012&pages=502-510&pmid=22446964&doi=10.1038/ng.2205&)

45. Murphy A. et al. Mapping of numerous disease-associated expression polymorphisms in primary peripheral blood CD4+ lymphocytes. Hum. Mol. Genet. 19, 4745–4757 (2010).  [DOI](https://doi.org/10.1093/hmg/ddq392) | [PMC free article](/articles/PMC2972694/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20833654/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum.%20Mol.%20Genet.&title=Mapping%20of%20numerous%20disease-associated%20expression%20polymorphisms%20in%20primary%20peripheral%20blood%20CD4+%20lymphocytes&author=A.%20Murphy&volume=19&publication_year=2010&pages=4745-4757&pmid=20833654&doi=10.1093/hmg/ddq392&)

46. Heinzen E. L. et al. Tissue-specific genetic control of splicing: implications for the study of complex traits. PLoS Biol. 6, e1 (2008).  [DOI](https://doi.org/10.1371/journal.pbio.1000001) | [PMC free article](/articles/PMC2605930/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19222302/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Biol.&title=Tissue-specific%20genetic%20control%20of%20splicing:%20implications%20for%20the%20study%20of%20complex%20traits&author=E.%20L.%20Heinzen&volume=6&publication_year=2008&pages=e1&pmid=19222302&doi=10.1371/journal.pbio.1000001&)

47. Zeller T. et al. Genetics and beyond-the transcriptome of human monocytes and disease susceptibility. PLoS ONE 5, e10693 (2010).  [DOI](https://doi.org/10.1371/journal.pone.0010693) | [PMC free article](/articles/PMC2872668/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20502693/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&title=Genetics%20and%20beyond-the%20transcriptome%20of%20human%20monocytes%20and%20disease%20susceptibility&author=T.%20Zeller&volume=5&publication_year=2010&pages=e10693&pmid=20502693&doi=10.1371/journal.pone.0010693&)

48. Barreiro L. B. et al. Deciphering the genetic architecture of variation in the immune response to Mycobacterium tuberculosis infection. Proc. Natl Acad. Sci. USA 109, 1204–1209 (2012).  [DOI](https://doi.org/10.1073/pnas.1115761109) | [PMC free article](/articles/PMC3268270/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22233810/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl%20Acad.%20Sci.%20USA&title=Deciphering%20the%20genetic%20architecture%20of%20variation%20in%20the%20immune%20response%20to%20Mycobacterium%20tuberculosis%20infection&author=L.%20B.%20Barreiro&volume=109&publication_year=2012&pages=1204-1209&pmid=22233810&doi=10.1073/pnas.1115761109&)

49. Greenawalt D. M. et al. A survey of the genetics of stomach, liver, and adipose gene expression from a morbidly obese cohort. Genome Res. 21, 1008–1016 (2011).  [DOI](https://doi.org/10.1101/gr.112821.110) | [PMC free article](/articles/PMC3129244/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21602305/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Res.&title=A%20survey%20of%20the%20genetics%20of%20stomach,%20liver,%20and%20adipose%20gene%20expression%20from%20a%20morbidly%20obese%20cohort&author=D.%20M.%20Greenawalt&volume=21&publication_year=2011&pages=1008-1016&pmid=21602305&doi=10.1101/gr.112821.110&)

50. Kompass K. S. & Witte J. S. Co-regulatory expression quantitative trait loci mapping: method and application to endometrial cancer. BMC Med. Genomics 4, 6 (2011).  [DOI](https://doi.org/10.1186/1755-8794-4-6) | [PMC free article](/articles/PMC3032645/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21226949/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Med.%20Genomics&title=Co-regulatory%20expression%20quantitative%20trait%20loci%20mapping:%20method%20and%20application%20to%20endometrial%20cancer&author=K.%20S.%20Kompass&author=J.%20S.%20Witte&volume=4&publication_year=2011&pages=6&pmid=21226949&doi=10.1186/1755-8794-4-6&)

51. Li Q. et al. Integrative eQTL-based analyses reveal the biology of breast cancer risk loci. Cell 152, 633–641 (2013).  [DOI](https://doi.org/10.1016/j.cell.2012.12.034) | [PMC free article](/articles/PMC4165609/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23374354/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell&title=Integrative%20eQTL-based%20analyses%20reveal%20the%20biology%20of%20breast%20cancer%20risk%20loci&author=Q.%20Li&volume=152&publication_year=2013&pages=633-641&pmid=23374354&doi=10.1016/j.cell.2012.12.034&)

52. Webster J. A. et al. Genetic control of human brain transcript expression in Alzheimer disease. Am. J. Hum. Genet. 84, 445–458 (2009).  [DOI](https://doi.org/10.1016/j.ajhg.2009.03.011) | [PMC free article](/articles/PMC2667989/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19361613/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Hum.%20Genet.&title=Genetic%20control%20of%20human%20brain%20transcript%20expression%20in%20Alzheimer%20disease&author=J.%20A.%20Webster&volume=84&publication_year=2009&pages=445-458&pmid=19361613&doi=10.1016/j.ajhg.2009.03.011&)

53. Zou F. et al. Brain expression genome-wide association study (eGWAS) identifies human disease-associated variants. PLoS Genet. 8, e1002707 (2012).  [DOI](https://doi.org/10.1371/journal.pgen.1002707) | [PMC free article](/articles/PMC3369937/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22685416/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet.&title=Brain%20expression%20genome-wide%20association%20study%20(eGWAS)%20identifies%20human%20disease-associated%20variants&author=F.%20Zou&volume=8&publication_year=2012&pages=e1002707&pmid=22685416&doi=10.1371/journal.pgen.1002707&)

54. Colantuoni C. et al. Temporal dynamics and genetic control of transcription in the human prefrontal cortex. Nature 478, 519–523 (2011).  [DOI](https://doi.org/10.1038/nature10524) | [PMC free article](/articles/PMC3510670/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22031444/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Temporal%20dynamics%20and%20genetic%20control%20of%20transcription%20in%20the%20human%20prefrontal%20cortex&author=C.%20Colantuoni&volume=478&publication_year=2011&pages=519-523&pmid=22031444&doi=10.1038/nature10524&)

55. Liu C. et al. Whole-genome association mapping of gene expression in the human prefrontal cortex. Mol. Psychiatry 15, 779–784 (2010).  [DOI](https://doi.org/10.1038/mp.2009.128) | [PMC free article](/articles/PMC3057235/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20351726/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol.%20Psychiatry&title=Whole-genome%20association%20mapping%20of%20gene%20expression%20in%20the%20human%20prefrontal%20cortex&author=C.%20Liu&volume=15&publication_year=2010&pages=779-784&pmid=20351726&doi=10.1038/mp.2009.128&)

56. Gibbs J. R. et al. Abundant quantitative trait loci exist for DNA methylation and gene expression in human brain. PLoS Genet. 6, e1000952 (2010).  [DOI](https://doi.org/10.1371/journal.pgen.1000952) | [PMC free article](/articles/PMC2869317/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20485568/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet.&title=Abundant%20quantitative%20trait%20loci%20exist%20for%20DNA%20methylation%20and%20gene%20expression%20in%20human%20brain&author=J.%20R.%20Gibbs&volume=6&publication_year=2010&pages=e1000952&pmid=20485568&doi=10.1371/journal.pgen.1000952&)

57. Zhang B1. et al. Integrated systems approach identifies genetic nodes and networks in late-onset Alzheimer's disease. Cell 153, 707–720 (2013).  [DOI](https://doi.org/10.1016/j.cell.2013.03.030) | [PMC free article](/articles/PMC3677161/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23622250/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell&title=Integrated%20systems%20approach%20identifies%20genetic%20nodes%20and%20networks%20in%20late-onset%20Alzheimer's%20disease&author=B1.%20Zhang&volume=153&publication_year=2013&pages=707-720&pmid=23622250&doi=10.1016/j.cell.2013.03.030&)

58. Schadt E. E. et al. Mapping the genetic architecture of gene expression in human liver. PLoS Biol. 6, e107 (2008).  [DOI](https://doi.org/10.1371/journal.pbio.0060107) | [PMC free article](/articles/PMC2365981/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18462017/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Biol.&title=Mapping%20the%20genetic%20architecture%20of%20gene%20expression%20in%20human%20liver&author=E.%20E.%20Schadt&volume=6&publication_year=2008&pages=e107&pmid=18462017&doi=10.1371/journal.pbio.0060107&)

59. Innocenti F. et al. Identification, replication, and functional fine-mapping of expression quantitative trait loci in primary human liver tissue. PLoS Genet. 7, e1002078 (2011).  [DOI](https://doi.org/10.1371/journal.pgen.1002078) | [PMC free article](/articles/PMC3102751/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21637794/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet.&title=Identification,%20replication,%20and%20functional%20fine-mapping%20of%20expression%20quantitative%20trait%20loci%20in%20primary%20human%20liver%20tissue&author=F.%20Innocenti&volume=7&publication_year=2011&pages=e1002078&pmid=21637794&doi=10.1371/journal.pgen.1002078&)

60. Grundberg E. et al. Population genomics in a disease targeted primary cell model. Genome Res. 19, 1942–1952 (2009).  [DOI](https://doi.org/10.1101/gr.095224.109) | [PMC free article](/articles/PMC2775606/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19654370/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Res.&title=Population%20genomics%20in%20a%20disease%20targeted%20primary%20cell%20model&author=E.%20Grundberg&volume=19&publication_year=2009&pages=1942-1952&pmid=19654370&doi=10.1101/gr.095224.109&)

61. Kabakchiev B. & Silverberg M. S. Expression quantitative trait loci analysis identifies associations between genotype and gene expression in human intestine. Gastroenterology 144, 1488–1496 (2013).  [DOI](https://doi.org/10.1053/j.gastro.2013.03.001) | [PMC free article](/articles/PMC3775712/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23474282/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gastroenterology&title=Expression%20quantitative%20trait%20loci%20analysis%20identifies%20associations%20between%20genotype%20and%20gene%20expression%20in%20human%20intestine&author=B.%20Kabakchiev&author=M.%20S.%20Silverberg&volume=144&publication_year=2013&pages=1488-1496&pmid=23474282&doi=10.1053/j.gastro.2013.03.001&)

62. Hao K. et al. Lung eQTLs to help reveal the molecular underpinnings of asthma. PLoS Genet. 8, e1003029 (2012).  [DOI](https://doi.org/10.1371/journal.pgen.1003029) | [PMC free article](/articles/PMC3510026/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23209423/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet.&title=Lung%20eQTLs%20to%20help%20reveal%20the%20molecular%20underpinnings%20of%20asthma&author=K.%20Hao&volume=8&publication_year=2012&pages=e1003029&pmid=23209423&doi=10.1371/journal.pgen.1003029&)

63. Ding J. et al. Gene expression in skin and lymphoblastoid cells: Refined statistical method reveals extensive overlap in cis-eQTL signals. Am. J. Hum. Genet. 87, 779–789 (2010).  [DOI](https://doi.org/10.1016/j.ajhg.2010.10.024) | [PMC free article](/articles/PMC2997368/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21129726/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Hum.%20Genet.&title=Gene%20expression%20in%20skin%20and%20lymphoblastoid%20cells:%20Refined%20statistical%20method%20reveals%20extensive%20overlap%20in%20cis-eQTL%20signals&author=J.%20Ding&volume=87&publication_year=2010&pages=779-789&pmid=21129726&doi=10.1016/j.ajhg.2010.10.024&)

64. Huang R. S. et al. Population differences in microRNA expression and biological implications. RNA Biol. 8, 692–701 (2011).  [DOI](https://doi.org/10.4161/rna.8.4.16029) | [PMC free article](/articles/PMC3225983/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21691150/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=RNA%20Biol.&title=Population%20differences%20in%20microRNA%20expression%20and%20biological%20implications&author=R.%20S.%20Huang&volume=8&publication_year=2011&pages=692-701&pmid=21691150&doi=10.4161/rna.8.4.16029&)

65. Rantalainen M. et al. MicroRNA expression in abdominal and gluteal adipose tissue is associated with mRNA expression levels and partly genetically driven. PLoS ONE 6, e27338 (2011).  [DOI](https://doi.org/10.1371/journal.pone.0027338) | [PMC free article](/articles/PMC3216936/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22102887/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&title=MicroRNA%20expression%20in%20abdominal%20and%20gluteal%20adipose%20tissue%20is%20associated%20with%20mRNA%20expression%20levels%20and%20partly%20genetically%20driven&author=M.%20Rantalainen&volume=6&publication_year=2011&pages=e27338&pmid=22102887&doi=10.1371/journal.pone.0027338&)

66. Wang K., Li M. & Hakonarson H. ANNOVAR: functional annotation of genetic variants from high-throughput sequencing data. Nucleic Acids Res. 38, e164 (2010).  [DOI](https://doi.org/10.1093/nar/gkq603) | [PMC free article](/articles/PMC2938201/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20601685/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&title=ANNOVAR:%20functional%20annotation%20of%20genetic%20variants%20from%20high-throughput%20sequencing%20data&author=K.%20Wang&author=M.%20Li&author=H.%20Hakonarson&volume=38&publication_year=2010&pages=e164&pmid=20601685&doi=10.1093/nar/gkq603&)

67. Ward L. D. & Kellis M. HaploReg: a resource for exploring chromatin states, conservation, and regulatory motif alterations within sets of genetically linked variants. Nucleic Acids Res. 40, D930–D934 (2012).  [DOI](https://doi.org/10.1093/nar/gkr917) | [PMC free article](/articles/PMC3245002/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22064851/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&title=HaploReg:%20a%20resource%20for%20exploring%20chromatin%20states,%20conservation,%20and%20regulatory%20motif%20alterations%20within%20sets%20of%20genetically%20linked%20variants&author=L.%20D.%20Ward&author=M.%20Kellis&volume=40&publication_year=2012&pages=D930-D934&pmid=22064851&doi=10.1093/nar/gkr917&)

68. Boyle A. P. et al. Annotation of functional variation in personal genomes using RegulomeDB. Genome Res. 22, 1790–1797 (2012).  [DOI](https://doi.org/10.1101/gr.137323.112) | [PMC free article](/articles/PMC3431494/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22955989/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Res.&title=Annotation%20of%20functional%20variation%20in%20personal%20genomes%20using%20RegulomeDB&author=A.%20P.%20Boyle&volume=22&publication_year=2012&pages=1790-1797&pmid=22955989&doi=10.1101/gr.137323.112&)

69. Dunham I. et al. An integrated encyclopedia of DNA elements in the human genome. Nature 489, 57–74 (2012).  [DOI](https://doi.org/10.1038/nature11247) | [PMC free article](/articles/PMC3439153/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22955616/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=An%20integrated%20encyclopedia%20of%20DNA%20elements%20in%20the%20human%20genome&author=I.%20Dunham&volume=489&publication_year=2012&pages=57-74&pmid=22955616&doi=10.1038/nature11247&)

70. Bernstein B. E. et al. The NIH Roadmap Epigenomics Mapping Consortium. Nat. Biotechnol. 28, 1045–1048 (2010).  [DOI](https://doi.org/10.1038/nbt1010-1045) | [PMC free article](/articles/PMC3607281/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20944595/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Biotechnol.&title=The%20NIH%20Roadmap%20Epigenomics%20Mapping%20Consortium&author=B.%20E.%20Bernstein&volume=28&publication_year=2010&pages=1045-1048&pmid=20944595&doi=10.1038/nbt1010-1045&)

71. Colhoun H. M. et al. Primary prevention of cardiovascular disease with atorvastatin in type 2 diabetes in the Collaborative Atorvastatin Diabetes Study (CARDS): multicentre randomised placebo-controlled trial. Lancet 364, 685–696 (2004).  [DOI](https://doi.org/10.1016/S0140-6736(04)16895-5) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15325833/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Primary%20prevention%20of%20cardiovascular%20disease%20with%20atorvastatin%20in%20type%202%20diabetes%20in%20the%20Collaborative%20Atorvastatin%20Diabetes%20Study%20(CARDS):%20multicentre%20randomised%20placebo-controlled%20trial&author=H.%20M.%20Colhoun&volume=364&publication_year=2004&pages=685-696&pmid=15325833&doi=10.1016/S0140-6736(04)16895-5&)
