# Genome-Wide Association Study of Serum Creatinine Levels during Vancomycin Therapy

## Metadata
**Authors:** Sara L Van Driest, Tracy L McGregor, Digna R Velez Edwards, Ben R Saville, Terrie E Kitchner, Scott J Hebbring, Murray Brilliant, Hayan Jouni, Iftikhar J Kullo, C Buddy Creech, Prince J Kannankeril, Susan I Vear, Kyle B Brothers, Erica A Bowton, Christian M Shaffer, Neelam Patel, Jessica T Delaney, Yuki Bradford, Sarah Wilson, Lana M Olson, Dana C Crawford, Amy L Potts, Richard H Ho, Dan M Roden, Josh C Denny
**Journal:** PLoS ONE
**Date:** 2015 Jun 1
**DOI:** [10.1371/journal.pone.0127791](https://doi.org/10.1371/journal.pone.0127791)
**PMID:** 26030142
**PMCID:** PMC4452656
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4452656/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC4452656/pdf/pone.0127791.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC4452656/pdf/pone.0127791.pdf)

## Abstract

Vancomycin, a commonly used antibiotic, can be nephrotoxic. Known risk factors such as age, creatinine clearance, vancomycin dose / dosing interval, and concurrent nephrotoxic medications fail to accurately predict nephrotoxicity. To identify potential genomic risk factors, we performed a genome-wide association study (GWAS) of serum creatinine levels while on vancomycin in 489 European American individuals and validated findings in three independent cohorts totaling 439 European American individuals. In primary analyses, the chromosome 6q22.31 locus was associated with increased serum creatinine levels while on vancomycin therapy (most significant variant rs2789047, risk allele A, β = -0.06, p = 1.1 x 10-7). SNPs in this region had consistent directions of effect in the validation cohorts, with a meta-p of 1.1 x 10-7. Variation in this region on chromosome 6, which includes the genes TBC1D32/C6orf170 and GJA1 (encoding connexin43), may modulate risk of vancomycin-induced kidney injury.

## Introduction

Vancomycin is a commonly used glycopeptide antibiotic with activity against gram positive bacteria, including methicillin-resistant *Staphylococcus aureus*.[[1](#pone.0127791.ref001)] Though vancomycin is not metabolized, but excreted unchanged by the kidney, vancomycin serum concentrations vary widely among individuals, even after adjustment for renal function, indicating variability in individual pharmacokinetics.[[2](#pone.0127791.ref002),[3](#pone.0127791.ref003)] Current guidelines recommend therapeutic vancomycin drug monitoring via measurement of trough concentrations and weight-based vancomycin dosing.[[4](#pone.0127791.ref004)]

One adverse event associated with vancomycin use is nephrotoxicity, occurring in 3–19% of patients treated with conventional drug doses.[[5](#pone.0127791.ref005),[6](#pone.0127791.ref006)] High dose therapy is associated with higher incidence of nephrotoxicity; vancomycin regimens of more than 4 g/day or targeting trough concentrations over 20 mcg/mL result in renal toxicity in 30–40% of patients.[[7](#pone.0127791.ref007)] Additional risk factors for vancomycin nephrotoxicity include elevated creatinine at baseline, concomitant nephrotoxic agents, age, and intensive care unit admission.[[6](#pone.0127791.ref006)] While the effects on the kidneys are reversible, renal injury during vancomycin therapy has the potential to further exacerbate drug toxicities, as renal elimination of this and other drugs may be impaired.[[7](#pone.0127791.ref007)]

Given the variability of vancomycin kinetics, frequency of nephrotoxicity, and potential clinical impact of variability in each, we sought to identify genomic loci associated with these outcomes. In a genome-wide association study (GWAS) of a retrospective cohort of individuals treated with vancomycin, we examined associations in the primary cohort of 489 individuals with peak creatinine level during the first two weeks of vancomycin therapy, as well as vancomycin trough levels and calculated renal elimination rate constant (Ke). Associations were validated via analysis and meta-analysis of data from independent cohorts of 343 and 59 individuals from two other medical centers, and an additional 37 individuals from the primary site. All three sites participate in the Electronic Medical Records and Genomics (eMERGE) Network and utilized electronic medical record (EMR) data and biobank specimens to assemble the investigated cohorts.[[8](#pone.0127791.ref008)]

## Methods

### Ethics Statement

The primary cohort for this study was derived from BioVU, Vanderbilt’s repository linking DNA from remnant clinical blood samples to de-identified EMR data.[[9](#pone.0127791.ref009),[10](#pone.0127791.ref010)] This resource has been approved as non-human subjects research by Vanderbilt's local Institutional Review Board and the federal Office of Human Research Protections (OHRP). This study was also reviewed by the Vanderbilt Institutional Review Board and determined to be non-human subjects research. Individuals at validation sites provided written consent as part of the DNA biobank at each site.[[11](#pone.0127791.ref011)] This study was approved by the Institutional Review Board at both validation sites (Marshfield Clinic and Mayo Clinic).

### Primary Cohort Identification

The primary cohort included only European American individuals, determined by genotype as described below, 18 years of age or older in BioVU with documentation of a vancomycin trough level after the third dose, the associated dose and dosing interval, and serum creatinine measurements. Exclusion criteria included: no vancomycin trough obtained after the third dose (reflecting steady state); dialysis therapy, extracorporeal membrane oxygenation, or heart transplantation prior to or during vancomycin therapy, identified by current procedural terminology (CPT) codes and manual review; and documentation of multiple dosing regimens of vancomycin prior to trough which were unable to be resolved by manual review.

### Outcome and Covariate Definitions for Peak Serum Creatinine

Peak serum creatinine was defined as the highest creatinine value obtained two dosing intervals after initiation of vancomycin through two weeks after the start of therapy ([S1 Fig](#pone.0127791.s001)). The *a priori* selected covariates for the outcome of peak creatinine were sex, age at time of vancomycin therapy, height, weight, vancomycin dose and dosing interval, vancomycin trough, and two serum creatinine values (baseline and creatinine at vancomycin start, [S1 Fig](#pone.0127791.s001)). Baseline creatinine was defined as the lowest creatinine value measured from one month before the start of vancomycin through the third dose and was included to reflect optimal renal function prior to or at the start of illness. The creatinine at vancomycin start, included to reflect renal function at the time of initiation of therapy, was defined as the creatinine value closest in time to the start of vancomycin therapy with preference for values obtained in the 24 hours prior to the start of therapy, then in the first 24 hours of therapy, then up to 30 days prior to the start of therapy. The frequency of acute kidney injury (AKI), defined as a rise in the serum creatinine of 0.3mg/dL or a 1.5-fold increase in serum creatinine from the baseline to peak value,[[12](#pone.0127791.ref012)] was assessed, and this dichotomous definition of AKI was evaluated as a secondary outcome.

Because concomitant medications, including diuretic and nephrotoxic medications, affect renal function, vancomycin excretion, and serum creatinine levels, all medication orders for loop diuretic medications (furosemide, bumetanide, torsemide, and ethacrynate) within 72 hours before vancomycin trough measurement were extracted from EMR data. Using drug, dose, frequency and route data, diuretic exposures were converted to IV furosemide equivalents given per 24 hours using the following conversions: 1mg oral furosemide = 0.5mg; 1mg oral or IV bumetanide = 40mg; 1mg oral or IV torsemide = 2mg; and 1mg oral or IV ethacrynate = 1mg. For non-loop diuretics, medication data were extracted to determine the total number of different non-loop diuretics given 72 hours prior to the vancomycin trough measurement; specific medications identified in these cohorts were eplerenone, hydrochlorothiazide, mannitol, metolazone, spironolactone, and triamterene. The number of different nephrotoxic medications given to each patient in the 72 hours prior to the vancomycin trough was tallied, excluding those given via topical, ophthalmic or otic routes of administration (listed in [S1 Table](#pone.0127791.s005)). Contrast agents were restricted to those administered intravenously. All “PRN” or “as needed” orders were manually reviewed, and included in the tally only if the EMR included evidence that the patient actually received the nephrotoxic medication.

### Outcome and Covariate Definitions for Vancomycin Trough and Ke

Vancomycin trough was defined as the first vancomycin trough documented in the EMR after at least three doses of vancomycin were given. Ke for each individual was calculated using the formula Ke = -ln[(Trough + [dose/(0.65 x weight)])/Trough]/(dosing interval—infusion time). Covariates included in the analysis of vancomycin trough were age, sex, height, weight, body surface area, vancomycin dose and dosing interval, creatinine at vancomycin start, and concomitant diuretic and nephrotoxic drugs, as defined above. For the analysis of calculated vancomycin Ke, weight, vancomycin dose, and vancomycin dosing interval were excluded as covariates, as they are used in the calculation of Ke. For these two outcomes, multiple imputation was used for missing covariate data points.

### Data Extraction and Validation

All outcome and covariate data were initially extracted from the BioVU repository using automated strategies. After data extraction, a portion of all records was manually reviewed to confirm data accuracy. Manual review included appropriate application of exclusion criteria (e.g. all individuals with any history of a dialysis CPT code were reviewed to ensure dialysis was not initiated during vancomycin therapy), confirmation of dosing data (e.g. all individuals with clinical orders indicating different dosage or interval for vancomycin therapy were reviewed to determine the dose relevant to the trough), and review of outliers (all values more than two standard deviations from the mean for age, height, weight, body surface area, vancomycin dose, vancomycin interval, creatinine, vancomycin trough, and all concomitant medication exposures). Any inaccuracies were manually corrected. All data were stored using the research database tool REDCap.[[13](#pone.0127791.ref013)]

### Primary Genotyping and Quality Control

DNA samples from the primary cohort were genotyped using the Omni1-Quad BeadChip array (Illumina, San Diego, CA). Quality control of genotyping included exclusion of samples with discordant gender compared to biobank records, genotyping efficiency (GE) < 98%, cryptic relatedness (two pairs of half-siblings identified, individuals with lower genotyping efficiency excluded from analysis), and duplicate samples. Single nucleotide polymorphisms (SNPs) were excluded from analysis if minor allele frequency was < 5%, GE < 98% across all samples, discordant genotypes called in duplicate samples, Mendelian errors identified among HapMap trios, or Hardy-Weinberg Equilibrium p-value was > 0.001.

### Primary Analysis and Imputation

All cohorts with genome-wide genotyping were restricted to those with European American ancestry indicated by analysis of ancestry informative markers using STRUCTURE.[[14](#pone.0127791.ref014)] Ancestry distribution for the primary cohort before and after elimination of individuals of non-European American ancestry is shown in [S2 Fig](#pone.0127791.s002). Association analysis of the primary genotyped dataset to each outcome was completed using PLINK v1.07.[[15](#pone.0127791.ref015)] Linear regression assuming an additive genetic model was used to test for single SNP association adjusting for covariates defined above. Individuals missing data for the outcome variable or any covariate data were excluded from the peak creatinine analysis. Due to leftward skew of the outcome variables, peak creatinine, vancomycin trough, and Ke values were log transformed to satisfy normality assumptions. Restricted cubic splines were initially considered to allow non-linear associations between the respective outcomes and continuous covariates, including age, height, weight, body surface area, creatinine, and vancomycin trough. For the outcome of peak creatinine, these were reduced to linear terms for easier interpretation after observing no evidence of non-linearity. After primary association analysis, individual chromosomes with the most significant loci for each outcome were imputed to determine additional SNPs using IMPUTE v2.2.2 and all 1000 Genomes populations as a reference set October 2012 build.[[16](#pone.0127791.ref016)]

### Validation Cohort Analyses

External validation cohorts were assembled using European American individuals from biobanks at the Marshfield Clinic in Marshfield, WI (343 individuals from the Personalized Medicine Research Project who were genotyped for this analysis) and the Mayo Clinic College of Medicine in Rochester, MN (59 individuals with previous genotype data available from prior analyses). An additional 37 European American individuals from the primary site, not included in the discovery analyses and genotyped as part of the Vanderbilt Genome-Electronic Records (VGER) project, were also identified. Inclusion criteria, exclusion criteria, outcome variables, covariates, and data validation procedures for these cohorts were the same as those for the primary cohort.

The most highly associated SNPs in each region identified in the analysis of genotyped data and imputed data as well as tag SNPs for each region identified were genotyped using Sequenom MassARRAY (Sequenom Inc., San Diego, CA) in the 343 DNA samples from the Marshfield Clinic cohort. Linear regression analyses to determine the association of each genotyped SNP to the outcomes of peak serum creatinine, vancomycin trough, and vancomycin Ke were performed with SNPTest v2.4.1.[[17](#pone.0127791.ref017)] An additional 96 samples from the Mayo Clinic and BioVU cohorts were identified with genotyping completed on the Human660W-Quad Bead Chip (Illumina, San Diego, CA). Single locus association analyses across primary and all three validation cohorts were further analyzed together with fixed-effects meta-analyses using METAL software.[[18](#pone.0127791.ref018)] The effective number of independent tests was calculated using SimpleM.[[19](#pone.0127791.ref019),[20](#pone.0127791.ref020)]

## Results

### Primary Cohort

We identified a primary cohort of 882 individuals in the Vanderbilt biorepository, BioVU ([Fig 1](#pone.0127791.g001)). After excluding individuals with no DNA sample available (N = 73), those who failed genotyping quality control (N = 28), and non-European-Americans (N = 36), 745 individuals remained. Of those, 256 were missing data for one or more of the requisite serum creatinine measurements, resulting in a primary cohort of 489 European American individuals for the analysis of peak creatinine while on vancomycin ([Table 1](#pone.0127791.t001)). The median age was 55 years, and 59% of the cohort was male. Most individuals were treated with 1000 mg of vancomycin every 12 hours and had vancomycin trough measurements in therapeutic range (interquartile range 7–16 mcg/mL). The majority of patients were not exposed to diuretic medications, but did receive one or more nephrotoxic drugs concomitant to vancomycin therapy. Peak serum creatinine measurements were higher than creatinine measurements at baseline or at vancomycin start (see [methods](#sec002) for definitions). Of the 489 individuals, 188 (38%) had serum creatinine measurements which met AKI criteria.

### Fig 1. Identification of primary cohort.

![Fig 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2e84/4452656/da2078b3ffd0/pone.0127791.g001.jpg)

Electronic medical records data were searched to identify 5,665 individuals exposed to vancomycin. Automated and manual algorithms were used to determine if each satisfied inclusion / exclusion criteria, as described in the methods, resulting in 882 confirmed cases. After exclusion of those without DNA, those who failed quality control (QC), and those of non-European-American ancestry, 745 individuals remained. Of those, 489 had serum creatinine measurements for the primary analysis.

### Table 1. Cohort demographics for primary and validation cohorts investigated by genome-wide association for the outcome of peak serum creatinine during vancomycin therapy.

|   | Primary Cohort | Marshfield Validation Cohort | Mayo Validation Cohort | VGER Validation Cohort |
| --- | --- | --- | --- | --- |
|   | (N = 489) | (N = 343) | (N = 59) | (N = 37) |
| Age (years)* | 55 (43–65) | 64 (53–75) | 70 (58–75) | 54 (45–66) |
| Male^ | 289 (59%) | 197 (57%) | 39 (66%) | 17 (46%) |
| Weight (kg)* | 81 (66–97) | 86 (68–103) | 88 (74–100) | 82 (68–96) |
| Height (m)* | 173 (165–180) | 168 (161–177) | 170 (162–182) | 175 (165–180) |
| Vancomycin Dose (mg)* | 1000 (1000–1000) | 1200 (1000–1500) | 1400 (1100–1500) | 1000 (1000–1000) |
| Vancomycin Dosing Interval (h)* | 12 (12–12) | 12 (12–24) | 12 (12–24) | 12 (8–12) |
| Vancomycin Trough (mcg/mL)* | 10 (7–16) | 11 (8–16) | 12 (10–15) | 12 (7–17) |
| Baseline Creatinine (mg/dL)* | 0.7 (0.6–0.9) | 0.8 (0.6–1.0) | 0.9 (0.8–1.0) | 0.8 (0.6–1.1) |
| Creatinine at Vancomycin Start (mg/dL)* | 0.9 (0.7–1.1) | 0.9 (0.7–1.1) | 1.0 (0.9–1.2) | 0.8 (0.7–1.0) |
| 24-hour Loop Diuretic Dose (mg)* | 0 (0–10) | 0 (0–0) | 0 (0–20) | 0 (0–0) |
| Loop Diuretic Exposure^ | 134 (27%) | 33 (10%) | 23 (39%) | 2 (5%) |
| Number of Non-loop Diuretics^ |   |   |   |   |
| 0 | 423 (87%) | 314 (92%) | 41 (69%) | 30 (81%) |
| 1 or more | 66 (13%) | 29 (8%) | 18 (31%) | 7 (19%) |
| Number of Nephrotoxic Drugs^ |   |   |   |   |
| 0 | 164 (34%) | 225 (66%) | 8 (14%) | 16 (43%) |
| 1 | 190 (39%) | 50 (15%) | 24 (41%) | 10 (27%) |
| 2 or more | 135 (28%) | 68 (20%) | 27 (46%) | 11 (30%) |
| Peak Creatinine (mg/dL) | 1.0 (0.8–1.3) | 1.0 (0.8–1.4) | 1.1 (0.9–1.4) | 0.9 (0.7–1.0) |

Table 1 Caption: *Median (interquartile range) ^N, %.

DNA samples were genotyped using the Illumina HumanOmni1-Quad platform, and SNPs were analyzed for quality control. Of the initial 1,138,747 variants, 19,793 SNPs were excluded due to genotyping efficiency less than 98%, 455 SNPs were excluded due to Mendelian errors, and 1,526 SNPs were excluded due to discordant calls across duplicate samples. After removal of SNPs with minor allele frequency < 0.05 and those out of Hardy-Weinberg Equilibrium, 711,284 SNPs remained for analysis. The effective number of independent tests calculated using SimpleM was 392,431, resulting in a calculated genome-wide alpha threshold of 1.27 x 10^–7^.[[19](#pone.0127791.ref019),[20](#pone.0127791.ref020)]

Linear regression analysis of genotyped SNPs, assuming an additive genetic model and adjusting for *a priori* determined covariates, identified the most significant SNP, rs2789047 (p = 1.1 x 10^–7^) at chromosome 6q22.31 ([Fig 2](#pone.0127791.g002)). Analysis of all genotyped and imputed SNPs on chromosome 6 identified 6 SNPs with information quality score > 0.9 at the same locus with similar p-values (rs2817952 and rs2817953, p = 9.5 x 10^–7^; rs2817955, rs2817954, rs2817957 and rs2251428, p = 9.6 x 10^–7^). Analysis of AKI as a dichotomous outcome did not identify any SNPs with p-values < 1x10^-6^.

### Fig 2. Association of genome-wide SNPs to peak creatinine while on vancomycin therapy in the primary cohort.

![Fig 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2e84/4452656/ae15347a0e2b/pone.0127791.g002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4452656_pone.0127791.g002.jpg)

A) Manhattan plot, where each dot represents a genotyped SNP, arranged along the x-axis by position of the SNP on each chromosome. The y-axis plots-log10(p-value) for the linear regression analysis of peak creatinine, adjusted for sex, age at time of vancomycin therapy, height, weight, vancomycin dose / dosing interval, vancomycin trough, and baseline serum creatinine measurements as described in the methods. B) LocusZoom plot of 6q22.31 locus, including genotyped and imputed SNPs. Each dot represents a SNP, arranged by position on chromosome 6 along the x-axis, and the color indicates degree of linkage disequilibrium with the index SNP, rs2789047. The left y-axis plots-log10(p-value) for each SNP. The blue line indicates estimated recombination rate, quantified on the right y-axis. Known genes in the region are indicted below the x-axis.

### Validation Cohorts and Meta-Analysis

We sought validation from three independent cohorts, including 343 individuals from the Marshfield Clinic, 59 individuals from Mayo Clinic, and 37 individuals from Vanderbilt not included in the primary analyses and genotyped as part of the Vanderbilt Genome-Electronic Records (VGER) project. All individuals were European Americans, ≥ 18 years of age, and received vancomycin during inpatient hospitalizations. Demographic characteristics and clinical covariates for the primary and validation cohorts are listed in [Table 1](#pone.0127791.t001). With the exception of the VGER cohort, the validation cohorts were older and had higher, less frequent vancomycin dosing than the primary cohort. Creatinine values were similar across cohorts, with peak creatinine values higher than those defined as baseline or at vancomycin start. Diuretic and nephrotoxic medication exposures varied in frequency across cohorts, with the Mayo cohort having higher frequencies of exposure to these medications.

A total of 65 SNPs with minor allele frequency > 1% from the chromosome 6 locus, including the most statistically significant SNPs and additional SNPs tagging known variants in the region, were genotyped in the validation set of 343 Marshfield samples, resulting in 12 SNPs with p-values for association with peak creatinine < 0.05 in adjusted analysis ([S2 Table](#pone.0127791.s006)). Meta-analysis of the primary and all validation cohorts for the SNPs in this region represented in the majority of datasets, either as genotyped or imputed SNPs, revealed 3 SNPs with p-values < 3 x 10^–7^ with consistent direction of effect in all but the supplemental VGER cohort ([Table 2](#pone.0127791.t002)); the p-value for the most statistically significant SNP in the meta-analysis (rs2789047, p = 1.1 x 10^–7^) exceeds the calculated genome-wide alpha threshold of 1.27 x 10^–7^. The three most significantly associated SNPs have identical effect sizes (β = -0.053) and are in strong LD, with r^2^ > 0.92 for all pairwise calculations of LD in the primary cohort. Genes in this region include *TBC1D32/C6orf170*, located ~211kb downstream and *GJA1*, encoding connexin43, located ~567kb downstream ([Fig 2](#pone.0127791.g002)).

### Table 2. Meta-analysis results for chromosome 6 SNP associations to peak creatinine during vancomycin therapy.

|   |  |  | Primary Cohort (N = 489) |  |  | Marshfield Validation (N = 343) |  | Mayo Validation (N = 59) |  | VGER Validation (N = 37) |  | Meta-Analysis (Primary + Validation) |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| SNP | Position | Risk Allele | RAF | β (SE) | P | β (SE) | P | β (SE) | P | β (SE) | P | β (SE) | Meta-P |
| rs2789047 | 121231501 | A | 0.28 | -0.06 (0.01) | 1.1x10-7 | -0.02 (0.04) | 0.57 | -0.001 (0.07) | 0.99 | 0.09 (0.37) | 0.81 | -0.05 (0.01) | 1.1x10-7 |
| rs12527161 | 121236583 | T | 0.27 | -0.06 (0.01) | 1.8x10-7 | -0.02 (0.04) | 0.63 | -0.02 (0.07) | 0.76 | 0.09 (0.37) | 0.81 | -0.05 (0.01) | 1.8x10-7 |
| rs9320773 | 121218196 | A | 0.27 | -0.06 (0.01) | 2.0x10-7 | -0.01 (0.04) | 0.82 | -0.04 (0.08) | 0.62 | 0.001 (0.34) | 1.0 | -0.05 (0.01) | 2.3x10-7 |
| rs1886249 | 121791852 | c | 0.14 | -0.02 (0.01) | 0.13 | -0.04 (0.04) | 0.35 | -0.03 (0.08) | 0.74 | 0.30 (0.43) | 0.50 | -0.023 (0.01) | 0.081 |
| rs3805787 | 121800606 | c | 0.23 | -0.01 (0.01) | 0.43 | -0.03 (0.04) | 0.37 | -0.04 (0.07) | 0.59 | 0.26 (0.46) | 0.58 | -0.012 (0.01) | 0.28 |
| rs17083553* | 121775368 | T | 0.19 | 0.01 (0.01) | 0.65 | 0.07 (0.05) | 0.12 | 0.01 (0.07) | 0.99 | -0.37 (0.44) | 0.41 | 0.009 (0.01) | 0.413 |
| rs6932373* | 121832689 | A | 0.38 | 0.002 (0.01) | 0.87 | -0.04 (0.03) | 0.27 | -0.09 (0.07) | 0.19 | 0.08 (0.23) | 0.74 | -0.003 (0.01) | 0.752 |
| rs2389541 | 121815698 | c | 0.31 | 0.01 (0.01) | 0.31 | -0.07 (0.04) | 0.05 | -0.09 (0.07) | 0.20 | 0.10 (0.26) | 0.70 | 0.002 (0.01) | 0.805 |
| rs17083598 | 121788169 | g | 0.06 | 0.01 (0.02) | 0.75 | -0.09 (0.06) | 0.18 | -0.08 (0.16) | 0.62 | 1.01 (0.63) | 0.12 | -0.002 (0.02) | 0.902 |

Table 2 Caption: RAF—Risk Allele Frequency; SE—Standard Error. *—Imputed SNP in Mayo and VGER cohorts.

### Analysis of Vancomycin Trough and Ke

No individuals in the primary cohort were missing outcome data for the analysis of vancomycin trough or calculated Ke, allowing analysis of 745 individuals ([S3 Table](#pone.0127791.s007)). In the analysis of the 711,284 genotyped SNPs in the primary cohort, both vancomycin trough and Ke analyses identified SNPs at chromosome 1q41 ([S3](#pone.0127791.s003) and [S4](#pone.0127791.s004) Figs). The most significantly associated SNP on chromosome 1 was rs10495197 (p = 4.5 x 10^–7^ for vancomycin trough and 2.4 x 10^–6^ for Ke). Imputation of chromosome 1 and analysis of imputed genotypes identified 13 additional SNPs at this locus associated with vancomycin trough and Ke with smaller p-values. Analysis of 27 SNPs on chromosome 1 genotyped in the Marshfield cohort, including SNPs with smallest p-values from the primary analysis and tag SNPs for known variants in these regions, found a minimum p-value of 0.13 for the outcome of vancomycin trough ([S4 Table](#pone.0127791.s008)). Effect sizes and directions of effect were variable for chromosome 1 SNPs between the primary and replication cohorts.

Chromosome locus 5q14.3 was also associated with these outcomes ([S3](#pone.0127791.s003) and [S4](#pone.0127791.s004) Figs). On chromosome 5, the most significant p-values for genotyped SNPs were obtained for rs12518285 (p = 9.2 x 10^–6^ for vancomycin trough and 1.8 x 10^–7^ for Ke), and 5 imputed SNPs had smaller p-values. Analysis of 5 associated and tag SNPs on chromosome 5 in the Marshfield cohort found a minimum p-value of 0.16 ([S4 Table](#pone.0127791.s008)). Meta-analysis of the primary and validation cohorts using both genotyped and imputed SNPs, for the outcome of vancomycin trough revealed the lowest p-value 6.0 x 10^–5^ (rs12518285, on chromosome 5, [S4 Table](#pone.0127791.s008)). Effect sizes and directions of effect were consistent for chromosome 5 SNPs between the primary and replication cohorts. The nearest known gene adjacent to the chromosome 5 locus is *EDIL3*, 557kb upstream.

Of note, SNPs at the chromosome 6 locus identified in the analysis of peak creatinine were not significantly associated with vancomycin trough or Ke. The most significant association in this region for both outcomes was for rs7748279 (p = 0.03 and 0.12 for association with vancomycin trough and Ke, respectively).

## Discussion

We utilized banked DNA and de-identified EMR data to assemble primary and validation cohorts to identify genetic variants associated with peak serum creatinine while on vancomycin, vancomycin trough concentration, and vancomycin Ke. The GWAS and meta-analysis identified a potential association of the chromosome 6q22.31 locus to peak creatinine during vancomycin therapy. Neither the primary analysis nor the meta-analysis resulted in p-values below 5 x 10^–8^, the typical threshold value for genome-wide significance, which may be due to the modest sample size of the primary and validation cohorts for this study design. However, p-values below the alpha threshold of 1.27x10^-7^ were found in the primary cohort and the meta-analysis across cohorts of peak creatinine levels while on vancomycin.

Vancomycin requires intravenous administration for systemic delivery due to poor oral bioavailability, so nearly all patients treated with vancomycin receive this drug during an inpatient hospitalization. At the primary institution and the validation sites, EMRs retain the patient data, laboratory values and medication exposures necessary for this study of vancomycin. BioVU, the Vanderbilt biorepository linked to de-identified EMR data, has DNA samples and clinical data available for over 180,000 individuals. We identified over 5,600 vancomycin-exposed individuals. However, the number of individuals with sufficient clinical documentation of outcomes (including multiple measured serum creatinine values and measurement of the vancomycin trough after at least 3 doses had been given) and covariates (including vancomycin dosing data) was much smaller. Cohort size was further reduced by the necessary step of restricting the analysis by ancestry. Despite the inclusion of a total of only 928 individuals from all sites for the primary outcome of peak serum creatinine while on vancomycin, the 6q22.31 locus was associated with a meta-p-value of 1.1 x 10^–7^.

The SNP with the strongest evidence of association with peak serum creatinine while on vancomycin (rs2789047) is near two known genes. The first, *TBC1D32/C6orf170*, encodes a ciliary protein and has been associated with oro-facial-digital syndrome type IX.[[21](#pone.0127791.ref021),[22](#pone.0127791.ref022)] There are no reported associations of this protein to renal function. The second gene in this region is *GJA1*, which encodes connexin43, a gap junction protein expressed in renal proximal tubules that has been previously associated with renal injury.[[23](#pone.0127791.ref023)–[25](#pone.0127791.ref025)] In rodents, podocyte injury and chronic kidney disease lead to increased expression of connexin43.[[23](#pone.0127791.ref023),[24](#pone.0127791.ref024)] In human proximal tubule epithelial cell cultures, aminoglycoside-sensitive cell lines have increased levels of expression of connexin43.[[25](#pone.0127791.ref025)] Connexin43 overexpression sensitized cells to aminoglycoside-induced injury, and siRNA inhibition of connexin43 expression attenuated cellular injury, as did functional inhibition.[[25](#pone.0127791.ref025)] We speculate that genetic variation in the locus we identified may affect connexin43 expression or function in humans and in turn play a role in nephrotoxicity from vancomycin. This region was not associated with vancomycin trough or Ke, suggesting that it may be an independent risk factor for kidney injury. Prior GWAS analyses in multiple populations have not identified this locus in association with chronic kidney disease,[[26](#pone.0127791.ref026)–[30](#pone.0127791.ref030)] indicating that this finding may be specific for nephrotoxicity in the setting of vancomycin exposure. Further experimental data are required to determine the specific role, if any, connexin43 may play in vancomycin-associated renal injury.

Analysis of the secondary outcomes (vancomycin trough and vancomycin Ke) failed to identify loci with p-values exceeding our alpha threshold in the primary or meta-analyses. The most strongly associated loci on chromosome 1 and chromosome 5 weakly associated with both outcomes, as expected since Ke was calculated from vancomycin trough, dose, and patient weight. Although a genetic association to variation in vancomycin clearance was not identified, our study demonstrates the use of EMR data to explore complex phenotypes such as pharmacokinetics.

Conclusions to be drawn from this initial GWAS are limited by issues common across the use of contemporary genomic approaches to study variability in drug action: modest cohort size, small effect size, and the selected nature of the individuals represented in these cohorts (i.e., those receiving vancomycin). The requirement for serum creatinine and vancomycin trough measurements in the retrospective cohorts may have resulted in over-representation of high-risk individuals. This selection bias for sicker patients is a major limitation to this study and impacts the generalizability of our findings. In addition, because the full spectrum of clinical outcomes may not be well-represented in our cohorts, the full effects of genetic variation may not be apparent from our data. However, many of these high-risk individuals did not develop renal injury (as defined by elevated serum creatinine) providing a cohort sufficient to identify the chromosome 6 locus for further study. Our analysis of AKI as a dichotomous outcome failed to identify any SNPs with significant association, likely due to loss of statistical power with a dichotomous outcome and the lack of specificity for true renal injury that may be inherent to the currently accepted creatinine thresholds for AKI.

Additional study of variants in the connexin43 gene and surrounding regulatory DNA will be required to test the hypothesis that genetic variation at this locus may influence susceptibility to vancomycin-associated renal toxicity and assess the potential clinical impact of this association for patients treated with vancomycin and other renally eliminated nephrotoxic medications. If genetic markers for nephrotoxic risk with clinically significant effect size are identified, pre-treatment screening may be employed to identify patients who merit close monitoring or modifications to their vancomycin dose. In addition, as the cellular mechanisms for vancomycin elimination and associated nephrotoxicity are unknown, further exploration of the role of connexin43 may lead to novel therapeutic interventions to prevent vancomycin-associated kidney injury.

## Supporting Information

## Acknowledgments

The authors thank the Vanderbilt Synthetic Derivative and BioVU teams for their contribution to this work.

## Data Availability

Genotype and phenotype files have been submitted to dbGaP (accession number phs000894.v1.p1, URL: [http://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000894.v1.p1](http://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000894.v1.p1)).

## Funding Statement

This study was supported by National Center for Research Resources / National Institutes of Health ([http://www.nih.gov/](http://www.nih.gov/)) grant UL1-RR024975, supporting the Vanderbilt Institute for Clinical and Translational Research (VICTR) (SLV, TLM, DRVE, BRS, CBC, PJK, SIV, KBB, EAB, CMS, NP, JTD, YB, SW, SMP, DCC, ALP, RHH, DMR, JCD); National Human Genome Research Institute / National Institutes of Health ([http://www.nih.gov/](http://www.nih.gov/)) grant U01-HG04603 (SLV, TLM, DRVE, BRS, CBC, PJK, SIV, KBB, EAB, CMS, NP, JTD, YB, SW, SMP, DCC, ALP, RHH, DMR, JCD); and National Institute of General Medical Sciences / National Institutes of Health ([http://www.nih.gov/](http://www.nih.gov/)) RC2-GM092618 (SLV, TLM, DRVE, BRS, CBC, PJK, SIV, KBB, EAB, CMS, NP, JTD, YB, SW, SMP, DCC, ALP, RHH, DMR, JCD). SLV and SIV have been supported by National Institute of General Medical Sciences / National Institutes of Health ([http://www.nih.gov/](http://www.nih.gov/)) Clinical Pharmacology Training Program 5T32-GM007569. SLV is supported by an Early Career Award from the Pharmaceutical Research and Manufacturers of America Foundation ([http://phrmafoundation.org/](http://phrmafoundation.org/)). TLM has been supported by National Institute of Environmental Health Sciences / National Institutes of Health ([http://www.nih.gov/](http://www.nih.gov/)) grant K12-ES015855 and the National Institute of Child Health and Human Development / National Institutes of Health ([http://www.nih.gov/](http://www.nih.gov/)) 1K23-HD000001. DRVE was supported by the National Institute of Child Health and Human Development / National Institutes of Health ([http://www.nih.gov/](http://www.nih.gov/)) Building Interdisciplinary Research Careers in Women’s Health career development program (2K12-HD043483-11). The dataset(s) used for the analyses described were obtained from Vanderbilt University Medical Center's BioVU which is supported by institutional funding and by the Vanderbilt CTSA grant UL1-TR000445 from National Center for Advancing Translational Sciences / National Institutes of Health ([http://www.nih.gov/](http://www.nih.gov/))(SLV, TLM, DRVE, BRS, CBC, PJK, SIV, KBB, EAB, CMS, NP, JTD, YB, SW, SMP, DCC, ALP, RHH, DMR, JCD); the Personalized Medicine Research Project of Marshfield Clinic which is supported by institutional funding and by National Human Genome Research Institute / National Institutes of Health ([http://www.nih.gov/](http://www.nih.gov/)) grant 1U01-HG006389 and from National Center for Advancing Translational Sciences / National Institutes of Health ([http://www.nih.gov/](http://www.nih.gov/)) grant UL1-TR000427 (TEK, SJH, MB); and the Mayo eMERGE Network Project which is supported by the National Human Genome Research Institute / National Institutes of Health ([http://www.nih.gov/](http://www.nih.gov/)) grants (U01-HG04599 and U01-HG006379) (HJ, IJK). The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Data Availability Statement

Genotype and phenotype files have been submitted to dbGaP (accession number phs000894.v1.p1, URL: [http://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000894.v1.p1](http://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000894.v1.p1)).

### Supplementary Materials

### Data Availability Statement

Genotype and phenotype files have been submitted to dbGaP (accession number phs000894.v1.p1, URL: [http://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000894.v1.p1](http://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000894.v1.p1)).

## References

1. Deck D, Winston L. Chapter 43. Beta-Lactam & Other Cell Wall- & Membrane-Active Antibiotics In: Katzung BG, Masters SB, Trevor AJ, editors. Basic & Clinical Pharmacology. 12th ed. New York: McGraw-Hill; 2012.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Basic%20&%20Clinical%20Pharmacology&author=D%20Deck&author=L%20Winston&author=BG%20Katzung&author=SB%20Masters&author=AJ%20Trevor&publication_year=2012&)

2. Murphy JE, Gillespie DE, Bateman CV. Predictability of vancomycin trough concentrations using seven approaches for estimating pharmacokinetic parameters. Am J Health Syst Pharm. 2006;63: 2365–2370. 10.2146/ajhp060047  [DOI](https://doi.org/10.2146/ajhp060047) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17106010/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Health%20Syst%20Pharm&title=Predictability%20of%20vancomycin%20trough%20concentrations%20using%20seven%20approaches%20for%20estimating%20pharmacokinetic%20parameters&author=JE%20Murphy&author=DE%20Gillespie&author=CV%20Bateman&volume=63&publication_year=2006&pages=2365-2370&pmid=17106010&doi=10.2146/ajhp060047&)

3. Marsot A, Boulamery A, Bruguerolle B, Simon N. Vancomycin. Clin Pharmacokinet. 2012;51: 1–13. 10.2165/11596390-000000000-00000  [DOI](https://doi.org/10.2165/11596390-000000000-00000) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22149255/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacokinet&title=Vancomycin&author=A%20Marsot&author=A%20Boulamery&author=B%20Bruguerolle&author=N%20Simon&volume=51&publication_year=2012&pages=1-13&pmid=22149255&doi=10.2165/11596390-000000000-00000&)

4. Rybak M, Lomaestro B, Rotschafer JC, Moellering R Jr, Craig W, Billeter M, et al. Therapeutic monitoring of vancomycin in adult patients: a consensus review of the American Society of Health-System Pharmacists, the Infectious Diseases Society of America, and the Society of Infectious Diseases Pharmacists. Am J Health Syst Pharm. 2009;66: 82–98. 10.2146/ajhp080434  [DOI](https://doi.org/10.2146/ajhp080434) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19106348/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Health%20Syst%20Pharm&title=Therapeutic%20monitoring%20of%20vancomycin%20in%20adult%20patients:%20a%20consensus%20review%20of%20the%20American%20Society%20of%20Health-System%20Pharmacists,%20the%20Infectious%20Diseases%20Society%20of%20America,%20and%20the%20Society%20of%20Infectious%20Diseases%20Pharmacists&author=M%20Rybak&author=B%20Lomaestro&author=JC%20Rotschafer&author=R%20Moellering&author=W%20Craig&volume=66&publication_year=2009&pages=82-98&pmid=19106348&doi=10.2146/ajhp080434&)

5. Wong-Beringer A, Joo J, Tse E, Beringer P. Vancomycin-associated nephrotoxicity: a critical appraisal of risk with high-dose therapy. Int J Antimicrob Agents. 2011;37: 95–101. 10.1016/j.ijantimicag.2010.10.013  [DOI](https://doi.org/10.1016/j.ijantimicag.2010.10.013) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21130609/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Antimicrob%20Agents&title=Vancomycin-associated%20nephrotoxicity:%20a%20critical%20appraisal%20of%20risk%20with%20high-dose%20therapy&author=A%20Wong-Beringer&author=J%20Joo&author=E%20Tse&author=P%20Beringer&volume=37&publication_year=2011&pages=95-101&pmid=21130609&doi=10.1016/j.ijantimicag.2010.10.013&)

6. Elyasi S, Khalili H, Dashti-Khavidaki S, Mohammadpour A. Vancomycin-induced nephrotoxicity: mechanism, incidence, risk factors and special populations. A literature review. Eur J Clin Pharmacol. 2012;68: 1243–1255. 10.1007/s00228-012-1259-9  [DOI](https://doi.org/10.1007/s00228-012-1259-9) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22411630/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Clin%20Pharmacol&title=Vancomycin-induced%20nephrotoxicity:%20mechanism,%20incidence,%20risk%20factors%20and%20special%20populations.%20A%20literature%20review&author=S%20Elyasi&author=H%20Khalili&author=S%20Dashti-Khavidaki&author=A%20Mohammadpour&volume=68&publication_year=2012&pages=1243-1255&pmid=22411630&doi=10.1007/s00228-012-1259-9&)

7. Hidayat LK, Hsu DI, Quist R, Shriner KA, Wong-Beringer A. High-dose vancomycin therapy for methicillin-resistant Staphylococcus aureus infections: efficacy and toxicity. Arch Intern Med. 2006;166: 2138–2144. 10.1001/archinte.166.19.2138  [DOI](https://doi.org/10.1001/archinte.166.19.2138) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17060545/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arch%20Intern%20Med&title=High-dose%20vancomycin%20therapy%20for%20methicillin-resistant%20Staphylococcus%20aureus%20infections:%20efficacy%20and%20toxicity&author=LK%20Hidayat&author=DI%20Hsu&author=R%20Quist&author=KA%20Shriner&author=A%20Wong-Beringer&volume=166&publication_year=2006&pages=2138-2144&pmid=17060545&doi=10.1001/archinte.166.19.2138&)

8. Crawford DC, Crosslin DR, Tromp G, Kullo IJ, Kuivaniemi H, Hayes MG, et al. eMERGEing progress in genomics-the first seven years. Front Genet. 2014;5: 184 10.3389/fgene.2014.00184  [DOI](https://doi.org/10.3389/fgene.2014.00184) | [PMC free article](/articles/PMC4060012/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24987407/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front%20Genet&title=eMERGEing%20progress%20in%20genomics-the%20first%20seven%20years&author=DC%20Crawford&author=DR%20Crosslin&author=G%20Tromp&author=IJ%20Kullo&author=H%20Kuivaniemi&volume=5&publication_year=2014&pages=184&pmid=24987407&doi=10.3389/fgene.2014.00184&)

9. Roden DM, Pulley JM, Basford MA, Bernard GR, Clayton EW, Balser JR, et al. Development of a large-scale de-identified DNA biobank to enable personalized medicine. Clin Pharmacol Ther. 2008;84: 362–369. 10.1038/clpt.2008.89  [DOI](https://doi.org/10.1038/clpt.2008.89) | [PMC free article](/articles/PMC3763939/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18500243/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Development%20of%20a%20large-scale%20de-identified%20DNA%20biobank%20to%20enable%20personalized%20medicine&author=DM%20Roden&author=JM%20Pulley&author=MA%20Basford&author=GR%20Bernard&author=EW%20Clayton&volume=84&publication_year=2008&pages=362-369&pmid=18500243&doi=10.1038/clpt.2008.89&)

10. Bowton E, Field JR, Wang S, Schildcrout JS, Van Driest SL, Delaney JT, et al. Biobanks and electronic medical records: enabling cost-effective research. Sci Transl Med. 2014;6: 234cm3. 10.1126/scitranslmed.3008604  [DOI](https://doi.org/10.1126/scitranslmed.3008604) | [PMC free article](/articles/PMC4226414/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24786321/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Sci%20Transl%20Med&title=Biobanks%20and%20electronic%20medical%20records:%20enabling%20cost-effective%20research&author=E%20Bowton&author=JR%20Field&author=S%20Wang&author=JS%20Schildcrout&author=SL%20Van%20Driest&volume=6&publication_year=2014&pmid=24786321&doi=10.1126/scitranslmed.3008604&)

11. McCarty CA, Chisholm RL, Chute CG, Kullo IJ, Jarvik GP, Larson EB, et al. The eMERGE Network: a consortium of biorepositories linked to electronic medical records data for conducting genomic studies. BMC Med Genomics. 2011;4: 13 10.1186/1755-8794-4-13  [DOI](https://doi.org/10.1186/1755-8794-4-13) | [PMC free article](/articles/PMC3038887/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21269473/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Med%20Genomics&title=The%20eMERGE%20Network:%20a%20consortium%20of%20biorepositories%20linked%20to%20electronic%20medical%20records%20data%20for%20conducting%20genomic%20studies&author=CA%20McCarty&author=RL%20Chisholm&author=CG%20Chute&author=IJ%20Kullo&author=GP%20Jarvik&volume=4&publication_year=2011&pages=13&pmid=21269473&doi=10.1186/1755-8794-4-13&)

12. Improving Global Outcomes (KDIGO) Acute Kidney Injury Work Group. KDIGO Clinical Practice Guideline for Acute Kidney Injury. Kidney Int. 2012;2: 1–138.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Kidney%20Int&title=KDIGO%20Clinical%20Practice%20Guideline%20for%20Acute%20Kidney%20Injury&volume=2&publication_year=2012&pages=1-138&)

13. Harris PA, Taylor R, Thielke R, Payne J, Gonzalez N, Conde JG. Research electronic data capture (REDCap)—a metadata-driven methodology and workflow process for providing translational research informatics support. J Biomed Inform. 2009;42: 377–381. 10.1016/j.jbi.2008.08.010  [DOI](https://doi.org/10.1016/j.jbi.2008.08.010) | [PMC free article](/articles/PMC2700030/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18929686/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biomed%20Inform&title=Research%20electronic%20data%20capture%20(REDCap)%E2%80%94a%20metadata-driven%20methodology%20and%20workflow%20process%20for%20providing%20translational%20research%20informatics%20support&author=PA%20Harris&author=R%20Taylor&author=R%20Thielke&author=J%20Payne&author=N%20Gonzalez&volume=42&publication_year=2009&pages=377-381&pmid=18929686&doi=10.1016/j.jbi.2008.08.010&)

14. Pritchard JK, Stephens M, Donnelly P. Inference of population structure using multilocus genotype data. Genetics. 2000;155: 945–959.  [DOI](https://doi.org/10.1093/genetics/155.2.945) | [PMC free article](/articles/PMC1461096/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10835412/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genetics&title=Inference%20of%20population%20structure%20using%20multilocus%20genotype%20data&author=JK%20Pritchard&author=M%20Stephens&author=P%20Donnelly&volume=155&publication_year=2000&pages=945-959&pmid=10835412&doi=10.1093/genetics/155.2.945&)

15. Purcell S, Neale B, Todd-Brown K, Thomas L, Ferreira MAR, Bender D, et al. PLINK: a tool set for whole-genome association and population-based linkage analyses. Am J Hum Genet. 2007;81: 559–575. 10.1086/519795  [DOI](https://doi.org/10.1086/519795) | [PMC free article](/articles/PMC1950838/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17701901/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=PLINK:%20a%20tool%20set%20for%20whole-genome%20association%20and%20population-based%20linkage%20analyses&author=S%20Purcell&author=B%20Neale&author=K%20Todd-Brown&author=L%20Thomas&author=MAR%20Ferreira&volume=81&publication_year=2007&pages=559-575&pmid=17701901&doi=10.1086/519795&)

16. Howie BN, Donnelly P, Marchini J. A flexible and accurate genotype imputation method for the next generation of genome-wide association studies. PLoS Genet. 2009;5: e1000529 10.1371/journal.pgen.1000529  [DOI](https://doi.org/10.1371/journal.pgen.1000529) | [PMC free article](/articles/PMC2689936/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19543373/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet&title=A%20flexible%20and%20accurate%20genotype%20imputation%20method%20for%20the%20next%20generation%20of%20genome-wide%20association%20studies&author=BN%20Howie&author=P%20Donnelly&author=J%20Marchini&volume=5&publication_year=2009&pages=e1000529&pmid=19543373&doi=10.1371/journal.pgen.1000529&)

17. Marchini J, Howie B, Myers S, McVean G, Donnelly P. A new multipoint method for genome-wide association studies by imputation of genotypes. Nat Genet. 2007;39: 906–913. 10.1038/ng2088  [DOI](https://doi.org/10.1038/ng2088) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17572673/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=A%20new%20multipoint%20method%20for%20genome-wide%20association%20studies%20by%20imputation%20of%20genotypes&author=J%20Marchini&author=B%20Howie&author=S%20Myers&author=G%20McVean&author=P%20Donnelly&volume=39&publication_year=2007&pages=906-913&pmid=17572673&doi=10.1038/ng2088&)

18. Willer CJ, Li Y, Abecasis GR. METAL: fast and efficient meta-analysis of genomewide association scans. Bioinformatics. 2010;26: 2190–2191. 10.1093/bioinformatics/btq340  [DOI](https://doi.org/10.1093/bioinformatics/btq340) | [PMC free article](/articles/PMC2922887/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20616382/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=METAL:%20fast%20and%20efficient%20meta-analysis%20of%20genomewide%20association%20scans&author=CJ%20Willer&author=Y%20Li&author=GR%20Abecasis&volume=26&publication_year=2010&pages=2190-2191&pmid=20616382&doi=10.1093/bioinformatics/btq340&)

19. Gao X, Starmer J, Martin ER. A multiple testing correction method for genetic association studies using correlated single nucleotide polymorphisms. Genet Epidemiol. 2008;32: 361–369. 10.1002/gepi.20310  [DOI](https://doi.org/10.1002/gepi.20310) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18271029/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genet%20Epidemiol&title=A%20multiple%20testing%20correction%20method%20for%20genetic%20association%20studies%20using%20correlated%20single%20nucleotide%20polymorphisms&author=X%20Gao&author=J%20Starmer&author=ER%20Martin&volume=32&publication_year=2008&pages=361-369&pmid=18271029&doi=10.1002/gepi.20310&)

20. Gao X, Becker LC, Becker DM, Starmer JD, Province MA. Avoiding the high Bonferroni penalty in genome-wide association studies. Genet Epidemiol. 2010;34: 100–105. 10.1002/gepi.20430  [DOI](https://doi.org/10.1002/gepi.20430) | [PMC free article](/articles/PMC2796708/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19434714/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genet%20Epidemiol&title=Avoiding%20the%20high%20Bonferroni%20penalty%20in%20genome-wide%20association%20studies&author=X%20Gao&author=LC%20Becker&author=DM%20Becker&author=JD%20Starmer&author=MA%20Province&volume=34&publication_year=2010&pages=100-105&pmid=19434714&doi=10.1002/gepi.20430&)

21. Adly N, Alhashem A, Ammari A, Alkuraya FS. Ciliary Genes TBC1D32/C6orf170 and SCLT1 are Mutated in Patients with OFD Type IX. Hum Mutat. 2014;35: 36–40. 10.1002/humu.22477  [DOI](https://doi.org/10.1002/humu.22477) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24285566/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mutat&title=Ciliary%20Genes%20TBC1D32/C6orf170%20and%20SCLT1%20are%20Mutated%20in%20Patients%20with%20OFD%20Type%20IX&author=N%20Adly&author=A%20Alhashem&author=A%20Ammari&author=FS%20Alkuraya&volume=35&publication_year=2014&pages=36-40&pmid=24285566&doi=10.1002/humu.22477&)

22. Ishikawa H, Thompson J, Yates JR III, Marshall WF. Proteomic Analysis of Mammalian Primary Cilia. Curr Biol. 2012;22: 414–419. 10.1016/j.cub.2012.01.031  [DOI](https://doi.org/10.1016/j.cub.2012.01.031) | [PMC free article](/articles/PMC3298568/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22326026/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Biol&title=Proteomic%20Analysis%20of%20Mammalian%20Primary%20Cilia&author=H%20Ishikawa&author=J%20Thompson&author=JR%20Yates&author=WF%20Marshall&volume=22&publication_year=2012&pages=414-419&pmid=22326026&doi=10.1016/j.cub.2012.01.031&)

23. Yaoita E, Yao J, Yoshida Y, Morioka T, Nameta M, Takata T, et al. Up-regulation of connexin43 in glomerular podocytes in response to injury. Am J Pathol. 2002;161: 1597–1606.  [DOI](https://doi.org/10.1016/s0002-9440(10)64438-0) | [PMC free article](/articles/PMC1850784/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12414508/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Pathol&title=Up-regulation%20of%20connexin43%20in%20glomerular%20podocytes%20in%20response%20to%20injury&author=E%20Yaoita&author=J%20Yao&author=Y%20Yoshida&author=T%20Morioka&author=M%20Nameta&volume=161&publication_year=2002&pages=1597-1606&pmid=12414508&doi=10.1016/s0002-9440(10)64438-0&)

24. Toubas J, Beck S, Pageaud A-L, Huby A-C, Mael-Ainin M, Dussaule J-C, et al. Alteration of connexin expression is an early signal for chronic kidney disease. Am J Physiol Renal Physiol. 2011;301: F24–32. 10.1152/ajprenal.00255.2010  [DOI](https://doi.org/10.1152/ajprenal.00255.2010) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21429966/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Physiol%20Renal%20Physiol&title=Alteration%20of%20connexin%20expression%20is%20an%20early%20signal%20for%20chronic%20kidney%20disease&author=J%20Toubas&author=S%20Beck&author=A-L%20Pageaud&author=A-C%20Huby&author=M%20Mael-Ainin&volume=301&publication_year=2011&pages=F24-32&pmid=21429966&doi=10.1152/ajprenal.00255.2010&)

25. Yao J, Huang T, Fang X, Chi Y, Zhu Y, Wan Y, et al. Disruption of gap junctions attenuates aminoglycoside-elicited renal tubular cell injury. Br J Pharmacol. 2010;160: 2055–2068. 10.1111/j.1476-5381.2010.00860.x  [DOI](https://doi.org/10.1111/j.1476-5381.2010.00860.x) | [PMC free article](/articles/PMC2958649/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20649601/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Pharmacol&title=Disruption%20of%20gap%20junctions%20attenuates%20aminoglycoside-elicited%20renal%20tubular%20cell%20injury&author=J%20Yao&author=T%20Huang&author=X%20Fang&author=Y%20Chi&author=Y%20Zhu&volume=160&publication_year=2010&pages=2055-2068&pmid=20649601&doi=10.1111/j.1476-5381.2010.00860.x&)

26. Pattaro C, Köttgen A, Teumer A, Garnaas M, Böger CA, Fuchsberger C, et al. Genome-wide association and functional follow-up reveals new loci for kidney function. PLoS Genet. 2012;8: e1002584 10.1371/journal.pgen.1002584  [DOI](https://doi.org/10.1371/journal.pgen.1002584) | [PMC free article](/articles/PMC3315455/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22479191/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet&title=Genome-wide%20association%20and%20functional%20follow-up%20reveals%20new%20loci%20for%20kidney%20function&author=C%20Pattaro&author=A%20K%C3%B6ttgen&author=A%20Teumer&author=M%20Garnaas&author=CA%20B%C3%B6ger&volume=8&publication_year=2012&pages=e1002584&pmid=22479191&doi=10.1371/journal.pgen.1002584&)

27. Kottgen A, Glazer NL, Dehghan A, Hwang S- J, Katz R, Li M, et al. Multiple Novel Loci are Associated with Indices of Renal Function and Chronic Kidney Disease. Nat Genet. 2009;41: 712–717. 10.1038/ng.377  [DOI](https://doi.org/10.1038/ng.377) | [PMC free article](/articles/PMC3039280/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19430482/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Multiple%20Novel%20Loci%20are%20Associated%20with%20Indices%20of%20Renal%20Function%20and%20Chronic%20Kidney%20Disease&author=A%20Kottgen&author=NL%20Glazer&author=A%20Dehghan&author=J%20Hwang%20S-&author=R%20Katz&volume=41&publication_year=2009&pages=712-717&pmid=19430482&doi=10.1038/ng.377&)

28. Köttgen A, Pattaro C, Böger CA, Fuchsberger C, Olden M, Glazer NL, et al. New loci associated with kidney function and chronic kidney disease. Nat Genet. 2010;42: 376–384. 10.1038/ng.568  [DOI](https://doi.org/10.1038/ng.568) | [PMC free article](/articles/PMC2997674/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20383146/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=New%20loci%20associated%20with%20kidney%20function%20and%20chronic%20kidney%20disease&author=A%20K%C3%B6ttgen&author=C%20Pattaro&author=CA%20B%C3%B6ger&author=C%20Fuchsberger&author=M%20Olden&volume=42&publication_year=2010&pages=376-384&pmid=20383146&doi=10.1038/ng.568&)

29. Chambers JC, Zhang W, Lord GM, van der Harst P, Lawlor DA, Sehmi JS, et al. Genetic loci influencing kidney function and chronic kidney disease in man. Nat Genet. 2010;42: 373–375. 10.1038/ng.566  [DOI](https://doi.org/10.1038/ng.566) | [PMC free article](/articles/PMC3748585/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20383145/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Genetic%20loci%20influencing%20kidney%20function%20and%20chronic%20kidney%20disease%20in%20man&author=JC%20Chambers&author=W%20Zhang&author=GM%20Lord&author=P%20van%20der%20Harst&author=DA%20Lawlor&volume=42&publication_year=2010&pages=373-375&pmid=20383145&doi=10.1038/ng.566&)

30. Gudbjartsson DF, Holm H, Indridason OS, Thorleifsson G, Edvardsson V, Sulem P, et al. Association of Variants at UMOD with Chronic Kidney Disease and Kidney Stones—Role of Age and Comorbid Diseases. PLoS Genet. 2010;6 10.1371/journal.pgen.1001039  [DOI](https://doi.org/10.1371/journal.pgen.1001039) | [PMC free article](/articles/PMC2912386/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20686651/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet&title=Association%20of%20Variants%20at%20UMOD%20with%20Chronic%20Kidney%20Disease%20and%20Kidney%20Stones%E2%80%94Role%20of%20Age%20and%20Comorbid%20Diseases&author=DF%20Gudbjartsson&author=H%20Holm&author=OS%20Indridason&author=G%20Thorleifsson&author=V%20Edvardsson&volume=6&publication_year=2010&pmid=20686651&doi=10.1371/journal.pgen.1001039&)
