# Association of a single nucleotide polymorphism near the interleukin-28B gene with response to hepatitis C therapy in HIV/hepatitis C virus-coinfected patients

## Metadata
**Authors:** Norma I Rallón, Susanna Naggie, José M Benito, José Medrano, Clara Restrepo, David Goldstein, Kevin V Shianna, Eugenia Vispo, Alex Thompson, John McHutchison, Vincent Soriano
**Journal:** AIDS (London, England)
**Date:** 2010 May 15
**DOI:** [10.1097/QAD.0b013e3283391d6d](https://doi.org/10.1097/QAD.0b013e3283391d6d)
**PMID:** 20389235
**PMCID:** PMC4892373
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4892373/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC4892373/pdf/nihms787052.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC4892373/pdf/nihms787052.pdf)

## Abstract

**Background:** 
Given that peginterferon–ribavirin treatment is poorly tolerated, there is interest in the identification of predictors of response, particularly in HIV/hepatitis C virus (HCV)-coinfected patients that respond less than HCV-monoinfected individuals. A single nucleotide polymorphism (SNP) near the IL28B gene (rs12979860) has been shown to predict treatment response in HCV-monoinfected patients carrying genotype 1. Information is lacking for HIV/HCV-coinfected individuals and/or other HCV genotypes.

**Methods:** 
From 650 HIV/HCV-coinfected patients, we identified those who had completed a course of peginterferon–ribavirin therapy with a validated outcome and available repository DNA. The rs12979860 SNP was examined in a blinded fashion.

**Results:** 
A total of 164 patients were included in the final IL28B genotyping analysis, 90 (55%) of whom achieved sustained virological response (SVR). HCV genotype distribution was as follows: HCV-1 58%, HCV-3 31% and HCV-4 11%. Overall, the SVR rate was higher in patients with CC than in those CT/TT genotypes: 56 of 75 (75%) versus 34 of 89 (38%) (P <0.0001). The effect of the SNP was seen in HCV genotypes 1 and 4 but not in HCV genotype 3 carriers. In the multivariable analysis (odds ratio; 95% confidence interval; P value), the rs12979860 CC genotype was a strong predictor of SVR (3.7; 1.6–8.5; 0.002), independent of HCV genotype 3 (8.0; 3.1–21.0; <0.001), serum HCV-RNA less than 600 000 IU/ml (11.9; 3.8–37.4; <0.001) and lack of advanced liver fibrosis (3.5; 1.4–8.9; 0.009).

**Conclusion:** 
The rs12979860 SNP located near the IL28B gene is associated with HCV treatment response in HIV-infected patients with chronic hepatitis C due to genotypes 1 or 4. Thus, IL28B genotyping should be considered as part of the treatment decision algorithm in this difficult-to-treat population.

Keywords: genetic polymorphism, hepatitis C virus treatment, HIV/hepatitis C virus coinfection, interferon-λ, IL-28B, pegylated interferon, rs12979860, single nucleotide polymorphism

### Background

Given that peginterferon–ribavirin treatment is poorly tolerated, there is interest in the identification of predictors of response, particularly in HIV/hepatitis C virus (HCV)-coinfected patients that respond less than HCV-monoinfected individuals. A single nucleotide polymorphism (SNP) near the *IL28B* gene (rs12979860) has been shown to predict treatment response in HCV-monoinfected patients carrying genotype 1. Information is lacking for HIV/HCV-coinfected individuals and/or other HCV genotypes.

### Methods

From 650 HIV/HCV-coinfected patients, we identified those who had completed a course of peginterferon–ribavirin therapy with a validated outcome and available repository DNA. The rs12979860 SNP was examined in a blinded fashion.

### Results

A total of 164 patients were included in the final *IL28B* genotyping analysis, 90 (55%) of whom achieved sustained virological response (SVR). HCV genotype distribution was as follows: HCV-1 58%, HCV-3 31% and HCV-4 11%. Overall, the SVR rate was higher in patients with CC than in those CT/TT genotypes: 56 of 75 (75%) versus 34 of 89 (38%) (*P* <0.0001). The effect of the SNP was seen in HCV genotypes 1 and 4 but not in HCV genotype 3 carriers. In the multivariable analysis (odds ratio; 95% confidence interval; *P* value), the rs12979860 CC genotype was a strong predictor of SVR (3.7; 1.6–8.5; 0.002), independent of HCV genotype 3 (8.0; 3.1–21.0; <0.001), serum HCV-RNA less than 600 000 IU/ml (11.9; 3.8–37.4; <0.001) and lack of advanced liver fibrosis (3.5; 1.4–8.9; 0.009).

### Conclusion

The rs12979860 SNP located near the *IL28B* gene is associated with HCV treatment response in HIV-infected patients with chronic hepatitis C due to genotypes 1 or 4. Thus, *IL28B* genotyping should be considered as part of the treatment decision algorithm in this difficult-to-treat population.

## Introduction

Hepatitis C virus (HCV) infects more than 175 million people worldwide [[1](#R1)]. In western countries, HCV is the leading cause of end-stage liver disease and hepatocellular carcinoma, as well as the main indication for liver transplantation [[1](#R1)]. HCV and HIV-1 share routes of transmission and establish chronic infections; therefore, coinfection is relatively common (15–40%) [[2](#R2)]. The course of HCV-related liver disease is accelerated in dually infected individuals [[3](#R3),[4](#R4)], and thereby HCV has emerged as an important cause of morbidity and mortality in persons infected with HIV-1 [[5](#R5)], especially because successful antiretroviral therapy has dramatically reduced the rate of opportunistic illnesses. Current therapy for chronic hepatitis C is based on a combination of peginterferon-α (pegIFN) and ribavirin (RBV) given for 6–18 months, depending on early viral kinetics and genotype [[6](#R6)]. Unfortunately, the medications are poorly tolerated and result in low response rates, with only half of patients achieving HCV clearance [[6](#R6)]. This figure is lower in HIV/HCV-coinfected patients [[2](#R2)]. Thus, identification of predictors of treatment success is desirable in order to select the best candidates for current therapy. Among others, infection due to HCV genotypes 2 or 3, low serum HCV-RNA, null or minimal liver fibrosis and younger age are the best predictors of response to therapy [[7](#R7)].

Three independent genome-wide association studies have recently identified several single nucleotide polymorphisms (SNPs) around the *IL28B* gene (coding for IFN-λ-3) that are strongly associated with treatment outcomes in HCV-monoinfected individuals [[8](#R8)–[10](#R10)]. A role for *IL28B* genetic variation in spontaneous HCV clearance has been subsequently established [[11](#R11),[12](#R12)]. The SNP with the strongest association, rs12979860, is located on chromosome 19q13, 3 kb upstream of the *IL28B* gene. The rs12979860 CC genotype is associated with a more than two-fold greater rate of sustained virological response (SVR) than the CT or TT genotypes in different HCV genotype 1 patients of different ethnicities [[8](#R8)]. However, information about the impact of this SNP on treatment outcomes in HIV/HCV-coinfected patients has not been examined yet, and similarly its effect on HCV genotypes other than 1 has only been explored in one study conducted in HCV-monoinfected individuals [[11](#R11)]. Herein, we investigate the role of the rs12979860 SNP in a well characterized cohort of HIV/HCV-coinfected individuals carrying distinct HCV genotypes that were treated with pegIFN–RBV.

## Participants and methods

### Study population

Patients were recruited at Hospital Carlos III, a reference HIV clinic located in Madrid, Spain. From an initial cohort of 650 HIV/HCV-coinfected patients with regular follow-up, whose characteristics have already been described elsewhere [[13](#R13)], we selected 198 individuals who had completed a course of therapy with pegIFN–RBV and had validated outcomes. Only patients who were IFN-naive were included in this analysis. Patients with hepatitis B virus (HBV) coinfection were excluded.

A second group of HIV-infected individuals with evidence of spontaneous HCV clearance was also identified and invited to participate in the study. They were IFN treatment-naive individuals with reactive HCV antibodies in the absence of detectable serum HCV-RNA at least 1 year after the first documented reactive HCV serology.

Treatment regimens included pegIFN alpha 2a or 2b at standard doses (180 μg per week or 1.5 μg/kg per week, respectively) plus weight-adjusted RBV (1000 mg/day for patients weighting <75 kg and 1200 mg/day for patients weighting >75 kg). Following international guidelines [[2](#R2)], patients with HCV genotypes 1 or 4 received either 48 or 72 weeks of treatment and patients with HCV genotype 3 received 24 or 48 weeks of treatment, according to virological response at week 4. Early stopping rules were applied for patients with suboptimal virological response at weeks 12 and 24 [[2](#R2)]. Finally, SVR was defined as undetectable serum HCV-RNA 24 weeks after the end of treatment; otherwise patients were considered as relapsers [[14](#R14)] and were excluded from the SVR group. For the purpose of this study, relapsers were considered along with nonresponders, who were patients experiencing suboptimal virological response during the treatment period, and accordingly did not complete the planned duration of therapy. Patients with poor drug compliance and/or who discontinued therapy due to side-effects were excluded from the nonresponder group.

To participate in the study, written informed consent for genetic testing including *IL28B* polymorphisms and other potential markers was obtained from all individuals, and the study protocol was evaluated and approved by the hospital ethics committee. Both plasma and peripheral blood mononuclear cells (PBMCs) were stored for all patients.

### Hepatitis C virus viremia and genotyping

Plasma HCV-RNA was measured using a real-time PCR assay (COBAS TaqMan, Roche, Barcelona, Spain), which has a lower limit of detection of 10 IU/ml. HCV genotyping was performed using a commercial real-time PCR hybridization assay (Versant HCV Genotype v2.0 LiPA; Siemens, Barcelona, Spain), which maximally reduces the chances of HCV genotype misclassification [[15](#R15)]. Plasma HIV-RNA was measured using Versant HIV-1 RNA v3.0 (Siemens), which has a lower limit of detection of 50 copies/ml.

### Liver fibrosis staging

The extent of liver fibrosis was measured using transient elastography by FibroScan (Echosens, Paris, France). Details about this noninvasive method, the examination procedure and correlation of liver fibrosis estimates with liver biopsy have been reported elsewhere [[16](#R16),[17](#R17)]. The median value of all tests per patient is expressed in kiloPascal (kPa) units. Advanced liver fibrosis (severe fibrosis or cirrhosis, corresponding to METAVIR scores F3 and F4) was defined for liver stiffness values 9.5 kPa or higher, according to previous reports from both HCV-monoinfected and HIV/HCV-coinfected patients [[18](#R18),[19](#R19)].

### rs12979860 single nucleotide polymorphism genotyping

Genotyping was performed by the Duke Institute for Genome Sciences and Policy. Genotyping was conducted in a blinded fashion on DNA specimens collected from each individual, using the 5′ nuclease assay with allele-specific TaqMan probes (ABI TaqMan allelic discrimination kit and the ABI7900HT Sequence Detection System (Applied Biosystems, Carlsbad, California, USA) [[20](#R20)]. Genotyping calls were manually inspected and verified prior to release.

### Statistical analyses

The main characteristics of the study population and the different parameters evaluated are expressed as median (interquartile range). Comparisons between groups were carried out using nonparametric analysis of variance (ANOVA). Associations between different qualitative parameters were explored using χ^2^-test or Fishers’s exact test, as appropriate. Univariable (χ^2^-test) and multivariable (logistic regression) analyses were used to determine predictors of treatment outcome. All statistical analyses were performed using the SPSS software version 13 (SPSS Inc., Chicago, Illinois, USA). All *P* values were two-tailed and were considered as significant only when lower than 0.05.

## Results

### Study population

Of the 198 HIV/HCV-coinfected patients treated for hepatitis C who met the study entry criteria, 34 did not have adequate stored human cell DNA material to be included in the genetic analysis or refused to provide genetic testing consent. Therefore, the final study population consisted of 164 HIV/HCV-coinfected patients who completed a full course of HCV therapy. From the second group of individuals who cleared HCV spontaneously, 24 could be enrolled in the study.

[Table 1](#T1) summarizes the main characteristics of the HIV/HCV-coinfected study population. Of note, all patients were of European ancestry and no Asians or Africans were present in this group. Up to 83% reported prior intravenous drug use as the primary route of transmission. The median CD4 count was 468 (345–647) cells/μl, with 73% of patients on antiretroviral therapy and with undetectable plasma HIV-RNA. In untreated HIV-infected individuals, median plasma HIV-RNA was 3.0 (2.1–3.7) log copies/ml and CD4 cell count was 456 (350–616) cells/μl. HCV genotype distribution was as follows: HCV-1 58%, HCV-3 31% and HCV-4 11%. Most patients (74%) had no evidence of advanced liver fibrosis.

### Table 1.

| Characteristics | Total population (N = 164) | SVR (N = 90) | NR (N = 74) | P (SVR vs. NR) |
| --- | --- | --- | --- | --- |
| Median age [years (interquartile range)] | 42 (3945) | 41 (38–45) | 43 (39–46) | 0.46 |
| Male sex [n (%)] | 122 (74) | 64 (53) | 58 (47) | 0.29 |
| Median CD4 cell count [cells/μl (interquartile range)] | 468 (345–647) | 456 (327–644) | 504 (371–720) | 0.40 |
| Patients on antiretroviral therapy [n (%)] | 120 (73) | 65 (72) | 55 (74) | 0.74 |
| Median plasma HIV-RNA in patients with no antiretroviral therapy [log copies/ml (interquartile range)] | 3.0 (2.1–3.7) | 3.0 (2.2–3.7) | 3.0 (2.1–3.8) | 0.89 |
| Median serum HCV-RNA [log IU/ml (interquartile range)] | 6.7 (5.9–7.0) | 6.2 (5.4–6.7) | 6.9 (6.6–7.2) | <0.0001 |
| HCV genotypes [n (%)] |  |  |  |  |
| 1 | 95 (58) | 40 (44) | 55 (74) | <0.0001 |
| 3 | 51 (31) | 43 (48) | 8 (11) |   |
| 4 | 18 (11) | 7 (8) | 11 (15) |   |
| Median liver fibrosis stiffness [kPa (interquartile range)] | 6.7 (5.0–10.0) | 6.0 (4.8–8.1) | 7.6 (6.0–13.1) | <0.001 |

Table 1 Caption: Main baseline characteristics of the hepatitis C treated study population.

The overall SVR rate in patients completing the planned duration of therapy was 55% (90/164). Patients with SVR and nonresponders were comparable in terms of age, CD4 cell counts and plasma HIV-RNA. However, these two groups differed in pretreatment serum HCV-RNA, liver fibrosis stage and HCV genotype distribution ([Table 1](#T1)). There were no significant differences according to HCV genotype in terms of HCV-RNA level, CD4 cell counts, age, liver fibrosis stage and proportion of patients with suppressed HIV viremia.

The main demographics of the group of 24 HIV-infected individuals who had cleared HCV spontaneously did not differ significantly from the group with HIV/HCV coinfection who had been treated with pegIFN–RBV. Briefly, their median age was 43 (35–54) years, 75% were men, median CD4 cell count was 498 (356–791) cells/μl and 75% were under antiretroviral therapy.

### Prevalence of rs12979860 genotypes in the study population

All 188 patients were genotyped for the rs12979860 allele. [Table 2](#T2) summarizes the distribution of polymorphisms according to HCV genotype and treatment response in the HIV/HCV-coinfected population. The population was in Hardy–Weinberg equilibrium (*P* = 0.77) and the C allele frequency was 68%. The prevalence of the distinct rs12979860 genotypes was as follows: CC 46%, CT 44% and TT 10%. Overall, CC, CT and TT carriers were comparable in terms of age, CD4 cell count, serum HCV-RNA, liver fibrosis stage and plasma HIV-RNA (data not shown). However, the proportion of patients infected with HCV genotype 3 was significantly higher in CC than in CT and TT carriers, whereas HCV genotypes 1 and 4 were more prevalent in TT and CT than in CC carriers ([Table 2](#T2)).

### Table 2.

|   | CC (N = 75) | CT (N = 73) | TT (N = 16) | P |
| --- | --- | --- | --- | --- |
| HCV genotype [n (%)] |  |  |  |  |
| 1 (N = 95) | 34 (45) | 51 (70) | 10 (63) | 0.001 |
| 3 (N = 51) | 35 (47) | 14 (19) | 2 (12) |   |
| 4 (N = 18) | 6 (8) | 8 (11) | 4 (25) |   |
| Treatment outcome [n (%)] |  |  |  |  |
| SVR (N = 90) | 56 (75) | 27 (37) | 7 (44) | <0.001 |
| NR (N = 74) | 19 (25) | 46 (63) | 9 (56) |   |

Table 2 Caption: Distribution of hepatitis C virus genotypes and treatment outcome according to rs12979860 genotypes.

The frequency of the rs12979860 CC genotype in patients with spontaneous HCV clearance was significantly higher than in chronically HIV/HCV-coinfected patients (75 vs. 46%, respectively; *P* = 0.007). In contrast with HCV genotypes 1 and 4, the frequency of the CC genotype did not differ significantly in patients chronically infected with HCV genotype 3 compared with those with spontaneous HCV clearance: 35 of 51 (69%) vs. 18 of 24 (75%), respectively.

### Treatment response according to rs12979860 genotype

In this cohort, the SVR rate was significantly higher in patients with the CC genotype than in those with CTor TT genotype (75, 37 and 44%, respectively, *P* <0.0001), with no significant differences between CT and TT genotypes, which is consistent with previous descriptions in HCV monoinfection [[8](#R8)]. For this reason, the recessive model (CC vs. CT/TT genotypes) instead of the additive model (CC, CT, TT) was used in subsequent analyses.

The SVR rate was higher in patients with CC genotypes than in those with CT/TT genotypes across all HCV genotypes. However, it was highly significant for HCV genotype 1 (65 vs. 30%; *P* = 0.001) and almost reached statistical significance for HCV genotype 4 (67 vs. 25%; *P* = 0.087), despite the limited number of patients examined who were infected with this genotype. In contrast, differences were negligible in HCV genotype 3 infected persons (86 vs. 81%; *P* = 0.684) ([Fig. 1](#F1)).

### Fig. 1.

![Fig. 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8877/4892373/c632a11109e1/nihms787052f1.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4892373_nihms787052f1.jpg)

Rate of sustained virological response in distinct hepatitis C virus genotypes according to rs12979860 single nucleotide polymorphism.

### Multivariable analysis

Given that several variables (age, sex, serum HCV-RNA, presence of detectable HIV-RNA, HCV genotype and extent of liver fibrosis) have previously been reported to influence HCV treatment outcomes, we completed univariable analyses (odds ratio; 95% confidence interval; *P* value) to assess the association of each of these factors with SVR in our HIV/HCV-coinfected population. In addition to the IL28B CC genotype (4.8; 2.4–9.4; <0.0001), HCV genotype 3 (7.5; 3.3–17.5; <0.0001), pretreatment serum HCV-RNA less than 600 000 IU/ml (8.4; 3.1–22.8; <0.0001) and lack of advanced liver fibrosis (2.5; 1.2–5.2; 0.01) were significantly associated with SVR. Age, sex and presence of detectable HIV-RNA were not significantly associated with SVR in the univariable analysis.

In the multivariable logistic regression analysis, we included those predictor variables, along with the rs12979860 genotype, as independent variables of treatment outcome. As shown in [Fig. 2](#F2), the rs12979860 CC genotype remained a strong predictor of SVR (3.7; 1.6–8.4; 0.002), independent of HCV genotype 3 (8.0; 3.1–21.0; <0.001), baseline serum HCV-RNA less than 600 000 IU/ml (11.9; 3.8–37.4; <0.001) and liver fibrosis stages F0–F2 (3.5; 1.4–8.9; 0.009). Similar results were observed when the logistic regression analysis was limited to HCV genotype 1 patients (rs12979860 CC genotype (5.6; 1.8–17.4; 0.003), baseline serum HCV-RNA less than 600 000 IU/ml (28.2; 4.8–164.4; <0.001) and liver fibrosis stages F0–F2 (6.6; 1.5–30.1; 0.014).

### Fig. 2.

![Fig. 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8877/4892373/edf32d32a3d6/nihms787052f2.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4892373_nihms787052f2.jpg)

Predictors of sustained virological response to peginterferon–ribavirin therapy in HIV/hepatitis C virus-coinfected patients.

The joint contribution of the four variables identified as independent predictors of HCV treatment outcome was further examined. Patients were stratified according to the number of protective factors (rs12979860 CC genotype, HCV genotype 3, baseline serum HCV-RNA <600 000 IU/ml and liver fibrosis stage F0–F2). SVR rate in patients presenting with less than two protective factors was very low, whereas this rate significantly increased in patients having two or more factors. This profile was reproduced when considering exclusively the subset of patients infected with HCV genotypes 1 and 4 (SVR rates were 12, 22, 68 and 100% for patients with 0, 1, 2 and 3 factors, respectively; *P* <0.0001). However, in patients infected with HCV genotype 3, the SVR was 50%, even in the absence of any other protective factors, and increased to more than 80% once one or more further factors were present ([Table 3](#T3)).

### Table 3.

|   | 0 | 1 | 2 | 3 | 4 | P |
| --- | --- | --- | --- | --- | --- | --- |
| Total population (N = 164) (%) | 12 | 23 | 75 | 88 | 100 | <0.0001 |
| HCV-1 and HCV-4 genotype patients (N = 113) (%) | 12 | 22 | 68 | 100 | NA | <0.0001 |
| HCV-3 genotype patients (N = 51) (%) | 50 | 84 | 82 | 100 | NA | 0.372 |

Table 3 Caption: Sustained virological response rates according to the number of protective factors (low serum HCV-RNA, hepatitis C virus genotype 3, lack of advanced liver fibrosis and rs12979860 CC genotype).

## Discussion

This is the first study to report a strong influence of the rs12979860 SNP located near the *IL28B* gene on response to pegIFN–RBV in HIV/HCV-coinfected patients. Overall, patients with the CC genotype had a 3.7-fold greater chance of achieving SVR than CTor TT carriers. This observation is in line with results recently obtained in HCV-monoinfected individuals carrying HCV genotype 1 [[8](#R8)]. Although, the overall impact of the CC genotype compared with other predictors of hepatitis C treatment response was attenuated in this HIV co-infected cohort. Other studies of this SNP found that it was the strongest predictor of SVR, more so than serum HCV-RNA less than 600 000 IU/ml or lack of advanced liver fibrosis [[8](#R8)]. This may be due to the overall lower SVR rates and higher HCV-RNA levels seen in HIV-infected individuals compared with HCV-monoinfected patients.

The improved SVR noted for carriers of the CC genotype infected with HCV genotypes 1 and 4 in our study did not translate to HCV genotype 3. Although the prevalence of the CC genotype and the SVR rate was higher in HCV genotype 3 carriers (69 and 84%, respectively), the CC genotype did not correlate with improved SVR in these patients. These differences suggest that the beneficial effect of the CC genotype on HCV treatment outcomes is mainly recognized in patients with difficult-to-treat HCV genotypes 1 and 4. This finding confirms that of Rauch *et al*. [[11](#R11)], who found no association of polymorphisms near the IL-28B gene with SVR in HCV-monoinfected patients carrying HCV genotypes 2 and 3.

The logistic regression model showed that the association of the CC genotype with SVR was independent of other well known predictors of SVR, such as low baseline serum HCV-RNA, HCV genotype 3 and lack of advanced liver fibrosis. In the study by Ge *et al*. [[8](#R8)], patients with the CC genotype had lower serum HCV-RNA, considering viremia as a continuous variable. However, this relationship was not significant when stratifying patients with HCV-RNA below or above 600 000 IU/ml. Likewise, we did not find any significant association between the CC genotype and serum HCV-RNA levels.

In our cohort of Spaniards who completed a course of pegIFN–RBV therapy, the overall CC genotype frequency was 46%, which is higher than that reported in North American whites (39%) [[8](#R8)]. It has long been recognized that treatment response rates in European HCV cohorts are generally better than in US cohorts; a higher frequency of patients with the good response CC genotype may be one of the explanations for this phenomenon. It was also notable that a higher prevalence of the CC genotype was present in patients who spontaneously had cleared HCV infection compared with chronically infected patients, consistent with the recognition in HIV-negative individuals that spontaneous HCV clearance is more frequent in CC than in CT/TT carriers [[12](#R12)]. However, this effect was not recognized for HCV genotype 3 and was driven by the protective effect of the CC genotype on chronification of HCV genotypes 1 and 4. It should be noted that our coinfected population was mainly represented by former intravenous drug users and most likely had been exposed to HCV before being infected with HIV; thus, any potential influence of HIV on the rate of HCV persistence could not be assessed.

Altogether these findings support a pivotal role of the CC genotype in the control of HCV replication in different scenarios, most likely mediated through innate immune responses. *IL28B* belongs to the IFN-λ family and interacts with its receptor to induce antiviral responses [[21](#R21)], including those against HCV [[22](#R22)]. Polymorphisms near the *IL28B* gene could affect the expression of IL28, thus influencing the response to pegIFN–RBV [[10](#R10)]. However, the precise mechanism of its effect remains unclear and further in-vitro and in-vivo studies are required to answer this question.

Beyond the relevance for the investigation of new therapeutic strategies against HCV, including IFN-λ, our findings suggest that *IL28B* genotyping may play a critical role in the clinical management of HIV/HCV-coinfected patients carrying HCV genotypes 1 and 4, the most difficult-to-treat population. In fact, consideration of the *IL28B* genotype along with other predictors of treatment outcome might help to make better treatment decisions regarding who should be treated with pegIFN–RBV. Treatment should be encouraged in patients with a high chance of success, such as those with two to four good predictors of SVR, whereas in the remaining patients with a very poor chance of SVR, advice to wait for new antivirals against HCV might be more appropriate, considering the frequent side-effects of the current medication while little hope for gain.

In summary, our results are the first providing important therapeutic information about the impact of *IL28B* variants on hepatitis C treatment outcomes in HIV/HCV-coinfected patients carrying distinct HCV genotypes. We demonstrate a significant influence of the rs12979860 SNP located near the *IL28B* gene on SVR in patients infected with HCV genotypes 1 and 4. The recognition that the CC genotype seems to favor spontaneous HCV clearance and enhance virus elimination following interferon-based therapy is of much interest. Of note, these effects were not seen for HCV genotype 3. Altogether, our findings support the fact that *IL28B* genotyping should be part of the treatment decision algorithm in this difficult-to-treat population.

## Acknowledgments

We would like to thank all patients who participated in the study. This work was supported in part by grants from Fundación Investigacion y Educacion en SIDA (IES), Red de Investigacion en SIDA (RIS, FIS-RD06/0006), Agencia Lain Entralgo and the NEAT European project.

## Footnotes

## References

1. Lauer G, Walker B. Hepatitis C virus infection. N Engl J Med. 2001;345:41–52. doi: 10.1056/NEJM200107053450107.  [DOI](https://doi.org/10.1056/NEJM200107053450107) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11439948/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Hepatitis%20C%20virus%20infection&author=G%20Lauer&author=B%20Walker&volume=345&publication_year=2001&pages=41-52&pmid=11439948&doi=10.1056/NEJM200107053450107&)

2. Soriano V, Puoti M, Sulkowski M, Cargnel A, Benhamou Y, Peters M, et al. Care of patients coinfected with HIV and hepatitis C virus: 2007 updated recommendations from the HCV-HIV International Panel. AIDS. 2007;21:1073–1089. doi: 10.1097/QAD.0b013e3281084e4d.  [DOI](https://doi.org/10.1097/QAD.0b013e3281084e4d) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17502718/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=AIDS&title=Care%20of%20patients%20coinfected%20with%20HIV%20and%20hepatitis%20C%20virus:%202007%20updated%20recommendations%20from%20the%20HCV-HIV%20International%20Panel&author=V%20Soriano&author=M%20Puoti&author=M%20Sulkowski&author=A%20Cargnel&author=Y%20Benhamou&volume=21&publication_year=2007&pages=1073-1089&pmid=17502718&doi=10.1097/QAD.0b013e3281084e4d&)

3. Graham C, Baden L, Yu E, Mrus J, Carnie J, Heeren T, et al. Influence of HIV infection on the course of hepatitis C virus infection: a meta-analysis. Clin Infect Dis. 2001;33:562–569. doi: 10.1086/321909.  [DOI](https://doi.org/10.1086/321909) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11462196/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Infect%20Dis&title=Influence%20of%20HIV%20infection%20on%20the%20course%20of%20hepatitis%20C%20virus%20infection:%20a%20meta-analysis&author=C%20Graham&author=L%20Baden&author=E%20Yu&author=J%20Mrus&author=J%20Carnie&volume=33&publication_year=2001&pages=562-569&pmid=11462196&doi=10.1086/321909&)

4. Martin-Carbonero L, Benhamou Y, Puoti M, Berenguer J, Mallolas J, Quereda C, et al. Incidence and predictors of severe liver fibrosis in HIV-infected patients with chronic hepatitis C: a European collaborative study. Clin Infect Dis. 2004;38:128–133. doi: 10.1086/380130.  [DOI](https://doi.org/10.1086/380130) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14679458/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Infect%20Dis&title=Incidence%20and%20predictors%20of%20severe%20liver%20fibrosis%20in%20HIV-infected%20patients%20with%20chronic%20hepatitis%20C:%20a%20European%20collaborative%20study&author=L%20Martin-Carbonero&author=Y%20Benhamou&author=M%20Puoti&author=J%20Berenguer&author=J%20Mallolas&volume=38&publication_year=2004&pages=128-133&pmid=14679458&doi=10.1086/380130&)

5. Weber R, Sabin C, Friis-Møller N, Reiss P, El-Sadr W, Kirk O, et al. Liver-related deaths in persons infected with the HIV: the D:A:D study. Arch Intern Med. 2006;166:1632–1641. doi: 10.1001/archinte.166.15.1632.  [DOI](https://doi.org/10.1001/archinte.166.15.1632) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16908797/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arch%20Intern%20Med&title=Liver-related%20deaths%20in%20persons%20infected%20with%20the%20HIV:%20the%20D:A:D%20study&author=R%20Weber&author=C%20Sabin&author=N%20Friis-M%C3%B8ller&author=P%20Reiss&author=W%20El-Sadr&volume=166&publication_year=2006&pages=1632-1641&pmid=16908797&doi=10.1001/archinte.166.15.1632&)

6. Ghany M, Strader D, Thomas D, Seeff L. American Association for the Study of Liver Diseases. Diagnosis, management, and treatment of hepatitis C: an update. Hepatology. 2009;49:1335–1374. doi: 10.1002/hep.22759.  [DOI](https://doi.org/10.1002/hep.22759) | [PMC free article](/articles/PMC7477893/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19330875/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hepatology&title=American%20Association%20for%20the%20Study%20of%20Liver%20Diseases.%20Diagnosis,%20management,%20and%20treatment%20of%20hepatitis%20C:%20an%20update&author=M%20Ghany&author=D%20Strader&author=D%20Thomas&author=L%20Seeff&volume=49&publication_year=2009&pages=1335-1374&pmid=19330875&doi=10.1002/hep.22759&)

7. Kau A, Vermehren J, Sarrazin C. Treatment predictors of a sustained virologic response in hepatitis B and C. J Hepatol. 2008;49:634–651. doi: 10.1016/j.jhep.2008.07.013.  [DOI](https://doi.org/10.1016/j.jhep.2008.07.013) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18715665/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Hepatol&title=Treatment%20predictors%20of%20a%20sustained%20virologic%20response%20in%20hepatitis%20B%20and%20C&author=A%20Kau&author=J%20Vermehren&author=C%20Sarrazin&volume=49&publication_year=2008&pages=634-651&pmid=18715665&doi=10.1016/j.jhep.2008.07.013&)

8. Ge D, Fellay J, Thompson A, Simon J, Shianna K, Urban T, et al. Genetic variation in IL28B predicts hepatitis C treatment-induced viral clearance. Nature. 2009;461:399–401. doi: 10.1038/nature08309.  [DOI](https://doi.org/10.1038/nature08309) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19684573/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Genetic%20variation%20in%20IL28B%20predicts%20hepatitis%20C%20treatment-induced%20viral%20clearance&author=D%20Ge&author=J%20Fellay&author=A%20Thompson&author=J%20Simon&author=K%20Shianna&volume=461&publication_year=2009&pages=399-401&pmid=19684573&doi=10.1038/nature08309&)

9. Suppiah V, Moldovan M, Ahlenstiel G, Berg T, Weltman M, Abate ML, et al. IL28B is associated with response to chronic hepatitis C interferon-alpha and ribavirin therapy. Nat Genet. 2009;41:1100–1104. doi: 10.1038/ng.447.  [DOI](https://doi.org/10.1038/ng.447) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19749758/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=IL28B%20is%20associated%20with%20response%20to%20chronic%20hepatitis%20C%20interferon-alpha%20and%20ribavirin%20therapy&author=V%20Suppiah&author=M%20Moldovan&author=G%20Ahlenstiel&author=T%20Berg&author=M%20Weltman&volume=41&publication_year=2009&pages=1100-1104&pmid=19749758&doi=10.1038/ng.447&)

10. Tanaka Y, Nishida N, Sugiyama M, Kurosaki M, Matsuura K, Sakamoto N, et al. Genome-wide association of IL28B with response to pegylated interferon-alpha and ribavirin therapy for chronic hepatitis C. Nat Genet. 2009;41:1105–1109. doi: 10.1038/ng.449.  [DOI](https://doi.org/10.1038/ng.449) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19749757/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Genome-wide%20association%20of%20IL28B%20with%20response%20to%20pegylated%20interferon-alpha%20and%20ribavirin%20therapy%20for%20chronic%20hepatitis%20C&author=Y%20Tanaka&author=N%20Nishida&author=M%20Sugiyama&author=M%20Kurosaki&author=K%20Matsuura&volume=41&publication_year=2009&pages=1105-1109&pmid=19749757&doi=10.1038/ng.449&)

11. Rauch A, Kutalik Z, Descombes P, Cai T, di Iulio J, Mueller T, et al. Genetic variation in IL28B Is associated with chronic hepatitis C and treatment failure: a genome wide association study. Gastroenterology. doi: 10.1053/j.gastro.2009.12.056. in press.  [DOI](https://doi.org/10.1053/j.gastro.2009.12.056) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20060832/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gastroenterology&title=Genetic%20variation%20in%20IL28B%20Is%20associated%20with%20chronic%20hepatitis%20C%20and%20treatment%20failure:%20a%20genome%20wide%20association%20study&author=A%20Rauch&author=Z%20Kutalik&author=P%20Descombes&author=T%20Cai&author=J%20di%20Iulio&pmid=20060832&doi=10.1053/j.gastro.2009.12.056&)

12. Thomas D, Thio C, Martin M, Qi Y, Ge D, O’Huigin C, et al. Genetic variation in IL28B and spontaneous clearance of hepatitis C virus. Nature. 2009;461:798–801. doi: 10.1038/nature08463.  [DOI](https://doi.org/10.1038/nature08463) | [PMC free article](/articles/PMC3172006/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19759533/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Genetic%20variation%20in%20IL28B%20and%20spontaneous%20clearance%20of%20hepatitis%20C%20virus&author=D%20Thomas&author=C%20Thio&author=M%20Martin&author=Y%20Qi&author=D%20Ge&volume=461&publication_year=2009&pages=798-801&pmid=19759533&doi=10.1038/nature08463&)

13. Medrano J, Resino S, Vispo E, Madejón A, Labarga P, Tuma P, et al. Hepatitis C virus (HCV) treatment uptake and changes in the prevalence of HCV genotypes in HIV-HCV coinfected patients. J Viral Hepat. doi: 10.1111/j.1365-2893.2010.01309.x. in press.  [DOI](https://doi.org/10.1111/j.1365-2893.2010.01309.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20456635/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Viral%20Hepat&title=Hepatitis%20C%20virus%20(HCV)%20treatment%20uptake%20and%20changes%20in%20the%20prevalence%20of%20HCV%20genotypes%20in%20HIV-HCV%20coinfected%20patients&author=J%20Medrano&author=S%20Resino&author=E%20Vispo&author=A%20Madej%C3%B3n&author=P%20Labarga&pmid=20456635&doi=10.1111/j.1365-2893.2010.01309.x&)

14. Medrano J, Barreiro P, Resino S, Tuma P, Rodríguez V, Vispo E, et al. Rate and timing of hepatitis C virus relapse after a successful course of pegylated interferon plus ribavirin in HIV-infected and HIV-uninfected patients. Clin Infect Dis. 2009;49:1397–1401. doi: 10.1086/630205.  [DOI](https://doi.org/10.1086/630205) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19814621/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Infect%20Dis&title=Rate%20and%20timing%20of%20hepatitis%20C%20virus%20relapse%20after%20a%20successful%20course%20of%20pegylated%20interferon%20plus%20ribavirin%20in%20HIV-infected%20and%20HIV-uninfected%20patients&author=J%20Medrano&author=P%20Barreiro&author=S%20Resino&author=P%20Tuma&author=V%20Rodr%C3%ADguez&volume=49&publication_year=2009&pages=1397-1401&pmid=19814621&doi=10.1086/630205&)

15. Chevaliez S, Bouvier-Alias M, Brillet R, Pawlotsky JM. Hepatitis C virus (HCV) genotype 1 subtype identification in new HCV drug development and future clinical practice. PLoS One. 2009;4:e8209. doi: 10.1371/journal.pone.0008209.  [DOI](https://doi.org/10.1371/journal.pone.0008209) | [PMC free article](/articles/PMC2785465/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19997618/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=Hepatitis%20C%20virus%20(HCV)%20genotype%201%20subtype%20identification%20in%20new%20HCV%20drug%20development%20and%20future%20clinical%20practice&author=S%20Chevaliez&author=M%20Bouvier-Alias&author=R%20Brillet&author=JM%20Pawlotsky&volume=4&publication_year=2009&pages=e8209&pmid=19997618&doi=10.1371/journal.pone.0008209&)

16. Ziol M, Handra-Luca A, Kettaneh A, Christidis C, Mal F, Kazemi F, et al. Noninvasive assessment of liver fibrosis by measurement of stiffness in patients with chronic hepatitis C. Hepatology. 2005;41:48–54. doi: 10.1002/hep.20506.  [DOI](https://doi.org/10.1002/hep.20506) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15690481/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hepatology&title=Noninvasive%20assessment%20of%20liver%20fibrosis%20by%20measurement%20of%20stiffness%20in%20patients%20with%20chronic%20hepatitis%20C&author=M%20Ziol&author=A%20Handra-Luca&author=A%20Kettaneh&author=C%20Christidis&author=F%20Mal&volume=41&publication_year=2005&pages=48-54&pmid=15690481&doi=10.1002/hep.20506&)

17. Castera L, Vergniol J, Foucher J, Le Bail B, Chanteloup E, Haaser M, et al. Prospective comparison of transient elastography, Fibrotest, APRI, and liver biopsy for the assessment of fibrosis in chronic hepatitis C. Gastroenterology. 2005;128:343–350. doi: 10.1053/j.gastro.2004.11.018.  [DOI](https://doi.org/10.1053/j.gastro.2004.11.018) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15685546/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gastroenterology&title=Prospective%20comparison%20of%20transient%20elastography,%20Fibrotest,%20APRI,%20and%20liver%20biopsy%20for%20the%20assessment%20of%20fibrosis%20in%20chronic%20hepatitis%20C&author=L%20Castera&author=J%20Vergniol&author=J%20Foucher&author=B%20Le%20Bail&author=E%20Chanteloup&volume=128&publication_year=2005&pages=343-350&pmid=15685546&doi=10.1053/j.gastro.2004.11.018&)

18. de Ledinghen V, Douvin C, Kettaneh A, Ziol M, Roulot D, Marcellin P, et al. Diagnosis of hepatic fibrosis and cirrhosis by transient elastography in HIV/hepatitis C virus-coinfected patients. J Acquir Immune Defic Syndr. 2006;41:175–179. doi: 10.1097/01.qai.0000194238.15831.c7.  [DOI](https://doi.org/10.1097/01.qai.0000194238.15831.c7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16394849/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Acquir%20Immune%20Defic%20Syndr&title=Diagnosis%20of%20hepatic%20fibrosis%20and%20cirrhosis%20by%20transient%20elastography%20in%20HIV/hepatitis%20C%20virus-coinfected%20patients&author=V%20de%20Ledinghen&author=C%20Douvin&author=A%20Kettaneh&author=M%20Ziol&author=D%20Roulot&volume=41&publication_year=2006&pages=175-179&pmid=16394849&doi=10.1097/01.qai.0000194238.15831.c7&)

19. Kirk G, Astemborski J, Mehta S, Spoler C, Fisher C, Allen D, et al. Assessment of liver fibrosis by transient elastography in persons with hepatitis C virus infection or HIV-hepatitis C virus coinfection. Clin Infect Dis. 2009;48:963–972. doi: 10.1086/597350.  [DOI](https://doi.org/10.1086/597350) | [PMC free article](/articles/PMC2715996/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19236273/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Infect%20Dis&title=Assessment%20of%20liver%20fibrosis%20by%20transient%20elastography%20in%20persons%20with%20hepatitis%20C%20virus%20infection%20or%20HIV-hepatitis%20C%20virus%20coinfection&author=G%20Kirk&author=J%20Astemborski&author=S%20Mehta&author=C%20Spoler&author=C%20Fisher&volume=48&publication_year=2009&pages=963-972&pmid=19236273&doi=10.1086/597350&)

20. Livak K. Allelic discrimination using fluorogenic probes and the 5′ nuclease assay. Genet Anal. 1999;14:143–149. doi: 10.1016/s1050-3862(98)00019-9.  [DOI](https://doi.org/10.1016/s1050-3862(98)00019-9) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10084106/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genet%20Anal&title=Allelic%20discrimination%20using%20fluorogenic%20probes%20and%20the%205%E2%80%B2%20nuclease%20assay&author=K%20Livak&volume=14&publication_year=1999&pages=143-149&pmid=10084106&doi=10.1016/s1050-3862(98)00019-9&)

21. Kotenko S, Gallager G, Baurin V, Lewis-Antes A, Shen M, Shah N, et al. IFN-lambdas mediate antiviral protection through a distinct class II cytokine receptor complex. Nat Immunol. 2003;4:69–77. doi: 10.1038/ni875.  [DOI](https://doi.org/10.1038/ni875) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12483210/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Immunol&title=IFN-lambdas%20mediate%20antiviral%20protection%20through%20a%20distinct%20class%20II%20cytokine%20receptor%20complex&author=S%20Kotenko&author=G%20Gallager&author=V%20Baurin&author=A%20Lewis-Antes&author=M%20Shen&volume=4&publication_year=2003&pages=69-77&pmid=12483210&doi=10.1038/ni875&)

22. Marcello T, Grakoui A, Barba-Spaeth G, Machlin E, Kotenko S, MacDonald M, et al. Interferons alpha and lambda inhibit HCV replication with distinct signal transduction and gene regulation kinetics. Gastroenterology. 2006;131:1887–1898. doi: 10.1053/j.gastro.2006.09.052.  [DOI](https://doi.org/10.1053/j.gastro.2006.09.052) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17087946/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gastroenterology&title=Interferons%20alpha%20and%20lambda%20inhibit%20HCV%20replication%20with%20distinct%20signal%20transduction%20and%20gene%20regulation%20kinetics&author=T%20Marcello&author=A%20Grakoui&author=G%20Barba-Spaeth&author=E%20Machlin&author=S%20Kotenko&volume=131&publication_year=2006&pages=1887-1898&pmid=17087946&doi=10.1053/j.gastro.2006.09.052&)
