# Pharmacogenetics of unboosted atazanavir in HIV-infected individuals in resource-limited settings: a sub-study of the AIDS Clinical Trials Group (ACTG) PEARLS study (NWCS 342)

## Metadata
**Authors:** Jose R Castillo-Mancilla, Christina L Aquilante, Michael F Wempe, Laura M Smeaton, Cynthia Firnhaber, Alberto M LaRosa, Nagalingeswaran Kumarasamy, Adriana Andrade, Gautam Baheti, Courtney V Fletcher, Thomas B Campbell, David W Haas, Samantha MaWhinney, Peter L Anderson
**Journal:** Journal of Antimicrobial Chemotherapy
**Date:** 2016 Feb 17
**DOI:** [10.1093/jac/dkw005](https://doi.org/10.1093/jac/dkw005)
**PMID:** 26892777
**PMCID:** PMC4867099
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4867099/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC4867099/pdf/dkw005.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC4867099/pdf/dkw005.pdf)

## Abstract

**Objectives:** 
The multinational PEARLS (ACTG A5175) study, conducted mainly in resource-limited settings, identified an increased treatment failure rate among HIV-infected individuals randomized to once-daily unboosted atazanavir, didanosine-EC, and emtricitabine compared with efavirenz-based regimens. We evaluated associations between selected human genetic polymorphisms and atazanavir pharmacokinetics in PEARLS.

**Methods:** 
Polymorphisms in CYP3A5, ABCB1, SLCO1B1 and NR1I2 were genotyped in PEARLS participants randomized to atazanavir plus didanosine-EC plus emtricitabine in Peru, South Africa and the USA, who also consented to genetic analysis. Non-linear mixed-effects population pharmacokinetic modelling was used to predict atazanavir oral clearance (CL/F) and concentration at 24 h (C24). Atazanavir mono-oxidation metabolites M1 and M2 were quantified from the same single-point plasma sample used to quantify the parent drug. Data were log10 transformed for statistical analysis using unpaired t-tests and one-way ANOVA and are presented as geometric mean (95% CI).

**Results:** 
Eighty-four HIV-infected participants were genotyped, including 44 Black Africans or African Americans and 28 women. Median age was 34 years. We identified 56 CYP3A5 expressers and 28 non-expressers. Atazanavir CL/F and C24 did not differ between CYP3A5 expressers and non-expressers: 13.2 (12.1–14.4) versus 12.7 L/h (11.7–13.9), P = 0.61, and 75.3 (46.1–123.0) versus 130.9 ng/mL (86.9–197.2), P = 0.14, respectively. M1/atazanavir and M2/atazanavir ratios were higher in expressers than in non-expressers: 0.0083 (0.0074–0.0094) versus 0.0063 (0.0053–0.0075), P = 0.008, and 0.0065 (0.0057–0.0073) versus 0.0050 (0.0042–0.0061), P = 0.02, respectively.

**Conclusions:** 
Expression of CYP3A5 appears to be associated with increased M1 and M2 atazanavir metabolite formation, without significantly affecting parent compound pharmacokinetics.

### Objectives

The multinational PEARLS (ACTG A5175) study, conducted mainly in resource-limited settings, identified an increased treatment failure rate among HIV-infected individuals randomized to once-daily unboosted atazanavir, didanosine-EC, and emtricitabine compared with efavirenz-based regimens. We evaluated associations between selected human genetic polymorphisms and atazanavir pharmacokinetics in PEARLS.

### Methods

Polymorphisms in *CYP3A5*, *ABCB1*, *SLCO1B1* and *NR1I2* were genotyped in PEARLS participants randomized to atazanavir plus didanosine-EC plus emtricitabine in Peru, South Africa and the USA, who also consented to genetic analysis. Non-linear mixed-effects population pharmacokinetic modelling was used to predict atazanavir oral clearance (CL/*F*) and concentration at 24 h (*C*_24_). Atazanavir mono-oxidation metabolites M1 and M2 were quantified from the same single-point plasma sample used to quantify the parent drug. Data were log_10_ transformed for statistical analysis using unpaired *t*-tests and one-way ANOVA and are presented as geometric mean (95% CI).

### Results

Eighty-four HIV-infected participants were genotyped, including 44 Black Africans or African Americans and 28 women. Median age was 34 years. We identified 56 CYP3A5 expressers and 28 non-expressers. Atazanavir CL/*F* and *C*_24_ did not differ between CYP3A5 expressers and non-expressers: 13.2 (12.1–14.4) versus 12.7 L/h (11.7–13.9), *P* = 0.61, and 75.3 (46.1–123.0) versus 130.9 ng/mL (86.9–197.2), *P* = 0.14, respectively. M1/atazanavir and M2/atazanavir ratios were higher in expressers than in non-expressers: 0.0083 (0.0074–0.0094) versus 0.0063 (0.0053–0.0075), *P* = 0.008, and 0.0065 (0.0057–0.0073) versus 0.0050 (0.0042–0.0061), *P* = 0.02, respectively.

### Conclusions

Expression of *CYP3A5* appears to be associated with increased M1 and M2 atazanavir metabolite formation, without significantly affecting parent compound pharmacokinetics.

## Introduction

Atazanavir, a PI used for the treatment of HIV-1 infection, is metabolized by cytochrome P450 (CYP) 3A4 and 3A5 subunits.^1^ To increase plasma exposure, atazanavir is typically prescribed with either low-dose ritonavir or cobicistat.^2,3^ CYP3A5 is a highly polymorphic enzyme with a bimodal pattern of hepatic expression; CYP3A5 expressers are individuals who carry at least one copy of the *CYP3A5*1* allele and exhibit up to 30-fold higher levels of protein than non-expressers.^4,5^ The frequency of *CYP3A5* expression is highest in Black Africans (50%–70%) and lowest in Caucasians (10%), and up to 25% of African Americans are homozygous for *CYP3A5*1.*^6^ The most frequent *CYP3A5* loss-of-function variant, *CYP3A5*3* (rs776746, 6986A → G), has been identified in all ethnic groups and is present in ∼85% of Caucasians, 85%–65% of Asians, 75% of Hispanics and 55% of African Americans.^5,6^ Less frequent loss-of-function variants include *CYP3A5*6* (rs10264272, 14690G → A) and *CYP3A5*7* (rs41303343, 27131-2ins T), both of which have been identified in Black Africans and African Americans, but not in other populations.^5,7^ Previous reports suggested associations between CYP3A5 non-expresser polymorphisms and metabolism of the PIs indinavir and saquinavir.^8–11^ Recent data in HIV-negative volunteers demonstrated faster atazanavir oral clearance (CL/*F*) and lower atazanavir trough concentrations (*C*_min_) in CYP3A5 expressers compared with non-expressers, particularly in non-African American males, which was lessened by ritonavir boosting.^12^ In a follow-up study, the atazanavir mono-oxidation metabolites 1 and 2 (M1 and M2) were produced faster via *CYP3A5* compared with *CYP3A4* *in vitro*, and the plasma M1/atazanavir and M2/atazanavir ratio values were 2-fold higher in CYP3A5 expressers compared with non-expressers *in vivo*. This novel metabolite signature was highly predictive of CYP3A5 expresser status and independent of race.^13^

Atazanavir is also a substrate of the efflux transporter P-glycoprotein, encoded by *ABCB1*.^14^ A study in HIV-negative healthy volunteers suggested that atazanavir CL/*F* and atazanavir *C*_min_ were associated with a particular *ABCB1* haplotype,^12^ but the relationships between PI pharmacokinetics and *ABCB1* polymorphisms have been inconsistent^15,16^ and *in vivo* effects have been modest.^17^ Organic anion transporting polypeptide 1, encoded by *SLCO1B1*, may also be relevant to atazanavir. The *SLCO1B1* c.521T → C (rs4149056) polymorphism has been associated with reduced hepatocyte uptake of lopinavir/ritonavir and increased plasma lopinavir exposure,^18,19^ but recent data did not identify an association between *SLCO1B1* polymorphisms and atazanavir pharmacokinetics.^20^ The pregnane X receptor (PXR), which is encoded by *NR1I2* and can be activated by PIs, regulates *CYP3A4*, *ABCB1* and *SLCO1B1* gene expression and has also been associated with atazanavir pharmacokinetics.^21^ Homozygosity for *NR1I2* 63396C → T (rs2472677) has been associated with increased *CYP3A4* activity, faster CL/*F* and lower atazanavir *C*_min_ in HIV-negative and HIV-positive individuals.^21,22^

Unboosted atazanavir is safe and effective compared with NNRTIs and other PIs.^23–25^ Potential advantages of unboosted atazanavir include a low incidence of hyperbilirubinaemia, lipid abnormalities and reduced pill burden.^26,27^ The Prospective Evaluation of Antiretrovirals in Resource Limited Settings (PEARLS, A5175, [NCT00084136](https://clinicaltrials.gov/ct2/show/NCT00084136)) study of the AIDS Clinical Trials Group (ACTG) randomized 1571 treatment-naive HIV-1-infected participants (47% women) from nine countries and four continents to efavirenz plus lamivudine/zidovudine, atazanavir plus didanosine-EC plus emtricitabine, or efavirenz plus emtricitabine/tenofovir disoproxil fumarate.^28^ In this study, the Data Safety Monitoring Board recommended the early discontinuation of the atazanavir plus didanosine plus emtricitabine arm after a median follow-up of 72 weeks, as it was found to have inferior efficacy when compared with efavirenz plus lamivudine/zidovudine and efavirenz plus emtricitabine/tenofovir disoproxil fumarate. A population pharmacokinetics analysis found slower atazanavir CL/*F* in women compared with men, and among participants from Haiti, India and the USA compared with participants from Brazil, Malawi, Peru, South Africa, Thailand and Zimbabwe.^29^ These associations suggested that pharmacogenetic variants might affect atazanavir exposure. The present study evaluated associations between *CYP3A5*, *ABCB1*, *SLCO1B1* and *NR1I2* polymorphisms and plasma pharmacokinetics and metabolism of unboosted atazanavir in participants enrolled in PEARLS.

## Methods

### Study design

We retrospectively evaluated associations between atazanavir pharmacokinetics and selected polymorphisms in PEARLS participants. The primary objective was to compare atazanavir pharmacokinetics and mono-oxidation metabolite-to-parent drug ratios in CYP3A5 expressers versus non-expressers. Secondary objectives included the pharmacokinetic and metabolite associations with *ABCB1*, *SLCO1B1* and *NR1I2* polymorphisms and the probability of developing virological failure according to CYP3A5 phenotype. Institutional review board and country-specific ethics committee approvals were obtained.

### Participants

HIV-infected, antiretroviral-naive participants enrolled in PEARLS who were randomized to once-daily atazanavir (400 mg by mouth once daily) plus didanosine-EC (250 or 400 mg by mouth once daily) plus emtricitabine (200 mg by mouth once daily) in Peru, South Africa and the USA and who provided informed consent for genetic testing were included. Only participants from these three countries were selected based on co-enrolment in either A5128 (US participants) or A5243 (non-US participants), two ongoing ACTG studies obtaining human biological samples for genetic analyses. Prior to study entry participants were at least 18 years of age, had CD4+ T cells <300/mm^3^ and <7 days of cumulative antiretroviral therapy.^28^ Women of reproductive potential were non-pregnant and agreed to the use of contraception if sexually active.

### Quantification of atazanavir and population pharmacokinetics analysis

A single atazanavir plasma sample was collected between 4 and 8 weeks after treatment initiation and stored at −80°C until analysis. Atazanavir was quantified by a validated HPLC method [with a lower limit of quantification (LLQ) of 20.6 ng/mL and a linear range of 20.6–20 000 ng/mL]. A population pharmacokinetics model using NON-linear Mixed Effects Modeling version 7 (NONMEM) was built to obtain individual subject estimates of predicted concentrations and atazanavir pharmacokinetics parameters, which included CL/*F* and concentration at 24 h (*C*_24_), as previously described.^29^

### Genotyping

Human genomic DNA was isolated from stored peripheral blood samples using a commercially available kit (QIAmp DNA Mini Kit; Qiagen, Valencia, CA, USA) and genotyped by PCR-pyrosequencing (PSQ 96MA; Qiagen). All genotyped samples were run in duplicate. Haplotypes were assigned using PHASE version 2.1.^30,31^ In *CYP3A5*, we genotyped *CYP3A5*3* (rs776746) and *CYP3A5*6* (rs10264272). The CYP3A5 expresser phenotype was assigned to individuals with at least one copy of the *CYP3A5*1* allele based on the *CYP3A5*3*/**6* haplotype. For heterozygosity, CYP3A5 phenotype was assigned after computational inference, as described above. In *ABCB1* we genotyped rs1128503, rs2032582 and rs1045642. *ABCB1* haplotypes were analysed according to the number of C-G-C copies (0, 1 or 2). We genotyped rs2306283 and rs4149056 in *SLCO1B1*, and rs2472677, rs763645 and rs1523130 in *NR1I2*.

### Metabolite analysis

Atazanavir mono-oxidation metabolites M1 and M2 were determined via a validated method using LC-MS/MS, as previously described.^13^ As authentic metabolite standards are not readily available, the apparent metabolite concentrations were estimates obtained from the standard curve of atazanavir. M1/atazanavir and M2/atazanavir ratios were calculated in all participants with a measurable atazanavir level from the same plasma sample as that used to perform the population pharmacokinetic analysis (single plasma sample collected between 4 and 8 weeks after treatment initiation).

### Statistical analysis

Pharmacogenetic association analysis was performed using atazanavir CL/*F* and *C*_24_ data. Statistical analysis for CYP3A5 phenotype (expresser versus non-expresser) and other predictors with two categories was conducted using unpaired *t*-tests. Genotypes with more than two categories were analysed using ANOVA with pairwise comparisons using the most frequent genotype (*1 for *CYP3A5*) as the reference group. If a significant difference was identified in unadjusted analyses, an overall *F*-test adjusting for gender and race was performed. To account for missing data, we performed a sensitivity-ranked analysis, which included the participants with an undetectable atazanavir level but measurable M1 and M2 (we assumed that these participants had recent dosing of atazanavir). In these participants we assigned the highest possible rank for CL/*F* and the lowest possible rank for *C*_24_ in addition to an atazanavir concentration of 10.3 ng/mL (midpoint between the LLQ and 0 ng/mL). Since the individual genotype analyses were considered to be hypothesis generating only, we did not adjust for multiple comparisons.^32,33^

A Kaplan–Meier time-to-event analysis was performed to estimate the cumulative probability of developing virological failure in CYP3A5 expressers versus non-expressers based on the median follow-up time for the atazanavir plus didanosine-EC plus emtricitabine arm. Virological failure was defined as two successive plasma HIV-1 RNA values >1000 copies/mL at week 16 or later.^28^

Data were log_10_ transformed for analysis and presented as geometric mean (95% CI) unless otherwise specified. Statistical analyses were conducted using SAS 9.3 (Cary, NC, USA) and GraphPad Prism version 6.00 for Windows. A *P* value <0.05 was considered significant.

## Results

### Study population

From all the participants randomized to the atazanavir arm in PEARLS (*n* = 526), a total of 84 individuals had genotyping performed. The main reason for not having available genotypes in all PEARLS participants was that the ACTG studies where DNA was collected for genotyping (A5128 and A5243) were not available during PEARLS enrolment and/or follow-up, except in Peru, South Africa and the USA. Demographic and genotypic characteristics are presented in Tables [1](#DKW005TB1) and [2](#DKW005TB2), respectively. Besides differences in racial, ethnic, gender and country distribution compared with the parent study, no statistically significant differences in age, HIV stage, viral load upon randomization and/or rate of virological failure were identified among those PEARLS participants who were enrolled in this study versus those who were not (*P* > 0.20, [Table S1, available as Supplementary data at JAC Online](http://jac.oxfordjournals.org/lookup/suppl/doi:10.1093/jac/dkw005/-/DC1)). Clinical and safety data of the parent trial have been presented elsewhere.^28^ Two of the 84 participants were excluded because they received concomitant ritonavir, leaving a total of 82 evaluable participants taking unboosted atazanavir. For pharmacokinetic analysis, 13 of 82 participants were excluded because atazanavir CL/*F* and *C*_24_ could not be calculated (7 with undetectable atazanavir level, 4 with atazanavir sampling ≤1 or ≥29 h post-dose and 2 with atazanavir level >6000 ng/mL). For the metabolite analysis, we excluded 7 of 82 participants with undetectable atazanavir levels since the M1/atazanavir and M2/atazanavir ratios could not be calculated. Overall, we analysed CL/*F* and *C*_24_ data for 69 participants and metabolite data for 75 participants.

### Table 1.

| Characteristic | n (%) or median (range) |
| --- | --- |
| Gender |  |
| male | 56 (67) |
| female | 28 (33) |
| Age, years | 34 (19–59) |
| Race |  |
| Black African or African American | 44 (52) |
| Non-Black African or African American | 40 (48) |
| White | 27 (32) |
| Mestizo | 6 (7) |
| >1 race/unknown | 4 (5) |
| American Indian/Hawaiian/Pacific Islander | 3 (4) |
| Country |  |
| USA | 54 (64) |
| South Africa | 24 (29) |
| Peru | 6 (7) |
| Baseline HIV viral load |  |
| ≥100 000 copies/mL | 44 (52) |
| <100 000 copies/mL | 40 (48) |
| Baseline CD4, cells/mm3 | 202 (6–298) |

Table 1 Caption: Study population demographic characteristics (n = 84)

### Table 2.

|   |   | Gender |  | Region |  |  | Race/Ethnicity |  |  |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
|   | Age, median (range) | male (n = 56) | female (n = 28) | USA (n = 54) | South Africa (n = 24) | Peru (n = 6) | Black Af/AA (n = 44) | White (n = 27) | Mestizo (n = 6) | Am Ind/Haw/PI (n = 3) | >1 race or unknown (n = 4) |
| CYP3A5 phenotype |  |  |  |  |  |  |  |  |  |  |  |
| expresser (n = 56) | 34 (19–55) | 35 | 21 | 32 | 23 | 1 | 40 | 11 | 1 | 1 | 3 |
| non-expresser (n = 28) | 34 (22–59) | 21 | 7 | 22 | 1 | 5 | 4 | 16 | 5 | 2 | 1 |
| CYP3A5 genotype |  |  |  |  |  |  |  |  |  |  |  |
| *1/*1 (n = 17) | 36 (22–55) | 7 | 10 | 8 | 9 | 0 | 14 | 2 | 0 | 1 | 0 |
| *1/*3 (n = 25) | 31 (23–48) | 17 | 8 | 20 | 4 | 1 | 12 | 9 | 1 | 0 | 3 |
| *1/*6 (n = 15) | 39 (19–51) | 12 | 3 | 5 | 10 | 0 | 14 | 1 | 0 | 0 | 0 |
| *3/*3 (n = 24) | 33 (22–59) | 18 | 6 | 19 | 0 | 5 | 2 | 14 | 5 | 2 | 1 |
| *6/*6 (n = 1) | 36 | 0 | 1 | 0 | 1 | 0 | 1 | 0 | 0 | 0 | 0 |
| *3/*6 (n = 2) | 34 (34–35) | 2 | 0 | 2 | 0 | 0 | 1 | 1 | 0 | 0 | 0 |
| ABCB1 (C-G-C copies) |  |  |  |  |  |  |  |  |  |  |  |
| 0 (n = 17) | 33 (22–59) | 13 | 4 | 14 | 0 | 3 | 1 | 11 | 3 | 0 | 2 |
| 1 (n = 28) | 31 (19–55) | 20 | 8 | 21 | 5 | 2 | 10 | 11 | 2 | 3 | 2 |
| 2 (n = 39) | 35 (19–51) | 23 | 16 | 19 | 19 | 1 | 33 | 5 | 1 | 0 | 0 |
| SLCO1B1 c.388 |  |  |  |  |  |  |  |  |  |  |  |
| AA (n = 17) | 33 (19–59) | 13 | 4 | 14 | 0 | 3 | 2 | 10 | 3 | 0 | 2 |
| AG (n = 18) | 32 (23–55) | 14 | 4 | 13 | 3 | 2 | 6 | 8 | 2 | 1 | 1 |
| GG (n = 49) | 35 (19–51) | 29 | 20 | 27 | 21 | 1 | 36 | 9 | 1 | 2 | 1 |
| SLCO1B1 c.521 |  |  |  |  |  |  |  |  |  |  |  |
| TT (n = 70) | 34 (19–59) | 46 | 24 | 43 | 23 | 4 | 41 | 18 | 4 | 3 | 4 |
| TC (n = 14) | 36 (19–55) | 10 | 4 | 11 | 1 | 2 | 3 | 9 | 2 | 0 | 0 |
| CC (n = 0) |   | 0 | 0 | 0 | 0 | 0 | 0 | 0 | 0 | 0 | 0 |
| NR1I2 g.44477 |  |  |  |  |  |  |  |  |  |  |  |
| TT (n = 37) | 34 (19–55) | 20 | 17 | 21 | 15 | 1 | 31 | 2 | 1 | 1 | 2 |
| TC (n = 34) | 34 (22–59) | 25 | 9 | 22 | 9 | 3 | 13 | 17 | 3 | 0 | 1 |
| CC (n = 13) | 39 (24–47) | 11 | 2 | 11 | 0 | 2 | 0 | 8 | 2 | 2 | 1 |
| NR1I2 g.63396 |  |  |  |  |  |  |  |  |  |  |  |
| CC (n = 33) | 34 (19–59) | 22 | 11 | 19 | 10 | 4 | 17 | 11 | 4 | 0 | 1 |
| CT (n = 34) | 37 (19–48) | 22 | 12 | 20 | 12 | 2 | 22 | 8 | 2 | 1 | 1 |
| TT (n = 17) | 32 (22–55) | 12 | 5 | 15 | 2 | 0 | 5 | 8 | 0 | 2 | 2 |
| NR1I2 g.69789 |  |  |  |  |  |  |  |  |  |  |  |
| AA (n = 54) | 34 (19–55) | 33 | 21 | 32 | 22 | 0 | 41 | 9 | 0 | 2 | 2 |
| AG (n = 14) | 34 (22–59) | 12 | 2 | 11 | 2 | 1 | 3 | 8 | 1 | 1 | 1 |
| GG (n = 16) | 32 (24–47) | 11 | 5 | 11 | 0 | 5 | 0 | 10 | 5 | 0 | 1 |

Table 2 Caption: Study population genotype characteristics (n = 84)

### Primary endpoint

Atazanavir population pharmacokinetics parameters and the metabolite-to-parent drug ratios according to CYP3A5 expresser status are shown in Table [3](#DKW005TB3). In an unadjusted analysis, atazanavir CL/*F* (*P* = 0.61) and *C*_24_ (*P* = 0.14) were not different in CYP3A5 expressers versus non-expressers, which remained unchanged after adjusting for race and gender. In contrast, M1/atazanavir and M2/atazanavir ratios were significantly higher in CYP3A5 expressers versus non-expressers: 0.0083 (0.0074–0.0094) versus 0.0063 (0.0053–0.0075), respectively, *P* = 0.008, for M1/atazanavir; 0.0065 (0.0057–0.0073) versus 0.0050 (0.0042–0.0061), respectively, *P* = 0.02, for M2/atazanavir (Figure [1](#DKW005F1), Table [3](#DKW005TB3)). After adjusting for race and gender, only the M2/atazanavir ratio remained significant (*P* = 0.04). A sub-analysis of CYP3A5 expresser status according to race (i.e. separate analysis of Black Africans/African Americans and non-Black Africans/African Americans, [Table S2](http://jac.oxfordjournals.org/lookup/suppl/doi:10.1093/jac/dkw005/-/DC1)) demonstrated that, in non-Black Africans/African Americans, the M1/atazanavir and M2/atazanavir ratios remained significantly higher in expressers (*n* = 12) versus non-expressers (*n* = 22): 0.0089 (0.0084–0.0095) versus 0.0060 (0.0049–0.0074), respectively, *P* = 0.0069, for M1/atazanavir; 0.0065 (0.0056–0.0075) versus 0.0047 (0.0038–0.0058), respectively, *P* = 0.03 for M2/atazanavir. Given sample size limitations in the non-expresser group, we could not make a conclusive inference in Black Africans/African Americans. In the ranked analysis (*n* = 73), which included four participants with undetectable atazanavir levels but measurable metabolites, only M1/atazanavir differed significantly in expressers versus non-expressers: 0.0083 (0.0074–0.0094) versus 0.0065 (0.0053–0.0080), respectively, *P* = 0.0027.

### Table 3.

|   | n = 69 (45 expressers; 24 non-expressers) |  |  |  | n = 75 (49 expressers; 26 non-expressers) |  |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- |
| CYP3A5 Phenotype | CL/F (L/h) | P | C24 (ng/mL) | P | M1/ATV | P | M2/ATV | P |
| Expressor | 13.2 (12.1–14.4) | 0.61 | 75.3 (46.1–123.0) | 0.14 | 0.0083 (0.0074–0.0094) | 0.008 | 0.0065 (0.0057–0.0073) | 0.02 |
| Non-expresser | 12.7 (11.7–13.9) |   | 130.9 (86.9–197.2) |   | 0.0063 (0.0053–0.0075) |   | 0.0050 (0.0042–0.0061) |   |

Table 3 Caption: Atazanavir (ATV) population pharmacokinetic parameters and metabolite ratios according to CYP3A5 phenotype

### Figure 1.

![Figure 1.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/57cc/4867099/2417ef9714dd/dkw00501.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4867099_dkw00501.jpg)

Ratios of mono-oxidation metabolites 1 and 2 (M1 and M2) to atazanavir (ATV) concentration in 75 CYP3A5 expressers (n = 49) and non-expressers (n = 26). (a) Log10 M1/atazanavir ratios in expressers versus non-expressers. (b) Log10 M2/atazanavir ratios in expressers versus non-expressers.

### CYP3A5 genotype

Population pharmacokinetics parameters and M1/atazanavir and M2/atazanavir ratios were analysed according to individual *CYP3A5* genotype (Table [4](#DKW005TB4)). In the unadjusted analysis, CL/*F* differed by *CYP3A5*3* genotype (*P* = 0.001, Figure [2](#DKW005F2)), with faster clearance with the *CYP3A5*1/*1* genotype versus *CYP3A5*1/*3* and *CYP3A5*3/*3* genotypes: 14.8 (13.9–15.8) versus 11.4 L/h (9.9–13.2), *P* = 0.0006, and 12.5 L/h (11.4–13.8), *P* = 0.02, respectively. This difference remained significant after adjusting for race and gender (*P* = 0.006), although only for *CYP3A5*1/*1* compared with *CYP3A5*1/*3* (*P* = 0.001). *C*_24_ also differed by *CYP3A5*3* genotype (*P* = 0.02, Figure [2](#DKW005F2)), with lower atazanavir levels with *CYP3A5*1/*1* genotype versus *CYP3A5*1/*3* and *CYP3A5*3/*3* genotypes [49.8 (28.4–87.2) versus 134.3 ng/mL (67.2–268.3), *P* = 0.01, and 135.4 ng/mL (84.9–215.7), *P* = 0.01, respectively], although this was not significant after adjusting for race and gender (*P* = 0.15). In the ranked analysis, CL/*F* was significantly faster in the *CYP3A5*1/*1* genotype versus *CYP3A5*1/*3* and *CYP3A5*3/*3* genotypes (*P* = 0.04). For the metabolite analysis, there was a significant overall difference in M1/atazanavir by *CYP3A5* genotype (*P* = 0.02, Figure [2](#DKW005F2)), with higher M1/atazanavir in *CYP3A5*1/*1* versus *CYP3A5*3/*3* genotype [0.0088 (0.0074–0.0105) versus 0.0063 (0.0052–0.0076), respectively, *P* = 0.005]. This was also the case for M2/atazanavir (*P* = 0.01, Figure [2](#DKW005F2)), which was higher with the *CYP3A5*1/*1* genotype versus *CYP3A5*3/*3* [0.0070 (0.0059–0.0083) versus 0.0049 (0.0040–0.0061), respectively, *P* = 0.004]. None of these differences was statistically significant after adjusting for race and gender or by ranked analysis.

### Table 4.

|   | n = 69 (45 expressers; 24 non-expressers) |  |  |  | n = 75 (49 expressers; 26 non-expressers) |  |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- |
|   | CL/F (L/h) | Pa | C24 (ng/mL) | Pa | M1/ATV | Pa | M2/ATV | Pa |
| CYP3A5*3 |   | 0.001 |   | 0.02 |   | 0.02 |   | 0.01 |
| *1/*1 (reference) | 14.8 (13.9–15.8) | (reference) | 49.8 (28.4–87.2) | (reference) | 0.0088 (0.0074–0.0105) | (reference) | 0.0070 (0.0059–0.0083) | (reference) |
| *1/*3 | 11.4 (9.9–13.2) | 0.0006 | 134.3 (67.2–268.3) | 0.01 | 0.0074 (0.0066–0.0084) | 0.15 | 0.0058 (0.0051–0.0065) | 0.11 |
| *3/*3 | 12.5 (11.4–13.8) | 0.02 | 135.4 (84.9–215.7) | 0.01 | 0.0063 (0.0052–0.0076) | 0.005 | 0.0049 (0.0040–0.0061) | 0.004 |
| CYP3A5*6 |   | 0.004 |   | 0.04 |   | 0.03 |   | 0.06 |
| *1/*1 (reference) | 12.4 (11.6–13.4) |   | 109.3 (73.5–161.8) |   | 0.0071 (0.0063–0.0080) |   | 0.0056 (0.0050–0.0063) |   |
| *1/*6 | 15.7 (14.6–16.8) |   | 42.9 (20.4–90.4) |   | 0.0093 (0.0078–0.0112) |   | 0.0071 (0.0060–0.0085) |   |
| ABCB1 (C-G-C copies) |   | 0.19 |   | 0.009b |   | 0.15 |   | 0.08 |
| 0 (reference) | 13.4 (11.8–15.2) |   | 115.8 (57.5–233.4) | (reference) | 0.0062 (0.0049–0.0078) |   | 0.0046 (0.0037–0.0058) |   |
| 1 | 12.1 (10.8–13.4) |   | 164.1 (102.3–263.1) | 0.08 | 0.0083 (0.0069–0.0099) |   | 0.0060 (0.0051–0.0072) |   |
| 2 | 13.7 (12.5–15.0) |   | 51.5 (29.4–90.1) | 0.46 | 0.0076 (0.0066–0.0088) |   | 0.0064 (0.0055–0.0074) |   |
| SLCO1B1 c.388 |   | 0.88 |   | 0.42 |   | 0.46 |   | 0.38 |
| AA (reference) | 12.6 (11.3–14.1) |   | 119.4 (80.7–176.7) |   | 0.0067 (0.0055–0.0081) |   | 0.0054 (0.0046–0.0064) |   |
| AG | 13.1 (11.6–14.9) |   | 119.1 (60.7–234.0) |   | 0.0076 (0.0061–0.0096) |   | 0.0055 (0.0044–0.0068) |   |
| GG | 13.1 (12.0–14.4) |   | 74.3 (43.1–128.0) |   | 0.0079 (0.0069–0.0090) |   | 0.0063 (0.0055–0.0073) |   |
| SLCO1B1 c.521 |   | 0.57 |   | 0.12 |   | 0.10 |   | 0.26 |
| TT (reference) | 13.2 (12.2–14.1) |   | 81.7 (54.7–121.8) |   | 0.0073 (0.0066–0.0081) |   | 0.0058 (0.0052–0.0065) |   |
| TC | 12.5 (10.8–14.4) |   | 176.2 (107.2–289.5) |   | 0.0092 (0.0069–0.0122) |   | 0.0068 (0.0053–0.0087) |   |
| NR1I2 g.44477 |   | 0.72 |   | 0.31 |   | 0.74 |   | 0.65 |
| TT (reference) | 13.4 (12.3–14.5) |   | 69.0 (37.1–128.2) |   | 0.0077 (0.0064–0.0091) |   | 0.0061 (0.0052–0.0071) |   |
| TC | 12.9 (11.7–14.2) |   | 100.6 (65.7–154.1) |   | 0.0077 (0.0067–0.0089) |   | 0.0060 (0.0051–0.0071) |   |
| CC | 12.5 (10.0–15.6) |   | 147.8 (58.2–375.7) |   | 0.0069 (0.0052–0.0091) |   | 0.0053 (0.0038–0.0074) |   |
| NR1I2 g.63396 |   | 0.17 |   | 0.008c |   | 0.96 |   | 0.89 |
| CC (reference) | 12.3 (10.9–13.9) |   | 126.8 (81.1–198.2) | (reference) | 0.0071 (0.0060–0.0085) |   | 0.0058 (0.0049–0.0070) |   |
| CT | 14.0 (13.0–15.1) |   | 48.4 (25.3–92.6) | 0.01 | 0.0081 (0.0067–0.0097) |   | 0.0064 (0.0053–0.0077) |   |
| TT | 12.5 (11.0–14.4) |   | 172.6 (106.2–280.3) | 0.50 | 0.0077 (0.0066–0.0090) |   | 0.0057 (0.0048–0.0068) |   |
| NR1I2 g.69789 |   | 0.82 |   | 0.73 |   | 0.48 |   | 0.67 |
| AA (reference) | 13.1 (12.1–14.0) |   | 81.7 (49.4–135.1) |   | 0.0082 (0.0081–0.0082) |   | 0.0063 (0.0047–0.0084) |   |
| AG | 13.4 (11.6–15.5) |   | 106.5 (64.9–175.0) |   | 0.0070 (0.0051–0.0096) |   | 0.0053 (0.0040–0.0069) |   |
| GG | 12.6 (10.0–15.8) |   | 112.8 (45.2–281.0) |   | 0.0077 (0.0070–0.0085) |   | 0.0061 (0.0055–0.0069) |   |

Table 4 Caption: Atazanavir population pharmacokinetic parameters and metabolite ratios according to individual genotype

### Figure 2.

![Figure 2.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/57cc/4867099/ff01b9ba4c2d/dkw00502.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4867099_dkw00502.jpg)

Pharmacokinetic parameters and ratios of mono-oxidation metabolites 1 and 2 (M1 and M2) to atazanavir (ATV) concentration according to individual CYP3A5*3 genotype. (a and b) Log10 C24 and log10 CL/F with CYP3A5*1/*1 (n = 27) versus CYP3A5*1/*3 (n = 21) versus CYPA3A5*3/*3 (n = 21), respectively. (c and d) Log10 M1/atazanavir and log10 M2/atazanavir with CYP3A5*1/*1 (n = 30) versus CYP3A5*1/*3 (n = 23) versus CYPA3A5*3/*3 (n = 22), respectively. Overall ANOVA: log10 C24 P = 0.02, log10 CL/F P = 0.001, log10 M1/atazanavir P = 0.01 and log10 M2/atazanavir P = 0.01.

Only one participant (a Black individual from South Africa) was homozygous for the *CYP3A5*6/*6* genotype. Thus, this participant was excluded from any further analyses and the genotypes compared were *CYP3A5*1/*1* versus *CYP3A5*1/*6*. In the unadjusted analysis, CL/*F* was slower with the *CYP3A5*1/*1* genotype versus the *CYP3A5*1/*6* genotype [12.4 (11.6–13.4) versus 15.7 L/h (14.6–16.8), respectively, *P* = 0.004], which remained significant after adjusting for race and gender (*P* = 0.01) and by ranked analysis (*P* = 0.009). Participants with the *CYP3A5*1/*1* genotype had higher *C*_24_ levels versus the *CYP3A5*1/*6* genotype [109.3 (73.5–161.8) versus 42.9 ng/mL (20.4–90.4), respectively, *P* = 0.04], although this lost significance by ranked analysis and after adjusting for race and gender. M1/atazanavir was lower in the *CYP3A5*1/*1* genotype versus the *CYP3A5*1/*6* genotype [0.0071 (0.0063–0.0080) versus 0.0093 (0.0078–0.0112), respectively, *P* = 0.03], but this was not significant in the adjusted or ranked analyses. There were no differences in M2/atazanavir according to *CYP3A5*6* genotype.

### ABCB1, SLCO1B1 and NR1I2

For *ABCB1*, *C*_24_ differed by number of C-G-C haplotype copies (*P* = 0.009, Table [4](#DKW005TB4)), with lower atazanavir *C*_24_ with 2 versus 1 copies [51.5 (29.4–90.1) versus 164.1 ng/mL (102.3–263.1), respectively, *P* = 0.003], but this difference was not significant after adjusting for race and gender (*P* = 0.06) or by ranked analysis (*P* = 0.22). No other differences in pharmacokinetic parameters of metabolite ratios were observed for *ABCB1*. For *SLCO1B1*, CL/*F*, *C*_24_ and M1/atazanavir and M2/atazanavir ratios did not differ by *SLCO1B1* c.388 or *SLCO1B1* c.521 genotypes (Table [4](#DKW005TB4)). In an unadjusted analysis, *C*_24_ was lower in participants heterozygous for *NR1I2* g.63396C → T (*P* = 0.008) versus those who were homozygous for the major or minor allele [48.4 (25.3–92.6) versus 126.8 ng/mL (81.1–198.2), *P* = 0.01, and 172.6 ng/mL (106.2–280.3), *P* = 0.007 (Table [4](#DKW005TB4)), respectively], which remained significant after adjusting for race and gender (*P* = 0.03) and by ranked analysis (*P* = 0.003). We found no difference in CL/*F*, *C*_24_, M1/atazanavir or M2/atazanavir by *NR1I2* g.44477T → C or *NR1I2* g.69789A → G genotypes (Table [4](#DKW005TB4)).

### Virological failure according to CYP3A5 phenotype

Among the 69 participants included in the pharmacokinetic analysis, 11 developed virological failure after randomization, including 7 out of 24 CYP3A5 non-expressers and 4 out of 45 CYP3A5 expressers. After 72 weeks (median study follow-up), the proportion of participants without virological failure was significantly higher in CYP3A5 expressers versus non-expressers [0.95 (95% CI 0.89–1) versus 0.73 (95% CI 0.57–0.94), *P* = 0.026] (Figure [3](#DKW005F3)).

### Figure 3.

![Figure 3.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/57cc/4867099/84ac2bb56fbd/dkw00503.jpg)

Kaplan–Meier estimates of time to development of virological failure (VF) in CYP3A5 expressers (n = 45) versus non-expressers (n = 24). The cumulative probability of virological failure is shown on the vertical axis for the two study groups. At week 72 (median study follow-up), the proportion of CYP3A5 expressers without VF was 0.95 (95% CI 0.89–1) versus 0.73 (95% CI 0.57–0.94) in non-expressers (P = 0.026). Racial distribution in expressers with VF (n = 4) included three Black African/African American participants (two from USA and one from South Africa) and one White participant. Racial distribution in non-expressers with VF (n = 7) included five White, one Peruvian and one African American participant from the USA.

## Discussion

This study identified a distinctive metabolic profile for unboosted atazanavir in a racially diverse cohort of HIV-infected CYP3A5 expressers versus non-expressers, which is consistent with previous findings in healthy volunteers and human liver microsomes.^13,34^ However, in contrast to previous observations,^12,20,35^ we found no association between CYP3A5 expresser status and pharmacokinetic parameters in the parent drug. This finding is unexpected since the increased metabolite ratios in CYP3A5 expressers suggest faster atazanavir metabolism. Despite this, individual genotype analysis for *CYP3A5*3* and *CYP3A5*6* identified significant pharmacokinetic differences in unboosted atazanavir. For example, when *CYP3A5*1/*3* was individually evaluated, CL/*F* and *C*_24_ revealed decreased atazanavir exposure compared with the C*YP3A5*1/*1* genotype. This suggests that, at least in this study cohort, the *CYP3A5*1/*3* genotype may more appropriately fit within the *CYP3A5*3/*3* category and the non-expresser phenotype. Although this discordance is opposite to what has been previously reported for atazanavir and other medications, including other PIs,^4,8,10,12,36^ an individual *CYP3A5* genotype effect has been previously observed with other drugs.^37–39^

Among the potential reasons for the *CYP3A5* pharmacokinetic differences between our study and previous reports could be that we derived pharmacokinetic data from a non-linear mixed effects model based on a single atazanavir measurement,^29^ in addition to the small sample size in our cohort. Similarly, there could be variability in drug adherence in this HIV-infected population, which is suggested by seven individuals with undetectable atazanavir levels. Although the adherence analysis in the parent study recently demonstrated low rates of non-adherence in participants randomized to a once daily compared with a twice daily regimen,^40^ adherence was assessed through a standardized interviewer-administered questionnaire and pill counts, both of which overestimate adherence.^41,42^ Concomitantly, the diverse ethnic distribution of the parent study from which this cohort is derived (i.e. a randomized clinical trial in resource-limited settings) resulted in a variable distribution of the CYP3A5 phenotype and other genotypes analysed, which could have limited our ability to identify genetic associations. Regardless of these limitations, our study adds novel and unique knowledge to the pharmacogenetics and metabolism of unboosted atazanavir in racially diverse HIV-infected individuals.

Two previously unreported findings in our study are worthy of individual discussion. First, participants with two copies of the *ABCB1* C-G-C haplotype had lower atazanavir *C*_24_ than individuals with one copy. Although this disagrees with previous reports in which atazanavir exposure was lower with 0 versus 1 and 2 C-G-C copies (even after controlling for CYP3A5 phenotype)^12^ and in individuals with the *ABCB1* 2677G → T/A genotype,^43^ it is not unexpected given the conflicting results involving the *ABCB1* pharmacogenetics of PIs.^11,15,16^ Second, *NR1I2* 63396 C/T heterozygosity was associated with lower atazanavir *C*_24_ than either homozygous genotype. This is inconsistent with previous observations for unboosted atazanavir, in which 63396 T/T homozygosity was associated with low drug exposure.^21,22^ To evaluate this further, we analysed the effect of the *NR1I2* 63396C → T genotype according to CYP3A5 phenotype and found that atazanavir *C*_24_ remained low in the CYP3A5 expressers (*P* = 0.02), but not in non-expressers (*P* = 0.11). The predominance of CYP3A5 expressers in our cohort may explain this finding, as previous observations of an association with *NR1I2* 63396C → T have mainly involved Caucasians, where 50–90% are CYP3A5 non-expressers.^21,22^ The discrepancies observed in our cohort on the pharmacogenetics of unboosted atazanavir in *ABCB1* and *NR1I2* merit additional study.

Although various studies have evaluated the relationship of antiretroviral pharmacogenetics with HIV outcomes, none has focused on unboosted atazanavir.^44–48^ In this study, virological failure was more frequent in phenotypic CYP3A5 non-expressers versus expressers. This contrasts with a recent report in which *CYP3A5*3* was not associated with lopinavir/ritonavir monotherapy failure,^49^ and is counterintuitive, since our M1 and M2 metabolite results suggest increased atazanavir metabolism in CYP3A5 expressers, which should lead to lower atazanavir exposure. A potential explanation for this finding is that reduced atazanavir metabolism in non-expressers could have led to higher drug exposure and side effects, resulting in sub-optimal adherence and treatment failure. On the other hand, this is unlikely to be due to an additional antiviral effect provided by the M1 and M2 metabolites in CYP3A5 expressers, as none of these atazanavir metabolites has demonstrated any *in vitro* antiviral activity.^1^

Among the main strengths of our study are the diversity of the studied population, the ability to study the pharmacokinetic and pharmacogenetic associations of atazanavir without the potential confounding effects of ritonavir and the multiple SNPs that were analysed. In addition, this study also demonstrates the feasibility of pharmacogenetic research in diverse settings. Our limitations include the small sample size, the sparse pharmacokinetic sampling and the restrictions inherent in subjective adherence monitoring. However, the evaluation of unboosted atazanavir pharmacogenetics in the context of a large randomized, multinational clinical trial in resource-limited settings makes this study unique in the field, as future studies of atazanavir as a single PI are unlikely to be repeated.

## Funding

This work was supported by Award Number U01AI068636 from the National Institute of Allergy and Infectious Diseases and supported by National Institute of Mental Health (NIMH), National Institute of Dental and Craniofacial Research (NIDCR). This work was also supported by NIH/NCRR Colorado CTSI (CCTSI) Grant Numbers UL1 RR025780 and UL1 TR001082; NIH/NIAID ACTG Minority Investigator Award AI0 68636; and NIH/NIAID K23 AI104315. The research utilized services of the Medicinal Chemistry Core Facility (MCCF) housed within the Department of Pharmaceutical Sciences (DOPS). The MCCF receives funding via CCTSI. Colorado Multiple Institutional Review Board COMIRB# 11-0106.

## Transparency declarations

All authors have none to declare.

## Supplementary data

[Tables S1 and S2 are available as Supplementary data at JAC Online (http://jac.oxfordjournals.org/)](http://jac.oxfordjournals.org/lookup/suppl/doi:10.1093/jac/dkw005/-/DC1).

## Disclaimer

The content is solely the responsibility of the authors and does not necessarily represent the official views of the National Institute of Allergy and Infectious Diseases or the National Institutes of Health.

## Supplementary Material

## Acknowledgements

We thank the AIDS CLINICAL TRIALS GROUP and the PEARLS (A5175) study team for their assistance with this manuscript. Study drugs were donated by Gilead Sciences, Bristol Myers Squibb, GlaxoSmithKline, ViiV Healthcare and Boehringer Ingelheim.

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Reyataz Prescribing Information. http://packageinserts.bms.com/pi/pi_reyataz.pdf.  [http://packageinserts.bms.com/pi/pi_reyataz.pdf](http://packageinserts.bms.com/pi/pi_reyataz.pdf)

2. Panel on Antiretroviral Guidelines for Adults and Adolescents. Guidelines for the Use of Antiretroviral Agents in HIV-1 Infected Adults and Adolescents. Department of Health and Human Services; https://aidsinfo.nih.gov/contentfiles/lvguidelines/adultandadolescentgl.pdf.  [https://aidsinfo.nih.gov/contentfiles/lvguidelines/adultandadolescentgl.pdf](https://aidsinfo.nih.gov/contentfiles/lvguidelines/adultandadolescentgl.pdf) | [Google Scholar](https://scholar.google.com/scholar_lookup?title=Guidelines%20for%20the%20Use%20of%20Antiretroviral%20Agents%20in%20HIV-1%20Infected%20Adults%20and%20Adolescents&)

3. Thompson MA, Aberg JA, Hoy JF et al. Antiretroviral treatment of adult HIV infection: 2012 recommendations of the International Antiviral Society-USA panel. JAMA 2012; 308: 387–402.  [DOI](https://doi.org/10.1001/jama.2012.7961) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22820792/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA&title=Antiretroviral%20treatment%20of%20adult%20HIV%20infection:%202012%20recommendations%20of%20the%20International%20Antiviral%20Society-USA%20panel&author=MA%20Thompson&author=JA%20Aberg&author=JF%20Hoy&volume=308&publication_year=2012&pages=387-402&pmid=22820792&doi=10.1001/jama.2012.7961&)

4. Wojnowski L, Kamdem LK. Clinical implications of CYP3A polymorphisms. Expert Opin Drug Metab Toxicol 2006; 2: 171–82.  [DOI](https://doi.org/10.1517/17425255.2.2.171) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16866606/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Expert%20Opin%20Drug%20Metab%20Toxicol&title=Clinical%20implications%20of%20CYP3A%20polymorphisms&author=L%20Wojnowski&author=LK%20Kamdem&volume=2&publication_year=2006&pages=171-82&pmid=16866606&doi=10.1517/17425255.2.2.171&)

5. Kuehl P, Zhang J, Lin Y et al. Sequence diversity in CYP3A promoters and characterization of the genetic basis of polymorphic CYP3A5 expression. Nat Genet 2001; 27: 383–91.  [DOI](https://doi.org/10.1038/86882) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11279519/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Sequence%20diversity%20in%20CYP3A%20promoters%20and%20characterization%20of%20the%20genetic%20basis%20of%20polymorphic%20CYP3A5%20expression&author=P%20Kuehl&author=J%20Zhang&author=Y%20Lin&volume=27&publication_year=2001&pages=383-91&pmid=11279519&doi=10.1038/86882&)

6. Thompson EE, Kuttab-Boulos H, Witonsky D et al. CYP3A variation and the evolution of salt-sensitivity variants. Am J Hum Genet 2004; 75: 1059–69.  [DOI](https://doi.org/10.1086/426406) | [PMC free article](/articles/PMC1182141/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15492926/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=CYP3A%20variation%20and%20the%20evolution%20of%20salt-sensitivity%20variants&author=EE%20Thompson&author=H%20Kuttab-Boulos&author=D%20Witonsky&volume=75&publication_year=2004&pages=1059-69&pmid=15492926&doi=10.1086/426406&)

7. Hustert E, Haberl M, Burk O et al. The genetic determinants of the CYP3A5 polymorphism. Pharmacogenetics 2001; 11: 773–9.  [DOI](https://doi.org/10.1097/00008571-200112000-00005) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11740341/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=The%20genetic%20determinants%20of%20the%20CYP3A5%20polymorphism&author=E%20Hustert&author=M%20Haberl&author=O%20Burk&volume=11&publication_year=2001&pages=773-9&pmid=11740341&doi=10.1097/00008571-200112000-00005&)

8. Anderson PL, Lamba J, Aquilante CL et al. Pharmacogenetic characteristics of indinavir, zidovudine, and lamivudine therapy in HIV-infected adults: a pilot study. J Acquir Immune Defic Syndr 2006; 42: 441–9.  [DOI](https://doi.org/10.1097/01.qai.0000225013.53568.69) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16791115/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Acquir%20Immune%20Defic%20Syndr&title=Pharmacogenetic%20characteristics%20of%20indinavir,%20zidovudine,%20and%20lamivudine%20therapy%20in%20HIV-infected%20adults:%20a%20pilot%20study&author=PL%20Anderson&author=J%20Lamba&author=CL%20Aquilante&volume=42&publication_year=2006&pages=441-9&pmid=16791115&doi=10.1097/01.qai.0000225013.53568.69&)

9. Frohlich M, Hoffmann MM, Burhenne J et al. Association of the CYP3A5 A6986G (CYP3A5*3) polymorphism with saquinavir pharmacokinetics. Br J Clin Pharmacol 2004; 58: 443–4.  [DOI](https://doi.org/10.1111/j.1365-2125.2004.02159.x) | [PMC free article](/articles/PMC1884607/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15373940/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Clin%20Pharmacol&title=Association%20of%20the%20CYP3A5%20A6986G%20(CYP3A5*3)%20polymorphism%20with%20saquinavir%20pharmacokinetics&author=M%20Frohlich&author=MM%20Hoffmann&author=J%20Burhenne&volume=58&publication_year=2004&pages=443-4&pmid=15373940&doi=10.1111/j.1365-2125.2004.02159.x&)

10. Josephson F, Allqvist A, Janabi M et al. CYP3A5 genotype has an impact on the metabolism of the HIV protease inhibitor saquinavir. Clin Pharmacol Ther 2007; 81: 708–12.  [DOI](https://doi.org/10.1038/sj.clpt.6100117) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17329995/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=CYP3A5%20genotype%20has%20an%20impact%20on%20the%20metabolism%20of%20the%20HIV%20protease%20inhibitor%20saquinavir&author=F%20Josephson&author=A%20Allqvist&author=M%20Janabi&volume=81&publication_year=2007&pages=708-12&pmid=17329995&doi=10.1038/sj.clpt.6100117&)

11. Solas C, Simon N, Drogoul MP et al. Minimal effect of MDR1 and CYP3A5 genetic polymorphisms on the pharmacokinetics of indinavir in HIV-infected patients. Br J Clin Pharmacol 2007; 64: 353–62.  [DOI](https://doi.org/10.1111/j.1365-2125.2007.02903.x) | [PMC free article](/articles/PMC2000655/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17517050/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Clin%20Pharmacol&title=Minimal%20effect%20of%20MDR1%20and%20CYP3A5%20genetic%20polymorphisms%20on%20the%20pharmacokinetics%20of%20indinavir%20in%20HIV-infected%20patients&author=C%20Solas&author=N%20Simon&author=MP%20Drogoul&volume=64&publication_year=2007&pages=353-62&pmid=17517050&doi=10.1111/j.1365-2125.2007.02903.x&)

12. Anderson PL, Aquilante CL, Gardner EM et al. Atazanavir pharmacokinetics in genetically determined CYP3A5 expressors versus non-expressors. J Antimicrob Chemother 2009; 64: 1071–9.  [DOI](https://doi.org/10.1093/jac/dkp317) | [PMC free article](/articles/PMC2760462/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19710077/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Antimicrob%20Chemother&title=Atazanavir%20pharmacokinetics%20in%20genetically%20determined%20CYP3A5%20expressors%20versus%20non-expressors&author=PL%20Anderson&author=CL%20Aquilante&author=EM%20Gardner&volume=64&publication_year=2009&pages=1071-9&pmid=19710077&doi=10.1093/jac/dkp317&)

13. Wempe MF, Anderson PL. Atazanavir metabolism according to CYP3A5 status: an in vitro-in vivo assessment. Drug Metab Dispos 2011; 39: 522–7.  [DOI](https://doi.org/10.1124/dmd.110.036178) | [PMC free article](/articles/PMC3061561/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21148251/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Atazanavir%20metabolism%20according%20to%20CYP3A5%20status:%20an%20in%20vitro-in%20vivo%20assessment&author=MF%20Wempe&author=PL%20Anderson&volume=39&publication_year=2011&pages=522-7&pmid=21148251&doi=10.1124/dmd.110.036178&)

14. Kimchi-Sarfaty C, Oh JM, Kim IW et al. A ‘silent’ polymorphism in the MDR1 gene changes substrate specificity. Science 2007; 315: 525–8.  [DOI](https://doi.org/10.1126/science.1135308) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17185560/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=A%20%E2%80%98silent%E2%80%99%20polymorphism%20in%20the%20MDR1%20gene%20changes%20substrate%20specificity&author=C%20Kimchi-Sarfaty&author=JM%20Oh&author=IW%20Kim&volume=315&publication_year=2007&pages=525-8&pmid=17185560&doi=10.1126/science.1135308&)

15. Perloff ES, Duan SX, Skolnik PR et al. Atazanavir: effects on P-glycoprotein transport and CYP3A metabolism in vitro. Drug Metab Dispos 2005; 33: 764–70.  [DOI](https://doi.org/10.1124/dmd.104.002931) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15764714/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Atazanavir:%20effects%20on%20P-glycoprotein%20transport%20and%20CYP3A%20metabolism%20in%20vitro&author=ES%20Perloff&author=SX%20Duan&author=PR%20Skolnik&volume=33&publication_year=2005&pages=764-70&pmid=15764714&doi=10.1124/dmd.104.002931&)

16. Lucia MB, Golotta C, Rutella S et al. Atazanavir inhibits P-glycoprotein and multidrug resistance-associated protein efflux activity. J Acquir Immune Defic Syndr 2005; 39: 635–7.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/16044020/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Acquir%20Immune%20Defic%20Syndr&title=Atazanavir%20inhibits%20P-glycoprotein%20and%20multidrug%20resistance-associated%20protein%20efflux%20activity&author=MB%20Lucia&author=C%20Golotta&author=S%20Rutella&volume=39&publication_year=2005&pages=635-7&pmid=16044020&)

17. Ma Q, Brazeau D, Zingman BS et al. Multidrug resistance 1 polymorphisms and trough concentrations of atazanavir and lopinavir in patients with HIV. Pharmacogenomics 2007; 8: 227–35.  [DOI](https://doi.org/10.2217/14622416.8.3.227) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17324111/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Multidrug%20resistance%201%20polymorphisms%20and%20trough%20concentrations%20of%20atazanavir%20and%20lopinavir%20in%20patients%20with%20HIV&author=Q%20Ma&author=D%20Brazeau&author=BS%20Zingman&volume=8&publication_year=2007&pages=227-35&pmid=17324111&doi=10.2217/14622416.8.3.227&)

18. Zhang X, Tierney C, Albrecht M et al. Discordant associations between SLCO1B1 521T-->C and plasma levels of ritonavir-boosted protease inhibitors in AIDS clinical trials group study A5146. Ther Drug Monit 2013; 35: 209–16.  [DOI](https://doi.org/10.1097/FTD.0b013e318280d0ad) | [PMC free article](/articles/PMC3603284/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23503447/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ther%20Drug%20Monit&title=Discordant%20associations%20between%20SLCO1B1%20521T-->C%20and%20plasma%20levels%20of%20ritonavir-boosted%20protease%20inhibitors%20in%20AIDS%20clinical%20trials%20group%20study%20A5146&author=X%20Zhang&author=C%20Tierney&author=M%20Albrecht&volume=35&publication_year=2013&pages=209-16&pmid=23503447&doi=10.1097/FTD.0b013e318280d0ad&)

19. Kohlrausch FB, de Cassia Estrela R, Barroso PF et al. The impact of SLCO1B1 polymorphisms on the plasma concentration of lopinavir and ritonavir in HIV-infected men. Br J Clin Pharmacol 2010; 69: 95–8.  [DOI](https://doi.org/10.1111/j.1365-2125.2009.03551.x) | [PMC free article](/articles/PMC2830602/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20078617/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Clin%20Pharmacol&title=The%20impact%20of%20SLCO1B1%20polymorphisms%20on%20the%20plasma%20concentration%20of%20lopinavir%20and%20ritonavir%20in%20HIV-infected%20men&author=FB%20Kohlrausch&author=R%20de%20Cassia%20Estrela&author=PF%20Barroso&volume=69&publication_year=2010&pages=95-8&pmid=20078617&doi=10.1111/j.1365-2125.2009.03551.x&)

20. Kile DA, MaWhinney S, Aquilante CL et al. A population pharmacokinetic-pharmacogenetic analysis of atazanavir. AIDS Res Hum Retroviruses 2012; 28: 1227–34.  [DOI](https://doi.org/10.1089/aid.2011.0378) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22394315/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=AIDS%20Res%20Hum%20Retroviruses&title=A%20population%20pharmacokinetic-pharmacogenetic%20analysis%20of%20atazanavir&author=DA%20Kile&author=S%20MaWhinney&author=CL%20Aquilante&volume=28&publication_year=2012&pages=1227-34&pmid=22394315&doi=10.1089/aid.2011.0378&)

21. Siccardi M, D'Avolio A, Baietto L et al. Association of a single-nucleotide polymorphism in the pregnane X receptor (PXR 63396C-->T) with reduced concentrations of unboosted atazanavir. Clin Infect Dis 2008; 47: 1222–5.  [DOI](https://doi.org/10.1086/592304) | [PMC free article](/articles/PMC3672984/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18831695/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Infect%20Dis&title=Association%20of%20a%20single-nucleotide%20polymorphism%20in%20the%20pregnane%20X%20receptor%20(PXR%2063396C-->T)%20with%20reduced%20concentrations%20of%20unboosted%20atazanavir&author=M%20Siccardi&author=A%20D'Avolio&author=L%20Baietto&volume=47&publication_year=2008&pages=1222-5&pmid=18831695&doi=10.1086/592304&)

22. Schipani A, Siccardi M, D'Avolio A et al. Population pharmacokinetic modeling of the association between 63396C->T pregnane X receptor polymorphism and unboosted atazanavir clearance. Antimicrob Agents Chemother 2010; 54: 5242–50.  [DOI](https://doi.org/10.1128/AAC.00781-10) | [PMC free article](/articles/PMC2981241/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20921307/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Antimicrob%20Agents%20Chemother&title=Population%20pharmacokinetic%20modeling%20of%20the%20association%20between%2063396C->T%20pregnane%20X%20receptor%20polymorphism%20and%20unboosted%20atazanavir%20clearance&author=A%20Schipani&author=M%20Siccardi&author=A%20D'Avolio&volume=54&publication_year=2010&pages=5242-50&pmid=20921307&doi=10.1128/AAC.00781-10&)

23. Squires K, Lazzarin A, Gatell JM et al. Comparison of once-daily atazanavir with efavirenz, each in combination with fixed-dose zidovudine and lamivudine, as initial therapy for patients infected with HIV. J Acquir Immune Defic Syndr 2004; 36: 1011–9.  [DOI](https://doi.org/10.1097/00126334-200408150-00003) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15247553/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Acquir%20Immune%20Defic%20Syndr&title=Comparison%20of%20once-daily%20atazanavir%20with%20efavirenz,%20each%20in%20combination%20with%20fixed-dose%20zidovudine%20and%20lamivudine,%20as%20initial%20therapy%20for%20patients%20infected%20with%20HIV&author=K%20Squires&author=A%20Lazzarin&author=JM%20Gatell&volume=36&publication_year=2004&pages=1011-9&pmid=15247553&doi=10.1097/00126334-200408150-00003&)

24. Malan DR, Krantz E, David N et al. Efficacy and safety of atazanavir, with or without ritonavir, as part of once-daily highly active antiretroviral therapy regimens in antiretroviral-naive patients. J Acquir Immune Defic Syndr 2008; 47: 161–7.  [DOI](https://doi.org/10.1097/QAI.0b013e31815ace6a) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17971713/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Acquir%20Immune%20Defic%20Syndr&title=Efficacy%20and%20safety%20of%20atazanavir,%20with%20or%20without%20ritonavir,%20as%20part%20of%20once-daily%20highly%20active%20antiretroviral%20therapy%20regimens%20in%20antiretroviral-naive%20patients&author=DR%20Malan&author=E%20Krantz&author=N%20David&volume=47&publication_year=2008&pages=161-7&pmid=17971713&doi=10.1097/QAI.0b013e31815ace6a&)

25. Sanne I, Piliero P, Squires K et al. Results of a phase 2 clinical trial at 48 weeks (AI424-007): a dose-ranging, safety, and efficacy comparative trial of atazanavir at three doses in combination with didanosine and stavudine in antiretroviral-naive subjects. J Acquir Immune Defic Syndr 2003; 32: 18–29.  [DOI](https://doi.org/10.1097/00126334-200301010-00004) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12514410/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Acquir%20Immune%20Defic%20Syndr&title=Results%20of%20a%20phase%202%20clinical%20trial%20at%2048%20weeks%20(AI424-007):%20a%20dose-ranging,%20safety,%20and%20efficacy%20comparative%20trial%20of%20atazanavir%20at%20three%20doses%20in%20combination%20with%20didanosine%20and%20stavudine%20in%20antiretroviral-naive%20subjects&author=I%20Sanne&author=P%20Piliero&author=K%20Squires&volume=32&publication_year=2003&pages=18-29&pmid=12514410&doi=10.1097/00126334-200301010-00004&)

26. Wood R, Phanuphak P, Cahn P et al. Long-term efficacy and safety of atazanavir with stavudine and lamivudine in patients previously treated with nelfinavir or atazanavir. J Acquir Immune Defic Syndr 2004; 36: 684–92.  [DOI](https://doi.org/10.1097/00126334-200406010-00005) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15167287/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Acquir%20Immune%20Defic%20Syndr&title=Long-term%20efficacy%20and%20safety%20of%20atazanavir%20with%20stavudine%20and%20lamivudine%20in%20patients%20previously%20treated%20with%20nelfinavir%20or%20atazanavir&author=R%20Wood&author=P%20Phanuphak&author=P%20Cahn&volume=36&publication_year=2004&pages=684-92&pmid=15167287&doi=10.1097/00126334-200406010-00005&)

27. Gatell J, Salmon-Ceron D, Lazzarin A et al. Efficacy and safety of atazanavir-based highly active antiretroviral therapy in patients with virologic suppression switched from a stable, boosted or unboosted protease inhibitor treatment regimen: the SWAN Study (AI424-097) 48-week results. Clin Infect Dis 2007; 44: 1484–92.  [DOI](https://doi.org/10.1086/517497) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17479947/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Infect%20Dis&title=Efficacy%20and%20safety%20of%20atazanavir-based%20highly%20active%20antiretroviral%20therapy%20in%20patients%20with%20virologic%20suppression%20switched%20from%20a%20stable,%20boosted%20or%20unboosted%20protease%20inhibitor%20treatment%20regimen:%20the%20SWAN%20Study%20(AI424-097)%2048-week%20results&author=J%20Gatell&author=D%20Salmon-Ceron&author=A%20Lazzarin&volume=44&publication_year=2007&pages=1484-92&pmid=17479947&doi=10.1086/517497&)

28. Campbell TB, Smeaton LM, Kumarasamy N et al. Efficacy and safety of three antiretroviral regimens for initial treatment of HIV-1: a randomized clinical trial in diverse multinational settings. PLoS Med 2012; 9: e1001290.  [DOI](https://doi.org/10.1371/journal.pmed.1001290) | [PMC free article](/articles/PMC3419182/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22936892/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Med&title=Efficacy%20and%20safety%20of%20three%20antiretroviral%20regimens%20for%20initial%20treatment%20of%20HIV-1:%20a%20randomized%20clinical%20trial%20in%20diverse%20multinational%20settings&author=TB%20Campbell&author=LM%20Smeaton&author=N%20Kumarasamy&volume=9&publication_year=2012&pages=e1001290&pmid=22936892&doi=10.1371/journal.pmed.1001290&)

29. Andrade A, Baheti G, Smeaton LM et al. Sex- and geographic-based differences in atazanavir pharmacokinetics in subjects treated with didanosine, emtricitabine, and atazanavir in the AIDS clinical trials group (ACTG) prospective evaluation of antiretrovirals in resource limited settings (PEARLS) study. In: Abstracts of the Eighteenth Conference on Retroviruses and Opportunistic Infections, 2011, Boston, Massachusetts. Abstract 648.  [Google Scholar](https://scholar.google.com/scholar_lookup?Andrade%20A,%20Baheti%20G,%20Smeaton%20LM%20et%20al.%20Sex-%20and%20geographic-based%20differences%20in%20atazanavir%20pharmacokinetics%20in%20subjects%20treated%20with%20didanosine,%20emtricitabine,%20and%20atazanavir%20in%20the%20AIDS%20clinical%20trials%20group%20(ACTG)%20prospective%20evaluation%20of%20antiretrovirals%20in%20resource%20limited%20settings%20(PEARLS)%20study.%20In:%20Abstracts%20of%20the%20Eighteenth%20Conference%20on%20Retroviruses%20and%20Opportunistic%20Infections,%202011,%20Boston,%20Massachusetts.%20Abstract%20648.)

30. Stephens M, Smith NJ, Donnelly P. A new statistical method for haplotype reconstruction from population data. Am J Hum Genet 2001; 68: 978–89.  [DOI](https://doi.org/10.1086/319501) | [PMC free article](/articles/PMC1275651/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11254454/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=A%20new%20statistical%20method%20for%20haplotype%20reconstruction%20from%20population%20data&author=M%20Stephens&author=NJ%20Smith&author=P%20Donnelly&volume=68&publication_year=2001&pages=978-89&pmid=11254454&doi=10.1086/319501&)

31. Stephens M, Donnelly P. A comparison of Bayesian methods for haplotype reconstruction from population genotype data. Am J Hum Genet 2003; 73: 1162–9.  [DOI](https://doi.org/10.1086/379378) | [PMC free article](/articles/PMC1180495/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14574645/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=A%20comparison%20of%20Bayesian%20methods%20for%20haplotype%20reconstruction%20from%20population%20genotype%20data&author=M%20Stephens&author=P%20Donnelly&volume=73&publication_year=2003&pages=1162-9&pmid=14574645&doi=10.1086/379378&)

32. Saville DJ. Basic statistics and the inconsistency of multiple comparison procedures. Can J Exp Psychol 2003; 57: 167–75.  [DOI](https://doi.org/10.1037/h0087423) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14596475/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Can%20J%20Exp%20Psychol&title=Basic%20statistics%20and%20the%20inconsistency%20of%20multiple%20comparison%20procedures&author=DJ%20Saville&volume=57&publication_year=2003&pages=167-75&pmid=14596475&doi=10.1037/h0087423&)

33. Rothman KJ. No adjustments are needed for multiple comparisons. Epidemiology 1990; 1: 43–6.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/2081237/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Epidemiology&title=No%20adjustments%20are%20needed%20for%20multiple%20comparisons&author=KJ%20Rothman&volume=1&publication_year=1990&pages=43-6&pmid=2081237&)

34. Tseng E, Walsky RL, Luzietti RA Jr et al. Relative contributions of cytochrome CYP3A4 versus CYP3A5 for CYP3A-cleared drugs assessed in vitro using a CYP3A4-selective inactivator (CYP3cide). Drug Metab Dispos 2014; 42: 1163–73.  [DOI](https://doi.org/10.1124/dmd.114.057000) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24737844/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Relative%20contributions%20of%20cytochrome%20CYP3A4%20versus%20CYP3A5%20for%20CYP3A-cleared%20drugs%20assessed%20in%20vitro%20using%20a%20CYP3A4-selective%20inactivator%20(CYP3cide)&author=E%20Tseng&author=RL%20Walsky&author=RA%20Luzietti&volume=42&publication_year=2014&pages=1163-73&pmid=24737844&doi=10.1124/dmd.114.057000&)

35. Savic RM, Barrail-Tran A, Duval X et al. Effect of adherence as measured by MEMS, ritonavir boosting, and CYP3A5 genotype on atazanavir pharmacokinetics in treatment-naive HIV-infected patients. Clin Pharmacol Ther 2012; 92: 575–83.  [DOI](https://doi.org/10.1038/clpt.2012.137) | [PMC free article](/articles/PMC3939416/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23033116/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Effect%20of%20adherence%20as%20measured%20by%20MEMS,%20ritonavir%20boosting,%20and%20CYP3A5%20genotype%20on%20atazanavir%20pharmacokinetics%20in%20treatment-naive%20HIV-infected%20patients&author=RM%20Savic&author=A%20Barrail-Tran&author=X%20Duval&volume=92&publication_year=2012&pages=575-83&pmid=23033116&doi=10.1038/clpt.2012.137&)

36. Hesselink DA, van Schaik RH, van der Heiden IP et al. Genetic polymorphisms of the CYP3A4, CYP3A5, and MDR-1 genes and pharmacokinetics of the calcineurin inhibitors cyclosporine and tacrolimus. Clin Pharmacol Ther 2003; 74: 245–54.  [DOI](https://doi.org/10.1016/S0009-9236(03)00168-1) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12966368/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Genetic%20polymorphisms%20of%20the%20CYP3A4,%20CYP3A5,%20and%20MDR-1%20genes%20and%20pharmacokinetics%20of%20the%20calcineurin%20inhibitors%20cyclosporine%20and%20tacrolimus&author=DA%20Hesselink&author=RH%20van%20Schaik&author=IP%20van%20der%20Heiden&volume=74&publication_year=2003&pages=245-54&pmid=12966368&doi=10.1016/S0009-9236(03)00168-1&)

37. Chen JS, Li LS, Cheng DR et al. Effect of CYP3A5 genotype on renal allograft recipients treated with tacrolimus. Transplant Proc 2009; 41: 1557–61.  [DOI](https://doi.org/10.1016/j.transproceed.2009.01.097) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19545678/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Transplant%20Proc&title=Effect%20of%20CYP3A5%20genotype%20on%20renal%20allograft%20recipients%20treated%20with%20tacrolimus&author=JS%20Chen&author=LS%20Li&author=DR%20Cheng&volume=41&publication_year=2009&pages=1557-61&pmid=19545678&doi=10.1016/j.transproceed.2009.01.097&)

38. Kim KA, Joo HJ, Lee HM et al. Influence of ABCB1 and CYP3A5 genetic polymorphisms on the pharmacokinetics of quetiapine in healthy volunteers. Pharmacogenet Genomics 2014; 24: 35–42.  [DOI](https://doi.org/10.1097/FPC.0000000000000020) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24240480/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Influence%20of%20ABCB1%20and%20CYP3A5%20genetic%20polymorphisms%20on%20the%20pharmacokinetics%20of%20quetiapine%20in%20healthy%20volunteers&author=KA%20Kim&author=HJ%20Joo&author=HM%20Lee&volume=24&publication_year=2014&pages=35-42&pmid=24240480&doi=10.1097/FPC.0000000000000020&)

39. Kim KA, Park PW, Lee OJ et al. Effect of polymorphic CYP3A5 genotype on the single-dose simvastatin pharmacokinetics in healthy subjects. J Clin Pharmacol 2007; 47: 87–93.  [DOI](https://doi.org/10.1177/0091270006295063) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17192506/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Pharmacol&title=Effect%20of%20polymorphic%20CYP3A5%20genotype%20on%20the%20single-dose%20simvastatin%20pharmacokinetics%20in%20healthy%20subjects&author=KA%20Kim&author=PW%20Park&author=OJ%20Lee&volume=47&publication_year=2007&pages=87-93&pmid=17192506&doi=10.1177/0091270006295063&)

40. Safren SA, Biello KB, Smeaton L et al. Psychosocial predictors of non-adherence and treatment failure in a large scale multi-national trial of antiretroviral therapy for HIV: data from the ACTG A5175/PEARLS trial. PLoS One 2014; 9: e104178.  [DOI](https://doi.org/10.1371/journal.pone.0104178) | [PMC free article](/articles/PMC4143224/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25153084/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=Psychosocial%20predictors%20of%20non-adherence%20and%20treatment%20failure%20in%20a%20large%20scale%20multi-national%20trial%20of%20antiretroviral%20therapy%20for%20HIV:%20data%20from%20the%20ACTG%20A5175/PEARLS%20trial&author=SA%20Safren&author=KB%20Biello&author=L%20Smeaton&volume=9&publication_year=2014&pages=e104178&pmid=25153084&doi=10.1371/journal.pone.0104178&)

41. Pearson CR, Simoni JM, Hoff P et al. Assessing antiretroviral adherence via electronic drug monitoring and self-report: an examination of key methodological issues. AIDS Behav 2007; 11: 161–73.  [DOI](https://doi.org/10.1007/s10461-006-9133-3) | [PMC free article](/articles/PMC5096443/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16804749/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=AIDS%20Behav&title=Assessing%20antiretroviral%20adherence%20via%20electronic%20drug%20monitoring%20and%20self-report:%20an%20examination%20of%20key%20methodological%20issues&author=CR%20Pearson&author=JM%20Simoni&author=P%20Hoff&volume=11&publication_year=2007&pages=161-73&pmid=16804749&doi=10.1007/s10461-006-9133-3&)

42. de Boer IM, Prins JM, Sprangers MA et al. Using different calculations of pharmacy refill adherence to predict virological failure among HIV-infected patients. J Acquir Immune Defic Syndr 2010; 55: 635–40.  [DOI](https://doi.org/10.1097/QAI.0b013e3181fba6ab) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21934556/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Acquir%20Immune%20Defic%20Syndr&title=Using%20different%20calculations%20of%20pharmacy%20refill%20adherence%20to%20predict%20virological%20failure%20among%20HIV-infected%20patients&author=IM%20de%20Boer&author=JM%20Prins&author=MA%20Sprangers&volume=55&publication_year=2010&pages=635-40&pmid=21934556&doi=10.1097/QAI.0b013e3181fba6ab&)

43. D'Avolio A, Carcieri C, Cusato J et al. Intracellular accumulation of atazanavir/ritonavir according to plasma concentrations and OATP1B1, ABCB1 and PXR genetic polymorphisms. J Antimicrob Chemother 2014; 69: 3061–6.  [DOI](https://doi.org/10.1093/jac/dku234) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24997317/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Antimicrob%20Chemother&title=Intracellular%20accumulation%20of%20atazanavir/ritonavir%20according%20to%20plasma%20concentrations%20and%20OATP1B1,%20ABCB1%20and%20PXR%20genetic%20polymorphisms&author=A%20D'Avolio&author=C%20Carcieri&author=J%20Cusato&volume=69&publication_year=2014&pages=3061-6&pmid=24997317&doi=10.1093/jac/dku234&)

44. Ribaudo HJ, Liu H, Schwab M et al. Effect of CYP2B6, ABCB1, and CYP3A5 polymorphisms on efavirenz pharmacokinetics and treatment response: an AIDS Clinical Trials Group study. J Infect Dis 2010; 202: 717–22.  [DOI](https://doi.org/10.1086/655470) | [PMC free article](/articles/PMC2919241/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20662624/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Infect%20Dis&title=Effect%20of%20CYP2B6,%20ABCB1,%20and%20CYP3A5%20polymorphisms%20on%20efavirenz%20pharmacokinetics%20and%20treatment%20response:%20an%20AIDS%20Clinical%20Trials%20Group%20study&author=HJ%20Ribaudo&author=H%20Liu&author=M%20Schwab&volume=202&publication_year=2010&pages=717-22&pmid=20662624&doi=10.1086/655470&)

45. Haas DW, Smeaton LM, Shafer RW et al. Pharmacogenetics of long-term responses to antiretroviral regimens containing efavirenz and/or nelfinavir: an Adult Aids Clinical Trials Group Study. J Infect Dis 2005; 192: 1931–42.  [DOI](https://doi.org/10.1086/497610) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16267764/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Infect%20Dis&title=Pharmacogenetics%20of%20long-term%20responses%20to%20antiretroviral%20regimens%20containing%20efavirenz%20and/or%20nelfinavir:%20an%20Adult%20Aids%20Clinical%20Trials%20Group%20Study&author=DW%20Haas&author=LM%20Smeaton&author=RW%20Shafer&volume=192&publication_year=2005&pages=1931-42&pmid=16267764&doi=10.1086/497610&)

46. Zhu P, Zhu Q, Zhang Y et al. ABCB1 variation and treatment response in AIDS patients: initial results of the Henan cohort. PLoS One 2013; 8: e55197.  [DOI](https://doi.org/10.1371/journal.pone.0055197) | [PMC free article](/articles/PMC3555879/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23372834/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=ABCB1%20variation%20and%20treatment%20response%20in%20AIDS%20patients:%20initial%20results%20of%20the%20Henan%20cohort&author=P%20Zhu&author=Q%20Zhu&author=Y%20Zhang&volume=8&publication_year=2013&pages=e55197&pmid=23372834&doi=10.1371/journal.pone.0055197&)

47. Fellay J, Marzolini C, Meaden ER et al. Response to antiretroviral treatment in HIV-1-infected individuals with allelic variants of the multidrug resistance transporter 1: a pharmacogenetics study. Lancet 2002; 359: 30–6.  [DOI](https://doi.org/10.1016/S0140-6736(02)07276-8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11809184/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Response%20to%20antiretroviral%20treatment%20in%20HIV-1-infected%20individuals%20with%20allelic%20variants%20of%20the%20multidrug%20resistance%20transporter%201:%20a%20pharmacogenetics%20study&author=J%20Fellay&author=C%20Marzolini&author=ER%20Meaden&volume=359&publication_year=2002&pages=30-6&pmid=11809184&doi=10.1016/S0140-6736(02)07276-8&)

48. Lubomirov R, Colombo S, di Iulio J et al. Association of pharmacogenetic markers with premature discontinuation of first-line anti-HIV therapy: an observational cohort study. J Infect Dis 2011; 203: 246–57.  [DOI](https://doi.org/10.1093/infdis/jiq043) | [PMC free article](/articles/PMC3071070/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21288825/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Infect%20Dis&title=Association%20of%20pharmacogenetic%20markers%20with%20premature%20discontinuation%20of%20first-line%20anti-HIV%20therapy:%20an%20observational%20cohort%20study&author=R%20Lubomirov&author=S%20Colombo&author=J%20di%20Iulio&volume=203&publication_year=2011&pages=246-57&pmid=21288825&doi=10.1093/infdis/jiq043&)

49. Berno G, Zaccarelli M, Gori C et al. Analysis of single-nucleotide polymorphisms (SNPs) in human CYP3A4 and CYP3A5 genes: potential implications for the metabolism of HIV drugs. BMC Med Genet 2014; 15: 76.  [DOI](https://doi.org/10.1186/1471-2350-15-76) | [PMC free article](/articles/PMC4083125/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24986243/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Med%20Genet&title=Analysis%20of%20single-nucleotide%20polymorphisms%20(SNPs)%20in%20human%20CYP3A4%20and%20CYP3A5%20genes:%20potential%20implications%20for%20the%20metabolism%20of%20HIV%20drugs&author=G%20Berno&author=M%20Zaccarelli&author=C%20Gori&volume=15&publication_year=2014&pages=76&pmid=24986243&doi=10.1186/1471-2350-15-76&)
