# Novel Sequence Variations in the Brain-Derived Neurotrophic Factor Gene and Association with Major Depression and Antidepressant Treatment Response

## Metadata
**Authors:** Julio Licinio, Chuanhui Dong, Ma-Li Wong
**Journal:** Archives of general psychiatry
**Date:** 2009 May
**DOI:** [10.1001/archgenpsychiatry.2009.38](https://doi.org/10.1001/archgenpsychiatry.2009.38)
**PMID:** 19414708
**PMCID:** PMC4272010
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4272010/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC4272010/pdf/nihms645315.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC4272010/pdf/nihms645315.pdf)

## Abstract

**Context:** 
Variations in the brain-derived neurotrophic factor (BDNF) gene have been associated with psychiatric disorders, such as schizophrenia, bipolar and major depressive (MDD) disorder, and with antidepressant action. Deep sequencing of the BDNF gene may identify new genetic variations and bring further insights into psychiatric genetics.

**Objective:** 
To better characterize sequence variability in the BDNF gene.

**Design:** 
A genomic DNA region of 72 kb that contained the entire BDNF coding sequence and 5kb of flanking regions was re-sequenced in more than 500 subjects.

**Setting and Participants:** 
Re-sequencing data was obtained in 264 controls and 272 MDD individuals collected from the Mexican-American community in Los Angeles; individuals were accessed by the same bilingual clinical research team.

**Main Outcome Measures:** 
Identification of novel genetic polymorphisms in the BDNF gene, assessment of their frequencies and associations with MDD risk or response to antidepressants.

**Results:** 
We identified 83 novel single nucleotide polymorphisms (SNP): 30 in untranslated regions, in coding sequences, 37 in introns, and 12 in upstream regions. 3 of 4 rare novel coding SNPs were non-synonymous. Association analyses of MDD and controls revealed that 6 SNPs were associated with MDD (rs12273539, rs11030103, rs6265, rs28722151, rs41282918, rs11030101) and two haplotypes in different blocks were significantly associated with MDD. One recently reported 5′ UTR (untranslated region) SNP, rs61888800, was associated with antidepressant response after adjusting for age, gender, medication and baseline HAM-D21 score.

**Conclusions:** 
We identified 83 new BDNF polymorphisms and found that genetic frequency variations across populations exist for this gene. One single SNP (rs12273539) and two haplotypes (one including Val66, another near exon VIIIh) remained significant after adjusting for multiple testing. One 5′ UTR SNP was associated with antidepressant response. Further studies using larger independent samples are needed to confirm this association and to understand the implications of these novel BDNF variations in psychiatric disorders.

### Context

Variations in the brain-derived neurotrophic factor (BDNF) gene have been associated with psychiatric disorders, such as schizophrenia, bipolar and major depressive (MDD) disorder, and with antidepressant action. Deep sequencing of the BDNF gene may identify new genetic variations and bring further insights into psychiatric genetics.

### Objective

To better characterize sequence variability in the BDNF gene.

### Design

A genomic DNA region of 72 kb that contained the entire BDNF coding sequence and 5kb of flanking regions was re-sequenced in more than 500 subjects.

### Setting and Participants

Re-sequencing data was obtained in 264 controls and 272 MDD individuals collected from the Mexican-American community in Los Angeles; individuals were accessed by the same bilingual clinical research team.

### Main Outcome Measures

Identification of novel genetic polymorphisms in the BDNF gene, assessment of their frequencies and associations with MDD risk or response to antidepressants.

### Results

We identified 83 novel single nucleotide polymorphisms (SNP): 30 in untranslated regions, in coding sequences, 37 in introns, and 12 in upstream regions. 3 of 4 rare novel coding SNPs were non-synonymous. Association analyses of MDD and controls revealed that 6 SNPs were associated with MDD (rs12273539, rs11030103, rs6265, rs28722151, rs41282918, rs11030101) and two haplotypes in different blocks were significantly associated with MDD. One recently reported 5′ UTR (untranslated region) SNP, rs61888800, was associated with antidepressant response after adjusting for age, gender, medication and baseline HAM-D21 score.

### Conclusions

We identified 83 new BDNF polymorphisms and found that genetic frequency variations across populations exist for this gene. One single SNP (rs12273539) and two haplotypes (one including Val66, another near exon VIIIh) remained significant after adjusting for multiple testing. One 5′ UTR SNP was associated with antidepressant response. Further studies using larger independent samples are needed to confirm this association and to understand the implications of these novel BDNF variations in psychiatric disorders.

## INTRODUCTION

The neutrophins are secreted peptides that are critically involved in differentiation and survival of neuronal populations ^1–3^. Brain-derived neurotrophic factor (BDNF) ^4–7^ is a neurotrophin that is abundantly and widely expressed in the CNS ^8, 9^. During the past decade, BDNF has emerged as a key factor implicated in complex behavioral patterns in the developing CNS and in disease. BDNF modulates signaling pathways that rapidly affect local synaptic function but it also has long-term effects on gene transcription. It promotes neuronal survival in the peripheral and CNS via the transcription factor cAMP-response element (CREB), which influences the expression of *Bcl-2*, a pro-survival gene. It also has important roles in excitatory synaptic transmission and plasticity^10–13^, memory processing and storage ^13–18^, and kindling and temporal lobe epilepsy ^19–22^. This relevance to crucial CNS functions has raised interest in its role in neurodegenerative and psychiatric disorders.

Allelic variations of the BDNF gene have been implicated in several conditions. Specifically, the allelic variation Thr2Ile (substitution of isoleucine for threonine at aminoacid position 2 in the coding sequence) has been implicated in the congenital central hypoventilation syndrome ^23^. The variations in BDNF have been extensively studied and implicated in the susceptibility to memory and hippocampal function impairments ^24^, and several psychiatric disorders^25^, such as obsessive-compulsive ^26^, eating ^27, 28^, bipolar ^29,30–34^, schizophrenia^35^, major depression ^36, 37^, and Alzheimer’s disease ^38–40^. In spite of conflicting findings in replication studies have been noted for several of these associations, it is interesting to note that the less frequent variation Met66, which is associated with poorer episodic memory and abnormal hippocampal activation in functional magnetic resonance imaging, generally confers a protective effect to neuropsychiatric conditions.

The genetic factors that contribute to human disease show enormous variation in the allelic spectra in number and population frequency of disease-predisposing alleles. Common complex disorders are multi-factorial and probably composed of both common genetic variants (common disease/common allele model) with small effect and rare sequence variants (rare variant/common disease model) with larger effect^41^. Although, the common allele is the prevalent of these two competing models in the genetic influence in common complex conditions, it has been predicted that re-sequencing studies may identify many rarer variants (>5%) of intermediate effect associated with common disorders and they may also be able to identify structural variations in genomic DNA, such as duplication and deletions of DNA sequences ^42, 43^.

Given the functional importance of BDNF in the CNS, the understanding of its allelic variants may be relevant to understanding its role in neuropsychiatric conditions. In spite a number of studies conducted to examine the association of BDNF variants, most of them have been focused on genotyping tag single nucleotide polymorphisms (tagSNPs) or the functional coding SNP rs6265. To our knowledge, no study has comprehensive surveyed BDNF sequence variation through direct sequencing and correlated the identified genetic variants with disease susceptibility. To discover new BDNF genetic variants and detect rare variants, we sequenced the whole BDNF gene and 5 kb flanking region in a total of 536 DNA samples comprised of 264 control and 272 depressed Mexican-American individuals. We further investigated all the identified genetic variants for association with risk for major depression and for relation to efficacy of antidepressant treatment.

## METHODS

### Participants

Participants were 264 controls and 272 patients with major depressive disorder (MDD) aged 19–68 years old. All participants were Mexican-Americans and had at least 3 grandparents born in Mexico. MDD was defined as a DSM-IV (Diagnostic and Statistical Manual of Mental Disorders, 4th Ed) diagnosis of current, unipolar major depressive episode and a HAM-D21 (21-Item Hamilton Depression Rating Scale) score of 18 or greater with item number 1 (depressed mood) rated 2 or greater. All MDD patients were enrolled in a pharmacogenetic study of antidepressant treatment response as previously described, and registered at clinical trials.gov (ID number [NCT00265291](https://clinicaltrials.gov/ct2/show/NCT00265291))^44, 45^. The demographic characteristics and the numbers of subjects in each subgroup were presented in [Supplementary Table 1](#SD1) and a flowchart ([Supplementary Figure 1](#SD1)). Briefly, in their primary language, all MDD patients had a comprehensive psychiatric and medical assessment based on the diagnostic and ratings instruments that had been fully validated in English and in Spanish. Exclusion criteria included active medical illnesses that could be etiologically related to the ongoing depressive episode, current or active suicidal ideation with a plan and strong intent, pregnancy, lactation, current use of medications with significant central nervous system activity, which interfere with EEG activity (e.g. benzodiazepines) or any other antidepressant treatment within the 2 weeks prior to enrollment, illicit drug use and/or alcohol abuse in the last 3 months or current enrollment in psychotherapy. Control individuals for our genomic studies were in general good health but were not screened for medical or psychiatric illness, they were age- and gender- matched and recruited from the same Mexican-American community in Los Angeles by the same bilingual clinical research team.

### Antidepressant Treatment

All patients had an initial comprehensive psychiatric and medical assessment and, if enrolled, had weekly structured follow-up assessments for 9 weeks. The study consisted of two phases: a 1-week single-blind placebo lead-in phase to minimize the impact of placebo responders followed, if subjects continued to meet the inclusion criteria after phase 1, by random assignment to one of the two treatment groups: fluoxetine 10–40 mg/day or desipramine 50–200 mg/day, administered in a double-blind manner for 8 weeks. Our primary clinical outcome measure was HAM-D21 score and clinical remission on antidepressants was defined as having a final (week 8) HAM-D21 score < 8^44^. In addition, the relative response change was also computed as the difference in HAM-D21 score between pre- and post-treatment divided by the pre-treatment HAM-D21 score.

### Genomic DNA Collection and Sequencing

At the initial visit, blood samples were collected under informed consent from the participating individuals into EDTA (K2EDTA) BD Vacutainer EDTA tubes (Becton Dickinson, Franklin Lakes, NJ), and genomic DNA was isolated by using Gentra Puregene DNA purification kits (Gentra Systems, Indianapolis, IN). BDNF DNA sequencing was completed to identify genetic polymorphisms by the Wellcome Trust Sanger Institute following their ExoSeq protocol ([http://www.sanger.ac.uk/humgen/exoseq/](http://www.sanger.ac.uk/humgen/exoseq/)). A 72 kb genomic DNA region, containing the entire BDNF coding region and 5kb flanking region, was sequenced. Briefly, DNA sequences were extracted from the Vega database ([http://vega.sanger.ac.uk/index.html](http://vega.sanger.ac.uk/index.html)). Primers were designed automatically using Primer3 to amplify DNA and primer pairs were checked for uniqueness prior to ordering and pre-screened to determine the optimum conditions for amplification. After amplification a sample of the products were visualized on an agarose gel, to confirm the size of the PCR product. The remaining PCR product was then cleaned-up using two enzymes, Exonuclease 1 and Shrimp Alkaline Phosphatase. Bi-directional sequencing of amplicons was carried out using Big DyeTM chemistry. SNPs were called using ExoTrace, a website algorithm developed for the detection of heterozygotes in sequence traces, which processes the sense and antisense sequence reads separately and subsequently and combines the results to allow SNP scoring.

### Nucleotide Diversity (θ) and Population Differentiation (FST) Estimation

Nucleotide diversity (*θ*) and its standard deviation (*S(θ*)) were calculated by SNP class under the assumption of an infinite neutral allele model as follows^46, 47^: *θ* = *K/aL, * S(θ)=aθL+b(θL)2/aL,a=∑i=2n1/(i-1), and b=∑i=2n1/(i-1)2, where *K* = the number of observed SNPs among *L* base pairs of genomic sequence in a sample of *n* alleles. All calculations were based on *n*=990 for all the sites given that the average sample size was 495 individuals across all the polymorphisms. The pairwise F_ST_ values were estimated for the dbSNPs which were both detected in our Mexican American sample and reported in HapMap sample and were calculated as described by Weir^48–50^.

### Hardy-Weinberg Equilibrium (HWE) Test and Population Stratification Analysis

Case-control study design is an efficient method to examine associations between candidate alleles and disease. But in order to compare allele frequencies and to be able to treat chromosomes as independent observations, the genotype frequencies must be in Hardy-Weinberg equilibrium^51^. Deviation from Hardy-Weinberg equilibrium was tested separately for healthy controls and patients by using PLINK program Version1.00 ([http://pngu.mgh.harvard.edu/~purcell/plink/](http://pngu.mgh.harvard.edu/~purcell/plink/))^52^. SNPs that are not in HWE in the healthy control group were excluded from the allele-based association analyses of cases and controls.

Another confounding factor that may impact the internal validity of case-control studies is the presence of population stratification. We used two approaches to test for the hidden stratification in our data. Firstly, 54 unlinked SNPs across 22 autosomal chromosomes were employed to analyze a combined sample with the genotype data download from three HapMap ethnic samples using STRUCTURE program. Three distinct clusters were identified with an average proportion of at least 92% of individuals correctly assigned to the given ethnic populations (CEU, CHB+JPT, YRI) ([Supplementary Figure 2A](#SD1)). We then used this panel of SNPs to test our sample and observed an almost equal proportion assigned to each clusters given K=2, 3, 4 in both cases and controls ([Supplementary Figure 2B](#SD1)). Secondly, genotype frequencies from each of the 54 unlinked SNPs were compared between cases and controls using the method described by Pritchard et al^53^. No significant difference was found based on an overall test statistic (χ^2^=100.50, *df* =108, *p*=0.68).

### Genetic Association Analyses of Case and Controls

For SNP-based association analysis, Fisher’s Exact test (2-tailed) was performed to compare allele frequencies and genotype distributions between depressed and healthy individuals by using PLINK program. In the allelic association analysis, each polymorphism was tested in controls to ensure the fitting with HWE; the odds ratio (OR) on the 2×2 contingency table of allele counts and its 95% confidence interval were also estimated for the polymorphism associated with the diagnosis of depression. In the genotypic association analysis, the SNP effects were tested under a codominant model on the 2×3 contingency table of genotype counts. In addition, logistic regression analyses were performed to test whether the observed SNP-depression association remained valid after controlling age and gender using SAS package (SAS Institute, Cary, NC).

For haplotype-based association analysis, haplotype blocks were identified by searching for “spine” of strong LD running from one marker to another along the legs of the triangle in the LD chart and haplotype population frequencies were estimated by using expectation maximization (EM) algorithm performed in computer program Haploview (Version 4, Broad Institute, [http://www.broad.mit.edu/mpg/haploview/](http://www.broad.mit.edu/mpg/haploview/)) ^54^. Haplotype frequencies were compared between depressed and control individuals to test whether a certain haplotype was associated with a diagnosis of depression.

To correct for multiple testing, 20,000 permutations were performed to estimate the adjusted *p* values for both single SNP-based analyses and haplotype-based analyses by using Haploview.

### Genetic Association Analysis of Response to Antidepressants

Data analyses were performed using both intention-to-treat (ITT) and completed-treatment samples. ITT sample consisted of patients who were randomized to one arm and received at least one dose of antidepressant medication and completed-treatment sample consisted to patients who completed 8-week of antidepressant treatment. The last observation carried forward (LOCF) approach was used to imput missing outcome in the ITT analysis. For discrete outcome (remission vs. non-remission), we investigated the allelic and genotypic association with the response to antidepressant treatment using the similar approaches to those in the analysis of cases and controls. For the quantitative outcome (relative reduction % in HAM-D21 scores between pre- and post-treatment), we conducted the analyses based on three genetic models (additive, dominant and recessive) and first performed the analyses using the combined samples of patients treated with desipramine or fluoxetine. We then performed the analysis separately by antidepressant medication (desipramine only, fluoxetine only). We employed a multiple linear regression model to examine the association between genotype and relative HAM-D21 score reduction by controlling for age, gender, and baseline (pre-treatment) HAM-D21 score using PLINK program.

### Power Calculation

Power to test the allelic association with depression was estimated with a range of effect size (odds ratio, OR) between 1.35 and 2.25 and minor allele frequency (MAF) between 0.1 and 0.25 using PAWE program ^55^. Power analyses showed that at a two-sided significance level of 0.05, sample sizes of 265 cases and 265 controls can achieve 80% power to detect an allelic OR of 1.68, 1.57, 1.50 and 1.46 with an MAF of 0.10, 0.15, 0.20 and 0.25, respectively. Power calculations for the association of BDNF variants with antidepressant treatment continuous outcome were given for a range of allele frequencies and Cohen’s effect size (mean difference in unit of standard deviation) based on dominant genetic model and using the Quanto (Version 1.2.3) program ^56, 57^. Sample size is assumed to be 200 for combined sample and 100 for each antidepressant treatment group based on an ITT design. Power analyses showed that at a two-sided significance level of 0.05 and when the allele frequency ≥ 0.15, the power is ≥ 89% to uncover a moderate effect size of 0.5 for a sample of 200 patients and ≥ 78% to detect a medium effect size of 0.6 for a sample of 100 patients.

## RESULTS

### Detection of Sequence Variation

Approximately 72 kb of DNA sequence containing 5 kb of flanking regions was systematically screened for novel nucleotide sequence variations in a sample of 536 Mexican American individuals, 264 controls and 272 depressed. A total of 130 nucleotide sequence variations were identified ([Table 1](#T1)). They included 83 novel SNPs and 47 dbSNPs: 40 in untranslated regions (UTRs), 6 in coding sequences, 62 in intronic sequences, and 22 in the flanking regions. Among 6 coding SNPs, 3 novel non-synonymous SNPs [[NT_009237](https://www.ncbi.nlm.nih.gov/nuccore/NT_009237).17_26467094 (Ala/Thr), [NT_009237](https://www.ncbi.nlm.nih.gov/nuccore/NT_009237).17_26467235 (His/Gly), [NT_009237](https://www.ncbi.nlm.nih.gov/nuccore/NT_009237).17_26467246 (Gly/Asp)], and 1 synonymous SNP ([NT_009237](https://www.ncbi.nlm.nih.gov/nuccore/NT_009237).17_26466714) were found and their minor allele frequencies were respectively 0.0019, 0.0019, 0.001, and 0.001 in the combined sample of cases and controls. Seventy-nine other novel polymorphisms included: 30 UTR SNPs, 37 intronic SNPs and 12 upstream SNPs ([Supplementary Table 2](#SD1)). The minor allele frequencies for the novel polymorphisms ranged from 0.0009 to 0.2445 with an allele distribution of ≤0.001, 37.6%; >0.001 and <0.01, 50.5%; and >0.01, 11.9% in the combined sample of cases and controls.

### Table 1.

| Location* | Sequence Screened (bp) | No. of Novel SNP | No. of dbSNP | No. of Total SNP | Nucleotide Diversity ± SD (x10−4) | Transition % |
| --- | --- | --- | --- | --- | --- | --- |
| Coding | 792 | 4 | 2 | 6 | 10.1 ± 4.5 | 100 |
| 5′ UTR | 2434 | 16 | 2 | 18 | 9.9 ± 2.9 | 72.2 |
| Intron | 60703 | 37 | 25 | 62 | 1.4 ± 0.3 | 71.0 |
| 3′ UTR | 2928 | 14 | 8 | 22 | 10.5 ± 2.8 | 69.6 |
| Upstream | 4989 | 12 | 10 | 22 | 5.9 ± 1.6 | 68.2 |
| Total | 71846 | 83 | 47 | 130 | 2.4 ± 0.5 | 71.8 |

Table 1 Caption: Detected BDNF SNPs in Mexican Americans

### Nucleotide Diversity

The nucleotide diversity was estimated in each class of sites (coding, 3′ UTR, 5′ UTR and intronic) by correcting for both sample size and the length of the screened site ([Table 1](#T1)). The nucleotide diversities were comparable for coding (0.0010 ± 0.0005), 3′UTR (0.0011 ± 0.0003) and 5′ UTR (0.0010 ± 0.0003) regions, but the estimate showed much lower nucleotide diversity in intronic region (0.00014 ± 0.00003). SNPs in UTRs or coding regions showed a 6-fold more diversity compared to those in intronic region. For the type of substitution, all the identified coding polymorphisms were transition, whereas the transition rates were 71.0%, 69.6%, 72.2% and 68.2% for intronic, 3′ UTR, 5′ UTR, and upstream regions, respectively.

### Population Differentiation

Among the 47 dbSNPs detected, 18 were reported in three HapMap ethnic groups: White (CEU), Black (YRI), and Asian (CHB+JPT) in the NCBI database as of 06/25/2008. Pairwise F_ST_ values between Mexican Americans (MA) and each HapMap ethnic sample were computed for the shared 18 SNPs and showed in [Table 2](#T2). Overall, the greater similarity in allele frequencies was found between Mexican Americans and Caucasians with a lower mean F_ST_ in MA vs CEU of 0.03, compared to that in MA vs YRI of 0.1 and in MA vs CHB+JPT of 0.09. For the single-locus estimates of F_ST_ values, large F_ST_ values (> 0.1) were observed at 4 SNPs (rs7124442, rs11819808, rs4923468, and rs7931755) in MA vs YRI (22.2%) and at 5 SNPs (rs6265, rs11030102, rs11030104, rs988748, and rs10767664) in MA vs CHB+JPT (27.8%), but less often (5.5%) in MA vs CEU (1 SNP: rs12273539).

### Table 2.

| SNP | Major/ Minor Allele | Minor Allele Frequency (MAF) |  |  |  |  |  | Fst |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| Mexican American (MA) |  |  | HapMap Sample |  |  |  |  |  |  |  |
| Cases | Controls | All | CEU | YRI | HCB+JPT | MA vs CEU | MA vs YRI | MA vs HCB+JPT |  |  |
| rs7124442 | T/C | 0.23 | 0.26 | 0.25 | 0.37 | 0.54 | 0.07 | 0.03 | 0.18 | 0.09 |
| rs6265 | G/A | 0.10 | 0.15 | 0.12 | 0.18 | 0.00 | 0.48 | 0.01 | 0.08 | 0.31 |
| rs11030101 | A/T | 0.26 | 0.33 | 0.29 | 0.40 | 0.12 | 0.31 | 0.02 | 0.07 | 0.00 |
| rs11819808 | C/T | 0.01 | 0.00 | 0.01 | 0.00 | 0.28 | 0.00 | 0.00 | 0.45 | 0.00 |
| rs11030102 | C/G | 0.17 | 0.18 | 0.18 | 0.29 | 0.07 | 0.01 | 0.04 | 0.04 | 0.11 |
| rs12273539 | C/T | 0.35 | 0.23 | 0.29 | 0.00 | 0.34 | 0.13 | 0.20 | 0.00 | 0.06 |
| rs11030104 | A/G | 0.12 | 0.15 | 0.14 | 0.20 | 0.00 | 0.49 | 0.01 | 0.08 | 0.29 |
| rs11030109 | G/A | 0.02 | 0.03 | 0.03 | 0.04 | 0.00 | 0.00 | 0.00 | 0.01 | 0.01 |
| rs988748 | C/G | 0.15 | 0.16 | 0.15 | 0.22 | 0.05 | 0.49 | 0.01 | 0.04 | 0.25 |
| rs4923468 | C/A | 0.01 | 0.02 | 0.02 | 0.02 | 0.19 | 0.02 | 0.00 | 0.26 | 0.00 |
| rs10767664 | A/T | 0.14 | 0.16 | 0.15 | 0.20 | 0.04 | 0.49 | 0.00 | 0.04 | 0.26 |
| rs7931755 | A/G | 0.01 | 0.01 | 0.01 | 0.00 | 0.25 | 0.00 | 0.00 | 0.41 | 0.00 |
| rs2030324 | G/A | 0.37 | 0.41 | 0.39 | 0.57 | 0.55 | 0.56 | 0.06 | 0.05 | 0.05 |
| rs12273363 | T/C | 0.15 | 0.16 | 0.16 | 0.19 | 0.07 | 0.01 | 0.00 | 0.03 | 0.09 |
| rs908867 | C/T | 0.04 | 0.05 | 0.04 | 0.12 | 0.10 | 0.04 | 0.04 | 0.03 | 0.00 |
| rs7931247 | C/T | 0.37 | 0.42 | 0.39 | 0.57 | 0.55 | 0.56 | 0.06 | 0.05 | 0.05 |
| rs12288512 | G/A | 0.14 | 0.15 | 0.15 | 0.19 | 0.07 | 0.01 | 0.00 | 0.02 | 0.09 |
| rs11030123 | G/A | 0.04 | 0.05 | 0.04 | 0.12 | 0.10 | 0.04 | 0.04 | 0.02 | 0.00 |

Table 2 Caption: Allele Frequencies and Fst Values for BDNF dbSNPs Shared by Mexican Americans and HapMap Samples

### Single SNP-Based Association Analyses of Cases and Controls

SNP-based allelic association analyses revealed that 6 polymorphisms were associated with MDD (rs12273539, *p*=0.00009; rs11030103, *p*=0.008; rs6265, *p*=0.009; rs28722151, *p*=0.01; rs41282918, *p*=0.01; rs11030101, *p*=0.02) ([Table 3](#T3)). All these 6 SNPs had a minor allele frequency of ≥ 0.14 and their genotypes were in Hardy-Weinberg equilibrium in controls. Genotyped-based analyses also showed that the 6 polymorphisms were associated with depression phenotype with a *p* ≤ 0.04 ([Table 3](#T3)). Among the 6 associated SNPs, 4 were intronic variants with OR ranging from 1.37 to 1.80; 1 SNP was 3′ UTR variant (rs41282918) with an effect of OR=2.13 (95% CI: 1.18–3.86); and 1 SNP was non-synonymous variant (rs6265) with an effect of OR=1.66 (95% CI: 1.14–2.41). Logistic regression analyses did not reveal a significant difference in age or gender between cases and controls and the associations of the 6 SNPs with depression remained similar after adjusting for age and gender. Permutation analysis showed that only SNP rs12273539 remained significant after adjusting for multiple tests with a corrected *p* value of 0.002.

### Table 3.

| SNP | Position | SNP Type | Risk/Non-risk Allele | Control Risk Allele Freq | OR (95%CI) | p* |
| --- | --- | --- | --- | --- | --- | --- |
| rs41282918 | 27635356 | 3′ UTR | A/C | 0.84 | 2.13(1.18, 3.86) | 0.01 (0.02) |
| rs6265 | 27636492 | Non-synonymous | G/A | 0.85 | 1.66(1.14, .41) | 0.009 (0.008) |
| rs11030101 | 27637320 | Intronic | A/T | 0.67 | 1.37(1.05, 1.78) | 0.02 (0.04) |
| rs28722151 | 27637752 | Intronic | C/G | 0.68 | 1.48(1.10, 1.99) | 0.01 (0.009) |
| rs11030103 | 27638909 | Intronic | G/A | 0.19 | 1.80(1.18, 2.74) | 0.008 (0.03) |
| rs12273539 | 27639887 | Intronic | T/C | 0.23 | 1.75(1.32, 2.31) | 0.00009 (0.0008) |

Table 3 Caption: BDNF Polymorphisms Associated with Depression

### Haplotype-Based Association Analysis of Cases and Controls

[Figure 1](#F1) shows that 7 haplotype blocks were identified by searching for the solid spine of strong LD, Among the 130 detected polymorphisms, 33 SNPs with a minor allele frequency ≥1.5% were included in the haplotype analyses. Several haplotypes were found to be associated with the diagnosis of depression in block 3 (5 SNPs: rs56820186, rs6265, rs11030101, rs28722151, and rs11030102) and block 4 (4 SNPs: rs57083135, [NT_009237](https://www.ncbi.nlm.nih.gov/nuccore/NT_009237).17_26469156, rs110303103, and rs12273539). Block 3 included three SNPs associated with depression ([Table 3](#T3)). The most significant association in block 3 was found for a common haplotype TGACC, and the haplotype frequency was 0.453 in cases and 0.316 in controls (χ^2^=20.80, *p*=0.000005; permutation adjusted *p*=0.0002). In block 4, the most significant association was found for haplotype CTGT, and the haplotype frequency was 0.337 in cases and 0.229 in controls (χ^2^=15.06, *p*=0.0001; permutation adjusted *p*=0.002). No other haplotypes were associated with depression after adjusting for multiple testing in the permutation tests.

### Figure 1. Linkage disequilibrium (LD) pattern in BDNF.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/acf2/4272010/58614c51be20/nihms645315f1.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4272010_nihms645315f1.jpg)

Standard color scheme in Haploview program is used to display the level of logarithm of odds (LOD) and the D′ (right inserted key). Estimated statistics of the D′ are shown in each box. They indicate the LD relationship between each pair of SNPs and they are not labeled if D′=1.00. The BDNF gene structure is illustrated by a long horizontal white bar with vertical lines indicating the relative positions of SNPs and black box representing alternative exons named by Pruunsild et al7. SNPs associated with depression are marked in orange (UTR), red (coding) and blue (intronic) circles. Left inset shows haplotype frequencies in cases and controls and the p values for the association analysis between haplotype and diagnosis of depression in blocks 3 and 4.

### Genetic Association Analysis of Response to Antidepressants

In the present study, there were 200 MDD patients who received at least one dose of antidepressant treatment (ITT sample of 103 received desipramine and 97 received fluoxetine) and 142 MDD patients who completed 8-week antidepressant treatment (completed-treatment sample of 68 with desipramine and 74 with fluoxetine). For the discrete outcome (remission vs non-remission), no detected polymorphisms were found to be significantly associated with the remission status in allelic and genotype-based analyses using ITT or completed-treatment samples. For the quantitative outcome (relative reduction in HAM-D21 score), one newly reported 5′ UTR SNP, rs61888800, was found to be associated with the better response to antidepressant treatment (*p*=0.02) after adjusting for age, gender, medication and baseline HAM-D21 score in the combined sample of patients treated with desipramine or fluoxetine in completed-treatment sample analysis. Patients who had GG genotype showed a larger average reduction of HAM-D21 score of 66.3% (95CI: 62.0–70.7%) compared to those who had non-GG genotype and had an average relative reduction of HAM-D21 score of 56.5% (95CI: 48.6–64.57%). For the medication-specific analyses, eight BDNF polymorphisms were found to be associated with the HAM-D score reduction among the patients treated with desipramine in both ITT and completed-treatment analyses with a p ≤ 0.05 after controlling for age, gender and baseline HAM-D score ([Table 4](#T4)), but no remained significant after adjusting for multiple testing. Among the 8 SNPs associated with response to desipramine treatment, all showed 14% larger reduction of HAM-D score in the patients homozygous for major allele except rs12273539, which showed 14% smaller reduction in patients homozygous for major allele in completed-treatment analysis and showed similar pattern but with a smaller reduction in ITT analysis. No associated polymorphism remained significant after adjusting for multiple testing through permutation and no detected SNPs were found significantly associated with the reduction of HAM-D scores in fluoxetine-treated group.

### Table 4.

| SNP (Type*) | Chr Position | Genotype | Intent-to-treat analysis^ |  |  |  | Complete-case analysis^ |  |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| N | Mean ± SD | β (95% CI) | p | N | Mean ± SD | β (95% CI) | p |  |  |  |
| rs7124442 (3′ UTR) | 27633617 | CC/CT | 43 | 37.97 ± 30.35 | 14.88 (2.99, 26.77) | 0.02 | 28 | 50.17 ± 25.58 | 14.6 (3.09, 26.11) | 0.02 |
|   | TT | 58 | 48.17 ± 30.77 | 40 | 60.88 ± 22.58 |  |  |  |  |  |
| rs11030102 (Intronic) | 27638172 | GG/GC | 33 | 35.64 ± 30.60 | 15.72 (3.14, 28.31) | 0.02 | 21 | 46.64 ± 27.50 | 15.09 (3.08, 27.11) | 0.02 |
|   | CC | 65 | 49.45 ± 30.04 | 47 | 60.86 ± 21.56 |  |  |  |  |  |
| rs12273539 (Intronic) | 27639887 | TT/TC | 53 | 47.30 ± 30.89 | −13.70 (−25.95, −1.45) | 0.03 | 35 | 61.13 ± 21.34 | −14.16 (−26.1, −2.23) | 0.02 |
|   | CC | 44 | 36.35 ± 31.42 | 28 | 49.66 ± 27.49 |  |  |  |  |  |
| rs61888800 (5′ UTR, Intronic) | 27678854 | TT/TG | 33 | 36.05 ± 30.83 | 17.12 (4.19, 30.05) | 0.01 | 21 | 47.28 ± 27.64 | 18.41 (6.24, 30.57) | 0.004 |
|   | GG | 60 | 49.84 ± 31.14 | 41 | 63.84 ± 20.96 |  |  |  |  |  |
| rs56133711 (Intronic) | 27679910 | AA/AG | 28 | 36.54 ± 29.80 | 15.52 (2.64, 28.40) | 0.02 | 18 | 48.57 ± 28.44 | 14.12 (1.20, 27.04) | 0.04 |
|   | GG | 63 | 50.16 ± 29.58 | 45 | 61.64 ± 21.44 |  |  |  |  |  |
| rs2030324 (Intronic) | 27683491 | AA/AG | 61 | 40.35 ± 30.63 | 12.81 (0.34, 25.29) | 0.05 | 37 | 52.44 ± 24.80 | 14.2 (2.78, 25.62) | 0.02 |
|   | GG | 39 | 48.79 ± 32.27 | 29 | 63.70 ± 21.84 |  |  |  |  |  |
| rs12273363 (Upstream) | 27701435 | CC/CT | 29 | 34.59 ± 28.61 | 15.81 (2.54, 29.09) | 0.02 | 19 | 48.29 ± 24.16 | 14.28 (1.47, 27.09) | 0.03 |
|   | TT | 71 | 46.39 ± 31.99 | 48 | 59.44 ± 24.00 |  |  |  |  |  |
| rs7931247 (Upstream) | 27703567 | TT/TC | 63 | 39.90 ± 30.23 | 13.2 (0.92, 25.49) | 0.04 | 38 | 51.72 ± 24.86 | 14.94 (3.53, 26.35) | 0.01 |
|   | CC | 39 | 48.79 ± 32.27 | 29 | 63.70 ± 21.84 |  |  |  |  |  |

Table 4 Caption: BDNF Polymorphisms Associated with Response to Antidepressant Treatment with Desipramine

## COMMENTS

We surveyed BDNF sequence variation by studying a 72kb genomic DNA region, which contained the entire BDNF coding and 5 kb of flanking sequences by direct sequencing. Our results provide a detailed description of BDNF sequence variations in Mexican Americans. Among the 130 SNPs that we detected in this study, 83 are novel and only 47 have been reported in NCBI dbSNP database, which has collected 254 BNDF SNPs to date([http://www.ncbi.nlm.nih.gov/projects/SNP](http://www.ncbi.nlm.nih.gov/projects/SNP)). Most of these new polymorphisms (89%) are rare variants with a minor allele below 1% ([Supplementary Table 2](#SD1)). This is not surprising because our study was conducted in large sample of 537 subjects of a specific ethnic group that has not been investigated extensively. The nucleotide diversity in that genomic region is 0.00024. Unexpectedly, we observed 6-fold more genetic diversity in coding regions (0.00101) compared to the intronic region (0.00014) although this estimate is very close to that (0.000238) observed in 6.8 kb BDNF non-coding region^58^. Some studies revealed that nucleotide diversity varies across both genes and functional classes, and gene-to-gene differences in SNP diversity are the most important factors that contribute to the variation^46^. To date, 254 BDNF SNPs (after excluding duplicates) have been reported in NCBI dbSNP database, including 7 in CDS (coding domain sequence) of 792 bp, 224 in intron of 60721 bp and 23 in UTR of 5362 bp. This also yields a higher frequency in coding regions (1 SNP per 113 bp) compared to intronic region (1 SNP per 271 bp). In addition, the sample size and population may also contribute to this frequency difference. Pairwise F_ST_ values revealed a substantial population differentiation in 18 dbSNPs using frequency data available from the NCBI database of 3 ethnic groups (CEU, YRI, CHB+JPT). For example, a high divergence of allele frequency was noted for non-synonymous SNP rs6265 across ethnic populations; minor allele (A allele) frequencies of 0.12, 0.18, 0.00 and 0.48 were found in Mexican American, Caucasian, African, and Asian, respectively. Our findings suggest that the genetic variation in the BDNF gene across different populations may be large and this heterogeneity may contribute to explain controversial findings in association of BNDF with depression from different populations.

It is noteworthy that rare variants in relevant genes in neurodevelopmental pathways have been associated with schizophrenia^59^, further supporting the rare variant/common disease model. The discovery of 83 mostly rare variants in BDNF, a gene that is found to be relevant to several psychiatric disorders, may therefore be of widespread interest.

We report here that 5 SNPs in the BDNF gene were significantly associated with depression, in addition to the non-synonymous SNP rs6265 which we reported previously^36^. Among the 6 SNPs, rs12273539, an intronic variant located 3.4 kb away from rs6265 and near alternative 5′exon VIIIh ([Figure 1](#F1)), showed the most significant association with depression and remained significant after adjustment for multiple testing. Unlike rs6265, rs12273539 showed much less similarity in allele frequency between Mexican Americans and Caucasians with a large Fst value of 0.20. Haplotype analyses revealed a strong LD (D′=1.00) between rs6265 and rs12273539 but they mapped to two LD blocks (blocks 3 and 4 in [Figure 1](#F1)). Two common haplotypes: TGACC that includes BDNF Val66 allele (G) in exon IX in block 3 and CTGT in block 4 near exon VIIIh, were found significantly associated with the increased risk for depression after correcting for multiple testing.

We also found that 8 SNPs were associated with drug response to desipramine treatment in both ITT and completed-treatment samples although the association did not remained significant after adjustment for multiple testing. Among the 8 SNPs, there were one 3′ UTR SNP (rs7124442) in block 1, two newly reported SNPs (5′ UTR SNP rs61888800 in exon Vh and intronic SNP56133711) in block 6, three SNPs (rs2030324 in intron; rs12273363 and rs7931247 in upstream region) in block 7, and one in each of block 3 (rs11030102) and block 4 (rs12273539) ([Figure 1](#F1)). Interestingly, SNP rs12273539, which showed the most significant association with depression status, was also associated with the drug response to desipramine treatment (β=−14.16%, p=0.024) in 8-week completers.

There are several implications to our findings. Firstly, they support the concept that BNDF genetic variants may differ in frequency and/or effect among different ethnic groups. For instance, our data support that in the variant rs6265 (Val66Met), the Val (G allele) carriers are at increased risk for depression, which is consistent with the data of several Caucasian studies ^60–62^. However, several studies in Asians have reported no association between depression and Val66Met^63–65^, or the association of the Met (A allele) variant with susceptibility to depression^66, 67^. Our population differentiation analysis also revealed that Mexican-Americans and Caucasians have a comparable Val66Met allele frequency (Fst=0.01), but they have substantial allele difference when compared to Asians (Fst=0.31). The observed results across ethnic groups may suggest heterogeneity in the BDNF allele frequencies and genetic polymorphisms among populations. Secondly, they suggest that other BDNF genetic variants besides Val66Met may contribute to susceptibility to depression. In this survey, we found 6 BDNF polymorphisms that were associated with depression risk. The strongest association was found to an intronic variant rs12273539. We also identified two haplotypes in different haplotype blocks, one containing rs6265 and the other containing rs12273539, that are significantly associated with depression after multiple testing adjustment (*p* ≤ 0.002). Thirdly, they suggest that the association of BDNF genetic variants with drug response to antidepressant treatment may be medication-specific and do not support a major role of Val66Met variant in antidepressant action. Among the 6 polymorphisms associated with depression in this study, only SNP rs12273539 was found to be associated with HAM-D21 score reduction in desipramine treatment in our sample. However, 7 other SNPs were found to be associated with desipramine treatment by showing ≥14% more average reduction in patients who are homozygous for major allele.

Three studies have recently assessed the association between Val66Met polymorphism and antidepressant response in MDD patients, but only one reported that Met carriers had a better response to 8-week citalopram treatment^63^. Gratacos et al reported a SNP rs908867 and a haplotype (TAT at rs12273363, rs908867 and rs1491850) in 5′ upstream region associated with antidepressant response^68^. Interestingly, in this region, we found 3 SNPs (rs2030324, rs12273363, and rs7931247 in block 7) associated with desipramine treatment although the association of rs908867 with response to antidepressant treatment was not significant in our study. The differential findings could be due to a number of factors such as medication type, outcome assessment, sample size, population substructure, and very importantly, the complexity and rich diversity in the regulation of BDNF multiple transcripts, in the coding and noncoding sequences,, and in the proBDNF and mature BDNF translation product sequences ^7, 69^.

Limitations of this study are related to the sample size is relative small, particularly for analyses of antidepressant treatment response. Although power analyses showed that at a single two-sided significance of 0.05 and allele frequency ≥ 0.15, a sample size of 200 patients can achieve 89% power to detect a moderate effect size of 0.5 that is close to what we observed in desipramine group, the power should be much lower if the genetic effect is medication-specific as our results suggest. Given the small sample size and that the lack of replication sample, the association with antidepressant treatment response should be interpreted with much caution and considered exploratory.

In conclusion, we have identified 83 novel BDNF genetic variants. Our data support the implication of BDNF in the susceptibility to major depressive disorder and in the therapeutic response to antidepressants. To our knowledge, this work is the most comprehensive genetic association study to date to have examined the association between BDNF sequence variation with both depression and antidepressant response. Given that a number of alternative BDNF tanscripts have been found to display complex splicing and expression patterns and the findings in different studies remain inconsistent, further comprehensive studies in larger independent samples are clearly warranted for conclusive results. Moreover, we suggest that deep sequencing of relevant genes in large numbers of patients can reveal substantial numbers of novel variants that may be useful targets for association studies.

## Supplementary Material

## Acknowledgments

**Funding sources:** NIH grants GM61394, RR017365, MH062777, RR000865, RR16996, HG002500, and DK063240, and Institutional Funds from the University of Miami, Department of Psychiatry & Behavioral Sciences

This research was supported in part by National Institutes of Health (NIH) grants GM61394, RR017365, MH062777, RR000865, RR16996, HG002500, and DK063240, and Institutional Funds from the University of Miami, Department of Psychiatry & Behavioral Sciences. No sponsors or funders had any role in the design and conduct of the study; collection, management, analysis, and interpretation of the data; and preparation, review, or approval of the manuscript. The Principal Investigator (J.L.) had full access to all of the data in the study and takes responsibility for the integrity of the data and the accuracy of the data analysis. All the DNA sequencing was completed by the Wellcome Trust Sanger Institute, UK. The authors gratefully acknowledge the cooperation of participants. The authors also thank all the staff from University of California in Los Angels and University of Miami for sample collection, clinical assessments and technical assistance.

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Barbacid M. Neurotrophic factors and their receptors. Curr Opin Cell Biol. 1995;7(2):148–155. doi: 10.1016/0955-0674(95)80022-0.  [DOI](https://doi.org/10.1016/0955-0674(95)80022-0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7612265/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Opin%20Cell%20Biol&title=Neurotrophic%20factors%20and%20their%20receptors&author=M%20Barbacid&volume=7&issue=2&publication_year=1995&pages=148-155&pmid=7612265&doi=10.1016/0955-0674(95)80022-0&)

2. Levi-Montalcini R. The nerve growth factor 35 years later. Science. 1987;237(4819):1154–1162. doi: 10.1126/science.3306916.  [DOI](https://doi.org/10.1126/science.3306916) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/3306916/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=The%20nerve%20growth%20factor%2035%20years%20later&author=R%20Levi-Montalcini&volume=237&issue=4819&publication_year=1987&pages=1154-1162&pmid=3306916&doi=10.1126/science.3306916&)

3. Segal RA, Greenberg ME. Intracellular signaling pathways activated by neurotrophic factors. Annu Rev Neurosci. 1996;19:463–489. doi: 10.1146/annurev.ne.19.030196.002335.  [DOI](https://doi.org/10.1146/annurev.ne.19.030196.002335) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8833451/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Annu%20Rev%20Neurosci&title=Intracellular%20signaling%20pathways%20activated%20by%20neurotrophic%20factors&author=RA%20Segal&author=ME%20Greenberg&volume=19&publication_year=1996&pages=463-489&pmid=8833451&doi=10.1146/annurev.ne.19.030196.002335&)

4. Barde YA, Edgar D, Thoenen H. Purification of a new neurotrophic factor from mammalian brain. Embo J. 1982;1(5):549–553. doi: 10.1002/j.1460-2075.1982.tb01207.x.  [DOI](https://doi.org/10.1002/j.1460-2075.1982.tb01207.x) | [PMC free article](/articles/PMC553086/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7188352/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Embo%20J&title=Purification%20of%20a%20new%20neurotrophic%20factor%20from%20mammalian%20brain&author=YA%20Barde&author=D%20Edgar&author=H%20Thoenen&volume=1&issue=5&publication_year=1982&pages=549-553&pmid=7188352&doi=10.1002/j.1460-2075.1982.tb01207.x&)

5. Hofer MM, Barde YA. Brain-derived neurotrophic factor prevents neuronal death in vivo. Nature. 1988;331(6153):261–262. doi: 10.1038/331261a0.  [DOI](https://doi.org/10.1038/331261a0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/3336438/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Brain-derived%20neurotrophic%20factor%20prevents%20neuronal%20death%20in%20vivo&author=MM%20Hofer&author=YA%20Barde&volume=331&issue=6153&publication_year=1988&pages=261-262&pmid=3336438&doi=10.1038/331261a0&)

6. Maisonpierre PC, Le Beau MM, Espinosa R, 3rd, Ip NY, Belluscio L, de la Monte SM, Squinto S, Furth ME, Yancopoulos GD. Human and rat brain-derived neurotrophic factor and neurotrophin-3: gene structures, distributions, and chromosomal localizations. Genomics. 1991;10(3):558–568. doi: 10.1016/0888-7543(91)90436-i.  [DOI](https://doi.org/10.1016/0888-7543(91)90436-i) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/1889806/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genomics&title=Human%20and%20rat%20brain-derived%20neurotrophic%20factor%20and%20neurotrophin-3:%20gene%20structures,%20distributions,%20and%20chromosomal%20localizations&author=PC%20Maisonpierre&author=MM%20Le%20Beau&author=R%20Espinosa&author=NY%20Ip&author=L%20Belluscio&volume=10&issue=3&publication_year=1991&pages=558-568&pmid=1889806&doi=10.1016/0888-7543(91)90436-i&)

7. Pruunsild P, Kazantseva A, Aid T, Palm K, Timmusk T. Dissecting the human BDNF locus: bidirectional transcription, complex splicing, and multiple promoters. Genomics. 2007;90(3):397–406. doi: 10.1016/j.ygeno.2007.05.004.  [DOI](https://doi.org/10.1016/j.ygeno.2007.05.004) | [PMC free article](/articles/PMC2568880/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17629449/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genomics&title=Dissecting%20the%20human%20BDNF%20locus:%20bidirectional%20transcription,%20complex%20splicing,%20and%20multiple%20promoters&author=P%20Pruunsild&author=A%20Kazantseva&author=T%20Aid&author=K%20Palm&author=T%20Timmusk&volume=90&issue=3&publication_year=2007&pages=397-406&pmid=17629449&doi=10.1016/j.ygeno.2007.05.004&)

8. Connor B, Dragunow M. The role of neuronal growth factors in neurodegenerative disorders of the human brain. Brain Res Brain Res Rev. 1998;27(1):1–39. doi: 10.1016/s0165-0173(98)00004-6.  [DOI](https://doi.org/10.1016/s0165-0173(98)00004-6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9639663/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Brain%20Res%20Brain%20Res%20Rev&title=The%20role%20of%20neuronal%20growth%20factors%20in%20neurodegenerative%20disorders%20of%20the%20human%20brain&author=B%20Connor&author=M%20Dragunow&volume=27&issue=1&publication_year=1998&pages=1-39&pmid=9639663&doi=10.1016/s0165-0173(98)00004-6&)

9. Murer MG, Yan Q, Raisman-Vozari R. Brain-derived neurotrophic factor in the control human brain, and in Alzheimer’s disease and Parkinson’s disease. Prog Neurobiol. 2001;63(1):71–124. doi: 10.1016/s0301-0082(00)00014-9.  [DOI](https://doi.org/10.1016/s0301-0082(00)00014-9) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11040419/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Prog%20Neurobiol&title=Brain-derived%20neurotrophic%20factor%20in%20the%20control%20human%20brain,%20and%20in%20Alzheimer%E2%80%99s%20disease%20and%20Parkinson%E2%80%99s%20disease&author=MG%20Murer&author=Q%20Yan&author=R%20Raisman-Vozari&volume=63&issue=1&publication_year=2001&pages=71-124&pmid=11040419&doi=10.1016/s0301-0082(00)00014-9&)

10. Bramham CR, Messaoudi E. BDNF function in adult synaptic plasticity: the synaptic consolidation hypothesis. Prog Neurobiol. 2005;76(2):99–125. doi: 10.1016/j.pneurobio.2005.06.003.  [DOI](https://doi.org/10.1016/j.pneurobio.2005.06.003) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16099088/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Prog%20Neurobiol&title=BDNF%20function%20in%20adult%20synaptic%20plasticity:%20the%20synaptic%20consolidation%20hypothesis&author=CR%20Bramham&author=E%20Messaoudi&volume=76&issue=2&publication_year=2005&pages=99-125&pmid=16099088&doi=10.1016/j.pneurobio.2005.06.003&)

11. Poo MM. Neurotrophins as synaptic modulators. Nat Rev Neurosci. 2001;2(1):24–32. doi: 10.1038/35049004.  [DOI](https://doi.org/10.1038/35049004) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11253356/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Neurosci&title=Neurotrophins%20as%20synaptic%20modulators&author=MM%20Poo&volume=2&issue=1&publication_year=2001&pages=24-32&pmid=11253356&doi=10.1038/35049004&)

12. Schinder AF, Poo M. The neurotrophin hypothesis for synaptic plasticity. Trends Neurosci. 2000;23(12):639–645. doi: 10.1016/s0166-2236(00)01672-6.  [DOI](https://doi.org/10.1016/s0166-2236(00)01672-6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11137155/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Trends%20Neurosci&title=The%20neurotrophin%20hypothesis%20for%20synaptic%20plasticity&author=AF%20Schinder&author=M%20Poo&volume=23&issue=12&publication_year=2000&pages=639-645&pmid=11137155&doi=10.1016/s0166-2236(00)01672-6&)

13. Tyler WJ, Alonso M, Bramham CR, Pozzo-Miller LD. From acquisition to consolidation: on the role of brain-derived neurotrophic factor signaling in hippocampal-dependent learning. Learn Mem. 2002;9(5):224–237. doi: 10.1101/lm.51202.  [DOI](https://doi.org/10.1101/lm.51202) | [PMC free article](/articles/PMC2806479/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12359832/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Learn%20Mem&title=From%20acquisition%20to%20consolidation:%20on%20the%20role%20of%20brain-derived%20neurotrophic%20factor%20signaling%20in%20hippocampal-dependent%20learning&author=WJ%20Tyler&author=M%20Alonso&author=CR%20Bramham&author=LD%20Pozzo-Miller&volume=9&issue=5&publication_year=2002&pages=224-237&pmid=12359832&doi=10.1101/lm.51202&)

14. Pang PT, Lu B. Regulation of late-phase LTP and long-term memory in normal and aging hippocampus: role of secreted proteins tPA and BDNF. Ageing Res Rev. 2004;3(4):407–430. doi: 10.1016/j.arr.2004.07.002.  [DOI](https://doi.org/10.1016/j.arr.2004.07.002) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15541709/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ageing%20Res%20Rev&title=Regulation%20of%20late-phase%20LTP%20and%20long-term%20memory%20in%20normal%20and%20aging%20hippocampus:%20role%20of%20secreted%20proteins%20tPA%20and%20BDNF&author=PT%20Pang&author=B%20Lu&volume=3&issue=4&publication_year=2004&pages=407-430&pmid=15541709&doi=10.1016/j.arr.2004.07.002&)

15. Rattiner LM, Davis M, Ressler KJ. Brain-derived neurotrophic factor in amygdala-dependent learning. Neuroscientist. 2005;11(4):323–333. doi: 10.1177/1073858404272255.  [DOI](https://doi.org/10.1177/1073858404272255) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16061519/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neuroscientist&title=Brain-derived%20neurotrophic%20factor%20in%20amygdala-dependent%20learning&author=LM%20Rattiner&author=M%20Davis&author=KJ%20Ressler&volume=11&issue=4&publication_year=2005&pages=323-333&pmid=16061519&doi=10.1177/1073858404272255&)

16. Bekinschtein P, Cammarota M, Izquierdo I, Medina JH. Reviews: BDNF and memory formation and storage. Neuroscientist. 2008;14(2):147–156. doi: 10.1177/1073858407305850.  [DOI](https://doi.org/10.1177/1073858407305850) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17911219/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neuroscientist&title=Reviews:%20BDNF%20and%20memory%20formation%20and%20storage&author=P%20Bekinschtein&author=M%20Cammarota&author=I%20Izquierdo&author=JH%20Medina&volume=14&issue=2&publication_year=2008&pages=147-156&pmid=17911219&doi=10.1177/1073858407305850&)

17. Bekinschtein P, Cammarota M, Katche C, Slipczuk L, Rossato JI, Goldin A, Izquierdo I, Medina JH. BDNF is essential to promote persistence of long-term memory storage. Proc Natl Acad Sci U S A. 2008;105(7):2711–2716. doi: 10.1073/pnas.0711863105.  [DOI](https://doi.org/10.1073/pnas.0711863105) | [PMC free article](/articles/PMC2268201/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18263738/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&title=BDNF%20is%20essential%20to%20promote%20persistence%20of%20long-term%20memory%20storage&author=P%20Bekinschtein&author=M%20Cammarota&author=C%20Katche&author=L%20Slipczuk&author=JI%20Rossato&volume=105&issue=7&publication_year=2008&pages=2711-2716&pmid=18263738&doi=10.1073/pnas.0711863105&)

18. Yamada K, Nabeshima T. Brain-derived neurotrophic factor/TrkB signaling in memory processes. J Pharmacol Sci. 2003;91(4):267–270. doi: 10.1254/jphs.91.267.  [DOI](https://doi.org/10.1254/jphs.91.267) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12719654/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharmacol%20Sci&title=Brain-derived%20neurotrophic%20factor/TrkB%20signaling%20in%20memory%20processes&author=K%20Yamada&author=T%20Nabeshima&volume=91&issue=4&publication_year=2003&pages=267-270&pmid=12719654&doi=10.1254/jphs.91.267&)

19. He XP, Kotloski R, Nef S, Luikart BW, Parada LF, McNamara JO. Conditional deletion of TrkB but not BDNF prevents epileptogenesis in the kindling model. Neuron. 2004;43(1):31–42. doi: 10.1016/j.neuron.2004.06.019.  [DOI](https://doi.org/10.1016/j.neuron.2004.06.019) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15233915/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neuron&title=Conditional%20deletion%20of%20TrkB%20but%20not%20BDNF%20prevents%20epileptogenesis%20in%20the%20kindling%20model&author=XP%20He&author=R%20Kotloski&author=S%20Nef&author=BW%20Luikart&author=LF%20Parada&volume=43&issue=1&publication_year=2004&pages=31-42&pmid=15233915&doi=10.1016/j.neuron.2004.06.019&)

20. Croll SD, Suri C, Compton DL, Simmons MV, Yancopoulos GD, Lindsay RM, Wiegand SJ, Rudge JS, Scharfman HE. Brain-derived neurotrophic factor transgenic mice exhibit passive avoidance deficits, increased seizure severity and in vitro hyperexcitability in the hippocampus and entorhinal cortex. Neuroscience. 1999;93(4):1491–1506. doi: 10.1016/s0306-4522(99)00296-1.  [DOI](https://doi.org/10.1016/s0306-4522(99)00296-1) | [PMC free article](/articles/PMC2504500/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10501474/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neuroscience&title=Brain-derived%20neurotrophic%20factor%20transgenic%20mice%20exhibit%20passive%20avoidance%20deficits,%20increased%20seizure%20severity%20and%20in%20vitro%20hyperexcitability%20in%20the%20hippocampus%20and%20entorhinal%20cortex&author=SD%20Croll&author=C%20Suri&author=DL%20Compton&author=MV%20Simmons&author=GD%20Yancopoulos&volume=93&issue=4&publication_year=1999&pages=1491-1506&pmid=10501474&doi=10.1016/s0306-4522(99)00296-1&)

21. Kokaia M, Ernfors P, Kokaia Z, Elmer E, Jaenisch R, Lindvall O. Suppressed epileptogenesis in BDNF mutant mice. Exp Neurol. 1995;133(2):215–224. doi: 10.1006/exnr.1995.1024.  [DOI](https://doi.org/10.1006/exnr.1995.1024) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7649227/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Exp%20Neurol&title=Suppressed%20epileptogenesis%20in%20BDNF%20mutant%20mice&author=M%20Kokaia&author=P%20Ernfors&author=Z%20Kokaia&author=E%20Elmer&author=R%20Jaenisch&volume=133&issue=2&publication_year=1995&pages=215-224&pmid=7649227&doi=10.1006/exnr.1995.1024&)

22. Lahteinen S, Pitkanen A, Koponen E, Saarelainen T, Castren E. Exacerbated status epilepticus and acute cell loss, but no changes in epileptogenesis, in mice with increased brain-derived neurotrophic factor signaling. Neuroscience. 2003;122(4):1081–1092. doi: 10.1016/j.neuroscience.2003.08.037.  [DOI](https://doi.org/10.1016/j.neuroscience.2003.08.037) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14643774/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neuroscience&title=Exacerbated%20status%20epilepticus%20and%20acute%20cell%20loss,%20but%20no%20changes%20in%20epileptogenesis,%20in%20mice%20with%20increased%20brain-derived%20neurotrophic%20factor%20signaling&author=S%20Lahteinen&author=A%20Pitkanen&author=E%20Koponen&author=T%20Saarelainen&author=E%20Castren&volume=122&issue=4&publication_year=2003&pages=1081-1092&pmid=14643774&doi=10.1016/j.neuroscience.2003.08.037&)

23. Weese-Mayer DE, Bolk S, Silvestri JM, Chakravarti A. Idiopathic congenital central hypoventilation syndrome: evaluation of brain-derived neurotrophic factor genomic DNA sequence variation. Am J Med Genet. 2002;107(4):306–310. doi: 10.1002/ajmg.10133.  [DOI](https://doi.org/10.1002/ajmg.10133) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11840487/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Med%20Genet&title=Idiopathic%20congenital%20central%20hypoventilation%20syndrome:%20evaluation%20of%20brain-derived%20neurotrophic%20factor%20genomic%20DNA%20sequence%20variation&author=DE%20Weese-Mayer&author=S%20Bolk&author=JM%20Silvestri&author=A%20Chakravarti&volume=107&issue=4&publication_year=2002&pages=306-310&pmid=11840487&doi=10.1002/ajmg.10133&)

24. Egan MF, Kojima M, Callicott JH, Goldberg TE, Kolachana BS, Bertolino A, Zaitsev E, Gold B, Goldman D, Dean M, Lu B, Weinberger DR. The BDNF val66met polymorphism affects activity-dependent secretion of BDNF and human memory and hippocampal function. Cell. 2003;112(2):257–269. doi: 10.1016/s0092-8674(03)00035-7.  [DOI](https://doi.org/10.1016/s0092-8674(03)00035-7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12553913/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell&title=The%20BDNF%20val66met%20polymorphism%20affects%20activity-dependent%20secretion%20of%20BDNF%20and%20human%20memory%20and%20hippocampal%20function&author=MF%20Egan&author=M%20Kojima&author=JH%20Callicott&author=TE%20Goldberg&author=BS%20Kolachana&volume=112&issue=2&publication_year=2003&pages=257-269&pmid=12553913&doi=10.1016/s0092-8674(03)00035-7&)

25. Licinio J, Wong ML. Brain-derived neurotrophic factor (BDNF) in stress and affective disorders. Molecular psychiatry. 2002;7(6):519. doi: 10.1038/sj.mp.4001211.  [DOI](https://doi.org/10.1038/sj.mp.4001211) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12140770/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Molecular%20psychiatry&title=Brain-derived%20neurotrophic%20factor%20(BDNF)%20in%20stress%20and%20affective%20disorders&author=J%20Licinio&author=ML%20Wong&volume=7&issue=6&publication_year=2002&pages=519&pmid=12140770&doi=10.1038/sj.mp.4001211&)

26. Hall D, Dhilla A, Charalambous A, Gogos JA, Karayiorgou M. Sequence variants of the brain-derived neurotrophic factor (BDNF) gene are strongly associated with obsessive-compulsive disorder. American journal of human genetics. 2003;73(2):370–376. doi: 10.1086/377003.  [DOI](https://doi.org/10.1086/377003) | [PMC free article](/articles/PMC1180373/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12836135/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=American%20journal%20of%20human%20genetics&title=Sequence%20variants%20of%20the%20brain-derived%20neurotrophic%20factor%20(BDNF)%20gene%20are%20strongly%20associated%20with%20obsessive-compulsive%20disorder&author=D%20Hall&author=A%20Dhilla&author=A%20Charalambous&author=JA%20Gogos&author=M%20Karayiorgou&volume=73&issue=2&publication_year=2003&pages=370-376&pmid=12836135&doi=10.1086/377003&)

27. Ribases M, Gratacos M, Armengol L, de Cid R, Badia A, Jimenez L, Solano R, Vallejo J, Fernandez F, Estivill X. Met66 in the brain-derived neurotrophic factor (BDNF) precursor is associated with anorexia nervosa restrictive type. Molecular psychiatry. 2003;8(8):745–751. doi: 10.1038/sj.mp.4001281.  [DOI](https://doi.org/10.1038/sj.mp.4001281) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12888803/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Molecular%20psychiatry&title=Met66%20in%20the%20brain-derived%20neurotrophic%20factor%20(BDNF)%20precursor%20is%20associated%20with%20anorexia%20nervosa%20restrictive%20type&author=M%20Ribases&author=M%20Gratacos&author=L%20Armengol&author=R%20de%20Cid&author=A%20Badia&volume=8&issue=8&publication_year=2003&pages=745-751&pmid=12888803&doi=10.1038/sj.mp.4001281&)

28. Ribases M, Gratacos M, Fernandez-Aranda F, Bellodi L, Boni C, Anderluh M, Cavallini MC, Cellini E, Di Bella D, Erzegovesi S, Foulon C, Gabrovsek M, Gorwood P, Hebebrand J, Hinney A, Holliday J, Hu X, Karwautz A, Kipman A, Komel R, Nacmias B, Remschmidt H, Ricca V, Sorbi S, Wagner G, Treasure J, Collier DA, Estivill X. Association of BDNF with anorexia, bulimia and age of onset of weight loss in six European populations. Hum Mol Genet. 2004;13(12):1205–1212. doi: 10.1093/hmg/ddh137.  [DOI](https://doi.org/10.1093/hmg/ddh137) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15115760/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mol%20Genet&title=Association%20of%20BDNF%20with%20anorexia,%20bulimia%20and%20age%20of%20onset%20of%20weight%20loss%20in%20six%20European%20populations&author=M%20Ribases&author=M%20Gratacos&author=F%20Fernandez-Aranda&author=L%20Bellodi&author=C%20Boni&volume=13&issue=12&publication_year=2004&pages=1205-1212&pmid=15115760&doi=10.1093/hmg/ddh137&)

29. Sklar P, Gabriel SB, McInnis MG, Bennett P, Lim YM, Tsan G, Schaffner S, Kirov G, Jones I, Owen M, Craddock N, DePaulo JR, Lander ES. Family-based association study of 76 candidate genes in bipolar disorder: BDNF is a potential risk locus. Brain-derived neutrophic factor. Molecular psychiatry. 2002;7(6):579–593. doi: 10.1038/sj.mp.4001058.  [DOI](https://doi.org/10.1038/sj.mp.4001058) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12140781/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Molecular%20psychiatry&title=Family-based%20association%20study%20of%2076%20candidate%20genes%20in%20bipolar%20disorder:%20BDNF%20is%20a%20potential%20risk%20locus.%20Brain-derived%20neutrophic%20factor&author=P%20Sklar&author=SB%20Gabriel&author=MG%20McInnis&author=P%20Bennett&author=YM%20Lim&volume=7&issue=6&publication_year=2002&pages=579-593&pmid=12140781&doi=10.1038/sj.mp.4001058&)

30. Geller B, Badner JA, Tillman R, Christian SL, Bolhofner K, Cook EH., Jr Linkage disequilibrium of the brain-derived neurotrophic factor Val66Met polymorphism in children with a prepubertal and early adolescent bipolar disorder phenotype. Am J Psychiatry. 2004;161(9):1698–1700. doi: 10.1176/appi.ajp.161.9.1698.  [DOI](https://doi.org/10.1176/appi.ajp.161.9.1698) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15337662/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Psychiatry&title=Linkage%20disequilibrium%20of%20the%20brain-derived%20neurotrophic%20factor%20Val66Met%20polymorphism%20in%20children%20with%20a%20prepubertal%20and%20early%20adolescent%20bipolar%20disorder%20phenotype&author=B%20Geller&author=JA%20Badner&author=R%20Tillman&author=SL%20Christian&author=K%20Bolhofner&volume=161&issue=9&publication_year=2004&pages=1698-1700&pmid=15337662&doi=10.1176/appi.ajp.161.9.1698&)

31. Lohoff FW, Sander T, Ferraro TN, Dahl JP, Gallinat J, Berrettini WH. Confirmation of association between the Val66Met polymorphism in the brain-derived neurotrophic factor (BDNF) gene and bipolar I disorder. Am J Med Genet B Neuropsychiatr Genet. 2005;139(1):51–53. doi: 10.1002/ajmg.b.30215.  [DOI](https://doi.org/10.1002/ajmg.b.30215) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16152572/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Med%20Genet%20B%20Neuropsychiatr%20Genet&title=Confirmation%20of%20association%20between%20the%20Val66Met%20polymorphism%20in%20the%20brain-derived%20neurotrophic%20factor%20(BDNF)%20gene%20and%20bipolar%20I%20disorder&author=FW%20Lohoff&author=T%20Sander&author=TN%20Ferraro&author=JP%20Dahl&author=J%20Gallinat&volume=139&issue=1&publication_year=2005&pages=51-53&pmid=16152572&doi=10.1002/ajmg.b.30215&)

32. Neves-Pereira M, Mundo E, Muglia P, King N, Macciardi F, Kennedy JL. The brain-derived neurotrophic factor gene confers susceptibility to bipolar disorder: evidence from a family-based association study. American journal of human genetics. 2002;71(3):651–655. doi: 10.1086/342288.  [DOI](https://doi.org/10.1086/342288) | [PMC free article](/articles/PMC379201/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12161822/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=American%20journal%20of%20human%20genetics&title=The%20brain-derived%20neurotrophic%20factor%20gene%20confers%20susceptibility%20to%20bipolar%20disorder:%20evidence%20from%20a%20family-based%20association%20study&author=M%20Neves-Pereira&author=E%20Mundo&author=P%20Muglia&author=N%20King&author=F%20Macciardi&volume=71&issue=3&publication_year=2002&pages=651-655&pmid=12161822&doi=10.1086/342288&)

33. Okada T, Hashimoto R, Numakawa T, Iijima Y, Kosuga A, Tatsumi M, Kamijima K, Kato T, Kunugi H. A complex polymorphic region in the brain-derived neurotrophic factor (BDNF) gene confers susceptibility to bipolar disorder and affects transcriptional activity. Molecular psychiatry. 2006;11(7):695–703. doi: 10.1038/sj.mp.4001822.  [DOI](https://doi.org/10.1038/sj.mp.4001822) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16568151/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Molecular%20psychiatry&title=A%20complex%20polymorphic%20region%20in%20the%20brain-derived%20neurotrophic%20factor%20(BDNF)%20gene%20confers%20susceptibility%20to%20bipolar%20disorder%20and%20affects%20transcriptional%20activity&author=T%20Okada&author=R%20Hashimoto&author=T%20Numakawa&author=Y%20Iijima&author=A%20Kosuga&volume=11&issue=7&publication_year=2006&pages=695-703&pmid=16568151&doi=10.1038/sj.mp.4001822&)

34. Tramontina J, Frey BN, Andreazza AC, Zandona M, Santin A, Kapczinski F. Val66met polymorphism and serum brain-derived neurotrophic factor levels in bipolar disorder. Molecular psychiatry. 2007;12(3):230–231. doi: 10.1038/sj.mp.4001941.  [DOI](https://doi.org/10.1038/sj.mp.4001941) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17325716/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Molecular%20psychiatry&title=Val66met%20polymorphism%20and%20serum%20brain-derived%20neurotrophic%20factor%20levels%20in%20bipolar%20disorder&author=J%20Tramontina&author=BN%20Frey&author=AC%20Andreazza&author=M%20Zandona&author=A%20Santin&volume=12&issue=3&publication_year=2007&pages=230-231&pmid=17325716&doi=10.1038/sj.mp.4001941&)

35. Neves-Pereira M, Cheung JK, Pasdar A, Zhang F, Breen G, Yates P, Sinclair M, Crombie C, Walker N, St Clair DM. BDNF gene is a risk factor for schizophrenia in a Scottish population. Molecular psychiatry. 2005;10(2):208–212. doi: 10.1038/sj.mp.4001575.  [DOI](https://doi.org/10.1038/sj.mp.4001575) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15630410/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Molecular%20psychiatry&title=BDNF%20gene%20is%20a%20risk%20factor%20for%20schizophrenia%20in%20a%20Scottish%20population&author=M%20Neves-Pereira&author=JK%20Cheung&author=A%20Pasdar&author=F%20Zhang&author=G%20Breen&volume=10&issue=2&publication_year=2005&pages=208-212&pmid=15630410&doi=10.1038/sj.mp.4001575&)

36. Ribeiro L, Busnello JV, Cantor RM, Whelan F, Whittaker P, Deloukas P, Wong ML, Licinio J. The brain-derived neurotrophic factor rs6265 (Val66Met) polymorphism and depression in Mexican-Americans. Neuroreport. 2007;18(12):1291–1293. doi: 10.1097/WNR.0b013e328273bcb0.  [DOI](https://doi.org/10.1097/WNR.0b013e328273bcb0) | [PMC free article](/articles/PMC2686836/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17632285/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neuroreport&title=The%20brain-derived%20neurotrophic%20factor%20rs6265%20(Val66Met)%20polymorphism%20and%20depression%20in%20Mexican-Americans&author=L%20Ribeiro&author=JV%20Busnello&author=RM%20Cantor&author=F%20Whelan&author=P%20Whittaker&volume=18&issue=12&publication_year=2007&pages=1291-1293&pmid=17632285&doi=10.1097/WNR.0b013e328273bcb0&)

37. Schumacher J, Jamra RA, Becker T, Ohlraun S, Klopp N, Binder EB, Schulze TG, Deschner M, Schmal C, Hofels S, Zobel A, Illig T, Propping P, Holsboer F, Rietschel M, Nothen MM, Cichon S. Evidence for a relationship between genetic variants at the brain-derived neurotrophic factor (BDNF) locus and major depression. Biol Psychiatry. 2005;58(4):307–314. doi: 10.1016/j.biopsych.2005.04.006.  [DOI](https://doi.org/10.1016/j.biopsych.2005.04.006) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16005437/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biol%20Psychiatry&title=Evidence%20for%20a%20relationship%20between%20genetic%20variants%20at%20the%20brain-derived%20neurotrophic%20factor%20(BDNF)%20locus%20and%20major%20depression&author=J%20Schumacher&author=RA%20Jamra&author=T%20Becker&author=S%20Ohlraun&author=N%20Klopp&volume=58&issue=4&publication_year=2005&pages=307-314&pmid=16005437&doi=10.1016/j.biopsych.2005.04.006&)

38. Bagnoli S, Nacmias B, Tedde A, Guarnieri BM, Cellini E, Petruzzi C, Bartoli A, Ortenzi L, Sorbi S. Brain-derived neurotrophic factor genetic variants are not susceptibility factors to Alzheimer’s disease in Italy. Ann Neurol. 2004;55(3):447–448. doi: 10.1002/ana.10842.  [DOI](https://doi.org/10.1002/ana.10842) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14991827/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Neurol&title=Brain-derived%20neurotrophic%20factor%20genetic%20variants%20are%20not%20susceptibility%20factors%20to%20Alzheimer%E2%80%99s%20disease%20in%20Italy&author=S%20Bagnoli&author=B%20Nacmias&author=A%20Tedde&author=BM%20Guarnieri&author=E%20Cellini&volume=55&issue=3&publication_year=2004&pages=447-448&pmid=14991827&doi=10.1002/ana.10842&)

39. Riemenschneider M, Schwarz S, Wagenpfeil S, Diehl J, Muller U, Forstl H, Kurz A. A polymorphism of the brain-derived neurotrophic factor (BDNF) is associated with Alzheimer’s disease in patients lacking the Apolipoprotein E epsilon4 allele. Molecular psychiatry. 2002;7(7):782–785. doi: 10.1038/sj.mp.4001073.  [DOI](https://doi.org/10.1038/sj.mp.4001073) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12192623/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Molecular%20psychiatry&title=A%20polymorphism%20of%20the%20brain-derived%20neurotrophic%20factor%20(BDNF)%20is%20associated%20with%20Alzheimer%E2%80%99s%20disease%20in%20patients%20lacking%20the%20Apolipoprotein%20E%20epsilon4%20allele&author=M%20Riemenschneider&author=S%20Schwarz&author=S%20Wagenpfeil&author=J%20Diehl&author=U%20Muller&volume=7&issue=7&publication_year=2002&pages=782-785&pmid=12192623&doi=10.1038/sj.mp.4001073&)

40. Kunugi H, Ueki A, Otsuka M, Isse K, Hirasawa H, Kato N, Nabika T, Kobayashi S, Nanko S. A novel polymorphism of the brain-derived neurotrophic factor (BDNF) gene associated with late-onset Alzheimer’s disease. Molecular psychiatry. 2001;6(1):83–86. doi: 10.1038/sj.mp.4000792.  [DOI](https://doi.org/10.1038/sj.mp.4000792) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11244490/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Molecular%20psychiatry&title=A%20novel%20polymorphism%20of%20the%20brain-derived%20neurotrophic%20factor%20(BDNF)%20gene%20associated%20with%20late-onset%20Alzheimer%E2%80%99s%20disease&author=H%20Kunugi&author=A%20Ueki&author=M%20Otsuka&author=K%20Isse&author=H%20Hirasawa&volume=6&issue=1&publication_year=2001&pages=83-86&pmid=11244490&doi=10.1038/sj.mp.4000792&)

41. Campbell H, Manolio T. Commentary: rare alleles, modest genetic effects and the need for collaboration. Int J Epidemiol. 2007;36(2):445–448. doi: 10.1093/ije/dym055.  [DOI](https://doi.org/10.1093/ije/dym055) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17470492/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Epidemiol&title=Commentary:%20rare%20alleles,%20modest%20genetic%20effects%20and%20the%20need%20for%20collaboration&author=H%20Campbell&author=T%20Manolio&volume=36&issue=2&publication_year=2007&pages=445-448&pmid=17470492&doi=10.1093/ije/dym055&)

42. Jakobsson M, Scholz SW, Scheet P, Gibbs JR, VanLiere JM, Fung HC, Szpiech ZA, Degnan JH, Wang K, Guerreiro R, Bras JM, Schymick JC, Hernandez DG, Traynor BJ, Simon-Sanchez J, Matarin M, Britton A, van de Leemput J, Rafferty I, Bucan M, Cann HM, Hardy JA, Rosenberg NA, Singleton AB. Genotype, haplotype and copy-number variation in worldwide human populations. Nature. 2008;451(7181):998–1003. doi: 10.1038/nature06742.  [DOI](https://doi.org/10.1038/nature06742) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18288195/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Genotype,%20haplotype%20and%20copy-number%20variation%20in%20worldwide%20human%20populations&author=M%20Jakobsson&author=SW%20Scholz&author=P%20Scheet&author=JR%20Gibbs&author=JM%20VanLiere&volume=451&issue=7181&publication_year=2008&pages=998-1003&pmid=18288195&doi=10.1038/nature06742&)

43. Redon R, Ishikawa S, Fitch KR, Feuk L, Perry GH, Andrews TD, Fiegler H, Shapero MH, Carson AR, Chen W, Cho EK, Dallaire S, Freeman JL, Gonzalez JR, Gratacos M, Huang J, Kalaitzopoulos D, Komura D, MacDonald JR, Marshall CR, Mei R, Montgomery L, Nishimura K, Okamura K, Shen F, Somerville MJ, Tchinda J, Valsesia A, Woodwark C, Yang F, Zhang J, Zerjal T, Zhang J, Armengol L, Conrad DF, Estivill X, Tyler-Smith C, Carter NP, Aburatani H, Lee C, Jones KW, Scherer SW, Hurles ME. Global variation in copy number in the human genome. Nature. 2006;444(7118):444–454. doi: 10.1038/nature05329.  [DOI](https://doi.org/10.1038/nature05329) | [PMC free article](/articles/PMC2669898/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17122850/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Global%20variation%20in%20copy%20number%20in%20the%20human%20genome&author=R%20Redon&author=S%20Ishikawa&author=KR%20Fitch&author=L%20Feuk&author=GH%20Perry&volume=444&issue=7118&publication_year=2006&pages=444-454&pmid=17122850&doi=10.1038/nature05329&)

44. Wong ML, Whelan F, Deloukas P, Whittaker P, Delgado M, Cantor RM, McCann SM, Licinio J. Phosphodiesterase genes are associated with susceptibility to major depression and antidepressant treatment response. Proc Natl Acad Sci U S A. 2006;103(41):15124–15129. doi: 10.1073/pnas.0602795103.  [DOI](https://doi.org/10.1073/pnas.0602795103) | [PMC free article](/articles/PMC1581426/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17008408/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&title=Phosphodiesterase%20genes%20are%20associated%20with%20susceptibility%20to%20major%20depression%20and%20antidepressant%20treatment%20response&author=ML%20Wong&author=F%20Whelan&author=P%20Deloukas&author=P%20Whittaker&author=M%20Delgado&volume=103&issue=41&publication_year=2006&pages=15124-15129&pmid=17008408&doi=10.1073/pnas.0602795103&)

45. Licinio J, O’Kirwan F, Irizarry K, Merriman B, Thakur S, Jepson R, Lake S, Tantisira KG, Weiss ST, Wong ML. Association of a corticotropin-releasing hormone receptor 1 haplotype and antidepressant treatment response in Mexican-Americans. Molecular psychiatry. 2004;9(12):1075–1082. doi: 10.1038/sj.mp.4001587.  [DOI](https://doi.org/10.1038/sj.mp.4001587) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15365580/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Molecular%20psychiatry&title=Association%20of%20a%20corticotropin-releasing%20hormone%20receptor%201%20haplotype%20and%20antidepressant%20treatment%20response%20in%20Mexican-Americans&author=J%20Licinio&author=F%20O%E2%80%99Kirwan&author=K%20Irizarry&author=B%20Merriman&author=S%20Thakur&volume=9&issue=12&publication_year=2004&pages=1075-1082&pmid=15365580&doi=10.1038/sj.mp.4001587&)

46. Halushka MK, Fan JB, Bentley K, Hsie L, Shen N, Weder A, Cooper R, Lipshutz R, Chakravarti A. Patterns of single-nucleotide polymorphisms in candidate genes for blood-pressure homeostasis. Nature genetics. 1999;22(3):239–247. doi: 10.1038/10297.  [DOI](https://doi.org/10.1038/10297) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10391210/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature%20genetics&title=Patterns%20of%20single-nucleotide%20polymorphisms%20in%20candidate%20genes%20for%20blood-pressure%20homeostasis&author=MK%20Halushka&author=JB%20Fan&author=K%20Bentley&author=L%20Hsie&author=N%20Shen&volume=22&issue=3&publication_year=1999&pages=239-247&pmid=10391210&doi=10.1038/10297&)

47. Li WH. Molecular Evolution. Sunderland, MA: Sinauer Associates; 1997.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Molecular%20Evolution&author=WH%20Li&publication_year=1997&)

48. Weir BS. Genetic Data Analysis II. Sunderland, MA: Sinauer; 1996.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Genetic%20Data%20Analysis%20II&author=BS%20Weir&publication_year=1996&)

49. Weir BS, Cockerham CC. Estinmating F-statistics for the analysis of population structure. Evoluation. 1984;38:1358–1370. doi: 10.1111/j.1558-5646.1984.tb05657.x.  [DOI](https://doi.org/10.1111/j.1558-5646.1984.tb05657.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28563791/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Evoluation&title=Estinmating%20F-statistics%20for%20the%20analysis%20of%20population%20structure&author=BS%20Weir&author=CC%20Cockerham&volume=38&publication_year=1984&pages=1358-1370&pmid=28563791&doi=10.1111/j.1558-5646.1984.tb05657.x&)

50. Weir BS, Hill WG. Estimating F-statistics. Annu Rev Genet. 2002;36:721–750. doi: 10.1146/annurev.genet.36.050802.093940.  [DOI](https://doi.org/10.1146/annurev.genet.36.050802.093940) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12359738/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Annu%20Rev%20Genet&title=Estimating%20F-statistics&author=BS%20Weir&author=WG%20Hill&volume=36&publication_year=2002&pages=721-750&pmid=12359738&doi=10.1146/annurev.genet.36.050802.093940&)

51. Balding DJ, Martin Bishop M, Cannings C, editors. Handbook of Statistical Genetics. 3. New York: JOHN WILEY & SONS, LTD; 2001.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Handbook%20of%20Statistical%20Genetics&publication_year=2001&)

52. Purcell S, Neale B, Todd-Brown K, Thomas L, Ferreira MA, Bender D, Maller J, Sklar P, de Bakker PI, Daly MJ, Sham PC. PLINK: a tool set for whole-genome association and population-based linkage analyses. American journal of human genetics. 2007;81(3):559–575. doi: 10.1086/519795.  [DOI](https://doi.org/10.1086/519795) | [PMC free article](/articles/PMC1950838/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17701901/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=American%20journal%20of%20human%20genetics&title=PLINK:%20a%20tool%20set%20for%20whole-genome%20association%20and%20population-based%20linkage%20analyses&author=S%20Purcell&author=B%20Neale&author=K%20Todd-Brown&author=L%20Thomas&author=MA%20Ferreira&volume=81&issue=3&publication_year=2007&pages=559-575&pmid=17701901&doi=10.1086/519795&)

53. Pritchard JK, Rosenberg NA. Use of unlinked genetic markers to detect population stratification in association studies. American journal of human genetics. 1999;65(1):220–228. doi: 10.1086/302449.  [DOI](https://doi.org/10.1086/302449) | [PMC free article](/articles/PMC1378093/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10364535/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=American%20journal%20of%20human%20genetics&title=Use%20of%20unlinked%20genetic%20markers%20to%20detect%20population%20stratification%20in%20association%20studies&author=JK%20Pritchard&author=NA%20Rosenberg&volume=65&issue=1&publication_year=1999&pages=220-228&pmid=10364535&doi=10.1086/302449&)

54. Barrett JC, Fry B, Maller J, Daly MJ. Haploview: analysis and visualization of LD and haplotype maps. Bioinformatics. 2005;21(2):263–265. doi: 10.1093/bioinformatics/bth457.  [DOI](https://doi.org/10.1093/bioinformatics/bth457) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15297300/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=Haploview:%20analysis%20and%20visualization%20of%20LD%20and%20haplotype%20maps&author=JC%20Barrett&author=B%20Fry&author=J%20Maller&author=MJ%20Daly&volume=21&issue=2&publication_year=2005&pages=263-265&pmid=15297300&doi=10.1093/bioinformatics/bth457&)

55. Gordon D, Finch SJ, Nothnagel M, Ott J. Power and sample size calculations for case-control genetic association tests when errors are present: application to single nucleotide polymorphisms. Human heredity. 2002;54(1):22–33. doi: 10.1159/000066696.  [DOI](https://doi.org/10.1159/000066696) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12446984/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Human%20heredity&title=Power%20and%20sample%20size%20calculations%20for%20case-control%20genetic%20association%20tests%20when%20errors%20are%20present:%20application%20to%20single%20nucleotide%20polymorphisms&author=D%20Gordon&author=SJ%20Finch&author=M%20Nothnagel&author=J%20Ott&volume=54&issue=1&publication_year=2002&pages=22-33&pmid=12446984&doi=10.1159/000066696&)

56. Gauderman WJ. Sample size requirements for association studies of gene-gene interaction. American journal of epidemiology. 2002;155(5):478–484. doi: 10.1093/aje/155.5.478.  [DOI](https://doi.org/10.1093/aje/155.5.478) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11867360/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=American%20journal%20of%20epidemiology&title=Sample%20size%20requirements%20for%20association%20studies%20of%20gene-gene%20interaction&author=WJ%20Gauderman&volume=155&issue=5&publication_year=2002&pages=478-484&pmid=11867360&doi=10.1093/aje/155.5.478&)

57. Gauderman WJ. Sample size requirements for matched case-control studies of gene-environment interaction. Statistics in medicine. 2002;21(1):35–50. doi: 10.1002/sim.973.  [DOI](https://doi.org/10.1002/sim.973) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11782049/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Statistics%20in%20medicine&title=Sample%20size%20requirements%20for%20matched%20case-control%20studies%20of%20gene-environment%20interaction&author=WJ%20Gauderman&volume=21&issue=1&publication_year=2002&pages=35-50&pmid=11782049&doi=10.1002/sim.973&)

58. Winantea J, Hoang MN, Ohlraun S, Rietschel M, Cichon S, Propping P, Nothen MM, Freudenberg J, Freudenberg-Hua Y. A summary statistic approach to sequence variation in noncoding regions of six schizophrenia-associated gene loci. Eur J Hum Genet. 2006;14(9):1037–1043. doi: 10.1038/sj.ejhg.5201664.  [DOI](https://doi.org/10.1038/sj.ejhg.5201664) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16736033/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Hum%20Genet&title=A%20summary%20statistic%20approach%20to%20sequence%20variation%20in%20noncoding%20regions%20of%20six%20schizophrenia-associated%20gene%20loci&author=J%20Winantea&author=MN%20Hoang&author=S%20Ohlraun&author=M%20Rietschel&author=S%20Cichon&volume=14&issue=9&publication_year=2006&pages=1037-1043&pmid=16736033&doi=10.1038/sj.ejhg.5201664&)

59. Walsh T, McClellan JM, McCarthy SE, Addington AM, Pierce SB, Cooper GM, Nord AS, Kusenda M, Malhotra D, Bhandari A, Stray SM, Rippey CF, Roccanova P, Makarov V, Lakshmi B, Findling RL, Sikich L, Stromberg T, Merriman B, Gogtay N, Butler P, Eckstrand K, Noory L, Gochman P, Long R, Chen Z, Davis S, Baker C, Eichler EE, Meltzer PS, Nelson SF, Singleton AB, Lee MK, Rapoport JL, King MC, Sebat J. Rare Structural Variants Disrupt Multiple Genes in Neurodevelopmental Pathways in Schizophrenia. Science. 2008 doi: 10.1126/science.1155174.  [DOI](https://doi.org/10.1126/science.1155174) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18369103/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=Rare%20Structural%20Variants%20Disrupt%20Multiple%20Genes%20in%20Neurodevelopmental%20Pathways%20in%20Schizophrenia&author=T%20Walsh&author=JM%20McClellan&author=SE%20McCarthy&author=AM%20Addington&author=SB%20Pierce&publication_year=2008&pmid=18369103&doi=10.1126/science.1155174&)

60. Lang UE, Hellweg R, Kalus P, Bajbouj M, Lenzen KP, Sander T, Kunz D, Gallinat J. Association of a functional BDNF polymorphism and anxiety-related personality traits. Psychopharmacology (Berl) 2005;180(1):95–99. doi: 10.1007/s00213-004-2137-7.  [DOI](https://doi.org/10.1007/s00213-004-2137-7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15918078/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Psychopharmacology%20(Berl)&title=Association%20of%20a%20functional%20BDNF%20polymorphism%20and%20anxiety-related%20personality%20traits&author=UE%20Lang&author=R%20Hellweg&author=P%20Kalus&author=M%20Bajbouj&author=KP%20Lenzen&volume=180&issue=1&publication_year=2005&pages=95-99&pmid=15918078&doi=10.1007/s00213-004-2137-7&)

61. Sen S, Nesse RM, Stoltenberg SF, Li S, Gleiberman L, Chakravarti A, Weder AB, Burmeister M. A BDNF coding variant is associated with the NEO personality inventory domain neuroticism, a risk factor for depression. Neuropsychopharmacology. 2003;28(2):397–401. doi: 10.1038/sj.npp.1300053.  [DOI](https://doi.org/10.1038/sj.npp.1300053) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12589394/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neuropsychopharmacology&title=A%20BDNF%20coding%20variant%20is%20associated%20with%20the%20NEO%20personality%20inventory%20domain%20neuroticism,%20a%20risk%20factor%20for%20depression&author=S%20Sen&author=RM%20Nesse&author=SF%20Stoltenberg&author=S%20Li&author=L%20Gleiberman&volume=28&issue=2&publication_year=2003&pages=397-401&pmid=12589394&doi=10.1038/sj.npp.1300053&)

62. Strauss J, Barr CL, George CJ, Devlin B, Vetro A, Kiss E, Baji I, King N, Shaikh S, Lanktree M, Kovacs M, Kennedy JL. Brain-derived neurotrophic factor variants are associated with childhood-onset mood disorder: confirmation in a Hungarian sample. Molecular psychiatry. 2005;10(9):861–867. doi: 10.1038/sj.mp.4001685.  [DOI](https://doi.org/10.1038/sj.mp.4001685) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15940299/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Molecular%20psychiatry&title=Brain-derived%20neurotrophic%20factor%20variants%20are%20associated%20with%20childhood-onset%20mood%20disorder:%20confirmation%20in%20a%20Hungarian%20sample&author=J%20Strauss&author=CL%20Barr&author=CJ%20George&author=B%20Devlin&author=A%20Vetro&volume=10&issue=9&publication_year=2005&pages=861-867&pmid=15940299&doi=10.1038/sj.mp.4001685&)

63. Choi MJ, Kang RH, Lim SW, Oh KS, Lee MS. Brain-derived neurotrophic factor gene polymorphism (Val66Met) and citalopram response in major depressive disorder. Brain research. 2006;1118(1):176–182. doi: 10.1016/j.brainres.2006.08.012.  [DOI](https://doi.org/10.1016/j.brainres.2006.08.012) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16979146/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Brain%20research&title=Brain-derived%20neurotrophic%20factor%20gene%20polymorphism%20(Val66Met)%20and%20citalopram%20response%20in%20major%20depressive%20disorder&author=MJ%20Choi&author=RH%20Kang&author=SW%20Lim&author=KS%20Oh&author=MS%20Lee&volume=1118&issue=1&publication_year=2006&pages=176-182&pmid=16979146&doi=10.1016/j.brainres.2006.08.012&)

64. Hong CJ, Huo SJ, Yen FC, Tung CL, Pan GM, Tsai SJ. Association study of a brain-derived neurotrophic-factor genetic polymorphism and mood disorders, age of onset and suicidal behavior. Neuropsychobiology. 2003;48(4):186–189. doi: 10.1159/000074636.  [DOI](https://doi.org/10.1159/000074636) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14673216/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neuropsychobiology&title=Association%20study%20of%20a%20brain-derived%20neurotrophic-factor%20genetic%20polymorphism%20and%20mood%20disorders,%20age%20of%20onset%20and%20suicidal%20behavior&author=CJ%20Hong&author=SJ%20Huo&author=FC%20Yen&author=CL%20Tung&author=GM%20Pan&volume=48&issue=4&publication_year=2003&pages=186-189&pmid=14673216&doi=10.1159/000074636&)

65. Tsai SJ, Cheng CY, Yu YW, Chen TJ, Hong CJ. Association study of a brain-derived neurotrophic-factor genetic polymorphism and major depressive disorders, symptomatology, and antidepressant response. Am J Med Genet B Neuropsychiatr Genet. 2003;123(1):19–22. doi: 10.1002/ajmg.b.20026.  [DOI](https://doi.org/10.1002/ajmg.b.20026) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14582140/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Med%20Genet%20B%20Neuropsychiatr%20Genet&title=Association%20study%20of%20a%20brain-derived%20neurotrophic-factor%20genetic%20polymorphism%20and%20major%20depressive%20disorders,%20symptomatology,%20and%20antidepressant%20response&author=SJ%20Tsai&author=CY%20Cheng&author=YW%20Yu&author=TJ%20Chen&author=CJ%20Hong&volume=123&issue=1&publication_year=2003&pages=19-22&pmid=14582140&doi=10.1002/ajmg.b.20026&)

66. Hwang JP, Tsai SJ, Hong CJ, Yang CH, Lirng JF, Yang YM. The Val66Met polymorphism of the brain-derived neurotrophic-factor gene is associated with geriatric depression. Neurobiol Aging. 2006;27(12):1834–1837. doi: 10.1016/j.neurobiolaging.2005.10.013.  [DOI](https://doi.org/10.1016/j.neurobiolaging.2005.10.013) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16343697/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neurobiol%20Aging&title=The%20Val66Met%20polymorphism%20of%20the%20brain-derived%20neurotrophic-factor%20gene%20is%20associated%20with%20geriatric%20depression&author=JP%20Hwang&author=SJ%20Tsai&author=CJ%20Hong&author=CH%20Yang&author=JF%20Lirng&volume=27&issue=12&publication_year=2006&pages=1834-1837&pmid=16343697&doi=10.1016/j.neurobiolaging.2005.10.013&)

67. Iga J, Ueno S, Yamauchi K, Numata S, Tayoshi-Shibuya S, Kinouchi S, Nakataki M, Song H, Hokoishi K, Tanabe H, Sano A, Ohmori T. The Val66Met polymorphism of the brain-derived neurotrophic factor gene is associated with psychotic feature and suicidal behavior in Japanese major depressive patients. Am J Med Genet B Neuropsychiatr Genet. 2007;144(8):1003–1006. doi: 10.1002/ajmg.b.30520.  [DOI](https://doi.org/10.1002/ajmg.b.30520) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17510948/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Med%20Genet%20B%20Neuropsychiatr%20Genet&title=The%20Val66Met%20polymorphism%20of%20the%20brain-derived%20neurotrophic%20factor%20gene%20is%20associated%20with%20psychotic%20feature%20and%20suicidal%20behavior%20in%20Japanese%20major%20depressive%20patients&author=J%20Iga&author=S%20Ueno&author=K%20Yamauchi&author=S%20Numata&author=S%20Tayoshi-Shibuya&volume=144&issue=8&publication_year=2007&pages=1003-1006&pmid=17510948&doi=10.1002/ajmg.b.30520&)

68. Gratacos M, Soria V, Urretavizcaya M, Gonzalez JR, Crespo JM, Bayes M, de Cid R, Menchon JM, Vallejo J, Estivill X. A brain-derived neurotrophic factor (BDNF) haplotype is associated with antidepressant treatment outcome in mood disorders. Pharmacogenomics J. 2007 doi: 10.1038/sj.tpj.6500460.  [DOI](https://doi.org/10.1038/sj.tpj.6500460) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17505499/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=A%20brain-derived%20neurotrophic%20factor%20(BDNF)%20haplotype%20is%20associated%20with%20antidepressant%20treatment%20outcome%20in%20mood%20disorders&author=M%20Gratacos&author=V%20Soria&author=M%20Urretavizcaya&author=JR%20Gonzalez&author=JM%20Crespo&publication_year=2007&pmid=17505499&doi=10.1038/sj.tpj.6500460&)

69. Liu QR, Lu L, Zhu XG, Gong JP, Shaham Y, Uhl GR. Rodent BDNF genes, novel promoters, novel splice variants, and regulation by cocaine. Brain research. 2006;1067(1):1–12. doi: 10.1016/j.brainres.2005.10.004.  [DOI](https://doi.org/10.1016/j.brainres.2005.10.004) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16376315/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Brain%20research&title=Rodent%20BDNF%20genes,%20novel%20promoters,%20novel%20splice%20variants,%20and%20regulation%20by%20cocaine&author=QR%20Liu&author=L%20Lu&author=XG%20Zhu&author=JP%20Gong&author=Y%20Shaham&volume=1067&issue=1&publication_year=2006&pages=1-12&pmid=16376315&doi=10.1016/j.brainres.2005.10.004&)
