# Genome-wide analysis links NFATC2 with asparaginase hypersensitivity

## Metadata
**Authors:** Christian A Fernandez, Colton Smith, Wenjian Yang, Charles G Mullighan, Chunxu Qu, Eric Larsen, W Paul Bowman, Chengcheng Liu, Laura B Ramsey, Tamara Chang, Seth E Karol, Mignon L Loh, Elizabeth A Raetz, Naomi J Winick, Stephen P Hunger, William L Carroll, Sima Jeha, Ching-Hon Pui, William E Evans, Meenakshi Devidas, Mary V Relling
**Journal:** Blood
**Date:** 2015 May 18
**DOI:** [10.1182/blood-2015-02-628800](https://doi.org/10.1182/blood-2015-02-628800)
**PMID:** 25987655
**PMCID:** PMC4492197
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4492197/

## Abstract

The rs6021191 variant in NFATC2 is associated with an increased risk of asparaginase hypersensitivity and is an expression quantitative trait locus associated with expression of NFATC2.

Exome interrogation confirms the importance of the HLA-DRB1*07:01 allele in asparaginase hypersensitivity.

## Introduction

Current acute lymphoblastic leukemia (ALL) and lymphoma therapies depend heavily upon the use of glucocorticoids and asparaginase, among other drugs. Asparaginase is a heterologous enzyme that can influence the exposure of other drugs, including glucocorticoids.^1^ Moreover, allergy to asparaginase can compromise the effectiveness of both asparaginase and dexamethasone by decreasing plasma exposure to both agents, and can result in a higher risk of relapse.^2^ Hypersensitivity reactions to asparaginase during treatment are common, and have been associated with anti-asparaginase immunoglobulin G (IgG) antibodies rather than IgE antibodies.^3,4^ However, recent reports indicate that patients receiving PEGylated *Escherichia coli* asparaginase can develop hypersensitivity reactions to the drug without any evidence of detectable antibodies; because IgE may be bound to mast cells and may be present for only a limited time period, it is possible that IgE plays a role in asparaginase-induced reactions but eludes detection.^5^ Therefore, multiple pathways of asparaginase hypersensitivity are possible and currently they are not well understood.

Few studies have investigated genetic risk factors for asparaginase hypersensitivity. In a study of 485 pediatric ALL patients on St. Jude Children’s Research Hospital (SJCRH) Total XV, the rs4958351 variant in the glutamate receptor gene *GRIA1* was associated with asparaginase hypersensitivity.^6^ Our recent study investigating the role of *HLA-DRB1* genes on asparaginase hypersensitivity in 1870 patients of European ancestry identified an association with the *HLA-DRB1**07:01 allele.^7^ However, additional agnostic genome-wide studies that encompass the racial diversity of patients with ALL are required to better understand the risk and the mechanisms of asparaginase hypersensitivity.

The present study is the largest investigation of genetic predispositions to asparaginase hypersensitivity and includes 3308 patients of diverse ancestry. We used a genome-wide association approach, including a focus on exonic variants in addition to noncoding variants, to identify genetic loci associated with asparaginase hypersensitivity.

## Materials and methods

### Patients

Asparaginase hypersensitivity was assessed in children with ALL enrolled on SJCRH protocols Total XIIIA (n = 154), Total XV (n = 498, [ClinicalTrials.gov](http://ClinicalTrials.gov) #[NCT00137111](https://clinicaltrials.gov/ct2/show/NCT00137111)), and Total XVI (n = 271, [ClinicalTrials.gov](http://ClinicalTrials.gov) #[NCT00549848](https://clinicaltrials.gov/ct2/show/NCT00549848)) or Children’s Oncology Group (COG) protocols POG 9906 (n = 222, [ClinicalTrials.gov](http://ClinicalTrials.gov) #[NCT00005603](https://clinicaltrials.gov/ct2/show/NCT00005603)) and AALL0232 (n = 2163, [ClinicalTrials.gov](http://ClinicalTrials.gov) #[NCT00075725](https://clinicaltrials.gov/ct2/show/NCT00075725)). Informed consent from the parents or guardians, and consent from the patients as appropriate were obtained according to Institutional Review Board guidelines for genomic research and for treatment. The schedules and doses of asparaginase used during treatment varied by protocol and have been described elsewhere.^7^ The association between HLA variants and asparaginase allergy among patients of European ancestry enrolled in these studies has been reported previously^7^ (see supplemental Table 1, available on the *Blood* Web site), and we have previously reported an analysis of standard noncoding “GWAS” array single nucleotide polymorphisms (SNPs) for allergy for a subset of the Total XV patients^6^ (supplemental Table 1). Patients enrolled on protocols Total XIIIA, Total XV, and POG 9906 received native *E coli* asparaginase (26%), whereas patients on Total XVI and AALL0232 received PEGylated *E coli* asparaginase (74%). Hypersensitivities were graded using the scales described in the National Cancer Institute’s common toxicity criteria version 1.0 for Total XIIIA,^8^ version 2.0 for POG 9906, version 3.0 for Total XV^6^ and Total XVI, and version 4.0 for COG AALL0232. Hypersensitivity reactions of grade 2 and above were considered cases. The clinical symptoms of reactions grade 2 and above can include rash, flushing, urticaria, fever ≥38°C, symptomatic bronchospasm, and anaphylaxis.

### Genotyping

Germline DNA was genotyped using the Affymetrix Human Mapping 500K Array Set, the Affymetrix Genome-Wide Human SNP Array 6.0, or the Illumina Exome BeadChip array.^9^ Hardy-Weinberg equilibrium tests were performed using PLINK in patients of European ancestry. SNPs with genotyping call rates <95% and SNPs that were not in Hardy-Weinberg equilibrium (*P* < .001 for SNPs with a minor allele frequency [MAF] ≥1%) were excluded from the association analysis. The genetic ancestry of patients was estimated using STRUCTURE as previously described,^10^ with the European-ancestry group defined as having >90% Northern European ancestry (CEU), the Asian-ancestry group defined as having >90% East Asian ancestry (CHB/JBT), the African-ancestry group defined as having >70% West African ancestry (YRI), the Hispanics defined as having Native American ancestry >10% and whose Native American ancestry was greater than their % of African ancestry, and finally, others whose ancestry was outside the above boundaries. Affymetrix genotyping was available for 94% of patients with asparaginase hypersensitivity data (supplemental Table 1). Illumina exome array genotyping was available for 95% of the patients in the combined patient cohort (supplemental Table 1). Overall, 90% of the ALL patients with asparaginase hypersensitivity data had genotyping available from both platforms (supplemental Table 1).

### Nuclear factor of activated T cells 2 (NFATC2) gene expression analysis

*NFATC2* gene expression was assessed in ALL tumor samples from patients enrolled on SJCRH protocols Total XV (n = 57) and Total XVI (n = 8). RNA-seq was performed as previously described.^11,12^ Expression levels of *NFATC2* were estimated as fragment per kilobase of transcript per million mapped reads (FPKM), and gene FPKM values were computed by summing the transcript FPKM values for each gene as described previously.^13,14^ *NFATC2* gene expression and genotyping was also available for 52 unrelated Yoruba (YRI) HapMap samples ([GSE9703](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE9703)).^15,16^ *NFATC2* expression levels in HapMap YRI samples were estimated by adding the hybridization signals of probes within spliced messenger RNA regions of the gene.

### Data analysis

Univariate and multivariate logistic regression was used to identify clinical risk factors associated with asparaginase hypersensitivity. The association analysis between SNP genotypes and asparaginase hypersensitivity reactions was performed using logistic regression, adjusting for significant covariates and assuming an additive genetic model.^17^ Genotype and phenotype association analysis was performed in PLINK.^17^ The association analysis between *NFATC2* gene expression and rs6021191 genotyping were performed using a general linear model with R statistical software (version 2.13.2).

### Pathway analysis using ingenuity

Ingenuity Pathway Analysis (IPA) was used to prioritize biological pathways related to asparaginase hypersensitivity reactions within the genome-wide analysis. Genes containing interrogated SNPs with the strongest associations (lowest 0.1% *P* values, with *P* ≤ 1.1 × 10^−3^) to asparaginase hypersensitivity were included in the pathway analysis (334 genes representing 499 SNPs; supplemental Table 2). Fisher’s exact test was used to determine if the genes most associated with asparaginase hypersensitivity were over-represented in each canonical pathway.

## Results

Hypersensitivity reaction data to native *E coli* asparaginase or PEGylated *E coli* asparaginase were available from a total of 3308 pediatric ALL patients enrolled on 5 different protocols ([Table 1](#T1)). Their median age was 9.3 years (range, 0.17 to 30), there were slightly more males than females (56.2% vs 43.8%; [Table 1](#T1)), and 59.5% of the genotyped patients were of European genetic ancestry ([Table 1](#T1)).

### Table 1.

| Characteristics | Treatment protocol |  |  |  |  |  |
| --- | --- | --- | --- | --- | --- | --- |
| SJCRH total XIIIA (%) | SJCRH total XV (%) | SJCRH total XVI (%) | POG 9906 (%) | COG AALL0232 (%) | Combined |  |
| Sample size (n) | 154 (4.7%)* | 498 (15.1%)* | 271 (8.2%)* | 222 (6.7%)* | 2163 (65.4%)* | 3308* |
| Age | 43 (28.3%),†7.1 | 126 (25.3%),†7.0 | 73 (26.9%),†7.2 | 146 (65.8%),†11.2 | 1372 (63.4%),†10.1 | 1760 (53.2%),†9.3 |
| (n ≥ 10, mean age) |  |  |  |  |  |  |
| Gender | 90 (58.4%)† | 279 (56.0%)† | 156 (57.6%)† | 150 (67.6%)† | 1183 (54.7%)† | 1858 (56.2%)† |
| (n, Male) |  |  |  |  |  |  |
| Ancestry‡ |   |   |   |   |   |   |
| European | 99 (75.6%)† | 322 (66.1%)† | 187 (69.5%)† | 122 (55.0%)† | 1,215 (56.2%)† | 1959 (59.5%)† |
| Hispanic | 11 (8.4%)† | 66 (13.6%)† | 21 (7.8%)† | 64 (28.8%)† | 560 (25.9%)† | 725 (22.0%)† |
| African | 19 (17.6%)† | 72 (14.8%)† | 11 (4.1%)† | 14 (6.3%)† | 134 (6.2%)† | 254 (7.7%)† |
| Asian | 0 (0.0%)† | 6 (1.2%)† | 1 (0.4%)† | 4 (1.8%)† | 42 (1.9%)† | 53 (1.6%)† |
| Other | 2 (1.5%)† | 21 (4.3%)† | 49 (18.2%)† | 18 (8.1%)† | 212 (9.8%)† | 302 (9.2%)† |
| ALL lineage§ | 130 (85.5%)† | 423 (84.9%)† | 225 (83.0%)† | 222 (100.0%)† | 2153 (100.0%)† | 3153 (95.6%)† |
| (n, B-ALL) |  |  |  |  |  |  |

Table 1 Caption: Demographics of ALL patients with asparaginase hypersensitivity data

A total of 589 patients had asparaginase hypersensitivity reactions (17.8%; [Table 2](#T2)). By univariate analysis both treatment protocol (*P* = 2.1 × 10^−51^; [Table 2](#T2)) and asparaginase formulation (*P* = 2.1 × 10^−50^; [Table 2](#T2)) were associated with hypersensitivity. Protocols including native *E coli* asparaginase had a higher frequency of hypersensitivity than those using PEGylated *E coli* asparaginase (35.1% vs 11.6%; [Table 2](#T2)). Age <10 years, ancestry, and B-cell ALL were also associated with the risk of hypersensitivity ([Table 2](#T2)). In multivariate analysis including treatment protocol, gender, age group, ancestry, and ALL lineage ([Table 2](#T2)), only protocol (*P* = 2.3 × 10^−46^), gender (*P* = .04), and ALL lineage (*P* = .003) were associated with asparaginase hypersensitivity. Patients with T-cell ALL were only enrolled on SJCRH protocols; within the standard/high-risk arms of these studies, there were more asparaginase hypersensitivity reactions among patients with B-lineage compared with those with T-cell ALL receiving identical treatment (supplemental Figure 1; *P* = .001). Asparaginase preparation was confounded with protocol, but it also associated with hypersensitivity in multivariate analysis (*P* = 7.2 × 10^−45^) when removing protocol as a covariate from the regression model.

### Table 2.

| Covariate | % Asparaginase allergy* | Univariate P value | Multivariate P value† |
| --- | --- | --- | --- |
| Protocol (% of cases) | Total XV: 204 (41.0%) | 2.1 × 10−51 | 2.3 × 10−46 |
| Total XIIIA: 50 (32.5%) |  |  |  |
| POG 9906: 53 (23.9%) |  |  |  |
| Total XVI: 38 (14.0%) |  |  |  |
| COG AALL0232: 244 (11.3%) |  |  |  |
| Combined: 589 (17.8%) |  |  |  |
| Asparaginase preparation‡ (% of cases) | E coli asparaginase: 307 (35.1%) | 2.1 × 10−50 | 7.2 × 10−45 |
| PEGylated E coli asparaginase: 282 (11.6%) |  |  |  |
| Gender (% of cases) | Male: 351 (18.9%) | .06 | .04 |
| Female: 238 (16.4%) |  |  |  |
| Age (<10, ≥10) | <10: 314 (20.3%) | 3.3 × 10−4 | .11 |
| (% of cases) | ≥10: 273 (15.5%) |  |  |
| Ancestry§ (% of cases) | African: 51 (20.1%) | 2.1 × 10−4 | .07 |
| European: 385 (19.7%) |  |  |  |
| Hispanic: 102 (14.1%) |  |  |  |
| Other: 40 (13.2%) |  |  |  |
| Asian: 4 (7.5%) |  |  |  |
| ALL lineage\|\| (% of cases) | B-lineage¶: 263 (33.8%) | 5.3 × 10−4 | .003 |
| T-lineage¶: 27 (18.8%) |  |  |  |

Table 2 Caption: Analysis of covariates associated with asparaginase hypersensitivity

To identify genomic variants associated with asparaginase hypersensitivity, a multivariate logistic regression model adjusted for treatment arm, ancestry (as a categorical covariate), gender, age, and ALL lineage was used. The minor allele of an intronic polymorphism in *NFATC2* (rs6021191) was associated with a higher risk of hypersensitivity at the genome-wide significance threshold (*P* = 4.1 × 10^−8^, odds ratio [OR] = 3.11; [Figure 1A-B](#F1)). Focusing on nonsynonymous SNPs, the minor allele at rs17885382 in the HLA-DRB1 gene (*P* = 3.23 × 10^−26^, OR = 1.63; [Figure 1A,C](#F1)) had the strongest association with asparaginase hypersensitivity.

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b769/4492197/6369409d30d2/69f1.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4492197_69f1.jpg)

Manhattan plot of P values for genome-wide SNP association with asparaginase hypersensitivity. (A) The Manhattan plot shows the negative log10 of the P values for association with asparaginase hypersensitivity vs each chromosome on the x-axis. The open circles indicate SNPs genotyped using the Affymetrix 6.0 array, whereas the open squares indicate SNPs genotyped by the Illumina Exome Beadchip array. The dashed line denotes the genome-wide threshold (P = 5 × 10−8). (B) SNPs genotyped using the Affymetrix SNP 6.0 array identified the NFATC2 rs6021191 variant associated with asparaginase hypersensitivity (P = 4.1 × 10−8, OR = 3.11, CI = 2.07-4.66). AA, subjects homozygous for the reference allele (A); TT, subjects homozygous for the variant allele (T); AT, heterozygotes. (C) Illumina Exome array genotyping identified the strongest association with the HLA-DRB1 rs17885382 variant (P = 3.2 × 10−6, OR = 1.63, CI = 1.33-2.00). CC, subjects homozygous for the reference allele (C); TT, subjects homozygous for the variant allele (T); CT, heterozygotes. Associations were determined using a general linear model adjusted for treatment, ALL immunophenotype, gender, age group, and ancestry.

The MAF for the *NFATC2* rs6021191 variant was highest in patients of non-European ancestry (0.062; supplemental Table 3), and only 0.001 among patients of European ancestry, MAFs that are consistent with those reported in the general population (supplemental Table 3). The rs6021191 variant was associated with hypersensitivity across all non-European ancestries (supplemental Table 3), with the largest effect size seen in those with primarily Asian ancestry (OR = 8.71; supplemental Figure 2A), followed by those with African ancestry (OR = 4.45; supplemental Figure 2A). The association was also present across all protocols (supplemental Figure 2B) and with both asparaginase preparations (supplemental Figure 2C).

The *NFATC2* rs6021191 variant is located in an intron of the gene and it is in a region with weak enhancer activity in HUVEC and K562 cell lines,^18^ suggesting that the SNP may influence the expression of *NFATC2*. We found higher *NFATC2* gene expression in both ALL tumor samples and in YRI HapMap lymphoblastoid cell line samples carrying the rs6021191 variant allele (*P* = 1.1 × 10^−3^ and 0.03, respectively; [Figure 2A-B](#F2)), indicating that the SNP is an expression quantitative trait locus (eQTL).

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b769/4492197/1906a231f972/69f2.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4492197_69f2.jpg)

The NFATC2 rs6021191 variant is an eQTL. RNA-seq data were available from the ALL tumor samples of 65 SJCRH patients and from 52 unrelated YRI HapMap samples to investigate the association between rs6021191 and NFATC2 gene expression. We found higher NFATC2 gene expression in samples with the rs6021191 variant compared with samples without the variant in both (A) ALL tumor samples (P = 1.1 × 10−3) and (B) YRI HapMap samples (P = .03). Allele abbreviations are explained in Figure 1.

The MAF for the *HLA-DRB1* rs17885382 variant was highest in patients of European ancestry (MAF = 0.125; supplemental Table 4) and present in all non-European ancestries at a MAF ≥0.05 (supplemental Table 4). The effect size was most pronounced in patients of European ancestry (*P* = 5.9 × 10^−5^, OR = 1.68; supplemental Figure 3A), but elevated in other ancestries, for all treatment protocols, and for both asparaginase preparations (supplemental Figure 3A-C).

The rs17885382 variant is located in exon 2 of the *HLA-DRB1* gene (Arg > Gln, 54). Our previous candidate gene analysis of HLA alleles associated with asparaginase allergies in patients of European ancestry identified an association with 10 amino acids in HLA-DRB1, and position 54 (amino acid position 25 when excluding the leader signal sequence) was among those implicated.^7^ From a total of 1540 *HLA-DRB1* alleles with amino acid sequences included in the International ImMunoGene Tics/HLA database,^19^ only *HLA-DRB1**01:13, *HLA-DRB1**03:54, *HLA-DRB1**13:92, *HLA-DRB1**14:26, *HLA-DRB1**15:24, and *HLA-DRB1**07 encode for a Gln in position 54. Our prior HLA-focused analysis discovered that *HLA-DRB1**07:01 was associated with hypersensitivity,^7^ and it is the only allele among those encoding for Gln at position 54 with a MAF >1%. Consistent with our previous analysis, the *HLA-DRB1* rs17885382 variant was in linkage disequilibrium with the *HLA-DRB1**07:01 allele (R^2^ = 0.94); therefore, it is likely that the haplotype is driving the association between rs17885382 and asparaginase hypersensitivity.

In a multivariate analysis, including both rs6021191 (*NFATC2*) and rs17885382 (*HLA-DRB1*) genotype, both variants remained associated with asparaginase hypersensitivity (P_rs6021191_ = 1.8 × 10^−8^, P_rs17885382_ = 3.2 × 10^−6^), and modest increases in effect size were seen after adjusting for the genotypes (OR_rs6021191_ = 3.35, OR_rs17885382_ = 1.66). The frequency of asparaginase hypersensitivity in patients with neither of these variants was 14.8% (321 of 2173). In comparison, 168 of the 738 (22.7%) patients with one variant (at rs6021191 or rs17885382) developed hypersensitivity ([Figure 3](#F3)) (OR_rs6021191_ = 3.07, P_rs6021191_ = 5.4 × 10^−6^; OR_rs17885382_ = 1.66, P_rs17885382_ = 3.5 × 10^−5^). The risk of hypersensitivity was higher still in patients carrying both alleles (9 out of 19 or 47.4%) as compared with patients with no variant alleles ([Figure 3](#F3)) (OR = 8.58, *P* = 1.7 × 10^−5^) and to patients carrying a single variant allele (OR_rs6021191_ = 2.79, P_rs6021191_ = 0.05; OR_rs17885382_ = 5.18, P_rs17885382_ = 1.2 × 10^−3^). Similar results were found taking the preparation of asparaginase into consideration. Patients receiving native *E coli* asparaginase carrying both risk alleles had an incidence of allergy of 50% compared with 32% among noncarriers of risk alleles, whereas patients receiving PEGylated *E coli* asparaginase had an incidence of allergy of 47% compared with 10% among noncarriers. Furthermore, after conditioning on the association for the *NFATC2* rs6021191 and *HLA-DRB1* rs17885382 variants, no other SNP reached the genome-wide significance threshold.

### Figure 3.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b769/4492197/23f32098a060/69f3.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4492197_69f3.jpg)

Patients carrying both the HLA-DRB1 rs17885382 and NFATC2 rs6021191 variant have a higher risk of developing asparaginase hypersensitivity compared with patients with no risk variant. The risk of developing hypersensitivity was determined for patients carrying a single-risk variant (HLA-DRB1 rs17885382 or NFATC2 rs6021191) or for patients carrying both risk variants (NFATC2 rs6021191 and HLA-DRB1 rs17885382). The risk of hypersensitivity was higher in patients carrying a single variant (ORrs6021191 = 3.07, CIrs6021191 = 1.87-4.94, Prs6021191 = 5.4 × 10−6; ORrs17885382 = 1.66, CIrs17885382 = 1.3-2.1, Prs17885382 = 3.5 × 10−5) or both variants (OR = 8.58, CI = 3.14-22.92, P = 1.7 × 10−5) compared with patients with no risk alleles. The associations were determined using a general linear model adjusted for treatment, ALL immunophenotype, gender, age group, and ancestry.

Among the top 20 pathways with *P* values <3 × 10^−3^ invoked by the 334 genes having top ranked SNPs associated with asparaginase hypersensitivity, 13 pathways involved T-cell apoptosis, T-cell signaling, T-cell activation, or T-cell disorders ([Figure 4A](#F4)). Genes within the top 20 pathways included *PPP3CA*, *PRKCQ*, and *PRKCE*, which are involved in T-cell function ([Figure 4B](#F4)). Therefore, the pathway analysis suggests that genetic variants within genes involved in T-cell function may play a role in the development of asparaginase hypersensitivity.

### Figure 4.

![Figure 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b769/4492197/1aac46bbb023/69f4.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4492197_69f4.jpg)

Genes involved in T-cell function may contribute to the risk of developing asparaginase hypersensitivity reactions. IPA was used to determine if genes most associated with asparaginase hypersensitivity were overrepresented in specific biological pathways. A total of 334 genes invoked by 499 SNPs associated with asparaginase hypersensitivity annotated to 234 distinct IPA pathways. (A) The top 20 pathways identified, all with P < 3 × 10−3, were enriched for genes involved in T-cell apoptosis, T-cell signaling, T-cell activation, or T-cell disorders. (B) A total of 33 genes containing SNPs associated with hypersensitivity were included within the top 20 canonical pathways (shown on x-axis). Many of these genes were included in multiple pathways. The number of pathways (out of 20) in which each of these 33 genes is involved is plotted on the y-axis. For example, HLA-DRB1 is involved in 14 out of the 20 top canonical pathways. GPCR, G-protein coupled receptor; nNOS, neuronal NOS; IL, interleukin; iCOS, inducible T-cell costimulator and its ligand, iCOSL.

## Discussion

Asparaginase is a mainstay in the treatment of ALL, but immune responses to asparaginase can decrease the systemic exposure to asparaginase and result in a higher risk of relapse.^2^ In this study, we used a genome-wide association approach to identify genetic loci associated with asparaginase hypersensitivity within a racially diverse set of pediatric patients with ALL. We found that the rs6021191 variant is associated with a higher risk of asparaginase hypersensitivity at the genome-wide significance level ([Figure 1A-B](#F1)), and the variant is associated with higher NFATC2 messenger RNA expression ([Figure 2A-B](#F2)). The *NFATC2* variant was rare in patients of European ancestry (supplemental Table 3); nevertheless, we were able to identify the association between the *NFATC2* variant and asparaginase hypersensitivity because a multiethnic cohort was available for our investigation. Other variants in *NFATC2* have also been related to immune-mediated phenotypes, such as narcolepsy,^20^ self-reported allergies (cat, pollen, and dust-mite allergy),^21^ and thiazolidinedione-induced edema.^22^ Interestingly, both self-reported allergies and narcolepsy also have associations with HLA genetic variants.^21,23^

The *NFATC2* gene encodes a cytoplasmic component of the NFAT transcription factor family. Upon T-cell receptor stimulation, cytoplasmic NFATC2 is dephosphorylated and translocated to the nucleus where it participates in gene regulation.^24^ The role of *NFATC2* on the risk of drug-induced allergy is unknown, yet studies have shown that *NFATC2* can influence the development and function of regulatory T cells and can have either a negative^25^ or positive^26^ regulatory influence on the immune response, depending on the antigen. Nevertheless, several studies using NFAT inhibitors for different immune-related disease models support that inhibition of the NFAT pathway can attenuate an immune response.^27-32^ Moreover, among individuals with Down syndrome, the inhibition of the NFAT pathway (partly via overexpression of *DYRK1A* and *DSCR1* due to trisomy 21)^33,34^ is hypothesized to be related to the cardiac, neurologic, and immunologic defects associated with Down syndrome,^34^ and individuals with Down syndrome may have attenuated antibody responses to several vaccine antigens.^35^ Based on this role of the NFAT pathway in Down syndrome, and on our finding of a higher incidence of asparaginase hypersensitivity among those carrying the *NFATC2* variant, we hypothesized that Down syndrome patients might have a lower incidence of asparaginase hypersensitivity. Indeed, we observed a lower incidence of asparaginase hypersensitivity among ALL patients with Down syndrome than in those without Down syndrome (6% vs 17%, *P* multivariate = 8.4 × 10^−3^; supplemental Figure 4), and this effect was found among protocols using either preparation of asparaginase, with 8% (4 of 48) of patients with Down syndrome having allergic reactions to PEGylated *E coli* asparaginase compared to 12% of patients without Down syndrome, and 0% (0 of 22) of patients with Down syndrome having allergic reactions to native *E coli* asparaginase compared with 37% among patients without Down syndrome. These results further support the notion that variability in NFATC2 plays a role in the risk of allergy to asparaginase, and that inhibition of NFATC2 might mitigate sensitization to asparaginase.

Our study included not only standard “GWAS” arrays (mostly noncoding SNPs), but also the exome array for interrogation of coding variants. The top hit for a coding variant was in the *HLA-DRB1* gene and the SNP was in linkage disequilibrium with the *HLA-DRB1*07:01* allele.^7^ Therefore, the top coding variant that we identified using the exome array was the same variant identified by our candidate gene study. However, the patients included in our candidate gene study were solely of European ancestry (supplemental Table 1), and the current study extends our previous report identifying *HLA-DBRB1**07:01 as a risk allele for asparaginase hypersensitivity by showing that *HLA-DRB1**07:01 is also a risk allele in non-European patients (supplemental Figure 3). Furthermore, we show that the association between the *HLA-DRB1* rs17885382 variant is independent from the *NFATC2* rs6021191 variant and that patients carrying both alleles have a higher risk of developing asparaginase hypersensitivity compared with patients with no risk alleles or with a single risk allele ([Figure 3](#F3)). Interestingly, the *GRIA1* rs4958351 variant that we previously found to associate with asparaginase hypersensitivity^6^ was also associated with the risk of asparaginase hypersensitivity in our combined patient cohort (*P* = .03; OR = 1.2). However, the risk of allergy was stronger in patients receiving native *E coli* asparaginase (OR = 1.55) than in patients receiving PEGylated *E coli* asparaginase (OR = 1.02).

In conclusion, we have identified that the *NFATC2* rs6021191 variant is an eQTL that is associated with the risk of developing asparaginase hypersensitivity. This is the first study to identify an association between *NFATC2* and a drug-induced hypersensitivity. Moreover, the results from our single SNP and pathway analyses suggest that T cells play an important role in the development of asparaginase hypersensitivity, likely due to their role in B-cell activation and differentiation.

## Acknowledgments

This study was supported by grants GM 92666, CA 21765, CA 142665, CA 36401, RCA 156449, and U10 CA98543, COG Statistical Center (U10 CA98413), and COG Specimen Banking (U24 CA114766) from the National Insitutes of Health and by the American Lebanese Syrian Associated Charities.

## Footnotes

## Authorship

Contribution: C.A.F. and M.V.R. designed the project, analyzed and interpreted the data, and drafted the manuscript; C.A.F., C.S., and W.Y. performed statistical analysis; and C.S., W.Y., C.G.M., C.Q., E.L.,W.P.B., C.L., L.B.R., T.C., S.E.K., M.L.L., E.A.R., N.J.W. S.P.H., W.L.C., S.J., C.-H.P., W.E.E., and M.D. contributed to the writing of the manuscript.

Conflict-of-interest disclosure: The authors declare no competing financial interests.

Correspondence: Mary V. Relling, St. Jude Children’s Research Hospital, 262 Danny Thomas Place, Mail Stop 313, Memphis, TN 38105; e-mail: mary.relling@stjude.org.

## References

1. Yang L, Boyd K, Kaste SC, Kamdem Kamdem L, Rahija RJ, Relling MV. A mouse model for glucocorticoid-induced osteonecrosis: effect of a steroid holiday. J Orthop Res. 2009;27(2):169–175. doi: 10.1002/jor.20733.  [DOI](https://doi.org/10.1002/jor.20733) | [PMC free article](/articles/PMC2718787/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18683891/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Orthop%20Res&title=A%20mouse%20model%20for%20glucocorticoid-induced%20osteonecrosis:%20effect%20of%20a%20steroid%20holiday.&author=L%20Yang&author=K%20Boyd&author=SC%20Kaste&author=L%20Kamdem%20Kamdem&author=RJ%20Rahija&volume=27&issue=2&publication_year=2009&pages=169-175&pmid=18683891&doi=10.1002/jor.20733&)

2. Kawedia JD, Liu C, Pei D, et al. Dexamethasone exposure and asparaginase antibodies affect relapse risk in acute lymphoblastic leukemia. Blood. 2012;119(7):1658–1664. doi: 10.1182/blood-2011-09-381731.  [DOI](https://doi.org/10.1182/blood-2011-09-381731) | [PMC free article](/articles/PMC3286344/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22117041/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Dexamethasone%20exposure%20and%20asparaginase%20antibodies%20affect%20relapse%20risk%20in%20acute%20lymphoblastic%20leukemia.&author=JD%20Kawedia&author=C%20Liu&author=D%20Pei&volume=119&issue=7&publication_year=2012&pages=1658-1664&pmid=22117041&doi=10.1182/blood-2011-09-381731&)

3. Killander D, Dohlwitz A, Engstedt L, et al. Hypersensitive reactions and antibody formation during L-asparaginase treatment of children and adults with acute leukemia. Cancer. 1976;37(1):220–228. doi: 10.1002/1097-0142(197601)37:1<220::aid-cncr2820370132>3.0.co;2-w.  [DOI](https://doi.org/10.1002/1097-0142(197601)37:1<220::aid-cncr2820370132>3.0.co;2-w) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/1061636/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer&title=Hypersensitive%20reactions%20and%20antibody%20formation%20during%20L-asparaginase%20treatment%20of%20children%20and%20adults%20with%20acute%20leukemia.&author=D%20Killander&author=A%20Dohlwitz&author=L%20Engstedt&volume=37&issue=1&publication_year=1976&pages=220-228&pmid=1061636&doi=10.1002/1097-0142(197601)37:1<220::aid-cncr2820370132>3.0.co;2-w&)

4. Liu C, Kawedia JD, Cheng C, et al. Clinical utility and implications of asparaginase antibodies in acute lymphoblastic leukemia. Leukemia. 2012;26(11):2303–2309. doi: 10.1038/leu.2012.102.  [DOI](https://doi.org/10.1038/leu.2012.102) | [PMC free article](/articles/PMC3516853/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22484422/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Leukemia&title=Clinical%20utility%20and%20implications%20of%20asparaginase%20antibodies%20in%20acute%20lymphoblastic%20leukemia.&author=C%20Liu&author=JD%20Kawedia&author=C%20Cheng&volume=26&issue=11&publication_year=2012&pages=2303-2309&pmid=22484422&doi=10.1038/leu.2012.102&)

5. Fernandez CA, Stewart E, Panetta JC, et al. Successful challenges using native E. coli asparaginase after hypersensitivity reactions to PEGylated E. coli asparaginase. Cancer Chemother Pharmacol. 2014;73(6):1307–1313. doi: 10.1007/s00280-014-2464-2.  [DOI](https://doi.org/10.1007/s00280-014-2464-2) | [PMC free article](/articles/PMC4137479/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24771103/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Chemother%20Pharmacol&title=Successful%20challenges%20using%20native%20E.%20coli%20asparaginase%20after%20hypersensitivity%20reactions%20to%20PEGylated%20E.%20coli%20asparaginase.&author=CA%20Fernandez&author=E%20Stewart&author=JC%20Panetta&volume=73&issue=6&publication_year=2014&pages=1307-1313&pmid=24771103&doi=10.1007/s00280-014-2464-2&)

6. Chen SH, Pei D, Yang W, et al. Genetic variations in GRIA1 on chromosome 5q33 related to asparaginase hypersensitivity. Clin Pharmacol Ther. 2010;88(2):191–196. doi: 10.1038/clpt.2010.94.  [DOI](https://doi.org/10.1038/clpt.2010.94) | [PMC free article](/articles/PMC3000799/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20592726/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Genetic%20variations%20in%20GRIA1%20on%20chromosome%205q33%20related%20to%20asparaginase%20hypersensitivity.&author=SH%20Chen&author=D%20Pei&author=W%20Yang&volume=88&issue=2&publication_year=2010&pages=191-196&pmid=20592726&doi=10.1038/clpt.2010.94&)

7. Fernandez CA, Smith C, Yang W, et al. HLA-DRB1*07:01 is associated with a higher risk of asparaginase allergies. Blood. 2014;124(8):1266–1276. doi: 10.1182/blood-2014-03-563742.  [DOI](https://doi.org/10.1182/blood-2014-03-563742) | [PMC free article](/articles/PMC4141516/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24970932/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=HLA-DRB1*07:01%20is%20associated%20with%20a%20higher%20risk%20of%20asparaginase%20allergies.&author=CA%20Fernandez&author=C%20Smith&author=W%20Yang&volume=124&issue=8&publication_year=2014&pages=1266-1276&pmid=24970932&doi=10.1182/blood-2014-03-563742&)

8. Woo MH, Hak LJ, Storm MC, et al. Hypersensitivity or development of antibodies to asparaginase does not impact treatment outcome of childhood acute lymphoblastic leukemia. J Clin Oncol. 2000;18(7):1525–1532. doi: 10.1200/JCO.2000.18.7.1525.  [DOI](https://doi.org/10.1200/JCO.2000.18.7.1525) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10735901/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Hypersensitivity%20or%20development%20of%20antibodies%20to%20asparaginase%20does%20not%20impact%20treatment%20outcome%20of%20childhood%20acute%20lymphoblastic%20leukemia.&author=MH%20Woo&author=LJ%20Hak&author=MC%20Storm&volume=18&issue=7&publication_year=2000&pages=1525-1532&pmid=10735901&doi=10.1200/JCO.2000.18.7.1525&)

9. Treviño LR, Yang W, French D, et al. Germline genomic variants associated with childhood acute lymphoblastic leukemia. Nat Genet. 2009;41(9):1001–1005. doi: 10.1038/ng.432.  [DOI](https://doi.org/10.1038/ng.432) | [PMC free article](/articles/PMC2762391/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19684603/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Germline%20genomic%20variants%20associated%20with%20childhood%20acute%20lymphoblastic%20leukemia.&author=LR%20Trevi%C3%B1o&author=W%20Yang&author=D%20French&volume=41&issue=9&publication_year=2009&pages=1001-1005&pmid=19684603&doi=10.1038/ng.432&)

10. Yang JJ, Cheng C, Devidas M, et al. Ancestry and pharmacogenomics of relapse in acute lymphoblastic leukemia. Nat Genet. 2011;43(3):237–241. doi: 10.1038/ng.763.  [DOI](https://doi.org/10.1038/ng.763) | [PMC free article](/articles/PMC3104508/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21297632/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Ancestry%20and%20pharmacogenomics%20of%20relapse%20in%20acute%20lymphoblastic%20leukemia.&author=JJ%20Yang&author=C%20Cheng&author=M%20Devidas&volume=43&issue=3&publication_year=2011&pages=237-241&pmid=21297632&doi=10.1038/ng.763&)

11. Goya R, Sun MG, Morin RD, et al. SNVMix: predicting single nucleotide variants from next-generation sequencing of tumors. Bioinformatics. 2010;26(6):730–736. doi: 10.1093/bioinformatics/btq040.  [DOI](https://doi.org/10.1093/bioinformatics/btq040) | [PMC free article](/articles/PMC2832826/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20130035/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=SNVMix:%20predicting%20single%20nucleotide%20variants%20from%20next-generation%20sequencing%20of%20tumors.&author=R%20Goya&author=MG%20Sun&author=RD%20Morin&volume=26&issue=6&publication_year=2010&pages=730-736&pmid=20130035&doi=10.1093/bioinformatics/btq040&)

12. Roberts KG, Morin RD, Zhang J, et al. Genetic alterations activating kinase and cytokine receptor signaling in high-risk acute lymphoblastic leukemia. Cancer Cell. 2012;22(2):153–166. doi: 10.1016/j.ccr.2012.06.005.  [DOI](https://doi.org/10.1016/j.ccr.2012.06.005) | [PMC free article](/articles/PMC3422513/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22897847/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Cell&title=Genetic%20alterations%20activating%20kinase%20and%20cytokine%20receptor%20signaling%20in%20high-risk%20acute%20lymphoblastic%20leukemia.&author=KG%20Roberts&author=RD%20Morin&author=J%20Zhang&volume=22&issue=2&publication_year=2012&pages=153-166&pmid=22897847&doi=10.1016/j.ccr.2012.06.005&)

13. Shah S, Schrader KA, Waanders E, et al. A recurrent germline PAX5 mutation confers susceptibility to pre-B cell acute lymphoblastic leukemia. Nat Genet. 2013;45(10):1226–1231. doi: 10.1038/ng.2754.  [DOI](https://doi.org/10.1038/ng.2754) | [PMC free article](/articles/PMC3919799/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24013638/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=A%20recurrent%20germline%20PAX5%20mutation%20confers%20susceptibility%20to%20pre-B%20cell%20acute%20lymphoblastic%20leukemia.&author=S%20Shah&author=KA%20Schrader&author=E%20Waanders&volume=45&issue=10&publication_year=2013&pages=1226-1231&pmid=24013638&doi=10.1038/ng.2754&)

14. Trapnell C, Williams BA, Pertea G, et al. Transcript assembly and quantification by RNA-seq reveals unannotated transcripts and isoform switching during cell differentiation. Nat Biotechnol. 2010;28(5):511–515. doi: 10.1038/nbt.1621.  [DOI](https://doi.org/10.1038/nbt.1621) | [PMC free article](/articles/PMC3146043/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20436464/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Biotechnol&title=Transcript%20assembly%20and%20quantification%20by%20RNA-seq%20reveals%20unannotated%20transcripts%20and%20isoform%20switching%20during%20cell%20differentiation.&author=C%20Trapnell&author=BA%20Williams&author=G%20Pertea&volume=28&issue=5&publication_year=2010&pages=511-515&pmid=20436464&doi=10.1038/nbt.1621&)

15. Zhang W, Dolan ME. Use of cell lines in the investigation of pharmacogenetic loci. Curr Pharm Des. 2009;15(32):3782–3795. doi: 10.2174/138161209789649475.  [DOI](https://doi.org/10.2174/138161209789649475) | [PMC free article](/articles/PMC2782819/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19925429/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Pharm%20Des&title=Use%20of%20cell%20lines%20in%20the%20investigation%20of%20pharmacogenetic%20loci.&author=W%20Zhang&author=ME%20Dolan&volume=15&issue=32&publication_year=2009&pages=3782-3795&pmid=19925429&doi=10.2174/138161209789649475&)

16. Zhang W, Duan S, Bleibel WK, et al. Identification of common genetic variants that account for transcript isoform variation between human populations. Hum Genet. 2009;125(1):81–93. doi: 10.1007/s00439-008-0601-x.  [DOI](https://doi.org/10.1007/s00439-008-0601-x) | [PMC free article](/articles/PMC2665168/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19052777/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Genet&title=Identification%20of%20common%20genetic%20variants%20that%20account%20for%20transcript%20isoform%20variation%20between%20human%20populations.&author=W%20Zhang&author=S%20Duan&author=WK%20Bleibel&volume=125&issue=1&publication_year=2009&pages=81-93&pmid=19052777&doi=10.1007/s00439-008-0601-x&)

17. Purcell S, Neale B, Todd-Brown K, et al. PLINK: a tool set for whole-genome association and population-based linkage analyses. Am J Hum Genet. 2007;81(3):559–575. doi: 10.1086/519795.  [DOI](https://doi.org/10.1086/519795) | [PMC free article](/articles/PMC1950838/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17701901/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=PLINK:%20a%20tool%20set%20for%20whole-genome%20association%20and%20population-based%20linkage%20analyses.&author=S%20Purcell&author=B%20Neale&author=K%20Todd-Brown&volume=81&issue=3&publication_year=2007&pages=559-575&pmid=17701901&doi=10.1086/519795&)

18. Hoffman MM, Buske OJ, Wang J, Weng Z, Bilmes JA, Noble WS. Unsupervised pattern discovery in human chromatin structure through genomic segmentation. Nat Methods. 2012;9(5):473–476. doi: 10.1038/nmeth.1937.  [DOI](https://doi.org/10.1038/nmeth.1937) | [PMC free article](/articles/PMC3340533/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22426492/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Methods&title=Unsupervised%20pattern%20discovery%20in%20human%20chromatin%20structure%20through%20genomic%20segmentation.&author=MM%20Hoffman&author=OJ%20Buske&author=J%20Wang&author=Z%20Weng&author=JA%20Bilmes&volume=9&issue=5&publication_year=2012&pages=473-476&pmid=22426492&doi=10.1038/nmeth.1937&)

19. Robinson J, Mistry K, McWilliam H, Lopez R, Parham P, Marsh SG. The IMGT/HLA database. Nucleic Acids Res. 2011;39(database issue):D1171–D1176. doi: 10.1093/nar/gkq998.  [DOI](https://doi.org/10.1093/nar/gkq998) | [PMC free article](/articles/PMC3013815/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21071412/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=The%20IMGT/HLA%20database.&author=J%20Robinson&author=K%20Mistry&author=H%20McWilliam&author=R%20Lopez&author=P%20Parham&volume=39&issue=database%20issue&publication_year=2011&pages=D1171-D1176&pmid=21071412&doi=10.1093/nar/gkq998&)

20. Shimada M, Miyagawa T, Kawashima M, et al. An approach based on a genome-wide association study reveals candidate loci for narcolepsy. Hum Genet. 2010;128(4):433–441. doi: 10.1007/s00439-010-0862-z.  [DOI](https://doi.org/10.1007/s00439-010-0862-z) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20677014/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Genet&title=An%20approach%20based%20on%20a%20genome-wide%20association%20study%20reveals%20candidate%20loci%20for%20narcolepsy.&author=M%20Shimada&author=T%20Miyagawa&author=M%20Kawashima&volume=128&issue=4&publication_year=2010&pages=433-441&pmid=20677014&doi=10.1007/s00439-010-0862-z&)

21. Hinds DA, McMahon G, Kiefer AK, et al. A genome-wide association meta-analysis of self-reported allergy identifies shared and allergy-specific susceptibility loci. Nat Genet. 2013;45(8):907–911. doi: 10.1038/ng.2686.  [DOI](https://doi.org/10.1038/ng.2686) | [PMC free article](/articles/PMC3753407/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23817569/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=A%20genome-wide%20association%20meta-analysis%20of%20self-reported%20allergy%20identifies%20shared%20and%20allergy-specific%20susceptibility%20loci.&author=DA%20Hinds&author=G%20McMahon&author=AK%20Kiefer&volume=45&issue=8&publication_year=2013&pages=907-911&pmid=23817569&doi=10.1038/ng.2686&)

22. Bailey SD, Xie C, Do R, et al. DREAM investigators. Variation at the NFATC2 locus increases the risk of thiazolidinedione-induced edema in the Diabetes REduction Assessment with ramipril and rosiglitazone Medication (DREAM) study. Diabetes Care. 2010;33(10):2250–2253. doi: 10.2337/dc10-0452.  [DOI](https://doi.org/10.2337/dc10-0452) | [PMC free article](/articles/PMC2945168/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20628086/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Diabetes%20Care&title=Variation%20at%20the%20NFATC2%20locus%20increases%20the%20risk%20of%20thiazolidinedione-induced%20edema%20in%20the%20Diabetes%20REduction%20Assessment%20with%20ramipril%20and%20rosiglitazone%20Medication%20(DREAM)%20study.&author=SD%20Bailey&author=C%20Xie&author=R%20Do&volume=33&issue=10&publication_year=2010&pages=2250-2253&pmid=20628086&doi=10.2337/dc10-0452&)

23. Miyagawa T, Toyoda H, Hirataka A, et al. New susceptibility variants to narcolepsy identified in HLA class II region. Hum Mol Genet. 2015;24(3):891–898. doi: 10.1093/hmg/ddu480.  [DOI](https://doi.org/10.1093/hmg/ddu480) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25256355/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mol%20Genet&title=New%20susceptibility%20variants%20to%20narcolepsy%20identified%20in%20HLA%20class%20II%20region.&author=T%20Miyagawa&author=H%20Toyoda&author=A%20Hirataka&volume=24&issue=3&publication_year=2015&pages=891-898&pmid=25256355&doi=10.1093/hmg/ddu480&)

24. Rao A, Luo C, Hogan PG. Transcription factors of the NFAT family: regulation and function. Annu Rev Immunol. 1997;15:707–747. doi: 10.1146/annurev.immunol.15.1.707.  [DOI](https://doi.org/10.1146/annurev.immunol.15.1.707) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9143705/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Annu%20Rev%20Immunol&title=Transcription%20factors%20of%20the%20NFAT%20family:%20regulation%20and%20function.&author=A%20Rao&author=C%20Luo&author=PG%20Hogan&volume=15&publication_year=1997&pages=707-747&pmid=9143705&doi=10.1146/annurev.immunol.15.1.707&)

25. Xanthoudakis S, Viola JP, Shaw KT, et al. An enhanced immune response in mice lacking the transcription factor NFAT1. Science. 1996;272(5263):892–895. doi: 10.1126/science.272.5263.892.  [DOI](https://doi.org/10.1126/science.272.5263.892) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8629027/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=An%20enhanced%20immune%20response%20in%20mice%20lacking%20the%20transcription%20factor%20NFAT1.&author=S%20Xanthoudakis&author=JP%20Viola&author=KT%20Shaw&volume=272&issue=5263&publication_year=1996&pages=892-895&pmid=8629027&doi=10.1126/science.272.5263.892&)

26. Karwot R, Übel C, Bopp T, Schmitt E, Finotto S. Increased immunosuppressive function of CD4(+)CD25(+)Foxp3(+)GITR+ T regulatory cells from NFATc2((-/-)) mice controls allergen-induced experimental asthma. Immunobiology. 2012;217(9):905–911. doi: 10.1016/j.imbio.2012.01.004.  [DOI](https://doi.org/10.1016/j.imbio.2012.01.004) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22317750/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Immunobiology&title=Increased%20immunosuppressive%20function%20of%20CD4(+)CD25(+)Foxp3(+)GITR+%20T%20regulatory%20cells%20from%20NFATc2((-/-))%20mice%20controls%20allergen-induced%20experimental%20asthma.&author=R%20Karwot&author=C%20%C3%9Cbel&author=T%20Bopp&author=E%20Schmitt&author=S%20Finotto&volume=217&issue=9&publication_year=2012&pages=905-911&pmid=22317750&doi=10.1016/j.imbio.2012.01.004&)

27. Nagamoto-Combs K, Combs CK. Microglial phenotype is regulated by activity of the transcription factor, NFAT (nuclear factor of activated T cells). J Neurosci. 2010;30(28):9641–9646. doi: 10.1523/JNEUROSCI.0828-10.2010.  [DOI](https://doi.org/10.1523/JNEUROSCI.0828-10.2010) | [PMC free article](/articles/PMC2914496/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20631193/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Neurosci&title=Microglial%20phenotype%20is%20regulated%20by%20activity%20of%20the%20transcription%20factor,%20NFAT%20(nuclear%20factor%20of%20activated%20T%20cells).&author=K%20Nagamoto-Combs&author=CK%20Combs&volume=30&issue=28&publication_year=2010&pages=9641-9646&pmid=20631193&doi=10.1523/JNEUROSCI.0828-10.2010&)

28. Choi JM, Sohn JH, Park TY, Park JW, Lee SK. Cell permeable NFAT inhibitory peptide Sim-2-VIVIT inhibits T-cell activation and alleviates allergic airway inflammation and hyper-responsiveness. Immunol Lett. 2012;143(2):170–176. doi: 10.1016/j.imlet.2012.01.016.  [DOI](https://doi.org/10.1016/j.imlet.2012.01.016) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22342853/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Immunol%20Lett&title=Cell%20permeable%20NFAT%20inhibitory%20peptide%20Sim-2-VIVIT%20inhibits%20T-cell%20activation%20and%20alleviates%20allergic%20airway%20inflammation%20and%20hyper-responsiveness.&author=JM%20Choi&author=JH%20Sohn&author=TY%20Park&author=JW%20Park&author=SK%20Lee&volume=143&issue=2&publication_year=2012&pages=170-176&pmid=22342853&doi=10.1016/j.imlet.2012.01.016&)

29. Elloumi HZ, Maharshak N, Rao KN, et al. A cell permeable peptide inhibitor of NFAT inhibits macrophage cytokine expression and ameliorates experimental colitis. PLoS ONE. 2012;7(3):e34172. doi: 10.1371/journal.pone.0034172.  [DOI](https://doi.org/10.1371/journal.pone.0034172) | [PMC free article](/articles/PMC3313977/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22479554/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&title=A%20cell%20permeable%20peptide%20inhibitor%20of%20NFAT%20inhibits%20macrophage%20cytokine%20expression%20and%20ameliorates%20experimental%20colitis.&author=HZ%20Elloumi&author=N%20Maharshak&author=KN%20Rao&volume=7&issue=3&publication_year=2012&pages=e34172&pmid=22479554&doi=10.1371/journal.pone.0034172&)

30. Noguchi H, Matsushita M, Okitsu T, et al. A new cell-permeable peptide allows successful allogeneic islet transplantation in mice. Nat Med. 2004;10(3):305–309. doi: 10.1038/nm994.  [DOI](https://doi.org/10.1038/nm994) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14770176/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Med&title=A%20new%20cell-permeable%20peptide%20allows%20successful%20allogeneic%20islet%20transplantation%20in%20mice.&author=H%20Noguchi&author=M%20Matsushita&author=T%20Okitsu&volume=10&issue=3&publication_year=2004&pages=305-309&pmid=14770176&doi=10.1038/nm994&)

31. Kuriyama M, Matsushita M, Tateishi A, et al. A cell-permeable NFAT inhibitor peptide prevents pressure-overload cardiac hypertrophy. Chem Biol Drug Des. 2006;67(3):238–243. doi: 10.1111/j.1747-0285.2006.00360.x.  [DOI](https://doi.org/10.1111/j.1747-0285.2006.00360.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16611217/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Chem%20Biol%20Drug%20Des&title=A%20cell-permeable%20NFAT%20inhibitor%20peptide%20prevents%20pressure-overload%20cardiac%20hypertrophy.&author=M%20Kuriyama&author=M%20Matsushita&author=A%20Tateishi&volume=67&issue=3&publication_year=2006&pages=238-243&pmid=16611217&doi=10.1111/j.1747-0285.2006.00360.x&)

32. Zhang S, Luo L, Wang Y, Gomez MF, Thorlacius H. Nuclear factor of activated T cells regulates neutrophil recruitment, systemic inflammation, and T-cell dysfunction in abdominal sepsis. Infect Immun. 2014;82(8):3275–3288. doi: 10.1128/IAI.01569-14.  [DOI](https://doi.org/10.1128/IAI.01569-14) | [PMC free article](/articles/PMC4136225/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24866796/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Infect%20Immun&title=Nuclear%20factor%20of%20activated%20T%20cells%20regulates%20neutrophil%20recruitment,%20systemic%20inflammation,%20and%20T-cell%20dysfunction%20in%20abdominal%20sepsis.&author=S%20Zhang&author=L%20Luo&author=Y%20Wang&author=MF%20Gomez&author=H%20Thorlacius&volume=82&issue=8&publication_year=2014&pages=3275-3288&pmid=24866796&doi=10.1128/IAI.01569-14&)

33. Roberts I, Izraeli S. Haematopoietic development and leukaemia in Down syndrome. Br J Haematol. 2014;167(5):587–599. doi: 10.1111/bjh.13096.  [DOI](https://doi.org/10.1111/bjh.13096) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25155832/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Haematol&title=Haematopoietic%20development%20and%20leukaemia%20in%20Down%20syndrome.&author=I%20Roberts&author=S%20Izraeli&volume=167&issue=5&publication_year=2014&pages=587-599&pmid=25155832&doi=10.1111/bjh.13096&)

34. Arron JR, Winslow MM, Polleri A, et al. NFAT dysregulation by increased dosage of DSCR1 and DYRK1A on chromosome 21. Nature. 2006;441(7093):595–600. doi: 10.1038/nature04678.  [DOI](https://doi.org/10.1038/nature04678) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16554754/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=NFAT%20dysregulation%20by%20increased%20dosage%20of%20DSCR1%20and%20DYRK1A%20on%20chromosome%2021.&author=JR%20Arron&author=MM%20Winslow&author=A%20Polleri&volume=441&issue=7093&publication_year=2006&pages=595-600&pmid=16554754&doi=10.1038/nature04678&)

35. Kusters MA, Verstegen RH, Gemen EF, de Vries E. Intrinsic defect of the immune system in children with Down syndrome: a review. Clin Exp Immunol. 2009;156(2):189–193. doi: 10.1111/j.1365-2249.2009.03890.x.  [DOI](https://doi.org/10.1111/j.1365-2249.2009.03890.x) | [PMC free article](/articles/PMC2759463/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19250275/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Exp%20Immunol&title=Intrinsic%20defect%20of%20the%20immune%20system%20in%20children%20with%20Down%20syndrome:%20a%20review.&author=MA%20Kusters&author=RH%20Verstegen&author=EF%20Gemen&author=E%20de%20Vries&volume=156&issue=2&publication_year=2009&pages=189-193&pmid=19250275&doi=10.1111/j.1365-2249.2009.03890.x&)
