# Inflammation-related genetic variants predict toxicities following definitive-radiotherapy for lung cancer

## Metadata
**Authors:** Xia Pu, Liewei Wang, Joe Y Chang, Michelle AT Hildebrandt, Yuanqing Ye, Charles Lu, Heath D Skinner, Nifang Niu, Gregory D Jenkins, Ritsuko Komaki, John D Minna, Jack A Roth, Richard M Weinshilboum, Xifeng Wu
**Journal:** Clinical pharmacology and therapeutics
**Date:** 2014 Jul 23
**DOI:** [10.1038/clpt.2014.154](https://doi.org/10.1038/clpt.2014.154)
**PMID:** 25054431
**PMCID:** PMC4206576
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4206576/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC4206576/pdf/nihms614416.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC4206576/pdf/nihms614416.pdf)

## Abstract

Definitive radiotherapy improves locoregional control and survival in inoperable non-small cell lung cancer (NSCLC) patients. However, radiation-induced toxicities (pneumonitis/esophagitis) are common dose-limiting inflammatory conditions. We therefore conducted a pathway-based analysis to identify inflammation-related SNPs associated with radiation-induced pneumonitis or esophagitis. 11,930 SNPs were genotyped in 201 stage I-III NSCLC patients treated with definitive radiotherapy. Validation was performed in an additional 220 NSCLC cases. After validation, 19 SNPs remained significant. A polygenic risk score (PRS) was generated to summarize the effect from validated SNPs. Significant improvements in discriminative ability were observed by adding the PRS into the clinical/epidemiological variable-based model. We then used 277 lymphoblastoid cell-lines to assess radiation sensitivity and eQTL relationships of the identified SNPs. Three genes (PRKCE,DDX58 and TNFSF7) were associated with radiation sensitivity. We concluded that inflammation-related genetic variants could contribute to the development of radiation-induced toxicities. These loci could assist in predicting those unfavorable events.

Keywords: NSCLC, inflammation, radiation, pneumonitis, esophagitis, single nucleotide polymorphism

## INTRODUCTION

Locally advanced Non-small cell lung cancer (NSCLC) is frequently treated with radiotherapy either alone or with chemotherapy. Although higher doses of radiotherapy have been associated with improved outcomes ([1](#R1)), some patients will develop radiation-induced toxicity even death that often necessitates dose reductions that hamper the effectiveness of therapy ([2](#R2)).

Acute normal tissue toxicity (pneumonitis or esophagitis) following radiotherapy for NSCLC is primarily due to hyper-inflammation of the lung or the esophagus following exposure to radiation. The development of either toxicity is multifactorial and difficult to predict. Although some clinical and dosimetric variables are associated with toxicity, these variables by themselves are not reliable predictors due to individual variation([3](#R3)). Thus, objective biomarkers are sorely needed to minimize events and improve treatment efficacy.

Since local inflammation is a key mediator of radiation-induced toxicity, studies have been conducted to identify inflammation-related biomarkers, focusing mostly on circulating inflammatory-molecules([3](#R3)). However, this approach has yet to identify reliable predictive biomarkers, mainly due to the variation of methodology and sample processing. On the other hand, inherited genetic variations could be more reliable alternatives due to their stability and repeatability. Single nucleotide polymorphisms (SNPs) have been reported to associate with radiation-induced pneumonitis or esophagitis in NSCLC([4](#R4)–[8](#R8)). However, previous studies have been limited by sample size, scope of variants analyzed, and lack of validation ([9](#R9)).

Therefore, in the current study, we broadened the scope of assessed genetic variants by systematically evaluating nearly 12,000 SNPs from inflammation-related genes in NSCLC patients treated with definitive radiation therapy. To limit false positives, a validation analysis was undertaken, bringing the total population of NSCLC patients included in the current analysis to 465 – the largest studies of its type yet performed. To better understand the potential functional consequences of the identified toxicity-associated variants, we utilized a lymphoblastoid cell line (LCL) model system of radiation sensitivity. Together, this approach incorporates discovery, validation, and functional assessment of inflammation-related genetic variation as a step towards the identification of meaningful genetic markers that can be used in the clinic to guide treatment decisions for NSCLC patients receiving radiation.

## RESULTS

### Patient Characteristics

A total of 201 NSCLC patients (109 men and 92 women) with a mean age of 65 years were included in the discovery phase. Grade 2 or higher pneumonitis or esophagitis were present in 70 (37%) and 90 (45%) patients, respectively. Thirty patients (15%) experienced both toxicities. All patients were smokers with a median smoking history of 51.7 pack-years. The majority of patients had stage IIIA (49%) or IIIB (30%) disease. Only 8% patients had a pre-treatment performance score of 2 or higher. 117 (58%) patients received concurrent chemoradiotherapy and the majority (55%) of these patients were treated with Intensity-Modulated Radiation Therapy (IMRT), to a median dose to the lung of 15.9 Gy and median esophageal dose of 28.4 Gy. The validation phase included 220 NSCLC patients. Of this group of patients, 77 (35%) had grade 2 or higher pneumonitis and 113 (51%) had esophagitis with 47 (21%) patients having both pneumonitis and esophagitis. Age, sex, smoking status, clinical stage, and pre-treatment performance status were comparable between validation and discovery phases.

### Individual SNP Analysis

A total of 11,930 SNPs from 904 inflammation-related genes were included in the discovery analysis. Of these, 1,208 were significantly associated with esophagitis and 1,321 with pneumonitis at p<0.05. Genotyping data for the validation phase from a previous GWAS for lung cancer risk([12](#R12)) was available for 226 SNPs for esophagitis and 234 SNPs for pneumonitis.

#### Esophagitis

Ten SNPs were validated as significantly associated with esophagitis ([Table 2](#T2)). The most significant SNP, rs1239344, is located in the 3’UTR region of *OSMR* (encoding for oncostatin M receptor) and is located in a predicted miRNA binding site for seven miRNAs based on predictions in the PolymiRTS database([15](#R15)). This SNP was associated with a greater than two-fold increased risk of developing esophagitis in both the discovery (OR= 2.45, 95% CI=1.14–5.26, P=0.021) and validation (OR= 4.15, 95% CI=1.68–10.28, P=0.002) phases. This finding was highly significant (P=1.78×10^−4^) in the combined meta-analysis with a 3.05-fold increase in risk of esophagitis (95% CI=1.70–5.57).

|   |   |   |   |   | Discovery |  | Validation |  | Meta Analysis |  |   |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| Chr | Gene | Location | SNP | Model | OR(95%CI)* | P | OR(95%CI)* | P | OR(95%CI) | P | P-het |
| Esophagitis |  |  |  |  |  |  |  |  |  |  |  |
| 5 | OSMR | 3’-UTR | rs1239344 | DOM | 2.45(1.14– 5.26) | 0.021 | 4.15(1.68–10.28) | 0.002 | 3.05(1.70– 5.47) | 1.78×10−4 | 0.383 |
| 19 | TNFSF7 | 3’-flanking | rs7259857 | ADD | 0.50(0.30– 0.84) | 0.008 | 0.50(0.28– 0.90) | 0.021 | 0.50(0.34– 0.74) | 4.48×10−4 | 0.981 |
| 2 | PRKCE | intron | rs940052 | DOM | 0.34(0.16– 0.75) | 0.007 | 0.37(0.15– 0.90) | 0.030 | 0.35(0.19– 0.64) | 5.49×10−4 | 0.912 |
| 13 | FGF14 | 5’-flanking | rs4772468 | DOM | 2.56(1.20– 5.47) | 0.015 | 2.76(1.12– 6.80) | 0.027 | 2.64(1.48– 4.72) | 1.04×10−3 | 0.897 |
| 6 | TAP1 | intron | rs3819721 | DOM | 2.30(1.12– 4.72) | 0.023 | 2.55(1.14– 5.68) | 0.022 | 2.41(1.41– 4.11) | 1.28×10−3 | 0.851 |
| 12 | CD4 | intron | rs2707212 | DOM | 2.70(1.23– 5.92) | 0.013 | 2.23(1.04– 4.79) | 0.040 | 2.45(1.42– 4.23) | 1.34×10−3 | 0.730 |
| 19 | LILRP2 | exon^ | rs270771 | DOM | 0.28(0.08– 0.93) | 0.037 | 0.10(0.02– 0.66) | 0.017 | 0.21(0.08– 0.57) | 2.30×10−3 | 0.375 |
| 10 | IL15RA | 3’-flanking | rs1998521 | ADD | 1.78(1.03– 3.08) | 0.037 | 1.82(1.02– 3.26) | 0.043 | 1.80(1.21– 2.68) | 3.67×10−3 | 0.958 |
| 2 | TANK | 3’-UTR | rs7309 | DOM | 2.42(1.06– 5.55) | 0.036 | 3.02(1.04– 8.77) | 0.043 | 2.63(1.37– 5.06) | 3.76×10−3 | 0.750 |
| 6 | AGER | 5’-flanking | rs204993 | ADD | 0.56(0.32– 0.98) | 0.043 | 0.54(0.30– 1.00) | 0.050 | 0.55(0.36– 0.83) | 4.82×10−3 | 0.950 |
| Pneumonitis |  |  |  |  |  |  |  |  |  |  |  |
| 10 | CDC2 | 3’-UTR | rs10711 | DOM | 2.67(1.26– 5.63) | 0.010 | 2.33(1.21– 4.48) | 0.011 | 2.47(1.51– 4.04) | 3.08×10−4 | 0.792 |
| 9 | DDX58 | intron | rs11795343 | ADD | 1.79(1.06– 3.03) | 0.030 | 1.95(1.21– 3.14) | 0.006 | 1.88(1.32– 2.67) | 4.86×10−4 | 0.817 |
| 9 | DDX58 | intron | rs7865082 | ADD | 2.32(1.33– 4.03) | 0.003 | 1.62(1.00– 2.63) | 0.050 | 1.89(1.32– 2.72) | 5.85×10−4 | 0.338 |
| 10 | CDC2 | intron | rs1871445 | DOM | 2.41(1.16– 4.99) | 0.018 | 2.28(1.19– 4.36) | 0.013 | 2.34(1.44– 3.79) | 5.93×10−4 | 0.911 |
| 4 | FGF5 | 3’-UTR | rs3733336 | DOM | 0.34(0.16– 0.72) | 0.005 | 0.50(0.26– 0.96) | 0.038 | 0.42(0.26– 0.69) | 6.32×10−4 | 0.459 |
| 21 | ETS2 | intron | rs2298560 | DOM | 0.37(0.16– 0.84) | 0.018 | 0.41(0.20– 0.84) | 0.015 | 0.39(0.23– 0.67) | 6.91×10−4 | 0.869 |
| 2 | LIMS1 | intron | rs12469016 | ADD | 1.86(1.11– 3.13) | 0.019 | 1.70(1.09– 2.65) | 0.020 | 1.77(1.26– 2.48) | 9.96×10−4 | 0.790 |
| 5 | GHR | intron | rs4292454 | ADD | 2.09(1.19– 3.69) | 0.011 | 1.60(1.00– 2.57) | 0.049 | 1.79(1.25– 2.57) | 1.63×10−3 | 0.479 |
| 6 | TFEB | intron | rs13202921 | ADD | 0.50(0.29– 0.87) | 0.015 | 0.61(0.38– 0.98) | 0.040 | 0.56(0.39– 0.81) | 1.65×10−3 | 0.584 |

Table Caption: Inflammation-related genetic variants associated with radiation-induced pneumonitis and esophagitis

#### Pneumonitis

Nine SNPs were significantly associated with pneumonitis in the validation population ([Table 2](#T2)). The most significant SNP, rs10711, is located in the 3’UTR region of *CDK1* (encoding for cyclin-dependent kinase 1), and predicted to create a new binding site for miR-1306-5p([15](#R15)). This SNP was significantly associated with a higher risk of pneumonitis in both phases of the study under the dominant model (OR_discovery_=2.67, 95% CI=1.26– 5.63, P= 0.010; OR_validation_=2.33, 95% CI=1.21- 4.48, P=0.011). In the meta-analysis, this increase remained significant (OR_meta_=2.47, 95% CI=1.51– 4.04, P=3.08×10^−4^).

### Polygenetic risk scores analysis

To quantitate the effect of multiple risk genotypes, polygenic risk scores (PRS) were calculated to better assist in identifying those at highest risk for radiation-induced toxicity ([Table 3](#T3)).

### Table 3.

|   | Discovery |  |   | Validation |  |  |   | Meta Analysis |  |  |   |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| Toxicity | EventN(%) | No EventN(%) | OR(95%CI)* | P | EventN(%) | No EventN(%) | OR(95%CI)* | P | OR(95%CI) | P | P-het |
| Esophagitis (10 SNPs) | 90(44.8%) | 111(55.2%) | 3.73(2.42–5.75) | 2.72×10−9 | 113(51.8%) | 105(48.2%) | 3.03(2.03–4.53) | 6.38×10−8 | 3.33(2.48–4.48) | 1.11×10−15 | 0.493 |
| Pneumonitis (9 SNPs) | 70(37.0%) | 119(63.0%) | 1.97(1.54–2.52) | 8.29×10−8 | 77(38.9%) | 121(61.1%) | 1.84(1.45–2.32) | 3.62×10−7 | 1.90(1.60–2.25) | 1.58×10−13 | 0.693 |

Table 3 Caption: Polygenetic Risk Score (PRS) for radiation-induced toxicity

The mean PRS for esophagitis was similar for both the discovery population (6.10; range: 2.44–9.34) and validation population (6.12; range: 2.42–9.34). There was a consistent association with increased risk of developing esophagitis with per score increase in the PRS (OR_discovery_=3.73, 95% CI=2.42–5.75, P= 2.72×10^−9^; OR_validation_=3.03, 95% CI= 2.03–4.53, P=6.38×10^−8^; OR_meta_=3.33, 95% CI=2.48–4.48, P=1.11×10^−15^). A similar effect was observed for pneumonitis, with a mean PRS of 5.20 (range: 1.61–10.19) in the discovery population, and 5.07 (range: 0.88–9.34) in the validation population. The PRS was positively associated with trend of significantly increased risk of pneumonitis (OR_discovery_=1.97, 95%CI=1.54- 2.52, P= 8.29×10^−8^; OR_validation_=1.84, 95% CI=1.45- 2.32, P=3.62×10^−7^; OR_meta_=1.90, 95% CI=1.60–2.25, P=1.58×10^−13^).

We then tested the ability of the identified genetic variants to enhance prediction of radiation-induced toxicity in a subset of population with complete clinical and genotyping information. A strong improvement of discrimination ability was observed for esophagitis when adding identified loci into the risk model. In the baseline model created with the clinical and epidemiological variables included in the main effect analysis, the AUC for the ROC was 0.799. With the inclusion of the PGS, there was a significant shift in the AUC to 0.936. Bootstrap resampling confirmed the significant increase in the AUC (Δ AUC=0.137 95%CI =0.111–0.236; [Figure 1A](#F1)). A shift in the AUC was also observed for pneumonitis. The AUC for the baseline model was at 0.755 and with the addition of the PGS into the baseline model, the AUC increased to 0.794. This improvement in the prediction discrimination when adding genetic markers was shown to be significant following 1,000 bootstrap resamplings (Δ AUC=0.039, 95%CI=0.001–0.123; [Figure 1B](#F1)).

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ce47/4206576/873528fd80e6/nihms614416f1.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4206576_nihms614416f1.jpg)

Receiver operating characteristic (ROC) curves showing the discriminatory power to predict: A. esophagitis; B. pneumonitis with and without PGS. PGS was generated based on SNPs that showed consistent effects in both discovery and validation phases.

### Functional Correlation with Radiosensitivity of Significant SNPs

Following imputation, 4,786 additional SNPs were identified in the 18 candidate regions harboring the 19 validated SNPs. Of these, 135 (116 imputed and 19 genotyped) were significantly associated with radiation-induced toxicities in the combined discovery and validation population. We selected these SNPs to assess for functional correlation with radiation sensitivity via the LCL model system that incorporates baseline host gene expression and cytotoxicity following radiation treatment. 45 SNPs in three genes (*PRKCE*, *DDX58*, and *TNFSF7*) were found to be significant associated with radiation response, which are more than 5.83 that would be expected by chance alone (p=1.21×10^−27^). While *PRKCE* and *TNFSF7* SNPs also showed significant eQTL relationships ([Supplementary Table 1](#SD1)).

*PRKCE* is located on chromosome 2 and encodes for protein kinase C, epsilon. The genotyped variant, rs940052, is located in an intron and was associated with significantly decreased risk for esophagitis (OR_meta_=0.34, 95% CI=0.19–0.62, P=4.03×10^−4^). Although this SNP was not correlated with radiosensitivity, 40 imputed SNPs within the region surrounding *PRKCE* were found to be significantly associated with not only risk of esophagitis, but also radiation response and cis-regulation of *PRKCE*. For example, an imputed SNP (rs11125035) located 3’ to rs940052 also in an intronic region, was associated with significant decreased risk of esophagitis (OR_meta_=0.34, 95% CI=0.19–0.62, P=4.03×10^−4^). This same variant was a significant eQTL for *PRKCE* (r=0.263, P=1.05×10^−5^, [Figure 2A](#F2)) and associated with radiosensitivity being correlated with a significantly decreased GI50 to gamma-radiation in the LCLs (r=−0.171, P=0.005; [Figure 2B](#F2)).

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ce47/4206576/3b5667878ba8/nihms614416f2.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4206576_nihms614416f2.jpg)

Effect of rs11125035 on gene expression and response to radiation exposure in the LCL model system: A. eQTL analysis; B. GI50. Expression and GI50 values are adjusted for race, sex, and PCs

The variant rs7259857, which is 3’ to *TNFSF7* (encoding for CD70), was associated with risk of developing esophagitis in both discovery and validation populations (OR_meta_=0.51, 95% CI=0.35- 0.75, P= 6.39×10^−4^). In the LCL system, this SNP was also borderline significantly associated with radiation response (GI50: r=0.158, P=0.074) and had a significant eQTL relationship with *TNFSF7* expression (r=0.130, P=0.033). Imputation further identified three SNPs (rs389898, rs427105, and rs2029743) in the 3’ flanking region of *TNFSF7* that conferred similar effects.

Rs11795343 is located in the intronic region of *DDX58* (DEAD box polypeptide 58) and was consistently significant associated with increased risk of developing pneumonitis (OR_meta_=1.88, 95% CI=1.32- 2.68, P=4.63×10^−4^). This SNP was significantly associated with radiation response both in terms of AUC (r=0.145, P=0.019) and GI50 (r=0.123, P=0.046). Moreover, this SNP showed a borderline significant cis-eQTL relationship with *DDX58* expression (r=0.115, P=0.058).

## DISCUSSION

Inflammation is believed to be the most important cellular process contributing to the etiology of esophagitis and pneumonitis. In this study, we applied a targeted, systematic approach to assess the association of inflammation-related SNPs with radiation-induced toxicity. We focused on Caucasian NSCLC patients to maintain a homogeneous population, and the effect of population substructure is minimal for both pneumonitis and esophagitis (data not shown). To minimize potential false positive findings, our study design included a validation step with analysis of additional samples and also incorporated functional genomic analyses to provide potential biological basis for the observed associations.

Among the variants identified in the analysis of pneumonitis or esophagitis, several loci showed functional significances. The intronic SNP rs11795343 in *DDX58*, associated with increased pneumonitis risk, was significantly associated with radiation responses and host gene expression in the LCLs. *DDX58* encodes a DEAD box protein involved in host immune response([16](#R16)). The homolog of *DDX58* in a pig model system has been shown to play a role in infectious disease([17](#R17)). Schneider et al. found that the expression of genes involved in oxidative stress and viral infection response, including *DDX58*, were increased in airway epithelial cells from COPD patients([18](#R18)). This variant is not located in known ENCODE regulatory elements, suggesting that a currently unknown element is present in this region or that this variant is in linkage disequilibrium with a yet undiscovered causal SNP that mediates *DDX58* function.

Variants in *PRKCE* and *TNFSF7* resulted in decreased risk of esophagitis and also found to be correlated with radiation response and host gene expression in the LCLs. Protein kinase C epsilon (*PRKCE*) is a member of protein kinase C (PKC) family that can phosphorylate a number of protein targets and participates in a diverse array of cellular processes([19](#R19), [20](#R20)). It is also known to promote NSCLC growth([21](#R21)) and enhance lung cancer cell survival through suppression of apoptosis([22](#R22)). Studies have found that *PRKCE* is associated with radiation-induced cellular changes ([23](#R23)),([24](#R24)). In our study, rs940052 and 35 imputed SNPs in *PRKCE* were significantly associated with esophagitis, radiation response, and host gene expression. It is likely that this SNP could result in altered expression of *PRKCE* therefore affect response to radiation toxicity in the surrounding normal cells. The third significant variant was located within*TNFSF7* encoding for a tumor necrosis factor (TNF) ligand family member, which contributes to T cell proliferation and activation. TNFSF7 (also known as CD70) is upregulated by radiation exposure which results in T-cell activation([25](#R25)). Our results for the first time suggest a potential important role of this gene in the development of esophagitis and radiosensitivity.

Overall, the majority of the 17 genes implicated in risk of acute radiation-induced toxicity are involved in a set of key cellular processes related to inflammation – this includes NFkappaB, MAPK/JNK, and JAK/STAT. All of these pathways have extensive support for their role in inflammation. More relevant to the current study, there is evidence that they play a role in response to radiation (reviewed by Dent([26](#R26)) and Valerie([27](#R27))), which supports the observed associations of these genes with the risk of developing pneumonitis and esophagitis.

Increasingly, the focus in genetic association studies has moved from single variant to the combined effect of multiple variants on risk for complex diseases and traits([28](#R28)). Previous studies have demonstrated the benefits of using polygenic risk scores for risk estimates of BMI([29](#R29)), prostate and breast cancer([30](#R30)), and bladder cancer([31](#R31)). Similarly, in this study we tested the effect of accumulated genetic information and potential for improved risk stratification to identify those at high risk of radiation-induced toxicities. Two PRS were developed based on the validated loci for pneumonitis and esophagitis. These scores were able to identify those at high risk in both discovery and validation populations with relatively high effect sizes. Rather than consider the effect from each SNP individually, the generated PRS summarized the information from all pre-selected genetic loci and more accurately represent the genetic risk of each patient. Therefore, as shown in the prediction models developed, this approach of assessing the cumulative effect across a panel of genetic variants holds more power in accurately predicting toxicity risk, increasing the potential for clinical application.

Previous studies have developed prediction models based on clinical and dosimetric variables to identify high risk patients for developing radiation-induced toxicities with varied accuracy([32](#R32)–[35](#R35)). In the current study, we demonstrated the ability of genetic information to improve prediction discriminative accuracy. The AUCs for the models that incorporated genetic information were within the range that could have the potential for clinical translation. The esophagitis model with the PRS information reached a prediction power of over 93%, making it potentially a strong tool to better stratify patients into different risk groups and help to minimize toxicity. Further efforts will be needed to validate and calibrate these models in external populations.

In conclusion, our study provides strong evidence that genetic variation in inflammation-related pathways have a significant effect on the risk of developing radiation-induced acute normal tissue toxicities in lung cancer patients. Functional analysis from the LCL model system provides additional evidence supporting our findings and information on the potential mechanism underlying our identified associations. Although not a definitive evidence regarding a biological effect underlying the observed associations, the functional assessment in the LCL model system provide additional evidence in support of the validity of the genetic loci being associated with radiation toxicity in NSCLC patients and presents potential mechanistic links for these events that are worthy of further analysis. Our hope is that these findings can guide future, in-depth investigation, including the in vitro experiments proposed here, regarding these mechanisms. Together, the validated genetic loci together with the constructed PRS could have great power to be used in the clinic to guide personalized decisions regarding optimal radiation dosages for NSCLC patients who receive primary radiation therapy.

## METHODS

### Patient population and data collection

Patients were recruited as part of an ongoing lung cancer case-control study at the University of Texas MD Anderson Cancer Center (MDACC) initiated in 1995. All patients were histologically confirmed; stage I–III NSCLC, Caucasian patients, who were treated with definitive radiotherapy or concurrent chemoradiation therapy. The dose and fraction were prescribed according to tumor stage, size and site, and also considered the surgery status, and patients generally received 55–70 Gy at 1.8–2 Gy per fraction with or without chemotherapy. We only included patients with a radiation total dosage ≥45 Gy to focus only on those received definitive radiation treatment. We excluded any patient treated with stereotactic or hypofractionated radiotherapy (defined as >3 Gy per treatment). During an in-person interview, demographic and epidemiology variables were collected. Clinical and treatment information (pre-treatment performance status, treatment regimens, and radiation dosimetric variables) were abstracted from medical records. Grade two or higher pneumonitis and esophagitis were scored based on the National Cancer Institute Common Terminology Criteria for Adverse Events (version 3.0) guidelines. All patients signed a written informed consent prior to enrollment and this study was approved by the MDACC Institutional Review Board.

### SNP Selection and Genotyping

Inflammation-related gene definitions and SNP identification were described previously([10](#R10)). Briefly, inflammatory-related genes were defined based on a pre-defined inflammation panel, and refined through database exploration (T1DBase [[http://www.t1dbase.org](http://www.t1dbase.org)]; University of Cambridge, Cambridge, UK). Tagging SNPs within a 10kb flanking region of each candidate gene region were identified using the Tagger pairwise method([11](#R11)) with an r^2^ threshold of 0.8 and minor allele frequency ≥0.05 based on genotyping data from the Caucasian population of the International HapMap Project. The final SNP list was used to build a customized Infinium II iSelect Custom Genotyping BeadChip (Illumina, San Diego, CA, USA). Genomic DNA was extracted from peripheral blood using the QIAamp DNA extraction kit (Qiagen, CA, USA). SNP genotyping was performed following the standard Infinium II assay protocol. Only SNPs that had genotype data from ≥95% of all samples and samples with genotype data from ≥95% of all SNPs were included in final data report. The genotyping for validation phase was performed using Illumina 300k BeadChips following the same quality control criteria as in discovery phase([12](#R12)). We defined any association with a P-value less than 0.05 as statistical significant.

### Cell Line Based Assays

Detailed information regarding the cell line-based radiosensitivity assays has been previously described([13](#R13)). Briefly, 277 EBV-transformed LCLs (93 African-American, 89 Caucasian-American, 95 Chinese-American) from unrelated healthy subjects were purchased from the Coriell Cell Repository. Total RNA was extracted and basal gene expression was measured using Affymetrix U113 Plus 2.0 GeneChips (Santa Clara, CA, USA). Cell cultures were exposed to ^137^Cesium gamma-rays and radiation response was measured in triplicate for each dosage. Cell proliferation was measured via MTS assays (CellTiter 96 Aqueous Non-Radioactive Cell Proliferation Assay; Promega, Madison, WI) and the radiation dose required to inhibit 50% growth (GI50) was determined. DNA was extracted and SNPs were genotyped using Illumina 550K & 510S SNP arrays. The quality control procedure was similar as described above – only SNPs with a call rate >95% and a MAF >5% were kept in the analysis, while removing any SNPs that did not match the Hardy-Weinberg equilibrium with a cut-off value of p<0.001.

### Statistical Analysis

Multivariate-logistic regression was used to assess the effect of individual SNPs on the risk of developing toxicity under the additive and dominant models of inheritance. Analysis of toxicity risk included adjustments for age, sex, clinical stage, smoking pack-year, performance status, concurrent chemoradiotherapy, radiation treatment type, pre-treatment lung function (DLCO and FEV1), planning target volume, and median dose (median lung dose for pneumonitis, median esophageal dose for esophagitis). Meta-analysis was used to summarize effect based on the fixed-effect model. Imputation for each candidate gene region was performed using IMPUTE2([14](#R14)) with info score ≥ 0.9 based on 1000 Genomes Project data. Polygenetic risk scores were calculated based on weighted sum of the risk alleles from all validated loci, where the weight was determined by the estimation of each SNP with their association with risk of toxicities (pneumonitis or esophagitis). The improvement of discriminative ability of genetic information adding to baseline clinical-variable based model was determined by comparing area under the curve (AUC) for each receiver operative curves (ROC). Bootstrap results based on 1,000 resamplings were used to confirm the significance of improvement in AUC when adding genetic information. eQTL and radiation response analyses were as previously described using Pearson correlations with the expression and radiation response phenotypes adjusted for race, sex, and PCs([13](#R13)).

## Supplementary Material

### What is the current knowledge on the topic?

| Variable |  | Discovery,n (%) | Validation,n (%) |
| --- | --- | --- | --- |
| Age, mean(SD) |  | 65.0 (9.5) | 62.9 (10.4) |
| Sex |  |   |   |
| Male |  | 109 (54%) | 122 (56%) |
| Female |  | 92 (46%) | 98 (44%) |
| Smoking pack year (SD) |  | 51.7(29.0) | 53.7 (28.9) |
| Clinical stage |  |   |   |
| Stage I |  | 18 (9%) | 26 (12%) |
| Stage II |  | 25 (12%) | 24 (11%) |
| Stage IIIA |  | 98 (49%) | 104 (47%) |
| Stage IIIB |  | 60 (30%) | 66 (30%) |
| Performance status |  |   |   |
| 0 |  | 60 (30%) | 51 (23%) |
| 1 |  | 96 (48%) | 99 (45%) |
| 2–4 |  | 17 (8%) | 30 (14%) |
| FEV1 percentage (Mean, SD) |  | 68.2 (21.1) | 67.6 (18.8) |
| DLCO percentage (Mean, SD) |  | 66.7 (21.6) | 62.1 (19.0) |
| Planned target volume (Mean, SD) |  | 667.0 (440.8) | 755.0 (481.0) |
| Median esophageal dose (Mean, SD) |  | 26.8 (11.6) | 30.1 (13.1) |
| Median lung dose (Mean, SD) |  | 15.4 (5.1) | 18.8 (9.4) |
| Radiation type |  |   |   |
| 2D |  | 24 (12%) | 94 (43%) |
| 3D |  | 30 (15%) | 102 (46%) |
| IMRT |  | 111 (55%) | 24 (11%) |
| Proton |  | 36 (18%) | na |
| Concurrent Chemoradiotherapy |  | 117 (58%) | 135 (61%) |
| Total |  | 201 | 220 |

Table Caption: Patient Characteristics

### What is the current knowledge on the topic?

### What is the current knowledge on the topic?

### What is the current knowledge on the topic?

Radiation-induced acute normal tissue toxicities (esophagitis and pneumonitis) are dosage-limiting events for definitive radiotherapy in locally advanced NSCLC patients, which can compromise the overall treatment effect.

### What question did this study address?

This study aimed to identify accurate genetic biomarkers that can be used to identify NSCLC patients who are at high risk of developing toxicities following definitive radiation therapy.

### What this study adds to our knowledge?

Genetic variants in three genes (*PRKCE*, *DDX58,* and *TNFSF7*) were associated with risk of developing radiation-induced acute toxicities and also displayed correlation with radiation response in a lymphoblastoid cell line model system.

### How this might change clinical pharmacology and therapeutics?

These identified biomarkers and the associated polygenic risk scores developed have the potential to assist in personalization of radiation therapeutic regimens, enabling radiation oncologists to adjust planned doses to minimize toxicity while optimizing effectiveness of treatment.

## ACKNOWLEDGEMENTS

This study was supported by U.S. NIH grants R01-CA111646 (X. Wu), P50CA070907 (J.D. Minna, J.A. Roth, and X. Wu), R01-CA127615 (X. Wu), and CPRIT RP130502 (X. Wu), U19-GM61388 (L. Wang) and R01-CA138461 (L. Wang). Additional support was provided by the Center for Translational and Public Health Genomics of the Duncan Family Institute for Cancer Prevention and Risk Assessment, The University of Texas MD Anderson Cancer Center (X. Wu).

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Machtay M, et al. Higher biologically effective dose of radiotherapy is associated with improved outcomes for locally advanced non-small cell lung carcinoma treated with chemoradiation: an analysis of the Radiation Therapy Oncology Group. Int J Radiat Oncol Biol Phys. 2012;82:425–434. doi: 10.1016/j.ijrobp.2010.09.004.  [DOI](https://doi.org/10.1016/j.ijrobp.2010.09.004) | [PMC free article](/articles/PMC5764542/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20980108/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Radiat%20Oncol%20Biol%20Phys&title=Higher%20biologically%20effective%20dose%20of%20radiotherapy%20is%20associated%20with%20improved%20outcomes%20for%20locally%20advanced%20non-small%20cell%20lung%20carcinoma%20treated%20with%20chemoradiation:%20an%20analysis%20of%20the%20Radiation%20Therapy%20Oncology%20Group&author=M%20Machtay&volume=82&publication_year=2012&pages=425-434&pmid=20980108&doi=10.1016/j.ijrobp.2010.09.004&)

2. O'Rourke N, Roque IFM, Farre Bernado N, Macbeth F. Concurrent chemoradiotherapy in non-small cell lung cancer. Cochrane Database Syst Rev. 2010:CD002140. doi: 10.1002/14651858.CD002140.pub3.  [DOI](https://doi.org/10.1002/14651858.CD002140.pub3) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20556756/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cochrane%20Database%20Syst%20Rev&title=Concurrent%20chemoradiotherapy%20in%20non-small%20cell%20lung%20cancer&author=N%20O'Rourke&author=IFM%20Roque&author=N%20Farre%20Bernado&author=F%20Macbeth&publication_year=2010&pages=CD002140&pmid=20556756&doi=10.1002/14651858.CD002140.pub3&)

3. Madani I, De Ruyck K, Goeminne H, De Neve W, Thierens H, Van Meerbeeck J. Predicting risk of radiation-induced lung injury. J Thorac Oncol. 2007;2:864–874. doi: 10.1097/JTO.0b013e318145b2c6.  [DOI](https://doi.org/10.1097/JTO.0b013e318145b2c6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17805067/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Thorac%20Oncol&title=Predicting%20risk%20of%20radiation-induced%20lung%20injury&author=I%20Madani&author=K%20De%20Ruyck&author=H%20Goeminne&author=W%20De%20Neve&author=H%20Thierens&volume=2&publication_year=2007&pages=864-874&pmid=17805067&doi=10.1097/JTO.0b013e318145b2c6&)

4. Guerra JL, et al. Association between single nucleotide polymorphisms of the transforming growth factor beta1 gene and the risk of severe radiation esophagitis in patients with lung cancer. Radiother Oncol. 2012;105:299–304. doi: 10.1016/j.radonc.2012.08.014.  [DOI](https://doi.org/10.1016/j.radonc.2012.08.014) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23022171/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Radiother%20Oncol&title=Association%20between%20single%20nucleotide%20polymorphisms%20of%20the%20transforming%20growth%20factor%20beta1%20gene%20and%20the%20risk%20of%20severe%20radiation%20esophagitis%20in%20patients%20with%20lung%20cancer&author=JL%20Guerra&volume=105&publication_year=2012&pages=299-304&pmid=23022171&doi=10.1016/j.radonc.2012.08.014&)

5. Hildebrandt MA, et al. Genetic variants in inflammation-related genes are associated with radiation-induced toxicity following treatment for non-small cell lung cancer. PLoS One. 2010;5:e12402. doi: 10.1371/journal.pone.0012402.  [DOI](https://doi.org/10.1371/journal.pone.0012402) | [PMC free article](/articles/PMC2928273/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20811626/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=Genetic%20variants%20in%20inflammation-related%20genes%20are%20associated%20with%20radiation-induced%20toxicity%20following%20treatment%20for%20non-small%20cell%20lung%20cancer&author=MA%20Hildebrandt&volume=5&publication_year=2010&pages=e12402&pmid=20811626&doi=10.1371/journal.pone.0012402&)

6. Palma DA, et al. Predicting radiation pneumonitis after chemoradiation therapy for lung cancer: an international individual patient data meta-analysis. Int J Radiat Oncol Biol Phys. 2013;85:444–450. doi: 10.1016/j.ijrobp.2012.04.043.  [DOI](https://doi.org/10.1016/j.ijrobp.2012.04.043) | [PMC free article](/articles/PMC3448004/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22682812/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Radiat%20Oncol%20Biol%20Phys&title=Predicting%20radiation%20pneumonitis%20after%20chemoradiation%20therapy%20for%20lung%20cancer:%20an%20international%20individual%20patient%20data%20meta-analysis&author=DA%20Palma&volume=85&publication_year=2013&pages=444-450&pmid=22682812&doi=10.1016/j.ijrobp.2012.04.043&)

7. West CM, Dunning AM, Rosenstein BS. Genome-wide association studies and prediction of normal tissue toxicity. Semin Radiat Oncol. 2012;22:91–99. doi: 10.1016/j.semradonc.2011.12.007.  [DOI](https://doi.org/10.1016/j.semradonc.2011.12.007) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22385916/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Semin%20Radiat%20Oncol&title=Genome-wide%20association%20studies%20and%20prediction%20of%20normal%20tissue%20toxicity&author=CM%20West&author=AM%20Dunning&author=BS%20Rosenstein&volume=22&publication_year=2012&pages=91-99&pmid=22385916&doi=10.1016/j.semradonc.2011.12.007&)

8. Yuan X, et al. Single nucleotide polymorphism at rs1982073:T869C of the TGFbeta 1 gene is associated with the risk of radiation pneumonitis in patients with non-small-cell lung cancer treated with definitive radiotherapy. J Clin Oncol. 2009;27:3370–3378. doi: 10.1200/JCO.2008.20.6763.  [DOI](https://doi.org/10.1200/JCO.2008.20.6763) | [PMC free article](/articles/PMC4385796/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19380441/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Single%20nucleotide%20polymorphism%20at%20rs1982073:T869C%20of%20the%20TGFbeta%201%20gene%20is%20associated%20with%20the%20risk%20of%20radiation%20pneumonitis%20in%20patients%20with%20non-small-cell%20lung%20cancer%20treated%20with%20definitive%20radiotherapy&author=X%20Yuan&volume=27&publication_year=2009&pages=3370-3378&pmid=19380441&doi=10.1200/JCO.2008.20.6763&)

9. Kelsey CR, Rosenstein BS, Marks LB. Predicting toxicity from radiation therapy--it's genetic, right? Cancer. 2012;118:3450–3454. doi: 10.1002/cncr.26670.  [DOI](https://doi.org/10.1002/cncr.26670) | [PMC free article](/articles/PMC4287226/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22144075/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer&title=Predicting%20toxicity%20from%20radiation%20therapy--it's%20genetic,%20right?&author=CR%20Kelsey&author=BS%20Rosenstein&author=LB%20Marks&volume=118&publication_year=2012&pages=3450-3454&pmid=22144075&doi=10.1002/cncr.26670&)

10. Pu X, et al. Predictors of Survival in Never-Smokers with Non–Small Cell Lung Cancer: A Large-Scale, Two-Phase Genetic Study. Clinical Cancer Research. 2012;18:5983–5991. doi: 10.1158/1078-0432.CCR-12-0774.  [DOI](https://doi.org/10.1158/1078-0432.CCR-12-0774) | [PMC free article](/articles/PMC3640870/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22977190/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clinical%20Cancer%20Research&title=Predictors%20of%20Survival%20in%20Never-Smokers%20with%20Non%E2%80%93Small%20Cell%20Lung%20Cancer:%20A%20Large-Scale,%20Two-Phase%20Genetic%20Study&author=X%20Pu&volume=18&publication_year=2012&pages=5983-5991&pmid=22977190&doi=10.1158/1078-0432.CCR-12-0774&)

11. de Bakker PI, Yelensky R, Pe'er I, Gabriel SB, Daly MJ, Altshuler D. Efficiency and power in genetic association studies. Nat Genet. 2005;37:1217–1223. doi: 10.1038/ng1669.  [DOI](https://doi.org/10.1038/ng1669) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16244653/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Efficiency%20and%20power%20in%20genetic%20association%20studies&author=PI%20de%20Bakker&author=R%20Yelensky&author=I%20Pe'er&author=SB%20Gabriel&author=MJ%20Daly&volume=37&publication_year=2005&pages=1217-1223&pmid=16244653&doi=10.1038/ng1669&)

12. Amos CI, et al. Genome-wide association scan of tag SNPs identifies a susceptibility locus for lung cancer at 15q25.1. Nat Genet. 2008;40:616–622. doi: 10.1038/ng.109.  [DOI](https://doi.org/10.1038/ng.109) | [PMC free article](/articles/PMC2713680/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18385676/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Genome-wide%20association%20scan%20of%20tag%20SNPs%20identifies%20a%20susceptibility%20locus%20for%20lung%20cancer%20at%2015q25.1&author=CI%20Amos&volume=40&publication_year=2008&pages=616-622&pmid=18385676&doi=10.1038/ng.109&)

13. Niu N, et al. Radiation pharmacogenomics: a genome-wide association approach to identify radiation response biomarkers using human lymphoblastoid cell lines. Genome Res. 2010;20:1482–1492. doi: 10.1101/gr.107672.110.  [DOI](https://doi.org/10.1101/gr.107672.110) | [PMC free article](/articles/PMC2963812/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20923822/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Res&title=Radiation%20pharmacogenomics:%20a%20genome-wide%20association%20approach%20to%20identify%20radiation%20response%20biomarkers%20using%20human%20lymphoblastoid%20cell%20lines&author=N%20Niu&volume=20&publication_year=2010&pages=1482-1492&pmid=20923822&doi=10.1101/gr.107672.110&)

14. Howie BN, Donnelly P, Marchini J. A flexible and accurate genotype imputation method for the next generation of genome-wide association studies. PLoS Genet. 2009;5:e1000529. doi: 10.1371/journal.pgen.1000529.  [DOI](https://doi.org/10.1371/journal.pgen.1000529) | [PMC free article](/articles/PMC2689936/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19543373/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet&title=A%20flexible%20and%20accurate%20genotype%20imputation%20method%20for%20the%20next%20generation%20of%20genome-wide%20association%20studies&author=BN%20Howie&author=P%20Donnelly&author=J%20Marchini&volume=5&publication_year=2009&pages=e1000529&pmid=19543373&doi=10.1371/journal.pgen.1000529&)

15. Bhattacharya A, Ziebarth JD, Cui Y. PolymiRTS Database 3.0: linking polymorphisms in microRNAs and their target sites with human diseases and biological pathways. Nucleic Acids Res. 2013 doi: 10.1093/nar/gkt1028.  [DOI](https://doi.org/10.1093/nar/gkt1028) | [PMC free article](/articles/PMC3965097/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24163105/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=PolymiRTS%20Database%203.0:%20linking%20polymorphisms%20in%20microRNAs%20and%20their%20target%20sites%20with%20human%20diseases%20and%20biological%20pathways&author=A%20Bhattacharya&author=JD%20Ziebarth&author=Y%20Cui&publication_year=2013&pmid=24163105&doi=10.1093/nar/gkt1028&)

16. Seth RB, Sun L, Chen ZJ. Antiviral innate immunity pathways. Cell Res. 2006;16:141–147. doi: 10.1038/sj.cr.7310019.  [DOI](https://doi.org/10.1038/sj.cr.7310019) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16474426/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell%20Res&title=Antiviral%20innate%20immunity%20pathways&author=RB%20Seth&author=L%20Sun&author=ZJ%20Chen&volume=16&publication_year=2006&pages=141-147&pmid=16474426&doi=10.1038/sj.cr.7310019&)

17. Kojima-Shibata C, et al. Differences in distribution of single nucleotide polymorphisms among intracellular pattern recognition receptors in pigs. Immunogenetics. 2009;61:153–160. doi: 10.1007/s00251-008-0350-y.  [DOI](https://doi.org/10.1007/s00251-008-0350-y) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19099299/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Immunogenetics&title=Differences%20in%20distribution%20of%20single%20nucleotide%20polymorphisms%20among%20intracellular%20pattern%20recognition%20receptors%20in%20pigs&author=C%20Kojima-Shibata&volume=61&publication_year=2009&pages=153-160&pmid=19099299&doi=10.1007/s00251-008-0350-y&)

18. Schneider D, et al. Increased cytokine response of rhinovirus-infected airway epithelial cells in chronic obstructive pulmonary disease. Am J Respir Crit Care Med. 2010;182:332–340. doi: 10.1164/rccm.200911-1673OC.  [DOI](https://doi.org/10.1164/rccm.200911-1673OC) | [PMC free article](/articles/PMC2921598/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20395558/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Respir%20Crit%20Care%20Med&title=Increased%20cytokine%20response%20of%20rhinovirus-infected%20airway%20epithelial%20cells%20in%20chronic%20obstructive%20pulmonary%20disease&author=D%20Schneider&volume=182&publication_year=2010&pages=332-340&pmid=20395558&doi=10.1164/rccm.200911-1673OC&)

19. Toton E, Ignatowicz E, Skrzeczkowska K, Rybczynska M. Protein kinase Cepsilon as a cancer marker and target for anticancer therapy. Pharmacol Rep. 2011;63:19–29. doi: 10.1016/s1734-1140(11)70395-4.  [DOI](https://doi.org/10.1016/s1734-1140(11)70395-4) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21441608/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacol%20Rep&title=Protein%20kinase%20Cepsilon%20as%20a%20cancer%20marker%20and%20target%20for%20anticancer%20therapy&author=E%20Toton&author=E%20Ignatowicz&author=K%20Skrzeczkowska&author=M%20Rybczynska&volume=63&publication_year=2011&pages=19-29&pmid=21441608&doi=10.1016/s1734-1140(11)70395-4&)

20. Wu-Zhang AX, Newton AC. Protein kinase C pharmacology: refining the toolbox. Biochem J. 2013;452:195–209. doi: 10.1042/BJ20130220.  [DOI](https://doi.org/10.1042/BJ20130220) | [PMC free article](/articles/PMC4079666/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23662807/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem%20J&title=Protein%20kinase%20C%20pharmacology:%20refining%20the%20toolbox&author=AX%20Wu-Zhang&author=AC%20Newton&volume=452&publication_year=2013&pages=195-209&pmid=23662807&doi=10.1042/BJ20130220&)

21. Caino MC, Lopez-Haber C, Kim J, Mochly-Rosen D, Kazanietz MG. Proteins kinase Cvarepsilon is required for non-small cell lung carcinoma growth and regulates the expression of apoptotic genes. Oncogene. 2012;31:2593–2600. doi: 10.1038/onc.2011.428.  [DOI](https://doi.org/10.1038/onc.2011.428) | [PMC free article](/articles/PMC3432976/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21996750/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncogene&title=Proteins%20kinase%20Cvarepsilon%20is%20required%20for%20non-small%20cell%20lung%20carcinoma%20growth%20and%20regulates%20the%20expression%20of%20apoptotic%20genes&author=MC%20Caino&author=C%20Lopez-Haber&author=J%20Kim&author=D%20Mochly-Rosen&author=MG%20Kazanietz&volume=31&publication_year=2012&pages=2593-2600&pmid=21996750&doi=10.1038/onc.2011.428&)

22. Ding L, Wang H, Lang W, Xiao L. Protein kinase C-epsilon promotes survival of lung cancer cells by suppressing apoptosis through dysregulation of the mitochondrial caspase pathway. J Biol Chem. 2002;277:35305–35313. doi: 10.1074/jbc.M201460200.  [DOI](https://doi.org/10.1074/jbc.M201460200) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12121973/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=Protein%20kinase%20C-epsilon%20promotes%20survival%20of%20lung%20cancer%20cells%20by%20suppressing%20apoptosis%20through%20dysregulation%20of%20the%20mitochondrial%20caspase%20pathway&author=L%20Ding&author=H%20Wang&author=W%20Lang&author=L%20Xiao&volume=277&publication_year=2002&pages=35305-35313&pmid=12121973&doi=10.1074/jbc.M201460200&)

23. Lee YJ, et al. PKC epsilon -mediated ERK1/2 activation involved in radiation-induced cell death in NIH3T3 cells. Biochim Biophys Acta. 2003;1593:219–229. doi: 10.1016/s0167-4889(02)00392-0.  [DOI](https://doi.org/10.1016/s0167-4889(02)00392-0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12581866/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochim%20Biophys%20Acta&title=PKC%20epsilon%20-mediated%20ERK1/2%20activation%20involved%20in%20radiation-induced%20cell%20death%20in%20NIH3T3%20cells&author=YJ%20Lee&volume=1593&publication_year=2003&pages=219-229&pmid=12581866&doi=10.1016/s0167-4889(02)00392-0&)

24. Hu B, Shen B, Su Y, Geard CR, Balajee AS. Protein kinase C epsilon is involved in ionizing radiation induced bystander response in human cells. Int J Biochem Cell Biol. 2009;41:2413–2421. doi: 10.1016/j.biocel.2009.06.012.  [DOI](https://doi.org/10.1016/j.biocel.2009.06.012) | [PMC free article](/articles/PMC2784166/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19577658/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Biochem%20Cell%20Biol&title=Protein%20kinase%20C%20epsilon%20is%20involved%20in%20ionizing%20radiation%20induced%20bystander%20response%20in%20human%20cells&author=B%20Hu&author=B%20Shen&author=Y%20Su&author=CR%20Geard&author=AS%20Balajee&volume=41&publication_year=2009&pages=2413-2421&pmid=19577658&doi=10.1016/j.biocel.2009.06.012&)

25. Huang J, et al. Irradiation enhances human T-cell function by upregulating CD70 expression on antigen-presenting cells in vitro. J Immunother. 2011;34:327–335. doi: 10.1097/CJI.0b013e318216983d.  [DOI](https://doi.org/10.1097/CJI.0b013e318216983d) | [PMC free article](/articles/PMC3094909/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21499130/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Immunother&title=Irradiation%20enhances%20human%20T-cell%20function%20by%20upregulating%20CD70%20expression%20on%20antigen-presenting%20cells%20in%20vitro&author=J%20Huang&volume=34&publication_year=2011&pages=327-335&pmid=21499130&doi=10.1097/CJI.0b013e318216983d&)

26. Dent P, et al. Stress and radiation-induced activation of multiple intracellular signaling pathways. Radiat Res. 2003;159:283–300. doi: 10.1667/0033-7587(2003)159[0283:sariao]2.0.co;2.  [DOI](https://doi.org/10.1667/0033-7587(2003)159%5B0283:sariao%5D2.0.co;2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12600231/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Radiat%20Res&title=Stress%20and%20radiation-induced%20activation%20of%20multiple%20intracellular%20signaling%20pathways&author=P%20Dent&volume=159&publication_year=2003&pages=283-300&pmid=12600231&doi=10.1667/0033-7587(2003)159%5B0283:sariao%5D2.0.co;2&)

27. Valerie K, et al. Radiation-induced cell signaling: inside-out and outside-in. Mol Cancer Ther. 2007;6:789–801. doi: 10.1158/1535-7163.MCT-06-0596.  [DOI](https://doi.org/10.1158/1535-7163.MCT-06-0596) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17363476/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Cancer%20Ther&title=Radiation-induced%20cell%20signaling:%20inside-out%20and%20outside-in&author=K%20Valerie&volume=6&publication_year=2007&pages=789-801&pmid=17363476&doi=10.1158/1535-7163.MCT-06-0596&)

28. Dudbridge F. Power and predictive accuracy of polygenic risk scores. PLoS Genet. 2013;9:e1003348. doi: 10.1371/journal.pgen.1003348.  [DOI](https://doi.org/10.1371/journal.pgen.1003348) | [PMC free article](/articles/PMC3605113/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23555274/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet&title=Power%20and%20predictive%20accuracy%20of%20polygenic%20risk%20scores&author=F%20Dudbridge&volume=9&publication_year=2013&pages=e1003348&pmid=23555274&doi=10.1371/journal.pgen.1003348&)

29. Peterson RE, et al. Genetic risk sum score comprised of common polygenic variation is associated with body mass index. Hum Genet. 2011;129:221–230. doi: 10.1007/s00439-010-0917-1.  [DOI](https://doi.org/10.1007/s00439-010-0917-1) | [PMC free article](/articles/PMC3403709/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21104096/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Genet&title=Genetic%20risk%20sum%20score%20comprised%20of%20common%20polygenic%20variation%20is%20associated%20with%20body%20mass%20index&author=RE%20Peterson&volume=129&publication_year=2011&pages=221-230&pmid=21104096&doi=10.1007/s00439-010-0917-1&)

30. Witte JS, Hoffmann TJ. Polygenic modeling of genome-wide association studies: an application to prostate and breast cancer. OMICS. 2011;15:393–398. doi: 10.1089/omi.2010.0090.  [DOI](https://doi.org/10.1089/omi.2010.0090) | [PMC free article](/articles/PMC3125548/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21348634/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=OMICS&title=Polygenic%20modeling%20of%20genome-wide%20association%20studies:%20an%20application%20to%20prostate%20and%20breast%20cancer&author=JS%20Witte&author=TJ%20Hoffmann&volume=15&publication_year=2011&pages=393-398&pmid=21348634&doi=10.1089/omi.2010.0090&)

31. Garcia-Closas M, et al. Common genetic polymorphisms modify the effect of smoking on absolute risk of bladder cancer. Cancer Res. 2013;73:2211–2220. doi: 10.1158/0008-5472.CAN-12-2388.  [DOI](https://doi.org/10.1158/0008-5472.CAN-12-2388) | [PMC free article](/articles/PMC3688270/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23536561/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=Common%20genetic%20polymorphisms%20modify%20the%20effect%20of%20smoking%20on%20absolute%20risk%20of%20bladder%20cancer&author=M%20Garcia-Closas&volume=73&publication_year=2013&pages=2211-2220&pmid=23536561&doi=10.1158/0008-5472.CAN-12-2388&)

32. De Ruyck K, et al. Development of a Multicomponent Prediction Model for Acute Esophagitis in Lung Cancer Patients Receiving Chemoradiotherapy. Int J Radiat Oncol Biol Phys. 2011 doi: 10.1016/j.ijrobp.2011.03.012.  [DOI](https://doi.org/10.1016/j.ijrobp.2011.03.012) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21605946/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Radiat%20Oncol%20Biol%20Phys&title=Development%20of%20a%20Multicomponent%20Prediction%20Model%20for%20Acute%20Esophagitis%20in%20Lung%20Cancer%20Patients%20Receiving%20Chemoradiotherapy&author=K%20De%20Ruyck&publication_year=2011&pmid=21605946&doi=10.1016/j.ijrobp.2011.03.012&)

33. Dehing-Oberije C, De Ruysscher D, van Baardwijk A, Yu S, Rao B, Lambin P. The importance of patient characteristics for the prediction of radiation-induced lung toxicity. Radiother Oncol. 2009;91:421–426. doi: 10.1016/j.radonc.2008.12.002.  [DOI](https://doi.org/10.1016/j.radonc.2008.12.002) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19147245/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Radiother%20Oncol&title=The%20importance%20of%20patient%20characteristics%20for%20the%20prediction%20of%20radiation-induced%20lung%20toxicity&author=C%20Dehing-Oberije&author=D%20De%20Ruysscher&author=A%20van%20Baardwijk&author=S%20Yu&author=B%20Rao&volume=91&publication_year=2009&pages=421-426&pmid=19147245&doi=10.1016/j.radonc.2008.12.002&)

34. Jenkins P, Watts J. An improved model for predicting radiation pneumonitis incorporating clinical and dosimetric variables. Int J Radiat Oncol Biol Phys. 2011;80:1023–1029. doi: 10.1016/j.ijrobp.2010.03.058.  [DOI](https://doi.org/10.1016/j.ijrobp.2010.03.058) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21543165/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Radiat%20Oncol%20Biol%20Phys&title=An%20improved%20model%20for%20predicting%20radiation%20pneumonitis%20incorporating%20clinical%20and%20dosimetric%20variables&author=P%20Jenkins&author=J%20Watts&volume=80&publication_year=2011&pages=1023-1029&pmid=21543165&doi=10.1016/j.ijrobp.2010.03.058&)

35. Zhu SC, Shen WB, Liu ZK, Li J, Su JW, Wang YX. Dosimetric and clinical predictors of radiation-induced lung toxicity in esophageal carcinoma. Tumori. 2011;97:596–602. doi: 10.1177/030089161109700510.  [DOI](https://doi.org/10.1177/030089161109700510) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22158490/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Tumori&title=Dosimetric%20and%20clinical%20predictors%20of%20radiation-induced%20lung%20toxicity%20in%20esophageal%20carcinoma&author=SC%20Zhu&author=WB%20Shen&author=ZK%20Liu&author=J%20Li&author=JW%20Su&volume=97&publication_year=2011&pages=596-602&pmid=22158490&doi=10.1177/030089161109700510&)
