# Exonic Variants Associated with Development of Aspirin Exacerbated Respiratory Diseases

## Metadata
**Authors:** Seung-Woo Shin, Byung Lae Park, HunSoo Chang, Jong Sook Park, Da-Jeong Bae, Hyun-Ji Song, Inseon S Choi, Mi-Kyeong Kim, Hea-Sim Park, Lyoung Hyo Kim, Suhg Namgoong, Ji On Kim, Hyoung Doo Shin, Choon-Sik Park
**Journal:** PLoS ONE
**Date:** 2014 Nov 5
**DOI:** [10.1371/journal.pone.0111887](https://doi.org/10.1371/journal.pone.0111887)
**PMID:** 25372592
**PMCID:** PMC4221198
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4221198/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC4221198/pdf/pone.0111887.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC4221198/pdf/pone.0111887.pdf)

## Abstract

Aspirin-exacerbated respiratory disease (AERD) is one phenotype of asthma, often occurring in the form of a severe and sudden attack. Due to the time-consuming nature and difficulty of oral aspirin challenge (OAC) for AERD diagnosis, non-invasive biomarkers have been sought. The aim of this study was to identify AERD-associated exonic SNPs and examine the diagnostic potential of a combination of these candidate SNPs to predict AERD. DNA from 165 AERD patients, 397 subjects with aspirin-tolerant asthma (ATA), and 398 normal controls were subjected to an Exome BeadChip assay containing 240K SNPs. 1,023 models (210-1) were generated from combinations of the top 10 SNPs, selected by the p-values in association with AERD. The area under the curve (AUC) of the receiver operating characteristic (ROC) curves was calculated for each model. SNP Function Portal and PolyPhen-2 were used to validate the functional significance of candidate SNPs. An exonic SNP, exm537513 in HLA-DPB1, showed the lowest p-value (p = 3.40×10−8) in its association with AERD risk. From the top 10 SNPs, a combination model of 7 SNPs (exm537513, exm83523, exm1884673, exm538564, exm2264237, exm396794, and exm791954) showed the best AUC of 0.75 (asymptotic p-value of 7.94×10−21), with 34% sensitivity and 93% specificity to discriminate AERD from ATA. Amino acid changes due to exm83523 in CHIA were predicted to be “probably damaging” to the structure and function of the protein, with a high score of ‘1’. A combination model of seven SNPs may provide a useful, non-invasive genetic marker combination for predicting AERD.

## Introduction

Aspirin (acetylsalicylic acid, ASA) - hypersensitivity refers to de­velopment of aspirin-exacerbated respiratory diseases (AERD), and ocular and skin manifestations following ingestion of aspirin or other nonsteroidal anti-inflammatory drugs (NSAIDs) [[1]](#pone.0111887-Lee1). The prevalence of aspirin hypersensitivity in adult asthmatics varies widely depending on whether it is identified by clinical history alone or after challenge with ASA [[2]](#pone.0111887-Szczeklik1). Based on patients’ histories alone, the incidence of AERD in adults is 38%, but this percentage increases three-fold after ASA challenge via the oral or bronchial route [[3]](#pone.0111887-Hedman1). It is noteworthy that approximately one-fifth of patients are unaware that they suffer from aspirin intolerance [[2]](#pone.0111887-Szczeklik1). Thus, identification of aspirin hypersensitivity, especially in hidden cases, is essential to avoid serious bronchospasm attacks in asthmatics.

Diagnosis of AERD can be established with certainty only by provocation tests such as Oral aspirin challenge (OAC). However, OAC is a time-consuming procedure, and in some cases, serious complications can occur. Thus, the development of non-invasive diagnostic methods is necessary to prevent the unexpected complications of aspirin use in susceptible patients.

Fewer than 100 genetic variants have been identified to date. They include biologically plausible genes responsible for the over- or under-production of critical modulators in the metabolism of arachidonic acids, and their receptors include *LTC4S* [[4]](#pone.0111887-Sanak1) *ALOX5* [[5]](#pone.0111887-Choi1), *CYSLT1R* [[6]](#pone.0111887-Kim1), *CYSLT2R* [[7]](#pone.0111887-Park1), *PTGER* [[8]](#pone.0111887-Jinnai1)–[[10]](#pone.0111887-Park2), *TBXAS1* [[11]](#pone.0111887-Oh1), and *TBXA2R* [[12]](#pone.0111887-Kim3). as well as those in the immune response and inflammatory pathways, including DPB1*0301 [[13]](#pone.0111887-Dekker1), *IL-4* [[14]](#pone.0111887-Kim4), *T-Box* [[15]](#pone.0111887-Akahoshi1), *FcepsilonR1* [[16]](#pone.0111887-Kim5), [[17]](#pone.0111887-Palikhe1), *TLR3* [[18]](#pone.0111887-Palikhe2), *NLRP3* [[19]](#pone.0111887-Hitomi1), *ADAM33* [[20]](#pone.0111887-Sakagami1), *ADORA1* [[21]](#pone.0111887-Kim6), *ACE* [[22]](#pone.0111887-Kim7), *CRTH2* [[23]](#pone.0111887-Palikhe3), *PPARG* [[24]](#pone.0111887-Oh2), *KIF3A* [[25]](#pone.0111887-Kim8), *SLC6A12* [[26]](#pone.0111887-Pasaje1), *SLC22A2* [[27]](#pone.0111887-Park3) and *CACNG6* [[28]](#pone.0111887-Lee2). The statistical significance of these SNPs, however, was marginal, with an odds ratio (OR) ranging from 0.5∼2.0 [[29]](#pone.0111887-Park4).

Our previous genome-wide association studies (GWAS) using 100,000 SNPs, revealed that rs7275857 in exon 2 of *CEP68* and rs1042151 in *HLA-DPB1* (Met105Val) were associated with AERD with an OR of 2.63 [[30]](#pone.0111887-Kim9) and 2.40 [[31]](#pone.0111887-Park5) respectively. The limiting factors of these studies included the analysis of common variants with >5% minor allele frequency (MAF) and minimal coverage of exonic SNPs [[32]](#pone.0111887-Pritchard1), [[33]](#pone.0111887-McCarthy1). Thus, recent genetic studies have focused on the possible contributions of rare variants (MAF <1%) that confer a substantial risk of disease. On the basis of growing information on these rare variants, a new Human Exome BeadChip (Illumina Inc., San Diego, CA, USA) has been developed to cover the putative functional exonic variants selected from >12,000 individual exome- and whole-genome sequences [[34]](#pone.0111887-Illumina1). We performed a GWAS using the HumanExome BeadChip v1.1 (Illumina Inc.) to identify new genetic variants and, in particular, exonic variants associated with the risk of AERD, and we evaluated the diagnostic potential of these candidate SNPs to predict AERD in Korean patients with asthma.

## Materials and Methods

### Ethics statement

Written informed consent was obtained from all study participants. For minors/children, it obtained from next of kin, caretakers, or guardians by written form. The protocol was approved by the Soonchunhyang Bucheon Hospital’s ethics committee (SCHBC_IRB_05_02 and schbc-biobank-2012-004). Genomic DNA from all the ethnic Korean study subjects was obtained from a biobank at Soonchunhyang Bucheon Hospital.

### Subjects

Study subjects were recruited from the Asthma Genome Research Center consisting of four university hospitals. They were ethnic Koreans. Asthma was diagnosed by physicians. All patients had a history of dyspnea and wheezing in the previous 12 months, plus one of the following: (1) >15% increase in forced expiratory volume in 1 s (FEV1) or >12% increase plus 200 mL following inhalation of a short-acting bronchodilator, (2) <10 mg/mL PC20 methacholine and (3) >20% increase in FEV1 following 2 weeks of treatment with inhaled or systemic corticosteroids and they met the criteria for asthma according to the Global Initiative for Asthma (GINA) guidelines [[35]](#pone.0111887-Bateman1). Total IgE was measured by the CAP system (Pharmacia Diagnostics, Uppsala, Sweden). Twenty-four commonly inhaled allergens were used for a skin-prick test. Atopy was defined as a wheal reaction with a diameter of 3 mm or greater than the reaction to histamine. Questions related to five specific elements of aspirin hypersensitivity (dyspnea, wheezing, nasal blockage, skin eruption, and loss of consciousness after aspirin ingestion) were incorporated in the questionnaires. Oral aspirin challenge (OAC) was performed to all of the asthmatics. OAC was performed with increasing doses of aspirin using the methods previously described [[36]](#pone.0111887-Chang1), [[37]](#pone.0111887-NizankowskaMogilnicka1). Patients having a history of aspirin hypersensitivity were given a dose of 30 mg and those with no history started 100 mg of aspirin orally. Symptoms, external signs (urticaria, angioedema, and rhinorrhea), blood pressure, and FEV1 were documented every 30 min for a period of 2 h. In the absence of any symptoms or signs suggestive of an adverse reaction, 100 mg of aspirin were administered and the same measurements were repeated every hour, with doses of 450 mg given until the patient developed a reaction. The test was deemed negative if no reaction occurred within 4 h after the final dose. Aspirin-induced bronchospasm was calculated as the pre-challenge FEV1 minus the post-challenge FEV1 divided by the pre challenge FEV1. OAC reactions were categorized into two groups as follows: (1) 15% or greater decreases of FEV1 or appearance of naso-ocular reactions (AERD); and (2) decreases of less than 15% of FEV1 without naso-ocular and cutaneous reactions [aspirin-tolerant asthma (ATA)]. Normal controls were spouses of the subjects or general populations, gave negative answers to a screening questionnaire for respiratory symptoms and showed >70% of FEV_1_/FVC, and a normal chest X-ray.

### Genome – wide exon SNP genotyping and quality control

Approximately 200 ng of genomic DNA was used to genotype each sample on the Illumina HumanExome v1.1 BeadChip (Illumina, Inc., San Diego, CA, USA). Samples were processed according to the Illumina Infinium assay manual. Briefly, each sample was whole-genome amplified, fragmented, precipitated, and resuspended in an appropriate hybridization buffer. Denatured samples were hybridized on a prepared HumanExome v1.1 BeadChip for a minimum of 16 hours at 48°C. Following hybridization, the beadchips were processed for the single-base extension reaction, stained, and imaged on an Illumina Bead Array Reader. Normalized bead intensity data obtained for each sample were loaded into the GenomeStudio software (Illumina, Inc.), which converted fluorescent intensities into SNP genotypes. SNP clusters for genotype calling were examined for all SNPs using the GenomeStudio software. For quality control, only SNPs that were genotyped in more than 98% of samples were included in the further analysis, and SNPs that met over 0.97 of call rate and polymorphic in current subjects were retained. Cluster plots were then assessed visually, and SNPs with poor cluster quality were removed. To examine population substructure, the population stratification analyze was performed.

### Selection of candidate SNPs to predict AERD

First, the case-control study was applied to the exome chip to select the candidate 10 SNPs with AERD. The best combination of the 10 SNPs for discriminating between the AERD and ATA, multiple logistic regressions were performed using ‘R’ software (ver. 2.13.1; [http://www.r-project.org/](http://www.r-project.org/)). ROC curves were obtained for all models, and the area under the curve (AUC) was calculated for each model in order to select the best combination of marker SNPs for discriminating AERD from ATA. The 5-fold cross-validation was applied to validate the best prediction model obtained by using multiple logistic regressions.

### The functional implication and structure effect of the candidate SNPs

To find the functional significance of SNPs, SNP functional Portal was used [[38]](#pone.0111887-Wang1). It was designed to be a clearing house for all public domain SNP functional annotation data, as well as in-house functional annotations derived from different data sources. It contains SNP functional annotations for genomic elements, transcription regulation, protein function, pathway, disease and population genetics.

Because the most SNPs in exome chip were located in the exon region, the structure stability test for the selected SNPs was performed using PolyPhen-2 [[39]](#pone.0111887-Adzhubei1). The Polyphen uses protein sequences from UniProtKB/UniRef100 Release 2011_12 (14-Dec-2011), structures from PDB/DSSP Snapshot 03-Jan-2012 (78,304 entries) and UCSC MultiZ multiple alignments of 45 vertebrate genomes with hg19/GRCh37 human genome (08-Oct-2009). The result for the structure stability was presented as a probability from ‘0’ to ‘1’. ‘0’ means that there is no impact of amino acid substitutions on the structure and function of human proteins and ‘1’ means much impact on the structure and function.

### Statistical analyses

Statistical analyses were performed using R software (ver. 2.13.1; [http://www.r-project.org/](http://www.r-project.org/)). For sex, smoking and atopy variable that were summarized as frequencies, the fisher’s exact test was applied. Because age, onset of asthma, decline (%) of FEV, BMI, Blood eosinophil (%), FEV1% predicted, FVC% predicted and PC20 histamine (mg/mL) didn’t satisfy the normality assumption, they were expressed as medians and ranges and Mann-Whitney U test was applied. The statistical significance was defined as *p*-value<0.05. To examine the association between genotypes and AERD and to calculate odds ratios (ORs) per allele, allelic tests, genotype tests, and Cochran-Armitage trend tests were used. A quantile-quantile plot was used to evaluate overall significance of the genome-wide exon association results and the impacts of population stratification.

## Results

### Clinical characteristics of the study populations

Total 560 asthmatics (165 AERD and 397 ATA) and 398 normal controls were included. Age, onset age of asthma, smoking status, decline (%) of FEV1 by aspirin provocation, BMI, FEV1, % predicted and PC20 methacholine were significantly different between the two groups (*p*-value<0.01). ([Table 1](#pone-0111887-t001)).

### Table 1. Clinical profiles of the study subjects.

|   | AERD | ATA | NC |
| --- | --- | --- | --- |
| Number of subjects (n) | 165 | 397 | 398 |
| Age | 41 (17–72)* , ** | 48 (15–77)** | 57 (11–86) |
| Age at asthma onset | 34 (1–68)* | 41 (1–76) | N/A |
| Sex (male/female) | 165 (63/102) | 397 (120/277)** | 295 (173/225) |
| Smoker/Ex-smoker (%) | 12.12/6.06* | 8.06/14.60** | 18.34/15.32 |
| Decline (%) of FEV1 by aspirin provocation | 28.10 (17.6–100)* | 4 (−15.00–14.7) | ND |
| FEV1, % predicted | 82.5 (68–110)* , ** | 78 (68–130)** | 105.00 (75–168) |
| PC20 methacholine (mg/mL) | 1.00 (0.07–25.00)* , ** | 1.85 (0.01–25.00)** | 25.00 (13.6–25.00) |
| Positive skin prick test (%) | 50.03** | 50.88** | 23.86 |
| Body mass index (kg/m2) | 23.69±4.27* , ** | 24.45±3.74 | 24.34 (14.8–36.21) |

Table 1 Caption: All data are presented as medians (range) or means ± standard deviation. *: p-value<0.01 vs. ATA. **: p-value<0.01 vs. NC.

### Genome – wide exon SNP genotyping and quality control

Firstly, we defined the cut-off value of call rate as 97%. Then genotype clusters for 500 variants showing call rate nearby 97% (500 above 97%) were plotted using GenomeStudio (Illumina, Inc., San Diego, CA, USA) software optimized for Korean population. In this study, among 242,985 SNPs displayed on the chip, 1,132 SNPs with call rate <97% and with poor cluster quality were removed. Among the remaining 241,853 SNPs, we checked MAF and removed additional 187,298 SNPs which were monomorphic in our study subjects. Finally, a total 54,555 autosomal SNPs were used for further statistical analysis. The overall call rate for all SNPs was 99.91% and only 6,878 SNPs among the 545,555 SNPs (12.6%) were found to have a MAF<1%. The result of the MDS (multidimensional scaling) plot analysis showed there was no the population substructure ([Figure S1](#pone.0111887.s001)). These SNPs were then used for association analysis with AERD under the additive mode using logistic regression and because age, smoking and BMI were significantly different between the two groups in the present and other studies [[36]](#pone.0111887-Chang1), [[40]](#pone.0111887-Eisner1), [[41]](#pone.0111887-Stevenson1), and because female are predominant [[42]](#pone.0111887-BergesGimeno1), [[43]](#pone.0111887-ASzczeklik1) and Atopy is lower in AERD than general asthma [[44]](#pone.0111887-Lotvall1). All these parameters were used as covariate. The comparison between the observed and the expected association *p*-values in the quantile-quantile (Q-Q) plot revealed that λ value was 1.0013 ([Fig. 1](#pone-0111887-g001)). The data discussed has been deposited in NCBI’s Gene Expression Omnibus [[45]](#pone.0111887-Edgar1) and is accessible through GEO Series accession number [GSE61129](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE61129) ([http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE61129](http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE61129)).

### Figure 1. QQ-plot analysis of 54,555 SNPs between for 165 AERD vs. 397 ATA.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4f25/4221198/ce32228f95e7/pone.0111887.g001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4221198_pone.0111887.g001.jpg)

### Association of the exonic SNPs with the risk of AERD

The *p*-values of all SNPs in association with the risk of AERD under the additive mode are presented in a Manhattan plot ([Fig. 2](#pone-0111887-g002)). The exm537513 on *HLA-DPB1* showed the lowest *p*-value of 3.4×10^−8^ (OR: 3.28,) in association with AERD. The *p*-value remained significant after multiple comparisons using Bonferroni’s correction (*p* = 0.0082).

### Figure 2. Manhattan plot of the p-values of 54,555 autosomal SNPs in association with AERD.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4f25/4221198/c51bf321e509/pone.0111887.g002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4221198_pone.0111887.g002.jpg)

The top 100 SNPs are presented according to their p-values in [Table S1](#pone.0111887.s003). Sixteen of the top 20 SNPs were located on chromosome 6. Among them, six SNPs were exonic and located within the *HLA* genes (three in *HLA-DPB1*, two in *HLA-DPA1* and one in *HLA-DPB2*). The remaining 10 SNPs were located at/near a pseudogene or an intron or within an intergenic area ([Table S1](#pone.0111887.s003)). There was strong linkage disequilibrium (LD) between the six exonic SNPs in the *HLA* genes, although not complete LD ([Table S2](#pone.0111887.s004)). AUCs for the six SNP combinations (2^6^− 1) were calculated by multiple logistic regression ([Table S3](#pone.0111887.s005)). A combination of four SNPs (exm537513, exm537522, exm-rs3097671, and exm-rs3129294) had the best AUC value of 0.629.

To determine the top 10 SNPs, the top six SNPs sorted by *p*-value were added to the four *HLA* SNPs ([Table 2](#pone-0111887-t002)). The *p*-values of the top 10 SNPs ranged from 3.4×10^−8^ 2.4×10^−4^ with ORs ranging from 0.13∼13.61 ([Table 2](#pone-0111887-t002)). Among them, *THRA* and *SLC30A9* were rare variants with a frequency <1% in one or more of the study groups. ORs of these two SNPs were significantly higher or lower (*THRA*, 13.6 and *SLC30A9*, 0.13) compared with those of the other common variants with a MAF >5% (1.79< OR<3.28 or OR = 0.57). The AUC of the top 10 SNPs ranged from 0.53∼0.6. The top 100 SNPs with lower *p*-values between AERD and ATA are listed in [Table S1](#pone.0111887.s003). Sixty SNPs had *p*-values<0.001 between AERD and ATA. MAFs of these SNPs in subjects with ATA were almost similar to those of the normal controls ([Table S1](#pone.0111887.s003) and [Table S2](#pone.0111887.s004)).

### Table 2. List of 10 SNPs with low p-values associated with AERD.

| SNPid | Gene(s) | Chr | Allele | Mutation(s) | MAF |  |  | AERD vs. ATA |  | AUC | AERD vs. NC |  | ATA vs. NC |  | statisticalpower | HWP |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
|   |   |   |   |   | AERD (n = 165) | ATA (n = 397) | NC (n = 398) | OR (95% CI) | p-value |   | OR(95% CI) | p-value | OR(95% CI) | p-value |   |   |
| exm537513* | HLA-DPB1 | 6 | A>C | Missense_I94L | 0.167 | 0.057 | 0.062 | 3.28(2.13–5.04) | 3.40E-08 | 0.60 | 3.20(1.98–5.17) | 9.23E-07 | 1.01(0.66–1.56) | 9.60E-01 | 0.77 | 0.427 |
| exm537522 | HLA-DPB1 | 6 | A>G | Missense_M10V | 0.194 | 0.087 | 0.085 | 2.50(1.71–3.67) | 2.38E-06 | 0.59 | 3.20(1.98–5.17) | 9.23E-07 | 1.07(0.73–1.55) | 7.31E-01 | 0.81 | 0.372 |
| exm-rs3097671 | HLA-DPA1 | 6 | G>C | Silent | 0.191 | 0.088 | 0.084 | 2.47(1.69–3.62) | 3.19E-06 | 0.59 | 2.59(1.68–3.99) | 1.05E-05 | 1.10(0.76–1.60) | 6.19E-01 | 0.81 | 0.317 |
| exm-rs3129294 | HLA-DPB2 | 6 | T>G | Silent | 0.324 | 0.207 | 0.206 | 1.91(1.41–2.58) | 2.75E-05 | 0.60 | 1.85(1.32–2.60) | 3.42E-04 | 0.96(0.74–1.24) | 7.59E-01 | 0.9 | 0.902 |
| exm83523* | CHIA | 1 | G>A | Missense_G102R | 0.133 | 0.064 | 0.065 | 2.55(1.64–3.95) | 3.00E-05 | 0.56 | 2.02(1.23–3.31) | 4.98E-03 | 0.91(0.59–1.42) | 6.86E-01 | 0.7 | 0.039 |
| exm1884673* | THRA | 17 | A>G | Missense_I170V | 0.033 | 0.003 | 0.006 | 13.61(2.89–63.96) | 6.20E-05 | 0.53 | 5.30(1.40–20.12) | 9.39E-03 | 0.54(0.09–3.18) | 4.86E-01 | 0.28 | 0.658 |
| exm538564* | ZBTB22 | 6 | T>C | Missense_T310A | 0.385 | 0.273 | 0.265 | 1.79(1.33–2.39) | 9.25E-05 | 0.59 | 2.39(1.69–3.39) | 4.51E-07 | 1.08(0.85–1.39) | 5.28E-01 | 0.85 | 0.423 |
| exm2264237* | LOC401320 | 7 | A>C | Silent | 0.330 | 0.438 | 0.436 | 0.57(0.42–0.76) | 1.10E-04 | 0.58 | 0.56(0.41–0.77) | 3.02E-04 | 1.03(0.83–1.29) | 7.79E-01 | 0.91 | 1.000 |
| exm396794* | SLC30A9 | 4 | G>A | Missense_M50V | 0.009 | 0.052 | 0.040 | 0.13(0.03–0.53) | 1.40E-04 | 0.54 | 0.13(0.03–0.60) | 9.48E-04 | 1.50(0.89–2.54) | 1.26E-01 | 0.11 | 0.906 |
| exm791954* | TSC1 | 9 | T>C | Synonymous_E444E | 0.130 | 0.063 | 0.062 | 2.38(1.50–3.76) | 2.40E-04 | 0.56 | 2.35(1.38–4.02) | 1.81E-03 | 1.05(0.67–1.64) | 8.23E-01 | 0.7 | 0582 |

Table 2 Caption: *Indicates the 7 SNPs that were part of a model.

### Multiple logistic regression and ROC curve analysis of the top 10 SNPs

A multiple logistic regression analysis was undertaken for all power sets composed of the 10 SNPs under the additive mode. ROC curves and the AUC were calculated for the 1,023 (2^10^−1) combinations of the 10 SNPs. The AUC values and *p*-values of the 1,023 models are presented in [Table S4](#pone.0111887.s006). The 192 candidate models of 1,023 models turned out to be that all *p*-values of the coefficients were less than 0.05. One model of 192 models, composed of seven SNPs (exm537513, exm83523, exm1884673, exm538564, exm2264237, exm396794, and exm791954; marked by an ‘*’ in [Table 2](#pone-0111887-t002)), had the highest AUC of 0.75 (an asymptotic *p*-value of 7.94×10^−21^ and an asymptotic 95% confidence interval of 0.706−0.795) ([Fig. 3](#pone-0111887-g003)).

### Figure 3. ROC curve of the best AUC model consisting of 7 SNPs.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4f25/4221198/1c641da1f29a/pone.0111887.g003.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4221198_pone.0111887.g003.jpg)

The results of the contingency using the seven SNPs are presented in [Table 3](#pone-0111887-t003). The combination of these SNPs had 34% sensitivity and 93% specificity to discriminate AERD from ATA. The results showed a positive predictive value of 68.2% and a negative predictive value of 77.1% with an accuracy of 76%. The result of 5-fold cross-validation for the combination model of 7 SNPs (exm537513, exm83523, exm1884673, exm538564, exm2264237, exm396794, and exm791954) showed 21.6–47.2% sensitivity (mean: 30.4%) and 90.7–97.4% specificity (mean: 93.6%), which were comparable to those obtained using multiple logistic regressions.

### Table 3. The contingency of the seven SNPs that formed the model.

|   | Actual value |  | Sensitivity | Specificity |
| --- | --- | --- | --- | --- |
| Predicted outcome | ATA | AERD |   |   |
| ATA | 368 | 110 | 33% | 93% |
| AERD | 29 | 55 |   |   |
### Functional implication of SNP alleles and structure effect of the selected seven SNPs

CpG Islands, DNAse I hypersensitive sites, and microRNA target sites were evaluated using the SNP functional Portal ([http://brainarray.mbni.med.umich.edu/Brainarray/Database/SearchSNP/snpfunc.aspx](http://brainarray.mbni.med.umich.edu/Brainarray/Database/SearchSNP/snpfunc.aspx)). Any of the seven SNPs were not related with these sites. Of the seven exonic SNPs, five were missense SNPs leading to changes of amino acids, or changes in side-chain polarity and charges ([Table 4](#pone-0111887-t004)). The exm83523 was predicted to be “probably damaging” with the probability of 1 PloyPhen2 score due to changes in polarity and charge. The exm538564 was predicted to have different polarity from polar to non-polar change. The other three SNPs were predicted no change in either charge or polarity.

### Table 4. The structural change and amino acid properties change of five missense SNPs.

| Exomid | SNPid | Gene | AA Change | Polyphen2 | The change of side-chain property |  |
| --- | --- | --- | --- | --- | --- | --- |
|   |   |   |   | Score | Polarity change | Charge change |
| exm537513 | rs1042136 | HLA-DPB1 | I→L | 0 | Nonpolar → nonpolar | Neutral → neutral |
| exm83523 | rs3818822 | CHIA | G→R | 1 | Polar → Basic polar | Neutral → positive |
| exm1884673 | rs118191745 | THRA | I→V | 0 | Nonpolar → nonpolar | Neutral → neutral |
| exm538564 | rs3130100 | ZBTB22 | T→A | 0 | Polar → nonpolar | Neutral → neutral |
| exm396794 | rs1047626 | SLC30A9 | M→V | 0 | Nonpolar → nonpolar | Neutral → neutral |
## Discussion

We analyzed the Exome BeadChip data for a total of 562 asthmatics (AERD, 165; ATA, 397) and 398 normal controls. The comparison between the observed and the expected association *p*-values in the Q-Q plot revealed a λ value of 1.0013, indicating the absence of stratification of the study population and the presence of SNPs with distinct deviations from the expected.

A previous GWAS study [[31]](#pone.0111887-Park5), showed that the *HLA-DPB1* rs1042151 (Met105Val) SNP was most significantly associated with susceptibility to AERD with an OR of 2.40. In the present study, three exonic SNPs on *HLA-DPB1* occurred in the top 20 SNPs: exm537513, exm537522 and exm537523. Exm537513 on *HLA-DPB1* had the lowest *p*-value (3.4×10^−8^; OR, 3.28) and, to the best of our knowledge, is a novel SNP associated with AERD. The exm537513 SNP was not amongst the SNPs in our previous GWAS study [[31]](#pone.0111887-Park5), even though the MAF was >5% (6% in NC and ATA and 17% in AERD). The exm537522 SNP was the same as *HLA-DPB1* rs1042151 in our previous GWAS data.

Because the AUC value of exm537513 on *HLA-DPB1* was 0.60, with very low sensitivity (3.6%) for predicting AERD, multiple logistic regression was performed using six exonic SNPs in the *HLA* genes (three in *HLA-DPB1*, two in *HLA-DPA1* and one in *HLA-DPB2*) present in the top 20 exonic SNPs. The combination of four SNPs (exm537513, exm537522, exm-rs3097671, and exm-rs3129294) raised the AUC to 0.629, although the combination of all six SNPs showed an AUC of 0.625. This may be due to the strong LD between exm537485 and exm537522 (D’ = 1) and between exm537522 and exm537523 (D’ = 1), as shown in [Table S2](#pone.0111887.s004), since they are located very close to each other; 204bps between exm537485 and exm537522 and 2bps between exm537523 and exm537522.

To further enhance the prediction of AERD, the six SNPs were combined with the four SNPs in the *HLA* gene, and the AUC was calculated for the 1,023 (2^10^−1) combinations. The greater the number of SNPs used for the analysis, the higher the possible predictive combination. However, combinations of more than 10 SNPs were impractical. Here, one model composed of seven SNPs showed the highest AUC of 0.75, representing a novel combination to predict AERD. To validate the best prediction model of the 7 SNPs obtained by using multiple logistic regressions, 5-fold cross-validation was applied. The sensitivity and specificity to discriminate AERD from ATA were comparable between the two methods. This data suggests that the best prediction of combination model of 7 SNPs seem to be reliable. In the present study, predictive modeling for discrimination of AERD from ATA had the high specificity but the low sensitivity. This may be due to small number of AERD compared to ATA patients in this study. When a balanced sample (AERD: 165, ATA: 165) is used for the prediction model consisting of the 7 SNPs (exm537513, exm83523, exm1884673, exm538564, exm2264237, exm396794, and exm791954), the prediction of AERD is 67% sensitivity and 73% specificity. The result shows increase in the sensitivity while decrease in specificity more than that of the best model using unbalanced samples (AERD: 165, ATA: 397). Thus, the prediction model in the present study may increase the sensitivity when the more AERD patients are included, and our prediction model may be reliable at the moment.

*HLA-DPB1* has a strong link with asthma, ATA, AERD and inflammation [[13]](#pone.0111887-Dekker1), [[31]](#pone.0111887-Park5), [[46]](#pone.0111887-Park6)–[[48]](#pone.0111887-Kim10). To date, 873 SNPs in *HLA-DPB1* and 1,146 SNPs in *HLA-DPA1* have been identified ([http://www.ncbi.nlm.nih.gov/snp/?term=HLA-DPB1+and+human](http://www.ncbi.nlm.nih.gov/snp/?term=HLA-DPB1+and+human), [http://www.ncbi.nlm.nih.gov/snp/?term=HLA-DPA1+and+human](http://www.ncbi.nlm.nih.gov/snp/?term=HLA-DPA1+and+human)). Among them, 260 and 205 exonic SNPs are present in *HLA-DPB1* and *HLA-DPA1*, respectively. In the present study, 12 SNPs in the *HLA-DPB1* and 20 SNPs in the *HLA-DPA1* were investigated in association with AERD. This indicates that the number of SNPs on the chip used in the present study is insufficient to cover all SNPs identified to date. This minimal coverage may be a limitation in the identification of AERD- associated rare variants.

Recently, rare variants were analyzed in an association study to detect SNPs with higher impacts on the genetic risk for multi-complex diseases [[49]](#pone.0111887-Schork1). Among the 54,555 SNPs investigated in the present study, 6,878 SNPs (12.6%) were found to have a MAF<1% in AERD, ATA, or NC. In the top 10 SNPs, *THRA* and *SLC30A9* was rare variants with a frequency <1% in one or two of the study groups. The AUC values of these SNPs were lower than those of the other eight top SNPs, with a MAF>5%. This indicates that rare variants may have less discrimination power than common variants for the prediction of AERD.

*HLA-DP* is an αβ-heterodimer cell-surface receptor. The α-helical domain forms the side of the peptide binding groove. The β-sheet regions form the base of the binding groove, the bulk of the molecule, as well as the inter-subunit binding region [[50]](#pone.0111887-Dai1). In the present study, five SNPs in *HLA-DP2* (three exonic SNPs in *HLA-DPB1* and two exonic SNPs in *HLA-DPA1*) were among the top 20 SNPs. There were four missense SNPs, and three of these (exm537513, exm537522, and exm537523) were located in the binding site of the small peptides ([Figure S2](#pone.0111887.s002)), with amino acid changes at these loci potentially changing the binding affinity of antigens related to AERD.

The functional relationship of the remaining six top genes associated with AERD has not yet been clarified. Interleukin 13 is induced by *CHIA*, variations of which may lead to asthma susceptibility [[51]](#pone.0111887-Birben1). *THRA*, also known as nuclear receptor subfamily 1, group A, member 1 (NR1A1), is a nuclear receptor protein [[52]](#pone.0111887-Spurr1). *TSC1* inhibits the nutrient-mediated or growth factor-stimulated phosphorylation of S6K1 and EIF4EBP1 by negatively regulating mTORC1 signaling [[53]](#pone.0111887-Tee1). The function of the *ZBTB22, LOC401320* and *SLC30A9* genes has not been identified in the PubMed database ([http://www.ncbi.nlm.nih.gov/pubmed](http://www.ncbi.nlm.nih.gov/pubmed)).

The SNP functional Portal was used to find out the effect on the transcription of the selected seven SNPs for CpG Island, DNAse I hypersensitive site, microRNA target site. The result showed that they had no effect on gene expression. The expected impacts of five missense site of the seven exonic SNPs were evaluated on the proteins structure and function. Among them, exm83523 *CHIA* showed changes in side-chain polarity (polar → Basic polar) and in side-chain charge (neutral → positive) showing the score of “probably damaging”. This data suggest that exm83523 *CHIA* might have the greatest impact on the function of *CHIA* protein.

We have previously reported the results of a 600K GWAS in association with the risk of AERD [[54]](#pone.0111887-Shin1). When the nine genes of the top 10 SNPs in the present exome study were searched among the top 100 genes of the 600K GWAS data ([Table S5](#pone.0111887.s007)), only two genes (*HLA-DPB2* and *TSC1*) were identified. This may be due to a minimal overlap of the 600K GWAS and the Exome BeadChip. There was an overlap of only 9,932 SNPs between the Exome BeadChip (4.09% of 242985 SNPs) and the GWAS chip (2.31% of 430487 SNPs). Another limitation of this study was as follows. First, no replication study was performed, and the number of study subjects was relatively small to detect significance of rare variants. Second, a functional study for the candidate SNPs was not performed. Therefore, a follow-up study for the replication and a functional study of the candidate SNPs would be needed to clarify a relationship between the SNPs and the development of AERD.

## Supporting Information

## Acknowledgments

The DNA samples were generously provided by Soonchunhyang University, Bucheon Hospital Biobank, and a member of the National Biobank of Korea, supported by the Ministry of Health, Welfare and Family Affairs, Republic of Korea.

## Data Availability

The authors confirm that all data underlying the findings are fully available without restriction. For the exmome chip data, the data discussed in this publication have been deposited in NCBI’s Gene Expression Omnibus (Edgar et al., 2002) and are accessible through GEO Series accession number GSE61129 ([http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE61129](http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE61129)).

## Funding Statement

This study was supported by a grant from the Korea Healthcare Technology R& D Project, Ministry for Health, Welfare & Family Affairs, and Republic of Korea (HI13C0319). The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Data Availability Statement

The authors confirm that all data underlying the findings are fully available without restriction. For the exmome chip data, the data discussed in this publication have been deposited in NCBI’s Gene Expression Omnibus (Edgar et al., 2002) and are accessible through GEO Series accession number GSE61129 ([http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE61129](http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE61129)).

### Supplementary Materials

### Data Availability Statement

The authors confirm that all data underlying the findings are fully available without restriction. For the exmome chip data, the data discussed in this publication have been deposited in NCBI’s Gene Expression Omnibus (Edgar et al., 2002) and are accessible through GEO Series accession number GSE61129 ([http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE61129](http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE61129)).

## References

1. Lee RU, Stevenson DD (2011) Aspirin-exacerbated respiratory disease: evaluation and management. Allergy Asthma Immunol Res 3: 3–10.  [DOI](https://doi.org/10.4168/aair.2011.3.1.3) | [PMC free article](/articles/PMC3005316/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21217919/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Allergy%20Asthma%20Immunol%20Res&title=Aspirin-exacerbated%20respiratory%20disease:%20evaluation%20and%20management&author=RU%20Lee&author=DD%20Stevenson&volume=3&publication_year=2011&pages=3-10&pmid=21217919&doi=10.4168/aair.2011.3.1.3&)

2. Szczeklik A, Nizankowska E, Duplaga M (2000) Natural history of aspirin-induced asthma. AIANE Investigators. European Network on Aspirin-Induced Asthma. Eur Respir J 16: 432–436.  [DOI](https://doi.org/10.1034/j.1399-3003.2000.016003432.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11028656/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20Respir%20J&title=Natural%20history%20of%20aspirin-induced%20asthma.%20AIANE%20Investigators.%20European%20Network%20on%20Aspirin-Induced%20Asthma&author=A%20Szczeklik&author=E%20Nizankowska&author=M%20Duplaga&volume=16&publication_year=2000&pages=432-436&pmid=11028656&doi=10.1034/j.1399-3003.2000.016003432.x&)

3. Hedman J, Kaprio J, Poussa T, Nieminen MM (1999) Prevalence of asthma, aspirin intolerance, nasal polyposis and chronic obstructive pulmonary disease in a population-based study. Int J Epidemiol 28: 717–722.  [DOI](https://doi.org/10.1093/ije/28.4.717) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10480701/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Epidemiol&title=Prevalence%20of%20asthma,%20aspirin%20intolerance,%20nasal%20polyposis%20and%20chronic%20obstructive%20pulmonary%20disease%20in%20a%20population-based%20study&author=J%20Hedman&author=J%20Kaprio&author=T%20Poussa&author=MM%20Nieminen&volume=28&publication_year=1999&pages=717-722&pmid=10480701&doi=10.1093/ije/28.4.717&)

4. Sanak M, Pierzchalska M, Bazan-Socha S, Szczeklik A (2000) Enhanced expression of the leukotriene C(4) synthase due to overactive transcription of an allelic variant associated with aspirin-intolerant asthma. Am J Respir Cell Mol Biol 23: 290–296.  [DOI](https://doi.org/10.1165/ajrcmb.23.3.4051) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10970818/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Respir%20Cell%20Mol%20Biol&title=Enhanced%20expression%20of%20the%20leukotriene%20C(4)%20synthase%20due%20to%20overactive%20transcription%20of%20an%20allelic%20variant%20associated%20with%20aspirin-intolerant%20asthma&author=M%20Sanak&author=M%20Pierzchalska&author=S%20Bazan-Socha&author=A%20Szczeklik&volume=23&publication_year=2000&pages=290-296&pmid=10970818&doi=10.1165/ajrcmb.23.3.4051&)

5. Choi JH, Park HS, Oh HB, Lee JH, Suh YJ, et al. (2004) Leukotriene-related gene polymorphisms in ASA-intolerant asthma: an association with a haplotype of 5-lipoxygenase. Hum Genet 114: 337–344.  [DOI](https://doi.org/10.1007/s00439-004-1082-1) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14749922/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Genet&title=Leukotriene-related%20gene%20polymorphisms%20in%20ASA-intolerant%20asthma:%20an%20association%20with%20a%20haplotype%20of%205-lipoxygenase&author=JH%20Choi&author=HS%20Park&author=HB%20Oh&author=JH%20Lee&author=YJ%20Suh&volume=114&publication_year=2004&pages=337-344&pmid=14749922&doi=10.1007/s00439-004-1082-1&)

6. Kim SH, Oh JM, Kim YS, Palmer LJ, Suh CH, et al. (2006) Cysteinyl leukotriene receptor 1 promoter polymorphism is associated with aspirin-intolerant asthma in males. Clin Exp Allergy 36: 433–439.  [DOI](https://doi.org/10.1111/j.1365-2222.2006.02457.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16630147/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Exp%20Allergy&title=Cysteinyl%20leukotriene%20receptor%201%20promoter%20polymorphism%20is%20associated%20with%20aspirin-intolerant%20asthma%20in%20males&author=SH%20Kim&author=JM%20Oh&author=YS%20Kim&author=LJ%20Palmer&author=CH%20Suh&volume=36&publication_year=2006&pages=433-439&pmid=16630147&doi=10.1111/j.1365-2222.2006.02457.x&)

7. Park JS, Chang HS, Park CS, Lee JH, Lee YM, et al. (2005) Association analysis of cysteinyl-leukotriene receptor 2 (CYSLTR2) polymorphisms with aspirin intolerance in asthmatics. Pharmacogenet Genomics 15: 483–492.  [DOI](https://doi.org/10.1097/01.fpc.0000166456.84905.a0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15970796/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Association%20analysis%20of%20cysteinyl-leukotriene%20receptor%202%20(CYSLTR2)%20polymorphisms%20with%20aspirin%20intolerance%20in%20asthmatics&author=JS%20Park&author=HS%20Chang&author=CS%20Park&author=JH%20Lee&author=YM%20Lee&volume=15&publication_year=2005&pages=483-492&pmid=15970796&doi=10.1097/01.fpc.0000166456.84905.a0&)

8. Jinnai N, Sakagami T, Sekigawa T, Kakihara M, Nakajima T, et al. (2004) Polymorphisms in the prostaglandin E2 receptor subtype 2 gene confer susceptibility to aspirin-intolerant asthma: a candidate gene approach. Hum Mol Genet 13: 3203–3217.  [DOI](https://doi.org/10.1093/hmg/ddh332) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15496426/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mol%20Genet&title=Polymorphisms%20in%20the%20prostaglandin%20E2%20receptor%20subtype%202%20gene%20confer%20susceptibility%20to%20aspirin-intolerant%20asthma:%20a%20candidate%20gene%20approach&author=N%20Jinnai&author=T%20Sakagami&author=T%20Sekigawa&author=M%20Kakihara&author=T%20Nakajima&volume=13&publication_year=2004&pages=3203-3217&pmid=15496426&doi=10.1093/hmg/ddh332&)

9. Kim SH, Kim YK, Park HW, Jee YK, Bahn JW, et al. (2007) Association between polymorphisms in prostanoid receptor genes and aspirin-intolerant asthma. Pharmacogenet Genomics 17: 295–304.  [DOI](https://doi.org/10.1097/01.fpc.0000239977.61841.fe) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17496729/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Association%20between%20polymorphisms%20in%20prostanoid%20receptor%20genes%20and%20aspirin-intolerant%20asthma&author=SH%20Kim&author=YK%20Kim&author=HW%20Park&author=YK%20Jee&author=JW%20Bahn&volume=17&publication_year=2007&pages=295-304&pmid=17496729&doi=10.1097/01.fpc.0000239977.61841.fe&)

10. Park BL, Park SM, Park JS, Uh ST, Choi JS, et al. (2010) Association of PTGER gene family polymorphisms with aspirin intolerant asthma in Korean asthmatics. BMB Rep 43: 445–449.  [DOI](https://doi.org/10.5483/bmbrep.2010.43.6.445) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20587336/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMB%20Rep&title=Association%20of%20PTGER%20gene%20family%20polymorphisms%20with%20aspirin%20intolerant%20asthma%20in%20Korean%20asthmatics&author=BL%20Park&author=SM%20Park&author=JS%20Park&author=ST%20Uh&author=JS%20Choi&volume=43&publication_year=2010&pages=445-449&pmid=20587336&doi=10.5483/bmbrep.2010.43.6.445&)

11. Oh SH, Kim YH, Park SM, Cho SH, Park JS, et al. (2011) Association analysis of thromboxane A synthase 1 gene polymorphisms with aspirin intolerance in asthmatic patients. Pharmacogenomics 12: 351–363.  [DOI](https://doi.org/10.2217/pgs.10.181) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21449675/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Association%20analysis%20of%20thromboxane%20A%20synthase%201%20gene%20polymorphisms%20with%20aspirin%20intolerance%20in%20asthmatic%20patients&author=SH%20Oh&author=YH%20Kim&author=SM%20Park&author=SH%20Cho&author=JS%20Park&volume=12&publication_year=2011&pages=351-363&pmid=21449675&doi=10.2217/pgs.10.181&)

12. Kim SH, Choi JH, Park HS, Holloway JW, Lee SK, et al. (2005) Association of thromboxane A2 receptor gene polymorphism with the phenotype of acetyl salicylic acid-intolerant asthma. Clin Exp Allergy 35: 585–590.  [DOI](https://doi.org/10.1111/j.1365-2222.2005.02220.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15898979/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Exp%20Allergy&title=Association%20of%20thromboxane%20A2%20receptor%20gene%20polymorphism%20with%20the%20phenotype%20of%20acetyl%20salicylic%20acid-intolerant%20asthma&author=SH%20Kim&author=JH%20Choi&author=HS%20Park&author=JW%20Holloway&author=SK%20Lee&volume=35&publication_year=2005&pages=585-590&pmid=15898979&doi=10.1111/j.1365-2222.2005.02220.x&)

13. Dekker JW, Nizankowska E, Schmitz-Schumann M, Pile K, Bochenek G, et al. (1997) Aspirin-induced asthma and HLA-DRB1 and HLA-DPB1 genotypes. Clin Exp Allergy 27: 574–577.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/9179433/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Exp%20Allergy&title=Aspirin-induced%20asthma%20and%20HLA-DRB1%20and%20HLA-DPB1%20genotypes&author=JW%20Dekker&author=E%20Nizankowska&author=M%20Schmitz-Schumann&author=K%20Pile&author=G%20Bochenek&volume=27&publication_year=1997&pages=574-577&pmid=9179433&)

14. Kim BS, Park SM, Uhm TG, Kang JH, Park JS, et al. (2010) Effect of single nucleotide polymorphisms within the interleukin-4 promoter on aspirin intolerance in asthmatics and interleukin-4 promoter activity. Pharmacogenet Genomics 20: 748–758.  [DOI](https://doi.org/10.1097/FPC.0b013e3283402155) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20921925/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Effect%20of%20single%20nucleotide%20polymorphisms%20within%20the%20interleukin-4%20promoter%20on%20aspirin%20intolerance%20in%20asthmatics%20and%20interleukin-4%20promoter%20activity&author=BS%20Kim&author=SM%20Park&author=TG%20Uhm&author=JH%20Kang&author=JS%20Park&volume=20&publication_year=2010&pages=748-758&pmid=20921925&doi=10.1097/FPC.0b013e3283402155&)

15. Akahoshi M, Obara K, Hirota T, Matsuda A, Hasegawa K, et al. (2005) Functional promoter polymorphism in the TBX21 gene associated with aspirin-induced asthma. Hum Genet 117: 16–26.  [DOI](https://doi.org/10.1007/s00439-005-1285-0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15806396/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Genet&title=Functional%20promoter%20polymorphism%20in%20the%20TBX21%20gene%20associated%20with%20aspirin-induced%20asthma&author=M%20Akahoshi&author=K%20Obara&author=T%20Hirota&author=A%20Matsuda&author=K%20Hasegawa&volume=117&publication_year=2005&pages=16-26&pmid=15806396&doi=10.1007/s00439-005-1285-0&)

16. Kim SH, Bae JS, Holloway JW, Lee JT, Suh CH, et al. (2006) A polymorphism of MS4A2 (−109T>C) encoding the beta-chain of the high-affinity immunoglobulin E receptor (FcepsilonR1beta) is associated with a susceptibility to aspirin-intolerant asthma. Clin Exp Allergy 36: 877–883.  [DOI](https://doi.org/10.1111/j.1365-2222.2006.02443.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16839402/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Exp%20Allergy&title=A%20polymorphism%20of%20MS4A2%20(%E2%88%92109T>C)%20encoding%20the%20beta-chain%20of%20the%20high-affinity%20immunoglobulin%20E%20receptor%20(FcepsilonR1beta)%20is%20associated%20with%20a%20susceptibility%20to%20aspirin-intolerant%20asthma&author=SH%20Kim&author=JS%20Bae&author=JW%20Holloway&author=JT%20Lee&author=CH%20Suh&volume=36&publication_year=2006&pages=877-883&pmid=16839402&doi=10.1111/j.1365-2222.2006.02443.x&)

17. Palikhe NS, Kim SH, Cho BY, Ye YM, Hur GY, et al. (2008) Association of three sets of high-affinity IgE receptor (FcepsilonR1) polymorphisms with aspirin-intolerant asthma. Respir Med 102: 1132–1139.  [DOI](https://doi.org/10.1016/j.rmed.2008.03.017) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18595682/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Respir%20Med&title=Association%20of%20three%20sets%20of%20high-affinity%20IgE%20receptor%20(FcepsilonR1)%20polymorphisms%20with%20aspirin-intolerant%20asthma&author=NS%20Palikhe&author=SH%20Kim&author=BY%20Cho&author=YM%20Ye&author=GY%20Hur&volume=102&publication_year=2008&pages=1132-1139&pmid=18595682&doi=10.1016/j.rmed.2008.03.017&)

18. Palikhe NS, Kim SH, Kim JH, Losol P, Ye YM, et al. (2011) Role of Toll-like Receptor 3 Variants in Aspirin-Exacerbated Respiratory Disease. Allergy Asthma Immunol Res 3: 123–127.  [DOI](https://doi.org/10.4168/aair.2011.3.2.123) | [PMC free article](/articles/PMC3062791/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21461252/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Allergy%20Asthma%20Immunol%20Res&title=Role%20of%20Toll-like%20Receptor%203%20Variants%20in%20Aspirin-Exacerbated%20Respiratory%20Disease&author=NS%20Palikhe&author=SH%20Kim&author=JH%20Kim&author=P%20Losol&author=YM%20Ye&volume=3&publication_year=2011&pages=123-127&pmid=21461252&doi=10.4168/aair.2011.3.2.123&)

19. Hitomi Y, Ebisawa M, Tomikawa M, Imai T, Komata T, et al.. (2009) Associations of functional NLRP3 polymorphisms with susceptibility to food-induced anaphylaxis and aspirin-induced asthma. J Allergy Clin Immunol 124: 779–785 e776.  [DOI](https://doi.org/10.1016/j.jaci.2009.07.044) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19767079/)

20. Sakagami T, Jinnai N, Nakajima T, Sekigawa T, Hasegawa T, et al. (2007) ADAM33 polymorphisms are associated with aspirin-intolerant asthma in the Japanese population. J Hum Genet 52: 66–72.  [DOI](https://doi.org/10.1007/s10038-006-0081-6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17061022/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Hum%20Genet&title=ADAM33%20polymorphisms%20are%20associated%20with%20aspirin-intolerant%20asthma%20in%20the%20Japanese%20population&author=T%20Sakagami&author=N%20Jinnai&author=T%20Nakajima&author=T%20Sekigawa&author=T%20Hasegawa&volume=52&publication_year=2007&pages=66-72&pmid=17061022&doi=10.1007/s10038-006-0081-6&)

21. Kim SH, Kim YK, Park HW, Ye YM, Min KU, et al. (2009) Adenosine deaminase and adenosine receptor polymorphisms in aspirin-intolerant asthma. Respir Med 103: 356–363.  [DOI](https://doi.org/10.1016/j.rmed.2008.10.008) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19019667/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Respir%20Med&title=Adenosine%20deaminase%20and%20adenosine%20receptor%20polymorphisms%20in%20aspirin-intolerant%20asthma&author=SH%20Kim&author=YK%20Kim&author=HW%20Park&author=YM%20Ye&author=KU%20Min&volume=103&publication_year=2009&pages=356-363&pmid=19019667&doi=10.1016/j.rmed.2008.10.008&)

22. Kim TH, Chang HS, Park SM, Nam BY, Park JS, et al. (2008) Association of angiotensin I-converting enzyme gene polymorphisms with aspirin intolerance in asthmatics. Clin Exp Allergy 38: 1727–1737.  [DOI](https://doi.org/10.1111/j.1365-2222.2008.03082.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18727619/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Exp%20Allergy&title=Association%20of%20angiotensin%20I-converting%20enzyme%20gene%20polymorphisms%20with%20aspirin%20intolerance%20in%20asthmatics&author=TH%20Kim&author=HS%20Chang&author=SM%20Park&author=BY%20Nam&author=JS%20Park&volume=38&publication_year=2008&pages=1727-1737&pmid=18727619&doi=10.1111/j.1365-2222.2008.03082.x&)

23. Palikhe NS, Kim SH, Cho BY, Ye YM, Choi GS, et al. (2010) Genetic variability in CRTH2 polymorphism increases eotaxin-2 levels in patients with aspirin exacerbated respiratory disease. Allergy 65: 338–346.  [DOI](https://doi.org/10.1111/j.1398-9995.2009.02158.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19796209/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Allergy&title=Genetic%20variability%20in%20CRTH2%20polymorphism%20increases%20eotaxin-2%20levels%20in%20patients%20with%20aspirin%20exacerbated%20respiratory%20disease&author=NS%20Palikhe&author=SH%20Kim&author=BY%20Cho&author=YM%20Ye&author=GS%20Choi&volume=65&publication_year=2010&pages=338-346&pmid=19796209&doi=10.1111/j.1398-9995.2009.02158.x&)

24. Oh SH, Park SM, Park JS, Jang AS, Lee YM, et al. (2009) Association analysis of peroxisome proliferator-activated receptors gamma gene polymorphisms with asprin hypersensitivity in asthmatics. Allergy Asthma Immunol Res 1: 30–35.  [DOI](https://doi.org/10.4168/aair.2009.1.1.30) | [PMC free article](/articles/PMC2831569/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20224667/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Allergy%20Asthma%20Immunol%20Res&title=Association%20analysis%20of%20peroxisome%20proliferator-activated%20receptors%20gamma%20gene%20polymorphisms%20with%20asprin%20hypersensitivity%20in%20asthmatics&author=SH%20Oh&author=SM%20Park&author=JS%20Park&author=AS%20Jang&author=YM%20Lee&volume=1&publication_year=2009&pages=30-35&pmid=20224667&doi=10.4168/aair.2009.1.1.30&)

25. Kim JH, Cha JY, Cheong HS, Park JS, Jang AS, et al. (2011) KIF3A, a cilia structural gene on chromosome 5q31, and its polymorphisms show an association with aspirin hypersensitivity in asthma. J Clin Immunol 31: 112–121.  [DOI](https://doi.org/10.1007/s10875-010-9462-x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20922562/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Immunol&title=KIF3A,%20a%20cilia%20structural%20gene%20on%20chromosome%205q31,%20and%20its%20polymorphisms%20show%20an%20association%20with%20aspirin%20hypersensitivity%20in%20asthma&author=JH%20Kim&author=JY%20Cha&author=HS%20Cheong&author=JS%20Park&author=AS%20Jang&volume=31&publication_year=2011&pages=112-121&pmid=20922562&doi=10.1007/s10875-010-9462-x&)

26. Pasaje CF, Kim JH, Park BL, Cheong HS, Chun JY, et al. (2010) Association of SLC6A12 variants with aspirin-intolerant asthma in a Korean population. Ann Hum Genet 74: 326–334.  [DOI](https://doi.org/10.1111/j.1469-1809.2010.00584.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20597903/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Hum%20Genet&title=Association%20of%20SLC6A12%20variants%20with%20aspirin-intolerant%20asthma%20in%20a%20Korean%20population&author=CF%20Pasaje&author=JH%20Kim&author=BL%20Park&author=HS%20Cheong&author=JY%20Chun&volume=74&publication_year=2010&pages=326-334&pmid=20597903&doi=10.1111/j.1469-1809.2010.00584.x&)

27. Park TJ, Kim JH, Bae JS, Park BL, Cheong HS, et al. (2011) Possible association of SLC22A2 polymorphisms with aspirin-intolerant asthma. Int Arch Allergy Immunol 155: 395–402.  [DOI](https://doi.org/10.1159/000321267) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21346370/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20Arch%20Allergy%20Immunol&title=Possible%20association%20of%20SLC22A2%20polymorphisms%20with%20aspirin-intolerant%20asthma&author=TJ%20Park&author=JH%20Kim&author=JS%20Bae&author=BL%20Park&author=HS%20Cheong&volume=155&publication_year=2011&pages=395-402&pmid=21346370&doi=10.1159/000321267&)

28. Lee JS, Kim JH, Bae JS, Kim JY, Park TJ, et al. (2010) Association of CACNG6 polymorphisms with aspirin-intolerance asthmatics in a Korean population. BMC Med Genet 11: 138.  [DOI](https://doi.org/10.1186/1471-2350-11-138) | [PMC free article](/articles/PMC2954844/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20860846/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Med%20Genet&title=Association%20of%20CACNG6%20polymorphisms%20with%20aspirin-intolerance%20asthmatics%20in%20a%20Korean%20population&author=JS%20Lee&author=JH%20Kim&author=JS%20Bae&author=JY%20Kim&author=TJ%20Park&volume=11&publication_year=2010&pages=138&pmid=20860846&doi=10.1186/1471-2350-11-138&)

29. Park S-M, Park JS, Park H-S, Park C-S (2013) Unraveling the Genetic Basis of Aspirin Hypersensitivity in Asthma Beyond Arachidonate Pathways. Allergy Asthma Immunol Res 5.  [DOI](https://doi.org/10.4168/aair.2013.5.5.258) | [PMC free article](/articles/PMC3756172/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24003382/)

30. Kim JH, Park BL, Cheong HS, Bae JS, Park JS, et al. (2010) Genome-wide and follow-up studies identify CEP68 gene variants associated with risk of aspirin-intolerant asthma. PLoS One 5: e13818.  [DOI](https://doi.org/10.1371/journal.pone.0013818) | [PMC free article](/articles/PMC2972220/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21072201/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=Genome-wide%20and%20follow-up%20studies%20identify%20CEP68%20gene%20variants%20associated%20with%20risk%20of%20aspirin-intolerant%20asthma&author=JH%20Kim&author=BL%20Park&author=HS%20Cheong&author=JS%20Bae&author=JS%20Park&volume=5&publication_year=2010&pages=e13818&pmid=21072201&doi=10.1371/journal.pone.0013818&)

31. Park BL, Kim TH, Kim JH, Bae JS, Pasaje CF, et al. (2013) Genome-wide association study of aspirin-exacerbated respiratory disease in a Korean population. Hum Genet 132: 313–321.  [DOI](https://doi.org/10.1007/s00439-012-1247-2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23180272/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Genet&title=Genome-wide%20association%20study%20of%20aspirin-exacerbated%20respiratory%20disease%20in%20a%20Korean%20population&author=BL%20Park&author=TH%20Kim&author=JH%20Kim&author=JS%20Bae&author=CF%20Pasaje&volume=132&publication_year=2013&pages=313-321&pmid=23180272&doi=10.1007/s00439-012-1247-2&)

32. Pritchard JK (2001) Are rare variants responsible for susceptibility to complex diseases? Am J Hum Genet 69: 124–137.  [DOI](https://doi.org/10.1086/321272) | [PMC free article](/articles/PMC1226027/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11404818/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=Are%20rare%20variants%20responsible%20for%20susceptibility%20to%20complex%20diseases?&author=JK%20Pritchard&volume=69&publication_year=2001&pages=124-137&pmid=11404818&doi=10.1086/321272&)

33. McCarthy MI, Hirschhorn JN (2008) Genome-wide association studies: potential next steps on a genetic journey. Hum Mol Genet 17: R156–165.  [DOI](https://doi.org/10.1093/hmg/ddn289) | [PMC free article](/articles/PMC2782356/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18852205/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mol%20Genet&title=Genome-wide%20association%20studies:%20potential%20next%20steps%20on%20a%20genetic%20journey&author=MI%20McCarthy&author=JN%20Hirschhorn&volume=17&publication_year=2008&pages=R156-165&pmid=18852205&doi=10.1093/hmg/ddn289&)

34. Illumina website (datasheet_humanexome_beadchips). Available: http://res.illumina.com/documents/products/datasheets/datasheet_humanexome_beadchips.pdf. Accessed 2014 Oct 13.  [http://res.illumina.com/documents/products/datasheets/datasheet_humanexome_beadchips.pdf](http://res.illumina.com/documents/products/datasheets/datasheet_humanexome_beadchips.pdf)

35. Bateman ED, Hurd SS, Barnes PJ, Bousquet J, Drazen JM, et al. (2008) Global strategy for asthma management and prevention: GINA executive summary. Eur Respir J 31: 143–178.  [DOI](https://doi.org/10.1183/09031936.00138707) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18166595/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20Respir%20J&title=Global%20strategy%20for%20asthma%20management%20and%20prevention:%20GINA%20executive%20summary&author=ED%20Bateman&author=SS%20Hurd&author=PJ%20Barnes&author=J%20Bousquet&author=JM%20Drazen&volume=31&publication_year=2008&pages=143-178&pmid=18166595&doi=10.1183/09031936.00138707&)

36. Chang HS, Park JS, Jang AS, Park SW, Uh ST, et al. (2011) Diagnostic value of clinical parameters in the prediction of aspirin-exacerbated respiratory disease in asthma. Allergy Asthma Immunol Res 3: 256–264.  [DOI](https://doi.org/10.4168/aair.2011.3.4.256) | [PMC free article](/articles/PMC3178824/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21966606/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Allergy%20Asthma%20Immunol%20Res&title=Diagnostic%20value%20of%20clinical%20parameters%20in%20the%20prediction%20of%20aspirin-exacerbated%20respiratory%20disease%20in%20asthma&author=HS%20Chang&author=JS%20Park&author=AS%20Jang&author=SW%20Park&author=ST%20Uh&volume=3&publication_year=2011&pages=256-264&pmid=21966606&doi=10.4168/aair.2011.3.4.256&)

37. Nizankowska-Mogilnicka E, Bochenek G, Mastalerz L, Swierczynska M, Picado C, et al. (2007) EAACI/GA2LEN guideline: aspirin provocation tests for diagnosis of aspirin hypersensitivity. Allergy 62: 1111–1118.  [DOI](https://doi.org/10.1111/j.1398-9995.2007.01409.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17521312/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Allergy&title=EAACI/GA2LEN%20guideline:%20aspirin%20provocation%20tests%20for%20diagnosis%20of%20aspirin%20hypersensitivity&author=E%20Nizankowska-Mogilnicka&author=G%20Bochenek&author=L%20Mastalerz&author=M%20Swierczynska&author=C%20Picado&volume=62&publication_year=2007&pages=1111-1118&pmid=17521312&doi=10.1111/j.1398-9995.2007.01409.x&)

38. Wang P, Dai M, Xuan W, McEachin RC, Jackson AU, et al. (2006) SNP Function Portal: a web database for exploring the function implication of SNP alleles. Bioinformatics 22: e523–529.  [DOI](https://doi.org/10.1093/bioinformatics/btl241) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16873516/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=SNP%20Function%20Portal:%20a%20web%20database%20for%20exploring%20the%20function%20implication%20of%20SNP%20alleles&author=P%20Wang&author=M%20Dai&author=W%20Xuan&author=RC%20McEachin&author=AU%20Jackson&volume=22&publication_year=2006&pages=e523-529&pmid=16873516&doi=10.1093/bioinformatics/btl241&)

39. Adzhubei IA, Schmidt S, Peshkin L, Ramensky VE, Gerasimova A, et al. (2010) A method and server for predicting damaging missense mutations. Nat Methods 7: 248–249.  [DOI](https://doi.org/10.1038/nmeth0410-248) | [PMC free article](/articles/PMC2855889/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20354512/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Methods&title=A%20method%20and%20server%20for%20predicting%20damaging%20missense%20mutations&author=IA%20Adzhubei&author=S%20Schmidt&author=L%20Peshkin&author=VE%20Ramensky&author=A%20Gerasimova&volume=7&publication_year=2010&pages=248-249&pmid=20354512&doi=10.1038/nmeth0410-248&)

40. Eisner MD, Iribarren C (2007) The influence of cigarette smoking on adult asthma outcomes. Nicotine Tob Res 9: 53–56.  [DOI](https://doi.org/10.1080/14622200601078293) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17365736/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nicotine%20Tob%20Res&title=The%20influence%20of%20cigarette%20smoking%20on%20adult%20asthma%20outcomes&author=MD%20Eisner&author=C%20Iribarren&volume=9&publication_year=2007&pages=53-56&pmid=17365736&doi=10.1080/14622200601078293&)

41. Stevenson DD, Szczeklik A (2006) Clinical and pathologic perspectives on aspirin sensitivity and asthma. J Allergy Clin Immunol 118: 773–786; quiz 787–778.  [DOI](https://doi.org/10.1016/j.jaci.2006.07.024) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17030227/)

42. Berges-Gimeno MP, Simon RA, Stevenson DD (2002) The natural history and clinical characteristics of aspirin-exacerbated respiratory disease. Ann Allergy Asthma Immunol 89: 474–478.  [DOI](https://doi.org/10.1016/S1081-1206(10)62084-4) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12452205/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Allergy%20Asthma%20Immunol&title=The%20natural%20history%20and%20clinical%20characteristics%20of%20aspirin-exacerbated%20respiratory%20disease&author=MP%20Berges-Gimeno&author=RA%20Simon&author=DD%20Stevenson&volume=89&publication_year=2002&pages=474-478&pmid=12452205&doi=10.1016/S1081-1206(10)62084-4&)

43. A. Szczeklik EN, M Duplaga (2000) Natural history of aspirin-induced asthma. Eur Respir J 16: 432–436.  [DOI](https://doi.org/10.1034/j.1399-3003.2000.016003432.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11028656/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20Respir%20J&title=Natural%20history%20of%20aspirin-induced%20asthma&author=EN%20A.%20Szczeklik&author=Duplaga%20M&volume=16&publication_year=2000&pages=432-436&pmid=11028656&doi=10.1034/j.1399-3003.2000.016003432.x&)

44. Lotvall J, Akdis CA, Bacharier LB, Bjermer L, Casale TB, et al. (2011) Asthma endotypes: a new approach to classification of disease entities within the asthma syndrome. J Allergy Clin Immunol 127: 355–360.  [DOI](https://doi.org/10.1016/j.jaci.2010.11.037) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21281866/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Allergy%20Clin%20Immunol&title=Asthma%20endotypes:%20a%20new%20approach%20to%20classification%20of%20disease%20entities%20within%20the%20asthma%20syndrome&author=J%20Lotvall&author=CA%20Akdis&author=LB%20Bacharier&author=L%20Bjermer&author=TB%20Casale&volume=127&publication_year=2011&pages=355-360&pmid=21281866&doi=10.1016/j.jaci.2010.11.037&)

45. Edgar R, Domrachev M, Lash AE (2002) Gene Expression Omnibus: NCBI gene expression and hybridization array data repository. Nucleic Acids Res 30: 207–210.  [DOI](https://doi.org/10.1093/nar/30.1.207) | [PMC free article](/articles/PMC99122/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11752295/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=Gene%20Expression%20Omnibus:%20NCBI%20gene%20expression%20and%20hybridization%20array%20data%20repository&author=R%20Edgar&author=M%20Domrachev&author=AE%20Lash&volume=30&publication_year=2002&pages=207-210&pmid=11752295&doi=10.1093/nar/30.1.207&)

46. Park HS, Kim SH, Sampson AP, Lee KW, Park CS (2004) The HLA-DPB1*0301 marker might predict the requirement for leukotriene receptor antagonist in patients with aspirin-intolerant asthma. J Allergy Clin Immunol 114: 688–689.  [DOI](https://doi.org/10.1016/j.jaci.2004.05.051) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15446291/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Allergy%20Clin%20Immunol&title=The%20HLA-DPB1*0301%20marker%20might%20predict%20the%20requirement%20for%20leukotriene%20receptor%20antagonist%20in%20patients%20with%20aspirin-intolerant%20asthma&author=HS%20Park&author=SH%20Kim&author=AP%20Sampson&author=KW%20Lee&author=CS%20Park&volume=114&publication_year=2004&pages=688-689&pmid=15446291&doi=10.1016/j.jaci.2004.05.051&)

47. Noguchi E, Sakamoto H, Hirota T, Ochiai K, Imoto Y, et al. (2011) Genome-wide association study identifies HLA-DP as a susceptibility gene for pediatric asthma in Asian populations. PLoS Genet 7: e1002170.  [DOI](https://doi.org/10.1371/journal.pgen.1002170) | [PMC free article](/articles/PMC3140987/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21814517/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet&title=Genome-wide%20association%20study%20identifies%20HLA-DP%20as%20a%20susceptibility%20gene%20for%20pediatric%20asthma%20in%20Asian%20populations&author=E%20Noguchi&author=H%20Sakamoto&author=T%20Hirota&author=K%20Ochiai&author=Y%20Imoto&volume=7&publication_year=2011&pages=e1002170&pmid=21814517&doi=10.1371/journal.pgen.1002170&)

48. Kim SH, Choi JH, Lee KW, Shin ES, Oh HB, et al. (2005) The human leucocyte antigen-DRB1*1302-DQB1*0609-DPB1*0201 haplotype may be a strong genetic marker for aspirin-induced urticaria. Clin Exp Allergy 35: 339–344.  [DOI](https://doi.org/10.1111/j.1365-2222.2004.02197.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15784113/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Exp%20Allergy&title=The%20human%20leucocyte%20antigen-DRB1*1302-DQB1*0609-DPB1*0201%20haplotype%20may%20be%20a%20strong%20genetic%20marker%20for%20aspirin-induced%20urticaria&author=SH%20Kim&author=JH%20Choi&author=KW%20Lee&author=ES%20Shin&author=HB%20Oh&volume=35&publication_year=2005&pages=339-344&pmid=15784113&doi=10.1111/j.1365-2222.2004.02197.x&)

49. Schork NJ, Murray SS, Frazer KA, Topol EJ (2009) Common vs. rare allele hypotheses for complex diseases. Curr Opin Genet Dev 19: 212–219.  [DOI](https://doi.org/10.1016/j.gde.2009.04.010) | [PMC free article](/articles/PMC2914559/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19481926/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Opin%20Genet%20Dev&title=Common%20vs.%20rare%20allele%20hypotheses%20for%20complex%20diseases&author=NJ%20Schork&author=SS%20Murray&author=KA%20Frazer&author=EJ%20Topol&volume=19&publication_year=2009&pages=212-219&pmid=19481926&doi=10.1016/j.gde.2009.04.010&)

50. Dai S, Murphy GA, Crawford F, Mack DG, Falta MT, et al. (2010) Crystal structure of HLA-DP2 and implications for chronic beryllium disease. Proc Natl Acad Sci U S A 107: 7425–7430.  [DOI](https://doi.org/10.1073/pnas.1001772107) | [PMC free article](/articles/PMC2867715/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20356827/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&title=Crystal%20structure%20of%20HLA-DP2%20and%20implications%20for%20chronic%20beryllium%20disease&author=S%20Dai&author=GA%20Murphy&author=F%20Crawford&author=DG%20Mack&author=MT%20Falta&volume=107&publication_year=2010&pages=7425-7430&pmid=20356827&doi=10.1073/pnas.1001772107&)

51. Birben E, Sackesen C, Kazani S, Tincer G, Karaaslan C, et al. (2011) The effects of an insertion in the 5′UTR of the AMCase on gene expression and pulmonary functions. Respir Med 105: 1160–1169.  [DOI](https://doi.org/10.1016/j.rmed.2011.03.017) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21511453/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Respir%20Med&title=The%20effects%20of%20an%20insertion%20in%20the%205%E2%80%B2UTR%20of%20the%20AMCase%20on%20gene%20expression%20and%20pulmonary%20functions&author=E%20Birben&author=C%20Sackesen&author=S%20Kazani&author=G%20Tincer&author=C%20Karaaslan&volume=105&publication_year=2011&pages=1160-1169&pmid=21511453&doi=10.1016/j.rmed.2011.03.017&)

52. Spurr NK, Solomon E, Jansson M, Sheer D, Goodfellow PN, et al. (1984) Chromosomal localisation of the human homologues to the oncogenes erbA and B. EMBO J. 3: 159–163.  [DOI](https://doi.org/10.1002/j.1460-2075.1984.tb01777.x) | [PMC free article](/articles/PMC557313/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/6323162/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Spurr%20NK,%20Solomon%20E,%20Jansson%20M,%20Sheer%20D,%20Goodfellow%20PN,%20et%20al.%20(1984)%20Chromosomal%20localisation%20of%20the%20human%20homologues%20to%20the%20oncogenes%20erbA%20and%20B.%20EMBO%20J.%203:%20159%E2%80%93163.)

53. Tee AR, Fingar DC, Manning BD, Kwiatkowski DJ, Cantley LC, et al. (2002) Tuberous sclerosis complex-1 and -2 gene products function together to inhibit mammalian target of rapamycin (mTOR)-mediated downstream signaling. Proc Natl Acad Sci U S A 99: 13571–13576.  [DOI](https://doi.org/10.1073/pnas.202476899) | [PMC free article](/articles/PMC129715/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12271141/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&title=Tuberous%20sclerosis%20complex-1%20and%20-2%20gene%20products%20function%20together%20to%20inhibit%20mammalian%20target%20of%20rapamycin%20(mTOR)-mediated%20downstream%20signaling&author=AR%20Tee&author=DC%20Fingar&author=BD%20Manning&author=DJ%20Kwiatkowski&author=LC%20Cantley&volume=99&publication_year=2002&pages=13571-13576&pmid=12271141&doi=10.1073/pnas.202476899&)

54. Shin SW, Park J, Kim YJ, Uh ST, Choi BW, et al. (2012) A highly sensitive and specific genetic marker to diagnose aspirin-exacerbated respiratory disease using a genome-wide association study. DNA Cell Biol 31: 1604–1609.  [DOI](https://doi.org/10.1089/dna.2012.1688) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22994212/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=DNA%20Cell%20Biol&title=A%20highly%20sensitive%20and%20specific%20genetic%20marker%20to%20diagnose%20aspirin-exacerbated%20respiratory%20disease%20using%20a%20genome-wide%20association%20study&author=SW%20Shin&author=J%20Park&author=YJ%20Kim&author=ST%20Uh&author=BW%20Choi&volume=31&publication_year=2012&pages=1604-1609&pmid=22994212&doi=10.1089/dna.2012.1688&)
