# Genome-Wide Pharmacogenomic Study on Methadone Maintenance Treatment Identifies SNP rs17180299 and Multiple Haplotypes on CYP2B6, SPON1, and GSG1L Associated with Plasma Concentrations of Methadone R- and S-enantiomers in Heroin-Dependent Patients

## Metadata
**Authors:** Hsin-Chou Yang, Shih-Kai Chu, Chieh-Liang Huang, Hsiang-Wei Kuo, Sheng-Chang Wang, Sheng-Wen Liu, Ing-Kang Ho, Yu-Li Liu
**Journal:** PLoS Genetics
**Date:** 2016 Mar 24
**DOI:** [10.1371/journal.pgen.1005910](https://doi.org/10.1371/journal.pgen.1005910)
**PMID:** 27010727
**PMCID:** PMC4806848
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4806848/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC4806848/pdf/pgen.1005910.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC4806848/pdf/pgen.1005910.pdf)

## Abstract

Methadone maintenance treatment (MMT) is commonly used for controlling opioid dependence, preventing withdrawal symptoms, and improving the quality of life of heroin-dependent patients. A steady-state plasma concentration of methadone enantiomers, a measure of methadone metabolism, is an index of treatment response and efficacy of MMT. Although the methadone metabolism pathway has been partially revealed, no genome-wide pharmacogenomic study has been performed to identify genetic determinants and characterize genetic mechanisms for the plasma concentrations of methadone R- and S-enantiomers. This study was the first genome-wide pharmacogenomic study to identify genes associated with the plasma concentrations of methadone R- and S-enantiomers and their respective metabolites in a methadone maintenance cohort. After data quality control was ensured, a dataset of 344 heroin-dependent patients in the Han Chinese population of Taiwan who underwent MMT was analyzed. Genome-wide single-locus and haplotype-based association tests were performed to analyze four quantitative traits: the plasma concentrations of methadone R- and S-enantiomers and their respective metabolites. A significant single nucleotide polymorphism (SNP), rs17180299 (raw p = 2.24 × 10−8), was identified, accounting for 9.541% of the variation in the plasma concentration of the methadone R-enantiomer. In addition, 17 haplotypes were identified on SPON1, GSG1L, and CYP450 genes associated with the plasma concentration of methadone S-enantiomer. These haplotypes accounted for approximately one-fourth of the variation of the overall S-methadone plasma concentration. The association between the S-methadone plasma concentration and CYP2B6, SPON1, and GSG1L were replicated in another independent study. A gene expression experiment revealed that CYP2B6, SPON1, and GSG1L can be activated concomitantly through a constitutive androstane receptor (CAR) activation pathway. In conclusion, this study revealed new genes associated with the plasma concentration of methadone, providing insight into the genetic foundation of methadone metabolism. The results can be applied to predict treatment responses and methadone-related deaths for individualized MMTs.

## Introduction

Heroin dependence is a severe psychiatric disorder characterized by a heroin craving behavior and the inability to stop using heroin. The World Health Organization reported that there were more than 9 million heroin users in 2014, and the prevalence of heroin dependence is increasing globally. Heroin abuse can incur medical complications and generate social problems; these problems have substantially increased the burden on social security and health insurance systems [[1](#pgen.1005910.ref001), [2](#pgen.1005910.ref002)].

Replacement or maintenance therapy with an opioid analog is among the most commonly used treatments for heroin dependence, and reduces craving and withdrawal symptoms, increases treatment compliance, and improves the quality of life of patients [[3](#pgen.1005910.ref003)]. Methadone, a synthetic opioid, is a commonly used medication for treating heroin dependence [[4](#pgen.1005910.ref004)]. Methadone maintenance treatment (MMT) reportedly retains patients and decreases heroin use more effectively than no treatment [[4](#pgen.1005910.ref004)–[6](#pgen.1005910.ref006)].

The chemical structure of methadone has a chiral center that produces the racemic enantiomers of *R*- and *S*-form [[7](#pgen.1005910.ref007)]. Methadone *R*- and *S*-enantiomers have different metabolic preferences for the liver cytochrome P-450 (CYP450) isozyme [[8](#pgen.1005910.ref008)]: the CYP2C19 isozyme metabolizes methadone *R*-enantiomer [[8](#pgen.1005910.ref008), [9](#pgen.1005910.ref009)] and CYP2B6 isozyme metabolizes methadone *S*-enantiomer [[10](#pgen.1005910.ref010), [11](#pgen.1005910.ref011)]. Several pharmacogenetic studies have found that the genetic variants located in the *CYP2B6* region were associated with methadone [[12](#pgen.1005910.ref012)] and could be observed to predict methadone-related deaths [[13](#pgen.1005910.ref013), [14](#pgen.1005910.ref014)]. Through an interaction between *CYP3A4* and *CYP2C19*, a genetic matrix exhibited a titration prediction effect for the required methadone dose [[15](#pgen.1005910.ref015)].

The steady-state plasma concentration of methadone is an index for quantifying methadone metabolism and can serve as a surrogate marker for the treatment responses and efficacy of MMT. Previous pharmacogenetic studies have reported the association between *CYP2B6* and the plasma concentration of methadone *S*-enantiomer [[16](#pgen.1005910.ref016)–[18](#pgen.1005910.ref018)]. We previously established a system used to measure methadone enantiomers and their metabolism product, metabolite 2-ethylidene-1,5-dimethyl-3,3-diphenyl pyrrolidine (EDDP) [[9](#pgen.1005910.ref009)]. Pharmacogenetic studies on the *CYP2B6* gene in a methadone maintenance cohort in Taiwan revealed the association between the plasma concentration of methadone *S*-enantiomer and single nucleotide polymorphisms (SNPs) located at the region of *CYP2B6* gene [[16](#pgen.1005910.ref016)]. In addition, the pregnane X receptor, a steroid and xenobiotic sensing nuclear receptor, possibly interacts with genetic variants in *CYP2B6* to associate with the plasma concentration of methadone *S*-enantiomer [[19](#pgen.1005910.ref019)]. In addition to genes in the *CYP450* gene family, *ABCB1* was reported to be associated with the plasma concentration of methadone [[20](#pgen.1005910.ref020)]. However, the association remains unclear because the finding was not replicated in other studies [[18](#pgen.1005910.ref018), [21](#pgen.1005910.ref021)]. Dennis et al. provided a systematic review on the effect of *ABCB1* and *CYP2B6* on methadone metabolism [[18](#pgen.1005910.ref018)].

Although the methadone metabolism pathway has been partially revealed [[22](#pgen.1005910.ref022)], there is still no report of genome-wide association analyses that characterize the plasma concentrations of the methadone *R*- and *S*-enantiomers. Moreover, the metabolism regulatory genes associated with the plasma concentration of methadone remain unclear. Previous pharmacogenetic studies on the plasma concentration of methadone enantiomer have used a candidate gene approach and single locus association analyses [[16](#pgen.1005910.ref016), [17](#pgen.1005910.ref017)]. Because of the clinical importance of treatment responses to MMT and lack of results from genome-wide pharmacogenomic research, the aim of this study was to conduct the first genome-wide pharmacogenomic investigation of the plasma concentrations of methadone enantiomers and their metabolites. We recruited and genotyped a Han Chinese methadone maintenance cohort comprising 360 heroin-dependent patients in Taiwan by using the Axiom Genome-Wide CHB 1 Array. Genome-wide single-locus and multilocus methods were applied to identify the susceptibility loci and their contribution in the variability of the methadone plasma concentration. Furthermore, an independent methadone maintenance cohort comprising 78 heroin-dependent patients under the same inclusion and exclusion criteria was recruited to replicate the susceptibility loci identified in the genome-wide pharmacogenomic study (discovery stage). In addition to identifying novel genes associated with the plasma concentration of methadone, this pharmacogenomic study was the first to verify the results of previous biological metabolic tests and the methadone pathways.

## Results

### Quality control

[Fig 1(A)](#pgen.1005910.g001) shows flow charts and results of all statistical analyses, and [Fig 1(B)](#pgen.1005910.g001) depicts the results of quality control. In total, 360 MMT patients were genotyped with the Axiom Genome-Wide CHB 1 Array. Five patients (306–026, 306–023, 401–023, 206–055, and 301–041), who exhibited a homozygosity pattern of X chromosome(s) inconsistent with their self-reported gender, were removed ([S1 Fig](#pgen.1005910.s008)). One patient (206–059) was removed because of a low genotyping call rate (GCR) of <0.95. Three patients (217–024, 301–156, and 306–013) with an overly high genome-wide homozygosity rate were removed ([S2 Fig](#pgen.1005910.s009)). Two of the three patients (217–024, [S3(A) Fig](#pgen.1005910.s010); 301–156, [S3(B) Fig](#pgen.1005910.s010)) with an overly high genome-wide homozygosity rate were found to have excessive runs of homozygosity on multiple chromosomes ([S3(C) Fig](#pgen.1005910.s010)). Five pairs of individuals exhibiting cryptic relatedness were identified; the identity-by-descent (IBD) estimates were 0.5891, 0.5052, 0.4431, 0.4571, and 0.4641. We removed five individuals with the lower GCR in each pair (211–004, 217–027, 206–066, 301–032, and 206–007). Finally, we removed two patients (301–107 and 301–011) who exhibited divergent ancestry; their first two principal components of allele frequency were outside the confidence bands of our population, the Han Chinese population in Taiwan ([S4 Fig](#pgen.1005910.s011)). In total, 16 individuals were removed from the subsequent association analyses. After the quality examination of SNP, 14,342 nonautosomal SNPs were removed, 7,926 SNPs with a GCR of <0.95 or a minor allele frequency (MAF) of <0.01 were removed, and 2,277 additional SNPs were removed because of a deviation from the Hardy-Weinberg equilibrium (HWE) (*p* < 8.1 × 10^−8^). In this study, 615,216 valid SNPs (approximately 98% of the autosomal SNPs on the Axiom Genome-Wide CHB 1 Array) were analyzed.

### Fig 1. Analysis flow of this study.

![Fig 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7c41/4806848/166cc78ed81b/pgen.1005910.g001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4806848_pgen.1005910.g001.jpg)

(A) Flow of all statistical analyses. (B) Flow of quality control.

### Three covariates and four quantitative traits

We examined three covariates (age, sex, and body mass index [BMI]) and four quantitative traits (the plasma concentrations of the methadone *R*- and *S*-enantiomers and their metabolites). Based on the post-quality-control data, the distributions of the three covariates and four quantitative traits are summarized in [Table 1](#pgen.1005910.t001). Among the male patients, the mean age and BMI were 39.3061 (standard deviation [SD] = 7.6587) and 23.9176 (SD = 3.4537), respectively. Among the female patients, the mean age and BMI were 33.0318 (SD = 5.4506) and 22.3980 (SD = 3.5792), respectively. The normality of the quantitative trait was a critical assumption required for the association analysis. The normality of the four quantitative traits was examined; [Table 1](#pgen.1005910.t001) shows a summary of the results. Before any data transformation, the Kolmogorov–Smirnov goodness-of-fit tests for normality [[23](#pgen.1005910.ref023)] revealed that the four quantitative traits significantly violated normality. After data transformation by winsorization, log transformation, and standardization, the normality assumption was achieved for all of the plasma concentrations of *R*-methadone (*p* = 0.6255), *S*-methadone (*p* = 0.0802), *R*-EDDP (*p* = 0.7903), and *S*-EDDP (*p* = 0.1876). Data distributions of the four quantitative traits before and after data transformation are displayed in [S5(A) and S5(B) Fig](#pgen.1005910.s012), respectively.

### Table 1. Summary statistics of covariates and the raw data and transformed data of quantitative traits by gender.

|   | Male |  | Female |  |   |
| --- | --- | --- | --- | --- | --- |
| Characteristics | Sample size | Mean ± SD | Sample size | Mean ± SD | Normality test (p value) |
| Age (years) | 281 | 39.3061 ± 7.6587 | 63 | 33.0318 ± 5.4506 | - |
| BMI (kg/m2) | 278 | 23.9176 ± 3.4537 | 63 | 22.398 ± 3.5792 | - |
| Raw plasma R-methadone/dose (ng/ml/mg) | 281 | 3.871 ± 2.4659 | 63 | 3.755 ± 1.7789 | 1.41×10−6 |
| Raw plasma S-methadone/dose (ng/ml/mg) | 281 | 2.787 ± 1.5619 | 63 | 2.566 ± 1.4909 | 2.67×10−3 |
| Raw plasma R-EDDP/dose (ng/ml/mg) | 272 | 0.2975 ± 0.455 | 63 | 0.2964 ± 0.4587 | < 2.2×10−16 |
| Raw plasma S-EDDP/dose (ng/ml/mg) | 277 | 0.3286 ± 0.5477 | 63 | 0.282 ± 0.1842 | < 2.2×10−16 |
| Transformed plasma R-methadone/dose (ng/ml/mg) | 281 | 0.0030 ± 0.9946 | 63 | -0.0136 ± 1.0316 | 0.6255 |
| Transformed plasma S-methadone/dose (ng/ml/mg) | 281 | 0.0173 ± 1.0227 | 63 | -0.0774 ± 0.8953 | 0.0802 |
| Transformed plasma R-EDDP/dose (ng/ml/mg) | 272 | 0.0212 ± 0.9787 | 63 | -0.0916 ± 1.0908 | 0.7903 |
| Transformed plasma S-EDDP/dose (ng/ml/mg) | 277 | -0.0004 ± 1.0075 | 63 | 0.0019 ± 0.9741 | 0.1876 |

Table 1 Caption: The number of individuals (i.e., sample size), means ± standard deviations (SD) for covariates and quantitative traits are provided. The final column provides the p value of the Kolmogorov-Smirnov Good-of-Fit test for normality for the raw data and transformed data of the four quantitative traits.

### Genome-wide singe-locus association analysis

As shown in the Manhattan plot ([Fig 2(A)](#pgen.1005910.g002)) and Quantile–Quantile plot ([Fig 2(B)](#pgen.1005910.g002)), the genome-wide single-locus association analysis identified only SNP rs17180299 to be significantly associated with the plasma concentration of *R*-methadone after a multiple-test correction of a false discovery rate (raw *p* = 2.24 × 10^−8^). [Fig 2(C)](#pgen.1005910.g002) depicts the regional association plot for the flanking genomic region of SNP rs17180299 on chromosome 9. Rs17180299 was an *A*/*G* polymorphism; the frequency of minor allele *G* was 0.09. A trend of the dose-response effect of rs17180299 on the plasma concentration of *R*-methadone was observed ([Fig 2(D)](#pgen.1005910.g002)). Patients who had more allele *G* tended to have a higher plasma concentration of *R*-methadone. A SNP cluster plot showed that individuals with different genotypes were clearly separated, implying that the genotype calls were reliable. However, this SNP was located in an intergenic region.

### Fig 2. Genome-wide single-locus association test for plasma concentration of R-methadone.

![Fig 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7c41/4806848/0d64edd112ef/pgen.1005910.g002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4806848_pgen.1005910.g002.jpg)

(A) Manhattan plot of genome-wide single-locus association test. The vertical axis indicates the raw p values of the association tests (in a scale of -log10). The horizontal axis is physical position of a SNP on different autosomes. Each point indicates the raw p value of a SNP. A significant SNP is circled in blue. (B) Quantile–Quantile plot of genome-wide single-locus association test. The vertical axis indicates the raw p values of the association tests (in a scale of -log10). The horizontal axis indicates the expected p values of the association tests (in a scale of -log10) under a null hypothesis, i.e., no genetic association. (C) Regional association plot of the significant SNP, rs17180299. The vertical axis indicates the raw p values of the association tests (in a scale of -log10). The horizontal axis is physical position of SNP (in unit of kb). (D) Distribution of plasma concentration of R-methadone for three genotypes of rs17180299.

For the other three traits, no single SNP was significant after a multiple-test correction of the false discovery rate. The Manhattan plots of genome-wide SNPs (in a scale of -log_10_) for the plasma concentration of *S*-methadone ([S6(A) Fig](#pgen.1005910.s013)), the plasma concentration of *R*-EDDP ([S6(B) Fig](#pgen.1005910.s013)), and the plasma concentration of *S*-EDDP ([S6(C) Fig](#pgen.1005910.s013)) were created. The SNPs with the smallest *p* values in the genome-wide association analysis of the plasma concentration of *S*-methadone, *R*-EDDP, and *S*-EDDP were AX-16534452 (raw *p* = 4.83 × 10^−7^), rs1448332 (raw *p* = 8.18 × 10^−7^), and AX-13599782 (raw *p* = 2.08 × 10^−6^), respectively. The SNP AX-16534452 was located on *SPON1* on chromosome 11. The SNPs rs1448332 and AX-13599782 were located in an intergenic region on chromosome 3 and chromosome 21, respectively.

### Genome-wide haplotype-based association analysis

We performed sliding-window haplotype-based association analyses with a window size of 2, 3, 4, 5, and 10 individually. For each window size, we separately performed haplotype-based association analyses for the four quantitative traits, adjusting for age, sex, and BMI. Only the results on a basis of a window size of 5 were reported because of the following two reasons. First, the locations of significant haplotypes in the haplotype-based association analyses of different window sizes were quite consistent. Among the 25 most significant windows in each of the analyses of different window sizes, higher than 92.5% of the significant windows were overlapped. Second, in the study population, a window size of 5 provided a distribution of window width nearest to the distribution of the block size of the LD of the Asian population in the International HapMap II Project ([S1 Table](#pgen.1005910.s001)) [[24](#pgen.1005910.ref024), [25](#pgen.1005910.ref025)].

The results for the plasma concentration of *R*-methadone are shown in the Manhattan plot ([Fig 3(A)](#pgen.1005910.g003)) and Quantile–Quantile plot ([Fig 3(B)](#pgen.1005910.g003)). The results for the plasma concentration of *S*-methadone are shown in the Manhattan plot ([Fig 4(A)](#pgen.1005910.g004)) and Quantile–Quantile plot ([Fig 4(B)](#pgen.1005910.g004)). The results for the plasma concentration of *R*-EDDP ([S7(A) Fig](#pgen.1005910.s014)) and the plasma concentration of *S*-EDDP ([S7(B) Fig](#pgen.1005910.s014)) are also provided. After a multiple-test correction of the false discovery rate [[26](#pgen.1005910.ref026)], we determined that four sliding windows were significantly associated with the plasma concentration of *R*-methadone ([Fig 3(A)](#pgen.1005910.g003)), and 23 sliding windows were associated with the plasma concentration of *S*-methadone ([Fig 4(A)](#pgen.1005910.g004)). No significant results were found in the haplotype-based association analyses of the plasma concentration of *R*-EDDP ([S7(A) Fig](#pgen.1005910.s014)) or the plasma concentration of *S*-EDDP ([S7(B) Fig](#pgen.1005910.s014)).

### Fig 3. Sliding-window genome-wide haplotype analysis of plasma concentration of R-methadone.

![Fig 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7c41/4806848/5b133e2fa6ff/pgen.1005910.g003.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4806848_pgen.1005910.g003.jpg)

(A) Manhattan plot of genome-wide haplotype-based association test. The vertical axis is the raw p values of the omnibus haplotype tests (in a scale of -log10) and the horizontal axis is physical position of the initial SNP of a haplotype in different autosomes. Each point indicates the raw p value of a window. The position of a significant haplotype is indicated by a blue arrow. (B) Quantile–Quantile plot of genome-wide haplotype-based association test. The vertical axis indicates the raw p values of the association tests (in a scale of -log10). The horizontal axis indicates the expected p values of the association tests (in a scale of -log10) under a null hypothesis, i.e., no genetic association. (C) Regional association plot and LD plot on chromosome 4. In the regional association plot, the vertical axis is the raw p values of the association tests (in a scale of -log10) and the horizontal axis is physical position of initial SNP of a haplotype (in unit of kb). Raw p values of single SNPs (orange circle) and haplotypes which were significant after adjusting for false discovery rate [26] (symbol: green square with frame) and were not significant (symbol: green square without frame) are shown. Moreover, haplotypes which passed a multiple-test correction of a false discovery rate [26] in a LD block (purple triangle) are shown. In the LD plot, pairwise LD of SNPs was measured by D’ [27]. SNPs with a strong LD were framed in a black inverse diamond block which was defined according to the confidence interval method [24]. (D) Regional association plot and LD plot on chromosome 9. (E) Distribution of plasma concentration of R-methadone for the haplotypes on chromosome 9 significantly associated with the plasma concentration of R-methadone.

### Fig 4. Sliding-window genome-wide haplotype analysis of plasma concentration of S-methadone.

![Fig 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7c41/4806848/d917c1419cbd/pgen.1005910.g004.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4806848_pgen.1005910.g004.jpg)

(A) Manhattan plot of genome-wide haplotype-based association test. The vertical axis is the raw p values of the omnibus haplotype tests (in a scale of -log10) and the horizontal axis is physical position of the initial SNP of a haplotype in different autosomes. Each point indicates the raw p value of a window. The positions of significant haplotypes are indicated by blue arrows. (B) Quantile–Quantile plot of genome-wide haplotype-based association test. The vertical axis indicates the raw p values of the association tests (in a scale of -log10). The horizontal axis indicates the expected p values of the association tests (in a scale of -log10) under a null hypothesis, i.e., no genetic association. (C) Regional association plot and LD plot on chromosome 11. In the regional association plot, the vertical axis is the raw p values of the association tests (in a scale of -log10) and the horizontal axis is physical position of initial SNP of a haplotype (in unit of kb). Raw p values of single SNPs (orange circle) and haplotypes which were significant after adjusting for false discovery rate [26] (symbol: green square with frame) and were not significant (symbol: green square without frame) are shown. Moreover, haplotypes which passed a multiple-test correction of a false discovery rate [26] in a LD block (purple triangle) are shown. In the LD plot, pairwise LD of SNPs was measured by D’ [27]. SNPs with a strong LD were framed in a black inverse diamond block which was defined according to the confidence interval method [24]. (D) Regional association plot and LD plot on chromosome 16. (E) Regional association plot and LD plot on chromosome 19.

The four sliding windows associated with the plasma concentration of *R*-methadone are summarized in [Table 2](#pgen.1005910.t002); the four windows consisted of one window located on chromosome 4 ([Fig 3(C)](#pgen.1005910.g003)) and three windows located in consecutive genomic regions on chromosome 9 ([Fig 3(D)](#pgen.1005910.g003)). The significant window regions were expanded to encompass the flanking region on both sides of the significant windows. The expanded regions on chromosomes 4 and 9 contained two LD blocks ([Fig 3(C)](#pgen.1005910.g003)) and five LD blocks ([Fig 3(D)](#pgen.1005910.g003)), respectively. We further examined whether individual haplotypes in the LD blocks were associated with the plasma concentration of *R*-methadone. No significant individual haplotype was found in the significant region of chromosome 4 ([Table 2](#pgen.1005910.t002) and [Fig 3(C)](#pgen.1005910.g003)). Significant individual haplotypes were found in the first four LD blocks in the region of chromosome 9 ([Table 2](#pgen.1005910.t002) and [Fig 3(D)](#pgen.1005910.g003)). The fourth LD block contained the only significant SNP (rs17180299) identified during the genome-wide single-locus association test. The slope estimates and their standard errors and the corresponding 95% confidence intervals of the 4 significant haplotypes are summarized in [S2 Table](#pgen.1005910.s002). All of the SNPs were located in intergenic regions. Trends of dose-response effects exerted by the four significant haplotypes on the plasma concentration of *R*-methadone were found ([Fig 3(E)](#pgen.1005910.g003)). Patients with more minor haplotypes tended to exhibit a higher plasma concentration of *R*-methadone.

### Table 2. The significant haplotypes associated with plasma concentration of R-methadone.

| Chrom. | Window a | LD block | Gene | Significant haplotype | Haplotype frequency | Raw p value (Adjusted p) b |
| --- | --- | --- | --- | --- | --- | --- |
| 4 | w1 | 1, 2 | --- | --- | --- | NS |
| 9 | w1- | 1 | --- | TTC(rs10115245, rs62572435, and rs1572144) | 0.095 | 2.26×10−6 (4.07×10−5) |
| 9 | w1 | 2 | --- | GC(rs1333935, and rs1333934) | 0.097 | 2.49×10−6 (4.48×10−5) |
| 9 | w1 | 3 | --- | CCA(rs4085128, rs4877551, and rs4877552) | 0.093 | 4.55×10−8 (8.19×10−7) |
| 9 | w2 ~ w3 | 4 | --- | CGGCG(rs970905, rs17180299, rs12000653, rs11138610, and rs72744905) | 0.090 | 2.24×10−8 (4.03×10−7) |

Table 2 Caption: We list the chromosome, window, linkage disequilibrium (LD) block, and gene where the haplotypes are located. Haplotype frequencies and raw and adjusted p values of the significant haplotypes are provided in the final two columns.

The 23 sliding windows associated with the plasma concentration of *S*-methadone are summarized in [Table 3](#pgen.1005910.t003); the 23 windows were located within three consecutive genomic regions ([Fig 4(A)](#pgen.1005910.g004)). They contained one window on chromosome 11 ([Fig 4(C)](#pgen.1005910.g004)), two windows on chromosome 16 ([Fig 4(D)](#pgen.1005910.g004)), and 20 windows on chromosome 19 ([Fig 4(E)](#pgen.1005910.g004)). An individual haplotype analysis in LD blocks revealed that 17 haplotypes in the 23 windows were significantly associated with the plasma concentration of *S*-methadone. The slope estimates and the corresponding 95% confidence intervals of the 17 significant haplotypes are summarized in [S2 Table](#pgen.1005910.s002). The significant genomic region of chromosome 11 contained two significant haplotypes in LD blocks B2 and B3 on *SPON1* ([Table 3](#pgen.1005910.t003) and [Fig 4(C)](#pgen.1005910.g004)). The significant genomic region of chromosome 16 contained five significant haplotypes in LD blocks B1 and B2 on *GSG1L* ([Table 3](#pgen.1005910.t003) and [Fig 4(D)](#pgen.1005910.g004)). Finally, the significant genomic region of chromosome 19 contained 10 significant haplotypes in LD blocks B2–B7 ([Table 3](#pgen.1005910.t003) and [Fig 4(E)](#pgen.1005910.g004)). This genomic region contained multiple *CYP450* genes. For all the 17 significant haplotypes, trends of dose-response effects exerted by the significant haplotypes on the plasma concentration of *S*-methadone were found; the dose-response effects of the 2, 5, 10 significant haplotypes on chromosomes 11, 16, and 19 are shown in [Fig 5(A), 5(B) and 5(C)](#pgen.1005910.g005), respectively. [S3](#pgen.1005910.s003) and [S4](#pgen.1005910.s004) Tables summarize all haplotypes with a minor haplotype frequency of >0.01 whatever they were significant or not in association tests of individual haplotypes for the plasma concentrations of methadone *R*- and *S*-enantiomers, respectively.

### Table 3. The significant haplotypes associated with plasma concentration of S-methadone.

| Chrom. | Window a | LD block | Gene name | Significant haplotype | Haplotype frequency | Raw p value (Adjusted p) |
| --- | --- | --- | --- | --- | --- | --- |
| 11 | w1+ | 2 | SPON1 | TTA(rs4756776, rs4757240, and rs4756779) | 0.153 | 1.27×10−6 (1.52×10−5) |
| 11 | w1+ | 3 | SPON1 | TC(rs4757242, and rs7936301) | 0.279 | 5.45×10−5 (6.54×10−4) |
| 16 | w1 ~ w2 | 1 | GSG1L | TCACT(rs60857987, rs713547, rs705912, rs4787995, and rs8182215) | 0.368 | 1.73×10−3 (1.38×10−2) |
| 16 | w1 ~ w2 | 1 | GSG1L | TCGCT(rs60857987, rs713547, rs705912, rs4787995, and rs8182215) | 0.122 | 1.78×10−3 (1.42×10−2) |
| 16 | w1 ~ w2 | 1 | GSG1L | TCGTT(rs60857987, rs713547, rs705912, rs4787995, and rs8182215) | 0.154 | 3.54×10−3 (2.83×10−2) |
| 16 | w2+ | 2 | GSG1L | CTGC(AX-13044173, rs772972, rs772973, and rs11639671) | 0.096 | 2.96×10−3 (2.37×10−2) |
| 16 | w2+ | 2 | GSG1L | TTAC(AX-13044173, rs772972, rs772973, and rs11639671) | 0.363 | 3.92×10−3 (3.14×10−2) |
| 19 | w1- | 2 | CYP2A7P1, CYP2B7P1 | AGC(rs12461727, rs73038469, and rs4803406) | 0.270 | 1.58×10−4 (7.11×10−3) |
| 19 | w1 | 3 | CYP2A7P1, CYP2B7P1 | GT(rs8110485, and rs4124633) | 0.259 | 2.43×10−6 (1.09×10−4) |
| 19 | w1 ~ w8 | 4 | CYP2A7P1, CYP2B7P1 | CTTCCGCAT(rs4560022, rs3889806, rs4803410, rs1017384, rs7251950, rs1808682, AX-13442755, rs3760657, and rs2054675) | 0.219 | 5.17×10−6 (2.33×10−4) |
| 19 | w1 ~ w8 | 4 | CYP2A7P1, CYP2B7P1 | TCTACGCAC(rs4560022, rs3889806, rs4803410, rs1017384, rs7251950, rs1808682, AX-13442755, rs3760657, and rs2054675) | 0.179 | 2.31×10−5 (1.04×10−3) |
| 19 | w5 ~ w14 | 5 | CYP2A7P1, CYP2B6 | TAATCG(rs8100458, rs7250601, rs7250991, rs11882424, rs8192719, and rs10853744) | 0.311 | 2.03×10−7 (9.14×10−6) |
| 19 | w5 ~ w14 | 5 | CYP2A7P1, CYP2B6 | TCCTTT(rs8100458, rs7250601, rs7250991, rs11882424, rs8192719, and rs10853744) | 0.126 | 9.04×10−6 (4.07×10−4) |
| 19 | w12 ~ w17 | 6 | CYP2A7P1 | CTAAG(rs61073883, rs1552222, rs7255904, AX-13442812, and rs11666982) | 0.179 | 6.81×10−4 (3.06×10−2) |
| 19 | w12 ~ w17 | 6 | CYP2A7P1 | CTGAT(rs61073883, rs1552222, rs7255904, AX-13442812, and rs11666982) | 0.330 | 2.04×10−7 (9.18×10−6) |
| 19 | w18 ~ w20 | 7 | --- | CGCG(rs17726861, rs7257703, rs17726963, and rs12982859) | 0.392 | 1.11×10−5 (5.00×10−4) |
| 19 | w18 ~ w20 | 7 | --- | GAAG(rs17726861, rs7257703, rs17726963, and rs12982859) | 0.337 | 2.08×10−6 (9.36×10−5) |

Table 3 Caption: We list the chromosome, window, linkage disequilibrium (LD) block, and gene where the haplotypes are located. Haplotype frequencies and p values of the significant haplotypes are provided in the final two columns.

### Fig 5. Distribution of plasma concentration of S-methadone for the significant haplotypes.

![Fig 5](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7c41/4806848/d8ae62d518ac/pgen.1005910.g005.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4806848_pgen.1005910.g005.jpg)

Distribution of plasma concentration of each significant haplotype is described in a subfigure; the haplotype frequency is provided in the title. In each subfigure, three box plots for patient groups with respect to 0, 1, and 2 haplotypes of interest are shown. In each box plot, five bands including the 1.5 interquartile range (IQR) of the lower quartile, first quartile, second quartile (i.e., median), third quartile, and 1.5 IQR of the lower quartile of plasma concentration are plotted. The vertical axis is the plasma concentration and the horizontal axis provides haplotype and the proportion of haplotype in our data set. (A) Box plots for the significant haplotypes on chromosome 11. (B) Box plots for the significant haplotypes on chromosome 16. (C) Box plots for the significant haplotypes on chromosome 19.

### Analysis of the variation in plasma concentration explained by significant SNP and haplotypes

[Table 4](#pgen.1005910.t004) shows the proportions of the variations in the plasma concentrations of methadone *R*- and *S*-enantiomers explained by significant haplotypes. Regarding the plasma concentration of *R*-methadone, one SNP (rs17180299) and four haplotypes on chromosome 9 were significant. Rs17180299 accounted for 9.541% of the variation in the plasma concentration of *R*-methadone. The most significant haplotype, *C-G-G-C-G*, was located in LD block B4 ([Fig 3(D)](#pgen.1005910.g003)). The SNP rs17180299, which was the only significant SNP identified during the genome-wide single-locus association test, was a tag SNP of the other four SNPs in this region. Therefore, this haplotype accounted for the same amount of variation as did rs17180299. According to the effect exerted by haplotype *C-G-G-C-G*, the remaining three significant haplotypes accounted for an extra variation of 1.809%. In total, 11.350% of the variation in the plasma concentration of *R*-methadone was accounted for by the identified SNPs and haplotypes. Regarding the plasma concentration of *S*-methadone, 17 haplotypes in the 23 windows was included into the final model sequentially; a haplotype which contributed a higher increment of model R^2^ was included earlier in a sequential order. The 17 significant haplotypes accounted for 23.96% of the variation in the plasma concentration of *S*-methadone. The five most significant haplotypes, which included two significant haplotypes in *CYP2B6*, accounted for more than 20% of the variation in the plasma concentration of *S*-methadone.

### Table 4. The proportion of variation explained by the significant haplotypes we identified.

| Transformed plasma concentration | Step | Chrom. | LD block | Significant haplotypes | a Univariate R2 | b Model R2 |
| --- | --- | --- | --- | --- | --- | --- |
| R-methadone | 1 | 9 | 4 | CGGCG | 0.095408 | 0.095408 |
| R-methadone | 2 | 9 | 1 | TTC | 0.069709 | 0.107298 |
| R-methadone | 3 | 9 | 2 | GC | 0.070183 | 0.113362 |
| R-methadone | 4 | 9 | 3 | CCA | 0.091044 | 0.113498 |
| S-methadone | 1 | 19 | 5 | TAATCG | 0.07653744 | 0.075978 |
| S-methadone | 2 | 11 | 2 | TTA | 0.06973389 | 0.136224 |
| S-methadone | 3 | 19 | 5 | TCCTTT | 0.05598868 | 0.166829 |
| S-methadone | 4 | 16 | 1 | TCGCT | 0.02923673 | 0.186557 |
| S-methadone | 5 | 16 | 1 | TCACT | 0.02532389 | 0.206716 |
| S-methadone | 6 | 19 | 7 | CGCG | 0.05109671 | 0.21637 |
| S-methadone | 7 | 19 | 2 | AGC | 0.0424036 | 0.223295 |
| S-methadone | 8 | 16 | 2 | TTAC | 0.02020137 | 0.230003 |
| S-methadone | 9 | 19 | 3 | GT | 0.0663698 | 0.233462 |
| S-methadone | 10 | 19 | 4 | TCTACGCAC | 0.0518725 | 0.235165 |
| S-methadone | 11 | 19 | 6 | CTGAT | 0.07624549 | 0.236899 |
| S-methadone | 12 | 16 | 1 | TCGTT | 0.02337932 | 0.238038 |
| S-methadone | 13 | 19 | 6 | CTAAG | 0.03132347 | 0.238553 |
| S-methadone | 14 | 11 | 3 | TC | 0.04704134 | 0.239014 |
| S-methadone | 15 | 16 | 2 | CTGC | 0.02666334 | 0.239358 |
| S-methadone | 16 | 19 | 4 | CTTCCGCAT | 0.0613232 | 0.239564 |
| S-methadone | 17 | 19 | 7 | GAAG | 0.06556728 | 0.239564 |

Table 4 Caption: In accordance with the inclusion steps of haplotypes in each analysis of plasma concentration of R-methadone and S-methadone, we list the chromosome and linkage disequilibrium (LD) block where the haplotypes are located. A haplotype which contributed a higher increment of model R2 was included earlier in a sequential order. Univariate R2 and model R2 of the significant haplotypes are provided in the final two columns.

### Replication analysis of the significant SNP and haplotypes identified in the genome-wide pharmacogenomic study

[S8(A) Fig](#pgen.1005910.s015) shows flow charts and results of all statistical analyses, and [S8(B) Fig](#pgen.1005910.s015) depicts the results of quality control in the replication study. In total, 78 MMT patients were genotyped with the Axiom Genome-Wide CHB 1 Array. Following the same procedures of quality control used at the discovery stage, 76 patients and 613,414 valid SNPs were remained; two (SS-401063 and SS-501002) of 78 patients were removed because of abnormal homozygosity rates. The distributions of the three covariates and two plasma concentrations at the discovery stage and replication stage were similar ([Table 1](#pgen.1005910.t001) and [S5 Table](#pgen.1005910.s005)). The results of association analysis are summarized in [S6 Table](#pgen.1005910.s006). Association analysis showed that haplotypes *T-C-G-T-T* on *GSG1L* (raw *p* = 4.40 × 10^−2^), *C-T-T-C-C-G-C-A-T* (raw *p* = 1.90 × 10^−2^) and *T-C-T-A-C-G-C-A-C* (raw *p* = 2.60 × 10^−2^) on *CYP2B7P1* and *CYP2A7P1* were associated with the plasma concentration of *S*-methadone. For clinical applications, we considered urine morphine test, which is an indicator for treatment response to MMT, as a stratified variable. In the test-positive group (i.e., poor-response group), *T*-*T*-*A* (raw *p* = 2.94 × 10^−2^) on *SPON1* was significant; in the test-negative group (i.e., better-response group), *C-T-T-C-C-G-C-A-T* (raw *p* = 3.97 × 10^−2^), *T-C-T-A-C-G-C-A-C* (raw *p* = 3.83 × 10^−2^), and *T-A-A-T-C-G* (raw *p* = 4.37 × 10^−2^) on *CYP2B7P1*, *CYP2A7P1*, and *CYP2B6* were significant. Other haplotypes had a *p* value of >0.05. The five replicated haplotypes (i.e., the significant haplotypes in the 1^st^, 2^nd^, 10^th^, 12^th^, and 16^th^ inclusion steps for the plasma concentration of *S*-methadone in [Table 4](#pgen.1005910.t004)) accounted for 17.75% and 19.51% of the variation in the plasma concentration of *S*-methadone at the discovery stage and replication stage, respectively. Regression coefficients of the replicated haplotypes at the discovery and replication stages were in the same direction. A meta-analysis by combining *p*-values at the discovery and replication stages showed that most of the haplotypes identified at the discovery stage were still statistically significant ([S7 Table](#pgen.1005910.s007)). However, we may have overlooked some important findings such as the significance of rs17180299 (raw *p* = 0.3578 in the replication study) and some important haplotypes ([S6 Table](#pgen.1005910.s006)) because of a small sample size in the replication study.

### Constitutive androstane receptor (CAR)-activated coexpression of CYP2B6, SPON1, and GSG1L

We investigated the roles of *SPON1* and *GSG1L* genes and their coexpression with *CYP2B6* by using a regulated plasma concentration of *S*-methadone in the human liver ([Fig 6(A)](#pgen.1005910.g006)). The mRNA expression levels of *CYP2B6*, *SPON1*, and *GSG1L* in human hepatoblastoma (HepG2) cells were measured after 24 h of exposure to CAR agonists. The results showed that when the concentration of CAR agonist 6-(4-chlorophenyl)imidazo-[2,1-b][1,3]thiazole-5-carbaldehyde-O-(3,4-dichloro-benzyl)-oxime (CITCO) was increased from 0 μM to 1 μM, the mRNA expression levels of *CYP2B6*, *SPON1*, and *GSG1L* increased concomitantly and significantly ([Fig 6(B)](#pgen.1005910.g006)). For *CYP2B6*, the mRNA expression increased from 1.293 × 10^−3^ (SD = 4.619 × 10^−5^) to 1.523 × 10^−3^ (SD = 2.082 × 10^−5^), a 1.18-fold increase. For *SPON1*, the mRNA expression increased from 1.200 × 10^−4^ (SD = 1.000 × 10^−5^) to 1.700 × 10^−4^ (SD = 2.000 × 10^−5^), a 1.42-fold increase. For *GSG1L*, the mRNA expression increased from 7.000 × 10^−5^ (SD = 2.000 × 10^−5^) to 1.267 × 10^−4^ (SD = 1.528 × 10^−5^); this was a 1.81-fold increase. The relative gene expression showed that the CAR agonist induced the gene expression of *CYP2B6*, *SPON1*, and *GSG1L*.

### Fig 6. Co-expression of CYP2B6, SPON1, and GSG1L through the CAR activation pathway.

![Fig 6](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7c41/4806848/05d1f6ff9764/pgen.1005910.g006.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4806848_pgen.1005910.g006.jpg)

(A) Schema diagram to show the relation among CYP2B6, SPON1, and GSG1L through the CAR activation pathway. (B) Induction of mRNA expression of CYP2B6, SPON1, and GSG1L in HepG2 after a 24-hr treatment with the CAR activator CITCO. In the bar graph, data were presented as the mean of relative mRNA expression ± SD (n = 3). Asterisks above the bars indicate a significant difference in the relative mRNA expression of a gene between a concentration of CITCO of 0 and 1 μM (p < 0.05).

## Discussion

*R*- and *S*-methadone can be metabolized through various enzymatic pathways. Results from a previous candidate gene study of *CYP2B6* [[16](#pgen.1005910.ref016)] and other pharmacogenetic studies [[10](#pgen.1005910.ref010), [11](#pgen.1005910.ref011)] have shown that the methadone *S*-enantiomer was frequently metabolized by CYP2B6 isozymes. The present genome-wide association study confirmed that *CYP2B6* is associated with the plasma concentration of methadone *S*-enantiomer. The two significant haplotypes, *T-A-A-T-C-G* and *T-C-C-T-T-T* of rs8100458, rs7250601, rs7250991, rs11882424, rs8192719, and rs10853744, on *CYP2B6* that were identified in this study accounted for 10.72% of the variation in the plasma concentration of *S*-methadone. Haplotype *T-C-C-T-T-T* was found to be associated with a urine morphine test in a permutation-based logistic regression analysis with 100,000 random shuffles of test-positive and test-negative patients (empirical *p* = 4.04 × 10^−3^). The results implied that the identified haplotypes were relevant to the response to MMT.

We further examined the genetic contribution of *CYP2B6* by performing an integrative analysis that involved seven SNPs (rs8100458, rs7250601, rs7250991, rs11882424, rs8192719, rs10853744, and rs1042389) on *CYP2B6* from the present study and 10 SNPs (rs8100458, rs10500282, rs10403955, rs2279342, rs3745274, rs2279343, rs2279345, rs1038376, rs707265, and rs1042389) from a previous study of *CYP2B6* [[16](#pgen.1005910.ref016)]. Overall, the haplotype analysis involved 15 distinct SNPs, including two SNPs (rs8100458 and rs1042389) that were investigated in the two studies. LD blocks were constructed and haplotype association tests were conducted in each LD block by using the same association test described in the **Methods** section. The analysis revealed two LD blocks ([S9 Fig](#pgen.1005910.s016)). The block near the 3’ end was an untranslated region and contained three SNPs (rs1038376, rs707265, and rs1042389 [i.e., AX-13442801]). Haplotypes *A*-*A*-*T* (raw *p* = 2.25 × 10^−7^, haplotype frequency = 0.332) and *T*-*G*-*T* (raw *p* = 5.64 × 10^−4^, haplotype frequency = 0.142) were significantly associated with the plasma concentration of the methadone *S*-enantiomer. The block near the 5’ end contained 12 SNPs. Haplotypes *T-A-A-T-T-A-G-G-T-T-C-G* (raw *p* = 1.10 × 10^−3^, haplotype frequency = 0.024), *T-A-A-T-T-A-G-A-T-T-C-G* (raw *p* = 1.65 × 10^−4^, haplotype frequency = 0.278), and *T-C-C-C-G-A-T-G-C-T-T-T* (raw *p* = 7.04 × 10^−6^, haplotype frequency = 0.126) were significant.

The results of the two haplotype analyses from the current study and the aforementioned combined analysis of 15 SNPs were compared. Two significant haplotypes of six SNPs on *CYP2B6* identified in the current study were also in the LD block near the 5’ end. The patterns of these two haplotypes were compatible with the three significant haplotypes identified in the combined analysis of 15 SNPs. The two significant haplotypes can be presented as *T-A-A-X-X-X-X-X-X-T-X-C-G* (raw *p* = 2.03 × 10^−7^, haplotype frequency = 0.311) and *T-C-C-X-X-X-X-X-X-T-X-T-T* (raw *p* = 9.04 × 10^−6^, haplotype frequency = 0.126), where *X* denotes an unknown allele caused by the unavailability of the SNPs in the Axiom Genome-Wide CHB 1 Array. The proportion of the variation in the plasma concentration of methadone *S*-enantiomers accounted for by the five haplotypes on *CYP2B6* increased from 10.725% to 13.433% after the SNPs of the Axiom Genome-Wide CHB 1 Array and the SNPs from our previous candidate gene study of *CYP2B6* were combined. Of the 15 SNPs, two were missense SNPs: rs3745274 (raw *p* = 8.08 × 10^−7^, minor allele frequency = 0.185) and rs2279343 (raw *p* = 3.62×10^−3^, minor allele frequency = 0.263). These two SNPs have been associated with methadone in heroin-dependent patients [[12](#pgen.1005910.ref012)] and poor metabolism, plasma concentration, and efavirenz clearance in children with human immunodeficiency virus [[28](#pgen.1005910.ref028)–[30](#pgen.1005910.ref030)].

*CYP2B6* was highly expressed in the liver ([S10(A) Fig](#pgen.1005910.s017)) and influenced methadone metabolism. We investigated whether the identified haplotypes near *CYP2B6* can regulate the gene expression of *CYP2B6* through the mechanism of cis-regulation. An RT-PCR experiment was conducted to measure the gene expression of *CYP2B6* in a random sample of 55 patients. The results showed that none of the 10 haplotypes associated with the plasma concentration of methadone *S*-enantiomer in the third significant region of chromosome 19 ([Fig 4(E)](#pgen.1005910.g004)) achieved significance. Only haplotype *A*-*G*-*C* of rs12461727, rs73038469, and rs4803406 in LD block B2 in *CYP2A7P1* and *CYP2B7P1* showed a dose-response pattern and had a *p* value near a marginal significance level (raw *p* = 0.0644). More samples are needed to verify that haplotype *A*-*G*-*C* is an expression quantitative trait locus.

In addition to the association between the *CYP2B6* gene and the plasma concentration of methadone *S*-enantiomers, we discovered that the haplotype SNPs located in the gene regions of *SPON1* and *GSG1L* were significantly associated with the plasma concentration of methadone *S*-enantiomers. To investigate the roles of *SPON1* and *GSG1L* genes and their coexpression with *CYP2B6* in a regulated plasma concentration of *S*-methadone in the liver, the cultured human liver cell line HepG2 was incubated with the CAR agonist CITCO, a CYP2B6 inducer [[31](#pgen.1005910.ref031)]. The mRNA expression levels of *CYP2B6*, *SPON1*, and *GSG1L* in HepG2 cells were measured; the results revealed that the gene expression of *CYP2B6*, *SPON1*, and *GSG1L* can be activated concomitantly through the CAR activation pathway. Methadone reportedly activates the CAR [[32](#pgen.1005910.ref032)], a phenomenon that partially explains why the three genes were significantly associated with the plasma concentrations of *S*-methadone. *SPON1* was expressed in all the tissues, including the brain, lymph node, kidney, and intestine ([S10(B) Fig](#pgen.1005910.s017)). *GSG1L* was expressed mostly in connective tissue and the tissues of the brain, eye, and testis ([S10(C) Fig](#pgen.1005910.s017)). Both *SPON1* and *GSG1L* were expressed in brain tissue. A previous study found that *GSG1L* was an α-amino-3-hydroxy-5-methyl-4-isoxazolepropionic acid (AMPA) glutamate receptor relevant to neurotransmission [[33](#pgen.1005910.ref033)]. In the present study, a permutation-based association test with 100,000 random shuffles was conducted, revealing that haplotype *C-T-G-C* was associated with treatment emergent symptoms for adverse events related to MMT (empirical *p* = 1.82 × 10^−2^). In addition, the preliminary interaction analysis suggested that there was an interactive effect between haplotype *T*-*C* in *SPON1* and haplotype *T*-*C*-*G*-*C*-*T* in *GSG1L* (raw *p* = 7.55 × 10^−3^). According to these findings, the roles of *SPON1* and *GSG1L* in the brain and the methadone action pathway will be further investigated in future studies.

The methadone *R*-enantiomer reportedly tends to be metabolized by the CYP2C19 isozyme [[8](#pgen.1005910.ref008), [9](#pgen.1005910.ref009)]. Our previous study found that a genotype combination of two functional SNPs (rs4986893 in exon 4 and rs4244285 in exon 5) on *CYP2C19* were associated with the plasma concentration of methadone *R*-enantiomers (*p* = 0.007; the *p* values of rs4986893 and rs4244285 were 0.8 and 0.013, respectively). Moreover, the genotype combination can provide a gene dosage model for classifying poor, intermediate, and extensive metabolizers [[34](#pgen.1005910.ref034)]. On the basis of the same dataset, the current analysis further revealed that rs4986893, rs4244285, and their genotype combination accounted for 0.198%, 2.014%, and 2.351% of the variation in the plasma concentration of methadone *R*-enantiomer, respectively.

This genome-wide single-locus association analysis did not find a single SNP on *CYP2C19* that exceeded the genome-wide significance level in the association analysis of the plasma concentration of methadone *R*-enantiomer. In addition, rs4244285 on *CYP2C19* was not designed in the Axiom CHB arrays. In contrast to the candidate gene study, this genome-wide single-locus association analysis identified that only the SNP rs17180299 (raw *p* = 2.24 × 10^−8^) had a strong signal of genetic association. Patients with more allele *G* of rs17180299 on average had higher plasma concentration of methadone *R*-enantiomer. Rs17180299, which accounted for 9.541% of the variation in the plasma concentration of methadone *R*-enantiomer, accounted for a substantially greater variation than did rs4986893, rs4244285, and their genotype combination. This SNP may be crucial for predicting the plasma concentration of methadone *R*-enantiomer and classifying heroin-dependent patients with different methadone metabolism responses. Permutation-based association tests with 100,000 random shuffles further revealed that haplotype *C-G-G-C-G* ([Fig 3(D)](#pgen.1005910.g003)), which contained rs17180299, was associated with the weight-adjusted methadone dosage (empirical *p* = 2.46 × 10^−2^). Previous studies have suggested that methadone *R*-enantiomers induce an enhanced analgesic effect; however, it also exerts an adverse and toxic cardiac effect [[35](#pgen.1005910.ref035), [36](#pgen.1005910.ref036)]. Therefore, patients with more allele *G* can be administered lower doses of methadone *R*-enantiomers to minimize the adverse effect.

Although rs17180299 is located in an intergenic region, the data from the Roadmap Epigenomics Projects revealed a relationship between rs17180299 and heterochromatic histone H3 lysine 9 trimethylation (H3K9me3) in the primary T regulatory cells from peripheral blood (*p*-value signal score was 2.47354). Several studies reported the association of histone mark H3K9me3 and substance abuse. For example, cocaine dynamically regulates H3K9me3 [[37](#pgen.1005910.ref037)] and H3K9me3 is a key player in the regulation of affective vulnerability and substance abuse [[38](#pgen.1005910.ref038)]. The information suggests that rs17180299 may play a role in the regulation of plasma concentration of methadone *R*-enantiomer through epigenetic histone modification of H3K9me3.

We also imputed untyped SNPs using the SHAPEIT software [[39](#pgen.1005910.ref039)] and IMPUTE2 software [[40](#pgen.1005910.ref040)] and ran association analysis in a 2-Mb flanking region of rs17180299 on chromosome 9. Strong association signals were found in the almost complete LD region closest to rs17180299 ([S11 Fig](#pgen.1005910.s018)). Among the significant SNPs, the imputed SNP rs17083093 at 82,938,858 bp (Raw *p* = 6.80 ×10^−9^) was even more significant than rs17180299 at 82,944,201 bp (Raw *p* = 2.24 ×10^−8^). All of the significant SNPs were located in intergenic regions and only rs17180299 provides direct biological explanations for substance abuse.

In general, the available sample sizes in pharmacogenomic studies are smaller than the sample sizes in genome-wide association studies for complex diseases. In this MMT studies, most patients were not willing to come to the clinic in daily basis for more than 5 days to fulfil the requirement of the steady-state status in which the half-life of methadone was considered with an average of 24 hours. This study overcame the difficulty and recruited a moderate sample size of steady-state MMT patients through multicenter collaborations. Furthermore, we successfully replicated that *CYP2B6*, *SPON1*, and *GSG1L* are associated with the plasma concentration of methadone *S*-enantiomer. However, we may have overlooked some important findings such as the significance of rs17180299 and some important haplotypes because of a small sample size in the replication study. SNP rs17180299 cannot be replicated because of a low frequency of minor allele *G* in the replication study; several MMT patients with genotype *GG* were observed at the discovery stage but no patients with genotype *GG* were observed at the replication stage. We calculated the sample size required to replicate the association of significant individual haplotypes by using Quanto software [[41](#pgen.1005910.ref041)]. For significant individual haplotypes with a stronger effect (e.g., the haplotypes on *CYP2B6*), on the basis of the current sample size, we had a power of 0.65 to replicate the results. For the haplotypes with a weaker effect (e.g., the haplotypes on *GSG1L*), on the basis of the current sample size, we had only a power of 0.25 to replicate the results. In the latter situation, we will need 330 replication samples in order to attain a power of 0.80. On the other hand, the replication of the association findings was mainly based on haplotype-specific association tests. Further investigation should be undertaken to prevent spurious association findings. We also evaluated the influence of “winner’s curse” on the estimation of the proportion of variation explained by the five replicated haplotypes. The replicated haplotypes accounted for 17.75% and 19.51% of the variation in the plasma concentration of *S*-methadone at the discovery stage and replication stage, respectively. Because the two proportions of variation explained were close, this information suggested no winner’s curse in this study. In this study, we examined samples from a single East-Asian population to reduce false-positive and false-negative results caused by genetic heterogeneity. Currently, no other genome-wide pharmacogenomic study on the plasma concentration of methadone in other populations exists for comparison.

In conclusion, this study was the first genome-wide pharmacogenomic study to identify genes associated with the plasma concentrations of methadone *R*- and *S*-enantiomers and their respective metabolites in a methadone maintenance cohort of heroin-dependent patients. A significant intergenic SNP (rs17180299) and four intergenic haplotypes on chromosome 9 accounted for 11.350% of the variation in the overall plasma concentration of methadone *R*-enantiomers. Regarding the plasma concentration of methadone *S*-enantiomers, 17 significant haplotypes were identified: two haplotypes on *SPON1*, five haplotypes on *GSG1L*, and 10 haplotypes on *CYP450* genes, including *CYP2B6*. These haplotypes accounted for approximately one-fourth of the variation in the plasma concentration of *S*-methadone. These results revealed that additional candidate genes are associated with the plasma concentration of methadone, affording insights into the mechanism of methadone metabolism, and facilitating further enhancement of MMT.

## Materials and Methods

### Ethics statement

Written informed consent was obtained from all participants during their initial clinical visit. The genome-wide pharmacogenomic study was approved by the Internal Review Boards of National Health Research Institutes (Zhunan, Taiwan; Permit Number: EC0970504) and the seven participating clinical centers, Taoyuan Mental Hospital, En-Chu-Kong Hospital, Far-Eastern Memorial Hospital, Taipei City Hospital Song-De Campus, China Medical University and Hospital, Taipei City Hospital Yang-Ming Campus, and Wei Gong Memorial Hospital (Miaoli, Taiwan). In addition, this study was registered with the U.S. National Institutes of Health Clinical Trial Registry ([http://www.clinicaltrial.gov/ct/show/NCT01059747](http://www.clinicaltrial.gov/ct/show/NCT01059747)). The replication study was approved by the Institutional Review Boards of China Medical University (Taichung, Taiwan; Permit Number: DMR101-IRB1-218 EC0970504).

### Study participants

This genome-wide pharmacogenomic study recruited 360 heroin-dependent patients who underwent the MMT. The inclusion and exclusion criteria for participation are described as follows. This study included patients who (1) resided in Taiwan and were Han Chinese; (2) were older than 18 years; (3) could participate in a clinical assessment in Chinese (including Mandarin and Taiwanese dialects); (4) were diagnosed as a patient with heroin dependence based on the Diagnostic and Statistical Manual of Mental Disorders, Fourth Edition; (5) underwent the MMT for at least 3 months; (6) received the MMT regularly in the past one week; (7) had no change of >10 mg in methadone dosage throughout the past one week; and (8) signed the informed consent form. This study excluded patients (1) who were pregnant and (2) who had severe cognitive impairment or severe comorbid mental disorders, including organic mental disorders and schizophrenia. For replication, we recruited another 78 independent samples under the same inclusion and exclusion criteria of the genome-wide pharmacogenomic study.

### Methadone and its metabolites in the plasma

The plasma concentrations of methadones and its metabolite EDDP enantiomers were measured using high-performance liquid chromatography according to the settings described in our previous report [[41](#pgen.1005910.ref041)]. A Chiral-AGP analytical column was used (5 μm, 100 × 3 mm; Chrom Tech, Cheshire, UK) with a Chiral-AGP guard column (10 × 3 mm) (Chrom Tech). Methadone, EDDP, and amitriptyline as an internal standard (40 ng) were extracted from the plasma samples by using a C18-E Strata solid-phase extraction column with a 100-mg/mL capacity (Phenomenex, Torrance, CA). In addition, 800-μL aliquots of each serum sample and 40 ng of the amitriptyline internal standard were added. The column was then washed and the remaining compounds were eluted. The collected eluent was then evaporated and the remaining residue was dissolved in 100 μL of mobile phase. A total sample volume of 50 μL was chromatographed. The intraday and interday coefficients of variation were 3.3% and 6.6% for *R*-methadone, 2.5% and 5.6% for *S*-methadone, 1.6% and 3.9% for *R*-EDDP, and 2.8% and 5.5% for *S*-EDDP, respectively. The recovery rates for *R*-methadone, *S*-methadone, *R*-EDDP, and *S*-EDDP were 109.0 ± 7.6%, 96.7 ± 8.6%, 96.6 ± 6.6%, and 87.4 ± 3.2%, respectively. The recovery rate for the internal standard was 60.2 ± 4.8%. The details are described in our previous publications [[42](#pgen.1005910.ref042), [43](#pgen.1005910.ref043)].

### DNA samples and SNP genotyping experiments

The genomic DNA of 360 Han Chinese MMT patients at discovery stage and 78 Han Chinese MMT patients at replication stage were isolated from leukocytes by using a Puregene Blood kit C (QIAGEN Sciences, Germantown, Maryland). The DNA concentration was measured and adjusted to 15–20 ng/μL by using a NanoDrop ND-1000 Spectrophotometer (NanoDrop Technologies, DE). All samples were genotyped at the National Center of Genomic Medicine (Taipei, Taiwan) by using the Axiom Genome-Wide CHB 1 Array (Affymetrix, Inc., San Diego, CA) following the manufacturer’s protocols ([http://www.affymetrix.com](http://www.affymetrix.com)). The genotype calling was performed using the Genotyping Console 4.0 with default parameters ([http://www.affymetrix.com](http://www.affymetrix.com)). This SNP genotyping platform optimized the genomic coverage of common SNPs having a minor allele frequency of >5% of the genomes of Han Chinese populations. In total, this array comprised 642,832 SNPs selected from the Axiom Genomic Database, which includes SNP probes from the International HapMap Project ([http://hapmap.ncbi.nlm.nih.gov/](http://hapmap.ncbi.nlm.nih.gov/)), the 1,000 Genomes Project ([http://www.1000genomes.org/](http://www.1000genomes.org/)), and the dbSNP database ([http://www.ncbi.nlm.nih.gov/SNP/](http://www.ncbi.nlm.nih.gov/SNP/)). Based on the NetAffyx Annotation Version 32.1, the SNPs included 625,360 autosomal SNPs, 14,342 nonautosomal SNPs, 59 SNPs without chromosome information, and 3,071 control SNPs.

### Preparation of liver cell lines and real-time polymerase chain reaction (RT-PCR) analysis

CAR agonist CITCO was purchased from Sigma-Aldrich (St. Louis, MO). CITCO (1 mM) was dissolved in 100% dimethyl sulphoxide (DMSO) and diluted with 10 times as much culture media (final DMSO concentration of 0.1%). HepG2 cells (Passage 96) were obtained from the Culture Collection and Research Centre (Food Industry Research and Development Institute, Hsinchu, Taiwan). The cells were cultured in Eagle’s Minimum Essential Medium supplemented with 10% fetal bovine serum (FBS) (Biological Industries, Kibbutz Beit Haemek, Israel). The cultures were maintained in a humidified atmosphere with 5% CO_2_ at 37°C, and the medium was refreshed every three or four days.

The HepG2 cells (Passages 98–100) were seeded on 10-cm petri dishes with a density of 2.5 × 10^5^ cells/10 mL of culture medium supplemented with 10% FBS. After 24 h of overnight cellular attachment, the medium was replaced with a medium containing different concentrations of CITCO or a vehicle (0.1% DMSO). The drug was incubated with the cells for 24 h. The entire cellular RNA was isolated using a Trizol reagent according to the manufacturer’s protocol (Life Technologies, Carlsbad, CA).

RT-PCR was conducted using a RevertAid H Minus First Strand cDNA Synthesis Kit (Fermentas, Waltham, MA) featuring a random hexamer and the RT-PCR operating on an ABI StepOne Plus System. The RT-PCR was performed for *CYP2B6*, *SPON1*, *GSG1L*, and a housekeeping gene, TATA-box binding protein (*TBP*), by using predesigned gene-specific TaqMan probes and primer sets (Hs03044634_m1 for *CYP2B6*, Hs01120488_m1 for *SPON1*, Hs00896151_m1 for *GSG1L*, and Hs00920497_m1 for *TBP*) purchased from Applied Biosystems (Applied Biosystems, Foster City, CA). Gene expression was quantified relative to *TBP* expression by using ABI StepOne Plus Software. The relative expression level of *CYP2B6*, *SPON1*, or *GSG1L* compared with that of *TBP* was defined as CT = [CT_target gene_CT_TBP_], where CT is the cycle threshold. The ratio of the mRNA of a target gene to the mRNA of *TBP* was calculated by using 2 ^CT^. The relative expression level of *CYP2B6*, *SPON1*, or *GSG1L* within various concentrations of the CITCO was calculated and compared according to an unpaired Student’s *t* test by using GraphPAD Prism, Version 5 (GraphPad Software, Inc., La Jolla, CA). Data were represented as mean ± SD of triplicates of the experiment.

### Statistical analyses

[Fig 1(A)](#pgen.1005910.g001) depicts a flowchart of the overall statistical analyses. Quality of samples and SNPs were controlled, and genome-wide single-locus association tests and haplotype association tests were performed. The identified SNPs, haplotypes, and genes were further examined through a replication study. The details of analysis procedures are described below.

The quality of samples and SNPs was examined according to the procedures in the analysis protocol [[44](#pgen.1005910.ref044)] in addition to examinations developed in this study. [Fig 1(B)](#pgen.1005910.g001) shows a summary of the procedures and results. Prior to examination, 3,071 control SNPs provided in the Axiom Genome-Wide CHB 1 Array were removed. Regarding sample quality control, first, we assessed the concordance of the sex information according to the self-reported gender and homozygosity pattern on X chromosomes; male participants with a homozygosity rate of <0.2 and female participants with a homozygosity rate of >0.8 were identified as having a sex error, and the homozygosity pattern of their X chromosomes was further examined. After the sex examination, we excluded 14,342 nonautosomal SNPs from the subsequent analyses. Second, we calculated the GCR; patients who had a GCR of <0.95 were identified as poor-quality samples. Third, the average genome-wide homozygosity rate was calculated and the 99.7% confidence interval was constructed; patients who had a genome-wide homozygosity rate outside the confidence interval were identified as outlying patients, and the pattern of their run of homozygosity was further examined. Fourth, we identified patients with unknown familial relationships; the IBD of each pair of individuals was estimated, and the pairs of individuals who had an estimated IBD of >0.1875 (i.e., greater than the average expected IBD values for second- and third-degree relatives) were identified as having cryptic relatedness. Finally, we identified individuals who showed evidence of divergent ancestry; the confidence bands of the first two principal components of allele frequency were constructed for each of five populations, including the Han Chinese population in Taiwan and four additional populations from the International HapMap II Project. The patients from Taiwan whose values of the first two principal components were located outside the confidence bands of the Han Chinese population in Taiwan were regarded as having divergent ancestry.

Regarding SNP quality, we first calculated the GCR of the SNPs; the SNPs that had a GCR of <0.95 were identified as poor-quality SNPs. Second, the MAF was estimated using an allele counting approach; the SNPs that had an MAF of <0.01 were identified as nonpolymorphic markers. We examined the HWE by using an exact HWE test [[45](#pgen.1005910.ref045)]; SNPs that had a *p* value that was less than the Bonferroni genome-wide significance level were identified as deviations from the HWE. Finally, after the SNPs associated with the four quantitative traits of study were identified, we examined the SNP cluster plots to ensure that the genotype calls were reliable.

In the aforementioned quality control of samples and SNPs, the patterns of homozygosity on X chromosomes and excessive runs of homozygosity in the human genome were examined using LOHAS software [[46](#pgen.1005910.ref046)]. The R programs in this study were used to construct principal components of allele frequency and their confidence bands and identify patients with divergent ancestry. Other statistical quality control procedures were performed using PLINK software [[47](#pgen.1005910.ref047)].

Four quantitative traits (i.e., plasma concentrations of the methadone *R*- and *S*-enantiomers and their metabolites) were examined in the genetic association analysis. Before the genetic association analysis, data transformation to normality was performed as follows: the quantitative traits were transformed through a winsorization procedure with a threshold of 0.01, followed by a log transformation, and then standardized by subtracting the mean from the traits and dividing the results by the standard deviation. Normality of the transformed data was tested by a Kolmogorov–Smirnov goodness-of-fit test [[23](#pgen.1005910.ref023)]. The aforementioned data transformation was completed using R programs.

The normalized quantitative traits were studied through genetic association analyses, including the genome-wide single-locus association test and haplotype association test. The genome-wide single-locus association test was performed according to a linear regression model by using PLINK software [[47](#pgen.1005910.ref047)]. The four normalized quantitative traits were individually considered as dependent variables. The independent variables included a SNP variable coded as 0, 1, and 2 for genotypes *AA*, *Aa*, and *aa*, respectively, and three additional variables for covariate adjustments, including age, sex, and BMI. The *p* value of a one degree of freedom Wald test was obtained for each SNP to examine the additive genetic effect. During each analysis of the four quantitative traits, the *p* values on a scale of -log_10_ obtained from genome-wise single-locus association tests were displayed in a Manhattan plot and a Quantile–Quantile plot. The false discovery rate [[48](#pgen.1005910.ref048)] was determined for a multiple-test correction. Finally, the significant SNPs were depicted in a Regional Association Plot [[49](#pgen.1005910.ref049)].

We performed genome-wide sliding-window haplotype-based association analyses with different window sizes individually. The human genome was partitioned into a series of sliding windows according to a specific window size. For each sliding window, a four-step haplotype analysis was performed as follows: First, haplotype configurations were inferred using a standard expectation-maximization algorithm [[50](#pgen.1005910.ref050)]. Only haplotype configurations with a posterior probability of >0.01 and only haplotypes with a minor haplotype frequency of >0.01 were analyzed in the subsequent steps. Second, an omnibus haplotype association test with linear regression analysis with an adjustment for age, sex, and BMI was performed using PLINK software. The omnibus test, which involved a Wald test with a degree of freedom of *h*-1 (where *h* is the number of distinct haplotypes in the window), was used to examine whether all haplotypes in the study window were associated with an overall study trait.

Third, a window that was significant in the second step was expanded to encompass the flanking region on either side. The width of extension on each side was one half of the window width (i.e., the physical position of the last SNP minus the position of the first SNP in the window). If windows overlapped, they were combined to form a new window before the region was expanded. Finally, in each extended genomic region defined at the third step, LD blocks were constructed based on the Gabriel’s method [[24](#pgen.1005910.ref024)] by using HAPLOVIEW [[51](#pgen.1005910.ref051)]. In each LD block, a linear regression analysis with an adjustment for age, sex, and BMI was conducted using PLINK to examine the association between the study traits and individual haplotypes versus that of other haplotypes. All haplotypes with a minor haplotype frequency of >0.01 were analyzed and the raw *p* values of individual haplotypes were adjusted for multiple testing by using the false discovery rate in the entire extended regions of the significant windows and also the Bonferroni correction by the extended regions of the significant windows. Only the haplotypes which passed both of the corrections were claimed to be significant.

For each of the studied quantitative traits, the proportion of variation accounted for by the significant SNPs and haplotypes were calculated as follows. Based on the variable(s) or covariate(s) in a regression model, the next SNP or haplotype was included if the SNP or haplotype produced the maximal increment of model R^2^. Using this variable inclusion procedure, we included all SNPs and haplotypes significantly associated with a quantitative trait in the regression model sequentially. Model R^2^ revealed the coefficient of determination of a full regression model that contained one or more haplotypes. In addition, the marginal R^2^ was calculated for each SNP or haplotype according to the regression model that contained only that SNP or haplotype.

In a replication study, we evaluated quality control of samples and SNPs using the same quality control procedures at the discovery stage. The identified SNPs and haplotypes in the genome-wide pharmacogenomic study were further examined using the same genetic association tests at the discovery stage. A meta-analysis by combining *p*-values at the discovery and replication stages was performed by using Fisher’s product *p*-value method [[52](#pgen.1005910.ref052)]. [S8(A) Fig](#pgen.1005910.s015) depicts a flowchart and results of the statistical analyses and [S8(B) Fig](#pgen.1005910.s015) shows a summary of the procedures and results of quality control of samples and SNPs in the replication study.

## Supporting Information

## Acknowledgments

We acknowledge those at the National Center for Genomic Medicine at Academia Sinica for their assistance in genotyping. We thank the Clinical Trial Information Management System (CTIMeS) at National Health Research Institutes for collecting and cleaning clinical data for this study. We thank research nurses for recruiting patients. We sincerely thank two anonymous reviewers for their very constructive and insightful comments that helped in preparing our manuscript.

## Data Availability

All phenotype and genotype files are available from the GEO database (accession number GSE78098).

## Funding Statement

HCY was supported by grants from the Ministry of Science and Technology of Taiwan (MOST 103-2314-B-001-008-MY3 and NSC100-2314-B-001-005-MY3) and the Career Development Award of Academia Sinica (AS-100-CDA-M03). YLL was supported by grants from the National Research Program for Genomic Medicine [NSC 100-3112-B-400-015], National Science Council [NSC 100-2314-B-400-002-MY3] and the National Health Research Institutes, Taiwan [NP-104-PP-04, NP-105-PP-04, NP-104-SP-04 and NP-105-SP-04]. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Data Availability Statement

All phenotype and genotype files are available from the GEO database (accession number GSE78098).

### Supplementary Materials

### Data Availability Statement

All phenotype and genotype files are available from the GEO database (accession number GSE78098).

## References

1. Mark TL, Woody GE, Juday T, Kleber HD. The economic costs of heroin addiction in the United States. Drug Alcohol Depend. 2001;61(2):195–206. Epub 2001/01/04. S0376-8716(00)00162-9 [pii]. .  [DOI](https://doi.org/10.1016/s0376-8716(00)00162-9) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11137285/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Alcohol%20Depend&title=The%20economic%20costs%20of%20heroin%20addiction%20in%20the%20United%20States&author=TL%20Mark&author=GE%20Woody&author=T%20Juday&author=HD%20Kleber&volume=61&issue=2&publication_year=2001&pages=195-206&pmid=11137285&doi=10.1016/s0376-8716(00)00162-9&)

2. Brown R. Heroin dependence. WMJ. 2004;103(4):20–6. Epub 2004/10/16. .  [PubMed](https://pubmed.ncbi.nlm.nih.gov/15481866/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=WMJ&title=Heroin%20dependence&author=R%20Brown&volume=103&issue=4&publication_year=2004&pages=20-6&pmid=15481866&)

3. Uchtenhagen A. Substitution management in opioid dependence. Journal of neural transmission Supplementum. 2003;(66):33–60. .  [DOI](https://doi.org/10.1007/978-3-7091-0541-2_3) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14582802/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20of%20neural%20transmission%20Supplementum&title=Substitution%20management%20in%20opioid%20dependence&author=A%20Uchtenhagen&issue=66&publication_year=2003&pages=33-60&pmid=14582802&doi=10.1007/978-3-7091-0541-2_3&)

4. Mattick RP, Breen C, Kimber J, Davoli M. Methadone maintenance therapy versus no opioid replacement therapy for opioid dependence. The Cochrane database of systematic reviews. 2009;(3):CD002209 10.1002/14651858.CD002209.pub2 .  [DOI](https://doi.org/10.1002/14651858.CD002209.pub2) | [PMC free article](/articles/PMC7097731/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19588333/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=The%20Cochrane%20database%20of%20systematic%20reviews&title=Methadone%20maintenance%20therapy%20versus%20no%20opioid%20replacement%20therapy%20for%20opioid%20dependence&author=RP%20Mattick&author=C%20Breen&author=J%20Kimber&author=M%20Davoli&issue=3&publication_year=2009&pages=CD002209&pmid=19588333&doi=10.1002/14651858.CD002209.pub2&)

5. Mattick RP, Breen C, Kimber J, Davoli M. Methadone maintenance therapy versus no opioid replacement therapy for opioid dependence. The Cochrane database of systematic reviews. 2002;(4):CD002209 10.1002/14651858.CD002209 .  [DOI](https://doi.org/10.1002/14651858.CD002209) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12519570/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=The%20Cochrane%20database%20of%20systematic%20reviews&title=Methadone%20maintenance%20therapy%20versus%20no%20opioid%20replacement%20therapy%20for%20opioid%20dependence&author=RP%20Mattick&author=C%20Breen&author=J%20Kimber&author=M%20Davoli&issue=4&publication_year=2002&pages=CD002209&pmid=12519570&doi=10.1002/14651858.CD002209&)

6. Mattick RP, Breen C, Kimber J, Davoli M. Methadone maintenance therapy versus no opioid replacement therapy for opioid dependence. The Cochrane database of systematic reviews. 2003;(2):CD002209 10.1002/14651858.CD002209 .  [DOI](https://doi.org/10.1002/14651858.CD002209) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12804430/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=The%20Cochrane%20database%20of%20systematic%20reviews&title=Methadone%20maintenance%20therapy%20versus%20no%20opioid%20replacement%20therapy%20for%20opioid%20dependence&author=RP%20Mattick&author=C%20Breen&author=J%20Kimber&author=M%20Davoli&issue=2&publication_year=2003&pages=CD002209&pmid=12804430&doi=10.1002/14651858.CD002209&)

7. Gerber JG, Rhodes RJ, Gal J. Stereoselective metabolism of methadone N-demethylation by cytochrome P4502B6 and 2C19. Chirality. 2004;16(1):36–44. 10.1002/chir.10303 .  [DOI](https://doi.org/10.1002/chir.10303) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14628297/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Chirality&title=Stereoselective%20metabolism%20of%20methadone%20N-demethylation%20by%20cytochrome%20P4502B6%20and%202C19&author=JG%20Gerber&author=RJ%20Rhodes&author=J%20Gal&volume=16&issue=1&publication_year=2004&pages=36-44&pmid=14628297&doi=10.1002/chir.10303&)

8. Chang Y, Fang WB, Lin SN, Moody DE. Stereo-selective metabolism of methadone by human liver microsomes and cDNA-expressed cytochrome P450s: a reconciliation. Basic & clinical pharmacology & toxicology. 2011;108(1):55–62. 10.1111/j.1742-7843.2010.00628.x  [DOI](https://doi.org/10.1111/j.1742-7843.2010.00628.x) | [PMC free article](/articles/PMC3005981/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20825389/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Basic%20&%20clinical%20pharmacology%20&%20toxicology&title=Stereo-selective%20metabolism%20of%20methadone%20by%20human%20liver%20microsomes%20and%20cDNA-expressed%20cytochrome%20P450s:%20a%20reconciliation&author=Y%20Chang&author=WB%20Fang&author=SN%20Lin&author=DE%20Moody&volume=108&issue=1&publication_year=2011&pages=55-62&pmid=20825389&doi=10.1111/j.1742-7843.2010.00628.x&)

9. Wang SC, Ho IK, Wu SL, Liu SC, Kuo HW, Lin KM, et al. Development of a method to measure methadone enantiomers and its metabolites without enantiomer standard compounds for the plasma of methadone maintenance patients. Biomedical chromatography: BMC. 2010;24(7):782–8. 10.1002/bmc.1363 .  [DOI](https://doi.org/10.1002/bmc.1363) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19904716/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biomedical%20chromatography:%20BMC&title=Development%20of%20a%20method%20to%20measure%20methadone%20enantiomers%20and%20its%20metabolites%20without%20enantiomer%20standard%20compounds%20for%20the%20plasma%20of%20methadone%20maintenance%20patients&author=SC%20Wang&author=IK%20Ho&author=SL%20Wu&author=SC%20Liu&author=HW%20Kuo&volume=24&issue=7&publication_year=2010&pages=782-8&pmid=19904716&doi=10.1002/bmc.1363&)

10. Totah RA, Sheffels P, Roberts T, Whittington D, Thummel K, Kharasch ED. Role of CYP2B6 in stereoselective human methadone metabolism. Anesthesiology. 2008;108(3):363–74. 10.1097/ALN.0b013e3181642938 .  [DOI](https://doi.org/10.1097/ALN.0b013e3181642938) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18292673/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Anesthesiology&title=Role%20of%20CYP2B6%20in%20stereoselective%20human%20methadone%20metabolism&author=RA%20Totah&author=P%20Sheffels&author=T%20Roberts&author=D%20Whittington&author=K%20Thummel&volume=108&issue=3&publication_year=2008&pages=363-74&pmid=18292673&doi=10.1097/ALN.0b013e3181642938&)

11. Totah RA, Allen KE, Sheffels P, Whittington D, Kharasch ED. Enantiomeric metabolic interactions and stereoselective human methadone metabolism. The Journal of pharmacology and experimental therapeutics. 2007;321(1):389–99. 10.1124/jpet.106.117580 .  [DOI](https://doi.org/10.1124/jpet.106.117580) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17259447/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=The%20Journal%20of%20pharmacology%20and%20experimental%20therapeutics&title=Enantiomeric%20metabolic%20interactions%20and%20stereoselective%20human%20methadone%20metabolism&author=RA%20Totah&author=KE%20Allen&author=P%20Sheffels&author=D%20Whittington&author=ED%20Kharasch&volume=321&issue=1&publication_year=2007&pages=389-99&pmid=17259447&doi=10.1124/jpet.106.117580&)

12. Levran O, Peles E, Hamon S, Randesi M, Adelson M, Kreek MJ. CYP2B6 SNPs are associated with methadone dose required for effective treatment of opioid addiction. Addict Biol. 2013;18(4):709–16. 10.1111/j.1369-1600.2011.00349.x  [DOI](https://doi.org/10.1111/j.1369-1600.2011.00349.x) | [PMC free article](/articles/PMC3735354/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21790905/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Addict%20Biol&title=CYP2B6%20SNPs%20are%20associated%20with%20methadone%20dose%20required%20for%20effective%20treatment%20of%20opioid%20addiction&author=O%20Levran&author=E%20Peles&author=S%20Hamon&author=M%20Randesi&author=M%20Adelson&volume=18&issue=4&publication_year=2013&pages=709-16&pmid=21790905&doi=10.1111/j.1369-1600.2011.00349.x&)

13. Bunten H, Liang WJ, Pounder DJ, Seneviratne C, Osselton D. OPRM1 and CYP2B6 gene variants as risk factors in methadone-related deaths. Clinical pharmacology and therapeutics. 2010;88(3):383–9. 10.1038/clpt.2010.127 .  [DOI](https://doi.org/10.1038/clpt.2010.127) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20668445/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clinical%20pharmacology%20and%20therapeutics&title=OPRM1%20and%20CYP2B6%20gene%20variants%20as%20risk%20factors%20in%20methadone-related%20deaths&author=H%20Bunten&author=WJ%20Liang&author=DJ%20Pounder&author=C%20Seneviratne&author=D%20Osselton&volume=88&issue=3&publication_year=2010&pages=383-9&pmid=20668445&doi=10.1038/clpt.2010.127&)

14. Bunten H, Liang WJ, Pounder D, Seneviratne C, Osselton MD. CYP2B6 and OPRM1 gene variations predict methadone-related deaths. Addiction biology. 2011;16(1):142–4. .  [DOI](https://doi.org/10.1111/j.1369-1600.2010.00274.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21158011/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Addiction%20biology&title=CYP2B6%20and%20OPRM1%20gene%20variations%20predict%20methadone-related%20deaths&author=H%20Bunten&author=WJ%20Liang&author=D%20Pounder&author=C%20Seneviratne&author=MD%20Osselton&volume=16&issue=1&publication_year=2011&pages=142-4&pmid=21158011&doi=10.1111/j.1369-1600.2010.00274.x&)

15. Tsai HJ, Wang SC, Liu SW, Ho IK, Chang YS, Tsai YT, et al. Assessment of CYP450 genetic variability effect on methadone dose and tolerance. Pharmacogenomics. 2014;15(7):977–86. Epub 2014/06/24. 10.2217/pgs.14.19 .  [DOI](https://doi.org/10.2217/pgs.14.19) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24956251/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Assessment%20of%20CYP450%20genetic%20variability%20effect%20on%20methadone%20dose%20and%20tolerance&author=HJ%20Tsai&author=SC%20Wang&author=SW%20Liu&author=IK%20Ho&author=YS%20Chang&volume=15&issue=7&publication_year=2014&pages=977-86&pmid=24956251&doi=10.2217/pgs.14.19&)

16. Wang SC, Ho IK, Tsou HH, Tian JN, Hsiao CF, Chen CH, et al. CYP2B6 polymorphisms influence the plasma concentration and clearance of the methadone S-enantiomer. Journal of clinical psychopharmacology. 2011;31(4):463–9. 10.1097/JCP.0b013e318222b5dd .  [DOI](https://doi.org/10.1097/JCP.0b013e318222b5dd) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21694616/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20of%20clinical%20psychopharmacology&title=CYP2B6%20polymorphisms%20influence%20the%20plasma%20concentration%20and%20clearance%20of%20the%20methadone%20S-enantiomer&author=SC%20Wang&author=IK%20Ho&author=HH%20Tsou&author=JN%20Tian&author=CF%20Hsiao&volume=31&issue=4&publication_year=2011&pages=463-9&pmid=21694616&doi=10.1097/JCP.0b013e318222b5dd&)

17. Dobrinas M, Crettol S, Oneda B, Lahyani R, Rotger M, Choong E, et al. Contribution of CYP2B6 alleles in explaining extreme (S)-methadone plasma levels: a CYP2B6 gene resequencing study. Pharmacogenetics and genomics. 2013;23(2):84–93. 10.1097/FPC.0b013e32835cb2e2 .  [DOI](https://doi.org/10.1097/FPC.0b013e32835cb2e2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23249875/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics%20and%20genomics&title=Contribution%20of%20CYP2B6%20alleles%20in%20explaining%20extreme%20(S)-methadone%20plasma%20levels:%20a%20CYP2B6%20gene%20resequencing%20study&author=M%20Dobrinas&author=S%20Crettol&author=B%20Oneda&author=R%20Lahyani&author=M%20Rotger&volume=23&issue=2&publication_year=2013&pages=84-93&pmid=23249875&doi=10.1097/FPC.0b013e32835cb2e2&)

18. Dennis BB, Bawor M, Thabane L, Sohani Z, Samaan Z. Impact of ABCB1 and CYP2B6 genetic polymorphisms on methadone metabolism, dose and treatment response in patients with opioid addiction: a systematic review and meta-analysis. PLoS ONE. 2014;9(1):e86114 Epub 2014/02/04. 10.1371/journal.pone.0086114 PONE-D-13-29310 [pii].  [DOI](https://doi.org/10.1371/journal.pone.0086114) | [PMC free article](/articles/PMC3906028/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24489693/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&title=Impact%20of%20ABCB1%20and%20CYP2B6%20genetic%20polymorphisms%20on%20methadone%20metabolism,%20dose%20and%20treatment%20response%20in%20patients%20with%20opioid%20addiction:%20a%20systematic%20review%20and%20meta-analysis&author=BB%20Dennis&author=M%20Bawor&author=L%20Thabane&author=Z%20Sohani&author=Z%20Samaan&volume=9&issue=1&publication_year=2014&pages=e86114&pmid=24489693&doi=10.1371/journal.pone.0086114&)

19. Tsai HJ, Wang SC, Tian JN, Chang TK, Ho IK, Hsiao CF, et al. PXR polymorphisms interacted with CYP2B6 polymorphisms on methadone metabolites. Journal of clinical psychopharmacology. 2013;33(1):137–40. 10.1097/01.jcp.0000426186.34421.de .  [DOI](https://doi.org/10.1097/01.jcp.0000426186.34421.de) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23288240/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20of%20clinical%20psychopharmacology&title=PXR%20polymorphisms%20interacted%20with%20CYP2B6%20polymorphisms%20on%20methadone%20metabolites&author=HJ%20Tsai&author=SC%20Wang&author=JN%20Tian&author=TK%20Chang&author=IK%20Ho&volume=33&issue=1&publication_year=2013&pages=137-40&pmid=23288240&doi=10.1097/01.jcp.0000426186.34421.de&)

20. Crettol S, Deglon JJ, Besson J, Croquette-Krokar M, Hammig R, Gothuey I, et al. ABCB1 and cytochrome P450 genotypes and phenotypes: influence on methadone plasma levels and response to treatment. Clin Pharmacol Ther. 2006;80(6):668–81. Epub 2006/12/21. S0009-9236(06)00390-0 [pii] 10.1016/j.clpt.2006.09.012 .  [DOI](https://doi.org/10.1016/j.clpt.2006.09.012) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17178267/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=ABCB1%20and%20cytochrome%20P450%20genotypes%20and%20phenotypes:%20influence%20on%20methadone%20plasma%20levels%20and%20response%20to%20treatment&author=S%20Crettol&author=JJ%20Deglon&author=J%20Besson&author=M%20Croquette-Krokar&author=R%20Hammig&volume=80&issue=6&publication_year=2006&pages=668-81&pmid=17178267&doi=10.1016/j.clpt.2006.09.012&)

21. Fonseca F, de la Torre R, Diaz L, Pastor A, Cuyas E, Pizarro N, et al. Contribution of cytochrome P450 and ABCB1 genetic variability on methadone pharmacokinetics, dose requirements, and response. PLoS ONE. 2011;6(5):e19527 Epub 2011/05/19. 10.1371/journal.pone.0019527 PONE-D-10-05589 [pii].  [DOI](https://doi.org/10.1371/journal.pone.0019527) | [PMC free article](/articles/PMC3093392/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21589866/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&title=Contribution%20of%20cytochrome%20P450%20and%20ABCB1%20genetic%20variability%20on%20methadone%20pharmacokinetics,%20dose%20requirements,%20and%20response&author=F%20Fonseca&author=R%20de%20la%20Torre&author=L%20Diaz&author=A%20Pastor&author=E%20Cuyas&volume=6&issue=5&publication_year=2011&pages=e19527&pmid=21589866&doi=10.1371/journal.pone.0019527&)

22. Fredheim OM, Moksnes K, Borchgrevink PC, Kaasa S, Dale O. Clinical pharmacology of methadone for pain. Acta anaesthesiologica Scandinavica. 2008;52(7):879–89. Epub 2008/03/12. 10.1111/j.1399-6576.2008.01597.x AAS1597 [pii]. .  [DOI](https://doi.org/10.1111/j.1399-6576.2008.01597.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18331375/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Acta%20anaesthesiologica%20Scandinavica&title=Clinical%20pharmacology%20of%20methadone%20for%20pain&author=OM%20Fredheim&author=K%20Moksnes&author=PC%20Borchgrevink&author=S%20Kaasa&author=O%20Dale&volume=52&issue=7&publication_year=2008&pages=879-89&pmid=18331375&doi=10.1111/j.1399-6576.2008.01597.x&)

23. Massey FJ. The Kolmogorov-Smirnov test for goodness of fit. J Amer Statist Ass. 1951;46(253):68–78. WOS:A1951UY90300007.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Amer%20Statist%20Ass&title=The%20Kolmogorov-Smirnov%20test%20for%20goodness%20of%20fit&author=FJ%20Massey&volume=46&issue=253&publication_year=1951&pages=68-78&)

24. Gabriel SB, Schaffner SF, Nguyen H, Moore JM, Roy J, Blumenstiel B, et al. The structure of haplotype blocks in the human genome. Science. 2002;296(5576):2225–9. 10.1126/science.1069424 WOS:000176379000060.  [DOI](https://doi.org/10.1126/science.1069424) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12029063/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=The%20structure%20of%20haplotype%20blocks%20in%20the%20human%20genome&author=SB%20Gabriel&author=SF%20Schaffner&author=H%20Nguyen&author=JM%20Moore&author=J%20Roy&volume=296&issue=5576&publication_year=2002&pages=2225-9&pmid=12029063&doi=10.1126/science.1069424&)

25. Wang Y, Fong PY, Leung FC, Mak W, Sham PC. Increased gene coverage and Alu frequency in large linkage disequilibrium blocks of the human genome. Gen Mol Res. 2007;6(4):1131–41. Epub 2008/02/15. gmr359 [pii]. .  [PubMed](https://pubmed.ncbi.nlm.nih.gov/18273807/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gen%20Mol%20Res&title=Increased%20gene%20coverage%20and%20Alu%20frequency%20in%20large%20linkage%20disequilibrium%20blocks%20of%20the%20human%20genome&author=Y%20Wang&author=PY%20Fong&author=FC%20Leung&author=W%20Mak&author=PC%20Sham&volume=6&issue=4&publication_year=2007&pages=1131-41&pmid=18273807&)

26. Benjamini Y, Hochberg Y. Controlling the false discovery rate: A practical and powerful approach to multiple testing. J R Statist Soc B. 1995;57(1):289–300. WOS:A1995QE45300017.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20R%20Statist%20Soc%20B&title=Controlling%20the%20false%20discovery%20rate:%20A%20practical%20and%20powerful%20approach%20to%20multiple%20testing&author=Y%20Benjamini&author=Y%20Hochberg&volume=57&issue=1&publication_year=1995&pages=289-300&)

27. Lewontin RC. The detection of linkage disequilibrium in molecular sequence data. Genetics. 1995;140(1):377–88. WOS:A1995QV52100031.  [DOI](https://doi.org/10.1093/genetics/140.1.377) | [PMC free article](/articles/PMC1206563/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7635301/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genetics&title=The%20detection%20of%20linkage%20disequilibrium%20in%20molecular%20sequence%20data&author=RC%20Lewontin&volume=140&issue=1&publication_year=1995&pages=377-88&pmid=7635301&doi=10.1093/genetics/140.1.377&)

28. Leger P, Dillingham R, Beauharnais CA, Kashuba AD, Rezk NL, Fitzgerald DW, et al. CYP2B6 variants and plasma efavirenz concentrations during antiretroviral therapy in Port-au-Prince, Haiti. J Infect Dis. 2009;200(6):955–64. Epub 2009/08/08. 10.1086/605126  [DOI](https://doi.org/10.1086/605126) | [PMC free article](/articles/PMC2754599/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19659438/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Infect%20Dis&title=CYP2B6%20variants%20and%20plasma%20efavirenz%20concentrations%20during%20antiretroviral%20therapy%20in%20Port-au-Prince,%20Haiti&author=P%20Leger&author=R%20Dillingham&author=CA%20Beauharnais&author=AD%20Kashuba&author=NL%20Rezk&volume=200&issue=6&publication_year=2009&pages=955-64&pmid=19659438&doi=10.1086/605126&)

29. Sukasem C, Cressey TR, Prapaithong P, Tawon Y, Pasomsub E, Srichunrusami C, et al. Pharmacogenetic markers of CYP2B6 associated with efavirenz plasma concentrations in HIV-1 infected Thai adults. Br J Clin Pharmacol. 2012;74(6):1005–12. Epub 2012/04/05. 10.1111/j.1365-2125.2012.04288.x  [DOI](https://doi.org/10.1111/j.1365-2125.2012.04288.x) | [PMC free article](/articles/PMC3522814/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22471906/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Clin%20Pharmacol&title=Pharmacogenetic%20markers%20of%20CYP2B6%20associated%20with%20efavirenz%20plasma%20concentrations%20in%20HIV-1%20infected%20Thai%20adults&author=C%20Sukasem&author=TR%20Cressey&author=P%20Prapaithong&author=Y%20Tawon&author=E%20Pasomsub&volume=74&issue=6&publication_year=2012&pages=1005-12&pmid=22471906&doi=10.1111/j.1365-2125.2012.04288.x&)

30. Salem AH, Fletcher CV, Brundage RC. Pharmacometric characterization of efavirenz developmental pharmacokinetics and pharmacogenetics in HIV-infected children. Antimicrobial agents and chemotherapy 2014;58(1):136–43. Epub 2013/10/23. 10.1128/AAC.01738-13 AAC.01738-13 [pii].  [DOI](https://doi.org/10.1128/AAC.01738-13) | [PMC free article](/articles/PMC3910794/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24145522/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Antimicrobial%20agents%20and%20chemotherapy&title=Pharmacometric%20characterization%20of%20efavirenz%20developmental%20pharmacokinetics%20and%20pharmacogenetics%20in%20HIV-infected%20children&author=AH%20Salem&author=CV%20Fletcher&author=RC%20Brundage&volume=58&issue=1&publication_year=2014&pages=136-43&pmid=24145522&doi=10.1128/AAC.01738-13&)

31. Westerink WM, Schoonen WG. Cytochrome P450 enzyme levels in HepG2 cells and cryopreserved primary human hepatocytes and their induction in HepG2 cells. Toxicol In Vitro. 2007;21(8):1581–91. Epub 2007/07/20. S0887-2333(07)00171-3 [pii] 10.1016/j.tiv.2007.05.014 .  [DOI](https://doi.org/10.1016/j.tiv.2007.05.014) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17637504/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Toxicol%20In%20Vitro&title=Cytochrome%20P450%20enzyme%20levels%20in%20HepG2%20cells%20and%20cryopreserved%20primary%20human%20hepatocytes%20and%20their%20induction%20in%20HepG2%20cells&author=WM%20Westerink&author=WG%20Schoonen&volume=21&issue=8&publication_year=2007&pages=1581-91&pmid=17637504&doi=10.1016/j.tiv.2007.05.014&)

32. Tolson AH, Li H, Eddington ND, Wang H. Methadone induces the expression of hepatic drug-metabolizing enzymes through the activation of pregnane X receptor and constitutive androstane receptor. Drug Metab Disposition. 2009;37(9):1887–94. Epub 2009/06/13. 10.1124/dmd.109.027854 dmd.109.027854 [pii].  [DOI](https://doi.org/10.1124/dmd.109.027854) | [PMC free article](/articles/PMC2729327/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19520773/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Disposition&title=Methadone%20induces%20the%20expression%20of%20hepatic%20drug-metabolizing%20enzymes%20through%20the%20activation%20of%20pregnane%20X%20receptor%20and%20constitutive%20androstane%20receptor&author=AH%20Tolson&author=H%20Li&author=ND%20Eddington&author=H%20Wang&volume=37&issue=9&publication_year=2009&pages=1887-94&pmid=19520773&doi=10.1124/dmd.109.027854&)

33. Sumioka A. Auxiliary subunits provide new insights into regulation of AMPA receptor trafficking. Journal of Biochemistry. 2013;153(4):331–7. Epub 2013/02/22. 10.1093/jb/mvt015 mvt015 [pii]. .  [DOI](https://doi.org/10.1093/jb/mvt015) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23426437/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Biochemistry&title=Auxiliary%20subunits%20provide%20new%20insights%20into%20regulation%20of%20AMPA%20receptor%20trafficking&author=A%20Sumioka&volume=153&issue=4&publication_year=2013&pages=331-7&pmid=23426437&doi=10.1093/jb/mvt015&)

34. Wang SC, Ho IK, Tsou HH, Liu SW, Hsiao CF, Chen CH, et al. Functional genetic polymorphisms in CYP2C19 gene in relation to cardiac side effects and treatment dose in a methadone maintenance cohort. Omics: a journal of integrative biology. 2013;17(10):519–26. Epub 2013/09/11. 10.1089/omi.2012.0068  [DOI](https://doi.org/10.1089/omi.2012.0068) | [PMC free article](/articles/PMC3783925/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24016178/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Omics:%20a%20journal%20of%20integrative%20biology&title=Functional%20genetic%20polymorphisms%20in%20CYP2C19%20gene%20in%20relation%20to%20cardiac%20side%20effects%20and%20treatment%20dose%20in%20a%20methadone%20maintenance%20cohort&author=SC%20Wang&author=IK%20Ho&author=HH%20Tsou&author=SW%20Liu&author=CF%20Hsiao&volume=17&issue=10&publication_year=2013&pages=519-26&pmid=24016178&doi=10.1089/omi.2012.0068&)

35. Kristensen K, Blemmer T, Angelo HR, Christrup LL, Drenck NE, Rasmussen SN, et al. Stereoselective pharmacokinetics of methadone in chronic pain patients. Therapeutic drug monitoring. 1996;18(3):221–7. Epub 1996/06/01. .  [DOI](https://doi.org/10.1097/00007691-199606000-00001) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8738759/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Therapeutic%20drug%20monitoring&title=Stereoselective%20pharmacokinetics%20of%20methadone%20in%20chronic%20pain%20patients&author=K%20Kristensen&author=T%20Blemmer&author=HR%20Angelo&author=LL%20Christrup&author=NE%20Drenck&volume=18&issue=3&publication_year=1996&pages=221-7&pmid=8738759&doi=10.1097/00007691-199606000-00001&)

36. Ansermot N, Albayrak Ö, Schläpfer J, Crettol S, Croquette-Krokar M, Bourquin M, et al. Substitution of (R, S)-methadone by (R)-methadone: impact on QTc interval. Archives of internal medicine. 2010;170(6):529–36. 10.1001/archinternmed.2010.26  [DOI](https://doi.org/10.1001/archinternmed.2010.26) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20308640/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Archives%20of%20internal%20medicine&title=Substitution%20of%20(R,%20S)-methadone%20by%20(R)-methadone:%20impact%20on%20QTc%20interval&author=N%20Ansermot&author=%C3%96%20Albayrak&author=J%20Schl%C3%A4pfer&author=S%20Crettol&author=M%20Croquette-Krokar&volume=170&issue=6&publication_year=2010&pages=529-36&pmid=20308640&doi=10.1001/archinternmed.2010.26&)

37. van Leeuwen EM, Kanterakis A, Deelen P, Kattenberg MV, Slagboom PE, de Bakker PI, et al. Population-specific genotype imputations using minimac or IMPUTE2. Nat Protoc. 2015;10(9):1285–96. Epub 2015/08/01. 10.1038/nprot.2015.077 nprot.2015.077 [pii]. .  [DOI](https://doi.org/10.1038/nprot.2015.077) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26226460/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Protoc&title=Population-specific%20genotype%20imputations%20using%20minimac%20or%20IMPUTE2&author=EM%20van%20Leeuwen&author=A%20Kanterakis&author=P%20Deelen&author=MV%20Kattenberg&author=PE%20Slagboom&volume=10&issue=9&publication_year=2015&pages=1285-96&pmid=26226460&doi=10.1038/nprot.2015.077&)

38. Sun L, Dimitromanolakis A, Faye LL, Paterson AD, Waggott D, Bull SB. BR-squared: a practical solution to the winner's curse in genome-wide scans. Hum Genet. 2011;129(5):545–52. Epub 2011/01/20. 10.1007/s00439-011-0948-2  [DOI](https://doi.org/10.1007/s00439-011-0948-2) | [PMC free article](/articles/PMC3074069/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21246217/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Genet&title=BR-squared:%20a%20practical%20solution%20to%20the%20winner's%20curse%20in%20genome-wide%20scans&author=L%20Sun&author=A%20Dimitromanolakis&author=LL%20Faye&author=AD%20Paterson&author=D%20Waggott&volume=129&issue=5&publication_year=2011&pages=545-52&pmid=21246217&doi=10.1007/s00439-011-0948-2&)

39. Horvath S. DNA methylation age of human tissues and cell types. Genome Biol. 2013;14(10):R115 Epub 2013/10/22. gb-2013-14-10-r115 [pii] 10.1186/gb-2013-14-10-r115  [DOI](https://doi.org/10.1186/gb-2013-14-10-r115) | [PMC free article](/articles/PMC4015143/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24138928/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Biol&title=DNA%20methylation%20age%20of%20human%20tissues%20and%20cell%20types&author=S%20Horvath&volume=14&issue=10&publication_year=2013&pages=R115&pmid=24138928&doi=10.1186/gb-2013-14-10-r115&)

40. Howie BN, Donnelly P, Marchini J. A flexible and accurate genotype imputation method for the next generation of genome-wide association studies. PLoS Genet. 2009;5(6):e1000529 Epub 2009/06/23. 10.1371/journal.pgen.1000529  [DOI](https://doi.org/10.1371/journal.pgen.1000529) | [PMC free article](/articles/PMC2689936/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19543373/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet&title=A%20flexible%20and%20accurate%20genotype%20imputation%20method%20for%20the%20next%20generation%20of%20genome-wide%20association%20studies&author=BN%20Howie&author=P%20Donnelly&author=J%20Marchini&volume=5&issue=6&publication_year=2009&pages=e1000529&pmid=19543373&doi=10.1371/journal.pgen.1000529&)

41. Gauderman WJ, Morrison JM. QUANTO 1.1: A computer program for power and sample size calculations for genetic-epidemiology studies. http://hydrauscedu/gxe. 2006.  [http://hydrauscedu/gxe](http://hydrauscedu/gxe)

42. Chen CH, Wang SC, Tsou HH, Ho IK, Tian JN, Yu CJ, et al. Genetic polymorphisms in CYP3A4 are associated with withdrawal symptoms and adverse reactions in methadone maintenance patients. Pharmacogenomics. 2011;12(10):1397–406. Epub 2011/09/10. 10.2217/pgs.11.103 .  [DOI](https://doi.org/10.2217/pgs.11.103) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21902501/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Genetic%20polymorphisms%20in%20CYP3A4%20are%20associated%20with%20withdrawal%20symptoms%20and%20adverse%20reactions%20in%20methadone%20maintenance%20patients&author=CH%20Chen&author=SC%20Wang&author=HH%20Tsou&author=IK%20Ho&author=JN%20Tian&volume=12&issue=10&publication_year=2011&pages=1397-406&pmid=21902501&doi=10.2217/pgs.11.103&)

43. Wang SC, Ho IK, Tsou HH, Tian JN, Hsiao CF, Chen CH, et al. CYP2B6 polymorphisms influence the plasma concentration and clearance of the methadone S-enantiomer. Journal of Clinical Psychopharmacology. 2011;31(4):463–9. Epub 2011/06/23. 10.1097/JCP.0b013e318222b5dd .  [DOI](https://doi.org/10.1097/JCP.0b013e318222b5dd) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21694616/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Clinical%20Psychopharmacology&title=CYP2B6%20polymorphisms%20influence%20the%20plasma%20concentration%20and%20clearance%20of%20the%20methadone%20S-enantiomer&author=SC%20Wang&author=IK%20Ho&author=HH%20Tsou&author=JN%20Tian&author=CF%20Hsiao&volume=31&issue=4&publication_year=2011&pages=463-9&pmid=21694616&doi=10.1097/JCP.0b013e318222b5dd&)

44. Anderson CA, Pettersson FH, Clarke GM, Cardon LR, Morris AP, Zondervan KT. Data quality control in genetic case-control association studies. Nat Protoc. 2010;5(9):1564–73. Epub 2010/11/19. nprot.2010.116 [pii] 10.1038/nprot.2010.116  [DOI](https://doi.org/10.1038/nprot.2010.116) | [PMC free article](/articles/PMC3025522/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21085122/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Protoc&title=Data%20quality%20control%20in%20genetic%20case-control%20association%20studies&author=CA%20Anderson&author=FH%20Pettersson&author=GM%20Clarke&author=LR%20Cardon&author=AP%20Morris&volume=5&issue=9&publication_year=2010&pages=1564-73&pmid=21085122&doi=10.1038/nprot.2010.116&)

45. Wigginton JE, Cutler DJ, Abecasis GR. A note on exact tests of Hardy-Weinberg equilibrium. Am J Hum Genet. 2005;76(5):887–93. WOS:000228198300016.  [DOI](https://doi.org/10.1086/429864) | [PMC free article](/articles/PMC1199378/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15789306/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=A%20note%20on%20exact%20tests%20of%20Hardy-Weinberg%20equilibrium&author=JE%20Wigginton&author=DJ%20Cutler&author=GR%20Abecasis&volume=76&issue=5&publication_year=2005&pages=887-93&pmid=15789306&doi=10.1086/429864&)

46. Yang HC, Chang LC, Huggins RM, Chen CH, Mullighan CG. LOHAS: loss-of-heterozygosity analysis suite. Genet Epidemiol. 2011;35(4):247–60. Epub 2011/02/12. 10.1002/gepi.20573 .  [DOI](https://doi.org/10.1002/gepi.20573) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21312262/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genet%20Epidemiol&title=LOHAS:%20loss-of-heterozygosity%20analysis%20suite&author=HC%20Yang&author=LC%20Chang&author=RM%20Huggins&author=CH%20Chen&author=CG%20Mullighan&volume=35&issue=4&publication_year=2011&pages=247-60&pmid=21312262&doi=10.1002/gepi.20573&)

47. Purcell S, Neale B, Todd-Brown K, Thomas L, Ferreira MAR, Bender D, et al. PLINK: A tool set for whole-genome association and population-based linkage analyses. Am J Hum Genet. 2007;81(3):559–75. 10.1086/519795 WOS:000249128200012.  [DOI](https://doi.org/10.1086/519795) | [PMC free article](/articles/PMC1950838/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17701901/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=PLINK:%20A%20tool%20set%20for%20whole-genome%20association%20and%20population-based%20linkage%20analyses&author=S%20Purcell&author=B%20Neale&author=K%20Todd-Brown&author=L%20Thomas&author=MAR%20Ferreira&volume=81&issue=3&publication_year=2007&pages=559-75&pmid=17701901&doi=10.1086/519795&)

48. Storey JD. A direct approach to false discovery rates. J Roy Stat Soc Ser B (Stat Method). 2002;64(3):479–98.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Roy%20Stat%20Soc%20Ser%20B%20(Stat%20Method)&title=A%20direct%20approach%20to%20false%20discovery%20rates&author=JD%20Storey&volume=64&issue=3&publication_year=2002&pages=479-98&)

49. Johnson AD, Handsaker RE, Pulit SL, Nizzari MM, O'Donnell CJ, de Bakker PI. SNAP: a web-based tool for identification and annotation of proxy SNPs using HapMap. Bioinformatics. 2008;24(24):2938–9. 10.1093/bioinformatics/btn564  [DOI](https://doi.org/10.1093/bioinformatics/btn564) | [PMC free article](/articles/PMC2720775/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18974171/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=SNAP:%20a%20web-based%20tool%20for%20identification%20and%20annotation%20of%20proxy%20SNPs%20using%20HapMap&author=AD%20Johnson&author=RE%20Handsaker&author=SL%20Pulit&author=MM%20Nizzari&author=CJ%20O'Donnell&volume=24&issue=24&publication_year=2008&pages=2938-9&pmid=18974171&doi=10.1093/bioinformatics/btn564&)

50. Excoffier L, Slatkin M. Maximum-likelihood estimation of molecular haplotype frequencies in a diploid population. Mol Biol Evol. 1995;12(5):921–7. WOS:A1995RQ66400021.  [DOI](https://doi.org/10.1093/oxfordjournals.molbev.a040269) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7476138/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Biol%20Evol&title=Maximum-likelihood%20estimation%20of%20molecular%20haplotype%20frequencies%20in%20a%20diploid%20population&author=L%20Excoffier&author=M%20Slatkin&volume=12&issue=5&publication_year=1995&pages=921-7&pmid=7476138&doi=10.1093/oxfordjournals.molbev.a040269&)

51. Barrett JC, Fry B, Maller J, Daly MJ. Haploview: Analysis and visualization of LD and haplotype maps. Bioinformatics. 2005;21(2):263–5. 10.1093/bioinformatics/bth457 WOS:000226308500016.  [DOI](https://doi.org/10.1093/bioinformatics/bth457) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15297300/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=Haploview:%20Analysis%20and%20visualization%20of%20LD%20and%20haplotype%20maps&author=JC%20Barrett&author=B%20Fry&author=J%20Maller&author=MJ%20Daly&volume=21&issue=2&publication_year=2005&pages=263-5&pmid=15297300&doi=10.1093/bioinformatics/bth457&)

52. Fisher RA. Statistical Methods for Research Workers. London: Oliver and Boyd; 1925.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Statistical%20Methods%20for%20Research%20Workers&author=RA%20Fisher&publication_year=1925&)

53. Frazer KA, Ballinger DG, Cox DR, Hinds DA, Stuve LL, Gibbs RA, et al. A second generation human haplotype map of over 3.1 million SNPs. Nature. 2007;449(7164):851–61. 10.1038/nature06258 WOS:000250230600036.  [DOI](https://doi.org/10.1038/nature06258) | [PMC free article](/articles/PMC2689609/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17943122/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=A%20second%20generation%20human%20haplotype%20map%20of%20over%203.1%20million%20SNPs&author=KA%20Frazer&author=DG%20Ballinger&author=DR%20Cox&author=DA%20Hinds&author=LL%20Stuve&volume=449&issue=7164&publication_year=2007&pages=851-61&pmid=17943122&doi=10.1038/nature06258&)

54. The GTEx Consortium. The Genotype-Tissue Expression (GTEx) project. Nat Genet. 2013;45(6):580–5. Epub 2013/05/30. 10.1038/ng.2653 ng.2653 [pii]. .  [DOI](https://doi.org/10.1038/ng.2653) | [PMC free article](/articles/PMC4010069/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23715323/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=The%20Genotype-Tissue%20Expression%20(GTEx)%20project&volume=45&issue=6&publication_year=2013&pages=580-5&pmid=23715323&doi=10.1038/ng.2653&)
