# Regulation of CETP expression by upstream polymorphisms: Reduced expression associated with rs247616

## Metadata
**Authors:** Adam Suhy, Katherine Hartmann, Audrey Papp, Danxin Wang, Wolfgang Sadee
**Journal:** Pharmacogenetics and genomics
**Date:** 2015 Aug
**DOI:** [10.1097/FPC.0000000000000151](https://doi.org/10.1097/FPC.0000000000000151)
**PMID:** 26061659
**PMCID:** PMC4499003
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4499003/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC4499003/pdf/nihms-691206.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC4499003/pdf/nihms-691206.pdf)

## Abstract

**Introduction:** 
Cholesteryl ester transfer protein (CETP) is involved in reverse cholesterol transport by exchanging cholesteryl esters for triglycerides between HDL and LDL particles, effectively decreasing HDL cholesterol levels. Variants within a large haplotype block upstream of CETP (rs247616, rs173539) have been shown to be significantly associated with reduced expression; however, the underlying mechanism had not been identified.

**Methods:** 
We analyzed the linkage structure of our top candidate Single Nucleotide Polymorphism (SNP), rs247616, and assessed each SNP of the haplotype block for potential interactions with transcription factor binding sites. We then used a reporter gene assay to assess the effect of 3 SNPs (rs247616, rs173539, and rs1723150) on expression in vitro.

**Results:** 
Several variants in the upstream haplotype, including rs247616, rs173539, and rs1723150, disrupt or generate transcription factor binding sites. In reporter gene assays, rs247616 and rs173539 significantly affected expression in HepG2 cells, whereas and rs17231506 had no effect. rs247616 decreased expression 1.7-fold (p<0.0001), while rs173539 increased expression 2.2 fold (p=0.0006).

**Conclusions:** 
SNPs, rs247616 and rs173539, are in high linkage disequilibrium (R2=0.96, D’=1.00) and have the potential to regulate CETP expression. While opposing effects suggest that regulation of CETP expression could vary between tissues, the minor allele of rs247616 and SNPs in high linkage with it were found to be associated with reduced expression across all tissues.

Keywords: Cholesterol Metabolism, Molecular Genetics, Gene expression, Statins, Transcription, Genetics, HDL, LDL, CETP

### Introduction

Cholesteryl ester transfer protein (CETP) is involved in reverse cholesterol transport by exchanging cholesteryl esters for triglycerides between HDL and LDL particles, effectively decreasing HDL cholesterol levels. Variants within a large haplotype block upstream of CETP (rs247616, rs173539) have been shown to be significantly associated with reduced expression; however, the underlying mechanism had not been identified.

### Methods

We analyzed the linkage structure of our top candidate Single Nucleotide Polymorphism (SNP), rs247616, and assessed each SNP of the haplotype block for potential interactions with transcription factor binding sites. We then used a reporter gene assay to assess the effect of 3 SNPs (rs247616, rs173539, and rs1723150) on expression in vitro.

### Results

Several variants in the upstream haplotype, including rs247616, rs173539, and rs1723150, disrupt or generate transcription factor binding sites. In reporter gene assays, rs247616 and rs173539 significantly affected expression in HepG2 cells, whereas and rs17231506 had no effect. rs247616 decreased expression 1.7-fold (p<0.0001), while rs173539 increased expression 2.2 fold (p=0.0006).

### Conclusions

SNPs, rs247616 and rs173539, are in high linkage disequilibrium (R2=0.96, D’=1.00) and have the potential to regulate CETP expression. While opposing effects suggest that regulation of CETP expression could vary between tissues, the minor allele of rs247616 and SNPs in high linkage with it were found to be associated with reduced expression across all tissues.

## Introduction

Cholesteryl Ester Transfer Protein (CETP) is involved in reverse cholesterol transport from the periphery to the liver where it is converted to bile acids and excreted. Expressed highly in spleen and liver [[1](#R1)–[3](#R3)], CETP is secreted into the circulation [[4](#R4)]. There, CETP facilitates the transfer of cholesteryl esters from high-density lipoprotein (HDL) particles to low-density lipoprotein (LDL) particles, in exchange for triglycerides [[4](#R4)]. Increased CETP activity reduces the HDL/total cholesterol ratio, which is associated with an increased risk for coronary artery disease (CAD) [[5](#R5)–[7](#R7)].

Genetic CETP deficiency resulting in increased HDLc and decreased LDLc in Japanese populations, causes various cardiovascular abnormalities, and was reported to be anti-atherogenic [[8](#R8)] or atherogenic [[9](#R9)]. Targeting CETP for inhibition has been a goal of pharmaceuticals; however, initial clinical results have shown no benefit or enhanced risk [[10](#R10),[11](#R11)]. Off-target effects of CETP inhibitors are suspected to play a role, but genetic CETP variants may also affect disease risk and treatment outcomes. CETP variants have been associated with HDL levels; however, observed effects on CAD risk are weak [[12](#R12)]. The minor allele of the marker Taq1B (rs708272) has been associated with reduced CETP activity and reduced benefit from statin therapy, suggesting a possible gene-drug interaction [[13](#R13),[14](#R14)]. Regulation of CETP expression has since been shown to be influenced significantly by SNPs 5-10kb upstream of the gene through GWAS analysis and molecular genetics studies [[15](#R15),[16](#R16)]. Multiple SNPs in this region are in high LD with each other on a long haplotype block stretching across the upstream region into the 5’ portion of CETP and including Taq1B [[16](#R16)]. The upstream enhancer SNPs (rs247616, rs173539, and rs3764261), in partial LD with Taq1B, were shown to be highly associated with CETP mRNA expression in the liver, and with reduced CETP activity and increased HDLc levels [[16](#R16)–[18](#R18)]. Subsequently, Leusink et al. found that CETP enhancer variant rs3764261, in partial LD with Taq1B (R2=0.442, D’=0.886), and in high LD with rs247616 (R2=1, D’=1), also associates with lower efficacy of statins in preventing CAD [[19](#R19)].

Previous studies had focused on non-synonymous single nucleotide polymorphisms (SNPs) in the coding region (such as I405V [rs5882]) and on variants within the proximal promoter region up to 1kb from the transcription start site [[20](#R20),[21](#R21)], which also reside on the large 5’ haplotype block with upstream enhancer variants. In addition, a splicing SNP (rs5883), generating an mRNA lacking exon 9 and a protein devoid of cholesteryl ester transfer activity, had been associated with reduced CETP activity and increased HDLc levels [[16](#R16)–[18](#R18)].

Genetic studies have also been carried out in transgenic mice using human or simian CETP [[22](#R22)–[28](#R28)]. However, the inserted CETP regions did not include the enhancer regions located upstream of CETP, and mice do not express CETP. Therefore, further study is required to understand the complex regulation of CETP in humans.

By measuring allelic expression imbalance (AEI) in human livers and associating it with genotype, we have previously identified rs247616 as the most likely variant responsible for reduced CETP mRNA expression [[16](#R16),[29](#R29)]. However, other promoter/enhancer variants could also have been causing or contributing to CETP transcription, and CETP is likely to harbor multiple regulatory variants. The current study aimed to determine the role of rs247616 in regulation of CETP mRNA expression, and distinguish whether it is solely responsible for the observed AEI, is in strong linkage disequilibrium (LD) with the functional SNP, or acts in concert with additional functional variants. We use transcription factor binding site prediction tools to inform our search, and a reporter gene assay to assess the potential function of the SNP. Given the pervasive effect of CETP on HDL and LDL levels, a better understanding of the molecular mechanisms underlying genetic effects is critical to resolving the relationship between CETP variants, lipid levels, and cardiovascular disease.

## Materials and Methods

### Identifying rs247616 linkage structure

Genotypes from the 1000 Genomes project (1000genomes.org) were analyzed using PLINK (Harvard) [[30](#R30)]. Variants up to 30 megabases upstream and downstream of rs247616 were assessed for linkage disequilibrium to rs247616 in Caucasians.

### Association with Expression in tissues

We measured the association between each SNP and CETP expression in liver, spleen and across 53 tissue types using genotypes and RNA-seq expression data from the GTEx database. Genotype data was accessed through dbGAP project 5358 under IRB protocol 2013H0096. Significance of association was determined using the student’s t-test.

### Transcription factor binding site prediction

MatInspector (Genomatix, Germany) was used to predict transcription factor binding sites lost or gained by the minor allele of SNPs found by sequencing. Fifty base-pair regions centered on each SNP were submitted with the minor and major alleles, and differences were recorded. The difference between the matrix similarity (a weighted score of the match of the inputted sequence to the defined matrix) and the optimized matrix similarity (a threshold defined for each matrix) was used to determine the quality of the match. Positive/negative scores indicate that the allele has a matrix similarity greater/lesser than the optimized matrix similarity, suggesting gain/loss of a potential new binding site for the transcription factor. Sites without differing scores between the major and minor allele, or sites with scores below the threshold for both alleles (suggesting poor binding for both) were discarded leaving only sites with allelic differences in binding [[31](#R31),[32](#R32)]. Potential interactions between SNPs and transcription factors were assessed by visual inspection of the ENCODE tracks using the UCSC genome browser [[33](#R33),[34](#R34)].

### Coexpression analysis

Using data from The Genotype-Tissue Expression (GTEx) project ([www.gtexportal.org](http://www.gtexportal.org), Broad Institute) [[2](#R2)], we correlated the expression of CETP in liver and spleen samples with the expression of transcription factors found to be influenced by SNPs in LD with our lead candidate SNP, rs247616 (n=19). The statistical package, R (r-project.org), was used to correlate the expression data using the Kendall method [[35](#R35)], and Benjamini-Hochberg [[36](#R36)] correction was used to adjust the p-values to account for multiple comparisons. A generalized linear model was tested in R to determine whether rs247616 genotype status affects the coexpression of CETP with transcription factors. Further, a multivariate analysis of CETP expression, transcription factor expression and predicted SNP interaction was conducted with a generalized linear model in R.

### Reporter gene assays for enhancer activity using luciferase expression

To assess the effect of the minor allele of a SNP on expression, we inserted the candidate enhancer region, containing either the major or minor allele, into the multiple cloning site of a pGL-4.23 minimal promoter luciferase expression vector (Promega, USA) using the infusion system (Clonetech, USA) and primers shown in [Table 1](#T1). The region around the candidate SNP of approximately 400bps was amplified by PCR. The vector was transformed into XL-10 gold E. coli. Colonies were isolated and validated by genotyping for the SNP. The sequence of the remaining insert was confirmed by Sanger sequencing. Three validated colonies for each insert were selected and combined for the luciferase expression assay.

### Table 1.

| Primer | Sequence (5′-3′) |
| --- | --- |
| rs247616-F infusion | GCTAGCCTCGAGGATATCGACTCAACAACAGGGCCACA |
| rs247616-R infusion | AGGCCAGATCTTGATATCGACAACAGAGGGACACTCTCTCTAATAAT |
| rs173539-F infusion | GCTAGCCTCGAGGATATCCCTGTGGTCCCAGTTACTTAGGA |
| rs173539-R infusion | AGGCCAGATCTTGATATCGCCATTTCCACTATACGGATCC |
| rs17231506-F infusion | GCTAGCCTCGAGGATATCCCATTATCCCCACCCTTGG |
| rs17231506-R infusion | AGGCCAGATCTTGATATCGGCTGGAGGAACTTCATTCATTA |

Table 1 Caption: Primers used in PCR and infusion cloning reactions

Each expression vector was co-transfected into HEK-293 and HepG2 cells with a renilla luciferase control vector using Lipofectamine 2000 (Life Technologies, USA) and the manufacturer’s procedure. Cells were plated into 12 well plates and incubated for 24 hours. The cells were then gently removed with trypsin, and counted and diluted approximately 1 to 10. Cells were placed in wells of a white, opaque 96-well plate. The Dual-Glo Luciferase Assay System (Promega, USA) was used to measure the firefly and renilla luciferase signals. Chemiluminescence measurements were taken with a Fusion plate reader (Perkin Elmer, USA). Transfections were carried out in triplicate or greater. Each graph shows the mean of 3 transfections (n=6 for rs247616), and error bars represent the standard error of the mean.

## Results

### Identifying rs247616 linkage structure

Previous work had identified rs247616, a SNP 6.2kb upstream of CETP, as the SNP most highly associated with Allelic Expression Imbalance (AEI); the minor allele confers a reduction in expression [[16](#R16)]. To search for additional candidate regulatory variants in the rs247616 haplotype block, we first identified additional SNPs in high LD with rs247616. Using CEU (Caucasian) population data from the 1000genomes project (1000genomes.org) and PLINK (Harvard) we identified all SNPs approximately 30Mb up and downstream of rs247616, and then calculated their LD to rs247616. Thirteen SNPs have an R^2^ greater than 0.77 ([Table 2](#T2)). An additional 3 SNPs have an R2 less than 0.77 and greater than 0.50, of which all are more highly linked with *Taq1B* (rs708272) ([Table 3](#T3)). These SNPs were not included in our analysis due to the previous lack of association of rs708272 with AEI for CETP in liver [[16](#R16)]. This indicates these three SNPs are unlikely to be regulatory variants affecting CETP mRNA expression.

### Table 2.

| SNP | Position | MAF | R2 | D' | Association to HDLca |
| --- | --- | --- | --- | --- | --- |
| rs12446515 | 56987015 | 0.33 | 1.00 | 1.00 | NTb |
| rs173539 | 56988044 | 0.33 | 1.00 | 1.00 | 4.65E-29 |
| rs247616 | 56989590 | 0.33 | 1.00 | 1.00 | 7.18E-29 |
| rs247617 | 56990716 | 0.33 | 1.00 | 1.00 | 1.52E-27 |
| rs183130 | 56991363 | 0.33 | 1.00 | 1.00 | 4.80E-27 |
| rs3764261 | 56993324 | 0.33 | 1.00 | 1.00 | 7.21E-29 |
| rs821840 | 56993886 | 0.33 | 1.00 | 1.00 | NT |
| rs36229491 | 56994244 | 0.33 | 1.00 | 1.00 | NT |
| rs17231506 | 56994528 | 0.33 | 1.00 | 1.00 | 4.11E-29 |
| rs56156922 | 56987369 | 0.33 | 0.98 | 1.00 | NT |
| rs56228609 | 56987765 | 0.31 | 0.93 | 1.00 | 4.23E-24 |
| rs12149545 | 56993161 | 0.31 | 0.93 | 1.00 | 3.19E-28 |
| rs200751500 | 57001274 | 0.31 | 0.87 | 0.97 | NT |
| rs72786786 | 56985514 | 0.33 | 0.78 | 0.88 | NT |
| rs7205804 | 57004889 | 0.45 | 0.51 | 0.93 | 5.16E-25 |
| rs1532625 | 57005301 | 0.45 | 0.51 | 0.93 | 4.83E-25 |
| rs1532624 | 57005479 | 0.45 | 0.51 | 0.93 | 6.35E-25 |

Table 2 Caption: Linkage structure of rs247616. Shaded rows indicate SNPs more strongly linked to rs708272

### Table 3.

| SNP | Position | MAF | R2 | D' | Association to HDLca |
| --- | --- | --- | --- | --- | --- |
| rs711752 | 56996211 | 0.45 | 1.00 | 1.00 | 2.31E-26 |
| rs708272 | 56996288 | 0.45 | 1.00 | 1.00 | 8.77E-27 |
| rs34620476 | 56996649 | 0.45 | 1.00 | 1.00 | 9.89E-24 |
| rs34145065 | 56996645 | 0.44 | 0.98 | 1.00 | NTb |
| rs12720926 | 56998918 | 0.44 | 0.96 | 1.00 | 5.57E-19 |
| rs11508026 | 56999328 | 0.44 | 0.96 | 1.00 | 3.99E-24 |
| rs4784741 | 57001216 | 0.44 | 0.96 | 1.00 | 2.39E-23 |
| rs12444012 | 57001438 | 0.44 | 0.96 | 1.00 | 1.76E-23 |
| rs200373219 | 57001581 | 0.44 | 0.96 | 1.00 | NT |
| rs7205804 | 57004889 | 0.45 | 0.90 | 0.96 | 5.16E-25 |
| rs1532625 | 57005301 | 0.45 | 0.90 | 0.96 | 4.83E-25 |
| rs1532624 | 57005479 | 0.45 | 0.90 | 0.96 | 6.35E-25 |
| rs1800775 | 56995236 | 0.49 | 0.78 | 1.00 | 9.83E-25 |
| rs3816117 | 56996158 | 0.49 | 0.78 | 1.00 | 2.53E-24 |
| rs56228609 | 56987765 | 0.31 | 0.56 | 1.00 | 4.23E-24 |
| rs12149545 | 56993161 | 0.31 | 0.56 | 1.00 | 3.19E-28 |
| rs200751500 | 57001274 | 0.31 | 0.55 | 1.00 | NT |

Table 3 Caption: Linkage structure of rs708272. Shaded rows indicate SNPs more strongly linked to rs247616

To assess their biological functions, we compared previously published associations for each of the 13 SNPs with HDL levels ([Table 2](#T2)) [[16](#R16)]. All tested were significantly associated with HDL, and owing to their high LD, could not be differentiated, thus necessitating examination of molecular mechanism.

### Association of CETP variants with CETP mRNA expression in human tissues

CETP is broadly expressed in tissues such as adipose, liver, breast and thyroid, with the highest expression in spleen ([Figure 1](#F1)). We used CETP expression in all sequenced tissues and genotyping data for rs247616, rs173539, and rs17231506 from the GTEx database. We compared the number of minor alleles of each SNP to the relative expression of CETP in liver and spleen ([Figure 2](#F2)). Using the student’s t-test, we found significantly lower expression of CETP associated with the minor allele of each SNP in spleen (p=0.008), which displayed the highest CETP mRNA expression, indicating that this haplotype block is important for regulation of CETP expression. We observed slightly lower average expression in liver (520 counts versus 396 counts for 0 versus 2 alleles), but this did not reach significance (p=0.63). When analyzed together, all tissues sequenced by GTEx showed a significant association (p=0.024). In view of our targeted analyses on allele-selective CETP mRNA expression [[16](#R16)], it is apparent that rs247616 is associated with hepatic expression, but at a level undetectable in GTEX liver expression data as an eQTL. Additionally, we stratified samples by rs247616 genotype and found no additional SNPs to be significant after Bonferroni multiple test correction.

### Figure 1. CETP expression in GTEx samples.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d4cb/4499003/067f4a99a73c/nihms-691206-f0001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4499003_nihms-691206-f0001.jpg)

Count of RNA-seq reads for CETP in tissues. (www.gtexportal.org/home/gene/CETP)

### Figure 2. mRNA expression of CETP in tissues.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d4cb/4499003/0ba77de25d83/nihms-691206-f0002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4499003_nihms-691206-f0002.jpg)

CETP expression in Liver (left) and spleen (right), as a function of rs247616 genotype, obtained from GTEx (www.gtexportal.org). Each bar represents the expression of CETP for 0, 1, or 2 copies of the minor allele of rs247616. CETP levels in spleen are significantly lower in carriers of the minor alleles of rs247616, with similar data obtained with rs175539, and rs17231506, because of high LD between them. No significant effect is observed in other tissues analyzed separately, such as liver, showing lower CETP expression and less genotype influence. Nevertheless, rs247616 is significant across all tissues in GTEx combined (p-value=0.024; data not shown).

These results suggest that overall, the minor allele reduces expression in most or all tissues however, the magnitude of the effect may differ between tissues. Because of the high expression and significant eQTL values in the spleen, we subsequently focused on differences in transcription factor expression in both liver and spleen.

### Transcription Factor Binding Site prediction

To determine whether these SNPs have a functional role, we assessed potential interactions with transcription factors. Sequence surrounding each SNP in LD with rs247616 and an R2 > 0.77 was submitted in pairs with the major and minor allele to MatInspector (Genomatix, Germany) to analyze lost or gained transcription factor binding sites. Using GTEx expression data, we identified transcription factors that are expressed in liver. Our findings reveal that many of the SNPs analyzed lie within a putative transcription factor binding site for transcription factors that are expressed in the liver, and modify the predicted ability of the transcription factor to bind ([Table 4](#T4)). Ten SNPs in high LD with rs247616 produce changes in the putative transcription factor binding sites in which the SNPs reside. These SNPs are found between 10.3kb upstream and 5.4kb downstream of the CETP transcriptional start site. Three SNPs cause a large increase (rs17231506, rs173539) or decrease (rs247616) in the matrix similarity score of transcription factors that are highly expressed in liver with RPKM (reads per kilobase per million reads sequenced) values greater than 25.

### Table 4.

| SNP | LD with rs247616 (R2) | Distance from CETP start | Matrix Similarity Score (Major) | Matrix Similarity Score (minor) | Gain or Loss | Transcription Factor | Median Liver RPKM | Median Spleen RPKM | Observed in ENCODE (Y/N) |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| rs72786786 | 0.78 | −10321 | 0.055 NDa | 0.099 0.029 | gain gain | SRF LMO2 | 7.6 1.4 | 20.4 18.6 | Y N |
| rs12446515 | 1.00 | −8820 | −0.104 −0.134 | 0.017 0.027 | gain gain | RREB1 ZFX | 6.6 2.4 | 9.1 4.3 | N Y |
| rs56228609 | 0.93 | −8070 | −0.018 | 0.053 | gain | NR1D1 | 9.5 | 8.5 | N |
| rs173539 | 1.00 | −7791 | ND | 0.06 | gain | RARA/RXRA | 10.2/25.7 | 22.7/16.4 | N |
| rs247616 | 1.00 | −6245 | 0.014 0.016 −0.128 | −0.075 −0.019 0.044 | loss loss gain | YBX1 CEBPA LIN54 | 75.3 54.4 1.2 | 168 9.9 1.6 | N N N |
| rs247617 | 1.00 | −5119 | 0.04 −0.027 | 0.076 0.028 | gain gain | NR1I2/RXRA TP53 | 21.4/25.7 5 | 0.18/16.4 17 | N Y |
| rs183130 | 1.00 | −4472 | −0.021 0.067 0.026 | 0.037 −0.097 0.001 | gain loss loss | HMGA1 NFKB1 DEAF1 | 5.5 4.9 2.9 | 46.2 13.5 7.6 | N Y N |
| rs36229491 | 1.00 | −1591 | 0.012 ND | 0.006 0.015 | loss gain | HNF1A ZFHX3 | 4.5 1 | 0 1.2 | N N |
| rs17231506 | 1.00 | −1307 | −0.036 0.047 | 0.059 −0.067 | gain loss | HNF4A DEAF1 | 41.6 2.9 | 0 7.6 | N N |
| rs200751500 | 0.87 | 5439 | 0.013 | 0.03 | gain | KLF4 | 2.1 | 26.4 | N |

Table 4 Caption: SNPs in high Linkage Disequilibrium with rs247616 that alter a putative transcription factor binding site for a factor that is have high RPKM values in liver.

rs247616 strongly alters the putative binding sites of both Y-box binding protein 1 (YBX1) and CCAAT/enhancer binding protein alpha (CEBPA). Both transcription factors are highly expressed in the GTEx sequenced livers with median RPKM values of 75 and 54, respectively. The matrix similarity scores decrease from a favorable binding prediction of 0.014 and 0.016 with the major allele, to unfavorable scores of −0.075 and −.019, respectively, with the minor allele, indicating a change from potentially favorable binding to strongly unfavorable binding. YBX1 is highly expressed in many tissues, including liver and spleen, as shown in [Figure 3](#F3). CEBPA expression in liver is highly variable (SD=56 RPKM), but expression is generally higher than in spleen.

### Figure 3. Expression of top 5 transcription factors.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d4cb/4499003/653c337c34e8/nihms-691206-f0003.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4499003_nihms-691206-f0003.jpg)

Expression is plotted on a log10 axis, expression in liver is shaded in gray. HNF4A shows no expression in spleen.

The minor allele of rs17231506 increases the matrix similarity score for Hepatocyte nuclear factor 4-alpha (HNF4A) from −0.036 to 0.059, indicating a change from an unfavorable binding site to a favorable one. HNF4A is also highly expressed in livers sequenced by the GTEx consortium, and has a median RPKM value of 42 indicating relatively high expression in liver. HNF4A expression was undetectable in spleen ([Figure 3](#F3)). While HepG2 is a hepatic cell line, and does express HNF4A, it is possible that it is not as well expressed as in hepatic tissues, and could explain a lack of effect from rs17231506.

The minor allele of rs173539 further causes an increase in the predicted matrix similarity score for the retinoid X receptor-alpha/retinoic acid receptor-alpha (RXRA/RARA) heterodimer. RARA and RXRA are well expressed in the liver and spleen, with median RPKM values of 10 and 26, respectively, in liver, and 23 and 16, respectively, in spleen ([Figure 3](#F3)). The matrix similarity is below the reported threshold when tested with the major allele, but provides a positive, favorable score of 0.06 for the minor allele. GTEx RNA-seq shows that multiple isoforms exist for RXRA; the expression of the first exon of the only protein coding isoform, which is also the only exon exclusive to the transcript (ensemble.org; ENST00000481739), is expressed 2.4 times more frequently in liver than spleen, indicating that the expression difference of RXRA between the tissues is greater than the difference in total expression would imply.

We used HaploReg v2 to access ENCODE date to corroborate our findings [[37](#R37)]. The predicted transcription factor and SNP interactions that were predicted in ENCODE are noted in [Table 4](#T4). We find that some potential interactions are observed in ENCODE, but many are not. None of the transcription factors predicted to interact with CETP SNPs are found to bind by ChiP analysis in ENCODE. However, some SNPs such as rs72786786 and rs247617 were found to bind transcription factor CTCF by ChiP.

### Coexpression analysis between CETP and transcription factors

We compared the expression levels of 19 transcription factors found to potentially interact with our candidate SNPs ([Table 4](#T4)) with CETP in livers (n=35), and spleens (n=35) sequenced by the GTEx Consortium. Using the R statistical package, we correlated the expression levels of transcription factors by the Kendall tau rank correlation coefficient method to identify concordant and discordant pairs. After adjusting the p-values with the Benjamini-Hochberg correction, no transcription factors showed statistically significant correlation with CETP expression. Additionally, R was used to run a generalized linear model incorporating rs247616 expression with coexpression of CETP and transcription factors. No significant correlations were observed in this test. Multivariate analysis of CETP expression, transcription factor expression, and interacting SNP genotype also resulted in no significant results after multiple test correction. This result indicates that no single transcription factor had sufficient influence to alter hepatic or splenic CETP expression in this group with high significance. A larger set of samples would be required to explore transcription factor interactions.

### Reporter gene assays for enhancer activity using luciferase expression

The three SNPs listed above show the strongest indication of playing a functional role in CETP regulation in the liver by interacting with well-expressed transcription factors. We used a reporter assay to examine the effects of rs247616, rs173539, and rs17231506 on expression of a luciferase reporter gene. The results show that rs247616 and rs173539 significantly altered the luciferase expression in both HEK-293 and HepG2. The minor allele of rs247616 produced a 1.7-fold decrease in expression of the reporter compared to the major allele in both cell lines (p<0.001) ([Figure 4A](#F4)). The minor allele of rs173539 induced an over twofold increase in expression over the major allele, again in both cell lines (p=0.003-0.005) ([Figure 4B](#F4)). No significant difference in expression was observed between the major and minor alleles of rs17231506 in HepG2 cells ([Figure 4C](#F4)), with a small reduction observed in HEK-293 cells (p=0.05).

### Figure 4. The effect of expression of the luciferase reporter gene on HEK-293 and HepG2 cells.

![Figure 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d4cb/4499003/9da4e9156ac2/nihms-691206-f0004.jpg)

Signals are normalized to the wildtype allele and represent the ratio of the experimental firefly luciferase signal to the renilla luciferase control. A) Significant decrease in expression due to the minor allele of rs247616 in HEK-293 and HepG2 cells, B) significant increase in expression due to the minor allele of rs173539 in HEK-293 and HepG2 cells, C) small but significant decrease in expression due to the minor allele of rs17231506 in HEK-293 cells but no significant effect in HepG2 cells.

## Discussion

The luciferase results of this study indicate that both rs247616 and rs173539 may directly play a role in the regulation of CETP. These experiments take the understanding of CETP regulation a step beyond statistical associations indicating that there are molecular mechanisms that can explain some of the genetic regulation of CETP. When coupled with AEI data that demonstrate reduced expression for the minor alleles of these SNPs in high LD with each other [[16](#R16)], it is apparent that in human liver the effect of the minor allele of rs247616 supersedes any effect of rs173539, which may be due to hepatic transcription factors that actually bind to the putative binding sites. Moreover, rs247616 was more strongly associated with measured allelic CETP mRNA expression ratios in the liver than both rs173539 and rs17231506 [[16](#R16)]). As allelic expression ratios are highly sensitive markers of regulatory factors, this finding further supports rs247616 as the main causative variant. Further, analysis of CETP expression in liver and spleen, sequenced by the GTEx consortium, indicates that SNPs in this haplotype block are associated with a significant decrease in CETP mRNA expression in the spleen, with high CETP expression. Failure to detect a significant association of rs247616 with hepatic mRNA expression in GTEx is likely a result of a less pronounced genetic effect compared to that in spleen, confounded by considerable noise in mRNA expression data versus allelic mRNA expression ratios [[16](#R16)]. Since the association of rs247616 with CETP mRNA expression remains significant when all tissues are combined in GTEx, we conclude that the main effect of the minor rs247616 allele is a reduction of mRNA expression across all tissues, consistent with enhanced HDL blood levels [[16](#R16)]. Our results here further emphasize that the high expression of CETP in spleen could suggest a physiological function other than that in lipid metabolism relevant to cardiovascular diseases.

Based on our transcription factor binding predictions we propose a mechanism to account for CETP expression in human tissues. Highly expressed factors YBX1 and CEBPA are unable to bind to the enhancer region containing the minor allele of rs247616 accounting for reduced expression in our reporter gene assay. This effect is somewhat mitigated in the liver by the gain of function of RARA/RXRA in the presence of the rs172539 minor allele. In the spleen, lower expression of the only protein coding isoform of RXRA allows the decrease in CETP expression by rs247616 to be predominant. The expression of CEBPA targeting the rs247616 site is higher in liver than in spleen; however, its expression level is highly variable, which may explain the variation observed in allelic mRNA ratios, noted previously [[16](#R16)]. We propose that rs247616 is the main regulatory variant causing reduced CETP expression in humans. The observation of additional interactions in ENCODE may indicate additional regulatory functions of SNPs in this haplotype block. We cannot exclude that additional SNPs may be involved in regulation in other tissues as well.

Future clinical association studies determining the influence of genetic CETP variants on CAD risk, and residual risk of statin therapy should include rs247616. We suggest that clinical trials of CETP inhibitors determine the status of these CETP variants in patients when analyzing outcomes; however, further work is needed to determine whether additional variants beyond rs247616 and the splicing SNP rs5883 [[16](#R16),[17](#R17)] exist that determine the overall genetic contribution to CETP activity.

In summary, rs247616 has shown the strongest association with liver allelic expression imbalance (AEI) in our earlier studies [[16](#R16)], reduces expression in a reporter gene assay, and is a lead candidate SNP for the regulation of CETP. Together with our previously reported splicing SNP, rs5883 [[17](#R17)], rs247616 may be an important marker for predicting CETP activity, and directing treatments such as statin therapy or CETP inhibition.

## Acknowledgements

This study was supported by NIH grant U01 GM092655, part of the NIH Pharmacogenomics Research Network.

## Footnotes

## References

1. Su AI, Wiltshire T, Batalov S, Lapp H, Ching K a, Block D, et al. A gene atlas of the mouse and human protein-encoding transcriptomes. Proc. Natl. Acad. Sci. U. S. A. 2004;101:6062–7. doi: 10.1073/pnas.0400782101.  [DOI](https://doi.org/10.1073/pnas.0400782101) | [PMC free article](/articles/PMC395923/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15075390/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A&title=A%20gene%20atlas%20of%20the%20mouse%20and%20human%20protein-encoding%20transcriptomes&author=AI%20Su&author=T%20Wiltshire&author=S%20Batalov&author=H%20Lapp&author=K%20a%20Ching&volume=101&publication_year=2004&pages=6062-7&pmid=15075390&doi=10.1073/pnas.0400782101&)

2. Lonsdale J, Thomas J, Salvatore M, Phillips R, Lo E, Shad S, et al. The Genotype-Tissue Expression (GTEx) project. Nat. Genet. 2013;45:580–5. doi: 10.1038/ng.2653.  [DOI](https://doi.org/10.1038/ng.2653) | [PMC free article](/articles/PMC4010069/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23715323/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet&title=The%20Genotype-Tissue%20Expression%20(GTEx)%20project&author=J%20Lonsdale&author=J%20Thomas&author=M%20Salvatore&author=R%20Phillips&author=E%20Lo&volume=45&publication_year=2013&pages=580-5&pmid=23715323&doi=10.1038/ng.2653&)

3. Drayna D, Jarnagin A, McLean J, Henzel W. Cloning and sequencing of human cholesteryl ester transfer protein cDNA. Nature. 1987;327:632–4. doi: 10.1038/327632a0.  [DOI](https://doi.org/10.1038/327632a0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/3600759/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Cloning%20and%20sequencing%20of%20human%20cholesteryl%20ester%20transfer%20protein%20cDNA&author=D%20Drayna&author=A%20Jarnagin&author=J%20McLean&author=W%20Henzel&volume=327&publication_year=1987&pages=632-4&pmid=3600759&doi=10.1038/327632a0&)

4. Hesler C, Swenson T, Tall A. Purification and Characterization of a Human Plasma Cholesteryl Ester Transfer Protein. J. Biol. Chem. 1987;262:2275–82.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/3818596/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem&title=Purification%20and%20Characterization%20of%20a%20Human%20Plasma%20Cholesteryl%20Ester%20Transfer%20Protein&author=C%20Hesler&author=T%20Swenson&author=A%20Tall&volume=262&publication_year=1987&pages=2275-82&pmid=3818596&)

5. Gordon DJ, Probstfield JL, Garrison RJ, Neaton JD, Castelli WP, Knoke JD, et al. High-density lipoprotein cholesterol and cardiovascular disease. Four prospective American studies. Circulation. 1989;79:8–15. doi: 10.1161/01.cir.79.1.8.  [DOI](https://doi.org/10.1161/01.cir.79.1.8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/2642759/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circulation&title=High-density%20lipoprotein%20cholesterol%20and%20cardiovascular%20disease.%20Four%20prospective%20American%20studies&author=DJ%20Gordon&author=JL%20Probstfield&author=RJ%20Garrison&author=JD%20Neaton&author=WP%20Castelli&volume=79&publication_year=1989&pages=8-15&pmid=2642759&doi=10.1161/01.cir.79.1.8&)

6. Roger VL, Go AS, Lloyd-Jones DM, Benjamin EJ, Berry JD, Borden WB, et al. Heart disease and stroke statistics--2012 update: a report from the American Heart Association. Circulation. 2012;125:e2–220. doi: 10.1161/CIR.0b013e31823ac046.  [DOI](https://doi.org/10.1161/CIR.0b013e31823ac046) | [PMC free article](/articles/PMC4440543/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22179539/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circulation&title=Heart%20disease%20and%20stroke%20statistics--2012%20update:%20a%20report%20from%20the%20American%20Heart%20Association&author=VL%20Roger&author=AS%20Go&author=DM%20Lloyd-Jones&author=EJ%20Benjamin&author=JD%20Berry&volume=125&publication_year=2012&pages=e2-220&pmid=22179539&doi=10.1161/CIR.0b013e31823ac046&)

7. Murray RK, Granner DK, Rodwell VW. Harper’s Biochemistry. 27th Appleton & Lange; Stamford: 2006.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Harper%E2%80%99s%20Biochemistry&author=RK%20Murray&author=DK%20Granner&author=VW%20Rodwell&publication_year=2006&)

8. Inazu A, Brown M, Hesler C, Agellon L, Koizumi J, Takata K, et al. Increased high-density lipoprotein levels caused by a common cholesteryl-ester transfer protein gene mutation. N. Engl. J. Med. 1990;323:1234–8. doi: 10.1056/NEJM199011013231803.  [DOI](https://doi.org/10.1056/NEJM199011013231803) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/2215607/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N.%20Engl.%20J.%20Med&title=Increased%20high-density%20lipoprotein%20levels%20caused%20by%20a%20common%20cholesteryl-ester%20transfer%20protein%20gene%20mutation&author=A%20Inazu&author=M%20Brown&author=C%20Hesler&author=L%20Agellon&author=J%20Koizumi&volume=323&publication_year=1990&pages=1234-8&pmid=2215607&doi=10.1056/NEJM199011013231803&)

9. Nagano M, Yamashita S, Hirano K-I, Takano M, Maruyama T, Ishihara M, et al. Molecular mechanisms of cholesteryl ester transfer protein deficiency in Japanese. J. Atheroscler. Thromb. 2004;11:110–21. doi: 10.5551/jat.11.110.  [DOI](https://doi.org/10.5551/jat.11.110) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15256762/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Atheroscler.%20Thromb&title=Molecular%20mechanisms%20of%20cholesteryl%20ester%20transfer%20protein%20deficiency%20in%20Japanese&author=M%20Nagano&author=S%20Yamashita&author=K-I%20Hirano&author=M%20Takano&author=T%20Maruyama&volume=11&publication_year=2004&pages=110-21&pmid=15256762&doi=10.5551/jat.11.110&)

10. Barter P, Caulfield M. Effects of torcetrapib in patients at high risk for coronary events. N. Engl. J. Med. 2007;357:2109–22. doi: 10.1056/NEJMoa0706628.  [DOI](https://doi.org/10.1056/NEJMoa0706628) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17984165/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N.%20Engl.%20J.%20Med&title=Effects%20of%20torcetrapib%20in%20patients%20at%20high%20risk%20for%20coronary%20events&author=P%20Barter&author=M%20Caulfield&volume=357&publication_year=2007&pages=2109-22&pmid=17984165&doi=10.1056/NEJMoa0706628&)

11. Schwartz GG, Olsson AG, Abt M, Ballantyne CM, Barter PJ, Brumm J, et al. Effects of dalcetrapib in patients with a recent acute coronary syndrome. N. Engl. J. Med. 2012;367:2089–99. doi: 10.1056/NEJMoa1206797.  [DOI](https://doi.org/10.1056/NEJMoa1206797) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23126252/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N.%20Engl.%20J.%20Med&title=Effects%20of%20dalcetrapib%20in%20patients%20with%20a%20recent%20acute%20coronary%20syndrome&author=GG%20Schwartz&author=AG%20Olsson&author=M%20Abt&author=CM%20Ballantyne&author=PJ%20Barter&volume=367&publication_year=2012&pages=2089-99&pmid=23126252&doi=10.1056/NEJMoa1206797&)

12. Boekholdt SM, Sacks FM, Jukema JW, Shepherd J, Freeman DJ, McMahon D, et al. Cholesteryl ester transfer protein TaqIB variant, high-density lipoprotein cholesterol levels, cardiovascular risk, and efficacy of pravastatin treatment: individual patient meta-analysis of 13,677 subjects. Circulation. 2005;111:278–87. doi: 10.1161/01.CIR.0000153341.46271.40.  [DOI](https://doi.org/10.1161/01.CIR.0000153341.46271.40) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15655129/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circulation&title=Cholesteryl%20ester%20transfer%20protein%20TaqIB%20variant,%20high-density%20lipoprotein%20cholesterol%20levels,%20cardiovascular%20risk,%20and%20efficacy%20of%20pravastatin%20treatment:%20individual%20patient%20meta-analysis%20of%2013,677%20subjects&author=SM%20Boekholdt&author=FM%20Sacks&author=JW%20Jukema&author=J%20Shepherd&author=DJ%20Freeman&volume=111&publication_year=2005&pages=278-87&pmid=15655129&doi=10.1161/01.CIR.0000153341.46271.40&)

13. Kuivenhoven J. The role of a common variant of the cholesteryl ester transfer protein gene in the progression of coronary atherosclerosis. N. Engl. J. Med. 1998;338:86–93. doi: 10.1056/NEJM199801083380203.  [DOI](https://doi.org/10.1056/NEJM199801083380203) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9420339/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N.%20Engl.%20J.%20Med&title=The%20role%20of%20a%20common%20variant%20of%20the%20cholesteryl%20ester%20transfer%20protein%20gene%20in%20the%20progression%20of%20coronary%20atherosclerosis&author=J%20Kuivenhoven&volume=338&publication_year=1998&pages=86-93&pmid=9420339&doi=10.1056/NEJM199801083380203&)

14. Regieli JJ, Jukema JW, Grobbee DE, Kastelein JJP, Kuivenhoven JA, Zwinderman AH, et al. CETP genotype predicts increased mortality in statin-treated men with proven cardiovascular disease: an adverse pharmacogenetic interaction. Eur. Heart J. 2008;29:2792–9. doi: 10.1093/eurheartj/ehn465.  [DOI](https://doi.org/10.1093/eurheartj/ehn465) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18957472/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur.%20Heart%20J&title=CETP%20genotype%20predicts%20increased%20mortality%20in%20statin-treated%20men%20with%20proven%20cardiovascular%20disease:%20an%20adverse%20pharmacogenetic%20interaction&author=JJ%20Regieli&author=JW%20Jukema&author=DE%20Grobbee&author=JJP%20Kastelein&author=JA%20Kuivenhoven&volume=29&publication_year=2008&pages=2792-9&pmid=18957472&doi=10.1093/eurheartj/ehn465&)

15. Marmot MGG, Smith GDD, Stansfeld S, Patel C, North F, Head J, et al. Health inequalities among British civil servants: the Whitehall II study. Lancet. 1991;337:1387–93. doi: 10.1016/0140-6736(91)93068-k.  [DOI](https://doi.org/10.1016/0140-6736(91)93068-k) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/1674771/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Health%20inequalities%20among%20British%20civil%20servants:%20the%20Whitehall%20II%20study&author=MGG%20Marmot&author=GDD%20Smith&author=S%20Stansfeld&author=C%20Patel&author=F%20North&volume=337&publication_year=1991&pages=1387-93&pmid=1674771&doi=10.1016/0140-6736(91)93068-k&)

16. Papp AC, Pinsonneault JK, Wang D, Newman LC, Gong Y, Johnson J a, et al. Cholesteryl Ester Transfer Protein (CETP) polymorphisms affect mRNA splicing, HDL levels, and sex-dependent cardiovascular risk. PLoS One. 2012;7:e31930. doi: 10.1371/journal.pone.0031930.  [DOI](https://doi.org/10.1371/journal.pone.0031930) | [PMC free article](/articles/PMC3293889/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22403620/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=Cholesteryl%20Ester%20Transfer%20Protein%20(CETP)%20polymorphisms%20affect%20mRNA%20splicing,%20HDL%20levels,%20and%20sex-dependent%20cardiovascular%20risk&author=AC%20Papp&author=JK%20Pinsonneault&author=D%20Wang&author=LC%20Newman&author=Y%20Gong&volume=7&publication_year=2012&pages=e31930&pmid=22403620&doi=10.1371/journal.pone.0031930&)

17. Suhy A, Hartmann K, Newman L, Papp A, Toneff T, Hook V, et al. Genetic variants affecting alternative splicing of human cholesteryl ester transfer protein. Biochem. Biophys. Res. Commun. 2014;443:1270–4. doi: 10.1016/j.bbrc.2013.12.127.  [DOI](https://doi.org/10.1016/j.bbrc.2013.12.127) | [PMC free article](/articles/PMC3929938/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24393849/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem.%20Biophys.%20Res.%20Commun&title=Genetic%20variants%20affecting%20alternative%20splicing%20of%20human%20cholesteryl%20ester%20transfer%20protein&author=A%20Suhy&author=K%20Hartmann&author=L%20Newman&author=A%20Papp&author=T%20Toneff&volume=443&publication_year=2014&pages=1270-4&pmid=24393849&doi=10.1016/j.bbrc.2013.12.127&)

18. Barber MJ, Mangravite LM, Hyde CL, Chasman DI, Smith JD, McCarty C a, et al. Genome-wide association of lipid-lowering response to statins in combined study populations. PLoS One. 2010;5:e9763. doi: 10.1371/journal.pone.0009763.  [DOI](https://doi.org/10.1371/journal.pone.0009763) | [PMC free article](/articles/PMC2842298/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20339536/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=Genome-wide%20association%20of%20lipid-lowering%20response%20to%20statins%20in%20combined%20study%20populations&author=MJ%20Barber&author=LM%20Mangravite&author=CL%20Hyde&author=DI%20Chasman&author=JD%20Smith&volume=5&publication_year=2010&pages=e9763&pmid=20339536&doi=10.1371/journal.pone.0009763&)

19. Leusink M, Onland-Moret NC, Asselbergs FW, Ding B, Kotti S, van Zuydam NR, et al. Cholesteryl Ester Transfer Protein Polymorphisms, Statin Use, and Their Impact on Cholesterol Levels and Cardiovascular Events. Clin. Pharmacol. Ther. 2013;95:314–20. doi: 10.1038/clpt.2013.194.  [DOI](https://doi.org/10.1038/clpt.2013.194) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24080640/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther&title=Cholesteryl%20Ester%20Transfer%20Protein%20Polymorphisms,%20Statin%20Use,%20and%20Their%20Impact%20on%20Cholesterol%20Levels%20and%20Cardiovascular%20Events&author=M%20Leusink&author=NC%20Onland-Moret&author=FW%20Asselbergs&author=B%20Ding&author=S%20Kotti&volume=95&publication_year=2013&pages=314-20&pmid=24080640&doi=10.1038/clpt.2013.194&)

20. Corbex M, Poirier O, Fumeron F, Betoulle D, Evans A, Ruidavets JB, et al. Extensive association analysis between the CETP gene and coronary heart disease phenotypes reveals several putative functional polymorphisms and gene-environment interaction. Genet. Epidemiol. 2000;19:64–80. doi: 10.1002/1098-2272(200007)19:1<64::AID-GEPI5>3.0.CO;2-E.  [DOI](https://doi.org/10.1002/1098-2272(200007)19:1<64::AID-GEPI5>3.0.CO;2-E) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10861897/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genet.%20Epidemiol&title=Extensive%20association%20analysis%20between%20the%20CETP%20gene%20and%20coronary%20heart%20disease%20phenotypes%20reveals%20several%20putative%20functional%20polymorphisms%20and%20gene-environment%20interaction&author=M%20Corbex&author=O%20Poirier&author=F%20Fumeron&author=D%20Betoulle&author=A%20Evans&volume=19&publication_year=2000&pages=64-80&pmid=10861897&doi=10.1002/1098-2272(200007)19:1<64::AID-GEPI5>3.0.CO;2-E&)

21. Frisdal E, Klerkx AHEM, Le Goff W, Tanck MWT, Lagarde J-P, Jukema JW, et al. Functional interaction between -629C/A, -971G/A and -1337C/T polymorphisms in the CETP gene is a major determinant of promoter activity and plasma CETP concentration in the REGRESS Study. Hum. Mol. Genet. 2005;14:2607–18. doi: 10.1093/hmg/ddi291.  [DOI](https://doi.org/10.1093/hmg/ddi291) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16049032/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum.%20Mol.%20Genet&title=Functional%20interaction%20between%20-629C/A,%20-971G/A%20and%20-1337C/T%20polymorphisms%20in%20the%20CETP%20gene%20is%20a%20major%20determinant%20of%20promoter%20activity%20and%20plasma%20CETP%20concentration%20in%20the%20REGRESS%20Study&author=E%20Frisdal&author=AHEM%20Klerkx&author=W%20Le%20Goff&author=MWT%20Tanck&author=J-P%20Lagarde&volume=14&publication_year=2005&pages=2607-18&pmid=16049032&doi=10.1093/hmg/ddi291&)

22. MacLean P, Vadlamudi S. Differential expression of cholesteryl ester transfer protein in the liver and plasma of fasted and fed transgenic mice. J. Nutr. Biochem. 2000;11:318–25. doi: 10.1016/s0955-2863(00)00084-x.  [DOI](https://doi.org/10.1016/s0955-2863(00)00084-x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11002127/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Nutr.%20Biochem&title=Differential%20expression%20of%20cholesteryl%20ester%20transfer%20protein%20in%20the%20liver%20and%20plasma%20of%20fasted%20and%20fed%20transgenic%20mice&author=P%20MacLean&author=S%20Vadlamudi&volume=11&publication_year=2000&pages=318-25&pmid=11002127&doi=10.1016/s0955-2863(00)00084-x&)

23. Harder C, Lau P, Meng A, Whitman SC, McPherson R. Cholesteryl ester transfer protein (CETP) expression protects against diet induced atherosclerosis in SR-BI deficient mice. Arterioscler. Thromb. Vasc. Biol. 2007;27:858–64. doi: 10.1161/01.ATV.0000259357.42089.dc.  [DOI](https://doi.org/10.1161/01.ATV.0000259357.42089.dc) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17272756/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arterioscler.%20Thromb.%20Vasc.%20Biol&title=Cholesteryl%20ester%20transfer%20protein%20(CETP)%20expression%20protects%20against%20diet%20induced%20atherosclerosis%20in%20SR-BI%20deficient%20mice&author=C%20Harder&author=P%20Lau&author=A%20Meng&author=SC%20Whitman&author=R%20McPherson&volume=27&publication_year=2007&pages=858-64&pmid=17272756&doi=10.1161/01.ATV.0000259357.42089.dc&)

24. Honzumi S, Shima A, Hiroshima A, Koieyama T, Ubukata N, Terasaka N. LXRalpha regulates human CETP expression in vitro and in transgenic mice. Atherosclerosis. 2010;212:139–45. doi: 10.1016/j.atherosclerosis.2010.04.025.  [DOI](https://doi.org/10.1016/j.atherosclerosis.2010.04.025) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20494359/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Atherosclerosis&title=LXRalpha%20regulates%20human%20CETP%20expression%20in%20vitro%20and%20in%20transgenic%20mice&author=S%20Honzumi&author=A%20Shima&author=A%20Hiroshima&author=T%20Koieyama&author=N%20Ubukata&volume=212&publication_year=2010&pages=139-45&pmid=20494359&doi=10.1016/j.atherosclerosis.2010.04.025&)

25. Gautier T, de Haan W, Grober J, Ye D, Bahr MJ, Claudel T, et al. Farnesoid X receptor activation increases cholesteryl ester transfer protein expression in humans and transgenic mice. J. Lipid Res. 2013;54:2195–205. doi: 10.1194/jlr.M038141.  [DOI](https://doi.org/10.1194/jlr.M038141) | [PMC free article](/articles/PMC3708369/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23620138/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Lipid%20Res&title=Farnesoid%20X%20receptor%20activation%20increases%20cholesteryl%20ester%20transfer%20protein%20expression%20in%20humans%20and%20transgenic%20mice&author=T%20Gautier&author=W%20de%20Haan&author=J%20Grober&author=D%20Ye&author=MJ%20Bahr&volume=54&publication_year=2013&pages=2195-205&pmid=23620138&doi=10.1194/jlr.M038141&)

26. Escolà-Gil JC, Julve J, Marzal-Casacuberta À , Ordóñez-Llanos J, González-Sastre F, Blanco-Vaca F. ApoA-II expression in CETP transgenic mice increases VLDL production and impairs VLDL clearance. J. Lipid Res. 2001;42:241–8.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/11181754/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Lipid%20Res&title=ApoA-II%20expression%20in%20CETP%20transgenic%20mice%20increases%20VLDL%20production%20and%20impairs%20VLDL%20clearance&author=JC%20Escol%C3%A0-Gil&author=J%20Julve&author=%C3%80%20Marzal-Casacuberta&author=J%20Ord%C3%B3%C3%B1ez-Llanos&author=F%20Gonz%C3%A1lez-Sastre&volume=42&publication_year=2001&pages=241-8&pmid=11181754&)

27. Foger B, Chase M, Amar MJ, Vaisman BL, Shamburek RD, Paigen B, et al. Cholesteryl Ester Transfer Protein Corrects Dysfunctional High Density Lipoproteins and Reduces Aortic Atherosclerosis in Lecithin Cholesterol Acyltransferase Transgenic Mice. J. Biol. Chem. 1999;274:36912–20. doi: 10.1074/jbc.274.52.36912.  [DOI](https://doi.org/10.1074/jbc.274.52.36912) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10601244/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem&title=Cholesteryl%20Ester%20Transfer%20Protein%20Corrects%20Dysfunctional%20High%20Density%20Lipoproteins%20and%20Reduces%20Aortic%20Atherosclerosis%20in%20Lecithin%20Cholesterol%20Acyltransferase%20Transgenic%20Mice&author=B%20Foger&author=M%20Chase&author=MJ%20Amar&author=BL%20Vaisman&author=RD%20Shamburek&volume=274&publication_year=1999&pages=36912-20&pmid=10601244&doi=10.1074/jbc.274.52.36912&)

28. Marotti K, Castle C. The role of cholesteryl ester transfer protein in primate apolipoprotein AI metabolism. Insights from studies with transgenic mice. Arterioscler. Thromb. Vasc. Biol. 1992;12:736–44. doi: 10.1161/01.atv.12.6.736.  [DOI](https://doi.org/10.1161/01.atv.12.6.736) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/1591233/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arterioscler.%20Thromb.%20Vasc.%20Biol&title=The%20role%20of%20cholesteryl%20ester%20transfer%20protein%20in%20primate%20apolipoprotein%20AI%20metabolism.%20Insights%20from%20studies%20with%20transgenic%20mice&author=K%20Marotti&author=C%20Castle&volume=12&publication_year=1992&pages=736-44&pmid=1591233&doi=10.1161/01.atv.12.6.736&)

29. Johnson A, Zhang Y, Papp A, Pinsonneault J, Lim J, Saffen D, et al. Polymorphisms affecting gene transcription and mRNA processing in pharmacogenetic candidate genes: detection through allelic expression imbalance in human target tissues. Pharmacogenet. Genomics. 2008;18:781–91. doi: 10.1097/FPC.0b013e3283050107.  [DOI](https://doi.org/10.1097/FPC.0b013e3283050107) | [PMC free article](/articles/PMC2779843/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18698231/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet.%20Genomics&title=Polymorphisms%20affecting%20gene%20transcription%20and%20mRNA%20processing%20in%20pharmacogenetic%20candidate%20genes:%20detection%20through%20allelic%20expression%20imbalance%20in%20human%20target%20tissues&author=A%20Johnson&author=Y%20Zhang&author=A%20Papp&author=J%20Pinsonneault&author=J%20Lim&volume=18&publication_year=2008&pages=781-91&pmid=18698231&doi=10.1097/FPC.0b013e3283050107&)

30. Purcell S, Neale B, Todd-Brown K, Thomas L, Ferreira M a R, Bender D, et al. PLINK: a tool set for whole-genome association and population-based linkage analyses. Am. J. Hum. Genet. 2007;81:559–75. doi: 10.1086/519795.  [DOI](https://doi.org/10.1086/519795) | [PMC free article](/articles/PMC1950838/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17701901/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Hum.%20Genet&title=PLINK:%20a%20tool%20set%20for%20whole-genome%20association%20and%20population-based%20linkage%20analyses&author=S%20Purcell&author=B%20Neale&author=K%20Todd-Brown&author=L%20Thomas&author=M%20a%20R%20Ferreira&volume=81&publication_year=2007&pages=559-75&pmid=17701901&doi=10.1086/519795&)

31. Cartharius K, Frech K, Grote K, Klocke B, Haltmeier M, Klingenhoff a, et al. MatInspector and beyond: promoter analysis based on transcription factor binding sites. Bioinformatics. 2005;21:2933–42. doi: 10.1093/bioinformatics/bti473.  [DOI](https://doi.org/10.1093/bioinformatics/bti473) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15860560/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=MatInspector%20and%20beyond:%20promoter%20analysis%20based%20on%20transcription%20factor%20binding%20sites&author=K%20Cartharius&author=K%20Frech&author=K%20Grote&author=B%20Klocke&author=M%20Haltmeier&volume=21&publication_year=2005&pages=2933-42&pmid=15860560&doi=10.1093/bioinformatics/bti473&)

32. Quandt K, Freeh K, Karas H, Wingender E, Werner T. Matlnd and Matlnspector?: new fast and versatile tools for detection of consensus matches in nucleotide sequence data. Nucleic Acids Res. 1995;23:4878–84. doi: 10.1093/nar/23.23.4878.  [DOI](https://doi.org/10.1093/nar/23.23.4878) | [PMC free article](/articles/PMC307478/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8532532/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=Matlnd%20and%20Matlnspector?:%20new%20fast%20and%20versatile%20tools%20for%20detection%20of%20consensus%20matches%20in%20nucleotide%20sequence%20data&author=K%20Quandt&author=K%20Freeh&author=H%20Karas&author=E%20Wingender&author=T%20Werner&volume=23&publication_year=1995&pages=4878-84&pmid=8532532&doi=10.1093/nar/23.23.4878&)

33. Rosenbloom KR, Sloan C a, Malladi VS, Dreszer TR, Learned K, Kirkup VM, et al. ENCODE Data in the UCSC Genome Browser: Year 5 update. Nucleic Acids Res. 2013;41:56–63. doi: 10.1093/nar/gks1172.  [DOI](https://doi.org/10.1093/nar/gks1172) | [PMC free article](/articles/PMC3531152/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23193274/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=ENCODE%20Data%20in%20the%20UCSC%20Genome%20Browser:%20Year%205%20update&author=KR%20Rosenbloom&author=C%20a%20Sloan&author=VS%20Malladi&author=TR%20Dreszer&author=K%20Learned&volume=41&publication_year=2013&pages=56-63&pmid=23193274&doi=10.1093/nar/gks1172&)

34. Kent WJ, Sugnet CW, Furey TS, Roskin KM, Pringle TH, Zahler AM, et al. The Human Genome Browser at UCSC The Human Genome Browser at UCSC. Genome Res. 2002:996–1006. doi: 10.1101/gr.229102.  [DOI](https://doi.org/10.1101/gr.229102) | [PMC free article](/articles/PMC186604/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12045153/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Res&title=The%20Human%20Genome%20Browser%20at%20UCSC%20The%20Human%20Genome%20Browser%20at%20UCSC&author=WJ%20Kent&author=CW%20Sugnet&author=TS%20Furey&author=KM%20Roskin&author=TH%20Pringle&publication_year=2002&pages=996-1006&pmid=12045153&doi=10.1101/gr.229102&)

35. Kumari S, Nie J, Chen H-S, Ma H, Stewart R, Li X, et al. Evaluation of gene association methods for coexpression network construction and biological knowledge discovery. PLoS One. 2012;7:e50411. doi: 10.1371/journal.pone.0050411.  [DOI](https://doi.org/10.1371/journal.pone.0050411) | [PMC free article](/articles/PMC3511551/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23226279/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=Evaluation%20of%20gene%20association%20methods%20for%20coexpression%20network%20construction%20and%20biological%20knowledge%20discovery&author=S%20Kumari&author=J%20Nie&author=H-S%20Chen&author=H%20Ma&author=R%20Stewart&volume=7&publication_year=2012&pages=e50411&pmid=23226279&doi=10.1371/journal.pone.0050411&)

36. Benjamini Y, Hochberg Y. Controlling the false discovery rate: a practical and powerful approach to multiple testing. J. R. Stat. Soc. Ser. B. 1995;57:289–300.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20R.%20Stat.%20Soc.%20Ser.%20B&title=Controlling%20the%20false%20discovery%20rate:%20a%20practical%20and%20powerful%20approach%20to%20multiple%20testing&author=Y%20Benjamini&author=Y%20Hochberg&volume=57&publication_year=1995&pages=289-300&)

37. Ward LD, Kellis M. HaploReg: A resource for exploring chromatin states, conservation, and regulatory motif alterations within sets of genetically linked variants. Nucleic Acids Res. 2012;40:1–5. doi: 10.1093/nar/gkr917.  [DOI](https://doi.org/10.1093/nar/gkr917) | [PMC free article](/articles/PMC3245002/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22064851/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=HaploReg:%20A%20resource%20for%20exploring%20chromatin%20states,%20conservation,%20and%20regulatory%20motif%20alterations%20within%20sets%20of%20genetically%20linked%20variants&author=LD%20Ward&author=M%20Kellis&volume=40&publication_year=2012&pages=1-5&pmid=22064851&doi=10.1093/nar/gkr917&)
