# Next Generation Sequencing of RYR1 and CACNA1S in Malignant Hyperthermia and Exertional Heat Illness

## Metadata
**Authors:** Dorota Fiszer, Marie-Anne Shaw, Nickla A Fisher, Ian M Carr, Pawan K Gupta, Elizabeth J Watkins, Daniel Roiz de Sa, Jerry H Kim, Philip M Hopkins
**Journal:** Anesthesiology
**Date:** 2015 May
**DOI:** [10.1097/ALN.0000000000000610](https://doi.org/10.1097/ALN.0000000000000610)
**PMID:** 25658027
**PMCID:** PMC4472733
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4472733/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC4472733/pdf/nihms690187.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC4472733/pdf/nihms690187.pdf)

## Abstract

**Background:** 
Variants in RYR1 are associated with the majority of cases of malignant hyperthermia (MH), a form of heat illness pharmacogenetically triggered by general anesthetics, and they have also been associated with exertional heat illness. CACNA1S has also been implicated in MH. We applied a targeted next generation sequencing approach to identify variants in RYR1 and CACNA1S in a cohort of unrelated patients diagnosed with MH susceptibility. We also provide the first comprehensive report of sequencing of these two genes in a cohort of survivors of exertional heat illness.

**Methods:** 
DNA extracted from blood was genotyped using a “long” polymerase chain reaction technique, with sequencing on the Illumina GAII® or MiSeq® platforms (Illumina Inc., San Diego, CA). Variants were assessed for pathogenicity using bioinformatic approaches. For further follow up DNA from additional family members and up to 211 MH normal and 556 MH susceptible unrelated individuals was tested.

**Results:** 
In 29 MH patients we identified three pathogenic and four novel RYR1 variants, with a further five RYR1 variants previously reported in association with MH. Three novel RYR1 variants were found in the exertional heat illness cohort (n = 28) along with two more previously reported in association with MH. Two other variants were reported previously associated with centronuclear myopathy. We found one and three rare variants of unknown significance in CACNA1S in the MH and exertional heat illness cohorts respectively.

**Conclusion:** 
Targeted next generation sequencing proved efficient at identifying diagnostically useful and potentially implicated variants in RYR1 and CACNA1S in MH and exertional heat illness.

### Background

Variants in *RYR1* are associated with the majority of cases of malignant hyperthermia (MH), a form of heat illness pharmacogenetically triggered by general anesthetics, and they have also been associated with exertional heat illness. *CACNA1S* has also been implicated in MH. We applied a targeted next generation sequencing approach to identify variants in *RYR1* and *CACNA1S* in a cohort of unrelated patients diagnosed with MH susceptibility. We also provide the first comprehensive report of sequencing of these two genes in a cohort of survivors of exertional heat illness.

### Methods

DNA extracted from blood was genotyped using a “long” polymerase chain reaction technique, with sequencing on the Illumina GAII^®^ or MiSeq^®^ platforms (Illumina Inc., San Diego, CA). Variants were assessed for pathogenicity using bioinformatic approaches. For further follow up DNA from additional family members and up to 211 MH normal and 556 MH susceptible unrelated individuals was tested.

### Results

In 29 MH patients we identified three pathogenic and four novel *RYR1* variants, with a further five *RYR1* variants previously reported in association with MH. Three novel *RYR1* variants were found in the exertional heat illness cohort (n = 28) along with two more previously reported in association with MH. Two other variants were reported previously associated with centronuclear myopathy. We found one and three rare variants of unknown significance in *CACNA1S* in the MH and exertional heat illness cohorts respectively.

### Conclusion

Targeted next generation sequencing proved efficient at identifying diagnostically useful and potentially implicated variants in *RYR1* and *CACNA1S* in MH and exertional heat illness.

## Introduction

Malignant Hyperthermia (MH) is a potentially fatal adverse drug reaction, triggered during general anesthesia by volatile halogenated anesthetics and succinylcholine. The prevalence of genetic variants predisposing to MH has been estimated to be as high as 1:2,000 ^1,2^. In most families MH shows an autosomal dominant pattern of inheritance. Following a suspected MH episode, the diagnosis can be confirmed using an *in vitro* contracture test (IVCT), where skeletal muscle, obtained through biopsy of the vastus muscle, is exposed to incremental concentrations of caffeine or halothane and responses recorded ^3^.

Physiological and biochemical studies show a severe defect in MH skeletal muscle calcium regulation ^4^. The ryanodine receptor (RYR1) forms a calcium channel in the sarcoplasmic reticulum and in its active state releases Ca^2+^ from the sarcoplasmic reticulum leading to muscle contraction^5^. Ca_v_1.1 is structurally a calcium channel located within the t-tubule membrane, but functionally it is the voltage sensor that activates RYR1. During an MH episode myoplasmic calcium concentration rises rapidly resulting in a hypermetabolic state characterised systemically by hyperthermia, acidosis, tachycardia, cardiac arrhythmias, skeletal muscle rigidity and rhabdomyolysis. Similar features are observed in exertional heat illness (EHI) suggesting possible links between MH and EHI^6^.

Variants in the *RYR1* gene (Mendelian inheritance in man, MIM #180901) are thought to be responsible for about 75% of cases susceptible to MH (MHS). Less frequently MH is associated with variants in *CACNA1S* (MIM #114208), encoding the α subunit of Ca_v_1.1. Mutations in either of these genes may disrupt signalling between Ca_v_1.1 and RYR1 and thus confer MH susceptibility^7–9^. A significant number of families do not have a variant in a major locus identified to date.

Approximately 200 missense variants in *RYR1* have been described in association with MH^10^, with 31 of them known to be functionally relevant and used diagnostically^11^, while only 2 variants described in *CACNA1S* in association with MH have been functionally characterised^12,13^. Variants in *RYR1* are also known to cause distinct rare myopathies such as multiminicore disease and central core disease, with some variants associated both with MH and central core disease^14^.

Because of the large size of *RYR1* screening the whole coding sequence (~15,000 bp) using Sanger sequencing is expensive and time-consuming. In a clinical setting, often only the limited number of diagnostic variants is screened. With the advent of Next Generation Sequencing (NGS) limitations of current approaches can be overcome with a potential for increased variant detection rate compared with Sanger sequencing^15^. We have assessed the feasibility of NGS using groups of MH susceptible and EHI patients previously screened for *RYR1* diagnostic variants and found to be negative, for analysis of the whole coding sequences of *RYR1* and *CACNA1S*. We applied a Long Range polymerase chain reaction (PCR) technique as an enrichment method, to amplify coding sequences, and sequenced using Illumina GAII^®^ and MiSeq^®^ technology (Illumina Inc., San Diego, CA).

## Materials and Methods

### Samples

Blood samples were obtained from patients referred to the Malignant Hyperthermia Unit, University of Leeds for genetic and functional (IVCT) tests used in the diagnosis of MH-susceptibility. The research was approved by the Leeds East Local Research Ethics Committee (Reference 10/H1306/70; Leeds, United Kingdom) and written consent was obtained from each patient. DNA was extracted from peripheral blood lymphocytes using a salting out method. Briefly, after erythrocyte lysis with a buffer containing 155 mM NH_4_Cl, 10 mM KHCO_3_ and 1.0 mM EDTA, the leukocytes were pelleted and subjected to lysis with 2% sodium dodecyl sulphate, 25 mM EDTA solution. Next, a protein precipitation with 10 M ammonium acetate was performed. Then the proteins were pelleted and supernatant containing DNA was transferred into a new tube. DNA precipitation was carried out using isopropanol.

Fifty-seven DNA samples were chosen for sequencing the entire coding sequence of *RYR1* and *CACNA1S*, as the two main genes known to contribute to MH susceptibility. The majority of samples had already been routinely screened for the 16 most common diagnostic variants found in the United Kingdom population: DNA results from these patients have not been reported in previous publications. We sequenced DNA from 29 MH susceptible individuals, with susceptibility confirmed by IVCT, and 28 EHI individuals. All of the EHI individuals were military personnel who had had a clinical episode of heat illness during military exercises that required hospital treatment. They were referred for testing for MH susceptibility having all subsequently failed to demonstrate normal thermoregulation during a heat tolerance test carried out at the Institute of Naval Medicine (see [appendix 1](#APP1)) on at least two separate occasions. As the sensitivity of the IVCT to detect genetic predisposition to EHI, as opposed to MH susceptibility, is unknown, we included EHI individuals in this study irrespective of their IVCT result. In the group of EHI samples, 16 were from MH normal individuals (MHN, responding to neither triggering agent used in the IVCT) and 12 from MH susceptible individuals, where MH susceptibility is defined by the IVCT.

Each of the 57 samples represented a single family. For follow up of NGS, DNA from additional family members and at least 150 MH normal and up to 556 MH susceptible genetically independent individuals, including cases of EHI, were tested. If a variant is not found in the 300 chromosomes of 150 individuals it can be concluded with 95% confidence that the prevalence of the variant in the population is < 1%, assuming the data are from a Binomial distribution. Such a variant, by definition, is not a polymorphism.

### Long PCR

The samples were processed in four batches. The coding sequence of *RYR1* and *CACNA1S* was amplified in 23 and 9 PCR products respectively ([tables 1](#T1) and [2](#T2)). Amplification was carried out in 20 μl reaction volumes containing 20 ng genomic DNA template, using SequalPrep Long PCR Kits with deoxyribonucleotide triphosphates (Invitrogen, Carlsbad, CA). Reactions were optimized according to the manufacturer’s recommendations and contained 1x reaction buffer, 2% dimethyl sulfoxide, 0.5x SequalPrep EnhancerA, 1.8 U SequalPrep Long Polymerase and 250 nM each of the forward and reverse primers. The thermal cycling program was: 2 min at 94 °C for denaturation, followed by 35 cycles of 10 s at 94 °C, 30 s at 60 °C and 6 min 30 s at 68 °C. The final extension step was 5 min at 72 °C.

### Table 1.

|   | Genomic start and end of amplicon | Exons covered | Length of Amplicon (bp) | Primer sequence |
| --- | --- | --- | --- | --- |
| Fr1 | 11192355 | Ex 1 | 1285bp | 5′-GAAAGCGCAGGTACCTCCTA-3′ |
|   | 11193639 |   |   | 5′-CTCCCATATTTACGACCCCA-3′ |
| Fr2 | 11198817 | Ex 2 – 7 | 4781bp | 5′-AAAAAGCAATTTAACTTGAGCAAGT-3′ |
|   | 11203597 |   |   | 5′-GGGCAACATTAAGGGTCTGTT-3′ |
| Fr3 | 11205200 | Ex 8 – 11 | 3279bp | 5′-CAACTGCAAAACCCCAAACT-3′ |
|   | 11208478 |   |   | 5′-GGAACAGGTCGCTACAGCTC-3′ |
| Fr4 | 11209942 | Ex 12 – 16 | 5606bp | 5′-CCAGTGCCTAGAACAGAGCC-3′ |
|   | 11215547 |   |   | 5′-GCAACAGGAACTTGTAGGGC-3′ |
| Fr5 | 11216101 | Ex 17 – 20 | 4181bp | 5′-TCCTTCCTTACATTGCTGATACC-3′ |
|   | 11220281 |   |   | 5′-ACTGCGATTCTGCTCTGACAG-3′ |
| Fr6 | 11222082 | Ex 21 – 27 | 6469pb | 5′-TTCTGAGAATGCTAAGCCCTC-3′ |
|   | 11228550 |   |   | 5′-GCTGTGGTTTGCCAAAGTTAC-3′ |
| Fr7 | 11231956 | Ex 28 – 31 | 5677bp | 5′-GGAGGTTTTGGAAGTTACCCC-3′ |
|   | 11237632 |   |   | 5′-GGGCCTTAGTTTCATGGAAG-3′ |
| Fr8 | 11241613 | Ex 32 – 34 | 3576bp | 5′-TGAATGAGTATGCTTGGGAGAG-3′ |
|   | 11245158 |   |   | 5′-TTGGATGCATAAATAGAGGAGGAT-3′ |
| Fr9 | 11247847 | Ex 35 – 38 | 3762bp | 5′-GACCTGGGTGGATCTTAAGGAG-3′ |
|   | 11251608 |   |   | 5′-CTATCTACCCCTCCCTTGCATCT-3′ |
| Fr10 | 11252848 | Ex 39 – 42 | 3246bp | 5′-AGATTCTTGGGCGAATAGAAAA-3′ |
|   | 11256093 |   |   | 5′-GGAAGTTAGAGTGCGGATCC-3′ |
| Fr11 | 11257937 | Ex 43 – 49 | 4036bp | 5′-GTCTCTGACTGAGCCCCTTCT-3′ |
|   | 11261972 |   |   | 5′-GAGATTTCTACGGGGACGCT-3′ |
| Fr12 | 11262753 | Ex 50 – 58 | 4004bp | 5′-GCATCCATATGCCCATTTACTC-3′ |
|   | 11266756 |   |   | 5′-TAGGTGAGTCTGGTCTGCAGAA-3′ |
| Fr13 | 11269208 | Ex 59 – 63 | 2265bp | 5′-GGTTTATCTCAAAGCCAACACA-3′ |
|   | 11271472 |   |   | 5′-ACTGACTGCATGCTTTGCTTA-3′ |
| Fr14 | 11273519 | Ex 64 – 67 | 4937bp | 5′-GAGGAAGTACCCCTCACTTTCA-3′ |
|   | 11278455 |   |   | 5′-GAAACCAGGAGGAAGAGTCAGA-3′ |
| Fr15 | 11281751 | Ex 68 – 76 | 6296bp | 5′-CAGATGACCCTAGAAACCCCAT-3′ |
|   | 11288046 |   |   | 5′-ATTTAAAGATCCGTCTGTCTTGGG-3′ |
| Fr16 | 11291290 | Ex 77 – 84 | 5608bp | 5′-TTATAAGATGGGGGTCCTCTCC-3′ |
|   | 11296897 |   |   | 5′-GCGGAAACTAATTCAGTCAACC-3′ |
| Fr17 | 11302085 | Ex 85 – 89 | 5703bp | 5′-TGCTTTCTGGCATACAATAGGA-3′ |
|   | 11307787 |   |   | 5′-CGGTTCTCATCTGTGTTAATGC-3′ |
| Fr18 | 11318013 | Ex 90 | 3492bp | 5′-GCAAATCACAAGAGCAAAGTCTA-3′ |
|   | 11321504 |   |   | 5′-CAAGGGTTTGTGTACTGGCTC-3′ |
| Fr19 | 11322259 | Ex 91 | 3282bp | 5′-ATTACAGCCATGTGCAACCA-3′ |
|   | 11325540 |   |   | 5′-CAAGTGCAGCTCTGCTTGTC-3′ |
| Fr20 | 11325725 | Ex 92 – 93 | 1334bp | 5′-CGCTTAGGGTGAGGACTCAG-3′ |
|   | 11327058 |   |   | 5′-AGGATGGAGGTGGAAGTGTG-3′ |
| Fr21 | 11329396 | Ex 94 – 96 | 2821bp | 5′-GATGGGATGAATTCTCCAGG-3′ |
|   | 11332216 |   |   | 5′-AAACCCAGGTCCCCTCCT-3′ |
| Fr22 | 11334714 | Ex 97 – 101 | 5034bp | 5′-GACAGCTCTGATCCCTCTGG-3′ |
|   | 11339747 |   |   | 5′-ATGCATCAGCTTGCCAAACT-3′ |
| Fr23 | 11343628 | Ex 102 – 106 | 3010bp | 5′-GACCATTTCTGGCTGTTGGT-3′ |
|   | 11346637 |   |   | 5′-CTGCTCCCAGCAACCTCTAC-3′ |

Table 1 Caption: Primer Sequences for 23 Amplicons Covering All RYR1 Exons with Splice Site Intronic Fragments

### Table 2.

|   | Genomic start and end of amplicon | Exons covered | Length of Amplicon (bp) | Primer sequence |
| --- | --- | --- | --- | --- |
| Fr1 | 201081807 | Ex 1–2 |   | 5′-ATCCCAGGCCTGTGTGTAAG-3′ |
|   | 201078421 |   | 3406bp | 5′-CAGAGAGACCAAGCTGGAGG-3′ |
| Fr2 | 201063193 | Ex 3–5 |   | 5′-TCAGAGAGCACAGAGGCATG-3′ |
|   | 201060615 |   | 2598bp | 5′-AGTAGAGCTGCCCTTCCTCC-3′ |
| Fr3 | 201058618 | Ex 6–10 |   | 5′-GATGTTCCCTTTGCCTTCTG-3′ |
|   | 201052161 |   | 6477bp | 5′-TCCAGCACTCATGGTCCATA-3′ |
| Fr4 | 201047287 | Ex 11–16 |   | 5′-GGAGTCAGGAGAAGGGAAGG-3′ |
|   | 201041788 |   | 5519bp | 5′-ATGGAGGGGTACAGGTAGGG-3′ |
| Fr5 | 201039853 | Ex 17–22 |   | 5′-CCCCCAGCCTATGGATATTT-3′ |
|   | 201034869 |   | 5004bp | 5′-TGGAAATCTGTGCATTCGAA-3′ |
| Fr6 | 201031968 | Ex 23–28 |   | 5′-ACTCATGTCCATCCCCTCAG-3′ |
|   | 201027075 |   | 4913bp | 5′-AGCCTTGCCCTCTCTGTTCT-3′ |
| Fr7 | 201023879 | Ex 29–32 |   | 5′-AGCTTCAGTCCTCACTCCCA-3′ |
|   | 201021442 |   | 2457bp | 5′-GGAGGAAAATGCAGCACTTC-3′ |
| Fr8 | 201020682 | Ex 33–38 |   | 5′-TTGTGACTCAGCCAGAGTGG-3′ |
|   | 201015627 |   | 5075bp | 5′-CCCATACCCAGTGGAACATT-3′ |
| Fr9 | 201013956 | Ex 39–44 |   | 5′-TAAGCCATACGAAGCGGTTC-3′ |
|   | 201008352 |   | 5604bp | 5′-CCAATGCAACATGGTCGTAG-3′ |

Table 2 Caption: Primer Sequences for Nine Amplicons Covering All CACNA1S Exons with Splice Site Intronic Fragments

Amplicon size was verified on 0.7% agarose gels and concentrations were measured using QuantIT PicoGreen BR Kits (Invitrogen, Eugene, OR). For each patient, the PCR products were pooled in an equimolar ratio. The pooled PCR products were sheared using an ultrasonicator (Covaris S2, KBioscience, Hertfordshire, United Kingdom) to produce fragments of approximately 200 bp. Subsequently, sheared samples were purified with MiniElute Purification Kits (Qiagen GmbH, Hilden, Germany) and the quality checked on an Agilent 2100 Bioanalyser (Agilent Technologies, Waldbronn, Germany). The SPRIworks Fragment Library System I (Beckman-Coulter, Brea, CA) was used for tagged library preparation. Ten samples can be processed in parallel. During the process unique indexed adaptors are ligated to each pooled sample. In the next step, individual libraries were subjected to enrichment. Phusion High-Fidelity Master Mix (ThermoFisherScientific, Waltham, MA) and 3′PTO modified primers (Fisher Scientific UK Ltd., Loughborough, United Kingdom) were used to amplify the libraries:

PTO-F

PTO-R

PCR was carried out in 50 μl volumes containing 5 μl of library DNA, 25 μl of Phusion HF Master Mix and 500 nM of each primer. The thermal cycling program was: 98 °C for 30 s, then 12 cycles of 98 °C for 10 s, 65 °C for 30 s and 72 °C for 30 s, and the final extension was 72 °C for 5 min. Enriched libraries were then purified with Agencourt^®^AMPure^®^XP (Beckman Coulter) according to the manufacturer’s protocol. Purified libraries are pooled in equimolar ratios and loaded on one lane of a flow cell for cluster generation. Libraries were subsequently sequenced on an Illumina GAII^®^ or MiSeq^®^.

Output files from the sequencing platform were sorted according to barcode tags using Illuminator Data Extractor (preprocessing software) and sequence analysis was performed using Illuminator software^16^, both of which are freely available on the University of Leeds website^*^. This software filters out base calls with a Phred (Q) score < 25. Reads were aligned against GeneBank genomic and coding reference sequences. For *RYR1* the genomic reference sequence was [NC_000019.9](https://www.ncbi.nlm.nih.gov/nuccore/NC_000019.9) (38923137-39079419) and transcript [NM_000540.2](https://www.ncbi.nlm.nih.gov/nuccore/NM_000540.2). For *CACNA1S* the genomic reference sequence was NC_0000001.10 (201008331-201081807) and transcript [BC133671.1](https://www.ncbi.nlm.nih.gov/nuccore/BC133671.1).

### Testing of missing fragments, variants in family segregation studies and independent samples

Sanger sequencing was used routinely for *RYR1* exon 91, since amplification of Long PCR fragments failed and for exon 102 in all but one case for the same reason, despite protracted attempts to optimise primers. The primers used for Sanger sequencing to provide full coverage of these exons are provided in [appendix 2](#APP2). The primers for Sanger sequencing of other exons are available from the authors on request.

When a variant was identified in an MH patient or an EHI patient who tested MH susceptible by IVCT, there was a possibility of testing further family members to look for cosegregation of variant and disease phenotype. Sanger sequencing was used to test *RYR1* and *CACNA1S* variants, identified through NGS, on additional family members (primers not supplied).

PCR based restriction fragment length polymorphism and amplification-refractory mutation system assays were designed for specific variants and used to screen unrelated MH susceptible and MH normal DNA samples. c.641C>T was screened by PCR and restriction fragment length polymorphism (F5′ ACC CTT GGC CTG AAA ATA CC, R5′ TGA AGT CAA GGG TTC AGC TC and *BccI*). c.9676G>C was also screened by PCR and restriction fragment length polymorphism (F5′ GCA CTG CAG CCT GAG TAA CA, R5′ CCC CCG AAC CAT AAA CTC TG and *BsrBI*). c.11958C>G was tested using an assay combining PCR products from exon 87 (the mutation site) and exon 24 to differentiate products after digestion with *EcoNI* (Exon 87 F5′ GTG ATC CCT GAT CCC TTC TC, R5′ GAA GCA GGT GGA TGG AGA C; Exon 24 F5′ GAC AAG GGT CAG CAG TCA GG, R5′ GGG TCA GAG TTG GGG TAG GA). c.12028G>A in exon 88 was tested by an amplification-refractory mutation system assay (Mutation specific primers: F′ CTA GGA CTC AAG CCA GAT CA, R′ GAT GGG GTT GAG GAT TAG GG and exon 44 control primers: F′ GGG AGG TCT CTG ATG GTG, R′ CGG GAG ACT CAC TGC TCG).

### Prediction of variant pathogenicity

To predict the possible functional relevance of all new variants, the impact of the amino acid substitution on the structure and function of the protein was estimated using the PolyPhen-2 (Polymorphism Phenotyping v2)^17^ and Combined Annotation–Dependent Depletion^18^ bioinformatics tools. The PolyPhen-2 program assigns the variant a score between 0 and 1.0: the higher the score, the more likely is the variant to be pathogenic. It should be noted that no bioinformatics tool has been validated for *RYR1* or *CACNA1S*. Our previous work suggests that a PolyPhen-2 score of >0.95 is sensitive but not specific for pathogenic *RYR1* variants^15^. The Combined Annotation–Dependent Depletion is a recently published method that combines information from multiple sources into a single C-score. The C-score provides a better correlation with pathogenicity than any other *in silico* prediction program or combinations of these^18^ but has not been validated for, or even applied in the literature to, *RYR1* and *CACNA1S* variants.

## Results

### Quality of sequencing

For all samples the sequence quality reached the quality threshold and read depth was above the commonly applied minimum for diagnostic use of 50 by a considerable margin in the great majority of fragments. The mean read depth by exon was 403 (see [table 3](#T3) for read depth for each exon). Despite extensive efforts to optimise primers, sequence quality was insufficient to reach the base-calling threshold in all samples for *RYR1* exon 91 and all but one sample for *RYR1* exon 102. Sanger sequencing was used for these exons and other exons that failed the quality threshold in individual samples. From the first batch of samples *RYR1* fragments 4 and 15 ([table 1](#T1)) failed in three samples and two samples respectively. For subsequent batches of samples the primers for these fragments were redesigned with good results ([table 3](#T3)). Otherwise *RYR1* fragments 13 and 16 ([table 1](#T1)) and *CACNA1S* fragments 1 and 3 ([table 2](#T2)) failed in one patient each, presumably as a result of pipetting errors.

### Table 3.

| RYR1 |  |  |  |  |  | CACNA1S |  |
| --- | --- | --- | --- | --- | --- | --- | --- |
| Exon | Average read depth | Exon | Average read depth | Exon | Average read depth | Exon | Average read depth |
| 1 | 358 | 48 | 218 | 95 | 370 | 1 | 360 |
| 2 | 346 | 49 | 437 | 96 | 842 | 2 | 465 |
| 3 | 319 | 50 | 394 | 97 | 599 | 3 | 454 |
| 4 | 393 | 51 | 431 | 98 | 358 | 4 | 371 |
| 5 | 405 | 52 | 369 | 99 | 327 | 5 | 468 |
| 6 | 348 | 53 | 453 | 100 | 446 | 6 | 402 |
| 7 | 420 | 54 | 465 | 101 | 360 | 7 | 433 |
| 8 | 328 | 55 | 418 | 102 | 1 | 8 | 418 |
| 9 | 429 | 56 | 440 | 103 | 368 | 9 | 326 |
| 10 | 354 | 57 | 344 | 104 | 418 | 10 | 417 |
| 11 | 372 | 58 | 902 | 105 | 416 | 11 | 349 |
| 12 | 487 | 59 | 199 | 106 | 313 | 12 | 471 |
| 13 | 425 | 60 | 417 |   |   | 13 | 402 |
| 14 | 495 | 61 | 544 |   |   | 14 | 440 |
| 15 | 524 | 62 | 375 |   |   | 15 | 409 |
| 16 | 436 | 63 | 459 |   |   | 16 | 700 |
| 17 | 403 | 64 | 357 |   |   | 17 | 381 |
| 18 | 338 | 65 | 327 |   |   | 18 | 427 |
| 19 | 374 | 66 | 317 |   |   | 19 | 435 |
| 20 | 361 | 67 | 356 |   |   | 20 | 399 |
| 21 | 254 | 68 | 251 |   |   | 21 | 326 |
| 22 | 296 | 69 | 476 |   |   | 22 | 964 |
| 23 | 341 | 70 | 487 |   |   | 23 | 435 |
| 24 | 316 | 71 | 406 |   |   | 24 | 416 |
| 25 | 344 | 72 | 459 |   |   | 25 | 475 |
| 26 | 353 | 73 | 378 |   |   | 26 | 428 |
| 27 | 353 | 74 | 472 |   |   | 27 | 429 |
| 28 | 298 | 75 | 437 |   |   | 28 | 468 |
| 29 | 395 | 76 | 534 |   |   | 29 | 450 |
| 30 | 318 | 77 | 547 |   |   | 30 | 492 |
| 31 | 334 | 78 | 291 |   |   | 31 | 544 |
| 32 | 434 | 79 | 320 |   |   | 32 | 371 |
| 33 | 326 | 80 | 404 |   |   | 33 | 408 |
| 34 | 353 | 81 | 394 |   |   | 34 | 415 |
| 35 | 378 | 82 | 199 |   |   | 35 | 379 |
| 36 | 371 | 83 | 305 |   |   | 36 | 386 |
| 37 | 447 | 84 | 278 |   |   | 37 | 410 |
| 38 | 563 | 85 | 274 |   |   | 38 | 410 |
| 39 | 374 | 86 | 281 |   |   | 39 | 414 |
| 40 | 397 | 87 | 301 |   |   | 40 | 409 |
| 41 | 364 | 88 | 327 |   |   | 41 | 373 |
| 42 | 358 | 89 | 335 |   |   | 42 | 416 |
| 43 | 675 | 90 | 405 |   |   | 43 | 364 |
| 44 | 253 | 91 | 0 |   |   | 44 | 390 |
| 45 | 376 | 92 | 1128 |   |   |   |   |
| 46 | 334 | 93 | 423 |   |   |   |   |
| 47 | 351 | 94 | 360 |   |   |   |   |

Table 3 Caption: Average Read Depth for Each Exon of RYR1 and CACNA1S

#### RYR1

Nineteen *RYR1* variants, 4 of which are novel (p. 2248R>C, p.3226E>Q, p.4010E>K, p.4230M>R), were found in 20 MH patients ([table 4](#T4)) and 11, 3 of which are novel (p.492R>H, p.4282L>V, p.4331A>T), in 13 EHI patients ([table 5](#T5)). Three polymorphic changes were found in the *RYR1* coding sequence, based on frequencies supplied for all populations on the Exome Variant Server^†^ ([table 6](#T6)), and a further two variants have been reported not to segregate with the MH phenotype (p.1787P>L^19^ and p.3253I>T^20^). All variants were seen as heterozygotes. *RYR1* variants described in this paper are illustrated in [figure 1](#F1) and their PolyPhen-2 scores and C-scores given in [table 6](#T6).

| Sample ID | RYR1 | CACNA1S |
| --- | --- | --- |
| MH1 | c.1840C>T; p.614R>C | c.1373T>A; p.458L>H |
| MH2 | c.641C>T ; p.214T>M | c.1373T>A; p.458L>H |
| MH3 | c.9676G>C; p.3226E>Q | c.1373T>A; p.458L>H- |
| MH4 | c.14168G>A; p.4723R>H | c.1373T>A; p.458L>H |
| MH5 | - | - |
| MH6 | - | c.1373T>A; p.458L>H |
| MH7 | c.488G>T; p.163R>L | c.4615C>T; p.1539R>C |
| MH8 | - | c.1373T>A; p.458L>Hc.3026C>A; p.1009T>K- |
| MH9 | - | - |
| MH10 | - | c.206C>G; p.69A>Gc.4615C>T; p.1539R>C |
| MH11 | - | c.4615C>T; p.1539R>C |
| MH12 | c.1654C>T; p.552R>Wc.11266C>G; p.3756Q>E | c.4615C>T; p.1539R>C |
| MH13 | - | c.1373T>A; p.458L>H-:c.1817G>A; p.606S>N- |
| MH14 | c.12689T>G; p.4230M>R | c.206C>G; p.69A>Gc.1373T>A; p.458L>H |
| MH15 | - | c.206C>G; p.69A>G:c.1373T>A; p.458L>H |
| MH16 | c.6178G>T; p.2060G>Cc.8327C>T; p.2776S>F | c.1817G>A; p.606S>N |
| MH17 | - | c.1373T>A; p.458L>H |
| MH18 | c.529C>T; p.177R>C | c.206C>G; p.69A>Gc.1373T>A; p.458L>H |
| MH19 | c.4711A>G; p.1571I>Vc.6178G>T; p.2060G>Cc.10097G>A; p.3366R>Hc.11798A>G; p.3933Y>Cc.113964G>A; p.4010E>K | - |
| MH20 | - | c.1373T>A; p.458L>H |
| MH21 | c.6742C>T; p.2248R>C | - |
| MH22 | c.6178G>T; p.2060G>C | c.206C>G; p.69A>Gc.1373T>A; p.458L>H |
| MH23 | - | c.1373T>A, p.458L>H |
| MH24 | c.6178G>T, p.2060G>C | c.5399T>C, p.1800S>L |
| MH25 | c.4178A>G, p.1393K>R | c.206C>G, p.69A>Gc.1373T>A, p.458L>H |
| MH26 | - | c.1373T>A, p.458L>H |
| MH27 | c.6178G>T, p.2060G>Cc.11958C>G, p.3986D>E | - |
| MH28 | - | c.1373T>A, p.458L>H |
| MH29 | c.4405C>T, p.1469R>Wc.6178G>T, p.2060G>C | c.1373T>A, p.458L>H |

Table Caption: Variants Detected in RYR1 and CACNA1S in 29 Malignant Hyperthermia Samples

| Sample ID | IVCT Result | RYR1 | CACNA1S |
| --- | --- | --- | --- |
| EHI1 | MHN | c.12844C>G; p.4282L>V | - |
| EHI2 | MHN | c.5360C>T; p.1787P>Lc.6178G>T; p.2060G>Cc.12884C>T; p.4295A>V | c.1373T>A; p.458L>H |
| EHI3 | MHN | - | c.206C>G; p.69A>Gc.1373T>A; p.458L>H |
| EHI4 | MHN | c.11266C>G; p.3756Q>E | c.1373T>A; p.458L>H |
| EHI5 | MHN | - | c.1373T>A; p.458L>H |
| EHI6 | MHN | c.6178G>C; p.2060G>C;c.12991G>A; p.4331A>T | - |
| EHI7 | MHN | - | c.1373T>A; p.458L>H |
| EHI8 | MHN | c.6178G>T; p.2060G>C;c.10747G>C; p.3583E>Q | c.1373T>A; p.458L>H |
| EHI9 | MHN | - | c.5515C>T; p.1839P>S |
| EHI10 | MHN | - | - |
| EHI11 | MHN | c.5360C>T; p.1787P>Lc.6178G>T; p.2060G>C | c.1493G>A; p.498R>Hc.4615C>T; p.1539R>C |
| EHI12 | MHN | - | c.1373T>A; p.458L>H |
| EHI13 | MHN | - | - |
| EHI14 | MHN | c.6178G>T, p.2060G>C | c.4615C>T, p.1539R>C |
| EHI15 | MHN | c.6178G>T, p.2060G>C | - |
| EHI16 | MHN | c.6178G>T, p.2060G>C | c.4615C>T, p.1539R>C |
| EHI17 | MHS | - | c.1373T>A; p.458L>H |
| EHI18 | MHS | - | c.1817G>A; p.606S>N |
| EHI19 | MHS | c.4024A>G; p.1342S>Gc.11941C>T; p.3981H>Y | c.1373T>A; p.458L>Hc.2047C>T; p.683R>C |
| EHI20 | MHS | c.6178G>T; p.2060G>Cc.11266C>G; p.3756Q>E | c.1373T>A; p.458L>H |
| EHI21 | MHS | - | - |
| EHI22 | MHS | c.4024A>G; p.1342S>G | - |
| EHI23 | MHS | c.9758T>C; p.3253I>T | c.1817G>A; p.606S>N |
| EHI24 | MHS | - | c.4615C>T; p.1539R>C |
| EHI25 | MHS | - | - |
| EHI26 | MHS | c.1475G>A; p.492R>H | c.4615C>T; p.1539R>C |
| EHI27 | MHS | - | - |
| EHI28 | MHS | - | c.206C>G; p.69A>Gc.1373T>A; p.458L>H |

Table Caption: Variants Detected in RYR1 and CACNA1S in 28 Exertional Heat Illness Samples

| Exon | Nucleotide change | Amino acid change | Sample phenotype (this study) | Reported disease association or polymorphism | Variant ID | MAF (from EVS) |
| --- | --- | --- | --- | --- | --- | --- |
| 6 | c.488G>T | p.163R>L | 1 MHS | MH diagnostic | rs193922753 | - |
| 6 | c.529C>T | p.177R>C | 1 MHS | MH uncharacterized | rs193922757 | - |
| 8 | c.641C>T | p.214T>M | 1 MHS | MH uncharacterized | - | - |
| 14 | c.1475G>A | p.492R>H | 1 EHI (MHS) | - | - | - |
| 15 | c.1654C>T | p.552R>W | 1 MHS | MH diagnostic | rs193922770 | - |
| 17 | c.1840C>T | p.614R>C | 1 MHS | MH/CCD diagnostic | rs118192172 | 0.000077 |
| 28 | c.4024A>G | p.1342S>G | 2 EHI (MHS) | MH uncharacterized | rs34694816 | 0.04 |
| 29 | c.4178A>G | p.1393K>R | 1 MHS | MH uncharacterised | rs137933390 | 0.004 |
| 30 | c.4405C>T | p.1469R>W | 1MHS | CNM | rs200546266 | 0.000077 |
| 33 | c.4711A>G | p.1571I>V | 1 MHS | MH uncharacterized | rs146429605 | 0.000929 |
| 34 | c.5360C>T | p.1787P>L | 2 EHI (MHN) | - | rs34934920 | 0.00538 |
| 38 | c.6178G>T | p.2060G>C | 6 MHS1 EHI(MHS)7 EHI (MHN) | Polymorphism | rs35364374 | 0.05 |
| 41 | c.6742C>T | p.2248R>C | 1 MHS | - | - | c.6743G>A 0.000077 |
| 53 | c.8327C>T | p.2776S>F | 1 MHS | - | rs147707463 | 0.000923 |
| 65 | c.9676G>C | p.3226E>Q | 1 MHS | - | - | - |
| 66 | c.9758T>C | p.3253I>T | 1 EHI (MHS) | - | Rs375626634 | 0.000077 |
| 67 | c.10097G>A | p.3366R>H | 1 MHS | MH/myopathy | rs137932199 | 0.00846 |
| 72 | c.10747G>C | p.3583E>Q | 1 EHI (MHS) | Polymorphism | rs55876273 | 0.03 |
| 79 | c.11266C>G | p.3756Q>E | 1 MHS1 EHI (MHS)1 EHI (MHN) | Polymorphism | rs4802584 | 0.018 |
| 86 | c.11798A>G | p.3933Y>C | 1 MHS | MH/myopathy | rs147136339 | 0.000846 |
| 87 | c.11941C>T | p.3981H>Y | 1 EHI (MHS) | CNM | rs148772854 | 0.0039 |
| 87 | c.11958C>G | p.3986D>E | 1 MHS | MH uncharacterized | rs193922842 | - |
| 88 | c.12028G>A | p.4010E>K | 1 MHS | Unknown | - | - |
| 91 | c.12689T>G | p.4230M>R | 1 MHS | Unknown | - | - |
| 91 | c.12844C>G | p.4282L>V | 1 EHI (MHN) | Unknown | - | - |
| 91 | c.12884C>T | p.4295A>V | 1 EHI (MHN) | MH/minicores | rs193922855 | - |
| 91 | c.12991G>A | p.4331A>T | 1 EHI (MHN) | - | - | - |
| 97 | c.14168G>A | p.4723R>H | 1 MHS | - | rs200766617 | c.14167C>T 0.000077 |

Table Caption: RYR1 Variant Characteristics

![Fig. 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8fec/4472733/87b030d66ec6/nihms690187f1.jpg)

Map of the RYR1 gene indicating the position of variants found in this study.

We found three variants (p.163L>R^14,21^, p.552R>W^22^, p.614R>C^23^), previously reported in MH patients, that have functionally characterised effects consistent with a pathogenic role in malignant hyperthermia. Four further variants previously described in the literature in MH patients (p.177R>C, p.1342S>G, p.1571I>V, p.3986D>E)^14,19,24–26^ are yet to be functionally characterised. p.177R>C affects a well-conserved amino acid, segregates with the MHS phenotype and was not found in 100 control chromosomes^14,19,24^. We found *RYR1* c.11958C>G p.3986D>E in a further 3 of 541 independent MH susceptible individuals and none of 177 MH negative controls. Of the family members tested in all four families, disease phenotype and variant status were concordant but only in a small total number of meioses. p.3986D>E has a PolyPhen-2 score of 0.999 and C-score of 12.60.

The variant c.641C>T; p.214T>M was found in a patient who had a clinically suspected MH reaction during anesthesia but who did not have an IVCT. Two brothers of the proband did have an IVCT, however, and the p.214T>M variant was found in one brother who tested MH susceptible by IVCT but not the other brother who tested MH normal, which is consistent with the variant being associated with MH susceptibility. The PolyPhen-2 score for p.214T>M is 1.000 and the C-score 14.48. Furthermore, p.214T>M was found in three of 556 MH susceptible samples and all members of the three families show concordance of disease and/or IVCT phenotype with variant status. All of 177 MH negative samples tested lacked the substitution.

Variant p.4295A>V was described by Jeong *et al*.^27^ as a compound mutation (with p.2435R>H) in an MH family with histological minicores and elevated serum creatine kinase: some individuals developed a late onset myopathy. In our study this variant appears in an EHI patient who tested MH normal by IVCT.

*RYR1* c.14168G>A; p.4723R>H was found in a patient who had a clinically suspected MH reaction confirmed by IVCT. Despite annotation through the 1000 Genomes Project^28^, the variant did not appear on the Exome Variant Server ([table 6](#T6)). However the variant was found in the proband’s son who was MH negative by IVCT. No further work was carried out on this variant.

*RYR1* c.4178A>G p.1393K>R was found in a patient with a clinically suspected MH reaction and detected in 2 further MH families through screening 556 unrelated MH susceptible samples. Subsequent testing of family members revealed individuals discordant for variant status and disease phenotype in both families, which is perhaps not surprising considering the minor allele frequency reported in the Exome Variant Server, and suggesting that this variant is unrelated to MH status.

The four new variants found in MH patients were located in exons 41, 65, 88, and 91. The c.9676G>C; p.3226E>Q *RYR1* variant was found in a MH proband whose mother was tested MH susceptible and also carried the variant. The brother and maternal cousin of the proband did not carry the variant and were tested MH negative by the IVCT. The PolyPhen-2 score was 0.997 and the C-score 14.93. This substitution was not seen in 211 MH negative or 530 independent MH susceptible samples. There were no family members to conduct further studies on p.2248R>C (PolyPhen-2 1.000, C-score 15.92) or p.4230T>G (PolyPhen-2 0.932, C-score 12.83).

Variant p.4010E>K was found in a MH susceptible woman who had four other *RYR1* variants ([table 4](#T4)): p.1571I>V, p.2060G>C, p.3366R>H and p.3933Y>C. Her son was the clinical proband and was too ill for an IVCT: he was negative for all variants except p.4010E>K. The Polyphen-2 score for this variant is 0.07 which suggests that it is unlikely to be damaging, although the C-score is 15.14. Two hundred MH negative controls and 422 MH susceptible individuals tested negative for the variant. The father of the proband was MH negative by IVCT and negative for all variants.

One of three new *RYR1* variants found in EHI patients is c.1475G>A; p.492R>H, which occurs in an EHI patient found to be MH susceptible by IVCT. At the same amino acid position there is another variant described in the 1000 Genomes Project^28^, c.1474C>T; p.492R>C. There were no family members available for segregation analysis. The PolyPhen-2 result for the new variant is 0.986 while the C-score is only 7.26. The other new variants identified in EHI patients, p.4282L>V and p.4331A>T, occur in EHI patients found to be MHN by IVCT and so no further family members have been tested by IVCT. The PolyPhen-2 scores and C-scores ([table 6](#T6)) suggest that they are benign.

#### CACNA1S

Eight of nine missense variants found in the *CACNA1S* coding sequence were previously described^7,8^. All of these are listed in the 1000 Genomes Project ([table 7](#T7)). In our patients four polymorphic changes were found in the *CACNA1S* coding sequence, based on frequencies supplied for all populations on the Exome Variant Server, and a single additional low frequency variant was seen, which had been reported as polymorphic elsewhere ([table 7](#T7)). Consistent with polymorphic status, c.206C>G; p.69A>G and c.1373T>A; p.458L>H were homozygous in some of the sequenced samples. A variant with a low minor allele frequency (p.683R>C; minor allele frequency 0.00976) was found in an EHI patient who had tested MH negative by IVCT ([table 5](#T5)). The relevance of this variant to EHI remains to be established.

| Exon | Nucleotide change | Amino acid change | Sample phenotype (this study) | Reported disease association | Variant ID | MAF (from EVS) |
| --- | --- | --- | --- | --- | --- | --- |
| 5 | c.206C>G | p.69A>G | 6 MHS1 EHI (MHS)1 EHI (MHN) | Polymorphism | rs12406479 | 0.0392 |
| 10 | c.1373T>A | p.458L>H | 20 MHS4 EHI (MHS)7 EHI (MHN) | Polymorphism | rs12742169 | 0.274 |
| 11 | c.1493G>A | p.498R>H | 1 EHI (MHN) | Unknown | rs150590855 | 0.00084 |
| 12 | c.1817G>A | p.606S>N | 2 MHS2 EHI (MHS) | Unknown | rs142356235 | 0.00868 |
| 14 | c.2047C>T | p.683R>C | 1 EHI (MHS) | Unknown | rs35708442 | 0.00976 |
| 24 | c.3026C>A | p.1009T>K | 1 MHS | Unknown | - | c.3026C>T 0.000154 |
| 38 | c.4615C>T | p.1539R>C | 4 MHS2 EHI (MHS)3 EHI (MHN) | Polymorphism | rs3850625 | 0.0892 |
| 44 | c.5399T>C | p.1800L>S | 3 MHS4 EHI (MHS)2 EHI (MHN) | Polymorphism | rs12139527 | 0.269 |
| 44 | c.5515C>T | p.1839P>S | 1 EHI (MHN) | Unknown | rs149547196 | 0.00253 |

Table Caption: CACNA1S Variant Characteristics

In the MH group we found *CACNA1S* variant c.3026C>A, A**C**G/A**A**G; p.1009T>K in one individual ([tables 4](#T4) and [7](#T7)): this variant was previously found by exome sequencing in another family from the United Kingdom^15^. In the 1000 Genomes Project there is an annotated variant changing threonine at position 1,009 to methionine (c.3026C>T; A**C**G/A**T**G). The Exome Variant Server provides a minor allele frequency of 0.000154 for the T>M substitution. Both substitutions are predicted to give probably damaging amino acid changes with PolyPhen-2 scores of 1.0 and high C-scores ([table 7](#T7)). Segregation studies showed the new variant not to segregate with the MH IVCT phenotype in the family from the current study. Although three MHS individuals carried the variant and six MHN individuals did not carry the variant, there were two MHN individuals who carried the variant. This is a similar situation to the family with the same variant reported by Kim *et al*^15^, where there was a LOD (logarithm of odds) score of 2.12 but one MHN individual carried the p.1009T>K variant.

## Discussion

We have used NGS technology to sequence the coding regions of the *RYR1* and *CACNA1S* genes in 29 MH patients and 28 EHI patients. By limiting our analyses to two genes we were able to simultaneously sequence samples from multiple patients while achieving very high read depths ([table 3](#T3)) in all *CACNA1S* exons and all but two *RYR1* exons. Combined with the high base-calling quality of the sequencing platforms used, this provides a high level of accuracy of sequencing in an efficient manner (once the primers were optimized, preparation of the enriched libraries for sequencing took 2 weeks). We had to resort to Sanger sequencing for *RYR1* exons 91 and 102, where the quality of NGS did not meet our quality thresholds, presumably because of the high proportion of guanine and cytosine bases in these regions.

Testing for diagnostic variants by amplification-refractory mutation system and PCR- restriction fragment length polymorphism methods is time consuming and usually a staged procedure, taking place over a considerable period of time. Necessarily the most prevalent variants are likely to be assayed first. An NGS diagnostic approach is more efficient with individuals fully sequenced for all variants at one time, with both the diagnostic variants and the wealth of uncharacterized variants, many of which may still prove to have functional relevance, detected. An NGS approach using long-range PCR has been employed successfully for diagnostic detection of *BRCA1* and *BRCA2* mutations in cases of familial breast cancer^29,30^. A real clinical and possible ethical problem with NGS approaches to diagnosis is deciding what information is given to referring clinicians and patients concerning variants of unknown or uncertain significance. Guidelines for reporting variants have been produced by professional bodies in Europe^‡^ (joint British/Dutch guidelines) and the United States^31^. The joint British/Dutch guidelines are especially useful as they provide a consensus view on the evidence required for inclusion in each class of variant. This includes the necessity to demonstrate the functional effects of missense variants before the variant can be considered to be pathologic, reiterating the position of the European MH Group with respect to the use of DNA findings in MH diagnosis^32^. The guidelines also caution against reliance on *in silico* predictions of pathogenicity in the diagnostic context and we would agree with this based on our previous analyses of *RYR1* and *CACNA1S* in control populations^15^. We propose to limit the use of PolyPhen-2 and C-scores as part of the process to prioritize variants for functional studies.

In the MH cohort we detected three variants that meet the European MH Group criteria for diagnostic use in assigning high risk of MH susceptibility. DNA from the individuals carrying these variants had not been sequenced prior to this study and screening for a full panel of diagnostic mutations was not complete in all cases. One of the variants detected was yet to be screened in the affected individual and another had been missed on routine screening, highlighting the sensitivity of NGS sequencing compared to conventional screening, including Sanger sequencing in this context^15,29^. Of the remaining 26 MH patients we detected variants of possible clinical relevance in 10 patients. In the 28 EHI patients we detected variants of possible clinical relevance in 7 individuals, 2 of whom had abnormal IVCT and 5 who had normal IVCT responses. In the MH cohort there was only one *CACNA1S* of possible relevance, the remainder being in *RYR1*, while 3 out of 7 of the variants of possible relevance in the EHI cohort were in *CACNA1S*.

This study represents the first report of sequencing the complete coding sequence of *RYR1* and *CACNA1S* in a cohort of survivors of EHI who have been physiologically characterized using heat tolerance testing and investigated by IVCT. Other than for several common variants, the functional and clinical relevance of the variants found in our EHI cohort is unclear. Uncommon variants were found in *RYR1* and *CACNA1S* in EHI patients who tested either positive or negative in the IVCT. However the prevalence of susceptibility to EHI is unknown, making interpretation based on variant frequency difficult. This is compounded by the absence of family segregation studies and indeed uncertainty regarding the genetic model operating in heritable predisposition to EHI. It is more than 20 yr since we first reported familial IVCT abnormalities in survivors of EHI^33^ but the sensitivity and specificity of the IVCT for identifying skeletal muscle abnormalities associated with EHI is unknown. Rather than trying to define crude IVCT cut-offs for this purpose (which has been clinically useful in MH susceptibility) it may be more fruitful to utilize the extensive quantitative information that can be derived from these tests, as we have done when analyzing the effects of different *RYR1* mutations in MH susceptible patients ^34^. As all of the variants that were found in both MH and EHI cohorts were polymorphisms or of uncertain significance, our findings are not helpful in advancing understanding of the relationship between the two conditions.

Variants in the present study were detected throughout *RYR1*. Due to the size of *RYR1*, attention historically focused on three hotspot regions where the first diagnostic variants were found ^34^. Sequencing of the entire gene is revealing variants throughout the gene. The RYR1 protein has many direct, diverse protein and subunit interactions. Whilst particular regions and residues will be conserved and functionally important, the expanding range of variants is not surprising. To date, the assumption has been made that primary mutations causing MH are likely to be missense, coding variants. However, the expression of messenger RNA in heterozygous MH susceptible individuals has been explored by Grievink and Stowell ^35^ and their findings suggest allele specific differences in *RYR1* expression may also contribute to phenotype. The possibility that noncoding variants are involved has not been explored.

Although MH shows an autosomal dominant pattern of inheritance, in some MH families discordant individuals have been identified, where IVCT confirmed phenotype is discordant with genotype ^36–38^, which suggests complexities in the genetics of the disease^11^. One explanation for discordancy in IVCT tested MH susceptible individuals is that there are some as yet undiscovered variants in *RYR1* or *CACNA1S* or other genes, which contribute to MH phenotype, while modifier loci have been implicated^39,40^: NGS may help to resolve discordancy in some families.

The current strategy for MH is to regard individuals carrying a familial diagnostic variant as susceptible. Diagnosis is conservative and these individuals would not require an IVCT. However individuals who do not carry a familial diagnostic variant cannot be regarded as unaffected, due to our incomplete knowledge of the genetics of MH, and these individuals undergo the IVCT. Of course this strategy is only appropriate for families where a proband has been screened for mutations and a familial diagnostic variant already identified. Incomplete knowledge concerning the genetics of MH means that a sensitive test based on identification of particular diagnostic variants is some way away. The last decade has seen many groups sequencing and cataloguing variants relevant to MH, some with information relating to diagnosis, focusing on *RYR1* and *CACNA1S*. Recently NGS methods have been employed to sequence genes involved in excitation-contraction coupling^20^ or the whole exome^15,41^. To expand the range of diagnostic variants parallel efforts are required to identify further variants in *RYR1*, *CACNA1S* and other loci, and to obtain functional information on the nature of the variants.

A number of samples sequenced for *RYR1* and *CACNA1S* in the present study carried several variants, particularly those of higher frequency. Minor allele frequencies from public databases, or inferred from our own screening, were combined with other information for interpretation of the likely nature of variants (see [appendix 3](#APP3) for an example). The description of variants as ‘polymorphic’ was not necessarily consistent across possible sources. Although the prevalence of MH is difficult to assess, primary causative missense mutations are highly unlikely to have frequencies greater than 0.001. However, the notion of common polymorphisms, or even rare variants, modifying the effects of mutations, such as the predicted damaging change *CACNA1S* c.3026C>A, cannot be ruled out in some instances^42,43^. *RYR1* variants are discussed here in the context of MH. However *RYR1* mutations are also known to be responsible for rare, congenital myopathies, which show both dominant and recessive modes of inheritance^44^ such as autosomal recessive central core disease^45,46^. NGS will undoubtedly reveal variants of interest to a number of disorders.

In summary, we have reported NGS using long range PCR of *RYR1* and *CACNA1S*, the only two loci with variants with functional evidence supporting their role in MH. NGS is a fast and efficient diagnostic tool and a significant improvement on the current screening of diagnostic genetic variants. Additionally, complete coding sequence information will rapidly expand our knowledge of variants associated with MH and related disorders. However, functional studies will then be required to define the importance of these variants to the condition in question.

### What we already know about this topic

### What we already know about this topic

### What we already know about this topic

### What we already know about this topic

### What this article tells us that is new

## Acknowledgments

**Funding was provided by:**

The National Institutes of Health (Bethesda, Maryland): National Institute of Arthritis, Musculoskeletal, and Skin Diseases (2P01 AR05235 to Dr. Hopkins); National Institute of General Medical Sciences (T32 GM086270 to Dr. Kim); National Center for Advancing Translational Sciences (KL2 TR000421 to Dr. Kim). The content is solely the responsibility of the authors and does not necessarily represent the official views of the National Institutes of Health.

The Medical Research Council (London, United Kingdom), Medical Bioinformatics Centre grant MR/L01629X/1 to The University of Leeds

## Appendix 1. Summary of the Heat Tolerance Assessment Undertaken at the Institute of Naval Medicine, Alverstoke, United Kingdom

For the British military there are published guidelines^47^, which require all military personnel who have suffered a significant episode of heat illness or required urgent admission to hospital with heat illness to be subsequently referred and assessed by the Institute of Naval Medicine. A function of the assessment is to establish if thermoregulation is affected by an underlying metabolic, biochemical or physiological disorder.

Exertional Heat Illness encompasses a spectrum of disorders deriving from the combined stresses of exertion and thermoregulation^48^. Such disorders may include heat related dehydration, stroke, cramps, exhaustion, injury, stress, syncope, rhabdomyolysis, acute renal failure and hyponatremia^49,50^. Individuals across the whole spectrum may be assessed. The assessment usually takes place a minimum of 6 calendar weeks following the original episode, or following full biochemical recovery. Individuals also undergo a limited guided and supervised return to physical activity prior to the assessment.

The heat tolerance assessment is conducted in an environmentally controlled climatic chamber (dry bulb temperature 34 °C (±0.25 °C), 40% (± 2%) relative humidity, Wet Bulb Globe Temperature (WBGT) index 27 °C, and wind speed 7 km·h^−1^). Patients first undergo a full medical examination, including cardiac screening and 12 lead electrocardiogram. Anthropometric measurements, height weight, estimation of body surface area, lean mass and fat mass are taken. This is followed by a maximal oxygen uptake assessment (VO_2_ max) with electrocardogram telemetry. This exercise test is conducted wearing shorts and a T-shirt on a treadmill in the climatic conditions described. Following this assessment patients are allowed an hours rest in a cool room to allow body temperature to return to normal. They are also allowed to drink ad libitum.

Prior to undergoing the full heat tolerance assessment, patients are weighed nude, they self insert a rectal thermistor to a depth of 10 cm, are instrumented with further skin thermistors for the measurement of skin temperature^51^, electrocardiogram telemetry and heart rate monitors. They then dress in combat trousers, T-shirt and combat jacket. They reenter the environmental chamber and are asked to march on the treadmill, carrying a weighted rucksack at a speed and gradient that is adjusted until they are working at 60% of their VO_2_ max. Steady state VO_2_ measurements are taken at regular intervals throughout the assessment. The duration of the assessment is a minimum of 60 min but may proceed to a maximum of 90 min. After 30 min of marching with the load, subjects remove the rucksack and jacket and then continue walking. Patients who are able to rapidly attenuate the rate of rise of their deep body temperatures and demonstrate the ability to achieve plateau in their core temperature are considered to be able to demonstrate physiological thermoregulation and the attainment of thermal equilibrium. Those who cannot do so, or those whose deep body temperatures rise rapidly towards the safety threshold of 40 °C are considered heat intolerant. Other measures such as sweat rate, which can be shown for body surface area, VO_2_ consumption, heart rate, rating of perceived exertion are made and occasionally blood sampling for biochemical analysis is considered if clinically indicated.

Patients who have an uneventful heat tolerance assessment are passed as fit to return to fitness and exercise training and are eventually returned to full military activities. Those who are thermally intolerant of exercise in the proscribed conditions remain in a protected medical category. They are retested at intervals of no less than 6 weeks. Persistent heat intolerance under these conditions, especially when this is combined with evidence of exertional rhabdomyolyis or elevations of muscle markers after the assessment and the absence of other medical conditions may be referred to The Malignant Hyperthermia Investigation Unit at the University of Leeds, United Kingdom.

## Appendix 2. Primers for Sanger Sequencing

### RYR1 Exon 91

Fragment 1

Fragment 2

Fragment 3

### RYR1 Exon 102

## Appendix 3. Interpretation of the Results of a Malignant Hyperthermia Sample in which Five Missense Variants in RYR1 Were Found

In one MH sample, five missense variants in the gene coding for RYR1 were found (p.1571I>V, p.2060G>C, p.3366R>H, p.3933Y>C, p.4010E>K). Histopathology showed nonspecific myopathic changes in the skeletal muscle of our patient, but serum creatine kinase concentration was normal. The variants p.1571I>V and p.3933Y>C in *cis* were first described in one family with four missense changes by Tammaro *et al*.^26^, while we found p.1571I>V in one United Kingdom family as a single variant (unpublished sequence data, 2013, Hopkins PM, Leeds, United Kingdom). The Italian family members carrying these two variants were MH negative but the patient who inherited an additional variant from her father (p.3903R>Q) was diagnosed by IVCT as MH susceptible. The MH susceptible phenotype seemed to segregate with the p.3903R>Q variant but there were discordant members in this family who were MH negative while carrying the p.3903R>Q variant, which may suggest that more than one variant is necessary to develop MH susceptibility^26^. Duarte *et al*.^45^ also reported p.3366R>H and p.3933Y>C inherited in *cis* in a patient with a family history of serious adverse reactions to anesthetics. These variants were considered pathogenic as they were not detected in 150 control samples and involve with phylogenetically conserved residues. Since the *RYR1* variant p.2060G>C is polymorphic^23,14^ and the novel variant p.4010E>K shows a very low PolyPhen-2 score of 0.07, it could be suggested that a compound effect of the three other variants may contribute to the susceptibility of our patient. However a limited family study does not support this suggestion since p.4010E>K was the only variant clearly segregating with disease in this particular family. It may well be that variants in other genes contribute to, or are wholly responsible for, MH susceptibility in these cases.

## Footnotes

## References

1. Monnier N, Krivosic-Horber R, Payen JF, Kozak-Ribbens G, Nivoche Y, Adnet P, Reyford H, Lunardi J. Presence of two different genetic traits in malignant hyperthermia families: Implication for genetic analysis, diagnosis, and incidence of malignant hyperthermia susceptibility. Anesthesiology. 2002;97:1067–74. doi: 10.1097/00000542-200211000-00007.  [DOI](https://doi.org/10.1097/00000542-200211000-00007) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12411788/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Anesthesiology&title=Presence%20of%20two%20different%20genetic%20traits%20in%20malignant%20hyperthermia%20families:%20Implication%20for%20genetic%20analysis,%20diagnosis,%20and%20incidence%20of%20malignant%20hyperthermia%20susceptibility&author=N%20Monnier&author=R%20Krivosic-Horber&author=JF%20Payen&author=G%20Kozak-Ribbens&author=Y%20Nivoche&volume=97&publication_year=2002&pages=1067-74&pmid=12411788&doi=10.1097/00000542-200211000-00007&)

2. Brady JE, Sun LS, Rosenberg H, Li G. Prevalence of malignant hyperthermia due to anesthesia in New York State, 2001–2005. Anesth Analg. 2009;109:1162–6. doi: 10.1213/ane.0b013e3181ac1548.  [DOI](https://doi.org/10.1213/ane.0b013e3181ac1548) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19762744/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Anesth%20Analg&title=Prevalence%20of%20malignant%20hyperthermia%20due%20to%20anesthesia%20in%20New%20York%20State,%202001%E2%80%932005&author=JE%20Brady&author=LS%20Sun&author=H%20Rosenberg&author=G%20Li&volume=109&publication_year=2009&pages=1162-6&pmid=19762744&doi=10.1213/ane.0b013e3181ac1548&)

3. Ording H, Brancadoro V, Cozzolino S, Ellis FR, Glauber V, Gonano EF, Halsall PJ, Hartung E, Heffron JJ, Heytens L, Kozak-Ribbens G, Kress H, Krivosic-Horber R, Lehmann-Horn F, Mortier W, Nivoche Y, Ranklev-Twetman E, Sigurdsson S, Snoeck M, Stieglitz P, Tegazzin V, Urwyler A, Wappler F. In vitro contracture test for diagnosis of malignant hyperthermia following the protocol of the European MH Group: Results of testing patients surviving fulminant MH and unrelated low-risk subjects. The European Malignant Hyperthermia Group. Acta Anaesthesiol Scand. 1997;41:955–66. doi: 10.1111/j.1399-6576.1997.tb04820.x.  [DOI](https://doi.org/10.1111/j.1399-6576.1997.tb04820.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9311391/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Acta%20Anaesthesiol%20Scand&title=In%20vitro%20contracture%20test%20for%20diagnosis%20of%20malignant%20hyperthermia%20following%20the%20protocol%20of%20the%20European%20MH%20Group:%20Results%20of%20testing%20patients%20surviving%20fulminant%20MH%20and%20unrelated%20low-risk%20subjects.%20The%20European%20Malignant%20Hyperthermia%20Group&author=H%20Ording&author=V%20Brancadoro&author=S%20Cozzolino&author=FR%20Ellis&author=V%20Glauber&volume=41&publication_year=1997&pages=955-66&pmid=9311391&doi=10.1111/j.1399-6576.1997.tb04820.x&)

4. Feng W, Barrientos GC, Cherednichenko G, Yang T, Padilla IT, Truong K, Allen PD, Lopez JR, Pessah IN. Functional and biochemical properties of ryanodine receptor type 1 channels from heterozygous R163C malignant hyperthermia-susceptible mice. Mol Pharmacol. 2011;79:420–31. doi: 10.1124/mol.110.067959.  [DOI](https://doi.org/10.1124/mol.110.067959) | [PMC free article](/articles/PMC3061367/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21156754/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Pharmacol&title=Functional%20and%20biochemical%20properties%20of%20ryanodine%20receptor%20type%201%20channels%20from%20heterozygous%20R163C%20malignant%20hyperthermia-susceptible%20mice&author=W%20Feng&author=GC%20Barrientos&author=G%20Cherednichenko&author=T%20Yang&author=IT%20Padilla&volume=79&publication_year=2011&pages=420-31&pmid=21156754&doi=10.1124/mol.110.067959&)

5. Protasi F, Paolini C, Nakai J, Beam KG, Franzini-Armstrong C, Allen PD. Multiple regions of RyR1 mediate functional and structural interactions with alpha(1S)-dihydropyridine receptors in skeletal muscle. Biophys J. 2002;83:3230–44. doi: 10.1016/S0006-3495(02)75325-3.  [DOI](https://doi.org/10.1016/S0006-3495(02)75325-3) | [PMC free article](/articles/PMC1302400/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12496092/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biophys%20J&title=Multiple%20regions%20of%20RyR1%20mediate%20functional%20and%20structural%20interactions%20with%20alpha(1S)-dihydropyridine%20receptors%20in%20skeletal%20muscle&author=F%20Protasi&author=C%20Paolini&author=J%20Nakai&author=KG%20Beam&author=C%20Franzini-Armstrong&volume=83&publication_year=2002&pages=3230-44&pmid=12496092&doi=10.1016/S0006-3495(02)75325-3&)

6. Bendahan D, Kozak-Ribbens G, Confort-Gouny S, Ghattas B, Figarella-Branger D, Aubert M, Cozzone PJ. A noninvasive investigation of muscle energetics supports similarities between exertional heat stroke and malignant hyperthermia. Anesth Analg. 2001;93:683–9. doi: 10.1097/00000539-200109000-00030.  [DOI](https://doi.org/10.1097/00000539-200109000-00030) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11524341/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Anesth%20Analg&title=A%20noninvasive%20investigation%20of%20muscle%20energetics%20supports%20similarities%20between%20exertional%20heat%20stroke%20and%20malignant%20hyperthermia&author=D%20Bendahan&author=G%20Kozak-Ribbens&author=S%20Confort-Gouny&author=B%20Ghattas&author=D%20Figarella-Branger&volume=93&publication_year=2001&pages=683-9&pmid=11524341&doi=10.1097/00000539-200109000-00030&)

7. Carpenter D, Ringrose C, Leo V, Morris A, Robinson RL, Halsall PJ, Hopkins PM, Shaw MA. The role of CACNA1S in predisposition to malignant hyperthermia. BMC Med Genet. 2009;10:104. doi: 10.1186/1471-2350-10-104.  [DOI](https://doi.org/10.1186/1471-2350-10-104) | [PMC free article](/articles/PMC2770053/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19825159/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Med%20Genet&title=The%20role%20of%20CACNA1S%20in%20predisposition%20to%20malignant%20hyperthermia&author=D%20Carpenter&author=C%20Ringrose&author=V%20Leo&author=A%20Morris&author=RL%20Robinson&volume=10&publication_year=2009&pages=104&pmid=19825159&doi=10.1186/1471-2350-10-104&)

8. Monnier N, Procaccio V, Stieglitz P, Lunardi J. Malignant-hyperthermia susceptibility is associated with a mutation of the alpha 1-subunit of the human dihydropyridine-sensitive L-type voltage-dependent calcium-channel receptor in skeletal muscle. Am J Hum Genet. 1997;60:1316–25. doi: 10.1086/515454.  [DOI](https://doi.org/10.1086/515454) | [PMC free article](/articles/PMC1716149/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9199552/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=Malignant-hyperthermia%20susceptibility%20is%20associated%20with%20a%20mutation%20of%20the%20alpha%201-subunit%20of%20the%20human%20dihydropyridine-sensitive%20L-type%20voltage-dependent%20calcium-channel%20receptor%20in%20skeletal%20muscle&author=N%20Monnier&author=V%20Procaccio&author=P%20Stieglitz&author=J%20Lunardi&volume=60&publication_year=1997&pages=1316-25&pmid=9199552&doi=10.1086/515454&)

9. Stewart SL, Hogan K, Rosenberg H, Fletcher JE. Identification of the Arg1086His mutation in the alpha subunit of the voltage-dependent calcium channel (CACNA1S) in a North American family with malignant hyperthermia. Clin Genet. 2001;59:178–84. doi: 10.1034/j.1399-0004.2001.590306.x.  [DOI](https://doi.org/10.1034/j.1399-0004.2001.590306.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11260227/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Genet&title=Identification%20of%20the%20Arg1086His%20mutation%20in%20the%20alpha%20subunit%20of%20the%20voltage-dependent%20calcium%20channel%20(CACNA1S)%20in%20a%20North%20American%20family%20with%20malignant%20hyperthermia&author=SL%20Stewart&author=K%20Hogan&author=H%20Rosenberg&author=JE%20Fletcher&volume=59&publication_year=2001&pages=178-84&pmid=11260227&doi=10.1034/j.1399-0004.2001.590306.x&)

10. Brandom BW, Bina S, Wong CA, Wallace T, Visoiu M, Isackson PJ, Vladutiu GD, Sambuughin N, Muldoon SM. Ryanodine receptor type 1 gene variants in the malignant hyperthermia-susceptible population of the United States. Anesth Analg. 2013;116:1078–86. doi: 10.1213/ANE.0b013e31828a71ff.  [DOI](https://doi.org/10.1213/ANE.0b013e31828a71ff) | [PMC free article](/articles/PMC3633164/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23558838/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Anesth%20Analg&title=Ryanodine%20receptor%20type%201%20gene%20variants%20in%20the%20malignant%20hyperthermia-susceptible%20population%20of%20the%20United%20States&author=BW%20Brandom&author=S%20Bina&author=CA%20Wong&author=T%20Wallace&author=M%20Visoiu&volume=116&publication_year=2013&pages=1078-86&pmid=23558838&doi=10.1213/ANE.0b013e31828a71ff&)

11. Robinson RL, Anetseder MJ, Brancadoro V, van Broekhoven C, Carsana A, Censier K, Fortunato G, Girard T, Heytens L, Hopkins PM, Jurkat-Rott K, Klinger W, Kozak-Ribbens G, Krivosic R, Monnier N, Nivoche Y, Olthoff D, Rueffert H, Sorrentino V, Tegazzin V, Mueller CR. Recent advances in the diagnosis of malignant hyperthermia susceptibility: How confident can we be of genetic testing? Eur J Hum Genet. 2003;11:342–8. doi: 10.1038/sj.ejhg.5200964.  [DOI](https://doi.org/10.1038/sj.ejhg.5200964) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12700608/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Hum%20Genet&title=Recent%20advances%20in%20the%20diagnosis%20of%20malignant%20hyperthermia%20susceptibility:%20How%20confident%20can%20we%20be%20of%20genetic%20testing?&author=RL%20Robinson&author=MJ%20Anetseder&author=V%20Brancadoro&author=C%20van%20Broekhoven&author=A%20Carsana&volume=11&publication_year=2003&pages=342-8&pmid=12700608&doi=10.1038/sj.ejhg.5200964&)

12. Weiss RG, O’Connell KM, Flucher BE, Allen PD, Grabner M, Dirksen RT. Functional analysis of the R1086H malignant hyperthermia mutation in the DHPR reveals an unexpected influence of the III-IV loop on skeletal muscle EC coupling. Am J Physiol Cell Physiol. 2004;287:C1094–102. doi: 10.1152/ajpcell.00173.2004.  [DOI](https://doi.org/10.1152/ajpcell.00173.2004) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15201141/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Physiol%20Cell%20Physiol&title=Functional%20analysis%20of%20the%20R1086H%20malignant%20hyperthermia%20mutation%20in%20the%20DHPR%20reveals%20an%20unexpected%20influence%20of%20the%20III-IV%20loop%20on%20skeletal%20muscle%20EC%20coupling&author=RG%20Weiss&author=KM%20O%E2%80%99Connell&author=BE%20Flucher&author=PD%20Allen&author=M%20Grabner&volume=287&publication_year=2004&pages=C1094-102&pmid=15201141&doi=10.1152/ajpcell.00173.2004&)

13. Eltit JM, Bannister RA, Moua O, Altamirano F, Hopkins PM, Pessah IN, Molinski TF, Lopez JR, Beam KG, Allen PD. Malignant hyperthermia susceptibility arising from altered resting coupling between the skeletal muscle L-type Ca2+ channel and the type 1 ryanodine receptor. Proc Natl Acad Sci U S A. 2012;109:7923–8. doi: 10.1073/pnas.1119207109.  [DOI](https://doi.org/10.1073/pnas.1119207109) | [PMC free article](/articles/PMC3356662/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22547813/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&title=Malignant%20hyperthermia%20susceptibility%20arising%20from%20altered%20resting%20coupling%20between%20the%20skeletal%20muscle%20L-type%20Ca2+%20channel%20and%20the%20type%201%20ryanodine%20receptor&author=JM%20Eltit&author=RA%20Bannister&author=O%20Moua&author=F%20Altamirano&author=PM%20Hopkins&volume=109&publication_year=2012&pages=7923-8&pmid=22547813&doi=10.1073/pnas.1119207109&)

14. Robinson R, Carpenter D, Shaw MA, Halsall J, Hopkins P. Mutations in RYR1 in malignant hyperthermia and central core disease. Hum Mutat. 2006;27:977–89. doi: 10.1002/humu.20356.  [DOI](https://doi.org/10.1002/humu.20356) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16917943/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mutat&title=Mutations%20in%20RYR1%20in%20malignant%20hyperthermia%20and%20central%20core%20disease&author=R%20Robinson&author=D%20Carpenter&author=MA%20Shaw&author=J%20Halsall&author=P%20Hopkins&volume=27&publication_year=2006&pages=977-89&pmid=16917943&doi=10.1002/humu.20356&)

15. Kim JH, Jarvik GP, Browning BL, Rajagopalan R, Gordon AS, Rieder MJ, Robertson PD, Nickerson DA, Fisher NA, Hopkins PM. Exome sequencing reveals novel rare variants in the ryanodine receptor and calcium channel genes in malignant hyperthermia families. Anesthesiology. 2013;119:1054–65. doi: 10.1097/ALN.0b013e3182a8a998.  [DOI](https://doi.org/10.1097/ALN.0b013e3182a8a998) | [PMC free article](/articles/PMC4115638/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24013571/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Anesthesiology&title=Exome%20sequencing%20reveals%20novel%20rare%20variants%20in%20the%20ryanodine%20receptor%20and%20calcium%20channel%20genes%20in%20malignant%20hyperthermia%20families&author=JH%20Kim&author=GP%20Jarvik&author=BL%20Browning&author=R%20Rajagopalan&author=AS%20Gordon&volume=119&publication_year=2013&pages=1054-65&pmid=24013571&doi=10.1097/ALN.0b013e3182a8a998&)

16. Carr IM, Morgan JE, Diggle CP, Sheridan E, Markham AF, Logan CV, Inglehearn CF, Taylor GR, Bonthron DT. Illuminator, a desktop program for mutation detection using short-read clonal sequencing. Genomics. 2011;98:302–9. doi: 10.1016/j.ygeno.2011.05.004.  [DOI](https://doi.org/10.1016/j.ygeno.2011.05.004) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21621601/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genomics&title=Illuminator,%20a%20desktop%20program%20for%20mutation%20detection%20using%20short-read%20clonal%20sequencing&author=IM%20Carr&author=JE%20Morgan&author=CP%20Diggle&author=E%20Sheridan&author=AF%20Markham&volume=98&publication_year=2011&pages=302-9&pmid=21621601&doi=10.1016/j.ygeno.2011.05.004&)

17. Adzhubei IA, Schmidt S, Peshkin L, Ramensky VE, Gerasimova A, Bork P, Kondrashov AS, Sunyaev SR. A method and server for predicting damaging missense mutations. Nat Methods. 2010;7:248–9. doi: 10.1038/nmeth0410-248.  [DOI](https://doi.org/10.1038/nmeth0410-248) | [PMC free article](/articles/PMC2855889/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20354512/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Methods&title=A%20method%20and%20server%20for%20predicting%20damaging%20missense%20mutations&author=IA%20Adzhubei&author=S%20Schmidt&author=L%20Peshkin&author=VE%20Ramensky&author=A%20Gerasimova&volume=7&publication_year=2010&pages=248-9&pmid=20354512&doi=10.1038/nmeth0410-248&)

18. Kircher M, Witten DW, Jain P, O’Roak BJ, Cooper GM, Shendure J. A general framework for estimating the relative pathogenicity of human genetic variants. Nat Genet. 2014;46:310–5. doi: 10.1038/ng.2892.  [DOI](https://doi.org/10.1038/ng.2892) | [PMC free article](/articles/PMC3992975/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24487276/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=A%20general%20framework%20for%20estimating%20the%20relative%20pathogenicity%20of%20human%20genetic%20variants&author=M%20Kircher&author=DW%20Witten&author=P%20Jain&author=BJ%20O%E2%80%99Roak&author=GM%20Cooper&volume=46&publication_year=2014&pages=310-5&pmid=24487276&doi=10.1038/ng.2892&)

19. Gillard EF, Otsu K, Fujii J, Duff C, de Leon S, Khanna VK, Britt BA, Worton RG, MacLennan DH. Polymorphisms and deduced amino acid substitutions in the coding sequence of the ryanodine receptor (RYR1) gene in individuals with malignant hyperthermia. Genomics. 1992;13:1247–54. doi: 10.1016/0888-7543(92)90042-q.  [DOI](https://doi.org/10.1016/0888-7543(92)90042-q) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/1354642/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genomics&title=Polymorphisms%20and%20deduced%20amino%20acid%20substitutions%20in%20the%20coding%20sequence%20of%20the%20ryanodine%20receptor%20(RYR1)%20gene%20in%20individuals%20with%20malignant%20hyperthermia&author=EF%20Gillard&author=K%20Otsu&author=J%20Fujii&author=C%20Duff&author=S%20de%20Leon&volume=13&publication_year=1992&pages=1247-54&pmid=1354642&doi=10.1016/0888-7543(92)90042-q&)

20. Schiemann AH, Durholt EM, Pollock N, Stowell KM. Sequence capture and massively parallel sequencing to detect mutations associated with malignant hyperthermia. Br J Anaesth. 2013;110:122–7. doi: 10.1093/bja/aes341.  [DOI](https://doi.org/10.1093/bja/aes341) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23035052/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Anaesth&title=Sequence%20capture%20and%20massively%20parallel%20sequencing%20to%20detect%20mutations%20associated%20with%20malignant%20hyperthermia&author=AH%20Schiemann&author=EM%20Durholt&author=N%20Pollock&author=KM%20Stowell&volume=110&publication_year=2013&pages=122-7&pmid=23035052&doi=10.1093/bja/aes341&)

21. Monnier N, Kozak-Ribbens G, Krivosic-Horber R, Nivoche Y, Qi D, Kraev N, Loke J, Sharma P, Tegazzin V, Figarella-Branger D, Romero N, Mezin P, Bendahan D, Payen JF, Depret T, Maclennan DH, Lunardi J. Correlations between genotype and pharmacological, histological, functional, and clinical phenotypes in malignant hyperthermia susceptibility. Hum Mutat. 2005;26:413–25. doi: 10.1002/humu.20231.  [DOI](https://doi.org/10.1002/humu.20231) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16163667/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mutat&title=Correlations%20between%20genotype%20and%20pharmacological,%20histological,%20functional,%20and%20clinical%20phenotypes%20in%20malignant%20hyperthermia%20susceptibility&author=N%20Monnier&author=G%20Kozak-Ribbens&author=R%20Krivosic-Horber&author=Y%20Nivoche&author=D%20Qi&volume=26&publication_year=2005&pages=413-25&pmid=16163667&doi=10.1002/humu.20231&)

22. Keating KE, Giblin L, Lynch PJ, Quane KA, Lehane M, Heffron JJ, McCarthy TV. Detection of a novel mutation in the ryanodine receptor gene in an Irish malignant hyperthermia pedigree: Correlation of the IVCT response with the affected and unaffected haplotypes. J Med Genet. 1997;34:291–6. doi: 10.1136/jmg.34.4.291.  [DOI](https://doi.org/10.1136/jmg.34.4.291) | [PMC free article](/articles/PMC1050914/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9138151/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Med%20Genet&title=Detection%20of%20a%20novel%20mutation%20in%20the%20ryanodine%20receptor%20gene%20in%20an%20Irish%20malignant%20hyperthermia%20pedigree:%20Correlation%20of%20the%20IVCT%20response%20with%20the%20affected%20and%20unaffected%20haplotypes&author=KE%20Keating&author=L%20Giblin&author=PJ%20Lynch&author=KA%20Quane&author=M%20Lehane&volume=34&publication_year=1997&pages=291-6&pmid=9138151&doi=10.1136/jmg.34.4.291&)

23. Gillard EF, Otsu K, Fujii J, Khanna VK, de Leon S, Derdemezi J, Britt BA, Duff CL, Worton RG, MacLennan DH. A substitution of cysteine for arginine 614 in the ryanodine receptor is potentially causative of human malignant hyperthermia. Genomics. 1991;11:751–5. doi: 10.1016/0888-7543(91)90084-r.  [DOI](https://doi.org/10.1016/0888-7543(91)90084-r) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/1774074/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genomics&title=A%20substitution%20of%20cysteine%20for%20arginine%20614%20in%20the%20ryanodine%20receptor%20is%20potentially%20causative%20of%20human%20malignant%20hyperthermia&author=EF%20Gillard&author=K%20Otsu&author=J%20Fujii&author=VK%20Khanna&author=S%20de%20Leon&volume=11&publication_year=1991&pages=751-5&pmid=1774074&doi=10.1016/0888-7543(91)90084-r&)

24. Galli L, Orrico A, Lorenzini S, Censini S, Falciani M, Covacci A, Tegazzin V, Sorrentino V. Frequency and localization of mutations in the 106 exons of the RYR1 gene in 50 individuals with malignant hyperthermia. Hum Mutat. 2006;27:830. doi: 10.1002/humu.9442.  [DOI](https://doi.org/10.1002/humu.9442) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16835904/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mutat&title=Frequency%20and%20localization%20of%20mutations%20in%20the%20106%20exons%20of%20the%20RYR1%20gene%20in%2050%20individuals%20with%20malignant%20hyperthermia&author=L%20Galli&author=A%20Orrico&author=S%20Lorenzini&author=S%20Censini&author=M%20Falciani&volume=27&publication_year=2006&pages=830&pmid=16835904&doi=10.1002/humu.9442&)

25. Levano S, Vukcevic M, Singer M, Matter A, Treves S, Urwyler A, Girard T. Increasing the number of diagnostic mutations in malignant hyperthermia. Hum Mutat. 2009;30:590–8. doi: 10.1002/humu.20878.  [DOI](https://doi.org/10.1002/humu.20878) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19191329/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mutat&title=Increasing%20the%20number%20of%20diagnostic%20mutations%20in%20malignant%20hyperthermia&author=S%20Levano&author=M%20Vukcevic&author=M%20Singer&author=A%20Matter&author=S%20Treves&volume=30&publication_year=2009&pages=590-8&pmid=19191329&doi=10.1002/humu.20878&)

26. Tammaro A, Di Martino A, Bracco A, Cozzolino S, Savoia G, Andria B, Cannavo A, Spagnuolo M, Piluso G, Aurino S, Nigro V. Novel missense mutations and unexpected multiple changes of RYR1 gene in 75 malignant hyperthermia families. Clin Genet. 2011;79:438–47. doi: 10.1111/j.1399-0004.2010.01493.x.  [DOI](https://doi.org/10.1111/j.1399-0004.2010.01493.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20681998/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Genet&title=Novel%20missense%20mutations%20and%20unexpected%20multiple%20changes%20of%20RYR1%20gene%20in%2075%20malignant%20hyperthermia%20families&author=A%20Tammaro&author=A%20Di%20Martino&author=A%20Bracco&author=S%20Cozzolino&author=G%20Savoia&volume=79&publication_year=2011&pages=438-47&pmid=20681998&doi=10.1111/j.1399-0004.2010.01493.x&)

27. Jeong SK, Kim DC, Cho YG, Sunwoo IN, Kim DS. A double mutation of the ryanodine receptor type 1 gene in a malignant hyperthermia family with multiminicore myopathy. J Clin Neurol. 2008;4:123–30. doi: 10.3988/jcn.2008.4.3.123.  [DOI](https://doi.org/10.3988/jcn.2008.4.3.123) | [PMC free article](/articles/PMC2686874/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19513315/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Neurol&title=A%20double%20mutation%20of%20the%20ryanodine%20receptor%20type%201%20gene%20in%20a%20malignant%20hyperthermia%20family%20with%20multiminicore%20myopathy&author=SK%20Jeong&author=DC%20Kim&author=YG%20Cho&author=IN%20Sunwoo&author=DS%20Kim&volume=4&publication_year=2008&pages=123-30&pmid=19513315&doi=10.3988/jcn.2008.4.3.123&)

28. Abecasis GR, Auton A, Brooks LD, DePristo MA, Durbin RM, Handsaker RE, Kang HM, Marth GT, McVean GA 1000 Genomes Project Consortium. An integrated map of genetic variation from 1,092 human genomes. Nature. 2012;491:56–65. doi: 10.1038/nature11632.  [DOI](https://doi.org/10.1038/nature11632) | [PMC free article](/articles/PMC3498066/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23128226/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=An%20integrated%20map%20of%20genetic%20variation%20from%201,092%20human%20genomes&author=GR%20Abecasis&author=A%20Auton&author=LD%20Brooks&author=MA%20DePristo&author=RM%20Durbin&volume=491&publication_year=2012&pages=56-65&pmid=23128226&doi=10.1038/nature11632&)

29. Morgan JE, Carr IM, Sheridan E, Chu CE, Hayward B, Camm N, Lindsay HA, Mattocks CJ, Markham AF, Bonthron DT, Taylor GR. Genetic diagnosis of familial breast cancer using clonal sequencing. Hum Mutat. 2010;31:484–91. doi: 10.1002/humu.21216.  [DOI](https://doi.org/10.1002/humu.21216) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20127978/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mutat&title=Genetic%20diagnosis%20of%20familial%20breast%20cancer%20using%20clonal%20sequencing&author=JE%20Morgan&author=IM%20Carr&author=E%20Sheridan&author=CE%20Chu&author=B%20Hayward&volume=31&publication_year=2010&pages=484-91&pmid=20127978&doi=10.1002/humu.21216&)

30. Ozcelik H, Shi X, Chang MC, Tram E, Vlasschaert M, Di Nicola N, Kiselova A, Yee D, Goldman A, Dowar M, Sukhu B, Kandel R, Siminovitch K. Long-range PCR and next-generation sequencing of BRCA1 and BRCA2 in breast cancer. J Mol Diagn. 2012;14:467–75. doi: 10.1016/j.jmoldx.2012.03.006.  [DOI](https://doi.org/10.1016/j.jmoldx.2012.03.006) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22874498/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Mol%20Diagn&title=Long-range%20PCR%20and%20next-generation%20sequencing%20of%20BRCA1%20and%20BRCA2%20in%20breast%20cancer&author=H%20Ozcelik&author=X%20Shi&author=MC%20Chang&author=E%20Tram&author=M%20Vlasschaert&volume=14&publication_year=2012&pages=467-75&pmid=22874498&doi=10.1016/j.jmoldx.2012.03.006&)

31. Richards CS, Bale S, Bellissimo DB, Das S, Grody WW, Hegde MR, Lyon E, Ward BE Molecular Subcommittee of the ACMG Laboratory Quality Assurance Committee. ACMG recommendations for standards for interpretation and reporting of sequence variations: Revisions 2007. Genet Med. 2008;10:294–300. doi: 10.1097/GIM.0b013e31816b5cae.  [DOI](https://doi.org/10.1097/GIM.0b013e31816b5cae) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18414213/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genet%20Med&title=ACMG%20recommendations%20for%20standards%20for%20interpretation%20and%20reporting%20of%20sequence%20variations:%20Revisions%202007&author=CS%20Richards&author=S%20Bale&author=DB%20Bellissimo&author=S%20Das&author=WW%20Grody&volume=10&publication_year=2008&pages=294-300&pmid=18414213&doi=10.1097/GIM.0b013e31816b5cae&)

32. Urwyler A, Deufel T, McCarthy T, West S European Malignant Hyperthermia Group. Guidelines for molecular genetic detection of susceptibility to malignant hyperthermia. Br J Anaesth. 2001;86:283–7. doi: 10.1093/bja/86.2.283.  [DOI](https://doi.org/10.1093/bja/86.2.283) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11573677/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Anaesth&title=Guidelines%20for%20molecular%20genetic%20detection%20of%20susceptibility%20to%20malignant%20hyperthermia&author=A%20Urwyler&author=T%20Deufel&author=T%20McCarthy&author=S%20West&volume=86&publication_year=2001&pages=283-7&pmid=11573677&doi=10.1093/bja/86.2.283&)

33. Hopkins PM, Ellis FR, Halsall PJ. Evidence for related myopathies in exertional heat stroke and malignant hyperthermia. Lancet. 1991;338:1491–2. doi: 10.1016/0140-6736(91)92304-k.  [DOI](https://doi.org/10.1016/0140-6736(91)92304-k) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/1683922/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Evidence%20for%20related%20myopathies%20in%20exertional%20heat%20stroke%20and%20malignant%20hyperthermia&author=PM%20Hopkins&author=FR%20Ellis&author=PJ%20Halsall&volume=338&publication_year=1991&pages=1491-2&pmid=1683922&doi=10.1016/0140-6736(91)92304-k&)

34. Carpenter D, Robinson RL, Quinnell RJ, Ringrose C, Hogg M, Casson F, Booms P, Iles DE, Halsall PJ, Steele DS, Shaw MA, Hopkins PM. Genetic variation in RYR1 and malignant hyperthermia phenotypes. Br J Anaesth. 2009;103:538–48. doi: 10.1093/bja/aep204.  [DOI](https://doi.org/10.1093/bja/aep204) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19648156/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Anaesth&title=Genetic%20variation%20in%20RYR1%20and%20malignant%20hyperthermia%20phenotypes&author=D%20Carpenter&author=RL%20Robinson&author=RJ%20Quinnell&author=C%20Ringrose&author=M%20Hogg&volume=103&publication_year=2009&pages=538-48&pmid=19648156&doi=10.1093/bja/aep204&)

35. Grievink H, Stowell KM. Allele-specific differences in ryanodine receptor 1 mRNA expression levels may contribute to phenotypic variability in malignant hyperthermia. Orphanet J Rare Dis. 2010;5:10. doi: 10.1186/1750-1172-5-10.  [DOI](https://doi.org/10.1186/1750-1172-5-10) | [PMC free article](/articles/PMC2895584/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20482855/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Orphanet%20J%20Rare%20Dis&title=Allele-specific%20differences%20in%20ryanodine%20receptor%201%20mRNA%20expression%20levels%20may%20contribute%20to%20phenotypic%20variability%20in%20malignant%20hyperthermia&author=H%20Grievink&author=KM%20Stowell&volume=5&publication_year=2010&pages=10&pmid=20482855&doi=10.1186/1750-1172-5-10&)

36. Adeokun AM, West SP, Ellis FR, Halsall PJ, Hopkins PM, Foroughmand AM, Iles DE, Robinson RL, Stewart AD, Curran JL. The G1021A substitution in the RYR1 gene does not cosegregate with malignant hyperthermia susceptibility in a British pedigree. Am J Hum Genet. 1997;60:833–41.  [PMC free article](/articles/PMC1712455/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9106529/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=The%20G1021A%20substitution%20in%20the%20RYR1%20gene%20does%20not%20cosegregate%20with%20malignant%20hyperthermia%20susceptibility%20in%20a%20British%20pedigree&author=AM%20Adeokun&author=SP%20West&author=FR%20Ellis&author=PJ%20Halsall&author=PM%20Hopkins&volume=60&publication_year=1997&pages=833-41&pmid=9106529&)

37. Heytens L. Molecular genetic detection of susceptibility to malignant hyperthermia in Belgian families. Acta Anaesthesiol Belg. 2007;58:113–8.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/17710899/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Acta%20Anaesthesiol%20Belg&title=Molecular%20genetic%20detection%20of%20susceptibility%20to%20malignant%20hyperthermia%20in%20Belgian%20families&author=L%20Heytens&volume=58&publication_year=2007&pages=113-8&pmid=17710899&)

38. Fortunato G, Carsana A, Tinto N, Brancadoro V, Canfora G, Salvatore F. A case of discordance between genotype and phenotype in a malignant hyperthermia family. Eur J Hum Genet. 1999;7:415–20. doi: 10.1038/sj.ejhg.5200314.  [DOI](https://doi.org/10.1038/sj.ejhg.5200314) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10352931/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Hum%20Genet&title=A%20case%20of%20discordance%20between%20genotype%20and%20phenotype%20in%20a%20malignant%20hyperthermia%20family&author=G%20Fortunato&author=A%20Carsana&author=N%20Tinto&author=V%20Brancadoro&author=G%20Canfora&volume=7&publication_year=1999&pages=415-20&pmid=10352931&doi=10.1038/sj.ejhg.5200314&)

39. Robinson RL, Curran JL, Ellis FR, Halsall PJ, Hall WJ, Hopkins PM, Iles DE, West SP, Shaw MA. Multiple interacting gene products may influence susceptibility to malignant hyperthermia. Ann Hum Genet. 2000;64:307–20. doi: 10.1017/S0003480000008186.  [DOI](https://doi.org/10.1017/S0003480000008186) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11415515/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Hum%20Genet&title=Multiple%20interacting%20gene%20products%20may%20influence%20susceptibility%20to%20malignant%20hyperthermia&author=RL%20Robinson&author=JL%20Curran&author=FR%20Ellis&author=PJ%20Halsall&author=WJ%20Hall&volume=64&publication_year=2000&pages=307-20&pmid=11415515&doi=10.1017/S0003480000008186&)

40. Robinson R, Hopkins P, Carsana A, Gilly H, Halsall J, Heytens L, Islander G, Jurkat-Rott K, Muller C, Shaw MA. Several interacting genes influence the malignant hyperthermia phenotype. Hum Genet. 2003;112:217–8. doi: 10.1007/s00439-002-0864-6.  [DOI](https://doi.org/10.1007/s00439-002-0864-6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12522565/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Genet&title=Several%20interacting%20genes%20influence%20the%20malignant%20hyperthermia%20phenotype&author=R%20Robinson&author=P%20Hopkins&author=A%20Carsana&author=H%20Gilly&author=J%20Halsall&volume=112&publication_year=2003&pages=217-8&pmid=12522565&doi=10.1007/s00439-002-0864-6&)

41. Gonsalves SG, Ng D, Johnston JJ, Teer JK, Stenson PD, Cooper DN, Mullikin JC, Biesecker LG, Program NCS. Using exome data to identify malignant hyperthermia susceptibility mutations. Anesthesiology. 2013;119:1043–53. doi: 10.1097/ALN.0b013e3182a8a8e7.  [DOI](https://doi.org/10.1097/ALN.0b013e3182a8a8e7) | [PMC free article](/articles/PMC4077354/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24195946/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Anesthesiology&title=Using%20exome%20data%20to%20identify%20malignant%20hyperthermia%20susceptibility%20mutations&author=SG%20Gonsalves&author=D%20Ng&author=JJ%20Johnston&author=JK%20Teer&author=PD%20Stenson&volume=119&publication_year=2013&pages=1043-53&pmid=24195946&doi=10.1097/ALN.0b013e3182a8a8e7&)

42. Tennessen JA, Bigham AW, O’Connor TD, Fu W, Kenny EE, Gravel S, McGee S, Do R, Liu X, Jun G, Kang HM, Jordan D, Leal SM, Gabriel S, Rieder MJ, Abecasis G, Altshuler D, Nickerson DA, Boerwinkle E, Sunyaev S, Bustamante CD, Bamshad MJ, Akey JM, Broad GO, Seattle GO, Project NES. Evolution and functional impact of rare coding variation from deep sequencing of human exomes. Science. 2012;337:64–9. doi: 10.1126/science.1219240.  [DOI](https://doi.org/10.1126/science.1219240) | [PMC free article](/articles/PMC3708544/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22604720/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=Evolution%20and%20functional%20impact%20of%20rare%20coding%20variation%20from%20deep%20sequencing%20of%20human%20exomes&author=JA%20Tennessen&author=AW%20Bigham&author=TD%20O%E2%80%99Connor&author=W%20Fu&author=EE%20Kenny&volume=337&publication_year=2012&pages=64-9&pmid=22604720&doi=10.1126/science.1219240&)

43. Casals F, Bertranpetit J. Genetics: Human genetic variation, shared and private. Science. 2012;337:39–40. doi: 10.1126/science.1224528.  [DOI](https://doi.org/10.1126/science.1224528) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22767915/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=Genetics:%20Human%20genetic%20variation,%20shared%20and%20private&author=F%20Casals&author=J%20Bertranpetit&volume=337&publication_year=2012&pages=39-40&pmid=22767915&doi=10.1126/science.1224528&)

44. Klein A, Lillis S, Munteanu I, Scoto M, Zhou H, Quinlivan R, Straub V, Manzur AY, Roper H, Jeannet PY, Rakowicz W, Jones DH, Jensen UB, Wraige E, Trump N, Schara U, Lochmuller H, Sarkozy A, Kingston H, Norwood F, Damian M, Kirschner J, Longman C, Roberts M, Auer-Grumbach M, Hughes I, Bushby K, Sewry C, Robb S, Abbs S, Jungbluth H, Muntoni F. Clinical and genetic findings in a large cohort of patients with ryanodine receptor 1 gene-associated myopathies. Hum Mutat. 2012;33:981–8. doi: 10.1002/humu.22056.  [DOI](https://doi.org/10.1002/humu.22056) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22473935/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mutat&title=Clinical%20and%20genetic%20findings%20in%20a%20large%20cohort%20of%20patients%20with%20ryanodine%20receptor%201%20gene-associated%20myopathies&author=A%20Klein&author=S%20Lillis&author=I%20Munteanu&author=M%20Scoto&author=H%20Zhou&volume=33&publication_year=2012&pages=981-8&pmid=22473935&doi=10.1002/humu.22056&)

45. Duarte ST, Oliveira J, Santos R, Pereira P, Barroso C, Conceicao I, Evangelista T. Dominant and recessive RYR1 mutations in adults with core lesions and mild muscle symptoms. Muscle Nerve. 2011;44:102–8. doi: 10.1002/mus.22009.  [DOI](https://doi.org/10.1002/mus.22009) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21674524/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Muscle%20Nerve&title=Dominant%20and%20recessive%20RYR1%20mutations%20in%20adults%20with%20core%20lesions%20and%20mild%20muscle%20symptoms&author=ST%20Duarte&author=J%20Oliveira&author=R%20Santos&author=P%20Pereira&author=C%20Barroso&volume=44&publication_year=2011&pages=102-8&pmid=21674524&doi=10.1002/mus.22009&)

46. Carpenter D, Ismail A, Robinson RL, Ringrose C, Booms P, Iles DE, Halsall PJ, Steele D, Shaw MA, Hopkins PM. A RYR1 mutation associated with recessive congenital myopathy and dominant malignant hyperthermia in Asian families. Muscle Nerve. 2009;40:633–9. doi: 10.1002/mus.21397.  [DOI](https://doi.org/10.1002/mus.21397) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19645060/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Muscle%20Nerve&title=A%20RYR1%20mutation%20associated%20with%20recessive%20congenital%20myopathy%20and%20dominant%20malignant%20hyperthermia%20in%20Asian%20families&author=D%20Carpenter&author=A%20Ismail&author=RL%20Robinson&author=C%20Ringrose&author=P%20Booms&volume=40&publication_year=2009&pages=633-9&pmid=19645060&doi=10.1002/mus.21397&)

47. JSP 539. Climatic Illness and Injury in the Armed Forces: Force Protection and Initial Medical Treatment. V2.4 Ministry of Defence; London: Aug, 2014.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Climatic%20Illness%20and%20Injury%20in%20the%20Armed%20Forces:%20Force%20Protection%20and%20Initial%20Medical%20Treatment&publication_year=2014&)

48. Gardner J, Kark J. Medical Aspects of Harsh Environments. Office of the Surgeon General’s Department of the Army; USA: 2003. Clinical Management, Diagnoses and Surveillance of Exertional Heat Illness.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Medical%20Aspects%20of%20Harsh%20Environments&author=J%20Gardner&author=J%20Kark&publication_year=2003&)

49. Bouchama A, Knochel J. Heat Stroke. New Engl J Med. 2004;346:1987–8. doi: 10.1056/NEJMra011089.  [DOI](https://doi.org/10.1056/NEJMra011089) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12075060/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=New%20Engl%20J%20Med&title=Heat%20Stroke&author=A%20Bouchama&author=J%20Knochel&volume=346&publication_year=2004&pages=1987-8&pmid=12075060&doi=10.1056/NEJMra011089&)

50. Defence Medical Services Department. Op TELIC. Heat Related Illness Audit. Ministry of Defence; London: 2003. DMSD/962/30/1, 1–11.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Op%20TELIC.%20Heat%20Related%20Illness%20Audit&publication_year=2003&)

51. Ramanathan N. A new weighting system for mean surface temperature of the human body. J Appl Physiol. 1964;19:531–3. doi: 10.1152/jappl.1964.19.3.531.  [DOI](https://doi.org/10.1152/jappl.1964.19.3.531) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14173555/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Appl%20Physiol&title=A%20new%20weighting%20system%20for%20mean%20surface%20temperature%20of%20the%20human%20body&author=N%20Ramanathan&volume=19&publication_year=1964&pages=531-3&pmid=14173555&doi=10.1152/jappl.1964.19.3.531&)
