# Association of positively selected eIF3a polymorphisms with toxicity of platinum-based chemotherapy in NSCLC patients

## Metadata
**Authors:** Ji-ye Yin, Xiang-guang Meng, Chen-yue Qian, Xiang-ping Li, Juan Chen, Yi Zheng, Rong Liu, Hong-hao Zhou, Zhao-qian Liu
**Journal:** Acta Pharmacologica Sinica
**Date:** 2015 Mar 3
**DOI:** [10.1038/aps.2014.160](https://doi.org/10.1038/aps.2014.160)
**PMID:** 25732572
**PMCID:** PMC4349930
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4349930/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC4349930/pdf/aps2014160a.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC4349930/pdf/aps2014160a.pdf)

## Abstract

**Aim:** 
Eukaryotic translation initiation factor 3 subunit A (eIF3a) plays critical roles in regulating the initiation of protein translation, and eIF3a is highly expressed in lung cancer. In this study, we investigated the association of the positively selected SNPs of eIF3a with the response to and toxicity of platinum-based chemotherapy in Chinese patients with non-small cell lung cancer (NSCLC).

**Methods:** 
SNP data for eIF3a locus were downloaded from HapMap database. For each SNP, haplotype, LD profile and population differentiation were analyzed. The long-range haplotype (LRH) test was employed to identify positively selected SNPs of eIF3a. A total of 325 NSCLC patients were enrolled and genotyped for these SNPs.

**Results:** 
Five positively selected (rs1409314, rs4752219, rs4752220, rs7091672 and rs10510050) and 5 non-positively selected SNPs (rs10886342, rs11198804, rs2275112, rs10787899 and rs4752269) were identified in the LRH test. However, none of them was correlated with the platinum-based chemotherapy response. In contrast, 4 of the positively selected SNPs (rs1409314, rs4752219, rs4752220 and rs7091672) were significantly correlated with the toxicities tested (neutropenia, anemia, thrombocytopenia, emesis and hepatotoxicity). In addition, rs10510050 was significantly correlated with thrombocytopenia, emesis and hepatotoxicity. None of the 5 non-positively selected SNPs was correlated with the 5 toxicities.

**Conclusion:** 
The positively selected SNPs of eIF3a are significantly correlated with platinum-based chemotherapy toxicities in Chinese NSCLC patients.

Keywords: eIF3a, positive selection, polymorphism, lung cancer, platinum-based chemotherapy, chemotherapy toxicity, Chinese NSCLC patients

### Aim:

Eukaryotic translation initiation factor 3 subunit A (eIF3a) plays critical roles in regulating the initiation of protein translation, and eIF3a is highly expressed in lung cancer. In this study, we investigated the association of the positively selected SNPs of eIF3a with the response to and toxicity of platinum-based chemotherapy in Chinese patients with non-small cell lung cancer (NSCLC).

### Methods:

SNP data for eIF3a locus were downloaded from HapMap database. For each SNP, haplotype, LD profile and population differentiation were analyzed. The long-range haplotype (LRH) test was employed to identify positively selected SNPs of eIF3a. A total of 325 NSCLC patients were enrolled and genotyped for these SNPs.

### Results:

Five positively selected (rs1409314, rs4752219, rs4752220, rs7091672 and rs10510050) and 5 non-positively selected SNPs (rs10886342, rs11198804, rs2275112, rs10787899 and rs4752269) were identified in the LRH test. However, none of them was correlated with the platinum-based chemotherapy response. In contrast, 4 of the positively selected SNPs (rs1409314, rs4752219, rs4752220 and rs7091672) were significantly correlated with the toxicities tested (neutropenia, anemia, thrombocytopenia, emesis and hepatotoxicity). In addition, rs10510050 was significantly correlated with thrombocytopenia, emesis and hepatotoxicity. None of the 5 non-positively selected SNPs was correlated with the 5 toxicities.

### Conclusion:

The positively selected SNPs of eIF3a are significantly correlated with platinum-based chemotherapy toxicities in Chinese NSCLC patients.

## Introduction

Translation is one of the major mechanisms of gene expression regulation. It consists of three major steps: initiation, elongation and termination. Among these, initiation is the rate-limiting step and involves at least 11 translation initiation factors (eIFs)^1^. EIF3 is the largest and most complex, consisting of at least 13 subunits named eIF3a to eIF3m^2,3^. EIF3a is the largest subunit with a molecular weight of 170 kDa, and the gene encoding it has 24 exons spanning approximately 45.8 kb of chromosome 10q26^4^.

EIF3a plays critical roles in all steps of cap-dependent translation initiation, including binding with the 40S ribosomal subunit, scanning and recognizing the start codon^2^. In addition, it has been reported to be involved in internal ribosomal entry site (IRES)-mediated cap-independent translation initiation^5,6^. However, recent studies have shown that eIF3a is overexpressed in many cancers, including breast, lung, cervix, stomach and esophagus^2^. Our previous study also showed that eIF3a regulates drug response by altering the expression of some DNA repair proteins^7^. These studies thus implicate eIF3a as a potential oncogene.

Single nucleotide polymorphisms (SNPs) are the most abundant human genome variations. Our previous study demonstrated that two *eIF3a* SNPs were correlated with drug response and toxicity^8,9^. We thus hypothesized that additional *eIF3a* SNPs may correlate with response and toxicity of platinum-based chemotherapy in non-small cell lung cancer (NSCLC) patients. It has been reported that screening SNPs with evidence of positive selection is a valid strategy for identifying functional SNPs^10,11,12^. This method is based on the theory that phenotypic differences between individuals are the functional consequences of genetic variants that facilitated the survival of ancestral human populations under different environmental stressors^13^. Therefore, SNPs under positive selection are likely to be functionally important for their linked genes. This strategy has been used successfully to identify functional SNPs associated with many genes, including glucose-6-phosphate dehydrogenase (*G6PD*)^13^, multidrug resistance 1 (*MDR1*)^10^, cytochrome P450, family 3, subfamily A (*CYP3A*)^11^ and proprotein convertase subtilisin-like kexin type 9 (*PCSK9*)^12^.

In the present study, we employed the long-range haplotype (LRH) test to identify positively selected SNPs of *eIF3a* and explored whether they were correlated with response and toxicity of platinum-based chemotherapy in Chinese NSCLC patients.

## Materials and methods

### Study population, acquisition of clinical information and genotyping

The study protocol was approved by the Ethics Committee of Xiangya School of Medicine, Central South University. A total of 325 lung cancer patients were enrolled in this study. All individuals provided written informed consent in compliance with the code of ethics of the World Medical Association (Declaration of Helsinki) before this study was initiated. Eligible subjects were treated at the Affiliated Cancer Hospital or Xiangya Hospital of Central South University (Changsha, Hunan, China) between November 2011 and May 2013. For this study, patients had to meet the following criteria: (1) histologic or cytologic confirmation of lung cancer and (2) presence of a primary tumor. Exclusion criteria included (1) pregnancy or lactation, (2) active infection, (3) symptomatic brain or leptomeningeal metastases and (4) previous or concomitant other malignancies. All demographic and clinical information was obtained from the two hospitals. The clinical characteristics of all subjects are summarized in [Table 1](#tbl1). We applied this study for clinical admission in the Chinese Clinical Trial Register (registration number: ChiCTR-RCC-12002830).

### Table 1. Baseline characteristic of NSCLC patients treated with platinum-based chemotherapy.

| Characteristics | All patients (%) | Responders (%) | Non-responders (%) | P value |
| --- | --- | --- | --- | --- |
| Total | 325 (100) | 132 (40.62) | 193 (59.38) |   |
| Age (mean±SD) | 53.82±9.24 | 55.35±9.27 | 52.78±9.09 | 0.010 |
| Gender |  |  |  |  |
| Male | 209 (64.31) | 82 (62.12) | 127 (65.80) |   |
| Female | 116 (35.69) | 50 (37.88) | 66 (34.20) | 0.556 |
| Histology |  |  |  |  |
| NSCLC | 246 (75.69) | 97 (73.48) | 149 (77.20) |   |
| SCLC | 79 (24.31) | 35 (26.52) | 44 (22.80) | 0.511 |
| Smoking status |  |  |  |  |
| Smokers | 169 (52.00) | 63 (47.73) | 106 (54.92) |   |
| Non-smokers | 156 (48.00) | 69 (52.27) | 87 (45.08) | 0.215 |
| ECOG PS |  |  |  |  |
| 0–1 | 311 (95.69) | 123 (93.18) | 188 (97.41) |   |
| ≥2 | 14 (4.31) | 9 (6.82) | 5 (2.59) | 0.093 |
| Chemotherapy |  |  |  |  |
| Platinum+gemcitabine (GP) | 94 (28.92) | 32 (24.24) | 62 (32.12) |   |
| Platinum+etoposide (EP) | 78 (24.00) | 46 (34.85) | 32 (16.58) |   |
| Platinum+paclitaxel (TP) | 16 (4.92) | 3 (2.27) | 13 (6.74) |   |
| Platinum+pemetrexed (PP) | 79 (24.31) | 27 (20.45) | 52 (26.94) |   |
| Other | 58 (17.85) | 24 (18.19) | 34 (17.62) | 0.002 |
The chemotherapy response was evaluated by the response evaluation criteria in solid tumor (RECIST) guidelines. Complete response (CR) and partial response (PR) were defined as drug response, whereas stable disease (SD) and progressive disease (PD) were defined as drug resistance. In the current study, hematologic toxicity (anemia, neutropenia and thrombocytopenia) and gastrointestinal toxicity (emesis and hepatotoxicity) were evaluated. Toxicity was classified according to the National Cancer Institute Common Toxicity Criteria 3.0 (NCI-CTC 3.0) at each cycle of chemotherapy. For each patient, the most severe grade during the entire study interval was evaluated. In brief, for all side effects, grade 0 and grade ≥1 were considered non-toxicity and toxicity, respectively. Genomic DNA of all subjects was isolated from the peripheral blood sample using a FlexiGene DNA Kit (Qiagen, Hilden, Germany) and stored at 4 °C until use. Genotyping was conducted by Sequenom's MassARRAY system (Sequenom, San Diego, California, USA).

### Polymorphism data and SNP marker selection

Genotype data of SNPs were retrieved from the International HapMap Project database (Phase III Release #28, [www.hapmap.org](http://www.hapmap.org)). Six populations were included in this study, including CEU, CHB, JPT, YRI, ASW, and CHD. The ancestral allele of each SNP was defined by aligning the sequence of each SNP to a chimpanzee (Pan troglodyte) sequence from the UCSC database ([genome.ucsc.edu](http://genome.ucsc.edu)).

SNPs spanning approximately 45.8 kb of the *eIF3* a locus were selected. To generate a more evenly distributed and denser SNP map, we extended the selection region to 200 kb upstream and downstream of the *eIF3a* locus. SNPs in the following categories were excluded from the study: (1) SNPs with a minor allele frequency <0.05, (2) SNPs that were not genotyped in at least one population and (3) SNPs departing from Hardy-Weinberg equilibrium. In total, 127 SNPs were included, and a flow chart describing SNP selection and information regarding included SNPs are provided in [Figure S1](#sup1) and [Table S1](#sup1), respectively.

### Data analysis

Data analysis was performed using R and Bioconductor. In addition, the Haploview program was utilized to perform genetic analysis, including allele frequency description and selection, LD and haplotype block analysis^14^. Lewontin's coefficient |*D*'| and Pearson's correlation *r*^2^ were employed to estimate LD between SNP pairs. Statistical significance of LD between SNP pairs was analyzed by Arlequin^16^. The classical LD decay equation *D*=*D*_0_(1–θ)^t^ was utilized to calculate LD decay trend line, which evaluated the overall LD in the populations^17^. Half-LD (LD_0.5_) was defined as the distance at which |*D*'|=0.5. Haplotype reconstruction was conducted in the PHASE program^18^. An unbiased estimator of F_ST_ (F-statistic/fixation index) from pairwise population comparisons was calculated. Randomization method (1000 permutations) was used to test the statistical significance of F_ST_ in the comparison using the FSTAT program^19^. A modified LRH test was used to assess for positive selection, and single SNP rather than haplotype was referred as “core”^13^. Haplotype homozygosity (HH) was calculated in a stepwise manner as extended haplotype homozygosity (EHH)^13^. Relative EHH (REHH) was the ratio of the EHH on the test core allele compared with the EHH of the grouped set of core alleles at the region not including the core allele tested^13^. Haplotype bifurcation diagram (HBD) was used to graphically evaluate the LD decay of both ancestral and derived alleles for a given core SNP. To test the statistical significance of the LRH test, an empirical distribution of core haplotype frequencies versus REHH was obtained by screening the entire chromosome 10 HapMap data in ASN, CEU and YRI populations. The HBD and LRH statistical tests were performed using Sweep software.

### Statistical analysis of association study

All continuous variables were presented as means±SD and confidence intervals (95% CI), and normally distributed data were analyzed by the two-sample *t* test. Noncontiguous variables, genotype distributions, and allelic frequencies in different groups were compared using the χ^2^ test. Unconditional logistic regression was performed to estimate the association of drug response and toxicity with eIF3a polymorphisms by calculating odds ratios (OR) and their 95% confidence intervals (CI). The *P* value was two-sided and *P*<0.05 was considered statistically significant. The aforementioned statistical analyses were performed using PLINK^20^ and SPSS 18.0 software (SPSS Inc, Chicago, Illinois, USA). Bonferroni correction was used to correct multiple testing.

## Results

### Haplotype and LD profile

We first detected the haplotype and LD profile of the involved SNPs, and the pair-wise |*D*'| and *r*^2^ were calculated to determine the *eIF3a* LD pattern. As shown in [Figure 1](#fig1), the patterns of LD structure were highly different across the entire locus between Africans and non-Africans. There are 18, 16, 17, 14, 29, and 29 blocks observed in CHD, CHB, JPT, CEU, ASW, and YRI populations, respectively. The longest blocks spanned 91 kb in CHD, 91 kb in CHB, 91 kb in JPT, 94 kb in CEU, 48 kb in ASW, and 26 kb in YRI.

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dc08/4349930/26669a549d64/aps2014160f1.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4349930_aps2014160f1.jpg)

LD pattern of eIF3a polymorphisms in the CHD, CHB, JPT, CEU, ASW, and YRI populations. Pairwise LD between polymorphisms was measured by |D'|. Haplotype blocks partitioned in each population were also numbered above them. The analysis was performed in the Haploview program (version 4.2).

We then plotted the |*D*'| and *r*^2^ between SNP pairs against their physical distance, as shown in [Figure 2](#fig2A). The LD generally decreased with physical distance. A curve was then fitted to determine the half LD (LD_50_) in all populations. The genetically older populations were expected to have lower LDs, and this was the case in our study. The LD_50_s of African populations YRI (44.5 kb) and ASW (61.5 kb) were shorter than those of non-African populations, while the longest LD_50_s presented in the Chinese population were CHD (78.3 kb) and CHB (75.4 kb).

### Figure 2A.

![Figure 2A](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dc08/4349930/98cf0f067b60/aps2014160f2A.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4349930_aps2014160f2A.jpg)

LD decay profile of eIF3a polymorphisms in the CHD, CHB, JPT, CEU, ASW, and YRI populations. (A) |D'| between eIF3a SNP pairs were calculated and plotted against their physical distance in the CHD, CHB, JPT, CEU, ASW, and YRI populations. A classical LD decay equation was utilized to calculate LD decay trend line, which is indicated by a red line in the figure. The LD50 was also shown. Details of the calculation were described in the Materials and methods section.

Furthermore, we calculated the proportion of SNP pairs with significant LD in these populations and observed clear differences. As shown in [Figure S2A](#sup1), African populations had fewer SNP pairs with significant LDs: the proportions were 23.66% and 19.03% in YRI and ASW populations. For non-African populations, however, 39.07%, 38.72%, 36.71%, and 35.88% SNP pairs showed significant LD in the CHD, CEU, CHB, and JPT populations, respectively. We also plotted the proportion of SNP pairs with insignificant LDs against physical distance for ASN (East Asian population; combined CHB and JPT populations), CEU and YRI populations ([Figure S2B](#sup1)). In accordance with previous observation, the African YRI population showed lower LD levels, while the ASN population showed higher allelic association.

Together, the haplotype and LD pattern displayed high diversity among the 6 populations analyzed, especially between African (ASW and YRI) and non-African (CHD, CHB, CEU, and JPT) populations.

### Population differentiation

To further test for differences in *eIF3a* allelic frequency among populations, we calculated F_ST_ values. Considering that the CHB and JPT populations exist in the same region and have substantial similarity in their genetic makeup, they were combined to make one ASN population in further analyses. Additionally, the following analyses were conducted on only the ASN, CEU, and YRI populations, which reflected a substantial amount of the genetic variation found throughout the world^21^.

Based on the pairwise F_ST_ values, *eIF3a* showed high population differentiation. SNPs with a proportion of 21.71% in CEU versus ASN, 43.41% in CEU versus YRI and 33.33% in ASN versus YRI showed significantly high F_ST_, respectively. We next plotted the F_ST_ values against physical distance in [Figure S3](#sup1). As shown, 7 SNPs in CEU versus YRI and 5 SNPs in ASN versus YRI had F_ST_ values above the threshold of 0.45, which was considered the threshold value of selection pressure^22,23^. However, none of the SNPs in CEU versus ASN had F_ST_ values above 0.45, indicating that *eIF3a* had low levels of differentiation between these two populations.

These results indicated that there was significantly different population differentiation among the ASN, CEU, and YRI populations, and that some SNPs may be under selective pressure.

### Signature of positive selection of eIF3a

To test for evidence of positive selection occurring at these SNPs with *eIF3a*, we performed an LRH test. We first plotted the relative EHH (REHH) of core SNPs against their allele frequencies. Based on the empirical distribution, 41, 33, and 13 SNPs showed significant departure from evolutionary neutrality in the YRI, ASN, and CEU populations, respectively. The top 5 SNPs with lowest *P* values were then involved in further study. As shown in [Figure 3](#fig3), A allele of SNP rs1409314 (*P*=0.0012), C allele of SNP rs4752219 (*P*=0.0025), A allele of SNP rs4752220 (*P*=0.0025), T allele of SNP rs7091672 (*P*=0.0043) and A allele of SNP rs10510050 (*P*=0.0046) deviated significantly from evolutionary neutrality. To assess this result, the EHH decay of these 5 SNPs was plotted against the physical distance. Generally, the overall EHH of the rs1409314A, rs4752219C, rs4752220A, rs7091672T, and rs10510050A alleles decayed more slowly than their corresponding ancestral/derived alleles in all three populations ([Figure S4](#sup1)). To confirm this result, we plotted haplotype branching diagrams (HBD) for these 5 SNPs, as indicated in [Figure S5](#sup1). Consistent with previous results, the rs1409314A, rs4752219C, rs4752220A, rs7091672T, and rs10510050A alleles had obvious long-range LD across all populations, especially in YRI, as indicated by a single distinctly predominant thick branch. Taken these results together, we concluded that rs1409314, rs4752219, rs4752220, rs7091672, and rs10510050 showed a signature of recent positive selection.

### Figure 3.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dc08/4349930/e422f17ee567/aps2014160f3.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4349930_aps2014160f3.jpg)

LRH test for 5 positively selected SNPs. The REHH value of 5 core alleles was plotted against its frequency in the ASN, CEU and YRI populations. The points of REHH against allele frequency were based on the calculation of chromosome 10 phased data downloaded from the HapMap database (data point not shown in the figure). The 50th, 75th, 90th, 95th, 99th, and 99.9th percentile lines were obtained by binning the empirical data by allele frequencies into 20 bins with equal intervals of 0.05. All analysis was performed in the Sweep programe.

As a control, we further analyzed 5 randomly selected SNPs (rs10886342, rs11198804, rs2275112, rs10787899, and rs4752269), which didn't show evidence of positive selection. These 5 SNPs were selected as representative non-positively selected SNPs of *eIF3* a for use in the following study.

### Association of selected SNPs and platinum-based chemotherapy response

Our previous study demonstrated that eIF3a may act as an oncogene and may be correlated with lung cancer chemotherapy response and toxicity^4,6,7^. Thus, we tested whether these positively selected SNPs were associated with platinum-based chemotherapy response in Chinese NSCLC patients. A total of 325 lung cancer patients were recruited for genotyping of the aforementioned 5 positively selected and 5 non-positively selected SNPs. There were significant differences between responders and non-responders in terms of age and chemotherapy regimen, suggesting that these factors need to be adjusted in the statistical analysis ([Table 1](#tbl1)). All tested SNPs were consistent with Hardy-Weinberg equilibrium in both responders and non-responders.

[Table 2](#tbl2) summarizes the results of association study of 10 SNPs in additive, dominant and recessive models. As indicated, neither the 5 positively selected SNPs (rs1409314, rs4752219, rs4752220, rs7091672 and rs10510050) nor the 5 non-positively selected SNPs (rs10886342, rs11198804, rs2275112, rs10787899, and rs4752269) were significantly associated with NSCLC patients' platinum-based chemotherapy response.

### Table 2. Association of eIF3a SNPs with platinum-based chemotherapy response in the Chinese patients.

| Polymorphisms | Genotype | Responders n (%) | Non-responders n (%) | Call rate (%) | MAF | Additive |  | Dominant |  | Recessive |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| OR (95% CI) | P | OR (95% CI) | P | OR (95% CI) | P |  |  |  |  |  |  |
| rs1409314 | AA | 27 (20.93) | 38 (20.32) | 97.74 | 0.43 | 0.83 (0.61–1.12) | 0.23 | 0.66 (0.41–1.05) | 0.08 | 0.96 (0.55–1.68) | 0.90 |
|   | AG | 62 (48.06) | 73 (39.04) |   |   |   |   |   |   |   |   |
|   | GG | 40 (31.01) | 76 (40.64) |   |   |   |   |   |   |   |   |
| rs4752219 | CC | 45 (36.29) | 64 (35.75) | 96.20 | 0.42 | 1.16 (0.85–1.58) | 0.35 | 1.02 (0.64–1.65) | 0.92 | 1.60 (0.90–2.85) | 0.11 |
|   | CT | 58 (46.77) | 71 (39.66) |   |   |   |   |   |   |   |   |
|   | TT | 21 (16.94) | 44 (24.59) |   |   |   |   |   |   |   |   |
| rs4752220 | AA | 48 (36.92) | 67 (35.83) | 98.62 | 0.42 | 1.16 (0.86–1.57) | 0.33 | 1.05 (0.66–1.67) | 0.84 | 1.56 (0.88–2.75) | 0.13 |
|   | AG | 60 (46.15) | 75 (40.11) |   |   |   |   |   |   |   |   |
|   | GG | 22 (16.93) | 45 (24.06) |   |   |   |   |   |   |   |   |
| rs7091672 | TT | 19 (15.45) | 29 (16.20) | 96.03 | 0.39 | 0.89 (0.64–1.23) | 0.47 | 0.76 (0.47–1.21) | 0.24 | 1.06 (0.56–2.00) | 0.86 |
|   | CT | 58 (47.15) | 71 (39.66) |   |   |   |   |   |   |   |   |
|   | CC | 46 (37.40) | 79 (44.14) |   |   |   |   |   |   |   |   |
| rs10510050 | AA | 25 (19.23) | 34 (18.28) | 98.45 | 0.40 | 0.83 (0.61–1.12) | 0.23 | 0.68 (0.43–1.08) | 0.10 | 0.94 (0.53–1.67) | 0.83 |
|   | AG | 59 (45.38) | 69 (37.10) |   |   |   |   |   |   |   |   |
|   | GG | 46 (35.39) | 83 (44.62) |   |   |   |   |   |   |   |   |
| rs10886342 | AA | 27 (20.93) | 45 (23.94) | 98.27 | 0.44 | 1.06 (0.78–1.42) | 0.73 | 1.00 (0.63–1.60) | 1.00 | 1.19 (0.69–2.04) | 0.53 |
|   | AG | 56 (43.41) | 76 (40.43) |   |   |   |   |   |   |   |   |
|   | GG | 46 (35.66) | 67 (35.63) |   |   |   |   |   |   |   |   |
| rs11198804 | TT | 29 (22.31) | 53 (28.19) | 98.79 | 0.49 | 1.12 (0.83–1.52) | 0.46 | 1.01 (0.62–1.66) | 0.96 | 1.37 (0.81–2.30) | 0.24 |
|   | TC | 64 (49.23) | 82 (43.62) |   |   |   |   |   |   |   |   |
|   | CC | 37 (28.46) | 53 (28.19) |   |   |   |   |   |   |   |   |
| rs2275112 | GG | 82 (65.60) | 128 (71.11) | 96.55 | 0.16 | 0.83 (0.52–1.31) | 0.42 | 0.77 (0.47–1.27) | 0.31 | 2.10 (0.22–20.44) | 0.52 |
|   | GA | 42 (33.60) | 49 (27.22) |   |   |   |   |   |   |   |   |
|   | AA | 1 (0.80) | 3 (1.67) |   |   |   |   |   |   |   |   |
| rs10787899 | AA | 19 (15.32) | 32 (17.88) | 96.03 | 0.43 | 1.09 (0.78–1.53) | 0.60 | 1.08 (0.67–1.75) | 0.76 | 1.20 (0.65–2.24) | 0.56 |
|   | AG | 62 (50.00) | 88 (49.16) |   |   |   |   |   |   |   |   |
|   | GG | 43 (34.68) | 59 (32.96) |   |   |   |   |   |   |   |   |
| rs4752269 | TT | 4 (3.23) | 7 (3.89) | 96.37 | 0.19 | 1.44 (0.94–2.20) | 0.09 | 1.58 (0.97–2.56) | 0.06 | 1.21 (0.35–4.24) | 0.76 |
|   | TC | 34 (27.42) | 67 (37.22) |   |   |   |   |   |   |   |   |
|   | CC | 86 (69.35) | 106 (58.89) |   |   |   |   |   |   |   |   |
### Association of selected SNPs and platinum-based chemotherapy toxicity

We next tested whether positively selected SNPs were correlated with chemotherapy toxicity in these patients. In the present study, we investigated 5 toxicities: neutropenia, anemia, thrombocytopenia, emesis and hepatotoxicity. As indicated in [Table 3](#tbl3), each was divided into two groups: toxicity (0 grade) and non-toxicity (≥1 grade). In general, 64%–68% patients showed at least one toxicity and only 32%–36% patients did not suffer from any toxicity.

### Table 3. Groups of graded platinum-based chemotherapy toxicities.

| Side effect | 0 grade n (%) | ≥1 grade n (%) |
| --- | --- | --- |
| Neutropenia | 112 (34.46) | 213 (65.54) |
| Anemia | 114 (35.08) | 211 (64.92) |
| Thrombocytopenia | 114 (35.08) | 211 (64.92) |
| Emesis | 116 (35.69) | 209 (64.31) |
| Hepatotoxicity | 104 (32.00) | 221 (68.00) |
It is interesting to note that 4 (rs1409314, rs4752219, rs4752220, and rs7091672) of the 5 positively selected SNPs were significantly correlated with all tested toxicities ([Table 4](#tbl4)). In addition, a significant association was also detected between rs10510050 and thrombocytopenia (OR=4.34, 95% CI=1.47–12.81, *P*=0.00), emesis (OR=0.39, 95% CI=0.18–0.83, *P*=0.01) and hepatotoxicity (OR=0.58, 95% CI=0.34–0.99, *P*=0.04). In contrast, of the 5 non-positively selected SNPs (rs10886342, rs11198804, rs2275112, rs10787899, and rs4752269), none showed significant association with any of the 5 tested toxicities. These results together suggest that the aforementioned 5 positively selected eIF3a SNPs are significantly correlated with platinum-based chemotherapy toxicities in Chinese NSCLC patients.

### Table 4. Association of eIF3a SNPs with platinum-based chemotherapy toxicities in the Chinese patients. bP<0.05.

| SNPs | Leukopenia |  | Anemia |  | Thrombocytopenia |  | Emesis |  | Hepatotoxicity |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| OR (95% CI) | P | OR (95% CI) | P | OR (95% CI) | P | OR (95% CI) | P | OR (95% CI) | P |  |
| rs1409314 | 2.36 (1.23–4.53) | 0.01b | 2.44 (1.27–4.67) | 0.01b | 2.44 (1.27–4.67) | 0.01b | 2.47 (1.29–4.74) | 0.01b | 1.58 (1.14–2.61) | 0.01b |
| rs4752219 | 1.64 (1.12–2.39) | 0.01b | 1.58 (1.09–2.30) | 0.02b | 1.64 (1.07–2.75) | 0.02b | 1.55 (1.07–2.25) | 0.02b | 1.52 (1.02–2.53) | 0.02b |
| rs4752220 | 1.64 (1.12–2.39) | 0.01b | 1.58 (1.09–2.30) | 0.02b | 1.64 (1.07–2.75) | 0.02b | 1.55 (1.07–2.25) | 0.02b | 1.52 (1.02–2.53) | 0.02b |
| rs7091672 | 3.44 (1.32–8.98) | 0.01b | 2.84 (1.16–6.92) | 0.02b | 1.60 (1.05–2.68) | 0.02b | 2.88 (1.18–7.02) | 0.02b | 1.64 (1.07–2.75) | 0.02b |
| rs10510050 | 1.01 (0.47–2.21) | 0.97 | 0.78 (0.36–1.69) | 0.52 | 4.34 (1.47–12.81) | 0.00b | 0.39 (0.18–0.83) | 0.01b | 0.58 (0.34–0.99) | 0.04b |
| rs10886342 | 1.13 (0.53–2.42) | 0.75 | 1.31 (0.63–2.74) | 0.46 | 0.51 (0.16–1.65) | 0.25 | 0.89 (0.48–1.66) | 0.72 | 0.95 (0.59–1.53) | 0.84 |
| rs11198804 | 1.42 (0.66–3.06) | 0.36 | 1.21 (0.58–2.53) | 0.61 | 1.05 (0.37–3.04) | 0.92 | 1.40 (0.77–2.57) | 0.27 | 1.06 (0.66–1.69) | 0.81 |
| rs2275112 | 1.24 (0.46–3.37) | 0.67 | 0.85 (0.29–2.52) | 0.76 | 2.11 (0.65–6.87) | 0.21 | 0.91 (0.48–1.82) | 0.85 | 1.27 (0.69–2.34) | 0.44 |
| rs10787899 | 0.50 (0.21–1.21) | 0.12 | 1.06 (0.49–2.27) | 0.89 | 0.78 (0.26–2.34) | 0.65 | 1.26 (0.90–1.77) | 0.18 | 1.39 (0.87–2.25) | 0.17 |
| rs4752269 | 1.48 (0.61–3.61) | 0.39 | 0.85 (0.32–2.29) | 0.75 | 2.24 (0.74–6.81) | 0.14 | 0.68 (0.28–1.66) | 0.39 | 0.84 (0.46–1.56) | 0.59 |
## Discussion

In the present study, we first characterized the haplotype and LD profile of *eIF3a* in data obtained from the HapMap database, then analyzed the population differentiation in three populations (ASN, CEU, and YRI). Moreover, we identified 5 SNPs (rs1409314, rs4752219, rs4752220, rs7091672, and rs10510050) of *eIF3a* that had undergone recent positive selection using the LRH test. Finally, we demonstrated that these SNPs were correlated with platinum-based chemotherapy toxicities in Chinese NSCLC patients.

With the explosion of newly identified SNPs, it is becoming impractical to investigate every SNP individually for correlation with a disease or drug response. Although new methods of DNA sequencing and genotyping have been developed, these are time consuming and costly. Thus, bioinformatics methods have been developed to identify potential functional SNPs. Based on the theories of natural selection and the aforementioned “Out-of-Africa” hypothesis, searching for signatures of recent positive selection can be used to identify potential functional SNPs^24^. With the dispersal of humans from Africa to the other regions of the world, different populations faced various selective pressures presented by climate, pathogens and sources of food. Thus, functional SNPs that facilitated survival in ancestral human populations may result in phenotypic differences among modern individuals or groups^25^. These SNPs may be retained under the pressure of positive selection; for example, a classic G6PD-202A variant that confers malaria resistance showed a strong evidence of recent positive selection^13^. Other groups have successfully utilized this method to detect functional SNPs in *PCSK9*^12^ and ABC drug transporter genes^26^. In the current study, we employed this method to identify 5 positively selected *eIF3a* SNPs and confirmed that they were significantly correlated with platinum-based chemotherapy toxicities in Chinese NSCLC patients. Importantly, another 5 non-positively selected control SNPs did not show evidence of association. Although our previous studies identified a number of platinum-based chemotherapy responses and toxicities associated SNPs^27,28,29^, this time- and cost-saving method is a potentially powerful tool for specifically identifying SNPs that correlate with drug response in the future.

Although eIF3a is the largest subunit of eIF3, which is itself the largest and most complex initiation factor, it has been shown to be dispensable for the function of eIF3-medicated translation initiation^2,30^. By contrast, previous studies by our group and others have suggested that eIF3a is a potential oncogene. Thus, it is important to understand the contributions of *eIF3a* polymorphisms to its correlation with cancers. Our previous study investigated the correlation of *eIF3a* polymorphisms and platinum-based chemotherapy resistance and severe toxicity in the Chinese population^8,9^. Olson *et al* found that *eIF3a* rs10787899 was significantly correlated with breast cancer risk in Caucasian female populations^31^. In the current study, we further identified 5 polymorphisms which correlated with lung cancer platinum-based chemotherapy toxicities in the Chinese population. While the mechanisms of the correlation remain unknown, we speculate that these SNPs may be potential genetic markers for drug response and toxicity.

In the present investigation, we extended the SNP selection region to 200 kb upstream and downstream of the *eIF3a* locus to generate a more even distribution. However, some regions selected overlapped with other genes, including family with sequence similarity 45, member A (*FAM45A*) in the *eIF3a* 5′ untranslated region (5′ UTR) and nanos homolog 1 (*Drosophila*) (*NANOS1*) in the *eIF3a* 3′ UTR regions. Indeed, the 5 positively selected SNPs (rs1409314, rs4752219, rs4752220, rs7091672, and rs10510050) were located in the overlap region of *NANOS1* and *eIF3a*. Thus, they may be considered genetic markers for both genes. This point is especially important for *in vitro* functional study, where both genes should be considered in the investigation. It should be noted that some limitations need to be considered when interpreting results of this study. First, although we confirmed the function of these positively selected SNPs in clinical patients, we did not conduct a study *in vitro*. Therefore, how these SNPs affected eIF3a function still remains unknown. Another limitation came from our sample size. The current study had a relatively small sample size and lacked confirmation from other validation stage populations. Additionally, we only detected the correlation in the Chinese population. Thus, this result needs to be further confirmed in a larger sample size and in different ethnic populations.

In summary, we identified 5 positively selected *eIF3a* SNPs using the LRH test and confirmed that they were significantly correlated with platinum-based chemotherapy toxicities in Chinese NSCLC patients. Our results imply that this strategy of SNP selection may be an inexpensive, effective way to discovery of SNPs correlated to drug response and toxicities.

## Author contribution

Ji-ye YIN, Hong-hao ZHOU, and Zhao-qian LIU conceived and designed the experiments. Ji-ye YIN, Xiang-guang MENG, Juan CHEN, and Yi ZHENG performed the experiments. Ji-ye YIN, Xiang-guang MENG, and Rong LIU analyzed the data. Chen-yue QIAN and Xiang-ping LI contributed reagents, materials and analysis tools. Ji-ye YIN and Zhao-qian LIU wrote the paper.

![Figure 2B](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dc08/4349930/4bdf43f59b1d/aps2014160f2B.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4349930_aps2014160f2B.jpg)

LD decay profile of eIF3a polymorphisms in the CHD, CHB, JPT, CEU, ASW, and YRI populations. (B) r2 between eIF3a SNP pairs were calculated and plotted against their physical distance in the CHD, CHB, JPT, CEU, ASW, and YRI populations. Details of the calculation were described in the Materials and methods section.

## Acknowledgments

This work was supported by National High-tech R&D Program of China (863 Program Grant, No 2012AA02A517 and 2012AA02A518), National Natural Science Foundation of China (No 81173129, 81202595, 81373490, and 81273595), and Hunan Provincial Natural Science Foundation of China (No 2015JJ1024).

## Footnotes

## Supplementary Information

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Sonenberg N, Hershey JWB, Mathews MB.Translational control of gene expressionCold Spring Harbor (NY): Cold Spring Harbor Laboratory Press; 2000.

2. Yin JY, Dong Z, Liu ZQ, Zhang JT. Translational control gone awry: a new mechanism of tumorigenesis and novel targets of cancer treatments. Biosci Rep. 2011;31:1–15. doi: 10.1042/BSR20100077.  [DOI](https://doi.org/10.1042/BSR20100077) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20964625/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biosci%20Rep&title=Translational%20control%20gone%20awry:%20a%20new%20mechanism%20of%20tumorigenesis%20and%20novel%20targets%20of%20cancer%20treatments&author=JY%20Yin&author=Z%20Dong&author=ZQ%20Liu&author=JT%20Zhang&volume=31&publication_year=2011&pages=1-15&pmid=20964625&doi=10.1042/BSR20100077&)

3. Dong Z, Zhang JT. Initiation factor eIF3 and regulation of mRNA translation, cell growth, and cancer. Crit Rev Oncol Hematol. 2006;59:169–80. doi: 10.1016/j.critrevonc.2006.03.005.  [DOI](https://doi.org/10.1016/j.critrevonc.2006.03.005) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16829125/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Crit%20Rev%20Oncol%20Hematol&title=Initiation%20factor%20eIF3%20and%20regulation%20of%20mRNA%20translation,%20cell%20growth,%20and%20cancer&author=Z%20Dong&author=JT%20Zhang&volume=59&publication_year=2006&pages=169-80&pmid=16829125&doi=10.1016/j.critrevonc.2006.03.005&)

4. Pincheira R, Chen Q, Zhang JT. Identification of a 170-kDa protein over-expressed in lung cancers. Br J Cancer. 2001;84:1520–7. doi: 10.1054/bjoc.2001.1828.  [DOI](https://doi.org/10.1054/bjoc.2001.1828) | [PMC free article](/articles/PMC2363652/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11384103/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Cancer&title=Identification%20of%20a%20170-kDa%20protein%20over-expressed%20in%20lung%20cancers&author=R%20Pincheira&author=Q%20Chen&author=JT%20Zhang&volume=84&publication_year=2001&pages=1520-7&pmid=11384103&doi=10.1054/bjoc.2001.1828&)

5. Buratti E, Tisminetzky S, Zotti M, Baralle FE. Functional analysis of the interaction between HCV 5'UTR and putative subunits of eukaryotic translation initiation factor eIF3. Nucleic Acids Res. 1998;26:3179–87. doi: 10.1093/nar/26.13.3179.  [DOI](https://doi.org/10.1093/nar/26.13.3179) | [PMC free article](/articles/PMC147697/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9628916/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=Functional%20analysis%20of%20the%20interaction%20between%20HCV%205'UTR%20and%20putative%20subunits%20of%20eukaryotic%20translation%20initiation%20factor%20eIF3&author=E%20Buratti&author=S%20Tisminetzky&author=M%20Zotti&author=FE%20Baralle&volume=26&publication_year=1998&pages=3179-87&pmid=9628916&doi=10.1093/nar/26.13.3179&)

6. Yin JY, Dong ZZ, Liu RY, Chen J, Liu ZQ, Zhang JT. Translational regulation of RPA2 via internal ribosomal entry site and by eIF3a. Carcinogenesis. 2013;34:1224–31. doi: 10.1093/carcin/bgt052.  [DOI](https://doi.org/10.1093/carcin/bgt052) | [PMC free article](/articles/PMC3670257/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23393223/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Carcinogenesis&title=Translational%20regulation%20of%20RPA2%20via%20internal%20ribosomal%20entry%20site%20and%20by%20eIF3a&author=JY%20Yin&author=ZZ%20Dong&author=RY%20Liu&author=J%20Chen&author=ZQ%20Liu&volume=34&publication_year=2013&pages=1224-31&pmid=23393223&doi=10.1093/carcin/bgt052&)

7. Yin JY, Shen J, Dong ZZ, Huang Q, Zhong MZ, Feng DY, et al. Effect of eIF3a on response of lung cancer patients to platinum-based chemotherapy by regulating DNA repair. Clin Cancer Res. 2011;17:4600–9. doi: 10.1158/1078-0432.CCR-10-2591.  [DOI](https://doi.org/10.1158/1078-0432.CCR-10-2591) | [PMC free article](/articles/PMC11531068/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21610145/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&title=Effect%20of%20eIF3a%20on%20response%20of%20lung%20cancer%20patients%20to%20platinum-based%20chemotherapy%20by%20regulating%20DNA%20repair&author=JY%20Yin&author=J%20Shen&author=ZZ%20Dong&author=Q%20Huang&author=MZ%20Zhong&volume=17&publication_year=2011&pages=4600-9&pmid=21610145&doi=10.1158/1078-0432.CCR-10-2591&)

8. Xu X, Han L, Duan L, Zhao Y, Yang H, Zhou B, et al. Association between eIF3alpha polymorphism and severe toxicity caused by platinum-based chemotherapy in non-small cell lung cancer patients. Br J Clin Pharmacol. 2013;75:516–23. doi: 10.1111/j.1365-2125.2012.04379.x.  [DOI](https://doi.org/10.1111/j.1365-2125.2012.04379.x) | [PMC free article](/articles/PMC3579266/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22804784/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Clin%20Pharmacol&title=Association%20between%20eIF3alpha%20polymorphism%20and%20severe%20toxicity%20caused%20by%20platinum-based%20chemotherapy%20in%20non-small%20cell%20lung%20cancer%20patients&author=X%20Xu&author=L%20Han&author=L%20Duan&author=Y%20Zhao&author=H%20Yang&volume=75&publication_year=2013&pages=516-23&pmid=22804784&doi=10.1111/j.1365-2125.2012.04379.x&)

9. Xu X, Han L, Yang H, Duan L, Zhou B, Zhao Y, et al. The A/G allele of eIF3a rs3740556 predicts platinum-based chemotherapy resistance in lung cancer patients. Lung Cancer. 2013;79:65–72. doi: 10.1016/j.lungcan.2012.10.005.  [DOI](https://doi.org/10.1016/j.lungcan.2012.10.005) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23127338/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lung%20Cancer&title=The%20A/G%20allele%20of%20eIF3a%20rs3740556%20predicts%20platinum-based%20chemotherapy%20resistance%20in%20lung%20cancer%20patients&author=X%20Xu&author=L%20Han&author=H%20Yang&author=L%20Duan&author=B%20Zhou&volume=79&publication_year=2013&pages=65-72&pmid=23127338&doi=10.1016/j.lungcan.2012.10.005&)

10. Tang K, Wong LP, Lee EJ, Chong SS, Lee CG. Genomic evidence for recent positive selection at the human MDR1 gene locus. Hum Mol Genet. 2004;13:783–97. doi: 10.1093/hmg/ddh099.  [DOI](https://doi.org/10.1093/hmg/ddh099) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14976162/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mol%20Genet&title=Genomic%20evidence%20for%20recent%20positive%20selection%20at%20the%20human%20MDR1%20gene%20locus&author=K%20Tang&author=LP%20Wong&author=EJ%20Lee&author=SS%20Chong&author=CG%20Lee&volume=13&publication_year=2004&pages=783-97&pmid=14976162&doi=10.1093/hmg/ddh099&)

11. Chen X, Wang H, Zhou G, Zhang X, Dong X, Zhi L, et al. Molecular population genetics of human CYP3A locus: signatures of positive selection and implications for evolutionary environmental medicine. Environ Health Perspect. 2009;117:1541–8. doi: 10.1289/ehp.0800528.  [DOI](https://doi.org/10.1289/ehp.0800528) | [PMC free article](/articles/PMC2790508/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20019904/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Environ%20Health%20Perspect&title=Molecular%20population%20genetics%20of%20human%20CYP3A%20locus:%20signatures%20of%20positive%20selection%20and%20implications%20for%20evolutionary%20environmental%20medicine&author=X%20Chen&author=H%20Wang&author=G%20Zhou&author=X%20Zhang&author=X%20Dong&volume=117&publication_year=2009&pages=1541-8&pmid=20019904&doi=10.1289/ehp.0800528&)

12. Ding K, Kullo IJ. Molecular population genetics of PCSK9: a signature of recent positive selection. Pharmacogenet Genomics. 2008;18:169–79. doi: 10.1097/FPC.0b013e3282f44d99.  [DOI](https://doi.org/10.1097/FPC.0b013e3282f44d99) | [PMC free article](/articles/PMC2842919/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18300938/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Molecular%20population%20genetics%20of%20PCSK9:%20a%20signature%20of%20recent%20positive%20selection&author=K%20Ding&author=IJ%20Kullo&volume=18&publication_year=2008&pages=169-79&pmid=18300938&doi=10.1097/FPC.0b013e3282f44d99&)

13. Sabeti PC, Reich DE, Higgins JM, Levine HZ, Richter DJ, Schaffner SF, et al. Detecting recent positive selection in the human genome from haplotype structure. Nature. 2002;419:832–7. doi: 10.1038/nature01140.  [DOI](https://doi.org/10.1038/nature01140) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12397357/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Detecting%20recent%20positive%20selection%20in%20the%20human%20genome%20from%20haplotype%20structure&author=PC%20Sabeti&author=DE%20Reich&author=JM%20Higgins&author=HZ%20Levine&author=DJ%20Richter&volume=419&publication_year=2002&pages=832-7&pmid=12397357&doi=10.1038/nature01140&)

14. Barrett JC, Fry B, Maller J, Daly MJ. Haploview: analysis and visualization of LD and haplotype maps. Bioinformatics. 2005;21:263–5. doi: 10.1093/bioinformatics/bth457.  [DOI](https://doi.org/10.1093/bioinformatics/bth457) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15297300/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=Haploview:%20analysis%20and%20visualization%20of%20LD%20and%20haplotype%20maps&author=JC%20Barrett&author=B%20Fry&author=J%20Maller&author=MJ%20Daly&volume=21&publication_year=2005&pages=263-5&pmid=15297300&doi=10.1093/bioinformatics/bth457&)

15. Lewontin RC. Annotation: the analysis of variance and the analysis of causes. Am J Hum Genet. 1974;26:400–11.  [PMC free article](/articles/PMC1762622/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/4827368/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=Annotation:%20the%20analysis%20of%20variance%20and%20the%20analysis%20of%20causes&author=RC%20Lewontin&volume=26&publication_year=1974&pages=400-11&pmid=4827368&)

16. Excoffier L, Lischer HE. Arlequin suite ver 3.5: a new series of programs to perform population genetics analyses under Linux and Windows. Mol Ecol Resour. 2010;10:564–7. doi: 10.1111/j.1755-0998.2010.02847.x.  [DOI](https://doi.org/10.1111/j.1755-0998.2010.02847.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21565059/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Ecol%20Resour&title=Arlequin%20suite%20ver%203.5:%20a%20new%20series%20of%20programs%20to%20perform%20population%20genetics%20analyses%20under%20Linux%20and%20Windows&author=L%20Excoffier&author=HE%20Lischer&volume=10&publication_year=2010&pages=564-7&pmid=21565059&doi=10.1111/j.1755-0998.2010.02847.x&)

17. Schulze TG, Chen YS, Akula N, Hennessy K, Badner JA, McInnis MG, et al. Can long-range microsatellite data be used to predict short-range linkage disequilibrium. Hum Mol Genet. 2002;11:1363–72. doi: 10.1093/hmg/11.12.1363.  [DOI](https://doi.org/10.1093/hmg/11.12.1363) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12023978/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mol%20Genet&title=Can%20long-range%20microsatellite%20data%20be%20used%20to%20predict%20short-range%20linkage%20disequilibrium&author=TG%20Schulze&author=YS%20Chen&author=N%20Akula&author=K%20Hennessy&author=JA%20Badner&volume=11&publication_year=2002&pages=1363-72&pmid=12023978&doi=10.1093/hmg/11.12.1363&)

18. Stephens M, Smith NJ, Donnelly P. A new statistical method for haplotype reconstruction from population data. Am J Hum Genet. 2001;68:978–89. doi: 10.1086/319501.  [DOI](https://doi.org/10.1086/319501) | [PMC free article](/articles/PMC1275651/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11254454/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=A%20new%20statistical%20method%20for%20haplotype%20reconstruction%20from%20population%20data&author=M%20Stephens&author=NJ%20Smith&author=P%20Donnelly&volume=68&publication_year=2001&pages=978-89&pmid=11254454&doi=10.1086/319501&)

19. JG FSTAT, a program to estimate and test gene diversities and fixation indices (version2.9.3). Available from http://www.unil.ch/izea/softwares/fstat.html . Updated from Goudet (1995). 2001  [http://www.unil.ch/izea/softwares/fstat.html](http://www.unil.ch/izea/softwares/fstat.html)

20. Purcell S, Neale B, Todd-Brown K, Thomas L, Ferreira MA, Bender D, et al. PLINK: a tool set for whole-genome association and population-based linkage analyses. Am J Hum Genet. 2007;81:559–75. doi: 10.1086/519795.  [DOI](https://doi.org/10.1086/519795) | [PMC free article](/articles/PMC1950838/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17701901/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=PLINK:%20a%20tool%20set%20for%20whole-genome%20association%20and%20population-based%20linkage%20analyses&author=S%20Purcell&author=B%20Neale&author=K%20Todd-Brown&author=L%20Thomas&author=MA%20Ferreira&volume=81&publication_year=2007&pages=559-75&pmid=17701901&doi=10.1086/519795&)

21. International HapMap Consortium. The International HapMap Project Nature 2003426789–96.  [DOI](https://doi.org/10.1038/nature02168) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14685227/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=The%20International%20HapMap%20Project&volume=426&publication_year=2003&pages=789-96&pmid=14685227&doi=10.1038/nature02168&)

22. Akey JM, Zhang G, Zhang K, Jin L, Shriver MD. Interrogating a high-density SNP map for signatures of natural selection. Genome Res. 2002;12:1805–14. doi: 10.1101/gr.631202.  [DOI](https://doi.org/10.1101/gr.631202) | [PMC free article](/articles/PMC187574/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12466284/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Res&title=Interrogating%20a%20high-density%20SNP%20map%20for%20signatures%20of%20natural%20selection&author=JM%20Akey&author=G%20Zhang&author=K%20Zhang&author=L%20Jin&author=MD%20Shriver&volume=12&publication_year=2002&pages=1805-14&pmid=12466284&doi=10.1101/gr.631202&)

23. Rosenberg NA, Pritchard JK, Weber JL, Cann HM, Kidd KK, Zhivotovsky LA, et al. Genetic structure of human populations. Science. 2002;298:2381–5. doi: 10.1126/science.1078311.  [DOI](https://doi.org/10.1126/science.1078311) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12493913/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=Genetic%20structure%20of%20human%20populations&author=NA%20Rosenberg&author=JK%20Pritchard&author=JL%20Weber&author=HM%20Cann&author=KK%20Kidd&volume=298&publication_year=2002&pages=2381-5&pmid=12493913&doi=10.1126/science.1078311&)

24. Biswas S, Akey JM. Genomic insights into positive selection. Trends Genet. 2006;22:437–46. doi: 10.1016/j.tig.2006.06.005.  [DOI](https://doi.org/10.1016/j.tig.2006.06.005) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16808986/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Trends%20Genet&title=Genomic%20insights%20into%20positive%20selection&author=S%20Biswas&author=JM%20Akey&volume=22&publication_year=2006&pages=437-46&pmid=16808986&doi=10.1016/j.tig.2006.06.005&)

25. Bamshad M, Wooding SP. Signatures of natural selection in the human genome. Nat Rev Genet. 2003;4:99–111. doi: 10.1038/nrg999.  [DOI](https://doi.org/10.1038/nrg999) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12560807/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Genet&title=Signatures%20of%20natural%20selection%20in%20the%20human%20genome&author=M%20Bamshad&author=SP%20Wooding&volume=4&publication_year=2003&pages=99-111&pmid=12560807&doi=10.1038/nrg999&)

26. Wang Z, Wang J, Tantoso E, Wang B, Tai AY, Ooi LL, et al. Signatures of recent positive selection at the ATP-binding cassette drug transporter superfamily gene loci. Hum Mol Genet. 2007;16:1367–80. doi: 10.1093/hmg/ddm087.  [DOI](https://doi.org/10.1093/hmg/ddm087) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17412754/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mol%20Genet&title=Signatures%20of%20recent%20positive%20selection%20at%20the%20ATP-binding%20cassette%20drug%20transporter%20superfamily%20gene%20loci&author=Z%20Wang&author=J%20Wang&author=E%20Tantoso&author=B%20Wang&author=AY%20Tai&volume=16&publication_year=2007&pages=1367-80&pmid=17412754&doi=10.1093/hmg/ddm087&)

27. Li XP, Yin JY, Wang Y, He H, Li X, Gong WJ, et al. The ATP7B genetic polymorphisms predict clinical outcome to platinum-based chemotherapy in lung cancer patients. Tumour Biol. 2014;35:8259–65. doi: 10.1007/s13277-014-2072-0.  [DOI](https://doi.org/10.1007/s13277-014-2072-0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24852429/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Tumour%20Biol&title=The%20ATP7B%20genetic%20polymorphisms%20predict%20clinical%20outcome%20to%20platinum-based%20chemotherapy%20in%20lung%20cancer%20patients&author=XP%20Li&author=JY%20Yin&author=Y%20Wang&author=H%20He&author=X%20Li&volume=35&publication_year=2014&pages=8259-65&pmid=24852429&doi=10.1007/s13277-014-2072-0&)

28. Wang Y, Yin JY, Li XP, Chen J, Qian CY, Zheng Y, et al. The association of transporter genes polymorphisms and lung cancer chemotherapy response. PLoS One. 2014;9:e91967. doi: 10.1371/journal.pone.0091967.  [DOI](https://doi.org/10.1371/journal.pone.0091967) | [PMC free article](/articles/PMC3958404/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24643204/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=The%20association%20of%20transporter%20genes%20polymorphisms%20and%20lung%20cancer%20chemotherapy%20response&author=Y%20Wang&author=JY%20Yin&author=XP%20Li&author=J%20Chen&author=CY%20Qian&volume=9&publication_year=2014&pages=e91967&pmid=24643204&doi=10.1371/journal.pone.0091967&)

29. Wang Y, Li XP, Yin JY, Zhang Y, He H, Qian CY, et al. Association of HMGB1 and HMGB2 genetic polymorphisms with lung cancer chemotherapy response. Clin Exp Pharmacol Physiol. 2014;41:408–15. doi: 10.1111/1440-1681.12232.  [DOI](https://doi.org/10.1111/1440-1681.12232) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24684392/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Exp%20Pharmacol%20Physiol&title=Association%20of%20HMGB1%20and%20HMGB2%20genetic%20polymorphisms%20with%20lung%20cancer%20chemotherapy%20response&author=Y%20Wang&author=XP%20Li&author=JY%20Yin&author=Y%20Zhang&author=H%20He&volume=41&publication_year=2014&pages=408-15&pmid=24684392&doi=10.1111/1440-1681.12232&)

30. Chaudhuri J, Chakrabarti A, Maitra U. Biochemical characterization of mammalian translation initiation factor 3 (eIF3). Molecular cloning reveals that p110 subunit is the mammalian homologue of Saccharomyces cerevisiae protein Prt1. J Biol Chem. 1997;272:30975–83. doi: 10.1074/jbc.272.49.30975.  [DOI](https://doi.org/10.1074/jbc.272.49.30975) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9388245/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=Biochemical%20characterization%20of%20mammalian%20translation%20initiation%20factor%203%20(eIF3).%20Molecular%20cloning%20reveals%20that%20p110%20subunit%20is%20the%20mammalian%20homologue%20of%20Saccharomyces%20cerevisiae%20protein%20Prt1&author=J%20Chaudhuri&author=A%20Chakrabarti&author=U%20Maitra&volume=272&publication_year=1997&pages=30975-83&pmid=9388245&doi=10.1074/jbc.272.49.30975&)

31. Olson JE, Wang X, Goode EL, Pankratz VS, Fredericksen ZS, Vierkant RA, et al. Variation in genes required for normal mitosis and risk of breast cancer. Breast Cancer Res Treat. 2010;119:423–30. doi: 10.1007/s10549-009-0386-1.  [DOI](https://doi.org/10.1007/s10549-009-0386-1) | [PMC free article](/articles/PMC2818059/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19377877/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Breast%20Cancer%20Res%20Treat&title=Variation%20in%20genes%20required%20for%20normal%20mitosis%20and%20risk%20of%20breast%20cancer&author=JE%20Olson&author=X%20Wang&author=EL%20Goode&author=VS%20Pankratz&author=ZS%20Fredericksen&volume=119&publication_year=2010&pages=423-30&pmid=19377877&doi=10.1007/s10549-009-0386-1&)
