# Pharmacogenetic characterization of naturally occurring germline NT5C1A variants to chemotherapeutic nucleoside analogs

## Metadata
**Authors:** Jason Saliba, Ryan Zabriskie, Rajarshi Ghosh, Bradford C Powell, Stephanie Hicks, Marek Kimmel, Qingchang Meng, Deborah I Ritter, David A Wheeler, Richard A Gibbs, Francis T F Tsai, Sharon E Plon
**Journal:** Pharmacogenetics and genomics
**Date:** 2016 Jun
**DOI:** [10.1097/FPC.0000000000000208](https://doi.org/10.1097/FPC.0000000000000208)
**PMID:** 26906009
**PMCID:** PMC4853247
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4853247/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC4853247/pdf/nihms-755133.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC4853247/pdf/nihms-755133.pdf)

## Abstract

**Background:** 
Mutations or alteration in expression of the 5’ nucleotidase gene family can confer altered responses to treatment with nucleoside analogs. While investigating leukemia susceptibility genes, we discovered a very rare p.L254P NT5C1A missense variant in the substrate recognition motif. Given the paucity of cellular drug response data from NT5C1A germline variation, we characterized p.L254P and eight rare variants of NT5C1A from genomic databases.

**Methods:** 
Through lentiviral infection, we created HEK293 cell lines that stably overexpress wildtype NT5C1A, p.L254P, or eight NT5C1A variants reported in the NHLBI Exome Variant server (one truncating and seven missense). IC50 values were determined by cytotoxicity assays after exposure to chemotherapeutic nucleoside analogs (Cladribine, Gemcitabine, 5-Fluorouracil). In addition, we used structure-based homology modeling to generate a 3D model for the C-terminal region of NT5C1A.

**Results:** 
The p.R180X (truncating), p.A214T, and p.L254P missense changes were the only variants that significantly impaired protein function across all nucleotide analogs tested (>5-fold difference versus WT; p<.05). Several of the remaining variants individually displayed differential effects (both more and less resistant) across the analogs tested. The homology model provided a structural framework to understand the impact of NT5C1A mutants on catalysis and drug processing. The model predicted active site residues within NT5C1A motif III and we experimentally confirmed that p.K314 (not p.K320) is required for NT5C1A activity.

**Conclusion:** 
We characterized germline variation and predicted protein structures of NT5C1A. Individual missense changes showed substantial variation in response to the different nucleoside analogs tested, which may impact patients’ responses to treatment.

Keywords: nucleoside analogs, cytosolic 5’ nucleotidase IA, pharmacogenetics, functional testing, cytotoxicity assays, germline variants

### Background

Mutations or alteration in expression of the 5’ nucleotidase gene family can confer altered responses to treatment with nucleoside analogs. While investigating leukemia susceptibility genes, we discovered a very rare p.L254P NT5C1A missense variant in the substrate recognition motif. Given the paucity of cellular drug response data from *NT5C1A* germline variation, we characterized p.L254P and eight rare variants of *NT5C1A* from genomic databases.

### Methods

Through lentiviral infection, we created HEK293 cell lines that stably overexpress wildtype NT5C1A, p.L254P, or eight NT5C1A variants reported in the NHLBI Exome Variant server (one truncating and seven missense). IC50 values were determined by cytotoxicity assays after exposure to chemotherapeutic nucleoside analogs (Cladribine, Gemcitabine, 5-Fluorouracil). In addition, we used structure-based homology modeling to generate a 3D model for the C-terminal region of NT5C1A.

### Results

The p.R180X (truncating), p.A214T, and p.L254P missense changes were the only variants that significantly impaired protein function across all nucleotide analogs tested (>5-fold difference versus WT; p<.05). Several of the remaining variants individually displayed differential effects (both more and less resistant) across the analogs tested. The homology model provided a structural framework to understand the impact of NT5C1A mutants on catalysis and drug processing. The model predicted active site residues within NT5C1A motif III and we experimentally confirmed that p.K314 (not p.K320) is required for NT5C1A activity.

### Conclusion

We characterized germline variation and predicted protein structures of *NT5C1A.* Individual missense changes showed substantial variation in response to the different nucleoside analogs tested, which may impact patients’ responses to treatment.

## Introduction

Nucleoside analogs are modified purines or pyrimidines, which induce cell death through a variety of mechanisms and are used in the treatment of a variety of cancers and other diseases. Cellular enzymes activate most nucleoside analogs through serial phosphorylation [[1](#R1)]. Conversely, 5’ nucleotidases dephosphorylate the monophosphate form of the analog followed by cellular export of the unphosphorylated analog. 5’ nucleotidases vary in their expression patterns and affinities for different substrates and thus may act differentially on nucleoside analog medications [[2](#R2)]. Polymorphisms altering mRNA expression of two 5’nucleotidases, *NT5C2* or *NT5C3A*, correlate with sensitivity of hematologic cells to cytosine analogs [[3](#R3),[4](#R4)] Activating somatic mutations of *NT5C2*, which confer resistance to thiopurine analogs, were found in acute lymphoblastic leukemia (ALL) patient samples at relapse [[5](#R5),[6](#R6)].

NT5C1A assists in the maintenance of metabolite levels within the adenosine pathway ([Fig. 1a](#F1)). NT5C1A functions as a tetramer with AMP as the primary substrate [[2](#R2),[7](#R7)]. Experimental overexpression of NT5C1A in a variety of human and murine cell lines results in increased cellular resistance to a variety of nucleoside analogs [[7](#R7),[8](#R8)]. The catalogue of somatic mutation in cancer (COSMIC) reports 308 tumor samples with *NT5C1A* overexpression and none having underexpression; similar ratios are observed in other 5’ nucleotidases [[9](#R9),[10](#R10)]. Additionally, the cBioPortal database reports *NT5C1A* gene amplification in a wide range of cancers [[11](#R11)]. *NT5C1A* (at chromosome 1p34) is part of an amplicon containing *MYCL1,* a member of the *MYC* oncogene family amplified in lung and other cancers [[12](#R12),[13](#R13)].

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1864/4853247/e15a5654e372/nihms-755133-f0001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4853247_nihms-755133-f0001.jpg)

(a) Schematic of NT5C1A in the adenosine metabolic pathway. NT5C1A is primarily responsible for assisting in the maintenance and balance of the adenosine metabolic pathway through the dephosphorylation of AMP to adenosine. NT5C1A can also process monophosphated nucleoside analogs. (b) Schematic of NT5C1A motifs and location of missense variants selected for functional analysis. Variants within one of the three motifs of NT5C1A were selected for functional characterization due the high conservation and predicted functional role of these motifs in addition to the one truncating mutation (p.R180X) and two rare missense changes (p.R111W and p.I275V) outside these motifs as controls. *Motif III was previously predicted to begin at p.K320, but our data indicates the motif should be expanded to begin at p.K314. (c) Table of germline missense variants of NT5C1A found in the NHLBI Exome Variant Server (EVS) selected for functional analysis. The Minor Allele Frequency percentage (MAF%) for each variant as reported by the NHLBI EVS and the ExAC databases.

Our laboratory performed whole exome sequencing to identify mutations underlying familial leukemia/lymphoma susceptibility [[14](#R14)]. We identified a rare unreported (dbSNP [[15](#R15)] or the NHLBI Exome variant server [[16](#R16)]) germline missense variant of *NT5C1A*, p.L254P, segregating with cancer in one kindred. Initial functional assays ([Supplementary Fig. 1](#SD5)) suggested the p.L254P mutation negatively impacted NT5C1A protein function.

Given the lack of functional and molecular evaluation of the naturally occurring *NT5C1A* missense variants, we performed a quantitative functional and structural assessment of an allelic series of missense variants in *NT5C1A*, which demonstrate the differential impact of missense mutations on processing of nucleoside analogs. Individuals carrying these rare alleles may represent unusual responders to chemotherapeutic nucleoside analog agents.

## Methods

### Creation of myc-NT5C1A pLenti6.3 vectors

A myc-NT5C1A wild-type (WT) cDNA sequence underwent site directed mutagenesis to create 11 different mutations using the QuikChange Lightning kit (Agilent Technologies, Santa Clara, California, USA) and primers (Sigma-Genosys, Woodlands, Texas, USA). All plasmids were harvested through the Qiagen miniprep kit (Qiagen Inc., Valencia, California, USA) and the *NT5C1A* mutations confirmed by Sanger sequencing (Eurofins MWG Operon, Huntsville, Alabama, USA).

WT and variant *myc-NT5C1A* cDNAs were amplified by PCR with primers containing *attB* sites on the 5’ and 3’ ends and transferred using Invitrogen's Gateway® system and BP/LR II recombinase enzymes into pDONR221 vectors and ultimately into the pLenti6.3 destination vector. Vector recombination was confirmed based on *BsrgI* (New England Biolabs, Ipswich, Massachusetts, USA) restriction digest and by Sanger sequencing.

### Creation of myc-NT5C1A wildtype and variant overexpression cell lines

HEK293 cells grown in IDMEM (Invitrogen, Carlsbad, California, USA) and 10% FBS (Thermo Scientific, Waltham, Massachusetts, USA) were transfected with pLenti-6.3-myc-NT51A WT or variant vectors, pDM2D, and pΔ874 (4ug DNA total) using Lipofectamine 2000 (Invitrogen) and Opti-MEM (Invitrogen).

Lentivirus was harvested four times over 72 hours after transfection and used to infect HEK293s for 24 hours. Infected HEK293s were maintained in IDMEM +10% FBS and 5.5 ug/ml blasticidin S HCL (Invitrogen) throughout subsequent assays. Confirmation of stable overexpression by Western blot is described in detail in the [Supplementary Methods](#SD4).

### Cytotoxicity assays

Stable cell lines were plated 2.025×10^3 cells/well in 96 well plates using either the Beckman 1000 automated work station or the Eppendorf epMotion 5075 (Eppendorf, Hamburg, Germany). Twenty-four hours after plating, cells were treated with varying drug concentrations (10^−3^ M to 10^−10^ M or 10^−4^ M to 10^−11^ M) of individual nucleoside analogs (Cladribine, Gemcitabine, Fluorouracil (5-FU)) for 96 hours at 37°C. Viability was then assessed by exposure to 3-(4,5-Dimethylthiazol-2yl)-2,5-diphenyletetrazolium bromide (MTT) (Sigma-Aldrich, St. Louis, Missouri, USA) as previously described [[17](#R17)]. Data from six wells per drug concentration for each run were averaged and plotted in cytotoxicity curves (Kaleidagraph 4.5, Synergy Software, Reading, Pennsylvania, USA). IC50 values were calculated as the concentration of drug necessary to produce 50% inhibition of cell growth compared to untreated control cells. Median IC50 values for each experiment are provided in the [supplementary information](#SD4) ([Supplementary Tables 1](#SD1)-[3](#SD3)).

### Statistical Analysis

Analysis of variance (ANOVA) was used to assess significant difference in log_10_ median IC50 values between variant and WT stable lentiviral overexpression cell lines for each drug. Next, a post hoc Tukey Honest Significant Difference (HSD) test was performed on the pairwise comparisons to identify variant line response differing significantly from the WT response. The analyses and plots were generated using R Statistical software and the packages ggplot2 and plyr [[18](#R18)].

### Generation of a 3D NT5C1A homology-based structural model

A search of the RCSC Protein Data Bank (PDB) for proteins of known 3D structure using Protein BLAST did not yield any significant homologous structure to NT5C1A [[19](#R19)]. We used HHpred, which implements pairwise comparison of profile hidden Markov models, to identify more distant structural homologs of human NT5C1A [[20](#R20)] which revealed several members of the bacterial hydrolase superfamily ([Supplementary Fig. 2](#SD6)). The top solution was found with the periplasmic acid phosphatase, AphA (PDB ID: 2B8J), a class B bacterial phosphatase belonging to the DDDD superfamily of phosphohydrolases [[21](#R21)]. To generate a 3D model for human NT5C1A, the top solution of HHpred was used as input into Modeller [[22](#R22)]. NT5C1A mutants were generated *in silico* followed by real-space refinement in COOT to optimize the geometry [[23](#R23)]. Ribbon diagrams of NT5C1A wildtype and variants were generated using PyMOL (Schrödinger, New York, New York, USA) [[24](#R24)].

## Results

### Selection of NT5C1A sequence variants found in the general population

Following the discovery of a very rare p.L254P missense variant segregating with cancer in a single pediatric leukemia and lymphoma kindred, we performed initial studies that demonstrated the p.L254P mutant NT5C1A protein had no activity against a wide variety of nucleoside analogs tested ([Supplementary Methods](#SD4) and [Supplementary Fig. 1](#SD5)). We performed a systematic evaluation of missense variants in publically available databases. NT5C1A is evolutionary conserved, specifically within Motifs I and III, responsible for the protein's catalytic activity, and Motif S, responsible for substrate recognition [[25](#R25)]. The National Heart Lung and Blood Institute (NHLBI) Exome Variant server (EVS) contains whole exome sequencing data of 6500 individuals (both European and African American descent) from multiple non-cancer cohorts [[16](#R16)]. The NHLBI EVS data demonstrate little variation with only 25 rare (less than 1% minor allele frequency (MAF)) missense mutations and a single putative loss of function mutation (nonsense p.R180X) suggesting preservation of NT5C1A function. We selected the five rare variants within the conserved motifs (p.A214T, p.P244L, p.R322H, p.I325V, p.F326Y), p.I275V located between Motif S and Motif III, the R111W variant in the N-terminal region, the rare p.L254P variant from the kindred, and the sole truncating mutation (p.R180X) to study ([Fig. 1b-c](#F1)).

*NT5C1A* cDNA encoding WT, p.L254P, or one of the eight selected NHLBI EVS variants were stably expressed in HEK293 cell lines through the use of pLenti6.3 vectors and subsequent lentiviral infection. Confirmation of each NT5C1A protein expression was assessed by Western blot analysis ([Supplementary Fig. 3](#SD7)). Only the truncating variant showed diminished NT5C1A expression.

### Variants within the functional motifs of NT5C1A result in variable cytotoxic responses to nucleoside analogs

We performed cytotoxicity assays (see Methods) of this *NT5C1A* allelic series of stable lentiviral infected HEK293 cells to assess pharmacogenetic responses against three nucleoside analogs commonly used in cancer treatments (Cladribine, Gemcitabine, and 5-Flourouracil). Assays were performed in duplicate at cell passages three and five. Statistical significance (95% confidence interval) was determined for each mutant line compared to cells overexpressing WT protein ([Fig. 2 a-d](#F2)).

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1864/4853247/b4aaf025da08/nihms-755133-f0002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4853247_nihms-755133-f0002.jpg)

Analysis of WT and variant NT5C1A lentiviral stable overexpression cell lines. (a-c) Median IC50 dot plots for the three nucleoside analogs tested [(a) Cladribine, (b) Gemcitabine, (c) 5-Fluorouracil] against stable WT, GFP and mutant NT5C1A overexpression cell lines. (d) Confidence Intervals of activity when compared with WT NT5C1A. ANOVA analysis followed by TukeyHSD post-hoc testing was used to determine if the median IC50 value of each variant was significantly different from the median IC50 value of the WT cell line. The lines emanating horizontally from the individual dots indicate the 95% confidence interval of the IC50 values for each mutant profiled (n=2-4 replicate experiments). Plots that do not contact the vertical dotted line at zero are statistically significantly better (right) or worse (left) at processing the nucleoside analog indicated (p<0.5). Plots that intersect the zero line are not statistically different from the WT overexpression line in their ability to process the analog.

As expected, cell lines containing either GFP or the p.R180X nonsense mutation were the most sensitive (25-50 fold lower IC50) to all three nucleoside when compared to those overexpressing WT NT5C1A (p<.0001). Cell lines expressing the p.A214T and p.L254P missense variants were the only other lines that displayed significantly reduced NT5C1A activity against all three nucleoside analogs (Cladribine (>7 fold, p<.0001;), Gemcitabine (>5 fold, p<.04), and 5-FU (>6 fold, p<.0001).

In contrast, the remaining NT5C1A missense variants showed differential sensitivity to the analogs tested. Cells overexpressing p.P244L mutant NT5C1A had the most disparate cytotoxicity response profiles compared to WT including only a twofold (non-significant) increased resistance to Cladribine, an analog of adenosine the primary NT5C1A substrate. Conversely, the p.P244L overexpressing cell line demonstrated significant loss of processing of Gemcitabine (6.3-fold, p<.02) and 5-FU (6.3-fold p<.0002). The p.P244L cell line loss of activity against these two agents was similar to the p.A214T and p.L254P lines, despite maintained activity against Cladribine.

The p.I325V and p.F326Y adjacent missense variants presented similarly disparate resistance patterns across analogs, but with lower overall impact. The p.I325V and p.F326Y lines were >5-fold (p<.001) more resistant to Cladribine, but only demonstrated a slight increase in processing of 5-FU. Conversely, p.I325V and p.F326Y lines were 1.8- and 3.3-fold, respectively, more sensitive (although not statistically significant) to Gemcitabine than the WT line.

The p.I275V and p.R322H NT5C1A overexpressing lines showed no statistically significant difference (<2-fold difference) in response to the three analogs tested. The more common p.R111W N-terminus missense variant outside the conserved motifs also processed Gemcitabine and 5-FU treatments similarly to WT. Unexpectedly, the p.R111W cell lines demonstrated >4-fold (p<.001) resistance to Cladribine.

Overall, assessment of the nine NT5C1A variants found in the general population revealed a diverse cytotoxic profile in response to the nucleoside analogs tested with multiple variants demonstrating increased processing (as measured by increased resistance) to Cladribine compared to WT, but diminished processing of other analogs.

### 3D homology model of NT5C1A and expansion of Motif III

The atomic structure of NT5C1A remains elusive. Thus, as described in detail in the Methods, we generated an *in silico* 3D homology model for the C-terminal region of NT5C1A (residues 171-368), which contain all three conserved functional motifs. This model allows for visualization of the missense variants studied here and for inferences to be made based on the other intracellular 5’ nucleotidases ([Fig. 3](#F3)). Previous structural studies of intracellular 5’ nucleotidase family members show the catalytic core is formed by two aspartates in Motif I and two aspartates in Motif III oriented near each other [[25](#R25)–[29](#R29)]. Our homology model for NT5C1A suggests a similar catalytic core composed of p.D211, p.D213, p.D328, and p.D329 (blue). Prior analyses also include a lysine at the N terminal end of Motif III (proposed to be p.K320 for NT5C1A) interacting with the bound substrate's phosphate group [[2](#R2),[25](#R25),[29](#R29)]. However, our structural model predicts p.K320 (light blue) to be remote from the catalytic core, whereas p.K314 (red), which is evolutionarily conserved, is predicted to be more proximal and is more likely to provide the protons needed to stabilize the NT5C1A dephosphorylation reaction ([Fig. 4a](#F4)).

### Figure 3.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1864/4853247/0ffef16e8883/nihms-755133-f0003.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4853247_nihms-755133-f0003.jpg)

Predicted NT5C1A 3D homology model. This predicted 3D homology model of NT5C1A generated by HHpred is based on the structure of Apha, a class B bacterial phosphatase found in E. coli. NT5C1A is modeled from residue 171 to the C-terminal residue, 368. The model predicts the aspartates from Motif I (p.D211 & p.D213) and Motif III (p.D328 and p.D329) (blue) to be oriented near each other, which would allow these four residues to act as integral members of the enzyme's catalytic core. Each variant tested within the C-terminal region of NT5C1A for impact on cytotoxic response is pictured in the model along with their side chain to portray their predicted location versus the catalytic core. p.A214 (orange) is located adjacent to one of the main core residues. p.P244 (green) and p.L254 (black) are positioned on the surface of the protein, but are located at opposite ends of the substrate recognition motif. p.I275 (purple) and p.R322 (magenta) are located on the surface of the protein away from the core. p.I325 (brown) and p.F326 (lavender) are located in a β-strand near the catalytic core of NT5C1A.

### Figure 4.

![Figure 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1864/4853247/f1b27803dc29/nihms-755133-f0004.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4853247_nihms-755133-f0004.jpg)

(a) p.K314 and p.K320 location versus the core. The 3D homology model predicts p.K320 (light blue) to be remote from the catalytic core (blue); however p.K314 (red) is predicted to be closer to the core and thus the p.K314 side chain could provide the protons used to stabilize the removal of the phosphate from the substrate. (b) Analysis of lysine residues potentially involved in the catalytic core of NT5C1A. 95% confidence intervals of K314E, K320E, or GFP (control) overexpression cell lines’ IC50s compared to WT derived cell viability assays following chemotherapy treatments. Statistical analysis as described in Fig. 2d.

HEK293 cell lines that stably overexpressed NT5C1A and substituted glutamic acid for lysine resulting in p.K314E and p.K320E mutant forms of NT5C1A were created and underwent cytotoxicity assays, as described previously ([Fig. 4b](#F4) and [Supplementary Fig. 4](#SD8)). The p.K320E overexpression line responded similarly to the WT overexpression line (no statistical difference) after exposure to both Cladribine and Gemcitabine. In contrast, the p.K314E overexpression line resulted in a median IC50 to both analogs that was two logs lower compared to the WT overexpression line (p<.0001) and was comparable to the GFP negative control. These results suggest p.K320 is not needed for catalysis and p.K314 is the required lysine in Motif III of NT5C1A.

## Discussion

Nucleoside analogs, used to treat a variety of cancer types, are prodrugs that become activated through serial phosphorylation [[1](#R1)]. The 5’ nucleotidase enzymes balance nucleoside concentrations through the dephosphorylation of nucleoside monophosphates and their analogs. Various 5’ nucleotidases have been linked to enhancing cellular resistance to nucleoside analog drugs [[2](#R2)–[6](#R6),[30](#R30)].

Prior work has shown overexpression of WT NT5C1A within HEK293 and Jurkat cell lines can increase resistance to a range of nucleoside analogs [[7](#R7)] and multiple data sets demonstrate *NT5C1A* overexpression in a variety of cancer types [[10](#R10)–[13](#R13),[31](#R31)]. Thus, genetic variation of *NT5C1A* could have pharmacogenetic impact on the protein's ability to process nucleoside analogs.

We describe here the first characterization of an allelic series of *NT5C1A* variants in response to several nucleoside analogs. We created multiple HEK293 cell lines that stably overexpressed WT NT5C1A, the rare p.L254P variant found to segregate in a pediatric cancer kindred, one truncating mutation, and seven missense variants found in the NHLBI EVS database of non-cancer cohorts. To rigorously assess the differences in analog response, we utilized an ANOVA followed by a Tukey post hoc test to generate IC50 95% confidence intervals of the mean differences of replicate experiments compared to WT NT5C1A.

Changes in cytotoxicity from these single amino acid substitutions suggest either complete loss of activity or altered substrate specificity. The p.R180X truncating mutation and p.A214T and p.L254P missense mutations were the only variants to negatively affect NT5C1A function across all three drugs, although p.A214T and p.L254P retain limited function in processing Cladribine and Gemcitabine compared to negative controls. Our homology model suggests p.A214T and p.L254P are likely to impair drug metabolism through different means. The p.A214T residue, adjacent to p.D213, could alter the removal of the phosphate from the substrate. Conversely, the p.L254P variant, on the surface of the protein, may unravel an α-helix (aa 251-262) eliminating substrate recognition/binding or preventing disassociation of the hydrolyzed substrate.

The other six variants displayed differential effects across the three chemotherapeutic drugs, highlighting that pharmacogenomic responses should not be predicted based on analysis of a single nucleoside analog. For example, p.P244L showed complete loss of function when treated with 5-FU, retained limited residual activity after Gemcitabine treatment, but responded similarly to WT when treated with Cladribine. This difference in response is consistent with the structural model prediction that p.P244L (at the other end of the substrate recognition motif from p.L254P) may reduce structural flexibility and alter nucleoside substrate recognition, thereby enhancing recognition and/or binding of Cladribine, but diminishing recognition of Gemcitabine and 5-FU. Thus an individual with the p.P244L NT5C1A allele would most likely show higher levels of the active forms of Gemcitabine or 5-FU, although not Cladribine.

No variant enhanced protein activity across all three nucleoside analogs, but three (p.R111W, p.I325V, p.F326Y) significantly enhanced the ability of NT5C1A to process Cladribine. Residues p.I325V and p.F326V are in close proximity to and could modify the catalytic core conformation or alter bonding within the β-sheet consistent with improved processing of dAMP and dUMP analogs, but hindered processing of the dCMP analog. Overall, the widest dynamic range across the missense variants was found in processing of Cladribine, the analog of a primary nucleoside substrate of NT5C1A.

The remaining missense mutations contained in the model are minor and/or conservative mutations on the surface of the protein away from the catalytic core or substrate recognition residues consistent with the lack of significant change in response to the analogs tested.

Our homology model shows NT5C1A has a catalytic core with four aspartates resembling that of other intracellular 5’ nucleotidases. However, our model and subsequent experiments demonstrate p.K314 and not p.K320 is absolutely required for NT5C1A activity. Thus, Motif III of NT5C1A should be expanded to include p.K314 ([Fig. 5](#F5)).

### Figure 5.

![Figure 5](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1864/4853247/7727c8c1e8d1/nihms-755133-f0005.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4853247_nihms-755133-f0005.jpg)

Revised conserved motifs of 5’ nucleotidases. The intracellular 5’ nucleotidases share three highly conserved functional motifs as identified by solved crystal structures and predicted by sequence analyses to date [2,25–29]. Residues in Motif I and III are involved in catalysis and residues in Motif S are involved in substrate recognition. The consensus sequence for each motif among the gene family is shaded grey and the numbers represent the amino acid start and end positions for each motif. The intracellular 5’ nucleotidases’ Motif III sequence begins with the lysine involved in the stabilization of the nucleophilic reaction that takes place in the protein's catalytic core. Based on the cytotoxicity data (Fig 4 and Supplementary Fig. 4) the Motif III sequence of NT5C1A is expanded to begin at p.K314, not p.K320.

The Broad Exome Aggregation Consortium (ExAC) ([www.exac.broadinstitute.org](http://www.exac.broadinstitute.org)), has now aggregated whole exome data of 60,706 individuals from a spectrum of cohorts [[32](#R32)]. ExAC contains 138 *NT5C1A* missense mutations, none with a MAF% >0.2% and the majority found in a single individual with only six rare truncating mutations. This much larger data set clearly displays conservation of *NT5C1A* function. However, the rare individual with a deleterious *NT5C1A* allele may represent unique responders to specific nucleoside analogs within large clinical trials.

## Supplementary Material

## Acknowledgements

We thank the laboratory of Kenneth Scott, PhD at Baylor College of Medicine for providing vectors and expertise on development of the lentiviral transduced cell lines.

Funding Source: Cancer Prevention and Research Institute of Texas RP 101089 (S.E.P) and National Institutes of Health grants, R01-CA138836 (S.E.P), R01-GM104980 (F.T.), R01-GM111084 (F.T.) and K12-GM084897 (D.I.R.)

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Galmarini CM, Mackey JR, Dumontet C. Nucleoside analogues and nucleobases in cancer treatment. Lancet Oncol. 2002;3:415–24. doi: 10.1016/s1470-2045(02)00788-x.  [DOI](https://doi.org/10.1016/s1470-2045(02)00788-x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12142171/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet%20Oncol&title=Nucleoside%20analogues%20and%20nucleobases%20in%20cancer%20treatment.&author=CM%20Galmarini&author=JR%20Mackey&author=C%20Dumontet&volume=3&publication_year=2002&pages=415-24&pmid=12142171&doi=10.1016/s1470-2045(02)00788-x&)

2. Hunsucker SA, Mitchell BS, Spychala J. The 5′-nucleotidases as regulators of nucleotide and drug metabolism. Pharmacol. Ther. 2005;107:1–30. doi: 10.1016/j.pharmthera.2005.01.003.  [DOI](https://doi.org/10.1016/j.pharmthera.2005.01.003) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15963349/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacol.%20Ther&title=The%205%E2%80%B2-nucleotidases%20as%20regulators%20of%20nucleotide%20and%20drug%20metabolism.&author=SA%20Hunsucker&author=BS%20Mitchell&author=J%20Spychala&volume=107&publication_year=2005&pages=1-30&pmid=15963349&doi=10.1016/j.pharmthera.2005.01.003&)

3. Mitra AK, Crews KR, Pounds S, Cao X, Feldberg T, Ghodke Y, et al. Genetic variants in cytosolic 5’-nucleotidase II are associated with its expression and cytarabine sensitivity in HapMap cell lines and in patients with acute myeloid leukemia. J. Pharmacol. Exp. Ther. [Internet] 2011;339:9–23. doi: 10.1124/jpet.111.182873. Available from: http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=3186292&tool=pmcentrez&rendertype=abstract.  [http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=3186292&tool=pmcentrez&rendertype=abstract](http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=3186292&tool=pmcentrez&rendertype=abstract) | [DOI](https://doi.org/10.1124/jpet.111.182873) | [PMC free article](/articles/PMC3186292/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21712425/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Pharmacol.%20Exp.%20Ther.%20%5BInternet%5D&title=Genetic%20variants%20in%20cytosolic%205%E2%80%99-nucleotidase%20II%20are%20associated%20with%20its%20expression%20and%20cytarabine%20sensitivity%20in%20HapMap%20cell%20lines%20and%20in%20patients%20with%20acute%20myeloid%20leukemia.&author=AK%20Mitra&author=KR%20Crews&author=S%20Pounds&author=X%20Cao&author=T%20Feldberg&volume=339&publication_year=2011&pages=9-23&pmid=21712425&doi=10.1124/jpet.111.182873&)

4. Li A, Fridley B, Kalari K, Jenkins G, Batzler A, Safgren S, et al. Gemcitabine and cytosine arabinoside cytotoxicity: Association with lymphoblastoid cell expression. Cancer Res. 2008;68:7050–8. doi: 10.1158/0008-5472.CAN-08-0405.  [DOI](https://doi.org/10.1158/0008-5472.CAN-08-0405) | [PMC free article](/articles/PMC2562356/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18757419/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=Gemcitabine%20and%20cytosine%20arabinoside%20cytotoxicity:%20Association%20with%20lymphoblastoid%20cell%20expression.&author=A%20Li&author=B%20Fridley&author=K%20Kalari&author=G%20Jenkins&author=A%20Batzler&volume=68&publication_year=2008&pages=7050-8&pmid=18757419&doi=10.1158/0008-5472.CAN-08-0405&)

5. Tzoneva G, Perez-Garcia A, Carpenter Z, Khiabanian H, Tosello V, Allegretta M, et al. Activating mutations in the NT5C2 nucleotidase gene drive chemotherapy resistance in relapsed ALL. Nat. Med. [Internet] 2013;19:368–71. doi: 10.1038/nm.3078. Available from: http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=3594483&tool=pmcentrez&rendertype=abstract.  [http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=3594483&tool=pmcentrez&rendertype=abstract](http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=3594483&tool=pmcentrez&rendertype=abstract) | [DOI](https://doi.org/10.1038/nm.3078) | [PMC free article](/articles/PMC3594483/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23377281/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Med.%20%5BInternet%5D&title=Activating%20mutations%20in%20the%20NT5C2%20nucleotidase%20gene%20drive%20chemotherapy%20resistance%20in%20relapsed%20ALL.&author=G%20Tzoneva&author=A%20Perez-Garcia&author=Z%20Carpenter&author=H%20Khiabanian&author=V%20Tosello&volume=19&publication_year=2013&pages=368-71&pmid=23377281&doi=10.1038/nm.3078&)

6. Meyer J a, Wang J, Hogan LE, Yang JJ, Dandekar S, Patel JP, et al. Relapse-specific mutations in NT5C2 in childhood acute lymphoblastic leukemia. Nat. Genet. [Internet] 2013;45:290–4. doi: 10.1038/ng.2558. Available from: http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=3681285&tool=pmcentrez&rendertype=abstract.  [http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=3681285&tool=pmcentrez&rendertype=abstract](http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=3681285&tool=pmcentrez&rendertype=abstract) | [DOI](https://doi.org/10.1038/ng.2558) | [PMC free article](/articles/PMC3681285/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23377183/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet.%20%5BInternet%5D&title=Relapse-specific%20mutations%20in%20NT5C2%20in%20childhood%20acute%20lymphoblastic%20leukemia.&author=J%20a%20Meyer&author=J%20Wang&author=LE%20Hogan&author=JJ%20Yang&author=S%20Dandekar&volume=45&publication_year=2013&pages=290-4&pmid=23377183&doi=10.1038/ng.2558&)

7. Hunsucker SA, Spychala J, Mitchell BS. Human cytosolic 5′-nucleotidase I: Characterization and role in nucleoside analog resistance. J. Biol. Chem. 2001;276:10498–504. doi: 10.1074/jbc.M011218200.  [DOI](https://doi.org/10.1074/jbc.M011218200) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11133996/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem&title=Human%20cytosolic%205%E2%80%B2-nucleotidase%20I:%20Characterization%20and%20role%20in%20nucleoside%20analog%20resistance.&author=SA%20Hunsucker&author=J%20Spychala&author=BS%20Mitchell&volume=276&publication_year=2001&pages=10498-504&pmid=11133996&doi=10.1074/jbc.M011218200&)

8. Gray T, Morrey EL, Gangadharan B, Sumter TF, Spychala J, Archer DR, et al. Enforced expression of cytosolic 5’-nucleotidase I confers resistance to nucleoside analogues in vitro but systemic chemotherapy toxicity precludes in vivo selection. Cancer Chemother. Pharmacol. 2006;58:117–28. doi: 10.1007/s00280-005-0156-7.  [DOI](https://doi.org/10.1007/s00280-005-0156-7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16362297/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Chemother.%20Pharmacol&title=Enforced%20expression%20of%20cytosolic%205%E2%80%99-nucleotidase%20I%20confers%20resistance%20to%20nucleoside%20analogues%20in%20vitro%20but%20systemic%20chemotherapy%20toxicity%20precludes%20in%20vivo%20selection.&author=T%20Gray&author=EL%20Morrey&author=B%20Gangadharan&author=TF%20Sumter&author=J%20Spychala&volume=58&publication_year=2006&pages=117-28&pmid=16362297&doi=10.1007/s00280-005-0156-7&)

9. [2015 Mar 1];COSMIC [Internet] Available from: cancer.sanger.ac.uk.  [cancer.sanger.ac.uk](http://cancer.sanger.ac.uk)

10. Forbes S, Beare D, Gunasekaran P, Leung K, Bindal N, Boutselakis H, et al. COSMIC: exploring the world's knowledge of somatic mutations in human cancer. 2015 doi: 10.1093/nar/gku1075.  [DOI](https://doi.org/10.1093/nar/gku1075) | [PMC free article](/articles/PMC4383913/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25355519/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=COSMIC:%20exploring%20the%20world's%20knowledge%20of%20somatic%20mutations%20in%20human%20cancer&author=S%20Forbes&author=D%20Beare&author=P%20Gunasekaran&author=K%20Leung&author=N%20Bindal&publication_year=2015&pmid=25355519&doi=10.1093/nar/gku1075&)

11. Gao J, Aksoy BA, Dogrusoz U, Dresdner G, Gross B, Sumer SO, et al. Integrative analysis of complex cancer genomics and clinical profiles using the cBioPortal. Sci. Signal. [Internet] 2013;6:11. doi: 10.1126/scisignal.2004088. Available from: http://www.ncbi.nlm.nih.gov/pubmed/23550210.  [http://www.ncbi.nlm.nih.gov/pubmed/23550210](http://www.ncbi.nlm.nih.gov/pubmed/23550210) | [DOI](https://doi.org/10.1126/scisignal.2004088) | [PMC free article](/articles/PMC4160307/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23550210/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Sci.%20Signal.%20%5BInternet%5D&title=Integrative%20analysis%20of%20complex%20cancer%20genomics%20and%20clinical%20profiles%20using%20the%20cBioPortal.&author=J%20Gao&author=BA%20Aksoy&author=U%20Dogrusoz&author=G%20Dresdner&author=B%20Gross&volume=6&publication_year=2013&pages=11&pmid=23550210&doi=10.1126/scisignal.2004088&)

12. Iwakawa R, Takenaka M, Kohno T, Shimada Y, Totoki Y, Shibata T, et al. Genome-wide identification of genes with amplification and/or fusion in small cell lung cancer. Genes. Chromosomes Cancer. 2013;52:802–16. doi: 10.1002/gcc.22076.  [DOI](https://doi.org/10.1002/gcc.22076) | [PMC free article](/articles/PMC3806277/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23716474/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genes.%20Chromosomes%20Cancer&title=Genome-wide%20identification%20of%20genes%20with%20amplification%20and/or%20fusion%20in%20small%20cell%20lung%20cancer.&author=R%20Iwakawa&author=M%20Takenaka&author=T%20Kohno&author=Y%20Shimada&author=Y%20Totoki&volume=52&publication_year=2013&pages=802-16&pmid=23716474&doi=10.1002/gcc.22076&)

13. Bell D, Berchuck A, Birrer M, Chien J, Cramer DW, Dao F, et al. Integrated genomic analyses of ovarian carcinoma. Nature. 2011:609–15. doi: 10.1038/nature10166.  [DOI](https://doi.org/10.1038/nature10166) | [PMC free article](/articles/PMC3163504/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21720365/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Integrated%20genomic%20analyses%20of%20ovarian%20carcinoma.&author=D%20Bell&author=A%20Berchuck&author=M%20Birrer&author=J%20Chien&author=DW%20Cramer&publication_year=2011&pages=609-15&pmid=21720365&doi=10.1038/nature10166&)

14. Powell BC, Jiang L, Muzny DM, Treviño LR, Dreyer ZE, Strong LC, et al. Identification of TP53 as an acute lymphocytic leukemia susceptibility gene through exome sequencing. Pediatr. Blood Cancer. 2013:60. doi: 10.1002/pbc.24417.  [DOI](https://doi.org/10.1002/pbc.24417) | [PMC free article](/articles/PMC3926299/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23255406/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pediatr.%20Blood%20Cancer&title=Identification%20of%20TP53%20as%20an%20acute%20lymphocytic%20leukemia%20susceptibility%20gene%20through%20exome%20sequencing.&author=BC%20Powell&author=L%20Jiang&author=DM%20Muzny&author=LR%20Trevi%C3%B1o&author=ZE%20Dreyer&publication_year=2013&pages=60&pmid=23255406&doi=10.1002/pbc.24417&)

15. Sherry ST, Ward MH, Kholodov M, Baker J, Phan L, Smigielski EM, et al. dbSNP: the NCBI database of genetic variation. Nucleic Acids Res. 2001;29:308–11. doi: 10.1093/nar/29.1.308.  [DOI](https://doi.org/10.1093/nar/29.1.308) | [PMC free article](/articles/PMC29783/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11125122/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=dbSNP:%20the%20NCBI%20database%20of%20genetic%20variation.&author=ST%20Sherry&author=MH%20Ward&author=M%20Kholodov&author=J%20Baker&author=L%20Phan&volume=29&publication_year=2001&pages=308-11&pmid=11125122&doi=10.1093/nar/29.1.308&)

16. Exome Variant Server, NHLBI GO Exome Sequencing Project (ESP) Seattle, WA: [2010 Jan 20]. [Internet]. Available from: http://evs.gs.washington.edu/EVS/  [http://evs.gs.washington.edu/EVS/](http://evs.gs.washington.edu/EVS/) | [Google Scholar](https://scholar.google.com/scholar_lookup?title=Exome%20Variant%20Server,%20NHLBI%20GO%20Exome%20Sequencing%20Project%20(ESP)&)

17. Horton TM, Gannavarapu A, Blaney SM, D'Argenio DZ, Plon SE, Berg SL. Bortezomib interactions with chemotherapy agents in acute leukemia in vitro. Cancer Chemother. Pharmacol. 2006;58:13–23. doi: 10.1007/s00280-005-0135-z.  [DOI](https://doi.org/10.1007/s00280-005-0135-z) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16292537/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacol&title=Bortezomib%20interactions%20with%20chemotherapy%20agents%20in%20acute%20leukemia%20in%20vitro.%20Cancer%20Chemother.&author=TM%20Horton&author=A%20Gannavarapu&author=SM%20Blaney&author=DZ%20D'Argenio&author=SE%20Plon&volume=58&publication_year=2006&pages=13-23&pmid=16292537&doi=10.1007/s00280-005-0135-z&)

18. R Core Team . R: A language and environment for statistical computing. R Foundation for Statistical Computing. Vienna, Austria: 2014.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=R:%20A%20language%20and%20environment%20for%20statistical%20computing.%20R%20Foundation%20for%20Statistical%20Computing&publication_year=2014&)

19. Altschul SF, Gish W, Miller W, Myers EW, Lipman DJ. Basic local alignment search tool. J. Mol. Biol. 1990;215:403–10. doi: 10.1016/S0022-2836(05)80360-2.  [DOI](https://doi.org/10.1016/S0022-2836(05)80360-2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/2231712/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Mol.%20Biol&title=Basic%20local%20alignment%20search%20tool.&author=SF%20Altschul&author=W%20Gish&author=W%20Miller&author=EW%20Myers&author=DJ%20Lipman&volume=215&publication_year=1990&pages=403-10&pmid=2231712&doi=10.1016/S0022-2836(05)80360-2&)

20. Söding J, Biegert A, Lupas AN. The HHpred interactive server for protein homology detection and structure prediction. Nucleic Acids Res. 2005:33. doi: 10.1093/nar/gki408.  [DOI](https://doi.org/10.1093/nar/gki408) | [PMC free article](/articles/PMC1160169/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15980461/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=The%20HHpred%20interactive%20server%20for%20protein%20homology%20detection%20and%20structure%20prediction.&author=J%20S%C3%B6ding&author=A%20Biegert&author=AN%20Lupas&publication_year=2005&pages=33&pmid=15980461&doi=10.1093/nar/gki408&)

21. Calderone V, Forleo C, Benvenuti M, Thaller MC, Rossolini GM, Mangani S. A structure-based proposal for the catalytic mechanism of the bacterial acid phosphatase AphA belonging to the DDDD superfamily of phosphohydrolases. J. Mol. Biol. 2006;355:708–21. doi: 10.1016/j.jmb.2005.10.068.  [DOI](https://doi.org/10.1016/j.jmb.2005.10.068) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16330049/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Mol.%20Biol&title=A%20structure-based%20proposal%20for%20the%20catalytic%20mechanism%20of%20the%20bacterial%20acid%20phosphatase%20AphA%20belonging%20to%20the%20DDDD%20superfamily%20of%20phosphohydrolases.&author=V%20Calderone&author=C%20Forleo&author=M%20Benvenuti&author=MC%20Thaller&author=GM%20Rossolini&volume=355&publication_year=2006&pages=708-21&pmid=16330049&doi=10.1016/j.jmb.2005.10.068&)

22. Sali A, Blundell TL. Comparative protein modelling by satisfaction of spatial restraints. J. Mol. Biol. 1993;234:779–815. doi: 10.1006/jmbi.1993.1626.  [DOI](https://doi.org/10.1006/jmbi.1993.1626) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8254673/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Mol.%20Biol&title=Comparative%20protein%20modelling%20by%20satisfaction%20of%20spatial%20restraints.&author=A%20Sali&author=TL%20Blundell&volume=234&publication_year=1993&pages=779-815&pmid=8254673&doi=10.1006/jmbi.1993.1626&)

23. Emsley P, Lohkamp B, Scott WG, Cowtan K. Features and development of Coot. Acta Crystallogr. Sect. D Biol. Crystallogr. International Union of Crystallography. 2010;66:486–501. doi: 10.1107/S0907444910007493.  [DOI](https://doi.org/10.1107/S0907444910007493) | [PMC free article](/articles/PMC2852313/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20383002/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=International%20Union%20of%20Crystallography&title=Features%20and%20development%20of%20Coot.%20Acta%20Crystallogr.%20Sect.%20D%20Biol.%20Crystallogr.&author=P%20Emsley&author=B%20Lohkamp&author=WG%20Scott&author=K%20Cowtan&volume=66&publication_year=2010&pages=486-501&pmid=20383002&doi=10.1107/S0907444910007493&)

24. Schrodinger LLC. The PyMOL Molecular Graphics System, Version 1.7.4. 2010  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=The%20PyMOL%20Molecular%20Graphics%20System,%20Version%201.7.4&author=LLC%20Schrodinger&publication_year=2010&)

25. Rinaldo-Matthis A, Rampazzo C, Reichard P, Bianchi V, Nordlund P. Crystal structure of a human mitochondrial deoxyribonucleotidase. Nat. Struct. Biol. 2002;9:779–87. doi: 10.1038/nsb846.  [DOI](https://doi.org/10.1038/nsb846) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12352955/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Struct.%20Biol&title=Crystal%20structure%20of%20a%20human%20mitochondrial%20deoxyribonucleotidase.&author=A%20Rinaldo-Matthis&author=C%20Rampazzo&author=P%20Reichard&author=V%20Bianchi&author=P%20Nordlund&volume=9&publication_year=2002&pages=779-87&pmid=12352955&doi=10.1038/nsb846&)

26. Walldén K, Rinaldo-Matthis A, Ruzzenente B, Rampazzo C, Bianchi V, Nordlund P. Crystal structures of human and murine deoxyribonucleotidases: Insights into recognition of substrates and nucleotide analogues. Biochemistry. 2007;46:13809–18. doi: 10.1021/bi7014794.  [DOI](https://doi.org/10.1021/bi7014794) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17985935/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochemistry&title=Crystal%20structures%20of%20human%20and%20murine%20deoxyribonucleotidases:%20Insights%20into%20recognition%20of%20substrates%20and%20nucleotide%20analogues.&author=K%20Walld%C3%A9n&author=A%20Rinaldo-Matthis&author=B%20Ruzzenente&author=C%20Rampazzo&author=V%20Bianchi&volume=46&publication_year=2007&pages=13809-18&pmid=17985935&doi=10.1021/bi7014794&)

27. Walldén K, Stenmark P, Nyman T, Flodin S, Gräslund S, Loppnau P, et al. Crystal structure of human cytosolic 5′-nucleotidase II: Insights into allosteric regulation and substrate recognition. J. Biol. Chem. 2007;282:17828–36. doi: 10.1074/jbc.M700917200.  [DOI](https://doi.org/10.1074/jbc.M700917200) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17405878/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem&title=Crystal%20structure%20of%20human%20cytosolic%205%E2%80%B2-nucleotidase%20II:%20Insights%20into%20allosteric%20regulation%20and%20substrate%20recognition.&author=K%20Walld%C3%A9n&author=P%20Stenmark&author=T%20Nyman&author=S%20Flodin&author=S%20Gr%C3%A4slund&volume=282&publication_year=2007&pages=17828-36&pmid=17405878&doi=10.1074/jbc.M700917200&)

28. Bitto E, Bingman C a., Wesenberg GE, McCoy JG, Phillips GN. Structure of pyrimidine 5' nucleotidase type 1: Insight into mechanism of action and inhibition during lead poisoning. J. Biol. Chem. 2006;281:20521–9. doi: 10.1074/jbc.M602000200.  [DOI](https://doi.org/10.1074/jbc.M602000200) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16672222/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem&title=Structure%20of%20pyrimidine%205'%20nucleotidase%20type%201:%20Insight%20into%20mechanism%20of%20action%20and%20inhibition%20during%20lead%20poisoning.&author=E%20Bitto&author=C%20a.%20Bingman&author=GE%20Wesenberg&author=JG%20McCoy&author=GN%20Phillips&volume=281&publication_year=2006&pages=20521-9&pmid=16672222&doi=10.1074/jbc.M602000200&)

29. Monecke T, Buschmann J, Neumann P, Wahle E, Ficner R. Crystal structures of the novel cytosolic 5' nucleotidase IIIB explain its preference for m7GMP. PLoS One. 2014;9:1–13. doi: 10.1371/journal.pone.0090915.  [DOI](https://doi.org/10.1371/journal.pone.0090915) | [PMC free article](/articles/PMC3946280/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24603684/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=Crystal%20structures%20of%20the%20novel%20cytosolic%205'%20nucleotidase%20IIIB%20explain%20its%20preference%20for%20m7GMP.&author=T%20Monecke&author=J%20Buschmann&author=P%20Neumann&author=E%20Wahle&author=R%20Ficner&volume=9&publication_year=2014&pages=1-13&pmid=24603684&doi=10.1371/journal.pone.0090915&)

30. Cividini F, Pesi R, Chaloin L, Allegrini S, Camici M, Cros-Perrial E, et al. Biochem. Pharmacol. [Internet] Elsevier Inc.; 2015. The purine analog fludarabine acts as a cytosolic 5′-nucleotidase II inhibitor. pp. 1–6. Available from: http://linkinghub.elsevier.com/retrieve/pii/S0006295215000799.  [http://linkinghub.elsevier.com/retrieve/pii/S0006295215000799](http://linkinghub.elsevier.com/retrieve/pii/S0006295215000799) | [DOI](https://doi.org/10.1016/j.bcp.2015.01.010) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25656700/) | [Google Scholar](https://scholar.google.com/scholar_lookup?title=Biochem.%20Pharmacol.%20%5BInternet%5D&author=F%20Cividini&author=R%20Pesi&author=L%20Chaloin&author=S%20Allegrini&author=M%20Camici&publication_year=2015&)

31. Network C genome AR. Comprehensive molecular characterization of urothelial bladder carcinoma. Nature [Internet] 2014;507:315–22. doi: 10.1038/nature12965. Available from: http://www.ncbi.nlm.nih.gov/pubmed/24476821.  [http://www.ncbi.nlm.nih.gov/pubmed/24476821](http://www.ncbi.nlm.nih.gov/pubmed/24476821) | [DOI](https://doi.org/10.1038/nature12965) | [PMC free article](/articles/PMC3962515/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24476821/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature%20%5BInternet%5D&title=Comprehensive%20molecular%20characterization%20of%20urothelial%20bladder%20carcinoma.&author=AR%20Network%20C%20genome&volume=507&publication_year=2014&pages=315-22&pmid=24476821&doi=10.1038/nature12965&)

32. Exome Aggregation Consortium (ExAC) Cambridge, MA: [2015 Mar 1]. [Internet]. Available from: http://exac.broadinstitute.org.  [http://exac.broadinstitute.org](http://exac.broadinstitute.org) | [Google Scholar](https://scholar.google.com/scholar_lookup?title=Exome%20Aggregation%20Consortium%20(ExAC)&)
