# The Role of Abcb5 Alleles in Susceptibility to Haloperidol-Induced Toxicity in Mice and Humans

## Metadata
**Authors:** Ming Zheng, Haili Zhang, David L Dill, J David Clark, Susan Tu, Arielle L Yablonovitch, Meng How Tan, Rui Zhang, Dan Rujescu, Manhong Wu, Lino Tessarollo, Wilfred Vieira, Michael M Gottesman, Suhua Deng, Livia S Eberlin, Richard N Zare, Jean-Martin Billard, Jean-Pierre Gillet, Jin Billy Li, Gary Peltz
**Journal:** PLoS Medicine
**Date:** 2015 Feb 3
**DOI:** [10.1371/journal.pmed.1001782](https://doi.org/10.1371/journal.pmed.1001782)
**PMID:** 25647612
**PMCID:** PMC4315575
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4315575/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC4315575/pdf/pmed.1001782.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC4315575/pdf/pmed.1001782.pdf)

## Abstract

**Background:** 
We know very little about the genetic factors affecting susceptibility to drug-induced central nervous system (CNS) toxicities, and this has limited our ability to optimally utilize existing drugs or to develop new drugs for CNS disorders. For example, haloperidol is a potent dopamine antagonist that is used to treat psychotic disorders, but 50% of treated patients develop characteristic extrapyramidal symptoms caused by haloperidol-induced toxicity (HIT), which limits its clinical utility. We do not have any information about the genetic factors affecting this drug-induced toxicity. HIT in humans is directly mirrored in a murine genetic model, where inbred mouse strains are differentially susceptible to HIT. Therefore, we genetically analyzed this murine model and performed a translational human genetic association study.

**Methods and Findings:** 
A whole genome SNP database and computational genetic mapping were used to analyze the murine genetic model of HIT. Guided by the mouse genetic analysis, we demonstrate that genetic variation within an ABC-drug efflux transporter (Abcb5) affected susceptibility to HIT. In situ hybridization results reveal that Abcb5 is expressed in brain capillaries, and by cerebellar Purkinje cells. We also analyzed chromosome substitution strains, imaged haloperidol abundance in brain tissue sections and directly measured haloperidol (and its metabolite) levels in brain, and characterized Abcb5 knockout mice. Our results demonstrate that Abcb5 is part of the blood-brain barrier; it affects susceptibility to HIT by altering the brain concentration of haloperidol. Moreover, a genetic association study in a haloperidol-treated human cohort indicates that human ABCB5 alleles had a time-dependent effect on susceptibility to individual and combined measures of HIT. Abcb5 alleles are pharmacogenetic factors that affect susceptibility to HIT, but it is likely that additional pharmacogenetic susceptibility factors will be discovered.

**Conclusions:** 
ABCB5 alleles alter susceptibility to HIT in mouse and humans. This discovery leads to a new model that (at least in part) explains inter-individual differences in susceptibility to a drug-induced CNS toxicity.

### Background

We know very little about the genetic factors affecting susceptibility to drug-induced central nervous system (CNS) toxicities, and this has limited our ability to optimally utilize existing drugs or to develop new drugs for CNS disorders. For example, haloperidol is a potent dopamine antagonist that is used to treat psychotic disorders, but 50% of treated patients develop characteristic extrapyramidal symptoms caused by haloperidol-induced toxicity (HIT), which limits its clinical utility. We do not have any information about the genetic factors affecting this drug-induced toxicity. HIT in humans is directly mirrored in a murine genetic model, where inbred mouse strains are differentially susceptible to HIT. Therefore, we genetically analyzed this murine model and performed a translational human genetic association study.

### Methods and Findings

A whole genome SNP database and computational genetic mapping were used to analyze the murine genetic model of HIT. Guided by the mouse genetic analysis, we demonstrate that genetic variation within an ABC-drug efflux transporter *(Abcb5)* affected susceptibility to HIT. In situ hybridization results reveal that Abcb5 is expressed in brain capillaries, and by cerebellar Purkinje cells. We also analyzed chromosome substitution strains, imaged haloperidol abundance in brain tissue sections and directly measured haloperidol (and its metabolite) levels in brain, and characterized *Abcb5* knockout mice. Our results demonstrate that *Abcb5* is part of the blood-brain barrier; it affects susceptibility to HIT by altering the brain concentration of haloperidol. Moreover, a genetic association study in a haloperidol-treated human cohort indicates that human *ABCB5* alleles had a time-dependent effect on susceptibility to individual and combined measures of HIT. *Abcb5* alleles are pharmacogenetic factors that affect susceptibility to HIT, but it is likely that additional pharmacogenetic susceptibility factors will be discovered.

### Conclusions

*ABCB5* alleles alter susceptibility to HIT in mouse and humans. This discovery leads to a new model that (at least in part) explains inter-individual differences in susceptibility to a drug-induced CNS toxicity.

### Background

The brain is the control center of the human body. This complex organ controls thoughts, memory, speech, and movement, it is the seat of intelligence, and it regulates the function of many organs. The brain comprises many different parts, all of which work together but all of which have their own special functions. For example, the forebrain is involved in intellectual activities such as thinking whereas the hindbrain controls the body’s vital functions and movements. Messages are passed between the various regions of the brain and to other parts of the body by specialized cells called neurons, which release and receive signal molecules known as neurotransmitters. Like all the organs in the body, blood vessels supply the brain with the oxygen, water, and nutrients it needs to function. Importantly, however, the brain is protected from infectious agents and other potentially dangerous substances circulating in the blood by the “blood-brain barrier,” a highly selective permeability barrier that is formed by the cells lining the fine blood vessels (capillaries) within the brain.

### Why Was This Study Done?

Although drugs have been developed to treat various brain disorders, more active and less toxic drugs are needed to improve the treatment of many if not most of these conditions. Unfortunately, relatively little is known about how the blood-brain barrier regulates the entry of drugs into the brain or about the genetic factors that affect the brain’s susceptibility to drug-induced toxicities. It is not known, for example, why about half of patients given haloperidol—a drug used to treat psychotic disorders (conditions that affect how people think, feel, or behave)—develop tremors and other symptoms caused by alterations in the brain region that controls voluntary movements. Here, to improve our understanding of how drugs enter the brain and impact its function, the researchers investigate the genetic factors that affect haloperidol-induced toxicity by genetically analyzing several inbred mouse strains (every individual in an inbred mouse strain is genetically identical) with different susceptibilities to haloperidol-induced toxicity and by undertaking a human genetic association study (a study that looks for non-chance associations between specific traits and genetic variants).

### What Did the Researchers Do and Find?

The researchers used a database of genetic variants called single nucleotide polymorphisms (SNPs) and a computational genetic mapping approach to show first that variations within the gene encoding Abcb5 affected susceptibility to haloperidol-induced toxicity (indicated by changes in the length of time taken by mice to move their paws when placed on an inclined wire-mesh screen) among inbred mouse strains. Abcb5 is an ATP-binding cassette transporter, a type of protein that moves molecules across cell membranes. The researchers next showed that Abcb5 is expressed in brain capillaries, which is the location of the blood-brain barrier. Abcb5 was also expressed in cerebellar Purkinje cells, which help to control motor (intentional) movements. They also measured the measured the effect of haloperidol and the haloperidol concentration in brain tissue sections in mice that were genetically engineered to make no Abcb5 (*Abcb5* knockout mice). Finally, the researchers investigated whether specific alleles (alternative versions) of *ABCB5* are associated with haloperidol-induced toxicity in people. Among a group of 85 patients treated with haloperidol for a psychotic illness, one specific *ABCB5* allele was associated with haloperidol-induced toxicity during the first few days of treatment.

### What Do These Findings Mean?

These findings indicate that Abcb5 is a component of the blood-brain barrier in mice and suggest that genetic variants in the gene encoding this protein underlie, at least in part, the differences in susceptibility to haloperidol-induced toxicity seen among inbred mice strains. Moreover, the human genetic association study indicates that a specific *ABCB5* allele also affects the susceptibility of people to haloperidol-induced toxicity. The researchers note that other *ABCB5* alleles or other genetic factors that affect haloperidol-induced toxicity in people might emerge if larger groups of patients were studied. However, based on their findings, the researchers propose a new model for the genetic mechanisms that underlie inter-individual and cell type-specific differences in susceptibility to haloperidol-induced brain toxicity. If confirmed in future studies, this model might facilitate the development of more effective and less toxic drugs to treat a range of brain disorders.

### Additional Information

Please access these websites via the online version of this summary at [http://dx.doi.org/10.1371/journal.pmed.1001782](http://dx.doi.org/10.1371/journal.pmed.1001782).

## Introduction

Relatively little is known about the mechanisms regulating the entry of many drugs into the brain, or about the genetic mechanisms affecting susceptibility to many drug-induced central nervous system (CNS) toxicities. Because of this lack of information, candidate drugs affecting the CNS have the lowest rate of survival during clinical development (<2%), and drug distribution across the blood-brain barrier has been identified as a contributing cause for drug candidate failure [[1](#pmed.1001782.ref001)]. The blood-brain barrier is a very complex multicellular vascular structure, which separates the brain from the peripheral blood by actively controlling the entry and efflux of drugs [[2](#pmed.1001782.ref002)]. Therefore, the mechanistic insight obtained by identification of genetic factors affecting a drug-induced CNS toxicity could aid in the development of new medicines and could improve our ability to optimally utilize existing treatments. For example, haloperidol is a potent dopamine receptor antagonist that is used to treat psychotic disorders [[3](#pmed.1001782.ref003)]. However, haloperidol-induced alterations in the extrapyramidal motor system depress the ability to initiate voluntary movements. This effect triggers the characteristic extrapyramidal symptoms (EPS) that develop in 40%–76% of chronically treated human patients, which include tremors, Parkinsonian rigidity, and decreased spontaneous movement [[4](#pmed.1001782.ref004),[5](#pmed.1001782.ref005)]. The same symptoms appear in some (but not all) of the 27 inbred strains that were treated with haloperidol [[6](#pmed.1001782.ref006)]. In this murine genetic model, the time required (latency) for a mouse to move all four paws after being placed on an inclined wire-mesh screen was measured as an index of the haloperidol-induced Parkinsonian rigidity that is observed in haloperidol-treated human patients. Although all strains had plasma drug levels that were similar to those measured in human patients, the extent of the haloperidol-induced toxicity (HIT) varied in a highly (85%) heritable manner [[6](#pmed.1001782.ref006)] among the inbred strains.

Because no pharmacogenetic factors for HIT have previously been identified in humans or mice, we wanted to analyze this murine genetic model of HIT. Of concern, mouse genetic studies have, overall, produced rather disappointing results; they have definitively identified <1% of the genes affecting the studied traits [[7](#pmed.1001782.ref007)]. Beyond the well described problems associated with murine linkage studies [[8](#pmed.1001782.ref008)], several analyses have predicted that the conventional methods used to analyze genome wide association studies (GWAS) using inbred mouse strains will produce even more disappointing results, owing to their low power and a high false positive rate [[7](#pmed.1001782.ref007),[9](#pmed.1001782.ref009),[10](#pmed.1001782.ref010)]. However, the availability of next generation sequence (NGS) data, which can provide complete genome-wide variant information for an expanded number of inbred strains [[11](#pmed.1001782.ref011)], could enable murine GWAS to identify causative genetic factors for many biomedical traits [[12](#pmed.1001782.ref012)]. The detailed genetic variation information obtained using NGS has increased power for pinpointing underlying causal variants relative to studies using SNP array data [[13](#pmed.1001782.ref013)]. Unlike the conventional murine GWAS methods, haplotype-based computational genetic mapping (HBCGM) [[14](#pmed.1001782.ref014)] has provided an alternative method for analysis of murine GWAS data that has identified causative genetic factors for many biomedical traits (reviewed in [[12](#pmed.1001782.ref012)]). HBCGM first organizes the allelic information into haplotype blocks, which facilitates the identification of the causative genetic factors through correlation of the phenotypic data with the haplotype-based pattern of genetic variation. Since our prior studies analyzed much less complex datasets, which had fewer SNPs and haplotype blocks, we did not know if HBCGM could analyze the far more complex datasets produced using NGS data. To determine if HBCGM could be successfully combined with an SNP database generated from whole genome sequence data, we analyzed a murine genetic model of HIT in order to identify causative genetic factors. A translational human genetic association study was then performed to determine if similar genetic factors affected human susceptibility to HIT.

## Methods

### Next-Generation Sequencing of the Genome of Nine Inbred Mouse Strains

Genomic DNA was obtained from the Jackson Laboratory ([http://www.jax.org](http://www.jax.org)). A total of 3 μg of DNA from each of nine inbred mouse strains was sonicated using a Covaris instrument with the following settings: duty cycle, 10%; intensity, 4; cycles per burst, 200; and time, 60 seconds. After end repair of the DNA fragments and addition of “A” base to the 3′ ends, standard Illumina genomic DNA sequencing adaptors were ligated. The samples were run on a 3% agarose gel, gel slices between 300–500 bp were excised, and the DNA was purified with a Qiagen column. The DNA was divided into three aliquots of equal quantity, and amplified separately using one of the following three DNA polymerases: Phusion HF (Finnzymes), Kapa HiFi (Kapa Biosystems), and Herculase (Agilent). For analysis of the SJL strain, Phusion GC (Finnzymes) was also used to amplify a separate aliquot. The PCR conditions for amplification were as follows: 98°C for 3 minutes, followed by nine cycles of 98°C for 20 seconds, 62°C for 30 seconds, and 72°C for 45 seconds, and then a final 72°C for 5 minutes. The amplified products were separated on an agarose gel, slices between 400 and 600 bp were excised, and the DNA was eluted from the gel slices and sequenced on an Illumina HiSeq 2000 instrument.

### Sequence Analysis and Variant Calling

The quality statistics for the allele calls at each position were evaluated using the FASTX-Toolkit ([http://hannonlab.cshl.edu/fastx_toolkit/](http://hannonlab.cshl.edu/fastx_toolkit/)). A rapid decay in sequence quality was observed toward the end of each read, usually within the last 10 (or sometimes 20) bps. To avoid artifacts caused by low quality sequence data, the last 10 or 20 bps of each read were trimmed, based upon a visual inspection of the quality statistics plot for the reads in each lane. The extent of trimming was designed to ensure that the quality scores should be ≥30 in the 1st quartile and ≥20 in the 5th percentile. All sequences were trimmed by 10 bps; with the exceptions being one of the three lanes for the MA/MyJ and MRL, and two of the three SM/J lanes. Since a much larger number of sequences were obtained for SJL, SJL sequences were trimmed by 20 bps to ensure the overall quality of the base calls. The read sequences were then aligned to the reference C57BL/6J mouse genome sequence (MGSCv37 assembly) using BWA [[15](#pmed.1001782.ref015)]. Optical and PCR duplicates were then marked using the Picard tool ([http://picard.sourceforge.net/](http://picard.sourceforge.net/)). The sequence variants (SNPs and INDELs) for each strain were identified using Samtools [[16](#pmed.1001782.ref016)], and the default settings were applied. The raw sequence data for the 17 inbred mouse strains described in [[11](#pmed.1001782.ref011)] were obtained from the European Nucleotide Archive ([http://www.ebi.ac.uk/ena/](http://www.ebi.ac.uk/ena/)), and the same data processing procedures were applied, but these sequences were not trimmed.

To ensure high quality variant calls, the variants were further filtered using two additional criteria. (i) To ensure a high level of confidence in the existence of the variant, we require the QUAL score, which represents the (Phred-scaled) probability that all observed samples are homozygous reference alleles, should be ≥50. (ii) To ensure a high level of confidence that the variant allele is indeed homozygous, the genotype call (the “GT” value) must be a homozygous alternative call, and that the Phred-scaled genotype likelihood (the “PL” value) for the homozygous alternative call is at least 20 units above that of any other PL value. If the variant allele call for a strain does not meet these criteria, the call is converted to an “N” (undetermined) instead of being disregarded, which ensures that we don’t mistakenly assume that it is the reference allele. Information about the fold-coverage and other aspects of the sequencing methods are provided in [S1 Text](#pmed.1001782.s013).

### Haplotype Block Construction and Genetic Mapping in Mice

HBCGM utilizes bi-allelic SNPs that are polymorphic among the strains. Only bi-allelic SNPs with at least one definitive homozygous alternative call among the 23 strains were included, while all other variants (including INDELs) were marked and excluded from HBCGM analysis. The locations and potential codon-changes caused by the SNPs are then annotated relative to the encoded genes using predictive gene models from Ensembl version 65. Only SNPs meeting the following criteria were used for haplotype block construction: (i) polymorphic among the strains with input trait data; and (ii) there were at least eight strains (or one-half of the number of input strains) with unambiguous allele calls. Haplotype blocks with 2, 3, 4, or haplotypes were then dynamically produced as described in [[17](#pmed.1001782.ref017)], and the correlation between the input phenotypic data and the haplotype pattern within each identified block was evaluated as described [[14](#pmed.1001782.ref014)]. The genes are then sorted based upon the ANOVA *p*-value (in increasing order) for numeric data or by the F statistic (in decreasing order) for categorical data. For a gene that is covered by multiple blocks, the smallest *p*-value (or the largest F statistic) obtained for all blocks within that gene was used.

The genetic effect size is calculated using our previously described method [[18](#pmed.1001782.ref018)]: Genetic effect size = (SSB − (*k* − 1) * MSE)/(SStotal + MSE), where SSB is the between-group sum-of-squares of the ANOVA model, SStotal is the total sum-of-squares, *k* is the number of groups, and MSE is within-group sum-of-squares divided by (*n* − *k*), where *n* is the total number of objects in the model. Given the very large number of haplotype blocks produced by the HBCGM method, all results were automatically filtered according to pre-determined criteria using software that selected the correlated genes that were expressed in a designated target organ and had missense SNPs.

### Robustness Assessment

To assess the robustness of the HBCGM output obtained using the data obtained from different sets of input strains, we developed software that would automatically perform a re-sampling analysis. For each iteration, a number (1, 2, or 3) was randomly selected to indicate the number of strains whose data would be randomly excluded from the 17 strains with available latency data. Next, the selected number of strains (and each of the individual strains to be deleted were also randomly selected) was excluded for this iteration. Then, HBCGM were run on this re-sampled data and the result was recorded (using *p* = 0.01 as the cutoff). This process was repeated 100 times. If a re-sampled strain panel was identical to one that had already been re-sampled, this iteration was disregarded, and another subset was randomly chosen using the same procedure. To measure the overall association of a gene with the phenotype among the 100 random re-sampled sets of output, we defined a robustness score based on the *p*-values obtained, which was calculated as: Score=∑i=1100-log10(pi), where *p* _i_ was the resulting *p*-value for that gene in the ith re-sampled experiment (if the *p*-value was bigger than the cutoff and therefore censored in the result, *p* _i_ was set to 0.01, i.e., the cutoff). Genes with a larger robustness index have stronger overall correlation among the different subsets analyzed. Their calculated scores then were used to rank the genes, and those with the highest scores were output.

### Statistical Analysis of Haloperidol Latency Data in Mice

The haloperidol-induced latency data measured on days 0, 3, 7, 30, 60, and 120 (MPD datasets: 39407, 39408, 39409, 39410, 39411, and 39445, respectively) and the plasma haloperidol concentrations measured on days 30, 60, 90, and 120 (MPD numbers: 39403–39406, respectively) were obtained from the Mouse PHENOME Database ([http://phenome.jax.org/](http://phenome.jax.org/)). The correlation analyses were performed using the phenotypic data for all 27 strains that were evaluated. The non-parametric Spearman’s correlation coefficient (*rho*) was calculated [[19](#pmed.1001782.ref019)], and the statistical significance of the correlation between the tested datasets was evaluated using this result. This non-parametric test was used to avoid producing spurious results that could be caused by the non-normal distribution of the data and by the presence of extreme outlying values. The *p*-values were then adjusted for multiple testing using the Benjamin-Hochberg method [[20](#pmed.1001782.ref020)]. To minimize the effect produced by the strain with an extreme outlying value and to better visualize the relationship of the variables, the haloperidol-induced latency data from day 7 and day 30 were log-transformed (10-based). (It should be noted that the log-transformation has no effect on the Spearman’s *rho*, nor did it affect the corresponding test result.)

### Haloperidol Toxicity Measurements

All animal experiments were performed according to protocols approved by the Stanford Institutional Animal Care and Use Committee. All mice were obtained from Jackson Labs, and were used at 10–12 weeks of age, and the results are reported according to the ARRIVE guidelines ([S1 Text](#pmed.1001782.s013)) [[21](#pmed.1001782.ref021)]. Haloperidol was administered to the mice by one of two methods: (i) a haloperidol pellet (Innovative Research of America) was subcutaneously implanted, which continuously released a dosage equivalent to 3 mg/kg/day, using published methods [[22](#pmed.1001782.ref022)]; or (ii) with 10 mg/kg haloperidol (Sigma) IP qd for measurement of drug and metabolite concentrations. The haloperidol-induced latency was measured as the time required for a mouse to move all four paws after being placed on a vertical metal mesh screen as previously described [[22](#pmed.1001782.ref022)].

### Haloperidol Quantitation

Haloperidol was purchased from Sigma-Aldrich; haloperidol-d4 was purchased from Toronto Research Chemicals Inc.; HPLC grade acetonitrile and water were from Honeywell Burdick and Jackson International Inc.; and formic acid was purchased from Pierce Thermo Scientific.

Mouse brain tissues of known weight were homogenized in methanol using a Precelly 24 (Bertin Technologies) homogenizer. 50 μl of homogenate was aliquoted, and haloperidol-d4 was added as internal standard to a final concentration of 100 ng/ml. After the mixture was incubated with three volumes of cold acetonitrile at −20°, it was centrifuged at 14,000 rpm for 10 minutes. The supernatant was dried in a speed vacuum and re-suspended in the original volume. A standard curve was prepared using brain homogenate from an untreated mouse in a linear range from 0.5 ng/ml to 250 ng/ml. Quantitative analysis of haloperidol and its metabolite (HPP+) was performed on an Agilent accurate mass QTOF 6520A coupled with an UHPLC Infinity 1290. The analytes were separated on a Phenomenex Kinetex XB-C18 column 2.1 × 100 mm. The flow rate was 0.5 ml/min with a gradient of solvent B (acetonitrile with 0.1% formic acid; solvent A is 0.1% formic acid) from 5% to 35% in 10 minutes, then 35% to 95% in 5 minutes. Positive electrospray full scan in the range of m/z 110–1,000 spectra were collected. Data analysis was done using Agilent quantitative analysis software.

### Desorption Electrospray Ionization Mass Spectrometroscopic Imaging

High mass resolution/high mass accuracy mass spectrometry tissue imaging was performed using a lab-built desorption electrospray ionization mass spectrometry imaging (DESI-MSI) source coupled to an LTQ-Orbitrap XL mass spectrometer (Thermo Scientific). DESI-MSI was performed in the positive ion mode from *m/z* 200–500, using the orbitrap as the mass analyzer at a resolving power of 60,000. The spatial resolution of the imaging experiments was of 150 μm. A histologically compatible solvent system dimethylformamide:acetonitrile (DMF:ACN) 1:1 (v/v) was used for analysis [[23](#pmed.1001782.ref023)] at a flow rate of 0.8 μl/min. The N_2_ pressure was set to 175 psi. After DESI-MSI, the same tissue section was subjected to hematoxylin–eosin (HE) staining. The software ImgGenerator (freeware, [http://www.msimaging.net/](http://www.msimaging.net/)) was used for converting raw files into 2-D images. Spatially accurate ion images were assembled using BioMap software (freeware, [http://www.maldi-msi.org/](http://www.maldi-msi.org/)). The protonated form of haloperidol was detected in the tissue sections at *m/z* 376.14679 (mass error of −1.7 ppm). The isotopic distribution of the ion at *m/z* 376.14679 was also found to agree with the molecular formula of haloperidol. DESI-MS ion images of haloperidol were compared to optical images of the same tissue in HE-stained tissue sections. The total abundance of haloperidol in each pixel from a DESI-MS ion image was extracted and averaged to obtain the average abundance of haloperidol per tissue section. This procedure was performed for each tissue section analyzed for each strain of mice. Based on the average total abundance of haloperidol for each strain, the fold change was then calculated.

### RNA In Situ Hybridization

*Abcb5* and *Drd2* probes were designed to hybridize to their corresponding mRNAs. The 741 bp *Abcb5* probe was PCR amplified from RIKEN_cDNA_clone B020023O04 using the primer set: AGGCCAGAAACAGAGGATTG; and CCTGGTAGAGCATGGCTTTG. The 1,126 bp *Drd2* probe was PCR amplified from total cDNA from a C57BL/6J mouse using the primer set: CCGTGAACCCCATCATCTAT and GGTTTGGTGCATGTATGGTG. PCR reactions were performed using Phusion High-fidelity DNA polymerase (Thermo Scientific) and the products were cloned into pCR-BluntII-TOPO vector (Life Technology). Clones with correct insertions were linearized using either BamHI or NotI (New England Biolabs) for in vitro transcription. Anti-sense and sense RNA probes were then in vitro transcribed using Riboprobe System-SP6/T7 (Promega) and DIG RNA labeling mix (Roche), DNase treated, precipitated in lithium chloride (Ambion), and stored at −80°C in aliquots. Freshly dissected mouse brain was immediately bisected sagittally, and fixed in 4% paraformaldehyde in DEPC treated phosphate buffered saline at 4°C overnight. Fixed tissues were then saturated in DEPC-treated 20% sucrose at 4°C overnight, then flash frozen in OCT compound (Tissue-Tek) and stored at −80°C. The brain sections were cut at 20 μm thickness, and serial sections were collected on Superfrost Plus glass slides (Fisher Scientific). Sets of serial sections across the whole brain were then hybridized anti-sense probes, and adjacent sections were hybridized with the sense probe (as negative controls), at 63°C in a moist chamber overnight. After washing and blocking, tissue sections were then incubated with AP-conjugated anti-DIG antibody (Roche) at 4°C overnight. Tissue sections were then washed and chromogenic reactions were then performed by incubation with the AP substrate BM Purple (Roche) at room temperature. The sections were then examined using a light microscope, and a purple/blue color indicates the localization of the targeted mRNA within the tissue.

### Statistical Analysis of Data Obtained from Inbred and Chromosome Substitution Strains

The latency measurements from chromosome substitution strain (CSS) mice were log-transformed, and a two-sample *t*-test was applied to compare the latency of CSS12 to that of C57BL/6. The differences between the latency on the log-scale and the corresponding 95% confidence intervals were evaluated, and then transformed back to the original scale. Similarly, the measured haloperidol and HPP^+^ concentrations were log-transformed and a two-sample *t*-test was applied to compare the concentration in A/J to that in C57BL/6. The differences between the latency on the log-scale and the corresponding 95% confidence intervals were evaluated and then transformed back to the original scale. For comparison of the HPP^+^ levels in A/J, CSS12, and C57BL/6 strains, the concentrations were log-transformed and an F-test was applied. The differences between the latency on the log-scale and the corresponding 95% confidence intervals were evaluated using SAS (version 9.3), and were then transformed back to the original scale.

### Analysis of Abcb5 Knockout Mice

Mice with a homozygous *Abcb5* gene deletion were prepared and bred onto the C57BL/6 by nine backcrosses, and these mice did not express *Abcb5* mRNA (JPG, MMG, and colleagues, personal communication). The latency in control wild-type littermate and *Abcb5* knockout mice were analyzed after 0 to 7 days of haloperidol 10 mg/kg/day IP administration. The statistical significance of the difference in the latency measurements between wild-type and *Abcb5* knockout mice was determined using a two-sample two-sided *t*-test on log-transformed data for each day as described in the preceding paragraph. Since gender could affect the haloperidol response, we performed a two-way ANOVA to assess the effect of gender. However, the gender effect was only significant on day 4 (*p* = 0.046) and was not significant for all of the other days. Therefore, an adjustment for the gender effect was not performed.

### Human Genetic Association Analysis: Human ABCB5 Alleles and Haloperidol Toxicity

The study was approved by the ethics committee at the University of Munich, and was carried out in accordance with the ethical standards laid down in the 1964 Declaration of Helsinki and later revisions (S2 Text). Additional description of these participants and their characteristics can be found elsewhere [[24](#pmed.1001782.ref024)]. Written informed consent was obtained at the time of recruitment for each participant. In brief, this human cohort consists of 101 patients of European descent that were treated with haloperidol as a monotherapy for a period of one month during the acute phase of their psychotic illness. The patients were enrolled over a 3-year period (1999–2002) at the Department of Psychiatry, Ludwig-Maximilians-University of Munich, Germany. Patients were excluded if they had a known contra-indication for treatment with haloperidol, had tardive dyskinesia previously, or severe neurological or medical disorders, organic brain diseases, pregnancy, or acute suicidal tendencies. Furthermore, the patients were excluded if they received a co-medication, such as beta blockers, or antidepressants that could possibly influence the antipsychotic treatment or its side effects. The patients were treated with haloperidol during the first phase of the illness’s episode, and then shifted to another antipsychotic treatment if there was a lack of response or if severe side effects developed. Patients were treated with haloperidol according to clinical need without any dose limitation during the acute phase of the illness. The mean dose of haloperidol was 8.76 mg/d (SD = 4.4) for day 1, 10.54 mg/d (SD = 6.0) for day 3, 10.88 mg/d (SD = 5.9) for day 7, 10.43 mg/d (SD = 6.2) for day 14, 10.95 mg/d (SD = 7.3) for day 21. As this was a naturalistic study doses were adapted to clinical requirements. Diagnosis was obtained through the Structured Clinical Interview for DSM-IV (SCID-I) interview, and psychopathological measurements were administered by two psychiatrists with highly reliable inter-rater evaluation results (k > 0.80). Haloperidol plasma levels, positive and negative syndrome scale (PANSS), ESRS, and UKU scores were assessed at baseline and on days 3, 7, 12, 21, and 28 as described [[24](#pmed.1001782.ref024)]. Since genotyping was performed after the clinical part of the study was closed, it was not possible for any of the evaluators to be aware of the genetic results.

The alleles at 38 SNPs in *ABCB5* were genotyped on a MassARRAY platform (Sequenom) for 85 patients, which represents all of the available DNA samples obtained from the individuals in this cohort. Briefly, after a multiplex PCR and shrimp alkaline phosphatase treatment, a single base extension reaction was performed. Following the de-salting of extension products with SpectroCLEAN resin (Sequenom), samples were spotted on SpecroCHIPs GenII (Sequenom), and analyzed with a MassARRAY MALDI-TOF mass spectrometer. Allele specific extension products and the resulting genotypes were identified using Typer 3.4 Software (Sequenom). For genotyping quality assurance, the CEU HapMap Trios (Coriell Institute for Medical Research) were included and compared with the HapMap database ([www.hapmap.org](http://www.hapmap.org)). Nine of the SNPs were not polymorphic in these 85 patients, and were excluded from further analysis. None of the remaining 29 SNPs violated the HWE condition, since the smallest *p*-value obtained using a traditional Chi-squared test [[25](#pmed.1001782.ref025)] was 0.11. Of note, 26 SNPs exhibited all three possible genotypes (i.e., homozygous major allele, heterozygous and homozygous minor allele), while three SNPs had only two genotypes. For the 26 SNPs with all three possible genotypes, we used an additive model to assess the association using ANOVA model. For the three SNPs with only two observed genotypes (homozygous major allele and heterozygous), only one grouping was tested using two-sample *t*-test. For each patient, the degree of parkinsonoid toxicity was rated as 0 (absent), 1 (mild), 2 (moderate), or 3 (severe) for each time point. A patient was classified as exhibiting parkinsonoid toxicity if the score was ≥2, as not exhibiting this toxicity if the score was ≤1 at each time point. Patients were similarly rated and classified for dyskinesia and akathesia, the other two toxicities. A combined measure of HIT at each time point was defined as follows: a patient was designated as exhibiting toxicity for (i.e., 1) if any of the three toxicity measurements was ≥2, and as non-toxic (i.e., 0) if the three toxicity measurements were all ≤1. Finally, as a cumulative index of the toxic effect of haloperidol, the area-under-curve (AUC) of the combined toxicity measures for days 0, 1, 3, and 7 was calculated. This value represents the cumulative toxicity induced by haloperidol over the indicated period, and it assumes one of the following values: 0, 1, 2, 3, or 4.

## Results

### Whole Genome-Genetic Association Mapping of Susceptibility to HIT

As described in [S1 Text](#pmed.1001782.s013), we analyzed NGS data to produce a 16 million SNP database with alleles covering 26 strains, including the 12 strains sequenced here, 13 strains analyzed in [[11](#pmed.1001782.ref011)], and the reference C57BL/6 strain ([S1 Fig.](#pmed.1001782.s003); [S1 Table](#pmed.1001782.s008)). Since our prior studies analyzed much less complex datasets with fewer SNPs and blocks, we did not know if HBCGM could analyze this far more complex (~900-fold more blocks) whole genome NGS-based dataset. However, we found that whole genome HBCGM could identify the known genetic factors for four biomedical traits ([S2 Fig.](#pmed.1001782.s004); [S2 Table](#pmed.1001782.s009); S3 Text). Moreover, we also found that its performance was superior to that of another genetic association method, which analyzed the correlation with a selected set of individual SNPs, especially when the inbred strains exhibited three or more distinct phenotypic responses ([S3 Table](#pmed.1001782.s010)).

We then examined available data on HIT (measured after 3, 7, 30, 60, or 120 days of haloperidol treatment) for the 17 strains with available genomic sequence ([S2B Table](#pmed.1001782.s009)). As described in the [S2B Table](#pmed.1001782.s009), our analysis of these data indicated that the prolonged latency appearing in some of the inbred strains (i) was due to haloperidol treatment; (ii) was not correlated with plasma drug levels; and (iii) had its relative magnitude in the strains maintained across all treatment periods (ranging from 3 to 120 days) analyzed. The lack of correlation between latency and plasma haloperidol level was expected, since the effect of CNS-acting drugs is dependent upon the brain (and not the plasma) drug concentration. HBCGM analysis indicated that genetic variation within *Abcb5* had the highest correlation (*p* = 8.3 × 10^−11^) with the day 30 latency values ([Fig. 1A](#pmed.1001782.g001)). HBCGM analysis also indicated that *Abcb5* was highly correlated with the latencies measured after 60 (rank number 3, *p*-value 2.9 × 10^−13^) or 120 (rank number 2, *p*-value 1.7 × 10^−10^) days of haloperidol treatment. Several other factors also indicated that *Abcb5* was a likely candidate gene: (i) When the robustness of the mapping results were assessed by randomly deleting sets of strains and iteratively repeating the analysis, *Abcb5* was among the top genes identified after the latency data were analyzed for treatment days 30, 60, or 120 ([S4 Table](#pmed.1001782.s011)). (ii) Among the 440 SNPs within or near *Abcb5*, eight missense SNPs divided the 17 strains into three haplotypic groupings with different latencies: group 1 (range: 3–44 seconds; 14 strains); group 2 (144–171 seconds; NZW, NZO); and group 3 (234 seconds, A/J). The three strains with extreme latency values share alleles at three missense SNPs that are within a nucleotide-binding region of the protein ([Fig. 1B](#pmed.1001782.g001)). Also, another codon changing SNP (cSNP) (rs30781912) converted a serine, which is conserved in all vertebrates, to an alanine in the strain (A/J) with the highest latency. (iii) We investigated whether *Abcb5* alleles could predict the measured latency for two inbred strains with extreme latency values (NON/ShiLtJ, 2.8 seconds; and NZL/LtJ, 211 seconds) whose genomic sequence was not yet characterized. After characterizing their *Abcb5* alleles, we found that their measured latencies were well within the range predicted by their *Abcb5* haplotype ([Table 1](#pmed.1001782.t001)). Lastly, genetic variation within *Abcb5* could affect susceptibility to HIT, since it is a member of the ABC-transporter gene family. Although little is known about murine *Abcb5*, its human homologue (*ABCB5)* is an energy-dependent efflux transporter that mediates the resistance to multiple drugs and chemotherapeutic agents [[26](#pmed.1001782.ref026)–[30](#pmed.1001782.ref030)]. Drug efflux pumps located at the luminal membrane of capillary endothelial cells in the brain, which export a drug back into the circulating blood after it has entered the brain, are major determinants of drug levels in the brain [[31](#pmed.1001782.ref031)].

### Fig 1. A pharmacogenetic factor for HIT.

![Fig 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/212d/4315575/72934cd33a57/pmed.1001782.g001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4315575_pmed.1001782.g001.jpg)

(A) The average latency measured after 30 days of haloperidol administration to 17 inbred strains is shown in the top panel. The bottom panel shows ten genes whose genetic pattern was most highly correlated with the latency data by the HBCGM program. The genes within correlated haplotype blocks are indicated by their symbol; an orange, yellow, or white background indicates whether SNPs causing a significant, minor, or no amino acid change are present, respectively. (A blue background indicates a SNP that affects a potential splice site alteration is present.) The haplotypic pattern is shown as colored rectangles that are arranged in the same order as the input data. Strains with the same colored rectangle have the same haplotype within the block. The p-values and genetic effect size were calculated as previously described [14]. (B) Missense SNPs in Abcb5. The 1,255 amino acids in Abcb5 contain two transmembrane (shaded blocks) and two ATP binding transporter domains (solid blocks). The alleles in eight missense SNPs in the 17 analyzed strains are shown, and the alleles that differ from the C57BL/6 reference strain are highlighted. The Gly170Cys is within the first transmembrane domain, and Gly491Ser and Lys493Glu are within the first transporter domain. These three SNPs divide the strains into three haplotypic groupings (each indicated by a different color) that correspond with the haloperidol-induced latency.

### Table 1. Abcb5 alleles predict haloperidol-induced latency.

| SNP | NON/ShiLtJ | NZL/LtJ |
| --- | --- | --- |
| T7A | T | T |
| L10S | L | S |
| V27A | V | — |
| G170C | G | C |
| G491S | G | S |
| K493E | K | E |
| S619F | S | F |
| K1206R | K | R |
| Haplotype | 1 | 2–3 |
| Predicted latency | 0–44 | 147–234 |
| Measured latency | 2.75 | 211 |

Table 1 Caption: The alleles for eight missense SNPs in Abcb5 that were determined for two inbred strains are shown. The alleles place NON/ShiLtJ mice in haplotypic group 1, and NZL/LtJ mice in haplotypic group 2–3. The measured haloperidol-induced latencies on treatment day 30 are well within the range (based upon data in Fig. 2) predicted by their haplotype.

### Analysis of Chromosome Substitution Strains and Brain Haloperidol Levels

A series of C57BL/6J-A/J CSS have been produced where each strain is homosomic for a single specified A/J chromosome on an otherwise C57BL/6 genetic background [[32](#pmed.1001782.ref032)]. Since A/J and C57BL/6 exhibited extreme latency values, we tested the haloperidol-induced latency in four CSS. Of these, only CSS12 (which have A/J alleles for every gene on Chromosome 12) had a significantly increased latency after 5 (6.7-fold, t value = 12.98, *p* < 0.0001, 95% CI 4.9–9.3) and 10 days (8.5-fold, t = 8.41, *p* < 0.0001, 95% CI 4.8–14.8) of haloperidol treatment ([Fig. 2A](#pmed.1001782.g002)). Among the 34 genes on Chromosome 12 that were correlated with susceptibility to HIT, *Abcb5* had the highest level of correlation and was the most plausible candidate based on its biological function ([S5 Table](#pmed.1001782.s012)).

### Fig 2. (A) HIT in C57BL/6J, A/J, and four chromosome substitution strains.

![Fig 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/212d/4315575/e4a8b36f32da/pmed.1001782.g002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4315575_pmed.1001782.g002.jpg)

The time (latency) required for a mouse to make a coordinated movement on a vertical mesh screen after treatment with haloperidol (3 mg/kg/day IP) for the indicated number of days is shown. Each bar represents the average ± SEM for three to four mice per strain. Only CSS12 mice (which have A/J alleles for every gene on Chromosome 12) had a significantly increased latency (p < 0.005) after 5 and 10 days of haloperidol treatment. (B) The amount of haloperidol and its metabolite (HPP+) in brain tissue obtained from A/J and C57BL/6 mice after 4 days of treatment with haloperidol (10 mg/kg/day IP) was assessed in two independent experiments. Each bar represents the average ± SEM of the LC/MS determined abundance (molecules per unit mass of brain tissue) for n = 4 mice per strain. (C) The brain level of the oxidative metabolite of haloperidol (HPP+) was measured after 10 days of haloperidol administration (3 mg/kg/day) to A/J (n = 6), C57BL/6 (n = 6), and Chromosome 12 substitution strain (CSS12) (n = 8) mice. Each bar shows the average ± standard error for the six to eight measurements for each strain, and the p-value is calculated using the Student’s t test on log2-transformed relative abundance data. Of note, CSS12 mice have a significantly higher (1.3-fold, p = 0.004) brain HPP+ level than C57BL/6J mice, but this level is below that in A/J mice.

Haloperidol is oxidatively metabolized to a pyridinium species (HPP^+^) that is present in brain tissue obtained from haloperidol-treated humans [[33](#pmed.1001782.ref033)] or rodents [[34](#pmed.1001782.ref034)]. HPP^+^ is a structural analog of the oxidative metabolite (MPP^+^) of a Parkinson-inducing neurotoxin N-methyl-4-phenyl-1,2,3,6-tetrahydropyridine (MPTP), and HPP^+^ has been shown to induce toxicity in dopaminergic neurons [[33](#pmed.1001782.ref033),[35](#pmed.1001782.ref035),[36](#pmed.1001782.ref036)]. To determine if differences in brain drug (or metabolite) levels were associated with susceptibility, we measured the concentration of haloperidol and HPP^+^ in brain tissue obtained from C57BL/6J and A/J mice after treatment with haloperidol 10 mg/kg/day for 4 days. This higher dose of haloperidol was used to ensure that we could measure the brain concentrations of both haloperidol and HPP^+^, since the metabolite is present at much lower (~100-fold) concentrations than the parent drug. C57BL/6 mice treated with this haloperidol dose do not exhibit an increased latency (see below). Consistent with HIT occurring in A/J but not C57BL/6J mice, the brain levels of haloperidol (2.2-fold and 2.3-fold, t values 4.88 and 4.99, *p*-values 0.008 and 0.002, 95% CI 1.4–3.5 and 1.5–3.5, respectively) and HPP^+^ (1.4-fold and 1.7-fold, t values 4.75 and 2.39, *p*-values 0.009 and 0.05, 95% CI 1.1–1.7 and 1.0–2.6, respectively) were both higher in A/J than in C57BL/6J mice in two independently performed experiments ([Fig. 2B](#pmed.1001782.g002)).

We also examined HPP^+^ levels in brains obtained from A/J, C57BL/6J, and CSS12 mice after a prolonged period (10 days) of haloperidol administration (3 mg/kg/day). The brain HPP^+^ level in CSS12 mice was significantly above (1.3-fold, t = 3.22, *p* = 0.004, 95% CI 1.1–1.5) that in C57BL/6 mice, but was below that of A/J mice, which had 1.9-fold (t = 7.65, *p* = 3 × 10^−5^, 95% CI 1.6–2.2) increase in HPP^+^ relative to C57BL/6 ([Fig. 2C](#pmed.1001782.g002)). These results suggest that the differential strain susceptibility to HIT could result from differences in Abcb5-regulated drug transport, and that an A/J Chromosome 12 on an otherwise C57BL/6 genetic background increases brain HPP^+^ levels. The correlation between the relative brain abundance of HPP^+^ in A/J, C57BL/6, and CSS12 mice and their haloperidol-induced latencies ([Fig. 2A](#pmed.1001782.g002)) is consistent with the other studies [[33](#pmed.1001782.ref033),[35](#pmed.1001782.ref035),[36](#pmed.1001782.ref036)] indicating that HPP^+^ is causative of HIT.

### Desorption Electrospray Ionization-Mass Spectrometry Imaging of Haloperidol Abundance in Brain Sections

In DESI-MSI, thin tissue sections are bombarded with solvent microdroplets to dissolve hundreds of chemicals from the sample surface; the secondary microdroplets formed enter a mass spectrometer, and are analyzed for their chemical content. A movable stage enables multiple pixels (in a row by column matrix) encompassing the entire thin section to be sampled, which generates a detailed 2-D chemical map of the distribution of molecules within the sample surface. The relative amount of haloperidol in each region was analyzed by this method to produce a 2-D ion image of its relative abundance throughout the entire brain section. We analyzed independently prepared sagittal brain sections (two per mouse) from A/J and C57BL/6 mice (three per strain) after treatment with haloperidol (10 mg/kg/d IP) for 5 days. This analysis revealed that haloperidol was diffusely distributed throughout all brain regions, but the amount of haloperidol in A/J brain tissue was consistently greater (~6-fold) than in C57BL/6 brain tissue ([Fig. 3](#pmed.1001782.g003)). Because of its low abundance, we could not detect the oxidative metabolite by this method. The inter-strain differences indicated by DESI-MSI were greater than those identified by quantitative MS analysis of extracted whole brain. This difference could be attributed to the fact that DESI-MSI was performed on tissue sections in a specific sagittal plane, which does not account for the abundance of haloperidol in other brain regions that were sampled by quantitative MS analysis of extracted whole brain. Nevertheless, direct analysis of brain tissue and DESI-MS imaging both clearly show that haloperidol was more abundant in A/J mouse brain.

### Fig 3. DESI-MSI of haloperidol in brain tissue.

![Fig 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/212d/4315575/825fbf24d008/pmed.1001782.g003.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4315575_pmed.1001782.g003.jpg)

A/J and C57BL/6J mice (n = 3 per strain) were treated with haloperidol (10 mg/kg/day IP) for 5 days. Two sagittal sections, which were separated by a distance of 120 μm, were prepared the same brain region of haloperidol-treated mice. The amount of haloperidol was analyzed DESI-MSI, and the relative abundance of haloperidol is indicated by the color shown in the scale bar. The HE-stained brain section shown at the bottom shows the sagittal plane analyzed, and the location of the frontal cortex (FC) and cerebellum (CB) are indicated for orientation. Haloperidol was diffusely present in the brain tissue obtained from both strains. Although the amount of haloperidol in brain tissue varied among different mice, A/J brain tissue had remarkably increased amount of haloperidol relative to C57BL/6J brain tissue.

### Abcb5 mRNA Is Expressed near Brain Capillaries and by Purkinje Cells

Since we did not know if *Abcb5* is expressed in brain, in situ hybridization studies were performed on tissue sections from mouse brain. The hybridizations were performed using an anti-sense probe that hybridizes to *Abcb5* mRNA, and with an *Abcb5* sense probe that serves as a negative control. *Abcb5* mRNA expression was observed in the perivascular region of brain capillaries ([Fig. 4A](#pmed.1001782.g004)). Since this is the anatomic location of the blood-brain barrier function, this expression pattern is consistent with *Abcb5* having a role in regulating brain haloperidol levels. Of importance, Purkinje cells in the cerebellum also expressed *Abcb5* mRNA; and these cells also expressed dopamine receptor D2 (*Drd2*) mRNA, which encodes the receptor that binds haloperidol ([Fig. 4B](#pmed.1001782.g004)). Interestingly, *Abcb5* mRNA was not detected in the substantia nigra ([S3 Fig.](#pmed.1001782.s005)). A low level of *Abcb5* mRNA expression was also detected in regions of the hippocampus and frontal cortex ([S4 Fig.](#pmed.1001782.s006)), but not by the *Drd2*-positive cells in the substantia nigra ([S3 Fig.](#pmed.1001782.s005)).

### Fig 4. Abcb5 mRNA expression in mouse brain.

![Fig 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/212d/4315575/7d5b07ddb48e/pmed.1001782.g004.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4315575_pmed.1001782.g004.jpg)

In situ hybridization was performed using anti-sense probes for Abcb5 and Drd2 mRNA on C57BL/6 mouse brain sections. For each anti-sense probe, negative-control hybridizations were also performed using the corresponding sense RNA probe on adjacent tissue sections. (A) Abcb5 mRNA was specifically expressed in the perivascular regions, as shown in two separate views of cortical and cerebellar regions. (B) Purkinje cells in cerebellum express Abcb5 mRNA, and these cells also express the dopamine receptor D2 (Drd2) that is bound by haloperidol. The left and right images are shown at 4× magnification, and magnified views (20×) of the boxed regions are shown in the center image.

### Abcb5 Knockout Mice Have a Prolonged Haloperidol-Induced Latency and Increased Brain Haloperidol Levels

To more precisely determine whether *Abcb5* plays a role in HIT, we measured the latencies in homozygous *Abcb5* knockout mice (on a C57BL/6 background) before and after treatment with haloperidol (10 mg/kg/day IP). Before drug treatment, there was no difference in the latency of *Abcb5* knockout mice relative to wild-type littermates (t = 1.54, *p* = 0.185) ([Fig. 5A](#pmed.1001782.g005)). Although there were only marginal differences on treatment day 4 (t = 2.15, *p* = 0.05), the latencies in *Abcb5* knockout mice were significantly prolonged after 5 (10.1-fold, t = 3.09, *p*-value = 0.03, 95% CI 1.5–69.2), 6 (13.6-fold, t = 4.79, *p*-value = 0.005, 95% CI 3.5–55.1), and 7 (50.3-fold, t = 7.91, *p*-value = 0.0005, 95% CI 14.1–179.6) days of haloperidol treatment relative to wild-type mice ([Fig. 5](#pmed.1001782.g005)). In a second independently performed experiment, haloperidol treatment induced a markedly prolonged latency in *Abcb5* knockout mice ([Fig. 5B](#pmed.1001782.g005)). In contrast, the latencies measured in wild-type littermates did not increase with haloperidol treatment and were comparable to those of C57BL/6 mice. Furthermore, DESI-MSI revealed that the amount of haloperidol in brain tissue obtained from *Abcb5* knockout mice (*n* = 3, 6,282 ± 2,279) was 6.8-fold higher than in wild-type littermates (*n* = 3, 935 ± 200, *p* = 0.02) ([Fig. 5C](#pmed.1001782.g005)). The prolonged latency and increased brain haloperidol levels observed in *Abcb5* knockout mice confirms that *Abcb5* affects susceptibility to HIT by increasing brain haloperidol levels.

### Fig 5. (A, B) Abcb5 knockout mice have a prolonged haloperidol-induced latency.

![Fig 5](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/212d/4315575/5b5e132266af/pmed.1001782.g005.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4315575_pmed.1001782.g005.jpg)

In two independent experiments shown in (A) and (B), Abcb5 knockout mice and wild-type littermates (gender indicated by male or female) were treated with haloperidol (10 mg/kg IP) for 7 days, and the haloperidol-induced latency was measured on the indicated days. In both experiments, the latencies in Abcb5 knockout mice were significantly prolonged relative to wild-type mice after 4, 5, 6, and 7 days of haloperidol treatment. The p-values comparing the latencies measured in the Abcb5 knockout and wild-type groups are shown for each treatment day. (C) DESI-MSI of haloperidol in brain tissues of Abcb5 knockout (n = 3) and wild-type littermate (n = 3) mice. The mice were treated with haloperidol (10mg/kg/day IP) for 4 days, and brain tissues were collected 4 hours after the last treatment. The amount of haloperidol was analyzed by DESI-MSI on 20 μm brain sections, and the relative abundance of haloperidol is indicated by the color shown in the scale bar. The HE-stained brain section shown at the bottom shows the sagittal plane analyzed. Abcb5 knockout mice have a 7-fold increased brain haloperidol level relative to wild-type littermates (p = 0.02).

### Human ABCB5 Alleles Associate with Haloperidol Toxicity

A human cohort with 85 patients of European descent was treated with haloperidol as a monotherapy for one month during the acute phase of their psychotic illness, and their response to therapy and the appearance of HIT was assessed at baseline and after 1, 3, 7, 14, and 21 days of treatment as described [[24](#pmed.1001782.ref024)]. Their clinical and demographic characteristics are described in [Table 2](#pmed.1001782.t002). We investigated whether *ABCB5* SNP alleles were associated with the appearance of the three types of symptoms of HIT (Parkinsonoid, Dyskinesia, and Akathisia scores) that were measured in this cohort. Of the 29 SNPs analyzed: six were codon changing (cSNPs), three were in the 5′ UTR, and the others were intronic ([Table 3](#pmed.1001782.t003)). We first examined the effect that treatment duration had on the incidence of each of the three types of HIT toxicities throughout the course of treatment (S6 Table). The incidence of akathesia and parkinsonoid features increased through day 14, while dyskinesia decreased after day 3 ([Fig. 6A](#pmed.1001782.g006)), which may be due to fact that the onset of the other toxicities masked the dyskinesia. Therefore, we analyzed a composite index of all three symptoms of HIT, since our mouse genetic analysis identified a genetic factor that alters the brain haloperidol concentration, which will affect susceptibility to all of these symptoms. In examining the allelic associations, we found a clear time-dependent allelic effect on HIT that was maximal on days 1, 3, and 7 (Figs. [6B](#pmed.1001782.g006) and [S5](#pmed.1001782.s007)). Therefore, it is likely that the protective allelic effect would dissipate after prolonged dosing owing to haloperidol (or metabolite) accumulation, but individuals with a protective *ABCB5* allele would exhibit less toxicity during the early treatment period (i.e., days 3–7).

### Table 2. Demographic and clinical data for the 85 patients.

| Variable | Results |
| --- | --- |
| Age (y) (mean SD) | 35.4 11.7 (Range: 18–64) |
| Sex | M = 45 (53%); F = 40 (47%) |
| Diagnosis | Schizophrenia, paranoid type = 46 (54.1%) |
| Schizophrenia, undifferentiated type = 2 (2.4%) |  |
| Schizophrenia, residual type = 2 (2.4%) |  |
| Schizophrenia, catatonic type = 4 (4.7%) |  |
| Schizophrenia, disorganized type = 3 (3.5%) |  |
| Schizoaffective disorder = 15 (17.6%) |  |
| Brief psychotic disorder = 7 (8.2%) |  |
| Delusional disorder = 1 (1.2%) |  |
| Schizophreniform disorder = 5 (5.9%) |  |
| Co-medication (till day 7) | Levomepromazin/ppromethazin = 52 (61.2%) |
| Cholinergic (biperiden/amitriptylin) = 26 (30.6%) |  |
| β-blocking agents = 8 (9.4%) |  |
| Gabaergic agents and benzodiazepine = 45 (52.9%) |  |
| Comorbid illness | Phobia = 2 (2.4%) |
| Alcoholism = 3 (3.5%) |  |
| Depression = 3 (3.5%) |  |
| Suicide attempt = 5 (5.8%) |  |
| Reason for dropout |   |
| Day 14 | 4 discharged after good recovery (4.7%) |
| Day 21 | 1 discharged after good recovery (1.1%) |
| 7 switched to other meds after good recovery (8.2%) |  |
| 7 switched after good recovery but with side effects (8.2%) |  |
| 3 without recovery (3.5%) |  |
| 9 intolerable side effects (10.5%) |  |
### Table 3. Genetic association results for alleles at 29 SNPs with the area under curve calculated by summing the combined toxicity measures on days 0–7.

| SNP ID | Position | Annotation | Region | MAF | AUC | Permutation p-Value |
| --- | --- | --- | --- | --- | --- | --- |
| rs2106562 | 20655268 | 5′UTR |   | 0.35 | 0.044 | 0.523 |
| rs73076550 | 20655392 | 5′UTR |   | 0.11 | 0.272 | 0.988 |
| rs78414512 | 20668277 | INTRON |   | 0.16 | 0.005 | 0.086 |
| rs17143212 | 20682884 | T131I | 1 | 0.03 | 0.002 | 0.034 |
| rs2893006 | 20687181 | 5′UTR / Synonymous | 1 | 0.24 | 0.308 | 0.995 |
| rs75784515 | 20687332 | INTRON | 1 | 0.24 | 0.287 | 0.992 |
| rs61732039 | 20687604 | D370G | 1 | 0.14 | 0.03 | 0.402 |
| rs3213622 | 20690885 | INTRON | 1 | 0.34 | 0.434 | 0.999 |
| rs34603556 | 20691047 | Alternative start codon / M446T | 1 | 0.24 | 0.308 | 0.995 |
| rs61227829 | 20691219 | Synonymous | 1 | 0.15 | 0.038 | 0.459 |
| rs17816709 | 20691860 | INTRON | 1 | 0.24 | 0.287 | 0.992 |
| rs2301641 | 20698270 | K115E / K560E | 1 | 0.36 | 0.588 | 1 |
| rs6952128 | 20700361 | INTRON | 1 | 0.36 | 0.463 | 1 |
| rs17143258 | 20726621 | INTRON | 1 | 0.18 | 0.531 | 1 |
| rs12700229 | 20729931 | INTRON | 2 | 0.26 | 0.719 | 1 |
| rs2190409 | 20732020 | INTRON | 2 | 0.19 | 0.961 | 1 |
| rs17218211 | 20740648 | INTRON | 2 | 0.21 | 0.011 | 0.195 |
| rs10488579 | 20741343 | INTRON | 2 | 0.24 | 0.021 | 0.316 |
| rs17218839 | 20745418 | INTRON | 2 | 0.26 | 0.026 | 0.361 |
| rs2108259 | 20745756 | INTRON | 2 | 0.26 | 0.024 | 0.338 |
| rs62453384 | 20762646 | G365V / G810V |   | 0.38 | 0.478 | 1 |
| rs62453385 | 20762937 | INTRON |   | 0.38 | 0.478 | 1 |
| rs6461513 | 20763907 | INTRON |   | 0.32 | 0.031 | 0.402 |
| rs4719626 | 20768794 | INTRON |   | 0.26 | 0.102 | 0.798 |
| rs6960186 | 20773995 | INTRON | 3 | 0.31 | 0.478 | 1 |
| rs6461515 | 20778646 | E970K / E525K / E137K | 3 | 0.19 | 0.117 | 0.853 |
| rs6461516 | 20778773 | INTRON | 3 | 0.17 | 0.189 | 0.961 |
| rs6461517 | 20787273 | INTRON | 3 | 0.38 | 0.706 | 1 |
| rs1015646 | 20793720 | INTRON | 3 | 0.39 | 0.681 | 1 |

Table 3 Caption: The SNP ID, genomic position (based on NCBI Build 37), minor allele frequency (MAF), and the p-value (uncorrected for multiple testing) for the allelic association with the combined toxicity measure are shown. The annotation indicates the location of the SNP (intron, 5′ or 3′ UTR), and the amino acid residue altered by the SNP (because of alternative ABCB5 transcripts, more than one location is indicated for some SNPs). All p-values < 0.05 are in bold. To account for multiple testing a permutation test was performed for each SNP, and the p-value is shown in a separate column. Of note, rs17143212 is a missense SNP (Thr131ILE) that it is the only SNP with a significant p-value in the permutation test. This indicates that its association with the area under the curve (AUC) remains significant even after accounting for the fact that 29 SNPs were simultaneously tested.

### Fig 6. Human ABCB5 alleles and HIT.

![Fig 6](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/212d/4315575/d4badd136a70/pmed.1001782.g006.jpg)

(A) The average of three toxicity measurements (± SEM) for 85 patients determined after the indicated time of haloperidol treatment is shown. Of note, the average Dyskinesia score peaked at 3 days post treatment and then declined, while the two other toxicity measurements (akathesia and parkinsonoid) increased with time after treatment until day 14. For this analysis, the toxicity measurement data for days 0, 1, 3, 7, 14, and 21 were available for 85, 85, 85, 85, 81, and 54, respectively, of the 85 patients in this cohort. (B) The graph shows the combined toxicity measurement (y-axis) relative to the time (x-axis) for ABCB5 SNP rs17143212. The average combined toxicity measurement was calculated for each genotype at each SNP, and then plotted (± SEM) according to the colors shown in the legend. (C) The LD map for genetic variants in ABCB5, which was compiled using data obtained from the International HapMap project (http://hapmap.ncbi.nlm.nih.gov/). The three regions where the alleles have a high level of LD are enclosed in boxes. Region 2 contains a cluster of four SNPs with alleles that were associated with HIT, and arrows in the diagram at the top of the figure indicate their relative positions.

Since each of these three different toxicities could all be caused by an increased drug (or drug metabolite) concentration in the brain, we examined whether *ABCB5* alleles were associated with a combined toxicity index, which indicated whether a patient manifested any of the three measured toxicities on a given day. To reduce the number of hypotheses that were tested simultaneously, the area under the curve (AUC) for the cumulative toxicity measurements on days 0 to 7 was calculated, and its association with each of the 29 SNPs was tested. (A truncated dataset was used because many of the values for day 14 and 21 were not available for many patients.) Remarkably, a SNP (rs17143212) that was a missense SNP (Thr131Ile) had a particularly strong association (raw *p* = 0.002) with the toxicity index, and a total of ten (out of 29) SNPs showed a plausible association (with raw *p* < 0.05) with the toxicity index ([Table 2](#pmed.1001782.t002)). Among these SNPs, four were consecutive SNPs within the second identified region of linkage disequilibrium in the *ABCB5* gene ([Fig. 6C](#pmed.1001782.g006)). The probability that ten SNPs would randomly have these *p*-values (< 0.05) is <8 × 10^−7^, indicating that it is highly unlikely that all of these associations were false positives resulting from a random match. Since the AUC values didn’t follow a Gaussian distribution and the sample size may not be large enough, the resulting *p*-values from the ANOVA models could be biased. Therefore, we performed a permutation test, which accounts for multiple testing and makes no assumption about the data distribution, to rigorously assess the statistical significance of each association. To do this, the combined toxicity value for the 85 patients was permutated, while their genotypes remained unchanged; this enabled the linkage disequilibrium among the SNPs to be retained. The distribution of the minimum *p*-value for the 29 SNPs was evaluated by performing this permutation experiment 1,000 times. The observed raw *p*-value for each SNP was then compared to this distribution to obtain the true *p*-value for that SNP under the null hypothesis that this region was not associated with HIT. Strikingly, rs17143212, which caused a significant (131Thr → ILE) amino acid change in the predicted protein, still exhibited a significant association (permutation test *p*-value = 0.034) after the permutation test. The four consecutive SNPs that had significant raw *p*-values didn’t have significant permutation test *p*-values. However, if additional patients were available for analysis, we would be able to better assess the significance of their associations. Nevertheless, at least one human *ABCB5* SNP shows a significant association (after correction for multiple testing) with a combined measure of HIT.

## Discussion

This study identifies murine *Abcb5* alleles as pharmacogenetic factors affecting susceptibility to HIT. Because Abcb5 is expressed in brain capillaries, it can function as part of the blood-brain barrier to regulate the level of drugs in the brain [[37](#pmed.1001782.ref037)]. This function is consistent with the measured differences in the brain levels of haloperidol and its metabolite (HPP^+^) in inbred and chromosome substitution strains. These findings suggest a new model for the genetic mechanisms underlying inter-individual and cell type-specific differences in susceptibility to haloperidol-induced CNS toxicity ([Fig. 7](#pmed.1001782.g007)).

### Fig 7. A diagram of the effect of Abcb5-mediated haloperidol transport on HIT.

![Fig 7](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/212d/4315575/1a67c7cd0b1f/pmed.1001782.g007.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4315575_pmed.1001782.g007.jpg)

Abcb5 is expressed in brain capillaries and by cerebellar Purkinje cells. Haloperidol will enter the brain due to its intrinsic properties, but vectorial transport by perivascular Abcb5 will reduce the brain concentration of haloperidol. An individual with alleles that reduce Abcb5 activity are more susceptible to HIT because they will have an increased brain haloperidol concentration. Cerebellar Purkinje cells and dopaminergic neurons in the substantia nigra both express the Drd2 receptor that is bound by haloperidol. In Drd2-expressing dopaminergic neurons haloperidol is converted to its oxidative metabolite (HPP+), which inhibits mitochondrial energy generation and causes the characteristic Parkinsonian-like symptoms to develop. Although Purkinje cells express Drd2, they also express Abcb5, which exports of haloperidol from these cells and protects them from HIT.

After haloperidol enters the brain (because of its lipophilicity or other intrinsic properties), the brain concentrations of haloperidol and its toxic metabolite will be affected by Abcb5 function at the brain-blood barrier. Thus, a strain with an *Abcb5* allele with reduced transporter function is more susceptible to HIT because of the increased haloperidol concentration in the brain. Like the causative toxins (MPTP, rotenone) used in the other animal models of toxin-induced Parkinson’s disease [[38](#pmed.1001782.ref038)–[41](#pmed.1001782.ref041)], the oxidative metabolite of haloperidol (HPP^+^) is a potent inhibitor of NADH-linked mitochondrial respiration [[36](#pmed.1001782.ref036)]. Brain enzymes convert the Parkinsonian neurotoxin MPTP to its toxic metabolite [[42](#pmed.1001782.ref042)–[44](#pmed.1001782.ref044)], which is concentrated in cells with a high affinity dopamine uptake system [[41](#pmed.1001782.ref041),[45](#pmed.1001782.ref045)], which causes the Parkinsonian-like symptoms to develop. HPP+ is a more potent inhibitor of mitochondrial electron transport than the oxidative metabolite of MPTP [[36](#pmed.1001782.ref036)]. The toxicity of paraquat, which is another neurotoxin that selectively damages dopaminergic neurons, is also dependent upon dopamine transporter activity [[46](#pmed.1001782.ref046)]. Thus, we propose that haloperidol in Drd2-expressing dopaminergic neurons is converted to its oxidative metabolite HPP^+^ by the drug metabolizing enzymes present in these neurons [[47](#pmed.1001782.ref047),[48](#pmed.1001782.ref048)]; and the mitochondrial toxicity induced by this metabolite produces the Parkinsonian-like symptoms.

Of interest, we found that Abcb5 and Drd2 were also expressed in cerebellar Purkinje cells. Given their large size, high metabolic requirements, and Drd2 expression, Purkinje cells should also be affected by HIT, since they are particularly susceptible to toxin-induced injury [[49](#pmed.1001782.ref049)]. However, the symptoms of HIT observed in mice and humans are predominantly extrapyramidal and do not have a substantial cerebellar component. Why does HIT produce Parkinsonian symptoms, but not cerebellar toxicity? We believe that Abcb5 expression by Purkinje cells protects them from HIT by their ability to transport haloperidol out of the cells. In contrast, haloperidol-targeted neurons in the substantia nigra, which lack Abcb5 expression, will be susceptible to HIT ([Fig. 7](#pmed.1001782.g007)).

Although a relatively small cohort of patients who were treated with haloperidol was evaluated here, a human *ABCB5* SNP, which causes a significant amino acid change in the protein, was also associated with susceptibility to HIT. However, the human allelic affect is exerted during the early period of drug treatment. The time-dependent effect observed in humans may be caused by the fact that continued haloperidol treatment may overwhelm the ability of the protective allele to clear the toxic metabolite. Despite some evidence for familial clustering [[50](#pmed.1001782.ref050)] and scattered reports (from studies that evaluated a small number of patients) of candidate gene associations [[51](#pmed.1001782.ref051),[52](#pmed.1001782.ref052)], we did not have any solid information about human pharmacogenetic factors for HIT. A recently completed, large human GWAS did not reveal any significant associations with antipsychotic drug-induced toxicities [[53](#pmed.1001782.ref053)]. However, this study did not perform the detailed analysis of HIT that was performed here. Our results suggest that an examination of the time dependence of individual and the aggregation of cumulative toxicity measurements may be required to identify pharmacogenetic factors for antipsychotic agents. Like the murine gene, human *ABCB5* is also highly polymorphic; alleles at ten codon changing SNPs (cSNPs) are predicted to affect human ABCB5 transporter function, including two (Iso648Thre, Glu679Lys) within the nucleotide-binding region [[54](#pmed.1001782.ref054)]. So, it is certainly possible that analysis of a larger cohort could identify other human *ABCB5* alleles that also affect ABCB5 function, and possibly susceptibility to drug toxicity. It is also possible that analysis of a larger human cohort could reveal SNP alleles in other genes that also affect susceptibility to HIT. Although haloperidol is a first-generation antipsychotic agent, the more commonly used second-generation antipsychotics (aripiprazole, clozapine, olanzapine, risperidone) are also associated with a significant incidence of EPS-related toxicities, which is higher than was originally appreciated [[55](#pmed.1001782.ref055)]. It would be of interest to determine whether genetic variation in *ABCB5*, or other efflux transporters, could affect susceptibility to their toxicity. Simulation studies have convincingly demonstrated that using pharmacogenetic information for drug selection [[56](#pmed.1001782.ref056)] or dose adjustment [[57](#pmed.1001782.ref057)] could reduce drug toxicity. Characterization of genetic factors affecting the toxicity caused by haloperidol (or second-generation antipsychotics) could lead to pharmacogenetic strategies that would improve the outcome for patients taking these drugs.

Lastly, analysis of chromosomal substitution strains indicated that the presence of A/J alleles on Chromosome 12 could partially induce susceptibility to HIT. However, the magnitude of the haloperidol-induced latency and the brain HPP^+^ levels in CSS12 mice were significantly below those of A/J mice. Thus, other genetic factors could also affect susceptibility to HIT in mice, which raises the possibility that there are other human pharmacogenetic factors to be found.

## Conclusions

These findings indicate that Abcb5 is a component of the blood-brain barrier in mice and suggest that genetic variants in this gene underlie, at least in part, the differences in susceptibility to haloperidol-induced toxicity among inbred mice strains. Moreover, the human genetic association study indicates that a specific *ABCB5* allele also affects the susceptibility of people to haloperidol-induced toxicity.

## Supporting Information

## Abbreviations:

## Funding Statement

GP and MZ were partially supported by funding from a transformative RO1 award (1R01DK090992-01) DLD was supported by a King Abdullah University of Science and Technology (KAUST) research grant under the KAUST Stanford Academic Excellence Alliance program. ST was supported by Stanford’s CURIS program for summer research interns. LSE was supported by a postdoctoral fellowship from the Center for Molecular Analysis and Design (CMAD). The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Ohtsuki S, Terasaki T (2007) Contribution of carrier-mediated transport systems to the blood-brain barrier as a supporting and protecting interface for the brain; importance for CNS drug discovery and development. Pharm Res 24: 1745–1758.  [DOI](https://doi.org/10.1007/s11095-007-9374-5) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17619998/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharm%20Res&title=Contribution%20of%20carrier-mediated%20transport%20systems%20to%20the%20blood-brain%20barrier%20as%20a%20supporting%20and%20protecting%20interface%20for%20the%20brain;%20importance%20for%20CNS%20drug%20discovery%20and%20development&author=S%20Ohtsuki&author=T%20Terasaki&volume=24&publication_year=2007&pages=1745-1758&pmid=17619998&doi=10.1007/s11095-007-9374-5&)

2. Obermeier B, Daneman R, Ransohoff RM (2013) Development, maintenance and disruption of the blood-brain barrier. Nat Med 19: 1584–1596. 10.1038/nm.3407  [DOI](https://doi.org/10.1038/nm.3407) | [PMC free article](/articles/PMC4080800/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24309662/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Med&title=Development,%20maintenance%20and%20disruption%20of%20the%20blood-brain%20barrier&author=B%20Obermeier&author=R%20Daneman&author=RM%20Ransohoff&volume=19&publication_year=2013&pages=1584-1596&pmid=24309662&doi=10.1038/nm.3407&)

3. Beresford R, Ward A (1987) Haloperidol decanoate. A preliminary review of its pharmacodynamic and pharmacokinetic properties and therapeutic use in psychosis. Drugs 33: 31–49.  [DOI](https://doi.org/10.2165/00003495-198733010-00002) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/3545764/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drugs&title=Haloperidol%20decanoate.%20A%20preliminary%20review%20of%20its%20pharmacodynamic%20and%20pharmacokinetic%20properties%20and%20therapeutic%20use%20in%20psychosis&author=R%20Beresford&author=A%20Ward&volume=33&publication_year=1987&pages=31-49&pmid=3545764&doi=10.2165/00003495-198733010-00002&)

4. Parkes GE (1982) Haloperidol: profile of side effects. Johnson DAW, editor. Haloperidol decanoate and the treatment of chronic schizophrenia. New York: ADIS Press; pp. 58–63.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Haloperidol%20decanoate%20and%20the%20treatment%20of%20chronic%20schizophrenia&author=GE%20Parkes&author=DAW%20Johnson&publication_year=1982&)

5. Soares-Weiser K, Fernandez HH (2007) Tardive dyskinesia. Semin Neurol 27: 159–169.  [DOI](https://doi.org/10.1055/s-2007-971169) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17390261/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Semin%20Neurol&title=Tardive%20dyskinesia&author=K%20Soares-Weiser&author=HH%20Fernandez&volume=27&publication_year=2007&pages=159-169&pmid=17390261&doi=10.1055/s-2007-971169&)

6. Crowley JJ, Kim Y, Szatkiewicz JP, Pratt AL, Quackenbush CR, et al. (2012) Genome-wide association mapping of loci for antipsychotic-induced extrapyramidal symptoms in mice. Mamm Genome 23: 322–335. 10.1007/s00335-011-9385-8  [DOI](https://doi.org/10.1007/s00335-011-9385-8) | [PMC free article](/articles/PMC3356790/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22207321/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mamm%20Genome&title=Genome-wide%20association%20mapping%20of%20loci%20for%20antipsychotic-induced%20extrapyramidal%20symptoms%20in%20mice&author=JJ%20Crowley&author=Y%20Kim&author=JP%20Szatkiewicz&author=AL%20Pratt&author=CR%20Quackenbush&volume=23&publication_year=2012&pages=322-335&pmid=22207321&doi=10.1007/s00335-011-9385-8&)

7. Flint J, Valdar W, Shifman S, Mott R (2005) Strategies for mapping and cloning quantitative trait genes in rodents. Nat Rev Genet 6: 271–286.  [DOI](https://doi.org/10.1038/nrg1576) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15803197/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Genet&title=Strategies%20for%20mapping%20and%20cloning%20quantitative%20trait%20genes%20in%20rodents&author=J%20Flint&author=W%20Valdar&author=S%20Shifman&author=R%20Mott&volume=6&publication_year=2005&pages=271-286&pmid=15803197&doi=10.1038/nrg1576&)

8. Nadeau JH, Frankel WN (2000) The roads from phenotypic variation to gene discovery: mutagenesis versus QTLs. Nat Genet 25: 381–384.  [DOI](https://doi.org/10.1038/78051) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10932178/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=The%20roads%20from%20phenotypic%20variation%20to%20gene%20discovery:%20mutagenesis%20versus%20QTLs&author=JH%20Nadeau&author=WN%20Frankel&volume=25&publication_year=2000&pages=381-384&pmid=10932178&doi=10.1038/78051&)

9. Payseur BA, Place M (2007) Prospects for association mapping in classical inbred mouse strains. Genetics 175: 1999–2008.  [DOI](https://doi.org/10.1534/genetics.106.067868) | [PMC free article](/articles/PMC1855108/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17277361/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genetics&title=Prospects%20for%20association%20mapping%20in%20classical%20inbred%20mouse%20strains&author=BA%20Payseur&author=M%20Place&volume=175&publication_year=2007&pages=1999-2008&pmid=17277361&doi=10.1534/genetics.106.067868&)

10. Su WL, Sieberts SK, Kleinhanz RR, Lux K, Millstein J, et al. (2010) Assessing the prospects of genome-wide association studies performed in inbred mice. Mamm Genome 21: 143–152. 10.1007/s00335-010-9249-7  [DOI](https://doi.org/10.1007/s00335-010-9249-7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20135320/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mamm%20Genome&title=Assessing%20the%20prospects%20of%20genome-wide%20association%20studies%20performed%20in%20inbred%20mice&author=WL%20Su&author=SK%20Sieberts&author=RR%20Kleinhanz&author=K%20Lux&author=J%20Millstein&volume=21&publication_year=2010&pages=143-152&pmid=20135320&doi=10.1007/s00335-010-9249-7&)

11. Keane TM, Goodstadt L, Danecek P, White MA, Wong K, et al. (2011) Mouse genomic variation and its effect on phenotypes and gene regulation. Nature 477: 289–294. 10.1038/nature10413  [DOI](https://doi.org/10.1038/nature10413) | [PMC free article](/articles/PMC3276836/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21921910/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Mouse%20genomic%20variation%20and%20its%20effect%20on%20phenotypes%20and%20gene%20regulation&author=TM%20Keane&author=L%20Goodstadt&author=P%20Danecek&author=MA%20White&author=K%20Wong&volume=477&publication_year=2011&pages=289-294&pmid=21921910&doi=10.1038/nature10413&)

12. Zheng M, Dill D, Peltz G (2012) A better prognosis for genetic association studies in mice. Trends Genet 28: 62–69. 10.1016/j.tig.2011.10.006  [DOI](https://doi.org/10.1016/j.tig.2011.10.006) | [PMC free article](/articles/PMC3268904/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22118772/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Trends%20Genet&title=A%20better%20prognosis%20for%20genetic%20association%20studies%20in%20mice&author=M%20Zheng&author=D%20Dill&author=G%20Peltz&volume=28&publication_year=2012&pages=62-69&pmid=22118772&doi=10.1016/j.tig.2011.10.006&)

13. Lappalainen T, Sammeth M, Friedlander MR, t Hoen PA, Monlong J, et al. (2013) Transcriptome and genome sequencing uncovers functional variation in humans. Nature 501: 506–511. 10.1038/nature12531  [DOI](https://doi.org/10.1038/nature12531) | [PMC free article](/articles/PMC3918453/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24037378/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Transcriptome%20and%20genome%20sequencing%20uncovers%20functional%20variation%20in%20humans&author=T%20Lappalainen&author=M%20Sammeth&author=MR%20Friedlander&author=PA%20t%20Hoen&author=J%20Monlong&volume=501&publication_year=2013&pages=506-511&pmid=24037378&doi=10.1038/nature12531&)

14. Liao G, Wang J, Guo J, Allard J, Cheng J, et al. (2004) In Silico Genetics: Identification of a Functional Element Regulating H2-Ea Gene Expression. Science 306: 690–695.  [DOI](https://doi.org/10.1126/science.1100636) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15499019/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=In%20Silico%20Genetics:%20Identification%20of%20a%20Functional%20Element%20Regulating%20H2-Ea%20Gene%20Expression&author=G%20Liao&author=J%20Wang&author=J%20Guo&author=J%20Allard&author=J%20Cheng&volume=306&publication_year=2004&pages=690-695&pmid=15499019&doi=10.1126/science.1100636&)

15. Li H, Durbin R (2009) Fast and accurate short read alignment with Burrows-Wheeler transform. Bioinformatics 25: 1754–1760. 10.1093/bioinformatics/btp324  [DOI](https://doi.org/10.1093/bioinformatics/btp324) | [PMC free article](/articles/PMC2705234/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19451168/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=Fast%20and%20accurate%20short%20read%20alignment%20with%20Burrows-Wheeler%20transform&author=H%20Li&author=R%20Durbin&volume=25&publication_year=2009&pages=1754-1760&pmid=19451168&doi=10.1093/bioinformatics/btp324&)

16. Li H, Handsaker B, Wysoker A, Fennell T, Ruan J, et al. (2009) The Sequence Alignment/Map format and SAMtools. Bioinformatics 25: 2078–2079. 10.1093/bioinformatics/btp352  [DOI](https://doi.org/10.1093/bioinformatics/btp352) | [PMC free article](/articles/PMC2723002/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19505943/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=The%20Sequence%20Alignment/Map%20format%20and%20SAMtools&author=H%20Li&author=B%20Handsaker&author=A%20Wysoker&author=T%20Fennell&author=J%20Ruan&volume=25&publication_year=2009&pages=2078-2079&pmid=19505943&doi=10.1093/bioinformatics/btp352&)

17. Peltz G, Zaas AK, Zheng M, Solis NV, Zhang MX, et al. (2011) Next-generation computational genetic analysis: multiple complement alleles control survival after Candida Albicans infection. Infect Immun 79: 4472–4479. 10.1128/IAI.05666-11  [DOI](https://doi.org/10.1128/IAI.05666-11) | [PMC free article](/articles/PMC3257944/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21875959/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Infect%20Immun&title=Next-generation%20computational%20genetic%20analysis:%20multiple%20complement%20alleles%20control%20survival%20after%20Candida%20Albicans%20infection&author=G%20Peltz&author=AK%20Zaas&author=M%20Zheng&author=NV%20Solis&author=MX%20Zhang&volume=79&publication_year=2011&pages=4472-4479&pmid=21875959&doi=10.1128/IAI.05666-11&)

18. Wang J, Peltz G (2005) Haplotype-based computational genetic analysis in mice. computational genetics and genomics: new tools for understanding disease. Totowa (New Jersey): Humana Press Inc; pp. 51–70.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=computational%20genetics%20and%20genomics:%20new%20tools%20for%20understanding%20disease&author=J%20Wang&author=G%20Peltz&publication_year=2005&)

19. Hollander M, Wolfe DA (1973) Nonparametric statistical methods. New York: John Wiley & Sons;  [Google Scholar](https://scholar.google.com/scholar_lookup?Hollander%20M,%20Wolfe%20DA%20(1973)%20Nonparametric%20statistical%20methods.%20New%20York:%20John%20Wiley%20&%20Sons;)

20. Benjamini Y, Hochberg Y (1995) Controlling the false discovery rate: a practical and powerful approach to multiple testing. J Roy Stat Soc B 57: 289–300.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Roy%20Stat%20Soc%20B&title=Controlling%20the%20false%20discovery%20rate:%20a%20practical%20and%20powerful%20approach%20to%20multiple%20testing&author=Y%20Benjamini&author=Y%20Hochberg&volume=57&publication_year=1995&pages=289-300&)

21. Kilkenny C, Browne WJ, Cuthill IC, Emerson M, Altman DG (2010) Improving bioscience research reporting: the ARRIVE guidelines for reporting animal research. PLoS Biol 8: e1000412 10.1371/journal.pbio.1000412  [DOI](https://doi.org/10.1371/journal.pbio.1000412) | [PMC free article](/articles/PMC2893951/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20613859/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Biol&title=Improving%20bioscience%20research%20reporting:%20the%20ARRIVE%20guidelines%20for%20reporting%20animal%20research&author=C%20Kilkenny&author=WJ%20Browne&author=IC%20Cuthill&author=M%20Emerson&author=DG%20Altman&volume=8&publication_year=2010&pages=e1000412&pmid=20613859&doi=10.1371/journal.pbio.1000412&)

22. Crowley JJ, Adkins DE, Pratt AL, Quackenbush CR, van den Oord EJ, et al. (2012) Antipsychotic-induced vacuous chewing movements and extrapyramidal side effects are highly heritable in mice. Pharmacogenomics J 12: 147–155. 10.1038/tpj.2010.82  [DOI](https://doi.org/10.1038/tpj.2010.82) | [PMC free article](/articles/PMC3117923/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21079646/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=Antipsychotic-induced%20vacuous%20chewing%20movements%20and%20extrapyramidal%20side%20effects%20are%20highly%20heritable%20in%20mice&author=JJ%20Crowley&author=DE%20Adkins&author=AL%20Pratt&author=CR%20Quackenbush&author=EJ%20van%20den%20Oord&volume=12&publication_year=2012&pages=147-155&pmid=21079646&doi=10.1038/tpj.2010.82&)

23. Eberlin LS, Ferreira CR, Dill AL, Ifa DR, Cheng L, et al. (2011) Nondestructive, histologically compatible tissue imaging by desorption electrospray ionization mass spectrometry. Chembiochem 12: 2129–2132. 10.1002/cbic.201100411  [DOI](https://doi.org/10.1002/cbic.201100411) | [PMC free article](/articles/PMC3678526/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21793152/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Chembiochem&title=Nondestructive,%20histologically%20compatible%20tissue%20imaging%20by%20desorption%20electrospray%20ionization%20mass%20spectrometry&author=LS%20Eberlin&author=CR%20Ferreira&author=AL%20Dill&author=DR%20Ifa&author=L%20Cheng&volume=12&publication_year=2011&pages=2129-2132&pmid=21793152&doi=10.1002/cbic.201100411&)

24. Giegling I, Drago A, Schafer M, Hartmann AM, Sander T, et al. (2011) Lack of association between 71 variations located in candidate genes and response to acute haloperidol treatment. Psychopharmacology (Berl) 214: 719–728. 10.1007/s00213-010-2080-8  [DOI](https://doi.org/10.1007/s00213-010-2080-8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21079921/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Psychopharmacology%20(Berl)&title=Lack%20of%20association%20between%2071%20variations%20located%20in%20candidate%20genes%20and%20response%20to%20acute%20haloperidol%20treatment&author=I%20Giegling&author=A%20Drago&author=M%20Schafer&author=AM%20Hartmann&author=T%20Sander&volume=214&publication_year=2011&pages=719-728&pmid=21079921&doi=10.1007/s00213-010-2080-8&)

25. Rohlfs RV, Weir BS (2008) Distributions of Hardy-Weinberg equilibrium test statistics. Genetics 180: 1609–1616. 10.1534/genetics.108.088005  [DOI](https://doi.org/10.1534/genetics.108.088005) | [PMC free article](/articles/PMC2581961/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18791257/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genetics&title=Distributions%20of%20Hardy-Weinberg%20equilibrium%20test%20statistics&author=RV%20Rohlfs&author=BS%20Weir&volume=180&publication_year=2008&pages=1609-1616&pmid=18791257&doi=10.1534/genetics.108.088005&)

26. Kawanobe T, Kogure S, Nakamura S, Sato M, Katayama K, et al. (2012) Expression of human ABCB5 confers resistance to taxanes and anthracyclines. Biochem Biophys Res Commun 418: 736–741. 10.1016/j.bbrc.2012.01.090  [DOI](https://doi.org/10.1016/j.bbrc.2012.01.090) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22306008/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem%20Biophys%20Res%20Commun&title=Expression%20of%20human%20ABCB5%20confers%20resistance%20to%20taxanes%20and%20anthracyclines&author=T%20Kawanobe&author=S%20Kogure&author=S%20Nakamura&author=M%20Sato&author=K%20Katayama&volume=418&publication_year=2012&pages=736-741&pmid=22306008&doi=10.1016/j.bbrc.2012.01.090&)

27. Frank NY, Margaryan A, Huang Y, Schatton T, Waaga-Gasser AM, et al. (2005) ABCB5-mediated doxorubicin transport and chemoresistance in human malignant melanoma. Cancer Res 65: 4320–4333.  [DOI](https://doi.org/10.1158/0008-5472.CAN-04-3327) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15899824/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=ABCB5-mediated%20doxorubicin%20transport%20and%20chemoresistance%20in%20human%20malignant%20melanoma&author=NY%20Frank&author=A%20Margaryan&author=Y%20Huang&author=T%20Schatton&author=AM%20Waaga-Gasser&volume=65&publication_year=2005&pages=4320-4333&pmid=15899824&doi=10.1158/0008-5472.CAN-04-3327&)

28. Frank NY, Pendse SS, Lapchak PH, Margaryan A, Shlain D, et al. (2003) Regulation of progenitor cell fusion by ABCB5 P-glycoprotein, a novel human ATP-binding cassette transporter. J Biol Chem 278: 47156–47165.  [DOI](https://doi.org/10.1074/jbc.M308700200) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12960149/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=Regulation%20of%20progenitor%20cell%20fusion%20by%20ABCB5%20P-glycoprotein,%20a%20novel%20human%20ATP-binding%20cassette%20transporter&author=NY%20Frank&author=SS%20Pendse&author=PH%20Lapchak&author=A%20Margaryan&author=D%20Shlain&volume=278&publication_year=2003&pages=47156-47165&pmid=12960149&doi=10.1074/jbc.M308700200&)

29. Fukunaga-Kalabis M, Martinez G, Nguyen TK, Kim D, Santiago-Walker A, et al. (2010) Tenascin-C promotes melanoma progression by maintaining the ABCB5-positive side population. Oncogene 29: 6115–6124. 10.1038/onc.2010.350  [DOI](https://doi.org/10.1038/onc.2010.350) | [PMC free article](/articles/PMC2991494/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20729912/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncogene&title=Tenascin-C%20promotes%20melanoma%20progression%20by%20maintaining%20the%20ABCB5-positive%20side%20population&author=M%20Fukunaga-Kalabis&author=G%20Martinez&author=TK%20Nguyen&author=D%20Kim&author=A%20Santiago-Walker&volume=29&publication_year=2010&pages=6115-6124&pmid=20729912&doi=10.1038/onc.2010.350&)

30. Wilson BJ, Schatton T, Zhan Q, Gasser M, Ma J, et al. (2011) ABCB5 identifies a therapy-refractory tumor cell population in colorectal cancer patients. Cancer Res 71: 5307–5316. 10.1158/0008-5472.CAN-11-0221  [DOI](https://doi.org/10.1158/0008-5472.CAN-11-0221) | [PMC free article](/articles/PMC3395026/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21652540/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=ABCB5%20identifies%20a%20therapy-refractory%20tumor%20cell%20population%20in%20colorectal%20cancer%20patients&author=BJ%20Wilson&author=T%20Schatton&author=Q%20Zhan&author=M%20Gasser&author=J%20Ma&volume=71&publication_year=2011&pages=5307-5316&pmid=21652540&doi=10.1158/0008-5472.CAN-11-0221&)

31. Seeger MA, van Veen HW (2009) Molecular basis of multidrug transport by ABC transporters. Biochim Biophys Acta 1794: 725–737. 10.1016/j.bbapap.2008.12.004  [DOI](https://doi.org/10.1016/j.bbapap.2008.12.004) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19135557/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochim%20Biophys%20Acta&title=Molecular%20basis%20of%20multidrug%20transport%20by%20ABC%20transporters&author=MA%20Seeger&author=HW%20van%20Veen&volume=1794&publication_year=2009&pages=725-737&pmid=19135557&doi=10.1016/j.bbapap.2008.12.004&)

32. Nadeau JH, Singer JB, Matin A, Lander ES (2000) Analysing complex genetic traits with chromosome substitution strains. Nat Genet 24: 221–225.  [DOI](https://doi.org/10.1038/73427) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10700173/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Analysing%20complex%20genetic%20traits%20with%20chromosome%20substitution%20strains&author=JH%20Nadeau&author=JB%20Singer&author=A%20Matin&author=ES%20Lander&volume=24&publication_year=2000&pages=221-225&pmid=10700173&doi=10.1038/73427&)

33. Eyles DW, Avent KM, Stedman TJ, Pond SM (1997) Two pyridinium metabolites of haloperidol are present in the brain of patients at post-mortem. Life Sci 60: 529–534.  [DOI](https://doi.org/10.1016/s0024-3205(96)00656-x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9042387/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Life%20Sci&title=Two%20pyridinium%20metabolites%20of%20haloperidol%20are%20present%20in%20the%20brain%20of%20patients%20at%20post-mortem&author=DW%20Eyles&author=KM%20Avent&author=TJ%20Stedman&author=SM%20Pond&volume=60&publication_year=1997&pages=529-534&pmid=9042387&doi=10.1016/s0024-3205(96)00656-x&)

34. Igarashi K, Kasuya F, Fukui M, Usuki E, Castagnoli N Jr. (1995) Studies on the metabolism of haloperidol (HP): the role of CYP3A in the production of the neurotoxic pyridinium metabolite HPP+ found in rat brain following ip administration of HP. Life Sci 57: 2439–2446.  [DOI](https://doi.org/10.1016/0024-3205(95)02240-5) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8847965/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Life%20Sci&title=Studies%20on%20the%20metabolism%20of%20haloperidol%20(HP):%20the%20role%20of%20CYP3A%20in%20the%20production%20of%20the%20neurotoxic%20pyridinium%20metabolite%20HPP+%20found%20in%20rat%20brain%20following%20ip%20administration%20of%20HP&author=K%20Igarashi&author=F%20Kasuya&author=M%20Fukui&author=E%20Usuki&author=N%20Castagnoli&volume=57&publication_year=1995&pages=2439-2446&pmid=8847965&doi=10.1016/0024-3205(95)02240-5&)

35. Bloomquist J, King E, Wright A, Mytilineou C, Kimura K, et al. (1994) 1-Methyl-4-phenylpyridinium-like neurotoxicity of a pyridinium metabolite derived from haloperidol: cell culture and neurotransmitter uptake studies. J Pharmacol Exp Ther 270: 822–830.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/8071874/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharmacol%20Exp%20Ther&title=1-Methyl-4-phenylpyridinium-like%20neurotoxicity%20of%20a%20pyridinium%20metabolite%20derived%20from%20haloperidol:%20cell%20culture%20and%20neurotransmitter%20uptake%20studies&author=J%20Bloomquist&author=E%20King&author=A%20Wright&author=C%20Mytilineou&author=K%20Kimura&volume=270&publication_year=1994&pages=822-830&pmid=8071874&)

36. Rollema H, Skolnik M, D’Engelbronner J, Igarashi K, Usuki E, et al. (1994) MPP(+)-like neurotoxicity of a pyridinium metabolite derived from haloperidol: in vivo microdialysis and in vitro mitochondrial studies. J Pharmacol Exp Ther 268: 380–387.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/8301579/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharmacol%20Exp%20Ther&title=MPP(+)-like%20neurotoxicity%20of%20a%20pyridinium%20metabolite%20derived%20from%20haloperidol:%20in%20vivo%20microdialysis%20and%20in%20vitro%20mitochondrial%20studies&author=H%20Rollema&author=M%20Skolnik&author=J%20D%E2%80%99Engelbronner&author=K%20Igarashi&author=E%20Usuki&volume=268&publication_year=1994&pages=380-387&pmid=8301579&)

37. Decleves X, Jacob A, Yousif S, Shawahna R, Potin S, et al. (2011) Interplay of drug metabolizing CYP450 enzymes and ABC transporters in the blood-brain barrier. Curr Drug Metab 12: 732–741.  [DOI](https://doi.org/10.2174/138920011798357024) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21623707/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Drug%20Metab&title=Interplay%20of%20drug%20metabolizing%20CYP450%20enzymes%20and%20ABC%20transporters%20in%20the%20blood-brain%20barrier&author=X%20Decleves&author=A%20Jacob&author=S%20Yousif&author=R%20Shawahna&author=S%20Potin&volume=12&publication_year=2011&pages=732-741&pmid=21623707&doi=10.2174/138920011798357024&)

38. Blandini F, Armentero MT (2012) Animal models of Parkinson’s disease. FEBS J 279: 1156–1166. 10.1111/j.1742-4658.2012.08491.x  [DOI](https://doi.org/10.1111/j.1742-4658.2012.08491.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22251459/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=FEBS%20J&title=Animal%20models%20of%20Parkinson%E2%80%99s%20disease&author=F%20Blandini&author=MT%20Armentero&volume=279&publication_year=2012&pages=1156-1166&pmid=22251459&doi=10.1111/j.1742-4658.2012.08491.x&)

39. Ramsay RR, Singer TP (1986) Energy-dependent uptake of N-methyl-4-phenylpyridinium, the neurotoxic metabolite of 1-methyl-4-phenyl-1,2,3,6-tetrahydropyridine, by mitochondria. J Biol Chem 261: 7585–7587.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/3486869/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=Energy-dependent%20uptake%20of%20N-methyl-4-phenylpyridinium,%20the%20neurotoxic%20metabolite%20of%201-methyl-4-phenyl-1,2,3,6-tetrahydropyridine,%20by%20mitochondria&author=RR%20Ramsay&author=TP%20Singer&volume=261&publication_year=1986&pages=7585-7587&pmid=3486869&)

40. Ramsay RR, Salach JI, Singer TP (1986) Uptake of the neurotoxin 1-methyl-4-phenylpyridine (MPP+) by mitochondria and its relation to the inhibition of the mitochondrial oxidation of NAD+-linked substrates by MPP+. Biochem Biophys Res Commun 134: 743–748.  [DOI](https://doi.org/10.1016/s0006-291x(86)80483-1) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/2868716/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem%20Biophys%20Res%20Commun&title=Uptake%20of%20the%20neurotoxin%201-methyl-4-phenylpyridine%20(MPP+)%20by%20mitochondria%20and%20its%20relation%20to%20the%20inhibition%20of%20the%20mitochondrial%20oxidation%20of%20NAD+-linked%20substrates%20by%20MPP+&author=RR%20Ramsay&author=JI%20Salach&author=TP%20Singer&volume=134&publication_year=1986&pages=743-748&pmid=2868716&doi=10.1016/s0006-291x(86)80483-1&)

41. Javitch JA, Snyder SH (1984) Uptake of MPP(+) by dopamine neurons explains selectivity of parkinsonism-inducing neurotoxin, MPTP. Eur J Pharmacol 106: 455–456.  [DOI](https://doi.org/10.1016/0014-2999(84)90740-4) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/6335692/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Pharmacol&title=Uptake%20of%20MPP(+)%20by%20dopamine%20neurons%20explains%20selectivity%20of%20parkinsonism-inducing%20neurotoxin,%20MPTP&author=JA%20Javitch&author=SH%20Snyder&volume=106&publication_year=1984&pages=455-456&pmid=6335692&doi=10.1016/0014-2999(84)90740-4&)

42. Nicklas WJ, Vyas I, Heikkila RE (1985) Inhibition of NADH-linked oxidation in brain mitochondria by 1-methyl-4-phenyl-pyridine, a metabolite of the neurotoxin, 1-methyl-4-phenyl-1,2,5,6-tetrahydropyridine. Life Sci 36: 2503–2508.  [DOI](https://doi.org/10.1016/0024-3205(85)90146-8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/2861548/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Life%20Sci&title=Inhibition%20of%20NADH-linked%20oxidation%20in%20brain%20mitochondria%20by%201-methyl-4-phenyl-pyridine,%20a%20metabolite%20of%20the%20neurotoxin,%201-methyl-4-phenyl-1,2,5,6-tetrahydropyridine&author=WJ%20Nicklas&author=I%20Vyas&author=RE%20Heikkila&volume=36&publication_year=1985&pages=2503-2508&pmid=2861548&doi=10.1016/0024-3205(85)90146-8&)

43. Viaggi C, Vaglini F, Pardini C, Sgado P, Caramelli A, et al. (2007) CYP 2E1 mutant mice are resistant to DDC-induced enhancement of MPTP toxicity. J Neural Transm Suppl: 159–163.  [DOI](https://doi.org/10.1007/978-3-211-73574-9_20) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17982890/)

44. Markey SP, Johannessen JN, Chiueh CC, Burns RS, Herkenham MA (1984) Intraneuronal generation of a pyridinium metabolite may cause drug-induced parkinsonism. Nature 311: 464–467.  [DOI](https://doi.org/10.1038/311464a0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/6332988/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Intraneuronal%20generation%20of%20a%20pyridinium%20metabolite%20may%20cause%20drug-induced%20parkinsonism&author=SP%20Markey&author=JN%20Johannessen&author=CC%20Chiueh&author=RS%20Burns&author=MA%20Herkenham&volume=311&publication_year=1984&pages=464-467&pmid=6332988&doi=10.1038/311464a0&)

45. Javitch JA, Uhl GR, Snyder SH (1984) Parkinsonism-inducing neurotoxin, N-methyl-4-phenyl-1,2,3,6-tetrahydropyridine: characterization and localization of receptor binding sites in rat and human brain. Proc Natl Acad Sci U S A 81: 4591–4595.  [DOI](https://doi.org/10.1073/pnas.81.14.4591) | [PMC free article](/articles/PMC345637/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/6611553/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&title=Parkinsonism-inducing%20neurotoxin,%20N-methyl-4-phenyl-1,2,3,6-tetrahydropyridine:%20characterization%20and%20localization%20of%20receptor%20binding%20sites%20in%20rat%20and%20human%20brain&author=JA%20Javitch&author=GR%20Uhl&author=SH%20Snyder&volume=81&publication_year=1984&pages=4591-4595&pmid=6611553&doi=10.1073/pnas.81.14.4591&)

46. Rappold PM, Cui M, Chesser AS, Tibbett J, Grima JC, et al. (2011) Paraquat neurotoxicity is mediated by the dopamine transporter and organic cation transporter-3. Proc Natl Acad Sci U S A 108: 20766–20771. 10.1073/pnas.1115141108  [DOI](https://doi.org/10.1073/pnas.1115141108) | [PMC free article](/articles/PMC3251116/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22143804/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&title=Paraquat%20neurotoxicity%20is%20mediated%20by%20the%20dopamine%20transporter%20and%20organic%20cation%20transporter-3&author=PM%20Rappold&author=M%20Cui&author=AS%20Chesser&author=J%20Tibbett&author=JC%20Grima&volume=108&publication_year=2011&pages=20766-20771&pmid=22143804&doi=10.1073/pnas.1115141108&)

47. Miksys SL, Tyndale RF (2002) Drug-metabolizing cytochrome P450s in the brain. J Psychiatry Neurosci 27: 406–415.  [PMC free article](/articles/PMC161713/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12491573/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Psychiatry%20Neurosci&title=Drug-metabolizing%20cytochrome%20P450s%20in%20the%20brain&author=SL%20Miksys&author=RF%20Tyndale&volume=27&publication_year=2002&pages=406-415&pmid=12491573&)

48. Lowndes HE, Beiswanger CM, Philbert MA, Reuhl KR (1994) Substrates for neural metabolism of xenobiotics in adult and developing brain. Neurotoxicology 15: 61–73.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/8090363/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neurotoxicology&title=Substrates%20for%20neural%20metabolism%20of%20xenobiotics%20in%20adult%20and%20developing%20brain&author=HE%20Lowndes&author=CM%20Beiswanger&author=MA%20Philbert&author=KR%20Reuhl&volume=15&publication_year=1994&pages=61-73&pmid=8090363&)

49. Manto M (2012) Toxic agents causing cerebellar ataxias. Handb Clin Neurol 103: 201–213. 10.1016/B978-0-444-51892-7.00012-7  [DOI](https://doi.org/10.1016/B978-0-444-51892-7.00012-7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21827890/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Handb%20Clin%20Neurol&title=Toxic%20agents%20causing%20cerebellar%20ataxias&author=M%20Manto&volume=103&publication_year=2012&pages=201-213&pmid=21827890&doi=10.1016/B978-0-444-51892-7.00012-7&)

50. Muller DJ, Schulze TG, Knapp M, Held T, Krauss H, et al. (2001) Familial occurrence of tardive dyskinesia. Acta Psychiatr Scand 104: 375–379.  [DOI](https://doi.org/10.1034/j.1600-0447.2001.00401.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11722319/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Acta%20Psychiatr%20Scand&title=Familial%20occurrence%20of%20tardive%20dyskinesia&author=DJ%20Muller&author=TG%20Schulze&author=M%20Knapp&author=T%20Held&author=H%20Krauss&volume=104&publication_year=2001&pages=375-379&pmid=11722319&doi=10.1034/j.1600-0447.2001.00401.x&)

51. Bakker PR, van Harten PN, van Os J (2008) Antipsychotic-induced tardive dyskinesia and polymorphic variations in COMT, DRD2, CYP1A2 and MnSOD genes: a meta-analysis of pharmacogenetic interactions. Mol Psychiatry 13: 544–556. 10.1038/sj.mp.4002142  [DOI](https://doi.org/10.1038/sj.mp.4002142) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18180754/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Psychiatry&title=Antipsychotic-induced%20tardive%20dyskinesia%20and%20polymorphic%20variations%20in%20COMT,%20DRD2,%20CYP1A2%20and%20MnSOD%20genes:%20a%20meta-analysis%20of%20pharmacogenetic%20interactions&author=PR%20Bakker&author=PN%20van%20Harten&author=J%20van%20Os&volume=13&publication_year=2008&pages=544-556&pmid=18180754&doi=10.1038/sj.mp.4002142&)

52. Arranz MJ, de Leon J (2007) Pharmacogenetics and pharmacogenomics of schizophrenia: a review of last decade of research. Mol Psychiatry 12: 707–747.  [DOI](https://doi.org/10.1038/sj.mp.4002009) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17549063/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Psychiatry&title=Pharmacogenetics%20and%20pharmacogenomics%20of%20schizophrenia:%20a%20review%20of%20last%20decade%20of%20research&author=MJ%20Arranz&author=J%20de%20Leon&volume=12&publication_year=2007&pages=707-747&pmid=17549063&doi=10.1038/sj.mp.4002009&)

53. Aberg K, Adkins DE, Bukszar J, Webb BT, Caroff SN, et al. (2010) Genomewide association study of movement-related adverse antipsychotic effects. Biol Psychiatry 67: 279–282. 10.1016/j.biopsych.2009.08.036  [DOI](https://doi.org/10.1016/j.biopsych.2009.08.036) | [PMC free article](/articles/PMC3388725/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19875103/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biol%20Psychiatry&title=Genomewide%20association%20study%20of%20movement-related%20adverse%20antipsychotic%20effects&author=K%20Aberg&author=DE%20Adkins&author=J%20Bukszar&author=BT%20Webb&author=SN%20Caroff&volume=67&publication_year=2010&pages=279-282&pmid=19875103&doi=10.1016/j.biopsych.2009.08.036&)

54. Moitra K, Scally M, McGee K, Lancaster G, Gold B, et al. (2011) Molecular evolutionary analysis of ABCB5: the ancestral gene is a full transporter with potentially deleterious single nucleotide polymorphisms. PLoS One 6: e16318 10.1371/journal.pone.0016318  [DOI](https://doi.org/10.1371/journal.pone.0016318) | [PMC free article](/articles/PMC3029322/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21298007/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=Molecular%20evolutionary%20analysis%20of%20ABCB5:%20the%20ancestral%20gene%20is%20a%20full%20transporter%20with%20potentially%20deleterious%20single%20nucleotide%20polymorphisms&author=K%20Moitra&author=M%20Scally&author=K%20McGee&author=G%20Lancaster&author=B%20Gold&volume=6&publication_year=2011&pages=e16318&pmid=21298007&doi=10.1371/journal.pone.0016318&)

55. Correll CU, Schenk EM (2008) Tardive dyskinesia and new antipsychotics. Curr Opin Psychiatry 21: 151–156. 10.1097/YCO.0b013e3282f53132  [DOI](https://doi.org/10.1097/YCO.0b013e3282f53132) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18332662/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Opin%20Psychiatry&title=Tardive%20dyskinesia%20and%20new%20antipsychotics&author=CU%20Correll&author=EM%20Schenk&volume=21&publication_year=2008&pages=151-156&pmid=18332662&doi=10.1097/YCO.0b013e3282f53132&)

56. Trusheim MR, Burgess B, Hu SX, Long T, Averbuch SD, et al. (2011) Quantifying factors for the success of stratified medicine. Nat Rev Drug Discov 10: 817–833. 10.1038/nrd3557  [DOI](https://doi.org/10.1038/nrd3557) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22037040/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Drug%20Discov&title=Quantifying%20factors%20for%20the%20success%20of%20stratified%20medicine&author=MR%20Trusheim&author=B%20Burgess&author=SX%20Hu&author=T%20Long&author=SD%20Averbuch&volume=10&publication_year=2011&pages=817-833&pmid=22037040&doi=10.1038/nrd3557&)

57. Guo Y, Shafer S, Weller P, Usuka J, Peltz G (2005) Pharmacogenomics and drug development. Pharmacogenomics 6: 857–864.  [DOI](https://doi.org/10.2217/14622416.6.8.857) | [PMC free article](/articles/PMC1473028/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16296948/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Pharmacogenomics%20and%20drug%20development&author=Y%20Guo&author=S%20Shafer&author=P%20Weller&author=J%20Usuka&author=G%20Peltz&volume=6&publication_year=2005&pages=857-864&pmid=16296948&doi=10.2217/14622416.6.8.857&)
