# A Novel Admixture-Based Pharmacogenetic Approach to Refine Warfarin Dosing in Caribbean Hispanics

## Metadata
**Authors:** Jorge Duconge, Alga S Ramos, Karla Claudio-Campos, Giselle Rivera-Miranda, Luis Bermúdez-Bosch, Jessicca Y Renta, Carmen L Cadilla, Iadelisse Cruz, Juan F Feliu, Cunegundo Vergara, Gualberto Ruaño
**Journal:** PLoS ONE
**Date:** 2016 Jan 8
**DOI:** [10.1371/journal.pone.0145480](https://doi.org/10.1371/journal.pone.0145480)
**PMID:** 26745506
**PMCID:** PMC4706412
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706412/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC4706412/pdf/pone.0145480.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC4706412/pdf/pone.0145480.pdf)

## Abstract

**Aim:** 
This study is aimed at developing a novel admixture-adjusted pharmacogenomic approach to individually refine warfarin dosing in Caribbean Hispanic patients.

**Patients & Methods:** 
A multiple linear regression analysis of effective warfarin doses versus relevant genotypes, admixture, clinical and demographic factors was performed in 255 patients and further validated externally in another cohort of 55 individuals.

**Results:** 
The admixture-adjusted, genotype-guided warfarin dosing refinement algorithm developed in Caribbean Hispanics showed better predictability (R2 = 0.70, MAE = 0.72mg/day) than a clinical algorithm that excluded genotypes and admixture (R2 = 0.60, MAE = 0.99mg/day), and outperformed two prior pharmacogenetic algorithms in predicting effective dose in this population. For patients at the highest risk of adverse events, 45.5% of the dose predictions using the developed pharmacogenetic model resulted in ideal dose as compared with only 29% when using the clinical non-genetic algorithm (p<0.001). The admixture-driven pharmacogenetic algorithm predicted 58% of warfarin dose variance when externally validated in 55 individuals from an independent validation cohort (MAE = 0.89 mg/day, 24% mean bias).

**Conclusions:** 
Results supported our rationale to incorporate individual’s genotypes and unique admixture metrics into pharmacogenetic refinement models in order to increase predictability when expanding them to admixed populations like Caribbean Hispanics.

**Trial Registration:** 
ClinicalTrials.gov NCT01318057

### Aim

This study is aimed at developing a novel admixture-adjusted pharmacogenomic approach to individually refine warfarin dosing in Caribbean Hispanic patients.

### Patients & Methods

A multiple linear regression analysis of effective warfarin doses versus relevant genotypes, admixture, clinical and demographic factors was performed in 255 patients and further validated externally in another cohort of 55 individuals.

### Results

The admixture-adjusted, genotype-guided warfarin dosing refinement algorithm developed in Caribbean Hispanics showed better predictability (R^2^ = 0.70, MAE = 0.72mg/day) than a clinical algorithm that excluded genotypes and admixture (R^2^ = 0.60, MAE = 0.99mg/day), and outperformed two prior pharmacogenetic algorithms in predicting effective dose in this population. For patients at the highest risk of adverse events, 45.5% of the dose predictions using the developed pharmacogenetic model resulted in ideal dose as compared with only 29% when using the clinical non-genetic algorithm (p<0.001). The admixture-driven pharmacogenetic algorithm predicted 58% of warfarin dose variance when externally validated in 55 individuals from an independent validation cohort (MAE = 0.89 mg/day, 24% mean bias).

### Conclusions

Results supported our rationale to incorporate individual’s genotypes and unique admixture metrics into pharmacogenetic refinement models in order to increase predictability when expanding them to admixed populations like Caribbean Hispanics.

### Trial Registration

ClinicalTrials.gov [NCT01318057](https://clinicaltrials.gov/ct2/show/NCT01318057)

## Introduction

Warfarin is an oral anticoagulant used to treat or prevent multiple thromboembolic disorders, including atrial fibrillation, heart valve replacement, recurrent stroke, deep vein thrombosis, pulmonary embolism, acute myocardial infarction and cerebrovascular accidents [[1](#pone.0145480.ref001)–[2](#pone.0145480.ref002)]. Although several clinical trials have compared its effectiveness and safety to the new oral anticoagulants (NOACs) [[3](#pone.0145480.ref003)], warfarin continues to be the standard of care in oral anticoagulation. According to IMS Health, over 19 million prescriptions of warfarin sodium were dispensed annually in the US by 2012, ranking #18^th^ overall [[4](#pone.0145480.ref004)–[5](#pone.0145480.ref005)]. However, it is still in the top 10 drug products by reports of adverse events in outpatients [[6](#pone.0145480.ref006)–[7](#pone.0145480.ref007)]. Despite its proven clinical benefits and its use for a long time, warfarin therapy is still a challenge due to a wide inter-individual variation in dose requirements, narrow therapeutic range and risk of serious bleeding [[1](#pone.0145480.ref001),[8](#pone.0145480.ref008)].

Pharmacogenetic-guided warfarin dosing algorithms have been developed to explain about 60% of dose variability in various populations, mostly in Caucasians [[1](#pone.0145480.ref001)]. Nonetheless, their impact on clinical outcomes remains controversial and reimbursement for genotyping is debated [[9](#pone.0145480.ref009)–[10](#pone.0145480.ref010)]. On the other hand, Hispanics have been largely excluded from the corresponding derivation cohorts [[11](#pone.0145480.ref011)–[13](#pone.0145480.ref013)]. Accordingly, these algorithms are limited to account for the effect of multi-hybrid admixtures and the unique stratification observed in Hispanics, which increase dramatically the disparities in translating benefits from pharmacogenomics to this medically underserved, minority population. We believe that a better approach for global pharmacogenetics (PGt) is to guide warfarin dosing by using a pharmacogenetic-based algorithm that also accounts for the effect of admixture or ancestry proportions.

Fostered by an urgent need to eliminate potential ethnic disparities in health care while implementing the personalized medicine paradigm, this study is aimed at developing a novel admixture-adjusted pharmacogenomic approach to predict individual maintenance doses of warfarin, after the fourth day of therapy (refinement model), in Caribbean Hispanic patients receiving anticoagulation therapy at the Veterans Affairs Caribbean Healthcare System (VACHS). We also compare the predictability of the developed algorithm in Caribbean Hispanics versus a clinical non-genetic algorithm and two other previously established pharmacogenetic-guided warfarin dosing models.

## Methods

### Study Design and Patient Cohort

All research activities involving human participants in this study were properly approved by the Institutional Review Boards (IRBs) at both VACHS (#00558) and UPR-MSC (A4070109) in November 2010 ([S1 Text](#pone.0145480.s001)). All clinical investigations were conducted according to the principles expressed in the Declaration of Helsinki. Written informed consent was obtained from each single participant of the study prior to enrollment ([S2 Text](#pone.0145480.s002) and [S3 Text](#pone.0145480.s003)).

This is an open-label, single-center, population-based, observational, retrospective cohort study ([ClinicalTrial.gov](http://ClinicalTrial.gov) Identifier [NCT01318057](https://clinicaltrials.gov/ct2/show/NCT01318057)) as depicted in [Fig 1](#pone.0145480.g001). Four hundred and twenty one (N = 421) warfarin-treated outpatients from the VACHS anticoagulation clinics in San Juan, Puerto Rico, were assessed for eligibility between February 2011 and March 2014. A total of 275 eligible patients who met inclusion criteria were approached regarding participation in this trial. Two patients declined to participate in the study, resulting in 273 subjects enrolled between March 22 and April 7, 2011. To be in compliance with the Ethical Principles for Medical Research Involving Human Subjects, the first participant was recruited 5 days after the study registration date in [ClinicalTrials.gov](http://ClinicalTrials.gov) (March 17, 2011). Another eighteen (18) patients were excluded from further analysis due to lack of complete clinical data from the CPRS and incomplete genotype information (e.g., poor call rates) or admixture estimates. This resulted in 255 patients available to include their clinical, demographic and genomic data for full analysis. Because this study required a blood sample at enrollment followed by a retrospective CPRS medical record review, no patients were withdrawn or lost to follow-up. The study completion date was July 2014. The authors confirm that all ongoing and related trials for this drug/intervention are registered.

### Fig 1. CONSORT flowchart illustrating an open-label, single-center, population-based, observational, retrospective cohorts study with cross-sectional genotyping analysis.

![Fig 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ec3f/4706412/70a46bf2e461/pone.0145480.g001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4706412_pone.0145480.g001.jpg)

The study uses existing data collected in the past (CPRS) to preliminarily identify eligible participants from the defined study population (i.e., Caribbean Hispanics), determine their stabilization status and retrieve relevant covariates for performing regression analysis and association testing. GNT stands for genotypes. PHT stands for phenotypes (e.g., therapeutic warfarin dose). CPRS stands for computerized patient record system.

The eligible candidates were Caribbean Hispanics from Puerto Rico (≥21 years-old). The anticoagulation clinic at this urban medical institution serves a population of over 1,640 patients, of which approximately 95% are Caribbean Hispanics who reside in the Commonwealth of Puerto Rico. The subjects who met the inclusion criteria–stable warfarin dose with at least three consecutive INR measurements on target (2–3 or 2.5–3.5, according to indication for warfarin use) for the same average weekly dose, no anemia, malignancies, renal failure, hepatic disease or any other acute illness, no use of other anticoagulants–were selected from the study population based on their warfarin initiation date (after January 2001). The requirement for warfarin therapy was determined on the basis of the American College of Chest Physicians (ACCP) evidence-based clinical practice guidelines [[14](#pone.0145480.ref014)–[15](#pone.0145480.ref015)]. Patients with low compliance (<80% medication possession ratio) were excluded. Although varying from case to case, the majority of patients were initiated on warfarin therapy following induction regimens that rely on age-adjusted fixed-dose schemes.

### Genetic Assays

The participants provided a small blood sample (5ml) for cross-sectional genotyping as well as to infer genetic structure and measure admixture. Whole blood samples were processed for genomic DNA extraction using a QIAamp DNA Blood Maxi Kit (spin protocol; QIAGEN Inc., Venlo, Limburg) according to manufacturer’s instructions. DNA quantification was performed by using the NanoDrop® Spectrophotometer (ThermoScientific, Wilmington, DE, US) and fluorescent staining of double-stranded DNA (PicoGreen® dsDNA Quantitation Kit, Molecular Probes, Eugene, OR, US). Fluorescent intensity was measured using a fluorescent micro-titer plate reader (BMG Labtech, FluoStar Optima, Ortenberg, Germany). Isolated DNA specimens were first tested for relevant polymorphisms on candidate warfarin-related pharmacogenes using the DMET^TM^ plus assay (Affymetrix®, Santa Clara, CA, USA) at the RCMI Core Lab of Molecular Genetics (UPR-MSC), and confirmed by either pre-developed TaqMan® Genotyping Assay Reagent kits for Allelic Discrimination (Applied Biosystems, CA, USA) or Tag-It™ Mutation Detection Assay for warfarin (LUMINEX xMAP® Technology) at the CLIA-certified Laboratory of Personalized Health (Genomas Inc., Hartford, CT). The African-related rare variant *CYP2C9**8 (c.449G>A; p.R150H) and two other polymorphisms in the promoter region of the gene (i.e., c.-1188T>C and c.-1766T>C), which are not included in the commercial DMET^TM^ plus panel, were individually detected by their corresponding TaqMan® SNP Genotyping Custom Assay Reagent kits for Allelic Discrimination (Life Technologies, Carlsbad, CA, USA) following manufacturer’s instructions. A full description of these assays and a complete list of variants ascertained can be found elsewhere [[16](#pone.0145480.ref016)–[21](#pone.0145480.ref021)]. The *CYP2C9* allele designations refer to those defined by the Cytochrome P450 Allele Nomenclature Committee [[22](#pone.0145480.ref022)–[23](#pone.0145480.ref023)].

Samples were also genotyped on a *PhysioGenomic* (PG) array that interrogates 384 single nucleotide polymorphisms (SNPs) and ancestry informative markers across 222 cardio-metabolic and neuroendocrine genes [[24](#pone.0145480.ref024)]. Careful manual analysis was performed on the alignments underlying the genotype calls using GenCall 6.1.3.24 and 50 SNPs with a slight degree of uncertainty about calling accuracy were not included in the analysis, leaving 332 SNPs from 196 genes. Genotyping was accomplished using Illumina® BeadArray™ technology [[25](#pone.0145480.ref025)–[26](#pone.0145480.ref026)]. A full list of the genes and SNPs in the PG chip is provided in our physiogenomic analysis study [[24](#pone.0145480.ref024)].

Departure from Hardy-Weinberg equilibrium (HWE) was estimated under the null hypothesis of the predictable segregation ratio of specific matching genotypes (p>0.05) by use of χ2 goodness of fit test with 1 degree of freedom. Clinical non-genetic and demographic information from the participants was retrospectively extracted from the corresponding computerized patient record system (CPRS). Individual warfarin doses at the first day of therapy (D1, usually 2.5 or 5 mg as per the clinical nomogram in the VACHS anticoagulation clinic) were retrieved from the corresponding records in CPRS. Statistical calculation indicated 80% power to detect SNPs with allele frequencies of 0.1–0.4 in the study sample, accounting for 10% of phenotypic variation at <5% significance level.

### Quantification of Admixture

A Bayesian’s clustering of individuals according to their maximal PG-derived genotype sharing was performed using the *STRUCTURE* v2.3.4 software package [[27](#pone.0145480.ref027)–[29](#pone.0145480.ref029)]. Three Markov Chain Monte Carlo (MCMC) runs were performed using default parameters and an assumed cluster size (K) of 3, which was chosen based on our knowledge of migration history in the Caribbean basin to reflect admixture from Amerindian (Tainos Arawaks), West-African, and European ancestors as well as to match previous reports for a 3-hybrid model [[30](#pone.0145480.ref030)–[35](#pone.0145480.ref035)]. A burn-in time of 30,000 iterations was followed by 70,000 further iterations. The log likelihood for each run was determined as *a posteriori* probability for the different numbers of clusters. Each individual was represented by a dot in the triangle plot of ancestral proportions. Proximity values (between 0 and 1) corresponding to each of 3 vertices in the triangle plot were calculated for each dot. A categorical criterion of proximity was added requiring that in order for an individual to be assigned to a specific cluster, its higher proximity value must be greater than the other two by at least 0.2. This criterion excluded those individuals not clearly positioned towards any vertex but rather located near the middle of the triangle (i.e., the mixed ones).

### Multiple Linear Regression Modeling

We selected all the patients with complete genetic and clinical data as the “derivation cohort” (n = 255) for developing dose-prediction models. Like in earlier reports, this “derivation cohort” size allowed inclusion of up to 13 independent variables in the final model [[36](#pone.0145480.ref036)]. A sample of another 55 patients from the Brownstone outpatient clinic in Hartford Hospital (Hartford, CT, USA) constituted the external ‘validation cohort’, which was used for testing validity of the developed pharmacogenetic model. Patients in the validation cohort were selected based on the same inclusion criteria as those in the derivation cohort. The investigators who performed the modeling and analysis did not have access to this validation set until after the final model was selected.

A forward stepwise entry and backward stepwise elimination approach in multiple linear regression analysis was used to ascertain the association between average therapeutic warfarin doses (dependent variable) and individual's genetics (i.e., genotypes and admixture index), demographics (i.e., age, body size) and non-genetic clinical information (i.e., dose-adjusted INR, target INR, co-meds use, indications, smoking status, comorbidities, diet) available on day 3 of warfarin therapy. Clinically relevant genetic variants on the following candidate pharmacogenes were tested: *CYP2C9*, *VKORC1*, *EPHX1*, *CYP4F2*, and *NQO1*. Variables were included in the final linear regression model if they were significantly (p<0.05) associated with the daily warfarin dose or were marginally significant (0.05 ≤ p ≤ 0.20) with strong biological plausibility.

The average therapeutic warfarin dose (mg/day) is defined as the dose that led to stable therapeutic anticoagulation levels (i.e., INR values in range on at least three consecutive visits for the same average dose). The numerical designations of “0” for the *CYP2C9**1/*1 genotype (wild-type); “1” for the *1/*n genotypes (n stands for *3,*5,*6,*8,*11 “loss-of-function” variants detected in the genotyping assay) and “2” for *n/*n genotypes (e.g., *2/*2, *3/*3, *3/*5) were used to correspond to the relative capacity of the CYP system to metabolize warfarin. *VKORC1*-1639 AA status is a carrier code, where 1 = AA and 0 = otherwise; *VKORC1*-1639 GA status is another code, where 1 = GA and 0 = otherwise. Likewise, the numerical designations of “0” was used for the wild-types on either *CYP4F2* or *NQO1* loci; whereas, “1” was set for the respective single carrier (i.e., *CYP4F2**1/*3 or *NQO1**1/*2) and “2” for the double-carriers of the corresponding allele variants in each of these two warfarin-related genetic loci (i.e., *CYP4F2**3/*3 or *NQO1**2/*2). The admixture index was defined as follows (1 = European ancestry, 2 = Native American ancestry, 3 = African ancestry, 4 = Admixed individual). All discrete variables were treated as dichotomous (e.g., Amiodarone user: Yes = 1, No = 0). Effect on the estimates of the maintenance warfarin dose was calculated per number of variant alleles (genetics), per decade (age) and per 0.25-unit increase in the dose-adjusted INR response at day 3.

All discrete covariates entered the analysis transformed as dummy variables to quantify how the individual categories affected daily mean warfarin dose. To this purpose, an inter-correlation matrix to identify genetic, demographic, and other clinical variables that correlate with maintenance warfarin daily dose was constructed.

Comparisons of the arithmetic mean values (or percentages) between carriers and wild-types were calculated with the use of the two-tailed Mann–Whitney U/Wilcoxon’s rank-sum test (for warfarin dose, age, height, weight, target INR range, INR/dose at third day) and χ2 or Fisher’s exact test (for co-medications, smoking status, race and comorbidities) at 5% significance. Analyses of variance to accommodate unequal cell sizes were used to test the effect of genotypes as an independent variable.

#### Model Diagnostics

The mean absolute error (MAE, mg/day, defined as the mean of the absolute values for the difference between the predicted and actual doses) was used to evaluate the model’s predictive accuracy. We selected the final model as the one that had the lowest predictive MAE. The bias of the dosing algorithm estimates (precision) was assessed by calculating the mean percentage of difference from the observed dose, where mean percentage of difference is equal to the MAE between predicted and actual dose divided by the actual dose ([predicted dose—observed dose]/observed dose) × 100%. Finally, the effect size of each independent predictor covariate on the daily dose of warfarin was also computed.

We compared dose predictions from our admixture-adjusted pharmacogenetic model for dose-refinement with those from three other separate models: 1) a clinical algorithm that uses the same equation of the admixture-adjusted pharmacogenetic model, but excluding admixture and genotypes; 2) the Lenzini *et al*. refinement model [[36](#pone.0145480.ref036)] and 3) the multi-ethnic IWPC algorithm [[37](#pone.0145480.ref037)]. We evaluated the potential clinical value of each algorithm by calculating the percentage of patients in both study cohorts whose predicted dose of warfarin was within 20% of the actual stable therapeutic dose. In addition, we calculated the percentage of patients for whom the predicted dose according to each algorithm was at least 20% higher than the actual dose (overestimation) or at least 20% lower than the actual dose (underestimation). These values represent a difference of 1 mg/day relative to the traditional starting dose of 5 mg/day, a difference clinicians would define as clinically relevant. We also assessed the performance of the algorithms in three dose groups: participants requiring a low dose (≤3 mg/day), those requiring a high dose (≥7 mg/day) and those requiring intermediate doses (>3 and <7 mg/day) for effective anticoagulation. McNemar’s test of paired proportions was used for the comparisons. Significance level of all statistical analyses was set at p<0.05. Statistical analyses were performed using the SmallSTATA® software v.12.

## Results

### Derivation Cohort

The derivation cohort (n = 255 patients) consists of patients with a mean age of 68 ± 9.81 years, 99% were male and 88% were self-identified in their electronic medical record as Hispanic Whites ([Table 1](#pone.0145480.t001)). The average therapeutic warfarin daily dose ranged from 1.43 to 11.07 mg/day, a difference representing approximately five standard deviations. From the total of patients enrolled, 49 patients were sensitive (≤3 mg/day) and 28 were resistant (≥7 mg/day) to warfarin. Of these, 10 patients showed discrepancies between actual dose requirements and dose predictions using a previously developed pharmacogenetic algorithm in Puerto Ricans [[38](#pone.0145480.ref038)]. [Table 1](#pone.0145480.t001) summarizes the most important facts on demographics and non-genetic clinical data of this study cohort. Atrial fibrillation (AFib, n = 136 patients), deep vein thrombosis (DVT, n = 36 patients) and pulmonary thromboembolism (PE, n = 10) were the most common indications for oral anticoagulant therapy with warfarin in this cohort. Potential drug interactions with warfarin were found with statins (simvastatin and fluvastatin, n = 145) and less common with amiodarone (n = 6). Demographic and clinical variables were similarly distributed between carriers and non-carriers.

### Table 1. Descriptive clinical (non-genetic) and demographic characteristics of the derivation cohort (N = 255 warfarin-treated patients from VACHS) and the independent sample of 55 warfarin-treated Puerto Ricans from the Brownstone Clinic in Hartford, CT (validation cohort).

| Characteristics | Derivation Cohort All Patients (N = 255) | Derivation Cohort Non-carriers (n = 74; %) | Derivation Cohort Carriers (n = 181; %) | p-valuea | Validation Cohort (N = 55) |
| --- | --- | --- | --- | --- | --- |
| Stable Warfarin Dose (mg/day)b |  |  |  |  |  |
| Meanc (±SD) | 4.62 (1.77) | 5.34 (1.70) | 4.33 (1.71) | 0.0001 | 5.0 (2) |
| Range (min–max) | 1.43–11.07 | 1.57–10.00 | 1.43–11.07 |   | 1.5–10.0 |
| Age (years) |  |  |  |  |  |
| Mean (±SD) | 68.1 (10) | 68 (10) | 68 (10) | 0.99 | 60.7 (14.4) |
| Range (min–max) | 31–94 | 37–94 | 31–90 |   | 23–90 |
| Weight (lbs.) |  |  |  |  |  |
| Mean (±SD) | 186.9 (37) | 190 (36) | 185 (37) | 0.44 | 179.0 (41.9) |
| Range (min–max) | 87–366 | 136–366 | 87–341 |   | 114.8–277 |
| Height (inches) |  |  |  |  |  |
| Mean (±SD) | 67.8 (7) | 67 (2) | 67 (3) | 0.05 | 62.7 (3.6) |
| Range (min–max) | 61–76 | 63–73 | 61–76 |   | 53–69 |
| Atrial fibrillationd/flutter, N(%) | 136 (53) | 45 (60.81) | 135 (74.59) | 0.03 | 33 (60) |
| Deep Vein Thrombosis, N (%) | 36 (14) | 15 (20.27) | 28 (15.47) | 0.36 | 10 (18.2) |
| Pulmonary embolism, N (%) | 10 (4) | 6 (8.11) | 7 (3.87) | 0.21 | 8 (14.5) |
| Other indication, N (%) | 73 (28.6) | 24 (32.43) | 61 (33.70) | 0.88 | 4 (7.3) |
| Smoker, N (%) | 13 (5.10) | 2 (2.70) | 11 (6.08) | 0.36 | N/A |
| Amiodarone users, N (%) | 6 (2.35) | 2 (2.70) | 4 (2.21) | 0.99 | 2 (3.6) |
| Azoles users, N (%) | 1 (0.39) | 1 (1.35) | 0 (0.00) | 0.29 | N/A |
| Statins users, N (%) | 145 (56.86) | 38 (51.35) | 107 (59.12) | 0.27 | N/A |
| Racee |  |  |  |  |  |
| Whites, N (%) | 226 (88.63) | 61 (82.43) | 165 (91.16) | 0.05 | 51 (92.7) |
| Blacks, N (%) | 29 (11.37) | 13 (17.57) | 16 (8.84) |   | 4 (7.3) |
| INR/Dose at 3rd day |  |  |  |  |  |
| Mean (±SD) | 0.63 (0.37) | 0.57 (0.41) | 0.66 (0.36) | 0.08 | 0.65 (0.53) |
| Range (min-max) | 0.14–2.27 | 0.14–2.26 | 0.17–2.27 |   | 0.14–2.47 |
| Target INR |  |  |  |  |  |
| Mean (±SD) | 2.53 (0.13) | 2.55 (0.16) | 2.52 (0.12) | 0.10 | 2.5 (0.5) |
| Range (min-max) | 2.25–3.00 | 2.50–3.00 | 2.25–3.00 |   | 2–3.5 |
| Gender |  |  |  |  |  |
| Male, N (%) | 254 (99.6) | 74 (100) | 180 (99.45) | 0.99 | 20 (36) |
| Females, N (%) | 1 (0.4) | 0 (0) | 1 (0.55) |   | 35 (64) |

Table 1 Caption: ap-values for the difference between the carriers and non-carriers were calculated with the use of the two-tailed U Mann-Whitney/Wilcoxon’s rank-sum test (for warfarin dose, age, height, weight, target INR, INR/Dose at 3rd day) and chi-square or Fisher’s exact test (for co-medications, smoking status, race, gender and comorbidities). Co-medications (brand name in parenthesis when used): Amiodarone; Statins: Simvastatin, Rosuvastatin (Crestor), Fluvastatin (Lescol) and Pravastatin; Azoles: Fluconazole (Diflucan), Itraconazole, Ketoconazole. bDose titrated empirically based on current clinical guidelines for warfarin patient management and nomogram at VACHS. cMeans refer to “arithmetic” means. dChronic, paroxysmal or post-operative included. N/A stands for information not available. eOne subject self-identified as mestizo; another one declined to self-report his race/ethnicity. Age, height and weight are determined at the time of stabilization. NS means not significant differences (p≥0.05).

The *CYP2C9* and *VKORC1* polymorphic alleles were highly prevalent in this study population (**Table 2**). Ninety-four (94, 37%) individuals are wild-types for both the *CYP2C9* and the *VKORC1* loci combined; whereas, 161 (63%) presented a variant genotype that are subdivided into 37% single, 21% double, 5% triple and 0.4% quadruple carriers of polymorphic alleles in *CYP2C9* and/or *VKORC1* genes. Among all these patients, 30% had at least one polymorphism in *CYP2C9*, whereas ~62% were carriers of the *VKORC1-*1639G>A promoter variant. All alleles were in Hardy-Weinberg equilibrium.

### Table 2. Allele and genotype frequency distributions of CYP2C9, CYP4F2, NQO1 and VKORC1 polymorphisms in Hispanic patients treated with warfarin at the VACHS-San Juan, PR (n = 255).

| Genotype/Alleles a | n; % (95% CI; Margin of Error) | χ2 test |
| --- | --- | --- |
| VKORC1-1639 G>A genotype |  | 0.41 |
| GG | 98; 38.4% (32.7–44.5; 0.06) |   |
| GA | 124; 48.6% (42.5–54.7; 0.06) |   |
| AA | 33; 13% (9.3–17.6; 0.04) |   |
| G | 320; 63% (58.5–66.8; 0.04) |   |
| A | 190; 37% (33.2–41.5; 0.04) |   |
| VKORC1 (c.134T>C, p.V45A, rs104894540): Resistant |  | <0.001 |
| TT | 254; 99.6% (97.6–99.9; 0.0129) |   |
| TC | 1; 0.39% (0.01–2.42; 0.0129) |   |
| T | 509; 99.8% (98.8–99.9; 0.0065) |   |
| C | 1; 0.2% (0.01–1.22; 0.0065) |   |
| CYP2C9 SNPs |  | 3.74 |
| *1/*1 | 179; 70% (64.3–75.5; 0.05) |   |
| *1/*2 | 36; 14.1% (10.3–18.9; 0.04) |   |
| *1/*3 | 17; 6.7% (4.1–10.5; 0.03) |   |
| *1/*4 | 4; 1.6% (0.5–4.1; 0.02) |   |
| *1/*5 | 2; 0.8% (0.03–3; 0.01) |   |
| *1/*6 | 1; 0.4% (0.01–2.5; 0.01) |   |
| *1/*8 | 3; 1.2% (0.25–3.5; 0.01) |   |
| *1/*11 | 1; 0.4% (0.01–2.5; 0.01) |   |
| *2/*2 | 2; 0.8% (0.03–3; 0.01) |   |
| *2/*3 | 6; 2.4% (0.9–5.2; 0.02) |   |
| *2/*5 | 1; 0.4% (0.01–2.5; 0.01) |   |
| *3/*5 | 2; 0.8% (0.03–3; 0.01) |   |
| *3/*6 | 1; 0.4% (0.01–2.5; 0.01) |   |
| *1 | 422; 83% (79.2–85.8; 0.03) |   |
| *2 | 47; 9% (7–12; 0.02) |   |
| *3 | 26; 5% (3.5–7.4; 0.02) |   |
| *4 | 4; 0.8% (0.2–2.1; 0.01) |   |
| *5 | 5; 1% (0.35–2.3; 0.01) |   |
| *6 | 2; 0.4% (0.01–1.5; 0.01) |   |
| *8 | 3; 0.6% (0.1–1.8; 0.01) |   |
| *11 | 1; 0.2% (0.009–1.2; 0.006) |   |
| CYP2C9 (c.-1188T>C)b |  | 1.18 |
| T/T | 37; 44% (33.9–54.7; 0.104) |   |
| C/T | 34; 40.5% (30.6–51.2; 0.103) |   |
| C/C | 13; 15.5% (9.13–24.8; 0.08) |   |
| T | 108; 64.3% (56.8–71.1; 0.07) |   |
| C | 60; 35.7% (28.9–43.2; 0.07) |   |
| CYP4F2*3 (g.18000G>A, p.V433M, rs2108622) |  | 2.82 |
| *1/*1 (G/G) | 203; 79.6% (74.2–84.1; 0.05) |   |
| *1/*3 (G/A) | 46; 18% (13.8–23.2; 0.05) |   |
| *3/*3 (A/A) | 6; 2.4% (1–5; 0.02) |   |
| *1 (G) | 452; 88.5% (85.6–91.1; 0.03) |   |
| *3 (A) | 58; 11.5% (8.9–14.4; 0.03) |   |
| NQO1*2 (g.559C>T, p.P187S, rs1800566) |  | 3.09 |
| *1/*1 (C/C) | 210; 82.3% (77.2–86.6; 0.05) |   |
| *1/*2 (C/T) | 40; 15.7% (11.7–20.7; 0.04) |   |
| *2/*2 (T/T) | 5; 2% (0.7–4.6; 0.02) |   |
| *1 (C) | 460; 90.2% (87.3–92.5; 0.03) |   |
| *2 (T) | 50; 9.8% (7.5–12.7; 0.03) |   |

Table 2 Caption: Data expressed as percent (Wald-adjusted 95% Confidence Interval, CI). χ2 <3.84 meet HWE for genotypes, one degree of freedom, p<0.05.

Notably, three patients were carriers of the *CYP2C9**8 allele (all heterozygous) and another five (5) were carriers of the *CYP2C9**5 allele (c.1080C>G transversion in exon 7, p.N360E). The *5 allele has been mainly reported in individuals of African descent and has been early associated with a substrate-dependent decrease in *in vitro* intrinsic clearances for CYP2C9 pathway that ranges from 8 to 18% of wild-type [[39](#pone.0145480.ref039)–[40](#pone.0145480.ref040)]. Additionally, the *8 allele has been identified in African-American patients [[39](#pone.0145480.ref039), [41](#pone.0145480.ref041)–[42](#pone.0145480.ref042)] and also accounts for a decrease in CYP2C9 enzyme activity of 30% for S-warfarin [[43](#pone.0145480.ref043)]. Moreover, the *CYP2C9**6 allele (g.10601delA; p.K273X) was identified in two subjects of this study cohort (both heterozygous for this variant, but one of them showing a *3/*6 double-carrier status, patient# WPR255); also the rare variants *CYP2C9**4 (g.42615T>C, p.I359T, *rs56165452*) and *CYP2C9**11 (g.42542C>T, p.R335W, *rs28371685*) were detected in four and one additional participants, respectively (all heterozygous). Another rare variant found in these patients was the c.-1188 T>C SNP in the promoter region of the *CYP2C9* gene (MAF = 0.357; frequencies of genotypes C/T: 0.405 ±0.31–0.51, Margin of Error = 0.10 and C/C: 0.155 ±0.09–0.25, Margin of Error = 0.08; HWE test χ^2^ = 1.18), but not the c.-1766 T>C polymorphism (rs9332094). Notably, both of them are often in strong linkage disequilibrium with the *8 allele [[44](#pone.0145480.ref044)].

The frequencies of the *CYP4F2**3 (g.18000G>A, p.V433M, *rs2108622)* and *NQO1**2 (g.559C>T, p.P187S, *rs1800566)* “resistant” polymorphisms in the study population were 0.11 (95%CI: 0.09–0.14) and 0.10 (95%CI: 0.07–0.13), respectively. There are 18% heterozygous (G/A) for the *CYP4F2**3 allele (95%CI: 0.14–0.23) and 2.4% homozygous (A/A) for this variant allele (95%CI: 0.01–0.05); whereas, about 15.7% of patients carried the C/T heterozygous genotype for the *NQO1**2 SNP (95%CI: 0.12–0.21) and 2% were double carriers (T/T) of this variant (95%CI: 0.007–0.05). Moreover, one patient (# WPR274) was identified as heterozygous for the “rare” missense variant c.134T>C transition in the *VKORC1* gene (*rs104894540*), which is an autosomal dominant warfarin resistance genotype resulting in a valine-to-alanine substitution at codon 45 (p.V45A). Another eight (8) participants are carriers of the H7 “resistant” haplotype, including the homozygous H7/H7 patient # WPR265.

### Validation Cohort

Subjects enrolled in the external “independent” validation cohort (n = 55) are younger (mean age: 60.7 years; range: 23–90 years) and included more female (approx. 64%) than those in the derivation sample. Sixty percent (60%) had AFib as their primary indication for therapy with warfarin. The other clinical and demographic variables are fairly well balanced between the two cohorts ([Table 1](#pone.0145480.t001)). About thirty-six percent (36.4%) of subjects in this cohort are wild-types, 11% are single carriers of polymorphisms on either *CYP2C9* (9%) or *VKORC1* (2%); 5.5% are double carriers for *CYP2C9* variants (including one *2/*3) and 34.5% are *VKORC1*-1639 AA; whereas, 9% are triple carriers for variants on both *CYP2C9* and *VKORC1* (i.e., *CYP2C9**1/*2 + *VKORC1* AA) and 3.6% are quadruple carriers (i.e., *CYP2C9**2/*2 + *VKORC1* AA).

### Model Development

The respective regression coefficients, p-values, the percent of effect on dose requirements and the contribution to explain observed variability that is accounted for by the model, as measured by the partial R^2^ statistics, are presented in [Table 3](#pone.0145480.t003) for all the variables that were entered into the regression model. The performance of this pharmacogenetic model is depicted in [Fig 2](#pone.0145480.g002).

### Table 3. Summary of attributes of the pharmacogenetic equation for warfarin dosing refinement in Caribbean Hispanics.

| Model Summary |  |  |  |  |  |
| --- | --- | --- | --- | --- | --- |
| Variablesa | Partial Regression Coefficient | Std. Error | Adjusted-R2 after entry | Effect on warfarin dose b | p-value |
| Constant | 3.246 | 0.656 |   |   |   |
| Age | -0.0181 | 0.0067 | 0.049 | -5% | 0.008 |
| D1 | 0.425 | 0.0488 | 0.463 | 13% | <0.0001 |
| tINR | 0.516 | 0.253 | 0.471 | 15% | 0.043 |
| Dose-adjusted INRc | -1.419 | 0.1437 | 0.591 | -11% | <0.0001 |
| CYP2C9*2 | -0.245 | 0.1654 | 0.613 | -7% | 0.014 |
| CYP2C9*3 | -0.544 | 0.2224 | 0.625 | -16% | 0.015 |
| CYP2C9*8 | -1.00 | 0.6014 | 0.638 | -31%; -18%d | 0.098 |
| VKAA | -0.896 | 0.2169 | 0.641 | -12% | <0.0001 |
| VKGA | -0.379 | 0.1391 | 0.659 | -14%; -13%e | 0.007 |
| CYP4F2*3 | 0.560 | 0.2274 | 0.673 | 17% | 0.014 |
| NQO1*2 | 0.346 | 0.2246 | 0.689 | 10% | 0.125 |
| Admixture | -0.093 | 0.0577 | 0.694 | -7% | 0.108 |
| Amiodarone | -0.795 | 0.4173 | 0.699 | -24% | 0.06 |

Table 3 Caption: Derivation cohort (n = 255 patients from the VACHS).

### Fig 2. Admixture-adjusted pharmacogenetic warfarin dose refinement algorithm in Caribbean Hispanic patients, developed by using a multiple regression analysis in a derivation cohort of 255 individuals.

![Fig 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ec3f/4706412/2f2db43b3e63/pone.0145480.g002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4706412_pone.0145480.g002.jpg)

The solid “identity” line illustrates perfect prediction. MAE and MSE stand for the mean absolute error and the mean standard error of estimate, respectively.

Combined *VKORC1*-1639AA and GA genotypes explained approximately 2.1% of the dose variability in this population. Moreover, this covariate was associated with a 13% reduction in the therapeutic warfarin dose per number of A alleles. The number of loss-of-function *CYP2C9* polymorphisms combined explained another 5% of the observed inter-individual variability in dose requirements, and it was associated with an average of 18% decrement in the warfarin dose per variant allele. Notably, *CYP4F2**3 and *NQO1**2 variants were independently associated with a 17% and 10% increase of the dose per variant allele, respectively; whereas, the admixed status of the admixture index decreases the dose by 7%. None of the tested *EPHX1* variants entered into the final model. Dose-adjusted INR at day 3 was one of the non-genetic clinical variables to enter the model, and each 0.25-unit increase in this interaction variable was associated with an 11% decrease in the therapeutic warfarin dose. This variable explained approximately 12% (95% CI: 6.3–25.6) of the observed dose variability. Other factors that entered the regression model were age, dose at day 1, target INR range and co-therapy with amiodarone. Overall, this admixture-adjusted pharmacogenetic model in Caribbean Hispanics contained thirteen significant ‘explanatory’ variables. These independent predictors explained more than two-thirds of the observed variance in the therapeutic warfarin dose (R^2^ = 70%; MAE = 0.72mg/day; MSE = 0.99; p< 0.001), which is better than that of a clinical non-genetic algorithm in the same derivation cohort (R^2^ = 60%; MAE = 0.99mg/day; MSE = 1.26). Interaction terms between the genotypes and key co-medications (e.g., amiodarone, statins, azoles and sulfamethoxazole) were not statistically significant.

A polynomial regression model was discarded as the distribution did not suggest a curvilinear relationship between warfarin doses and the model variables. A multiple, least-squares, linear regression model proved to be the best option for the available data, according to the lowest MAE criterion. The stepwise regression analysis yielded the following PGt algorithm to predict warfarin doses in Hispanic Puerto Ricans (as seen in [Fig 2](#pone.0145480.g002)):

| WarfarinDose(mgday)=[3.2463−(0.0181∙Age)+(0.4246∙D1)+(0.5157∙tINR)−(1.419∙INRD3)−(0.2450∙CYP2C9*2)−(0.5442∙CYP2C9*3)−(1.00∙CYP2C9*8)−(0.3793∙VKORC1GA)−(0.8958∙VKORC1AA)+(0.5602∙CYP4F2*3)+(0.3457∙NQO1*2)−(0.7948∙Amiodarone)−(0.0932∙Admixture)] | (1) |
| --- | --- |
Where INR/D3 is the corresponding ratio of the dose-adjusted INR measures on day 3 of initiation of warfarin therapy; D1 is the initial dose of warfarin (mg/day) on Day 1 of therapy; tINR stands for the middle value of the target INR range, as per indication to warfarin (e.g., 2–3 for Afib is entered as 2.5); *VKORC1* AA status is a carrier code, where 1 = AA and 0 = otherwise; *VKORC1* GA status is a code, where 1 = GA and 0 = otherwise; all *CYP2C9* status, as well as the *CYP4F2**3 and the *NQO1**2 status, are codes, where 0 = wild-type, 1 = one mutated allele and 2 = two mutated alleles; amiodarone user code = 1 if the patient is taking that drug and 0 = otherwise; and age is in years. Finally, the admixture index code is as follows: 1 = European ancestry, 2 = Native American ancestry, 3 = African ancestry, 4 = Admixed individual.

To select candidate variables for the multiple linear-regression model of warfarin dosing prediction in Caribbean Hispanics, we initially examined the independent effects of demographics (i.e., age, gender, height and weight or body mass index), clinical (i.e., treatment indication, comorbidities, use of co-medications [amiodarone, antifungal azoles, statins, Bactrim], target INR range, initial doses (days 1 thru 3 of therapy), dose-adjusted INR at day 3, diet and smoking status) and genetic variables (i.e., *VKORC1* genotypes, number of *CYP2C9* variants, *CYP4F2*3 18000G>A (V433M*, *rs2108622)* and *NQO1*2 c*.*559C>T (P187S*, *rs1800566)* carrier status) individually.

The admixture-driven pharmacogenomic algorithm developed in Caribbean Hispanics predicted effective warfarin dose with R^2^ values of 63%, 50% and 20% for low, intermediate and high-dose groups, respectively ([Table 4](#pone.0145480.t004)). These R^2^ values surpassed predictability of the clinical non-genetic algorithm for the same dosing groups in the derivation cohort of Caribbean Hispanic patients (i.e., 50%, 45% and 2%, respectively). As expected, our model’s predictability also outperformed both the Lenzini’s and the IWPC’s algorithms in this study cohort (i.e., Lenzini’s model: 24%, 20%, and 7% and IWPC’s model: 23%, 13% and 2% of variability explained for the same dosing groups, respectively). Furthermore, in more than half of patients the MAE value was <1 mg/day (i.e., falling within 20% of the actual dose, which is a well-accepted criterion for accuracy in dose estimation).

### Table 4. Warfarin daily dose predictability of the admixture-adjusted pharmacogenetic refinement model developed in Caribbean Hispanics (CH), the Lenzini et al. pharmacogenetic refinement model [36], the IWPC pharmacogenetic initiation algorithm [37] and the clinical algorithma as compared with the actual doses of warfarin for the therapeutic effect in patients requiring low (≤3 mg/day), intermediate (>3 and <7 mg/day) or high (≥7 mg/day) exposure.

| Prediction Model | IWPC Algorithm | Lenzini Algorithm | Clinical Algorithma | CH Algorithm |
| --- | --- | --- | --- | --- |
| Low Doses (≤3 mg daily) |  |  |  |  |
| MAE (mg/day) 95% CI | 1.27 (1.02–1.52) | 0.63 (0.49–0.77) | 1.22 (1.05–1.39) | 0.51 (0.39–0.63) |
| % bias (precision) | 53% | 26% | 50% | 21% |
| R2 (%) | 23% | 24% | 50% | 63% |
| Intermediate Doses (>3 and <7 mg daily) |  |  |  |  |
| MAE (mg/day) 95% CI | 0.97 (0.85–1.09) | 1.2 (1.08–1.32) | 0.83 (0.74–0.92) | 0.61 (0.53–0.69) |
| % bias (precision) | 21% | 26% | 18% | 13% |
| R2 (%) | 13% | 20% | 45% | 50% |
| High Doses (≥7 mg daily) |  |  |  |  |
| MAE (mg/day) 95% CI | 2.48 (2.03–2.93) | 3.20 (2.73–3.67) | 1.53 (1.03–2.02) | 1.53 (1.17–1.89) |
| % bias (precision) | 31% | 40% | 19% | 19% |
| R2 (%) | 2% | 7% | 2% | 20% |

Table 4 Caption: Study cohort of 255 warfarin patients at the VACHS anticoagulation clinic.

For patients at the highest risk who required either ≤3mg (~19.2% of the study cohorts) or ≥7 mg/day of warfarin per day (~11% of the study cohorts), 54.5% of the dose predictions using our developed model resulted within 20% of the actual dose (‘ideal dose’) as compared with 29% when using the clinical non-genetic algorithm (p<0.001), 34% with the Lenzini’s model (p<0.001) and 24.7% with the IWPC’s algorithm (p<0.001). Similarly, for patients requiring between 3 and 7 mg/day (intermediate-dose group: 69.8% of the total cohort), our admixture-adjusted pharmacogenetic algorithm predicted doses in the ideal range for significantly more patients than the clinical algorithm or the two earlier published pharmacogenetic models (i.e., 78 vs 61, 39 and 62%, respectively; p<0.001 for each comparisons, [Table 5](#pone.0145480.t005)).

### Table 5. Predictions of ideal doses by using the admixture-adjusted pharmacogenetic refinement model developed in Caribbean Hispanics (CH), the Lenzini et al. pharmacogenetic refinement model [36], the IWPC pharmacogenetic initiation algorithm [37] and the clinical algorithmb in patients requiring low (≤3 mg/day), intermediate (>3 and <7 mg/day) or high (≥7 mg/day) exposure.

| Model | Ideal Dose (%) | Underestimated (%) | Overestimated (%) | p-valuea |
| --- | --- | --- | --- | --- |
| Low Doses (≤3 mg daily) |  |  |  |  |
| IWPC algorithm | 24.5 | 4.08 | 71.43 | 0.008 |
| Lenzini model | 44.9 | 8.20 | 46.9 | 0.48 |
| Clinical algorithmb | 10.20 | 0.00 | 89.80 | 0.0001 |
| CH Algorithm | 53.06 | 2.04 | 44.9 | - |
| Intermediate Doses (>3 and <7 mg daily) |  |  |  |  |
| IWPC algorithm | 62.4 | 18.54 | 19.1 | 0.0021 |
| Lenzini model | 39.3 | 53.9 | 6.74 | 0.0001 |
| Clinical algorithm | 61.8 | 2.8 | 35.39 | 0.0005 |
| CH Algorithm | 77.0 | 8.43 | 14.61 | - |
| High Doses (≥7 mg daily) |  |  |  |  |
| IWPC algorithm | 25.00 | 75.00 | 0.00 | 0.0159 |
| Lenzini model | 14.29 | 85.71 | 0.00 | 0.006 |
| Clinical Algorithm | 60.7 | 39.3 | 0.00 | 0.68 |
| CH Algorithm | 57.14 | 42.86 | 0.00 | - |

Table 5 Caption: Ideal dose is defined as within 20% of the actual warfarin dose at stabilization.

In general, the admixture-based pharmacogenomic algorithm tailored specifically for the Caribbean Hispanic population yielded consistently better dose predictions as compared to the Lenzini’s and IWPC’s algorithms. In particular, we observed good predictions of ideal doses in ~70% of the entire study cohorts when using our model, with significantly fewer under- and overestimations of the ideal doses (11% and 19%, respectively). In contrast, the Lenzini’s model barely predicted ideal doses in less than a half of subjects that came up with good predictions after using our model (i.e., ~38%, 96/255); whereas, the IWPC’s model only predicted correctly the ideal dose in approx. 51% of participants.

A correlation plot of the predicted warfarin daily dose based on the admixture-driven pharmacogenetic algorithm developed for Caribbean Hispanics as compared with the actual daily dose in the validation cohort is shown in [Fig 3A](#pone.0145480.g003). As can be observed, our pharmacogenetic model predicted 59% of the variance in warfarin dose when externally validated in 55 genotyped individuals from the independent Hartford’s patient cohort (MAE = 0.89 mg/day, 24% mean bias). On the contrary, the two others reference PGt algorithms showed poorer dose predictability in this independent cohort. [Fig 3B and 3C](#pone.0145480.g003) depicts the corresponding statistics for these algorithms (i.e., IWPC’s model: R^2^ = 39%, MAE = 1.60 mg/day, 34% mean bias; Lenzini’s model: R^2^ = 27%, MAE = 1.62 mg/day, 29% mean bias). On the other hand, the clinical algorithm that excluded genotypes and admixture only accounted for 50% of dose variability in this independent cohort (MAE = 1.34 mg/day, 34% mean bias, [Fig 3D](#pone.0145480.g003)).

### Fig 3. Validation of the admixture-adjusted pharmacogenetic algorithm for dose refinement in Caribbean Hispanics (upper plot A) as compared to two publicly available algorithms (i.e., IWPC-derived [37] and Lenzini et al [36] models, plots B and C, respectively) as well as a clinical non-genetic algorithm (plot D) in an independent sample of 55 Puerto Ricans from the Brownstone Clinic in Hartford, CT.

![Fig 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ec3f/4706412/c4772ccbc9ba/pone.0145480.g003.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4706412_pone.0145480.g003.jpg)

Each filled diamond represents the observed versus predicted dose of each patient. The upper solid line is (predicted + 1 mg/day) of the actual dose, the middle solid line (i.e. 45% degree line) illustrates perfect prediction in this validation cohort, and the lower solid line is (predicted −1 mg/day) of the actual dose. 1 mg/day change in warfarin dose is sufficient to change the INR by 0.5, a clinically meaningful difference.

[Fig 4](#pone.0145480.g004) depicts a typical STRUCTURE-derived triangular landscape with ancestry-specific vertices (i.e., European, African and Native-American in a counterclockwise sense) on which we plotted the results of the Bayesian’s clustering analysis used to measure individual admixture proportions of participants (i.e., the ones used for the algorithm derivation). Notably, a portion of these individuals (red dots) are scattered throughout the triangle area (i.e., not clearly positioned toward any of the vertices but displayed in the middle of the triangle), showing hence a marked admixture. We found that the Native-American sector in the right-most vertex of the SRUCTURE triangle (in green) showed a higher proportion of patients with low-dose requirements (<3 mg/day) than the rest of the clusters (i.e., 33% vs. 19%, p<0.01).

### Fig 4. Association between the degree of Amerindian ancestry and low dose requirements.

![Fig 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ec3f/4706412/8ce9277d57ff/pone.0145480.g004.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4706412_pone.0145480.g004.jpg)

ID codes depicted for those requiring <3mg/day.

## Discussion

The sampled cohort is representative of the target population. The observed frequency distribution of multiple ethno-specific polymorphic variants in these warfarin-related pharmacogenes confirmed a richer genetic diversity in this highly admixed population of Caribbean Hispanics as compared to other parental groups like Caucasians. Noteworthy, some of these variants have been previously described exclusively in either people of African ancestry (e.g., *5,*6,*8, *11) or Caucasians (e.g., *2,*3), but not often coexisting in both of them. Besides, the combination of *CYP4F2**3 and *NQO1**2 “resistant” variants, identified in as many as 20.4% and 17.7% of patients in the study cohort, had only been previously described to be associated with higher warfarin dose requirements in another US Hispanic population [[12](#pone.0145480.ref012)]. *CYP4F2* gene encodes cytochrome P450, family 4, subfamily F polypeptide, a member of the cytochrome P450 superfamily of enzymes that is involved in vitamin K1 metabolism. The NAD(P)H dehydrogenase, quinone 1 (*NQO1*) gene encodes for a FAD-binding cytoplasmic 2-electron reductase enzyme that catalyzes the reduction of vitamin K to its active hydroquinone form, and is therefore involved in the vitamin K-dependent prothrombin synthesis.

Most of existing pharmacogenetic-guided algorithms include only the *CYP2C9**2, *3 and *VKORC1*-1639G>A variants commonly found among Caucasians [[45](#pone.0145480.ref045)–[52](#pone.0145480.ref052)]. Uncommon mutations in Europeans (e.g., *CYP2C9**8, *NQO1**2) are often overlooked and, consequently, the utility of existing algorithms is limited in patients with mixed ancestry like Caribbean Hispanics. A recent clinical trial (COAG, [NCT00839657](https://clinicaltrials.gov/ct2/show/NCT00839657)) raised some concerns about clinical utility of genotyping in warfarin dose predictions [[53](#pone.0145480.ref053)]; however, we firmly believe that failure to account for variants relevant in underrepresented ethnicities (e.g., African Americans or admixed Hispanics) will lead to significant dosing error in these populations.

Similar to our previous pharmacogenetic-guided algorithm in Puerto Ricans [[38](#pone.0145480.ref038)], carrier status for *VKORC1*-1639G>A and loss-of-function *CYP2C9* polymorphisms were significant predictors of therapeutic dose in the herein developed admixture-adjusted PGt algorithm of dose-refinement within Caribbean Hispanics. However, the *CYP2C9* carrier status (i.e., combinatorial *CYP2C9**2 plus *3 alleles, along with the *8 variant), was this time a better predictor of therapeutic doses in our derivation cohort.

Despite of being the first variable to show significance in the previous algorithm, *VKORC1* GA and AA were now the 8^th^ and 9^th^ variables to enter a stepwise regression model that predicted the refined dose in the new algorithm ([Table 3](#pone.0145480.t003)). Interestingly, *VKORC1* genotypes combined contributed only ~2.1% of the total R^2^, which suggests that the INR/dose3 value captured most of the information on warfarin sensitivity in the derivation cohort. A similar observation was earlier noted by Lenzini *et al*. [[36](#pone.0145480.ref036)] in another dose-refinement model developed in a cohort of mostly Caucasian patients. Moreover, previous reports also found that *VKORC1* correlates with INR response over the first week of treatment; whereas, a strong association with dose vanished during this initial week of therapy [[36](#pone.0145480.ref036)]. It has been hypothesized by others that *CYP2C9* variants could overcome the predictive effect of *VKORC1* in PGt algorithms for dose-refinements because of their direct impact on S-warfarin metabolism, a process that seems to be more clinically relevant after the first few days. Since INR measure reflects warfarin sensitivity but not warfarin clearance, this value will mask the *VKOC1* effect over time. Nonetheless, the addition of *VKORC1* in the final model is expected to partially reduce the INR effect on dose variability, thus minimizing the impact of errors in initial INR measurements. Lenzini *et al*. have previously acknowledged this expectancy; particularly in patients being treated with unfractionated or low-molecular-weight heparin that sometimes inflate initial INRs [[36](#pone.0145480.ref036)].

Although the admixture index only explained less than 1% of observed dose variability in this cohort, it represents approximately a 7% of dose decrement for admixed individuals. Accordingly, we strongly believe this variable is a clinically relevant predictor for our model in Caribbean Hispanics given the well-documented stratification of this population in the Island of Puerto Rico and the Caribbean basin at large. An earlier paper by Via *et al*. [[54](#pone.0145480.ref054)] provides evidence in support of the need for adjustments by admixture degree in pharmacogenetics studies within Caribbean Hispanics in order to control for significant population stratification given its marked ethno-geographic heterogeneity and distinctive clines of admixture proportions across the Island (i.e., people with the higher African ancestry mainly located in the east side of the Island; whereas, individuals who have the greatest European contribution mostly living in the west region).

To the best of our knowledge, no previous trials have included a measure of admixture in warfarin dose predictions, even though a few of them have certainly added ethnicity or race as a proxy [[12](#pone.0145480.ref012)]. Admixture can be a confounder that may result in false associations due to differences in genetic backgrounds across individuals in the derivation cohort [[11](#pone.0145480.ref011),[13](#pone.0145480.ref013), [55](#pone.0145480.ref055)–[56](#pone.0145480.ref056)]. Findings depicted in [Fig 3](#pone.0145480.g003) suggest caution on use of pharmacogenetic models in Hispanics without accounting for admixture or ethnic-specific SNPs. [Fig 4](#pone.0145480.g004) shows result of the Bayesian’s clustering analysis of PG array data in order to infer the unique genetic ancestral roots among members of the admixed population under study [[27](#pone.0145480.ref027)]. Based on this result, we concluded that there is an association between the degree of individual Amerindian ancestry and low warfarin dose requirements. This is not surprising given the fact that people of Asian or Amerindian ancestry are more likely to need lower doses because of their higher *VKORC1* haplotype A frequency. Admixed populations may differ from other populations in frequency, distribution and combination of allelic variants in their genomes [[57](#pone.0145480.ref057)–[59](#pone.0145480.ref059)]. There are also important differences in linkage disequilibrium (LD) by ancestry that might have significant consequences for the association between genotypes and warfarin dosing across different populations.

Other variables that were not important predictors of dose revisions in our study cohort were body size, gender, vitamin K intake, statin use, azoles use, smoking status, liver function, diabetes, and indications. Type 2 Diabetes shows high prevalence among subjects in the study cohort and has been earlier reported as a marker for lower warfarin dose requirements [[60](#pone.0145480.ref060)]. A possible explanation is that these variables were all taken into proper consideration when initial doses were estimated at the anticoagulation clinic and their predictive ability appears to wane over time.

In practice, current turnaround time for genotyping results and ancestral proportions estimation could take ~2–3 days. Such an inherent delay limits our ability to initiate dosing based on pharmacogenetic information. Furthermore, various experts consider genotypes irrelevant once INR measurements become available after some days of therapy ([[36](#pone.0145480.ref036)]. This apparent lack of explicit cost-benefit for patients is compounded by the poor predictability of prior PGt-guided initiation algorithms in admixed populations like Hispanics. Consequently, it is not surprising the lack of fully endorsement by professional organizations in the field [[61](#pone.0145480.ref061)–[63](#pone.0145480.ref063)].

However, there is a well-documented higher incidence of warfarin-related bleeding events during and after the second week of warfarin therapy [[45](#pone.0145480.ref045), [64](#pone.0145480.ref064)–[68](#pone.0145480.ref068)]. Accordingly, PGt-guided models that refine warfarin dosing are expected to remain relevant in clinical settings after the first 3–5 doses even though these initial doses were not tailored to genotypes. A dose-refinement PGt algorithm can thus allow us to adjust therapeutic warfarin dose once relevant INR measures are also available. Previous developments of PGt refinement algorithms, including the work by the International Warfarin Dose-Refinement Collaboration (Warfarin DR) group [[36](#pone.0145480.ref036)], have been successful in determining utility of genotypes in patients from different ethnicities, even after INR levels become available on day 4–5^th^ of therapy, but none in Caribbean Hispanics.

Herein we present results for a newly developed and validated admixture-driven PGt algorithm for warfarin dose-refinement in Caribbean Hispanics, which showed very good performance, accuracy and robustness in predicting therapeutic doses in the target population. Our findings provide further support in favor of the proposed pharmacogenetic approach to predict optimal individual’s warfarin doses, even after 4 days of treatment, as compared to a standard clinical method.

On the other hand, our model outperformed two previous pharmacogenetic-guided algorithms (i.e., a dose-refinement algorithm by Lenzini *et al* [[36](#pone.0145480.ref036)] and the PGt algorithm for dose initiation developed by the IWPC that evaluate dosing variability at earlier time points [[37](#pone.0145480.ref037)]), with higher R^2^ (i.e., 0.70 versus 0.34 and 0.27, respectively), less scatter and lower MAE (MAE = 0.72 mg/day versus 1.33 and 1.2 mg/days, respectively), when using dataset from our derivation cohort of 255 patients. Accordingly, these existing pharmacogenetic-guided algorithms derived from other populations (mostly Caucasians) were poor predictors of the actual therapeutic warfarin dose in Caribbean Hispanics, explaining no more than 30% of the total dose variation in Caribbean Hispanics as compared to ≥ 60% in Caucasians [[36](#pone.0145480.ref036)–[37](#pone.0145480.ref037)].

Likewise, the developed algorithm also performed better for the Caribbean Hispanic population than a clinical algorithm that excludes genotypes and admixture (R^2^ = 0.6 and MAE = 0.99 mg/day). After adjusting for INR at 3^rd^ day and admixture, clinical polymorphisms on major warfarin-related pharmacogenes (*CYP2C9* and *VKORC1*) remained significant predictors of effective dose (p<0.05). Therefore, this admixture-driven PGt algorithm estimates effective warfarin doses after 4 days of therapy more accurately than the one that only used clinical factors.

Analysis in [Table 5](#pone.0145480.t005) shows no significant difference between our admixture-adjusted PGt model to predict optimal doses in the high-dose subset of patients (≥7 mg/day) and the clinical non-genetic algorithm (p = 0.683), but the admixture-adjusted PGt algorithm was significantly more accurate in the subsets with the intermediate (p = 0.0005) and low-dose requirements (p = 0.0001). We consider that our PGt algorithm is particularly more useful in patients who are carriers of the “high-risk” unusual genotypes. As a matter of fact, 54% of dose predictions in patients at the highest risk for poor outcomes (i.e. ≤3mg or ≥7 mg/day) resulted in ideal doses with our developed model, as compared with only 29% with the clinical algorithm. Previous reports indicate that algorithms guided by genetic information are able to correct the genetic propensity to high INRs in individuals with variant SNPs [[68](#pone.0145480.ref068)].

Although one study limitation is the single-center nature of the study design, the fact that all participants were managed by the same anticoagulation services (i.e., clinicians, procedures, formulary, dosing strategies, CPRS data entry, etc.) minimizes differential managements that could represent a potential bias to account for variability in dose requirements. Important to remark that the San Juan VACHS-affiliated anticoagulation clinic is the main and largest anticoagulation center in the Caribbean area, with patients from all ethno-geographic regions of the Island of Puerto Rico that ensure representativeness. Nonetheless, potential biases remain in any non-randomized study. Accordingly, we propose this algorithm to be investigated in a larger randomized, prospective clinical study in Caribbean Hispanics. Other limitations of the study include gender and elderly, as well as issues related to the observational and retrospective nature of the study design. All these limitations are also valid for the clinical non-genetic algorithm.

In general, our findings in Caribbean Hispanics are consistent with those reported in previous studies of warfarin PGt in other populations to predict dose-refinements, including those by Lenzini *et al*. and Horne *et al*. [[36](#pone.0145480.ref036), [68](#pone.0145480.ref068)]. The addition of genotypes and admixture improves the R^2^ by ~10%. The absolute difference in predicted vs. observed doses in the overall patient cohort was 0.72 mg/day, which corresponds to a 14.4% of mean absolute percentage of difference (precision). This prediction error of only 14% represents a difference of <1 mg/day relative to the traditional starting dose of 5 mg/day, which clinicians define as not clinically relevant. By systematically using this novel admixture-adjusted PGt algorithm, we anticipate that clinicians could now estimate a more accurate “tailored-to-genotype” therapeutic warfarin dose in Caribbean Hispanics after four days of therapy that translate into a safer and more effective therapy, with those at the highest-risk likely gaining the most benefit from this approach.

A multi-center, randomized clinical trial ([ClinicalTrials.gov](http://ClinicalTrials.gov) Identifier: [NCT02345356](https://clinicaltrials.gov/ct2/show/NCT02345356)), aimed at comparing our admixture-adjusted PGt algorithm for warfarin dose predictions in Caribbean Hispanics versus current standard of care, will be conducted soon in order to evidence its clinical utility.

## Supporting Information

## Acknowledgments

The material presented herein is the result of work supported with resources from and the use of facilities at the Veteran Affairs Caribbean Health System in San Juan, Puerto Rico. We thank Genomas Inc., Hartford, CT, and the UPR-MSC RCMI Center for Genomics in Health Disparities and Rare Disorders for their support and for providing the resources and facilities for performing genetic assays. The authors also want to thank Dr. Richard L. Seip, PhD; Mohan Kocherla, MSc; Jean C. Ortiz, PharmD; Dr. Cynthia Perez and Dr. Heidi L Venegas-Rios for their help in this survey. Finally, we want to thank all the patients for their participation in this study.

## Data Availability

All relevant data are within the paper and its Supporting Information files.

## Funding Statement

This investigation was supported in part by the grant number SC1 HL123911 from the National Heart, Lung, and Blood Institute (NHLBI) and the SCORE Program of the National Institute of General Medical Sciences (NIGMS); by the Research Center in Minority Institutions (RCMI) award number 8G12 MD007600 from the National Institute on Minority Health and Health Disparities (NIMHD); by the grant number R25 GM061838 from the Minority Biomedical Research Support-Research Initiative for Scientific Enhancement (MBRS-RISE) at the University of Puerto Rico Medical Sciences Campus; and by award number 8U54 MD007587 from NIMHD of the National Institutes of Health (NIH). Genomas Inc. provided support in the form of salaries for authors [GR], but did not have any additional role in the study design, data collection and analysis, decision to publish, or preparation of the manuscript.

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Data Availability Statement

All relevant data are within the paper and its Supporting Information files.

### Supplementary Materials

### Data Availability Statement

All relevant data are within the paper and its Supporting Information files.

## References

1. Johnson JA, Gong L, Whirl-Carrillo M, Gage BF, Scott SA, Stein CM, et al. Clinical Pharmacogenetics Implementation Consortium Guidelines for CYP2C9 and VKORC1 Genotypes and Warfarin Dosing. J Clin Pharmacol Ther 2011; 90: 625–629.  [DOI](https://doi.org/10.1038/clpt.2011.185) | [PMC free article](/articles/PMC3187550/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21900891/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Pharmacol%20Ther&title=Clinical%20Pharmacogenetics%20Implementation%20Consortium%20Guidelines%20for%20CYP2C9%20and%20VKORC1%20Genotypes%20and%20Warfarin%20Dosing&author=JA%20Johnson&author=L%20Gong&author=M%20Whirl-Carrillo&author=BF%20Gage&author=SA%20Scott&volume=90&publication_year=2011&pages=625-629&pmid=21900891&doi=10.1038/clpt.2011.185&)

2. Bristol-Myers Squibb. Coumadin (warfarin sodium) package insert. Princeton, New Jersey, USA. October, 2011. http://packageinserts.bms.com/pi/pi_coumadin.pdf. Accessed 11 April 2015.  [http://packageinserts.bms.com/pi/pi_coumadin.pdf](http://packageinserts.bms.com/pi/pi_coumadin.pdf)

3. Ruff CT, Giugliano RP, Braunwald E, Morrow DA, Murphy SA, Kuder JF, et al. Comparison of the efficacy and safety of new oral anticoagulants with warfarin in patients with atrial fibrillation: a meta-analysis of randomized trials. Lancet 2014; 383: 955–62. 10.1016/S0140-6736(13)62343-0  [DOI](https://doi.org/10.1016/S0140-6736(13)62343-0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24315724/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Comparison%20of%20the%20efficacy%20and%20safety%20of%20new%20oral%20anticoagulants%20with%20warfarin%20in%20patients%20with%20atrial%20fibrillation:%20a%20meta-analysis%20of%20randomized%20trials&author=CT%20Ruff&author=RP%20Giugliano&author=E%20Braunwald&author=DA%20Morrow&author=SA%20Murphy&volume=383&publication_year=2014&pages=955-62&pmid=24315724&doi=10.1016/S0140-6736(13)62343-0&)

4. Bartholow M. Top 200 Drugs of 2012. Pharmacy Times 2013. July 13; 79(7):42, 44.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacy%20Times&title=Top%20200%20Drugs%20of%202012&author=M%20Bartholow&volume=79&issue=7&publication_year=2013&)

5. Declining medicine use and costs: for better or worse? A review of the use of medicines in the United States in 2012; IMS Institute for Healthcare Informatics; Available: http://www.imshealth.com/deployedfiles/ims/Global/Content/Insights/IMS%20Institute%20for%20Healthcare%20Informatics/2012%20U.S.%20Medicines%20Report/2012_U.S.Medicines_Report.pdf. Accessed 11 April 2015.  [http://www.imshealth.com/deployedfiles/ims/Global/Content/Insights/IMS%20Institute%20for%20Healthcare%20Informatics/2012%20U.S.%20Medicines%20Report/2012_U.S.Medicines_Report.pdf](http://www.imshealth.com/deployedfiles/ims/Global/Content/Insights/IMS%20Institute%20for%20Healthcare%20Informatics/2012%20U.S.%20Medicines%20Report/2012_U.S.Medicines_Report.pdf) | [Google Scholar](https://scholar.google.com/scholar_lookup?title=A%20review%20of%20the%20use%20of%20medicines%20in%20the%20United%20States%20in%202012&)

6. Wysowski DK, Nourjah P, Swartz L. Bleeding complications with warfarin use: a prevalent adverse effect resulting in regulatory action. Arch Intern Med 2007; 167: 1414–1419.  [DOI](https://doi.org/10.1001/archinte.167.13.1414) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17620536/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arch%20Intern%20Med&title=Bleeding%20complications%20with%20warfarin%20use:%20a%20prevalent%20adverse%20effect%20resulting%20in%20regulatory%20action&author=DK%20Wysowski&author=P%20Nourjah&author=L%20Swartz&volume=167&publication_year=2007&pages=1414-1419&pmid=17620536&doi=10.1001/archinte.167.13.1414&)

7. Budnitz DS, Pollock DA, Weidenbach KN, Mendelsohn AB, Schroeder TJ, Annest JL. National surveillance of emergency department visits for outpatient adverse drug events. JAMA 2006; 296: 1858–66.  [DOI](https://doi.org/10.1001/jama.296.15.1858) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17047216/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA&title=National%20surveillance%20of%20emergency%20department%20visits%20for%20outpatient%20adverse%20drug%20events&author=DS%20Budnitz&author=DA%20Pollock&author=KN%20Weidenbach&author=AB%20Mendelsohn&author=TJ%20Schroeder&volume=296&publication_year=2006&pages=1858-66&pmid=17047216&doi=10.1001/jama.296.15.1858&)

8. Gulseth MP, Grice GR, Dager WE. Pharmacogenomics of Warfarin: Uncovering a Piece of the Warfarin Mystery. Clinical Frontiers 2009; 66, 123–133.  [DOI](https://doi.org/10.2146/ajhp080127) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19139476/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clinical%20Frontiers&title=Pharmacogenomics%20of%20Warfarin:%20Uncovering%20a%20Piece%20of%20the%20Warfarin%20Mystery&author=MP%20Gulseth&author=GR%20Grice&author=WE%20Dager&volume=66&publication_year=2009&pages=123-133&pmid=19139476&doi=10.2146/ajhp080127&)

9. Zineh I, Pacanowski M, Woodcock J. Pharmacogenetics and coumarin dosing—recalibrating expectations. N Engl J Med 2013; 369: 2273–75. 10.1056/NEJMp1314529  [DOI](https://doi.org/10.1056/NEJMp1314529) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24328463/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Pharmacogenetics%20and%20coumarin%20dosing%E2%80%94recalibrating%20expectations&author=I%20Zineh&author=M%20Pacanowski&author=J%20Woodcock&volume=369&publication_year=2013&pages=2273-75&pmid=24328463&doi=10.1056/NEJMp1314529&)

10. Centers for Medicaid and Medicare Services. Decision memo for pharmacogenomic testing for warfarin response (CAG-00400N). Available: http://www.cms.gov/medicare-coverage-database/details/nca-decision-memo.aspx?NCAId=224. Accessed 21 April 2015.  [http://www.cms.gov/medicare-coverage-database/details/nca-decision-memo.aspx?NCAId=224](http://www.cms.gov/medicare-coverage-database/details/nca-decision-memo.aspx?NCAId=224)

11. Duconge J, Cadilla CL, Seip RL, Ruaño G. Why admixture matters in genetically-guided therapy: Missed targets in the COAG and EU-PACT trials. PR Health Sci. J 2015; 34(3): 175–177.  [PMC free article](/articles/PMC4770896/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26454897/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PR%20Health%20Sci.%20J&title=Why%20admixture%20matters%20in%20genetically-guided%20therapy:%20Missed%20targets%20in%20the%20COAG%20and%20EU-PACT%20trials&author=J%20Duconge&author=CL%20Cadilla&author=RL%20Seip&author=G%20Rua%C3%B1o&volume=34&issue=3&publication_year=2015&pages=175-177&pmid=26454897&)

12. Bress A, Patel SR, Perera MA, Campbell RT, Kittles RA, Cavallari LH. Effect of NQO1 and CYP4F2 genotypes on warfarin dose requirements in Hispanic-Americans and African-Americans Pharmacogenomics 2012; 13(16):1925–35. 10.2217/pgs.12.164  [DOI](https://doi.org/10.2217/pgs.12.164) | [PMC free article](/articles/PMC3586586/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23215885/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Effect%20of%20NQO1%20and%20CYP4F2%20genotypes%20on%20warfarin%20dose%20requirements%20in%20Hispanic-Americans%20and%20African-Americans&author=A%20Bress&author=SR%20Patel&author=MA%20Perera&author=RT%20Campbell&author=RA%20Kittles&volume=13&issue=16&publication_year=2012&pages=1925-35&pmid=23215885&doi=10.2217/pgs.12.164&)

13. Duconge J. Population Heterogeneity and Genomic Admixture: Relevance for Global Pharmacogenetics. Pharmacogenomics Pharmacoproteomics 2014; 5:5 Available: 10.4172/2153-0645.1000e141  [DOI](https://doi.org/10.4172/2153-0645.1000e141) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20Pharmacoproteomics&title=Population%20Heterogeneity%20and%20Genomic%20Admixture:%20Relevance%20for%20Global%20Pharmacogenetics&author=J%20Duconge&volume=5&publication_year=2014&pages=5&doi=10.4172/2153-0645.1000e141&)

14. Antithrombotic therapy and prevention of thrombosis, 9th ed. In: American College of Chest Physicians evidence-based clinical practice guidelines. Chest 2012; p.141 (suppl. 2).  [DOI](https://doi.org/10.1378/chest.11-2294) | [PMC free article](/articles/PMC3278067/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22315258/)

15. Wigle P, Hein B, Bloomfield HE, Tubb M, Doherty M. Updated Guidelines on Outpatient Anticoagulation. Am Fam Physician. 2013; 87(8):556–566.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/23668445/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20Fam%20Physician&title=Updated%20Guidelines%20on%20Outpatient%20Anticoagulation&author=P%20Wigle&author=B%20Hein&author=HE%20Bloomfield&author=M%20Tubb&author=M%20Doherty&volume=87&issue=8&publication_year=2013&pages=556-566&pmid=23668445&)

16. Sissung TM, English BC, Venzon D, Figg WD, Deeken JF. Clinical pharmacology and pharmacogenetics in a genomics era: the DMET platform. Pharmacogenomics. 2010; 11(1):89–103. 10.2217/pgs.09.154  [DOI](https://doi.org/10.2217/pgs.09.154) | [PMC free article](/articles/PMC6448402/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20017675/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Clinical%20pharmacology%20and%20pharmacogenetics%20in%20a%20genomics%20era:%20the%20DMET%20platform&author=TM%20Sissung&author=BC%20English&author=D%20Venzon&author=WD%20Figg&author=JF%20Deeken&volume=11&issue=1&publication_year=2010&pages=89-103&pmid=20017675&doi=10.2217/pgs.09.154&)

17. Guzzi PH, Agapito G, Teresa M, Martino D, Arbitrio M, Tagliaferri P, et al. DMET-Analyzer: automatic analysis of Affymetrix DMET Data. BMC Bioinformatics 2012; 13:258 10.1186/1471-2105-13-258  [DOI](https://doi.org/10.1186/1471-2105-13-258) | [PMC free article](/articles/PMC3496574/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23035929/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Bioinformatics&title=DMET-Analyzer:%20automatic%20analysis%20of%20Affymetrix%20DMET%20Data&author=PH%20Guzzi&author=G%20Agapito&author=M%20Teresa&author=D%20Martino&author=M%20Arbitrio&volume=13&publication_year=2012&pages=258&pmid=23035929&doi=10.1186/1471-2105-13-258&)

18. Orengo-Mercado C, Nieves B, López L, Vallés-Ortiz N, Renta JY, Santiago-Borrero PJ, et al. Frequencies of Functional Polymorphisms in three Pharmacokinetic Genes of Clinical Interest within the Admixed Puerto Rican Population. J Pharmacogenom Pharmacoproteomics 2013, 4:113 10.4172/2153-0645.1000113  [DOI](https://doi.org/10.4172/2153-0645.1000113) | [PMC free article](/articles/PMC3769800/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24040574/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharmacogenom%20Pharmacoproteomics&title=Frequencies%20of%20Functional%20Polymorphisms%20in%20three%20Pharmacokinetic%20Genes%20of%20Clinical%20Interest%20within%20the%20Admixed%20Puerto%20Rican%20Population&author=C%20Orengo-Mercado&author=B%20Nieves&author=L%20L%C3%B3pez&author=N%20Vall%C3%A9s-Ortiz&author=JY%20Renta&volume=4&publication_year=2013&pages=113&pmid=24040574&doi=10.4172/2153-0645.1000113&)

19. Livak KJ. Allelic discrimination using fluorogenic probes and the 5’ nuclease assay. Genet Anal-Biomol Eng. 1999; 14(5–6):143–9.  [DOI](https://doi.org/10.1016/s1050-3862(98)00019-9) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10084106/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genet%20Anal-Biomol%20Eng&title=Allelic%20discrimination%20using%20fluorogenic%20probes%20and%20the%205%E2%80%99%20nuclease%20assay&author=KJ%20Livak&volume=14&issue=5%E2%80%936&publication_year=1999&pages=143-9&pmid=10084106&doi=10.1016/s1050-3862(98)00019-9&)

20. Gordon J, Merante F, Weiss S, Zastawny R. Pharmacogenetic P-450 screening using the Tag-It universal bead-based array platform In: Wong SH, Linder MW, Valdes R, eds. Pharmacogenomics and Proteomics: Enabling the Practice of Personalized Medicine. Washington, DC: AAC Press, 2006.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Pharmacogenomics%20and%20Proteomics:%20Enabling%20the%20Practice%20of%20Personalized%20Medicine&author=J%20Gordon&author=F%20Merante&author=S%20Weiss&author=R%20Zastawny&author=SH%20Wong&publication_year=2006&)

21. Ruaño G, Thompson PD, Villagra D, Bower B, Kocherla M, Yazdanpanah G, et al. High carrier prevalence of combinatorial CYP2C9 and VKORC1 genotypes affecting warfarin dosing. Personalized Medicine 2008; 5(3):225–232.  [DOI](https://doi.org/10.2217/17410541.5.3.225) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29783498/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Personalized%20Medicine&title=High%20carrier%20prevalence%20of%20combinatorial%20CYP2C9%20and%20VKORC1%20genotypes%20affecting%20warfarin%20dosing&author=G%20Rua%C3%B1o&author=PD%20Thompson&author=D%20Villagra&author=B%20Bower&author=M%20Kocherla&volume=5&issue=3&publication_year=2008&pages=225-232&pmid=29783498&doi=10.2217/17410541.5.3.225&)

22. Home Page of the Human Cytochrome P450 (CYP) Allele Nomenclature Committee. Available: http://www.cypalleles.ki.se/cyp2c9.htm  [http://www.cypalleles.ki.se/cyp2c9.htm](http://www.cypalleles.ki.se/cyp2c9.htm)

23. Sim SC, Ingelman-Sundberg M. The human cytochrome P450 Allele Nomenclature Committee Website: submission criteria, procedures, and objectives. Methods Mol Biol 2006; 320:183–191.  [DOI](https://doi.org/10.1385/1-59259-998-2:183) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16719391/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Methods%20Mol%20Biol&title=The%20human%20cytochrome%20P450%20Allele%20Nomenclature%20Committee%20Website:%20submission%20criteria,%20procedures,%20and%20objectives&author=SC%20Sim&author=M%20Ingelman-Sundberg&volume=320&publication_year=2006&pages=183-191&pmid=16719391&doi=10.1385/1-59259-998-2:183&)

24. Ruano G, Duconge J, Windemuth A, Cadilla CL, Kocherla M, Villagra D, et al. Physiogenomic analysis of the Puerto Rican population. Pharmacogenomics 2009; 10(4): 565–577. 10.2217/pgs.09.5  [DOI](https://doi.org/10.2217/pgs.09.5) | [PMC free article](/articles/PMC2846824/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19374515/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Physiogenomic%20analysis%20of%20the%20Puerto%20Rican%20population&author=G%20Ruano&author=J%20Duconge&author=A%20Windemuth&author=CL%20Cadilla&author=M%20Kocherla&volume=10&issue=4&publication_year=2009&pages=565-577&pmid=19374515&doi=10.2217/pgs.09.5&)

25. Oliphant A, Barker DL, Stuelpnagel JR, Chee MS. BeadArray technology: enabling an accurate, cost-effective approach to high-throughput genotyping. Biotechniques Suppl 2002; 56–58:60–61.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/12083399/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biotechniques%20Suppl&title=BeadArray%20technology:%20enabling%20an%20accurate,%20cost-effective%20approach%20to%20high-throughput%20genotyping&author=A%20Oliphant&author=DL%20Barker&author=JR%20Stuelpnagel&author=MS%20Chee&volume=56%E2%80%9358&publication_year=2002&pages=60-61&pmid=12083399&)

26. Fan JB, Oliphant A, Shen R, Kermani BG, Garcia F, Gunderson KL, et al. Highly parallel SNP genotyping. Cold Spring Harb Symp Quant Biol 2003; 68:69–78.  [DOI](https://doi.org/10.1101/sqb.2003.68.69) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15338605/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cold%20Spring%20Harb%20Symp%20Quant%20Biol&title=Highly%20parallel%20SNP%20genotyping&author=JB%20Fan&author=A%20Oliphant&author=R%20Shen&author=BG%20Kermani&author=F%20Garcia&volume=68&publication_year=2003&pages=69-78&pmid=15338605&doi=10.1101/sqb.2003.68.69&)

27. Pritchard JK, Stephens M, Donnelly P. Inference of population structure using multilocus genotype data Genetics 2000; 155:945–959.  [DOI](https://doi.org/10.1093/genetics/155.2.945) | [PMC free article](/articles/PMC1461096/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10835412/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genetics&title=Inference%20of%20population%20structure%20using%20multilocus%20genotype%20data&author=JK%20Pritchard&author=M%20Stephens&author=P%20Donnelly&volume=155&publication_year=2000&pages=945-959&pmid=10835412&doi=10.1093/genetics/155.2.945&)

28. Falush D, Stephens M, Pritchard JK. Inference of population structure using multilocus genotype data: dominant markers and null alleles. Mol Ecol Notes 2007; 7:574–578.  [DOI](https://doi.org/10.1111/j.1471-8286.2007.01758.x) | [PMC free article](/articles/PMC1974779/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18784791/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Ecol%20Notes&title=Inference%20of%20population%20structure%20using%20multilocus%20genotype%20data:%20dominant%20markers%20and%20null%20alleles&author=D%20Falush&author=M%20Stephens&author=JK%20Pritchard&volume=7&publication_year=2007&pages=574-578&pmid=18784791&doi=10.1111/j.1471-8286.2007.01758.x&)

29. The University of Chicago Department of Biological Sciences, Pritchard Lab. Available: http://pritch.bsd.uchicago.edu. Accessed 11 April 2015.  [http://pritch.bsd.uchicago.edu](http://pritch.bsd.uchicago.edu)

30. Parra EJ, Kittles RA, Shriver MD. Implications of correlations between skin color and genetic ancestry for biomedical research Nat Genet 2004; 36:S54–S60.  [DOI](https://doi.org/10.1038/ng1440) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15508005/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Implications%20of%20correlations%20between%20skin%20color%20and%20genetic%20ancestry%20for%20biomedical%20research&author=EJ%20Parra&author=RA%20Kittles&author=MD%20Shriver&volume=36&publication_year=2004&pages=S54-S60&pmid=15508005&doi=10.1038/ng1440&)

31. González-Burchard E, Borrell LN, Choudhry S, Naqvi M, Tsai HJ, Rodríguez-Santana JR, et al. Latino populations: a unique opportunity for the study of race, genetics, and social environment in epidemiological research. Am J Public Health 2005; 95(12):2161–2168.  [DOI](https://doi.org/10.2105/AJPH.2005.068668) | [PMC free article](/articles/PMC1449501/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16257940/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Public%20Health&title=Latino%20populations:%20a%20unique%20opportunity%20for%20the%20study%20of%20race,%20genetics,%20and%20social%20environment%20in%20epidemiological%20research&author=E%20Gonz%C3%A1lez-Burchard&author=LN%20Borrell&author=S%20Choudhry&author=M%20Naqvi&author=HJ%20Tsai&volume=95&issue=12&publication_year=2005&pages=2161-2168&pmid=16257940&doi=10.2105/AJPH.2005.068668&)

32. Salari K, Choudhry S, Tang H, Naqvi M, Lind D, Avila PC, et al. Genetic admixture and asthma-related phenotypes in Mexican American and Puerto Rican asthmatics. Genet Epidemiol 2005; 29(1):76–86.  [DOI](https://doi.org/10.1002/gepi.20079) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15918156/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genet%20Epidemiol&title=Genetic%20admixture%20and%20asthma-related%20phenotypes%20in%20Mexican%20American%20and%20Puerto%20Rican%20asthmatics&author=K%20Salari&author=S%20Choudhry&author=H%20Tang&author=M%20Naqvi&author=D%20Lind&volume=29&issue=1&publication_year=2005&pages=76-86&pmid=15918156&doi=10.1002/gepi.20079&)

33. Bertoni B, Budowle B, Sans M, Barton SA, Chakraborty R. Admixture in Hispanics: distribution of ancestral population contributions in the Continental United States. Hum Biol 2003; 75:1–11.  [DOI](https://doi.org/10.1353/hub.2003.0016) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12713142/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Biol&title=Admixture%20in%20Hispanics:%20distribution%20of%20ancestral%20population%20contributions%20in%20the%20Continental%20United%20States&author=B%20Bertoni&author=B%20Budowle&author=M%20Sans&author=SA%20Barton&author=R%20Chakraborty&volume=75&publication_year=2003&pages=1-11&pmid=12713142&doi=10.1353/hub.2003.0016&)

34. Martínez-Cruzado JC, Toro-Labrador G, Viera-Vera J, Rivera-Vega MY, Startek J, Latorre-Esteves M, et al. Reconstructing the population history of Puerto Rico by means of mtDNA phylogeographic analysis Am J Phys Anthropol 2005; 128:131–155.  [DOI](https://doi.org/10.1002/ajpa.20108) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15693025/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Phys%20Anthropol&title=Reconstructing%20the%20population%20history%20of%20Puerto%20Rico%20by%20means%20of%20mtDNA%20phylogeographic%20analysis&author=JC%20Mart%C3%ADnez-Cruzado&author=G%20Toro-Labrador&author=J%20Viera-Vera&author=MY%20Rivera-Vega&author=J%20Startek&volume=128&publication_year=2005&pages=131-155&pmid=15693025&doi=10.1002/ajpa.20108&)

35. Choudhry S, Coyle NE, Tang H, Salari K, Lind D, Clark SL, et al. Population stratification confounds genetic association studies among Latinos. Hum Genet 2006; 118:652–64.  [DOI](https://doi.org/10.1007/s00439-005-0071-3) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16283388/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Genet&title=Population%20stratification%20confounds%20genetic%20association%20studies%20among%20Latinos&author=S%20Choudhry&author=NE%20Coyle&author=H%20Tang&author=K%20Salari&author=D%20Lind&volume=118&publication_year=2006&pages=652-64&pmid=16283388&doi=10.1007/s00439-005-0071-3&)

36. Lenzini P, Wadelius M, Kimmel S, Anderson JL, Jorgensen AL, Pirmohamed M, et al. Integration of genetic, clinical, and laboratory data to refine warfarin dosing. Clin Pharmacol Ther. 2010; 87:572–578. 10.1038/clpt.2010.13  [DOI](https://doi.org/10.1038/clpt.2010.13) | [PMC free article](/articles/PMC2858245/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20375999/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Integration%20of%20genetic,%20clinical,%20and%20laboratory%20data%20to%20refine%20warfarin%20dosing&author=P%20Lenzini&author=M%20Wadelius&author=S%20Kimmel&author=JL%20Anderson&author=AL%20Jorgensen&volume=87&publication_year=2010&pages=572-578&pmid=20375999&doi=10.1038/clpt.2010.13&)

37. International Warfarin Pharmacogenetics Consortium, Klein TE, Altman RB, Eriksson N, Gage BF, Kimmel SE, Lee MT, et al. Estimation of the warfarin dose with clinical and pharmacogenetic data N Engl J Med. 2009; 360:753–764. 10.1056/NEJMoa0809329  [DOI](https://doi.org/10.1056/NEJMoa0809329) | [PMC free article](/articles/PMC2722908/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19228618/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Estimation%20of%20the%20warfarin%20dose%20with%20clinical%20and%20pharmacogenetic%20data&author=TE%20Klein&author=RB%20Altman&author=N%20Eriksson&author=BF%20Gage&author=SE%20Kimmel&volume=360&publication_year=2009&pages=753-764&pmid=19228618&doi=10.1056/NEJMoa0809329&)

38. Ramos AS, Seip RL, Rivera-Miranda G, Felici-Giovanini ME, Garcia-Berdecia R, Alejandro-Cowan Y, et al. Development of a pharmacogenetic-guided warfarin dosing algorithm for Puerto Rican patients. Pharmacogenomics 2012; 13(16):1937–50. 10.2217/pgs.12.171  [DOI](https://doi.org/10.2217/pgs.12.171) | [PMC free article](/articles/PMC3538136/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23215886/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Development%20of%20a%20pharmacogenetic-guided%20warfarin%20dosing%20algorithm%20for%20Puerto%20Rican%20patients&author=AS%20Ramos&author=RL%20Seip&author=G%20Rivera-Miranda&author=ME%20Felici-Giovanini&author=R%20Garcia-Berdecia&volume=13&issue=16&publication_year=2012&pages=1937-50&pmid=23215886&doi=10.2217/pgs.12.171&)

39. Cavallari LH, Langaee TY, Momary KM, Shapiro NL, Nutescu EA, Coty WA, et al. Genetic and clinical predictors of warfarin dose requirements in African Americans. Clin Pharmacol Ther. 2010; 87:459–464. 10.1038/clpt.2009.223  [DOI](https://doi.org/10.1038/clpt.2009.223) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20072124/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Genetic%20and%20clinical%20predictors%20of%20warfarin%20dose%20requirements%20in%20African%20Americans&author=LH%20Cavallari&author=TY%20Langaee&author=KM%20Momary&author=NL%20Shapiro&author=EA%20Nutescu&volume=87&publication_year=2010&pages=459-464&pmid=20072124&doi=10.1038/clpt.2009.223&)

40. Dickmann LJ, Rettie AE, Kneller MB, Kim RB, Wood AJJ, Stein CM, et al. Identification and functional characterization of a new CYP2C9 variant (CYP2C9*5) expressed among African Americans. Mol Pharmacol 2001; 60: 382–387.  [DOI](https://doi.org/10.1124/mol.60.2.382) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11455026/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Pharmacol&title=Identification%20and%20functional%20characterization%20of%20a%20new%20CYP2C9%20variant%20(CYP2C9*5)%20expressed%20among%20African%20Americans&author=LJ%20Dickmann&author=AE%20Rettie&author=MB%20Kneller&author=RB%20Kim&author=AJJ%20Wood&volume=60&publication_year=2001&pages=382-387&pmid=11455026&doi=10.1124/mol.60.2.382&)

41. Drozda K, Wong S, Patel SR, Bress AP, Nutescu EA, Kittles RA, et al. Poor warfarin dose prediction with pharmacogenetic algorithms that exclude genotypes important for African Americans Pharmacogenet Genomics. 2015; 25(2):73–81  [DOI](https://doi.org/10.1097/FPC.0000000000000108) | [PMC free article](/articles/PMC4280295/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25461246/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Americans%20Pharmacogenet%20Genomics&title=Poor%20warfarin%20dose%20prediction%20with%20pharmacogenetic%20algorithms%20that%20exclude%20genotypes%20important%20for%20African&author=K%20Drozda&author=S%20Wong&author=SR%20Patel&author=AP%20Bress&author=EA%20Nutescu&volume=25&issue=2&publication_year=2015&pages=73-81&pmid=25461246&doi=10.1097/FPC.0000000000000108&)

42. Scott SA, Jaremko M, Lubitz SA, Kornreich R, Halperin JL, Desnick RJ. CYP2C9*8 is prevalent among African-Americans: implications for pharmacogenetic dosing. Pharmacogenomics 2009; 10(8):1243–55. 10.2217/pgs.09.71  [DOI](https://doi.org/10.2217/pgs.09.71) | [PMC free article](/articles/PMC2737687/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19663669/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=CYP2C9*8%20is%20prevalent%20among%20African-Americans:%20implications%20for%20pharmacogenetic%20dosing&author=SA%20Scott&author=M%20Jaremko&author=SA%20Lubitz&author=R%20Kornreich&author=JL%20Halperin&volume=10&issue=8&publication_year=2009&pages=1243-55&pmid=19663669&doi=10.2217/pgs.09.71&)

43. Liu Y, Jeong H, Takahashi H, Drozda K, Patel SR, Shapiro NL, et al. Decreased Warfarin Clearance associated with CYP2C9R150H (*8) polymorphism. Clin Pharmacol Ther 2012; 91(4): 660–665. 10.1038/clpt.2011.269  [DOI](https://doi.org/10.1038/clpt.2011.269) | [PMC free article](/articles/PMC3879795/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22378156/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Decreased%20Warfarin%20Clearance%20associated%20with%20CYP2C9R150H%20(*8)%20polymorphism&author=Y%20Liu&author=H%20Jeong&author=H%20Takahashi&author=K%20Drozda&author=SR%20Patel&volume=91&issue=4&publication_year=2012&pages=660-665&pmid=22378156&doi=10.1038/clpt.2011.269&)

44. Cavallari LH, Vaynshteyn D, Freeman KM, Wang D, Perera MA, Takahashi H, et al. CYP2C9 promoter region single-nucleotide polymorphisms linked to the R150H polymorphism are functional suggesting their role in CYP2C9*8-mediated effects. Pharmacogenet Genomics. 2013; 23(4):228–31. 10.1097/FPC.0b013e32835e95c7  [DOI](https://doi.org/10.1097/FPC.0b013e32835e95c7) | [PMC free article](/articles/PMC3629689/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23376925/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=CYP2C9%20promoter%20region%20single-nucleotide%20polymorphisms%20linked%20to%20the%20R150H%20polymorphism%20are%20functional%20suggesting%20their%20role%20in%20CYP2C9*8-mediated%20effects&author=LH%20Cavallari&author=D%20Vaynshteyn&author=KM%20Freeman&author=D%20Wang&author=MA%20Perera&volume=23&issue=4&publication_year=2013&pages=228-31&pmid=23376925&doi=10.1097/FPC.0b013e32835e95c7&)

45. Higashi MK, Veenstra DL, Kondo LM, Wittkowsky AK, Srinouanprachanh SL, Farin FM, et al. Association between CYP2C9 genetic variants and anticoagulation-related outcomes during warfarin therapy JAMA 2002; 287: 1690–98.  [DOI](https://doi.org/10.1001/jama.287.13.1690) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11926893/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA&title=Association%20between%20CYP2C9%20genetic%20variants%20and%20anticoagulation-related%20outcomes%20during%20warfarin%20therapy&author=MK%20Higashi&author=DL%20Veenstra&author=LM%20Kondo&author=AK%20Wittkowsky&author=SL%20Srinouanprachanh&volume=287&publication_year=2002&pages=1690-98&pmid=11926893&doi=10.1001/jama.287.13.1690&)

46. Aithal GP, Day CP, Kesteven PJ, Daly AK. Association of polymorphisms in the Cytochrome P450 CYP2C9 with warfarin dose requirement and risk of bleeding complications Lancet 1999; 353: 717–719.  [DOI](https://doi.org/10.1016/S0140-6736(98)04474-2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10073515/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Association%20of%20polymorphisms%20in%20the%20Cytochrome%20P450%20CYP2C9%20with%20warfarin%20dose%20requirement%20and%20risk%20of%20bleeding%20complications&author=GP%20Aithal&author=CP%20Day&author=PJ%20Kesteven&author=AK%20Daly&volume=353&publication_year=1999&pages=717-719&pmid=10073515&doi=10.1016/S0140-6736(98)04474-2&)

47. Reynolds KK, Valdes R Jr, Hartung BR, Linder MW. Individualizing Warfarin Therapy Personalized Medicine 2007; 4: 11–31.  [DOI](https://doi.org/10.2217/17410541.4.1.11) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29793302/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Individualizing%20Warfarin%20Therapy%20Personalized%20Medicine&author=KK%20Reynolds&author=R%20Valdes&author=BR%20Hartung&author=MW%20Linder&volume=4&publication_year=2007&pages=11-31&pmid=29793302&doi=10.2217/17410541.4.1.11&)

48. Sanderson S, Emery J, Higgins J. CYP2C9 gene variants, drug dose, and bleeding risk in warfarin treated patients: A HuGEnet™ systemic review and meta-analysis. Genet Med. 2005; 7: 97–104.  [DOI](https://doi.org/10.1097/01.gim.0000153664.65759.cf) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15714076/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genet%20Med&title=CYP2C9%20gene%20variants,%20drug%20dose,%20and%20bleeding%20risk%20in%20warfarin%20treated%20patients:%20A%20HuGEnet%E2%84%A2%20systemic%20review%20and%20meta-analysis&author=S%20Sanderson&author=J%20Emery&author=J%20Higgins&volume=7&publication_year=2005&pages=97-104&pmid=15714076&doi=10.1097/01.gim.0000153664.65759.cf&)

49. Takahashi H, Wilkinson GR, Nutescu EA, Morita T, Ritchie MD, Scordo MG, et al. Different contributions of polymorphisms in VKORC1 and CYP2C9 to intra- and inter-population differences in maintenance doses of warfarin in Japanese, Caucasians and African Americans. Pharmacogenet Genomics 2006; 16: 101–110.  [DOI](https://doi.org/10.1097/01.fpc.0000184955.08453.a8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16424822/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Different%20contributions%20of%20polymorphisms%20in%20VKORC1%20and%20CYP2C9%20to%20intra-%20and%20inter-population%20differences%20in%20maintenance%20doses%20of%20warfarin%20in%20Japanese,%20Caucasians%20and%20African%20Americans&author=H%20Takahashi&author=GR%20Wilkinson&author=EA%20Nutescu&author=T%20Morita&author=MD%20Ritchie&volume=16&publication_year=2006&pages=101-110&pmid=16424822&doi=10.1097/01.fpc.0000184955.08453.a8&)

50. Veenstra DL, You JHS, Rieder MJ, Farin FM, Wilkerson HW, Blough DK, et al. Association of Vitamin K epoxide reductase complex 1 (VKORC1) variants with warfarin dose in a Hong Kong Chinese patient population. Pharmacogenet Genomics 2005; 15: 687–691.  [DOI](https://doi.org/10.1097/01.fpc.0000174789.77614.68) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16141794/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Association%20of%20Vitamin%20K%20epoxide%20reductase%20complex%201%20(VKORC1)%20variants%20with%20warfarin%20dose%20in%20a%20Hong%20Kong%20Chinese%20patient%20population&author=DL%20Veenstra&author=JHS%20You&author=MJ%20Rieder&author=FM%20Farin&author=HW%20Wilkerson&volume=15&publication_year=2005&pages=687-691&pmid=16141794&doi=10.1097/01.fpc.0000174789.77614.68&)

51. Caldwell MD, Berg RL, Zhang KQ, Glurich I, Schmelzer JR, Yale SH, et al. Evaluation of Genetic Factors for Warfarin Dose Prediction. Clinical Medicine & Research 2007; 5(1): 8–16.  [DOI](https://doi.org/10.3121/cmr.2007.724) | [PMC free article](/articles/PMC1855340/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17456829/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clinical%20Medicine%20&%20Research&title=Evaluation%20of%20Genetic%20Factors%20for%20Warfarin%20Dose%20Prediction&author=MD%20Caldwell&author=RL%20Berg&author=KQ%20Zhang&author=I%20Glurich&author=JR%20Schmelzer&volume=5&issue=1&publication_year=2007&pages=8-16&pmid=17456829&doi=10.3121/cmr.2007.724&)

52. Goldstein JA. Clinical relevance of genetic polymorphisms in the human CYP2C subfamily. Br J Clin Pharmacol 2001; 52: 349–355.  [DOI](https://doi.org/10.1046/j.0306-5251.2001.01499.x) | [PMC free article](/articles/PMC2014584/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11678778/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Clin%20Pharmacol&title=Clinical%20relevance%20of%20genetic%20polymorphisms%20in%20the%20human%20CYP2C%20subfamily&author=JA%20Goldstein&volume=52&publication_year=2001&pages=349-355&pmid=11678778&doi=10.1046/j.0306-5251.2001.01499.x&)

53. Kimmel SE, French B, Kasner SE, Johnson JA, Anderson JL, Gage BF, et al. A pharmacogenetic versus a clinical algorithm for warfarin dosing. N. Engl. J. Med. 2013; 369 (24): 2283–93. 10.1056/NEJMoa1310669  [DOI](https://doi.org/10.1056/NEJMoa1310669) | [PMC free article](/articles/PMC3942158/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24251361/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N.%20Engl.%20J.%20Med.&title=A%20pharmacogenetic%20versus%20a%20clinical%20algorithm%20for%20warfarin%20dosing&author=SE%20Kimmel&author=B%20French&author=SE%20Kasner&author=JA%20Johnson&author=JL%20Anderson&volume=369&issue=24&publication_year=2013&pages=2283-93&pmid=24251361&doi=10.1056/NEJMoa1310669&)

54. Via M, Gignoux CR, Roth LA, Fejerman L, Galanter J, Choudhry S, et al. History shaped the geographic distribution of genomic admixture on the island of Puerto Rico. PLoS One 2011; 6(1):e16513 10.1371/journal.pone.0016513  [DOI](https://doi.org/10.1371/journal.pone.0016513) | [PMC free article](/articles/PMC3031579/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21304981/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=History%20shaped%20the%20geographic%20distribution%20of%20genomic%20admixture%20on%20the%20island%20of%20Puerto%20Rico&author=M%20Via&author=CR%20Gignoux&author=LA%20Roth&author=L%20Fejerman&author=J%20Galanter&volume=6&issue=1&publication_year=2011&pages=e16513&pmid=21304981&doi=10.1371/journal.pone.0016513&)

55. Villagra D, Duconge J, Windemuth A, Cadilla CL, Kocherla M, Gorowski K, et al. CYP2C9 and VKORC1 genotypes in Puerto Ricans: A case for admixture-matching in clinical pharmacogenetic studies. Clin Chim Acta 2010; 411(17–18):1306–11. 10.1016/j.cca.2010.05.021  [DOI](https://doi.org/10.1016/j.cca.2010.05.021) | [PMC free article](/articles/PMC2903218/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20488169/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Chim%20Acta&title=CYP2C9%20and%20VKORC1%20genotypes%20in%20Puerto%20Ricans:%20A%20case%20for%20admixture-matching%20in%20clinical%20pharmacogenetic%20studies&author=D%20Villagra&author=J%20Duconge&author=A%20Windemuth&author=CL%20Cadilla&author=M%20Kocherla&volume=411&issue=17%E2%80%9318&publication_year=2010&pages=1306-11&pmid=20488169&doi=10.1016/j.cca.2010.05.021&)

56. Duconge J, Ruaño G. Emerging Role of Admixture in the Pharmacogenetics of Puerto Rican Hispanics. J Pharmacogenom Pharmacoproteomics 2010; (4)1:101.  [DOI](https://doi.org/10.4172/2153-0645.1000101) | [PMC free article](/articles/PMC3515058/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23227441/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharmacogenom%20Pharmacoproteomics&title=Emerging%20Role%20of%20Admixture%20in%20the%20Pharmacogenetics%20of%20Puerto%20Rican%20Hispanics&author=J%20Duconge&author=G%20Rua%C3%B1o&volume=1&publication_year=2010&pages=101&pmid=23227441&doi=10.4172/2153-0645.1000101&)

57. Suarez-Kurtz G. Pharmacogenomics in admixed populations Austin, Texas: Landes Bioscience, 2007.  [Google Scholar](https://scholar.google.com/scholar_lookup?Suarez-Kurtz%20G.%20Pharmacogenomics%20in%20admixed%20populations%20Austin,%20Texas:%20Landes%20Bioscience,%202007.)

58. Suarez-Kurtz G, Pena SD. Pharmacogenomics in the Americas: the impact of genetic admixture. Curr Drug Targets 2006; 7:1649–58.  [DOI](https://doi.org/10.2174/138945006779025392) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17168840/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Drug%20Targets&title=Pharmacogenomics%20in%20the%20Americas:%20the%20impact%20of%20genetic%20admixture&author=G%20Suarez-Kurtz&author=SD%20Pena&volume=7&publication_year=2006&pages=1649-58&pmid=17168840&doi=10.2174/138945006779025392&)

59. Claudio-Campos K, Duconge J, Cadilla CL, Ruaño G. Pharmacogenetics of drug metabolizing enzymes in U.S. Hispanics. Drug Metabol Personal Ther. 2015. June 1; 30(2):87–105. 10.1515/dmdi-2014-0023  [DOI](https://doi.org/10.1515/dmdi-2014-0023) | [PMC free article](/articles/PMC4447600/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25431893/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metabol%20Personal%20Ther&title=Pharmacogenetics%20of%20drug%20metabolizing%20enzymes%20in%20U.S.%20Hispanics&author=K%20Claudio-Campos&author=J%20Duconge&author=CL%20Cadilla&author=G%20Rua%C3%B1o&volume=30&issue=2&publication_year=2015&pages=87-105&pmid=25431893&doi=10.1515/dmdi-2014-0023&)

60. Hillman MA, Wilke RA, Caldwell MD, Berg RL, Glurich I, Burmester JK. Relative impact of covariates in prescribing warfarin according to CYP2C9 genotype. Pharmacogenetics 2004; 14: 539–47.  [DOI](https://doi.org/10.1097/01.fpc.0000114760.08559.dc) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15284536/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=Relative%20impact%20of%20covariates%20in%20prescribing%20warfarin%20according%20to%20CYP2C9%20genotype&author=MA%20Hillman&author=RA%20Wilke&author=MD%20Caldwell&author=RL%20Berg&author=I%20Glurich&volume=14&publication_year=2004&pages=539-47&pmid=15284536&doi=10.1097/01.fpc.0000114760.08559.dc&)

61. Ansell J, Hirsh J, Hylek E, Jacobson A, Crowther M, Palareti G; American College of Chest Physicians. Pharmacology and management of the vitamin K antagonists: American College of Chest Physicians Evidence-Based Clinical Practice Guidelines (8th Edition). Chest 2008; 133(6 Suppl.): 160S–198S. 10.1378/chest.08-0670  [DOI](https://doi.org/10.1378/chest.08-0670) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18574265/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Chest&title=Pharmacology%20and%20management%20of%20the%20vitamin%20K%20antagonists:%20American%20College%20of%20Chest%20Physicians%20Evidence-Based%20Clinical%20Practice%20Guidelines%20(8th%20Edition)&author=J%20Ansell&author=J%20Hirsh&author=E%20Hylek&author=A%20Jacobson&author=M%20Crowther&volume=133&issue=6%20Suppl.&publication_year=2008&pages=160S-198S&pmid=18574265&doi=10.1378/chest.08-0670&)

62. Flockhart DA, O’Kane D, Williams MS, Watson MS, Flockhart DA, Gage B, et al. Pharmacogenetic testing of CYP2C9 and VKORC1 alleles for warfarin. Genet Med 2008; 10:139–50. 10.1097/GIM.0b013e318163c35f  [DOI](https://doi.org/10.1097/GIM.0b013e318163c35f) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18281922/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genet%20Med&title=Pharmacogenetic%20testing%20of%20CYP2C9%20and%20VKORC1%20alleles%20for%20warfarin&author=DA%20Flockhart&author=D%20O%E2%80%99Kane&author=MS%20Williams&author=MS%20Watson&author=DA%20Flockhart&volume=10&publication_year=2008&pages=139-50&pmid=18281922&doi=10.1097/GIM.0b013e318163c35f&)

63. McClain MR, Palomaki GE, Piper M, Haddow JE. A rapid-ACCE review of CYP2C9 and VKORC1 alleles testing to inform warfarin dosing in adults at elevated risk for thrombotic events to avoid serious bleeding Genet Med 2008; 10: 89–98. 10.1097/GIM.0b013e31815bf924  [DOI](https://doi.org/10.1097/GIM.0b013e31815bf924) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18281915/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genet%20Med&title=A%20rapid-ACCE%20review%20of%20CYP2C9%20and%20VKORC1%20alleles%20testing%20to%20inform%20warfarin%20dosing%20in%20adults%20at%20elevated%20risk%20for%20thrombotic%20events%20to%20avoid%20serious%20bleeding&author=MR%20McClain&author=GE%20Palomaki&author=M%20Piper&author=JE%20Haddow&volume=10&publication_year=2008&pages=89-98&pmid=18281915&doi=10.1097/GIM.0b013e31815bf924&)

64. Wadelius M, Chen LY, Lindh JD, Eriksson N, Ghori MJ, Bumpstead S, et al. The largest prospective warfarin-treated cohort supports genetic forecasting. Blood 2009; 113:784–792. 10.1182/blood-2008-04-149070  [DOI](https://doi.org/10.1182/blood-2008-04-149070) | [PMC free article](/articles/PMC2630264/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18574025/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=The%20largest%20prospective%20warfarin-treated%20cohort%20supports%20genetic%20forecasting&author=M%20Wadelius&author=LY%20Chen&author=JD%20Lindh&author=N%20Eriksson&author=MJ%20Ghori&volume=113&publication_year=2009&pages=784-792&pmid=18574025&doi=10.1182/blood-2008-04-149070&)

65. Schwarz UI, Ritchie MD, Bradford Y, Li C, Dudek SM, Frye-Anderson A, et al. Genetic determinants of response to warfarin during initial anticoagulation N Engl J Med. 2008; 358:999–1008. 10.1056/NEJMoa0708078  [DOI](https://doi.org/10.1056/NEJMoa0708078) | [PMC free article](/articles/PMC3894627/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18322281/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Genetic%20determinants%20of%20response%20to%20warfarin%20during%20initial%20anticoagulation&author=UI%20Schwarz&author=MD%20Ritchie&author=Y%20Bradford&author=C%20Li&author=SM%20Dudek&volume=358&publication_year=2008&pages=999-1008&pmid=18322281&doi=10.1056/NEJMoa0708078&)

66. Lindh JD, Holm L, Dahl ML, Alfredsson L, Rane A. Incidence and predictors of severe bleeding during warfarin treatment. J Thromb Thrombolysis 2008; 25:151–159.  [DOI](https://doi.org/10.1007/s11239-007-0048-2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17514429/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Thromb%20Thrombolysis&title=Incidence%20and%20predictors%20of%20severe%20bleeding%20during%20warfarin%20treatment&author=JD%20Lindh&author=L%20Holm&author=ML%20Dahl&author=L%20Alfredsson&author=A%20Rane&volume=25&publication_year=2008&pages=151-159&pmid=17514429&doi=10.1007/s11239-007-0048-2&)

67. Limdi NA, McGwin G, Goldstein JA, Beasley TM, Arnett DK, Adler BK, et al. Influence of CYP2C9 and VKORC1 1173C/T genotype on the risk of hemorrhagic complications in African-American and European-American patients on warfarin. Clin Pharmacol Ther. 2008; 83:312–321.  [DOI](https://doi.org/10.1038/sj.clpt.6100290) | [PMC free article](/articles/PMC2683398/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17653141/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Influence%20of%20CYP2C9%20and%20VKORC1%201173C/T%20genotype%20on%20the%20risk%20of%20hemorrhagic%20complications%20in%20African-American%20and%20European-American%20patients%20on%20warfarin&author=NA%20Limdi&author=G%20McGwin&author=JA%20Goldstein&author=TM%20Beasley&author=DK%20Arnett&volume=83&publication_year=2008&pages=312-321&pmid=17653141&doi=10.1038/sj.clpt.6100290&)

68. Horne BD, Lenzini PA, Wadelius M, Jorgensen AL, Kimmel SE, Ridker PM, et al. Pharmacogenetic Warfarin Dose Refinements Remain Significantly Influenced by Genetic Factors after One Week of Therapy. Thromb Haemost. 2012; 107(2): 232–240. 10.1160/TH11-06-0388  [DOI](https://doi.org/10.1160/TH11-06-0388) | [PMC free article](/articles/PMC3292349/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22186998/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Thromb%20Haemost&title=Pharmacogenetic%20Warfarin%20Dose%20Refinements%20Remain%20Significantly%20Influenced%20by%20Genetic%20Factors%20after%20One%20Week%20of%20Therapy&author=BD%20Horne&author=PA%20Lenzini&author=M%20Wadelius&author=AL%20Jorgensen&author=SE%20Kimmel&volume=107&issue=2&publication_year=2012&pages=232-240&pmid=22186998&doi=10.1160/TH11-06-0388&)
