# Genetic variant in folate homeostasis is associated with lower warfarin dose in African Americans

## Metadata
**Authors:** Roxana Daneshjou, Eric R Gamazon, Ben Burkley, Larisa H Cavallari, Julie A Johnson, Teri E Klein, Nita Limdi, Sara Hillenmeyer, Bethany Percha, Konrad J Karczewski, Taimour Langaee, Shitalben R Patel, Carlos D Bustamante, Russ B Altman, Minoli A Perera
**Journal:** Blood
**Date:** 2014 Jul 30
**DOI:** [10.1182/blood-2014-04-568436](https://doi.org/10.1182/blood-2014-04-568436)
**PMID:** 25079360
**PMCID:** PMC4183989
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4183989/

## Abstract

A population-specific genetic variant involved in folate homeostasis is associated with lower warfarin dose in African Americans.

## Introduction

With >33 million prescriptions in 2011, warfarin is the most commonly used anticoagulant for preventing thromboembolic events.^1^ Warfarin has a 20-fold interindividual dose variability and a narrow therapeutic index, and it is responsible for one-third of adverse drug event hospitalizations in older Americans.^2^ Alternatives to warfarin, such as direct thrombin inhibitors and factor Xa inhibitors, are now available. However, these are more expensive, irreversible, and may cause a higher rate of acute coronary events compared with warfarin.^3,4^ Thus, warfarin remains a mainstay of anticoagulant therapy, and better methods of dosing warfarin will lead to fewer adverse events.

In addition to clinical modifiers, genetic polymorphisms are also known to affect an individual’s stable therapeutic warfarin dose.^5^ These polymorphisms modulate warfarin’s pharmocodynamic or pharmacokinetic pathways and include variants in genes coding for the warfarin’s target, the vitamin K epoxide reductase complex, subunit 1 (*VKORC1*), and warfarin’s main metabolizer, cytochrome p450 family 2, subfamily C, polypeptide 9 (*CYP2C9*).^6,7^ The Food and Drug Administration approved the addition of dosing recommendations based on genotype to warfarin labeling.^8^ In 2012, Anderson et al demonstrated in a clinical effectiveness trial that using pharmacogenetics to predict therapeutic warfarin dose is superior than the standard of care in establishing and maintaining therapeutic anticoagulation, as measured by international normalized ratio, and may reduce the risk for adverse events.^8^

In 2013, as part of the Clarification of Optimal Anticoagulation through Genetics (COAG) trial, Kimmel et al conducted a double-blind randomized controlled trial that showed no difference in time in therapeutic range over 4 weeks when dosing warfarin with a pharmacogenetic algorithm vs a clinical algorithm.^9,10^ The implications of these findings are unclear because clinical algorithms are not the current standard of care.^10^ In parallel, Pirmohamed et al conducted a single-blind randomized controlled trial and showed the superiority of using a modified version of the International Warfarin Pharmacogenetics Consortium (IWPC) pharmacogenetic equation over the current standard of care in the United Kingdom and Sweden.^11^ Although both of these studies focused on European populations, Kimmel et al included 275 African American subjects and concluded that the tested variants performed poorly, leading to a significant overprediction of dose.^9^ Importantly, *VKORC1* −1639 G/A, *CYP2C9**2, and *CYP2C9**3, the only variants studied by Kimmel et al, are less prevalent in African Americans, who have some of the greatest known dose variability.^12-14^ Thus, this study highlighted the need for population-specific data in warfarin pharmacogenetics. Variants relevant to warfarin dose with greater frequency in African Americans have been discovered through candidate gene studies, targeted sequencing of the warfarin pharmacodynamics and pharmacokinetic pathways, and a genome-wide association study.^15,16^ For instance, a variant in a *VKORC1* regulator, calumenin (*CALU*), leads to higher predicted warfarin doses in African Americans, whereas the *CYP2C9**8 variant leads to lower doses.^17-19^ Additionally, a recent genome-wide association study showed a variant in the CYP2C cluster associates with lower warfarin dose in African Americans.^15^ However, these genetic factors and all the known clinical variables explain only 29% to 41% of the variability in warfarin dose in this population compared with >50% in European descent populations.^5,19^

Therefore, identifying additional genetic factors that affect warfarin dose in populations of African descent is critical for extending the benefits of genotype guidance to these populations. To this end, we used an extreme phenotype strategy paired with exome sequencing to find novel variants that influence warfarin dosing among African Americans.^20^ We report a novel association between a genetic variant affecting folate homeostasis and lower warfarin dose requirement among African Americans.

## Methods

### Discovery cohort

#### Selection.

The discovery cohort was selected using the dose distribution of 360 African Americans on warfarin and selecting individuals who were at least ∼0.25 standard deviations away from the mean: 44.5 mg/week (supplemental Figure 1).^15^ The cutoffs were 35 mg/week for the low-dose cohort and 49 mg/week for the high-dose cohort. One hundred six individuals met these criteria. Samples were collected from sites at the University of Chicago, University of Illinois at Chicago, and University of Florida. All patients consented using an Institutional Review Board–approved process at each respective collection site in accordance with the Declaration of Helsinki.

#### Sequencing, mapping, and variant calling.

Exome capture and sequencing were performed using 2.5 to 3 µg of DNA (concentration ≥ 200 ng/µL) with Illumina’s TruSeq v1 and sequenced using the Illumina GAIIx platform. Reads were mapped using BWA 0.6.1 (with the -q 20 flag). Duplicate reads were marked with Picard 1.56, and local realignment around potential indels (including the Single Nucleotide Polymorphysm database [dbSNP] 132) was performed with the Genome Analysis Toolkit (GATK; 1.3-21-gcb284ee). GATK was used to recalibrate quality scores with dbSNP 132 as a training set and to calculate depth of coverage.

#### Variant calling.

SNPs were called with GATK's UnifiedGenotyper only at sites covered by the exome sequencing platform and the 50 bp surrounding these sites. Quality scores were calibrated with Variant Quality Score Recalibration using the default settings. VCFtools was used to calculate the missing data rate, heterozygosity, and depth per site, which was redundant with depth from mapping. Indels were called using GATK’s Unified Genotyper, and quality scores were calibrated with Variant Quality Score Recalibration at default settings. With quality score filtering, we called ∼500 000 SNPs.

#### Quality control.

We submitted 1 sample from a subject of European descent who had previously undergone exome sequencing at 30× coverage by the same Illumina platform.^21^ Moreover, 86 samples had been previously genotyped on the Illumina 610 Quad BeadChip (Illumina, San Diego, CA). Concordance between the previously exome sequenced sample of European descent and the high-quality exome calls was 99.8%.

As an additional quality control step, we confirmed that identity by descent (IBD) analysis showed that 85 of 86 exome-sequenced samples shared 100% IBD with their genotyped data. One sample had a concordance of 79%; this sample had the lowest coverage (mean coverage ∼18×) and was removed from subsequent analyses. The mean depth of coverage of the remaining samples was about 68×. Finally, IBD revealed that 3 samples were unexpectedly related; this relationship was confirmed by consulting the clinical charts. For the final analysis, we removed 2 of these samples so that the samples used for analysis were completely unrelated. Samples had previously been shown to meet the criteria for African-American descent.^15^

In addition to quality score filtering, we filtered out SNPs with a minor allele frequency <0.05 and loci missing calls in >20 individuals (∼20% missingness). We only tested a total of 155 186 SNPs.

#### Clinical data.

Demographic and clinical data were collected as previously described.^14,15^ Collected clinical covariates included gender, age, height, weight, concurrent use of aspirin, amiodarone, enzyme inducers (rifampin, phenytoin, and carbamazepine), indication for warfarin, and stable dose of warfarin. One individual was missing height, which was estimated by using the sex-matched mean. One individual was missing dose indication, and this was set to the default of no venous thromboembolism (VTE).

#### Analysis.

A total of 103 unrelated samples met the above quality control requirements. Genotyped covariates were *VKORC1* −1639G>A (rs9923231), *CYP2C9**2 (rs1799853), *CYP2C9**3 (rs1057910), *CYP2C9**5 (rs28371686), *CYP2C9**8 (rs7900194), and *CYP2C9**11 (rs28371685). The *VKORC1* −1639G>A genotype was modeled as an additive covariate, whereas the *CYP2C9* haplotype was modeled as a composite additive covariate.

To account for population stratification, we used principal component analysis, which was calculated using the Nonlinear Iterative Partial Least Squares algorithm in the Principal Component Analysis module python package (version 1.1.02).^22^

We selected which clinical covariates and previously identified genetic covariates to use in our model based on univariate association with log-transformed warfarin dose (supplemental Table 1, available on the *Blood* Web site) using the linear model (*lm*) function in the R statistical package (version 2.15.3). We modeled log-transformed dose on these significant covariates and on every SNP that passed the aforementioned quality control requirements using linear model function in the R statistical package (version 2.15.3). *P* values were adjusted using Bonferroni correction, with a significance cutoff of 3.22 × 10^−7^ based on the 155 186 SNPs tested. Linkage disequilibrium (LD) between top SNPs was evaluated using PLINK (version 1.07).^23^

Previous extreme-phenotype sequencing studies have performed rare variant burden tests across genes to identify genes of interest.^20^ We used this same test, the RTV1 test, to perform a gene-based evaluation of our exomes.^24^ We mapped variants within RefSeq exon boundaries and used an allele frequency cutoff frequency of <0.125. For 42 284 evaluated reference sequences, we used a Bonferroni cutoff of 1.18 × 10^−6^.

Separately, we also specifically evaluated the presence of rare, damaging mutations in known pharmacokinetic and pharmacodynamic genes involved in warfarin dosing: *CYP2C9* and *VKORC1*. Known *CYP2C9* variants had been genotyped previously.

### Replication cohort

The replication cohort consisted of 479 African Americans on a stable dose of warfarin enrolled from the University of Chicago, University of Illinois at Chicago, University of Florida, and University of Alabama at Birmingham; these samples were independent from the original 360 samples used for the extreme-phenotype study design in the discovery cohort. All patients provided written, informed consent for study participation. In the replication cohort, samples were not selected for an extreme dose distribution (supplemental Figure 2).

#### Genotyping.

Samples from University of Chicago, University of Illinois, and the University of Florida were genotyped via pyrosequencing, as previously described.^25^ A list of primers can be found in the supplemental Table 2.^7^ Genotypes for rs7856096 in the University of Alabama at Birmingham cohort were obtained from Genome-Wide Association Study Illumina Human 1M Duo 3.0 genotype data (directly interrogated) as previously described.^15^

#### Quality control.

Because race was self-reported, we tested ancestry as described below. Samples from the University of Alabama at Birmingham were assessed for ancestry using principal component analysis, as previously described.^15^ Samples from the University of Chicago and University of Illinois were assessed for ancestry using ancestry informative markers using previously described methods.^26,27^ In the replication, 83 subjects were found to have a significantly lower percentage of African ancestry based on previously described cutoffs and were removed prior to analysis.^26,27^

#### Clinical data.

Available clinical data from the replication data set included gender, age, height, weight, amiodarone use, and VTE status. Missing height and weight were imputed using the gender-matched mean.

#### Analysis.

A total of 372 samples met the quality control and missingness criteria. Genetic covariates included *VKORC1* −1639G>A coded as an additive genetic variant and *CYP2C9* haplotype data as a composite score. The University of Alabama at Birmingham data included a *CYP2C9* composite score based on variants in *CYP2C9**2, *CYP2C9**3, *CYP2C9**5, *CYP2C9**6, and *CYP2C9**11. The University of Chicago and University of Illinois *CYP2C9* composite scores included information about the presence of *CYP2C9**2, *CYP2C9**3, *CYP2C9**6, *CYP2C9**8, and *CYP2C9**11. For the replication model, we selected the clinical covariates that were used in the discovery model that were available. There were no data available on aspirin use in the replication cohort, and SNP array data were not available on the entire data set for principal component analysis. Thus, the reduced model used on the replication cohort incorporated age, weight, amiodarone, VTE status, *VKORC1* −1639G>A, and *CYP2C9* composite score. Log2-transformed dose was modeled using these covariates and the top hit using the linear model function in the R statistical package (version 2.15.3). Because we tested only the 1 SNP that was significant in our discovery cohort, we did not correct for multiple hypothesis testing.

We ran this same reduced model on the discovery cohort to ensure that the reduced model did not affect the significance of our top hit.

### Information added to the IWPC model

We then evaluated the contribution of the SNP to the IWPC pharmacogenetic algorithm. We calculated the predicted warfarin dose of each individual in both the discovery and replication cohorts using the IWPC dose equation.^5^ We treated *CYP2C9**11 and *CYP2C9**8 as equivalent to *CYP2C9**2 and *CYP2C9**5 as equivalent to *CYP2C9**3 and *CYP2C9**6 as equivalent to *CYP2C9* unknown.^5,15,16^ The residual dose unexplained by the equation was defined as the predicted dose subtracted from the actual dose. The discovery cohort included information on rifampin, carbamazepine, and phenytoin use.

For the replication cohort, participants from the University of Alabama at Birmingham were not on enzyme inducers, and data on carbamazepine use were available from the University of Chicago and University of Illinois at Chicago replication cohorts.

The novel SNP was analyzed as an additive variant, and we computed the additional information contributed to this residual dose using the *lm* function in R (version 2.15.3).

### Functional analysis

We tested whether rs7856096 overlaps experimentally predicted functional elements from the Encyclopedia of DNA Elements (ENCODE) and the National Institutes of Health Epigenomics Roadmap.^28,29^ We downloaded chromatin state segmentation data (generated by ENCODE/Broad using ChIP-seq data) derived from 9 human cell types from the University of California, Santa Cruz Genome Browser.^30^

We also analyzed lymphoblastoid cell line (LCL) microarray gene expression data in the 3 HapMap African populations: the Maasai from Kenyawa, Kenya (MKK, N = 135), the Luhya in Webuye, Kenya (LWK; N = 83), and the Yoruba in Ibadan, Nigeria (YRI; N = 108).^31^ In particular, we tested, assuming an additive model, the effect of each risk allele on the expression of *FPGS* using the *lm* function in R (version 2.15.4), with or without population indicator as a covariate.

## Results

### Discovery cohort: rs7856096 significantly associated with dose in the extreme phenotype study

Our discovery cohort consisted of 103 unrelated African Americans on stable doses of warfarin. These subjects were selected for an extreme phenotype distribution. Population demographics and clinical data were collected as previously described and are summarized in [Table 1](#T1).^14^ The allele frequency of *VKORC1* −1639G>A in this cohort was 0.117. For *CYP2C9* composite score, 15.5% of individuals had only 1 CYP2C9 and 6.8% had 2 *CYP2C9* variants. Variables that significantly explained dose in a univariate model were age, weight, aspirin use, amiodarone use, warfarin indication, VTE status, *VKORC1* −1639G>A, and composite *CYP2C9* status (supplemental Table 1). The top 10 SNPs and *P* values from a linear model of log2-transformed warfarin dose are shown in [Table 2](#T2). The coefficients for the covariates in this linear model can be found in supplemental Table 3. Only 1 SNP, rs7856096, was significantly associated with warfarin dose (uncorrected *P* value: 1.82 × 10^−8^; [Figure 1](#F1)). rs7856096 is upstream of exon 2 of the folylpolyglutamate synthase gene (*FPGS*), which encodes a mitochondrial enzyme involved in folate sequestration. This SNP was in LD with the second most significant SNP, rs11554716, with an *R*^2^ value of 0.85. An LD plot using the HapMap Phase III ASW data can be found in refs. ^32^ and ^33^. [Figure 2](#F2) shows a QQ-plot of expected vs observed *P* values. The risk allele, which was the ancestral allele, is most prevalent on the African continent ([Figure 3](#F3)) and has a frequency of 0.204 in our discovery population.^32,34,35^

### Table 1.

| Covariate | Discovery mean/% (all) (n = 103) | Discovery mean/% (>49 mg/week) (n = 58) | Discovery mean (<35 mg/week) (n = 45) | Replication cohort mean/% |
| --- | --- | --- | --- | --- |
| Dose | 49.16 mg/wk | 66.38 mg/wk | 26.97 mg/wk | 43.80 mg/wk |
| Gender | 68.00% female | 67.2% female | 68.89% female | 61.39% female |
| Age | 56.75 y | 50.38 y | 64.96 y | 57.58 y |
| Height | 168.20 cm | 168.83 cm | 167.40 cm | 169.60 cm |
| Weight | 92.37 kg | 96.43 kg | 87.13 kg | 92.05 kg |
| Aspirin | 26.21% | 12.07% | 44.44% | — |
| Amiodarone | 0.97% | 0% | 2.22% | 4.30% |
| Phenytoin | 0.97% | 0% | 2.22% | — |
| Carbamazepine | 1.94% | 3.45% | 0% | — |
| Rifampin | 2.91% | 3.45% | 2.22% | — |
| Indication (VTE) | 64.71% | 82.75% | 40.91% | 47.04% |

Table 1 Caption: Clinical demographics of the discovery cohort and replication cohort

### Table 2.

| SNP | Chromosome:position | Gene | P value (uncorrected) |
| --- | --- | --- | --- |
| rs7856096 | chr9:130566539 | FPGS | 1.82 × 10−8 |
| rs11554716 | chr9:130565326 | FPGS | 2.93 × 10−7 |
| rs28694704 | chr17:18098253 | ALKBH5 | 6.56 × 10−6 |
| rs3750371 | chr11:50257362 | LOC441601 | 9.87 × 10−6 |
| rs2991344 | chr1:41978566 | HIVEP3 | 1.09 × 10−5 |
| rs2483689 | chr1:41978825 | HIVEP3 | 1.09 × 10−5 |
| rs82825 | chr3:52471942 | SEMA3G | 2.25 × 10−5 |
| rs7257009 | chr19:43866906 | CD177 | 2.47 × 10−5 |
| rs7406 | chr1:225589473 | LBR | 2.86 × 10−5 |
| rs1062098 | chr19:44930738 | ZNF229 | 2.87 × 10−5 |

Table 2 Caption: Top 10 SNPs in discovery cohort analysis

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/75af/4183989/f33852082902/2298f1.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4183989_2298f1.jpg)

Manhattan plot of discovery cohort. Bonferroni cutoff of 3.22 × 10−7 for significance. The 2 top SNPs are in LD and are in FPGS.

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/75af/4183989/3e724f6c3097/2298f2.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4183989_2298f2.jpg)

QQ-plot of expected and observed P values.

### Figure 3.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/75af/4183989/52d3e99fb330/2298f3.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4183989_2298f3.jpg)

Distribution of associated risk allele across diverse populations. The risk allele, which is the ancestral allele, is most prevalent on the African continent. Map generated using Human Genome Diversity Project (HGDP) data through the HGDP Selection Browser.34,35

### Gene-based analysis shows no novel associations

No single gene reached statistical significance after Bonferroni correction. The list of the top 10 genes can be found in supplemental Table 4.

No novel, previously undescribed nonsynonymous exonic variants were observed in *VKORC1* in the discovery cohort. The only nonsynonymous exonic variant observed was rs7200749, with an allele frequency of 0.157. This variant had been previously reported as affecting warfarin resistance in a small cohort of South African descent; however, in our model, the variant had a *P* value of .311.^36^ For *CYP2C9*, we found no novel nonsynonymous variants. However, previously described coding variation was observed and was also genotyped, and a summary can be found in supplemental Table 5.

### Replication cohort: rs7856096 replicates in an independent cohort of 372 African Americans on warfarin

We replicated our finding in a cohort of 372 African Americans with known stable dose but who were not selected for extreme dose phenotype (supplemental Figure 2). Key demographics of this group can be found in [Table 1](#T1). Available clinical covariates included gender, age, height, weight, amiodarone use (0 or 1), and indication (VTE or not). We used a linear model of log2-transformed dose and available covariates from the discovery model to test whether rs7856096 was significant. Genome-wide SNP array data were only available for the University of Alabama at Birmingham samples, and in this subset of the data, the principal components did not significantly correlate with dose. For the genetic covariates used, the allele frequency of *VKORC1* −1639G>A allele was 0.0941. For the CYP2C9 composite score, 14.5% of individuals had 1 variant and 2.1% had 2 variants. The tested SNP, rs7856096, had an allele frequency of 0.205. In this replication cohort of 372 individuals, the association of rs7856096 with warfarin dose remained significant (*P* = .0458). The coefficients for the covariates in this linear model can be found in supplemental Table 6. Because our replication model was missing 2 covariates from the discovery model (principal component 1 and aspirin use), we tested the reduced model on the discovery cohort (supplemental Table 7) and confirmed that rs7856096 remained significant in the discovery cohort using the reduced model.

### rs7856096 predicts warfarin sensitivity in the IWPC dose equation

We tested the degree to which the new SNP, rs7856096, could improve the prediction of warfarin dose. In particular, we modeled the added value of this SNP to the IWPC pharmacogenetic dose equation in a combined cohort, consisting of both our discovery and replication cohorts totaling 476 individuals. The distribution of residual dose in this combined cohort is shown in supplemental Figure 4. Although the discovery cohort had been selected for extreme dose, the residual or unexplained dose was normally distributed. In the combined cohort, each minor allele of rs7856096 contributed to the −5.81 mg/week change in the predicted dose (*P* = 3.93 × 10^−5^). The percentage variance in dose explained by the IWPC dose equation alone was 30.0%, as measured by adjusted *R*^2^, and rs7856096 explained an additional 3.30 percentage points of the dose variability.

### rs7856096 is a cis-acting eQTL for FPGS in LCLs

ENCODE data show that rs7856096 overlaps active enhancers in multiple cell types, including a lymphoblastoid cell line (GM12878) (supplemental Figure 5).^28^ This raises the possibility that allelic variants may affect enhancer activity and alter the expression of the target gene. We therefore evaluated the SNP for its association with the expression of *FPGS* in transformed lymphoblastoid cell lines derived from 3 HapMap African populations (MKK, LWK, and YRI). Transformed lymphoblastoid cell lines express FPGS to the same level as liver tissue (supplemental Figure 6).^37^ The SNP was significantly associated with the expression of the gene (*P* = .02; [Figure 4](#F4)). When we used the population indicator (which was significantly associated with FPGS expression at *P* = .007 in a univariate association analysis) as a covariate, the SNP remained significant (*P* = .01). The SNP had concordant direction of effect in all 3 African populations.

### Figure 4.

![Figure 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/75af/4183989/cc7da3a7387b/2298f4.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4183989_2298f4.jpg)

Effect of SNP on FPGS expression in LCLs across 3 African populations (MKK, LWK, and YRI). Effect on expression was evaluated using a linear model with population and rs7856096 (modeled additively) as covariates.

## Discussion

Here, we describe a population-specific variant in a folate homeostasis gene that is associated with lower warfarin dose in African Americans. This variant not only adds to the list of variants influencing warfarin dose in African Americans but is also the first genetic evidence linking folate pathways to warfarin dosing.

Warfarin’s interindividual dose variability and narrow therapeutic index pose a challenge for accurate dosing. Dosing algorithms using currently available pharmacogenetic and clinical factors are 1 way to try and predict an individual’s dose.^5^ The COAG trial found no difference in percentage of time in the international normalized ratio when comparing patients dosed using a clinical algorithm (not currently standard of care) and patients dosed using a pharmcogenetic algorithm. However, the trial was underpowered to detect differences in adverse events, such as bleeding or thromboembolism.^10^ Additionally, another study found that pharmacogenetic algorithms can improve dose prediction compared with the current standard of care in the United Kingdom and Sweden.^11^ The COAG trial included 27% African Americans and concluded that using the genetic variants *VKORC1* −1639G>A, *CYP2C9**2, and *CYP2C9**3 led to dose overprediction in this population compared with using clinical variables alone.^9^ However, these variants occur at a significantly lower frequency in African Americans, and other population-specific variants may improve prediction and improve performance in this population.^38,39^ In our study, we conditioned our model on the *CYP2C9* variants that are more prevalent in African Americans, such as CYP2C9*5, *8, and *11, and discovered a novel variant affecting warfarin dose in this population.^38^ The resulting variant, rs7856096, is in a gene (and therefore pathway) not previously associated with the pharmacogenetics of warfarin.

Extreme phenotype design can help identify genetic factors affecting a phenotype when the discovery cohort is small.^20,40,41^ Because an extreme-phenotype approach coupled with sequencing has never been applied to the warfarin phenotype, our study was uniquely designed to find signals not detected in previous genome-wide association studies. After conditioning on known clinical factors and genetic factors (including *CYP2C9* variants specific to African Americans), our approach yielded rs7856096, and this SNP was genome-wide significance after Bonferroni correction. This SNP is on chromosome 9, in the *FPGS* gene, which encodes a mitochondrial enzyme involved in folate homeostasis. rs7856096 is at the 5′ end upstream of exon 2 of *FPGS*. Within this data set, rs7856096 is in strong LD (D′ = 0.93) with another intronic SNP, rs11554716, but is not in LD with any other sequenced FPGS variants, leading to a sparse peak in the Manhattan plot. This variant was discovered because of its proximity to the coding region, and we found no coding variants in LD with it. The ancestral allele, which is the risk allele, is most prevalent in the African subcontinent, and would not likely have been observed in any of the studies in individuals of European descent.^32^ Additionally, the use of an extreme phenotype design allowed the identification of an association for which the signal may have been masked by the middle of the dose distribution. However, we did replicate this SNP in an independent cohort in which doses were not selected for an extreme phenotype.

In both the discovery and replication cohort analyses, the coefficient for the association between the minor rs7856096 allele and warfarin dose is negative, indicating that the presence of this variant is correlated with lower dose. We assessed the clinical value by calculating the information added to the IWPC dose equation using a combination of the discovery and replication cohorts. As mentioned in the methods, we included African American-specific *CYP2C9* variants in this calculation, such as *CYP2C9**5, *6, *8, and *11. We found in the combined cohort each risk allele corresponded to a dose reduction of 5.81 mg/week (0.83 mg/day). To truly assess the clinical impact of this variant and update the IWPC dose equation, additional studies in independent larger cohorts are required. However, this first clinical analysis suggests that individuals homozygous for the risk allele could require a clinically significant reduction of 11.62 mg/week (1.66 mg/day) from the IWPC predicted dose. In our combined cohort, 3.3% of individuals were homozygous at this locus. For comparison, currently, the Food and Drug Administration label on warfarin recommends a reduction of 2 to 4 mg/day for wild-type VKORC1 with CYP2C9 *1/*3, *2/*2, or *2/*3 and no reduction for CYP2C9 *1/*2.^42^

Preliminary evidence from 3 HapMap African populations suggests that the action of this SNP is through the regulation of *FPGS* expression. Changes in *FPGS* expression are known to affect folate sequestration and utilization in the folate pathway.^43,44^ This analysis was based on transformed lymphoblastoid cell lines because African liver samples were not available; however, RNA-Seq data in European populations show that transformed lymphoblastoid cell lines express *FPGS* at the same level as liver tissue (supplemental Figure 6).^37^ *FPGS* is expressed and active in the liver, the primary site of warfarin metabolism (supplemental Figure 7).^45,46^ From a biological standpoint, the discovery of this variant is the first genetic evidence of a connection between the warfarin and folate pathways. Previously, folate had been linked to warfarin through drug-drug interaction studies—folate intake has been shown to affect warfarin clearance, and a small study demonstrated that increased folate intake affected warfarin metabolism rates.^47^ Additionally, folate has been shown to affect coagulation status, so this variant could affect the warfarin pharmacodynamic pathway as well.^48,49^ In this study, we had no information on folate intake; however, one could postulate that, because this variant affects folate sequestration, a possible genetic-environmental interaction could occur. Studies looking at how this variant and folate interact in the setting of warfarin could shed light on any possible synergistic effects.

In conclusion, we present the first report of a SNP involved in folate homeostasis, which significantly influences warfarin dose in African Americans, a population whose dose variability has been understudied.

## Acknowledgments

Work by L.H.C. was supported by American Heart Association Midwest Affiliate Spring 2010 grant-in-aid 10GRNT3750024 and a University of Illinois at Chicago College of Pharmacy Hans Vahlteich research award. Work by N.L. was supported in part by National Heart Lung and Blood Institute grant RO1HL092173 and National Centre for Advancing Translational Sciences grant UL1 TR000165. Work by J.A.J. and T.L. was supported in part by National Institutes of Health, National Institute of General Medical Sciences grant U01 GM074492. Work by M.A.P. was supported in part by National Heart Lung and Blood Institute grant K23 HL089808 and American Heart Association grant 10CRP3740026. Work by R.B.A. and T.E.K. is supported in part by National Institutes of Health, National Institute of General Medical Sciences grant GM61374 and gifts from Microsoft and Lightspeed Ventures. Work by R.D. was supported in part by the Howard Hughes Medical Institute Medical Fellows Program, the Stanford Medical Scientist Training Program, and the Stanford Genetics Training Program. Illumina sequencing services were performed by the Stanford Center for Genomics and Personalized Medicine.

## Footnotes

## Authorship

Contribution: R.D., E.R.G., L.H.C., J.A.J., T.E.K., S.H., B.P., K.J.K., C.D.B., R.B.A., and M.A.P. helped design the research; E.R.G., B.B., L.H.C., J.A.J., N.L., T.L., S.R.P., and M.A.P. helped generate the data; R.D., E.R.G., S.H., B.P., K.J.K. helped analyze the data; R.D. and E.R.G. drafted the manuscript; and all authors edited the manuscript.

Conflict-of-interest disclosure: L.H.C. is a coinvestigator for US Utility Patent application no. 12/572 908, titled “CYP2C9*8 alleles correlate with decreased warfarin metabolism and increased warfarin sensitivity”: published May 27, 2010; Pub. No. US 2010/0130599. R.B.A. is a founder of and consultant for Personalis. C.D.B. is on the advisory board of a project at 23andMe and on the scientific advisory boards of Personalis, Inc.; InVitae; Etalon, Inc.; and [Ancestry.com](http://Ancestry.com). T.E.K. is a consultant for Personalis. The remaining authors declare no competing financial interests.

The current affiliation for L.H.C. is Department of Pharmacotherapy and Translational Research, University of Florida, Gainesville, FL.

Correspondence: Russ Altman, 443 Via Ortega, Room 209, MC: 4245, Stanford, CA 94305-4125; e-mail: russ.altman@stanford.edu; or Minoli Perera, 900 E 57th St, Chicago, IL 60637; e-mail: mperera@bsd.uchicago.edu.

## References

1. IMS Institute for Healthcare Informatics. The Use of Medicines in the United States. Review of 2011; April 2012. Report by the IMS Institute for Healthcare Informatics. Available at: www.imshealth.com/ims/Global/Content/Insights/IMS%20Institute%20for%20Healthcare%20Informatics/IHII_Medicines_in_U.S_Report_2011.pdf.  [www.imshealth.com/ims/Global/Content/Insights/IMS%20Institute%20for%20Healthcare%20Informatics/IHII_Medicines_in_U.S_Report_2011.pdf](http://www.imshealth.com/ims/Global/Content/Insights/IMS%20Institute%20for%20Healthcare%20Informatics/IHII_Medicines_in_U.S_Report_2011.pdf) | [Google Scholar](https://scholar.google.com/scholar_lookup?IMS%20Institute%20for%20Healthcare%20Informatics.%20The%20Use%20of%20Medicines%20in%20the%20United%20States.%20Review%20of%202011;%20April%202012.%20Report%20by%20the%20IMS%20Institute%20for%20Healthcare%20Informatics.%20Available%20at:%20www.imshealth.com/ims/Global/Content/Insights/IMS%20Institute%20for%20Healthcare%20Informatics/IHII_Medicines_in_U.S_Report_2011.pdf.)

2. Budnitz DS, Lovegrove MC, Shehab N, Richards CL. Emergency hospitalizations for adverse drug events in older Americans. N Engl J Med. 2011;365(21):2002–2012. doi: 10.1056/NEJMsa1103053.  [DOI](https://doi.org/10.1056/NEJMsa1103053) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22111719/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Emergency%20hospitalizations%20for%20adverse%20drug%20events%20in%20older%20Americans.&author=DS%20Budnitz&author=MC%20Lovegrove&author=N%20Shehab&author=CL%20Richards&volume=365&issue=21&publication_year=2011&pages=2002-2012&pmid=22111719&doi=10.1056/NEJMsa1103053&)

3. Bauer KA. Recent progress in anticoagulant therapy: oral direct inhibitors of thrombin and factor Xa. J Thromb Haemost. 2011;9(Suppl 1):12–19. doi: 10.1111/j.1538-7836.2011.04321.x.  [DOI](https://doi.org/10.1111/j.1538-7836.2011.04321.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21781237/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Thromb%20Haemost&title=Recent%20progress%20in%20anticoagulant%20therapy:%20oral%20direct%20inhibitors%20of%20thrombin%20and%20factor%20Xa.&author=KA%20Bauer&volume=9&issue=Suppl%201&publication_year=2011&pages=12-19&pmid=21781237&doi=10.1111/j.1538-7836.2011.04321.x&)

4. Uchino K, Hernandez AV. Dabigatran association with higher risk of acute coronary events: meta-analysis of noninferiority randomized controlled trials. Arch Intern Med. 2012;172(5):397–402. doi: 10.1001/archinternmed.2011.1666.  [DOI](https://doi.org/10.1001/archinternmed.2011.1666) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22231617/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arch%20Intern%20Med&title=Dabigatran%20association%20with%20higher%20risk%20of%20acute%20coronary%20events:%20meta-analysis%20of%20noninferiority%20randomized%20controlled%20trials.&author=K%20Uchino&author=AV%20Hernandez&volume=172&issue=5&publication_year=2012&pages=397-402&pmid=22231617&doi=10.1001/archinternmed.2011.1666&)

5. Klein TE, Altman RB, Eriksson N, et al. International Warfarin Pharmacogenetics Consortium. Estimation of the warfarin dose with clinical and pharmacogenetic data. N Engl J Med. 2009;360(8):753–764. doi: 10.1056/NEJMoa0809329.  [DOI](https://doi.org/10.1056/NEJMoa0809329) | [PMC free article](/articles/PMC2722908/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19228618/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Estimation%20of%20the%20warfarin%20dose%20with%20clinical%20and%20pharmacogenetic%20data.&author=TE%20Klein&author=RB%20Altman&author=N%20Eriksson&volume=360&issue=8&publication_year=2009&pages=753-764&pmid=19228618&doi=10.1056/NEJMoa0809329&)

6. D’Andrea G, D’Ambrosio RL, Di Perna P, et al. A polymorphism in the VKORC1 gene is associated with an interindividual variability in the dose-anticoagulant effect of warfarin. Blood. 2005;105(2):645–649. doi: 10.1182/blood-2004-06-2111.  [DOI](https://doi.org/10.1182/blood-2004-06-2111) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15358623/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=A%20polymorphism%20in%20the%20VKORC1%20gene%20is%20associated%20with%20an%20interindividual%20variability%20in%20the%20dose-anticoagulant%20effect%20of%20warfarin.&author=G%20D%E2%80%99Andrea&author=RL%20D%E2%80%99Ambrosio&author=P%20Di%20Perna&volume=105&issue=2&publication_year=2005&pages=645-649&pmid=15358623&doi=10.1182/blood-2004-06-2111&)

7. Aquilante CL, Langaee TY, Lopez LM, et al. Influence of coagulation factor, vitamin K epoxide reductase complex subunit 1, and cytochrome P450 2C9 gene polymorphisms on warfarin dose requirements. Clin Pharmacol Ther. 2006;79(4):291–302. doi: 10.1016/j.clpt.2005.11.011.  [DOI](https://doi.org/10.1016/j.clpt.2005.11.011) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16580898/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Influence%20of%20coagulation%20factor,%20vitamin%20K%20epoxide%20reductase%20complex%20subunit%201,%20and%20cytochrome%20P450%202C9%20gene%20polymorphisms%20on%20warfarin%20dose%20requirements.&author=CL%20Aquilante&author=TY%20Langaee&author=LM%20Lopez&volume=79&issue=4&publication_year=2006&pages=291-302&pmid=16580898&doi=10.1016/j.clpt.2005.11.011&)

8. Anderson JL, Horne BD, Stevens SM, et al. A randomized and clinical effectiveness trial comparing two pharmacogenetic algorithms and standard care for individualizing warfarin dosing (CoumaGen-II). Circulation. 2012;125(16):1997–2005. doi: 10.1161/CIRCULATIONAHA.111.070920.  [DOI](https://doi.org/10.1161/CIRCULATIONAHA.111.070920) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22431865/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circulation&title=A%20randomized%20and%20clinical%20effectiveness%20trial%20comparing%20two%20pharmacogenetic%20algorithms%20and%20standard%20care%20for%20individualizing%20warfarin%20dosing%20(CoumaGen-II).&author=JL%20Anderson&author=BD%20Horne&author=SM%20Stevens&volume=125&issue=16&publication_year=2012&pages=1997-2005&pmid=22431865&doi=10.1161/CIRCULATIONAHA.111.070920&)

9. Kimmel SE, French B, Kasner SE, et al. COAG Investigators. A pharmacogenetic versus a clinical algorithm for warfarin dosing. N Engl J Med. 2013;369(24):2283–2293. doi: 10.1056/NEJMoa1310669.  [DOI](https://doi.org/10.1056/NEJMoa1310669) | [PMC free article](/articles/PMC3942158/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24251361/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=A%20pharmacogenetic%20versus%20a%20clinical%20algorithm%20for%20warfarin%20dosing.&author=SE%20Kimmel&author=B%20French&author=SE%20Kasner&volume=369&issue=24&publication_year=2013&pages=2283-2293&pmid=24251361&doi=10.1056/NEJMoa1310669&)

10. Zineh I, Pacanowski M, Woodcock J. Pharmacogenetics and coumarin dosing—recalibrating expectations. N Engl J Med. 2013;369(24):2273–2275. doi: 10.1056/NEJMp1314529.  [DOI](https://doi.org/10.1056/NEJMp1314529) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24328463/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Pharmacogenetics%20and%20coumarin%20dosing%E2%80%94recalibrating%20expectations.&author=I%20Zineh&author=M%20Pacanowski&author=J%20Woodcock&volume=369&issue=24&publication_year=2013&pages=2273-2275&pmid=24328463&doi=10.1056/NEJMp1314529&)

11. Pirmohamed M, Burnside G, Eriksson N, et al. EU-PACT Group. A randomized trial of genotype-guided dosing of warfarin. N Engl J Med. 2013;369(24):2294–2303. doi: 10.1056/NEJMoa1311386.  [DOI](https://doi.org/10.1056/NEJMoa1311386) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24251363/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=A%20randomized%20trial%20of%20genotype-guided%20dosing%20of%20warfarin.&author=M%20Pirmohamed&author=G%20Burnside&author=N%20Eriksson&volume=369&issue=24&publication_year=2013&pages=2294-2303&pmid=24251363&doi=10.1056/NEJMoa1311386&)

12. Schelleman H, Chen Z, Kealey C, et al. Warfarin response and vitamin K epoxide reductase complex 1 in African Americans and Caucasians. Clin Pharmacol Ther. 2007;81(5):742–747. doi: 10.1038/sj.clpt.6100144.  [DOI](https://doi.org/10.1038/sj.clpt.6100144) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17329985/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Warfarin%20response%20and%20vitamin%20K%20epoxide%20reductase%20complex%201%20in%20African%20Americans%20and%20Caucasians.&author=H%20Schelleman&author=Z%20Chen&author=C%20Kealey&volume=81&issue=5&publication_year=2007&pages=742-747&pmid=17329985&doi=10.1038/sj.clpt.6100144&)

13. Kealey C, Chen Z, Christie J, et al. Warfarin and cytochrome P450 2C9 genotype: possible ethnic variation in warfarin sensitivity. Pharmacogenomics. 2007;8(3):217–225. doi: 10.2217/14622416.8.3.217.  [DOI](https://doi.org/10.2217/14622416.8.3.217) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17324110/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Warfarin%20and%20cytochrome%20P450%202C9%20genotype:%20possible%20ethnic%20variation%20in%20warfarin%20sensitivity.&author=C%20Kealey&author=Z%20Chen&author=J%20Christie&volume=8&issue=3&publication_year=2007&pages=217-225&pmid=17324110&doi=10.2217/14622416.8.3.217&)

14. Limdi NA, Arnett DK, Goldstein JA, et al. Influence of CYP2C9 and VKORC1 on warfarin dose, anticoagulation attainment and maintenance among European-Americans and African-Americans. Pharmacogenomics. 2008;9(5):511–526. doi: 10.2217/14622416.9.5.511.  [DOI](https://doi.org/10.2217/14622416.9.5.511) | [PMC free article](/articles/PMC2757655/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18466099/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Influence%20of%20CYP2C9%20and%20VKORC1%20on%20warfarin%20dose,%20anticoagulation%20attainment%20and%20maintenance%20among%20European-Americans%20and%20African-Americans.&author=NA%20Limdi&author=DK%20Arnett&author=JA%20Goldstein&volume=9&issue=5&publication_year=2008&pages=511-526&pmid=18466099&doi=10.2217/14622416.9.5.511&)

15. Perera MA, Cavallari LH, Limdi NA, et al. Genetic variants associated with warfarin dose in African-American individuals: a genome-wide association study. Lancet. 2013;382(9894):790–796. doi: 10.1016/S0140-6736(13)60681-9.  [DOI](https://doi.org/10.1016/S0140-6736(13)60681-9) | [PMC free article](/articles/PMC3759580/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23755828/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Genetic%20variants%20associated%20with%20warfarin%20dose%20in%20African-American%20individuals:%20a%20genome-wide%20association%20study.&author=MA%20Perera&author=LH%20Cavallari&author=NA%20Limdi&volume=382&issue=9894&publication_year=2013&pages=790-796&pmid=23755828&doi=10.1016/S0140-6736(13)60681-9&)

16. Liu Y, Jeong H, Takahashi H, et al. Decreased warfarin clearance associated with the CYP2C9 R150H (*8) polymorphism. Clin Pharmacol Ther. 2012;91(4):660–665. doi: 10.1038/clpt.2011.269.  [DOI](https://doi.org/10.1038/clpt.2011.269) | [PMC free article](/articles/PMC3879795/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22378156/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Decreased%20warfarin%20clearance%20associated%20with%20the%20CYP2C9%20R150H%20(*8)%20polymorphism.&author=Y%20Liu&author=H%20Jeong&author=H%20Takahashi&volume=91&issue=4&publication_year=2012&pages=660-665&pmid=22378156&doi=10.1038/clpt.2011.269&)

17. Voora D, Koboldt DC, King CR, et al. A polymorphism in the VKORC1 regulator calumenin predicts higher warfarin dose requirements in African Americans. Clin Pharmacol Ther. 2010;87(4):445–451. doi: 10.1038/clpt.2009.291.  [DOI](https://doi.org/10.1038/clpt.2009.291) | [PMC free article](/articles/PMC2928561/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20200517/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=A%20polymorphism%20in%20the%20VKORC1%20regulator%20calumenin%20predicts%20higher%20warfarin%20dose%20requirements%20in%20African%20Americans.&author=D%20Voora&author=DC%20Koboldt&author=CR%20King&volume=87&issue=4&publication_year=2010&pages=445-451&pmid=20200517&doi=10.1038/clpt.2009.291&)

18. Scott SA, Jaremko M, Lubitz SA, Kornreich R, Halperin JL, Desnick RJ. CYP2C9*8 is prevalent among African-Americans: implications for pharmacogenetic dosing. Pharmacogenomics. 2009;10(8):1243–1255. doi: 10.2217/pgs.09.71.  [DOI](https://doi.org/10.2217/pgs.09.71) | [PMC free article](/articles/PMC2737687/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19663669/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=CYP2C9*8%20is%20prevalent%20among%20African-Americans:%20implications%20for%20pharmacogenetic%20dosing.&author=SA%20Scott&author=M%20Jaremko&author=SA%20Lubitz&author=R%20Kornreich&author=JL%20Halperin&volume=10&issue=8&publication_year=2009&pages=1243-1255&pmid=19663669&doi=10.2217/pgs.09.71&)

19. Ramirez AH, Shi Y, Schildcrout JS, et al. Predicting warfarin dosage in European-Americans and African-Americans using DNA samples linked to an electronic health record. Pharmacogenomics. 2012;13(4):407–418. doi: 10.2217/pgs.11.164.  [DOI](https://doi.org/10.2217/pgs.11.164) | [PMC free article](/articles/PMC3361510/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22329724/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Predicting%20warfarin%20dosage%20in%20European-Americans%20and%20African-Americans%20using%20DNA%20samples%20linked%20to%20an%20electronic%20health%20record.&author=AH%20Ramirez&author=Y%20Shi&author=JS%20Schildcrout&volume=13&issue=4&publication_year=2012&pages=407-418&pmid=22329724&doi=10.2217/pgs.11.164&)

20. Emond MJ, Louie T, Emerson J, et al. National Heart, Lung, and Blood Institute (NHLBI) GO Exome Sequencing Project; Lung GO. Exome sequencing of extreme phenotypes identifies DCTN4 as a modifier of chronic Pseudomonas aeruginosa infection in cystic fibrosis. Nat Genet. 2012;44(8):886–889. doi: 10.1038/ng.2344.  [DOI](https://doi.org/10.1038/ng.2344) | [PMC free article](/articles/PMC3702264/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22772370/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Exome%20sequencing%20of%20extreme%20phenotypes%20identifies%20DCTN4%20as%20a%20modifier%20of%20chronic%20Pseudomonas%20aeruginosa%20infection%20in%20cystic%20fibrosis.&author=MJ%20Emond&author=T%20Louie&author=J%20Emerson&volume=44&issue=8&publication_year=2012&pages=886-889&pmid=22772370&doi=10.1038/ng.2344&)

21. Clark MJ, Chen R, Lam HYK, et al. Performance comparison of exome DNA sequencing technologies. Nat Biotechnol. 2011;29(10):908–914. doi: 10.1038/nbt.1975.  [DOI](https://doi.org/10.1038/nbt.1975) | [PMC free article](/articles/PMC4127531/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21947028/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Biotechnol&title=Performance%20comparison%20of%20exome%20DNA%20sequencing%20technologies.&author=MJ%20Clark&author=R%20Chen&author=HYK%20Lam&volume=29&issue=10&publication_year=2011&pages=908-914&pmid=21947028&doi=10.1038/nbt.1975&)

22. Price AL, Patterson NJ, Plenge RM, Weinblatt ME, Shadick NA, Reich D. Principal components analysis corrects for stratification in genome-wide association studies. Nat Genet. 2006;38(8):904–909. doi: 10.1038/ng1847.  [DOI](https://doi.org/10.1038/ng1847) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16862161/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Principal%20components%20analysis%20corrects%20for%20stratification%20in%20genome-wide%20association%20studies.&author=AL%20Price&author=NJ%20Patterson&author=RM%20Plenge&author=ME%20Weinblatt&author=NA%20Shadick&volume=38&issue=8&publication_year=2006&pages=904-909&pmid=16862161&doi=10.1038/ng1847&)

23. Purcell S, Neale B, Todd-Brown K, et al. PLINK: a tool set for whole-genome association and population-based linkage analyses. Am J Hum Genet. 2007;81(3):559–575. doi: 10.1086/519795.  [DOI](https://doi.org/10.1086/519795) | [PMC free article](/articles/PMC1950838/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17701901/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=PLINK:%20a%20tool%20set%20for%20whole-genome%20association%20and%20population-based%20linkage%20analyses.&author=S%20Purcell&author=B%20Neale&author=K%20Todd-Brown&volume=81&issue=3&publication_year=2007&pages=559-575&pmid=17701901&doi=10.1086/519795&)

24. Morris AP, Zeggini E. An evaluation of statistical approaches to rare variant analysis in genetic association studies. Genet Epidemiol. 2010;34(2):188–193. doi: 10.1002/gepi.20450.  [DOI](https://doi.org/10.1002/gepi.20450) | [PMC free article](/articles/PMC2962811/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19810025/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genet%20Epidemiol&title=An%20evaluation%20of%20statistical%20approaches%20to%20rare%20variant%20analysis%20in%20genetic%20association%20studies.&author=AP%20Morris&author=E%20Zeggini&volume=34&issue=2&publication_year=2010&pages=188-193&pmid=19810025&doi=10.1002/gepi.20450&)

25. Dimeski G, Badrick T, Flatman R, Ormiston B. Pyrosequencing method for genotyping cytochrome P450 CYP2C8 and CYP2C9 enzymes. Clin Chem. 2004;50(12):2391–2392. doi: 10.1373/clinchem.2004.040071.  [DOI](https://doi.org/10.1373/clinchem.2004.040071) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15563491/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Chem&title=Pyrosequencing%20method%20for%20genotyping%20cytochrome%20P450%20CYP2C8%20and%20CYP2C9%20enzymes.&author=G%20Dimeski&author=T%20Badrick&author=R%20Flatman&author=B%20Ormiston&volume=50&issue=12&publication_year=2004&pages=2391-2392&pmid=15563491&doi=10.1373/clinchem.2004.040071&)

26. Falush D, Stephens M, Pritchard JK. Inference of population structure using multilocus genotype data: linked loci and correlated allele frequencies. Genetics. 2003;164(4):1567–1587. doi: 10.1093/genetics/164.4.1567.  [DOI](https://doi.org/10.1093/genetics/164.4.1567) | [PMC free article](/articles/PMC1462648/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12930761/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genetics&title=Inference%20of%20population%20structure%20using%20multilocus%20genotype%20data:%20linked%20loci%20and%20correlated%20allele%20frequencies.&author=D%20Falush&author=M%20Stephens&author=JK%20Pritchard&volume=164&issue=4&publication_year=2003&pages=1567-1587&pmid=12930761&doi=10.1093/genetics/164.4.1567&)

27. Robbins C, Torres JB, Hooker S, et al. Confirmation study of prostate cancer risk variants at 8q24 in African Americans identifies a novel risk locus. Genome Res. 2007;17(12):1717–1722. doi: 10.1101/gr.6782707.  [DOI](https://doi.org/10.1101/gr.6782707) | [PMC free article](/articles/PMC2099580/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17978284/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Res&title=Confirmation%20study%20of%20prostate%20cancer%20risk%20variants%20at%208q24%20in%20African%20Americans%20identifies%20a%20novel%20risk%20locus.&author=C%20Robbins&author=JB%20Torres&author=S%20Hooker&volume=17&issue=12&publication_year=2007&pages=1717-1722&pmid=17978284&doi=10.1101/gr.6782707&)

28. Dunham I, Kundaje A, Aldred SF, et al. ENCODE Project Consortium. An integrated encyclopedia of DNA elements in the human genome. Nature. 2012;489(7414):57–74. doi: 10.1038/nature11247.  [DOI](https://doi.org/10.1038/nature11247) | [PMC free article](/articles/PMC3439153/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22955616/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=An%20integrated%20encyclopedia%20of%20DNA%20elements%20in%20the%20human%20genome.&author=I%20Dunham&author=A%20Kundaje&author=SF%20Aldred&volume=489&issue=7414&publication_year=2012&pages=57-74&pmid=22955616&doi=10.1038/nature11247&)

29. NIH Roadmap Epigenomics Mapping Consortium. Roadmap epigenomics project. Available at: http://www.roadmapepigenomics.org. Accessed March 2, 2014.  [http://www.roadmapepigenomics.org](http://www.roadmapepigenomics.org) | [DOI](https://doi.org/10.1038/nbt1010-1045) | [PMC free article](/articles/PMC3607281/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20944595/)

30. Ernst J, Kellis M. Discovery and characterization of chromatin states for systematic annotation of the human genome. Nat Biotechnol. 2010;28(8):817–825. doi: 10.1038/nbt.1662.  [DOI](https://doi.org/10.1038/nbt.1662) | [PMC free article](/articles/PMC2919626/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20657582/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Biotechnol&title=Discovery%20and%20characterization%20of%20chromatin%20states%20for%20systematic%20annotation%20of%20the%20human%20genome.&author=J%20Ernst&author=M%20Kellis&volume=28&issue=8&publication_year=2010&pages=817-825&pmid=20657582&doi=10.1038/nbt.1662&)

31. Stranger BE, Montgomery SB, Dimas AS, et al. Patterns of cis regulatory variation in diverse human populations. PLoS Genet. 2012;8(4):e1002639. doi: 10.1371/journal.pgen.1002639.  [DOI](https://doi.org/10.1371/journal.pgen.1002639) | [PMC free article](/articles/PMC3330104/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22532805/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet&title=Patterns%20of%20cis%20regulatory%20variation%20in%20diverse%20human%20populations.&author=BE%20Stranger&author=SB%20Montgomery&author=AS%20Dimas&volume=8&issue=4&publication_year=2012&pages=e1002639&pmid=22532805&doi=10.1371/journal.pgen.1002639&)

32. Altshuler DM, Gibbs RA, Peltonen L, et al. International HapMap 3 Consortium. Integrating common and rare genetic variation in diverse human populations. Nature. 2010;467(7311):52–58. doi: 10.1038/nature09298.  [DOI](https://doi.org/10.1038/nature09298) | [PMC free article](/articles/PMC3173859/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20811451/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Integrating%20common%20and%20rare%20genetic%20variation%20in%20diverse%20human%20populations.&author=DM%20Altshuler&author=RA%20Gibbs&author=L%20Peltonen&volume=467&issue=7311&publication_year=2010&pages=52-58&pmid=20811451&doi=10.1038/nature09298&)

33. Johnson AD, Handsaker RE, Pulit SL, Nizzari MM, O’Donnell CJ, de Bakker PIW. SNAP: a web-based tool for identification and annotation of proxy SNPs using HapMap. Bioinformatics. 2008;24(24):2938–2939. doi: 10.1093/bioinformatics/btn564.  [DOI](https://doi.org/10.1093/bioinformatics/btn564) | [PMC free article](/articles/PMC2720775/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18974171/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=SNAP:%20a%20web-based%20tool%20for%20identification%20and%20annotation%20of%20proxy%20SNPs%20using%20HapMap.&author=AD%20Johnson&author=RE%20Handsaker&author=SL%20Pulit&author=MM%20Nizzari&author=CJ%20O%E2%80%99Donnell&volume=24&issue=24&publication_year=2008&pages=2938-2939&pmid=18974171&doi=10.1093/bioinformatics/btn564&)

34. Pickrell JK, Coop G, Novembre J, et al. Signals of recent positive selection in a worldwide sample of human populations. Genome Res. 2009;19(5):826–837. doi: 10.1101/gr.087577.108.  [DOI](https://doi.org/10.1101/gr.087577.108) | [PMC free article](/articles/PMC2675971/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19307593/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Res&title=Signals%20of%20recent%20positive%20selection%20in%20a%20worldwide%20sample%20of%20human%20populations.&author=JK%20Pickrell&author=G%20Coop&author=J%20Novembre&volume=19&issue=5&publication_year=2009&pages=826-837&pmid=19307593&doi=10.1101/gr.087577.108&)

35. Coop G, Pickrell JK, Novembre J, et al. The role of geography in human adaptation. PLoS Genet. 2009;5(6):e1000500. doi: 10.1371/journal.pgen.1000500.  [DOI](https://doi.org/10.1371/journal.pgen.1000500) | [PMC free article](/articles/PMC2685456/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19503611/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet&title=The%20role%20of%20geography%20in%20human%20adaptation.&author=G%20Coop&author=JK%20Pickrell&author=J%20Novembre&volume=5&issue=6&publication_year=2009&pages=e1000500&pmid=19503611&doi=10.1371/journal.pgen.1000500&)

36. Mitchell C, Gregersen N, Krause A. Novel CYP2C9 and VKORC1 gene variants associated with warfarin dosage variability in the South African black population. Pharmacogenomics. 2011;12(7):953–963. doi: 10.2217/pgs.11.36.  [DOI](https://doi.org/10.2217/pgs.11.36) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21635147/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Novel%20CYP2C9%20and%20VKORC1%20gene%20variants%20associated%20with%20warfarin%20dosage%20variability%20in%20the%20South%20African%20black%20population.&author=C%20Mitchell&author=N%20Gregersen&author=A%20Krause&volume=12&issue=7&publication_year=2011&pages=953-963&pmid=21635147&doi=10.2217/pgs.11.36&)

37. Consortium TG GTEx Consortium. The Genotype-Tissue Expression (GTEx) project. Nat Genet. 2013;45(6):580–585. doi: 10.1038/ng.2653.  [DOI](https://doi.org/10.1038/ng.2653) | [PMC free article](/articles/PMC4010069/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23715323/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=The%20Genotype-Tissue%20Expression%20(GTEx)%20project.&author=TG%20Consortium&volume=45&issue=6&publication_year=2013&pages=580-585&pmid=23715323&doi=10.1038/ng.2653&)

38. Perera MA, Cavallari LH, Johnson JA. Warfarin pharmacogenetics: an illustration of the importance of studies in minority populations. Clin Pharmacol Ther. 2014;95(3):242–244. doi: 10.1038/clpt.2013.209.  [DOI](https://doi.org/10.1038/clpt.2013.209) | [PMC free article](/articles/PMC4131541/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24548987/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Warfarin%20pharmacogenetics:%20an%20illustration%20of%20the%20importance%20of%20studies%20in%20minority%20populations.&author=MA%20Perera&author=LH%20Cavallari&author=JA%20Johnson&volume=95&issue=3&publication_year=2014&pages=242-244&pmid=24548987&doi=10.1038/clpt.2013.209&)

39. Daneshjou R, Klein TE, Altman RB. Genotype-guided dosing of vitamin K antagonists. N Engl J Med. 2014;370(18):1762–1763. doi: 10.1056/NEJMc1402521#SA4.  [DOI](https://doi.org/10.1056/NEJMc1402521#SA4) | [PMC free article](/articles/PMC4145721/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24785217/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Genotype-guided%20dosing%20of%20vitamin%20K%20antagonists.&author=R%20Daneshjou&author=TE%20Klein&author=RB%20Altman&volume=370&issue=18&publication_year=2014&pages=1762-1763&pmid=24785217&doi=10.1056/NEJMc1402521#SA4&)

40. Fahmi S, Yang C, Esmail S, Hobbs HH, Cohen JC. Functional characterization of genetic variants in NPC1L1 supports the sequencing extremes strategy to identify complex trait genes. Hum Mol Genet. 2008;17(14):2101–2107. doi: 10.1093/hmg/ddn108.  [DOI](https://doi.org/10.1093/hmg/ddn108) | [PMC free article](/articles/PMC2441721/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18413323/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mol%20Genet&title=Functional%20characterization%20of%20genetic%20variants%20in%20NPC1L1%20supports%20the%20sequencing%20extremes%20strategy%20to%20identify%20complex%20trait%20genes.&author=S%20Fahmi&author=C%20Yang&author=S%20Esmail&author=HH%20Hobbs&author=JC%20Cohen&volume=17&issue=14&publication_year=2008&pages=2101-2107&pmid=18413323&doi=10.1093/hmg/ddn108&)

41. Lange LA, Hu Y, Zhang H, et al. NHLBI Grand Opportunity Exome Sequencing Project. Whole-exome sequencing identifies rare and low-frequency coding variants associated with LDL cholesterol. Am J Hum Genet. 2014;94(2):233–245. doi: 10.1016/j.ajhg.2014.01.010.  [DOI](https://doi.org/10.1016/j.ajhg.2014.01.010) | [PMC free article](/articles/PMC3928660/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24507775/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=Whole-exome%20sequencing%20identifies%20rare%20and%20low-frequency%20coding%20variants%20associated%20with%20LDL%20cholesterol.&author=LA%20Lange&author=Y%20Hu&author=H%20Zhang&volume=94&issue=2&publication_year=2014&pages=233-245&pmid=24507775&doi=10.1016/j.ajhg.2014.01.010&)

42. Johnson JA, Gong L, Whirl-Carrillo M, et al. Clinical Pharmacogenetics Implementation Consortium. Clinical Pharmacogenetics Implementation Consortium Guidelines for CYP2C9 and VKORC1 genotypes and warfarin dosing. Clin Pharmacol Ther. 2011;90(4):625–629. doi: 10.1038/clpt.2011.185.  [DOI](https://doi.org/10.1038/clpt.2011.185) | [PMC free article](/articles/PMC3187550/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21900891/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Clinical%20Pharmacogenetics%20Implementation%20Consortium%20Guidelines%20for%20CYP2C9%20and%20VKORC1%20genotypes%20and%20warfarin%20dosing.&author=JA%20Johnson&author=L%20Gong&author=M%20Whirl-Carrillo&volume=90&issue=4&publication_year=2011&pages=625-629&pmid=21900891&doi=10.1038/clpt.2011.185&)

43. Wang Y, Zhao R, Goldman ID. Decreased expression of the reduced folate carrier and folypolyglutamate synthetase is the basis for acquired resistance to the pemetrexed antifolate (LY231514) in an L1210 murine leukemia cell line. Biochem Pharmacol. 2003;65(7):1163–1170. doi: 10.1016/s0006-2952(03)00007-8.  [DOI](https://doi.org/10.1016/s0006-2952(03)00007-8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12663051/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem%20Pharmacol&title=Decreased%20expression%20of%20the%20reduced%20folate%20carrier%20and%20folypolyglutamate%20synthetase%20is%20the%20basis%20for%20acquired%20resistance%20to%20the%20pemetrexed%20antifolate%20(LY231514)%20in%20an%20L1210%20murine%20leukemia%20cell%20line.&author=Y%20Wang&author=R%20Zhao&author=ID%20Goldman&volume=65&issue=7&publication_year=2003&pages=1163-1170&pmid=12663051&doi=10.1016/s0006-2952(03)00007-8&)

44. Kager L, Cheok M, Yang W, et al. Folate pathway gene expression differs in subtypes of acute lymphoblastic leukemia and influences methotrexate pharmacodynamics. J Clin Invest. 2005;115(1):110–117. doi: 10.1172/JCI22477.  [DOI](https://doi.org/10.1172/JCI22477) | [PMC free article](/articles/PMC539195/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15630450/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Invest&title=Folate%20pathway%20gene%20expression%20differs%20in%20subtypes%20of%20acute%20lymphoblastic%20leukemia%20and%20influences%20methotrexate%20pharmacodynamics.&author=L%20Kager&author=M%20Cheok&author=W%20Yang&volume=115&issue=1&publication_year=2005&pages=110-117&pmid=15630450&doi=10.1172/JCI22477&)

45. Clarke L, Waxman DJ. Human liver folylpolyglutamate synthetase: biochemical characterization and interactions with folates and folate antagonists. Arch Biochem Biophys. 1987;256(2):585–596. doi: 10.1016/0003-9861(87)90616-3.  [DOI](https://doi.org/10.1016/0003-9861(87)90616-3) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/3619447/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arch%20Biochem%20Biophys&title=Human%20liver%20folylpolyglutamate%20synthetase:%20biochemical%20characterization%20and%20interactions%20with%20folates%20and%20folate%20antagonists.&author=L%20Clarke&author=DJ%20Waxman&volume=256&issue=2&publication_year=1987&pages=585-596&pmid=3619447&doi=10.1016/0003-9861(87)90616-3&)

46. Kim M-S, Pinto SM, Getnet D, et al. A draft map of the human proteome. Nature. 2014;509(7502):575–581. doi: 10.1038/nature13302.  [DOI](https://doi.org/10.1038/nature13302) | [PMC free article](/articles/PMC4403737/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24870542/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=A%20draft%20map%20of%20the%20human%20proteome.&author=M-S%20Kim&author=SM%20Pinto&author=D%20Getnet&volume=509&issue=7502&publication_year=2014&pages=575-581&pmid=24870542&doi=10.1038/nature13302&)

47. Muszkat M, Bialer O, Blotnick S, et al. Effects of folic acid supplementation on the pharmacokinetics and anticoagulant effect of warfarin: an open-label, prospective study of long-term administration in adults. Clin Ther. 2010;32(2):347–356. doi: 10.1016/j.clinthera.2010.02.008.  [DOI](https://doi.org/10.1016/j.clinthera.2010.02.008) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20206792/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Ther&title=Effects%20of%20folic%20acid%20supplementation%20on%20the%20pharmacokinetics%20and%20anticoagulant%20effect%20of%20warfarin:%20an%20open-label,%20prospective%20study%20of%20long-term%20administration%20in%20adults.&author=M%20Muszkat&author=O%20Bialer&author=S%20Blotnick&volume=32&issue=2&publication_year=2010&pages=347-356&pmid=20206792&doi=10.1016/j.clinthera.2010.02.008&)

48. Mayer O, Jr, Simon J, Rosolová H, Hromádka M, Subrt I, Vobrubová I. The effects of folate supplementation on some coagulation parameters and oxidative status surrogates. Eur J Clin Pharmacol. 2002;58(1):1–5. doi: 10.1007/s00228-001-0421-6.  [DOI](https://doi.org/10.1007/s00228-001-0421-6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11956665/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Clin%20Pharmacol&title=The%20effects%20of%20folate%20supplementation%20on%20some%20coagulation%20parameters%20and%20oxidative%20status%20surrogates.&author=O%20Mayer&author=J%20Simon&author=H%20Rosolov%C3%A1&author=M%20Hrom%C3%A1dka&author=I%20Subrt&volume=58&issue=1&publication_year=2002&pages=1-5&pmid=11956665&doi=10.1007/s00228-001-0421-6&)

49. Deol PS, Barnes TA, Dampier K, John Pasi K, Oppenheimer C, Pavord SR. The effects of folic acid supplements on coagulation status in pregnancy. Br J Haematol. 2004;127(2):204–208. doi: 10.1111/j.1365-2141.2004.05172.x.  [DOI](https://doi.org/10.1111/j.1365-2141.2004.05172.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15461627/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Haematol&title=The%20effects%20of%20folic%20acid%20supplements%20on%20coagulation%20status%20in%20pregnancy.&author=PS%20Deol&author=TA%20Barnes&author=K%20Dampier&author=K%20John%20Pasi&author=C%20Oppenheimer&volume=127&issue=2&publication_year=2004&pages=204-208&pmid=15461627&doi=10.1111/j.1365-2141.2004.05172.x&)
