# Inflammation-related Genetic Variations and Survival for Advanced Non-Small Cell Lung Cancer Receiving First-line Chemotherapy

## Metadata
**Authors:** Xia Pu, Michelle A Hildebrandt, Charles Lu, Jack A Roth, David J Stewart, Yang Zhao, Rebecca S Heist, Yuanqing Ye, David W Chang, Li Su, John D Minna, Scott M Lippman, Margaret R Spitz, David C Christiani, Xifeng Wu
**Journal:** Clinical pharmacology and therapeutics
**Date:** 2014 Apr 22
**DOI:** [10.1038/clpt.2014.89](https://doi.org/10.1038/clpt.2014.89)
**PMID:** 24755914
**PMCID:** PMC4141040
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4141040/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC4141040/pdf/nihms-587120.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC4141040/pdf/nihms-587120.pdf)

## Abstract

**Background:** 
accurate prognostic prediction is challenging for advanced-stage non-small cell lung cancer (NSCLC) patients.

**Methods:** 
we systematically investigated genetic variants within inflammation pathway as potential prognostic markers for advanced-stage NSCLC patients treated with first-line chemotherapy. A discovery phase in 502 patients and an internal validation in 335 patients were completed at MD Anderson Cancer Center. External validation was performed in 371 patients at Harvard University.

**Results:** 
a missense SNP (HLA-DOB:rs2071554) predicted to influence protein function was significantly associated with poor survival in the discovery (HR:1.46, 95% CI:1.02-2.09), internal validation (HR:1.51, 95% CI:1.02-2.25), and external validation (HR:1.52, 95% CI:1.01-2.29) populations. KLRK1:rs2900420 was associated with a reduced risk in the discovery (HR:0.76, 95% CI:0.60-0.96), internal validation (HR:0.77, 95% CI:0.61-0.99), and external validation (HR:0.80, 95% CI:0.63-1.02) populations. A strong cumulative effect was observed for these SNPs on overall survival.

**Conclusions:** 
Genetic variations in inflammation-related genes could have potential to complement prediction of prognosis.

Keywords: NSCLC, advanced stage, overall survival, inflammation, single nucleotide polymorphism

### Background

accurate prognostic prediction is challenging for advanced-stage non-small cell lung cancer (NSCLC) patients.

### Methods

we systematically investigated genetic variants within inflammation pathway as potential prognostic markers for advanced-stage NSCLC patients treated with first-line chemotherapy. A discovery phase in 502 patients and an internal validation in 335 patients were completed at MD Anderson Cancer Center. External validation was performed in 371 patients at Harvard University.

### Results

a missense SNP (*HLA-DOB:*rs2071554) predicted to influence protein function was significantly associated with poor survival in the discovery (HR:1.46, 95% CI:1.02-2.09), internal validation (HR:1.51, 95% CI:1.02-2.25), and external validation (HR:1.52, 95% CI:1.01-2.29) populations. *KLRK1:*rs2900420 was associated with a reduced risk in the discovery (HR:0.76, 95% CI:0.60-0.96), internal validation (HR:0.77, 95% CI:0.61-0.99), and external validation (HR:0.80, 95% CI:0.63-1.02) populations. A strong cumulative effect was observed for these SNPs on overall survival.

### Conclusions

Genetic variations in inflammation-related genes could have potential to complement prediction of prognosis.

## INTRODUCTION

Lung cancer is a highly lethal disease and was responsible for an estimated 160,000 deaths in 2013 in the US ([1](#R1)). Patients are typically diagnosed at advanced stage (stage III/IV) with a dismal 5-year survival rate([2](#R2)). Combination chemotherapy is the standard of care for stage IV non-small cell lung cancer (NSCLC), while combined platinum-based chemoradiation or chemoradiation/surgery is the standard therapy for stage III NSCLC ([3](#R3), [4](#R4)). While some patients benefit from standard of care, others do not. Prognostic biomarkers that improve the accuracy of outcome prediction for individual patients could be useful clinically.

Inflammation is estimated to contribute to 15% of all cancer deaths ([5](#R5)). The lung is a frequent site of inflammation due to environmental exposures, and inflammatory diseases of the lung, such as chronic obstructive pulmonary disease, have been related to increased incidence and a poor prognosis of lung cancer ([6](#R6), [7](#R7)). Evidence has shown that inflammatory molecules and effectors are independently associated with tumor progression and survival in advanced-stage lung cancer patients ([8](#R8), [9](#R9)). Moreover, chemotherapeutic agents are known to induce cellular damage, which could trigger an acute inflammatory response ([10](#R10), [11](#R11)). Uncontrolled inflammation can attenuate treatment effectiveness and lead to the development of chemoresistance ([12](#R12)) or toxicities, both of which worsen prognosis ([13](#R13)). Taken together, these findings suggest that genetic markers of inflammation might be promising prognostic biomarkers for patients with advanced lung cancer.

Genome-wide association studies (GWAS) have recently been used to detect loci as potential biomarkers of risk and outcomes for various diseases, including lung cancer ([14](#R14)-[19](#R19)). However, some genomic regions have relatively low coverage due to weak linkage disequilibrium relationships and the design of the GWAS arrays. Thus, a comprehensive evaluation of genetic regions of interest based on prior knowledge of the disease biology utilizing pathway-based or gene-based approaches are needed to compliment GWAS findings ([20](#R20)). Towards this, we conducted a multi-phase, pathway-based study to evaluate single nucleotide polymorphisms (SNPs) in major inflammation genes for their effect on overall survival in patients with advanced NSCLC treated with first-line primary chemotherapy (either alone or in combination with radiotherapy), with the goal of identifying potential prognostic biomarkers that will benefit this group of patients.

## RESULTS

### Patient characteristics

A total of 837 (discovery: 502, validation: 335) patients from MD Anderson and 371 patients from Harvard were included in the analysis ([Table 1](#T1)). MD Anderson populations have a relatively longer median survival time (MST) (discovery: 16.5 months, validation: 16.8 months) compared to the Harvard population (12.2 months). The median follow-up time of patients in MD Anderson discovery phase is relatively short (30.5 months) which is probably due to the higher percentage of stage IV patients. All the patients were non-Hispanic whites with Stage III or IV NSCLC and age was not significantly different between patients who had died and those who were alive.

### Table 1.

|   | MD Anderson Discovery |  |  | MD Anderson Validation |  |  | Harvard Validation |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| Variables | Dead (%) | Alive (%) | P | Dead (%) | Alive (%) | P | Dead (%) | Alive (%) | P |
| MST (months) | 16.5 |  |   | 16.8 |  |   | 12.2 |  |   |
| MFT (months) | 30.5 |  |   | 89.6 |  |   | 60.0 |  |   |
| Age, mean(SD) | 60.7(11.2) | 62.4(10.5) | 0.099 | 59.3(10.4) | 57.5(9.0) | 0.374 | 63.58(10.55) | 60.45(10.76) | 0.053 |
| Sex |  |  |  |  |  |  |  |  |  |
| Male | 166(67) | 80(33) |   | 196(94) | 12(6) |   | 171(89) | 22(11) |   |
| Female | 174(68) | 82(32) | 0.907 | 110(87) | 17(13) | 0.016 | 147(83) | 31(17) | 0.098 |
| Smoking status |  |  |  |  |  |  |  |  |  |
| Never | 129(76) | 41(24) |   | 4(100) | 0(0) |   | 25(76) | 8(24) |   |
| Former | 117(61) | 74(39) |   | 145(92) | 13(8) |   | 154(87) | 23(13) |   |
| Current & RQ | 94(67) | 47(33) | 0.012 | 157(91) | 16(9) | 0.782 | 139(86) | 22(14) | 0.227 |
| Clinical stage |  |  |  |  |  |  |  |  |  |
| Stage III | 99(58) | 72(42) |   | 142(88) | 20(12) |   | 118(84) | 22(16) |   |
| Stage IV | 241(73) | 90(27) | 0.001 | 164(95) | 9(5) | 0.020 | 200(87) | 31(13) | 0.540 |
| Chemotherapy regimens |  |  |  |  |  |  |  |  |  |
| Platinum-based + other agent* | 276(67) | 136(33) | 0.449 | 253(90) | 27(10) | 0.148 | 264(85) | 48(15) | 0.251 |
| Non platinum-based | 64(71) | 26(29) |   | 53(96) | 2(4) |   | 35(90) | 4(10) |   |
| Missing |   |   |   |   |   |   | 19(95) | 1(5) |   |
| Total | 340 | 162 |   | 306 | 29 |   | 318 | 53 |   |

Table 1 Caption: Characteristics of the study populations at the time of analysis

### Association of inflammation-related SNPs on overall survival

A total of 11,930 SNPs from 904 genes were genotyped, of which 11,689 passed quality control measures and were included in the MD Anderson discovery analysis ([Figure 1](#F1)). 1,123 SNPs were significantly associated with overall survival in this group (P<0.05). Among these SNPs, genotyping data from a previously published GWAS ([19](#R19)) was available for 267 SNPs. After removing 413 SNPs which had insignificant (P<0.2) proxy SNPs (r^2^>0.8) on the GWAS chip, we genotyped an additional 443 SNPs using a custom designed iSelect BeadChip.

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/95f9/4141040/483ad3dd1663/nihms-587120-f0001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4141040_nihms-587120-f0001.jpg)

Schematic of study design, SNP selection, and populations for MD Anderson discovery, MD Anderson internal validation, and Harvard external validation.

After quality control, 657 SNPs (390 genotyped and 267 using existing genotype data) were selected for analysis in the internal validation. We validated the association with overall survival for 49 SNPs (HRs consistent and P < 0.05 in both phases). We then performed an external validation of 32 of the 49 SNPs (those that had existing data available from previously published GWAS) in the Harvard population ([17](#R17)). Seventeen SNPs were found to have consistent effects on overall survival in all three populations, with two significant (or borderline significant) in all three phases ([Table 2](#T2)).

### Table 2.

|   |   |   | DiscoveryMD Anderson |  | Internal ValidationMD Anderson |  | External ValidationHarvard population |  | Combined Validation |  |  | Combined Overall |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| SNP | Gene | Model | HR(95% CI)* | P | HR(95% CI)* | P | HR(95% CI)* | P | HR(95% CI)** | P | P-het | HR(95% CI)** | P | P-het |
| rs2071554 | HLA-DOB | DOM | 1.46(1.02-2.09) | 0.040 | 1.51(1.02-2.25) | 0.041 | 1.52(1.01-2.29) | 0.045 | 1.52(1.14-2.02) | 0.004 | 0.982 | 1.49(1.19-1.87) | 4.32×10−4 | 0.987 |
| rs2900420 | KLRK1 | DOM | 0.76(0.60-0.96) | 0.021 | 0.77(0.61-0.99) | 0.038 | 0.80(0.63-1.02) | 0.069 | 0.79(0.66-0.93) | 0.006 | 0.832 | 0.78(0.68-0.89) | 3.51×10−4 | 0.944 |
| rs12141256 | FAF1 | DOM | 0.75(0.57-0.97) | 0.031 | 0.71(0.52-0.97) | 0.033 | 0.87(0.66-1.13) | 0.295 | 0.80(0.65-0.98) | 0.029 | 0.399 | 0.78(0.66-0.91) | 2.27×10−3 | 0.601 |
| rs1986649 | FOXO1A | DOM | 0.76(0.60-0.96) | 0.020 | 0.75(0.59-0.95) | 0.018 | 0.88(0.69-1.13) | 0.322 | 0.81(0.68-0.96) | 0.017 | 0.442 | 0.79(0.69-0.91) | 9.43×10−4 | 0.584 |
| rs7972757 | KLRK1 | DOM | 0.73(0.55-0.98) | 0.035 | 0.67(0.49-0.92) | 0.012 | 0.87(0.66-1.15) | 0.331 | 0.78(0.63-0.95) | 0.016 | 0.314 | 0.76(0.64-0.90) | 1.42×10−3 | 0.452 |
| rs17446614 | FOXO1A | DOM | 0.72(0.56-0.93) | 0.011 | 0.69(0.53-0.90) | 0.006 | 0.89(0.68-1.16) | 0.386 | 0.78(0.65-0.94) | 0.010 | 0.293 | 0.76(0.65-0.88) | 3.34×10−4 | 0.364 |
| rs216136 | CSF1R | ADD | 1.21(1.03-1.42) | 0.023 | 1.17(1.00-1.37) | 0.046 | 1.07(0.91-1.25) | 0.410 | 1.12(1.00-1.25) | 0.047 | 0.367 | 1.15(1.05-1.25) | 3.46×10−3 | 0.528 |
| rs2189521 | IL21R | REC | 1.41(1.03-1.94) | 0.032 | 1.43(1.08-1.89) | 0.014 | 1.13(0.85-1.50) | 0.415 | 1.27(1.04-1.55) | 0.020 | 0.160 | 1.31(1.10-1.55) | 1.85×10−3 | 0.438 |
| rs1509 | CAPN10 | ADD | 0.83(0.69-0.99) | 0.038 | 0.83(0.68-1.00) | 0.048 | 0.93(0.78-1.11) | 0.433 | 0.88(0.77-1.00) | 0.055 | 0.460 | 0.86(0.78-0.96) | 5.53×10−3 | 0.567 |
| rs10964912 | IFNA14 | REC | 1.49(1.01-2.19) | 0.044 | 2.00(1.26-3.17) | 0.003 | 1.16(0.78-1.72) | 0.462 | 1.46(1.08-1.97) | 0.013 | 0.012 | 1.47(1.16-1.86) | 1.38×10−3 | 0.208 |
| rs971768 | IL17RA | DOM | 1.47(1.09-1.98) | 0.012 | 1.46(1.00-2.12) | 0.047 | 1.16(0.78-1.74) | 0.465 | 1.31(1.00-1.73) | 0.051 | 0.355 | 1.38(1.13-1.69) | 1.71×10−3 | 0.626 |
| rs10000856 | IRF2 | ADD | 1.26(1.07-1.50) | 0.007 | 1.22(1.03-1.44) | 0.020 | 1.06(0.90-1.25) | 0.506 | 1.13(1.01-1.28) | 0.036 | 0.148 | 1.18(1.07-1.29) | 1.07×10−3 | 0.288 |
| rs2133092 | TLN2 | DOM | 1.30(1.04-1.63) | 0.023 | 1.30(1.03-1.64) | 0.027 | 1.08(0.84-1.38) | 0.543 | 1.19(1.01-1.41) | 0.043 | 0.207 | 1.23(1.07-1.41) | 2.88×10−3 | 0.472 |
| rs11903566 | PRKCE | DOM | 1.60(1.15-2.24) | 0.006 | 1.45(1.04-2.03) | 0.029 | 1.11(0.74-1.67) | 0.625 | 1.30(1-1.69) | 0.046 | 0.266 | 1.41(1.15-1.73) | 1.05×10−3 | 0.381 |
| rs908742 | PRKCZ | DOM | 1.28(1.03-1.60) | 0.024 | 1.33(1.06-1.67) | 0.015 | 1.03(0.82-1.29) | 0.794 | 1.17(0.99-1.37) | 0.188 | 0.042 | 1.21(1.06-1.37) | 4.44×10−3 | 0.234 |
| rs3749166 | CAPN10 | REC | 1.41(1.04-1.92) | 0.029 | 1.42(1.02-1.99) | 0.038 | 1.00(0.71-1.41) | 0.992 | 1.20(0.94-1.52) | 0.177 | 0.070 | 1.27(1.06-1.54) | 0.012 | 0.254 |

Table 2 Caption: Seventeen inflammation-related SNPs with consistent effects on overall survival across three analytical phases

Rs2071554, a missense variation in the first exon of *HLA-DOB* (major histocompatibility complex class II, DO beta), was associated with increased risk of death in all three populations ([Figure 2a](#F2)). In the MD Anderson discovery population (HR=1.46, 95% CI=1.02- 2.09, P=0.040), patients carrying at least one variant allele (AG or AA) had a significant survival disparity of six months, from 17 months to 11 months, compared with those who were homozygous for the common allele (GG, P for log-rank test=0.009, [Figure 3a](#F3)). In the MD Anderson internal validation population, rs2071554 was also associated with increased risk of death (HR=1.51, 95% CI=1.02-2.25, P=0.041), and a non-significant, but appreciable seven month shortened MST ([Figure 3b](#F3)). A similar effect was observed in the Harvard external validation population. The variant allele was associated with shortened overall survival (HR=1.52, 95% CI=1.01- 2.29, P=0.045); patients carrying at least one copy of the variant allele had a shorter MST than patients who were homozygous for the common allele (P for log-rank test=0.007; [Figure 3c](#F3)). Meta-analysis of the association of rs2071554 with overall survival under the fixed effects model showed a P value of 4.3×10^−4^ (HR=1.49, 95% CI=1.19-1.87, P for heterogeneity=0.988, [Figure 2a](#F2)).

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/95f9/4141040/8102260f9c2b/nihms-587120-f0002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4141040_nihms-587120-f0002.jpg)

Forest plot for meta-analysis of the association of single nucleotide polymorphisms (A) HLA-DOB:rs2071554 and (B) KLRK1:rs2900420, as well as (C) cumulative effect, with overall survival in discovery and internal validation populations from MD Anderson and external validation population from Harvard. HR, hazard ratio; CI, confidence interval; NSCLCs, number of patients with non-small cell lung cancer.

### Figure 3.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/95f9/4141040/6f06d33c8f30/nihms-587120-f0003.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4141040_nihms-587120-f0003.jpg)

Kaplan-Meier estimates of HLA-DOB:rs2071554 genotypes and overall survival in advanced NSCLC patients treated with chemotherapy: (A) MD Anderson discovery; (B) MD Anderson internal validation; (C) Harvard external validation. N=A/B, A: number of patients dead, B: total number of patients. MST: median survival time.

*KLRK1*:rs2900420, which is located in the 3′ flanking region of the *KLRK1* (killer cell lectin-like receptor subfamily K, member 1) gene, a component of the natural killer cell signaling pathway, was associated with reduced risk in the MD Anderson discovery population (HR=0.76, 95% CI=0.60-0.96, P=0.021) and in the MD Anderson internal validation population (HR=0.77, 95% CI=0.61-0.99, P=0.038), while borderline significant in Harvard external validation population (HR=0.80, 95% CI=0.63-1.02, P=0.069; [Figure 2b](#F2)). Significant survival time advantages were observed for patients who carried at least one variant allele compared with patients who were homozygous for the common allele (discovery: GG, 15 months; AG and AA, 20 months; P for log-rank test=0.011; internal validation: GG, 15 months; AG and AA, 18 months; P for log-rank test=0.087). In the Harvard external validation population, the association of rs2900420 with overall survival reached borderline significance (HR=0.80, 95% CI=0.63-1.02, P=0.069). However, meta-analysis of the validation populations showed a significant effect (P=0.006) and in the overall meta-analysis the effect was highly significant at p=3.5×10^−4^ (HR=0.78, 95% CI=0.68-0.89, P for heterogeneity=0.945).

In addition, ten other variants were significant in the MD Anderson discovery and internal validation populations and did not reach significance in the Harvard external validation, but did show significance in a meta-analysis of the validation results and the overall meta-analysis.

### Cumulative effects

In the cumulative effects analysis, we observed a significant “SNP-dosage” effect of these SNPs on overall survival: the more risk genotypes a patient carried, the greater the deleterious effects on overall survival ([Figure 2c](#F2)). Compared to individuals without any UFGs, patients carrying one UFG had combined 31% increased risk of death (MD Anderson discovery: HR=1.37, 95% CI=1.07-1.76, P=0.013; MD Anderson internal validation: HR=1.32, 95% CI=1.03-1.71, P=0.031; Harvard external validation: HR=1.25, 95% CI=0.98-1.60, P=0.073). This raised to an 83% increase in risk in the overall population for those with two UFG (MD Anderson discovery: HR=1.83, 95% CI=1.14-2.94, P=0.012; MD Anderson internal validation: HR=1.96, 95% CI=1.17-3.30, P=0.011; Harvard external validation: HR=1.75, 95% CI=1.07-2.85, P=0.025) and significantly decreased median survival times ([Figure 4](#F4)).

### Figure 4.

![Figure 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/95f9/4141040/913cb05d3222/nihms-587120-f0004.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4141040_nihms-587120-f0004.jpg)

Kaplan-Meier estimates of UFGs and overall survival in advanced NSCLC patients treated with chemotherapy: (A) MD Anderson discovery; (B) MD Anderson internal validation; (C) Harvard external validation. N=A/B, A: number of patients dead, B: total number of patients. MST: median survival time.

### in silico function analysis of HLA-DOB rs2071554

To determine the potential consequences of this variant and explore the underlying mechanism, we applied bioinformatics tools to *in silico* evaluate the effect on protein structure and function. rs2071554 is a missense variation that results in an arginine to glutamine substitution in the first exon of *HLA-DOB*. Polyphen2 analysis suggested that this amino acid change may potentially damage protein function (Polyphen2: 0.923, sensitivity: 0.80, specificity: 0.94). Similarly, SIFT predicted this SNP to be deleterious (SIFT score: 0.02). Both tools provide additional evidence in support of the potential importance of this SNP on protein function.

## DISCUSSION

We systematically evaluated the effects of SNPs from major inflammation genes on overall survival of advanced NSCLC patients who received first-line chemotherapy. In our 3-phase pathway-based association study, we identified two potential prognostic biomarkers: *HLA-DOB:*rs2071554 and *KLRK1:*rs2900420. *HLA-DOB* variant increased risk with a corresponding decrease in median survival time, while the *KLRK1* SNP was protective and prolonged overall survival. Moreover, the *HLA-DOB* variant was predicted to alter function through *in silico* analysis, consistent with the observed association of increased risk of death and shortened survival time.

HLA-DOB is the beta subunit of the HLA-DO class II paralogs. It functions as a negative regulator of major histocompatibility complex class II molecules by inhibiting HLA-DM molecules in a pH-dependent manner. The DO:DM ratio dictates major histocompatibility complex class II restricted-antigen presentation efficiency ([21](#R21)). Evidence has shown that dysregulation of the antigen presentation pathway is involved in cancer development ([22](#R22)). Moreover, major histocompatibility complex class II molecules are key immune response molecules, which have been reported to have a positive relationship with prognosis in various cancers ([23](#R23), [24](#R24)). In our study, we determined that this missense SNP may alter protein structure and function, and we identified a robust adverse effect on survival across all three populations. Currently, no studies have implicated this gene as playing a role in lung cancer risk or clinical outcomes. Our results suggest a potential predictive role of this locus, making it worthy of future deep sequencing to identify the causal variant and functional analysis *in vitro* to elucidate the mechanisms responsible.

*KLRK1* (member 1 of the killer cell lectin-like receptor subfamily K) encodes for a transmembrane protein that interacts with various ligands to activate natural killer and T cells, leading to lysis of targeted cells, including tumor cells. This gene has been previously shown to be involved in chemoresistance for osteosarcoma ([25](#R25)) and ligands binding to *KLRK1* have been found to prevent cisplatin-induced cytotoxic lymphocyte killing ([26](#R26)). Studies have also reported that lung adenocarcinoma cells were able to escape from the innate immune response of natural killer cells by expressing heterogeneous ligands for KLRK1 ([27](#R27)). Furthermore, this gene has been identified as a promising target for immunotherapy for cancer ([28](#R28), [29](#R29)). However, similar to HLA-DOB, no previous studies have linked KLRK1 to lung risk or clinical outcomes, highlighting the ability of targeted approaches in identifying novel predictors. *KLRK1*:rs2900420 is located three kilobases 3′ to the KLRK1 gene. In our study, it was associated with prolonged overall survival in the MD Anderson populations and its association with prolonged overall survival was nearly significant in the Harvard external validation population. It is very likely that with increased sample size the results would reach statistical significance. An additional KLRK1 variant (rs7972757) was significant in the MD Anderson discovery and internal validation populations, but not replicated in the Harvard external validation, providing additional support to the potential importance of this gene in lung cancer. Further exploration of the potential underlying biological mechanism(s) of this association would increase our understanding of this relationship and solidify the role of *KLRK1* in lung cancer prognosis.

To minimize differences in tumor characteristics and treatment regimens between the two study sites (MD Anderson and Harvard), we followed strict inclusion criteria based on stage and treatment. For example, a majority (>80%) of the patients in all three study populations were treated with platinum-based chemotherapy ([Table 1](#T1)), most commonly with the addition of a taxane although other agents included pemetrexed, gemcitabine, bevacizumab, and erlotinib. However, even with these measures in place, there are always subtle, often unidentifiable, differences in the patient populations among different hospitals, which could result in differences in survival times as we observed between the MD Anderson and Harvard cohorts. For example, patients who died at Harvard cohort were at a slightly older age (63.6 years in Harvard, compared to 60.7 years in MD Anderson discovery and 59.3 years in MD Anderson validation). These slight differences in the populations underscore the potential impact of the two validated SNPs - the effects are stronger than any differences among the study populations making the findings more transferable across the general population of lung cancer patients and not study site specific. Several other genetic variants in inflammation genes were significant in the MD Anderson discovery and validation populations, but did not reach significance in the Harvard external validation. Ten did become significant in the validation meta-analysis ([Table 2](#T2)), suggesting that they may indeed be additional predictors of overall survival. These candidate variants are located in several well-known inflammation genes, including the receptors for several circulating cytokines (*CSF1R*, *IL21R*, *IL17RA*), cytokines (*IRF2*, *IFNA14*), and cellular signaling molecules (*PRKCE*, *PRKCZ*). Further analysis of these genetic variants and genes would be of interest to definitively establish or abolish a relationship with overall survival in advanced lung cancer patients.

To our knowledge, this is the first study to systematically investigate the effects of inflammation-related genetic variations on survival of advanced NSCLC patients. The major strength of this study was the three-phase screening and validation approach using two independent patient populations, which were drawn from the largest lung cancer pharmacogenetic clinical outcome studies in the United States. All patients were at advanced stages treated with first line chemotherapy with or without radiotherapy. In addition, we developed a comprehensive panel of inflammation-related genetic variations, which covered major cellular processes involved in inflammation responses and regulatory processes. With this extensive coverage, our results provide a broad overview of the role of genetic variation within the overall inflammation network in modulating patients’ clinical outcomes.

In conclusion, we identified and validated two potential genetic markers within the inflammation pathway that may affect overall survival in patients with advanced NSCLC treated with first-line chemotherapy. Given the important role of inflammation throughout the cancer continuum, these genetic markers may be promising prognostic markers to help in treatment decision-making in the clinic.

## METHODS

### Study populations and data collection

#### MD Anderson discovery and validation populations

Patients from The University of Texas MD Anderson Cancer Center included in this study are part of an ongoing lung study that has been recruiting since 1995. All patients were non-Hispanic white, had histologically confirmed advanced-stage (stage III or IV, AJCC v6.) NSCLC, did not undergo surgery, and received first-line chemotherapy with or without radiotherapy at MD Anderson. A total of 502 patients were included in the discovery population with an additional 335 in the validation analysis. A structured questionnaire was used to collect epidemiologic and demographic data during an in-person interview. In addition, genomic DNA was extracted from peripheral blood samples using the QIAamp DNA extraction kit (Qiagen, Valencia, CA). Clinical and follow-up data were obtained from medical records. Each patient signed informed consent, and this study was approved by the MD Anderson Institutional Review Board.

#### Harvard external validation

The details of the Harvard lung cancer population have been described in detail previously ([30](#R30)). In brief, participants were non-Hispanic white patients newly diagnosed with histologically confirmed lung cancer. From this population, we selected patients with advanced NSCLC who had received first-line chemotherapy with/without radiation therapy and had not undergone surgery were included in the external validation population. A total of 371 patients met these criteria. An interviewer-administered questionnaire was used to collect epidemiologic data. Peripheral blood was drawn for DNA extraction. Informed consent was signed by each study participant and Harvard Institutional Review Board approved this study.

### Genotyping and quality control

#### MD Anderson discovery

A custom Illumina iSelect genotyping BeadChip was designed to genotype genetic variants in inflammation-related genes (study design detailed in [Figure 1](#F1)). Genes involved in inflammatory responses and regulation were retrieved using the T1Dbase ([http://www.t1dbase.org](http://www.t1dbase.org); University of Cambridge), which focuses on diabetes-related and inflammation-related genes. Additional gene information was obtained from the WKINFLAM panel ([31](#R31)). Tagging SNPs for each gene were selected from within a 10-kb flanking region using CEU data from the HapMap Project ([http://www.hapmap.org](http://www.hapmap.org)), based on the NCBI B36 assembly and dbSNP b126 using the Tagger Pairwise method (r^2^> 0.8 and minor allele frequency [MAF]≥0.05) ([32](#R32)). Candidate SNPs were then submitted to Illumina (San Diego, CA) and tested for designability using the Assay Design Tool. SNPs with a score >0.6 were considered qualified for the creation of the iSelect BeadChip.

Detailed genotyping and quality control methods used in the discovery phase have been previously described ([33](#R33)). Briefly, genotyping was performed according to the standard Infinium II assay protocol for the iSelect HD BeadChips. Quality control measures were applied to the datasets, excluding any DNA samples or SNPs with a call rate (percentage of data available for all SNPs or samples) <95%. For patients with direct relatives also enrolled in the study, only one patient within the relationship, the one whose DNA sample had a higher SNP call rate was included in the final analysis. SNPs with MAF <0.01 were excluded.

#### MD Anderson internal validation

Genotyping for SNPs selected for the validation phase was done either through the design of a custom iSelect BeadChip or using existing HumanHap300/HumanHap317/HumanHap660 genotyping data. Quality control for the iSelect BeadChip was performed on the basis of sample and SNP call rates; we removed any samples or SNPs with a call rate <95%. Detailed quality control measures for the HumanHap300/HumanHap317/HumanHap660 BeadChip have been described previously; these were also based on genotyping call rate (call rate >95% for all samples and SNPs included). SNPs with MAF<0.01 were also excluded ([34](#R34)).

#### Harvard external validation

Genotypes for external validation were obtained from the Illumina HumanHap610-Quad chip following standard protocol, as previously described ([18](#R18)). Quality control measures were similar to those used in the MD Anderson populations: only SNPs and samples with a genotyping call rate >95% and SNPs with MAF>0.01 were included in the analysis.

### Statistical analyses

For each phase, multivariable Cox proportional hazards regression models, with corresponding hazard ratios (HRs) and 95% confidence intervals (CIs), were used to estimate the effect of a single SNP on overall survival (the time between diagnosis and death or last follow-up), adjusting for age at diagnosis, sex, smoking status (current, former, or never), clinical stage (stage III or IV), and treatment regimen (chemotherapy and/or radiotherapy). Patients who had smoked fewer than 100 cigarettes over their lifetime were defined as never-smokers; ever-smokers were defined as patients who had smoked > 100 cigarettes over their lifetime, including former smokers (those who had quit smoking >1 year before diagnosis), and current smokers and recent quitters (those who had quit smoking within a year before diagnosis).

Kaplan-Meier survival curves and corresponding log-rank tests were used to test the survival difference between genotypes of each SNP. Meta-analysis was performed to obtain summary HRs and 95% CIs. Heterogeneity was tested with chi-square-based Q-statistics. A fixed-effect model was used when heterogeneity was absent (P for heterogeneity >0.05).

The cumulative effect of the top two validated SNPs within each population was determined by counting the number of risk genotypes each patient carried and using patients without any risk genotypes as a reference group. Polyphen-2 ([http://genetics.bwh.harvard.edu/pph2/index.shtml](http://genetics.bwh.harvard.edu/pph2/index.shtml)) ([35](#R35)) and SIFT ([http://sift.bii.a-star.edu.sg/](http://sift.bii.a-star.edu.sg/)) ([36](#R36)) were used *in silico* to predict the influence of the SNP on protein function.

The potential effect of population stratification was evaluated using quantile-quantile plots of the test statistics in the MD Anderson discovery population. We calculated the inflation factor (λ) by dividing the observed median of test statistics by expected median (from χ^2^ distribution with 1 degree of freedom) value. The obtained λ is close to 1 (0.92), indicating that population substructure has no substantial effect on the test statistics in the discovery stage analysis.

### What is the current knowledge on the topic?

### What is the current knowledge on the topic?

### What is the current knowledge on the topic?

### What is the current knowledge on the topic?

For advanced-stage NSCLC patients, platinum-based chemotherapy is the standard of care. However, a major hurdle is variation in response and thus, the ability to predict prognosis for these patients to enable selection of optimal treatment regimens.

### What question did this study address?

This study aimed to identify accurate biomarkers that can be used to guide identification of advanced-stage NSCLC patients who would benefit from standard of care, as well as those who are at increased risk of a poor outcome under the same treatment regimen.

### What this study adds to our knowledge?

Two common germline genetic variants in inflammation-related genes were identified as being consistently associated with survival in advanced-stage NSCLC patients treated with first-line chemotherapy. Furthermore, this study is the first to implicate *HLA-DOB* and *KLRK1* as being involved in lung cancer outcomes.

### How this might change clinical pharmacology and therapeutics?

These novel loci have potential to complement clinicopathological variables in prediction of prognosis. This would assist the personalization of treatment regimens for those predicted to have a poor outcome.

## ACKNOWLEDGEMENTS

This work was supported by National Cancer Institute grants R01 CA111646, P50 CA070907, R01 CA127219, R01 CA074386, R01 CA092824, and P50 CA090578 in part by the National Institutes of Health through MD Anderson’s Cancer Center Support Grant, CA016672.

## Footnotes

## References

1. Siegel R, Naishadham D, Jemal A. Cancer statistics, 2013. CA: A Cancer Journal for Clinicians. 2013 doi: 10.3322/caac.21166. n/a-n/a.  [DOI](https://doi.org/10.3322/caac.21166) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23335087/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=CA:%20A%20Cancer%20Journal%20for%20Clinicians&title=Cancer%20statistics,%202013&author=R%20Siegel&author=D%20Naishadham&author=A%20Jemal&publication_year=2013&pmid=23335087&doi=10.3322/caac.21166&)

2. Jemal A, Siegel R, Xu J, Ward E. Cancer statistics. CA Cancer J Clin. 2010;60:277–300. doi: 10.3322/caac.20073.  [DOI](https://doi.org/10.3322/caac.20073) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20610543/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=CA%20Cancer%20J%20Clin&title=Cancer%20statistics&author=A%20Jemal&author=R%20Siegel&author=J%20Xu&author=E%20Ward&volume=60&publication_year=2010&pages=277-300&pmid=20610543&doi=10.3322/caac.20073&)

3. Ardizzoni A, et al. Cisplatin-versus carboplatin-based chemotherapy in first-line treatment of advanced non-small-cell lung cancer: an individual patient data meta-analysis. J Natl Cancer Inst. 2007;99:847–57. doi: 10.1093/jnci/djk196.  [DOI](https://doi.org/10.1093/jnci/djk196) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17551145/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Natl%20Cancer%20Inst&title=Cisplatin-versus%20carboplatin-based%20chemotherapy%20in%20first-line%20treatment%20of%20advanced%20non-small-cell%20lung%20cancer:%20an%20individual%20patient%20data%20meta-analysis&author=A%20Ardizzoni&volume=99&publication_year=2007&pages=847-57&pmid=17551145&doi=10.1093/jnci/djk196&)

4. Ung YC, Yu E, Falkson C, Haynes AE, Stys-Norman D, Evans WK. The role of high-dose-rate brachytherapy in the palliation of symptoms in patients with non-small-cell lung cancer: a systematic review. Brachytherapy. 2006;5:189–202. doi: 10.1016/j.brachy.2006.05.001.  [DOI](https://doi.org/10.1016/j.brachy.2006.05.001) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16864071/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Brachytherapy&title=The%20role%20of%20high-dose-rate%20brachytherapy%20in%20the%20palliation%20of%20symptoms%20in%20patients%20with%20non-small-cell%20lung%20cancer:%20a%20systematic%20review&author=YC%20Ung&author=E%20Yu&author=C%20Falkson&author=AE%20Haynes&author=D%20Stys-Norman&volume=5&publication_year=2006&pages=189-202&pmid=16864071&doi=10.1016/j.brachy.2006.05.001&)

5. Ramos-Nino ME. The Role of Chronic Inflammation in Obesity-Associated Cancers. ISRN Oncology. 2013;201325 doi: 10.1155/2013/697521.  [DOI](https://doi.org/10.1155/2013/697521) | [PMC free article](/articles/PMC3683483/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23819063/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=ISRN%20Oncology&title=The%20Role%20of%20Chronic%20Inflammation%20in%20Obesity-Associated%20Cancers&author=ME%20Ramos-Nino&volume=2013&publication_year=2013&pmid=23819063&doi=10.1155/2013/697521&)

6. Zaynagetdinov R, et al. Epithelial nuclear factor-kappaB signaling promotes lung carcinogenesis via recruitment of regulatory T lymphocytes. Oncogene. 2011 doi: 10.1038/onc.2011.480.  [DOI](https://doi.org/10.1038/onc.2011.480) | [PMC free article](/articles/PMC3266969/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22002309/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncogene&title=Epithelial%20nuclear%20factor-kappaB%20signaling%20promotes%20lung%20carcinogenesis%20via%20recruitment%20of%20regulatory%20T%20lymphocytes&author=R%20Zaynagetdinov&publication_year=2011&pmid=22002309&doi=10.1038/onc.2011.480&)

7. Vlachostergios PJ, et al. Baseline insulin-like growth factor-I plasma levels, systemic inflammation, weight loss and clinical outcome in metastatic non-small cell lung cancer patients. Oncology. 2011;81:113–8. doi: 10.1159/000331685.  [DOI](https://doi.org/10.1159/000331685) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21986483/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncology&title=Baseline%20insulin-like%20growth%20factor-I%20plasma%20levels,%20systemic%20inflammation,%20weight%20loss%20and%20clinical%20outcome%20in%20metastatic%20non-small%20cell%20lung%20cancer%20patients&author=PJ%20Vlachostergios&volume=81&publication_year=2011&pages=113-8&pmid=21986483&doi=10.1159/000331685&)

8. Ulrich CM, Bigler J, Potter JD. Non-steroidal anti-inflammatory drugs for cancer prevention: promise, perils and pharmacogenetics. Nat Rev Cancer. 2006;6:130–40. doi: 10.1038/nrc1801.  [DOI](https://doi.org/10.1038/nrc1801) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16491072/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Cancer&title=Non-steroidal%20anti-inflammatory%20drugs%20for%20cancer%20prevention:%20promise,%20perils%20and%20pharmacogenetics&author=CM%20Ulrich&author=J%20Bigler&author=JD%20Potter&volume=6&publication_year=2006&pages=130-40&pmid=16491072&doi=10.1038/nrc1801&)

9. Kasymjanova G, et al. The predictive value of pre-treatment inflammatory markers in advanced non-small-cell lung cancer. Curr Oncol. 2010;17:52–8. doi: 10.3747/co.v17i4.567.  [DOI](https://doi.org/10.3747/co.v17i4.567) | [PMC free article](/articles/PMC2913830/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20697515/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Oncol&title=The%20predictive%20value%20of%20pre-treatment%20inflammatory%20markers%20in%20advanced%20non-small-cell%20lung%20cancer&author=G%20Kasymjanova&volume=17&publication_year=2010&pages=52-8&pmid=20697515&doi=10.3747/co.v17i4.567&)

10. Mukhopadhyay P, et al. Poly(ADP-ribose) polymerase-1 is a key mediator of cisplatin-induced kidney inflammation and injury. Free Radic Biol Med. 2011;51:1774–88. doi: 10.1016/j.freeradbiomed.2011.08.006.  [DOI](https://doi.org/10.1016/j.freeradbiomed.2011.08.006) | [PMC free article](/articles/PMC3207278/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21884784/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Free%20Radic%20Biol%20Med&title=Poly(ADP-ribose)%20polymerase-1%20is%20a%20key%20mediator%20of%20cisplatin-induced%20kidney%20inflammation%20and%20injury&author=P%20Mukhopadhyay&volume=51&publication_year=2011&pages=1774-88&pmid=21884784&doi=10.1016/j.freeradbiomed.2011.08.006&)

11. Nakano S, Niho Y. 5-Fluorouracil and cisplatin sequential chemotherapy; mechanism of action and clinical application in advanced, unresectable non-small cell lung cancer. Intern Med. 1995;34:288–91. doi: 10.2169/internalmedicine.34.288.  [DOI](https://doi.org/10.2169/internalmedicine.34.288) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7606103/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Intern%20Med&title=5-Fluorouracil%20and%20cisplatin%20sequential%20chemotherapy;%20mechanism%20of%20action%20and%20clinical%20application%20in%20advanced,%20unresectable%20non-small%20cell%20lung%20cancer&author=S%20Nakano&author=Y%20Niho&volume=34&publication_year=1995&pages=288-91&pmid=7606103&doi=10.2169/internalmedicine.34.288&)

12. Mahon KL, Henshall SM, Sutherland RL, Horvath LG. Pathways of chemotherapy resistance in castration-resistant prostate cancer. Endocr Relat Cancer. 2011;18:R103–23. doi: 10.1530/ERC-10-0343.  [DOI](https://doi.org/10.1530/ERC-10-0343) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21565970/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Endocr%20Relat%20Cancer&title=Pathways%20of%20chemotherapy%20resistance%20in%20castration-resistant%20prostate%20cancer&author=KL%20Mahon&author=SM%20Henshall&author=RL%20Sutherland&author=LG%20Horvath&volume=18&publication_year=2011&pages=R103-23&pmid=21565970&doi=10.1530/ERC-10-0343&)

13. Yeh I, George E, Fleckman P. Eccrine hidradenitis sine neutrophils: a toxic response to chemotherapy. J Cutan Pathol. 2011;38:905–10. doi: 10.1111/j.1600-0560.2011.01776.x.  [DOI](https://doi.org/10.1111/j.1600-0560.2011.01776.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21955315/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Cutan%20Pathol&title=Eccrine%20hidradenitis%20sine%20neutrophils:%20a%20toxic%20response%20to%20chemotherapy&author=I%20Yeh&author=E%20George&author=P%20Fleckman&volume=38&publication_year=2011&pages=905-10&pmid=21955315&doi=10.1111/j.1600-0560.2011.01776.x&)

14. Amos CI, et al. Genome-wide association scan of tag SNPs identifies a susceptibility locus for lung cancer at 15q25.1. Nat Genet. 2008;40:616–22. doi: 10.1038/ng.109.  [DOI](https://doi.org/10.1038/ng.109) | [PMC free article](/articles/PMC2713680/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18385676/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Genome-wide%20association%20scan%20of%20tag%20SNPs%20identifies%20a%20susceptibility%20locus%20for%20lung%20cancer%20at%2015q25.1&author=CI%20Amos&volume=40&publication_year=2008&pages=616-22&pmid=18385676&doi=10.1038/ng.109&)

15. Hung RJ, et al. A susceptibility locus for lung cancer maps to nicotinic acetylcholine receptor subunit genes on 15q25. Nature. 2008;452:633–7. doi: 10.1038/nature06885.  [DOI](https://doi.org/10.1038/nature06885) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18385738/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=A%20susceptibility%20locus%20for%20lung%20cancer%20maps%20to%20nicotinic%20acetylcholine%20receptor%20subunit%20genes%20on%2015q25&author=RJ%20Hung&volume=452&publication_year=2008&pages=633-7&pmid=18385738&doi=10.1038/nature06885&)

16. Thorgeirsson TE, et al. A variant associated with nicotine dependence, lung cancer and peripheral arterial disease. Nature. 2008;452:638–42. doi: 10.1038/nature06846.  [DOI](https://doi.org/10.1038/nature06846) | [PMC free article](/articles/PMC4539558/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18385739/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=A%20variant%20associated%20with%20nicotine%20dependence,%20lung%20cancer%20and%20peripheral%20arterial%20disease&author=TE%20Thorgeirsson&volume=452&publication_year=2008&pages=638-42&pmid=18385739&doi=10.1038/nature06846&)

17. Huang YT, et al. Genome-wide analysis of survival in early-stage non-small-cell lung cancer. J Clin Oncol. 2009;27:2660–7. doi: 10.1200/JCO.2008.18.7906.  [DOI](https://doi.org/10.1200/JCO.2008.18.7906) | [PMC free article](/articles/PMC2690391/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19414679/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Genome-wide%20analysis%20of%20survival%20in%20early-stage%20non-small-cell%20lung%20cancer&author=YT%20Huang&volume=27&publication_year=2009&pages=2660-7&pmid=19414679&doi=10.1200/JCO.2008.18.7906&)

18. Li Y, et al. Genetic variants and risk of lung cancer in never smokers: a genome-wide association study. Lancet Oncol. 2010;11:321–30. doi: 10.1016/S1470-2045(10)70042-5.  [DOI](https://doi.org/10.1016/S1470-2045(10)70042-5) | [PMC free article](/articles/PMC2945218/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20304703/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet%20Oncol&title=Genetic%20variants%20and%20risk%20of%20lung%20cancer%20in%20never%20smokers:%20a%20genome-wide%20association%20study&author=Y%20Li&volume=11&publication_year=2010&pages=321-30&pmid=20304703&doi=10.1016/S1470-2045(10)70042-5&)

19. Wu X, et al. Genome-wide association study of survival in non-small cell lung cancer patients receiving platinum-based chemotherapy. J Natl Cancer Inst. 2011;103:817–25. doi: 10.1093/jnci/djr075.  [DOI](https://doi.org/10.1093/jnci/djr075) | [PMC free article](/articles/PMC3096796/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21483023/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Natl%20Cancer%20Inst&title=Genome-wide%20association%20study%20of%20survival%20in%20non-small%20cell%20lung%20cancer%20patients%20receiving%20platinum-based%20chemotherapy&author=X%20Wu&volume=103&publication_year=2011&pages=817-25&pmid=21483023&doi=10.1093/jnci/djr075&)

20. Li M, Li C, Guan W. Evaluation of coverage variation of SNP chips for genome-wide association studies. Eur J Hum Genet. 2008;16:635–43. doi: 10.1038/sj.ejhg.5202007.  [DOI](https://doi.org/10.1038/sj.ejhg.5202007) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18253166/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Hum%20Genet&title=Evaluation%20of%20coverage%20variation%20of%20SNP%20chips%20for%20genome-wide%20association%20studies&author=M%20Li&author=C%20Li&author=W%20Guan&volume=16&publication_year=2008&pages=635-43&pmid=18253166&doi=10.1038/sj.ejhg.5202007&)

21. Denzin LK, Sant’Angelo DB, Hammond C, Surman MJ, Cresswell P. Negative regulation by HLA-DO of MHC class II-restricted antigen processing. Science. 1997;278:106–9. doi: 10.1126/science.278.5335.106.  [DOI](https://doi.org/10.1126/science.278.5335.106) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9311912/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=Negative%20regulation%20by%20HLA-DO%20of%20MHC%20class%20II-restricted%20antigen%20processing&author=LK%20Denzin&author=DB%20Sant%E2%80%99Angelo&author=C%20Hammond&author=MJ%20Surman&author=P%20Cresswell&volume=278&publication_year=1997&pages=106-9&pmid=9311912&doi=10.1126/science.278.5335.106&)

22. Seliger B. Different regulation of MHC class I antigen processing components in human tumors. J Immunotoxicol. 2008;5:361–7. doi: 10.1080/15476910802482870.  [DOI](https://doi.org/10.1080/15476910802482870) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19404870/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Immunotoxicol&title=Different%20regulation%20of%20MHC%20class%20I%20antigen%20processing%20components%20in%20human%20tumors&author=B%20Seliger&volume=5&publication_year=2008&pages=361-7&pmid=19404870&doi=10.1080/15476910802482870&)

23. Chamuleau ME, Ossenkoppele GJ, van de Loosdrecht AA. MHC class II molecules in tumour immunology: prognostic marker and target for immune modulation. Immunobiology. 2006;211:619–25. doi: 10.1016/j.imbio.2006.05.005.  [DOI](https://doi.org/10.1016/j.imbio.2006.05.005) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16920500/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Immunobiology&title=MHC%20class%20II%20molecules%20in%20tumour%20immunology:%20prognostic%20marker%20and%20target%20for%20immune%20modulation&author=ME%20Chamuleau&author=GJ%20Ossenkoppele&author=AA%20van%20de%20Loosdrecht&volume=211&publication_year=2006&pages=619-25&pmid=16920500&doi=10.1016/j.imbio.2006.05.005&)

24. Souwer Y, et al. Detection of aberrant transcription of major histocompatibility complex class II antigen presentation genes in chronic lymphocytic leukaemia identifies HLA-DOA mRNA as a prognostic factor for survival. Br J Haematol. 2009;145:334–43. doi: 10.1111/j.1365-2141.2009.07625.x.  [DOI](https://doi.org/10.1111/j.1365-2141.2009.07625.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19245431/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Haematol&title=Detection%20of%20aberrant%20transcription%20of%20major%20histocompatibility%20complex%20class%20II%20antigen%20presentation%20genes%20in%20chronic%20lymphocytic%20leukaemia%20identifies%20HLA-DOA%20mRNA%20as%20a%20prognostic%20factor%20for%20survival&author=Y%20Souwer&volume=145&publication_year=2009&pages=334-43&pmid=19245431&doi=10.1111/j.1365-2141.2009.07625.x&)

25. Buddingh EP, et al. Chemotherapy-resistant osteosarcoma is highly susceptible to IL-15-activated allogeneic and autologous NK cells. Cancer Immunol Immunother. 2011;60:575–86. doi: 10.1007/s00262-010-0965-3.  [DOI](https://doi.org/10.1007/s00262-010-0965-3) | [PMC free article](/articles/PMC3061210/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21240486/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Immunol%20Immunother&title=Chemotherapy-resistant%20osteosarcoma%20is%20highly%20susceptible%20to%20IL-15-activated%20allogeneic%20and%20autologous%20NK%20cells&author=EP%20Buddingh&volume=60&publication_year=2011&pages=575-86&pmid=21240486&doi=10.1007/s00262-010-0965-3&)

26. Conejo-Garcia JR, et al. Ovarian Carcinoma Expresses the NKG2D Ligand Letal and Promotes the Survival and Expansion of CD28-Antitumor T Cells. Cancer Research. 2004;64:2175–82. doi: 10.1158/0008-5472.can-03-2194.  [DOI](https://doi.org/10.1158/0008-5472.can-03-2194) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15026360/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Research&title=Ovarian%20Carcinoma%20Expresses%20the%20NKG2D%20Ligand%20Letal%20and%20Promotes%20the%20Survival%20and%20Expansion%20of%20CD28-Antitumor%20T%20Cells&author=JR%20Conejo-Garcia&volume=64&publication_year=2004&pages=2175-82&pmid=15026360&doi=10.1158/0008-5472.can-03-2194&)

27. Le Maux Chansac B, et al. Potentiation of NK cell-mediated cytotoxicity in human lung adenocarcinoma: role of NKG2D-dependent pathway. Int Immunol. 2008;20:801–10. doi: 10.1093/intimm/dxn038.  [DOI](https://doi.org/10.1093/intimm/dxn038) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18441340/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20Immunol&title=Potentiation%20of%20NK%20cell-mediated%20cytotoxicity%20in%20human%20lung%20adenocarcinoma:%20role%20of%20NKG2D-dependent%20pathway&author=B%20Le%20Maux%20Chansac&volume=20&publication_year=2008&pages=801-10&pmid=18441340&doi=10.1093/intimm/dxn038&)

28. Zhang T, Sentman CL. Cancer immunotherapy using a bispecific NK receptor fusion protein that engages both T cells and tumor cells. Cancer Res. 2011;71:2066–76. doi: 10.1158/0008-5472.CAN-10-3200.  [DOI](https://doi.org/10.1158/0008-5472.CAN-10-3200) | [PMC free article](/articles/PMC3095211/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21282338/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=Cancer%20immunotherapy%20using%20a%20bispecific%20NK%20receptor%20fusion%20protein%20that%20engages%20both%20T%20cells%20and%20tumor%20cells&author=T%20Zhang&author=CL%20Sentman&volume=71&publication_year=2011&pages=2066-76&pmid=21282338&doi=10.1158/0008-5472.CAN-10-3200&)

29. Park MJ, Bae JH, Chung JS, Kim SH, Kang CD. Induction of NKG2D ligands and increased sensitivity of tumor cells to NK cell-mediated cytotoxicity by hematoporphyrin-based photodynamic therapy. Immunol Invest. 2011;40:367–82. doi: 10.3109/08820139.2010.551435.  [DOI](https://doi.org/10.3109/08820139.2010.551435) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21314289/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Immunol%20Invest&title=Induction%20of%20NKG2D%20ligands%20and%20increased%20sensitivity%20of%20tumor%20cells%20to%20NK%20cell-mediated%20cytotoxicity%20by%20hematoporphyrin-based%20photodynamic%20therapy&author=MJ%20Park&author=JH%20Bae&author=JS%20Chung&author=SH%20Kim&author=CD%20Kang&volume=40&publication_year=2011&pages=367-82&pmid=21314289&doi=10.3109/08820139.2010.551435&)

30. Asomaning K, et al. Second hand smoke, age of exposure and lung cancer risk. Lung Cancer. 2008;61:13–20. doi: 10.1016/j.lungcan.2007.11.013.  [DOI](https://doi.org/10.1016/j.lungcan.2007.11.013) | [PMC free article](/articles/PMC2515267/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18191495/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lung%20Cancer&title=Second%20hand%20smoke,%20age%20of%20exposure%20and%20lung%20cancer%20risk&author=K%20Asomaning&volume=61&publication_year=2008&pages=13-20&pmid=18191495&doi=10.1016/j.lungcan.2007.11.013&)

31. Loza MJ, McCall CE, Li L, Isaacs WB, Xu J, Chang BL. Assembly of inflammation-related genes for pathway-focused genetic analysis. PLoS One. 2007;2:e1035. doi: 10.1371/journal.pone.0001035.  [DOI](https://doi.org/10.1371/journal.pone.0001035) | [PMC free article](/articles/PMC2001184/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17940599/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=Assembly%20of%20inflammation-related%20genes%20for%20pathway-focused%20genetic%20analysis&author=MJ%20Loza&author=CE%20McCall&author=L%20Li&author=WB%20Isaacs&author=J%20Xu&volume=2&publication_year=2007&pages=e1035&pmid=17940599&doi=10.1371/journal.pone.0001035&)

32. de Bakker PI, Yelensky R, Pe’er I, Gabriel SB, Daly MJ, Altshuler D. Efficiency and power in genetic association studies. Nat Genet. 2005;37:1217–23. doi: 10.1038/ng1669.  [DOI](https://doi.org/10.1038/ng1669) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16244653/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Efficiency%20and%20power%20in%20genetic%20association%20studies&author=PI%20de%20Bakker&author=R%20Yelensky&author=I%20Pe%E2%80%99er&author=SB%20Gabriel&author=MJ%20Daly&volume=37&publication_year=2005&pages=1217-23&pmid=16244653&doi=10.1038/ng1669&)

33. Spitz MR, et al. Variants in Inflammation Genes Are Implicated in Risk of Lung Cancer in Never Smokers Exposed to Second-hand Smoke. Cancer Discov. 2011;1:10. doi: 10.1158/2159-8290.CD-11-0080.  [DOI](https://doi.org/10.1158/2159-8290.CD-11-0080) | [PMC free article](/articles/PMC3919666/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22586632/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Discov&title=Variants%20in%20Inflammation%20Genes%20Are%20Implicated%20in%20Risk%20of%20Lung%20Cancer%20in%20Never%20Smokers%20Exposed%20to%20Second-hand%20Smoke&author=MR%20Spitz&volume=1&publication_year=2011&pages=10&pmid=22586632&doi=10.1158/2159-8290.CD-11-0080&)

34. Wu X, et al. Genome-wide association study of survival in non-small cell lung cancer patients receiving platinum-based chemotherapy. J Natl Cancer Inst. 103:817–25. doi: 10.1093/jnci/djr075.  [DOI](https://doi.org/10.1093/jnci/djr075) | [PMC free article](/articles/PMC3096796/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21483023/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Natl%20Cancer%20Inst&title=Genome-wide%20association%20study%20of%20survival%20in%20non-small%20cell%20lung%20cancer%20patients%20receiving%20platinum-based%20chemotherapy&author=X%20Wu&volume=103&pages=817-25&pmid=21483023&doi=10.1093/jnci/djr075&)

35. Adzhubei IA, et al. A method and server for predicting damaging missense mutations. Nat Methods. 2010;7:248–9. doi: 10.1038/nmeth0410-248.  [DOI](https://doi.org/10.1038/nmeth0410-248) | [PMC free article](/articles/PMC2855889/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20354512/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Methods&title=A%20method%20and%20server%20for%20predicting%20damaging%20missense%20mutations&author=IA%20Adzhubei&volume=7&publication_year=2010&pages=248-9&pmid=20354512&doi=10.1038/nmeth0410-248&)

36. Kumar P, Henikoff S, Ng PC. Predicting the effects of coding non-synonymous variants on protein function using the SIFT algorithm. Nat Protoc. 2009;4:1073–81. doi: 10.1038/nprot.2009.86.  [DOI](https://doi.org/10.1038/nprot.2009.86) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19561590/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Protoc&title=Predicting%20the%20effects%20of%20coding%20non-synonymous%20variants%20on%20protein%20function%20using%20the%20SIFT%20algorithm&author=P%20Kumar&author=S%20Henikoff&author=PC%20Ng&volume=4&publication_year=2009&pages=1073-81&pmid=19561590&doi=10.1038/nprot.2009.86&)
