# A genome-wide association study identifies variants in KCNIP4 associated with ACE inhibitor-induced cough

## Metadata
**Authors:** J D Mosley, C M Shaffer, S L Van Driest, P E Weeke, Q S Wells, J H Karnes, D R Velez Edwards, W-Q Wei, P L Teixeira, L Bastarache, D C Crawford, R Li, T A Manolio, E P Bottinger, C A McCarty, J G Linneman, M H Brilliant, J A Pacheco, W Thompson, R L Chisholm, G P Jarvik, D R Crosslin, D S Carrell, E Baldwin, J Ralston, E B Larson, J Grafton, A Scrol, H Jouni, I J Kullo, G Tromp, K M Borthwick, H Kuivaniemi, D J Carey, M D Ritchie, Y Bradford, S S Verma, C G Chute, A Veluchamy, M K Siddiqui, C N A Palmer, A Doney, S H MahmoudPour, A H Maitland-van der Zee, A D Morris, J C Denny, D M Roden
**Journal:** The Pharmacogenomics Journal
**Date:** 2015 Jul 14
**DOI:** [10.1038/tpj.2015.51](https://doi.org/10.1038/tpj.2015.51)
**PMID:** 26169577
**PMCID:** PMC4713364
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4713364/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC4713364/pdf/tpj201551a.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC4713364/pdf/tpj201551a.pdf)

## Abstract

The most common side effect of angiotensin-converting enzyme inhibitor (ACEi) drugs is cough. We conducted a genome-wide association study (GWAS) of ACEi-induced cough among 7080 subjects of diverse ancestries in the Electronic Medical Records and Genomics (eMERGE) network. Cases were subjects diagnosed with ACEi-induced cough. Controls were subjects with at least 6 months of ACEi use and no cough. A GWAS (1595 cases and 5485 controls) identified associations on chromosome 4 in an intron of KCNIP4. The strongest association was at rs145489027 (minor allele frequency=0.33, odds ratio (OR)=1.3 (95% confidence interval (CI): 1.2–1.4), P=1.0 × 10−8). Replication for six single-nucleotide polymorphisms (SNPs) in KCNIP4 was tested in a second eMERGE population (n=926) and in the Genetics of Diabetes Audit and Research in Tayside, Scotland (GoDARTS) cohort (n=4309). Replication was observed at rs7675300 (OR=1.32 (1.01–1.70), P=0.04) in eMERGE and at rs16870989 and rs1495509 (OR=1.15 (1.01–1.30), P=0.03 for both) in GoDARTS. The combined association at rs1495509 was significant (OR=1.23 (1.15–1.32), P=1.9 × 10−9). These results indicate that SNPs in KCNIP4 may modulate ACEi-induced cough risk.

## Introduction

Angiotensin-converting enzyme inhibitor drugs (ACEi) are frequently used in clinical medicine for the treatment of elevated blood pressure, heart failure and renal protection in chronic kidney disease.^1^ Although this class of medications is generally well tolerated, adverse reactions may prevent their use in some individuals. The most common side effect is a persistent, nonproductive cough that can start within days to months after initiating therapy and requires cessation of ACEi use.^2^ Women are 1.5–2 times more likely to develop a cough than men.^3^ Some epidemiological features of ACEi-induced cough suggest a genetic predisposition to this adverse effect. In particular, there is racial variation in the incidence of cough, with the highest rates observed in east Asian populations where rates are 20–45%,^4^ as compared with European American populations where rates are approximately 10%.^5^

The mechanism of ACEi-induced cough is not certain. ACEis block the proteolytic enzyme angiotensin I-converting enzyme (*ACE*), which cleaves a number of target proteins, including angiotensin I, the primary mediator of the blood pressure-lowering effects, and pro-inflammatory kinins.^6^ One suspected mechanism of cough is the accumulation of these inflammatory kinins, which may sensitize vagal afferent fibers leading to a neurogenically mediated cough.^7^ There have been multiple candidate gene studies testing for associations between genetic variation in *ACE* and bradykinin pathway members and ACEi-induced cough. There has been intense focus on the association of cough with a common 287-bp insertion–deletion in the ACE gene.^3, 8, 9, 10, 11^ A meta-analysis of 11 of these studies only found a consistent association in a subgroup analysis of subjects aged >60 years.^12^ Significant single-nucleotide polymorphism (SNP) associations have also been reported between mediators in the bradykinin pathway. including the bradykinin B2 receptor (*BDKRB2*), membrane metallo-endopeptidase (*MME*), prostaglandin receptor E (*PTGER3*), neurokinin 2 receptor (*NK2R*) and *ACE.*^9, 13, 14, 15, 16^ However, these findings have not been consistently observed across studies.^3, 10, 17, 18^ Additional studies also suggest a role for SNPs located within the *ABO* gene, which have been shown to regulate the plasma ACE levels and have also been associated with ACEi cough.^10, 14, 19, 20^

In the present study, we used a genome-wide association study (GWAS) approach to carry out a comprehensive analysis of genetic determinants of ACEi-induced cough. Subjects were identified through the Electronic Medical Records and Genomics (eMERGE) network, a consortium of medical centers that utilize electronic medical records (EMR) as a tool for genomic research.^21^ Candidate SNPs were first identified using a multi-racial discovery cohort. The most significant candidate SNPs were then evaluated in two independent replication cohorts. We report here a significant signal at the *KCNIP4* locus indicating that variant potassium channel function or neuronal signaling contributes to the risk of a cough.

## Materials and methods

### Study population

The discovery study population comprised 7080 adult subjects collected from six sites participating in the phases I and II of eMERGE Network (Phase I: Vanderbilt University (VUMC), Marshfield Clinic, Northwestern University, Mayo Clinic and Group Health Research Institute; and Phase II: Geisinger Health System and Mount Sinai).^21, 22^ Subjects used for the discovery population were selected to maximize the representation across sites while minimizing the number of genotyping platforms represented. In addition, cases and controls genotyped on OMNI-QUAD platforms and that were part of the Vanderbilt Electronic Systems for Pharmacogenomic Assessment (VESPA) study that examined the genomics of drug response phenotypes were also included.^23^ For subgroup analyses, genetic ancestry assignment was determined using STRUCTURE^24^ in conjunction with 1917 ancestry informative markers (from the Illumina Test Panel^25^), with European ancestry defined as >90% probability of being in the CEU cluster and African ancestry defined as >70% probability of being in the YRI cluster, using a HapMap population as the reference ([Supplementary Figures S1–S3](#sup1)). Site-specific subject counts are shown in [Supplementary Table S1](#sup1).

### Phenotype data

The phenotype evaluated was cough attributable to the use of an ACEi. The algorithm identifying cases and controls was developed at VUMC and subsequently validated and deployed at the other eMERGE sites. The phenotype definition incorporated an iterative process whereby automated case and control assignment algorithms were validated against assignments made by manual review of the EMR.^26^ Phenotyping algorithms were refined until the positive predictive value reached the predesignated target of ⩾95%. In the final algorithms, cases were defined as subjects whose records contained either an ACEi drug name or ACEi class designator and ‘cough' on the same line within the structured ‘Allergy' section of the medical record. Hence, cases represent ACEi-induced cough recorded by a health-care provider. Controls were subjects who had an ACEi drug name or ACEi class on two medication listings with dates separated by at least 6 months and did not have a documented cough associated with ACEi use in the Allergy section. Complete details of the algorithm are available from PheKB ([http://phekb.org/phenotypes](http://phekb.org/phenotypes)). The mean positive predictive value for cases and controls in the VUMC and replication sites was 100% and 97.5%, respectively ([Supplementary Table S2](#sup1)).

Birth decade and sex were also extracted in addition to diagnosis codes (International Classification of Diseases (ICD)-9 codes^27^) related to comorbidities potentially contributive to the risk of cough, including asthma, postnasal drip, gastroesophageal reflux disease, bronchitis, emphysema, bronchiectasis, allergic alveolitis and chronic obstructive pulmonary disease. Lists of the ICD-9 codes used to define each of these conditions are shown in [Supplementary Table S3](#sup1). Subjects with ⩾1 ICD-9 codes for any of the conditions were considered to have the comorbidity. Smoking status, categorized as ‘Ever', ‘Never' or ‘Missing', was also extracted from either structured EMR data or validated algorithms.^28^

### Genotyping data

SNP genotype data were acquired on the Illumina HumanOmni1-Quad (Vanderbilt), HumanOmni5-Quad (VUMC), Human1M-Duov3_B (VUMC), HumanOmniExpress-12v1.0 (Geisinger), HumanOmniExpress (Mount Sinai) and Human660W-Quadv1_A (all other sites and VUMC) ([Supplementary Table S1](#sup1)). Genotyping data for each platform were individually cleaned. Quality control (QC) steps included identifying sex mismatches, SNPs failing concordance with HapMap, Mendelian errors and duplicate removal. After QC, a merged data set was created that contained 267 485 SNPs present on all platforms and with a call rate >98%. Cryptic relatedness was assessed on the merged platforms by identical-by-descent analysis, and one of a pair of subjects (*n*=343 total) more closely related than half-siblings was randomly excluded. Imputation was performed on the merged intersection data set using IMPUTE2^29^ in conjunction with the 1000 Genomes phase 3 reference panel for all populations. Prior to imputation, strand alignment between study and reference genotypes and prephasing of study genotypes into haplotypes was performed using SHAPEIT.^30^ Only those imputed SNPs with a genotype probability >90% were analyzed. SNPs with an Info score^29^ (measuring the average probabilities for a given SNP) <0.7 were excluded from analyses. The final analyses were restricted to 1 931 830 SNPs with a call rate >98%, a Hardy–Weinberg *P*>1 × 10^−6^ and minor allele frequency (MAF)>0.01. Principal components (PCs) fit to the preimputed SNP data set were computed using EIGENSTRAT^31^ to adjust for population structure.

The primary single SNP tests of association were performed using logistic regression assuming an additive genetic model, adjusting for 10 PCs, birth year and sex. A model that also incorporated binary covariates for smoking status (captured by two binary variables: ‘Ever smoked' and ‘Never smoked') and each of the comorbidities of asthma, postnatal drip, gastroesophageal reflux disease and lung disease (bronchitis, emphysema, bronchiectasis, allergic alveolitis and chronic obstructive pulmonary disease) was evaluated.

### Replication cohorts

SNPs with an association *P*-value <5x10^−6^ were evaluated in two replication populations. The first replication set comprised additional subjects available through the eMERGE network who had SNP genotyping as part of eMERGE Phase II in addition to 53 additional cases and controls from VUMC that were not included in the original analyses. Covariate and phenotype extraction from the EMR was as described above. Subjects were genotyped on the Affymetrix Human SNP Array 6.0 (Mount Sinai, Marshfield), Human660W-Quad (Marshfield), Human610-Quad (Mayo) and HumanOmniExpress (Group Health, Northwestern) platforms ([Supplementary Table S4](#sup1)). QC steps, as described above, were performed per the QC protocol established by the eMERGE Genomics Working Group.^32^ Each data set was then individually imputed by platform using IMPUTE2^29^ and the 1000 Genomes cosmopolitan reference panel.^33^ Imputed data were extracted and merged using the same QC filters as described above. The replication analyses were conducted in 157 cases and 769 controls that had >80% probability of being in the CEU cluster by STRUCTURE analyses. Multivariable analyses adjusting for PCs and covariates were performed, as described above. A replication *P*-value of 0.05 was considered statistically significant.

The second replication set was derived from the GoDARTS (Genetics of Diabetes Audit and Research in Tayside, Scotland) 2011 cohort and contained prescription information between 1990 and 2011 for 17 601 self-reported Caucasian diabetics and non-diabetics from the Tayside area. Cases were defined as subjects who switched from an ACEi to an angiotensin receptor blocker, a drug that targets a downstream receptor in the angiotensin–renin–aldosterone pathway. The positive predictive values of a possible and probable ACEi adverse drug reaction using this case definition were 90.5% and 68.3%, respectively. The most frequent adverse reactions associated with this case definition were cough and the rare reaction of angioedema. Controls were defined as those who had filled ACEi prescriptions within 9 months of the study's censor date or their date of death. Subjects who were ever concurrently on ACEis and angiotensin receptor blockers were also categorized as controls. A total of 710 cases and 3599 controls were available for analysis. Subjects were genotyped on the Affymetrix 6.0 (Affymetrix, Santa Clara, CA, USA) (381 cases and 1924 controls) or Illumina HumanOmniExpress (Illumina, San Diego, CA, USA) (329 cases and 1675 controls) platforms. Both platforms were imputed using IMPUTE2^29^ and the 1000 Genomes reference panel. SNPs deviating from Hardy–Weinberg equation (*P*<1 × 10^−6^) or with an Info Score<0.4 were excluded. An additive genetic model adjusting for age and sex was first computed separately for each genotyping platform and the results were meta-analyzed using GWAMA.^34^

### Data analysis

All QC analyses and GWA analyses were performed using PLINK v1.07.^35^ GoDARTS data were also analyzed using SNPTEST v2.5^36^ and GWAMA v2.1.^34^ All other analyses were performed using SAS v9.3 (SAS Institute, Cary, NC, USA). SNP data around the *KCNIP4* gene were visualized using LocusZoom^37^. The forest plot was generated using the Metafor package.^38^ The publicly available GTEx^39^, HaploReg v3^40^ and the NCBI expression quantitative trait locus (eQTL) databases were used to identify eQTLs and functional motifs associated with the most significant candidate SNPs.

### Ethics statement

The eMERGE study was approved by the Institutional Review Board at each site.^21, 41^ The GoDARTS study was approved by the Tayside Medical Ethics Committee and informed consent was obtained for all participants.

## Results

A total of 1595 cases of ACEi-induced cough and 5485 controls were analyzed in the discovery cohort ([Table 1](#tbl1)). The majority of subjects were of European ancestry, with about 15% of subjects belonging to other racial/ethnic groups. As compared with controls, the cases had a higher proportion of females (*P*<0.001), were younger, as measured by birth decade (*P*<0.001), more likely to have been smokers (*P*<0.001) and differed with respect to frequencies of diagnoses of gastroesophageal reflux disease (*P*<0.001), postnatal drip (*P*<0.001) and structural lung disease (*P*<0.001), all of which may contribute to a chronic cough. However, the directions of associations for these variables were not consistent, as smoking and lung disease were associated with decreased rates of ACEi cough, whereas the other variables were associated with increased rates.

### Table 1. Population characteristics of the discovery cohort.

|   | Cases, n (%) | Controls, n (%) | P-valuea |
| --- | --- | --- | --- |
| Male | 626 (39.3) | 3029 (55.2) | <0.001 |
| Female | 969 (60.7) | 2456 (44.8) |   |
|   |   |   |   |
| Genetic ancestry |  |  |  |
| European ancestry | 1346 (84.3) | 4661 (85.0) | <0.001 |
| African ancestry | 178 (11.2) | 701 (12.8) |   |
| Other | 71 (4.5) | 123 (2.2) |   |
|   |   |   |   |
| Birth decade (IQR) | 1930–1950 | 1920–1940 | <0.001 |
| Ever smoked | 515 (32.9) | 2403 (43.8) | <0.001b |
|   |   |   |   |
| History of |  |  |  |
| Asthma | 262 (16.4) | 882 (16.1) | 0.74 |
| GERD | 668 (41.9) | 2035 (37.1) | <0.001 |
| PND | 530 (33.2) | 1507 (27.5) | <0.001 |
| Other lung disease | 331 (20.8) | 1626 (29.7) | <0.001 |

Table 1 Caption: Abbreviations: GERD, gastroesophageal reflux disease; IQR, interqquartile range; PND, postnasal drip. aP-value obtained from chi-square analysis. bCalculation excludes 776 subjects (184 cases, 592 controls) with an unknown smoking history.

In multivariable regression analyses adjusting for PCs, sex and birth year, two SNPs located on chromosome 4 reached genome-wide significance ([Figure 1](#fig1), [Supplementary Figure S4](#sup1)). All of the SNPs with an association *P*-value<5x10^−6^ were exclusively located in intron 4 in the gene ‘Kv Channel Interacting Protein 4' (*KCNIP4*) ([Figure 2](#fig2) and [Supplementary Figure S5](#sup1)). The strongest association was in the imputed SNP rs145489027 (MAF=0.30, odds ratio (OR)=1.3 (95% confidence interval (CI): 1.2–1.4), *P*=1.0 × 10^−8^; [Table 2](#tbl2)). When results were stratified by site, there was a consistent direction of association across sites, with ORs ranging from 1.1 to 1.5 ([Supplementary Table S5](#sup1)). When the analyses were additionally adjusted for cough risk factors, the association at the SNP remained significant (OR=1.3 (95% CI: 1.2–1.4), *P*=1.0 × 10^−8^; [Supplementary Figure S6](#sup1)). Similar results were observed when subjects with lung disease were excluded (OR=1.3 (95% CI: 1.2–1.5), *P*=3.0x10^−8^; [Supplementary Figure S7](#sup1)). A regression model adjusting for the rs145489027 SNP eliminated the associations (*P*>0.05) for the other most significant SNPs, indicating that the SNPs represent a common signal (the *r*^2^ between this SNP and other most significant SNPs was 0.68 for rs7661530 and rs6838116 and 0.98 for rs7675300, rs16870989 and rs1495509 in a 1000G European ancestry population). In a subset analysis of African Americans and European Americans, this SNP was most strongly associated with cough in European Americans (MAF=0.33, OR=1.3 (95% CI: 1.2–1.4), *P*=2.5 × 10^−7^; [Table 3](#tbl3)). Although the direction of the effect was similar in African Americans, these associations were not significant in this smaller subset of subjects (*n*=879). ACEi-induced cough has been observed to occur more frequently in women^3^. In analyses stratified by sex, ORs were similar for men (OR=1.3 (95% CI: 1.2–1.5), *P*=1.4 × 10^−4^) and women (OR=1.3 (95% CI: 1.1–1.5), *P*=1.1 × 10^−5^). A search of publicly available data sets did not identify eQTLs or functional motifs associated the most significant SNPs. We specifically examined the associations using all SNPs located in 16 previously reported candidate genes (see [Supplementary Materials](#sup1)). No significant associations (*P*<0.05) were observed after applying a Bonferroni correction adjusting for multiple testing or in a subset analysis of older subjects born before 1960.

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a791/4879182/046ee8b915ad/tpj201551f1.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4879182_tpj201551f1.jpg)

Manhattan plot of genotyped single-nucleotide polymorphisms associated with angiotensin-converting enzyme inhibitor-induced cough using an additive model adjusted for 10 principal components, age, sex, history of asthma, reflux, postnasal drip and lung disease. The red line indicates the genome-wide significance threshold of alpha=5 × 10−8.

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a791/4879182/56179d7fa408/tpj201551f2.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4879182_tpj201551f2.jpg)

LocusZoom plot of most strongly associated single-nucleotide polymorphisms (SNPs) from genome-wide association study located in the region of KCNIP4 (chr4:20330238–22350374), centered around SNP rs145489027 (shown in purple). Linkage disequilibrium (based on r2 values) with respect to rs145489027 are based on the CEU reference population. Imputed SNPs are denoted by squares and genotyped SNPs by circles.

### Table 2. Most significantly associated SNPs.

| SNP | Minor allele | MAF | OR | 95% CI | P-value |
| --- | --- | --- | --- | --- | --- |
| Discoverya |  |  |  |  |  |
| rs7661530 | T | 0.33 | 1.27 | (1.17–1.39) | 6.1 × 10−8 |
| rs6838116 | A | 0.33 | 1.28 | (1.17–1.40) | 3.1 × 10−8 |
| rs7675300 | A | 0.31 | 1.27 | (1.16–1.39) | 1.1 × 10−7 |
| rs16870989 | A | 0.31 | 1.27 | (1.16–1.39) | 1.1 × 10−7 |
| rs145489027 | A | 0.30 | 1.30 | (1.19–1.42) | 1.0 × 10−8 |
| rs1495509 | C | 0.31 | 1.27 | (1.17–1.39) | 7.8 × 10−8 |
|   |   |   |   |   |   |
| Replication |  |  |  |  |  |
| eMERGEb |  |  |  |  |  |
| rs7661530 | T | 0.35 | 1.24 | (0.96–1.59) | 0.09 |
| rs7675300 | A | 0.33 | 1.32 | (1.01–1.70) | 0.04 |
| rs16870989 | A | 0.33 | 1.28 | (0.98–1.65) | 0.06 |
| rs145489027 | A | 0.32 | 1.28 | (0.98–1.68) | 0.07 |
| rs1495509 | C | 0.33 | 1.28 | (0.98–1.65) | 0.06 |
| GoDARTSc |   |   |   |   |   |
| rs7661530 | T | 0.37 | 1.05 | (0.89–1.23) | 0.84 |
| rs6838116 | A | 0.37 | 1.08 | (0.95–1.20) | 0.57 |
| rs16870989 | A | 0.34 | 1.15 | (1.01–1.30) | 0.03 |
| rs145489027 | A | 0.33 | 1.12 | (0.99–1.27) | 0.08 |
| rs1495509 | C | 0.34 | 1.15 | (1.01–1.30) | 0.03 |
|   |   |   |   |   |   |
| Meta-analysis |  |  |  |  |  |
| rs7661530 | T | — | 1.22 | (1.13–1.31) | 9.6 × 10−8 |
| rs6838116d | A | — | 1.17 | (1.10–1.24) | 4.2 × 10−7 |
| rs7675300d | A | — | 1.27 | (1.17–1.39) | 1.4 × 10−8 |
| rs16870989 | A | — | 1.23 | (1.15–1.32) | 4.9 × 10−9 |
| rs145489027 | A | — | 1.24 | (1.16–1.33) | 2 × 10−9 |
| rs1495509 | C | — | 1.23 | (1.15–1.32) | 1.9 × 10−9 |

Table 2 Caption: Abbreviations: CI, confidence interval; eMERGE, Electronic Medical Records and Genomics; GoDARTS, Genetics of Diabetes Audit and Research in Tayside, Scotland; MAF, minor allele frequency; OR, odds ratio; SNP, single-nucleotide polymorphism. aFrom a multivariable logistic regression assuming an additive genetic model and adjusted for 10 principal components (PCs), birth year and sex. bFrom a multivariable logistic regression (157 cases and 769 controls) assuming an additive genetic model and adjusted for 3 PCs, birth year and sex. cFrom a meta-analysis (710 cases and 3,599 controls) adjusted for age and sex. The P-value for the Q-score P-value was >0.05 for each SNP. dBased on only two data sets.

### Table 3. Association analyses by racial groups for SNP rs145489027.

| Racial group | Cases | Controls | MAF cases | MAF controls | OR | 95% CI | P-valuea |
| --- | --- | --- | --- | --- | --- | --- | --- |
| European ancestry | 1346 | 4661 | 0.37 | 0.32 | 1.3 | (1.2–1.4) | 2.5 × 10−7 |
| African ancestry | 178 | 701 | 0.13 | 0.10 | 1.2 | (0.9–1.8) | 0.24 |
| Otherb | 71 | 123 | 0.29 | 0.20 | 1.4 | (0.8–2.6) | 0.25 |

Table 3 Caption: Abbreviations: CI, confidence interval; MAF, minor allele frequency; OR, odds ratio; SNP, single-nucleotide polymorphism. aFrom a multivariable logistic regression assuming an additive genetic model and adjusting for 10 principal components, birth year and sex. bIncludes subjects not meeting criteria for inclusion with African ancestry and European ancestry.

We sought replication for the SNPs in *KCNIP4* with an association *P*-value <5 × 10^−6^ in two independent data sets ([Supplementary Tables S6 and S7](#sup1)). Of the six SNPs identified in the discovery cohort, five were available in each of the final QC'ed eMERGE and GoDARTS replication sets. In both replication sets, the MAFs and association statistics were similar to those observed in the European ancestry subjects in the discovery population ([Table 2](#tbl2)). The most significantly associated SNP from the discovery analysis (rs145489027) did not reach statistical significance in either replication population ([Table 2](#tbl2)). One SNP (rs7675300) was significantly associated with ACEi cough in the eMERGE set, (OR=1.32 (1.01–1.70), *P*=0.04). Two SNPs were significantly associated in the GoDARTS set: rs16870989 (OR=1.15 (1.01–1.30), *P*=0.03) and rs1495509 (OR=1.15 (1.01–1.30), *P*=0.03). In a meta-analysis performed using both replication sets, three of the four SNPs present in both sets had an association at *P*<0.05 (rs16870989: OR=1.17 (1.03–1.34), *P*=0.009; rs145489027: OR=1.15 (1.01–1.31), *P*=0.02); and rs1495509: OR=1.17 (1.03–1.34), *P*=0.009); [Supplementary Table S8](#sup1)). In a meta-analysis across all discovery and replication populations, four of the most significant SNPs reached genome-wide significance ([Table 2](#tbl2)) with the non-imputed SNP rs1495509 (highlighted in [Figure 2](#fig2)), showing the strongest association ([Figure 3](#fig3)). The combined association statistic across the replication sets at this SNP was (OR=1.23 (1.15–1.32), *P*=1.9 × 10^−9^) and did not demonstrate significant heterogeneity across studies (*P*=0.42).

### Figure 3.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a791/4879182/cb13995f5187/tpj201551f3.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4879182_tpj201551f3.jpg)

Forest plot for the association between angiotensin-converting enzyme inhibitor cough and KCNIP4 intronic single-nucleotide polymorphism rs1495509 for the discovery (eMERGE (Electronic Medical Records and Genomics) Network) and replication (eMERGE and GoDARTS (Genetics of Diabetes Audit and Research in Tayside, Scotland)) data sets. There was no heterogeneity across studies (P=0.43). CI, confidence interval.

## Discussion

We describe the first large GWAS investigating SNP variants associated with clinically diagnosed ACEi-associated cough. All phenotype and genotype data for this study were derived from clinical research settings that incorporate an EMR data system. We found significant associations in a set of intronic SNPs located within the gene *KCNIP4*. Several of these significant associations were independently replicated in two European ancestry populations. In summary, these analyses identify a novel candidate gene that may have a role in this common adverse reaction.

The SNPs showing the strongest association with the cough phenotype are located exclusively in intron 4 of *KCNIP4*. *KCNIP4* is a member of the KChIP family of EF hand (helix-loop-helix)-containing calcium-binding proteins. A major function attributed to *KCNIP4* is the regulation of Kv4 potassium channels, which are significant contributors to action potential activity in neurons and cardiac myocytes.^42^ The amino terminus of the *KCNIP4* gene product undergoes extensive alternative splicing and at least six splice variants have been described.^43^ Of note, several of these splice variants remove the flanking exons surrounding the intronic region identified in these analyses. Alternative splicing has been shown to have functional significance and can result in modulation of Kv4 channel functions and their subcellular location.^42, 44^ *KCNIP4* isoforms are predominantly expressed in neuronal structures in the brain and spinal cord, though some isoforms are found in the kidney, stomach and small intestine.^43^ Little-to-no expression of the gene has been observed in lung extracts.^45, 46^

The most strongly associated SNPs are located within a single intron of *KCNIP4*. These SNPs are not in linkage disequilibrium with an amino-acid changing variant that would alter the primary structure of the protein. Hence, it is more likely that these SNPs have a regulatory role likely related to mRNA splicing or expression. A leading hypothesized mechanism of the ACEi-induced cough is stimulation of sensory nerve afferents within the lung resulting from the accumulation of inflammatory mediators that are normally cleaved by the ACE enzyme.^47^ This hypothesis has served as the basis for candidate gene studies that have focused on variation in inflammatory pathways within the lung. Our results would suggest that the important source of this variation may be directly related to the sensory nerves themselves, as *KCNIP4* has been found to be in both central and peripheral neuronal structures. Indeed, if *KCNIP4* expression in the lung were restricted to sensory nerves, its protein and mRNA levels would be expected to be low in samples derived from lung whole-cell homogenates, especially if samples were taken from the lung periphery. This restricted expression pattern could account for why *KCNIP4* has not been detected in the lung. In further support of a role for *KCNIP4* in lung physiology, a GWAS in mice identified an association between *KCNIP4* and airway hyper-responsiveness, which was confirmed in studies of human asthma and airway hyper-reactivity.^48^

Epidemiological studies have identified several factors associated with an increased risk of ACEi-induced cough, including sex and race. In particular, the prevalence of cough is higher among women and east Asian populations.^3, 4^ In our replication set, the allele frequencies and effect sizes in subjects of European ancestry were generally comparable to those observed in the discovery sets. The ORs were weaker in the GoDARTS replication set, which is likely attributable to the lower positive predictive value and specificity of the case definition (switching from an ACEi medication to an angiotensin receptor blocker) used in this cohort.^49, 50^ Among African Americans, the associated alleles had lower MAFs. Although the association statistics trended in the same direction, they did not reach statistical significance. However, there was only 5% power to replicate the association in this group owing to small numbers of subjects and lower allele frequencies. Analyses across races suggest that this genomic region around *KCNIP4* might be a trans-population risk factor and allele frequencies may contribute to prevalence differences in ACEi-induced cough. However, a comparison of the MAFs for the most significantly associated SNPs among HapMap European and Asian populations show that the minor alleles are very similar in these groups. Hence, racial differences in the frequencies of these SNPs would not account for the prevalence differences between these populations and would suggest that other independent or modifying genetic factors may be contributing to racial differences.

There are several limitations to this study. Cases and controls were identified using the EMR data sources. A limitation of EMR data is that data collection is not systematic and can be incomplete. These limitations can lead to both differential and nondifferential misclassification, which can skew or weaken associations. For instance, subjects who had been switched from ACEi therapy due to cough prior to the time period captured by the EMR could be inappropriately assigned to the control group. In addition, information pertaining to ACEi dosage, treatment duration and indication could not be systematically extracted for analysis. Hence, the contributions of these factors to ACEi cough could not be evaluated. We were also unable to incorporate elements of ACEi cough phenotype definitions that have been used in some studies, such as evaluating that effect of cessation of an ACEi on cough at a fixed time interval,^18^ as these protocols are not standard in clinical practice. The study also had limited power to detect associations for SNPs with an MAF<10%. Hence, the contribution of low frequency variants to the phenotype could not be quantified. Genotyping was performed using a number of commercial SNP-genotyping platforms. Imputation across multiple genotyping platforms can give rise to systematic frequency differences, which can lead to inflated type I error in GWAS studies.^51^ This problem is exacerbated when there are case and control imbalances across platforms. To attenuate this bias, the genotype data in the discovery set were imputed from an intersection of SNPs across each platform being evaluated, an approach that has been shown to decrease type I error rates.^52^ We did not demonstrate a functional role for *KCNIP4* in the cough phenotype. Hence, it is possible the SNPs we identified are involved in the regulation of a nearby gene unrelated to *KCNIP4*, as has been reported for a number of other SNPs located within or in close proximity of a gene.^53, 54^ Epidemiological studies have observed higher rates of ACEi-induced cough in some Asian populations. This study did not assess the risks of the candidate SNPs in this population.

In conclusion, we used a GWAS to identify SNP variants associated with ACEi-induced cough using data derived from eMERGE, a network of medical centers that utilize electronic medical records as a tool for genomic research. We identified SNPs in the intron of the gene *KCNIP4* as potential candidates for this adverse reaction. The mechanisms by which *KCNIP4* may contribute to the cough are not known, and functional studies are needed to further elucidate the pathophysiological mechanisms.

## Acknowledgments

This work was supported by the VUMC Clinical Pharmacology Training grant (T32 GM07569), the electronic MEdical Records and GEnomics (eMERGE) Network U01-HG-04603 (Vanderbilt), 1U02HG004608-01, 1U01HG006389 and NCATS/NIH grant UL1TR000427 (Marshfield/EIRH/Penn State), U01HG006375 and U01AG06781 (Group Health and University of Washington), U01-HG04599 (Mayo Clinic), U01-HG004609 (Northwestern University), U01HG006382 (Geisinger), an ARRA grant RC2 GM092618, R01 LM 010685, a Vanderbilt PGRN grant U19 HL065962 and the Vanderbilt CTSA grant UL1 TR000445 from NCATS/NIH. At Geisinger, the sample collection was supported by NIH (P30DK072488, R01DK088231 and R01DK091601), Pennsylvania Commonwealth Universal Research Enhancement Program, the Ben Franklin Technology Development Fund of PA, the Geisinger Clinical Research Fund and a Grant-In-Aid from the American Heart Association. GoDARTS was funded by the Wellcome Trust (084727/Z/08/Z, 085475/Z/08/Z, 085475/B/08/Z) and as part of the EU IMI-SUMMIT program.

The authors declare no conflict of interest.

## Footnotes

## Supplementary Material

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Zaman MA, Oparil S, Calhoun DA. Drugs targeting the renin-angiotensin-aldosterone system. Nat Rev Drug Discov 2002; 1: 621–636.  [DOI](https://doi.org/10.1038/nrd873) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12402502/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Drug%20Discov&title=Drugs%20targeting%20the%20renin-angiotensin-aldosterone%20system&volume=1&publication_year=2002&pages=621&pmid=12402502&doi=10.1038/nrd873&)

2. Overlack A. ACE inhibitor-induced cough and bronchospasm. Incidence, mechanisms and management. Drug Saf 1996; 15: 72–78.  [DOI](https://doi.org/10.2165/00002018-199615010-00006) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8862965/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Saf&title=ACE%20inhibitor-induced%20cough%20and%20bronchospasm.%20Incidence,%20mechanisms%20and%20management&volume=15&publication_year=1996&pages=72&pmid=8862965&doi=10.2165/00002018-199615010-00006&)

3. Lee YJ, Tsai JC. Angiotensin-converting enzyme gene insertion/deletion, not bradykinin B2 receptor -58 T/C gene polymorphism, associated with angiotensin-converting enzyme inhibitor-related cough in Chinese female patients with non-insulin-dependent diabetes mellitus. Metab Clin Exp 2001; 50: 1346–1350.  [DOI](https://doi.org/10.1053/meta.2001.27212) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11699055/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Metab%20Clin%20Exp&title=Angiotensin-converting%20enzyme%20gene%20insertion/deletion,%20not%20bradykinin%20B2%20receptor%20-58%E2%80%89T/C%20gene%20polymorphism,%20associated%20with%20angiotensin-converting%20enzyme%20inhibitor-related%20cough%20in%20Chinese%20female%20patients%20with%20non-insulin-dependent%20diabetes%20mellitus&volume=50&publication_year=2001&pages=1346&pmid=11699055&doi=10.1053/meta.2001.27212&)

4. Woo KS, Nicholls MG. High prevalence of persistent cough with angiotensin converting enzyme inhibitors in Chinese. Br J Clin Pharmacol 1995; 40: 141–144.  [PMC free article](/articles/PMC1365173/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8562296/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Clin%20Pharmacol&title=High%20prevalence%20of%20persistent%20cough%20with%20angiotensin%20converting%20enzyme%20inhibitors%20in%20Chinese&volume=40&publication_year=1995&pages=141&pmid=8562296&)

5. Wood R. Bronchospasm and cough as adverse reactions to the ACE inhibitors captopril, enalapril and lisinopril. A controlled retrospective cohort study. Br J Clin Pharmacol 1995; 39: 265–270.  [DOI](https://doi.org/10.1111/j.1365-2125.1995.tb04447.x) | [PMC free article](/articles/PMC1365002/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7619667/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Clin%20Pharmacol&title=Bronchospasm%20and%20cough%20as%20adverse%20reactions%20to%20the%20ACE%20inhibitors%20captopril,%20enalapril%20and%20lisinopril.%20A%20controlled%20retrospective%20cohort%20study&volume=39&publication_year=1995&pages=265&pmid=7619667&doi=10.1111/j.1365-2125.1995.tb04447.x&)

6. Bernstein KE, Ong FS, Blackwell W-LB, Shah KH, Giani JF, Gonzalez-Villalobos RA et al. A modern understanding of the traditional and nontraditional biological functions of angiotensin-converting enzyme. Pharmacol Rev 2013; 65: 1–46.  [DOI](https://doi.org/10.1124/pr.112.006809) | [PMC free article](/articles/PMC3565918/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23257181/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacol%20Rev&title=A%20modern%20understanding%20of%20the%20traditional%20and%20nontraditional%20biological%20functions%20of%20angiotensin-converting%20enzyme&volume=65&publication_year=2013&pages=1&pmid=23257181&doi=10.1124/pr.112.006809&)

7. Fox AJ, Lalloo UG, Belvisi MG, Bernareggi M, Chung KF, Barnes PJ. Bradykinin-evoked sensitization of airway sensory nerves: a mechanism for ACE-inhibitor cough. Nat Med 1996; 2: 814–817.  [DOI](https://doi.org/10.1038/nm0796-814) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8673930/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Med&title=Bradykinin-evoked%20sensitization%20of%20airway%20sensory%20nerves:%20a%20mechanism%20for%20ACE-inhibitor%20cough&volume=2&publication_year=1996&pages=814&pmid=8673930&doi=10.1038/nm0796-814&)

8. Furuya K, Yamaguchi E, Hirabayashi T, Itoh A, Hizawa N, Ohnuma N et al. Angiotensin-I-converting enzyme gene polymorphism and susceptibility to cough. Lancet 1994; 343: 354.  [DOI](https://doi.org/10.1016/s0140-6736(94)91190-8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7905161/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Angiotensin-I-converting%20enzyme%20gene%20polymorphism%20and%20susceptibility%20to%20cough&volume=343&publication_year=1994&pages=354&pmid=7905161&doi=10.1016/s0140-6736(94)91190-8&)

9. Mukae S, Itoh S, Aoki S, Iwata T, Nishio K, Sato R et al. Association of polymorphisms of the renin-angiotensin system and bradykinin B2 receptor with ACE-inhibitor-related cough. J Hum Hypertens 2002; 16: 857–863.  [DOI](https://doi.org/10.1038/sj.jhh.1001486) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12522467/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Hum%20Hypertens&title=Association%20of%20polymorphisms%20of%20the%20renin-angiotensin%20system%20and%20bradykinin%20B2%20receptor%20with%20ACE-inhibitor-related%20cough&volume=16&publication_year=2002&pages=857&pmid=12522467&doi=10.1038/sj.jhh.1001486&)

10. Woo SW, Bang S, Chung MW, Jin SK, Kim YS, Lee SH. Lack of association between ACE and bradykinin B2 receptor gene polymorphisms and ACE inhibitor-induced coughing in hypertensive Koreans. J Clin Pharm Ther 2009; 34: 561–567.  [DOI](https://doi.org/10.1111/j.1365-2710.2009.01028.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19744011/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Pharm%20Ther&title=Lack%20of%20association%20between%20ACE%20and%20bradykinin%20B2%20receptor%20gene%20polymorphisms%20and%20ACE%20inhibitor-induced%20coughing%20in%20hypertensive%20Koreans&volume=34&publication_year=2009&pages=561&pmid=19744011&doi=10.1111/j.1365-2710.2009.01028.x&)

11. Takahashi T, Yamaguchi E, Furuya K, Kawakami Y. The ACE gene polymorphism and cough threshold for capsaicin after cilazapril usage. Respir Med 2001; 95: 130–135.  [DOI](https://doi.org/10.1053/rmed.2000.1005) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11217909/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Respir%20Med&title=The%20ACE%20gene%20polymorphism%20and%20cough%20threshold%20for%20capsaicin%20after%20cilazapril%20usage&volume=95&publication_year=2001&pages=130&pmid=11217909&doi=10.1053/rmed.2000.1005&)

12. Li Y-F, Zhu X-M, Liu F, Xiao C-S, Bian Y-F, Li H et al. Angiotensin-converting enzyme (ACE) gene insertion/deletion polymorphism and ACE inhibitor-related cough: a meta-analysis. PLoS One 2012; 7: e37396.  [DOI](https://doi.org/10.1371/journal.pone.0037396) | [PMC free article](/articles/PMC3378563/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22723835/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=Angiotensin-converting%20enzyme%20(ACE)%20gene%20insertion/deletion%20polymorphism%20and%20ACE%20inhibitor-related%20cough:%20a%20meta-analysis&volume=7&publication_year=2012&pages=e37396&pmid=22723835&doi=10.1371/journal.pone.0037396&)

13. Grilo A, Sáez-Rosas MP, Santos-Morano J, Sánchez E, Moreno-Rey C, Real LM et al. Identification of genetic factors associated with susceptibility to angiotensin-converting enzyme inhibitors-induced cough. Pharmacogenet Genomics 2011; 21: 10–17.  [DOI](https://doi.org/10.1097/FPC.0b013e328341041c) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21052031/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Identification%20of%20genetic%20factors%20associated%20with%20susceptibility%20to%20angiotensin-converting%20enzyme%20inhibitors-induced%20cough&volume=21&publication_year=2011&pages=10&pmid=21052031&doi=10.1097/FPC.0b013e328341041c&)

14. Mas S, Gassò P, Alvarez S, Ortiz J, Sotoca JM, Francino A et al. Pharmacogenetic predictors of angiotensin-converting enzyme inhibitor-induced cough: the role of ACE, ABO, and BDKRB2 genes. Pharmacogenet Genomics 2011; 21: 531–538.  [DOI](https://doi.org/10.1097/FPC.0b013e328348c6db) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21832968/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Pharmacogenetic%20predictors%20of%20angiotensin-converting%20enzyme%20inhibitor-induced%20cough:%20the%20role%20of%20ACE,%20ABO,%20and%20BDKRB2%20genes&volume=21&publication_year=2011&pages=531&pmid=21832968&doi=10.1097/FPC.0b013e328348c6db&)

15. Mukae S, Aoki S, Itoh S, Iwata T, Ueda H, Katagiri T. Bradykinin B(2) receptor gene polymorphism is associated with angiotensin-converting enzyme inhibitor-related cough. Hypertension 2000; 36: 127–131.  [DOI](https://doi.org/10.1161/01.hyp.36.1.127) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10904024/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hypertension&title=Bradykinin%20B(2)%20receptor%20gene%20polymorphism%20is%20associated%20with%20angiotensin-converting%20enzyme%20inhibitor-related%20cough&volume=36&publication_year=2000&pages=127&pmid=10904024&doi=10.1161/01.hyp.36.1.127&)

16. Kim T-B, Oh S-Y, Park H-K, Jeon S-G, Chang Y-S, Lee K-Y et al. Polymorphisms in the neurokinin-2 receptor gene are associated with angiotensin-converting enzyme inhibitor-induced cough. J Clin Pharm Ther 2009; 34: 457–464.  [DOI](https://doi.org/10.1111/j.1365-2710.2008.01018.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19583679/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Pharm%20Ther&title=Polymorphisms%20in%20the%20neurokinin-2%20receptor%20gene%20are%20associated%20with%20angiotensin-converting%20enzyme%20inhibitor-induced%20cough&volume=34&publication_year=2009&pages=457&pmid=19583679&doi=10.1111/j.1365-2710.2008.01018.x&)

17. Mahmoudpour SH, Leusink M, Putten L, van der, Terreehorst I, Asselbergs FW, de Boer A et al. Pharmacogenetics of ACE inhibitor-induced angioedema and cough: a systematic review and meta-analysis. Pharmacogenomics 2013; 14: 249–260.  [DOI](https://doi.org/10.2217/pgs.12.206) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23394388/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Pharmacogenetics%20of%20ACE%20inhibitor-induced%20angioedema%20and%20cough:%20a%20systematic%20review%20and%20meta-analysis&volume=14&publication_year=2013&pages=249&pmid=23394388&doi=10.2217/pgs.12.206&)

18. Zee RY, Rao VS, Paster RZ, Sweet CS, Lindpaintner K. Three candidate genes and angiotensin-converting enzyme inhibitor-related cough: a pharmacogenetic analysis. Hypertension 1998; 31: 925–928.  [DOI](https://doi.org/10.1161/01.hyp.31.4.925) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9535416/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hypertension&title=Three%20candidate%20genes%20and%20angiotensin-converting%20enzyme%20inhibitor-related%20cough:%20a%20pharmacogenetic%20analysis&volume=31&publication_year=1998&pages=925&pmid=9535416&doi=10.1161/01.hyp.31.4.925&)

19. Terao C, Bayoumi N, McKenzie CA, Zelenika D, Muro S, Mishima M et al. Quantitative variation in plasma angiotensin-I converting enzyme activity shows allelic heterogeneity in the ABO blood group locus. Ann Hum Genet 2013; 77: 465–471.  [DOI](https://doi.org/10.1111/ahg.12034) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23937567/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Hum%20Genet&title=Quantitative%20variation%20in%20plasma%20angiotensin-I%20converting%20enzyme%20activity%20shows%20allelic%20heterogeneity%20in%20the%20ABO%20blood%20group%20locus&volume=77&publication_year=2013&pages=465&pmid=23937567&doi=10.1111/ahg.12034&)

20. Luo J-Q, He F-Z, Luo Z-Y, Wen J-G, Wang L-Y, Sun N-L et al. Rs495828 polymorphism of the ABO gene is a predictor of enalapril-induced cough in Chinese patients with essential hypertension. Pharmacogenet Genomics 2014; 24: 306–313.  [DOI](https://doi.org/10.1097/FPC.0000000000000050) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24743543/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Rs495828%20polymorphism%20of%20the%20ABO%20gene%20is%20a%20predictor%20of%20enalapril-induced%20cough%20in%20Chinese%20patients%20with%20essential%20hypertension&volume=24&publication_year=2014&pages=306&pmid=24743543&doi=10.1097/FPC.0000000000000050&)

21. Gottesman O, Kuivaniemi H, Tromp G, Faucett WA, Li R, Manolio TA et al. The Electronic Medical Records and Genomics (eMERGE) Network: past, present, and future. Genet Med 2013; 15: 761–771.  [DOI](https://doi.org/10.1038/gim.2013.72) | [PMC free article](/articles/PMC3795928/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23743551/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genet%20Med&title=The%20Electronic%20Medical%20Records%20and%20Genomics%20(eMERGE)%20Network:%20past,%20present,%20and%20future&volume=15&publication_year=2013&pages=761&pmid=23743551&doi=10.1038/gim.2013.72&)

22. Crawford DC, Crosslin DR, Tromp G, Kullo IJ, Kuivaniemi H, Hayes MG et al. eMERGEing progress in genomics-the first seven years. Front Genet 2014; 5: 184.  [DOI](https://doi.org/10.3389/fgene.2014.00184) | [PMC free article](/articles/PMC4060012/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24987407/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front%20Genet&title=eMERGEing%20progress%20in%20genomics-the%20first%20seven%20years&volume=5&publication_year=2014&pages=184&pmid=24987407&doi=10.3389/fgene.2014.00184&)

23. Bowton E, Field JR, Wang S, Schildcrout JS, Van Driest SL, Delaney JT et al. Biobanks and electronic medical records: enabling cost-effective research. Sci Transl Med 2014; 6: 234cm3.  [DOI](https://doi.org/10.1126/scitranslmed.3008604) | [PMC free article](/articles/PMC4226414/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24786321/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Sci%20Transl%20Med&title=Biobanks%20and%20electronic%20medical%20records:%20enabling%20cost-effective%20research&volume=6&publication_year=2014&pages=234cm3&pmid=24786321&doi=10.1126/scitranslmed.3008604&)

24. Pritchard JK, Stephens M, Donnelly P. Inference of population structure using multilocus genotype data. Genetics 2000; 155: 945–959.  [DOI](https://doi.org/10.1093/genetics/155.2.945) | [PMC free article](/articles/PMC1461096/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10835412/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genetics&title=Inference%20of%20population%20structure%20using%20multilocus%20genotype%20data&volume=155&publication_year=2000&pages=945&pmid=10835412&doi=10.1093/genetics/155.2.945&)

25. Dumitrescu L, Ritchie MD, Brown-Gentry K, Pulley JM, Basford M, Denny JC et al. Assessing the accuracy of observer-reported ancestry in a biorepository linked to electronic medical records. Genet Med 2010; 12: 648–650.  [DOI](https://doi.org/10.1097/GIM.0b013e3181efe2df) | [PMC free article](/articles/PMC2952033/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20733501/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genet%20Med&title=Assessing%20the%20accuracy%20of%20observer-reported%20ancestry%20in%20a%20biorepository%20linked%20to%20electronic%20medical%20records&volume=12&publication_year=2010&pages=648&pmid=20733501&doi=10.1097/GIM.0b013e3181efe2df&)

26. Kho AN, Pacheco JA, Peissig PL, Rasmussen L, Newton KM, Weston N et al. Electronic medical records for genetic research: results of the eMERGE consortium. Sci Transl Med 2011; 3: 79re1.  [DOI](https://doi.org/10.1126/scitranslmed.3001807) | [PMC free article](/articles/PMC3690272/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21508311/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Sci%20Transl%20Med&title=Electronic%20medical%20records%20for%20genetic%20research:%20results%20of%20the%20eMERGE%20consortium&volume=3&publication_year=2011&pages=79re1&pmid=21508311&doi=10.1126/scitranslmed.3001807&)

27. Israel RA. The International Classification of Disease. Two hundred years of development. Public Health Rep 1978; 93: 150–152.  [PMC free article](/articles/PMC1431884/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/635089/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Public%20Health%20Rep&title=The%20International%20Classification%20of%20Disease.%20Two%20hundred%20years%20of%20development&volume=93&publication_year=1978&pages=150&pmid=635089&)

28. Liu M, Shah A, Jiang M, Peterson NB, Dai Q, Aldrich MC et al. A study of transportability of an existing smoking status detection module across institutions. AMIA Annu Symp Proc 2012; 2012: 577–586.  [PMC free article](/articles/PMC3540509/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23304330/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=AMIA%20Annu%20Symp%20Proc&title=A%20study%20of%20transportability%20of%20an%20existing%20smoking%20status%20detection%20module%20across%20institutions&volume=2012&publication_year=2012&pages=577&pmid=23304330&)

29. Howie BN, Donnelly P, Marchini J. A flexible and accurate genotype imputation method for the next generation of genome-wide association studies. PLoS Genet 2009; 5: e1000529.  [DOI](https://doi.org/10.1371/journal.pgen.1000529) | [PMC free article](/articles/PMC2689936/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19543373/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet&title=A%20flexible%20and%20accurate%20genotype%20imputation%20method%20for%20the%20next%20generation%20of%20genome-wide%20association%20studies&volume=5&publication_year=2009&pages=e1000529&pmid=19543373&doi=10.1371/journal.pgen.1000529&)

30. Delaneau O, Zagury J-F, Marchini J. Improved whole-chromosome phasing for disease and population genetic studies. Nat Methods 2013; 10: 5–6.  [DOI](https://doi.org/10.1038/nmeth.2307) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23269371/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Methods&title=Improved%20whole-chromosome%20phasing%20for%20disease%20and%20population%20genetic%20studies&volume=10&publication_year=2013&pages=5&pmid=23269371&doi=10.1038/nmeth.2307&)

31. Patterson N, Price AL, Reich D. Population structure and eigenanalysis. PLoS Genet 2006; 2: e190.  [DOI](https://doi.org/10.1371/journal.pgen.0020190) | [PMC free article](/articles/PMC1713260/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17194218/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet&title=Population%20structure%20and%20eigenanalysis&volume=2&publication_year=2006&pages=e190&pmid=17194218&doi=10.1371/journal.pgen.0020190&)

32. Zuvich RL, Armstrong LL, Bielinski SJ, Bradford Y, Carlson CS, Crawford DC et al. Pitfalls of merging GWAS data: lessons learned in the eMERGE network and quality control procedures to maintain high data quality. Genet Epidemiol 2011; 35: 887–898.  [DOI](https://doi.org/10.1002/gepi.20639) | [PMC free article](/articles/PMC3592376/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22125226/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genet%20Epidemiol&title=Pitfalls%20of%20merging%20GWAS%20data:%20lessons%20learned%20in%20the%20eMERGE%20network%20and%20quality%20control%20procedures%20to%20maintain%20high%20data%20quality&volume=35&publication_year=2011&pages=887&pmid=22125226&doi=10.1002/gepi.20639&)

33. Verma SS, de Andrade M, Tromp G, Kuivaniemi H, Pugh E, Namjou B et al. Imputation and quality control steps for combining multiple genome-wide datasets. Front Genet 2014; 5: 370.  [DOI](https://doi.org/10.3389/fgene.2014.00370) | [PMC free article](/articles/PMC4263197/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25566314/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front%20Genet&title=Imputation%20and%20quality%20control%20steps%20for%20combining%20multiple%20genome-wide%20datasets&volume=5&publication_year=2014&pages=370&pmid=25566314&doi=10.3389/fgene.2014.00370&)

34. Mägi R, Morris AP. GWAMA: software for genome-wide association meta-analysis. BMC Bioinformatics 2010; 11: 288.  [DOI](https://doi.org/10.1186/1471-2105-11-288) | [PMC free article](/articles/PMC2893603/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20509871/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Bioinformatics&title=GWAMA:%20software%20for%20genome-wide%20association%20meta-analysis&volume=11&publication_year=2010&pages=288&pmid=20509871&doi=10.1186/1471-2105-11-288&)

35. Purcell S, Neale B, Todd-Brown K, Thomas L, Ferreira MAR, Bender D et al. PLINK: a tool set for whole-genome association and population-based linkage analyses. Am J Hum Genet 2007; 81: 559–575.  [DOI](https://doi.org/10.1086/519795) | [PMC free article](/articles/PMC1950838/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17701901/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=PLINK:%20a%20tool%20set%20for%20whole-genome%20association%20and%20population-based%20linkage%20analyses&volume=81&publication_year=2007&pages=559&pmid=17701901&doi=10.1086/519795&)

36. Marchini J, Howie B, Myers S, McVean G, Donnelly P. A new multipoint method for genome-wide association studies by imputation of genotypes. Nat Genet 2007; 39: 906–913.  [DOI](https://doi.org/10.1038/ng2088) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17572673/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=A%20new%20multipoint%20method%20for%20genome-wide%20association%20studies%20by%20imputation%20of%20genotypes&volume=39&publication_year=2007&pages=906&pmid=17572673&doi=10.1038/ng2088&)

37. Pruim RJ, Welch RP, Sanna S, Teslovich TM, Chines PS, Gliedt TP et al. LocusZoom: regional visualization of genome-wide association scan results. Bioinformatics 2010; 26: 2336–2337.  [DOI](https://doi.org/10.1093/bioinformatics/btq419) | [PMC free article](/articles/PMC2935401/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20634204/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=LocusZoom:%20regional%20visualization%20of%20genome-wide%20association%20scan%20results&volume=26&publication_year=2010&pages=2336&pmid=20634204&doi=10.1093/bioinformatics/btq419&)

38. Viechtbauer W. Conducting meta-analyses in R with the metafor package. J Stat Softw 2010; 36: 1–48.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Stat%20Softw&title=Conducting%20meta-analyses%20in%20R%20with%20the%20metafor%20package&volume=36&publication_year=2010&pages=1&)

39. GTEx Consortium. The Genotype-Tissue Expression (GTEx) project. Nat Genet 2013; 45: 580–585.  [DOI](https://doi.org/10.1038/ng.2653) | [PMC free article](/articles/PMC4010069/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23715323/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=The%20Genotype-Tissue%20Expression%20(GTEx)%20project&volume=45&publication_year=2013&pages=580&pmid=23715323&doi=10.1038/ng.2653&)

40. Ward LD, Kellis M. HaploReg: a resource for exploring chromatin states, conservation, and regulatory motif alterations within sets of genetically linked variants. Nucleic Acids Res 2012; 40: D930–D934.  [DOI](https://doi.org/10.1093/nar/gkr917) | [PMC free article](/articles/PMC3245002/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22064851/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=HaploReg:%20a%20resource%20for%20exploring%20chromatin%20states,%20conservation,%20and%20regulatory%20motif%20alterations%20within%20sets%20of%20genetically%20linked%20variants&volume=40&publication_year=2012&pages=D930&pmid=22064851&doi=10.1093/nar/gkr917&)

41. Roden DM, Pulley JM, Basford MA, Bernard GR, Clayton EW, Balser JR et al. Development of a large-scale de-identified DNA biobank to enable personalized medicine. Clin Pharmacol Ther 2008; 84: 362–369.  [DOI](https://doi.org/10.1038/clpt.2008.89) | [PMC free article](/articles/PMC3763939/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18500243/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Development%20of%20a%20large-scale%20de-identified%20DNA%20biobank%20to%20enable%20personalized%20medicine&volume=84&publication_year=2008&pages=362&pmid=18500243&doi=10.1038/clpt.2008.89&)

42. Shibata R, Misonou H, Campomanes CR, Anderson AE, Schrader LA, Doliveira LC et al. A fundamental role for KChIPs in determining the molecular properties and trafficking of Kv4.2 potassium channels. J Biol Chem 2003; 278: 36445–36454.  [DOI](https://doi.org/10.1074/jbc.M306142200) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12829703/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=A%20fundamental%20role%20for%20KChIPs%20in%20determining%20the%20molecular%20properties%20and%20trafficking%20of%20Kv4.2%20potassium%20channels&volume=278&publication_year=2003&pages=36445&pmid=12829703&doi=10.1074/jbc.M306142200&)

43. Pruunsild P, Timmusk T. Structure, alternative splicing, and expression of the human and mouse KCNIP gene family. Genomics 2005; 86: 581–593.  [DOI](https://doi.org/10.1016/j.ygeno.2005.07.001) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16112838/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genomics&title=Structure,%20alternative%20splicing,%20and%20expression%20of%20the%20human%20and%20mouse%20KCNIP%20gene%20family&volume=86&publication_year=2005&pages=581&pmid=16112838&doi=10.1016/j.ygeno.2005.07.001&)

44. Massone S, Vassallo I, Castelnuovo M, Fiorino G, Gatta E, Robello M et al. RNA polymerase III drives alternative splicing of the potassium channel-interacting protein contributing to brain complexity and neurodegeneration. J Cell Biol 2011; 193: 851–866.  [DOI](https://doi.org/10.1083/jcb.201011053) | [PMC free article](/articles/PMC3105541/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21624954/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Cell%20Biol&title=RNA%20polymerase%20III%20drives%20alternative%20splicing%20of%20the%20potassium%20channel-interacting%20protein%20contributing%20to%20brain%20complexity%20and%20neurodegeneration&volume=193&publication_year=2011&pages=851&pmid=21624954&doi=10.1083/jcb.201011053&)

45. Xiong H, Kovacs I, Zhang Z. Differential distribution of KChIPs mRNAs in adult mouse brain. Brain Res Mol Brain Res 2004; 128: 103–111.  [DOI](https://doi.org/10.1016/j.molbrainres.2004.06.024) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15363885/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Brain%20Res%20Mol%20Brain%20Res&title=Differential%20distribution%20of%20KChIPs%20mRNAs%20in%20adult%20mouse%20brain&volume=128&publication_year=2004&pages=103&pmid=15363885&doi=10.1016/j.molbrainres.2004.06.024&)

46. Morohashi Y, Hatano N, Ohya S, Takikawa R, Watabiki T, Takasugi N et al. Molecular cloning and characterization of CALP/KChIP4, a novel EF-hand protein interacting with presenilin 2 and voltage-gated potassium channel subunit Kv4. J Biol Chem 2002; 277: 14965–14975.  [DOI](https://doi.org/10.1074/jbc.M200897200) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11847232/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=Molecular%20cloning%20and%20characterization%20of%20CALP/KChIP4,%20a%20novel%20EF-hand%20protein%20interacting%20with%20presenilin%202%20and%20voltage-gated%20potassium%20channel%20subunit%20Kv4&volume=277&publication_year=2002&pages=14965&pmid=11847232&doi=10.1074/jbc.M200897200&)

47. Takahama K, Araki T, Fuchikami J, Kohjimoto Y, Miyata T. Studies on the magnitude and the mechanism of cough potentiation by angiotensin-converting enzyme inhibitors in guinea-pigs: involvement of bradykinin in the potentiation. J Pharm Pharmacol 1996; 48: 1027–1033.  [DOI](https://doi.org/10.1111/j.2042-7158.1996.tb05895.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8953504/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharm%20Pharmacol&title=Studies%20on%20the%20magnitude%20and%20the%20mechanism%20of%20cough%20potentiation%20by%20angiotensin-converting%20enzyme%20inhibitors%20in%20guinea-pigs:%20involvement%20of%20bradykinin%20in%20the%20potentiation&volume=48&publication_year=1996&pages=1027&pmid=8953504&doi=10.1111/j.2042-7158.1996.tb05895.x&)

48. Himes BE, Sheppard K, Berndt A, Leme AS, Myers RA, Gignoux CR et al. Integration of mouse and human genome-wide association data identifies KCNIP4 as an asthma gene. PLoS One 2013; 8: e56179.  [DOI](https://doi.org/10.1371/journal.pone.0056179) | [PMC free article](/articles/PMC3572953/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23457522/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=Integration%20of%20mouse%20and%20human%20genome-wide%20association%20data%20identifies%20KCNIP4%20as%20an%20asthma%20gene&volume=8&publication_year=2013&pages=e56179&pmid=23457522&doi=10.1371/journal.pone.0056179&)

49. Wojczynski MK, Tiwari HK. Definition of phenotype. Adv Genet 2008; 60: 75–105.  [DOI](https://doi.org/10.1016/S0065-2660(07)00404-X) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18358317/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Adv%20Genet&title=Definition%20of%20phenotype&volume=60&publication_year=2008&pages=75&pmid=18358317&doi=10.1016/S0065-2660(07)00404-X&)

50. Rice JP, Saccone NL, Rasmussen E. Definition of the phenotype. Adv Genet 2001; 42: 69–76.  [DOI](https://doi.org/10.1016/s0065-2660(01)42015-3) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11037314/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Adv%20Genet&title=Definition%20of%20the%20phenotype&volume=42&publication_year=2001&pages=69&pmid=11037314&doi=10.1016/s0065-2660(01)42015-3&)

51. Sinnott JA, Kraft P. Artifact due to differential error when cases and controls are imputed from different platforms. Hum Genet 2012; 131: 111–119.  [DOI](https://doi.org/10.1007/s00439-011-1054-1) | [PMC free article](/articles/PMC3217156/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21735171/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Genet&title=Artifact%20due%20to%20differential%20error%20when%20cases%20and%20controls%20are%20imputed%20from%20different%20platforms&volume=131&publication_year=2012&pages=111&pmid=21735171&doi=10.1007/s00439-011-1054-1&)

52. Johnson EO, Hancock DB, Levy JL, Gaddis NC, Saccone NL, Bierut LJ et al. Imputation across genotyping arrays for genome-wide association studies: assessment of bias and a correction strategy. Hum Genet 2013; 132: 509–522.  [DOI](https://doi.org/10.1007/s00439-013-1266-7) | [PMC free article](/articles/PMC3628082/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23334152/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Genet&title=Imputation%20across%20genotyping%20arrays%20for%20genome-wide%20association%20studies:%20assessment%20of%20bias%20and%20a%20correction%20strategy&volume=132&publication_year=2013&pages=509&pmid=23334152&doi=10.1007/s00439-013-1266-7&)

53. Smemo S, Tena JJ, Kim K-H, Gamazon ER, Sakabe NJ, Gómez-Marín C et al. Obesity-associated variants within FTO form long-range functional connections with IRX3. Nature 2014; 507: 371–375.  [DOI](https://doi.org/10.1038/nature13138) | [PMC free article](/articles/PMC4113484/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24646999/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Obesity-associated%20variants%20within%20FTO%20form%20long-range%20functional%20connections%20with%20IRX3&volume=507&publication_year=2014&pages=371&pmid=24646999&doi=10.1038/nature13138&)

54. Van den Boogaard M, Smemo S, Burnicka-Turek O, Arnolds DE, van de Werken HJG, Klous P et al. A common genetic variant within SCN10A modulates cardiac SCN5A expression. J Clin Invest 2014; 124: 1844–1852.  [DOI](https://doi.org/10.1172/JCI73140) | [PMC free article](/articles/PMC3973109/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24642470/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Invest&title=A%20common%20genetic%20variant%20within%20SCN10A%20modulates%20cardiac%20SCN5A%20expression&volume=124&publication_year=2014&pages=1844&pmid=24642470&doi=10.1172/JCI73140&)
