# Genome-Wide Meta-Analysis of Cotinine Levels in Cigarette Smokers Identifies Locus at 4q13.2

## Metadata
**Authors:** Jennifer J Ware, Xiangning Chen, Jacqueline Vink, Anu Loukola, Camelia Minica, Rene Pool, Yuri Milaneschi, Massimo Mangino, Cristina Menni, Jingchun Chen, Roseann E Peterson, Kirsi Auro, Leo-Pekka Lyytikäinen, Juho Wedenoja, Alexander I Stiby, Gibran Hemani, Gonneke Willemsen, Jouke Jan Hottenga, Tellervo Korhonen, Markku Heliövaara, Markus Perola, Richard J Rose, Lavinia Paternoster, Nic Timpson, Catherine A Wassenaar, Andy Z X Zhu, George Davey Smith, Olli T Raitakari, Terho Lehtimäki, Mika Kähönen, Seppo Koskinen, Timothy Spector, Brenda W J H Penninx, Veikko Salomaa, Dorret I Boomsma, Rachel F Tyndale, Jaakko Kaprio, Marcus R Munafò
**Journal:** Scientific Reports
**Date:** 2016 Feb 1
**DOI:** [10.1038/srep20092](https://doi.org/10.1038/srep20092)
**PMID:** 26833182
**PMCID:** PMC4735517
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4735517/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC4735517/pdf/srep20092.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC4735517/pdf/srep20092.pdf)

## Abstract

Genome-wide association studies (GWAS) of complex behavioural phenotypes such as cigarette smoking typically employ self-report phenotypes. However, precise biomarker phenotypes may afford greater statistical power and identify novel variants. Here we report the results of a GWAS meta-analysis of levels of cotinine, the primary metabolite of nicotine, in 4,548 daily smokers of European ancestry. We identified a locus close to UGT2B10 at 4q13.2 (minimum p = 5.89 × 10−10 for rs114612145), which was consequently replicated. This variant is in high linkage disequilibrium with a known functional variant in the UGT2B10 gene which is associated with reduced nicotine and cotinine glucuronidation activity, but intriguingly is not associated with nicotine intake. Additionally, we observed association between multiple variants within the 15q25.1 region and cotinine levels, all located within the CHRNA5-A3-B4 gene cluster or adjacent genes, consistent with previous much larger GWAS using self-report measures of smoking quantity. These results clearly illustrate the increase in power afforded by using precise biomarker measures in GWAS. Perhaps more importantly however, they also highlight that biomarkers do not always mark the phenotype of interest. The use of metabolite data as a proxy for environmental exposures should be carefully considered in the context of individual differences in metabolic pathways.

## Results

Variants in two genomic regions were found to be associated with cotinine levels, including 15q25.1 (a region previously identified in association with smoking quantity[1](#b1),[3](#b3),[4](#b4)) and a locus at 4q13.2 (see [Figs 1](#f1) and [2](#f2), [Table 1](#t1), [Table S2](#S1)). All 96 variants that met or exceeded the threshold for genome-wide significance (*p* < 5 × 10^−8^) on chromosome 4 lay between 69.6 and 69.9 Mb within a region of *UGT* genes, including *UGT2B10* and *UGT2A3* (see [Fig. 2](#f2)). The variant with the lowest *p*-value in this region was rs114612145 (rs77107237 in GRCh38) (*p* = 5.89 × 10^−10^), which lies between *UGT2B10* and *UGT2A3* ([Table 1](#t1), [Fig. 2](#f2)). The minor allele (G) was associated with a 0.22 SD increase in cotinine level, equating to ~39 ng/ml increase in plasma/serum cotinine ([Table 1](#t1), [Fig. 2](#f2)). The cotinine quantification method did not affect the result. This SNP accounted for 0.87% of the variance in cotinine levels.

### Figure 1. Manhattan and quantile-quantile plots illustrating genome-wide meta-analysis results.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f2d8/4735517/193e52715405/srep20092-f1.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4735517_srep20092-f1.jpg)

Manhattan plot (A): All SNPs plotted on x-axis according to their position on each chromosome, against their association with cotinine level, as shown on the y-axis as –log10 p-value. QQ plot (B): The observed distribution of p-values (y-axis) against the expected distribution of p-values under the null hypothesis (x-axis). Plots includes variants which were genotyped or imputed in at least 3,000 individuals only (~7 M SNPs).

### Figure 2. Forest and regional plots of associations for cotinine from genome-wide meta-analysis.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f2d8/4735517/b92d1823fd91/srep20092-f2.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4735517_srep20092-f2.jpg)

Forest plots illustrate effect size and 95% confidence intervals (CIs) observed in each contributing study for chromosome 15 (A) and chromosome 4 (B) SNPs with smallest p-values (“top” SNPs). Regional plots show SNPs plotted by their positions on chromosomes against –log10 p-value for their association with cotinine level in genome-wide meta-analysis. The top SNP in each region is highlighted in purple. The SNPs surrounding each top SNP are colour coded to reflect their LD with this variant (see legend). Estimated recombination rates are plotted in pale blue to reflect local LD structure on secondary y-axis. Genome build = hg19; LD population = 1000 Genomes March 2012 release (EUR). Regional plots generated using Locus Zoom.

### Table 1. Summary information for top SNPs identified from cotinine genome-wide meta-analysis.

| SNP | Chr | Gene | Position | EA | EAF | n | Beta | SE | p value |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| rs10851907 | 15 | Intergenic (CHRNB4/CHRNA3) | 78915864 | A | 0.41 | 4330 | 0.19 | 0.02 | 1.46 × 10−19 |
| rs114612145 | 4 | Intergenic (UGT2B10/UGT2A3) | 69746647 | G | 0.10 | 4290 | 0.22 | 0.04 | 5.89 × 10−10 |

Table 1 Caption: A total of 279 SNPs on chromosome 15 and 96 SNPs on chromosome 4 exceeded genome-wide significance for association with cotinine. The top SNP on each chromosome is shown. Position refers to base pair position in genome build hg19/GRCh37. EA: effect allele; EAF: effect allele frequency; SE: standard error; Beta: change in standard deviation of cotinine level per copy of the effect allele in an additive model.

To investigate whether the association with cotinine at 4q13.2 could be completely explained by rs114612145, we repeated the association analysis conditioning on this SNP ([Figure S1](#S1), bottom panel). No residual signal was detected, suggesting that the variants identified in this region represent a single independent signal. For further confirmation of the cotinine association signal at 4q13.2, selected variants from this region were examined in relation to cotinine levels in two independent samples (see [Tables S3](#S1) and [S4](#S1)). Strong evidence for association was observed in both ([Table S4](#S1)). The rs114612145 SNP identified in our sample is in high LD with a functional missense variant in *UGT2B10*, rs144647471 (also known as Asp67Tyr or rs61750900 in build GRCh38) (r^2^ = 0.90). Whilst this missense variant did not reach the threshold for genome-wide significance in our discovery sample (*p* = 1.91 × 10^−5^), this is likely due to the reduced sample size upon which association was determined (SNP imputed in only 2,585 individuals). However, we observed evidence of association in an independent sample, in the same direction to that observed in the discovery sample (*p* = 0.020; [Table S3](#S1)).

Our meta-analysis also highlighted a locus on chromosome 15, which has consistently been identified in GWAS of smoking quantity. All 279 genome-wide significant variants identified on this chromosome lay within the *CHRNA5-A3-B4* nicotinic receptor gene cluster, adjacent genes (*IREB2*, *AGPHD1*, *PSMA4* and *ADAMTS7*), or intergenic regions (see [Fig. 2](#f2)). The variant with the lowest *p*-value (both in this region and overall) was rs10851907 (*p* = 1.46 × 10^−19^), located in an intergenic region between *CHRNB4* and *CHRNA3* ([Table 1](#t1), [Fig. 2](#f2)). The minor allele (A) was associated with a 0.19 SD increase in cotinine levels, equating to a ~34 ng/ml increase in plasma/serum cotinine ([Table 1](#t1), [Fig. 2](#f2)). This SNP, which is in LD with rs16969968 (r^2^ = 0.71), accounted for 1.75% of the variance in cotinine levels.

To investigate whether the association with cotinine at 15q25.1 could be explained completely by rs10851907, we repeated the association analysis conditioning on this SNP ([Figure S2](#S1)). Residual association was detected within the region, with strongest evidence for association at rs57064725 (*p*_c_ = 2.92 × 10^−8^), located in an intron of *PSMA4* ([Figure S2](#S1), middle panel). Conditioning on both of these variants left no residual signal ([Figure S2](#S1), bottom panel). Given previous robust evidence linking measures of smoking quantity to the nonsynonymous SNP rs16969968, a variant which also exceeded the threshold for genome-wide significance in our analyses (*p* = 6.91 × 10^−17^), we also re-ran the association analysis conditioning on this SNP ([Figure S3](#S1)). Residual association similar to that observed after conditioning on rs10851907 was detected within the region, with strongest evidence for association at rs7170068 (*p*_c_ = 1.51 × 10^−9^), located in an intron in *CHRNA3* ([Figure S3](#S1), middle panel). Conditioning on rs16969968 and rs7170068 left no residual signal in this region ([Figure S3](#S1), bottom panel). It is notable that earlier dissection of the 15q25.1 region has suggested a third distinct signal for smoking quantity, marked by rs588765, apparent upon adjustment for rs16969968[9](#b9),[10](#b10). However, this signal was not identified in three previous large GWAS meta-analyses after conditioning on rs16969968[1](#b1),[3](#b3),[4](#b4), and was not apparent in our data, before or after adjustment for rs16969968 (*p* = 0.10; *p*_c_ = 1.18 × 10^−3^).

## Discussion

We conducted a meta-analysis of genome-wide association data on cotinine levels in current smokers, in order to identify genetic variants associated with smoking behaviour. We identified 375 genetic variants, representing three independent signals. Two of these signals were located within *CHRNA5-A3-B4* on chromosome 15, a region that is now well-established as being associated with smoking heaviness. More importantly however, we also identified a signal within *UGT2B10* on chromosome 4, a locus which has previously been found to associate with cotinine and nicotine metabolism, but *not* with nicotine dose, highlighting an important limitation of the use of metabolite data (such as cotinine) as a proxy for an environmental exposure.

UGT2B10 plays a key role in nicotine and cotinine glucuronidation, converting nicotine to nicotine-glucuronide (a minor nicotine metabolic pathway, accounting for 3–5% nicotine clearance[11](#b11),[12](#b12),[13](#b13)), and cotinine to cotinine-glucuronide (accounting for 12–17% of cotinine clearance[11](#b11)). The minor allele at rs144647471 (i.e., that associated with higher cotinine levels in our sample) is associated with a reduction in UGT2B10 function for both nicotine and cotinine glucuronidation[14](#b14),[15](#b15).

Several factors contribute to cotinine levels: consumption of nicotine (i.e., heaviness of smoking), conversion of nicotine to cotinine (catalysed primarily by CYP2A6), and cotinine clearance (including cotinine glucuronidation). It is theoretically possible that the reduced function *UGT2B10* variant identified in this GWAS function could alter consumption. Reduced clearance of nicotine through glucuronidation (a minor pathway) would theoretically result in modestly longer lasting circulating levels of nicotine, which could *reduce* smoking if individuals self-titrate nicotine levels. Consistent with this theory, a previous study found that *ad libitum* smokers with this reduced function *UGT2B10* variant had lower nicotine intake[14](#b14), as indexed by total nicotine equivalent levels. However, other studies (including a much larger study from the same group) have found that this variant has no effect on consumption[16](#b16),[17](#b17). Further, a decrease in nicotine intake via *ad libitum* smoking is *inconsistent* with higher cotinine levels, thus a change in consumption does not explain the association we note with the reduced function *UGT2B10* variant and higher cotinine levels. A more likely explanation for our findings is a reduction in cotinine metabolism, through decreased cotinine glucuronidation, which is consistent with the association noted between the reduced function *UGT2B10* variant and increased cotinine levels. Indeed, a recent GWAS conducted by Patel and colleagues[17](#b17) concluded that genetic variation within *UGT2B10* contributes significantly to nicotine and cotinine glucuronidation but not to nicotine dose.

Cotinine is a strong biomarker of cigarette consumption in daily smokers[5](#b5),[18](#b18),[19](#b19),[20](#b20),[21](#b21). The confirmation of the *CHRNA5-A3-B4* locus by our meta-analysis, and the conditional analyses results which parallel those of much larger GWAS of self-reported cigarette consumption[3](#b3) demonstrate the utility of cotinine to identify genetic variants responsible for smoking quantity with greater statistical power (see also [Table S5](#S1)). However, since cotinine is an intermediate metabolite, its concentration is influenced by multiple contributing pathways. Individual differences in nicotine and cotinine metabolism, both via glucuronidation, as suggested here, and by oxidative metabolism as previously observed[18](#b18) have confounding effects. While substantially more accurate than self-report smoking levels, future studies seeking an even more comprehensive assessment of tobacco exposure may consider using total nicotine equivalents (i.e., sum of nicotine and all its metabolites), a robust measure of nicotine exposure which intrinsically controls for variation in genes influencing both nicotine and cotinine metabolism (including variation in *UGT2B10* and *CYP2A6*)[12](#b12),[22](#b22).

In conclusion, we observed evidence of association between cotinine levels in current smokers and a locus at 4q13.2 encompassing *UGT2B10*, which encodes an enzyme playing a key role in nicotine and cotinine glucuronidation, in addition to the 15q25.1 region, previously shown to robustly associate with smoking quantity. Our analyses clearly illustrate the benefit of using precise, objective phenotypes in GWAS. However, they also importantly illustrate that biomarkers do not always capture the phenotype of interest. The use of metabolite data (such as cotinine) as a proxy for environmental exposures should be carefully considered in the context of individual differences in metabolic pathways.

## Methods

### Contributing studies

A total of 11 studies (collectively forming the Cotinine Consortium) contributed to the GWAS meta-analysis: Avon Longitudinal Study of Parents and Children (ALSPAC), Coronary Artery Risk Development in Young Adults (CARDIA), FinnTwin, FINRISK, Framingham Heart Study, Health2000 GenMets study (GenMets), Multi-Ethnic Study of Atherosclerosis (MESA), Netherlands Study of Depression and Anxiety (NESDA), Netherlands Twin Register (NTR), TwinsUK, and Cardiovascular Risk in Young Finns Study (YFS) (see [Table 2](#t2)). These 11 samples resulted in a collective sample size of *n* = 4,548. Further information on each study/cohort is provided in Text S1.

### Table 2. Descriptive characteristics of the 11 studies contributing to the genome-wide meta-analysis.

| Study | n | Sex (% male) | Age (years) |  | Cotinine (ng/ml)a |  | Medium | Method |
| --- | --- | --- | --- | --- | --- | --- | --- | --- |
| Mean | SD | Mean | SD |  |  |  |  |  |
| ALSPAC | 258 | 50.4 | 17.8 | 0.4 | 182.1 | 107.2 | Plasma | Immunoassay |
| CARDIA | 387 | 47.8 | 25.3 | 3.4 | 202.0 | 137.8 | Plasma | Radioimmunoassay |
| FinnTwin | 145 | 46.2 | 23.0 | 1.5 | 206.6 | 107.5 | Serum | Mass spectrometry |
| FINRISK | 218 | 59.8 | 48.4 | 11.6 | 223.6 | 167.7 | Serum | Mass spectrometry |
| Framingham | 93 | 43.0 | N/A | N/A | 101.3 | 55.6 | Plasma/serum | Mass spectrometry |
| GenMets | 485 | 57.8 | 47.3 | 11.2 | 490.1 | 250.6 | Serum | Immunoassay |
| MESA | 189 | 57.5 | 59.6 | 8.9 | 4818.2 | 4105.5 | Urine | Immunoassay |
| NESDA | 808 | 36.5 | 41.5 | 12.4 | 260.9 | 224.4 | Plasma | Immunoassay |
| NTR | 897 | 44.1 | 43.6 | 13.9 | 278.5 | 269.7 | Plasma | Immunoassay |
| TwinsUK | 676 | 8.9 | 48.1 | 13.7 | 175.3 | 63.6 | Plasma | Mass spectrometry |
| YFS | 392 | 55.4 | 33.8 | 6.2 | 200.6 | 108.9 | Serum | Mass spectrometry |

Table 2 Caption: aCotinine mean and standard deviation values refer to raw values prior to standardisation (i.e., conversion to Z-scores). Further study details available in Text S1. SD: standard deviation.

### Phenotype definition

Cotinine levels were determined from plasma, serum or urine samples, and quantified using immunoassay, radioimmunoassay or mass spectrometry (see [Table 2](#t2) and Text S1 for further details). Cotinine levels show good agreement within samples across assessment methods, and further are highly correlated across different types of biological samples, including urine and blood[23](#b23),[24](#b24). However, given that absolute cotinine levels are much higher in urine relative to blood, cotinine data were transformed and standardised prior to the conduct of individual study-level GWAS analysis, to ensure meaningful comparison and synthesis of results across studies. Within each study sample, cotinine levels were assayed from one specific type of biological sample, using a singular assessment method.

### Sample inclusion criteria

Individuals within each sample were eligible for inclusion in analyses provided they were assessed for cotinine level at or after 17 years of age, of European ancestry, successfully genotyped genome-wide, and current *daily* smokers at the time of cotinine assessment. To minimise inclusion of non-smokers and non-daily smokers in our analyses, specific cotinine level inclusion thresholds were imposed, determined on the basis of receiver operating characteristic (ROC) analyses conducted in representative samples with smoking status self-report data and cotinine data derived from plasma and urine samples. These were conservatively set at 10 ng/ml cotinine in serum/plasma samples assessed using mass spectrometry, 50 ng/ml cotinine in serum/plasma samples assessed using immunoassay, and 80 ng/ml cotinine in urine samples assessed using immunoassay. Full descriptive characteristics of the studies in the Cotinine Consortium are presented in [Table 2](#t2).

### Genotyping and imputation

All contributing studies performed their own genotyping, quality control, and imputation to the 1000 Genomes Phase 1 Version 3 reference panel (see [Table S1](#S1)). Study samples were genotyped on a number of different platforms. Each study applied its own set of quality control filters. Genotype imputation was performed using IMPUTE, IMPUTE2 or MACH prior to genome-wide association analyses.

### Study specific GWAS analysis

Prior to study-specific genome-wide association analyses, cotinine data were transformed if necessary to correct for positive skew (using natural logarithm or square-root), and then standardised (i.e., converted to Z-scores). An additive genetic model was used for association analyses. Linear regression was used to establish evidence of association, with standardised cotinine level as the dependent variable and allele dose (0, 1 or 2 copies of the minor allele) as the independent variable. All analyses adjusted for sex and age, with the exception of Framingham, which controlled only for sex as age data for this sample were not available. For family-based studies (e.g., NTR and FinnTwin), only one observation per family was included.

### Meta-analysis of GWAS results

All 11 GWAS summary data files were delivered to the co-ordinating site via secure file-sharing services. Imputation quality control procedures were centrally imposed. Specifically, variants were excluded if: a) MAF < 1%, and/or b) info score < 0.4 or r^2^ < 0.3. Once the quality of each data file was confirmed, files were imported into METAL (March 2011 Release) ([http://www.sph.umich.edu/csg/abecasis/metal/index.html](http://www.sph.umich.edu/csg/abecasis/metal/index.html)), a software tool for meta-analysis of whole genome association data. Genomic control was enabled (appropriate genomic control correction applied to input files) to correct for population structure. A fixed-effects meta-analysis was then performed for each SNP by combining allelic effects weighted by the inverse of their variance. Secondary correction for population structure via genomic control of summary statistics was not performed as the genomic control parameter (λGC) for meta-analysis summary statistics was 0.992. The fixed threshold for genome-wide significance was set at *p* < 5 × 10^−8^. The meta-analysis was completed for ~11 M variants. Results were limited to the ~7 M variants which had been genotyped/imputed in at least 3,000 individuals. The meta-analysis was also repeated using GWAMA 2.1[25](#b25) assuming random effects and an additive model, and the same results were obtained.

### Conditional analyses

Conditional analyses were performed with GCTA software[26](#b26) using full meta-analysis summary level statistics and a European reference panel (ALSPAC mothers cohort; n = 8,890) to determine the number of independent signals present in regions identified in our meta-analysis. For each region of interest, we re-ran the meta-analysis conditioning on our SNP with the lowest *p*-value. The next signal was identified from the conditional meta-analysis results, and included in the second conditional meta-analysis. This process was repeated in an iterative fashion until no residual signal remained below a threshold of *p* < 5 × 10^−8^ (see [Figure S1 and S2](#S1)). For 15q25.1, we also employed an additional strategy, conditioning on missense SNP rs16969968 first. The entire process was also repeated using an alternative sample (Netherlands Twin Register) as the LD reference panel (*n* = 7,000), and similar results obtained.

### Independent replication

We examined the association between the novel signal identified in our meta-analysis and cotinine levels in two independent samples, FINRISK2007 and FinnTwin (persons not included in discovery GWAS). Descriptive characteristics, genotyping and imputation information on these two samples is presented in [Table S3](#S1).

## Acknowledgements

JJW is supported by a Post-Doctoral Research Fellowship from the Oak Foundation ([http://www.oakfnd.org/](http://www.oakfnd.org/)). JJW, GH, LP, NT, GDS and MRM are members of the Medical Research Council (MRC) Integrative Epidemiology Unit: support from the MRC is gratefully acknowledged (MC_UU_12013/6). JJW and MRM are members of the UK Centre for Tobacco and Alcohol Studies, a UK Clinical Research Council Public Health Research: Centre of Excellence. Funding from British Heart Foundation, Cancer Research UK, Economic and Social Research Council, Medical Research Council, and the National Institute for Health Research, under the auspices of the UK Clinical Research Collaboration, is gratefully acknowledged. XC, JC and REP acknowledge support from the National Institute on Drug Abuse (NIDA) (R01DA032246). REP also acknowledges support from NIDA (R25 DA26119) and National Institute of Mental Health (NIMH) (T32 MH20030). AIS is supported by a Wellcome Trust 4-year PhD studentship in molecular, genetic, and lifecourse epidemiology (WT083431MA). LP is funded by a UK MRC Population Health Scientist fellowship (MR/J012165/1). VS was funded by the Academy of Finland (139635) and the Finnish Foundation for Cardiovascular Research. RJR acknowledges support from National Institute on Alcohol Abuse and Alcoholism (AA-12502). KA received sponsorship from the Finnish Medical Association. MP acknowledges EU FP7 Grants BBMRI-Large Prospective Cohorts (313010), and BioSHaRE (261433). YM is supported by GGZinGeest, the mental health care organization supporting NESDA. CM is supported by European Research Council (ERC) starting grant 284167 (P.I. JV). DB also acknowledges ERC support (230374). CW is supported by a Canadian Institutes of Health Research (CIHR) doctoral award. MM receives support from the National Institute for Health Research (NIHR) BioResource Clinical Research Facility and Biomedical Research Centre based at Guy’s and St Thomas’ NHS Foundation Trust and King’s College London. RFT is an Endowed Chair in Addiction for the Department of Psychiatry at the University of Toronto (CIHR grant TMH109787, NIH PGRN grant DA020830). JK is supported by Academy of Finland grants 265240 and 263278. Acknowledgements specific to individual cohorts are listed in [Supplementary Information](#S1).

## Additional Information

**Data availability**: Cotinine GWAS meta-analysis summary results (doi: 10.5523/bris.182rhz19hg3lz1172a7yfcap9v) accessible in full at [https://data.bris.ac.uk/data/dataset/182rhz19hg3lz1172a7yfcap9v](https://data.bris.ac.uk/data/dataset/182rhz19hg3lz1172a7yfcap9v).

**How to cite this article**: Ware, J. J. *et al.* Genome-Wide Meta-Analysis of Cotinine Levels in Cigarette Smokers Identifies Locus at 4q13.2. *Sci. Rep.* **6**, 20092; doi: 10.1038/srep20092 (2016).

## Supplementary Material

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Furberg H. et al. Genome-wide meta-analyses identify multiple loci associated with smoking behavior. Nature Genetics 42, 441–447, 10.1038/ng.571 (2010).  [DOI](https://doi.org/10.1038/ng.571) | [PMC free article](/articles/PMC2914600/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20418890/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature%20Genetics&title=Genome-wide%20meta-analyses%20identify%20multiple%20loci%20associated%20with%20smoking%20behavior&author=H.%20Furberg&volume=42&publication_year=2010&pages=441-447&pmid=20418890&doi=10.1038/ng.571&)

2. Thorgeirsson T. E. et al. A variant associated with nicotine dependence, lung cancer and peripheral arterial disease. Nature 452, 638–642, 10.1038/nature06846 (2008).  [DOI](https://doi.org/10.1038/nature06846) | [PMC free article](/articles/PMC4539558/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18385739/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=A%20variant%20associated%20with%20nicotine%20dependence,%20lung%20cancer%20and%20peripheral%20arterial%20disease&author=T.%20E.%20Thorgeirsson&volume=452&publication_year=2008&pages=638-642&pmid=18385739&doi=10.1038/nature06846&)

3. Liu J. Z. et al. Meta-analysis and imputation refines the association of 15q25 with smoking quantity. Nature Genetics 42, 436–440, 10.1038/ng.572 (2010).  [DOI](https://doi.org/10.1038/ng.572) | [PMC free article](/articles/PMC3612983/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20418889/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature%20Genetics&title=Meta-analysis%20and%20imputation%20refines%20the%20association%20of%2015q25%20with%20smoking%20quantity&author=J.%20Z.%20Liu&volume=42&publication_year=2010&pages=436-440&pmid=20418889&doi=10.1038/ng.572&)

4. Thorgeirsson T. E. et al. Sequence variants at CHRNB3-CHRNA6 and CYP2A6 affect smoking behavior. Nature Genetics 42, 448–453, 10.1038/ng.573 (2010).  [DOI](https://doi.org/10.1038/ng.573) | [PMC free article](/articles/PMC3080600/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20418888/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature%20Genetics&title=Sequence%20variants%20at%20CHRNB3-CHRNA6%20and%20CYP2A6%20affect%20smoking%20behavior&author=T.%20E.%20Thorgeirsson&volume=42&publication_year=2010&pages=448-453&pmid=20418888&doi=10.1038/ng.573&)

5. Munafo M. R. et al. Association between genetic variants on chromosome 15q25 locus and objective measures of tobacco exposure. Journal of the National Cancer Institute 104, 740–748, 10.1093/jnci/djs191 (2012).  [DOI](https://doi.org/10.1093/jnci/djs191) | [PMC free article](/articles/PMC3352832/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22534784/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20of%20the%20National%20Cancer%20Institute&title=Association%20between%20genetic%20variants%20on%20chromosome%2015q25%20locus%20and%20objective%20measures%20of%20tobacco%20exposure&author=M.%20R.%20Munafo&volume=104&publication_year=2012&pages=740-748&pmid=22534784&doi=10.1093/jnci/djs191&)

6. Keskitalo K. et al. Association of serum cotinine level with a cluster of three nicotinic acetylcholine receptor genes (CHRNA3/CHRNA5/CHRNB4) on chromosome 15. Human Molecular Genetics 18, 4007–4012, 10.1093/hmg/ddp322 (2009).  [DOI](https://doi.org/10.1093/hmg/ddp322) | [PMC free article](/articles/PMC2748889/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19628476/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Human%20Molecular%20Genetics&title=Association%20of%20serum%20cotinine%20level%20with%20a%20cluster%20of%20three%20nicotinic%20acetylcholine%20receptor%20genes%20(CHRNA3/CHRNA5/CHRNB4)%20on%20chromosome%2015&author=K.%20Keskitalo&volume=18&publication_year=2009&pages=4007-4012&pmid=19628476&doi=10.1093/hmg/ddp322&)

7. McNeill A. & Munafo M. R. Reducing harm from tobacco use. Journal of Psychopharmacology 27, 13–18, 10.1177/0269881112458731 (2013).  [DOI](https://doi.org/10.1177/0269881112458731) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23035032/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Psychopharmacology&title=Reducing%20harm%20from%20tobacco%20use&author=A.%20McNeill&author=M.%20R.%20Munafo&volume=27&publication_year=2013&pages=13-18&pmid=23035032&doi=10.1177/0269881112458731&)

8. Strasser A. A., Lerman C., Sanborn P. M., Pickworth W. B. & Feldman E. A. New lower nicotine cigarettes can produce compensatory smoking and increased carbon monoxide exposure. Drug and Alcohol Dependence 86, 294–300, 10.1016/j.drugalcdep.2006.06.017 (2007).  [DOI](https://doi.org/10.1016/j.drugalcdep.2006.06.017) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16930853/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20and%20Alcohol%20Dependence&title=New%20lower%20nicotine%20cigarettes%20can%20produce%20compensatory%20smoking%20and%20increased%20carbon%20monoxide%20exposure&author=A.%20A.%20Strasser&author=C.%20Lerman&author=P.%20M.%20Sanborn&author=W.%20B.%20Pickworth&author=E.%20A.%20Feldman&volume=86&publication_year=2007&pages=294-300&pmid=16930853&doi=10.1016/j.drugalcdep.2006.06.017&)

9. Saccone N. L. et al. Multiple independent loci at chromosome 15q25.1 affect smoking quantity: a meta-analysis and comparison with lung cancer and COPD. PLoS Genetics 6, 10.1371/journal.pgen.1001053 (2010).  [DOI](https://doi.org/10.1371/journal.pgen.1001053) | [PMC free article](/articles/PMC2916847/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20700436/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genetics&title=Multiple%20independent%20loci%20at%20chromosome%2015q25.1%20affect%20smoking%20quantity:%20a%20meta-analysis%20and%20comparison%20with%20lung%20cancer%20and%20COPD&author=N.%20L.%20Saccone&volume=6&publication_year=2010&pmid=20700436&doi=10.1371/journal.pgen.1001053&)

10. Hallfors J. et al. Scrutiny of the CHRNA5-CHRNA3-CHRNB4 smoking behavior locus reveals a novel association with alcohol use in a Finnish population based study. International Journal of Molecular Epidemiology and Genetics 4, 109–119 (2013).  [PMC free article](/articles/PMC3709115/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23875064/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=International%20Journal%20of%20Molecular%20Epidemiology%20and%20Genetics&title=Scrutiny%20of%20the%20CHRNA5-CHRNA3-CHRNB4%20smoking%20behavior%20locus%20reveals%20a%20novel%20association%20with%20alcohol%20use%20in%20a%20Finnish%20population%20based%20study&author=J.%20Hallfors&volume=4&publication_year=2013&pages=109-119&pmid=23875064&)

11. Hukkanen J., Jacob P. 3rd & Benowitz N. L. Metabolism and disposition kinetics of nicotine. Pharmacological Reviews 57, 79–115, 10.1124/pr.57.1.3 (2005).  [DOI](https://doi.org/10.1124/pr.57.1.3) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15734728/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacological%20Reviews&title=Metabolism%20and%20disposition%20kinetics%20of%20nicotine&author=J.%20Hukkanen&author=P.%20Jacob&author=N.%20L.%20Benowitz&volume=57&publication_year=2005&pages=79-115&pmid=15734728&doi=10.1124/pr.57.1.3&)

12. Zhu A. Z. et al. Alaska Native smokers and smokeless tobacco users with slower CYP2A6 activity have lower tobacco consumption, lower tobacco-specific nitrosamine exposure and lower tobacco-specific nitrosamine bioactivation. Carcinogenesis 34, 93–101, 10.1093/carcin/bgs306 (2013).  [DOI](https://doi.org/10.1093/carcin/bgs306) | [PMC free article](/articles/PMC3534190/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23027621/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Carcinogenesis&title=Alaska%20Native%20smokers%20and%20smokeless%20tobacco%20users%20with%20slower%20CYP2A6%20activity%20have%20lower%20tobacco%20consumption,%20lower%20tobacco-specific%20nitrosamine%20exposure%20and%20lower%20tobacco-specific%20nitrosamine%20bioactivation&author=A.%20Z.%20Zhu&volume=34&publication_year=2013&pages=93-101&pmid=23027621&doi=10.1093/carcin/bgs306&)

13. Chen G. et al. Glucuronidation genotypes and nicotine metabolic phenotypes: importance of functional UGT2B10 and UGT2B17 polymorphisms. Cancer Research 70, 7543–7552, 10.1158/0008-5472.CAN-09-4582 (2010).  [DOI](https://doi.org/10.1158/0008-5472.CAN-09-4582) | [PMC free article](/articles/PMC2998997/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20876810/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Research&title=Glucuronidation%20genotypes%20and%20nicotine%20metabolic%20phenotypes:%20importance%20of%20functional%20UGT2B10%20and%20UGT2B17%20polymorphisms&author=G.%20Chen&volume=70&publication_year=2010&pages=7543-7552&pmid=20876810&doi=10.1158/0008-5472.CAN-09-4582&)

14. Berg J. Z. et al. UGT2B10 genotype influences nicotine glucuronidation, oxidation, and consumption. Cancer Epidemiology, Biomarkers & Prevention 19, 1423–1431, 10.1158/1055-9965.EPI-09-0959 (2010).  [DOI](https://doi.org/10.1158/1055-9965.EPI-09-0959) | [PMC free article](/articles/PMC2882998/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20501767/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Epidemiology,%20Biomarkers%20&%20Prevention&title=UGT2B10%20genotype%20influences%20nicotine%20glucuronidation,%20oxidation,%20and%20consumption&author=J.%20Z.%20Berg&volume=19&publication_year=2010&pages=1423-1431&pmid=20501767&doi=10.1158/1055-9965.EPI-09-0959&)

15. Bloom A. J. et al. The contribution of common UGT2B10 and CYP2A6 alleles to variation in nicotine glucuronidation among European Americans. Pharmacogenetics and Genomics 23, 706–716, 10.1097/FPC.0000000000000011 (2013).  [DOI](https://doi.org/10.1097/FPC.0000000000000011) | [PMC free article](/articles/PMC3919513/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24192532/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics%20and%20Genomics&title=The%20contribution%20of%20common%20UGT2B10%20and%20CYP2A6%20alleles%20to%20variation%20in%20nicotine%20glucuronidation%20among%20European%20Americans&author=A.%20J.%20Bloom&volume=23&publication_year=2013&pages=706-716&pmid=24192532&doi=10.1097/FPC.0000000000000011&)

16. Zhu A. Z. et al. Variation in trans-3′-hydroxycotinine glucuronidation does not alter the nicotine metabolite ratio or nicotine intake. PloS One 8, e70938, 10.1371/journal.pone.0070938 (2013).  [DOI](https://doi.org/10.1371/journal.pone.0070938) | [PMC free article](/articles/PMC3732272/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23936477/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PloS%20One&title=Variation%20in%20trans-3%E2%80%B2-hydroxycotinine%20glucuronidation%20does%20not%20alter%20the%20nicotine%20metabolite%20ratio%20or%20nicotine%20intake&author=A.%20Z.%20Zhu&volume=8&publication_year=2013&pages=e70938&pmid=23936477&doi=10.1371/journal.pone.0070938&)

17. Patel Y. M. et al. The contribution of common genetic variation to nicotine and cotinine glucuronidation in multiple ethnic/racial populations. Cancer Epidemiology, Biomarkers & Prevention 24, 119–127, 10.1158/1055-9965.EPI-14-0815 (2015).  [DOI](https://doi.org/10.1158/1055-9965.EPI-14-0815) | [PMC free article](/articles/PMC4294952/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25293881/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Epidemiology,%20Biomarkers%20&%20Prevention&title=The%20contribution%20of%20common%20genetic%20variation%20to%20nicotine%20and%20cotinine%20glucuronidation%20in%20multiple%20ethnic/racial%20populations&author=Y.%20M.%20Patel&volume=24&publication_year=2015&pages=119-127&pmid=25293881&doi=10.1158/1055-9965.EPI-14-0815&)

18. Zhu A. Z. et al. The ability of plasma cotinine to predict nicotine and carcinogen exposure is altered by differences in CYP2A6: the influence of genetics, race, and sex. Cancer Epidemiology, Biomarkers & Prevention 22, 708–718, 10.1158/1055-9965.EPI-12-1234-T (2013).  [DOI](https://doi.org/10.1158/1055-9965.EPI-12-1234-T) | [PMC free article](/articles/PMC3617060/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23371292/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Epidemiology,%20Biomarkers%20&%20Prevention&title=The%20ability%20of%20plasma%20cotinine%20to%20predict%20nicotine%20and%20carcinogen%20exposure%20is%20altered%20by%20differences%20in%20CYP2A6:%20the%20influence%20of%20genetics,%20race,%20and%20sex&author=A.%20Z.%20Zhu&volume=22&publication_year=2013&pages=708-718&pmid=23371292&doi=10.1158/1055-9965.EPI-12-1234-T&)

19. Malaiyandi V., Goodz S. D., Sellers E. M. & Tyndale R. F. CYP2A6 genotype, phenotype, and the use of nicotine metabolites as biomarkers during ad libitum smoking. Cancer Epidemiology, Biomarkers & Prevention 15, 1812–1819, 10.1158/1055-9965.EPI-05-0723 (2006).  [DOI](https://doi.org/10.1158/1055-9965.EPI-05-0723) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17035386/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Epidemiology,%20Biomarkers%20&%20Prevention&title=CYP2A6%20genotype,%20phenotype,%20and%20the%20use%20of%20nicotine%20metabolites%20as%20biomarkers%20during%20ad%20libitum%20smoking&author=V.%20Malaiyandi&author=S.%20D.%20Goodz&author=E.%20M.%20Sellers&author=R.%20F.%20Tyndale&volume=15&publication_year=2006&pages=1812-1819&pmid=17035386&doi=10.1158/1055-9965.EPI-05-0723&)

20. Benowitz N. L., Dains K. M., Dempsey D., Yu L. & Jacob P. 3rd. Estimation of nicotine dose after low-level exposure using plasma and urine nicotine metabolites. Cancer Epidemiology, Biomarkers & Prevention 19, 1160–1166, 10.1158/1055-9965.EPI-09-1303 (2010).  [DOI](https://doi.org/10.1158/1055-9965.EPI-09-1303) | [PMC free article](/articles/PMC2867075/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20447913/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Epidemiology,%20Biomarkers%20&%20Prevention&title=Estimation%20of%20nicotine%20dose%20after%20low-level%20exposure%20using%20plasma%20and%20urine%20nicotine%20metabolites&author=N.%20L.%20Benowitz&author=K.%20M.%20Dains&author=D.%20Dempsey&author=L.%20Yu&author=P.%20Jacob&volume=19&publication_year=2010&pages=1160-1166&pmid=20447913&doi=10.1158/1055-9965.EPI-09-1303&)

21. Perez-Stable E. J., Benowitz N. L. & Marin G. Is serum cotinine a better measure of cigarette smoking than self-report? Preventive Medicine 24, 171–179 (1995).  [DOI](https://doi.org/10.1006/pmed.1995.1031) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7597020/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Preventive%20Medicine&title=Is%20serum%20cotinine%20a%20better%20measure%20of%20cigarette%20smoking%20than%20self-report?&author=E.%20J.%20Perez-Stable&author=N.%20L.%20Benowitz&author=G.%20Marin&volume=24&publication_year=1995&pages=171-179&pmid=7597020&doi=10.1006/pmed.1995.1031&)

22. St Helen G., Dempsey D., Wilson M., Jacob P. 3rd & Benowitz N. L. Racial differences in the relationship between tobacco dependence and nicotine and carcinogen exposure. Addiction 108, 607–617, 10.1111/j.1360-0443.2012.04077.x (2013).  [DOI](https://doi.org/10.1111/j.1360-0443.2012.04077.x) | [PMC free article](/articles/PMC3553231/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22971134/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Addiction&title=Racial%20differences%20in%20the%20relationship%20between%20tobacco%20dependence%20and%20nicotine%20and%20carcinogen%20exposure&author=G.%20St%20Helen&author=D.%20Dempsey&author=M.%20Wilson&author=P.%20Jacob&author=N.%20L.%20Benowitz&volume=108&publication_year=2013&pages=607-617&pmid=22971134&doi=10.1111/j.1360-0443.2012.04077.x&)

23. Tanner J. A. et al. Nicotine Metabolite Ratio (3-Hydroxycotinine/Cotinine) in Plasma and Urine by Different Analytical Methods and Laboratories: Implications for Clinical Implementation. Cancer Epidemiology, Biomarkers & Prevention 24, 1239–1246, 10.1158/1055-9965.EPI-14-1381 (2015).  [DOI](https://doi.org/10.1158/1055-9965.EPI-14-1381) | [PMC free article](/articles/PMC4526326/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26014804/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Epidemiology,%20Biomarkers%20&%20Prevention&title=Nicotine%20Metabolite%20Ratio%20(3-Hydroxycotinine/Cotinine)%20in%20Plasma%20and%20Urine%20by%20Different%20Analytical%20Methods%20and%20Laboratories:%20Implications%20for%20Clinical%20Implementation&author=J.%20A.%20Tanner&volume=24&publication_year=2015&pages=1239-1246&pmid=26014804&doi=10.1158/1055-9965.EPI-14-1381&)

24. Vine M. F. et al. Cotinine concentrations in semen, urine, and blood of smokers and nonsmokers. American Journal of Public Health 83, 1335–1338 (1993).  [DOI](https://doi.org/10.2105/ajph.83.9.1335) | [PMC free article](/articles/PMC1694994/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8363014/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=American%20Journal%20of%20Public%20Health&title=Cotinine%20concentrations%20in%20semen,%20urine,%20and%20blood%20of%20smokers%20and%20nonsmokers&author=M.%20F.%20Vine&volume=83&publication_year=1993&pages=1335-1338&pmid=8363014&doi=10.2105/ajph.83.9.1335&)

25. Magi R. & Morris A. P. GWAMA: software for genome-wide association meta-analysis. BMC Bioinformatics 11, 288 (2010).  [DOI](https://doi.org/10.1186/1471-2105-11-288) | [PMC free article](/articles/PMC2893603/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20509871/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Bioinformatics&title=GWAMA:%20software%20for%20genome-wide%20association%20meta-analysis&author=R.%20Magi&author=A.%20P.%20Morris&volume=11&publication_year=2010&pages=288&pmid=20509871&doi=10.1186/1471-2105-11-288&)

26. Yang J., Lee S. H., Goddard M. E. & Visscher P. M. GCTA: a tool for Genome-wide Complex Trait Analysis. American Journal of Human Genetics 88, 76–82 (2011).  [DOI](https://doi.org/10.1016/j.ajhg.2010.11.011) | [PMC free article](/articles/PMC3014363/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21167468/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=American%20Journal%20of%20Human%20Genetics&title=GCTA:%20a%20tool%20for%20Genome-wide%20Complex%20Trait%20Analysis&author=J.%20Yang&author=S.%20H.%20Lee&author=M.%20E.%20Goddard&author=P.%20M.%20Visscher&volume=88&publication_year=2011&pages=76-82&pmid=21167468&doi=10.1016/j.ajhg.2010.11.011&)
