# Sequencing of Charcot-Marie-Tooth disease genes in a toxic polyneuropathy

## Metadata
**Authors:** Andreas S Beutler, Amit A Kulkarni, Rahul Kanwar, Christopher J Klein, Terry M Therneau, Rui Qin, Michaela S Banck, Ganesh K Boora, Kathryn J Ruddy, Yanhong Wu, Regenia L Smalley, Julie M Cunningham, Nguyet Anh Le-Lindqwister, Peter Beyerlein, Gary P Schroth, Anthony J Windebank, Stephan Züchner, Charles L Loprinzi
**Journal:** Annals of neurology
**Date:** 2014 Sep 17
**DOI:** [10.1002/ana.24265](https://doi.org/10.1002/ana.24265)
**PMID:** 25164601
**PMCID:** PMC4388308
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4388308/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC4388308/pdf/nihms624988.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC4388308/pdf/nihms624988.pdf)

## Abstract

**Objective:** 
Mutations in Charcot-Marie-Tooth disease (CMT) genes are the cause of rare familial forms of polyneuropathy. Whether allelic variability in CMT genes is also associated with common forms of polyneuropathy—considered “acquired” in medical parlance—is unknown. Chemotherapy induced peripheral neuropathy (CIPN) occurs commonly in cancer patients and is individually unpredictable. We used CIPN as clinical model to investigate the association of non-CMT polyneuropathy with CMT genes.

**Methods:** 
269 neurologically asymptomatic cancer patients were enrolled in the clinical trial Alliance N08C1 to receive the neurotoxic drug paclitaxel, while undergoing prospective assessments for polyneuropathy. 49 CMT genes were analyzed by targeted massively parallel sequencing of genomic DNA from patient blood.

**Results:** 
119 (of 269) patients were identified from the two ends of the polyneuropathy phenotype distribution: patients that were most- and least susceptible to paclitaxel polyneuropathy. The CMT gene PRX was found to be deleteriously mutated in patients who were susceptible to CIPN but not in controls (p=8×10−3). Genetic variation in another CMT gene, ARHGEF10, was highly significantly associated with CIPN (p=5×10−4). Three non-synonymous recurrent single nucleotide variants contributed to the ARHGEF10 signal: rs9657362, rs2294039, and rs17683288. Of these, rs9657362 had the strongest effect (odds ratio of 4.8, p=4×10−4).

**Interpretation:** 
The results reveal an association of CMT gene allelic variability with susceptibility to CIPN. The findings raise the possibility that other acquired polyneuropathies may also be co-determined by genetic etiological factors, of which some may be related to genes already known to cause the phenotypically related Mendelian disorders of CMT.

### Objective

Mutations in Charcot-Marie-Tooth disease (CMT) genes are the cause of rare familial forms of polyneuropathy. Whether allelic variability in CMT genes is also associated with common forms of polyneuropathy—considered “acquired” in medical parlance—is unknown. Chemotherapy induced peripheral neuropathy (CIPN) occurs commonly in cancer patients and is individually unpredictable. We used CIPN as clinical model to investigate the association of non-CMT polyneuropathy with CMT genes.

### Methods

269 neurologically asymptomatic cancer patients were enrolled in the clinical trial Alliance N08C1 to receive the neurotoxic drug paclitaxel, while undergoing prospective assessments for polyneuropathy. 49 CMT genes were analyzed by targeted massively parallel sequencing of genomic DNA from patient blood.

### Results

119 (of 269) patients were identified from the two ends of the polyneuropathy phenotype distribution: patients that were most- and least susceptible to paclitaxel polyneuropathy. The CMT gene *PRX* was found to be deleteriously mutated in patients who were susceptible to CIPN but not in controls (p=8×10^−3^). Genetic variation in another CMT gene, *ARHGEF10*, was highly significantly associated with CIPN (p=5×10^−4^). Three non-synonymous recurrent single nucleotide variants contributed to the *ARHGEF10* signal: rs9657362, rs2294039, and rs17683288. Of these, rs9657362 had the strongest effect (odds ratio of 4.8, p=4×10^−4^).

### Interpretation

The results reveal an association of CMT gene allelic variability with susceptibility to CIPN. The findings raise the possibility that other acquired polyneuropathies may also be co-determined by genetic etiological factors, of which some may be related to genes already known to cause the phenotypically related Mendelian disorders of CMT.

## INTRODUCTION

Polyneuropathies fall into two main categories, acquired—toxic, inflammatory, nutritional, metabolic, and others—and inherited polyneuropathies, also known as Charcot-Marie-Tooth disease (CMT). The most successful studies of the origin of these diseases, arguably, have been performed for CMT because most, if not all of CMT, is transmitted in a monogenic (dominant, recessive, or X-linked), highly penetrant mode. Approximately 80 genes have been reported to cause CMT, including the 49 established CMT genes that became the focus of the present work.^1–4^ The most recent CMT gene discoveries were possible thanks to the new genome analysis technology of massively parallel short read (“next generation”) sequencing.^4, 5^ These techniques are now standard and allow for comprehensive assessment of gene variants.

As the number of CMT genes increased, it became clear that their value might reach beyond the diagnosis of affected families. In fact, these genes represent key elements in biological mechanisms that are likely also affected in some acquired, non-CMT polyneuropathies.^2, 4^ This notion raises the hypothesis that these genes harbor additional genetic variants that are not associated with CMT, but predispose to acquired forms of polyneuropathies. We explored this possibility by next-generation sequencing of CMT genes in patients with an acquired polyneuropathy, chemotherapy induced peripheral neuropathy (CIPN). CIPN lent itself to the study objective because it can be investigated in well-designed human trials providing a level of control that can resemble experimental laboratory models of neuropathy. Furthermore, CMT patients are predisposed to developing severe CIPN when exposed to vincristine^6–9^ (a drug used in multi-agent chemotherapy of lymphomas) and occasionally have been diagnosed with CMT only after initiation of chemotherapy.^8–14^

CIPN is also an important clinical entity in its own right because it is the most important unmitigated toxicity of numerous cancer drugs such as paclitaxel, frequently impairing patients’ quality of life and occasionally leading to irreversible debility. CIPN cannot be predicted from clinical parameters in neurologically asymptomatic subjects, further suggesting that it may be a high-yield model to search for a genetic basis.

The present study, Alliance N08C1, implemented the above clinical trial design features to test whether CMT gene allelic variability may be associated with susceptibility to (or protection from) developing CIPN. Specifically, we hypothesized that a genetic association would be due to non-CMT alleles (in CMT genes), because we examined patients that were unselected and neurologically normal at study outset and therefore should not carry mutations already known in CMT. Patients in the study received paclitaxel chemotherapy and were phenotyped prospectively by serial (repeat) administration of a previously validated, disease-specific instrument, the European Organization for Research and Treatment of Cancer (EORTC) Quality of Life Chemotherapy-Induced Peripheral Neuropathy 20 question questionnaire (CIPN20), quantifying symptoms of peripheral neuropathy.^15–17^ A pre-determined set of 49 canonical CMT genes were analyzed by massively parallel, short-read sequencing of germline DNA. CMT gene sequencing results were then statistically tested for an association of common and rare single nucleotide variants (SNV) with the development of peripheral neuropathy.

## PATIENTS AND METHODS

### Patients

NCCTG N08C1 is an observational study of CIPN in patients receiving paclitaxel at a dose of either 70-90 mg/m^2^ weekly or 175 mg/m^2^ every 3 weeks with or without carboplatin; clinical observations from the study have been reported previously.^18, 19^ The study was performed by the North Central Cancer Treatment Group (NCCTG), an NIH supported multi-institutional clinical trials consortium, which recently joined the Alliance for Clinical Trials in Oncology (Alliance). Patients were at least 18 years old and were able to provide written consent and to complete study questionnaires. They had to have a life expectancy of greater than 6 months and an Eastern Cooperative Oncology Group (ECOG) performance status of 0 or 1. Patients were excluded if they had been diagnosed with peripheral neuropathy (from any cause) or fibromyalgia; were to receive concurrent neutrophil colony-stimulating factor therapy; or had previous exposure to paclitaxel or other neurotoxic chemotherapy drugs. Informed consent was obtained from all patients for the serial CIPN phenotype assessment and for collection of blood for genetic analysis (Mayo IRB# 08-006970). Patient characteristics are shown in [Table 1](#T1).

### Table 1.

|   |   | Control | Cases |
| --- | --- | --- | --- |
| Age | Mean [years] | 56.6 | 56.5 |
| Sex | Female | 89% | 92% |
| Male | 11% | 8% |  |
| Ethnicity | Caucasian | 76% | 76% |
| African American | 15% | 18% |  |
| Others/unknown | 9% | 6% |  |
| Primary site | Breast | 61% | 63% |
| Ovarian/Fallopian | 20% | 14% |  |
| Lung | 13% | 10% |  |
| Head and neck | 4% | 4% |  |
| Endometrial | 0.0% | 4% |  |
| Others | 2% | 6% |  |
| Tumor status | Recurrent | 4% | 8% |
| Resected with known residual | 15% | 15% |  |
| Resected with no residual | 59% | 49% |  |
| Unresected | 22% | 28% |  |
| Diabetes mellitus |  | 11.6% | 12.5% |

Table 1 Caption: Patient characteristics

### Selection of CIPN controls and cases by a study-wide phenotype model

CIPN was phenotypically assessed with the CIPN20, a dedicated instrument that has been well validated.^15–17^ The CIPN20 solicits patient responses to 20 questions about neuropathy symptoms on a four-tier ordinal scale (1=“Not at All”; 2=“A Little”; 3=“Quite a Bit”; 4=“Very Much”).

The CIPN20 tool was administered prior to each chemotherapy treatment. Patients were classified as “uncategorized”, “control”, or “case” based on the rate of symptom progression as follows. A study-wide statistical model was fit to the raw CIPN20 response data employing a Rasch method.^20^ The procedure is designed for the analysis of categorical response data and can accommodate missing values. Each patient/question response is modeled as the sum of two effects, the underlying (“true”) patient CIPN state of susceptibility and a question specific modifier. In our analysis the patient state of susceptibility was summarized by a slope over time (the rate of CIPN onset). A shallow slope (no symptoms arising or a slow onset) is found in patients, who are resistant to developing CIPN and a steep slope in patients susceptible to CIPN. “CIPN susceptibility score” refers to this slope. A standard error and corresponding confidence intervals (CI) were computed for each CIPN susceptibility score providing a measure of the precision of the CIPN susceptibility assessment. Patients whose 95% confidence interval (CI) overlapped with the median slope for their respective treatment regimen with the N08C1 study, i.e., patients with susceptibility scores that were not statistically significantly different from the group median, were classified as “uncategorized”. All other patients had a non-overlapping CI indicating that their CIPN susceptibility was either significantly lower or significantly higher than the study average; these patients were accordingly defined as “controls” and CIPN “cases”. Patient selection and CIPN20 are depicted in [Fig 1](#F1).

### Figure 1. Neuropathy phenotyping by serial symptom assessment in paclitaxel-treated patients to score CIPN susceptibility.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/187d/4388308/24dd68748a5b/nihms624988f1.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4388308_nihms624988f1.jpg)

A. A CIPN susceptibility score was determined for each of the n=248 patients in N08C1, for whom three (or more) serial CIPN20 assessments were available. All patients were treated with paclitaxel for up to twelve weeks. Patients with scores that were significantly different from the study median of 1.0 were chosen for exome sequencing, n=119. Patients with a significantly higher score constituted CIPN-susceptible cases and patients with a significantly lower score were the controls. Patients with intermediate susceptibility scores (close to 1), or few or missing data points, or unreliable responses (indicated by large standard errors), n=129, remained uncategorized. Thereby the approach yielded two “extreme phenotype” groups for genetic comparison. Data points shown are the CIPN susceptibility scores with standard errors for all patients normalized to the median slope for the specific paclitaxel regimen received: paclitaxel single agent (top); paclitaxel with carboplatin (bottom); weekly administration (circles); every-three-week administration (squares). B. Raw CIPN20 scores are shown for two illustrative patients, a control- and a CIPN subject over a period of up to twelve weekly administrations of paclitaxel. The heat map depicts responses to the CIPN20 questionnaire items in rows and time points (weekly) in columns. Both patients were free of neuropathy symptoms at study outset as confirmed by their response “not at all” (a score of 1) to all questions at the earliest time point. Subsequently one of the patients continued to answer “not at all” to almost all CIPN20 items in the twelve repeat assessments indicating lack of susceptibility to neuropathy (or “resistance” to neuropathy); as a consequence, this patient was assigned to the control group. The other depicted patient responded with scores rising from week to week (yellow, red), indicating early onset of neuropathy and progressive worsening over time; treatment was terminated after fewer than twelve treatments. This patient was classified as belonging to the group of cases, susceptible to CIPN, because the slope of worsening of CIPN symptoms was steep.

### CMT gene sequencing and bioinformatic SNV calling

Genomic DNA (gDNA) was isolated from peripheral blood leukocytes and used to construct sequencing libraries with the commercially available TruSeq reagent kit (Illumina). DNA was subjected to acoustic shearing (Covaris) with a target fragment size peak of 300–400bp followed by end repair. Bar-coded adapters were used. The quality of libraries was confirmed by electrophoresis on an Agilent bio-analyzer. Exome enrichment was carried out using the TruSeq exome enrichment kit (Illumina) pooling six libraries (each labeled by barcoded adapters) per enrichment reaction. Libraries of cases and controls were processed in random order and randomly assigned to pools to preclude the possibility of a batch effect. Each pool was sequenced twice in one flow cell lane on a HiSeq 2000 genome analyzer (Illumina) in paired-end mode to a length of 100bp x 2.

Raw sequencing reads were pre-processed by trimming adapter sequences and nucleotides at the end of a read with a phred score of <20. Reads were then aligned to the human reference genome (grch37) using Novoalign version 2.07 ([http://novocraft.com](http://novocraft.com)) with the following flags (novoalign --hdrhd off -v 120 -c 4 -i PE 425,80 -x 5 -r Random –d). PCR duplicates were marked using Picard ([http://picard.sourceforge.net](http://picard.sourceforge.net)). The Genome Analysis Toolkit (GATK) version 2.2^21^ was employed for local read realignment, base quality score recalibration (BQSR), multi-sample SNV calling across the exome target region (demarcated by the Illumina True Seq BED file [http://support.illumina.com/sequencing/sequencing_kits/truseq_exome_enrichment_kit/downloads.ilmn](http://support.illumina.com/sequencing/sequencing_kits/truseq_exome_enrichment_kit/downloads.ilmn)) resulting in a study-wide variant call format (VCF) file containing all candidate SNV. SNV were then ranked by variant quality score recalibration (VQSR) according to the variant quality log-odds (VQSLOD). A VQSLOD corresponding to a predicted study-wide specificity of >99.9% and sensitivity >90% was chosen to select the final SNV set for subsequent analysis. The PLINK/SEQ version 0.08 ([http://atgu.mgh.harvard.edu/plinkseq](http://atgu.mgh.harvard.edu/plinkseq)) genomic database environment was used for subsequent high-level analyses. SNV were annotated using SnpEff version 3.2^22^ and ANNOVAR.^23^ Annotations provided are based on Ensemble GRCh37.75. Intronic and synonymous (i.e., silent) SNV were omitted from subsequent high-level analyses because those are unlikely to have any biological impact. Accordingly, the CMT gene analysis described below was performed with non-silent SNV: SNV altering the encoded protein, also termed non-synonymous SNV, which consisted of missense and non-sense SNV.

### Orthogonal validation of SNV by traditional methods

TaqMan PCR was used in conjunction with the SDS 2.0 software (Applied Biosystems) to analyze all patient DNA sampled for validation of the recurrent SNV detected in *ARHGEF10*. Sanger sequencing was used for validation of the singleton SNV detected by sequencing in *PRX*.

### Statistical testing of the association of HN genes with CIPN

The primary endpoint, predefined before sequencing commenced, was to test the association of recurrent SNV in CMT genes with CIPN. 49 CMT genes were judged to be canonical (thoroughly established) CMT genes at the time when the primary endpoint was set. While the number of CMT genes is larger by most current accounts (as alluded to in the introduction), the primary analysis of the present study kept the analysis plan focused on the CMT genes shown in [Fig 2C](#F2). For the same reason, keeping the prospectively defined analysis plan unaltered, the gene *KIF1B* was not removed from the set of 49 genes even though a further appraisal of the literature made it clear that its link to CMT is questionable.^24, 25^ The present study did not find any evidence to link *KIF1B* to CIPN and thus its likely erroneous inclusion at the outset did not affect the results. Genes found to harbor at least one recurrent SNV in the present study, were tested for an association with CIPN by C-alpha,^26^ a test that has been used in other recent nextgen sequencing studies. It is implemented in the PLINK/SEQ analysis environment employed in the study. Each CMT gene was tested separately for an association with CIPN by C-alpha.

### Figure 2. CMT gene categorization and -testing.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/187d/4388308/d70cb468de2a/nihms624988f2.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4388308_nihms624988f2.jpg)

A. Sequencing read coverage. The genomic target region consisting of 235 kB nucleotides covering the 801 exons of 49 canonical CMT genes was sequenced at a multiplicity (“depth”) exceeding the requirement of >20-fold unique read coverage at >80% of base pair positions. Shown is the average and 95% CI for all study samples. B. SNV discovery and grouping. SNV were categorized according to whether they occurred only in one patient in the study (“singleton”) or were seen in several patients (“recurrent”). SNV counts across the target region are shown for the entire study (left panel) and as average per patient (right panel). Distinct singleton SNV were nearly as common as distinct recurrent SNV. However, because the same recurrent SNV occurs in several patients, recurrent SNV are in aggregate far more commonly found than singleton SNV as shown on the right. SNV were further sub-grouped into synonymous SNV, expected to be biologically silent in most cases, and non-synonymous SNV, i.e., missense or non-sense SNV, which are predicted to alter the encoded protein. All subsequent analyses were based on non-synonymous SNV. C. Categorization of CMT genes according to non-synonymous SNV. CMT genes were categorized according to whether they harbored singleton and/or recurrent SNV. Only non-synonymous SNV were considered. D. Burden testing of CMT genes for singleton SNV association with CIPN-susceptibility. A p-p plot shows the results of Burden tests performed on genes harboring ≥5 singleton SNV. The gene PRX was found to be associated with CIPN-susceptibility. E. C-alpha testing of CMT genes for recurrent SNV association with CIPN-susceptibility. A p-p plot shows the results of C-alpha tests performed on genes harboring ≥1 recurrent SNV. The gene ARHGEF10 was found to be associated with CIPN-susceptibility.

Under a secondary endpoint, a mutational Burden test was performed (as implemented in the PLINK/SEQ environment) for genes harboring a non-silent singleton SNV in ≥5 study patients.

## RESULTS

### CIPN susceptibility determined by serial neuropathy phenotyping

Clinical patient characteristics are provided in [Table 1](#T1). Using the EORTC-CIPN20 instrument for serial neuropathy phenotyping we identified 73 CIPN-susceptible patients and 46 controls for genetic analysis out of a cohort of 269 patients ([Fig 1](#F1)). [Fig 1A](#F1) shows individual CIPN susceptibility scores in each group with standard error for each patient indicating the reliability of the serial measurements in the subject. The two groups subjected to genetic analysis represented the phenotypic extremes as patients with intermediate scores or unreliable responses were excluded by the method. The phenotyping was carried out prospectively and repeated serially on average 8.8 times per patient to establish the time-course of symptoms ([Fig 1B](#F1)). The CIPN susceptibility score can be understood as a slope: symptoms as a function of chemotherapy cycle. Slopes were computed using a Rasch-type study-wide model.

### Categorization of SNV and CMT genes

Sequencing read depth for the 119 patients met criteria for efficient SNV discovery ([Fig 2A](#F2)). We identified 517 unique SNV study-wide in the coding region of the 49 CMT genes ([Fig 2B–C](#F2)). Of these, 272 were synonymous SNV, i.e., predicted to be silent in regards to the encoded protein; these were excluded from subsequent analyses. 245 non-silent SNV were found: 242 missense SNV and 3 non-sense SNV. Of the non-silent SNV, 115 were identified in several patients in the study cohort: termed “recurrent” SNV. The remaining 130 SNV were found only in one patient, termed “singleton” SNV ([Fig 2B](#F2)). On average, each patient harbored 22 non-synonymous SNV across the 49 genes ([Fig 2B](#F2)).

The 49 CMT genes could be categorized according to whether they harbored singleton and/or recurrent SNV ([Fig 2C](#F2)). 10 genes harbored ≥5 singleton SNV and 32 genes harbored ≥1 recurrent SNV.

### CMT gene association with CIPN

The CMT genes harboring ≥5 singleton SNV were assessed for an association with susceptibility to CIPN by mutational Burden testing, which yielded the significance values shown as p-p plot in [Fig 2D](#F2). In a p-p plot a comparison is made between the observed results and a distribution of hypothetical significance values that would be expected to occur by chance. Points falling along the diagonal line are expected for the majority of genes, while a point found far above the diagonal is evidence for a true association supporting the link, in this case, of singleton SNV in *PRX* with susceptibility to CIPN. The test result for the gene periaxin (*PRX*) was above the diagonal line with a significance value of p=0.008.

The CMT genes harboring ≥1 recurrent SNV were assessed for an association with susceptibility to CIPN by the C-alpha test. The corresponding p-p plot is shown as [Fig 2E](#F2). The CMT gene *ARHGEF10* was found to be highly significantly associated with CIPN with p=5×10^−4^. The analysis of CMT genes harboring recurrent variants with the C-alpha test was the prospectively defined, primary endpoint of the study. Adjustment for multiple testing by the Bonferroni method yielded a study-wide significance level of p=0.016.

### SNV in PRX

*PRX* was found to be disrupted by unique non-synonymous singleton SNV in 8 of the patients in the CIPN-susceptible group and in none of the patients in the control groups. Of the 8 singleton SNV, 7 were predicted to be deleterious by two independent prediction algorithms, Polyphen^27^ and Sift.^28^ Therefore, non-synonymous singleton SNV in *PRX* had characteristics of “mutations” disrupting gene function. Interestingly, 7 of these SNV were in exon 7, where the majority of known CMT mutations are found.^29–33^ Yet, none of the identified rare variants were reported previously in Mendelian CMT families. We also did not identify compound heterozygous variation in our patients as would be expected in CMT patients. The presence of an increased “burden” of coding non-synonymous variation in *PRX* in a heterozygous state illustrates current challenges of defining “modifier variation” and “CMT mutations” in individual patients. Details and validation of the identified variants are shown in [Fig 3A](#F3).

### Figure 3. Allelic variants in CIPN-susceptibility associated CMT genes.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/187d/4388308/40a1daf7bd17/nihms624988f3.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4388308_nihms624988f3.jpg)

A. PRX singleton SNV. Of the CIPN-susceptible patients, 8 carried a non-synonymous SNV in PRX. The protein encoded by PRX is shown with the positions of SNV found in CIPN susceptible patients (blue) and sites of known CMT mutations42 (grey). B. Validation by Sanger sequencing. Individual SNV in PRX were confirmed by Sanger sequencing. Shown are stacks of next generation sequencing reads from the present study and confirmatory electrophoretic tracings obtained by capillary sequencing for the heterozygous variant at chr19:40909664 in a CIPN-susceptible patient and the reference genotype (in another patient). C. AHRGEF10 recurrent SNV. Three recurrent non-synonymous SNV were detected. The observed MAF was similar to the MAF reported in the 1000 Genomes Project and NHLBI_6500 exome sequencing reference datasets. The three SNV were not correlated (occur independently) in reference cohorts as shown in a linkage disequilibrium (LD) plot generated from 1000 Genomes Project data. D. Observed frequency of hetero- and homozygous genotypes and validation by TaqMan PCR. For each SNV the relative frequencies of the three possible genotypes observed in the study (homozygous for the major allele, or heterozygous, or homozygous for the minor allele) were consistent with the Hardy Weinberg equilibrium. The accuracy of SNV was confirmed by orthogonal laboratory techniques. E. AHRGEF10 alleles in CIPN-susceptible patients and controls. The association of ARHGEF10 alleles with CIPN-susceptibility was dominated by rs9657362. Absence of its minor allele was strongly associated with CIPN-susceptibility with an OR=4.8; the same result can also be expressed as the inverse, i.e., OR=0.21, to mean that the minor allele was associated with control group status, i.e. it was protective. The association of rs9657362 was statistically highly significant with p=4×10−4. The SNV rs17683288 and rs2294039 showed a trend toward a protective and a risk-effect respectively. A Venn diagram shows the frequency of all possible AHRGEF10 genotypes (all 8 possible combinations of carrying a minor allele at rs9657362, rs17683288 and rs2294039) among CIPN-susceptible and control patients.

### SNV in ARHGEF10

*ARHGEF10* was identified by C-alpha,^26^ a test that can detect the association of a combination of recurrent risk- and protective SNV in a gene with a trait. This was the pre-specified primary endpoint and the pre-specified testing method of the study. Per-gene testing can provide greater statistical power than testing of individual SNV;^34^ but a positive per-gene result such as for *ARHGEF10* may implicate several genetic variants that require further investigation. In the case of *ARHGEF10*, three non-synonymous recurrent SNV contributed to the association test: rs9657362, rs2294039, and rs17683288.

For each of the variants, sequencing was informative in all patients. The technical results passed all quality control criteria. Sequencing depth was 59.3 to 72.5 fold on average for the three SNV. Major and minor allele detection rates were consistent with those reported in two large reference datasets ([Fig 3C](#F3)). In reference studies, the three SNV were not in linkage disequilibrium (LD) meaning that minor alleles at rs9657362, rs2294039, and rs17683288 could be expected to occur independently ([Fig 3C](#F3)). The counts of heterozygous and homozygous genotypes across the study population was consistent with the Hardy Weinberg equilibrium ([Fig 3D](#F3)).

Of the three SNV, rs9657362 fulfilled criteria for a strong association with CIPN: The odds ratio (OR) was 4.8 with a significance of p<10^−3^. The minor allele was associated with protection against CIPN (control group status).

Results for rs2294039 and rs17683288 suggested a risk- and a protective- effect respectively, which appeared weaker than the effect of rs9657362. An exploratory analysis of all possible *AHRGEF10* genotypes, up to 8 possible combinations of major and minor alleles for the three SNV in each individual, raised the possibility that a protective effect of rs17683288 may be seen if it occurs alongside rs9657362, because patients with a combination of the two protective alleles at rs17683288 and rs9657362 appeared to be most strongly protected from CIPN. These results are depicted in [Fig 3E](#F3).

### Analysis of other CMT genes for previously reported mutations

All 49 CMT genes analyzed in the present study were reviewed and no previously reported, canonical CMT mutations were found in cases or controls. Because the study enrolled unselected patients that were asymptomatic at study outset, finding a CMT mutation in a cohort of this size was unlikely and therefore this result could have been considered an expected outcome.

## DISCUSSION

In the present study CIPN served as a clinical research model to investigate, whether allelic variability in CMT genes might be linked to an acquired toxic neuropathy. Why would the search for a genetic basis of an acquired neuropathy focus on CMT genes? Mendelian CMT genes have been identified by unbiased methods from linkage studies to whole exome analysis. Further, family segregation studies have allowed for rigorous testing of clinical relevance. Each known CMT gene, however rare a cause of peripheral neuropathies, thus represents a key player in the pathophysiology of peripheral nerve degeneration. We argue that the biology of peripheral nerve maintenance, reaction to “stress”, and eventual degeneration is shared by a limited set of pathways and gene networks. The growing number of validated CMT genes, thus provides a “keyhole view” into these mechanisms and provided a framework that could be tested in the current study.^3^

By using state of the art genomic analysis tools our study provided evidence in support of this hypothesis by identifying candidate associations of two CMT genes with CIPN-susceptibility. The study included 269 individuals and targeted sequencing was performed on 119 of them, which were carefully selected to represent a group of highly CIPN-susceptible patients and a group of CIPN-resistant controls. This study design and size is comparable to few very recent next generation sequencing reports for other non-Mendelian disorders. Notably, sequencing-based studies were successful even in small cohorts such as the National Heart, Lung, and Blood Institute (NHLBI) acute lung injury exome sequencing study consisting of 88 subjects (discussed in^35^) and the NHLBI chronic *P. aeruginosa* infection study with 91 subjects.^36^ In the present study, we used exome sequencing technology because it was the most economically efficacious approach to test the primary endpoint: allelic variability in CMT genes.

CIPN provided a highly controlled research model of an acquired, toxic neuropathy. Because CIPN is iatrogenic, the causal toxicity was well standardized across a group of patients resembling a controlled laboratory experiment. CIPN developed rapidly over only a few weeks in all susceptible patients minimizing the effects of other clinical variables. Neurologically normal patients were enrolled in the study, a cohort specifically chosen to exclude patients with clinical CMT. The incidence of CIPN was high in the study population making it a truly common neuropathy. Most importantly, phenotyping for CIPN could be carried out prospectively and was repeated serially to establish the time-course of symptoms. Importantly, the study control group could be selected from the same, prospectively accrued cohort, from which the neuropathy cases were drawn. These methodological study design features resemble a laboratory environment in terms of experimental induction of neuropathy. The rate of progression of polyneuropathy over the study period (slope of symptom progression) was quantified with a patient reported outcome instrument, the CIPN20, which was developed and validated by neurologist-led teams for the assessment of peripheral neuropathy in cancer patients.^15, 37^ In the present study, patients were not assessed by electrophysiological testing and no other serial examination data was collected, which limits the value of the study to some extent. Such additional exams may lead to improved studies in the future. Yet, the CIPN20 methodology (and other patient reported outcome measures) was recently demonstrated to cross validate well with neurological exams (including vibration sensory testing) leading the authors of the study to conclude that patient reported- and physician assessed outcomes were “two sides of the same coin” similarly quantifying the polyneuropathy of CIPN.^37^ Furthermore, it is a well-known issue that standardization of clinical exams across multiple examiners can be challenging especially in the assessment of polyneuropathy and may compromise the reliability of research data.^38^ The present study was performed at multiple cancer centers (as is most common for larger trials in oncology), where standardized administration of the CIPN20 was performed 8.8 times on average per patient (over the course of chemotherapy treatment), providing a comprehensive set of data that could be rigorously analyzed to select cases and controls according to individual CIPN-susceptibility.

*PRX* was nominated as CIPN-gene through singleton, non-synonymous SNV in the present study, shown in detail in [Fig 3 A–B](#F3). 7 out of 8 variants were present in exon 7, which also contains most of the reported CMT mutations. Given that *PRX* is a recessive CMT gene with a presumed loss of function effect, the identified heterozygous changes in the current study may represent a haploinsufficiency mechanism of action. Importantly, the *PRX* finding emphasizes the importance of including rare SNV in the design of future studies on the genetic basis of CIPN and other acquired neuropathies. Clearly, better approaches to differentiate modifier alleles from mutations are required. It is, however, conceivable that known *PRX* CMT mutations in carriers could lead to increased susceptibility to polyneuropathy after application of paclitaxel. The *PRX* gene product in Schwann cells is thought to have a role in the stabilization of myelin in the peripheral nervous system. Mice lacking a functional *PRX* gene assemble compact PNS myelin. However, the sheath is unstable, leading to demyelination and reflex behaviors that are associated with painful conditions caused by peripheral nerve damage.^39^

*ARHGEF10* was linked to CIPN-susceptibility with a strongly significant association, p=5×10^−4^. The result was observed under the pre-specified primary study endpoint: testing of canonical CMT genes by C-alpha. *ARHGEF10* regulates actin cytoskeleton- and microtubule dynamics and is involved in neuronal morphogenesis, including cell migration; axon growth; and axonal guidance. It has a role in peripheral nerve myelination through RhoA signaling.^40^ Mutations in *ARHGEF10* are associated with slowed nerve conduction velocity with autosomal dominant segreation.^41^ Given that the *ARHGEF10* result were driven by three independent SNV, it appears reasonable to speculate that future personalized medicine approaches will develop risk algorithms (not necessarily additive in nature) based on major effect alleles. A simplified schematic is provided in a Venn diagram in [Fig 3E](#F3), illustrating the proportion of patients that carry more than one risk allele. Such approaches may be used to stratify patients prior to chemotherapy to balance cancer-therapy outcome with potential side effects via drug choice and dosage. As genomes may soon be available to most cancer patients at nominal cost (or as a by-product of tumor sequencing), refinement of CIPN risk assessment might be among the first clinically relevant—and further prospectively testable—applications of the new technology.

In the case of *ARHGEF10* two of the SNV provide protective effects and thus represent attractive considerations for drug targeting. Potentially, adjuvant drugs given in combination with regular chemotherapy might be able to mitigate unwarranted side effects in the future. Thus, one of the important outcomes of this study is the identification of alleles that should be targeted in future functional follow-up studies. Because this part of the analysis was performed under a secondary endpoint, it should be considered preliminary requiring further study.

In numerous CMT genes, the present study did either not detect any non-synonymous SNV or did not demonstrate an association with CIPN. Future studies on CIPN genetics may be able to identify additional CMT genes, which may have been missed in the present study for several possible reasons including an overall low allele frequency amongst the world population, incomplete coverage of the target gene region by sequencing reads, or the overall size of the present cohort.

The findings of the present study raise the possibility that other “acquired” polyneuropathies such as from diabetes or chronic alcoholism may also be co-determined by genetic etiological factors, of which some may be related to genes already known to cause the phenotypically related Mendelian disorders of CMT. Future studies will be required to test this notion, while implementing a study design that follows patients prospectively (i.e., to assess individual susceptibility to a disorder such as diabetic neuropathy) may require a longer time frame than the present study (years instead of months). The present study on CIPN may provide an outline of what results could be expected, which may support the design of such ambitious future projects in other acquired polyneuropathies.

Finally, the identified alleles should inform the development of animal models for CIPN. Preclinical testing of side effects is best carried out in precise genetic models of the phenotype in question. Our work indicates a path to study and prevent CIPN in a targeted, hypothesis driven manner.

In conclusion, this study confirms the connectedness of distinctive areas of medicine. Much progress can be made by crossing fields that are traditionally distant, such as neurology and oncology or inherited peripheral neuropathies and toxic peripheral neuropathies. As genomic research rapidly identifies the key nodes and edges of a connected biology, one may expect an increase in the pace of discovery and translation of meaningful results into clinical practice.

## Acknowledgments

A.S.B. is supported by grants from the National Institute of Neurological Disorders and Stroke (NINDS) of the National Institutes of Health (NIH) (R01NS063022), the Schulze Family Foundation, and the Mayo Clinic Center for Individualized Medicine. S.Z is supported by grants from the NINDS of the NIH (U54NS0657, R01NS075764, R01NS072248) and the Muscular Dystrophy Association. C.L.L. is supported by a grant from the National Cancer Institute (NCI) of the NIH (U10CA37404-27).

## Footnotes

## References

1. Klein CJ, Duan X, Shy ME. Inherited neuropathies: clinical overview and update. Muscle & nerve. 2013 Oct;48(4):604–22. doi: 10.1002/mus.23775.  [DOI](https://doi.org/10.1002/mus.23775) | [PMC free article](/articles/PMC3918879/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23801417/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Muscle%20&%20nerve&title=Inherited%20neuropathies:%20clinical%20overview%20and%20update&author=CJ%20Klein&author=X%20Duan&author=ME%20Shy&volume=48&issue=4&publication_year=2013&pages=604-22&pmid=23801417&doi=10.1002/mus.23775&)

2. Saporta AS, Sottile SL, Miller LJ, Feely SM, Siskind CE, Shy ME. Charcot-Marie-Tooth disease subtypes and genetic testing strategies. Annals of neurology. 2011 Jan;69(1):22–33. doi: 10.1002/ana.22166.  [DOI](https://doi.org/10.1002/ana.22166) | [PMC free article](/articles/PMC3058597/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21280073/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Annals%20of%20neurology&title=Charcot-Marie-Tooth%20disease%20subtypes%20and%20genetic%20testing%20strategies&author=AS%20Saporta&author=SL%20Sottile&author=LJ%20Miller&author=SM%20Feely&author=CE%20Siskind&volume=69&issue=1&publication_year=2011&pages=22-33&pmid=21280073&doi=10.1002/ana.22166&)

3. Zuchner S, Vance JM. Mechanisms of disease: a molecular genetic update on hereditary axonal neuropathies. Nature clinical practice Neurology. 2006 Jan;2(1):45–53. doi: 10.1038/ncpneuro0071.  [DOI](https://doi.org/10.1038/ncpneuro0071) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16932520/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature%20clinical%20practice%20Neurology&title=Mechanisms%20of%20disease:%20a%20molecular%20genetic%20update%20on%20hereditary%20axonal%20neuropathies&author=S%20Zuchner&author=JM%20Vance&volume=2&issue=1&publication_year=2006&pages=45-53&pmid=16932520&doi=10.1038/ncpneuro0071&)

4. Timmerman V, Strickland AV, Zuchner S. Genetics of Charcot-Marie-Tooth (CMT) Disease within the Frame of the Human Genome Project Success. Genes. 2014;5(1):13–32. doi: 10.3390/genes5010013.  [DOI](https://doi.org/10.3390/genes5010013) | [PMC free article](/articles/PMC3978509/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24705285/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genes&title=Genetics%20of%20Charcot-Marie-Tooth%20(CMT)%20Disease%20within%20the%20Frame%20of%20the%20Human%20Genome%20Project%20Success&author=V%20Timmerman&author=AV%20Strickland&author=S%20Zuchner&volume=5&issue=1&publication_year=2014&pages=13-32&pmid=24705285&doi=10.3390/genes5010013&)

5. Klein CJ, Middha S, Duan X, et al. Application of whole exome sequencing in undiagnosed inherited polyneuropathies. Journal of neurology, neurosurgery, and psychiatry. 2014 Mar 6; doi: 10.1136/jnnp-2013-306740.  [DOI](https://doi.org/10.1136/jnnp-2013-306740) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24604904/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20of%20neurology,%20neurosurgery,%20and%20psychiatry&title=Application%20of%20whole%20exome%20sequencing%20in%20undiagnosed%20inherited%20polyneuropathies&author=CJ%20Klein&author=S%20Middha&author=X%20Duan&publication_year=2014&pmid=24604904&doi=10.1136/jnnp-2013-306740&)

6. Dickerhoff R, Lindner W, Scheiber W. Severe vincristine neurotoxicity in a patient with Charcot-Marie-Tooth disease. Pediatric hematology and oncology. 1988;5(1):61–4. doi: 10.3109/08880018809031253.  [DOI](https://doi.org/10.3109/08880018809031253) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/3152952/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pediatric%20hematology%20and%20oncology&title=Severe%20vincristine%20neurotoxicity%20in%20a%20patient%20with%20Charcot-Marie-Tooth%20disease&author=R%20Dickerhoff&author=W%20Lindner&author=W%20Scheiber&volume=5&issue=1&publication_year=1988&pages=61-4&pmid=3152952&doi=10.3109/08880018809031253&)

7. Graf WD, Chance PF, Lensch MW, Eng LJ, Lipe HP, Bird TD. Severe vincristine neuropathy in Charcot-Marie-Tooth disease type 1A. Cancer. 1996 Apr 1;77(7):1356–62. doi: 10.1002/(SICI)1097-0142(19960401)77:7<1356::AID-CNCR20>3.0.CO;2-#.  [DOI](https://doi.org/10.1002/(SICI)1097-0142(19960401)77:7<1356::AID-CNCR20>3.0.CO;2-#) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8608515/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer&title=Severe%20vincristine%20neuropathy%20in%20Charcot-Marie-Tooth%20disease%20type%201A&author=WD%20Graf&author=PF%20Chance&author=MW%20Lensch&author=LJ%20Eng&author=HP%20Lipe&volume=77&issue=7&publication_year=1996&pages=1356-62&pmid=8608515&doi=10.1002/(SICI)1097-0142(19960401)77:7<1356::AID-CNCR20>3.0.CO;2-#&)

8. Hogan-Dann CM, Fellmeth WG, McGuire SA, Kiley VA. Polyneuropathy following vincristine therapy in two patients with Charcot-Marie-Tooth syndrome. JAMA: the journal of the American Medical Association. 1984 Nov 23–30;252(20):2862–3.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/6092732/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA:%20the%20journal%20of%20the%20American%20Medical%20Association&title=Polyneuropathy%20following%20vincristine%20therapy%20in%20two%20patients%20with%20Charcot-Marie-Tooth%20syndrome&author=CM%20Hogan-Dann&author=WG%20Fellmeth&author=SA%20McGuire&author=VA%20Kiley&volume=252&issue=20&publication_year=1984&pages=2862-3&pmid=6092732&)

9. Naumann R, Mohm J, Reuner U, Kroschinsky F, Rautenstrauss B, Ehninger G. Early recognition of hereditary motor and sensory neuropathy type 1 can avoid life-threatening vincristine neurotoxicity. British journal of haematology. 2001 Nov;115(2):323–5. doi: 10.1046/j.1365-2141.2001.03126.x.  [DOI](https://doi.org/10.1046/j.1365-2141.2001.03126.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11703329/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=British%20journal%20of%20haematology&title=Early%20recognition%20of%20hereditary%20motor%20and%20sensory%20neuropathy%20type%201%20can%20avoid%20life-threatening%20vincristine%20neurotoxicity&author=R%20Naumann&author=J%20Mohm&author=U%20Reuner&author=F%20Kroschinsky&author=B%20Rautenstrauss&volume=115&issue=2&publication_year=2001&pages=323-5&pmid=11703329&doi=10.1046/j.1365-2141.2001.03126.x&)

10. Mercuri E, Poulton J, Buck J, et al. Vincristine treatment revealing asymptomatic hereditary motor sensory neuropathy type 1A. Archives of disease in childhood. 1999 Nov;81(5):442–3. doi: 10.1136/adc.81.5.442.  [DOI](https://doi.org/10.1136/adc.81.5.442) | [PMC free article](/articles/PMC1718125/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10519723/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Archives%20of%20disease%20in%20childhood&title=Vincristine%20treatment%20revealing%20asymptomatic%20hereditary%20motor%20sensory%20neuropathy%20type%201A&author=E%20Mercuri&author=J%20Poulton&author=J%20Buck&volume=81&issue=5&publication_year=1999&pages=442-3&pmid=10519723&doi=10.1136/adc.81.5.442&)

11. Nakamura T, Hashiguchi A, Suzuki S, Uozumi K, Tokunaga S, Takashima H. Vincristine exacerbates asymptomatic Charcot-Marie-tooth disease with a novel EGR2 mutation. Neurogenetics. 2012 Feb;13(1):77–82. doi: 10.1007/s10048-012-0313-1.  [DOI](https://doi.org/10.1007/s10048-012-0313-1) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22271166/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neurogenetics&title=Vincristine%20exacerbates%20asymptomatic%20Charcot-Marie-tooth%20disease%20with%20a%20novel%20EGR2%20mutation&author=T%20Nakamura&author=A%20Hashiguchi&author=S%20Suzuki&author=K%20Uozumi&author=S%20Tokunaga&volume=13&issue=1&publication_year=2012&pages=77-82&pmid=22271166&doi=10.1007/s10048-012-0313-1&)

12. Neumann Y, Toren A, Rechavi G, et al. Vincristine treatment triggering the expression of asymptomatic Charcot-Marie-Tooth disease. Medical and pediatric oncology. 1996 Apr;26(4):280–3. doi: 10.1002/(SICI)1096-911X(199604)26:4<280::AID-MPO12>3.0.CO;2-J.  [DOI](https://doi.org/10.1002/(SICI)1096-911X(199604)26:4<280::AID-MPO12>3.0.CO;2-J) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8600343/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Medical%20and%20pediatric%20oncology&title=Vincristine%20treatment%20triggering%20the%20expression%20of%20asymptomatic%20Charcot-Marie-Tooth%20disease&author=Y%20Neumann&author=A%20Toren&author=G%20Rechavi&volume=26&issue=4&publication_year=1996&pages=280-3&pmid=8600343&doi=10.1002/(SICI)1096-911X(199604)26:4<280::AID-MPO12>3.0.CO;2-J&)

13. Olek MJ, Bordeaux B, Leshner RT. Charcot-Marie-Tooth disease type I diagnosed in a 5-year-old boy after vincristine neurotoxicity, resulting in maternal diagnosis. The Journal of the American Osteopathic Association. 1999 Mar;99(3):165–7. doi: 10.7556/jaoa.1999.99.3.165.  [DOI](https://doi.org/10.7556/jaoa.1999.99.3.165) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10217912/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=The%20Journal%20of%20the%20American%20Osteopathic%20Association&title=Charcot-Marie-Tooth%20disease%20type%20I%20diagnosed%20in%20a%205-year-old%20boy%20after%20vincristine%20neurotoxicity,%20resulting%20in%20maternal%20diagnosis&author=MJ%20Olek&author=B%20Bordeaux&author=RT%20Leshner&volume=99&issue=3&publication_year=1999&pages=165-7&pmid=10217912&doi=10.7556/jaoa.1999.99.3.165&)

14. Chauvenet AR, Shashi V, Selsky C, Morgan E, Kurtzberg J, Bell B. Vincristine-induced neuropathy as the initial presentation of charcot-marie-tooth disease in acute lymphoblastic leukemia: a Pediatric Oncology Group study. Journal of pediatric hematology/oncology. 2003 Apr;25(4):316–20. doi: 10.1097/00043426-200304000-00010.  [DOI](https://doi.org/10.1097/00043426-200304000-00010) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12679647/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20of%20pediatric%20hematology/oncology&title=Vincristine-induced%20neuropathy%20as%20the%20initial%20presentation%20of%20charcot-marie-tooth%20disease%20in%20acute%20lymphoblastic%20leukemia:%20a%20Pediatric%20Oncology%20Group%20study&author=AR%20Chauvenet&author=V%20Shashi&author=C%20Selsky&author=E%20Morgan&author=J%20Kurtzberg&volume=25&issue=4&publication_year=2003&pages=316-20&pmid=12679647&doi=10.1097/00043426-200304000-00010&)

15. Postma TJ, Aaronson NK, Heimans JJ, et al. The development of an EORTC quality of life questionnaire to assess chemotherapy-induced peripheral neuropathy: the QLQ-CIPN20. Eur J Cancer. 2005 May;41(8):1135–9. doi: 10.1016/j.ejca.2005.02.012.  [DOI](https://doi.org/10.1016/j.ejca.2005.02.012) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15911236/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Cancer&title=The%20development%20of%20an%20EORTC%20quality%20of%20life%20questionnaire%20to%20assess%20chemotherapy-induced%20peripheral%20neuropathy:%20the%20QLQ-CIPN20&author=TJ%20Postma&author=NK%20Aaronson&author=JJ%20Heimans&volume=41&issue=8&publication_year=2005&pages=1135-9&pmid=15911236&doi=10.1016/j.ejca.2005.02.012&)

16. Lavoie Smith EM, Barton DL, Qin R, Steen PD, Aaronson NK, Loprinzi CL. Assessing patient-reported peripheral neuropathy: the reliability and validity of the European Organization for Research and Treatment of Cancer QLQ-CIPN20 Questionnaire. Quality of life research: an international journal of quality of life aspects of treatment, care and rehabilitation. 2013 Mar 30; doi: 10.1007/s11136-013-0379-8.  [DOI](https://doi.org/10.1007/s11136-013-0379-8) | [PMC free article](/articles/PMC4383166/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23543373/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Quality%20of%20life%20research:%20an%20international%20journal%20of%20quality%20of%20life%20aspects%20of%20treatment,%20care%20and%20rehabilitation&title=Assessing%20patient-reported%20peripheral%20neuropathy:%20the%20reliability%20and%20validity%20of%20the%20European%20Organization%20for%20Research%20and%20Treatment%20of%20Cancer%20QLQ-CIPN20%20Questionnaire&author=EM%20Lavoie%20Smith&author=DL%20Barton&author=R%20Qin&author=PD%20Steen&author=NK%20Aaronson&publication_year=2013&pmid=23543373&doi=10.1007/s11136-013-0379-8&)

17. Cavaletti G, Cornblath DR, Merkies IS, et al. The chemotherapy-induced peripheral neuropathy outcome measures standardization study: from consensus to the first validity and reliability findings. Annals of oncology: official journal of the European Society for Medical Oncology/ESMO. 2013 Feb;24(2):454–62. doi: 10.1093/annonc/mds329.  [DOI](https://doi.org/10.1093/annonc/mds329) | [PMC free article](/articles/PMC3551481/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22910842/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Annals%20of%20oncology:%20official%20journal%20of%20the%20European%20Society%20for%20Medical%20Oncology/ESMO&title=The%20chemotherapy-induced%20peripheral%20neuropathy%20outcome%20measures%20standardization%20study:%20from%20consensus%20to%20the%20first%20validity%20and%20reliability%20findings&author=G%20Cavaletti&author=DR%20Cornblath&author=IS%20Merkies&volume=24&issue=2&publication_year=2013&pages=454-62&pmid=22910842&doi=10.1093/annonc/mds329&)

18. Loprinzi CL, Reeves BN, Dakhil SR, et al. Natural history of paclitaxel-associated acute pain syndrome: prospective cohort study NCCTG N08C1. Journal of clinical oncology: official journal of the American Society of Clinical Oncology. 2011 Apr 10;29(11):1472–8. doi: 10.1200/JCO.2010.33.0308.  [DOI](https://doi.org/10.1200/JCO.2010.33.0308) | [PMC free article](/articles/PMC3082985/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21383290/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20of%20clinical%20oncology:%20official%20journal%20of%20the%20American%20Society%20of%20Clinical%20Oncology&title=Natural%20history%20of%20paclitaxel-associated%20acute%20pain%20syndrome:%20prospective%20cohort%20study%20NCCTG%20N08C1&author=CL%20Loprinzi&author=BN%20Reeves&author=SR%20Dakhil&volume=29&issue=11&publication_year=2011&pages=1472-8&pmid=21383290&doi=10.1200/JCO.2010.33.0308&)

19. Reeves BN, Dakhil SR, Sloan JA, et al. Further data supporting that paclitaxel-associated acute pain syndrome is associated with development of peripheral neuropathy: North Central Cancer Treatment Group trial N08C1. Cancer. 2012 Oct 15;118(20):5171–8. doi: 10.1002/cncr.27489.  [DOI](https://doi.org/10.1002/cncr.27489) | [PMC free article](/articles/PMC3376686/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22415454/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer&title=Further%20data%20supporting%20that%20paclitaxel-associated%20acute%20pain%20syndrome%20is%20associated%20with%20development%20of%20peripheral%20neuropathy:%20North%20Central%20Cancer%20Treatment%20Group%20trial%20N08C1&author=BN%20Reeves&author=SR%20Dakhil&author=JA%20Sloan&volume=118&issue=20&publication_year=2012&pages=5171-8&pmid=22415454&doi=10.1002/cncr.27489&)

20. Rasch G. Probabilistic models for some intelligence and attainment tests. Chicago: University of Chicago Press; 1980. Expanded ed.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Probabilistic%20models%20for%20some%20intelligence%20and%20attainment%20tests&author=G%20Rasch&publication_year=1980&)

21. McKenna A, Hanna M, Banks E, et al. The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data. Genome research. 2010 Sep;20(9):1297–303. doi: 10.1101/gr.107524.110.  [DOI](https://doi.org/10.1101/gr.107524.110) | [PMC free article](/articles/PMC2928508/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20644199/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20research&title=The%20Genome%20Analysis%20Toolkit:%20a%20MapReduce%20framework%20for%20analyzing%20next-generation%20DNA%20sequencing%20data&author=A%20McKenna&author=M%20Hanna&author=E%20Banks&volume=20&issue=9&publication_year=2010&pages=1297-303&pmid=20644199&doi=10.1101/gr.107524.110&)

22. Cingolani P, Platts A, Wang le L, et al. A program for annotating and predicting the effects of single nucleotide polymorphisms, SnpEff: SNPs in the genome of Drosophila melanogaster strain w1118; iso-2; iso-3. Fly. 2012 Apr-Jun;6(2):80–92. doi: 10.4161/fly.19695.  [DOI](https://doi.org/10.4161/fly.19695) | [PMC free article](/articles/PMC3679285/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22728672/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Fly&title=A%20program%20for%20annotating%20and%20predicting%20the%20effects%20of%20single%20nucleotide%20polymorphisms,%20SnpEff:%20SNPs%20in%20the%20genome%20of%20Drosophila%20melanogaster%20strain%20w1118;%20iso-2;%20iso-3&author=P%20Cingolani&author=A%20Platts&author=L%20Wang%20le&volume=6&issue=2&publication_year=2012&pages=80-92&pmid=22728672&doi=10.4161/fly.19695&)

23. Wang K, Li M, Hakonarson H. ANNOVAR: functional annotation of genetic variants from high-throughput sequencing data. Nucleic acids research. 2010 Sep;38(16):e164. doi: 10.1093/nar/gkq603.  [DOI](https://doi.org/10.1093/nar/gkq603) | [PMC free article](/articles/PMC2938201/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20601685/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20acids%20research&title=ANNOVAR:%20functional%20annotation%20of%20genetic%20variants%20from%20high-throughput%20sequencing%20data&author=K%20Wang&author=M%20Li&author=H%20Hakonarson&volume=38&issue=16&publication_year=2010&pages=e164&pmid=20601685&doi=10.1093/nar/gkq603&)

24. Zuchner S, Mersiyanova IV, Muglia M, et al. Mutations in the mitochondrial GTPase mitofusin 2 cause Charcot-Marie-Tooth neuropathy type 2A. Nature genetics. 2004 May;36(5):449–51. doi: 10.1038/ng1341.  [DOI](https://doi.org/10.1038/ng1341) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15064763/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature%20genetics&title=Mutations%20in%20the%20mitochondrial%20GTPase%20mitofusin%202%20cause%20Charcot-Marie-Tooth%20neuropathy%20type%202A&author=S%20Zuchner&author=IV%20Mersiyanova&author=M%20Muglia&volume=36&issue=5&publication_year=2004&pages=449-51&pmid=15064763&doi=10.1038/ng1341&)

25. Bissar-Tadmouri N, Nelis E, Zuchner S, et al. Absence of KIF1B mutation in a large Turkish CMT2A family suggests involvement of a second gene. Neurology. 2004 May 11;62(9):1522–5. doi: 10.1212/01.wnl.0000123253.57555.3a.  [DOI](https://doi.org/10.1212/01.wnl.0000123253.57555.3a) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15136675/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neurology&title=Absence%20of%20KIF1B%20mutation%20in%20a%20large%20Turkish%20CMT2A%20family%20suggests%20involvement%20of%20a%20second%20gene&author=N%20Bissar-Tadmouri&author=E%20Nelis&author=S%20Zuchner&volume=62&issue=9&publication_year=2004&pages=1522-5&pmid=15136675&doi=10.1212/01.wnl.0000123253.57555.3a&)

26. Neale BM, Rivas MA, Voight BF, et al. Testing for an unusual distribution of rare variants. PLoS genetics. 2011 Mar;7(3):e1001322. doi: 10.1371/journal.pgen.1001322.  [DOI](https://doi.org/10.1371/journal.pgen.1001322) | [PMC free article](/articles/PMC3048375/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21408211/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20genetics&title=Testing%20for%20an%20unusual%20distribution%20of%20rare%20variants&author=BM%20Neale&author=MA%20Rivas&author=BF%20Voight&volume=7&issue=3&publication_year=2011&pages=e1001322&pmid=21408211&doi=10.1371/journal.pgen.1001322&)

27. Adzhubei IA, Schmidt S, Peshkin L, et al. A method and server for predicting damaging missense mutations. Nature methods. 2010 Apr;7(4):248–9. doi: 10.1038/nmeth0410-248.  [DOI](https://doi.org/10.1038/nmeth0410-248) | [PMC free article](/articles/PMC2855889/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20354512/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature%20methods&title=A%20method%20and%20server%20for%20predicting%20damaging%20missense%20mutations&author=IA%20Adzhubei&author=S%20Schmidt&author=L%20Peshkin&volume=7&issue=4&publication_year=2010&pages=248-9&pmid=20354512&doi=10.1038/nmeth0410-248&)

28. Kumar P, Henikoff S, Ng PC. Predicting the effects of coding non-synonymous variants on protein function using the SIFT algorithm. Nature protocols. 2009;4(7):1073–81. doi: 10.1038/nprot.2009.86.  [DOI](https://doi.org/10.1038/nprot.2009.86) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19561590/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature%20protocols&title=Predicting%20the%20effects%20of%20coding%20non-synonymous%20variants%20on%20protein%20function%20using%20the%20SIFT%20algorithm&author=P%20Kumar&author=S%20Henikoff&author=PC%20Ng&volume=4&issue=7&publication_year=2009&pages=1073-81&pmid=19561590&doi=10.1038/nprot.2009.86&)

29. Guilbot A, Williams A, Ravise N, et al. A mutation in periaxin is responsible for CMT4F, an autosomal recessive form of Charcot-Marie-Tooth disease. Human molecular genetics. 2001 Feb 15;10(4):415–21. doi: 10.1093/hmg/10.4.415.  [DOI](https://doi.org/10.1093/hmg/10.4.415) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11157804/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Human%20molecular%20genetics&title=A%20mutation%20in%20periaxin%20is%20responsible%20for%20CMT4F,%20an%20autosomal%20recessive%20form%20of%20Charcot-Marie-Tooth%20disease&author=A%20Guilbot&author=A%20Williams&author=N%20Ravise&volume=10&issue=4&publication_year=2001&pages=415-21&pmid=11157804&doi=10.1093/hmg/10.4.415&)

30. Takashima H, Boerkoel CF, De Jonghe P, et al. Periaxin mutations cause a broad spectrum of demyelinating neuropathies. Annals of neurology. 2002 Jun;51(6):709–15. doi: 10.1002/ana.10213.  [DOI](https://doi.org/10.1002/ana.10213) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12112076/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Annals%20of%20neurology&title=Periaxin%20mutations%20cause%20a%20broad%20spectrum%20of%20demyelinating%20neuropathies&author=H%20Takashima&author=CF%20Boerkoel&author=P%20De%20Jonghe&volume=51&issue=6&publication_year=2002&pages=709-15&pmid=12112076&doi=10.1002/ana.10213&)

31. Kabzinska D, Drac H, Sherman DL, et al. Charcot-Marie-Tooth type 4F disease caused by S399fsx410 mutation in the PRX gene. Neurology. 2006 Mar 14;66(5):745–7. doi: 10.1212/01.wnl.0000201269.46071.35.  [DOI](https://doi.org/10.1212/01.wnl.0000201269.46071.35) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16534116/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neurology&title=Charcot-Marie-Tooth%20type%204F%20disease%20caused%20by%20S399fsx410%20mutation%20in%20the%20PRX%20gene&author=D%20Kabzinska&author=H%20Drac&author=DL%20Sherman&volume=66&issue=5&publication_year=2006&pages=745-7&pmid=16534116&doi=10.1212/01.wnl.0000201269.46071.35&)

32. Barankova L, Siskova D, Huhne K, et al. A 71-nucleotide deletion in the periaxin gene in a Romani patient with early-onset slowly progressive demyelinating CMT. European journal of neurology: the official journal of the European Federation of Neurological Societies. 2008 Jun;15(6):548–51. doi: 10.1111/j.1468-1331.2008.02104.x.  [DOI](https://doi.org/10.1111/j.1468-1331.2008.02104.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18410371/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=European%20journal%20of%20neurology:%20the%20official%20journal%20of%20the%20European%20Federation%20of%20Neurological%20Societies&title=A%2071-nucleotide%20deletion%20in%20the%20periaxin%20gene%20in%20a%20Romani%20patient%20with%20early-onset%20slowly%20progressive%20demyelinating%20CMT&author=L%20Barankova&author=D%20Siskova&author=K%20Huhne&volume=15&issue=6&publication_year=2008&pages=548-51&pmid=18410371&doi=10.1111/j.1468-1331.2008.02104.x&)

33. Nouioua S, Hamadouche T, Funalot B, et al. Novel mutations in the PRX and the MTMR2 genes are responsible for unusual Charcot-Marie-Tooth disease phenotypes. Neuromuscular disorders: NMD. 2011 Aug;21(8):543–50. doi: 10.1016/j.nmd.2011.04.013.  [DOI](https://doi.org/10.1016/j.nmd.2011.04.013) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21741241/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neuromuscular%20disorders:%20NMD&title=Novel%20mutations%20in%20the%20PRX%20and%20the%20MTMR2%20genes%20are%20responsible%20for%20unusual%20Charcot-Marie-Tooth%20disease%20phenotypes&author=S%20Nouioua&author=T%20Hamadouche&author=B%20Funalot&volume=21&issue=8&publication_year=2011&pages=543-50&pmid=21741241&doi=10.1016/j.nmd.2011.04.013&)

34. Goldstein DB, Allen A, Keebler J, et al. Sequencing studies in human genetics: design and interpretation. Nature reviews Genetics. 2013 Jul;14(7):460–70. doi: 10.1038/nrg3455.  [DOI](https://doi.org/10.1038/nrg3455) | [PMC free article](/articles/PMC4117319/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23752795/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature%20reviews%20Genetics&title=Sequencing%20studies%20in%20human%20genetics:%20design%20and%20interpretation&author=DB%20Goldstein&author=A%20Allen&author=J%20Keebler&volume=14&issue=7&publication_year=2013&pages=460-70&pmid=23752795&doi=10.1038/nrg3455&)

35. Altekruse SF, Korsary C, Krapcho M, et al. SEER Cancer Statistics Review, 1975–2007. National Cancer Institute; Bethesda, MD: 2010. 2/18/14; http://seer.cancer.gov/archive/csr/1975_2007/  [http://seer.cancer.gov/archive/csr/1975_2007/](http://seer.cancer.gov/archive/csr/1975_2007/) | [Google Scholar](https://scholar.google.com/scholar_lookup?title=SEER%20Cancer%20Statistics%20Review,%201975%E2%80%932007&author=SF%20Altekruse&author=C%20Korsary&author=M%20Krapcho&publication_year=2010&)

36. Matsuda T, Ajiki W, Marugame T, Ioka A, Tsukuma H, Sobue T. Population-based survival of cancer patients diagnosed between 1993 and 1999 in Japan: a chronological and international comparative study. Jpn J Clin Oncol. 2011 Jan;41(1):40–51. doi: 10.1093/jjco/hyq167.  [DOI](https://doi.org/10.1093/jjco/hyq167) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20819833/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Jpn%20J%20Clin%20Oncol&title=Population-based%20survival%20of%20cancer%20patients%20diagnosed%20between%201993%20and%201999%20in%20Japan:%20a%20chronological%20and%20international%20comparative%20study&author=T%20Matsuda&author=W%20Ajiki&author=T%20Marugame&author=A%20Ioka&author=H%20Tsukuma&volume=41&issue=1&publication_year=2011&pages=40-51&pmid=20819833&doi=10.1093/jjco/hyq167&)

37. Alberti P, Rossi E, Cornblath DR, et al. Physician-assessed and patient-reported outcome measures in chemotherapy-induced sensory peripheral neurotoxicity: two sides of the same coin. Annals of oncology: official journal of the European Society for Medical Oncology/ESMO. 2014 Jan;25(1):257–64. doi: 10.1093/annonc/mdt409.  [DOI](https://doi.org/10.1093/annonc/mdt409) | [PMC free article](/articles/PMC3868322/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24256846/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Annals%20of%20oncology:%20official%20journal%20of%20the%20European%20Society%20for%20Medical%20Oncology/ESMO&title=Physician-assessed%20and%20patient-reported%20outcome%20measures%20in%20chemotherapy-induced%20sensory%20peripheral%20neurotoxicity:%20two%20sides%20of%20the%20same%20coin&author=P%20Alberti&author=E%20Rossi&author=DR%20Cornblath&volume=25&issue=1&publication_year=2014&pages=257-64&pmid=24256846&doi=10.1093/annonc/mdt409&)

38. Dyck PJ, Overland CJ, Low PA, et al. Signs and symptoms versus nerve conduction studies to diagnose diabetic sensorimotor polyneuropathy: Cl vs. NPhys trial. Muscle & nerve. 2010 Aug;42(2):157–64. doi: 10.1002/mus.21661.  [DOI](https://doi.org/10.1002/mus.21661) | [PMC free article](/articles/PMC2956592/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20658599/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Muscle%20&%20nerve&title=Signs%20and%20symptoms%20versus%20nerve%20conduction%20studies%20to%20diagnose%20diabetic%20sensorimotor%20polyneuropathy:%20Cl%20vs.%20NPhys%20trial&author=PJ%20Dyck&author=CJ%20Overland&author=PA%20Low&volume=42&issue=2&publication_year=2010&pages=157-64&pmid=20658599&doi=10.1002/mus.21661&)

39. Gillespie CS, Sherman DL, Fleetwood-Walker SM, et al. Peripheral demyelination and neuropathic pain behavior in periaxin-deficient mice. Neuron. 2000 May;26(2):523–31. doi: 10.1016/s0896-6273(00)81184-8.  [DOI](https://doi.org/10.1016/s0896-6273(00)81184-8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10839370/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neuron&title=Peripheral%20demyelination%20and%20neuropathic%20pain%20behavior%20in%20periaxin-deficient%20mice&author=CS%20Gillespie&author=DL%20Sherman&author=SM%20Fleetwood-Walker&volume=26&issue=2&publication_year=2000&pages=523-31&pmid=10839370&doi=10.1016/s0896-6273(00)81184-8&)

40. Chaya T, Shibata S, Tokuhara Y, et al. Identification of a negative regulatory region for the exchange activity and characterization of T332I mutant of Rho guanine nucleotide exchange factor 10 (ARHGEF10) The Journal of biological chemistry. 2011 Aug 26;286(34):29511–20. doi: 10.1074/jbc.M111.236810.  [DOI](https://doi.org/10.1074/jbc.M111.236810) | [PMC free article](/articles/PMC3190991/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21719701/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=The%20Journal%20of%20biological%20chemistry&title=Identification%20of%20a%20negative%20regulatory%20region%20for%20the%20exchange%20activity%20and%20characterization%20of%20T332I%20mutant%20of%20Rho%20guanine%20nucleotide%20exchange%20factor%2010%20(ARHGEF10)&author=T%20Chaya&author=S%20Shibata&author=Y%20Tokuhara&volume=286&issue=34&publication_year=2011&pages=29511-20&pmid=21719701&doi=10.1074/jbc.M111.236810&)

41. Verhoeven K, De Jonghe P, Van de Putte T, et al. Slowed conduction and thin myelination of peripheral nerves associated with mutant rho Guanine-nucleotide exchange factor 10. American journal of human genetics. 2003 Oct;73(4):926–32. doi: 10.1086/378159.  [DOI](https://doi.org/10.1086/378159) | [PMC free article](/articles/PMC1180612/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14508709/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=American%20journal%20of%20human%20genetics&title=Slowed%20conduction%20and%20thin%20myelination%20of%20peripheral%20nerves%20associated%20with%20mutant%20rho%20Guanine-nucleotide%20exchange%20factor%2010&author=K%20Verhoeven&author=P%20De%20Jonghe&author=T%20Van%20de%20Putte&volume=73&issue=4&publication_year=2003&pages=926-32&pmid=14508709&doi=10.1086/378159&)

42. Tokunaga S, Hashiguchi A, Yoshimura A, et al. Late-onset Charcot-Marie-Tooth disease 4F caused by periaxin gene mutation. Neurogenetics. 2012 Nov;13(4):359–65. doi: 10.1007/s10048-012-0338-5.  [DOI](https://doi.org/10.1007/s10048-012-0338-5) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22847150/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neurogenetics&title=Late-onset%20Charcot-Marie-Tooth%20disease%204F%20caused%20by%20periaxin%20gene%20mutation&author=S%20Tokunaga&author=A%20Hashiguchi&author=A%20Yoshimura&volume=13&issue=4&publication_year=2012&pages=359-65&pmid=22847150&doi=10.1007/s10048-012-0338-5&)
