# Thiopurine pharmacogenomics: association of SNPs with clinical response and functional validation of candidate genes

## Metadata
**Authors:** Alice Matimba, Fang Li, Alina Livshits, Cher S Cartwright, Stephen Scully, Brooke L Fridley, Gregory Jenkins, Anthony Batzler, Liewei Wang, Richard Weinshilboum, Lynne Lennard
**Journal:** Pharmacogenomics
**Date:** 2014 Mar
**DOI:** [10.2217/pgs.13.226](https://doi.org/10.2217/pgs.13.226)
**PMID:** 24624911
**PMCID:** PMC4027966
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4027966/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC4027966/pdf/nihms577420.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC4027966/pdf/nihms577420.pdf)

## Abstract

**Aim:** 
We investigated candidate genes associated with thiopurine metabolism and clinical response in childhood acute lymphoblastic leukemia.

**Materials & methods:** 
We performed genome-wide SNP association studies of 6-thioguanine and 6-mercaptopurine cytotoxicity using lymphoblastoid cell lines. We then genotyped the top SNPs associated with lymphoblastoid cell line cytotoxicity, together with tagSNPs for genes in the ‘thiopurine pathway’ (686 total SNPs), in DNA from 589 Caucasian UK ALL97 patients. Functional validation studies were performed by siRNA knockdown in cancer cell lines.

**Results:** 
SNPs in the thiopurine pathway genes ABCC4, ABCC5, IMPDH1, ITPA, SLC28A3 and XDH, and SNPs located within or near ATP6AP2, FRMD4B, GNG2, KCNMA1 and NME1, were associated with clinical response and measures of thiopurine metabolism. Functional validation showed shifts in cytotoxicity for these genes.

**Conclusion:** 
The clinical response to thiopurines may be regulated by variation in known thiopurine pathway genes and additional novel genes outside of the thiopurine pathway.

Keywords: 6-mercaptopurine, 6-thioguanine, childhood acute lymphoblastic leukemia, single nucleotide polymorphisms, thiopurine pharmacogenomics

### Aim

We investigated candidate genes associated with thiopurine metabolism and clinical response in childhood acute lymphoblastic leukemia.

### Materials & methods

We performed genome-wide SNP association studies of 6-thioguanine and 6-mercaptopurine cytotoxicity using lymphoblastoid cell lines. We then genotyped the top SNPs associated with lymphoblastoid cell line cytotoxicity, together with tagSNPs for genes in the ‘thiopurine pathway’ (686 total SNPs), in DNA from 589 Caucasian UK ALL97 patients. Functional validation studies were performed by siRNA knockdown in cancer cell lines.

### Results

SNPs in the thiopurine pathway genes *ABCC4*, *ABCC5*, *IMPDH1*, *ITPA*, *SLC28A3* and *XDH*, and SNPs located within or near *ATP6AP2*, *FRMD4B*, *GNG2*, *KCNMA1* and *NME1*, were associated with clinical response and measures of thiopurine metabolism. Functional validation showed shifts in cytotoxicity for these genes.

### Conclusion

The clinical response to thiopurines may be regulated by variation in known thiopurine pathway genes and additional novel genes outside of the thiopurine pathway.

## Materials & methods

### Cell lines

LCLs from 58 African–American, 58 European–American, and 60 Han Chinese–American unrelated healthy subjects (sample sets HD100AA, HD100CAU and HD100CHI) were purchased from the Coriell Cell Repository (NJ, USA). These cell lines had been generated from blood samples collected and anonymized by the National Institute of General Medical Sciences (NIGMS). All subjects had provided written consent for the use of their samples for research purposes. This study was reviewed and approved by the Mayo Clinic Institutional Review Board. HeLa (human cervical carcinoma), U87MG and U251 (human glioma) as well as OVCAR10 (ovarian cancer) cell lines were purchased from the American Type Culture Collection (VA, USA).

### Drugs & cell proliferation assays

6-MP and 6-TG were purchased from Sigma Aldrich (MO, USA). Cytotoxicity assays were performed using the CellTiter 96^®^ Aqueous Non-Radioactive Cell Proliferation Assay (Promega Corporation, WI, USA) after 72 h drug exposure over a range of drug concentrations (see [Supplementary table 1](#SD1) at [www.futuremedicine.com/doi/suppl/10.2217/pgs.13.266](http://www.futuremedicine.com/doi/suppl/10.2217/pgs.13.266) for concentrations) for lymphoblastoid, HeLa, OVCAR10, U87MG and U251 cell lines.

### LCL genomic data

DNA from the Human Variation Panel LCLs had been genotyped in the Mayo Clinic Genotype Shared Resource (GSR) using Illumina HumanHap 550K and 510S BeadChips. The Coriell Institute also genotyped and made available Affymetrix SNP Array 6.0 Chip data for these cell lines. As a result, approximately 1.3 million unique SNPs were available for each cell line. Total RNA was also extracted from each of the LCLs using the Qiagen RNeasy Mini kit (Qiagen, Inc., CA, USA) and RNA quality was tested using an Agilent 2100 Bioanalyzer, followed by hybridization to Affymetrix U133 Plus 2.0 GeneChips. Basal mRNA expression data were normalized using guanine cytosine robust multiarray analysis [[36](#R36)]. These data are available in the NCBI database under accession number [GSE24277](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE24277). In an analysis of 1.3 million SNPs one would predict 130 SNPs to be associated with 6-TG or 6-MP IC_50_ at a 1 × 10^−4^ level. No adjustment was made for multiple testing, although a conservative Bonferroni cutoff of p < 7.28 × 10^−5^ would account for the 686 SNPs analyzed. For these reasons SNPs with an association <1 × 10^−4^ were considered for further study.

### UK ALL97 clinical trial

The UK Medical Research Council (MRC) ALL97 trial (UK ALL97, registration number ISRCTN26727615) compared the efficacy and toxicity of dexamethasone and 6-TG (experimental arm) with prednisone and 6-MP (standard approach arm). This trial recruited ALL patients aged 1–18 years who were newly diagnosed with their first episode of ALL. The patients were diagnosed in the UK and Ireland between January 1997 and June 2002. Patient ethnicity was recorded at trial recruitment by asking the parents. Treatment centers obtained local ethics committee approval and informed consent from patients and/or parents. The trial included an add-on pharmacogenetic and drug metabolism study designed to investigate inter- and intra-patient variability in response to thiopurines. Background treatment regimens underwent several modifications during the trial, but randomizations and biological studies were retained. Patients were randomly allocated to daily oral 6-TG (standard dose 40 mg/m^2^) or 6-MP (standard dose 75 mg/m^2^) during interim and continuing maintenance. Thiopurine doses were titrated to achieve target neutrophil and platelet counts. Details of the titration protocols, treatment regimens and modifications have been reported previously [[3](#R3),[37](#R37)].

Blood samples were requested at disease diagnosis and during thiopurine maintenance chemotherapy. During maintenance chemotherapy, blood samples were obtained after at least 7 days at the standard protocol thiopurine dose or at the maximum tolerated dose, but not within 2 months of a red cell transfusion. Blood samples were obtained at the earliest convenient time point after the criteria described above were met. If maintenance therapy had been interrupted or the dosage reduced, the sample was obtained after the recovery of cell counts during the next cycle of thiopurine therapy. The use of these DNA samples and associated clinical data to perform the present study was reviewed and approved by the Sheffield Local Research Ethics Committee (reference 08/H1308/279).

### TPMT activity & thiopurine metabolite assays

The concentrations of the 6-TGN and methyl-mercaptopurine nucleotide (MeMPN) thiopurine metabolites and TPMT activity were measured by HPLC, as described previously [[38](#R38),[39](#R39)]. The TPMT activity assay was modified to measure the methylmercaptopurine product directly [[2](#R2)]. Metabolite assay results are reported as pmol/8 × 10^8^ RBCs and TPMT activity as units/ml packed RBCs.

### Genotyping

DNA extracted from blood samples (Nucleon BACC1, Manchester, UK) was genotyped for *TPMT*3A*, *TPMT*3B* and *TPMT*3C* by amplification of exons 7 and 10 using intron-based primers [[40](#R40)]. A 10 μl aliquot of the PCR product was digested for 4 h with *Mwo1* at 60°C to detect *TPMT*3B* (wild-type fragments 226 bp and 100 bp, variant allele, 326 bp, not digested) and *Acc1* at 37°C to detect *TPMT*3C* (wild-type, 437 bp, not digested, variant allele fragments 268 bp and 169 bp). *TPMT*3A* is a double mutant containing both the exon 7 and exon 10 variant alleles. The restriction digestion products were analyzed on 4% poly-acrylamide gels. *TPMT*2* was determined by sequencing exon 5 of the *TPMT* gene using an ABI 3730 capillary sequencer (Applied Biosystems, Warrington, UK) with dye-primer chemistry. SNPs selected for genotyping using DNA from the ALL97 patients were quality-tested by Illumina, arranged in a customized genotyping panel and genotyped using the Illumina GoldenGate system (CA, USA).

### Transient transfection & RNAi

siRNAs used to perform the gene knockdown studies were purchased from Dharmacon RNAi Technologies (CO, USA). The cell lines studied were transfected with Lipofectamine^™^ RNAiMAX (Invitrogen, CA, USA). Briefly, approximately 4–5 × 10^3^ cells were plated in a 96-well format and were transfected with 5 nM of the appropriate siRNA. After 24 h, a series of drug concentrations was added, followed by incubation for 72 h. Cytotoxicity was measured as described previously, and total RNA was isolated from the cells using the Qiagen RNeasy kit. Primers were purchased from Qiagen, and quantitative reverse transcription (qRT)-PCR was performed using the one-step, Brilliant SYBR^®^ Green qRT-PCR master mix kit (Stratagene, CA, USA) with the StepOnePlus^™^ Real-Time PCR System (Applied Biosystems, CA, USA). All experiments were performed in triplicate with β-actin as an internal control. The efficiency of siRNA knockdown was estimated by quantitative PCR, comparing the mRNA expression of the siRNA knockdown genes with nontargeting siRNA as a control. The nontargeting siRNA was designed to target no known genes in the human cells lines and so determine baseline gene-expression levels. The efficiency of gene-expression knockdown ranged from 70–90%.

### Statistical methods & association analysis

The statistical methods for LCL analysis, including adjusting for ethnicity, have been previously described [[30](#R30)]. For the functional validation studies best fit curves, before and after gene knockdown, were fitted using GraphPad Prism Software (CA, USA). The data was fitted to a four parameter nonlinear regression, curve fit, model selecting a variable slope and sigmoidal dose–response curve. Statistical significance was calculated using the Student’s t-test for paired data in the GraphPad software. Survival was calculated from the start of treatment to death or last follow-up, while relapse-free survival was calculated from start of treatment to relapse/death, or last follow-up. Association of survival and relapse phenotypes and SNPs was assessed using Cox proportional hazard models. All other phenotypes were transformed because of skewness, using a van der Waerden rank transformation, and association with SNPs was assessed using linear regression. Increase in model R^2^ was derived from the difference in model R^2^ by adding genotype beyond covariates, and direction of the genotype effect was the sign of the effect of the genotype in the regression model. All analyses were adjusted for TPMT ‘metabolizer status’, and the treatment received. SNP genotypes were modeled with an additive genetic model (count of rare alleles).

## Results

For the thiopurine studies, within the UK ALL97 clinical trial, 87% of the patients enrolled were Caucasian and 13% belonged to other ethnic groups (5.4% Asian [Indian or Pakistani decent], 3.3% mixed race, 1.5% black, 0.5% Oriental and 2.3% unknown or non-Caucasian). Analysis was restricted to the Caucasian cohort; blood and DNA samples were obtained from 589 Caucasian children. Demographic data for these patients are summarized in [table 1](#T1). The SNPs genotyped using DNA samples from these patients were selected from two sources: SNPs identified during 6-TG and 6-MP cytotoxicity GWAS conducted with the Human Variation Panel of LCLs; and tagSNPs for genes encoding proteins in the thiopurine pathway as defined by PharmGKB [[101](#R48)].

### Table 1.

| Variable | Name or value | n (%) or value (range) |
| --- | --- | --- |
| Gender | Male | 330 (56.0) |
| Female | 259 (44.0) |  |
|   |  |  |
| Age, median (range) | – | 4 (1–16) |
|   |  |  |
| WBC group | <10 | 250 (42.4) |
| 10–19.9 | 88 (14.9) |  |
| 20–50.9 | 101 (17.1) |  |
| 50–99.9 | 63 (10.7) |  |
| 100+ | 87 (14.8) |  |
|   |  |  |
| Outcome, relapse | Male | 74 |
| Female | 43 |  |
|   |  |  |
| Outcome, death | Male | 44 |
| Female | 39 |  |
|   |  |  |
| 6-MP† | Sample number | 258 |
| Dose, mg/m2 | 75 (14–125) |  |
| Derived TGNs | 377 (66–1928) |  |
| Derived MeMPNs | 7698 (66–1928) |  |
| TPMT activity | 14.1 (2.8–23) |  |
|   |  |  |
| 6-TG† | Sample number | 217 |
| Dose, mg/m2 | 40 (10–78) |  |
| Derived TGNs | 2030 (96–6874) |  |
| TPMT activity | 13.7 (0–21) |  |
|   |  |  |
| Genotype available | Diagnosis only | 219 (37.2) |
| Chemotherapy only | 266 (45.2) |  |
| Diagnosis and chemotherapy | 104 (17.7) |  |
|   |  |  |
| TPMT genotypes‡ | *1/*1 | 530 (90) |
| *1/*3A | 51 (8.7) |  |
| *1/*3C | 5 (0.8) |  |
| *1/*2 | 1 (0.2) |  |
| *3A/*3A | 1 (0.2) |  |
| *3C/*3C | 1 (0.2) |  |

Table 1 Caption: Demographic data for the 589 UK ALL97 Caucasian children studied.

Subsequent paragraphs describe first, the Human Variation Panel LCL Model System GWAS that was used to identify the majority of SNPs used to genotype the clinical samples; second the results of the clinical sample genotyping; and, third, the functional validation of selected genes related to SNPs that displayed associations with the clinical phenotypes studied.

### Human Variation Panel 6-TG & 6-MP cytotoxicity GWAS

The Manhattan plots for the LCL thiopurine GWAS, depicted graphically in [Figure 1](#F1), showed that 153 SNPs with p < 1 × 10^−4^ were associated with 6-TG IC_50_ values and 130 with 6-MP IC_50_ values, but only six of those SNPs were common to both drugs ([Figure 1C](#F1)). All SNPs with p < 1 × 10^−4^ for both LCL 6-TG and 6-MP GWAS are listed in [Supplementary tables 2a & 2b](#SD1), with an asterix indicating SNPs that were common to both 6-TG and 6-MP. The top SNP for 6-TG cytotoxicity was rs6097295 (p = 2.34 × 10^−7^, r = 0.40). This SNP mapped to chromosome 20 ([Figure 1a](#F1)) in the first intron of the *TSHZ2* gene and the variant allele for this SNP was associated with increased resistance. Two other SNPs with low p-values mapped to the same *TSHZ2* intron. The most striking SNP signal observed in the 6-MP GWAS mapped to chromosome 17 ([Figure 1b](#F1)), with the top SNP, rs4794227 (r = −0.38; p = 1.31 × 10^−6^), located in the 3′-UTR of the *UTP18* gene. The variant allele for this SNP was associated with decreased 6-MP IC_50_, that is, enhanced sensitivity.

### Figure 1. Genome-wide association studies for SNP associations with thiopurine IC50 values in lymphoblastoid cell lines.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/98ec/4027966/bdb81dfb3277/nihms577420f1.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4027966_nihms577420f1.jpg)

Manhattan plots for lymphoblastoid cell line GWAS for IC50 values for (A) 6-TG and (B) 6-MP. (C) Overlap of top SNPs for 6-TG and 6-MP with p-values of <10−4. The blue line denotes p < 1 × 10−4; SNPs with an association <1 × 10−4 were considered for further study. Red dots denote thiopurine pathway SNPs. 6-MP: 6-mercaptopurine; 6-TG: 6-thioguanine; GWAS: Genome-wide association study.

SNPs with p-values <1 × 10^−4^ from the LCL GWAS ([Figure 1C](#F1)) as well as a group of tagSNPs for thiopurine pathway genes were then genotyped using DNA samples from patients enrolled in the UK ALL97 trials [[3](#R3)]. A graphical outline of the process of SNP selection is shown in [Figure 2](#F2). Specifically, 403 SNPs were selected from the 6-TG and 6-MP LCL GWAS (p < 1 × 10^−4^; [Supplementary tables 2a, 2b](#SD1), [Figure 1C](#F1) & [Figure 2a](#F2)). That number included 114 from the SNP peak on a 380 kb region of chromosome 17 observed during the 6-MP GWAS ([Figure 1b](#F1)). SNPs with low p-values within this region other than those genotyped during the GWAS were imputed using CEU HapMap 3 data. The PharmGKB database and HapMap 2 data were used to select an additional 387 thiopurine pathway tagSNPs ([Figure 2b](#F2)). Therefore, 790 SNPs were submitted to Illumina, and a final total of 686 SNPs passed quality control, were successfully genotyped using DNA from UK ALL97 patients and were analyzed for their possible association with clinical phenotypes.

### Figure 2. Selection of SNPs for genotyping samples from acute lymphoblastic leukemia patients.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/98ec/4027966/7d2980006116/nihms577420f2.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4027966_nihms577420f2.jpg)

(A) Top SNPs associated with 6-TG and 6-MP IC50 values in LCLs. Additional SNPs in a region of chromosome 17 were selected from the CEU Hapmap2 data over approximately 380 kb extending across a SNP ‘signal’ on chromosome 17 that was observed during the LCL GWAS for 6-MP IC50 value. (B) Thiopurine pathway SNPs. These SNPs were in or near genes (<200 kb) encoding proteins known to be involved in the thiopurine pathway as defined by PharmGKB. 6-MP: 6-mercaptopurine; 6-TG: 6-thioguanine; GWAS: Genome-wide association study; LCL: Lymphoblastoid cell lines.

### Association analysis for UK ALL97 patients

Possible associations between the SNPs genotyped in the 589 patients studied and patient-related phenotypes were investigated. These phenotypes included patient RBC TPMT activities at both disease diagnosis and relapse, patient RBC drug metabolite concentrations (6-TGNs and MeMPNs), disease relapse and patient survival. It should be noted that some of the DNA samples had been obtained at the time of initial diagnosis while others were obtained during remission maintenance chemotherapy. For the survival, relapse and TPMT activity phenotypes, genotypes from both ‘diagnosis’ and ‘chemotherapy’ samples were used to perform the association analyses, while – for obvious reasons – only genotypes from the chemotherapy samples could be used to perform association analyses for 6-TGN and MeMPN concentrations or the MeMPN:6-TGN ratio because those phenotypes were only available after the initiation of chemotherapy. Comparison of genotypes showed a 99% concordance between genotypes for diagnosis and chemotherapy samples from the patients for which both types of DNA samples were available (n = 104). Although there is good agreement between the genotypes, it is not complete. For this reason, because the at diagnosis genotype sample was derived from white blood cells containing predominantly leukemic cells, the diagnosis and chemotherapy samples were analyzed separately in the association analysis of survival and relapse. [tables 2](#T2) & [3](#T3) summarize the association of SNPs and clinical phenotypes.

### Table 2.

| Phenotype | SNP ID | Chr | Gene | Location | MAF | HR | p-value |
| --- | --- | --- | --- | --- | --- | --- | --- |
| Survival (‘chemotherapy’ genotypes) | rs9838667 | 3 | ABCC5 | Intron | 0.22 | 0.40 | 9.19 × 10−3 |
| rs4148586 | 3 | ABCC5 | Intron | 0.34 | 2.23 | 1.66 × 10−4 |  |
| rs3817404 | 3 | ABCC5 | Intron | 0.34 | 2.23 | 1.67 × 10−4 |  |
| rs7636910 | 3 | ABCC5 | Intron | 0.40 | 1.82 | 4.51 × 10−3 |  |
| rs2293001 | 3 | ABCC5 | Intron | 0.40 | 1.82 | 4.51 × 10−3 |  |
| rs4731449 | 7 | IMPDH1 | Intron | 0.13 | 1.96 | 7.89 × 10−3 |  |
| rs1729767 | 13 | ABCC4 | Intron | 0.28 | 0.39 | 2.30 × 10−3 |  |
|   |  |  |  |  |  |  |  |
| Survival (‘diagnosis’ genotypes) | rs206846 | 2 | XDH | Intron | 0.39 | 2.60 | 1.65 × 10−3 |
| rs4584690 | 13 | ABCC4 | Intron | 0.12 | 3.11 | 1.58 × 10−4 |  |
| rs1750190 | 13 | ABCC4 | Intron | 0.46 | 0.39 | 8.83 × 10−3 |  |
| rs1189443 | 13 | ABCC4 | Intron | 0.16 | 2.54 | 2.63 × 10−3 |  |
| rs1729747 | 13 | ABCC4 | Intron | 0.15 | 2.42 | 4.19 × 10−3 |  |
| rs12864844 | 13 | ABCC4 | Intron | 0.12 | 2.91 | 6.51 × 10−4 |  |
| rs7321807 | 13 | ABCC4 | Intron | 0.03 | 4.48 | 7.64 × 10−3 |  |
| rs7325256 | 13 | ABCC4 | Intron | 0.03 | 4.48 | 7.64 × 10−3 |  |
|   |  |  |  |  |  |  |  |
| Relapse (‘chemotherapy’ genotypes) | rs4148586 | 3 | ABCC5 | Intron | 0.34 | 2.17 | 4.45 × 10−4 |
| rs3817404 | 3 | ABCC5 | Intron | 0.34 | 2.18 | 4.16 × 10−4 |  |
| rs7636910 | 3 | ABCC5 | Intron | 0.40 | 1.79 | 8.36 × 10−3 |  |
| rs2293001 | 3 | ABCC5 | Intron | 0.40 | 1.79 | 8.36 × 10−3 |  |
| rs1729767 | 13 | ABCC4 | Intron | 0.28 | 0.34 | 2.13 × 10−3 |  |
|   |  |  |  |  |  |  |  |
| Relapse (‘diagnosis’ genotypes) | rs206846 | 2 | XDH | Intron | 0.39 | 2.55 | 7.01 × 10−3 |
| rs4584690 | 13 | ABCC4 | Intron | 0.12 | 2.61 | 7.71 × 10−3 |  |
| rs7321807 | 13 | ABCC4 | Intron | 0.03 | 6.28 | 1.45 × 10−3 |  |
| rs7325256 | 13 | ABCC4 | Intron | 0.03 | 6.28 | 1.45 × 10−3 |  |
| rs7270101 | 20 | ITPA | Intron | 0.15 | 2.72 | 6.76 × 10−3 |  |

Table 2 Caption: Top thiopurine pathway SNPs associated with clinical phenotypes in UK ALL97 patients: treatment outcomes.

### Table 3.

| Phenotype | SNP ID | Chr | Gene | Location | MAF | (Direction) Increase in R2† | p-value |
| --- | --- | --- | --- | --- | --- | --- | --- |
| TPMT activity | rs494852 | 2 | XDH | Intron | 0.19 | (−) 0.015 | 5.24 × 10−4 |
| rs4731448 | 7 | IMPDH1 | Intron | 0.24 | (−) 0.033 | 3.03 × 10−3 |  |
| rs17428030 | 9 | SLC28A3 | Intron | 0.09 | (−) 0.030 | 3.93 × 10−3 |  |
| rs17268122 | 13 | ABCC4 | Intron | 0.22 | (−) 0.030 | 4.24 × 10−3 |  |
|   |  |  |  |  |  |  |  |
| 6-TG, 6-TGN | rs9556455 | 13 | ABCC4 | Intron | 0.14 | (−) 0.046 | 5.76 × 10−3 |
|   |  |  |  |  |  |  |  |
| 6-MP, MeMPN | rs12201199 | 6 | TPMT | Intron | 0.06 | (−) 0.032 | 6.02 × 10−3 |
| rs4588940 | 9 | SLC28A3 | Intron | 0.47 | (−) 0.071 | 3.38 × 10−5 |  |
| rs4305983 | 9 | SLC28A3 | Intron | 0.28 | (−) 0.089 | 3.07 × 10−6 |  |
| rs7043257 | 9 | SLC28A3 | Intron | 0.20 | (−) 0.042 | 1.59 × 10−3 |  |
| rs7035753 | 9 | SLC28A3 | Intron | 0.38 | (−) 0.031 | 6.99 × 10−3 |  |
| rs17087144 | 9 | SLC28A3 | Intron | 0.33 | (+) 0.049 | 6.00 × 10−4 |  |
|   |  |  |  |  |  |  |  |
| 6-MP, MeMPN/6-TGN | rs1142345 | 6 | TPMT | Exon (ns) | 0.05 | (−) 0.026 | 7.90 × 10−3 |
| rs12201199 | 6 | TPMT | Intron | 0.06 | (−) 0.028 | 6.41 × 10−3 |  |
| rs4588940 | 9 | SLC28A3 | Intron | 0.47 | (−) 0.055 | 1.12 × 10−4 |  |
| rs4305983 | 9 | SLC28A3 | Intron | 0.28 | (−) 0.063 | 3.42 × 10−5 |  |
| rs7043257 | 9 | SLC28A3 | Intron | 0.20 | (−) 0.030 | 4.38 × 10−3 |  |
| rs17087144 | 9 | SLC28A3 | Intron | 0.33 | (+) 0.030 | 4.48 × 10−3 |  |

Table 3 Caption: Top thiopurine pathway SNPs associated with clinical phenotypes in UK ALL97 patients: TPMT activity and thiopurine metabolites.

### TPMT genotype associations

*TPMT* genotype is associated with striking variation in TPMT activity and, as a result, with risk for the occurrence of serious adverse drug reactions in patients treated with thiopurines [[4](#R4)]. Our association analysis ([Supplementary table 3](#SD1)) confirmed that *TPMT* genotype was highly associated with level of TPMT activity during chemotherapy (2.90 × 10^−43^), RBC MeMPN:6-TGN ratio (5.63 × 10^−6^) and RBC 6-TGN concentrations (p = 1.62 × 10^−6^) in these patients. However, *TPMT* genotype (analyzed as wild-type or variant allele) was not significantly associated with either survival or relapse ([Supplementary table 3](#SD1)). The ALL97 protocol adjusted drug dose in TPMT deficient patients to a starting dose of 10% of the standard 75 mg m^−2^ protocol dose. For all patients the thiopurine dose was titrated to target blood cell counts for a degree of controlled myelosuppression. A small cohort of problem patients (drug sensitivity or nonresponse) had thiopurine dosage adjusted based on drug metabolite levels. This lack of association of TPMT with outcome measures has been observed in similar protocols [[41](#R41)]. In order to determine whether additional SNPs, beyond those in *TPMT*, might be associated with clinical phenotypes in ALL patients, association analyses were performed after adjusting for *TPMT* genotype and TPMT enzyme activity as covariates. Since results adjusted for *TPMT* genotype and TPMT enzyme activity during chemotherapy did not differ significantly, in subsequent paragraphs we will only report results adjusted for *TPMT* genotype.

### Associations with treatment outcomes

Included among the strongest associations with treatment outcomes were those seen for the rs7321807 and rs7325256 (linkage disequilibrium r^2^ = 1) SNPs in the thiopurine pathway gene *ABCC4*, with hazard ratio values of 6.28 and 4.48 for relapse and survival, respectively ([tables 2](#T2) & [3](#T3)). A hazard ratio >1 indicates a worse survival or an increased risk of relapse. Other thiopurine pathway SNPs associated with these clinical disease-related outcomes mapped to the introns of *ABCC5*, *IMPDH1*, *ITPA* and *XDH.* An intronic SNP in *ITPA*, rs7270101, previously reported to be associated with thiopurine metabolism and febrile neutropenia in US ALL patients [[20](#R20)] and a poorer event-free survival in Asian ALL patients [[42](#R42)], was also associated with increased risk for relapse in the ALL97 patients ([tables 2](#T2) & [3](#T3)).

Nonthiopurine pathway SNPs associated with the survival and relapse phenotypes were located in or near the *ZP4*, *RAD51AP2*, *VSNL1*, *A2BP1* and *MAPK4*genes ([tables 4](#T4) & [5](#T5)). Since these SNPs had been selected from among top SNPs associated with 6-TG and 6-MP IC_50_ values observed during the LCL GWAS, and since SNPs that were positively associated with IC_50_ values during those studies would be expected to be related to drug resistance, potentially resulting in worse outcomes (hazard ratio >1), the results listed in [table 4](#T4) & [5](#T5) for outcomes were all compatible with predictions from the LCL thiopurine cytotoxicity GWAS.

### Table 4.

| Phenotype | SNP ID | Chr | Gene/nearby genes | Location | MAF | HR | p-value |
| --- | --- | --- | --- | --- | --- | --- | --- |
| Survival | rs1565430† | 1 | ZP4 | 5′-FR | 0.24 | 2.72 | 8.06 × 10−3 |
| rs424827† | 2 | RAD51AP2 | 5′-FR | 0.31 | 2.32 | 6.57 × 10−3 |  |
| rs2710688† | 2 | VSNL1 | Intron | 0.36 | 2.51 | 3.67 × 10−3 |  |
| rs5917990† | X | ATP6AP2 | Flanking | 0.34 | 1.88 | 8.47 × 10−3 |  |
|   |  |  |  |  |  |  |  |
| Relapse | rs665312† | 2 | RAD51AP2 | Exon (s) | 0.32 | 2.58 | 8.04 × 10−3 |
| rs424827† | 2 | RAD51AP2 | UTR | 0.31 | 2.62 | 5.14 × 10−3 |  |
| rs7017705† | 8 |   |   | 0.48 | 0.32 | 2.92 × 10−3 |  |
| rs13273490† | 8 |   |   | 0.49 | 3.84 | 1.03 × 10−3 |  |
| rs9924075† | 16 | A2BP1 | Intron | 0.43 | 2.65 | 5.92 × 10−3 |  |
| rs11662176 | 18 | MAPK4 | Intron | 0.33 | 1.86 | 5.95 × 10−3 |  |
| rs9953685 | 18 | MAPK4 | Intron | 0.33 | 1.80 | 8.37 × 10−3 |  |

Table 4 Caption: Top ‘nonthiopurine pathway’ SNPs associated with clinical phenotypes in UK ALL97 patients: treatment outcomes.

### Table 5.

| Phenotype | SNP ID | Chr | Gene/nearby gene(s) | Location | MAF | (Direction) Increase in R2† | p-value |
| --- | --- | --- | --- | --- | --- | --- | --- |
| TPMT activity | rs1040637 | 6 | FAM8A6P | 5′-FR | 0.38 | (+) 0.013 | 5.82 × 10−3 |
| rs200148‡ | 6 | HIVEP2, AIG1 | 5′-FR | 0.44 | (−) 0.028 | 4.95 × 10−3 |  |
|   |  |  |  |  |  |  |  |
| 6-TG, 6-TGN | rs2215290 | 17 | NME1 | 5′-FR | 0.10 | (−) 0.048 | 4.56 × 10−3 |
| rs3760467 | 17 | NME1, NME2 | TFBS | 0.10 | (−) 0.048 | 4.56 × 10−3 |  |
| rs1558254 | 17 | NME1, NME2 | Intron | 0.10 | (−) 0.048 | 4.56 × 10−3 |  |
|   |  |  |  |  |  |  |  |
| 6-MP, 6-TGN | rs877419 | 8 | PAG1 | 3′-FR | 0.49 | (−) 0.052 | 6.99 × 10−4 |
| rs11001976 | 10 | KCNMA1 | Intron | 0.11 | (+) 0.038 | 4.06 × 10−3 |  |
| rs17480656 | 10 | KCNMA1 | Intron | 0.11 | (+) 0.042 | 2.31 × 10−3 |  |
| rs12765834 | 10 | KCNMA1 | Intron | 0.11 | (+) 0.042 | 2.31 × 10−3 |  |
| rs17389791 | 10 | KCNMA1 | Intron | 0.10 | (+) 0.042 | 2.31 × 10−3 |  |
| rs11001997 | 10 | KCNMA1 | Intron | 0.10 | (+) 0.042 | 2.31 × 10−3 |  |
|   |  |  |  |  |  |  |  |
| 6-MP, MeMPN | rs11707515 | 3 | FRMD4B | Intron | 0.25 | (−) 0.028 | 9.86 × 10−3 |
| rs6549198 | 3 | FRMD4B | Intron | 0.25 | (−) 0.031 | 7.03 × 10−3 |  |
| rs10473594 | 5 | CDH12 | Intron | 0.22 | (−) 0.036 | 3.62 × 10−3 |  |
| rs12886319 | 14 | GNG2 | Intron | 0.15 | (−) 0.069 | 4.21 × 10−5 |  |
|   |  |  |  |  |  |  |  |
| 6-MP, MeMPN/6-TGN | rs12886319 | 14 | GNG2 | Intron | 0.15 | (−) 0.046 | 4.32 × 10−4 |
| rs3748483 | 20 | SLC24A3 | Flanking | 0.31 | (+) 0.036 | 1.89 × 10−3 |  |

Table 5 Caption: Top ‘nonthiopurine pathway’ SNPs associated with clinical phenotypes in UK ALL97 patients: TPMT activity and thiopurine metabolites.

### Associations with RBC TPMT activity & thiopurine metabolite concentrations

Reductions in on-chemotherapy RBC TPMT activity were associated with SNPs in introns of the thiopurine pathway genes *XDH*, *IMPDH1*, *SLC28A3* and *ABCC4* ([tables 2](#T2) & [3](#T3)). A SNP in the nonthiopurine pathway gene *FAM8A6P* (rs1040637) was associated with increased TPMT activity, while rs200148, which mapped between *AIG1* and *HIVEP2* ([tables 4](#T4) & [5](#T5)), was associated with reduced enzyme activity. In the LCLs, the rs200148 SNP was associated with increased sensitivity to 6-TG and 6-MP, as would be expected for a reduction in TPMT activity [[4](#R4)].

As anticipated [[4](#R4)], levels of 6-TGNs, active thiopurine metabolites, were negatively correlated with on-chemotherapy TPMT activity (6-MP: r_S_ = −0.247, p ≤ 0.0001, 6-TG: r_S_ = −0.266, p ≤ 0.0001). 6-TGN and MeMPN concentrations and the MeMPN:TGN ratio were associated with SNPs in the thiopurine pathway genes *TPMT*, *ABCC4* and in *SLC28A3* ([tables 2](#T2) & [3](#T3)). Intronic SNPs in the nonthiopurine pathway genes *CDH12*, *FRMD4B* and *GNG2* were associated with reduced levels of MeMPNs (p < 0.01). SNPs that mapped to *KCNMA1* and *PAG1* were associated with increased 6-TGN levels in patients treated with 6-MP ([tables 4](#T4) & [5](#T5)).

### SNP associations with gene expression

Since SNPs can regulate gene expression in either a *cis* or *trans* fashion [[35](#R35)], we took advantage of the LCL model system, with the full realization that LCLs are not lymphoblasts, to test for associations of SNPs that were associated with clinical phenotypes ([tables 2](#T2), [3](#T3), [4](#T4) & [5](#T5)) with mRNA expression data in the Human Variation Panel LCLs ([Supplementary table 4](#SD1)). For example, the *ITPA* intronic SNP rs7270101 was associated (p = 5.07 × 10^−9^) with reduced expression of its gene. These observations confirmed a previous report that this SNP altered splicing and was associated with decreased *ITPA* expression and decreased ITPA enzyme activity [[43](#R43)]. The rs7321807 and rs7325256 SNPs in *ABCC4* were in linkage disequilibrium (r^2^= 1) and were associated (p = 3.37 × 10^−3^ and 3.90 × 10^−3^) with increased expression of *ABCC4* in LCLs ([Supplementary table 4](#SD1)). These associations for *ABCC4* SNPs were not significant after correcting for multiple comparisons, but their possible functional role was supported by their association with worse clinical outcomes for the UK ALL97 patients ([tables 2](#T2) & [3](#T3)). By studying possible *trans* expression associations for SNPs, we found that rs200148 in the region between *HIVEP2* and *AIG1* was associated with both reduced TPMT activity ([table 5](#T5)) and with expression of the thiopurine pathway genes *ADA* (p = 7.39 × 10^−4^) and *GART* (p = 1.46 × 10^−7^; [Supplementary table 4](#SD1)).

### Candidate gene functional validation

As the final step in our series of experiments, we selected a panel of genes to study for their effects on thiopurine cytotoxicity in cancer cell lines as a functional test of the associations that we had observed. Genes were selected for functional validation if SNPs associated with at least one phenotype with p < 0.01 ([tables 2](#T2), [3](#T3), [4](#T4) & [5](#T5)) were located in or near (<200 kb) that gene. GWAS identified six SNPs associated with both 6-TG and 6-MP cytotoxicity, functional validation was performed for two of these (*CCND2* and *C10orf118*) together with SNPs associated with clinical outcome. We did not prioritize pseudo-genes or genes without names, rather we investigated SNPs in known functional genes. The *ADA* and *GART* genes were added as a result of our *trans* expression association analysis of top SNPs in the LCLs ([Supplementary table 4](#SD1)). A total of 21 genes were selected for functional testing. Each of these genes was required to be expressed in at least two tumor cell lines out of a panel composed of HeLa, OVCAR10, U251 and U87MG. We first obtained basal expression array data for all four cell lines. In order to proceed with testing, we required an intensity level that indicated sufficient mRNA for siRNA gene knockdown to be meaningful and for qRT-PCR to measure both basal gene expression and the efficiency of siRNA knockdown.

Validation required that siRNA knockdown result in a significant shift of the cytotoxicity curve for at least two cell lines. For example, after the knockdown of *ATP6AP2*, *FRMD4B*, *GNG2*, *KCNMA1, UTP18* and *NME1*, the cell lines were more resistant to both drugs ([Figure 3](#F3)). Functional validation of thiopurine pathway genes showed significant shifts of cytotoxicity curves in the anticipated direction (p < 0.05) for *TPMT* and *ITPA* ([Figure 3](#F3)) as well as *ABCC4*, *ADA* and *SLC28A3* in HeLa cells ([Supplementary Figure 1](#SD1)). The anticipated increase in sensitivity was observed after *TPMT* knockdown, while increased resistance was observed after *SLC28A3* knockdown – as anticipated based on our previous LCL-expression studies [[30](#R30)]. The overall results of the functional analyses are summarized in [Supplementary table 5](#SD1).

### Figure 3. Functional validation of candidate genes in cancer cell lines.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/98ec/4027966/78e39e09fb5b/nihms577420f3.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4027966_nihms577420f3.jpg)

(A) ‘Nonthiopurine pathway’ genes. (B) Thiopurine pathway genes. Cytotoxicity curves after gene knockdown followed by drug treatment are shown for top candidate genes that showed consistent ‘shifts’ in at least two cell lines. In this screening test best fit parameters of a four parameter nonlinear regression model were calculated and statistical significance was calculated. p-values are indicated if they were <0.05. In cases where the curves did not fit the model (GNG2 and KCNMA1 in U87M cells and NME1 in U251 cells, all for 6-TG cytotoxicity) the curves could not be compared therefore p-values could not be calculated although a clear curve shift can be observed. 6-MP: 6-mercaptopurine; 6-TG: 6-thioguanine.

## Discussion

We have applied a ‘three-tiered’ experimental strategy in an attempt to identify and functionally validate novel associations of SNPs, and their corresponding genes, with variation in thiopurine response phenotypes in UK ALL97 patients. The SNPs were initially identified on the basis of 6-TG and 6-MP cytotoxicity GWAS performed with LCLs or as tagSNPs for thiopurine pathway genes. Obviously, p-values of 1 × 10^−4^ during the LCL GWAS were not genome-wide significant after correction for multiple comparisons, but the purpose was to identify ‘candidate’ SNPs to test by genotyping clinical samples to determine whether they might be associated with clinical phenotypes for ALL patients treated with thiopurines. It should be emphasized that all of these patients were treated with thiopurines, but they were also treated with several other drugs. We anticipated, correctly, that associations determined using clinical samples would probably be inconclusive, which is why we also included a final functional validation step for candidate SNPs associated with clinical phenotypes as the final step in our three-tiered strategy.

The LCL studies identified only six SNPs associated with both 6-TG and 6-MP IC_50_ values. This is perhaps a direct reflection of the differing activation pathways for the two thiopurines (6-TG forms TGNs directly whereas 6-MP forms a variety of nucleotide metabolites prior to TGN formation), coupled to the limitations of studying drug metabolism in a cell-line system, a system that cannot reflect the presystemic metabolism and intra- and inter-cellular drug metabolite transfer that occurs *in vivo*. Therefore the LCL model system is of greatest value for the study of pharmacodynamic effects rather than pharmacokinetic effects. Because the LCL system is unable to address many genes encoding proteins involved in pharmacokinetics we added tagSNPs for thiopurine pathway genes when we analyzed the clinical samples. In our validation studies, when we knocked-down both thiopurine pathway and nonthiopurine pathway genes, we found that 20 (out of 25 genes tested) altered the resistance to both 6-TG and 6-MP.

The most significant associations observed were for intronic SNPs in *SLC28A3* with reduced 6-TGN levels and MeMPN:6-TGN ratios during 6-MP therapy and for *ABCC4* intronic SNPs with relapse and survival ([tables 2](#T2) & [3](#T3)). Knockdown of *SLC28A3* resulted in increased drug resistance ([Supplementary Figure 1](#SD1)). A large number of SNPs associated with treatment outcome in ALL97 patients mapped to genes encoding the ABCC4 and ABCC5 transporters ([tables 2](#T2) & [3](#T3)). These genes encode proteins that can extrude nucleotide monophosphates from the cell [[16](#R16),[30](#R30)]. Increased expression of *ABCC4* is associated with the rs7321807 and rs7325256 SNPs ([Supplementary table 4](#SD1)), which may explain drug resistance with increased risk of relapse, as observed in this study. As anticipated, knockdown of *ABCC4* in HeLa cells resulted in lower IC_50_ values and increased thiopurine sensitivity ([Supplementary Figure 1](#SD1)), in agreement with results reported by Krishnamurthy *et al.* and Li *et al.*, both of whom showed that decreased expression of *ABCC4* results in the intracellular accumulation of 6-TGNs and increased drug sensitivity [[16](#R16),[30](#R30)].

The rs7270101 SNP in *ITPA* has been reported to alter splicing and is associated with decreased *ITPA* expression and enzyme activity [[43](#R43)], resulting in the accumulation of thioinosine triphosphate and reduced production of 6-TGNs. Our knockdown of *ITPA* ([Figure 3](#F3)) resulted in increased IC_50_ values in OVCAR10 and U251 cells, demonstrating that reduced *ITPA* expression in tumor cells results in thiopurine resistance. These observations may be related to both the increased risk of relapse in the patients in this study who carried this polymorphism ([tables 2](#T2) & [3](#T3)) and to the decreased survival associated with *ITPA* polymorphisms reported in Asian ALL patients [[42](#R42)].

TPMT enzyme activity is inversely correlated with 6-TGN levels [[4](#R4)]. Reduced RBC 6-TGN concentrations as a result of elevated TPMT activity may increase the risk of relapse [[44](#R44)], while high levels of 6-TGNs may result in myelosuppression [[4](#R4),[8](#R8)]. However, SNPs located in regions of the genome other than *TPMT* itself may also influence level of TPMT activity [[27](#R27),[45](#R45)]. For example, we observed an association of reduced TPMT activity with the rs200148 SNP that lies between *AIG1* and *HIVEP2* ([tables 4](#T4) & [5](#T5)). This SNP was also negatively associated with expression of the thiopurine pathway gene *ADA* in LCLs ([Supplementary table 4](#SD1)). We also showed that siRNA knockdown of *ADA* resulted in increased sensitivity to 6-TG and 6-MP in HeLa cells ([Supplementary Figure 1](#SD1)), similar to the effect of *TPMT* knockdown. Although it was not possible to perform siRNA knockdown for all of the candidate genes that we identified, we did show that knockdown of *ATP6AP2, GNG2, FRMD4B, NME1, KCNMA1* and *UTP18* resulted in increased resistance to thiopurine drug-induced cytotoxicity (Figure 4 & [Supplementary table 5](#SD1)). Possible mechanisms responsible for these observations should be pursued in the course of future studies.

It should be emphasized that LCLs, like all model systems, have limitations. However, it should be emphasized once again that this LCL model system has already repeatedly demonstrated its power to generate pharmacogenomic hypotheses [[28](#R28),[32](#R32),[33](#R33),[46](#R46)] and to pursue pharmacogenomic hypotheses from clinical GWAS [[31](#R31),[34](#R34),[47](#R47)]. The present study shows that SNPs identified during GWAS performed with this model system may help lead us to genomic regions of possible importance for thiopurine treatment response. It should also be emphasized, as pointed out earlier, that mutations in the tumor genome obviously play a critical role in variation in antineoplastic drug response, but our studies show that germline polymorphisms can also play an important role in variation in response to antineoplastic drug therapy [[10](#R10)].

In summary, we have applied a three-tiered research strategy to identify, test using clinical samples, and functionally validate the relationships among SNPs, clinical outcomes, gene expression and the cytotoxicity of thiopurine drugs used to treat childhood ALL. We observed a series of novel associations with clinical response phenotypes in ALL patients, but few of these associations were significant after correcting for multiple comparisons (p < 7.1 × 10^−5^). That is why our use of a three-step research strategy, utilizing three different systems, was essential. We used HeLa, U251, U87M and OVCAR10 cancer cell lines for the final functional validation; chosen because the target genes were expressed in these cell lines and their use was without the technical difficulties associated with validation in nonadherent cell lines.

## Conclusion

In conclusion, the results of the studies described here contribute to our understanding of the pharmacogenomics of thiopurine response in the therapy of childhood ALL. We did not identify any new, strong candidate genes but confirmed earlier reports of the functional importance, and impact, of genes in the thiopurine pathway – especially ABC and SLC transporter genes – and added novel nonthiopurine pathway candidate genes of potential importance for clinical drug-response phenotypes. These results may also point to a future in which panels of biomarkers might help to better optimize and monitor thiopurine therapy of childhood ALL.

## Future perspective

Our study has shown that germline polymorphisms can play an important role in the variation in response to the chemotherapy of childhood ALL. We specifically concentrated on the response to thiopurine therapy because, for many successful ALL protocols, thiopurine maintenance chemotherapy is the backbone of the long-term (2 to 3 years) continuing chemotherapy. Treatment outcome after disease relapse remains poor and so the delivery of optimum drug therapy in first remission is of prime importance. The most significant associations with treatment outcome were observed for SNPs in the genes encoding ABC and SLC transporters: a large number of SNPs mapped to genes encoding *ABCC4* and *ABCC5,* observations that confirm earlier reports of their clinical importance. Also within genes influencing thiopurine metabolite formation, we observed an increased relapse risk with the rs7270101 *ITPA* polymorphism and reduced ITPA expression in the knockdown studies resulted in increased thiopurine resistance. These results indicate SNPs that could modulate the response to thiopurine drugs and future studies should investigate these SNPs as a biomarker panel, along with indices of TPMT status and thiopurine metabolite formation, which could be used to optimize the delivery of thiopurine drugs. Potentially, the biomarkers could identify thiopurine ‘good response’ patients in whom the intensity of total chemotherapy could potentially be reduced and *vice versa*, the identification of thiopurine nonresponders who may require additional risk-directed intensification of therapy.

Additionally, our studies showed that knockdown of *ATP6AP2, GNG2, FRMD4B, NME1, KCNMA1* and *UTP18,* genes that were associated with clinical phenotypes but not directly associated with thiopurine metabolism or recirculation, resulted in an increased resistance to thiopurine drug-induced cytotoxicity. Possible mechanisms responsible for these observations should be pursued in the course of future studies; this may enable an increased understanding of the underlying processes that play a role in thiopurine response and treatment outcome.

## Supplementary Material

### Aim

### Aim

### Aim

### Aim

### ‘Human variation panel’ 6-thioguanine & 6-mercaptopurine cytotoxicity genome-wide association study

### Association analysis for UK ALL97 patients

### Candidate gene functional validation

### Conclusion

## Acknowledgments

The authors wish to thank all the clinicians who entered patients into UK ALL97 and the participating children and their families. The authors thank the Clinical Trials Service Unit, University of Oxford, for patient outcome and demographic data and the UK Medical Research Council funded UK Childhood Leukaemia Working Party, which oversaw recruitment to the trial and enabled the thiopurine studies. The authors thank L Wussow for her assistance with the preparation of this manuscript.

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Escherich GM, Richards S, Stork LC, Vora AJ. Meta-analysis of randomised trials comparing thiopurines in childhood acute lymphoblastic leukaemia. Leukemia. 2011;25:953–959. doi: 10.1038/leu.2011.37.  [DOI](https://doi.org/10.1038/leu.2011.37) | [PMC free article](/articles/PMC3112460/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21372841/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Leukemia&title=Meta-analysis%20of%20randomised%20trials%20comparing%20thiopurines%20in%20childhood%20acute%20lymphoblastic%20leukaemia&author=GM%20Escherich&author=S%20Richards&author=LC%20Stork&author=AJ%20Vora&volume=25&publication_year=2011&pages=953-959&pmid=21372841&doi=10.1038/leu.2011.37&)

2. Lennard L, Richards S, Cartwright CS, Mitchell C, Lilleyman JS, Vora A. The thiopurine methyltransferase genetic polymorphism is associated with thioguanine-related veno-occlusive disease of the liver in children with acute lymphoblastic leukemia. Clin Pharmacol Ther. 2006;80(4):375–383. doi: 10.1016/j.clpt.2006.07.002.  [DOI](https://doi.org/10.1016/j.clpt.2006.07.002) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17015055/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=The%20thiopurine%20methyltransferase%20genetic%20polymorphism%20is%20associated%20with%20thioguanine-related%20veno-occlusive%20disease%20of%20the%20liver%20in%20children%20with%20acute%20lymphoblastic%20leukemia&author=L%20Lennard&author=S%20Richards&author=CS%20Cartwright&author=C%20Mitchell&author=JS%20Lilleyman&volume=80&issue=4&publication_year=2006&pages=375-383&pmid=17015055&doi=10.1016/j.clpt.2006.07.002&)

3. Vora A, Mitchell CD, Lennard L, et al. Toxicity and efficacy of 6-thioguanine versus 6-mercaptopurine in childhood lymphoblastic leukaemia: a randomised trial. Lancet. 2006;368(9544):1339–1348. doi: 10.1016/S0140-6736(06)69558-5.  [DOI](https://doi.org/10.1016/S0140-6736(06)69558-5) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17046466/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Toxicity%20and%20efficacy%20of%206-thioguanine%20versus%206-mercaptopurine%20in%20childhood%20lymphoblastic%20leukaemia:%20a%20randomised%20trial&author=A%20Vora&author=CD%20Mitchell&author=L%20Lennard&volume=368&issue=9544&publication_year=2006&pages=1339-1348&pmid=17046466&doi=10.1016/S0140-6736(06)69558-5&)

4. Wang L, Weinshilboum R. Thiopurine S-methyltransferase pharmacogenetics: insights, challenges and future directions. Oncogene. 2006;25(11):1629–1638. doi: 10.1038/sj.onc.1209372. Key conclusions with respect to the TPMT genetic polymorphism. Discusses unresolved issues and areas for future emphasis.  [DOI](https://doi.org/10.1038/sj.onc.1209372) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16550163/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncogene&title=Thiopurine%20S-methyltransferase%20pharmacogenetics:%20insights,%20challenges%20and%20future%20directions&author=L%20Wang&author=R%20Weinshilboum&volume=25&issue=11&publication_year=2006&pages=1629-1638&pmid=16550163&doi=10.1038/sj.onc.1209372&)

5. Tallen G, Ratei R, Mann G, et al. Long-term outcome in children with relapsed acute lymphoblastic leukemia after time-point and site-of-relapse stratification and intensified short-course multidrug chemotherapy: results of trial ALL-REZ BFM 90. J Clin Oncol. 2010;28(14):2339–2347. doi: 10.1200/JCO.2009.25.1983.  [DOI](https://doi.org/10.1200/JCO.2009.25.1983) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20385996/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Long-term%20outcome%20in%20children%20with%20relapsed%20acute%20lymphoblastic%20leukemia%20after%20time-point%20and%20site-of-relapse%20stratification%20and%20intensified%20short-course%20multidrug%20chemotherapy:%20results%20of%20trial%20ALL-REZ%20BFM%2090&author=G%20Tallen&author=R%20Ratei&author=G%20Mann&volume=28&issue=14&publication_year=2010&pages=2339-2347&pmid=20385996&doi=10.1200/JCO.2009.25.1983&)

6. Schmiegelow K, Al-Modhwahi I, Andersen MK, et al. Methotrexate/6-mercaptopurine maintenance therapy influences the risk of a second malignant neoplasm after childhood acute lymphoblastic leukemia: results from the NOPHO ALL-92 study. Blood. 2009;113(24):6077–6084. doi: 10.1182/blood-2008-11-187880.  [DOI](https://doi.org/10.1182/blood-2008-11-187880) | [PMC free article](/articles/PMC2699230/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19224761/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Methotrexate/6-mercaptopurine%20maintenance%20therapy%20influences%20the%20risk%20of%20a%20second%20malignant%20neoplasm%20after%20childhood%20acute%20lymphoblastic%20leukemia:%20results%20from%20the%20NOPHO%20ALL-92%20study&author=K%20Schmiegelow&author=I%20Al-Modhwahi&author=MK%20Andersen&volume=113&issue=24&publication_year=2009&pages=6077-6084&pmid=19224761&doi=10.1182/blood-2008-11-187880&)

7. Lund B, Asberg A, Heyman M, et al. Risk factors for treatment related mortality in childhood acute lymphoblastic leukaemia. Pediatric Blood Cancer. 2010;56(4):551–559. doi: 10.1002/pbc.22719.  [DOI](https://doi.org/10.1002/pbc.22719) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21298739/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pediatric%20Blood%20Cancer&title=Risk%20factors%20for%20treatment%20related%20mortality%20in%20childhood%20acute%20lymphoblastic%20leukaemia&author=B%20Lund&author=A%20Asberg&author=M%20Heyman&volume=56&issue=4&publication_year=2010&pages=551-559&pmid=21298739&doi=10.1002/pbc.22719&)

8. Lennard L, Rees CA, Lilleyman JS, Maddocks JL. Childhood leukaemia: a relationship between intracellular 6-mercaptopurine metabolites and neutropenia. 1983. Br J Clin Pharmacol. 2004;58(7):S867–S871. doi: 10.1111/j.1365-2125.2004.02314.x. discussion S872–S864.  [DOI](https://doi.org/10.1111/j.1365-2125.2004.02314.x) | [PMC free article](/articles/PMC1884668/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15595985/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Clin%20Pharmacol&title=Childhood%20leukaemia:%20a%20relationship%20between%20intracellular%206-mercaptopurine%20metabolites%20and%20neutropenia.%201983&author=L%20Lennard&author=CA%20Rees&author=JS%20Lilleyman&author=JL%20Maddocks&volume=58&issue=7&publication_year=2004&pages=S867-S871&pmid=15595985&doi=10.1111/j.1365-2125.2004.02314.x&)

9. Gaynon PS. Childhood acute lymphoblastic leukaemia and relapse. Br J Haematol. 2005;131(5):579–587. doi: 10.1111/j.1365-2141.2005.05773.x.  [DOI](https://doi.org/10.1111/j.1365-2141.2005.05773.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16351633/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Haematol&title=Childhood%20acute%20lymphoblastic%20leukaemia%20and%20relapse&author=PS%20Gaynon&volume=131&issue=5&publication_year=2005&pages=579-587&pmid=16351633&doi=10.1111/j.1365-2141.2005.05773.x&)

10. Wang L, McLeod HL, Weinshilboum RM. Genomics and drug response. N Engl J Med. 2011;364(12):1144–1153. doi: 10.1056/NEJMra1010600.  [DOI](https://doi.org/10.1056/NEJMra1010600) | [PMC free article](/articles/PMC3184612/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21428770/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Genomics%20and%20drug%20response&author=L%20Wang&author=HL%20McLeod&author=RM%20Weinshilboum&volume=364&issue=12&publication_year=2011&pages=1144-1153&pmid=21428770&doi=10.1056/NEJMra1010600&)

11. Zaza G, Cheok M, Krynetskaia N, et al. Thiopurine pathway. Pharmacogenet Genomics. 2010;20(9):573–574. doi: 10.1097/FPC.0b013e328334338f.  [DOI](https://doi.org/10.1097/FPC.0b013e328334338f) | [PMC free article](/articles/PMC3098750/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19952870/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Thiopurine%20pathway&author=G%20Zaza&author=M%20Cheok&author=N%20Krynetskaia&volume=20&issue=9&publication_year=2010&pages=573-574&pmid=19952870&doi=10.1097/FPC.0b013e328334338f&)

12. Lennard L, Lewis IJ, Michelagnoli M, Lilleyman JS. Thiopurine methyltransferase deficiency in childhood lymphoblastic leukaemia: 6-mercaptopurine dosage strategies. Med Pediatr Oncol. 1997;29(4):252–255. doi: 10.1002/(sici)1096-911x(199710)29:4<252::aid-mpo3>3.0.co;2-l.  [DOI](https://doi.org/10.1002/(sici)1096-911x(199710)29:4<252::aid-mpo3>3.0.co;2-l) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9251729/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Med%20Pediatr%20Oncol&title=Thiopurine%20methyltransferase%20deficiency%20in%20childhood%20lymphoblastic%20leukaemia:%206-mercaptopurine%20dosage%20strategies&author=L%20Lennard&author=IJ%20Lewis&author=M%20Michelagnoli&author=JS%20Lilleyman&volume=29&issue=4&publication_year=1997&pages=252-255&pmid=9251729&doi=10.1002/(sici)1096-911x(199710)29:4<252::aid-mpo3>3.0.co;2-l&)

13. Lennard L. The clinical pharmacology of 6-mercaptopurine. Euro J Clin Pharmacol. 1992;43(4):329–339. doi: 10.1007/BF02220605.  [DOI](https://doi.org/10.1007/BF02220605) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/1451710/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Euro%20J%20Clin%20Pharmacol&title=The%20clinical%20pharmacology%20of%206-mercaptopurine&author=L%20Lennard&volume=43&issue=4&publication_year=1992&pages=329-339&pmid=1451710&doi=10.1007/BF02220605&)

14. Relling MV, Gardner EE, Sandborn WJ, et al. Clinical Pharmacogenetics Implementation Consortium guidelines for thiopurine methyltransferase genotype and thiopurine dosing. Clin Pharmacol Ther. 2011;89(3):387–391. doi: 10.1038/clpt.2010.320. Summarizes the evidence for the use of TPMT in dosing thiopurine drugs and optimizing thiopurine therapy and states dosing recommendations based on TPMT genotype.  [DOI](https://doi.org/10.1038/clpt.2010.320) | [PMC free article](/articles/PMC3098761/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21270794/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Clinical%20Pharmacogenetics%20Implementation%20Consortium%20guidelines%20for%20thiopurine%20methyltransferase%20genotype%20and%20thiopurine%20dosing&author=MV%20Relling&author=EE%20Gardner&author=WJ%20Sandborn&volume=89&issue=3&publication_year=2011&pages=387-391&pmid=21270794&doi=10.1038/clpt.2010.320&)

15. Duley JA, Somogi AA, Martin JH. The future of thiopurine pharmacogenenomics. Pharmacogenomics. 2012;13(14):1549–1552. doi: 10.2217/pgs.12.140. Discusses the pharmacogenomic potential of candidate genes to modify the prescription of thiopurine drugs. Our work identified top SNPs in some of these genes which were associated with metabolite concentrations in UK acute lymphoblastic leukemia patients.  [DOI](https://doi.org/10.2217/pgs.12.140) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23148628/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=The%20future%20of%20thiopurine%20pharmacogenenomics&author=JA%20Duley&author=AA%20Somogi&author=JH%20Martin&volume=13&issue=14&publication_year=2012&pages=1549-1552&pmid=23148628&doi=10.2217/pgs.12.140&)

16. Krishnamurthy P, Schwab M, Takenaka K, et al. Transporter-mediated protection against thiopurine-induced hematopoietic toxicity. Cancer Res. 2008;68(13):4983–4989. doi: 10.1158/0008-5472.CAN-07-6790.  [DOI](https://doi.org/10.1158/0008-5472.CAN-07-6790) | [PMC free article](/articles/PMC3323115/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18593894/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=Transporter-mediated%20protection%20against%20thiopurine-induced%20hematopoietic%20toxicity&author=P%20Krishnamurthy&author=M%20Schwab&author=K%20Takenaka&volume=68&issue=13&publication_year=2008&pages=4983-4989&pmid=18593894&doi=10.1158/0008-5472.CAN-07-6790&)

17. Janke D, Mehralivand S, Strand D, et al. 6-mercaptopurine and 9-(2-phosphonyl-methoxyethyl) adenine (PMEA) transport altered by two missense mutations in the drug transporter gene ABCC4. Hum Mutat. 2008;29(5):659–669. doi: 10.1002/humu.20694.  [DOI](https://doi.org/10.1002/humu.20694) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18300232/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mutat&title=6-mercaptopurine%20and%209-(2-phosphonyl-methoxyethyl)%20adenine%20(PMEA)%20transport%20altered%20by%20two%20missense%20mutations%20in%20the%20drug%20transporter%20gene%20ABCC4&author=D%20Janke&author=S%20Mehralivand&author=D%20Strand&volume=29&issue=5&publication_year=2008&pages=659-669&pmid=18300232&doi=10.1002/humu.20694&)

18. Ansari M, Sauty G, Labuda M, et al. Polymorphisms in multidrug resistance-associated protein gene 4 is associated with outcome in childhood acute lymphoblastic leukemia. Blood. 2009;114(7):1383–1386. doi: 10.1182/blood-2008-11-191098.  [DOI](https://doi.org/10.1182/blood-2008-11-191098) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19515727/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Polymorphisms%20in%20multidrug%20resistance-associated%20protein%20gene%204%20is%20associated%20with%20outcome%20in%20childhood%20acute%20lymphoblastic%20leukemia&author=M%20Ansari&author=G%20Sauty&author=M%20Labuda&volume=114&issue=7&publication_year=2009&pages=1383-1386&pmid=19515727&doi=10.1182/blood-2008-11-191098&)

19. Stocco G, Cheok MH, Crews KR, et al. Genetic polymorphism of inosine triphosphate pyrophosphatase is a determinant of mercaptopurine metabolism and toxicity during treatment for acute lymphoblastic leukemia. Clin Pharmacol Ther. 2009;85(2):164–172. doi: 10.1038/clpt.2008.154. Highlights the influence of SNPs in thiopurine pathways genes, other than TPMT, on thiopurine toxicity.  [DOI](https://doi.org/10.1038/clpt.2008.154) | [PMC free article](/articles/PMC2762405/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18685564/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Genetic%20polymorphism%20of%20inosine%20triphosphate%20pyrophosphatase%20is%20a%20determinant%20of%20mercaptopurine%20metabolism%20and%20toxicity%20during%20treatment%20for%20acute%20lymphoblastic%20leukemia&author=G%20Stocco&author=MH%20Cheok&author=KR%20Crews&volume=85&issue=2&publication_year=2009&pages=164-172&pmid=18685564&doi=10.1038/clpt.2008.154&)

20. Hawwa AF, Millership JS, Collier PS, et al. Pharmacogenomic studies of the anticancer and immunosuppressive thiopurines mercaptopurine and azathioprine. Br J Clin Pharmacol. 2008;66(4):517–528. doi: 10.1111/j.1365-2125.2008.03248.x.  [DOI](https://doi.org/10.1111/j.1365-2125.2008.03248.x) | [PMC free article](/articles/PMC2561120/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18662289/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Clin%20Pharmacol&title=Pharmacogenomic%20studies%20of%20the%20anticancer%20and%20immunosuppressive%20thiopurines%20mercaptopurine%20and%20azathioprine&author=AF%20Hawwa&author=JS%20Millership&author=PS%20Collier&volume=66&issue=4&publication_year=2008&pages=517-528&pmid=18662289&doi=10.1111/j.1365-2125.2008.03248.x&)

21. Han S, Lee KM, Park SK, et al. Genome-wide association study of childhood acute lymphoblastic leukemia in Korea. Leuk Res. 2010;34(10):1271–1274. doi: 10.1016/j.leukres.2010.02.001.  [DOI](https://doi.org/10.1016/j.leukres.2010.02.001) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20189245/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Leuk%20Res&title=Genome-wide%20association%20study%20of%20childhood%20acute%20lymphoblastic%20leukemia%20in%20Korea&author=S%20Han&author=KM%20Lee&author=SK%20Park&volume=34&issue=10&publication_year=2010&pages=1271-1274&pmid=20189245&doi=10.1016/j.leukres.2010.02.001&)

22. Papaemmanuil E, Hosking FJ, Vijayakrishnan J, et al. Loci on 7p12.2, 10q21.2 and 14q11.2 are associated with risk of childhood acute lymphoblastic leukemia. Nat Genet. 2009;41(9):1006–1010. doi: 10.1038/ng.430.  [DOI](https://doi.org/10.1038/ng.430) | [PMC free article](/articles/PMC4915548/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19684604/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Loci%20on%207p12.2,%2010q21.2%20and%2014q11.2%20are%20associated%20with%20risk%20of%20childhood%20acute%20lymphoblastic%20leukemia&author=E%20Papaemmanuil&author=FJ%20Hosking&author=J%20Vijayakrishnan&volume=41&issue=9&publication_year=2009&pages=1006-1010&pmid=19684604&doi=10.1038/ng.430&)

23. Trevino LR, Yang W, French D, et al. Germline genomic variants associated with childhood acute lymphoblastic leukemia. Nat Genet. 2009;41(9):1001–1005. doi: 10.1038/ng.432.  [DOI](https://doi.org/10.1038/ng.432) | [PMC free article](/articles/PMC2762391/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19684603/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Germline%20genomic%20variants%20associated%20with%20childhood%20acute%20lymphoblastic%20leukemia&author=LR%20Trevino&author=W%20Yang&author=D%20French&volume=41&issue=9&publication_year=2009&pages=1001-1005&pmid=19684603&doi=10.1038/ng.432&)

24. Horinouchi M, Yagi M, Imanishi H, et al. Association of genetic polymorphisms with hepatotoxicity in patients with childhood acute lymphoblastic leukemia or lymphoma. Pediatr Hematol Oncol. 2010;27(5):344–354. doi: 10.3109/08880011003739422.  [DOI](https://doi.org/10.3109/08880011003739422) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20670164/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pediatr%20Hematol%20Oncol&title=Association%20of%20genetic%20polymorphisms%20with%20hepatotoxicity%20in%20patients%20with%20childhood%20acute%20lymphoblastic%20leukemia%20or%20lymphoma&author=M%20Horinouchi&author=M%20Yagi&author=H%20Imanishi&volume=27&issue=5&publication_year=2010&pages=344-354&pmid=20670164&doi=10.3109/08880011003739422&)

25. Yang JJ, Cheng C, Yang W, et al. Genome-wide interrogation of germline genetic variation associated with treatment response in childhood acute lymphoblastic leukemia. JAMA. 2009;301(4):393–403. doi: 10.1001/jama.2009.7.  [DOI](https://doi.org/10.1001/jama.2009.7) | [PMC free article](/articles/PMC2664534/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19176441/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA&title=Genome-wide%20interrogation%20of%20germline%20genetic%20variation%20associated%20with%20treatment%20response%20in%20childhood%20acute%20lymphoblastic%20leukemia&author=JJ%20Yang&author=C%20Cheng&author=W%20Yang&volume=301&issue=4&publication_year=2009&pages=393-403&pmid=19176441&doi=10.1001/jama.2009.7&)

26. Yang JJ, Cheng C, Devidas M, et al. Genome-wide association study identifies germline polymorphisms associated with relapse of childhood acute lymphoblastic leukemia. Blood. 2012;120(20):4197–4204. doi: 10.1182/blood-2012-07-440107.  [DOI](https://doi.org/10.1182/blood-2012-07-440107) | [PMC free article](/articles/PMC3501717/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23007406/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Genome-wide%20association%20study%20identifies%20germline%20polymorphisms%20associated%20with%20relapse%20of%20childhood%20acute%20lymphoblastic%20leukemia&author=JJ%20Yang&author=C%20Cheng&author=M%20Devidas&volume=120&issue=20&publication_year=2012&pages=4197-4204&pmid=23007406&doi=10.1182/blood-2012-07-440107&)

27. Stocco GS, Yang W, Crews KR, et al. PACSIN2 polymorphism influences TPMT activity and mercaptopurine-related gastrointestinal toxicity. Hum Mol Genet. 2012;21(21):4793–4804. doi: 10.1093/hmg/dds302.  [DOI](https://doi.org/10.1093/hmg/dds302) | [PMC free article](/articles/PMC3471396/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22846425/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mol%20Genet&title=PACSIN2%20polymorphism%20influences%20TPMT%20activity%20and%20mercaptopurine-related%20gastrointestinal%20toxicity&author=GS%20Stocco&author=W%20Yang&author=KR%20Crews&volume=21&issue=21&publication_year=2012&pages=4793-4804&pmid=22846425&doi=10.1093/hmg/dds302&)

28. Li L, Fridley B, Kalari K, et al. Gemcitabine and cytosine arabinoside cytotoxicity: association with lymphoblastoid cell expression. Cancer Res. 2008;68(17):7050–7058. doi: 10.1158/0008-5472.CAN-08-0405.  [DOI](https://doi.org/10.1158/0008-5472.CAN-08-0405) | [PMC free article](/articles/PMC2562356/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18757419/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=Gemcitabine%20and%20cytosine%20arabinoside%20cytotoxicity:%20association%20with%20lymphoblastoid%20cell%20expression&author=L%20Li&author=B%20Fridley&author=K%20Kalari&volume=68&issue=17&publication_year=2008&pages=7050-7058&pmid=18757419&doi=10.1158/0008-5472.CAN-08-0405&)

29. Wheeler HE, Dolan ME. Lymphoblastoid cell lines in pharmacogenomic discovery and clinical translation. Pharmacogenomics. 2012;13(1):55–70. doi: 10.2217/pgs.11.121.  [DOI](https://doi.org/10.2217/pgs.11.121) | [PMC free article](/articles/PMC3292907/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22176622/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Lymphoblastoid%20cell%20lines%20in%20pharmacogenomic%20discovery%20and%20clinical%20translation&author=HE%20Wheeler&author=ME%20Dolan&volume=13&issue=1&publication_year=2012&pages=55-70&pmid=22176622&doi=10.2217/pgs.11.121&)

30. Li F, Fridley BL, Matimba A, et al. Ecto-5′-nucleotidase and thiopurine cellular circulation: association with cytotoxicity. Drug Metab Dispos. 2010;38(12):2329–2338. doi: 10.1124/dmd.110.035220. Demonstrates the intercellular cycling of thiopurines by the ABC and SLC transporter proteins and illustrates how the impaired function of these transporters (specifically SNPs in ABCC4 and SLC28A3) can result in the accumulation of toxic drug metabolites.  [DOI](https://doi.org/10.1124/dmd.110.035220) | [PMC free article](/articles/PMC2993460/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20855458/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Ecto-5%E2%80%B2-nucleotidase%20and%20thiopurine%20cellular%20circulation:%20association%20with%20cytotoxicity&author=F%20Li&author=BL%20Fridley&author=A%20Matimba&volume=38&issue=12&publication_year=2010&pages=2329-2338&pmid=20855458&doi=10.1124/dmd.110.035220&)

31. Ingle JN, Schaid DJ, Goss PE, et al. Genome-wide associations and functional genomic studies of musculoskeletal adverse events in women receiving aromatase inhibitors. J Clin Oncol. 2010;28(31):4674–4682. doi: 10.1200/JCO.2010.28.5064. Demonstrates the use of genome-wide association study-informed functional genomics in cancer therapeutics.  [DOI](https://doi.org/10.1200/JCO.2010.28.5064) | [PMC free article](/articles/PMC3020700/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20876420/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Genome-wide%20associations%20and%20functional%20genomic%20studies%20of%20musculoskeletal%20adverse%20events%20in%20women%20receiving%20aromatase%20inhibitors&author=JN%20Ingle&author=DJ%20Schaid&author=PE%20Goss&volume=28&issue=31&publication_year=2010&pages=4674-4682&pmid=20876420&doi=10.1200/JCO.2010.28.5064&)

32. Pei H, Li L, Fridley BL, et al. FKBP51 affects cancer cell response to chemotherapy by negatively regulating Akt. Cancer Cell. 2009;16(3):259–266. doi: 10.1016/j.ccr.2009.07.016.  [DOI](https://doi.org/10.1016/j.ccr.2009.07.016) | [PMC free article](/articles/PMC2755578/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19732725/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Cell&title=FKBP51%20affects%20cancer%20cell%20response%20to%20chemotherapy%20by%20negatively%20regulating%20Akt&author=H%20Pei&author=L%20Li&author=BL%20Fridley&volume=16&issue=3&publication_year=2009&pages=259-266&pmid=19732725&doi=10.1016/j.ccr.2009.07.016&)

33. Hou J, Wang L. FKBP5 as a selection biomarker for gemcitabine and Akt inhibitors in treatment of pancreatic cancer. PLoS ONE. 2012;7(5):e36252. doi: 10.1371/journal.pone.0036252.  [DOI](https://doi.org/10.1371/journal.pone.0036252) | [PMC free article](/articles/PMC3348935/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22590527/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&title=FKBP5%20as%20a%20selection%20biomarker%20for%20gemcitabine%20and%20Akt%20inhibitors%20in%20treatment%20of%20pancreatic%20cancer&author=J%20Hou&author=L%20Wang&volume=7&issue=5&publication_year=2012&pages=e36252&pmid=22590527&doi=10.1371/journal.pone.0036252&)

34. Liu M, Ingle JN, Fridley BL, et al. TSPYL5 SNPs: association with plasma estradiol concentrations and aromatase expression. Mol Endocrinol. 2013;27(4):657–670. doi: 10.1210/me.2012-1397.  [DOI](https://doi.org/10.1210/me.2012-1397) | [PMC free article](/articles/PMC3607698/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23518928/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Endocrinol&title=TSPYL5%20SNPs:%20association%20with%20plasma%20estradiol%20concentrations%20and%20aromatase%20expression&author=M%20Liu&author=JN%20Ingle&author=BL%20Fridley&volume=27&issue=4&publication_year=2013&pages=657-670&pmid=23518928&doi=10.1210/me.2012-1397&)

35. Li L, Fridley BL, Kalari K, et al. Gemcitabine and arabinosylcytosine pharmacogenomics: genome-wide association and drug response biomarkers. PLoS ONE. 2009;4(11):e7765. doi: 10.1371/journal.pone.0007765.  [DOI](https://doi.org/10.1371/journal.pone.0007765) | [PMC free article](/articles/PMC2770319/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19898621/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&title=Gemcitabine%20and%20arabinosylcytosine%20pharmacogenomics:%20genome-wide%20association%20and%20drug%20response%20biomarkers&author=L%20Li&author=BL%20Fridley&author=K%20Kalari&volume=4&issue=11&publication_year=2009&pages=e7765&pmid=19898621&doi=10.1371/journal.pone.0007765&)

36. Wu Z, Irizarry RA, Gentleman R, Martinez-Murillo F, Spencer F. A model-based background adjustment for oligonucleotide expression arrays. J Am Stat Assoc. 2004;99(468):909–917.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Am%20Stat%20Assoc&title=A%20model-based%20background%20adjustment%20for%20oligonucleotide%20expression%20arrays&author=Z%20Wu&author=RA%20Irizarry&author=R%20Gentleman&author=F%20Martinez-Murillo&author=F%20Spencer&volume=99&issue=468&publication_year=2004&pages=909-917&)

37. Mitchell CD, Richards SM, Kinsey SE, et al. Benefit of dexamethasone compared with prednisolone for childhood acute lymphoblastic leukaemia: results of the UK medical research council ALL97 randomized trial. Br J Haematol. 2005;129(6):734–745. doi: 10.1111/j.1365-2141.2005.05509.x.  [DOI](https://doi.org/10.1111/j.1365-2141.2005.05509.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15952999/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Haematol&title=Benefit%20of%20dexamethasone%20compared%20with%20prednisolone%20for%20childhood%20acute%20lymphoblastic%20leukaemia:%20results%20of%20the%20UK%20medical%20research%20council%20ALL97%20randomized%20trial&author=CD%20Mitchell&author=SM%20Richards&author=SE%20Kinsey&volume=129&issue=6&publication_year=2005&pages=734-745&pmid=15952999&doi=10.1111/j.1365-2141.2005.05509.x&)

38. Lennard L, Singleton HJ. High-performance liquid chromatographic assay of human red blood cell thiopurine methyltransferase activity. J Chromatogr. 1994;661(1):25–33. doi: 10.1016/0378-4347(94)00327-0.  [DOI](https://doi.org/10.1016/0378-4347(94)00327-0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7866549/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Chromatogr&title=High-performance%20liquid%20chromatographic%20assay%20of%20human%20red%20blood%20cell%20thiopurine%20methyltransferase%20activity&author=L%20Lennard&author=HJ%20Singleton&volume=661&issue=1&publication_year=1994&pages=25-33&pmid=7866549&doi=10.1016/0378-4347(94)00327-0&)

39. Lennard L, Singleton HJ. High-performance liquid chromatographic assay of the methyl and nucleotide metabolites of 6-mercaptopurine: quantitation of red blood cell 6-thioguanine nucleotide, 6-thioinosinic acid and 6-methylmercaptopurine metabolites in a single sample. J Chromatogr. 1992;583(1):83–90. doi: 10.1016/0378-4347(92)80347-s.  [DOI](https://doi.org/10.1016/0378-4347(92)80347-s) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/1484095/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Chromatogr&title=High-performance%20liquid%20chromatographic%20assay%20of%20the%20methyl%20and%20nucleotide%20metabolites%20of%206-mercaptopurine:%20quantitation%20of%20red%20blood%20cell%206-thioguanine%20nucleotide,%206-thioinosinic%20acid%20and%206-methylmercaptopurine%20metabolites%20in%20a%20single%20sample&author=L%20Lennard&author=HJ%20Singleton&volume=583&issue=1&publication_year=1992&pages=83-90&pmid=1484095&doi=10.1016/0378-4347(92)80347-s&)

40. Otterness D, Szumlanski C, Lennard L, et al. Human thiopurine methyltransferase pharmacogenetics: gene sequence polymorphisms. Clin Pharmacol Ther. 1997;62(1):60–73. doi: 10.1016/S0009-9236(97)90152-1.  [DOI](https://doi.org/10.1016/S0009-9236(97)90152-1) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9246020/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Human%20thiopurine%20methyltransferase%20pharmacogenetics:%20gene%20sequence%20polymorphisms&author=D%20Otterness&author=C%20Szumlanski&author=L%20Lennard&volume=62&issue=1&publication_year=1997&pages=60-73&pmid=9246020&doi=10.1016/S0009-9236(97)90152-1&)

41. Relling MV, Pui CH, Evans E. Thiopurine methyltransferase in acute lymphoblastic leukaemia. Blood. 2006;107(2):843–844. doi: 10.1182/blood-2005-08-3379.  [DOI](https://doi.org/10.1182/blood-2005-08-3379) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16401827/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Thiopurine%20methyltransferase%20in%20acute%20lymphoblastic%20leukaemia&author=MV%20Relling&author=CH%20Pui&author=E%20Evans&volume=107&issue=2&publication_year=2006&pages=843-844&pmid=16401827&doi=10.1182/blood-2005-08-3379&)

42. Kim H, Kang HJ, Kim HJ, et al. Pharmacogenetic analysis of prediatric patients with acute lymphblastic leukaemia: a possible association between survival rate and ITPA polymorphism. PLoS ONE. 2012;7(9):e45558. doi: 10.1371/journal.pone.0045558.  [DOI](https://doi.org/10.1371/journal.pone.0045558) | [PMC free article](/articles/PMC3454425/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23029095/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&title=Pharmacogenetic%20analysis%20of%20prediatric%20patients%20with%20acute%20lymphblastic%20leukaemia:%20a%20possible%20association%20between%20survival%20rate%20and%20ITPA%20polymorphism&author=H%20Kim&author=HJ%20Kang&author=HJ%20Kim&volume=7&issue=9&publication_year=2012&pages=e45558&pmid=23029095&doi=10.1371/journal.pone.0045558&)

43. Arenas M, Duley J, Sumi S, Sanderson J, Marinaki A. The ITPA c.94C>A and g IVS2+21A>C sequence variants contribute to missplicing of the ITPA gene. Biochim Biophys Acta. 2007;1772(1):96–102. doi: 10.1016/j.bbadis.2006.10.006.  [DOI](https://doi.org/10.1016/j.bbadis.2006.10.006) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17113761/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochim%20Biophys%20Acta&title=The%20ITPA%20c.94C>A%20and%20g%20IVS2+21A>C%20sequence%20variants%20contribute%20to%20missplicing%20of%20the%20ITPA%20gene&author=M%20Arenas&author=J%20Duley&author=S%20Sumi&author=J%20Sanderson&author=A%20Marinaki&volume=1772&issue=1&publication_year=2007&pages=96-102&pmid=17113761&doi=10.1016/j.bbadis.2006.10.006&)

44. Schmiegelow K, Forestier E, Kristinsson J, et al. Thiopurine methyltransferase activity is related to the risk of relapse of childhood acute lymphoblastic leukemia: results from the NOPHO ALL-92 study. Leukemia. 2009;23(3):557–564. doi: 10.1038/leu.2008.316.  [DOI](https://doi.org/10.1038/leu.2008.316) | [PMC free article](/articles/PMC3898327/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18987654/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Leukemia&title=Thiopurine%20methyltransferase%20activity%20is%20related%20to%20the%20risk%20of%20relapse%20of%20childhood%20acute%20lymphoblastic%20leukemia:%20results%20from%20the%20NOPHO%20ALL-92%20study&author=K%20Schmiegelow&author=E%20Forestier&author=J%20Kristinsson&volume=23&issue=3&publication_year=2009&pages=557-564&pmid=18987654&doi=10.1038/leu.2008.316&)

45. Jones TS, Yang W, Evans WE, Relling MV. Using HapMap tools in pharmacogenomic discovery: the thiopurine methyltransferase polymorphism. Clin Pharmacol Ther. 2007;81(5):729–734. doi: 10.1038/sj.clpt.6100135.  [DOI](https://doi.org/10.1038/sj.clpt.6100135) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17329987/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Using%20HapMap%20tools%20in%20pharmacogenomic%20discovery:%20the%20thiopurine%20methyltransferase%20polymorphism&author=TS%20Jones&author=W%20Yang&author=WE%20Evans&author=MV%20Relling&volume=81&issue=5&publication_year=2007&pages=729-734&pmid=17329987&doi=10.1038/sj.clpt.6100135&)

46. Niu N, Qin Y, Fridley BL, et al. Radiation pharmacogenomics: a genome-wide association approach to identify radiation response biomarkers using human lymphoblastoid cell lines. Genome Res. 2010;20(11):1482–1492. doi: 10.1101/gr.107672.110.  [DOI](https://doi.org/10.1101/gr.107672.110) | [PMC free article](/articles/PMC2963812/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20923822/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Res&title=Radiation%20pharmacogenomics:%20a%20genome-wide%20association%20approach%20to%20identify%20radiation%20response%20biomarkers%20using%20human%20lymphoblastoid%20cell%20lines&author=N%20Niu&author=Y%20Qin&author=BL%20Fridley&volume=20&issue=11&publication_year=2010&pages=1482-1492&pmid=20923822&doi=10.1101/gr.107672.110&)

47. Liu M, Wang L, Bongartz T, et al. Aromatase inhibitors, estrogens and musculoskeletal pain: estrogen-dependent T-cell leukemia 1A (TCL1A) gene-mediated regulation of cytokine expression. Breast Cancer Res. 2012;14(2):R41. doi: 10.1186/bcr3137.  [DOI](https://doi.org/10.1186/bcr3137) | [PMC free article](/articles/PMC3446375/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22405131/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Breast%20Cancer%20Res&title=Aromatase%20inhibitors,%20estrogens%20and%20musculoskeletal%20pain:%20estrogen-dependent%20T-cell%20leukemia%201A%20(TCL1A)%20gene-mediated%20regulation%20of%20cytokine%20expression&author=M%20Liu&author=L%20Wang&author=T%20Bongartz&volume=14&issue=2&publication_year=2012&pages=R41&pmid=22405131&doi=10.1186/bcr3137&)
