# Using Exome Data to Identify Malignant Hyperthermia Susceptibility Mutations

## Metadata
**Authors:** Stephen G Gonsalves, David Ng, Jennifer J Johnston, Jamie K Teer, NISC Comparative Sequencing Program‖, Peter D Stenson, David N Cooper, James C Mullikin, Leslie G Biesecker
**Journal:** Anesthesiology
**Date:** 2013 Nov
**DOI:** [10.1097/ALN.0b013e3182a8a8e7](https://doi.org/10.1097/ALN.0b013e3182a8a8e7)
**PMID:** 24195946
**PMCID:** PMC4077354
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4077354/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC4077354/pdf/nihms522989.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC4077354/pdf/nihms522989.pdf)

## Abstract

**Background:** 
Malignant hyperthermia susceptibility (MHS) is a life-threatening, inherited disorder of muscle calcium metabolism, triggered by anesthetics and depolarizing muscle relaxants. An unselected cohort was screened for MHS mutations using exome sequencing. Our aim was to pilot a strategy for the RYR1 and CACNA1S genes.

**Methods:** 
Exome sequencing was performed on 870 volunteers not ascertained for MHS. Variants in RYR1 and CACNA1S were annotated using an algorithm that filtered results based on mutation type, frequency, and information in mutation databases. Variants were scored on a six-point pathogenicity scale. Medical histories and pedigrees were reviewed for malignant hyperthermia and related disorders.

**Results:** 
We identified 70 RYR1 and 53 CACNA1S variants among 870 exomes. Sixty-three RYR1 and 41 CACNA1S variants passed the quality and frequency metrics but we excluded synonymous variants. In RYR1, we identified 65 missense mutations, one nonsense, two that affected splicing, and one non frameshift indel. In CACNA1S, 48 missense, one frameshift deletion, one splicing and one non frameshift indel were identified. RYR1 variants predicted to be pathogenic for MHS were found in three participants without medical or family histories of MHS. Numerous variants, previously described as pathogenic in mutation databases, were reclassified by us to be of unknown pathogenicity.

**Conclusions:** 
Exome sequencing can identify asymptomatic patients at risk for MHS, although the interpretation of exome variants can be challenging. The use of exome sequencing in unselected cohorts is an important tool to understand the prevalence and penetrance of MHS, a critical challenge for the field.

### Background

Malignant hyperthermia susceptibility (MHS) is a life-threatening, inherited disorder of muscle calcium metabolism, triggered by anesthetics and depolarizing muscle relaxants. An unselected cohort was screened for MHS mutations using exome sequencing. Our aim was to pilot a strategy for the *RYR1* and *CACNA1S* genes.

### Methods

Exome sequencing was performed on 870 volunteers not ascertained for MHS. Variants in *RYR1* and *CACNA1S* were annotated using an algorithm that filtered results based on mutation type, frequency, and information in mutation databases. Variants were scored on a six-point pathogenicity scale. Medical histories and pedigrees were reviewed for malignant hyperthermia and related disorders.

### Results

We identified 70 *RYR1* and 53 *CACNA1S* variants among 870 exomes. Sixty-three *RYR1* and 41 *CACNA1S* variants passed the quality and frequency metrics but we excluded synonymous variants. In *RYR1*, we identified 65 missense mutations, one nonsense, two that affected splicing, and one non frameshift indel. In *CACNA1S*, 48 missense, one frameshift deletion, one splicing and one non frameshift indel were identified. *RYR1* variants predicted to be pathogenic for MHS were found in three participants without medical or family histories of MHS. Numerous variants, previously described as pathogenic in mutation databases, were reclassified by us to be of unknown pathogenicity.

### Conclusions

Exome sequencing can identify asymptomatic patients at risk for MHS, although the interpretation of exome variants can be challenging. The use of exome sequencing in unselected cohorts is an important tool to understand the prevalence and penetrance of MHS, a critical challenge for the field.

## Introduction

Malignant Hyperthermia Susceptibility (MHS) is a rare disorder of calcium dysregulation triggered by volatile anesthetics and the depolarizing muscle relaxant succinylcholine. It is an important cause of morbidity and mortality, and in its fulminant form manifests nearly always as metabolic and/or respiratory acidosis, rhabdomyolysis and hyperkalemia, as well some or all of the following symptoms: tachycardia, tachypnea, arrhythmias, skeletal muscle rigidity and lethal hyperthermia. It is inherited in a predominately autosomal dominant pattern and associated with RYR1 or *CACNA1S* mutations, with other mapped loci. Seventy to 86% of patients with MHS have *RYR1* mutations^1-5^ and 1% have *CACNA1S* mutations^6^. The prevalence and penetrance of MHS mutations are difficult to determine because the pharmacologic exposure rate is low and it is an inconsistently manifesting gene-environment interaction; i.e. when a susceptible patient is exposed to a triggering agent, the probability of Malignant Hyperthermia (MH) is <100%.

Most MHS gene and variant studies have been performed on families with multiple generations affected with typical MHS. Studying these families made possible the discovery of the two implicated genes. However, these studies had ascertainment biases for those with severe reactions to the drugs. This has complicated efforts to establish the true prevalence and penetrance of MHS mutations.

In addition, assigning pathogenicity to *RYR1* and *CACNA1S* variants is challenging for several reasons. First is the issue of locus heterogeneity. With several mapped loci without identified genes, some *RYR1* and *CACNA1S* variants may have been erroneously determined to be pathogenic when there was a causative variant in another (untested) gene. In addition, *RYR1* and *CACNA1S* are large genes with 106 and 44 exons, respectively, making mutation screening challenging. Thus, some *RYR1* and *CACNA1S* variants previously determined to be pathogenic may be benign, as has been shown for other genes^7^.

New sequencing technologies, including exome sequencing (ES), have made sequencing of the human exome (exons of known genes) feasible. This provides the opportunity to detect mutations in MHS genes in a less biased manner. Using this approach, we can improve our understanding of the mutational spectra of the *RYR1* and *CACNA1S* genes, and estimate their penetrance. Our objective was to identify mutations in *RYR1* and *CACNA1S* in a population not ascertained for MHS as a pilot for the use of exome data for predictive medicine.

## Materials and Methods

To pilot the identification of MHS in an unselected population (mostly from the metropolitan Washington D.C. and Baltimore areas of the United States), we evaluated ES data from the ClinSeq® study^8^ (n=870)—a longitudinal cohort design to study the technical, medical, and genetic counseling issues associated with medical sequencing on large scale (i.e., exome or genome sequencing). The ClinSeq® study was reviewed and approved by the National Human Genome Research Institute's Institutional Review Board (Bethesda, MD) and all subjects provided informed consent to publish results and deposit sequence data in databases. Participants were 45 to 65 years of age at enrollment with a median age of 57 years. These volunteers were unselected for MHS because they were ascertained for a spectrum of coronary artery disease, which is not associated with MHS. This sample of 870 participants was 89% Caucasian, 96.3% not of Hispanic or Latino background, and 49.7% female. Family history, race, ethnicity, current medical status and clinical data were collected at enrollment, although a personal or family history of MHS was not specifically solicited. Race and ethnicity was determined by self-report on an intake questionnaire. First-degree relatives of another participant were excluded but one dyad of participants were first cousins and one dyad were first cousins once removed. During their initial visit, participants underwent an electrocardiogram, echocardiogram, and computed tomography scan for coronary calcium, clinical chemistries, and blood sample collection for genomic analysis. Sequence variants deemed clinically relevant were validated in a Clinical Laboratory Improvement Amendments-certified laboratory and the results returned to the participant.

The sequence data were generated at the National Institutes of Health's Intramural Sequencing Center. The sequencing method used solution-hybridization exome capture, performed with the *SureSelect All Exon System* (Version 1.0) by Agilent Technologies (Santa Clara, CA). The sequencing of 101 base-pair (paired-end reads) was performed with the GAIIx sequencer from Illumina, Inc. (San Diego, CA). One or two 101 base-pair, paired-end flow-cell lanes were sufficient to generate greater than 85 percent coverage of the targeted exome with high-quality variant detection^9^. Filters were applied with the VarSifter Next-Gen variation analysis software^*10^. DNA isolation, library preparation, capture, sequencing, and alignment and base calling were performed as described^11^.

*RYR1* and *CACNA1S* variants were filtered for mutation type, frequency, and information in locus-specific mutation databases (LSDBs). The complementary DNA variants and their predicted protein changes are referred to by their protein designations in the text (See variant tables, [Supplemental Digital Content 1 and Supplemental Digital Content 2](#SD1) which are tables of the *RYR1* and *CACNA1S* variants identified in this study, respectively). *RYR1* nucleotide numbering is based on transcript [NM_000540.2](https://www.ncbi.nlm.nih.gov/nuccore/NM_000540.2), and *CACNA1S* [NM_000069.2](https://www.ncbi.nlm.nih.gov/nuccore/NM_000069.2), according to the Human Genome Variation Society nomenclature^†^. Variants with low genotype quality were designated class 0; the remainder were scored 1-5 using an adaptation of published criteria^12-17^. Briefly, class 1 variants were definitely benign, class 2 probably benign, class 3 of uncertain pathogenicity, class 4 probably pathogenic, and class 5 definitely pathogenic. Further evaluation of the variants was performed using the Human Gene Mutation Database^‡18^ and the LSDB, Leiden Open Variation Database (LOVD)^§19^ and for potentially pathogenic variants, review of the medical literature ([Table 1](#T2)). We elected not to use amino acid substitution mutation analysis tools because their predictive power is variable.

### Table 1.

| DatabaseLiteratureDesignation | Novel(Not Published) |  | Published as pathogenic |  | Published as VUS | Published asbenign |
| --- | --- | --- | --- | --- | --- | --- |
| Mutation Type | MissenseIn Frame Insertion/Deletions | NonsenseFrameshiftSplice | Missense | NonsenseFrameshiftSplice | Any | Any |
| Class 5 |   | ClinSeq®/NHLBI ESP Mean Allele Frequency < 1%/0.5% |  |  |   |   |
| (Pathogenic) | Similar to disease causing mutation and consistent family history | On EMHG list of 31 approved diagnostic (causative) mutations, and/or NAMH Group's mutation panel OR Two or more reports as pathogenic and no evidence against | On EMHG list of 31 approved diagnostic (causative) mutations, and/or NAMH Group's mutation panel OR Single report as pathogenic with supporting evidence |  |  |  |
| Class 4Likely pathogenic) | Similar to disease causing mutation and inconsistent family history | Two or more reports as pathogenic with single evidence against OR Single report as pathogenic with supporting evidence | Two or more reports as pathogenic with single evidence against OR Single report as pathogenic without supporting evidence |  |  |  |
| Class 3(Uncertain) | All novel missense or in frame insertions/deletions without supporting publications | No similar disease causing mutation reported as pathogenic OR Inconsistent family history | Two or more reports as pathogenic with multiple evidence against OR Single report as pathogenic without supporting evidence | Two or more reports as pathogenic with multiple evidence against OR Single report as pathogenic with single evidence against | Reported as VUS (no convincing evidence they have a causative effect, no evidence to support polymorphism) OR single case reported as pathogenic | Single report as benign with insufficient supporting evidence |
| Class 2(Likely not pathogenic) |   |  | Single report as pathogenic with multiple evidence against |  | Some evidence to support as polymorphism OR Multiple evidence against pathogenicity | On EMHG list of 156 nonpathogenic variants, OR Multiple cases reported as benign with insufficient evidence OR multiple report as benign with supporting evidence |
| Class 1(Not pathogenic) | ClinSeq®/NHLBI ESP Mean Allele Frequency > 1%/0.5% |  |  |  |  |  |

Table 1 Caption: Variant Pathogenicity Classification System, Criteria for assignment of pathogenicity class 1 to 5 for MH gene variants RYR1 and CACNA1S variants were filtered for quality and frequency, and then assigned to pathogenicity classes based on data available in the Human Gene Mutation Database (HGMD), locus specific databases (LSDBs) and family history, as well as from the European Malignant Hyperthermia Group's (EMHG) list of diagnostic and non-pathogenic variants and the North American Malignant Hyperthermia (NAMH) mutation panel. Variants that did not pass quality filters were defined as class 0, variants that did not pass frequency filters were defined as class 1, all other variants were assessed according to the criteria presented in the table.

Medical histories of the probands and their pedigrees were reviewed for diagnoses or symptoms of MHS and related disorders. We learned retrospectively that one participant self-referred to the study because of a clinical diagnosis of MHS (subsequently found to have RYR1 p.Asp3986Glu). Clinically relevant results were returned to participants for management. For the family with a history of MH, we used standard linkage methods, typing short tandem repeat polymorphism markers and polymerase chain reaction amplification and Sanger sequencing of exons not covered by exome data.

## Results

The sequencing coverage (defined as the number of coding base-pairs with quality calls/total number of targeted base-pairs) of the coding exons was 83% (*RYR1*) and 93% (*CACNA1S*) ([Figure 1](#F1)) and there exists an inherent risk of false-negatives. Sequence coverage is dependent on many factors including DNA quality, capture efficiency, GC content, and repeat elements. Our average depth of coverage in the target region for each sample was 89×.

### Figure 1. Box and Whisker Plots.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/463f/4077354/506172434a5f/nihms522989f1.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4077354_nihms522989f1.jpg)

Box and whisker plots showing base coverage for the RYR1 and CACNA1S genes for a cohort of 870 probands. The bottom and top lines of the box represent the 1st and 3rd quartile, respectively; the mid-line represents the median value. The bottom and top whiskers represent the lowest and highest values within 1.5 times the interquartile range. Outliers have been excluded. Values on the y-axis are represented as fraction of total coding exonic bases for each gene.

One *CACNA1S* variant was a false-positive, recognized by its marginal most probable genotype score and confirmed by manual review of sequence reads. We identified 123 total variants, 70 in *RYR1* and 53 in *CACNA1S*, among 870 exomes. These variants were identified in one to 419 participants, each. Seventeen of the 122 variants (seven *RYR1* and 11 *CACNA1S*) were excluded because they were too common (see [Figure 2](#F2), and the *RYR1* and *CACNA1S* variant tables [Supplemental Digital Content 1 and Supplemental Digital Content 2](#SD1)). The National Heart, Lung, and Blood Institute's, Exome Variant Server^**^ frequency threshold was set to 0.5% as this was ∼10-fold higher than the higher end of the MHS prevalence estimate^1^ and the ClinSeq® frequency was set to 1% because it includes about 1/10 as many exomes as the Exome Variant Server and therefore chance variation could inadvertently exclude a variant. Our focus was on the identification of highly penetrant alleles that cause autosomal dominant MHS, though we did detect some recessive myopathy alleles.

### Figure 2. Quality/Frequency Filter Algorithm.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/463f/4077354/413d8bba8f33/nihms522989f2.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4077354_nihms522989f2.jpg)

Filtering criteria used for coding-variant interpretation. Variants were filtered on genotype quality, coverage and allele frequencies. Variants removed by quality filters were classified as 0 and frequency filters as Class 1, with the remaining assessed for pathogenicity (Class 2-5) based on data present in the Human Gene Mutation Database (HGMD) and locus-specific databases (LSDBs). MPG =most probable genotype. MAF =minor allele frequency. NHLBI ESP =The National Heart, Lung, and Blood Institute, exome sequencing project.

The remaining 104 variants (63 in *RYR1* and 41 in *CACNA1S*) were considered rare variants. Seventeen of the 63 *RYR1* rare variants were listed in the HGMD as “disease-causing” for either MHS, central core disease, multi-minicore disease, atypical periodic paralysis, or congenital myopathy. Three of the 63 *RYR1* variants were not present in HGMD but were listed in the LSDB as pathogenic. One of the 41 CACNA1S variants (p.Thr1354Ser) was listed in HGMD as pathogenic for MHS, and one (p.Arg498His) was listed in the LSDB as pathogenic but without any supporting evidence. Of the 20 *RYR1* variants (present in HGMD or LSDBs, and with an allele frequency <1%), only four met our criteria ([Table 1](#T2)) for class 5 pathogenicity; the remaining 16 were scored as a 3 (variants of unknown significance) or class 2 (likely not pathogenic).

Four class 5 *RYR1* variants were identified in 870 exomes. The p.Arg614Cys variant was found in one participant and listed in HGMD as pathogenic based on three publications^20-22^ and reported 37 times in LOVD. All of the submitting authors of these entries had concluded that it was pathogenic. This p.Arg614Cys variant is one of the 31 RYR1 mutations on the European Malignant Hyperthermia Group^††^ list of pathogenic mutations and is also included in the 2002 North American MH consensus list of 17 causative mutations^23^. We designated this mutation as class 5, pathogenic. It is interesting to note that the 62 year-old female participant with this variant had no family or personal history of MHS, despite having surgery with general anesthesia thrice.

The second class 5 RYR1 pathogenic variant, p.Arg2241X, was detected in two participants. It was described as pathogenic in HGMD, based on a single patient with congenital myopathy, episodes of generalized, atypical normokalemic paralysis, and multi-minicore disease with external ophthalmoplegia and episodes of atypical periodic paralysis^24^. The molecular data in this published report were complex. The patient had, in addition to p.Arg2241X, p.Asp708Asn in *cis* and p.Arg2939Lys in *trans* to p.Arg2241X with apparent nonsense-mediated messenger RNA decay of the p.Arg2241X-bearing allele. In another study of 37 patients with dominant or recessive *RYR1*-related myopathies, the p.Arg2241X variant was described in three patients with recessive myopathies and ophthalmoparesis^25^. In two siblings, seven and five years old, the p.Arg2241X variant co-occurred with the previously described putatively pathogenic variant p.Arg109Trp^26,27^, and in a third patient the p.Arg2241X variant co-occurred with two missense variants, the putatively pathogenic p.Arg2939Lys^27^ and p.Asp708Asn (these three variants were likely from the same patient reported in two case series by this same group).^24,27^ The RYR1 variant p.Arg2241X was also categorized as a pathogenic recessive mutation in a patient with a congenital myopathy and muscle biopsy finding of an *RYR1*-related myopathy from a study of 71 families with *RYR1* mutations^28^. The patient had two additional recessive pathogenic variants, p.Asp708Asn and p.Met485Val, and a synonymous variant of unknown significance c.11547G>A (p(=)). The p.Arg2241X variant was not detected in the Exome Variant Server. We categorized p.Arg2241X as class 5, since it was described in affected patients and of the category of variants (nonsense) strongly predictive of an autosomal recessive *RYR1*-related myopathy.

The third class 5 *RYR1* variant, c.5183C>T; p.Ser1728Phe, was listed in HGMD with references to two studies as pathogenic^5,22^. We found this variant in a 47-year-old (Irish/British ancestry) female (1/1,740 alleles) without a personal or family history of MHS. The p.Ser1728Phe variant was reported in three independent families from an analysis of the United Kingdom MH patients^5^. In a subsequent genotype-phenotype correlation study, the p.Ser1728Phe variant was found in seven individuals and two families—six with a weaker *in vitro* contraction test phenotype compared to the known pathogenic p.Gly2434Arg mutation, suggesting a lesser effect on channel function as compared to their control^22^. Since the rare p.Ser1728Phe variant (1/10,757 alleles in the Exome Variant Server) was reported multiple times as pathogenic, with no evidence against, it was scored as a class 5, pathogenic variant.

The fourth class 5 *RYR1* variant, c.11958C>G; p.Asp3986Glu, was listed in HGMD with references to the same two United Kingdom studies cited above.^5,22^. The variant was seen in five MH patients with MH disease status and associated with more severe static caffeine contractures and higher creatine kinase levels than the p.Gly2434Arg control or other variants. It was also identified in one 45 year-old (Irish/German Ancestry) male, ClinSeq® volunteer (1/1,740 alleles) with a history of MH. The volunteer had a history of multiple fulminant MH events—symptoms of myopathy, myotonia (dysphagia), proximal muscle weakness, and a positive *in vitro* contracture test and a serum creatine kinase value of 1,271 U/L and lactate dehydrogenase level of 238 U/L (See participant description table, [Supplementary Digital Content 3](#SD1) which is a table containing the characteristics of the ClinSeq® volunteers with *RYR1* variants). In addition, he had a family history of myotonia and positive *in vitro* contracture test in three siblings. This rare variant from *RYR1* MHS mutational hotspot region III was *not* found in over 6,500 human exomes in Exome Variant Server. Reported multiple times as pathogenic with no evidence against, we concluded this is a Class 5, pathogenic mutation.

Of the 20 rare *RYR1* variants (17 in HGMD & the LSDBs, three in LSDBs only) that were identified in ClinSeq®–ten were assigned to class 3, and six to class 2, based on the criteria in the pathogenicity table. The reasoning for these assignments is described in the [Supplemental Digital Content 4, Supplemental Methods](#SD1), which describes the variants—of less than class 5 pathogenicity—identified in ClinSeq® and in databases. The family, personal, medical and surgical histories of all participants with *RYR1* and *CACNA1S* variants were reviewed; all but two were negative for MHS (See participant description tables, [Supplementary Digital Content 3 and Supplementary Digital Content 5](#SD1), containing the characteristics of the ClinSeq® volunteers with *RYR1* and *CACNA1S* variants, respectively).

One participant was found to have a novel RYR1 missense variant p.Arg3498Gly and a three-generation family history of MHS with an *in vitro* contracture test diagnostic for MHS^29^. To assess the potential pathogenicity of this variant, we performed a segregation analysis of the variant in the family. The variant did not segregate with the MHS phenotype (See [Supplemental Digital Content 6](#SD1), a pedigree of the malignant hyperthermia family). We ruled out an error in phenotyping, after acquiring muscle biopsy and caffeine halothane contracture test results for seven of the family members from The North American Malignant Hyperthermia Registry^‡‡^. We next performed a candidate linkage analysis of the *RYR1* locus. Genotyping and manual haplotyping showed that a *RYR1* haplotype cosegregated with the phenotype, but this haplotype was in *trans* to p.Arg3498Gly. We concluded that p.Arg3498Gly was not pathogenic and hypothesized that this family most likely had MHS attributable to an undetected *RYR1* variant in *trans* to p.Arg3498Gly in the proband. We next evaluated the exon coverage of *RYR1* in this proband and found that he had 91.9% sequence coverage. We Sanger sequenced exons with poor exome sequence read-depth but found no mutations. We concluded that the exome sequencing of *RYR1* generated both a false-negative and a false-positive result in that the p.Arg3498Gly is not pathogenic and the participant likely has a mutation in *RYR1* not captured by ES or Sanger sequencing.

One of the 41 *CACNA1S* rare variants, p.Arg498His, identified in one exome, was listed in LOVD as pathogenic (it was not listed in HGMD). However, the pathogenicity of this entry was not supported by the primary literature, nor did LOVD provide details of the *CACNA1S*-associated phenotype. We contacted the LOVD curators and learned that the variant had been recategorized as ‘unknown pathogenicity’, although the database itself had not been updated. We therefore categorized it as a variant of uncertain significance (score 3).

The *CACNA1S* variant p.Thr1354Ser was identified in 9/870 ClinSeq® exomes (mean allele frequency 0.7%) and in the Exome Variant Server with an allele count of 48/12,958 (mean allele frequency 0.4%). HGMD listed this variant as pathogenic, citing a publication showing segregation of p.Thr1354Ser in one family, its absence in 282 controls, and functional data demonstrating abnormal Ca^++^ flux^30^. However, we concluded that this was more likely a benign variant in linkage disequilibrium with the (undetected) true pathogenic variant in the family described by Pirone and colleagues.^30^ Of the remaining 39 *CACNA1S* rare variants, none of these were present in either HGMD or the LSDBs. These variants were also assigned to class 3. None of these patients had a personal or family history of MHS (See [Supplemental Digital Content 5](#SD1) with the characteristics of the ClinSeq® volunteers with *CACNA1S* variants).

## Discussion

Four examples of both the power and the limitations of ES for studying MHS were identified in this study. First, we detected a causative (class 5) RYR1 mutation, p.Arg614Cys, in a proband who had no clinical/phenotypic evidence of MHS and a negative family history (See participant description table, [Supplemental Digital Content 3](#SD1)). The p.Arg614Cys variant was included in both the North American MHS and the European Malignant Hyperthermia Group causative mutation lists. We conclude that this represents a presymptomatic diagnosis of MHS in this participant, which is an example of the predictive, personalized genomic medicine in practice. We confirmed this variant in a clinical testing lab, returned it to the participant with medical and genetic counseling, and referred her for consideration for caffeine halothane contracture test testing and enrollment in the Malignant Hyperthermia Association of the United States registry. Until such testing is performed on the patient or she has a reaction to a triggering agent, we cannot claim to have proven she has MHS. However, because this variant is listed in both the North American MHS and the European Malignant Hyperthermia Group causative mutation lists, we believe that it is extremely unlikely that this variant is benign solely because it was ascertained in context of this study design. Second, the p.Thr1354Ser *CACNA1S* variant, previously assumed to be pathogenic, was deemed likely to be class 3 i.e., of uncertain pathogenicity. The frequency of this single variant was ∼20 times higher than the frequency of MHS attributed to all loci and all mutations (0.74-1% p.Thr1354Ser heterozygotes). Although there are good functional data implicating this variant^30^ in MHS, we believe that the population genetic data mandate that it should be scored class 3, of unknown pathogenicity. Our findings, supported by the Exome Variant Server *CACNA1S* allele frequencies, suggest that other previously implicated MHS variants may be benign. Caution is warranted regarding variants claimed to be causative for MHS, especially when used for predictive individualized medicine. Third, we found a novel RYR1 p.Arg3498Gly variant that was *not* pathogenic in an individual positive for MH by the caffeine halothane contracture test and a family history of MHS. The variant was rare but did not segregate with the phenotype and this family most likely had MHS attributable to an undetected *RYR1* variant, or, less likely, a variant at another locus. We suggest that other previously reported rare *RYR1* variants without robust genetic data may have been misclassified as pathogenic. Fourth, we identified the class 5 variant, p.Arg2241X, which has been associated with phenotypes inherited in a recessive pattern, but recent publications have questioned the pathogenicity of this variant^15,24-26,28^. The risk of MHS in most recessive myopathies is uncertain, and has only been proven for central core disease^31^. This example shows that even when one can identify pathogenic variants, it can be challenging to associate them unequivocally with specific phenotypes.

Using ES, we identified 123 distinct variants (70 *RYR1* and 53 *CACNA1S*) among 870 participants ([Figures 3](#F3) and [4](#F4)). Our analyses yielded a spectrum of pathogenicity scores from benign to pathogenic ([Figures 5](#F5) and [6](#F6)). All but two of the *RYR1* variants classified as “disease causing mutations” in HGMD were reclassified by us as benign, probably benign, or variant of unknown significance, scores 1-3. We reclassified these variants based on the criteria in [Table 1](#T2), under the assumption that a variant was benign, unless a critical review of the data supported a higher pathogenicity category. It is critical to recognize that our assessment of ‘benign’ or ‘probably benign’ is limited to the specific context of using such a variant for individualized predictive medicine and that it is certainly not our intention for it to be interpreted to mean that the variant has no role in the pathogenicity of MHS, myopathy, or other phenotypes. In addition, more than half of the *RYR1* variants (43/69, 62%) we identified were not listed by HGMD or the LSDB databases, or in biomedical literature citations. Because we screened a cohort unselected for MHS, we predicted that most of the novel variants would be benign. More than half (40/69) of the *RYR1* variants were rare and *not* found in the Exome Variant Server. A fifth (10/51) of the *CACNA1S* variants were common polymorphisms, which we assigned to class 1 (benign), with the remaining assigned to class 3 (unknown) ([Figure 6](#F6)). Four individuals (three males, one female) had more than one *RYR1* variant and two of the four participants with two *RYR1* variants had benign *CACNA1S* variants as well (See [Supplemental Digital Content 3 and Supplemental Digital Content 5](#SD1) containing the characteristics of the ClinSeq® volunteers with *RYR1* and *CACNA1S* variants).

### Figure 3. Frequency Histogram of RYR1 Variants.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/463f/4077354/487be1c0e774/nihms522989f3.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4077354_nihms522989f3.jpg)

Frequency histogram of the 69 RYR1 variants with predicted protein changes from the ClinSeq® 870 cohort. The three RYR1 hotspot regions (Region 1/N-terminal, Region 2/central and Region 3/C-terminal) are emphasized for purposes of orientation. Blue arrows in the figure point to variants referenced in the text. (ClinSeq® trademark held by National Institutes of Health, Bethesda, Maryland.)

### Figure 4. Frequency Histogram of CACNA1S Variants.

![Figure 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/463f/4077354/54a9a92da6da/nihms522989f4.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4077354_nihms522989f4.jpg)

Frequency histogram of 51 CACNA1S variants with predicted protein changes from the ClinSeq® 870 cohort. Blue arrows in the figure point to variants referenced in the text. (ClinSeq® trademark held by National Institutes of Health, Bethesda, Maryland.)

### Figure 5. Percentage of ClinSeq® RYR1 Variants, Class 1-5.

![Figure 5](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/463f/4077354/69eb6b7ec038/nihms522989f5.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4077354_nihms522989f5.jpg)

Proportion of 69 RYR1 variants from the ClinSeq® 870 cohort in pathogenicity class 1 through 5. (ClinSeq® trademark held by National Institutes of Health, Bethesda, Maryland.)

### Figure 6. Percentage of ClinSeq® CACNA1S Variants, Class 1-5.

![Figure 6](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/463f/4077354/0104f183d049/nihms522989f6.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4077354_nihms522989f6.jpg)

Proportion of 51 CACNA1S variants from the ClinSeq® 870 cohort in pathogenicity classes 1 through 5. (ClinSeq® trademark held by National Institutes of Health, Bethesda, Maryland.)

The purpose of this study was to identify high penetrance variants associated with MHS. As noted above, that we conclude a variant is class 1-3 does not automatically mean that the variant has no physiological effects. Moreover, the data did not allow us to evaluate whether interactions could have occurred among variants in a given individual,^5^ although this should be specifically addressed in future studies. We deliberately set our threshold for pathogenicity high to avoid the error of wrongly diagnosing an individual as susceptible in an ascertainment mode where the prior probability that they are affected was low. The risk of false-negatives in exome sequencing will diminish as future ES and follow-up studies generate additional data.

The filtering process for analysis of MHS variants from ES requires a manual method of evaluating variants to extract meaningful information. We used allele frequency, genotype-phenotype databases and the primary literature to identify pathogenic variants. Unfortunately, there is at present no single information source that allows one to reliably ascertain if a variant is benign or pathogenic. Many sequence databases (e.g., the Exome Variant Server and The Single Nucleotide Polymorphism Database) include pathogenic, potentially pathogenic and non-pathogenic variants and do not include phenotype data. Further, there is often no indication as to whether some individuals harbor multiple variants within a single gene, which limited our ability to evaluate these data. Our evaluation of 870 exomes using HGMD and LSDBs indicated likely significant levels of misclassification and variability in the pathogenicity determination not only in HGMD and the LSDBs, which is primarily attributable to the source literature.

ES has some limitations: the method can miss pathogenic variants such as structural variation, or copy-number variants, in the genome— larger insertions and deletions, duplications and inversions. Although the technology has improved target coverage over the years, it will most likely never reach 100%. In view of the distribution of variants and the complexity of the genome, ES remains an efficient way to identify most mutations altering protein sequence in any single DNA sample. However, to our knowledge, the only genomic variants so far associated with MH are missense variants in coding exons, so most of these limitations do not pertain, given our current knowledge of the disorder.

The published prevalence of MHS mutations varies widely from 1 in 2,000^1,32^ to 1/10,000^4^ but the penetrance has been difficult to determine. Our study of 870 exomes, although it represents a prodigious amount of data, is still too small to estimate the prevalence of MHS. The ES of patients not ascertained for a personal or family history of MHS allows, in principle, an unbiased approach to genotype-phenotype correlation that has not been feasible with previous technologies. We conclude that some *RYR1* and *CACNA1S* variants may have been misclassified as pathogenic without adequate genetic (e.g., cosegregation) or functional data. It is important to stress that in addition to robust genetic analysis, there is a critical need for a robust and non-invasive functional test for MHS, which together with genetic data could allow accurate determination of the prevalence and penetrance of this trait. Presently, ES cannot replace clinical investigations, but rather assists clinicians in determining which patients should undergo further genetic and/or functional analyses. This approach to variant identification in MHS should be extended to other cohorts undergoing ES, and may be useful as a first screening approach, before more invasive and time-consuming investigations. Analysis of thousands of exomes has the potential to provide the MHS field with an exhaustive catalog of variants to determine the true prevalence, penetrance, and expressivity of this life-threatening disorder. While the assessment of the pathogenicity of both known and novel variants remains challenging, we demonstrate that causative mutations can be identified from ES data. These data suggest that clinically relevant mutations can be identified as incidental findings in exomes sequenced for clinical care and clinical research. This should inform the debate on the return of such secondary results to research participants. Further, the application of ES technology to large and diverse cohorts has the potential to accelerate the pace of MHS gene mutation discovery. We speculate that the results of these studies will allow the development of clinical genomic screening for MHS, which should reduce the incidence of life-threatening events and increase life expectancy for those individuals who harbor pathogenic variants in these genes.

## Supplementary Material

### What we already know about this topic

### What we already know about this topic

### What we already know about this topic

### What we already know about this topic

### What this article tells us that is new

## Acknowledgments

The authors thank the staff at the NIH Intramural Sequencing Center, NIH Clinical Center and the ClinSeq® study participants. Caitlin Joy Krause, BS provided technical support and Flavia M. Facio, MS, CGC provided clinical support. Dr. Sheila Muldoon, MD, Professor, Department of Anesthesiology of the Uniformed Services University of the Health Sciences provided advice and support.

**Funding Disclosure**: This study was funded by the Division of Intramural Research of the National Human Genome Research Institute, National Institutes of Health, Bethesda, MD.

## Appendix. The NIH Intramural Sequencing Center Group

5625 Fishers Ln

Rockville, MD 20892-9400

| Center |   |
| --- | --- |
| Director | Jim Mullikin, Ph.D. |
| Deputy Director | Jim Thomas, Ph.D. |
| Sequencing Group |   |
| Group Director | Robert Blakesley, Ph.D. |
| Group Deputy Director | Alice Young, B.A. |
| Robotic Specialist | Sean Lovett, B.Sc. |
| Library Construction |   |
| Co-Leader | Joel Han, B.Sc. |
| Co-Leader | Richelle Legaspi, M.Sc. |
| Library Technician | Christina Sison, B.Sc. |
| Library Technician | Casandra Montemayor, M.Sc. |
| Sequence Production |   |
| Leader | Michael Gregory, M.Sc. |
| Production Technician | April Hargrove, B.Sc. |
| Production Technician | Taccara Johnson, B.Sc. |
| Production Technician | Nancy Riebow, B.Sc. |
| NextGen Production |   |
| Leader | Brian Schmidt, B.Sc. |
| Sequence Finishing |   |
| Leader | Jyoti Gupta, M.Sc. |
| Finishing Technician | Betty Benjamin, B.Sc. |
| Finishing Technician | Shelise Brooks, B.Sc. |
| Finishing Technician | Holly Coleman, M.Sc. |
| Finishing Technician | Shi-ling Ho, B.Sc. |
| Finishing Technician | Karen Schandler, M.Sc. |
| Finishing Technician | Mal Stantripop, B.Sc. |
| Instrumentation |   |
| Leader | Quino Maduro, B.Sc. |
| Bioinformatics Group |   |
| Group Director | Dr. Gerry Bouffard, Ph.D. |
| Staff Bioinformatician | Mila Dekhtyar, M.Sc. |
| Staff Bioinformatician | Dr. Xiaobin Guan, Ph.D. |
| Staff Bioinformatician | Cathy Masiello, M.Sc. |
| Staff Bioinformatician | Baishali Maskeri, Ph.D. |
| Staff Bioinformatician | Jenny McDowell, Ph.D. |
| Staff Bioinformatician | Morgan Park, Ph.D. |
| Staff Bioinformatician | Meg Vemulapalli, M.Sc. |
## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Ibarra MC, Wu S, Murayama K, Minami N, Ichihara Y, Kikuchi H, Noguchi S, Hayashi YK, Ochiai R, Nishino I. Malignant hyperthermia in Japan: Mutation screening of the entire ryanodine receptor type 1 gene coding region by direct sequencing. Anesthesiology. 2006;104:1146–54. doi: 10.1097/00000542-200606000-00008.  [DOI](https://doi.org/10.1097/00000542-200606000-00008) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16732084/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Anesthesiology&title=Malignant%20hyperthermia%20in%20Japan:%20Mutation%20screening%20of%20the%20entire%20ryanodine%20receptor%20type%201%20gene%20coding%20region%20by%20direct%20sequencing&author=MC%20Ibarra&author=S%20Wu&author=K%20Murayama&author=N%20Minami&author=Y%20Ichihara&volume=104&publication_year=2006&pages=1146-54&pmid=16732084&doi=10.1097/00000542-200606000-00008&)

2. Sambuughin N, Holley H, Muldoon S, Brandom BW, de Bantel AM, Tobin JR, Nelson TE, Goldfarb LG. Screening of the entire ryanodine receptor type 1 coding region for sequence variants associated with malignant hyperthermia susceptibility in the north american population. Anesthesiology. 2005;102:515–21. doi: 10.1097/00000542-200503000-00007.  [DOI](https://doi.org/10.1097/00000542-200503000-00007) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15731587/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Anesthesiology&title=Screening%20of%20the%20entire%20ryanodine%20receptor%20type%201%20coding%20region%20for%20sequence%20variants%20associated%20with%20malignant%20hyperthermia%20susceptibility%20in%20the%20north%20american%20population&author=N%20Sambuughin&author=H%20Holley&author=S%20Muldoon&author=BW%20Brandom&author=AM%20de%20Bantel&volume=102&publication_year=2005&pages=515-21&pmid=15731587&doi=10.1097/00000542-200503000-00007&)

3. Galli L, Orrico A, Lorenzini S, Censini S, Falciani M, Covacci A, Tegazzin V, Sorrentino V. Frequency and localization of mutations in the 106 exons of the RYR1 gene in 50 individuals with malignant hyperthermia. Hum Mutat. 2006;27:830. doi: 10.1002/humu.9442.  [DOI](https://doi.org/10.1002/humu.9442) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16835904/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mutat&title=Frequency%20and%20localization%20of%20mutations%20in%20the%20106%20exons%20of%20the%20RYR1%20gene%20in%2050%20individuals%20with%20malignant%20hyperthermia&author=L%20Galli&author=A%20Orrico&author=S%20Lorenzini&author=S%20Censini&author=M%20Falciani&volume=27&publication_year=2006&pages=830&pmid=16835904&doi=10.1002/humu.9442&)

4. Monnier N, Krivosic-Horber R, Payen JF, Kozak-Ribbens G, Nivoche Y, Adnet P, Reyford H, Lunardi J. Presence of two different genetic traits in malignant hyperthermia families: Implication for genetic analysis, diagnosis, and incidence of malignant hyperthermia susceptibility. Anesthesiology. 2002;97:1067–74. doi: 10.1097/00000542-200211000-00007.  [DOI](https://doi.org/10.1097/00000542-200211000-00007) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12411788/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Anesthesiology&title=Presence%20of%20two%20different%20genetic%20traits%20in%20malignant%20hyperthermia%20families:%20Implication%20for%20genetic%20analysis,%20diagnosis,%20and%20incidence%20of%20malignant%20hyperthermia%20susceptibility&author=N%20Monnier&author=R%20Krivosic-Horber&author=JF%20Payen&author=G%20Kozak-Ribbens&author=Y%20Nivoche&volume=97&publication_year=2002&pages=1067-74&pmid=12411788&doi=10.1097/00000542-200211000-00007&)

5. Robinson R, Carpenter D, Shaw MA, Halsall J, Hopkins P. Mutations in RYR1 in malignant hyperthermia and central core disease. Hum Mutat. 2006;27:977–89. doi: 10.1002/humu.20356.  [DOI](https://doi.org/10.1002/humu.20356) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16917943/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mutat&title=Mutations%20in%20RYR1%20in%20malignant%20hyperthermia%20and%20central%20core%20disease&author=R%20Robinson&author=D%20Carpenter&author=MA%20Shaw&author=J%20Halsall&author=P%20Hopkins&volume=27&publication_year=2006&pages=977-89&pmid=16917943&doi=10.1002/humu.20356&)

6. Stewart SL, Hogan K, Rosenberg H, Fletcher JE. Identification of the Arg1086His mutation in the alpha subunit of the voltage-dependent calcium channel (CACNA1S) in a North American family with malignant hyperthermia. Clin Genet. 2001;59:178–84. doi: 10.1034/j.1399-0004.2001.590306.x.  [DOI](https://doi.org/10.1034/j.1399-0004.2001.590306.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11260227/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Genet&title=Identification%20of%20the%20Arg1086His%20mutation%20in%20the%20alpha%20subunit%20of%20the%20voltage-dependent%20calcium%20channel%20(CACNA1S)%20in%20a%20North%20American%20family%20with%20malignant%20hyperthermia&author=SL%20Stewart&author=K%20Hogan&author=H%20Rosenberg&author=JE%20Fletcher&volume=59&publication_year=2001&pages=178-84&pmid=11260227&doi=10.1034/j.1399-0004.2001.590306.x&)

7. Bell CJ, Dinwiddie DL, Miller NA, Hateley SL, Ganusova EE, Mudge J, Langley RJ, Zhang L, Lee CC, Schilkey FD, Sheth V, Woodward JE, Peckham HE, Schroth GP, Kim RW, Kingsmore SF. Carrier testing for severe childhood recessive diseases by next-generation sequencing. Sci Transl Med. 2011;3:65ra4. doi: 10.1126/scitranslmed.3001756.  [DOI](https://doi.org/10.1126/scitranslmed.3001756) | [PMC free article](/articles/PMC3740116/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21228398/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Sci%20Transl%20Med&title=Carrier%20testing%20for%20severe%20childhood%20recessive%20diseases%20by%20next-generation%20sequencing&author=CJ%20Bell&author=DL%20Dinwiddie&author=NA%20Miller&author=SL%20Hateley&author=EE%20Ganusova&volume=3&publication_year=2011&pages=65ra4&pmid=21228398&doi=10.1126/scitranslmed.3001756&)

8. Biesecker LG, Mullikin JC, Facio FM, Turner C, Cherukuri PF, Blakesley RW, Bouffard GG, Chines PS, Cruz P, Hansen NF, Teer JK, Maskeri B, Young AC, Manolio TA, Wilson AF, Finkel T, Hwang P, Arai A, Remaley AT, Sachdev V, Shamburek R, Cannon RO, Green ED. The ClinSeq Project: Piloting large-scale genome sequencing for research in genomic medicine. Genome Res. 2009;19:1665–74. doi: 10.1101/gr.092841.109.  [DOI](https://doi.org/10.1101/gr.092841.109) | [PMC free article](/articles/PMC2752125/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19602640/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Res&title=The%20ClinSeq%20Project:%20Piloting%20large-scale%20genome%20sequencing%20for%20research%20in%20genomic%20medicine&author=LG%20Biesecker&author=JC%20Mullikin&author=FM%20Facio&author=C%20Turner&author=PF%20Cherukuri&volume=19&publication_year=2009&pages=1665-74&pmid=19602640&doi=10.1101/gr.092841.109&)

9. Teer JK, Bonnycastle LL, Chines PS, Hansen NF, Aoyama N, Swift AJ, Abaan HO, Albert TJ, Margulies EH, Green ED, Collins FS, Mullikin JC, Biesecker LG. Systematic comparison of three genomic enrichment methods for massively parallel DNA sequencing. Genome Res. 2010;20:1420–31. doi: 10.1101/gr.106716.110.  [DOI](https://doi.org/10.1101/gr.106716.110) | [PMC free article](/articles/PMC2945191/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20810667/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Res&title=Systematic%20comparison%20of%20three%20genomic%20enrichment%20methods%20for%20massively%20parallel%20DNA%20sequencing&author=JK%20Teer&author=LL%20Bonnycastle&author=PS%20Chines&author=NF%20Hansen&author=N%20Aoyama&volume=20&publication_year=2010&pages=1420-31&pmid=20810667&doi=10.1101/gr.106716.110&)

10. Teer JK, Green ED, Mullikin JC, Biesecker LG. VarSifter: Visualizing and analyzing exome-scale sequence variation data on a desktop computer. Bioinformatics. 2012;28:599–600. doi: 10.1093/bioinformatics/btr711.  [DOI](https://doi.org/10.1093/bioinformatics/btr711) | [PMC free article](/articles/PMC3278764/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22210868/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=VarSifter:%20Visualizing%20and%20analyzing%20exome-scale%20sequence%20variation%20data%20on%20a%20desktop%20computer&author=JK%20Teer&author=ED%20Green&author=JC%20Mullikin&author=LG%20Biesecker&volume=28&publication_year=2012&pages=599-600&pmid=22210868&doi=10.1093/bioinformatics/btr711&)

11. Johnston JJ, Rubinstein WS, Facio FM, Ng D, Singh LN, Teer JK, Mullikin JC, Biesecker LG. Secondary variants in individuals undergoing exome sequencing: Screening of 572 individuals identifies high-penetrance mutations in cancer-susceptibility genes. Am J Hum Genet. 2012;91:97–108. doi: 10.1016/j.ajhg.2012.05.021.  [DOI](https://doi.org/10.1016/j.ajhg.2012.05.021) | [PMC free article](/articles/PMC3397257/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22703879/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=Secondary%20variants%20in%20individuals%20undergoing%20exome%20sequencing:%20Screening%20of%20572%20individuals%20identifies%20high-penetrance%20mutations%20in%20cancer-susceptibility%20genes&author=JJ%20Johnston&author=WS%20Rubinstein&author=FM%20Facio&author=D%20Ng&author=LN%20Singh&volume=91&publication_year=2012&pages=97-108&pmid=22703879&doi=10.1016/j.ajhg.2012.05.021&)

12. Plon SE, Eccles DM, Easton D, Foulkes WD, Genuardi M, Greenblatt MS, Hogervorst FB, Hoogerbrugge N, Spurdle AB, Tavtigian SV. Sequence variant classification and reporting: Recommendations for improving the interpretation of cancer susceptibility genetic test results. Hum Mutat. 2008;29:1282–91. doi: 10.1002/humu.20880.  [DOI](https://doi.org/10.1002/humu.20880) | [PMC free article](/articles/PMC3075918/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18951446/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mutat&title=Sequence%20variant%20classification%20and%20reporting:%20Recommendations%20for%20improving%20the%20interpretation%20of%20cancer%20susceptibility%20genetic%20test%20results&author=SE%20Plon&author=DM%20Eccles&author=D%20Easton&author=WD%20Foulkes&author=M%20Genuardi&volume=29&publication_year=2008&pages=1282-91&pmid=18951446&doi=10.1002/humu.20880&)

13. Akbari MR, Zhang S, Fan I, Royer R, Li S, Risch H, McLaughlin J, Rosen B, Sun P, Narod SA. Clinical impact of unclassified variants of the BRCA1 and BRCA2 genes. J Med Genet. 2011;48:783–6. doi: 10.1136/jmedgenet-2011-100305.  [DOI](https://doi.org/10.1136/jmedgenet-2011-100305) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21965345/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Med%20Genet&title=Clinical%20impact%20of%20unclassified%20variants%20of%20the%20BRCA1%20and%20BRCA2%20genes&author=MR%20Akbari&author=S%20Zhang&author=I%20Fan&author=R%20Royer&author=S%20Li&volume=48&publication_year=2011&pages=783-6&pmid=21965345&doi=10.1136/jmedgenet-2011-100305&)

14. Bergman JE, Janssen N, van der Sloot AM, de Walle HE, Schoots J, Rendtorff ND, Tranebjaerg L, Hoefsloot LH, van Ravenswaaij-Arts CM, Hofstra RM. A novel classification system to predict the pathogenic effects of CHD7 missense variants in CHARGE syndrome. Hum Mutat. 2012;33:1251–60. doi: 10.1002/humu.22106.  [DOI](https://doi.org/10.1002/humu.22106) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22539353/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mutat&title=A%20novel%20classification%20system%20to%20predict%20the%20pathogenic%20effects%20of%20CHD7%20missense%20variants%20in%20CHARGE%20syndrome&author=JE%20Bergman&author=N%20Janssen&author=AM%20van%20der%20Sloot&author=HE%20de%20Walle&author=J%20Schoots&volume=33&publication_year=2012&pages=1251-60&pmid=22539353&doi=10.1002/humu.22106&)

15. Greenblatt MS, Brody LC, Foulkes WD, Genuardi M, Hofstra RM, Olivier M, Plon SE, Sijmons RH, Sinilnikova O, Spurdle AB. Locus-specific databases and recommendations to strengthen their contribution to the classification of variants in cancer susceptibility genes. Hum Mutat. 2008;29:1273–81. doi: 10.1002/humu.20889.  [DOI](https://doi.org/10.1002/humu.20889) | [PMC free article](/articles/PMC3446852/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18951438/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mutat&title=Locus-specific%20databases%20and%20recommendations%20to%20strengthen%20their%20contribution%20to%20the%20classification%20of%20variants%20in%20cancer%20susceptibility%20genes&author=MS%20Greenblatt&author=LC%20Brody&author=WD%20Foulkes&author=M%20Genuardi&author=RM%20Hofstra&volume=29&publication_year=2008&pages=1273-81&pmid=18951438&doi=10.1002/humu.20889&)

16. Jordan DM, Kiezun A, Baxter SM, Agarwala V, Green RC, Murray MF, Pugh T, Lebo MS, Rehm HL, Funke BH, Sunyaev SR. Development and validation of a computational method for assessment of missense variants in hypertrophic cardiomyopathy. Am J Hum Genet. 2011;88:183–92. doi: 10.1016/j.ajhg.2011.01.011.  [DOI](https://doi.org/10.1016/j.ajhg.2011.01.011) | [PMC free article](/articles/PMC3035712/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21310275/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=Development%20and%20validation%20of%20a%20computational%20method%20for%20assessment%20of%20missense%20variants%20in%20hypertrophic%20cardiomyopathy&author=DM%20Jordan&author=A%20Kiezun&author=SM%20Baxter&author=V%20Agarwala&author=RC%20Green&volume=88&publication_year=2011&pages=183-92&pmid=21310275&doi=10.1016/j.ajhg.2011.01.011&)

17. Thompson BA, Greenblatt MS, Vallee MP, Herkert JC, Tessereau C, Young EL, Adzhubey IA, Li B, Bell R, Feng B, Mooney SD, Radivojac P, Sunyaev SR, Frebourg T, Hofstra RM, Sijmons RH, Boucher K, Thomas A, Goldgar DE, Spurdle AB, Tavtigian SV. Calibration of multiple in silico tools for predicting pathogenicity of mismatch repair gene missense substitutions. Hum Mutat. 2013;34:255–65. doi: 10.1002/humu.22214.  [DOI](https://doi.org/10.1002/humu.22214) | [PMC free article](/articles/PMC4318556/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22949387/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mutat&title=Calibration%20of%20multiple%20in%20silico%20tools%20for%20predicting%20pathogenicity%20of%20mismatch%20repair%20gene%20missense%20substitutions&author=BA%20Thompson&author=MS%20Greenblatt&author=MP%20Vallee&author=JC%20Herkert&author=C%20Tessereau&volume=34&publication_year=2013&pages=255-65&pmid=22949387&doi=10.1002/humu.22214&)

18. Stenson PD, Ball EV, Mort M, Phillips AD, Shiel JA, Thomas NS, Abeysinghe S, Krawczak M, Cooper DN. Human Gene Mutation Database (HGMD): 2003 update. Hum Mutat. 2003;21:577–81. doi: 10.1002/humu.10212.  [DOI](https://doi.org/10.1002/humu.10212) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12754702/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mutat&title=Human%20Gene%20Mutation%20Database%20(HGMD):%202003%20update&author=PD%20Stenson&author=EV%20Ball&author=M%20Mort&author=AD%20Phillips&author=JA%20Shiel&volume=21&publication_year=2003&pages=577-81&pmid=12754702&doi=10.1002/humu.10212&)

19. Fokkema IF, Taschner PE, Schaafsma GC, Celli J, Laros JF, den Dunnen JT. LOVD v.2.0: the next generation in gene variant databases. Hum Mutat. 2011;32:557–63. doi: 10.1002/humu.21438.  [DOI](https://doi.org/10.1002/humu.21438) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21520333/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mutat&title=LOVD%20v.2.0:%20the%20next%20generation%20in%20gene%20variant%20databases&author=IF%20Fokkema&author=PE%20Taschner&author=GC%20Schaafsma&author=J%20Celli&author=JF%20Laros&volume=32&publication_year=2011&pages=557-63&pmid=21520333&doi=10.1002/humu.21438&)

20. Vladutiu GD, Isackson PJ, Kaufman K, Harley JB, Cobb B, Christopher-Stine L, Wortmann RL. Genetic risk for malignant hyperthermia in non-anesthesia-induced myopathies. Mol Genet Metab. 2011;104:167–73. doi: 10.1016/j.ymgme.2011.07.001.  [DOI](https://doi.org/10.1016/j.ymgme.2011.07.001) | [PMC free article](/articles/PMC3171598/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21795085/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Genet%20Metab&title=Genetic%20risk%20for%20malignant%20hyperthermia%20in%20non-anesthesia-induced%20myopathies&author=GD%20Vladutiu&author=PJ%20Isackson&author=K%20Kaufman&author=JB%20Harley&author=B%20Cobb&volume=104&publication_year=2011&pages=167-73&pmid=21795085&doi=10.1016/j.ymgme.2011.07.001&)

21. Gillard EF, Otsu K, Fujii J, Khanna VK, de Leon S, Derdemezi J, Britt BA, Duff CL, Worton RG, MacLennan DH. A substitution of cysteine for arginine 614 in the ryanodine receptor is potentially causative of human malignant hyperthermia. Genomics. 1991;11:751–5. doi: 10.1016/0888-7543(91)90084-r.  [DOI](https://doi.org/10.1016/0888-7543(91)90084-r) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/1774074/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genomics&title=A%20substitution%20of%20cysteine%20for%20arginine%20614%20in%20the%20ryanodine%20receptor%20is%20potentially%20causative%20of%20human%20malignant%20hyperthermia&author=EF%20Gillard&author=K%20Otsu&author=J%20Fujii&author=VK%20Khanna&author=S%20de%20Leon&volume=11&publication_year=1991&pages=751-5&pmid=1774074&doi=10.1016/0888-7543(91)90084-r&)

22. Carpenter D, Robinson RL, Quinnell RJ, Ringrose C, Hogg M, Casson F, Booms P, Iles DE, Halsall PJ, Steele DS, Shaw MA, Hopkins PM. Genetic variation in RYR1 and malignant hyperthermia phenotypes. Br J Anaesth. 2009;103:538–48. doi: 10.1093/bja/aep204.  [DOI](https://doi.org/10.1093/bja/aep204) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19648156/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Anaesth&title=Genetic%20variation%20in%20RYR1%20and%20malignant%20hyperthermia%20phenotypes&author=D%20Carpenter&author=RL%20Robinson&author=RJ%20Quinnell&author=C%20Ringrose&author=M%20Hogg&volume=103&publication_year=2009&pages=538-48&pmid=19648156&doi=10.1093/bja/aep204&)

23. Sei Y, Sambuughin N, Muldoon S. Anesthesiology 2004. Vol. 100. Bethesda, Maryland: Sep 4-5, 2002. Malignant hyperthermia genetic testing in North America Working Group Meeting; pp. 464–5.  [DOI](https://doi.org/10.1097/00000542-200402000-00058) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14870754/) | [Google Scholar](https://scholar.google.com/scholar_lookup?title=Anesthesiology%202004&author=Y%20Sei&author=N%20Sambuughin&author=S%20Muldoon&publication_year=2002&)

24. Zhou H, Lillis S, Loy RE, Ghassemi F, Rose MR, Norwood F, Mills K, Al-Sarraj S, Lane RJ, Feng L, Matthews E, Sewry CA, Abbs S, Buk S, Hanna M, Treves S, Dirksen RT, Meissner G, Muntoni F, Jungbluth H. Multi-minicore disease and atypical periodic paralysis associated with novel mutations in the skeletal muscle ryanodine receptor (RYR1) gene. Neuromuscul Disord. 2010;20:166–73. doi: 10.1016/j.nmd.2009.12.005.  [DOI](https://doi.org/10.1016/j.nmd.2009.12.005) | [PMC free article](/articles/PMC3136116/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20080402/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neuromuscul%20Disord&title=Multi-minicore%20disease%20and%20atypical%20periodic%20paralysis%20associated%20with%20novel%20mutations%20in%20the%20skeletal%20muscle%20ryanodine%20receptor%20(RYR1)%20gene&author=H%20Zhou&author=S%20Lillis&author=RE%20Loy&author=F%20Ghassemi&author=MR%20Rose&volume=20&publication_year=2010&pages=166-73&pmid=20080402&doi=10.1016/j.nmd.2009.12.005&)

25. Klein A, Jungbluth H, Clement E, Lillis S, Abbs S, Munot P, Pane M, Wraige E, Schara U, Straub V, Mercuri E, Muntoni F. Muscle magnetic resonance imaging in congenital myopathies due to ryanodine receptor type 1 gene mutations. Arch Neurol. 2011;68:1171–9. doi: 10.1001/archneurol.2011.188.  [DOI](https://doi.org/10.1001/archneurol.2011.188) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21911697/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arch%20Neurol&title=Muscle%20magnetic%20resonance%20imaging%20in%20congenital%20myopathies%20due%20to%20ryanodine%20receptor%20type%201%20gene%20mutations&author=A%20Klein&author=H%20Jungbluth&author=E%20Clement&author=S%20Lillis&author=S%20Abbs&volume=68&publication_year=2011&pages=1171-9&pmid=21911697&doi=10.1001/archneurol.2011.188&)

26. Zhou H, Yamaguchi N, Xu L, Wang Y, Sewry C, Jungbluth H, Zorzato F, Bertini E, Muntoni F, Meissner G, Treves S. Characterization of recessive RYR1 mutations in core myopathies. Hum Mol Genet. 2006;15:2791–803. doi: 10.1093/hmg/ddl221.  [DOI](https://doi.org/10.1093/hmg/ddl221) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16940308/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mol%20Genet&title=Characterization%20of%20recessive%20RYR1%20mutations%20in%20core%20myopathies&author=H%20Zhou&author=N%20Yamaguchi&author=L%20Xu&author=Y%20Wang&author=C%20Sewry&volume=15&publication_year=2006&pages=2791-803&pmid=16940308&doi=10.1093/hmg/ddl221&)

27. Zhou H, Jungbluth H, Sewry CA, Feng L, Bertini E, Bushby K, Straub V, Roper H, Rose MR, Brockington M, Kinali M, Manzur A, Robb S, Appleton R, Messina S, D'Amico A, Quinlivan R, Swash M, Muller CR, Brown S, Treves S, Muntoni F. Molecular mechanisms and phenotypic variation in RYR1-related congenital myopathies. Brain. 2007;130:2024–36. doi: 10.1093/brain/awm096.  [DOI](https://doi.org/10.1093/brain/awm096) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17483490/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Brain&title=Molecular%20mechanisms%20and%20phenotypic%20variation%20in%20RYR1-related%20congenital%20myopathies&author=H%20Zhou&author=H%20Jungbluth&author=CA%20Sewry&author=L%20Feng&author=E%20Bertini&volume=130&publication_year=2007&pages=2024-36&pmid=17483490&doi=10.1093/brain/awm096&)

28. Klein A, Lillis S, Munteanu I, Scoto M, Zhou H, Quinlivan R, Straub V, Manzur AY, Roper H, Jeannet PY, Rakowicz W, Jones DH, Jensen UB, Wraige E, Trump N, Schara U, Lochmuller H, Sarkozy A, Kingston H, Norwood F, Damian M, Kirschner J, Longman C, Roberts M, Auer-Grumbach M, Hughes I, Bushby K, Sewry C, Robb S, Abbs S, Jungbluth H, Muntoni F. Clinical and genetic findings in a large cohort of patients with ryanodine receptor 1 gene-associated myopathies. Hum Mutat. 2012;33:981–8. doi: 10.1002/humu.22056.  [DOI](https://doi.org/10.1002/humu.22056) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22473935/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mutat&title=Clinical%20and%20genetic%20findings%20in%20a%20large%20cohort%20of%20patients%20with%20ryanodine%20receptor%201%20gene-associated%20myopathies&author=A%20Klein&author=S%20Lillis&author=I%20Munteanu&author=M%20Scoto&author=H%20Zhou&volume=33&publication_year=2012&pages=981-8&pmid=22473935&doi=10.1002/humu.22056&)

29. Rosenberg H, Sambuughin N, Dirksen R. In: Malignant Hyperthermia Susceptibility, GeneReviews. Pagon RA, Bird TD, Dolan CR, Stephens K, Seattle WA, editors. University of Washington; Seattle: 1993.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Malignant%20Hyperthermia%20Susceptibility,%20GeneReviews&author=H%20Rosenberg&author=N%20Sambuughin&author=R%20Dirksen&publication_year=1993&)

30. Pirone A, Schredelseker J, Tuluc P, Gravino E, Fortunato G, Flucher BE, Carsana A, Salvatore F, Grabner M. Identification and functional characterization of malignant hyperthermia mutation T1354S in the outer pore of the Cavalpha1S-subunit. Am J Physiol Cell Physiol. 2010;299:C1345–54. doi: 10.1152/ajpcell.00008.2010.  [DOI](https://doi.org/10.1152/ajpcell.00008.2010) | [PMC free article](/articles/PMC3006335/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20861472/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Physiol%20Cell%20Physiol&title=Identification%20and%20functional%20characterization%20of%20malignant%20hyperthermia%20mutation%20T1354S%20in%20the%20outer%20pore%20of%20the%20Cavalpha1S-subunit&author=A%20Pirone&author=J%20Schredelseker&author=P%20Tuluc&author=E%20Gravino&author=G%20Fortunato&volume=299&publication_year=2010&pages=C1345-54&pmid=20861472&doi=10.1152/ajpcell.00008.2010&)

31. Klingler W, Rueffert H, Lehmann-Horn F, Girard T, Hopkins PM. Core myopathies and risk of malignant hyperthermia. Anesth Analg. 2009;109:1167–73. doi: 10.1213/ANE.0b013e3181b5ae2d.  [DOI](https://doi.org/10.1213/ANE.0b013e3181b5ae2d) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19762745/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Anesth%20Analg&title=Core%20myopathies%20and%20risk%20of%20malignant%20hyperthermia&author=W%20Klingler&author=H%20Rueffert&author=F%20Lehmann-Horn&author=T%20Girard&author=PM%20Hopkins&volume=109&publication_year=2009&pages=1167-73&pmid=19762745&doi=10.1213/ANE.0b013e3181b5ae2d&)

32. Rosenberg H, Davis M, James D, Pollock N, Stowell K. Malignant hyperthermia. Orphanet J Rare Dis. 2007;2:21. doi: 10.1186/1750-1172-2-21.  [DOI](https://doi.org/10.1186/1750-1172-2-21) | [PMC free article](/articles/PMC1867813/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17456235/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Orphanet%20J%20Rare%20Dis&title=Malignant%20hyperthermia&author=H%20Rosenberg&author=M%20Davis&author=D%20James&author=N%20Pollock&author=K%20Stowell&volume=2&publication_year=2007&pages=21&pmid=17456235&doi=10.1186/1750-1172-2-21&)
