# Performance and Value of IFN-Lambda3 and IFN-Lambda4 Genotyping in Patients with Chronic Hepatitis C (CHC) Genotype 2/3 in a Real World Setting

## Metadata
**Authors:** Steffen B Wiegand, Benjamin Heidrich, Simone Susser, Magdalena Rogalska-Taranta, Jörg Petersen, Klaus H W Böker, Natalia Grigorian, Ralph Link, Uwe Naumann, Christine John, Stefan Lueth, Peter Malfertheiner, Michael P Manns, Heiner Wedemeyer, Christoph Sarrazin, Markus Cornberg
**Journal:** PLoS ONE
**Date:** 2015 Dec 23
**DOI:** [10.1371/journal.pone.0145622](https://doi.org/10.1371/journal.pone.0145622)
**PMID:** 26699619
**PMCID:** PMC4689517
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4689517/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC4689517/pdf/pone.0145622.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC4689517/pdf/pone.0145622.pdf)

## Abstract

**Background:** 
SNPs near the interferon lambda (IFNL) 3 gene are predictors for sustained virological response (SVR) in patients with chronic hepatitis C genotype (GT) 1. In addition, a dinucleotide frame shift in ss469415590 was described, which generates IFNL4. In this study, we compared the role of IFNL4 variants with IFNL3-(rs12979860) and IFNL3-(rs8099917) on response to pegylated (PEG)-IFN and Ribavirin (RBV) in patients with chronic hepatitis C GT2/3.

**Methods:** 
We recruited 1006 patients with chronic hepatitis C and GT2/3 in a large German registry. A treatment with PEG-IFN and Ribavirin was started by 959 patients. We performed genotyping of IFNL3 (rs12979860, n = 726; rs8099917, n = 687) and of IFNL4 (ss469415590; n = 631).

**Results:** 
Both preferable IFNL3 genotypes were associated with RVR (both p<0.0001) rather than with SVR (rs12979860: p = 0.251; rs8099917: p = 0.447). Only RVR was linked to SVR in univariate and multivariate analyzes (both p<0.001). Concordance of genotyping in patients with available serum samples and EDTA blood samples (n = 259) was more than 96% for both IFNL3 SNPs. IFNL3-(rs12979860) correlated with IFNL4: 99.2% of patients with IFNL3-(rs12979860)-CC were IFNL4-(ss469415590)-TT/TT. IFNL3-(rs12979860)-CT was linked with IFNL4-(ss469415590)-TT/ΔG (98.0%) and IFNL3-(rs12979860)-TT was associated with IFNL4-(ss469415590)-ΔG/ΔG (97.6%).

**Conclusion:** 
IFNL3 genotyping from serum was highly efficient and can be used as an alternative if EDTA whole blood is not available. In Caucasian GT2/3 patients genotyping for INFL4-(ss469415590) does not lead to additional information for the decision-making process. Importantly, IFNL3 SNPs were not associated with SVR but with RVR. Even in the era of new direct acting antiviral (DAA) therapies, IFNL3 testing may therefore still be considered for naïve GT2/3 patients to decide if dual Peg-IFN/RBV therapy is an option in resource limited regions.

### Background

SNPs near the interferon lambda (IFNL) 3 gene are predictors for sustained virological response (SVR) in patients with chronic hepatitis C genotype (GT) 1. In addition, a dinucleotide frame shift in ss469415590 was described, which generates IFNL4. In this study, we compared the role of IFNL4 variants with IFNL3-(rs12979860) and IFNL3-(rs8099917) on response to pegylated (PEG)-IFN and Ribavirin (RBV) in patients with chronic hepatitis C GT2/3.

### Methods

We recruited 1006 patients with chronic hepatitis C and GT2/3 in a large German registry. A treatment with PEG-IFN and Ribavirin was started by 959 patients. We performed genotyping of IFNL3 (rs12979860, n = 726; rs8099917, n = 687) and of IFNL4 (ss469415590; n = 631).

### Results

Both preferable IFNL3 genotypes were associated with RVR (both p<0.0001) rather than with SVR (rs12979860: p = 0.251; rs8099917: p = 0.447). Only RVR was linked to SVR in univariate and multivariate analyzes (both p<0.001). Concordance of genotyping in patients with available serum samples and EDTA blood samples (n = 259) was more than 96% for both IFNL3 SNPs. IFNL3-(rs12979860) correlated with IFNL4: 99.2% of patients with IFNL3-(rs12979860)-CC were IFNL4-(ss469415590)-TT/TT. IFNL3-(rs12979860)-CT was linked with IFNL4-(ss469415590)-TT/ΔG (98.0%) and IFNL3-(rs12979860)-TT was associated with IFNL4-(ss469415590)-ΔG/ΔG (97.6%).

### Conclusion

IFNL3 genotyping from serum was highly efficient and can be used as an alternative if EDTA whole blood is not available. In Caucasian GT2/3 patients genotyping for INFL4-(ss469415590) does not lead to additional information for the decision-making process. Importantly, IFNL3 SNPs were not associated with SVR but with RVR. Even in the era of new direct acting antiviral (DAA) therapies, IFNL3 testing may therefore still be considered for naïve GT2/3 patients to decide if dual Peg-IFN/RBV therapy is an option in resource limited regions.

## Introduction

World-wide 64–103 million people are chronically infected with the hepatitis C virus (HCV) [[1](#pone.0145622.ref001)]. There are seven HCV genotypes, which show different distributions around the world. Genotype 2 is frequent in some parts of Asia. Genotype 3 is very frequent in South-East Asia and accounts for around one third of infections in Europe [[2](#pone.0145622.ref002)]. HCV genotypes have influence on the natural course of chronic hepatitis, i.e. genotype 2 is associated with ALT flares while genotype 3 infected patients show faster disease progression and higher mortality [[3](#pone.0145622.ref003),[4](#pone.0145622.ref004)]. In addition, different genotypes require different treatment concepts, now and in the past.

For decades IFN in combination with RBV was the standard of care chronic hepatitis C. While patients with genotype 1 were treated for 48 weeks resulting in 50% SVR, patients with genotypes 2 and 3 achieved up to 90% SVR with 24 weeks therapy [[5](#pone.0145622.ref005),[6](#pone.0145622.ref006)]. Similar to HCV genotype 1, IFN based therapy can be individualized based on viral and host factors. For example, patients with high viral load at baseline achieved significantly less often SVR [[7](#pone.0145622.ref007)]. Host factors such as age, gender, stage of fibrosis, origin and body mass index affect treatment outcome in interferon based therapy regimes [[8](#pone.0145622.ref008)]. In recent years genome-wide association studies (GWAS) revealed several genetic polymorphisms (rs12979860; rs8099917; etc.) in the interferon lambda 3 (IFNL3) gene region (also known as IL28B), which are associated with SVR rates in patients with HCV genotype 1 undergoing combination therapy with pegylated interferon and ribavirin (PegIFN/RBV) [[9](#pone.0145622.ref009)–[11](#pone.0145622.ref011)].

In the last few years several new DAA have been approved for the treatment of hepatitis C. DAA combinations with or without IFN showed improved SVR rates up to 95%, especially in genotype 1 patients with shorter treatment duration [[12](#pone.0145622.ref012)]. However, in 2015 many of the available DAAs have some limitations in efficacy in genotype 2 and especially in genotype 3. In addition, novel DAA are not available in all parts of the world at the same time. Thus, IFN based therapies; especially for genotype 3 are still an important therapeutic concept [[13](#pone.0145622.ref013)]. So far, the data regarding an association of different IFNL3 genotypes with SVR in HCV genotype 2 and 3 patients is still controversial [[14](#pone.0145622.ref014)–[19](#pone.0145622.ref019)]. One study by Eslam et al. showed an associations between IFLN3 genotypes and SVR in patients with HCV genotype 2 and 3 [[18](#pone.0145622.ref018)], while other studies have demonstrated less clear results [[14](#pone.0145622.ref014)–[17](#pone.0145622.ref017),[19](#pone.0145622.ref019)]. One reason for the discrepancy may be due to small sample sizes used in some of these trials and the heterogeneity of subgroups analyzed [[14](#pone.0145622.ref014),[17](#pone.0145622.ref017),[18](#pone.0145622.ref018),[20](#pone.0145622.ref020)]. Recently, IFNL4 a new variant in the CpG region upstream of IFNL3 (IL28b) was observed which showed also strong association with HCV clearance [[21](#pone.0145622.ref021)]. Susser et al., showed that in genotype 3 infected patients, best SVR prediction was based on IFNL4 and not on IFNL3 genotype [[22](#pone.0145622.ref022)]. However, also in this study the number of genotype 3 patients was lower than 200. Therefore, the aim of this study was to analyze the association of IFNL3 and IFNL4 genotypes with therapy outcome in more than 600 treatment-naïve patients infected with genotypes 2 and 3 treated with pegylated interferon alfa-2b and ribavirin within a large multi-center nationwide prospective registry.

## Material and Methods

### Patient population

Overall, 152 centers participated in this prospective German nationwide multicenter registry and 1006 patients were recruited between June 2008 and December 2012. Eligibility criteria for the registry were age ≥18, HCV genotype 2 or 3, detectable HCV RNA (>15 IU/mL) and positivity of anti-HCV antibodies as well as no history of antiviral therapy ([Fig 1](#pone.0145622.g001)). Overall, 959 started treatment with PEG-IFN and Ribavirin with 114 being HCV genotype 2 and 567 being HCV genotype 3 with available data on IFNL3 and 4. Cirrhosis was present in 7% of the overall population. Detailed in- and exclusion criteria and baseline characteristics can be found in the supplementary materials ([S1 Fig](#pone.0145622.s002) and [S1 Table](#pone.0145622.s003)).

### Fig 1. Flow chart of patients with available IFNL3 and IFN-L4 genotype.

![Fig 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/11bd/4689517/d23d23734ef2/pone.0145622.g001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4689517_pone.0145622.g001.jpg)

### Interferon Lambda 3 (IFNL3) genotyping In EDTA-blood

IFNL3 (formerly IL28B) single-nucleotide polymorphism rs12979860 and rs8099917 genotyping was performed using real-time polymerase chain reaction and melting curve analysis in the Light Cycler 480 II System (Roche, Mannheim, Germany). DNA was extracted from EDTA-blood samples using the DNeasy purification Kit (QIAGEN, Hilden, Germany). Primers and hybridization probes were purchased from TIB MOLBIOL (Berlin, Germany).

### Genotyping of IFNL3 (IL28B) and IFN-L4 in serum

Genotypes of rs8099917 (*IFNL3*), rs12979860 (*IFNL3*) and ss469415590 (*IFN-L4*) were determined using an inventoried TaqMan^®^ SNP Genotyping assay (Life Technologies GmbH, Darmstadt, Germany) for rs8099917 and custom-designed TaqMan^®^ SNP Genotyping assays (Life Technologies GmbH, Darmstadt, Germany) for rs12979860 and ss469415590.

All reactions were set up with 1μL of isolated gDNA and TaqMan^®^ Genotyping Master Mix (Life Technologies GmbH, Darmstadt, Germany). The genotyping ran on a StepOnePlus^TM^ Real Time PCR System (Life Technologies GmbH, Darmstadt, Germany). Genotyping was performed at the Biomedical Research Laboratory of Medical Clinic 1, Goethe-University Hospital, Frankfurt, Germany.

### HCV RNA quantification

HCV RNA quantification was performed at baseline (BL), W4, -12, end of treatment and 24 weeks after cessation of therapy. Quantification was done locally with different assays and thresholds [[13](#pone.0145622.ref013)].

### Response definitions

HCV-RNA below 15 IU/mL at week 4 of treatment with pegylated interferon and ribavirin was defined as rapid virological response (RVR) according to the German Guidelines and based on the analysis by Sarrazin et al., [[23](#pone.0145622.ref023),[24](#pone.0145622.ref024)]. A decrease of more than 2 log10 from baseline in HCV RNA or HCV RNA negativity at week 12 was defined early virological response (EVR); end of treatment response (EOT) and sustained virological (SVR) were defined as HCV-RNA below detection limit 24 weeks after the end of treatment. In the intention-to-treat (ITT) analysis all patients with at least one dose of PegIFN and RBV were included. Missing results for RVR and SVR were considered as negative results. For the per-protocol (PP) analysis, patients treated for at least 12 weeks and with available results at week 4 and/or 24 weeks after end of therapy were considered for each evaluation.

### Laboratory tests

Hematological and biochemical parameters were assessed locally at baseline. Alanine aminotransferase (ALT), aspartate aminotransferase (AST), gamma-glutamyltransferase (GGT) and alkaline phosphatase (ALP) as well as bilirubin, creatinine and albumin were part of the biochemical work-up. Hematological parameters included platelet counts.

### Definition of liver cirrhosis

Non-invasive methods like ultrasound, FibroScan® or biochemical results and liver histology were used for defining presence or absence of cirrhosis. F4 in Metavir or F5-6 in ISHAK in liver biopsies was considered as cirrhosis [[25](#pone.0145622.ref025),[26](#pone.0145622.ref026)]. Diagnosis of cirrhosis in ultrasound was based on assessment of the local physician. Same holds true for steatosis. Liver stiffness ≥12.5kPa was considered as cirrhosis [[27](#pone.0145622.ref027)]. Patients with at least two of the following criteria: AST/ALT ratio >1 [[28](#pone.0145622.ref028)], bilirubin >1,5ULN, platelets <100/nL and albumin <35g/L fulfilled biochemical assessment of cirrhosis. If one of the definitions above was met, presence of cirrhosis was considered.

### Statistical analysis

GraphPad Prism 5 (GraphPad Software, Inc., La Jolla, CA, USA) was used for graphic design. Statistical analysis was performed with SPSS, version 22 (2013, SPSS, Munich, Germany). Quantitative values are indicated in median or mean and statistical differences were assessed by using Student’s t test. Qualitative data was analyzed by using Chi square test. Differences were considered significant at p<0.05. An online Tool ([http://www.oege.org/software/hwe-mr-calc.shtml](http://www.oege.org/software/hwe-mr-calc.shtml)) for calculating the Hardy-Weinberg equilibrium based on Rodriguez publication was used [[29](#pone.0145622.ref029)].

### Ethical approval

The ethics committee of Hannover Medical School, Medical University of Leipzig, Regensburg, Cologne, Essen, Bonn, Bochum, Münster, Rostock, Halle-Wittenberg, Würzburg and the State Chamber for Physicians of Thuringia, Saxony, Saxony-Anhalt, North-Rhine, Lippe-Westphalia, Saarland and Bremen approved the registry. Each patient signed a written informed consent form. The registry has been performed according to the current World Medical Association Declaration of Helsinki. The procedures have been approved by the local ethics committee of the Hannover Medical School (Vote No. 3860) and are in line with German law.

## Results

### IFNL3 and IFNL4 genotyping

Overall, 726 out of 1006 (72%) individuals have been genotyped for IFNL3 rs12979860 and 687 samples were tested for IFNL3 rs8099917 ([Fig 1](#pone.0145622.g001)). Additionally, 631 samples could be genotyped for IFN-L4 ss469415590. The genotype distribution of the IFNL4 double nucleotide polymorphism and the two IFNL3 SNPs was in Hardy-Weinberg equilibrium. The overall genotype distribution of IFNL3 rs8099917 TT, GT, and GG was 64.3%, 31.4%, and 4.2% and the distribution of rs12979860 CC, CT, and TT was 41.5%, 45.7%, and 12.8%, respectively. For IFNL4 ss469415590 TT/TT, TT/ ΔG, and ΔG/ΔG were seen in 40.9%, 46.1%, and 13.0% of patients, respectively.

### Concordance in IFNL3 genotyping in EDTA whole blood and serum

Testing for IFNL3 rs12979860, IFNL3 rs8099917 and IFNL4 ss469415590 was not scheduled in the initiation phase of this registry and was performed with available stored EDTA whole blood samples and/ or serum samples as illustrated in [Fig 1](#pone.0145622.g001).

Overall, in almost all samples concordant results between serum and EDTA whole blood samples were observed in IFNL3 rs12979860 and rs8099917 genotypes with concordance rates of 99% (132 out of 134) and 96% (120 out of 125), respectively.

In those two cases with discordance for rs12979860 genotype one patient had CT in EDTA whole blood and CC in serum and the other had TT in EDTA whole blood and CC in serum ([Fig 1](#pone.0145622.g001)). For genotyping of rs8099917 discordant results were observed in five cases (1 sample: GT (EDTA) vs. GG (serum); 2 samples: GT (EDTA) vs. TT (serum); 2 samples: TT (EDTA) vs. GT (serum)) ([Fig 1](#pone.0145622.g001)).

### IFLN4 ss469415590 in Caucasian patients with HCV G2/3

In addition, serum samples of 631 patients were tested for IFLN4 genotype. The overall genotype distribution of IFNL4 ss469415590 TT/TT, TT/ΔG, and ΔG/ΔG was 40.9%, 46.1%, and 13.0%. We observed a very high correlation between IFNL3 (rs12979860) and IFN-L4 (ss469415590) (r = 0.9777; p<0.0001). IFNL3 (rs12979860) CC was associated with IFN-L4 (ss469415590) TT/TT in 99.2% of patients. IFNL3 (rs12979860) CT was linked with IFN-L4 (ss469415590) TT/ΔG (98.0%) and IFNL3 (rs12979860) TT was associated with IFN-L4 (ss469415590) ΔG/ΔG (97.6%).

### Treatment response in correlation with IFNL3 and IFLN4 genotypes

Due to the high association between IFNL3 (rs12979860) and IFN-L4 (ss469415590) we only used IFNL3 SNPs for further analyses. There was no statistical difference considering SVR rates in ITT as well as PP analysis in different SNPs (rs12979860 and rs8099917) of IFNL3 ([Table 1](#pone.0145622.t001) and [Fig 2A](#pone.0145622.g002)). The so-called preferable genotypes CC-rs12979860 and TT- rs8099917 showed higher RVR rates compared to the non-preferable ones (82% vs. 63%; p<0.0001 and 80% vs. 55%; p<0.0001) ([Fig 2B](#pone.0145622.g002)).

### Table 1. SVR rates in relation to IFLN3 genotype.

| Analysis | rs12979860 |  |  | p-value* | rs8099917 |  |  | p-value* |
| --- | --- | --- | --- | --- | --- | --- | --- | --- |
|   | CC | CT | TT |   | TT | GT | GG |   |
| ITT | 51.2% | 49.2% | 37.9% | 0.09 | 51.1% | 50.0% | 33.3% | 0.20 |
| PP | 89.6% | 89.2% | 89.2% | 0.99 | 90.4% | 88.9% | 81.8% | 0.62 |
|   | CC | Non-CC |  |   | TT | Non-TT |  |   |
| ITT | 51.2% | 46.8% |  | 0.25 | 51.1% | 48.1% |  | 0.45 |
| PP | 89.6% | 89.2% |  | 0.87 | 90.4% | 88.3% |  | 0.50 |

Table 1 Caption: * Chi square test

### Fig 2. SVR and RVR rates according to INF-L3 and HCV genotype.

![Fig 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/11bd/4689517/4ec95f8db0c2/pone.0145622.g002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4689517_pone.0145622.g002.jpg)

(A) SVR rates in relation to IFNL3 genotype (PP analysis). (B) RVR rates in relation to IFNL3 genotype (PP analysis). (C) SVR rates in relation to IFNL3 genotype in HCV genotype 2 patients. (D) SVR rates in relation to IFNL3 genotype in HCV genotype 3 patients. (E) RVR rates in relation to IFNL3 genotype in HCV genotype 3 patients. (F) RVR rates in relation to IFNL3 genotype in HCV genotype 2 patients.

During this study period we observed 32 relapses and only 5 breakthroughs in the per-protocol analysis. The majority of patients with relapse were HCV genotype 3 (n = 27; 84%) and had more often the favorable genotype TT-rs8099917 rather than non-TT-rs8099917 (n = 16 (62%) vs. n = 10 (38%)). In contrast, the favorable genotype CC-rs12979860 was observed in the minority of patients with relapse (n = 11 (39%) vs. n = 17 (61%)).

### Correlation of IFNL3 genotypes and treatment response in different HCV genotypes

The overall SVR was high (91%) for both HCV genotypes. However, patients infected with HCV genotype 2 had significantly higher SVR rates than patients with HCV genotype 3 (96% vs. 90%; p = 0.0479). Week 4 responses were comparable to SVR results but without being statistical significant (78% vs. 70%; p = 0.0520).

The preferable genotype distribution of both SNPs was similar in GT2 and GT3 (rs12979860: 44% vs. 41%; p = 0.5563 and rs8099917: 66% vs. 64%; p = 0.6449) ([Table 2](#pone.0145622.t002)). Regarding SVR rates, there were no statistical significant differences between preferable and non-preferable IFNL3 genotype in HCV genotype 2 and 3 ([Fig 2C and 2D](#pone.0145622.g002)). Patients with HCV genotype 3 had significantly higher RVR rates if both preferable IFNL3 SNPs were present (CC 80% vs. non-CC 61%; p<0.0001 and TT 79% vs. Non-TT 53%; p<0.0001) ([Fig 2E](#pone.0145622.g002)). In patients with HCV genotype 2 only IFNL3 rs8099917 and not rs12979860 showed significant differences in RVR rates (TT: 84% vs. non-TT: 63%; p = 0.0162; CC: 86% vs. non CC: 71%; p = 0.0606) ([Fig 2F](#pone.0145622.g002)).

### Table 2. IFNL3 genotype distribution in HCV genotype 2 and 3.

|   | rs12979860 |  | p-value | rs8099917 |  | p-value |
| --- | --- | --- | --- | --- | --- | --- |
|   | CC | non-CC |   | TT | non-TT |   |
|   | n = 301 | n = 425 |   | n = 442 | n = 245 |   |
| In GT2 (%) | 44 | 56 | 0.5141 | 66 | 34 | 0.6449 |
| In GT3 (%) | 41 | 59 | 0.5141 | 64 | 36 | 0.6449 |
### Predictor of treatment responses

Pretreatment clinical variables and on-treatment responses were analyzed considering association to SVR. Only younger age, stage of liver fibrosis, HCV genotype and RVR were linked with SVR-PP in a univariate analysis. All parameters with a p-value ≤0.05 were considered for multivariate analysis. Only RVR was independently associated with SVR-PP in multivariate analysis (p<0.001) ([Table 3](#pone.0145622.t003)). A retrospective power analysis showed a requirement of n>383 for rs12979860 genotype and n>391 for rs8099917 genotype in order to have 90% power to detect an odds ratio of 2 with a p-value of 0.05. SVR-ITT analyses fulfilled these criteria and in addition revealed an effect of gender and BMI in the univariate analysis, but only RVR remained independently correlated with SVR ([Table 3](#pone.0145622.t003)).

### Table 3. Univariate and multivariate analysis of pretreament and on treatment parameters considering SVR.

|   | SVR-PP | Univariate analysis | Multivariate analysis | SVR-ITT | Univariate analysis | Multivariate analysis |
| --- | --- | --- | --- | --- | --- | --- |
|   | n = (%) | p = | p = | n = (%) | p = | p = |
| Age<40 | 167 (96) | 0.050 | 0.600 | 167 (45) | 0.026 | 0.783 |
| Age>40 | 298 (88) | 0.050 | 0.600 | 298 (53) | 0.600 | 0.600 |
| Male | 279 (89) | 0.069 |   | 279 (45) | <0.001 | 0.170 |
| Female | 190 (94) | 0.069 |   | 190 (59) | 0.069 | 0.170 |
| BMI>27 | 162 (92) | 0.716 |   | 162 (55) | 0.022 | 0.978 |
| BMI<27 | 297 (91) | 0.716 |   | 297 (47) | 0.022 | 0.978 |
| Cirrhosis | 25 (74) | <0.001 | 0.435 | 25 (33) | 0.005 | 0.580 |
| Non-cirrhosis | 444 (92) | <0.001 | 0.435 | 444 (50) | 0.022 | 0.580 |
| Steatosis | 128 (90) | 0.359 |   | 128 (57) | 0.103 |   |
| Non-steatosis | 281 (92) | 0.359 |   | 281 (50) | 0.103 |   |
| Genotype 2 | 99 (96) | 0.048 | 0.217 | 99 (61) | <0.001 | 0.147 |
| Genotype 3 | 366 (90) | 0.048 | 0.217 | 366 (48) | <0.001 | 0.147 |
| BL <800,000 IU/mL | 241 (92) | 0.243 |   | 241 (48) | 0.603 |   |
| BL >800,000 IU/mL | 218 (89) | 0.243 |   | 218 (50) | 0.603 |   |
| CC- rs12979860 | 147 (90) | 0.893 |   | 147 (51) | 0.251 |   |
| Non-CC- rs12979860 | 190 (89) | 0.893 |   | 190 (47) | 0.251 |   |
| TT- rs8099917 | 217 (90) | 0.590 |   | 217 (51) | 0.447 |   |
| Non-TT-rs8099917 | 113 (88) | 0.590 |   | 113 (48) | 0.447 |   |
| RVR (PP or ITT) | 340 (95) | <0.001 | <0.001 | 340 (59) | <0.001 | <0.001 |
| Non-RVR (PP or ITT) | 80 (77) | <0.001 | <0.001 | 130 (34) | <0.001 | <0.001 |
### Association between viral load and IFNL3 genotypes

Patients with HCV genotype 2 and 3 had similar median levels of HCV RNA at baseline (6.2±1.6 log10 IU/mL vs. 5.7±1.4 log10 IU/mL; p = 0.1838). However, in HCV genotype 2 the frequency of patients with high viral load above 800,000 IU/mL was significantly higher compared to genotype 3 patients (62% vs. 44%; p<0.0001).

Patients with CC genotype (rs12979860) had significantly higher median baseline HCV RNA levels compared with non-CC genotype (6.1±1.2 log10 IU/mL vs. 5.6±1.0 log10 IU/mL; p = 0.0292) and had more often high viral load (57% vs. 41%; p<0.0001) ([Fig 3A](#pone.0145622.g003)). Interestingly, these differences seen in the overall population could be observed only in HCV genotype 3 patients. The association between rs8099917 genotypes and HCV viral load revealed similar results (TT: 6.0±1.2 log10 IU/mL vs. non-TT: 5.6±1.0 log10 IU/mL; p = 0.0362) ([Fig 3B](#pone.0145622.g003)).

### Fig 3. HCV RNA levels at baseline according to IFNL3 genotypes.

![Fig 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/11bd/4689517/c557195774e8/pone.0145622.g003.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4689517_pone.0145622.g003.jpg)

(A) HCV RNA levels at baseline for patients with favourable and non-favorable rs1297860 genotype. (B) HCV RNA levels at baseline for patients with favourable and non-favorable rs8099917 genotype.

### Week 4 response in HCV genotype 2/3 patients according to viral load and IFNL3 genotypes

Patients with genotype 3 and low viral load, defined as HCV RNA below 800,000 IU/mL at baseline, achieved more often RVR (74% vs 64%; p = 0.0074) ([Fig 4](#pone.0145622.g004)). Low viral load at baseline and each favorable genotype were associated with significantly higher RVR rates. Individuals with favorable IFNL3 genotype and low viral load at baseline compared to patients with non-favorable IFNL3 genotype and high viral load at baseline, had more pronounced differences in RVR rates (rs12979860: 87% vs. 48%; p<0.0001 and rs8099917: 87% vs. 30%; p<0.0001) ([Fig 4](#pone.0145622.g004)). In patients with beneficial IFNL3 genotype but high viral load RVR rates were comparable to individuals with non-beneficial genotype but low viral load, (rs12979860: 76% vs. 70%; p = 0.3264 and rs8099917: 72% vs. 67%; p = 0.3836) ([Fig 4](#pone.0145622.g004)).

### Fig 4. RVR in HCV genotype 3 according to baseline viral load in relation to IFNL3 genotype.

![Fig 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/11bd/4689517/52b727333669/pone.0145622.g004.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4689517_pone.0145622.g004.jpg)

Data with those virological and IFNL3 configuration considering SVR rates are located in the supporting information ([S1A Fig](#pone.0145622.s002)).

The same analyses, considering RVR and SVR rates, have been performed for G2 patients with similar results in parts ([S1B and S1C](#pone.0145622.s002) Fig). However, the numbers of patients in the subgroups were low.

## Discussion

In this large cohort of more than 600 patients chronically infected with HCV genotype 2 or 3 and available IFNL3 data we could show (i) association of IFNL3 and IFNL4 with RVR rather than SVR, (ii) strong association between IFNL4 ss469415590 and IFNL3 rs12979860 genotypes (iii) and high concordance between IFNL3 genotypes determined in serum and in EDTA-blood.

The role for IFNL3 or IFNL4 testing for patients with genotype 2 and 3 still remains controversial. Many studies have shown an association of IFL3 and RVR but not SVR ([[14](#pone.0145622.ref014)–[19](#pone.0145622.ref019)]). However, recently, Eslam et al. [[18](#pone.0145622.ref018)] demonstrated a high association of both preferable SNPs with SVR rates. Eslam et al. discussed that prior studies [[14](#pone.0145622.ref014)–[19](#pone.0145622.ref019)] revealed no association of IFNL3 SNPs with SVR because patient numbers in those studies were too low to have the ability to detect any effect of IFNL3 SNPs on the outcome of treatment. In order to refute a lack of power due to high SVR rates in PP analysis a retrospective power analysis considering SVR ITT results was performed. Therefore, the number of included individuals in this study should be sufficient. However, despite the high number of patients included in this registry, neither in the PP nor in the ITT analysis an association between INFL3 SNPs and SVR could be observed. However, there was a linkage between IFNL3 SNPs and RVR and RVR remained the only independent factor associated with SVR in multivariate analysis. This was consistent with previous studies [[23](#pone.0145622.ref023)–[28](#pone.0145622.ref028)]. There are some limitations in our but also in the study by Eslam. Neither study is a prospective clinical trial. In Eslams study a fixed treatment duration of 24 weeks was implemented, whereas in this registry each participating physician could determine individual therapy durations, leading in general to an overtherapy like we published before [[13](#pone.0145622.ref013)]. Eslam et al., did not report the ribavirin doses actually used, which may impact the response in more difficult to treat patients. In our registry the recommendation were based on the German Clinical Practice Guideline (12–15 mg/kg) [[23](#pone.0145622.ref023)] and is often higher compared to the fixed 800mg dose initially suggested [[30](#pone.0145622.ref030)]. Our patients received 13.7 mg/kg in average. Our registry only included naïve patients while the study by Eslam et al., did not report if the patients were naïve or have been treated with PEG-IFN/RBV before [[18](#pone.0145622.ref018)]. Also the number of patients with moderate/severe fibrosis was higher in the study by Eslam. The association of IFNL3 and SVR may be lost in naïve easier to treat patients. The study by Eslam also included not only Caucasian patient.

Although there was no association between IFNL3 SNPs to SVR, a strong association to RVR was present and RVR remains the strongest factor for achieving SVR. In the end this may be even the more relevant association in the times of new emerging DAA therapies. In those “easy-to-treat” patients with low viral load, no signs of cirrhosis and the important milestone RVR, combination therapy of Peg-IFN and RBV may still be considered as first line treatment option. This concept is far less expensive [[13](#pone.0145622.ref013)] and could save resources for the more difficult to treat patients (cirrhosis, non-RVR) which should be treated with DAAs. We therefore suggest that the preferred patient populations for IFNL3 testing are naïve genotype 2 and especially naïve genotype 3 patients with low baseline viremia and without signs of cirrhosis. For example, G3 patients positive for IFNL3 CC genotype had 80% RVR of whom 94% achieved SVR in the PP analysis with a median of 24 weeks PEG-IFN/RBV.

In a recent study Susser et al. showed that IFN-L4 genotype was the best single predictor for SVR in genotype 3 infected patients [[22](#pone.0145622.ref022)]. However, we could not observe better predictive values for IFNL4 compared to IFNL3 in Genotype 3 patients. In the study of Susser et al. a high association of IFNL4 ss469415590 and IFNL3 rs12979860 was as well observed. Additionally, in their study they investigated only 191 patients with HCV genotype 3 which might influence results. Finally, the calculated p-value of 0.044 for the association of IFNL4 ss469415590 and SVR was only borderline significant.

Analysis of host genetic polymorphism usually requires EDTA blood. Sometimes, retrospective analyses of large cohorts are not possible due to the lack of stored samples. One of our other aims in this study was to assess if IFNL3 and IFNL4 genotyping was possible from serum samples. Our study revealed high concordance and in case of discordance results differed only by one base. Initially 712 samples had to be retested for IFNL3 genotyping with 687 cases (97%) revealing results for both SNPs. Thus, IFNL3 polymorphisms could easily be analysed from stored serum samples.

In conclusion, IFNL3 polymorphisms testing may be a tool for genotype 2/3 patients to decide if dual Peg-IFN/RBV therapy is an option to save resources in the times of new DAA therapies. IFNL4 showed similar predictive value in Caucasian patients and is not recommended in this setting.

## Supporting Information

## Acknowledgments

We thank all the contributors of the study and all patients recruited in this registry.

Sven Wollschläger, Johannes Wiegand, Andreas Stallmach, Renate Heyne, Thomas Berg, Jörg Gölz, Dorothea Schleehauf, Klaus Schmidt, Rainer Günther, Joachim C. Arnold, Johann Ockenga, Stefanie Holm, Christoph Roggel, Jörg Petersen, Christian Hoffmann, Dietrich Hüppe, Guido Gerken, Andreas Lügering, Herman E. Wasmuth, Ulrich Spengler, Stephanie Sippel, Stefan Zeuzem, Maria S. Leuschner, Frank Lammert, Michael Schuster, Stephan Schiekofer, Manfred V. Singer, Christoph Eisenbach, Andreas Trein, Michael Gregor, Martin Rössle, Bernd Moeller, Michael R. Kraus, Nektarios Dikopoulos, Wolfgang Norgauer, Reiner Wiest, Kurt Grüngreiff, H.-J. Cordes, Bruno Tulaj, Elke Lauenroth-Mai, Michael Damm, Michael Mroß, Burkhard Otremba, Hermann Steffens, Oliver Marschal, Stephan Pfäffl, Slawomir Sworszt, Hartmut Bordel, Franke Armand von Lucadou, Holger Hinrichsen, Markus Reiser, Michael Brack, Uwe Deuster, Dorothea Schleehauf, Hartwig Klinker, Elmar Zehnter, Manfred Wiese, Bertram Schmack, Beate Quarterman, Sabine Morgenschweis, Axel Baumgarten, Karl-Heinz Kastner, Karl-Wilhelm Wietholt, Gunnild Baldini, Walter Steege, Rolf Zimmer, Karl Metz, Siegfried Köppe, Stefan Schubert, Wolfgang Mohl, Sabine Mauruschat, Heribert Renner, Bernd Weber, Manfred Lohnstein, Michael Falk, Manfred Kuhn, Reinhard Diedrich, Andreas Schober, Thomas Wünsche, Jürgen Krause, Lutwinus Weitner, Rajan Somasundaram, Hans-Jörg Epple, H. Lochs, Micha Löbermann, Wolfgang Klemm, Ann Michelsen, Gunter Brauer, Christian Jellinek, Torsten Hoheisel, Rüdiger Behrens, Dieter Heidbreder, Sebastian Hinz, Wolfgang Steigleder, Matthias Dollinger, Philipp Thermann, Gerlinde Teuber, Alexander Zipprich, Patrick Krummenerl, Ralf Bade, Gero Moog, Peter Klein, Eduard Boniakowski, Rolf Lange, M. Freitag, Andreas Zipf, Jörg Claus, Axel Ring, U. Ehrle, Wolfgang Gickler, Heiner W. Busch, Reinhard Baumgartner, Willi Schiffelholz, Wolfram Graf Finck v. Finkenstein, Dagobert Ross, Th. Koch, Gerd Klausen, Karl-Heinz Hey, B. Küppers, Annett Borkenhagen, Ulrich Keller, Thomas Stritzke, Frank Doering, Claus Niederau, Eugen Fürmann, Fajr Bannout, Stefan Roth, Maria Stalter, Klaus Huntenburg, Katrin Heine, Hans-Jürgen Stellbrink, Astrid Göttsche, Diether Ludwig, Lorenz L. Hillebrand, Rainer Wailersbacher, Kai U. Rehbehn, Thomas Käser, Peter L. Berg, Jochen George, Rainer Bindert, Karl-Georg Simon, Roland Reininghaus, Mirjam Reiner, Markus Sellinger, Horst Feyerabend, Martin Hoffstadt, Johannes Benninger, Andreas Baron, Walter Krämer, Ulrich Banhardt, Marcus Schuchmann, Martin Hunstiger, Thomas Witthöft, Petra Sandow, Geert Vogt

## Data Availability

All relevant data are within the paper and its Supporting Information files.

## Funding Statement

This study was supported financially by MSD Sharp & Dohme ([www.msd.de](http://www.msd.de)) and the German Liver Foundation ([http://www.deutsche-leberstiftung.de](http://www.deutsche-leberstiftung.de)). MSD had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript. The HepNet Study House of the German Liver Foundation was used for sample storage and data collection and had no role in study design, decision to publish or preparation of the manuscript.

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Data Availability Statement

All relevant data are within the paper and its Supporting Information files.

### Supplementary Materials

### Data Availability Statement

All relevant data are within the paper and its Supporting Information files.

## References

1. Mohd Hanafiah K, Groeger J, Flaxman AD, Wiersma ST. Global epidemiology of hepatitis C virus infection: new estimates of age-specific antibody to HCV seroprevalence. Hepatol Baltim Md. 2013. April;57(4):1333–42.  [DOI](https://doi.org/10.1002/hep.26141) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23172780/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hepatol%20Baltim%20Md&title=Global%20epidemiology%20of%20hepatitis%20C%20virus%20infection:%20new%20estimates%20of%20age-specific%20antibody%20to%20HCV%20seroprevalence&author=K%20Mohd%20Hanafiah&author=J%20Groeger&author=AD%20Flaxman&author=ST%20Wiersma&volume=57&issue=4&publication_year=2013&pages=1333-42&pmid=23172780&doi=10.1002/hep.26141&)

2. Cornberg M, Razavi HA, Alberti A, Bernasconi E, Buti M, Cooper C, et al. A systematic review of hepatitis C virus epidemiology in Europe, Canada and Israel. Liver Int Off J Int Assoc Study Liver. 2011. July;31 Suppl 2:30–60.  [DOI](https://doi.org/10.1111/j.1478-3231.2011.02539.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21651702/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Liver%20Int%20Off%20J%20Int%20Assoc%20Study%20Liver&title=A%20systematic%20review%20of%20hepatitis%20C%20virus%20epidemiology%20in%20Europe,%20Canada%20and%20Israel&author=M%20Cornberg&author=HA%20Razavi&author=A%20Alberti&author=E%20Bernasconi&author=M%20Buti&volume=31&issue=Suppl%202&publication_year=2011&pages=30-60&pmid=21651702&doi=10.1111/j.1478-3231.2011.02539.x&)

3. Hiraga N, Suzuki F, Akuta N, Suzuki Y, Sezaki H, Hosaka T, et al. Clinical and virological characteristics of untreated patients with chronic hepatitis C who develop serum alanine aminotransferase flare-up. J Med Virol. 2005. February;75(2):240–8.  [DOI](https://doi.org/10.1002/jmv.20263) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15602722/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Med%20Virol&title=Clinical%20and%20virological%20characteristics%20of%20untreated%20patients%20with%20chronic%20hepatitis%20C%20who%20develop%20serum%20alanine%20aminotransferase%20flare-up&author=N%20Hiraga&author=F%20Suzuki&author=N%20Akuta&author=Y%20Suzuki&author=H%20Sezaki&volume=75&issue=2&publication_year=2005&pages=240-8&pmid=15602722&doi=10.1002/jmv.20263&)

4. Bochud P-Y, Cai T, Overbeck K, Bochud M, Dufour J-F, Müllhaupt B, et al. Genotype 3 is associated with accelerated fibrosis progression in chronic hepatitis C. J Hepatol. 2009. October;51(4):655–66. 10.1016/j.jhep.2009.05.016  [DOI](https://doi.org/10.1016/j.jhep.2009.05.016) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19665246/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Hepatol&title=Genotype%203%20is%20associated%20with%20accelerated%20fibrosis%20progression%20in%20chronic%20hepatitis%20C&author=P-Y%20Bochud&author=T%20Cai&author=K%20Overbeck&author=M%20Bochud&author=J-F%20Dufour&volume=51&issue=4&publication_year=2009&pages=655-66&pmid=19665246&doi=10.1016/j.jhep.2009.05.016&)

5. Cornberg M, Hüppe D, Wiegand J, Felten G, Wedemeyer H, Manns MP. [Treatment of chronic hepatitis C with PEG-interferon alpha-2b and ribavirin: 24 weeks of therapy are sufficient for HCV genotype 2 and 3]. Z Für Gastroenterol. 2003. June;41(6):517–22.  [DOI](https://doi.org/10.1055/s-2003-39649) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12806536/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Z%20F%C3%BCr%20Gastroenterol&title=Treatment%20of%20chronic%20hepatitis%20C%20with%20PEG-interferon%20alpha-2b%20and%20ribavirin:%2024%20weeks%20of%20therapy%20are%20sufficient%20for%20HCV%20genotype%202%20and%203&author=M%20Cornberg&author=D%20H%C3%BCppe&author=J%20Wiegand&author=G%20Felten&author=H%20Wedemeyer&volume=41&issue=6&publication_year=2003&pages=517-22&pmid=12806536&doi=10.1055/s-2003-39649&)

6. Manns MP, Wedemeyer H, Cornberg M. Treating viral hepatitis C: efficacy, side effects, and complications. Gut. 2006. September;55(9):1350–9.  [DOI](https://doi.org/10.1136/gut.2005.076646) | [PMC free article](/articles/PMC1860034/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16905701/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gut&title=Treating%20viral%20hepatitis%20C:%20efficacy,%20side%20effects,%20and%20complications&author=MP%20Manns&author=H%20Wedemeyer&author=M%20Cornberg&volume=55&issue=9&publication_year=2006&pages=1350-9&pmid=16905701&doi=10.1136/gut.2005.076646&)

7. Munir S, Saleem S, Idrees M, Tariq A, Butt S, Rauff B, et al. Hepatitis C treatment: current and future perspectives. Virol J. 2010;7:296 10.1186/1743-422X-7-296  [DOI](https://doi.org/10.1186/1743-422X-7-296) | [PMC free article](/articles/PMC2984595/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21040548/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Virol%20J&title=Hepatitis%20C%20treatment:%20current%20and%20future%20perspectives&author=S%20Munir&author=S%20Saleem&author=M%20Idrees&author=A%20Tariq&author=S%20Butt&volume=7&publication_year=2010&pages=296&pmid=21040548&doi=10.1186/1743-422X-7-296&)

8. Ferenci P. Response guided therapy in patients with chronic hepatitis C—yesterday, today and tomorrow. Best Pract Res Clin Gastroenterol. 2012. August;26(4):463–9. 10.1016/j.bpg.2012.09.002  [DOI](https://doi.org/10.1016/j.bpg.2012.09.002) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23199505/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Best%20Pract%20Res%20Clin%20Gastroenterol&title=Response%20guided%20therapy%20in%20patients%20with%20chronic%20hepatitis%20C%E2%80%94yesterday,%20today%20and%20tomorrow&author=P%20Ferenci&volume=26&issue=4&publication_year=2012&pages=463-9&pmid=23199505&doi=10.1016/j.bpg.2012.09.002&)

9. Ge D, Fellay J, Thompson AJ, Simon JS, Shianna KV, Urban TJ, et al. Genetic variation in IL28B predicts hepatitis C treatment-induced viral clearance. Nature. 2009. September 17;461(7262):399–401. 10.1038/nature08309  [DOI](https://doi.org/10.1038/nature08309) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19684573/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Genetic%20variation%20in%20IL28B%20predicts%20hepatitis%20C%20treatment-induced%20viral%20clearance&author=D%20Ge&author=J%20Fellay&author=AJ%20Thompson&author=JS%20Simon&author=KV%20Shianna&volume=461&issue=7262&publication_year=2009&pages=399-401&pmid=19684573&doi=10.1038/nature08309&)

10. Thompson AJ, Muir AJ, Sulkowski MS, Ge D, Fellay J, Shianna KV, et al. Interleukin-28B polymorphism improves viral kinetics and is the strongest pretreatment predictor of sustained virologic response in genotype 1 hepatitis C virus. Gastroenterology. 2010. July;139(1):120–9.e18.  [DOI](https://doi.org/10.1053/j.gastro.2010.04.013) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20399780/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=C%20virus.%20Gastroenterology&title=Interleukin-28B%20polymorphism%20improves%20viral%20kinetics%20and%20is%20the%20strongest%20pretreatment%20predictor%20of%20sustained%20virologic%20response%20in%20genotype%201%20hepatitis&author=AJ%20Thompson&author=AJ%20Muir&author=MS%20Sulkowski&author=D%20Ge&author=J%20Fellay&volume=139&issue=1&publication_year=2010&pages=120-9.e18&pmid=20399780&doi=10.1053/j.gastro.2010.04.013&)

11. Suppiah V, Moldovan M, Ahlenstiel G, Berg T, Weltman M, Abate ML, et al. IL28B is associated with response to chronic hepatitis C interferon-alpha and ribavirin therapy. Nat Genet. 2009. October;41(10):1100–4. 10.1038/ng.447  [DOI](https://doi.org/10.1038/ng.447) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19749758/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=IL28B%20is%20associated%20with%20response%20to%20chronic%20hepatitis%20C%20interferon-alpha%20and%20ribavirin%20therapy&author=V%20Suppiah&author=M%20Moldovan&author=G%20Ahlenstiel&author=T%20Berg&author=M%20Weltman&volume=41&issue=10&publication_year=2009&pages=1100-4&pmid=19749758&doi=10.1038/ng.447&)

12. Manns MP, von Hahn T. Novel therapies for hepatitis C—one pill fits all? Nat Rev Drug Discov. 2013. August;12(8):595–610. 10.1038/nrd4050  [DOI](https://doi.org/10.1038/nrd4050) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23807378/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Drug%20Discov&title=Novel%20therapies%20for%20hepatitis%20C%E2%80%94one%20pill%20fits%20all?&author=MP%20Manns&author=T%20von%20Hahn&volume=12&issue=8&publication_year=2013&pages=595-610&pmid=23807378&doi=10.1038/nrd4050&)

13. Heidrich B, Wiegand SB, Buggisch P, Hinrichsen H, Link R, Möller B, et al. Treatment of Naïve Patients with Chronic Hepatitis C Genotypes 2 and 3 with Pegylated Interferon Alpha and Ribavirin in a Real World Setting: Relevance for the New Era of DAA. PloS One. 2014;9(10):e108751 10.1371/journal.pone.0108751  [DOI](https://doi.org/10.1371/journal.pone.0108751) | [PMC free article](/articles/PMC4193764/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25302676/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PloS%20One&title=Treatment%20of%20Na%C3%AFve%20Patients%20with%20Chronic%20Hepatitis%20C%20Genotypes%202%20and%203%20with%20Pegylated%20Interferon%20Alpha%20and%20Ribavirin%20in%20a%20Real%20World%20Setting:%20Relevance%20for%20the%20New%20Era%20of%20DAA&author=B%20Heidrich&author=SB%20Wiegand&author=P%20Buggisch&author=H%20Hinrichsen&author=R%20Link&volume=9&issue=10&publication_year=2014&pages=e108751&pmid=25302676&doi=10.1371/journal.pone.0108751&)

14. Mangia A, Thompson AJ, Santoro R, Piazzolla V, Tillmann HL, Patel K, et al. An IL28B polymorphism determines treatment response of hepatitis C virus genotype 2 or 3 patients who do not achieve a rapid virologic response. Gastroenterology. 2010. September;139(3):821–7, 827.e1. 10.1053/j.gastro.2010.05.079  [DOI](https://doi.org/10.1053/j.gastro.2010.05.079) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20621700/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gastroenterology&title=An%20IL28B%20polymorphism%20determines%20treatment%20response%20of%20hepatitis%20C%20virus%20genotype%202%20or%203%20patients%20who%20do%20not%20achieve%20a%20rapid%20virologic%20response&author=A%20Mangia&author=AJ%20Thompson&author=R%20Santoro&author=V%20Piazzolla&author=HL%20Tillmann&volume=139&issue=3&publication_year=2010&pages=821-7&pmid=20621700&doi=10.1053/j.gastro.2010.05.079&)

15. Sarrazin C, Susser S, Doehring A, Lange CM, Müller T, Schlecker C, et al. Importance of IL28B gene polymorphisms in hepatitis C virus genotype 2 and 3 infected patients. J Hepatol. 2011. March;54(3):415–21. 10.1016/j.jhep.2010.07.041  [DOI](https://doi.org/10.1016/j.jhep.2010.07.041) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21112657/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Hepatol&title=Importance%20of%20IL28B%20gene%20polymorphisms%20in%20hepatitis%20C%20virus%20genotype%202%20and%203%20infected%20patients&author=C%20Sarrazin&author=S%20Susser&author=A%20Doehring&author=CM%20Lange&author=T%20M%C3%BCller&volume=54&issue=3&publication_year=2011&pages=415-21&pmid=21112657&doi=10.1016/j.jhep.2010.07.041&)

16. Stättermayer AF, Stauber R, Hofer H, Rutter K, Beinhardt S, Scherzer TM, et al. Impact of IL28B genotype on the early and sustained virologic response in treatment-naïve patients with chronic hepatitis C. Clin Gastroenterol Hepatol Off Clin Pract J Am Gastroenterol Assoc. 2011. April;9(4):344–50.e2.  [DOI](https://doi.org/10.1016/j.cgh.2010.07.019) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20728570/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Gastroenterol%20Hepatol%20Off%20Clin%20Pract%20J%20Am%20Gastroenterol%20Assoc&title=Impact%20of%20IL28B%20genotype%20on%20the%20early%20and%20sustained%20virologic%20response%20in%20treatment-na%C3%AFve%20patients%20with%20chronic%20hepatitis%20C&author=AF%20St%C3%A4ttermayer&author=R%20Stauber&author=H%20Hofer&author=K%20Rutter&author=S%20Beinhardt&volume=9&issue=4&publication_year=2011&pages=344-50&pmid=20728570&doi=10.1016/j.cgh.2010.07.019&)

17. Moghaddam A, Melum E, Reinton N, Ring-Larsen H, Verbaan H, Bjøro K, et al. IL28B genetic variation and treatment response in patients with hepatitis C virus genotype 3 infection. Hepatol Baltim Md. 2011. March;53(3):746–54.  [DOI](https://doi.org/10.1002/hep.24154) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21374656/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hepatol%20Baltim%20Md&title=IL28B%20genetic%20variation%20and%20treatment%20response%20in%20patients%20with%20hepatitis%20C%20virus%20genotype%203%20infection&author=A%20Moghaddam&author=E%20Melum&author=N%20Reinton&author=H%20Ring-Larsen&author=H%20Verbaan&volume=53&issue=3&publication_year=2011&pages=746-54&pmid=21374656&doi=10.1002/hep.24154&)

18. Eslam M, Leung R, Romero-Gomez M, Mangia A, Irving WL, Sheridan D, et al. IFNL3 polymorphisms predict response to therapy in chronic hepatitis C genotype 2/3 infection. J Hepatol. 2014. August;61(2):235–41. 10.1016/j.jhep.2014.03.039  [DOI](https://doi.org/10.1016/j.jhep.2014.03.039) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24768758/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Hepatol&title=IFNL3%20polymorphisms%20predict%20response%20to%20therapy%20in%20chronic%20hepatitis%20C%20genotype%202/3%20infection&author=M%20Eslam&author=R%20Leung&author=M%20Romero-Gomez&author=A%20Mangia&author=WL%20Irving&volume=61&issue=2&publication_year=2014&pages=235-41&pmid=24768758&doi=10.1016/j.jhep.2014.03.039&)

19. Rangnekar AS, Fontana RJ. IL-28B polymorphisms and the response to antiviral therapy in HCV genotype 2 and 3 varies by ethnicity: a meta-analysis. J Viral Hepat. 2013. June;20(6):377–84. 10.1111/jvh.12039  [DOI](https://doi.org/10.1111/jvh.12039) | [PMC free article](/articles/PMC4551399/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23647954/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Viral%20Hepat&title=IL-28B%20polymorphisms%20and%20the%20response%20to%20antiviral%20therapy%20in%20HCV%20genotype%202%20and%203%20varies%20by%20ethnicity:%20a%20meta-analysis&author=AS%20Rangnekar&author=RJ%20Fontana&volume=20&issue=6&publication_year=2013&pages=377-84&pmid=23647954&doi=10.1111/jvh.12039&)

20. Stenkvist J, Sönnerborg A, Weiland O. HCV RNA decline in chronic HCV genotype 2 and 3 during standard of care treatment according to IL28B polymorphism. J Viral Hepat. 2013. March;20(3):193–9. 10.1111/j.1365-2893.2012.01645.x  [DOI](https://doi.org/10.1111/j.1365-2893.2012.01645.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23383658/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Viral%20Hepat&title=HCV%20RNA%20decline%20in%20chronic%20HCV%20genotype%202%20and%203%20during%20standard%20of%20care%20treatment%20according%20to%20IL28B%20polymorphism&author=J%20Stenkvist&author=A%20S%C3%B6nnerborg&author=O%20Weiland&volume=20&issue=3&publication_year=2013&pages=193-9&pmid=23383658&doi=10.1111/j.1365-2893.2012.01645.x&)

21. Prokunina-Olsson L, Muchmore B, Tang W, Pfeiffer RM, Park H, Dickensheets H, et al. A variant upstream of IFNL3 (IL28B) creating a new interferon gene IFNL4 is associated with impaired clearance of hepatitis C virus. Nat Genet. 2013. February;45(2):164–71. 10.1038/ng.2521  [DOI](https://doi.org/10.1038/ng.2521) | [PMC free article](/articles/PMC3793390/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23291588/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=A%20variant%20upstream%20of%20IFNL3%20(IL28B)%20creating%20a%20new%20interferon%20gene%20IFNL4%20is%20associated%20with%20impaired%20clearance%20of%20hepatitis%20C%20virus&author=L%20Prokunina-Olsson&author=B%20Muchmore&author=W%20Tang&author=RM%20Pfeiffer&author=H%20Park&volume=45&issue=2&publication_year=2013&pages=164-71&pmid=23291588&doi=10.1038/ng.2521&)

22. Susser S, Herrmann E, Lange C, Hamdi N, Müller T, Berg T, et al. Predictive value of interferon-lambda gene polymorphisms for treatment response in chronic hepatitis C. PloS One. 2014;9(11):e112592 10.1371/journal.pone.0112592  [DOI](https://doi.org/10.1371/journal.pone.0112592) | [PMC free article](/articles/PMC4231027/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25393304/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PloS%20One&title=Predictive%20value%20of%20interferon-lambda%20gene%20polymorphisms%20for%20treatment%20response%20in%20chronic%20hepatitis%20C&author=S%20Susser&author=E%20Herrmann&author=C%20Lange&author=N%20Hamdi&author=T%20M%C3%BCller&volume=9&issue=11&publication_year=2014&pages=e112592&pmid=25393304&doi=10.1371/journal.pone.0112592&)

23. Sarrazin C, Berg T, Ross RS, Schirmacher P, Wedemeyer H, Neumann U, et al. Prophylaxis, diagnosis and therapy of hepatitis C virus (HCV) infection: the German guidelines on the management of HCV infection]. Z Für Gastroenterol. 2010. February;48(2):289–351.  [DOI](https://doi.org/10.1055/s-0028-1110008) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20119896/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Z%20F%C3%BCr%20Gastroenterol&title=Prophylaxis,%20diagnosis%20and%20therapy%20of%20hepatitis%20C%20virus%20(HCV)%20infection:%20the%20German%20guidelines%20on%20the%20management%20of%20HCV%20infection%5D&author=C%20Sarrazin&author=T%20Berg&author=RS%20Ross&author=P%20Schirmacher&author=H%20Wedemeyer&volume=48&issue=2&publication_year=2010&pages=289-351&pmid=20119896&doi=10.1055/s-0028-1110008&)

24. Sarrazin C, Shiffman ML, Hadziyannis SJ, Lin A, Colucci G, Ishida H, et al. Definition of rapid virologic response with a highly sensitive real-time PCR-based HCV RNA assay in peginterferon alfa-2a plus ribavirin response-guided therapy. J Hepatol. 2010. June;52(6):832–8. 10.1016/j.jhep.2010.01.030  [DOI](https://doi.org/10.1016/j.jhep.2010.01.030) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20385421/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Hepatol&title=Definition%20of%20rapid%20virologic%20response%20with%20a%20highly%20sensitive%20real-time%20PCR-based%20HCV%20RNA%20assay%20in%20peginterferon%20alfa-2a%20plus%20ribavirin%20response-guided%20therapy&author=C%20Sarrazin&author=ML%20Shiffman&author=SJ%20Hadziyannis&author=A%20Lin&author=G%20Colucci&volume=52&issue=6&publication_year=2010&pages=832-8&pmid=20385421&doi=10.1016/j.jhep.2010.01.030&)

25. Poynard T, Bedossa P, Opolon P. Natural history of liver fibrosis progression in patients with chronic hepatitis C. The OBSVIRC, METAVIR, CLINIVIR, and DOSVIRC groups. Lancet. 1997. March 22;349(9055):825–32.  [DOI](https://doi.org/10.1016/s0140-6736(96)07642-8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9121257/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Natural%20history%20of%20liver%20fibrosis%20progression%20in%20patients%20with%20chronic%20hepatitis%20C.%20The%20OBSVIRC,%20METAVIR,%20CLINIVIR,%20and%20DOSVIRC%20groups&author=T%20Poynard&author=P%20Bedossa&author=P%20Opolon&volume=349&issue=9055&publication_year=1997&pages=825-32&pmid=9121257&doi=10.1016/s0140-6736(96)07642-8&)

26. Knodell RG, Ishak KG, Black WC, Chen TS, Craig R, Kaplowitz N, et al. Formulation and application of a numerical scoring system for assessing histological activity in asymptomatic chronic active hepatitis. Hepatol Baltim Md. 1981. Oct;1(5):431–5.  [DOI](https://doi.org/10.1002/hep.1840010511) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7308988/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hepatol%20Baltim%20Md&title=Formulation%20and%20application%20of%20a%20numerical%20scoring%20system%20for%20assessing%20histological%20activity%20in%20asymptomatic%20chronic%20active%20hepatitis&author=RG%20Knodell&author=KG%20Ishak&author=WC%20Black&author=TS%20Chen&author=R%20Craig&volume=1&issue=5&publication_year=1981&pages=431-5&pmid=7308988&doi=10.1002/hep.1840010511&)

27. Castéra L, Vergniol J, Foucher J, Le Bail B, Chanteloup E, Haaser M, et al. Prospective comparison of transient elastography, Fibrotest, APRI, and liver biopsy for the assessment of fibrosis in chronic hepatitis C. Gastroenterology. 2005. February;128(2):343–50.  [DOI](https://doi.org/10.1053/j.gastro.2004.11.018) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15685546/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gastroenterology&title=Prospective%20comparison%20of%20transient%20elastography,%20Fibrotest,%20APRI,%20and%20liver%20biopsy%20for%20the%20assessment%20of%20fibrosis%20in%20chronic%20hepatitis%20C&author=L%20Cast%C3%A9ra&author=J%20Vergniol&author=J%20Foucher&author=B%20Le%20Bail&author=E%20Chanteloup&volume=128&issue=2&publication_year=2005&pages=343-50&pmid=15685546&doi=10.1053/j.gastro.2004.11.018&)

28. De Ritis F, Coltorti M, Giusti G. Serum-transaminase activities in liver disease. Lancet. 1972. March 25;1(7752):685–7.  [DOI](https://doi.org/10.1016/s0140-6736(72)90487-4) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/4125186/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Serum-transaminase%20activities%20in%20liver%20disease&author=F%20De%20Ritis&author=M%20Coltorti&author=G%20Giusti&volume=1&issue=7752&publication_year=1972&pages=685-7&pmid=4125186&doi=10.1016/s0140-6736(72)90487-4&)

29. Rodriguez S, Gaunt TR, Day INM. Hardy-Weinberg equilibrium testing of biological ascertainment for Mendelian randomization studies. Am J Epidemiol. 2009. February 15;169(4):505–14. 10.1093/aje/kwn359  [DOI](https://doi.org/10.1093/aje/kwn359) | [PMC free article](/articles/PMC2640163/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19126586/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Epidemiol&title=Hardy-Weinberg%20equilibrium%20testing%20of%20biological%20ascertainment%20for%20Mendelian%20randomization%20studies&author=S%20Rodriguez&author=TR%20Gaunt&author=INM%20Day&volume=169&issue=4&publication_year=2009&pages=505-14&pmid=19126586&doi=10.1093/aje/kwn359&)

30. Hadziyannis SJ, Sette H, Morgan TR, Balan V, Diago M, Marcellin P, et al. Peginterferon-alpha2a and ribavirin combination therapy in chronic hepatitis C: a randomized study of treatment duration and ribavirin dose. Ann Intern Med. 2004. March 2;140(5):346–55.  [DOI](https://doi.org/10.7326/0003-4819-140-5-200403020-00010) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14996676/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Intern%20Med&title=Peginterferon-alpha2a%20and%20ribavirin%20combination%20therapy%20in%20chronic%20hepatitis%20C:%20a%20randomized%20study%20of%20treatment%20duration%20and%20ribavirin%20dose&author=SJ%20Hadziyannis&author=H%20Sette&author=TR%20Morgan&author=V%20Balan&author=M%20Diago&volume=140&issue=5&publication_year=2004&pages=346-55&pmid=14996676&doi=10.7326/0003-4819-140-5-200403020-00010&)
