# Genetic variation in the UGT1A locus is associated with simvastatin efficacy in a clinical practice setting

## Metadata
**Authors:** Otito F Iwuchukwu, QiPing Feng, Wei-Qi Wei, Lan Jiang, Min Jiang, Hua Xu, Joshua C Denny, Russell A Wilke, Ronald M Krauss, Dan M Roden, C Michael Stein
**Journal:** Pharmacogenomics
**Date:** 2014 Nov
**DOI:** [10.2217/pgs.14.128](https://doi.org/10.2217/pgs.14.128)
**PMID:** 25493567
**PMCID:** PMC4292894
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4292894/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC4292894/pdf/nihms-652640.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC4292894/pdf/nihms-652640.pdf)

## Abstract

**Aim:** 
Simvastatin is a lactone prodrug that exists in equilibrium with its active hydroxyacid through a process mediated by UGT1A enzymes. The UGT1A locus has been associated with simvastatin response and disposition in humans. Therefore, we fine-mapped the UGT1A locus to identify genetic variations contributing to simvastatin disposition and response variability.

**Methods:** 
Using de-identified electronic medical records linked to a DNA biobank, we extracted information about dose and low-density lipo protein cholesterol (LDL-C) concentrations for patients who received more than two different doses of simvastatin. Pharmacodynamic measures of simvastatin potency and efficacy were calculated from dose–response curves (E0 = baseline LDL-C, ED50 = dose yielding 50% maximum response, and Emax = maximum decrease in LDL-C) in 1100 patients. We selected 153 polymorphisms in UGT1A1 and UGT1A3 for genotyping and conducted genotype–phenotype associations using a prespecified additive model.

**Results:** 
Two variants in UGT1A1 (rs2003569 and rs12052787) were associated with Emax (p = 0.0059 and 0.031, respectively; for rs2003569 the mean Emax was 59.3 ± 23.0, 62.0 ± 22.4, and 69.7 ± 24.8 mg/dl, for patients with 0, 1 or 2 copies of the minor A allele, respectively). When stratified by race, the difference in response was greater in African–Americans than in European Americans. Rs2003569 was also negatively associated with total serum bilirubin levels (p = 7.85 × 10−5). Four rare SNPs were nominally associated with E0 and ED50.

**Conclusion:** 
We identified a UGT1A1 promoter variant (rs2003569) associated with simvastatin efficacy.

Keywords: LDL-cholesterol, pharmacodynamics, pharmacokinetics, promoter variants, simvastatin, UGT1A1

### Aim

Simvastatin is a lactone prodrug that exists in equilibrium with its active hydroxyacid through a process mediated by UGT1A enzymes. The *UGT1A* locus has been associated with simvastatin response and disposition in humans. Therefore, we fine-mapped the *UGT1A* locus to identify genetic variations contributing to simvastatin disposition and response variability.

### Methods

Using de-identified electronic medical records linked to a DNA biobank, we extracted information about dose and low-density lipo protein cholesterol (LDL-C) concentrations for patients who received more than two different doses of simvastatin. Pharmacodynamic measures of simvastatin potency and efficacy were calculated from dose–response curves (E_0_ = baseline LDL-C, ED_50_ = dose yielding 50% maximum response, and E_max_ = maximum decrease in LDL-C) in 1100 patients. We selected 153 polymorphisms in *UGT1A1* and *UGT1A3* for genotyping and conducted genotype–phenotype associations using a prespecified additive model.

### Results

Two variants in *UGT1A1* (rs2003569 and rs12052787) were associated with E_max_ (p = 0.0059 and 0.031, respectively; for rs2003569 the mean E_max_ was 59.3 ± 23.0, 62.0 ± 22.4, and 69.7 ± 24.8 mg/dl, for patients with 0, 1 or 2 copies of the minor A allele, respectively). When stratified by race, the difference in response was greater in African–Americans than in European Americans. Rs2003569 was also negatively associated with total serum bilirubin levels (p = 7.85 × 10^−5^). Four rare SNPs were nominally associated with E_0_ and ED_50_.

### Conclusion

We identified a *UGT1A1* promoter variant (rs2003569) associated with simvastatin efficacy.

## Background

HMG-CoA reductase inhibitors (statins) lower plasma levels of low-density lipo-protein cholesterol (LDL-C) as much as 60% [[1](#R1)]. This LDL-C lowering effect has significant therapeutic benefits, including reduction in cardiovascular events and deaths. As such, statins have become one of the most widely prescribed groups of drugs in the developed world. Simvastatin is one of the most commonly used statins due to its relative potency and favorable pharmacoeconomics [[2](#R2)].

The overall disposition of simvastatin is complex, widely variable, and involves interplay between CYP450 enzymes (CYPs), uridine-diphospho glucuronosyl transferases (UGTs) and drug transporters, all with established functional genetic poly morphisms. There is substantial information about CYP and transporter pharmaco genetics as related to simvastatin disposition and response, but the effects of genetic variation in UGT enzymes are poorly understood. There is interest in defining the underlying variability in genetic architecture that contributes to the wide range of treatment response in patients receiving statins.

Unlike atorvastatin and most other statins, simvastatin is a prodrug, administered as the inactive lactone and then converted to the open hydroxyacid form that is responsible for its lipid lowering effects. This active open hydroxyacid exists in equilibrium with the inactive lactone, through an interconversion mediated both enzymatically and by spontaneous chemical hydrolysis ([Figure 1](#F1)). Simvastatin acid is converted enzymatically to its corresponding lactone through the formation of an acyl glucuronide intermediate [[3](#R3)], a reaction catalyzed by certain members of the UGT superfamily.

### Figure 1. Simvastatin acyl glucuronide formation and interconversion to simvastatin open hydroxy-acid.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1915/4292894/94448f857631/nihms-652640-f0001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4292894_nihms-652640-f0001.jpg)

Proposed metabolism of statins depicting the acid to lactone interconversion by various pathways. Statin lactones are hydrolyzed to the open acid chemically and enzymatically. The statin acids are then rapidly converted to corresponding lactones by glucuronidation (via their unstable acyl glucuronides) and to a lesser extent by a coenzyme-A-dependent pathway. The same considerations apply to the oxidative metabolites of both lactone and hyroxyacid forms of statins. Green text represents active uptake by OATP1B1 (or SLCO1B1) transport proteins. Blue text represents phase I metabolism by CYP3A4 enzymes. Red text represents UGTs responsible for glucuronidation. Adapted from Prueksaritanont et al. [3] with Permission from American Society for Pharmacology and Experimental Therapeutics [ASPET]). Please see color figure at www.futuremedicine.com/doi/pdf/10.2217/pgs.14.128

UGTs are classified into two main families – UGT1 and UGT2 – that are then further subclassified into UGT1A, 1B, 2A and 2B subfamilies with individual enzymes or isoforms usually identified by numbers after the respective subfamily; for example, UGT1A1 and UGT2B7. The main enzymes responsible for simvastatin glucuronidation are UGT1A1 and UGT1A3, both encoded at the *UGT1A* gene locus and differentially expressed in the liver [[3](#R3)]. In addition to these two enzymes, the 200 kb locus encodes nine other active UGT1A gene products and two pseudogenes. Each active *UGT1A* transcript is composed of the same four exons (exons 2–5) spliced to a unique first exon for which the gene product is named (e.g., UGT1A1–1A12) [[4](#R4)–[6](#R6)].

Genetic polymorphisms are reported for almost all of the active *UGT1A* genes, and UGT activity can vary by genotype. A well-known and widely studied *UGT1A1* polymorphism is defined by a variable-length (5–8) dinucleotide TA short tandem repeat (STR) in the *UGT1A1* gene promoter [[7](#R7)–[9](#R9)]. This polymorphism confers several genotypes depending on the number of TA repeats; an increase in the number of TA repeats is associated with decreased promoter activity and reduced protein expression and activity. The importance of this genetic variation for clinical practice is underscored by FDA label changes identifying its con- tribution to UGT1A1 pharmacogenetics as potentially important in the prescribing of irinotecan, nilotinib and indacaterol [[10](#R10)].

In a previous study, we confirmed known associations between key pharmacodynamic and pharmaco kinetic variants and simvastatin efficacy and potency, and also discovered a significant association between simvastatin potency and a *UGT1A1* variant [[11](#R11)]. Therefore, in the present study we use a fine-mapping approach to further define the associations between simvastatin pharmacodynamics and the *UGT1A1* locus.

## Methods

### Study population

From a previous cohort of 2026 subjects in whom we had characterized exposure and response to simvastatin [[11](#R11)], we selected 1152 patients with readily available DNA samples for fine mapping of UGT1A1. We extracted information from the electronic medical records (EMRs) of all patients who had baseline LDL-C levels (defined as no prior statin exposure) and were subsequently exposed to two or more doses of simvastatin with accompanying LDL-C concentrations obtained during routine clinical care at Vanderbilt University Medical Center (VUMC), TN, USA. A de-identified version of a fully integrated (inpatient–outpatient) EMR in a VUMC practice-based setting is available for research use. The clinical practice-based biobank, named BioVU, enables the linking of de-identified DNA samples to a comprehensive de-identified EMR and has extended the use of EMRs as a resource for genetic association studies. BioVU currently serves as a repository for DNA linked to the EMRs for over 170,000 patients, and is the nation's largest clinical practice-based biobank. As BioVU is primarily a regional resource, the majority of records (80%) are from subjects of European ancestry, reflective of the surrounding community's racial makeup. The enrollment and biobanking approach used for BioVU has been published [[12](#R12),[13](#R13)]. The Institutional Review Board of VUMC approved this study.

### Phenotyping

The phenotyping methods used in this study have been validated and described in detail [[14](#R14),[15](#R15)]. Briefly, natural language processing software was used to extract and reconstruct all unstructured retrospective drug exposure history for each patient. We obtained all LDL-C levels from structured laboratory records and plotted these longitudinally alongside simvastatin exposure. We filtered all lipid data and only used steady-state LDL-C levels (obtained after at least a 6-week window following dose initiation or dose changes) for calculations. Median LDL-C values were calculated for each simvastatin dose to construct individual dose–response curves for each patient in the selected population using the pharmacodynamic E_max_ (maximal effect) model. In this model, change in LDL-C is a function of simvastatin dose, with individual parameters varying around a population average.

Parameters estimated for phenotypic traits were: first, E_0_ (baseline LDL-C levels); second, E_max_ (the maximal LDL-C lowering efficacy); and third, ED_50_ (the simvastatin dose required to achieve half-maximal effect for each subject within the study population). Dose–response parameters were estimated for the study subjects using our previously published dose–response equation [[14](#R14),[15](#R15)]. We also extracted total serum bilirubin levels from structured laboratory records (at the time of baseline LDL-C) to be used as a validation pheno-type for individuals genotyped for the UGT1A1 TA repeat polymorphism.

### SNP selection & genotyping

The UGT locus has been extensively studied and sequence data are publicly available. We focused on polymorphisms in the two *UGT1A* genes linked to statin disposition, *UGT1A1* and *UGT1A3*. A systematic search of three different databases: dbSNP (the free public archive for genetic variation) [[16](#R16)], the 1000 Genomes Project variation catalog [[17](#R17)] and the UGT gene database [[18](#R18)], was conducted for all UGT variants. Polymorphisms of interest were selected using various criteria: first, all nonsynonymous SNPs in Exon 1, the promoter region up to 4 kb from the transcription start site and the 3′-UTR region. For *UGT1A1*, this ensures capture of the region surrounding the TATA box and the phenobarbital response enhancer module. Second, common and rare variants in the shared exons that had been previously validated by frequency, function or clinical significance with a minor allele frequency (MAF) cut-off of 0.1%. Third, where applicable, all intronic nonsynonymous SNPs common to the *UGT1A* family with MAF greater than 0.1%, as in criterion 2.

Using the criteria above, we selected 183 SNPs, one deletion polymorphism, and the *UGT1A1* STR for genotyping. Of the183 SNPs, 25 failed either the design or assay phase for the Sequenom MassARRAY® iPlex Gold Assay (Sequenom Inc., CA, USA). Thus, 158 SNPs were multiplexed for subsequent genotyping. The TA repeat polymorphism was genotyped by DNA sequencing followed by fragment analysis using capillary electrophoresis on the Applied Biosystems® 3730 DNA Analyzer (Life Technologies, NY, USA).

### Quality control & statistical analyses

All analyses were conducted using the PLINK genetic analysis toolset version 1.07 [[19](#R19)]. Quality control procedures for genotype data included checking minor allele frequency (MAF), Hardy–Weinberg equilibrium (HWE), mendelian error for HapMap controls, and missingness. We removed markers and samples with missing data. Genotyping was successful (call rate greater than 99%) for 153 of the multiplexed 158 SNPs. We also excluded SNPs that were either monomorphic or deviated significantly from HWE (p < 0.001) leaving a total of 39 SNPs for final analysis. Only samples with available phenotypic information were used in association studies. We conducted the genotype–phenotype association tests for E_0_, ED_50_ and E_max_ using linear regression based on a prespecified additive model and analyzed differences between means by the Tukey–Kramer's *post hoc* comparison test using JMP11 analytical software suite (SAS Institute, NC, USA). Next, we carried out separate regression analysis using third party (administratively) assigned race/ethnicity for Whites (European–American) and Blacks (African–American). The accuracy of such observer-reported ancestry has been shown to cor respond well (greater than 98% concordance) with genetically inferred ancestry in BioVU [[20](#R20)].

Because this study represents the fine mapping of a locus previously associated with simvastatin efficacy [[11](#R11)] all results are presented as unadjusted p-values. By definition, all common and rare variants in this study are correlated across three blocks of linkage disequilibrium.

## Results

### Study cohort

The mean age for the 1100 subjects genotyped (579 females and 521 males) was 69 ± 12.7 years and their race distribution reflects those of the surrounding community ([Table 1](#T1)). Population averages (±SD) for the predefined phenotypic traits of simvastatin response were E_0_ = 137.8 ± 34 mg/dl, E_max_ = 60.3 ± 23.0 mg/dl and ED_50_ = 7.49 ± 3.1 mg/day.

### Table 1.

| Variable | Value |
| --- | --- |
| Age | 69 ± 12.7 years |
| Gender, n (frequencies) |   |
| – Females | 579 (0.53) |
| – Males | 521 (0.47) |
| Race, n (frequencies) |   |
| – European–American | 827 (0.75) |
| – African–American | 222 (0.20) |
| – Asian/others | 51 (0.05) |
| E0 | 137.8 ± 34 mg/dl |
| Emax | 60.3 ± 23.0 mg/dl |
| ED50 | 7.49 ± 3.1 mg/day |

Table 1 Caption: Age, gender and quantitative trait distributions for study subjects, n = 1100.

### Association with UGT genotype

The trait of primary interest, maximal LDL-C lowering or E_max_, was significantly associated (p = 0.0059) with a putative functional variant in the *UGT1A1* promoter, rs2003569 ([Table 2](#T2)). The rs2003569 variant confers a clinically meaningful difference in response; mean E_max_ (±SD) was 59.3 ± 23.0 mg/dl, 62.0 ± 22.4 mg/dl, and 69.7 ± 24.8 mg/dl, for study subjects with 0, 1 or 2 copies of the minor A allele, respectively ([Table 2](#T2) & [Figure 2](#F2)). When stratified by race, the associated difference in response was greater in African–Americans (p = 0.001) than in European–Americans (p = 0.2). The magnitude of response per 0, 1, and 2 copies of the A allele was 54.5 ± 23.6 mg/dl, 60.7 ± 20.4 mg/dl, and 78.4 ± 34.5 mg/dl in African–Americans, respectively, and 60.2 ± 22.8 mg/dl, 62.2 ± 22.3 mg/dl, and 64.2 ± 18.2 mg/dl in European–Americans, respectively ([Table 3](#T3) & [Figure 2](#F2)). Thus, this variant appears predictive of greater simvastatin efficacy, particularly in African–Americans. We also observed a nominally significant association between E_max_ and another UGT1A1 promoter variant, rs12052787 in the entire population, but not in subsequent analysis by race. This SNP is in partial LD with rs2003569 only in HapMap CEU subjects (r^2^ = 0.48) and not in subjects of African ancestry.

### Table 2.

| SNP | MM | mM | mm | p-value |
| --- | --- | --- | --- | --- |
| rs4124874 | 59.54 ± 23.93 | 61.27 ± 23.27 | 59.46 ± 21.03 | 0.949 |
| rs10929302 | 60.89 ± 22.09 | 59.98 ± 23.18 | 58.86 ± 26.32 | 0.345 |
| rs4399719 | 59.82 ± 23.99 | 61.13 ± 23.24 | 59.42 ± 21.02 | 0.909 |
| rs12052787 | 59.77 ± 22.68 | 64.24 ± 24.56 | 61.42 ± 4.617 | 0.037 |
| rs3755319 | 60.24 ± 24.06 | 60.92 ± 23.06 | 59.05 ± 20.98 | 0.546 |
| rs2003569 | 59.27 ± 23.0* | 62.01 ± 22.4 | 69.7 ± 24.8* | 0.0059 |
| rs34916116 | 60.19 ± 23.21 | 61.34 ± 21.04 | 59.33 ± 14.74 | 0.656 |
| rs887829 | 61.09 ± 22.36 | 60.16 ± 22.23 | 58.29 ± 26.92 | 0.203 |

Table 2 Caption: Mean maximum decrease in low-density lipoprotein cholesterol concentrations (Emax) for SNPs in the UGT1A1 promoter with minor allele frequency > 0.05.

### Figure 2. Population distribution of the phenotypic trait Emax by rs2003569 genotype.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1915/4292894/485486966556/nihms-652640-f0002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4292894_nihms-652640-f0002.jpg)

Mean differences by genotype for the phenotypic trait Emax (maximal LDL-C lowering) for the top SNP rs2003569 in (A) the entire cohort, (B) African–Americans and (C) European Americans. *p-value less than 0.05 for difference in means (GG and AA) and †p-value less than 0.05 for difference in means (AG and AA). LDL-C: Low-density lipoprotein cholesterol.

### Table 3.

| Race | MM | mM | mm |
| --- | --- | --- | --- |
| African-Americans | 54.5 ± 23.6* | 60.7 ± 20.4* | 78.4 ± 34.5†,* |
| European-Americans | 60.2 ± 22.8 | 62.2 ± 22.3 | 64.2 ± 18.2 |
| All | 59.27 ± 23.0* | 62.01 ± 22.4 | 69.7 ± 24.8* |

Table 3 Caption: Mean maximum decrease in low-density lipoprotein cholesterol concentrations (Emax) stratified by race for the rs2003569 variant.

Two rare variants were nominally associated with baseline LDL-C (E_0_); a variant in the common intron 3 region, rs10445705 (MAF 0.013), and another in *UGT1A3* exon 1, rs28898619 (MAF 0.0069). Two other rare variants were also nominally associated with ED_50_: rs143033456 (in the shared exon 5) and rs61757317 (a deletion polymorphism in the 3′-UTR region; data not shown). These associations represent very rare SNPs and would need replication in multiple cohorts, as our study was limited in their representation.

The number of *UGT1A1* TA repeats was not significantly associated with E_0_ (p = 0.629), ED_50_ (p = 0.238), or E_max_ (p = 0.166) There was, however, a strong positive association between the number of TA repeats and bilirubin levels (p = 1.43 × 10^−16^).

### Concomitant analysis of key variations in simvastatin pharmacokinetic genes

The full impact of pharmacokinetic (PK) candidate gene variants should be interpreted within the context of variation in the interrelated processes of absorption, distribution, metabolism and elimination [[21](#R21)]. Simvastatin disposition and response is affected not only by phase I (oxidative) and phase II (conjugative) metabolism, but also by variability in membrane transport. For example, the organic anion transporting poly-peptide OATP-1B1 (gene name *SLCO1B1*) expressed on the sinusoidal membrane of hepatocytes facilitates the hepatic uptake of simvastatin acid. A coding variant in *SLCO1B1*, rs4149056, is associated with altered hepatic uptake of simvastatin and differential risk for statin-related muscle toxicity [[22](#R22)–[24](#R24)].

We have previously shown that simvastatin efficacy (defined by E_max_) was associated with functional variants in *CYP3A4* (rs2740574) and *CYP3A5* (rs776746) and that its potency (defined by ED_50_) was associated with *SLCO1B1* (rs4149056) in the current study cohort [[11](#R11)]. The association between simvastatin efficacy and rs776746 and a lack of association with rs4149056 has been reported by others [[25](#R25),[26](#R26)]. It is, thus, possible that our findings with *UGT1A1* variants are driven by an interaction with a variant in these other PK candidate genes. We therefore adjusted the *UGT1A1* analysis for the aforementioned PK gene variants to ascertain if there was an independent association between rs2003569 and simvastatin E_max_. The association between rs2003569 and E_max_ was independent of the three variants previously shown to affect simvastatin response ([Table 4](#T4)). In fact, there was a modest reduction in the p-value with inclusion of the *CYP3A* variants, suggesting that phase I and II variants may have an additive effect.

### Table 4.

| SNP | Gene | X SNP | Gene | p-value |
| --- | --- | --- | --- | --- |
| rs2003569 (A) | UGT1A1 | rs2740574 (G) | CYP3A4 | 0.00219 |
| rs2003569 (A) | UGT1A1 | rs776746 (A) | CYP3A5 | 0.00229 |
| rs2003569 (A) | UGT1A1 | rs4149056 (G) | SLCO1B1 | 0.00628 |

Table 4 Caption: Conditioning analysis for rs2003569 on key simvastatin pharmacokinetic genes/SNPs for effect on Emax.

## Discussion

The major new finding of this study was that rs2003569, a *UGT1A1* promoter variant, is associated with a statistically significant difference in simvastatin efficacy, particularly in African–Americans. Variation in *UGT1A1* is known to have important clinical consequences. For example, structural variations in the promoter region are associated with inborn errors of metabolism such as Crigler-Najjar and Gilbert's syndromes [[8](#R8),[27](#R27),[28](#R28)]. The *UGT1A1* TA repeat promoter polymorphism (rs8175347), a key variation underlying the hyperbilirubinemia observed in Gilbert's syndrome, has an established ethnic difference in allele frequencies, with the 7 and 8 TA repeats or TA(7)/(8) alleles occurring at a higher frequencies in African–Americans compared with European–Americans [[9](#R9)]. This variation has been associated with differences in the phase II metabolism of some anti-cancer and anti-HIV drugs [[29](#R29)–[34](#R34)]. The relationship between genetic variability in *UGT1A1* and measures of simvastatin response has not been defined previously, although there are studies on *UGT1A3* variants and the *UGT1A1*28* polymorphism on atorvastatin disposition [[35](#R35),[36](#R36)].

Because *UGT1A1* is known to contribute to simvastatin disposition both *in vitro* and *in vivo* [[3](#R3),[37](#R37)], it was important to refine the observation from our previous study that a marker in this gene was nominally associated with simvastatin dose response [[11](#R11)]. Thus, we performed the current fine-mapping of the *UGT1A* locus, which identified rs2003569 as a putative functional variant. This variant is a G>A single nucleotide variation with a global MAF of 0.16 and an MAF of 0.12, 0.19 and 0.22 in the CEU, ASW and YRI HapMap populations. Rs2003569 is found approximately 1 kb upstream of the TATA box (and the ATG transcription start site) and approximately 2 kb downstream of the phenobarbital response enhancer module of the *UGT1A1* promoter region ([Figure 3](#F3)). Genetic polymorphisms in these regions have been shown to have significant influences on transcription and gene expression [[38](#R38)]. We found within our entire cohort that individuals with two copies of the rs2003569 minor A allele had a 10 mg/dl increase in E_max_ (or LDL-C lowering). This observation is compatible with a concept of increased metabolism of the inactive lactone to the active open beta-hydroxy acid, due to a gain of function, thus leading to increased efficacy.

### Figure 3. Location of the UGT1A1 variant rs2003569.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1915/4292894/b59da63c2ffa/nihms-652640-f0003.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4292894_nihms-652640-f0003.jpg)

Linearized depiction of the UGT1A1 gene and its location on Chromosome 2 showing the transcribed intronic and exonic regions (exons 2–5 are common for all UGT1A1 gene products) and the approximate location of the rs2003569 variant in relation to the transcription start site and the TA repeat polymorphism, rs8175347. MAF: Minor allele frequency.

The *UGT1A* locus has been studied extensively in pharmacogenetic association (candidate gene and genome wide) studies and the key *UGT1A1* gene variant in this study, rs2003569, has been associated with functional consequences in both GWAS and functional PK studies [[39](#R39),[40](#R40)]. In a population PK study [[40](#R40)], rs2003569 influenced the pharmacokinetics of etravirine (a second-generation non-nucleoside reverse transcriptase inhibitor), where it was associated with higher clearance in a discovery cohort. Again, this finding of increased clearance with the A allele of rs2003569 is compatible with a gain-of-function effect. In a GWAS meta-analysis study, this variant was positively associated with serum bilirubin levels but at a p-value below the predefined GWAS threshold [[39](#R39)]. In our study, however, rs2003569 was negatively associated with total serum bilirubin levels (p = 7.85 × 10^−5^), which is also compatible with a gain-of-function since UGTs are responsible for bili-rubin metabolism and ultimately clearance through the bile. The *UGT1A1* TA repeat polymorphism has consistently been positively associated with bilirubin levels in many studies and a meta-analysis of GWAS studies [[39](#R39)]. The association between this TA repeat polymorphism and bilirubin concentrations was confirmed in our study; however, this variant was not significantly associated with any measures of simvastatin response.

Because simvastatin is a pro-drug, metabolism to the active moiety by UGTs is a key determinant of efficacy. However, this metabolic activation occurs in conjunction with variability in other key PK processes such as absorption, hepatic uptake and distribution that could also alter PD outcomes, a relationship that can be clarified by PK/PD modeling. Such models have been defined for statins and a dynamic systems biology modeling approach with statin bio-transformation by UGTs and CYPs has been used to explain interindividual differences in LDL-C lowering effects [[41](#R41),[42](#R42)]. Models such as these could potentially be further refined by incorporating pharmacogenetic information. In our analysis models, we considered *UGT1A1* alone, and in conjunction with *CYP3A4/5* and *SLCO1B1* key variants, and found that the *UGT1A1* findings were independent of other simvastatin PK genes.

Our study had several limitations. We set out to investigate the effects of rare and common UGT variants on simvastatin dose-response, but our study was underpowered to detect the effects of variants with MAF less than 1%. Another limitation of this study was that we did not functionally characterize the key UGT variants *in vitro* with regard to their putative effects on simvastatin metabolism.

## Conclusion & future perspective

In conclusion, using a retrospective study design, we identified a *UGT1A1* promoter variant significantly associated with simvastatin efficacy. Our findings suggest that this rs2003569 variant may be a gain-of-function allele resulting in increased activity that is particularly important in those of African ancestry. This study highlights the power of large biobanks to quantify the impact of genetic variation on an important clinical trait in the context of routine care. Additionally, it advanced our understanding of the genetic architecture underlying the lipid-lowering efficacy of simvastatin. Future work would be directed at replication of this finding in cohorts of diverse ancestry and assessment of clinical generalizability in the context of cardiovascular event reduction.

**Background**

**Methods**

**Results**

**Conclusion**

**Background**

**Methods**

**Results**

**Conclusion**

## Acknowledgments

This work was supported in part by funding by the NIH through the Vanderbilt CTSA UL1TR000445 and NIH grants U19HL069757, U19 HL065962, GM109145 and T32 GM007569.

## Footnotes

## References

1. Law MR, Wald NJ, Rudnicka AR. Quantifying effect of statins on low density lipoprotein cholesterol, ischaemic heart disease, and stroke: systematic review and meta-analysis. BMJ. 2003;326(7404):1423. doi: 10.1136/bmj.326.7404.1423.  [DOI](https://doi.org/10.1136/bmj.326.7404.1423) | [PMC free article](/articles/PMC162260/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12829554/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMJ&title=Quantifying%20effect%20of%20statins%20on%20low%20density%20lipoprotein%20cholesterol,%20ischaemic%20heart%20disease,%20and%20stroke:%20systematic%20review%20and%20meta-analysis.&author=MR%20Law&author=NJ%20Wald&author=AR%20Rudnicka&volume=326&issue=7404&publication_year=2003&pages=1423&pmid=12829554&doi=10.1136/bmj.326.7404.1423&)

2. Evaluating statin drugs to treat high cholesterol and heart disease. www.consumerreports.org/health/resources/pdf/best-buy-drugs/StatinsUpdate-FINAL.pdf.  [www.consumerreports.org/health/resources/pdf/best-buy-drugs/StatinsUpdate-FINAL.pdf](http://www.consumerreports.org/health/resources/pdf/best-buy-drugs/StatinsUpdate-FINAL.pdf)

3. Prueksaritanont T, Subramanian R, Fang X, et al. Glucuronidation of statins in animals and humans: a novel mechanism of statin lactonization. Drug Metab. Dispos. 2002;30(5):505–512. doi: 10.1124/dmd.30.5.505. [First study to show formation and quantification of acyl glucuronide conjugates of the hydroxy acid forms of simvastatin and two other statins in humans and animals.]  [DOI](https://doi.org/10.1124/dmd.30.5.505) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11950779/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab.%20Dispos&title=Glucuronidation%20of%20statins%20in%20animals%20and%20humans:%20a%20novel%20mechanism%20of%20statin%20lactonization.&author=T%20Prueksaritanont&author=R%20Subramanian&author=X%20Fang&volume=30&issue=5&publication_year=2002&pages=505-512&pmid=11950779&doi=10.1124/dmd.30.5.505&)

4. Mackenzie PI, Owens IS, Burchell B, et al. The UDP glycosyltransferase gene superfamily: recommended nomenclature update based on evolutionary divergence. Pharmacogenetics. 1997;7(4):255–269. doi: 10.1097/00008571-199708000-00001.  [DOI](https://doi.org/10.1097/00008571-199708000-00001) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9295054/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=The%20UDP%20glycosyltransferase%20gene%20superfamily:%20recommended%20nomenclature%20update%20based%20on%20evolutionary%20divergence.&author=PI%20Mackenzie&author=IS%20Owens&author=B%20Burchell&volume=7&issue=4&publication_year=1997&pages=255-269&pmid=9295054&doi=10.1097/00008571-199708000-00001&)

5. Owens IS, Ritter JK. Gene structure at the human UGT1 locus creates diversity in isozyme structure, substrate specificity, and regulation. Prog. Nucleic Acid Res. Mol. Biol. 1995;51:305–338. doi: 10.1016/s0079-6603(08)60882-x.  [DOI](https://doi.org/10.1016/s0079-6603(08)60882-x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7659777/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Prog.%20Nucleic%20Acid%20Res.%20Mol.%20Biol&title=Gene%20structure%20at%20the%20human%20UGT1%20locus%20creates%20diversity%20in%20isozyme%20structure,%20substrate%20specificity,%20and%20regulation.&author=IS%20Owens&author=JK%20Ritter&volume=51&publication_year=1995&pages=305-338&pmid=7659777&doi=10.1016/s0079-6603(08)60882-x&)

6. Ritter JK, Chen F, Sheen YY, et al. A novel complex locus UGT1 encodes human bilirubin, phenol, and other UDP-glucuronosyltransferase isozymes with identical carboxyl termini. J. Biol. Chem. 1992;267(5):3257–3261.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/1339448/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem&title=A%20novel%20complex%20locus%20UGT1%20encodes%20human%20bilirubin,%20phenol,%20and%20other%20UDP-glucuronosyltransferase%20isozymes%20with%20identical%20carboxyl%20termini.&author=JK%20Ritter&author=F%20Chen&author=YY%20Sheen&volume=267&issue=5&publication_year=1992&pages=3257-3261&pmid=1339448&)

7. Beutler E, Gelbart T, Demina A. Racial variability in the UDP-glucuronosyltransferase 1 (UGT1A1) promoter: a balanced polymorphism for regulation of bilirubin metabolism? Proc. Natl Acad. Sci. USA. 1998;95(14):8170–8174. doi: 10.1073/pnas.95.14.8170. [First to show the racial variability in carriers and homozygotes for the UGT1A1*28 variant.]  [DOI](https://doi.org/10.1073/pnas.95.14.8170) | [PMC free article](/articles/PMC20948/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9653159/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl%20Acad.%20Sci.%20USA&title=Racial%20variability%20in%20the%20UDP-glucuronosyltransferase%201%20(UGT1A1)%20promoter:%20a%20balanced%20polymorphism%20for%20regulation%20of%20bilirubin%20metabolism?&author=E%20Beutler&author=T%20Gelbart&author=A%20Demina&volume=95&issue=14&publication_year=1998&pages=8170-8174&pmid=9653159&doi=10.1073/pnas.95.14.8170&)

8. Bosma PJ, Chowdhury JR, Bakker C, et al. The genetic basis of the reduced expression of bilirubin UDP-glucuronosyltransferase 1 in Gilbert's syndrome. N. Engl. J. Med. 1995;333(18):1171–1175. doi: 10.1056/NEJM199511023331802. [Seminal paper describing the genetic basis for the reduced expression of UGT protein and activity seen with the UGT1A1*28 variant in patients with Gilbert’s syndrome.]  [DOI](https://doi.org/10.1056/NEJM199511023331802) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7565971/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N.%20Engl.%20J.%20Med&title=The%20genetic%20basis%20of%20the%20reduced%20expression%20of%20bilirubin%20UDP-glucuronosyltransferase%201%20in%20Gilbert's%20syndrome.&author=PJ%20Bosma&author=JR%20Chowdhury&author=C%20Bakker&volume=333&issue=18&publication_year=1995&pages=1171-1175&pmid=7565971&doi=10.1056/NEJM199511023331802&)

9. Williams JA, Andersson T, Andersson TB, et al. PhRMA white paper on ADME pharmacogenomics. J. Clin. Pharmacol. 2008;48(7):849–889. doi: 10.1177/0091270008319329.  [DOI](https://doi.org/10.1177/0091270008319329) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18524998/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Pharmacol&title=PhRMA%20white%20paper%20on%20ADME%20pharmacogenomics.&author=JA%20Williams&author=T%20Andersson&author=TB%20Andersson&volume=48&issue=7&publication_year=2008&pages=849-889&pmid=18524998&doi=10.1177/0091270008319329&)

10. Table of Pharmacogenomic Biomarkers in Drug Labeling. www.fda.gov/drugs/scienceresearch/researchareas/ pharmacogenetics/ucm083378.htm.  [www.fda.gov/drugs/scienceresearch/researchareas/ pharmacogenetics/ucm083378.htm](http://www.fda.gov/drugs/scienceresearch/researchareas/pharmacogenetics/ucm083378.htm)

11. Wei WQ, Feng Q, Jiang L, et al. Characterization of statin dose response in electronic medical records. Clin. Pharmacol. Ther. 2013;95(3):331–338. doi: 10.1038/clpt.2013.202. [Characterized and replicated some of the genetic basis of statin dose responses in the electronic medical records of two major medical centers.]  [DOI](https://doi.org/10.1038/clpt.2013.202) | [PMC free article](/articles/PMC3944214/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24096969/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther&title=Characterization%20of%20statin%20dose%20response%20in%20electronic%20medical%20records.&author=WQ%20Wei&author=Q%20Feng&author=L%20Jiang&volume=95&issue=3&publication_year=2013&pages=331-338&pmid=24096969&doi=10.1038/clpt.2013.202&)

12. Roden DM, Pulley JM, Basford MA, et al. Development of a large-scale de-identified DNA biobank to enable personalized medicine. Clin. Pharmacol. Ther. 2008;84(3):362–369. doi: 10.1038/clpt.2008.89.  [DOI](https://doi.org/10.1038/clpt.2008.89) | [PMC free article](/articles/PMC3763939/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18500243/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther&title=Development%20of%20a%20large-scale%20de-identified%20DNA%20biobank%20to%20enable%20personalized%20medicine.&author=DM%20Roden&author=JM%20Pulley&author=MA%20Basford&volume=84&issue=3&publication_year=2008&pages=362-369&pmid=18500243&doi=10.1038/clpt.2008.89&)

13. Pulley J, Clayton E, Bernard GR, Roden DM, Masys DR. Principles of human subjects protections applied in an optout, de-identified biobank. Clin. Transl. Sci. 2010;3(1):42–48. doi: 10.1111/j.1752-8062.2010.00175.x.  [DOI](https://doi.org/10.1111/j.1752-8062.2010.00175.x) | [PMC free article](/articles/PMC3075971/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20443953/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Transl.%20Sci&title=Principles%20of%20human%20subjects%20protections%20applied%20in%20an%20optout,%20de-identified%20biobank.&author=J%20Pulley&author=E%20Clayton&author=GR%20Bernard&author=DM%20Roden&author=DR%20Masys&volume=3&issue=1&publication_year=2010&pages=42-48&pmid=20443953&doi=10.1111/j.1752-8062.2010.00175.x&)

14. Peissig P, Sirohi E, Berg RL, et al. Construction of atorvastatin dose-response relationships using data from a large population-based DNA biobank. Basic Clin. Pharmacol. Toxicol. 2007;100(4):286–288. doi: 10.1111/j.1742-7843.2006.00035.x.  [DOI](https://doi.org/10.1111/j.1742-7843.2006.00035.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17371534/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Basic%20Clin.%20Pharmacol.%20Toxicol&title=Construction%20of%20atorvastatin%20dose-response%20relationships%20using%20data%20from%20a%20large%20population-based%20DNA%20biobank.&author=P%20Peissig&author=E%20Sirohi&author=RL%20Berg&volume=100&issue=4&publication_year=2007&pages=286-288&pmid=17371534&doi=10.1111/j.1742-7843.2006.00035.x&)

15. Wilke RA, Berg RL, Linneman JG, Zhao C, Mccarty CA, Krauss RM. Characterization of low-density lipoprotein cholesterol-lowering efficacy for atorvastatin in a population-based DNA biorepository. Basic Clin. Pharmacol. Toxicol. 2008;103(4):354–359. doi: 10.1111/j.1742-7843.2008.00291.x.  [DOI](https://doi.org/10.1111/j.1742-7843.2008.00291.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18834356/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Basic%20Clin.%20Pharmacol.%20Toxicol&title=Characterization%20of%20low-density%20lipoprotein%20cholesterol-lowering%20efficacy%20for%20atorvastatin%20in%20a%20population-based%20DNA%20biorepository.&author=RA%20Wilke&author=RL%20Berg&author=JG%20Linneman&author=C%20Zhao&author=CA%20Mccarty&volume=103&issue=4&publication_year=2008&pages=354-359&pmid=18834356&doi=10.1111/j.1742-7843.2008.00291.x&)

16. dbSNP Home Page. www.ncbi.nlm.nih.gov/projects/SNP.  [www.ncbi.nlm.nih.gov/projects/SNP](http://www.ncbi.nlm.nih.gov/projects/SNP)

17. 1000 Genomes. http://browser.1000genomes.org.  [http://browser.1000genomes.org](http://browser.1000genomes.org)

18. University of Laval Pharmacogenomics Laboratory; www.pharmacogenomics.pha.ulaval.ca.  [www.pharmacogenomics.pha.ulaval.ca](http://www.pharmacogenomics.pha.ulaval.ca) | [Google Scholar](https://scholar.google.com/scholar_lookup?University%20of%20Laval%20Pharmacogenomics%20Laboratory;%20www.pharmacogenomics.pha.ulaval.ca.)

19. PLINK. http://pngu.mgh.harvard.edu/~purcell/plink.  [http://pngu.mgh.harvard.edu/~purcell/plink](http://pngu.mgh.harvard.edu/~purcell/plink)

20. Dumitrescu L, Ritchie MD, Brown-Gentry K, et al. Assessing the accuracy of observer-reported ancestry in a biorepository linked to electronic medical records. Genet. Med. 2010;12(10):648–650. doi: 10.1097/GIM.0b013e3181efe2df.  [DOI](https://doi.org/10.1097/GIM.0b013e3181efe2df) | [PMC free article](/articles/PMC2952033/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20733501/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genet.%20Med&title=Assessing%20the%20accuracy%20of%20observer-reported%20ancestry%20in%20a%20biorepository%20linked%20to%20electronic%20medical%20records.&author=L%20Dumitrescu&author=MD%20Ritchie&author=K%20Brown-Gentry&volume=12&issue=10&publication_year=2010&pages=648-650&pmid=20733501&doi=10.1097/GIM.0b013e3181efe2df&)

21. Wilke RA, Reif DM, Moore JH. Combinatorial pharmacogenetics. Nat. Rev. Drug Discov. 2005;4(11):911–918. doi: 10.1038/nrd1874.  [DOI](https://doi.org/10.1038/nrd1874) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16264434/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Drug%20Discov&title=Combinatorial%20pharmacogenetics.&author=RA%20Wilke&author=DM%20Reif&author=JH%20Moore&volume=4&issue=11&publication_year=2005&pages=911-918&pmid=16264434&doi=10.1038/nrd1874&)

22. Romaine SP, Bailey KM, Hall AS, Balmforth AJ. The influence of SLCO1B1 (OATP1B1) gene polymorphisms on response to statin therapy. Pharmacogenomics J. 2010;10(1):1–11. doi: 10.1038/tpj.2009.54.  [DOI](https://doi.org/10.1038/tpj.2009.54) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19884908/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=The%20influence%20of%20SLCO1B1%20(OATP1B1)%20gene%20polymorphisms%20on%20response%20to%20statin%20therapy.&author=SP%20Romaine&author=KM%20Bailey&author=AS%20Hall&author=AJ%20Balmforth&volume=10&issue=1&publication_year=2010&pages=1-11&pmid=19884908&doi=10.1038/tpj.2009.54&)

23. Voora D, Shah SH, Spasojevic I, et al. The SLCO1B1*5 genetic variant is associated with statin-induced side effects. J. Am. Coll. Cardiol. 2009;54(17):1609–1616. doi: 10.1016/j.jacc.2009.04.053. [Identified the SLCO1B1*5 variant as the key variant involved in statin muscle toxicity.]  [DOI](https://doi.org/10.1016/j.jacc.2009.04.053) | [PMC free article](/articles/PMC3417133/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19833260/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Am.%20Coll.%20Cardiol&title=The%20SLCO1B1*5%20genetic%20variant%20is%20associated%20with%20statin-induced%20side%20effects.&author=D%20Voora&author=SH%20Shah&author=I%20Spasojevic&volume=54&issue=17&publication_year=2009&pages=1609-1616&pmid=19833260&doi=10.1016/j.jacc.2009.04.053&)

24. Wilke RA, Ramsey LB, Johnson SG, et al. The clinical pharmacogenomics implementation consortium: CPIC guideline for SLCO1B1 and simvastatin-induced myopathy. Clin. Pharmacol. Ther. 2012;92(1):112–117. doi: 10.1038/clpt.2012.57. [Published guidelines for the recommendation and use of SLCO1B1 (rs4149056) genotyping in the clinic.]  [DOI](https://doi.org/10.1038/clpt.2012.57) | [PMC free article](/articles/PMC3384438/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22617227/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther&title=The%20clinical%20pharmacogenomics%20implementation%20consortium:%20CPIC%20guideline%20for%20SLCO1B1%20and%20simvastatin-induced%20myopathy.&author=RA%20Wilke&author=LB%20Ramsey&author=SG%20Johnson&volume=92&issue=1&publication_year=2012&pages=112-117&pmid=22617227&doi=10.1038/clpt.2012.57&)

25. Giannakopoulou E, Ragia G, Kolovou V, et al. No impact of SLCO1B1 521T>C, 388A>G and 411G>A polymorphisms on response to statin therapy in the Greek population. Mol. Biol. Reports. 2014;41(7):4631–4638. doi: 10.1007/s11033-014-3334-z.  [DOI](https://doi.org/10.1007/s11033-014-3334-z) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24668570/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol.%20Biol.%20Reports&title=No%20impact%20of%20SLCO1B1%20521T>C,%20388A>G%20and%20411G>A%20polymorphisms%20on%20response%20to%20statin%20therapy%20in%20the%20Greek%20population.&author=E%20Giannakopoulou&author=G%20Ragia&author=V%20Kolovou&volume=41&issue=7&publication_year=2014&pages=4631-4638&pmid=24668570&doi=10.1007/s11033-014-3334-z&)

26. Kolovou G, Ragia G, Kolovou V, et al. Impact of CYP3A5 gene polymorphism on efficacy of simvastatin. Open Cardiovasc. Med. J. 2014;8:12–17. doi: 10.2174/1874192401408010012.  [DOI](https://doi.org/10.2174/1874192401408010012) | [PMC free article](/articles/PMC3959175/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24653785/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Open%20Cardiovasc.%20Med.%20J&title=Impact%20of%20CYP3A5%20gene%20polymorphism%20on%20efficacy%20of%20simvastatin.&author=G%20Kolovou&author=G%20Ragia&author=V%20Kolovou&volume=8&publication_year=2014&pages=12-17&pmid=24653785&doi=10.2174/1874192401408010012&)

27. Tukey R, Strassburg C. Human UDP-glucuronosyltransferases: metabolism, expression, and disease. Annu. Rev. Pharmacol. Toxicol. 2000;40:581–616. doi: 10.1146/annurev.pharmtox.40.1.581.  [DOI](https://doi.org/10.1146/annurev.pharmtox.40.1.581) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10836148/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Annu.%20Rev.%20Pharmacol.%20Toxicol&title=Human%20UDP-glucuronosyltransferases:%20metabolism,%20expression,%20and%20disease.&author=R%20Tukey&author=C%20Strassburg&volume=40&publication_year=2000&pages=581-616&pmid=10836148&doi=10.1146/annurev.pharmtox.40.1.581&)

28. Ehmer U, Kalthoff S, Fakundiny B, et al. Gilbert syndrome redefined: a complex genetic haplotype influences the regulation of glucuronidation. Hepatology. 2012;55(6):1912–1921. doi: 10.1002/hep.25561.  [DOI](https://doi.org/10.1002/hep.25561) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22213127/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hepatology&title=Gilbert%20syndrome%20redefined:%20a%20complex%20genetic%20haplotype%20influences%20the%20regulation%20of%20glucuronidation.&author=U%20Ehmer&author=S%20Kalthoff&author=B%20Fakundiny&volume=55&issue=6&publication_year=2012&pages=1912-1921&pmid=22213127&doi=10.1002/hep.25561&)

29. Palomaki GE, Bradley LA, Douglas MP, Kolor K, Dotson WD. Can UGT1A1 genotyping reduce morbidity and mortality in patients with metastatic colorectal cancer treated with irinotecan? An evidence-based review. Genet. Med. 2009;11(1):21–34. doi: 10.1097/GIM.0b013e31818efd77.  [DOI](https://doi.org/10.1097/GIM.0b013e31818efd77) | [PMC free article](/articles/PMC2743611/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19125129/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genet.%20Med&title=Can%20UGT1A1%20genotyping%20reduce%20morbidity%20and%20mortality%20in%20patients%20with%20metastatic%20colorectal%20cancer%20treated%20with%20irinotecan?%20An%20evidence-based%20review.&author=GE%20Palomaki&author=LA%20Bradley&author=MP%20Douglas&author=K%20Kolor&author=WD%20Dotson&volume=11&issue=1&publication_year=2009&pages=21-34&pmid=19125129&doi=10.1097/GIM.0b013e31818efd77&)

30. Iyer L, Das S, Janisch L, et al. UGT1A1*28 polymorphism as a determinant of irinotecan disposition and toxicity. Pharmacogenomics J. 2002;2(1):43–47. doi: 10.1038/sj.tpj.6500072.  [DOI](https://doi.org/10.1038/sj.tpj.6500072) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11990381/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=UGT1A1*28%20polymorphism%20as%20a%20determinant%20of%20irinotecan%20disposition%20and%20toxicity.&author=L%20Iyer&author=S%20Das&author=L%20Janisch&volume=2&issue=1&publication_year=2002&pages=43-47&pmid=11990381&doi=10.1038/sj.tpj.6500072&)

31. Dias MM, Mckinnon RA, Sorich MJ. Impact of the UGT1A1*28 allele on response to irinotecan: a systematic review and meta-analysis. Pharmacogenomics. 2012;13(8):889–899. doi: 10.2217/pgs.12.68.  [DOI](https://doi.org/10.2217/pgs.12.68) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22676194/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Impact%20of%20the%20UGT1A1*28%20allele%20on%20response%20to%20irinotecan:%20a%20systematic%20review%20and%20meta-analysis.&author=MM%20Dias&author=RA%20Mckinnon&author=MJ%20Sorich&volume=13&issue=8&publication_year=2012&pages=889-899&pmid=22676194&doi=10.2217/pgs.12.68&)

32. Lubomirov R, Colombo S, Di Iulio J, et al. Association of pharmacogenetic markers with premature discontinuation of first-line anti-HIV therapy: an observational cohort study. J. Infect. Dis. 2011;203(2):246–257. doi: 10.1093/infdis/jiq043.  [DOI](https://doi.org/10.1093/infdis/jiq043) | [PMC free article](/articles/PMC3071070/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21288825/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Infect.%20Dis&title=Association%20of%20pharmacogenetic%20markers%20with%20premature%20discontinuation%20of%20first-line%20anti-HIV%20therapy:%20an%20observational%20cohort%20study.&author=R%20Lubomirov&author=S%20Colombo&author=J%20Di%20Iulio&volume=203&issue=2&publication_year=2011&pages=246-257&pmid=21288825&doi=10.1093/infdis/jiq043&)

33. Ribaudo HJ, Daar ES, Tierney C, et al. Impact of UGT1A1 Gilbert variant on discontinuation of ritonavir-boosted atazanavir in AIDS Clinical Trials Group study A5202. J. Infect. Dis. 2013;207(3):420–425. doi: 10.1093/infdis/jis690.  [DOI](https://doi.org/10.1093/infdis/jis690) | [PMC free article](/articles/PMC3537445/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23148286/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Infect.%20Dis&title=Impact%20of%20UGT1A1%20Gilbert%20variant%20on%20discontinuation%20of%20ritonavir-boosted%20atazanavir%20in%20AIDS%20Clinical%20Trials%20Group%20study%20A5202.&author=HJ%20Ribaudo&author=ES%20Daar&author=C%20Tierney&volume=207&issue=3&publication_year=2013&pages=420-425&pmid=23148286&doi=10.1093/infdis/jis690&)

34. Rotger M, Taffé P, Bleiber G, et al. Gilbert Syndrome and the development of antiretroviral therapy-associated hyperbilirubinemia. J. Infect. Dis. 2005;192(8):1381–1386. doi: 10.1086/466531.  [DOI](https://doi.org/10.1086/466531) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16170755/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Infect.%20Dis&title=Gilbert%20Syndrome%20and%20the%20development%20of%20antiretroviral%20therapy-associated%20hyperbilirubinemia.&author=M%20Rotger&author=P%20Taff%C3%A9&author=G%20Bleiber&volume=192&issue=8&publication_year=2005&pages=1381-1386&pmid=16170755&doi=10.1086/466531&)

35. Riedmaier S, Klein K, Hofmann U, et al. UDP-glucuronosyltransferase (UGT) polymorphisms affect atorvastatin lactonization in vitro and in vivo. Clin. Pharmacol. Ther. 2010;87(1):65–73. doi: 10.1038/clpt.2009.181.  [DOI](https://doi.org/10.1038/clpt.2009.181) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19794410/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther&title=UDP-glucuronosyltransferase%20(UGT)%20polymorphisms%20affect%20atorvastatin%20lactonization%20in%20vitro%20and%20in%20vivo.&author=S%20Riedmaier&author=K%20Klein&author=U%20Hofmann&volume=87&issue=1&publication_year=2010&pages=65-73&pmid=19794410&doi=10.1038/clpt.2009.181&)

36. Stormo C, Bogsrud MP, Hermann M, et al. UGT1A1*28 is associated with decreased systemic exposure of atorvastatin lactone. Mol. Diag. Ther. 2013;17(4):233–237. doi: 10.1007/s40291-013-0031-x.  [DOI](https://doi.org/10.1007/s40291-013-0031-x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23580084/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol.%20Diag.%20Ther&title=UGT1A1*28%20is%20associated%20with%20decreased%20systemic%20exposure%20of%20atorvastatin%20lactone.&author=C%20Stormo&author=MP%20Bogsrud&author=M%20Hermann&volume=17&issue=4&publication_year=2013&pages=233-237&pmid=23580084&doi=10.1007/s40291-013-0031-x&)

37. Shitara Y, Sugiyama Y. Pharmacokinetic and pharmacodynamic alterations of 3-hydroxy-3-methylglutaryl coenzyme A (HMG-CoA) reductase inhibitors: drug–drug interactions and interindividual differences in transporter and metabolic enzyme functions. Pharmacol. Ther. 2006;112(1):71–105. doi: 10.1016/j.pharmthera.2006.03.003.  [DOI](https://doi.org/10.1016/j.pharmthera.2006.03.003) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16714062/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacol.%20Ther&title=Pharmacokinetic%20and%20pharmacodynamic%20alterations%20of%203-hydroxy-3-methylglutaryl%20coenzyme%20A%20(HMG-CoA)%20reductase%20inhibitors:%20drug%E2%80%93drug%20interactions%20and%20interindividual%20differences%20in%20transporter%20and%20metabolic%20enzyme%20functions.&author=Y%20Shitara&author=Y%20Sugiyama&volume=112&issue=1&publication_year=2006&pages=71-105&pmid=16714062&doi=10.1016/j.pharmthera.2006.03.003&)

38. Li Y, Buckley D, Wang S, Klaassen CD, Zhong XB. Genetic polymorphisms in the TATA box and upstream phenobarbital-responsive enhancer module of the UGT1A1 promoter have combined effects on UDP-glucuronosyltransferase 1A1 transcription mediated by constitutive androstane receptor, pregnane X receptor, or glucocorticoid receptor in human liver. Drug Metab. Dispos. 2009;37(9):1978–1986. doi: 10.1124/dmd.109.027409.  [DOI](https://doi.org/10.1124/dmd.109.027409) | [PMC free article](/articles/PMC2729330/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19541828/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab.%20Dispos&title=Genetic%20polymorphisms%20in%20the%20TATA%20box%20and%20upstream%20phenobarbital-responsive%20enhancer%20module%20of%20the%20UGT1A1%20promoter%20have%20combined%20effects%20on%20UDP-glucuronosyltransferase%201A1%20transcription%20mediated%20by%20constitutive%20androstane%20receptor,%20pregnane%20X%20receptor,%20or%20glucocorticoid%20receptor%20in%20human%20liver.&author=Y%20Li&author=D%20Buckley&author=S%20Wang&author=CD%20Klaassen&author=XB%20Zhong&volume=37&issue=9&publication_year=2009&pages=1978-1986&pmid=19541828&doi=10.1124/dmd.109.027409&)

39. Johnson AD, Kavousi M, Smith AV, et al. Genome-wide association meta-analysis for total serum bilirubin levels. Hum. Mol. Genet. 2009;18(14):2700–2710. doi: 10.1093/hmg/ddp202. [Meta-analysis of three different genome-wide association studies for genetic contributors to variability in serum bilirubin.]  [DOI](https://doi.org/10.1093/hmg/ddp202) | [PMC free article](/articles/PMC2701336/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19414484/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum.%20Mol.%20Genet&title=Genome-wide%20association%20meta-analysis%20for%20total%20serum%20bilirubin%20levels.&author=AD%20Johnson&author=M%20Kavousi&author=AV%20Smith&volume=18&issue=14&publication_year=2009&pages=2700-2710&pmid=19414484&doi=10.1093/hmg/ddp202&)

40. Lubomirov R, Arab-Alameddine M, Rotger M, et al. Pharmacogenetics-based population pharmacokinetic analysis of etravirine in HIV-1 infected individuals. Pharmacogenet. Genomics. 2013;23(1):9–18. doi: 10.1097/FPC.0b013e32835ade82.  [DOI](https://doi.org/10.1097/FPC.0b013e32835ade82) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23111422/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet.%20Genomics&title=Pharmacogenetics-based%20population%20pharmacokinetic%20analysis%20of%20etravirine%20in%20HIV-1%20infected%20individuals.&author=R%20Lubomirov&author=M%20Arab-Alameddine&author=M%20Rotger&volume=23&issue=1&publication_year=2013&pages=9-18&pmid=23111422&doi=10.1097/FPC.0b013e32835ade82&)

41. Faltaos DW, Urien S, Carreau V, et al. Use of an indirect effect model to describe the LDL cholesterol-lowering effect by statins in hypercholesterolaemic patients. Fundam. Clin. Pharmacol. 2006;20(3):321–330. doi: 10.1111/j.1472-8206.2006.00404.x.  [DOI](https://doi.org/10.1111/j.1472-8206.2006.00404.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16671968/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Fundam.%20Clin.%20Pharmacol&title=Use%20of%20an%20indirect%20effect%20model%20to%20describe%20the%20LDL%20cholesterol-lowering%20effect%20by%20statins%20in%20hypercholesterolaemic%20patients.&author=DW%20Faltaos&author=S%20Urien&author=V%20Carreau&volume=20&issue=3&publication_year=2006&pages=321-330&pmid=16671968&doi=10.1111/j.1472-8206.2006.00404.x&)

42. Bucher J, Riedmaier S, Schnabel A, et al. A systems biology approach to dynamic modeling and inter-subject variability of statin pharmacokinetics in human hepatocytes. BMC Syst. Biol. 2011;5:66. doi: 10.1186/1752-0509-5-66. [Demonstrates the successful application of a systems biology modeling approach to explain variability in statin pharmacokinetics in humans.]  [DOI](https://doi.org/10.1186/1752-0509-5-66) | [PMC free article](/articles/PMC3117731/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21548957/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Syst.%20Biol&title=A%20systems%20biology%20approach%20to%20dynamic%20modeling%20and%20inter-subject%20variability%20of%20statin%20pharmacokinetics%20in%20human%20hepatocytes.&author=J%20Bucher&author=S%20Riedmaier&author=A%20Schnabel&volume=5&publication_year=2011&pages=66&pmid=21548957&doi=10.1186/1752-0509-5-66&)
