# The influence of the CYP2C19*10 allele on clopidogrel activation and CYP2C19*2 genotyping

## Metadata
**Authors:** Taimour Langaee, Hao-Jie Zhu, Xinwen Wang, Nihal El Rouby, John S Markowitz, Joyce A Goldstein, Julie A Johnson
**Journal:** Pharmacogenetics and genomics
**Date:** 2014 Aug
**DOI:** [10.1097/FPC.0000000000000068](https://doi.org/10.1097/FPC.0000000000000068)
**PMID:** 24945780
**PMCID:** PMC4090277
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4090277/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC4090277/pdf/nihms608938.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC4090277/pdf/nihms608938.pdf)

## Abstract

The polymorphic hepatic enzyme CYP2C19 catalyzes the metabolism of clinically important drugs such as clopidogrel, proton-pump inhibitors, and others and clinical pharmacogenetic testing for clopidogrel is increasingly common. The CYP2C19*10 SNP is located 1 bp upstream the CYP2C19*2 SNP. Despite the low frequency of the CYP2C19*10 allele, its impact on metabolism of CYP2C19 substrates and CYP2C19*2 genotyping makes it an important SNP to consider for pharmacogenetic testing of CYP2C19. However, the effect of the CYP2C19*10 allele on clopidogrel metabolism has not been explored to date. We measured the enzymatic activity of the CYP2C19.10 protein against clopidogrel. The catalytic activity of CYP2C19.10 in the biotransformation of clopidogrel and 2-oxo-clopidgorel was significantly decreased relative to wild type CYP2C19.1B. We also report that the CYP2C19*10 SNP interferes with the CYP2C19*2 TaqMan® genotyping assay, resulting in miscalling of CYP2C19*10/*2 as CYP2C19*2/*2. Our data provide evidence of CYP2C19.10’s reduced metabolism of clopidogrel and 2-oxo-clopidogrel.

Keywords: CYP2C19*10, Clopidogrel, Pharmacokinetic, Pharmacogenetic, genotyping

## INTRODUCTION

CYP2C19 contains the largest frequency of null alleles among the *CYP2C* gene family as well as various reduced function alleles, and it catalyzes the metabolism of many clinically important drugs. The contribution of CYP2C19 catalytic activity varies between different substrates such as the anticonvulsant mephenytoin, the proton-pump inhibitors omeprazole and lansoprazole, and the bioactivation of antiplatelet drug clopidogrel [[1](#R1)–[3](#R3)]. There are more than 30 variant alleles of the *CYP2C19* gene ([http://www.cypalleles.ki.se/cyp2c19.htm](http://www.cypalleles.ki.se/cyp2c19.htm)). The *CYP2C19*2 and CYP2C19*3* null alleles and the *CYP2C19*17* gain of function allele are the most common *CYP2C19* alleles. The frequency of *CYP2C19*2* allele varies from ~15% for Caucasians and Africans to ~29–35% in Asians [[4](#R4)].

The non-synonymous *CYP2C19*10* SNP rs6413438 (C680T, Pro227Leu) is located 1 bp upstream of the null *CYP2C19*2 SNP* rs4244285 (G681A, a cryptic splice site). The *CYP2C19*10* SNP codes for a reduced function allelic protein that was first reported by Blaisdell et al. 2002. The lower *in vitro* metabolic activity of recombinant CYP2C19.10 protein against the two commonly used probe substrates of CYP2C19, S-mephenytoin and omeprazole, has been previously reported [[1](#R1), [2](#R2)]. The effect of the *CYP2C19.10* allelic protein on clopidogrel metabolism has not been evaluated. In this study, we measured the enzymatic activity of recombinant CYP2C19.10 protein on clopidogrel, S-mephenytoin and omeprazole *in vitro*.

It has also been reported that the presence of *CYP2C19*10* SNP before *CYP2C19*2* SNP on exon 5 of *CYP2C19* gene interferes with *CYP2C19*2* genotyping in a PCR-RFLP assay [[5](#R5)] and the INFINITI *CYP2C19*2* genotyping assay ([www.autogenomics.com/pdf/INFINITICYP2C19PackageInsertUSA](http://www.autogenomics.com/pdf/INFINITICYP2C19PackageInsertUSA)) resulting in misclassification of the *CYP2C19*10* allele as the *CY2C19*2* allele.

Clopidogrel is an antiplatelet prodrug that undergoes a complex multi-step oxidative metabolism and bioactivation to its active metabolite in the liver by CYP2C19 and a few other CYP enzymes. The *CYP2C19*2* and *CYP2C19*3* null polymorphisms in *CYP2C19* gene has been associated with reduced clinical response to clopidogrel [[4](#R4)]. Clopidogrel is a major focus of our Personalized Medicine Program at the University of Florida, and in other clinical phamacogenetic programs. As a part of our clinical pharmacogenetic program, we noted a potential problem with genotyping *CYP2C19*2* when *CYP2C19*10* is present. We then recognized the absence of data on *CYP2C19.10* on clopidogrel metabolism. Hence we describe the impact of CYP2C19.10 on metabolism of clopidogrel and 2-oxo-clopidogrel and also the effect of the *CYP2C19*10* SNP on the *CYP2C19*2* TaqMan® genotyping assay.

## MATERIALS AND METHODS

### Chemicals and Materials

Clopidogrel, 2-oxo-clopidogrel, d4-clopidogrel, and the 2-bromo-3’-methoxy acetophenone (MPB) derivative of racemic *cis*-clopidogrel thiol active metabolite (clopidogrel-AM) were obtained from Toronto Research Chemicals (catalogue number C587256, Ontario, Canada). Omeprazole, *S*-mephenytoin, 5-hydroxyomeprazole, 4-hydroxymephenytoin, recombinant human NADPH-P450 oxidoreductase, human cytochorome b5, and other regents were purchased from Sigma-Aldrich (St. Louis, MO). Partially purified recombinant bacterially expressed CYP2C19.1B (wild type) and CYP2C19.10 proteins were provided by J. Goldstein’s laboratory [[1](#R1)].

### CYP2C19.10 kinetic study

Recombinant CYP2C19.1 and CYP2C19.10 were modified at the N-terminus for optimal expression in bacteria by replacing the first 8 amino acids with those of bovine cytochrome P450 17-α. These amino acids are not involved in substrate specificity. The allelic proteins were expressed in *E coli* DH5α and purified as described extensively in [[1](#R1), [2](#R2), [12](#R12)].

The *in vitro* incubation study was conducted following a previously published method with some minor modifications [[1](#R1)]. In brief, the partially purified recombinant CYP2C19.1B (wild type) and CYP2C19.10 proteins were pre-incubated with recombinant human NADPH-P450 oxidoreductase (4 pmol/pmol P450) and human cytochrome b5 (2 pmol/ pmol P450) at 37°C for 5 min, followed by adding various concentrations of the tested substrates, including clopidogrel (300, 400, 500 µM), 2-oxo-clopidogrel (10, 20, 50, 100, 200, 386 µM), omeprazole (2, 5, 10, 20, 50, 100, 200, 500 µM), or *S*-mephenytoin (50, 100, 200, 400, 600, 1000 µM) in a reaction buffer containing 20 mM HEPES, 0.1 mM EDTA and 1.25 mM MgCl_2_ (pH 7.4) for 3 min. The reaction was initiated by adding NADPH-generating reagents (0.1 mg/ml of yeast glucose-6-phosphate dehydrogenase, 3 mg/ml of NADP+, and 0.07 M glucose-6-phosphate). The final concentrations of the CYP2C19 enzymes were 2.24 µM, 1.12 µM, 0.046 µM, and 0.11 µM for the reactions of clopidogrel, 2-oxo-clopidogrel, omeprazole, and *S*-mephenytoin, respectively. The incubation times were 2 h for clopidogrel, 20 min for 2-oxo-clopidogrel, and 10 min for omeprazole and *S*-mephenytoin. The amounts of the CYP2C19 enzymes and the incubation times were evaluated in preliminary studies to ensure that the formation of the metabolites was linear with respect to the concentrations of enzymes and incubation time.

The metabolic reactions of clopidogrel and 2-oxo-clopidogrel were terminated by adding a two-fold volume of acetonitrile containing the internal standard d4-clopidogrel (10 ng/ml) and the derivatizing reagent MPB, (5 mM). MPB was utilized to convert the unstable clopidogrel-AM to its stable MPB derivative to facilitate analysis. The samples were vortexed and left at room temperature for 10 min to allow the completion of derivatization. The reactions for the hydroxylation of omeprazole and mephenyoin were terminated by adding two-fold volume of acetonitrile containing the internal standards d3-hydroxyomeprazole (10 ng/ml). All samples were then vortexed and centrifuged at 16,000 xg for 20 min at 4°C. The supernatants, containing parent compounds and formed metabolites (clopidogrel-AM, 5-hydroxyomeprazole, and 4-hydroxymephenytoin) were collected and subjected to liquid chromatograph tandem mass spectrometry (LC-MS/MS) analysis utilizing assays described below.

### LC-MS/MS assays

The LC-MS/MS system consisted of a Shimadzu HPLC system coupled with an AB Sciex API 3000 triple quadrupole mass spectrometer. The concentrations of clopidogrel, 2-oxo-clopidogrel, and clopidogrel-AM were determined using an LC-MS/MS assay described in our recent publication [[13](#R13)]. The assays for the quantification of omeprazole, mephenytoin and their respective hydroxylation metabolites were developed based on the published methods with some modifications [[14](#R14), [15](#R15)]. Briefly, the analytes were separated on a Phenomenex (Torrance, CA) 3-µm C18 reverse phase column (2.0×150 mm) at a flow rate of 0.2 mL/min. The mobile phase was 90% methanol with 0.1% formic acid for omeprazole and 80% acetonitrile with 0.2% formic acid for *S*-mephenytoin. Ionization was achieved via ESI in the positive mode and ions were monitored by multiple reaction monitoring. Omeprazole, 5-hydroxyomeprazole, *S*-mephenytoin, 4-hydroxymephenytoin and d3-hydroxyomeprazole (internal standard for omeprazole and *S*-mephenytoin) were monitored via the m/z transition 346.1/198.1, 362.3/213.9, 219.1/134.1, 235.1/150.2 and 365.3/213.9, respectively. The collision energy was 32V and 25V for omeprazole and *S*-mephenytoin, respectively. Ionspray voltage, source temperature, and curtain gas was 5000 V, 400 °C and 12 L/h, respectively, for both methods. The assays were validated with precision (CV% < 10%) and accuracy (88% – 114%) for all analytes.

### CYP2C19*10 allele genotyping

Genomic DNA from the INternatioinal Verapamil SR Trandolapil Study (INVEST), and Pharmacogenomic Evaluation of Antihypertensive Responses (PEAR) were isolated from mouthwash or blood by using commercially available kits (PureGene, Gentra Systems Inc., Minneapolis, MN, and QIAGEN DNA Blood Isolation Kit, QIAGEN, Valencia, CA). DNA samples from a total of 181 African-American and 202 Hispanic-Latinos DNA samples from PEAR [[9](#R9)] and INVEST [[10](#R10)] clinical studies were genotyped for *CYP2C19*2* and **10* by pyrosequencing [[11](#R11)]. The pyrosequencing genotyping assay for *CYP2C19*2* and *CYP2C19*10* were designed by Pyrosequencing assay design software. The PCR and sequencing primers, annealing temperature and sequence to analyze for pyrosequencing assays are as follow, the *CYP2C19*10 and CYP2C19*2* forward Bio-5’-TTACAACCAGAGCTTGGCATAT-3’, Reverse-5’-CGCAAGCAGTCACATAACTAAGC-3’PCR primers, reverse sequencing primer 5’- AAGTAATTTGTTATGGGTTC-3’, annealing temperature 58° C, and sequence to analyze, CC/TG/AGGAAATAATCAATGA. Both *CYP2C19*2* and *CYP2C19*10* SNPs are genotyped by single PCR and pyrosequencing reaction. The predicted histogram and pyrogram along with genotype calls for a sample are shown in [figure 1](#F1).

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b0bb/4090277/8ade45784e68/nihms608938f1.jpg)

Predicted histogram and pyrogram along with genotype call for an individual with CYP2C19*10 (C/T) and CYP2C19*2 (G/A) genotype are shown. The order of nucleotide dispensation is located at the bottom of histogram and pyrogram.

### Statistical Analysis

Data are presented as mean ± SD of duplicated independent experiments. Data were fit to the Michaelis-Menten equation, and the kinetic parameters K_m_ and V_max_ were calculated via nonlinear regression analysis (Graphpad Prism software version 4.0; Graphpad Software, Inc., San Diego, CA). Apparent intrinsic clearance (Cl_int_) was calculated as the ratio of V_max_ to K_m_. The differences in kinetic parameters between CYP2C19.1B and CYP2C19.10 were analyzed using the student’s t-test and P<0.05 is considered statistically significant. Hardy-Weinberg Equilibrium (HWE) was tested using a χ^2^ (chi square) goodness-of-fit test with one degree of freedom.

## RESULTS

### CYP2C19.10 kinetic analysis

The *in vitro* incubation studies demonstrated that CYP2C19.1B can efficiently catalyze the hydroxylation of the known CYP2C19 substrates omeprazole and *S*-mephenytoin ([Fig. 2A and Fig. 2B](#F2)). The V_max_ values of CYP2C19.1B were determined to be 2.77 ± 0.04 nmole/min/nmole and 1.17 ± 0.13 nmole/min/nmole for omeprazole and *S*-mephenytoin, respectively, under the experimental conditions. The K_m_ values for omeprazole and *S*-mephenytoin, were 32 ± 2 µM and 119 ± 21 µM, respectively. Consistent with previous reports, CYP2C19.10 behaved as a loss-of-function variant in the metabolism of *S*-mephenytoin, while exhibiting markedly decreased enzymatic activity towards omeprazole hydroxylation. The observed Cl_int_ values of CYP2C19.10 (2.2 ± 0.04) for omeprazole 5’-hydroxylation were less than 3% of that of CYP2C19.1B (87 ± 2.6).

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b0bb/4090277/c43c7222a025/nihms608938f2.jpg)

Kinetic analysis of catalytic activity of CYP2C19.1B (wild type) and CYP2C19.10 proteins on the metabolism of omeprazole (A), S-mephenytoin (B), 2-oxo-clopidogrel (C), and clopidogrel (D). Enzymatic activity was determined by the measurements of respective metabolites as indicated in the Y axis title of each figure. Values are means ± SD of two independent experiments.

The clopidogrel intermediate metabolite 2-oxo-clopidogrel was efficiently metabolized to clopidogrel-AM by CYP2C19.1B ([Fig. 2C](#F2)). However, the catalytic activity of CYP2C19.10 on the activation of 2-oxo-clopidgorel was significantly impaired relative to the wild type enzyme. The calculated Cl_int_ value of CYP2C19.10 on 2-oxo-clopdigrel activation was approximately 25% that of CYP2C19.1B ([Table 1](#T1)). Significantly decreased enzymatic activity of CYP2C19.10 was also observed for the activation of clopidogrel ([Fig. 2D](#F2)). The observed velocities of clopidogrel-AM formation from 2-oxo-clopidogrel and clopidogrel ([Fig. 2C and Fig. 2D](#F2)) indicate that the activation of clopidogrel mediated by CYP2C19 was less efficient relative to 2-oxo-clopidogrel activation. We were only able to measure the activation of clopidogrel at three relatively high substrate concentrations (i.e. 300, 400, 500 µM). The kinetic parameters V_max_ and K_m_ of clopidogrel activation were not determined due to the limited range of substrate concentrations available for analysis. The kinetic parameters of other tested substrates are summarized in [Table 1](#T1).

### Table 1.

|   | 2-oxo-clopidogrel |  |  | Omeprazole |  |  | S-mephenytoin |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| Vmax | Km (µM) | Clint | Vmax | Km (µM) | Clint | Vmax | Km (µM) | Clint |  |
| 2C19.1B | 17.4 ± 4.1 | 409.2 ± 159.5 | 0.052 ± 0.008 | 2.77 ± 0.04 | 32 ± 2 | 87.2 ± 2.6 | 1.17 ± 0.13 | 119 ± 21 | 9.7 ± 2.7 |
| 2C19.10 | 6.1 ± 2.3 | 302.3 ± 202.4 | 0.013 ± 0.001** | 0.74 ± 0.02** | 336 ± 18** | 2.2 ± 0.04** | N/A | N/A | N/A |

Table 1 Caption: Enzyme kinetic parameters for CYP2C19.1B and CYP2C19.10

### CYP2C19*10 genotyping and frequency

The presence of the *CYP2C19*10* SNP in close proximity to *CYP2C19*2* interferes with the *CYP2C19*2* TaqMan® genotyping assay and results in miscalling of *CYP2C19*10/*2* as *CYP2C19*2/*2.* This only has been observed with *2 allele and does not affect other alleles such as *1 (*1/*10) or *17 (*10/*17). The frequency of the *CYP2C19*10* allele is 0.21% for African Americans and Mexicans and 0.04% for Caucasians ([http://www.ncbi.nlm.nih.gov/snp](http://www.ncbi.nlm.nih.gov/snp)). The presence of *CYP2C19*10* SNP disrupted the *CYP2C19*2* TaqMan® assay genotyping process and heterozygous samples with *CYP2C19*10/*2* genotypes were called as homozygous *CYP2C19*2/*2*. The sample was then genotyped by a pyrosequencing method and the actual genotype was confirmed as *CYP2C19*10/*2.* The genotype calls for an individual sample with *CYP2C19*10 (C/T) and CYP2C19*2* (G/A) genotype are shown in [figure 1](#F1). The *CYP2C19*10* allele frequency for African-Americans, Hispanics and Caucasians was 0.8%, 0.25%, and 0%, respectively.

## DISCUSSION

The non-synonymous *CYP2C19*10* SNP results in substitution of proline 227, a highly conserved amino acid among the CYP2C family, to leucine. Proline 227 is located within the F-G loop, forming a part of the substrate access channel in CYP2C19 that plays an important role in substrate specificity [[6](#R6), [7](#R7)]. Although CYP2C19.10 is not a null variant, it was shown to have significantly decreased enzymatic activity toward menphenytoin and omeperazole relative to the wild type enzyme, behaving functionally like a loss of function allele [[1](#R1), [8](#R8), [2](#R2)]. The potential impact of the CYP2C19.10 variant on metabolism of other CYP2C19 substrates, *CYP2C19*2* genotyping and the absence of kinetic data on clopidogrel bioactivation were the reasons we performed this study. To our knowledge, this is the first study to measure the enzymatic activity of CYP2C19.10 protein toward clopidogrel and 2-oxo-clopidogrel. *S*-mephenytoin and omeprazole were used as positive controls in this study and consistent with previous reports ([1](#R1), [8](#R8)), the CYP2C19.10 variant behaved as a loss of function allele for the metabolism of S-mephenytoin, and exhibited a significantly decreased enzymatic activity toward omeprazole hydroxylation. The activation of 2-oxo-clopidogrel by CYP2C19.10 variant was significantly impaired and its calculated Cl_int_ value was approximately 25% that of the wild type CYP2C19.1B enzyme. Thus while the *CYP2C19.10* allelic protein shows dramatic loss of function for S-mephenytoin and omeperazole, it clearly exhibits substrate specificity and would be characterized as a reduced function allele towards clopidogrel and 2-oxo-clopidogrel. Thus the impact of CYP2C19.10 on metabolism of various substrates cannot be precisely defined and will require further study with each substrate of interest.

The CYP2C19*1B-mediated activation of clopidogrel was less efficient relative to 2-oxo-clopidogrel activation. We speculate that this may be due to the complex two-step metabolism of clopidogrel compared to one step metabolism of 2-oxo-clopidogrel. Additionally, clopidogrel is a mechanism-based CYP2C19 inhibitor as determined in an in vitro study [[16](#R16)]. Thus, the presence of the parent compound clopidogrel may have attenuated the second step activation (i.e. 2-oxo-clopidogrel > clopidogrel-AM) catalyzed by CYP2C19. We were also not able to determine the kinetic parameters V_max_ and K_m_ of clopidogrel metabolism due to the limited range of substrate concentrations.

Since the presence of *CYP2C19*10* allele affects the *CY2C19*2* SNP genotyping and results in misclassification of *CYP2C19*10/*2* as *CYP2C19*2/*2* in some reported genotyping methods, we suggest that samples with *CYP2C19*2/*2* genotypes be checked for the presence of the *CYP2C19*10* SNP. Particularly since CYP2C19.10 does not behave as null allelic protein for clopidogrel. For CYP2C19 substrates such as S-mephenytoin in which *CYP2C19*10* is essentially a loss of function allele, the metabolism status wouldn’t differ from that of carriers of *CYP2C19*2*; however, for substrates such as clopidogrel that are partially metabolized by CYP2C19.10, it is important to genotype for both *CYP2C19*10* and *CYP2C19*2* SNPs. Although the *CYP2C19*10* allele frequency for African-Americans (0.8%), Hispanics (0.25), and Caucasians (0%), was relatively low, the important impact of this allele on CYP2C19 substrate metabolism and *CY2C19*2* SNP genotyping justifies its inclusion in *CYP2C19* pharmacogenetic testing panels. For *CYP2C19* genotyping, we suggest to use assays that can distinguish *CYP2C19*10* from *CYP2C19*2*.

In conclusion, our data provide evidence that CYP2C19.10 variant partially metabolizes clopidogrel and 2-oxo-clopidogrel, and the presence of *CYP2C19*10* allele affects the *CY2C19*2* TaqMan® genotyping assay and results in misclassification of *CYP2C19*10/*2* as *CYP2C19*2/*2.*

## Acknowledgments

This work was partially supported by the Intramural Research Program of the National Institute of Environmental Health Sciences (Project ES021024-32) (J.A.G.) and partially by Grants NIAID 1R21AI096345 (J.M and H.J.Z.), UL1RR029890-03S3 (UL1TR000064) (J.A.J.), U01 GM074492 (J.A.J and T.L.), and U01 GM074492 (J.A.J. and T.L.).

## Footnotes

## References

1. Blaisdell J, Mohrenweiser H, Jackson J, Ferguson S, Coulter S, Chanas B, et al. Identification and functional characterization of new potentially defective alleles of human CYP2C19. Pharmacogenetics. 2002;12:703–711. doi: 10.1097/00008571-200212000-00004.  [DOI](https://doi.org/10.1097/00008571-200212000-00004) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12464799/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=Identification%20and%20functional%20characterization%20of%20new%20potentially%20defective%20alleles%20of%20human%20CYP2C19&author=J%20Blaisdell&author=H%20Mohrenweiser&author=J%20Jackson&author=S%20Ferguson&author=S%20Coulter&volume=12&publication_year=2002&pages=703-711&pmid=12464799&doi=10.1097/00008571-200212000-00004&)

2. Wang H, Kim RA, Sun D, Gao Y, Wang H, Zhu J, et al. Evaluation of the effects of 18 non-synonymous single-nucleotide polymorphisms of CYP450 2C19 on in vitro drug inhibition potential by a fluorescence-based high-throughput assay. Xenobiotica. 2011;41(9):826–835. doi: 10.3109/00498254.2011.582893.  [DOI](https://doi.org/10.3109/00498254.2011.582893) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21692664/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Xenobiotica&title=Evaluation%20of%20the%20effects%20of%2018%20non-synonymous%20single-nucleotide%20polymorphisms%20of%20CYP450%202C19%20on%20in%20vitro%20drug%20inhibition%20potential%20by%20a%20fluorescence-based%20high-throughput%20assay&author=H%20Wang&author=RA%20Kim&author=D%20Sun&author=Y%20Gao&author=H%20Wang&volume=41&issue=9&publication_year=2011&pages=826-835&pmid=21692664&doi=10.3109/00498254.2011.582893&)

3. Ancrenaz V, Desmeules J, James R, Fontana P, Reny JL, Dayer P, et al. The paraoxonase-1 pathway is not a major bioactivation pathway of clopidogrel in vitro. Br J Pharmacol. 2012;166(8):2362–2370. doi: 10.1111/j.1476-5381.2012.01946.x.  [DOI](https://doi.org/10.1111/j.1476-5381.2012.01946.x) | [PMC free article](/articles/PMC3448899/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22428615/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Pharmacol&title=The%20paraoxonase-1%20pathway%20is%20not%20a%20major%20bioactivation%20pathway%20of%20clopidogrel%20in%20vitro&author=V%20Ancrenaz&author=J%20Desmeules&author=R%20James&author=P%20Fontana&author=JL%20Reny&volume=166&issue=8&publication_year=2012&pages=2362-2370&pmid=22428615&doi=10.1111/j.1476-5381.2012.01946.x&)

4. Scott SA, Sangkuhl K, Gardner EE, Stein CM, Hulot JS, Johnson JA, et al. Clinical Pharmacogenetics Implementation Consortium. Clinical Pharmacogenetics Implementation Consortium guidelines for cytochrome P450-2C19 (CYP2C19) genotype and clopidogrel therapy. Clin Pharmacol Ther. 2011;90(2):328–332. doi: 10.1038/clpt.2011.132.  [DOI](https://doi.org/10.1038/clpt.2011.132) | [PMC free article](/articles/PMC3234301/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21716271/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Clinical%20Pharmacogenetics%20Implementation%20Consortium.%20Clinical%20Pharmacogenetics%20Implementation%20Consortium%20guidelines%20for%20cytochrome%20P450-2C19%20(CYP2C19)%20genotype%20and%20clopidogrel%20therapy&author=SA%20Scott&author=K%20Sangkuhl&author=EE%20Gardner&author=CM%20Stein&author=JS%20Hulot&volume=90&issue=2&publication_year=2011&pages=328-332&pmid=21716271&doi=10.1038/clpt.2011.132&)

5. Rasmussen H, Werge T. Misclassification of allele CYP2C19*10 as CY2C19*2 by a commonly used PCR-RFLP procedure. GenetTest. 2008;12(1):57–58. doi: 10.1089/gte.2007.0055.  [DOI](https://doi.org/10.1089/gte.2007.0055) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18373404/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=GenetTest&title=Misclassification%20of%20allele%20CYP2C19*10%20as%20CY2C19*2%20by%20a%20commonly%20used%20PCR-RFLP%20procedure&author=H%20Rasmussen&author=T%20Werge&volume=12&issue=1&publication_year=2008&pages=57-58&pmid=18373404&doi=10.1089/gte.2007.0055&)

6. Tsao CC, Wester MR, Ghanayem B, Coulter SJ, Chanas B, Johnson EF, et al. Identification of human CYP2C19 residues that confer S-mephenytoin 4'-hydroxylation activity to CYP2C9. Biochemistry. 2001;40(7):1937–1944. doi: 10.1021/bi001678u.  [DOI](https://doi.org/10.1021/bi001678u) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11329260/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochemistry&title=Identification%20of%20human%20CYP2C19%20residues%20that%20confer%20S-mephenytoin%204'-hydroxylation%20activity%20to%20CYP2C9&author=CC%20Tsao&author=MR%20Wester&author=B%20Ghanayem&author=SJ%20Coulter&author=B%20Chanas&volume=40&issue=7&publication_year=2001&pages=1937-1944&pmid=11329260&doi=10.1021/bi001678u&)

7. Wada Y, Mitsuda M, Ishihara Y, Watanabe M, Iwasaki M, Asahi S. Important amino acid residues that confer CYP2C19 selective activity to CYP2C9. J Biochem. 2008;144(3):323–333. doi: 10.1093/jb/mvn070.  [DOI](https://doi.org/10.1093/jb/mvn070) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18511451/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biochem&title=Important%20amino%20acid%20residues%20that%20confer%20CYP2C19%20selective%20activity%20to%20CYP2C9.&author=Y%20Wada&author=M%20Mitsuda&author=Y%20Ishihara&author=M%20Watanabe&author=M%20Iwasaki&volume=144&issue=3&publication_year=2008&pages=323-333&pmid=18511451&doi=10.1093/jb/mvn070&)

8. Lee SJ, Kim WY, Kim H, Shon JH, Lee SS, Shin JG. Identification of new CYP2C19 variants exhibiting decreased enzyme activity in the metabolism of S-mephenytoin and omeprazole. Drug Metab Dispos. 2009;37(11):2262–2269. doi: 10.1124/dmd.109.028175.  [DOI](https://doi.org/10.1124/dmd.109.028175) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19661214/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Identification%20of%20new%20CYP2C19%20variants%20exhibiting%20decreased%20enzyme%20activity%20in%20the%20metabolism%20of%20S-mephenytoin%20and%20omeprazole&author=SJ%20Lee&author=WY%20Kim&author=H%20Kim&author=JH%20Shon&author=SS%20Lee&volume=37&issue=11&publication_year=2009&pages=2262-2269&pmid=19661214&doi=10.1124/dmd.109.028175&)

9. Johnson JA, Boerwinkle E, Zineh I, Chapman AB, Bailey K, Cooper-DeHoff RM, et al. Pharmacogenomics of antihypertensive drugs: rationale and design of the Pharmacogenomic Evaluation of Antihypertensive Responses (PEAR) study. Am. Heart. J. 2009;157:442–449. doi: 10.1016/j.ahj.2008.11.018.  [DOI](https://doi.org/10.1016/j.ahj.2008.11.018) | [PMC free article](/articles/PMC2671287/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19249413/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20Heart.%20J&title=Pharmacogenomics%20of%20antihypertensive%20drugs:%20rationale%20and%20design%20of%20the%20Pharmacogenomic%20Evaluation%20of%20Antihypertensive%20Responses%20(PEAR)%20study&author=JA%20Johnson&author=E%20Boerwinkle&author=I%20Zineh&author=AB%20Chapman&author=K%20Bailey&volume=157&publication_year=2009&pages=442-449&pmid=19249413&doi=10.1016/j.ahj.2008.11.018&)

10. Pepine CJ, Handberg-Thurmond E, Marks RG, Conlon M, Cooper-DeHoff R, Volkers P, et al. Rationale and design of the international verapamil SR/trandolapril study (INVEST): an internet-based randomized trial in coronary artery disease patients with hypertension. J. Am Coll. Cardiol. 1998;32:1228–1237. doi: 10.1016/s0735-1097(98)00423-9.  [DOI](https://doi.org/10.1016/s0735-1097(98)00423-9) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9809930/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Am%20Coll.%20Cardiol&title=Rationale%20and%20design%20of%20the%20international%20verapamil%20SR/trandolapril%20study%20(INVEST):%20an%20internet-based%20randomized%20trial%20in%20coronary%20artery%20disease%20patients%20with%20hypertension&author=CJ%20Pepine&author=E%20Handberg-Thurmond&author=RG%20Marks&author=M%20Conlon&author=R%20Cooper-DeHoff&volume=32&publication_year=1998&pages=1228-1237&pmid=9809930&doi=10.1016/s0735-1097(98)00423-9&)

11. Langaee TY, Ronaghi M. Genetic Variation Analyses by Pyrosequencing. Mutation Research. 2005;573(1–2):96–102. doi: 10.1016/j.mrfmmm.2004.07.023.  [DOI](https://doi.org/10.1016/j.mrfmmm.2004.07.023) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15829240/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mutation%20Research&title=Genetic%20Variation%20Analyses%20by%20Pyrosequencing&author=TY%20Langaee&author=M%20Ronaghi&volume=573&issue=1%E2%80%932&publication_year=2005&pages=96-102&pmid=15829240&doi=10.1016/j.mrfmmm.2004.07.023&)

12. Ibeanu GC, Blaisdell J, Ghanayem BI, Beyeler C, Benhamou S, Bouchardy C, et al. An additional defective allele, CYP2C19*5, contributes to the S-mephenytoin poor metabolizer phenotype in Caucasians. Pharmacogenetics. 1998;8(2):129–135. doi: 10.1097/00008571-199804000-00006.  [DOI](https://doi.org/10.1097/00008571-199804000-00006) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10022751/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=An%20additional%20defective%20allele,%20CYP2C19*5,%20contributes%20to%20the%20S-mephenytoin%20poor%20metabolizer%20phenotype%20in%20Caucasians&author=GC%20Ibeanu&author=J%20Blaisdell&author=BI%20Ghanayem&author=C%20Beyeler&author=S%20Benhamou&volume=8&issue=2&publication_year=1998&pages=129-135&pmid=10022751&doi=10.1097/00008571-199804000-00006&)

13. Zhu HJ, Wang X, Gawronski BE, Brinda BJ, Angiolillo DJ, Markowitz JS. Carboxylesterase 1 as a determinant of clopidogrel metabolism and activation. J Pharmacol Exp Ther. 2013;344(3):665–672. doi: 10.1124/jpet.112.201640.  [DOI](https://doi.org/10.1124/jpet.112.201640) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23275066/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharmacol%20Exp%20Ther&title=Carboxylesterase%201%20as%20a%20determinant%20of%20clopidogrel%20metabolism%20and%20activation&author=HJ%20Zhu&author=X%20Wang&author=BE%20Gawronski&author=BJ%20Brinda&author=DJ%20Angiolillo&volume=344&issue=3&publication_year=2013&pages=665-672&pmid=23275066&doi=10.1124/jpet.112.201640&)

14. Kim MJ, Kim H, Cha IJ, Park JS, Shon JH, Liu KH, Shin JG. High-throughput screening of inhibitory potential of nine cytochrome P450 enzymes in vitro using liquid chromatography/tandem mass spectrometry. Rapid communications in mass spectrometry : RCM. 2005;19:2651–2658. doi: 10.1002/rcm.2110.  [DOI](https://doi.org/10.1002/rcm.2110) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16124035/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Rapid%20communications%20in%20mass%20spectrometry%20:%20RCM&title=High-throughput%20screening%20of%20inhibitory%20potential%20of%20nine%20cytochrome%20P450%20enzymes%20in%20vitro%20using%20liquid%20chromatography/tandem%20mass%20spectrometry&author=MJ%20Kim&author=H%20Kim&author=IJ%20Cha&author=JS%20Park&author=JH%20Shon&volume=19&publication_year=2005&pages=2651-2658&pmid=16124035&doi=10.1002/rcm.2110&)

15. Zhang W, Han F, Guo P, Zhao H, Lin ZJ, Huang MQ, Bertelsen K, Weng N. Simultaneous determination of tolbutamide, omeprazole, midazolam and dextromethorphan in human plasma by LC-MS/MS--a high throughput approach to evaluate drug-drug interactions. Journal of chromatography B, Analytical technologies in the biomedical and life sciences. 2010;878:1169–1177. doi: 10.1016/j.jchromb.2010.03.026.  [DOI](https://doi.org/10.1016/j.jchromb.2010.03.026) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20378422/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20of%20chromatography%20B,%20Analytical%20technologies%20in%20the%20biomedical%20and%20life%20sciences&title=Simultaneous%20determination%20of%20tolbutamide,%20omeprazole,%20midazolam%20and%20dextromethorphan%20in%20human%20plasma%20by%20LC-MS/MS--a%20high%20throughput%20approach%20to%20evaluate%20drug-drug%20interactions&author=W%20Zhang&author=F%20Han&author=P%20Guo&author=H%20Zhao&author=ZJ%20Lin&volume=878&publication_year=2010&pages=1169-1177&pmid=20378422&doi=10.1016/j.jchromb.2010.03.026&)

16. Nishiya Y, Hagihara K, Kurihara A, Okudaira N, Farid NA, Okazaki O, Ikeda T. Comparison of mechanism-based inhibition of human cytochrome P450 2C19 by ticlopidine, clopidogrel, and prasugrel. Xenobiotica. 2009;39:836–843. doi: 10.3109/00498250903191427.  [DOI](https://doi.org/10.3109/00498250903191427) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19845434/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Xenobiotica&title=Comparison%20of%20mechanism-based%20inhibition%20of%20human%20cytochrome%20P450%202C19%20by%20ticlopidine,%20clopidogrel,%20and%20prasugrel&author=Y%20Nishiya&author=K%20Hagihara&author=A%20Kurihara&author=N%20Okudaira&author=NA%20Farid&volume=39&publication_year=2009&pages=836-843&pmid=19845434&doi=10.3109/00498250903191427&)
