# Inherited GATA3 variants are associated with Ph-like childhood acute lymphoblastic leukemia and risk of relapse

## Metadata
**Authors:** Virginia Perez-Andreu, Kathryn G Roberts, Richard C Harvey, Wenjian Yang, Cheng Cheng, Deqing Pei, Heng Xu, Julie Gastier-Foster, E Shuyu, Joshua Yew-Suang Lim, I-Ming Chen, Yiping Fan, Meenakshi Devidsa, Michael J Borowitz, Colton Smith, Geoffrey Neale, Esteban G Burchard, Dara G Torgerson, Federico Antillon Klussmann, Cesar Rolando Najera Villagran, Naomi J Winick, Bruce M Camitta, Elizabeth Raetz, Brent Wood, Feng Yue, William L Carroll, Eric Larsen, W Paul Bowman, Mignon L Loh, Michael Dean, Deepa Bhojwani, Ching-Hon Pui, William E Evans, Mary V Relling, Stephen P Hunger, Cheryl L Willman, Charles G Mullighan, Jun J Yang
**Journal:** Nature genetics
**Date:** 2013 Oct 20
**DOI:** [10.1038/ng.2803](https://doi.org/10.1038/ng.2803)
**PMID:** 24141364
**PMCID:** PMC4039076
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4039076/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC4039076/pdf/nihms-568109.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC4039076/pdf/nihms-568109.pdf)

## Abstract

Recent genomic profiling of childhood acute lymphoblastic leukemia (ALL) identified a novel high-risk subtype with a gene expression signature resembling Philadelphia chromosome-positive ALL and a poor prognosis (Ph-like ALL). However, the role of inherited genetic variation in Ph-like ALL pathogenesis remains unknown. In a genome-wide association study (GWAS) of 511 ALL cases and 6,661 non-ALL controls, we identified a single susceptibility locus for Ph-like ALL (GATA3, rs3824662, P=2.17×10−14, odds ratio [OR]=3.85, for Ph-like ALL vs. non-ALL; P=1.05×10−8, OR=3.25, for Ph-like ALL vs. non-Ph-like ALL) that was independently validated. The rs3824662 risk allele was associated with somatic lesions underlying Ph-like ALL (i.e., CRLF2 rearrangement, JAK mutation, and IKZF1 deletion) and directly influenced GATA3 transcription. Finally, GATA3 SNP genotype was also associated with early treatment response and the risk of ALL relapse. Our results provide insights into interactions between host and tumor genomes and their importance in ALL pathogenesis and prognosis.

## ONLINE METHODS

### Subjects and genotyping

The ALL cases investigated comprised children with newly-diagnosed B-precursor ALL who were treated on the Children’s Oncology Group (COG) trials AALL0232, P9905^32^ and P9906^10^ ([Supplementary Table 2](#SD1)), and non-ALL controls included 12,416 subjects^14,33-35^. The number of subjects included in each analysis was described in [Supplementary Figs 15](#SD1), [16](#SD1), and [17](#SD1), and in the text as appropriate. This study was approved by the Institutional Review Boards with proper informed consent.

Germline genomic DNA was extracted from peripheral blood or bone marrow samples obtained during clinical remission for children with ALL. Genotyping was done for COG AALL0232 and COG P9905 cohorts and for non-ALL controls using the Affymetrix Human SNP Array 6.0. Quality control was performed for samples and SNPs according to call rate and minor allele frequency ([Supplementary Fig. 1](#SD1)). Theta (allele signal intensity) plots were constructed using Affymetrix Genotyping Console for rs3824662 and rs3781093 ([Supplementary Fig. 18](#SD1)). *GATA3* SNPs (rs3824662 and rs3781093) were genotyped in the COG P9906 cohort and in the Guatemalan samples by Sanger sequencing ([Supplementary Table 3](#SD1)).

Genetic ancestry was determined by using STRUCTURE^21,36^ and was used to define ethnicity ([Supplementary Note](#SD1)).

### Ph-like ALL and GWAS

Ph-like ALL was identified in the COG ALL0232 cohort and in the COG P9906 cohort on the basis of unsupervised hierarchical clustering analysis of global gene expression profile, as described previously^7,9,37^ .

The discovery GWAS of Ph-like ALL comprised 511 ALL cases enrolled on the COG AALL0232 protocol and 6,661 non-ALL controls from the dbGaP MESA dataset. We performed two association tests to identify germline SNPs related to Ph-like ALL: we compared the genotype frequency at each SNP 1) in Ph-like ALL (N=75) *vs.* non-ALL controls (N=6,661) and 2) in Ph-like ALL (N=75) *vs.* ALL cases without Ph-like profile (“non-Ph-like ALL”, N=436). Association was evaluated with logistic regression under an additive model with genetic ancestry as covariates. Population stratification was assessed by the construction of a quantile-quantile (Q-Q) plot ([Supplementary Fig. 19](#SD1)). SNPs that reached *P*≤5×10^−8^ in the discovery GWAS were tested in an independent replication cohort: 171 ALL cases from the COG P9906 protocol and 5,755 non-ALL controls. Association with Ph-like ALL was evaluated by logistic regression with genetic ancestries as covariates by comparing 1) Ph-like ALL (N=32) *vs.* non-ALL controls (N=5,755) and 2) Ph-like ALL (N=32) *vs.* non-Ph-like ALL (N=139). Independently, the Ph-like phenotype was also identified by the recognition of outliers by sampling ends (ROSE) algorithm ([Supplementary Fig. 20](#SD1)). *GATA3* SNPs (rs3824662 and rs3781093) and expression were also evaluated in a separate cohort of patients with Ph+ ALL ([Supplementary Note](#SD1) and [Supplementary Fig. 21](#SD1)).

Functional characterization of *GATA3* SNPs was performed by examining the association of SNP genotype with *GATA3* expression, local DNase hypersensitivity, and global gene expression in ALL ([Supplementary Note](#SD1), [Supplementary Table 4](#SD1), [Fig. 2](#F2), [Supplementary Figs. 6](#SD1),[7](#SD1), [12](#SD1), [22](#SD1), and [23](#SD1)), partly using previously published data sets^30,31,38^. Associations of *GATA3* SNPs with *CRLF2, JAK*, and *IKZF1* somatic lesions were evaluated in the COG AALL0232, COG P9906, and COG P9905 cohorts, and with relapse in the COG P9906 and COG P9905 cohorts ([Supplementary Note](#SD1)). Germline SNPs within the *JAK2* gene were tested for association with somatic *JAK2* mutation in ALL ([Supplementary Table 5](#SD1)). R 2.15.1 statistical software was used for all analyses unless indicated otherwise ([Supplementary Note](#SD1)). Statistical tests were chosen as appropriate and according to the phenotype distribution (e.g., normally or binomially distributed for continuous or categorical variables, respectively).

## Supplementary Material

## Acknowledgements

We thank the patients and parents who participated in the COG protocols included in this study, the clinicians and research staff at COG institutions and J. Pullen (University of Mississippi at Jackson) for assistance in classification of patients with ALL. Genome-wide genotyping of COG P9905 samples was performed by the Center for Molecular Medicine with the generous financial support from the Jeffrey Pride Foundation and the National Childhood Cancer Foundation. V.P.A is supported by the Spanish Ministry of Education Fellowship Grant and by the St. Jude Children’s Research Hospital Academic Programs Special Fellowship. J.J.Y. is supported by the American Society of Hematology Scholar Award, Alex Lemonade Stand Foundation for Childhood Cancer Young Investigator Grant, and by the Order of St. Francis Foundation. K.G.R. is supported by a National Health and Medical Research Council (Australia) Overseas Training Fellowship and a Haematology Society of Australia and New Zealand Novartis New Investigator Scholarship. C.G.M. is a Pew Scholar in the Biomedical Sciences and a St. Baldrick’s Scholar. We thank M. Shriver (Pennsylvania State University) for sharing SNP genotype data of the Native American references, Jonathan Pritchard and Jacob Degner (University of Chicago) for sharing DNase hypersensitivity data of HapMap Yoruba cell lines, Raul C. Ribeiro (St. Jude Children’s Research Hospital) and Pedro De Alarcon (University of Illinois College of Medicine at Peoria) for coordinating collaborations in Guatemala. This work was supported by the National Institutes of Health (grant numbers CA156449, CA21765, CA36401, CA98543, CA114766, CA98413, CA140729 and GM92666), in part by the intramural Program of the National Cancer Institute, and by the American Lebanese Syrian Associated Charities (ALSAC). The study sponsors were not directly involved in the design of the study, the collection, analysis, and interpretation of the data, the writing of the manuscript, or the decision to submit the manuscript.

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Pui CH, Evans WE. Treatment of acute lymphoblastic leukemia. N Engl J Med. 2006;354:166–78. doi: 10.1056/NEJMra052603.  [DOI](https://doi.org/10.1056/NEJMra052603) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16407512/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Treatment%20of%20acute%20lymphoblastic%20leukemia&author=CH%20Pui&author=WE%20Evans&volume=354&publication_year=2006&pages=166-78&pmid=16407512&doi=10.1056/NEJMra052603&)

2. Pui CH, Mullighan CG, Evans WE, Relling MV. Pediatric acute lymphoblastic leukemia: where are we going and how do we get there? Blood. 2012;120:1165–74. doi: 10.1182/blood-2012-05-378943.  [DOI](https://doi.org/10.1182/blood-2012-05-378943) | [PMC free article](/articles/PMC3418713/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22730540/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Pediatric%20acute%20lymphoblastic%20leukemia:%20where%20are%20we%20going%20and%20how%20do%20we%20get%20there?&author=CH%20Pui&author=CG%20Mullighan&author=WE%20Evans&author=MV%20Relling&volume=120&publication_year=2012&pages=1165-74&pmid=22730540&doi=10.1182/blood-2012-05-378943&)

3. Hunger SP, et al. Improved survival for children and adolescents with acute lymphoblastic leukemia between 1990 and 2005: a report from the children’s oncology group. J Clin Oncol. 2012;30:1663–9. doi: 10.1200/JCO.2011.37.8018.  [DOI](https://doi.org/10.1200/JCO.2011.37.8018) | [PMC free article](/articles/PMC3383113/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22412151/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Improved%20survival%20for%20children%20and%20adolescents%20with%20acute%20lymphoblastic%20leukemia%20between%201990%20and%202005:%20a%20report%20from%20the%20children%E2%80%99s%20oncology%20group&author=SP%20Hunger&volume=30&publication_year=2012&pages=1663-9&pmid=22412151&doi=10.1200/JCO.2011.37.8018&)

4. Stanulla M, et al. Integrating molecular information into treatment of childhood acute lymphoblastic leukemia--a perspective from the BFM Study Group. Blood Cells Mol Dis. 2007;39:160–3. doi: 10.1016/j.bcmd.2007.04.005.  [DOI](https://doi.org/10.1016/j.bcmd.2007.04.005) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17532236/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood%20Cells%20Mol%20Dis&title=Integrating%20molecular%20information%20into%20treatment%20of%20childhood%20acute%20lymphoblastic%20leukemia--a%20perspective%20from%20the%20BFM%20Study%20Group&author=M%20Stanulla&volume=39&publication_year=2007&pages=160-3&pmid=17532236&doi=10.1016/j.bcmd.2007.04.005&)

5. Biondi A, et al. Imatinib after induction for treatment of children and adolescents with Philadelphia-chromosome-positive acute lymphoblastic leukaemia (EsPhALL): a randomised, open-label, intergroup study. Lancet Oncol. 2012;13:936–45. doi: 10.1016/S1470-2045(12)70377-7.  [DOI](https://doi.org/10.1016/S1470-2045(12)70377-7) | [PMC free article](/articles/PMC3431502/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22898679/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet%20Oncol&title=Imatinib%20after%20induction%20for%20treatment%20of%20children%20and%20adolescents%20with%20Philadelphia-chromosome-positive%20acute%20lymphoblastic%20leukaemia%20(EsPhALL):%20a%20randomised,%20open-label,%20intergroup%20study&author=A%20Biondi&volume=13&publication_year=2012&pages=936-45&pmid=22898679&doi=10.1016/S1470-2045(12)70377-7&)

6. Den Boer ML, et al. A subtype of childhood acute lymphoblastic leukaemia with poor treatment outcome: a genome-wide classification study. Lancet Oncol. 2009;10:125–34. doi: 10.1016/S1470-2045(08)70339-5.  [DOI](https://doi.org/10.1016/S1470-2045(08)70339-5) | [PMC free article](/articles/PMC2707020/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19138562/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet%20Oncol&title=A%20subtype%20of%20childhood%20acute%20lymphoblastic%20leukaemia%20with%20poor%20treatment%20outcome:%20a%20genome-wide%20classification%20study&author=ML%20Den%20Boer&volume=10&publication_year=2009&pages=125-34&pmid=19138562&doi=10.1016/S1470-2045(08)70339-5&)

7. Roberts KG, et al. Genetic alterations activating kinase and cytokine receptor signaling in high-risk acute lymphoblastic leukemia. Cancer Cell. 2012;22:153–66. doi: 10.1016/j.ccr.2012.06.005.  [DOI](https://doi.org/10.1016/j.ccr.2012.06.005) | [PMC free article](/articles/PMC3422513/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22897847/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Cell&title=Genetic%20alterations%20activating%20kinase%20and%20cytokine%20receptor%20signaling%20in%20high-risk%20acute%20lymphoblastic%20leukemia&author=KG%20Roberts&volume=22&publication_year=2012&pages=153-66&pmid=22897847&doi=10.1016/j.ccr.2012.06.005&)

8. Harvey RC, et al. Identification of novel cluster groups in pediatric high-risk B-precursor acute lymphoblastic leukemia with gene expression profiling: correlation with genome-wide DNA copy number alterations, clinical characteristics, and outcome. Blood. 2010;116:4874–84. doi: 10.1182/blood-2009-08-239681.  [DOI](https://doi.org/10.1182/blood-2009-08-239681) | [PMC free article](/articles/PMC3321747/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20699438/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Identification%20of%20novel%20cluster%20groups%20in%20pediatric%20high-risk%20B-precursor%20acute%20lymphoblastic%20leukemia%20with%20gene%20expression%20profiling:%20correlation%20with%20genome-wide%20DNA%20copy%20number%20alterations,%20clinical%20characteristics,%20and%20outcome&author=RC%20Harvey&volume=116&publication_year=2010&pages=4874-84&pmid=20699438&doi=10.1182/blood-2009-08-239681&)

9. Mullighan CG, et al. Deletion of IKZF1 and prognosis in acute lymphoblastic leukemia. N Engl J Med. 2009;360:470–80. doi: 10.1056/NEJMoa0808253.  [DOI](https://doi.org/10.1056/NEJMoa0808253) | [PMC free article](/articles/PMC2674612/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19129520/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Deletion%20of%20IKZF1%20and%20prognosis%20in%20acute%20lymphoblastic%20leukemia&author=CG%20Mullighan&volume=360&publication_year=2009&pages=470-80&pmid=19129520&doi=10.1056/NEJMoa0808253&)

10. Harvey RC, et al. Rearrangement of CRLF2 is associated with mutation of JAK kinases, alteration of IKZF1, Hispanic/Latino ethnicity, and a poor outcome in pediatric B-progenitor acute lymphoblastic leukemia. Blood. 2010;115:5312–21. doi: 10.1182/blood-2009-09-245944.  [DOI](https://doi.org/10.1182/blood-2009-09-245944) | [PMC free article](/articles/PMC2902132/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20139093/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Rearrangement%20of%20CRLF2%20is%20associated%20with%20mutation%20of%20JAK%20kinases,%20alteration%20of%20IKZF1,%20Hispanic/Latino%20ethnicity,%20and%20a%20poor%20outcome%20in%20pediatric%20B-progenitor%20acute%20lymphoblastic%20leukemia&author=RC%20Harvey&volume=115&publication_year=2010&pages=5312-21&pmid=20139093&doi=10.1182/blood-2009-09-245944&)

11. Loh ML, et al. Tyrosine kinome sequencing of pediatric acute lymphoblastic leukemia: a report from the Children’s Oncology Group TARGET Project. Blood. 2013;121:485–8. doi: 10.1182/blood-2012-04-422691.  [DOI](https://doi.org/10.1182/blood-2012-04-422691) | [PMC free article](/articles/PMC3548168/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23212523/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Tyrosine%20kinome%20sequencing%20of%20pediatric%20acute%20lymphoblastic%20leukemia:%20a%20report%20from%20the%20Children%E2%80%99s%20Oncology%20Group%20TARGET%20Project&author=ML%20Loh&volume=121&publication_year=2013&pages=485-8&pmid=23212523&doi=10.1182/blood-2012-04-422691&)

12. Papaemmanuil E, et al. Loci on 7p12.2, 10q21.2 and 14q11.2 are associated with risk of childhood acute lymphoblastic leukemia. Nat Genet. 2009;41:1006–10. doi: 10.1038/ng.430.  [DOI](https://doi.org/10.1038/ng.430) | [PMC free article](/articles/PMC4915548/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19684604/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Loci%20on%207p12.2,%2010q21.2%20and%2014q11.2%20are%20associated%20with%20risk%20of%20childhood%20acute%20lymphoblastic%20leukemia&author=E%20Papaemmanuil&volume=41&publication_year=2009&pages=1006-10&pmid=19684604&doi=10.1038/ng.430&)

13. Sherborne AL, et al. Variation in CDKN2A at 9p21.3 influences childhood acute lymphoblastic leukemia risk. Nat Genet. 2010;42:492–4. doi: 10.1038/ng.585.  [DOI](https://doi.org/10.1038/ng.585) | [PMC free article](/articles/PMC3434228/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20453839/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Variation%20in%20CDKN2A%20at%209p21.3%20influences%20childhood%20acute%20lymphoblastic%20leukemia%20risk&author=AL%20Sherborne&volume=42&publication_year=2010&pages=492-4&pmid=20453839&doi=10.1038/ng.585&)

14. Trevino LR, et al. Germline genomic variants associated with childhood acute lymphoblastic leukemia. Nat Genet. 2009;41:1001–5. doi: 10.1038/ng.432.  [DOI](https://doi.org/10.1038/ng.432) | [PMC free article](/articles/PMC2762391/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19684603/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Germline%20genomic%20variants%20associated%20with%20childhood%20acute%20lymphoblastic%20leukemia&author=LR%20Trevino&volume=41&publication_year=2009&pages=1001-5&pmid=19684603&doi=10.1038/ng.432&)

15. Xu H, et al. Novel susceptibility variants at 10p12.31-12.2 for childhood acute lymphoblastic leukemia in ethnically diverse populations. J Natl Cancer Inst. 2013;105:733–42. doi: 10.1093/jnci/djt042.  [DOI](https://doi.org/10.1093/jnci/djt042) | [PMC free article](/articles/PMC3691938/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23512250/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Natl%20Cancer%20Inst&title=Novel%20susceptibility%20variants%20at%2010p12.31-12.2%20for%20childhood%20acute%20lymphoblastic%20leukemia%20in%20ethnically%20diverse%20populations&author=H%20Xu&volume=105&publication_year=2013&pages=733-42&pmid=23512250&doi=10.1093/jnci/djt042&)

16. Xu H, et al. ARID5B genetic polymorphisms contribute to racial disparities in the incidence and treatment outcome of childhood acute lymphoblastic leukemia. J Clin Oncol. 2012;30:751–7. doi: 10.1200/JCO.2011.38.0345.  [DOI](https://doi.org/10.1200/JCO.2011.38.0345) | [PMC free article](/articles/PMC3295551/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22291082/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=ARID5B%20genetic%20polymorphisms%20contribute%20to%20racial%20disparities%20in%20the%20incidence%20and%20treatment%20outcome%20of%20childhood%20acute%20lymphoblastic%20leukemia&author=H%20Xu&volume=30&publication_year=2012&pages=751-7&pmid=22291082&doi=10.1200/JCO.2011.38.0345&)

17. Paulsson K, et al. Genetic landscape of high hyperdiploid childhood acute lymphoblastic leukemia. Proc Natl Acad Sci U S A. 2010;107:21719–24. doi: 10.1073/pnas.1006981107.  [DOI](https://doi.org/10.1073/pnas.1006981107) | [PMC free article](/articles/PMC3003126/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21098271/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&title=Genetic%20landscape%20of%20high%20hyperdiploid%20childhood%20acute%20lymphoblastic%20leukemia&author=K%20Paulsson&volume=107&publication_year=2010&pages=21719-24&pmid=21098271&doi=10.1073/pnas.1006981107&)

18. Jones AV, et al. JAK2 haplotype is a major risk factor for the development of myeloproliferative neoplasms. Nat Genet. 2009;41:446–9. doi: 10.1038/ng.334.  [DOI](https://doi.org/10.1038/ng.334) | [PMC free article](/articles/PMC4120192/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19287382/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=JAK2%20haplotype%20is%20a%20major%20risk%20factor%20for%20the%20development%20of%20myeloproliferative%20neoplasms&author=AV%20Jones&volume=41&publication_year=2009&pages=446-9&pmid=19287382&doi=10.1038/ng.334&)

19. Olcaydu D, et al. A common JAK2 haplotype confers susceptibility to myeloproliferative neoplasms. Nat Genet. 2009;41:450–4. doi: 10.1038/ng.341.  [DOI](https://doi.org/10.1038/ng.341) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19287385/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=A%20common%20JAK2%20haplotype%20confers%20susceptibility%20to%20myeloproliferative%20neoplasms&author=D%20Olcaydu&volume=41&publication_year=2009&pages=450-4&pmid=19287385&doi=10.1038/ng.341&)

20. Kilpivaara O, et al. A germline JAK2 SNP is associated with predisposition to the development of JAK2(V617F)-positive myeloproliferative neoplasms. Nat Genet. 2009;41:455–9. doi: 10.1038/ng.342.  [DOI](https://doi.org/10.1038/ng.342) | [PMC free article](/articles/PMC3676425/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19287384/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=A%20germline%20JAK2%20SNP%20is%20associated%20with%20predisposition%20to%20the%20development%20of%20JAK2(V617F)-positive%20myeloproliferative%20neoplasms&author=O%20Kilpivaara&volume=41&publication_year=2009&pages=455-9&pmid=19287384&doi=10.1038/ng.342&)

21. Yang JJ, et al. Ancestry and pharmacogenomics of relapse in acute lymphoblastic leukemia. Nat Genet. 2011;43:237–41. doi: 10.1038/ng.763.  [DOI](https://doi.org/10.1038/ng.763) | [PMC free article](/articles/PMC3104508/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21297632/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Ancestry%20and%20pharmacogenomics%20of%20relapse%20in%20acute%20lymphoblastic%20leukemia&author=JJ%20Yang&volume=43&publication_year=2011&pages=237-41&pmid=21297632&doi=10.1038/ng.763&)

22. Fujiwara T, et al. Discovering hematopoietic mechanisms through genome-wide analysis of GATA factor chromatin occupancy. Mol Cell. 2009;36:667–81. doi: 10.1016/j.molcel.2009.11.001.  [DOI](https://doi.org/10.1016/j.molcel.2009.11.001) | [PMC free article](/articles/PMC2784893/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19941826/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Cell&title=Discovering%20hematopoietic%20mechanisms%20through%20genome-wide%20analysis%20of%20GATA%20factor%20chromatin%20occupancy&author=T%20Fujiwara&volume=36&publication_year=2009&pages=667-81&pmid=19941826&doi=10.1016/j.molcel.2009.11.001&)

23. Wei G, et al. Genome-wide analyses of transcription factor GATA3-mediated gene regulation in distinct T cell types. Immunity. 2011;35:299–311. doi: 10.1016/j.immuni.2011.08.007.  [DOI](https://doi.org/10.1016/j.immuni.2011.08.007) | [PMC free article](/articles/PMC3169184/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21867929/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Immunity&title=Genome-wide%20analyses%20of%20transcription%20factor%20GATA3-mediated%20gene%20regulation%20in%20distinct%20T%20cell%20types&author=G%20Wei&volume=35&publication_year=2011&pages=299-311&pmid=21867929&doi=10.1016/j.immuni.2011.08.007&)

24. Yagi R, Zhu J, Paul WE. An updated view on transcription factor GATA3-mediated regulation of Th1 and Th2 cell differentiation. Int Immunol. 2011;23:415–20. doi: 10.1093/intimm/dxr029.  [DOI](https://doi.org/10.1093/intimm/dxr029) | [PMC free article](/articles/PMC3123974/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21632975/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20Immunol&title=An%20updated%20view%20on%20transcription%20factor%20GATA3-mediated%20regulation%20of%20Th1%20and%20Th2%20cell%20differentiation&author=R%20Yagi&author=J%20Zhu&author=WE%20Paul&volume=23&publication_year=2011&pages=415-20&pmid=21632975&doi=10.1093/intimm/dxr029&)

25. Zhang J, et al. The genetic basis of early T-cell precursor acute lymphoblastic leukaemia. Nature. 2012;481:157–63. doi: 10.1038/nature10725.  [DOI](https://doi.org/10.1038/nature10725) | [PMC free article](/articles/PMC3267575/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22237106/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=The%20genetic%20basis%20of%20early%20T-cell%20precursor%20acute%20lymphoblastic%20leukaemia&author=J%20Zhang&volume=481&publication_year=2012&pages=157-63&pmid=22237106&doi=10.1038/nature10725&)

26. Enciso-Mora V, et al. A genome-wide association study of Hodgkin’s lymphoma identifies new susceptibility loci at 2p16.1 (REL), 8q24.21 and 10p14 (GATA3) Nat Genet. 2010;42:1126–30. doi: 10.1038/ng.696.  [DOI](https://doi.org/10.1038/ng.696) | [PMC free article](/articles/PMC4268499/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21037568/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=A%20genome-wide%20association%20study%20of%20Hodgkin%E2%80%99s%20lymphoma%20identifies%20new%20susceptibility%20loci%20at%202p16.1%20(REL),%208q24.21%20and%2010p14%20(GATA3)&author=V%20Enciso-Mora&volume=42&publication_year=2010&pages=1126-30&pmid=21037568&doi=10.1038/ng.696&)

27. Pasquet M, et al. High frequency of GATA2 mutations in patients with mild chronic neutropenia evolving to MonoMac syndrome, myelodysplasia, and acute myeloid leukemia. Blood. 2013;121:822–9. doi: 10.1182/blood-2012-08-447367.  [DOI](https://doi.org/10.1182/blood-2012-08-447367) | [PMC free article](/articles/PMC3714670/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23223431/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=High%20frequency%20of%20GATA2%20mutations%20in%20patients%20with%20mild%20chronic%20neutropenia%20evolving%20to%20MonoMac%20syndrome,%20myelodysplasia,%20and%20acute%20myeloid%20leukemia&author=M%20Pasquet&volume=121&publication_year=2013&pages=822-9&pmid=23223431&doi=10.1182/blood-2012-08-447367&)

28. Hahn CN, et al. Heritable GATA2 mutations associated with familial myelodysplastic syndrome and acute myeloid leukemia. Nat Genet. 2011;43:1012–7. doi: 10.1038/ng.913.  [DOI](https://doi.org/10.1038/ng.913) | [PMC free article](/articles/PMC3184204/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21892162/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Heritable%20GATA2%20mutations%20associated%20with%20familial%20myelodysplastic%20syndrome%20and%20acute%20myeloid%20leukemia&author=CN%20Hahn&volume=43&publication_year=2011&pages=1012-7&pmid=21892162&doi=10.1038/ng.913&)

29. Davies SM, et al. Pharmacogenetics of minimal residual disease response in children with B-precursor acute lymphoblastic leukemia: a report from the Children’s Oncology Group. Blood. 2008;111:2984–90. doi: 10.1182/blood-2007-09-114082.  [DOI](https://doi.org/10.1182/blood-2007-09-114082) | [PMC free article](/articles/PMC2265447/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18182569/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Pharmacogenetics%20of%20minimal%20residual%20disease%20response%20in%20children%20with%20B-precursor%20acute%20lymphoblastic%20leukemia:%20a%20report%20from%20the%20Children%E2%80%99s%20Oncology%20Group&author=SM%20Davies&volume=111&publication_year=2008&pages=2984-90&pmid=18182569&doi=10.1182/blood-2007-09-114082&)

30. Huang RS, et al. A genome-wide approach to identify genetic variants that contribute to etoposide-induced cytotoxicity. Proc Natl Acad Sci U S A. 2007;104:9758–63. doi: 10.1073/pnas.0703736104.  [DOI](https://doi.org/10.1073/pnas.0703736104) | [PMC free article](/articles/PMC1887589/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17537913/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&title=A%20genome-wide%20approach%20to%20identify%20genetic%20variants%20that%20contribute%20to%20etoposide-induced%20cytotoxicity&author=RS%20Huang&volume=104&publication_year=2007&pages=9758-63&pmid=17537913&doi=10.1073/pnas.0703736104&)

31. Degner JF, et al. DNase I sensitivity QTLs are a major determinant of human expression variation. Nature. 2012;482:390–4. doi: 10.1038/nature10808.  [DOI](https://doi.org/10.1038/nature10808) | [PMC free article](/articles/PMC3501342/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22307276/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=DNase%20I%20sensitivity%20QTLs%20are%20a%20major%20determinant%20of%20human%20expression%20variation&author=JF%20Degner&volume=482&publication_year=2012&pages=390-4&pmid=22307276&doi=10.1038/nature10808&)

32. Borowitz MJ, et al. Clinical significance of minimal residual disease in childhood acute lymphoblastic leukemia and its relationship to other prognostic factors: a Children’s Oncology Group study. Blood. 2008;111:5477–85. doi: 10.1182/blood-2008-01-132837.  [DOI](https://doi.org/10.1182/blood-2008-01-132837) | [PMC free article](/articles/PMC2424148/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18388178/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Clinical%20significance%20of%20minimal%20residual%20disease%20in%20childhood%20acute%20lymphoblastic%20leukemia%20and%20its%20relationship%20to%20other%20prognostic%20factors:%20a%20Children%E2%80%99s%20Oncology%20Group%20study&author=MJ%20Borowitz&volume=111&publication_year=2008&pages=5477-85&pmid=18388178&doi=10.1182/blood-2008-01-132837&)

33. Shi J, et al. Common variants on chromosome 6p22.1 are associated with schizophrenia. Nature. 2009;460:753–7. doi: 10.1038/nature08192.  [DOI](https://doi.org/10.1038/nature08192) | [PMC free article](/articles/PMC2775422/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19571809/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Common%20variants%20on%20chromosome%206p22.1%20are%20associated%20with%20schizophrenia&author=J%20Shi&volume=460&publication_year=2009&pages=753-7&pmid=19571809&doi=10.1038/nature08192&)

34. Purcell SM, et al. Common polygenic variation contributes to risk of schizophrenia and bipolar disorder. Nature. 2009;460:748–52. doi: 10.1038/nature08185.  [DOI](https://doi.org/10.1038/nature08185) | [PMC free article](/articles/PMC3912837/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19571811/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Common%20polygenic%20variation%20contributes%20to%20risk%20of%20schizophrenia%20and%20bipolar%20disorder&author=SM%20Purcell&volume=460&publication_year=2009&pages=748-52&pmid=19571811&doi=10.1038/nature08185&)

35. Burchard EG, et al. Lower bronchodilator responsiveness in Puerto Rican than in Mexican subjects with asthma. Am J Respir Crit Care Med. 2004;169:386–92. doi: 10.1164/rccm.200309-1293OC.  [DOI](https://doi.org/10.1164/rccm.200309-1293OC) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14617512/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Respir%20Crit%20Care%20Med&title=Lower%20bronchodilator%20responsiveness%20in%20Puerto%20Rican%20than%20in%20Mexican%20subjects%20with%20asthma&author=EG%20Burchard&volume=169&publication_year=2004&pages=386-92&pmid=14617512&doi=10.1164/rccm.200309-1293OC&)

36. Pritchard JK, Stephens M, Donnelly P. Inference of population structure using multilocus genotype data. Genetics. 2000;155:945–59. doi: 10.1093/genetics/155.2.945.  [DOI](https://doi.org/10.1093/genetics/155.2.945) | [PMC free article](/articles/PMC1461096/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10835412/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genetics&title=Inference%20of%20population%20structure%20using%20multilocus%20genotype%20data&author=JK%20Pritchard&author=M%20Stephens&author=P%20Donnelly&volume=155&publication_year=2000&pages=945-59&pmid=10835412&doi=10.1093/genetics/155.2.945&)

37. Tibshirani R, Hastie T, Narasimhan B, Chu G. Diagnosis of multiple cancer types by shrunken centroids of gene expression. Proc Natl Acad Sci U S A. 2002;99:6567–72. doi: 10.1073/pnas.082099299.  [DOI](https://doi.org/10.1073/pnas.082099299) | [PMC free article](/articles/PMC124443/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12011421/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&title=Diagnosis%20of%20multiple%20cancer%20types%20by%20shrunken%20centroids%20of%20gene%20expression&author=R%20Tibshirani&author=T%20Hastie&author=B%20Narasimhan&author=G%20Chu&volume=99&publication_year=2002&pages=6567-72&pmid=12011421&doi=10.1073/pnas.082099299&)

38. Spielman RS, et al. Common genetic variants account for differences in gene expression among ethnic groups. Nat Genet. 2007;39:226–31. doi: 10.1038/ng1955.  [DOI](https://doi.org/10.1038/ng1955) | [PMC free article](/articles/PMC3005333/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17206142/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Common%20genetic%20variants%20account%20for%20differences%20in%20gene%20expression%20among%20ethnic%20groups&author=RS%20Spielman&volume=39&publication_year=2007&pages=226-31&pmid=17206142&doi=10.1038/ng1955&)
