# A genetic risk score is associated with statin-induced low-density lipoprotein cholesterol lowering

## Metadata
**Authors:** Maarten Leusink, Anke H Maitland-van der Zee, Bo Ding, Fotios Drenos, Erik PA van Iperen, Helen R Warren, Mark J Caulfield, L Adrienne Cupples, Mary Cushman, Aroon D Hingorani, Ron C Hoogeveen, G Kees Hovingh, Meena Kumari, Leslie A Lange, Patricia B Munroe, Fredrik Nyberg, Pamela J Schreiner, Suthesh Sivapalaratnam, Paul IW de Bakker, Anthonius de Boer, Brendan J Keating, Folkert W Asselbergs, N Charlotte Onland-Moret
**Journal:** Pharmacogenomics
**Date:** 2016 Apr 5
**DOI:** [10.2217/pgs.16.8](https://doi.org/10.2217/pgs.16.8)
**PMID:** 27045730
**PMCID:** PMC5558527
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5558527/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC5558527/pdf/pgs-17-583.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC5558527/pdf/pgs-17-583.pdf)

## Abstract

**Aim:** 
To find new genetic loci associated with statin response, and to investigate the association of a genetic risk score (GRS) with this outcome.

**Patients & methods:** 
In a discovery meta-analysis (five studies, 1991 individuals), we investigated the effects of approximately 50000 single nucleotide polymorphisms on statin response, following up associations with p < 1 × 10-4 (three independent studies, 5314 individuals). We further assessed the effect of a GRS based on SNPs in ABCG2, LPA and APOE.

**Results:** 
No new SNPs were found associated with statin response. The GRS was associated with reduced statin response: 0.0394 mmol/l per allele (95% CI: 0.0171–0.0617, p = 5.37 × 10-4).

**Conclusion:** 
The GRS was associated with statin response, but the small effect size (˜2% of the average low-density lipoprotein cholesterol reduction) limits applicability.

Keywords: : pharmacogenetics cholesterol, risk score, statin therapy

### Aim:

To find new genetic loci associated with statin response, and to investigate the association of a genetic risk score (GRS) with this outcome.

### Patients & methods:

In a discovery meta-analysis (five studies, 1991 individuals), we investigated the effects of approximately 50000 single nucleotide polymorphisms on statin response, following up associations with p < 1 × 10^-4^ (three independent studies, 5314 individuals). We further assessed the effect of a GRS based on SNPs in *ABCG2*, *LPA* and *APOE*.

### Results:

No new SNPs were found associated with statin response. The GRS was associated with reduced statin response: 0.0394 mmol/l per allele (95% CI: 0.0171–0.0617, p = 5.37 × 10^-4^).

### Conclusion:

The GRS was associated with statin response, but the small effect size (˜2% of the average low-density lipoprotein cholesterol reduction) limits applicability.

## Methods

### Study selection

In our focus on a gene-centric discovery approach, we selected studies that used the 50k Cardiochip array [[10](#B10)] for genotyping, having more than one measurement of LDLc, information on statin use and participants of European ancestry. We included five studies, the ARIC study, FHS, MESA, WHII and AMC-PAS. All these studies were observational. Replication of the results was sought in other studies, as detailed below. The baseline measurement was defined in these studies as the last measurement of LDLc before reporting use of statins. The baseline value is therefore not necessarily measured on the same visit for all participants. The on-statin measurement was defined as the first measurement of LDLc after the first report of statin use. We then calculated LDLc change (δ LDLc) by subtracting the baseline value from the on-statin value. Details of the set-up and analysis for each study can be found in the [Supplementary Material](http:///doi/suppl/10.2217/pgs.16.8/suppl_file/pgs-17-583-s1.docx). Informed consent for DNA analysis was received from each respective local institutional and/or national ethical review board.

### Genotyping & quality control

Genotyping was performed using the gene-centric IBC array [[10](#B10)]. Several studies have been published using this array and have confirmed previously established associations and identified new associations of SNPs with several phenotypes and disease outcomes, including blood pressure [[11](#B11)], coronary artery disease [[12](#B12),[13](#B13)], plasma lipids [[14](#B14)] and Type 2 diabetes (T2D) [[15](#B15)]. Array data were clustered into genotypes with Illumina BeadStudio software (Illumina, CA, USA). Quality-control filters were applied within each cohort at the sample and SNP levels and are described in the [Supplementary Methods](http:///doi/suppl/10.2217/pgs.16.8/suppl_file/pgs-17-583-s1.docx). SNPs with a call rate below 95% or Hardy–Weinberg p-value <1 × 10^-06^ were excluded from analysis. Only SNPs with a minor allele frequency >1% were included in this analysis.

### Association analysis

We used linear regression to test the association of a SNP with LDLc change. Association testing was performed using an additive genetic model, adjusting for age, sex, baseline LDLc and other relevant study covariates (see [Supplementary Material](http:///doi/suppl/10.2217/pgs.16.8/suppl_file/pgs-17-583-s1.docx)). Samples were excluded based on ancestry outliers by PCA, removals of duplicates and first and second degree relatives. Each study was further adjusted for population stratification using the principal component approach in EIGENSTRAT [[16](#B16)]. Adjustment for baseline LDLc was necessary because many of the SNPs previously associated with statin response are also associated with cholesterol levels before treatment with statins. Meta-analysis was performed with METAL [[17](#B17)], using the inverse variance method and a fixed-effects model and I² as a measure of between-study heterogeneity [[18](#B18)]. To maintain the conventional 5% false-positive rate, the appropriate multiple-testing-corrected threshold for statistical significance was set at p < 2.4 × 10^-6^, taking into account the approximately 20,500 independent tests generated for individuals of European descent [[19](#B19)]. We selected all SNPs reaching p < 1 × 10^-4^ for subsequent replication. Power calculations for the chip-wide association analysis were done using QUANTO [[20](#B20)], assuming a mean LDLc-reduction of 1.3 mmol/l with a SD of 0.6, and a minor allele frequency of 20%.

### Replication

Independent replication was attempted for associations with p < 1 × 10^-4^. Look-ups were performed in three additional studies, the UK and Irish subjects from the observational and the randomized arm of the ASCOT-UK RCT, ASCOT-UK OBS and the RCT JUPITER, with data from a total of 5314 individuals in the three studies. Details of characteristics and methodological details for cohorts can be found in the [Supplementary Material](http:///doi/suppl/10.2217/pgs.16.8/suppl_file/pgs-17-583-s1.docx). The association of the genetic risk score was also investigated in these replication studies.

### Risk score composition

We calculated a genetic risk score (genetic risk score) on the basis of previous findings related to statin response, selecting risk alleles achieving genome-wide significance (p < 5 × 10^-8^) in GWAS of LDLc change, replicated in at least one other study. In total, we selected three SNPs reported by Chasman *et al*. [[9](#B9)], independently found by Deshmukh *et al*. [[8](#B8)] and Tomlinson *et al*. [[21](#B21)]. These SNPs were rs2231142 (a proxy SNP with r^2^ = 0.92 for rs2199936 in *ABCG2*), rs10455872 in *LPA* and rs2075650 (a proxy SNP with r^2^ = 0.95 for rs71352238 in *APOE*). For all three loci, the lead SNP or a good proxy SNP (r^2^ > 0.8) was available. One of the studies, AMC-PAS, was not included in this analysis due to the unavailability of one of the SNPs in the risk score.

The approach used to calculate the genetic risk score was a simple count of the alleles decreasing statin response, in other words, lowering the efficacy and bringing the response closer to zero. The simple count approach, rather than weighted scores, was chosen as SNP effects found in GWAS were similar.

Association of the risk score with delta LDLc was investigated using linear regression, using the risk score as a continuous variable. Models were adjusted for age, sex, baseline LDLc and relevant study covariates (e.g., correction for population stratification with principal components) as specified in the [Supplementary Material](http:///doi/suppl/10.2217/pgs.16.8/suppl_file/pgs-17-583-s1.docx).

Study-specific estimates of the risk score effect (β) were subsequently combined using random effects meta-analysis with the inverse variance weighting method, using R [[22](#B22)] version 3.0.2 and the ‘meta’ package [[23](#B23)] 3.5–0. A p-value < 0.05 was considered to constitute a statistically significant effect. We used I² to quantify between-study heterogeneity [[18](#B18)].

## Results

### Characteristics of studies

For the discovery dataset, we included five studies with a total of 1991 participants ([Table 1](#T1)). This gave us 80% statistical power to detect effects of 0.125 mmol/l per allele, which is close to the effect sizes found by other studies [[9](#B9)]. LDLc levels were consistently lower on statin use compared with before statin use, with an average reduction of 1.18 mmol/l. In the replication phase, a total of 5314 participants in three studies were included with an average LDLc reduction of 1.25 mmol/l. For the risk score analysis, one study was excluded due to unavailable SNPs, and the total sample size was 7121.

### Table 1. . Baseline characteristics and statin effect in the discovery and replication phase studies.

| Study | N | Age | Males | BMI | Diabetes | Hypertension | Ever smoked | Type of statin | Before statin LDLc | On-statin LDLc | Change in LDLc |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| Discovery phase |  |  |  |  |  |  |  |  |  |  |  |
| ARIC | 819 | 55.3 (5.4) | 396 (48%) | 59.2 (6.0) | 102 (13%) | 281 (34%) | 475 (58%) | Mixed | 4.34 (1.03) | 3.48 (0.91) | -0.85 (0.98) |
| AMC-PAS | 184 | 43.7 (4.9) | 141 (77%) | 26.9 (4.2) | 22 (12%) | 69 (38%) | 154 (84%) | Mixed | 3.97 (1.15) | 2.46 (0.80) | -1.51 (1.11) |
| FHS | 395 | 46.6 (8.5) | 221 (56%) | 28.2 (4.6) | 66 (17%) | 205 (52%) | 280 (71%) | Mixed | 4.06 (0.93) | 2.92 (0.91) | -1.07 (0.93) |
| MESA | 303 | 63.7 (9.8) | 154 (51%) | 29.0 (5.2) | 32 (11%) | 145 (48%) | 167 (55%) | Mixed | 3.58 (0.82) | 2.31 (0.59) | -1.25 (0.72) |
| WHII | 290 | 58.2 (5.7) | 227 (78%) | 27.0 (3.5) | 21 (7.2%) | 67 (23%) | 169 (60%) | Mixed | 4.54 (0.97) | 2.57 (0.76) | -1.94 (0.91) |
| Replication phase |  |  |  |  |  |  |  |  |  |  |  |
| ASCOT OBS | 975 | 63.4 (8.0) | 882 (84%) | 29.0 (4.6) | 208 (21%) | 975 (100%) | 679 (70%) | Mixed | 3.38 (0.89) | 2.25 (0.72) | -1.13 (0.80) |
| ASCOT RCT | 894 | 64.2 (8.0) | 975 (89%) | 28.9 (5.1) | 185 (21%) | 894 (100%) | 624 (70%) | Atorvastatin | 3.46 (0.71) | 2.18 (0.60) | -1.28 (0.62) |
| JUPITER | 3445 | 65.8 (7.6) | 2356 (68%) | 29.5 (5.7) | 11 (0.32%) | 1938 (56%) | 436 (13%) | Rosuvastatin | 2.76 (0.44) | 1.41 (0.50) | -1.35 (0.54) |

Table 1 Caption: Data are given as the mean (standard deviation) for continuous variables, and n (%) for categorical variables unless indicated otherwise. Cholesterol values are in mmol/l. LDLc: Low-density lipoprotein cholesterol; N: Number of participant.

### Association analysis

A total of 37,465 SNPs in five cohort studies were tested in the association analysis. No SNPs reached the significance threshold of p < 2.4 × 10^-6^ ([Supplementary Figure 1](http:///doi/suppl/10.2217/pgs.16.8/suppl_file/pgs-17-583-s1.docx): QQ-plot, [Supplementary Figure 2](http:///doi/suppl/10.2217/pgs.16.8/suppl_file/pgs-17-583-s1.docx): Manhattan plot). One SNP reached p < 1 × 10^-4^ ([Table 2](#T2)), rs17171676 on chromosome 7, and was taken to the replication phase. This SNP was not statistically significantly associated in this replication phase (replication p > 0.05), nor in a joint analysis of the discovery and replication phase ([Table 2](#T2)).

### Table 2. . SNP associated with statin response in the chip-wide analysis at p < 1 × 10-4.

|   |   |   |   | Discovery |  | Replication |  | Overall |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| SNP | CHR | Position | Gene | Effect (95% CI) | p-value | Effect (95% CI) | p-value | Effect (95% CI) | p-value |
| rs17171676 | 7 | 40307154 | C7orf10 | -0.299 (-0.4504; -0.1492) | 9.57 × 10-5 | 0.0315 (-0.0292; 0.0922) | 0.3167 | -0.0160 (-0.0730; 0.0410) | 0.582 |

Table 2 Caption: Effect is in mmol/l. CHR: Chromosome; BP: Base pair, based on GRCh36.

### Association of genetic risk score

The distribution of the risk score ([Table 3](#T3)) shows that only 4% of the individuals of European ancestry have four or more risk alleles, with the majority (more than 95%) having one to three alleles. Although statins show a consistent average reduction in LDLc, a higher genetic risk score is associated with a smaller change in LDLc due to statin treatment. The risk score decreased the effect of statins on δ LDLc by 0.0394 mmol/l per allele (95% CI: 0.0171; 0.0617), with a p-value of 5.37 × 10^-4^ ([Figure 1](#F0001)). Heterogeneity was low in this analysis (I^2^ = 18%). When adjustment for baseline LDLc was not included, the effect size was similar, 0.0354 mmol/l per allele but with a larger 95% CI (0.0054; 0.0654), which is still statistically significant (p = 0. 00206). The effect size was larger in the observational studies (β = 0.0509 mmol/l per allele, 95% CI: 0.0172; 0.0847, p = 0.0031) than in the RCTs (β = 0.0391 mmol/l per allele, 95% CI: -0.0097; 0.0880, p = 0.117).

### Table 3. . Distribution of genetic risk score in samples of ARIC study, MESA and FHS.

| 0 alleles | 1 alleles | 2 alleles | 3 alleles | 4 alleles | 5 alleles | 6 alleles |
| --- | --- | --- | --- | --- | --- | --- |
| 0.66% | 13.07% | 58.10% | 24.11% | 3.84% | 0.22% | 0.01% |

Table 3 Caption: Risk alleles were defined as alleles associated with smaller LDLc reduction in the genes ABCG2, LPA and APOE.

### Figure 1. . Forest plot for the association between the genetic risk score and low-density lipoprotein cholesterol reduction.

![Figure 1. ](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/827c/5558527/c800f447c085/pgs-17-583-g1.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5558527_pgs-17-583-g1.jpg)

Analyses were adjusted for baseline LDLc values. LDLc: Low-density lipoprotein cholesterol.

## Discussion

Using data from six population-based cohort studies and two RCTs, with a total of 7305 participants, we investigated the effect of common genetic variation on the LDLc response to statins using a cardiovascular gene-centric SNP array. The LDLc lowering effect of statins we found in the whole population, -1.30 mmol/l, is in the range of the values reported in a large meta-analyses of clinical trials (from -1.08 to -1.8 mmol/l) [[1](#B1),[24](#B24)].

No new loci were found to be associated to LDLc response to statins. Although one locus reached a p-value of p < 1 × 10^-4^, it could be replicated. These results are consistent with a genetic model involving multiple genes, each with modest effects, on LDLc change. The observation that GWAS have revealed robust associations with statin-induced response suggests that there are genuine genetic factors, but that much larger samples will be needed to find them with unequivocal evidence.

We also investigated the association of a genetic risk score, consisting of SNPs previously associated with statin response at the genome-wide level, with statin-induced LDLc lowering. The direction of effect of the risk score agreed with what was previously found. However, the effect was small, 0.0313 mmol/l per extra allele, which is about 2% of the total lowering of LDLc we found in the studies included in our analysis. Furthermore, more than 95% of individuals with European ancestry only have 1–3 risk alleles, resulting in a maximum of 0.09 mmol/l difference in statin response. The small effect of the risk score, the modest effective range of a risk score based on three SNPs, and the fact that LDLc was substantially reduced in most participants starting statin therapy, suggest that the possible role of genetics in the prediction of statin efficacy is small. The same conclusion was also reached by investigators studying the effects of single SNPs [[8](#B8),[9](#B9),[25](#B25)]. This study adds to the existing knowledge that taking the genetic background of a patient into account when prescribing statins, based on the known SNPs that affect statin-efficacy, does not seem justified. Although larger sample sizes in future studies may well result in the identification of additional loci that influence LDLc response, it seems unlikely that a revised genetic risk score would explain a sufficiently large fraction of the observed interindividual variation in LDLc response for it to be clinically useful. These limitations notwithstanding, genetic research into adverse events such as myopathy [[26](#B26)], and rhabdomyolysis might be used to predict and prevent these side effects, as has been done for other drugs and side effects, as effect sizes for adverse drug reactions may be much larger [[27](#B27),[28](#B28)].

It is important to distinguish between direct genetic effects on (baseline) LDLc and genetic effects on statin-induced LDLc reduction. These can be difficult to separate when there are also associations with baseline LDLc, as is the case for the SNPs used in the risk score [[5](#B5),[9](#B9)]. Adjusting for baseline LDLc values slightly increased the effect of the risk score, and narrowed the 95% CI. This indicates that associations with LDLc change can be obscured by effects of SNPs on baseline LDLc, and underscores the importance of this adjustment.

A strong point of this study is that it includes evidence from both observational cohort studies and clinical trials, demonstrating that the genetic risk score has a similar effect in both types of studies. A limitation of this study, however, is that we were not able to investigate the genetic effects for the different types of statins separately. This is partly a result of lack of data: the studies in the discovery phase have incomplete dosing and statin type data or complete absence of it. This is different from the data of clinical trials ASCOT and JUPITER (10 mg/day atorvastatin and 20 mg/dat rosuvastatin, respectively). However, when we would restrict our analyses to those subjects with information on dose and type of statin, the power reduction due to a lower participant count would negatively impact the possible identification of SNPs affecting statin effectiveness. Further, the study is small compared with meta-analyses of main SNP effects, which now typically include more than 100,000 participants [[5](#B5)]. This is a result of the fact that only a subset of the participants in observational studies (and also of placebo-controlled statin trials) is prescribed statin medication. Additionally, several population-based studies only have cholesterol measures at only one time point, excluding the possibility of investigating longitudinal changes in LDLc, while other studies do not have (reliable) information on drug use. For pharmacogenetic research, which is hampered by relatively small sample sizes [[29](#B29)], it would be valuable if drug use data and longitudinal follow-up data were routinely collected in large samples.

## Conclusion

In summary, we investigated whether new individual SNPs associated with statin-induced change in LDLc levels could be identified from a gene-centric discovery approach; we further investigated whether a genetic risk score, composed of SNPs previously associated with statin response, was associated with statin-induced change in LDLc levels. We did not find new SNPs associated with the magnitude of statin-induced LDLc reduction, but we found that there is a small but statistically significant association of the risk score with statin-induced LDLc reduction. However, use of statins lowers LDLc substantially regardless of genotype, and each additional allele in the genetic risk score decreases the total LDLc change only by 2%. Therefore, genotype-based clinical decision making for statin therapy is unlikely to improve efficacy, and at this moment there is no role for genetic testing in clinical practice to guide statin treatment.

## Future perspective

To date, several polymorphisms have been found to affect response to statin therapy based on the achieved lowering of LDLc. A further increase in the number of loci associated with LDLc response can probably be achieved with larger studies using GWAS methodology, although large numbers of statin users are required to effectively perform these. Because of the small effect sizes identified, there does not seem to be a genetically identifiable subgroup in which statins do not lower LDLc. The utility of genetic research into statin response might lie more in a better understanding of pharmacokinetics and pharmacodynamics, and ultimately the development of new drugs, than in direct clinical application (e.g., choosing a drug type or dose). Alternatively, research into the (prediction of) adverse drug reactions side effects of statin therapy might have a faster impact on clinical decisions.

#### Background

#### Background

#### Background

#### Patients & methods

#### Results: main findings

## Supplementary Material

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Mihaylova B, Emberson J, Blackwell L, et al. The effects of lowering LDL cholesterol with statin therapy in people at low risk of vascular disease: meta-analysis of individual data from 27 randomised trials. Lancet. 2012;380(9841):581–590. doi: 10.1016/S0140-6736(12)60367-5.  [DOI](https://doi.org/10.1016/S0140-6736(12)60367-5) | [PMC free article](/articles/PMC3437972/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22607822/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=The%20effects%20of%20lowering%20LDL%20cholesterol%20with%20statin%20therapy%20in%20people%20at%20low%20risk%20of%20vascular%20disease:%20meta-analysis%20of%20individual%20data%20from%2027%20randomised%20trials&author=B%20Mihaylova&author=J%20Emberson&author=L%20Blackwell&volume=380&issue=9841&publication_year=2012&pages=581-590&pmid=22607822&doi=10.1016/S0140-6736(12)60367-5&)

2. Baigent C, Blackwell L, Emberson J, et al. Efficacy and safety of more intensive lowering of LDL cholesterol: a meta-analysis of data from 170,000 participants in 26 randomised trials. Lancet. 2010;376(9752):1670–1681. doi: 10.1016/S0140-6736(10)61350-5.  [DOI](https://doi.org/10.1016/S0140-6736(10)61350-5) | [PMC free article](/articles/PMC2988224/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21067804/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Efficacy%20and%20safety%20of%20more%20intensive%20lowering%20of%20LDL%20cholesterol:%20a%20meta-analysis%20of%20data%20from%20170,000%20participants%20in%2026%20randomised%20trials&author=C%20Baigent&author=L%20Blackwell&author=J%20Emberson&volume=376&issue=9752&publication_year=2010&pages=1670-1681&pmid=21067804&doi=10.1016/S0140-6736(10)61350-5&)

3. Chasman DI, Posada D, Subrahmanyan L, Cook NR, Stanton VP, Ridker PM. Pharmacogenetic study of statin therapy and cholesterol reduction. JAMA. 2004;291(23):2821–2827. doi: 10.1001/jama.291.23.2821.  [DOI](https://doi.org/10.1001/jama.291.23.2821) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15199031/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA&title=Pharmacogenetic%20study%20of%20statin%20therapy%20and%20cholesterol%20reduction&author=DI%20Chasman&author=D%20Posada&author=L%20Subrahmanyan&author=NR%20Cook&author=VP%20Stanton&volume=291&issue=23&publication_year=2004&pages=2821-2827&pmid=15199031&doi=10.1001/jama.291.23.2821&)

4. Mega JL, Morrow DA, Brown A, Cannon CP, Sabatine MS. Identification of genetic variants associated with response to statin therapy. Arterioscler. Thromb. Vasc. Biol. 2009;29(9):1310–1315. doi: 10.1161/ATVBAHA.109.188474.  [DOI](https://doi.org/10.1161/ATVBAHA.109.188474) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19667110/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arterioscler.%20Thromb.%20Vasc.%20Biol.&title=Identification%20of%20genetic%20variants%20associated%20with%20response%20to%20statin%20therapy&author=JL%20Mega&author=DA%20Morrow&author=A%20Brown&author=CP%20Cannon&author=MS%20Sabatine&volume=29&issue=9&publication_year=2009&pages=1310-1315&pmid=19667110&doi=10.1161/ATVBAHA.109.188474&)

5. Teslovich TM, Musunuru K, Smith AV, et al. Biological, clinical and population relevance of 95 loci for blood lipids. Nature. 2010;466(7307):707–713. doi: 10.1038/nature09270.  [DOI](https://doi.org/10.1038/nature09270) | [PMC free article](/articles/PMC3039276/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20686565/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Biological,%20clinical%20and%20population%20relevance%20of%2095%20loci%20for%20blood%20lipids&author=TM%20Teslovich&author=K%20Musunuru&author=AV%20Smith&volume=466&issue=7307&publication_year=2010&pages=707-713&pmid=20686565&doi=10.1038/nature09270&)

6. Voora D, Shah SH, Reed CR, et al. Pharmacogenetic predictors of statin-mediated low-density lipoprotein cholesterol reduction and dose response. Circ. Cardiovasc. Genet. 2008;1(2):100–106. doi: 10.1161/CIRCGENETICS.108.795013.  [DOI](https://doi.org/10.1161/CIRCGENETICS.108.795013) | [PMC free article](/articles/PMC2995295/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20031551/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circ.%20Cardiovasc.%20Genet.&title=Pharmacogenetic%20predictors%20of%20statin-mediated%20low-density%20lipoprotein%20cholesterol%20reduction%20and%20dose%20response&author=D%20Voora&author=SH%20Shah&author=CR%20Reed&volume=1&issue=2&publication_year=2008&pages=100-106&pmid=20031551&doi=10.1161/CIRCGENETICS.108.795013&)

7. Thompson JF, Hyde CL, Wood LS, et al. Comprehensive whole-genome and candidate gene analysis for response to statin therapy in the treating to new targets (TNT) cohort. Circ. Cardiovasc. Genet. 2009;2(2):173–181. doi: 10.1161/CIRCGENETICS.108.818062.  [DOI](https://doi.org/10.1161/CIRCGENETICS.108.818062) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20031582/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circ.%20Cardiovasc.%20Genet.&title=Comprehensive%20whole-genome%20and%20candidate%20gene%20analysis%20for%20response%20to%20statin%20therapy%20in%20the%20treating%20to%20new%20targets%20(TNT)%20cohort&author=JF%20Thompson&author=CL%20Hyde&author=LS%20Wood&volume=2&issue=2&publication_year=2009&pages=173-181&pmid=20031582&doi=10.1161/CIRCGENETICS.108.818062&)

8. Deshmukh HA, Colhoun HM, Johnson T, et al. Genome-wide association study of genetic determinants of LDL-C response to atorvastatin therapy: importance of Lp(a) J. Lipid Res. 2012;53(5):1000–1011. doi: 10.1194/jlr.P021113.  [DOI](https://doi.org/10.1194/jlr.P021113) | [PMC free article](/articles/PMC3329377/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22368281/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Lipid%20Res.&title=Genome-wide%20association%20study%20of%20genetic%20determinants%20of%20LDL-C%20response%20to%20atorvastatin%20therapy:%20importance%20of%20Lp(a)&author=HA%20Deshmukh&author=HM%20Colhoun&author=T%20Johnson&volume=53&issue=5&publication_year=2012&pages=1000-1011&pmid=22368281&doi=10.1194/jlr.P021113&)

9. Chasman DI, Giulianini F, MacFadyen J, Barratt BJ, Nyberg F, Ridker PM. Genetic determinants of statin-induced low-density lipoprotein cholesterol reduction: the Justification for the Use of Statins in Prevention: an Intervention Trial Evaluating Rosuvastatin (JUPITER) trial. Circ. Cardiovasc. Genet. 2012;5(2):257–264. doi: 10.1161/CIRCGENETICS.111.961144.  [DOI](https://doi.org/10.1161/CIRCGENETICS.111.961144) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22331829/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circ.%20Cardiovasc.%20Genet.&title=Genetic%20determinants%20of%20statin-induced%20low-density%20lipoprotein%20cholesterol%20reduction:%20the%20Justification%20for%20the%20Use%20of%20Statins%20in%20Prevention:%20an%20Intervention%20Trial%20Evaluating%20Rosuvastatin%20(JUPITER)%20trial&author=DI%20Chasman&author=F%20Giulianini&author=J%20MacFadyen&author=BJ%20Barratt&author=F%20Nyberg&volume=5&issue=2&publication_year=2012&pages=257-264&pmid=22331829&doi=10.1161/CIRCGENETICS.111.961144&)

10. Keating BJ, Tischfield S, Murray SS, et al. Concept, design and implementation of a cardiovascular gene-centric 50 k SNP array for large-scale genomic association studies. PLoS ONE. 2008;3(10):e3583. doi: 10.1371/journal.pone.0003583.  [DOI](https://doi.org/10.1371/journal.pone.0003583) | [PMC free article](/articles/PMC2571995/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18974833/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&title=Concept,%20design%20and%20implementation%20of%20a%20cardiovascular%20gene-centric%2050%20k%20SNP%20array%20for%20large-scale%20genomic%20association%20studies&author=BJ%20Keating&author=S%20Tischfield&author=SS%20Murray&volume=3&issue=10&publication_year=2008&pages=e3583&pmid=18974833&doi=10.1371/journal.pone.0003583&)

11. Johnson T, Gaunt TR, Newhouse SJ, et al. Blood pressure loci identified with a gene-centric array. Am. J. Hum. Genet. 2011;89(6):688–700. doi: 10.1016/j.ajhg.2011.10.013.  [DOI](https://doi.org/10.1016/j.ajhg.2011.10.013) | [PMC free article](/articles/PMC3234370/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22100073/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Hum.%20Genet.&title=Blood%20pressure%20loci%20identified%20with%20a%20gene-centric%20array&author=T%20Johnson&author=TR%20Gaunt&author=SJ%20Newhouse&volume=89&issue=6&publication_year=2011&pages=688-700&pmid=22100073&doi=10.1016/j.ajhg.2011.10.013&)

12. Clarke R, Peden JF, Hopewell JC, et al. Genetic variants associated with Lp(a) lipoprotein level and coronary disease. N. Engl. J. Med. 2009;361(26):2518–2528. doi: 10.1056/NEJMoa0902604.  [DOI](https://doi.org/10.1056/NEJMoa0902604) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20032323/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N.%20Engl.%20J.%20Med.&title=Genetic%20variants%20associated%20with%20Lp(a)%20lipoprotein%20level%20and%20coronary%20disease&author=R%20Clarke&author=JF%20Peden&author=JC%20Hopewell&volume=361&issue=26&publication_year=2009&pages=2518-2528&pmid=20032323&doi=10.1056/NEJMoa0902604&)

13. Butterworth A, Braund P, Farrall M, et al. Large-scale gene-centric analysis identifies novel variants for coronary artery disease. PLoS Genet. 2011;7:e1002260. doi: 10.1371/journal.pgen.1002260.  [DOI](https://doi.org/10.1371/journal.pgen.1002260) | [PMC free article](/articles/PMC3178591/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21966275/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet.&title=Large-scale%20gene-centric%20analysis%20identifies%20novel%20variants%20for%20coronary%20artery%20disease&author=A%20Butterworth&author=P%20Braund&author=M%20Farrall&volume=7&publication_year=2011&pages=e1002260&pmid=21966275&doi=10.1371/journal.pgen.1002260&)

14. Asselbergs FW, Guo Y, van Iperen EPA, et al. Large-scale gene-centric meta-analysis across 32 studies identifies multiple lipid loci. Am. J. Hum. Genet. 2012;91(5):823–838. doi: 10.1016/j.ajhg.2012.08.032.  [DOI](https://doi.org/10.1016/j.ajhg.2012.08.032) | [PMC free article](/articles/PMC3487124/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23063622/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Hum.%20Genet.&title=Large-scale%20gene-centric%20meta-analysis%20across%2032%20studies%20identifies%20multiple%20lipid%20loci&author=FW%20Asselbergs&author=Y%20Guo&author=EPA%20van%20Iperen&volume=91&issue=5&publication_year=2012&pages=823-838&pmid=23063622&doi=10.1016/j.ajhg.2012.08.032&)

15. Saxena R, Elbers CC, Guo Y, et al. Large-scale gene-centric meta-analysis across 39 studies identifies Type 2 diabetes loci. Am. J. Hum. Genet. 2012;90(3):410–425. doi: 10.1016/j.ajhg.2011.12.022.  [DOI](https://doi.org/10.1016/j.ajhg.2011.12.022) | [PMC free article](/articles/PMC3309185/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22325160/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Hum.%20Genet.&title=Large-scale%20gene-centric%20meta-analysis%20across%2039%20studies%20identifies%20Type%202%20diabetes%20loci&author=R%20Saxena&author=CC%20Elbers&author=Y%20Guo&volume=90&issue=3&publication_year=2012&pages=410-425&pmid=22325160&doi=10.1016/j.ajhg.2011.12.022&)

16. Price AL, Patterson NJ, Plenge RM, Weinblatt ME, Shadick NA, Reich D. Principal components analysis corrects for stratification in genome-wide association studies. Nature. 2006;38(8):904–909. doi: 10.1038/ng1847.  [DOI](https://doi.org/10.1038/ng1847) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16862161/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Principal%20components%20analysis%20corrects%20for%20stratification%20in%20genome-wide%20association%20studies&author=AL%20Price&author=NJ%20Patterson&author=RM%20Plenge&author=ME%20Weinblatt&author=NA%20Shadick&volume=38&issue=8&publication_year=2006&pages=904-909&pmid=16862161&doi=10.1038/ng1847&)

17. Willer CJ, Li Y, Abecasis GR. METAL: fast and efficient meta-analysis of genomewide association scans. Bioinformatics. 2010;26(17):2190–2191. doi: 10.1093/bioinformatics/btq340.  [DOI](https://doi.org/10.1093/bioinformatics/btq340) | [PMC free article](/articles/PMC2922887/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20616382/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=METAL:%20fast%20and%20efficient%20meta-analysis%20of%20genomewide%20association%20scans&author=CJ%20Willer&author=Y%20Li&author=GR%20Abecasis&volume=26&issue=17&publication_year=2010&pages=2190-2191&pmid=20616382&doi=10.1093/bioinformatics/btq340&)

18. Higgins JPT, Thompson SG, Deeks JJ, Altman DG. Measuring inconsistency in meta-analyses. BMJ. 2003;327(7414):557–560. doi: 10.1136/bmj.327.7414.557.  [DOI](https://doi.org/10.1136/bmj.327.7414.557) | [PMC free article](/articles/PMC192859/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12958120/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMJ&title=Measuring%20inconsistency%20in%20meta-analyses&author=JPT%20Higgins&author=SG%20Thompson&author=JJ%20Deeks&author=DG%20Altman&volume=327&issue=7414&publication_year=2003&pages=557-560&pmid=12958120&doi=10.1136/bmj.327.7414.557&)

19. Lanktree MB, Guo Y, Murtaza M, et al. Meta-analysis of dense genecentric association studies reveals common and uncommon variants associated with height. Am. J. Hum. Genet. 2011;88(1):6–18. doi: 10.1016/j.ajhg.2010.11.007.  [DOI](https://doi.org/10.1016/j.ajhg.2010.11.007) | [PMC free article](/articles/PMC3014369/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21194676/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Hum.%20Genet.&title=Meta-analysis%20of%20dense%20genecentric%20association%20studies%20reveals%20common%20and%20uncommon%20variants%20associated%20with%20height&author=MB%20Lanktree&author=Y%20Guo&author=M%20Murtaza&volume=88&issue=1&publication_year=2011&pages=6-18&pmid=21194676&doi=10.1016/j.ajhg.2010.11.007&)

20. Gauderman WJ. Sample size requirements for association studies of gene-gene interaction. Am. J. Epidemiol. 2002;155(5):478–484. doi: 10.1093/aje/155.5.478.  [DOI](https://doi.org/10.1093/aje/155.5.478) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11867360/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Epidemiol.&title=Sample%20size%20requirements%20for%20association%20studies%20of%20gene-gene%20interaction&author=WJ%20Gauderman&volume=155&issue=5&publication_year=2002&pages=478-484&pmid=11867360&doi=10.1093/aje/155.5.478&)

21. Tomlinson B, Hu M, Lee VWY, et al. ABCG2 polymorphism is associated with the low-density lipoprotein cholesterol response to rosuvastatin. Clin. Pharmacol. Ther. 2010;87(5):558–562. doi: 10.1038/clpt.2009.232.  [DOI](https://doi.org/10.1038/clpt.2009.232) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20130569/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=ABCG2%20polymorphism%20is%20associated%20with%20the%20low-density%20lipoprotein%20cholesterol%20response%20to%20rosuvastatin&author=B%20Tomlinson&author=M%20Hu&author=VWY%20Lee&volume=87&issue=5&publication_year=2010&pages=558-562&pmid=20130569&doi=10.1038/clpt.2009.232&)

22. R Core Team (2015). R: a language and environment for statistical computing. R Foundation for Statistical Computing, Vienna, Austria. 2012  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=R%20Foundation%20for%20Statistical%20Computing,%20Vienna,%20Austria&title=R%20Core%20Team%20(2015).%20R:%20a%20language%20and%20environment%20for%20statistical%20computing&publication_year=2012&)

23. Schwarzer G. Meta: meta-analysis with R. 2012. https://cran.r-project.org/web/packages/meta/index.html  [https://cran.r-project.org/web/packages/meta/index.html](https://www.cran.r-project.org/web/packages/meta/index.html)

24. Law MR, Wald NJ, Rudnicka AR. Quantifying effect of statins on low density lipoprotein cholesterol, ischaemic heart disease, and stroke: systematic review and meta-analysis. BMJ. 2003;326:1423. doi: 10.1136/bmj.326.7404.1423.  [DOI](https://doi.org/10.1136/bmj.326.7404.1423) | [PMC free article](/articles/PMC162260/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12829554/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMJ&title=Quantifying%20effect%20of%20statins%20on%20low%20density%20lipoprotein%20cholesterol,%20ischaemic%20heart%20disease,%20and%20stroke:%20systematic%20review%20and%20meta-analysis&author=MR%20Law&author=NJ%20Wald&author=AR%20Rudnicka&volume=326&publication_year=2003&pages=1423&pmid=12829554&doi=10.1136/bmj.326.7404.1423&)

25. Hopewell JC, Parish S, Offer A, et al. Impact of common genetic variation on response to simvastatin therapy among 18 705 participants in the Heart Protection Study. Eur. Heart J. 2013;34(13):982–992. doi: 10.1093/eurheartj/ehs344.  [DOI](https://doi.org/10.1093/eurheartj/ehs344) | [PMC free article](/articles/PMC3612775/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23100282/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur.%20Heart%20J.&title=Impact%20of%20common%20genetic%20variation%20on%20response%20to%20simvastatin%20therapy%20among%2018%20705%20participants%20in%20the%20Heart%20Protection%20Study&author=JC%20Hopewell&author=S%20Parish&author=A%20Offer&volume=34&issue=13&publication_year=2013&pages=982-992&pmid=23100282&doi=10.1093/eurheartj/ehs344&)

26. Link E, Parish S, Armitage J, et al. SLCO1B1 variants and statin-induced myopathy – a genomewide study. N. Engl. J. Med. 2008;359(8):789–799. doi: 10.1056/NEJMoa0801936.  [DOI](https://doi.org/10.1056/NEJMoa0801936) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18650507/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N.%20Engl.%20J.%20Med.&title=SLCO1B1%20variants%20and%20statin-induced%20myopathy%20%E2%80%93%20a%20genomewide%20study&author=E%20Link&author=S%20Parish&author=J%20Armitage&volume=359&issue=8&publication_year=2008&pages=789-799&pmid=18650507&doi=10.1056/NEJMoa0801936&)

27. Daly AK, Donaldson PT, Bhatnagar P, et al. HLA-B*5701 genotype is a major determinant of drug-induced liver injury due to flucloxacillin. Nat. Genet. 2009;41(7):816–819. doi: 10.1038/ng.379.  [DOI](https://doi.org/10.1038/ng.379) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19483685/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet.&title=HLA-B*5701%20genotype%20is%20a%20major%20determinant%20of%20drug-induced%20liver%20injury%20due%20to%20flucloxacillin&author=AK%20Daly&author=PT%20Donaldson&author=P%20Bhatnagar&volume=41&issue=7&publication_year=2009&pages=816-819&pmid=19483685&doi=10.1038/ng.379&)

28. Chung W-H, Hung S-I, Hong H-S, et al. Medical genetics: a marker for Stevens–Johnson syndrome. Nature. 2004;428(6982):486. doi: 10.1038/428486a.  [DOI](https://doi.org/10.1038/428486a) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15057820/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Medical%20genetics:%20a%20marker%20for%20Stevens%E2%80%93Johnson%20syndrome&author=W-H%20Chung&author=S-I%20Hung&author=H-S%20Hong&volume=428&issue=6982&publication_year=2004&pages=486&pmid=15057820&doi=10.1038/428486a&)

29. Motsinger-Reif AA, Jorgenson E, Relling MV, et al. Genome-wide association studies in pharmacogenomics: successes and lessons. Pharmacogenet. Genomics. 2013;23(8):383–394. doi: 10.1097/FPC.0b013e32833d7b45.  [DOI](https://doi.org/10.1097/FPC.0b013e32833d7b45) | [PMC free article](/articles/PMC3003940/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20639796/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet.%20Genomics&title=Genome-wide%20association%20studies%20in%20pharmacogenomics:%20successes%20and%20lessons&author=AA%20Motsinger-Reif&author=E%20Jorgenson&author=MV%20Relling&volume=23&issue=8&publication_year=2013&pages=383-394&pmid=20639796&doi=10.1097/FPC.0b013e32833d7b45&)
