# Genotypic and Phenotypic Factors Influencing Drug Response in Mexican Patients With Type 2 Diabetes Mellitus

## Metadata
**Authors:** Hector E Sanchez-Ibarra, Luisa M Reyes-Cortes, Xian-Li Jiang, Claudia M Luna-Aguirre, Dionicio Aguirre-Trevino, Ivan A Morales-Alvarado, Rafael B Leon-Cachon, Fernando Lavalle-Gonzalez, Faruck Morcos, Hugo A Barrera-Saldaña
**Journal:** Frontiers in Pharmacology
**Date:** 2018 Apr 6
**DOI:** [10.3389/fphar.2018.00320](https://doi.org/10.3389/fphar.2018.00320)
**PMID:** 29681852
**PMCID:** PMC5898372
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5898372/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC5898372/pdf/fphar-09-00320.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC5898372/pdf/fphar-09-00320.pdf)

## Abstract

The treatment of Type 2 Diabetes Mellitus (T2DM) consists primarily of oral antidiabetic drugs (OADs) that stimulate insulin secretion, such as sulfonylureas (SUs) and reduce hepatic glucose production (e.g., biguanides), among others. The marked inter-individual differences among T2DM patients’ response to these drugs have become an issue on prescribing and dosing efficiently. In this study, fourteen polymorphisms selected from Genome-wide association studies (GWAS) were screened in 495 T2DM Mexican patients previously treated with OADs to find the relationship between the presence of these polymorphisms and response to the OADs. Then, a novel association screening method, based on global probabilities, was used to globally characterize important relationships between the drug response to OADs and genetic and clinical parameters, including polymorphisms, patient information, and type of treatment. Two polymorphisms, ABCC8-Ala1369Ser and KCNJ11-Glu23Lys, showed a significant impact on response to SUs. Heterozygous ABCC8-Ala1369Ser variant (A/C) carriers exhibited a higher response to SUs compared to homozygous ABCC8-Ala1369Ser variant (A/A) carriers (p-value = 0.029) and to homozygous wild-type genotypes (C/C) (p-value = 0.012). The homozygous KCNJ11-Glu23Lys variant (C/C) and wild-type (T/T) genotypes had a lower response to SUs compared to heterozygous (C/T) carriers (p-value = 0.039). The screening of OADs response related genetic and clinical factors could help improve the prescribing and dosing of OADs for T2DM patients and thus contribute to the design of personalized treatments.

Keywords: pharmacogenetics, pharmacogenomics, diabetes, sulfonylureas, biguanides, Mexican, direct coupling analysis, direct information

## Introduction

Type 2 Diabetes Mellitus (T2DM) is the most common form of diabetes in adults. T2DM is associated with multiple complications, such as blindness, lower limb amputation, and premature death ([Marchetti et al., 2009](#B25); [Barquera et al., 2013](#B5)). According to the International Diabetes Federation (IDF), China, India, United States, Brazil, Russia, and Mexico are the countries with the highest incidence. It is estimated that life expectancy is reduced in diabetic individuals by 5–10 years, mainly due to lack of early treatment. In Mexico, the average age for death by diabetes or its complications was 66.7 in 2010, compared with the lifespan of 76 years of non-diabetic individuals ([Agudelo-Botero and Davila-Cervantes, 2015](#B1)). The average annual economic cost from 2006 to 2010 of T2DM patients in Mexico was $941,345,886 USD of direct cost, $177,220,390 USD of indirect cost, and $27,969,427 USD from its complications. This immense cost, coupled with the issues of inequity and access to healthcare in Mexico, where 51% of the cost comes from household income, represents a huge social burden ([Arredondo and De Icaza, 2011](#B3); [Barquera et al., 2013](#B5)).

Several classes of oral antidiabetic drugs (OADs) are currently available and primarily include agents that stimulate insulin secretion (sulfonylureas), reduce hepatic glucose production (biguanides), delay the digestion and absorption of intestinal carbohydrate (alpha-glucosidase inhibitors), or improve insulin function (thiazolidinediones) ([Krentz and Bailey, 2005](#B23); [Nathan et al., 2009](#B30)). Additionally, OADs include other classes of drugs such as meglitinides, glucagon-like peptide-1 (GLP-1) agonists, dipeptidylpeptidase-4 (DPP-4) inhibitors, dopamine-2 agonists, and amylin analogs ([Inzucchi et al., 2012](#B21)). There is a wide variability in adverse events and glucose-lowering response to OADs among different patients, which may be attributed to factors like age, sex, and body weight, but also to genetic variation related to pharmacokinetic and pharmacodynamic properties of the OADs ([Becker et al., 2013](#B7); [Emami-Riedmaier et al., 2015](#B14)).

Biguanide, especially metformin, which is the only one available OAD in some countries, is recommended as the first-choice therapy for T2DM ([Inzucchi et al., 2012](#B21)). Metformin inhibits the activity of mitochondrial respiratory-chain complex I, resulting in decreased ATP synthesis and an accumulation of AMP leading to the activation of AMP-activated protein kinase (AMPK) and the subsequent suppression of hepatic gluconeogenesis ([Foretz et al., 2010](#B17)). Pharmacokinetic studies suggest that metformin is actively absorbed from the gut and is excreted unchanged in the urine ([Zhou et al., 2009](#B47)). The organic cation transporter 1 (OCT1), encoded by *SLC22A1* gene, is expressed in the basolateral membrane of hepatocytes and mediates the metformin uptake, while OCT2 (encoded by *SCL22A2*), expressed in the basolateral membrane of kidney tubular cells, facilitates almost 80% of metformin excretion ([Pearson, 2009](#B31); [Pernicova and Korbonits, 2014](#B32)). Associations of intronic variants in *SLC22A1* and *SLC22A2* with glucose-lowering response to metformin in T2DM patients have been previously reported ([Tkac et al., 2013](#B42)). *SLC22A1* gene is highly polymorphic, with common function-reducing polymorphisms such as Arg61Cys (rs12208357), Gly401Ser (rs34130495), and Gly465Arg (rs34059508), which having been associated with decreased transportation and therefore the reduced therapeutic effect of metformin ([Distefano and Watanabe, 2010](#B12)). *In vitro* studies have shown that all three polymorphisms might be associated with reduced metformin uptake ([van Dam et al., 2005](#B45)). However, *in vivo* studies show controversial results ([Tzvetkov et al., 2009](#B44)).

Sulfonylureas (SUs) target an ATP-dependent potassium (K-_ATP_) channel present in pancreatic β-cells. K-_ATP_ channels are hetero-octamers composed of Kir6.2 pore subunit encoded by the gene *KCNJ11*, and the SUR1 receptor subunit encoded by the gene *ABCC8*. SUs lower glycemia by enhancing insulin secretion from pancreatic β-cells by inducing K-_ATP_ channel closure ([Tkac, 2015](#B41)). SUs, such as tolbutamide, glimepiride, and glipizide, are mainly metabolized by the enzyme cytochrome P450 encoded by the *CYP2C9* isoform gene. Several SNPs have been related to their effect on insulin secretion enhancing ([Holstein et al., 2005](#B19)). Reduced drug-metabolizing activity has been reported in individuals carrying two allelic variants namely *CYP2C9*^∗^2 (rs1799853) leading to a missense amino acid polymorphism Arg144Cys, and *CYP2C9*^∗^3 (rs1057910) leading to the missense amino acid polymorphism Ile359Leu ([Huang and Florez, 2011](#B20)). The Ile359Leu polymorphism has a more profound effect ([Ragia et al., 2014](#B34)). These alleles encode proteins with a diminished enzymatic activity and are correlated with elevated serum levels of SUs ([Ragia et al., 2009](#B33)). However, *CYP2C9*-Arg144Cys polymorphism is not associated with diabetes susceptibility ([Semiz et al., 2010](#B36)).

Regarding SUs target (K-_ATP_ channels), most studies researched two linked non-synonymous common variants in both *ABCC8* and *KCNJ11* genes. *KCNJ11* variants are implicated in glycemic progression to either prediabetes or T2DM. One of the most common *KCNJ11* polymorphisms is Glu23Lys (rs5219). The functional effects of the Glu23Lys variant on insulin secretion and sensitivity yield controversial results, even though recent larger studies demonstrate a significantly reduced insulin secretion, lower insulin levels, and improved insulin sensitivity, consistent with the enhanced K-_ATP_ channels activity in pancreatic β-cells ([Villareal et al., 2009](#B46)). More recently, the associations of the Glu23Lys variant and a different *KCNJ11* variant, Ile1337Val (rs5215), with T2DM have been confirmed in several genome-wide association studies (GWAS), rekindling the interest in its potential role as a genetic marker for T2DM development ([Cheung et al., 2011](#B11)). On the other hand, the *ABCC8*-Ala1369Ser (rs757110) polymorphism has been associated with a reduction of glycated hemoglobin (HbA1c) in the Chinese population with SUs treatment ([Feng et al., 2008](#B15); [Sokolova et al., 2015](#B37)).

In addition to pharmacogenetic factors, the response to OADs is conditional on different phenotypic or clinical aspects. With the accessibility of cohorts of this T2DM patient information, various statistical approaches can be used to determine the contributing factors affecting response to OADs. Traditional statistical tools are used to measure the co-occurrence of factor variable and treatment response at a time ([Turner et al., 2009](#B43); [Stransky et al., 2015](#B39)). However, the human trait factors may internally relate or function together to affect the drug response. Although these tests provide real statistical connections among variables in patient data, these relationships tend to be composed with both strong and weak correlations making it difficult to disentangle direct effects that explain the influence of some variables over a factor of interest. Therefore, important efforts have been dedicated to the development of statistical models to better describe relationship networks related to human disease. In the field of pharmacogenomics, a variety of statistical models have been built, such as Bayesian networks and Elastic net regression ([Barretina et al., 2012](#B6)), which have exhibited great performance on finding genes highly connected to drug response. Recently, a global statistical model, direct coupling analysis (DCA), also has been demonstrated to be applicable in pharmacogenetic data ([Jiang et al., 2017](#B22)). DCA efficiently computes estimates of a joint probability distribution of multivariate patient profiles constructed with clinical data. The parameters of such distribution estimated by DCA are used to quantify with high success the degree of connectivity of variables in the model. The ability to disentangle direct couplings from indirect couplings has been successful in the field of structural biology where directly coupled residue pairs have been used to predict co-evolution of amino acids ([dos Santos et al., 2015](#B13)), predict the structure of proteins ([Sulkowska et al., 2012](#B40)) with an accuracy not seen before as well as predict the molecular plasticity and complexes ([Morcos et al., 2013](#B27); [dos Santos et al., 2015](#B13)). Recently, we have used this framework to study protein expression level–based protein–protein interactions and in a pharmacogenomics approach to infer gene–drug interactions in cancer tissues and cell lines where information on drug sensitivity is available ([Jiang et al., 2017](#B22)). This is the first time that direct information (DI) is used as a metric of correlation in high throughput profiling data. It not only captures the connections between well-known drug response predictors, including some drug targets for certain anti-cancer agents, but also predicts some potential biomarkers and generates gene–drug networks. DCA is used in this study to find highly coupled factors for response to OADs and to construct a network for the patient cohort data. A metric called DI is computed to evaluate the association intensity of two variables, including the connections between two potential factors and between factors and drug response.

In addition to genetic variations traits containing pharmacogenetic data, the phenotypic traits of patients, such as age, sex, health status, have been suggested to have influences on the outcome of OADs treatment for T2DM. Thus, a T2DM patient database including genetic data and patient phenotypic data is advantageous. This study collects 495 T2DM patients with information about age, origin, sex, body index, health status, history of OADs treatment, polymorphisms, and results of glycated hemoglobin (HbA1c) tests. HbA1c is a recognized target for diabetes control used in international guidelines and is the most suitable parameter to be studied in pharmacogenetic studies ([Lo et al., 2012](#B24)). Here, we propose a new structure-learning approach for Bayesian network construction by using direct information and Chow-Liu trees. Chow-Liu algorithm is commonly used to learn Bayesian network structure ([Almudevar, 2010](#B2)), and mutual information is used by this algorithm to estimate the dependence of two variables ([Chen et al., 2008](#B10)). Due to the better performance of DI on measuring direct associations when compared to mutual information, we integrated DI and the Chow-Liu algorithm to recover global connections between clinical factors for T2DM patients.

Genetic variations or patient phenotypic data affecting the drug responses to T2DM treatments often lead to the necessity of treatment changes and adjustments, resulting in higher expenses for the patients. The aim of this study was to establish an association between patient clinical data, such as habits, treatment history, polymorphisms, and variability in the response to OAD treatments in a Mexican population. Therefore, biomarkers could help prescribe the right drug and its dosage, for better control of the disease and its consequences, including treatment savings and reduced impact in productivity.

## Materials and Methods

### Design

A cross-sectional and retrospective study with convenience sampling was carried out in T2DM patients treated with OADs, in monotherapy or in combination for at least 6 months, to determine possible association between patient data, gene variants, and drug response assessed by HbA1c values. This study was conducted according to Good Clinical Practice standards and guidelines of the Declarations of Helsinki and Tokyo. Furthermore, the protocol was approved by the Ethics and Research Committee from the Medical School of the Universidad Autonoma de Nuevo León (IRB00005579).

### Patients

We recruited male and female patients with T2DM from northeastern Mexico who attended the Clinic of Diabetes of the Endocrinology Service at the Dr. José Eleuterio González Hospital in Monterrey, Mexico. The recruitment period lasted 12 months. The inclusion criteria were: patients over 18 years old with T2DM and treated with oral antihyperglycemic agents or OADs, in monotherapy or in combination for at least 6 months. The exclusion criteria were: diabetes type 1, gestational diabetes, other non-T2DM types of diabetes, active cancer, heart failure, co-treatment with corticosteroids or estrogens, conditions that can cause hyperglycemia, addiction to alcohol or illegal drugs, and dementia or severe psychiatric disorders. The co-treatments with corticosteroids and estrogens were excluded. The disease status was confirmed using the American Diabetes Association criteria and a physical examination. Blood pressure, body height, and body weight measurement were done. The body mass index (BMI) was calculated from anthropometric measurements.

All patients were apprised about the aims of the study, and a written informed consent was obtained. In addition, information on the history of diabetes and the presence of arterial hypertension, hyperlipidemia, and chronic-degenerative diseases, smoking status, and other medications was obtained from the medical records and from the interview for inclusion in the study.

### Definition of Response

A fasting blood sample was drawn for the determination of HbA1c. HbA1c was measured at least 3 months after drug prescription and determined using Tina-quant^®^ HbA1C Gen. 3 (Cobas-Mira Roche). The approach taken for the treatment of the patients was “treat to target,” defined as failure to reach levels of HbA1c ≤ 7%. The initial HbA1c of each patient was at least 7%.

### DNA Isolation

Peripheral blood from patients was extracted in a tube with EDTA and genomic DNA was isolated with Wizard Genomic DNA Purification Kit (Promega, Madison, WI, United States). Protocol was followed according to manufacturer’s instructions. Genomic DNA was quantified by UV absorbance using Nanodrop (Thermo Scientific, Wilmington, DE, United States). The quality of DNA was measured with the A260/280 ratio, a value of 1.8–2 was considered of good quality. Samples were kept at -20°C in small working aliquots until analysis to avoid recurrent cycles of freezing and thawing to minimize degradation.

### Pharmacogenetic Tests (Genotyping)

A total of 14 single nucleotide polymorphisms distributed in 5 different genes associated with response to anti-diabetic treatments were genotyped by Real-Time PCR system using validated Genotyping Assays (Applied Biosystems, Foster City, CA, United States) according to the manufacturer’s instructions. Two additional polymorphisms in *SLC22A1* gene (Met61Val and Met420Del) were included in the study and analyzed in 50 responders and 50 non-responder patients. These additional polymorphisms were determined by nucleotide sequencing method in a Genetic Analyzer 3100 (Applied Biosystems). As a quality control measure, genotyping for the polymorphisms were required to pass three tests for inclusion in subsequent association studies: the genotype call rate (> 0.90 completeness to obtain 99.8% accuracy), the Hardy-Weinberg equilibrium (HWE) test (*p*-value > 0.05), and the minor allele frequency (MAF) criterion (> 0.01).

### Analysis of Statistical Significance

Standard descriptive and comparative analyses were performed. The responder’s phenotypes classification was made using Hb1Ac parameter applied a cut-off ≤ 7 for responder’s and > 7 for non-responder’s [including first-line therapy (FLT), second-line therapy (SLT), third-line therapy (TLT), monotherapy, and combination therapy]. The HWE was determined by comparing the genotype frequencies with the expected values using the maximum likelihood method. To detect significant differences between two groups, Student’s *t*-test or the Mann–Whitney *U*-test were used for parametric or non-parametric distributions, respectively. Differences between more than two groups were assessed by one-way ANOVA and the Kruskal–Wallis *H*-test for parametric or non-parametric distributions, respectively. *Post hoc* tests (LSD and Tamhane’s T2) were used for pairwise comparisons. Possible associations between genotypes and phenotypes were assessed using contingency tables *X*^2^ statistics and Fisher’s exact tests. The association was evaluated under four different models (dominant, over dominant, recessive, and additive). Odds ratios were estimated with 95% confidence intervals. Aforementioned analyses were performed with SPSS for Windows, V.20 (IBM Corp., Armonk, NY, United States). All *p*-values were two-tailed. The corrected *P* (*P*c)-values were adjusted by using Bonferroni’s correction. A *p*-value ≤ 0.05 was considered statistically significant.

### Computational Modeling: Direct Coupling Analysis

To study the association between diabetes-related SNPs, patient data and antidiabetic drug response, we have developed a metric called DI, which is derived from the inference framework DCA ([Morcos et al., 2011](#B28)). DCA is a statistical method that infers efficiently the parameters of probability distributions with a large set of variables. DCA can be computed efficiently and is able to capture and evaluate direct pairwise correlations among potentially thousands of variable connections. The probability distribution of large sets of data is modeled with the following Boltzmann-like distribution:

| P(dat)=1Zexp{∑eij+∑hi} |
| --- |
where *dat* represents a profile with *L* variables that are indexed by *i* and *j* and *Z* is a normalization constant. The parameters of this distribution are all possible *eij* and *hi* for *i, j* ≤*L* and contain information about pairwise direct connectivity (*eij*) of the variables in the dataset. They are typically hard to be calculated exactly, but can be estimated using DCA. Once the parameters have been estimated, we can use them to compute pairwise probabilities. The following expression shows the form of DI based on the probabilities computed using the parameters, *eij* and *hi.*

| DIij=∑xi,xjPij(xi,xj)logPij(xi,xj)fi(xi)fj(xj) |
| --- |
Here *xi* is the quantized value of the clinical variable in the profile. The values of the *DIij* pairs tell us how connected are two variables in the distribution.

### Analysis on T2DM Patient Data

The DCA was applied to the complete cohort of data as described in **Figure 1**. The responder’s phenotypes classification was made at a cut-off 7 as defined before. Patient’s body indexes, such as weight, height, BMI, age, duration of diabetes, systolic pressure, diastolic pressure, are classified based on decade spans. To find the influential factors for response to OADs, a matrix containing all patient phenotypic informatics, 14 polymorphisms, HbA1c test result is generated as the input for DCA algorithm ([Morcos et al., 2011](#B28)). The T2DM database consists of patient profiles from 495 patients, including basic information, first, second, third line therapy information, 14 polymorphisms, health conditions, and the HbA1c test result estimating the glucose-lowering effect of OADs. The patient profile columns also include the 21 OADs separately, representing the usage and doses of a specific OAD for certain patients. All of those profiles data are classified and organized in an input matrix for DCA. DI is computed from DCA as a metric of connectivity strength for pairwise variables. The higher DI values, the stronger the correlation between these two variables. DI has been successfully applied to model molecular interactions in protein folds ([Morcos et al., 2011](#B28),[Morcos et al., 2014](#B29); [dos Santos et al., 2015](#B13); [Boyd et al., 2016](#B9)) as well as to identify drug-gene connections in cancer datasets ([Jiang et al., 2017](#B22)). Then, DI values for each variable pair is computed by DCA algorithm and then is used to find a complete network by using a minimum spanning tree approach and then a Bayesian network is built with undirected edges.

### FIGURE 1.

![FIGURE 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a613/5898372/674347dfc68f/fphar-09-00320-g001.jpg)

Workflow of global probabilistic modeling on T2DM patient data. Strategy of using T2DM patient datasets to compute the direct information metric between patient genetic or clinical factors and the drug response of OAD treatments. After DI values were calculated, they were used in the Chow-Liu tree for a structural learning for Bayesian network.

### Predictive Model for OAD Treatment Response

The direct connectivity (*eij*) estimates the strength of couplings between two variables at certain states. The summation of *eij* over all of patient profile factors with drug response provides a score to evaluate each patient’s glucose lowering response after taking OADs under his specific genetic and clinical profiles.

When summing all the *eij* with the *j* defined as the HbA1c level ≤ 7%, the Score represents how likely the patient is responding to the current OAD treatment based on his/her body indexes, treatment strategy, polymorphisms, health condition.

| ScoreRes=∑ieij(xi,Res) |
| --- |
where *i* denotes a genetic or clinical factor of patient, and *xi* represents the class of the factor belongs to. Additionally, the score for a patient’s inert responses to the OAD is calculated based on the *eij* with *j* representing HbA1c level > 7%.

| ScoreNonRes=∑ieij(xi,NonRes) |
| --- |
The two scores for each patient are compared and the treatment response is predicted based on which score is larger. The leave one out cross-validation is conducted to evaluate the performance of this predictive model.

## Results

### Descriptive Statistics and Phenotype Classification

A total of 495 patients treated with hypoglycemic drugs were included in this study. The subjects were Mexican, mainly from northeastern of Mexico. The average age of patients was 56.30 ± 12.16 for males and 56.41 ± 11.45 for females. No significant differences were found for the age of diagnosis, diabetes duration, and HbA1c values between males and females. However, the BMI was statistically higher in females (**Table 1**). Regarding to co-morbidities, the most frequent co-morbidity was hypertension with 24.4%, followed by hypertension-dyslipidemia with 13.1%, only dyslipidemia (7.5%), hypothyroidism (6.3%), and hypertension-hypothyroidism (4.6%).

### Table 1.

| Patients | N | Age | Diagnosis age | Diabetes duration | BMI | HbA1c |
| --- | --- | --- | --- | --- | --- | --- |
| Males | 156 (31.5%) | 56.30 ± 12.16 | 45.12 ± 12.013 | 11.45 ± 8.03 | 28.90 ± 4.46£ | 8.69 ± 2.24 |
| Females | 339 (68.5%) | 56.41 ± 11.45 | 45.32 ± 10.705 | 10.95 ± 8.63 | 30.66 ± 6.78 | 8.45 ± 2.10 |
| Non-responders | 353 (71.3%) | 56.30 ± 11.51 | 44.14 ± 11.00∆, ¥ | 12.14 ± 8.25 | 29.83 ± 5.95 | 9.40 ± 1.92£,§ |
| MT non-responders | 332 (67.1%) | 56.47 ± 11.48 | 44.39 ± 10.95 | 11.00 ± 8.26 | 30.00 ± 6.07 | 9.33 ± 1.94 |
| CT non-responders | 30 (6.1%) | 55.27 ± 11.51 | 42.53 ± 11.23 | 13.07 ± 8.28 | 28.50 ± 4.80 | 9.6 ± 1.81 |
| Responders (any type) | 142 (28.7%) | 56.56 ± 12.09 | 48.02 ± 10.98 | 8.54 ± 8.39 | 30.79 ± 6.72 | 6.34 ± 0.47 |
| MT responders | 127 (25.7%) | 55.88 ± 12.08 | 47.92 ± 11.36 | 8.06 ± 7.88 | 30.91 ± 6.72 | 6.32 ± 0.43 |
| CT responders | 7 (1.4%) | 65.29 ± 13.16 | 49.29 ± 6.90 | 15.43 ± 14.26 | 25.83 ± 3.78 | 5.97 ± 0.79 |
| FLT responders | 93 (18.8%) | 56.61 ± 11.11 | 49.68 ± 10.83i | 6.92 ± 6.89Ȼ | 31.46 ± 6.85 | 6.30 ± 0.50 |
| SLT responders | 39 (7.9%) | 56.77 ± 12.98 | 46.62 ± 10.33 | 10.19 ± 8.79 | 30.13 ± 6.15 | 6.43 ± 0.41 |
| TLT Responders | 10 (2.0%) | 55.20 ± 17.69 | 38.10 ± 9.67 | 17.10 ± 13.07 | 27.13 ± 6.88 | 6.40 ± 0.44 |

Table 1 Caption: Demographic and clinical data of patients.

The phenotype classification based on HbA1c values (**Table 1**) was significantly different between the responder’s and non-responder’s (*p* = 6.29 × 10^-68^). More than half of the patients (353) did not respond to any type of therapy (HbA1c > 7%), failing in 71.3% of the cases, and the treatment was effective (HbA1c ≤ 7%) in 142 individuals. The average diagnosis age of non-responders showed significant lower values (*p* = 4.25 × 10^-4^) compared to responder’s, but showed statistically significant higher values of diabetes duration (*p* = 2.5 × 10^-7^). A total of 93 patients (18.8%) responded to FLT, and they showed higher values of diagnosis age (*p* = 0.025), although for lower values of diabetes duration (*p* ≤ 0.049), compared to responder’s to TLT. None other therapies had a significant difference.

The drug most commonly used for the FLT was metformin in monotherapy (46.7%). The second most used drug in FLT was a SU in combination with metformin (34.6%). For SLT and TLT, metformin was also very commonly used (16.7 and 8.0%, respectively). For FLT, SLT, and TLT, the third most common option was SU in monotherapy (9.3, 13.3, and 5.3%, respectively). Insulin was the most common treatment choice in SLT and TLT (55.2 and 69.3%, respectively), although it was the fifth option in FLT (2.2%) (**Table 2**).

### Table 2.

|   | First-line therapy |  | Second-line therapy |  | Third-line therapy |  |
| --- | --- | --- | --- | --- | --- | --- |
| Drug | N | Percent | N | Percent | N | Percent |
| Metformin | 231 | 46.7 | 45 | 16.7 | 6 | 8 |
| Metformin/Sulfonylurea | 171 | 34.6 | 3 | 1.1 | 1 | 1.3 |
| Sulfonylurea | 46 | 9.3 | 40 | 14.8 | 5 | 6.8 |
| Other | 36 | 7.2 | 33 | 12.2 | 11 | 14.6 |
| Insulin | 11 | 2.2 | 149 | 55.2 | 52 | 69.3 |
| Total | 495 | 100 | 270 | 100 | 75 | 100 |

Table 2 Caption: Scheme for the treatment of T2DM.

### Pharmacogenetic Findings by Standard Statistical Methods

The polymorphisms M165I and R400C in *SLC22A2* gene were not in HWE equilibrium. The SNPs G401S and R465G in *SLC22A1* gene, and K432Q in *SLC22A2* gene, had a Minor Allele Frequency (MAF) < 0.01. The polymorphisms were excluded from subsequent analyses. As a result a total of 9 SNPs remained for statistical analysis. Two polymorphisms, Ala1369Ser in gene *ABCC8* and Glu23Lys in gene *KCNJ11*, showed a significant impact on response to SUs.

The effect of *ABCC8*-Ala1369Ser polymorphism on Hb1Ac under SU treatment was statistically significant. Heterozygous variant (C/T) carriers had lower HbA1c values compared to homozygous wild-type (A/A) carriers (*p* = 0.029) and compared to homozygous wild-type and variant (A/A+C/C) carriers (*p* = 0.012). The genotypes resulting from the *KCNJ11*-Glu23Lys polymorphism also had a significant impact on HbA1c under SU treatment. First, the homozygous wild-type and variant (C/C+T/T) carriers had higher HbA1c values (*p* = 0.039) as compared to heterozygous carrier (C/T). None of the other 7 polymorphisms tested had a significant impact on clinical parameters (**Table 3**).

### Table 3.

| Polymorphism | N | BMI | Diagnosis age | HbA1c |
| --- | --- | --- | --- | --- |
| ABCC8-Ala1369Ser |  |  |  |  |
| A/A | 180 | 30.74 ± 6.84 | 46.23 ± 11.37 | 8.69 ± 2.07Δ |
| A/C | 241 | 29.84 ± 5.93 | 45.16 ± 11.34 | 8.34 ± 2.21 |
| C/C | 74 | 29.44 ± 5.26 | 43.20 ± 9.52 | 8.74 ± 2.09 |
| A/A+C/C | 254 | 30.36 ± 6.44 | 45.35 ± 10.93 | 8.70 ± 2.07 |
| CYP2C9-Arg144Cys |  |  |  |  |
| C/C | 423 | 30.07 ± 6.26 | 45.41 ± 11.03 | 8.49 ± 2.17 |
| C/T | 67 | 30.30 ± 6.04 | 44.69 ± 11.40 | 8.67 ± 2.00 |
| T/T | 5 | 30.17 ± 2.09 | 40.00 ± 16.33 | 9.60 ± 2.13 |
| CYP2C9-Ile359Leu |  |  |  |  |
| A/A | 460 | 30.06 ± 6.01 | 45.35 ± 10.96 | 8.52 ± 2.13 |
| C/A | 35 | 30.67 ± 8.28 | 44.06 ± 13.24 | 8.55 ± 2.34 |
| KCNJ11-Glu23Lys |  |  |  |  |
| C/C | 179 | 30.70 ± 6.87 | 46.26 ± 11.56 | 8.64 ± 2.08 |
| C/T | 246 | 29.91 ± 5.96 | 44.95 ± 11.25 | 8.37 ± 2.19 |
| T/T | 70 | 29.26 ± 5.01 | 43.79 ± 9.28 | 8.75 ± 2.13 |
| C/C+T/T | 249 | 30.29 ± 6.42 | 45.56 ± 11.00 | 8.67 ± 2.09£ |
| KCNJ11-Ile1337Val |  |  |  |  |
| C/C | 71 | 29.36 ± 5.05 | 43.75 ± 9.22 | 8.74 ± 2.12 |
| C/T | 247 | 29.97 ± 6.17 | 44.91 ± 11.34 | 8.38 ± 2.20 |
| T/T | 177 | 30.59 ± 6.62 | 46.34 ± 11.46 | 8.64 ± 2.07 |
| SLC22A1-Arg61Cys |  |  |  |  |
| C/C | 475 | 30.05 ± 6.21 | 45.40 ± 11.03 | 8.53 ± 2.16 |
| C/T | 20 | 31.43 ± 5.83 | 41.90 ± 13.02 | 8.51 ± 1.77 |
| SLC22A1-Met61Val |  |  |  |  |
| G/G | 92 | 30.77 ± 6.22 | 46.41 ± 9.96 | 8.18 ± 2.04 |
| A/G | 26 | 29.74 ± 4.45 | 45.58 ± 14.77 | 8.14 ± 1.74 |
| A/A | 6 | 28.66 ± 4.03 | 43.67 ± 10.65 | 8.15 ± 1.83 |
| SLC22A2-Ala270Ser |  |  |  |  |
| A/C | 58 | 30.04 ± 6.35 | 46.03 ± 11.17 | 8.04 ± 1.56 |
| C/C | 437 | 30.11 ± 6.18 | 45.15 ± 11.12 | 8.59 ± 2.20 |
| SLC22A2-Met420Del |  |  |  |  |
| ATG/ATG | 52 | 30.56 ± 6.30 | 44.65 ± 9.87 | 8.35 ± 2.09 |
| ATG/delTGA | 49 | 30.72 ± 5.77 | 48.10 ± 12.73 | 8.05 ± 2.00 |
| delGAT/delGAT | 23 | 29.61 ± 4.78 | 45.13 ± 9.59 | 8.02 ± 1.56 |

Table 3 Caption: Association values between gene polymorphisms and clinical parameters.

The association was evaluated under genetic models for only nine polymorphisms that had passed a quality control. We found that two of the nine polymorphisms were associated with the responder phenotype. The A/C genotype of *ABCC8*-Ala1369Ser and the C/T genotype of *KCNJ11*-Glu23Lys were significantly associated with responder phenotype using over dominant model. This association remained statistically significant after adjusting using Bonferroni’s correction (*p* < 0.05) (**Table 4**).

### Table 4.

| Gene | Polymorphism | Model | OR (95% CI) | p-value | Pc-value |
| --- | --- | --- | --- | --- | --- |
| ABCC8 | Ala1369Ser | Over-dominant (A/A+C/C vs. A/C) | A/A+C/C = 1.33 (1.11–1.59) | 0.03 | 0.04∗ |
|   |   |   | A/C = 0.736 (0.59–0.92) |   |   |
| KCNJ11 | Glu23Lys | Over-dominant (C/C+T/T vs. C/T) | C/C+T/T = 1.27 (1.06–1.51) | 0.013 | 0.018∗ |
|   |   |   | C/T = 0.77 (0.62–0.96) |   |   |

Table 4 Caption: Association values between genotypes and response using dominant, over-dominant, and additive models.

### Pharmacogenetic and Clinical Parametric Findings From T2DM Patient Profiles by Direct Coupling Analysis

The DCA finds factor-drug response connections from a global statistical model computed from an estimate of the joint probability distribution of all clinical variables in the study. **Figure 1** shows the classification process that the patient clinical and genetic data undergoes to form the input discrete matrix for DCA algorithm. The outcome is a set of pairs with DI values. To uncover the minimal set of relevant connections between those factors, a Bayesian network is constructed by using the Chow-Liu algorithm as shown in **Figure 1**. However, this study refines Chow-Liu algorithm by replacing the typical use of mutual information with DI from DCA to calculate the Kullback–Leibler distance. This is a novel approach to generate the Bayesian network. Some factors cluster together and are connected showing previously known relationships, such as the connections between weight, height, BMI, and gender. These known associations of factors can be seen as validation of the links found by the algorithm. The time lengths of treatment (first line and second line), age, age of diagnosis, and diabetes diagnosis span are clustered; however, the treatment history for the third line therapy is more likely to be associated with weight.

In agreement with the pharmacogenomics finding that *KCNJ11* Glu23Lys affects the response to SUs, while *KCNJ11* Glu23Lys is generally connected to response to OADs. However, the *ABCC8* Ala1369Ser variant is not connected to any drug in this network and is linked to *KCNJ11* Ile1337Val variant. Polymorphisms in the *SLC22A2* gene have been identified and shown to cause inter-patient variability in the pharmacokinetic and pharmacodynamic profile of metformin. Three gene variants, M165I (rs8177507), Ala270Ser (rs316019), and R400C (rs8177516), of the *SLC22A2* gene were reported with reduced uptake of OCT2 substrate, whereas a fourth one, K432Q (rs8177517), showed an increased uptake activity compared to the wild-type allele. However, attempts to translate those findings into altered response to metformin of diabetic patients in several populations have not been successful ([Meyer zu Schwabedissen et al., 2010](#B26)). As shown in **Figure 2**, 3 out of 4 polymorphisms in *SLC22A2* have connections to metformin in combination with other drugs. The genetic variants of *SLC22A2* identified in a Korean population appear to have a significant impact on the disposition of metformin. As expected from the primary distribution of OCT2 in the kidney, the tubular excretion was influenced mainly by the M165I, Ala270Ser, and R400C variants of *SLC22A2*, leading to an increase in plasma metformin concentrations in subjects with these variants ([Song et al., 2008](#B38)). MET is connected to FLT cluster and SLT cluster, being consistent with the fact that MET is the most commonly used drug in FLT and the second common drug in SLT. Two SU drugs, GLIB and GLIM, are connected together.

### FIGURE 2.

![FIGURE 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a613/5898372/a2e385168513/fphar-09-00320-g002.jpg)

Bayesian network of OADs and factors built from direct information. Hexagonal shapes indicate OADs and ovals denote clinical parameters or polymorphisms.

To systematically investigate the connection between blood glucose lowering outcome and other factors, we studied the couplings between those factors and the drug response HbA1c test results. In the input matrix, the values in columns for each drug identify their presence or absence in the treatment. The overall ranking of each drug response connection is shown in the heatmap of **Figure 3A**. Treatment time and doses are highly associated with HbA1c results. Age and place of origin appear to be strongly influential. The administration of GLIB or MET in monotherapy is also highly connected to HbA1c results, partially corresponding with the fact that Metformin is the most commonly used treatment for T2DM. Among the body indexes parameter, weight and BMI still have high rankings, which suggests that in prediction of treatment outcome those two factors are worthy of consideration. The rankings of polymorphisms have the highest influence at *KCNJ11* Glu23Lys, which is observed to be correlated with drug response to SUs in both the statistical significance study and the DI-based Bayesian network.

### FIGURE 3.

![FIGURE 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a613/5898372/52b24241806d/fphar-09-00320-g003.jpg)

(A) Heat map for the ranking of each factors and HbA1c test result among all the pairwise pool. The lower value of the ranking indicates stronger connectivity. (B) A predictive model for patient’s drug response to OAD treatments.

In order to predict the glucose-lowering efficacy of each OADs and determine a better therapy strategy based on a given profile of patient, we develop a predictive model on DI (**Figure 3B**). DI is a metric of direct coupling among variables but it does not reveal the directionality of this connection. It is possible to use the parameters of the global joint distribution, to quantify how a large number of factors account for a possible outcome, i.e., responsive or non-responsive treatment. This additive model uses the *eij(xi,xj)* estimates connecting factors to response with the aims to distinguish between the responder and non-responder group. We conducted a leave one out cross-validation on the 495 T2DM patients dataset, and reached an average of prediction rate at 0.70, with the maximum response vs. non-response prediction rate at 0.76.

## Discussion

### Association Between Gene Polymorphisms and Clinical Parameters

From the nine analyzed pharamcogentic polymorphisms seeking to explain the relationship between diverse genotypes of diabetic patients and their response to different OADs, only two polymorphisms, *ABCC8*-Ala1369Ser and *KCNJ11*-Glu23Lys, showed a significant impact on response on the reduction of Hb1Ac with SU treatment. None of the other seven polymorphisms tested had a significant impact on clinical parameters. These results confirm the association of *ABCC8*-Ala1369Ser polymorphism and reduction of HbA1c level in the Chinese population with SU treatment ([Feng et al., 2008](#B15)). Nevertheless, studies in Caucasian populations showed no association of *KCNJ11*-Glu23Lys with Hb1Ac reduction in response to SUs ([Ragia et al., 2012](#B35)).

The *CYP2C9* polymorphisms included in this study, Arg144Cys and I1359L, showed no significant differences in response to SUs in comparison with studies carried on Caucasian population in which they described a higher sensitivity to SUs for Ile359Leu and Arg144Cys variant carriers ([Becker et al., 2008](#B8); [Ragia et al., 2014](#B34)). The *KCNJ11*-I337 polymorphism showed no evidence of being related in the response to SUs as a study carried on Chinese population suggests ([Cheung et al., 2011](#B11)). The *SLC22A1* polymorphisms, Arg61Cys and Met61Val, showed no significant evidence of being related in the response to metformin in comparison with a study carried in Caucasian population in which they found a significant reduction of Hb1Ac after 6 months of metformin treatment ([Tkac et al., 2013](#B42)). The *SLC22A2* polymorphisms showed no evidence of being related in the response to metformin, contrary of what has been suggested ([Avery et al., 2009](#B4)).

### Association Between Genotypes and Phenotypes

Only the A/C nucleotide change from polymorphism Ala1369Ser (gene *ABCC8*) and the C/T nucleotide change from polymorphism Glu23Lys (gene *KCNJ1*) were significantly associated with responder phenotype using an over dominant model. *KCNJ11* and *ABCC8* encode for the subunits KIR6.2 and SUR1, respectively, of the heteroctomer K_ATP_ channel ([Emami-Riedmaier et al., 2015](#B14)). K_ATP_ channels regulate membrane K^+^ flux for various cell types including pancreatic β-cells, where increased glucose metabolism results in the closure of the K_ATP_ channels leading to calcium influx and subsequent insulin secretion ([Nathan et al., 2009](#B30)). Notably, *KCNJ11* and *ABCC8* genes lie close to each other on chromosome 11, with strong linkage disequilibrium. In a Caucasian population study, Ala1369Ser was correlated with Glu23Lys, where for every K allele of *KCNJ11* gene found there was A allele of *ABCC8*, thus constituting a possible haplotype ([Florez et al., 2004](#B16)), whereas several studies and meta-analyses showed the association of *KCNJ11*, but not of *ABCC8* polymorphisms, with susceptibility to type 2 diabetes ([van Dam et al., 2005](#B45); [Gong et al., 2012](#B18)).

We showed that it is possible to use patient data in this comprehensive study to generate a model of the global distribution of patient profiles. This model includes phenotypic factors, health conditions, treatment information, and polymorphisms with clinical treatment outcome variable. Although we found agreement between the standard statistical tests and the global pairwise DCA model about how *KCNJ11*-Glu23Lys affects the efficacy of SUs drug, we also found novel relationships when modeling the dataset with global techniques. We uncover a network connecting OADs, gene polymorphisms, and patient information. Connections with the HbA1c test and metrics for the association between each pairwise variables can inform better how a large set of factors interact during disease progression.

A predictive model for OAD drug response is proposed based on direct coupling parameters *eij* in this study and its predictive performance has been validated by cross validation. The overall prediction rate both for predicting as responding or non-responding can be as high as 0.76. This model has the potential to be used as a guide to modify factors to predict higher response scores. This is a topic of further research that can have applications in personalized therapies. With increasing well-phenotyped cohorts and new methods, such as Next Generation Sequencing and global statistical analyses, the next few years promise a renewed interest in the use of pharmacogenetics to unravel drug and disease mechanisms, as well as the possibility to individualize T2DM therapy by genotype.

## Author Contributions

HB-S and LR-C conceived of the idea of analyzing genetic factors altering the oral antidiabetic drugs response in Mexican type II diabetes patients. LR-C, CL-A, HS-I, and IM-A carried out the real-time PCR experiments and Sanger sequencing experiments. FL-G provided the biospecimens and their clinical data. FM, X-LJ, and RL-C analyzed the data and designed the models and the computational framework. X-LJ and FM developed the predictive model. HB-S was in charge of overall direction and planning. HS-I, FM, X-LJ, LR-C, and DA-T wrote the manuscript. HB-S and FM should be considered as co-corresponding authors.

## Conflict of Interest Statement

The authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.

## Acknowledgments

We thank Esteban Lopez for the comments that greatly improved the manuscript. We would also like to show our gratitude to Roxana Rivera, Ph.D., and Ricardo Cerda for their comments on earlier versions of the manuscript.

## Footnotes

## References

1. Agudelo-Botero M., Davila-Cervantes C. A. (2015). Burden of mortality due to diabetes mellitus in Latin America 2000-2011: the case of Argentina, Chile, Colombia, and Mexico. Gac. Sanit. 29 172–177. 10.1016/j.gaceta.2015.01.015  [DOI](https://doi.org/10.1016/j.gaceta.2015.01.015) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25746417/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gac.%20Sanit.&title=Burden%20of%20mortality%20due%20to%20diabetes%20mellitus%20in%20Latin%20America%202000-2011:%20the%20case%20of%20Argentina,%20Chile,%20Colombia,%20and%20Mexico.&author=M.%20Agudelo-Botero&author=C.%20A.%20Davila-Cervantes&volume=29&publication_year=2015&pages=172-177&pmid=25746417&doi=10.1016/j.gaceta.2015.01.015&)

2. Almudevar A. (2010). A hypothesis test for equality of bayesian network models. EURASIP J. Bioinform. Syst. Biol. 2010:947564. 10.1155/2010/947564  [DOI](https://doi.org/10.1155/2010/947564) | [PMC free article](/articles/PMC3171365/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20981254/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=EURASIP%20J.%20Bioinform.%20Syst.%20Biol.&title=A%20hypothesis%20test%20for%20equality%20of%20bayesian%20network%20models.&author=A.%20Almudevar&volume=2010&issue=947564&publication_year=2010&pmid=20981254&doi=10.1155/2010/947564&)

3. Arredondo A., De Icaza E. (2011). The cost of diabetes in Latin America: evidence from Mexico. Value Health 14(5 Suppl. 1) S85–S88. 10.1016/j.jval.2011.05.022  [DOI](https://doi.org/10.1016/j.jval.2011.05.022) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21839907/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Value%20Health&title=The%20cost%20of%20diabetes%20in%20Latin%20America:%20evidence%20from%20Mexico.&author=A.%20Arredondo&author=E.%20De%20Icaza&volume=14(5%20Suppl.%201)&publication_year=2011&pages=S85-S88&pmid=21839907&doi=10.1016/j.jval.2011.05.022&)

4. Avery P., Mousa S. S., Mousa S. A. (2009). Pharmacogenomics in type II diabetes mellitus management: steps toward personalized medicine. Pharmgenomics Pers. Med. 2 79–91.  [DOI](https://doi.org/10.2147/pgpm.s5806) | [PMC free article](/articles/PMC3513204/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23226037/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmgenomics%20Pers.%20Med.&title=Pharmacogenomics%20in%20type%20II%20diabetes%20mellitus%20management:%20steps%20toward%20personalized%20medicine.&author=P.%20Avery&author=S.%20S.%20Mousa&author=S.%20A.%20Mousa&volume=2&publication_year=2009&pages=79-91&pmid=23226037&doi=10.2147/pgpm.s5806&)

5. Barquera S., Campos-Nonato I., Aguilar-Salinas C., Lopez-Ridaura R., Arredondo A., Rivera-Dommarco J. (2013). Diabetes in Mexico: cost and management of diabetes and its complications and challenges for health policy. Glob. Health 9:3. 10.1186/1744-8603-9-3  [DOI](https://doi.org/10.1186/1744-8603-9-3) | [PMC free article](/articles/PMC3599194/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23374611/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Glob.%20Health&title=Diabetes%20in%20Mexico:%20cost%20and%20management%20of%20diabetes%20and%20its%20complications%20and%20challenges%20for%20health%20policy.&author=S.%20Barquera&author=I.%20Campos-Nonato&author=C.%20Aguilar-Salinas&author=R.%20Lopez-Ridaura&author=A.%20Arredondo&volume=9&issue=3&publication_year=2013&pmid=23374611&doi=10.1186/1744-8603-9-3&)

6. Barretina J., Caponigro G., Stransky N., Venkatesan K., Margolin A. A., Kim S., et al. (2012). The cancer cell line Encyclopedia enables predictive modelling of anticancer drug sensitivity. Nature 483 603–607. 10.1038/nature11003  [DOI](https://doi.org/10.1038/nature11003) | [PMC free article](/articles/PMC3320027/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22460905/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=The%20cancer%20cell%20line%20Encyclopedia%20enables%20predictive%20modelling%20of%20anticancer%20drug%20sensitivity.&author=J.%20Barretina&author=G.%20Caponigro&author=N.%20Stransky&author=K.%20Venkatesan&author=A.%20A.%20Margolin&volume=483&publication_year=2012&pages=603-607&pmid=22460905&doi=10.1038/nature11003&)

7. Becker M. L., Pearson E. R., Tkac I. (2013). Pharmacogenetics of oral antidiabetic drugs. Int. J. Endocrinol. 2013:686315. 10.1155/2013/686315  [DOI](https://doi.org/10.1155/2013/686315) | [PMC free article](/articles/PMC3845331/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24324494/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Endocrinol.&title=Pharmacogenetics%20of%20oral%20antidiabetic%20drugs.&author=M.%20L.%20Becker&author=E.%20R.%20Pearson&author=I.%20Tkac&volume=2013&issue=686315&publication_year=2013&pmid=24324494&doi=10.1155/2013/686315&)

8. Becker M. L., Visser L. E., Trienekens P. H., Hofman A., van Schaik R., Stricker B. (2008). Cytochrome P450 2C9∗2 and ∗3 polymorphisms and the dose and effect of sulfonylurea in type II diabetes mellitus. Clin. Pharmacol. Ther. 83 288–292. 10.1038/sj.clpt.6100273  [DOI](https://doi.org/10.1038/sj.clpt.6100273) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17597710/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Cytochrome%20P450%202C9%E2%88%972%20and%20%E2%88%973%20polymorphisms%20and%20the%20dose%20and%20effect%20of%20sulfonylurea%20in%20type%20II%20diabetes%20mellitus.&author=M.%20L.%20Becker&author=L.%20E.%20Visser&author=P.%20H.%20Trienekens&author=A.%20Hofman&author=R.%20van%20Schaik&volume=83&publication_year=2008&pages=288-292&pmid=17597710&doi=10.1038/sj.clpt.6100273&)

9. Boyd J. S., Cheng R. R., Paddock M. L., Sancar C., Morcos F., Golden S. S. (2016). A combined computational and genetic approach uncovers network interactions of the cyanobacterial circadian clock. J. Bacteriol. 198 2439–2447. 10.1128/JB.00235-16  [DOI](https://doi.org/10.1128/JB.00235-16) | [PMC free article](/articles/PMC4999937/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27381914/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Bacteriol.&title=A%20combined%20computational%20and%20genetic%20approach%20uncovers%20network%20interactions%20of%20the%20cyanobacterial%20circadian%20clock.&author=J.%20S.%20Boyd&author=R.%20R.%20Cheng&author=M.%20L.%20Paddock&author=C.%20Sancar&author=F.%20Morcos&volume=198&publication_year=2016&pages=2439-2447&pmid=27381914&doi=10.1128/JB.00235-16&)

10. Chen X. W., Anantha G., Lin X. T. (2008). Improving Bayesian network structure learning with mutual information-based node ordering in the K2 algorithm. IEEE Transac. Knowl. Data Eng. 20 628–640. 10.1109/Tkde.2007.190732  [DOI](https://doi.org/10.1109/Tkde.2007.190732) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=IEEE%20Transac.%20Knowl.%20Data%20Eng.&title=Improving%20Bayesian%20network%20structure%20learning%20with%20mutual%20information-based%20node%20ordering%20in%20the%20K2%20algorithm.&author=X.%20W.%20Chen&author=G.%20Anantha&author=X.%20T.%20Lin&volume=20&publication_year=2008&pages=628-640&doi=10.1109/Tkde.2007.190732&)

11. Cheung C. Y., Tso A. W., Cheung B. M., Xu A., Fong C. H., Ong K. L., et al. (2011). The KCNJ11 E23K polymorphism and progression of glycaemia in Southern Chinese: a long-term prospective study. PLoS One 6:e28598. 10.1371/journal.pone.0028598  [DOI](https://doi.org/10.1371/journal.pone.0028598) | [PMC free article](/articles/PMC3230634/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22163043/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=The%20KCNJ11%20E23K%20polymorphism%20and%20progression%20of%20glycaemia%20in%20Southern%20Chinese:%20a%20long-term%20prospective%20study.&author=C.%20Y.%20Cheung&author=A.%20W.%20Tso&author=B.%20M.%20Cheung&author=A.%20Xu&author=C.%20H.%20Fong&volume=6&issue=e28598&publication_year=2011&pmid=22163043&doi=10.1371/journal.pone.0028598&)

12. Distefano J. K., Watanabe R. M. (2010). Pharmacogenetics of anti-diabetes drugs. Pharmaceuticals 3 2610–2646. 10.3390/ph3082610  [DOI](https://doi.org/10.3390/ph3082610) | [PMC free article](/articles/PMC2951319/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20936101/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmaceuticals&title=Pharmacogenetics%20of%20anti-diabetes%20drugs.&author=J.%20K.%20Distefano&author=R.%20M.%20Watanabe&volume=3&publication_year=2010&pages=2610-2646&pmid=20936101&doi=10.3390/ph3082610&)

13. dos Santos R. N., Morcos F., Jana B., Andricopulo A. D., Onuchic J. N. (2015). Dimeric interactions and complex formation using direct coevolutionary couplings. Sci. Rep. 5:13652. 10.1038/srep13652  [DOI](https://doi.org/10.1038/srep13652) | [PMC free article](/articles/PMC4559900/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26338201/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep.&title=Dimeric%20interactions%20and%20complex%20formation%20using%20direct%20coevolutionary%20couplings.&author=R.%20N.%20dos%20Santos&author=F.%20Morcos&author=B.%20Jana&author=A.%20D.%20Andricopulo&author=J.%20N.%20Onuchic&volume=5&issue=13652&publication_year=2015&pmid=26338201&doi=10.1038/srep13652&)

14. Emami-Riedmaier A., Schaeffeler E., Nies A. T., Morike K., Schwab M. (2015). Stratified medicine for the use of antidiabetic medication in treatment of type II diabetes and cancer: where do we go from here? J. Intern. Med. 277 235–247. 10.1111/joim.12330  [DOI](https://doi.org/10.1111/joim.12330) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25418285/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Intern.%20Med.&title=Stratified%20medicine%20for%20the%20use%20of%20antidiabetic%20medication%20in%20treatment%20of%20type%20II%20diabetes%20and%20cancer:%20where%20do%20we%20go%20from%20here?&author=A.%20Emami-Riedmaier&author=E.%20Schaeffeler&author=A.%20T.%20Nies&author=K.%20Morike&author=M.%20Schwab&volume=277&publication_year=2015&pages=235-247&pmid=25418285&doi=10.1111/joim.12330&)

15. Feng Y., Mao G. Y., Ren X. W., Xing H. X., Tang G. F., Li Q., et al. (2008). Ser1369Ala variant in sulfonylurea receptor gene ABCC8 Is associated with antidiabetic efficacy of gliclazide in Chinese type 2 diabetic patients. Diabetes Care 31 1939–1944. 10.2337/dc07-2248  [DOI](https://doi.org/10.2337/dc07-2248) | [PMC free article](/articles/PMC2551631/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18599530/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Diabetes%20Care&title=Ser1369Ala%20variant%20in%20sulfonylurea%20receptor%20gene%20ABCC8%20Is%20associated%20with%20antidiabetic%20efficacy%20of%20gliclazide%20in%20Chinese%20type%202%20diabetic%20patients.&author=Y.%20Feng&author=G.%20Y.%20Mao&author=X.%20W.%20Ren&author=H.%20X.%20Xing&author=G.%20F.%20Tang&volume=31&publication_year=2008&pages=1939-1944&pmid=18599530&doi=10.2337/dc07-2248&)

16. Florez J. C., Burtt N., de Bakker P. I., Almgren P., Tuomi T., Holmkvist J., et al. (2004). Haplotype structure and genotype-phenotype correlations of the sulfonylurea receptor and the islet ATP-sensitive potassium channel gene region. Diabetes Metab. Res. Rev. 53 1360–1368. 10.2337/diabetes.53.5.1360  [DOI](https://doi.org/10.2337/diabetes.53.5.1360) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15111507/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Diabetes%20Metab.%20Res.%20Rev.&title=Haplotype%20structure%20and%20genotype-phenotype%20correlations%20of%20the%20sulfonylurea%20receptor%20and%20the%20islet%20ATP-sensitive%20potassium%20channel%20gene%20region.&author=J.%20C.%20Florez&author=N.%20Burtt&author=P.%20I.%20de%20Bakker&author=P.%20Almgren&author=T.%20Tuomi&volume=53&publication_year=2004&pages=1360-1368&pmid=15111507&doi=10.2337/diabetes.53.5.1360&)

17. Foretz M., Hébrard S., Leclerc J., Zarrinpashneh E., Soty M., Mithieux G., et al. (2010). Metformin inhibits hepatic gluconeogenesis in mice independently of the LKB1/AMPK pathway via a decrease in hepatic energy state. J. Clin. Invest. 120 2355–2369. 10.1172/JCI40671  [DOI](https://doi.org/10.1172/JCI40671) | [PMC free article](/articles/PMC2898585/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20577053/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Invest.&title=Metformin%20inhibits%20hepatic%20gluconeogenesis%20in%20mice%20independently%20of%20the%20LKB1/AMPK%20pathway%20via%20a%20decrease%20in%20hepatic%20energy%20state.&author=M.%20Foretz&author=S.%20H%C3%A9brard&author=J.%20Leclerc&author=E.%20Zarrinpashneh&author=M.%20Soty&volume=120&publication_year=2010&pages=2355-2369&pmid=20577053&doi=10.1172/JCI40671&)

18. Gong B., Yu J., Li H., Li W., Tong X. (2012). The effect of KCNJ11 polymorphism on the risk of type 2 diabetes: a global meta-analysis based on 49 case-control studies. DNA Cell Biol. 31 801–810. 10.1089/dna.2011.1445  [DOI](https://doi.org/10.1089/dna.2011.1445) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22082043/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=DNA%20Cell%20Biol.&title=The%20effect%20of%20KCNJ11%20polymorphism%20on%20the%20risk%20of%20type%202%20diabetes:%20a%20global%20meta-analysis%20based%20on%2049%20case-control%20studies.&author=B.%20Gong&author=J.%20Yu&author=H.%20Li&author=W.%20Li&author=X.%20Tong&volume=31&publication_year=2012&pages=801-810&pmid=22082043&doi=10.1089/dna.2011.1445&)

19. Holstein A., Plaschke A., Ptak M., Egberts E. H., El-Din J., Brockmoller J., et al. (2005). Association between CYP2C9 slow metabolizer genotypes and severe hypoglycaemia on medication with sulphonylurea hypoglycaemic agents. Br. J. Clin. Pharmacol. 60 103–106. 10.1111/j.1365-2125.2005.02379.x  [DOI](https://doi.org/10.1111/j.1365-2125.2005.02379.x) | [PMC free article](/articles/PMC1884896/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15963101/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br.%20J.%20Clin.%20Pharmacol.&title=Association%20between%20CYP2C9%20slow%20metabolizer%20genotypes%20and%20severe%20hypoglycaemia%20on%20medication%20with%20sulphonylurea%20hypoglycaemic%20agents.&author=A.%20Holstein&author=A.%20Plaschke&author=M.%20Ptak&author=E.%20H.%20Egberts&author=J.%20El-Din&volume=60&publication_year=2005&pages=103-106&pmid=15963101&doi=10.1111/j.1365-2125.2005.02379.x&)

20. Huang C. M., Florez J. C. (2011). Pharmacogenetics in type 2 diabetes: potential implications for clinical practice. Genome Med. 3:76. 10.1186/gm292  [DOI](https://doi.org/10.1186/gm292) | [PMC free article](/articles/PMC3308031/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22126607/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Med.&title=Pharmacogenetics%20in%20type%202%20diabetes:%20potential%20implications%20for%20clinical%20practice.&author=C.%20M.%20Huang&author=J.%20C.%20Florez&volume=3&issue=76&publication_year=2011&pmid=22126607&doi=10.1186/gm292&)

21. Inzucchi S. E., Bergenstal R. M., Buse J. B., Diamant M., Ferrannini E., Nauck M., et al. (2012). Management of hyperglycaemia in type 2 diabetes: a patient-centered approach. position statement of the American diabetes association (ADA) and the European Association for the Study of Diabetes (EASD). Diabetologia 55 1577–1596. 10.1007/s00125-012-2534-0  [DOI](https://doi.org/10.1007/s00125-012-2534-0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22526604/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Diabetologia&title=Management%20of%20hyperglycaemia%20in%20type%202%20diabetes:%20a%20patient-centered%20approach.%20position%20statement%20of%20the%20American%20diabetes%20association%20(ADA)%20and%20the%20European%20Association%20for%20the%20Study%20of%20Diabetes%20(EASD).&author=S.%20E.%20Inzucchi&author=R.%20M.%20Bergenstal&author=J.%20B.%20Buse&author=M.%20Diamant&author=E.%20Ferrannini&volume=55&publication_year=2012&pages=1577-1596&pmid=22526604&doi=10.1007/s00125-012-2534-0&)

22. Jiang X. L., Martinez-Ledesma E., Morcos F. (2017). Revealing protein networks and gene-drug connectivity in cancer from direct information. Sci. Rep. 7:3739. 10.1038/s41598-017-04001-3  [DOI](https://doi.org/10.1038/s41598-017-04001-3) | [PMC free article](/articles/PMC5473890/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28623316/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep.&title=Revealing%20protein%20networks%20and%20gene-drug%20connectivity%20in%20cancer%20from%20direct%20information.&author=X.%20L.%20Jiang&author=E.%20Martinez-Ledesma&author=F.%20Morcos&volume=7&issue=3739&publication_year=2017&pmid=28623316&doi=10.1038/s41598-017-04001-3&)

23. Krentz A. J., Bailey C. J. (2005). Oral antidiabetic agents - Current role in type 2 diabetes mellitus. Drugs 65 385–411. 10.2165/00003495-200565030-00005  [DOI](https://doi.org/10.2165/00003495-200565030-00005) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15669880/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drugs&title=Oral%20antidiabetic%20agents%20-%20Current%20role%20in%20type%202%20diabetes%20mellitus.&author=A.%20J.%20Krentz&author=C.%20J.%20Bailey&volume=65&publication_year=2005&pages=385-411&pmid=15669880&doi=10.2165/00003495-200565030-00005&)

24. Lo C., Lui M., Ranasinha S., Teede H. J., Kerr P. G., Polkinghorne K., et al. (2012). Defining the relationship between HbA(1c) and average plasma glucose in type 2 diabetes and chronic kidney disease. Diabetologia 55 S453–S453. 10.1016/j.diabres.2014.01.020  [DOI](https://doi.org/10.1016/j.diabres.2014.01.020) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24573088/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Diabetologia&title=Defining%20the%20relationship%20between%20HbA(1c)%20and%20average%20plasma%20glucose%20in%20type%202%20diabetes%20and%20chronic%20kidney%20disease.&author=C.%20Lo&author=M.%20Lui&author=S.%20Ranasinha&author=H.%20J.%20Teede&author=P.%20G.%20Kerr&volume=55&publication_year=2012&pages=S453-S453&pmid=24573088&doi=10.1016/j.diabres.2014.01.020&)

25. Marchetti P., Lupi R., Del Guerra S., Bugliani M., D’Aleo V., Occhipinti M., et al. (2009). Goals of treatment for Type 2 diabetes beta-Cell preservation for glycemic control. Diabetes Care 32 S178–S183. 10.2337/dc09-S306  [DOI](https://doi.org/10.2337/dc09-S306) | [PMC free article](/articles/PMC2811441/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19875548/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Diabetes%20Care&title=Goals%20of%20treatment%20for%20Type%202%20diabetes%20beta-Cell%20preservation%20for%20glycemic%20control.&author=P.%20Marchetti&author=R.%20Lupi&author=S.%20Del%20Guerra&author=M.%20Bugliani&author=V.%20D%E2%80%99Aleo&volume=32&publication_year=2009&pages=S178-S183&pmid=19875548&doi=10.2337/dc09-S306&)

26. Meyer zu Schwabedissen H. E., Verstuyft C., Kroemer H. K., Becquemont L., Kim R. B. (2010). Human multidrug and toxin extrusion 1 (MATE1/SLC47A1) transporter: functional characterization, interaction with OCT2 (SLC22A2), and single nucleotide polymorphisms. Am. J. Physiol. Renal Physiol. 298 F997–F1005. 10.1152/ajprenal.00431.2009  [DOI](https://doi.org/10.1152/ajprenal.00431.2009) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20053795/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Physiol.%20Renal%20Physiol.&title=Human%20multidrug%20and%20toxin%20extrusion%201%20(MATE1/SLC47A1)%20transporter:%20functional%20characterization,%20interaction%20with%20OCT2%20(SLC22A2),%20and%20single%20nucleotide%20polymorphisms.&author=H.%20E.%20Meyer%20zu%20Schwabedissen&author=C.%20Verstuyft&author=H.%20K.%20Kroemer&author=L.%20Becquemont&author=R.%20B.%20Kim&volume=298&publication_year=2010&pages=F997-F1005&pmid=20053795&doi=10.1152/ajprenal.00431.2009&)

27. Morcos F., Jana B., Hwa T., Onuchic J. N. (2013). Coevolutionary signals across protein lineages help capture multiple protein conformations. Proc. Natl. Acad. Sci. U.S.A. 110 20533–20538. 10.1073/pnas.1315625110  [DOI](https://doi.org/10.1073/pnas.1315625110) | [PMC free article](/articles/PMC3870752/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24297889/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.S.A.&title=Coevolutionary%20signals%20across%20protein%20lineages%20help%20capture%20multiple%20protein%20conformations.&author=F.%20Morcos&author=B.%20Jana&author=T.%20Hwa&author=J.%20N.%20Onuchic&volume=110&publication_year=2013&pages=20533-20538&pmid=24297889&doi=10.1073/pnas.1315625110&)

28. Morcos F., Pagnani A., Lunt B., Bertolino A., Marks D. S., Sander C. (2011). Direct-coupling analysis of residue coevolution captures native contacts across many protein families. Proc. Natl. Acad. Sci. U.S.A. 108 E1293–E1301. 10.1073/pnas.1111471108  [DOI](https://doi.org/10.1073/pnas.1111471108) | [PMC free article](/articles/PMC3241805/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22106262/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.S.A.&title=Direct-coupling%20analysis%20of%20residue%20coevolution%20captures%20native%20contacts%20across%20many%20protein%20families.&author=F.%20Morcos&author=A.%20Pagnani&author=B.%20Lunt&author=A.%20Bertolino&author=D.%20S.%20Marks&volume=108&publication_year=2011&pages=E1293-E1301&pmid=22106262&doi=10.1073/pnas.1111471108&)

29. Morcos F., Schafer N. P., Cheng R. R., Onuchic J. N., Wolynes P. G. (2014). Coevolutionary information, protein folding landscapes, and the thermodynamics of natural selection. Proc. Natl. Acad. Sci. U.S.A. 111 12408–12413. 10.1073/pnas.1413575111  [DOI](https://doi.org/10.1073/pnas.1413575111) | [PMC free article](/articles/PMC4151759/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25114242/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.S.A.&title=Coevolutionary%20information,%20protein%20folding%20landscapes,%20and%20the%20thermodynamics%20of%20natural%20selection.&author=F.%20Morcos&author=N.%20P.%20Schafer&author=R.%20R.%20Cheng&author=J.%20N.%20Onuchic&author=P.%20G.%20Wolynes&volume=111&publication_year=2014&pages=12408-12413&pmid=25114242&doi=10.1073/pnas.1413575111&)

30. Nathan D. M., Buse J. B., Davidson M. B., Ferrannini E., Holman R. R., Sherwin R. (2009). Medical management of hyperglycemia in type 2 diabetes: a consensus algorithm for the initiation and adjustment of therapy A consensus statement of the American diabetes association and the European association for the study of diabetes. Diabetes Care 32 193–203. 10.2337/dc08-9025  [DOI](https://doi.org/10.2337/dc08-9025) | [PMC free article](/articles/PMC2606813/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18945920/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Diabetes%20Care&title=Medical%20management%20of%20hyperglycemia%20in%20type%202%20diabetes:%20a%20consensus%20algorithm%20for%20the%20initiation%20and%20adjustment%20of%20therapy%20A%20consensus%20statement%20of%20the%20American%20diabetes%20association%20and%20the%20European%20association%20for%20the%20study%20of%20diabetes.&author=D.%20M.%20Nathan&author=J.%20B.%20Buse&author=M.%20B.%20Davidson&author=E.%20Ferrannini&author=R.%20R.%20Holman&volume=32&publication_year=2009&pages=193-203&pmid=18945920&doi=10.2337/dc08-9025&)

31. Pearson E. (2009). Pharmacogenetics in diabetes. Curr. Diab. Rep. 9 172–181. 10.1007/s11892-009-0028-3  [DOI](https://doi.org/10.1007/s11892-009-0028-3) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19323963/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr.%20Diab.%20Rep.&title=Pharmacogenetics%20in%20diabetes.&author=E.%20Pearson&volume=9&publication_year=2009&pages=172-181&pmid=19323963&doi=10.1007/s11892-009-0028-3&)

32. Pernicova I., Korbonits M. (2014). Metformin-mode of action and clinical implications for diabetes and cancer. Nat. Rev. Endocrinol. 10 143–156. 10.1038/nrendo.2013.256  [DOI](https://doi.org/10.1038/nrendo.2013.256) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24393785/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Endocrinol.&title=Metformin-mode%20of%20action%20and%20clinical%20implications%20for%20diabetes%20and%20cancer.&author=I.%20Pernicova&author=M.%20Korbonits&volume=10&publication_year=2014&pages=143-156&pmid=24393785&doi=10.1038/nrendo.2013.256&)

33. Ragia G., Petridis I., Tavridou A., Christakidis D., Manolopoulos V. G. (2009). Presence of CYP2C9∗3 allele increases risk for hypoglycemia in Type 2 diabetic patients treated with sulfonylureas. Pharmacogenomics 10 1781–1787. 10.2217/Pgs.09.96  [DOI](https://doi.org/10.2217/Pgs.09.96) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19891554/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Presence%20of%20CYP2C9%E2%88%973%20allele%20increases%20risk%20for%20hypoglycemia%20in%20Type%202%20diabetic%20patients%20treated%20with%20sulfonylureas.&author=G.%20Ragia&author=I.%20Petridis&author=A.%20Tavridou&author=D.%20Christakidis&author=V.%20G.%20Manolopoulos&volume=10&publication_year=2009&pages=1781-1787&pmid=19891554&doi=10.2217/Pgs.09.96&)

34. Ragia G., Tavridou A., Elens L., Van Schaik R. H. N., Manolopoulos V. G. (2014). CYP2C9∗2 Allele Increases Risk for Hypoglycemia in POR∗1/∗1 Type 2 Diabetic Patients Treated with Sulfonylureas. Exp. Clin. Endocrinol. Diabetes 122 60–63. 10.1055/s-0033-1361097  [DOI](https://doi.org/10.1055/s-0033-1361097) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24464600/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Exp.%20Clin.%20Endocrinol.%20Diabetes&title=CYP2C9%E2%88%972%20Allele%20Increases%20Risk%20for%20Hypoglycemia%20in%20POR%E2%88%971/%E2%88%971%20Type%202%20Diabetic%20Patients%20Treated%20with%20Sulfonylureas.&author=G.%20Ragia&author=A.%20Tavridou&author=L.%20Elens&author=R.%20H.%20N.%20Van%20Schaik&author=V.%20G.%20Manolopoulos&volume=122&publication_year=2014&pages=60-63&pmid=24464600&doi=10.1055/s-0033-1361097&)

35. Ragia G., Tavridou A., Petridis I., Manolopoulos V. G. (2012). Association of KCNJ11 E23K gene polymorphism with hypoglycemia in sulfonylurea-treated Type 2 diabetic patients. Diabetes Res. Clin. Pract. 98 119–124. 10.1016/j.diabres.2012.04.017  [DOI](https://doi.org/10.1016/j.diabres.2012.04.017) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22591706/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Diabetes%20Res.%20Clin.%20Pract.&title=Association%20of%20KCNJ11%20E23K%20gene%20polymorphism%20with%20hypoglycemia%20in%20sulfonylurea-treated%20Type%202%20diabetic%20patients.&author=G.%20Ragia&author=A.%20Tavridou&author=I.%20Petridis&author=V.%20G.%20Manolopoulos&volume=98&publication_year=2012&pages=119-124&pmid=22591706&doi=10.1016/j.diabres.2012.04.017&)

36. Semiz S., Dujic T., Ostanek B., Prnjavorac B., Bego T., Malenica M. (2010). Analysis of CYP2C9∗2 CYP2C19∗2 and CYP2D6∗4 polymorphisms in patients with type 2 diabetes mellitus. Bosn. J. Basic Med. Sci. 10 287–291. 10.17305/bjbms.2010.2662  [DOI](https://doi.org/10.17305/bjbms.2010.2662) | [PMC free article](/articles/PMC5489036/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21108610/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bosn.%20J.%20Basic%20Med.%20Sci.&title=Analysis%20of%20CYP2C9%E2%88%972%20CYP2C19%E2%88%972%20and%20CYP2D6%E2%88%974%20polymorphisms%20in%20patients%20with%20type%202%20diabetes%20mellitus.&author=S.%20Semiz&author=T.%20Dujic&author=B.%20Ostanek&author=B.%20Prnjavorac&author=T.%20Bego&volume=10&publication_year=2010&pages=287-291&pmid=21108610&doi=10.17305/bjbms.2010.2662&)

37. Sokolova E. A., Bondar I. A., Shabelnikova O. Y., Pyankova O. V., Filipenko M. L. (2015). Replication of KCNJ11 (p.E23K) and ABCC8 (p.S1369A) Association in Russian diabetes mellitus 2 Type cohort and meta-analysis. PLoS One 10:e0124662. 10.1371/journal.pone.0124662  [DOI](https://doi.org/10.1371/journal.pone.0124662) | [PMC free article](/articles/PMC4425644/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25955821/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=Replication%20of%20KCNJ11%20(p.E23K)%20and%20ABCC8%20(p.S1369A)%20Association%20in%20Russian%20diabetes%20mellitus%202%20Type%20cohort%20and%20meta-analysis.&author=E.%20A.%20Sokolova&author=I.%20A.%20Bondar&author=O.%20Y.%20Shabelnikova&author=O.%20V.%20Pyankova&author=M.%20L.%20Filipenko&volume=10&issue=e0124662&publication_year=2015&pmid=25955821&doi=10.1371/journal.pone.0124662&)

38. Song I. S., Shin H. J., Shim E. J., Jung I. S., Kim W. Y., Shon J. H. (2008). Genetic variants of the organic cation transporter 2 influence the disposition of metformin. Clin. Pharmacol. Ther. 84 559–562. 10.1038/clpt.2008.61  [DOI](https://doi.org/10.1038/clpt.2008.61) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18401339/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Genetic%20variants%20of%20the%20organic%20cation%20transporter%202%20influence%20the%20disposition%20of%20metformin.&author=I.%20S.%20Song&author=H.%20J.%20Shin&author=E.%20J.%20Shim&author=I.%20S.%20Jung&author=W.%20Y.%20Kim&volume=84&publication_year=2008&pages=559-562&pmid=18401339&doi=10.1038/clpt.2008.61&)

39. Stransky N., Ghandi M., Kryukov G. V., Garraway L. A., Lehar J., Liu M. (2015). Pharmacogenomic agreement between two cancer cell line data sets. Nature 528 84–87. 10.1038/nature15736  [DOI](https://doi.org/10.1038/nature15736) | [PMC free article](/articles/PMC6343827/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26570998/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Pharmacogenomic%20agreement%20between%20two%20cancer%20cell%20line%20data%20sets.&author=N.%20Stransky&author=M.%20Ghandi&author=G.%20V.%20Kryukov&author=L.%20A.%20Garraway&author=J.%20Lehar&volume=528&publication_year=2015&pages=84-87&pmid=26570998&doi=10.1038/nature15736&)

40. Sulkowska J. I., Morcos F., Weigt M., Hwa T., Onuchic J. N. (2012). Genomics-aided structure prediction. Proc. Natl. Acad. Sci. U.S.A. 109 10340–10345. 10.1073/pnas.1207864109  [DOI](https://doi.org/10.1073/pnas.1207864109) | [PMC free article](/articles/PMC3387073/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22691493/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.S.A.&title=Genomics-aided%20structure%20prediction.&author=J.%20I.%20Sulkowska&author=F.%20Morcos&author=M.%20Weigt&author=T.%20Hwa&author=J.%20N.%20Onuchic&volume=109&publication_year=2012&pages=10340-10345&pmid=22691493&doi=10.1073/pnas.1207864109&)

41. Tkac I. (2015). Genetics of Drug Response in Type 2 Diabetes. Curr. Diab. Rep. 15:43. 10.1007/s11892-015-0617-2  [DOI](https://doi.org/10.1007/s11892-015-0617-2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25975599/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr.%20Diab.%20Rep.&title=Genetics%20of%20Drug%20Response%20in%20Type%202%20Diabetes.&author=I.%20Tkac&volume=15&issue=43&publication_year=2015&pmid=25975599&doi=10.1007/s11892-015-0617-2&)

42. Tkac I., Klimcakova L., Javorsky M., Fabianova M., Schroner Z., Hermanova H. (2013). Pharmacogenomic association between a variant in SLC47A1 gene and therapeutic response to metformin in type 2 diabetes. Diabetes Obes. Metab. 15 189–191. 10.1111/j.1463-1326.2012.01691.x  [DOI](https://doi.org/10.1111/j.1463-1326.2012.01691.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22882994/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Diabetes%20Obes.%20Metab.&title=Pharmacogenomic%20association%20between%20a%20variant%20in%20SLC47A1%20gene%20and%20therapeutic%20response%20to%20metformin%20in%20type%202%20diabetes.&author=I.%20Tkac&author=L.%20Klimcakova&author=M.%20Javorsky&author=M.%20Fabianova&author=Z.%20Schroner&volume=15&publication_year=2013&pages=189-191&pmid=22882994&doi=10.1111/j.1463-1326.2012.01691.x&)

43. Turner S. D., Crawford D. C., Ritchie M. D. (2009). Methods for optimizing statistical analyses in pharmacogenomics research. Expert Rev. Clin. Pharmacol. 2 559–570. 10.1586/ecp.09.32  [DOI](https://doi.org/10.1586/ecp.09.32) | [PMC free article](/articles/PMC2835152/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20221410/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Expert%20Rev.%20Clin.%20Pharmacol.&title=Methods%20for%20optimizing%20statistical%20analyses%20in%20pharmacogenomics%20research.&author=S.%20D.%20Turner&author=D.%20C.%20Crawford&author=M.%20D.%20Ritchie&volume=2&publication_year=2009&pages=559-570&pmid=20221410&doi=10.1586/ecp.09.32&)

44. Tzvetkov M. V., Vormfelde S. V., Balen D., Meineke I., Schmidt T., Sehrt D. (2009). The effects of genetic polymorphisms in the organic cation transporters OCT1 OCT2 and OCT3 on the renal clearance of metformin. Clin. Pharmacol. Ther. 86 299–306. 10.1038/clpt.2009.92  [DOI](https://doi.org/10.1038/clpt.2009.92) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19536068/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=The%20effects%20of%20genetic%20polymorphisms%20in%20the%20organic%20cation%20transporters%20OCT1%20OCT2%20and%20OCT3%20on%20the%20renal%20clearance%20of%20metformin.&author=M.%20V.%20Tzvetkov&author=S.%20V.%20Vormfelde&author=D.%20Balen&author=I.%20Meineke&author=T.%20Schmidt&volume=86&publication_year=2009&pages=299-306&pmid=19536068&doi=10.1038/clpt.2009.92&)

45. van Dam R. M., Hoebee B., Seidell J. C., Schaap M. M., de Bruin T. W., Feskens E. J. (2005). Common variants in the ATP-sensitive K+ channel genes KCNJ11 (Kir6.2) and ABCC8 (SUR1) in relation to glucose intolerance: population-based studies and meta-analyses. Diabetes Med. 22 590–598. 10.1111/j.1464-5491.2005.01465.x  [DOI](https://doi.org/10.1111/j.1464-5491.2005.01465.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15842514/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Diabetes%20Med.&title=Common%20variants%20in%20the%20ATP-sensitive%20K+%20channel%20genes%20KCNJ11%20(Kir6.2)%20and%20ABCC8%20(SUR1)%20in%20relation%20to%20glucose%20intolerance:%20population-based%20studies%20and%20meta-analyses.&author=R.%20M.%20van%20Dam&author=B.%20Hoebee&author=J.%20C.%20Seidell&author=M.%20M.%20Schaap&author=T.%20W.%20de%20Bruin&volume=22&publication_year=2005&pages=590-598&pmid=15842514&doi=10.1111/j.1464-5491.2005.01465.x&)

46. Villareal D. T., Koster J. C., Robertson H., Akrouh A., Miyake K., Bell G. I. (2009). Kir6.2 Variant E23K Increases ATP-Sensitive K(+) channel activity and is associated with impaired insulin release and enhanced insulin sensitivity in adults with normal glucose tolerance. Diabetes 58 1869–1878. 10.2337/db09-0025  [DOI](https://doi.org/10.2337/db09-0025) | [PMC free article](/articles/PMC2712777/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19491206/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Diabetes&title=Kir6.2%20Variant%20E23K%20Increases%20ATP-Sensitive%20K(+)%20channel%20activity%20and%20is%20associated%20with%20impaired%20insulin%20release%20and%20enhanced%20insulin%20sensitivity%20in%20adults%20with%20normal%20glucose%20tolerance.&author=D.%20T.%20Villareal&author=J.%20C.%20Koster&author=H.%20Robertson&author=A.%20Akrouh&author=K.%20Miyake&volume=58&publication_year=2009&pages=1869-1878&pmid=19491206&doi=10.2337/db09-0025&)

47. Zhou K. X., Donnelly L. A., Kimber C. H., Donnan P. T., Doney A. S. F., Leese G. (2009). Reduced-Function SLC22A1 polymorphisms encoding organic cation transporter 1 and glycemic response to metformin: a GoDARTS study. Diabetes Metab. Res. Rev. 58 1434–1439. 10.2337/db08-0896  [DOI](https://doi.org/10.2337/db08-0896) | [PMC free article](/articles/PMC2682689/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19336679/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Diabetes%20Metab.%20Res.%20Rev.&title=Reduced-Function%20SLC22A1%20polymorphisms%20encoding%20organic%20cation%20transporter%201%20and%20glycemic%20response%20to%20metformin:%20a%20GoDARTS%20study.&author=K.%20X.%20Zhou&author=L.%20A.%20Donnelly&author=C.%20H.%20Kimber&author=P.%20T.%20Donnan&author=A.%20S.%20F.%20Doney&volume=58&publication_year=2009&pages=1434-1439&pmid=19336679&doi=10.2337/db08-0896&)
