# TSPAN5, ERICH3 and selective serotonin reuptake inhibitors in major depressive disorder: pharmacometabolomics-informed pharmacogenomics

## Metadata
**Authors:** M Gupta, D Neavin, D Liu, J Biernacka, D Hall-Flavin, W V Bobo, M A Frye, M Skime, G D Jenkins, A Batzler, K Kalari, W Matson, S S Bhasin, H Zhu, T Mushiroda, Y Nakamura, M Kubo, L Wang, R Kaddurah-Daouk, R M Weinshilboum
**Journal:** Molecular Psychiatry
**Date:** 2016 Feb 23
**DOI:** [10.1038/mp.2016.6](https://doi.org/10.1038/mp.2016.6)
**PMID:** 26903268
**PMCID:** PMC5003027
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5003027/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC5003027/pdf/mp20166a.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC5003027/pdf/mp20166a.pdf)

## Abstract

Millions of patients suffer from major depressive disorder (MDD), but many do not respond to selective serotonin reuptake inhibitor (SSRI) therapy. We used a pharmacometabolomics-informed pharmacogenomics research strategy to identify genes associated with metabolites that were related to SSRI response. Specifically, 306 MDD patients were treated with citalopram or escitalopram and blood was drawn at baseline, 4 and 8 weeks for blood drug levels, genome-wide single nucleotide polymorphism (SNP) genotyping and metabolomic analyses. SSRI treatment decreased plasma serotonin concentrations (P<0.0001). Baseline and plasma serotonin concentration changes were associated with clinical outcomes (P<0.05). Therefore, baseline and serotonin concentration changes were used as phenotypes for genome-wide association studies (GWAS). GWAS for baseline plasma serotonin concentrations revealed a genome-wide significant (P=7.84E-09) SNP cluster on chromosome four 5' of TSPAN5 and a cluster across ERICH3 on chromosome one (P=9.28E-08) that were also observed during GWAS for change in serotonin at 4 (P=5.6E-08 and P=7.54E-07, respectively) and 8 weeks (P=1.25E-06 and P=3.99E-07, respectively). The SNPs on chromosome four were expression quantitative trait loci for TSPAN5. Knockdown (KD) and overexpression (OE) of TSPAN5 in a neuroblastoma cell line significantly altered the expression of serotonin pathway genes (TPH1, TPH2, DDC and MAOA). Chromosome one SNPs included two ERICH3 nonsynonymous SNPs that resulted in accelerated proteasome-mediated degradation. In addition, ERICH3 and TSPAN5 KD and OE altered media serotonin concentrations. Application of a pharmacometabolomics-informed pharmacogenomic research strategy, followed by functional validation, indicated that TSPAN5 and ERICH3 are associated with plasma serotonin concentrations and may have a role in SSRI treatment outcomes.

## Introduction

Major depressive disorder (MDD) is the most common psychiatric disorder worldwide, with a lifetime prevalence of approximately 13%.^1, 2^ MDD is associated with marked morbidity and premature mortality.^3^ Although the causes of MDD are not fully understood, relative deficiency of the neurotransmitter serotonin appears to have a role in the pathophysiology of MDD and, as a result, drugs that enhance serotonergic neurotransmission are used to treat MDD.^4, 5^ Selective serotonin reuptake inhibitors (SSRIs), drugs that increase serotonin signaling in the central nervous system by blocking its presynaptic reuptake, are first-line pharmacologic therapy for MDD.^6, 7, 8^ However, response to SSRIs is highly variable, with less than half of MDD patients achieving remission during therapy with these drugs.^9, 10, 11^

Twin and other genetic studies suggest that inheritance contributes both to MDD risk^12, 13, 14, 15^ and to variation in SSRI response.^16, 17, 18^ We^10, 11^ and others^19, 20, 21, 22, 23, 24^ have performed antidepressant response candidate gene and genome-wide association studies (GWAS), but with only limited success and with few replicated findings.^17, 25, 26, 27^ Relative lack of power, variation in study design and phenotypic heterogeneity may all contribute to this state of affairs. The addition of other ‘omics' to genomics might make it possible to achieve enhanced patient subclassification, thus making it possible to identify novel genetic factors that contribute to variation in SSRI response.

We have previously used pharmacometabolomics to help guide and inform genomic studies of SSRI clinical response.^28, 29^ Metabolomics is being used increasingly to identify ‘biosignatures' for disease subclassification and/or drug response phenotype(s).^30, 31, 32^ Pharmacometabolomics is an emerging field that uses ‘metabolic profiles' to characterize biological response to drug treatment.^28, 29, 33, 34, 35^ In the present study, 306 MDD patients were randomly selected from the Mayo Clinic Pharmacogenomics Research Network Antidepressant Medication Pharmacogenomics Study (PGRN-AMPS) SSRI trial who were included in our ‘Clinical SSRI Response' and ‘Citalopram and Escitalopram Metabolism' GWA studies.^11, 36, 37^ Plasma samples from those patients were used to perform metabolomic studies through the Pharmacometabolomics Research Network at baseline and after 4 and 8 weeks of SSRI therapy, for a total of 918 samples assayed. Among the metabolites analyzed, plasma serotonin concentrations and changes in plasma serotonin concentrations were associated with the largest number of SSRI treatment outcome measures. Specifically, patients with higher baseline plasma serotonin concentrations and/or greater decreases in plasma serotonin concentrations responded better to SSRI therapy. We then moved from metabolomics to genomics by performing GWAS to identify genes associated with variation in plasma serotonin concentrations or changes in serotonin concentrations during SSRI therapy, followed by the functional pursuit of those genes in neuronal cell models.

Specifically, when GWAS was performed with baseline plasma serotonin concentrations as the phenotype, a genome-wide significant (*P*=7.84E-09) single nucleotide polymorphism (SNP) signal that was 5' of the *Tetraspanin 5* (*TSPAN5*) gene on chromosome four and a cluster of SNPs across the *Glutamate-rich 3* (*ERICH3*) gene on chromosome one (*P*=9.28E-08) were identified. Those same SNP signals were identified during GWAS for change in plasma serotonin concentrations after 4 and 8 weeks of SSRI therapy. In addition, the Genome Tissue Expression (GTEx) Database^38^ showed that both of those genes were highly expressed in the brain. The SNPs 5' of *TSPAN5* were cis-expression quantitative trait loci (eQTLs) for that gene. Follow-up functional genomic experiments performed by knocking down or overexpressing *TSPAN5* in a neuroblastoma cell line resulted in significant alterations in the expression of genes encoding serotonin pathway enzymes as well as changes in the concentration of serotonin in the cell culture media. Two of the SNPs in the *ERICH3* SNP cluster encoded nonsynonymous variants (ns) that were associated with accelerated proteasome-mediated degradation of ERICH3. In addition, changes in *ERICH3* expression significantly altered media serotonin concentrations but did not influence serotonin pathway gene expression. Finally, one of the *ERICH3* nsSNPs (rs11580409, *P*=1.12E-07) was associated with clinical SSRI response in the International SSRI Pharmacogenomics Consortium (ISPC), an observation that was replicated in the Sequenced Treatment Alternatives to Relieve Depression (STAR*D) study. In summary, the application of a ‘pharmacometabolomics-informed pharmacogenomic' research strategy made it possible to identify two novel genes related to plasma serotonin concentration—a phenotype that was associated with SSRI clinical response.

## Materials and methods

### Trial design, samples and metabolomic assays

Patient selection, treatment outcomes and blood sample collection for the Pharmacogenomics Research Network Antidepressant Medication Pharmacogenomics Study (PGRN-AMPS) SSRI trial have been described in detail elsewhere.^11, 36, 37^ Plasma metabolite concentrations were assayed using samples from 306 randomly selected MDD patients at baseline and after 4 and 8 weeks of SSRI therapy using a high-performance liquid chromatography electrochemical coulometric array metabolomics platform.^31, 39^ See [Supplementary Text](#sup1) for details.

### Genotyping and statistical analyses

DNA from PGRN-AMPS SSRI trial patients was genotyped at the RIKEN Center for Genomic Medicine (Yokohama, Japan) using Illumina human 610-Quad BeadChips (Illumina, San Diego, CA, USA), as described previously.^11, 37^ GWAS were performed using approximately 7.5 million SNPs. Patients were removed from the analysis for non-compliance or non-Caucasian heritage. Baseline analyses were adjusted for age and sex. Metabolite concentrations and changes in metabolite concentrations after SSRI treatment were tested for association with QIDS-C16 percent change, response and remission. See [Supplementary Text](#sup1) for details.

### TSPAN5, ERICH3 and SNP function

Lymphoblastoid cell lines (LCLs) were selected from the 'Human Variation Panel' based on *TSPAN5* or *ERICH3* SNP genotypes to determine whether the SNPs were eQTLs for those genes. The 300 LCLs (100 European-American, 100 African-American and 100 Han Chinese-American subjects) in the ‘Human Variation Panel' that had been SNP genotyped previously have been utilized repeatedly to generate and test pharmacogenomic hypotheses.^40, 41, 42, 43, 44^ *TSPAN5* SNP function was assessed using electrophoretic mobility shift assays and dual luciferase reporter gene assays. Expression constructs for *ERICH3* that encoded wild type (WT) as well as one or both nsSNPs (rs11580409 or rs11210490) were expressed with or without the proteasome inhibitor MG132 or the autophagy inhibitor 3-methyladenosine. See [Supplementary Text](#sup1) for details.

### TSPAN5 and ERICH3 expression and the serotonin pathway

After TSPAN5 or ERICH3 knockdown (KD) or overexpression (OE) in neurally derived cell lines, serotonin pathway enzyme expression was assessed by quantitative real-time polymerase chain reaction (qRT-PCR) and quantitative western blot. Cell culture media serotonin concentrations were measured by Bioanalytical Systems (BASi, West Lafayette, IN, USA). See [Supplementary Text](#sup1) for details.

## Results

### Plasma metabolite concentrations and their association with clinical outcomes

We set out to use plasma metabolomic profiles of MDD patients being treated with SSRIs to identify metabolites that were correlated with SSRI clinical outcomes and, subsequently, SNPs/genes associated with those metabolite concentrations for functional study in neuronal cell lines. This approach made it possible to move from peripheral plasma metabolomics to genomics and then to test genomic candidates in neural cells—addressing concerns with regard to the relevance of peripheral biomarkers for neuronal function.

Specifically, a liquid chromatography electrochemical coulometric array metabolomics platform was used to quantify 31 known plasma metabolites ([Supplementary Table 1](#sup1)), primarily metabolites in the tryptophan, tyrosine, purine and tocopherol pathways, at three time points—baseline and after 4 and 8 weeks of SSRI therapy. We then determined the association of those metabolites with measures of clinical response (remission, response and percent change in QIDS-C16) after 4 and 8 weeks of SSRI therapy. Plasma serotonin concentrations at baseline as well as their change after 4 and 8 weeks of SSRI treatment were more highly associated with SSRI response phenotypes than those for any other metabolite ([Table 1](#tbl1) and [Supplementary Table 2](#sup1)). The associations listed in [Table 1](#tbl1) are ‘nominal' and have not been corrected for multiple comparisons because the purpose was to identify metabolites to use for GWAS. Plasma serotonin concentrations decreased significantly after SSRI treatment at both 4 (*P*<0.0001) and 8 weeks (*P*<0.0001) ([Figure 1](#fig1)). The odds ratios (OR) and correlation coefficients (*r*) listed in [Table 1](#tbl1) indicated that higher baseline plasma serotonin concentrations as well as larger decreases in plasma serotonin concentrations between baseline and 4 or 8 weeks of therapy were both associated with better clinical outcomes. We then performed GWAS using baseline plasma serotonin concentrations and change in plasma serotonin concentrations at 4 and 8 weeks of SSRI therapy as phenotypes.

### Table 1. Association of plasma serotonin concentrations with clinical outcomes.

| Clinical outcomes | Remission at 4 weeks | Remission at 8 weeks | Response at 4 weeks | Response at 8 weeks | % change at 4 weeks | % change at 8 weeks |
| --- | --- | --- | --- | --- | --- | --- |
| Baseline | P=0.012 | P=0.028 | P=0.007 | P=0.047 | P=0.015 | P=0.019 |
|   | OR=1.41 | OR=1.31 | OR=1.40 | OR=1.30 | r=−0.14 | r=−0.14 |
| Change after 4 weeks | P=0.011 | P=0.041 | P=0.026 | P=0.060 | P=0.021 | P=0.024 |
|   | OR=1.40 | OR=1.27 | OR=1.31 | OR=1.27 | r=−0.13 | r=−0.13 |
| Change after 8 weeks | P=0.069 | P=0.147 | P=0.037 | P=0.130 | P=0.041 | P=0.06 |
|   | OR=1.27 | OR=1.19 | OR=1.29 | OR=1.21 | r=−0.12 | r=−0.11 |

Table 1 Caption: Abbreviation: OR, odds ratio; SSRI, selective serotonin reuptake inhibitor. Plasma serotonin concentrations at baseline and decreases in plasma serotonin concentrations after 4 weeks of SSRI treatment were nominally associated with remission, response and percent change in QIDS-C16 score. The decrease in plasma serotonin between baseline and 8 weeks of SSRI treatment was associated only with the response at 4 weeks and percent change in QIDS-C16 at 4 weeks. OR>1 indicates improvement (associated with higher baseline and larger changes in plasma serotonin concentrations) and the negative r values indicate a decrease in QIDS-C16 scores, i.e., improvement. P-values<0.05 have been bolded.

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9280/5116483/3dea0a483a50/mp20166f1.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5116483_mp20166f1.jpg)

Patient plasma serotonin concentrations. Relative plasma serotonin concentrations (expressed as a ratio of the standard) in major depressive disorder patient samples were decreased significantly after 4 and 8 weeks of selective serotonin reuptake inhibitor treatment when compared with baseline. ***P<0.0001.

### GWAS for plasma serotonin and change in serotonin concentrations

The Manhattan plot of the GWAS for baseline plasma serotonin concentrations showed a genome-wide significant SNP cluster on chromosome four that consisted of 15 SNPs in tight linkage disequilibrium that mapped 15–25 kilobases (kb) 5' of the *Tetraspanin 5* (*TSPAN5*) gene, with the lowest *P*-value (7.84E-09) for the rs11947402 SNP ([Figures 2a and c](#fig2) and [Supplementary Table 3](#sup1)). In addition, there was a SNP cluster on chromosome one across the *Glutamate-rich 3* (*ERICH3*) gene that included two nsSNPs (rs11580409 and rs11210490), with the lowest *P*-value (9.28E-08) for the rs696692 SNP ([Figures 2a and b](#fig2)). These same two signals were observed in Manhattan plots of GWAS data for change in plasma serotonin concentrations after SSRI therapy for 4 (*TSPAN5*: rs11947402, *P*=5.6E-08; *ERICH3*: rs696692, *P*=7.54E-07) and 8 weeks (*TSPAN5*: rs11947402, *P*=1.25E-06; *ERICH3*: rs699848, *P*=3.99E-07) ([Supplementary Figure 1](#sup1)). QQ plots for these GWAS are shown in [Supplementary Figure 2](#sup1).

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9280/5116483/b32d6f0673b6/mp20166f2.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5116483_mp20166f2.jpg)

Baseline serotonin concentration GWAS. (a) GWAS for baseline plasma serotonin concentrations revealed a genome-wide significant signal on chromosome 4 as well as a suggestive SNP cluster on chromosome 1. (b) The locus zoom shows that the SNPs on chromosome 1 are across ERICH3. The SNP most highly associated with baseline plasma serotonin concentration from this cluster was rs696692 (P=9.28E-08). (c) The locus zoom for the genome-wide significant SNP cluster on chromosome 4 shows that the SNPs are approximately 15–25 kb 5' of TSPAN5, with rs11947402 as the most highly associated with baseline plasma serotonin concentration (P=7.84E-09). GWAS, genome-wide association study; SNP, single nucleotide polymorphism.

Variant *TSPAN5* SNP genotypes were associated with higher baseline plasma serotonin concentrations and greater decreases in plasma serotonin concentration during SSRI therapy ([Supplementary Figure 3a-c](#sup1)). Conversely, *ERICH3* variant allele genotypes were associated with lower baseline plasma serotonin concentrations and smaller decreases in plasma serotonin concentrations during SSRI therapy ([Supplementary Figure 3d-f](#sup1)).

The minor allele frequency for the SNPs 5' of *TSPAN5* was 7% in our European-American MDD patients, consistent with the 6.7% value reported for European populations by the 1000 Genomes Project.^45^ The relatively low minor allele frequency value complicated efforts to examine the effect of homozygosity for the variant allele in the functional genomic studies described subsequently. The SNPs across the *ERICH3* gene had a minor allele frequency of 35% in the PGRN-AMPS patients, similar to the 34.1% figure reported for European populations by the 1000 Genomes Project.^45^ As the SNPs near *TSPAN5* were genome-wide significant and the SNPs across *ERICH3* were highly suggestive, we pursued the possible functional implications of both signals.

### TSPAN5 and ERICH3 SNPs as eQTLs

The initial question that we asked with regard to the SNPs 5' of *TSPAN5* and across *ERICH3* was whether they might be cis-eQTLs for those genes. Specifically, we selected LCLs from the Human Variation Panel that were either homozygous WT or variant (V) for the SNPs 5' of *TSPAN5* or those across *ERICH3* and performed qRT-PCR analysis. TSPAN5 mRNA was decreased in LCLs homozygous for the variant genotype as compared with LCLs homozygous for the WT genotype (*P*<0.05; [Figure 3a](#fig3)). However, there was no difference in *ERICH3* expression between LCLs homozygous WT and homozygous variant for the *ERICH3* SNPs (data not shown).

### Figure 3.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9280/5116483/1f6525e77118/mp20166f3.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5116483_mp20166f3.jpg)

TSPAN5 and ERICH3 SNP function. TSPAN5 expression is decreased for cells or tissues homozygous for variant (V/V) or with heterozygous (WT/V) SNP genotypes as compared with homozygous wild type (WT/WT) in (a) LCLs, (b) cerebral cortex and (c) frontal cortex. (d) Luciferase assay results comparing WT and variant SNP genotypes (rs1918743, rs59961429 and rs56095565) effects on transcriptional activities indicate decreased transcription for the variant TSPAN5 SNPs in SK-N-BE(2) neuroblastoma cells; (e) ERICH3 plasmids that were WT or contained one or both of the nonsynonymous SNPs (rs11580409 and rs11210490) were expressed in HEK-293T/17 cells. Both P264A (rs11210490) and L1056V (rs11580409) were associated with decreased protein levels as compared with WT, but L1056V was associated with a much greater decrease in protein level. (f) Quantification of ERICH3 protein relative to the GAPDH control for the ERICH3 western blots shown in (e). (g) Plasmids encoding ERICH3 allozymes that were WT or contained one or both of the amino acid substitutions (P264A and L1056V) were expressed in HEK-293T/17 cells with and without a protease inhibitor (MG132) or an autophagy inhibitor (3MA). MG132 prevented ERICH3 SNP-dependent protein degradation but 3MA did not. (h) Quantification of proteasome and autophagy inhibition of the ERICH3 allozyme degradation studies shown in (g). EV, empty vector; NS, non-significant; *P<0.05; **P<0.01; ***P<0.0001. LCL, lymphoblastoid cell lines; SNP, single nucleotide polymorphism.

We next consulted eQTL databases to determine whether the SNPs 5' of *TSPAN5* or across *ERICH3* might be cis-eQTLs. The GTEx Database^38^ showed that brain displayed high expression for both TSPAN5 and ERICH3 ([Supplementary Figures 4 and 5](#sup1)). Previous studies in mice had reported that the tissue with the highest TSPAN5 protein level was brain.^46, 47, 48, 49^ Unfortunately, GTEx had too few samples to provide reliable eQTL data for *TSPAN5*. However, the Brain eQTL Almanac (BRAINEAC) database included data for 134 human brain samples.^50^ We found that, in the brain areas with the highest TSPAN5 expression (cerebral and frontal cortex), the SNPs were once again cis-eQTLs for TSPAN5 (*P*=6.9E-05 and 0.027, respectively), with lower expression for homozygous variant or heterozygous genotypes—just as we found for LCLs ([Figures 3b and c](#fig3)). Finally, we used the Blood eQTL browser that contains data for 5311 individual blood samples.^51^ Once again, the SNPs were identified as eQTLs for *TSPAN5* (*P*=1.36E-14) but, in this case, the Z-score of 7.8 indicated that the variant allele was associated with higher *TSPAN5* expression—opposite to what we found using LCLs or BRAINEAC. Therefore, these SNPs are cis-eQTLs for *TSPAN5*, but there appears to be tissue-specific directionality of associations with the SNPs—an important factor for the interpretation of subsequent functional studies. The SNPs across *ERICH3* were not cis-eQTLs in any of these databases.

We next attempted to determine which of the SNPs 5' of *TSPAN5* might influence expression. Specifically, we used the TRANSFAC 6.0 database to identify transcription factors that might bind to DNA sequences that contained the SNPs. Eight of the SNPs were predicted to potentially disrupt or create transcription factor-binding sites. Electrophoretic mobility shift assays performed using nuclear protein extracts from neuroblastoma SK-N-BE(2) cells revealed differences in nuclear protein-binding patterns between WT and variant SNP sequences for three of the eight SNPs (rs1918743, rs59961429 and rs56095565) ([Supplementary Figure 6](#sup1)). In an attempt to more directly determine the possible role of these three SNPs in transcription, we performed luciferase reporter assays by transfecting luciferase reporter gene constructs containing each of the SNPs into SK-N-BE(2) cells. Each of the variant SNP genotypes significantly decreased luciferase activity when compared with the WT genotype, indicating decreased transcriptional activity in SK-N-BE(2) neuroblastoma cells ([Figure 3d](#fig3))—compatible with the results of our eQTL analyses for brain and LCLs.

### ERICH3 SNPs and proteasome-mediated degradation

As the SNPs across *ERICH3* were not cis-eQTLs for that gene, we tested the possibility that the two nsSNPs might affect ERICH3 protein concentrations. Proteasome-mediated degradation is a common functional mechanism for the effect of nsSNPs.^52, 53, 54, 55^ *ERICH3* cDNA constructs that were WT or contained one or both of the nsSNPs (rs11580409 and rs11210490) were transfected into HEK-293T/17 cells. The rs11210490 SNP (Pro264Ala) was associated with a small (28%), but significant (*P*<0.05) reduction in ERICH3 protein, while the rs11580409 SNP (Leu1056Val) was associated with an 80% decrease of ERICH protein (*P*<0.001). Constructs with both nsSNPs were associated with a 93% reduction in ERICH3 protein (*P*<0.001) ([Figures 3e and f](#fig3)). Furthermore, the proteasome inhibitor (MG132) increased ERICH3 variant allozyme concentrations, but the autophagy inhibitor (3-methyladenosine) did not ([Figures 3g and h](#fig3)), indicating that degradation of the variant ERICH3 allozymes was proteasome-mediated. We next attempted to identify the potential functional relationship of TSPAN5 and ERICH3 with baseline and change in plasma serotonin concentrations.

### TSPAN5 and ERICH3 expression and serotonin pathway enzyme gene expression

SK-N-BE(2) neuroblastoma cells were used to perform *TSPAN5* functional genomic studies because they are derived from neural cells and express TSPAN5 and serotonin pathway enzymes ([Figure 4a](#fig4)). When TSPAN5 was knocked down more than 70% in SK-N-BE(2) cells, there was a significant decrease of mRNA and protein levels for the serotonin pathway enzymes TPH1, TPH2, DDC, MAOA, ([Figure 4a](#fig4)) as well as the serotonin transporter SLC6A4 ([Figures 4b and d](#fig4)). Furthermore, OE of *TSPAN5* was associated with increased expression of *TPH1*, *TPH2*, *DDC* and *MAOA* ([Figures 4b and d](#fig4)). Protein levels were not induced to the same extent as mRNA after *TSPAN5* OE, perhaps because *TSPAN5* is highly expressed in those cells.

### Figure 4.

![Figure 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9280/5116483/ccaf6d055ae0/mp20166f4.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5116483_mp20166f4.jpg)

TSPAN5 and ERICH3 association with serotonin biosynthesis and metabolism. (a) Serotonin biosynthesis and metabolism pathway. (b) mRNA expression of genes encoding serotonin pathway enzymes as measured by qRT-PCR were decreased after TSPAN5 KD (black) and increased after TSPAN5 OE (cross-hatched) in SK-N-BE(2) neuroblastoma cells. (c, d) Western blot analysis indicated decreased serotonin enzyme protein levels after TSPAN5 KD but no significant change after TSPAN5 OE, as quantified in (d). (e, f) Culture media serotonin concentrations after TSPAN5 (e) KD and (f) OE. (g, h) Cell culture media serotonin concentrations after ERICH3 (g) KD and (h) OE. *P<0.05; **P<0.01; TPH1/2, tryptophan hydroxylase 1/2; DDC, dopa decarboxylase; KD, knockdown; MAOA/B, monoamine oxidase A/B; OE, overexpression; SLC6A4, serotonin transporter; EV, empty vector.

Similar ERICH3 KD and OE experiments were performed using neurally derived cells including neuroblastoma cells (SK-N-SH and SK-N-BE(2)), human neural progenitor-derived neurons and glioblastoma cells (U251). However, ERICH3 KD and OE did not alter the expression of serotonin pathway enzymes (data not shown).

### RBPJ-κ expression and serotonin pathway enzyme gene expression

TSPAN5 has been reported to be involved in Notch signaling^56^ through ADAM10 recruitment^57, 58, 59, 60^ and the Notch-Recombination Signal Binding Protein for Immunoglobulin Kappa J Region (RBPJ-κ) has been implicated in regulation of the expression of serotonin pathway genes.^61^ Therefore, we knocked down RBPJ-κ in SK-N-BE(2) cells, and observed increased expression of *TPH1, TPH2*, *DDC* and *SLC6A4* ([Supplementary Figure 7](#sup1)). This may be one mechanism by which TSPAN5, an integral membrane protein, may influence serotonin biosynthesis, as described in more detail in the Discussion.

### TSPAN5 and ERICH3 and serotonin concentrations in cell culture media

Serotonin concentrations in cell culture media decreased significantly after TSPAN5 KD, but TSPAN5 OE did not result in significant changes in cell culture media serotonin concentrations ([Figures 4e and f](#fig4))—consistent with the changes observed in TPH1, TPH2, DDC, MAOA and SLC6A4 protein levels observed after TSPAN5 KD and OE ([Figures 4c and d](#fig4)). ERICH3 KD in human neural progenitor cell-derived neurons and OE in SK-N-BE(2) neuroblastoma cells were associated with significantly altered cell culture media serotonin concentrations ([Figures 4g and h](#fig4)), perhaps indicating that ERICH3 influences plasma serotonin concentrations through a mechanism that does not directly involve the expression of serotonin biosynthesis and metabolism enzyme genes. These two cell lines were chosen for study because both express the serotonin biosynthesis and metabolism enzymes and because ERICH3 is highly expressed in human neural progenitor cell-derived neurons, but is not expressed in SK-N-BE(2) neuroblastoma cells.

### SNP combinations and plasma serotonin concentrations

We next analyzed the possible association of baseline and change in plasma serotonin concentrations in our patients with combinations of genotypes for the top *TSPAN5* SNP (rs11947402) and the *ERICH3* rs11580409 nsSNP. As anticipated, patients who were homozygous WT for the *TSPAN5* SNP (rs11947402) and who were homozygous variant for the *ERICH3* SNP (rs11580409) had lower average baseline (*P*=1.76E-12) and smaller average decreases in serotonin concentrations after 4 (*P*=6.09E-11) and 8 weeks (*P*=1.84E-09) of SSRI treatment as compared with patients carrying the *TSPAN5* variant SNP allele and/or the *ERICH3* nsSNP WT allele (rs11580409). The R-squared values indicate that the TSPAN5 SNP (rs11947402) and *ERICH3* nsSNP (rs11580409) account for 18.8% of the baseline variation of serotonin concentrations and 15.4% and 13% of the variation in the change in serotonin concentrations after 4 and 8 weeks of SSRI treatment in this MDD population, respectively ([Supplementary Figure 8](#sup1)). The numbers of patients who had each SNP genotype combination are shown in [Supplementary Table 4](#sup1).

### ERICH3 and TSPAN5 SNPs and clinical phenotypes

Finally, we attempted to determine whether these SNPs might be associated with SSRI clinical response in the PGRN-AMPS GWAS^11^ and/or in two independent SSRI response GWAS (STAR*D^24^ and ISPC^10^). The nsSNP (rs11580409) in *ERICH3* that displayed striking proteasome-mediated degradation was associated with response at 4 weeks in the ISPC population (*P*=0.022, OR=1.25) and response at 6 weeks in the STAR*D population (*P*=0.041, OR=1.17). The SNPs in the cluster 5' of *TSPAN5* were not significantly associated with clinical response in any of these SSRI GWAS ([Supplementary Table 5](#sup1)).

## Discussion

A major goal of molecular psychiatry is to develop a molecular subclassification of psychiatric disease. In theory, that might allow a rational selection of optimal therapy for each patient—that is, ‘Precision' pharmacotherapy. In the present study, we used the most commonly prescribed antidepressant medication—SSRIs—as probes for molecular mechanisms associated with drug response. Specifically, we applied a pharmacometabolomics-informed pharmacogenomic research strategy during which we utilized plasma samples from 306 MDD patients enrolled in the Mayo PGRN-AMPS SSRI trial^11, 37^ to perform metabolomic assays for 31 metabolites, primarily metabolites from pathways related to monoamine neurotransmitters. The goal was to associate individual variation in these plasma metabolites with SSRI treatment outcomes—with the understanding that this represented only one step toward determining whether molecular mechanisms identified in the periphery might also have a role in neurotransmitter function in the brain. For that reason, we used GWAS to identify novel genes that might influence concentrations of the metabolite(s) identified in the periphery and then determined whether those same genes might also influence neuronal cell phenotypes.

We found that plasma serotonin concentrations in MDD patients decreased dramatically after SSRI treatment ([Figure 1](#fig1)). The effect of SSRIs on plasma serotonin concentrations is not well understood, and published results are contradictory—with some studies reporting decreased plasma serotonin after SSRI treatment,^32, 62, 63, 64, 65^ while others report increased concentrations,^66, 67, 68, 69^ which may be due, in part, to the plasma collection method or the platform used to assay serotonin. However, we observed a clear decrease in those concentrations in PGRN-AMPS MDD patients as measured with a liquid chromatography electrochemical coulometric array platform—a highly sensitive, quantitative method.^31, 39^ We also observed that both higher baseline plasma serotonin concentrations and greater decreases in plasma serotonin concentrations after 4 and 8 weeks of SSRI therapy were associated with better clinical outcomes (response, remission and QIDS-C16 percent change) than were observed for any of the other metabolites assayed ([Table 1](#tbl1) and [Supplementary Table 1](#sup1)). In an attempt to identify genes associated with individual variation in plasma serotonin concentrations and/or changes in those concentrations during SSRI therapy, we performed a GWAS for both phenotypes.

Unlike previous SSRI response GWAS in which no genome-wide significant SNPs were identified, our GWAS for baseline plasma serotonin concentrations included a genome-wide significant (7.84E-09) SNP cluster 5' of the *TSPAN5* gene on chromosome four ([Figures 2a and c](#fig2)). We also observed a SNP cluster across the *ERICH3* gene on chromosome one ([Figures 2a and b](#fig2)). The same SNP signals were observed when GWA studies were performed for change in plasma serotonin concentrations after 4 or 8 weeks of SSRI therapy ([Supplementary Figure 1](#sup1)). These observations raised the possibility that TSPAN5 and/or ERICH3 might be involved in the regulation of genes encoding enzymes in the serotonin metabolic pathway—a hypothesis that we tested in neurally derived cell lines.

In an attempt to understand the possible role of the SNPs 5' of *TSPAN5* in the regulation of serotonin biosynthesis, metabolism or transport, we first determined that those SNPs were cis-eQTLs for *TSPAN5* in LCLs, brain tissue and blood samples ([Figures 3d and g](#fig3) and [Supplementary Figure 6](#sup1)). However, the variant allele was associated with lower TSPAN5 expression in LCLs and brain tissue but higher expression in blood samples. We then showed that TSPAN5 KD and OE in neuroblastoma cells were associated with changes in the expression of serotonin pathway genes. In addition, TSPAN5 KD in neuroblastoma cells was associated with a significant decrease in serotonin concentration in the cell culture media ([Figures 4a and f](#fig4)). These results indicated that the SNPs 5' of *TSPAN5* could influence its expression, which, in turn appeared to have a role in the regulation of serotonin-related pathways.

TSPAN5 is a member of the tetraspanin superfamily, a family of proteins characterized by four hydrophobic transmembrane domains.^49, 70^ Tetraspanins form molecular complexes within the plasma membrane that can modulate cellular signaling.^71, 72^ TSPAN5 function has not been investigated extensively and has not previously been implicated in the regulation of serotonin or variation in SSRI response. However, several recent studies reported that TSPAN5 may promote Notch signaling^56^ by facilitating the transport of ADAM10, an α-secretase involved in cleaving the Notch receptor, to the cell membrane.^57, 58, 59, 60^ The Notch intracellular domain is then transported to the nucleus where it binds to transcription factors on gene promoters, inducing changes in gene expression.^73, 74^ A recent study reported that RBPJ-κ may have a role in the expression of *DDC* and *MAOA*,^61^ and we showed that RBPJ-κ KD resulted in increased expression of *TPH1/2*, *DDC* and *SLC6A4* in SK-N-BE(2) neuroblastoma cells ([Supplementary Figure 7](#sup1)). Obviously, future studies will be required to clarify the possible functional relationships among ADAM10, TSPAN5, Notch and RBPJ-κ.

We also pursued the function of the chromosome one SNP signal across *ERICH3* that included two nsSNPs. These same SNPs were also associated with response in both the ISPC (*P*=0.022, OR=1.3) and STAR*D (*P*=0.041, OR=1.2) studies. The SNPs across *ERICH3* were not cis-eQTLs, but rather, the nsSNPs were associated with the proteasome-mediated degradation of ERICH3 protein. KD and OE of ERICH3 did not alter the expression of genes encoding serotonin pathway enzymes but were associated with significant changes of serotonin concentrations in the cell culture media. The functional mechanism by which ERICH3 influences serotonin concentrations is unclear.

The present study has shown that baseline plasma serotonin concentrations and decreases in plasma serotonin concentrations after 4 and 8 weeks of SSRI therapy were associated with clinical outcomes in our MDD patients ([Table 1](#tbl1)). Furthermore, *TSPAN5* variant and WT *ERICH3* SNP genotypes ([Figure 2](#fig2) and [Supplementary Figure 1](#sup1)) were associated with higher baseline plasma serotonin concentrations and larger decreases in plasma serotonin concentrations after SSRI therapy ([Supplementary Figure 3](#sup1)). The SNPs across ERICH3 included a nsSNP that resulted in ERICH3 proteasome-mediated degradation. The SNPs 5' of *TSPAN5* were eQTLs for that gene in LCLs ([Figure 3a](#fig3)), brain ([Figures 3b and c](#fig3)) and blood—although with tissue-specific differences in directionality. Higher *TSPAN5* expression was associated with the variant SNP genotype in blood, which, based on our functional studies, would suggest higher expression of serotonin pathway genes and elevated serotonin synthesis ([Figure 4](#fig4))—consistent with the higher baseline plasma serotonin concentrations observed in our clinical data—although that hypothesis will require future validation. Finally, the *ERICH3* nsSNP was associated with clinical response in two independent SSRI studies.

In summary, the present series of experiments have demonstrated that metabolomics can be a useful tool to help identify novel biology—especially when it is used to guide and inform subsequent genomic studies. By integrating pharmacometabolomic and pharmacogenomic data related to SSRI treatment response, we identified SNPs that are cis-eQTLs for *TSPAN5—*a gene not previously known to be involved in either SSRI response or the regulation of serotonin-related pathways and nsSNPs in *ERICH3* that altered the quantity of ERICH3 protein. Virtually nothing was known about *ERICH3* prior to the studies reported here. Future experiments will be required to pursue these observations in depressed and non-depressed populations as will additional pharmacometabolomics-informed pharmacogenomic studies to help us move toward the goal of enhanced molecular subclassification of psychiatric disease and its response to drug therapy.

## Acknowledgments

This work was supported, in part, by U19 GM61388 (The Pharmacogenomics Research Network), RO1 GM28157, by R24 GM078233 (The Metabolomics Research Network for Drug Response Phenotype) and by RC2 GM092729 (The Metabolomics Network for Drug Response Phenotype).

The authors declare no conflict of interest.

## Footnotes

## Supplementary Material

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Alonso J, Angermeyer MC, Bernert S, Bruffaerts R, Brugha TS, Bryson H et al. Prevalence of mental disorders in Europe: results from the European Study of the Epidemiology of Mental Disorders (ESEMeD) project. Acta Psychiatr Scand Suppl 2004; 420: 21–27.  [DOI](https://doi.org/10.1111/j.1600-0047.2004.00327.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15128384/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Acta%20Psychiatr%20Scand%20Suppl&title=Prevalence%20of%20mental%20disorders%20in%20Europe:%20results%20from%20the%20European%20Study%20of%20the%20Epidemiology%20of%20Mental%20Disorders%20(ESEMeD)%20project&volume=420&publication_year=2004&pages=21&pmid=15128384&doi=10.1111/j.1600-0047.2004.00327.x&)

2. Bromet E, Andrade LH, Hwang I, Sampson NA, Alonso J, de Girolamo G et al. Cross-national epidemiology of DSM-IV major depressive episode. BMC Med 2011; 9: 90.  [DOI](https://doi.org/10.1186/1741-7015-9-90) | [PMC free article](/articles/PMC3163615/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21791035/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Med&title=Cross-national%20epidemiology%20of%20DSM-IV%20major%20depressive%20episode&volume=9&publication_year=2011&pages=90&pmid=21791035&doi=10.1186/1741-7015-9-90&)

3. Chesney E, Goodwin GM, Fazel S. Risks of all-cause and suicide mortality in mental disorders: a meta-review. World Psychiatry 2014; 13: 153–160.  [DOI](https://doi.org/10.1002/wps.20128) | [PMC free article](/articles/PMC4102288/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24890068/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=World%20Psychiatry&title=Risks%20of%20all-cause%20and%20suicide%20mortality%20in%20mental%20disorders:%20a%20meta-review&volume=13&publication_year=2014&pages=153&pmid=24890068&doi=10.1002/wps.20128&)

4. Morrissette DA, Stahl SM. Modulating the serotonin system in the treatment of major depressive disorder. CNS Spectr 2014; 19: 54–68.  [DOI](https://doi.org/10.1017/S1092852914000613) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25544378/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=CNS%20Spectr&title=Modulating%20the%20serotonin%20system%20in%20the%20treatment%20of%20major%20depressive%20disorder&volume=19&publication_year=2014&pages=54&pmid=25544378&doi=10.1017/S1092852914000613&)

5. Gorman JM, Korotzer A, Su G. Efficacy comparison of escitalopram and citalopram in the treatment of major depressive disorder: pooled analysis of placebo-controlled trials. CNS Spectr 2002; 7(4 Suppl 1): 40–44.  [DOI](https://doi.org/10.1017/s1092852900028595) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15131492/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=CNS%20Spectr&title=Efficacy%20comparison%20of%20escitalopram%20and%20citalopram%20in%20the%20treatment%20of%20major%20depressive%20disorder:%20pooled%20analysis%20of%20placebo-controlled%20trials&volume=7&publication_year=2002&pages=40&pmid=15131492&doi=10.1017/s1092852900028595&)

6. El Mansari M, Guiard BP, Chernoloz O, Ghanbari R, Katz N, Blier P. Relevance of norepinephrine-dopamine interactions in the treatment of major depressive disorder. CNS Neurosci Ther 2010; 16: e1–17.  [DOI](https://doi.org/10.1111/j.1755-5949.2010.00146.x) | [PMC free article](/articles/PMC2904493/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20406250/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=CNS%20Neurosci%20Ther&title=Relevance%20of%20norepinephrine-dopamine%20interactions%20in%20the%20treatment%20of%20major%20depressive%20disorder&volume=16&publication_year=2010&pages=e1&pmid=20406250&doi=10.1111/j.1755-5949.2010.00146.x&)

7. Anderson HD, Pace WD, Libby AM, West DR, Valuck RJ. Rates of 5 common antidepressant side effects among new adult and adolescent cases of depression: a retrospective US claims study. Clin Ther 2012; 34: 113–123.  [DOI](https://doi.org/10.1016/j.clinthera.2011.11.024) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22177545/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Ther&title=Rates%20of%205%20common%20antidepressant%20side%20effects%20among%20new%20adult%20and%20adolescent%20cases%20of%20depression:%20a%20retrospective%20US%20claims%20study&volume=34&publication_year=2012&pages=113&pmid=22177545&doi=10.1016/j.clinthera.2011.11.024&)

8. Smith AJ, Sketris I, Cooke C, Gardner D, Kisely S, Tett SE. A comparison of antidepressant use in Nova Scotia, Canada and Australia. Pharmacoepidemiol Drug Saf 2008; 17: 697–706.  [DOI](https://doi.org/10.1002/pds.1541) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18181227/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacoepidemiol%20Drug%20Saf&title=A%20comparison%20of%20antidepressant%20use%20in%20Nova%20Scotia,%20Canada%20and%20Australia&volume=17&publication_year=2008&pages=697&pmid=18181227&doi=10.1002/pds.1541&)

9. Trivedi MH, Rush AJ, Wisniewski SR, Nierenberg AA, Warden D, Ritz L et al. Evaluation of outcomes with citalopram for depression using measurement-based care in STAR*D: implications for clinical practice. Am J Psychiatry 2006; 163: 28–40.  [DOI](https://doi.org/10.1176/appi.ajp.163.1.28) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16390886/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Psychiatry&title=Evaluation%20of%20outcomes%20with%20citalopram%20for%20depression%20using%20measurement-based%20care%20in%20STAR*D:%20implications%20for%20clinical%20practice&volume=163&publication_year=2006&pages=28&pmid=16390886&doi=10.1176/appi.ajp.163.1.28&)

10. Biernacka JM, Sangkuhl K, Jenkins G, Whaley RM, Barman P, Batzler A et al. The International SSRI Pharmacogenomics Consortium (ISPC): a genome-wide association study of antidepressant treatment response. Transl Psychiatry 2015; 5: e553.  [DOI](https://doi.org/10.1038/tp.2015.47) | [PMC free article](/articles/PMC4462610/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25897834/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Transl%20Psychiatry&title=The%20International%20SSRI%20Pharmacogenomics%20Consortium%20(ISPC):%20a%20genome-wide%20association%20study%20of%20antidepressant%20treatment%20response&volume=5&publication_year=2015&pages=e553&pmid=25897834&doi=10.1038/tp.2015.47&)

11. Ji Y, Biernacka JM, Hebbring S, Chai Y, Jenkins GD, Batzler A et al. Pharmacogenomics of selective serotonin reuptake inhibitor treatment for major depressive disorder: genome-wide associations and functional genomics. Pharmacogenomics J 2013; 13: 456–463.  [DOI](https://doi.org/10.1038/tpj.2012.32) | [PMC free article](/articles/PMC3941038/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22907730/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=Pharmacogenomics%20of%20selective%20serotonin%20reuptake%20inhibitor%20treatment%20for%20major%20depressive%20disorder:%20genome-wide%20associations%20and%20functional%20genomics&volume=13&publication_year=2013&pages=456&pmid=22907730&doi=10.1038/tpj.2012.32&)

12. Sullivan PF, Neale MC, Kendler KS. Genetic epidemiology of major depression: review and meta-analysis. Am J Psychiatry 2000; 157: 1552–1562.  [DOI](https://doi.org/10.1176/appi.ajp.157.10.1552) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11007705/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Psychiatry&title=Genetic%20epidemiology%20of%20major%20depression:%20review%20and%20meta-analysis&volume=157&publication_year=2000&pages=1552&pmid=11007705&doi=10.1176/appi.ajp.157.10.1552&)

13. Bierut LJ, Heath AC, Bucholz KK, Dinwiddie SH, Madden PA, Statham DJ et al. Major depressive disorder in a community-based twin sample: are there different genetic and environmental contributions for men and women? Arch Gen Psychiatry 1999; 56: 557–563.  [DOI](https://doi.org/10.1001/archpsyc.56.6.557) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10359473/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arch%20Gen%20Psychiatry&title=Major%20depressive%20disorder%20in%20a%20community-based%20twin%20sample:%20are%20there%20different%20genetic%20and%20environmental%20contributions%20for%20men%20and%20women&volume=56&publication_year=1999&pages=557&pmid=10359473&doi=10.1001/archpsyc.56.6.557&)

14. McGuffin P, Katz R, Watkins S, Rutherford J. A hospital-based twin register of the heritability of DSM-IV unipolar depression. Arch Gen Psychiatry 1996; 53: 129–136.  [DOI](https://doi.org/10.1001/archpsyc.1996.01830020047006) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8629888/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arch%20Gen%20Psychiatry&title=A%20hospital-based%20twin%20register%20of%20the%20heritability%20of%20DSM-IV%20unipolar%20depression&volume=53&publication_year=1996&pages=129&pmid=8629888&doi=10.1001/archpsyc.1996.01830020047006&)

15. Kendler KS, Neale MC, Kessler RC, Heath AC, Eaves LJ. The lifetime history of major depression in women. Reliability of diagnosis and heritability. Arch Gen Psychiatry 1993; 50: 863–870.  [DOI](https://doi.org/10.1001/archpsyc.1993.01820230054003) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8215812/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arch%20Gen%20Psychiatry&title=The%20lifetime%20history%20of%20major%20depression%20in%20women.%20Reliability%20of%20diagnosis%20and%20heritability&volume=50&publication_year=1993&pages=863&pmid=8215812&doi=10.1001/archpsyc.1993.01820230054003&)

16. Serretti A, Benedetti F, Zanardi R, Smeraldi E. The influence of Serotonin Transporter Promoter Polymorphism (SERTPR) and other polymorphisms of the serotonin pathway on the efficacy of antidepressant treatments. Prog Neuropsychopharmacol Biol Psychiatry 2005; 29: 1074–1084.  [DOI](https://doi.org/10.1016/j.pnpbp.2005.03.013) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15939518/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Prog%20Neuropsychopharmacol%20Biol%20Psychiatry&title=The%20influence%20of%20Serotonin%20Transporter%20Promoter%20Polymorphism%20(SERTPR)%20and%20other%20polymorphisms%20of%20the%20serotonin%20pathway%20on%20the%20efficacy%20of%20antidepressant%20treatments&volume=29&publication_year=2005&pages=1074&pmid=15939518&doi=10.1016/j.pnpbp.2005.03.013&)

17. Fabbri C, Minarini A, Niitsu T, Serretti A. Understanding the pharmacogenetics of selective serotonin reuptake inhibitors. Exp Opin Drug Metab Toxicol 2014; 10: 1093–1118.  [DOI](https://doi.org/10.1517/17425255.2014.928693) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24930681/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Exp%20Opin%20Drug%20Metab%20Toxicol&title=Understanding%20the%20pharmacogenetics%20of%20selective%20serotonin%20reuptake%20inhibitors&volume=10&publication_year=2014&pages=1093&pmid=24930681&doi=10.1517/17425255.2014.928693&)

18. Laje G, Perlis RH, Rush AJ, McMahon FJ. Pharmacogenetics studies in STAR*D: strengths, limitations, and results. Psychiatr Serv 2009; 60: 1446–1457.  [DOI](https://doi.org/10.1176/appi.ps.60.11.1446) | [PMC free article](/articles/PMC3775610/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19880459/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Psychiatr%20Serv&title=Pharmacogenetics%20studies%20in%20STAR*D:%20strengths,%20limitations,%20and%20results&volume=60&publication_year=2009&pages=1446&pmid=19880459&doi=10.1176/appi.ps.60.11.1446&)

19. Myung W, Kim J, Lim SW, Shim S, Won HH, Kim S et al. A genome-wide association study of antidepressant response in Koreans. Transl Psychiatry 2015; 5: e633.  [DOI](https://doi.org/10.1038/tp.2015.127) | [PMC free article](/articles/PMC5068817/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26348319/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Transl%20Psychiatry&title=A%20genome-wide%20association%20study%20of%20antidepressant%20response%20in%20Koreans&volume=5&publication_year=2015&pages=e633&pmid=26348319&doi=10.1038/tp.2015.127&)

20. Niitsu T, Fabbri C, Bentini F, Serretti A. Pharmacogenetics in major depression: a comprehensive meta-analysis. Prog Neuropsychopharmacol Biol Psychiatry 2013; 45: 183–194.  [DOI](https://doi.org/10.1016/j.pnpbp.2013.05.011) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23733030/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Prog%20Neuropsychopharmacol%20Biol%20Psychiatry&title=Pharmacogenetics%20in%20major%20depression:%20a%20comprehensive%20meta-analysis&volume=45&publication_year=2013&pages=183&pmid=23733030&doi=10.1016/j.pnpbp.2013.05.011&)

21. GENDEP Investigators, MARS Investigators, STAR*D Investigators.. Common genetic variation and antidepressant efficacy in major depressive disorder: a meta-analysis of three genome-wide pharmacogenetic studies. Am J Psychiatry 2013; 170: 207–217.  [DOI](https://doi.org/10.1176/appi.ajp.2012.12020237) | [PMC free article](/articles/PMC10416089/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23377640/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Psychiatry&title=Common%20genetic%20variation%20and%20antidepressant%20efficacy%20in%20major%20depressive%20disorder:%20a%20meta-analysis%20of%20three%20genome-wide%20pharmacogenetic%20studies&volume=170&publication_year=2013&pages=207&pmid=23377640&doi=10.1176/appi.ajp.2012.12020237&)

22. Tansey KE, Guipponi M, Perroud N, Bondolfi G, Domenici E, Evans D et al. Genetic predictors of response to serotonergic and noradrenergic antidepressants in major depressive disorder: a genome-wide analysis of individual-level data and a meta-analysis. PLoS Med 2012; 9: e1001326.  [DOI](https://doi.org/10.1371/journal.pmed.1001326) | [PMC free article](/articles/PMC3472989/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23091423/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Med&title=Genetic%20predictors%20of%20response%20to%20serotonergic%20and%20noradrenergic%20antidepressants%20in%20major%20depressive%20disorder:%20a%20genome-wide%20analysis%20of%20individual-level%20data%20and%20a%20meta-analysis&volume=9&publication_year=2012&pages=e1001326&pmid=23091423&doi=10.1371/journal.pmed.1001326&)

23. Uher R, Perroud N, Ng MY, Hauser J, Henigsberg N, Maier W et al. Genome-wide pharmacogenetics of antidepressant response in the GENDEP project. Am J Psychiatry 2010; 167: 555–564.  [DOI](https://doi.org/10.1176/appi.ajp.2009.09070932) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20360315/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Psychiatry&title=Genome-wide%20pharmacogenetics%20of%20antidepressant%20response%20in%20the%20GENDEP%20project&volume=167&publication_year=2010&pages=555&pmid=20360315&doi=10.1176/appi.ajp.2009.09070932&)

24. Garriock HA, Kraft JB, Shyn SI, Peters EJ, Yokoyama JS, Jenkins GD et al. A genomewide association study of citalopram response in major depressive disorder. Biol Psychiatry 2010; 67: 133–138.  [DOI](https://doi.org/10.1016/j.biopsych.2009.08.029) | [PMC free article](/articles/PMC2794921/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19846067/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biol%20Psychiatry&title=A%20genomewide%20association%20study%20of%20citalopram%20response%20in%20major%20depressive%20disorder&volume=67&publication_year=2010&pages=133&pmid=19846067&doi=10.1016/j.biopsych.2009.08.029&)

25. Weizman S, Gonda X, Dome P, Faludi G. Pharmacogenetics of antidepressive drugs: a way towards personalized treatment of major depressive disorder. Neuropsychopharmacol Hung 2012; 14: 87–101.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/22710850/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neuropsychopharmacol%20Hung&title=Pharmacogenetics%20of%20antidepressive%20drugs:%20a%20way%20towards%20personalized%20treatment%20of%20major%20depressive%20disorder&volume=14&publication_year=2012&pages=87&pmid=22710850&)

26. Fabbri C, Serretti A. Pharmacogenetics of major depressive disorder: top genes and pathways toward clinical applications. Curr Psychiatry Rep 2015; 17: 594.  [DOI](https://doi.org/10.1007/s11920-015-0594-9) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25980509/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Psychiatry%20Rep&title=Pharmacogenetics%20of%20major%20depressive%20disorder:%20top%20genes%20and%20pathways%20toward%20clinical%20applications&volume=17&publication_year=2015&pages=594&pmid=25980509&doi=10.1007/s11920-015-0594-9&)

27. Singh AB, Bousman CA, Ng C, Berk M. Antidepressant pharmacogenetics. Curr Opin Psychiatry 2014; 27: 43–51.  [DOI](https://doi.org/10.1097/YCO.0000000000000023) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24270480/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Opin%20Psychiatry&title=Antidepressant%20pharmacogenetics&volume=27&publication_year=2014&pages=43&pmid=24270480&doi=10.1097/YCO.0000000000000023&)

28. Ji Y, Hebbring S, Zhu H, Jenkins GD, Biernacka J, Snyder K et al. Glycine and a glycine dehydrogenase (GLDC) SNP as citalopram/escitalopram response biomarkers in depression: pharmacometabolomics-informed pharmacogenomics. Clin Pharmacol Ther 2011; 89: 97–104.  [DOI](https://doi.org/10.1038/clpt.2010.250) | [PMC free article](/articles/PMC3034442/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21107318/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Glycine%20and%20a%20glycine%20dehydrogenase%20(GLDC)%20SNP%20as%20citalopram/escitalopram%20response%20biomarkers%20in%20depression:%20pharmacometabolomics-informed%20pharmacogenomics&volume=89&publication_year=2011&pages=97&pmid=21107318&doi=10.1038/clpt.2010.250&)

29. Abo R, Hebbring S, Ji Y, Zhu H, Zeng ZB, Batzler A et al. Merging pharmacometabolomics with pharmacogenomics using '1000 Genomes' single-nucleotide polymorphism imputation: selective serotonin reuptake inhibitor response pharmacogenomics. Pharmacogenet Genomics 2012; 22: 247–253.  [DOI](https://doi.org/10.1097/FPC.0b013e32835001c9) | [PMC free article](/articles/PMC3303952/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22322242/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Merging%20pharmacometabolomics%20with%20pharmacogenomics%20using%20'1000%20Genomes'%20single-nucleotide%20polymorphism%20imputation:%20selective%20serotonin%20reuptake%20inhibitor%20response%20pharmacogenomics&volume=22&publication_year=2012&pages=247&pmid=22322242&doi=10.1097/FPC.0b013e32835001c9&)

30. Monteiro MS, Carvalho M, Bastos ML, Guedes de Pinho P. Metabolomics analysis for biomarker discovery: advances and challenges. Curr Med Chem 2013; 20: 257–271.  [DOI](https://doi.org/10.2174/092986713804806621) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23210853/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Med%20Chem&title=Metabolomics%20analysis%20for%20biomarker%20discovery:%20advances%20and%20challenges&volume=20&publication_year=2013&pages=257&pmid=23210853&doi=10.2174/092986713804806621&)

31. Zhu H, Bogdanov MB, Boyle SH, Matson W, Sharma S, Matson S et al. Pharmacometabolomics of response to sertraline and to placebo in major depressive disorder - possible role for methoxyindole pathway. PloS One 2013; 8: e68283.  [DOI](https://doi.org/10.1371/journal.pone.0068283) | [PMC free article](/articles/PMC3714282/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23874572/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PloS%20One&title=Pharmacometabolomics%20of%20response%20to%20sertraline%20and%20to%20placebo%20in%20major%20depressive%20disorder%20-%20possible%20role%20for%20methoxyindole%20pathway&volume=8&publication_year=2013&pages=e68283&pmid=23874572&doi=10.1371/journal.pone.0068283&)

32. Kaddurah-Daouk R, Bogdanov MB, Wikoff WR, Zhu H, Boyle SH, Churchill E et al. Pharmacometabolomic mapping of early biochemical changes induced by sertraline and placebo. Transl Psychiatry 2013; 3: e223.  [DOI](https://doi.org/10.1038/tp.2012.142) | [PMC free article](/articles/PMC3566722/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23340506/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Transl%20Psychiatry&title=Pharmacometabolomic%20mapping%20of%20early%20biochemical%20changes%20induced%20by%20sertraline%20and%20placebo&volume=3&publication_year=2013&pages=e223&pmid=23340506&doi=10.1038/tp.2012.142&)

33. Kaddurah-Daouk R, Weinshilboum R. Metabolomic signatures for drug response phenotypes: Pharmacometabolomics enables precision medicine. Clin Pharmacol Ther 2015; 98: 71–75.  [DOI](https://doi.org/10.1002/cpt.134) | [PMC free article](/articles/PMC4938244/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25871646/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Metabolomic%20signatures%20for%20drug%20response%20phenotypes:%20Pharmacometabolomics%20enables%20precision%20medicine&volume=98&publication_year=2015&pages=71&pmid=25871646&doi=10.1002/cpt.134&)

34. Ellero-Simatos S, Lewis JP, Georgiades A, Yerges-Armstrong LM, Beitelshees AL, Horenstein RB et al. Pharmacometabolomics reveals that serotonin is implicated in aspirin response variability. CPT Pharmacometrics Syst Pharmacol 2014; 3: e125.  [DOI](https://doi.org/10.1038/psp.2014.22) | [PMC free article](/articles/PMC4120016/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25029353/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=CPT%20Pharmacometrics%20Syst%20Pharmacol&title=Pharmacometabolomics%20reveals%20that%20serotonin%20is%20implicated%20in%20aspirin%20response%20variability&volume=3&publication_year=2014&pages=e125&pmid=25029353&doi=10.1038/psp.2014.22&)

35. Yerges-Armstrong LM, Ellero-Simatos S, Georgiades A, Zhu H, Lewis JP, Horenstein RB et al. Purine pathway implicated in mechanism of resistance to aspirin therapy: pharmacometabolomics-informed pharmacogenomics. Clin Pharmacol Thera 2013; 94: 525–532.  [DOI](https://doi.org/10.1038/clpt.2013.119) | [PMC free article](/articles/PMC4001726/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23839601/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Thera&title=Purine%20pathway%20implicated%20in%20mechanism%20of%20resistance%20to%20aspirin%20therapy:%20pharmacometabolomics-informed%20pharmacogenomics&volume=94&publication_year=2013&pages=525&pmid=23839601&doi=10.1038/clpt.2013.119&)

36. Mrazek DA, Biernacka JM, McAlpine DE, Benitez J, Karpyak VM, Williams MD et al. Treatment outcomes of depression: the pharmacogenomic research network antidepressant medication pharmacogenomic study. J Clin Psychopharmacol 2014; 34: 313–317.  [DOI](https://doi.org/10.1097/JCP.0000000000000099) | [PMC free article](/articles/PMC3992481/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24743713/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Psychopharmacol&title=Treatment%20outcomes%20of%20depression:%20the%20pharmacogenomic%20research%20network%20antidepressant%20medication%20pharmacogenomic%20study&volume=34&publication_year=2014&pages=313&pmid=24743713&doi=10.1097/JCP.0000000000000099&)

37. Ji Y, Schaid DJ, Desta Z, Kubo M, Batzler AJ, Snyder K et al. Citalopram and escitalopram plasma drug and metabolite concentrations: genome-wide associations. Br J Clin Pharmacol 2014; 78: 373–383.  [DOI](https://doi.org/10.1111/bcp.12348) | [PMC free article](/articles/PMC4137829/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24528284/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Clin%20Pharmacol&title=Citalopram%20and%20escitalopram%20plasma%20drug%20and%20metabolite%20concentrations:%20genome-wide%20associations&volume=78&publication_year=2014&pages=373&pmid=24528284&doi=10.1111/bcp.12348&)

38. GTEx Consortium. The Genotype-Tissue Expression (GTEx) project. Nat Genet 2013; 45: 580–585.  [DOI](https://doi.org/10.1038/ng.2653) | [PMC free article](/articles/PMC4010069/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23715323/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=The%20Genotype-Tissue%20Expression%20(GTEx)%20project&volume=45&publication_year=2013&pages=580&pmid=23715323&doi=10.1038/ng.2653&)

39. Matson WR, Langlais P, Volicer L, Gamache PH, Bird E, Mark KA. n-Electrode three-dimensional liquid chromatography with electrochemical detection for determination of neurotransmitters. Clin Chem 1984; 30: 1477–1488.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/6147209/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Chem&title=n-Electrode%20three-dimensional%20liquid%20chromatography%20with%20electrochemical%20detection%20for%20determination%20of%20neurotransmitters&volume=30&publication_year=1984&pages=1477&pmid=6147209&)

40. Liu M, Ingle JN, Fridley BL, Buzdar AU, Robson ME, Kubo M et al. TSPYL5 SNPs: association with plasma estradiol concentrations and aromatase expression. Mol Endocrinol 2013; 27: 657–670.  [DOI](https://doi.org/10.1210/me.2012-1397) | [PMC free article](/articles/PMC3607698/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23518928/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Endocrinol&title=TSPYL5%20SNPs:%20association%20with%20plasma%20estradiol%20concentrations%20and%20aromatase%20expression&volume=27&publication_year=2013&pages=657&pmid=23518928&doi=10.1210/me.2012-1397&)

41. Niu N, Qin Y, Fridley BL, Hou J, Kalari KR, Zhu M et al. Radiation pharmacogenomics: A genome-wide association approach to identify radiation response biomarkers using human lymphoblastoid cell lines. Genome Res 2010; 20: 1482–1492.  [DOI](https://doi.org/10.1101/gr.107672.110) | [PMC free article](/articles/PMC2963812/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20923822/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Res&title=Radiation%20pharmacogenomics:%20A%20genome-wide%20association%20approach%20to%20identify%20radiation%20response%20biomarkers%20using%20human%20lymphoblastoid%20cell%20lines&volume=20&publication_year=2010&pages=1482&pmid=20923822&doi=10.1101/gr.107672.110&)

42. Li L, Fridley B, Kalari K, Jenkins G, Batzler A, Safgren S et al. Gemcitabine and cytosine arabinoside cytotoxicity: association with lymphoblastoid cell expression. Cancer Res 2008; 68: 7050–7058.  [DOI](https://doi.org/10.1158/0008-5472.CAN-08-0405) | [PMC free article](/articles/PMC2562356/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18757419/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=Gemcitabine%20and%20cytosine%20arabinoside%20cytotoxicity:%20association%20with%20lymphoblastoid%20cell%20expression&volume=68&publication_year=2008&pages=7050&pmid=18757419&doi=10.1158/0008-5472.CAN-08-0405&)

43. Ingle JN, Liu M, Wickerham DL, Schaid DJ, Wang L, Mushiroda T et al. Selective estrogen receptor modulators and pharmacogenomic variation in ZNF423 regulation of BRCA1 expression: individualized breast cancer prevention. Cancer Discov 2013; 3: 812–825.  [DOI](https://doi.org/10.1158/2159-8290.CD-13-0038) | [PMC free article](/articles/PMC3710533/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23764426/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Discov&title=Selective%20estrogen%20receptor%20modulators%20and%20pharmacogenomic%20variation%20in%20ZNF423%20regulation%20of%20BRCA1%20expression:%20individualized%20breast%20cancer%20prevention&volume=3&publication_year=2013&pages=812&pmid=23764426&doi=10.1158/2159-8290.CD-13-0038&)

44. Liu M, Goss PE, Ingle JN, Kubo M, Furukawa Y, Batzler A et al. Aromatase inhibitor-associated bone fractures: a case-cohort GWAS and functional genomics. Mol Endocrinol 2014; 28: 1740–1751.  [DOI](https://doi.org/10.1210/me.2014-1147) | [PMC free article](/articles/PMC4179631/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25148458/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Endocrinol&title=Aromatase%20inhibitor-associated%20bone%20fractures:%20a%20case-cohort%20GWAS%20and%20functional%20genomics&volume=28&publication_year=2014&pages=1740&pmid=25148458&doi=10.1210/me.2014-1147&)

45. Abecasis GR, Auton A, Brooks LD, DePristo MA, Durbin RM, Handsaker RE et al. An integrated map of genetic variation from 1,092 human genomes. Nature 2012; 491: 56–65.  [DOI](https://doi.org/10.1038/nature11632) | [PMC free article](/articles/PMC3498066/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23128226/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=An%20integrated%20map%20of%20genetic%20variation%20from%201,092%20human%20genomes&volume=491&publication_year=2012&pages=56&pmid=23128226&doi=10.1038/nature11632&)

46. Garcia-Frigola C, Burgaya F, Calbet M, de Lecea L, Soriano E. Mouse Tspan-5, a member of the tetraspanin superfamily, is highly expressed in brain cortical structures. Neuroreport 2000; 11: 3181–3185.  [DOI](https://doi.org/10.1097/00001756-200009280-00027) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11043545/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neuroreport&title=Mouse%20Tspan-5,%20a%20member%20of%20the%20tetraspanin%20superfamily,%20is%20highly%20expressed%20in%20brain%20cortical%20structures&volume=11&publication_year=2000&pages=3181&pmid=11043545&doi=10.1097/00001756-200009280-00027&)

47. Garcia-Frigola C, Burgaya F, de Lecea L, Soriano E. Pattern of expression of the tetraspanin Tspan-5 during brain development in the mouse. Mech Dev 2001; 106: 207–212.  [DOI](https://doi.org/10.1016/s0925-4773(01)00436-1) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11472858/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mech%20Dev&title=Pattern%20of%20expression%20of%20the%20tetraspanin%20Tspan-5%20during%20brain%20development%20in%20the%20mouse&volume=106&publication_year=2001&pages=207&pmid=11472858&doi=10.1016/s0925-4773(01)00436-1&)

48. Juenger H, Holst MI, Duffe K, Jankowski J, Baader SL. Tetraspanin-5 (Tm4sf9) mRNA expression parallels neuronal maturation in the cerebellum of normal and L7En-2 transgenic mice. J Comp Neurol 2005; 483: 318–328.  [DOI](https://doi.org/10.1002/cne.20439) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15682397/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Comp%20Neurol&title=Tetraspanin-5%20(Tm4sf9)%20mRNA%20expression%20parallels%20neuronal%20maturation%20in%20the%20cerebellum%20of%20normal%20and%20L7En-2%20transgenic%20mice&volume=483&publication_year=2005&pages=318&pmid=15682397&doi=10.1002/cne.20439&)

49. Todd SC, Doctor VS, Levy S. Sequences and expression of six new members of the tetraspanin/TM4SF family. Biochim Biophys Acta 1998; 1399: 101–104.  [DOI](https://doi.org/10.1016/s0167-4781(98)00087-6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9714763/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochim%20Biophys%20Acta&title=Sequences%20and%20expression%20of%20six%20new%20members%20of%20the%20tetraspanin/TM4SF%20family&volume=1399&publication_year=1998&pages=101&pmid=9714763&doi=10.1016/s0167-4781(98)00087-6&)

50. Ramasamy A, Trabzuni D, Guelfi S, Varghese V, Smith C, Walker R et al. Genetic variability in the regulation of gene expression in ten regions of the human brain. Nat Neurosci 2014; 17: 1418–1428.  [DOI](https://doi.org/10.1038/nn.3801) | [PMC free article](/articles/PMC4208299/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25174004/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Neurosci&title=Genetic%20variability%20in%20the%20regulation%20of%20gene%20expression%20in%20ten%20regions%20of%20the%20human%20brain&volume=17&publication_year=2014&pages=1418&pmid=25174004&doi=10.1038/nn.3801&)

51. Westra HJ, Peters MJ, Esko T, Yaghootkar H, Schurmann C, Kettunen J et al. Systematic identification of trans eQTLs as putative drivers of known disease associations. Nat Genet 2013; 45: 1238–1243.  [DOI](https://doi.org/10.1038/ng.2756) | [PMC free article](/articles/PMC3991562/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24013639/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Systematic%20identification%20of%20trans%20eQTLs%20as%20putative%20drivers%20of%20known%20disease%20associations&volume=45&publication_year=2013&pages=1238&pmid=24013639&doi=10.1038/ng.2756&)

52. Weinshilboum R, Wang L. Pharmacogenetics: inherited variation in amino acid sequence and altered protein quantity. Clin Pharmacol Ther 2004; 75: 253–258.  [DOI](https://doi.org/10.1016/j.clpt.2003.12.002) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15080131/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Pharmacogenetics:%20inherited%20variation%20in%20amino%20acid%20sequence%20and%20altered%20protein%20quantity&volume=75&publication_year=2004&pages=253&pmid=15080131&doi=10.1016/j.clpt.2003.12.002&)

53. Wang L, Nguyen TV, McLaughlin RW, Sikkink LA, Ramirez-Alvarado M, Weinshilboum RM. Human thiopurine S-methyltransferase pharmacogenetics: variant allozyme misfolding and aggresome formation. Proc Natl Acad Sci USA 2005; 102: 9394–9399.  [DOI](https://doi.org/10.1073/pnas.0502352102) | [PMC free article](/articles/PMC1153717/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15967990/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20USA&title=Human%20thiopurine%20S-methyltransferase%20pharmacogenetics:%20variant%20allozyme%20misfolding%20and%20aggresome%20formation&volume=102&publication_year=2005&pages=9394&pmid=15967990&doi=10.1073/pnas.0502352102&)

54. Li F, Wang L, Burgess RJ, Weinshilboum RM. Thiopurine S-methyltransferase pharmacogenetics: autophagy as a mechanism for variant allozyme degradation. Pharmacogenet Genomics 2008; 18: 1083–1094.  [DOI](https://doi.org/10.1097/FPC.0b013e328313e03f) | [PMC free article](/articles/PMC2583164/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18820593/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Thiopurine%20S-methyltransferase%20pharmacogenetics:%20autophagy%20as%20a%20mechanism%20for%20variant%20allozyme%20degradation&volume=18&publication_year=2008&pages=1083&pmid=18820593&doi=10.1097/FPC.0b013e328313e03f&)

55. Wang L, Sullivan W, Toft D, Weinshilboum R. Thiopurine S-methyltransferase pharmacogenetics: chaperone protein association and allozyme degradation. Pharmacogenetics 2003; 13: 555–564.  [DOI](https://doi.org/10.1097/01.fpc.0000054124.14659.99) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12972954/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=Thiopurine%20S-methyltransferase%20pharmacogenetics:%20chaperone%20protein%20association%20and%20allozyme%20degradation&volume=13&publication_year=2003&pages=555&pmid=12972954&doi=10.1097/01.fpc.0000054124.14659.99&)

56. Dunn CD, Sulis ML, Ferrando AA, Greenwald I. A conserved tetraspanin subfamily promotes Notch signaling in Caenorhabditis elegans and in human cells. Proc Natl Acad Sci USA 2010; 107: 5907–5912.  [DOI](https://doi.org/10.1073/pnas.1001647107) | [PMC free article](/articles/PMC2851858/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20220101/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20USA&title=A%20conserved%20tetraspanin%20subfamily%20promotes%20Notch%20signaling%20in%20Caenorhabditis%20elegans%20and%20in%20human%20cells&volume=107&publication_year=2010&pages=5907&pmid=20220101&doi=10.1073/pnas.1001647107&)

57. Dornier E, Coumailleau F, Ottavi JF, Moretti J, Boucheix C, Mauduit P et al. TspanC8 tetraspanins regulate ADAM10/Kuzbanian trafficking and promote Notch activation in flies and mammals. J Cell Biol 2012; 199: 481–496.  [DOI](https://doi.org/10.1083/jcb.201201133) | [PMC free article](/articles/PMC3483123/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23091066/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Cell%20Biol&title=TspanC8%20tetraspanins%20regulate%20ADAM10/Kuzbanian%20trafficking%20and%20promote%20Notch%20activation%20in%20flies%20and%20mammals&volume=199&publication_year=2012&pages=481&pmid=23091066&doi=10.1083/jcb.201201133&)

58. Haining EJ, Yang J, Bailey RL, Khan K, Collier R, Tsai S et al. The TspanC8 subgroup of tetraspanins interacts with A disintegrin and metalloprotease 10 (ADAM10) and regulates its maturation and cell surface expression. J Biol Chem 2012; 287: 39753–39765.  [DOI](https://doi.org/10.1074/jbc.M112.416503) | [PMC free article](/articles/PMC3501075/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23035126/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=The%20TspanC8%20subgroup%20of%20tetraspanins%20interacts%20with%20A%20disintegrin%20and%20metalloprotease%2010%20(ADAM10)%20and%20regulates%20its%20maturation%20and%20cell%20surface%20expression&volume=287&publication_year=2012&pages=39753&pmid=23035126&doi=10.1074/jbc.M112.416503&)

59. Iwai K, Ishii M, Ohshima S, Miyatake K, Saeki Y. Expression and function of transmembrane-4 superfamily (tetraspanin) proteins in osteoclasts: reciprocal roles of Tspan-5 and NET-6 during osteoclastogenesis. Allergol Int 2007; 56: 457–463.  [DOI](https://doi.org/10.2332/allergolint.O-07-488) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17965585/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Allergol%20Int&title=Expression%20and%20function%20of%20transmembrane-4%20superfamily%20(tetraspanin)%20proteins%20in%20osteoclasts:%20reciprocal%20roles%20of%20Tspan-5%20and%20NET-6%20during%20osteoclastogenesis&volume=56&publication_year=2007&pages=457&pmid=17965585&doi=10.2332/allergolint.O-07-488&)

60. Zhou J, Fujiwara T, Ye S, Li X, Zhao H. Downregulation of Notch modulators, tetraspanin 5 and 10, inhibits osteoclastogenesis in vitro. Calcif Tissue Int 2014; 95: 209–217.  [DOI](https://doi.org/10.1007/s00223-014-9883-2) | [PMC free article](/articles/PMC4139439/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24935633/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Calcif%20Tissue%20Int&title=Downregulation%20of%20Notch%20modulators,%20tetraspanin%205%20and%2010,%20inhibits%20osteoclastogenesis%20in%20vitro&volume=95&publication_year=2014&pages=209&pmid=24935633&doi=10.1007/s00223-014-9883-2&)

61. Shi M, Hu ZL, Zheng MH, Song NN, Huang Y, Zhao G et al. Notch-Rbpj signaling is required for the development of noradrenergic neurons in the mouse locus coeruleus. J Cell Sci 2012; 125(Pt 18): 4320–4332.  [DOI](https://doi.org/10.1242/jcs.102152) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22718343/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Cell%20Sci&title=Notch-Rbpj%20signaling%20is%20required%20for%20the%20development%20of%20noradrenergic%20neurons%20in%20the%20mouse%20locus%20coeruleus&volume=125&publication_year=2012&pages=4320&pmid=22718343&doi=10.1242/jcs.102152&)

62. Alvarez JC, Gluck N, Fallet A, Gregoire A, Chevalier JF, Advenier C et al. Plasma serotonin level after 1day of fluoxetine treatment: a biological predictor for antidepressant response? Psychopharmacology (Berl) 1999; 143: 97–101.  [DOI](https://doi.org/10.1007/s002130050924) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10227085/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Psychopharmacology%20(Berl)&title=Plasma%20serotonin%20level%20after%201day%20of%20fluoxetine%20treatment:%20a%20biological%20predictor%20for%20antidepressant%20response&volume=143&publication_year=1999&pages=97&pmid=10227085&doi=10.1007/s002130050924&)

63. Moreno J, Campos MG, Lara C, Lopez G, Pavon L, Hernandez ME et al. Tryptophan and serotonin in blood and platelets of depressed patients: Effect of an antidepressant treatment. Salud Mental 2006; 29: 4.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Salud%20Mental&title=Tryptophan%20and%20serotonin%20in%20blood%20and%20platelets%20of%20depressed%20patients:%20Effect%20of%20an%20antidepressant%20treatment&volume=29&publication_year=2006&pages=4&)

64. Urbina M, Pineda S, Pinango L, Carreira I, Lima L. [3H]Paroxetine binding to human peripheral lymphocyte membranes of patients with major depression before and after treatment with fluoxetine. Int J Immunopharmacol 1999; 21: 631–646.  [DOI](https://doi.org/10.1016/s0192-0561(99)00035-1) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12609459/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Immunopharmacol&title=%5B3H%5DParoxetine%20binding%20to%20human%20peripheral%20lymphocyte%20membranes%20of%20patients%20with%20major%20depression%20before%20and%20after%20treatment%20with%20fluoxetine&volume=21&publication_year=1999&pages=631&pmid=12609459&doi=10.1016/s0192-0561(99)00035-1&)

65. Tyrer SP, Marshall EF, Griffiths HW. The relationship between response to fluoxetine, plasma drug levels, imipramine binding to platelet membranes and whole-blood 5-HT. Prog Neuropsychopharmacol Biol Psychiatry 1990; 14: 797–805.  [DOI](https://doi.org/10.1016/0278-5846(90)90051-h) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/1963498/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Prog%20Neuropsychopharmacol%20Biol%20Psychiatry&title=The%20relationship%20between%20response%20to%20fluoxetine,%20plasma%20drug%20levels,%20imipramine%20binding%20to%20platelet%20membranes%20and%20whole-blood%205-HT&volume=14&publication_year=1990&pages=797&pmid=1963498&doi=10.1016/0278-5846(90)90051-h&)

66. Hernandez ME, Mendieta D, Martinez-Fong D, Loria F, Moreno J, Estrada I et al. Variations in circulating cytokine levels during 52 week course of treatment with SSRI for major depressive disorder. Eur Neuropsychopharmacol 2008; 18: 917–924.  [DOI](https://doi.org/10.1016/j.euroneuro.2008.08.001) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18805677/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20Neuropsychopharmacol&title=Variations%20in%20circulating%20cytokine%20levels%20during%2052%20week%20course%20of%20treatment%20with%20SSRI%20for%20major%20depressive%20disorder&volume=18&publication_year=2008&pages=917&pmid=18805677&doi=10.1016/j.euroneuro.2008.08.001&)

67. Blardi P, De Lalla A, Leo A, Auteri A, Iapichino S, Di Muro A et al. Serotonin and fluoxetine levels in plasma and platelets after fluoxetine treatment in depressive patients. J Clin Psychopharmacol 2002; 22: 131–136.  [DOI](https://doi.org/10.1097/00004714-200204000-00005) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11910257/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Psychopharmacol&title=Serotonin%20and%20fluoxetine%20levels%20in%20plasma%20and%20platelets%20after%20fluoxetine%20treatment%20in%20depressive%20patients&volume=22&publication_year=2002&pages=131&pmid=11910257&doi=10.1097/00004714-200204000-00005&)

68. Kotzailias N, Marker M, Jilma B. Early effects of paroxetine on serotonin storage, plasma levels, and urinary excretion: a randomized, double-blind, placebo-controlled trial. J Clin Psychopharmacol 2004; 24: 536–539.  [DOI](https://doi.org/10.1097/01.jcp.0000138765.08235.46) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15349011/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Psychopharmacol&title=Early%20effects%20of%20paroxetine%20on%20serotonin%20storage,%20plasma%20levels,%20and%20urinary%20excretion:%20a%20randomized,%20double-blind,%20placebo-controlled%20trial&volume=24&publication_year=2004&pages=536&pmid=15349011&doi=10.1097/01.jcp.0000138765.08235.46&)

69. Blardi P, de Lalla A, Urso R, Auteri A, Dell'Erba A, Bossini L et al. Activity of citalopram on adenosine and serotonin circulating levels in depressed patients. J Clin Psychopharmacol 2005; 25: 262–266.  [DOI](https://doi.org/10.1097/01.jcp.0000161500.58266.90) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15876907/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Psychopharmacol&title=Activity%20of%20citalopram%20on%20adenosine%20and%20serotonin%20circulating%20levels%20in%20depressed%20patients&volume=25&publication_year=2005&pages=262&pmid=15876907&doi=10.1097/01.jcp.0000161500.58266.90&)

70. Maecker HT, Todd SC, Levy S. The tetraspanin superfamily: molecular facilitators. FASEB J 1997; 11: 428–442.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/9194523/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=FASEB%20J&title=The%20tetraspanin%20superfamily:%20molecular%20facilitators&volume=11&publication_year=1997&pages=428&pmid=9194523&)

71. Kurita-Taniguchi M, Hazeki K, Murabayashi N, Fukui A, Tsuji S, Matsumoto M et al. Molecular assembly of CD46 with CD9, alpha3-beta1 integrin and protein tyrosine phosphatase SHP-1 in human macrophages through differentiation by GM-CSF. Mol Immunol 2002; 38: 689–700.  [DOI](https://doi.org/10.1016/s0161-5890(01)00100-6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11858824/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Immunol&title=Molecular%20assembly%20of%20CD46%20with%20CD9,%20alpha3-beta1%20integrin%20and%20protein%20tyrosine%20phosphatase%20SHP-1%20in%20human%20macrophages%20through%20differentiation%20by%20GM-CSF&volume=38&publication_year=2002&pages=689&pmid=11858824&doi=10.1016/s0161-5890(01)00100-6&)

72. Claas C, Stipp CS, Hemler ME. Evaluation of prototype transmembrane 4 superfamily protein complexes and their relation to lipid rafts. J Biol Chem 2001; 276: 7974–7984.  [DOI](https://doi.org/10.1074/jbc.M008650200) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11113129/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=Evaluation%20of%20prototype%20transmembrane%204%20superfamily%20protein%20complexes%20and%20their%20relation%20to%20lipid%20rafts&volume=276&publication_year=2001&pages=7974&pmid=11113129&doi=10.1074/jbc.M008650200&)

73. Kovall RA, Hendrickson WA. Crystal structure of the nuclear effector of Notch signaling, CSL, bound to DNA. EMBO J 2004; 23: 3441–3451.  [DOI](https://doi.org/10.1038/sj.emboj.7600349) | [PMC free article](/articles/PMC516623/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15297877/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=EMBO%20J&title=Crystal%20structure%20of%20the%20nuclear%20effector%20of%20Notch%20signaling,%20CSL,%20bound%20to%20DNA&volume=23&publication_year=2004&pages=3441&pmid=15297877&doi=10.1038/sj.emboj.7600349&)

74. Borggrefe T, Oswald F. The Notch signaling pathway: transcriptional regulation at Notch target genes. Cell Mol Life Sci 2009; 66: 1631–1646.  [DOI](https://doi.org/10.1007/s00018-009-8668-7) | [PMC free article](/articles/PMC11115614/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19165418/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell%20Mol%20Life%20Sci&title=The%20Notch%20signaling%20pathway:%20transcriptional%20regulation%20at%20Notch%20target%20genes&volume=66&publication_year=2009&pages=1631&pmid=19165418&doi=10.1007/s00018-009-8668-7&)
