# Discovering novel SNPs that are correlated with patient outcome in a Singaporean cancer patient cohort treated with gemcitabine-based chemotherapy

## Metadata
**Authors:** Vachiranee Limviphuvadh, Chee Seng Tan, Fumikazu Konishi, Piroon Jenjaroenpun, Joy Shengnan Xiang, Yuliya Kremenska, Yar Soe Mu, Nicholas Syn, Soo Chin Lee, Ross A Soo, Frank Eisenhaber, Sebastian Maurer-Stroh, Wei Peng Yong
**Journal:** BMC Cancer
**Date:** 2018 May 11
**DOI:** [10.1186/s12885-018-4471-x](https://doi.org/10.1186/s12885-018-4471-x)
**PMID:** 29751792
**PMCID:** PMC5948914
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5948914/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC5948914/pdf/12885_2018_Article_4471.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC5948914/pdf/12885_2018_Article_4471.pdf)

## Abstract

**Background:** 
Single Nucleotide Polymorphisms (SNPs) can influence patient outcome such as drug response and toxicity after drug intervention. The purpose of this study is to develop a systematic pathway approach to accurately and efficiently predict novel non-synonymous SNPs (nsSNPs) that could be causative to gemcitabine-based chemotherapy treatment outcome in Singaporean non-small cell lung cancer (NSCLC) patients.

**Methods:** 
Using a pathway approach that incorporates comprehensive protein-protein interaction data to systematically extend the gemcitabine pharmacologic pathway, we identified 77 related nsSNPs, common in the Singaporean population. After that, we used five computational criteria to prioritize the SNPs based on their importance for protein function. We specifically selected and screened six candidate SNPs in a patient cohort with NSCLC treated with gemcitabine-based chemotherapy.

**Result:** 
We performed survival analysis followed by hematologic toxicity analyses and found that three of six candidate SNPs are significantly correlated with the patient outcome (P < 0.05) i.e. ABCG2 Q141K (rs2231142), SLC29A3 S158F (rs780668) and POLR2A N764K (rs2228130).

**Conclusions:** 
Our computational SNP candidate enrichment workflow approach was able to identify several high confidence biomarkers predictive for personalized drug treatment outcome while providing a rationale for a molecular mechanism of the SNP effect.

**Trial registration:** 
NCT00695994. Registered 10 June, 2008 ‘retrospectively registered’.

**Electronic supplementary material:** 
The online version of this article (10.1186/s12885-018-4471-x) contains supplementary material, which is available to authorized users.

Keywords: Gemcitabine, NSCLC, Pharmacogenetics, SNPs, Patient outcome

### Background

Single Nucleotide Polymorphisms (SNPs) can influence patient outcome such as drug response and toxicity after drug intervention. The purpose of this study is to develop a systematic pathway approach to accurately and efficiently predict novel non-synonymous SNPs (nsSNPs) that could be causative to gemcitabine-based chemotherapy treatment outcome in Singaporean non-small cell lung cancer (NSCLC) patients.

### Methods

Using a pathway approach that incorporates comprehensive protein-protein interaction data to systematically extend the gemcitabine pharmacologic pathway, we identified 77 related nsSNPs, common in the Singaporean population. After that, we used five computational criteria to prioritize the SNPs based on their importance for protein function. We specifically selected and screened six candidate SNPs in a patient cohort with NSCLC treated with gemcitabine-based chemotherapy.

### Result

We performed survival analysis followed by hematologic toxicity analyses and found that three of six candidate SNPs are significantly correlated with the patient outcome (*P* < 0.05) i.e. ABCG2 Q141K (rs2231142), SLC29A3 S158F (rs780668) and POLR2A N764K (rs2228130).

### Conclusions

Our computational SNP candidate enrichment workflow approach was able to identify several high confidence biomarkers predictive for personalized drug treatment outcome while providing a rationale for a molecular mechanism of the SNP effect.

### Trial registration

[NCT00695994](https://clinicaltrials.gov/ct2/show/NCT00695994). Registered 10 June, 2008 ‘retrospectively registered’.

### Electronic supplementary material

The online version of this article (10.1186/s12885-018-4471-x) contains supplementary material, which is available to authorized users.

## Background

Gemcitabine (2′-2′ difluorodeoxycytidine) is a deoxycytidine analogue with antitumor activity against a variety of solid tumors such as non-small cell lung cancer (NSCLC), breast cancer [[1](#CR1)] and pancreatic cancer [[2](#CR2)]. Gemcitabine requires phosphorylation to mono-, di-, and triphosphates (dFdCTP) to be active. This mechanism results in a unique pattern of self-potentiation of the drug and when this drug is incorporated into the DNA during replication, it causes chain termination. Gemcitabine also has multiple intracellular targets. Up- or downregulation of these targets may confer resistance to this drug.

Wider availability and lower costs of genome and expression profile sequencing made application of those techniques in clinical practice feasible; thus, the scientific question of how patient-specific mutations and chromosomal aberrations influence personal clinical outcomes via biomolecular mechanisms has become acute [[3](#CR3), [4](#CR4)]. For example, pharmacogenetics studies published in the last decades have provided evidence that Single Nucleotide Polymorphisms (SNPs) can causally influence patient outcome such as drug response and toxicity after drug intervention [[5](#CR5)]. Most SNPs associated with patient outcome have been found in genes involved in the drug pharmacology i.e. affecting drug transport, metabolism and/or activity with drugs. Soo et al. tested 26 SNPs from nine genes that are already known to be directly associated with gemcitabine transport, metabolism and activity [[6](#CR6)]. They found several SNPs that were associated with patient outcome in Singaporean NSCLC patients treated with gemcitabine [[6](#CR6)]. However, a systematic approach to investigate the relationship between gene variants and patient outcome is still lacking. Therefore, the purpose of this study is to develop a systematic pathway approach to accurately and efficiently predict novel non-synonymous SNPs (nsSNPs) that could be causative to gemcitabine-based chemotherapy treatment outcome in Singaporean NSCLC patients. After detailed SNP analysis, we prioritized the SNPs based on their importance in protein function and molecular mechanism. From the top-ranking SNPs, we specifically selected six final candidate SNPs for clinical validation. We genotyped these six SNPs in a Singaporean patient cohort and have found that three out of the six SNPs correlated with patient outcome.

## Methods

### Pyrimidine metabolism as a starting pathway to find more genes in the gemcitabine pharmacologic pathway

Pyrimidine metabolism is known to be critical in the pharmacologic pathway of gemcitabine, which is a pyrimidine analogue. Therefore, we used the pyrimidine metabolism pathway in KEGG (hsa00240; [http://www.genome.jp/kegg-bin/show_pathway?org_name=hsa&mapno=00240&mapscale=&show_fdescription=show](http://www.genome.jp/kegg-bin/show_pathway?org_name=hsa&mapno=00240&mapscale=&show_fdescription=show)) [[7](#CR7)] which contains 100 genes as a starting point. In addition, we also did extensive literature search to find more genes that are directly associated with gemcitabine transport [[6](#CR6), [8](#CR8), [9](#CR9)]. Apart from literature review, information from PharmGKB ([http://www.pharmgkb.org/](http://www.pharmgkb.org/)) has also been referenced. As a result, six membrane transporters implicated in the uptake of gemcitabine i.e. SLC28A1 (Entrez GeneID:9154), SLC28A2 (Entrez GeneID:9153), SLC28A3 (Entrez GeneID:64078), SLC29A1 (Entrez GeneID:2030), SLC29A2 (Entrez GeneID:3177), SLC29A3 (Entrez GeneID:55315) and three transporters implicated in the efflux of gemcitabine i.e. ABCC5 (Entrez GeneID:10057), ABCC10 (Entrez GeneID:89845) and ABCG2 (Entrez GeneID:9429) were added to the pathway. In total, 109 genes (the 100 genes in pyrimidine metabolism and the 9 membrane transporter genes) were used as starting proteins to find more interaction partners by using our in-house comprehensive protein-protein interaction (PPI) data.

### Adding more potentially related proteins to the pathway using comprehensive PPI data

We used our in-house comprehensive PPI data to find additional proteins that could be related to the gemcitabine pharmacologic pathway. Comprehensive PPI data was consolidated by integrating experimentally-validated PPIs from nine databases, i.e. BIND [[10](#CR10)], BioGRID [[11](#CR11)], IntAct [[12](#CR12)], DIP [[13](#CR13)], MINT [[14](#CR14)], MPact [[15](#CR15)], HPRD [[16](#CR16)], GNP ([http://genomenetwork.nig.ac.jp/index_e.html](http://genomenetwork.nig.ac.jp/index_e.html)) and MPPI [[17](#CR17)], to provide unique PPIs together with the accumulation of evidence such as experimental type and PubMed IDs. The method we used to integrate multiple databases is provided in detail in this website i.e. [http://ipid.bii.a-star.edu.sg/annie/home.do#ui-tabs-1](http://ipid.bii.a-star.edu.sg/annie/home.do#ui-tabs-1). The final interaction set contains 1,148,484 unique PPIs including 227,731 human PPIs. We used only human PPIs in this study. We extended the pyrimidine metabolism in KEGG (hsa00240) using the conservative requirement that the new protein must have been reported to interact with at least two out of the 100 proteins that are already in the pathway. After that, we collected nsSNPs from NCBI/dbSNPs build 136 [[18](#CR18)] that are linked to these human genes using the NCBI E-utilities tool [[19](#CR19)] with search terms “missense”, “nonsense” or “frameshift”. Information from databases “ensembl_mart_66” and “homo_sapiens_variation_66_37” [[20](#CR20)] was then used to annotate each of the nsSNPs retrieved from E-utilities i.e. Ensembl’s genotype, Ensembl’s transcription ID, Ensembl’s consequence type, NCBI’s consequence type, HGVS genomic, HGVS coding, HGVS protein, PolyPhen-2 and SIFT prediction for reference. A script was written in python and was run on 1st March 2012.

### Finding common SNPs in the Singaporean population

We used allele frequency information from the Singapore Genome Variation Project (SGVP) [[21](#CR21)] to find common SNPs in the Singaporean population among the retrieved nsSNPs. The SGVP provides a publicly available resource of 1.6 million SNPs genotyped in 268 individuals from the Chinese, Malay, and Indian ethnicities in the Singaporean population. In this study, a common SNP is defined as one with a minor allele frequency (MAF) of ≥5% in at least one out of three ethnic groups i.e. Chinese, Malays or Indians.

### Five criteria to filter candidate SNPs

After common SNPs are selected from the retrieved SNPs, five criteria were used to narrow down the common SNPs to select only those that are likely to affect protein function i.e.

### Table 1.

| Characteristics at diagnosis | NSCLC patients (n = 92)a |
| --- | --- |
| Ethnicity |   |
| Chinese | 80 |
| Malay | 9 |
| Indian | 0 |
| Others | 2 |
| No data | 1 |
| Gender |   |
| Male | 67 |
| Female | 24 |
| No data | 1 |
| Stage of Cancer |   |
| Stage III | 14 |
| Stage IV | 77 |
| No data | 1 |
| Performance Status (ECOG) |   |
| 0 | 58 |
| 1 | 33 |
| No data | 1 |

Table 1 Caption: Characteristics of patients who were treated with gemcitabine-based chemotherapy

Finally, after consideration of the five criteria in each of the 77 SNPs, we selected only the top-ranking candidate SNPs for genotyping in the Singaporean patient cohort with known clinical trial data.

### Study population

The Singapore National Healthcare Group Domain Specific Review Board reviewed and approved the study. All the patients provided written informed consent before study entry. The study was conducted in accordance to Good Clinical Practice guidelines. A total of 92 non-small cell lung cancer (NSCLC) patients were recruited for the study and were analysed. All the patients received their treatments in the Department of Haematology-Oncology at National University Hospital of Singapore. Patients with not more than two lines of prior systemic chemotherapy were recruited to receive gemcitabine (750-1000 mg/m^2^ on day 1 and day 8) and carboplatin (AUC 5 mg/ml on day 1) every 3 weeks. Radiographic assessments were done to evaluate tumor response every two cycles according to RECIST criteria. Safety assessments were performed at every cycle including weekly full blood counts to monitor haematological toxicities. Demographic profiles of the patients are summarized in Table [1](#Tab1). We could not retrieve any data from one NSCLC patient and there is another stage IV NSCLC Chinese, male with ECOG = 0 patient who had no survival data as well. So in total, 90 NSCLC patients were available for survival and toxicity analysis.

### Blood collection and genomic DNA extraction

A total of 8 ml peripheral blood was obtained from each patient. The blood was drawn into heparinized vacutainer tubes (Becton Dickinson) and mononuclear cells isolated by Ficoll-Hypaque density gradient centrifugation according to manufacturer’s instructions (GE Healthcare, Chalfont St Giles, United Kingdom). The DNA in turn was extracted from the mononuclear cells using the Puregene DNA purification kit (Gentra Systems, Minneapolis, MN).

### PCR (polymerase chain reaction) and pyrosequencing

First, PCR products were immobilized on streptavidin-coated beads and denatured to produce single-stranded products. Pyrosequencing was performed using the PyroMark Gold Q24 reagent and the PyroMark Q24 system (Qiagen), according to the manufacturer’s protocol. Primers for pyrosequencing were designed with the PyroMark Assay Design Software 2.0. Primers, including biotin-labelled and sequencing primers are represented in Additional file [1](#MOESM1): Table S1. Sequencing analysis was performed using PyroMark Q24 version 2.0.6 software in the allele quantification analysis (QA) mode.

### Statistical analysis to find correlation between the candidate SNPs and patient outcome

Kaplan-Meier methods and log-rank test were used to analyse results for overall survival and progression-free survival in the NSCLC patient cohort. Grade 3 or 4 haematological toxicities and its association with gene variants were analysed using Chi-squared test. All statistical analyses were two-sided and the SPSS software version 16.0 was used. *P* value of less than 0.05 were considered to indicate nominal statistical significance. Predictor variables – including gender, age, stage and ECOG), and the 6 SNPs – were initially correlated with categorical outcomes (grade 3/4 neutropenia and thrombocytopenia) using the chi-squared test, and with time-to-event outcomes (overall and progression free survival) using the log-rank test in univariate fashion. Next, clinical variables and SNPs which were found to be significant in the univariate analyses were included in multivariate Cox or logistic regression to obtain adjusted *p* values and effect sizes.

## Results

### 5046 nsSNPs were found to be linked to the 178 genes in the gemcitabine pharmacologic pathway

The overall workflow and result in each step are described in Fig. [1](#Fig1). We used 100 proteins in the human pyrimidine metabolism metabolic pathway (KEGG:hsa00240) as a starting point and then used our in-house comprehensive PPI data which comprise of unique 227,731 human PPIs integrated from nine public databases (detail in Method section) to extend the pathway by using a conservative requirement that the new protein needs to connect to at least two other out of the 100 proteins that are already in the pyrimidine pathway. By using this criterion, we found an additional 69 proteins from the comprehensive PPI data that can be connected to the pathway. Therefore, 169 genes (100 genes in the pyrimidine metabolism and an additional 69 new genes) together with the 9 membrane transporters from literature review were used to find nsSNPs that are linked to these genes. By using NCBI E-utilities, 5046 nsSNPs were found to be linked to all 178 genes from dbSNPs. 2540 of these nsSNPs (50.34%) came from the newly added proteins. Next, using allele frequency data from SGVP [[21](#CR21)], we found that only 77 (in 54 genes) of the retrieved nsSNPs have MAF with more than or equal to 5% in at least one ethnicity in the Singaporean population. We called these 77 nsSNPs as common SNPs in this study. These common SNPs contain 73 missense, 3 nonsense and 1 frameshift mutations. Detailed information of all common nsSNPs are described in Additional file [1](#MOESM1): Table S2. Among the 77 common SNPs, eight were found to be previously tested in the Singaporean patient cohort in NSCLC [[6](#CR6)] (please refer “*” after rsIDs in Additional file [1](#MOESM1): Table S2). Out of the eight, three of them were proven to be associated with patient outcome i.e. SLC28A1 D521N (rs2242046), SLC28A2 P22L (rs11854484) and SLC28A2 S75R (rs1060896) although these SNPs passed only one or none in our criteria (Additional file [1](#MOESM1): Table S2). Later, POLA2 G583R (rs487989) which is one of the eight SNPs and passed two of our criteria was proven to be strongly associated with mortality rate and survival time among Singaporean NSCLC patients treated with gemcitabine [[31](#CR31)].

### Fig. 1.

![Fig. 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ca01/5948914/a1de6b352b04/12885_2018_4471_Fig1_HTML.jpg)

Overall workflow and summary of results in each step. The pyrimidine metabolism (KEGG PATHWAY: hsa00240, 100 genes) has been chosen as a starting point and then using comprehensive PPI to extend the pathway to add more proteins that could be potentially related to the pathway in which 69 new proteins can be added. We also added 9 membrane transporters that have been known to be associated with the gemcitabine pharmacologic pathway. 5046 nsSNPs are found to be linked to the 178 genes (100 together with the new 69 and the 9 transporters’ genes). 77 of them are found to be common in Singaporean population. After that, five criteria have been used to prioritize the common SNPs. We did detailed SNP analysis for 15 common nsSNPs that passed at least 3 out of 5 criteria and some borderline SNPs. Finally, after thorough literature review, we selected six SNPs to be genotyped in the NSCLC Singaporean patient cohort. PPI: Protein-protein interaction, SGVP: Singaporean Genome Variation Project

### 15 out of 77 common nsSNPs have significant results in three out of five criteria

We used five criteria (as described in detail in the Method section) to narrow down the common SNPs to select only those that are likely to affect protein function (Fig. [1](#Fig1)). The first criterion was to select SNPs that have higher MAF in the Chinese population since the majority of our patients are Singaporean Chinese. 23 out of the 77 nsSNPs were found to match this criterion. The second and third criteria were based on prediction results from PolyPhen-2 and SIFT, respectively, both of which are evolutionary sequence conservation-based approaches. We used the batch-query tool of PolyPhen-2 to parse pre-calculated results of all common nsSNPs. For those SNPs that could not fetch results from the batch-query tool, we used RefSeq amino acid sequence ID of each gene as input to retrieve the result from the PolyPhen-2 website. From the PolyPhen-2 result (i.e. the second criteria), there were 17 common SNPs predicted to be either “probably” or “possibly damaging” (Additional file [1](#MOESM1): Table S2). For the SIFT analysis (i.e. the third criteria), we used the “SIFT dbSNP batch tool” to retrieve prediction results for all common nsSNPs and found that only 9 of them were predicted as “Deleterious” with the SIFT score equal to or less than 0.05 (Additional file [1](#MOESM1): Table S2). The fourth criterion is to check if the SNPs lie on any functional domain. To do this, we retrieved HMMER results against Pfam and SMART using “Prim-Seq-An w/Pfam” analysis on ANNOTATOR and found that 35 out of the 77 SNPs are located in a functional domain of their proteins. The final criterion was to investigate whether the SNP affects protein structural stability using FoldX. Only 27 common SNPs were in a region with a known structure or high similarity to a known structure which allows us to do homology modeling and FoldX analysis. 17 of them returned significant results from FoldX. Results of the five criteria of all 77 common nsSNPs are described in Additional file [1](#MOESM1): Table S2.

Finally, we were able to narrow down the 77 common nsSNPs to 15 which have significant results in at least three out of the five criteria (Additional file [1](#MOESM1): Table S2). We also considered some SNPs that retrieved border line results. Lastly, literature review was performed to understand the functional role of the genes associated with the SNPs to further select SNPs that are most likely to affect the gemcitabine pathway. Finally, we identified the following six SNPs, that is, ABCG2 Q141K (c.421C > A, rs2231142), SLC29A3 S158F (c.473C > T, rs780668), HELB T980I (C > T. rs1168312), NT5C2 D549E (c.1647C > T, rs3740387), POLR2A N764K (c.2292C > T, rs2228130) and CTDP1 T221M (c.662C > T, rs2279103) as final candidate SNPs based on the importance of the SNPs for protein function and drug-related molecular mechanisms (Fig. [2](#Fig2)). Five out of the six final candidate SNPs passed three out of five criteria and only CTDP1 T221M is selected based on a borderline result of the computational selection criteria because it seemed plausible from our literature study. These SNPs are then genotyped in a Singaporean NSCLC patient cohort with known patient outcome for gemcitabine-based therapy.

### Fig. 2.

![Fig. 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ca01/5948914/e5a6da566067/12885_2018_4471_Fig2_HTML.jpg)

Schematic diagram of gemcitabine pharmacologic pathway. Key genes that are directly involved in the gemcitabine pharmacologic pathway are shown. Genes in blue have been studied or tested with NSCLC patient samples previously in other publications. Other genes are found from our pathway-based approach. Nine membrane transporters that are included in this study are also shown in this diagram i.e. ABCC10, ABCC5, ABCG2, SLC28A1, SLC28A2, SLC28A3, SLC29A1, SLC29A2 and SLC29A3. The six SNPs which belong to six genes (in red box) were selected as final candidate SNPs in our study

### Genotyping of the six final candidate SNPs

A total of 90 NSCLC patients that have survival data available were genotyped for the six final candidate SNPs. Genotype information of 90 samples is shown in Additional file [1](#MOESM1): Table S3. Genotype of HELB T980I could not be retrieved from 2 out of 90 patients. Therefore, we used 88 NSCLC patient data to perform survival and toxicity analyses in the next step. We could not find any patient who has the TT genotype of POLR2A N764K.

### ABCG2 Q141K and SLC29A3 S158F are associated with increased survival in NSCLC

Kaplan-Meier analysis was performed to determine any correlation of the six final candidate SNPs with overall survival (OS) and progression free survival (PFS) in the NSCLC patient cohort (*n* = 88). ABCG2 Q141K (c.421 C > A, rs2231142) was found to be associated with increased median PFS. Patients with CA/AA genotype were shown to have longer PFS compared to CC genotype i.e. 9.12 months [95% CI 1.83-16.4 months] vs 5.51 months [95% CI 4.31-6.71 months] respectively, HR 0.51 (95% CI 0.31-0.83), adjusted *P =* 0.007 (Table [2](#Tab2)).

### Table 2.

|   | Factors/Genotype | Number/ | Progression Free Survival (PFS) |  |  |  | Number/ | Overall Survival (OS) |  |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| Median PFS (months) | Univariate analysis |  | Multivariate analysis |  | Median OS (months) | Univariate analysis |  | Multivariate analysis |  |  |  |
|   | HR (95% CI) | P value | HR (95%CI) | P value (adjusted) |   | HR (95% CI) | P value | HR (95%CI) | P value (adjusted) |  |  |
| Gender | Male | 66 | 0.92 (0.55-1.55) | 0.767 | – | – | 66 | 0.76 (0.43-1.36) | 0.357 | – | – |
| Female | 22 | 22 |  |  |  |  |  |  |  |  |  |
| Age | < 62 | 42 | 0.81 (0.51-1.28) | 0.370 | – | – | 42 | 0.85 (0.51-1.40) | 0.523 | – | – |
| ≥62 | 46 | 46 |  |  |  |  |  |  |  |  |  |
| Stage | 3 | 19 | 1.59 (0.91-2.76) | 0.100 | – | – | 19 | 1.60 (0.84-3.02) | 0.151 | – | – |
| 4 | 69 | 69 |  |  |  |  |  |  |  |  |  |
| ECOG | 0 | 27 | 1.22 (0.73-2.04) | 0.451 | – | – | 27 | 1.95 (1.06-3.57) | 0.031 | 2.13 (1.14-3.97) | 0.018 |
| 1 | 61 | 61 |  |  |  |  |  |  |  |  |  |
| ABCG2 Q141K | CC | 5.51 | 0.48 (0.29-0.78) | 0.003 | 0.51 (0.31-0.83) | 0.007 | 11.57 | 0.52 (0.31-0.89) | 0.017 | 0.61 (0.35-1.06) | 0.080 |
| CA/AA | 9.12 | 20.19 |  |  |  |  |  |  |  |  |  |
| SLC29A3 S158F | CC | 5.08 | 0.56 (0.33-0.96) | 0.034 | 0.64 (0.37-1.10) | 0.108 | 8.43 | 0.52 (0.30-0.92) | 0.024 | 0.49 (0.27-0.88) | 0.017 |
| CT/TT | 7.84 | 17.64 |  |  |  |  |  |  |  |  |  |
| NT5C2 D549Ea | CC | 5.25 | 0.71 (0.41-1.21) | 0.206 | – | – | 10.49 | 0.67 (0.38-1.17) | 0.157 | – | – |
| CT/TT | 7.57 | 17.31 |  |  |  |  |  |  |  |  |  |
| HELB T980I | CC | 8.2 | 1.42 (0.90-2.25) | 0.133 | – | – | 17.05 | 1.23 (0.75-2.04) | 0.412 | – | – |
| CT/TT | 5.74 | 11.28 |  |  |  |  |  |  |  |  |  |
| CTDP1 T221M | CC | 7.02 | 1.24 (0.74-2.09) | 0.421 | – | – | 13.35 | 1.10 (0.62-1.96) | 0.741 | – | – |
| CT/TT | 6.56 | 16.85 |  |  |  |  |  |  |  |  |  |
| POLR2A N764K | CC | 7.02 | 1.15 (0.55-2.42) | 0.711 | – | – | 13.34 | 1.14 (0.54-2.40) | 0.734 | – | – |
| CT | 7.57 | 14.03 |  |  |  |  |  |  |  |  |  |

Table 2 Caption: Univariate and multivariate Cox regression analyses of progression-free survival (PFS) and overall survival (OS) of the six final candidate SNPs with clinical parameters in the NSCLC cohort

SLC29A3 S158F (c.473C > T, rs780668) was found to be associated with increased OS. Patients with CT/TT genotype were shown to have longer median OS compared to CC genotype i.e. 17.64 months [95% CI 10.55-24.73 months] vs 8.43 months [94% CI 1.21-15.64 months], HR 0.49 (95% CI 0.27-0.88), adjusted *P =* 0.017 (Table [2](#Tab2)). Association with OS/PFS could not be found in four other variants (Table [2](#Tab2)).

### ABCG2 Q141K and POLR2A N764K are correlated with gemcitabine cytotoxicity

The ABCG2 Q141K variant (the CA/AA genotype) was not only associated with improved PFS but was also found to be associated with increased toxicity i.e. higher risk of grade 3 or 4 thrombocytopenia (low platelet count) compared to the wild-type genotype (CC) (70.7% vs 44.7% respectively, HR 3.79 (95% CI 1.42-10.1) adjusted *P* = 0.008) (Table [3](#Tab3)). Interestingly, the wild-type CC genotype of POLR2A N764K variant was found to be associated with a higher risk of grade 3 or 4 thrombocytopenia at 61.5% compared to 20.0% of the SNP’s CT genotype, HR 0.18 (95% CI 0.03-0.98), adjusted *P* = 0.048 (Table [3](#Tab3)).

### Table 3.

|   | Factors/Genotype | Number/ Percentage of grade 3/4 neutropenia (all cycles) | Grade 3/4 neutropenia |  |  |  | Number/Percentage of grade 3/4 thrombocytopenia (all cycles) | Grade 3/4 thrombocytopenia |  |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| Univariate analysis |  | Multivariate analysis |  | Univariate analysis |  | Multivariate analysis |  |  |  |  |  |
| OR (95%CI) | P value | OR (95%CI) | P value (adjusted) | OR (95%CI) | P value | OR (95%CI) | P value (adjusted) |  |  |  |  |
| Gender | Male | 66 | 2.74 (0.83-9.04) | 0.097 | – | – | 66 | 4.78 (1.46-15.7) | 0.01 | 4.44 (1.27-15.6) | 0.020 |
| Female | 22 | 22 |  |  |  |  |  |  |  |  |  |
| Age | < 62 | 42 | 1.03 (0.42-2.51) | 0.942 | – | – | 42 | 0.56 (0.24-1.31) | 0.178 | – | – |
| ≥62 | 46 | 46 |  |  |  |  |  |  |  |  |  |
| Stage | 3 | 19 | 2.21 (0.78-6.24) | 0.136 | – | – | 19 | 2.84 (0.99-9.12) | 0.052 | – | – |
| 4 | 69 | 69 |  |  |  |  |  |  |  |  |  |
| ECOG | 0 | 27 | 0.80 (0.30-2.14) | 0.659 | – | – | 27 | 0.87 (0.35-2.17) | 0.758 | – | – |
| 1 | 61 | 61 |  |  |  |  |  |  |  |  |  |
| ABCG2 Q141K | CC | 59.60% | 2.10 (0.84-5.28) | 0.113 | – | – | 44.70% | 2.99 (1.23-7.25) | 0.015 | 3.79 (1.42-10.1) | 0.008 |
| CA/AA | 75.60% | 70.70% |  |  |  |  |  |  |  |  |  |
| SLC29A3 S158F | CC | 50.00% | 2.67 (0.99-7.22) | 0.054 | – | – | 45.60% | 1.85 (0.70-4.89) | 0.109 | – | – |
| CT/TT | 72.70% | 60.60% |  |  |  |  |  |  |  |  |  |
| NT5C2 D549E* | CC | 57.10% | 1.76 (0.64-4.84) | 0.272 | – | – | 57.10% | 0.98 (0.37-2.65) | 0.973 | – | – |
| CT/TT | 70.20% | 56.70% |  |  |  |  |  |  |  |  |  |
| HELB T980I | CC | 70.00% | 0.73 (0.30-1.80) | 0.499 | – | – | 56.00% | 1.08 (0.46-2.53) | 0.859 | – | – |
| CT/TT | 63.20% | 57.90% |  |  |  |  |  |  |  |  |  |
| CTDP1 T221M | CC | 66.70% | 1.07 (0.38-3.01) | 0.090 | – | – | 54.60% | 1.46 (0.54-3.94) | 0.457 | – | – |
| CT/TT | 68.20% | 63.60% |  |  |  |  |  |  |  |  |  |
| POLR2A N764K | CC CT | 70.50 40.00% | 0.28 (0.07-1.08) | 0.065 | – | – | 61.50 20.00% | 0.16 (0.03-0.79) | 0.024 | 0.18 (0.03-0.98) | 0.048 |

Table 3 Caption: Univariate and multivariate analyses by chi square and logistic regression, respectively, of grade 3 or 4 neutropenia and thrombocytopenia of the six final candidate SNPs with clinical parameters in the NSCLC cohort

## Discussion

In this study, three out of the six candidate SNPs were confirmed to be associated with NSCLC patient outcome i.e. OS, PFS and side effect. To the best of our knowledge, this is the first study showing association of ABCG2 Q141K (rs2231142), SLC29A3 S158F (rs780668) and POLR2A N764K (rs2228130) with NSCLC patient outcome treated with gemcitabine-based chemotherapy. ABCG2 belongs to the ABCG subfamily and ABC transporter superfamily. The ABCG family has five members i.e. ABCG2, ABCG1, ABCG4, ABCG5 and ABCG8. ABCG2 consists of a nucleotide-binding domain (NBD) in the amino terminus followed by six putative transmembrane domains (Fig. [3a](#Fig3)). The ABCG2 Q141K SNP is located at the NBD in the cytoplasmic part of the protein. The c.421A allele frequency of ABCG2 Q141K is known as one of the common SNPs in Asian people (about 26-35%) [[32](#CR32)]. Moreover, this SNP has been shown to be associated with increased risk of gout [[33](#CR33)]. When we created our own detailed multiple sequence alignment using all members in the ABCG family, we found that glutamine in this position is well conserved in ABCG2 orthologs but not in other members in the family, therefore Q141 can be considered as an ABCG2-subfamily specific conserved residue (Fig. [3b](#Fig3)). ABCG2 is the only member in this family that is not involved in cholesterol efflux but it mediates the efflux of a wide range of xenobiotics including gemcitabine, using ATP as an energy source [[34](#CR34)]. There is in vitro evidence that ABCG2 Q141K decreases efflux activity and increases intracellular gemcitabine levels and it has been known to be associated with impaired ABCG2 activity by lowering protein expression level or decreasing ATPase activity [[35](#CR35)]. The study supports the observation that ABCG2 itself plays a role in decreasing intracellular concentration of gemcitabine. Another in vitro study demonstrates significantly worse overall survival for carriers of the ABCG2 421A-allele treated with platinum-based drugs [[36](#CR36)]. Mizuarai et al. described that the ATPase activity of the Q141K variant was reduced approximately 1.3-fold compared to the activity of the wild type ABCG2 in polarized LLC-PK1 cell lines, resulting in increased drug accumulation and decreased drug efflux in the variant ABCG2-expressing cells [[36](#CR36)]. According to BLAST against PDB, a crystal structure of Malk, the ATP subunit of the maltose transporter from *E.coli* (PDB:1Q12 chain A) [[37](#CR37)] was the top hit with a E-value of 2.0E-17. We used this template to do homology modeling of the NBD region (position 41-299) of ABCG2 using MODELLER. The ABCG2 model is shown in Fig. [3c](#Fig3). We used this model to calculate the stability change upon mutation by FoldX and found the average free energy changes (ddG) when mutating Q to K at position 141 of ABCG2 to be 1.93 kcal/mol with a standard deviation (SD) of 0.10 kcal/mol. This suggests that the SNP has a destabilizing effect on the protein structure which is in agreement with a recent finding that Q141 causes instability in the NBD [[38](#CR38)]. The SNP is located in the loop region which is relatively near the ATP binding site of the dimer and changing the neutral side-chain glutamine to positively-charged side-chain lysine may affect the scaffold of the neighboring ATP binding site formed by the homodimer (Fig. [3c](#Fig3)). Therefore, it can be proposed that if a patient has this variant and is treated with gemcitabine, efflux of gemcitabine can be diminished resulting in an increase in the intracellular concentration of gemcitabine in cancer cells and it is thus more effective at killing cancer cells. However, since normal cells also have this SNP which causes accumulation of the drug and other substrates exported by this protein, this SNP is also linked to increased toxicity in normal cells (Fig. [4](#Fig4)).

### Fig. 3.

![Fig. 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ca01/5948914/a769d032eef4/12885_2018_4471_Fig3_HTML.jpg)

Result of detailed SNP analysis of ABCG2 Q141K. a Domain architecture of ABCG2. ABCG2 contains a nucleotide-binding domain (NBD) in the cytoplasmic region and a membrane-spanning domain transmembrane domain (MSD) consisting of 6 putative transmembrane segments. ABCG2 Q141K is located in the NBD. b Multiple alignment using all five members in the ABCG subfamily. Orthologs of each member were retrieved from OMA browser (omabrowser.org/). MAFFT with L-INS-i was used to create the multiple alignment. We used seven representative organisms to show conservation of SNP’s region. HUMAN: H. sapiens, MACMU: M. mulatta, BOVIN: B. taurus, CANFA: C. familiaris, MOUSE: M.musculus, MONDO: M.domestica, ANOCA: A. carolinensis. c Homology model of the nucleotide-binding domain of ABCG2 using the ATP subunit of the maltose transporter from E.coli (PDB:1Q12 chain A) [37] as a template is shown in green. ATPs are shown in blue and Q141 is shown in red. Superimposition of the model and chain B of the template (shown in purple) was done to show the homodimer of the region

### Fig. 4.

![Fig. 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ca01/5948914/380854b58169/12885_2018_4471_Fig4_HTML.jpg)

Proposed mechanisms of ABCG2 Q141K SNP and patient outcome. NSCLC patient who is treated with gemcitabine and have ABCG2 Q141K either heterozygous or homozygous allele could increase their survival because of accumulating more gemcitabine inside cancer cells and is thus more effective in killing cancer cells. However, probability of increasing toxicity can occur since other substrates of ABCG2 can be accumulated inside normal/healthy cells and thus cause cell death. These healthy cells can include cells in the bone marrow that produce blood e.g. platelets

Our study also showed for the first time that patients who were carrying either the CT or TT of SLC29A3 473 C > T (rs780668) were associated with increased OS. SLC29A3 belongs to the equilibrative nucleoside transporter (ENT) family, responsible for passive nucleoside transport and has 11 transmembrane helices (TMs) within the nucleoside transporter domain (Pfam:PF01733) (Fig. [5a](#Fig5)). SLC29A3 S158 is likely to be a subfamily specific residue since serine at this position is fully conserved from human to fish among SLC29A3 orthologs but cannot be seen in other members in the family (Fig. [5b](#Fig5)). This residue may relate to its unique function compared to other members in that it seems to function in the inner membrane of mitochondria and/or in the lysosome which requires an acidic pH environment and the position of the SNP seems to localize outside of the inner membrane of mitochondria [[39](#CR39)]. For homology modeling of SLC29A3, we had a problem retrieving correct TMs models using MODELLER with loop refinement. The template used was a crystal structure of the glycerol-3-phosphate transporter from *E.Coli* (PDB:1PW4) chain A which has only 12% identity to our query. This problem is common when we used MODELLER which is more suitable for modeling soluble proteins than membrane proteins. Therefore, we used another software called Memoir [[40](#CR40)] which is a homology modelling algorithm designed specifically for membrane proteins. A homology model from this software retrieved 11 TMs with long N-terminus and long loop regions between TM6 and 7 and the correct SNP’s position on the 3D structure (Fig. [5c](#Fig5)). According to FoldX, SLC29A3 S158F was predicted to have a significant destabilizing effect with average ddG of 2.18 kcal/mol which could be explained by the strong change from the polar side chain serine to the larger and more hydrophobic side chain phenylalanine (Fig. [5d](#Fig5)). Besides potential involvement of the conserved wildtype serine in the transport process, increasing the hydrophobicity through the mutation at the outside interface with the membrane may result in deeper insertion of the affected helix in the membrane. SLC29A3 can transport gemcitabine into organelles, e.g., mitochondria [[39](#CR39)]. Moreover, SLC29A3 could be involved in the mitochondrial toxicity of nucleoside drugs [[8](#CR8)]. Since SLC29A3 S158F is most likely to affect protein function, it may have a significant impact on transporting gemcitabine into mitochondria.

### Fig. 5.

![Fig. 5](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ca01/5948914/5c4445eb6190/12885_2018_4471_Fig5_HTML.jpg)

Result of detailed SNP analysis of SLC29A3 S158F. a Domain architecture of SLC29A3. The protein has 11 transmembrane helices and SLC29A3 S158F (red lollipop) lies between TM3 and TM4 which is localized extracellularly. Residues 169-473 correspond to the nucleoside transporter domain (Pfam:PF01733). b Multiple alignment using all four members in the SLC29 gene family i.e. SLC29A1-4. Orthologs of each member were retrieved using Orthologue search on ANNOTATOR [25]. MAFFT with L-INS-i was used to create the multiple alignment. c Homology model of SLC29A3 created by Memoir based on the template of a crystal structure of the glycerol-3-phosphate transporter from E. coli (PDB ID: 1PW4 chain A). This template was retrieved using HHPRED against PDB on ANNOTATOR [e-value = 1.7e-08]. d Screen shot of the FoldX result showing wild type and after mutation. The SNP was predicted to be stabilizing on protein structure (average ddG run over 5 times = − 1.14 kcal/mol SD = 0.04)

POLR2A (DNA-directed RNA polymerase II subunit RPB1) encodes the largest subunit (out of 12 subunits) of RNA polymerase II (Pol II) which catalyzes the RNA synthesis from DNA. POLR2A contains a carboxy terminal domain (CTD) which is composed of 52 heptapeptide repeats that are necessary for the polymerase activity (Fig. [6a](#Fig6)). POLR2A N764K is located in the RNA polymerase’s domain 4 (Pfam:PF05000) which is also known as the funnel domain. The N764 is highly conserved among orthologs (Fig. [6b](#Fig6)) and both PolyPhen-2 and SIFT analyses predicted that the SNP affects protein function (Additional file [1](#MOESM1): Table S2). We created a homology model of POLR2A without the CTD using a crystal structure of yeast RNA polymerase II (PDB:1I3Q chain A) as a template (%identity = 50.3%) (Fig. [6c](#Fig6)). The SNP is in the loop region and it was predicted to have a destabilizing effect by FoldX (average ddG 1.13 kcal/mol) (Fig. [6d](#Fig6)). This could be due to the longer and charged lysine side chain causing a change in the conformation of the local loop structure. In our study, interestingly, wild type (CC) is found to be associated with higher grade 3 or 4 thrombocytopenia when compared to the CT variant (Table [3](#Tab3)). Further analysis is needed to understand the mechanism for this. There is an evidence that dFdCTP is incorporated into RNA which is concentration- and time-dependent, resulting in inhibition of RNA synthesis [[41](#CR41)]. In human parental NSCLC cells with a different inherent gemcitabine resistance, sensitivity to gemcitabine was related to differences in RNA corporation [[42](#CR42)]. Since the SNP is found to be strongly deleterious from our analyses, it would be interesting to investigate whether POLR2A itself plays a role in the gemcitabine pharmacologic pathway and whether POLR2A N764K SNP has any implications on RNA synthesis.

### Fig. 6.

![Fig. 6](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ca01/5948914/2c1aef19be25/12885_2018_4471_Fig6_HTML.jpg)

Result of detailed SNP analysis of POLR2A N764K. a Domain architecture of POLR2A. b Multiple alignment of POLR2A. Orthologs of POLR2A were retrieved using Orthologue search on ANNOTATOR [25]. MAFFT with L-INS-i was used to create the multiple alignment. c Homology model of POLR2A was created by MODELLER with loop refinement using a crystal structure of yeast RNA polymerase II (PDB:1I3Q chain A) as a template. This template was retrieved using BLAST against PDB on ANNOTATOR [e-value = 0.0]. d Screen shot of the FoldX result showing wild type and after mutation. The SNP was predicted to destabilize protein structure (average ddG run over 5 times = 1.13 kcal/mol, SD = 0.09)

NT5C2 (5′-nucleotidase, cytosolic II) encodes a hydrolase that serves a crucial role in cellular purine metabolism by acting primarily on inosine 5′-monophosphate (IMP) or guanosine monophosphate (GMP). The 5′-nucleotidase is a huge family of enzymes that catalyze the dephosphorylation of deoxy- and ribonucleoside monosphosphates to nucleoside analogues and inorganic phosphates [[43](#CR43), [44](#CR44)]. NT5C2 is known to dephosphorylate monophosphorylated gemcitabine. The NT5C2 D549 is the first charged amino acid in the last 13 acidic residues on the C-terminus of the protein (Fig. [7a](#Fig7)). A study in 1999 proved the importance of the highly acidic C-terminus in NT5C2 protein using cDNA constructs encoding proteins lacking either N- or C-terminus and obtained the kinetic and molecular characteristics of the recombinant proteins [[45](#CR45)]. When the last 13 acidic residues on the C-terminus were eliminated, there was a drastic reduction in the catalytic competence of the enzyme by lowering both the substrate affinity and the specific productivity. Furthermore, the capability of the protein to form a tetramer was significantly compromised. From the experiment above, it was concluded that the region of glutamic and aspartic acid residues in the C-terminus of the enzyme is necessary for the complete function of NT5C2. Another study, on bovine NT5C2 found that the C-terminus is perhaps involved with the modulation of enzyme function [[46](#CR46)]. In our detailed SNP analysis, the D549 is well conserved among orthologs of NT5C2 (Fig. [7b](#Fig7)) but is not conserved throughout the NT5C enzyme family (data not shown). D549E was predicted as deleterious by both PolyPhen-2 and SIFT. However, D and E are both negatively charged amino acids and the following acidic C-terminus shows a mixed pattern of the two so it is not mechanistically clear how this mutation could alter the enzyme activity of NT5C2. In agreement with this, we do not find significant correlations with survival or toxicity for NT5C2 D549E.

### Fig. 7.

![Fig. 7](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ca01/5948914/d4e20c171e4f/12885_2018_4471_Fig7_HTML.jpg)

Result of detailed SNP analysis of NT5C2 D549E. a Domain architecture of NT5C2. b Multiple alignment of NT5C2. Orthologs of NT5C2 were retrieved using Orthologue search on ANNOTATOR [25]. MAFFT with L-INS-i was used to create the multiple alignment

HELB T980I and CTDP1 T221M, unlike the other four SNPs that we predicted to directly affect the gemcitabine pharmacologic pathway, were selected because they could potentially affect the pathway indirectly in order to establish if more remotely related SNPs are still useful candidates for further studies. According to our criteria, HELB T980I passed three criteria i.e. significant results in SIFT, PolyPhen-2 and high MAF in Singaporean Chinese (Additional file [1](#MOESM1): Table S2) while CTDP1 T221M passed two criteria (borderline) i.e. the SNP is in a domain region and has significant FoldX stability change (Additional file [1](#MOESM1): Table S2). HELB T980I lies close to the phosphorylation sites in the phosphorylation regulated subcellular localization (PSLD) domain at the C-terminus of the protein. The PSLD domain has been suggested to play a significant role in regulating the subcellular localization of HELB [[47](#CR47)]. The same study also suggested a possible role of HELB in DNA repair. Another study gives further evidence that HELB is recruited by Replication Protein A to mitigate replication stress [[48](#CR48)]. So, we hypothesized that the SNP could impair HELB’s DNA replication stress mitigating effect of gemcitabine-induced DNA damage which could lead to reduced recovery from gemcitabine-induced replication stress, and hence to a better anti-cancer activity of gemcitabine. CTDP1 dephosphorylates a phosphorylated C-terminal domain of POLR2A to facilitate Pol II recycling for transcription [[49](#CR49)]. There is evidence suggesting that CTDP1 may play a role in DNA damage response as well [[50](#CR50)]. However, from our clinical results, we could not see any correlation between these two SNPs and the patient outcome. Therefore, it can be suggested that when we choose SNPs in the final step, we should expect a high chance for correlation with drug response only for those that are affecting the pharmacologic pathway directly.

## Conclusions

Overall, our bioinformatics approach can be used to select a small number of potential causative SNPs that can be rationalized with molecular mechanisms of their effects. Our workflow can also be applied to any pathway of interest that could affect other phenotypes. Nowadays, using GWAS to find SNPs associated with common diseases or adverse drug reactions are the norm but even though thousands of case samples have been used, one review found that about 30% of the results lead to a null finding [[51](#CR51)]. Furthermore, significant GWAS hits are often not easily explainable to have functional effects e.g. when they are intronic or synonymous SNPs. Although our study may be limited in sample size, we managed to find that three out of six final candidate SNPs are associated with patient outcome. However, more association studies and possibly molecular and cellular studies are needed to further establish the value of these three SNPs as biomarkers of patient outcome. While we have to acknowledge that this approach of selective filters does not guarantee to find all SNPs involved in a studied phenotype, the main benefit is that one can reduce the space of possible candidates to a small experimentally tractable number with higher chance of being relevant. We do believe that other candidate SNPs that passed three out of five criteria would also be potential candidates to be tested further. We hope our approach and findings will pave the way to more meaningful biomarkers and personalized treatment options in the future.

## Additional file

## Acknowledgements

We are thankful to Hong Sain Ooi who helped us on collecting protein-protein interaction data from different public databases.

### Funding

This work was supported by the Agency for Science, Technology and Research (A*STAR) Joint Council Office (JCO) Grant JCOAG04_FG03_2009. The clinical trial was supported by the BMRC grant 06/1/26/19/482. RAS is supported by the National Research Foundation, Singapore and the Singapore Ministry of Education under its Research Centers of Excellence initiative. The funders had no role in the design of the study, data collection and analysis, interpretation of data or preparation of the manuscript.

### Availability of data and materials

All data analysed during this study are included in this published article and Additional file.

## Abbreviations

## Authors’ contributions

VL, FK, and SM conceived and designed the bioinformatics study. VL, FK, PJ and JSX performed bioinformatics analysis. CT, LS, RAS contributed to patient recruitment. YK performed genotyping of patient samples. Statistical analysis were performed by YM, NS and CT. VL, JSX and SM contributed to the interpretation of the data. VL, CT, JSX, YK, YM and SM wrote the manuscript. FE, SM and WY served as principal investigators of this project. All authors read and approved the final manuscript.

### Ethics approval and consent to participate

### Ethics approval and consent to participate

The Singapore National Healthcare Group Domain Specific Review Board reviewed and approved the study. All the patients provided written informed consent before study entry. The study was conducted in accordance to Good Clinical Practice guidelines.

### Competing interests

The authors declare that they have no competing interests.

### Publisher’s Note

Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.

## Footnotes

## Contributor Information

Vachiranee Limviphuvadh, Email: vachiraneel@bii.a-star.edu.sg.

Chee Seng Tan, Email: chee_seng_tan@nuhs.edu.sg.

Fumikazu Konishi, Email: konishi@acls.titech.ac.jp.

Piroon Jenjaroenpun, Email: piroonj@bii.a-star.edu.sg.

Joy Shengnan Xiang, Email: jsxiang@stanford.edu.

Yuliya Kremenska, Email: yuliya.kremenska@gmail.com.

Yar Soe Mu, Email: yar_soe_mu@nuhs.edu.sg.

Nicholas Syn, Email: nicholas.syn@u.nus.edu.

Soo Chin Lee, Email: soo_chin_lee@nuhs.edu.sg.

Ross A. Soo, Email: ross_soo@nuhs.edu.sg

Frank Eisenhaber, Email: franke@bii.a-star.edu.sg.

Sebastian Maurer-Stroh, Email: sebastianms@bii.a-star.edu.sg.

Wei Peng Yong, Email: Wei_Peng_Yong@nuhs.edu.sg.

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Data Availability Statement

All data analysed during this study are included in this published article and Additional file.

### Supplementary Materials

### Data Availability Statement

All data analysed during this study are included in this published article and Additional file.

## References

1. Blackstein M, Vogel CL, Ambinder R, Cowan J, Iglesias J, Melemed A. Gemcitabine as first-line therapy in patients with metastatic breast cancer: a phase II trial. Oncology. 2002;62:2–8. doi: 10.1159/000048240.  [DOI](https://doi.org/10.1159/000048240) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11810037/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncology&title=Gemcitabine%20as%20first-line%20therapy%20in%20patients%20with%20metastatic%20breast%20cancer:%20a%20phase%20II%20trial&author=M%20Blackstein&author=CL%20Vogel&author=R%20Ambinder&author=J%20Cowan&author=J%20Iglesias&volume=62&publication_year=2002&pages=2-8&pmid=11810037&doi=10.1159/000048240&)

2. Burris HA, 3rd, Moore MJ, Andersen J, Green MR, Rothenberg ML, Modiano MR, Cripps MC, Portenoy RK, Storniolo AM, Tarassoff P, Nelson R, Dorr FA, Stephens CD, Von Hoff DD. Improvements in survival and clinical benefit with gemcitabine as first-line therapy for patients with advanced pancreas cancer: a randomized trial. J Clin Oncol. 1997;15:2403–2413. doi: 10.1200/JCO.1997.15.6.2403.  [DOI](https://doi.org/10.1200/JCO.1997.15.6.2403) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9196156/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Improvements%20in%20survival%20and%20clinical%20benefit%20with%20gemcitabine%20as%20first-line%20therapy%20for%20patients%20with%20advanced%20pancreas%20cancer:%20a%20randomized%20trial&author=HA%20Burris&author=MJ%20Moore&author=J%20Andersen&author=MR%20Green&author=ML%20Rothenberg&volume=15&publication_year=1997&pages=2403-2413&pmid=9196156&doi=10.1200/JCO.1997.15.6.2403&)

3. Kuznetsov V, Lee HK, Maurer-Stroh S, Molnár MJ, Pongor S, Eisenhaber B, Eisenhaber F. How bioinformatics influences health informatics: usage of biomolecular sequences, expression profiles and automated microscopic image analyses for clinical needs and public health. Health Inf Sci Syst. 2013;1. https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4336111/.  [https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4336111/](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4336111/) | [DOI](https://doi.org/10.1186/2047-2501-1-2) | [PMC free article](/articles/PMC4336111/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25825654/)

4. Eisenhaber F. A decade after the first full human genome sequencing: when will we understand our own genome? J Bioinforma Comput Biol. 2012;10:1271001. doi: 10.1142/S0219720012710011.  [DOI](https://doi.org/10.1142/S0219720012710011) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22849370/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Bioinforma%20Comput%20Biol&title=A%20decade%20after%20the%20first%20full%20human%20genome%20sequencing:%20when%20will%20we%20understand%20our%20own%20genome?&author=F%20Eisenhaber&volume=10&publication_year=2012&pages=1271001&pmid=22849370&doi=10.1142/S0219720012710011&)

5. Ma Q, Lu AYH. Pharmacogenetics, pharmacogenomics, and individualized medicine. Pharmacol Rev. 2011;63:437–459. doi: 10.1124/pr.110.003533.  [DOI](https://doi.org/10.1124/pr.110.003533) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21436344/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacol%20Rev&title=Pharmacogenetics,%20pharmacogenomics,%20and%20individualized%20medicine&author=Q%20Ma&author=AYH%20Lu&volume=63&publication_year=2011&pages=437-459&pmid=21436344&doi=10.1124/pr.110.003533&)

6. Soo RA, Wang LZ, Ng SS, Chong PY, Yong WP, Lee SC, Liu JJ, Choo TB, Tham LS, Lee HS, Goh BC, Soong R. Distribution of gemcitabine pathway genotypes in ethnic Asians and their association with outcome in non-small cell lung cancer patients. Lung Cancer. 2009;63:121–127. doi: 10.1016/j.lungcan.2008.04.010.  [DOI](https://doi.org/10.1016/j.lungcan.2008.04.010) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18538445/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lung%20Cancer&title=Distribution%20of%20gemcitabine%20pathway%20genotypes%20in%20ethnic%20Asians%20and%20their%20association%20with%20outcome%20in%20non-small%20cell%20lung%20cancer%20patients&author=RA%20Soo&author=LZ%20Wang&author=SS%20Ng&author=PY%20Chong&author=WP%20Yong&volume=63&publication_year=2009&pages=121-127&pmid=18538445&doi=10.1016/j.lungcan.2008.04.010&)

7. Kanehisa M, Goto S. KEGG: Kyoto encyclopedia of genes and genomes. Nucleic Acids Res. 2000;28:27–30. doi: 10.1093/nar/28.1.27.  [DOI](https://doi.org/10.1093/nar/28.1.27) | [PMC free article](/articles/PMC102409/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10592173/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=KEGG:%20Kyoto%20encyclopedia%20of%20genes%20and%20genomes&author=M%20Kanehisa&author=S%20Goto&volume=28&publication_year=2000&pages=27-30&pmid=10592173&doi=10.1093/nar/28.1.27&)

8. Errasti-Murugarren E, Pastor-Anglada M. Drug transporter pharmacogenetics in nucleoside-based therapies. Pharmacogenomics. 2010;11:809–841. doi: 10.2217/pgs.10.70.  [DOI](https://doi.org/10.2217/pgs.10.70) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20504255/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Drug%20transporter%20pharmacogenetics%20in%20nucleoside-based%20therapies&author=E%20Errasti-Murugarren&author=M%20Pastor-Anglada&volume=11&publication_year=2010&pages=809-841&pmid=20504255&doi=10.2217/pgs.10.70&)

9. Fukunaga AK, Marsh S, Murry DJ, Hurley TD, McLeod HL. Identification and analysis of single-nucleotide polymorphisms in the gemcitabine pharmacologic pathway. Pharm J. 2004;4:307–314. doi: 10.1038/sj.tpj.6500259.  [DOI](https://doi.org/10.1038/sj.tpj.6500259) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15224082/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharm%20J&title=Identification%20and%20analysis%20of%20single-nucleotide%20polymorphisms%20in%20the%20gemcitabine%20pharmacologic%20pathway&author=AK%20Fukunaga&author=S%20Marsh&author=DJ%20Murry&author=TD%20Hurley&author=HL%20McLeod&volume=4&publication_year=2004&pages=307-314&pmid=15224082&doi=10.1038/sj.tpj.6500259&)

10. Bader GD, Betel D, Hogue CWV. BIND: the biomolecular interaction network database. Nucleic Acids Res. 2003;31:248–250. doi: 10.1093/nar/gkg056.  [DOI](https://doi.org/10.1093/nar/gkg056) | [PMC free article](/articles/PMC165503/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12519993/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=BIND:%20the%20biomolecular%20interaction%20network%20database&author=GD%20Bader&author=D%20Betel&author=CWV%20Hogue&volume=31&publication_year=2003&pages=248-250&pmid=12519993&doi=10.1093/nar/gkg056&)

11. Stark C, Breitkreutz B-J, Reguly T, Boucher L, Breitkreutz A, Tyers M. BioGRID: a general repository for interaction datasets. Nucleic Acids Res. 2006;34(Database issue):D535–D539. doi: 10.1093/nar/gkj109.  [DOI](https://doi.org/10.1093/nar/gkj109) | [PMC free article](/articles/PMC1347471/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16381927/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=BioGRID:%20a%20general%20repository%20for%20interaction%20datasets&author=C%20Stark&author=B-J%20Breitkreutz&author=T%20Reguly&author=L%20Boucher&author=A%20Breitkreutz&volume=34&issue=Database%20issue&publication_year=2006&pages=D535-D539&pmid=16381927&doi=10.1093/nar/gkj109&)

12. Kerrien S, Alam-Faruque Y, Aranda B, Bancarz I, Bridge A, Derow C, Dimmer E, Feuermann M, Friedrichsen A, Huntley R, Kohler C, Khadake J, Leroy C, Liban A, Lieftink C, Montecchi-Palazzi L, Orchard S, Risse J, Robbe K, Roechert B, Thorneycroft D, Zhang Y, Apweiler R, Hermjakob H. IntAct--open source resource for molecular interaction data. Nucleic Acids Res. 2007;35(Database issue):D561–D565. doi: 10.1093/nar/gkl958.  [DOI](https://doi.org/10.1093/nar/gkl958) | [PMC free article](/articles/PMC1751531/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17145710/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=IntAct--open%20source%20resource%20for%20molecular%20interaction%20data&author=S%20Kerrien&author=Y%20Alam-Faruque&author=B%20Aranda&author=I%20Bancarz&author=A%20Bridge&volume=35&issue=Database%20issue&publication_year=2007&pages=D561-D565&pmid=17145710&doi=10.1093/nar/gkl958&)

13. Salwinski L, Miller CS, Smith AJ, Pettit FK, Bowie JU, Eisenberg D. The database of interacting proteins: 2004 update. Nucleic Acids Res. 2004;32(Database issue):D449–D451. doi: 10.1093/nar/gkh086.  [DOI](https://doi.org/10.1093/nar/gkh086) | [PMC free article](/articles/PMC308820/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14681454/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=The%20database%20of%20interacting%20proteins:%202004%20update&author=L%20Salwinski&author=CS%20Miller&author=AJ%20Smith&author=FK%20Pettit&author=JU%20Bowie&volume=32&issue=Database%20issue&publication_year=2004&pages=D449-D451&pmid=14681454&doi=10.1093/nar/gkh086&)

14. Chatr-aryamontri A, Ceol A, Palazzi LM, Nardelli G, Schneider MV, Castagnoli L, Cesareni G. MINT: the molecular INTeraction database. Nucleic Acids Res. 2007;35(Database issue):D572–D574. doi: 10.1093/nar/gkl950.  [DOI](https://doi.org/10.1093/nar/gkl950) | [PMC free article](/articles/PMC1751541/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17135203/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=MINT:%20the%20molecular%20INTeraction%20database&author=A%20Chatr-aryamontri&author=A%20Ceol&author=LM%20Palazzi&author=G%20Nardelli&author=MV%20Schneider&volume=35&issue=Database%20issue&publication_year=2007&pages=D572-D574&pmid=17135203&doi=10.1093/nar/gkl950&)

15. Güldener U, Münsterkötter M, Oesterheld M, Pagel P, Ruepp A, Mewes H-W, Stümpflen V. MPact: the MIPS protein interaction resource on yeast. Nucleic Acids Res. 2006;34(Database issue):D436–D441. doi: 10.1093/nar/gkj003.  [DOI](https://doi.org/10.1093/nar/gkj003) | [PMC free article](/articles/PMC1347366/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16381906/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=MPact:%20the%20MIPS%20protein%20interaction%20resource%20on%20yeast&author=U%20G%C3%BCldener&author=M%20M%C3%BCnsterk%C3%B6tter&author=M%20Oesterheld&author=P%20Pagel&author=A%20Ruepp&volume=34&issue=Database%20issue&publication_year=2006&pages=D436-D441&pmid=16381906&doi=10.1093/nar/gkj003&)

16. Keshava Prasad TS, Goel R, Kandasamy K, Keerthikumar S, Kumar S, Mathivanan S, Telikicherla D, Raju R, Shafreen B, Venugopal A, Balakrishnan L, Marimuthu A, Banerjee S, Somanathan DS, Sebastian A, Rani S, Ray S, Harrys Kishore CJ, Kanth S, Ahmed M, Kashyap MK, Mohmood R, Ramachandra YL, Krishna V, Rahiman BA, Mohan S, Ranganathan P, Ramabadran S, Chaerkady R, Pandey A. Human protein reference database--2009 update. Nucleic Acids Res. 2009;37(Database issue):D767–D772. doi: 10.1093/nar/gkn892.  [DOI](https://doi.org/10.1093/nar/gkn892) | [PMC free article](/articles/PMC2686490/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18988627/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=Human%20protein%20reference%20database--2009%20update&author=TS%20Keshava%20Prasad&author=R%20Goel&author=K%20Kandasamy&author=S%20Keerthikumar&author=S%20Kumar&volume=37&issue=Database%20issue&publication_year=2009&pages=D767-D772&pmid=18988627&doi=10.1093/nar/gkn892&)

17. Pagel P, Kovac S, Oesterheld M, Brauner B, Dunger-Kaltenbach I, Frishman G, Montrone C, Mark P, Stümpflen V, Mewes H-W, Ruepp A, Frishman D. The MIPS mammalian protein-protein interaction database. Bioinformatics. 2005;21:832–834. doi: 10.1093/bioinformatics/bti115.  [DOI](https://doi.org/10.1093/bioinformatics/bti115) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15531608/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=The%20MIPS%20mammalian%20protein-protein%20interaction%20database&author=P%20Pagel&author=S%20Kovac&author=M%20Oesterheld&author=B%20Brauner&author=I%20Dunger-Kaltenbach&volume=21&publication_year=2005&pages=832-834&pmid=15531608&doi=10.1093/bioinformatics/bti115&)

18. Sherry ST, Ward MH, Kholodov M, Baker J, Phan L, Smigielski EM. Sirotkin K: dbSNP: the NCBI database of genetic variation. Nucleic Acids Res. 2001;29:308–311. doi: 10.1093/nar/29.1.308.  [DOI](https://doi.org/10.1093/nar/29.1.308) | [PMC free article](/articles/PMC29783/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11125122/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=Sirotkin%20K:%20dbSNP:%20the%20NCBI%20database%20of%20genetic%20variation&author=ST%20Sherry&author=MH%20Ward&author=M%20Kholodov&author=J%20Baker&author=L%20Phan&volume=29&publication_year=2001&pages=308-311&pmid=11125122&doi=10.1093/nar/29.1.308&)

19. Entrez Programming Utilities Help [http://www.ncbi.nlm.nih.gov/books/NBK25501/].  [http://www.ncbi.nlm.nih.gov/books/NBK25501/](http://www.ncbi.nlm.nih.gov/books/NBK25501)

20. Flicek P, Amode MR, Barrell D, Beal K, Brent S, Carvalho-Silva D, Clapham P, Coates G, Fairley S, Fitzgerald S, Gil L, Gordon L, Hendrix M, Hourlier T, Johnson N, Kähäri AK, Keefe D, Keenan S, Kinsella R, Komorowska M, Koscielny G, Kulesha E, Larsson P, Longden I, McLaren W, Muffato M, Overduin B, Pignatelli M, Pritchard B, Riat HS, et al. Ensembl 2012. Nucleic Acids Res. 2012;40:D84–D90. doi: 10.1093/nar/gkr991.  [DOI](https://doi.org/10.1093/nar/gkr991) | [PMC free article](/articles/PMC3245178/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22086963/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=Ensembl%202012&author=P%20Flicek&author=MR%20Amode&author=D%20Barrell&author=K%20Beal&author=S%20Brent&volume=40&publication_year=2012&pages=D84-D90&pmid=22086963&doi=10.1093/nar/gkr991&)

21. Teo Y-Y, Sim X, Ong RTH, Tan AKS, Chen J, Tantoso E, Small KS, Ku C-S, Lee EJD, Seielstad M, Chia K-S. Singapore genome variation project: a haplotype map of three southeast Asian populations. Genome Res. 2009;19:2154–2162. doi: 10.1101/gr.095000.109.  [DOI](https://doi.org/10.1101/gr.095000.109) | [PMC free article](/articles/PMC2775604/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19700652/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Res&title=Singapore%20genome%20variation%20project:%20a%20haplotype%20map%20of%20three%20southeast%20Asian%20populations&author=Y-Y%20Teo&author=X%20Sim&author=RTH%20Ong&author=AKS%20Tan&author=J%20Chen&volume=19&publication_year=2009&pages=2154-2162&pmid=19700652&doi=10.1101/gr.095000.109&)

22. Adzhubei I, Jordan DM, Sunyaev SR. Predicting functional effect of human missense mutations using PolyPhen-2. Curr Protoc Hum Genet. 2013; Chapter 7:Unit7.20. https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4480630/.  [https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4480630/](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4480630/) | [DOI](https://doi.org/10.1002/0471142905.hg0720s76) | [PMC free article](/articles/PMC4480630/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23315928/)

23. Kumar P, Henikoff S, Ng PC. Predicting the effects of coding non-synonymous variants on protein function using the SIFT algorithm. Nat Protoc. 2009;4:1073–1081. doi: 10.1038/nprot.2009.86.  [DOI](https://doi.org/10.1038/nprot.2009.86) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19561590/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Protoc&title=Predicting%20the%20effects%20of%20coding%20non-synonymous%20variants%20on%20protein%20function%20using%20the%20SIFT%20algorithm&author=P%20Kumar&author=S%20Henikoff&author=PC%20Ng&volume=4&publication_year=2009&pages=1073-1081&pmid=19561590&doi=10.1038/nprot.2009.86&)

24. Schneider A, Dessimoz C, Gonnet GH. OMA browser—exploring orthologous relations across 352 complete genomes. Bioinformatics. 2007;23:2180–2182. doi: 10.1093/bioinformatics/btm295.  [DOI](https://doi.org/10.1093/bioinformatics/btm295) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17545180/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=OMA%20browser%E2%80%94exploring%20orthologous%20relations%20across%20352%20complete%20genomes&author=A%20Schneider&author=C%20Dessimoz&author=GH%20Gonnet&volume=23&publication_year=2007&pages=2180-2182&pmid=17545180&doi=10.1093/bioinformatics/btm295&)

25. Eisenhaber B, Kuchibhatla D, Sherman W, Sirota FL, Berezovsky IN, Wong W-C, Eisenhaber F. The recipe for protein sequence-based function prediction and its implementation in the ANNOTATOR software environment. Methods Mol Biol. 2016;1415:477–506. doi: 10.1007/978-1-4939-3572-7_25.  [DOI](https://doi.org/10.1007/978-1-4939-3572-7_25) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27115649/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Methods%20Mol%20Biol&title=The%20recipe%20for%20protein%20sequence-based%20function%20prediction%20and%20its%20implementation%20in%20the%20ANNOTATOR%20software%20environment&author=B%20Eisenhaber&author=D%20Kuchibhatla&author=W%20Sherman&author=FL%20Sirota&author=IN%20Berezovsky&volume=1415&publication_year=2016&pages=477-506&pmid=27115649&doi=10.1007/978-1-4939-3572-7_25&)

26. Katoh K, Toh H. Recent developments in the MAFFT multiple sequence alignment program. Brief Bioinform. 2008;9:286–298. doi: 10.1093/bib/bbn013.  [DOI](https://doi.org/10.1093/bib/bbn013) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18372315/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Brief%20Bioinform&title=Recent%20developments%20in%20the%20MAFFT%20multiple%20sequence%20alignment%20program&author=K%20Katoh&author=H%20Toh&volume=9&publication_year=2008&pages=286-298&pmid=18372315&doi=10.1093/bib/bbn013&)

27. Waterhouse AM, Procter JB, DMA M, Clamp M, Barton GJ. Jalview version 2--a multiple sequence alignment editor and analysis workbench. Bioinformatics. 2009;25:1189–1191. doi: 10.1093/bioinformatics/btp033.  [DOI](https://doi.org/10.1093/bioinformatics/btp033) | [PMC free article](/articles/PMC2672624/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19151095/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=Jalview%20version%202--a%20multiple%20sequence%20alignment%20editor%20and%20analysis%20workbench&author=AM%20Waterhouse&author=JB%20Procter&author=M%20DMA&author=M%20Clamp&author=GJ%20Barton&volume=25&publication_year=2009&pages=1189-1191&pmid=19151095&doi=10.1093/bioinformatics/btp033&)

28. Schymkowitz J, Borg J, Stricher F, Nys R, Rousseau F, Serrano L. The FoldX web server: an online force field. Nucleic Acids Res. 2005;33(Web Server issue):W382–W388. doi: 10.1093/nar/gki387.  [DOI](https://doi.org/10.1093/nar/gki387) | [PMC free article](/articles/PMC1160148/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15980494/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=The%20FoldX%20web%20server:%20an%20online%20force%20field&author=J%20Schymkowitz&author=J%20Borg&author=F%20Stricher&author=R%20Nys&author=F%20Rousseau&volume=33&issue=Web%20Server%20issue&publication_year=2005&pages=W382-W388&pmid=15980494&doi=10.1093/nar/gki387&)

29. Van Durme J, Delgado J, Stricher F, Serrano L, Schymkowitz J, Rousseau F. A graphical interface for the FoldX forcefield. Bioinformatics. 2011;27:1711–1712. doi: 10.1093/bioinformatics/btr254.  [DOI](https://doi.org/10.1093/bioinformatics/btr254) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21505037/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=A%20graphical%20interface%20for%20the%20FoldX%20forcefield&author=J%20Van%20Durme&author=J%20Delgado&author=F%20Stricher&author=L%20Serrano&author=J%20Schymkowitz&volume=27&publication_year=2011&pages=1711-1712&pmid=21505037&doi=10.1093/bioinformatics/btr254&)

30. Eswar N, Eramian D, Webb B, Shen M-Y, Sali A. Protein structure modeling with MODELLER. Methods Mol Biol. 2008;426:145–159. doi: 10.1007/978-1-60327-058-8_8.  [DOI](https://doi.org/10.1007/978-1-60327-058-8_8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18542861/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Methods%20Mol%20Biol&title=Protein%20structure%20modeling%20with%20MODELLER&author=N%20Eswar&author=D%20Eramian&author=B%20Webb&author=M-Y%20Shen&author=A%20Sali&volume=426&publication_year=2008&pages=145-159&pmid=18542861&doi=10.1007/978-1-60327-058-8_8&)

31. Mah TL, Yap XN, Limviphuvadh V, Li N, Sridharan S, Kuralmani V, Feng M, Liem N, Adhikari S, Yong WP, Soo RA, Maurer-Stroh S, Eisenhaber F, Tong JC. Novel SNP improves differential survivability and mortality in non-small cell lung cancer patients. BMC Genomics. 2014;15(Suppl 9):S20. doi: 10.1186/1471-2164-15-S9-S20.  [DOI](https://doi.org/10.1186/1471-2164-15-S9-S20) | [PMC free article](/articles/PMC4290611/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25521664/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Genomics&title=Novel%20SNP%20improves%20differential%20survivability%20and%20mortality%20in%20non-small%20cell%20lung%20cancer%20patients&author=TL%20Mah&author=XN%20Yap&author=V%20Limviphuvadh&author=N%20Li&author=S%20Sridharan&volume=15&issue=Suppl%209&publication_year=2014&pages=S20&pmid=25521664&doi=10.1186/1471-2164-15-S9-S20&)

32. de Jong FA, Marsh S, Mathijssen RHJ, King C, Verweij J, Sparreboom A, McLeod HL. ABCG2 pharmacogenetics: ethnic differences in allele frequency and assessment of influence on irinotecan disposition. Clin Cancer Res. 2004;10:5889–5894. doi: 10.1158/1078-0432.CCR-04-0144.  [DOI](https://doi.org/10.1158/1078-0432.CCR-04-0144) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15355921/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&title=ABCG2%20pharmacogenetics:%20ethnic%20differences%20in%20allele%20frequency%20and%20assessment%20of%20influence%20on%20irinotecan%20disposition&author=FA%20de%20Jong&author=S%20Marsh&author=RHJ%20Mathijssen&author=C%20King&author=J%20Verweij&volume=10&publication_year=2004&pages=5889-5894&pmid=15355921&doi=10.1158/1078-0432.CCR-04-0144&)

33. Dong Z, Guo S, Yang Y, Wu J, Guan M, Zou H, Jin L, Wang J. Association between ABCG2 Q141K polymorphism and gout risk affected by ethnicity and gender: a systematic review and meta-analysis. Int J Rheum Dis. 2015;18:382–391. doi: 10.1111/1756-185X.12519.  [DOI](https://doi.org/10.1111/1756-185X.12519) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25639607/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Rheum%20Dis&title=Association%20between%20ABCG2%20Q141K%20polymorphism%20and%20gout%20risk%20affected%20by%20ethnicity%20and%20gender:%20a%20systematic%20review%20and%20meta-analysis&author=Z%20Dong&author=S%20Guo&author=Y%20Yang&author=J%20Wu&author=M%20Guan&volume=18&publication_year=2015&pages=382-391&pmid=25639607&doi=10.1111/1756-185X.12519&)

34. Furukawa T, Wakabayashi K, Tamura A, Nakagawa H, Morishima Y, Osawa Y, Ishikawa T. Major SNP (Q141K) variant of human ABC transporter ABCG2 undergoes lysosomal and proteasomal degradations. Pharm Res. 2009;26:469–479. doi: 10.1007/s11095-008-9752-7.  [DOI](https://doi.org/10.1007/s11095-008-9752-7) | [PMC free article](/articles/PMC2628956/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18958403/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharm%20Res&title=Major%20SNP%20(Q141K)%20variant%20of%20human%20ABC%20transporter%20ABCG2%20undergoes%20lysosomal%20and%20proteasomal%20degradations&author=T%20Furukawa&author=K%20Wakabayashi&author=A%20Tamura&author=H%20Nakagawa&author=Y%20Morishima&volume=26&publication_year=2009&pages=469-479&pmid=18958403&doi=10.1007/s11095-008-9752-7&)

35. de Wolf C, Jansen R, Yamaguchi H, de Haas M, van de Wetering K, Wijnholds J, Beijnen J, Borst P. Contribution of the drug transporter ABCG2 (breast cancer resistance protein) to resistance against anticancer nucleosides. Mol Cancer Ther. 2008;7:3092–3102. doi: 10.1158/1535-7163.MCT-08-0427.  [DOI](https://doi.org/10.1158/1535-7163.MCT-08-0427) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18765824/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Cancer%20Ther&title=Contribution%20of%20the%20drug%20transporter%20ABCG2%20(breast%20cancer%20resistance%20protein)%20to%20resistance%20against%20anticancer%20nucleosides&author=C%20de%20Wolf&author=R%20Jansen&author=H%20Yamaguchi&author=M%20de%20Haas&author=K%20van%20de%20Wetering&volume=7&publication_year=2008&pages=3092-3102&pmid=18765824&doi=10.1158/1535-7163.MCT-08-0427&)

36. Mizuarai S, Aozasa N, Kotani H. Single nucleotide polymorphisms result in impaired membrane localization and reduced atpase activity in multidrug transporter ABCG2. Int J Cancer. 2004;109:238–246. doi: 10.1002/ijc.11669.  [DOI](https://doi.org/10.1002/ijc.11669) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14750175/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Cancer&title=Single%20nucleotide%20polymorphisms%20result%20in%20impaired%20membrane%20localization%20and%20reduced%20atpase%20activity%20in%20multidrug%20transporter%20ABCG2&author=S%20Mizuarai&author=N%20Aozasa&author=H%20Kotani&volume=109&publication_year=2004&pages=238-246&pmid=14750175&doi=10.1002/ijc.11669&)

37. Chen J, Lu G, Lin J, Davidson AL, Quiocho FA. A tweezers-like motion of the ATP-binding cassette dimer in an ABC transport cycle. Mol Cell. 2003;12:651–661. doi: 10.1016/j.molcel.2003.08.004.  [DOI](https://doi.org/10.1016/j.molcel.2003.08.004) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14527411/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Cell&title=A%20tweezers-like%20motion%20of%20the%20ATP-binding%20cassette%20dimer%20in%20an%20ABC%20transport%20cycle&author=J%20Chen&author=G%20Lu&author=J%20Lin&author=AL%20Davidson&author=FA%20Quiocho&volume=12&publication_year=2003&pages=651-661&pmid=14527411&doi=10.1016/j.molcel.2003.08.004&)

38. Woodward OM, Tukaye DN, Cui J, Greenwell P, Constantoulakis LM, Parker BS, Rao A, Köttgen M, Maloney PC, Guggino WB. Gout-causing Q141K mutation in ABCG2 leads to instability of the nucleotide-binding domain and can be corrected with small molecules. Proc Natl Acad Sci U S A. 2013;110:5223–5228. doi: 10.1073/pnas.1214530110.  [DOI](https://doi.org/10.1073/pnas.1214530110) | [PMC free article](/articles/PMC3612674/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23493553/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&title=Gout-causing%20Q141K%20mutation%20in%20ABCG2%20leads%20to%20instability%20of%20the%20nucleotide-binding%20domain%20and%20can%20be%20corrected%20with%20small%20molecules&author=OM%20Woodward&author=DN%20Tukaye&author=J%20Cui&author=P%20Greenwell&author=LM%20Constantoulakis&volume=110&publication_year=2013&pages=5223-5228&pmid=23493553&doi=10.1073/pnas.1214530110&)

39. Govindarajan R, Leung GPH, Zhou M, Tse C-M, Wang J, Unadkat JD. Facilitated mitochondrial import of antiviral and anticancer nucleoside drugs by human equilibrative nucleoside transporter-3. Am J Physiol Gastrointest Liver Physiol. 2009;296:G910–G922. doi: 10.1152/ajpgi.90672.2008.  [DOI](https://doi.org/10.1152/ajpgi.90672.2008) | [PMC free article](/articles/PMC2670673/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19164483/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Physiol%20Gastrointest%20Liver%20Physiol&title=Facilitated%20mitochondrial%20import%20of%20antiviral%20and%20anticancer%20nucleoside%20drugs%20by%20human%20equilibrative%20nucleoside%20transporter-3&author=R%20Govindarajan&author=GPH%20Leung&author=M%20Zhou&author=C-M%20Tse&author=J%20Wang&volume=296&publication_year=2009&pages=G910-G922&pmid=19164483&doi=10.1152/ajpgi.90672.2008&)

40. Ebejer J-P, Hill JR, Kelm S, Shi J, Deane CM. Memoir: template-based structure prediction for membrane proteins. Nucleic Acids Res. 2013;41(Web Server issue):W379–W383. doi: 10.1093/nar/gkt331.  [DOI](https://doi.org/10.1093/nar/gkt331) | [PMC free article](/articles/PMC3692111/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23640332/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=Memoir:%20template-based%20structure%20prediction%20for%20membrane%20proteins&author=J-P%20Ebejer&author=JR%20Hill&author=S%20Kelm&author=J%20Shi&author=CM%20Deane&volume=41&issue=Web%20Server%20issue&publication_year=2013&pages=W379-W383&pmid=23640332&doi=10.1093/nar/gkt331&)

41. Ruiz van Haperen VW, Veerman G, Vermorken JB, Peters GJ. 2′,2′-Difluoro-deoxycytidine (gemcitabine) incorporation into RNA and DNA of tumour cell lines. Biochem Pharmacol. 1993;46:762–766. doi: 10.1016/0006-2952(93)90566-F.  [DOI](https://doi.org/10.1016/0006-2952(93)90566-F) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8363650/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem%20Pharmacol&title=2%E2%80%B2,2%E2%80%B2-Difluoro-deoxycytidine%20(gemcitabine)%20incorporation%20into%20RNA%20and%20DNA%20of%20tumour%20cell%20lines&author=VW%20Ruiz%20van%20Haperen&author=G%20Veerman&author=JB%20Vermorken&author=GJ%20Peters&volume=46&publication_year=1993&pages=762-766&pmid=8363650&doi=10.1016/0006-2952(93)90566-F&)

42. Bergman AM, Pinedo HM, Peters GJ. Determinants of resistance to 2′,2′-difluorodeoxycytidine (gemcitabine) Drug Resist Updat. 2002;5:19–33. doi: 10.1016/S1368-7646(02)00002-X.  [DOI](https://doi.org/10.1016/S1368-7646(02)00002-X) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12127861/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Resist%20Updat&title=Determinants%20of%20resistance%20to%202%E2%80%B2,2%E2%80%B2-difluorodeoxycytidine%20(gemcitabine)&author=AM%20Bergman&author=HM%20Pinedo&author=GJ%20Peters&volume=5&publication_year=2002&pages=19-33&pmid=12127861&doi=10.1016/S1368-7646(02)00002-X&)

43. Bianchi V, Spychala J. Mammalian 5′-Nucleotidases. J Biol Chem. 2003;278:46195–46198. doi: 10.1074/jbc.R300032200.  [DOI](https://doi.org/10.1074/jbc.R300032200) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12947102/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=Mammalian%205%E2%80%B2-Nucleotidases&author=V%20Bianchi&author=J%20Spychala&volume=278&publication_year=2003&pages=46195-46198&pmid=12947102&doi=10.1074/jbc.R300032200&)

44. Bogan KL, Brenner C. 5′-Nucleotidases and their new roles in NAD+ and phosphate metabolism. New J Chem. 2010;34:845–853. doi: 10.1039/b9nj00758j.  [DOI](https://doi.org/10.1039/b9nj00758j) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=New%20J%20Chem&title=5%E2%80%B2-Nucleotidases%20and%20their%20new%20roles%20in%20NAD+%20and%20phosphate%20metabolism&author=KL%20Bogan&author=C%20Brenner&volume=34&publication_year=2010&pages=845-853&doi=10.1039/b9nj00758j&)

45. Spychala J, Chen V, Oka J, Mitchell BS. ATP and phosphate reciprocally affect subunit association of human recombinant high km 5′-nucleotidase. Role for the C-terminal polyglutamic acid tract in subunit association and catalytic activity. Eur J Biochem. 1999;259:851–858. doi: 10.1046/j.1432-1327.1999.00099.x.  [DOI](https://doi.org/10.1046/j.1432-1327.1999.00099.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10092873/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Biochem&title=ATP%20and%20phosphate%20reciprocally%20affect%20subunit%20association%20of%20human%20recombinant%20high%20km%205%E2%80%B2-nucleotidase.%20Role%20for%20the%20C-terminal%20polyglutamic%20acid%20tract%20in%20subunit%20association%20and%20catalytic%20activity&author=J%20Spychala&author=V%20Chen&author=J%20Oka&author=BS%20Mitchell&volume=259&publication_year=1999&pages=851-858&pmid=10092873&doi=10.1046/j.1432-1327.1999.00099.x&)

46. Allegrini S, Scaloni A, Careddu MG, Cuccu G, D’Ambrosio C, Pesi R, Camici M, Ferrara L, Tozzi MG. Mechanistic studies on bovine cytosolic 5′-nucleotidase II, an enzyme belonging to the HAD superfamily. Eur J Biochem. 2004;271:4881–4891. doi: 10.1111/j.1432-1033.2004.04457.x.  [DOI](https://doi.org/10.1111/j.1432-1033.2004.04457.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15606776/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Biochem&title=Mechanistic%20studies%20on%20bovine%20cytosolic%205%E2%80%B2-nucleotidase%20II,%20an%20enzyme%20belonging%20to%20the%20HAD%20superfamily&author=S%20Allegrini&author=A%20Scaloni&author=MG%20Careddu&author=G%20Cuccu&author=C%20D%E2%80%99Ambrosio&volume=271&publication_year=2004&pages=4881-4891&pmid=15606776&doi=10.1111/j.1432-1033.2004.04457.x&)

47. Gu J, Xia X, Yan P, Liu H, Podust VN, Reynolds AB, Fanning E. Cell cycle-dependent regulation of a human DNA helicase that localizes in DNA damage foci. Mol Biol Cell. 2004;15:3320–3332. doi: 10.1091/mbc.e04-03-0227.  [DOI](https://doi.org/10.1091/mbc.e04-03-0227) | [PMC free article](/articles/PMC452586/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15146062/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Biol%20Cell&title=Cell%20cycle-dependent%20regulation%20of%20a%20human%20DNA%20helicase%20that%20localizes%20in%20DNA%20damage%20foci&author=J%20Gu&author=X%20Xia&author=P%20Yan&author=H%20Liu&author=VN%20Podust&volume=15&publication_year=2004&pages=3320-3332&pmid=15146062&doi=10.1091/mbc.e04-03-0227&)

48. Guler GD, Liu H, Vaithiyalingam S, Arnett DR, Kremmer E, Chazin WJ, Fanning E. Human DNA helicase B (HDHB) binds to replication protein a and facilitates cellular recovery from replication stress. J Biol Chem. 2012;287:6469–6481. doi: 10.1074/jbc.M111.324582.  [DOI](https://doi.org/10.1074/jbc.M111.324582) | [PMC free article](/articles/PMC3307258/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22194613/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=Human%20DNA%20helicase%20B%20(HDHB)%20binds%20to%20replication%20protein%20a%20and%20facilitates%20cellular%20recovery%20from%20replication%20stress&author=GD%20Guler&author=H%20Liu&author=S%20Vaithiyalingam&author=DR%20Arnett&author=E%20Kremmer&volume=287&publication_year=2012&pages=6469-6481&pmid=22194613&doi=10.1074/jbc.M111.324582&)

49. Ghosh A, Shuman S, Lima CD. The structure of Fcp1, an essential RNA polymerase II CTD phosphatase. Mol Cell. 2008;32:478–490. doi: 10.1016/j.molcel.2008.09.021.  [DOI](https://doi.org/10.1016/j.molcel.2008.09.021) | [PMC free article](/articles/PMC2645342/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19026779/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Cell&title=The%20structure%20of%20Fcp1,%20an%20essential%20RNA%20polymerase%20II%20CTD%20phosphatase&author=A%20Ghosh&author=S%20Shuman&author=CD%20Lima&volume=32&publication_year=2008&pages=478-490&pmid=19026779&doi=10.1016/j.molcel.2008.09.021&)

50. Jeong S-J, Kim H-J, Yang Y-J, Seol J-H, Jung B-Y, Han J-W, Lee H-W, Cho E-J. Role of RNA polymerase II carboxy terminal domain phosphorylation in DNA damage response. J Microbiol. 2005;43:516–522.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/16410768/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Microbiol&title=Role%20of%20RNA%20polymerase%20II%20carboxy%20terminal%20domain%20phosphorylation%20in%20DNA%20damage%20response&author=S-J%20Jeong&author=H-J%20Kim&author=Y-J%20Yang&author=J-H%20Seol&author=B-Y%20Jung&volume=43&publication_year=2005&pages=516-522&pmid=16410768&)

51. Chan SL, Jin S, Loh M, Brunham LR. Progress in understanding the genomic basis for adverse drug reactions: a comprehensive review and focus on the role of ethnicity. Pharmacogenomics. 2015:1–19.  [DOI](https://doi.org/10.2217/pgs.15.54) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25978008/)
