# Significant association of glutamate receptor, ionotropic N-methyl-D-aspartate 3A (GRIN3A), with nicotine dependence in European- and African-American smokers

## Metadata
**Authors:** Jennie Z Ma, Thomas J Payne, Ming D Li
**Journal:** Human genetics
**Date:** 2010 Jan 19
**DOI:** [10.1007/s00439-010-0787-6](https://doi.org/10.1007/s00439-010-0787-6)
**PMID:** 20084518
**PMCID:** PMC5573871
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5573871/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC5573871/pdf/nihms898153.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC5573871/pdf/nihms898153.pdf)

## Abstract

The glutamate receptor gene, ionotropic N-methyl-D-aspartate 3A (GRIN3A), is one of the seven that code for subunits of N-methyl-D-aspartate receptors, which play an essential role at many synapses in the brain, regulating ion flow across membranes in response to glutamate signaling. In this study, we analyzed 25 single nucleotide polymorphisms (SNPs) within GRIN3A for association with nicotine dependence (ND), which was assessed by smoking quantity, heaviness of smoking index, and the Fagerström test for ND. Both individual SNP and haplotype association tests were performed in African-American (AA) and European-American (EA) samples as well as in the pooled sample consisting of 2,037 individuals from 602 nuclear families. Individual SNP analysis revealed significant associations of 5, 5, and 4 SNPs with at least one ND measure in the pooled, EA, and AA samples, respectively. Of them, SNPs rs17189632 and rs10121600 in the pooled sample and rs11788456 in the EA sample remained significant after correction for multiple testing. On the basis of the blocks determined with Haploview, we performed haplo-type-based association analysis and found 2, 4, and 1 haplotype(s) that are significantly associated with at least one ND measure in the pooled, EA, and AA samples, respectively. Some of them remained significant after correction for multiple testing. We concluded that GRIN3A represents a strong candidate for involvement in the etiology of ND and warrants further investigation in independent samples.

## Introduction

Despite widespread public awareness of the catastrophic health effects of using tobacco products, approximately 438,000 American lives and more than $167 billion are lost to tobacco each year ([CDC 2007](#R6); [Mokdad et al. 2004](#R23)). Many tobacco users show a desire to quit, but the highly addictive nature of nicotine often makes their efforts futile. Numerous epidemiological studies have revealed the importance of genetic factors in determining individual liability to nicotine dependence (ND) ([Li et al. 2003a](#R16); [Sullivan and Kendler 1999](#R30)). This genetic significance highlights a need for better molecular understanding of addiction. Effective pharmaceutical treatments designed to target ND depend highly on an advanced understanding of genetic factors that influence the development and maintenance of addiction.

The glutamate receptor gene, ionotropic *N*-methyl-D-aspartate 3A (*GRIN3A*), consists of nine exons and localizes to 9q34, a region that has been associated with ND in several genome-wide linkage studies ([Bergen et al. 1999](#R3); [Bierut et al. 2004](#R5); [Gelernter et al. 2007](#R10); [Li et al. 2003b](#R17), [2006](#R19)). *GRIN3A* codes for glutamate *N*-methyl-D-aspartate (NMDA) receptor subunit 3A precursor, a 1,115-residue protein representing one of the subunits of *N*-methyl-D-aspartate receptors (NMDARs), which play an essential role at many synapses in the brain by regulating ion flow across membranes in response to glutamate signaling. Glutamate, an abundant amino acid throughout the body, is among the most common neurotransmitters in the human central nervous system. The NMDARs consist of heteromeric assemblies composed from a pool of three homologous subunits: NR1, NR2, and NR3. The NR2 subunit specifically binds glutamate, whereas NR1 and NR3 bind glycine, an agonist essential for NMDAR function. With eight NR1 splice variants from a single gene, four NR2 subunits (NR2A-D) from four independent genes, and two NR3 subunits (NR3A-B) from two genes, there is a large pool of components from which many NMDAR variants can be assembled ([Andersson et al. 2001](#R1)). Each functional receptor is thought to consist most often of a minimum of two NR1 and two NR2 subunits. Initially, NR3 was thought to be expressed only in certain cell types ([Furukawa et al. 2005](#R8)); however, a recent study ([Nilsson et al. 2007](#R24)) showed that NR3A is distributed throughout much of the human forebrain. Further, this study showed that both NR3A and NR1 bind glycine, although with significantly different affinities and solubilities. This compositional and functional diversity has provided challenges for understanding the function of NMDARs, as it has been difficult to ascribe specific functional variations to particular changes in subunit composition ([Paoletti and Neyton 2007](#R27)). This diversity, however, also makes delivering drugs to specific NMDARs more plausible.

Although the details of specific NMDAR subtypes have not been completely uncovered, much progress has been made in understanding their general functions. The most noticeable feature of NMDARs is their high permeability to calcium ions. This function is thought to be directly related to synaptic plasticity under physiologic conditions, as well as neuronal death during excitotoxic pathological conditions ([Paoletti and Neyton 2007](#R27)).

Of specific interest in the studies described in this communication is the function of the NR3 subunits. The NR1 and NR2 sequences that line the inside of the channel pore are highly conserved, and ion permeability varies little between subtypes. However, receptors that incorporate NR3 show a substantial decrease in single-channel conductance and Ca^++^ permeability ([Furukawa et al. 2005](#R8)). This implies that their incorporation is used to induce significant variation in the basic properties of the receptor.

Given the plausible biological function of *GRIN3A* and its chromosomal location in a region linked to ND, it is of great interest to determine whether *GRIN3A* is associated with ND. Twenty-five SNPs selected uniformly from the whole gene were examined for possible association with ND in a dataset consisting of 2,037 individuals from 602 nuclear families of either EA or AA ancestry, with ND being assessed by three commonly used ND measures: smoking quantity (SQ), heaviness of smoking index (HSI), and the Fagerström test for ND (FTND).

## Materials and methods

### Participants and ND measures

Participants in this family-based association study were recruited primarily from the Mid-South states in the US between 1999 and 2004. Criteria for eligible proband smokers consisted of: being at least 18 years of age, having consumed an average of 20 cigarettes per day for 12 months prior to recruitment, having smoked for a minimum of five consecutive years, and being free of any current psychiatric diagnosis with the exception of alcohol use/abuse. To perform a family-based genetic study, we also recruited as many as possible the biological parents and sibling of the probands regardless of their smoking status. Data collected from all the participants included demographics such as age, sex, race, biological relationships, weight, height, education, and marital status. Data related to medical histories, current and past smoking behavior, and personality traits were collected using various questionnaires that are available at the NIDA Genetics Consortium website ([http://zork.wustl.edu/nida](http://zork.wustl.edu/nida)). All participants provided informed consent, and the study protocol and forms were approved by all participating Institutional Review Boards.

To assess the ND status of each smoker, three common scales were employed: SQ, HSI (0–6 scale), and FTND (0–10 scale) ([Heatherton et al. 1991](#R12)). As there is no consensus as which measure provides the best overall assessment of ND, we performed association analysis on the three measures at both the individual SNP and haplotype levels with the goal of achieving different insights that selecting one specific measure cannot. Briefly, the SQ is a straightforward index of total nicotine consumption, whereas HSI also assesses smoking urgency (i.e., “How soon after you wake up do you smoke your first cigarette?”). The FTND includes the HSI factors, but adds other behavioral indicators that relate smoking to specific circumstances. Furthermore, the use of multiple measures allows cross-referencing with other studies, regardless of their chosen measure. Given the overlaps among the three measures, there exists a fairly robust correlation between them, ranging from 0.88 to 0.94 for three populations ([Beuten et al. 2005](#R4); [Li et al. 2005](#R18); [Ma et al. 2005](#R21)). Thus, we prefer not to correct for them while we perform correction for multiple testing.

The sample consisted of 2,037 participants from 602 nuclear families, with 1,366 individuals from 402 AA families and 671 individuals from 200 EA families. The average participant age was 39.4 ± 14.4 (SD) years for the AA sample and 40.5 ± 15.5 years for the EA sample. The average nuclear family size was 3.14 ± 0.75 for AAs and 3.17 ± 0.69 for EAs. The average HSI and FTND scores were 3.7 ± 1.4 and 6.25 ± 2.15 for AA smokers and 3.9 ± 1.4 and 6.33 ± 2.22 for EA smokers, respectively. The average number of cigarettes smoked per day by AA and EA smokers was 19.4 ± 13.33 and 19.5 ± 13.4, respectively. For detailed information on the clinical characteristics of the samples used in the study, please refer to [Table 1](#T1).

### Table 1.

| Characteristic | African-Americans | European-Americans | Pooled |
| --- | --- | --- | --- |
| No. of nuclear families | 402 | 200 | 602 |
| Average no. of members/family | 3.14 ± 0.75 | 3.17 ± 0.69 | 3.15 ± 0.73 |
| No. of subjects | 1,366 | 671 | 2,037 |
| Females (%) | 66.1 | 69.5 | 67.2 |
| Age (years) | 39.4 ± 14.4 | 40.5 ± 15.5 | 39.7 ± 14.8 |
| No. of smokers | 1,053 | 515 | 1,568 |
| Age of smoking onset | 17.3 ± 4.7 | 15.5 ± 4.4 | 16.7 ± 4.7 |
| Years smoked | 20.4 ± 12.5 | 23.2 ± 13.5 | 21.3 ± 12.9 |
| Smoking quantity/day | 19.4 ± 13.3 | 19.5 ± 13.4 | 19.5 ± 13.3 |
| HSI | 3.7 ± 1.4 | 3.9 ± 1.4 | 3.8 ± 1.4 |
| FTND score | 6.26 ± 2.15 | 6.33 ± 2.22 | 6.29 ± 2.17 |

Table 1 Caption: Clinical characteristics of the African-American, European-American, and pooled samples

### DNA extraction, SNP selection, and SNP genotyping

The DNA was extracted from participant blood samples using the QIAamp^©^ DNA Blood Maxi Kit, purchased from Qiagen Inc (Valencia, CA, USA). The SNP selection was based on the high heterozygosity with a minor allele frequency (MAF) >0.05 (as listed in the NCBI dbSNP database and shown in the last two columns of [Table 2](#T2)), potential biological functions within a gene, and optimal coverage of the gene. General information regarding each genotyped SNP is shown in [Table 2](#T2).

### Table 2.

| SNP number | dbSNP ID | Alleles | Chromosome position | SNP location | SNP function | Reported minor allele frequency |  |
| --- | --- | --- | --- | --- | --- | --- | --- |
|   |  |  |  |  |  |  |  |
| YRI | CEU |  |  |  |  |  |  |
| 1 | rs7030238 | A/C | 103372316 | Exon 9 | 3′-UTR | 0.283 (C) | 0.192 (C) |
| 2 | rs10512282 | G/T | 103374154 | Exon 9 | 3′-UTR | 0.067 (G) | 0.075 (G) |
| 3 | rs3739722 | A/G | 103375503 | Exon 9 | Arg1041Gln | 0.200 (A) | 0.125 (A) |
| 4 | rs11788456 | A/G | 103387971 | Intron 6 |   | 0.492 (G) | 0.467 (G) |
| 5 | rs3739723 | A/T | 103394171 | Intron 6 |   | 0.144 (A) | 0.070 (A) |
| 6 | rs1407877 | A/G | 103395276 | Intron 6 |   | 0.183 (A) | 0.117 (A) |
| 7 | rs17189632 | A/T | 103407823 | Intron 6 |   | 0.292 (A) | 0.450 (A) |
| 8 | rs729688 | A/C | 103412287 | Intron 6 |   | 0.350 (C) | 0.458 (C) |
| 9 | rs10121600 | C/T | 103417824 | Intron 5 |   | 0.492 (C) | 0.350 (T) |
| 10 | rs1570514 | A/C | 103428684 | Intron 4 |   | 0.417 (C) | 0.408 (A) |
| 11 | rs10989573 | A/G | 103438865 | Intron 3 |   | 0.45 (G) | 0.467 (A) |
| 12 | rs1016428 | C/T | 103441356 | Intron 3 |   | 0.417 (C) | 0.467 (T) |
| 13 | rs1005682 | C/T | 103450724 | Intron 3 |   | 0.475 (T) | 0.242 (T) |
| 14 | rs1924032 | A/G | 103457633 | Intron 3 |   | 0.417 (G) | 0.442 (G) |
| 15 | rs942142 | A/C | 103472694 | Exon 3 | Ala607Ala | 0.154 (C) | 0.404 (C) |
| 16 | rs10512285 | A/G | 103472855 | Exon 3 | Leu554Leu | 0.110 (G) | 0.430 (G) |
| 17 | rs10989589 | C/T | 103473056 | Exon 3 | Gly487Arg | 0.100 (T) | 0.358 (T) |
| 18 | rs2506354 | C/T | 103488884 | Exon 2 | Gly373Gly | 0.125 (C) | 0.158 (C) |
| 19 | rs2485533 | C/T | 103489719 | Intron 1 |   | 0.408 (C) | 0.457 (T) |
| 20 | rs10819978 | C/T | 103504006 | Intron 1 |   | 0.475 (T) | 0.458 (T) |
| 21 | rs2485528 | C/T | 103509088 | Intron 1 |   | 0.375 (C) | 0.467 (T) |
| 22 | rs10819979 | G/T | 103514059 | Intron 1 |   | 0.183 (T) | 0.392 (T) |
| 23 | rs2050640 | C/T | 103529305 | Intron 1 |   | 0.425 (T) | 0.475 (C) |
| 24 | rs1415644 | C/T | 103533587 | Intron 1 |   | 0.133 (T) | 0.183 (T) |
| 25 | rs2067056 | C/T | 103541388 | 5′ near gene |   | 0.467 (C) | 0.450 (T) |

Table 2 Caption: SNPs genotyped in the present study

Of the 25 SNPs, eight (i.e., rs3739722, rs3739723, rs729688, rs1016428, rs1924032, rs10512285, and rs2485528) were genotyped using a *Taq*Man SNP Genotyping Assay purchased from Applied Biosystems (Foster City, CA, USA). The remaining 17 SNPs were genotyped using the Illumina BeadChip system at the Center for Inherited Disease Research (CIDR) at Johns Hopkins University. All reactions for the *Taq*Man assay were run in 384-well plates, with a total reaction volume of 7 μl. Each reaction mixture contained an MGB probe, primers, 15 ng of template DNA, and 2.5 μl of *Taq*Man Universal PCR Mastermix. The PCR cycling conditions were 50°C for 2 min, 95°C for 10 min, and 40 cycles of 95°C for 25 s and 60°C for 1 min. Allelic discrimination analysis was carried out with an ABI Prism 7900HT Sequence Detection System (Applied Biosystems). Each plate was quality-controlled with four no-template controls, as well as eight DNA-containing positive controls, which were monitored for consistency.

### LD and association analysis

We used the PedCheck program ([O’Connell and Weeks 1998](#R26)) to detect genotyping inconsistencies for Mendelian inheritance. One hundred twenty-two inconsistencies, with 81 in the AA sample and 41 in the EA sample, were detected in approximately 51,000 assays (i.e., 0.24% genotyping error) for the 25 SNPs across all DNA samples and were excluded from subsequent statistical analysis. The program Haploview ([Barrett et al. 2005](#R2)) was used to determine pair-wise linkage disequilibrium (LD) between all SNPs using the option of determining blocks on the basis of the criteria defined by [Gabriel et al. (2002)](#R9).

The three phenotypic ND measures discussed earlier were used individually to assess the association of ND with all 25 SNPs using the PBAT program ([Lange et al. 2003](#R14)). Age and sex were used as covariates within the PBAT program because of evidence that they influence ND differently ([Edwards et al. 1995](#R7); [Li et al. 2003a](#R16); [Madden et al. 1999](#R22); [Perez-Stable et al. 1998](#R28); [Perkins et al. 1999](#R29)). The FBAT ([Horvath et al. 2004](#R13)) program was used to measure association between each haplotype and the three ND measures in the pooled, EA, and AA samples separately. For both individual SNP and haplotype analysis, only the additive genetic model was examined. All associations designated significant were corrected for multiple testing using the SNP spectral decomposition (SNPSpD) method ([Nyholt 2004](#R25)) for individual SNP analysis. The SNPSpD method was favored because Bonferroni correction tends to be too conservative and overcorrects for inflated false-positive rates, resulting in a reduction of power. Bonferroni correction was used for haplotype-based association analysis for major haplotypes (>5% frequency), however, as the SNPSpD method cannot handle correction for haplotypes.

## Results

### Association analysis of individual SNPs

Previous studies have shown that genotypic differences exist between various ethnic populations for some smoking-related genes ([Li 2008](#R15); [Li et al. 2005](#R18), [2008](#R20)). Although significant differences between ethnic samples are less obvious for most of the SNPs investigated here, examining individual populations can provide a clear view of the association of the gene with ND in each ethnic sample and give insight into the evolutionary history of a gene. Therefore, all association analyses were performed separately on AA and EA samples, and the results for significantly associated SNPs with ND measures in each ethnic sample are provided in [Table 3](#T3). To increase statistical power and illustrate the conserved nature of the gene, [Table 3](#T3) also shows the association results for the pooled sample.

### Table 3.

| dbSNP ID | Risk/minor allele | Pooled sample |  |  |  | European-American sample |  |  |  | African-American sample |  |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
|   |  |  |  |   |  |  |  |   |  |  |  |  |  |
| MAF | SQ | HSI | FTND | MAF | SQ | HSI | FTND | MAF | SQ | HSI | FTND |  |  |
| rs7030238 | C/C | 0.286 | 0.0030 | 0.0033 | 0.0044 | 0.226 | 0.1047 | 0.0873 | 0.2407 | 0.317 | 0.0145 | 0.0163 | 0.0100 |
| rs10512282 | G/G | 0.082 | 0.0256 | 0.0500 | 0.0683 | 0.091 | 0.0456 | 0.0419 | 0.0549 | 0.078 | 0.2055 | 0.349 | 0.3965 |
| rs11788456 | G/G | 0.451 | 0.0674 | 0.2436 | 0.1963 | 0.455 | 0.0016 | 0.0102 | 0.0014 | 0.449 | 0.8465 | 0.869 | 0.690 |
| rs17189632 | T/A | 0.389 | 0.0003 | 0.0003 | 0.0002 | 0.446 | 0.0042 | 0.0094 | 0.0129 | 0.359 | 0.0103 | 0.0071 | 0.0037 |
| rs10121600 | C/a | 0.481 (T) | 0.0005 | 0.0032 | 0.0050 | 0.358 (T) | 0.0039 | 0.0367 | 0.0383 | 0.454 (C) | 0.0218 | 0.0288 | 0.0428 |
| rs1924032 | A/G | 0.491 | 0.0435 | 0.1164 | 0.1270 | 0.498 | 0.4877 | 0.8943 | 0.6218 | 0.486 | 0.0477 | 0.0889 | 0.1495 |
| rs2067056 | C/a | 0.418 (C) | 0.0911 | 0.3091 | 0.2883 | 0.496 (T) | 0.0236 | 0.1401 | 0.0561 | 0.372 (C) | 0.5822 | 0.7873 | 0.9348 |

Table 3 Caption: Probability (p) values for risk alleles significantly associated with at least one ND measure under the additive model in pooled, European-American, and African-American samples

Individual SNP analysis yielded five significant associations under the additive model in the pooled sample. Of them, three SNPs remained significant after correction for multiple tests for the number of SNPs tested in this study, and the same principle for all multiple corrections was used throughout the whole study. The ‘T’ allele of SNP rs17189632 showed a significant association with all the three ND measures (*p* = 0.0002–0.0003; [Table 3](#T3)) as did the ‘C’ allele of SNPs rs7030238 and rs10121600 with SQ and HSI (*p* = 0.0033–0.0005), respectively. In the EA sample, we also found that five SNPs were significantly associated with at least one ND measure; however, only rs11788456 remained significant with SQ (*p* = 0.0016) and FTND (*p* = 0.0014) after correction for multiple testing ([Table 3](#T3)). As for the AA sample, although we found significant associations of four SNPs (rs7030238, rs17189632, rs10121600, and rs1924032) with at least one ND measure, only the association of the ‘T’ allele of rs17189632 with FTND remained significant after correction for multiple testing (*p* = 0.0037).

### LD structure and haplotype-based association analysis

Pair-wise *D*′ values for all SNPs were calculated with the Haploview program ([Barrett et al. 2005](#R2)), and all blocks were defined as described by [Gabriel et al. (2002)](#R9). A total of seven blocks were detected in the pooled sample and consist of the following SNPs: rs7030238 and rs10512282 (Block 1); rs11788456 and rs3739723 (Block 2); rs1407877 and rs17189632 (Block 3); rs1570514 and rs10989573 (Block 4); rs1924032, rs942142, rs10512285 and rs10989589 (Block 5); rs2485533 and rs10819978 (Block 6); and rs10819979, rs2050640, and rs1415644 (Block 7). Blocks were also calculated using *R*^2^ values, which resulted in identical block structures. Similarly, we performed LD analysis using Haploview on each ethnic sample individually, and a total of six blocks was identified for each sample. A detailed LD structure and their corresponding block information are shown in [Fig. 1](#F1).

### Fig. 1.

![Fig. 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6a09/5573871/a893794458e2/nihms898153f1a.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5573871_nihms898153f1a.jpg)

LD structures for 25 SNPs in GRIN3A in the pooled (a), EA (b), and AA (c) samples. Haploview (v. 3.2) (Barrett et al. 2005) was used to calculate all D′ values, and haplotype blocks were defined according to Gabriel et al. (2002). The number in each box represents the D′ value for each SNP pair surrounding that box

On the basis of the blocks determined by Haploview, we performed haplotype-based association analysis with three ND measures under the additive model for each block separately in each sample ([Table 4](#T4)). In the pooled sample, we found that Blocks 1 and 3, located at the 3′-UTR of the gene, showed significant association with all three ND measures; their corresponding global *p* value ranged from 0.0026 to 0.0035 for Block 1 and from 0.0022 to 0.0047 for Block 3. For Block 1, we detected three major haplotypes, with the first one (i.e., A–T with a frequency of 73.6%) showing a significant positive association with all three ND measures (*Z* = 2.941–3.038; *p* = 0.0033–0.0024), while the other two (i.e., C–T and C–G with a frequency of 19.4 and 7%, respectively) showed significant inverse associations with at least two of the three ND measures. However, only the associations of haplotypes A–T with the three ND measures and C–G with SQ remain significant after correction for multiple testing. For Block 3, we detected two significant ND-associated haplotypes, with the first one (i.e., G–T with a frequency of 48.7%) being inversely associated with HSI (*Z* = −1.987; *p* = 0.0469) and FTND (*Z* = −1.989; *p* = 0.0468) and the second one (i.e., G–A with a frequency of 37.2%) being positively associated with the three ND measures (*Z* = 3.194–3.410; *p* = 0.0014–0.0006).

### Table 4.

| SNP combinations/haplotype blocks | Frequency | SQ |  |  | HSI |  |  | FTND |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
|   |  |  |   |  |  |   |  |  |  |  |
| No. of families | Z | p | No. of families | Z | p | No. of families | Z | p |  |  |
| (a) Pooled sample |  |  |  |  |  |  |  |  |  |  |
| rs7030238–rs10512282 (Block 1) |  |  |  |  |  |  |  |  |  |  |
| A–T | 0.736 | 291 | 3.024 | 0.0025 | 306 | 3.038 | 0.0024 | 313 | 2.941 | 0.0033 |
| C–T | 0.194 | 244 | −1.914 | 0.0556 | 256 | −2.096 | 0.0361 | 262 | −2.144 | 0.0345 |
| C–G | 0.070 | 108 | −2.490 | 0.0128 | 112 | −2.163 | 0.0305 | 114 | −1.949 | 0.0513 |
| Global p value |   |   |   | 0.0026 |   |   | 0.0026 |   |   | 0.0035 |
| rs1407877–rs17189632 (Block 3) |  |  |  |  |  |  |  |  |  |  |
| G–T | 0.487 | 303 | −1.900 | 0.0574 | 318 | −1.987 | 0.0469 | 322 | −1.989 | 0.0468 |
| G–A | 0.372 | 296 | 3.194 | 0.0014 | 314 | 3.205 | 0.0014 | 317 | 3.410 | 0.0006 |
| A–T | 0.140 | 194 | −1.596 | 0.1105 | 201 | −1.473 | 0.1407 | 203 | −1.702 | 0.0888 |
| Global p value |   |   |   | 0.0038 |   |   | 0.0047 |   |   | 0.0022 |
| (b) European-American sample |  |  |  |  |  |  |  |  |  |  |
| rs7030238–rs10512282 (Block 1) |  |  |  |  |  |  |  |  |  |  |
| A–T | 0.799 | 73 | 1.995 | 0.0460 | 75 | 2.076 | 0.0379 | 78 | 1.477 | 0.1395 |
| C–T | 0.121 | 48 | −0.526 | 0.5986 | 50 | −0.563 | 0.5732 | 53 | 0.126 | 0.8998 |
| C–G | 0.080 | 39 | −2.200 | 0.0278 | 39 | −2.200 | 0.0278 | 39 | −2.036 | 0.0418 |
| Global p value |   |   |   | 0.0667 |   |   | 0.0611 |   |   | 0.0900 |
| rs729688–rs10121600 (Block 2) |  |  |  |  |  |  |  |  |  |  |
| C–C | 0.451 | 79 | −1.716 | 0.0861 | 81 | −0.762 | 0.4461 | 83 | −1.354 | 0.1757 |
| A–T | 0.360 | 78 | 3.087 | 0.0020 | 81 | 2.324 | 0.0201 | 82 | 2.322 | 0.0202 |
| A–C | 0.179 | 57 | −2.270 | 0.0232 | 58 | −2.134 | 0.0328 | 63 | −1.504 | 0.1326 |
| Global p value |   |   |   | 0.0026 |   |   | 0.0431 |   |   | 0.0422 |
| rs1924032–rs942142–rs10512285–rs10989589 (Block 4) |  |  |  |  |  |  |  |  |  |  |
| A–A–A–C | 0.508 | 84 | −1.034 | 0.3011 | 86 | −0.567 | 0.5707 | 87 | −0.802 | 0.4225 |
| G–C–G–T | 0.387 | 89 | 1.835 | 0.0665 | 93 | 1.499 | 0.1338 | 92 | 1.736 | 0.0825 |
| G–A–A–C | 0.097 | 37 | −2.029 | 0.0425 | 41 | −2.023 | 0.0431 | 42 | −2.139 | 0.0324 |
| Global p value |   |   |   | 0.0971 |   |   | 0.1603 |   |   | 0.0951 |
| rs10819979–rs2050640–rs1415644 (Block 6) |  |  |  |  |  |  |  |  |  |  |
| G–T–C | 0.465 | 81 | −1.838 | 0.0660 | 81 | −1.065 | 0.2867 | 84 | −1.576 | 0.1150 |
| T–C–C | 0.250 | 59 | 2.279 | 0.0227 | 63 | 1.981 | 0.0476 | 64 | 2.093 | 0.0363 |
| T–C–T | 0.175 | 58 | −0.960 | 0.3369 | 59 | −1.273 | 0.2029 | 61 | −0.733 | 0.4636 |
| G–C–C | 0.109 | 34 | 0.562 | 0.5741 | 36 | 0.085 | 0.9320 | 38 | 0.165 | 0.8690 |
| Global p value |   |   |   | 0.0301 |   |   | 0.0767 |   |   | 0.0580 |
| (c) African-American sample |  |  |  |  |  |  |  |  |  |  |
| rs7030238–rs10512282 (Block 1) |  |  |  |  |  |  |  |  |  |  |
| A–T | 0.704 | 220 | 2.384 | 0.0171 | 230 | 2.387 | 0.0170 | 233 | 2.556 | 0.0106 |
| C–T | 0.231 | 196 | −1.852 | 0.0641 | 206 | −2.022 | 0.0432 | 209 | −2.272 | 0.0231 |
| C–G | 0.065 | 69 | −1.467 | 0.1423 | 72 | −1.082 | 0.2794 | 74 | −0.914 | 0.3606 |
| Global p value |   |   |   | 0.0234 |   |   | 0.0226 |   |   | 0.0105 |

Table 4 Caption: Haplotype-based association analysis results with three ND measures under additive genetic model for the pooled (a), European-American (b), and African-American (c) samples

For the EA sample, we found four blocks containing at least one haplotype that showed significant association with one of the three ND measures ([Table 4b](#T4)). For example, in Block 2, formed by SNPs rs729688 and rs10121600, we found two major haplotypes that showed significant association with ND. The first haplotype, A–T with a frequency of 36%, showed a significant association with all three ND measures (*Z* = 2.322–3.087; *p* = 0.0202–0.0020). The second haplotype, A–C with a frequency of 17.9%, showed significant inverse association with SQ (*Z* = −2.270; *p* = 0.0232) and HSI (*Z* = −2.134; *p* = 0.0328). Detailed results on other blocks can be seen in [Table 4](#T4) as well.

Regarding the AA sample, only Block 1, formed by SNPs rs7030238 and rs10512282, showed a significant association with ND under the additive model ([Table 4c](#T4)). This block contains three major haplotypes, with the first one, A–T (with a frequency of 70.4%), showing significant association with the three ND measures (*Z* = 2.384–2.556; *p* = 0.0171–0.0106) and the second one, C–T (with a frequency of 23.1), showing significant inverse association with HSI (*Z* = −2.022; *p* = 0.0432) and FTND (*Z* = −2.272; *p* = 0.0231). The third haplotype, C–G with a frequency of 6.5%, showed no significant association with any ND measure.

## Discussion

The fundamental roles that *GRIN3A* plays in general nervous system function by way of NMDARs, along with its location in a chromosomal region linked to smoking behavior, make this gene a plausible candidate for a role in the pathology of ND and other neurological diseases. To test for association with ND, 25 SNPs within the *GRIN3A* gene were genotyped for 2,037 individuals from 602 nuclear families of either AA or EA origin. Three measures of ND were used to phenotype individuals in the study, and associations for each of these phenotypes were performed for all SNPs as well as all haplotypes within each block determined specifically for the samples investigated here. By calculating the association in both the EA and AA samples as well as in the pooled sample, we were able to check for potential population stratification between the two ethnic samples as well as to show the combined association in the pooled sample.

Association analysis of 25 SNPs within *GRIN3A* revealed multiple significant individual SNP and haplotype associations with ND. Individual SNP analysis revealed a significant association of five SNPs in the pooled and EA samples and of four SNPs in the AA sample with multiple ND measures under the additive model. Especially, we found SNPs rs17189632 to be associated with all the ND measures and rs10121600 with SQ in the pooled sample following correction for multiple testing. Both of these SNPs showed significant associations with the three ND measures in the AA and EA samples prior to correction for multiple tests. This result highlights the conserved nature of the gene, as combining the EA and AA samples into one pooled dataset increases the power of the test and yields highly significant associations. In addition, we found rs11788456 to be significantly associated with all the three ND measures in the EA sample, and it remained significant for SQ and FTND after correction for multiple testing.

Although individual SNP testing yielded significant associations for various numbers of SNPs in all the three samples, haplotype-based association analysis within each block yielded significant associations of two blocks in the pooled sample, four blocks in the EA sample, and one block in the AA sample with multiple ND measures. Although the number of blocks associated with ND differed among samples, we found Block 1, formed by rs7030238 and rs10512282 located at the 3′-UTR of the gene, to be significantly associated with ND in all the three samples, suggesting this region likely harbors functional SNP(s) and warrants further investigation at the molecular level such as through deep sequencing analysis of this region.

These significant statistical associations complement the established role of *GRIN3A* in regulating neural transmission, marking the gene as a strong candidate for involvement in the etiology of ND. Our study is complementary to others that have implicated *GRIN3A* in schizophrenia and bipolar disorder ([Gulli et al. 2007](#R11)). Although *GRIN3A* has not been associated with ND per se, the glutamate receptor signaling pathway the gene belongs to has been associated with smoking cessation in a pathway-based association analysis of smoking behavior ([Wang and Li 2010](#R33)). The genes in this glutamate signaling pathway that have been associated with other smoking behaviors such as initiation and cessation in genome-wide association studies include *GRIN2A*, *GRIN2B*, *GRINK1*, and *GRINK2*, to name a few ([Uhl et al. 2008](#R31); [Vink et al. 2009](#R32)). These findings provide further evidence for the involvement of *GRIN3A* in the etiology of ND, supporting the conclusions drawn from this report.

In summary, our results indicate that *GRIN3A* is significantly associated with ND. This significant statistical association at both the individual SNP and the haplotype level, along with *GRIN3A*’s central role in regulating neurotransmission and its location in a region linked to smoking behavior, makes the gene a strong candidate for future investigation into the molecular mechanisms that underlie ND. Future understanding of the role of this gene, especially in the 3′-UTR region, may provide the insight necessary to target ND on a molecular level and reduce or eliminate the addictive properties of nicotine that plague millions of individuals.

## Acknowledgments

We are grateful for the invaluable contributions of clinical information and blood by all participants in the genetic study, as well as the dedicated work of many research staff at different clinical sites. This project was funded by National Institutes of Health grants DA-12844 and DA-13783.

## Contributor Information

Jennie Z. Ma, Department of Public Health Sciences, University of Virginia, P. O. Box 800717, Charlottesville, VA 22908, USA

Thomas J. Payne, Department of Otolaryngology, ACT Center for Tobacco Treatment, University of Mississippi Medical Center, Jackson, MS 39216, USA

Ming D. Li, Department of Psychiatry and Neurobehavioral Sciences, University of Virginia, Charlottesville, VA 22911, USA

## References

1. Andersson O, Stenqvist A, Attersand A, von Euler G. Nucleotide sequence, genomic organization, and chromosomal localization of genes encoding the human NMDA receptor subunits NR3A and NR3B. Genomics. 2001;78:178–184. doi: 10.1006/geno.2001.6666.  [DOI](https://doi.org/10.1006/geno.2001.6666) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11735224/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genomics&title=Nucleotide%20sequence,%20genomic%20organization,%20and%20chromosomal%20localization%20of%20genes%20encoding%20the%20human%20NMDA%20receptor%20subunits%20NR3A%20and%20NR3B&author=O%20Andersson&author=A%20Stenqvist&author=A%20Attersand&author=G%20von%20Euler&volume=78&publication_year=2001&pages=178-184&pmid=11735224&doi=10.1006/geno.2001.6666&)

2. Barrett JC, Fry B, Maller J, Daly MJ. Haploview: analysis and visualization of LD and haplotype maps. Bioinformatics. 2005;21:263–265. doi: 10.1093/bioinformatics/bth457.  [DOI](https://doi.org/10.1093/bioinformatics/bth457) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15297300/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=Haploview:%20analysis%20and%20visualization%20of%20LD%20and%20haplotype%20maps&author=JC%20Barrett&author=B%20Fry&author=J%20Maller&author=MJ%20Daly&volume=21&publication_year=2005&pages=263-265&pmid=15297300&doi=10.1093/bioinformatics/bth457&)

3. Bergen AW, Korczak JF, Weissbecker KA, Goldstein AM. A genome-wide search for loci contributing to smoking and alcoholism. Genet Epidemiol. 1999;17(Suppl 1):S55–S60. doi: 10.1002/gepi.1370170710.  [DOI](https://doi.org/10.1002/gepi.1370170710) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10597412/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genet%20Epidemiol&title=A%20genome-wide%20search%20for%20loci%20contributing%20to%20smoking%20and%20alcoholism&author=AW%20Bergen&author=JF%20Korczak&author=KA%20Weissbecker&author=AM%20Goldstein&volume=17&issue=Suppl%201&publication_year=1999&pages=S55-S60&pmid=10597412&doi=10.1002/gepi.1370170710&)

4. Beuten J, Ma JZ, Payne TJ, Dupont RT, Quezada P, Huang W, Crews KM, Li MD. Significant association of BDNF haplotypes in European-American male smokers but not in European-American female or African-American smokers. Am J Med Genet B Neuropsychiatr Genet. 2005;139:73–80. doi: 10.1002/ajmg.b.30231.  [DOI](https://doi.org/10.1002/ajmg.b.30231) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16152573/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Med%20Genet%20B%20Neuropsychiatr%20Genet&title=Significant%20association%20of%20BDNF%20haplotypes%20in%20European-American%20male%20smokers%20but%20not%20in%20European-American%20female%20or%20African-American%20smokers&author=J%20Beuten&author=JZ%20Ma&author=TJ%20Payne&author=RT%20Dupont&author=P%20Quezada&volume=139&publication_year=2005&pages=73-80&pmid=16152573&doi=10.1002/ajmg.b.30231&)

5. Bierut LJ, Rice JP, Goate A, Hinrichs AL, Saccone NL, Foroud T, Edenberg HJ, Cloninger CR, Begleiter H, Conneally PM, Crowe RR, Hesselbrock V, Li TK, Nurnberger JI, Jr, Porjesz B, Schuckit MA, Reich T. A genomic scan for habitual smoking in families of alcoholics: common and specific genetic factors in substance dependence. Am J Med Genet. 2004;124A:19–27. doi: 10.1002/ajmg.a.20329.  [DOI](https://doi.org/10.1002/ajmg.a.20329) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14679582/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Med%20Genet&title=A%20genomic%20scan%20for%20habitual%20smoking%20in%20families%20of%20alcoholics:%20common%20and%20specific%20genetic%20factors%20in%20substance%20dependence&author=LJ%20Bierut&author=JP%20Rice&author=A%20Goate&author=AL%20Hinrichs&author=NL%20Saccone&volume=124A&publication_year=2004&pages=19-27&pmid=14679582&doi=10.1002/ajmg.a.20329&)

6. CDC. Cigarette smoking among adults—United States, 2006. MMWR Morb Mortal Wkly Rep. 2007;56:1157–1161.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/17989644/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=MMWR%20Morb%20Mortal%20Wkly%20Rep&title=Cigarette%20smoking%20among%20adults%E2%80%94United%20States,%202006&volume=56&publication_year=2007&pages=1157-1161&pmid=17989644&)

7. Edwards KL, Austin MA, Jarvik GP. Evidence for genetic influences on smoking in adult women twins. Clin Genet. 1995;47:236–244. doi: 10.1111/j.1399-0004.1995.tb04303.x.  [DOI](https://doi.org/10.1111/j.1399-0004.1995.tb04303.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7554348/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Genet&title=Evidence%20for%20genetic%20influences%20on%20smoking%20in%20adult%20women%20twins&author=KL%20Edwards&author=MA%20Austin&author=GP%20Jarvik&volume=47&publication_year=1995&pages=236-244&pmid=7554348&doi=10.1111/j.1399-0004.1995.tb04303.x&)

8. Furukawa H, Singh SK, Mancusso R, Gouaux E. Subunit arrangement and function in NMDA receptors. Nature. 2005;438:185–192. doi: 10.1038/nature04089.  [DOI](https://doi.org/10.1038/nature04089) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16281028/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Subunit%20arrangement%20and%20function%20in%20NMDA%20receptors&author=H%20Furukawa&author=SK%20Singh&author=R%20Mancusso&author=E%20Gouaux&volume=438&publication_year=2005&pages=185-192&pmid=16281028&doi=10.1038/nature04089&)

9. Gabriel SB, Schaffner SF, Nguyen H, Moore JM, Roy J, Blumenstiel B, Higgins J, DeFelice M, Lochner A, Faggart M, Liu-Cordero SN, Rotimi C, Adeyemo A, Cooper R, Ward R, Lander ES, Daly MJ, Altshuler D. The structure of haplotype blocks in the human genome. Science. 2002;296:2225–2229. doi: 10.1126/science.1069424.  [DOI](https://doi.org/10.1126/science.1069424) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12029063/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=The%20structure%20of%20haplotype%20blocks%20in%20the%20human%20genome&author=SB%20Gabriel&author=SF%20Schaffner&author=H%20Nguyen&author=JM%20Moore&author=J%20Roy&volume=296&publication_year=2002&pages=2225-2229&pmid=12029063&doi=10.1126/science.1069424&)

10. Gelernter J, Panhuysen C, Weiss R, Brady K, Poling J, Krauthammer M, Farrer L, Kranzler HR. Genomewide linkage scan for nicotine dependence: identification of a chromosome 5 risk locus. Biol Psychiatry. 2007;61:119–126. doi: 10.1016/j.biopsych.2006.08.023.  [DOI](https://doi.org/10.1016/j.biopsych.2006.08.023) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17081504/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biol%20Psychiatry&title=Genomewide%20linkage%20scan%20for%20nicotine%20dependence:%20identification%20of%20a%20chromosome%205%20risk%20locus&author=J%20Gelernter&author=C%20Panhuysen&author=R%20Weiss&author=K%20Brady&author=J%20Poling&volume=61&publication_year=2007&pages=119-126&pmid=17081504&doi=10.1016/j.biopsych.2006.08.023&)

11. Gulli R, Masnata B, Bonvicini C, Tura GB, Manglaviti L, Vaggi M, Mollica M, Bellone E, Mandich P, Gennarelli M, Di Maria E. A putative regulatory subunit (NR3A) of the NMDA receptor complex as candidate gene for susceptibility to schizophrenia: a case-control study. Psychiatr Genet. 2007;17:355–356. doi: 10.1097/YPG.0b013e328133f74f.  [DOI](https://doi.org/10.1097/YPG.0b013e328133f74f) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18075478/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Psychiatr%20Genet&title=A%20putative%20regulatory%20subunit%20(NR3A)%20of%20the%20NMDA%20receptor%20complex%20as%20candidate%20gene%20for%20susceptibility%20to%20schizophrenia:%20a%20case-control%20study&author=R%20Gulli&author=B%20Masnata&author=C%20Bonvicini&author=GB%20Tura&author=L%20Manglaviti&volume=17&publication_year=2007&pages=355-356&pmid=18075478&doi=10.1097/YPG.0b013e328133f74f&)

12. Heatherton TF, Kozlowski LT, Frecker RC, Fagerstrom KO. The Fagerstrom test for nicotine dependence: a revision of the Fagerstrom tolerance questionnaire. Br J Addict. 1991;86:1119–1127. doi: 10.1111/j.1360-0443.1991.tb01879.x.  [DOI](https://doi.org/10.1111/j.1360-0443.1991.tb01879.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/1932883/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Addict&title=The%20Fagerstrom%20test%20for%20nicotine%20dependence:%20a%20revision%20of%20the%20Fagerstrom%20tolerance%20questionnaire&author=TF%20Heatherton&author=LT%20Kozlowski&author=RC%20Frecker&author=KO%20Fagerstrom&volume=86&publication_year=1991&pages=1119-1127&pmid=1932883&doi=10.1111/j.1360-0443.1991.tb01879.x&)

13. Horvath S, Xu X, Lake SL, Silverman EK, Weiss ST, Laird NM. Family-based tests for associating haplotypes with general phenotype data: application to asthma genetics. Genet Epidemiol. 2004;26:61–69. doi: 10.1002/gepi.10295.  [DOI](https://doi.org/10.1002/gepi.10295) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14691957/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genet%20Epidemiol&title=Family-based%20tests%20for%20associating%20haplotypes%20with%20general%20phenotype%20data:%20application%20to%20asthma%20genetics&author=S%20Horvath&author=X%20Xu&author=SL%20Lake&author=EK%20Silverman&author=ST%20Weiss&volume=26&publication_year=2004&pages=61-69&pmid=14691957&doi=10.1002/gepi.10295&)

14. Lange C, Silverman EK, Xu X, Weiss ST, Laird NM. A multivariate family-based association test using generalized estimating equations: FBAT-GEE. Biostatistics. 2003;4:195–206. doi: 10.1093/biostatistics/4.2.195.  [DOI](https://doi.org/10.1093/biostatistics/4.2.195) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12925516/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biostatistics&title=A%20multivariate%20family-based%20association%20test%20using%20generalized%20estimating%20equations:%20FBAT-GEE&author=C%20Lange&author=EK%20Silverman&author=X%20Xu&author=ST%20Weiss&author=NM%20Laird&volume=4&publication_year=2003&pages=195-206&pmid=12925516&doi=10.1093/biostatistics/4.2.195&)

15. Li MD. Identifying susceptibility loci for nicotine dependence: 2008 update based on recent genome-wide linkage analyses. Hum Genet. 2008;123:119–131. doi: 10.1007/s00439-008-0473-0.  [DOI](https://doi.org/10.1007/s00439-008-0473-0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18205015/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Genet&title=Identifying%20susceptibility%20loci%20for%20nicotine%20dependence:%202008%20update%20based%20on%20recent%20genome-wide%20linkage%20analyses&author=MD%20Li&volume=123&publication_year=2008&pages=119-131&pmid=18205015&doi=10.1007/s00439-008-0473-0&)

16. Li MD, Cheng R, Ma JZ, Swan GE. A meta-analysis of estimated genetic and environmental effects on smoking behavior in male and female adult twins. Addiction. 2003a;98:23–31. doi: 10.1046/j.1360-0443.2003.00295.x.  [DOI](https://doi.org/10.1046/j.1360-0443.2003.00295.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12492752/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Addiction&title=A%20meta-analysis%20of%20estimated%20genetic%20and%20environmental%20effects%20on%20smoking%20behavior%20in%20male%20and%20female%20adult%20twins&author=MD%20Li&author=R%20Cheng&author=JZ%20Ma&author=GE%20Swan&volume=98&publication_year=2003a&pages=23-31&pmid=12492752&doi=10.1046/j.1360-0443.2003.00295.x&)

17. Li MD, Ma JZ, Cheng R, Dupont RT, Williams NJ, Crews KM, Payne TJ, Elston RC. A genome-wide scan to identify loci for smoking rate in the Framingham Heart Study population. BMC Genet. 2003b;4(Suppl 1):S103. doi: 10.1186/1471-2156-4-S1-S103.  [DOI](https://doi.org/10.1186/1471-2156-4-S1-S103) | [PMC free article](/articles/PMC1866441/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14975171/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Genet&title=A%20genome-wide%20scan%20to%20identify%20loci%20for%20smoking%20rate%20in%20the%20Framingham%20Heart%20Study%20population&author=MD%20Li&author=JZ%20Ma&author=R%20Cheng&author=RT%20Dupont&author=NJ%20Williams&volume=4&issue=Suppl%201&publication_year=2003b&pages=S103&pmid=14975171&doi=10.1186/1471-2156-4-S1-S103&)

18. Li MD, Beuten J, Ma JZ, Payne TJ, Lou XY, Garcia V, Duenes AS, Crews KM, Elston RC. Ethnic- and gender-specific association of the nicotinic acetylcholine receptor alpha4 subunit gene (CHRNA4) with nicotine dependence. Hum Mol Genet. 2005;14:1211–1219. doi: 10.1093/hmg/ddi132.  [DOI](https://doi.org/10.1093/hmg/ddi132) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15790597/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mol%20Genet&title=Ethnic-%20and%20gender-specific%20association%20of%20the%20nicotinic%20acetylcholine%20receptor%20alpha4%20subunit%20gene%20(CHRNA4)%20with%20nicotine%20dependence&author=MD%20Li&author=J%20Beuten&author=JZ%20Ma&author=TJ%20Payne&author=XY%20Lou&volume=14&publication_year=2005&pages=1211-1219&pmid=15790597&doi=10.1093/hmg/ddi132&)

19. Li MD, Payne TJ, Ma JZ, Lou XY, Zhang D, Dupont RT, Crews KM, Somes G, Williams NJ, Elston RC. A genomewide search finds major susceptibility Loci for nicotine dependence on chromosome 10 in african americans. Am J Hum Genet. 2006;79:745–751. doi: 10.1086/508208.  [DOI](https://doi.org/10.1086/508208) | [PMC free article](/articles/PMC1592559/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16960812/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=A%20genomewide%20search%20finds%20major%20susceptibility%20Loci%20for%20nicotine%20dependence%20on%20chromosome%2010%20in%20african%20americans&author=MD%20Li&author=TJ%20Payne&author=JZ%20Ma&author=XY%20Lou&author=D%20Zhang&volume=79&publication_year=2006&pages=745-751&pmid=16960812&doi=10.1086/508208&)

20. Li MD, Ma JZ, Payne TJ, Lou XY, Zhang D, Dupont RT, Elston RC. Genome-wide linkage scan for nicotine dependence in European Americans and its converging results with African Americans in the Mid-South Tobacco Family sample. Mol Psychiatry. 2008;13:407–416. doi: 10.1038/sj.mp.4002038.  [DOI](https://doi.org/10.1038/sj.mp.4002038) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17579606/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Psychiatry&title=Genome-wide%20linkage%20scan%20for%20nicotine%20dependence%20in%20European%20Americans%20and%20its%20converging%20results%20with%20African%20Americans%20in%20the%20Mid-South%20Tobacco%20Family%20sample&author=MD%20Li&author=JZ%20Ma&author=TJ%20Payne&author=XY%20Lou&author=D%20Zhang&volume=13&publication_year=2008&pages=407-416&pmid=17579606&doi=10.1038/sj.mp.4002038&)

21. Ma JZ, Beuten J, Payne TJ, Dupont RT, Elston RC, Li MD. Haplotype analysis indicates an association between the DOPA decarboxylase (DDC) gene and nicotine dependence. Hum Mol Genet. 2005;14:1691–1698. doi: 10.1093/hmg/ddi177.  [DOI](https://doi.org/10.1093/hmg/ddi177) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15879433/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mol%20Genet&title=Haplotype%20analysis%20indicates%20an%20association%20between%20the%20DOPA%20decarboxylase%20(DDC)%20gene%20and%20nicotine%20dependence&author=JZ%20Ma&author=J%20Beuten&author=TJ%20Payne&author=RT%20Dupont&author=RC%20Elston&volume=14&publication_year=2005&pages=1691-1698&pmid=15879433&doi=10.1093/hmg/ddi177&)

22. Madden PA, Heath AC, Pedersen NL, Kaprio J, Koskenvuo MJ, Martin NG. The genetics of smoking persistence in men and women: a multicultural study. Behav Genet. 1999;29:423–431. doi: 10.1023/a:1021674804714.  [DOI](https://doi.org/10.1023/a:1021674804714) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10857247/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Behav%20Genet&title=The%20genetics%20of%20smoking%20persistence%20in%20men%20and%20women:%20a%20multicultural%20study&author=PA%20Madden&author=AC%20Heath&author=NL%20Pedersen&author=J%20Kaprio&author=MJ%20Koskenvuo&volume=29&publication_year=1999&pages=423-431&pmid=10857247&doi=10.1023/a:1021674804714&)

23. Mokdad AH, Marks JS, Stroup DF, Gerberding JL. Actual causes of death in the United States, 2000. JAMA. 2004;291:1238–1245. doi: 10.1001/jama.291.10.1238.  [DOI](https://doi.org/10.1001/jama.291.10.1238) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15010446/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA&title=Actual%20causes%20of%20death%20in%20the%20United%20States,%202000&author=AH%20Mokdad&author=JS%20Marks&author=DF%20Stroup&author=JL%20Gerberding&volume=291&publication_year=2004&pages=1238-1245&pmid=15010446&doi=10.1001/jama.291.10.1238&)

24. Nilsson A, Eriksson M, Muly EC, Akesson E, Samuelsson EB, Bogdanovic N, Benedikz E, Sundstrom E. Analysis of NR3A receptor subunits in human native NMDA receptors. Brain Res. 2007;1186:102–112. doi: 10.1016/j.brainres.2007.09.008.  [DOI](https://doi.org/10.1016/j.brainres.2007.09.008) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17997397/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Brain%20Res&title=Analysis%20of%20NR3A%20receptor%20subunits%20in%20human%20native%20NMDA%20receptors&author=A%20Nilsson&author=M%20Eriksson&author=EC%20Muly&author=E%20Akesson&author=EB%20Samuelsson&volume=1186&publication_year=2007&pages=102-112&pmid=17997397&doi=10.1016/j.brainres.2007.09.008&)

25. Nyholt DR. A simple correction for multiple testing for single-nucleotide polymorphisms in linkage disequilibrium with each other. Am J Hum Genet. 2004;74:765–769. doi: 10.1086/383251.  [DOI](https://doi.org/10.1086/383251) | [PMC free article](/articles/PMC1181954/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14997420/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=A%20simple%20correction%20for%20multiple%20testing%20for%20single-nucleotide%20polymorphisms%20in%20linkage%20disequilibrium%20with%20each%20other&author=DR%20Nyholt&volume=74&publication_year=2004&pages=765-769&pmid=14997420&doi=10.1086/383251&)

26. O’Connell JR, Weeks DE. PedCheck: a program for identification of genotype incompatibilities in linkage analysis. Am J Hum Genet. 1998;63:259–266. doi: 10.1086/301904.  [DOI](https://doi.org/10.1086/301904) | [PMC free article](/articles/PMC1377228/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9634505/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=PedCheck:%20a%20program%20for%20identification%20of%20genotype%20incompatibilities%20in%20linkage%20analysis&author=JR%20O%E2%80%99Connell&author=DE%20Weeks&volume=63&publication_year=1998&pages=259-266&pmid=9634505&doi=10.1086/301904&)

27. Paoletti P, Neyton J. NMDA receptor subunits: function and pharmacology. Curr Opin Pharmacol. 2007;7(1):39–47. doi: 10.1016/j.coph.2006.08.011.  [DOI](https://doi.org/10.1016/j.coph.2006.08.011) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17088105/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Opin%20Pharmacol&title=NMDA%20receptor%20subunits:%20function%20and%20pharmacology&author=P%20Paoletti&author=J%20Neyton&volume=7&issue=1&publication_year=2007&pages=39-47&pmid=17088105&doi=10.1016/j.coph.2006.08.011&)

28. Perez-Stable EJ, Herrera B, Jacob P, 3rd, Benowitz NL. Nicotine metabolism and intake in black and white smokers. JAMA. 1998;280:152–156. doi: 10.1001/jama.280.2.152.  [DOI](https://doi.org/10.1001/jama.280.2.152) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9669788/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA&title=Nicotine%20metabolism%20and%20intake%20in%20black%20and%20white%20smokers&author=EJ%20Perez-Stable&author=B%20Herrera&author=P%20Jacob&author=NL%20Benowitz&volume=280&publication_year=1998&pages=152-156&pmid=9669788&doi=10.1001/jama.280.2.152&)

29. Perkins KA, Donny E, Caggiula AR. Sex differences in nicotine effects and self-administration: review of human and animal evidence. Nicotine Tob Res. 1999;1:301–315. doi: 10.1080/14622299050011431.  [DOI](https://doi.org/10.1080/14622299050011431) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11072427/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nicotine%20Tob%20Res&title=Sex%20differences%20in%20nicotine%20effects%20and%20self-administration:%20review%20of%20human%20and%20animal%20evidence&author=KA%20Perkins&author=E%20Donny&author=AR%20Caggiula&volume=1&publication_year=1999&pages=301-315&pmid=11072427&doi=10.1080/14622299050011431&)

30. Sullivan PF, Kendler KS. The genetic epidemiology of smoking. Nicotine Tob Res. 1999;1(Suppl 2):S51–S57. doi: 10.1080/14622299050011811. discussion S69–S70.  [DOI](https://doi.org/10.1080/14622299050011811) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11768187/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nicotine%20Tob%20Res&title=The%20genetic%20epidemiology%20of%20smoking&author=PF%20Sullivan&author=KS%20Kendler&volume=1&issue=Suppl%202&publication_year=1999&pages=S51-S57&pmid=11768187&doi=10.1080/14622299050011811&)

31. Uhl GR, Liu QR, Drgon T, Johnson C, Walther D, Rose JE, David SP, Niaura R, Lerman C. Molecular genetics of successful smoking cessation: convergent genome-wide association study results. Arch Gen Psychiatry. 2008;65:683–693. doi: 10.1001/archpsyc.65.6.683.  [DOI](https://doi.org/10.1001/archpsyc.65.6.683) | [PMC free article](/articles/PMC2430596/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18519826/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arch%20Gen%20Psychiatry&title=Molecular%20genetics%20of%20successful%20smoking%20cessation:%20convergent%20genome-wide%20association%20study%20results&author=GR%20Uhl&author=QR%20Liu&author=T%20Drgon&author=C%20Johnson&author=D%20Walther&volume=65&publication_year=2008&pages=683-693&pmid=18519826&doi=10.1001/archpsyc.65.6.683&)

32. Vink JM, Smit AB, de Geus EJ, Sullivan P, Willemsen G, Hottenga JJ, Smit JH, Hoogendijk WJ, Zitman FG, Peltonen L, Kaprio J, Pedersen NL, Magnusson PK, Spector TD, Kyvik KO, Morley KI, Heath AC, Martin NG, Westendorp RG, Slagboom PE, Tiemeier H, Hofman A, Uitterlinden AG, Aulchenko YS, Amin N, van Duijn C, Penninx BW, Boomsma DI. Genome-wide association study of smoking initiation and current smoking. Am J Hum Genet. 2009;84:367–379. doi: 10.1016/j.ajhg.2009.02.001.  [DOI](https://doi.org/10.1016/j.ajhg.2009.02.001) | [PMC free article](/articles/PMC2667987/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19268276/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=Genome-wide%20association%20study%20of%20smoking%20initiation%20and%20current%20smoking&author=JM%20Vink&author=AB%20Smit&author=EJ%20de%20Geus&author=P%20Sullivan&author=G%20Willemsen&volume=84&publication_year=2009&pages=367-379&pmid=19268276&doi=10.1016/j.ajhg.2009.02.001&)

33. Wang J, Li MD. Common and unique biological pathways associated with smoking initiation/progression, nicotine dependence, and smoking cessation. Neuropsychopharmacology. 2010 doi: 10.1038/npp.2009.178. [Epub ahead of print]  [DOI](https://doi.org/10.1038/npp.2009.178) | [PMC free article](/articles/PMC2814000/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19890259/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neuropsychopharmacology&title=Common%20and%20unique%20biological%20pathways%20associated%20with%20smoking%20initiation/progression,%20nicotine%20dependence,%20and%20smoking%20cessation&author=J%20Wang&author=MD%20Li&publication_year=2010&pmid=19890259&doi=10.1038/npp.2009.178&)
