# A Comprehensive Functional Assessment of Carboxylesterase 1 Nonsynonymous Polymorphisms

## Metadata
**Authors:** Xinwen Wang, Nada Rida, Jian Shi, Audrey H Wu, Barry E Bleske, Hao-Jie Zhu
**Journal:** Drug Metabolism and Disposition
**Date:** 2017 Nov
**DOI:** [10.1124/dmd.117.077669](https://doi.org/10.1124/dmd.117.077669)
**PMID:** 28838926
**PMCID:** PMC5637814
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5637814/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC5637814/pdf/dmd.117.077669.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC5637814/pdf/dmd.117.077669.pdf)

## Abstract

Carboxylesterase 1 (CES1) is the predominant human hepatic hydrolase responsible for the metabolism of many clinically important medications. CES1 expression and activity vary markedly among individuals; and genetic variation is a major contributing factor to CES1 interindividual variability. In this study, we comprehensively examined the functions of CES1 nonsynonymous single nucleotide polymorphisms (nsSNPs) and haplotypes using transfected cell lines and individual human liver tissues. The 20 candidate variants include CES1 nsSNPs with a minor allele frequency >0.5% in a given population or located in close proximity to the CES1 active site. Five nsSNPs, including L40Ter (rs151291296), G142E (rs121912777), G147C (rs146456965), Y170D (rs148947808), and R171C (rs201065375), were loss-of-function variants for metabolizing the CES1 substrates clopidogrel, enalapril, and sacubitril. In addition, A158V (rs202121317), R199H (rs2307243), E220G (rs200707504), and T290M (rs202001817) decreased CES1 activity to a lesser extent in a substrate-dependent manner. Several nsSNPs, includingL40Ter (rs151291296), G147C (rs146456965), Y170D (rs148947808), and R171C (rs201065375), significantly reduced CES1 protein and/or mRNA expression levels in the transfected cells. Functions of the common nonsynonymous haplotypes D203E-A269S and S75N-D203E-A269S were evaluated using cells stably expressing the haplotypes and a large set of the human liver. Neither CES1 expression nor activity was affected by the two haplotypes. In summary, this study revealed several functional nsSNPs with impaired activity on the metabolism of CES1 substrate drugs. Clinical investigations are warranted to determine whether these nsSNPs can serve as biomarkers for the prediction of therapeutic outcomes of drugs metabolized by CES1.

## Introduction

Carboxylesterase 1 (CES1) is the most abundant drug-metabolizing enzyme in the human liver ([Achour et al., 2017](#B1)), contributing to 80%–95% of total hepatic hydrolytic activity ([Imai, 2006](#B5)). CES1 is responsible for the metabolism of a wide range of therapeutic agents, endogenous compounds, and environmental toxins ([Laizure et al., 2013](#B6)). Of particular clinical relevance, CES1 catalyzes the hydrolysis of numerous clinically important medications, such as angiotensin-converting enzyme inhibitor prodrugs ([Wang et al., 2016b](#B25)), clopidogrel (Plavix; Bristol-Myers Squibb, New York, NY) ([Zhu et al., 2013](#B32)), sacubitril (Entresto; Novartis, Basel, Switzerland) ([Shi et al., 2016b](#B18)), methylphenidate (Ritalin; Novartis) ([Zhu et al., 2008](#B31)), oseltamivir (Tamiflu; Genentech, San Francisco, CA) ([Shi et al., 2016a](#B17)), and dabigatran etexilate (Pradaxa; Boehringer Ingelheim Pharmaceuticals, Ridgefield, CT) ([Laizure et al., 2014](#B7); [Shi et al., 2016c](#B19)). Marked interindividual variability in CES1 expression and activity has been well documented ([Hosokawa et al., 1995](#B4); [Shi et al., 2006](#B16); [Yoshimura et al., 2008](#B27); [Hagihara et al., 2009](#B3); [Yang et al., 2009](#B26); [Zhu et al., 2009a](#B28); [Ross et al., 2012](#B14)), which may affect the pharmacokinetics and pharmacodynamics of drugs metabolized by CES1. Genetic polymorphisms are increasingly recognized as an important factor contributing to CES1 variability and varied responses to CES1 substrate drugs ([Geshi et al., 2005](#B2); [Zhu et al., 2008](#B31); [Nemoda et al., 2009](#B10); [Sai et al., 2010](#B15); [Rasmussen et al., 2015](#B13); [Tarkiainen et al., 2015a](#B22)). For example, the *CES1* nonsynonymous single nucleotide polymorphism (nsSNP) G143E (rs71647871) was associated with diminishing CES1 activity on metabolizing several CES1 substrate drugs, including methylphenidate ([Zhu et al., 2008](#B31); [Nemoda et al., 2009](#B10); [Stage et al., 2017](#B20)), oseltamivir ([Zhu and Markowitz, 2009](#B30); [Tarkiainen et al., 2012](#B21)), enalapril ([Tarkiainen et al., 2015b](#B23); [Wang et al., 2016b](#B25)), clopidogrel ([Lewis et al., 2013](#B8); [Tarkiainen et al., 2015a](#B22)), dabigatran etexilate ([Shi et al., 2016c](#B19)), and sacubitril ([Shi et al., 2016b](#B18)). These findings indicate that *CES1* genetic variants may significantly impact responses to drugs metabolized by CES1.

The *CES1* gene is located on chromosome 16q13-q22.1 and is highly polymorphic, with >600 nsSNPs across its 14 exons. The frequencies of *CES1* nsSNPs vary markedly among different ethnic groups. Several nsSNPs were rare in Caucasians but common in other populations, including the variants T167S [rs147694791, minor allele frequency (MAF) = 5.48% in Africans], R186P [rs60054861, MAF = 12.39% in Africans], A158V [rs202121317, MAF = 6.69% in South Asians], and E220G [rs200707504, MAF = 3.1% in Koreans]. A recent in silico structure-based analysis predicted that eight *CES1* nsSNPs, including G142E (rs121912777), G143E (rs71647871), G147C (rs146456965), Q169P (rs143718310), Y170D (rs148947808), R171C (rs201065375), G173D (rs4784575), and E220G (rs200707504), reside within 5 Å from the enzyme active site ([Nzabonimpa et al., 2016](#B11)) and thus are likely to alter enzyme-substrate binding affinity. However, except for a very few variants that have a relatively high MAF in Caucasians (e.g., G143E), the majority of *CES1* nsSNPs have not been thoroughly studied for their functional consequences.

In this study, we conducted a comprehensive in vitro functional study using transfected cell lines and individual human liver tissues to evaluate the functions of *CES1* nsSNPs that have a MAF >0.5% in a given population and the common haplotypes D203E-A269S and S75N-D203E-A269S. Furthermore, nsSNPs located in close proximity to the CES1 active site were also subjected to the functional study. This study revealed that several nsSNPs significantly impaired CES1 activity on the metabolism of the CES1 substrates enalapril, clopidogrel, and sacubitril. The findings reinforce the notion that CES1 genetic variants play an important role in interindividual variability in response to medications metabolized by CES1.

## Materials and Methods

### Materials

The QuikChange Lightning Multi Site-Directed Mutagenesis Kit was purchased from Agilent Technologies (Santa Clara, CA). The products purchased from Thermo Fisher Scientific Co. (Waltham, MA) included the Flp-In-293 cell line, pOG44 plasmid, S.N.A.P. Plasmid DNA MiniPrep Kit, Lipofectamine 2000 Transfection Reagent, Dulbecco’s modified Eagle’s medium (DMEM), fetal bovine serum (FBS), Hygromycin B in phosphate-buffered saline (50 mg/ml) and a 100× antibiotics mixture containing penicillin (100 IU/ml) and streptomycin (100 *μ*g/ml), Trypsin-EDTA (0.25%), the SILAC Protein Quantitation Kit-DMEM containing SILAC DMEM (deficient in arginine and lysine), ^13^C_6_ l-lysine-2HCl, ^13^C_6_^15^N_4_ l-arginine-HCl, dialyzed FBS, urea, dl-dithiothreitol, trifluoroacetic acid, TRIzol RNA isolation reagent, and acetonitrile. Iodoacetamide was from Acros Organics (Morris Plains, NJ). L-1-tosylamide-2-phenylethyl chloromethyl ketone-treated trypsin was obtained from Worthington Biochemical Corporation (Freehold, NJ). Water Oasis HLB columns were from Waters Corporation (Milford, MA). Recombinant CES1 (purity >95%) was from R&D Systems (Minneapolis, MN).

Enalapril maleate was purchased from Sigma-Aldrich (St. Louis, MO). The enalapril hydrolytic metabolite enalaprilat dehydrate was from Selleck Chemicals (Houston, TX). *S*-(l)-clopidogrel and clopidogrel carboxylate were obtained from Toronto Research Chemicals, Inc. (Toronto, ON, Canada). Sacubitril was purchased from MedKoo Biosciences (Chapel Hill, NC). The active metabolite of sacubitril, LBQ657 (*γS*-[(3-carboxy-1-oxopropyl)amino]-*αR*-methyl-[1,1′-biphenyl]-4-pentanoic acid), was obtained from our laboratory after incubation of 100 *μ*M sacubitril with 50 ng/*μ*l recombinant human CES1 at 37°C for 2 hours ([Shi et al., 2016b](#B18)). Sacubitril was completely hydrolyzed to LBQ657 after incubation as determined by liquid chromatography (LC)–tandem mass spectrometry (MS/MS) analysis ([Shi et al., 2016b](#B18)). The High-Capacity cDNA Reverse Transcription Kit and SYBR Green PCR Master Mix were from Applied Biosystems (Foster City, CA).

A total of 104 individual normal human liver samples were obtained from XenoTech LLC (Lenexa, KS) and the Cooperative Human Tissue Network (Columbus, OH). The liver samples were from 48 men and 56 women aged 22–81 years. The donors included 94 Caucasians, six African Americans, two Hispanics, and two classified as “others.”

### Establishment of Cell Lines Stably Expressing Wild-Type and Mutant CES1

A total of 20 *CES1* variants were selected for an in vitro functional study based on the criteria that the SNPs or haplotypes have a MAF >0.5% in a population or the variants are located within a 5-Å distance from the CES1 active site ([Table 1](#T1)). Mutant *CES1* plasmids were generated by a site-directed mutagenesis assay with specific mutagenic primers ([Supplemental Table 1](http://dmd.aspetjournals.org/lookup/suppl/doi:10.1124/dmd.117.077669/-/DC1)) and transfected to human embryonic kidney cells (Flp-In-293) based on previously published methods ([Zhu et al., 2008](#B31)). The desired sequences of wild-type (WT) and mutant *CES1* plasmids were confirmed by DNA sequencing analysis. Verified *CES1* plasmids were co-transfected with a pOG44 plasmid at a ratio of 1:10 into Flp-In-293 cells with Lipofectamine 2000. Six hours after transfection, cells were gently rinsed to remove transfection reagents and were then cultured in complete medium (DMEM containing 10% FBS). After 12 hours, the culture medium was replaced with a complete medium supplemented with the antibiotic hygromycin B (100 *μ*g/ml) for the selection of CES1 transfected cells. Stable WT and mutant CES1-expressing cell lines were established after culture with hygromycin B for at least 3 weeks. All transfected cell lines were validated by DNA sequencing analysis.

### TABLE 1.

| nsSNP or Haplotype | Protein Residue Changes | Nucleotide Changes | MAF |  |  |  |  |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| African | European (FINN) | European (Non-FINN) | East Asian | South Asian | Latino | Other |  |  |  |
|   |   |   | % |  |  |  |  |  |  |
| nsSNP |   |   |   |   |   |   |   |   |   |
| rs151291296 | L40Ter | T227G | 0.51 | 0.00 | 0.00 | 0.00 | 0.00 | 0.00 | 0.00 |
| rs2307240 | S75N | G332A | 4.30 | 3.83 | 5.37 | 4.68 | 7.44 | 2.06 | 5.29 |
| rs121912777 | G142E | G533A | 0.00 | 0.00 | 0.00 | 0.00 | 0.00 | 0.00 | 0.00 |
| rs146456965 | G147C | G547T | 0.03 | 0.00 | 0.08 | 0.00 | 0.00 | 0.00 | 0.00 |
| rs202121317 | A158V | C581T | 0.04 | 0.00 | 0.00 | 0.03 | 6.69 | 0.04 | 0.11 |
| rs147694791 | T167S | C608G | 5.48 | 0.00 | 0.01 | 0.00 | 0.01 | 0.32 | 0.00 |
| rs143718310 | Q169P | A614C | 0.00 | 0.00 | 0.00 | 0.00 | 0.00 | 0.00 | 0.00 |
| rs148947808 | Y170D | T616G | 0.00 | 0.00 | 0.00 | 0.00 | 0.00 | 0.00 | 0.00 |
| rs201065375 | R171C | C619T | 0.01 | 0.00 | 0.02 | 0.00 | 0.00 | 0.05 | 0.00 |
| rs4784575 | G173D | G626A | 0.01 | 0.00 | 0.00 | 0.39 | 0.02 | 0.12 | 0.00 |
| rs60054861 | R186P | G665C | 12.39 | 0.02 | 0.24 | 0.01 | 0.02 | 0.69 | 1.23 |
| rs2307243 | R199H | G704A | 0.74 | 0.00 | 0.03 | 0.00 | 0.00 | 0.03 | 0.00 |
| rs2307227 | D203E | C717A | 8.30 | 0.71 | 3.16 | 1.18 | 4.33 | 1.70 | 3.53 |
| rs200707504 | E220G | A767G | 0.00 | 0.00 | 0.00 | 0.55 | 0.00 | 0.00 | 0.00 |
| rs115629050 | A269S | G913T | 8.25 | 0.71 | 3.15 | 1.24 | 4.31 | 1.71 | 3.52 |
| rs114119971 | H284Q | C960G | 0.18 | 0.27 | 0.71 | 0.00 | 0.06 | 0.26 | 0.44 |
| rs202001817 | T290M | C977T | 1.21 | 0.03 | 0.19 | 0.02 | 2.86 | 1.87 | 0.11 |
| rs576295379 | N340K | T1128G | 0.00 | 0.00 | 0.01 | 0.00 | 0.55 | 0.01 | 0.11 |
| Haplotype |   |   |   |   |   |   |   |   |   |
| D203E-A269S |   |   | 5.22 | 0.00 | 0.00 | 0.00 | 0.00 | 0.00 | NA |
| S75N-D203E-A269S |   |   | 3.56 | 1.01 | 4.46 | 1.49 | 4.29 | 2.02 | NA |

Table 1 Caption: MAFs of CES1 nsSNPs and haplotypes

### Preparation of S9 Fractions from Transfected Cells and Individual Human Livers

For the preparation of S9 fractions from the transfected cell lines, cells were cultured in 175-cm^2^ flasks until they reached approximately 95% confluence. The cells were rinsed twice with ice-cold phosphate-buffered saline before they were harvested in 0.5 ml of the same buffer. The cells were then lysed by sonication, followed by centrifugation at 9000*g* at 4°C for 30 minutes. The supernatant (S9 fraction) was collected in 1.5-ml protein low binding tubes and stored at −80°C until use. Individual human liver S9 fractions were prepared according to a previous publication ([Wang et al., 2016b](#B25)).

### Enzymatic Activity Assays

In vitro incubation studies were conducted to determine CES1 activities of the prepared S9 fraction samples on hydrolysis of the CES1 substrates enalapril, clopidogrel, and sacubitril. Enalapril incubation was carried out in 4-ml silanized glass vials because of significant nonspecific bindings of enalapril and its hydrolytic metabolite enalaprilat to plastic Eppendorf tubes ([Wang et al., 2016b](#B25)). For clopidogrel and sacubitril, incubations were performed in 1.5-ml Eppendorf tubes. Enalapril (500 *μ*M), clopidogrel (100 *μ*M), and sacubitril (200 *μ*M) were incubated with S9 fractions at final protein concentrations of 0.2, 0.1, and 0.05 mg/ml, respectively, at 37°C for 10 minutes. The hydrolysis reactions were terminated by the addition of a 4-fold volume of methanol containing the internal standard (IS) 5-hydroxy omeprazole (20 ng/ml) for enalapril, a 2-fold volume of acetonitrile with the IS *d*_4_-clopidogrel carboxylic acid (25 ng/ml) for clopidogrel, and a 3-fold volume of acetonitrile with the IS ritalinic acid (7.2 *μ*g/ml) for sacubitril. After centrifugation at 17,000*g* at 4°C for 20 minutes, the supernatant was collected and analyzed for metabolite concentrations using high-performance liquid chromatography (HPLC)-MS/MS methods described previously ([Zhu et al., 2013](#B32); [Shi et al., 2016b](#B18); [Wang et al., 2016b](#B25)). In addition, S9 fractions from vector-transfected cells served as a blank control in the CES1 activity studies.

### Quantification of CES1 Protein in Human Livers and WT and Mutant CES1 Transfected Cells

Absolute CES1 protein expression levels in individual human liver samples were quantified using a SILAC-based LC-MS/MS assay that we previously established ([Wang et al., 2016a](#B24)).

Relative CES1 protein quantifications in S9 fractions from transfected cells were conducted using a similar SILAC-based LC-MS/MS assay without having CES1 calibration curves. The intensities of several selected CES1 unique peptides were normalized by the SILAC counterpart peptides (i.e., IS), and glyceraldehyde 3-phosphate dehydrogenase (GAPDH) was used as the internal reference protein for quality control. Relative CES1 protein expression in each mutant cell line was calculated by comparison with the CES1 expression level in WT *CES1* transfected cells.

### Analysis of CES1 mRNA Expression in WT and Mutant CES1 Transfected Cells

WT and mutant *CES1* transfected cells were cultured in six-well plates. One microgram of RNA extracted from the cells was reverse transcribed to cDNA using oligo dT primers. Quantitative real-time polymerase chain reactions (qRT-PCRs) were performed on a 7300 Real-Time PCR system (Applied Biosystems) using SYBR green fluorescence. GAPDH was detected as the internal control for normalization. *CES1* gene expression in mutant cell lines relative to that in WT CES1 cells were determined by the 2^−ΔΔCT^ method. In addition, RNA isolated from vector-transfected cells was included in the qRT-PCR study as a blank control. The primers and experimental conditions for the qRT-PCR experiments are described in [Supplemental Tables 1 and 2](http://dmd.aspetjournals.org/lookup/suppl/doi:10.1124/dmd.117.077669/-/DC1), respectively.

### cDNA Sequencing of Human Liver Samples

Total RNA was isolated from 50 mg frozen individual human liver tissues using TRIzol reagent according to the manufacturer’s instructions. cDNA was synthesized from 1 *μ*g total RNA using the reverse transcription protocol described above. Nested PCR was subsequently applied to amplification of the whole length of the CES1 cDNA. Final PCR products were purified and subjected to bidirectional sequencing with four pairs of primers. The primers and nested PCR conditions for the CES1 cDNA amplification are listed in [Supplemental Tables 1 and 2](http://dmd.aspetjournals.org/lookup/suppl/doi:10.1124/dmd.117.077669/-/DC1), respectively.

### HPLC-MS/MS Analysis

The HPLC-MS/MS analysis was performed on a Shimadzu HPLC system (Shimadzu, Tokyo, Japan) coupled with an Applied Biosystems API 4000 triple quadruple mass spectrometer.

#### LC-MS/MS Method for CES1 Activity Assay.

The hydrolytic metabolites enalaprilat, clopidogrel carboxylate, and LBQ657 were quantitated using previously described assays with some modifications ([Zhu et al., 2013](#B32); [Shi et al., 2016b](#B18); [Wang et al., 2016b](#B25)). The analytes were isolated on a Shimadzu VP-ODS column (5 *μ*m, 150 × 2.0 mm; Shimadzu) with the mobile phase delivered at a constant flow rate (0.3 ml/min for enalaprilat, 0.25 ml/min for clopidogrel carboxylate, and 0.2 ml/min for LBQ657). The gradient conditions are summarized in [Supplemental Table 3](http://dmd.aspetjournals.org/lookup/suppl/doi:10.1124/dmd.117.077669/-/DC1). The column temperature was set at 50°C for enalaprilat and 40°C for clopidogrel carboxylate and LBQ657. An injection volume of 10 *μ*l was used for all analytes. Positive electrospray ionization mode was applied, and ions were monitored by multiple reaction monitoring with the following mass-to-charge transitions: enalaprilat (349.0 > 206.0), clopidogrel carboxylate (308.0 > 197.9), LBQ657 (384.7 > 266.7), 5-hydroxy omeprazole (362.34 > 213.9), clopidogrel *d*_4_-carboxylic acid (312.1 > 202.0), and ritalinic acid (220.5 > 84.6).

#### LC-MS/MS Method for CES1 Protein Quantification.

To quantitate the relative CES1 protein expression levels in the transfected cells, an established targeted CES1 proteomics assay was adopted with some modifications ([Wang et al., 2016a](#B24)). In brief, six CES1 unique peptides (i.e., AISESGVALTSVLVK, FLSLDLQGDPR, TAMSLLWK, SYPLVC[CAM]IAK, ELIPEATEK, FWANFAR) and one GAPDH unique peptide (GALQNIIPASTGAAK) and the corresponding heavy isotope-labeled SILAC peptides were separated on a Zorbax 300SB-C18 column (5 *μ*M, 150 × 2.1 mm; Agilent Technologies). The column temperature was set at 40°C. The mobile phase was delivered at a constant flow rate of 0.2 ml/min with the gradient conditions listed in [Supplemental Table 3](http://dmd.aspetjournals.org/lookup/suppl/doi:10.1124/dmd.117.077669/-/DC1). Under positive ionization mode, analytes were monitored at the transitions described previously ([Wang et al., 2016a](#B24)). The data were analyzed using Skyline software (University of Washington, Seattle, WA).

### Data Analysis

Data are representative of three independent experiments and values are expressed as means ± S.D. The unpaired, two-tailed *t* test was used to determine the differences in CES1 expression and activity between *CES1* variants and WT controls (GraphPad Prism software, version 6.0; GraphPad Software, San Diego, CA). *P* < 0.05 was considered statistically significant.

## Results

### Enzymatic Activity of WT and Mutant CES1 on Enalapril, Clopidogrel, and Sacubitril Hydrolysis

A total of 21 stable CES1 expressing cell lines were established to evaluate the effect of the candidate *CES1* nsSNPs and haplotypes on CES1 activity utilizing the CES1 substrates enalapril, clopidogrel, and sacubitril ([Fig. 1](#F1)). All three CES1 substrates were efficiently hydrolyzed by WT CES1. No appreciable hydrolytic metabolites were formed after incubation of the three substrates with the S9 fractions from cells expressing the *CES1* nsSNPs L40Ter, G142E, G147C, Y170D, and R171C, indicating that these five nsSNPs were loss-of-function variants for CES1. Among the five loss-of-function CES1 mutations, except for L40Ter, all variants were located close to the CES1 active site. Q169P (rs143718310), another nsSNP located near the CES1 active site, exhibited approximately 50% of CES1 activity on hydrolysis of the three substrates relative to the WT enzyme. Moreover, the CES1 SNP E220G, which has an MAF of 0.55% in South Asians and 3.1% in Koreans but is rare in other populations, significantly decreased CES1 hydrolytic activity on enalapril, clopidogrel, and sacubitril by 78.8% ± 2.6%, 82.3% ± 3.5%, and 80.3% ± 3.0%, respectively. The variants A158V (MAF = 6.69% in South Asians) and R199H (MAF = 0.74% in Africans) showed significantly decreased activity on enalapril hydrolysis (53.4% ± 1.5% and 59.5% ± 2.0%, respectively, of the WT CES1). Interestingly, these two nsSNPs did not significantly affect the hydrolysis of clopidogrel and sacubitril. This substrate-specific effect was also observed for T290M (MAF = 1.24% in Africans, 2.86% in South Asians, and 1.87% in Latinos), which reduced the hydrolysis of clopidogrel by 50.7% ± 16.8% while having no significant effect on enalapril and sacubitril metabolism. In addition to the individual nsSNPs, the two common haplotypes D203E-A269S and S75N-D203E-A269S were also studied for their potential effect on CES1 catalytic activity. The in vitro incubation study indicated that none of the haplotypes significantly affect CES1 activity on hydrolyzing enalapril, clopidogrel, and sacubitril.

### Fig. 1.

![Fig. 1.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ace4/5637814/656fd358d4b4/dmd.117.077669f1.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5637814_dmd.117.077669f1.jpg)

(A–C) Effect of CES1 nsSNPs and haplotypes on hydrolysis of enalapril (A), clopidogrel (B), and sacubitril (C) in CES1 transfected cell lines. Enalapril (500 μM), clopidogrel (100 μM), and sacubitril (200 μM) were incubated with S9 fractions prepared from cells stably transfected with WT CES1 and candidate variants (18 nsSNPs and two haplotypes: D203E-A269S and S75N-D203E-A269S). Hydrolysis rates of enalapril, clopidogrel, and sacubitril were determined by measuring the formation of the respective hydrolytic metabolites enalaprilat, clopidogrel carboxylic acid, and LBQ465. Vector-transfected cells were used as a blank control. Data are expressed as means ± S.D. (n = 3). *P < 0.05; **P < 0.001 (CES1 variants versus WT CES1).

### CES1 Protein and mRNA Expression Levels in WT and Mutant CES1 Cell Lines

CES1 mRNA and protein were abundantly expressed in the WT *CES1* transfected cells, with levels comparable to that in the human liver (data not shown). No detectable CES1 mRNA and protein was found in the vector control cells. nsSNP G147C decreased CES1 mRNA and protein expression levels by 80.5% ± 0.7% and 86.7% ± 2.8%, respectively. L40Ter-expressing cells displayed normal CES1 mRNA expression levels but null CES1 protein expression, which is likely due to the premature stop codon in exon 2 introduced by this variant. Interestingly, the CES1 protein levels of the Y170D and R171C variants were only 7.4% ± 2.8% and 5.9 ± 3.6% of WT CES1, respectively, whereas mRNA expression levels were not significantly affected. Neither haplotype S75N-D203E-A269S nor D203E-A269S affected the CES1 protein or mRNA expression levels in the transfected cells.

### CES1 nsSNPs S75N, D203E, and A269S Had No Effect on CES1 Protein Expression and Activity in the Human Liver

CES1 mRNA extracted from a total of 104 individual human liver samples was reverse transcribed to cDNA and subsequently sequenced for *CES1* nsSNPs. Twelve subjects were S75N heterozygotes and 10 subjects were heterozygous for D203E and A269S. We noted that D203E and A269S were in complete linkage disequilibrium (LD) with each other in our samples; 9 of the 10 D203E and A269S carriers were also heterozygous for the S75N variant (i.e., S75N-D203E-A269S haplotype). No other candidate *CES1* nsSNPs were found in the liver samples, which is likely because 94 of the 104 subjects were Caucasians and the other candidate *CES1* nsSNPs are rare in Caucasians. In addition, the known loss-of-function *CES1* nsSNP G143E and several rare nsSNPs that are not on our candidate nsSNP list were identified in the human liver samples, including I49V (rs3826193), L97I (rs571416840), A93V (rs202111709), and A156T (rs187158640). [Table 2](#T2) provides a summary of the genotypes, CES1 protein expression, CES1 activity on enalapril metabolism, and the CES1 activity normalized by protein expression of all CES1 nsSNPs carriers from the 104 human liver samples. Consistent with the results from the transfected cells, neither CES1 protein expression nor activity was affected by S75N and the haplotype S75N-D203E-A269S ([Fig. 3](#F3)). The effect of G143E on enalapril metabolism in the human liver was described in a recently published study ([Wang et al., 2016b](#B25)). Statistical analysis was not performed for other variants due to the small sample size of the carriers.

### TABLE 2.

| Subject Number | S75N | D203E | A269S | G143E | I49V | L97I | A93V | A156T | Enalapril Hydrolysis Rate | CES1 Protein Expression | Normalized CES1 Activity |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
|   |   |   |   |   |   |   |   |   | pmol/min per mg protein | pmol/mg protein | pmol/min per pmol CES1 |
| L2 | S/N | D/D | A/A | G/G | I/I | L/L | A/A | A/A | 40.3 | 104.17 | 0.39 |
| L16 | S/N | D/D | A/A | G/G | I/I | L/L | A/A | A/A | 45.3 | 84.45 | 0.54 |
| L40 | S/N | D/D | A/A | G/G | I/I | L/L | A/A | A/A | 93.4 | 170.44 | 0.55 |
| L8 | S/N | D/E | A/S | G/G | I/I | L/L | A/A | A/A | 62.4 | 184.06 | 0.34 |
| L31 | S/N | D/E | A/S | G/G | I/I | L/L | A/A | A/A | 12.6 | 104.39 | 0.12 |
| L45 | S/N | D/E | A/S | G/G | I/I | L/L | A/A | A/A | 41.1 | 118.41 | 0.35 |
| L60 | S/N | D/E | A/S | G/G | I/I | L/L | A/A | A/A | 64.4 | 194.12 | 0.33 |
| L61 | S/N | D/E | A/S | G/G | I/I | L/L | A/A | A/A | 50.3 | 141.49 | 0.36 |
| L72 | S/N | D/E | A/S | G/G | I/I | L/L | A/A | A/A | 153.7 | 266.06 | 0.58 |
| L90 | S/N | D/E | A/S | G/G | I/I | L/L | A/A | A/A | 36.3 | 102.01 | 0.36 |
| L95 | S/N | D/E | A/S | G/G | I/I | L/L | A/A | A/A | 87.8 | 215.71 | 0.41 |
| L102 | S/N | D/E | A/S | G/G | I/I | L/L | A/A | A/A | 83.7 | 104.85 | 0.80 |
| L99 | S/S | D/E | A/S | G/G | I/I | L/L | A/A | A/A | 20.2 | 69.99 | 0.29 |
| L41 | S/S | D/D | A/A | G/E | I/I | L/L | A/A | A/A | 15.9 | 236.16 | 0.07 |
| L73 | S/S | D/D | A/A | G/E | I/I | L/L | A/A | A/T | 22.4 | 96.33 | 0.23 |
| L94 | S/S | D/D | A/A | G/E | I/I | L/L | A/A | A/A | 28.4 | 156.11 | 0.18 |
| L103 | S/S | D/D | A/A | G/E | I/I | L/L | A/A | A/A | 19.9 | 163.28 | 0.12 |
| L104 | S/S | D/D | A/A | G/E | I/I | L/L | A/A | A/A | 17.6 | 273.32 | 0.06 |
| L3 | S/S | D/D | A/A | G/G | I/V | L/L | A/A | A/A | 0.0 | 155.26 | 0.00 |
| L5 | S/S | D/D | A/A | G/G | I/V | L/I | A/A | A/A | 26.4 | 102.37 | 0.26 |
| L89 | S/S | D/D | A/A | G/G | I/I | L/L | A/V | A/A | 183.6 | 435.68 | 0.42 |

Table 2 Caption: CES1 genotypes, CES1 protein expression, CES1 activity on enalapril metabolism, and the activity normalized by CES1 protein expression in human livers carrying CES1 nsSNPs

### Fig. 3.

![Fig. 3.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ace4/5637814/f5105acca044/dmd.117.077669f3.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5637814_dmd.117.077669f3.jpg)

(A and B) Hepatic CES1 activity on enalapril activation (A) and absolute CES1 expression (B) among different CES1 genotypes (i.e., S75N, n = 3; D203E-A269S, n = 1; S75N-D203E-A269S, n = 9; and WT, n = 33). The WT control contains 33 subjects who do not carry any nsSNPs. Statistical analysis was not performed for the D203E-A269S haplotype due to the very limited carrier number.

## Discussion

CES1, the most abundant hepatic enzyme in humans ([Achour et al., 2017](#B1)), catalyzes the hydrolysis of many clinically important drugs ([Shi et al., 2006](#B16), [2016a](#B17),[b](#B18),[c](#B19); [Zhu et al., 2008](#B31), [2013](#B32); [Wang et al., 2016b](#B25); [Stage et al., 2017](#B20)). Considerable interindividual variability in CES1 expression and activity has been consistently demonstrated by our laboratory and others ([Zhu et al., 2008](#B31), [2013](#B32); [Laizure et al., 2013](#B6); [Rasmussen et al., 2015](#B13); [Shi et al., 2016a](#B17); [Wang et al., 2016b](#B25); [Oh et al., 2017](#B12); [Stage et al., 2017](#B20)). Varied CES1 function may lead to variability in the pharmacokinetics and pharmacodynamics of drugs metabolized by CES1. *CES1* genetic variation has been established as a significant contributor to CES1 variability ([Zhu et al., 2008](#B31), [2013](#B32); [Rasmussen et al., 2015](#B13); [Shi et al., 2016a](#B17),[b](#B18),[c](#B19); [Wang et al., 2016b](#B25); [Stage et al., 2017](#B20)). Among thousands of variants identified within the *CES1* gene, nsSNPs are of particular interest because, relative to other types of genetic variants, nsSNPs are more likely to be functionally significant due to associated changes in amino acid sequences. The discovery of the loss-of-function variant G143E has exemplified the functional significance and clinical implications of *CES1* nsSNPs ([Zhu et al., 2008](#B31)). However, only a very few *CES1* nsSNPs have been studied so far ([Zhu et al., 2008](#B31); [Shi et al., 2016c](#B19); [Stage et al., 2017](#B20)), leaving a large portion of CES1 variability unexplained. Several in silico programs such as SIFT and Polyphen2 were developed to predict functional consequences of nsSNPs. However, our previous study demonstrated that the in silico analysis was not predictive of the effect of *CES1* nsSNPs on enzyme function ([Zhu et al., 2013](#B32)). In this study, we conducted a comprehensive in vitro functional study to determine the impact of *CES1* nsSNPs on CES1 expression and activity using *CES1* transfected cell lines and individual human liver tissues.

The MAFs of *CES1* nsSNPs vary markedly among different ethnic groups ([Table 1](#T1)); however, the majority of previous *CES1* pharmacogenomics studies have focused on common variants in Caucasians. This study included *CES1* nsSNPs with an MAF >0.5% in a given population to make the assessment applicable to all ethnic groups. Five *CES1* nsSNPs (i.e., L40Ter, E220G, A158V, R199H, and T290M) were functionally significant. L40Ter (MAF = 0.55% in Africans) was a loss-of-function variant for the hydrolysis of enalapril, clopidogrel, and sacubitril ([Fig. 1](#F1)). This variant resulted in a premature stop codon in exon2, abolishing the production of mature CES1 protein, although the *CES1* mRNA level in the L40Ter transfected cells was comparable to the WT control ([Fig. 2](#F2)). E220G showed a significant decreasing effect on the hydrolysis of all three tested CES1 substrates. Consistent with our in vitro data, E220G was predicted in silico to be deleterious to CES1 activity, as it resides near the active site ([Nzabonimpa et al., 2016](#B11)). Given that CES1 protein and mRNA levels were unaltered in E220G-expressing cells ([Fig. 2](#F2)), the effect of E220G on CES1 activity was likely due to the disruption of CES1-substrate bindings. The MAF of this variant was 0.55% in East Asians according to the dbSNP database; however, a recent study reported that this variant is very common in Koreans (MAF = 3.1%) ([Oh et al., 2017](#B12)). This study also revealed that E220G was associated with a greater systemic exposure to oseltamivir in healthy subjects who received a single dose of oseltamivir (75 mg). This clinical observation is in agreement with our in vitro study, indicating that E220G reduces CES1 activity.

### Fig. 2.

![Fig. 2.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ace4/5637814/f2879a13d09a/dmd.117.077669f2.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5637814_dmd.117.077669f2.jpg)

(A and B) CES1 protein (A) and mRNA (B) expression levels in the CES1 nsSNP and haplotype transfected cells. Relative CES1 protein and mRNA levels of the CES1 variants compared with WT CES1 were determined using LC-MS/MS and qRT-PCR assays, respectively. WT CES1 and vector-transfected cells were included as the positive and negative controls, respectively. Data are presented as means ± S.D. (n = 3). *P < 0.05; **P < 0.001 (CES1 variants versus WT CES1).

It is interesting that the effects of the *CES1* nsSNPs A158V, R199H, and T290M on CES1 activity are substrate dependent ([Fig. 1](#F1)). Enalapril hydrolysis was reduced to 46.6% ± 1.5% and 40.5% ± 2.0% of the WT enzyme by A158V and R199H, respectively, whereas the hydrolysis of clopidogrel and sacubitril was not affected by the variants. In addition, T290M decreased clopidogrel hydrolysis by 50.7% ± 16.8%, while having no effect on enalapril and sacubitril metabolism ([Fig. 1](#F1)). Given the high frequencies of A158V in South Asians (MAF = 6.69%) and R199H in Africans (MAF = 0.74%), A158V and R199H could be clinically significant for optimizing enalapril therapy in South Asians and Africans, respectively. T290M is relatively prevalent in Africans, South Asians, and Latinos, with an MAF of 1.24%, 2.86%, and 1.87%, respectively. It should be noted that >85% of ingested clopidogrel is metabolized by CES1, and the *CES1* nsSNP G143E has shown significant effects on clopidogrel activation as well as its therapeutic outcomes ([Lewis et al., 2013](#B8); [Zhu et al., 2013](#B32); [Tarkiainen et al., 2015a](#B22)). Therefore, T290M could be a significant contributor to the interindividual variation in clopidogrel therapy in relevant populations. It should be noted that the differential effect of *CES1* genetic polymorphisms on different CES1 substrates has been observed in previous studies. For example, G143E was a loss-of-function variant for the metabolism of methylphenidate, clopidogrel, enalapril, and dabigatran but a decreased function variant for oseltamivir metabolism ([Zhu et al., 2008](#B31); [Zhu and Markowitz, 2009](#B30); [Shi et al., 2016b](#B18); [Wang et al., 2016b](#B25)). Similar to the G143E nsSNP, A158V, R199H, and T290M diminished CES1 activity without altering the mRNA or protein expression levels ([Figs. 1](#F1)–[3](#F3)), indicating that these variants might directly affect the enzyme-substrate interaction. Given that chemical structures differ significantly among enalapril, clopidogrel, and sacubitril, we speculate that the involvement of amino acid residues in catalyzing the cleavage of the ester bonds of these compounds may vary, which may have resulted in the observed substrate-dependent effect of the three nsSNPs.

A recent healthy volunteer study reported that the area under the curve of d-methylphenidate plasma concentrations was increased by 68% in A269S carriers compared with noncarriers after subjects were administered a single dose of 10 mg methylphenidate ([Lyauk et al., 2016](#B9)). However, our in vitro studies of transfected cells and human livers showed that A269S was not associated with CES1 expression or activity. Given the fact that A269S is in complete LD (D’ = 1, *R*^2^ = 1) with D203E and is in high LD with S75N (D’ = 0.664, *R*^2^ = 0.399), we established cell lines that stably expressed the haplotypes D203E-A269S and S75N-D203E-A269S and we evaluated the impact of the haplotypes on CES1 expression and the metabolism of the CES1 substrates clopidogrel, enalapril, and sacubitril. We also genotyped a large set of human liver tissues for the *CES1* nsSNPs and haplotypes and determined the CES1 expression and activity on enalapril hydrolysis. As shown in [Figs. 1](#F1)–[3](#F3), neither D203E-A269S nor S75N-D203E-A269S exhibited a significant effect on CES1 protein expression or activity in the transfected cells and human liver tissues.

Eight *CES1* nsSNPs, including G142E, G143E, G147E, Q169P, Y170D, R171C, G173D, and E220G, are located within a 5-Å distance from the CES1 active site, according to analysis of the CES1 three-dimensional structure ([Nzabonimpa et al., 2016](#B11)). One of the variants, G143E, showed a profound impact on CES1 activity and consequently impaired the metabolism of several CES1 substrate drugs ([Zhu et al., 2008](#B31), [2009b](#B29); [Nemoda et al., 2009](#B10); [Zhu and Markowitz, 2009](#B30); [Lewis et al., 2013](#B8); [Tarkiainen et al., 2015a](#B22),[b](#B23); [Shi et al., 2016b](#B18); [Wang et al., 2016a](#B24)). In this study, stable cell lines transfected with CES1 variants were developed to assess the functions of the rest of the seven *CES1* nsSNPs located in close proximity to the CES1 active site. As expected, all seven nsSNPs affected CES1 activity to a certain extent, although effect magnitude and substrate specificity varied among the nsSNPs ([Fig. 1](#F1)).

CES1 protein and mRNA expression levels are highly correlated among the transfected cell lines with the exception of the L40T, Y170D, and R171C variants ([Supplemental Fig. 1](http://dmd.aspetjournals.org/lookup/suppl/doi:10.1124/dmd.117.077669/-/DC1)). The three variants markedly impaired CES1 protein expression without affecting mRNA levels ([Fig. 2](#F2)). As expected, the correlation between CES1 activity and protein expression was significantly higher than that between the activity and mRNA expression for the tested variants ([Supplemental Fig. 2](http://dmd.aspetjournals.org/lookup/suppl/doi:10.1124/dmd.117.077669/-/DC1)). Furthermore, a number of nsSNPs, such as G142E and E220G, significantly attenuated CES1 activity while imposing no effect on CES1 protein and mRNA expression levels.

In summary, to our knowledge, this study is the first comprehensive functional assessment of *CES1* nonsynonymous variants and haplotypes and reveals a number of nsSNPs that exhibit significant effects on the metabolism of CES1 substrates. Clinical investigations are warranted to determine the influence of the identified functional nsSNPs on the pharmacokinetics and pharmacodynamics of medications metabolized by CES1. It is noted that the functional nsSNPs identified from this study may only explain a small portion of CES1 variability based on the frequencies of the variants. Thus, further investigations are needed to elucidate genetic and nongenetic elements regulating this important drug-metabolizing enzyme.

## Abbreviations

## Authorship Contributions

*Participated in research design:* Wang, Shi, Wu, Bleske, Zhu.

*Conducted experiments:* Wang, Rida, Shi.

*Performed data analysis:* Wang, Shi, Zhu.

*Wrote or contributed to the writing of the manuscript:* Wang, Rida, Shi, Wu, Bleske, Zhu.

## Footnotes

## References

1. Achour B, Al Feteisi H, Lanucara F, Rostami-Hodjegan A, Barber J. (2017) Global proteomic analysis of human liver microsomes: rapid characterization and quantification of hepatic drug-metabolizing enzymes. Drug Metab Dispos 45:666–675.  [DOI](https://doi.org/10.1124/dmd.116.074732) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28373266/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Global%20proteomic%20analysis%20of%20human%20liver%20microsomes:%20rapid%20characterization%20and%20quantification%20of%20hepatic%20drug-metabolizing%20enzymes&author=B%20Achour&author=H%20Al%20Feteisi&author=F%20Lanucara&author=A%20Rostami-Hodjegan&author=J%20Barber&volume=45&publication_year=2017&pages=666-675&pmid=28373266&doi=10.1124/dmd.116.074732&)

2. Geshi E, Kimura T, Yoshimura M, Suzuki H, Koba S, Sakai T, Saito T, Koga A, Muramatsu M, Katagiri T. (2005) A single nucleotide polymorphism in the carboxylesterase gene is associated with the responsiveness to imidapril medication and the promoter activity. Hypertens Res 28:719–725.  [DOI](https://doi.org/10.1291/hypres.28.719) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16419644/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hypertens%20Res&title=A%20single%20nucleotide%20polymorphism%20in%20the%20carboxylesterase%20gene%20is%20associated%20with%20the%20responsiveness%20to%20imidapril%20medication%20and%20the%20promoter%20activity&author=E%20Geshi&author=T%20Kimura&author=M%20Yoshimura&author=H%20Suzuki&author=S%20Koba&volume=28&publication_year=2005&pages=719-725&pmid=16419644&doi=10.1291/hypres.28.719&)

3. Hagihara K, Kazui M, Kurihara A, Yoshiike M, Honda K, Okazaki O, Farid NA, Ikeda T. (2009) A possible mechanism for the differences in efficiency and variability of active metabolite formation from thienopyridine antiplatelet agents, prasugrel and clopidogrel. Drug Metab Dispos 37:2145–2152.  [DOI](https://doi.org/10.1124/dmd.109.028498) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19704027/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=A%20possible%20mechanism%20for%20the%20differences%20in%20efficiency%20and%20variability%20of%20active%20metabolite%20formation%20from%20thienopyridine%20antiplatelet%20agents,%20prasugrel%20and%20clopidogrel&author=K%20Hagihara&author=M%20Kazui&author=A%20Kurihara&author=M%20Yoshiike&author=K%20Honda&volume=37&publication_year=2009&pages=2145-2152&pmid=19704027&doi=10.1124/dmd.109.028498&)

4. Hosokawa M, Endo T, Fujisawa M, Hara S, Iwata N, Sato Y, Satoh T. (1995) Interindividual variation in carboxylesterase levels in human liver microsomes. Drug Metab Dispos 23:1022–1027.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/8654188/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Interindividual%20variation%20in%20carboxylesterase%20levels%20in%20human%20liver%20microsomes&author=M%20Hosokawa&author=T%20Endo&author=M%20Fujisawa&author=S%20Hara&author=N%20Iwata&volume=23&publication_year=1995&pages=1022-1027&pmid=8654188&)

5. Imai T. (2006) Human carboxylesterase isozymes: catalytic properties and rational drug design. Drug Metab Pharmacokinet 21:173–185.  [DOI](https://doi.org/10.2133/dmpk.21.173) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16858120/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Pharmacokinet&title=Human%20carboxylesterase%20isozymes:%20catalytic%20properties%20and%20rational%20drug%20design&author=T%20Imai&volume=21&publication_year=2006&pages=173-185&pmid=16858120&doi=10.2133/dmpk.21.173&)

6. Laizure SC, Herring V, Hu Z, Witbrodt K, Parker RB. (2013) The role of human carboxylesterases in drug metabolism: have we overlooked their importance? Pharmacotherapy 33:210–222.  [DOI](https://doi.org/10.1002/phar.1194) | [PMC free article](/articles/PMC4572478/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23386599/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacotherapy&title=The%20role%20of%20human%20carboxylesterases%20in%20drug%20metabolism:%20have%20we%20overlooked%20their%20importance?&author=SC%20Laizure&author=V%20Herring&author=Z%20Hu&author=K%20Witbrodt&author=RB%20Parker&volume=33&publication_year=2013&pages=210-222&pmid=23386599&doi=10.1002/phar.1194&)

7. Laizure SC, Parker RB, Herring VL, Hu Z-Y. (2014) Identification of carboxylesterase-dependent dabigatran etexilate hydrolysis. Drug Metab Dispos 42:201–206.  [DOI](https://doi.org/10.1124/dmd.113.054353) | [PMC free article](/articles/PMC3912543/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24212379/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Identification%20of%20carboxylesterase-dependent%20dabigatran%20etexilate%20hydrolysis&author=SC%20Laizure&author=RB%20Parker&author=VL%20Herring&author=Z-Y%20Hu&volume=42&publication_year=2014&pages=201-206&pmid=24212379&doi=10.1124/dmd.113.054353&)

8. Lewis JP, Horenstein RB, Ryan K, O’Connell JR, Gibson Q, Mitchell BD, Tanner K, Chai S, Bliden KP, Tantry US, et al. (2013) The functional G143E variant of carboxylesterase 1 is associated with increased clopidogrel active metabolite levels and greater clopidogrel response. Pharmacogenet Genomics 23:1–8.  [DOI](https://doi.org/10.1097/FPC.0b013e32835aa8a2) | [PMC free article](/articles/PMC3682407/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23111421/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=The%20functional%20G143E%20variant%20of%20carboxylesterase%201%20is%20associated%20with%20increased%20clopidogrel%20active%20metabolite%20levels%20and%20greater%20clopidogrel%20response&author=JP%20Lewis&author=RB%20Horenstein&author=K%20Ryan&author=JR%20O%E2%80%99Connell&author=Q%20Gibson&volume=23&publication_year=2013&pages=1-8&pmid=23111421&doi=10.1097/FPC.0b013e32835aa8a2&)

9. Lyauk YK, Stage C, Bergmann TK, Ferrero-Milliani L, Bjerre D, Thomsen R, Dalhoff KP, Rasmussen HB, Jürgens G. (2016) Population pharmacokinetics of methylphenidate in healthy adults emphasizing novel and known effects of several carboxylesterase 1 (CES1) variants. Clin Transl Sci 9:337–345.  [DOI](https://doi.org/10.1111/cts.12423) | [PMC free article](/articles/PMC5351003/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27754602/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Transl%20Sci&title=Population%20pharmacokinetics%20of%20methylphenidate%20in%20healthy%20adults%20emphasizing%20novel%20and%20known%20effects%20of%20several%20carboxylesterase%201%20(CES1)%20variants&author=YK%20Lyauk&author=C%20Stage&author=TK%20Bergmann&author=L%20Ferrero-Milliani&author=D%20Bjerre&volume=9&publication_year=2016&pages=337-345&pmid=27754602&doi=10.1111/cts.12423&)

10. Nemoda Z, Angyal N, Tarnok Z, Gadoros J, Sasvari-Szekely M. (2009) Carboxylesterase 1 gene polymorphism and methylphenidate response in ADHD. Neuropharmacology 57:731–733.  [DOI](https://doi.org/10.1016/j.neuropharm.2009.08.014) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19733552/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neuropharmacology&title=Carboxylesterase%201%20gene%20polymorphism%20and%20methylphenidate%20response%20in%20ADHD&author=Z%20Nemoda&author=N%20Angyal&author=Z%20Tarnok&author=J%20Gadoros&author=M%20Sasvari-Szekely&volume=57&publication_year=2009&pages=731-733&pmid=19733552&doi=10.1016/j.neuropharm.2009.08.014&)

11. Nzabonimpa GS, Rasmussen HB, Brunak S, Taboureau O, and Consortium I (2016) Investigating the impact of missense mutations in hCES1 by in silico structure-based approaches. Drug Metab Pers Ther DOI: 10.1515/dmpt-2015-0034 [published ahead of print].  [DOI](https://doi.org/10.1515/dmpt-2015-0034) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26900165/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Nzabonimpa%20GS,%20Rasmussen%20HB,%20Brunak%20S,%20Taboureau%20O,%20and%20Consortium%20I%20(2016)%20Investigating%20the%20impact%20of%20missense%20mutations%20in%20hCES1%20by%20in%20silico%20structure-based%20approaches.%20Drug%20Metab%20Pers%20Ther%20DOI:%2010.1515/dmpt-2015-0034%20%5Bpublished%20ahead%20of%20print%5D.)

12. Oh J, Lee S, Lee H, Cho JY, Yoon SH, Jang IJ, Yu KS, Lim KS. (2017) The novel carboxylesterase 1 variant c.662A>G may decrease the bioactivation of oseltamivir in humans. PLoS One 12:e0176320.  [DOI](https://doi.org/10.1371/journal.pone.0176320) | [PMC free article](/articles/PMC5402961/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28437488/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=The%20novel%20carboxylesterase%201%20variant%20c.662A>G%20may%20decrease%20the%20bioactivation%20of%20oseltamivir%20in%20humans&author=J%20Oh&author=S%20Lee&author=H%20Lee&author=JY%20Cho&author=SH%20Yoon&volume=12&publication_year=2017&pages=e0176320&pmid=28437488&doi=10.1371/journal.pone.0176320&)

13. Rasmussen HB, Bjerre D, Linnet K, Jürgens G, Dalhoff K, Stefansson H, Hankemeier T, Kaddurah-Daouk R, Taboureau O, Brunak S, et al. INDICES Consortium (2015) Individualization of treatments with drugs metabolized by CES1: combining genetics and metabolomics. Pharmacogenomics 16:649–665.  [DOI](https://doi.org/10.2217/pgs.15.7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25896426/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Individualization%20of%20treatments%20with%20drugs%20metabolized%20by%20CES1:%20combining%20genetics%20and%20metabolomics&author=HB%20Rasmussen&author=D%20Bjerre&author=K%20Linnet&author=G%20J%C3%BCrgens&author=K%20Dalhoff&volume=16&publication_year=2015&pages=649-665&pmid=25896426&doi=10.2217/pgs.15.7&)

14. Ross MK, Borazjani A, Wang R, Crow JA, Xie S. (2012) Examination of the carboxylesterase phenotype in human liver. Arch Biochem Biophys 522:44–56.  [DOI](https://doi.org/10.1016/j.abb.2012.04.010) | [PMC free article](/articles/PMC3569858/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22525521/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arch%20Biochem%20Biophys&title=Examination%20of%20the%20carboxylesterase%20phenotype%20in%20human%20liver&author=MK%20Ross&author=A%20Borazjani&author=R%20Wang&author=JA%20Crow&author=S%20Xie&volume=522&publication_year=2012&pages=44-56&pmid=22525521&doi=10.1016/j.abb.2012.04.010&)

15. Sai K, Saito Y, Tatewaki N, Hosokawa M, Kaniwa N, Nishimaki-Mogami T, Naito M, Sawada J, Shirao K, Hamaguchi T, et al. (2010) Association of carboxylesterase 1A genotypes with irinotecan pharmacokinetics in Japanese cancer patients. Br J Clin Pharmacol 70:222–233.  [DOI](https://doi.org/10.1111/j.1365-2125.2010.03695.x) | [PMC free article](/articles/PMC2911552/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20653675/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Clin%20Pharmacol&title=Association%20of%20carboxylesterase%201A%20genotypes%20with%20irinotecan%20pharmacokinetics%20in%20Japanese%20cancer%20patients&author=K%20Sai&author=Y%20Saito&author=N%20Tatewaki&author=M%20Hosokawa&author=N%20Kaniwa&volume=70&publication_year=2010&pages=222-233&pmid=20653675&doi=10.1111/j.1365-2125.2010.03695.x&)

16. Shi D, Yang J, Yang D, LeCluyse EL, Black C, You L, Akhlaghi F, Yan B. (2006) Anti-influenza prodrug oseltamivir is activated by carboxylesterase human carboxylesterase 1, and the activation is inhibited by antiplatelet agent clopidogrel. J Pharmacol Exp Ther 319:1477–1484.  [DOI](https://doi.org/10.1124/jpet.106.111807) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16966469/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharmacol%20Exp%20Ther&title=Anti-influenza%20prodrug%20oseltamivir%20is%20activated%20by%20carboxylesterase%20human%20carboxylesterase%201,%20and%20the%20activation%20is%20inhibited%20by%20antiplatelet%20agent%20clopidogrel&author=D%20Shi&author=J%20Yang&author=D%20Yang&author=EL%20LeCluyse&author=C%20Black&volume=319&publication_year=2006&pages=1477-1484&pmid=16966469&doi=10.1124/jpet.106.111807&)

17. Shi J, Wang X, Eyler RF, Liang Y, Liu L, Mueller BA, Zhu HJ. (2016a) Association of oseltamivir activation with gender and carboxylesterase 1 genetic polymorphisms. Basic Clin Pharmacol Toxicol 119:555–561.  [DOI](https://doi.org/10.1111/bcpt.12625) | [PMC free article](/articles/PMC5118077/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27228223/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Basic%20Clin%20Pharmacol%20Toxicol&title=Association%20of%20oseltamivir%20activation%20with%20gender%20and%20carboxylesterase%201%20genetic%20polymorphisms.&author=J%20Shi&author=X%20Wang&author=RF%20Eyler&author=Y%20Liang&author=L%20Liu&volume=119&publication_year=2016&pages=555-561&pmid=27228223&doi=10.1111/bcpt.12625&)

18. Shi J, Wang X, Nguyen J, Wu AH, Bleske BE, Zhu HJ. (2016b) Sacubitril is selectively activated by carboxylesterase 1 (CES1) in the liver and the activation is affected by CES1 genetic variation. Drug Metab Dispos 44:554–559.  [DOI](https://doi.org/10.1124/dmd.115.068536) | [PMC free article](/articles/PMC4810765/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26817948/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Sacubitril%20is%20selectively%20activated%20by%20carboxylesterase%201%20(CES1)%20in%20the%20liver%20and%20the%20activation%20is%20affected%20by%20CES1%20genetic%20variation&author=J%20Shi&author=X%20Wang&author=J%20Nguyen&author=AH%20Wu&author=BE%20Bleske&volume=44&publication_year=2016&pages=554-559&pmid=26817948&doi=10.1124/dmd.115.068536&)

19. Shi J, Wang X, Nguyen JH, Bleske BE, Liang Y, Liu L, Zhu HJ. (2016c) Dabigatran etexilate activation is affected by the CES1 genetic polymorphism G143E (rs71647871) and gender. Biochem Pharmacol 119:76–84.  [DOI](https://doi.org/10.1016/j.bcp.2016.09.003) | [PMC free article](/articles/PMC5061634/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27614009/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem%20Pharmacol&title=Dabigatran%20etexilate%20activation%20is%20affected%20by%20the%20CES1%20genetic%20polymorphism%20G143E%20(rs71647871)%20and%20gender&author=J%20Shi&author=X%20Wang&author=JH%20Nguyen&author=BE%20Bleske&author=Y%20Liang&volume=119&publication_year=2016&pages=76-84&pmid=27614009&doi=10.1016/j.bcp.2016.09.003&)

20. Stage C, Jürgens G, Guski LS, Thomsen R, Bjerre D, Ferrero-Miliani L, Lyauk YK, Rasmussen HB, Dalhoff K, INDICES Consortium (2017) The impact of CES1 genotypes on the pharmacokinetics of methylphenidate in healthy Danish subjects. Br J Clin Pharmacol 83:1506–1514.  [DOI](https://doi.org/10.1111/bcp.13237) | [PMC free article](/articles/PMC5465325/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28087982/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Clin%20Pharmacol&title=The%20impact%20of%20CES1%20genotypes%20on%20the%20pharmacokinetics%20of%20methylphenidate%20in%20healthy%20Danish%20subjects&author=C%20Stage&author=G%20J%C3%BCrgens&author=LS%20Guski&author=R%20Thomsen&author=D%20Bjerre&volume=83&publication_year=2017&pages=1506-1514&pmid=28087982&doi=10.1111/bcp.13237&)

21. Tarkiainen EK, Backman JT, Neuvonen M, Neuvonen PJ, Schwab M, Niemi M. (2012) Carboxylesterase 1 polymorphism impairs oseltamivir bioactivation in humans. Clin Pharmacol Ther 92:68–71.  [DOI](https://doi.org/10.1038/clpt.2012.13) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22588607/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Carboxylesterase%201%20polymorphism%20impairs%20oseltamivir%20bioactivation%20in%20humans&author=EK%20Tarkiainen&author=JT%20Backman&author=M%20Neuvonen&author=PJ%20Neuvonen&author=M%20Schwab&volume=92&publication_year=2012&pages=68-71&pmid=22588607&doi=10.1038/clpt.2012.13&)

22. Tarkiainen EK, Holmberg MT, Tornio A, Neuvonen M, Neuvonen PJ, Backman JT, Niemi M. (2015a) Carboxylesterase 1 c.428G>A single nucleotide variation increases the antiplatelet effects of clopidogrel by reducing its hydrolysis in humans. Clin Pharmacol Ther 97:650–658.  [DOI](https://doi.org/10.1002/cpt.101) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25704243/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Carboxylesterase%201%20c.428G>A%20single%20nucleotide%20variation%20increases%20the%20antiplatelet%20effects%20of%20clopidogrel%20by%20reducing%20its%20hydrolysis%20in%20humans&author=EK%20Tarkiainen&author=MT%20Holmberg&author=A%20Tornio&author=M%20Neuvonen&author=PJ%20Neuvonen&volume=97&publication_year=2015&pages=650-658&pmid=25704243&doi=10.1002/cpt.101&)

23. Tarkiainen EK, Tornio A, Holmberg MT, Launiainen T, Neuvonen PJ, Backman JT, Niemi M. (2015b) Effect of carboxylesterase 1 c.428G > A single nucleotide variation on the pharmacokinetics of quinapril and enalapril. Br J Clin Pharmacol 80:1131–1138.  [DOI](https://doi.org/10.1111/bcp.12667) | [PMC free article](/articles/PMC4631185/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25919042/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Clin%20Pharmacol&title=Effect%20of%20carboxylesterase%201%20c.428G%20>%20A%20single%20nucleotide%20variation%20on%20the%20pharmacokinetics%20of%20quinapril%20and%20enalapril&author=EK%20Tarkiainen&author=A%20Tornio&author=MT%20Holmberg&author=T%20Launiainen&author=PJ%20Neuvonen&volume=80&publication_year=2015&pages=1131-1138&pmid=25919042&doi=10.1111/bcp.12667&)

24. Wang X, Liang Y, Liu L, Shi J, Zhu HJ. (2016a) Targeted absolute quantitative proteomics with SILAC internal standards and unlabeled full-length protein calibrators (TAQSI). Rapid Commun Mass Spectrom 30:553–561.  [DOI](https://doi.org/10.1002/rcm.7482) | [PMC free article](/articles/PMC6301059/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26842578/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Rapid%20Commun%20Mass%20Spectrom&title=Targeted%20absolute%20quantitative%20proteomics%20with%20SILAC%20internal%20standards%20and%20unlabeled%20full-length%20protein%20calibrators%20(TAQSI)&author=X%20Wang&author=Y%20Liang&author=L%20Liu&author=J%20Shi&author=HJ%20Zhu&volume=30&publication_year=2016&pages=553-561&pmid=26842578&doi=10.1002/rcm.7482&)

25. Wang X, Wang G, Shi J, Aa J, Comas R, Liang Y, Zhu HJ. (2016b) CES1 genetic variation affects the activation of angiotensin-converting enzyme inhibitors. Pharmacogenomics J 16:220–230.  [DOI](https://doi.org/10.1038/tpj.2015.42) | [PMC free article](/articles/PMC6329299/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26076923/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=CES1%20genetic%20variation%20affects%20the%20activation%20of%20angiotensin-converting%20enzyme%20inhibitors&author=X%20Wang&author=G%20Wang&author=J%20Shi&author=J%20Aa&author=R%20Comas&volume=16&publication_year=2016&pages=220-230&pmid=26076923&doi=10.1038/tpj.2015.42&)

26. Yang D, Pearce RE, Wang X, Gaedigk R, Wan YJ, Yan B. (2009) Human carboxylesterases HCE1 and HCE2: ontogenic expression, inter-individual variability and differential hydrolysis of oseltamivir, aspirin, deltamethrin and permethrin. Biochem Pharmacol 77:238–247.  [DOI](https://doi.org/10.1016/j.bcp.2008.10.005) | [PMC free article](/articles/PMC2671154/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18983829/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem%20Pharmacol&title=Human%20carboxylesterases%20HCE1%20and%20HCE2:%20ontogenic%20expression,%20inter-individual%20variability%20and%20differential%20hydrolysis%20of%20oseltamivir,%20aspirin,%20deltamethrin%20and%20permethrin&author=D%20Yang&author=RE%20Pearce&author=X%20Wang&author=R%20Gaedigk&author=YJ%20Wan&volume=77&publication_year=2009&pages=238-247&pmid=18983829&doi=10.1016/j.bcp.2008.10.005&)

27. Yoshimura M, Kimura T, Ishii M, Ishii K, Matsuura T, Geshi E, Hosokawa M, Muramatsu M. (2008) Functional polymorphisms in carboxylesterase1A2 (CES1A2) gene involves specific protein 1 (Sp1) binding sites. Biochem Biophys Res Commun 369:939–942.  [DOI](https://doi.org/10.1016/j.bbrc.2008.02.120) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18328811/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem%20Biophys%20Res%20Commun&title=Functional%20polymorphisms%20in%20carboxylesterase1A2%20(CES1A2)%20gene%20involves%20specific%20protein%201%20(Sp1)%20binding%20sites&author=M%20Yoshimura&author=T%20Kimura&author=M%20Ishii&author=K%20Ishii&author=T%20Matsuura&volume=369&publication_year=2008&pages=939-942&pmid=18328811&doi=10.1016/j.bbrc.2008.02.120&)

28. Zhu HJ, Appel DI, Jiang Y, Markowitz JS. (2009a) Age- and sex-related expression and activity of carboxylesterase 1 and 2 in mouse and human liver. Drug Metab Dispos 37:1819–1825.  [DOI](https://doi.org/10.1124/dmd.109.028209) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19487248/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Age-%20and%20sex-related%20expression%20and%20activity%20of%20carboxylesterase%201%20and%202%20in%20mouse%20and%20human%20liver&author=HJ%20Zhu&author=DI%20Appel&author=Y%20Jiang&author=JS%20Markowitz&volume=37&publication_year=2009&pages=1819-1825&pmid=19487248&doi=10.1124/dmd.109.028209&)

29. Zhu HJ, Appel DI, Johnson JA, Chavin KD, Markowitz JS. (2009b) Role of carboxylesterase 1 and impact of natural genetic variants on the hydrolysis of trandolapril. Biochem Pharmacol 77:1266–1272.  [DOI](https://doi.org/10.1016/j.bcp.2008.12.017) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19185566/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem%20Pharmacol&title=Role%20of%20carboxylesterase%201%20and%20impact%20of%20natural%20genetic%20variants%20on%20the%20hydrolysis%20of%20trandolapril&author=HJ%20Zhu&author=DI%20Appel&author=JA%20Johnson&author=KD%20Chavin&author=JS%20Markowitz&volume=77&publication_year=2009&pages=1266-1272&pmid=19185566&doi=10.1016/j.bcp.2008.12.017&)

30. Zhu HJ, Markowitz JS. (2009) Activation of the antiviral prodrug oseltamivir is impaired by two newly identified carboxylesterase 1 variants. Drug Metab Dispos 37:264–267.  [DOI](https://doi.org/10.1124/dmd.108.024943) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19022936/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Activation%20of%20the%20antiviral%20prodrug%20oseltamivir%20is%20impaired%20by%20two%20newly%20identified%20carboxylesterase%201%20variants&author=HJ%20Zhu&author=JS%20Markowitz&volume=37&publication_year=2009&pages=264-267&pmid=19022936&doi=10.1124/dmd.108.024943&)

31. Zhu HJ, Patrick KS, Yuan HJ, Wang JS, Donovan JL, DeVane CL, Malcolm R, Johnson JA, Youngblood GL, Sweet DH, et al. (2008) Two CES1 gene mutations lead to dysfunctional carboxylesterase 1 activity in man: clinical significance and molecular basis. Am J Hum Genet 82:1241–1248.  [DOI](https://doi.org/10.1016/j.ajhg.2008.04.015) | [PMC free article](/articles/PMC2427248/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18485328/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=Two%20CES1%20gene%20mutations%20lead%20to%20dysfunctional%20carboxylesterase%201%20activity%20in%20man:%20clinical%20significance%20and%20molecular%20basis&author=HJ%20Zhu&author=KS%20Patrick&author=HJ%20Yuan&author=JS%20Wang&author=JL%20Donovan&volume=82&publication_year=2008&pages=1241-1248&pmid=18485328&doi=10.1016/j.ajhg.2008.04.015&)

32. Zhu HJ, Wang X, Gawronski BE, Brinda BJ, Angiolillo DJ, Markowitz JS. (2013) Carboxylesterase 1 as a determinant of clopidogrel metabolism and activation. J Pharmacol Exp Ther 344:665–672.  [DOI](https://doi.org/10.1124/jpet.112.201640) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23275066/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharmacol%20Exp%20Ther&title=Carboxylesterase%201%20as%20a%20determinant%20of%20clopidogrel%20metabolism%20and%20activation&author=HJ%20Zhu&author=X%20Wang&author=BE%20Gawronski&author=BJ%20Brinda&author=DJ%20Angiolillo&volume=344&publication_year=2013&pages=665-672&pmid=23275066&doi=10.1124/jpet.112.201640&)
