# Effect of genetic variation in UGT1A and ABCB1 on moxifloxacin pharmacokinetics in South African patients with tuberculosis

## Metadata
**Authors:** Anushka Naidoo, Veron Ramsuran, Maxwell Chirehwa, Paolo Denti, Helen McIlleron, Kogieleum Naidoo, Nonhlanhla Yende-Zuma, Ravesh Singh, Sinaye Ngcapu, Mamoonah Chaudhry, Michael S Pepper, Nesri Padayatchi
**Journal:** Pharmacogenomics
**Date:** 2017 Dec 6
**DOI:** [10.2217/pgs-2017-0144](https://doi.org/10.2217/pgs-2017-0144)
**PMID:** 29210323
**PMCID:** PMC5753622
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5753622/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC5753622/pdf/pgs-19-17.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC5753622/pdf/pgs-19-17.pdf)

## Abstract

**Aim:** 
We assessed the effect of genetic variability in UGT1A and ABCB1 genes on moxifloxacin pharmacokinetics.

**Methods:** 
Genotypes for selected UGT1A and ABCB1 SNPs were determined using a TaqMan® Genotyping OpenArray™ and high-resolution melt analysis for rs8175347. A nonlinear mixed-effects model was used to describe moxifloxacin pharmacokinetics.

**Results:** 
Genotypes of UGT1A SNPs, rs8175347 and rs3755319 (20.6% lower and 11.6% increased clearance, respectively) and ABCB1 SNP rs2032582 (40% reduced bioavailability in one individual) were significantly associated with changes in moxifloxacin pharmacokinetic parameters.

**Conclusion:** 
Genetic variation in UGT1A as represented by rs8175347 to a lesser extent rs3755319 and the ABCB1 rs2032582 SNP is modestly associated with the interindividual variability reported in moxifloxacin pharmacokinetics and exposure. Clinical relevance of the effects of genetic variation on moxifloxacin pharmacokinetic requires further investigation.

Keywords: : ABCB1, moxifloxacin, pharmacogenetics, pharmacokinetics, UGT1A

### Aim:

We assessed the effect of genetic variability in *UGT1A* and *ABCB1* genes on moxifloxacin pharmacokinetics.

### Methods:

Genotypes for selected *UGT1A* and *ABCB1* SNPs were determined using a TaqMan^®^ Genotyping OpenArray^™^ and high-resolution melt analysis for rs8175347. A nonlinear mixed-effects model was used to describe moxifloxacin pharmacokinetics.

### Results:

Genotypes of *UGT1A* SNPs, rs8175347 and rs3755319 (20.6% lower and 11.6% increased clearance, respectively) and *ABCB1* SNP rs2032582 (40% reduced bioavailability in one individual) were significantly associated with changes in moxifloxacin pharmacokinetic parameters.

### Conclusion:

Genetic variation in *UGT1A* as represented by rs8175347 to a lesser extent rs3755319 and the *ABCB1* rs2032582 SNP is modestly associated with the interindividual variability reported in moxifloxacin pharmacokinetics and exposure. Clinical relevance of the effects of genetic variation on moxifloxacin pharmacokinetic requires further investigation.

## Methods

A prospective PK substudy was conducted within the ongoing Improving Retreatment Success (IMPRESS) open-label randomized controlled trial ([NCT02114684](https://clinicaltrials.gov/ct2/show/NCT02114684)) from October 2013 in KwaZulu-Natal, Durban, South Africa. Details on study objectives, design and inclusion or exclusion criteria are available at [ClinicalTrials.gov](http://ClinicalTrials.gov) ([https://clinicaltrials.gov/ct2/show/NCT02114684](https://clinicaltrials.gov/ct2/show/NCT02114684)).

Patients in the intervention arm of the study receiving moxifloxacin, who provided informed consent to be included in the PK substudy, had blood samples collected for PK analysis at predefined time points. Baseline whole blood samples were collected in all patients consenting to storage of samples for later genetic testing. All participants recruited to the study were: >18 years of age, had a confirmed history of TB within the last 3 years and had been diagnosed with sputum smear positive, rifampicin sensitive, *M. tuberculosis* based on microscopy and GeneXpert technology. Only those with no predefined laboratory or clinical abnormalities were included, regardless of HIV status.

### Drug regimens

Patients randomized to the intervention arm of the study received daily moxifloxacin 400 mg (Avelox^®^ – Bayer Healthcare, Leverkusen, Germany), weight-based rifampicin 450 or 600 mg and isoniazid 225 or 300 mg, for patients 38–54 kg and ≥55 kg, respectively. Rifampicin, isoniazid and moxifloxacin were included during the 2-month intensive phase and 4-month continuation phase of TB treatment. Pyrazinamide 1500 and 2000 mg, in patients between 38–54 kg and ≥55 kg, respectively, was used during the intensive phase of treatment.

### Follow-up

Patients were followed up for 24 months, and clinical and safety monitoring was done bimonthly for the first 6 months, or as clinically indicated. Adherence to TB treatment was measured using pill count based on the number of tablets dispensed, physically returned, reported remaining or lost as well as participant self-report of missed or incomplete doses in the 4 days prior to the day of study visit or PK sampling. HIV co-infected patients received standard first-line antiretroviral therapy (ART) containing efavirenz, emtricitabine and tenofovir. Treatment and prophylaxis for opportunistic infections and concomitant treatment used were recorded on case report forms.

### PK sample collection

Plasma samples were collected prior to drug dose and at 2.5, 6 and 24 h post dose at month 1 and/or 2 during the intensive phase of TB treatment, at month 6 during the continuation phase of TB treatment and at approximately 4 weeks after completion of TB treatment following a single dose of moxifloxacin. Plasma, collected in ethylenediaminetetraacetic acid (EDTA) tubes, was centrifuged at 3000 r.p.m., placed on ice, sent within 1 h of collection to the CAPRISA laboratory to be stored at -80° C. Moxifloxacin concentrations were quantified in clinical plasma samples at the KwaZulu-Natal Research Institute for tuberculosis and HIV (KRITH) pharmacology laboratory using validated HPLC and tandem mass spectrometry. The bioanalytical method was developed and validated according to US FDA guidelines (2011) [[25](#B25)]. Sample preparation included a protein precipitation with acetonitrile and subsequent dilution with water. Analytes were chromatographically separated using a Zorbax C18, 3.5 μm, 50 mm × 2.1 mm column and detected using the ABI Sciex 5500 (AB Sciex LLC, MA, USA). QTrap mass spectrometer (AB Sciex LLC) operated in positive mode. The following transitions were used: precursor ion → product ion (all in units of m/z): moxifloxacin: 402.1 → 358.2 and 402.1 → 364.1. The internal standard used was ciprofloxacin: 331.6 → 231.0 and 331.6 → 288.1. Moxifloxacin was analyzed isocratically with a 22% acetonitrile/water/0.1% formic acid mobile phase. The injection volume was 2 μl and the total analytical run time was 5 min. The method was validated over the concentration range of 50–5000 ng/ml. Overall precision, based on quality control samples evaluated at low, medium and high concentrations during the validation and analysis of samples ranged from 8.4 to 19.4% and accuracy ranged from 101.9 to 105%. Calculated carry over at the lower limit of quantification was 5.4%. The liquid chromatography–tandem mass spectrometry system was interfaced with a DELL^®^ Windows^®^7 computer running Analyst^®^ software version 1.6.2, which was used for chromatographic data acquisition, peak integration and quantification of analytes.

### Genotyping

A total of 14 SNPs, six for *ABCB1* and eight for *UGT1A*, were selected for analysis in this study based on previous evidence of functional significance on drug response using the PharmGKB database and relevant literature [[18](#B18),[24](#B24),[26](#B26)]. The SNPs selected included rs10276036, rs1128503, rs2032582, rs1045642, rs2235033 and rs2235013 for *ABCB1*; and rs4148323, rs2003569, rs3755319, rs11692021, rs2070959, rs28900377, rs1983023 and rs8175347 for *UGT1A*.

Genotypes were determined for each of the SNPs using a TaqMan^®^ Genotyping OpenArray™ (Thermo Fisher Scientific, MA, USA). For rs8175347, due to the limited ability of TaqMan probes to reliably distinguish all genotypes, a high resolution melt analysis was used to determine genotype [[27](#B27)]. The TaqMan Genotyping OpenArray was performed as follows: genomic DNA was extracted from whole blood for each sample using the Qiagen QIAamp MiniKit (Germany, cat. no. 51306) according to manufacturer's instructions. Approximately 50 μg/ml of DNA and 2× TaqMan Genotyping OpenArray Master Mix was used for the assay. Plates were loaded using the Accufill liquid handling robot and run on the Applied Biosystems™ QuantStudio™ 12 K Flex Real-Time PCR instrument according to Applied Biosystems™ (CA, USA) user guide. Analysis was performed using the TaqMan Genotyper Software version 3.1. The genotypes for *UGT1A1**28 (rs8175347) were determined using genomic DNA as previously described [[27](#B27)] with minor modifications. The real-time PCR conditions were changed as follows: 95°C for 5 mins, followed by 45 cycles of 95°C for 10 s, 67°C for 10 s and 76°C for 10 s and finally 76°C for 1 min. Linkage disequilibrium (LD) was tested and haplotypes generated using LDlink 2.0. online software [[28](#B28)]. SNPs from *UGT1A* (rs11692021 and rs2070959) were found to be in perfect LD (D′ = 1, r^2^ = 1). In addition, *ABCB1* SNPs (rs2235033 and rs2235013) were found to be in perfect LD (D′ = 1, r^2^ = 1). The rs11692021 and rs2235033 SNPs were randomly selected in order to exclude them from the haplotypes generated.

### Study oversight

Ethical approval for the study was provided by the Biomedical Research Ethics Committee of the University of KwaZulu-Natal (BFC029/13) and the Medicines Control Council of South Africa (MCC ref:20130510).

### Statistical analysis

The moxifloxacin concentration-time data were analyzed using nonlinear mixed-effects (NLME) modelling, implemented with the software NONMEM (version 7.3) [[29](#B29)]. Perl-speaks-NONMEM, Xpose and Pirana were used for model diagnostics and to track model development [[30](#B30)]. A previously reported model developed on the same PK data was used as a starting point [[31](#B31)]. The model had a semimechanistic structure describing the effect of the liver both on first-pass extraction and systemic clearance. The original analysis [[31](#B31)] identified and characterized the effect of several factors on the pharmacokinetics of moxifloxacin, including body size (using fat-free mass of the individual patients, calculated based on weight, height and sex as suggested by Janmahasatian *et al*. [[32](#B32)]), co-administration of efavirenz and a difference between the exposure obtained at steady-state during rifampicin-based TB treatment and at single dose. After accounting for these effects, a moderate amount of variability still remained in the data, and this analysis aimed to further improve the model by exploring and including the effect of genetic polymorphisms. As a first exploratory step, random effects for the between-subject and -occasion variability in intrinsic clearance (CL_int_), prehepatic bioavailability and absorption parameters were extracted from the original model. These represent the portion of variability not yet explained by the factors previously included in the model (mentioned above). Linear mixed-effects models implemented in SAS, version 9.4 (SAS Institute, NC, USA) were used to explore the relationship between this variability and genotypes for the *UGT1A* and *ABCB1* SNPs. A compound symmetry correlation structure was assumed to describe the correlation between individual variability estimates at different visits. After this screening step, all the effects for which a trend was identified (p < 0.05) were further assessed within an updated version of the NLME model using stepwise covariate model building implemented in Perl-speaks-NONMEM [[30](#B30)]. Each SNP was included as a categorical variable, and grouped according to the corresponding genotype – that is, wild-type, heterozygous, or homozygous mutant. In certain instances, the heterozygous group was combined with either the wild-type or homozygous mutant whenever no statistical significant difference could be detected between two of the variants. A drop in the NONMEM objective function value (OFV) of 3.84 (corresponding 5% level of significance, 1 degree of freedom) was considered for inclusion of the effect of an SNP. A more stringent criterion of a change in OFV of 6.64 (corresponding to 1% level of significance) was applied for retaining the effect of an SNP in the final model. Further diagnostics were used, including inspection of goodness of fit plots and visual predictive checks, testing the robustness of the detected effect with a nonparametric bootstrap with replacement (n = 200), and inspection of the individual contributions to the change of the OFV.

## Results

### Baseline data

We included 172 South African TB patients: 119 (69.2%) male, 170 (98.8%) of Black African ethnicity and 127 (73.8%) HIV co-infected ([Table 1](#T1)). Moxifloxacin concentration-time data were available for 58 of 172 patients. Of these: median weight, fat-free mass and age were 56.9 kg (interquartile range [IQR]: 51.1–65.2), 46.8 kg (IQR: 42.5–50.3) and 37 years (IQR: 31–42), respectively. Forty-one (70.7%) patients were male, 42 (72.4%) HIV co-infected, with 40/42 (95%) on efavirenz-based ART ([Table 1](#T1)).

### Table 1. . Baseline data.

| Variable | n = 172 | n = 58 |
| --- | --- | --- |
| Age (years), median (IQR) | 35 (30–41) | 37 (31–42) |
|   |  |  |
| Male, n (%) | 119 (69.2) | 41 (70.7) |
|   |  |  |
| Race, n (%) – Black African ethnicity/Caucasian/colored‡ | 170 (98.8)/1 (0.6)/1 (0.6) | 56 (96.6)/1 (1.7)/1 (1.7) |
|   |  |  |
| Weight (kg), median (IQR) | 55.7 (50.3–62.1) | 56.9 (51.1–65.2) |
|   |  |  |
| Fat-free mass (kg) | – | 46.8 (42.5–50.3) |
|   |  |  |
| BMI (kg/m2), median (IQR) | 19.7 (18.3–22.5) | 19.6 (18.0–23.3) |
|   |  |  |
| HIV status, n (%) positive/negative | 127 (73.8)/45 (26.2) | 42 (72.4)/16 (27.6) |
|   |  |  |
| ART, n (%)† |  |  |
|   |  |  |
| Efavirenz + Emtricitabine + Tenofovir | 117 (95.1) | 40 (95.2) |
|   |  |  |
| Lopinavir/Ritonavir + Lamivudine + Tenofovir | 2 (1.6) | 2 (4.8) |
|   |  |  |
| Lopinavir/Ritonavir + Lamivudine + Zidovudine | 2 (1.6) | – |
|   |  |  |
| Lopinavir/Ritonavir + Emtricitabine + Tenofovir | 1 (0.8) | – |
|   |  |  |
| Efavirenz/Lamivudine/Zidovudine | 1 (0.8) | – |
|   |  |  |
| CD4+ count (cells/mm3), median (IQR)†§ | 241.0 (129.0–407.0) | 277.0 (139.0–384.0) |
|   |  |  |
| Viral load (log 10 copies/ml)†¶ | 3.7 (1.3–5.0) | 3.3 (1.3–4.2) |

Table 1 Caption: †Mixed race. ‡Only for HIV+ patients. §10/172, 4/58 missing data. ¶12/172, 5/58 missing data. ART: Antiretroviral therapy; IQR: Interquartile range.

### Genotype frequencies

The genotypic frequencies for *UGT1A* and *ABCB1* are presented in [Table 2](#T2) for all patients (n = 172) included in the study and the subset of 58 patients with moxifloxacin drug level data. Allelic frequencies for all SNPs are presented in [Table 3](#T3). Haplotypes were created for *UGT1A* based on SNPs rs2070959, rs1983023, rs28900377, rs3755319, rs2003569 and rs4148323 while haplotypes for *ABCB1* were created based on SNPs rs1128503, rs2032582 and rs1045642, in the order listed. Haplotype frequencies are presented in [Table 2](#T2).

### Table 2. . Genotype and haplotype frequency for UGT1A and ABCB1 SNPs.

| Frequency UGT1A |  |  |  | Frequency ABCB1 |  |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- |
|   |  |  |  |  |  |  |  |
| SNP | Genotype | n = 172 | n = 58 | SNP | Genotype | n = 172 | n = 58 |
| rs11692021 622T>C | CC | 0.04 | 0.05 | rs10276036 1000–44G>A | TT | 0.74 | 0.74 |
|   |  |  |  |  |  |  |  |
|   | CT | 0.18 | 0.21 |   | CT | 0.23 | 0.24 |
|   |  |  |  |  |  |  |  |
|   | TT | 0.78 | 0.73 |   | CC | 0.03 | 0.02 |
|   |  |  |  |  |  |  |  |
| rs4148323 211G>A | GG | 1.00 | 1.00 | rs1128503 1236C>T | AA | 0.01 | 0.00 |
|   |  |  |  |  |  |  |  |
|   | GA | 0.00 | 0.00 |   | AG | 0.14 | 0.16 |
|   |  |  |  |  |  |  |  |
|   | AA | 0.00 | 0.00 |   | GG | 0.85 | 0.84 |
|   |  |  |  |  |  |  |  |
| rs2003569 −997G>A | AA | 0.18 | 0.09 | rs2032582 2677G>T/A | AA | 0.00 | 0.00 |
|   |  |  |  |  |  |  |  |
|   | AG | 0.38 | 0.40 |   | CA | 0.02 | 0.03 |
|   |  |  |  |  |  |  |  |
|   | GG | 0.45 | 0.51 |   | CC | 0.98 | 0.97 |
|   |  |  |  |  |  |  |  |
| rs3755319 −1352A>C | AA | 0.06 | 0.05 | rs1045642 3435C>T | AA | 0.02 | 0.00 |
|   |  |  |  |  |  |  |  |
|   | AC | 0.32 | 0.41 |   | AG | 0.14 | 0.19 |
|   |  |  |  |  |  |  |  |
|   | CC | 0.62 | 0.54 |   | GG | 0.84 | 0.81 |
|   |  |  |  |  |  |  |  |
| rs2070959 855 + 20756A>G | AA | 0.79 | 0.74 | rs2235013 1725 + 38G>A | TT | 0.29 | 0.38 |
|   |  |  |  |  |  |  |  |
|   | AG | 0.18 | 0.21 |   | CT | 0.47 | 0.43 |
|   |  |  |  |  |  |  |  |
|   | GG | 0.04 | 0.05 |   | CC | 0.24 | 0.20 |
|   |  |  |  |  |  |  |  |
| rs28900377 856–23423C>T | TT | 0.01 | 0.00 | rs2235033 1554 + 24T>C | AA | 0.24 | 0.20 |
|   |  |  |  |  |  |  |  |
|   | CT | 0.06 | 0.09 |   | AG | 0.76 | 0.80 |
|   |  |  |  |  |  |  |  |
|   | CC | 0.93 | 0.91 |   | GG | 0.00 | 0.00 |
|   |  |  |  |  |  |  |  |
| rs1983023 −751T>C | TT | 0.06 | 0.07 |   |   |   |   |
|   |  |  |  |  |  |  |  |
|   | CT | 0.39 | 0.36 |   |   |   |   |
|   |  |  |  |  |  |  |  |
|   | CC | 0.56 | 0.57 |   |   |   |   |
|   |  |  |  |  |  |  |  |
| rs8175347 TA [5–8] | TA5/6 – UGT1A1*36/*1 | 0.11 | 0.10 |   |   |   |   |
|   |  |  |  |  |  |  |  |
|   | TA5/7 – UGT1A1*36/*28 | 0.01 | 0.00 |   |   |   |   |
|   |  |  |  |  |  |  |  |
|   | TA5/8 – UGT1A1*36/*37 | 0.01 | 0.00 |   |   |   |   |
|   |  |  |  |  |  |  |  |
|   | TA6/6 – UGT1A1*1/*1 | 0.48 | 0.50 |   |   |   |   |
|   |  |  |  |  |  |  |  |
|   | TA6/7 – UGT1A1*1/*28 | 0.16 | 0.12 |   |   |   |   |
|   |  |  |  |  |  |  |  |
|   | TA6/8 – UGT1A1*1/*37 | 0.02 | 0.02 |   |   |   |   |
|   |  |  |  |  |  |  |  |
|   | TA7/7 – UGT1A1*28/*28 | 0.08 | 0.12 |   |   |   |   |
|   |  |  |  |  |  |  |  |
|   | TA7/8 – UGT1A1*28/*37 | 0.13 | 0.14 |   |   |   |   |
|   |  |  |  |  |  |  |  |
| UGT1A haplotype† |   | n = 172 | n = 58 | ABCB1 | Haplotype‡ | n = 172 | n = 58 |
|   |  |  |  |  |  |  |  |
| Hap1 | ACCCGG | 0.71 | 0.76 |   | CGC | 0.83 | 0.84 |
|   |  |  |  |  |  |  |  |
| Hap2 | GCCCGG | 0.20 | 0.24 |   | CGT | 0.15 | 0.20 |
|   |  |  |  |  |  |  |  |
| Hap3 | ACCAGG | 0.35 | 0.40 |   | CTC | 0.02 | 0.02 |
|   |  |  |  |  |  |  |  |
| Hap4 | ACCCAG | 0.51 | 0.44 |   | TGC | 0.15 | 0.16 |
|   |  |  |  |  |  |  |  |
| Hap5 | ATCAGG | 0.24 | 0.25 |   | TGT | 0.10 | 0.14 |
|   |  |  |  |  |  |  |  |
| Hap6 | ATCCAG | 0.27 | 0.22 |   |   |   |   |
|   |  |  |  |  |  |  |  |
| Hap7 | ATTCAG | 0.05 | 0.05 |   |   |   |   |
|   |  |  |  |  |  |  |  |
| Hap8 | GCTCGG | 0.01 | 0.04 |   |   |   |   |

Table 2 Caption: † UGT1A haplotype consists of rs2070959, rs1983023, rs28900377, rs3755319, rs2003569 and rs4148323, respectively. ‡ ABCB1 haplotype consists of rs1128503, rs2032582 and rs1045642, respectively. The genotypes represented for ABCB1 represent the forward orientation while the gene is transcribed in the reverse direction.

### Table 3. . Allele frequencies for UGT1A and ABCB1 SNPs (n = 172).

| UGT1A |  |  |  |  |  |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- |
|   |  |  |  |  |  |  |  |
| 211G>A | -997G>A | -1352A>C | 855 + 20756A>G | 856–23423C>T | -751T>C | TA(n) | 622T>C |
|   |  |  |  |  |  |  |  |
| rs4148323 | rs2003569 | rs3755319 | rs2070959 | rs28900377 | rs1983023 | rs8175347 | rs11692021 |
|   |  |  |  |  |  |  |  |
| G (1.00) | A (0.36) | A (0.22) | A (0.87) | T (0.04) | T (0.25) | 5 (0.07) | C (0.13) |
|   |  |  |  |  |  |  |  |
| A (0.00) | G (0.64) | C (0.78) | G (0.13) | C (0.96) | C (0.75) | 6 (0.63) | T (0.87) |
|   |  |  |  |  |  |  |  |
|   |   |   |   |   |   | 7 (0.23) |   |
|   |  |  |  |  |  |  |  |
|   |   |   |   |   |   | 8 (0.08) |   |
|   |  |  |  |  |  |  |  |
| ABCB1† |  |  |  |  |  |  |  |
|   |  |  |  |  |  |  |  |
| 1000–44G>A | 1236C>T | 2677G>T/A | 3435C>T | 1725 + 38G>A | 1554 + 24T>C |   |   |
|   |  |  |  |  |  |  |  |
| rs10276036 | rs1128503 | rs2032582 | rs1045642 | rs2235013 | rs2235033 |   |   |
|   |  |  |  |  |  |  |  |
| T (0.85) | A (0.08) | A (0.01) | A (0.09) | T (0.53) | A (0.47) |   |   |
|   |  |  |  |  |  |  |  |
| C (0.15) | G (0.92) | C (0.99) | G (0.91) | C (0.47) | G (0.53) |   |   |

Table 3 Caption: †The genotypes represented for ABCB1 represent the forward orientation while the gene is transcribed in the reverse direction. TA(n): Number of TA repeats.

### Effects of UGT1A and ABCB1 SNPs on moxifloxacin pharmacokinetics

The variability in mean moxifloxacin plasma concentrations (AUC) in each patient obtained from the base NLME model and stratified by genotypes for eight *UGT1A* and six *ABCB1* SNPs are shown in [Figure 1](#F0001)A and B. These values are affected by other effects contained in the base model, and are only reported to show the distribution of variability in AUC among genotypes tested and for comparison of exposures with other studies. The exploration of pharmacogenetics was executed after adjusting for all the fixed effects already included in the model.

### Figure 1. . Distribution of moxifloxacin AUC and genotypes/linkage disequilibrium plots.

![Figure 1. ](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7026/5753622/42e538aa1f9d/pgs-19-17-g1.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5753622_pgs-19-17-g1.jpg)

Distributions of (A) UGT1A and (B) ABCB1 genotypes using variability in mean moxifloxacin concentration (AUC) μg·h/ml from the base nonlinear mixed-effects model, and linkage disequilibrium between (C) UGT1A (D) and ABCB1 SNPs.

An LD test was performed on *UGT1A* and *ABCB1* SNPs ([Figure 1](#F0001)C & D). For *UGT1A* we tested seven SNPs. Of these, rs11692021 and rs2070959 were found to be in perfect LD (D′ = 1, r^2^ = 1). For this reason, rs11692021 was randomly selected and excluded from the generation of haplotypes. The LD was not calculated for rs4148323 because only one genotype (GG) was observed in the population ([Figure 1](#F0001)C). LD analysis was performed on five SNPs for *ABCB1*; rs2032582 was not included in the LD analysis as this SNP possessed multiple alleles that did not fit within the LD test. We found rs2235033 and rs2235013 to be in perfect LD (D′ = 1, r^2^ = 1; [Figure 1](#F0001)D); rs2235033 was randomly chosen to be excluded from the generation of haplotypes.

The SNPs found to be significant in the screening step employing linear models were then tested in NLME models that allow for re-estimation of other covariates that may affect moxifloxacin PK parameters. The effects of two *UGT* SNPs were confirmed to influence the pharmacokinetics of moxifloxacin in the final NLME PK model ([Table 4](#T4)). Having the TA 5/6 repeat in rs8175347 was associated with a 20.6% lower clearance and approximately 26% higher AUC, after adjusting for other covariates (such as efavirenz and/or rifampicin co-administration) compared with those with TA 6/6, 6/7, 7/7 and 7/8 (p = 0.001). Patients with AC and AA genotypes for rs3755319 were found to have 11.6% increased clearance when compared with the CC genotype in the model (p = 0.032). However, effect of the SNP was not strong enough to be retained in the final model using the cut-off value of p = 0.01. Only one individual within the cohort examined had a CA genotype for the *ABCB1* SNP rs2032582. When the effect of the rs2032582 SNP was included in the population PK model, a 40% reduced prehepatic bioavailability and similar decrease in AUC were observed for the patient with the CA genotype (p = 0.01). However, although this effect significantly improved the model fit in terms of OFV, it was not retained in the final model due to the fact that only one patient was affected. When testing the outcome of presence versus absence of haplotypes for both *ABCB1* and *UGT* within individual patients in the NLME models, no significant difference was observed. [Figure 2](#F0002) depicts the visual predictive checks for the final model stratified by genotypes for rs8175347. The visual predictive checks show that the model describes the data adequately after inclusion of the effect of the SNP.

### Table 4. . Population parameter estimates of moxifloxacin pharmacokinetics.

| Parameter description | Typical value (CI)§ | Random variability (CI)§ |
| --- | --- | --- |
| Intrinsic clearance during rifampicin-based TB treatment (l/h)†,‡ | 50.0 (45.9; 56.8) | BSV: 12.1% (2.6; 16.7)BOV: 12.2% (4.6; 18.9) |
|   |  |  |
| Volume of central compartment (l)‡ | 127 (109; 137) | BSV: 8.6% (0.5; 13.5) |
|   |  |  |
| Intercompartmental clearance (l/h)‡ | 2.12 (1.55; 4.51) | – |
|   |  |  |
| Volume of peripheral compartment (l)‡ | 31.0 (22.3; 50.8) | – |
|   |  |  |
| Prehepatic bioavailability (.)# | 1 fixed | BOV: 35.6% (28.2; 42.1) |
|   |  |  |
| Absorption lag time (h)prior | 0.55 (0.45; 0.73) | – |
|   |  |  |
| Ka – absorption rate (1/h)prior | 2.80 (1.18; 3.45) | BOV: 93.2% (0.9; 121.7) |
|   |  |  |
| Hepatic plasma flow (l/h) | 50 fixed |   |
|   |  |  |
| Moxifloxacin fraction unbound (%) | 50% fixed |   |
|   |  |  |
| Change in intrinsic clearance while on single dose of moxifloxacin (%) | -28.9% (-36.5; -21.9) |   |
|   |  |  |
| Change in prehepatic bioavailability while on single dose of moxifloxacin (%) | -22.4% (-32.4; -11.6) |   |
|   |  |  |
| Change in intrinsic clearance while on efavirenz-based ART (%) | +42.9% (32.6; 56.1) |   |
|   |  |  |
| Change in intrinsic clearance due to rs8175347 (five out of six genotypes) | -20.6% (-29.3; -13.6) |   |
|   |  |  |
| Scaling factor for variability in bioavailability while on single dose of moxifloxacin (-fold)†† | 0.62-fold (0.41; 0.85) |   |
|   |  |  |
| Scaling factor for variability in bioavailability for unobserved doses (-fold)†† | 2.48-fold (1.73; 3.75) |   |
|   |  |  |
| Proportional error (%) | 17.4% (12.2; 21.2) |   |
|   |  |  |
| Additive error (mg/l) | 0.011 (0.004; 0.017) |   |

Table 4 Caption: †Intrinsic clearance of moxifloxacin when given at steady state within rifampicin-based TB treatment and no efavirenz. ‡All clearance and volume parameters have been allometrically scaled with fat-free mass, and the typical values reported here refer to the typical patient, with a fat-free mass of 47 kg. §CI – empirical 95% CI obtained with a nonparametric bootstrap (n = 300). #Prehepatic bioavailability is the fraction of the drug, which is absorbed, crosses the gut wall without any change, thus entering the portal vein and reaching the liver. ††These scaling factors modulate the size of the between-occasion variability in prehepatic bioavailability for the sections of data indicated (single dose and unobserved doses). priorThese parameters were estimated using a prior, as detailed in text of the original analysis [29]. ART: Antiretroviral therapy; BSV: Between-subject variability; BOV: Between-occasion variability; TB: Tuberculosis.

### Figure 2. . Visual predictive checks stratified by genotype of rs8175347.

![Figure 2. ](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7026/5753622/65bf717dc45e/pgs-19-17-g2.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5753622_pgs-19-17-g2.jpg)

The dashed and solid lines are the 5th, median and 95th percentiles of the observed concentrations, while the shaded regions represent the corresponding 95% CIs for the same percentiles. The subplot in each stratum shows the same visual predictive checks with a logarithmic transformation applied to the y-axis.

## Discussion

We describe the prevalence of genetic variability in selected *UGT1A* and *ABCB1* SNPs in a cohort of South African TB patients, the majority being of Black African ethnicity. Our data add to the current evidence of genetic variants having pharmacogenetic relevance among Africans. The impact of genetic variation on drug metabolizing and transporter enzymes UGT and ABCB1, has not been described previously for moxifloxacin in the context of TB treatment. We found differences between genotypes of *UGT1A* SNPs rs8175347 and rs3755319, which significantly associated with changes in moxifloxacin PK parameters. Genotypes of the *ABCB1* SNP rs2032582 were significantly associated with a decrease in prehepatic bioavailability in the NLME model.

Moxifloxacin exhibits extensive interindividual variability in PK parameters [[6–8](#B6),[33](#B33),[34](#B34)]. We tested the effect of genetic variants in *UGT1A* and *ABCB1* as covariates in NLME models used to determine moxifloxacin concentration-time data and found that these are associated with some of the between subject variability observed in moxifloxacin PK parameters not accounted for by other relevant covariates in our study. We previously reported low overall moxifloxacin concentrations in this patient cohort [[31](#B31)], which are further decreased by up to 30% [[8](#B8),[31](#B31)] as a result of drug interactions with efavirenz-based ART and rifampicin – a known inducer of UGT and P-glycoprotein [[35](#B35)]. After adjusting for these and other covariates, some of the unexplained between subject variability in moxifloxacin PK may be attributed to genetic variability.

More than 113 different *UGT1A1* variants have been described, which can confer reduced or increased activity as well as inactive or normal enzymatic phenotypes [[18](#B18),[20](#B20)]. The *UGT1A1* rs8175347 SNP in the gene promoter region covering the TATA box consists of variation in a short (TA; n) sequence from five to eight TA repeats. This SNP has been extensively studied for association between the *UGT1A1*28* (TA7) allele and toxicity related to the cancer drug irinotecan [[18](#B18),[36](#B36)]. The TA6 and TA7 variants are frequently seen in most populations studied, while the TA5 (*UGT1A1*36*) and TA8 (*UGT1A1*37*) are found almost exclusively in African populations; TA6 (*UGT1A1*1*) being the wild-type [[18](#B18),[37](#B37)]. Frequencies of the TA repeats observed in our cohort were similar to those previously reported in South African patients; however, the frequency of the TA6 allele was higher (0.63 vs 0.5) and TA7 lower (0.23 vs 0.37) in our cohort compared with a previous published black South African cohort [[37](#B37)]. Homozygotes for TA7 (*UGT1A1*28/*28*) and less commonly the TA8 genotype have been linked to decreased transcriptional activity and lower activity of the gene compared with the wild-type, resulting in decreased glucuronidation of the toxic SN-38 metabolite of irinotecan. The TA7 and TA8 containing genotypes are, therefore, expected to result in decreased metabolism, lower clearance, higher bioavailability and drug exposure as seen with thyroxine (T4) [[38](#B38)]. These and other studies suggest that an increased number of TA repeats may be associated with higher drug exposure. However, we demonstrate higher moxifloxacin concentrations with heterozygous TA5/6 genotypes compared with TA6/6, TA6/7 and TA7/8 genotypes. Similar findings were reported with telmisartan [[39](#B39)] raloxifene [[40](#B40)] and moxifloxacin in healthy individuals [[19](#B19)], where higher concentrations of metabolites and resultant lower drug exposures were observed with TA7 (*UGT1A1*28*). The reasons for this contrasting effect are not entirely clear, but may be linked to extensive sharing of exons in the *UGT1A* family and genetic linkage between many UGT variants (which may differ between ethnic groups) contributing to discordance between findings. Furthermore, uncertainty exists on the extent of functional activity of the UGT variants studied.

We report effects of the AC and AA variants in the rs3755319 *UGT1A* SNP resulting in higher clearance (CL) and decreased moxifloxacin exposures; however, these effects were not robust enough to be included in the final NLME model. Previous studies have reported nonsignificant effects of this SNP on exposure to other drugs such as ABT-751 used to treat patients with neoplasms [[41](#B41)]. No significant effects of *UGT* haplotypes were observed in our study.

We found very limited effects of genotypes in the *ABCB1* SNPs studied, with the exception of rs2032582 where we found 40% reduced prehepatic bioavailability associated with lower moxifloxacin concentrations in the presence of the CA genotype. Similar effects of this variant have been reported previously [[23](#B23)]. However, we had only one individual in this genotype group, likely as a result of our limited sample size. A previous study testing the effect of the rs2032582 SNP on efavirenz concentrations in South African patients found limited effects on drug exposure [[42](#B42)]. We found no significant effects of *ABCB1* haplotypes in our study. The *ABCB1* variants 1236C>T (rs1128503), 2677G>T/A (rs2032582) and 3435C>T (rs1045642) occur at high-allele frequencies and create a common haplotype; therefore, they have been widely studied in relation to a number of drug classes [[23](#B23)]. A recent review investigating the effects of these SNPs on drug exposure and PK parameters in over 300 studies concluded that although these had significant effects on drug disposition and response, findings were in part conflicting and the clinical implications limited [[23](#B23)].

We acknowledge the limitations of our study. This includes a relatively small sample of patients with drug concentration data, and the lack of significant associations observed between SNPs studied and PK parameters for moxifloxacin may be due to inadequate numbers of patients in the groups for each of the genotypes studied. The concentrations of moxifloxacin metabolites M1 and M2 were not measured and we are unable to draw any conclusions on the effects of the genetic variation in UGT on moxifloxacin glucuronidation to its inactive metabolites. We included a limited number of SNPs in our study based on previous literature that showed association between these SNPs and drug response, toxicity or disease susceptibility. Other relevant *UGT1A* and *ABCB1* SNPs that were not included in this study may significantly impact moxifloxacin PK parameters. Genotype frequencies in our population may differ from other populations and our findings may therefore be limited to similar African populations. The PK data were collected with sparse sampling. While NLME modeling has been shown to be suitable to analyze sparse data, our model had sometimes limited capability to separate effect on CL_int_ versus prehepatic bioavailability, parameters which both affect total exposure, as seen with TA5/6 repeat in rs8175347. In such cases, the most statistically significant effect was included in the model. We previously reported significant effect of efavirenz on moxifloxacin pharmacokinetics (42% increased clearance and resultant 30% lower AUC) due to induction of the UGT enzyme in these patients. While the NLME model adjusted for the effect of efavirenz on moxifloxacin PK and should be able to separate this from the genotype effect it is possible that if efavirenz induction mimics the effects of genetic variants on enzyme activity or if mutations in the UGT enzyme binding sites alter the extent of efavirenz induction, that this may interfere with the model's ability to detect the effect of the genotypes or differentiate between these effects.

## Conclusion

We have observed increased prevalence of genetic variability in *UGT1A* and *ABCB1* in our cohort of South African patients. Genotypes of *UGT1A* SNPs (rs8175347) and to a lesser extent *UGT1A* (rs3755319) and *ABCB1* (rs2032582) were associated with significant differences in moxifloxacin drug concentrations or PK parameters. The clinical relevance of these changes in moxifloxacin drug exposure on TB treatment outcomes needs further investigation, although given that moxifloxacin exhibits concentration-dependent bactericidal activity, changes in drug concentrations are likely to impact efficacy.

PK–PD targets including AUC: MIC ratios >53 or 100 have been suggested to be associated with improved outcomes and reduced risk of emergent drug resistance. Moxifloxacin drug concentrations in our study were low overall. It is therefore plausible that SNPs associated with lower or higher drug concentrations may affect individual patient's likelihood of achieving these targets.

## Acknowledgements

The authors gratefully acknowledge the study participants for their participation in the IMPRESS pharmacokinetic study; study nurses, counsellors, clinicians, the study co-ordinator D Govender and the IMPRESS study team for their work in the pharmacokinetic study. Special acknowledgement to M Govender and the pharmacy team for support on the study. We thank the following laboratories and laboratory staff; CAPRISA laboratory (N Samsunder, and lab staff), KwaZulu-Natal Research Institute for Tuberculosis and HIV (KRITH) Pharmacology Core and Medical Microbiology Laboratories and Global Clinical & Viral Laboratories, for laboratory support. We thank D Munatsi and P Radebe for data management on the study, and J Bredenkamp (Forest Molecular Genetics Programme, University of Pretoria) for the TaqMan^®^ OpenArray genotyping assay analysis. The Division of Clinical Pharmacology at the University of Cape Town, acknowledges Novartis Pharma for their support of the development of pharmacometric skills in Africa. The sponsor of the study had no role in study design, data collection, data analysis, data interpretation or writing of the report. All authors had access to data, commented on drafts and approved the final report. The corresponding author had final responsibility for the decision to submit for publication.

## Footnotes

## References

1. Bayer HealthCare AG. Bayer HealthCare: Germany; 2011. Moxifloxacin investigator brochure version 14.0.  [Google Scholar](https://scholar.google.com/scholar_lookup?Bayer%20HealthCare%20AG.%20Bayer%20HealthCare:%20Germany;%202011.%20Moxifloxacin%20investigator%20brochure%20version%2014.0.)

2. WHO. www.who.int/tb/publications/global_report/en/ Global Tuberculosis Report 2016.  [www.who.int/tb/publications/global_report/en/](http://www.who.int/tb/publications/global_report/en/)

3. Dawson R, Diacon AH, Everitt D, et al. Efficiency and safety of the combination of moxifloxacin, pretomanid (PA-824), and pyrazinamide during the first 8 weeks of antituberculosis treatment: a Phase 2b, open-label, partly randomised trial in patients with drug-susceptible or drug-resistant pulmonary tuberculosis. Lancet. 2015;385(9979):1738–1747. doi: 10.1016/S0140-6736(14)62002-X.  [DOI](https://doi.org/10.1016/S0140-6736(14)62002-X) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25795076/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Efficiency%20and%20safety%20of%20the%20combination%20of%20moxifloxacin,%20pretomanid%20(PA-824),%20and%20pyrazinamide%20during%20the%20first%208%20weeks%20of%20antituberculosis%20treatment:%20a%20Phase%202b,%20open-label,%20partly%20randomised%20trial%20in%20patients%20with%20drug-susceptible%20or%20drug-resistant%20pulmonary%20tuberculosis&author=R%20Dawson&author=AH%20Diacon&author=D%20Everitt&volume=385&issue=9979&publication_year=2015&pages=1738-1747&pmid=25795076&doi=10.1016/S0140-6736(14)62002-X&)

4. Moodley R, Godec TR. Short-course treatment for multidrug-resistant tuberculosis: the STREAM trials. Eur. Respir. Rev. 2016;25(139):29–35. doi: 10.1183/16000617.0080-2015.  [DOI](https://doi.org/10.1183/16000617.0080-2015) | [PMC free article](/articles/PMC9487666/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26929418/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur.%20Respir.%20Rev.&title=Short-course%20treatment%20for%20multidrug-resistant%20tuberculosis:%20the%20STREAM%20trials&author=R%20Moodley&author=TR%20Godec&volume=25&issue=139&publication_year=2016&pages=29-35&pmid=26929418&doi=10.1183/16000617.0080-2015&)

5. Gumbo T, Louie A, Deziel MR, Parsons LM, Salfinger M, Drusano GL. Selection of a moxifloxacin dose that suppresses drug resistance in Mycobacterium tuberculosis, by use of an in vitro pharmacodynamic infection model and mathematical modeling. J. Infect. Dis. 2004;190(9):1642–1651. doi: 10.1086/424849.  [DOI](https://doi.org/10.1086/424849) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15478070/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Infect.%20Dis.&title=Selection%20of%20a%20moxifloxacin%20dose%20that%20suppresses%20drug%20resistance%20in%20Mycobacterium%20tuberculosis,%20by%20use%20of%20an%20in%20vitro%20pharmacodynamic%20infection%20model%20and%20mathematical%20modeling&author=T%20Gumbo&author=A%20Louie&author=MR%20Deziel&author=LM%20Parsons&author=M%20Salfinger&volume=190&issue=9&publication_year=2004&pages=1642-1651&pmid=15478070&doi=10.1086/424849&)

6. MacGowan AP. Moxifloxacin (Bay 12–8039): a new methoxy quinolone antibacterial. Expert Opin. Investigational Drugs. 1999;8(2):181–199. doi: 10.1517/13543784.8.2.181.  [DOI](https://doi.org/10.1517/13543784.8.2.181) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15992072/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Expert%20Opin.%20Investigational%20Drugs&title=Moxifloxacin%20(Bay%2012%E2%80%938039):%20a%20new%20methoxy%20quinolone%20antibacterial&author=AP%20MacGowan&volume=8&issue=2&publication_year=1999&pages=181-199&pmid=15992072&doi=10.1517/13543784.8.2.181&)

7. Nijland HM, Ruslami R, Suroto AJ, et al. Rifampicin reduces plasma concentrations of moxifloxacin in patients with tuberculosis. Clin. Infect. Dis. 2007;45(8):1001–1007. doi: 10.1086/521894.  [DOI](https://doi.org/10.1086/521894) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17879915/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Infect.%20Dis.&title=Rifampicin%20reduces%20plasma%20concentrations%20of%20moxifloxacin%20in%20patients%20with%20tuberculosis&author=HM%20Nijland&author=R%20Ruslami&author=AJ%20Suroto&volume=45&issue=8&publication_year=2007&pages=1001-1007&pmid=17879915&doi=10.1086/521894&)

8. Weiner M, Burman W, Luo CC, et al. Effects of rifampin and multidrug resistance gene polymorphism on concentrations of moxifloxacin. Antimicrob. Agents Chemother. 2007;51(8):2861–2866. doi: 10.1128/AAC.01621-06.  [DOI](https://doi.org/10.1128/AAC.01621-06) | [PMC free article](/articles/PMC1932492/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17517835/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Antimicrob.%20Agents%20Chemother.&title=Effects%20of%20rifampin%20and%20multidrug%20resistance%20gene%20polymorphism%20on%20concentrations%20of%20moxifloxacin&author=M%20Weiner&author=W%20Burman&author=CC%20Luo&volume=51&issue=8&publication_year=2007&pages=2861-2866&pmid=17517835&doi=10.1128/AAC.01621-06&)

9. Alffenaar JW, de Vries PM, Luijckx GJ, van Soolingen D, van der Werf TS, van Altena R. Plasma and cerebrospinal fluid pharmacokinetics of moxifloxacin in a patient with tuberculous meningitis. Antimicrob. Agents Chemother. 2008;52(6):2293–2295. doi: 10.1128/AAC.01637-07.  [DOI](https://doi.org/10.1128/AAC.01637-07) | [PMC free article](/articles/PMC2415813/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18362186/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Antimicrob.%20Agents%20Chemother.&title=Plasma%20and%20cerebrospinal%20fluid%20pharmacokinetics%20of%20moxifloxacin%20in%20a%20patient%20with%20tuberculous%20meningitis&author=JW%20Alffenaar&author=PM%20de%20Vries&author=GJ%20Luijckx&author=D%20van%20Soolingen&author=TS%20van%20der%20Werf&volume=52&issue=6&publication_year=2008&pages=2293-2295&pmid=18362186&doi=10.1128/AAC.01637-07&)

10. Pranger AD, Kosterink JG, van Altena R, et al. Limited-sampling strategies for therapeutic drug monitoring of moxifloxacin in patients with tuberculosis. Ther. Drug Monit. 2011;33(3):350–354. doi: 10.1097/FTD.0b013e31821b793c.  [DOI](https://doi.org/10.1097/FTD.0b013e31821b793c) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21544017/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ther.%20Drug%20Monit.&title=Limited-sampling%20strategies%20for%20therapeutic%20drug%20monitoring%20of%20moxifloxacin%20in%20patients%20with%20tuberculosis&author=AD%20Pranger&author=JG%20Kosterink&author=R%20van%20Altena&volume=33&issue=3&publication_year=2011&pages=350-354&pmid=21544017&doi=10.1097/FTD.0b013e31821b793c&)

11. Naidoo A, Naidoo K, McIlleron H, Essack S, Padayatchi N. A review of moxifloxacin for the treatment of drug-susceptible tuberculosis. J. Clin. Pharmacol. 2017;57(11):1369–1389. doi: 10.1002/jcph.968.  [DOI](https://doi.org/10.1002/jcph.968) | [PMC free article](/articles/PMC5663285/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28741299/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Pharmacol.&title=A%20review%20of%20moxifloxacin%20for%20the%20treatment%20of%20drug-susceptible%20tuberculosis&author=A%20Naidoo&author=K%20Naidoo&author=H%20McIlleron&author=S%20Essack&author=N%20Padayatchi&volume=57&issue=11&publication_year=2017&pages=1369-1389&pmid=28741299&doi=10.1002/jcph.968&)

12. Ramachandran G, Swaminathan S. Role of pharmacogenomics in the treatment of tuberculosis: a review. Pharmgenomics Pers. Med. 2012;5:89–98. doi: 10.2147/PGPM.S15454.  [DOI](https://doi.org/10.2147/PGPM.S15454) | [PMC free article](/articles/PMC3513231/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23226065/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmgenomics%20Pers.%20Med.&title=Role%20of%20pharmacogenomics%20in%20the%20treatment%20of%20tuberculosis:%20a%20review&author=G%20Ramachandran&author=S%20Swaminathan&volume=5&publication_year=2012&pages=89-98&pmid=23226065&doi=10.2147/PGPM.S15454&)

13. Mboowa G. Genetics of sub-Saharan African human population: implications for HIV/AIDS, tuberculosis, and malaria. Int. J. Evol. Biol. 2014:108291. doi: 10.1155/2014/108291. 2014.  [DOI](https://doi.org/10.1155/2014/108291) | [PMC free article](/articles/PMC4151494/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25202468/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Evol.%20Biol.&title=Genetics%20of%20sub-Saharan%20African%20human%20population:%20implications%20for%20HIV/AIDS,%20tuberculosis,%20and%20malaria&author=G%20Mboowa&publication_year=2014&pages=108291&pmid=25202468&doi=10.1155/2014/108291&)

14. Chigutsa E, Visser ME, Swart EC, et al. The SLCO1B1 rs4149032 polymorphism is highly prevalent in South Africans and is associated with reduced rifampin concentrations: dosing implications. Antimicrob. Agents Chemother. 2011;55(9):4122–4127. doi: 10.1128/AAC.01833-10.  [DOI](https://doi.org/10.1128/AAC.01833-10) | [PMC free article](/articles/PMC3165308/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21709081/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Antimicrob.%20Agents%20Chemother.&title=The%20SLCO1B1%20rs4149032%20polymorphism%20is%20highly%20prevalent%20in%20South%20Africans%20and%20is%20associated%20with%20reduced%20rifampin%20concentrations:%20dosing%20implications&author=E%20Chigutsa&author=ME%20Visser&author=EC%20Swart&volume=55&issue=9&publication_year=2011&pages=4122-4127&pmid=21709081&doi=10.1128/AAC.01833-10&)

15. Gengiah TN, Botha JH, Soowamber D, Naidoo K, Abdool Karim SS. Low rifampicin concentrations in tuberculosis patients with HIV infection. J. Infect. Dev. Ctries. 2014;8(8):987–993. doi: 10.3855/jidc.4696.  [DOI](https://doi.org/10.3855/jidc.4696) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25116663/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Infect.%20Dev.%20Ctries&title=Low%20rifampicin%20concentrations%20in%20tuberculosis%20patients%20with%20HIV%20infection&author=TN%20Gengiah&author=JH%20Botha&author=D%20Soowamber&author=K%20Naidoo&author=SS%20Abdool%20Karim&volume=8&issue=8&publication_year=2014&pages=987-993&pmid=25116663&doi=10.3855/jidc.4696&)

16. McIlleron H, Willemse M, Werely CJ, et al. Isoniazid plasma concentrations in a cohort of South African children with tuberculosis: implications for international pediatric dosing guidelines. Clin. Infect. Dis. 2009;48(11):1547–1553. doi: 10.1086/598192.  [DOI](https://doi.org/10.1086/598192) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19392636/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Infect.%20Dis.&title=Isoniazid%20plasma%20concentrations%20in%20a%20cohort%20of%20South%20African%20children%20with%20tuberculosis:%20implications%20for%20international%20pediatric%20dosing%20guidelines&author=H%20McIlleron&author=M%20Willemse&author=CJ%20Werely&volume=48&issue=11&publication_year=2009&pages=1547-1553&pmid=19392636&doi=10.1086/598192&)

17. Moise PA, Birmingham MC, Schentag JJ. Pharmacokinetics and metabolism of moxifloxacin. Drugs Today. 2000;36(4):229–244. doi: 10.1358/dot.2000.36.4.570201.  [DOI](https://doi.org/10.1358/dot.2000.36.4.570201) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12879119/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drugs%20Today&title=Pharmacokinetics%20and%20metabolism%20of%20moxifloxacin&author=PA%20Moise&author=MC%20Birmingham&author=JJ%20Schentag&volume=36&issue=4&publication_year=2000&pages=229-244&pmid=12879119&doi=10.1358/dot.2000.36.4.570201&)

18. Barbarino JM, Haidar CE, Klein TE, Altman RB. PharmGKB summary: very important pharmacogene information for UGT1A1 . Pharmacogenet. Genomics. 2014;24(3):177–183. doi: 10.1097/FPC.0000000000000024.  [DOI](https://doi.org/10.1097/FPC.0000000000000024) | [PMC free article](/articles/PMC4091838/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24492252/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet.%20Genomics&title=PharmGKB%20summary:%20very%20important%20pharmacogene%20information%20for%20UGT1A1&author=JM%20Barbarino&author=CE%20Haidar&author=TE%20Klein&author=RB%20Altman&volume=24&issue=3&publication_year=2014&pages=177-183&pmid=24492252&doi=10.1097/FPC.0000000000000024&)

19. Hasunuma T, Tohkin M, Kaniwa N, et al. Absence of ethnic differences in the pharmacokinetics of moxifloxacin, simvastatin, and meloxicam among three east Asian populations and Caucasians. Br. J. Clin. Pharmacol. 2016;81(6):1078–1090. doi: 10.1111/bcp.12884.  [DOI](https://doi.org/10.1111/bcp.12884) | [PMC free article](/articles/PMC4876172/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26774055/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br.%20J.%20Clin.%20Pharmacol.&title=Absence%20of%20ethnic%20differences%20in%20the%20pharmacokinetics%20of%20moxifloxacin,%20simvastatin,%20and%20meloxicam%20among%20three%20east%20Asian%20populations%20and%20Caucasians&author=T%20Hasunuma&author=M%20Tohkin&author=N%20Kaniwa&volume=81&issue=6&publication_year=2016&pages=1078-1090&pmid=26774055&doi=10.1111/bcp.12884&)

20. Stingl JC, Bartels H, Viviani R, Lehmann ML, Brockmoller J. Relevance of UDP-glucuronosyltransferase polymorphisms for drug dosing: a quantitative systematic review. Pharmacol. Ther. 2014;141(1):92–116. doi: 10.1016/j.pharmthera.2013.09.002.  [DOI](https://doi.org/10.1016/j.pharmthera.2013.09.002) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24076267/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacol.%20Ther.&title=Relevance%20of%20UDP-glucuronosyltransferase%20polymorphisms%20for%20drug%20dosing:%20a%20quantitative%20systematic%20review&author=JC%20Stingl&author=H%20Bartels&author=R%20Viviani&author=ML%20Lehmann&author=J%20Brockmoller&volume=141&issue=1&publication_year=2014&pages=92-116&pmid=24076267&doi=10.1016/j.pharmthera.2013.09.002&)

21. Ito T, Yano I, Tanaka K, Inui KI. Transport of quinolone antibacterial drugs by human P-glycoprotein expressed in a kidney epithelial cell line, LLC-PK1. J. Pharmacol. Exp. Ther. 1997;282(2):955–960.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/9262363/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Pharmacol.%20Exp.%20Ther.&title=Transport%20of%20quinolone%20antibacterial%20drugs%20by%20human%20P-glycoprotein%20expressed%20in%20a%20kidney%20epithelial%20cell%20line,%20LLC-PK1&author=T%20Ito&author=I%20Yano&author=K%20Tanaka&author=KI%20Inui&volume=282&issue=2&publication_year=1997&pages=955-960&pmid=9262363&)

22. Fromm MF. The influence of MDR1 polymorphisms on P-glycoprotein expression and function in humans. Adv. Drug Deliv. Rev. 2002;54(10):1295–1310. doi: 10.1016/s0169-409x(02)00064-9.  [DOI](https://doi.org/10.1016/s0169-409x(02)00064-9) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12406646/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Adv.%20Drug%20Deliv.%20Rev.&title=The%20influence%20of%20MDR1%20polymorphisms%20on%20P-glycoprotein%20expression%20and%20function%20in%20humans&author=MF%20Fromm&volume=54&issue=10&publication_year=2002&pages=1295-1310&pmid=12406646&doi=10.1016/s0169-409x(02)00064-9&)

23. Wolking S, Schaeffeler E, Lerche H, Schwab M, Nies AT. Impact of genetic polymorphisms of ABCB1 (MDR1, P-glycoprotein) on drug disposition and potential clinical implications: update of the literature. Clin. Pharmacokinet. 2015;54(7):709–735. doi: 10.1007/s40262-015-0267-1.  [DOI](https://doi.org/10.1007/s40262-015-0267-1) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25860377/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacokinet.&title=Impact%20of%20genetic%20polymorphisms%20of%20ABCB1%20(MDR1,%20P-glycoprotein)%20on%20drug%20disposition%20and%20potential%20clinical%20implications:%20update%20of%20the%20literature&author=S%20Wolking&author=E%20Schaeffeler&author=H%20Lerche&author=M%20Schwab&author=AT%20Nies&volume=54&issue=7&publication_year=2015&pages=709-735&pmid=25860377&doi=10.1007/s40262-015-0267-1&)

24. Hodges LM, Markova SM, Chinn LW, et al. Very important pharmacogene summary: ABCB1 (MDR1, P-glycoprotein) Pharmacogenet. Genomics. 2011;21(3):152–161. doi: 10.1097/FPC.0b013e3283385a1c.  [DOI](https://doi.org/10.1097/FPC.0b013e3283385a1c) | [PMC free article](/articles/PMC3098758/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20216335/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet.%20Genomics&title=Very%20important%20pharmacogene%20summary:%20ABCB1%20(MDR1,%20P-glycoprotein)&author=LM%20Hodges&author=SM%20Markova&author=LW%20Chinn&volume=21&issue=3&publication_year=2011&pages=152-161&pmid=20216335&doi=10.1097/FPC.0b013e3283385a1c&)

25. FDA. www.fda.gov/downloads/Drugs/GuidanceComplianceRegulatoryInformation/Guidances/ucm070107.pdf Guidance for industry bioanalytical method validation, Department of Health and Human Services, Food and Drug Administration.  [www.fda.gov/downloads/Drugs/GuidanceComplianceRegulatoryInformation/Guidances/ucm070107.pdf](http://www.fda.gov/downloads/Drugs/GuidanceComplianceRegulatoryInformation/Guidances/ucm070107.pdf)

26. Whirl-Carrillo M, McDonagh EM, Hebert JM, et al. Pharmacogenomics knowledge for personalized medicine. Clin. Pharmacol. Ther. 2012;92(4):414–417. doi: 10.1038/clpt.2012.96.  [DOI](https://doi.org/10.1038/clpt.2012.96) | [PMC free article](/articles/PMC3660037/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22992668/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Pharmacogenomics%20knowledge%20for%20personalized%20medicine&author=M%20Whirl-Carrillo&author=EM%20McDonagh&author=JM%20Hebert&volume=92&issue=4&publication_year=2012&pages=414-417&pmid=22992668&doi=10.1038/clpt.2012.96&)

27. Farrar JS, Palais RA, Wittwer CT. Snapback primer genotyping of the Gilbert syndrome UGT1A1 (TA)(n) promoter polymorphism by high-resolution melting. Clin. Chem. 2011;57(9):1303–1310. doi: 10.1373/clinchem.2011.166306.  [DOI](https://doi.org/10.1373/clinchem.2011.166306) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21771946/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Chem.&title=Snapback%20primer%20genotyping%20of%20the%20Gilbert%20syndrome%20UGT1A1%20(TA)(n)%20promoter%20polymorphism%20by%20high-resolution%20melting&author=JS%20Farrar&author=RA%20Palais&author=CT%20Wittwer&volume=57&issue=9&publication_year=2011&pages=1303-1310&pmid=21771946&doi=10.1373/clinchem.2011.166306&)

28. Machiela MJ, Chanock SJ. LDlink: a web-based application for exploring population-specific haplotype structure and linking correlated alleles of possible functional variants. Bioinformatics. 2015;31(21):3555–3557. doi: 10.1093/bioinformatics/btv402.  [DOI](https://doi.org/10.1093/bioinformatics/btv402) | [PMC free article](/articles/PMC4626747/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26139635/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=LDlink:%20a%20web-based%20application%20for%20exploring%20population-specific%20haplotype%20structure%20and%20linking%20correlated%20alleles%20of%20possible%20functional%20variants&author=MJ%20Machiela&author=SJ%20Chanock&volume=31&issue=21&publication_year=2015&pages=3555-3557&pmid=26139635&doi=10.1093/bioinformatics/btv402&)

29. Beal S, Sheiner L, Boeckmann A, Bauer R. ICON Development Solutions; MD, USA: 2009. NONMEM users guides (1989–2009)  [Google Scholar](https://scholar.google.com/scholar_lookup?Beal%20S,%20Sheiner%20L,%20Boeckmann%20A,%20Bauer%20R.%20ICON%20Development%20Solutions;%20MD,%20USA:%202009.%20NONMEM%20users%20guides%20(1989%E2%80%932009))

30. Keizer RJ, Karlsson MO, Hooker A. Modeling and simulation workbench for NONMEM: tutorial on Pirana, PsN, and Xpose. CPT: Pharmacometrics Systems Pharmacology. 2013;2:e50. doi: 10.1038/psp.2013.24.  [DOI](https://doi.org/10.1038/psp.2013.24) | [PMC free article](/articles/PMC3697037/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23836189/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=CPT:%20Pharmacometrics%20Systems%20Pharmacology&title=Modeling%20and%20simulation%20workbench%20for%20NONMEM:%20tutorial%20on%20Pirana,%20PsN,%20and%20Xpose&author=RJ%20Keizer&author=MO%20Karlsson&author=A%20Hooker&volume=2&publication_year=2013&pages=e50&pmid=23836189&doi=10.1038/psp.2013.24&)

31. Naidoo A, Chirehwa M, McIlleron H, et al. Effect of rifampicin and efavirenz on moxifloxacin concentrations when co-administered in patients with drug-susceptible TB. J. Antimicrob. Chemother. 2017;72(5):1441–1449. doi: 10.1093/jac/dkx004.  [DOI](https://doi.org/10.1093/jac/dkx004) | [PMC free article](/articles/PMC5890691/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28175315/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Antimicrob.%20Chemother.&title=Effect%20of%20rifampicin%20and%20efavirenz%20on%20moxifloxacin%20concentrations%20when%20co-administered%20in%20patients%20with%20drug-susceptible%20TB&author=A%20Naidoo&author=M%20Chirehwa&author=H%20McIlleron&volume=72&issue=5&publication_year=2017&pages=1441-1449&pmid=28175315&doi=10.1093/jac/dkx004&)

32. Janmahasatian S, Duffull SB, Ash S, Ward LC, Byrne NM, Green B. Quantification of lean bodyweight. Clin. Pharmacokinet. 2005;44(10):1051–1065. doi: 10.2165/00003088-200544100-00004.  [DOI](https://doi.org/10.2165/00003088-200544100-00004) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16176118/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacokinet.&title=Quantification%20of%20lean%20bodyweight&author=S%20Janmahasatian&author=SB%20Duffull&author=S%20Ash&author=LC%20Ward&author=NM%20Byrne&volume=44&issue=10&publication_year=2005&pages=1051-1065&pmid=16176118&doi=10.2165/00003088-200544100-00004&)

33. Peloquin CA, Hadad DJ, Molino LP, et al. Population pharmacokinetics of levofloxacin, gatifloxacin, and moxifloxacin in adults with pulmonary tuberculosis. Antimicrob. Agents Chemother. 2008;52(3):852–857. doi: 10.1128/AAC.01036-07.  [DOI](https://doi.org/10.1128/AAC.01036-07) | [PMC free article](/articles/PMC2258503/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18070980/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Antimicrob.%20Agents%20Chemother.&title=Population%20pharmacokinetics%20of%20levofloxacin,%20gatifloxacin,%20and%20moxifloxacin%20in%20adults%20with%20pulmonary%20tuberculosis&author=CA%20Peloquin&author=DJ%20Hadad&author=LP%20Molino&volume=52&issue=3&publication_year=2008&pages=852-857&pmid=18070980&doi=10.1128/AAC.01036-07&)

34. Pranger AD, van Altena R, Aarnoutse RE, et al. Evaluation of moxifloxacin for the treatment of tuberculosis: 3 years of experience. Eur. Respir. J. 2011;38(4):888–894. doi: 10.1183/09031936.00176610.  [DOI](https://doi.org/10.1183/09031936.00176610) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21310881/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur.%20Respir.%20J.&title=Evaluation%20of%20moxifloxacin%20for%20the%20treatment%20of%20tuberculosis:%203%20years%20of%20experience&author=AD%20Pranger&author=R%20van%20Altena&author=RE%20Aarnoutse&volume=38&issue=4&publication_year=2011&pages=888-894&pmid=21310881&doi=10.1183/09031936.00176610&)

35. Kim KA, Park PW, Liu KH, et al. Effect of rifampin, an inducer of CYP3A and P-glycoprotein, on the pharmacokinetics of risperidone. J. Clin. Pharmacol. 2008;48(1):66–72. doi: 10.1177/0091270007309888.  [DOI](https://doi.org/10.1177/0091270007309888) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18094221/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Pharmacol.&title=Effect%20of%20rifampin,%20an%20inducer%20of%20CYP3A%20and%20P-glycoprotein,%20on%20the%20pharmacokinetics%20of%20risperidone&author=KA%20Kim&author=PW%20Park&author=KH%20Liu&volume=48&issue=1&publication_year=2008&pages=66-72&pmid=18094221&doi=10.1177/0091270007309888&)

36. Minami H, Sai K, Saeki M, et al. Irinotecan pharmacokinetics/pharmacodynamics and UGT1A genetic polymorphisms in Japanese: roles of UGT1A1*6 and *28 . Pharmacogenet. Genomics. 2007;17(7):497–504. doi: 10.1097/FPC.0b013e328014341f.  [DOI](https://doi.org/10.1097/FPC.0b013e328014341f) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17558305/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet.%20Genomics&title=Irinotecan%20pharmacokinetics/pharmacodynamics%20and%20UGT1A%20genetic%20polymorphisms%20in%20Japanese:%20roles%20of%20UGT1A1*6%20and%20*28&author=H%20Minami&author=K%20Sai&author=M%20Saeki&volume=17&issue=7&publication_year=2007&pages=497-504&pmid=17558305&doi=10.1097/FPC.0b013e328014341f&)

37. Horsfall LJ, Zeitlyn D, Tarekegn A, et al. Prevalence of clinically relevant UGT1A alleles and haplotypes in African populations. Annals Human Genet. 2011;75(2):236–246. doi: 10.1111/j.1469-1809.2010.00638.x.  [DOI](https://doi.org/10.1111/j.1469-1809.2010.00638.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21309756/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Annals%20Human%20Genet.&title=Prevalence%20of%20clinically%20relevant%20UGT1A%20alleles%20and%20haplotypes%20in%20African%20populations&author=LJ%20Horsfall&author=D%20Zeitlyn&author=A%20Tarekegn&volume=75&issue=2&publication_year=2011&pages=236-246&pmid=21309756&doi=10.1111/j.1469-1809.2010.00638.x&)

38. Santoro AB, Vargens DD, Barros Filho Mde C, et al. Effect of UGT1A1, UGT1A3, DIO1 and DIO2 polymorphisms on L-thyroxine doses required for TSH suppression in patients with differentiated thyroid cancer. Br. J. Clin. Pharmacol. 2014;78(5):1067–1075. doi: 10.1111/bcp.12437.  [DOI](https://doi.org/10.1111/bcp.12437) | [PMC free article](/articles/PMC4243881/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24910925/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br.%20J.%20Clin.%20Pharmacol.&title=Effect%20of%20UGT1A1,%20UGT1A3,%20DIO1%20and%20DIO2%20polymorphisms%20on%20L-thyroxine%20doses%20required%20for%20TSH%20suppression%20in%20patients%20with%20differentiated%20thyroid%20cancer&author=AB%20Santoro&author=DD%20Vargens&author=C%20Barros%20Filho%20Mde&volume=78&issue=5&publication_year=2014&pages=1067-1075&pmid=24910925&doi=10.1111/bcp.12437&)

39. Shin HJ, Kim JE, Lim SJ, et al. Effect of UGT1A1 and UGT1A3 polymorphisms on pharmacokinetics of telmisartan in Korean. Clin. Ther. 2015;37(8):e87.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Ther.&title=Effect%20of%20UGT1A1%20and%20UGT1A3%20polymorphisms%20on%20pharmacokinetics%20of%20telmisartan%20in%20Korean&author=HJ%20Shin&author=JE%20Kim&author=SJ%20Lim&volume=37&issue=8&publication_year=2015&pages=e87&)

40. Trontelj J, Marc J, Zavratnik A, Bogataj M, Mrhar A. Effects of UGT1A1*28 polymorphism on raloxifene pharmacokinetics and pharmacodynamics. Br. J. Clin. Pharmacol. 2009;67(4):437–444. doi: 10.1111/j.1365-2125.2009.03363.x.  [DOI](https://doi.org/10.1111/j.1365-2125.2009.03363.x) | [PMC free article](/articles/PMC2679107/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19371317/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br.%20J.%20Clin.%20Pharmacol.&title=Effects%20of%20UGT1A1*28%20polymorphism%20on%20raloxifene%20pharmacokinetics%20and%20pharmacodynamics&author=J%20Trontelj&author=J%20Marc&author=A%20Zavratnik&author=M%20Bogataj&author=A%20Mrhar&volume=67&issue=4&publication_year=2009&pages=437-444&pmid=19371317&doi=10.1111/j.1365-2125.2009.03363.x&)

41. Innocenti F, Ramirez J, Obel J, et al. Preclinical discovery of candidate genes to guide pharmacogenetics during Phase I development: the example of the novel anticancer agent ABT-751. Pharmacogenet. Genomics. 2013;23(7):374–381. doi: 10.1097/FPC.0b013e3283623e81.  [DOI](https://doi.org/10.1097/FPC.0b013e3283623e81) | [PMC free article](/articles/PMC3858967/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23670235/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet.%20Genomics&title=Preclinical%20discovery%20of%20candidate%20genes%20to%20guide%20pharmacogenetics%20during%20Phase%20I%20development:%20the%20example%20of%20the%20novel%20anticancer%20agent%20ABT-751&author=F%20Innocenti&author=J%20Ramirez&author=J%20Obel&volume=23&issue=7&publication_year=2013&pages=374-381&pmid=23670235&doi=10.1097/FPC.0b013e3283623e81&)

42. Swart M, Ren Y, Smith P, Dandara C. ABCB1 4036A>G and 1236C>T polymorphisms affect plasma efavirenz levels in South African HIV/AIDS patients. Front. Genet. 2012;3:236. doi: 10.3389/fgene.2012.00236.  [DOI](https://doi.org/10.3389/fgene.2012.00236) | [PMC free article](/articles/PMC3488761/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23133441/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front.%20Genet.&title=ABCB1%204036A>G%20and%201236C>T%20polymorphisms%20affect%20plasma%20efavirenz%20levels%20in%20South%20African%20HIV/AIDS%20patients&author=M%20Swart&author=Y%20Ren&author=P%20Smith&author=C%20Dandara&volume=3&publication_year=2012&pages=236&pmid=23133441&doi=10.3389/fgene.2012.00236&)
