# Influence of pharmacogenetic polymorphisms and demographic variables on metformin pharmacokinetics in an admixed Brazilian cohort

## Metadata
**Authors:** Ana Beatriz Santoro, Mariana Rodrigues Botton, Claudio José Struchiner, Guilherme Suarez‐Kurtz
**Journal:** British Journal of Clinical Pharmacology
**Date:** 2018 Feb 26
**DOI:** [10.1111/bcp.13522](https://doi.org/10.1111/bcp.13522)
**PMID:** 29352482
**PMCID:** PMC5903234
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5903234/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC5903234/pdf/BCP-84-987.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC5903234/pdf/BCP-84-987.pdf)

## Abstract

**Aims:** 
To identify pharmacogenetic and demographic variables that influence the systemic exposure to metformin in an admixed Brazilian cohort.

**Methods:** 
The extreme discordant phenotype was used to select 106 data sets from nine metformin bioequivalence trials, comprising 256 healthy adults. Eleven single‐nucleotide polymorphisms in SLC22A1, SLC22A2, SLC47A1 SLC47A2 and in transcription factor SP1 were genotyped and a validated panel of ancestry informative markers was used to estimate the individual proportions of biogeographical ancestry. Two‐step (univariate followed by multivariate) regression modelling was developed to identify covariates associated with systemic exposure to metformin, accessed by the area under the plasma concentration–time curve, between 0 and 48 h (AUC0–48h), after single oral doses of metformin (500 or 1000 mg).

**Results:** 
The individual proportions of African, Amerindian and European ancestry varied widely, as anticipated from the structure of the Brazilian population The dose‐adjusted, log‐transformed AUC0–48h's (ng h ml−1 mg−1) differed largely in the two groups at the opposite ends of the distribution histogram, namely 0.82, 0.79–0.85 and 1.08, 1.06–1.11 (mean, 95% confidence interval; P = 6.10−26, t test). Multivariate modelling revealed that metformin AUC0–48h increased with age, food and carriage of rs12208357 in SLC22A1 but was inversely associated with body surface area and individual proportions of African ancestry.

**Conclusions:** 
A pharmacogenetic marker in OCT1 (SLC22A1 rs12208357), combined with demographic covariates (age, body surface area and individual proportion of African ancestry) and a food effect explained 29.7% of the variability in metformin AUC0–48h.

Keywords: biogeographical ancestry, extreme discordant phenotype, metformin pharmacokinetics, SLC22, SLC47, SP1

### Aims

To identify pharmacogenetic and demographic variables that influence the systemic exposure to metformin in an admixed Brazilian cohort.

### Methods

The extreme discordant phenotype was used to select 106 data sets from nine metformin bioequivalence trials, comprising 256 healthy adults. Eleven single‐nucleotide polymorphisms in SLC22A1, SLC22A2, SLC47A1 SLC47A2 and in transcription factor SP1 were genotyped and a validated panel of ancestry informative markers was used to estimate the individual proportions of biogeographical ancestry. Two‐step (univariate followed by multivariate) regression modelling was developed to identify covariates associated with systemic exposure to metformin, accessed by the area under the plasma concentration–time curve, between 0 and 48 h (AUC_0–48h_), after single oral doses of metformin (500 or 1000 mg).

### Results

The individual proportions of African, Amerindian and European ancestry varied widely, as anticipated from the structure of the Brazilian population The dose‐adjusted, log‐transformed AUC_0–48h_'s (ng h ml^−1^ mg^−1^) differed largely in the two groups at the opposite ends of the distribution histogram, namely 0.82, 0.79–0.85 and 1.08, 1.06–1.11 (mean, 95% confidence interval; P = 6.10^−26^, t test). Multivariate modelling revealed that metformin AUC_0–48h_ increased with age, food and carriage of rs12208357 in SLC22A1 but was inversely associated with body surface area and individual proportions of African ancestry.

### Conclusions

A pharmacogenetic marker in OCT1 (SLC22A1 rs12208357), combined with demographic covariates (age, body surface area and individual proportion of African ancestry) and a food effect explained 29.7% of the variability in metformin AUC_0–48h_.

## What is Already Known about this Subject

## What this Study Adds

## Introduction

The pharmacokinetics of [http://www.guidetopharmacology.org/GRAC/LigandDisplayForward?ligandId=4779](http://www.guidetopharmacology.org/GRAC/LigandDisplayForward?ligandId=4779), a first‐line drug therapy for type 2 diabetes (DM2), is modulated by several transmembrane transporters, notably organic cation transporters (OCTs) and multidrug and toxin extrusion proteins (MATEs), encoded by [http://www.guidetopharmacology.org/GRAC/FamilyDisplayForward?familyId=196](http://www.guidetopharmacology.org/GRAC/FamilyDisplayForward?familyId=196) and [http://www.guidetopharmacology.org/GRAC/FamilyDisplayForward?familyId=236](http://www.guidetopharmacology.org/GRAC/FamilyDisplayForward?familyId=236) genes, respectively [1](#bcp13522-bib-0004), [2](#bcp13522-bib-0005). OCT1 (*SCL22A1*) and OCT3 (*SCL22A3*) contribute to the extrusion of metformin from enterocytes into blood. Conversely, OCT1 and to a lesser extent OCT3 promote the uptake of metformin into hepatocytes, essential for the drug's inhibitory effect on gluconeogenesis. Both OCT1 and OCT3 transport metformin into skeletal muscle, which may be a major site of the drug's action. OCT2 (*SCL22A2*) expressed in the basolateral membrane of renal tubule cells, mediate cellular influx of metformin, whereas MATE1 (*SLC47A1*) and MATE2 (*SLC47A2*), present in the apical membrane extrude the drug out of the tubule lining cells into urine. OCT1 has also been detected in the apical membranes of proximal and distal tubules, and may be involved in metformin resorption [3](#bcp13522-bib-0006).

The impact of OCT and MATE transporters on metformin pharmacokinetics is subject to modulation by coadministered drugs, that inhibit or induce the transporters [4](#bcp13522-bib-0007), [5](#bcp13522-bib-0008), by genetic polymorphisms affecting expression and/or function of the transport proteins and their transcription factors, and by gene–gene interactions [6](#bcp13522-bib-0009), [7](#bcp13522-bib-0010), [8](#bcp13522-bib-0011), [9](#bcp13522-bib-0012), [10](#bcp13522-bib-0013). Metformin pharmacogenetics has been extensively investigated in European and Asian populations, with controversial results regarding the drug's pharmacokinetics and clinical effects. The present study explores the impact of pharmacogenetic variants on metformin pharmacokinetics in a Brazilian cohort of healthy adults. The present day Brazilian population of over 200 million, is highly heterogeneous and admixed, a fact that has far reaching pharmacogenetics implications [11](#bcp13522-bib-0014), [12](#bcp13522-bib-0015), [13](#bcp13522-bib-0016). The extent of admixture is well documented in a wealth of population genetic studies in Brazilians, and it is evident that most individuals have significant degrees of European and/or African ancestry, while a sizeable number display also Amerindian ancestry [14](#bcp13522-bib-0017), [15](#bcp13522-bib-0018). Based on previous reports of association with metformin pharmacokinetic parameters, 11 single‐nucleotide polymorphisms (SNPs) in *SCL22A1*, *SCL22A2*, *SCL47*A1, *SCL47*A2 and in the transcription factor SP1 (specificity protein 1) locus were selected for this study. The participants were also typed with a panel of ancestry informative markers to estimate the influence of individual proportions of Amerindian (Native), African and European ancestry on metformin pharmacokinetics.

## Methods

### Study cohort

We used demographic and pharmacokinetic data available for 256 healthy, adult Brazilians previously enrolled in nine bioequivalence trials, performed according to the guidelines of the Brazilian Health Surveillance Agency [16](#bcp13522-bib-0019) and approved by the respective institutional review boards. Each volunteer (24–34 per trial) provided written, informed consent and was asked to self‐identify according to the *race/Colour* classification scheme adopted by the Brazilian census, which relies on self‐perception of skin colour [17](#bcp13522-bib-0003). The term *Colour* and the Colour categories are capitalized to call attention to their meaning in the context of the Brazilian Census, Colour (in Portuguese, *cor*) denoting the Brazilian equivalent of the English term *race*. Details of the study protocol were reported previously [18](#bcp13522-bib-0020). Briefly, the bioequivalence trials adopted an open‐label, randomized, two‐sequence, two‐period crossover design, in which administration of the reference (Glifage; Merck & Co., Rio de Janeiro, Brazil) and a test metformin formulation were separated by a 7‐day washout interval. The oral metformin doses were 500 mg (four trials) and 1000 mg (five trials). Metformin was administered after a 12–h fast in four trials and 30 min after a standard low‐fat breakfast in five trials. Consecutive blood samples (*n* = 14–18) were collected immediately before and in the 48 h following metformin administration. The concentration of metformin in plasma was determined by liquid chromatography–tandem mass spectrometry, and linear trapezoidal interpolation was used to determine the area under the plasma concentration *vs*. time curve between 0 and 48 h (AUC_0–48h_). All statistical analyses were performed on log‐transformed AUC_0–48h,_ adjusted for the individual doses, expressed as ng h ml^−1^ mg (dose)^−1^.

### Extreme discordant phenotype

To increase the statistical power of the study to detect associations of genetic and nongenetic variables with metformin AUC_0–48h_ we applied the extreme discordant phenotype (EDP) methodology [19](#bcp13522-bib-0021). The EDP approach contrasts the most sensitive and the most resistant phenotype groups, which in the case of a quantitative trait such as the individual metformin AUC_0–48h_ correspond to the lower and upper ends of the distribution histogram. Data sets for the reference formulation from the 256 individuals enrolled in the bioequivalence trials were used to construct the distribution histogram of the AUC_0–48h_. The cut‐off point for selection of individuals to be recruited for the study was set at the 20^th^ percentile of the overall study cohort (*n* = 256), which corresponds to 51 individuals. As a preventive measure in the case of missing genotype data, two additional subjects from each end of the distribution histogram were recruited. Only these 106 individuals (2 × 53) were genotyped for pharmacogenetic polymorphisms and for ancestry informative markers, as described below.

### Genetic component of metformin pharmacokinetics

The repeated drug administration (RDA) method for estimating the genetic component (r_GC_) of pharmacokinetic variation [20](#bcp13522-bib-0022) was applied to the dose‐adjusted, log‐transformed AUC_0–48_ values for the reference and the test formulations administered to the 106 subjects selected for this study. The between‐ (SD_b_ ^2^) and within‐person (SD_w_ ^2^) variances were calculated and the expression r_GC_ = (SD_b_ ^2^ – SD_w_ ^2^)/ SD_b_ ^2^ was used to estimate the genetic component of interindividual variability in AUC_0–48._ Accordingly, r_GC_ values approaching 1.0, point to overwhelming genetic control, whereas r_GC_ values close to zero suggest that environmental factors dominate. Our rationale for adopting the RDA approach is that application of bioequivalent drug formulations mimics repeated administration of a given formulation, as originally proposed for the r_GC_ analyses [20](#bcp13522-bib-0022).

### Genotyping

Genomic DNA was isolated from peripheral blood by standard procedures. Taqman assays and a Fast 7500 Real‐Time System (Applied Biosystems, Foster City, CA, USA) were used for allele discrimination at 11 polymorphic loci in *SLC22A1, SLC22A2, SLC47A1, SCL47A2* and the SP1 locus (Table [1](#bcp13522-tbl-0001)). SNPs were selected on the basis of previous reports of association with metformin pharmacokinetic parameters. Allele and genotype frequency were derived by gene counting. Deviations from Hardy–Weinberg equilibrium were assessed by the goodness‐of‐fit χ^2^ test.

### Table 1.

| Gene (protein) | dbSNP ID | Nucleotide | Amino acid |   |
| --- | --- | --- | --- | --- |
| change | change | Functiona |  |  |
| SLC22A1 (OCT1) | rs1867351 | c. –510 T > C |   | htSNP, decreased expression 33 |
| rs622342 | intron A > C |   | unknown function |  |
| rs12208357 | c. 181C > T | R61C | reduced activity 31 |  |
| rs72552763 | c. 1260GAT > del | M420del | reduced activity 33 |  |
| rs34059508 | c. 1393G > A | G465R | reduced activity 31 |  |
| SLC22A2 (OCT2) | rs316019 | c. 808G > T | A270S | reduced activity 43 |
| SLC47A1 (MATE1) | rs2252281 | c. –66 T > C |   | decreased promoter activity 44 |
|   | rs2289669 | intron G > A |   | unknown function |
| SLC47A2 (MATE2) | rs12943590 | c. –130 G > A |   | increased promoter activity 45 |
| AMHR2 (SP1 locus) | rs784888 | intron G > C |   | unknown function |
| rs784892 | intron G > A |   | unknown function |  |

Table 1 Caption: Single nucleotide polymorphisms genotyped in the study

### Linkage disequilibrium and haplotype analyses

Pairwise linkage disequilibrium (LD) analyses of polymorphisms in *SLC22A1, SLC47A1* and *SP1* genes were conducted online using the CUBEX software [21](#bcp13522-bib-0023). The level of LD between loci was assessed using the coefficients *D´* and *r* ^2^. *SLC22A1* haplotypes, comprising rs1867351, rs622342, rs12208357, rs72552763 and rs34059508 were statistically inferred using the software Phase v2.1.1 [22](#bcp13522-bib-0024). Haplotypes are labelled by numbers preceded by Br (e.g. HapBr1, the wild‐type haplotype), a reference to this Brazilian cohort. Haplotypes containing one or more variant alleles are designated variant haplotypes. Diplotypes were classified according to the number of wildtype haplotypes: homozygous wildtype (*n* = 2), heterozygous (*n* = 1) and homozygous variant (*n* = 0).

### Estimation of individual admixture

The 106 individuals selected by the EDP approach were genotyped with a panel of biallelic short insertion–deletion (indel) polymorphisms, validated as ancestry informative markers for the Brazilian population [23](#bcp13522-bib-0001). The individual proportions of biogeographical ancestry (Amerindian, European and African) were estimated using the program *Structure* v.2.3.4 [24](#bcp13522-bib-0002).

### Multivariate regression model

To select candidate variables for the modelling of metformin AUC_0–48h_, we initially examined the independent effects of demographic variables, food effect (i.e. metformin administration to fast or fed individuals), the 11 genotyped SNPs and the inferred *SLC22A1* diplotypes. Genetic variables were analysed according to codominant and dominant models. In the latter, carriers of one or two copies of the variant alleles at the *SLC22, SCL47* and SP1 SNPs and carriers of one or two copies of variant *SLC22A1* halotypes were contrasted with the respective homozygotes for the variant wild‐type SNP alleles and the *SLC22A1* wild‐type halplotype HapBr1. Discrete covariates entered the analysis transformed as dummy variables and their effects are described by electing a baseline category and comparing the transformed response predicted at this category with the remaining category or categories of interest. Variables that associated (*P* < 0.15) with the log‐transformed, dose‐adjusted metformin AUC_0–48h_ in the univariate analyses were considered further in multivariate regression models. Variables that associated (*P* < 0.05) with AUC_0–48h_ in successive multivariate analyses were included in the final model. The relative contribution of each variable to the model was estimated by the partial *r* ^2^ statistics, which measure the degree of association between two random variables, with the effect of a set of controlling random variables removed. We assessed the goodness of fit of our models based on the *R* ^2^ and *R* ^2^‐adjusted statistics. The former is a measure of how well the regression line approximates the observed data points. It can also be interpreted as the proportion of the total variance in the observed data that is explained by the model. Since *R* ^2^ always increases with the number of variables in the model, this measure can be accounted for by adjusting for the number of covariates in the model, yielding the statistics known as the adjusted *R* ^2^.

### Statistical analyses

Demographic variables are presented as median ± interquartile range (1–3). Allele, genotype and haplotype frequencies are shown with the respective 95% confidence intervals [95% confidence interval (CI)]. The log‐transformed, dose adjusted AUC_0–48h_ is presented as mean ± standard deviation (SD). The software G*Power 3.1.9.2 [25](#bcp13522-bib-0025) was used to calculate statistical power to detect differences in AUC_0–48h_ among genotypes and *SLC22A1* haplotypes.

### Nomenclature of targets and ligands

Key protein targets and ligands in this article are hyperlinked to corresponding entries in [http://www.guidetopharmacology.org](http://www.guidetopharmacology.org), the common portal for data from the IUPHAR/BPS Guide to PHARMACOLOGY [26](#bcp13522-bib-0026), and are permanently archived in the Concise Guide to PHARMACOLOGY 2017/18 [27](#bcp13522-bib-0027).

## Results

Table [2](#bcp13522-tbl-0002) presents demographic information for the 106 healthy adult individuals (52 women) included in the study. Four race/Colour categories of the Brazilian census were represented in the study cohort and distributed in excellent agreement with their prevalence in the overall Brazilian population [15](#bcp13522-bib-0018). The individual proportions of Amerindian, European and African biogeographical ancestry (Table [2](#bcp13522-tbl-0002), Figure [S1](#bcp13522-supitem-0001)) varied widely among the participants, as anticipated from the structure of the heterogenous, admixed Brazilian population [13](#bcp13522-bib-0016), [15](#bcp13522-bib-0018).

### Table 2.

| Demographic information | Median | IQR |
| --- | --- | --- |
| Age (years) | 31.1 | 26.1–36.3 |
| Weight (kg) | 68.9 | 64.2–76.8 |
| Body surface area (m 2 ) | 1.79 | 1.69–1.90 |
| Proportion of ancestry |   |   |
| African | 0.048 | 0.019–0.201 |
| Amerindian | 0.028 | 0.016–0.071 |
| European | 0.904 | 0.710–0.954 |
| Sex | n | % |
| Female | 52 | 49.1 |
| Male | 54 | 50.9 |
| Race/Colour a |   |   |
| White | 48 | 4.2 |
| Brown | 43 | 40.6 |
| Black | 13 | 12.3 |
| Amerindian | 2 | 1.9 |

Table 2 Caption: Demographic characteristics of the study cohort

### Metformin pharmacokinetics

Figure [1](#bcp13522-fig-0001) (inset) shows the frequency distribution histogram of metformin AUC_0–48h_ in the overall cohort (256 volunteers). Using the EDP approach, 53 individuals at the opposite extremes of the histogram were selected for this study. The dose‐adjusted, log‐transformed metformin AUC_0–48h_ in the two groups, namely 0.82, 0.79–0.85 ng h ml^−1^ mg^−1^ and 1.08, 1.06–1.11 ng h ml^−1^ mg^−1^ (mean, 95%CI) differed significantly (*P* = 6.10^−26^, *t* test). The metformin plasma concentration‐time curves for the two groups are shown in Figure [1](#bcp13522-fig-0001).

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8bde/5903234/20f5af55cfe1/BCP-84-987-g001.jpg)

Metformin plasma concentration (mean, standard deviation) vs. time curves for single oral doses (500 or 1000 mg) of the reference formulation, administered to 106 healthy adult individuals enrolled in bioequivalence trials. The plasma concentrations are adjusted to the individual dose (mg). For better visualization, the standard deviation bars are shown in only one direction. Solid circles and open squares refer, respectively, to 53 individuals at each end of the frequency distribution histogram (inset) of the log‐transformed, dose‐adjusted AUC0–48h for the 256 individuals participating in the bioequivalence trials. The grey bars highlight the 106 individuals classified in the 20th percentile of the histogram

### Genetic component of variability of systemic exposure to metformin

The dose‐adjusted, log transformed AUC_0–48_ data for the reference and the test formulations were used for the calculation of the r_GC_ AUC_0–48h_ value_,_ according to the RDA method [20](#bcp13522-bib-0022). For the 106 subjects selected based on the EDP approach, an r_GC_ value of 0.93 (0.90–0.95, 95% CI) was obtained, which is suggestive of a strong genetic component in the variation in systemic exposure to metformin following single oral doses of 500–1000 mg.

### Pharmacogenetic polymorphisms

Table [3](#bcp13522-tbl-0003) shows the frequency distribution of alleles and genotypes at polymorphic loci in *SLC22A1, SLC22A2, SLC47A1, SCL47A2* and in the SP1 region. Deviation from Hardy–Weinberg expectations was not observed at any locus. The two SNPs in the SP1 locus, rs584888 and rs 784 892, were in strong LD, with pairwise *D´* = 1.0 and *r* ^2^ = 0.93.

### Table 3.

| Gene (protein) | dbSNP ID | Genotype frequency (95% CI) |  |  | Allele frequency (95% CI) |  |
| --- | --- | --- | --- | --- | --- | --- |
| A/Aa | A/B | B/B | A | B |  |  |
| SLC22A1 (OCT1) | rs1867351 | 0.59 (0.49–0.67) | 0.32 (0.21–0.41) | 0.08 (0.04–0.15) | 0.75 (0.68–0.80) | 0.24 (0.19–0.31) |
| rs622342 | 0.40 (0.31–0.49) | 0.44 (0.34–0.53) | 0.16 (0.10–0.24) | 0.62 (0.54–0.68) | 0.38 (0.31–0.44) |  |
| rs12208357 | 0.90 (0.82–0.94) | 0.09 (0.05–0.16) | 0.01 (<0.01–0.05) | 0.94 (0.9–0.97) | 0.06 (0.03–0.09) |  |
| rs72552763 | 0.67 (0.57–0.75) | 0.27 (0.20–0.37) | 0.06 (0.02–0.12) | 0.81 (0.75–0.85) | 0.19 (0.15–0.25) |  |
| rs34059508 | 0.98 (0.93–0.99) | 0.02 (<0.1–0.06) | 0 | 0.99 (0.97–1.00) | 0.01 (<0.1–0.03) |  |
| SLC22A2 (OCT2) | rs316019 | 0.77 (0.67–0.83) | 0.23 (0.16–0.32) | 0.01 (<0.01–0.05) | 0.88 (0.83–0.91) | 0.12 (0.08–0.17) |
| SLC47A1 (MATE1) | rs2252281 | 0.35 (0.27–0.45) | 0.44 (0.35–0.54) | 0.19 (0.12–0.28) | 0.58 (0.51–0.64) | 0.41 (0.35–0.48) |
| rs2289669 | 0.44 (0.35–0.54) | 0.43 (0.33–0.52) | 0.13 (0.08–0.21) | 0.66 (0.59–0.72) | 0.34 (0.28–0.41) |  |
| SLC47A2 (MATE2) | rs12943590 | 0.37 (0.28–0.46) | 0.47 (0.38–0.57) | 0.15 (0.09–0.23) | 0.60 (0.54–0.67) | 0.39 (0.32–0.45) |
| SP1 (SP1) | rs784888 | 0.74 (0.65–0.81) | 0.20 (0.14–0.29) | 0.04 (0.02–0.10) | 0.84 (0.79–0.88) | 0.15 (0.11–0.20) |
| rs784892 | 0.75 (0.66–0.82) | 0.20 (0.14–0.29) | 0.03 (0.01–0.09) | 0.85 (0.80–0.89) | 0.14 (0.10–0.19) |  |

Table 3 Caption: Allele and genotype frequency in the study cohort

Haplotype analysis of the five *SCL22A1* SNPs identified nine haplotypes with frequencies greater than 1%, which together accounted for 99.5% of the overall genetic diversity (Table [4](#bcp13522-tbl-0004)). The wildtype haplotype (HapBr1) represented 35.4%, whereas variant haplotypes comprising one (HapBr2 – HapBr6) or two MAF alleles (HapBr7 – HapBr9) accounted for 43.9% and 18.9%, respectively. Diplotypes combining two, one or zero copies of the wildtype HapBr1 haplotype occurred in 14 (13.2%), 47 (44.3%) and 45 (42.5%) individuals, respectively.

### Table 4.

| Haplotype ID | Haplotypea | n (frequency) |
| --- | --- | --- |
| HapBr1 | A/C/GAT/A/G | 75 (0.354) |
| HapBr2 | A/C/GAT/C/G | 37 (0.175) |
| HapBr3 | G/C/GAT/A/G | 34 (0.160) |
| HapBr4 | A/T/GAT/A/G | 11 (0.052) |
| HapBr5 | A/C/−/A/G | 8 (0.038) |
| HapBr6 | A/C/GAT/A/A | 3 (0.014) |
| HapBr7 | A/C/−/C/G | 30 (0.142) |
| HapBr8 | G/C/GAT/C/G | 9 (0.042) |
| HapBr9 | G/C/−/C/G | 4 (0.019) |

Table 4 Caption: SLC22A1 haplotypes in the study cohort

### Regression modelling of metformin AUC0–48h

The initial univariate analyses assessed the independent effect of demographic variables (Table [2](#bcp13522-tbl-0002)), food *vs*. fasting prior to metformin administration, and pharmacogenetic variants (SNPs and *SCL22A1* diplotypes). Using a cut‐off value of *P <* 0.15 (Methods), the univariate analyses disclosed age, BSA, self‐reported brown or black colour, individual proportions of African and European ancestry, feeding status, rs12208357 (dominant model) and rs1867351 (codominant model) as being associated with metformin AUC_0–48h_ (Table [5](#bcp13522-tbl-0005)). None of the other nine SNPs nor the *SLC22A1* diplytopes under dominant or codominant model, associated (*P <* 0.15) with the metformin AUC_0–48h ._ The covariates significantly associated with AUC_0–48h_ in the univariate analyses were then entered into multivariate regression modelling, which retained age, BSA, individual proportions of African ancestry, food effect and rs12208357 in the final model (Table [5](#bcp13522-tbl-0005)). The partial *r* ^2^ statistics showed that age (partial *r* ^2^ = 0.091) and food (0.083) had the largest influence on metformin AUC_0–48h_, followed by BSA (0.050), rs12208357 (0.031) and African ancestry (0.029). Metformin AUC_0–48h_ increased with age, food and carriage of rs12208357, but was inversely associated with BSA and individual proportion of African ancestry. Based on the adjusted *R* ^2^ value, the final regression model explained 29.7% of the overall variability in metformin AUC_0–48h._

### Table 5.

| Covariates | Univariate regression | Multivariate regression |  |  |
| --- | --- | --- | --- | --- |
| P value | Regression coefficient | P valueb | Partial r 2 |  |
| Age | 0.004 | 0.007 | <0.001 | 0.091 |
| BSA | 0.012 | −0.245 | <0.008 | 0.050 |
| Race/Colour a |   |   |   |   |
| Brown | 0.066 |   |   |   |
| Black | 0.047 |   |   |   |
| Food effect a | <0.001 | 0.103 | 0.001 | 0.083 |
| European ancestry | 0.130 |   |   |   |
| African ancestry | 0.060 | −0.137 | 0.042 | 0.029 |
| SCL22A1 rs1867351 | 0.152 (codominant) |   |   |   |
| SCL22A1 rs12208357 | 0.077 (dominant) | 0.101 | 0.038 | 0.031 |
|   |   | R 2 = 0.332 |  |  |
|   |   | R 2 adjusted = 0.297 |  |  |

Table 5 Caption: Regression modelling of metformin AUC0–48h in Brazilians

Finally, prompted by the findings of Christensen *et al*. [8](#bcp13522-bib-0011) we examined the interaction between the OCT2 (rs316019) and MATE1 (rs2252281) polymorphisms with respect to the systemic exposure to metformin. Of the 37 individuals homozygous for the rs2252281 reference allele (Table [3](#bcp13522-tbl-0003)), 28 were also homozygous for the rs316019 reference allele, nine were heterozygous and none was homozygous for the variant allele. The metformin AUC_0–48h_ was larger in the wildtype homozygous (0.986 ± 0.172 ng h ml^−1^ mg^−1^) compared to the heterozygous individuals (0.908 ± 0.139 ng h ml^−1^ mg^−1^), which is consistent with the data on metformin clearance reported by of Christensen *et al*. [8](#bcp13522-bib-0011). However, the difference in AUC_0–48h_ between the rs316019 genotypes was not statistically significant (*P* = 0.21*)* in our study cohort.

## Discussion

Pharmacokinetic data from bioequivalence trials in healthy adult Brazilians were used to identify pharmacogenetic markers associated with interindividual variability in the systemic exposure to metformin, assessed by the parameter AUC_0–48h_. To increase statistical power to detect pharmacogenetic associations, we applied the EDP method [19](#bcp13522-bib-0021) to 256 bioequivalence data sets available and selected 106 individuals from the opposite ends of the distribution histogram of the AUC_0–48h._ We then applied the RDA method [20](#bcp13522-bib-0022) to these 106 data sets, and obtained an r_GC_ value of 0.93 (0.90–0.95), which suggests that the observed variability in the log‐transformed dose‐adjusted AUC_0–48h_ is largely under genetic control [20](#bcp13522-bib-0022). However, the multivariate regression modelling of the contribution of genetic and nongenetic covariates to the log‐transformed metformin AUC_0–48h_ disclosed a minor contribution of the genetic variants investigated. Thus, of the 11 SNPs examined, only rs12208357 was retained in the final multivariate regression model, accounting for 3.1% of the total variability in systemic exposure to metformin. Of notice, diplotypes comprising rs12208357 and four other SNPs in *SLC22A1* showed no association with metformin AUC_0–48h_.

The apparent quantitative discrepancy between the r_GC_ and the multivariate regression analysis, may involve several factors. First, the r_GC_ analysis was based on data from bioequivalence studies, in which the impact of environmental variables is considerably attenuated, due to the implicit strict control of timing of drug administration, food status, coadministered drugs or xenobiotics, smoking, alcohol intake etc. Consequently, it may be argued that the r_GC_ obtained from bioequivalence trials overestimates the genetic component of pharmacokinetic variability. Nevertheless, a similarly high r_GC_ value (0.94) was reported for the renal clearance of metformin measured in multiple occasions in individuals receiving the reference drug formulation [28](#bcp13522-bib-0028). This finding, taken as evidence for strong genetic control of metformin pharmacokinetic variability [28](#bcp13522-bib-0028), provides support for a similar interpretation of our results derived from bioequivalence trials. Second, we genotyped a selected number of SNPs in a set of metformin transporter genes, which did not include PMAT (*SLC29A4),* OCTN1 (*SLC22A4*) and the thiamine transporter, THTR‐2 *(SLC19A3*), that may be determinants of metformin pharmacokinetics. Genome‐wide association studies on metformin pharmacokinetics, might reveal additional variants that would account for the large difference in the pharmacogenomic impact inferred using the r_GC_ method *vs*. multivariate regression modelling. In the related context of metformin pharmacodynamics, genome‐wide association studies revealed two SNPs, one at a locus containing the ataxia telangiectasia mutated (*ATM*) gene and another in *SLC2A2*, encoding the facilitated glucose transporter GLUT2, that associated significantly with metformin‐induced clinical response. Of note, metformin was not a substrate or an inhibitor of GLUT2 in Xenopus laevis oocytes [29](#bcp13522-bib-0029), [30](#bcp13522-bib-0030). In addition, a recently published meta‐analysis failed to detect significant effects of nine SNPs in metformin transporters (seven of which were also investigated in the present study) on the glycaemic response to metformin [10](#bcp13522-bib-0013).

The statistical power of this study to detect significant associations of individual SNPs and *SLC22A1* diplotypes with metformin AUC_0–48h,_ is a third factor to be considered. Based on genotype frequencies and a dominant genetic model, the study was powered (β = 0.2, α = 0.05) to detect a genotype effect of reduction or increase of metformin AUC_0–48h_ ≥ 15% for rs12208357 and ≥10% for all other candidate SNPs, except the rare (MAF <0.01) rs34059508. Regarding the *SLC22A1* diplotypes*,* there was statistical power to detect increase or decrease of metformin AUC_0–48h_ ≥10% under a dominant model and ≥15% under a recessive model. However, our two‐step regression modelling adopted a nominal significance level of α = 0.15 in the initial, univariate analyses for selection of covariates to be entered into the multivariable regression analyses. Accordingly, a genotype or diplotype effect of reduction or increase of metformin AUC_0–48h_ >5% in the univariate analysis allowed inclusion of the respective covariate in the multivariate regression modelling. This was the case, of both rs12208357 and rs1867351, with *P* values of 0.08 and 0.15, respectively, in the univariate analyses (Table [5](#bcp13522-tbl-0005)). The final regression model retained only rs12208357, which accounted for 3.1% of the variability of metformin AUC_0–48h_ in the study cohort.

The rs12208357 in *SLC22A1* is a nonsynonymous transition (181C > T, R61C), first shown by Shu and colleagues [31](#bcp13522-bib-0031), [32](#bcp13522-bib-0032) to abolish OCT1 activity, leading to reduced metformin uptake into hepatocytes, attenuated effect of the drug on glucose tolerance tests and higher AUC. The latter finding was reproduced in the present study, despite differences in ethnicity, metformin dose and experimental protocol. The increased systemic exposure to metformin in carriers of rs12208357 conflicts with the increased renal clearance of metformin reported by Tvezkof *et al*. [33](#bcp13522-bib-0033). The latter observation, however, was not reproduced by Christensen and colleagues [34](#bcp13522-bib-0034), [35](#bcp13522-bib-0035). The latter authors examined the influence of *SLC22A1* (OCT1) diplotypes comprising rs12208357 and three other reduced‐function alleles on the metformin plasma concentration at steady state, and found the trough value to decrease with increasing number of reduced‐function haplotypes in DM2 patients [34](#bcp13522-bib-0034) but not in healthy volunteers [35](#bcp13522-bib-0035). A statistical type 2 error inflicted by the small sample size of the healthy cohort (*n* = 12) was considered a possible explanation for these discordant observations [35](#bcp13522-bib-0035). However, in our considerably larger (*n* = 106) cohort of healthy subjects, diplotypes comprising the five genotyped SNPs in *SCL22A1* also failed to influence metformin AUC_0–48._ A recent study using noninvasive ^11^C–metformin PET/CT scan revealed a novel implication of *SLC22A1* polymorphisms: the hepatic uptake of metformin was significantly reduced in carriers of rs12208357 (and rs72552763) irrespective of changes in circulating levels of metformin. This led to the conclusion that plasma levels of metformin do not reflect the drug concentration in hepatocytes where it exerts its pharmacodynamic effects [36](#bcp13522-bib-0036).

In addition to rs12208357, the final multivariate regression model of covariates significantly associated with metformin AUC_0–48_ retained age, BSA, food and individual proportions of African ancestry. Metformin AUC_0–48_ increased with age and food but was inversely associated with BSA and individual proportion of African ancestry. The effect of age on systemic exposure to metformin has been reported previously and ascribed to age‐related decline in kidney function [37](#bcp13522-bib-0037). Our cohort ranged in age from 18 to 48 years, and studies using GFR estimates on population‐based data suggested that the decline of kidney function may begin after the second decade of life [38](#bcp13522-bib-0038). The greater metformin AUC_0–48h_ in fed compared to fasted subjects in our study contrasts with previous reports of decreased or no change in bioavailability of metformin after high‐fat and/or high‐calorie meals [39](#bcp13522-bib-0039), [40](#bcp13522-bib-0040), [41](#bcp13522-bib-0041). Differences in fat and caloric content across studies may account for these discordant observations. The inverse relationship between BSA and metformin AUC_0–48_ in our cohort is a likely consequence of increasing volume of distribution with the increase in BSA. Finally, the inverse relationship between African ancestry and metformin AUC_0–48h_ is a novel observation which was made possible by enrolment of an admixed Brazilian cohort covering a wide range of individual proportions of European and African ancestry (Figure [S1](#bcp13522-supitem-0001)). Of notice, the individual self‐reported race/Colour was not retained in the final model, which is consistent with the notion that among Brazilians, correlation between skin colour and biogeographical ancestry is tenuous, at best [13](#bcp13522-bib-0016), [15](#bcp13522-bib-0018), [42](#bcp13522-bib-0042). Nevertheless, the inverse relationship between African ancestry and metformin AUC_0–48h_ is in line with the higher apparent clearance of metformin in African Americans, compared to European Americans [7](#bcp13522-bib-0010). Accordingly, Goswami *et al*. [7](#bcp13522-bib-0010) concluded that “for African Americans to achieve similar metformin exposure to that of European Americans a 26% increase in dose should be considered”. Importantly, attempts to extrapolate data from African Americans to Black Brazilians must take into consideration that although individual proportions of African ancestry vary widely among both groups, the average proportion of African ancestry is considerably higher in African Americans, compared to Black Brazilians [11](#bcp13522-bib-0014).

During the editorial reviewing process, it has been suggested that it would be interesting to use the remaining 150 individuals in the overall cohort to validate the final regression model for metformin AUC_0–48_. We agree and acknowledge as a limitation of the study the impossibility of carrying out such validation, because DNA from the remaining individuals was not available for genotyping the pharmacogenetic polymorphisms or the ancestry informative markers.

In conclusion, using the EDP approach and a multivariate regression modelling we identified a pharmacogenetic marker in OCT1 (*SLC22A1* rs12208357), which combined with demographic covariates (age, BSA and individual proportion of African ancestry) and a food effect explained 29% of the variability in the systemic exposure to metformin, administered as a single dose to an admixed Brazilian cohort of healthy subjects. Whether these findings may be extended to DM2 patients under chronic treatment with metformin requires further studies.

## Competing Interests

All authors have completed the Unified Competing Interest form at [http://www.icmje.org/coi_disclosure.pdf](http://www.icmje.org/coi_disclosure.pdf), and declare: G.S.K. received grant support from the Brazilian agencies Conselho Nacional de Desenvolvimento Científico e Tecnológico (CNPq), Fundação de Amparo à Pesquisa do Estado do Rio de Janeiro (FAPERJ) and Departamento de Ciência e Tecnologia, Ministério da Saúde; C.S.J. is supported by CNPq and FAPERJ; the other authors had no support from any organization; all authors had no financial relationships with any organizations that might have an interest in the submitted work in the previous 3 years and no other relationships or activities that could appear to have influenced the submitted work.

*Research in G.S.K.'s laboratory is supported by grants from CNPq, Faperj and DECIT/Ministry of Health, Brazil*.

## Supporting information

Santoro, A. B. , Botton, M. R. , Struchiner, C. J. , and Suarez‐Kurtz, G. (2018) Influence of pharmacogenetic polymorphisms and demographic variables on metformin pharmacokinetics in an admixed Brazilian cohort. Br J Clin Pharmacol, 84: 987–996. doi: [10.1111/bcp.13522](https://doi.org/10.1111/bcp.13522).

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Graham GG, Punt J, Arora M, Day RO, Doogue MP, Duong JK, et al Clinical pharmacokinetics of metformin. Clin Pharmacokinet 2011; 50: 81–98.  [DOI](https://doi.org/10.2165/11534750-000000000-00000) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21241070/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacokinet&title=Clinical%20pharmacokinetics%20of%20metformin&author=GG%20Graham&author=J%20Punt&author=M%20Arora&author=RO%20Day&author=MP%20Doogue&volume=50&publication_year=2011&pages=81-98&pmid=21241070&doi=10.2165/11534750-000000000-00000&)

2. Liang X, Giacomini KM. Transporters involved in metformin pharmacokinetics and treatment response. J Pharm Sci 2017; 106: 2245–2250.  [DOI](https://doi.org/10.1016/j.xphs.2017.04.078) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28495567/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharm%20Sci&title=Transporters%20involved%20in%20metformin%20pharmacokinetics%20and%20treatment%20response&author=X%20Liang&author=KM%20Giacomini&volume=106&publication_year=2017&pages=2245-2250&pmid=28495567&doi=10.1016/j.xphs.2017.04.078&)

3. Todd JN, Florez JC. An update on the pharmacogenomics of metformin: progress, problems and potential. Pharmacogenomics 2014; 15: 529–539.  [DOI](https://doi.org/10.2217/pgs.14.21) | [PMC free article](/articles/PMC4121960/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24624919/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=An%20update%20on%20the%20pharmacogenomics%20of%20metformin:%20progress,%20problems%20and%20potential&author=JN%20Todd&author=JC%20Florez&volume=15&publication_year=2014&pages=529-539&pmid=24624919&doi=10.2217/pgs.14.21&)

4. Kajbaf F, De Broe ME, Lalau JD. Therapeutic concentrations of metformin: a systematic review. Clin Pharmacokinet 2016; 55: 439–459.  [DOI](https://doi.org/10.1007/s40262-015-0323-x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26330026/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacokinet&title=Therapeutic%20concentrations%20of%20metformin:%20a%20systematic%20review&author=F%20Kajbaf&author=ME%20De%20Broe&author=JD%20Lalau&volume=55&publication_year=2016&pages=439-459&pmid=26330026&doi=10.1007/s40262-015-0323-x&)

5. Stage TB, Brøsen K, Christensen MM. A comprehensive review of drug‐drug interactions with metformin. Clin Pharmacokinet 2015; 54: 811–824.  [DOI](https://doi.org/10.1007/s40262-015-0270-6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25943187/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacokinet&title=A%20comprehensive%20review%20of%20drug%E2%80%90drug%20interactions%20with%20metformin&author=TB%20Stage&author=K%20Br%C3%B8sen&author=MM%20Christensen&volume=54&publication_year=2015&pages=811-824&pmid=25943187&doi=10.1007/s40262-015-0270-6&)

6. Pawlyk AC, Giacomini KM, McKeon C, Shuldiner AR, Florez JC. Metformin pharmacogenomics: current status and future directions. Diabetes 2014; 63: 2590–2599.  [DOI](https://doi.org/10.2337/db13-1367) | [PMC free article](/articles/PMC4113063/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25060887/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Diabetes&title=Metformin%20pharmacogenomics:%20current%20status%20and%20future%20directions&author=AC%20Pawlyk&author=KM%20Giacomini&author=C%20McKeon&author=AR%20Shuldiner&author=JC%20Florez&volume=63&publication_year=2014&pages=2590-2599&pmid=25060887&doi=10.2337/db13-1367&)

7. Goswami S, Yee SW, Stocker S, Mosley JD, Kubo M, Castro R, et al Genetic variants in transcription factors are associated with the pharmacokinetics and pharmacodynamics of metformin. Clin Pharmacol Ther 2014; 96: 370–379.  [DOI](https://doi.org/10.1038/clpt.2014.109) | [PMC free article](/articles/PMC4171106/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24853734/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Genetic%20variants%20in%20transcription%20factors%20are%20associated%20with%20the%20pharmacokinetics%20and%20pharmacodynamics%20of%20metformin&author=S%20Goswami&author=SW%20Yee&author=S%20Stocker&author=JD%20Mosley&author=M%20Kubo&volume=96&publication_year=2014&pages=370-379&pmid=24853734&doi=10.1038/clpt.2014.109&)

8. Christensen MM, Pedersen RS, Stage TB, Brasch‐Andersen C, Nielsen F, Damkier P, et al A gene–gene interaction between polymorphisms in the OCT2 and MATE1 genes influences the renal clearance of metformin. Pharmacogenet Genomics 2013; 23: 526–534.  [DOI](https://doi.org/10.1097/FPC.0b013e328364a57d) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23873119/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=A%20gene%E2%80%93gene%20interaction%20between%20polymorphisms%20in%20the%20OCT2%20and%20MATE1%20genes%20influences%20the%20renal%20clearance%20of%20metformin&author=MM%20Christensen&author=RS%20Pedersen&author=TB%20Stage&author=C%20Brasch%E2%80%90Andersen&author=F%20Nielsen&volume=23&publication_year=2013&pages=526-534&pmid=23873119&doi=10.1097/FPC.0b013e328364a57d&)

9. Arimany‐Nardi C, Koepsell H, Pastor‐Anglada M. Role of SLC22A1 polymorphic variants in drug disposition, therapeutic responses, and drug‐drug interactions. Pharmacogenomics J 2015; 15: 473–487.  [DOI](https://doi.org/10.1038/tpj.2015.78) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26526073/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=Role%20of%20SLC22A1%20polymorphic%20variants%20in%20drug%20disposition,%20therapeutic%20responses,%20and%20drug%E2%80%90drug%20interactions&author=C%20Arimany%E2%80%90Nardi&author=H%20Koepsell&author=M%20Pastor%E2%80%90Anglada&volume=15&publication_year=2015&pages=473-487&pmid=26526073&doi=10.1038/tpj.2015.78&)

10. Dujic T, Zhou K, Yee SW, van Leeuwen N, de Keyser CE, Javorský M, et al Variants in pharmacokinetic transporters and glycemic response to metformin: a Metgen meta‐analysis. Clin Pharmacol Ther 2017; 101: 763–772.  [DOI](https://doi.org/10.1002/cpt.567) | [PMC free article](/articles/PMC5425333/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27859023/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Variants%20in%20pharmacokinetic%20transporters%20and%20glycemic%20response%20to%20metformin:%20a%20Metgen%20meta%E2%80%90analysis&author=T%20Dujic&author=K%20Zhou&author=SW%20Yee&author=N%20van%20Leeuwen&author=CE%20de%20Keyser&volume=101&publication_year=2017&pages=763-772&pmid=27859023&doi=10.1002/cpt.567&)

11. Suarez‐Kurtz G, Pena SD. Pharmacogenomics in the Americas: the impact of genetic admixture. Curr Drug Targets 2006; 7: 1649–1658.  [DOI](https://doi.org/10.2174/138945006779025392) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17168840/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Drug%20Targets&title=Pharmacogenomics%20in%20the%20Americas:%20the%20impact%20of%20genetic%20admixture&author=G%20Suarez%E2%80%90Kurtz&author=SD%20Pena&volume=7&publication_year=2006&pages=1649-1658&pmid=17168840&doi=10.2174/138945006779025392&)

12. Suarez‐Kurtz G. Pharmacogenetics in the Brazilian population. Front Pharmacol 2010; 1: 118.  [DOI](https://doi.org/10.3389/fphar.2010.00118) | [PMC free article](/articles/PMC3153000/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21833165/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front%20Pharmacol&title=Pharmacogenetics%20in%20the%20Brazilian%20population&author=G%20Suarez%E2%80%90Kurtz&volume=1&publication_year=2010&pages=118&pmid=21833165&doi=10.3389/fphar.2010.00118&)

13. Suarez‐Kurtz G, Pena SD, Struchiner CJ, Hutz MH. Pharmacogenomic diversity among Brazilians: Influence of ancestry, self‐reported color, and geographical origin. Front Pharmacol 2012; 3: 191.  [DOI](https://doi.org/10.3389/fphar.2012.00191) | [PMC free article](/articles/PMC3490152/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23133420/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front%20Pharmacol&title=Pharmacogenomic%20diversity%20among%20Brazilians:%20Influence%20of%20ancestry,%20self%E2%80%90reported%20color,%20and%20geographical%20origin&author=G%20Suarez%E2%80%90Kurtz&author=SD%20Pena&author=CJ%20Struchiner&author=MH%20Hutz&volume=3&publication_year=2012&pages=191&pmid=23133420&doi=10.3389/fphar.2012.00191&)

14. Salzano FM, Bortololini MC. Genetics and evolution of Latin American populations. Cambridge: Cambridge University Press, 2002.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Genetics%20and%20evolution%20of%20Latin%20American%20populations&author=FM%20Salzano&author=MC%20Bortololini&publication_year=2002&)

15. Pena SD, Di Pietro G, Fuchshuber‐Moraes M, Genro JP, Hutz MH, Kehdy Fde S, et al The genomic ancestry of individuals from different geographical regions of Brazil is more uniform than expected. PLoS One 2011; 6: e17063.  [DOI](https://doi.org/10.1371/journal.pone.0017063) | [PMC free article](/articles/PMC3040205/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21359226/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=The%20genomic%20ancestry%20of%20individuals%20from%20different%20geographical%20regions%20of%20Brazil%20is%20more%20uniform%20than%20expected&author=SD%20Pena&author=G%20Di%20Pietro&author=M%20Fuchshuber%E2%80%90Moraes&author=JP%20Genro&author=MH%20Hutz&volume=6&publication_year=2011&pages=e17063&pmid=21359226&doi=10.1371/journal.pone.0017063&)

16. Anvisa . Brazilian National Surveillance Agency. Available at http://www.portal.anvisa.gov.br (last accessed 10 August 2017).  [http://www.portal.anvisa.gov.br](http://www.portal.anvisa.gov.br)

17. IBGE, Instituto Brasileiro de Geografia e Estatística . Available at https://sidra.ibge.gov.br/tabela/2093 (last accessed 10 August 2017).  [https://sidra.ibge.gov.br/tabela/2093](https://sidra.ibge.gov.br/tabela/2093)

18. Santoro AB, Stage TB, Struchiner CJ, Christensen MM, Brosen K, Suarez‐Kurtz G. Limited sampling strategy for determining metformin area under the plasma concentration‐time curve. Br J Clin Pharmacol 2016; 82: 1002–1010.  [DOI](https://doi.org/10.1111/bcp.13049) | [PMC free article](/articles/PMC5137825/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27324407/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Clin%20Pharmacol&title=Limited%20sampling%20strategy%20for%20determining%20metformin%20area%20under%20the%20plasma%20concentration%E2%80%90time%20curve&author=AB%20Santoro&author=TB%20Stage&author=CJ%20Struchiner&author=MM%20Christensen&author=K%20Brosen&volume=82&publication_year=2016&pages=1002-1010&pmid=27324407&doi=10.1111/bcp.13049&)

19. Nebert DW. Extreme discordant phenotype methodology: An intuitive approach to clinical pharmacogenetics. Eur J Pharmacol 2000; 410: 107–120.  [DOI](https://doi.org/10.1016/s0014-2999(00)00809-8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11134663/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Pharmacol&title=Extreme%20discordant%20phenotype%20methodology:%20An%20intuitive%20approach%20to%20clinical%20pharmacogenetics&author=DW%20Nebert&volume=410&publication_year=2000&pages=107-120&pmid=11134663&doi=10.1016/s0014-2999(00)00809-8&)

20. Kalow W, Tang BK, Endrenyi L. Hypothesis: comparisons of inter‐ and intraindividual variations can substitute for twin studies in drug research. Pharmacogenetics 1998; 8: 283–289.  [DOI](https://doi.org/10.1097/00008571-199808000-00001) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9731714/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=Hypothesis:%20comparisons%20of%20inter%E2%80%90%20and%20intraindividual%20variations%20can%20substitute%20for%20twin%20studies%20in%20drug%20research&author=W%20Kalow&author=BK%20Tang&author=L%20Endrenyi&volume=8&publication_year=1998&pages=283-289&pmid=9731714&doi=10.1097/00008571-199808000-00001&)

21. CubeX: Cubic Exact Solution. Available at http://www.oege.org/software/cubex (last accessed 10 August 2017).  [http://www.oege.org/software/cubex](http://www.oege.org/software/cubex)

22. Stephens M, Donnelly P. A comparison of Bayesian methods for haplotype reconstruction from population genotype data. Am J Hum Genet 2003; 73: 1162–1169.  [DOI](https://doi.org/10.1086/379378) | [PMC free article](/articles/PMC1180495/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14574645/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=A%20comparison%20of%20Bayesian%20methods%20for%20haplotype%20reconstruction%20from%20population%20genotype%20data&author=M%20Stephens&author=P%20Donnelly&volume=73&publication_year=2003&pages=1162-1169&pmid=14574645&doi=10.1086/379378&)

23. Bastos‐Rodrigues L, Pimenta JR, Pena SDJ. The genetic structure of human populations studied through short insertion–deletion polymorphisms. Ann Hum Genet 2006; 70: 658–665.  [DOI](https://doi.org/10.1111/j.1469-1809.2006.00287.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29488221/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Hum%20Genet&title=The%20genetic%20structure%20of%20human%20populations%20studied%20through%20short%20insertion%E2%80%93deletion%20polymorphisms&author=L%20Bastos%E2%80%90Rodrigues&author=JR%20Pimenta&author=SDJ%20Pena&volume=70&publication_year=2006&pages=658-665&pmid=29488221&doi=10.1111/j.1469-1809.2006.00287.x&)

24. Pritchard JK, Stephens M, Donnelly P. Inference of population structure using multilocus genotype data. Genetics 2000; 155: 945–959.  [DOI](https://doi.org/10.1093/genetics/155.2.945) | [PMC free article](/articles/PMC1461096/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10835412/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genetics&title=Inference%20of%20population%20structure%20using%20multilocus%20genotype%20data&author=JK%20Pritchard&author=M%20Stephens&author=P%20Donnelly&volume=155&publication_year=2000&pages=945-959&pmid=10835412&doi=10.1093/genetics/155.2.945&)

25. G*Power: Statistical Power Analyses . Available at http://www.gpower.hhu.de/en.html (last accessed 10 August 2017).  [http://www.gpower.hhu.de/en.html](http://www.gpower.hhu.de/en.html)

26. Harding SD, Sharman JL, Faccenda E, Southan C, Pawson AJ, Ireland S, et al The IUPHAR/BPS Guide to PHARMACOLOGY in 2018: updates and expansion to encompass the new guide to IMMUNOPHARMACOLOGY. Nucl Acid Res 2018; 46: D1091–D1106.  [DOI](https://doi.org/10.1093/nar/gkx1121) | [PMC free article](/articles/PMC5753190/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29149325/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucl%20Acid%20Res&title=The%20IUPHAR/BPS%20Guide%20to%20PHARMACOLOGY%20in%202018:%20updates%20and%20expansion%20to%20encompass%20the%20new%20guide%20to%20IMMUNOPHARMACOLOGY&author=SD%20Harding&author=JL%20Sharman&author=E%20Faccenda&author=C%20Southan&author=AJ%20Pawson&volume=46&publication_year=2018&pages=D1091-D1106&pmid=29149325&doi=10.1093/nar/gkx1121&)

27. Alexander SPH, Kelly E, Marrion NV, Peters JA, Faccenda E, Harding SD, et al The Concise Guide to PHARMACOLOGY 2017/18: Transporters. Br J Pharmacol 2017; 174: S360–S446.  [DOI](https://doi.org/10.1111/bph.13883) | [PMC free article](/articles/PMC5650669/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29055035/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Pharmacol&title=The%20Concise%20Guide%20to%20PHARMACOLOGY%202017/18:%20Transporters&author=SPH%20Alexander&author=E%20Kelly&author=NV%20Marrion&author=JA%20Peters&author=E%20Faccenda&volume=174&publication_year=2017&pages=S360-S446&pmid=29055035&doi=10.1111/bph.13883&)

28. Leabman MK, Giacomini KM. Estimating the contribution of genes and environment to variation in renal drug clearance. Pharmacogenetics 2003; 13: 581–584.  [DOI](https://doi.org/10.1097/00008571-200309000-00007) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12972957/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=Estimating%20the%20contribution%20of%20genes%20and%20environment%20to%20variation%20in%20renal%20drug%20clearance&author=MK%20Leabman&author=KM%20Giacomini&volume=13&publication_year=2003&pages=581-584&pmid=12972957&doi=10.1097/00008571-200309000-00007&)

29. GoDARTS and UKPDS Diabetes Pharmacogenetics Study Group , Wellcome Trust Case Control Consortium , Zhou K, Bellenguez C, Spencer CC, Bennett AJ, et al Common variants near ATM are associated with glycemic response to metformin in type 2 diabetes. Nat Genet 2011; 43: 117–120.  [DOI](https://doi.org/10.1038/ng.735) | [PMC free article](/articles/PMC3030919/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21186350/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Common%20variants%20near%20ATM%20are%20associated%20with%20glycemic%20response%20to%20metformin%20in%20type%202%20diabetes&author=K%20Zhou&author=C%20Bellenguez&author=CC%20Spencer&author=AJ%20Bennett&volume=43&publication_year=2011&pages=117-120&pmid=21186350&doi=10.1038/ng.735&)

30. Hedderson MM, Palmer CNA, Florez JC, Giacomini KM, Pearson ER. Variation in the glucose transporter gene SLC2A2 is associated with glycemic response to metformin. Nat Genet 2016; 48: 1055–1059.  [DOI](https://doi.org/10.1038/ng.3632) | [PMC free article](/articles/PMC5007158/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27500523/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Variation%20in%20the%20glucose%20transporter%20gene%20SLC2A2%20is%20associated%20with%20glycemic%20response%20to%20metformin&author=MM%20Hedderson&author=CNA%20Palmer&author=JC%20Florez&author=KM%20Giacomini&author=ER%20Pearson&volume=48&publication_year=2016&pages=1055-1059&pmid=27500523&doi=10.1038/ng.3632&)

31. Shu Y, Leabman MK, Feng B, Mangravite LM, Huang CC, Stryke D, et al Evolutionary conservation predicts function of variants of the human organic cation transporter, OCT1. Proc Natl Acad Sci U S A 2003; 100: 5902–5907.  [DOI](https://doi.org/10.1073/pnas.0730858100) | [PMC free article](/articles/PMC156299/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12719534/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&title=Evolutionary%20conservation%20predicts%20function%20of%20variants%20of%20the%20human%20organic%20cation%20transporter,%20OCT1&author=Y%20Shu&author=MK%20Leabman&author=B%20Feng&author=LM%20Mangravite&author=CC%20Huang&volume=100&publication_year=2003&pages=5902-5907&pmid=12719534&doi=10.1073/pnas.0730858100&)

32. Shu Y, Brown C, Castro RA, Shi RJ, Lin ET, Owen RP, et al Effect of genetic variation in the organic cation transporter 1, OCT1, on metformin pharmacokinetics. Clin Pharmacol Ther 2008; 83: 273–280.  [DOI](https://doi.org/10.1038/sj.clpt.6100275) | [PMC free article](/articles/PMC2976713/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17609683/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Effect%20of%20genetic%20variation%20in%20the%20organic%20cation%20transporter%201,%20OCT1,%20on%20metformin%20pharmacokinetics&author=Y%20Shu&author=C%20Brown&author=RA%20Castro&author=RJ%20Shi&author=ET%20Lin&volume=83&publication_year=2008&pages=273-280&pmid=17609683&doi=10.1038/sj.clpt.6100275&)

33. Tzvetkov MV, Vormfelde SV, Balen D, Meineke I, Schmidt T, Sehrt D, et al The effects of genetic polymorphisms in the organic cation transporters OCT1, OCT2, and OCT3 on the renal clearance of metformin. Clin Pharmacol Ther 2009; 86: 299–306.  [DOI](https://doi.org/10.1038/clpt.2009.92) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19536068/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=The%20effects%20of%20genetic%20polymorphisms%20in%20the%20organic%20cation%20transporters%20OCT1,%20OCT2,%20and%20OCT3%20on%20the%20renal%20clearance%20of%20metformin&author=MV%20Tzvetkov&author=SV%20Vormfelde&author=D%20Balen&author=I%20Meineke&author=T%20Schmidt&volume=86&publication_year=2009&pages=299-306&pmid=19536068&doi=10.1038/clpt.2009.92&)

34. Christensen MM, Brasch‐Andersen C, Green H, Nielsen F, Damkier P, Beck‐Nielsen H, et al The pharmacogenetics of metformin and its impact on plasma metformin steady‐state levels and glycosylated hemoglobin A1c. Pharmacogenet Genomics 2011; 21: 837–850.  [DOI](https://doi.org/10.1097/FPC.0b013e32834c0010) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21989078/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=The%20pharmacogenetics%20of%20metformin%20and%20its%20impact%20on%20plasma%20metformin%20steady%E2%80%90state%20levels%20and%20glycosylated%20hemoglobin%20A1c&author=MM%20Christensen&author=C%20Brasch%E2%80%90Andersen&author=H%20Green&author=F%20Nielsen&author=P%20Damkier&volume=21&publication_year=2011&pages=837-850&pmid=21989078&doi=10.1097/FPC.0b013e32834c0010&)

35. Christensen MM, Højlund K, Hother‐Nielsen O, Stage TB, Damkier P, Beck‐Nielsen H, et al Steady‐state pharmacokinetics of metformin is independent of the OCT1 genotype in healthy volunteers. Eur J Clin Pharmacol 2015; 71: 691–697.  [DOI](https://doi.org/10.1007/s00228-015-1853-8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25939711/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Clin%20Pharmacol&title=Steady%E2%80%90state%20pharmacokinetics%20of%20metformin%20is%20independent%20of%20the%20OCT1%20genotype%20in%20healthy%20volunteers&author=MM%20Christensen&author=K%20H%C3%B8jlund&author=O%20Hother%E2%80%90Nielsen&author=TB%20Stage&author=P%20Damkier&volume=71&publication_year=2015&pages=691-697&pmid=25939711&doi=10.1007/s00228-015-1853-8&)

36. Christensen M, Brøsen K, Frøkiaer J, Jessen N. Genetic polymorphisms in organic cation transporter 1 attenuates hepatic metformin exposure in humans. Clin Pharmacol Ther 2017; 102: 841–848.  [DOI](https://doi.org/10.1002/cpt.701) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28380657/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Genetic%20polymorphisms%20in%20organic%20cation%20transporter%201%20attenuates%20hepatic%20metformin%20exposure%20in%20humans&author=M%20Christensen&author=K%20Br%C3%B8sen&author=J%20Fr%C3%B8kiaer&author=N%20Jessen&volume=102&publication_year=2017&pages=841-848&pmid=28380657&doi=10.1002/cpt.701&)

37. Sambol NC, Chiang J, Lin ET, Goodman AM, Liu CY, Benet LZ, et al Kidney function and age are both predictors of pharmacokinetics of metformin. J Clin Pharmacol 1995; 35: 1094–1102.  [DOI](https://doi.org/10.1002/j.1552-4604.1995.tb04033.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8626883/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Pharmacol&title=Kidney%20function%20and%20age%20are%20both%20predictors%20of%20pharmacokinetics%20of%20metformin&author=NC%20Sambol&author=J%20Chiang&author=ET%20Lin&author=AM%20Goodman&author=CY%20Liu&volume=35&publication_year=1995&pages=1094-1102&pmid=8626883&doi=10.1002/j.1552-4604.1995.tb04033.x&)

38. Coresh J, Selvin E, Stevens LA, Manzi J, Kusek JW, Eggers P, et al Prevalence of chronic kidney disease in the United States. JAMA 2007; 298: 2038–2047.  [DOI](https://doi.org/10.1001/jama.298.17.2038) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17986697/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA&title=Prevalence%20of%20chronic%20kidney%20disease%20in%20the%20United%20States&author=J%20Coresh&author=E%20Selvin&author=LA%20Stevens&author=J%20Manzi&author=JW%20Kusek&volume=298&publication_year=2007&pages=2038-2047&pmid=17986697&doi=10.1001/jama.298.17.2038&)

39. Sambol NC, Brookes LG, Chiang J, Goodman AM, Lin ET, Liu CY, et al Food intake and dosage level, but not tablet vs solution dosage form, affect the absorption of metformin HCl in man. Br J Clin Pharmacol 1996; 42: 510–512.  [DOI](https://doi.org/10.1111/j.1365-2125.1996.tb00017.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8904626/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Clin%20Pharmacol&title=Food%20intake%20and%20dosage%20level,%20but%20not%20tablet%20vs%20solution%20dosage%20form,%20affect%20the%20absorption%20of%20metformin%20HCl%20in%20man&author=NC%20Sambol&author=LG%20Brookes&author=J%20Chiang&author=AM%20Goodman&author=ET%20Lin&volume=42&publication_year=1996&pages=510-512&pmid=8904626&doi=10.1111/j.1365-2125.1996.tb00017.x&)

40. Marathe PH, Arnold ME, Meeker J, Greene DS, Barbhaiya RH. Pharmacokinetics and bioavailability of a metformin/glyburide tablet administered alone and with food. J Clin Pharmacol 2000; 40: 1494–1502.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/11185672/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Pharmacol&title=Pharmacokinetics%20and%20bioavailability%20of%20a%20metformin/glyburide%20tablet%20administered%20alone%20and%20with%20food&author=PH%20Marathe&author=ME%20Arnold&author=J%20Meeker&author=DS%20Greene&author=RH%20Barbhaiya&volume=40&publication_year=2000&pages=1494-1502&pmid=11185672&)

41. Karim A, Slater M, Bradford D, Schwartz L, Laurent A. Oral antidiabetic drugs: effect of food on absorption of pioglitazone and metformin from a fixed‐dose combination tablet. J Clin Pharmacol 2007; 47: 48–55.  [DOI](https://doi.org/10.1177/0091270006293756) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17192501/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Pharmacol&title=Oral%20antidiabetic%20drugs:%20effect%20of%20food%20on%20absorption%20of%20pioglitazone%20and%20metformin%20from%20a%20fixed%E2%80%90dose%20combination%20tablet&author=A%20Karim&author=M%20Slater&author=D%20Bradford&author=L%20Schwartz&author=A%20Laurent&volume=47&publication_year=2007&pages=48-55&pmid=17192501&doi=10.1177/0091270006293756&)

42. Parra FC, Amado RC, Lambertucci JR, Rocha J, Antunes CM, Pena SD. Color and genomic ancestry in Brazilians. Proc Natl Acad Sci U S A 2003; 100: 177–182.  [DOI](https://doi.org/10.1073/pnas.0126614100) | [PMC free article](/articles/PMC140919/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12509516/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&title=Color%20and%20genomic%20ancestry%20in%20Brazilians&author=FC%20Parra&author=RC%20Amado&author=JR%20Lambertucci&author=J%20Rocha&author=CM%20Antunes&volume=100&publication_year=2003&pages=177-182&pmid=12509516&doi=10.1073/pnas.0126614100&)

43. Song IS, Shin HJ, Shim EJ, Jung IS, Kim WY, Shon JH, et al Genetic variants of the organic cation transporter 2 influence the disposition of metformin. Clin Pharmacol Ther 2008; 84: 559–562.  [DOI](https://doi.org/10.1038/clpt.2008.61) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18401339/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Genetic%20variants%20of%20the%20organic%20cation%20transporter%202%20influence%20the%20disposition%20of%20metformin&author=IS%20Song&author=HJ%20Shin&author=EJ%20Shim&author=IS%20Jung&author=WY%20Kim&volume=84&publication_year=2008&pages=559-562&pmid=18401339&doi=10.1038/clpt.2008.61&)

44. Choi JH, Yee SW, Kim MJ, Nguyen L, Lee JH, Kang J‐O, et al Identification and characterization of novel polymorphisms in the basal promoter of the human transporter, MATE. Pharmacogenet Genomics 2009; 19: 770–780.  [DOI](https://doi.org/10.1097/FPC.0b013e328330eeca) | [PMC free article](/articles/PMC2976711/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19745787/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Identification%20and%20characterization%20of%20novel%20polymorphisms%20in%20the%20basal%20promoter%20of%20the%20human%20transporter,%20MATE&author=JH%20Choi&author=SW%20Yee&author=MJ%20Kim&author=L%20Nguyen&author=JH%20Lee&volume=19&publication_year=2009&pages=770-780&pmid=19745787&doi=10.1097/FPC.0b013e328330eeca&)

45. Choi JH, Yee SW, Ramirez AH, Morrissey KM, Jang GH, Joski PJ, et al A Common 5'‐UTR Variant in MATE2‐K is associated with poor response to metformin. Clin Pharmacol Ther 2011; 90: 674–684.  [DOI](https://doi.org/10.1038/clpt.2011.165) | [PMC free article](/articles/PMC3329222/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21956618/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=A%20Common%205'%E2%80%90UTR%20Variant%20in%20MATE2%E2%80%90K%20is%20associated%20with%20poor%20response%20to%20metformin&author=JH%20Choi&author=SW%20Yee&author=AH%20Ramirez&author=KM%20Morrissey&author=GH%20Jang&volume=90&publication_year=2011&pages=674-684&pmid=21956618&doi=10.1038/clpt.2011.165&)
