# An integrated pharmacokinetic/pharmacogenomic analysis of ABCB1 and SLCO1B1 polymorphisms on edoxaban exposure

## Metadata
**Authors:** A G Vandell, J Lee, M Shi, I Rubets, K S Brown, J R Walker
**Journal:** The Pharmacogenomics Journal
**Date:** 2016 Nov 29
**DOI:** [10.1038/tpj.2016.82](https://doi.org/10.1038/tpj.2016.82)
**PMID:** 27897269
**PMCID:** PMC5817390
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5817390/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC5817390/pdf/tpj201682a.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC5817390/pdf/tpj201682a.pdf)

## Abstract

Edoxaban and its low-abundance, active metabolite M4 are substrates of P-glycoprotein (P-gp; MDR1) and organic anion transporter protein 1B1 (OATP1B1), respectively, and pharmacological inhibitors of P-gp and OATP1B1 can affect edoxaban and M4 pharmacokinetics (PK). In this integrated pharmacogenomic analysis, genotype and concentration–time data from 458 healthy volunteers in 14 completed phase 1 studies were pooled to examine the impact on edoxaban PK parameters of allelic variants of ABCB1 (rs1045642: C3435T) and SLCO1B1 (rs4149056: T521C), which encode for P-gp and OATP1B1. Although some pharmacologic inhibitors of P-gp and OATP1B1 increase edoxaban exposure, neither the ABCB1 C3435T nor the SLCO1B1 T521C polymorphism affected edoxaban PK. A slight elevation in M4 exposure was observed among SLCO1B1 C-allele carriers; however, this elevation is unlikely to be clinically significant as plasma M4 concentrations comprise <10% of total edoxaban levels.

## Introduction

Edoxaban is a once-daily, orally administered, selective and direct inhibitor of factor Xa, a key mediator of both the intrinsic and extrinsic pathways of the clotting cascade.^1, 2^ By binding to free Xa as well as Xa bound to the prothrombinase complex, edoxaban attenuates thrombin generation, and consequently, thrombus formation.^1, 2^ Edoxaban is approved in Japan for the prevention of venous thromboembolism (VTE) following major orthopedic surgery.^3^ Based on the results of 2 large clinical trials,^4, 5^ edoxaban recently received approval for the prevention of stroke and systemic embolism in patients with nonvalvular atrial fibrillation and the treatment of venous thromboembolism in Japan,^6^ the United States,^7^ Switzerland^8^ and the European Union.^9^

Edoxaban is a substrate of the efflux pump P-glycoprotein (P-gp; also known as multidrug resistance protein 1 or MDR1) and its metabolite M4 is a substrate of the hepatic uptake transporter organic anion transporter protein 1B1 (OATP1B1), as demonstrated in *in vitro* studies.^10^ Drug–drug interaction studies have demonstrated that changes in transporter function may influence edoxaban and M4 exposure in a clinically significant manner. P-gp inhibitors, including quinidine and verapamil, increased edoxaban exposure in healthy subjects and in nonvalvular atrial fibrillation patients.^11, 12^ Cyclosporine, which inhibits several transporters including OATP1B1, and rifampin increase M4 levels in healthy volunteers.^13, 14^

Genetic variations in *ABCB1* (ATP-binding cassette, sub-family B, member 1) and *SLCO1B1* (solute carrier organic anion transporter family, member 1B1), the genes that encode for P-gp and OATP1B1, respectively, are known to affect transporter function and thus substrate exposure. In particular, studies have indicated that a single-nucleotide polymorphism (SNP) in *ABCB1*, rs1045642—a C to T substitution at position 3435 of the coding sequence—alters P-gp expression and may affect the pharmacokinetics (PK) of some P-gp substrates, although reports regarding the nature of the effect are conflicting.^15, 16^ Similarly, the *SLCO1B1* polymorphism, rs4149056—a T to C substitution at position 521 of the coding sequence—is associated with reduced transporter activity; augments exposure of OATP1B1 substrates such as atorvastatin, rosuvastatin and pravastatin;^17, 18^ and is associated with increased risk of statin-induced myalgia and rhabdomyolysis.^19^

Edoxaban undergoes minimal metabolism as nearly 73% of the absorbed drug is eliminated unmodified.^20^ M4 (also known as D21-2393), which has anticoagulant properties similar to edoxaban (data on file), is a low-abundance, active metabolite of edoxaban, accounting for <10% of the total anticoagulant exposure.^20^ The metabolic enzyme, carboxylesterase 1 (CES1), catalyzes the formation of M4 from edoxaban.^20^ The terminal half-life of edoxaban ranges from 10 to 14 h,^21^ and ~50% of the absorbed dose is renally eliminated.^20, 22^ The *in vitro* plasma protein binding of edoxaban and M4 are ~55% and 80%, respectively.^21, 23^

Although increased edoxaban and M4 exposures are observed upon administration of the P-gp inhibitors quinidine and verapamil or the OATP1B1 inhibitor cyclosporine, the effects of common genetic variants that alter the function of these transporters on edoxaban and M4 PK are unknown. To answer these questions, a prospective-retrospective integrated analysis using pooled genotype and PK data from 14 completed edoxaban phase 1 studies was conducted.

## Materials and methods

### Analysis design and population selection

Twenty-four phase 1 edoxaban studies were considered for inclusion in the present integrated analysis. Completed studies were included in the current analysis provided they enrolled healthy subjects, included a treatment arm in which a single 60-mg edoxaban dose was administered under fasting conditions, and collected DNA samples that were available for genotyping. Only data from the edoxaban 60 mg treatment groups, in which edoxaban was administered alone and not with any potentially interacting drugs or food, were included in the analysis. Based on these criteria, DNA samples and PK data from 458 healthy subjects (365 males and 93 females) from 14 completed edoxaban phase 1 clinical studies were pooled for this analysis ([Supplementary Table 1](#sup1)). All subjects provided written informed consent to participate in the clinical studies and supply genetic material. Each study was approved by an institutional review board and was carried out in compliance with the Declaration of Helsinki and the International Conference on Harmonisation.

For each of the included phase 1 studies, serial blood samples were collected at predose and up to 96 h postdose to determine edoxaban and M4 plasma concentrations in accordance with the specific study protocols.

### Bioanalytic methods

Edoxaban and M4 plasma concentrations were measured by a validated liquid chromatography tandem mass spectrometry (LC–MS/MS) method for all studies by Advion BioServices (Ithaca, New York, USA). The lower limit of quantitation for edoxaban and M4 was 0.764 ng ml^−1^ and 0.0792 ng ml^−1^, respectively.

### Genotype analysis

Validated commercial Taqman assays (Life Technologies, Waltham, MA, USA) were performed to determine *ABCB1* rs1045642 and *SLCO1B1* rs4149056 genotypes by Gentris Corporation (Morrisville, North Carolina, USA). Assays were carried out in duplicate. If results were not in agreement, the sample was genotyped using Sanger sequencing. Results were further confirmed by reanalyzing ~25 (~5%) randomly selected samples for both genotypes using Sanger sequencing in both the forward and reverse direction. Using this approach, the genotype call rate was 100%.

### Pharmacokinetic and statistical analysis

An integrated database consisting of PK parameters and genotype data from each of the 458 subjects included in this analysis was created. The PK parameters AUC_inf_, AUC_0–24_, AUC_last_, *C*_max_, and *C*_24_ were computed using a noncompartmental approach with WinNonlin Professional software Version 4.0 or 5.2 (Pharsight Corporation, Montreal, Canada). PK parameters were statistically compared between genotypes using analysis of variance (ANOVA) in SAS Proc MIXED (SAS Institute, Cary, North Carolina, USA), with genotype and study ID as fixed effects. Point estimates and 90% confidence intervals (CI) for the ratios of the PK parameters between different allele variations of *ABCB1* rs1045642 (CC vs CT, CC vs TT, and CT vs TT) and *SLCO1B1* rs4149056 (C-carriers vs TT) were calculated using multiple pairwise comparisons by applying exponential transformation to the difference in least squares (LS) means calculated using ln-transformed values and CIs obtained from the ANOVA model.

Assuming the expected mean ratio was within the range of 95% to 105% with Bonferroni adjustment for multiple comparisons among the three rs1045642 variant subgroups, a genotype subgroup of at least 37, 105 and 98 subjects for edoxaban AUC_inf_, *C*_max_ and *C*_24_, respectively, was required in order to have at least 80% power so that the 90% CI for the mean ratio would fall within the range of 80–125%. Using similar criteria, rs4149056 genotype subgroups required a minimum of 26, 75 and 70 subjects for edoxaban AUC_inf_, *C*_max_ and *C*_24_, respectively, while the corresponding PK measures for M4 required 57, 103 and 96 subjects, respectively. PK parameters were not considered to be different between genotype groups if the 90% CIs for the geometric LS means ratios of the analysis groups were within the range of 80–125%.

The metabolite-to-parent ratios (MPRs) were calculated for AUC_last_, AUC_inf_ and *C*_max_ with adjustments for molecular weights and summarized by *SLCO1B1* rs4149056 genotype (TT and C-carriers). The molecular weights used in the MPR calculations were 548.07 g mol^−1^ for edoxaban (parent) and 520.99 g mol^−1^ for the metabolite M4.

## Results

### Demographics and baseline characteristics

The demographics and baseline characteristics of subjects included in this analysis are listed by *ABCB1* and *SLCO1B1* genotype in [Table 1](#tbl1). The mean age, body mass index, and renal function were similar across genotype groups. The *ABCB1* and *SLCO1B1* SNPs were in Hardy–Weinberg equilibrium for all race groups examined. The observed frequency of *ABCB1* genotypes varied across ethnic groups, consistent with previously published findings.^24^

### Table 1. Demographic summary of ABCB1 (rs1045642) and SLCO1B1 (rs4149056) genotyped population.

|   | ABCB1 (rs1045642) |  |  | N=458 | SLCO1B1 (rs4149056) |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- |
|   | CC (n=206) | CT (n=193) | TT (n=59) |   | TT (n=384) | CT (n=71) | CC (n=3) |
| Sex, n (%) |  |  |  |  |  |  |  |
| Male | 167 (81.1) | 150 (77.7) | 48 (81.4) | 365 (79.7) | 311 (81.0) | 53 (74.6) | 1 (33.3) |
| Female | 39 (18.9) | 43 (22.3) | 11 (18.6) | 93 (20.3) | 73 (19.0) | 18 (25.4) | 2 (66.7) |
|   |  |  |  |  |  |  |  |
| Race, n (%) |  |  |  |  |  |  |  |
| American Indian/Alaskan | 0 (0.0) | 2 (1.0) | 0 (0.0) | 2 (0.4) | 1 (0.3) | 1 (1.4) | 0 (0.0) |
| Asian | 1 (0.5) | 6 (3.1) | 3 (5.1) | 10 (2.2) | 9 (2.3) | 1 (1.4) | 0 (0.0) |
| Black or African American | 150 (72.8) | 91 (47.2) | 9 (15.3) | 250 (54.6) | 226 (58.9) | 24 (33.8) | 0 (0.0) |
| White | 43 (20.9) | 89 (46.1) | 45 (76.3) | 177 (38.6) | 134 (34.9) | 40 (56.3) | 3 (100.0) |
| Other | 12 (5.8) | 5 (2.6) | 2 (3.4) | 19 (4.1) | 14 (3.6) | 5 (7.0) | 0 (0.0) |
|   |  |  |  |  |  |  |  |
| Ethnicity, n (%) |  |  |  |  |  |  |  |
| Hispanic/Latino | 38 (18.4) | 51 (26.4) | 16 (27.1) | 105 (22.9) | 84 (21.9) | 20 (28.2) | 1 (33.3) |
| Not Hispanic/Latino | 168 (81.6) | 142 (73.6) | 43 (72.9) | 353 (77.1) | 300 (78.1) | 51 (71.8) | 2 (66.7) |
|   |  |  |  |  |  |  |  |
| Age (years) |  |  |  |  |  |  |  |
| Mean±s.d. | 32.1±7.4 | 30.2±6.9 | 30.9±8.5 | 31.1±7.4 | 31.4±7.4 | 29.6±6.9 | 29.3±12.1 |
|   |  |  |  |  |  |  |  |
| Weight (kg) |  |  |  |  |  |  |  |
| Mean±s.d. | 81.5±12.1 | 79.0±12.1 | 76.6±12.0 | 79.8±12.2 | 80.1±12.1 | 78.8±12.3 | 64.0±4.0 |
|   |  |  |  |  |  |  |  |
| BMI (kg m−2) |  |  |  |  |  |  |  |
| Mean±s.d. | 26.6±3.1 | 25.9±3.0 | 25.2±3.0 | 26.1±3.1 | 26.1±3.0 | 26.2±3.2 | 24.9±3.9 |
|   |  |  |  |  |  |  |  |
| CLcr (ml min−1) |  |  |  |  |  |  |  |
| Mean±s.d. | 134.0±26.7 | 137.0±25.2 | 138.0±25.3 | 136.0±25.9 | 134.5±26.3 | 141.0±22.7 | 146.7±25.2 |

Table 1 Caption: Abbreviations: BMI, body mass index; CLcr, creatinine clearance; SD, standard deviation.

Due to the infrequent occurrence of *SLCO1B1* CC homozygotes (0.66%) in the overall population, subjects with the CT or CC genotypes were combined together and analyzed as C-carriers. Thus, overall, 16.2% of subjects were C-carriers, and 83.8% of subjects were TT homozygotes. As previously reported in the literature, the frequency of *SLCO1B1* T521C variants differed between races.^25^

### Pharmacokinetic parameters

As shown in [Figure 1a](#fig1), *ABCB1* genotype did not significantly alter mean plasma edoxaban concentration–time curves. Edoxaban PK parameters including total plasma exposure (AUC_inf_), peak (*C*_max_) and trough plasma concentrations (*C*_24_) were not affected by *ABCB1* C3435T genotype ([Figures 1b–d](#fig1) and [Table 2](#tbl2)). In addition, plasma exposure from the time of dosing to the last measurable concentration (AUC_last_) and plasma exposure up to 24 h after dosing (AUC_0–24_) exhibited a similar range of values across genotypes ([Table 2](#tbl2)). The values of ratios of geometric LS means for AUC_inf_, *C*_max_ and *C*_24_ were 103.5 (90% CI, 98.2–109.1), 109.8 (90% CI, 100.3–120.2), and 93.4 (90% CI, 85.7–101.9), respectively, for a CT vs CC comparison, and 104.9 (90% CI, 96.8–113.6), 106.2 (90% CI, 92.6–121.8), and 92.3 (90% CI, 80.8–105.4), respectively, for a TT vs CC comparison. These data reveal that the *ABCB1* C3435T SNP does not significantly affect edoxaban exposure.

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d4b3/5817390/90a6a99de2c0/tpj201682f1.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5817390_tpj201682f1.jpg)

The effect of the ABCB1 rs1045642 variant on edoxaban pharmacokinetics. (a) Mean concentration–time profiles by genotype. Box and whisker plots demonstrating the effect of ABCB1 rs1045642 genotypes on (b) AUCinf, (c) Cmax and (d) C24. Inset figure in (a) shows profile of interval from initial dosing to 12 h postdose. Error bars in (a) represent the standard deviation. Whiskers in (b, c and d) indicate the 10th and 90th percentiles; individual dots represent points outside of the 10th and 90th percentiles. Median values are represented numerically and as lines in box plots.

### Table 2. Summary of pharmacokinetic parameters by ABCB1 rs1045642 genotype.

| Parameter | ABCB1 (rs1045642) genotypes |  |  |  |  |  |
| --- | --- | --- | --- | --- | --- | --- |
|   | CC (n=206) | n | CT (n=193) | n | TT (n=59) | n |
| AUClast (ng•h ml−1) | 1760±447.2 | 206 | 1814±417.0 | 193 | 1820±434.8 | 59 |
| AUC0–24 (ng•h ml−1) | 1608±441.2 | 206 | 1671±420.3 | 193 | 1680±441.5 | 59 |
| AUCinf (ng•h ml−1) | 1789±450.2 | 205 | 1845±417.8 | 190 | 1863±436.5 | 58 |
| Cmax (ng ml−1) | 246±96.6 | 206 | 268±95.2 | 193 | 261±101.3 | 59 |
| C24 (ng ml−1) | 12.5±5.11 | 205 | 11.8±5.07 | 193 | 12.3±6.46 | 58 |

Table 2 Caption: Abbreviations: AUC, area under the curve; AUC0–24, AUC from the time of dosing over a 0–24 h postdose time interval; C24, observed plasma concentration at 24 h postdose; AUCinf, AUC from the time of dosing to the last measurable concentration and extrapolated to infinity; AUClast, area under the plasma drug concentration–time curve from the time of dosing to the last measurable concentration; Cmax, maximum observed plasma concentration. Data presented as mean±s.d.

The mean plasma concentration vs time profiles of edoxaban were also comparable among subjects independent of *SLCO1B1* genotype ([Figure 2a](#fig2), left panel). Consistent with this observation, AUC_inf_, *C*_max_, and *C*_24_ of edoxaban were similar between C-carriers and TT homozygotes ([Figure 2b](#fig2) and [Table 3](#tbl3)). Furthermore, the range of values for AUC_last_ and AUC_0–24_ did not differ between C-carriers and TT homozygotes ([Table 3](#tbl3)). In comparing C-carriers vs TT homozygotes, the geometric LS means ratios for edoxaban AUC_inf_, *C*_max_ and *C*_24_ were 101.0 (90% CI, 95.9–106.3), 105.4 (90% CI, 96.5–115.1) and 95.3 (90% CI, 87.5–103.7), respectively.

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d4b3/5817390/7dd80315dae4/tpj201682f2.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5817390_tpj201682f2.jpg)

The effect of the SLCO1B1 rs4149056 variant on edoxaban and M4 pharmacokinetics. Mean concentration–time profiles by genotype of (a) edoxaban (left panel) and M4 (right panel). Box and whisker plots demonstrating the effect of SLCO1B1 rs4149056 genotypes on AUCinf, Cmax and C24 for (b) edoxaban and (c) M4. Inset figure in (a) shows profile of interval from initial dosing to 12 h postdose. Error bars in (a) represent the standard deviation. Whiskers in (b and c) indicate the 10th and 90th percentiles; individual dots represent points outside of the 10th and 90th percentiles. Median values are represented numerically and as lines in box plots.

### Table 3. Summary of pharmacokinetic parameters of edoxaban and M4 by SLCO1B1 rs4149056 genotype.

| SLCO1B1 (rs4149056) genotypes |  |  |  |  |
| --- | --- | --- | --- | --- |
| PK parameter | TT | n | C-carriers | n |
| Edoxaban |  |  |  |  |
| AUClast (ng • h ml−1) | 1792.9±442.8 | 384 | 1777.2±380.8 | 74 |
| AUC0–24 (ng • h ml−1) | 1644.1±440.6 | 384 | 1643.8±392.9 | 74 |
| AUCinf (ng • h ml−1) | 1823.5±444.9 | 381 | 1815.5±382.4 | 72 |
| Cmax (ng ml−1) | 255.1±97.4 | 384 | 268.0±94.7 | 74 |
| C24 (ng ml−1) | 12.3±5.4 | 382 | 11.6±4.8 | 74 |
|   |  |  |  |  |
| M4 |  |  |  |  |
| AUClast (ng • h ml−1) | 147.1±60.7 | 384 | 184.6±75.6 | 74 |
| AUC0–24 (ng • h ml−1) | 134.7±57.2 | 384 | 170.0±73.0 | 74 |
| AUCinf (ng • h ml−1) | 150.2±61.4 | 379 | 191.9±76.4 | 72 |
| Cmax (ng ml−1) | 21.2±10.7 | 384 | 26.7±13.6 | 74 |
| C24 (ng ml−1) | 1.1±0.57 | 382 | 1.3±0.74 | 74 |
| MPR AUClast | 8.7±2.9 | 384 | 10.9±3.6 | 74 |
| MPR AUCinf | 8.7±2.9 | 378 | 11.1±3.5 | 72 |
| MPR Cmax | 8.8±3.1 | 384 | 10.5±3.3 | 74 |

Table 3 Caption: Abbreviations: AUC0–24, AUC from the time of dosing over a 0–24 h postdose time interval; C24, observed plasma concentration at 24 h postdose; AUCinf, AUC from the time of dosing to the last measurable concentration and extrapolated to infinity; AUClast, area under the plasma drug concentration–time curve from the time of dosing to the last measurable concentration; Cmax, maximum observed plasma concentration; MPR, metabolite to parent ratio of relevant PK parameter. Data shown as mean±s.d.

Plasma M4 exposure was slightly higher among *SLCO1B1* C-carriers compared to TT homozygotes, as assessed by the mean plasma concentration vs time curves of M4 ([Figure 2a](#fig2), right panel). Accordingly, C-carriers exhibited modestly higher levels of median total exposure and peak plasma concentration ([Figure 2c](#fig2)). Median M4 trough levels were comparable between the 2 genotypes ([Figure 2c](#fig2)), while mean AUC_last_ and AUC_0–24_ values were greater among C-carriers than TT homozygotes ([Table 3](#tbl3)). The geometric LS means ratios comparing C-carriers with TT homozygotes for AUC_inf_, *C*_max_ and *C*_24_ were 126.9 (90% CI, 117.4–137.1), 124.1 (90% CI, 111.9–137.5) and 114.9 (90% CI, 104.0–127.0), respectively; the 90% CI of the ratios of geometric LS means of C-carriers vs TT homozygotes exceeded the predefined bioequivalence range (90% CI, 80–125%) for each of these PK measures. Taken together, these data confirm that while the *SLCO1B1* T521C SNP does not influence edoxaban exposure, M4 exposure is elevated among C-carriers.

Edoxaban and M4 have similar anticoagulant activities on a molar basis (data on file). MPRs for AUC_last_, AUC_inf_ and *C*_max_ of M4 to edoxaban ranged from 9% to 11%, indicating that M4 contributes a small amount to the overall anticoagulant effect ([Table 3](#tbl3) and [Figure 3](#fig3)). Together, these findings demonstrate that the *SLCO1B1* T521C polymorphism increases M4 exposure, but not in a clinically meaningful way, as M4 plasma concentration is approximately one-tenth that of edoxaban.

### Figure 3.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d4b3/5817390/844a2cb3eb63/tpj201682f3.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5817390_tpj201682f3.jpg)

Summary pharmacokinetics of edoxaban and M4 in plasma by SLCO1B1 rs4149056 genotype. Bar graphs represent mean values, and the error bars indicate standard deviation.

### Safety

The single 60-mg dose of edoxaban was generally well tolerated by the healthy subjects in these 14 phase 1 studies.^13, 14, 26, 27, 28, 29^

## Discussion

Common genetic variants and pharmacologic inhibition can result in altered transporter function, and thereby affect the PK of certain substrates. Allelic variants of *ABCB1*, including C3435T, which codes for a synonymous substitution in exon 26, are reported to alter the disposition of some P-gp substrates.^15, 24, 30, 31^ For example, some reports indicate that exposure to digoxin, a P-gp substrate, is altered upon the administration of the P-gp inhibitor quinidine^32^ and is affected by the *ABCB1* C3435T SNP.^15, 30, 31^ The *SLCO1B1* T521C SNP results in a valine-to-alanine substitution in the uptake transporter OATP1B1, leading to decreased membrane expression^33^ and transport activity of OATP1B1.^19, 34, 35^ Concentrations of the OATP1B1 substrate rosuvastatin increase upon treatment with the OATP1B1 inhibitor cyclosporine^36^ and are also higher among *SLCO1B1* T521C CC homozygotes than TT homozygotes.^18^

Edoxaban is a substrate of the P-gp, and exposure to edoxaban increases upon coadministration of the P-gp inhibitors ketoconazole, quinidine, erythromycin, verapamil or dronedarone in both healthy subjects and nonvalvular atrial fibrillation patients.^11, 12^ M4, an active metabolite of edoxaban, is a substrate of OATP1B1,^10^ and coadministration of edoxaban with the OATP1B1 inhibitor cyclosporine increases exposure to M4.^13^

The present integrated analysis pooled PK and genotype data from 14 completed single-dose, phase 1 studies involving 458 healthy subjects to examine the impact of polymorphisms in two important transporter genes on edoxaban exposure. Results from this analysis demonstrate for the first time that common genetic variants of *ABCB1* (C3435T) and *SLCO1B1* (T521C) had minimal impact on edoxaban exposure. Edoxaban concentration–time profiles were comparable regardless of *ABCB1* C3435T genotype. In addition, mean and median AUC_inf_, *C*_max_ and *C*_24_ of edoxaban did not vary between the three *ABCB1* genotypes. With regard to the *SLCO1B1* T521C genotype, edoxaban concentration–time profiles and median PK parameters including AUC_inf_, *C*_max_ and *C*_24_ were similar between TT homozygotes and C-carriers. This was expected as edoxaban is not known to be an OATP1B1 substrate. While a modest increase in M4 plasma exposure is observed among C-allele carriers, this association is unlikely to be clinically relevant given that M4, at concentrations reaching <10% of edoxaban levels, does not significantly contribute to the total anticoagulant activity. Therefore, while some pharmacologic inhibitors of P-gp and OATP1B1 may increase edoxaban and M4 exposure in a clinically relevant way,^11, 12, 13^ these two common polymorphisms do not.

As concurrent administration of P-gp or OATP1B1 inhibitors with edoxaban increases edoxaban or M4 exposure, respectively, the effects of functional polymorphisms of *ABCB1* and *SLCO1B1* on edoxaban exposure were tested in this study. Neither *ABCB1* C3435T nor *SLCO1B1* T521C genotype status affected the exposure of edoxaban among healthy subjects, suggesting that these are not clinically significant polymorphisms. As such, the data reported here suggest that prescribers can follow approved dosing recommendations^21^ for edoxaban without concern for the presence of these two genetic variants.

This study was limited to candidate SNPs and thus did not capture the effects of other known *ABCB1* and *SLCO1B1* polymorphisms or of haplotypes^24, 25^ on edoxaban and M4 PK. The polymorphisms chosen for this analysis are common, well characterized, and known to be functionally active^24, 25^ and thus were selected for initial pharmacogenomic signal detection. Because of the lack of an observed effect with these polymorphisms, additional SNPs and haplotypes within *ABCB1* and *SLCO1B1* were not characterized, as they are unlikely to have a substantial impact on edoxaban or M4 PK.

In pharmacogenomics, integrated multistudy analyses can be especially useful to answer important questions that otherwise cannot be resolved within small individual studies. Some phase 1 studies—which characterize the PK parameters of new drugs and investigate the effects of food and potential interacting drugs on those PK parameters—include pharmacogenomics sample collection, although their typically small sample size limits the investigation of pharmacogenomic effects. Although phases 2 and 3 studies are often sufficiently powered for pharmacogenomics analyses, these studies usually lack the comprehensive concentration–time data needed to explore potential pharmacogenomic effects on PK parameters. The current integrated analysis circumvents these limitations by pooling genetic and concentration–time data from multiple phase 1 studies. The resulting data set, containing genotypes, covariates, and full PK profiles, is large enough to conduct an adequately powered pharmacogenomic analysis of PK parameters. This approach has been successfully employed in previous studies to investigate the effects of *CYP2C19* polymorphisms on the outcomes of clopidogrel treatment^37^ and *UGT1A1* polymorphisms on dolutegravir PK.^38^

Variability can be introduced into pooled analyses when different laboratories are used to collect genotype or concentration–time data. For the present study, all genetic data were generated by a single genotyping laboratory and all concentration–time data were generated by a single bioanalysis laboratory. However, some limitations remain. First, SNPs in *CES1*, which is involved in the metabolism of edoxaban to M4, have previously been reported to affect the disposition of other drugs^39, 40^ but were not investigated in this study. In addition, while this analysis controlled for race, the effect of *ABCB1* C3435T and *SLCO1B1* T521C on edoxaban and M4 PK were not analyzed by race. The present study was adequately powered; however, its sample size limits further evaluation of the effect of race, additional polymorphisms, and haplotypes. Larger follow-up replication studies will be required to address the limitations of this analysis.

## Acknowledgments

This integrated pharmacogenomic analysis was sponsored by Daiichi Sankyo, Inc. Writing and editorial support was provided by Sameera Kongara, PhD, of AlphaBioCom, LLC, and funded by Daiichi Sankyo. Dr Brown and Dr Walker were employees of Daiichi Sankyo when this work was conducted.

**Author contributions**

AGV, KB, JL, MS and JRW designed the study. IR performed the research. AGV, JL, IR and JRW analyzed data. All authors were involved in the development of the manuscript and provided approval for submission.

## Footnotes

## Supplementary Material

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Eriksson BI, Quinlan DJ, Eikelboom JW. Novel oral factor Xa and thrombin inhibitors in the management of thromboembolism. Annu Rev Med 2011; 62: 41–57.  [DOI](https://doi.org/10.1146/annurev-med-062209-095159) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21226611/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Annu%20Rev%20Med&title=Novel%20oral%20factor%20Xa%20and%20thrombin%20inhibitors%20in%20the%20management%20of%20thromboembolism&volume=62&publication_year=2011&pages=41&pmid=21226611&doi=10.1146/annurev-med-062209-095159&)

2. Furugohri T, Isobe K, Honda Y, Kamisato-Matsumoto C, Sugiyama N, Nagahara T et al. DU-176b, a potent and orally active factor Xa inhibitor: in vitro and in vivo pharmacological profiles. J Thromb Haemost 2008; 6: 1542–1549.  [DOI](https://doi.org/10.1111/j.1538-7836.2008.03064.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18624979/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Thromb%20Haemost&title=DU-176b,%20a%20potent%20and%20orally%20active%20factor%20Xa%20inhibitor:%20in%20vitro%20and%20in%20vivo%20pharmacological%20profiles&volume=6&publication_year=2008&pages=1542&pmid=18624979&doi=10.1111/j.1538-7836.2008.03064.x&)

3. Daiichi Sankyo Receives First Market Approval in Japan for LIXIANA (Edoxaban), a Direct Oral Factor Xa Inhibitor, for the Prevention of Venous Thromboembolism after Major Orthopedic Surgery. Available at http://www.daiichisankyo.com/media_investors/media_relations/press_releases/detail/005806.html. Accessed on 3 December 2014.

4. Giugliano RP, Ruff CT, Braunwald E, Murphy SA, Wiviott SD, Halperin JL et al. Edoxaban versus warfarin in patients with atrial fibrillation. N Engl J Med 2013; 369: 2093–2104.  [DOI](https://doi.org/10.1056/NEJMoa1310907) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24251359/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Edoxaban%20versus%20warfarin%20in%20patients%20with%20atrial%20fibrillation&volume=369&publication_year=2013&pages=2093&pmid=24251359&doi=10.1056/NEJMoa1310907&)

5. Hokusai VTE Investigators Buller HR Decousus H Grosso MA Mercuri M Middeldorp S et al. Edoxaban versus warfarin for the treatment of symptomatic venous thromboembolism. N Engl J Med 2013; 369: 1406–1415.  [DOI](https://doi.org/10.1056/NEJMoa1306638) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23991658/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Edoxaban%20versus%20warfarin%20for%20the%20treatment%20of%20symptomatic%20venous%20thromboembolism&volume=369&publication_year=2013&pages=1406&pmid=23991658&doi=10.1056/NEJMoa1306638&)

6. Daiichi Sankyo Receives Approval for Additional Indications of LIXIANA (edoxaban) in Japan. Available at http://www.daiichisankyo.com/media_investors/media_relations/press_releases/detail/006189.html. Accessed on 3 December 2014.

7. U.S. FDA Approves Daiichi Sankyo’s Once-Daily SAVAYSA (edoxaban) Tablets for Reduction of Stroke Risk in Non-Valvular Atrial Fibrillation and for the Treatment of Venous Thromboembolism. Available at www.daiichisankyo.com/media_investors/media_relations/press_releases/detail/006236.html. Accessed on 14 January 2015.

8. LIXIANA (edoxaban) approved in Switzerland. Available at http://www.daiichi-sankyo.eu/media/european-news/news-detail/article/lixianaR-edoxaban-approval-in-switzerland.html. Accessed on 16 April 2015.

9. Daiichi Sankyo’s Once-Daily LIXIANA (edoxaban) Approved in the EU for Stroke Prevention in Nonvalvular Atrial Fibrillation and for the Treatment and Prevention of Recurrent DVT and PE. Available at http://www.daiichisankyo.com/media_investors/media_relations/press_releases/detail/006314.html. Accessed on 25 June 2015.

10. Mikkaichi T, Yoshigae Y, Masumoto H, Imaoka T, Rozehnal V, Fischer T et al. Edoxaban transport via P-glycoprotein is a key factor for the drug's disposition. Drug Metab Dispos 2014; 42: 520–528.  [DOI](https://doi.org/10.1124/dmd.113.054866) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24459178/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Edoxaban%20transport%20via%20P-glycoprotein%20is%20a%20key%20factor%20for%20the%20drug's%20disposition&volume=42&publication_year=2014&pages=520&pmid=24459178&doi=10.1124/dmd.113.054866&)

11. Mendell J, Zahir H, Matsushima N, Noveck R, Lee F, Chen S et al. Drug-drug interaction studies of cardiovascular drugs involving P-glycoprotein, an efflux transporter, on the pharmacokinetics of edoxaban, an oral factor Xa inhibitor. Am J Cardiovasc Drugs 2013; 13: 331–342.  [DOI](https://doi.org/10.1007/s40256-013-0029-0) | [PMC free article](/articles/PMC3781304/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23784266/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Cardiovasc%20Drugs&title=Drug-drug%20interaction%20studies%20of%20cardiovascular%20drugs%20involving%20P-glycoprotein,%20an%20efflux%20transporter,%20on%20the%20pharmacokinetics%20of%20edoxaban,%20an%20oral%20factor%20Xa%20inhibitor&volume=13&publication_year=2013&pages=331&pmid=23784266&doi=10.1007/s40256-013-0029-0&)

12. Salazar DE, Mendell J, Kastrissios H, Green M, Carrothers TJ, Song S et al. Modelling and simulation of edoxaban exposure and response relationships in patients with atrial fibrillation. Thromb Haemost 2012; 107: 925–936.  [DOI](https://doi.org/10.1160/TH11-08-0566) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22398655/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Thromb%20Haemost&title=Modelling%20and%20simulation%20of%20edoxaban%20exposure%20and%20response%20relationships%20in%20patients%20with%20atrial%20fibrillation&volume=107&publication_year=2012&pages=925&pmid=22398655&doi=10.1160/TH11-08-0566&)

13. Matsushima N, Zahir H, Mendell J, Chen S, He L, Parasrampuria D. Coadministration of Cyclosporine Increased Edoxaban Exposure in Healthy Adults. 2014 American College of Clinical Pharmacology annual meeting September 14–16 Atlanta, Georgia. Clin Pharm Drug Dev 2014; 3: 1–59.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharm%20Drug%20Dev&title=Coadministration%20of%20Cyclosporine%20Increased%20Edoxaban%20Exposure%20in%20Healthy%20Adults.%202014%20American%20College%20of%20Clinical%20Pharmacology%20annual%20meeting%20September%2014%E2%80%9316%20Atlanta,%20Georgia&volume=3&publication_year=2014&pages=1&)

14. Mendell J, Chen S, He L, Desai M, Parasramupria DA. The effect of rifampin on the pharmacokinetics of edoxaban in healthy adults. Clin Drug Invest 2015; 35: 447–453.  [DOI](https://doi.org/10.1007/s40261-015-0298-2) | [PMC free article](/articles/PMC4488474/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26068927/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Drug%20Invest&title=The%20effect%20of%20rifampin%20on%20the%20pharmacokinetics%20of%20edoxaban%20in%20healthy%20adults&volume=35&publication_year=2015&pages=447&pmid=26068927&doi=10.1007/s40261-015-0298-2&)

15. Hoffmeyer S, Burk O, von Richter O, Arnold HP, Brockmoller J, Johne A et al. Functional polymorphisms of the human multidrug-resistance gene: multiple sequence variations and correlation of one allele with P-glycoprotein expression and activity in vivo. Proc Natl Acad Sci USA 2000; 97: 3473–3478.  [DOI](https://doi.org/10.1073/pnas.050585397) | [PMC free article](/articles/PMC16264/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10716719/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20USA&title=Functional%20polymorphisms%20of%20the%20human%20multidrug-resistance%20gene:%20multiple%20sequence%20variations%20and%20correlation%20of%20one%20allele%20with%20P-glycoprotein%20expression%20and%20activity%20in%20vivo&volume=97&publication_year=2000&pages=3473&pmid=10716719&doi=10.1073/pnas.050585397&)

16. Bonhomme-Faivre L, Devocelle A, Saliba F, Chatled S, Maccario J, Farinotti R et al. MDR-1 C3435T polymorphism influences cyclosporine a dose requirement in liver-transplant recipients. Transplantation 2004; 78: 21–25.  [DOI](https://doi.org/10.1097/01.tp.0000130981.55654.78) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15257034/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Transplantation&title=MDR-1%20C3435T%20polymorphism%20influences%20cyclosporine%20a%20dose%20requirement%20in%20liver-transplant%20recipients&volume=78&publication_year=2004&pages=21&pmid=15257034&doi=10.1097/01.tp.0000130981.55654.78&)

17. Nishizato Y, Ieiri I, Suzuki H, Kimura M, Kawabata K, Hirota T et al. Polymorphisms of OATP-C (SLC21A6) and OAT3 (SLC22A8) genes: consequences for pravastatin pharmacokinetics. Clin Pharmacol Ther 2003; 73: 554–565.  [DOI](https://doi.org/10.1016/S0009-9236(03)00060-2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12811365/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Polymorphisms%20of%20OATP-C%20(SLC21A6)%20and%20OAT3%20(SLC22A8)%20genes:%20consequences%20for%20pravastatin%20pharmacokinetics&volume=73&publication_year=2003&pages=554&pmid=12811365&doi=10.1016/S0009-9236(03)00060-2&)

18. Pasanen MK, Fredrikson H, Neuvonen PJ, Niemi M. Different effects of SLCO1B1 polymorphism on the pharmacokinetics of atorvastatin and rosuvastatin. Clin Pharmacol Ther 2007; 82: 726–733.  [DOI](https://doi.org/10.1038/sj.clpt.6100220) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17473846/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Different%20effects%20of%20SLCO1B1%20polymorphism%20on%20the%20pharmacokinetics%20of%20atorvastatin%20and%20rosuvastatin&volume=82&publication_year=2007&pages=726&pmid=17473846&doi=10.1038/sj.clpt.6100220&)

19. Gong IY, Kim RB. Impact of genetic variation in OATP transporters to drug disposition and response. Drug Metab Pharmacokinet 2013; 28: 4–18.  [DOI](https://doi.org/10.2133/dmpk.dmpk-12-rv-099) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23047721/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Pharmacokinet&title=Impact%20of%20genetic%20variation%20in%20OATP%20transporters%20to%20drug%20disposition%20and%20response&volume=28&publication_year=2013&pages=4&pmid=23047721&doi=10.2133/dmpk.dmpk-12-rv-099&)

20. Bathala MS, Masumoto H, Oguma T, He L, Lowrie C, Mendell J. Pharmacokinetics biotransformation, and mass balance of edoxaban, a selective, direct factor Xa inhibitor, in humans. Drug Metab Dispos 2012; 40: 2250–2255.  [DOI](https://doi.org/10.1124/dmd.112.046888) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22936313/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Pharmacokinetics%20biotransformation,%20and%20mass%20balance%20of%20edoxaban,%20a%20selective,%20direct%20factor%20Xa%20inhibitor,%20in%20humans&volume=40&publication_year=2012&pages=2250&pmid=22936313&doi=10.1124/dmd.112.046888&)

21. SAVAYSA (edoxaban) Tablets for Oral Use. Full Prescribing Information. Parsippany, New Jersey. Daiichi Sankyo, Inc 2015.

22. Matsushima N, Lee F, Sato T, Weiss D, Mendell J. Bioavailability and safety of the factor Xa inhibitor edoxaban and the effects of quinidine in healthy subjects. Clin Pharm Drug Dev 2013; 2: 358–366.  [DOI](https://doi.org/10.1002/cpdd.53) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27121940/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharm%20Drug%20Dev&title=Bioavailability%20and%20safety%20of%20the%20factor%20Xa%20inhibitor%20edoxaban%20and%20the%20effects%20of%20quinidine%20in%20healthy%20subjects&volume=2&publication_year=2013&pages=358&pmid=27121940&doi=10.1002/cpdd.53&)

23. Parasrampuria DA, Truitt KE. Pharmacokinetics and pharmacodynamics of edoxaban, a non-vitamin K antagonist oral anticoagulant that inhibits clotting factor Xa. Clin Pharmacokinet 2015; 55: 641–655.  [DOI](https://doi.org/10.1007/s40262-015-0342-7) | [PMC free article](/articles/PMC4875962/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26620048/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacokinet&title=Pharmacokinetics%20and%20pharmacodynamics%20of%20edoxaban,%20a%20non-vitamin%20K%20antagonist%20oral%20anticoagulant%20that%20inhibits%20clotting%20factor%20Xa&volume=55&publication_year=2015&pages=641&pmid=26620048&doi=10.1007/s40262-015-0342-7&)

24. Ieiri I. Functional significance of genetic polymorphisms in P-glycoprotein (MDR1, ABCB1) and breast cancer resistance protein (BCRP, ABCG2). Drug Metab Pharmacokinet 2012; 27: 85–105.  [DOI](https://doi.org/10.2133/dmpk.dmpk-11-rv-098) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22123128/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Pharmacokinet&title=Functional%20significance%20of%20genetic%20polymorphisms%20in%20P-glycoprotein%20(MDR1,%20ABCB1)%20and%20breast%20cancer%20resistance%20protein%20(BCRP,%20ABCG2)&volume=27&publication_year=2012&pages=85&pmid=22123128&doi=10.2133/dmpk.dmpk-11-rv-098&)

25. Niemi M, Pasanen MK, Neuvonen PJ. Organic anion transporting polypeptide 1B1: a genetically polymorphic transporter of major importance for hepatic drug uptake. Pharmacol Rev 2011; 63: 157–181.  [DOI](https://doi.org/10.1124/pr.110.002857) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21245207/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacol%20Rev&title=Organic%20anion%20transporting%20polypeptide%201B1:%20a%20genetically%20polymorphic%20transporter%20of%20major%20importance%20for%20hepatic%20drug%20uptake&volume=63&publication_year=2011&pages=157&pmid=21245207&doi=10.1124/pr.110.002857&)

26. Mendell J, Lee F, Chen S, Worland V, Shi M, Samama MM. The effects of the antiplatelet agents, aspirin and naproxen, on pharmacokinetics and pharmacodynamics of the anticoagulant edoxaban, a direct factor Xa inhibitor. J Cardiovasc Pharmacol 2013; 62: 212–221.  [DOI](https://doi.org/10.1097/FJC.0b013e3182970991) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23615159/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Cardiovasc%20Pharmacol&title=The%20effects%20of%20the%20antiplatelet%20agents,%20aspirin%20and%20naproxen,%20on%20pharmacokinetics%20and%20pharmacodynamics%20of%20the%20anticoagulant%20edoxaban,%20a%20direct%20factor%20Xa%20inhibitor&volume=62&publication_year=2013&pages=212&pmid=23615159&doi=10.1097/FJC.0b013e3182970991&)

27. Zahir H, Matsushima N, Halim AB, He L, Zhang G, Lee F et al. Edoxaban administration following enoxaparin: a pharmacodynamic, pharmacokinetic, and tolerability assessment in human subjects. Thromb Haemost 2012; 108: 166–175.  [DOI](https://doi.org/10.1160/TH11-09-0676) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22628060/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Thromb%20Haemost&title=Edoxaban%20administration%20following%20enoxaparin:%20a%20pharmacodynamic,%20pharmacokinetic,%20and%20tolerability%20assessment%20in%20human%20subjects&volume=108&publication_year=2012&pages=166&pmid=22628060&doi=10.1160/TH11-09-0676&)

28. Brown KS, Wickremasingha P, Parasrampuria D, Dishy V, Kochan J, Weiss D et al. The impact of prothrombin complex concentrate on the anticoagulatory effects of the factor Xa inhibitor edoxaban. J Am Coll Cardiol 2014; 63 (12_S). .  [DOI](https://doi.org/10.1016/j.thromres.2015.07.012) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26298619/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Am%20Coll%20Cardiol&title=The%20impact%20of%20prothrombin%20complex%20concentrate%20on%20the%20anticoagulatory%20effects%20of%20the%20factor%20Xa%20inhibitor%20edoxaban&volume=63&publication_year=2014&pmid=26298619&doi=10.1016/j.thromres.2015.07.012&)

29. Zahir H, Mendell J, Jin J, He L, Parasrampuria D. The effect of ketoconazole on the pharmacokinetics and pharmacodynamics of edoxaban in healthy adults. 2014 American College of Clinical Pharmacology annual meeting September 14–16 Atlanta, Georgia. Clin Pharm Drug Dev 2014; 3: 1–59.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharm%20Drug%20Dev&title=The%20effect%20of%20ketoconazole%20on%20the%20pharmacokinetics%20and%20pharmacodynamics%20of%20edoxaban%20in%20healthy%20adults.%202014%20American%20College%20of%20Clinical%20Pharmacology%20annual%20meeting%20September%2014%E2%80%9316%20Atlanta,%20Georgia&volume=3&publication_year=2014&pages=1&)

30. Sakaeda T, Nakamura T, Horinouchi M, Kakumoto M, Ohmoto N, Sakai T et al. MDR1 genotype-related pharmacokinetics of digoxin after single oral administration in healthy Japanese subjects. Pharm Res 2001; 18: 1400–1404.  [DOI](https://doi.org/10.1023/a:1012244520615) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11697464/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharm%20Res&title=MDR1%20genotype-related%20pharmacokinetics%20of%20digoxin%20after%20single%20oral%20administration%20in%20healthy%20Japanese%20subjects&volume=18&publication_year=2001&pages=1400&pmid=11697464&doi=10.1023/a:1012244520615&)

31. Verstuyft C, Schwab M, Schaeffeler E, Kerb R, Brinkmann U, Jaillon P et al. Digoxin pharmacokinetics and MDR1 genetic polymorphisms. Eur J Clin Pharmacol 2003; 58: 809–812.  [DOI](https://doi.org/10.1007/s00228-003-0567-5) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12698307/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Clin%20Pharmacol&title=Digoxin%20pharmacokinetics%20and%20MDR1%20genetic%20polymorphisms&volume=58&publication_year=2003&pages=809&pmid=12698307&doi=10.1007/s00228-003-0567-5&)

32. Fenner KS, Troutman MD, Kempshall S, Cook JA, Ware JA, Smith DA et al. Drug-drug interactions mediated through P-glycoprotein: clinical relevance, in vitro and in vivo correlation using digoxin as a probe drug. Clin Pharmacol Ther 2009; 85: 173–181.  [DOI](https://doi.org/10.1038/clpt.2008.195) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18987624/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Drug-drug%20interactions%20mediated%20through%20P-glycoprotein:%20clinical%20relevance,%20in%20vitro%20and%20in%20vivo%20correlation%20using%20digoxin%20as%20a%20probe%20drug&volume=85&publication_year=2009&pages=173&pmid=18987624&doi=10.1038/clpt.2008.195&)

33. Tirona RG, Leake BF, Merino G, Kim RB. Polymorphisms in OATP-C: identification of multiple allelic variants associated with altered transport activity among European- and African-Americans. J Biol Chem 2001; 276: 35669–35675.  [DOI](https://doi.org/10.1074/jbc.M103792200) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11477075/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=Polymorphisms%20in%20OATP-C:%20identification%20of%20multiple%20allelic%20variants%20associated%20with%20altered%20transport%20activity%20among%20European-%20and%20African-Americans&volume=276&publication_year=2001&pages=35669&pmid=11477075&doi=10.1074/jbc.M103792200&)

34. Ho RH, Tirona RG, Leake BF, Glaeser H, Lee W, Lemke CJ et al. Drug and bile acid transporters in rosuvastatin hepatic uptake: function, expression, and pharmacogenetics. Gastroenterology 2006; 130: 1793–1806.  [DOI](https://doi.org/10.1053/j.gastro.2006.02.034) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16697742/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gastroenterology&title=Drug%20and%20bile%20acid%20transporters%20in%20rosuvastatin%20hepatic%20uptake:%20function,%20expression,%20and%20pharmacogenetics&volume=130&publication_year=2006&pages=1793&pmid=16697742&doi=10.1053/j.gastro.2006.02.034&)

35. Kameyama Y, Yamashita K, Kobayashi K, Hosokawa M, Chiba K. Functional characterization of SLCO1B1 (OATP-C) variants, SLCO1B1*5, SLCO1B1*15 and SLCO1B1*15+C1007G, by using transient expression systems of HeLa and HEK293 cells. Pharmacogenet Genomics 2005; 15: 513–522.  [DOI](https://doi.org/10.1097/01.fpc.0000170913.73780.5f) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15970799/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Functional%20characterization%20of%20SLCO1B1%20(OATP-C)%20variants,%20SLCO1B1*5,%20SLCO1B1*15%20and%20SLCO1B1*15+C1007G,%20by%20using%20transient%20expression%20systems%20of%20HeLa%20and%20HEK293%20cells&volume=15&publication_year=2005&pages=513&pmid=15970799&doi=10.1097/01.fpc.0000170913.73780.5f&)

36. Simonson SG, Raza A, Martin PD, Mitchell PD, Jarcho JA, Brown CD et al. Rosuvastatin pharmacokinetics in heart transplant recipients administered an antirejection regimen including cyclosporine. Clin Pharmacol Ther 2004; 76: 167–177.  [DOI](https://doi.org/10.1016/j.clpt.2004.03.010) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15289793/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Rosuvastatin%20pharmacokinetics%20in%20heart%20transplant%20recipients%20administered%20an%20antirejection%20regimen%20including%20cyclosporine&volume=76&publication_year=2004&pages=167&pmid=15289793&doi=10.1016/j.clpt.2004.03.010&)

37. Mega JL, Simon T, Collet JP, Anderson JL, Antman EM, Bliden K et al. Reduced-function CYP2C19 genotype and risk of adverse clinical outcomes among patients treated with clopidogrel predominantly for PCI: a meta-analysis. JAMA 2010; 304: 1821–1830.  [DOI](https://doi.org/10.1001/jama.2010.1543) | [PMC free article](/articles/PMC3048820/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20978260/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA&title=Reduced-function%20CYP2C19%20genotype%20and%20risk%20of%20adverse%20clinical%20outcomes%20among%20patients%20treated%20with%20clopidogrel%20predominantly%20for%20PCI:%20a%20meta-analysis&volume=304&publication_year=2010&pages=1821&pmid=20978260&doi=10.1001/jama.2010.1543&)

38. Chen SSt, Jean P, Borland J, Song I, Yeo AJ, Piscitelli S et al. Evaluation of the effect of UGT1A1 polymorphisms on dolutegravir pharmacokinetics. Pharmacogenomics 2014; 15: 9–16.  [DOI](https://doi.org/10.2217/pgs.13.190) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24329186/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Evaluation%20of%20the%20effect%20of%20UGT1A1%20polymorphisms%20on%20dolutegravir%20pharmacokinetics&volume=15&publication_year=2014&pages=9&pmid=24329186&doi=10.2217/pgs.13.190&)

39. Lewis JP, Horenstein RB, Ryan K, O'Connell JR, Gibson Q, Mitchell BD et al. The functional G143E variant of carboxylesterase 1 is associated with increased clopidogrel active metabolite levels and greater clopidogrel response. Pharmacogenet Genomics 2013; 23: 1–8.  [DOI](https://doi.org/10.1097/FPC.0b013e32835aa8a2) | [PMC free article](/articles/PMC3682407/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23111421/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=The%20functional%20G143E%20variant%20of%20carboxylesterase%201%20is%20associated%20with%20increased%20clopidogrel%20active%20metabolite%20levels%20and%20greater%20clopidogrel%20response&volume=23&publication_year=2013&pages=1&pmid=23111421&doi=10.1097/FPC.0b013e32835aa8a2&)

40. Pare G, Eriksson N, Lehr T, Connolly S, Eikelboom J, Ezekowitz MD et al. Genetic determinants of dabigatran plasma levels and their relation to bleeding. Circulation 2013; 127: 1404–1412.  [DOI](https://doi.org/10.1161/CIRCULATIONAHA.112.001233) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23467860/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circulation&title=Genetic%20determinants%20of%20dabigatran%20plasma%20levels%20and%20their%20relation%20to%20bleeding&volume=127&publication_year=2013&pages=1404&pmid=23467860&doi=10.1161/CIRCULATIONAHA.112.001233&)
