# Polymorphisms associated with everolimus pharmacokinetics, toxicity and survival in metastatic breast cancer

## Metadata
**Authors:** Tomas Pascual, María Apellániz-Ruiz, Cristina Pernaut, Cecilia Cueto-Felgueroso, Pablo Villalba, Carlos Álvarez, Luis Manso, Lucia Inglada-Pérez, Mercedes Robledo, Cristina Rodríguez-Antona, Eva Ciruelos
**Journal:** PLoS ONE
**Date:** 2017 Jul 20
**DOI:** [10.1371/journal.pone.0180192](https://doi.org/10.1371/journal.pone.0180192)
**PMID:** 28727815
**PMCID:** PMC5519037
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5519037/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC5519037/pdf/pone.0180192.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC5519037/pdf/pone.0180192.pdf)

## Abstract

**Purpose:** 
Metastatic breast cancer (MBC) progressing after endocrine therapy frequently activates PI3K/AKT/mTOR pathway. The BOLERO-2 trial showed that everolimus-exemestane achieves increased progression free survival (PFS) compared with exemestane. However, there is great inter-patient variability in toxicity and response to exemestane-everolimus treatment. The objective of this study was to perform an exploratory study analyzing the implication of single nucleotide polymorphisms (SNPs) on outcomes from this treatment through a pharmacogenetic analysis.

**Patients and methods:** 
Blood was collected from 90 postmenopausal women with hormone receptor-positive, HER2-negative MBC treated with exemestane-everolimus following progression after prior treatment with a non-steroidal aromatase inhibitor. Everolimus pharmacokinetics was measured in 37 patients. Twelve SNPs in genes involved in everolimus pharmacokinetics and pharmacodynamics were genotyped and associations assessed with drug plasma levels, clinically relevant toxicities (non-infectious pneumonitis, mucositis, hyperglycemia and hematological toxicities), dose reductions or treatment suspensions due to toxicity, progression free survival (PFS) and overall survival.

**Results:** 
We found that CYP3A4 rs35599367 variant (CYP3A4*22 allele) carriers had higher everolimus blood concentration compared to wild type patients (P = 0.019). ABCB1 rs1045642 was associated with risk of mucositis (P = 0.031), while PIK3R1 rs10515074 and RAPTOR rs9906827 were associated with hyperglycemia and non-infectious pneumonitis (P = 0.016 and 0.024, respectively). Furthermore, RAPTOR rs9906827 was associated with PFS (P = 0.006).

**Conclusions:** 
CYP3A4*22 allele influenced plasma concentration of everolimus and several SNPs in PI3K/AKT/mTOR pathway genes were associated with treatment toxicities and prognosis. These results require replication, but suggest that germline variation could influence everolimus outcomes in MBC.

### Purpose

Metastatic breast cancer (MBC) progressing after endocrine therapy frequently activates PI3K/AKT/mTOR pathway. The BOLERO-2 trial showed that everolimus-exemestane achieves increased progression free survival (PFS) compared with exemestane. However, there is great inter-patient variability in toxicity and response to exemestane-everolimus treatment. The objective of this study was to perform an exploratory study analyzing the implication of single nucleotide polymorphisms (SNPs) on outcomes from this treatment through a pharmacogenetic analysis.

### Patients and methods

Blood was collected from 90 postmenopausal women with hormone receptor-positive, HER2-negative MBC treated with exemestane-everolimus following progression after prior treatment with a non-steroidal aromatase inhibitor. Everolimus pharmacokinetics was measured in 37 patients. Twelve SNPs in genes involved in everolimus pharmacokinetics and pharmacodynamics were genotyped and associations assessed with drug plasma levels, clinically relevant toxicities (non-infectious pneumonitis, mucositis, hyperglycemia and hematological toxicities), dose reductions or treatment suspensions due to toxicity, progression free survival (PFS) and overall survival.

### Results

We found that *CYP3A4* rs35599367 variant (*CYP3A4*22* allele) carriers had higher everolimus blood concentration compared to wild type patients (P = 0.019). *ABCB1* rs1045642 was associated with risk of mucositis (P = 0.031), while *PIK3R1* rs10515074 and *RAPTOR* rs9906827 were associated with hyperglycemia and non-infectious pneumonitis (P = 0.016 and 0.024, respectively). Furthermore, *RAPTOR* rs9906827 was associated with PFS (P = 0.006).

### Conclusions

*CYP3A4*22* allele influenced plasma concentration of everolimus and several SNPs in PI3K/AKT/mTOR pathway genes were associated with treatment toxicities and prognosis. These results require replication, but suggest that germline variation could influence everolimus outcomes in MBC.

## Introduction

Breast cancer is a life-threatening disease and is the second leading cause of cancer death among women. It has been estimated that in 2017 there will be 255,180 newly diagnosed breast cancer cases in the US, and approximately 40,610 women will die from breast cancer[[1](#pone.0180192.ref001)]. Although metastatic breast cancer is diagnosed in only 5% of cases at presentation, nearly one third of breast cancer patients with non-metastatic tumors will eventually develop metastases[[2](#pone.0180192.ref002)].

Breast cancer is considered a genetically heterogeneous and biologically diverse disease. Endocrine therapies that target estrogen receptor (ER) signaling pathways for ER-positive disease and HER2-targeted therapies for HER2-positive disease play a critical role in the treatment of most patients with breast cancer. For patients with ER-positive advanced breast cancer, endocrine therapy is the recommended initial treatment. Since most patients eventually develop resistance to these therapies, the guidelines recommend another endocrine agent when initial therapy fails [[3](#pone.0180192.ref003)].

Recent advances in elucidating the molecular mechanisms of disease progression have identified the existence of adaptive ‘‘cross-talk” between the ER and various growth factor receptor and intracellular signaling pathways, allowing breast cancer cells to escape the inhibitory effects of endocrine therapy[[4](#pone.0180192.ref004)]. These findings provide clinical rationale for enhancing or extending endocrine sensitivity by combining endocrine therapy with a targeted agent against compensatory pathways. The PI3K/AKT/mTOR pathway is a crucial mediator of tumor progression [[5](#pone.0180192.ref005),[6](#pone.0180192.ref006),[7](#pone.0180192.ref007)]. As the PI3K/Akt/mTOR pathway is heavily deregulated in breast cancer [[8](#pone.0180192.ref008),[9](#pone.0180192.ref009)], inhibitors of mTOR are of interest as potential therapeutic agents for breast cancer patients, with everolimus and temsirolimus being the key drugs considered ([S1 Table](#pone.0180192.s001)).

Robust clinical evidence favoring the use of everolimus plus exemestane rather than exemestane alone was demonstrated by the BOLERO-2 trial, a phase 3 study in postmenopausal women with ER-positive, HER2-negative advanced breast cancer progressing or recurring during or following treatment with non-steroidal aromatase inhibitors (AI) ([S1 Table](#pone.0180192.s001)). In addition, PFS benefits were consistent across patient subgroups defined by age, race, presence of visceral metastases, and prior chemotherapy [[10](#pone.0180192.ref010),[11](#pone.0180192.ref011),[12](#pone.0180192.ref012)].

The greater PFS benefit associated with everolimus plus exemestane was accompanied by increased toxicity, including a higher incidence of mTOR-inhibitor class-effect adverse events (AEs) such as stomatitis, non-infectious pneumonitis (NIP) and hyperglycemia, as well as higher incidences of hematologic toxicity, asthenia, fatigue, and weight loss. Most of these adverse events were mild to moderate in severity, and were generally manageable by dose reduction or interruption. In the BALLET expanded-access study (CRAD001YIC04), the primary reason for discontinuation during the first 3 months of treatment was AEs; for 16% of patients, treatment was discontinued due to toxicity. The most frequent drug-related AEs of grade 3 or more were stomatitis (10%), hyperglycemia (4%), asthenia (6%) and NIP (2%)[[13](#pone.0180192.ref013)].

A retrospective exploratory analysis of tumor tissue was unable to identify any predictive biomarkers of everolimus efficacy in patient subgroups defined by each of the 4 most frequently altered genes/pathways, when assessed individually (PIK3CA, CCND1, TP53 and FGFR1). Patients with low levels of markers of chromosomal instability had better PFS [[14](#pone.0180192.ref014)]. No studies to date have evaluated the contribution of germline variants to everolimus treatment outcome in MBC, although some have assessed a limited number of polymorphisms in patients with other types of tumors [[15](#pone.0180192.ref015),[16](#pone.0180192.ref016)].

Thus, there is an urgent need to identify patients that will have early relapse or severe toxicities from everolimus-exemestane, leading to discontinuation of treatment. We carried out an exploratory study with the aiming of defining the impact of germline variation on outcomes following everolimus-exemestane treatment. For this purpose we recruited MBC patients treated with this therapy. We genotyped twelve SNPs in genes involved in everolimus metabolism and transport and genes in the PI3K/AKT/mTOR pathway, and evaluated their association with everolimus pharmacokinetics and toxicity and survival.

## Materials and methods

### Patients

All patients were treated at the 12 de Octubre University Hospital, the hospital institutional ethics committee approved the study in accordance with the principles of Good Clinical Practice, the Declaration of Helsinki, and other applicable local regulations. Written informed consent was obtained from all patients before enrollment.

All patients had locally advanced or MBC and were previously exposed to AI in either the neoadjuvant/adjuvant or palliative setting. Eligible patients were postmenopausal women with ER-positive, HER2–non-amplified, advanced breast cancer whose disease was refractory to previous letrozole, anastrozole or exemestane. Letrozole, anastrozole or exemestane did not have to be the most recent treatment before enrollment, but recurrence or progression during receipt of the most recent systemic therapy had to be documented. Patients who had previously received other anticancer endocrine treatments or prior chemotherapy regimens for advanced disease were included. Patients also had to have an Eastern Cooperative Oncology Group (ECOG) performance status of 2 or less and adequate organ and hematologic functions. Exclusion criteria included previous treatment with mTOR inhibitors.

All patients were treated on an outpatient basis until disease progression or dose-limiting toxicity occurred. All patients had treatment initiated with a daily oral dose of 5 or 10 mg of everolimus and 25 mg of exemestane. Two dose reductions were allowed for grade 3 or higher hematologic or non-hematologic toxicities related to everolimus, or grade 2 for NIP.

### DNA isolation, SNP selection and genotyping

Blood samples for DNA isolation were collected from patients in treatment with exemestane-everolimus in any time of the treatment. Genomic DNA was isolated from peripheral blood using the FlexiGene DNA Kit (Qiagen, Valencia, CA, USA). DNA concentration was quantified by PicoGreen (Invitrogen, Carlsbad, CA, USA).

Twelve SNPs located in genes involved in everolimus metabolism (*CYP3A4*, *CYP3A5* and *CYP2C8*)[[17](#pone.0180192.ref017)] or everolimus transport (*ABCB1*)[[17](#pone.0180192.ref017)] or in genes belonging to the PI3K/AKT/mTOR pathway (*FGFR4*, *PHLPP2*, *AKT2*, *PIK3R1*, *RAPTOR* and *AKT1)*[[18](#pone.0180192.ref018),[19](#pone.0180192.ref019),[20](#pone.0180192.ref020)]were selected for genotyping taking into consideration the allele frequency and evidence of functionality ([Table 1](#pone.0180192.t001)). The latter included published data showing altered activity and/or expression. Reported associations with survival for patients with other tumor types were also considered.

### Table 1. SNPs included in the study and their genotype frequencies.

| Gene | Gene category | SNP | Variant type | MAF | Genotype countsa |  | Reference for selection |
| --- | --- | --- | --- | --- | --- | --- | --- |
| CYP3A4 | Everolimus metabolizing enzyme | rs35599367 C>T | Intronic | 0.04 | C/C | 83 (93%) | [17], [21] |
| C/T | 6 (7%) |  |  |  |  |  |  |
| T/T | 0 (0%) |  |  |  |  |  |  |
| CYP3A5 | Everolimus metabolizing enzyme | rs776746 G>A | Intronic (splicing defect) | 0.07 | G/G | 76 (85%) | [17,22] |
| G/A | 13 (15%) |  |  |  |  |  |  |
| A/A | 0 (0%) |  |  |  |  |  |  |
| CYP2C8 | Everolimus metabolizing enzyme | rs11572080 G>A | Missense (R139K) | 0.12 | G/G | 64 (71%) | [17,23] |
| G/A | 23 (26%) |  |  |  |  |  |  |
| A/A | 3 (3%) |  |  |  |  |  |  |
| ABCB1 | Everolimus transporter | rs1045642 C>T | Synonymous (I1145I) | 0.41 | C/C | 20 (24%) | [17,24,25] |
| T/C | 48 (56%) |  |  |  |  |  |  |
| T/T | 17 (20%) |  |  |  |  |  |  |
| ABCB1 | Everolimus transporter | rs1128503 C>T | Synonymous (G412G) | 0.40 | C/C | 30 (34% | [17,25] |
| C/T | 39 (45%) |  |  |  |  |  |  |
| T/T | 18 (21%) |  |  |  |  |  |  |
| ABCB1 | Everolimus transporter | rs2032582 G>T | Missense (A893S) | 0.35 | G/G | 28 (32%) | [17,25] |
| G/T | 46 (53%) |  |  |  |  |  |  |
| T/T | 13 (15%) |  |  |  |  |  |  |
| FGFR4 | mTOR pathway | rs351855G>A | Missense (G388R) | 0.29 | G/G | 48 (55%) | [15] |
| G/A | 35 (40%) |  |  |  |  |  |  |
| A/A | 5 (6%) |  |  |  |  |  |  |
| PHLPP2 | mTOR pathway | rs61733127 T>C | Missense (L1016S) | 0.16 | T/T | 61 (69%) | [16] |
| T/C | 25 (28%) |  |  |  |  |  |  |
| C/C | 3 (3%) |  |  |  |  |  |  |
| AKT2 | mTOR pathway | rs3730050 G>A | Intronic | 0.27 | G/G | 37 (42%) | [26] |
| G/A | 43 (48%) |  |  |  |  |  |  |
| A/A | 9 (10%) |  |  |  |  |  |  |
| PIK3R1 | mTOR pathway | rs10515074 A>G | Intronic | 20 | A/A | 64 (71%) | [26] |
| A/G | 24 (27%) |  |  |  |  |  |  |
| G/G | 2 (2%) |  |  |  |  |  |  |
| RAPTOR | mTOR pathway | rs9906827 C>T | Intronic | 49 | C/C | 24 (27%) | [26] |
| C/T | 44 (50%) |  |  |  |  |  |  |
| T/T | 20 (23%) |  |  |  |  |  |  |
| AKT1 | mTOR pathway | rs2494732 A>G | Intronic | 43 | A/A | 30 (34%) | [27] |
| A/G | 46 (52%) |  |  |  |  |  |  |
| G/G | 13 (15%) |  |  |  |  |  |  |

Table 1 Caption: MAF: minor allele frequency in this study. aThe number of genotyped patients was 90, but some samples failed genotyping for individual SNPs.

Genotyping was carried out on 15 ng of genomic DNA using the KASPar Technology (KBioscience, UK) and including DNA samples with known genotypes and negative controls. The Sequence Detection System ABI PRISM® 7900HT (Applied Biosystems) was used for the detection of fluorescence and allele assignment. The allele frequencies of the SNPs were similar to those described for Caucasians in1000 Genomes Project and all SNPs had P-values >0.05 for Hardy–Weinberg equilibrium, except *ABCB1* rs2032582 with a minor deviation (P = 0.03). After reviewing the cluster plots, this SNP was included in the analysis.

### Pharmacokinetics

The pharmacokinetic (PK) profile of everolimus was studied by analyzing blood concentration levels in individual samples. Blood samples for pharmacokinetic studies were collected from patients at 1 time point, on day 14 of the first month of treatment. All samples were collected in ethylenediaminetetraacetic acid-containing tubes.

Everolimus concentrations were determined using a chemiluminescentmicroparticle immunoassay (CMIA) by cross-reaction with anti-Sirolimus antibodies, on the Architect i2000SR System (Abbott). Prior to the analysis, a manual pre-treatment step was performed in which the whole blood sample was extracted using a precipitation reagent, then heated (42°C, 10 minutes) and centrifuged (11800 rpm, 4 minutes). The clear supernatant was decanted into a transplant pretreatment tube and analyzed using an Architect i2000SR analyzer. The calibration range of the assay was 0.0–30.0 ng/mL with a sensitivity of 1 ng/mL. The samples that had a concentration greater than 30 ng/ mL were diluted with Calibrator A and retested.

### Outcomes

AEs were recorded from the patients’ medical records retrospectively. Toxicity event was defined as the termination, temporary interruption and/or dose reduction of everolimus. Adverse events were graded using the National Cancer Institute Common Terminology Criteria for Adverse Events, Version 4.03 (NCI CTCAE). The efficacy of everolimus treatment was defined as PFS defined as the time elapsed between treatment initiation by everolimus and tumor progression or death from any cause. Tumor response was determinated at 6 to 12 weeks using Response Evaluation Criteria in Solid Tumors (RECIST) (version 1.1) by each investigators. Patients without documented clinical o radiographic disease progression were censored on the date of the last follow-up. OS was defined as the time elapsed between treatment initiation by everolimus and death from any cause.

The following clinical data were collected from the date of cancer diagnosis to the end of the study: demographic characteristics; number of metastases and their localization; cancer treatment (adjuvant hormonotherapy or chemotherapy; number of hormonotherapy lines in metastatic situation and number of chemotherapy lines in metastatic situation prior to everolimus treatment; everolimus treatment, everolimus initiation, initial dose, date and reason for everolimus termination, temporary(s) interruption(S) and/or dose reduction(S); biological results (hemoglobin, platelets, white cells, neutrophils, albumin, glycaemia, transaminases); presence of AEs.

### Statistical analysis

Associations between SNPs and everolimus concentrations were assessed using Mann-Whitney-U test. Associations with a selection of clinically relevant toxicities (NIP, mucositis, hyperglycemia, leukopenia, lymphopenia and thrombopenia) were assessed using logistic regression. Cox regression was applied to study the associations between SNPs and time to treatment modifications (treatment dose reduction or treatment interruption due to toxicity), PFS and OS.

Factors associated with the outcome variable under study with a P value <0.1 in univariate analyses were included as covariates in multivariable analyses; when no factors reached this threshold, relevant clinical factors were included as covariates, as indicated in the text. An additive (per-allele) genetic model was assessed initially, and for those with P<0.1, alternative genetic models were explored. SPSS v.19 was used for all statistical analyses. P-values less than 0.05 were considered statistically significant.

## Results

### Patient characteristics

A total of 90 women were recruited from October 2011 through January 2015. Baseline characteristics are described in [Table 2](#pone.0180192.t002). For 11 patients the starting dose of everolimus was 5 mg. The median duration of treatment with everolimus was 204 days (range, 13–815 days). Adverse events were consistent with those previously described and are listed in [S2 Table](#pone.0180192.s002).

### Table 2. Baseline demographic and clinical characteristics.

| Characteristic | N (%)* |
| --- | --- |
| Median age (range), in years | 62 (37–84) |
| Visceral involvement | 55 (61) |
| Liver | 36 (40) |
| Bone | 38 (42) |
| >3 metastatic sites | 37 (41) |
| ECOG performance status |   |
| 0 | 71 (79) |
| 1 | 10 (12) |
| 2 | 9 (10) |
| Breast cancer IHC |   |
| Estrogen receptor positive | 90 (100) |
| Progesterone receptor positive | 75 (84) |
| Prior hormone therapy in metastatic setting | 79 (87) |
| Median number of lines of therapy (range) | 1.5 (0–4) |
| Anastrozole/letrozole | 63 (70) |
| Fulvestrant | 41 (46) |
| Exemestane | 19 (22) |
| Tamoxifen | 23 (26) |
| Prior chemotherapy in metastatic setting | 45 (50) |
| Median number of lines of therapy (range) | 2.4 (0–7) |
| Taxanes | 18 (20) |
| Antracyclines | 17 (19) |
| Capecitabine | 32 (36) |

Table 2 Caption: IHC, immunohistochemistry * Unless otherwise indicated

### Everolimus pharmacokinetics

Pharmacokinetic analysis was carried out using data from 37 patients who had PK data on day 14 of the first everolimus cycle. At this time point, the median concentration of everolimus was 30.5 ng/mL (SD = 27.5) with a minimum of 2.8 ng/mL and a maximum of 130.6 ng/mL. Of the 6 SNPs involved in everolimus metabolism (*CYP3A4*, *CYP3A5* and *CYP2C8* genes) and transport (*ABCB1*), only *CYP3A4* rs35599367 (*CYP3A4*22* allele) showed a statistically significant association with everolimus concentration (P = 0.019). *CYP3A4*22* variant carriers (n = 4) had 2.7-fold higher everolimus concentration compared to wild type patients (median of 69.1ng/mL versus 25.7 ng/mL, respectively; [Fig 1](#pone.0180192.g001)). For *CYP3A5*1* carriers (n = 4) we found no significantly differences.

### Fig 1. Box plot representing everolimus blood concentration by CYP3A4 rs35599367 (CYP3A4*22) genotype.

![Fig 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d2dd/5519037/e9fc3caeb21e/pone.0180192.g001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5519037_pone.0180192.g001.jpg)

“C/C” corresponds to CYP3A4*22 wild type patients (n = 33), and “C/G” to CYP3A4*22 heterozygous carriers (n = 4). Comparison between groups was performed using the Mann-Whitney-U test.

### Everolimus toxicities

Of the 90 patients recruited in the study, 69 (76%) experienced at least one adverse event related to everolimus, 43 patients (48%) required a dose reduction to 5 mg daily and 6 (7%) required a second dose reduction to 2.5 mg daily. Eight patients (9%) discontinued treatment due to stomatitis, NIP, fatigue or thrombocytopenia (three, three, three and one patient, respectively). Two patients (2%) died due to AEs, but only one event (NIP) was considered drug-related ([S2 Table](#pone.0180192.s002)).

The analysis of time to treatment modification due to toxicity showed a trend (HR = 0.58, 95%CI = 0.33–1.01, P = 0.056) for *FGFR4* rs351855 under an additive genetic model ([Table 3](#pone.0180192.t003)). The association under a dominant model was statistically significant for both univariate (HR = 0.50, 95%CI = 0.27–0.93; P = 0.028) and multivariable analyses, the latter correcting for age at diagnosis (HR = 0.52, 95%CI = 0.28–0.97, P = 0.040).

### Table 3. SNPs associated with toxicity.

| Toxicity | Gene | SNP ID | Genetic model | Univariate analysis |  |  | Multivariable analysisa |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
|   |   |   |   | OR | 95% CI | P value | OR | 95% CI | P value |
| Time to treatment modifications due to toxicity | FGFR4 | rs351855 G>A | Additive | 0.58 | 0.33–1.01 | 0.056 | 0.60 | 0.35–1.06 | 0.077 |
|   |   |   | Dominant | 0.50 | 0.27–0.93 | 0.028 | 0.52 | 0.28–0.97 | 0.040 |
| Leucopenia | PIK3R1 | rs10515074 A>G | Additive | 4.67 | 1.80–12.1 | 0.0015 | 5.03 | 1.89–13.4 | 0.0012 |
| Hyperglycemia | PIK3R1 | rs10515074 A>G | Additive | 0.39 | 0.16–0.95 | 0.037 | 0.24 | 0.07–0.76 | 0.016 |
| Pneumonitis | RAPTOR | rs9906827 C>T | Additive | 0.40 | 0.18–0.91 | 0.028 | 0.38 | 0.16–0.88 | 0.024 |
| Mucositis | ABCB1 | rs1045642 C>T | Additive | 2.11 | 1.02–4.37 | 0.043 | 2.27 | 1.08–4.77 | 0.031 |
| Lymphopenia | ABCB1 | rs2032582 G>T | Additive | 2.33 | 1.12–4.81 | 0.023 | 2.23 | 1.07–4.67 | 0.033 |

Table 3 Caption: aThe multivariable analysis performed for time to treatment modifications, mucositis, pneumonitis, hyperglycemia, leucopenia and lymphopenia included the following covariates: age, presence of visceral disease, previous pneumonitis events, diabetes mellitus status, number of previous chemotherapy lines and number of previous chemotherapy lines, respectively.

Associations between SNPs and clinically relevant everolimus toxicities were detected using logistic regression analysis ([Table 3](#pone.0180192.t003)). For mucositis, patients with the T-allele of *ABCB1* rs1045642 had higher risk of toxicity (OR = 2.30, 95%CI = 1.08–4.77, P = 0.031; multivariable analysis). The A-allele of *RAPTOR* rs9906827 was associated with lower risk of non-infectious pneumonitis (OR = 0.38, 95%CI = 0.16–0.88, P = 0.024; multivariable analysis). The minor allele of *PIK3R1* rs10515074 was associated with reduced risk of hyperglycemia (OR = 0.24, 95%CI = 0.07–0.76, P = 0.016), but increased risk of leucopenia (OR = 5.03, 95%CI = 1.89–13.35, P = 0.001); there was also a trend with lymphopenia (OR = 2.48, 95%CI = 0.92–6.69, P = 0.073), all in multivariable analysis. *ABCB1* rs2032582 was significantly associated with lymphopenia risk (OR = 2.23, 95%CI = 1.07–4.67, P = 0.033; multivariable analysis). No SNPs were associated with thrombocytopenia.

### Progression-free survival and overall survival in metastatic breast cancer

The minor allele of *RAPTOR* rs9906827 was associated with longer PFS in univariate Cox regression analysis both in an additive and dominant genetic model (HR = 0.65, 95%CI = 0.45–0.94, P = 0.023 and HR = 0.49, 95%CI = 0.29–0.82, P = 0.007, respectively; [Fig 2](#pone.0180192.g002)). After the inclusion of age, number of previous chemotherapy lines ([Fig 2](#pone.0180192.g002)), number of previous lines of hormone-therapy and presence of previous visceral disease, the association remained statistically significant with no substantial changes in estimated HRs or P-values. No SNPs were significantly associated with OS.

### Fig 2. Kaplan-Meier curve for progression free survival by RAPTOR rs9906827 genotype.

![Fig 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d2dd/5519037/a0b44623daf0/pone.0180192.g002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5519037_pone.0180192.g002.jpg)

P-value corresponds to Cox regression analysis under a dominant genetic model including the number of previous chemotherapy lines as covariate. HR, hazard ratio; CI, confidence interval.

## Discussion

The BOLERO-2 trial demonstrated a clinically meaningful improvement in PFS with everolimus plus exemestane therapy in patients with ER–positive/HER2-negative advanced breast cancer [[10](#pone.0180192.ref010),[11](#pone.0180192.ref011)].Despite these encouraging results, not all patients benefit from everolimus, and an optimal target patient population for this drug is yet to be defined. Moreover, the benefits of treatment with everolimus come with an increased incidence of mTOR inhibitor-associated toxicities. Thus, there is an urgent need for biomarkers that can predict response to everolimus and better define the ideal target patient group.

Germline variants have previously been shown to be accurate biomarkers of drug toxicities, and their stability over a subject´s life and easy assessment make them especially attractive. The aim of this study was to explore whether germline variants are implicated in everolimus toxicity and response to treatment. We performed a pharmacogenetic analysis using 90 MBC patients treated with exemestane-everolimus, including a subset in which their PK profile was determined. To the best of our knowledge, this is the first study investigating germline predictors of exemestane-everolimus outcome in MBC. The impact on everolimus activity of SNPs located in PI3K/AKT/mTOR pathway genes and the alteration of everolimus plasma concentration caused by the*CYP3A4*22*variant, point towards germline variation as a relevant factor influencing everolimus outcomes in MBC. These results warrant replication in prospective patient series.

Everolimus is used in transplant recipients, tuberous sclerosis complex and metastatic breast, kidney and neuroendocrine pancreas cancer. The transplantation literature includes numerous associations between polymorphisms in genes encoding drug metabolizing enzymes and drug transporters and the pharmacokinetics of immunosuppressive drugs (e.g. tacrolimus, cyclosporine, everolimus). Most of these studies have been retrospective in design and the most relevant associations correspond to variants in *CYP3A5* and *ABCB1*[[28](#pone.0180192.ref028),[29](#pone.0180192.ref029)], [[30](#pone.0180192.ref030),[31](#pone.0180192.ref031)], however, most studies of everolimus pharmacokinetics have given negative results[[32](#pone.0180192.ref032),[33](#pone.0180192.ref033),[34](#pone.0180192.ref034),[35](#pone.0180192.ref035)]. In this study, we found that *CYP3A4*22* resulted in significantly higher plasma levels of everolimus, consistent with the decreased activity caused by the *CYP3A4*22* allele [[21](#pone.0180192.ref021)], while no effect was detected for *CYP3A5*3* with the same number of carrier patients This suggests a greater influence of *CYP3A4*22* than *CYP3A5*3* in everolimus pharmacokinetics, at least in MBC patients. Furthermore, everolimus dosing in cancer is much higher than in transplant patients (i.e. 5–10 mg/day versus 2-5mg/day), thus, it is conceivable that in the cancer setting, *CYP3A4*22* might exert a greater and more clinical relevant effect on drug toxicity than in the transplant setting. In this study we did not find statistically significant differences in the toxicities evaluated for *CYP3A4*22* variant carriers, however, only 6 carriers were identified, giving low statistical power. Thus, further studies are needed to clarify the impact of *CYP3A4*22* on everolimus toxicity.

Class-effect toxicities during mTOR inhibitor therapy are well characterized. The most common adverse events observed in everolimus clinical trials include stomatitis (50%), rash (40%), immunosuppression (40%), NIP (15%) and hyperglycemia (15%) [[10](#pone.0180192.ref010),[36](#pone.0180192.ref036),[37](#pone.0180192.ref037),[38](#pone.0180192.ref038),[39](#pone.0180192.ref039)]. Treatment modifications guided by biomarkers predictive of toxicity could minimize severe toxicities and increase optimal outcomes. SNP rs10515074 in *PIK3R1*, a gene encoding the 85 kD regulatory subunit of phosphatidylinositol 3-kinase enzyme, which is an upstream member that triggers thePI3K/AKT/mTOR signaling pathway, was associated with hyperglycemia and leucopenia. This variant has previously been associated with survival in muscle invasive and metastatic bladder cancer patients [[40](#pone.0180192.ref040)]. Furthermore, RAPTOR (regulatory associated protein of mTOR) has a positive role in nutrient signaling and in the control of cell size[[41](#pone.0180192.ref041)], and its intronic variant rs9906827 has been found to be associated with survival in muscle invasive and metastatic bladder cancer patients [[40](#pone.0180192.ref040)]. In our study in breast cancer patients treated with exemestane and everolimus, a statistically significant association in the same direction (variant allele associated with better outcome) was observed for *RAPTOR* rs9906827 and PFS. Thus, our results suggests that constitutive variation in the PI3K/AKT/mTOR pathway could result in alteration in susceptibility to toxicities caused by drugs inhibiting this pathway and also exert an effect in tumor outcome during treatment. In addition, SNPs in *ABCB1* were associated with increased risk of mucositis (rs1045642) and lymphopenia (rs2032582). These associations remained significant after multivariable analysis, and warrant validation in an independent series.

Key limitations affecting our study were the limited sample size and the lack of replication in an independent prospective study. However, this is a first exploratory study aimed at generating hypothesis, and with the exception of *CYP3A4*22* allele, the SNPs associated with everolimus outcome have a relative high allele frequency, increasing the power of the study. Another limitation was that schedule and dose modifications were not dictated by protocol (they reflect real life drug management), and the timing for radiological assessments was determined by individual clinicians. Thus, courses of treatment were not standardized and outcomes were assessed with regard to clinical practice.

In conclusion, this is the first study exploring the impact of germline variation on exemestane-everolimus outcome in MBC. Our results provide evidence that the *CYP3A4*22*variant influences everolimus PK and suggest that polymorphisms in *ABCB1* and PI3K/AKT/mTOR pathway genes could influence everolimus toxicity and response in MBC. These results require replication in an independent prospective series. If confirmed, these genetic variants could be used to inform individualized metastatic breast cancer treatment.

## Supporting information

## Data Availability

All relevant data are within the paper and its Supporting Information files.

## Funding Statement

This work was supported by projects from Mutua Madrileña (project number 2012/0088, Principal Investigator: Dr E. Ciruelos) and the Spanish Ministry of Economy and Competiveness (grant number SAF2012-35779). The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Data Availability Statement

All relevant data are within the paper and its Supporting Information files.

### Supplementary Materials

### Data Availability Statement

All relevant data are within the paper and its Supporting Information files.

## References

1. Siegel RL, Miller KD, Jemal A. Cancer Statistics, 2017. CA Cancer J Clin. 2017;67(1):7–30. doi: 10.3322/caac.21387 .  [DOI](https://doi.org/10.3322/caac.21387) | [DOI](https://doi.org/10.3322/caac.21387) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28055103/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=CA%20Cancer%20J%20Clin&title=Cancer%20Statistics,%202017&author=RL%20Siegel&author=KD%20Miller&author=A%20Jemal&volume=67&issue=1&publication_year=2017&pages=7-30&pmid=28055103&doi=10.3322/caac.21387&)

2. Gonzalez-Angulo AM, Morales-Vasquez F, Hortobagyi GN. Overview of resistance to systemic therapy in patients with breast cancer. Adv Exp Med Biol. 2007;608:1–22. .  [DOI](https://doi.org/10.1007/978-0-387-74039-3_1) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17993229/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Adv%20Exp%20Med%20Biol&title=Overview%20of%20resistance%20to%20systemic%20therapy%20in%20patients%20with%20breast%20cancer&author=AM%20Gonzalez-Angulo&author=F%20Morales-Vasquez&author=GN%20Hortobagyi&volume=608&publication_year=2007&pages=1-22&pmid=17993229&doi=10.1007/978-0-387-74039-3_1&)

3. Cardoso F, Costa A, Senkus E, Aapro M, Andre F, Barrios CH, et al. 3rd ESO-ESMO International Consensus Guidelines for Advanced Breast Cancer (ABC 3). Ann Oncol. 2016. doi: 10.1093/annonc/mdw544 .  [DOI](https://doi.org/10.1093/annonc/mdw544) | [DOI](https://doi.org/10.1093/annonc/mdw544) | [PMC free article](/articles/PMC5378224/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28177437/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Oncol&title=3rd%20ESO-ESMO%20International%20Consensus%20Guidelines%20for%20Advanced%20Breast%20Cancer%20(ABC%203)&author=F%20Cardoso&author=A%20Costa&author=E%20Senkus&author=M%20Aapro&author=F%20Andre&publication_year=2016&pmid=28177437&doi=10.1093/annonc/mdw544&)

4. Tryfonidis K, Zardavas D, Katzenellenbogen BS, Piccart M. Endocrine treatment in breast cancer: Cure, resistance and beyond. Cancer Treat Rev. 2016;50:68–81. doi: 10.1016/j.ctrv.2016.08.008 .  [DOI](https://doi.org/10.1016/j.ctrv.2016.08.008) | [DOI](https://doi.org/10.1016/j.ctrv.2016.08.008) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27643748/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Treat%20Rev&title=Endocrine%20treatment%20in%20breast%20cancer:%20Cure,%20resistance%20and%20beyond&author=K%20Tryfonidis&author=D%20Zardavas&author=BS%20Katzenellenbogen&author=M%20Piccart&volume=50&publication_year=2016&pages=68-81&pmid=27643748&doi=10.1016/j.ctrv.2016.08.008&)

5. Huang J, Manning BD. The TSC1-TSC2 complex: a molecular switchboard controlling cell growth. Biochem J. 2008;412(2):179–90. doi: 10.1042/BJ20080281 ; PubMed Central PMCID: PMC2735030.  [DOI](https://doi.org/10.1042/BJ20080281) | [DOI](https://doi.org/10.1042/BJ20080281) | [PMC free article](/articles/PMC2735030/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18466115/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem%20J&title=The%20TSC1-TSC2%20complex:%20a%20molecular%20switchboard%20controlling%20cell%20growth&author=J%20Huang&author=BD%20Manning&volume=412&issue=2&publication_year=2008&pages=179-90&pmid=18466115&doi=10.1042/BJ20080281&)

6. Margariti N, Fox SB, Bottini A, Generali D. "Overcoming breast cancer drug resistance with mTOR inhibitors". Could it be a myth or a real possibility in the short-term future? Breast Cancer Res Treat. 2011;128(3):599–606. doi: 10.1007/s10549-010-0986-9 .  [DOI](https://doi.org/10.1007/s10549-010-0986-9) | [DOI](https://doi.org/10.1007/s10549-010-0986-9) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20945086/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Breast%20Cancer%20Res%20Treat&title=%22Overcoming%20breast%20cancer%20drug%20resistance%20with%20mTOR%20inhibitors%22.%20Could%20it%20be%20a%20myth%20or%20a%20real%20possibility%20in%20the%20short-term%20future?&author=N%20Margariti&author=SB%20Fox&author=A%20Bottini&author=D%20Generali&volume=128&issue=3&publication_year=2011&pages=599-606&pmid=20945086&doi=10.1007/s10549-010-0986-9&)

7. Villarreal-Garza C, Cortes J, Andre F, Verma S. mTOR inhibitors in the management of hormone receptor-positive breast cancer: the latest evidence and future directions. Ann Oncol. 2012;23(10):2526–35. doi: 10.1093/annonc/mds075 .  [DOI](https://doi.org/10.1093/annonc/mds075) | [DOI](https://doi.org/10.1093/annonc/mds075) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22553196/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Oncol&title=mTOR%20inhibitors%20in%20the%20management%20of%20hormone%20receptor-positive%20breast%20cancer:%20the%20latest%20evidence%20and%20future%20directions&author=C%20Villarreal-Garza&author=J%20Cortes&author=F%20Andre&author=S%20Verma&volume=23&issue=10&publication_year=2012&pages=2526-35&pmid=22553196&doi=10.1093/annonc/mds075&)

8. Beeram M, Tan QT, Tekmal RR, Russell D, Middleton A, DeGraffenried LA. Akt-induced endocrine therapy resistance is reversed by inhibition of mTOR signaling. Ann Oncol. 2007;18(8):1323–8. doi: 10.1093/annonc/mdm170 .  [DOI](https://doi.org/10.1093/annonc/mdm170) | [DOI](https://doi.org/10.1093/annonc/mdm170) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17693645/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Oncol&title=Akt-induced%20endocrine%20therapy%20resistance%20is%20reversed%20by%20inhibition%20of%20mTOR%20signaling&author=M%20Beeram&author=QT%20Tan&author=RR%20Tekmal&author=D%20Russell&author=A%20Middleton&volume=18&issue=8&publication_year=2007&pages=1323-8&pmid=17693645&doi=10.1093/annonc/mdm170&)

9. Miller TW, Hennessy BT, Gonzalez-Angulo AM, Fox EM, Mills GB, Chen H, et al. Hyperactivation of phosphatidylinositol-3 kinase promotes escape from hormone dependence in estrogen receptor-positive human breast cancer. J Clin Invest. 2010;120(7):2406–13. doi: 10.1172/JCI41680 ; PubMed Central PMCID: PMC2898598.  [DOI](https://doi.org/10.1172/JCI41680) | [DOI](https://doi.org/10.1172/JCI41680) | [PMC free article](/articles/PMC2898598/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20530877/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Invest&title=Hyperactivation%20of%20phosphatidylinositol-3%20kinase%20promotes%20escape%20from%20hormone%20dependence%20in%20estrogen%20receptor-positive%20human%20breast%20cancer&author=TW%20Miller&author=BT%20Hennessy&author=AM%20Gonzalez-Angulo&author=EM%20Fox&author=GB%20Mills&volume=120&issue=7&publication_year=2010&pages=2406-13&pmid=20530877&doi=10.1172/JCI41680&)

10. Baselga J, Campone M, Piccart M, Burris HA 3rd, Rugo HS, Sahmoud T, et al. Everolimus in postmenopausal hormone-receptor-positive advanced breast cancer. N Engl J Med. 2012;366(6):520–9. doi: 10.1056/NEJMoa1109653 .  [DOI](https://doi.org/10.1056/NEJMoa1109653) | [DOI](https://doi.org/10.1056/NEJMoa1109653) | [PMC free article](/articles/PMC5705195/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22149876/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Everolimus%20in%20postmenopausal%20hormone-receptor-positive%20advanced%20breast%20cancer&author=J%20Baselga&author=M%20Campone&author=M%20Piccart&author=HA%20Burris&author=HS%20Rugo&volume=366&issue=6&publication_year=2012&pages=520-9&pmid=22149876&doi=10.1056/NEJMoa1109653&)

11. Yardley DA, Noguchi S, Pritchard KI, Burris HA 3rd, Baselga J, Gnant M, et al. Everolimus plus exemestane in postmenopausal patients with HR(+) breast cancer: BOLERO-2 final progression-free survival analysis. Adv Ther. 2013;30(10):870–84. doi: 10.1007/s12325-013-0060-1 ; PubMed Central PMCID: PMC3898123.  [DOI](https://doi.org/10.1007/s12325-013-0060-1) | [DOI](https://doi.org/10.1007/s12325-013-0060-1) | [PMC free article](/articles/PMC3898123/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24158787/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Adv%20Ther&title=Everolimus%20plus%20exemestane%20in%20postmenopausal%20patients%20with%20HR(+)%20breast%20cancer:%20BOLERO-2%20final%20progression-free%20survival%20analysis&author=DA%20Yardley&author=S%20Noguchi&author=KI%20Pritchard&author=HA%20Burris&author=J%20Baselga&volume=30&issue=10&publication_year=2013&pages=870-84&pmid=24158787&doi=10.1007/s12325-013-0060-1&)

12. Beck JT, Hortobagyi GN, Campone M, Lebrun F, Deleu I, Rugo HS, et al. Everolimus plus exemestane as first-line therapy in HR(+), HER2(-) advanced breast cancer in BOLERO-2. Breast Cancer Res Treat. 2014;143(3):459–67. doi: 10.1007/s10549-013-2814-5 ; PubMed Central PMCID: PMC3907668.  [DOI](https://doi.org/10.1007/s10549-013-2814-5) | [DOI](https://doi.org/10.1007/s10549-013-2814-5) | [PMC free article](/articles/PMC3907668/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24362951/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Breast%20Cancer%20Res%20Treat&title=Everolimus%20plus%20exemestane%20as%20first-line%20therapy%20in%20HR(+),%20HER2(-)%20advanced%20breast%20cancer%20in%20BOLERO-2&author=JT%20Beck&author=GN%20Hortobagyi&author=M%20Campone&author=F%20Lebrun&author=I%20Deleu&volume=143&issue=3&publication_year=2014&pages=459-67&pmid=24362951&doi=10.1007/s10549-013-2814-5&)

13. Jerusalem G, Mariani G, Ciruelos EM, Martin M, Tjan-Heijnen VC, Neven P, et al. Safety of everolimus plus exemestane in patients with hormone-receptor-positive, HER2-negative locally advanced or metastatic breast cancer progressing on prior non-steroidal aromatase inhibitors: primary results of a phase IIIb, open-label, single-arm, expanded-access multicenter trial (BALLET). Ann Oncol. 2016;27(9):1719–25. doi: 10.1093/annonc/mdw249 .  [DOI](https://doi.org/10.1093/annonc/mdw249) | [DOI](https://doi.org/10.1093/annonc/mdw249) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27358383/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Oncol&title=Safety%20of%20everolimus%20plus%20exemestane%20in%20patients%20with%20hormone-receptor-positive,%20HER2-negative%20locally%20advanced%20or%20metastatic%20breast%20cancer%20progressing%20on%20prior%20non-steroidal%20aromatase%20inhibitors:%20primary%20results%20of%20a%20phase%20IIIb,%20open-label,%20single-arm,%20expanded-access%20multicenter%20trial%20(BALLET)&author=G%20Jerusalem&author=G%20Mariani&author=EM%20Ciruelos&author=M%20Martin&author=VC%20Tjan-Heijnen&volume=27&issue=9&publication_year=2016&pages=1719-25&pmid=27358383&doi=10.1093/annonc/mdw249&)

14. Hortobagyi GN, Chen D, Piccart M, Rugo HS, Burris HA 3rd, Pritchard KI, et al. Correlative Analysis of Genetic Alterations and Everolimus Benefit in Hormone Receptor-Positive, Human Epidermal Growth Factor Receptor 2-Negative Advanced Breast Cancer: Results From BOLERO-2. J Clin Oncol. 2015. doi: 10.1200/JCO.2014.60.1971 .  [DOI](https://doi.org/10.1200/JCO.2014.60.1971) | [DOI](https://doi.org/10.1200/JCO.2014.60.1971) | [PMC free article](/articles/PMC5070556/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26503204/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Correlative%20Analysis%20of%20Genetic%20Alterations%20and%20Everolimus%20Benefit%20in%20Hormone%20Receptor-Positive,%20Human%20Epidermal%20Growth%20Factor%20Receptor%202-Negative%20Advanced%20Breast%20Cancer:%20Results%20From%20BOLERO-2&author=GN%20Hortobagyi&author=D%20Chen&author=M%20Piccart&author=HS%20Rugo&author=HA%20Burris&publication_year=2015&pmid=26503204&doi=10.1200/JCO.2014.60.1971&)

15. Serra S, Zheng L, Hassan M, Phan AT, Woodhouse LJ, Yao JC, et al. The FGFR4-G388R single-nucleotide polymorphism alters pancreatic neuroendocrine tumor progression and response to mTOR inhibition therapy. Cancer Res. 2012;72(22):5683–91. doi: 10.1158/0008-5472.CAN-12-2102 .  [DOI](https://doi.org/10.1158/0008-5472.CAN-12-2102) | [DOI](https://doi.org/10.1158/0008-5472.CAN-12-2102) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22986737/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=The%20FGFR4-G388R%20single-nucleotide%20polymorphism%20alters%20pancreatic%20neuroendocrine%20tumor%20progression%20and%20response%20to%20mTOR%20inhibition%20therapy&author=S%20Serra&author=L%20Zheng&author=M%20Hassan&author=AT%20Phan&author=LJ%20Woodhouse&volume=72&issue=22&publication_year=2012&pages=5683-91&pmid=22986737&doi=10.1158/0008-5472.CAN-12-2102&)

16. Seth Albert Bellister YZ, Sceusi Eric, Ellis Lee M., Yao James C. Prediction of prognosis in patients treated with everolimus for extrapancreatic neuroendocrine tumors by a single nucleotide polymorphism in PHLPP2. J Clin Oncol 2013;31((suppl 4; abstr 163)):163.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Prediction%20of%20prognosis%20in%20patients%20treated%20with%20everolimus%20for%20extrapancreatic%20neuroendocrine%20tumors%20by%20a%20single%20nucleotide%20polymorphism%20in%20PHLPP2&author=YZ%20Seth%20Albert%20Bellister&author=Eric%20Sceusi&author=Lee%20M.%20Ellis&author=James%20C.%20Yao&volume=31&issue=suppl%204&publication_year=2013&pages=163&)

17. Kirchner GI, Meier-Wiedenbach I, Manns MP. Clinical pharmacokinetics of everolimus. Clin Pharmacokinet. 2004;43(2):83–95. doi: 10.2165/00003088-200443020-00002 .  [DOI](https://doi.org/10.2165/00003088-200443020-00002) | [DOI](https://doi.org/10.2165/00003088-200443020-00002) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14748618/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacokinet&title=Clinical%20pharmacokinetics%20of%20everolimus&author=GI%20Kirchner&author=I%20Meier-Wiedenbach&author=MP%20Manns&volume=43&issue=2&publication_year=2004&pages=83-95&pmid=14748618&doi=10.2165/00003088-200443020-00002&)

18. Vivanco I, Sawyers CL. The phosphatidylinositol 3-Kinase AKT pathway in human cancer. Nat Rev Cancer. 2002;2(7):489–501. doi: 10.1038/nrc839 .  [DOI](https://doi.org/10.1038/nrc839) | [DOI](https://doi.org/10.1038/nrc839) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12094235/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Cancer&title=The%20phosphatidylinositol%203-Kinase%20AKT%20pathway%20in%20human%20cancer&author=I%20Vivanco&author=CL%20Sawyers&volume=2&issue=7&publication_year=2002&pages=489-501&pmid=12094235&doi=10.1038/nrc839&)

19. Vicier C, Dieci MV, Arnedos M, Delaloge S, Viens P, Andre F. Clinical development of mTOR inhibitors in breast cancer. Breast Cancer Res. 2014;16(1):203 doi: 10.1186/bcr3618 ; PubMed Central PMCID: PMC3978635.  [DOI](https://doi.org/10.1186/bcr3618) | [DOI](https://doi.org/10.1186/bcr3618) | [PMC free article](/articles/PMC3978635/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25189767/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Breast%20Cancer%20Res&title=Clinical%20development%20of%20mTOR%20inhibitors%20in%20breast%20cancer&author=C%20Vicier&author=MV%20Dieci&author=M%20Arnedos&author=S%20Delaloge&author=P%20Viens&volume=16&issue=1&publication_year=2014&pages=203&pmid=25189767&doi=10.1186/bcr3618&)

20. Laplante M, Sabatini DM. mTOR signaling in growth control and disease. Cell. 2012;149(2):274–93. doi: 10.1016/j.cell.2012.03.017 ; PubMed Central PMCID: PMC3331679.  [DOI](https://doi.org/10.1016/j.cell.2012.03.017) | [DOI](https://doi.org/10.1016/j.cell.2012.03.017) | [PMC free article](/articles/PMC3331679/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22500797/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell&title=mTOR%20signaling%20in%20growth%20control%20and%20disease&author=M%20Laplante&author=DM%20Sabatini&volume=149&issue=2&publication_year=2012&pages=274-93&pmid=22500797&doi=10.1016/j.cell.2012.03.017&)

21. Wang D, Guo Y, Wrighton SA, Cooke GE, Sadee W. Intronic polymorphism in CYP3A4 affects hepatic expression and response to statin drugs. Pharmacogenomics J. 2011;11(4):274–86. doi: 10.1038/tpj.2010.28 ; PubMed Central PMCID: PMC3248744.  [DOI](https://doi.org/10.1038/tpj.2010.28) | [DOI](https://doi.org/10.1038/tpj.2010.28) | [PMC free article](/articles/PMC3248744/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20386561/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=Intronic%20polymorphism%20in%20CYP3A4%20affects%20hepatic%20expression%20and%20response%20to%20statin%20drugs&author=D%20Wang&author=Y%20Guo&author=SA%20Wrighton&author=GE%20Cooke&author=W%20Sadee&volume=11&issue=4&publication_year=2011&pages=274-86&pmid=20386561&doi=10.1038/tpj.2010.28&)

22. Kuehl P, Zhang J, Lin Y, Lamba J, Assem M, Schuetz J, et al. Sequence diversity in CYP3A promoters and characterization of the genetic basis of polymorphic CYP3A5 expression. Nat Genet. 2001;27(4):383–91. doi: 10.1038/86882 .  [DOI](https://doi.org/10.1038/86882) | [DOI](https://doi.org/10.1038/86882) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11279519/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Sequence%20diversity%20in%20CYP3A%20promoters%20and%20characterization%20of%20the%20genetic%20basis%20of%20polymorphic%20CYP3A5%20expression&author=P%20Kuehl&author=J%20Zhang&author=Y%20Lin&author=J%20Lamba&author=M%20Assem&volume=27&issue=4&publication_year=2001&pages=383-91&pmid=11279519&doi=10.1038/86882&)

23. Dai D, Zeldin DC, Blaisdell JA, Chanas B, Coulter SJ, Ghanayem BI, et al. Polymorphisms in human CYP2C8 decrease metabolism of the anticancer drug paclitaxel and arachidonic acid. Pharmacogenetics. 2001;11(7):597–607. .  [DOI](https://doi.org/10.1097/00008571-200110000-00006) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11668219/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=Polymorphisms%20in%20human%20CYP2C8%20decrease%20metabolism%20of%20the%20anticancer%20drug%20paclitaxel%20and%20arachidonic%20acid&author=D%20Dai&author=DC%20Zeldin&author=JA%20Blaisdell&author=B%20Chanas&author=SJ%20Coulter&volume=11&issue=7&publication_year=2001&pages=597-607&pmid=11668219&doi=10.1097/00008571-200110000-00006&)

24. Fung KL, Gottesman MM. A synonymous polymorphism in a common MDR1 (ABCB1) haplotype shapes protein function. Biochim Biophys Acta. 2009;1794(5):860–71. doi: 10.1016/j.bbapap.2009.02.014 ; PubMed Central PMCID: PMC2810319.  [DOI](https://doi.org/10.1016/j.bbapap.2009.02.014) | [DOI](https://doi.org/10.1016/j.bbapap.2009.02.014) | [PMC free article](/articles/PMC2810319/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19285158/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochim%20Biophys%20Acta&title=A%20synonymous%20polymorphism%20in%20a%20common%20MDR1%20(ABCB1)%20haplotype%20shapes%20protein%20function&author=KL%20Fung&author=MM%20Gottesman&volume=1794&issue=5&publication_year=2009&pages=860-71&pmid=19285158&doi=10.1016/j.bbapap.2009.02.014&)

25. Kimchi-Sarfaty C, Oh JM, Kim IW, Sauna ZE, Calcagno AM, Ambudkar SV, et al. A "silent" polymorphism in the MDR1 gene changes substrate specificity. Science. 2007;315(5811):525–8. doi: 10.1126/science.1135308 .  [DOI](https://doi.org/10.1126/science.1135308) | [DOI](https://doi.org/10.1126/science.1135308) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17185560/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=A%20%22silent%22%20polymorphism%20in%20the%20MDR1%20gene%20changes%20substrate%20specificity&author=C%20Kimchi-Sarfaty&author=JM%20Oh&author=IW%20Kim&author=ZE%20Sauna&author=AM%20Calcagno&volume=315&issue=5811&publication_year=2007&pages=525-8&pmid=17185560&doi=10.1126/science.1135308&)

26. Chen QR, Song YK, Yu LR, Wei JS, Chung JY, Hewitt SM, et al. Global genomic and proteomic analysis identifies biological pathways related to high-risk neuroblastoma. J Proteome Res. 2010;9(1):373–82. Epub 2009/11/20. doi: 10.1021/pr900701v ; PubMed Central PMCID: PMC2801773.  [DOI](https://doi.org/10.1021/pr900701v) | [DOI](https://doi.org/10.1021/pr900701v) | [PMC free article](/articles/PMC2801773/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19921788/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Proteome%20Res&title=Global%20genomic%20and%20proteomic%20analysis%20identifies%20biological%20pathways%20related%20to%20high-risk%20neuroblastoma&author=QR%20Chen&author=YK%20Song&author=LR%20Yu&author=JS%20Wei&author=JY%20Chung&volume=9&issue=1&publication_year=2010&pages=373-82&pmid=19921788&doi=10.1021/pr900701v&)

27. Li Q, Yang J, Yu Q, Wu H, Liu B, Xiong H, et al. Associations between single-nucleotide polymorphisms in the PI3K-PTEN-AKT-mTOR pathway and increased risk of brain metastasis in patients with non-small cell lung cancer. Clin Cancer Res. 2013;19(22):6252–60. doi: 10.1158/1078-0432.CCR-13-1093 .  [DOI](https://doi.org/10.1158/1078-0432.CCR-13-1093) | [DOI](https://doi.org/10.1158/1078-0432.CCR-13-1093) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24077347/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&title=Associations%20between%20single-nucleotide%20polymorphisms%20in%20the%20PI3K-PTEN-AKT-mTOR%20pathway%20and%20increased%20risk%20of%20brain%20metastasis%20in%20patients%20with%20non-small%20cell%20lung%20cancer&author=Q%20Li&author=J%20Yang&author=Q%20Yu&author=H%20Wu&author=B%20Liu&volume=19&issue=22&publication_year=2013&pages=6252-60&pmid=24077347&doi=10.1158/1078-0432.CCR-13-1093&)

28. Staatz CE, Goodman LK, Tett SE. Effect of CYP3A and ABCB1 single nucleotide polymorphisms on the pharmacokinetics and pharmacodynamics of calcineurin inhibitors: Part II. Clin Pharmacokinet. 2010;49(4):207–21. doi: 10.2165/11317550-000000000-00000 .  [DOI](https://doi.org/10.2165/11317550-000000000-00000) | [DOI](https://doi.org/10.2165/11317550-000000000-00000) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20214406/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacokinet&title=Effect%20of%20CYP3A%20and%20ABCB1%20single%20nucleotide%20polymorphisms%20on%20the%20pharmacokinetics%20and%20pharmacodynamics%20of%20calcineurin%20inhibitors:%20Part%20II&author=CE%20Staatz&author=LK%20Goodman&author=SE%20Tett&volume=49&issue=4&publication_year=2010&pages=207-21&pmid=20214406&doi=10.2165/11317550-000000000-00000&)

29. Hronova K, Sima M, Svetlik S, Matouskova O, Slanar O. Pharmacogenetics and immunosuppressive drugs. Expert Rev Clin Pharmacol. 2014;7(6):821–35. doi: 10.1586/17512433.2014.966811 .  [DOI](https://doi.org/10.1586/17512433.2014.966811) | [DOI](https://doi.org/10.1586/17512433.2014.966811) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25301406/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Expert%20Rev%20Clin%20Pharmacol&title=Pharmacogenetics%20and%20immunosuppressive%20drugs&author=K%20Hronova&author=M%20Sima&author=S%20Svetlik&author=O%20Matouskova&author=O%20Slanar&volume=7&issue=6&publication_year=2014&pages=821-35&pmid=25301406&doi=10.1586/17512433.2014.966811&)

30. Zununi Vahed S, Ardalan M, Samadi N, Omidi Y. Pharmacogenetics and drug-induced nephrotoxicity in renal transplant recipients. Bioimpacts. 2015;5(1):45–54. doi: 10.15171/bi.2015.12 ; PubMed Central PMCID: PMCPMC4401167.  [DOI](https://doi.org/10.15171/bi.2015.12) | [DOI](https://doi.org/10.15171/bi.2015.12) | [PMC free article](/articles/PMC4401167/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25901296/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioimpacts&title=Pharmacogenetics%20and%20drug-induced%20nephrotoxicity%20in%20renal%20transplant%20recipients&author=Vahed%20S%20Zununi&author=M%20Ardalan&author=N%20Samadi&author=Y%20Omidi&volume=5&issue=1&publication_year=2015&pages=45-54&pmid=25901296&doi=10.15171/bi.2015.12&)

31. Knops N, Levtchenko E, van den Heuvel B, Kuypers D. From gut to kidney: transporting and metabolizing calcineurin-inhibitors in solid organ transplantation. Int J Pharm. 2013;452(1–2):14–35. doi: 10.1016/j.ijpharm.2013.05.033 .  [DOI](https://doi.org/10.1016/j.ijpharm.2013.05.033) | [DOI](https://doi.org/10.1016/j.ijpharm.2013.05.033) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23711732/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Pharm&title=From%20gut%20to%20kidney:%20transporting%20and%20metabolizing%20calcineurin-inhibitors%20in%20solid%20organ%20transplantation&author=N%20Knops&author=E%20Levtchenko&author=B%20van%20den%20Heuvel&author=D%20Kuypers&volume=452&issue=1%E2%80%932&publication_year=2013&pages=14-35&pmid=23711732&doi=10.1016/j.ijpharm.2013.05.033&)

32. Lemaitre F, Bezian E, Goldwirt L, Fernandez C, Farinotti R, Varnous S, et al. Population pharmacokinetics of everolimus in cardiac recipients: comedications, ABCB1, and CYP3A5 polymorphisms. Ther Drug Monit. 2012;34(6):686–94. doi: 10.1097/FTD.0b013e318273c899 .  [DOI](https://doi.org/10.1097/FTD.0b013e318273c899) | [DOI](https://doi.org/10.1097/FTD.0b013e318273c899) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23131698/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ther%20Drug%20Monit&title=Population%20pharmacokinetics%20of%20everolimus%20in%20cardiac%20recipients:%20comedications,%20ABCB1,%20and%20CYP3A5%20polymorphisms&author=F%20Lemaitre&author=E%20Bezian&author=L%20Goldwirt&author=C%20Fernandez&author=R%20Farinotti&volume=34&issue=6&publication_year=2012&pages=686-94&pmid=23131698&doi=10.1097/FTD.0b013e318273c899&)

33. Kniepeiss D, Renner W, Trummer O, Wagner D, Wasler A, Khoschsorur GA, et al. The role of CYP3A5 genotypes in dose requirements of tacrolimus and everolimus after heart transplantation. Clin Transplant. 2011;25(1):146–50. doi: 10.1111/j.1399-0012.2009.01198.x .  [DOI](https://doi.org/10.1111/j.1399-0012.2009.01198.x) | [DOI](https://doi.org/10.1111/j.1399-0012.2009.01198.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20041908/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Transplant&title=The%20role%20of%20CYP3A5%20genotypes%20in%20dose%20requirements%20of%20tacrolimus%20and%20everolimus%20after%20heart%20transplantation&author=D%20Kniepeiss&author=W%20Renner&author=O%20Trummer&author=D%20Wagner&author=A%20Wasler&volume=25&issue=1&publication_year=2011&pages=146-50&pmid=20041908&doi=10.1111/j.1399-0012.2009.01198.x&)

34. Moes DJ, Swen JJ, den Hartigh J, van der Straaten T, van der Heide JJ, Sanders JS, et al. Effect of CYP3A4*22, CYP3A5*3, and CYP3A Combined Genotypes on Cyclosporine, Everolimus, and Tacrolimus Pharmacokinetics in Renal Transplantation. CPT Pharmacometrics Syst Pharmacol. 2014;3:e100 doi: 10.1038/psp.2013.78 ; PubMed Central PMCID: PMCPMC3944116.  [DOI](https://doi.org/10.1038/psp.2013.78) | [DOI](https://doi.org/10.1038/psp.2013.78) | [PMC free article](/articles/PMC3944116/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24522145/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=CPT%20Pharmacometrics%20Syst%20Pharmacol&title=Effect%20of%20CYP3A4*22,%20CYP3A5*3,%20and%20CYP3A%20Combined%20Genotypes%20on%20Cyclosporine,%20Everolimus,%20and%20Tacrolimus%20Pharmacokinetics%20in%20Renal%20Transplantation&author=DJ%20Moes&author=JJ%20Swen&author=J%20den%20Hartigh&author=T%20van%20der%20Straaten&author=JJ%20van%20der%20Heide&volume=3&publication_year=2014&pages=e100&pmid=24522145&doi=10.1038/psp.2013.78&)

35. Schoeppler KE, Aquilante CL, Kiser TH, Fish DN, Zamora MR. The impact of genetic polymorphisms, diltiazem, and demographic variables on everolimus trough concentrations in lung transplant recipients. Clin Transplant. 2014;28(5):590–7. doi: 10.1111/ctr.12350 .  [DOI](https://doi.org/10.1111/ctr.12350) | [DOI](https://doi.org/10.1111/ctr.12350) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24628014/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Transplant&title=The%20impact%20of%20genetic%20polymorphisms,%20diltiazem,%20and%20demographic%20variables%20on%20everolimus%20trough%20concentrations%20in%20lung%20transplant%20recipients&author=KE%20Schoeppler&author=CL%20Aquilante&author=TH%20Kiser&author=DN%20Fish&author=MR%20Zamora&volume=28&issue=5&publication_year=2014&pages=590-7&pmid=24628014&doi=10.1111/ctr.12350&)

36. Jerusalem GM Gabriella; Ciruelos Eva M; Martin Miguel; Tjan-Heijnen Vivianne CG; Neven Patrick; Gavila Gregori Joaquin; Michelotti Andrea; Montemurro Filippo; Lang Istvan; Mardiak Josef; Naume Bjoem; Camozzi Maura; Lorizzo Katia; Brenski Dariusz; Conte Pierfranco. Abstract P5-19-02: Everolimus in combination with exemestane in hormone receptor-positive locally advanced or metastatic breast cancer (BC) patients progressing on prior non-steroidal AI (NSAIs): Ballet study (CRAD001YIC04). Cancer Research. 2014;75(9 Supplement):P5-19-02. doi: 10.1158/1538-7445.SABCS14-P5-19-02  [DOI](https://doi.org/10.1158/1538-7445.SABCS14-P5-19-02) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Research&title=Abstract%20P5-19-02:%20Everolimus%20in%20combination%20with%20exemestane%20in%20hormone%20receptor-positive%20locally%20advanced%20or%20metastatic%20breast%20cancer%20(BC)%20patients%20progressing%20on%20prior%20non-steroidal%20AI%20(NSAIs):%20Ballet%20study%20(CRAD001YIC04)&author=Gabriella%20Jerusalem%20GM&author=Eva%20M%20Ciruelos&author=Miguel%20Martin&author=Vivianne%20CG%20Tjan-Heijnen&author=Patrick%20Neven&volume=75&issue=9%20Supplement&publication_year=2014&)

37. Ciruelos E, Cortes-Funes H, Ghanem I, Manso L, Arteaga C. Role of inhibitors of mammalian target of rapamycin in the treatment of luminal breast cancer. Anticancer Drugs. 2013;24(8):769–80. doi: 10.1097/CAD.0b013e328363adc5 .  [DOI](https://doi.org/10.1097/CAD.0b013e328363adc5) | [DOI](https://doi.org/10.1097/CAD.0b013e328363adc5) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23838677/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Anticancer%20Drugs&title=Role%20of%20inhibitors%20of%20mammalian%20target%20of%20rapamycin%20in%20the%20treatment%20of%20luminal%20breast%20cancer&author=E%20Ciruelos&author=H%20Cortes-Funes&author=I%20Ghanem&author=L%20Manso&author=C%20Arteaga&volume=24&issue=8&publication_year=2013&pages=769-80&pmid=23838677&doi=10.1097/CAD.0b013e328363adc5&)

38. Yao JC, Shah MH, Ito T, Bohas CL, Wolin EM, Van Cutsem E, et al. Everolimus for advanced pancreatic neuroendocrine tumors. N Engl J Med. 2011;364(6):514–23. doi: 10.1056/NEJMoa1009290 ; PubMed Central PMCID: PMCPMC4208619.  [DOI](https://doi.org/10.1056/NEJMoa1009290) | [DOI](https://doi.org/10.1056/NEJMoa1009290) | [PMC free article](/articles/PMC4208619/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21306238/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Everolimus%20for%20advanced%20pancreatic%20neuroendocrine%20tumors&author=JC%20Yao&author=MH%20Shah&author=T%20Ito&author=CL%20Bohas&author=EM%20Wolin&volume=364&issue=6&publication_year=2011&pages=514-23&pmid=21306238&doi=10.1056/NEJMoa1009290&)

39. Motzer RJ, Escudier B, Oudard S, Hutson TE, Porta C, Bracarda S, et al. Phase 3 trial of everolimus for metastatic renal cell carcinoma: final results and analysis of prognostic factors. Cancer. 2010;116(18):4256–65. doi: 10.1002/cncr.25219 .  [DOI](https://doi.org/10.1002/cncr.25219) | [DOI](https://doi.org/10.1002/cncr.25219) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20549832/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer&title=Phase%203%20trial%20of%20everolimus%20for%20metastatic%20renal%20cell%20carcinoma:%20final%20results%20and%20analysis%20of%20prognostic%20factors&author=RJ%20Motzer&author=B%20Escudier&author=S%20Oudard&author=TE%20Hutson&author=C%20Porta&volume=116&issue=18&publication_year=2010&pages=4256-65&pmid=20549832&doi=10.1002/cncr.25219&)

40. Chen M, Gu J, Delclos GL, Killary AM, Fan Z, Hildebrandt MA, et al. Genetic variations of the PI3K-AKT-mTOR pathway and clinical outcome in muscle invasive and metastatic bladder cancer patients. Carcinogenesis. 2010;31(8):1387–91. doi: 10.1093/carcin/bgq110 ; PubMed Central PMCID: PMC2915631.  [DOI](https://doi.org/10.1093/carcin/bgq110) | [DOI](https://doi.org/10.1093/carcin/bgq110) | [PMC free article](/articles/PMC2915631/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20530239/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Carcinogenesis&title=Genetic%20variations%20of%20the%20PI3K-AKT-mTOR%20pathway%20and%20clinical%20outcome%20in%20muscle%20invasive%20and%20metastatic%20bladder%20cancer%20patients&author=M%20Chen&author=J%20Gu&author=GL%20Delclos&author=AM%20Killary&author=Z%20Fan&volume=31&issue=8&publication_year=2010&pages=1387-91&pmid=20530239&doi=10.1093/carcin/bgq110&)

41. Guertin DA, Sabatini DM. Defining the role of mTOR in cancer. Cancer Cell. 2007;12(1):9–22. doi: 10.1016/j.ccr.2007.05.008 .  [DOI](https://doi.org/10.1016/j.ccr.2007.05.008) | [DOI](https://doi.org/10.1016/j.ccr.2007.05.008) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17613433/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Cell&title=Defining%20the%20role%20of%20mTOR%20in%20cancer&author=DA%20Guertin&author=DM%20Sabatini&volume=12&issue=1&publication_year=2007&pages=9-22&pmid=17613433&doi=10.1016/j.ccr.2007.05.008&)
