# Genomewide Association Study Identifies Novel Genetic Loci That Modify Antiplatelet Effects and Pharmacokinetics of Clopidogrel

## Metadata
**Authors:** W‐P Zhong, H Wu, J‐Y Chen, X‐X Li, H‐M Lin, B Zhang, Z‐W Zhang, D‐L Ma, S Sun, H‐P Li, L‐P Mai, G‐D He, X‐P Wang, H‐P Lei, H‐K Zhou, L Tang, S‐W Liu, S‐L Zhong
**Journal:** Clinical Pharmacology and Therapeutics
**Date:** 2017 Feb 9
**DOI:** [10.1002/cpt.589](https://doi.org/10.1002/cpt.589)
**PMID:** 27981573
**PMCID:** PMC5485718
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5485718/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC5485718/pdf/CPT-101-791.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC5485718/pdf/CPT-101-791.pdf)

## Abstract

Genetic variants in the pharmacokinetic (PK) mechanism are the main underlying factors affecting the antiplatelet response to clopidogrel. Using a genomewide association study (GWAS) to identify new genetic loci that modify antiplatelet effects in Chinese patients with coronary heart disease, we identified novel variants in two transporter genes (SLC14A2 rs12456693, ATP‐binding cassette [ABC]A1 rs2487032) and in N6AMT1 (rs2254638) associated with P2Y12 reaction unit (PRU) and plasma active metabolite (H4) concentration. These new variants dramatically improved the predictability of PRU variability to 37.7%. The associations between these loci and PK parameters of clopidogrel and H4 were observed in additional patients, and its function on the activation of clopidogrel was validated in liver S9 fractions (P < 0.05). Rs2254638 was further identified to exert a marginal risk effect for major adverse cardiac events in an independent cohort. In conclusion, new genetic variants were systematically identified as risk factors for the reduced efficacy of clopidogrel treatment.

**WHAT IS THE CURRENT KNOWLEDGE ON THE TOPIC?**

☑ Clopidogrel is a first‐line antiplatelet therapy for CHD, but it is associated with substantial variability in PK and pharmacodynamic response. To date, gene variants explain only a small proportion of the variability.

**WHAT QUESTION DID THIS STUDY ADDRESS?**

☑ The study aimed to identify new genetic loci‐modifying antiplatelet response to clopidogrel in Chinese patients with CHD by a systematic analysis combining antiplatelet effects and PK.

**WHAT THIS STUDY ADDS TO OUR KNOWLEDGE**

☑ We identified novel variants in two transporter genes (rs12456693 in *SLC14A2* and rs2487032 in *ABCA1*) and in *N6AMT1* (rs2254638) associated with clopidogrel‐treated PRU and plasma H4 concentration. The new variants, together with *CYP2C19*2* and clinical factors, dramatically improved the predictability of PRU variability to 37.7% compared with the published value of ∼20%. Meanwhile, *N6AMT1* rs2254638 was identified to exert a marginal risk effect for MACE in an independent cohort.

**HOW THIS MIGHT CHANGE CLINICAL PHARMACOLOGY OR TRANSLATIONAL SCIENCE**

☑ Our study findings enhanced the understanding of the absorption and metabolic mechanisms that influence antiplatelet response to clopidogrel treatment.

Clopidogrel is a first‐line antiplatelet therapy for coronary heart disease (CHD) and an extensively prescribed drug worldwide.[1](#cpt589-bib-0001), [2](#cpt589-bib-0002) However, clopidogrel therapy is associated with highly variable pharmacodynamic response, and individuals with high residual on‐treatment platelet reactivity have a great risk of death, myocardial infarction, and stent thrombosis.[3](#cpt589-bib-0003), [4](#cpt589-bib-0004), [5](#cpt589-bib-0005), [6](#cpt589-bib-0006), [7](#cpt589-bib-0007), [8](#cpt589-bib-0008)

Reduced efficacy of clopidogrel may be caused not only by nongenetic factors, such as age and use of interacting drugs, but also by genetic polymorphisms, particularly those in transporters and enzymes participating in clopidogrel absorption and metabolic transformation. Several studies have been conducted using a candidate gene approach to identify genetic variants associated with antiplatelet and pharmacokinetic (PK) response to clopidogrel.[9](#cpt589-bib-0009), [10](#cpt589-bib-0010), [11](#cpt589-bib-0011), [12](#cpt589-bib-0012) However, such candidate gene studies have shown inconsistent results, except for those of *CYP2C19* variants. A recent genomewide association study (GWAS) identified an association with *CYP2C19*2* and diminished platelet response to clopidogrel treatment and poor cardiovascular outcomes,[12](#cpt589-bib-0012) and this result was confirmed by other studies.[11](#cpt589-bib-0011), [13](#cpt589-bib-0013), [14](#cpt589-bib-0014)

Gene variants that have been investigated to date explain only 12% of the variability in the clopidogrel response and 20.6% when combined with clinical factors.[12](#cpt589-bib-0012), [15](#cpt589-bib-0015) The substantial variability in the response to clopidogrel remains unexplained.[16](#cpt589-bib-0016), [17](#cpt589-bib-0017) Published GWAS may have missed single‐nucleotide polymorphisms (SNPs) with small or moderately sized genetic effects on the P2Y12 reaction unit (PRU) variations under clopidogrel treatment, because of the stringent cutoff imposed by Bonferroni correction. Furthermore, a vast racial difference exists in the distribution of genetic variants. Studies focusing on white subjects may have also overlooked significant associations with Chinese subjects.

Considering that genetic variants in PK mechanisms are the main underlying factors that modify the antiplatelet effects of clopidogrel, joint analysis of genetic variants modifying pharmacodynamic and PK response to clopidogrel should be effective for identifying the biochemical mechanism.[18](#cpt589-bib-0018), [19](#cpt589-bib-0019), [20](#cpt589-bib-0020) Therefore, additional GWAS using combination analysis of antiplatelet effects and PKs are necessary to identify new potential genetic variants that affect the antiplatelet response to clopidogrel,[21](#cpt589-bib-0021) particularly in non‐European populations.

In this study, we conducted a genome‐wide investigation on the effects of genetic variants on the clopidogrel‐treated antiplatelet response and H4 (an active metabolite of clopidogrel) exposure. For SNPs associated with platelet function in response to clopidogrel and H4 exposure, we then evaluated their relationship with PK parameters of clopidogrel and H4 in additional patients with CHD, as well as investigated the function of these SNPs on clopidogrel activation in human liver tissues. Finally, the SNPs suggestively associated with antiplatelet effects and H4 exposures were further determined by their link to major adverse cardiac events (MACEs) in patients with CHD after percutaneous coronary intervention.

## RESULTS

### Patient characteristics and their effects on the antiplatelet response to clopidogrel and H4 exposure

A schematic of the design of the GWAS is illustrated in **Figure** [1](#cpt589-fig-0001). In stage I, 3 of 120 patients were excluded for failure of the VerifyNow test, whereas 2 patients were excluded for deviation from the population, as shown by principal component analysis (PCA; **Supplementary Figure S1** online). Thus, 115 patients passed quality control and were included in the GWAS analysis. Baseline clinical characteristics and their effects on the H4 concentration and PRU of 115 patients are shown in **Supplementary Table S1** online. In stage IIa, baseline clinical characteristics and their effects on PKs of clopidogrel and H4 in 31 patients are shown in **Supplementary Table S2** online.

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9d1f/5485718/a50707b2d1b5/CPT-101-791-g001.jpg)

Schematic of the design of the genomewide association study (GWAS). CHD, coronary heart disease; PK, pharmacokinetic; PRU, P2Y12 reaction unit; SNP, single‐nucleotide polymorphism.

Patient characteristics and their effects on antiplatelet and PK response to clopidogrel in stages I and IIa can be found in the **Supplementary Data** online.

### Genomewide association study of the antiplatelet response to clopidogrel and H4 exposure

To identify SNPs associated with antiplatelet response to clopidogrel treatment, we performed a GWAS of PRU at 4 h in 115 patients with CHD. The antiplatelet effects of clopidogrel at 4 h widely varied (**Supplementary Figure S2a** online). Although no genomewide significant association signals were detected in single marker‐based GWAS of PRU or PRU >208 with a Bonferroni‐adjusted *P* value (*P* < 7.11 × 10^−8^), a total of 125 loci in 25 genes showed suggestive evidence of association with PRU or PRU >208 (*P* < 1 × 10^−4^) excluding *CYP2C19*3* (rs4986893, *P* _PRU >208_ = 0.5003, *P* _PRU_ = 0.3513).

The plasma concentration of clopidogrel, H4, and H3 at 2 h widely varied (**Supplementary Figure S2** online). Among 125 SNPs associated with PRU or PRU >208 (*P* < 1 × 10^−4^), 27 SNPs were associated with the H4 concentration at 2 h at the significance level of *P* < 0.01 (**Supplementary Table S3** online), among which 23 SNPs were within the *HELLS‐CYP2C18‐CYP2C19* cluster (**Figure** [2](#cpt589-fig-0002) **a**). These SNPs were in strong linkage disequilibrium (LD) with one another with pairwise r^2^ = 0.49–1.0, including *CYP2C19*2* (rs4244285, *P* = 1.66 × 10^−5^). Univariate linear regression analysis revealed that *CYP2C19*2* was associated with increased PRU, which explained 11% PRU variation. Others included rs12456693 in *SLC14A2* (*P* = 8.52 × 10^−3^; **Figure** [2](#cpt589-fig-0002) **b**), rs2254638 in *N6AMT1* (*P* = 6.29 × 10^−3^; **Figure** [2](#cpt589-fig-0002) **c**), rs4741806 on chromosome 9 (*P* = 2.40 × 10^−3^), and rs7292279 on chromosome 22 (*P* = 9.08 × 10^−3^).

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9d1f/5485718/c87324d68985/CPT-101-791-g002.jpg)

Linkage disequilibrium block analysis of single‐nucleotide polymorphisms in a 250 kb region upstream and downstream of CYP2C19 rs4244285 (*2) on chromosome 10 (a), SCL14A2 rs12456693 on chromosome 18 (b), N6AMT1 rs2254638 on chromosome 21 (c), and their relationship with P2Y12 reaction unit (PRU) and H4 concentration at 2 h in 115 patients with coronary heart disease. [Color figure can be viewed at wileyonlinelibrary.com]

### Association of single‐nucleotide polymorphisms in absorption, distribution, metabolism, and excretion genes with P2Y12 reaction unit and H4 concentration

A total of 399 loci in 95 candidate absorption, distribution, metabolism, and excretion (ADME) genes were significantly associated with PRU or PRU >208 (*P* < 0.01). Among these loci, 77 SNPs were associated with the H4 concentration (*P* < 0.05; **Supplementary Table S4** online). Besides SNPs in the *CYP2C19* cluster and *SLC14A2*, four loci (ATP‐binding cassette [*ABC*]*A1* rs2487032, *ABCA4* rs4147820, *MGST1* rs12369968, and *UGT2B11* rs13123057) were significantly associated with the H4 concentration (*P* < 0.01).

### Selected single‐nucleotide polymorphism genotyping for replication studies in stages II and III

SNPs were selected based on the following criteria for further validation: (1) *P* < 1.0 × 10^−4^ for PRU or PRU >208 for all SNPs and *P* < 0.05 for the H4 concentration; (2) *P* < 0.01 for PRU or PRU >208 for SNPs in ADME genes and *P* < 0.05 for the H4 concentration in stage I; (3) only the SNP with the lowest *P* value was chosen to represent multiple SNPs, which were in high LD (r^2^ ≥ 0.8) for each gene. Eighteen SNPs satisfied the criteria for replication (**Supplementary Table S5** online, **Table** [1](#cpt589-tbl-0001)).

### Table 1.

| SNP | CHR | Gene symbol | P value (n = 115) in stage I |  |  | P value (n = 31) in stage IIa |  |  |  |  |  | P value (n = 32) in stage IIb |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| PRU >208 | PRU | H4 C2h | Clop Tmax | Clop Cmax | Clop AUC0‐4h | H4 Tmax | H4 Cmax | H4 AUC0‐4h |  |  |  |  |
| rs74460025 | 5 | GABRB2 | 2.61 × 10‐5 | 5.08 × 10‐3 | 5.19 × 10‐3 | 0.679 | 0.973 | 0.672 | 0.760 | 0.740 | 0.668 | 0.9402 |
| rs17145154 | 7 | DNAH11 | 4.72 × 10‐5 | 6.36 × 10‐4 | 2.40 × 10‐2 | 0.672 | 0.557 | 0.799 | 0.0513 | 0.950 | 0.794 | 0.4269 |
| rs4741806 | 9 | Intergenic | 4.04 × 10‐4 | 6.28 × 10‐5 | 2.40 × 10‐3 | 0.972 | 0.483 | 0.544 | 0.495 | 0.918 | 0.541 | 0.9502 |
| rs1048196 | 10 | HELLS | 1.45 × 10‐5 | 4.05 × 10‐5 | 4.20 × 10‐4 | 0.978 | 0.402 | 0.587 | 0.742 | 0.248 | 0.0273 | 0.0135 |
| rs1926711 | 10 | CYP2C18 | 8.24 × 10‐6 | 3.51 × 10‐5 | 6.61 × 10‐3 | 0.667 | 0.153 | 0.291 | 0.773 | 0.517 | 0.0843 | 0.0153 |
| rs4244285 | 10 | CYP2C19 | 1.66 × 10‐5 | 6.01 × 10‐5 | 2.20 × 10‐4 | 0.955 | 0.125 | 0.205 | 0.842 | 0.172 | 0.00861 | 0.0334 |
| rs2852213 | 11 | GRIK4 | 8.77 × 10‐5 | 5.12 × 10‐3 | 1.62 × 10‐2 | 0.593 | 0.147 | 0.0794 | 0.121 | 0.679 | 0.495 | 0.9842 |
| rs774392 | 12 | GRIP1 | 2.49 × 10‐5 | 3.18 × 10‐4 | 6.72 × 10‐3 | 0.432 | 0.444 | 0.369 | 0.382 | 0.611 | 0.387 | 0.961 |
| rs12913988 | 15 | ATP10A | 1.57 × 10‐3 | 9.69 × 10‐6 | 1.47 × 10‐2 | 0.307 | 0.831 | 0.691 | 0.895 | 0.538 | 0.617 | 0.9948 |
| rs12456693 | 18 | SLC14A2 | 6.83 × 10‐5 | 1.07 × 10‐2 | 8.52 × 10‐3 | 0.00485 | 0.262 | 0.172 | 0.0444 | 0.0245 | 0.129 | 0.4131 |
| rs1571678 | 21 | N6AMT1 | 1.21 × 10‐3 | 2.65 × 10‐5 | 6.85 × 10‐3 | 0.0249 | 0.969 | 0.958 | 0.0551 | 0.286 | 0.358 | 0.0643 |
| rs2254638 | 21 | N6AMT1 | 1.87 × 10‐3 | 5.37 × 10‐5 | 6.29 × 10‐3 | 0.0249 | 0.969 | 0.958 | 0.0551 | 0.286 | 0.358 | 0.0386 |
| rs17209532 | 22 | Intergenic | 2.46 × 10‐3 | 1.92 × 10‐5 | 3.08 × 10‐2 | 0.218 | 0.266 | 0.167 | 0.0670 | 0.520 | 0.306 | 0.7581 |
| rs7292279 | 22 | Intergenic | 7.39 × 10‐4 | 4.34 × 10‐5 | 9.08 × 10‐3 | 0.730 | 0.689 | 0.863 | 0.385 | 0.944 | 0.860 | 0.1483 |
| rs13123057 | 4 | UGT2B11 | 8.69 × 10‐3 | 8.94 × 10‐3 | 8.69 × 10‐4 | 0.334 | 0.984 | 0.847 | 0.332 | 0.396 | 0.850 | 0.7324 |
| rs12369968 | 12 | MGST1 | 3.43 × 10‐4 | 3.57 × 10‐3 | 9.18 × 10‐4 | 0.459 | 0.440 | 0.448 | 0.131 | 0.0904 | 0.135 | 0.9919 |
| rs2487032 | 9 | ABCA1 | 3.46 × 10‐3 | 3.75 × 10‐2 | 5.34 × 10‐3 | 0.243 | 0.0211 | 0.00590 | 0.0189 | 0.382 | 0.329 | 0.8812 |
| rs4147820 | 1 | ABCA4 | 1.74 × 10‐3 | 8.42 × 10‐4 | 7.00 × 10‐3 | 0.495 | 0.448 | 0.604 | 0.759 | 0.673 | 0.679 | 0.662 |

Table 1 Caption: Summary of genomewide association study of antiplatelet effects and pharmacokinetic of clopidogrel in stage I and replication results of pharmacokinetic in stage II

### Contribution of the novel loci to variability of antiplatelet effects and pharmacokinetic of clopidogrel

To identify the independent contribution of each factor to the interindividual variability in antiplatelet and PK response to clopidogrel, multivariate linear regression analysis was used. The results showed that *CYP2C19*2*, rs2254638 in *N6AMT1*, rs2487032 in *ABCA1*, use of calcium channel blockers, and sex were independent predictors for variability of the antiplatelet response to clopidogrel. These factors explained 37.7% of the variance in PRU, among which 28.2% was explained by genotypes. *CYP2C19*2*, rs12456693 in *SLC14A2*, rs2254638 in *N6AMT1*, and age were independent predictors for variability of the H4 concentration. These predictors explained up to 28.3% of variability, of which 23.4% was explained by genotypes (**Table** [2](#cpt589-tbl-0002)).

### Table 2.

|   | Parameter | Estimate | Partial R2 | Model R2 | P value |
| --- | --- | --- | --- | --- | --- |
|   | Intercept | 181.89 |   |   | < 0.0001 |
|   | rs2254638 in N6ANT1 | ‐39.95 | 0.148 | 0.148 | < 0.0001 |
| PRU | rs4244285 in CYP2C19 | 33.74 | 0.109 | 0.257 | 0.0001 |
|   | CCB | 140.61 | 0.051 | 0.308 | 0.0063 |
|   | Male | ‐23.71 | 0.044 | 0.352 | 0.0085 |
|   | rs2487032 in ABCA1 | ‐19.76 | 0.025 | 0.377 | 0.0441 |
|   | Intercept | ‐3.15 |   |   | 0.648 |
|   | rs4244285 in CYP2C19 | 3.35 | 0.163 | 0.163 | < 0.0001 |
| H4 | rs2254638 in N6ANT1 | 3.77 | 0.045 | 0.208 | 0.0166 |
|   | Age | 0.27 | 0.048 | 0.256 | 0.0106 |
|   | rs12456693 in SLC14A2 | 3.35 | 0.027 | 0.283 | 0.0521 |

Table 2 Caption: Multiple regression analysis with forward selection for modeling P2Y12 reaction unit and H4 based on genotypes, interacting drugs, and age in 115 patients

### Effects of single‐nucleotide polymorphisms on the pharmacokinetic of clopidogrel and H4 in patients with coronary heart disease

**Supplementary Figure S3** online presents the plasma concentration‐time profiles of clopidogrel and its metabolites obtained in 31 patients after the administration of clopidogrel in stage IIa. The absorption of clopidogrel from the gastrointestinal tract widely varied. The PK parameters calculated for clopidogrel and its metabolites are presented in **Supplementary Table S6** online.

To investigate the absorption and metabolic mechanisms that influenced antiplatelet response to clopidogrel treatment, SNPs associated with antiplatelet effects and PKs of clopidogrel were further studied in 31 patients with CHD along with PK parameters of clopidogrel and H4 (**Supplementary Table S7** online, **Table** [1](#cpt589-tbl-0001)). Among the 18 significant SNPs identified in stage I, rs1048196 in *HELLS*, rs1926711 in *CYP2C18*, and *CYP2C19*2* in the *HELLS‐CYP2C18‐CYP2C19* cluster were associated with the mean H4 area under the curve (AUC_0–4h_; *P* = 0.0273, 0.0843, and 0.00861, respectively). These SNPs were in strong LD (r^2^ = 0.49–0.96). *CYP2C19*2* is shown as a representative (**Figure** [3](#cpt589-fig-0003) **a**). The AUC_0–4h_ of H4 for GG homozygotes was significantly higher than those for GA heterozygotes and AA homozygotes of *CYP2C19*2*.

### Figure 3.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9d1f/5485718/f51f27a38a7b/CPT-101-791-g003.jpg)

Association of significant single‐nucleotide polymorphisms (a) rs4244285 in CYP2C19, also known as CYP2C19*2; (b) rs12456693 in SLC14A2; (c) rs2254638 in N6AMT1; and (d) rs2487032 in ABCA1 with pharmacokinetic (PK) parameters of clopidogrel and H4 in 31 patients with coronary heart disease. The PK parameters include area under the concentration‐time curve (AUC), peak concentration (Cmax), and peak concentration (Tmax).

The rs12456693 T allele in *SLC14A2* was associated with decreased time of maximum plasma concentration (T_max_) of clopidogrel (*P* = 0.00485), decreased T_max_ of H4 (*P* = 0.0444), and increased peak plasma concentration (C_max_) of H4 (*P* = 0.0245; **Figure** [3](#cpt589-fig-0003) **b**). The rs2254638 C allele in *N6AMT1* was associated with increased T_max_ of clopidogrel (*P* = 0.0249) and increased T_max_ of H4 (*P* = 0.0551; **Figure** [3](#cpt589-fig-0003) **c**). The rs2487032 A allele in *ABCA1* was associated with decreased C_max_ (*P* = 0.0211) and decreased AUC_0–4h_ (*P* = 0.00590) of clopidogrel and decreased T_max_ of H4 (*P* = 0.0189; **Figure** [3](#cpt589-fig-0003) **d**).

### Effects of single‐nucleotide polymorphisms on the activation of clopidogrel in human liver S9 fractions

To investigate further whether these new variants have a potential function in the activation of clopidogrel, we used 32 individual human liver S9 fractions to compare H4 formation among different genotypes of the 18 SNPs (**Supplementary Table S8** online, **Table** [1](#cpt589-tbl-0001)). Rs1048196 in *HELLS*, rs1926711 in *CYP2C18*, and *CYP2C19*2* in the *HELLS‐CYP2C18‐CYP2C19* cluster were associated with decreased H4 concentration (*P* = 0.0135, 0.0153, and 0.0334, respectively). *CYP2C19*2* was associated with decreased H4 concentration (1.84 ± 1.57 ng/mL/mg protein for **2/*2*; 5.86 ± 5.08 ng/mL/mg protein for **1/*2*; and 8.53 ± 6.46 ng/mL/mg protein for **1/*1*; *P* = 0.0334). The *N6AMT1* rs2254638 was associated with decreased H4 concentration (9.21 ± 3.68 ng/mL/mg protein for TT; 7.69 ± 6.48 ng/mL/mg protein for TC; and 3.36 ± 3.75 ng/mL/mg protein for CC; *P* = 0.0386). Other SNPs were not associated with the H4 concentration in human liver S9 fractions (*P* > 0.05).

### Effects of the novel loci on the risk of major adverse cardiac event in patients with coronary heart disease

To validate whether the significant SNPs identified potentially influence the clinical outcome, these SNPs were genotyped for 299 patients with CHD after percutaneous coronary intervention by the Sequenom's MassARRAY system. Rs1571678 was not included, because it was in strong LD with rs2254638 in *N6AMT1* at r^2^ = 0.93. The call rate of the SNPs ranged from 96.33–99.80% (**Supplementary Table S9** online).

Logistic regression analysis showed that diabetes mellitus (odds ratio [OR] = 1.666; 95% confidence interval [CI] = 0.977–2.84; *P* = 0.0608), heart failure (OR = 5.101; 95% CI = 2.617–9.94; *P* < 0.0001), arrhythmia (OR = 3.695; 95% CI = 1.793–7.615; *P* = 0.0004), and use of calcium channel blockers (OR = 2.251; 95% CI = 1.338–3.788; *P* = 0.0022) exerted a significant effect on the risk of MACE (**Table** [3](#cpt589-tbl-0003)). Rs2254638 in *N6AMT1* was marginally associated with the occurrence of MACE (*P* = 0.0653; false discovery rate [FDR] = 0.4726). Patients with the C allele had a higher risk of MACE than those without (OR = 1.428; 95% CI = 0.978–2.086). Rs12456693 in *SLC14A2* was marginally associated with the occurrence of MACE (*P* = 0.0953; FDR = 0.4726). No significant association was observed between *CYP2C19*2* and MACE (*P* = 0.9463; FDR = 0.9463). Rs12913988 in *ATP10A* was found to be significantly associated with the occurrence of MACE (*P* = 0.0011; FDR = 0.0187). Patients with the T allele had a higher risk of MACE in 1.5 years (OR = 1.876; 95% CI = 1.285–2.740). Other SNPs were not associated with the occurrence of MACE (*P* > 0.1; FDR >0.5; **Table** [4](#cpt589-tbl-0004)).

### Table 3.

| Characteristics | Case | Control | P value | OR per variant allele (95% CI) |
| --- | --- | --- | --- | --- |
| No. (%) or | No. (%) or |  |  |  |
| Mean ± SD | Mean ± SD |  |  |  |
| Demographic data |   |   |   |   |
| Age, y | 65.71 ± 9.95 | 64.08 ± 11.43 | 0.2388 | 1.014 (0.991–1.037) |
| Male | 73 (81.11) | 166 (79.43) | 0.7386 | 1.112 (0.595–2.079) |
| Medical history |   |   |   |   |
| Diabetes mellitus | 32 (35.56) | 52 (24.88) | 0.0608a | 1.666 (0.977–2.84)a |
| Hypertension | 60 (66.67) | 121 (57.89) | 0.1556 | 1.455 (0.867–2.44) |
| Heart failure | 28 (31.11) | 17 (8.13) | < 0.0001a | 5.101 (2.617–9.94)a |
| ACS | 71 (78.89) | 155 (74.16) | 0.3837 | 1.302 (0.719–2.357) |
| Arrhythmia | 20 (22.22) | 15 (7.18) | 0.0004a | 3.695 (1.793–7.615)a |
| Biochemical measurements |   |   |   |   |
| ALT | 38.99 ± 63.15 | 29.48 ± 29.15 | 0.1311 | 1.005 (0.999–1.011) |
| AST | 38.23 ± 41.41 | 39.75 ± 72.11 | 0.8548 | 1 (0.996–1.004) |
| LDLC | 2.57 ± 1.02 | 2.84 ± 2.12 | 0.2416 | 0.862 (0.673–1.105) |
| HDLC | 0.97 ± 0.25 | 0.99 ± 0.25 | 0.5219 | 0.717 (0.259–1.986) |
| TRIG | 1.65 ± 1.24 | 1.69 ± 1.26 | 0.8092 | 0.975 (0.793–1.199) |
| APOA | 0.98 ± 0.25 | 1.03 ± 0.26 | 0.1739 | 0.45 (0.142–1.423) |
| CHOL | 4.24 ± 1.14 | 4.53 ± 1.25 | 0.0665 | 0.81 (0.647–1.014) |
| CK | 191.44 ± 295.10 | 181.62 ± 345.37 | 0.8293 | 1.0 (0.999–1.001) |
| CKMB | 9.22 ± 11.75 | 8.44 ± 9.83 | 0.5874 | 1.007 (0.982–1.032) |
| CREA | 102.32 ± 83.21 | 94.41 ± 43.20 | 0.3425 | 1.002 (0.998–1.006) |
| GLUC | 6.97 ± 2.82 | 6.45 ± 2.41 | 0.3109 | 1.002 (0.998–1.006) |
| LPa | 274.97 ± 323.41 | 306.37 ± 340.55 | 0.4993 | 1 (0.999–1.001) |
| Medication |   |   |   |   |
| Statins | 89 (98.89) | 208 (99.52) | 0.55 | 0.428 (0.026–6.918) |
| β‐blockers | 84 (93.33) | 195 (93.3) | 0.9919 | 1.005 (0.373–2.705) |
| ACE inhibitors | 72 (80.00) | 151 (72.25) | 0.1599 | 1.536 (0.844–2.795) |
| CCBs | 39 (43.33) | 53 (25.36) | 0.0022a | 2.251 (1.338–3.788)a |
| Proton pump inhibitors | 56 (62.22) | 116 (55.50) | 0.2815 | 1.32 (0.796–2.19) |

Table 3 Caption: Association of baseline characteristics with the occurrence of major adverse cardiac event in 299 patients with coronary heart disease

### Table 4.

| SNP | Gene symbol | Genotype | With MACE | Without MACE | OR per variant allele (95% CI) | P value | FDR |
| --- | --- | --- | --- | --- | --- | --- | --- |
| No. (%) | No. (%) |  |  |  |  |  |  |
| rs74460025 | GABRB2 | TT | 64 (71.91) | 160 (77.29) | 1.487 (0.9–2.457) | 0.1216 | 0.4726 |
|   |   | TC | 21 (23.60) | 46 (22.22) |   |   |   |
|   |   | CC | 4 (4.49) | 1 (0.48) |   |   |   |
| rs17145154 | DNAH11 | TT | 51 (59.3) | 104 (51.23) | 1.272 (0.859–1.884) | 0.2299 | 0.6292 |
|   |   | TG | 28 (32.56) | 78 (38.42) |   |   |   |
|   |   | GG | 7 (8.14) | 21 (10.34) |   |   |   |
| rs4741806 | Intergenic | AA | 72 (80.9) | 158 (76.33) | 0.718 (0.402–1.282) | 0.2623 | 0.6292 |
|   |   | AC | 17 (19.1) | 45 (21.74) |   |   |   |
|   |   | CC | 0 (0) | 4 (1.93) |   |   |   |
| rs1048196 | HELLS | CC | 41 (46.07) | 92 (45.1) | 0.942 (0.638–1.393) | 0.7661 | 0.9422 |
|   |   | CT | 41 (46.07) | 93 (45.59) |   |   |   |
|   |   | TT | 7 (7.78) | 19 (9.31) |   |   |   |
| rs1926711 | CYP2C18 | GG | 33 (37.5) | 74 (36.63) | 1.022 (0.693–1.505) | 0.9143 | 0.9463 |
|   |   | GA | 45 (51.14) | 105 (51.98) |   |   |   |
|   |   | AA | 10 (11.36) | 23 (11.39) |   |   |   |
| rs4244285 | CYP2C19 | GG | 42 (46.67) | 98 (47.34) | 0.987 (0.666–1.461) | 0.9463 | 0.9463 |
|   |   | GA | 42 (46.67) | 90 (43.48) |   |   |   |
|   |   | AA | 6 (6.67) | 19 (9.18) |   |   |   |
| rs2852213 | GRIK4 | CC | 13 (14.61) | 62 (30.10) | 1.302 (0.918–1.847) | 0.139 | 0.4726 |
|   |   | CT | 54 (60.67) | 89 (43.20) |   |   |   |
|   |   | TT | 22 (24.72) | 55 (26.70) |   |   |   |
| rs774392 | GRIP1 | CC | 3 (3.33) | 3 (1.44) | 0.904 (0.548–1.49) | 0.6913 | 0.9422 |
|   |   | CA | 20 (22.22) | 49 (23.56) |   |   |   |
|   |   | AA | 67 (74.44) | 156 (75.00) |   |   |   |
| rs12913988a | ATP10A | CC | 7 (7.78) | 43 (20.77) | 1.876 (1.285–2.740)a | 0.0011a | 0.0187a |
|   |   | CT | 40 (44.44) | 99 (47.83) |   |   |   |
|   |   | TT | 43 (47.78) | 65 (31.4) |   |   |   |
| rs12456693a | SLC14A2 | CC | 66 (74.16) | 141 (67.46) | 0.654 (0.398–1.077)a | 0.0953a | 0.4726a |
|   |   | CT | 23 (25.84) | 58 (27.75) |   |   |   |
|   |   | TT | 0 (0) | 10 (4.78) |   |   |   |
| rs2254638a | N6AMT1 | TT | 13 (14.77) | 42 (20.59) | 1.428 (0.978–2.086)a | 0.0653a | 0.4726a |
|   |   | TC | 45 (51.14) | 113 (55.39) |   |   |   |
|   |   | CC | 30 (34.09) | 49 (24.02) |   |   |   |
| rs17209532 | Intergenic | GG | 33 (36.67) | 92 (45.10) | 0.822 (0.57–1.187) | 0.2961 | 0.6292 |
|   |   | GA | 46 (51.11) | 88 (43.14) |   |   |   |
|   |   | AA | 11 (12.22) | 24 (11.76) |   |   |   |
| rs7292279 | Intergenic | GG | 31 (34.44) | 82 (39.42) | 0.974 (0.68–1.396) | 0.8876 | 0.9463 |
|   |   | GA | 48 (53.33) | 94 (44.98) |   |   |   |
|   |   | AA | 11 (12.22) | 33 (15.79) |   |   |   |
| rs13123057 | UGT2B11 | GG | 41 (45.56) | 98 (47.34) | 1.162 (0.771–1.751) | 0.4725 | 0.7302 |
|   |   | GT | 28 (31.11) | 72 (34.78) |   |   |   |
|   |   | AG | 7 (7.78) | 16 (7.73) |   |   |   |
|   |   | TT | 9 (10.00) | 12 (5.80) |   |   |   |
|   |   | AT | 5 (5.56) | 7 (3.38) |   |   |   |
|   |   | AA | 0 (0) | 2 (0.97) |   |   |   |
| rs12369968 | MGST1 | AA | 32 (33.71) | 73 (34.93) | 1.16 (0.814–1.655) | 0.4114 | 0.7052 |
|   |   | AG | 40 (44.94) | 104 (49.76) |   |   |   |
|   |   | GG | 19 (21.35) | 32 (15.31) |   |   |   |
| rs2487032 | ABCA1 | GG | 36 (40.91) | 83 (40.1) | 0.954 (0.688–1.322) | 0.7759 | 0.9422 |
|   |   | GA | 31 (35.23) | 82 (39.61) |   |   |   |
|   |   | AA | 21 (23.86) | 42 (20.29) |   |   |   |
| rs4147820 | ABCA4 | GG | 44 (48.89) | 120 (57.42) | 1.177 (0.796–1.739) | 0.4148 | 0.7052 |
|   |   | GA | 41 (45.56) | 73 (34.93) |   |   |   |
|   |   | AA | 5 (5.56) | 16 (7.66) |   |   |   |

Table 4 Caption: Association of genetic polymorphisms with the occurrence of major adverse cardiac event in 299 patients with coronary heart disease

## DISCUSSION

This work is the first systematic GWAS combining analysis of antiplatelet effects and PKs to investigate functional genetic variants that modify clopidogrel efficacy in Chinese patients with CHD. We identified new genetic variants that affect the antiplatelet response to clopidogrel by changing PK behavior in patients with CHD. The new variants in *N6AMT1* and *ABCA1*, together with *CYP2C19* and clinical factors, dramatically improved the predictability of PRU variability to 37.7% compared with the published value of ∼20%. Rs12456693 in *SLC14A2* was also an independent predictor for the variability of the H4 concentration. *N6AMT1* rs2254638 and *SLC14A2* rs12456693 were further identified to exert a marginal risk effect on the occurrence of MACE in an independent patient with CHD cohort.

We confirmed the association between variants in the *HELLS‐CYP2C18‐CYP2C19* cluster and antiplatelet effects and PK of clopidogrel.[12](#cpt589-bib-0012), [22](#cpt589-bib-0022) *CYP2C19*2* independently explained 16% of the variation in the logarithm of the H4 concentration at 2 h and 11% of PRU variation after LD of clopidogrel, similar to a published value of ∼12% of the variation in the clopidogrel response.[12](#cpt589-bib-0012) Whereas *CYP2C19*3* did not have a significant contribution to antiplatelet effects, PK, and activation of clopidogrel. The reason may be due to the limited effect of *CYP2C19*3* on the activation of clopidogrel, small sample number, and low variant allele frequency.[23](#cpt589-bib-0023) Although many studies have shown a positive association between *CYP2C19* functional variants and cardiovascular outcomes, we did not find a significant association between *CYP2C19*2* or **3* polymorphism and MACE. This result was in line with the findings of two previous meta‐analyses[24](#cpt589-bib-0024), [25](#cpt589-bib-0025) and the finding from a recent study.[26](#cpt589-bib-0026)

One of the novel findings in the study is that rs2254638 in *N6AMT1* may be a potential genetic biomarker to predict the individual difference of the intermediate efficacy response and clinical endpoint to clopidogrel treatment through metabolic and PK mechanisms. Our study showed that rs2254638 in *N6AMT1* could lower liver metabolic function, reduce the plasma H4 concentration, decrease the antiplatelet effects of clopidogrel with a higher treated PRU, and confer a risk effect on the occurrence of MACE. *N6AMT1* rs2254638 independently explained 4.5% of the variation in the logarithm of the H4 concentration 2 h and 14.8% of PRU variation. The mechanistic basis for the function of *N6AMT1* in clopidogrel activation remains unclear. The *N6AMT1* gene was identified in humans and encodes an N(6)‐adenine‐specific DNA methyltransferase.[27](#cpt589-bib-0027) This enzyme possesses specific activity in converting the arsenic metabolite, monomethylarsonous acid, to the less toxic dimethylarsinic acid.[28](#cpt589-bib-0028) Harari *et al*.[29](#cpt589-bib-0029) reported that *N6AMT1* polymorphisms are associated with arsenic methylation in Andean women. Whether *N6AMT1* is involved in the activation of clopidogrel directly or through other mechanisms, such as methylation of other metabolic enzymes, should be elucidated in further studies.

Compared with the metabolism of clopidogrel, the current knowledge about its absorption is limited. Only one transporter, a P‐glycoprotein encoded by *ABCB1*, has been reported to be involved in the intestinal absorption of clopidogrel.[30](#cpt589-bib-0030) Almost all previous publications concerning clopidogrel absorption variability focused on the effects of genetic polymorphism of *ABCB1* on antiplatelet and PK response to clopidogrel and clinical outcomes. However, published data on the association of *ABCB1* genetic variants and antiplatelet effects and PKs of clopidogrel are inconsistent.[30](#cpt589-bib-0030), [31](#cpt589-bib-0031), [32](#cpt589-bib-0032), [33](#cpt589-bib-0033) We did not find an SNP consistently associated with PRU and H4 concentration. *ABCB1* genetic variants may not be a key factor contributing to the absorption variability of clopidogrel. Genetic variants in other transporters involving clopidogrel disposition may contribute to the large coefficient of variation in T_max_ and C_max_ values. The PK of clopidogrel and H4 may be affected not only by the transporter in the absorption phase but also by the transporter in the excretion phase.

We identified novel variants in two transporter genes, *SLC14A2* and *ABCA1*, associated with antiplatelet or PK response to clopidogrel that may enhance our understanding of the absorption and excretion of clopidogrel. *ABCA1* was independently associated with PRU but not with H4 at 2 h after a loading dose of clopidogrel, whereas *SLC14A2* was independently associated with H4 in the elimination phase. These findings suggested that the mechanisms for *ABCA1* and *SLC14A2* involved in the PK of clopidogrel and H4 differed.

*SLC14A2* is a member of the solute carrier family and encodes six urea transporters (UTs), UT‐A1 to UT‐A6 isoforms, in mammals. UT transporters are found mainly in the kidneys and heart,[34](#cpt589-bib-0034) cochlea,[35](#cpt589-bib-0035) placenta,[36](#cpt589-bib-0036) colon, brain, and liver.[37](#cpt589-bib-0037) UTs facilitate the reabsorption and recycling of the urinary urea, thereby increasing medullary urea concentration.[38](#cpt589-bib-0038) Variants in *SLC14A2* were associated with blood urea nitrogen levels in a Japanese population, but no clear phenotypic effect was demonstrated.[39](#cpt589-bib-0039) We hypothesized that *SLC14A2* was involved in the excretion of clopidogrel, but this hypothesis should be tested in further studies. The newly identified genetic variants in *SLC14A2* associated with antiplatelet effects and PKs of clopidogrel were found to have a marginal effect on clinical outcomes for patients with CHD after percutaneous coronary intervention.

*ABCA1* encodes a member of the ABC transporter family and is located on chromosome 9q22.31. *ABCA1* is expressed in many tissues and cells, including enterocytes and hepatocytes,[40](#cpt589-bib-0040) and plays a major role in cholesterol efflux. In enterocytes, *ABCA1* is localized on the basolateral plasma membrane,[41](#cpt589-bib-0041) and induction of *ABCA1* enhances basolateral (systemic) absorption of dietary cholesterol that is apically assimilated by enterocytes.[42](#cpt589-bib-0042) McNeish *et al*.[43](#cpt589-bib-0043) reported that *ABCA1* disruption in mice leads to the accumulation of intestinally derived cholesterol in plasma very low‐density lipoprotein. Our results showed that a variant of rs2487032, located in an *ABCA1* promoter, increased the concentration of plasma clopidogrel's active metabolite H4. This effect increased inhibition of platelet function. The mechanism may involve an increase in the speed of absorption of clopidogrel and H4. However, the SNP was not associated with clinical outcomes.

We also identified a new genetic variant (rs12913988) in *ATP10A* that functions as a significant risk factor for MACE occurrence, and it enhances the antiplatelet effects of clopidogrel with a low treated PRU and increases the plasma H4 concentration in stage I. The underlying reason leading to the result is unclear and requires further study. *ATP10A* encodes a protein belonging to the family of P‐type cation transport ATPases and to the subfamily of aminophospholipid‐transporting ATPases. The appearance of phosphatidylserine on the surface of animal cells triggers phagocytosis and blood coagulation.[44](#cpt589-bib-0044) ATPases transport a variety of different compounds, including ions and phospholipids, across a membrane using ATP hydrolysis for energy.

The novel SNPs detected in our study escaped detection in a previous European GWAS.[12](#cpt589-bib-0012) One of the reasons could be frequency differences among populations for SNPs. For example, rs2254638 in *N6AMT1* and rs12913988 in *ATP10A* are more frequent in East Asians (∼50–60%) than Europeans (∼10–15%). Another reason could be the difference of chips used. For example, the previous GWAS in European populations adopted Affymetrix GeneChip Human Mapping 500K or 1M (version 6.0) arrays[12](#cpt589-bib-0012) did not capture rs12913988 in *ATP10A*.

Our study had two limitations that merit mention. (1) The sample size was small; therefore, findings may exhibit false positivity because of multiple comparisons, a problem inherent in GWAS. To address this issue, we jointly analyzed antiplatelet effects and PKs of clopidogrel with genomewide SNPs and adopted a *P* value at the significance level of *CYP2C19*2* for prioritizing functional SNPs. Furthermore, we used not only other patient populations to replicate and confirm findings but in liver S9 fractions to validate the impact of SNPs *in vitro*. The validated SNPs with biological plausibility of ADME have a high likelihood of being true‐positive findings. We further investigated the association of genetic polymorphisms with the clinical outcome in another large population. (2) The H4 concentrations at 2 h after clopidogrel were used as a substitute PK parameter in GWAS of stage I and may have compromised the power to some extent. The PK parameters, including AUC and clearance, were originally designed to be estimated by a nonlinear mixed‐effects modeling method. However, given the large individual differences in the absorption phase of clopidogrel, T_max_ reached 4 h, which was the last timepoint of blood drawn, in some patients. Nonlinear mixed‐effects modeling of population PK may be biased. In addition, the H4 concentration at 2 h was highly correlated with AUC_0–4h_ (r = 0.8141; *P* < 0.0001) in 31 patients (**Supplementary Figure S4** online).

In conclusion, we systematically identified new genetic variants as risk factors for reduced efficacy of clopidogrel and highlighted related genes that may be involved in antiplatelet effects and PK of clopidogrel. Our results enhance our understanding of the absorption and metabolic mechanisms that influence antiplatelet response to clopidogrel treatment.

## METHODS

A detailed Methods section is provided in the **Supplementary Data** online.

### Study population and study design

A three‐stage GWAS in Han Chinese patients with CHD was performed. In stage I, the GWAS of on‐treatment platelet reactivity was performed to identify SNPs associated with antiplatelet effects of clopidogrel in 115 patients with CHD. The concentration of H4 was analyzed to identify a PK explanation for SNPs associated with antiplatelet effects in the same cohort. In stage IIa, SNPs associated with antiplatelet effects and PKs of clopidogrel were further studied in 31 patients with CHD along with PK parameters of clopidogrel and H4. In stage IIb, the function of these SNPs on clopidogrel activation was further investigated in 32 human liver tissues. In stage III, clinical outcomes (MACE) were used to study the association between these SNPs and response to clopidogrel in 299 patients with CHD. This study was approved by the Medical Ethical Review Committee of Guangdong General Hospital. All subjects provided written informed consent.

### Sample processing

In stage I, patients received a 300 mg loading dose of clopidogrel and a 100 mg dose of aspirin. Blood samples were collected for determining PRU and the plasma concentration of clopidogrel, active metabolite isomer H4, and inactive metabolite isomer H3 at 2 and 4 h after the loading dose. In stage IIa, blood samples for the PK assessment of clopidogrel and its metabolites were collected at 0, 0.25, 0.5, 1, 2, and 4 h after a 300 mg loading dose of clopidogrel.

### Antiplatelet effect analysis

The antiplatelet effectiveness of clopidogrel was measured using the VerifyNow, assay as described by the manufacturer (Accumetrics, San Diego, CA). The test of platelet function using PRU was assessed separately as a continuous measure and a categorical measure, which was defined as high or low platelet reactivity on clopidogrel using a PRU cutoff value higher than 208 based on previous studies.[3](#cpt589-bib-0003), [45](#cpt589-bib-0045)

### Bioanalytical analysis

A high‐performance liquid chromatography tandem mass spectrometry assay was developed and validated for the simultaneous determination of clopidogrel, metabolite stereoisomers (H3 and H4), and internal standard carbamazepine in human plasma and mixtures of liver S9 fraction. The representative tandem mass spectrometry spectrum and liquid chromatography mass spectrometry profiles are shown in **Supplementary Figure S5** online. The PK parameters of clopidogrel, H3, and H4 were estimated using the noncompartmental analysis function in Phoenix WinNonlin version 6.3 software (Pharsight, Cary, NC).

### Genotyping and quality control in the genomewide association study

A total of 900,015 SNPs in a GWAS scan of 117 subjects were genotyped with the Illumina HumanOmniZhongHua‐8 BeadChip according to the Infinium HD protocol from Illumina. Prior to association analysis, a systematic quality control procedure was applied to the raw genotyping data to filter unqualified SNPs and samples. We detected population outliers using a method based on PCA. After quality filtering and cleaning, 115 subjects with 703,143 SNPs were included in further analyses.

### Candidate single‐nucleotide polymorphism association in absorption, distribution, metabolism, and excretion genes

We focused particularly on the genes involved in ADME to identify genetic variants with PK mechanisms associated with response to clopidogrel using combination analysis of antiplatelet effects and PK. We closely examined 16,202 SNPs in 295 candidate ADME genes from PharmaADME ([http://www.pharmaadme.org/](http://www.pharmaadme.org/)) and their surrounding regions (±2 kb) for association with PRU or PRU >208 and the H4 concentration in 115 patients with CHD.

### Pharmacokinetic replication study

To confirm the association between significant SNPs with PK parameters of clopidogrel and H4, as well as further assess possible mechanisms for the association between significant SNPs and PRU, a replication study in 31 patients with CHD was performed in stage IIa. The plasma concentration was determined at 0, 0.25, 0.5, 1, 2, and 4 h.

### Functional replication study on clopidogrel activation

In stage IIb, the function of these SNPs on clopidogrel activation was further investigated in 32 human liver tissues to identify a plausible mechanism for newly identified genetic variants. The formation of H4 in the liver S9 fraction was analyzed using a validated liquid chromatography tandem mass spectrometry assay. Genotyping of 18 SNPs was performed using the iPLEX MassARRAY platform (Sequenom).

### Clinical validation study

In stage III, we used clinical outcomes as a measure to study the association between these SNPs and clopidogrel efficacy. A total of 299 Han Chinese patients with CHD who received clopidogrel therapy were sequentially recruited and followed up to 1.5 years, including 91 patients with MACE and 208 patients without MACE as control. Genotyping was conducted by Sequenom's MassARRAY system (Sequenom). The effect of genetic variants on the clinical outcome of MACE in 1.5 years was assessed.

### Statistics

Details on statistical analysis can be found in the **Supplementary Data** online.

## SOURCE OF FUNDING

This work was supported by the National Nature Science Foundation of China (No. 81373486, 81673514), Science and Technology Development Projects of Guangdong Province, China (No. 2016B090918114 and 2013B021800157), and Science and Technology Development Projects of Guangzhou, Guangdong, China (201510010236 and 201604020096).

## AUTHOR CONTRIBUTIONS

S.‐L.Z., W.Z., H.W., J.‐Y.C., X.‐X.L., H.‐M.L., B.Z., Z.‐W.Z., D.‐L.M., S.S., H.‐P.L., L.‐P.M., G.‐D.H., X.‐P.W., H.L., H.‐K.Z., L.T., and S.‐W.L. wrote the manuscript. S.‐L.Z., W.Z., H.W., and J.‐Y.C. designed the research. W.Z., H.W., X.‐X.L., H.‐M.L., H.‐P.L., G.‐D.H., and H.L. performed the research. W.Z., H.W., X.‐X.L., H.‐M.L., X.‐P.W., H.‐K.Z., L.T., and S.‐W.L. analyzed the data.

## CONFLICT OF INTEREST

The authors declared no conflict of interest.

## Supporting information

## ACKNOWLEDGMENTS

We appreciate Gilles Tuffal, George Liu, and their colleagues of Sanofi‐Aventis for kind offering of the 3′‐methoxyacetophenone derivatives (MP‐H3 [SAR287550‐1] and MP‐H4 [SAR206251]) of free clopidogrel active metabolite stereoisomers (H3 and H4).

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Amsterdam, E.A. et al 2014 AHA/ACC guideline for the management of patients with non‐ST‐elevation acute coronary syndromes: a report of the American College of Cardiology/American Heart Association Task Force on Practice Guidelines. Circulation 130, e344–e426 (2014).  [DOI](https://doi.org/10.1161/CIR.0000000000000134) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25249585/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circulation&title=2014%20AHA/ACC%20guideline%20for%20the%20management%20of%20patients%20with%20non%E2%80%90ST%E2%80%90elevation%20acute%20coronary%20syndromes:%20a%20report%20of%20the%20American%20College%20of%20Cardiology/American%20Heart%20Association%20Task%20Force%20on%20Practice%20Guidelines&author=E.A.%20Amsterdam&volume=130&publication_year=2014&pages=e344-e426&pmid=25249585&doi=10.1161/CIR.0000000000000134&)

2. O'Gara, P.T. et al 2013 ACCF/AHA guideline for the management of ST‐elevation myocardial infarction: a report of the American College of Cardiology Foundation/American Heart Association Task Force on Practice Guidelines. Circulation 127, e362–e425 (2013).  [DOI](https://doi.org/10.1161/CIR.0b013e3182742cf6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23247304/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circulation&title=2013%20ACCF/AHA%20guideline%20for%20the%20management%20of%20ST%E2%80%90elevation%20myocardial%20infarction:%20a%20report%20of%20the%20American%20College%20of%20Cardiology%20Foundation/American%20Heart%20Association%20Task%20Force%20on%20Practice%20Guidelines&author=P.T.%20O'Gara&volume=127&publication_year=2013&pages=e362-e425&pmid=23247304&doi=10.1161/CIR.0b013e3182742cf6&)

3. Aradi, D. et al Bleeding and stent thrombosis on P2Y12‐inhibitors: collaborative analysis on the role of platelet reactivity for risk stratification after percutaneous coronary intervention. Eur. Heart J. 36, 1762–1771 (2015).  [DOI](https://doi.org/10.1093/eurheartj/ehv104) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25896078/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur.%20Heart%20J.&title=Bleeding%20and%20stent%20thrombosis%20on%20P2Y12%E2%80%90inhibitors:%20collaborative%20analysis%20on%20the%20role%20of%20platelet%20reactivity%20for%20risk%20stratification%20after%20percutaneous%20coronary%20intervention&author=D.%20Aradi&volume=36&publication_year=2015&pages=1762-1771&pmid=25896078&doi=10.1093/eurheartj/ehv104&)

4. Tantry, U.S. et al Consensus and update on the definition of on‐treatment platelet reactivity to adenosine diphosphate associated with ischemia and bleeding. J. Am. Coll. Cardiol. 62, 2261–2273 (2013).  [DOI](https://doi.org/10.1016/j.jacc.2013.07.101) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24076493/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Am.%20Coll.%20Cardiol.&title=Consensus%20and%20update%20on%20the%20definition%20of%20on%E2%80%90treatment%20platelet%20reactivity%20to%20adenosine%20diphosphate%20associated%20with%20ischemia%20and%20bleeding&author=U.S.%20Tantry&volume=62&publication_year=2013&pages=2261-2273&pmid=24076493&doi=10.1016/j.jacc.2013.07.101&)

5. Price, M.J. et al Platelet reactivity and cardiovascular outcomes after percutaneous coronary intervention: a time‐dependent analysis of the gauging responsiveness with a VerifyNow P2Y12 assay: impact on thrombosis and safety (GRAVITAS) trial. Circulation 124, 1132–1137 (2011).  [DOI](https://doi.org/10.1161/CIRCULATIONAHA.111.029165) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21875913/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circulation&title=Platelet%20reactivity%20and%20cardiovascular%20outcomes%20after%20percutaneous%20coronary%20intervention:%20a%20time%E2%80%90dependent%20analysis%20of%20the%20gauging%20responsiveness%20with%20a%20VerifyNow%20P2Y12%20assay:%20impact%20on%20thrombosis%20and%20safety%20(GRAVITAS)%20trial&author=M.J.%20Price&volume=124&publication_year=2011&pages=1132-1137&pmid=21875913&doi=10.1161/CIRCULATIONAHA.111.029165&)

6. Campo, G. et al Prospective evaluation of on‐clopidogrel platelet reactivity over time in patients treated with percutaneous coronary intervention relationship with gene polymorphisms and clinical outcome. J. Am. Coll. Cardiol. 57, 2474–2483 (2011).  [DOI](https://doi.org/10.1016/j.jacc.2010.12.047) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21679849/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Am.%20Coll.%20Cardiol.&title=Prospective%20evaluation%20of%20on%E2%80%90clopidogrel%20platelet%20reactivity%20over%20time%20in%20patients%20treated%20with%20percutaneous%20coronary%20intervention%20relationship%20with%20gene%20polymorphisms%20and%20clinical%20outcome&author=G.%20Campo&volume=57&publication_year=2011&pages=2474-2483&pmid=21679849&doi=10.1016/j.jacc.2010.12.047&)

7. Marcucci, R. et al Cardiovascular death and nonfatal myocardial infarction in acute coronary syndrome patients receiving coronary stenting are predicted by residual platelet reactivity to ADP detected by a point‐of‐care assay: a 12‐month follow‐up. Circulation 119, 237–242 (2009).  [DOI](https://doi.org/10.1161/CIRCULATIONAHA.108.812636) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19118249/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circulation&title=Cardiovascular%20death%20and%20nonfatal%20myocardial%20infarction%20in%20acute%20coronary%20syndrome%20patients%20receiving%20coronary%20stenting%20are%20predicted%20by%20residual%20platelet%20reactivity%20to%20ADP%20detected%20by%20a%20point%E2%80%90of%E2%80%90care%20assay:%20a%2012%E2%80%90month%20follow%E2%80%90up&author=R.%20Marcucci&volume=119&publication_year=2009&pages=237-242&pmid=19118249&doi=10.1161/CIRCULATIONAHA.108.812636&)

8. Valenti, R. et al Residual platelet reactivity to predict long‐term clinical outcomes after clopidogrel loading in patients with acute coronary syndromes: comparison of different cutoff values by light transmission aggregometry from the responsiveness to clopidogrel and stent thrombosis 2‐acute coronary syndrome (RECLOSE 2‐ACS) study. J. Thromb. Thrombolysis 40, 76–82 (2015).  [DOI](https://doi.org/10.1007/s11239-014-1159-1) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25502874/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Thromb.%20Thrombolysis&title=Residual%20platelet%20reactivity%20to%20predict%20long%E2%80%90term%20clinical%20outcomes%20after%20clopidogrel%20loading%20in%20patients%20with%20acute%20coronary%20syndromes:%20comparison%20of%20different%20cutoff%20values%20by%20light%20transmission%20aggregometry%20from%20the%20responsiveness%20to%20clopidogrel%20and%20stent%20thrombosis%202%E2%80%90acute%20coronary%20syndrome%20(RECLOSE%202%E2%80%90ACS)%20study&author=R.%20Valenti&volume=40&publication_year=2015&pages=76-82&pmid=25502874&doi=10.1007/s11239-014-1159-1&)

9. Zhu, H.J. , Wang, X. , Gawronski, B.E. , Brinda, B.J. , Angiolillo, D.J. & Markowitz, J.S. Carboxylesterase 1 as a determinant of clopidogrel metabolism and activation. J. Pharmacol. Exp. Ther. 344, 665–672 (2013).  [DOI](https://doi.org/10.1124/jpet.112.201640) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23275066/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Pharmacol.%20Exp.%20Ther.&title=Carboxylesterase%201%20as%20a%20determinant%20of%20clopidogrel%20metabolism%20and%20activation&author=H.J.%20Zhu&author=X.%20Wang&author=B.E.%20Gawronski&author=B.J.%20Brinda&author=D.J.%20Angiolillo&volume=344&publication_year=2013&pages=665-672&pmid=23275066&doi=10.1124/jpet.112.201640&)

10. Viviani Anselmi, C. et al Routine assessment of on‐clopidogrel platelet reactivity and gene polymorphisms in predicting clinical outcome following drug‐eluting stent implantation in patients with stable coronary artery disease. JACC Cardiovasc. Interv. 6, 1166–1175 (2013).  [DOI](https://doi.org/10.1016/j.jcin.2013.06.010) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24262617/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JACC%20Cardiovasc.%20Interv.&title=Routine%20assessment%20of%20on%E2%80%90clopidogrel%20platelet%20reactivity%20and%20gene%20polymorphisms%20in%20predicting%20clinical%20outcome%20following%20drug%E2%80%90eluting%20stent%20implantation%20in%20patients%20with%20stable%20coronary%20artery%20disease&author=C.%20Viviani%20Anselmi&volume=6&publication_year=2013&pages=1166-1175&pmid=24262617&doi=10.1016/j.jcin.2013.06.010&)

11. Mega, J.L. et al Reduced‐function CYP2C19 genotype and risk of adverse clinical outcomes among patients treated with clopidogrel predominantly for PCI: a meta‐analysis. JAMA 304, 1821–1830 (2010).  [DOI](https://doi.org/10.1001/jama.2010.1543) | [PMC free article](/articles/PMC3048820/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20978260/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA&title=Reduced%E2%80%90function%20CYP2C19%20genotype%20and%20risk%20of%20adverse%20clinical%20outcomes%20among%20patients%20treated%20with%20clopidogrel%20predominantly%20for%20PCI:%20a%20meta%E2%80%90analysis&author=J.L.%20Mega&volume=304&publication_year=2010&pages=1821-1830&pmid=20978260&doi=10.1001/jama.2010.1543&)

12. Shuldiner, A.R. et al Association of cytochrome P450 2C19 genotype with the antiplatelet effect and clinical efficacy of clopidogrel therapy. JAMA 302, 849–857 (2009).  [DOI](https://doi.org/10.1001/jama.2009.1232) | [PMC free article](/articles/PMC3641569/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19706858/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA&title=Association%20of%20cytochrome%20P450%202C19%20genotype%20with%20the%20antiplatelet%20effect%20and%20clinical%20efficacy%20of%20clopidogrel%20therapy&author=A.R.%20Shuldiner&volume=302&publication_year=2009&pages=849-857&pmid=19706858&doi=10.1001/jama.2009.1232&)

13. Perry, C.G. & Shuldiner, A.R. Pharmacogenomics of anti‐platelet therapy: how much evidence is enough for clinical implementation? J. Hum. Genet. 58, 339–345 (2013).  [DOI](https://doi.org/10.1038/jhg.2013.41) | [PMC free article](/articles/PMC3715315/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23697979/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Hum.%20Genet.&title=Pharmacogenomics%20of%20anti%E2%80%90platelet%20therapy:%20how%20much%20evidence%20is%20enough%20for%20clinical%20implementation?&author=C.G.%20Perry&author=A.R.%20Shuldiner&volume=58&publication_year=2013&pages=339-345&pmid=23697979&doi=10.1038/jhg.2013.41&)

14. Bokeriya, O.L. , Kudzoeva, Z.F. , Shvarts, V.A. , Koasari, A.K. & Donakanyan, S.A. The possibility of selecting optimal antiplatelet therapy in patients with coronary heart disease in terms of CYP2C19 polymorphism [in Russian]. Ter. Arkh. 88, 47–54 (2016).  [DOI](https://doi.org/10.17116/terarkh201688547-54) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27239927/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ter.%20Arkh.&title=The%20possibility%20of%20selecting%20optimal%20antiplatelet%20therapy%20in%20patients%20with%20coronary%20heart%20disease%20in%20terms%20of%20CYP2C19%20polymorphism%20%5Bin%20Russian%5D&author=O.L.%20Bokeriya&author=Z.F.%20Kudzoeva&author=V.A.%20Shvarts&author=A.K.%20Koasari&author=S.A.%20Donakanyan&volume=88&publication_year=2016&pages=47-54&pmid=27239927&doi=10.17116/terarkh201688547-54&)

15. Bouman, H.J. et al Variability in on‐treatment platelet reactivity explained by CYP2C19*2 genotype is modest in clopidogrel pretreated patients undergoing coronary stenting. Heart 97, 1239–1244 (2011).  [DOI](https://doi.org/10.1136/hrt.2010.220509) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21628721/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Heart&title=Variability%20in%20on%E2%80%90treatment%20platelet%20reactivity%20explained%20by%20CYP2C19*2%20genotype%20is%20modest%20in%20clopidogrel%20pretreated%20patients%20undergoing%20coronary%20stenting&author=H.J.%20Bouman&volume=97&publication_year=2011&pages=1239-1244&pmid=21628721&doi=10.1136/hrt.2010.220509&)

16. Frelinger, A.L. 3rd et al Clopidogrel pharmacokinetics and pharmacodynamics vary widely despite exclusion or control of polymorphisms (CYP2C19, ABCB1, PON1), noncompliance, diet, smoking, co‐medications (including proton pump inhibitors), and pre‐existent variability in platelet function. J. Am. Coll. Cardiol. 61, 872–879 (2013).  [DOI](https://doi.org/10.1016/j.jacc.2012.11.040) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23333143/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Am.%20Coll.%20Cardiol.&title=Clopidogrel%20pharmacokinetics%20and%20pharmacodynamics%20vary%20widely%20despite%20exclusion%20or%20control%20of%20polymorphisms%20(CYP2C19,%20ABCB1,%20PON1),%20noncompliance,%20diet,%20smoking,%20co%E2%80%90medications%20(including%20proton%20pump%20inhibitors),%20and%20pre%E2%80%90existent%20variability%20in%20platelet%20function&author=A.L.%20Frelinger&volume=61&publication_year=2013&pages=872-879&pmid=23333143&doi=10.1016/j.jacc.2012.11.040&)

17. Chan, N.C. et al Role of phenotypic and genetic testing in managing clopidogrel therapy. Blood 124, 689–699 (2014).  [DOI](https://doi.org/10.1182/blood-2014-01-512723) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24951432/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Role%20of%20phenotypic%20and%20genetic%20testing%20in%20managing%20clopidogrel%20therapy&author=N.C.%20Chan&volume=124&publication_year=2014&pages=689-699&pmid=24951432&doi=10.1182/blood-2014-01-512723&)

18. Ahn, K. , Luo, J. , Berg, A. , Keefe, D. & Wu, R. Functional mapping of drug response with pharmacodynamic‐pharmacokinetic principles. Trends Pharmacol. Sci. 31, 306–311 (2010).  [DOI](https://doi.org/10.1016/j.tips.2010.04.004) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20488563/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Trends%20Pharmacol.%20Sci.&title=Functional%20mapping%20of%20drug%20response%20with%20pharmacodynamic%E2%80%90pharmacokinetic%20principles&author=K.%20Ahn&author=J.%20Luo&author=A.%20Berg&author=D.%20Keefe&author=R.%20Wu&volume=31&publication_year=2010&pages=306-311&pmid=20488563&doi=10.1016/j.tips.2010.04.004&)

19. Wu, R. et al A conceptual framework for pharmacodynamic genome‐wide association studies in pharmacogenomics. Drug Discov. Today 16, 884–890 (2011).  [DOI](https://doi.org/10.1016/j.drudis.2011.09.001) | [PMC free article](/articles/PMC4029408/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21920452/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Discov.%20Today&title=A%20conceptual%20framework%20for%20pharmacodynamic%20genome%E2%80%90wide%20association%20studies%20in%20pharmacogenomics&author=R.%20Wu&volume=16&publication_year=2011&pages=884-890&pmid=21920452&doi=10.1016/j.drudis.2011.09.001&)

20. Goswami, S. et al Genetic variants in transcription factors are associated with the pharmacokinetics and pharmacodynamics of metformin. Clin. Pharmacol. Ther. 96, 370–379 (2014).  [DOI](https://doi.org/10.1038/clpt.2014.109) | [PMC free article](/articles/PMC4171106/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24853734/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Genetic%20variants%20in%20transcription%20factors%20are%20associated%20with%20the%20pharmacokinetics%20and%20pharmacodynamics%20of%20metformin&author=S.%20Goswami&volume=96&publication_year=2014&pages=370-379&pmid=24853734&doi=10.1038/clpt.2014.109&)

21. Karaźniewicz‐Łada, M. , Danielak, D. , Rubiś, B. , Burchardt, P. , Oszkinis, G. & Główka, F . The influence of genetic polymorphism of Cyp2c19 isoenzyme on the pharmacokinetics of clopidogrel and its metabolites in patients with cardiovascular diseases. J. Clin. Pharmacol. 54, 874–880 (2014).  [DOI](https://doi.org/10.1002/jcph.323) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24782221/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Pharmacol.&title=The%20influence%20of%20genetic%20polymorphism%20of%20Cyp2c19%20isoenzyme%20on%20the%20pharmacokinetics%20of%20clopidogrel%20and%20its%20metabolites%20in%20patients%20with%20cardiovascular%20diseases&author=M.%20Kara%C5%BAniewicz%E2%80%90%C5%81ada&author=D.%20Danielak&author=B.%20Rubi%C5%9B&author=P.%20Burchardt&author=G.%20Oszkinis&volume=54&publication_year=2014&pages=874-880&pmid=24782221&doi=10.1002/jcph.323&)

22. Brandt, J.T. et al Common polymorphisms of CYP2C19 and CYP2C9 affect the pharmacokinetic and pharmacodynamic response to clopidogrel but not prasugrel. J. Thromb. Haemost. 5, 2429–2436 (2007).  [DOI](https://doi.org/10.1111/j.1538-7836.2007.02775.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17900275/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Thromb.%20Haemost.&title=Common%20polymorphisms%20of%20CYP2C19%20and%20CYP2C9%20affect%20the%20pharmacokinetic%20and%20pharmacodynamic%20response%20to%20clopidogrel%20but%20not%20prasugrel&author=J.T.%20Brandt&volume=5&publication_year=2007&pages=2429-2436&pmid=17900275&doi=10.1111/j.1538-7836.2007.02775.x&)

23. Man, M. et al Genetic variation in metabolizing enzyme and transporter genes: comprehensive assessment in 3 major East Asian subpopulations with comparison to Caucasians and Africans. J. Clin. Pharmacol. 50, 929–940 (2010).  [DOI](https://doi.org/10.1177/0091270009355161) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20173083/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Pharmacol.&title=Genetic%20variation%20in%20metabolizing%20enzyme%20and%20transporter%20genes:%20comprehensive%20assessment%20in%203%20major%20East%20Asian%20subpopulations%20with%20comparison%20to%20Caucasians%20and%20Africans&author=M.%20Man&volume=50&publication_year=2010&pages=929-940&pmid=20173083&doi=10.1177/0091270009355161&)

24. Holmes, M.V. , Perel, P. , Shah, T. , Hingorani, A.D. & Casas, J.P. CYP2C19 genotype, clopidogrel metabolism, platelet function, and cardiovascular events: a systematic review and meta‐analysis. JAMA 306, 2704–2714 (2011).  [DOI](https://doi.org/10.1001/jama.2011.1880) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22203539/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA&title=CYP2C19%20genotype,%20clopidogrel%20metabolism,%20platelet%20function,%20and%20cardiovascular%20events:%20a%20systematic%20review%20and%20meta%E2%80%90analysis&author=M.V.%20Holmes&author=P.%20Perel&author=T.%20Shah&author=A.D.%20Hingorani&author=J.P.%20Casas&volume=306&publication_year=2011&pages=2704-2714&pmid=22203539&doi=10.1001/jama.2011.1880&)

25. Bauer, T. , Bouman, H.J. , van Werkum, J.W. , Ford, N.F. , ten Berg, J.M. & Taubert, D. Impact of CYP2C19 variant genotypes on clinical efficacy of antiplatelet treatment with clopidogrel: systematic review and meta‐analysis. BMJ 343, d4588 (2011).  [DOI](https://doi.org/10.1136/bmj.d4588) | [PMC free article](/articles/PMC3191560/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21816733/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMJ&title=Impact%20of%20CYP2C19%20variant%20genotypes%20on%20clinical%20efficacy%20of%20antiplatelet%20treatment%20with%20clopidogrel:%20systematic%20review%20and%20meta%E2%80%90analysis&author=T.%20Bauer&author=H.J.%20Bouman&author=J.W.%20van%20Werkum&author=N.F.%20Ford&author=J.M.%20ten%20Berg&volume=343&publication_year=2011&pages=d4588&pmid=21816733&doi=10.1136/bmj.d4588&)

26. Doll, J.A. et al Impact of CYP2C19 metabolizer status on patients with ACS treated with prasugrel versus clopidogrel. J. Am. Coll. Cardiol. 67, 936–947 (2016).  [DOI](https://doi.org/10.1016/j.jacc.2015.12.036) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26916483/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Am.%20Coll.%20Cardiol.&title=Impact%20of%20CYP2C19%20metabolizer%20status%20on%20patients%20with%20ACS%20treated%20with%20prasugrel%20versus%20clopidogrel&author=J.A.%20Doll&volume=67&publication_year=2016&pages=936-947&pmid=26916483&doi=10.1016/j.jacc.2015.12.036&)

27. Ratel, D. et al Undetectable levels of N6‐methyl adenine in mouse DNA: cloning and analysis of PRED28, a gene coding for a putative mammalian DNA adenine methyltransferase. FEBS Lett. 580, 3179–3184 (2006).  [DOI](https://doi.org/10.1016/j.febslet.2006.04.074) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16684535/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=FEBS%20Lett.&title=Undetectable%20levels%20of%20N6%E2%80%90methyl%20adenine%20in%20mouse%20DNA:%20cloning%20and%20analysis%20of%20PRED28,%20a%20gene%20coding%20for%20a%20putative%20mammalian%20DNA%20adenine%20methyltransferase&author=D.%20Ratel&volume=580&publication_year=2006&pages=3179-3184&pmid=16684535&doi=10.1016/j.febslet.2006.04.074&)

28. Ren, X. et al Involvement of N‐6 adenine‐specific DNA methyltransferase 1 (N6AMT1) in arsenic biomethylation and its role in arsenic‐induced toxicity. Environ. Health Perspect. 119, 771–777 (2011).  [DOI](https://doi.org/10.1289/ehp.1002733) | [PMC free article](/articles/PMC3114810/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21193388/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Environ.%20Health%20Perspect.&title=Involvement%20of%20N%E2%80%906%20adenine%E2%80%90specific%20DNA%20methyltransferase%201%20(N6AMT1)%20in%20arsenic%20biomethylation%20and%20its%20role%20in%20arsenic%E2%80%90induced%20toxicity&author=X.%20Ren&volume=119&publication_year=2011&pages=771-777&pmid=21193388&doi=10.1289/ehp.1002733&)

29. Harari, F. , Engström, K. , Concha, G. , Colque, G. , Vahter, M. & Broberg, K. N‐6‐adenine‐specific DNA methyltransferase 1 (N6AMT1) polymorphisms and arsenic methylation in Andean women. Environ. Health Perspect. 121, 797–803 (2013).  [DOI](https://doi.org/10.1289/ehp.1206003) | [PMC free article](/articles/PMC3702000/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23665909/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Environ.%20Health%20Perspect.&title=N%E2%80%906%E2%80%90adenine%E2%80%90specific%20DNA%20methyltransferase%201%20(N6AMT1)%20polymorphisms%20and%20arsenic%20methylation%20in%20Andean%20women&author=F.%20Harari&author=K.%20Engstr%C3%B6m&author=G.%20Concha&author=G.%20Colque&author=M.%20Vahter&volume=121&publication_year=2013&pages=797-803&pmid=23665909&doi=10.1289/ehp.1206003&)

30. Taubert, D. et al Impact of P‐glycoprotein on clopidogrel absorption. Clin. Pharmacol. Ther. 80, 486–501 (2006).  [DOI](https://doi.org/10.1016/j.clpt.2006.07.007) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17112805/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Impact%20of%20P%E2%80%90glycoprotein%20on%20clopidogrel%20absorption&author=D.%20Taubert&volume=80&publication_year=2006&pages=486-501&pmid=17112805&doi=10.1016/j.clpt.2006.07.007&)

31. Karaźniewicz‐Łada, M. et al Impact of common ABCB1 polymorphism on pharmacokinetics and pharmacodynamics of clopidogrel and its metabolites. J. Clin. Pharm. Ther. 40, 226–231 (2015).  [DOI](https://doi.org/10.1111/jcpt.12236) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25430046/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Pharm.%20Ther.&title=Impact%20of%20common%20ABCB1%20polymorphism%20on%20pharmacokinetics%20and%20pharmacodynamics%20of%20clopidogrel%20and%20its%20metabolites&author=M.%20Kara%C5%BAniewicz%E2%80%90%C5%81ada&volume=40&publication_year=2015&pages=226-231&pmid=25430046&doi=10.1111/jcpt.12236&)

32. Nassar, S. , Amro, O. , Abu‐Rmaileh, H. , Alshaer, I. , Korachi, M. & Ayesh, S. ABCB1 C3435T and CYP2C19*2 polymorphisms in a Palestinian and Turkish population: a pharmacogenetic perspective to clopidogrel. Meta Gene 2, 314–319 (2014).  [DOI](https://doi.org/10.1016/j.mgene.2014.01.009) | [PMC free article](/articles/PMC4287958/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25606414/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Meta%20Gene&title=ABCB1%20C3435T%20and%20CYP2C19*2%20polymorphisms%20in%20a%20Palestinian%20and%20Turkish%20population:%20a%20pharmacogenetic%20perspective%20to%20clopidogrel&author=S.%20Nassar&author=O.%20Amro&author=H.%20Abu%E2%80%90Rmaileh&author=I.%20Alshaer&author=M.%20Korachi&volume=2&publication_year=2014&pages=314-319&pmid=25606414&doi=10.1016/j.mgene.2014.01.009&)

33. Su, J.F. , Hu, X.H. & Li, C.Y. Risk factors for clopidogrel resistance in patients with ischemic cerebral infarction and the correlation with gene rs1045642 polymorphism. Exp. Ther. Med. 9, 267–271 (2015).  [DOI](https://doi.org/10.3892/etm.2014.2058) | [PMC free article](/articles/PMC4247296/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25452814/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Exp.%20Ther.%20Med.&title=Risk%20factors%20for%20clopidogrel%20resistance%20in%20patients%20with%20ischemic%20cerebral%20infarction%20and%20the%20correlation%20with%20gene%20rs1045642%20polymorphism&author=J.F.%20Su&author=X.H.%20Hu&author=C.Y.%20Li&volume=9&publication_year=2015&pages=267-271&pmid=25452814&doi=10.3892/etm.2014.2058&)

34. Duchesne, R. et al UT‐A urea transporter protein in heart: increased abundance during uremia, hypertension, and heart failure. Circ. Res. 89, 139–145 (2001).  [DOI](https://doi.org/10.1161/hh1401.093293) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11463720/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circ.%20Res.&title=UT%E2%80%90A%20urea%20transporter%20protein%20in%20heart:%20increased%20abundance%20during%20uremia,%20hypertension,%20and%20heart%20failure&author=R.%20Duchesne&volume=89&publication_year=2001&pages=139-145&pmid=11463720&doi=10.1161/hh1401.093293&)

35. Kwun, Y.S. et al Immunohistochemical localization of urea transporters A and B in the rat cochlea. Hear. Res. 183, 84–96 (2003).  [DOI](https://doi.org/10.1016/s0378-5955(03)00218-1) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/13679141/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hear.%20Res.&title=Immunohistochemical%20localization%20of%20urea%20transporters%20A%20and%20B%20in%20the%20rat%20cochlea&author=Y.S.%20Kwun&volume=183&publication_year=2003&pages=84-96&pmid=13679141&doi=10.1016/s0378-5955(03)00218-1&)

36. Damiano, A.E. , Zotta, E. & Ibarra, C. Functional and molecular expression of AQP9 channel and UT‐A transporter in normal and preeclamptic human placentas. Placenta 27, 1073–1081 (2006).  [DOI](https://doi.org/10.1016/j.placenta.2005.11.014) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16480766/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Placenta&title=Functional%20and%20molecular%20expression%20of%20AQP9%20channel%20and%20UT%E2%80%90A%20transporter%20in%20normal%20and%20preeclamptic%20human%20placentas&author=A.E.%20Damiano&author=E.%20Zotta&author=C.%20Ibarra&volume=27&publication_year=2006&pages=1073-1081&pmid=16480766&doi=10.1016/j.placenta.2005.11.014&)

37. Fenton, R.A. et al Characterization of mouse urea transporters UT‐A1 and UT‐A2. Am. J. Physiol. Renal Physiol. 283, F817–F825 (2002).  [DOI](https://doi.org/10.1152/ajprenal.00263.2001) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12217874/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Physiol.%20Renal%20Physiol.&title=Characterization%20of%20mouse%20urea%20transporters%20UT%E2%80%90A1%20and%20UT%E2%80%90A2&author=R.A.%20Fenton&volume=283&publication_year=2002&pages=F817-F825&pmid=12217874&doi=10.1152/ajprenal.00263.2001&)

38. Stewart, G. The emerging physiological roles of the SLC14A family of urea transporters. Br. J. Pharmacol. 164, 1780–1792 (2011).  [DOI](https://doi.org/10.1111/j.1476-5381.2011.01377.x) | [PMC free article](/articles/PMC3246703/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21449978/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br.%20J.%20Pharmacol.&title=The%20emerging%20physiological%20roles%20of%20the%20SLC14A%20family%20of%20urea%20transporters&author=G.%20Stewart&volume=164&publication_year=2011&pages=1780-1792&pmid=21449978&doi=10.1111/j.1476-5381.2011.01377.x&)

39. Kamatani, Y. et al Genome‐wide association study of hematological and biochemical traits in a Japanese population. Nat. Genet. 42, 210–215 (2010).  [DOI](https://doi.org/10.1038/ng.531) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20139978/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet.&title=Genome%E2%80%90wide%20association%20study%20of%20hematological%20and%20biochemical%20traits%20in%20a%20Japanese%20population&author=Y.%20Kamatani&volume=42&publication_year=2010&pages=210-215&pmid=20139978&doi=10.1038/ng.531&)

40. Wellington, C.L. et al ABCA1 mRNA and protein distribution patterns predict multiple different roles and levels of regulation. Lab. Invest. 82, 273–283 (2002).  [DOI](https://doi.org/10.1038/labinvest.3780421) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11896206/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lab.%20Invest.&title=ABCA1%20mRNA%20and%20protein%20distribution%20patterns%20predict%20multiple%20different%20roles%20and%20levels%20of%20regulation&author=C.L.%20Wellington&volume=82&publication_year=2002&pages=273-283&pmid=11896206&doi=10.1038/labinvest.3780421&)

41. Ohama, T. et al Dominant expression of ATP‐binding cassette transporter‐1 on basolateral surface of Caco‐2 cells stimulated by LXR/RXR ligands. Biochem. Biophys. Res. Commun. 296, 625–630 (2002).  [DOI](https://doi.org/10.1016/s0006-291x(02)00853-7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12176027/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem.%20Biophys.%20Res.%20Commun.&title=Dominant%20expression%20of%20ATP%E2%80%90binding%20cassette%20transporter%E2%80%901%20on%20basolateral%20surface%20of%20Caco%E2%80%902%20cells%20stimulated%20by%20LXR/RXR%20ligands&author=T.%20Ohama&volume=296&publication_year=2002&pages=625-630&pmid=12176027&doi=10.1016/s0006-291x(02)00853-7&)

42. Murthy, S. , Born, E. , Mathur, S.N. & Field, F.J. LXR/RXR activation enhances basolateral efflux of cholesterol in CaCo‐2 cells. J. Lipid Res. 43, 1054–1064 (2002).  [DOI](https://doi.org/10.1194/jlr.m100358-jlr200) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12091489/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Lipid%20Res.&title=LXR/RXR%20activation%20enhances%20basolateral%20efflux%20of%20cholesterol%20in%20CaCo%E2%80%902%20cells&author=S.%20Murthy&author=E.%20Born&author=S.N.%20Mathur&author=F.J.%20Field&volume=43&publication_year=2002&pages=1054-1064&pmid=12091489&doi=10.1194/jlr.m100358-jlr200&)

43. McNeish, J. et al High density lipoprotein deficiency and foam cell accumulation in mice with targeted disruption of ATP‐binding cassette transporter‐1. Proc. Natl. Acad. Sci. USA 97, 4245–4250 (2000).  [DOI](https://doi.org/10.1073/pnas.97.8.4245) | [PMC free article](/articles/PMC18215/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10760292/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&title=High%20density%20lipoprotein%20deficiency%20and%20foam%20cell%20accumulation%20in%20mice%20with%20targeted%20disruption%20of%20ATP%E2%80%90binding%20cassette%20transporter%E2%80%901&author=J.%20McNeish&volume=97&publication_year=2000&pages=4245-4250&pmid=10760292&doi=10.1073/pnas.97.8.4245&)

44. Tang, X. , Halleck, M.S. , Schlegel, R.A. & Williamson, P. A subfamily of P‐type ATPases with aminophospholipid transporting activity. Science 272, 1495–1497 (1996).  [DOI](https://doi.org/10.1126/science.272.5267.1495) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8633245/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=A%20subfamily%20of%20P%E2%80%90type%20ATPases%20with%20aminophospholipid%20transporting%20activity&author=X.%20Tang&author=M.S.%20Halleck&author=R.A.%20Schlegel&author=P.%20Williamson&volume=272&publication_year=1996&pages=1495-1497&pmid=8633245&doi=10.1126/science.272.5267.1495&)

45. Stone, G.W. et al Platelet reactivity and clinical outcomes after coronary artery implantation of drug‐eluting stents (ADAPT‐DES): a prospective multicentre registry study. Lancet 382, 614–623 (2013).  [DOI](https://doi.org/10.1016/S0140-6736(13)61170-8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23890998/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Platelet%20reactivity%20and%20clinical%20outcomes%20after%20coronary%20artery%20implantation%20of%20drug%E2%80%90eluting%20stents%20(ADAPT%E2%80%90DES):%20a%20prospective%20multicentre%20registry%20study&author=G.W.%20Stone&volume=382&publication_year=2013&pages=614-623&pmid=23890998&doi=10.1016/S0140-6736(13)61170-8&)
