# Association of STAT-3 rs1053004 and VDR rs11574077 With FOLFIRI-Related Gastrointestinal Toxicity in Metastatic Colorectal Cancer Patients

## Metadata
**Authors:** Elena De Mattia, Erika Cecchin, Marcella Montico, Adrien Labriet, Chantal Guillemette, Eva Dreussi, Rossana Roncato, Alessia Bignucolo, Angela Buonadonna, Mario D’Andrea, Luigi Coppola, Sara Lonardi, Eric Lévesque, Derek Jonker, Félix Couture, Giuseppe Toffoli
**Journal:** Frontiers in Pharmacology
**Date:** 2018 Apr 13
**DOI:** [10.3389/fphar.2018.00367](https://doi.org/10.3389/fphar.2018.00367)
**PMID:** 29706892
**PMCID:** PMC5908896
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5908896/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC5908896/pdf/fphar-09-00367.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC5908896/pdf/fphar-09-00367.pdf)

## Abstract

Pharmacogenomics has largely been applied to the personalization of irinotecan-based treatment, focusing mainly on the study of genetic variants in adsorption, distribution, metabolism, and excretion (ADME) genes. The transcriptional control of ADME gene expression is mediated by a set of nuclear factors responding to cancer-related inflammation, which could have pharmacological implications. The aim of the present study was to uncover novel genetic predictors of neutropenia and gastrointestinal toxicity risk among 246 haplotype-tagging polymorphisms in 22 genes encoding inflammation-related cytokines and transcriptional regulators of ADME genes. The study comprised overall more than 400 metastatic colorectal cancer (mCRC) patients treated with first-line FOLFIRI, grouped in a discovery and a replication cohorts. A concordant protective effect of STAT-3 rs1053004 polymorphism against the risk of grade 3–4 gastrointestinal toxicity was observed in both the cohorts of patients (OR = 0.51, p = 0.045, q = 0.521 and OR = 0.39, p = 0.043, respectively). VDR rs11574077 polymorphism was demonstrated to affect both irinotecan biliary index (BI) and glucuronidation ratio (GR) by a pharmacokinetic analysis. This effect was consistent with an increased risk of grade 3–4 gastrointestinal toxicity in the discovery cohort (OR = 4.46, p = 0.010, q = 0.305). The association was not significant in the replication cohort (OR = 1.44, p = 0.601). These findings suggest an effect of STAT-3 and VDR polymorphisms on FOLFIRI-related gastrointestinal toxicity. If prospectively validated as predictive markers, they could be used to improve the clinical management of mCRC.

Keywords: STAT-3, VDR, genetic markers, irinotecan (CPT-11), gastrointestinal toxicity, colorectal cancer, inflammation, FOLFIRI

## Introduction

Irinotecan in association with FOLFIRI regimen represents the standard of care for first-line treatment of mCRC ([Fujita et al., 2015](#B17)). Despite the established antineoplastic effectiveness of the FOLFIRI regimen, the sporadic occurrence of severe and occasionally life-threatening complications often causes a failure of the chemotherapy, negatively impacting patient care ([Rothenberg et al., 2001](#B41)). Exposure to active irinotecan metabolite SN-38 is the major cause of toxicity, with severe neutropenia and delayed diarrhea as the dose-limiting toxicity and great inter-individual variability ([Gupta et al., 1994](#B19), [1997](#B20)). In the last few years, pharmacogenomics has largely been applied to the personalization of CRC treatment, specifically focusing on the genetic variability in ADME genes ([De Mattia et al., 2015](#B12)) demonstrating the role of genetic markers in *UGT1A* and *ABC*, and *SLC* transporters, in combination with clinico-demographic features, in predicting FOLFIRI toxicity ([Toffoli et al., 2006](#B43); [Cecchin et al., 2009](#B6); [Toffoli et al., 2010](#B44); [De Mattia et al., 2013a](#B13); [Levesque et al., 2013](#B29); [Chen et al., 2015a](#B7),[b](#B8)). As in the case of *UGT1A1*^∗^28, these studies led to specific pharmacogenetic guidelines ([Swen et al., 2011](#B42); [Caudle et al., 2013](#B4)). Nevertheless, the optimization of irinotecan-based therapy remains sub-optimal and under-explored targets may significantly contribute to determining the likelihood of severe complications after chemotherapy.

The expression of irinotecan-related ADME genes is controlled upstream by crucial transcription factors that respond to endobiotic and xenobiotic stimuli (e.g., inflammation response, drug administration) with a demonstrated effect on irinotecan bioavailability. In particular, the inflammatory state, a condition linked to some cancers, including CRC ([Markman and Shiao, 2015](#B31)), has been shown to affect the pharmacokinetic and pharmacodynamic parameters of various chemotherapeutics by modulating drug metabolic enzymes and ABC/SLC transporters ([Cressman et al., 2012](#B10); [Harvey and Morgan, 2014](#B21)). This effect is mediated by transcriptional regulators, such as STAT-3 and NFκB1, whose activity is controlled by pro-inflammatory cytokine-induced signaling pathways ([Ho and Piquette-Miller, 2006](#B22); [Reuter et al., 2010](#B40)). More recently, nuclear receptors (NRs), another class of transcriptional regulators ([De Mattia et al., 2013b](#B14), [2016](#B11); [Cecchin et al., 2016](#B5)), have emerged as crucial regulators of ADME genes in the presence of cytokines released during the inflammation process ([Chen et al., 2012](#B9); [Cressman et al., 2012](#B10); [De Mattia et al., 2013b](#B14)). Altered transcriptional functionality could impact irinotecan drug pharmacokinetic and pharmacodynamic profiles via the regulation of ADME gene expression. In this context a significant effect on irinotecan exposure and clinical outcome of some genetic variants in *HNF1A* gene was previously reported ([Labriet et al., 2017](#B26)).

The present study addressed the effect of polymorphisms in genes encoding transcriptional regulators and pro-inflammatory cytokines impacting irinotecan-related ADME genes on the risk of toxicity. The aim of the study, adopting a TagSNP approach, was to evaluate the systemic variability of 22 transcriptional regulators and pro-inflammatory cytokines impacting irinotecan-related ADME genes to define novel genetic markers that may improve the prediction of the differential probability of developing neutropenia and gastrointestinal (GI) toxicities after FOLFIRI treatment. A discovery/replication study design was used with data on more than 400 mCRC patients treated with first-line FOLFIRI ([Toffoli et al., 2006](#B43); [Levesque et al., 2013](#B29)).

## Materials and Methods

### Patient Cohorts and Treatment

The discovery cohort included prospectively enrolled Northeastern Italian mCRC patients undergoing first-line FOLFIRI treatment and homogenously followed up between February 2002 and November 2005 ([Toffoli et al., 2006](#B43); [Cecchin et al., 2009](#B6)). Of the 267 subjects enrolled, 250 were found to be eligible and included in the final analysis of toxicity ([Toffoli et al., 2006](#B43)); DNA, clinical, and pharmacokinetic data were collected and described previously ([Toffoli et al., 2006](#B43); [Cecchin et al., 2009](#B6)). Patients were treated with either the Tournigand-modified FOLFIRI regimen ([Tournigand et al., 2004](#B45)) (>90% of total) or the FOLFIRI regimen, both based on a 180 mg/m^2^ intravenous dose of irinotecan. Details were published previously on eligibility criteria and treatment modalities, as well as the procedures for evaluating toxicity and data collection ([Toffoli et al., 2006](#B43)). In the present study, toxicity end-points were grade 3–4 neutropenia and GI (diarrhea, nausea, or vomiting) toxicities, which represent the major irinotecan-related side effects. The worst event recorded during the entire course of chemotherapy was considered. Criteria for therapy delay/discontinuation were reported previously ([Toffoli et al., 2006](#B43)).

The replication cohort included 167 Eastern Canadian mCRC patients receiving FOLFIRI-based regimens. All patients received a 180 mg/m^2^ intravenous dose of irinotecan every 2 weeks, and 75 patients also received co-treatments: bevacizumab, an experimental drug, or a placebo. Details on eligibility, treatment modalities, and toxicity data collection were previously documented elsewhere ([Levesque et al., 2013](#B29); [Chen et al., 2015a](#B7),[b](#B8)).

In both cohorts, the severity of neutropenia and GI toxicities were evaluated prospectively and according to the National Cancer Institute Common Terminology Criteria for Adverse Events, version 3.0, criteria.

All the patients in the study were self-reported Caucasian. The study protocol complied with the ethical guidelines of the 1975 Declaration of Helsinki. The protocol was approved by the Comitato Etico Indipendente-Centro di Riferimento Oncologico di Aviano and the CHU de Quebec Ethics Committees. All patients provided written informed consent for the genetic analysis before entering the study. All experiments were carried out in accordance with the relevant guidelines and regulations of the CHU de Québec and Centro di Riferimento Oncologico di Aviano.

### Candidate Genes and Polymorphism Selection

Target genes were selected initially on the basis of a literature search (PubMed-MEDLINE) prioritizing transcriptional controllers and cytokines clearly implicated in the regulation of transporters and phase I and II enzymes during inflammation. Particular attention was paid to the modulation of membrane carriers (i.e., ABCB1, ABCC1, ABCC2, ABCG2, and SLCO1B1) and metabolic proteins (i.e., UGT1A, CES, and CYPs) strictly involved in the FOLFIRI drug pathway. Genetic variants for each candidate gene were chosen successively using the TagSNP approach and the genotype frequency data downloaded from the HapMap website^1^; the filter parameters were HapMap CEU database (release #27) and MAF ≥ 0.05. This search permitted to obtain records about variants located in the exonic and intronic regions of the genes. The Tagger program implemented in Haploview^2^ (Broad Institute, Cambridge, MA, United States) was then employed to predict the TagSNPs (*r*^2^ = 0.80). For each block of linkage polymorphisms, a TagSNP was picked. If the initially selected TagSNPs were not amenable to genotype assay development (see section “Genetic analysis” below), an alternative TagSNP that captured the same information was chosen. Polymorphisms that tagged only themselves and did not have a substitute tag were rejected. Finally, the panel of selected TagSNPs was integrated with additional variants located in the 5′- and 3′ untranslated region of the gene chosen by screening the NCBI dbSNP database^3^ (Genomic Build: hg19/GRCh37 [Feb 2009]) using the following criteria: PubMed citation and MAF ≥ 0.05 in HapMap CEU population. At the end of this bioinformatics workflow, a set of 246 molecular markers in 22 candidate genes encoding NRs (PXR, LXR-A/B, FXR, RXR-A/B/G, CAR, VDR, PPAR-A/G/D, HNF4A, and HNF1A), transcription factors and related pathways (STAT-3, NF-kB1, IKBKB, and CHUK), and key pro-inflammatory cytokines (TNF, IL-1B, IL-6, and IFNγ), were selected (Supplementary Table [S1](#SM1)) and introduced into the pharmacogenetic analysis.

### Genetic Analysis

Genomic DNA was extracted from peripheral blood using the High Pure PCR Template Preparation Kit (Roche Diagnostics GmbH, Mannheim, Germany). DNA samples were genotyped using the Illumina BeadXpress platform based on Golden Gate chemistry and the allelic discrimination method based on the TaqMan system. A 192-plex and 48-plex Illumina VeraCode GoldenGate Genotyping Assay (Illumina, Inc., San Diego, CA, United States) was developed using the Assay Design Tool (ADT) available through Technical Support on the Illumina website^4^. Only the assays with a high final score (≥0.6) and optimal designability (=1) were considered compatible with successful GoldenGate genotyping and introduced into the final custom panel. Samples were prepared for the analysis according to the manufacturer’s protocol. VeraScan software (version 2.0) was employed for fluorescence detection and the GenomeStudio V2011.1 tool (Illumina, Inc.) for genotype clustering with a polymorphism call-threshold of 0.25 (on a scale of 0–1). Sample replicates were introduced into each analysis to assess the robustness of the output records and to provide duplicate data to aid in the redefinition of clustering. Only the DNA samples and polymorphisms with a call rate >90% were retained in the final report. The excluded markers and six residual polymorphisms of the selected pool were tested in an allelic discrimination reaction using predesigned TaqMan SNP genotyping assays. All commercial TaqMan assays were purchased from Applied Biosystems^5^ and the analyses performed using the Applera TaqMan Universal Master Mix on an ABI 7500 (AB Applied Biosystems, Foster City, CA, United States) according to the manufacturer’s instructions. Positive and negative control samples were included in each analysis. More details about the analytical procedures are available upon request.

Polymorphisms to be tested in the replication cohort were genotyped by the Canadian group using iPLEX matrix-assisted laser desorption/ionization time-of-flight mass spectrometry (Sequenom, San Diego, CA, United States). Negative controls and a 5% random sample duplicate population were used to ensure the robustness and reproducibility of the assay. All extension primers and PCR assays were designed using SpectroDESIGNER software (Sequenom, San Diego, CA, United States). Markers that could not be sequenced due to poor primer design or because they were located in duplicated regions were replaced with TagSNPs in complete linkage disequilibrium (LD) (*r*^2^ = 1.00).

### Pharmacokinetic Analysis

Pharmacokinetic data were available for 71 patients in the discovery cohort ([Toffoli et al., 2006](#B43)). The pharmacokinetic parameters of irinotecan, SN-38 (active form of irinotecan), SN-38G (inactive glucuronidation form of irinotecan), the GR, and BI were determined as reported previously ([Toffoli et al., 2006](#B43)). The GR was defined as the ratio of the SN38G AUC over the SN38 AUC. The BI was defined as the product of the irinotecan AUC and the ratio of the SN38 AUC over the SN38G AUC.

### Statistical Analysis

The overall study design is depicted in **Figure 1**. The analysis was carried out in three steps. The first step consisted of the selection of potential markers of severe toxicity (*p* < 0.05) in the discovery cohort. In the second step, these selected polymorphisms were independently tested in an independent cohort in order to find concordant associations (*p* < 0.05). Markers with a non-significant (*p* > 0.05) concordant effect (same genetic model, same effect) in both cohorts were still considered for possible association with irinotecan pharmacokinetic parameters, in a subgroup of patients from the discovery cohort.

### FIGURE 1.

![FIGURE 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/aee9/5908896/2684b9349e76/fphar-09-00367-g001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5908896_fphar-09-00367-g001.jpg)

Flow chart summarizing the study design. More details are provided in the text. Abbreviations: BV, bevacizumab; G3–G4, grade 3–4; GI, gastrointestinal; mCRC, metastatic colorectal cancer; tagSNP, tagging polymorphism.

Associations between genotype markers and outcome measures of FOLFIRI toxicity were assessed by multivariate logistic regression. Dominant, recessive, and additive genetic models were considered for each polymorphism by combining heterozygous and homozygous genotypes; the best-fitting genetic model was selected according to the Wald chi-square test. ORs and the corresponding 95% CIs were calculated and adjusted for the patients’ clinical and demographic characteristics available for each dataset (gender, age, first tumor site, stage at diagnosis, radical surgery, and adjuvant chemotherapy for the discovery cohort; gender, age, and co-treatment for the replication cohort). In all cases, significance was claimed for *p* < 0.05 (two-sided). To assess the effect of the multiple testing in the discovery cohort, a *q*-value [False discovery rate (FDR)-adjusted *p*-value] was evaluated ([Benjamini and Hochberg, 1995](#B3)). The distribution of pharmacokinetic parameters for irinotecan, SN-38, SN-38G, the BI, and GR according to genotype were compared by the Kruskal–Wallis test. All analyses were carried out in Stata 12.1.

## Results

### Genotyping

Genotyping was successful in 224/240 assays. BeadXpress technology failed at the analysis of 16 markers, which were successfully tested using the TaqMan method, as well as an additional six polymorphisms not included in the BeadXpress panel. The average genotype call rate was 0.99 (range: 0.90–1.00). Three of 250 samples were excluded from the study because they did not reach the fixed call rate threshold of 90%, probably due to low DNA quality; thus, genotype data were available for 247 patients that constituted the discovery set. Replicated samples included in the analyses presented an average concordance rate of 100%. Three randomly selected polymorphisms were used for analytical validation of the genotyping data using direct Sanger sequencing. Fifty samples were sequenced for *PXR* rs3732359, 49 for *RXRA* rs10881582, and 89 for *VDR* rs4237855. All of these markers had a concordance rate of 100%.

Analysis of the markers to be tested in the replication set, using the Sequenom platform was designed successfully for all candidate polymorphisms except rs880663, rs3093662, rs2744537, rs1800629, rs1800630, and rs2069840. All 167 samples constituting the replication set were positively genotyped; the average genotype call rate was 0.98 (range: 0.94–1.00).

### Patient Characteristics

The main demographic and clinical characteristics of the two study populations (discovery and replication cohorts) are reported in **Table 1**. All 247 genotyped patients were included in the cumulative toxicity evaluation (discovery cohort). The details of the toxicity assessment were previously reported ([Toffoli et al., 2006](#B43)). Severe neutropenia (grade 3–4) was present in 35 cases (14.2%) and corresponded to the most frequent severe toxicity. Diarrhea, nausea, and/or vomiting of high grade (grade 3–4) occurred in 26 cases (10.5%) and represented the predominant non-hematological toxicities. Among the 167 patients in the replication group, severe neutropenia was reported in 28 (16.8%), whereas grade 3–4 GI side effects were reported in 24 (14.4%).

### Table 1.

|   | Total (n = 414) | Italian cohort∗ (n = 247) Discovery set | Canadian cohort (n = 167) Replication set |
| --- | --- | --- | --- |
|   |   |  |  |
| Characteristic | N (%) | N (%) | N (%) |
| Gender |   |   |   |
| Male | 270 (65.2) | 160 (64.8) | 110 (65.87) |
| Female | 144 (34.8) | 87 (35.2) | 57 (34.13) |
| Age (years) |   |   |   |
| Median | 62.9 | 63.5 | 62 |
| Primary tumor site |  |  |  |
| Colon | 298 (72.0) | 176 (71.3) | 122 (73.1) |
| Right |   | 78 (44.3) | n.a |
| Left |   | 98 (55.7) | n.a |
| Rectum | 113 (27.3) | 71 (28.7) | 42 (25.1) |
| Unknown | 3 (0.7) | – | 3 (1.8) |
| Regimen |   |   |   |
| FOLFIRI | 414 (100.0) | 247 (100) | 167 (100) |
| Co-treatment |  |  |  |
| Bevacizumab |   | 0 | 69 (41.3) |
| Other drug |   | 0 | 6 (3.6) |
| Toxicity |   |   |   |
| Neutropenia (grade 3–4) | 63 (15.2) | 35 (14.2) | 28 (16.8) |
| Gastrointestinal (grade 3–4) | 50 (12.1) | 26 (10.5) | 24 (14.4) |
|   |  |  |  |

Table 1 Caption: Demographic and clinical characteristics of the patients included in the study (discovery and replication cohort).

### Markers of Neutropenia

The association of each polymorphism with the occurrence of grade 3–4 neutropenia was tested by logistic regression analysis and the results summarized in **Table 2**. In the discovery cohort, 18 genetic variations in genes encoding four NRs (HNF4α, PXR, PPARs, and VDR), one transcription factor (NF-κB), and one cytokine (TNF) emerged as significant (*p* < 0.05) predictors of severe neutropenia over the entire course of chemotherapy. The genotype distribution based on neutropenia grade is reported in Supplementary Table [S2](#SM2); the MAFs of these polymorphisms were checked and found to be in line with the data reported for the Caucasian population^6^. Of the 18 markers identified, 10 were associated with an increased risk of developing grade 3–4 neutropenia, with ORs ranging from 1.68 to 16.08, and the remaining eight were indicated to have a protective effect against the development of toxicity, with ORs ranging from 0.16 to 0.60. Among the 18 markers associated with neutropenia at *p* < 0.05 in the discovery cohort, the *q*-value ranged from 0.181 to 0.505.

### Table 2.

| Discovery set |  |  |  |  |  | Replication set |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- |
|   |  |  |  |  |  |   |  |  |
| Genes | SNP | Base Change | Model | OR (95% CI)a | p-value | Model | OR (95% CI)b | p-value |
| HNF4A | rs2425637 | G > T | Additive | 1.88 (1.05–3.38) | 0.035 | Dominant | 1.51 (0.53–4.27) | 0.442 |
| HNF4A | rs3212183 | T > C | Recessive | 2.50 (1.11–5.56) | 0.026 | Additive | 1.20 (0.65–2.21) | 0.558 |
| HNF4A | rs3212197 | C > T | Additive | 2.29 (1.07–4.91) | 0.033 | Dominant | 0.47 (0.09–2.35) | 0.357 |
| HNF4A | rs6093976 | C > T | Dominant | 0.16 (0.05–0.56) | 0.004 | Dominant | 0.67 (0.27–1.64) | 0.379 |
| HNF4A | rs6093978 | C > T | Dominant | 0.41 (0.18–0.93) | 0.033 | Recessive | 1.83 (0.59–5.68) | 0.292 |
| HNF4A | rs6130615 | C > T | Recessive | 16.08 (2.89–89.62) | 0.002 | Additive | 0.70 (0.27–1.79) | 0.457 |
| HNF4A | rs745975 | G > A | Dominant | 0.33 (0.13–0.85) | 0.021 | Dominant | 0.68 (0.29–1.61) | 0.380 |
| HNF4A | rs2425640 | G > A | Additive | 0.60 (0.36–0.98) | 0.042 | Recessive | 3.35 (0.82–13.66) | 0.092 |
| NR1I2 | rs16830505 | A > G | Dominant | 2.33 (1.01–5.34) | 0.046 | Additive | 1.87 (0.83–4.23) | 0.132 |
| NR1I2 | rs7643645 | A > G | Dominant | 0.28 (0.13–0.61) | 0.001 | Recessive | 3.01 (1.01–9.01) | 0.049 |
| PPARD | rs2076169 | T > C | Recessive | 8.99 (1.31–61.85) | 0.026 | Dominant | 0.18 (0.02–1.42) | 0.103 |
| PPARG | rs2972164 | T > C | Dominant | 0.36 (0.16–0.82) | 0.015 | Dominant | 1.34 (0.49–3.69) | 0.565 |
| PPARG | rs880663 | T > C | Additive | 0.36 (0.16–0.82) | 0.015 | – | – | – |
| NFKB1 | rs230539 | A > G | Additive | 1.68 (1.02–2.78) | 0.043 | Recessive | 0.42 (0.09–1.87) | 0.254 |
| TNF | rs3093662 | A > G | Dominant | 2.56 (1.10–5.96) | 0.029 | – | – | – |
| VDR | rs11168287 | A > G | Dominant | 3.12 (1.02–9.56) | 0.046 | Recessive | 1.98 (0.73–5.43) | 0.182 |
| VDR | rs11574026 | C > T | Dominant | 2.36 (1.06–5.23) | 0.035 | Recessive | 1.55 (0.63–3.77) | 0.339 |
| VDR | rs12717991 | G > A | Dominant | 0.36 (0.16–0.82) | 0.015 | Dominant | 0.84 (0.35–2.00) | 0.695 |
|   |  |  |  |  |  |  |  |  |

Table 2 Caption: Odds ratios (ORs) and 95% confidence interval (CI) for grade 3–4 vs. grade 0–2 cumulative neutropenia in the discovery (n = 247 mCRC patients) and replication (n = 167 mCRC patients) cohorts according to gene polymorphisms (SNPs).

None of these associations was significant in the replication set (*p* > 0.05).

### Markers of GI Toxicity

The association of each polymorphism with the occurrence of grade 3–4 GI toxicity was tested by logistic regression analysis and the data reported in **Table 3**. Twenty-two polymorphic variants in genes encoding five NRs (HNF4α, CAR, PPARs, RXRs, and VDR), one transcription factor (STAT-3), and three cytokines (TNF, IFN-γ, and IL-6) were significantly (*p* < 0.05) associated with the risk of developing severe GI toxicity during the entire course of chemotherapy. The genotype distribution by GI toxicity grade is reported in Supplementary Table [S3](#SM3); the MAFs of these polymorphisms were in line with the data reported for the Caucasian population (see foot note text 6). Of the 22 markers identified, 14 were associated with an increased chance of having grade 3–4 GI toxicity, with ORs ranging from 1.72 to 20.74, and the remaining eight correlated with an inferior risk of severe GI toxicity, with ORs ranging from 0.12 to 0.51. Among the 22 markers associated with GI toxicity at *p* < 0.05 in the discovery cohort, the *q*-value ranged from 0.244 to 0.521.

### Table 3.

| Discovery set |  |  |  |  |  | Replication set |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- |
|   |  |  |  |  |  |   |  |  |
| Genes | SNP | Base Change | Model | OR (95% CI)a | p-value | Model | OR (95% CI)b | p-value |
| HNF4A | rs1800961 | C > T | Dominant | 11.51 (2.02–65.61) | 0.006 | Dominant | 3.27 (0.57–18.69) | 0.182 |
| HNF4A | rs2071197 | G > A | Dominant | 4.42 (1.75–11.17) | 0.002 | Dominant | 1.43 (0.45–4.58) | 0.546 |
| HNF4A | rs6031587 | C > T | Additive | 3.24 (1.40–7.51) | 0.006 | Additive | 2.21 (0.86–5.70) | 0.100 |
| HNF4A | rs6093976 | C > T | Additive | 0.36 (0.13–0.98) | 0.046 | Recessive | 1.92 (0.35–10.65) | 0.457 |
| NR1I3 | rs2307424 | C > T | Dominant | 3.38 (1.23–9.27) | 0.018 | Additive | 0.66 (0.35–1.24) | 0.196 |
| NR1I3 | rs4073054 | T > G | Recessive | 0.14 (0.02–0.92) | 0.041 | Dominant | 0.63 (0.26–1.50) | 0.293 |
| PPARA | rs9626736 | A > G | Additive | 2.14 (1.14–4.00) | 0.018 | Dominant | 1.90 (0.71–5.08) | 0.203 |
| PPARD | rs2076169 | T > C | Recessive | 20.74 (2.88–149.18) | 0.003 | Recessive | 6.26 (0.36–109.01) | 0.208 |
| RXRG | rs380518 | T > C | Dominant | 4.69 (1.73–12.70) | 0.002 | Dominant | 2.15 (0.82–5.66) | 0.120 |
| RXRG | rs4657437 | C > A | Additive | 0.48 (0.24–0.99) | 0.047 | Dominant | 2.54 (0.90–7.22) | 0.080 |
| RXRG | rs283695 | G > A | Additive | 1.80 (1.04–3.12) | 0.036 | Recessive | 0.43 (0.10–1.98) | 0.282 |
| RXRG | rs3767344 | G > C | Dominant | 0.35 (0.13–0.93) | 0.034 | Dominant | 0.75 (0.28–2.03) | 0.574 |
| RXRG | rs157880 | C > T | Recessive | 11.70 (20.03–67.58) | 0.006 | Dominant | 1.44 (0.58–3.59) | 0.435 |
| RXRB | rs2744537 | G > T | Recessive | 5.34 (1.42–20.07) | 0.013 | – | – | – |
| VDR | rs11574077 | A > G | Additive | 4.46 (1.43–13.96) | 0.010 | Additive | 1.44 (0.37–5.63) | 0.601 |
| VDR | rs4760648 | C > T | Additive | 2.09 (1.13–3.84) | 0.018 | Additive | 0.71 (0.36–1.42) | 0.338 |
| VDR | rs2853564 | T > C | Additive | 0.38 (0.18–0.78) | 0.008 | Additive | 1.42 (0.72–2.78) | 0.310 |
| TNF | rs1800629 | A > G | Dominant | 4.07 (1.13–14.72) | 0.032 | – | – | – |
| TNF | rs1800630 | C > A | Additive | 1.72 (1.01–2.94) | 0.047 | – | – | – |
| STAT3 | rs1053004 | T > C | Additive | 0.51 (0.27–0.99) | 0.045 | Dominant | 0.39 (0.15–0.97) | 0.043 |
| INFG | rs2069716 | A > G | Dominant | 0.12 (0.01–0.97) | 0.047 | Dominant | 0.88 (0.20–3.88) | 0.862 |
| IL6 | rs2069840 | C > G | Dominant | 0.40 (0.17–0.95) | 0.039 | – | – | – |
|   |  |  |  |  |  |  |  |  |

Table 3 Caption: Odds ratios (ORs) and 95% confidence interval (CI) for grade 3–4 vs. grade 0–2 cumulative gastrointestinal toxicity in the discovery (n = 247 mCRC patients) and replication (n = 167 mCRC patients) cohort according to gene polymorphisms (SNPs).

One *STAT-3* marker (rs1053004) presented a concordant significant effect in the replication cohort (*p* < 0.05) as a protective factor against the development of grade 3–4 GI toxicity. The C allele at *STAT-3* rs1053004 was significantly associated with a decreased risk of toxicity in the discovery cohort (OR = 0.51, *p* = 0.045, *q* = 0.521) according to an additive model. Similarly, *STAT-3* rs1053004 C allele exerted a protective effect against the development of toxicity in the replication cohort (OR = 0.39, *p* = 0.043, *q* = 0.305) in a dominant model. None of the other associations between genetic markers and the risk of severe GI toxicity found in the discovery cohort were significant in the replication cohort (*p* > 0.05).

### Pharmacokinetics

The pharmacokinetic analysis focused on markers that, even if not presenting a significant effect in the replication cohort (*p* > 0.05), presented a concordant effect on the toxicity risk in both cohorts (same size effect according to the same genetic model, **Tables 2**, **3**). The association of these polymorphisms with the pharmacokinetic parameters was investigated in a subset of 71 patients from the discovery cohort, and the most relevant results (*p* < 0.1; concordant genetic model) summarized in Supplementary Table [S4](#SM4). *VDR* rs11574077 was associated to an inferior GR (*p* = 0.012) and an increased BI (*p* = 0.036) according to an additive model (**Figure 2**). None of the four patients harboring the rs11574077-G variant allele were homozygous for *UGT1A∗*28 polymorphism, excluding a potential confounding effect. The pharmacokinetic correlations with the other polymorphisms analyzed were not significant but still presented an effect consistent with that reported on the toxicity risk. *HNF4A* rs6031587 correlated with an increased AUC for irinotecan (*p* = 0.091), whereas *VDR* rs12717991 (*p* = 0.073) and *RXRG* rs3767344 (*p* = 0.078) were associated with an increased GR consistent with the hypothesized protective effect against the development of toxicity.

### FIGURE 2.

![FIGURE 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/aee9/5908896/88c15adb4520/fphar-09-00367-g002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5908896_fphar-09-00367-g002.jpg)

(Top) An histogram plot displays the percentage of patients experiencing mild (grade 0–2) or severe (grade 3–4) GI toxicity, according to VDR rs11574077 genotype in the discovery and replication sets. (Bottom) The biliary index and glucuronidation ratio values for each patient, according to VDR rs11574077 genotype, are shown. The analysis has been performed on 71 patients from the discovery set. Each dot represents a patient. Abbreviations: 95%CI, 95% confidence intervals; AUC, area under the curve; BI, biliary index [CPT-11 AUC X (SN38 AUC/SN38G AUC)]; CPT-11, irinotecan; GI, gastrointestinal toxicity; GR, glucuronidation ratio (SN-38G AUC/SN-38 AUC); OR, Odds ratios; SN-38, active form of irinotecan; PK, pharmacokinetic; SN-38G, inactive glucuronidation form of irinotecan.

## Discussion

Despite the introduction of targeted agents in the pharmacological treatment of mCRC, combinatorial chemotherapy regimens are still the backbone of each therapeutic scheme, burdening the patients with unpredictable adverse events. Neutropenia and GI toxicity are typically associated with irinotecan-containing regimens, such as FOLFIRI ([Gupta et al., 1994](#B19), [1997](#B20)). In the last few years, several pharmacogenetic studies have sought to identify predictive genetic markers impacting irinotecan ADME genes and, consequently, its pharmacokinetics and pharmacodynamics in order to help clinicians personalize irinotecan-based therapy. Despite these efforts, the only validated predictor of severe toxicity is currently *UGT1A1*^∗^28 polymorphism, adopted by international guidelines for adjusting the irinotecan dose ([Swen et al., 2011](#B42); [Caudle et al., 2013](#B4)), which does not completely address the issue. Additional work is required to define further molecular markers and better identify patients at risk for severe complications. A more recent field of investigation is the contribution of genetic variability in specific transcriptional regulators encoding genes to the impact on the control of ADME gene expression ([Ho and Piquette-Miller, 2006](#B22); [Reuter et al., 2010](#B40); [De Mattia et al., 2013b](#B14), [2016](#B11); [Cecchin et al., 2016](#B5)). The main findings of the present study were the identification of *STAT-3* rs1053004 polymorphism as a promising marker of grade 3–4 GI toxicity, and of *VDR* rs11574077 as a marker of irinotecan pharmacokinetics with a consistent effect on grade 3–4 GI toxicity risk. An independent cohort of FOLFIRI-treated mCRC patients was adopted to independently replicate these results.

The IL-6/STAT-3 cascade has been indirectly implicated in the regulation of key irinotecan pathway-related proteins, such as cytochromes (i.e., CYP3A4) ([Jover et al., 2002](#B23)) and ABC/SLC transporters (i.e., MDR-1) ([Duan et al., 2006](#B15); [Ho and Piquette-Miller, 2006](#B22); [Andrejko et al., 2008](#B2); [Zhang et al., 2011](#B48); [Zhu et al., 2012](#B49)), which play a crucial role in drug disposition. In this respect, STAT-3 could mediate the impact of CRC-associated inflammation ([Markman and Shiao, 2015](#B31)) on irinotecan-related ADME gene expression, affecting the toxicity profile ([Chen et al., 2012](#B9); [Cressman et al., 2012](#B10); [Harvey and Morgan, 2014](#B21)). STAT-3 is triggered in response to the binding of numerous pro-inflammatory cytokines and represents the critical factor in IL-6-induced gene control ([Aggarwal et al., 2009](#B1); [Qi and Yang, 2014](#B39); [Yu et al., 2014](#B47)). Once activated, STAT-3 transduces the signal across the cytoplasm and into the nucleus, where it mediates gene transcription, often in cooperation with NF-κB ([Aggarwal et al., 2009](#B1); [Qi and Yang, 2014](#B39); [Yu et al., 2014](#B47)). The *STAT-3* rs1053004 polymorphism located in the 3′ untranslated region of the gene in a putative binding site for miR-423-5p has been reported to regulate the expression of STAT-3 protein, probably by altering mRNA degradation, and to significantly affect its transcriptional activity ([Permuth-Wey et al., 2016](#B37)). The altered STAT-3 activity associated with this polymorphic variant could potentially contribute to altering the bioavailability of irinotecan and, consequently, the individual predisposition to experience severe GI side effects. As a mediator of cytokine signaling, STAT-3 could also be directly involved in the complex mechanism underlying the toxic damage to the GI epithelium (i.e., mucositis) induced by some chemotherapeutics, including irinotecan. The pathobiology of drug-related mucositis involves the mucosal immune system, with pro-inflammatory cytokine release playing an important role ([Logan et al., 2008](#B30); [Lee et al., 2014](#B28)). In particular, this mechanism has been implicated in the onset of intestinal injury subsequent to the administration of irinotecan ([Logan et al., 2008](#B30); [Melo et al., 2008](#B33)), as well 5-fluorouracil, the other drug included in the FOLFIRI regimen ([Lee et al., 2014](#B28)). Thus, IL-6/STAT-3 signaling could contribute to FOLFIRI-related mucosal damage by controlling the proliferation and survival of intestinal epithelial cells as observed in colitis ([Moriasi et al., 2012](#B34); [Nguyen et al., 2015](#B36)). Furthermore, an interaction between the IL-6/STAT-3 cascade and the gut microbioma, which contributes to the local accumulation of the cytotoxic active metabolite SN-38 ([Mathijssen et al., 2001](#B32)), has been described ([Fichtner-Feigl et al., 2015](#B16); [Jurjus et al., 2016](#B24)). Therefore, a change in the STAT-3 activity could significantly alter the mediation of pro-inflammatory cytokines, such as IL-6, IL-1β, and TNF-α, modifying the susceptibility of the GI mucosa to the toxic effect of irinotecan-based treatment as shown in the present study. A significant effect of some germ line genetic variations in these cytokines (i.e., IL-6, and TNFα) was demonstrated in the discovery cohort (**Table 3**). Unfortunately, the missing genotype data for the replication cohort did not permit to independently replicate their effect.

Concerning *VDR* rs11574077 polymorphism, it was found to significantly correlate with an increased BI and inferior GR, parameters indicating lower efficacy of the glucuronidation and detoxification pathways. This functional effect of the polymorphism on the pharmacokinetics of irinotecan is consistent with its detrimental impact on the occurrence of grade 3–4 GI toxicity, though it was not significantly replicated in the Canadian cohort, possibly due to the lower number of patients in this group and the low MAF of the polymorphism. Beyond its physiological role in calcium and phosphate homeostasis, VDR has been demonstrated to cooperate in the transcriptional regulation of ADME genes (i.e., *CYPs, UGT1As, ABC/SLC* transporters) ([Prakash et al., 2015](#B38)), possibly affecting the irinotecan disposition profile. Thus, *VDR* rs11574077 represents an intronic variant of unknown functional significance, although it was reported to impact the risk of developing some tumors and cardiovascular disease ([Verschuren et al., 2012](#B46); [Muindi et al., 2013](#B35)). Though the exact molecular mechanism of *VDR* expression/activity regulation by rs1799794 variant is currently unclear, and the hypothesis that its effect could be due to linkage disequilibrium with another functional marker cannot be excluded. However, the potentially altered VDR activity due to the polymorphism could influence drug bioavailability with significant consequences on the irinotecan-based modulation of GI toxicity.

Some of the genetic markers selected in the first step of the statistical analysis in the discovery cohort were not confirmed in the replication cohort. Other markers presented a concordant, but not significant, effect in the replication cohort (**Tables 2**, **3**). It must be kept in mind that some inhomogeneity existed between the two cohorts and we cannot exclude an ethnicity-specific effect of some of the analyzed genetic variants. For these markers, additional replication studies will be required to elucidate their possible role in determining the irinotecan toxicity risk.

Some limitations of the present study need to be considered. Firstly, the FOLFIRI regimen is no longer the standard therapy for mCRC patients, but it is still given in combination with monoclonal antibodies. A subset of the patients in the replication cohort was treated with a combination of FOLFIRI and bevacizumab, supporting a predictive effect of the markers in such poly-chemotherapy-treated patients. Secondly, the functional meanings of the markers highlighted in the present study are unknown. Formal functional analyses should be performed in order to better understand the molecular mechanism underlying the observed associations, but the concordant pharmacokinetic effect found for some of the highlighted markers supports a functional effect of the variant on the efficiency of the enzymatic activity of the encoded protein. Thirdly, the present study focused only on common genetic variants with a MAF ≥ 0.05. As recently pointed out, rarer genetic variants could account for a high percentage of inter-individual variability in drug metabolism, including NR genes ([Lauschke and Ingelman-Sundberg, 2016](#B27); [Kozyra et al., 2017](#B25)), and for the observed inter-individual heterogeneity in drug toxicity and pharmacokinetics. Therefore, future pharmacogenetic approaches should include these emerging markers in order to better describe patient phenotypes regarding the response to pharmacological treatment. It must be noticed that, when controlling the analysis for multiple testing, all the associations were above 18.1% of FDR, pointing out that the study results must be considered only as hypothesis-generating. It is nonetheless acknowledged that replicating a significant association in an independent set of patients, as in the present study, strengthens the reliability of the data and the interest to further clarify their potential clinical implication.

## Conclusion

The present study suggested for the first time that the *STAT-3* rs1053004 polymorphism could have an effect on the risk of severe GI toxicity after irinotecan therapy (FOLFIRI regimen) in mCRC patients. This finding further highlights the importance of inflammatory response mediators in the pathobiology of drug-induced mucosal injury, a topic that requires more attention in pharmacogenetic investigations. *VDR* rs11574077 also emerged as a promising novel determinant of irinotecan pharmacokinetic parameters and possibly of FOLFIRI-related GI toxicity risk, soliciting future research efforts in this direction. The discovery of novel factors contributing to the individual predisposition to the development of severe GI toxicity after chemotherapy is of great interest considering the increased recognition of the clinical and economic implications of the mucosal damage induced by treatment ([Gibson et al., 2015](#B18)). These findings could help elucidate the molecular mechanism underlying irinotecan-induced epithelial injury and, if validated, could be considered as additional criteria to improve the clinical management of FOLFIRI-related GI toxicity.

## Author Contributions

EDM and EC contributed equally to designing the study, writing the main manuscript, and elaborating the tables and figures. MM was involved in the statistical analysis and interpretation of data. AL participated in the molecular analysis for replication cohort. CG participated in the enrollment of the replication patient group and the collection of clinical and genotyping data on this set. ED, RR, and AlB participated in the molecular analysis for the discovery cohort. AnB, MD’A, LC, and SL participated in the enrollment of the discovery patient group and the collection of clinical data. EL, DJ, and FC participated in the design, enrolment of the replication cohort, and the collection of clinical data. GT was the guarantor. All authors reviewed the manuscript.

## Conflict of Interest Statement

The authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.

## Acknowledgments

The authors wish to thank Dr. Alessandro Fornasier, who developed the genetic data management software used in the present research.

## Abbreviations

## Supplementary Material

The Supplementary Material for this article can be found online at: [https://www.frontiersin.org/articles/10.3389/fphar.2018.00367/full#supplementary-material](https://www.frontiersin.org/articles/10.3389/fphar.2018.00367/full#supplementary-material)

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Aggarwal B. B., Kunnumakkara A. B., Harikumar K. B., Gupta S. R., Tharakan S. T., Koca C., et al. (2009). Signal transducer and activator of transcription-3, inflammation, and cancer: how intimate is the relationship? Ann. N. Y. Acad. Sci. 1171 59–76. 10.1111/j.1749-6632.2009.04911.x  [DOI](https://doi.org/10.1111/j.1749-6632.2009.04911.x) | [PMC free article](/articles/PMC3141289/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19723038/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann.%20N.%20Y.%20Acad.%20Sci.&title=Signal%20transducer%20and%20activator%20of%20transcription-3,%20inflammation,%20and%20cancer:%20how%20intimate%20is%20the%20relationship?&author=B.%20B.%20Aggarwal&author=A.%20B.%20Kunnumakkara&author=K.%20B.%20Harikumar&author=S.%20R.%20Gupta&author=S.%20T.%20Tharakan&volume=1171&publication_year=2009&pages=59-76&pmid=19723038&doi=10.1111/j.1749-6632.2009.04911.x&)

2. Andrejko K. M., Raj N. R., Kim P. K., Cereda M., Deutschman C. S. (2008). IL-6 modulates sepsis-induced decreases in transcription of hepatic organic anion and bile acid transporters. Shock 29 490–496. 10.1097/shk.0b013e318150762b  [DOI](https://doi.org/10.1097/shk.0b013e318150762b) | [PMC free article](/articles/PMC2667679/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17724432/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Shock&title=IL-6%20modulates%20sepsis-induced%20decreases%20in%20transcription%20of%20hepatic%20organic%20anion%20and%20bile%20acid%20transporters.&author=K.%20M.%20Andrejko&author=N.%20R.%20Raj&author=P.%20K.%20Kim&author=M.%20Cereda&author=C.%20S.%20Deutschman&volume=29&publication_year=2008&pages=490-496&pmid=17724432&doi=10.1097/shk.0b013e318150762b&)

3. Benjamini Y., Hochberg Y. (1995). Controlling the false discovery rate: a practical and powerful approach to multiple testing. J. R. Stat. Soc. Series B Stat. Methodol. 57 289–300.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20R.%20Stat.%20Soc.%20Series%20B%20Stat.%20Methodol.&title=Controlling%20the%20false%20discovery%20rate:%20a%20practical%20and%20powerful%20approach%20to%20multiple%20testing.&author=Y.%20Benjamini&author=Y.%20Hochberg&volume=57&publication_year=1995&pages=289-300&)

4. Caudle K. E., Thorn C. F., Klein T. E., Swen J. J., McLeod H. L., Diasio R. B., et al. (2013). Clinical Pharmacogenetics Implementation Consortium guidelines for dihydropyrimidine dehydrogenase genotype and fluoropyrimidine dosing. Clin. Pharmacol. Ther. 94 640–645. 10.1038/clpt.2013.172  [DOI](https://doi.org/10.1038/clpt.2013.172) | [PMC free article](/articles/PMC3831181/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23988873/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Clinical%20Pharmacogenetics%20Implementation%20Consortium%20guidelines%20for%20dihydropyrimidine%20dehydrogenase%20genotype%20and%20fluoropyrimidine%20dosing.&author=K.%20E.%20Caudle&author=C.%20F.%20Thorn&author=T.%20E.%20Klein&author=J.%20J.%20Swen&author=H.%20L.%20McLeod&volume=94&publication_year=2013&pages=640-645&pmid=23988873&doi=10.1038/clpt.2013.172&)

5. Cecchin E., De Mattia E., Toffoli G. (2016). Nuclear receptors and drug metabolism for the personalization of cancer therapy. Expert. Opin. Drug Metab. Toxicol. 12 291–306. 10.1517/17425255.2016.1141196  [DOI](https://doi.org/10.1517/17425255.2016.1141196) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26789519/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Expert.%20Opin.%20Drug%20Metab.%20Toxicol.&title=Nuclear%20receptors%20and%20drug%20metabolism%20for%20the%20personalization%20of%20cancer%20therapy.&author=E.%20Cecchin&author=E.%20De%20Mattia&author=G.%20Toffoli&volume=12&publication_year=2016&pages=291-306&pmid=26789519&doi=10.1517/17425255.2016.1141196&)

6. Cecchin E., Innocenti F., D’Andrea M., Corona G., De Mattia E., Biason P., et al. (2009). Predictive role of the UGT1A1, UGT1A7, and UGT1A9 genetic variants and their haplotypes on the outcome of metastatic colorectal cancer patients treated with fluorouracil, leucovorin, and irinotecan. J. Clin. Oncol. 27 2457–2465. 10.1200/JCO.2008.19.0314  [DOI](https://doi.org/10.1200/JCO.2008.19.0314) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19364970/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Oncol.&title=Predictive%20role%20of%20the%20UGT1A1,%20UGT1A7,%20and%20UGT1A9%20genetic%20variants%20and%20their%20haplotypes%20on%20the%20outcome%20of%20metastatic%20colorectal%20cancer%20patients%20treated%20with%20fluorouracil,%20leucovorin,%20and%20irinotecan.&author=E.%20Cecchin&author=F.%20Innocenti&author=M.%20D%E2%80%99Andrea&author=G.%20Corona&author=E.%20De%20Mattia&volume=27&publication_year=2009&pages=2457-2465&pmid=19364970&doi=10.1200/JCO.2008.19.0314&)

7. Chen S., Laverdiere I., Tourancheau A., Jonker D., Couture F., Cecchin E., et al. (2015a). A novel UGT1 marker associated with better tolerance against irinotecan-induced severe neutropenia in metastatic colorectal cancer patients. Pharmacogenomics J. 15 513–520. 10.1038/tpj.2015.12  [DOI](https://doi.org/10.1038/tpj.2015.12) | [PMC free article](/articles/PMC6174029/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25778466/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J.&title=A%20novel%20UGT1%20marker%20associated%20with%20better%20tolerance%20against%20irinotecan-induced%20severe%20neutropenia%20in%20metastatic%20colorectal%20cancer%20patients.&author=S.%20Chen&author=I.%20Laverdiere&author=A.%20Tourancheau&author=D.%20Jonker&author=F.%20Couture&volume=15&publication_year=2015a&pages=513-520&pmid=25778466&doi=10.1038/tpj.2015.12&)

8. Chen S., Villeneuve L., Jonker D., Couture F., Laverdiere I., Cecchin E., et al. (2015b). ABCC5 and ABCG1 polymorphisms predict irinotecan-induced severe toxicity in metastatic colorectal cancer patients. Pharmacogenet. Genomics 25 573–583. 10.1097/FPC.0000000000000168  [DOI](https://doi.org/10.1097/FPC.0000000000000168) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26352872/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet.%20Genomics&title=ABCC5%20and%20ABCG1%20polymorphisms%20predict%20irinotecan-induced%20severe%20toxicity%20in%20metastatic%20colorectal%20cancer%20patients.&author=S.%20Chen&author=L.%20Villeneuve&author=D.%20Jonker&author=F.%20Couture&author=I.%20Laverdiere&volume=25&publication_year=2015b&pages=573-583&pmid=26352872&doi=10.1097/FPC.0000000000000168&)

9. Chen Y., Tang Y., Guo C., Wang J., Boral D., Nie D. (2012). Nuclear receptors in the multidrug resistance through the regulation of drug-metabolizing enzymes and drug transporters. Biochem. Pharmacol. 83 1112–1126. 10.1016/j.bcp.2012.01.030  [DOI](https://doi.org/10.1016/j.bcp.2012.01.030) | [PMC free article](/articles/PMC3339266/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22326308/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem.%20Pharmacol.&title=Nuclear%20receptors%20in%20the%20multidrug%20resistance%20through%20the%20regulation%20of%20drug-metabolizing%20enzymes%20and%20drug%20transporters.&author=Y.%20Chen&author=Y.%20Tang&author=C.%20Guo&author=J.%20Wang&author=D.%20Boral&volume=83&publication_year=2012&pages=1112-1126&pmid=22326308&doi=10.1016/j.bcp.2012.01.030&)

10. Cressman A. M., Petrovic V., Piquette-Miller M. (2012). Inflammation-mediated changes in drug transporter expression/activity: implications for therapeutic drug response. Expert. Rev. Clin. Pharmacol. 5 69–89. 10.1586/ecp.11.66  [DOI](https://doi.org/10.1586/ecp.11.66) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22142160/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Expert.%20Rev.%20Clin.%20Pharmacol.&title=Inflammation-mediated%20changes%20in%20drug%20transporter%20expression/activity:%20implications%20for%20therapeutic%20drug%20response.&author=A.%20M.%20Cressman&author=V.%20Petrovic&author=M.%20Piquette-Miller&volume=5&publication_year=2012&pages=69-89&pmid=22142160&doi=10.1586/ecp.11.66&)

11. De Mattia E., Cecchin E., Roncato R., Toffoli G. (2016). Pregnane X receptor, constitutive androstane receptor and hepatocyte nuclear factors as emerging players in cancer precision medicine. Pharmacogenomics 17 1547–1571. 10.2217/pgs-2016-0095  [DOI](https://doi.org/10.2217/pgs-2016-0095) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27561454/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Pregnane%20X%20receptor,%20constitutive%20androstane%20receptor%20and%20hepatocyte%20nuclear%20factors%20as%20emerging%20players%20in%20cancer%20precision%20medicine.&author=E.%20De%20Mattia&author=E.%20Cecchin&author=R.%20Roncato&author=G.%20Toffoli&volume=17&publication_year=2016&pages=1547-1571&pmid=27561454&doi=10.2217/pgs-2016-0095&)

12. De Mattia E., Cecchin E., Toffoli G. (2015). Pharmacogenomics of intrinsic and acquired pharmacoresistance in colorectal cancer: toward targeted personalized therapy. Drug Resist. Updat. 20 39–70. 10.1016/j.drup.2015.05.003  [DOI](https://doi.org/10.1016/j.drup.2015.05.003) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26027741/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Resist.%20Updat.&title=Pharmacogenomics%20of%20intrinsic%20and%20acquired%20pharmacoresistance%20in%20colorectal%20cancer:%20toward%20targeted%20personalized%20therapy.&author=E.%20De%20Mattia&author=E.%20Cecchin&author=G.%20Toffoli&volume=20&publication_year=2015&pages=39-70&pmid=26027741&doi=10.1016/j.drup.2015.05.003&)

13. De Mattia E., Dreussi E., Cecchin E., Toffoli G. (2013a). Pharmacogenetics of the nuclear hormone receptors: the missing link between environment and drug effects? Pharmacogenomics 14 2035–2054. 10.2217/pgs.13.214  [DOI](https://doi.org/10.2217/pgs.13.214) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24279858/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Pharmacogenetics%20of%20the%20nuclear%20hormone%20receptors:%20the%20missing%20link%20between%20environment%20and%20drug%20effects?&author=E.%20De%20Mattia&author=E.%20Dreussi&author=E.%20Cecchin&author=G.%20Toffoli&volume=14&publication_year=2013a&pages=2035-2054&pmid=24279858&doi=10.2217/pgs.13.214&)

14. De Mattia E., Toffoli G., Polesel J., D’Andrea M., Corona G., Zagonel V., et al. (2013b). Pharmacogenetics of ABC and SLC transporters in metastatic colorectal cancer patients receiving first-line FOLFIRI treatment. Pharmacogenet. Genomics 23 549–557. 10.1097/FPC.0b013e328364b6cf  [DOI](https://doi.org/10.1097/FPC.0b013e328364b6cf) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24018773/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet.%20Genomics&title=Pharmacogenetics%20of%20ABC%20and%20SLC%20transporters%20in%20metastatic%20colorectal%20cancer%20patients%20receiving%20first-line%20FOLFIRI%20treatment.&author=E.%20De%20Mattia&author=G.%20Toffoli&author=J.%20Polesel&author=M.%20D%E2%80%99Andrea&author=G.%20Corona&volume=23&publication_year=2013b&pages=549-557&pmid=24018773&doi=10.1097/FPC.0b013e328364b6cf&)

15. Duan Z., Foster R., Bell D. A., Mahoney J., Wolak K., Vaidya A., et al. (2006). Signal transducers and activators of transcription 3 pathway activation in drug-resistant ovarian cancer. Clin. Cancer Res. 12 5055–5063. 10.1158/1078-0432.CCR-06-0861  [DOI](https://doi.org/10.1158/1078-0432.CCR-06-0861) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16951221/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Cancer%20Res.&title=Signal%20transducers%20and%20activators%20of%20transcription%203%20pathway%20activation%20in%20drug-resistant%20ovarian%20cancer.&author=Z.%20Duan&author=R.%20Foster&author=D.%20A.%20Bell&author=J.%20Mahoney&author=K.%20Wolak&volume=12&publication_year=2006&pages=5055-5063&pmid=16951221&doi=10.1158/1078-0432.CCR-06-0861&)

16. Fichtner-Feigl S., Kesselring R., Strober W. (2015). Chronic inflammation and the development of malignancy in the GI tract. Trends Immunol. 36 451–459. 10.1016/j.it.2015.06.007  [DOI](https://doi.org/10.1016/j.it.2015.06.007) | [PMC free article](/articles/PMC4533118/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26194796/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Trends%20Immunol.&title=Chronic%20inflammation%20and%20the%20development%20of%20malignancy%20in%20the%20GI%20tract.&author=S.%20Fichtner-Feigl&author=R.%20Kesselring&author=W.%20Strober&volume=36&publication_year=2015&pages=451-459&pmid=26194796&doi=10.1016/j.it.2015.06.007&)

17. Fujita K., Kubota Y., Ishida H., Sasaki Y. (2015). Irinotecan, a key chemotherapeutic drug for metastatic colorectal cancer. World J. Gastroenterol. 21 12234–12248. 10.3748/wjg.v21.i43.12234  [DOI](https://doi.org/10.3748/wjg.v21.i43.12234) | [PMC free article](/articles/PMC4649109/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26604633/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=World%20J.%20Gastroenterol.&title=Irinotecan,%20a%20key%20chemotherapeutic%20drug%20for%20metastatic%20colorectal%20cancer.&author=K.%20Fujita&author=Y.%20Kubota&author=H.%20Ishida&author=Y.%20Sasaki&volume=21&publication_year=2015&pages=12234-12248&pmid=26604633&doi=10.3748/wjg.v21.i43.12234&)

18. Gibson R. J., Bowen J. M., Coller J. K. (2015). What are the predictive factors in the risk and severity of chemotherapy-induced gastrointestinal toxicity? Future Oncol. 11 2367–2370. 10.2217/FON.15.138  [DOI](https://doi.org/10.2217/FON.15.138) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26270862/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Future%20Oncol.&title=What%20are%20the%20predictive%20factors%20in%20the%20risk%20and%20severity%20of%20chemotherapy-induced%20gastrointestinal%20toxicity?&author=R.%20J.%20Gibson&author=J.%20M.%20Bowen&author=J.%20K.%20Coller&volume=11&publication_year=2015&pages=2367-2370&pmid=26270862&doi=10.2217/FON.15.138&)

19. Gupta E., Lestingi T. M., Mick R., Ramirez J., Vokes E. E., Ratain M. J. (1994). Metabolic fate of irinotecan in humans: correlation of glucuronidation with diarrhea. Cancer Res. 54 3723–3725.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/8033091/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res.&title=Metabolic%20fate%20of%20irinotecan%20in%20humans:%20correlation%20of%20glucuronidation%20with%20diarrhea.&author=E.%20Gupta&author=T.%20M.%20Lestingi&author=R.%20Mick&author=J.%20Ramirez&author=E.%20E.%20Vokes&volume=54&publication_year=1994&pages=3723-3725&pmid=8033091&)

20. Gupta E., Mick R., Ramirez J., Wang X., Lestingi T. M., Vokes E. E., et al. (1997). Pharmacokinetic and pharmacodynamic evaluation of the topoisomerase inhibitor irinotecan in cancer patients. J. Clin. Oncol. 15 1502–1510. 10.1200/JCO.1997.15.4.1502  [DOI](https://doi.org/10.1200/JCO.1997.15.4.1502) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9193346/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Oncol.&title=Pharmacokinetic%20and%20pharmacodynamic%20evaluation%20of%20the%20topoisomerase%20inhibitor%20irinotecan%20in%20cancer%20patients.&author=E.%20Gupta&author=R.%20Mick&author=J.%20Ramirez&author=X.%20Wang&author=T.%20M.%20Lestingi&volume=15&publication_year=1997&pages=1502-1510&pmid=9193346&doi=10.1200/JCO.1997.15.4.1502&)

21. Harvey R. D., Morgan E. T. (2014). Cancer, inflammation, and therapy: effects on cytochrome p450-mediated drug metabolism and implications for novel immunotherapeutic agents. Clin. Pharmacol. Ther. 96 449–457. 10.1038/clpt.2014.143  [DOI](https://doi.org/10.1038/clpt.2014.143) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24987833/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Cancer,%20inflammation,%20and%20therapy:%20effects%20on%20cytochrome%20p450-mediated%20drug%20metabolism%20and%20implications%20for%20novel%20immunotherapeutic%20agents.&author=R.%20D.%20Harvey&author=E.%20T.%20Morgan&volume=96&publication_year=2014&pages=449-457&pmid=24987833&doi=10.1038/clpt.2014.143&)

22. Ho E. A., Piquette-Miller M. (2006). Regulation of multidrug resistance by pro-inflammatory cytokines. Curr. Cancer Drug Targets 6 295–311. 10.2174/156800906777441753  [DOI](https://doi.org/10.2174/156800906777441753) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16848721/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr.%20Cancer%20Drug%20Targets&title=Regulation%20of%20multidrug%20resistance%20by%20pro-inflammatory%20cytokines.&author=E.%20A.%20Ho&author=M.%20Piquette-Miller&volume=6&publication_year=2006&pages=295-311&pmid=16848721&doi=10.2174/156800906777441753&)

23. Jover R., Bort R., Gomez-Lechon M. J., Castell J. V. (2002). Down-regulation of human CYP3A4 by the inflammatory signal interleukin-6: molecular mechanism and transcription factors involved. FASEB J. 16 1799–1801. 10.1096/fj.02-0195fje  [DOI](https://doi.org/10.1096/fj.02-0195fje) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12354697/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=FASEB%20J.&title=Down-regulation%20of%20human%20CYP3A4%20by%20the%20inflammatory%20signal%20interleukin-6:%20molecular%20mechanism%20and%20transcription%20factors%20involved.&author=R.%20Jover&author=R.%20Bort&author=M.%20J.%20Gomez-Lechon&author=J.%20V.%20Castell&volume=16&publication_year=2002&pages=1799-1801&pmid=12354697&doi=10.1096/fj.02-0195fje&)

24. Jurjus A., Eid A., Al Kattar S., Zeenny M. N., Gerges-Geagea A., Haydar H., et al. (2016). Inflammatory bowel disease, colorectal cancer and type 2 diabetes mellitus: the links. BBA Clin. 5 16–24. 10.1016/j.bbacli.2015.11.002  [DOI](https://doi.org/10.1016/j.bbacli.2015.11.002) | [PMC free article](/articles/PMC4802401/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27051585/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BBA%20Clin.&title=Inflammatory%20bowel%20disease,%20colorectal%20cancer%20and%20type%202%20diabetes%20mellitus:%20the%20links.&author=A.%20Jurjus&author=A.%20Eid&author=S.%20Al%20Kattar&author=M.%20N.%20Zeenny&author=A.%20Gerges-Geagea&volume=5&publication_year=2016&pages=16-24&pmid=27051585&doi=10.1016/j.bbacli.2015.11.002&)

25. Kozyra M., Ingelman-Sundberg M., Lauschke V. M. (2017). Rare genetic variants in cellular transporters, metabolic enzymes, and nuclear receptors can be important determinants of interindividual differences in drug response. Genet. Med. 19 20–29. 10.1038/gim.2016.33  [DOI](https://doi.org/10.1038/gim.2016.33) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27101133/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genet.%20Med.&title=Rare%20genetic%20variants%20in%20cellular%20transporters,%20metabolic%20enzymes,%20and%20nuclear%20receptors%20can%20be%20important%20determinants%20of%20interindividual%20differences%20in%20drug%20response.&author=M.%20Kozyra&author=M.%20Ingelman-Sundberg&author=V.%20M.%20Lauschke&volume=19&publication_year=2017&pages=20-29&pmid=27101133&doi=10.1038/gim.2016.33&)

26. Labriet A., De Mattia E., Cecchin E., Lévesque E., Jonker D., Couture F., et al. (2017). Improved progression-free survival in irinotecan-treated metastatic colorectal cancer patients carrying the HNF1A coding variant p.I27L. Front. Pharmacol. 8:712. 10.3389/fphar.2017.00712  [DOI](https://doi.org/10.3389/fphar.2017.00712) | [PMC free article](/articles/PMC5641335/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29066969/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front.%20Pharmacol.&title=Improved%20progression-free%20survival%20in%20irinotecan-treated%20metastatic%20colorectal%20cancer%20patients%20carrying%20the%20HNF1A%20coding%20variant%20p.I27L.&author=A.%20Labriet&author=E.%20De%20Mattia&author=E.%20Cecchin&author=E.%20L%C3%A9vesque&author=D.%20Jonker&volume=8&issue=712&publication_year=2017&pmid=29066969&doi=10.3389/fphar.2017.00712&)

27. Lauschke V. M., Ingelman-Sundberg M. (2016). Precision medicine and rare genetic variants. Trends Pharmacol. Sci. 37 85–86. 10.1016/j.tips.2015.10.006  [DOI](https://doi.org/10.1016/j.tips.2015.10.006) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26705087/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Trends%20Pharmacol.%20Sci.&title=Precision%20medicine%20and%20rare%20genetic%20variants.&author=V.%20M.%20Lauschke&author=M.%20Ingelman-Sundberg&volume=37&publication_year=2016&pages=85-86&pmid=26705087&doi=10.1016/j.tips.2015.10.006&)

28. Lee C. S., Ryan E. J., Doherty G. A. (2014). Gastro-intestinal toxicity of chemotherapeutics in colorectal cancer: the role of inflammation. World J. Gastroenterol. 20 3751–3761. 10.3748/wjg.v20.i14.3751  [DOI](https://doi.org/10.3748/wjg.v20.i14.3751) | [PMC free article](/articles/PMC3983434/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24744571/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=World%20J.%20Gastroenterol.&title=Gastro-intestinal%20toxicity%20of%20chemotherapeutics%20in%20colorectal%20cancer:%20the%20role%20of%20inflammation.&author=C.%20S.%20Lee&author=E.%20J.%20Ryan&author=G.%20A.%20Doherty&volume=20&publication_year=2014&pages=3751-3761&pmid=24744571&doi=10.3748/wjg.v20.i14.3751&)

29. Levesque E., Belanger A. S., Harvey M., Couture F., Jonker D., Innocenti F., et al. (2013). Refining the UGT1A haplotype associated with irinotecan-induced hematological toxicity in metastatic colorectal cancer patients treated with 5-fluorouracil/irinotecan-based regimens. J. Pharmacol. Exp. Ther. 345 95–101. 10.1124/jpet.112.202242  [DOI](https://doi.org/10.1124/jpet.112.202242) | [PMC free article](/articles/PMC3920089/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23386248/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Pharmacol.%20Exp.%20Ther.&title=Refining%20the%20UGT1A%20haplotype%20associated%20with%20irinotecan-induced%20hematological%20toxicity%20in%20metastatic%20colorectal%20cancer%20patients%20treated%20with%205-fluorouracil/irinotecan-based%20regimens.&author=E.%20Levesque&author=A.%20S.%20Belanger&author=M.%20Harvey&author=F.%20Couture&author=D.%20Jonker&volume=345&publication_year=2013&pages=95-101&pmid=23386248&doi=10.1124/jpet.112.202242&)

30. Logan R. M., Gibson R. J., Bowen J. M., Stringer A. M., Sonis S. T., Keefe D. M. (2008). Characterisation of mucosal changes in the alimentary tract following administration of irinotecan: implications for the pathobiology of mucositis. Cancer Chemother. Pharmacol. 62 33–41. 10.1007/s00280-007-0570-0  [DOI](https://doi.org/10.1007/s00280-007-0570-0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17703303/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Chemother.%20Pharmacol.&title=Characterisation%20of%20mucosal%20changes%20in%20the%20alimentary%20tract%20following%20administration%20of%20irinotecan:%20implications%20for%20the%20pathobiology%20of%20mucositis.&author=R.%20M.%20Logan&author=R.%20J.%20Gibson&author=J.%20M.%20Bowen&author=A.%20M.%20Stringer&author=S.%20T.%20Sonis&volume=62&publication_year=2008&pages=33-41&pmid=17703303&doi=10.1007/s00280-007-0570-0&)

31. Markman J. L., Shiao S. L. (2015). Impact of the immune system and immunotherapy in colorectal cancer. J. Gastrointest. Oncol. 6 208–223. 10.3978/j.issn.2078-6891.2014.077  [DOI](https://doi.org/10.3978/j.issn.2078-6891.2014.077) | [PMC free article](/articles/PMC4311093/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25830040/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Gastrointest.%20Oncol.&title=Impact%20of%20the%20immune%20system%20and%20immunotherapy%20in%20colorectal%20cancer.&author=J.%20L.%20Markman&author=S.%20L.%20Shiao&volume=6&publication_year=2015&pages=208-223&pmid=25830040&doi=10.3978/j.issn.2078-6891.2014.077&)

32. Mathijssen R. H., van Alphen R. J., Verweij J., Loos W. J., Nooter K., Stoter G., et al. (2001). Clinical pharmacokinetics and metabolism of irinotecan (CPT-11). Clin. Cancer Res. 7 2182–2194.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/11489791/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Cancer%20Res.&title=Clinical%20pharmacokinetics%20and%20metabolism%20of%20irinotecan%20(CPT-11).&author=R.%20H.%20Mathijssen&author=R.%20J.%20van%20Alphen&author=J.%20Verweij&author=W.%20J.%20Loos&author=K.%20Nooter&volume=7&publication_year=2001&pages=2182-2194&pmid=11489791&)

33. Melo M. L., Brito G. A., Soares R. C., Carvalho S. B., Silva J. V., Soares P. M., et al. (2008). Role of cytokines (TNF-alpha, IL-1beta and KC) in the pathogenesis of CPT-11-induced intestinal mucositis in mice: effect of pentoxifylline and thalidomide. Cancer Chemother. Pharmacol. 61 775–784. 10.1007/s00280-007-0534-4  [DOI](https://doi.org/10.1007/s00280-007-0534-4) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17624531/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Chemother.%20Pharmacol.&title=Role%20of%20cytokines%20(TNF-alpha,%20IL-1beta%20and%20KC)%20in%20the%20pathogenesis%20of%20CPT-11-induced%20intestinal%20mucositis%20in%20mice:%20effect%20of%20pentoxifylline%20and%20thalidomide.&author=M.%20L.%20Melo&author=G.%20A.%20Brito&author=R.%20C.%20Soares&author=S.%20B.%20Carvalho&author=J.%20V.%20Silva&volume=61&publication_year=2008&pages=775-784&pmid=17624531&doi=10.1007/s00280-007-0534-4&)

34. Moriasi C., Subramaniam D., Awasthi S., Ramalingam S., Anant S. (2012). Prevention of colitis-associated cancer: natural compounds that target the IL-6 soluble receptor. Anticancer Agents Med. Chem. 12 1221–1238. 10.2174/187152012803833080  [DOI](https://doi.org/10.2174/187152012803833080) | [PMC free article](/articles/PMC4239476/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22583410/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Anticancer%20Agents%20Med.%20Chem.&title=Prevention%20of%20colitis-associated%20cancer:%20natural%20compounds%20that%20target%20the%20IL-6%20soluble%20receptor.&author=C.%20Moriasi&author=D.%20Subramaniam&author=S.%20Awasthi&author=S.%20Ramalingam&author=S.%20Anant&volume=12&publication_year=2012&pages=1221-1238&pmid=22583410&doi=10.2174/187152012803833080&)

35. Muindi J. R., Adjei A. A., Wu Z. R., Olson I., Huang H., Groman A., et al. (2013). Serum vitamin D metabolites in colorectal cancer patients receiving cholecalciferol supplementation: correlation with polymorphisms in the vitamin D genes. Horm. Cancer 4 242–250. 10.1007/s12672-013-0139-9  [DOI](https://doi.org/10.1007/s12672-013-0139-9) | [PMC free article](/articles/PMC3689467/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23456391/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Horm.%20Cancer&title=Serum%20vitamin%20D%20metabolites%20in%20colorectal%20cancer%20patients%20receiving%20cholecalciferol%20supplementation:%20correlation%20with%20polymorphisms%20in%20the%20vitamin%20D%20genes.&author=J.%20R.%20Muindi&author=A.%20A.%20Adjei&author=Z.%20R.%20Wu&author=I.%20Olson&author=H.%20Huang&volume=4&publication_year=2013&pages=242-250&pmid=23456391&doi=10.1007/s12672-013-0139-9&)

36. Nguyen P. M., Putoczki T. L., Ernst M. (2015). STAT3-activating cytokines: a therapeutic opportunity for inflammatory bowel disease? J. Interferon Cytokine Res. 35 340–350. 10.1089/jir.2014.0225  [DOI](https://doi.org/10.1089/jir.2014.0225) | [PMC free article](/articles/PMC4426323/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25760898/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Interferon%20Cytokine%20Res.&title=STAT3-activating%20cytokines:%20a%20therapeutic%20opportunity%20for%20inflammatory%20bowel%20disease?&author=P.%20M.%20Nguyen&author=T.%20L.%20Putoczki&author=M.%20Ernst&volume=35&publication_year=2015&pages=340-350&pmid=25760898&doi=10.1089/jir.2014.0225&)

37. Permuth-Wey J., Fulp W. J., Reid B. M., Chen Z., Georgeades C., Cheng J. Q., et al. (2016). STAT3 polymorphisms may predict an unfavorable response to first-line platinum-based therapy for women with advanced serous epithelial ovarian cancer. Int. J. Cancer 138 612–619. 10.1002/ijc.29799  [DOI](https://doi.org/10.1002/ijc.29799) | [PMC free article](/articles/PMC4819434/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26264211/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Cancer&title=STAT3%20polymorphisms%20may%20predict%20an%20unfavorable%20response%20to%20first-line%20platinum-based%20therapy%20for%20women%20with%20advanced%20serous%20epithelial%20ovarian%20cancer.&author=J.%20Permuth-Wey&author=W.%20J.%20Fulp&author=B.%20M.%20Reid&author=Z.%20Chen&author=C.%20Georgeades&volume=138&publication_year=2016&pages=612-619&pmid=26264211&doi=10.1002/ijc.29799&)

38. Prakash C., Zuniga B., Song C. S., Jiang S., Cropper J., Park S., et al. (2015). Nuclear receptors in drug metabolism, drug response and drug interactions. Nucl. Receptor Res. 2:101178. 10.11131/2015/101178  [DOI](https://doi.org/10.11131/2015/101178) | [PMC free article](/articles/PMC4963026/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27478824/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucl.%20Receptor%20Res.&title=Nuclear%20receptors%20in%20drug%20metabolism,%20drug%20response%20and%20drug%20interactions.&author=C.%20Prakash&author=B.%20Zuniga&author=C.%20S.%20Song&author=S.%20Jiang&author=J.%20Cropper&volume=2&issue=101178&publication_year=2015&pmid=27478824&doi=10.11131/2015/101178&)

39. Qi Q. R., Yang Z. M. (2014). Regulation and function of signal transducer and activator of transcription 3. World J. Biol. Chem. 5 231–239. 10.4331/wjbc.v5.i2.231  [DOI](https://doi.org/10.4331/wjbc.v5.i2.231) | [PMC free article](/articles/PMC4050116/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24921012/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=World%20J.%20Biol.%20Chem.&title=Regulation%20and%20function%20of%20signal%20transducer%20and%20activator%20of%20transcription%203.&author=Q.%20R.%20Qi&author=Z.%20M.%20Yang&volume=5&publication_year=2014&pages=231-239&pmid=24921012&doi=10.4331/wjbc.v5.i2.231&)

40. Reuter S., Gupta S. C., Chaturvedi M. M., Aggarwal B. B. (2010). Oxidative stress, inflammation, and cancer: how are they linked? Free Radic. Biol. Med. 49 1603–1616. 10.1016/j.freeradbiomed.2010.09.006  [DOI](https://doi.org/10.1016/j.freeradbiomed.2010.09.006) | [PMC free article](/articles/PMC2990475/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20840865/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Free%20Radic.%20Biol.%20Med.&title=Oxidative%20stress,%20inflammation,%20and%20cancer:%20how%20are%20they%20linked?&author=S.%20Reuter&author=S.%20C.%20Gupta&author=M.%20M.%20Chaturvedi&author=B.%20B.%20Aggarwal&volume=49&publication_year=2010&pages=1603-1616&pmid=20840865&doi=10.1016/j.freeradbiomed.2010.09.006&)

41. Rothenberg M. L., Meropol N. J., Poplin E. A., Van Cutsem E., Wadler S. (2001). Mortality associated with irinotecan plus bolus fluorouracil/leucovorin: summary findings of an independent panel. J. Clin. Oncol. 19 3801–3807. 10.1200/JCO.2001.19.18.3801  [DOI](https://doi.org/10.1200/JCO.2001.19.18.3801) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11559717/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Oncol.&title=Mortality%20associated%20with%20irinotecan%20plus%20bolus%20fluorouracil/leucovorin:%20summary%20findings%20of%20an%20independent%20panel.&author=M.%20L.%20Rothenberg&author=N.%20J.%20Meropol&author=E.%20A.%20Poplin&author=E.%20Van%20Cutsem&author=S.%20Wadler&volume=19&publication_year=2001&pages=3801-3807&pmid=11559717&doi=10.1200/JCO.2001.19.18.3801&)

42. Swen J. J., Nijenhuis M., de Boer A., Grandia L., Maitland-van der Zee A. H., Mulder H., et al. (2011). Pharmacogenetics: from bench to byte–an update of guidelines. Clin. Pharmacol. Ther. 89 662–673. 10.1038/clpt.2011.34  [DOI](https://doi.org/10.1038/clpt.2011.34) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21412232/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Pharmacogenetics:%20from%20bench%20to%20byte%E2%80%93an%20update%20of%20guidelines.&author=J.%20J.%20Swen&author=M.%20Nijenhuis&author=A.%20de%20Boer&author=L.%20Grandia&author=A.%20H.%20Maitland-van%20der%20Zee&volume=89&publication_year=2011&pages=662-673&pmid=21412232&doi=10.1038/clpt.2011.34&)

43. Toffoli G., Cecchin E., Corona G., Russo A., Buonadonna A., D’Andrea M., et al. (2006). The role of UGT1A1∗28 polymorphism in the pharmacodynamics and pharmacokinetics of irinotecan in patients with metastatic colorectal cancer. J. Clin. Oncol. 24 3061–3068. 10.1200/JCO.2005.05.5400  [DOI](https://doi.org/10.1200/JCO.2005.05.5400) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16809730/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Oncol.&title=The%20role%20of%20UGT1A1%E2%88%9728%20polymorphism%20in%20the%20pharmacodynamics%20and%20pharmacokinetics%20of%20irinotecan%20in%20patients%20with%20metastatic%20colorectal%20cancer.&author=G.%20Toffoli&author=E.%20Cecchin&author=G.%20Corona&author=A.%20Russo&author=A.%20Buonadonna&volume=24&publication_year=2006&pages=3061-3068&pmid=16809730&doi=10.1200/JCO.2005.05.5400&)

44. Toffoli G., Cecchin E., Gasparini G., D’Andrea M., Azzarello G., Basso U., et al. (2010). Genotype-driven phase I study of irinotecan administered in combination with fluorouracil/leucovorin in patients with metastatic colorectal cancer. J. Clin. Oncol. 28 866–871. 10.1200/JCO.2009.23.6125  [DOI](https://doi.org/10.1200/JCO.2009.23.6125) | [PMC free article](/articles/PMC4872310/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20038727/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Oncol.&title=Genotype-driven%20phase%20I%20study%20of%20irinotecan%20administered%20in%20combination%20with%20fluorouracil/leucovorin%20in%20patients%20with%20metastatic%20colorectal%20cancer.&author=G.%20Toffoli&author=E.%20Cecchin&author=G.%20Gasparini&author=M.%20D%E2%80%99Andrea&author=G.%20Azzarello&volume=28&publication_year=2010&pages=866-871&pmid=20038727&doi=10.1200/JCO.2009.23.6125&)

45. Tournigand C., Andre T., Achille E., Lledo G., Flesh M., Mery-Mignard D., et al. (2004). FOLFIRI followed by FOLFOX6 or the reverse sequence in advanced colorectal cancer: a randomized GERCOR study. J. Clin. Oncol. 22 229–237. 10.1200/JCO.2004.05.113  [DOI](https://doi.org/10.1200/JCO.2004.05.113) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14657227/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Oncol.&title=FOLFIRI%20followed%20by%20FOLFOX6%20or%20the%20reverse%20sequence%20in%20advanced%20colorectal%20cancer:%20a%20randomized%20GERCOR%20study.&author=C.%20Tournigand&author=T.%20Andre&author=E.%20Achille&author=G.%20Lledo&author=M.%20Flesh&volume=22&publication_year=2004&pages=229-237&pmid=14657227&doi=10.1200/JCO.2004.05.113&)

46. Verschuren J. J., Trompet S., Postmus I., Sampietro M. L., Heijmans B. T., Houwing-Duistermaat J. J., et al. (2012). Systematic testing of literature reported genetic variation associated with coronary restenosis: results of the GENDER Study. PLoS. One. 7:e42401. 10.1371/journal.pone.0042401  [DOI](https://doi.org/10.1371/journal.pone.0042401) | [PMC free article](/articles/PMC3411750/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22879966/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS.%20One.&title=Systematic%20testing%20of%20literature%20reported%20genetic%20variation%20associated%20with%20coronary%20restenosis:%20results%20of%20the%20GENDER%20Study.&author=J.%20J.%20Verschuren&author=S.%20Trompet&author=I.%20Postmus&author=M.%20L.%20Sampietro&author=B.%20T.%20Heijmans&volume=7&issue=e42401&publication_year=2012&pmid=22879966&doi=10.1371/journal.pone.0042401&)

47. Yu H., Lee H., Herrmann A., Buettner R., Jove R. (2014). Revisiting STAT3 signalling in cancer: new and unexpected biological functions. Nat. Rev. Cancer 14 736–746. 10.1038/nrc3818  [DOI](https://doi.org/10.1038/nrc3818) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25342631/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Cancer&title=Revisiting%20STAT3%20signalling%20in%20cancer:%20new%20and%20unexpected%20biological%20functions.&author=H.%20Yu&author=H.%20Lee&author=A.%20Herrmann&author=R.%20Buettner&author=R.%20Jove&volume=14&publication_year=2014&pages=736-746&pmid=25342631&doi=10.1038/nrc3818&)

48. Zhang X., Xiao W., Wang L., Tian Z., Zhang J. (2011). Deactivation of signal transducer and activator of transcription 3 reverses chemotherapeutics resistance of leukemia cells via down-regulating P-gp. PLoS. One. 6:e20965. 10.1371/journal.pone.0020965  [DOI](https://doi.org/10.1371/journal.pone.0020965) | [PMC free article](/articles/PMC3108986/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21677772/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS.%20One.&title=Deactivation%20of%20signal%20transducer%20and%20activator%20of%20transcription%203%20reverses%20chemotherapeutics%20resistance%20of%20leukemia%20cells%20via%20down-regulating%20P-gp.&author=X.%20Zhang&author=W.%20Xiao&author=L.%20Wang&author=Z.%20Tian&author=J.%20Zhang&volume=6&issue=e20965&publication_year=2011&pmid=21677772&doi=10.1371/journal.pone.0020965&)

49. Zhu H., Liu Z., Tang L., Liu J., Zhou M., Xie F., et al. (2012). Reversal of P-gp and MRP1-mediated multidrug resistance by H6, a gypenoside aglycon from Gynostemma pentaphyllum, in vincristine-resistant human oral cancer (KB/VCR) cells. Eur. J. Pharmacol. 696 43–53. 10.1016/j.ejphar.2012.09.046  [DOI](https://doi.org/10.1016/j.ejphar.2012.09.046) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23051672/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur.%20J.%20Pharmacol.&title=Reversal%20of%20P-gp%20and%20MRP1-mediated%20multidrug%20resistance%20by%20H6,%20a%20gypenoside%20aglycon%20from%20Gynostemma%20pentaphyllum,%20in%20vincristine-resistant%20human%20oral%20cancer%20(KB/VCR)%20cells.&author=H.%20Zhu&author=Z.%20Liu&author=L.%20Tang&author=J.%20Liu&author=M.%20Zhou&volume=696&publication_year=2012&pages=43-53&pmid=23051672&doi=10.1016/j.ejphar.2012.09.046&)
