# Genome-Wide Prioritization and Transcriptomics Reveal Novel Signatures Associated with Thiazide Diuretics Blood Pressure Response

## Metadata
**Authors:** Mohamed H Shahin, Ana C Sá, Amy Webb, Yan Gong, Taimour Langaee, Caitrin W McDonough, Alberto Riva, Amber L Beitleshees, Arlene B Chapman, John G Gums, Stephen T Turner, Eric Boerwinkle, Steven E Scherer, Wolfgang Sadee, Rhonda M Cooper-DeHoff, Julie A Johnson
**Journal:** Circulation. Cardiovascular genetics
**Date:** 2017 Jan
**DOI:** [10.1161/CIRCGENETICS.116.001404](https://doi.org/10.1161/CIRCGENETICS.116.001404)
**PMID:** 28115488
**PMCID:** PMC5298887
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5298887/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC5298887/pdf/nihms-837915.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC5298887/pdf/nihms-837915.pdf)

## Abstract

**Background:** 
Thiazide diuretics are among the most commonly prescribed antihypertensives. However, <50% of thiazide treated patients achieve blood pressure (BP) control. Herein, we used different omics (genomics and transcriptomics) to identify novel biomarkers of thiazide diuretics BP response.

**Methods and Results:** 
Genome-wide analysis included 228 white hypertensives with BP determined at baseline and after 9 weeks of hydrochlorothiazide (HCTZ). Single nucleotide polymorphisms (SNPs) with p<5×10−5 were prioritized according to their biological function, using RegulomeDB, haploreg and GWAVA. The results from the prioritization approach revealed rs10995 as the most likely functional SNP, among SNPs tested, that has been associated with HCTZ BP response. The rs10995 G-allele was associated with better BP response to HCTZ vs. non-carriers (ΔSBP/ΔDBP:−12.3/−8.2 vs. −6.8/−3.5 mmHg, respectively, ΔSBP p=3×10−4, ΔDBP p=5×10−5). This association was replicated in independent participants treated with chlorthalidone (CLT). In addition, rs10995 G-allele was associated with increased mRNA expression of VASP, encoding vasodilator-stimulated phosphoprotein. Moreover, baseline expression of the VASP mRNA was significantly higher in 25 good-responders to HCTZ compared to 25 poor-responders (p=0.01). This finding was replicated in independent participants treated with CLT (p=0.04). Lastly, allelic-specific expression analysis revealed a significant but modest imbalance with rs10995 and rs10156, a SNP in high linkage disequilibrium (r2=0.7) with rs10995, which both could contribute to the observed genetic effects by affecting VASP mRNA expression.

**Conclusions:** 
This study highlights the strength of using different omics to identify novel biomarkers of drug response and suggests VASP as a potential determinant of thiazide diuretics BP response.

**Clinical Trial Information:** 
ClinicalTrials.gov; Unique Identifiers: NCT00246519, NCT01203852.

Keywords: pharmacogenetics, hypertension, blood pressure, transcriptome, genomics, pharmacogenomics, antihypertensive agent, personalized medicine, RNA-Seq, thiazide diuretics

### Background

Thiazide diuretics are among the most commonly prescribed antihypertensives. However, <50% of thiazide treated patients achieve blood pressure (BP) control. Herein, we used different omics (genomics and transcriptomics) to identify novel biomarkers of thiazide diuretics BP response.

### Methods and Results

Genome-wide analysis included 228 white hypertensives with BP determined at baseline and after 9 weeks of hydrochlorothiazide (HCTZ). Single nucleotide polymorphisms (SNPs) with p<5×10^−5^ were prioritized according to their biological function, using RegulomeDB, haploreg and GWAVA. The results from the prioritization approach revealed rs10995 as the most likely functional SNP, among SNPs tested, that has been associated with HCTZ BP response. The rs10995 *G-*allele was associated with better BP response to HCTZ *vs.* non-carriers (ΔSBP/ΔDBP:−12.3/−8.2 *vs.* −6.8/−3.5 mmHg, respectively, ΔSBP p=3×10^−4^, ΔDBP p=5×10^−5^). This association was replicated in independent participants treated with chlorthalidone (CLT). In addition, rs10995 *G-allele* was associated with increased mRNA expression of *VASP*, encoding vasodilator-stimulated phosphoprotein. Moreover, baseline expression of the VASP mRNA was significantly higher in 25 good-responders to HCTZ compared to 25 poor-responders (p=0.01). This finding was replicated in independent participants treated with CLT (p=0.04). Lastly, allelic-specific expression analysis revealed a significant but modest imbalance with rs10995 and rs10156, a SNP in high linkage disequilibrium (r^2^=0.7) with rs10995, which both could contribute to the observed genetic effects by affecting VASP mRNA expression.

### Conclusions

This study highlights the strength of using different omics to identify novel biomarkers of drug response and suggests *VASP* as a potential determinant of thiazide diuretics BP response.

### Clinical Trial Information

[ClinicalTrials.gov](http://ClinicalTrials.gov); Unique Identifiers: [NCT00246519](https://clinicaltrials.gov/ct2/show/NCT00246519), [NCT01203852](https://clinicaltrials.gov/ct2/show/NCT01203852).

## Introduction

Hypertension (HTN) is the most common chronic disease and the primary cause of cardiovascular morbidity and mortality globally.^1^ Studies have shown that the risk for coronary diseases and stroke doubles with every 20 mmHg increase in systolic BP (SBP) or 10 mmHg increase in diastolic BP (DBP).^2^ Accordingly, using anti-hypertensive medications for controlling BP substantially reduces cardiovascular risk and the overall mortality associated with HTN.^3^ Thiazide diuretics, including hydrochlorothiazide (HCTZ), are ranked among the most commonly prescribed drugs for the treatment of HTN in the U.S.,^4^ and are highly recommended as first-line agents for most patients with uncomplicated essential HTN.^5^ Although they have been used as anti-hypertensives for more than half a century, the mechanism by which thiazide diuretics chronically lower BP has not been fully elucidated yet.^6^ Additionally, studies have shown that < 50% of thiazide treated patients achieve BP control.^7, 8^ Even with the use of other anti-HTN medications, acting on a variety of BP regulatory systems, only 44% of HTN treated patients achieve BP control.^9^ Given these facts, it is clear that the current approach for selecting anti-HTN medications is suboptimal and more work is still needed to optimize the use of these drugs.

In the past decade, application of genome-wide association studies (GWAS) has advanced our understanding of the potential role of genetics in variable response to drugs.^10^ Recently, we conducted a GWAS analysis in which we identified novel genetic regions associated with the BP response to HCTZ, such as protein kinase C, alpha (*PRKCA*) and *G-*protein alpha subunit-endothelian-3 (*GNAS-EDN3*).^11^ In this study, we propose that there are additional true genetic contributors that have not met the stringent genome-wide statistical threshold, but they are difficult to ascertain statistically using genomics data alone, particularly with the small sample sizes of the globally available HTN pharmacogenetics datasets. Thus, it is critical to leverage other information to effectively prioritize GWAS signals, increase replication rates and better understand the mechanism underlying HCTZ BP response.

Recent studies have shown that GWAS single nucleotide polymorphisms (SNPs) associated with complex traits are more likely to be expression quantitative trait loci (eQTLs).^12, 13^ Additionally, studies have demonstrated that the majority (~93%) of previously conducted GWAS findings lay in non-coding regions,^14^ and that these findings are significantly enriched in the regions that harbor functional elements, such as transcriptional factor binding sites (TFBSs), histone modification marked regions, DNase I hypersensitive sites (DHSs) and eQTLs.^15, 16^ Accordingly, we hypothesized that prioritizing the GWAS output based on regulatory functional signals that perturb gene expression might elucidate novel genetic signals affecting HCTZ BP response. Investigating genes where these signals are involved might open new avenues for a better understanding of the molecular determinants of the BP response. Thus, we aimed in the current study to use data from the Encyclopedia of DNA Elements (ENCODE) project, like transcriptional factor CHIP-seq, histone CHIP-seq, and DNase I hypersensitivity site data, along with publically available eQTL data, to prioritize and highlight novel genetic variants affecting the BP response to HCTZ. Additionally, we sought to use transcriptomics data to further confirm our genetic finding.

## Methods

### Pharmacogenomic Evaluation of Antihypertensive Responses (PEAR) study

The primary analysis included samples and data from participants recruited as part of the PEAR trial (clinicaltrials.gov # [NCT00246519](https://clinicaltrials.gov/ct2/show/NCT00246519)).^17^ In brief, PEAR was a prospective, randomized, multi-center study with one of its primary aims was to evaluate the role of genetics on the BP response of HCTZ treated participants. PEAR recruited mild to moderate hypertensive participants, defined as participants with untreated DBP ≤ 110 and SBP ≤ 180. After enrollment, all participants had an average of 4 weeks washout period followed by a randomization to either a monotherapy of 12.5 mg/daily HCTZ or 50 mg/daily atenolol (β-1-selective blocker) for a duration of three weeks, with the dose titrated upwards for additional 6 weeks (i.e. HCTZ 25 mg/daily or atenolol 100 mg/daily) in participants with BP > 120/70 mmHg.

### Pharmacogenomic Evaluation of Antihypertensive Responses 2 (PEAR-2) Study

We also used clinical data and biological samples from PEAR-2 participants to validate and replicate our findings from the PEAR primary analysis. In brief, PEAR-2 was a prospective, multi-center, sequential monotherapy study (clinicaltrials.gov # [NCT01203852](https://clinicaltrials.gov/ct2/show/NCT01203852))^18^ with one of its primary aims being to evaluate the role of genetics on the BP response to chlorthalidone (CLT; a thiazide-like diuretic). PEAR-2 recruited mild to moderate hypertensive participants, aged 18-65 years, from the University of Florida (Gainesville, FL), Emory University (Atlanta, GA), and the Mayo Clinic (Rochester, MN). All participants had an average 4 week washout period then they started on a monotherapy of 15 mg/daily CLT for a duration of two weeks, with the dose titrated upwards for additional 6 weeks (i.e. CLT 25 mg/daily) in participants with BP still > 120/70 mmHg. The Institutional Review Board approved both PEAR and PEAR-2 studies at each study site, and written informed consent was obtained from all participants.

### Thiazide Blood Pressure Response Measurement

The primary analysis of the current study included 228 European American (white) participants from the PEAR study with their BP measured pre-HCTZ (baseline) and 9 weeks post-HCTZ therapy. In PEAR, BP at home, office and ambulatory daytime and night-time were measured, as previously described^17^ and as explained in the [Supplement](#SD1). For the analysis of the PEAR HCTZ treated participants included in this study, we used a composite weighted average of the home, office and ambulatory daytime and nighttime, which we showed that it represents a more accurate measurement of BP response with a better signal-to-noise ratio and more power to identify genetic predictors of BP response.^19^

The SNP replication analysis within this study included 185 white participants from the PEAR-2 study with their BP measured pre-CLT (baseline) and 8 weeks post-CLT therapy. Both home and office BP were measured, as previously described.^18^ Please check the [Supplemental Material](#SD1) for more details.

### Genotyping

Genome-wide genotyping for the PEAR participants was performed as previously described^11^ and as explained in the [Supplement](#SD1). For PEAR-2, genotyping was conducted using Human Omni2.5 S BeadChip (Illumina, San Diego CA). Participants from PEAR or PEAR-2 were excluded if sample genotype call rates were below 95%. Additionally, SNPs with a genotype call rates below 95% were also excluded.

### Gene Expression

Gene expression was measured using RNA-Seq analysis on both PEAR (discovery) and PEAR-2 (replication) white participants treated with HCTZ and CLT, respectively. For PEAR RNA-Seq analysis, we collected whole blood samples at baseline from 50 white participants treated with HCTZ. Participants were selected from the upper and lower quartiles of HCTZ home DBP response (25 good BP responders and 25 poor BP responders). Similarly, for PEAR-2 gene expression, whole blood samples were collected at baseline from 50 white participants that were selected from the upper and lower quartiles of home DBP response to CLT (25 good BP responders and 25 poor BP responders) to conduct RNA-Seq analysis on these samples. Home DBP response was used for selection of participants for RNA-Seq analyses since it exhibits less variability and superior reproducibility compared to office BP response.^20^ Of note, out of the 50 samples used for the RNA-Seq analysis in PEAR, one sample was removed because it did not pass quality control.

Details for the RNA-Seq analysis for both PEAR and PEAR-2 samples are described in the [Supplement](#SD1). In brief, a strand-specific protocol was used to prepare libraries, as previously described.^21^ For data quality control purposes, read duplicates removal was implemented using Picard ([http://picard.sourceforge.net](http://picard.sourceforge.net)) Mark Duplicates option. The 100 bp reads generated in the paired-end RNA sequencing were uniquely mapped to the human reference genome (hg19) using TopHat v2.0.10 allowing for four reads mismatches, read edit distance of six, one mismatch in the anchor region of a spliced read, and a maximum of five multi-hits. Transcript assembly was performed using Cufflinks (v2.0.2) and gene expression values, reported in fragments per kilobase per million reads (FPKM), were calculated using cuffdiff (v2.2.1).^22^ Alternative tools for investigating differential gene expression were used to adjust for covariates such as age, gender and baseline BP. First, we removed duplicate BAM files from TopHat alignments, as an input to the function htseq-count,^23^ to count the number of reads for each gene (Gencode gene annotation release 18). Counts were modeled to a negative binomial distribution using a generalized linear model implemented in edgeR^24^ (Bioconductor R Package).

### Statistical Analyses

#### Genomics Analysis

GWAS analysis was previously conducted^11^ to test the association between dosage imputed genotype data with composite SBP and composite DBP responses to HCTZ in 228 white participants from the PEAR study. PLINK software was used to run the analysis with adjustment for age, gender, pre-HCTZ BP and population substructure by considering the first and second principal components (PC1 and PC2) in all our analysis. In this study, we investigated SNPs with a p-value <5×10^−5^ from the GWAS of HCTZ BP response, selected on the basis of quantile-quantile (Q-Q) plots showing that SNPs with p-values <5×10^−5^ deviated from expectation under the null hypothesis of no relationship between SNPs and HCTZ BP response in PEAR white participants treated with HCTZ ([Supplementary Figure S1](#SD1)). Deviation from Hardy-Weinberg Equilibrium (HWE) for the selected SNPs was tested using exact test, and SNPs with HWE p-value <1×10^−5^ were removed. The statistical analyses conducted in this study were carried out with SAS (version 9.3; SAS Institute) and SPSS software (version 17.0; SPSS Inc.).

### Single Nucleotide Polymorphisms Prioritization Approach

To address the aims of the current study, a SNP prioritization approach was conducted as shown in [Figure 1](#F1). For the prioritization approach, we selected 103 SNPs ([Supplementary Table S1](#SD1)), representing 41 independent genetic signals, with p-values < 5×10^−5^ from the GWAS analyses of both HCTZ SBP and DBP responses conducted on PEAR whites. It is likely that among the 103 selected SNPs, true positive SNPs might be in linkage disequilibrium (LD) with functional variants that are responsible for the significant association with HCTZ BP response. Thus, we used rAggr webserver ([http://raggr.usc.edu/](http://raggr.usc.edu/)) to select all SNPs in high LD (r^2^>0.8, according to Northern Europeans from Utah (CEU) population) with the 103 SNPs, within a distance of 500kb, based on 1000 Genomes Project, phase 3 release. This step yielded 1,082 SNPs which were then annotated and prioritized using three analytical tools: (1) RegulomeDB^25^ ([http://www.regulomedb.org/](http://www.regulomedb.org/)), (2) Genome-Wide Annotation of Variants (GWAVA)^26^ ([https://www.sanger.ac.uk/sanger/StatGen Gwava](https://www.sanger.ac.uk/sanger/StatGenGwava)) and (3) Haploreg v4.1^27^ webserver ([http://www.broadinstitute.org/mammals/ haploreg/haplo reg. php](http://www.broadinstitute.org/mammals/haploreg/haploreg.php)). These three tools annotate SNPs based on high-throughput data sets from the ENCODE Project, along with other publically available Expression Quantitative Trait Loci (eQTL) data sets, computational predictions and manual annotations. More details about each tool are described in the [Supplement](#SD1).

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1291/5298887/398a043b6884/nihms-837915-f0001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5298887_nihms-837915-f0001.jpg)

The overall framework of the experimental approaches used in this study. GWAS, genome-wide association study; SNPs, single nucleotide polymorphisms; eQTLs, expression quantitative trait loci.

In brief, the RegulomeDB has a scoring system that categorizes SNPs, ranging from 1 to 6, based on the degree of experimental or computational evidence and regulatory functional consequences of tested SNPs. For prioritization using RegulomeDB, we focused on SNPs with a score of 1 and 2, since the lower score indicates stronger evidence for a SNP to be located in a potentially functional region. Using GWAVA, SNPs were prioritized based on their TSS score (calculated based on nearest transcription start site). We used GWAVA TSS score to prioritize our signals because it takes into account different regulatory annotations and not influenced by any single annotation. Additionally, we selected SNPs with a TSS score ≥ 0.5, which represents the threshold used by GWAVA’s original authors to classify SNPs as functional.^26^ Using Haploreg v4.1, we selected all SNPs that had been previously shown to affect gene expression (eQTLs). SNPs that overlap between the three annotation tools were then moved forward to be tested for replication in PEAR-2 whites treated with CLT, as explained below.

### Replication

A general linear model was used to test the association between prioritized SNPs and CLT BP response in PEAR-2 whites, with adjustment for age, gender, pre-CLT BP and PC1 and 2.

### Further Validation

For SNPs replicated in PEAR-2, we used Haploreg v4.1 to check whether these SNPs had been shown to be significantly associated with the mRNA expression levels of any genes. We then selected these genes and tested differences in their mRNA expression levels between PEAR HCTZ good responders and poor responders. We further replicated our findings by testing the differences in mRNA expression levels, of the genes of interest, between PEAR-2 CLT good responders and poor responders.

We also tested replicated SNPs for allelic mRNA expression imbalance (AEI). SNP calls and allele-specific read counts were generated with SaMtools mpileup from the duplicate removed alignment file.^28^ Analysis was conducted in individuals heterozygous for the relevant transcript SNPs in both PEAR and PEAR-2 thiazide diuretics treated participants. Chi-square Goodness-of-Fit was applied to test if the allele ratio (minor allele/major allele) deviates from the expected ratio 0.5 (i.e. when the two alleles are expressed equally).^29^ Read counts for SNPs in high LD with each other were combined to yield a more accurate estimate of allelic ratios.

Lastly, for SNPs located in the 3’UTR of the gene, we tested them to check if they interfere with miRNA target site using *in silico* tools like MirSNP ([http://bioinfo.bjmu.edu.cn/mirsnp/search/](http://bioinfo.bjmu.edu.cn/mirsnp/search/))^30^ and PolymiRTS database 3.0 ([http://compbio.uthsc.edu/miRSNP/](http://compbio.uthsc.edu/miRSNP/)).^31^ More details about those tools and the analysis approach are present in the [Supplement](#SD1).

## Results

### Characteristics of Study Participants and Thiazide Diuretics Blood Pressure Response

Characteristics and thiazide diuretic BP responses of participants, included in the discovery and replication genetic analyses, are presented in [Table 1](#T1). We found that baseline characteristics such as age, gender, body mass index (BMI) and baseline BP were similar among PEAR and PEAR-2 participants. However, CLT treated participants in PEAR-2 had greater BP lowering effects compared to HCTZ treated participants in PEAR, likely due to the fact that CLT is approximately 1.5 to 2.0 times as potent as HCTZ therapy.^32^

### Table 1.

| Characteristics | PEAR HCTZ monotherapy (N=228) | PEAR-2 CLT monotherapy (N=185) |
| --- | --- | --- |
| Age, mean (SD) years | 50 ± 9.5 | 51.1 ± 8.9 |
| Women, N (%) | 91 (40) | 79 (42.5) |
| BMI, mean (SD) kg*m−2 | 30.30 ± 4.90 | 30.66 ± 4.95 |
| Pre-treatment home SBP, mean (SD) mmHg | 146 ± 9.96 | 147.43 ± 10.31 |
| Pre-treatment home DBP, mean (SD) mmHg | 93.61 ± 5.59 | 90.28 ± 5.04 |
| Home SBP response, mean (SD) mmHg | −7.68 ± 8.1 | −12.25 ± 9.1 |
| Home DBP response, mean (SD) mmHg | −4.23 ± 5.32 | −6.83 ± 5.43 |
| Composite SBP response, mean (SD) mmHg | −8.50 ± 7.02 | NA |
| Composite DBP response, mean (SD) mmHg | −4.68 ± 4.79 | NA |

Table 1 Caption: Characteristics of PEAR and PEAR-2 white participants

Characteristics of participants selected from either PEAR or PEAR-2 for the RNA-Seq analyses are presented in [Table 2](#T2) and [3](#T3), respectively. Additionally, histograms describing the distribution of BP response among HCTZ participants in PEAR and CLT participants in PEAR-2, who were included in the RNA-Seq analyses, are presented in the data [Supplement](#SD1) ([Supplementary Figure S2](#SD1)). For PEAR RNA-Seq selected participants, baseline characteristics such as age, gender, BMI and baseline BP were not significantly different between good responders and poor responders to HCTZ. However, in PEAR-2 RNA-Seq selected participants, age was marginally significant, and gender and baseline BP were both significantly different between CLT good responders and poor responders. Accordingly, we adjusted for age, gender and baseline DBP in the analyses conducted using PEAR-2 RNA-Seq data. Of note, the mapping statistics of the RNA-Seq runs are presented in the [Supplement](#SD1) ([Supplementary Table S2](#SD1)).

### Table 2.

| Characteristics | PEAR Good Responders (N=24)* | PEAR Poor Responders (N=25) | P-value |
| --- | --- | --- | --- |
| Age, mean (SD) years | 48 ± 11.94 | 47.5 ± 8.70 | 0.85 |
| Women, N (%) | 9 (36.0) | 12 (50.0) | 0.46 |
| BMI, mean (SD) kg*m−2 | 29.3 ± 5.13 | 31.8 ± 5.90 | 0.12 |
| Pre-treatment home SBP, mean (SD) mmHg | 145.6 ± 10.2 | 144.1 ± 9.82 | 0.60 |
| Pre-treatment home DBP, mean (SD) mmHg | 93.8 ± 5.0 | 94.1 ± 4.37 | 0.81 |
| Home SBP response, mean (SD) mmHg | −11.80 ± 6.85 | −0.9 ± 5.85 | 3E-07 |
| Home DBP response, mean (SD) mmHg | −8.36 ± 5.57 | 0.08 ± 3.65 | 1E-07 |
| Composite SBP response, mean (SD) mmHg | −12 ± 6.44 | −4 ± 5.60 | 2E-05 |
| Composite DBP response, mean (SD) mmHg | −7.56 ± 4.84 | −1.6 ± 3.69 | 1E-05 |

Table 2 Caption: Characteristics of PEAR white participants included in the RNA-Seq analysis

### Table 3.

| Characteristics | PEAR-2 Good Responders (N=25) | PEAR-2 Poor responders (N=25) | P-value |
| --- | --- | --- | --- |
| Age, mean (SD) years | 53.36 ± 7.80 | 48.48 ± 10.23 | 0.064 |
| Women, N (%) | 15 (60) | 5 (20) | 4E-03 |
| BMI, mean (SD) kg*m−2 | 32.60 ± 5.14 | 30.59 ± 5.31 | 0.18 |
| Pre-treatment home SBP, mean (SD) mmHg | 152.23 ± 10.95 | 143.90 ± 8.99 | 5E-03 |
| Pre-treatment home DBP, mean (SD) mmHg | 96.78 ± 6.59 | 92.87 ± 5.02 | 0.023 |
| Home SBP response, mean (SD) mmHg | −21.66 ± 7.67 | −1.56 ± 5.06 | 1.3E-14 |
| Home DBP response, mean (SD) mmHg | −14.25 ± 4.22 | −0.26 ± 2.24 | 2.4E-19 |

Table 3 Caption: Characteristics of PEAR-2 white participants included in the RNA-Seq analysis

### Single Nucleotide Polymorphisms Prioritization Approach

Out of the 103 SNPs, none of them deviated from HWE ([Supplementary Table S1](#SD1)). Selecting SNPs in LD (r^2^>0.8) with the 103 SNPs provided a list of 1,082 SNPs. Of the 1,082 SNPs, nineteen displayed a RegulomeDB score less than 3 ([Supplementary Table S3](#SD1)). Out of those nineteen, rs10995 located in the 3’UTR of the vasodilator-stimulated phosphoprotein gene (*VASP*) was the top prioritized SNP with a RegulomeDB score of 1d. Testing the 1,082 SNPs in GWAVA identified 29 SNPs with a TSS score ≥ 0.5 ([Supplementary Table S4](#SD1)). Again, GWAVA identified rs10995 as the top prioritized SNP with a score of 0.87. Using Haploreg v4.1, we were able to annotate 290 SNPs out of the 1,082 SNPs as eQTLs. Visualizing the combined results of these different tools ([Supplementary Figure S3, Supplementary Table S5](#SD1)) highlights rs10995 as the only overlapping and top prioritized signal associated with HCTZ BP response. Thus, we tested the rs10995 association for replication in PEAR-2 whites treated with CLT.

### Replication and Further Validation

The minor allele frequency of rs10995 G>A SNP in PEAR and PEAR-2 whites was 25.6% and 25.9%, respectively. Testing the rs10995 SNP in PEAR-2 whites revealed a significant association with SBP response to CLT, and a trend toward significance with DBP response ([Figure 2](#F2)), with adjustment for baseline BP, age, gender and PC1 and 2. Although we adjusted for baseline BP in our analysis, we also tested the association between rs10995 and baseline BP in PEAR and PEAR-2 participants included in this study. Our results showed no association between the rs10995 and baseline BP measured in PEAR (DBP=0.42, SBP=0.22) and PEAR-2 (DBP=0.17, SBP=0.88). Additionally, the *in silico* analysis using Haploreg v4.1 shows that rs10995 is associated with the mRNA expression levels of four genes (*VASP*, *FOSB*, *GPR4* and *KLC3*; [Supplementary Table S6](#SD1)). Therefore, we tested baseline expression levels of those genes between HCTZ good responders and poor responders in PEAR participants. Out of the four genes, only *VASP* displayed a significantly different mRNA levels between HCTZ good responders and poor responders (p=0.01; [Figure 3A](#F3), [Supplementary Table S7](#SD1)). We found that HCTZ good responders had higher VASP mRNA levels at baseline compared to poor responders. This association was replicated in the same direction in PEAR-2 participants treated with CLT (p=0.04; [Figure 3B](#F3), [Supplementary Table S7](#SD1)). However, after adjustment for age, gender and baseline BP using EdgeR, the association weakened (adjusted p=0.12) ([Supplementary Table S8](#SD1)).

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1291/5298887/f395f63bb68f/nihms-837915-f0002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5298887_nihms-837915-f0002.jpg)

Effect of the rs10995 polymorphism on the blood pressure response of whites treated with thiazide diuretics in the PEAR and PEAR-2 studies. Blood pressure responses were adjusted for baseline blood pressure, age, gender, and PC1 and 2. P-values represented are for contrast of adjusted means between different genotype groups. Error bars represent standard error of the mean. *One sided p-value based on a one-sided hypothesis tested in the replication study. A)systolic blood pressure response to hydrochlorothiazide in the PEAR study. B)diastolic blood pressure response in the PEAR study. C)systolic blood pressure response to chlorthalidone in the PEAR-2 study. D)diastolic blood pressure response to chlorthalidone in the PEAR-2 study.

### Figure 3.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1291/5298887/6e77461c0641/nihms-837915-f0003.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5298887_nihms-837915-f0003.jpg)

Plots showing the difference in the VASP baseline mRNA expression levels between thiazide diuretics good responders and poor responders. A)PEAR study. B)PEAR-2 study.

We also found a significant association between rs10995 and VASP mRNA expression at baseline in PEAR-2 (p=5×10^−3^), and a trend toward significance in PEAR participants (p=0.06). Our analysis showed that the rs10995 G/G genotype carriers (with better response to HCTZ) had higher expression levels of VASP mRNA at baseline compared to G/A and A/A genotype carriers ([Supplementary Figure S4](#SD1)). This finding was supported by publically available eQTL data showing the same effect of the rs10995 *G-*allele on the VASP mRNA expression in several tissues ([Supplementary Table S6](#SD1)). We also sought to test whether allelic VASP mRNA expression is higher in the presence of the rs10995 *G-*allele compared to the A-allele. For rs10995, of 39 heterozygous samples tested from PEAR and PEAR-2, two samples had G/A-allele ratios greater than 2 fold, evidence of AEI, and eighteen samples displayed a ≥ 1.2 fold change ([Supplementary Figure S5](#SD1)), while the remaining sample did not show allelic ratios suggestive of AEI. Because of inherent noise in allelic RNA ratio analysis, it is not possible to assign the presence of AEI unambiguously.^33^ Thus, we tested the SNPs in high LD (r^2^>0.8) with rs10995 ([Supplementary Table S9](#SD1)) and all SNPs in the exon region of the *VASP* gene with coverage >50 reads ([Supplementary Table S10](#SD1)). From this analysis, we only focused on SNPs that have been shown in at least two heterozygous samples to have an AEI fold change >1.3. According to this criterion, we found two SNPs, rs10156 and rs546716916 with samples that had a change between their reference and variant alleles counts >1.3. The latter SNP is a rare SNP located 78 base pairs away from the rs10995 SNP, but it is not in LD with the rs10995 SNP ([Supplementary Figure S6](#SD1)). While for the rs10156 SNP, we found that it is in high LD with the rs10995 SNP (r^2^=0.7, D’=1) and laying 173 base pairs away from the rs10995 signal. rs10156 T>C SNP had higher read counts and displayed allelic ratios in the same direction as rs10995 in the vast majority of samples ([Supplementary Figure S7](#SD1)), suggesting the copy of the gene containing the reference *GT-*alleles is expressed more than the *AC*-alleles with the two SNPs combined with a mean log2 allelic expression ratio of 0.48 (p=0.004). These results support a finding of AEI, while we cannot ascertain whether rs10995 or rs10156 or both, alter mRNA expression.

Lastly, since the rs10995 SNP is located in the 3’UTR of the *VASP* gene, we tested whether rs10995 SNP interferes with a miRNA binding site. Both MirSNP and PolymiRTS database 3.0 databases indicate that the rs10995 *G-*allele creates a new binding site for miRNA 3127-3p, which enhance its binding to VASP mRNA compared to the *A-*allele ([Supplementary Table S11](#SD1)). This result suggests an additional mechanism of action that could account at least in part for the observed clinical associations.

## Discussion

Thiazide diuretics have been the mainstay anti-HTN therapy for years and are currently ranked among the most commonly prescribed first-line anti-HTN in the US. Despite their widespread use, a wide inter-individual variability in response to thiazide diuretics has been reported, which has stimulated interest in identifying predictors that can be used for optimizing the BP response to this therapy. Over the past years, results from GWAS have advanced our understanding of the potential role of genetics on the inter-individual variability in response to different drugs, including thiazide diuretics.^6, 11^ However, GWAS stringent statistical thresholds hinder the discovery of additional true genetic variants, particularly with the small sample sizes of the globally available HTN pharmacogenetic studies. Thus, in this study, we sought to identify additional novel genetic variants associated with thiazide diuretics BP response by leveraging functional data generated from the ENCODE project and other publically available eQTL datasets. We hypothesized that this approach would help prioritize the genetic signals from the GWAS and increase the chances of refining and identifying additional true signals that might be missing.

In this study, out of the 1,082 SNPs tested in our prioritization analysis, we identified and replicated the rs10995 in the *VASP* gene as a novel genetic signal with clinically relevant effects on the BP response of thiazide diuretics. Additionally, using RNA-Seq data from thiazide diuretics treated individuals showed that the rs10995 SNP represents a *VASP* eQTL. Moreover, we found that HCTZ good BP responders had higher baseline expression levels of the *VASP* gene compared to poor responders, which suggests that *VASP* might be a potential predictor of thiazide diuretics BP response. This finding was replicated in PEAR-2 participants treated with CLT, where CLT good BP responders had significantly higher VASP mRNA levels at baseline compared to poor responders. While this association weakened after adjustment for age, gender and baseline BP, this might be attributed to the small sample size used for the RNA-Seq analysis.

For further validation, we also tested whether rs10995 SNP and an adjacent SNP in high LD, rs10156, display allele-specific VASP mRNA expression. Taken together, the allelic ratio results demonstrate the presence of AEI as a marker of a genetic variant, most likely rs10995 or rs10156, or both. Of note, we did not have the rs10156 among the genotyped or imputed SNPs in our GWAS data, thus we were not able to test its association with thiazide diuretics BP response. Future studies should consider testing this SNP to check its association with HCTZ BP response. Lastly, *in silico* testing reveals that rs10995 minor allele enhances the binding of miRNA 3127-3p in the 3’UTR region. Although these results are intriguing, more experimental work is needed to test the interaction between rs10995 SNP and miRNA 3127-3p, and its clinical relevance.

VASP is highly expressed in vascular endothelial cells, smooth muscle cells and platelets.^34, 35^ As a member of the ENA/VASP protein family, VASP is involved in actin polymerization, actin cytoskeleton regulation and intracellular pathways of integrin-extracellular matrix (ECM) interaction^36^ - pathways involved in the mechanism underlying smooth muscle contraction and BP regulation.^37, 38^ VASP has also been well characterized as a substrate for cAMP- and cGMP-dependent protein kinases,^39^ known for their role in regulating contraction of vascular smooth muscle.^40^ Additionally, VASP phosphorylation has been implicated in various cellular responses ranging from endothelial cell permeability and angiogenesis^41, 42^ to platelet aggregation and secretion.^43, 44^ Moreover, it serves as a biochemical marker for monitoring nitric oxide stimulated soluble guanylyl cyclase/cGMP-dependent protein kinase type I pathway,^45^ which is involved in BP regulation, vascular remodeling and platelet, cardiac and kidney function.^46-48^ Collectively, the results of this study along with literature evidence support the hypothesis that VASP might be involved in the BP lowering mechanism of thiazides, and might serve as an important determinant of thiazide diuretics BP response. Future replication of the findings presented in this study in large well-designed studies is needed to confirm their utility in guiding the selection of thiazide diuretics for a better control on BP response. Additionally, conducting mechanistic studies on the *VASP* gene might provide insight into the mechanism underlying thiazide diuretics BP response.

Our study has several limitations. First, the small sample sizes used for the genetic and transcriptomics analyses may have limited our power to identify additional novel markers and replicate some of our genetic and transcriptomics signals. Secondly, we used whole blood-based RNA for the RNA-Seq analyses, which might not be the most relevant tissue for thiazide BP response. However, response to anti-HTN drugs might arise from a variety of target tissues. Thus, it is very difficult to select a specific tissue for testing the expression of BP genes as BP genes might be expressed in blood, heart, brain, kidney or any other specific tissue. Additionally, such tissues are essentially impossible to obtain from otherwise healthy hypertensive individuals. Lastly, whole blood samples can have different ratios of cell types, which may result in artifacts of differences in expression levels of genes that are differentially expressed between cell types. However, we did not measure differences in cell type distributions in blood for our RNA-seq runs. Thus, we were not able to account for possible changes in cell type composition. Although, it is possible that differences in cell type distribution could affect gene expression levels, however, this happens particularly with a disease state or drug treatment suspected of changing cell type content, which is not the case with the drug treatment regimen used in this study.

Our study also has several strengths. The prioritization approach used in this study was successful in identifying a novel genetic signal that we were not able to identify using traditional approaches for analyzing the GWAS output. Our findings from this approach support the hypothesis that not all SNPs are equal,^49^ and mandate the importance of utilizing functional annotations to better prioritize the GWAS output and increase the chances of identifying and replicating true signals associated with variability in drug response. We believe that using such an approach could help us to take forward the large investment in GWAS and convert the output of this approach to identify additional genetic variants, and biologically relevant pathways associated with drug response.

In summary, to our knowledge, this is the first study to highlight the association between the *VASP* gene and thiazide diuretics BP response. This study illustrates the power of utilizing the publically available ENCODE data and eQTL datasets to prioritize the GWAS output and increase the probability of identifying novel genetic variants underlying drug response. Future use of these publically available data might give us more insight into the mechanism underlying BP response, which might facilitate the development of new drugs and therapeutic approaches that can be utilized for optimizing anti-hypertensive BP response.

## Supplementary Material

Thiazide diuretics have been the cornerstone in managing hypertension for more than half a century, yet when used as monotherapy, only about half of thiazide treated patients achieve their target blood pressure (BP) goal. Thus, identifying biomarkers to predict variability in BP response to thiazides may have future clinical utility. Herein, we used publically available ENCODE data along with expression quantitative trait loci (eQTL) data to prioritize for further investigation of genetic signals from a GWAS of BP response to hydrochlorothiazide. Our prioritization approach identified rs10995 variant in the vasodilator-stimulated phosphoprotein (*VASP)* as the most functional genetic signal associated with BP response. This genetic variant was then replicated in an independent group of hypertensives treated with chlorthalidone. We found rs10995 *G-*allele carriers had better BP response to thiazides *vs.* non-carriers, and the *G-allele* had higher mRNA expression of *VASP*. We also identified that baseline expression of VASP mRNA was significantly higher in people with a better BP response to thiazide diuretics. This finding was also replicated in an independent group of patients treated with chlorthalidone. The results from this study may help advance a personalized approach to antihypertensive therapy. The rs10995 variant should be tested in other independent cohorts to confirm its utility in guiding the selection of thiazide therapy. Additionally, future work on the VASP association with the BP response to thiazides may provide more insights into the chronic BP lowering mechanism underlying thiazide diuretics and may open new avenues for identifying novel antihypertensive drug targets.

Thiazide diuretics have been the cornerstone in managing hypertension for more than half a century, yet when used as monotherapy, only about half of thiazide treated patients achieve their target blood pressure (BP) goal. Thus, identifying biomarkers to predict variability in BP response to thiazides may have future clinical utility. Herein, we used publically available ENCODE data along with expression quantitative trait loci (eQTL) data to prioritize for further investigation of genetic signals from a GWAS of BP response to hydrochlorothiazide. Our prioritization approach identified rs10995 variant in the vasodilator-stimulated phosphoprotein (*VASP)* as the most functional genetic signal associated with BP response. This genetic variant was then replicated in an independent group of hypertensives treated with chlorthalidone. We found rs10995 *G-*allele carriers had better BP response to thiazides *vs.* non-carriers, and the *G-allele* had higher mRNA expression of *VASP*. We also identified that baseline expression of VASP mRNA was significantly higher in people with a better BP response to thiazide diuretics. This finding was also replicated in an independent group of patients treated with chlorthalidone. The results from this study may help advance a personalized approach to antihypertensive therapy. The rs10995 variant should be tested in other independent cohorts to confirm its utility in guiding the selection of thiazide therapy. Additionally, future work on the VASP association with the BP response to thiazides may provide more insights into the chronic BP lowering mechanism underlying thiazide diuretics and may open new avenues for identifying novel antihypertensive drug targets.

## Acknowledgments

We appreciate the valuable contributions of the Pharmacogenomic Evaluation of Antihypertensive Responses (PEAR) and PEAR-2 study participants, support staff and study physicians.

**Sources of Funding:** PEAR was supported by the National Institute of Health Pharmacogenetics Research Network grant U01 GM074492 and the National Center for Advancing Translational Sciences under the award number UL1 TR000064 (University of Florida); UL1 TR000454 (Emory University) and UL1 TR000135 (Mayo Clinic). PEAR was also supported by funds from the Mayo Foundation. Additional support for this work includes: U01 GM092655 (W.S.) and AHA pre-doctoral fellowship award #14PRE20460115 (M.H.S.).

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Oparil S, Schmieder RE. New approaches in the treatment of hypertension. Circ Res. 2015;116:1074–1095. doi: 10.1161/CIRCRESAHA.116.303603.  [DOI](https://doi.org/10.1161/CIRCRESAHA.116.303603) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25767291/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circ%20Res&title=New%20approaches%20in%20the%20treatment%20of%20hypertension&author=S%20Oparil&author=RE%20Schmieder&volume=116&publication_year=2015&pages=1074-1095&pmid=25767291&doi=10.1161/CIRCRESAHA.116.303603&)

2. Lewington S, Clarke R, Qizilbash N, Peto R, Collins R, Prospective Studies C Age-specific relevance of usual blood pressure to vascular mortality: a meta-analysis of individual data for one million adults in 61 prospective studies. Lancet. 2002;360:1903–1913. doi: 10.1016/s0140-6736(02)11911-8.  [DOI](https://doi.org/10.1016/s0140-6736(02)11911-8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12493255/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Age-specific%20relevance%20of%20usual%20blood%20pressure%20to%20vascular%20mortality:%20a%20meta-analysis%20of%20individual%20data%20for%20one%20million%20adults%20in%2061%20prospective%20studies&author=S%20Lewington&author=R%20Clarke&author=N%20Qizilbash&author=R%20Peto&author=R%20Collins&volume=360&publication_year=2002&pages=1903-1913&pmid=12493255&doi=10.1016/s0140-6736(02)11911-8&)

3. Mancia G. Blood pressure reduction and cardiovascular outcomes: past, present, and future. Am J Cardiol. 2007;100:3J–9J. doi: 10.1016/j.amjcard.2007.05.008.  [DOI](https://doi.org/10.1016/j.amjcard.2007.05.008) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17666195/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Cardiol&title=Blood%20pressure%20reduction%20and%20cardiovascular%20outcomes:%20past,%20present,%20and%20future&author=G%20Mancia&volume=100&publication_year=2007&pages=3J-9J&pmid=17666195&doi=10.1016/j.amjcard.2007.05.008&)

4. IMS Institue for Healthcare Informatics Medicines Use and Spending Shifts: A Review of the Use of Medicines in the U.S. 2014 Accessed October 4th, 2016.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Medicines%20Use%20and%20Spending%20Shifts:%20A%20Review%20of%20the%20Use%20of%20Medicines%20in%20the%20U.S&publication_year=2014&)

5. Weber MA, Schiffrin EL, White WB, Mann S, Lindholm LH, Kenerson JG, et al. Clinical practice guidelines for the management of hypertension in the community a statement by the American Society of Hypertension and the International Society of Hypertension. J Hypertens. 2014;32:3–15. doi: 10.1097/HJH.0000000000000065.  [DOI](https://doi.org/10.1097/HJH.0000000000000065) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24270181/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Hypertens&title=Clinical%20practice%20guidelines%20for%20the%20management%20of%20hypertension%20in%20the%20community%20a%20statement%20by%20the%20American%20Society%20of%20Hypertension%20and%20the%20International%20Society%20of%20Hypertension&author=MA%20Weber&author=EL%20Schiffrin&author=WB%20White&author=S%20Mann&author=LH%20Lindholm&volume=32&publication_year=2014&pages=3-15&pmid=24270181&doi=10.1097/HJH.0000000000000065&)

6. Shahin MH, Johnson JA. Mechanisms and pharmacogenetic signals underlying thiazide diuretics blood pressure response. Curr Opin Pharmacol. 2016;27:31–37. doi: 10.1016/j.coph.2016.01.005.  [DOI](https://doi.org/10.1016/j.coph.2016.01.005) | [PMC free article](/articles/PMC4915478/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26874237/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Opin%20Pharmacol&title=Mechanisms%20and%20pharmacogenetic%20signals%20underlying%20thiazide%20diuretics%20blood%20pressure%20response&author=MH%20Shahin&author=JA%20Johnson&volume=27&publication_year=2016&pages=31-37&pmid=26874237&doi=10.1016/j.coph.2016.01.005&)

7. Materson BJ. Variability in response to antihypertensive drugs. Am J Med. 2007;120:S10–20. doi: 10.1016/j.amjmed.2007.02.003.  [DOI](https://doi.org/10.1016/j.amjmed.2007.02.003) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17403377/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Med&title=Variability%20in%20response%20to%20antihypertensive%20drugs&author=BJ%20Materson&volume=120&publication_year=2007&pages=S10-20&pmid=17403377&doi=10.1016/j.amjmed.2007.02.003&)

8. Materson BJ, Reda DJ, Cushman WC, Massie BM, Freis ED, Kochar MS, et al. Single-drug therapy for hypertension in men. A comparison of six antihypertensive agents with placebo. The Department of Veterans Affairs Cooperative Study Group on Antihypertensive Agents. N Engl J Med. 1993;328:914–921. doi: 10.1056/NEJM199304013281303.  [DOI](https://doi.org/10.1056/NEJM199304013281303) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8446138/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Single-drug%20therapy%20for%20hypertension%20in%20men.%20A%20comparison%20of%20six%20antihypertensive%20agents%20with%20placebo.%20The%20Department%20of%20Veterans%20Affairs%20Cooperative%20Study%20Group%20on%20Antihypertensive%20Agents&author=BJ%20Materson&author=DJ%20Reda&author=WC%20Cushman&author=BM%20Massie&author=ED%20Freis&volume=328&publication_year=1993&pages=914-921&pmid=8446138&doi=10.1056/NEJM199304013281303&)

9. Thoenes M, Neuberger HR, Volpe M, Khan BV, Kirch W, Bohm M. Antihypertensive drug therapy and blood pressure control in men and women: an international perspective. J Hum Hypertens. 2010;24:336–344. doi: 10.1038/jhh.2009.76.  [DOI](https://doi.org/10.1038/jhh.2009.76) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19798089/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Hum%20Hypertens&title=Antihypertensive%20drug%20therapy%20and%20blood%20pressure%20control%20in%20men%20and%20women:%20an%20international%20perspective&author=M%20Thoenes&author=HR%20Neuberger&author=M%20Volpe&author=BV%20Khan&author=W%20Kirch&volume=24&publication_year=2010&pages=336-344&pmid=19798089&doi=10.1038/jhh.2009.76&)

10. Perera MA, Cavallari LH, Limdi NA, Gamazon ER, Konkashbaev A, Daneshjou R, et al. Genetic variants associated with warfarin dose in African-American individuals: a genome-wide association study. Lancet. 2013;382:790–796. doi: 10.1016/S0140-6736(13)60681-9.  [DOI](https://doi.org/10.1016/S0140-6736(13)60681-9) | [PMC free article](/articles/PMC3759580/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23755828/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Genetic%20variants%20associated%20with%20warfarin%20dose%20in%20African-American%20individuals:%20a%20genome-wide%20association%20study&author=MA%20Perera&author=LH%20Cavallari&author=NA%20Limdi&author=ER%20Gamazon&author=A%20Konkashbaev&volume=382&publication_year=2013&pages=790-796&pmid=23755828&doi=10.1016/S0140-6736(13)60681-9&)

11. Turner ST, Boerwinkle E, O'Connell JR, Bailey KR, Gong Y, Chapman AB, et al. Genomic association analysis of common variants influencing antihypertensive response to hydrochlorothiazide. Hypertension. 2013;62:391–397. doi: 10.1161/HYPERTENSIONAHA.111.00436.  [DOI](https://doi.org/10.1161/HYPERTENSIONAHA.111.00436) | [PMC free article](/articles/PMC3780966/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23753411/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hypertension&title=Genomic%20association%20analysis%20of%20common%20variants%20influencing%20antihypertensive%20response%20to%20hydrochlorothiazide&author=ST%20Turner&author=E%20Boerwinkle&author=JR%20O'Connell&author=KR%20Bailey&author=Y%20Gong&volume=62&publication_year=2013&pages=391-397&pmid=23753411&doi=10.1161/HYPERTENSIONAHA.111.00436&)

12. Nicolae DL, Gamazon E, Zhang W, Duan S, Dolan ME, Cox NJ. Trait-associated SNPs are more likely to be eQTLs: annotation to enhance discovery from GWAS. PLoS Genet. 2010;6:e1000888. doi: 10.1371/journal.pgen.1000888.  [DOI](https://doi.org/10.1371/journal.pgen.1000888) | [PMC free article](/articles/PMC2848547/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20369019/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet&title=Trait-associated%20SNPs%20are%20more%20likely%20to%20be%20eQTLs:%20annotation%20to%20enhance%20discovery%20from%20GWAS&author=DL%20Nicolae&author=E%20Gamazon&author=W%20Zhang&author=S%20Duan&author=ME%20Dolan&volume=6&publication_year=2010&pages=e1000888&pmid=20369019&doi=10.1371/journal.pgen.1000888&)

13. Stranger BE, Stahl EA, Raj T. Progress and promise of genome-wide association studies for human complex trait genetics. Genetics. 2011;187:367–383. doi: 10.1534/genetics.110.120907.  [DOI](https://doi.org/10.1534/genetics.110.120907) | [PMC free article](/articles/PMC3030483/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21115973/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genetics&title=Progress%20and%20promise%20of%20genome-wide%20association%20studies%20for%20human%20complex%20trait%20genetics&author=BE%20Stranger&author=EA%20Stahl&author=T%20Raj&volume=187&publication_year=2011&pages=367-383&pmid=21115973&doi=10.1534/genetics.110.120907&)

14. Maurano MT, Humbert R, Rynes E, Thurman RE, Haugen E, Wang H, et al. Systematic localization of common disease-associated variation in regulatory DNA. Science. 2012;337:1190–1195. doi: 10.1126/science.1222794.  [DOI](https://doi.org/10.1126/science.1222794) | [PMC free article](/articles/PMC3771521/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22955828/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=Systematic%20localization%20of%20common%20disease-associated%20variation%20in%20regulatory%20DNA&author=MT%20Maurano&author=R%20Humbert&author=E%20Rynes&author=RE%20Thurman&author=E%20Haugen&volume=337&publication_year=2012&pages=1190-1195&pmid=22955828&doi=10.1126/science.1222794&)

15. Dunham I, Kundaje A, Aldred SF, Collins PJ, Davis CA, Doyle F, et al. An integrated encyclopedia of DNA elements in the human genome. Nature. 2012;489:57–74. doi: 10.1038/nature11247.  [DOI](https://doi.org/10.1038/nature11247) | [PMC free article](/articles/PMC3439153/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22955616/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=An%20integrated%20encyclopedia%20of%20DNA%20elements%20in%20the%20human%20genome&author=I%20Dunham&author=A%20Kundaje&author=SF%20Aldred&author=PJ%20Collins&author=CA%20Davis&volume=489&publication_year=2012&pages=57-74&pmid=22955616&doi=10.1038/nature11247&)

16. Schaub MA, Boyle AP, Kundaje A, Batzoglou S, Snyder M. Linking disease associations with regulatory information in the human genome. Genome Res. 2012;22:1748–1759. doi: 10.1101/gr.136127.111.  [DOI](https://doi.org/10.1101/gr.136127.111) | [PMC free article](/articles/PMC3431491/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22955986/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Res&title=Linking%20disease%20associations%20with%20regulatory%20information%20in%20the%20human%20genome&author=MA%20Schaub&author=AP%20Boyle&author=A%20Kundaje&author=S%20Batzoglou&author=M%20Snyder&volume=22&publication_year=2012&pages=1748-1759&pmid=22955986&doi=10.1101/gr.136127.111&)

17. Johnson JA, Boerwinkle E, Zineh I, Chapman AB, Bailey K, Cooper-DeHoff RM, et al. Pharmacogenomics of antihypertensive drugs: rationale and design of the Pharmacogenomic Evaluation of Antihypertensive Responses (PEAR) study. Am Heart J. 2009;157:442–449. doi: 10.1016/j.ahj.2008.11.018.  [DOI](https://doi.org/10.1016/j.ahj.2008.11.018) | [PMC free article](/articles/PMC2671287/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19249413/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20Heart%20J&title=Pharmacogenomics%20of%20antihypertensive%20drugs:%20rationale%20and%20design%20of%20the%20Pharmacogenomic%20Evaluation%20of%20Antihypertensive%20Responses%20(PEAR)%20study&author=JA%20Johnson&author=E%20Boerwinkle&author=I%20Zineh&author=AB%20Chapman&author=K%20Bailey&volume=157&publication_year=2009&pages=442-449&pmid=19249413&doi=10.1016/j.ahj.2008.11.018&)

18. Hamadeh IS, Langaee TY, Dwivedi R, Garcia S, Burkley BM, Skaar TC, et al. Impact of CYP2D6 polymorphisms on clinical efficacy and tolerability of metoprolol tartrate. Clin Pharmacol Ther. 2014;96:175–181. doi: 10.1038/clpt.2014.62.  [DOI](https://doi.org/10.1038/clpt.2014.62) | [PMC free article](/articles/PMC4111800/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24637943/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Impact%20of%20CYP2D6%20polymorphisms%20on%20clinical%20efficacy%20and%20tolerability%20of%20metoprolol%20tartrate&author=IS%20Hamadeh&author=TY%20Langaee&author=R%20Dwivedi&author=S%20Garcia&author=BM%20Burkley&volume=96&publication_year=2014&pages=175-181&pmid=24637943&doi=10.1038/clpt.2014.62&)

19. Turner ST, Schwartz GL, Chapman AB, Beitelshees AL, Gums JG, Cooper-Dehoff RM, et al. Power to identify a genetic predictor of antihypertensive drug response using different methods to measure blood pressure response. J Transl Med. 2012;10:47. doi: 10.1186/1479-5876-10-47.  [DOI](https://doi.org/10.1186/1479-5876-10-47) | [PMC free article](/articles/PMC3342146/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22413836/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Transl%20Med&title=Power%20to%20identify%20a%20genetic%20predictor%20of%20antihypertensive%20drug%20response%20using%20different%20methods%20to%20measure%20blood%20pressure%20response&author=ST%20Turner&author=GL%20Schwartz&author=AB%20Chapman&author=AL%20Beitelshees&author=JG%20Gums&volume=10&publication_year=2012&pages=47&pmid=22413836&doi=10.1186/1479-5876-10-47&)

20. Stergiou GS, Baibas NM, Gantzarou AP, Skeva II, Kalkana CB, Roussias LG, et al. Reproducibility of home, ambulatory, and clinic blood pressure: implications for the design of trials for the assessment of antihypertensive drug efficacy. Am J Hypertens. 2002;15:101–104. doi: 10.1016/s0895-7061(01)02324-x.  [DOI](https://doi.org/10.1016/s0895-7061(01)02324-x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11863243/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hypertens&title=Reproducibility%20of%20home,%20ambulatory,%20and%20clinic%20blood%20pressure:%20implications%20for%20the%20design%20of%20trials%20for%20the%20assessment%20of%20antihypertensive%20drug%20efficacy&author=GS%20Stergiou&author=NM%20Baibas&author=AP%20Gantzarou&author=II%20Skeva&author=CB%20Kalkana&volume=15&publication_year=2002&pages=101-104&pmid=11863243&doi=10.1016/s0895-7061(01)02324-x&)

21. Levin JZ, Yassour M, Adiconis X, Nusbaum C, Thompson DA, Friedman N, et al. Comprehensive comparative analysis of strand-specific RNA sequencing methods. Nat Methods. 2010;7:709–715. doi: 10.1038/nmeth.1491.  [DOI](https://doi.org/10.1038/nmeth.1491) | [PMC free article](/articles/PMC3005310/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20711195/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Methods&title=Comprehensive%20comparative%20analysis%20of%20strand-specific%20RNA%20sequencing%20methods&author=JZ%20Levin&author=M%20Yassour&author=X%20Adiconis&author=C%20Nusbaum&author=DA%20Thompson&volume=7&publication_year=2010&pages=709-715&pmid=20711195&doi=10.1038/nmeth.1491&)

22. Trapnell C, Roberts A, Goff L, Pertea G, Kim D, Kelley DR, et al. Differential gene and transcript expression analysis of RNA-seq experiments with TopHat and Cufflinks. Nat Protoc. 2012;7:562–578. doi: 10.1038/nprot.2012.016.  [DOI](https://doi.org/10.1038/nprot.2012.016) | [PMC free article](/articles/PMC3334321/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22383036/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Protoc&title=Differential%20gene%20and%20transcript%20expression%20analysis%20of%20RNA-seq%20experiments%20with%20TopHat%20and%20Cufflinks&author=C%20Trapnell&author=A%20Roberts&author=L%20Goff&author=G%20Pertea&author=D%20Kim&volume=7&publication_year=2012&pages=562-578&pmid=22383036&doi=10.1038/nprot.2012.016&)

23. Anders S, Pyl PT, Huber W. HTSeq--a Python framework to work with high-throughput sequencing data. Bioinformatics. 2015;31:166–169. doi: 10.1093/bioinformatics/btu638.  [DOI](https://doi.org/10.1093/bioinformatics/btu638) | [PMC free article](/articles/PMC4287950/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25260700/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=HTSeq--a%20Python%20framework%20to%20work%20with%20high-throughput%20sequencing%20data&author=S%20Anders&author=PT%20Pyl&author=W%20Huber&volume=31&publication_year=2015&pages=166-169&pmid=25260700&doi=10.1093/bioinformatics/btu638&)

24. Robinson MD, McCarthy DJ, Smyth GK. edgeR: a Bioconductor package for differential expression analysis of digital gene expression data. Bioinformatics. 2010;26:139–140. doi: 10.1093/bioinformatics/btp616.  [DOI](https://doi.org/10.1093/bioinformatics/btp616) | [PMC free article](/articles/PMC2796818/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19910308/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=edgeR:%20a%20Bioconductor%20package%20for%20differential%20expression%20analysis%20of%20digital%20gene%20expression%20data&author=MD%20Robinson&author=DJ%20McCarthy&author=GK%20Smyth&volume=26&publication_year=2010&pages=139-140&pmid=19910308&doi=10.1093/bioinformatics/btp616&)

25. Boyle AP, Hong EL, Hariharan M, Cheng Y, Schaub MA, Kasowski M, et al. Annotation of functional variation in personal genomes using RegulomeDB. Genome Res. 2012;22:1790–1797. doi: 10.1101/gr.137323.112.  [DOI](https://doi.org/10.1101/gr.137323.112) | [PMC free article](/articles/PMC3431494/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22955989/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Res&title=Annotation%20of%20functional%20variation%20in%20personal%20genomes%20using%20RegulomeDB&author=AP%20Boyle&author=EL%20Hong&author=M%20Hariharan&author=Y%20Cheng&author=MA%20Schaub&volume=22&publication_year=2012&pages=1790-1797&pmid=22955989&doi=10.1101/gr.137323.112&)

26. Ritchie GR, Dunham I, Zeggini E, Flicek P. Functional annotation of noncoding sequence variants. Nat Methods. 2014;11:294–296. doi: 10.1038/nmeth.2832.  [DOI](https://doi.org/10.1038/nmeth.2832) | [PMC free article](/articles/PMC5015703/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24487584/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Methods&title=Functional%20annotation%20of%20noncoding%20sequence%20variants&author=GR%20Ritchie&author=I%20Dunham&author=E%20Zeggini&author=P%20Flicek&volume=11&publication_year=2014&pages=294-296&pmid=24487584&doi=10.1038/nmeth.2832&)

27. Ward LD, Kellis M. HaploReg: a resource for exploring chromatin states, conservation, and regulatory motif alterations within sets of genetically linked variants. Nucleic Acids Res. 2012;40:D930–934. doi: 10.1093/nar/gkr917.  [DOI](https://doi.org/10.1093/nar/gkr917) | [PMC free article](/articles/PMC3245002/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22064851/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=HaploReg:%20a%20resource%20for%20exploring%20chromatin%20states,%20conservation,%20and%20regulatory%20motif%20alterations%20within%20sets%20of%20genetically%20linked%20variants&author=LD%20Ward&author=M%20Kellis&volume=40&publication_year=2012&pages=D930-934&pmid=22064851&doi=10.1093/nar/gkr917&)

28. Li H, Handsaker B, Wysoker A, Fennell T, Ruan J, Homer N, et al. The Sequence Alignment/Map format and SAMtools. Bioinformatics. 2009;25:2078–2079. doi: 10.1093/bioinformatics/btp352.  [DOI](https://doi.org/10.1093/bioinformatics/btp352) | [PMC free article](/articles/PMC2723002/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19505943/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=The%20Sequence%20Alignment/Map%20format%20and%20SAMtools&author=H%20Li&author=B%20Handsaker&author=A%20Wysoker&author=T%20Fennell&author=J%20Ruan&volume=25&publication_year=2009&pages=2078-2079&pmid=19505943&doi=10.1093/bioinformatics/btp352&)

29. Li G, Bahn JH, Lee JH, Peng G, Chen Z, Nelson SF, et al. Identification of allele-specific alternative mRNA processing via transcriptome sequencing. Nucleic Acids Res. 2012;40:e104. doi: 10.1093/nar/gks280.  [DOI](https://doi.org/10.1093/nar/gks280) | [PMC free article](/articles/PMC3401465/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22467206/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=Identification%20of%20allele-specific%20alternative%20mRNA%20processing%20via%20transcriptome%20sequencing&author=G%20Li&author=JH%20Bahn&author=JH%20Lee&author=G%20Peng&author=Z%20Chen&volume=40&publication_year=2012&pages=e104&pmid=22467206&doi=10.1093/nar/gks280&)

30. Liu C, Zhang F, Li T, Lu M, Wang L, Yue W, et al. MirSNP, a database of polymorphisms altering miRNA target sites, identifies miRNA-related SNPs in GWAS SNPs and eQTLs. BMC Genomics. 2012;13:661. doi: 10.1186/1471-2164-13-661.  [DOI](https://doi.org/10.1186/1471-2164-13-661) | [PMC free article](/articles/PMC3582533/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23173617/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Genomics&title=MirSNP,%20a%20database%20of%20polymorphisms%20altering%20miRNA%20target%20sites,%20identifies%20miRNA-related%20SNPs%20in%20GWAS%20SNPs%20and%20eQTLs&author=C%20Liu&author=F%20Zhang&author=T%20Li&author=M%20Lu&author=L%20Wang&volume=13&publication_year=2012&pages=661&pmid=23173617&doi=10.1186/1471-2164-13-661&)

31. Bhattacharya A, Ziebarth JD, Cui Y. PolymiRTS Database 3.0: linking polymorphisms in microRNAs and their target sites with human diseases and biological pathways. Nucleic Acids Res. 2014;42:D86–91. doi: 10.1093/nar/gkt1028.  [DOI](https://doi.org/10.1093/nar/gkt1028) | [PMC free article](/articles/PMC3965097/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24163105/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=PolymiRTS%20Database%203.0:%20linking%20polymorphisms%20in%20microRNAs%20and%20their%20target%20sites%20with%20human%20diseases%20and%20biological%20pathways&author=A%20Bhattacharya&author=JD%20Ziebarth&author=Y%20Cui&volume=42&publication_year=2014&pages=D86-91&pmid=24163105&doi=10.1093/nar/gkt1028&)

32. Ernst ME, Carter BL, Goerdt CJ, Steffensmeier JJ, Phillips BB, Zimmerman MB, et al. Comparative antihypertensive effects of hydrochlorothiazide and chlorthalidone on ambulatory and office blood pressure. Hypertension. 2006;47:352–358. doi: 10.1161/01.HYP.0000203309.07140.d3.  [DOI](https://doi.org/10.1161/01.HYP.0000203309.07140.d3) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16432050/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hypertension&title=Comparative%20antihypertensive%20effects%20of%20hydrochlorothiazide%20and%20chlorthalidone%20on%20ambulatory%20and%20office%20blood%20pressure&author=ME%20Ernst&author=BL%20Carter&author=CJ%20Goerdt&author=JJ%20Steffensmeier&author=BB%20Phillips&volume=47&publication_year=2006&pages=352-358&pmid=16432050&doi=10.1161/01.HYP.0000203309.07140.d3&)

33. Smith RM, Webb A, Papp AC, Newman LC, Handelman SK, Suhy A, et al. Whole transcriptome RNA-Seq allelic expression in human brain. BMC Genomics. 2013;14:571. doi: 10.1186/1471-2164-14-571.  [DOI](https://doi.org/10.1186/1471-2164-14-571) | [PMC free article](/articles/PMC3765493/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23968248/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Genomics&title=Whole%20transcriptome%20RNA-Seq%20allelic%20expression%20in%20human%20brain&author=RM%20Smith&author=A%20Webb&author=AC%20Papp&author=LC%20Newman&author=SK%20Handelman&volume=14&publication_year=2013&pages=571&pmid=23968248&doi=10.1186/1471-2164-14-571&)

34. Markert T, Krenn V, Leebmann J, Walter U. High expression of the focal adhesion- and microfilament-associated protein VASP in vascular smooth muscle and endothelial cells of the intact human vessel wall. Basic Res Cardiol. 1996;91:337–343. doi: 10.1007/BF00788712.  [DOI](https://doi.org/10.1007/BF00788712) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8922250/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Basic%20Res%20Cardiol&title=High%20expression%20of%20the%20focal%20adhesion-%20and%20microfilament-associated%20protein%20VASP%20in%20vascular%20smooth%20muscle%20and%20endothelial%20cells%20of%20the%20intact%20human%20vessel%20wall&author=T%20Markert&author=V%20Krenn&author=J%20Leebmann&author=U%20Walter&volume=91&publication_year=1996&pages=337-343&pmid=8922250&doi=10.1007/BF00788712&)

35. Haffner C, Jarchau T, Reinhard M, Hoppe J, Lohmann SM, Walter U. Molecular cloning, structural analysis and functional expression of the proline-rich focal adhesion and microfilament-associated protein VASP. EMBO J. 1995;14:19–27. doi: 10.1002/j.1460-2075.1995.tb06971.x.  [DOI](https://doi.org/10.1002/j.1460-2075.1995.tb06971.x) | [PMC free article](/articles/PMC398048/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7828592/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=EMBO%20J&title=Molecular%20cloning,%20structural%20analysis%20and%20functional%20expression%20of%20the%20proline-rich%20focal%20adhesion%20and%20microfilament-associated%20protein%20VASP&author=C%20Haffner&author=T%20Jarchau&author=M%20Reinhard&author=J%20Hoppe&author=SM%20Lohmann&volume=14&publication_year=1995&pages=19-27&pmid=7828592&doi=10.1002/j.1460-2075.1995.tb06971.x&)

36. Krause M, Dent EW, Bear JE, Loureiro JJ, Gertler FB. Ena/VASP proteins: regulators of the actin cytoskeleton and cell migration. Annu Rev Cell Dev Biol. 2003;19:541–564. doi: 10.1146/annurev.cellbio.19.050103.103356.  [DOI](https://doi.org/10.1146/annurev.cellbio.19.050103.103356) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14570581/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Annu%20Rev%20Cell%20Dev%20Biol&title=Ena/VASP%20proteins:%20regulators%20of%20the%20actin%20cytoskeleton%20and%20cell%20migration&author=M%20Krause&author=EW%20Dent&author=JE%20Bear&author=JJ%20Loureiro&author=FB%20Gertler&volume=19&publication_year=2003&pages=541-564&pmid=14570581&doi=10.1146/annurev.cellbio.19.050103.103356&)

37. Yamin R, Morgan KG. Deciphering actin cytoskeletal function in the contractile vascular smooth muscle cell. J Physiol. 2012;590:4145–4154. doi: 10.1113/jphysiol.2012.232306.  [DOI](https://doi.org/10.1113/jphysiol.2012.232306) | [PMC free article](/articles/PMC3473273/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22687615/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Physiol&title=Deciphering%20actin%20cytoskeletal%20function%20in%20the%20contractile%20vascular%20smooth%20muscle%20cell&author=R%20Yamin&author=KG%20Morgan&volume=590&publication_year=2012&pages=4145-4154&pmid=22687615&doi=10.1113/jphysiol.2012.232306&)

38. Gunst SJ, Zhang W. Actin cytoskeletal dynamics in smooth muscle: a new paradigm for the regulation of smooth muscle contraction. Am J Physiol Cell Physiol. 2008;295:C576–587. doi: 10.1152/ajpcell.00253.2008.  [DOI](https://doi.org/10.1152/ajpcell.00253.2008) | [PMC free article](/articles/PMC2544441/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18596210/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Physiol%20Cell%20Physiol&title=Actin%20cytoskeletal%20dynamics%20in%20smooth%20muscle:%20a%20new%20paradigm%20for%20the%20regulation%20of%20smooth%20muscle%20contraction&author=SJ%20Gunst&author=W%20Zhang&volume=295&publication_year=2008&pages=C576-587&pmid=18596210&doi=10.1152/ajpcell.00253.2008&)

39. Walter U, Eigenthaler M, Geiger J, Reinhard M. Role of cyclic nucleotide-dependent protein kinases and their common substrate VASP in the regulation of human platelets. Adv Exp Med Biol. 1993;344:237–249. doi: 10.1007/978-1-4615-2994-1_19.  [DOI](https://doi.org/10.1007/978-1-4615-2994-1_19) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8209791/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Adv%20Exp%20Med%20Biol&title=Role%20of%20cyclic%20nucleotide-dependent%20protein%20kinases%20and%20their%20common%20substrate%20VASP%20in%20the%20regulation%20of%20human%20platelets&author=U%20Walter&author=M%20Eigenthaler&author=J%20Geiger&author=M%20Reinhard&volume=344&publication_year=1993&pages=237-249&pmid=8209791&doi=10.1007/978-1-4615-2994-1_19&)

40. Hofmann F, Feil R, Kleppisch T, Schlossmann J. Function of cGMP-dependent protein kinases as revealed by gene deletion. Physiol Rev. 2006;86:1–23. doi: 10.1152/physrev.00015.2005.  [DOI](https://doi.org/10.1152/physrev.00015.2005) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16371594/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Physiol%20Rev&title=Function%20of%20cGMP-dependent%20protein%20kinases%20as%20revealed%20by%20gene%20deletion&author=F%20Hofmann&author=R%20Feil&author=T%20Kleppisch&author=J%20Schlossmann&volume=86&publication_year=2006&pages=1-23&pmid=16371594&doi=10.1152/physrev.00015.2005&)

41. Comerford KM, Lawrence DW, Synnestvedt K, Levi BP, Colgan SP. Role of vasodilator-stimulated phosphoprotein in PKA-induced changes in endothelial junctional permeability. FASEB J. 2002;16:583–585. doi: 10.1096/fj.01-0739fje.  [DOI](https://doi.org/10.1096/fj.01-0739fje) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11919161/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=FASEB%20J&title=Role%20of%20vasodilator-stimulated%20phosphoprotein%20in%20PKA-induced%20changes%20in%20endothelial%20junctional%20permeability&author=KM%20Comerford&author=DW%20Lawrence&author=K%20Synnestvedt&author=BP%20Levi&author=SP%20Colgan&volume=16&publication_year=2002&pages=583-585&pmid=11919161&doi=10.1096/fj.01-0739fje&)

42. Chen H, Levine YC, Golan DE, Michel T, Lin AJ. Atrial natriuretic peptide-initiated cGMP pathways regulate vasodilator-stimulated phosphoprotein phosphorylation and angiogenesis in vascular endothelium. J Biol Chem. 2008;283:4439–4447. doi: 10.1074/jbc.M709439200.  [DOI](https://doi.org/10.1074/jbc.M709439200) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18079117/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=Atrial%20natriuretic%20peptide-initiated%20cGMP%20pathways%20regulate%20vasodilator-stimulated%20phosphoprotein%20phosphorylation%20and%20angiogenesis%20in%20vascular%20endothelium&author=H%20Chen&author=YC%20Levine&author=DE%20Golan&author=T%20Michel&author=AJ%20Lin&volume=283&publication_year=2008&pages=4439-4447&pmid=18079117&doi=10.1074/jbc.M709439200&)

43. An D, Rodrigues B. Role of changes in cardiac metabolism in development of diabetic cardiomyopathy. Am J Physiol Heart Circ Physiol. 2006;291:H1489–1506. doi: 10.1152/ajpheart.00278.2006.  [DOI](https://doi.org/10.1152/ajpheart.00278.2006) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16751293/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Physiol%20Heart%20Circ%20Physiol&title=Role%20of%20changes%20in%20cardiac%20metabolism%20in%20development%20of%20diabetic%20cardiomyopathy&author=D%20An&author=B%20Rodrigues&volume=291&publication_year=2006&pages=H1489-1506&pmid=16751293&doi=10.1152/ajpheart.00278.2006&)

44. Aszodi A, Pfeifer A, Ahmad M, Glauner M, Zhou XH, Ny L, et al. The vasodilator-stimulated phosphoprotein (VASP) is involved in cGMP- and cAMP-mediated inhibition of agonist-induced platelet aggregation, but is dispensable for smooth muscle function. EMBO J. 1999;18:37–48. doi: 10.1093/emboj/18.1.37.  [DOI](https://doi.org/10.1093/emboj/18.1.37) | [PMC free article](/articles/PMC1171100/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9878048/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=EMBO%20J&title=The%20vasodilator-stimulated%20phosphoprotein%20(VASP)%20is%20involved%20in%20cGMP-%20and%20cAMP-mediated%20inhibition%20of%20agonist-induced%20platelet%20aggregation,%20but%20is%20dispensable%20for%20smooth%20muscle%20function&author=A%20Aszodi&author=A%20Pfeifer&author=M%20Ahmad&author=M%20Glauner&author=XH%20Zhou&volume=18&publication_year=1999&pages=37-48&pmid=9878048&doi=10.1093/emboj/18.1.37&)

45. Oelze M, Mollnau H, Hoffmann N, Warnholtz A, Bodenschatz M, Smolenski A, et al. Vasodilator-stimulated phosphoprotein serine 239 phosphorylation as a sensitive monitor of defective nitric oxide/cGMP signaling and endothelial dysfunction. Circ Res. 2000;87:999–1005. doi: 10.1161/01.res.87.11.999.  [DOI](https://doi.org/10.1161/01.res.87.11.999) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11090544/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circ%20Res&title=Vasodilator-stimulated%20phosphoprotein%20serine%20239%20phosphorylation%20as%20a%20sensitive%20monitor%20of%20defective%20nitric%20oxide/cGMP%20signaling%20and%20endothelial%20dysfunction&author=M%20Oelze&author=H%20Mollnau&author=N%20Hoffmann&author=A%20Warnholtz&author=M%20Bodenschatz&volume=87&publication_year=2000&pages=999-1005&pmid=11090544&doi=10.1161/01.res.87.11.999&)

46. Feil R, Lohmann SM, de Jonge H, Walter U, Hofmann F. Cyclic GMP-dependent protein kinases and the cardiovascular system: insights from genetically modified mice. Circ Res. 2003;93:907–916. doi: 10.1161/01.RES.0000100390.68771.CC.  [DOI](https://doi.org/10.1161/01.RES.0000100390.68771.CC) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14615494/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circ%20Res&title=Cyclic%20GMP-dependent%20protein%20kinases%20and%20the%20cardiovascular%20system:%20insights%20from%20genetically%20modified%20mice&author=R%20Feil&author=SM%20Lohmann&author=H%20de%20Jonge&author=U%20Walter&author=F%20Hofmann&volume=93&publication_year=2003&pages=907-916&pmid=14615494&doi=10.1161/01.RES.0000100390.68771.CC&)

47. Warner TD, Mitchell JA, Sheng H, Murad F. Effects of cyclic GMP on smooth muscle relaxation. Adv Pharmacol. 1994;26:171–194. doi: 10.1016/s1054-3589(08)60054-x.  [DOI](https://doi.org/10.1016/s1054-3589(08)60054-x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7913615/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Adv%20Pharmacol&title=Effects%20of%20cyclic%20GMP%20on%20smooth%20muscle%20relaxation&author=TD%20Warner&author=JA%20Mitchell&author=H%20Sheng&author=F%20Murad&volume=26&publication_year=1994&pages=171-194&pmid=7913615&doi=10.1016/s1054-3589(08)60054-x&)

48. Buechler WA, Ivanova K, Wolfram G, Drummer C, Heim JM, Gerzer R. Soluble guanylyl cyclase and platelet function. Ann N Y Acad Sci. 1994;714:151–157. doi: 10.1111/j.1749-6632.1994.tb12039.x.  [DOI](https://doi.org/10.1111/j.1749-6632.1994.tb12039.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7912486/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20N%20Y%20Acad%20Sci&title=Soluble%20guanylyl%20cyclase%20and%20platelet%20function&author=WA%20Buechler&author=K%20Ivanova&author=G%20Wolfram&author=C%20Drummer&author=JM%20Heim&volume=714&publication_year=1994&pages=151-157&pmid=7912486&doi=10.1111/j.1749-6632.1994.tb12039.x&)

49. Schork AJ, Thompson WK, Pham P, Torkamani A, Roddey JC, Sullivan PF, et al. All SNPs are not created equal: genome-wide association studies reveal a consistent pattern of enrichment among functionally annotated SNPs. PLoS Genet. 2013;9:e1003449. doi: 10.1371/journal.pgen.1003449.  [DOI](https://doi.org/10.1371/journal.pgen.1003449) | [PMC free article](/articles/PMC3636284/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23637621/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet&title=All%20SNPs%20are%20not%20created%20equal:%20genome-wide%20association%20studies%20reveal%20a%20consistent%20pattern%20of%20enrichment%20among%20functionally%20annotated%20SNPs&author=AJ%20Schork&author=WK%20Thompson&author=P%20Pham&author=A%20Torkamani&author=JC%20Roddey&volume=9&publication_year=2013&pages=e1003449&pmid=23637621&doi=10.1371/journal.pgen.1003449&)
