# ABC Transporter Polymorphisms are Associated with Irinotecan Pharmacokinetics and Neutropenia

## Metadata
**Authors:** Megan Li, Eric L Seiser, R Michael Baldwin, Jacqueline Ramirez, Mark J Ratain, Federico Innocenti, Deanna L Kroetz
**Journal:** The pharmacogenomics journal
**Date:** 2016 Nov 15
**DOI:** [10.1038/tpj.2016.75](https://doi.org/10.1038/tpj.2016.75)
**PMID:** 27845419
**PMCID:** PMC5432414
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5432414/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC5432414/pdf/nihms837275.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC5432414/pdf/nihms837275.pdf)

## Abstract

Neutropenia is a common dose-limiting toxicity associated with irinotecan treatment. While UGT1A1 variants have been associated with neutropenia, a fraction of neutropenia risk remains unaccounted for. To identify additional genetic markers contributing to variability in irinotecan pharmacokinetics and neutropenia, a regression analysis was performed in 78 irinotecan-treated patients to comprehensively analyze three hepatic efflux transporter genes (ABCB1, ABCC1 and ABCG2). rs6498588 (ABCC1) and rs12720066 (ABCB1) were associated with increased SN-38 exposure, and rs17501331 (ABCC1) and rs12720066 were associated with lower absolute neutrophil count nadir. rs6498588 and a variant in high linkage disequilibrium are located in transcriptionally active regions or are predicted to alter transcription factor binding sites. While enhancer activity was not evident in vitro for genomic regions containing these SNPs, rs6498588 was significantly associated with ABCC1 expression in human liver. These results suggest that genetic variation in ABCC1 and ABCB1 may contribute to irinotecan-induced neutropenia by altering expression of transporters involved in irinotecan metabolite disposition.

## Introduction

Irinotecan is an antineoplastic prodrug approved for first-line therapy in combination with 5-fluorouracil and leucovorin for patients with metastatic carcinoma of the colon or rectum and as a single agent for second-line therapy of fluorouracil refractory metastatic colorectal cancer. Common dose-limiting toxicities are diarrhea and neutropenia^1^, with up to 34% of patients experiencing grade 3–4 neutropenia^2^. Interindividual variation in irinotecan toxicity has prompted the search for genetic biomarkers to guide safer irinotecan treatment.

Irinotecan is activated to form the more potent topoisomerase I inhibiting active metabolite, SN-38^3^. Elevated SN-38 plasma concentrations have been associated with neutropenia during irinotecan treatment^4–7^. SN-38 is subsequently inactivated to SN-38 glucuronide (SN-38G) by members of the uridine diphosphate glucuronosyl transferase family, primarily UGT1A1^8^ ([Figure 1](#F1)).

### Figure 1. Role of ABC transporters and UGT in irinotecan pharmacokinetics and toxicity.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/bdf0/5432414/7d0dc6365d22/nihms837275f1.jpg)

The pathway is taken from PharmGKB. Copyright PharmGKB. Permission has been given by PharmGKB and Stanford University. An original version is available online at https://www.pharmgkb.org/pathway/PA2001.

*UGT1A1**28 has been associated with increased risk of neutropenia during irinotecan treatment^5,9–12^. The TA repeat polymorphism is located in the proximal promoter region of *UGT1A1* and results in reduced expression relative to the *UGT1A1**1 reference allele. Patients homozygous for *UGT1A1**28 glucuronidate SN-38 less efficiently than patients who have one or more reference alleles^10^; subsequently, homozygous patients are especially susceptible to toxicities resulting from increased SN-38 systemic exposure (as measured by area under the plasma concentration-time curve (AUC)). *UGT1A1**93 (rs10929302), a promoter polymorphism in high linkage disequilibrium (LD) with *UGT1A1**28, has also been proposed as a predictor of neutropenia, with the variant being strongly associated with increased hematologic toxicity^13^, increased SN-38 exposure, and lower absolute neutrophil count (ANC) nadir^14–16^.

While *UGT1A1**28-guided irinotecan dosing does reduce dose-limiting toxicities^17^, variability in SN-38 AUC within genotype-normalized doses suggests that *UGT1A1**28 does not account for all the toxicity observed during irinotecan therapy. Other studies have demonstrated more modest effects of *UGT1A1* variants and propose that other metabolizing enzymes and drug transporters may be involved^18–20^.

Irinotecan toxicity and increased SN-38 exposure have been previously associated with several hepatic transporter polymorphisms^14,21–23^. The ATP-binding cassette (ABC) family includes proteins that transport irinotecan and its metabolites out of the cell^24^. *ABCB1* variants have been associated with increased exposure to irinotecan and SN-38^21^, reduced irinotecan and SN-38 clearance^21,25^, and other toxic events^15,26^. *ABCC1* variants were previously associated with ANC nadir, SN-38 AUC, and the ratio of SN-38G AUC to SN-38 AUC^14^, and a common *ABCG2* haplotype was also associated with neutropenia^27^. However, these associations have yet to be replicated and validated. Compared to *UGT1A1*, hepatic transporters are relatively unexplored yet could play an important role in the distribution and accumulation of irinotecan and its metabolites.

In the current study, a candidate gene and functional analysis were performed to examine associations of *ABCB1*, *ABCC1*, and *ABCG2* polymorphisms with irinotecan pharmacokinetics and neutropenia. An earlier pharmacogenetic analysis in this cohort included only a small number of polymorphisms in these three genes^14^. The present study covers the entire length and flanking regions of these genes for a more comprehensive analysis and tests the hypothesis that ABC transporter variants can contribute to additional variability in neutropenia observed during irinotecan treatment.

## Subjects and Methods

### Subjects

Human investigations were performed after review and approval by the Biological Sciences Division/University of Chicago Hospitals Institutional Review Board and in accordance with Federalwide Assurance for the protection of human subjects. Informed consent was obtained from all subjects. The study cohort consisted of 85 advanced cancer patients treated with single-agent irinotecan (300 mg/m^2^ or 350 mg/m^2^) every 3 weeks. Pharmacokinetic and clinical data, including race, sex, age, body surface area (BSA), SN-38 AUC, ANC nadir, and baseline ANC, were collected. This patient population was examined in previous studies^5,14^, and thus the sample size was not based on *a priori* calculation.

### Genotyping

Tag single nucleotide polymorphisms (SNP) were selected from genomic regions covering 25 kb upstream of the transcription start site to 5 kb downstream of the 3′-UTR of each candidate gene (*ABCB1*, *ABCC1*, *ABCG2*). Additional SNPs were chosen to directly interrogate nonsynonymous SNPs in the UCSF Pharmacogenetics of Membrane Transporters database ([http://pharmacogenetics.ucsf.edu/](http://pharmacogenetics.ucsf.edu/)), SNPs with published clinical associations, and SNPs in LD with clinically associated SNPs. Fifty *ABCB1* SNPs, 93 *ABCC1* SNPs, and 38 *ABCG2* tag SNPs were assayed. Genotyping was performed by the Mayo Clinic Genotyping Shared Resource using the Illumina VeraCode (BeadXpress) platform. Following quality control measures to eliminate suboptimal samples and markers and filtering for minor allele frequency ≥ 0.05 using PLINK v1.07^28^, 85 individuals and 123 SNPs were retained for analysis ([Supplementary Table 1](#SD1)).

### Statistical analysis

SNPs were tested for deviation from Hardy-Weinberg equilibrium using an exact test. Genotype-phenotype associations were analyzed using a two-stage regression analysis in R v3.2.2^29^. Dose-adjusted SN-38 AUC and ANC nadir values were log_10_-transformed for the analysis. In the univariate analysis, simple regressions were first performed on each variant and clinical covariate separately, and those with a significance level of P < 0.15 were kept for further analysis. Step-wise backward selection of this subset of genotypes using the rms package^30^ for R led to the final multivariable models, and model fit was assessed by examining R^2^ values. An additive genetic model was first assumed for all genotype associations. To eliminate the driving effect of any sparse homozygous variants, when a particular variant had n ≤3 observations in the homozygous genotype group and was significant, these genotypes were grouped with the heterozygous genotypes; the variant was kept in the model if still significant after combining genotypes.

Other statistical analyses were performed using GraphPad Prism v6.0 (GraphPad Software, La Jolla, CA, USA). Comparisons for the risk alleles analysis and luciferase reporter assays were analyzed using a Mann-Whitney test or, if there were more than two groups, a Kruskal-Wallis test followed by Dunn’s multiple comparisons test. Variance was assumed to be similar between compared groups, and two-sided P-values < 0.05 were considered statistically significant.

### Bioinformatic analysis

Significant SNPs and SNPs in high LD (r^2^ ≥ 0.8) were examined for evidence of changes in function and regulatory activity. HaploReg^31^ and RegulomeDB^32^ were used to determine whether the SNPs overlap with experimentally predicted functional elements (chromatin state segmentation^33^, DNase hypersensitivity peaks, ChIP-seq peaks) from the ENCODE Project^34^ and the Roadmap Epigenomics Project^35^. HaploReg^31^ and TRANSFAC Match^36^ were used to identify and predict changes in transcription factor (TF) binding.

### Measurement of enhancer activity in vitro

Primers were designed with the aid of Primer3^37,38^ to target 500 bp genomic regions flanking each significant SNP ([Supplementary Table 2](#SD1)). These regions were PCR-amplified using PfuTurbo (Agilent, Santa Clara, CA, USA) from human genomic DNA and cloned into the pGL4.23[*luc2*/minP] luciferase reporter vector (Promega, Madison, WI, USA). The QuikChange Lightning Site-Directed Mutagenesis Kit (Agilent) was used to create the desired point mutation. Correct sequences were verified by Sanger sequencing. Reporter plasmids containing reference or variant sequences were transfected into Hep G2 (ATCC^®^ HB-8065™, human hepatocellular carcinoma, passage 2–6) cells using Lipofectamine LTX and PLUS reagents (Invitrogen, Carlsbad, CA, USA). The *Renilla* luciferase construct pGL4.73 (Promega) was co-transfected to correct for transfection efficiency. Empty pGL4.23 vector and a known apolipoprotein E liver enhancer^39^ were also transfected as negative and positive controls, respectively. Cells were cultured in Eagle’s Minimum Essential Medium with EBSS, non-essential amino acids, 2 mM L-glutamine, 1 mM sodium pyruvate, 1500 mg/L sodium bicarbonate, and 10% fetal bovine serum. Twenty-four hours after transfection, luciferase activity was measured using the Dual-Luciferase Reporter Assay System (Promega) and quantified using the GloMax-96 Microplate Luminometer (Promega). Background-subtracted firefly luciferase activity was normalized with respect to *Renilla* luciferase activity and expressed as a mean value relative to empty pGL4.23 vector.

### Code availability

The R code for the univariate and multivariate regression analyses is available at [https://github.com/kroetzlab/irinotecan-ABC](https://github.com/kroetzlab/irinotecan-ABC).

## Results

### Cohort analysis

The original study cohort consisted of 85 patients, with the majority being Caucasian or African American. Seven subjects (2 Asian, 1 Filipino, 4 Hispanic) were omitted from the initial regression analyses to avoid dilution of association signals by genetic heterogeneity. Characteristics of patients included in the primary analysis are listed in [Supplementary Table 3](#SD1). The analyzed cohort had a median age of 57 and slightly more males (58%). The majority (76%) received a 350 mg/m^2^ dose of irinotecan, while 24% received a 300 mg/m^2^ dose. The outcomes examined in this study were SN-38 AUC (adjusted for irinotecan dose) and ANC nadir. Distributions of these phenotypes in the analyzed cohort are shown in [Supplementary Figure 1](#SD1). None of the clinical covariates (race, sex, age, BSA, baseline ANC) were significantly associated with SN-38 AUC or ANC nadir.

### Association of ABC SNPs with SN-38 exposure

Eight SNPs met the P < 0.15 threshold in univariate regression analyses for SN-38 exposure. Of these, rs12720066, rs6498588, rs35621, and rs2373586 had P-values < 0.05 ([Table 1](#T1)). The genotype frequencies for these SNPs in Caucasians and African Americans are shown in [Supplementary Table 4](#SD1). All eight ABC SNPs were taken into the multivariate analysis along with the *UGT1A1**93 genotype (rs10929302). The final log SN-38 AUC model included *UGT1A1**93, an intronic *ABCB1* variant (rs12720066), and an *ABCC1* variant (rs6498588) in the 5′-flanking region ([Table 1](#T1)). The model explained 34% (R^2^ = 0.345) of the variation in SN-38 exposure, with all three SNPs contributing similar variances to the model (rs12720066: β = −0.337; rs6498588: β = 0.325; rs10929302: β = 0.390). The *UGT1A1* and *ABCC1* SNPs were associated with increased exposure of patients to SN-38, while the *ABCB1* SNP was associated with decreased exposure ([Figure 2](#F2)).

### Table 1.

|   |   |   |   | Univariate |  |  | Multivariate3 |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| rsid | Gene | Function | Ref>Alt | Estimate | SE | P | Estimate | SE | P |
| rs127200662 | ABCB1 | intronic | A>C | −0.204 | 0.070 | 0.005 | −0.217 | 0.061 | 6.24 × 10−4 |
| rs6498588 | ABCC1 | 5′-gene | A>T | 0.111 | 0.042 | 0.010 | 0.123 | 0.036 | 9.50 × 10−4 |
| rs356212 | ABCC1 | intronic | C>T | 0.191 | 0.083 | 0.025 |   |   |   |
| rs2373586 | ABCB1 | intronic | A>C | 0.086 | 0.040 | 0.033 |   |   |   |
| rs8187843 | ABCC1 | intronic | G>A | −0.160 | 0.084 | 0.061 |   |   |   |
| rs37435272 | ABCC1 | 3′-UTR | C>T | 0.113 | 0.062 | 0.071 |   |   |   |
| rs2150952 | ABCC1 | intronic | G>A | 0.118 | 0.073 | 0.109 |   |   |   |
| rs212082 | ABCC1 | intronic | A>G | −0.080 | 0.054 | 0.142 |   |   |   |
| rs10929302 | UGT1A1 | promoter | G>A | 0.145 | 0.042 | 0.001 | 0.154 | 0.037 | 9.00 × 10−5 |

Table 1 Caption: Univariate and multivariate analysis results for log SN-38 AUC1

### Figure 2. Association of ABC and UGT variants with SN-38 AUC.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/bdf0/5432414/e2820eca660c/nihms837275f2.jpg)

The relationship between genotype and SN-38 AUC (adjusted for irinotecan dose) is shown for all analyzed patients. Horizontal bars represent the median values for each genotype group.

Inclusion of only Caucasian genotypes in the association analyses yielded similar results as obtained with the combined population ([Supplementary Table 5](#SD1)). The multivariable model for log SN-38 AUC also contained *UGT1A1**93, rs12720066, and rs6498588, with similar effect sizes as those for the model including African Americans. Inclusion of the seven Asian, Filipino, and Hispanic subjects into the final model ([Supplementary Table 6](#SD1)) resulted in similar coefficients and P-values.

### Association of ABC SNPs with neutropenia

In univariate linear regression analyses of ABC SNPs with ANC nadir, ten SNPs met the P < 0.15 threshold and three (rs17501331, rs12720066, and rs3743527) had P-values < 0.05 ([Table 2](#T2)). Genotype frequencies of these SNPs are listed in [Supplementary Table 4](#SD1). The final model following multivariable regression and inclusion of *UGT1A1**93 included an intronic SNP in *ABCB1* (rs12720066) and an intronic SNP in *ABCC1* (rs17501331) ([Table 2](#T2)). This model accounted for 39% (R^2^ = 0.390) of the variation in ANC nadir, with *UGT1A1**93 explaining more variance in the outcome (β = −0.479) than the other two SNPs (rs17501331: β = −0.295; rs12720066: β = 0.286). The *ABCB1* SNP was associated with increased ANC nadir and the *UGT1A1* and *ABCC1* SNPs were associated with reduced nadirs ([Figure 3](#F3)). As for the SN-38 AUC analysis, the final multivariable model for log ANC nadir for Caucasians only contained the same SNPs with similar effect sizes as those for the model including African Americans ([Supplementary Table 7](#SD1)). Inclusion of the seven Asian, Filipino, and Hispanic subjects into the final model ([Supplementary Table 8](#SD1)) also resulted in similar coefficients and P-values.

### Table 2.

|   |   |   |   | Univariate |  |  | Multivariate3 |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| rsid | Gene | Function | Ref>Alt | Estimate | SE | P | Estimate | SE | P |
| rs17501331 | ABCC1 | intronic | A>G | −0.255 | 0.106 | 0.019 | −0.281 | 0.088 | 0.002 |
| rs127200662 | ABCB1 | intronic | A>C | 0.227 | 0.102 | 0.030 | 0.261 | 0.084 | 0.003 |
| rs37435272 | ABCC1 | 3′-UTR | C>T | −0.194 | 0.089 | 0.033 |   |   |   |
| rs212090 | ABCC1 | 3′-UTR | T>A | 0.102 | 0.059 | 0.089 |   |   |   |
| rs1128503 | ABCB1 | synonymous | A>G | −0.097 | 0.056 | 0.090 |   |   |   |
| rs8050881 | ABCC1 | 5′-gene | A>G | −0.098 | 0.061 | 0.110 |   |   |   |
| rs2725264 | ABCG2 | intronic | C>T | −0.127 | 0.080 | 0.118 |   |   |   |
| rs35626 | ABCC1 | intronic | G>T | −0.107 | 0.069 | 0.123 |   |   |   |
| rs1967120 | ABCC1 | intronic | G>A | −0.108 | 0.070 | 0.124 |   |   |   |
| rs152023 | ABCC1 | intronic | C>T | −0.093 | 0.063 | 0.144 |   |   |   |
| rs10929302 | UGT1A1 | promoter | G>A | −0.262 | 0.057 | 1.98 × 10−5 | −0.268 | 0.052 | 1.99 × 10−6 |

Table 2 Caption: Univariate and multivariate analysis results for log ANC nadir1

### Figure 3. Association of ABC and UGT variants with ANC nadir.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/bdf0/5432414/620e772f109f/nihms837275f3.jpg)

The relationship between genotype and absolute neutrophil count nadir is shown for all analyzed patients. Horizontal bars represent the median values for each genotype group.

### Contribution of multiple risk alleles

To examine the combined effects of risk alleles, composite genotypes were constructed for each outcome. Patients carrying more risk alleles for rs12720066, rs6498588, and *UGT1A1**93 had significantly higher SN-38 AUC (P < 0.0001), while patients carrying more risk alleles for rs12720066, rs17501331, and *UGT1A1**93 had significantly lower ANC values (P = 0.0006) ([Supplementary Figure 2](#SD1)). Almost all (91%) patients with grade 3–4 neutropenia (< 1000 cells/μl) carried at least four risk alleles among all three ABC SNPs and *UGT1A1**93, compared to 53% of patients with no or less severe neutropenia (P = 0.001). Of the patients with grade 3–4 neutropenia, 91% are homozygous for rs12720066 and 46% are homozygous for *UGT1A1**93, while of patients with no or less severe neutropenia, 73% are homozygous for rs12720066 and 7% are homozygous for *UGT1A1**93. Dose-adjusted SN-38 AUC was also confirmed to be higher in patients with grade 3–4 neutropenia ([Supplementary Figure 3](#SD1); P = 0.0001).

### Bioinformatic analysis

To investigate the putative function of the SNPs in the final models, overlapping regulatory elements and protein binding sites were examined. The intronic *ABCC1* SNP rs17501331 is in a predicted region of transcriptional transition in Hep G2 cells and is predicted to alter two regulatory motifs, DMRT1 and SOX6. rs6498588, located 11 kb upstream of *ABCC1*, is predicted to alter three regulatory motifs (GR, Mef2, YY1). rs4148330, a SNP in high LD (r^2^ = 0.8) with rs6498588, resides 1.7 kb upstream of *ABCC1* in a region with H3K4 methylation in Hep G2 cells and DNase hypersensitivity in Hep G2 cells and hepatocytes, and is predicted to alter E2F and NRSF TF binding. rs12720066 and other SNPs in high LD with rs6498588 were also predicted to alter regulatory motifs but were not associated with an enrichment in enhancer or DNase activity in liver or liver-derived cells.

### Enhancer assays

Genomic regions containing the *ABCB1* and *ABCC1* SNPs in the final models for SN-38 AUC and ANC nadir plus additional SNPs in high LD with these variants ([Supplementary Table 2](#SD1)) were tested *in vitro* for enhancer activity and the effect of genetic variation on this activity. None of the tested genomic regions exhibited significant enhancer activity and the introduction of the indicated variants did not change the transcriptional activity ([Figure 4](#F4)).

### Figure 4. Analysis of genomic regions and genetic variants for enhancer activity.

![Figure 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/bdf0/5432414/a82d7d3d4922/nihms837275f4.jpg)

Enhancer activity was measured in Hep G2 cells using a luciferase reporter gene assay. The values plotted represent mean ± SD of the relative luciferase activity in cells transfected with the indicated genomic region without (Ref) and with (SNP) the variant compared to empty pGL4.23 construct (−). A known liver enhancer of the APOE gene was used as the positive (+) control. Data are representative of at least three independent experiments of six biological replicates each.

## Discussion

This study provides a comprehensive analysis of the genetic variants spanning three hepatic ABC transporter genes of the irinotecan pharmacokinetic pathway. Three SNPs (rs12720066, rs6498588, rs17501331) were significantly associated with changes in SN-38 AUC or ANC nadir. These variants could potentially predict neutropenia during irinotecan treatment and provide additional insight into the molecular mechanism of the toxicity.

*UGT1A1**93 (rs10929302) was confirmed as a strong predictor of irinotecan-induced neutropenia, reflected by associations with higher SN-38 AUC and lower ANC nadirs. The effect sizes of *UGT1A1**93 for associations with SN-38 AUC or ANC nadir were similar or slightly more significant than those of *UGT1A1**28. *UGT1A1**93 was previously associated with increased hematologic toxicity, increased SN-38 exposure, and lower ANC nadir^13–15^. A recent study confirmed *UGT1A1**93 as a more robust marker for neutropenia than *UGT1A1**28^16^.

The three ABC transporter SNPs in the final multivariable models are in noncoding regions, either intronic or upstream of their respective genes, and are not in high linkage disequilibrium with any coding SNPs. This was expected, considering the low number of high-frequency coding SNPs in these genes. Previous studies of the effects of nonsynonymous SNPs on the expression and activity of these ABC transporters have yielded mostly negative or inconclusive results^40–42^. Of the SNPs with P < 0.15 in the univariate analyses, only one (rs1128503) is in a coding region and one intronic SNP (rs2373586) is in high LD with a common missense SNP, rs2032582. rs1128503 and rs2032582 are both in *ABCB1* and, together with rs1045642, make up the *ABCB1**2 haplotype. This haplotype has been previously associated with early toxicity during irinotecan treatment^26^. However, these SNPs dropped out of significance during the selection of the multivariable models. These coding SNPs have been frequently studied together and individually for association with transporter activity and changes in exposure and response to numerous drugs, but with discordant results^40^.

In contrast to coding SNPs, due to lower selective pressures and more modest effects, variants in the regulatory regions of membrane transporter genes are more abundant and exist at higher allele frequencies^43,44^. Functional studies of variants in the proximal promoter^44–47^, untranslated^48,49^ and intronic^50,51^ regions of ABC and SLCO transporter genes have demonstrated changes in mRNA or protein expression levels and interindividual variation in pharmacokinetics^52,53^. In light of this, it is likely that noncoding genetic variation has a more widespread impact on transporter function. Our results highlight the importance of including noncoding SNPs in pharmacogenetic studies and the need to further characterize regulatory regions in pharmacogenes.

The significant variants identified in this study may contribute to irinotecan-induced neutropenia by altering expression of ABCB1 or ABCC1. *ABCB1* encodes P-glycoprotein, which is involved in irinotecan and SN-38 biliary excretion^54,55^. An increase in ABCB1 expression could result in increased biliary secretion of SN-38 and a corresponding reduction in plasma SN-38 levels, which are inversely correlated with ANC nadir. *ABCC1* encodes MRP1, a hepatic basolateral efflux transporter of SN-38^56^. Increased ABCC1 expression is hypothesized to result in increased systemic levels of SN-38, which is consistent with the decreased ANC nadir associated with this variant. It should be noted that ABCC1/MRP1 expression in the liver is relatively low^57^, so increased expression may have a significant impact on basolateral efflux of SN-38 back into the blood.

Bioinformatic analyses predicted 5′-flanking *ABCC1* SNP rs6498588 to alter TF binding and rs4148330 (in LD with rs6498588) to have both enhancer activity in hepatic cells and effects on TF binding. A preliminary eQTL meta-analysis of human liver samples found rs6498588 to be significantly associated (meta-adjusted P = 0.05) with increased ABCC1 expression^58^. These data are consistent with the association of rs6498588 with increased SN-38 AUC, as higher ABCC1 expression would result in increased transport of SN-38 into the blood. However, our *in vitro* enhancer assays did not confirm these predictions for either rs6498588 or rs4148330. Enhancer assays conducted in additional hepatic cell lines or with different-sized genomic regions are warranted.

A combined allele analysis showed that a higher number of risk alleles from our models is correlated with irinotecan-induced toxicity. A significantly greater proportion of patients with grade 3–4 neutropenia carried at least four risk alleles for both phenotypes when compared to the group with no or less severe neutropenia. rs12720066 and *UGT1A1**93 appear to be driving this effect, with the directions of their effects consistent with the toxicity in both pharmacokinetic and pharmacodynamic outcomes. While rs6498588 was only in the final SN-38 AUC model, a SNP in high LD (rs8050881) had a weaker univariate association with ANC nadir; the variant alleles of these SNPs are also associated with the toxicity in both outcomes. However, the contribution of rs17501331 is less clear; this variant was only associated with ANC nadir and not at all with SN-38 AUC, suggesting a mechanism that may affect neutrophil count independent of SN-38 exposure. Given the complexity of the many metabolizing enzymes and transporters acting on SN-38, further eQTL and functional characterization studies are needed to validate the effect of these SNPs before proposing a combined PK/PD model.

The addition of rs6498588 to our previously published model^14^ of log SN-38 AUC containing rs35605 (*ABCC1*), rs10276036 (*ABCB1*) and *UGT1A1**93 improved the percent variation of SN-38 AUC explained by genetic determinants (adjusted R^2^ increased from 0.221 to 0.284, P = 0.009, ANOVA), with all four variants in the model remaining significant. When rs12720066 and rs17501331 were added to our previous model of log ANC nadir^14^ containing rs3765129 (*ABCC1*), rs2306283 (*SLCO1B1*), and *UGT1A1**93, only rs17501331 had marginal significance (P = 0.05) and a smaller effect than the other variants in the initial model. The previously identified SNPs, especially *UGT1A1**93, may explain more of the genetic variation in ANC nadir. Again, a better mechanistic understanding of this toxicity is needed before proposing a predictive multivariable model for irinotecan-induced neutropenia.

There are no known clinical associations of the variants identified in the current association studies. However, there is still enough compelling functional evidence to suggest that genetic variants in *ABCB1* and *ABCC1* are relevant to irinotecan toxicity. MRP1-overexpressing KB-3–1 cells showed increased SN-38 resistance compared to normal cells^56^, affirming that expression of the transporter does affect SN-38 transport. In *ABCB1* knockout mice, irinotecan and SN-38 plasma levels were increased relative to the wildtype mice, consistent with a role for P-gp in biliary secretion of this drug/metabolite pair^55^. Cell lines overexpressing BCRP (encoded by *ABCG2*) are also resistant to irinotecan^59^ and SN-38^60^, and *ABCG2* 421C>A reduced gene expression and conferred irinotecan resistance in cancer cell lines^61^. In the current study, there was no association with any of the tested *ABCG2* variants and SN-38 exposure or ANC nadir.

Other genes in the irinotecan pharmacokinetic pathway including *ABCC2*, *SLCO1B1*, *CES1*, *CES2*, *CYP3A4* and additional variants in *UGT1A1* and other UGT genes may also contribute to SN-38 exposure and neutropenia and warrant further study^62–64^. In contrast, genetic variation in genes responsible for irinotecan pharmacodynamics do not seem to play a role^65^. More comprehensive genotyping of pharmacokinetic genes has the potential to identify additional genetic variants more strongly associated with irinotecan exposure and toxicity. Discovery of additional genetic and non-genetic determinants of irinotecan-induced neutropenia will improve the predictive power of a clinical pharmacogenetic algorithm over using a single *UGT1A1**28 test for drug selection and dosing. Replication and prospective studies in large independent cohorts as well as in other ethnic groups are necessary to validate and expand these findings.

A current gap in the field of pharmacogenetics is understanding the clinical value of variants. Our study used a pharmacokinetic/pharmacodynamic phenotyping approach to discover new variants associated with neutropenia during irinotecan treatment. Further investigation of these variants and others in ABC transporter genes may not only potentially contribute to the improvement of individualized treatment with irinotecan, but may also have broader implications for other substrates of ABC transporters.

## Supplementary Material

## Acknowledgments

This work was supported by NIH/NIGMS U01GM061390 (Pharmacogenetics of Membrane Transporters), NIH/NIGMS U01GM0061393 (PAAR-Pharmacogenomics of Anticancer Agents Research Group), NIH/NIGMS T32 GM007175, NIH/NIDDK R21DK081157, and NIH/NCI K07CA140390-01.

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. de Forni M, Bugat R, Chabot GG, Culine S, Extra JM, Gouyette A, et al. Phase I and pharmacokinetic study of the camptothecin derivative irinotecan, administered on a weekly schedule in cancer patients. Cancer Res. 1994;54:4347–4354.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/8044782/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=Phase%20I%20and%20pharmacokinetic%20study%20of%20the%20camptothecin%20derivative%20irinotecan,%20administered%20on%20a%20weekly%20schedule%20in%20cancer%20patients&author=M%20de%20Forni&author=R%20Bugat&author=GG%20Chabot&author=S%20Culine&author=JM%20Extra&volume=54&publication_year=1994&pages=4347-4354&pmid=8044782&)

2. Fuchs CS, Moore MR, Harker G, Villa L, Rinaldi D, Hecht JR. Phase III comparison of two irinotecan dosing regimens in second-line therapy of metastatic colorectal cancer. J Clin Oncol. 2003;21:807–814. doi: 10.1200/JCO.2003.08.058.  [DOI](https://doi.org/10.1200/JCO.2003.08.058) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12610178/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Phase%20III%20comparison%20of%20two%20irinotecan%20dosing%20regimens%20in%20second-line%20therapy%20of%20metastatic%20colorectal%20cancer&author=CS%20Fuchs&author=MR%20Moore&author=G%20Harker&author=L%20Villa&author=D%20Rinaldi&volume=21&publication_year=2003&pages=807-814&pmid=12610178&doi=10.1200/JCO.2003.08.058&)

3. Slatter JG, Su P, Sams JP, Schaaf LJ, Wienkers LC. Bioactivation of the anticancer agent CPT-11 to SN-38 by human hepatic microsomal carboxylesterases and the in vitro assessment of potential drug interactions. Drug Metab Dispos. 1997;25:1157–1164.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/9321519/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Bioactivation%20of%20the%20anticancer%20agent%20CPT-11%20to%20SN-38%20by%20human%20hepatic%20microsomal%20carboxylesterases%20and%20the%20in%20vitro%20assessment%20of%20potential%20drug%20interactions&author=JG%20Slatter&author=P%20Su&author=JP%20Sams&author=LJ%20Schaaf&author=LC%20Wienkers&volume=25&publication_year=1997&pages=1157-1164&pmid=9321519&)

4. Ramchandani RP, Wang Y, Booth BP, Ibrahim A, Johnson JR, Rahman A, et al. The role of SN-38 exposure, UGT1A1*28 polymorphism, and baseline bilirubin level in predicting severe irinotecan toxicity. J Clin Pharmacol. 2007;47:78–86. doi: 10.1177/0091270006295060.  [DOI](https://doi.org/10.1177/0091270006295060) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17192505/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Pharmacol&title=The%20role%20of%20SN-38%20exposure,%20UGT1A1*28%20polymorphism,%20and%20baseline%20bilirubin%20level%20in%20predicting%20severe%20irinotecan%20toxicity&author=RP%20Ramchandani&author=Y%20Wang&author=BP%20Booth&author=A%20Ibrahim&author=JR%20Johnson&volume=47&publication_year=2007&pages=78-86&pmid=17192505&doi=10.1177/0091270006295060&)

5. Innocenti F, Undevia SD, Iyer L, Chen PX, Das S, Kocherginsky M, et al. Genetic variants in the UDP-glucuronosyltransferase 1A1 gene predict the risk of severe neutropenia of irinotecan. J Clin Oncol. 2004;22:1382–1388. doi: 10.1200/JCO.2004.07.173.  [DOI](https://doi.org/10.1200/JCO.2004.07.173) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15007088/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Genetic%20variants%20in%20the%20UDP-glucuronosyltransferase%201A1%20gene%20predict%20the%20risk%20of%20severe%20neutropenia%20of%20irinotecan&author=F%20Innocenti&author=SD%20Undevia&author=L%20Iyer&author=PX%20Chen&author=S%20Das&volume=22&publication_year=2004&pages=1382-1388&pmid=15007088&doi=10.1200/JCO.2004.07.173&)

6. Minami H, Sai K, Saeki M, Saito Y, Ozawa S, Suzuki K, et al. Irinotecan pharmacokinetics/pharmacodynamics and UGT1A genetic polymorphisms in Japanese: roles of UGT1A1*6 and *28. Pharmacogenet Genomics. 2007;17:497–504. doi: 10.1097/FPC.0b013e328014341f.  [DOI](https://doi.org/10.1097/FPC.0b013e328014341f) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17558305/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Irinotecan%20pharmacokinetics/pharmacodynamics%20and%20UGT1A%20genetic%20polymorphisms%20in%20Japanese:%20roles%20of%20UGT1A1*6%20and%20*28&author=H%20Minami&author=K%20Sai&author=M%20Saeki&author=Y%20Saito&author=S%20Ozawa&volume=17&publication_year=2007&pages=497-504&pmid=17558305&doi=10.1097/FPC.0b013e328014341f&)

7. Han JY, Lim HS, Shin ES, Yoo YK, Park YH, Lee JE, et al. Comprehensive analysis of UGT1A polymorphisms predictive for pharmacokinetics and treatment outcome in patients with non-small-cell lung cancer treated with irinotecan and cisplatin. J Clin Oncol. 2006;24:2237–2244. doi: 10.1200/JCO.2005.03.0239.  [DOI](https://doi.org/10.1200/JCO.2005.03.0239) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16636344/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Comprehensive%20analysis%20of%20UGT1A%20polymorphisms%20predictive%20for%20pharmacokinetics%20and%20treatment%20outcome%20in%20patients%20with%20non-small-cell%20lung%20cancer%20treated%20with%20irinotecan%20and%20cisplatin&author=JY%20Han&author=HS%20Lim&author=ES%20Shin&author=YK%20Yoo&author=YH%20Park&volume=24&publication_year=2006&pages=2237-2244&pmid=16636344&doi=10.1200/JCO.2005.03.0239&)

8. Iyer L, King CD, Whitington PF, Green MD, Roy SK, Tephly TR, et al. Genetic predisposition to the metabolism of irinotecan (CPT-11). Role of uridine diphosphate glucuronosyltransferase isoform 1A1 in the glucuronidation of its active metabolite (SN-38) in human liver microsomes. J Clin Invest. 1998;101:847–854. doi: 10.1172/JCI915.  [DOI](https://doi.org/10.1172/JCI915) | [PMC free article](/articles/PMC508633/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9466980/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Invest&title=Genetic%20predisposition%20to%20the%20metabolism%20of%20irinotecan%20(CPT-11).%20Role%20of%20uridine%20diphosphate%20glucuronosyltransferase%20isoform%201A1%20in%20the%20glucuronidation%20of%20its%20active%20metabolite%20(SN-38)%20in%20human%20liver%20microsomes&author=L%20Iyer&author=CD%20King&author=PF%20Whitington&author=MD%20Green&author=SK%20Roy&volume=101&publication_year=1998&pages=847-854&pmid=9466980&doi=10.1172/JCI915&)

9. Ando Y, Saka H, Ando M, Sawa T, Muro K, Ueoka H, et al. Polymorphisms of UDP-glucuronosyltransferase gene and irinotecan toxicity: a pharmacogenetic analysis. Cancer Res. 2000;60:6921–6926.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/11156391/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=Polymorphisms%20of%20UDP-glucuronosyltransferase%20gene%20and%20irinotecan%20toxicity:%20a%20pharmacogenetic%20analysis&author=Y%20Ando&author=H%20Saka&author=M%20Ando&author=T%20Sawa&author=K%20Muro&volume=60&publication_year=2000&pages=6921-6926&pmid=11156391&)

10. Iyer L, Das S, Janisch L, Wen M, Ramirez J, Karrison T, et al. UGT1A1*28 polymorphism as a determinant of irinotecan disposition and toxicity. Pharmacogenomics J. 2002;2:43–47. doi: 10.1038/sj.tpj.6500072.  [DOI](https://doi.org/10.1038/sj.tpj.6500072) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11990381/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=UGT1A1*28%20polymorphism%20as%20a%20determinant%20of%20irinotecan%20disposition%20and%20toxicity&author=L%20Iyer&author=S%20Das&author=L%20Janisch&author=M%20Wen&author=J%20Ramirez&volume=2&publication_year=2002&pages=43-47&pmid=11990381&doi=10.1038/sj.tpj.6500072&)

11. Rouits E, Boisdron-Celle M, Dumont A, Guerin O, Morel A, Gamelin E. Relevance of different UGT1A1 polymorphisms in irinotecan-induced toxicity: a molecular and clinical study of 75 patients. Clin Cancer Res. 2004;10:5151–5159. doi: 10.1158/1078-0432.CCR-03-0548.  [DOI](https://doi.org/10.1158/1078-0432.CCR-03-0548) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15297419/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&title=Relevance%20of%20different%20UGT1A1%20polymorphisms%20in%20irinotecan-induced%20toxicity:%20a%20molecular%20and%20clinical%20study%20of%2075%20patients&author=E%20Rouits&author=M%20Boisdron-Celle&author=A%20Dumont&author=O%20Guerin&author=A%20Morel&volume=10&publication_year=2004&pages=5151-5159&pmid=15297419&doi=10.1158/1078-0432.CCR-03-0548&)

12. McLeod HL, Parodi L, Sargent DJ, Marsh S, Green E, Abreu P, et al. UGT1A1*28, toxicity and outcome in advanced colorectal cancer: results from trial N9741. J Clin Oncol. 2006;24(Suppl 18) Abstract 3520.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=UGT1A1*28,%20toxicity%20and%20outcome%20in%20advanced%20colorectal%20cancer:%20results%20from%20trial%20N9741&author=HL%20McLeod&author=L%20Parodi&author=DJ%20Sargent&author=S%20Marsh&author=E%20Green&volume=24&issue=Suppl%2018&publication_year=2006&)

13. Côté JF, Kirzin S, Kramar A, Mosnier JF, Diebold MD, Soubeyran I, et al. UGT1A1 polymorphism can predict hematologic toxicity in patients treated with irinotecan. Clin Cancer Res. 2007;13:3269–3275. doi: 10.1158/1078-0432.CCR-06-2290.  [DOI](https://doi.org/10.1158/1078-0432.CCR-06-2290) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17510208/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&title=UGT1A1%20polymorphism%20can%20predict%20hematologic%20toxicity%20in%20patients%20treated%20with%20irinotecan&author=JF%20C%C3%B4t%C3%A9&author=S%20Kirzin&author=A%20Kramar&author=JF%20Mosnier&author=MD%20Diebold&volume=13&publication_year=2007&pages=3269-3275&pmid=17510208&doi=10.1158/1078-0432.CCR-06-2290&)

14. Innocenti F, Kroetz DL, Schuetz E, Dolan ME, Ramírez J, Relling M, et al. Comprehensive pharmacogenetic analysis of irinotecan neutropenia and pharmacokinetics. J Clin Oncol. 2009;27:2604–2614. doi: 10.1200/JCO.2008.20.6300.  [DOI](https://doi.org/10.1200/JCO.2008.20.6300) | [PMC free article](/articles/PMC2690389/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19349540/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Comprehensive%20pharmacogenetic%20analysis%20of%20irinotecan%20neutropenia%20and%20pharmacokinetics&author=F%20Innocenti&author=DL%20Kroetz&author=E%20Schuetz&author=ME%20Dolan&author=J%20Ram%C3%ADrez&volume=27&publication_year=2009&pages=2604-2614&pmid=19349540&doi=10.1200/JCO.2008.20.6300&)

15. Lara PN, Jr, Natale R, Crowley J, Lenz HJ, Redman MW, Carleton JE, et al. Phase III trial of irinotecan/cisplatin compared with etoposide/cisplatin in extensive-stage small-cell lung cancer: clinical and pharmacogenomic results from SWOG S0124. J Clin Oncol. 2009;27:2530–2535. doi: 10.1200/JCO.2008.20.1061.  [DOI](https://doi.org/10.1200/JCO.2008.20.1061) | [PMC free article](/articles/PMC2684855/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19349543/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Phase%20III%20trial%20of%20irinotecan/cisplatin%20compared%20with%20etoposide/cisplatin%20in%20extensive-stage%20small-cell%20lung%20cancer:%20clinical%20and%20pharmacogenomic%20results%20from%20SWOG%20S0124&author=PN%20Lara&author=R%20Natale&author=J%20Crowley&author=HJ%20Lenz&author=MW%20Redman&volume=27&publication_year=2009&pages=2530-2535&pmid=19349543&doi=10.1200/JCO.2008.20.1061&)

16. Crona DJ, Ramirez J, Qiao W, de Graan AJ, Ratain MJ, van Schaik RH, et al. Clinical validity of new genetic biomarkers of irinotecan neutropenia: an independent replication study. Pharmacogenomics J. 2015 doi: 10.1038/tpj.2015.23. e-pub ahead of print 14 April 2015.  [DOI](https://doi.org/10.1038/tpj.2015.23) | [PMC free article](/articles/PMC4729652/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25869015/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=Clinical%20validity%20of%20new%20genetic%20biomarkers%20of%20irinotecan%20neutropenia:%20an%20independent%20replication%20study&author=DJ%20Crona&author=J%20Ramirez&author=W%20Qiao&author=AJ%20de%20Graan&author=MJ%20Ratain&publication_year=2015&pmid=25869015&doi=10.1038/tpj.2015.23&)

17. Innocenti F, Schilsky RL, Ramírez J, Janisch L, Undevia S, House LK, et al. Dose-finding and pharmacokinetic study to optimize the dosing of irinotecan according to the UGT1A1 genotype of patients with cancer. J Clin Oncol. 2014;32:2328–2334. doi: 10.1200/JCO.2014.55.2307.  [DOI](https://doi.org/10.1200/JCO.2014.55.2307) | [PMC free article](/articles/PMC4105486/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24958824/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Dose-finding%20and%20pharmacokinetic%20study%20to%20optimize%20the%20dosing%20of%20irinotecan%20according%20to%20the%20UGT1A1%20genotype%20of%20patients%20with%20cancer&author=F%20Innocenti&author=RL%20Schilsky&author=J%20Ram%C3%ADrez&author=L%20Janisch&author=S%20Undevia&volume=32&publication_year=2014&pages=2328-2334&pmid=24958824&doi=10.1200/JCO.2014.55.2307&)

18. Marcuello E, Altés A, Menoyo A, Del Rio E, Gómez-Pardo M, Baiget M. UGT1A1 gene variations and irinotecan treatment in patients with metastatic colorectal cancer. Br J Cancer. 2004;91:678–682. doi: 10.1038/sj.bjc.6602042.  [DOI](https://doi.org/10.1038/sj.bjc.6602042) | [PMC free article](/articles/PMC2364770/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15280927/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Cancer&title=UGT1A1%20gene%20variations%20and%20irinotecan%20treatment%20in%20patients%20with%20metastatic%20colorectal%20cancer&author=E%20Marcuello&author=A%20Alt%C3%A9s&author=A%20Menoyo&author=E%20Del%20Rio&author=M%20G%C3%B3mez-Pardo&volume=91&publication_year=2004&pages=678-682&pmid=15280927&doi=10.1038/sj.bjc.6602042&)

19. Braun MS, Richman SD, Thompson L, Daly CL, Meade AM, Adlard JW, et al. Association of molecular markers with toxicity outcomes in a randomized trial of chemotherapy for advanced colorectal cancer: the FOCUS trial. J Clin Oncol. 2009;27:5519–5528. doi: 10.1200/JCO.2008.21.6283.  [DOI](https://doi.org/10.1200/JCO.2008.21.6283) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19858398/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Association%20of%20molecular%20markers%20with%20toxicity%20outcomes%20in%20a%20randomized%20trial%20of%20chemotherapy%20for%20advanced%20colorectal%20cancer:%20the%20FOCUS%20trial&author=MS%20Braun&author=SD%20Richman&author=L%20Thompson&author=CL%20Daly&author=AM%20Meade&volume=27&publication_year=2009&pages=5519-5528&pmid=19858398&doi=10.1200/JCO.2008.21.6283&)

20. Schulz C, Heinemann V, Schalhorn A, Moosmann N, Zwingers T, Boeck S, et al. UGT1A1 gene polymorphism: impact on toxicity and efficacy of irinotecan-based regimens in metastatic colorectal cancer. World J Gastroenterol. 2009;15:5058–5066. doi: 10.3748/wjg.15.5058.  [DOI](https://doi.org/10.3748/wjg.15.5058) | [PMC free article](/articles/PMC2768885/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19859999/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=World%20J%20Gastroenterol&title=UGT1A1%20gene%20polymorphism:%20impact%20on%20toxicity%20and%20efficacy%20of%20irinotecan-based%20regimens%20in%20metastatic%20colorectal%20cancer&author=C%20Schulz&author=V%20Heinemann&author=A%20Schalhorn&author=N%20Moosmann&author=T%20Zwingers&volume=15&publication_year=2009&pages=5058-5066&pmid=19859999&doi=10.3748/wjg.15.5058&)

21. Mathijssen RH, Marsh S, Karlsson MO, Xie R, Baker SD, Verweij J, et al. Irinotecan pathway genotype analysis to predict pharmacokinetics. Clin Cancer Res. 2003;9:3246–3253.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/12960109/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&title=Irinotecan%20pathway%20genotype%20analysis%20to%20predict%20pharmacokinetics&author=RH%20Mathijssen&author=S%20Marsh&author=MO%20Karlsson&author=R%20Xie&author=SD%20Baker&volume=9&publication_year=2003&pages=3246-3253&pmid=12960109&)

22. Han JY, Lim HS, Yoo YK, Shin ES, Park YH, Lee SY, et al. Associations of ABCB1, ABCC2, and ABCG2 polymorphisms with irinotecan-pharmacokinetics and clinical outcome in patients with advanced non-small cell lung cancer. Cancer. 2007;110:138–147. doi: 10.1002/cncr.22760.  [DOI](https://doi.org/10.1002/cncr.22760) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17534875/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer&title=Associations%20of%20ABCB1,%20ABCC2,%20and%20ABCG2%20polymorphisms%20with%20irinotecan-pharmacokinetics%20and%20clinical%20outcome%20in%20patients%20with%20advanced%20non-small%20cell%20lung%20cancer&author=JY%20Han&author=HS%20Lim&author=YK%20Yoo&author=ES%20Shin&author=YH%20Park&volume=110&publication_year=2007&pages=138-147&pmid=17534875&doi=10.1002/cncr.22760&)

23. Han JY, Lim HS, Park YH, Lee SY, Lee JS. Integrated pharmacogenetic prediction of irinotecan pharmacokinetics and toxicity in patients with advanced non-small cell lung cancer. Lung Cancer. 2009;63:115–120. doi: 10.1016/j.lungcan.2007.12.003.  [DOI](https://doi.org/10.1016/j.lungcan.2007.12.003) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18221820/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lung%20Cancer&title=Integrated%20pharmacogenetic%20prediction%20of%20irinotecan%20pharmacokinetics%20and%20toxicity%20in%20patients%20with%20advanced%20non-small%20cell%20lung%20cancer&author=JY%20Han&author=HS%20Lim&author=YH%20Park&author=SY%20Lee&author=JS%20Lee&volume=63&publication_year=2009&pages=115-120&pmid=18221820&doi=10.1016/j.lungcan.2007.12.003&)

24. Sparreboom A, Danesi R, Ando Y, Chan J, Figg WD. Pharmacogenomics of ABC transporters and its role in cancer chemotherapy. Drug Resist Updat. 2003;6:71–84. doi: 10.1016/s1368-7646(03)00005-0.  [DOI](https://doi.org/10.1016/s1368-7646(03)00005-0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12729805/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Resist%20Updat&title=Pharmacogenomics%20of%20ABC%20transporters%20and%20its%20role%20in%20cancer%20chemotherapy&author=A%20Sparreboom&author=R%20Danesi&author=Y%20Ando&author=J%20Chan&author=WD%20Figg&volume=6&publication_year=2003&pages=71-84&pmid=12729805&doi=10.1016/s1368-7646(03)00005-0&)

25. Sai K, Kaniwa N, Itoda M, Saito Y, Hasegawa R, Komamura K, et al. Haplotype analysis of ABCB1/MDR1 blocks in a Japanese population reveals genotype-dependent renal clearance of irinotecan. Pharmacogenetics. 2003;13:741–757. doi: 10.1097/00008571-200312000-00005.  [DOI](https://doi.org/10.1097/00008571-200312000-00005) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14646693/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=Haplotype%20analysis%20of%20ABCB1/MDR1%20blocks%20in%20a%20Japanese%20population%20reveals%20genotype-dependent%20renal%20clearance%20of%20irinotecan&author=K%20Sai&author=N%20Kaniwa&author=M%20Itoda&author=Y%20Saito&author=R%20Hasegawa&volume=13&publication_year=2003&pages=741-757&pmid=14646693&doi=10.1097/00008571-200312000-00005&)

26. Glimelius B, Garmo H, Berglund A, Fredriksson LA, Berglund M, Kohnke H, et al. Prediction of irinotecan and 5-fluorouracil toxicity and response in patients with advanced colorectal cancer. Pharmacogenomics J. 2011;11:61–71. doi: 10.1038/tpj.2010.10.  [DOI](https://doi.org/10.1038/tpj.2010.10) | [PMC free article](/articles/PMC3036798/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20177420/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=Prediction%20of%20irinotecan%20and%205-fluorouracil%20toxicity%20and%20response%20in%20patients%20with%20advanced%20colorectal%20cancer&author=B%20Glimelius&author=H%20Garmo&author=A%20Berglund&author=LA%20Fredriksson&author=M%20Berglund&volume=11&publication_year=2011&pages=61-71&pmid=20177420&doi=10.1038/tpj.2010.10&)

27. Sai K, Saito Y, Maekawa K, Kim SR, Kaniwa N, Nishimaki-Mogami T, et al. Additive effects of drug transporter genetic polymorphisms on irinotecan pharmacokinetics/pharmacodynamics in Japanese cancer patients. Cancer Chemother Pharmacol. 2010;66:95–105. doi: 10.1007/s00280-009-1138-y.  [DOI](https://doi.org/10.1007/s00280-009-1138-y) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19771428/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Chemother%20Pharmacol&title=Additive%20effects%20of%20drug%20transporter%20genetic%20polymorphisms%20on%20irinotecan%20pharmacokinetics/pharmacodynamics%20in%20Japanese%20cancer%20patients&author=K%20Sai&author=Y%20Saito&author=K%20Maekawa&author=SR%20Kim&author=N%20Kaniwa&volume=66&publication_year=2010&pages=95-105&pmid=19771428&doi=10.1007/s00280-009-1138-y&)

28. Purcell S, Neale B, Todd-Brown K, Thomas L, Ferreira MA, Bender D, et al. PLINK: a tool set for whole-genome association and population-based linkage analyses. Am J Hum Genet. 2007;81:559–575. doi: 10.1086/519795.  [DOI](https://doi.org/10.1086/519795) | [PMC free article](/articles/PMC1950838/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17701901/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=PLINK:%20a%20tool%20set%20for%20whole-genome%20association%20and%20population-based%20linkage%20analyses&author=S%20Purcell&author=B%20Neale&author=K%20Todd-Brown&author=L%20Thomas&author=MA%20Ferreira&volume=81&publication_year=2007&pages=559-575&pmid=17701901&doi=10.1086/519795&)

29. R Core Team. R: A language and environment for statistical computing. R Foundation for Statistical Computing; Vienna, Austria: 2015. https://www.R-project.org/  [https://www.R-project.org/](https://www.R-project.org/) | [Google Scholar](https://scholar.google.com/scholar_lookup?title=R:%20A%20language%20and%20environment%20for%20statistical%20computing&publication_year=2015&)

30. Harrell FE. rms: Regression Modeling Strategies. R package version 4.4-0. 2015 http://CRAN.R-project.org/package=rms.  [http://CRAN.R-project.org/package=rms](http://CRAN.R-project.org/package=rms)

31. Ward LD, Kellis M. HaploReg: a resource for exploring chromatin states, conservation, and regulatory motif alterations within sets of genetically linked variants. Nucleic Acids Res. 2012;40:D930–D934. doi: 10.1093/nar/gkr917.  [DOI](https://doi.org/10.1093/nar/gkr917) | [PMC free article](/articles/PMC3245002/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22064851/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=HaploReg:%20a%20resource%20for%20exploring%20chromatin%20states,%20conservation,%20and%20regulatory%20motif%20alterations%20within%20sets%20of%20genetically%20linked%20variants&author=LD%20Ward&author=M%20Kellis&volume=40&publication_year=2012&pages=D930-D934&pmid=22064851&doi=10.1093/nar/gkr917&)

32. Boyle AP, Hong EL, Hariharan M, Cheng Y, Schaub MA, Kasowski M, et al. Annotation of functional variation in personal genomes using RegulomeDB. Genome Res. 2012;22:1790–1797. doi: 10.1101/gr.137323.112.  [DOI](https://doi.org/10.1101/gr.137323.112) | [PMC free article](/articles/PMC3431494/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22955989/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Res&title=Annotation%20of%20functional%20variation%20in%20personal%20genomes%20using%20RegulomeDB&author=AP%20Boyle&author=EL%20Hong&author=M%20Hariharan&author=Y%20Cheng&author=MA%20Schaub&volume=22&publication_year=2012&pages=1790-1797&pmid=22955989&doi=10.1101/gr.137323.112&)

33. Ernst J, Kheradpour P, Mikkelsen TS, Shoresh N, Ward LD, Epstein CB, et al. Mapping and analysis of chromatin state dynamics in nine human cell types. Nature. 2011;473:43–49. doi: 10.1038/nature09906.  [DOI](https://doi.org/10.1038/nature09906) | [PMC free article](/articles/PMC3088773/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21441907/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Mapping%20and%20analysis%20of%20chromatin%20state%20dynamics%20in%20nine%20human%20cell%20types&author=J%20Ernst&author=P%20Kheradpour&author=TS%20Mikkelsen&author=N%20Shoresh&author=LD%20Ward&volume=473&publication_year=2011&pages=43-49&pmid=21441907&doi=10.1038/nature09906&)

34. ENCODE Project Consortium. An integrated encyclopedia of DNA elements in the human genome. Nature. 2012;489:57–74. doi: 10.1038/nature11247.  [DOI](https://doi.org/10.1038/nature11247) | [PMC free article](/articles/PMC3439153/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22955616/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=An%20integrated%20encyclopedia%20of%20DNA%20elements%20in%20the%20human%20genome&volume=489&publication_year=2012&pages=57-74&pmid=22955616&doi=10.1038/nature11247&)

35. Roadmap Epigenomics Consortium. Kundaje A, Meuleman W, Ernst J, Bilenky M, Yen A, et al. Integrative analysis of 111 reference human epigenomes. Nature. 2015;518:317–330. doi: 10.1038/nature14248.  [DOI](https://doi.org/10.1038/nature14248) | [PMC free article](/articles/PMC4530010/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25693563/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Integrative%20analysis%20of%20111%20reference%20human%20epigenomes&author=A%20Kundaje&author=W%20Meuleman&author=J%20Ernst&author=M%20Bilenky&author=A%20Yen&volume=518&publication_year=2015&pages=317-330&pmid=25693563&doi=10.1038/nature14248&)

36. Kel AE, Gössling E, Reuter I, Cheremushkin E, Kel-Margoulis OV, Wingender E. MATCH: A tool for searching transcription factor binding sites in DNA sequences. Nucleic Acids Res. 2003;31:3576–3579. doi: 10.1093/nar/gkg585.  [DOI](https://doi.org/10.1093/nar/gkg585) | [PMC free article](/articles/PMC169193/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12824369/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=MATCH:%20A%20tool%20for%20searching%20transcription%20factor%20binding%20sites%20in%20DNA%20sequences&author=AE%20Kel&author=E%20G%C3%B6ssling&author=I%20Reuter&author=E%20Cheremushkin&author=OV%20Kel-Margoulis&volume=31&publication_year=2003&pages=3576-3579&pmid=12824369&doi=10.1093/nar/gkg585&)

37. Untergasser A, Cutcutache I, Koressaar T, Ye J, Faircloth BC, Remm M, et al. Primer3—new capabilities and interfaces. Nucleic Acids Res. 2012;40:e115. doi: 10.1093/nar/gks596.  [DOI](https://doi.org/10.1093/nar/gks596) | [PMC free article](/articles/PMC3424584/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22730293/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=Primer3%E2%80%94new%20capabilities%20and%20interfaces&author=A%20Untergasser&author=I%20Cutcutache&author=T%20Koressaar&author=J%20Ye&author=BC%20Faircloth&volume=40&publication_year=2012&pages=e115&pmid=22730293&doi=10.1093/nar/gks596&)

38. Koressaar T, Remm M. Enhancements and modifications of primer design program Primer3. Bioinformatics. 2007;23:1289–1291. doi: 10.1093/bioinformatics/btm091.  [DOI](https://doi.org/10.1093/bioinformatics/btm091) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17379693/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=Enhancements%20and%20modifications%20of%20primer%20design%20program%20Primer3&author=T%20Koressaar&author=M%20Remm&volume=23&publication_year=2007&pages=1289-1291&pmid=17379693&doi=10.1093/bioinformatics/btm091&)

39. Simonet WS, Bucay N, Lauer SJ, Taylor JM. A far-downstream hepatocyte-specific control region directs expression of the linked human apolipoprotein E and C-I genes in transgenic mice. J Biol Chem. 1993;268:8221–8229.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/7681840/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=A%20far-downstream%20hepatocyte-specific%20control%20region%20directs%20expression%20of%20the%20linked%20human%20apolipoprotein%20E%20and%20C-I%20genes%20in%20transgenic%20mice&author=WS%20Simonet&author=N%20Bucay&author=SJ%20Lauer&author=JM%20Taylor&volume=268&publication_year=1993&pages=8221-8229&pmid=7681840&)

40. Hodges LM, Markova SM, Chinn LW, Gow JM, Kroetz DL, Klein TE, et al. Very important pharmacogene summary: ABCB1 (MDR1, P-glycoprotein) Pharmacogenet Genomics. 2011;21:152–161. doi: 10.1097/FPC.0b013e3283385a1c.  [DOI](https://doi.org/10.1097/FPC.0b013e3283385a1c) | [PMC free article](/articles/PMC3098758/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20216335/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Very%20important%20pharmacogene%20summary:%20ABCB1%20(MDR1,%20P-glycoprotein)&author=LM%20Hodges&author=SM%20Markova&author=LW%20Chinn&author=JM%20Gow&author=DL%20Kroetz&volume=21&publication_year=2011&pages=152-161&pmid=20216335&doi=10.1097/FPC.0b013e3283385a1c&)

41. Létourneau IJ, Deeley RG, Cole SP. Functional characterization of non-synonymous single nucleotide polymorphisms in the gene encoding human multidrug resistance protein 1 (MRP1/ABCC1) Pharmacogenet Genomics. 2005;15:647–657. doi: 10.1097/01.fpc.0000173484.51807.48.  [DOI](https://doi.org/10.1097/01.fpc.0000173484.51807.48) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16041243/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Functional%20characterization%20of%20non-synonymous%20single%20nucleotide%20polymorphisms%20in%20the%20gene%20encoding%20human%20multidrug%20resistance%20protein%201%20(MRP1/ABCC1)&author=IJ%20L%C3%A9tourneau&author=RG%20Deeley&author=SP%20Cole&volume=15&publication_year=2005&pages=647-657&pmid=16041243&doi=10.1097/01.fpc.0000173484.51807.48&)

42. Conseil G, Cole SP. Two polymorphic variants of ABCC1 selectively alter drug resistance and inhibitor sensitivity of the multidrug and organic anion transporter multidrug resistance protein 1. Drug Metab Dispos. 2013;41:2187–2196. doi: 10.1124/dmd.113.054213.  [DOI](https://doi.org/10.1124/dmd.113.054213) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24080162/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Two%20polymorphic%20variants%20of%20ABCC1%20selectively%20alter%20drug%20resistance%20and%20inhibitor%20sensitivity%20of%20the%20multidrug%20and%20organic%20anion%20transporter%20multidrug%20resistance%20protein%201&author=G%20Conseil&author=SP%20Cole&volume=41&publication_year=2013&pages=2187-2196&pmid=24080162&doi=10.1124/dmd.113.054213&)

43. Yee SW, Chen L, Giacomini KM. Pharmacogenomics of membrane transporters: past, present and future. Pharmacogenomics. 2010;11:475–479. doi: 10.2217/pgs.10.22.  [DOI](https://doi.org/10.2217/pgs.10.22) | [PMC free article](/articles/PMC3234298/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20350125/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Pharmacogenomics%20of%20membrane%20transporters:%20past,%20present%20and%20future&author=SW%20Yee&author=L%20Chen&author=KM%20Giacomini&volume=11&publication_year=2010&pages=475-479&pmid=20350125&doi=10.2217/pgs.10.22&)

44. Hesselson SE, Matsson P, Shima JE, Fukushima H, Yee SW, Kobayashi Y, et al. Genetic variation in the proximal promoter of ABC and SLC superfamilies: liver and kidney specific expression and promoter activity predict variation. PLoS ONE. 2009;4:e6942. doi: 10.1371/journal.pone.0006942.  [DOI](https://doi.org/10.1371/journal.pone.0006942) | [PMC free article](/articles/PMC2735003/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19742321/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&title=Genetic%20variation%20in%20the%20proximal%20promoter%20of%20ABC%20and%20SLC%20superfamilies:%20liver%20and%20kidney%20specific%20expression%20and%20promoter%20activity%20predict%20variation&author=SE%20Hesselson&author=P%20Matsson&author=JE%20Shima&author=H%20Fukushima&author=SW%20Yee&volume=4&publication_year=2009&pages=e6942&pmid=19742321&doi=10.1371/journal.pone.0006942&)

45. Choi JH, Yee SW, Kim MJ, Nguyen L, Lee JH, Kang JO, et al. Identification and characterization of novel polymorphisms in the basal promoter of the human transporter, MATE1. Pharmacogenet Genomics. 2009;19:770–780. doi: 10.1097/FPC.0b013e328330eeca.  [DOI](https://doi.org/10.1097/FPC.0b013e328330eeca) | [PMC free article](/articles/PMC2976711/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19745787/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Identification%20and%20characterization%20of%20novel%20polymorphisms%20in%20the%20basal%20promoter%20of%20the%20human%20transporter,%20MATE1&author=JH%20Choi&author=SW%20Yee&author=MJ%20Kim&author=L%20Nguyen&author=JH%20Lee&volume=19&publication_year=2009&pages=770-780&pmid=19745787&doi=10.1097/FPC.0b013e328330eeca&)

46. Tahara H, Yee SW, Urban TJ, Hesselson S, Castro RA, Kawamoto M, et al. Functional genetic variation in the basal promoter of the organic cation/carnitine transporters OCTN1 (SLC22A4) and OCTN2 (SLC22A5) J Pharmacol Exp Ther. 2009;329:262–271. doi: 10.1124/jpet.108.146449.  [DOI](https://doi.org/10.1124/jpet.108.146449) | [PMC free article](/articles/PMC2670604/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19141711/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharmacol%20Exp%20Ther&title=Functional%20genetic%20variation%20in%20the%20basal%20promoter%20of%20the%20organic%20cation/carnitine%20transporters%20OCTN1%20(SLC22A4)%20and%20OCTN2%20(SLC22A5)&author=H%20Tahara&author=SW%20Yee&author=TJ%20Urban&author=S%20Hesselson&author=RA%20Castro&volume=329&publication_year=2009&pages=262-271&pmid=19141711&doi=10.1124/jpet.108.146449&)

47. Yee SW, Shima JE, Hesselson S, Nguyen L, De Val S, Lafond RJ, et al. Identification and characterization of proximal promoter polymorphisms in the human concentrative nucleoside transporter 2 (SLC28A2) J Pharmacol Exp Ther. 2009;328:699–707. doi: 10.1124/jpet.108.147207.  [DOI](https://doi.org/10.1124/jpet.108.147207) | [PMC free article](/articles/PMC2682268/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19098160/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharmacol%20Exp%20Ther&title=Identification%20and%20characterization%20of%20proximal%20promoter%20polymorphisms%20in%20the%20human%20concentrative%20nucleoside%20transporter%202%20(SLC28A2)&author=SW%20Yee&author=JE%20Shima&author=S%20Hesselson&author=L%20Nguyen&author=S%20De%20Val&volume=328&publication_year=2009&pages=699-707&pmid=19098160&doi=10.1124/jpet.108.147207&)

48. To KK, Zhan Z, Litman T, Bates SE. Regulation of ABCG2 expression at the 3′ untranslated region of its mRNA through modulation of transcript stability and protein translation by a putative microRNA in the S1 colon cancer cell line. Mol Cell Biol. 2008;28:5147–5161. doi: 10.1128/MCB.00331-08.  [DOI](https://doi.org/10.1128/MCB.00331-08) | [PMC free article](/articles/PMC2519722/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18573883/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Cell%20Biol&title=Regulation%20of%20ABCG2%20expression%20at%20the%203%E2%80%B2%20untranslated%20region%20of%20its%20mRNA%20through%20modulation%20of%20transcript%20stability%20and%20protein%20translation%20by%20a%20putative%20microRNA%20in%20the%20S1%20colon%20cancer%20cell%20line&author=KK%20To&author=Z%20Zhan&author=T%20Litman&author=SE%20Bates&volume=28&publication_year=2008&pages=5147-5161&pmid=18573883&doi=10.1128/MCB.00331-08&)

49. Pan YZ, Morris ME, Yu AM. MicroRNA-328 negatively regulates the expression of breast cancer resistance protein (BCRP/ABCG2) in human cancer cells. Mol Pharmacol. 2009;75:1374–1379. doi: 10.1124/mol.108.054163.  [DOI](https://doi.org/10.1124/mol.108.054163) | [PMC free article](/articles/PMC2684886/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19270061/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Pharmacol&title=MicroRNA-328%20negatively%20regulates%20the%20expression%20of%20breast%20cancer%20resistance%20protein%20(BCRP/ABCG2)%20in%20human%20cancer%20cells&author=YZ%20Pan&author=ME%20Morris&author=AM%20Yu&volume=75&publication_year=2009&pages=1374-1379&pmid=19270061&doi=10.1124/mol.108.054163&)

50. Kim MJ, Skewes-Cox P, Fukushima H, Hesselson S, Yee SW, Ramsey LB, et al. Functional characterization of liver enhancers that regulate drug-associated transporters. Clin Pharmacol Ther. 2011;89:571–578. doi: 10.1038/clpt.2010.353.  [DOI](https://doi.org/10.1038/clpt.2010.353) | [PMC free article](/articles/PMC3227682/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21368754/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Functional%20characterization%20of%20liver%20enhancers%20that%20regulate%20drug-associated%20transporters&author=MJ%20Kim&author=P%20Skewes-Cox&author=H%20Fukushima&author=S%20Hesselson&author=SW%20Yee&volume=89&publication_year=2011&pages=571-578&pmid=21368754&doi=10.1038/clpt.2010.353&)

51. O’Brien VP, Bokelmann K, Ramírez J, Jobst K, Ratain MJ, Brockmöller J, et al. Hepatocyte nuclear factor 1 regulates the expression of the organic cation transporter 1 via binding to an evolutionary conserved region in intron 1 of the OCT1 gene. J Pharmacol Exp Ther. 2013;347:181–192. doi: 10.1124/jpet.113.206359.  [DOI](https://doi.org/10.1124/jpet.113.206359) | [PMC free article](/articles/PMC3781413/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23922447/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharmacol%20Exp%20Ther&title=Hepatocyte%20nuclear%20factor%201%20regulates%20the%20expression%20of%20the%20organic%20cation%20transporter%201%20via%20binding%20to%20an%20evolutionary%20conserved%20region%20in%20intron%201%20of%20the%20OCT1%20gene&author=VP%20O%E2%80%99Brien&author=K%20Bokelmann&author=J%20Ram%C3%ADrez&author=K%20Jobst&author=MJ%20Ratain&volume=347&publication_year=2013&pages=181-192&pmid=23922447&doi=10.1124/jpet.113.206359&)

52. Poonkuzhali B, Lamba J, Strom S, Sparreboom A, Thummel K, Watkins P, et al. Association of breast cancer resistance protein/ABCG2 phenotypes and novel promoter and intron 1 single nucleotide polymorphisms. Drug Metab Dispos. 2008;36:780–795. doi: 10.1124/dmd.107.018366.  [DOI](https://doi.org/10.1124/dmd.107.018366) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18180275/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Association%20of%20breast%20cancer%20resistance%20protein/ABCG2%20phenotypes%20and%20novel%20promoter%20and%20intron%201%20single%20nucleotide%20polymorphisms&author=B%20Poonkuzhali&author=J%20Lamba&author=S%20Strom&author=A%20Sparreboom&author=K%20Thummel&volume=36&publication_year=2008&pages=780-795&pmid=18180275&doi=10.1124/dmd.107.018366&)

53. Zhou Q, Sparreboom A, Tan EH, Cheung YB, Lee A, Poon D, et al. Pharmacogenetic profiling across the irinotecan pathway in Asian patients with cancer. Br J Clin Pharmacol. 2005;59:415–424. doi: 10.1111/j.1365-2125.2004.02330.x.  [DOI](https://doi.org/10.1111/j.1365-2125.2004.02330.x) | [PMC free article](/articles/PMC1884809/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15801936/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Clin%20Pharmacol&title=Pharmacogenetic%20profiling%20across%20the%20irinotecan%20pathway%20in%20Asian%20patients%20with%20cancer&author=Q%20Zhou&author=A%20Sparreboom&author=EH%20Tan&author=YB%20Cheung&author=A%20Lee&volume=59&publication_year=2005&pages=415-424&pmid=15801936&doi=10.1111/j.1365-2125.2004.02330.x&)

54. Iyer L, Ramírez J, Shepard DR, Bingham CM, Hossfeld DK, Ratain MJ, et al. Biliary transport of irinotecan and metabolites in normal and P-glycoprotein-deficient mice. Cancer Chemother Pharmacol. 2002;49:336–341. doi: 10.1007/s00280-001-0420-4.  [DOI](https://doi.org/10.1007/s00280-001-0420-4) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11914914/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Chemother%20Pharmacol&title=Biliary%20transport%20of%20irinotecan%20and%20metabolites%20in%20normal%20and%20P-glycoprotein-deficient%20mice&author=L%20Iyer&author=J%20Ram%C3%ADrez&author=DR%20Shepard&author=CM%20Bingham&author=DK%20Hossfeld&volume=49&publication_year=2002&pages=336-341&pmid=11914914&doi=10.1007/s00280-001-0420-4&)

55. Tagen M, Zhuang Y, Zhang F, Harstead KE, Shen J, Schaiquevich P, et al. P-glycoprotein, but not multidrug resistance protein 4, plays a role in the systemic clearance of irinotecan and SN-38 in mice. Drug Metab Lett. 2010;4:195–201. doi: 10.2174/187231210792928251.  [DOI](https://doi.org/10.2174/187231210792928251) | [PMC free article](/articles/PMC4486004/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20583968/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Lett&title=P-glycoprotein,%20but%20not%20multidrug%20resistance%20protein%204,%20plays%20a%20role%20in%20the%20systemic%20clearance%20of%20irinotecan%20and%20SN-38%20in%20mice&author=M%20Tagen&author=Y%20Zhuang&author=F%20Zhang&author=KE%20Harstead&author=J%20Shen&volume=4&publication_year=2010&pages=195-201&pmid=20583968&doi=10.2174/187231210792928251&)

56. Chen ZS, Furukawa T, Sumizawa T, Ono K, Ueda K, Seto K, et al. ATP-Dependent efflux of CPT-11 and SN-38 by the multidrug resistance protein (MRP) and its inhibition by PAK-104P. Mol Pharmacol. 1999;55:921–928.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/10220571/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Pharmacol&title=ATP-Dependent%20efflux%20of%20CPT-11%20and%20SN-38%20by%20the%20multidrug%20resistance%20protein%20(MRP)%20and%20its%20inhibition%20by%20PAK-104P&author=ZS%20Chen&author=T%20Furukawa&author=T%20Sumizawa&author=K%20Ono&author=K%20Ueda&volume=55&publication_year=1999&pages=921-928&pmid=10220571&)

57. Roelofsen H, Vos TA, Schippers IJ, Kuipers F, Koning H, Moshage H, et al. Increased levels of the multidrug resistance protein in lateral membranes of proliferating hepatocyte-derived cells. Gastroenterology. 1997;112:511–521. doi: 10.1053/gast.1997.v112.pm9024305.  [DOI](https://doi.org/10.1053/gast.1997.v112.pm9024305) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9024305/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gastroenterology&title=Increased%20levels%20of%20the%20multidrug%20resistance%20protein%20in%20lateral%20membranes%20of%20proliferating%20hepatocyte-derived%20cells&author=H%20Roelofsen&author=TA%20Vos&author=IJ%20Schippers&author=F%20Kuipers&author=H%20Koning&volume=112&publication_year=1997&pages=511-521&pmid=9024305&doi=10.1053/gast.1997.v112.pm9024305&)

58. Seiser EL, Xia K, Wright FA, Innocenti F. Meta-analysis of liver eQTL studies and cross-tissue eQTL comparison using GTEx data (Abstract #667S). Presented at the 64th Annual Meeting of The American Society of Human Genetics; October 20, 2014; San Diego, CA.  [Google Scholar](https://scholar.google.com/scholar_lookup?Seiser%20EL,%20Xia%20K,%20Wright%20FA,%20Innocenti%20F.%20Meta-analysis%20of%20liver%20eQTL%20studies%20and%20cross-tissue%20eQTL%20comparison%20using%20GTEx%20data%20(Abstract%20#667S).%20Presented%20at%20the%2064th%20Annual%20Meeting%20of%20The%20American%20Society%20of%20Human%20Genetics;%20October%2020,%202014;%20San%20Diego,%20CA.)

59. Schellens JH, Maliepaard M, Scheper RJ, Scheffer GL, Jonker JW, Smit JW, et al. Transport of topoisomerase I inhibitors by the breast cancer resistance protein. Potential clinical implications. Ann N Y Acad Sci. 2000;922:188–194. doi: 10.1111/j.1749-6632.2000.tb07037.x.  [DOI](https://doi.org/10.1111/j.1749-6632.2000.tb07037.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11193894/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20N%20Y%20Acad%20Sci&title=Transport%20of%20topoisomerase%20I%20inhibitors%20by%20the%20breast%20cancer%20resistance%20protein.%20Potential%20clinical%20implications&author=JH%20Schellens&author=M%20Maliepaard&author=RJ%20Scheper&author=GL%20Scheffer&author=JW%20Jonker&volume=922&publication_year=2000&pages=188-194&pmid=11193894&doi=10.1111/j.1749-6632.2000.tb07037.x&)

60. Candeil L, Gourdier I, Peyron D, Vezzio N, Copois V, Bibeau F, et al. ABCG2 overexpression in colon cancer cells resistant to SN38 and in irinotecan-treated metastases. Int J Cancer. 2004;109:858–854. doi: 10.1002/ijc.20032.  [DOI](https://doi.org/10.1002/ijc.20032) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15027118/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Cancer&title=ABCG2%20overexpression%20in%20colon%20cancer%20cells%20resistant%20to%20SN38%20and%20in%20irinotecan-treated%20metastases&author=L%20Candeil&author=I%20Gourdier&author=D%20Peyron&author=N%20Vezzio&author=V%20Copois&volume=109&publication_year=2004&pages=858-854&pmid=15027118&doi=10.1002/ijc.20032&)

61. Imai Y, Nakane M, Kage K, Tsukahara S, Ishikawa E, Tsuruo T, et al. C421A polymorphism in the human breast cancer resistance protein gene is associated with low expression of Q141K protein and low-level drug resistance. Mol Cancer Ther. 2002;1:611–616.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/12479221/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Cancer%20Ther&title=C421A%20polymorphism%20in%20the%20human%20breast%20cancer%20resistance%20protein%20gene%20is%20associated%20with%20low%20expression%20of%20Q141K%20protein%20and%20low-level%20drug%20resistance&author=Y%20Imai&author=M%20Nakane&author=K%20Kage&author=S%20Tsukahara&author=E%20Ishikawa&volume=1&publication_year=2002&pages=611-616&pmid=12479221&)

62. Phelps MA, Sparreboom A. Irinotecan pharmacogenetics: a finished puzzle? J Clin Oncol. 2014;32:2287–2289. doi: 10.1200/JCO.2014.56.3387.  [DOI](https://doi.org/10.1200/JCO.2014.56.3387) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24958823/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Irinotecan%20pharmacogenetics:%20a%20finished%20puzzle?&author=MA%20Phelps&author=A%20Sparreboom&volume=32&publication_year=2014&pages=2287-2289&pmid=24958823&doi=10.1200/JCO.2014.56.3387&)

63. Fujiwara Y, Minami H. An overview of the recent progress in irinotecan pharmacogenetics. Pharmacogenomics. 2010;11:391–406. doi: 10.2217/pgs.10.19.  [DOI](https://doi.org/10.2217/pgs.10.19) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20235794/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=An%20overview%20of%20the%20recent%20progress%20in%20irinotecan%20pharmacogenetics&author=Y%20Fujiwara&author=H%20Minami&volume=11&publication_year=2010&pages=391-406&pmid=20235794&doi=10.2217/pgs.10.19&)

64. Marsh S, Hoskins JM. Irinotecan pharmacogenomics. Pharmacogenomics. 2010;11:1003–1010. doi: 10.2217/pgs.10.95.  [DOI](https://doi.org/10.2217/pgs.10.95) | [PMC free article](/articles/PMC2927346/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20602618/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Irinotecan%20pharmacogenomics&author=S%20Marsh&author=JM%20Hoskins&volume=11&publication_year=2010&pages=1003-1010&pmid=20602618&doi=10.2217/pgs.10.95&)

65. Hoskins JM, Rosner GL, Ratain MJ, McLeod HL, Innocenti F. Pharmacodynamic genes do not influence risk of neutropenia in cancer patients treated with moderately high-dose irinotecan. Pharmacogenomics. 2009;10:1139–1146. doi: 10.2217/pgs.09.35.  [DOI](https://doi.org/10.2217/pgs.09.35) | [PMC free article](/articles/PMC2748108/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19604089/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Pharmacodynamic%20genes%20do%20not%20influence%20risk%20of%20neutropenia%20in%20cancer%20patients%20treated%20with%20moderately%20high-dose%20irinotecan&author=JM%20Hoskins&author=GL%20Rosner&author=MJ%20Ratain&author=HL%20McLeod&author=F%20Innocenti&volume=10&publication_year=2009&pages=1139-1146&pmid=19604089&doi=10.2217/pgs.09.35&)

66. 1000 Genomes Project Consortium. Auton A, Brooks LD, Durbin RM, Garrison EP, Kang HM, et al. A global reference for human genetic variation. Nature. 2015;526:68–74. doi: 10.1038/nature15393.  [DOI](https://doi.org/10.1038/nature15393) | [PMC free article](/articles/PMC4750478/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26432245/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=A%20global%20reference%20for%20human%20genetic%20variation&author=A%20Auton&author=LD%20Brooks&author=RM%20Durbin&author=EP%20Garrison&author=HM%20Kang&volume=526&publication_year=2015&pages=68-74&pmid=26432245&doi=10.1038/nature15393&)

67. International HapMap Consortium. The International HapMap Project. Nature. 2003;426:789–796. doi: 10.1038/nature02168.  [DOI](https://doi.org/10.1038/nature02168) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14685227/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=The%20International%20HapMap%20Project&volume=426&publication_year=2003&pages=789-796&pmid=14685227&doi=10.1038/nature02168&)
