# CYP3A4 genotype is associated with sildenafil concentrations in patients with heart failure with preserved ejection fraction

## Metadata
**Authors:** Simon de Denus, Jean L Rouleau, Douglas L Mann, Gordon S Huggins, Naveen L Pereira, Svati H Shah, Thomas P Cappola, René Fouodjio, Ian Mongrain, Marie-Pierre Dubé
**Journal:** The pharmacogenomics journal
**Date:** 2017 Apr 25
**DOI:** [10.1038/tpj.2017.8](https://doi.org/10.1038/tpj.2017.8)
**PMID:** 28440343
**PMCID:** PMC5656562
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5656562/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC5656562/pdf/nihms856822.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC5656562/pdf/nihms856822.pdf)

## Abstract

Despite its established inter-individual variability, sildenafil has been the subject of only a few pharmacogenetic investigations, with limited data regarding the genetic modulators of its pharmacokinetics. We conducted a pharmacogenetic substudy of patients randomized to sildenafil (n = 85) in the RELAX trial, which investigated the impact of high-dose sildenafil in patients with heart failure with preserved left ventricular ejection fraction (HFpEF). In the overall population, the CYP3A4 inferred phenotype appeared associated with the dose-adjusted peak concentrations of sildenafil at week 12 and week 24 (adjusted P=0.045 for repeated measures analysis), although this P value did not meet our corrected significance threshold of 0.0167. In the more homogeneous Caucasian subgroup, this association was significant (adjusted P=0.0165 for repeated measures). Hence, CYP3A4 inferred phenotype is associated with peak sildenafil dose-adjusted concentrations in patients with HFpEF receiving high doses of sildenafil. The clinical impact of this association requires further investigation.

Keywords: Sildenafil, pharmacokinetics, heart failure, pharmacogenetics

## INTRODUCTION

The phosphodiesterase 5 inhibitor (PDE5) sildenafil (Viagra®) has been one of the most widely prescribed drugs in the world for nearly two decades. Despite the established inter-individual variability of its pharmacokinetics and efficacy,^1–3^ sildenafil has been the subject of only a few pharmacogenomic (PGx) investigations. Limited and inconsistent data from small to moderate size studies have suggested that genetic variants related to a variety of biological pathways could influence sildenafil’s hemodynamic effects in patients with pulmonary hypertension or its efficacy in treating erectile dysfunction.^4–9^

Considering that a majority of the markers judged to be clinically useful by the National Institutes of Health’s Clinical PGx Implementation Consortium are absorption, distribution, metabolism and elimination (ADME) genes,^10^ it is surprising that few studies have focused on variants of genes which could modulate sildenafil concentrations and dosing requirements. In particular, given that sildenafil is extensively metabolized in the liver by the cytochrome P450 3A (*CYP3A*) isoenzymes (major metabolizing route; 79%) and *CYP2C9* (20%),^11–13^ genetic variants coding for these isoenzymes would biologically appear to be likely genetic modulators of the effects of sildenafil. Although small studies (*n* < 25) have investigated whether *CYP3A5* and *CYP2C9* were associated with sildenafil concentrations and pharmacokinetics,^2,14^ the small sample size from these studies limits their statistical power to identify significant associations.

The RELAX (Phosphodiesterase-5 Inhibition to Improve Clinical Status and Exercise Capacity in Heart Failure with Preserved Ejection Fraction) trial, which investigated the impact of high-dose sildenafil on the exercise capacity and clinical status of patients with heart failure with preserved left ventricular ejection fraction (HFpEF), represents a unique opportunity to further explore the genetic determinants of serum concentrations of sildenafil in a larger population than previous studies. Indeed, as part of RELAX, peak sildenafil concentrations were measured after twelve and twenty-four weeks of treatment.^15^ Because no beneficial hemodynamic or remodelling effects were observed in RELAX, the aim of this pharmacogenetic sub-study was to identify predictors of sildenafil peak concentrations. Based on existing evidence,^2,14,16^ the primary goal of this ancillary study was to investigate the impact of variants in the *CYP2C9*, *CYP3A4* and *CYP3A5* genes on dose-adjusted peak concentrations of sildenafil measured after 12 and 24 weeks of treatment. We hypothesized that patients carrying genotypes associated with a greater metabolizing capacity for these isoenzymes, such as extensive metabolizers (EM), would present lower dose-adjusted peak concentrations than carriers of genetic variations associated with a lower metabolizing capacity, such as intermediate metabolizers (IM) or poor metabolizers (PM).

## METHODS

### Overview of study design

The methods and results of the RELAX trial ([clinicaltrials.gov](http://clinicaltrials.gov) Identifier: [NCT00763867](https://clinicaltrials.gov/ct2/show/NCT00763867)) have been reported previously.^15,17^ Briefly, RELAX was a multicenter randomized placebo-controlled trial which investigated the impact of high-dose sildenafil on exercise tolerance in patients with HFpEF (LVEF ≥ 50% in the last 12 months presenting New York Heart Association functional class II through IV) whose symptoms were stable while receiving medical therapy.^15^ The use of significant CYP3A4 inhibitors (e.g. ketoconazole, erythromycin), as well as a current or anticipated future need for nitrate therapy, were exclusion criteria.

Sildenafil was administered orally at 20 mg three times a day for 12 weeks. After 12 weeks, study endpoints were measured as previously described^15,17^ including peak sildenafil concentrations, which were obtained through phlebotomy 45 to 120 minutes after the scheduled dose. Following this, if the 20 mg three times a day dose was well tolerated, it was then increased to 60 mg three times a day for 12 weeks; otherwise the dose was maintained at 20 mg three times a day. Study endpoint measurements were repeated at week 24, including peak sildenafil concentrations. Sildenafil concentrations were measured by liquid chromatography tandem mass spectrometry (LC-MS/MS) as previously reported.^15^ Plasma cyclic GMP (cGMP) levels were measured as previously reported at week 24.^15^ For the current sub–study, we limited our investigations to the 85 patients in the sildenafil group who provided informed consent to participate in the genomic/pharmacogenomic sub-study and who provided a genetic sample.

### Genetic analyses

The methods of the Heart Failure Network (HFN) genomics/pharmacogenomics sub-studies, including DNA extraction, genotyping and quality control have been previously reported.^18^ The genotyping strategy included multiple commercial and custom platforms. Among the commercial platforms, we used Sequenom’s iPLEX^®^ ADME PGx Panel (Sequenom [now Agena Bioscience], San Diego, CA, USA) to genotype functional SNPs related to the absorption, distribution, metabolism and excretion (ADME). For the current report, we limited our investigations to the variants included on Sequenom’s iPLEX^®^ ADME PGx Panel which contains 192 genetic variants, including 183 SNPs which passed genotyping quality control. Following additional data clean-up specific to the group of participants and SNPs of the current analysis (see [supplementary information](#SD1)), 75 SNPs with a minor allele frequency (MAF) of ≥0.01 (from 32 genes; see [supplementary information](#SD1)) were included.

### Metabolizer status inference

Genetic variants genotyped on the Sequenom ADME panel were used to infer the metabolizer status of the primary genes of interest (*CYP2C9*, *CYP3A4*, *CYP3A5*) according to published and recognized scientific evidence in the field (see [supplementary information](#SD1)). This approach has the advantage of combining the information of multiple SNPs from a given gene into one variable, thus reducing the number of statistical tests and the severity adjustment made to control for multiple comparisons. This approach was also used for other selected secondary genes (*CYP2D6*, *CYP2C19*, *CYP2E1*, *CYP2B6*, *CYP2A6*, *CYP2C8*, *DPYD*, *SLCO1B1*, *UGT1A1*, *SUL1A1*, *NAT1;* see [supplementary information](#SD1)), although all secondary genes were also tested as individual SNPs, given that, with the exception of *CYP2D6* and *CYP2C19*, more limited consensus data is available regarding the functional consequences of variants in these genes. Based on previous evidence, and depending on the gene, the metabolizer status was defined, for example, as poor metabolizer (PM), intermediate metabolizer (IM), extensive metabolizer (EM), or ultra metabolizer (UM).

### Study objectives and endpoints

The primary objective of this RELAX pharmacogenetic sub-study was to test the association between the predicted phenotype for *CYP2C9*, *CYP3A4* and *CYP3A5* and the concentration:dose ratio of sildenafil measured at 12 and 24 weeks. Similar ratios have been previously used to take into account the impact of different doses on concentrations measured.^19–22^ Another important factor that motivated our selection of this phenotype as a primary outcome is that it was measured on 2 separate occasions in RELAX. Indeed, our group has used such repeated measures for analyses in previous pharmacogenomic studies^21,23^ to improve statistical power.

The main secondary objective was to evaluate the association between these three genes and concentrations of cGMP at week 24 in patients still receiving sildenafil. As part of the primary RELAX trial, sildenafil was shown to significantly increase cGMP concentrations compared to baseline, although this difference was not statistically significant compared with placebo.

Other secondary objectives included the evaluation of the association between these endpoints and other ADME genes included on the Sequenom iPlex ADME panel. Finally, as an exploratory analysis, we planned to investigate the relationship of variants significantly associated with the concentration: dose ratio of sildenafil with the occurrence of hypotension and flushing, which were more frequent in the sildenafil group than the placebo group in RELAX.

### Statistical analysis

For continuous variables, descriptive statistics are presented as mean ± standard deviation, while categorical variables are presented as counts and percentages of each category. Prior to the statistical analyses, the normality test was performed on each endpoint using Shapiro-Wilk test at the 0.01 significance level. As this test appeared to be significant, the normality assumption was rejected and the data was transformed using the log_10_, square root, inverse and the square functions. The square root of sildenafil peak concentration, the log_10_ of concentration/dose ratio of sildenafil, and the log_10_ of cGMP were selected to be the best functions to be statistically analyzed. Data are presented as non-transformed.

### Genetic association analyses

The concentration: dose ratio was statistically analyzed using the MIXED procedure of SAS, first including only non-genetic covariates into the analysis model. The repeating factor was the study visit. For cGMP, we used a GLM regression model that, again, was first fitted including non–genetic covariates into the analysis. For the risk of hypotension or flushing, a similar approach was utilized using a logistic regression. For all models, the non-genetic variables investigated were age, sex, body mass index, estimated glomerular filtration rate (eGFR) and amiodarone use. A univariate step selection was conducted between each endpoint and each covariate at 0.05 significance level. When more than one covariate was found to be significantly associated to the endpoint, a stepwise selection method was then performed between each endpoint and these clinical covariates with a *P*-value of less than 0.1 (entry *P*-value=0.1, stay *P*–value≤0.15).

The most important principal components (PC) of the population structure (components 1–3) were included in all genetic analyses to control for genetic ancestry and to avoid confounding by population structure,^24^ whether or not they were significant at the stepwise level. Thus, all the multivariate genetic models for each of the endpoints were adjusted for the non-genetic covariate meeting the aforementioned criteria and genetic ancestry (PC).

For genes, the inferred metabolizer status was defined, depending on the gene, as poor metabolizer (PM), intermediate metabolizer (IM), extensive metabolizer (EM), or ultra metabolizer (UM) and coded respectively as 0, 1, 2, and 3. For the individual SNP analyses, genotype was coded as 0, 1, or 2 according to the number of copies of minor alleles.

As sensitivity analyses, primary genes and any other genes or SNPs meeting our pre–defined statistical threshold were then analyzed in Caucasians only. Statistical analyses were performed using SAS, PLINK, and existing scripts.

### Significance threshold

Statistical tests performed were two-sided and adjusted to account for the multiple testing of genes. For the three genes for the primary analysis (*CYP2C9*, *CYP3A4* and *CYP3A5*) we used a Bonferroni-adjusted p value threshold of 0.05/3=0.0167. For all other variants, the significance threshold was adjusted using a Bonferroni correction based on the effective number of independent tests (M_eff_) of all candidate genes, including those of objective 1. The M_eff_ was computed using the method of Gao et al^25^ and was set at 49. Then, the adjusted significance threshold was calculated as 0.05/49 = 0.001.

### Ethical consideration

The RELAX study was approved by all local Institutional Review Boards (IRBs) and all patients provided a written consent to participate in the study. Moreover, in centers electing to participate in the genomics/pharmacogenomics sub-study of the HFN clinical trials, the sub-study was approved by the IRB at each side. Patients taking part in the RELAX trial in these participating centers were offered the possibility to participate in the genetics sub-study. All participants willing to participate in the study provided written informed consent.

## RESULTS

A total of 85 patients were included as part of this sub-study ([Table 1](#T1)). Study subjects were more likely to be males (54%), Caucasian (92%) and were 67.9±10.5 years old. The predicted phenotypes and related genotypes of the study participants for *CYP2C9*, *CYP3A4, CYP3A5* are detailed in [Table 2](#T2).

### Table 1.

| Characteristic | Patients treated with sildenafil(n = 85) |
| --- | --- |
| Female, n (%) | 39 (45.9) |
| Age, years | 67.9 ± 10.5 |
| Race, n (%) |   |
| White | 78 (91.8) |
| Black | 4 (4.7) |
| Other | 3 (3.5) |
| Diabetes, n (%) | 32 (37.6) |
| Blood pressure, mmHg | 126.7 ± 16.6/70.9 ± 10.0 |
| Heart rate, bpm | 69.9 ± 12.2 |
| Baseline LVEF, % | 61.6 ± 6.2 |
| NYHA class, II/III | 44 (51.8)/ 41 (48.2) |
| BMI, kg/m2 | 33.9 ± 7.3 |
| Creatinine, mg/dL | 1.2 ± 0.5 |
| eGFR, ml/min/1.73 m2 | 67.3 ± 23.1 |
| NT proBNP, pg/mL | 1012.5 ± 1212.4 |
| cGMP, pmol/mL | 82.5 ± 36.0 |
| Treatment |   |
| ACE inhibitor, n (%) | 34 (40.0) |
| Angiotensin receptor blocker, n (%) | 19 (22.4) |
| Beta blocker use, n (%) | 62 (72.9) |
| Aldosterone antagonist, n (%) | 11 (12.9) |
| Hydrochlorothiazide, n (%) | 11(12.9) |
| Metolazone, n (%) | 2 (2.4) |
| Loop diuretics, n (%) |   |
| Furosemide | 52 (61.2) |
| Torsemide | 7 (8.2) |
| Bumetanide | 4 (4.7) |
| Amiodarone, n (%) | 7 (8.2) |
| Calcium channel blocker, n (%) | 20 (23.5) |

Table 1 Caption: Baseline characteristics of the study population

### Table 2.

| Gene | Phenotype | Genotype | Frequency, n (%) |
| --- | --- | --- | --- |
| All patients |  |  |  |
| CYP2C9 | EM | *1/*1 | 52 (62.7) |
|   |   | *1/*9 | 2 (2.4) |
|   | IM | *1/*2 | 18 (21.7) |
|   |   | *1/*3 | 7 (8.4) |
|   | PM | *2/*2 | 1 (1.2) |
|   |   | *2/*3 | 3 (3.6) |
| CYP3A4 | EM | *1/*1 | 72 (85.7) |
|   | IM | *1/*22 | 12 (14.3) |
| CYP3A5 | EM | *1/*1 | 1 (1.2) |
|   | IM | *1/*3 | 7 (8.2) |
|   | PM | *3/*3 | 74 (87.1) |
|   |   | *3/*6 | 2 (2.4) |
|   |   | *3/*7 | 1 (1.2) |
| Caucasians |  |  |  |
| CYP2C9 | EM | *1/*1 | 48 (63.2) |
|   | IM | *1/*2 | 18 (23.7) |
|   |   | *1/*3 | 6 (7.9) |
|   | PM | *2/*2 | 1 (1.3) |
|   |   | *2/*3 | 3 (3.9) |
| CYP3A4 | EM | *1/*1 | 66 (85.7) |
|   | IM | *1/*22 | 11 (14.3) |
| CYP3A5 | IM | *1/*3 | 6 (7.7) |
|   | PM | *3/*3 | 72 (92.3) |

Table 2 Caption: Inferred phenotypes and related genotypes in the overall population and in the Caucasian subgroup

### Predictors of dose-concentration ratio of sildenafil

Of the 85 patients enrolled in this sub-study, 77 individuals had at least one sildenafil concentration measured (week 12: *n* = 71; week 24: *n* = 66). In univariate analysis, both age and BMI were significantly associated with concentration: dose ratios of sildenafil (all *P* < 0.0001).

In regards to the primary genes of interest, in the overall population, consistent with their predicted lower metabolising capacity, the IM for the CYP3A4 inferred phenotype presented a higher ratio of sildenafil than EM at both week 12 and week 24 ([Figure 1](#F1), *P* = 0.045 for repeated measures analysis, after adjusting for age, BMI and three PCs for genetic ancestry). All IM were *CYP3A4**22 carriers. Although this association p value was below the nominal threshold of 0.05, it did not meet our corrected significance threshold of 0.0167. Nevertheless, in the more homogeneous Caucasian subgroup, this association was found to be significant after adjusting for multiple comparisons (*P* = 0.0165 for repeated measures analysis, after adjusting for age, BMI and three PCs for genetic ancestry [[Figure 1](#F1)]).

### Figure 1. Concentration:dose ratios of sildenafil according to CYP3A4 inferred phenotype in (A) all patients and (B) Caucasians.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/844d/5656562/822b5fd917b0/nihms856822f1.jpg)

After adjusting for age, body mass index and three principal components for genetic ancestry, CYP3A4 inferred phenotype was significantly associated with the sildenafil concentration:dose ratio in Caucasians (P = 0.0165 for repeated measures analysis). EM, extensive metabolisers; IM, intermediate metabolizers.

No association was observed between the *CYP2C9* inferred phenotype and the sildenafil concentration:dose ratio ([Figure 2](#F2)). In regards to the *CYP3A5* inferred phenotype, although PM tended to have higher concentrations, particularly in the Caucasian subgroup, *P* values did not reach our significance corrected threshold in the overall population nor in the specific Caucasian subgroup (all *P* values > 0.14, [Figure 3](#F3)). In an exploratory analysis, we grouped Caucasian patients based on their inferred *CYP3A4* and *CYP3A5* phenotypes. We observed that increasing levels of predicted metabolizing capacity appeared to be associated with decreasing levels of dose-adjusted sildenafil concentrations (F test *P* = 0.0238; [Figure 4](#F4)). None of the pairwise comparisons between specific groups reached the pre-established threshold.

### Figure 2. Concentration:dose ratios of sildenafil according to CYP2C9 inferred phenotype in (A) all patients and (B) Caucasians.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/844d/5656562/c7c63631c085/nihms856822f2.jpg)

EM, extensive metabolisers; IM, intermediate metabolizers. Data presented as mean ± standard deviation.

### Figure 3. Concentration:dose ratios of sildenafil according to CYP3A5 inferred phenotype in (A) all patients and (B) Caucasians.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/844d/5656562/4b172b23e8b6/nihms856822f3.jpg)

EM, extensive metabolisers; IM, intermediate metabolizers; PM, poor metabolizers. Data presented as mean ± standard deviation.

### Figure 4. Concentration:dose ratios of sildenafil according to CYP3A4 and CYP3A5 inferred phenotype in Caucasians.

![Figure 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/844d/5656562/429ae8805e56/nihms856822f4.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5656562_nihms856822f4.jpg)

Increasing levels of predicted metabolizing capacity showed a trend with decreasing levels of dose-adjusted sildenafil concentrations (F test P = 0.0238). EM, extensive metabolisers; IM, intermediate metabolizers; PM, poor metabolizers. Data presented as mean ± standard deviation.

### Cyclic GMP

After adjusting for age, BMI, baseline cGMP and three PC of genetic ancestry, none of the primary genes were associated with cGMP at week 24.

### Secondary genes

None of the secondary genetic variants or inferred phenotypes reached our pre-established threshold after multiple comparisons (*P* = 0.001).

### Vascular adverse effects

Given the association between the *CYP3A4* inferred phenotype and the dose-adjusted concentrations of sildenafil, we explored the association of this genotype with the risk of hypotension and flushing. We observed no significant association, neither in the overall population (IM: 25% vs EM: 17%; *P* = 0.15), nor with the Caucasian subgroup (IM: 27% vs EM: 18%; *P* = 0.82).

## DISCUSSION

In the current study, we investigated the association between three candidate drug metabolism genes and peak dose-adjusted concentrations of sildenafil in HFpEF patients receiving sildenafil. To the best of our knowledge, this pharmacogenetic sub-study is the largest to have investigated the association between candidate genes and concentrations of sildenafil. Furthermore, we believe we are reporting for the first time that the inferred *CYP3A4* phenotype, which in this population corresponded to carriers of the *CYP3A4*22* variant, may be associated with dose–adjusted peak concentrations of sildenafil. While this association was only numerically apparent in our entire population, it reached our predetermined significance threshold in the Caucasian subgroup. Although we saw numerically higher dose-adjusted concentrations in *CYP3A5* PM, particularly in Caucasians, our results did not reach statistical significance. Moreover we found no indication that genetic variations in the *CYP2C9*-coding gene (*CYP2C9*), a minor elimination pathway of sildenafil, influenced sildenafil peak concentrations.

*CYP3A4* is the most abundant CYP in the liver.^26^ It contributes to the metabolism of approximately 30% of all drugs, including sildenafil,^13,26,27^ through oxidative biotransformation.^26^ Other substrates of *CYP3A4* include the immunosuppressant drug tacrolimus, the lipid-lowering drug atorvastatin and endoxifen, the principal active metabolite of the anti-neoplastic agent tamoxifen.^27,28^ Existing data suggest that sildenafil undergoes significant first-pass metabolism in the liver and that *CYP3A4* is the isoenzyme responsible for a majority of this metabolism,^12,13^ although *CYP3A5* may also contribute to the metabolism of sildenafil by the *CYP3A* subfamily.^16^ *CYP3A4* is also expressed in the gut wall and this pre-systemic metabolism is also thought to limit the bioavailability (40%) of sildenafil.^13^

The *CYP3A4*-coding gene, *CYP3A4*, is located in a cluster on chromosome 7 along with *CYP3A5*, *CYP3A7*, and *CYP3A43*^26,27^ Few genetic variants in *CYP3A4* have been shown to modulate *CYP3A4* activity. The intronic variant rs35599367 (C>T), which is located in intron 6 and which was allocated the **22* allele name by the Human Cytochrome P450 (CYP) Allele Nomenclature Database, is, to our knowledge, the only “common” (MAF > 0.01) SNP convincingly shown to influence *CYP3A4* expression.^27,29^ Existing data suggest that the hepatic *CYP3A4* mRNA level and enzyme activity are respectively 1.7- and 2.5-fold greater in individuals with the CC genotype than in carriers of the CT and TT carriers.^29^ *CYP3A4*22*’s MAF is reported to be of approximately 4% in Asians and African-Americans and 8% in Caucasians.^27^ The frequency in our Caucasian subgroup (7%) is thus consistent with existing data. Our observation that *CYP3A4*22* carriers present high dose-adjusted concentration of sildenafil is consistent with other reports showing that *CYP3A4*22* carriers present higher concentrations of *CYP3A4* substrates, including atorvastatin and tacrolimus.^30–32^

Despite the fact that we observed a significant association between *CYP3A4* and sildenafil concentrations, this did not translate into an association with its downstream biomarker cGMP. The hemodynamic effect of sildenafil and other PDE5 inhibitors in the treatment of erectile dysfunction and pulmonary arterial hypertension is secondary to the inhibition of cGMP degradation by PDE5, which leads to smooth muscle relaxation.^11,13^ Although this lack of association may appear surprising, it is consistent with the findings from the RELAX trial where sildenafil increased cGMP when compared to baseline but was not statistically significant compared to placebo.^15^ Moreover, the correlation between sildenafil peak concentrations and cGMP was weak in RELAX, perhaps reflecting that multiple other PDE are implicated in the metabolism of cGMP.^15^ Nevertheless, this lack of association underlines the uncertainty as to whether the observations in the current and other reports^30–32^ with *CYPA4*22* can be extended to the clinical response to drugs.

Given the high degree of similarity in DNA sequence between *CYP3A4* and *CYP3A5*, the known substrate overlap which exist between these isoenzymes,^26^ and existing data regarding the implication of *CYP3A5* in the metabolism of sildenafil,^16^ we hypothesised that *CYP3A5* could also influence the dose-adjusted peak sildenafil concentrations. In a similar fashion to a previous report in 21 healthy Korean males, we observed higher peak dose-adjusted concentrations in Caucasians participants who were *CYP3A5* PM in the present study.^2^ Nevertheless, as with that report, these trends were not statistically significant, which could be the result of our small sample size (see [supplementary material](#SD1)).^28^ Whether combining information from *CYP3A4* and *CYP3A5* further allows do decipher inter-individual differences in sildenafil’s pharmacokinetics requires additional investigation. If our preliminary observations are confirmed, given the apparent 2.5 fold difference in dose-adjusted concentrations between patients presenting with a CYP3A4 IM / CYP3A5 PM profile and those with a CYP3A4 EM / CYP3A5 IM profile, genotyping could be useful to personalise sildenafil dosing in the clinic.

We did not observe any association between sildenafil concentrations and the *CYP2C9* inferred phenotype, a minor route of sildenafil metabolism (20%).^11,12^ This is consistent with a prior study of 23 healthy males.^14^ Given the limited metabolism of sildenafil by *CYP2C9* but the know functional impact of *CYP2C9* variants on the pharmacokinetics of multiple drugs, including warfarin,^33^ we cannot exclude the possibility that *CYP2C9* could exert a modest impact on the pharmacokinetics of sildenafil. Demonstrating such an association would likely necessitate a much larger sample size than the one of the current investigation.

### Limitations

Our results should be interpreted while taking into consideration some inherent study limitations. First, our results are issued from the RELAX clinical trial which relied on three times per day dosing of sildenafil. The generalizability of our results to dosing schemes commonly used to treat erectile dysfunction requires further investigation.^13^ Nevertheless, the RELAX dosing regimen is similar to those investigated in pulmonary artery hypertension^11^ and thus, the potential clinical consequences of our findings may be of particular interest to investigate in these patients. Second, there was no measure or control for the types of food that may have been taken with sildenafil. Ingesting sildenafil with a high-fat meal is known to reduce its absorption rate and peak concentration,^12^ while foods that can modify *CYP3A4* activity such as grapefruit or pomelo juice could increase concentrations.^11,12^ Despite this, and the inherent heterogeneity in multicenter studies, we were able to identify a novel genetic association between the predicted *CYP3A4* phenotype, ultimately *CYP3A4*22*, and sildenafil dose-adjusted peak concentrations, the association would be expected to be even stronger in a study controlling for nutritional modifiers of the effect. Third, one could argue that because *CYP2C9* inhibitors were allowed as part of the study, this could potentially have contributed to our inability to detect genetic association with *CYP2C9* variants. As part of RELAX, restriction on *CYP2C9* inhibitors were not required given evidence suggesting that *CYP2C9* inhibitors exert no clinically significant impact on the pharmacokinetics of sildenafil.^11,12^ Consistent with this premise, the use of amiodarone, a known *CYP2C9* inhibitor,^26^ was not associated with our primary endpoint. A fourth limitation stems from the single concentration measure available at each visit, which limited our ability to conduct a more detailed pharmacokinetic study of sildenafil to assess other pharmacokinetic parameters of interest such as bioavailability, the elimination half-life or the area under the curve are influenced by *CYP3A4*. Fifth, despite constituting the largest pharmacogenetic study investigating the pharmacokinetics of sildenafil, our sample size was not sufficiently large to identify variants or inferred phenotypes with a more modest effect. Finally, it should be noted that there is a possibility that some patients may have carried rarer genetic variants that were not tested as part of our assay which could have contributed to misclassification of their metabolizer status. This may even be more likely for non-Caucasian patients.

In conclusion, we observed that the *CYP3A4* inferred phenotype, which reflected the *CYP3A4*22* carrier status, was associated with peak sildenafil dose adjusted concentrations in patients with HFpEF receiving high doses of sildenafil. Whether these results can be replicated in other populations for which sildenafil is currently indicated and influence its clinical response requires further investigation. Should this observation be validated, genotype-guided prescription of sildenafil in clinical practice or clinical trials could lead to a more predictable clinical response from this agent.

## Supplementary Material

## Acknowledgments

**Funding:** Supported by grant HL084904 from the National Heart, Lung, and Blood Institute.

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Kanjanawart S, Gaysonsiri D, Tangsucharit P, Vannaprasaht S, Phunikhom K, Kaewkamson T, et al. Comparative bioavailability of two sildenafil tablet formulations after single-dose administration in healthy Thai male volunteers. Int J Clin Pharmacol Ther. 2011;49:525–530. doi: 10.5414/cp201496.  [DOI](https://doi.org/10.5414/cp201496) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21781653/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Clin%20Pharmacol%20Ther&title=Comparative%20bioavailability%20of%20two%20sildenafil%20tablet%20formulations%20after%20single-dose%20administration%20in%20healthy%20Thai%20male%20volunteers&author=S%20Kanjanawart&author=D%20Gaysonsiri&author=P%20Tangsucharit&author=S%20Vannaprasaht&author=K%20Phunikhom&volume=49&publication_year=2011&pages=525-530&pmid=21781653&doi=10.5414/cp201496&)

2. Shon JH, Ku HY, Bae SY, Oh MK, Yeo CW, Bae SK, et al. The disposition of three phosphodiesterase type 5 inhibitors, vardenafil, sildenafil, and udenafil, is differently influenced by the CYP3A5 genotype. Pharmacogenet Genomics. 2011;21:820–828. doi: 10.1097/FPC.0b013e32834b79e6.  [DOI](https://doi.org/10.1097/FPC.0b013e32834b79e6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21934637/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=The%20disposition%20of%20three%20phosphodiesterase%20type%205%20inhibitors,%20vardenafil,%20sildenafil,%20and%20udenafil,%20is%20differently%20influenced%20by%20the%20CYP3A5%20genotype&author=JH%20Shon&author=HY%20Ku&author=SY%20Bae&author=MK%20Oh&author=CW%20Yeo&volume=21&publication_year=2011&pages=820-828&pmid=21934637&doi=10.1097/FPC.0b013e32834b79e6&)

3. Benard F, Carrier S, Lee JC, Talwar V, Defoy I. Men with mild erectile dysfunction benefit from sildenafil treatment. J Sex Med. 2010;7:3725–3735. doi: 10.1111/j.1743-6109.2010.02015.x.  [DOI](https://doi.org/10.1111/j.1743-6109.2010.02015.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20946161/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Sex%20Med&title=Men%20with%20mild%20erectile%20dysfunction%20benefit%20from%20sildenafil%20treatment&author=F%20Benard&author=S%20Carrier&author=JC%20Lee&author=V%20Talwar&author=I%20Defoy&volume=7&publication_year=2010&pages=3725-3735&pmid=20946161&doi=10.1111/j.1743-6109.2010.02015.x&)

4. Muniz JJ, Lacchini R, Rinaldi TO, Nobre YT, Cologna AJ, Martins AC, et al. Endothelial nitric oxide synthase genotypes and haplotypes modify the responses to sildenafil in patients with erectile dysfunction. Pharmacogenomics J. 2013;13:189–196. doi: 10.1038/tpj.2011.49.  [DOI](https://doi.org/10.1038/tpj.2011.49) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22064666/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=Endothelial%20nitric%20oxide%20synthase%20genotypes%20and%20haplotypes%20modify%20the%20responses%20to%20sildenafil%20in%20patients%20with%20erectile%20dysfunction&author=JJ%20Muniz&author=R%20Lacchini&author=TO%20Rinaldi&author=YT%20Nobre&author=AJ%20Cologna&volume=13&publication_year=2013&pages=189-196&pmid=22064666&doi=10.1038/tpj.2011.49&)

5. Eisenhardt A, Sperling H, Hauck E, Porst H, Stief C, Rubben H, et al. ACE gene I/D and NOS3 G894T polymorphisms and response to sildenafil in men with erectile dysfunction. Urology. 2003;62:152–157. doi: 10.1016/s0090-4295(03)00137-7.  [DOI](https://doi.org/10.1016/s0090-4295(03)00137-7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12837457/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Urology&title=ACE%20gene%20I/D%20and%20NOS3%20G894T%20polymorphisms%20and%20response%20to%20sildenafil%20in%20men%20with%20erectile%20dysfunction&author=A%20Eisenhardt&author=H%20Sperling&author=E%20Hauck&author=H%20Porst&author=C%20Stief&volume=62&publication_year=2003&pages=152-157&pmid=12837457&doi=10.1016/s0090-4295(03)00137-7&)

6. Peskircioglu L, Atac FB, Erdem SR, Deveci S, Verdi H, Ozkardes H. The association between intron 4 VNTR, E298A and IVF 23+10 G/T polymorphisms of ecNOS gene and sildenafil responsiveness in patients with erectile dysfunction. Int J Impot Res. 2007;19:149–153. doi: 10.1038/sj.ijir.3901501.  [DOI](https://doi.org/10.1038/sj.ijir.3901501) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16871271/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Impot%20Res&title=The%20association%20between%20intron%204%20VNTR,%20E298A%20and%20IVF%2023+10%20G/T%20polymorphisms%20of%20ecNOS%20gene%20and%20sildenafil%20responsiveness%20in%20patients%20with%20erectile%20dysfunction&author=L%20Peskircioglu&author=FB%20Atac&author=SR%20Erdem&author=S%20Deveci&author=H%20Verdi&volume=19&publication_year=2007&pages=149-153&pmid=16871271&doi=10.1038/sj.ijir.3901501&)

7. Lacchini R, Muniz JJ, Nobre YT, Cologna AJ, Martins AC, Tanus-Santos JE. VEGF genetic polymorphisms affect the responsiveness to sildenafil in clinical and postoperative erectile dysfunction. Pharmacogenomics J. 2013;13:437–442. doi: 10.1038/tpj.2012.39.  [DOI](https://doi.org/10.1038/tpj.2012.39) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23007311/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=VEGF%20genetic%20polymorphisms%20affect%20the%20responsiveness%20to%20sildenafil%20in%20clinical%20and%20postoperative%20erectile%20dysfunction&author=R%20Lacchini&author=JJ%20Muniz&author=YT%20Nobre&author=AJ%20Cologna&author=AC%20Martins&volume=13&publication_year=2013&pages=437-442&pmid=23007311&doi=10.1038/tpj.2012.39&)

8. Sekine A, Tanabe N, Sugiura T, Shigeta A, Jujo T, Nishimura R, et al. Polymorphism of the G Protein beta3 Subunit Gene Influences the Efficacy of Sildenafil in Patients with Pulmonary Hypertension. Int Med (Tokyo, Japan) 2014;53:291–297. doi: 10.2169/internalmedicine.53.0658.  [DOI](https://doi.org/10.2169/internalmedicine.53.0658) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24531084/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20Med%20(Tokyo,%20Japan)&title=Polymorphism%20of%20the%20G%20Protein%20beta3%20Subunit%20Gene%20Influences%20the%20Efficacy%20of%20Sildenafil%20in%20Patients%20with%20Pulmonary%20Hypertension&author=A%20Sekine&author=N%20Tanabe&author=T%20Sugiura&author=A%20Shigeta&author=T%20Jujo&volume=53&publication_year=2014&pages=291-297&pmid=24531084&doi=10.2169/internalmedicine.53.0658&)

9. Sperling H, Eisenhardt A, Virchow S, Hauck E, Lenk S, Porst H, et al. Sildenafil response is influenced by the G protein beta 3 subunit GNB3 C825T polymorphism: a pilot study. J Urol. 2003;169:1048–1051. doi: 10.1097/01.ju.0000058369.72348.ba.  [DOI](https://doi.org/10.1097/01.ju.0000058369.72348.ba) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12576843/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Urol&title=Sildenafil%20response%20is%20influenced%20by%20the%20G%20protein%20beta%203%20subunit%20GNB3%20C825T%20polymorphism:%20a%20pilot%20study&author=H%20Sperling&author=A%20Eisenhardt&author=S%20Virchow&author=E%20Hauck&author=S%20Lenk&volume=169&publication_year=2003&pages=1048-1051&pmid=12576843&doi=10.1097/01.ju.0000058369.72348.ba&)

10. Relling MV, Klein TE. CPIC: Clinical Pharmacogenetics Implementation Consortium of the Pharmacogenomics Research Network. Clin Pharmacol Ther. 2011;89:464–467. doi: 10.1038/clpt.2010.279.  [DOI](https://doi.org/10.1038/clpt.2010.279) | [PMC free article](/articles/PMC3098762/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21270786/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=CPIC:%20Clinical%20Pharmacogenetics%20Implementation%20Consortium%20of%20the%20Pharmacogenomics%20Research%20Network&author=MV%20Relling&author=TE%20Klein&volume=89&publication_year=2011&pages=464-467&pmid=21270786&doi=10.1038/clpt.2010.279&)

11. Chaumais MC, Perrin S, Sitbon O, Simonneau G, Humbert M, Montani D. Pharmacokinetic evaluation of sildenafil as a pulmonary hypertension treatment. Expert Opinion Drug Metabolism Toxicol. 2013;9:1193–1205. doi: 10.1517/17425255.2013.804063.  [DOI](https://doi.org/10.1517/17425255.2013.804063) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23944387/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Expert%20Opinion%20Drug%20Metabolism%20Toxicol&title=Pharmacokinetic%20evaluation%20of%20sildenafil%20as%20a%20pulmonary%20hypertension%20treatment&author=MC%20Chaumais&author=S%20Perrin&author=O%20Sitbon&author=G%20Simonneau&author=M%20Humbert&volume=9&publication_year=2013&pages=1193-1205&pmid=23944387&doi=10.1517/17425255.2013.804063&)

12. Gupta M, Kovar A, Meibohm B. The clinical pharmacokinetics of phosphodiesterase-5 inhibitors for erectile dysfunction. J Clin Pharmacol. 2005;45:987–1003. doi: 10.1177/0091270005276847.  [DOI](https://doi.org/10.1177/0091270005276847) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16100293/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Pharmacol&title=The%20clinical%20pharmacokinetics%20of%20phosphodiesterase-5%20inhibitors%20for%20erectile%20dysfunction&author=M%20Gupta&author=A%20Kovar&author=B%20Meibohm&volume=45&publication_year=2005&pages=987-1003&pmid=16100293&doi=10.1177/0091270005276847&)

13. Dorsey P, Keel C, Klavens M, Hellstrom WJ. Phosphodiesterase type 5 (PDE5) inhibitors for the treatment of erectile dysfunction. Expert Opinion Pharmacotherapy. 2010;11:1109–1122. doi: 10.1517/14656561003698131.  [DOI](https://doi.org/10.1517/14656561003698131) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20402554/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Expert%20Opinion%20Pharmacotherapy&title=Phosphodiesterase%20type%205%20(PDE5)%20inhibitors%20for%20the%20treatment%20of%20erectile%20dysfunction&author=P%20Dorsey&author=C%20Keel&author=M%20Klavens&author=WJ%20Hellstrom&volume=11&publication_year=2010&pages=1109-1122&pmid=20402554&doi=10.1517/14656561003698131&)

14. Jetter A, Lazar A, Schomig E, Fuhr U, Kinzig-Schippers M, Sorgel F. The CYP2C9 genotype does not influence sildenafil pharmacokinetics in healthy volunteers. Clin Pharmacol Ther. 2005;78:441–443. doi: 10.1016/j.clpt.2005.08.005.  [DOI](https://doi.org/10.1016/j.clpt.2005.08.005) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16198665/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=The%20CYP2C9%20genotype%20does%20not%20influence%20sildenafil%20pharmacokinetics%20in%20healthy%20volunteers&author=A%20Jetter&author=A%20Lazar&author=E%20Schomig&author=U%20Fuhr&author=M%20Kinzig-Schippers&volume=78&publication_year=2005&pages=441-443&pmid=16198665&doi=10.1016/j.clpt.2005.08.005&)

15. Redfield MM, Chen HH, Borlaug BA, Semigran MJ, Lee KL, Lewis G, et al. Effect of phosphodiesterase-5 inhibition on exercise capacity and clinical status in heart failure with preserved ejection fraction: a randomized clinical trial. JAMA. 2013;309:1268–1277. doi: 10.1001/jama.2013.2024.  [DOI](https://doi.org/10.1001/jama.2013.2024) | [PMC free article](/articles/PMC3835156/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23478662/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA&title=Effect%20of%20phosphodiesterase-5%20inhibition%20on%20exercise%20capacity%20and%20clinical%20status%20in%20heart%20failure%20with%20preserved%20ejection%20fraction:%20a%20randomized%20clinical%20trial&author=MM%20Redfield&author=HH%20Chen&author=BA%20Borlaug&author=MJ%20Semigran&author=KL%20Lee&volume=309&publication_year=2013&pages=1268-1277&pmid=23478662&doi=10.1001/jama.2013.2024&)

16. Ku HY, Ahn HJ, Seo KA, Kim H, Oh M, Bae SK, et al. The contributions of cytochromes P450 3A4 and 3A5 to the metabolism of the phosphodiesterase type 5 inhibitors sildenafil, udenafil, and vardenafil. Drug Metabolism Disposition: Biological Fate Chemicals. 2008;36:986–990. doi: 10.1124/dmd.107.020099.  [DOI](https://doi.org/10.1124/dmd.107.020099) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18308836/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metabolism%20Disposition:%20Biological%20Fate%20Chemicals&title=The%20contributions%20of%20cytochromes%20P450%203A4%20and%203A5%20to%20the%20metabolism%20of%20the%20phosphodiesterase%20type%205%20inhibitors%20sildenafil,%20udenafil,%20and%20vardenafil&author=HY%20Ku&author=HJ%20Ahn&author=KA%20Seo&author=H%20Kim&author=M%20Oh&volume=36&publication_year=2008&pages=986-990&pmid=18308836&doi=10.1124/dmd.107.020099&)

17. Redfield MM, Borlaug BA, Lewis GD, Mohammed SF, Semigran MJ, Lewinter MM, et al. PhosphdiesteRasE-5 Inhibition to Improve CLinical Status and EXercise Capacity in Diastolic Heart Failure (RELAX) trial: rationale and design. Circ Heart Fail. 2012;5:653–659. doi: 10.1161/CIRCHEARTFAILURE.112.969071.  [DOI](https://doi.org/10.1161/CIRCHEARTFAILURE.112.969071) | [PMC free article](/articles/PMC3530955/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22991405/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circ%20Heart%20Fail&title=PhosphdiesteRasE-5%20Inhibition%20to%20Improve%20CLinical%20Status%20and%20EXercise%20Capacity%20in%20Diastolic%20Heart%20Failure%20(RELAX)%20trial:%20rationale%20and%20design&author=MM%20Redfield&author=BA%20Borlaug&author=GD%20Lewis&author=SF%20Mohammed&author=MJ%20Semigran&volume=5&publication_year=2012&pages=653-659&pmid=22991405&doi=10.1161/CIRCHEARTFAILURE.112.969071&)

18. de Denus S, Rouleau JL, Mann DL, Huggins GS, Cappola TP, Shah SH, et al. A pharmacogenetic investigation of intravenous furosemide in decompensated heart failure: a meta-analysis of three clinical trials. Pharmacogenomics J. 2016 Mar 1; doi: 10.1038/tpj.2016.4. [Epub ahead of print]  [DOI](https://doi.org/10.1038/tpj.2016.4) | [PMC free article](/articles/PMC5009007/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26927285/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=A%20pharmacogenetic%20investigation%20of%20intravenous%20furosemide%20in%20decompensated%20heart%20failure:%20a%20meta-analysis%20of%20three%20clinical%20trials&author=S%20de%20Denus&author=JL%20Rouleau&author=DL%20Mann&author=GS%20Huggins&author=TP%20Cappola&publication_year=2016&pmid=26927285&doi=10.1038/tpj.2016.4&)

19. Zheng H, Zeevi A, Schuetz E, Lamba J, McCurry K, Griffith BP, et al. Tacrolimus dosing in adult lung transplant patients is related to cytochrome P4503A5 gene polymorphism. J Clin Pharmacol. 2004;44:135–140. doi: 10.1177/0091270003262108.  [DOI](https://doi.org/10.1177/0091270003262108) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14747421/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Pharmacol&title=Tacrolimus%20dosing%20in%20adult%20lung%20transplant%20patients%20is%20related%20to%20cytochrome%20P4503A5%20gene%20polymorphism&author=H%20Zheng&author=A%20Zeevi&author=E%20Schuetz&author=J%20Lamba&author=K%20McCurry&volume=44&publication_year=2004&pages=135-140&pmid=14747421&doi=10.1177/0091270003262108&)

20. Wehland M, Bauer S, Brakemeier S, Burgwinkel P, Glander P, Kreutz R, et al. Differential impact of the CYP3A5*1 and CYP3A5*3 alleles on pre-dose concentrations of two tacrolimus formulations. Pharmacogenet Genomics. 2011;21:179–184. doi: 10.1097/FPC.0b013e32833ea085.  [DOI](https://doi.org/10.1097/FPC.0b013e32833ea085) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20818295/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Differential%20impact%20of%20the%20CYP3A5*1%20and%20CYP3A5*3%20alleles%20on%20pre-dose%20concentrations%20of%20two%20tacrolimus%20formulations&author=M%20Wehland&author=S%20Bauer&author=S%20Brakemeier&author=P%20Burgwinkel&author=P%20Glander&volume=21&publication_year=2011&pages=179-184&pmid=20818295&doi=10.1097/FPC.0b013e32833ea085&)

21. de Denus S, Zakrzewski M, Barhdadi A, Leblanc MH, Racine N, Belanger F, et al. Association between renal function and CYP3A5 genotype in heart transplant recipients treated with calcineurin inhibitors. J Heart Lung Transplant. 2011;30:326–331. doi: 10.1016/j.healun.2010.09.015.  [DOI](https://doi.org/10.1016/j.healun.2010.09.015) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21094057/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Heart%20Lung%20Transplant&title=Association%20between%20renal%20function%20and%20CYP3A5%20genotype%20in%20heart%20transplant%20recipients%20treated%20with%20calcineurin%20inhibitors&author=S%20de%20Denus&author=M%20Zakrzewski&author=A%20Barhdadi&author=MH%20Leblanc&author=N%20Racine&volume=30&publication_year=2011&pages=326-331&pmid=21094057&doi=10.1016/j.healun.2010.09.015&)

22. de Denus S, Zakrzewski M, Barhdadi A, Leblanc MH, Racine N, Belanger F, et al. CYP3A5*1/*3 genetic polymorphism is associated with post cardiac transplant renal dysfunction in patients treated with calcineurin inhibitors. J Cardiac Fail. 2008;14:S3–S3.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Cardiac%20Fail&title=CYP3A5*1/*3%20genetic%20polymorphism%20is%20associated%20with%20post%20cardiac%20transplant%20renal%20dysfunction%20in%20patients%20treated%20with%20calcineurin%20inhibitors&author=S%20de%20Denus&author=M%20Zakrzewski&author=A%20Barhdadi&author=MH%20Leblanc&author=N%20Racine&volume=14&publication_year=2008&pages=S3-S3&)

23. Lachance K, Barhdadi A, Mongrain I, Normand V, Zakrzewski M, Leblanc MH, et al. PRKCB is associated with calcineurin inhibitor-induced renal dysfunction in heart transplant recipients. Pharmacogenet Genomics. 2012;5:336–343. doi: 10.1097/FPC.0b013e3283510a35.  [DOI](https://doi.org/10.1097/FPC.0b013e3283510a35) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22322241/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=PRKCB%20is%20associated%20with%20calcineurin%20inhibitor-induced%20renal%20dysfunction%20in%20heart%20transplant%20recipients&author=K%20Lachance&author=A%20Barhdadi&author=I%20Mongrain&author=V%20Normand&author=M%20Zakrzewski&volume=5&publication_year=2012&pages=336-343&pmid=22322241&doi=10.1097/FPC.0b013e3283510a35&)

24. Price AL, Patterson NJ, Plenge RM, Weinblatt ME, Shadick NA, Reich D. Principal components analysis corrects for stratification in genome-wide association studies. Nat Gen. 2006;38:904–909. doi: 10.1038/ng1847.  [DOI](https://doi.org/10.1038/ng1847) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16862161/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Gen&title=Principal%20components%20analysis%20corrects%20for%20stratification%20in%20genome-wide%20association%20studies&author=AL%20Price&author=NJ%20Patterson&author=RM%20Plenge&author=ME%20Weinblatt&author=NA%20Shadick&volume=38&publication_year=2006&pages=904-909&pmid=16862161&doi=10.1038/ng1847&)

25. Gao X, Starmer J, Martin ER. A multiple testing correction method for genetic association studies using correlated single nucleotide polymorphisms. Genet Epidemiol. 2008;32:361–369. doi: 10.1002/gepi.20310.  [DOI](https://doi.org/10.1002/gepi.20310) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18271029/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genet%20Epidemiol&title=A%20multiple%20testing%20correction%20method%20for%20genetic%20association%20studies%20using%20correlated%20single%20nucleotide%20polymorphisms&author=X%20Gao&author=J%20Starmer&author=ER%20Martin&volume=32&publication_year=2008&pages=361-369&pmid=18271029&doi=10.1002/gepi.20310&)

26. Zanger UM, Schwab M. Cytochrome P450 enzymes in drug metabolism: regulation of gene expression, enzyme activities, and impact of genetic variation. Pharmacol Ther. 2013;138:103–141. doi: 10.1016/j.pharmthera.2012.12.007.  [DOI](https://doi.org/10.1016/j.pharmthera.2012.12.007) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23333322/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacol%20Ther&title=Cytochrome%20P450%20enzymes%20in%20drug%20metabolism:%20regulation%20of%20gene%20expression,%20enzyme%20activities,%20and%20impact%20of%20genetic%20variation&author=UM%20Zanger&author=M%20Schwab&volume=138&publication_year=2013&pages=103-141&pmid=23333322&doi=10.1016/j.pharmthera.2012.12.007&)

27. Werk AN, Cascorbi I. Functional Gene Variants of CYP3A4. Clin Pharmacol Ther. 2014;96:340–348. doi: 10.1038/clpt.2014.129.  [DOI](https://doi.org/10.1038/clpt.2014.129) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24926778/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Functional%20Gene%20Variants%20of%20CYP3A4&author=AN%20Werk&author=I%20Cascorbi&volume=96&publication_year=2014&pages=340-348&pmid=24926778&doi=10.1038/clpt.2014.129&)

28. Binkhorst L, Mathijssen RHJ, Jager A, van Gelder T. Individualization of tamoxifen therapy: Much more than just CYP2D6 genotyping. Cancer Treatment Rev. 2015;41:289–299. doi: 10.1016/j.ctrv.2015.01.002.  [DOI](https://doi.org/10.1016/j.ctrv.2015.01.002) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25618289/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Treatment%20Rev&title=Individualization%20of%20tamoxifen%20therapy:%20Much%20more%20than%20just%20CYP2D6%20genotyping&author=L%20Binkhorst&author=RHJ%20Mathijssen&author=A%20Jager&author=T%20van%20Gelder&volume=41&publication_year=2015&pages=289-299&pmid=25618289&doi=10.1016/j.ctrv.2015.01.002&)

29. Wang D, Guo Y, Wrighton SA, Cooke GE, Sadee W. Intronic polymorphism in CYP3A4 affects hepatic expression and response to statin drugs. Pharmacogenomics J. 2011;11:274–286. doi: 10.1038/tpj.2010.28.  [DOI](https://doi.org/10.1038/tpj.2010.28) | [PMC free article](/articles/PMC3248744/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20386561/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=Intronic%20polymorphism%20in%20CYP3A4%20affects%20hepatic%20expression%20and%20response%20to%20statin%20drugs&author=D%20Wang&author=Y%20Guo&author=SA%20Wrighton&author=GE%20Cooke&author=W%20Sadee&volume=11&publication_year=2011&pages=274-286&pmid=20386561&doi=10.1038/tpj.2010.28&)

30. Klein K, Thomas M, Winter S, Nussler AK, Niemi M, Schwab M, et al. PPARA: a novel genetic determinant of CYP3A4 in vitro and in vivo. Clin Pharmacol Ther. 2012;91:1044–1052. doi: 10.1038/clpt.2011.336.  [DOI](https://doi.org/10.1038/clpt.2011.336) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22510778/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=PPARA:%20a%20novel%20genetic%20determinant%20of%20CYP3A4%20in%20vitro%20and%20in%20vivo&author=K%20Klein&author=M%20Thomas&author=S%20Winter&author=AK%20Nussler&author=M%20Niemi&volume=91&publication_year=2012&pages=1044-1052&pmid=22510778&doi=10.1038/clpt.2011.336&)

31. Pallet N, Jannot AS, El Bahri M, Etienne I, Buchler M, de Ligny BH, et al. Kidney transplant recipients carrying the CYP3A4*22 allelic variant have reduced tacrolimus clearance and often reach supratherapeutic tacrolimus concentrations. Am J Transplant. 2015;15:800–805. doi: 10.1111/ajt.13059.  [DOI](https://doi.org/10.1111/ajt.13059) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25588704/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Transplant&title=Kidney%20transplant%20recipients%20carrying%20the%20CYP3A4*22%20allelic%20variant%20have%20reduced%20tacrolimus%20clearance%20and%20often%20reach%20supratherapeutic%20tacrolimus%20concentrations&author=N%20Pallet&author=AS%20Jannot&author=M%20El%20Bahri&author=I%20Etienne&author=M%20Buchler&volume=15&publication_year=2015&pages=800-805&pmid=25588704&doi=10.1111/ajt.13059&)

32. Elens L, Capron A, van Schaik RH, De Meyer M, De Pauw L, Eddour DC, et al. Impact of CYP3A4*22 allele on tacrolimus pharmacokinetics in early period after renal transplantation: toward updated genotype-based dosage guidelines. Ther Drug Monit. 2013;35:608–616. doi: 10.1097/FTD.0b013e318296045b.  [DOI](https://doi.org/10.1097/FTD.0b013e318296045b) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24052064/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ther%20Drug%20Monit&title=Impact%20of%20CYP3A4*22%20allele%20on%20tacrolimus%20pharmacokinetics%20in%20early%20period%20after%20renal%20transplantation:%20toward%20updated%20genotype-based%20dosage%20guidelines&author=L%20Elens&author=A%20Capron&author=RH%20van%20Schaik&author=M%20De%20Meyer&author=L%20De%20Pauw&volume=35&publication_year=2013&pages=608-616&pmid=24052064&doi=10.1097/FTD.0b013e318296045b&)

33. Takeuchi F, McGinnis R, Bourgeois S, Barnes C, Eriksson N, Soranzo N, et al. A genome-wide association study confirms VKORC1, CYP2C9, and CYP4F2 as principal genetic determinants of warfarin dose. PLoS Genet. 2009;5:e1000433. doi: 10.1371/journal.pgen.1000433.  [DOI](https://doi.org/10.1371/journal.pgen.1000433) | [PMC free article](/articles/PMC2652833/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19300499/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet&title=A%20genome-wide%20association%20study%20confirms%20VKORC1,%20CYP2C9,%20and%20CYP4F2%20as%20principal%20genetic%20determinants%20of%20warfarin%20dose&author=F%20Takeuchi&author=R%20McGinnis&author=S%20Bourgeois&author=C%20Barnes&author=N%20Eriksson&volume=5&publication_year=2009&pages=e1000433&pmid=19300499&doi=10.1371/journal.pgen.1000433&)
