# Variants in the CYP2B6 3′UTR alter in vitro and in vivo CYP2B6 activity: potential role of microRNAs

## Metadata
**Authors:** Kimberly S Burgess, Joseph Ipe, Marelize Swart, Ingrid F Metzger, Jessica Lu, Brandon T Gufford, Nancy Thong, Zeruesenay Desta, Roger Gaedigk, Robin E Pearce, Andrea Gaedigk, Yunlong Liu, Todd C Skaar
**Journal:** Clinical pharmacology and therapeutics
**Date:** 2017 Oct 25
**DOI:** [10.1002/cpt.892](https://doi.org/10.1002/cpt.892)
**PMID:** 28960269
**PMCID:** PMC5871545
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5871545/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC5871545/pdf/nihms909412.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC5871545/pdf/nihms909412.pdf)

## Abstract

CYP2B6*6 and CYP2B6*18 are the most clinically important variants causing reduced CYP2B6 protein expression and activity. However, these variants do not account for all variability in CYP2B6 activity. Emerging evidence has shown that genetic variants in the 3′UTR may explain variable drug response by altering microRNA regulation. Five 3′UTR variants were associated with significantly altered efavirenz AUC0-48 (8-OH-EFV/EFV) ratios in healthy human volunteers. The rs70950385 (AG>CA) variant, predicted to create a microRNA binding site for miR-1275, was associated with 33% decreased CYP2B6 activity among normal metabolizers (AG/AG vs. CA/CA [p<0.05]). In vitro luciferase assays were used to confirm that the CA on the variant allele created a microRNA binding site causing a 11.3% decrease in activity compared to the AG allele when treated with miR-1275 (p=0.0035). Our results show that a 3′UTR variant contributes to variability in CYP2B6 activity.

Keywords: CYP, pharmacogenetics, efavirenz, bupropion, polymorphism, SNP, metabolism

## INTRODUCTION

*CYP2B6* is a highly polymorphic cytochrome P450. Of these variants, the *CYP2B6*6* (516G>T, Q172H and 785A>G, K262R) and *CYP2B6*18* (983T>C, I328T) haplotypes, have been shown to be the most clinically important by causing a reduction in protein expression and activity [[1](#R1)–[3](#R3)]. The *CYP2B6*6* haplotype has been associated with altered elimination and response of many clinically-relevant drugs, such as efavirenz, bupropion, methadone, and cyclophosphamide [[1](#R1), [4](#R4), [5](#R5)]; however, these and other frequently studied genetic variants account for only a portion of the variability observed in the biotransformation of drugs by CYP2B6.

Efavirenz, a non-nucleoside reverse transcriptase inhibitor, is prescribed as part of the World Health Organization recommended first-line combination antiretroviral therapy to treat HIV type 1 infection [[6](#R6)]. However, efavirenz has been associated with central nervous system adverse events such as hallucinations and insomnia and with treatment failure in some patients [[7](#R7)]. Higher efavirenz plasma concentrations have also been associated with increased likelihood of adverse CNS effects, whereas lower levels more likely result in treatment failure [[8](#R8)]. Efavirenz has also been shown to have wide variability in plasma concentrations among patients, some of which has been accounted for by genetic polymorphisms in *CYP2B6*, the enzyme primarily responsible for efavirenz biotransformation. Variability in CYP2B6-mediated biotransformation has been observed for other CYP2B6 substrates after the more well-studied polymorphisms have been accounted for, such as the variability in methadone metabolism and clearance [[9](#R9)].

Pharmacogenetics studies have focused primarily on genetic variants in the coding region of drug disposition genes. However, emerging evidence strongly suggests that genetic variants in the 3′UTR also contribute to variability in drug response by altering microRNA (miRNA) regulation [[10](#R10), [11](#R11)]. MicroRNAs, approximately 17–22 nucleotides in length, have been predicted to negatively regulate the expression of many genes including drug disposition genes, predominately through imperfect complimentary binding within the 3′UTR. Perfect binding of the seed sequence region, approximately seven nucleotides in length within the miRNA, has been the main focus for predicting miRNA regulation [[12](#R12)]. Variants within the seed sequence of miRNA binding sites may abolish miRNA binding sites, as well as create new miRNA binding sites [[13](#R13)]. These genetic variations in miRNA binding sites of CYPs may explain some of the inter-subject variability in pharmacokinetics and drug response.

In this study, we hypothesized that genetic variants in the 3′UTR of *CYP2B6* are associated with variability in efavirenz pharmacokinetics. To test this hypothesis, we identified genetic variants in the 3′UTR of *CYP2B6* of healthy volunteers that were administered efavirenz. Efavirenz has been shown to be a suitable probe drug substrate to measure CYP2B6 activity *in vitro* and *in vivo* [[14](#R14)]. AUC_0-48_ and C_max_ ratios (8-hydroxy-efavirenz/efavirenz) (8-OH-EFV/EFV) have been demonstrated to describe CYP2B6 activity over minor CYP pathways [[15](#R15), [16](#R16)]. Genetic variants that were associated with altered CYP2B6 activity were further characterized with *in vitro* functional assays to assess their impact on miRNA regulation of *CYP2B6* expression. Our results show that genetic variants in the 3′UTR of *CYP2B6* functionally alter miRNA regulation and are associated with variability in efavirenz pharmacokinetics.

## RESULTS

### Association of CYP2B6 3′UTR SNPs with efavirenz AUC0-48 metabolite:parent ratios in healthy volunteers

Among 200 healthy volunteers sequenced for *CYP2B6* 3′UTR variants, 197 were also genotyped for *CYP2B6*6* and *CYP2B6*18. CYP2B6*6 and CYP2B6*18* genotyping data were used to classify those 197 volunteers as genotypic normal (47%), intermediate (42%), or poor (11%) metabolizers ([Supplemental Table 1](#SD4)).

Twenty-five variants were observed in the 3′UTR of this population; however, only seven variants were present at a frequency that allowed us to assess pharmacokinetic differences among the genotypes ([Table 1](#T1)). These variants are not novel and have previously been reported in public databases such as The 1000 Genomes Project [[17](#R17)]. A schematic representation of the *CPY2B6* 3′UTR sequence and variant positions are provided in [Supplemental Figure 1](#SD1). Plasma efavirenz AUC_0-48_ ratios (8-OH-EFV/EFV) were available for 187 volunteers and were used to further describe the metabolic phenotype. The C_max_ ratios were in general agreement with the more robust AUC_0-48_ ratios and are only shown in the [supplemental materials](#SD8) ([Supplemental Table 2](#SD5) and [3](#SD6)).

### Table 1.

| SNP | Chromosome 19 Position1 | MAF | HWE | Genotype (n) |  |  |
| --- | --- | --- | --- | --- | --- | --- |
| rs3181842 | 41017111 | 0.343 | Yes | T/T (89) | T/C (85) | C/C (26) |
| rs7246465 | 41017398 | 0.323 | No | C/C (99) | C/T (73) | T/T (28) |
| rs28969420 | 41017899 | 0.050 | No | G/G (166) | G/T (24) | T/T (10) |
| rs1038376 | 41018104 | 0.333 | Yes | A/A (89) | A/T (89) | T/T (22) |
| rs707265 | 41018182 | 0.318 | Yes | G/G (101) | G/A (71) | A/A (28) |
| rs70950385 | 41018226 | 0.228 | Yes | AG/AG (122) | AG/CA (65) | CA/CA (13) |
| rs1042389 | 41018248 | 0.228 | Yes | C/C (122) | C/T (65) | T/T (13) |

Table 1 Caption: CYP2B6 3′UTR variant frequencies in healthy vol′unteers

Among the normal metabolizers, an increase in efavirenz metabolism was detected among 3′UTR variants rs3181842, rs7246465 and rs707265 ([Figure 1](#F1); [Supplemental Table 2](#SD5)). For rs3181842, an increase in CYP2B6 activity was observed between volunteers with T/T versus T/C (31.9% increase; *p*<0.05) and C/C genotypes (70.6% increase; *p*<0.0001) ([Figure 1A](#F1); [Supplemental Table 2](#SD5)). An increase in CYP2B6 activity was also seen among volunteers carrying the rs7246465 variant allele (C/C vs. C/T [38.0% increase; *p*<0.01] and T/T [67.9% increase; p<0.0001]) and the rs707265 variant allele (G/G vs. G/A [34.2% increase; *p*<0.05] and A/A [72.4% increase; p<0.0001]) ([Figure 1B, E](#F1); [Supplemental Table 2](#SD5)). These two variants were in partial linkage disequilibrium in our study cohort (r^2^=0.80, D′=0.91) ([Figure 2](#F2)).

### Figure 1. Association of CYP2B6 3′UTR variants with efavirenz metabolism in healthy volunteers with CYP2B6 genotypes predicting normal metabolism.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8f6e/5871545/184652ce9ae4/nihms909412f1.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5871545_nihms909412f1.jpg)

Efavirenz AUC metabolite:parent ratios (8-OH-EFV/EFV) among normal metabolizers; subjects with CYP2B6*6 and *18 alleles excluded. X-axis: CYP2B6 3′UTR genotypes F. rs70950385 variant is in complete LD with rs1042389 (data not shown separately). Y-axis: 8-hydroxy-efavirenz/efavirenz ratio. (*=p<0.05, **=p<0.01, ****=p<0.0001)

### Figure 2. r2 linkage disequilibrium diagram of CYP2B6 variants genotyped in samples from all four studies (3 clinical trials and one liver sample set).

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8f6e/5871545/d301697d4ee7/nihms909412f2.jpg)

The r2 values for CYP2B6 3′UTR variants and CYP2B6*6 (SNP 1 and 2) and CYP2B6*18 (SNP 3).

The rs70950385 and rs1042389 variant alleles were associated with a decrease in CYP2B6 activity when comparing AUC between AG/AG and CA/CA genotypes (32.7%; *p<*0.05) ([Figure 1F](#F1); [Supplemental Table 2](#SD5)). The rs70950385 and rs1042389 variants were in complete linkage disequilibrium (r^2^=1.0, D′=1.0) in our cohort and thus not shown separately ([Figure 2](#F2)). Comparable changes in efavirenz metabolism among the 3′UTR variants were also observed when normal, intermediate and poor metabolizers were combined ([Supplemental Figure 2](#SD2); [Supplemental Table 3](#SD6)).

To assess the changes in efavirenz metabolism among combinations of variants, haplotype and diplotype analyses were performed. Significant decreases in CYP2B6 activity, as measured by AUC ratios, were observed between haplotype 4 versus 2 (*p*<0.01), 12 (*p*<0.0001), 13 (*p*<05), and 24 (*p*<0.01), respectively ([Table 2](#T2)). Haplotypes 2, 4, 12, 13, and 24 varied among the following 3′UTR variants: rs3181842, rs7260525, rs7246465, rs28969420, rs1038376, rs707265, rs70950385, and rs1042389. A significant overall change was also observed in AUC ratios between the different diplotypes, but no significant changes were detected after post-hoc analysis (data not shown). The sample size for many haplotypes and diplotypes were too small to assess, i.e. many haplotypes and diplotypes were observed only within one of the metabolizer groups (data not shown).

### Table 2.

| Haplotype ID | Haplotype** | Count | Mean AUC0-48 (8-OH-EFV/EFV) | Mean OH-BUP Rates (pmol/mg protein/min) |
| --- | --- | --- | --- | --- |
| 1 | G7CCTACACAAGGACCGCCGAGAGTA | 7 | 0.33±0.19 | 265.7±167.6 |
| 2* | G7CCTACACAAGGACCGCCGTGAGTA | 25 | 0.33±0.03* | 453.5±375.3 |
| 4* | G7CCTACACAAGGATCGCCGAAAGTA | 128 | 0.55±0.03* | 572.0±135.8 |
| 9 | G7CTTACACAAGGACCGCCGAGAGTA | 62 | 0.40±0.03 | 480.8±104.4 |
| 12* | G7CTTACACAAGGACCGCCGTGAGTA | 128 | 0.38±0.02* | 512.1±106.1 |
| 13* | G7CTTACACAAGGACCTCCGAGCACA | 50 | 0.39±0.03* | 153.3±61.7 |
| 24* | G7CTTACACAAGGGCCGCCGAGCACA | 65 | 0.40±0.03* | 403.9±137.6 |
| 34 | G7CTAGCACAAGGACCGCCGTGAGTA | 22 | 0.37±0.05 | 342.4±168.6 |
| 37 | G8CCTACACAAGGATCGCCGAAAGTA | 32 | 0.51±0.05 | 444.8±162.7 |

Table 2 Caption: CYP2B6 3′UTR variant haplotype association with CYP2B6 activity in healthy volunteers and human liver microsomes

### 3′UTR SNPs association with altered CYP2B6 activity in human liver microsomes

To further evaluate the changes found in our healthy volunteer population, DNA from 90 human liver microsomal preparations isolated from pediatric and adult liver tissue samples were sequenced and characterized for CYP2B6 activity by measuring bupropion hydroxylation (one sample had an undetectable bupropion hydroxylation rate). In this tissue panel, there were 51 genotypic normal, 31 intermediate, 4 poor metabolizers, and 4 liver samples whose *CYP2B6*6* and *CYP2B6*18* genotypes were unknown (data not shown). A similar decrease in CYP2B6 activity was observed among samples with the rs70950385 homozygous (AG/AG) genotype (543.3 pmol/mg protein/min) versus carriers of the CA allele (326.3 pmol/mg protein/min) in the panel (39.9% decrease; *p*<0.05 based on untransformed data) ([Figure 3](#F3) – log_2_-transformed velocities for visualization purposes; [Supplemental Table 4](#SD7)). Differences were not significant in genotypic normal metabolizers, likely because there were too few samples in this subset. The rs70950385 and rs1042389 variants were in complete linkage disequilibrium in this sample population (r^2^=1.0, D=1.0) ([Figure 2](#F2)). There were no significant changes for any of the other variants, haplotypes, and diplotypes, as their frequencies were low ([Supplemental Figure 3](#SD3) and [Table 2](#T2)).

### Figure 3. Association of CYP2B6 rs70950385 with bupropion metabolism in human liver microsomes.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8f6e/5871545/7e8333346245/nihms909412f3.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5871545_nihms909412f3.jpg)

Bupropion hydroxylation activity observed in human liver microsomes among A. all samples and B. normal metabolizers. X-axis: Genotype (rs70950385 in complete linkage disequilibrium with rs1042389). Y-axis: log2 transformed rate of hydroxyl-bupropion (pmol/mg protein/min). (*=p<0.05) Note: Data was log-transformed for visual purposes only.

### miR-1275 alters luciferase activity in a plasmid containing the rs70950385 (CA) variant allele

The rs70950385 variant (AG>CA), a combination of rs12979270 (A>C) and rs12979898 (G>A), was found to be in complete linkage disequilibrium in our total population. The rs12979270 (A>C) was predicted by PolymiRTS to create miRNA binding site miR-625-5p ([Figure 4A](#F4)) through perfect complimentary alignment within the seed sequence region, nucleotides 2–8 from the 5′ end of the miRNA. Manual seed sequence comparisons revealed the dinucleotide variant created an additional base pair match for the first nucleotide of miR-1275 ([Figure 4B](#F4)). When this mechanism was tested in a luciferase transfection model in HepG2 cells without co-transfection of predicted miRNAs, there were no differences in luciferase activity between the plasmid containing the wild-type and variant miRNA binding sites with *C. elegans* negative control miRNA or controls without miRNA transfected ([Figure 5A](#F5)). However, when transfected with the miRNAs whose binding was predicted to be altered by the variant, miR-1275 reduced luciferase activity by 11.3% (*p*=0.0035) in the presence of the variant allele compared to the wild-type allele ([Figure 5B](#F5)). MicroRNA 625-5p did not impact luciferase activity.

### Figure 4. rs70950385 creates miR-625-5p and miR-1275 binding site in CYP2B6 3′UTR.

![Figure 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8f6e/5871545/cbc3ac27433f/nihms909412f4.jpg)

Proposed mechanism for miRNA A. 625-5p and B. 1275 binding to the CYP2B6 3′UTR among carriers of the rs70950385 (CA) allele.

### Figure 5. miRNA regulation of CYP2B6 3′UTR in vitro.

![Figure 5](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8f6e/5871545/bd44d2e353af/nihms909412f5.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5871545_nihms909412f5.jpg)

MicroRNA regulation of CYP2B6 expression through measurement of luciferase activity in pIS-0 plasmids containing either the reference (wild-type) or variant miRNA binding sites. A. X-axis: Treatment groups (none, C. elegans control, miR-1275, miR-625). Y-axis: Wild-type (firefly/Renilla luciferase) / Variant (firefly/Renilla luciferase). A ratio of 1=no change, >1=wild-type had higher luciferase activity, <1=variant had higher luciferase activity. B. X-axis: Wild-type and variant plasmids plus miR-1275. Y-axis: Firefly/Renilla luciferase ratio. (*=p<0.05)

## DISCUSSION

Variability in the pharmacokinetics of efavirenz and other drugs that undergo CYP2B6-mediated metabolism has been attributed to genetic variants in the coding region; however, variability in clinical responses and adverse events are not completely explained by these genetic differences. Data from this study indicate that genetic variations in the 3′UTR of *CYP2B6* have an effect on CYP2B6 activity, independent of CYP2B6 genotypic metabolizer status (assigned based on *CYP2B6*6* and **18*). There were 25 unique variants detected in the 3′UTR of *CYP2B6* in our study population. Of these variants, seven were observed at a frequency that allowed us to assess differences in efavirenz pharmacokinetics between the homozygous and heterozygous volunteers. The rs3181842, rs7246465, and rs707265 had similar increases in CYP2B6 activity to one another for homozygous volunteers ([Figure 1](#F1), panels A, B, and E). These three variants were not predicted to remove or create a miRNA binding site through the PolymiRTS database at the time the database was accessed. Although the mechanism behind these observed changes are unknown, the mechanism may be related to only one of the variants as there is partial linkage disequilibrium among the three variants. The rs3181842 vs. rs7246465, rs3181842 vs. rs707265, and rs7246465 vs. rs707265 have r^2^= 0.58, 0.58, and 0.80 values respectively ([Figure 2](#F2)).

The rs70950385 and rs1042389 variants were shown to be in complete LD in both our volunteers and human liver microsomes. These two variants were associated with decreased CYP2B6 activity with respect to efavirenz pharmacokinetics in subjects genotyped as normal metabolizers. They were also associated with decreased rates of bupropion hydroxylation in the panel of human liver microsomes. The rs1042389 variant has previously been associated with a decrease in median efavirenz plasma concentrations among black South African HIV/AIDS patients in a study that assessed the effect of 3′UTR variants on efavirenz response [[10](#R10)]. The rs70950385 and rs1042389 SNPs were shown to be in complete LD in this population as well. We propose that some of the observed decrease in CYP2B6 activity is mediated by miRNA regulation owing to the rs70950385 variant creating a miRNA binding site for miR-1275. This hypothesis is further corroborated by our *in vitro* data ([Figure 5](#F5)). In HepG2 cells treated with miR-1275 that expressed the variant (CA) miRNA binding site, reduced luciferase activity was observed compared to cells that expressed the reference (wild-type) (AG) miRNA binding site. Previously, we showed that miR-1275 expression is increased in pediatric and adult human livers compared to fetal livers, unequivocally demonstrating that this miRNA is expressed in the human liver [[18](#R18)]. These data suggest that this miRNA may play a role in regulating *CYP2B6* activity; however, further studies are warranted to validate that this is the underlying mechanism as opposed to other mechanisms, such as variant-induced structural changes in the mRNA that may affect its stability.

No significant changes in efavirenz pharmacokinetics were observed among the 3′UTR variants within the genetic intermediate and poor metabolizers in our human volunteer population for any variant with the exception of rs7246465 (C/T vs. T/T; *p*<0.05) (data not shown). All other variants yielded similar trends. This is likely due to inadequate sample numbers among the intermediate and poor metabolizer groups to detect these changes. Alternatively, this observation may be due to a limited effect size of the 3′UTR variants in the presence of coding region variants known to reduce CYP2B6 activity. The rs70950385 homozygous variant and other genotypes were only detected in normal metabolizers in our population. Interestingly, many 3′UTR haplotypes and diplotypes were unique to the metabolizer status. This may be due to partial LD with the *CYP2B6*6* and *CYP2B6*18* alleles or that our sample size was too small to detect an overlap.

This study shows that 3′UTR variants impact drug metabolism. Although the mechanisms are not known for all altered efavirenz pharmacokinetics associated with 3′UTR variants in *CYP2B6*, miRNA regulation is a likely contributor. Limitations on linking the other variants to miRNAs may be due to the limitations of miRNA prediction algorithms and tools which focus mostly on the seed sequence region as the main mechanism for miRNA binding. As in the case of rs70950385, our data suggest that miRNA regulation may explain, at least in part, the decreased CYP2B6 activity in healthy volunteers carrying the variant allele, as corroborated by our *in vitro* data. Although pharmacogenetics has primarily focused on genetic variants in the coding and promoter regions, our findings, i.e. that genetic variants in the 3′UTR contribute to variability in drug response, will open up the field to consider 3′UTR variants as a source of variability in the activities of many drug disposition genes.

## MATERIALS AND METHODS

### CYP2B6 Activity in Healthy Volunteers

Data from 200 healthy non-pregnant female (n=83) and male (n=117) volunteers (18–55 years old) administered a single of efavirenz dose, a probe drug substrate of CYP2B6 activity, was obtained from the single dose phase of three previous studies, published and unpublished [[19](#R19), [20](#R20)]. These studies were approved by the Indiana University School of Medicine Institutional Review Board, conducted at the Indiana University School of Medicine Clinical Research Center and registered at [http://www.clinicaltrials.gov](http://www.clinicaltrials.gov) (trial identifiers [NCT00668395](https://clinicaltrials.gov/ct2/show/NCT00668395), [NCT01104376](https://clinicaltrials.gov/ct2/show/NCT01104376), and [NCT02401256](https://clinicaltrials.gov/ct2/show/NCT02401256)). After written informed consent, volunteers were thoroughly screened using medical history, physical examination, and laboratory tests such as electrocardiography, HIV test, urinalysis and blood tests. During this screen, venous blood was drawn for DNA isolation. Dietary restrictions and inclusion and exclusion criteria have been previously reported [[21](#R21), [22](#R22)].

These open label studies were retrospectively used to evaluate the association of single dose efavirenz pharmacokinetics with the *CYP2B6* 3′UTR variants. Briefly, plasma samples were taken at pre-dose and 0.5, 1, 1.5, 2, 2.5, 3, 4, 6, 8, 10, 12, 16, 24, 48, 72 and 144 hours after single dose efavirenz administration. Participants from two studies (127 participants total) were administered 600mg of efavirenz and 73 participants from another study were administered 100mg of efavirenz (Sustiva, Bristol-Myers Squibb). Plasma efavirenz and 8-hydroxy-efavirenz concentrations were measured using a validated LC-MS/MS protocol as described previously [[19](#R19)]. Non-compartmental analysis of data was performed using Phoenix^®^ WinNonlin^®^ (version 7.0, Pharsight Corp., Cary, NC) to determine C_max_ (maximal plasma concentration) and AUC_0-48_ (area under the plasma concentration time curve from zero to 48 hours). C_max_ was assessed directly from the concentration-time profile and AUC_0-48_ was determined using the trapezoidal rule with linear up/log down interpolation.

### CYP2B6 Activity in Human Liver Microsomes (HLMs)

HLMs were prepared by differential centrifugation, as described by Lu and Levin (1972) [[23](#R23)]. *In vitro* microsomal incubations were performed according to Pearce et al 2015 [[24](#R24)]. Briefly, HLMs (30μg of microsomal protein), potassium phosphate buffer (50mM, pH 7.4), MgCl2 (3mM), EDTA (1mM), and bupropion (500μM) were incubated at the final concentrations listed. Reactions were initiated by the addition of a NADPH-generating system, consisting of NADP (1mM), glucose-6-phosphate (5mM), and glucose-6-phosphate dehydrogenase (1 U/ml), incubated at 37°C in a Thermo Forma Benchtop Orbital Shaker (Marietta, OH) and terminated after 30 minutes by the addition of 50μL of ice-cold acetonitrile. Incubations were performed in triplicate. Bupropion stock solutions were dissolved in methanol; however, the concentration of methanol present in the incubation mixtures did not exceed 0.25%. Hydroxybupropion and bupropion were resolved by isocratic, reversed-phase high-performance liquid chromatography based on a modification of the method by Faucette et al. 2000 [[25](#R25)] as described previously [[24](#R24)].

### Genotyping and Sequencing in Healthy Volunteers

Genomic DNA extracted from whole blood was used for genotyping and Sanger sequencing of the 3′UTR. *CYP2B6* genotyping was performed using TaqMan Genotyping Assays for rs3745274 (516G>T, Q172H) and rs28399499 (983T>C, I328T) (Life Technologies, Foster City, CA) according to the protocol provided. PCR was performed on BioRad iCycler and QuantStudio 12K Flex real-time PCR instruments. *CYP2B6* rs2279343 (785A>G, K262R) genotyping was performed by first amplifying exon 5 with primers 5′-CTCTCTCCCTGTGACCTGCTA-3′ (forward) and 5′-CTCCCTCTGTCTTTCATTCTGTC-3′ (reverse) (Integrated DNA Technologies, Coralville, IA) as described by Lang et al. [[26](#R26)], then using 1uL exon 5 PCR amplification product as a template for a custom TaqMan Genotyping Assay as previously described [[21](#R21)].

To sequence the 3′UTR region, this region was first amplified by adding the following reagents to each reaction: 1μL of DNA (10 ng), 12μL CloneAmp HiFi PCR Premix (CloneTech, Mountain View, CA), 10μL ultrapure distilled water and 1μL of each primer (final concentration: [0.4μM]) (Integrated DNA Technologies, Coralville, IA). The primer sequences used were 5′-GGCAAAATACCCCCAACATA-3′ (forward) and 5′-AGAGTTGGCATTGAGGTGAGAG-3′ (reverse) [[10](#R10)]. The PCR amplification conditions were 98°C for 5 minutes, 40 cycles at 98°C for 10 seconds, 69.5°C for 15 seconds, and 72°C for 2 minutes. Final extension was at 72°C for 7 minutes. PCR purification was performed using a QIAquick PCR Purification Kit (Qiagen, Valencia, CA) according to the user manual. DNA concentrations were measured with a Qubit dsDNA BR Assay Kit (Invitrogen, Eugene, OR) according to the user manual. Samples were prepared for sequencing using 10μL of DNA (50 ng) and 2μL of one of four different primers [1.7μM] and Sanger sequenced by ACGT (Germantown, MD). The 1.8 kb PCR product was sequenced in its entirety using the primers mentioned above and forward and reverse primers 5′-CTAAGCCTTGCTCTGTCTCC-3′ (forward) and 5′-GGAGAATCACTTGAACCCAGG-3′ (reverse).

### Genotyping and Sequencing in Human Liver Tissue Samples

Genomic DNA extracted from 90 human liver samples was used for genotyping and Sanger sequencing. Genotyping was performed on Applied Biosystems 7900 HT Real-Time PCR System or QuantStudio 12K Flex real-time PCR instruments using Life Technologies TaqMan assays for *CYP2B6* rs34223104 (282T>C); rs3745274 (516G>T, Q172H), rs28399499 (983T>C, I328T), and rs3211371 (1459C>T, R487C) (Life Technologies, Foster City, CA). To detect *CYP2B6* rs2279343 (785A>G, K262R) a high-resolution melting (HRM) assay was performed as previously described [[27](#R27)].

To sequence the 3′UTR, this region was first amplified by adding the following reagents to each reaction: 1μL of DNA (15 ng), 4μL KAPA LR Hotstart 2X, 2.4μL ultrapure distilled water and 0.3μL of each primer (final concentration: [0.375μM]) (Integrated DNA Technologies, Coralville, IA). The primer sequences used were 5′-AATCTGTTGCAGTGGACATTTG-3′ (forward) and 5′-AGAGTTGGCATTGAGGTGAGA-3′ (reverse). The PCR amplification conditions were 94°C for 3 minutes, 48 cycles at 94°C for 10 seconds, 61.5°C for 15 seconds, and 68°C for 2 minutes. Final extension was 68°C for 7 minutes. PCR purification was performed using ExoSAP-IT (USB, Cleveland, OH). A 1:20 dilution of the purified PCR product was used for sequencing, performed with BigDye terminator v3.1 and the STeP cycle sequencing protocol [[28](#R28)]. Briefly, three cycle steps were performed: 1) at 95°C for 10 sec, 50°C for 5 sec and 60°C for 1 min 15 sec for a total 15 cycles, 2) at 95°C for 10 sec, 50°C for 5 sec and 60°C for 1 min 30 sec, a total of 5 cycles and 3) at 95°C for 10 sec, 50°C for 5 sec and 60°C for 2 min for 15 cycles. The sequence reactions were cleaned up with AxyPrep Mag DyeClean (Axygen, Corning Life Sciences, Tewksbury, MA) and electrophoresis was performed on a 3730xl DNA Analyzer (Life Technologies, Foster City, CA). The 1962 bp PCR product was sequenced in its entirety using the following forward primers 5′-TGTGTCTGGGCTTAGGGAC-3′, 5′-GTGATTCACCCACCTTAGCC-3′, 5′-GTCTCAGCTCCCAAGTAGCTG-3′, and the reverse primers 5′-CAGGTCTCTCAGAGGCAGG-3′, 5′-AGTTCGAGACCAGCCTGG-3′, 5′-CCAGCTCCTCCAGAGGC-3′ and 5′-GCAGCTGGGAATGAGGA-3′.

As a quality control, genotyping was repeated in random samples and genotyping and sequencing results were verified by a second person to ensure accurate interpretation of the data. Sequences were analyzed using Sequencher (v.5.4.5, Gene Codes Corporation, Ann Arbor, MI) and BioEdit (v.7.2.5)[[29](#R29)] after alignment to the CYP2B6 reference sequence with GenBank Accession Number [NC_000019.10](https://www.ncbi.nlm.nih.gov/nuccore/NC_000019.10).

### CYP2B6 Allele, Genotype, Haplotype, and Diplotype Analyses

*CYP2B6* star allele designations were assigned in accordance with the Human Cytochrome P450 Allele Nomenclature Database ([http://cypalleles.ki.se/](http://cypalleles.ki.se/)). Allele and genotype frequencies, Hardy-Weinberg equilibrium analysis, and linkage disequilibrium analysis were performed using SHEsis [[30](#R30)]. Haplotype construction was performed using PHASE version 2.1.1 [[31](#R31), [32](#R32)]. Haplotype and diplotype Venn diagrams were generated using BioVenn [[33](#R33)].

### In Silico MicroRNA Predictions

The PolymiRTS Database 3.0 accessed June 2014 was used to identify SNPs predicted to create or abolish miRNA binding sites for *CYP2B6* [[13](#R13)]. For the dinucleotide variant, rs70950385, manual seed sequence alignment was performed.

### In Vitro Luciferase Assay

The pIS-0 firefly luciferase vector [[34](#R34)] (Addgene, Cambridge, MA) was used to study 3′UTR function in relation to miRNA regulation. The oligo containing either the wild-type (5′-GCCGTGTAATTCTAGGAGCTCGTCACACACTGCTGTAGTCTTCCCC**AG**TCCTCATCGTTCTAGAGTCGGGGC-3′) or variant (5′-GCCGTGTAATTCTAGGAGCTCGTCACACACTGCTGTAGTCTTCCCC**CA**TCCTCATCGTTCTAGAGTCGGGGC-3′) miRNA binding sites for the rs70950385 SNP (Integrated DNA Technologies, Coralville, IA) were amplified and then cloned into the pIS-0 vector 3′UTR using the NEBuilder HiFi DNA assembly system as instructed by the user manual (New England BioLabs, Ipswich, MA). The plasmids were then transformed into One Shot OmniMax 2 TI Chemically Competent *E. coli* (Invitrogen, Carlsbad, CA) and plated on ampicillin-treated agar plates. Individual colonies were selected and grown in ampicillin-treated liquid culture overnight. Plasmid DNA was isolated using a NucleoSpin Plasmid (NoLid) DNA Purification Kit (Macherey-Nagel, Düren, Germany) and quantified using a DNA dsDNA BR Assay Kit (Invitrogen, Eugene, OR). DNA was subsequently sequenced using the 5′-GTGGTTTGTCCAAACTCATC-3′ reverse primer to ensure clones with the correct wild-type and variant sequences were selected.

HepG2 human liver carcinoma cells were seeded at a density of 90,000 cells per well in 24 well plates and cultured for 48 hours. HepG2 cells were then transfected using Lipofectamine 3000 (Life Technologies, Foster City, CA) for 48 hours with each plasmid (500ng/well) with or without the miRNAs (hsa-miR-625-5p and hsa-miR-1275; final concentration: [30pM]) (GE Dharmacon, Chicago, IL) predicted to target the *CYP2B6* 3′UTR region encompassing the genetic variant. *Renilla* luciferase (10ng/well) was used as the transfection control. *C. elegans* cel-miR-67 was employed as a negative control. Luciferase activity was measured with the Dual-Glo Luciferase Assay as instructed by the user manual (Promega, Madison, WI).

### Statistical Analysis

Efavirenz AUC_0-48_ and C_max_ metabolite:parent ratios, and bupropion hydroxylase activities (untransformed data) were analyzed by genotype, haplotype, or diplotype using Kruskal-Wallis followed by Dunn’s post-hoc tests. Multiple comparisons within each SNP group were accounted for by Dunn’s post-hoc tests. Rather than increasing the stringency of our statistical test to reduce the false positive findings resulting from testing multiple *CYP2B6* variants, we followed up our findings with a human liver microsome population study and *in vitro* miRNA functional studies. *In vitro* luciferase assay results were analyzed using paired ratio t-tests of Firefly/*Renilla* luciferase activity ratios. All data analyses were performed using Graphpad Prism version 6 (La Jolla, CA); *p-*values <0.05 were considered statistically significant.

## Supplementary Material

### What is the current knowledge on the topic?

### What is the current knowledge on the topic?

### What is the current knowledge on the topic?

### What is the current knowledge on the topic?

Some of the wide pharmacokinetic variability among efavirenz and other drugs metabolized by CYP2B6 have been accounted for by the *CYP2B6***6* and **18* allelic variants; however, unexplained inter-individual variability still remains. Emerging evidence implicates genetic variants in the 3′UTR to impact variability in drug metabolism by altering microRNA regulation.

### What question does this study address?

We hypothesized that genetic variants in the 3′UTR contribute to the wide variability observed in CYP2B6 activity through microRNA regulation. We sequenced the 3′UTR of healthy human volunteers previously administered efavirenz and functionally validated the variants *in vitro*. Findings are supported by data obtained from a panel of human liver tissue samples.

### What this study adds to our knowledge

Genetic variants in the 3′UTR contributes to the variability in efavirenz metabolism among genotypic normal CYP2B6 metabolizers. The observed decrease in CYP2B6 activity in rs70950385 carriers of the variant allele appears to be related to microRNA regulation by miR-1275.

### How this might change clinical pharmacology or translational science

This study demonstrates an association between altered CYP2B6 activity and genetic variants in the 3′UTR. These findings help to further explain the variability in CYP2B6 activity among patients after other well-studied variants have been accounted for. Inclusion of this SNP into test panels may improve the prediction of CYP2B6 activity from genotype.

## Acknowledgments

This study was supported by funding from NIH/NIGMS F31-GM119401 (Burgess), Clinical Pharmacology Training Grant T32-GM008425 (Gufford), R01-GM078501 (Desta), R01-GM088076 (Skaar) and the Vera Bradley Foundation for Breast Cancer (Swart, Ipe). The National Institute of Child Health and Human Development Brain and Tissue Bank for Developmental Disorders at the University of Maryland is funded by the National Institutes of Health (NIH) contract HHSN275200900011C, reference number, N01-HD-9-0011 and the Liver Tissue Cell Distribution System is funded by NIH contract number N01-DK-7-0004/HHSN267200700004C.

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Desta Z, et al. Impact of CYP2B6 polymorphism on hepatic efavirenz metabolism in vitro. Pharmacogenomics. 2007;8(6):547–58. doi: 10.2217/14622416.8.6.547.  [DOI](https://doi.org/10.2217/14622416.8.6.547) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17559344/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Impact%20of%20CYP2B6%20polymorphism%20on%20hepatic%20efavirenz%20metabolism%20in%20vitro&author=Z%20Desta&volume=8&issue=6&publication_year=2007&pages=547-58&pmid=17559344&doi=10.2217/14622416.8.6.547&)

2. Klein K, et al. Genetic variability of CYP2B6 in populations of African and Asian origin: allele frequencies, novel functional variants, and possible implications for anti-HIV therapy with efavirenz. Pharmacogenet Genomics. 2005;15(12):861–73. doi: 10.1097/01213011-200512000-00004.  [DOI](https://doi.org/10.1097/01213011-200512000-00004) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16272958/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Genetic%20variability%20of%20CYP2B6%20in%20populations%20of%20African%20and%20Asian%20origin:%20allele%20frequencies,%20novel%20functional%20variants,%20and%20possible%20implications%20for%20anti-HIV%20therapy%20with%20efavirenz&author=K%20Klein&volume=15&issue=12&publication_year=2005&pages=861-73&pmid=16272958&doi=10.1097/01213011-200512000-00004&)

3. Hofmann MH, et al. Aberrant splicing caused by single nucleotide polymorphism c.516G>T [Q172H], a marker of CYP2B6*6, is responsible for decreased expression and activity of CYP2B6 in liver. J Pharmacol Exp Ther. 2008;325(1):284–92. doi: 10.1124/jpet.107.133306.  [DOI](https://doi.org/10.1124/jpet.107.133306) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18171905/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharmacol%20Exp%20Ther&title=Aberrant%20splicing%20caused%20by%20single%20nucleotide%20polymorphism%20c.516G>T%20%5BQ172H%5D,%20a%20marker%20of%20CYP2B6*6,%20is%20responsible%20for%20decreased%20expression%20and%20activity%20of%20CYP2B6%20in%20liver&author=MH%20Hofmann&volume=325&issue=1&publication_year=2008&pages=284-92&pmid=18171905&doi=10.1124/jpet.107.133306&)

4. Eap CB, et al. Stereoselective block of hERG channel by (S)-methadone and QT interval prolongation in CYP2B6 slow metabolizers. Clin Pharmacol Ther. 2007;81(5):719–28. doi: 10.1038/sj.clpt.6100120.  [DOI](https://doi.org/10.1038/sj.clpt.6100120) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17329992/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Stereoselective%20block%20of%20hERG%20channel%20by%20(S)-methadone%20and%20QT%20interval%20prolongation%20in%20CYP2B6%20slow%20metabolizers&author=CB%20Eap&volume=81&issue=5&publication_year=2007&pages=719-28&pmid=17329992&doi=10.1038/sj.clpt.6100120&)

5. Nakajima M, et al. Genetic polymorphisms of CYP2B6 affect the pharmacokinetics/pharmacodynamics of cyclophosphamide in Japanese cancer patients. Pharmacogenet Genomics. 2007;17(6):431–45. doi: 10.1097/FPC.0b013e328045c4fb.  [DOI](https://doi.org/10.1097/FPC.0b013e328045c4fb) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17502835/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Genetic%20polymorphisms%20of%20CYP2B6%20affect%20the%20pharmacokinetics/pharmacodynamics%20of%20cyclophosphamide%20in%20Japanese%20cancer%20patients&author=M%20Nakajima&volume=17&issue=6&publication_year=2007&pages=431-45&pmid=17502835&doi=10.1097/FPC.0b013e328045c4fb&)

6. Organization, W.H. Consolidated guidelines on the use of antiretroviral drugs for treating and preventing HIV infection: recommendations for a public health approach. 2016.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/27466667/) | [Google Scholar](https://scholar.google.com/scholar_lookup?title=Consolidated%20guidelines%20on%20the%20use%20of%20antiretroviral%20drugs%20for%20treating%20and%20preventing%20HIV%20infection:%20recommendations%20for%20a%20public%20health%20approach&publication_year=2016&)

7. Adkins JC, Noble S. Efavirenz. Drugs. 1998;56(6):1055–64. doi: 10.2165/00003495-199856060-00014. discussion 1065–6.  [DOI](https://doi.org/10.2165/00003495-199856060-00014) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9878993/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drugs&title=Efavirenz&author=JC%20Adkins&author=S%20Noble&volume=56&issue=6&publication_year=1998&pages=1055-64&pmid=9878993&doi=10.2165/00003495-199856060-00014&)

8. Marzolini C, et al. Efavirenz plasma levels can predict treatment failure and central nervous system side effects in HIV-1-infected patients. AIDS. 2001;15(1):71–5. doi: 10.1097/00002030-200101050-00011.  [DOI](https://doi.org/10.1097/00002030-200101050-00011) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11192870/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=AIDS&title=Efavirenz%20plasma%20levels%20can%20predict%20treatment%20failure%20and%20central%20nervous%20system%20side%20effects%20in%20HIV-1-infected%20patients&author=C%20Marzolini&volume=15&issue=1&publication_year=2001&pages=71-5&pmid=11192870&doi=10.1097/00002030-200101050-00011&)

9. Kharasch ED, et al. Methadone Pharmacogenetics: CYP2B6 Polymorphisms Determine Plasma Concentrations, Clearance, and Metabolism. Anesthesiology. 2015;123(5):1142–53. doi: 10.1097/ALN.0000000000000867.  [DOI](https://doi.org/10.1097/ALN.0000000000000867) | [PMC free article](/articles/PMC4667947/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26389554/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Anesthesiology&title=Methadone%20Pharmacogenetics:%20CYP2B6%20Polymorphisms%20Determine%20Plasma%20Concentrations,%20Clearance,%20and%20Metabolism&author=ED%20Kharasch&volume=123&issue=5&publication_year=2015&pages=1142-53&pmid=26389554&doi=10.1097/ALN.0000000000000867&)

10. Swart M, Dandara C. Genetic variation in the 3′-UTR of CYP1A2, CYP2B6, CYP2D6, CYP3A4, NR1I2, and UGT2B7: potential effects on regulation by microRNA and pharmacogenomics relevance. Front Genet. 2014;5:167. doi: 10.3389/fgene.2014.00167.  [DOI](https://doi.org/10.3389/fgene.2014.00167) | [PMC free article](/articles/PMC4044583/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24926315/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front%20Genet&title=Genetic%20variation%20in%20the%203%E2%80%B2-UTR%20of%20CYP1A2,%20CYP2B6,%20CYP2D6,%20CYP3A4,%20NR1I2,%20and%20UGT2B7:%20potential%20effects%20on%20regulation%20by%20microRNA%20and%20pharmacogenomics%20relevance&author=M%20Swart&author=C%20Dandara&volume=5&publication_year=2014&pages=167&pmid=24926315&doi=10.3389/fgene.2014.00167&)

11. Ramamoorthy A, et al. In silico and in vitro identification of microRNAs that regulate hepatic nuclear factor 4alpha expression. Drug Metab Dispos. 2012;40(4):726–33. doi: 10.1124/dmd.111.040329.  [DOI](https://doi.org/10.1124/dmd.111.040329) | [PMC free article](/articles/PMC3310421/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22232426/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=In%20silico%20and%20in%20vitro%20identification%20of%20microRNAs%20that%20regulate%20hepatic%20nuclear%20factor%204alpha%20expression&author=A%20Ramamoorthy&volume=40&issue=4&publication_year=2012&pages=726-33&pmid=22232426&doi=10.1124/dmd.111.040329&)

12. Liu J. Control of protein synthesis and mRNA degradation by microRNAs. Curr Opin Cell Biol. 2008;20(2):214–21. doi: 10.1016/j.ceb.2008.01.006.  [DOI](https://doi.org/10.1016/j.ceb.2008.01.006) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18329869/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Opin%20Cell%20Biol&title=Control%20of%20protein%20synthesis%20and%20mRNA%20degradation%20by%20microRNAs&author=J%20Liu&volume=20&issue=2&publication_year=2008&pages=214-21&pmid=18329869&doi=10.1016/j.ceb.2008.01.006&)

13. Bhattacharya A, Ziebarth JD, Cui Y. PolymiRTS Database 3.0: linking polymorphisms in microRNAs and their target sites with human diseases and biological pathways. Nucleic Acids Res. 2014;42(Database issue):D86–91. doi: 10.1093/nar/gkt1028.  [DOI](https://doi.org/10.1093/nar/gkt1028) | [PMC free article](/articles/PMC3965097/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24163105/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=PolymiRTS%20Database%203.0:%20linking%20polymorphisms%20in%20microRNAs%20and%20their%20target%20sites%20with%20human%20diseases%20and%20biological%20pathways&author=A%20Bhattacharya&author=JD%20Ziebarth&author=Y%20Cui&volume=42&issue=Database%20issue&publication_year=2014&pages=D86-91&pmid=24163105&doi=10.1093/nar/gkt1028&)

14. Administration, F.a.D. Guidance for industry: drug interaction studies-study design, data analysis, implications for dosing, and labeling recommendations. Center for Drug Evaluation and Research (CDER); Rockville: 2012.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Guidance%20for%20industry:%20drug%20interaction%20studies-study%20design,%20data%20analysis,%20implications%20for%20dosing,%20and%20labeling%20recommendations&publication_year=2012&)

15. Xu C, et al. CYP2B6 pharmacogenetics-based in vitro-in vivo extrapolation of efavirenz clearance by physiologically based pharmacokinetic modeling. Drug Metab Dispos. 2013;41(12):2004–11. doi: 10.1124/dmd.113.051755.  [DOI](https://doi.org/10.1124/dmd.113.051755) | [PMC free article](/articles/PMC3834132/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23846872/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=CYP2B6%20pharmacogenetics-based%20in%20vitro-in%20vivo%20extrapolation%20of%20efavirenz%20clearance%20by%20physiologically%20based%20pharmacokinetic%20modeling&author=C%20Xu&volume=41&issue=12&publication_year=2013&pages=2004-11&pmid=23846872&doi=10.1124/dmd.113.051755&)

16. Jiang F, et al. Effects of clopidogrel and itraconazole on the disposition of efavirenz and its hydroxyl metabolites: exploration of a novel CYP2B6 phenotyping index. Br J Clin Pharmacol. 2013;75(1):244–53. doi: 10.1111/j.1365-2125.2012.04314.x.  [DOI](https://doi.org/10.1111/j.1365-2125.2012.04314.x) | [PMC free article](/articles/PMC3555064/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22554354/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Clin%20Pharmacol&title=Effects%20of%20clopidogrel%20and%20itraconazole%20on%20the%20disposition%20of%20efavirenz%20and%20its%20hydroxyl%20metabolites:%20exploration%20of%20a%20novel%20CYP2B6%20phenotyping%20index&author=F%20Jiang&volume=75&issue=1&publication_year=2013&pages=244-53&pmid=22554354&doi=10.1111/j.1365-2125.2012.04314.x&)

17. Genomes Project, C., et al. A global reference for human genetic variation. Nature. 2015;526(7571):68–74. doi: 10.1038/nature15393.  [DOI](https://doi.org/10.1038/nature15393) | [PMC free article](/articles/PMC4750478/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26432245/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=A%20global%20reference%20for%20human%20genetic%20variation&volume=526&issue=7571&publication_year=2015&pages=68-74&pmid=26432245&doi=10.1038/nature15393&)

18. Burgess KS, et al. Age-related changes in microRNA expression and pharmacogenes in human liver. Clin Pharmacol Ther. 2015 doi: 10.1002/cpt.145.  [DOI](https://doi.org/10.1002/cpt.145) | [PMC free article](/articles/PMC4512918/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25968989/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Age-related%20changes%20in%20microRNA%20expression%20and%20pharmacogenes%20in%20human%20liver&author=KS%20Burgess&publication_year=2015&pmid=25968989&doi=10.1002/cpt.145&)

19. Robarge JD, et al. Population Pharmacokinetic Modeling To Estimate the Contributions of Genetic and Nongenetic Factors to Efavirenz Disposition. Antimicrob Agents Chemother. 2017;61(1) doi: 10.1128/AAC.01813-16.  [DOI](https://doi.org/10.1128/AAC.01813-16) | [PMC free article](/articles/PMC5192152/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27799204/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Antimicrob%20Agents%20Chemother&title=Population%20Pharmacokinetic%20Modeling%20To%20Estimate%20the%20Contributions%20of%20Genetic%20and%20Nongenetic%20Factors%20to%20Efavirenz%20Disposition&author=JD%20Robarge&volume=61&issue=1&publication_year=2017&pmid=27799204&doi=10.1128/AAC.01813-16&)

20. Desta Z, et al. Inhibition of Cytochrome P450 2B6 Activity by Voriconazole Profiled Using Efavirenz Disposition in Healthy Volunteers. Antimicrob Agents Chemother. 2016;60(11):6813–6822. doi: 10.1128/AAC.01000-16.  [DOI](https://doi.org/10.1128/AAC.01000-16) | [PMC free article](/articles/PMC5075090/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27600044/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Antimicrob%20Agents%20Chemother&title=Inhibition%20of%20Cytochrome%20P450%202B6%20Activity%20by%20Voriconazole%20Profiled%20Using%20Efavirenz%20Disposition%20in%20Healthy%20Volunteers&author=Z%20Desta&volume=60&issue=11&publication_year=2016&pages=6813-6822&pmid=27600044&doi=10.1128/AAC.01000-16&)

21. Michaud V, et al. Efavirenz-mediated induction of omeprazole metabolism is CYP2C19 genotype dependent. Pharmacogenomics J. 2014;14(2):151–9. doi: 10.1038/tpj.2013.17.  [DOI](https://doi.org/10.1038/tpj.2013.17) | [PMC free article](/articles/PMC3740059/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23629159/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=Efavirenz-mediated%20induction%20of%20omeprazole%20metabolism%20is%20CYP2C19%20genotype%20dependent&author=V%20Michaud&volume=14&issue=2&publication_year=2014&pages=151-9&pmid=23629159&doi=10.1038/tpj.2013.17&)

22. Michaud V, et al. Induction of CYP2C19 and CYP3A activity following repeated administration of efavirenz in healthy volunteers. Clin Pharmacol Ther. 2012;91(3):475–82. doi: 10.1038/clpt.2011.249.  [DOI](https://doi.org/10.1038/clpt.2011.249) | [PMC free article](/articles/PMC3667659/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22318618/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Induction%20of%20CYP2C19%20and%20CYP3A%20activity%20following%20repeated%20administration%20of%20efavirenz%20in%20healthy%20volunteers&author=V%20Michaud&volume=91&issue=3&publication_year=2012&pages=475-82&pmid=22318618&doi=10.1038/clpt.2011.249&)

23. Lu AY, Levin W. Partial purification of cytochromes P-450 and P-448 from rat liver microsomes. Biochem Biophys Res Commun. 1972;46(3):1334–9. doi: 10.1016/s0006-291x(72)80121-9.  [DOI](https://doi.org/10.1016/s0006-291x(72)80121-9) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/5012170/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem%20Biophys%20Res%20Commun&title=Partial%20purification%20of%20cytochromes%20P-450%20and%20P-448%20from%20rat%20liver%20microsomes&author=AY%20Lu&author=W%20Levin&volume=46&issue=3&publication_year=1972&pages=1334-9&pmid=5012170&doi=10.1016/s0006-291x(72)80121-9&)

24. Pearce RE, et al. Developmental Expression of CYP2B6: A Comprehensive Analysis of mRNA Expression, Protein Content and Bupropion Hydroxylase Activity and the Impact of Genetic Variation. Drug Metab Dispos. 2016;44(7):948–58. doi: 10.1124/dmd.115.067546.  [DOI](https://doi.org/10.1124/dmd.115.067546) | [PMC free article](/articles/PMC4931886/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26608082/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Developmental%20Expression%20of%20CYP2B6:%20A%20Comprehensive%20Analysis%20of%20mRNA%20Expression,%20Protein%20Content%20and%20Bupropion%20Hydroxylase%20Activity%20and%20the%20Impact%20of%20Genetic%20Variation&author=RE%20Pearce&volume=44&issue=7&publication_year=2016&pages=948-58&pmid=26608082&doi=10.1124/dmd.115.067546&)

25. Faucette SR, et al. Validation of bupropion hydroxylation as a selective marker of human cytochrome P450 2B6 catalytic activity. Drug Metab Dispos. 2000;28(10):1222–30.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/10997944/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Validation%20of%20bupropion%20hydroxylation%20as%20a%20selective%20marker%20of%20human%20cytochrome%20P450%202B6%20catalytic%20activity&author=SR%20Faucette&volume=28&issue=10&publication_year=2000&pages=1222-30&pmid=10997944&)

26. Lang T, et al. Extensive genetic polymorphism in the human CYP2B6 gene with impact on expression and function in human liver. Pharmacogenetics. 2001;11(5):399–415. doi: 10.1097/00008571-200107000-00004.  [DOI](https://doi.org/10.1097/00008571-200107000-00004) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11470993/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=Extensive%20genetic%20polymorphism%20in%20the%20human%20CYP2B6%20gene%20with%20impact%20on%20expression%20and%20function%20in%20human%20liver&author=T%20Lang&volume=11&issue=5&publication_year=2001&pages=399-415&pmid=11470993&doi=10.1097/00008571-200107000-00004&)

27. Twist GP, et al. High-resolution melt analysis to detect sequence variations in highly homologous gene regions: application to CYP2B6. Pharmacogenomics. 2013;14(8):913–22. doi: 10.2217/pgs.13.66.  [DOI](https://doi.org/10.2217/pgs.13.66) | [PMC free article](/articles/PMC3866959/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23746185/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=High-resolution%20melt%20analysis%20to%20detect%20sequence%20variations%20in%20highly%20homologous%20gene%20regions:%20application%20to%20CYP2B6&author=GP%20Twist&volume=14&issue=8&publication_year=2013&pages=913-22&pmid=23746185&doi=10.2217/pgs.13.66&)

28. Platt AR, Woodhall RW, George AL., Jr Improved DNA sequencing quality and efficiency using an optimized fast cycle sequencing protocol. Biotechniques. 2007;43(1):58, 60, 62. doi: 10.2144/000112499.  [DOI](https://doi.org/10.2144/000112499) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17695253/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biotechniques&title=Improved%20DNA%20sequencing%20quality%20and%20efficiency%20using%20an%20optimized%20fast%20cycle%20sequencing%20protocol&author=AR%20Platt&author=RW%20Woodhall&author=AL%20George&volume=43&issue=1&publication_year=2007&pages=58&pmid=17695253&doi=10.2144/000112499&)

29. Hall TA. BioEdit: a user-friendly biological sequence alignment editor and analysis program for Windows 95/98/NT. Nucleic acids symposium series. 1999  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20acids%20symposium%20series&title=BioEdit:%20a%20user-friendly%20biological%20sequence%20alignment%20editor%20and%20analysis%20program%20for%20Windows%2095/98/NT&author=TA%20Hall&publication_year=1999&)

30. Shi YY, He L. SHEsis, a powerful software platform for analyses of linkage disequilibrium, haplotype construction, and genetic association at polymorphism loci. Cell Res. 2005;15(2):97–8. doi: 10.1038/sj.cr.7290272.  [DOI](https://doi.org/10.1038/sj.cr.7290272) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15740637/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell%20Res&title=SHEsis,%20a%20powerful%20software%20platform%20for%20analyses%20of%20linkage%20disequilibrium,%20haplotype%20construction,%20and%20genetic%20association%20at%20polymorphism%20loci&author=YY%20Shi&author=L%20He&volume=15&issue=2&publication_year=2005&pages=97-8&pmid=15740637&doi=10.1038/sj.cr.7290272&)

31. Stephens M, Donnelly P. A comparison of bayesian methods for haplotype reconstruction from population genotype data. The American Journal of Human Genetics. 2003;73(5):1162–1169. doi: 10.1086/379378.  [DOI](https://doi.org/10.1086/379378) | [PMC free article](/articles/PMC1180495/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14574645/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=The%20American%20Journal%20of%20Human%20Genetics&title=A%20comparison%20of%20bayesian%20methods%20for%20haplotype%20reconstruction%20from%20population%20genotype%20data&author=M%20Stephens&author=P%20Donnelly&volume=73&issue=5&publication_year=2003&pages=1162-1169&pmid=14574645&doi=10.1086/379378&)

32. Stephens M, Smith NJ, Donnelly P. A new statistical method for haplotype reconstruction from population data. The American Journal of Human Genetics. 2001;68(4):978–989. doi: 10.1086/319501.  [DOI](https://doi.org/10.1086/319501) | [PMC free article](/articles/PMC1275651/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11254454/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=The%20American%20Journal%20of%20Human%20Genetics&title=A%20new%20statistical%20method%20for%20haplotype%20reconstruction%20from%20population%20data&author=M%20Stephens&author=NJ%20Smith&author=P%20Donnelly&volume=68&issue=4&publication_year=2001&pages=978-989&pmid=11254454&doi=10.1086/319501&)

33. Hulsen T, de Vlieg J, Alkema W. BioVenn - a web application for the comparison and visualization of biological lists using area-proportional Venn diagrams. BMC Genomics. 2008;9:488. doi: 10.1186/1471-2164-9-488.  [DOI](https://doi.org/10.1186/1471-2164-9-488) | [PMC free article](/articles/PMC2584113/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18925949/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Genomics&title=BioVenn%20-%20a%20web%20application%20for%20the%20comparison%20and%20visualization%20of%20biological%20lists%20using%20area-proportional%20Venn%20diagrams&author=T%20Hulsen&author=J%20de%20Vlieg&author=W%20Alkema&volume=9&publication_year=2008&pages=488&pmid=18925949&doi=10.1186/1471-2164-9-488&)

34. Yekta S, I, Shih H, Bartel DP. MicroRNA-directed cleavage of HOXB8 mRNA. Science. 2004;304(5670):594–6. doi: 10.1126/science.1097434.  [DOI](https://doi.org/10.1126/science.1097434) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15105502/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=MicroRNA-directed%20cleavage%20of%20HOXB8%20mRNA&author=S%20Yekta&author=H%20Shih&author=DP%20Bartel&volume=304&issue=5670&publication_year=2004&pages=594-6&pmid=15105502&doi=10.1126/science.1097434&)
