# Genetic variation in CFH predicts phenytoin-induced maculopapular exanthema in European-descent patients

## Metadata
**Authors:** Mark McCormack, Hongsheng Gui, Andrés Ingason, Doug Speed, Galen EB Wright, Eunice J Zhang, Rodrigo Secolin, Clarissa Yasuda, Maxwell Kwok, Stefan Wolking, Felicitas Becker, Sarah Rau, Andreja Avbersek, Kristin Heggeli, Costin Leu, Chantal Depondt, Graeme J Sills, Anthony G Marson, Pauls Auce, Martin J Brodie, Ben Francis, Michael R Johnson, Bobby PC Koeleman, Pasquale Striano, Antonietta Coppola, Federico Zara, Wolfram S Kunz, Josemir W Sander, Holger Lerche, Karl Martin Klein, Sarah Weckhuysen, Martin Krenn, Lárus J Gudmundsson, Kári Stefánsson, Roland Krause, Neil Shear, Colin JD Ross, Norman Delanty, for the EPIGEN Consortium;, Munir Pirmohamed, Bruce C Carleton, for the Canadian Pharmacogenomics Network for Drug Safety;, Fernando Cendes, Iscia Lopes-Cendes, Wei-ping Liao, Terence J O'Brien, Sanjay M Sisodiya, for the EpiPGX Consortium;, Stacey Cherny, Patrick Kwan, Larry Baum, for the International League Against Epilepsy Consortium on Complex Epilepsies;, Gianpiero L Cavalleri
**Journal:** Neurology
**Date:** 2018 Jan 23
**DOI:** [10.1212/WNL.0000000000004853](https://doi.org/10.1212/WNL.0000000000004853)
**PMID:** 29288229
**PMCID:** PMC5798660
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5798660/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC5798660/pdf/NEUROLOGY2017804252.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC5798660/pdf/NEUROLOGY2017804252.pdf)

## Abstract

**Objective:** 
To characterize, among European and Han Chinese populations, the genetic predictors of maculopapular exanthema (MPE), a cutaneous adverse drug reaction common to antiepileptic drugs.

**Methods:** 
We conducted a case-control genome-wide association study of autosomal genotypes, including Class I and II human leukocyte antigen (HLA) alleles, in 323 cases and 1,321 drug-tolerant controls from epilepsy cohorts of northern European and Han Chinese descent. Results from each cohort were meta-analyzed.

**Results:** 
We report an association between a rare variant in the complement factor H–related 4 (CFHR4) gene and phenytoin-induced MPE in Europeans (p = 4.5 × 10–11; odds ratio [95% confidence interval] 7 [3.2–16]). This variant is in complete linkage disequilibrium with a missense variant (N1050Y) in the complement factor H (CFH) gene. In addition, our results reinforce the association between HLA-A*31:01 and carbamazepine hypersensitivity. We did not identify significant genetic associations with MPE among Han Chinese patients.

**Conclusions:** 
The identification of genetic predictors of MPE in CFHR4 and CFH, members of the complement factor H–related protein family, suggest a new link between regulation of the complement system alternative pathway and phenytoin-induced hypersensitivity in European-ancestral patients.

### Objective

To characterize, among European and Han Chinese populations, the genetic predictors of maculopapular exanthema (MPE), a cutaneous adverse drug reaction common to antiepileptic drugs.

### Methods

We conducted a case-control genome-wide association study of autosomal genotypes, including Class I and II human leukocyte antigen (HLA) alleles, in 323 cases and 1,321 drug-tolerant controls from epilepsy cohorts of northern European and Han Chinese descent. Results from each cohort were meta-analyzed.

### Results

We report an association between a rare variant in the complement factor H–related 4 (*CFHR4*) gene and phenytoin-induced MPE in Europeans (*p* = 4.5 × 10^–11^; odds ratio [95% confidence interval] 7 [3.2–16]). This variant is in complete linkage disequilibrium with a missense variant (N1050Y) in the complement factor H (*CFH*) gene. In addition, our results reinforce the association between *HLA-A*31:01* and carbamazepine hypersensitivity. We did not identify significant genetic associations with MPE among Han Chinese patients.

### Conclusions

The identification of genetic predictors of MPE in CFHR4 and CFH, members of the complement factor H–related protein family, suggest a new link between regulation of the complement system alternative pathway and phenytoin-induced hypersensitivity in European-ancestral patients.

## Methods

### Standard protocol approvals, registrations, and patient consents

All study participants provided written, informed consent for genetic analysis. Local institutional review boards approved study protocols at each contributing site.

### Study design

We conducted a retrospective case-control study in individuals of European and Han Chinese ethnicity. Participants were exposed to carbamazepine, lamotrigine, phenytoin, or oxcarbazepine. Our analyses were structured to test genetic variants for association with MPE within and across both of the broad ancestral groups, through logistic regression of genotype dosage and subsequent meta-analysis of regression coefficients. We tested for association with (1) aromatic AED-induced MPE vs controls tolerant to at least 3 aromatic AEDs, (2) carbamazepine-induced MPE vs carbamazepine-tolerant controls, (3) lamotrigine-induced MPE vs lamotrigine-tolerant controls, and (4) phenytoin-induced MPE vs phenytoin-tolerant controls. Due to small sample size, oxcarbazepine-related MPE was not analyzed as an individual case cohort.

### Cohorts and phenotype definition

Epilepsy cohorts from the ILAE-CGC, EpiPGX, and EPIGEN Consortia were included in the discovery GWAS meta-analysis ([table 1](#T1)). A European-descent replication cohort was assembled from sites in Brazil, Canada (via CPNDS), Liverpool, and other sites across the United Kingdom. Cases were defined as having MPE attributed to carbamazepine, lamotrigine, phenytoin, or oxcarbazepine as determined by their clinician, occurring within 3 months of initiation and resolving upon dose reduction or AED withdrawal. Control patients trialed carbamazepine, lamotrigine, phenytoin, or oxcarbazepine for at least 3 months without reporting a cADR. Epilepsy-specific patient demographics are presented in table e-2 ([links.lww.com/WNL/A56](http://links.lww.com/WNL/A56)).

### Table 1.

Table 1 Caption: Breakdown of antiepileptic drug (AED)–induced maculopapular exanthema (MPE) cases and AED-tolerant controls in discovery dataset

### Genotyping and imputation

Genotyping of a subset of EpiPGX samples was performed at deCODE Genetics (Reykjavik, Iceland) using Illumina (San Diego, CA) OmniExpress-12 v1.1 and OmniExpress-24 v1.1 single nucleotide polymorphism (SNP) arrays. The remainder of samples were genotyped locally using various Illumina beadchip SNP arrays, details of which are published elsewhere.^13^ Genotyping and imputation quality control is described in appendix e-1 ([links.lww.com/WNL/A57](http://links.lww.com/WNL/A57)).

### Study power

We estimated that the study had 80% power to detect a genetic predictor of relative risk (approximated to odds ratio) ≥3 with an allele frequency ≥2% and an α level of 1.25 × 10^−8^. Power for AED-specific and population-specific analyses are detailed in appendix e-1 ([links.lww.com/WNL/A57](http://links.lww.com/WNL/A57)).

### Statistical analyses

Association analyses were conducted within the European and Asian subgroups using an additive logistic regression model. To account for genotype uncertainty, SNPTEST was used to apply a missing data likelihood score model that included sex, clinical site, and 5 principal components as covariates to control for bias and population stratification.^15^ Fixed effects meta-analyses were conducted across the European and Asian subgroups using the software package METAL, applying genomic control correction within cohorts.^16^ The threshold for statistical significance was set at 1.25 × 10^−8^, reflecting an empirical Bonferroni correction for 4 tests, of the standard 5 × 10^−8^ genome-wide significance threshold. Conditional association analysis was performed on loci containing significant markers to establish whether other genetic variants in the region (1 Mb upstream and downstream) were independently associated with MPE. The conditional threshold for significance was set at 5 × 10^−6^, based on a genome-wide estimation of 10,000 imputed variants per 2 MB region.^13^ We applied the Stouffer *z* trend test to the combined results from the discovery and replication cohorts.

### Confirmatory genotyping

Where an association signal satisfied the threshold for significance, additional genotyping and resequencing were performed in a subset of patients and results were compared with imputation dosage files. The variant rs78239784 was confirmed by Sanger sequencing in 100 patients from the original discovery cohort. For the purpose of replication, we genotyped the rs78239784 variant in an independent cohort of 13 phenytoin-induced MPE cases and 88 phenytoin-tolerant controls.

## Results

### Cohort description

In total, 375 MPE cases and 1,321 controls satisfied our criteria for inclusion in the discovery analyses (see Methods and [table 1](#T1)). There were 16 patients with cross-reactivity to 2 or more aromatic AEDs, 8 of whom were hypersensitive to carbamazepine and lamotrigine. Genome-wide array data for 323 cases and 1,321 controls were available for analysis. Broad European or Han Chinese ancestry was assigned to each participant according to principal components analysis (figure e-1, [links.lww.com/WNL/A55](http://links.lww.com/WNL/A55)).

### Genome-wide association analysis of broad aromatic AED–induced MPE

After quality control (see appendix e-1, figure e-2, [links.lww.com/WNL/A55](http://links.lww.com/WNL/A55), for details), 3,693,290 variants remained for analysis in the European dataset and 4,402,554 variants in the Han Chinese dataset. We only considered autosomal SNPs in our analyses. To test hypothesis (1), that population-specific genetic markers predispose to MPE, a logistic regression analysis of all MPE cases was performed separately in the European and Han Chinese ancestral subgroups. We did not observe any genome-wide significant markers for MPE due to any aromatic AED in either Europeans or Han Chinese. The study was powered to detect an effect of relative risk >3.5 in the European cohort and >5 in the Han Chinese.

To test hypothesis (2), that transethnic genetic markers predispose to MPE, a fixed-effects meta-analysis of the association results for European and Han Chinese ancestral subgroups was performed. We did not observe any genome-wide significant markers for MPE shared among European or Han Chinese subgroups ([figure 1A](#F1)). This analysis was powered to detect an effect size >3.

### Figure 1. Meta-analysis results across European and Han Chinese cohorts.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8d6c/5798660/1c46890176dd/NEUROLOGY2017804252FF1.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5798660_NEUROLOGY2017804252FF1.jpg)

Manhattan (a) and quantile–quantile (b) plots for the meta-analyses of maculopapular exanthema vs tolerant controls, for (A) any antiepileptic drug (genomic inflation factor [λ] = 1.01), (B) carbamazepine (λ = 1.01), (C) lamotrigine (λ = 0.99), and (D) phenytoin (λ = 0.98).

### Genome-wide association analysis of specific aromatic AED–induced MPE

To test hypothesis (3), that genetic variants for MPE are AED-specific and population-specific, logistic regression analyses of AED-specific MPE was performed separately in the European and Han Chinese ancestral groups (figures e-3 and e-4, [links.lww.com/WNL/A55](http://links.lww.com/WNL/A55)). Within the European subgroup, *HLA-A*31:01* was significantly associated with carbamazepine-induced MPE in Europeans (*p* = 1.47 × 10^−10^, odds ratio [OR] [95% confidence interval (CI)] 5.5 [3.0–10]). Conditioning on *HLA-A*31:01* did not reveal additional variants within the human leukocyte antigen (HLA) region that were independently contributing to carbamazepine-induced MPE. No genome-wide significant signals for lamotrigine-induced MPE were observed in Europeans. This analysis was powered to detect an effect size >6.

For phenytoin-induced MPE, we identified a significant association with rs78239784, an intronic variant of the complement factor H–related 4 gene (*CFHR4*). The risk allele, G, had a minor allele frequency of 12% in our European phenytoin-induced MPE cases compared to 1.5% in European phenytoin-tolerant controls (*p* = 2.94 × 10^−10^, OR [95% CI] 8.8 [4.0–19]; [figure 2](#F2)). Conditioning on rs78239784 did not reveal additional variants in this locus that were independently contributing to phenytoin-induced MPE. Using 1000 Genomes Phase III European population data, rs78239784 was found to be in complete linkage disequilibrium with rs35274867 (*r*^2^ = 1 and D' = 1), a missense variant coding for an asparagine to tyrosine substitution at amino acid 1,050 of the complement factor H (*CFH*) gene. The missense variant was not present in our association results, as it was filtered during quality control because the imputation score was <0.95. The imputation accuracy of the top variant rs78239784 was confirmed in our cohort via Sanger sequencing and TaqMan approaches. Of 100 samples tested, a 100% concordance rate was found between imputed and resequenced genotypes. Within the Han Chinese subgroup, no significant associations were found between autosomal SNPs or *HLA* alleles and AED-specific MPE. Summary results for known risk loci in our dataset were scrutinized and are presented in [table 2](#T2). None of these loci was even nominally significant (*p* > 0.05) in the Han Chinese subgroup.

### Figure 2. Intronic CFHR4 variants are associated with phenytoin-induced maculopapular exanthema (MPE) in Europeans.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8d6c/5798660/275811ec42f6/NEUROLOGY2017804252FF2.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5798660_NEUROLOGY2017804252FF2.jpg)

(A) Manhattan and (B) quantile–quantile plot of phenytoin-induced MPE in the European subgroup (λ = 1.01). The LocusZoom plot (C) highlights the most significant single nucleotide polymorphism (SNP), rs78239784 (purple dot), is an intronic variant in CFHR4.

### Table 2.

Table 2 Caption: Association test results for risk alleles for maculopapular exanthema (MPE) across ethnicities

In order to test hypothesis (4), that transethnic genetic markers predispose to AED-specific MPE, we meta-analyzed *p* values from association results for carbamazepine, lamotrigine, and phenytoin individually, across European and Han Chinese ancestral subgroups. There were no shared genome-wide significant markers among our meta-analyses of AED-specific MPE ([figure 1, B–D](#F1)).

### Replication of CFHR4 signal

To replicate the association with phenytoin-induced MPE in an independent cohort, the variant rs78239784 was genotyped in self-reported European-descent cases and controls recruited through centers in Liverpool (United Kingdom), Sao Paolo (Brazil), and across Canada (CPNDS). Two heterozygous carriers were identified among 13 phenytoin-induced MPE cases while only a single carrier was observed among 88 phenytoin-tolerant controls yielding a 2-tailed Fisher exact *p* value of 0.044. Pooling all cases and controls together, we report an overall *p* value of 4.5 × 10^−11^ (with a combined OR [95% CI] 7 [3.2–16]) for the association between rs78239784 and phenytoin-induced MPE in Europeans ([table 3](#T3)).

### Table 3.

Table 3 Caption: rs78239784 associates with phenytoin-induced maculopapular exanthema in Europeans

## Discussion

We detected a strong association between variants in the complement factor H regulatory pathway and phenytoin-induced MPE in a European-descent patient population. The presence of the associated genotype increases risk for MPE 6-fold. Our results indicate that risk variants for MPE tend to be drug-specific and population-specific.

These results point to the regulators of complement activation gene cluster as a genetic locus contributing to the onset of hypersensitivity to phenytoin. The most significant variant in our European subgroup analysis, rs78239784 (c.59-2448T>G), tags the missense variant rs35274867 (p.N1050Y) in *CFH*, suggesting aberrant complement activation as a potential causal mechanism in a subset of phenytoin-sensitive individuals. According to data from the Exome Aggregation Consortium, *CFH* N1050Y has an allele frequency of approximately 2% in Europeans, 3% in African subpopulations, less than 1% in South Asians, and is almost invariant in East Asians.^17^ Given the absence of this allele in East Asian populations, the lack of an association between the *CFH* locus and MPE in our Han Chinese cohort is unsurprising. We propose that population-specific, independent rare variants of large effect may explain a proportion of MPE cases, a similar paradigm to the rare variant model demonstrated in Crohn disease and ulcerative colitis.^18^ *CFH* N1050Y has previously been associated with type 2 diabetes–associated end-stage kidney disease in an African American cohort.^19^ Defects in CFH-related proteins are also associated with overactivation of the complement immune system and can lead to atypical hemolytic-uremic syndrome, C3 glomerulopathy, basal laminar drusen, immunoglobulin A nephropathies, and systemic lupus erythematosus.^20^ Further, genetic variants in *CFHR4* and *CFH* are associated with risk for age-related macular degeneration.^21^ We did not detect these symptoms among N1050Y carriers. While it is unclear whether phenytoin directly interacts with circulating CFH-related proteins, it does not specifically increase serum complement levels.^22^ Our findings offer an expanded insight into the role of the complement alternative pathway in hypersensitivity to AEDs.

Phenytoin is still used as a first-line treatment for epilepsy in many settings and is listed on the WHO list of essential medicines.^23^ Epidemiologic data on prescriptions of AEDs for epilepsy in the United Kingdom show that, in 2008, phenytoin accounted for 18% of all treated person-years in epilepsy and was most frequently used in the elderly.^24^ Therefore, a clinically useful prognostic test for phenytoin-induced cutaneous ADRs in European-ancestral individuals would be welcome. The sensitivity of the *CFHR4* variant as a prognostic marker is 16% and the specificity is 97%, which corresponds to a positive likelihood ratio of 5.93 (95% CI 2.8–12.6) and a negative likelihood ratio of 0.86 (95% CI 0.8–0.9). Assuming the pretest probability of the ADR is 5%, a positive test for this marker increases the probability of MPE to phenytoin sixfold to 30%, while a negative test reduces the probability marginally to 4.3%. There are an estimated 6 million people with epilepsy in Europe, which means approximately 90,000 people are at-risk carriers of this mutation.^25^ We estimate that 208 (95% CI 103–431) patients of European ancestry would need to be screened to prevent one case, based on a previously reported formula,^26^ which corresponds to an absolute risk reduction of 0.005 (95% CI 0.002–0.009). As a comparison, it is estimated that 442 Han Chinese patients would be needed to screened for *HLA-B*15:02* in order to prevent a single carbamazepine-induced SJS/TEN case. We would suggest that the clinical utility and cost-effectiveness of implementing preemptive screening be evaluated through a prospective study.

We did not replicate the association between *CYP2C9*3* (rs1057910) and MPE in our cohort, irrespective of ethnicity or AED. This is not surprising given that the original association with phenytoin in Han Chinese was largely driven by SJS/TEN cases, which were excluded from this analysis. We did, however, detect a nonsignificant enrichment of *CYP2C9*3* (*p* = 0.08) among the European phenytoin-induced MPE cases, but the effect size we observe (OR 1.8) is smaller than previously reported for phenytoin-induced MPE in Han Chinese (OR 5.5).^10^ Notably, 2 *CYP2C9*3* carriers among the phenytoin-induced MPE cases were also heterozygous for the *CFHR4* variant while only 3 of 560 controls were jointly heterozygous. The frequency of *CYP2C9*3* differs between controls from European and Han Chinese subgroups in our study, which is in accordance with background population frequency reported in the Exome Aggregation Consortium (European: 7%, East Asian: 3%). While our results do not support a significant effect of *CYP2C9*3* in MPE, larger cohorts including severe cADR cases may resolve the extent of the association across populations.

*HLA-A*31:01* was the most strongly associated marker with carbamazepine-induced MPE in Europeans in this study. Forty-three of the 95 cases studied here were also included in the discovery publication,^8^ but the effect of the allele remains significant when we restrict to new cases only (p = 4 × 10^−7^), thus providing an additional independent replication of the initial finding. We confirm that *HLA-B*15:02* is not associated with carbamazepine-induced MPE in Han Chinese, and no novel signals emerged for carbamazepine-induced MPE in either population. No significant predictors of lamotrigine-induced MPE were observed in either population tested. *HLA-A*24:02* was not significantly associated with lamotrigine-induced MPE in either of the European or Han Chinese ancestral subgroups; rather this allele was observed to be more frequent among our lamotrigine-tolerant controls.

Our meta-analyses did not reveal any significant transethnic genetic markers for MPE due to any AED. There were considerably more European-descent patients in this analysis than any other ethnicity, and we recognize this as a limitation of the study. Analysis of non-European cohorts is warranted. A second limitation of our study was the low number of Han Chinese lamotrigine-related MPE cases, relative to European-descent cases. Therefore we cannot conclusively rule out genetic predictors of modest effect size for MPE to lamotrigine in Han Chinese or other non-European descent populations. We recognize that our replication cohort for phenytoin is small and comprises self-reported ancestral Europeans. Since we did not have full genotype array data for these individuals, we relied on Fisher exact test for calculating significance rather than logistic regression with correction for principal components. Additional studies of larger sample size are required to further characterize the association, improve the estimation of the risk effect size, and determine the prognostic ability and economics of screening for this marker. Finally, as this study was not powered to investigate MPE attributed to oxcarbazepine due to low sample size, further investigation is warranted.

We have identified a genetic predictor for a common adverse reaction to phenytoin in European-descent patients, adding a new pharmacogenetic marker for potential use in the treatment of epilepsy. This finding adds to the list of genetic predictors of hypersensitivity to anticonvulsant therapy and opens up a new avenue for understanding the biology underlying cutaneous adverse reactions. This finding can advance genetic testing in the clinic as it expands the array of genetic tests available to aid clinicians in reducing overall rates of discontinuation due to adverse events and improving patient safety.

## Acknowledgment

The authors thank the following coinvestigators: the members of the EpiPGX Consortium, the members of the Canadian Pharmacogenomics Network for Drug Safety (CPNDS) Consortium, the members of the International League Against Epilepsy Consortium on Complex Epilepsies (ILAE-CGC), and the members of the EPIGEN Consortium (a full list of coinvestigators is found in the e-Appendix); Ana Fulgenico-Maisch, Simona Donatello, Clare O'Kennedy, Lisa Slattery, Paula Corr, and Joanna Fay for clinical assessment of patients, technical assistance with DNA extraction, and sample management; and the patients for their participation in the study.

## Glossary

## Footnotes

## Contributor Information

Collaborators: International Workshop on Comorbidity in Multiple Sclerosis, [Samuel F. Berkovic](https://pubmed.ncbi.nlm.nih.gov/?term=%22Berkovic%20SF%22%5BAuthor%5D), [Daniel H. Lowenstein](https://pubmed.ncbi.nlm.nih.gov/?term=%22Lowenstein%20DH%22%5BAuthor%5D), [Erin L. Heinzen](https://pubmed.ncbi.nlm.nih.gov/?term=%22Heinzen%20EL%22%5BAuthor%5D), [Hakon Hakonarson](https://pubmed.ncbi.nlm.nih.gov/?term=%22Hakonarson%20H%22%5BAuthor%5D), [Russell J. Buono](https://pubmed.ncbi.nlm.nih.gov/?term=%22Buono%20RJ%22%5BAuthor%5D), [Karen Oliver](https://pubmed.ncbi.nlm.nih.gov/?term=%22Oliver%20K%22%5BAuthor%5D), [Honsheng Gui](https://pubmed.ncbi.nlm.nih.gov/?term=%22Gui%20H%22%5BAuthor%5D), [Stacey Cherny](https://pubmed.ncbi.nlm.nih.gov/?term=%22Cherny%20S%22%5BAuthor%5D), [Patrick Kwan](https://pubmed.ncbi.nlm.nih.gov/?term=%22Kwan%20P%22%5BAuthor%5D), [Larry Baum](https://pubmed.ncbi.nlm.nih.gov/?term=%22Baum%20L%22%5BAuthor%5D), [Maxwell Kwok](https://pubmed.ncbi.nlm.nih.gov/?term=%22Kwok%20M%22%5BAuthor%5D), [Doug Speed](https://pubmed.ncbi.nlm.nih.gov/?term=%22Speed%20D%22%5BAuthor%5D), [Rodrigo Secolin](https://pubmed.ncbi.nlm.nih.gov/?term=%22Secolin%20R%22%5BAuthor%5D), [Clarissa Yasuda](https://pubmed.ncbi.nlm.nih.gov/?term=%22Yasuda%20C%22%5BAuthor%5D), [Fernando Cendes](https://pubmed.ncbi.nlm.nih.gov/?term=%22Cendes%20F%22%5BAuthor%5D), [Iscia Lopes–Cendes](https://pubmed.ncbi.nlm.nih.gov/?term=%22Lopes%E2%80%93Cendes%20I%22%5BAuthor%5D), [Liao Wei-ping](https://pubmed.ncbi.nlm.nih.gov/?term=%22Wei-ping%20L%22%5BAuthor%5D), [Terence J. O’Brien](https://pubmed.ncbi.nlm.nih.gov/?term=%22O%E2%80%99Brien%20TJ%22%5BAuthor%5D), [Andres Ingason](https://pubmed.ncbi.nlm.nih.gov/?term=%22Ingason%20A%22%5BAuthor%5D), [Lárus J. Gudmundsson](https://pubmed.ncbi.nlm.nih.gov/?term=%22Gudmundsson%20LJ%22%5BAuthor%5D), [Kári Stefánsson](https://pubmed.ncbi.nlm.nih.gov/?term=%22Stef%C3%A1nsson%20K%22%5BAuthor%5D), [Sanjay M. Sisodiya](https://pubmed.ncbi.nlm.nih.gov/?term=%22Sisodiya%20SM%22%5BAuthor%5D), [Andreja Avbersek](https://pubmed.ncbi.nlm.nih.gov/?term=%22Avbersek%20A%22%5BAuthor%5D), [Kristin Heggeli](https://pubmed.ncbi.nlm.nih.gov/?term=%22Heggeli%20K%22%5BAuthor%5D), [Costin Leu](https://pubmed.ncbi.nlm.nih.gov/?term=%22Leu%20C%22%5BAuthor%5D), [Joseph Willis](https://pubmed.ncbi.nlm.nih.gov/?term=%22Willis%20J%22%5BAuthor%5D), [Anna Tostevin](https://pubmed.ncbi.nlm.nih.gov/?term=%22Tostevin%20A%22%5BAuthor%5D), [Krishna Chinthapalli](https://pubmed.ncbi.nlm.nih.gov/?term=%22Chinthapalli%20K%22%5BAuthor%5D), [Chantal Depondt](https://pubmed.ncbi.nlm.nih.gov/?term=%22Depondt%20C%22%5BAuthor%5D), [Mojgansadat Borghei](https://pubmed.ncbi.nlm.nih.gov/?term=%22Borghei%20M%22%5BAuthor%5D), [Gianpiero L. Cavalleri](https://pubmed.ncbi.nlm.nih.gov/?term=%22Cavalleri%20GL%22%5BAuthor%5D), [Norman Delanty](https://pubmed.ncbi.nlm.nih.gov/?term=%22Delanty%20N%22%5BAuthor%5D), [Mark McCormack](https://pubmed.ncbi.nlm.nih.gov/?term=%22McCormack%20M%22%5BAuthor%5D), [Sinéad Heavin](https://pubmed.ncbi.nlm.nih.gov/?term=%22Heavin%20S%22%5BAuthor%5D), [Wolfram S. Kunz](https://pubmed.ncbi.nlm.nih.gov/?term=%22Kunz%20WS%22%5BAuthor%5D), [Holger Lerche](https://pubmed.ncbi.nlm.nih.gov/?term=%22Lerche%20H%22%5BAuthor%5D), [Stefan Wolking](https://pubmed.ncbi.nlm.nih.gov/?term=%22Wolking%20S%22%5BAuthor%5D), [Felicitas Becker](https://pubmed.ncbi.nlm.nih.gov/?term=%22Becker%20F%22%5BAuthor%5D), [Sarah Rau](https://pubmed.ncbi.nlm.nih.gov/?term=%22Rau%20S%22%5BAuthor%5D), [Christian Hengsbach](https://pubmed.ncbi.nlm.nih.gov/?term=%22Hengsbach%20C%22%5BAuthor%5D), [Yvonne Weber](https://pubmed.ncbi.nlm.nih.gov/?term=%22Weber%20Y%22%5BAuthor%5D), [Zvonka Rener Primec](https://pubmed.ncbi.nlm.nih.gov/?term=%22Primec%20ZR%22%5BAuthor%5D), [Pasquale Striano](https://pubmed.ncbi.nlm.nih.gov/?term=%22Striano%20P%22%5BAuthor%5D), [Antonietta Coppola](https://pubmed.ncbi.nlm.nih.gov/?term=%22Coppola%20A%22%5BAuthor%5D), [Federico Zara](https://pubmed.ncbi.nlm.nih.gov/?term=%22Zara%20F%22%5BAuthor%5D), [Antonio Gamberdella](https://pubmed.ncbi.nlm.nih.gov/?term=%22Gamberdella%20A%22%5BAuthor%5D), [Amedeo Bianchi](https://pubmed.ncbi.nlm.nih.gov/?term=%22Bianchi%20A%22%5BAuthor%5D), [Giuseppe Capovilla](https://pubmed.ncbi.nlm.nih.gov/?term=%22Capovilla%20G%22%5BAuthor%5D), [Anna Teresa Giallonardo](https://pubmed.ncbi.nlm.nih.gov/?term=%22Giallonardo%20AT%22%5BAuthor%5D), [Angela La Neve](https://pubmed.ncbi.nlm.nih.gov/?term=%22La%20Neve%20A%22%5BAuthor%5D), [Giovanni Crichiutt](https://pubmed.ncbi.nlm.nih.gov/?term=%22Crichiutt%20G%22%5BAuthor%5D), [Aglaia Vignoli](https://pubmed.ncbi.nlm.nih.gov/?term=%22Vignoli%20A%22%5BAuthor%5D), [Roberto Michelucci](https://pubmed.ncbi.nlm.nih.gov/?term=%22Michelucci%20R%22%5BAuthor%5D), [Francesca Bisulli](https://pubmed.ncbi.nlm.nih.gov/?term=%22Bisulli%20F%22%5BAuthor%5D), [Carla Marini](https://pubmed.ncbi.nlm.nih.gov/?term=%22Marini%20C%22%5BAuthor%5D), [Belcastro Vicenzo](https://pubmed.ncbi.nlm.nih.gov/?term=%22Vicenzo%20B%22%5BAuthor%5D), [Michael R. Johnson](https://pubmed.ncbi.nlm.nih.gov/?term=%22Johnson%20MR%22%5BAuthor%5D), [Sarah R. Langley](https://pubmed.ncbi.nlm.nih.gov/?term=%22Langley%20SR%22%5BAuthor%5D), [Graeme J. Sills](https://pubmed.ncbi.nlm.nih.gov/?term=%22Sills%20GJ%22%5BAuthor%5D), [Anthony G. Marson](https://pubmed.ncbi.nlm.nih.gov/?term=%22Marson%20AG%22%5BAuthor%5D), [Pauls Auce](https://pubmed.ncbi.nlm.nih.gov/?term=%22Auce%20P%22%5BAuthor%5D), [Ben Francis](https://pubmed.ncbi.nlm.nih.gov/?term=%22Francis%20B%22%5BAuthor%5D), [Andrea Jorgensen](https://pubmed.ncbi.nlm.nih.gov/?term=%22Jorgensen%20A%22%5BAuthor%5D), [Martin J. Brodie](https://pubmed.ncbi.nlm.nih.gov/?term=%22Brodie%20MJ%22%5BAuthor%5D), [Prashant Srivastava](https://pubmed.ncbi.nlm.nih.gov/?term=%22Srivastava%20P%22%5BAuthor%5D), [Roland Krause](https://pubmed.ncbi.nlm.nih.gov/?term=%22Krause%20R%22%5BAuthor%5D), [Martin Krenn](https://pubmed.ncbi.nlm.nih.gov/?term=%22Krenn%20M%22%5BAuthor%5D), [Fritz Zimprich](https://pubmed.ncbi.nlm.nih.gov/?term=%22Zimprich%20F%22%5BAuthor%5D), [Ekaterina Pataraia](https://pubmed.ncbi.nlm.nih.gov/?term=%22Pataraia%20E%22%5BAuthor%5D), [Karl Martin Klein](https://pubmed.ncbi.nlm.nih.gov/?term=%22Klein%20KM%22%5BAuthor%5D), [Felix Rosenow](https://pubmed.ncbi.nlm.nih.gov/?term=%22Rosenow%20F%22%5BAuthor%5D), [Hiltrud Muhle](https://pubmed.ncbi.nlm.nih.gov/?term=%22Muhle%20H%22%5BAuthor%5D), [Dieter Dennig](https://pubmed.ncbi.nlm.nih.gov/?term=%22Dennig%20D%22%5BAuthor%5D), [Bernhard Steinhoff](https://pubmed.ncbi.nlm.nih.gov/?term=%22Steinhoff%20B%22%5BAuthor%5D), [Susanne Schubert-Bast](https://pubmed.ncbi.nlm.nih.gov/?term=%22Schubert-Bast%20S%22%5BAuthor%5D), [Thomas Mayer](https://pubmed.ncbi.nlm.nih.gov/?term=%22Mayer%20T%22%5BAuthor%5D), [Hartmut Baier](https://pubmed.ncbi.nlm.nih.gov/?term=%22Baier%20H%22%5BAuthor%5D), [Rikke S. Møller](https://pubmed.ncbi.nlm.nih.gov/?term=%22M%C3%B8ller%20RS%22%5BAuthor%5D), [Marina Nikanorova](https://pubmed.ncbi.nlm.nih.gov/?term=%22Nikanorova%20M%22%5BAuthor%5D), [Sarah Weckhuysen](https://pubmed.ncbi.nlm.nih.gov/?term=%22Weckhuysen%20S%22%5BAuthor%5D), [Peter de Jonghe](https://pubmed.ncbi.nlm.nih.gov/?term=%22de%20Jonghe%20P%22%5BAuthor%5D), [Hannah Stamberger](https://pubmed.ncbi.nlm.nih.gov/?term=%22Stamberger%20H%22%5BAuthor%5D), [Josemir W. Sander](https://pubmed.ncbi.nlm.nih.gov/?term=%22Sander%20JW%22%5BAuthor%5D), [Bianca Berghuis](https://pubmed.ncbi.nlm.nih.gov/?term=%22Berghuis%20B%22%5BAuthor%5D), [Bobby P.C. Koeleman](https://pubmed.ncbi.nlm.nih.gov/?term=%22Koeleman%20BPC%22%5BAuthor%5D), [Anja C.M. Sonsma](https://pubmed.ncbi.nlm.nih.gov/?term=%22Sonsma%20ACM%22%5BAuthor%5D), [Carolien de Kovel](https://pubmed.ncbi.nlm.nih.gov/?term=%22de%20Kovel%20C%22%5BAuthor%5D), [David B. Goldstein](https://pubmed.ncbi.nlm.nih.gov/?term=%22Goldstein%20DB%22%5BAuthor%5D), [Slavé Petrovski](https://pubmed.ncbi.nlm.nih.gov/?term=%22Petrovski%20S%22%5BAuthor%5D), [Rodney A. Radtke](https://pubmed.ncbi.nlm.nih.gov/?term=%22Radtke%20RA%22%5BAuthor%5D), [Nicole Walley](https://pubmed.ncbi.nlm.nih.gov/?term=%22Walley%20N%22%5BAuthor%5D), [J. Eunice Zhang](https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20JE%22%5BAuthor%5D), [Munir Pirmohamed](https://pubmed.ncbi.nlm.nih.gov/?term=%22Pirmohamed%20M%22%5BAuthor%5D), [Galen E.B. Wright](https://pubmed.ncbi.nlm.nih.gov/?term=%22Wright%20GEB%22%5BAuthor%5D), [Colin J.D. Ross](https://pubmed.ncbi.nlm.nih.gov/?term=%22Ross%20CJD%22%5BAuthor%5D), [Bruce C. Carleton](https://pubmed.ncbi.nlm.nih.gov/?term=%22Carleton%20BC%22%5BAuthor%5D), [Michael Hayden](https://pubmed.ncbi.nlm.nih.gov/?term=%22Hayden%20M%22%5BAuthor%5D), [Stuart MacLeod](https://pubmed.ncbi.nlm.nih.gov/?term=%22MacLeod%20S%22%5BAuthor%5D), [Reza Ghannadan](https://pubmed.ncbi.nlm.nih.gov/?term=%22Ghannadan%20R%22%5BAuthor%5D), [Shahrad Rod Rassekh](https://pubmed.ncbi.nlm.nih.gov/?term=%22Rassekh%20SR%22%5BAuthor%5D), [Henk Visscher](https://pubmed.ncbi.nlm.nih.gov/?term=%22Visscher%20H%22%5BAuthor%5D), [Folefac Aminkeng](https://pubmed.ncbi.nlm.nih.gov/?term=%22Aminkeng%20F%22%5BAuthor%5D), [Michelle Higginson](https://pubmed.ncbi.nlm.nih.gov/?term=%22Higginson%20M%22%5BAuthor%5D), [Nasim Massah](https://pubmed.ncbi.nlm.nih.gov/?term=%22Massah%20N%22%5BAuthor%5D), [Fudan Miao](https://pubmed.ncbi.nlm.nih.gov/?term=%22Miao%20F%22%5BAuthor%5D), [Adrienne Borrie](https://pubmed.ncbi.nlm.nih.gov/?term=%22Borrie%20A%22%5BAuthor%5D), [Ursula Amstutz](https://pubmed.ncbi.nlm.nih.gov/?term=%22Amstutz%20U%22%5BAuthor%5D), [Claudette Hildebrand](https://pubmed.ncbi.nlm.nih.gov/?term=%22Hildebrand%20C%22%5BAuthor%5D), [Shevaun Hughes](https://pubmed.ncbi.nlm.nih.gov/?term=%22Hughes%20S%22%5BAuthor%5D), [Kaitlyn Shaw](https://pubmed.ncbi.nlm.nih.gov/?term=%22Shaw%20K%22%5BAuthor%5D), [Satvir Dhoot](https://pubmed.ncbi.nlm.nih.gov/?term=%22Dhoot%20S%22%5BAuthor%5D), [Amit P Bhavsar](https://pubmed.ncbi.nlm.nih.gov/?term=%22Bhavsar%20AP%22%5BAuthor%5D), [Yuling Li](https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20Y%22%5BAuthor%5D), [Jong W Lee](https://pubmed.ncbi.nlm.nih.gov/?term=%22Lee%20JW%22%5BAuthor%5D), [Kaarina Kowalec](https://pubmed.ncbi.nlm.nih.gov/?term=%22Kowalec%20K%22%5BAuthor%5D), [Jessica Stortz](https://pubmed.ncbi.nlm.nih.gov/?term=%22Stortz%20J%22%5BAuthor%5D), [Tessa Bendyshe-Walton](https://pubmed.ncbi.nlm.nih.gov/?term=%22Bendyshe-Walton%20T%22%5BAuthor%5D), [Mikaela Barker](https://pubmed.ncbi.nlm.nih.gov/?term=%22Barker%20M%22%5BAuthor%5D), [Duncan Waltrip](https://pubmed.ncbi.nlm.nih.gov/?term=%22Waltrip%20D%22%5BAuthor%5D), [Rachel Bader](https://pubmed.ncbi.nlm.nih.gov/?term=%22Bader%20R%22%5BAuthor%5D), [Britt Drögemöller](https://pubmed.ncbi.nlm.nih.gov/?term=%22Dr%C3%B6gem%C3%B6ller%20B%22%5BAuthor%5D), [Nicole McGoldrick](https://pubmed.ncbi.nlm.nih.gov/?term=%22McGoldrick%20N%22%5BAuthor%5D), [Cheri Nijssen-Jordan](https://pubmed.ncbi.nlm.nih.gov/?term=%22Nijssen-Jordan%20C%22%5BAuthor%5D), [David Johnson](https://pubmed.ncbi.nlm.nih.gov/?term=%22Johnson%20D%22%5BAuthor%5D), [Carnation Zhuwaki](https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhuwaki%20C%22%5BAuthor%5D), [Linda Verbeek](https://pubmed.ncbi.nlm.nih.gov/?term=%22Verbeek%20L%22%5BAuthor%5D), [Rick Kaczowka](https://pubmed.ncbi.nlm.nih.gov/?term=%22Kaczowka%20R%22%5BAuthor%5D), [Patti Stevenson](https://pubmed.ncbi.nlm.nih.gov/?term=%22Stevenson%20P%22%5BAuthor%5D), [Kevin Hall](https://pubmed.ncbi.nlm.nih.gov/?term=%22Hall%20K%22%5BAuthor%5D), [Nick Honcharik](https://pubmed.ncbi.nlm.nih.gov/?term=%22Honcharik%20N%22%5BAuthor%5D), [Geert 't Jong](https://pubmed.ncbi.nlm.nih.gov/?term=%22Jong%20G'%22%5BAuthor%5D), [Sara Israels](https://pubmed.ncbi.nlm.nih.gov/?term=%22Israels%20S%22%5BAuthor%5D), [Shanna Chan](https://pubmed.ncbi.nlm.nih.gov/?term=%22Chan%20S%22%5BAuthor%5D), [Byron Garnham](https://pubmed.ncbi.nlm.nih.gov/?term=%22Garnham%20B%22%5BAuthor%5D), [Michelle Staub](https://pubmed.ncbi.nlm.nih.gov/?term=%22Staub%20M%22%5BAuthor%5D), [Michael Rieder](https://pubmed.ncbi.nlm.nih.gov/?term=%22Rieder%20M%22%5BAuthor%5D), [Becky Malkin](https://pubmed.ncbi.nlm.nih.gov/?term=%22Malkin%20B%22%5BAuthor%5D), [Neil Shear](https://pubmed.ncbi.nlm.nih.gov/?term=%22Shear%20N%22%5BAuthor%5D), [Gideon Koren](https://pubmed.ncbi.nlm.nih.gov/?term=%22Koren%20G%22%5BAuthor%5D), [Shinya Ito](https://pubmed.ncbi.nlm.nih.gov/?term=%22Ito%20S%22%5BAuthor%5D), [Paul Nathan](https://pubmed.ncbi.nlm.nih.gov/?term=%22Nathan%20P%22%5BAuthor%5D), [Mark Greenberg](https://pubmed.ncbi.nlm.nih.gov/?term=%22Greenberg%20M%22%5BAuthor%5D), [Facundo Garcia Bournissen](https://pubmed.ncbi.nlm.nih.gov/?term=%22Bournissen%20FG%22%5BAuthor%5D), [Miho Inoue](https://pubmed.ncbi.nlm.nih.gov/?term=%22Inoue%20M%22%5BAuthor%5D), [Sachi Sakaguchi](https://pubmed.ncbi.nlm.nih.gov/?term=%22Sakaguchi%20S%22%5BAuthor%5D), [Toshihiro Tanaka](https://pubmed.ncbi.nlm.nih.gov/?term=%22Tanaka%20T%22%5BAuthor%5D), [Hisaki Fujii](https://pubmed.ncbi.nlm.nih.gov/?term=%22Fujii%20H%22%5BAuthor%5D), [Mina Ogawa](https://pubmed.ncbi.nlm.nih.gov/?term=%22Ogawa%20M%22%5BAuthor%5D), [Taro Kamiya](https://pubmed.ncbi.nlm.nih.gov/?term=%22Kamiya%20T%22%5BAuthor%5D), [Smita Karande](https://pubmed.ncbi.nlm.nih.gov/?term=%22Karande%20S%22%5BAuthor%5D), [Sholeh Ghayoori](https://pubmed.ncbi.nlm.nih.gov/?term=%22Ghayoori%20S%22%5BAuthor%5D), [Régis Vaillancourt](https://pubmed.ncbi.nlm.nih.gov/?term=%22Vaillancourt%20R%22%5BAuthor%5D), [Donna Johnston](https://pubmed.ncbi.nlm.nih.gov/?term=%22Johnston%20D%22%5BAuthor%5D), [Herpreet Mankoo](https://pubmed.ncbi.nlm.nih.gov/?term=%22Mankoo%20H%22%5BAuthor%5D), [Elaine Wong](https://pubmed.ncbi.nlm.nih.gov/?term=%22Wong%20E%22%5BAuthor%5D), [Brenda Wilson](https://pubmed.ncbi.nlm.nih.gov/?term=%22Wilson%20B%22%5BAuthor%5D), [Lauren O’Connor](https://pubmed.ncbi.nlm.nih.gov/?term=%22O%E2%80%99Connor%20L%22%5BAuthor%5D), [Caleb Hui](https://pubmed.ncbi.nlm.nih.gov/?term=%22Hui%20C%22%5BAuthor%5D), [Cindy Yuen](https://pubmed.ncbi.nlm.nih.gov/?term=%22Yuen%20C%22%5BAuthor%5D), [Andrea Walsh](https://pubmed.ncbi.nlm.nih.gov/?term=%22Walsh%20A%22%5BAuthor%5D), [Kaviyarasan Mahalingam](https://pubmed.ncbi.nlm.nih.gov/?term=%22Mahalingam%20K%22%5BAuthor%5D), [Jean-François Bussières](https://pubmed.ncbi.nlm.nih.gov/?term=%22Bussi%C3%A8res%20JF%22%5BAuthor%5D), [Denis Lebel](https://pubmed.ncbi.nlm.nih.gov/?term=%22Lebel%20D%22%5BAuthor%5D), [Pierre Barret](https://pubmed.ncbi.nlm.nih.gov/?term=%22Barret%20P%22%5BAuthor%5D), [Aurélie Clauson](https://pubmed.ncbi.nlm.nih.gov/?term=%22Clauson%20A%22%5BAuthor%5D), [Ève Courbon](https://pubmed.ncbi.nlm.nih.gov/?term=%22Courbon%20%C3%88%22%5BAuthor%5D), [Léna Cerruti](https://pubmed.ncbi.nlm.nih.gov/?term=%22Cerruti%20L%22%5BAuthor%5D), [Maud Harry](https://pubmed.ncbi.nlm.nih.gov/?term=%22Harry%20M%22%5BAuthor%5D), [Marine Aussedat](https://pubmed.ncbi.nlm.nih.gov/?term=%22Aussedat%20M%22%5BAuthor%5D), [Margaret Murray](https://pubmed.ncbi.nlm.nih.gov/?term=%22Murray%20M%22%5BAuthor%5D), [Marilyn Tiller](https://pubmed.ncbi.nlm.nih.gov/?term=%22Tiller%20M%22%5BAuthor%5D), and [Carol-anne Osborne](https://pubmed.ncbi.nlm.nih.gov/?term=%22Osborne%20CA%22%5BAuthor%5D)

## Affiliations

From Molecular and Cellular Therapeutics (M.M., N.D., G.L.C.) and FutureNeuro Research Centre (G.L.C.), Royal College of Surgeons in Ireland, Dublin; Centre for Genomic Sciences (H.G., S.C., L.B.), Li Ka Shing Faculty of Medicine, University of Hong Kong; deCODE Genetics/Amgen, Inc. (A.I., L.J.G., K.S.), Reykjavik, Iceland; UCL Genetics Institute (D.S.), University College London, UK; Centre for Molecular Medicine and Therapeutics, BC Children's Hospital Research Institute, Faculty of Medicine (G.E.B.W.), Departments of Pediatrics and Medical Genetics (C.J.D.R.), and Division of Translational Therapeutics, Department of Pediatrics (B.C.C.), University of British Columbia, Vancouver, Canada; Department of Molecular and Clinical Pharmacology (J.E.Z., G.J.S., A.G.M., P.A., M.P.), Institute of Translational Medicine, University of Liverpool, UK; Department of Medical Genetics (R.S., I.L.-C.), School of Medical Sciences, University of Campinas and the Brazilian Institute of Neuroscience and Neurotechnology (BRAINN); Department of Neurology (C.Y., F.C.), School of Medical Sciences, University of Campinas and the Brazilian Institute of Neuroscience and Neurotechnology (BRAINN), Campinas, SP, Brazil; Hertie Institute for Clinical Brain Research (M.K., H.L.), University of Tübingen, Germany; Department of Clinical and Experimental Epilepsy (S.W., F.B., S.R., A.A., K.H., C.L., S.M.S.) and NIHR University College London Hospitals Biomedical Research Centre (J.W.S.), UCL Institute of Neurology, London, UK; Laboratory of Experimental Neurology (C.D.), Hôpital Erasme, Université Libre de Bruxelles, Brussels, Belgium; The Walton Centre NHS Foundation Trust (A.G.M., P.A.), Liverpool; Epilepsy Unit (M.J.B.), West Glasgow ACH-Yorkhill; Department of Biostatistics (B.F.), Institute of Translational Medicine, University of Liverpool; Division of Brain Sciences (M.R.J.), Imperial College Faculty of Medicine, London, UK; Division of Neurosciences (B.P.C.K.) and Center for Molecular Medicine (M.M.), Universitair Medisch Centrum Utrecht, Netherlands; Pediatric Neurology and Muscular Diseases Unit, Department of Neurosciences, Rehabilitation, Ophthalmology, Genetics, Maternal and Child Health, University of Genoa (P.S.), and Laboratory of Neurogenetics and Neuroscience (F.Z.), Institute G. Gaslini; Department of Neuroscience (A.C.), Reproductive and Odontostomatological Sciences, Federico II University, Naples; Department of Epileptology (W.S.K.), University of Bonn, Germany; Stichting Epilepsie Instellingen Nederland (SEIN) (J.W.S.), Heemstede, Netherlands; The Chalfont Centre for Epilepsy (J.W.S., S.M.S.), Epilepsy Society, Chalfont St. Peters, Buckinghamshire, UK; Epilepsy Center Frankfurt Rhine-Main (K.M.K.), Department of Neurology, Center of Neurology and Neurosurgery, University Hospital, Goethe-University Frankfurt; Epilepsy Center Hessen (K.M.K.), Department of Neurology, University Hospitals Giessen & Marburg, and Philipps-University Marburg, Germany; Neurogenetics Group (S.W.), VIB-UAntwerp, Center for Molecular Neurology; Neurology Department (S.W.), University Hospital Antwerp, Belgium; Department of Neurology (M.K.), Medical University of Vienna, Austria; Faculty of Medicine (K.S.), University of Iceland, Reykjavik; Luxembourg Centre for Systems Biomedicine (R.K.), University of Luxembourg; University of Toronto (N.S.); Institute of Neuroscience and The Second Affiliated Hospital of Guangzhou Medical University (W.-p.L.), Key Laboratory of Neurogenetics and Channelopathies of Guangdong Province and the Ministry of Education of China, China; and Departments of Medicine and Neurology (T.J.O., P.K.), University of Melbourne, Royal Melbourne Hospital, Australia.

## Author contributions

M.M.C., H.G., L.B., S.C., P.K., and G.L.C. contributed to the conception and design of the study and the acquisition, analysis, and interpretation of data. M.M.C. and H.G. performed all statistical analysis. A.I., S.M.S., R.S., S.W., F.B., S.R., K.H., C.L., A.G.M., P.A., M.J.B., B.F., M.R.J., N.S., G.E.B.W., C.J.D., B.C.C., D.S., J.E.Z., M.K., M.P., A.A., C.D., G.J.S., B.P.C.K., P.S., F.Z., A.C., W.S.K., J.W.S., H.L., K.M.K., S.W., M.K., L.J.G., R.K., N.D., C.Y., F.C., I.L.-C., T.J.O., W.-p.L., L.J.G., and K.S. contributed to the acquisition, analysis, and interpretation of data. All authors contributed to the critical revision of the final version of the manuscript for important intellectual content.

## Study funding

This study was not industry-sponsored. The work was supported by a grant from the European Commission (7th Framework Programme Grant 279062, EpiPGX). M.M.C. and G.L.C. are supported by Science Foundation Ireland, grant 13/CDA/2223, and an RCSI seed funding grant GA 14-1899. This project was supported by the General Research Funds (HKU7623/08M and HKU7747/07M to S.C., CUHK4466/06M to P.K.) and Health and Medical Research Fund (HMRF 01120086 to P.K.) from Hong Kong. Some results presented in this article were prepared using the HPC facilities of the University of Luxembourg. This work was partly undertaken at UCLH/UCL, which received a proportion of funding from the Department of Health's NIHR Biomedical Research Centres funding scheme (J.W.S., S.M.S.). The work was also supported by the Epilepsy Society, UK (J.W.S., S.M.S.), by the foundation “no epilep,” the German Chapter of the ILAE (DGfE) (both to H.L.). F.C. and I.L.-C. are supported by Fundação de Amparo à Pesquisa do Estado de São Paulo, Brazil, through grant 2013/07559-3. J.E.Z. and M.P. thank the NHS Chair of Pharmacogenetics programme and MRC Centre for Drug Safety Science for support in Liverpool. B.C.C. and C.J.D.R. are supported by the Canadian Institutes of Health Research (CIHR) Drug Safety and Effectiveness Network (FRN-117588), the Canada Foundation for Innovation and the Canadian Dermatology Foundation. G.E.B.W. is supported by a CIHR Fellowship. The funders of the study had no role in the study design, data collection, data analysis, data interpretation, or writing of the report. M.M.C., H.G., and G.L.C. had full access to all the data in the study and the corresponding authors had final responsibility for the decision to submit for publication.

## Disclosure

M. McCormack and H. Gui report no disclosures relevant to the manuscript. A. Ingason is an employee of deCODE Genetics/Amgen. D. Speed, G. Wright, E. Zhang, R. Secolin, C. Yasuda, M. Kwok, S. Wolking, F. Becker, and S. Rau report no disclosures relevant to the manuscript. A. Avbersek is employed by UCB Pharma SPRL, Belgium, as Associate Director. K. Heggeli, C. Leu, C. Depondt, and G. Sills report no disclosures relevant to the manuscript. A. Marson was awarded grants from GSK, Eisai, and UCB Pharma, which funded the National Audit of Seizure Management in Hospitals. P. Auce, M. Brodie, B. Francis, M. Johnson, B. Koeleman, P. Striano, A. Coppola, F. Zara, and W. Kunz report no disclosures relevant to the manuscript. J. Sander has served on scientific advisory boards for UCB Pharma and Eisai; has served on speaker's bureaus for UCB Pharma, Eisai, Teva, and Lundbeck; has received research support from UCB Pharma, GSK, Eisai, The Marvin Weil Epilepsy Research Fund, and NL Nationaal Epilepsie Fonds; and his current position is endowed by the UK Epilepsy Society. H. Lerche has received speaker or consultancy fees or travel support from Bial, Desitin, Eisai, GlaxoSmithKline, Pfizer, UCB Pharma, or Valeant. K. Klein reports personal fees from UCB Pharma, Novartis Pharma AG, and Eisai, outside of the submitted work. AC was awarded a grant from EISAI and personal fees for speaking from Eisai, outside of the submitted work. S. Weckhuysen and M. Krenn report no disclosures relevant to the manuscript. L. Gudmundsson is an employee of deCODE Genetics/Amgen. K. Stefánsson is an employee of deCODE Genetics/Amgen. R. Krause, N. Shear, C. Ross, and N. Delanty report no disclosures relevant to the manuscript. M. Pirmohamed reports grants from MRC and grants from UK Department of Health during the conduct of the study. B. Carleton, through the Pharmaceutical Outcomes Programme (POPi), has received financial support for its pharmacogenomics research from the Canada Foundation for Innovation (CFI), Canadian Institutes of Health Research, Genome Canada, Genome British Columbia, and the Provincial Health Services Authority. Prof. Carelton has also received support by the University of British Columbia, Child & Family Research Institute, Vancouver, and Pfizer. Prof. Carleton has a patent and applications pending for biomarkers of anthracycline-induced cardiotoxicity and cisplatin-induced ototoxicity but no relevant financial disclosures relevant to this study. F. Cendes reports speaking fees from UCB Pharma, outside of the current work. I. Cendes-Lopes and W. Liao report no disclosures relevant to the manuscript. T. O'Brien reports grants from The Royal Melbourne Foundation during the conduct of the study. S. Sisodiya and S. Cherny report no disclosures relevant to the manuscript. P. Kwan has received speaker or consultancy fees and/or research grants from Eisai, GlaxoSmithKline, Johnson & Johnson, Pfizer, and UCB Pharma. L. Baum and G. Cavalleri report no disclosures relevant to the manuscript. Go to [Neurology.org/N](http://n.neurology.org/lookup/doi/10.1212/WNL.0000000000004853) for full disclosures.

## References

1. Li X, Yu K, Mei S, et al. HLA-B*1502 increases the risk of phenytoin or lamotrigine induced Stevens-Johnson syndrome/toxic epidermal necrolysis: evidence from a meta-analysis of nine case-control studies. Drug Res 2015;65:107–111.  [DOI](https://doi.org/10.1055/s-0034-1375684) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24871931/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Res&title=HLA-B*1502%20increases%20the%20risk%20of%20phenytoin%20or%20lamotrigine%20induced%20Stevens-Johnson%20syndrome/toxic%20epidermal%20necrolysis:%20evidence%20from%20a%20meta-analysis%20of%20nine%20case-control%20studies&volume=65&publication_year=2015&pages=107-111&pmid=24871931&doi=10.1055/s-0034-1375684&)

2. Chung WH, Hung SI, Hong HS, et al. Medical genetics: a marker for Stevens-Johnson syndrome. Nature 2004;428:486.  [DOI](https://doi.org/10.1038/428486a) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15057820/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Medical%20genetics:%20a%20marker%20for%20Stevens-Johnson%20syndrome&volume=428&publication_year=2004&pages=486&pmid=15057820&doi=10.1038/428486a&)

3. Man CB, Kwan P, Baum L, et al. Association between HLA-B*1502 allele and antiepileptic drug-induced cutaneous reactions in Han Chinese. Epilepsia 2007;48:1015–1018.  [DOI](https://doi.org/10.1111/j.1528-1167.2007.01022.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17509004/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Epilepsia&title=Association%20between%20HLA-B*1502%20allele%20and%20antiepileptic%20drug-induced%20cutaneous%20reactions%20in%20Han%20Chinese&volume=48&publication_year=2007&pages=1015-1018&pmid=17509004&doi=10.1111/j.1528-1167.2007.01022.x&)

4. Lonjou C, Thomas L, Borot N, et al. A marker for Stevens-Johnson syndrome: ethnicity matters. Pharmacogenomics J 2006;6:265–268.  [DOI](https://doi.org/10.1038/sj.tpj.6500356) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16415921/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=A%20marker%20for%20Stevens-Johnson%20syndrome:%20ethnicity%20matters&volume=6&publication_year=2006&pages=265-268&pmid=16415921&doi=10.1038/sj.tpj.6500356&)

5. Amstutz U, Ross CJ, Castro-Pastrana LI, et al. HLA-A 31:01 and HLA-B 15:02 as genetic markers for carbamazepine hypersensitivity in children. Clin Pharmacol Ther 2013;94:142–149.  [DOI](https://doi.org/10.1038/clpt.2013.55) | [PMC free article](/articles/PMC3839910/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23588310/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=HLA-A%2031:01%20and%20HLA-B%2015:02%20as%20genetic%20markers%20for%20carbamazepine%20hypersensitivity%20in%20children&volume=94&publication_year=2013&pages=142-149&pmid=23588310&doi=10.1038/clpt.2013.55&)

6. Kim SH, Lee KW, Song WJ, et al. Carbamazepine-induced severe cutaneous adverse reactions and HLA genotypes in Koreans. Epilepsy Res 2011;97:190–197.  [DOI](https://doi.org/10.1016/j.eplepsyres.2011.08.010) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21917426/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Epilepsy%20Res&title=Carbamazepine-induced%20severe%20cutaneous%20adverse%20reactions%20and%20HLA%20genotypes%20in%20Koreans&volume=97&publication_year=2011&pages=190-197&pmid=21917426&doi=10.1016/j.eplepsyres.2011.08.010&)

7. Ozeki T, Mushiroda T, Yowang A, et al. Genome-wide association study identifies HLA-A*3101 allele as a genetic risk factor for carbamazepine-induced cutaneous adverse drug reactions in Japanese population. Hum Mol Genet 2011;20:1034–1041.  [DOI](https://doi.org/10.1093/hmg/ddq537) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21149285/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mol%20Genet&title=Genome-wide%20association%20study%20identifies%20HLA-A*3101%20allele%20as%20a%20genetic%20risk%20factor%20for%20carbamazepine-induced%20cutaneous%20adverse%20drug%20reactions%20in%20Japanese%20population&volume=20&publication_year=2011&pages=1034-1041&pmid=21149285&doi=10.1093/hmg/ddq537&)

8. McCormack M, Alfirevic A, Bourgeois S, et al. HLA-A*3101 and carbamazepine-induced hypersensitivity reactions in Europeans. N Engl J Med 2011;364:1134–1143.  [DOI](https://doi.org/10.1056/NEJMoa1013297) | [PMC free article](/articles/PMC3113609/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21428769/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=HLA-A*3101%20and%20carbamazepine-induced%20hypersensitivity%20reactions%20in%20Europeans&volume=364&publication_year=2011&pages=1134-1143&pmid=21428769&doi=10.1056/NEJMoa1013297&)

9. Shi YW, Min FL, Zhou D, et al. HLA-A*24:02 as a common risk factor for antiepileptic drug-induced cutaneous adverse reactions. Neurology 2017;88:2183–2191.  [DOI](https://doi.org/10.1212/WNL.0000000000004008) | [PMC free article](/articles/PMC5467955/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28476759/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neurology&title=HLA-A*24:02%20as%20a%20common%20risk%20factor%20for%20antiepileptic%20drug-induced%20cutaneous%20adverse%20reactions&volume=88&publication_year=2017&pages=2183-2191&pmid=28476759&doi=10.1212/WNL.0000000000004008&)

10. Chung WH, Chang WC, Lee YS, et al. Genetic variants associated with phenytoin-related severe cutaneous adverse reactions. JAMA 2014;312:525–534.  [DOI](https://doi.org/10.1001/jama.2014.7859) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25096692/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA&title=Genetic%20variants%20associated%20with%20phenytoin-related%20severe%20cutaneous%20adverse%20reactions&volume=312&publication_year=2014&pages=525-534&pmid=25096692&doi=10.1001/jama.2014.7859&)

11. Tassaneeyakul W, Prabmeechai N, Sukasem C, et al. Associations between HLA class I and cytochrome P450 2C9 genetic polymorphisms and phenytoin-related severe cutaneous adverse reactions in a Thai population. Pharmacogenet Genomics 2016;26:225–234.  [DOI](https://doi.org/10.1097/FPC.0000000000000211) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26928377/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Associations%20between%20HLA%20class%20I%20and%20cytochrome%20P450%202C9%20genetic%20polymorphisms%20and%20phenytoin-related%20severe%20cutaneous%20adverse%20reactions%20in%20a%20Thai%20population&volume=26&publication_year=2016&pages=225-234&pmid=26928377&doi=10.1097/FPC.0000000000000211&)

12. McCormack M, Urban TJ, Shianna KV, et al. Genome-wide mapping for clinically relevant predictors of lamotrigine- and phenytoin-induced hypersensitivity reactions. Pharmacogenomics 2012;13:399–405.  [DOI](https://doi.org/10.2217/pgs.11.165) | [PMC free article](/articles/PMC3428903/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22379998/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Genome-wide%20mapping%20for%20clinically%20relevant%20predictors%20of%20lamotrigine-%20and%20phenytoin-induced%20hypersensitivity%20reactions&volume=13&publication_year=2012&pages=399-405&pmid=22379998&doi=10.2217/pgs.11.165&)

13. International League Against Epilepsy Consortium on Complex Epilepsies. Genetic determinants of common epilepsies: a meta-analysis of genome-wide association studies. Lancet Neurol 2014;13:893–903.  [DOI](https://doi.org/10.1016/S1474-4422(14)70171-1) | [PMC free article](/articles/PMC4189926/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25087078/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet%20Neurol&title=Genetic%20determinants%20of%20common%20epilepsies:%20a%20meta-analysis%20of%20genome-wide%20association%20studies&volume=13&publication_year=2014&pages=893-903&pmid=25087078&doi=10.1016/S1474-4422(14)70171-1&)

14. Carleton B, Poole R, Smith M, et al. Adverse drug reaction active surveillance: developing a national network in Canada's children's hospitals. Pharmacoepidemiol Drug Saf 2009;18:713–721.  [DOI](https://doi.org/10.1002/pds.1772) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19507171/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacoepidemiol%20Drug%20Saf&title=Adverse%20drug%20reaction%20active%20surveillance:%20developing%20a%20national%20network%20in%20Canada's%20children's%20hospitals&volume=18&publication_year=2009&pages=713-721&pmid=19507171&doi=10.1002/pds.1772&)

15. Marchini J, Howie B, Myers S, McVean G, Donnelly P. A new multipoint method for genome-wide association studies by imputation of genotypes. Nat Genet 2007;39:906–913.  [DOI](https://doi.org/10.1038/ng2088) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17572673/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=A%20new%20multipoint%20method%20for%20genome-wide%20association%20studies%20by%20imputation%20of%20genotypes&volume=39&publication_year=2007&pages=906-913&pmid=17572673&doi=10.1038/ng2088&)

16. Willer CJ, Li Y, Abecasis GR. METAL: fast and efficient meta-analysis of genomewide association scans. Bioinformatics 2010;26:2190–2191.  [DOI](https://doi.org/10.1093/bioinformatics/btq340) | [PMC free article](/articles/PMC2922887/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20616382/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=METAL:%20fast%20and%20efficient%20meta-analysis%20of%20genomewide%20association%20scans&volume=26&publication_year=2010&pages=2190-2191&pmid=20616382&doi=10.1093/bioinformatics/btq340&)

17. Lek M, Karczewski KJ, Minikel EV, et al. Analysis of protein-coding genetic variation in 60,706 humans. Nature 2016;536:285–291.  [DOI](https://doi.org/10.1038/nature19057) | [PMC free article](/articles/PMC5018207/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27535533/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Analysis%20of%20protein-coding%20genetic%20variation%20in%2060,706%20humans&volume=536&publication_year=2016&pages=285-291&pmid=27535533&doi=10.1038/nature19057&)

18. Beaudoin M, Goyette P, Boucher G, et al. Deep resequencing of GWAS loci identifies rare variants in CARD9, IL23R and RNF186 that are associated with ulcerative colitis. PLoS Genet 2013;9:e1003723.  [DOI](https://doi.org/10.1371/journal.pgen.1003723) | [PMC free article](/articles/PMC3772057/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24068945/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet&title=Deep%20resequencing%20of%20GWAS%20loci%20identifies%20rare%20variants%20in%20CARD9,%20IL23R%20and%20RNF186%20that%20are%20associated%20with%20ulcerative%20colitis&volume=9&publication_year=2013&pages=e1003723&pmid=24068945&doi=10.1371/journal.pgen.1003723&)

19. Bonomo JA, Palmer ND, Hicks PJ, et al. Complement factor H gene associations with end-stage kidney disease in African Americans. Nephrol Dial Transpl 2014;29:1409–1414.  [DOI](https://doi.org/10.1093/ndt/gfu036) | [PMC free article](/articles/PMC4081633/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24586071/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nephrol%20Dial%20Transpl&title=Complement%20factor%20H%20gene%20associations%20with%20end-stage%20kidney%20disease%20in%20African%20Americans&volume=29&publication_year=2014&pages=1409-1414&pmid=24586071&doi=10.1093/ndt/gfu036&)

20. Skerka C, Chen Q, Fremeaux-Bacchi V, Roumenina LT. Complement factor H related proteins (CFHRs). Mol Immunol 2013;56:170–180.  [DOI](https://doi.org/10.1016/j.molimm.2013.06.001) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23830046/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Immunol&title=Complement%20factor%20H%20related%20proteins%20(CFHRs)&volume=56&publication_year=2013&pages=170-180&pmid=23830046&doi=10.1016/j.molimm.2013.06.001&)

21. Fritsche LG, Igl W, Bailey JN, et al. A large genome-wide association study of age-related macular degeneration highlights contributions of rare and common variants. Nat Genet 2016;48:134–143.  [DOI](https://doi.org/10.1038/ng.3448) | [PMC free article](/articles/PMC4745342/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26691988/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=A%20large%20genome-wide%20association%20study%20of%20age-related%20macular%20degeneration%20highlights%20contributions%20of%20rare%20and%20common%20variants&volume=48&publication_year=2016&pages=134-143&pmid=26691988&doi=10.1038/ng.3448&)

22. Basaran N, Kansu E, Hincal F. Serum immunoglobulins, complement levels and lymphocyte subpopulations in phenytoin-treated epileptic patients. Immunopharmacol Immunotoxicol 1989;11:335–346.  [DOI](https://doi.org/10.3109/08923978909005374) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/2621321/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Immunopharmacol%20Immunotoxicol&title=Serum%20immunoglobulins,%20complement%20levels%20and%20lymphocyte%20subpopulations%20in%20phenytoin-treated%20epileptic%20patients&volume=11&publication_year=1989&pages=335-346&pmid=2621321&doi=10.3109/08923978909005374&)

23. The selection and use of essential medicines. World Health Organ Tech Rep Ser 2015: vii–xv:1–546.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/27183787/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=World%20Health%20Organ%20Tech%20Rep%20Ser&title=The%20selection%20and%20use%20of%20essential%20medicines&publication_year=2015&pages=vii-xv&pmid=27183787&)

24. Nicholas JM, Ridsdale L, Richardson MP, Ashworth M, Gulliford MC. Trends in antiepileptic drug utilisation in UK primary care 1993-2008: cohort study using the General Practice Research Database. Seizure 2012;21:466–470.  [DOI](https://doi.org/10.1016/j.seizure.2012.04.014) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22608976/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Seizure&title=Trends%20in%20antiepileptic%20drug%20utilisation%20in%20UK%20primary%20care%201993-2008:%20cohort%20study%20using%20the%20General%20Practice%20Research%20Database&volume=21&publication_year=2012&pages=466-470&pmid=22608976&doi=10.1016/j.seizure.2012.04.014&)

25. Baulac M, de Boer H, Elger C, et al. Epilepsy priorities in Europe: a report of the ILAE-IBE Epilepsy Advocacy Europe Task Force. Epilepsia 2015;56:1687–1695.  [DOI](https://doi.org/10.1111/epi.13201) | [PMC free article](/articles/PMC5019256/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26415919/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Epilepsia&title=Epilepsy%20priorities%20in%20Europe:%20a%20report%20of%20the%20ILAE-IBE%20Epilepsy%20Advocacy%20Europe%20Task%20Force&volume=56&publication_year=2015&pages=1687-1695&pmid=26415919&doi=10.1111/epi.13201&)

26. Chen Z, Liew D, Kwan P. Real-world efficiency of pharmacogenetic screening for carbamazepine-induced severe cutaneous adverse reactions. PLoS One 2014;9:e96990.  [DOI](https://doi.org/10.1371/journal.pone.0096990) | [PMC free article](/articles/PMC4013087/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24806465/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=Real-world%20efficiency%20of%20pharmacogenetic%20screening%20for%20carbamazepine-induced%20severe%20cutaneous%20adverse%20reactions&volume=9&publication_year=2014&pages=e96990&pmid=24806465&doi=10.1371/journal.pone.0096990&)
