# Genome‐Wide Association Approach Identified Novel Genetic Predictors of Heart Rate Response to β‐Blockers

## Metadata
**Authors:** Mohamed H Shahin, Daniela J Conrado, Daniel Gonzalez, Yan Gong, Maximilian T Lobmeyer, Amber L Beitelshees, Eric Boerwinkle, John G Gums, Arlene Chapman, Stephen T Turner, Rhonda M Cooper‐DeHoff, Julie A Johnson
**Journal:** Journal of the American Heart Association: Cardiovascular and Cerebrovascular Disease
**Date:** 2018 Feb 24
**DOI:** [10.1161/JAHA.117.006463](https://doi.org/10.1161/JAHA.117.006463)
**PMID:** 29478026
**PMCID:** PMC5866313
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5866313/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC5866313/pdf/JAH3-7-e006463.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC5866313/pdf/JAH3-7-e006463.pdf)

## Abstract

**Background:** 
For many indications, the negative chronotropic effect of β‐blockers is important to their efficacy, yet the heart rate (HR) response to β‐blockers varies. Herein, we sought to use a genome‐wide association approach to identify novel single nucleotide polymorphisms (SNPs) associated with HR response to β‐blockers.

**Methods and Results:** 
We first performed 4 genome‐wide association analyses for HR response to atenolol (a β1‐adrenergic receptor blocker) as: (1) monotherapy or (2) add‐on therapy, in 426 whites and 273 blacks separately from the PEAR (Pharmacogenomic Evaluation of Antihypertensive Responses) study. A meta‐analysis was then performed between the genome‐wide association analysis performed in PEAR atenolol monotherapy and add‐on therapy, in each race separately, using the inverse variance method assuming fixed effects. From this analysis, SNPs associated with HR response to atenolol at a P<1E‐05 were tested for replication in whites (n=200) and blacks (n=168) treated with metoprolol (a β1‐adrenergic receptor blocker). From the genome‐wide association meta‐analyses, SNP rs17117817 near olfactory receptor family10 subfamily‐p‐member1 (OR10P1), and SNP rs2364349 in sorting nexin‐9 (SNX9) replicated in blacks. The combined studies meta‐analysis P values for the rs17117817 and rs2364349 reached genome‐wide significance (rs17117817G‐allele; Meta‐β=5.53 beats per minute, Meta‐P=2E‐09 and rs2364349 A‐allele; Meta‐β=3.5 beats per minute, Meta‐P=1E‐08). Additionally, SNPs in the OR10P1 and SNX9 gene regions were also associated with HR response in whites.

**Conclusions:** 
This study highlights OR10P1 and SNX9 as novel genes associated with changes in HR in response to β‐blockers.

**Clinical Trial Registration:** 
URL: http://www.clinicaltrials.gov. Unique identifier: NCT00246519.

Keywords: atenolol, heart rate, metoprolol, pharmacogenomics, β‐blockers

Subject Categories: Genetic, Association Studies; Cardiovascular Disease; Hypertension

### Background

For many indications, the negative chronotropic effect of β‐blockers is important to their efficacy, yet the heart rate (HR) response to β‐blockers varies. Herein, we sought to use a genome‐wide association approach to identify novel single nucleotide polymorphisms (SNPs) associated with HR response to β‐blockers.

### Methods and Results

We first performed 4 genome‐wide association analyses for HR response to atenolol (a β1‐adrenergic receptor blocker) as: (1) monotherapy or (2) add‐on therapy, in 426 whites and 273 blacks separately from the PEAR (Pharmacogenomic Evaluation of Antihypertensive Responses) study. A meta‐analysis was then performed between the genome‐wide association analysis performed in PEAR atenolol monotherapy and add‐on therapy, in each race separately, using the inverse variance method assuming fixed effects. From this analysis, SNPs associated with HR response to atenolol at a *P*<1E‐05 were tested for replication in whites (n=200) and blacks (n=168) treated with metoprolol (a β1‐adrenergic receptor blocker). From the genome‐wide association meta‐analyses, SNP rs17117817 near olfactory receptor family10 subfamily‐p‐member1 (*OR10P1*), and SNP rs2364349 in sorting nexin‐9 (*SNX9*) replicated in blacks. The combined studies meta‐analysis *P* values for the rs17117817 and rs2364349 reached genome‐wide significance (rs17117817G‐allele; Meta‐β=5.53 beats per minute, Meta‐*P*=2E‐09 and rs2364349 A‐allele; Meta‐β=3.5 beats per minute, Meta‐*P*=1E‐08). Additionally, SNPs in the *OR10P1* and *SNX9* gene regions were also associated with HR response in whites.

### Conclusions

This study highlights *OR10P1* and *SNX9* as novel genes associated with changes in HR in response to β‐blockers.

### Clinical Trial Registration

URL: [http://www.clinicaltrials.gov](http://www.clinicaltrials.gov). Unique identifier: NCT00246519.

## Clinical Perspective

## Clinical Perspective

### What Is New?

### What Are the Clinical Implications?

Over the past 5 decades, β‐adrenergic receptor blockers (β‐blockers) have been a cornerstone therapy for heart failure, post–myocardial infarction, symptomatic angina, and other cardiovascular diseases.[1](#jah32988-bib-0001), [2](#jah32988-bib-0002) Additionally, they remain one of the most commonly prescribed classes of drugs in the United States.[3](#jah32988-bib-0003) β‐Blockers work by inhibiting the β‐adrenergic receptors in the heart, which prevent the binding of epinephrine and norepinephrine to these receptors and eventually reduce cardiac contractility and heart rate (HR).[4](#jah32988-bib-0004) HR lowering in response to β‐blockers has been associated with lower risk of incident heart failure and cardiovascular diseases.[5](#jah32988-bib-0005), [6](#jah32988-bib-0006) Additionally, results from clinical trials and meta‐analyses have shown a significant association between β‐blockers’ HR‐lowering effect and improvement in clinical outcomes.[7](#jah32988-bib-0007), [8](#jah32988-bib-0008), [9](#jah32988-bib-0009), [10](#jah32988-bib-0010) Altogether, these data highlight that the HR‐modulating effects of β‐blockers are a critical efficacy component for this class of drugs and contributes to their beneficial therapeutic effects.

Despite the widespread use of β‐blockers, interindividual differences in HR‐lowering response to β‐blockers have been observed.[11](#jah32988-bib-0011), [12](#jah32988-bib-0012) These differences might be attributed to several factors, including genetics.[13](#jah32988-bib-0013) Over the past 2 decades, considerable resources have been allocated to elucidate the genetic contributors of variability in drug response to β‐blockers. Most of this research has focused on explaining differences in their effects on blood pressure, heart failure phenotypes, and effects on cardiovascular outcomes.[14](#jah32988-bib-0014), [15](#jah32988-bib-0015), [16](#jah32988-bib-0016), [17](#jah32988-bib-0017), [18](#jah32988-bib-0018) However, only a few candidate gene pharmacogenetic studies have focused on evaluating the role of genetics in explaining the differences in β‐blockers’ negative chronotropic response. Not surprisingly, *ADRB1*, the gene encoding the β_1_ receptor, has been the focus of most studies evaluating variability in negative chronotropic response to β‐blockers.[12](#jah32988-bib-0012), [19](#jah32988-bib-0019), [20](#jah32988-bib-0020), [21](#jah32988-bib-0021), [22](#jah32988-bib-0022), [23](#jah32988-bib-0023), [24](#jah32988-bib-0024) However, there is a paucity of information about other potential genes that may be associated with changes in β‐blockers’ negative chronotropic response.

Since autonomic regulation of HR is dependent on a vast array of proteins, change in HR in response to a β‐blocker may be controlled via multiple genes. Therefore, a thorough understanding of genetic determinants associated with changes in HR in response to a β‐blocker is needed. Accordingly, in this study, we sought to conduct a genome‐wide association analysis to identify novel genetic predictors associated with changes in HR in response to β‐blockers therapy in European‐Americans (whites) and African‐Americans (blacks).

## Methods

### Study Design and Participants

#### PEAR study

A description of the Pharmacogenomic Evaluation of Antihypertensive Responses (PEAR) study (clinicaltrials.gov #[NCT00246519](https://clinicaltrials.gov/ct2/show/NCT00246519)), including the study rationale, design, protocol, and safety procedures, has been previously published.[25](#jah32988-bib-0025) PEAR genotype and phenotype data used in this study have been made publicly available at the database of Genotypes and Phenotypes (dbGaP; [https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000649.v1.p1](https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000649.v1.p1); dbgap study accession: phs000649.v1.p1). Briefly, PEAR was a randomized, open‐label, multicenter, prospective study designed to evaluate the role of genetic variation on the blood pressure response to the β‐blocker atenolol and the thiazide diuretic hydrochlorothiazide (HCTZ). In total, 768 individuals with mild‐to‐moderate essential hypertension, of any race or ethnicity, and between the ages of 17 and 65 years completed the PEAR study. Following a washout period and collection of baseline data, study participants were randomized to either atenolol 50 mg or HCTZ 12.5 mg, both administered once daily, with the dose doubled (ie, atenolol 100 mg or HCTZ 25 mg) in patients with a systolic blood pressure >120 mm Hg and/or diastolic blood pressure >70 mm Hg. Responses to treatment were assessed after 6 to 9 weeks of monotherapy (response assessment #1; Figure [1](#jah32988-fig-0001)), after which those subjects who were still not at goal had the alternate drug added with a similar dose titration scheme and a response assessment following 6 to 9 weeks on combination therapy (response assessment #2). In PEAR, >85% of the participants treated with atenolol as a mono‐ or add‐on therapy had atenolol dose titrated to 100 mg per day. Data collected included home, office, and 24‐hour ambulatory systolic blood pressure, diastolic blood pressure, and HR. Both HR measurement and blood pressure response were used to guide dosage adjustments, as those individuals with HR <55 beats per minute were precluded from receiving a higher dose of atenolol. Institutional Review Boards at each clinical research center approved the study protocol (University of Florida, Gainesville, FL; Emory University, Atlanta, GA; Mayo Clinic, Rochester, MN) and written informed consent was obtained from all participants.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3947/5866313/6129457237fd/JAH3-7-e006463-g001.jpg)

PEAR study design. ATEN indicates atenolol; HCTZ, hydrochlorothiazide; HR, heart rate; PEAR, Pharmacogenomics Evaluation of Antihypertensive Response.

#### PEAR‐2 study

PEAR‐2 was a multicenter, prospective, open‐label, sequential monotherapy clinical trial (clinicaltrials.gov #[NCT01203852](https://clinicaltrials.gov/ct2/show/NCT01203852)).[26](#jah32988-bib-0026) The PEAR‐2 genotype and phenotype data used in this study are currently in the process of being uploaded to dbGaP and will soon be available to other researchers under dbGaP accession phs000649.v2.p1. The Institutional Review Boards of each enrolling institution (University of Florida, Gainesville, FL; Emory University, Atlanta, GA; Mayo Clinic, Rochester, MN) approved the study and all subjects provided written informed consent. Eligible study participants received metoprolol tartrate 50 mg twice daily, and after 2 weeks if an inadequate response was observed (>120/70 mm Hg and HR >55 bpm), the dose was increased to 100 mg twice daily for an additional 6 weeks. Of note, >95% of the participants treated with metoprolol had their dose titrated to 100 mg twice day. Metoprolol and atenolol are both β1‐adrenergic receptor blockers, and they have the same mechanism of reducing HR via their inhibitory effect on the β1‐adrenergic receptor. Thus, we used data from PEAR‐2 metoprolol‐treated participants for replicating the results discovered from PEAR atenolol‐treated participants.

### Heart Rate Assessment

In both the PEAR and PEAR‐2 studies, HR data were generated from measurements collected using a home blood pressure monitor, the Microlife model 3AC1‐PC home blood pressure monitor (Minneapolis, MN), which has been validated for accuracy.[27](#jah32988-bib-0027) Blood pressure and HR were measured in triplicate mode in the morning and evening, averaged by the blood pressure, and recorded with a date and time stamp. At each assessment period, study participants were required to have 5 morning and evening recordings in the previous 7 days. Thus, HR values used in the analysis described herein were based on a minimum of 30 HR recordings. Baseline HR was measured at the end of an average 4‐week washout of antihypertensive drugs (HR_1_). HR response following monotherapy with atenolol or HCTZ was measured after 6 to 9 weeks (HR_2_), and HR response following addition of atenolol to HCTZ was measured after 6 to 9 weeks of combination therapy (HR_3_). In PEAR‐2, the HR response to metoprolol was noted as HR_2_ and the baseline HR was HR_1_.

### Genetic Analyses and Genotyping

#### Genome‐wide association study approach

Details of the genome‐wide genotyping, quality control, and imputation performed on PEAR samples were previously described.[28](#jah32988-bib-0028) In brief, PEAR DNA samples underwent genotyping using the Illumina Human Omni‐1Million Quad BeadChip (Illumina, San Diego CA). Genotypes were called using GenTrain2 Illumina clustering algorithm in the software package GenomeStudio (Illumina, San Diego, CA). MaCH software (version 1.0.16) was used to impute single nucleotide polymorphisms (SNPs) based on HapMapIII haplotypes. SNPs with minor allele frequency <5% or imputation *r* ^2^ <0.3 were excluded from the analysis. After quality control, we had ≈1.1 million SNPs that were included in the genome‐wide association discovery analyses in white participants in PEAR monotherapy and PEAR add‐on therapy and ≈1.2 million SNPs in blacks. For PEAR‐2, genotyping was conducted using Human Omni2.5 S BeadChip (Illumina), and imputation based on 1000 Genomes Phase I reference panels was performed using Minimac.[29](#jah32988-bib-0029) Participants from PEAR or PEAR‐2 were excluded if sample genotype call rates were below 95%. Sample contamination was tested by checking sex mismatches using X chromosome genotype data, and those who were discordant were excluded. Additionally, cryptic relatedness was estimated by pairwise identity‐by‐descent analysis implemented using PLINK.[30](#jah32988-bib-0030) We also ran principal component analysis on all samples, using the EIGENSTRAT method,[30](#jah32988-bib-0030) to assess the ancestral background. Heterozygosity was also assessed using PLINK, by estimating the inbreeding coefficient. After imputing PEAR2 genotyping data and running quality control procedures, as explained above, we had ≈6.5 million SNPs in whites and ≈8.9 million SNPs in blacks treated with metoprolol therapy, which were used for the replication efforts in this study.

The analysis approach used in this study is summarized in Figure [2](#jah32988-fig-0002). First, we aimed to identify new genetic variants that contribute to the observed interindividual variability in HR‐lowering effect in response to atenolol by conducting a genome‐wide association study (GWAS). A meta‐analysis was then performed, in each race separately, between PEAR atenolol monotherapy and PEAR atenolol add‐on therapy. To replicate our findings, SNPs with *P*<1E‐05 from the PEAR meta‐analysis were evaluated for association with changes in HR in response to metoprolol in PEAR‐2. Given the fact that patterns of linkage disequilibrium often differ across different ancestral groups, further validation of SNPs of interest was performed by looking up regions of the SNPs of interest in the alternate ancestral group.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3947/5866313/e839c437eb0a/JAH3-7-e006463-g002.jpg)

The overall analysis framework of the study. GWAS indicates genome‐wide association study; PEAR, Pharmacogenomics Evaluation of Antihypertensive Response; SNP, single nucleotide polymorphism.

### Statistical Analysis

In PEAR, the change in HR in response to atenolol was determined as follows (Figure [1](#jah32988-fig-0001)). For the monotherapy, HR_1_ was subtracted from HR_2_ (HRΔ_21_); for the add‐on therapy, HR_2_ was subtracted from HR_3_ (HRΔ_32_). Hence, HR_1_ and HR_2_ values were considered baseline HR for monotherapy and add‐on therapy, respectively. In PEAR‐2, the change in HR in response to metoprolol was obtained from subtracting HR_2_ from HR_1_ (HRΔ_21_).

For the GWAS discovery analyses, 4 separate genome‐wide association analyses were conducted for association with β‐blockers’ negative chronotropic response; analyses of response to atenolol as: (1) monotherapy, and (2) add‐on therapy, in whites and blacks. Association analyses between SNPs and HR responses were performed using a linear regression analysis in PLINK.[30](#jah32988-bib-0030) All analyses were conducted under an additive model of inheritance and adjusted for age, sex, baseline HR, and principal components 1 and 2. Although >85% of the atenolol‐treated participants in PEAR monotherapy and add‐on therapy were taking the same dose of therapy (atenolol 100 mg per day), we adjusted for differences in dose in the analyses to prevent any confounding results. A meta‐analysis was then performed between PEAR atenolol monotherapy and PEAR atenolol add‐on therapy, in each race separately, assuming fixed effects and using inverse‐variance weighting as implemented in the METAL software.[31](#jah32988-bib-0031) For the GWAS analysis, a typical genome‐wide *P* value of <5E‐08 was used to judge the significance of SNPs in both black and white participants. SNPs were considered suggestive with meta‐analysis *P*<1E‐05. Power analysis was performed using Quanto ([http://biostats.usc.edu/software](http://biostats.usc.edu/software)) to identify the power and effect sizes that can be detected using our study sample size. We found that our GWAS discovery analysis in whites (n=426) has >80% power to detect a 5 bpm difference in β‐blocker HR response by genotype, assuming a SD of 10 bpm and SNPs of minor allele frequency ≥15%, with 2‐sided α level=1E‐05 (suggestive α level used in the study) (Table [S1](#jah32988-sup-0001)). This shows that this study is only powered to detect common SNPs with large effect size. Similarly, the power calculation in blacks shows that we only have power to detect common SNPs with large effect sizes (Table [S1](#jah32988-sup-0001)). This reveals that the sample size used in this study is not powered to detect SNPs with small effect sizes.

PLINK[30](#jah32988-bib-0030) software was used to explore the linkage disequilibrium between SNPs with *P*<1E‐05 from PEAR atenolol mono‐ and add‐on therapy GWAS meta‐analysis. SNPs were then pruned based on linkage disequilibrium by removing any SNP with an *r* ^2^ >0.5 with any other SNP in a 50 SNP window. SNPs that remained after pruning were considered independent SNPs and were moved forward for replication in PEAR‐2 participants treated with metoprolol (Figure [2](#jah32988-fig-0002)). A Bonferroni correction was used to define a significant threshold for this analysis (0.05/number of independent SNPs tested). Deviation from Hardy–Weinberg equilibrium was tested by χ^2^ or Fisher exact test in each race separately, and SNPs with Hardy‐Weinberg *P*<1E‐06 were excluded from the analysis.

To evaluate the effect of multiple response alleles on the HR response to β‐blockers and to investigate the relative contribution of our genetic findings toward the phenotype, we constructed a genetic response score based on replicated SNPs. Points were given for the genotypes of the replicated SNPs in which the homozygous genotype of each SNP with the greatest HR‐lowering effect had 2 points, while heterozygous genotype had 1 point, and homozygous genotype associated with the worst HR‐lowering effect had 0. Alleles with HR‐lowering effect were then summed up for inclusion in a linear regression model to test the association between the response score and changes in HR in response to β‐blockers in PEAR and PEAR‐2 participants. The latter analyses were adjusted for age, sex, baseline HR, differences in dose, and PC1 and 2.

### Network and In Silico Analyses

To further investigate the potential physiological roles of the replicated SNPs identified in this study and the possible interactions between them, we used STRING database (version 10.5) ([https://string-db.org/](https://string-db.org/)).[32](#jah32988-bib-0032) Replicated SNPs, from the GWAS meta‐analysis, were used to build networks showing the potential interactions between the identified SNPs/genes and other potential interacting genes that might be involved in the mechanism underlying β‐blocker HR‐lowering effects.

## Results

Baseline demographic characteristics for the PEAR and PEAR‐2 study participants treated with β‐blockers are summarized in Table [1](#jah32988-tbl-0001). Age, sex, body mass index, baseline HR, and HR response were similar between white atenolol mono‐ and add‐on therapy and metoprolol monotherapy–treated participants. Similarly, age, sex, body mass index, baseline HR, and HR response were similar between black atenolol mono‐ and add‐on therapy and metoprolol monotherapy‐treated participants. The mean and 95% confidence interval for atenolol HR‐lowering response following mono‐ and add‐on therapy were similar, as shown in Table [S2](#jah32988-sup-0001), suggesting minimal chronotropic response following HCTZ treatment, which provides the justification for combining these in the meta‐analysis. We did not conduct any comparisons between whites and blacks because analyses in these groups were all done separately and the purpose of this study is not to evaluate differences between whites and blacks.

### Table 1.

| Characteristics | PEAR Atenolol Monotherapy (N=374) |  | PEAR Atenolol Add‐on (N=325) |  | PEAR‐2 Metoprolol Monotherapy (N=368) |  |
| --- | --- | --- | --- | --- | --- | --- |
| White (n=228) | Black (n=146) | White (n=198) | Black (n=127) | White (n=200) | Black (n=168) |  |
| Age, mean (SD) y | 49.5±9.5 | 47.2±8.5 | 49.9±9.5 | 47.4±8.8 | 51.0±9.0 | 50.0±9.2 |
| Women, N (%) | 109 (47.3) | 107 (73.2) | 91 (45.9) | 92 (77.3) | 110 (55.0) | 89 (53.0) |
| BMI, mean (SD) kg×m−2 | 30.3±5.6 | 31.6±6.3 | 30.3±4.9 | 31.5±5.4 | 30.8±5.1 | 30.8±5.2 |
| Pretreatment HR, mean (SD) bpm | 76.5±9.2 | 79.9±9.2 | 77.8±9.7 | 81.2±9.6 | 77.7±9.6 | 79.8±9.5 |
| Change in HR, mean (SD) bpm | −12.8±5.5 | −11.0±6.8 | −13.4±6.0 | −11.1±7.4 | −12.3±7.2 | −11.2±7.0 |

Table 1 Caption: Characteristics of β‐Blocker‐Treated Participants in PEAR and PEAR‐2

### Genome‐Wide Association Analyses

Four separate genome‐wide association analyses were conducted for association with β‐blockers’ negative chronotropic response; these included analyses of response to atenolol as monotherapy and add‐on therapy in whites and blacks, followed by meta‐analysis of the 2 data sets within race. SNPs with a *P*<1E‐05 for atenolol monotherapy associations in whites and blacks are reported in Tables [S3](#jah32988-sup-0001) and [S4](#jah32988-sup-0001), respectively, and those for associations with response to atenolol add‐on therapy are in Tables [S5](#jah32988-sup-0001) and [S6](#jah32988-sup-0001), respectively. Manhattan plots and q‐q plots for each GWAS analysis are presented in Figures [S1](#jah32988-sup-0001) through [S4](#jah32988-sup-0001).

In blacks, the meta‐analysis of atenolol monotherapy and add‐on therapy revealed 35 SNPs achieving the suggestive *P* value level, representing 20 independent genetic signals (Table [2](#jah32988-tbl-0002)); these were tested for replication in PEAR‐2 blacks treated with metoprolol. Replicated SNPs were defined as those with a Bonferroni corrected *P*<0.0025 (0.05/20 independent genetic signals), and with HR‐lowering effects in the same direction observed in the discovery analysis. From this analysis, we found 2 SNPs—SNP rs17117817 located 7 kb away from 5′ of olfactory receptor family10 subfamily p‐member1 (*OR10P1*) gene, and SNP rs2364349 within the sorting nexin‐9 gene (*SNX9*)—to be significantly associated with metoprolol's HR‐lowering effect in black participants (Table [2](#jah32988-tbl-0002)). Using an additive genetic model, blacks carrying the rs17117817 G‐allele or rs2364349 A‐allele had a smaller HR‐lowering response to β‐blockers (Figure [3](#jah32988-fig-0003)). The combined 3‐cohort meta‐analysis *P* value for the rs17117817 and rs2364349 reached genome‐wide significance (rs17117817G‐allele; β=5.53 bpm, *P*=2E‐09 and rs2364349 A‐allele; β=3.5 bpm, *P*=1E‐08, respectively). Additionally, we validated the association between the *OR10P1* and *SNX9* genetic regions and changes in HR in response to β‐blockers in whites where we found significant SNPs in the *OR10P1* and *SNX9* genetic regions associated with HR changes in response to β‐blockers (rs79860936 in *OR10P1* gene region; *P*=7.6E‐04, rs7772983 in the *SNX9* gene region; *P*=1.7E‐03, Figure [4](#jah32988-fig-0004)).

### Table 2.

| Race | SNP | A1 | FRQ | Rsqa | Meta Effect | Meta SE | Meta P Value | Dir | PEAR‐2 Effect | PEAR‐2 SE | PEAR‐2 P Value | Rsqb |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| Blackc | rs17117817c | Tc | 0.93c | 1.00 to 0.82c | −5.73c | 1.11c | 2E‐07c | −−c | −5.05c | 1.65c | 2.2E‐03c | 1.00c |
| Black | rs10809249 | T | 0.82 | 0.95 to 0.99 | −3.52 | 0.74 | 2E‐06 | −− | −0.57 | 0.98 | 0.56 | 0.99 |
| Black | rs12890215 | T | 0.82 | 1.01 to 1.03 | −3.4 | 0.72 | 2E‐06 | −− | −0.48 | 0.92 | 0.602 | 1.00 |
| Black | rs1045580 | T | 0.85 | 0.95 to 1.02 | 3.56 | 0.76 | 2E‐06 | ++ | 0.62 | 1.02 | 0.53 | 1.00 |
| Black | rs2806495 | A | 0.10 | 0.96 to 0.82 | 4.67 | 0.99 | 2E‐06 | ++ | −0.53 | 1.29 | 0.67 | 1.00 |
| Black | rs7738600 | A | 0.51 | 1.05 to 1.03 | −2.6 | 0.56 | 4E‐06 | −− | −0.24 | 0.71 | 0.73 | 1.00 |
| Black | rs11757000 | T | 0.77 | 0.99 to 0.91 | −3.05 | 0.67 | 5E‐06 | −− | 0.35 | 0.91 | 0.69 | 1.00 |
| Black | rs7042878 | C | 0.09 | 1.05 to 1.10 | 4.31 | 0.95 | 5E‐06 | ++ | 1.65 | 1.35 | 0.22 | 1.00 |
| Blackc | rs2364349c | Ac | 0.18c | 0.87 to 0.92c | 3.43c | 0.76c | 7E‐06c | ++c | 3.64c | 1.03c | 4E‐04c | 0.98c |
| Black | rs4733278 | T | 0.46 | 0.82 to 0.99 | −2.62 | 0.58 | 7E‐06 | −− | 0.01 | 0.75 | 0.98 | 0.99 |
| Black | rs11069252 | T | 0.92 | 0.92 to 1.00 | −4.76 | 1.06 | 7E‐06 | −− | 1.21 | 1.37 | 0.37 | 1.00 |
| Black | rs12417208 | A | 0.84 | 0.89 to 0.82 | −3.76 | 0.84 | 8E‐06 | −− | −1.43 | 1.09 | 0.19 | 0.93 |
| Black | rs3759422 | T | 0.46 | 0.91 to 1.06 | 2.5 | 0.56 | 8E‐06 | ++ | 0.77 | 0.69 | 0.26 | 1.00 |
| Black | rs1018353 | T | 0.9 | 0.95 to 0.85 | −4.29 | 0.96 | 8E‐06 | −− | 0.94 | 1.37 | 0.49 | 1.00 |
| Black | rs4554901 | T | 0.07 | 0.93 to 1.03 | 4.85 | 1.09 | 8E‐06 | ++ | −0.09 | 1.41 | 0.94 | 0.99 |
| Black | rs10145648 | T | 0.74 | 0.98 to 1.00 | −2.85 | 0.64 | 8E‐06 | −− | −0.34 | 0.77 | 0.65 | 1.00 |
| Black | rs10809367 | A | 0.14 | 1.18 to 1.18 | 3.41 | 0.77 | 9E‐06 | ++ | 0.95 | 1.18 | 0.42 | 1.00 |
| Black | rs10499 | A | 0.95 | 0.94 to 1.10 | −5.67 | 1.28 | 9E‐06 | −− | −2.01 | 2.00 | 0.31 | 1.00 |
| Blackc | rs6455914c | Tc | 0.22c | 0.97 to 0.93c | 3.08c | 0.7c | 1E‐05c | ++c | 1.73c | 0.83c | 0.03c | 0.99c |
| Black | rs6470259 | A | 0.80 | 0.94 to 0.88 | −3.2 | 0.72 | 1E‐05 | −− | −0.76 | 1.01 | 0.44 | 0.94 |
| White | rs955395 | A | 0.36 | 0.99 to 0.96 | −1.7 | 0.35 | 9E‐07 | −− | 0.21 | 0.59 | 0.69 | 0.99 |
| White | rs11727192 | T | 0.21 | 0.94 to 0.97 | −1.92 | 0.42 | 5E‐06 | −− | −0.80 | 0.77 | 0.29 | 1.00 |
| White | rs10516175 | A | 0.12 | 1.09 to 1.02 | 2.17 | 0.48 | 6E‐06 | ++ | 0.46 | 0.79 | 0.64 | 0.99 |
| White | rs13160161 | A | 0.37 | 1.07 to 0.97 | −1.5 | 0.33 | 7E‐06 | −− | −0.34 | 0.67 | 0.71 | 1.00 |
| White | rs11641210 | T | 0.89 | 0.99 to 1.00 | 2.32 | 0.52 | 8E‐06 | ++ | −0.50 | 1.00 | 0.61 | 0.99 |

Table 2 Caption: Replication of the Linkage‐Disequilibrium Pruned SNPs With P Value <0.00001 From the GWAS Meta‐Analysis of β‐Blockers HR Response in PEAR‐2 Participants Treated With Metoprolol

### Figure 3.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3947/5866313/ecbfafbd2c60/JAH3-7-e006463-g003.jpg)

Effect of rs17117817 and rs2364349 polymorphisms on the β‐blocker HR response in whites within PEAR and PEAR‐2 studies. A, rs17117817. B, rs2364349. The box represents the values from the 25% to 75% percentile. The horizontal line represents the median. The black diamond represents the mean. The vertical line extends from the minimum to the maximum value. Each blue dot represents an individual. HR response was adjusted for age, sex, baseline HR, differences in dose, and principal components 1 and 2. Two‐sided P values represented are for the contrast of adjusted means between different genotype groups. *Meta‐analysis was performed assuming fixed effects and using inverse‐variance weighting. bpm indicates beats per minute; HR, heart rate; OR10P1, olfactory receptor family 10 subfamily p member 1; PEAR, Pharmacogenomic Evaluation of Antihypertensive Responses; SNX9, sorting nexin 9.

### Figure 4.

![Figure 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3947/5866313/f40ba01ebf6e/JAH3-7-e006463-g004.jpg)

Regional plot showing the significance of the associations of SNPs with changes in HR in response to β‐blockers used in whites. A, In olfactory receptor family 10 subfamily p member 1 (OR10P1) genetic region. B, In sorting nexin‐9 (SNX9) genetic region. HR indicates heart rate; SNP, single nucleotide polymorphism.

To make sure that the 2 replicated signals were not previously associated with HR and that they are pharmacogenomics independent signals, we searched NHGRI GWAS Catalog, ([https://www.ebi.ac.uk/gwas/search?query=SNX9](https://www.ebi.ac.uk/gwas/search?query=SNX9)) and the dbGap phenotype‐genotype integrator ([https://www.ncbi.nlm.nih.gov/gap/phegeni](https://www.ncbi.nlm.nih.gov/gap/phegeni)) for any GWAS associations that have been previously reported between *OR10P1* or *SNX9* and HR. The results from this search showed no previously reported association between *SNX9* or *OR10P1* and HR. Although we adjusted for baseline HR in our analysis, we also tested the association between SNP rs2364349 and SNP rs17117817 and baseline HR in black participants treated with β‐blockers in PEAR and PEAR‐2 participants included in this study. We found no association between SNP rs2364349 and baseline HR in PEAR monotherapy (*P*=0.23), PEAR add‐on therapy (*P*=0.37), and PEAR‐2 (*P*=0.70). Similarly, we found no association between SNP rs17117817 and baseline HR in PEAR monotherapy (*P*=0.18), PEAR add‐on therapy (*P*=0.79), and PEAR‐2 (*P*=0.30).

To assess the relative contribution of the replicated SNPs toward our changes in HR in response to β‐blockers, we created a response score as previously discussed in the Methods section. As expected, in black participants treated with atenolol mono‐ and add‐on therapy in PEAR and metoprolol in PEAR2, individuals with a higher score had a greater HR reduction in response to β‐blockers compared with individuals with lower score (Figure [5](#jah32988-fig-0005)). The meta‐analysis of the 3 data sets reveal that, on average, for each HR‐lowering allele carried by an individual, there is ≈3.4 bpm reduction in HR in response to β‐blockers (*P*=2.4E‐12). Such an approach requires validation in an independent cohort but highlights the potential approach to clinical use of such data.

### Figure 5.

![Figure 5](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3947/5866313/c08a7dab299f/JAH3-7-e006463-g005.jpg)

β‐Blocker's HR response score in PEAR and PEAR‐2. HR responses were adjusted for age, sex, baseline HR, differences in dose, and principal components 1 and 2. Genetic variants were coded as follows: (A) rs17117817 (OR10P1 genetic region) T/T=2, G/T=1, G/G=zero, and (B) rs2364349 (SNX9 genetic region) G/G=2, A/G=1, A/A=zero. The box represents the values from the 25% to 75% percentile. The horizontal line represents the median. The vertical line extends from the minimum to the maximum value. Each blue dot represents an individual. bpm indicates beats per minute HR, heart rate; PEAR, Pharmacogenomic Evaluation of Antihypertensive Responses. *Meta‐analysis was performed between PEAR atenolol monotherapy, PEAR atenolol add‐on therapy, and PEAR‐2 metoprolol groups, assuming fixed effects and using inverse‐variance weighting.

In whites, the meta‐analysis between PEAR atenolol mono‐ and add‐on therapy revealed 13 SNPs with a *P*<1E‐05, which represent 5 independent genetic signals (Table [2](#jah32988-tbl-0002)). However, none of these SNPs were replicated when tested in white participants treated with metoprolol in PEAR‐2 (Table [2](#jah32988-tbl-0002)).

### Network Analysis

Lastly, we conducted a network analysis using the genes discovered in this study (*OR10P1* and *SNX9*) and the β‐adrenergic receptor genes (*ADRB1* and *ADRB2*), which encode the protein targets for β‐blockers, to identify potential interactions between the replicated genetic signals that might be involved in β‐blockers’ HR‐lowering effects mechanism. From this analysis, using STRING database, we found a direct interaction between SNX9 and ADRB2, and an indirect interaction between OR10P1 and ADRB1 and ADRB2 via the effect of OR10P1 on either ADRBK1 or GNB1 (Figure [6](#jah32988-fig-0006)). These results further support the association we found between *OR10P1* and *SNX9* and changes in HR in response to β‐blockers and suggest that OR10P1 and SNX9 may be playing a potential role in the HR‐lowering mechanism underlying β‐blocker therapy.

### Figure 6.

![Figure 6](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3947/5866313/e6bfbc602c24/JAH3-7-e006463-g006.jpg)

Interaction network of replicated genetic signals and protein targets for β‐blockers. Network was created using the STRING database (https://string-db.org/).

## Discussion

β‐Blockers are one of the most commonly prescribed classes of drugs and have been used for decades to treat various cardiovascular conditions, including myocardial infarction, heart failure, hypertension, and angina.[1](#jah32988-bib-0001), [3](#jah32988-bib-0003) The HR‐lowering effect of β‐blockers is a critical efficacy component for this class of drugs and contributes substantially to their beneficial effects in acute‐ and post–myocardial infarction, heart failure, angina, and arrhythmias.[5](#jah32988-bib-0005), [6](#jah32988-bib-0006), [33](#jah32988-bib-0033) However, considerable interpatient variability in HR response to β‐blockers has been observed,[11](#jah32988-bib-0011), [12](#jah32988-bib-0012) indicating that a substantial proportion of β‐blocker‐treated patients fail to achieve cardioprotection with β‐blockers. This reveals that the current approach for β‐blockers selection and achieving target HR is suboptimal. Thus, identifying factors that are associated with the observed changes in HR in response to β‐blockers is important and holds the promise to optimize the use of this class of drugs. Given the growing evidence in the literature highlighting the significant influence of genetics on HR,[34](#jah32988-bib-0034), [35](#jah32988-bib-0035), [36](#jah32988-bib-0036) we sought herein to identify genetic factors associated with the observed interindividual variability in the negative chronotropic response to atenolol and metoprolol using a genome‐wide association approach. To our knowledge, the present study is the first GWAS to investigate genetic variations associated with changes in HR in response to β‐blocker therapy in whites and blacks.

The results from the GWAS analyses have revealed 2 novel genetic signals, SNP rs17117817 in the *OR10P1* genetic region, and SNP rs2364349 in the *SNX9* gene, with clinically relevant effects on HR in response to β‐blocker therapy used in blacks. We also validated these findings in whites, by identifying significant signals, in the *OR10P1* and *SNX9* genetic regions, associated with changes in HR of whites treated with β‐blockers. According to the meta‐analysis results, rs17117817 T/T and G/T genotype carriers have ≈11 and 5.5 bpm reductions in their HR in response to β‐blockers, respectively, compared with noncarriers. Similarly, rs2364349 G/G and A/G genotype carriers have ≈7 and 3.5 bpm reductions in their HR in response to β‐blockers, respectively, compared with noncarriers. Moreover, we created a response score to assess the relative contribution of the replicated SNPs and their effect on changes in HR in response to β‐blockers. From this analysis, we found that for each HR‐lowering allele carried by an individual, there is ≈3.4 bpm reduction in HR in response to β‐blockers (*P*=2.4E‐12). These observed reductions in HR in response to β‐blockers represent clinically significant differences that have been previously reported to be significantly associated with cardiovascular morbidity and mortality in the general population and in patients with heart disease.[37](#jah32988-bib-0037), [38](#jah32988-bib-0038), [39](#jah32988-bib-0039), [40](#jah32988-bib-0040) Several studies have reported the pivotal role of β‐blockers HR reduction effect in reducing cardiovascular mortality in heart failure.[41](#jah32988-bib-0041), [42](#jah32988-bib-0042) For instance, a meta‐analysis of 35 studies has shown that β‐blocker HR reduction of 5 bpm could provide 14% risk reduction in mortality.[10](#jah32988-bib-0010) These data reveal the clinical significance of our findings on HR response to β‐blockers and their clinical implications on cardiovascular mortality in patients treated with β‐blockers. However, further validation of the response score and the genetic variants, identified in this study, in large well‐designed independent cohorts are still needed.

*OR10P1* is a member of the olfactory receptor gene family, which represents a large family of genes that encode G‐protein coupled receptors. OR10P1 has been shown to be highly expressed in the heart,[43](#jah32988-bib-0043) yet little is known about the role of OR10P1 in the regulation of HR. Herein, using network analyses, we have seen an indirect interaction between OR10P1 and ADRB2. Consistent with this finding, previous studies have demonstrated the interaction between protein members of the olfactory receptor family and β_2_‐adrenergic receptors.[44](#jah32988-bib-0044) Additionally, several studies have shown that inactivation of β‐adrenergic receptor kinase‐2 and β‐arrestin both have a well‐known regulatory role on β‐adrenergic receptors, results in increased olfactory receptor stimulation of cAMP formation,[45](#jah32988-bib-0045), [46](#jah32988-bib-0046); the latter mediates the catecholaminergic control on HR and contractility.[47](#jah32988-bib-0047) Moreover, a recent study by Jovancevic et al[48](#jah32988-bib-0048) has shown that the activation of olfactory receptors (ie, OR51E1) induces a negative chronotropic effect in human stem cell–derived cardiomyocytes in a dose‐dependent manner. This literature evidence, along with the results of this study, suggests that olfactory receptors, including OR10P1, may be important regulators of HR in response to β‐blockers.

SNX9, a member of the sorting nexin gene family, has been shown to be highly expressed in the heart and placenta.[49](#jah32988-bib-0049) It was first described in 1999 as a Src homology 3‐domain and phox homology‐domain protein that regulates metalloproteinases ADAM9 and ADAM15.[49](#jah32988-bib-0049) SNX9 has also been shown to play a substantial role in endocytosis via clathrin‐coated pits, which is well identified in the downregulation of transmembrane signal transduction but can also promote sustained signal transduction.[50](#jah32988-bib-0050) Interestingly, studies have reported that SNX9 interacts and stimulates the basal GTPase activity of dynamin, a GTPase that plays a central role in clathrin‐induced endocytosis and agonist‐mediated sequestration of ADRB2.[51](#jah32988-bib-0051), [52](#jah32988-bib-0052), [53](#jah32988-bib-0053) Studies have also shown that the rapid and efficient recycling of G‐protein coupled receptors, like ADRB2, to the plasma membrane after ligand‐induced endocytosis is essential for the functional resensitization of receptors‐mediated signaling.[54](#jah32988-bib-0054), [55](#jah32988-bib-0055) This highlights the importance of clathrin‐mediated endocytosis, which is regulated by SNX9, in controlling the activity of G‐protein coupled receptors like ADRB2.

In addition, studies have also revealed an interaction between SNX9 and adaptor protein 2,[56](#jah32988-bib-0056), [57](#jah32988-bib-0057) an important multimeric protein that binds to cargo proteins and regulates the assembly of the clathrin‐coated vesicle and their endocytosis.[58](#jah32988-bib-0058) Furthermore, adaptor protein 2 has been shown to interact with β‐arrestin,[59](#jah32988-bib-0059) an interaction that is required for the recycling of the ADRB2. Since β‐arrestin is known to block the activation of adenylyl cyclase, it eventually reduces ADRB responsiveness to catecholamines.[60](#jah32988-bib-0060) Thus, it may be possible that SNX9 mediates its effect on HR response to β‐blockers via β‐arrestin. Moreover, the results of the network analysis that we performed revealed that SNX9 might be interacting with ADRB2 in an indirect way via its effect on dynamin 1 (DNM1) or dynamin 2 (DNM2) proteins. Although all these possible hypotheses are intriguing, follow‐up studies are needed to test these hypotheses.

The present study has several strengths. To our knowledge, this is the first GWAS conducted to identify genetic variants associated with changes in HR in response to β‐blockers in whites and blacks. Using this genome‐wide approach, we are the first to uncover the novel association between *OR10P1* and *SNX9* and changes in HR in response to β‐blockers. Additionally, the replication and the validation of the *OR10P1* and *SNX9* signals in blacks and whites, respectively, highlight the novelty of these findings. Moreover, the highly consistent effect of the *OR10P1* and *SNX9* genetic signals on changes in HR, across 3 independent cohorts, with 2 β‐blockers, emphasizes the importance of these findings and suggests the involvement of the *OR10P1* and *SNX9* in the mechanism underlying β‐blockers’ HR‐lowering effect.

The present study also has a few noteworthy limitations. First, the small sample sizes used for the genetic analyses may have limited our power to identify additional novel markers and to replicate some of our genetic signals. Second, we have used resting instead of exercise HR as an end point. However, in clinical practice, dose titration of β‐blockers is performed based on resting HR and blood pressure response. Thus, this is the more clinically relevant phenotype.

In summary, the present study sheds light on *OR10P1* and *SNX9* and their association with changes in HR in response to β‐blockers. Future studies are still needed to investigate the role of *OR10P1* and *SNX9* in the mechanism underlying HR response to β‐blockers. Such studies may provide new insights into the HR‐lowering mechanism, and help in the identification of novel drug targets based on a deeper understanding of the determinants of HR response to β‐blockers. Perhaps as we move forward towards a personalized medicine approach, future use of the genetic signals identified in this study might help in guiding the selection of β‐blockers and optimize their use.

## Sources of Funding

PEAR was supported by the National Institute of Health Pharmacogenetics Research Network grant U01 GM074492 and the National Center for Advancing Translational Sciences under the award number UL1 TR000064 (University of Florida); UL1 TR000454 (Emory University); and UL1 TR000135 (Mayo Clinic). PEAR was also supported by funds from the Mayo Foundation. Gonzalez receives support for research from the National Institute of Child Health and Human Development (K23HD083465) and the nonprofit organization Thrasher Research Fund ([www.thrasherresearch.org](http://www.thrasherresearch.org)).

## Disclosures

None.

## Supporting information

## Acknowledgments

We appreciate the valuable contributions of the study participants of the Pharmacogenomic Evaluation of Antihypertensive Responses (PEAR) and PEAR‐2. We also would like to thank PEAR and PEAR‐2 support staff and study physicians.

(J Am Heart Assoc. 2018;7:e006463 DOI: [10.1161/JAHA.117.006463](https://doi.org/10.1161/JAHA.117.006463).)29478026

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Hollenberg NK. The role of beta‐blockers as a cornerstone of cardiovascular therapy. Am J Hypertens. 2005;18:165S–168S.  [DOI](https://doi.org/10.1016/j.amjhyper.2005.09.010) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16373194/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hypertens&title=The%20role%20of%20beta%E2%80%90blockers%20as%20a%20cornerstone%20of%20cardiovascular%20therapy&author=NK%20Hollenberg&volume=18&publication_year=2005&pages=165S-168S&pmid=16373194&doi=10.1016/j.amjhyper.2005.09.010&)

2. Shin J, Johnson JA. Beta‐blocker pharmacogenetics in heart failure. Heart Fail Rev. 2010;15:187–196.  [DOI](https://doi.org/10.1007/s10741-008-9094-x) | [PMC free article](/articles/PMC2851851/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18437562/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Heart%20Fail%20Rev&title=Beta%E2%80%90blocker%20pharmacogenetics%20in%20heart%20failure&author=J%20Shin&author=JA%20Johnson&volume=15&publication_year=2010&pages=187-196&pmid=18437562&doi=10.1007/s10741-008-9094-x&)

3. IMS Institute . Medicines use and spending in the U.S. A review of 2015 and outlook to 2020. Available at: https://structurecms-staging-psyclone.netdna-ssl.com/client_assets/dwonk/media/attachments/590c/6aa0/6970/2d2d/4182/0000/590c6aa069702d2d41820000.pdf?1493985952. Accessed February 7, 2017.  [https://structurecms-staging-psyclone.netdna-ssl.com/client_assets/dwonk/media/attachments/590c/6aa0/6970/2d2d/4182/0000/590c6aa069702d2d41820000.pdf?1493985952](https://structurecms-staging-psyclone.netdna-ssl.com/client_assets/dwonk/media/attachments/590c/6aa0/6970/2d2d/4182/0000/590c6aa069702d2d41820000.pdf?1493985952)

4. Ladage D, Schwinger RH, Brixius K. Cardio‐selective beta‐blocker: pharmacological evidence and their influence on exercise capacity. Cardiovasc Ther. 2013;31:76–83.  [DOI](https://doi.org/10.1111/j.1755-5922.2011.00306.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22279967/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cardiovasc%20Ther&title=Cardio%E2%80%90selective%20beta%E2%80%90blocker:%20pharmacological%20evidence%20and%20their%20influence%20on%20exercise%20capacity&author=D%20Ladage&author=RH%20Schwinger&author=K%20Brixius&volume=31&publication_year=2013&pages=76-83&pmid=22279967&doi=10.1111/j.1755-5922.2011.00306.x&)

5. Okin PM, Kjeldsen SE, Julius S, Hille DA, Dahlof B, Devereux RB. Effect of changing heart rate during treatment of hypertension on incidence of heart failure. Am J Cardiol. 2012;109:699–704.  [DOI](https://doi.org/10.1016/j.amjcard.2011.10.026) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22154318/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Cardiol&title=Effect%20of%20changing%20heart%20rate%20during%20treatment%20of%20hypertension%20on%20incidence%20of%20heart%20failure&author=PM%20Okin&author=SE%20Kjeldsen&author=S%20Julius&author=DA%20Hille&author=B%20Dahlof&volume=109&publication_year=2012&pages=699-704&pmid=22154318&doi=10.1016/j.amjcard.2011.10.026&)

6. Kaplan JR, Manuck SB, Adams MR, Weingand KW, Clarkson TB. Inhibition of coronary atherosclerosis by propranolol in behaviorally predisposed monkeys fed an atherogenic diet. Circulation. 1987;76:1364–1372.  [DOI](https://doi.org/10.1161/01.cir.76.6.1364) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/3677359/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circulation&title=Inhibition%20of%20coronary%20atherosclerosis%20by%20propranolol%20in%20behaviorally%20predisposed%20monkeys%20fed%20an%20atherogenic%20diet&author=JR%20Kaplan&author=SB%20Manuck&author=MR%20Adams&author=KW%20Weingand&author=TB%20Clarkson&volume=76&publication_year=1987&pages=1364-1372&pmid=3677359&doi=10.1161/01.cir.76.6.1364&)

7. Cucherat M. Quantitative relationship between resting heart rate reduction and magnitude of clinical benefits in post‐myocardial infarction: a meta‐regression of randomized clinical trials. Eur Heart J. 2007;28:3012–3019.  [DOI](https://doi.org/10.1093/eurheartj/ehm489) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17981830/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20Heart%20J&title=Quantitative%20relationship%20between%20resting%20heart%20rate%20reduction%20and%20magnitude%20of%20clinical%20benefits%20in%20post%E2%80%90myocardial%20infarction:%20a%20meta%E2%80%90regression%20of%20randomized%20clinical%20trials&author=M%20Cucherat&volume=28&publication_year=2007&pages=3012-3019&pmid=17981830&doi=10.1093/eurheartj/ehm489&)

8. Flannery G, Gehrig‐Mills R, Billah B, Krum H. Analysis of randomized controlled trials on the effect of magnitude of heart rate reduction on clinical outcomes in patients with systolic chronic heart failure receiving beta‐blockers. Am J Cardiol. 2008;101:865–869.  [DOI](https://doi.org/10.1016/j.amjcard.2007.11.023) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18328855/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Cardiol&title=Analysis%20of%20randomized%20controlled%20trials%20on%20the%20effect%20of%20magnitude%20of%20heart%20rate%20reduction%20on%20clinical%20outcomes%20in%20patients%20with%20systolic%20chronic%20heart%20failure%20receiving%20beta%E2%80%90blockers&author=G%20Flannery&author=R%20Gehrig%E2%80%90Mills&author=B%20Billah&author=H%20Krum&volume=101&publication_year=2008&pages=865-869&pmid=18328855&doi=10.1016/j.amjcard.2007.11.023&)

9. Kolloch R, Legler UF, Champion A, Cooper‐Dehoff RM, Handberg E, Zhou Q, Pepine CJ. Impact of resting heart rate on outcomes in hypertensive patients with coronary artery disease: findings from the INternational VErapamil‐SR/trandolapril STudy (INVEST). Eur Heart J. 2008;29:1327–1334.  [DOI](https://doi.org/10.1093/eurheartj/ehn123) | [PMC free article](/articles/PMC2805436/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18375982/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20Heart%20J&title=Impact%20of%20resting%20heart%20rate%20on%20outcomes%20in%20hypertensive%20patients%20with%20coronary%20artery%20disease:%20findings%20from%20the%20INternational%20VErapamil%E2%80%90SR/trandolapril%20STudy%20(INVEST)&author=R%20Kolloch&author=UF%20Legler&author=A%20Champion&author=RM%20Cooper%E2%80%90Dehoff&author=E%20Handberg&volume=29&publication_year=2008&pages=1327-1334&pmid=18375982&doi=10.1093/eurheartj/ehn123&)

10. McAlister FA, Wiebe N, Ezekowitz JA, Leung AA, Armstrong PW. Meta‐analysis: beta‐blocker dose, heart rate reduction, and death in patients with heart failure. Ann Intern Med. 2009;150:784–794.  [DOI](https://doi.org/10.7326/0003-4819-150-11-200906020-00006) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19487713/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Intern%20Med&title=Meta%E2%80%90analysis:%20beta%E2%80%90blocker%20dose,%20heart%20rate%20reduction,%20and%20death%20in%20patients%20with%20heart%20failure&author=FA%20McAlister&author=N%20Wiebe&author=JA%20Ezekowitz&author=AA%20Leung&author=PW%20Armstrong&volume=150&publication_year=2009&pages=784-794&pmid=19487713&doi=10.7326/0003-4819-150-11-200906020-00006&)

11. Materson BJ, Reda DJ, Cushman WC, Massie BM, Freis ED, Kochar MS, Hamburger RJ, Fye C, Lakshman R, Gottdiener J. Single‐drug therapy for hypertension in men. A comparison of six antihypertensive agents with placebo. The Department of Veterans Affairs Cooperative Study Group on Antihypertensive Agents. N Engl J Med. 1993;328:914–921.  [DOI](https://doi.org/10.1056/NEJM199304013281303) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8446138/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Single%E2%80%90drug%20therapy%20for%20hypertension%20in%20men.%20A%20comparison%20of%20six%20antihypertensive%20agents%20with%20placebo.%20The%20Department%20of%20Veterans%20Affairs%20Cooperative%20Study%20Group%20on%20Antihypertensive%20Agents&author=BJ%20Materson&author=DJ%20Reda&author=WC%20Cushman&author=BM%20Massie&author=ED%20Freis&volume=328&publication_year=1993&pages=914-921&pmid=8446138&doi=10.1056/NEJM199304013281303&)

12. Cotarlan V, Brofferio A, Gerhard GS, Chu X, Shirani J. Impact of beta(1)‐ and beta(2)‐adrenergic receptor gene single nucleotide polymorphisms on heart rate response to metoprolol prior to coronary computed tomographic angiography. Am J Cardiol. 2013;111:661–666.  [DOI](https://doi.org/10.1016/j.amjcard.2012.11.015) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23261005/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Cardiol&title=Impact%20of%20beta(1)%E2%80%90%20and%20beta(2)%E2%80%90adrenergic%20receptor%20gene%20single%20nucleotide%20polymorphisms%20on%20heart%20rate%20response%20to%20metoprolol%20prior%20to%20coronary%20computed%20tomographic%20angiography&author=V%20Cotarlan&author=A%20Brofferio&author=GS%20Gerhard&author=X%20Chu&author=J%20Shirani&volume=111&publication_year=2013&pages=661-666&pmid=23261005&doi=10.1016/j.amjcard.2012.11.015&)

13. Petrashevskaya NN, Koch SE, Bodi I, Schwartz A. Calcium cycling, historic overview and perspectives. Role for autonomic nervous system regulation. J Mol Cell Cardiol. 2002;34:885–896.  [DOI](https://doi.org/10.1006/jmcc.2002.2033) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12234761/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Mol%20Cell%20Cardiol&title=Calcium%20cycling,%20historic%20overview%20and%20perspectives.%20Role%20for%20autonomic%20nervous%20system%20regulation&author=NN%20Petrashevskaya&author=SE%20Koch&author=I%20Bodi&author=A%20Schwartz&volume=34&publication_year=2002&pages=885-896&pmid=12234761&doi=10.1006/jmcc.2002.2033&)

14. Johnson JA, Zineh I, Puckett BJ, McGorray SP, Yarandi HN, Pauly DF. Beta 1‐adrenergic receptor polymorphisms and antihypertensive response to metoprolol. Clin Pharmacol Ther. 2003;74:44–52.  [DOI](https://doi.org/10.1016/S0009-9236(03)00068-7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12844134/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Beta%201%E2%80%90adrenergic%20receptor%20polymorphisms%20and%20antihypertensive%20response%20to%20metoprolol&author=JA%20Johnson&author=I%20Zineh&author=BJ%20Puckett&author=SP%20McGorray&author=HN%20Yarandi&volume=74&publication_year=2003&pages=44-52&pmid=12844134&doi=10.1016/S0009-9236(03)00068-7&)

15. Liu J, Liu ZQ, Yu BN, Xu FH, Mo W, Zhou G, Liu YZ, Li Q, Zhou HH. beta1‐Adrenergic receptor polymorphisms influence the response to metoprolol monotherapy in patients with essential hypertension. Clin Pharmacol Ther. 2006;80:23–32.  [DOI](https://doi.org/10.1016/j.clpt.2006.03.004) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16815314/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=beta1%E2%80%90Adrenergic%20receptor%20polymorphisms%20influence%20the%20response%20to%20metoprolol%20monotherapy%20in%20patients%20with%20essential%20hypertension&author=J%20Liu&author=ZQ%20Liu&author=BN%20Yu&author=FH%20Xu&author=W%20Mo&volume=80&publication_year=2006&pages=23-32&pmid=16815314&doi=10.1016/j.clpt.2006.03.004&)

16. Mialet Perez J, Rathz DA, Petrashevskaya NN, Hahn HS, Wagoner LE, Schwartz A, Dorn GW, Liggett SB. Beta 1‐adrenergic receptor polymorphisms confer differential function and predisposition to heart failure. Nat Med. 2003;9:1300–1305.  [DOI](https://doi.org/10.1038/nm930) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14502278/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Med&title=Beta%201%E2%80%90adrenergic%20receptor%20polymorphisms%20confer%20differential%20function%20and%20predisposition%20to%20heart%20failure&author=J%20Mialet%20Perez&author=DA%20Rathz&author=NN%20Petrashevskaya&author=HS%20Hahn&author=LE%20Wagoner&volume=9&publication_year=2003&pages=1300-1305&pmid=14502278&doi=10.1038/nm930&)

17. Terra SG, Hamilton KK, Pauly DF, Lee CR, Patterson JH, Adams KF, Schofield RS, Belgado BS, Hill JA, Aranda JM, Yarandi HN, Johnson JA. Beta1‐adrenergic receptor polymorphisms and left ventricular remodeling changes in response to beta‐blocker therapy. Pharmacogenet Genomics. 2005;15:227–234.  [DOI](https://doi.org/10.1097/01213011-200504000-00006) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15864115/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Beta1%E2%80%90adrenergic%20receptor%20polymorphisms%20and%20left%20ventricular%20remodeling%20changes%20in%20response%20to%20beta%E2%80%90blocker%20therapy&author=SG%20Terra&author=KK%20Hamilton&author=DF%20Pauly&author=CR%20Lee&author=JH%20Patterson&volume=15&publication_year=2005&pages=227-234&pmid=15864115&doi=10.1097/01213011-200504000-00006&)

18. Liggett SB, Mialet‐Perez J, Thaneemit‐Chen S, Weber SA, Greene SM, Hodne D, Nelson B, Morrison J, Domanski MJ, Wagoner LE, Abraham WT, Anderson JL, Carlquist JF, Krause‐Steinrauf HJ, Lazzeroni LC, Port JD, Lavori PW, Bristow MR. A polymorphism within a conserved beta(1)‐adrenergic receptor motif alters cardiac function and beta‐blocker response in human heart failure. Proc Natl Acad Sci USA. 2006;103:11288–11293.  [DOI](https://doi.org/10.1073/pnas.0509937103) | [PMC free article](/articles/PMC1523317/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16844790/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20USA&title=A%20polymorphism%20within%20a%20conserved%20beta(1)%E2%80%90adrenergic%20receptor%20motif%20alters%20cardiac%20function%20and%20beta%E2%80%90blocker%20response%20in%20human%20heart%20failure&author=SB%20Liggett&author=J%20Mialet%E2%80%90Perez&author=S%20Thaneemit%E2%80%90Chen&author=SA%20Weber&author=SM%20Greene&volume=103&publication_year=2006&pages=11288-11293&pmid=16844790&doi=10.1073/pnas.0509937103&)

19. Beitelshees AL, Zineh I, Yarandi HN, Pauly DF, Johnson JA. Influence of phenotype and pharmacokinetics on beta‐blocker drug target pharmacogenetics. Pharmacogenomics J. 2006;6:174–178.  [DOI](https://doi.org/10.1038/sj.tpj.6500354) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16402084/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=Influence%20of%20phenotype%20and%20pharmacokinetics%20on%20beta%E2%80%90blocker%20drug%20target%20pharmacogenetics&author=AL%20Beitelshees&author=I%20Zineh&author=HN%20Yarandi&author=DF%20Pauly&author=JA%20Johnson&volume=6&publication_year=2006&pages=174-178&pmid=16402084&doi=10.1038/sj.tpj.6500354&)

20. de Groote P, Helbecque N, Lamblin N, Hermant X, Mc Fadden E, Foucher‐Hossein C, Amouyel P, Dallongeville J, Bauters C. Association between beta‐1 and beta‐2 adrenergic receptor gene polymorphisms and the response to beta‐blockade in patients with stable congestive heart failure. Pharmacogenet Genomics. 2005;15:137–142.  [DOI](https://doi.org/10.1097/01213011-200503000-00001) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15861037/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Association%20between%20beta%E2%80%901%20and%20beta%E2%80%902%20adrenergic%20receptor%20gene%20polymorphisms%20and%20the%20response%20to%20beta%E2%80%90blockade%20in%20patients%20with%20stable%20congestive%20heart%20failure&author=P%20de%20Groote&author=N%20Helbecque&author=N%20Lamblin&author=X%20Hermant&author=E%20Mc%20Fadden&volume=15&publication_year=2005&pages=137-142&pmid=15861037&doi=10.1097/01213011-200503000-00001&)

21. Karlsson J, Lind L, Hallberg P, Michaelsson K, Kurland L, Kahan T, Malmqvist K, Ohman KP, Nystrom E, Melhus H. Beta(1)‐adrenergic receptor gene polymorphisms and response to beta(1)‐adrenergic receptor blockade in patients with essential hypertension. Clin Cardiol. 2004;27:347–350.  [DOI](https://doi.org/10.1002/clc.4960270610) | [PMC free article](/articles/PMC6654438/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15237695/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cardiol&title=Beta(1)%E2%80%90adrenergic%20receptor%20gene%20polymorphisms%20and%20response%20to%20beta(1)%E2%80%90adrenergic%20receptor%20blockade%20in%20patients%20with%20essential%20hypertension&author=J%20Karlsson&author=L%20Lind&author=P%20Hallberg&author=K%20Michaelsson&author=L%20Kurland&volume=27&publication_year=2004&pages=347-350&pmid=15237695&doi=10.1002/clc.4960270610&)

22. Kurnik D, Li C, Sofowora GG, Friedman EA, Muszkat M, Xie HG, Harris PA, Williams SM, Nair UB, Wood AJ, Stein CM. Beta‐1‐adrenoceptor genetic variants and ethnicity independently affect response to beta‐blockade. Pharmacogenet Genomics. 2008;18:895–902.  [DOI](https://doi.org/10.1097/FPC.0b013e328309733f) | [PMC free article](/articles/PMC2757009/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18794726/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Beta%E2%80%901%E2%80%90adrenoceptor%20genetic%20variants%20and%20ethnicity%20independently%20affect%20response%20to%20beta%E2%80%90blockade&author=D%20Kurnik&author=C%20Li&author=GG%20Sofowora&author=EA%20Friedman&author=M%20Muszkat&volume=18&publication_year=2008&pages=895-902&pmid=18794726&doi=10.1097/FPC.0b013e328309733f&)

23. Liu J, Liu ZQ, Tan ZR, Chen XP, Wang LS, Zhou G, Zhou HH. Gly389Arg polymorphism of beta(1)‐adrenergic receptor is associated with the cardiovascular response to metoprolol. Clin Pharmacol Ther. 2003;74:372–379.  [DOI](https://doi.org/10.1016/S0009-9236(03)00224-8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14534524/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Gly389Arg%20polymorphism%20of%20beta(1)%E2%80%90adrenergic%20receptor%20is%20associated%20with%20the%20cardiovascular%20response%20to%20metoprolol&author=J%20Liu&author=ZQ%20Liu&author=ZR%20Tan&author=XP%20Chen&author=LS%20Wang&volume=74&publication_year=2003&pages=372-379&pmid=14534524&doi=10.1016/S0009-9236(03)00224-8&)

24. Sehrt D, Meineke I, Tzvetkov M, Gultepe S, Brockmoller J. Carvedilol pharmacokinetics and pharmacodynamics in relation to CYP2D6 and ADRB pharmacogenetics. Pharmacogenomics. 2011;12:783–795.  [DOI](https://doi.org/10.2217/pgs.11.20) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21599570/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Carvedilol%20pharmacokinetics%20and%20pharmacodynamics%20in%20relation%20to%20CYP2D6%20and%20ADRB%20pharmacogenetics&author=D%20Sehrt&author=I%20Meineke&author=M%20Tzvetkov&author=S%20Gultepe&author=J%20Brockmoller&volume=12&publication_year=2011&pages=783-795&pmid=21599570&doi=10.2217/pgs.11.20&)

25. Johnson JA, Boerwinkle E, Zineh I, Chapman AB, Bailey K, Cooper‐DeHoff RM, Gums J, Curry RW, Gong Y, Beitelshees AL, Schwartz G, Turner ST. Pharmacogenomics of antihypertensive drugs: rationale and design of the Pharmacogenomic Evaluation of Antihypertensive Responses (PEAR) study. Am Heart J. 2009;157:442–449.  [DOI](https://doi.org/10.1016/j.ahj.2008.11.018) | [PMC free article](/articles/PMC2671287/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19249413/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20Heart%20J&title=Pharmacogenomics%20of%20antihypertensive%20drugs:%20rationale%20and%20design%20of%20the%20Pharmacogenomic%20Evaluation%20of%20Antihypertensive%20Responses%20(PEAR)%20study&author=JA%20Johnson&author=E%20Boerwinkle&author=I%20Zineh&author=AB%20Chapman&author=K%20Bailey&volume=157&publication_year=2009&pages=442-449&pmid=19249413&doi=10.1016/j.ahj.2008.11.018&)

26. Hamadeh IS, Langaee TY, Dwivedi R, Garcia S, Burkley BM, Skaar TC, Chapman AB, Gums JG, Turner ST, Gong Y, Cooper‐DeHoff RM, Johnson JA. Impact of CYP2D6 polymorphisms on clinical efficacy and tolerability of metoprolol tartrate. Clin Pharmacol Ther. 2014;96:175–181.  [DOI](https://doi.org/10.1038/clpt.2014.62) | [PMC free article](/articles/PMC4111800/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24637943/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Impact%20of%20CYP2D6%20polymorphisms%20on%20clinical%20efficacy%20and%20tolerability%20of%20metoprolol%20tartrate&author=IS%20Hamadeh&author=TY%20Langaee&author=R%20Dwivedi&author=S%20Garcia&author=BM%20Burkley&volume=96&publication_year=2014&pages=175-181&pmid=24637943&doi=10.1038/clpt.2014.62&)

27. O'Brien E, Waeber B, Parati G, Staessen J, Myers MG. Blood pressure measuring devices: recommendations of the European Society of Hypertension. BMJ. 2001;322:531–536.  [DOI](https://doi.org/10.1136/bmj.322.7285.531) | [PMC free article](/articles/PMC1119736/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11230071/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMJ&title=Blood%20pressure%20measuring%20devices:%20recommendations%20of%20the%20European%20Society%20of%20Hypertension&author=E%20O'Brien&author=B%20Waeber&author=G%20Parati&author=J%20Staessen&author=MG%20Myers&volume=322&publication_year=2001&pages=531-536&pmid=11230071&doi=10.1136/bmj.322.7285.531&)

28. Turner ST, Boerwinkle E, O'Connell JR, Bailey KR, Gong Y, Chapman AB, McDonough CW, Beitelshees AL, Schwartz GL, Gums JG, Padmanabhan S, Hiltunen TP, Citterio L, Donner KM, Hedner T, Lanzani C, Melander O, Saarela J, Ripatti S, Wahlstrand B, Manunta P, Kontula K, Dominiczak AF, Cooper‐DeHoff RM, Johnson JA. Genomic association analysis of common variants influencing antihypertensive response to hydrochlorothiazide. Hypertension. 2013;62:391–397.  [DOI](https://doi.org/10.1161/HYPERTENSIONAHA.111.00436) | [PMC free article](/articles/PMC3780966/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23753411/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hypertension&title=Genomic%20association%20analysis%20of%20common%20variants%20influencing%20antihypertensive%20response%20to%20hydrochlorothiazide&author=ST%20Turner&author=E%20Boerwinkle&author=JR%20O'Connell&author=KR%20Bailey&author=Y%20Gong&volume=62&publication_year=2013&pages=391-397&pmid=23753411&doi=10.1161/HYPERTENSIONAHA.111.00436&)

29. Howie B, Fuchsberger C, Stephens M, Marchini J, Abecasis GR. Fast and accurate genotype imputation in genome‐wide association studies through pre‐phasing. Nat Genet. 2012;44:955–959.  [DOI](https://doi.org/10.1038/ng.2354) | [PMC free article](/articles/PMC3696580/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22820512/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Fast%20and%20accurate%20genotype%20imputation%20in%20genome%E2%80%90wide%20association%20studies%20through%20pre%E2%80%90phasing&author=B%20Howie&author=C%20Fuchsberger&author=M%20Stephens&author=J%20Marchini&author=GR%20Abecasis&volume=44&publication_year=2012&pages=955-959&pmid=22820512&doi=10.1038/ng.2354&)

30. Purcell S, Neale B, Todd‐Brown K, Thomas L, Ferreira MA, Bender D, Maller J, Sklar P, de Bakker PI, Daly MJ, Sham PC. PLINK: a tool set for whole‐genome association and population‐based linkage analyses. Am J Hum Genet. 2007;81:559–575.  [DOI](https://doi.org/10.1086/519795) | [PMC free article](/articles/PMC1950838/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17701901/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=PLINK:%20a%20tool%20set%20for%20whole%E2%80%90genome%20association%20and%20population%E2%80%90based%20linkage%20analyses&author=S%20Purcell&author=B%20Neale&author=K%20Todd%E2%80%90Brown&author=L%20Thomas&author=MA%20Ferreira&volume=81&publication_year=2007&pages=559-575&pmid=17701901&doi=10.1086/519795&)

31. Willer CJ, Li Y, Abecasis GR. METAL: fast and efficient meta‐analysis of genomewide association scans. Bioinformatics. 2010;26:2190–2191.  [DOI](https://doi.org/10.1093/bioinformatics/btq340) | [PMC free article](/articles/PMC2922887/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20616382/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=METAL:%20fast%20and%20efficient%20meta%E2%80%90analysis%20of%20genomewide%20association%20scans&author=CJ%20Willer&author=Y%20Li&author=GR%20Abecasis&volume=26&publication_year=2010&pages=2190-2191&pmid=20616382&doi=10.1093/bioinformatics/btq340&)

32. Szklarczyk D, Franceschini A, Wyder S, Forslund K, Heller D, Huerta‐Cepas J, Simonovic M, Roth A, Santos A, Tsafou KP, Kuhn M, Bork P, Jensen LJ, von Mering C. STRING v10: protein‐protein interaction networks, integrated over the tree of life. Nucleic Acids Res. 2015;43:D447–D452.  [DOI](https://doi.org/10.1093/nar/gku1003) | [PMC free article](/articles/PMC4383874/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25352553/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=STRING%20v10:%20protein%E2%80%90protein%20interaction%20networks,%20integrated%20over%20the%20tree%20of%20life&author=D%20Szklarczyk&author=A%20Franceschini&author=S%20Wyder&author=K%20Forslund&author=D%20Heller&volume=43&publication_year=2015&pages=D447-D452&pmid=25352553&doi=10.1093/nar/gku1003&)

33. Freemantle N, Cleland J, Young P, Mason J, Harrison J. beta Blockade after myocardial infarction: systematic review and meta regression analysis. BMJ. 1999;318:1730–1737.  [DOI](https://doi.org/10.1136/bmj.318.7200.1730) | [PMC free article](/articles/PMC31101/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10381708/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMJ&title=beta%20Blockade%20after%20myocardial%20infarction:%20systematic%20review%20and%20meta%20regression%20analysis&author=N%20Freemantle&author=J%20Cleland&author=P%20Young&author=J%20Mason&author=J%20Harrison&volume=318&publication_year=1999&pages=1730-1737&pmid=10381708&doi=10.1136/bmj.318.7200.1730&)

34. Eppinga RN, Hagemeijer Y, Burgess S, Hinds DA, Stefansson K, Gudbjartsson DF, van Veldhuisen DJ, Munroe PB, Verweij N, van der Harst P. Identification of genomic loci associated with resting heart rate and shared genetic predictors with all‐cause mortality. Nat Genet. 2016;48:1557–1563.  [DOI](https://doi.org/10.1038/ng.3708) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27798624/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Identification%20of%20genomic%20loci%20associated%20with%20resting%20heart%20rate%20and%20shared%20genetic%20predictors%20with%20all%E2%80%90cause%20mortality&author=RN%20Eppinga&author=Y%20Hagemeijer&author=S%20Burgess&author=DA%20Hinds&author=K%20Stefansson&volume=48&publication_year=2016&pages=1557-1563&pmid=27798624&doi=10.1038/ng.3708&)

35. den Hoed M, Eijgelsheim M, Esko T, Brundel BJ, Peal DS, Evans DM, Nolte IM, Segre AV, Holm H, Handsaker RE, Westra HJ, Johnson T, Isaacs A, Yang J, Lundby A, Zhao JH, Kim YJ, Go MJ, Almgren P, Bochud M, Boucher G, Cornelis MC, Gudbjartsson D, Hadley D, van der Harst P, Hayward C, den Heijer M, Igl W, Jackson AU, Kutalik Z, Luan J, Kemp JP, Kristiansson K, Ladenvall C, Lorentzon M, Montasser ME, Njajou OT, O'Reilly PF, Padmanabhan S, St Pourcain B, Rankinen T, Salo P, Tanaka T, Timpson NJ, Vitart V, Waite L, Wheeler W, Zhang W, Draisma HH, Feitosa MF, Kerr KF, Lind PA, Mihailov E, Onland‐Moret NC, Song C, Weedon MN, Xie W, Yengo L, Absher D, Albert CM, Alonso A, Arking DE, de Bakker PI, Balkau B, Barlassina C, Benaglio P, Bis JC, Bouatia‐Naji N, Brage S, Chanock SJ, Chines PS, Chung M, Darbar D, Dina C, Dorr M, Elliott P, Felix SB, Fischer K, Fuchsberger C, de Geus EJ, Goyette P, Gudnason V, Harris TB, Hartikainen AL, Havulinna AS, Heckbert SR, Hicks AA, Hofman A, Holewijn S, Hoogstra‐Berends F, Hottenga JJ, Jensen MK, Johansson A, Junttila J, Kaab S, Kanon B, Ketkar S, Khaw KT, Knowles JW, Kooner AS, Kors JA, Kumari M, Milani L, Laiho P, Lakatta EG, Langenberg C, Leusink M, Liu Y, Luben RN, Lunetta KL, Lynch SN, Markus MR, Marques‐Vidal P, Mateo Leach I, McArdle WL, McCarroll SA, Medland SE, Miller KA, Montgomery GW, Morrison AC, Muller‐Nurasyid M, Navarro P, Nelis M, O'Connell JR, O'Donnell CJ, Ong KK, Newman AB, Peters A, Polasek O, Pouta A, Pramstaller PP, Psaty BM, Rao DC, Ring SM, Rossin EJ, Rudan D, Sanna S, Scott RA, Sehmi JS, Sharp S, Shin JT, Singleton AB, Smith AV, Soranzo N, Spector TD, Stewart C, Stringham HM, Tarasov KV, Uitterlinden AG, Vandenput L, Hwang SJ, Whitfield JB, Wijmenga C, Wild SH, Willemsen G, Wilson JF, Witteman JC, Wong A, Wong Q, Jamshidi Y, Zitting P, Boer JM, Boomsma DI, Borecki IB, van Duijn CM, Ekelund U, Forouhi NG, Froguel P, Hingorani A, Ingelsson E, Kivimaki M, Kronmal RA, Kuh D, Lind L, Martin NG, Oostra BA, Pedersen NL, Quertermous T, Rotter JI, van der Schouw YT, Verschuren WM, Walker M, Albanes D, Arnar DO, Assimes TL, Bandinelli S, Boehnke M, de Boer RA, Bouchard C, Caulfield WL, Chambers JC, Curhan G, Cusi D, Eriksson J, Ferrucci L, van Gilst WH, Glorioso N, de Graaf J, Groop L, Gyllensten U, Hsueh WC, Hu FB, Huikuri HV, Hunter DJ, Iribarren C, Isomaa B, Jarvelin MR, Jula A, Kahonen M, Kiemeney LA, van der Klauw MM, Kooner JS, Kraft P, Iacoviello L, Lehtimaki T, Lokki ML, Mitchell BD, Navis G, Nieminen MS, Ohlsson C, Poulter NR, Qi L, Raitakari OT, Rimm EB, Rioux JD, Rizzi F, Rudan I, Salomaa V, Sever PS, Shields DC, Shuldiner AR, Sinisalo J, Stanton AV, Stolk RP, Strachan DP, Tardif JC, Thorsteinsdottir U, Tuomilehto J, van Veldhuisen DJ, Virtamo J, Viikari J, Vollenweider P, Waeber G, Widen E, Cho YS, Olsen JV, Visscher PM, Willer C, Franke L, Erdmann J, Thompson JR, Pfeufer A, Sotoodehnia N, Newton‐Cheh C, Ellinor PT, Stricker BH, Metspalu A, Perola M, Beckmann JS, Smith GD, Stefansson K, Wareham NJ, Munroe PB, Sibon OC, Milan DJ, Snieder H, Samani NJ, Loos RJ. Identification of heart rate‐associated loci and their effects on cardiac conduction and rhythm disorders. Nat Genet. 2013;45:621–631.  [DOI](https://doi.org/10.1038/ng.2610) | [PMC free article](/articles/PMC3696959/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23583979/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Identification%20of%20heart%20rate%E2%80%90associated%20loci%20and%20their%20effects%20on%20cardiac%20conduction%20and%20rhythm%20disorders&author=M%20den%20Hoed&author=M%20Eijgelsheim&author=T%20Esko&author=BJ%20Brundel&author=DS%20Peal&volume=45&publication_year=2013&pages=621-631&pmid=23583979&doi=10.1038/ng.2610&)

36. van den Berg ME, Warren HR, Cabrera CP, Verweij N, Mifsud B, Haessler J, Bihlmeyer NA, Fu YP, Weiss S, Lin HJ, Grarup N, Li‐Gao R, Pistis G, Shah N, Brody JA, Muller‐Nurasyid M, Lin H, Mei H, Smith AV, Lyytikainen LP, Hall LM, van Setten J, Trompet S, Prins BP, Isaacs A, Radmanesh F, Marten J, Entwistle A, Kors JA, Silva CT, Alonso A, Bis JC, de Boer R, de Haan HG, de Mutsert R, Dedoussis G, Dominiczak AF, Doney ASF, Ellinor PT, Eppinga RN, Felix SB, Guo X, Hagemeijer Y, Hansen T, Harris TB, Heckbert SR, Huang PL, Hwang SJ, Kahonen M, Kanters JK, Kolcic I, Launer LJ, Li M, Yao J, Linneberg A, Liu S, Macfarlane PW, Mangino M, Morris AD, Mulas A, Murray AD, Nelson CP, Orru M, Padmanabhan S, Peters A, Porteous DJ, Poulter N, Psaty BM, Qi L, Raitakari OT, Rivadeneira F, Roselli C, Rudan I, Sattar N, Sever P, Sinner MF, Soliman EZ, Spector TD, Stanton AV, Stirrups KE, Taylor KD, Tobin MD, Uitterlinden A, Vaartjes I, Hoes AW, van der Meer P, Volker U, Waldenberger M, Xie Z, Zoledziewska M, Tinker A, Polasek O, Rosand J, Jamshidi Y, van Duijn CM, Zeggini E, Jukema JW, Asselbergs FW, Samani NJ, Lehtimaki T, Gudnason V, Wilson J, Lubitz SA, Kaab S, Sotoodehnia N, Caulfield MJ, Palmer CNA, Sanna S, Mook‐Kanamori DO, Deloukas P, Pedersen O, Rotter JI, Dorr M, O'Donnell CJ, Hayward C, Arking DE, Kooperberg C, van der Harst P, Eijgelsheim M, Stricker BH, Munroe PB. Discovery of novel heart rate‐associated loci using the Exome Chip. Hum Mol Genet. 2017;26:2346–2363.  [DOI](https://doi.org/10.1093/hmg/ddx113) | [PMC free article](/articles/PMC5458336/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28379579/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mol%20Genet&title=Discovery%20of%20novel%20heart%20rate%E2%80%90associated%20loci%20using%20the%20Exome%20Chip&author=ME%20van%20den%20Berg&author=HR%20Warren&author=CP%20Cabrera&author=N%20Verweij&author=B%20Mifsud&volume=26&publication_year=2017&pages=2346-2363&pmid=28379579&doi=10.1093/hmg/ddx113&)

37. Palatini P, Julius S. Elevated heart rate: a major risk factor for cardiovascular disease. Clin Exp Hypertens. 2004;26:637–644.  [DOI](https://doi.org/10.1081/ceh-200031959) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15702618/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Exp%20Hypertens&title=Elevated%20heart%20rate:%20a%20major%20risk%20factor%20for%20cardiovascular%20disease&author=P%20Palatini&author=S%20Julius&volume=26&publication_year=2004&pages=637-644&pmid=15702618&doi=10.1081/ceh-200031959&)

38. Benetos A, Thomas F, Bean K, Albaladejo P, Palatini P, Guize L. Resting heart rate in older people: a predictor of survival to age 85. J Am Geriatr Soc. 2003;51:284–285.  [DOI](https://doi.org/10.1046/j.1532-5415.2003.51080.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12558738/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Am%20Geriatr%20Soc&title=Resting%20heart%20rate%20in%20older%20people:%20a%20predictor%20of%20survival%20to%20age%2085&author=A%20Benetos&author=F%20Thomas&author=K%20Bean&author=P%20Albaladejo&author=P%20Palatini&volume=51&publication_year=2003&pages=284-285&pmid=12558738&doi=10.1046/j.1532-5415.2003.51080.x&)

39. Kannel WB. Risk stratification in hypertension: new insights from the Framingham Study. Am J Hypertens. 2000;13:3S–10S.  [DOI](https://doi.org/10.1016/s0895-7061(99)00252-6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10678282/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hypertens&title=Risk%20stratification%20in%20hypertension:%20new%20insights%20from%20the%20Framingham%20Study&author=WB%20Kannel&volume=13&publication_year=2000&pages=3S-10S&pmid=10678282&doi=10.1016/s0895-7061(99)00252-6&)

40. Jouven X, Empana JP, Schwartz PJ, Desnos M, Courbon D, Ducimetiere P. Heart‐rate profile during exercise as a predictor of sudden death. N Engl J Med. 2005;352:1951–1958.  [DOI](https://doi.org/10.1056/NEJMoa043012) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15888695/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Heart%E2%80%90rate%20profile%20during%20exercise%20as%20a%20predictor%20of%20sudden%20death&author=X%20Jouven&author=JP%20Empana&author=PJ%20Schwartz&author=M%20Desnos&author=D%20Courbon&volume=352&publication_year=2005&pages=1951-1958&pmid=15888695&doi=10.1056/NEJMoa043012&)

41. Metra M, Torp‐Pedersen C, Swedberg K, Cleland JG, Di Lenarda A, Komajda M, Remme WJ, Lutiger B, Scherhag A, Lukas MA, Charlesworth A, Poole‐Wilson PA. Influence of heart rate, blood pressure, and beta‐blocker dose on outcome and the differences in outcome between carvedilol and metoprolol tartrate in patients with chronic heart failure: results from the COMET trial. Eur Heart J. 2005;26:2259–2268.  [DOI](https://doi.org/10.1093/eurheartj/ehi386) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16040619/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20Heart%20J&title=Influence%20of%20heart%20rate,%20blood%20pressure,%20and%20beta%E2%80%90blocker%20dose%20on%20outcome%20and%20the%20differences%20in%20outcome%20between%20carvedilol%20and%20metoprolol%20tartrate%20in%20patients%20with%20chronic%20heart%20failure:%20results%20from%20the%20COMET%20trial&author=M%20Metra&author=C%20Torp%E2%80%90Pedersen&author=K%20Swedberg&author=JG%20Cleland&author=A%20Di%20Lenarda&volume=26&publication_year=2005&pages=2259-2268&pmid=16040619&doi=10.1093/eurheartj/ehi386&)

42. Mulder P, Barbier S, Chagraoui A, Richard V, Henry JP, Lallemand F, Renet S, Lerebours G, Mahlberg‐Gaudin F, Thuillez C. Long‐term heart rate reduction induced by the selective I(f) current inhibitor ivabradine improves left ventricular function and intrinsic myocardial structure in congestive heart failure. Circulation. 2004;109:1674–1679.  [DOI](https://doi.org/10.1161/01.CIR.0000118464.48959.1C) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14981003/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circulation&title=Long%E2%80%90term%20heart%20rate%20reduction%20induced%20by%20the%20selective%20I(f)%20current%20inhibitor%20ivabradine%20improves%20left%20ventricular%20function%20and%20intrinsic%20myocardial%20structure%20in%20congestive%20heart%20failure&author=P%20Mulder&author=S%20Barbier&author=A%20Chagraoui&author=V%20Richard&author=JP%20Henry&volume=109&publication_year=2004&pages=1674-1679&pmid=14981003&doi=10.1161/01.CIR.0000118464.48959.1C&)

43. Human genomics. The Genotype‐Tissue Expression (GTEx) pilot analysis: multitissue gene regulation in humans. Science. 2015;348:648–660.  [DOI](https://doi.org/10.1126/science.1262110) | [PMC free article](/articles/PMC4547484/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25954001/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=Human%20genomics.%20The%20Genotype%E2%80%90Tissue%20Expression%20(GTEx)%20pilot%20analysis:%20multitissue%20gene%20regulation%20in%20humans&volume=348&publication_year=2015&pages=648-660&pmid=25954001&doi=10.1126/science.1262110&)

44. Hague C, Uberti MA, Chen Z, Bush CF, Jones SV, Ressler KJ, Hall RA, Minneman KP. Olfactory receptor surface expression is driven by association with the beta2‐adrenergic receptor. Proc Natl Acad Sci USA. 2004;101:13672–13676.  [DOI](https://doi.org/10.1073/pnas.0403854101) | [PMC free article](/articles/PMC518811/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15347813/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20USA&title=Olfactory%20receptor%20surface%20expression%20is%20driven%20by%20association%20with%20the%20beta2%E2%80%90adrenergic%20receptor&author=C%20Hague&author=MA%20Uberti&author=Z%20Chen&author=CF%20Bush&author=SV%20Jones&volume=101&publication_year=2004&pages=13672-13676&pmid=15347813&doi=10.1073/pnas.0403854101&)

45. Dawson TM, Arriza JL, Jaworsky DE, Borisy FF, Attramadal H, Lefkowitz RJ, Ronnett GV. Beta‐adrenergic receptor kinase‐2 and beta‐arrestin‐2 as mediators of odorant‐induced desensitization. Science. 1993;259:825–829.  [DOI](https://doi.org/10.1126/science.8381559) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8381559/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=Beta%E2%80%90adrenergic%20receptor%20kinase%E2%80%902%20and%20beta%E2%80%90arrestin%E2%80%902%20as%20mediators%20of%20odorant%E2%80%90induced%20desensitization&author=TM%20Dawson&author=JL%20Arriza&author=DE%20Jaworsky&author=FF%20Borisy&author=H%20Attramadal&volume=259&publication_year=1993&pages=825-829&pmid=8381559&doi=10.1126/science.8381559&)

46. Peppel K, Boekhoff I, McDonald P, Breer H, Caron MG, Lefkowitz RJ. G protein‐coupled receptor kinase 3 (GRK3) gene disruption leads to loss of odorant receptor desensitization. J Biol Chem. 1997;272:25425–25428.  [DOI](https://doi.org/10.1074/jbc.272.41.25425) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9325250/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=G%20protein%E2%80%90coupled%20receptor%20kinase%203%20(GRK3)%20gene%20disruption%20leads%20to%20loss%20of%20odorant%20receptor%20desensitization&author=K%20Peppel&author=I%20Boekhoff&author=P%20McDonald&author=H%20Breer&author=MG%20Caron&volume=272&publication_year=1997&pages=25425-25428&pmid=9325250&doi=10.1074/jbc.272.41.25425&)

47. Zaccolo M. cAMP signal transduction in the heart: understanding spatial control for the development of novel therapeutic strategies. Br J Pharmacol. 2009;158:50–60.  [DOI](https://doi.org/10.1111/j.1476-5381.2009.00185.x) | [PMC free article](/articles/PMC2795260/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19371331/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Pharmacol&title=cAMP%20signal%20transduction%20in%20the%20heart:%20understanding%20spatial%20control%20for%20the%20development%20of%20novel%20therapeutic%20strategies&author=M%20Zaccolo&volume=158&publication_year=2009&pages=50-60&pmid=19371331&doi=10.1111/j.1476-5381.2009.00185.x&)

48. Jovancevic N, Dendorfer A, Matzkies M, Kovarova M, Heckmann JC, Osterloh M, Boehm M, Weber L, Nguemo F, Semmler J, Hescheler J, Milting H, Schleicher E, Gelis L, Hatt H. Medium‐chain fatty acids modulate myocardial function via a cardiac odorant receptor. Basic Res Cardiol. 2017;112:13.  [DOI](https://doi.org/10.1007/s00395-017-0600-y) | [PMC free article](/articles/PMC5258789/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28116519/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Basic%20Res%20Cardiol&title=Medium%E2%80%90chain%20fatty%20acids%20modulate%20myocardial%20function%20via%20a%20cardiac%20odorant%20receptor&author=N%20Jovancevic&author=A%20Dendorfer&author=M%20Matzkies&author=M%20Kovarova&author=JC%20Heckmann&volume=112&publication_year=2017&pages=13&pmid=28116519&doi=10.1007/s00395-017-0600-y&)

49. Howard L, Nelson KK, Maciewicz RA, Blobel CP. Interaction of the metalloprotease disintegrins MDC9 and MDC15 with two SH3 domain‐containing proteins, endophilin I and SH3PX1. J Biol Chem. 1999;274:31693–31699.  [DOI](https://doi.org/10.1074/jbc.274.44.31693) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10531379/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=Interaction%20of%20the%20metalloprotease%20disintegrins%20MDC9%20and%20MDC15%20with%20two%20SH3%20domain%E2%80%90containing%20proteins,%20endophilin%20I%20and%20SH3PX1&author=L%20Howard&author=KK%20Nelson&author=RA%20Maciewicz&author=CP%20Blobel&volume=274&publication_year=1999&pages=31693-31699&pmid=10531379&doi=10.1074/jbc.274.44.31693&)

50. Thomsen AR, Plouffe B, Cahill TJ III, Shukla AK, Tarrasch JT, Dosey AM, Kahsai AW, Strachan RT, Pani B, Mahoney JP, Huang L, Breton B, Heydenreich FM, Sunahara RK, Skiniotis G, Bouvier M, Lefkowitz RJ. GPCR‐G protein‐beta‐arrestin super‐complex mediates sustained G protein signaling. Cell. 2016;166:907–919.  [DOI](https://doi.org/10.1016/j.cell.2016.07.004) | [PMC free article](/articles/PMC5418658/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27499021/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell&title=GPCR%E2%80%90G%20protein%E2%80%90beta%E2%80%90arrestin%20super%E2%80%90complex%20mediates%20sustained%20G%20protein%20signaling&author=AR%20Thomsen&author=B%20Plouffe&author=TJ%20Cahill&author=AK%20Shukla&author=JT%20Tarrasch&volume=166&publication_year=2016&pages=907-919&pmid=27499021&doi=10.1016/j.cell.2016.07.004&)

51. Soulet F, Yarar D, Leonard M, Schmid SL. SNX9 regulates dynamin assembly and is required for efficient clathrin‐mediated endocytosis. Mol Biol Cell. 2005;16:2058–2067.  [DOI](https://doi.org/10.1091/mbc.E04-11-1016) | [PMC free article](/articles/PMC1073683/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15703209/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Biol%20Cell&title=SNX9%20regulates%20dynamin%20assembly%20and%20is%20required%20for%20efficient%20clathrin%E2%80%90mediated%20endocytosis&author=F%20Soulet&author=D%20Yarar&author=M%20Leonard&author=SL%20Schmid&volume=16&publication_year=2005&pages=2058-2067&pmid=15703209&doi=10.1091/mbc.E04-11-1016&)

52. Shin N, Ahn N, Chang‐Ileto B, Park J, Takei K, Ahn SG, Kim SA, Di Paolo G, Chang S. SNX9 regulates tubular invagination of the plasma membrane through interaction with actin cytoskeleton and dynamin 2. J Cell Sci. 2008;121:1252–1263.  [DOI](https://doi.org/10.1242/jcs.016709) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18388313/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Cell%20Sci&title=SNX9%20regulates%20tubular%20invagination%20of%20the%20plasma%20membrane%20through%20interaction%20with%20actin%20cytoskeleton%20and%20dynamin%202&author=N%20Shin&author=N%20Ahn&author=B%20Chang%E2%80%90Ileto&author=J%20Park&author=K%20Takei&volume=121&publication_year=2008&pages=1252-1263&pmid=18388313&doi=10.1242/jcs.016709&)

53. Lundmark R, Carlsson SR. Sorting nexin 9 participates in clathrin‐mediated endocytosis through interactions with the core components. J Biol Chem. 2003;278:46772–46781.  [DOI](https://doi.org/10.1074/jbc.M307334200) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12952949/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=Sorting%20nexin%209%20participates%20in%20clathrin%E2%80%90mediated%20endocytosis%20through%20interactions%20with%20the%20core%20components&author=R%20Lundmark&author=SR%20Carlsson&volume=278&publication_year=2003&pages=46772-46781&pmid=12952949&doi=10.1074/jbc.M307334200&)

54. Wang Y, Lauffer B, Von Zastrow M, Kobilka BK, Xiang Y. N‐ethylmaleimide‐sensitive factor regulates beta2 adrenoceptor trafficking and signaling in cardiomyocytes. Mol Pharmacol. 2007;72:429–439.  [DOI](https://doi.org/10.1124/mol.107.037747) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17510209/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Pharmacol&title=N%E2%80%90ethylmaleimide%E2%80%90sensitive%20factor%20regulates%20beta2%20adrenoceptor%20trafficking%20and%20signaling%20in%20cardiomyocytes&author=Y%20Wang&author=B%20Lauffer&author=M%20Von%20Zastrow&author=BK%20Kobilka&author=Y%20Xiang&volume=72&publication_year=2007&pages=429-439&pmid=17510209&doi=10.1124/mol.107.037747&)

55. Hanyaloglu AC, von Zastrow M. Regulation of GPCRs by endocytic membrane trafficking and its potential implications. Annu Rev Pharmacol Toxicol. 2008;48:537–568.  [DOI](https://doi.org/10.1146/annurev.pharmtox.48.113006.094830) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18184106/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Annu%20Rev%20Pharmacol%20Toxicol&title=Regulation%20of%20GPCRs%20by%20endocytic%20membrane%20trafficking%20and%20its%20potential%20implications&author=AC%20Hanyaloglu&author=M%20von%20Zastrow&volume=48&publication_year=2008&pages=537-568&pmid=18184106&doi=10.1146/annurev.pharmtox.48.113006.094830&)

56. Lundmark R, Carlsson SR. SNX9—a prelude to vesicle release. J Cell Sci. 2009;122:5–11.  [DOI](https://doi.org/10.1242/jcs.037135) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19092055/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Cell%20Sci&title=SNX9%E2%80%94a%20prelude%20to%20vesicle%20release&author=R%20Lundmark&author=SR%20Carlsson&volume=122&publication_year=2009&pages=5-11&pmid=19092055&doi=10.1242/jcs.037135&)

57. Lundmark R, Carlsson SR. The beta‐appendages of the four adaptor‐protein (AP) complexes: structure and binding properties, and identification of sorting nexin 9 as an accessory protein to AP‐2. Biochem J. 2002;362:597–607.  [DOI](https://doi.org/10.1042/0264-6021:3620597) | [PMC free article](/articles/PMC1222423/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11879186/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem%20J&title=The%20beta%E2%80%90appendages%20of%20the%20four%20adaptor%E2%80%90protein%20(AP)%20complexes:%20structure%20and%20binding%20properties,%20and%20identification%20of%20sorting%20nexin%209%20as%20an%20accessory%20protein%20to%20AP%E2%80%902&author=R%20Lundmark&author=SR%20Carlsson&volume=362&publication_year=2002&pages=597-607&pmid=11879186&doi=10.1042/0264-6021:3620597&)

58. Pearse BM, Smith CJ, Owen DJ. Clathrin coat construction in endocytosis. Curr Opin Struct Biol. 2000;10:220–228.  [DOI](https://doi.org/10.1016/s0959-440x(00)00071-3) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10753805/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Opin%20Struct%20Biol&title=Clathrin%20coat%20construction%20in%20endocytosis&author=BM%20Pearse&author=CJ%20Smith&author=DJ%20Owen&volume=10&publication_year=2000&pages=220-228&pmid=10753805&doi=10.1016/s0959-440x(00)00071-3&)

59. Laporte SA, Oakley RH, Holt JA, Barak LS, Caron MG. The interaction of beta‐arrestin with the AP‐2 adaptor is required for the clustering of beta 2‐adrenergic receptor into clathrin‐coated pits. J Biol Chem. 2000;275:23120–23126.  [DOI](https://doi.org/10.1074/jbc.M002581200) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10770944/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=The%20interaction%20of%20beta%E2%80%90arrestin%20with%20the%20AP%E2%80%902%20adaptor%20is%20required%20for%20the%20clustering%20of%20beta%202%E2%80%90adrenergic%20receptor%20into%20clathrin%E2%80%90coated%20pits&author=SA%20Laporte&author=RH%20Oakley&author=JA%20Holt&author=LS%20Barak&author=MG%20Caron&volume=275&publication_year=2000&pages=23120-23126&pmid=10770944&doi=10.1074/jbc.M002581200&)

60. Shenoy SK, Lefkowitz RJ. Seven‐transmembrane receptor signaling through beta‐arrestin. Sci STKE. 2005;2005:cm10.  [DOI](https://doi.org/10.1126/stke.2005/308/cm10) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16267056/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Sci%20STKE&title=Seven%E2%80%90transmembrane%20receptor%20signaling%20through%20beta%E2%80%90arrestin&author=SK%20Shenoy&author=RJ%20Lefkowitz&volume=2005&publication_year=2005&pages=cm10&pmid=16267056&doi=10.1126/stke.2005/308/cm10&)
