# ABCG2 Regulatory SNPs Alter In Vivo Enhancer Activity and Expression

## Metadata
**Authors:** Rachel J Eclov, Mee J Kim, Aparna Chhibber, Robin P Smith, Nadav Ahituv, Deanna L Kroetz
**Journal:** Pharmacogenetics and genomics
**Date:** 2017 Dec
**DOI:** [10.1097/FPC.0000000000000312](https://doi.org/10.1097/FPC.0000000000000312)
**PMID:** 28930109
**PMCID:** PMC5678922
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5678922/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC5678922/pdf/nihms901528.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC5678922/pdf/nihms901528.pdf)

## Abstract

**Objectives:** 
The expression and activity of the breast cancer resistance protein (ABCG2) contributes to the pharmacokinetics of endogenous and xenobiotic substrates. The effect of genetic variation on the activity of cis-regulatory elements and nuclear response elements in the ABCG2 locus and their contribution to ABCG2 expression has not been systematically investigated. In this study, the effect of genetic variation on the in vitro and in vivo enhancer activity of six previously identified liver enhancers in the ABCG2 locus was examined.

**Methods:** 
Reference and variant liver enhancers were tested for their ability to alter luciferase activity in vitro in HepG2 and HEK293T cell lines and in vivo using a hydrodynamic tail vein assay. Positive in vivo SNPs were tested for association with gene expression and for altered protein binding in electrophoretic mobility shift assays (EMSA).

**Results:** 
Multiple SNPs were found to alter enhancer activity in vitro. Four of these variants (rs9999111, rs12508471, ABCG2RE1*2 and rs149713212) decreased and one (rs2725263) increased enhancer activity in vivo. Additionally, rs9999111 and rs12508471 were associated with ABCG2 expression in lymphoblastoid cell lines, lymphocytes and T cells, and showed increased HepG2 nuclear protein binding.

**Conclusion:** 
This study identifies SNPs within regulatory regions of the ABCG2 locus that alter enhancer activity in vitro and in vivo. Several of these SNPs correlate with tissue-specific ABCG2 expression and alter DNA/protein binding. These SNPs could contribute to reported tissue-specific variability in ABCG2 expression and may influence the correlation between ABCG2 expression and disease risk or the pharmacokinetics and pharmacodynamics of BCRP substrates.

Keywords: Enhancer, ABCG2, BCRP, breast cancer resistance protein, ABC transporter, pharmacogenomics, polymorphism, transcriptional regulation

### Objectives

The expression and activity of the breast cancer resistance protein (*ABCG2*) contributes to the pharmacokinetics of endogenous and xenobiotic substrates. The effect of genetic variation on the activity of *cis*-regulatory elements and nuclear response elements in the *ABCG2* locus and their contribution to ABCG2 expression has not been systematically investigated. In this study, the effect of genetic variation on the *in vitro* and *in vivo* enhancer activity of six previously identified liver enhancers in the *ABCG2* locus was examined.

### Methods

Reference and variant liver enhancers were tested for their ability to alter luciferase activity *in vitro* in HepG2 and HEK293T cell lines and *in vivo* using a hydrodynamic tail vein assay. Positive *in vivo* SNPs were tested for association with gene expression and for altered protein binding in electrophoretic mobility shift assays (EMSA).

### Results

Multiple SNPs were found to alter enhancer activity *in vitro*. Four of these variants (rs9999111, rs12508471, ABCG2RE1*2 and rs149713212) decreased and one (rs2725263) increased enhancer activity *in vivo*. Additionally, rs9999111 and rs12508471 were associated with ABCG2 expression in lymphoblastoid cell lines, lymphocytes and T cells, and showed increased HepG2 nuclear protein binding.

### Conclusion

This study identifies SNPs within regulatory regions of the *ABCG2* locus that alter enhancer activity *in vitro* and *in vivo*. Several of these SNPs correlate with tissue-specific ABCG2 expression and alter DNA/protein binding. These SNPs could contribute to reported tissue-specific variability in ABCG2 expression and may influence the correlation between ABCG2 expression and disease risk or the pharmacokinetics and pharmacodynamics of BCRP substrates.

## Introduction

The breast cancer resistance protein (BCRP), encoded by *ABCG2*, is a member of the ATP-binding cassette (ABC) membrane transporter family and is responsible for transport of its substrates across intestinal epithelial cells into the intestinal lumen, from the hepatocyte into the bile, into milk, away from the placenta and brain, and into the lumen of the renal proximal tubule[[1](#R1)]. Reduced expression and function of MXR are associated with a variety of adverse events, such as pheophorbide-induced phototoxicity[[2](#R2)], urate-induced gout[[3](#R3)], gefitinib-induced diarrhea[[4](#R4)], as well as reduced chemotherapy response and increased susceptibility to cancer[[5](#R5)–[8](#R8)]. Although reduced function coding variants of BCRP exist[[9](#R9)], variability in ABCG2 expression and BCRP substrate pharmacokinetics cannot be accounted for solely by these nonsynonymous variants. Even in individuals without these variants, there is a wide range of ABCG2 expression[[10](#R10)]. Identifying regulatory regions of *ABCG2* and functional single nucleotide polymorphisms (SNPs) within those regions may inform about the mechanisms of genetic regulation of ABCG2 expression.

BCRP is one of many transporters important in drug absorption, distribution, metabolism and excretion (ADME). Recently, the transcriptional regulation of ADME genes has been linked to *cis*-regulatory elements, and alterations in ADME gene expression due to variants in those regulatory elements are becoming more evident[[11](#R11),[12](#R12)]. Additionally, expression quantitative trait loci (eQTL) studies of human genes have implicated proximal regulatory variation as a prevalent cause of population variation in gene expression[[13](#R13),[14](#R14)]. *cis*-Regulatory elements include enhancers, suppressors, promoters, insulators and locus control regions that work to regulate the transcriptional activity of the basal transcription machinery. These genomic regions provide binding sites for transcription factors, either ubiquitous or tissue-specific, that work through complex interactions with histones, RNA polymerase and other transcription factors to determine gene transcription. In previous studies, we utilized comparative genomics along with *in vitro* and *in vivo* assays to identify six liver enhancers in the *ABCG2* gene locus[[15](#R15)].

In this study, the hypothesis that SNPs in regulatory regions of the *ABCG2* locus contribute to the variation in ABCG2 expression was tested. Variants in liver enhancer regions reported in publicly available databases were studied *in vitro* in kidney and liver cell lines, and variants with altered function were tested *in vivo* using a hydrodynamic tail vein assay. SNPs that significantly altered *in vivo* liver enhancer activity were then tested for association with ABCG2 expression in human liver, kidney, placenta, breast, lymphocytes, T cells and fibroblasts. Based on supporting ENCODE data indicating transcriptional activity for five of six of these enhancer regions, EMSAs were used to confirm changes in protein/DNA binding for SNPs that correlate with gene expression. The findings from this study provide insight into how non-coding genetic variants may lead to altered exposure to BCRP substrates.

## Methods

### Genetic Analysis of Enhancer Regions

SNPs in each of the *ABCG2 in vivo* enhancer regions were retrieved for all available ethnic populations from publicly available databases, including 1000 Genomes 20120214 phase 1 release[[16](#R16)], dbSNP build 135 and HapMap release 28[[17](#R17)]. SNPs in linkage disequilibrium with rs12508471, rs72873421, rs149713212, rs9999111 and rs2725263 (r^2^ threshold ≥ 0.8) were extracted from 1000 Genomes pilot 1 genotype data using the Broad Institute SNP annotation and proxy search (SNAP) version 2.2[[18](#R18)] for each population (CEU, YRI and CHB+JBT) separately and linkage analysis was performed using the Haploview program version 4.2[[19](#R19)]

### Variant Enhancer Plasmid Construction

Reference enhancer plasmids in the pGL4.23 vector were previously described[[15](#R15)]. Site-directed mutagenesis (SDM) on plasmids was performed using specific primers ([Supplemental Table 1](#SD1)) and Phusion High-Fidelity DNA Polymerase following the manufacturer’s protocol. PCR reaction conditions are available in the [supplemental materials and methods](#SD1). Primers and PCR conditions for the deletion SNP rs36105707 were designed according to a large deletion protocol[[20](#R20)]. Endotoxin-free DNA for all vectors were isolated using the GenElute HP Endotoxin-Free Maxiprep Kit following the manufacturer’s protocol.

### Cell Culture and Transfections

HEK293T/17 and HepG2 cell lines were cultured and transfected for *in vitro* luciferase assays with Lipofectamine 2000 following the manufacturer’s protocol as previously described[[15](#R15)]. Cells were lysed 18–24 hr after transfection and measured for firefly and *Renilla* luciferase activity using the Dual-Luciferase^®^ Reporter Assay System in a GloMax 96 microplate Dual Injector Luminometer (Promega, Madison, WI) following the manufacturer’s protocol. Each experiment also included the empty pGL4.23 vector and the *ApoE*-pGL4.23[[21](#R21)] or pGL4.13 plasmids. Enhancer activity was expressed as the ratio of the plasmid firefly to *Renilla* luciferase activity; activity of each variant plasmid was then normalized relative to the reference plasmid, setting the reference activity to one (100%).

### Hydrodynamic Tail Vein Assay

Selected positive *in vitro* variant enhancer elements were screened for their effect on *in vivo* liver enhancer activity using the hydrodynamic tail vein injection adapted for enhancer activity screening[[22](#R22),[23](#R23)]. Each variant enhancer, along with their reference enhancer plasmid, the *ApoE*[[21](#R21)] positive control liver enhancer and an empty pGL4.23 vector, were injected individually into the tail vein of 4–11 mice and hepatic luciferase activity measured after 24 hr as previously described[[15](#R15)]. Each plasmid’s firefly activity was normalized to *Renilla* luciferase activity and expressed as fold activity relative to the negative control, empty pGL4.23. All mouse work was done following a protocol approved by the University of California San Francisco Institutional Animal Care and Use Committee.

### Liver and Kidney Tissues

Kidney (n=60) and liver (n=60) samples were procured by the PMT research group at the University of California San Francisco (San Francisco, CA)[[24](#R24)] from Asterand (Detroit, MI), Capital Biosciences (Rockville, MD) and SRI International (Menlo Park, CA). DNA was extracted and purified from the tissues using a Qiagen AllPrep DNA/RNA Mini Kit and QIAquick PCR Purification Kit following the manufacturer’s protocols. RNA was extracted from the tissues following the protocol for Trizol reagent and cleaned-up with the Qiagen RNeasy MinElute Cleanup Kit following the manufacturer’s protocol. High quality RNA was isolated from 58 kidney and 60 liver samples and those with 260/280 >1.7, 260/230 >1.8, and RNA Integrity number from Bioanalyzer of 3–8 were used to correlate SNP genotype with total ABCG2 mRNA expression. RNA was reverse transcribed to cDNA using the High Capacity cDNA Reverse Transcription Kit (Applied Biosystems, Foster City, CA).

### ABCG2 mRNA Expression and Genotype in PMT Liver and Kidney Tissues

ABCG2 gene expression was evaluated in 58 kidney and 60 liver samples using the Biotrove Open Array™ qPCR platform (Life Technologies, Carlsbad, CA) according to the manufacturer’s protocol. ABCG2 mRNA expression was normalized to a geometric mean of the expression of GAPDH, β-2 microglobin, and β-actin and expressed as 2^−ΔΔCt^ per gene for each sample. All ∆Ct values for a given tissue type were quantile normalized across samples using the open source R preprocess Core package[[25](#R25),[26](#R26)]. Expression data was quality controlled using principal component analysis to identify outliers. Of these samples, 58 kidney and 34 liver samples were successfully genotyped on the Affymetrix Axiom genotyping platform using the Axiom® Genome-Wide CEU 1 Array Plate (Santa Clara, CA). The samples were tested for quality control (QC) using sex-check, identity by descent and call rate tests, of which six kidney samples failed and were excluded from further analysis. After initial QC, 52 kidney samples and 34 liver samples were included in subsequent analyses.

### Association of SNPs with Gene Expression

Genotype and expression data from PMT liver and kidney tissue, 195 samples from Schadt *et al*.[[27](#R27)] liver tissue and 62 samples from The Cancer Genome Atlas (TCGA) for breast tissue[[28](#R28)] were analyzed for associations between enhancer variants and ABCG2 expression levels. Genotypes were imputed using 1000 Genomes data and then tested for correlation between the expression of ABCG2, PPM1K and PKD2 and genotype using a linear regression and the Affymetrix Genotyping Console (Santa Clara, CA). The PMT liver and kidney linear regression were performed after adjusting for gender.

A database for integrated analysis and visualization of SNP-gene associations in eQTL studies, Genevar[[29](#R29)], includes data from several sequence and gene expression profiling studies: the MuTHER study[[30](#R30),[31](#R31)] with data from female twins in adipose (166 samples), skin (160 samples) and lymphoblastoid cell lines (LCLs, 156 samples), the Stranger study[[32](#R32)–[35](#R35)] with data from 726 HapMap LCL samples, and the GenCord study[[36](#R36)] with data from 85 human umbilical fibroblasts, LCLs and T cell samples. The expression of ABCG2, and the neighboring 5’ (PPM1K) and 3’ (PKD2) genes was correlated to the *ABCG2* locus SNPs that altered enhancer activity *in vivo*, or with SNPs in LD (r^2^>0.8, as determined above) with these SNPs. Using the GeneVar 3.2.0 eQTL analysis program, Spearman rank correlation coefficients (rho) for 10,000 permutations per SNP between reference, heterozygous and variant alleles were calculated.

### Electrophoretic Mobility Shift Assay

Electrophoretic mobility shift assays (EMSA) were performed using the Odyssey EMSA buffer kit following the manufacturer’s protocol. EMSA probes sequences are available in the [supplemental materials](#SD1). Competition assays were preformed by adding 40-fold molar excess of unlabeled reference or SNP oligonucleotide. DNA/protein complexes were separated from free probe by gel electrophoresis and imaged using the Licor system (Odyssey, Lincoln, NE).

### Statistical Analysis

Normalized polymorphic enhancer activities were compared to the reference enhancer per transfection (3–8 wells/plasmid) using an ANOVA analysis followed by a Bonferroni’s multiple comparison *t*-test with *P* < 0.05 considered significant. Polymorphic enhancer constructs identified for *in vivo* testing had either a 2-fold increase or decrease in activity and a *P* < 0.0001 in both cell lines. Results from the hydrodynamic tail vein injection were analyzed using an unpaired Student’s *t*-test between the reference and variant enhancer (for only one variant) or an ANOVA analysis followed by a Bonferroni’s multiple comparison *t*-test (for two or more variants); *P* < 0.05 was considered significant for both tests. All statistics were run using the GraphPad Prism 5 software program (San Diego, CA).

## Results

### Genetic Variation in the ABCG2 Locus Enhancer Regions

A total of fifty-three SNPs and three haplotypes in the six *in vivo* enhancer regions were obtained from publicly available databases ([Table 1](#T1)). There are three SNPs in ABCG2RE14, ten in ABCG2RE26 and eight in ABCG2RE6. None of these SNPs are in linkage disequilibrium (LD) with each other, and only rs9999111 and rs2725268 had a minor allele frequency (MAF) ≥ 5% in at least one population. There are six SNPs in the ABCG2RE1 region. In addition, ABCG2RE1 has one haplotype (ABCG2RE1*2), which is a combination of the SNPs rs72873421, rs12500008 and rs12508471. These three SNPs have MAFs ranging from 8.4–38% and are in near perfect LD (r^2^=0.96–0.98). There are twelve SNPs in ABCG2RE8 and one haplotype, ABCG2RE8*2, which is a combination of the two most frequent SNPs rs2725263 and rs2725264. These two SNPs have MAFs ranging from 8–87% depending on the ethnic population, and the ABCG2RE8*2 haplotype has a frequency similar to the individual SNPs (7–79%) with an r^2^ = 0.46 between the SNPs. There are fourteen SNPs and one haplotype in ABCG2RE9. The ABCG2RE9*2 (rs41282399 and rs2622628) haplotype has a frequency from 1.7–3.7% with an r^2^ = 0.13 between SNPs.

### Table 1.

| Region | SNP ID | Location1 | ΔNT3 | Conserved3 | MAF2 % |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- |
|   |  |  |  |  |  |  |  |
| AA | EUR | AS |  |  |  |  |  |
| ABCG2RE1 | rs72873421 | chr4:88923906 | G>A | Yes | 21.1 | 8.4 | 38.1 |
|   | rs117741074 | chr4:88924002 | G>A | Yes | 0.0 | 0.0 | 1.7 |
|   | rs12500008 | chr4:88924176 | C>A | Yes | 21.1 | 8.4 | 38.3 |
|   | rs2728131 | chr4:88924344 | C>T | Yes | 1.2 | 8.4 | 2.8 |
|   | rs12508471 | chr4:88924371 | A>G | Yes | 21.1 | 8.4 | 38.3 |
|   | rs78901673 | chr4:88924356 | A>G | Yes | nr | nr | nr |
|   | ABCG2RE1*24 | - | - | - | 21.1 | 8.4 | 37.2 |
|   |  |  |  |  |  |  |  |
| ABCG2RE6 | rs45510401 | chr4:89011422 | A>G | No | 4.5 | 0.0 | 0.0 |
|   | rs2725268 | chr4:89010983 | A>G | Yes | 7.5 | 47.4 | 24.0 |
|   | rs57351915 | chr4:89011141 | A>- | Yes | nr | nr | nr |
|   | rs58830217 | chr4:89011309 | ->A | No | nr | nr | nr |
|   | rs186188962 | chr4:89011051 | G>A | No | 0.0 | 0.5 | 0.0 |
|   | rs144180103 | chr4:89011129 | A>- | No | nr | nr | nr |
|   | rs190754327 | chr4:89011173 | C>G | Yes | nr | nr | nr |
|   | rs183322988 | chr4:89011437 | T>C | No | 0.0 | 0.3 | 0.3 |
|   |  |  |  |  |  |  |  |
| ABCG2RE8 | rs139101431 | chr4:89026007 | A>G | No | 1.6 | 0.0 | 0.0 |
|   | rs144062279 | chr4:89026087 | G>A | No | 0.4 | 0.0 | 0.0 |
|   | rs2725264 | chr4:89026109 | C>T | Yes | 83.7 | 8.2 | 21.5 |
|   | rs4148156 | chr4:89026242 | C>T | Yes | nr | nr | nr |
|   | rs145932752 | chr4:89026407 | A>G | No | 1.6 | 0.0 | 0.0 |
|   | rs6831395 | chr4:89026420 | G>A | Yes | 2.2 | 0.0 | 0.0 |
|   | rs2725263 | chr4:89026428 | A>C | Yes | 87.0 | 48.4 | 40.0 |
|   | rs192562676 | chr4:89026490 | C>T | No | 0.0 | 0.0 | 1.0 |
|   | rs182159263 | chr4:89026493 | C>G | No | 0.0 | 0.7 | 0.0 |
|   | rs187527722 | chr4:89026537 | A>G | No | 0.0 | 0.0 | 0.2 |
|   | rs192781547 | chr4:89026630 | A>G | No | 0.0 | 0.0 | 0.3 |
|   | rs184709106 | chr4:89026642 | C>T | No | 0.0 | 0.0 | 0.3 |
|   | ABCG2RE8*24 | - | - | - | 78.8 | 7.3 | 21.5 |
| ABCG2RE9 | rs2231148 | chr4:89028478 | T>A | Yes | 4.3 | 40.2 | 18.5 |
|   | rs190738974 | chr4:89028490 | A>G | No | 0.0 | 0.0 | 0.2 |
|   | rs117761897 | chr4:89028542 | C>T | Yes | 0.0 | 0.0 | 0.2 |
|   | rs41282399 | chr4:89028544 | A>C | Yes | 4.1 | 1.7 | 3.3 |
|   | rs113647079 | chr4:89028578 | C>G | Yes | nr | nr | nr |
|   | rs2054576 | chr4:89028775 | A>G | Yes | 1.2 | 7.9 | 25.0 |
|   | rs151266026 | chr4:89028935 | T>C | No | 0.0 | 0.0 | 0.7 |
|   | rs183315559 | chr4:89028979 | G>A | No | 0.0 | 0.0 | 0.3 |
|   | rs189214307 | chr4:89029111 | C>T | No | 0.0 | 0.0 | 0.3 |
|   | rs2622628 | chr4:89029252 | A>C | Yes | 45.5 | 4.4 | 20.8 |
|   | rs36105707 | chr4:89029304 | ->TTAAT | Yes | nr | nr | nr |
|   | rs141635727 | chr4:89029335 | A>G | No | 0.4 | 0.0 | 0.0 |
|   | rs190767980 | chr4:89029361 | C>T | No | 0.0 | 0.0 | 0.0 |
|   | rs147070185 | chr4:89029364 | G>A | No | 0.0 | 0.0 | 0.2 |
|   | ABCG2RE9*24 | - | - | - | 3.7 | 1.7 | 3.1 |
|   |  |  |  |  |  |  |  |
| ABCG2RE14 | rs9999111 | chr4:89073197 | A>C | Yes | 4.7 | 7.3 | 0.0 |
|   | rs138867860 | chr4:89073397 | C>A | No | 0.0 | 0.0 | 0.0 |
|   | rs114916387 | chr4:89073289 | T>C | Yes | 2.0 | 0.0 | 0.5 |
|   |  |  |  |  |  |  |  |
| ABCG2RE26 | rs137884075 | chr4:89189556 | C>T | No | 0.0 | 0.0 | 0.2 |
|   | rs142621223 | chr4:89189557 | G>A | No | 0.0 | 0.0 | 0.3 |
|   | rs139553964 | chr4:89189571 | G>A | No | 0.0 | 0.0 | 1.2 |
|   | rs149713212 | chr4:89189602 | G>A | No | 1.6 | 0.0 | 0.2 |
|   | rs144565932 | chr4:89189634 | G>A | No | 0.0 | 0.0 | 1.2 |
|   | rs62309980 | chr4:89189655 | C>T | Yes | 0.0 | 0.1 | 0.0 |
|   | rs76888829 | chr4:89190325 | T>C | Yes | 4.3 | 1.3 | 1.2 |
|   | rs9998634 | chr4:89190395 | G>C | Yes | 0.4 | 0.0 | 0.0 |
|   | rs77538297 | chr4:89189971 | C>T | Yes | 0.2 | 0.0 | 0.0 |
|   | rs35696062 | chr4:89190022 | G>- | Yes | nr | nr | nr |

Table 1 Caption: Variants in ABCG2 Locus Enhancer Elements

### Effect of SNPs on In Vitro Enhancer Activity

All fifty-three SNPs in the six enhancer regions, along with the ABCG2RE1*2, ABCG2RE8*2 and ABCG2RE9*2 haplotypes, were tested for differential regulatory activity in HepG2 ([Figure 1](#F1)) and HEK293T ([Figure 2](#F2)) cell lines ([Supplemental Table 2](#SD1)). Activity of the variant and reference enhancers was determined by their ability to drive the expression of luciferase. Six ABCG2RE26 SNPs and two ABCG2RE14 SNPs increased enhancer activity in both cell lines (*P* < 0.001). The ABCG2RE14 rs9999111 and ABCG2RE26 rs149713212 SNPs caused a reduction in activity in both cell lines to only 6–20% of the activity with the corresponding reference sequences (*P* < 0.001). Three ABCG2RE6 SNPs caused decreases in enhancer activity to levels that were 50–80% of control in both cell lines (*P* < 0.05). The only ABCG2RE6 SNP to increase enhancer activity (~1.5-fold, *P* < 0.001) in both cell lines was rs183322988. ABCG2RE1 rs72873421 caused a greater than 2-fold increase in enhancer activity and rs12508471 resulted in an almost complete loss of activity in both cell lines (*P* < 0.001). The ABCG2RE1*2 (rs72873421, rs12500008 and rs12508471) haplotype did not have a significant effect in HepG2 cells but reduced enhancer activity (*P* < 0.001) in the HEK293T cell line. The ABCG2RE8 SNPs rs2725263 and rs2725264 and the ABCG2RE9 SNP rs190738974 increased enhancer activity in the hepatic and renal cell lines 1.35- to 1.68-fold; ABCG2RE9 rs36105707 was the most potent enhancer in HepG2 cells (3.39-fold above control; *P* < 0.001) and one of the most potent enhancer in HEK293 cells (2.01-fold above control; *P* < 0.001). While both ABCG2RE8 SNPs rs2725263 and rs2725264 caused a modest increase in enhancer activity (1.36- to 1.64-fold; *P* < 0.05), the combination of these SNPs as the ABCG2RE8*2 haplotype did not affect enhancer activity.

### Figure 1. Effect of Enhancer Variants in HepG2 Cells.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6e1a/5678922/d2b1389ab221/nihms901528f1.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5678922_nihms901528f1.jpg)

The luciferase activity of reference and variant A) ABCG2RE14, B) ABCG2RE26, C) ABCG2RE6, D) ABCG2RE1, E) ABCG2RE8 and F) ABCG2RE9 enhancer regions were measured in a transiently transfected liver cell line. Enhancer activity is expressed as the ratio of firefly to Renilla luciferase activity normalized to the reference vector activity (reference is set to 1). SNPs are displayed respective to their genomic orientation. Data is expressed as the mean ± SEM from a representative experiment with 3–6 wells per sequence. Differences between reference and variant enhancers were tested by an ANOVA followed by a post-hoc Bonferroni’s multiple comparison t-test; * P < 0.05, ** P < 0.001, *** P < 0.0001.

### Figure 2. Effect of Enhancer Variants in HEK293T Cells.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6e1a/5678922/2b4d45b8a7b8/nihms901528f2.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5678922_nihms901528f2.jpg)

The luciferase activity of reference and variant A) ABCG2RE14, B) ABCG2RE26, C) ABCG2RE6, D) ABCG2RE1, E) ABCG2RE8 and F) ABCG2RE9 enhancer regions were measured in a transiently transfected kidney cell line. Enhancer activity is expressed as the ratio of firefly to Renilla luciferase activity normalized to the reference vector activity (reference is set to 1). SNPs are displayed respective to their genomic orientation. Data is expressed as the mean ± SEM from a representative experiment with 3–6 wells per sequence. Differences between reference and variant enhancers were tested by an ANOVA followed by a post-hoc Bonferroni’s multiple comparison t-test; * P < 0.05, ** P < 0.001, *** P < 0.0001.

For subsequent *in vivo* validation, SNPs were selected from amongst the most significant for each of the enhancer regions. The ABCG2RE14 rs9999111 and ABCG2RE26 rs149713212 SNPs were chosen because they reduced *in vitro* activity to <20% of control (*P* < 0.001). Of the four SNPs that significantly altered the ABCG2RE1 enhancer activity in both cell lines, two of them, rs72873421 and rs12508471, and the ABCG2RE1*2 haplotype, were chosen. Since ABCG2RE6 is a weak *in vivo* enhancer (~2- fold)[[15](#R15)], only rs183322988 was chosen for *in vivo* validation because it increased enhancer activity *in vitro* (*P* < 0.001). The ABCG2RE8 rs2725263 SNP was chosen because it had a more consistent increase in enhancer function in vitro compared to rs2725264. Finally, the ABCG2RE9 rs190738974 SNP was chosen for *in vivo* follow-up over the rs36105707 SNP based on predicted changes in TFBS (data not shown).

### Effect of SNPs on Enhancer Activity In Vivo

Seven SNPs and one haplotype were screened for their effect on *in vivo* liver enhancer activity using the hydrodynamic tail vein injection assay[[22](#R22)]. The *ApoE* liver specific enhancer[[21](#R21)] exhibited a 50-fold activity and the six reference enhancers had a range of 2- to 50-fold activity relative to pGL4.23 ([Figure 3](#F3)). Three of the variants resulted in decreased *in vivo* enhancer activity compared to their respective reference enhancer ([Figure 3](#F3), *P* < 0.05), [Supplemental Table 2](#SD1)). Enhancer activity for ABCG2RE14 rs9999111 was only 30% of the reference sequence (*P* < 0.05). Enhancer activity for ABCG2RE1*2 was almost completely absent (6% of control) *in vivo* (*P* < 0.001). ABCG2RE1 rs12508471 and ABCG2RE26 rs149713212 SNPs also both resulted in an almost complete loss of enhancer activity (*P* < 0.001). The only SNP to increase enhancer activity *in vivo* was the ABCG2RE8 SNP rs2725263, which increased enhancer activity by 1.5- fold (*P* < 0.05). ABCG2RE6 rs183322988, ABCG2RE1 rs72873421 and ABCG2RE9 rs190738974 SNPs had no effect on enhancer activity *in vivo*. All SNPs that significantly altered activity *in vivo* were consistent with their effect on activity *in vitro*.

### Figure 3. Effect of Enhancer Variants In Vivo.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6e1a/5678922/010ac2446300/nihms901528f3.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5678922_nihms901528f3.jpg)

The luciferase activity in mouse liver homogenates was measured 24 hr after plasmid injection. Enhancer activity is expressed as the ratio of firefly to Renilla luciferase activity normalized to the empty vector (pGL4.23) activity. An enhancer for ApoE was used as the positive control[21]. Data is expressed as the mean ± SEM for 4–11 mice. Differences between reference and variant enhancer elements were tested by an unpaired Student’s t-test (one variant) or an ANOVA followed by a Bonferroni’s multiple comparison t-test (for two or more variants); * P < 0.05, ** P < 0.001, *** P < 0.0001.

### Associations of SNPs with mRNA Expression Levels

SNPs that showed an effect on *in vivo* liver enhancer activity were tested for their association with expression levels of ABCG2 in selected tissues as described in the Methods section. The ABCG2RE14 SNP rs9999111 was significantly associated with lower ABCG2 expression in human umbilical cord LCLs and T cells from GenCord[[36](#R36)] tissues (*P* < 0.05; rho = −0.24, [Figure 4](#F4)). However, rs9999111 had no association with ABCG2 expression in the PMT liver and kidney samples ([Supplemental Figure 1](#SD1)). ABCG2 expression was also not correlated with rs9999111 in the Schadt[[27](#R27)] liver tissues (data not shown).

### Figure 4. Association of rs9999111 with ABCG2 expression.

![Figure 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6e1a/5678922/1e186cef422f/nihms901528f4.jpg)

Association of the ABCG2RE14 SNP rs9999111 with ABCG2 mRNA expression in 85 human umbilical cord (A) lymphocytes and (B) T-Cells from the GenCord study[36]. Analysis was done using the GeneVar[29] program with a linear regression and are displayed as gene expression versus rs9999111 genotype with rho (correlation coefficient), P value (P) and empirical P value (Pemp) indicated.

The ABCG2RE1 SNP rs12500008, which occurs only as the ABCG2RE1*2 haplotype and is in perfect LD with rs12508471, was associated with lower ABCG2 expression in LCLs of Chinese individuals (rho = −0.251, *P* = 0.025; [Figure 5A](#F5)) and trended toward significance with ABCG2 expression in the Kenyan population (rho = −0.181, *P* = 0.1; [Figure 5B](#F5)). ABCG2RE1*2 associated with lower PPM1K expression in adipose and skin (rho = −0.26, *P* = 0.015 and rho = −0.32, *P* = 0.005 respectively; [Supplemental Figure 2](#SD1)). ABCG2RE1*2 was also associated with lower *PPM1K* expression in LCLs using two probes for PPM1K mRNA (rho = −0.22, *P* = 0.06 and rho = −0.22, *P* = 0.047; [Supplemental Figure 2](#SD1)). All ABCG2RE1*2 associations with PPM1K and ABCG2 had similar effect sizes (rho ~ −0.25).

### Figure 5. Association of rs12500008 with ABCG2 expression.

![Figure 5](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6e1a/5678922/275e6ea31999/nihms901528f5.jpg)

Association of the ABCG2RE1 SNP rs12500008, which is in complete linkage disequilibrium with rs12508471 and part of the ABCG2RE1*2 haplotype, with ABCG2 mRNA expression in HapMap LCLs of (A) 80 Han Chinese and (B) 82 Kenyans from the Stranger study[35]. Analysis was done using the GeneVar[29] program with a linear regression and are displayed as gene expression versus rs12500008 genotype with rho (correlation coefficient), P value (P) and empirical P value (Pemp) indicated.

### Effect of Genetic Variants on Binding of DNA to Nuclear Proteins

The ABCG2RE1 rs12508471 and ABCG2RE14 rs9999111 reference and variant DNA were tested for their ability to alter DNA binding to protein in an EMSA. ABCG2RE1 and ABCG2RE14 reference probes showed binding to HepG2 nuclear proteins, with specific DNA/protein interactions being susceptible to competition by unlabeled oligonucleotides ([Figure 6](#F6)). Both rs12508471 and rs9999111 probes exhibited increased HepG2 nuclear protein binding compared to their reference sequence ([Figure 6](#F6)). Proteins binding to the variant probes were less susceptible to competition by unlabeled reference oligonucleotides than they were to the unlabeled variant probe ([Figure 6](#F6)).

### Figure 6. Effect of rs12508471 and rs9999111 on DNA-protein binding.

![Figure 6](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6e1a/5678922/cf29f97745d7/nihms901528f6.jpg)

Representative electrophoretic mobility shift assay using HepG2 nuclear extracts incubated with IRDye 700 labeled probes for (A) ABCG2RE1 reference or rs12508471 and (B) ABCG2RE14 reference or rs9999111 sequences. Competition assays were performed with 40-fold excess of unlabeled reference or variant oligonucleotides; specific DNA/protein bands are indicated by arrows. Reference and variant DNA sequences surrounding each SNP (±6 bp) are shown below their respective gel.

## Discussion

Although non-coding SNPs in the *ABCG2* gene locus have been correlated with drug response[[37](#R37)] and disease progression[[38](#R38)], relatively few have been correlated with gene expression[[10](#R10),[37](#R37)], and none have confirmed mechanisms of action. Recent studies on other ADME genes, including transporters, have identified SNPs in *cis*-regulatory regions that are responsible for altering gene expression and contributing to adverse drug effects[[11](#R11),[23](#R23),[39](#R39),[40](#R40)]. In order to identify non-coding SNPs that correlate with ABCG2 expression, this study investigated the *in vitro* effect of fifty-three SNPs and three haplotypes on the activity of six previously identified *ABCG2* locus enhancer elements[[15](#R15)]. The SNP from each region with the largest *in vitro* effect was followed up *in vivo*; in the case of ABCG2RE1, two SNPs and the ABCG2RE1*2 haplotype were followed-up *in vivo*.

ABCG2RE14 SNP rs9999111, which is located in intron 1 of *ABCG2*, significantly decreased the activity of the ABCG2RE14 enhancer sequence *in vivo* to levels that were only 30% of reference. However, rs9999111 was not associated with liver or kidney expression of ABCG2 in the present study or with liver or intestinal expression of ABCG2 in a previous study[[10](#R10)]. The discordance between these findings could be the result of the combination of the low activity ABCG2RE14 has as an *in vivo* liver enhancer and a change in binding of a tissue specific transcription factor or nuclear receptor that is not critical for constitutive expression of ABCG2. Despite a small sample size (n = 85), the rs9999111 SNP associated with decreased ABCG2 expression in human umbilical cord LCLs and T cells. The beta coefficient of rs9999111 in the umbilical cord (−0.24) and the relatively low frequency of rs9999111 (MAF = 0 – 7.3%), indicates this SNP contributes modestly to variation in ABCG2 expression. Using the transcription factor database (TRANSFAC) Match program, rs9999111 is predicted to reduce the binding of immune-related transcription factors (nuclear factor κ-B, Gfi and FOXD3) and ADME-related transcription factors (vitamin D receptor and hepatic nuclear factor-3β) and increase binding of aryl hydrocarbon receptor (data not shown). EMSA analysis of rs9999111 with HepG2 nuclear protein extract confirmed increased DNA/protein interaction compared to the reference sequence. While we can speculate that the EMSA reflects increased binding to a transcriptional repressor, consistent with the reduced *in vivo* enhancer activity, additional studies are needed to clearly define this interaction. BCRP is a part of the placental barrier important for protection of the fetus, and reduction of BCRP function has the potential to increase fetal exposure to toxic compounds like bile acids, topotecan and PhIP[[41](#R41)–[43](#R43)]. The role of rs9999111 in placental regulation of ABCG2 should be further investigated as it could affect fetal exposure to BCRP substrates. Follow-up studies in a cohort with more tissues could help clarify the association of this SNP with ABCG2 expression in the placenta.

ABCG2RE1*2 is a common haplotype with a frequency of 8.4% in Caucasians, 21% in African Americans and 38% in Asians; it is made up of rs12508471, rs72873421 and rs12500008. The rs12508471 and rs72873421 SNPs affected ABCG2RE1 enhancer activity *in vitro*, but only the rs12508471 eliminated ABCG2RE1 enhancer activity both *in vitro* and *in vivo*. The rs72873421 showed increased activity *in vitro*, which could limit the decrease in activity by rs12508471 when these SNPs occur together in the ABCG2RE1*2 haplotype. However, the ABCG2RE1*2 construct still exhibited a significant decrease in enhancer activity in HEK293T cells and an even greater loss of hepatic enhancer activity *in vivo*. Dexamethasone, a ligand for the GR, is capable of decreasing ABCG2 expression via GR, progesterone receptor and pregnane X receptor (PXR) dependent mechanisms[[44](#R44)]. Using the transcription factor database (TRANSFAC) Match program, rs12508471 gains predicted binding sites for nuclear response element binding, including PXR, peroxisome proliferator-activated receptor, GR, p300 and vitamin D receptor (data not shown). EMSA analysis of the rs12508471 sequence confirmed increased DNA/protein binding compared to the reference sequence. However, oligonucleotide with a GR consensus sequence was unable to compete with the rs12508471 probe (data not shown), and additional studies will be needed to identify the specific transcription factor(s) whose binding is altered by this SNP. ABCG2RE1*2 was not associated with ABCG2 mRNA levels in liver and kidney tissue sets. However, these were composed primarily of Caucasians, a population that has a low frequency of ABCG2RE1*2. Utilizing publically available African and Asian LCL datasets with higher ABCG2RE1*2 frequency, we found that ABCG2RE1*2 was associated with decreased expression of ABCG2 and PPM1K in several tissues, indicating that this haplotype may modestly influence the variability of ABCG2 and PPM1K expression. ABCG2RE1 is situated just upstream of the *PKD2* promoter, but it did not correlate with PKD2 expression in any of the tissue sets (data not shown). Enhancers can work as locus control regions to regulate the expression of neighboring genes and even modulate the tissue-specific expression of multiple genes[[45](#R45)]. Therefore, ABCG2RE1 regulates both ABCG2 and PPM1K expression, and the ABCG2RE1*2 haplotype contributes to altered expression of these genes with more effect in populations with higher frequency of the ABCG2RE1*2 haplotype.

Of the eight variants tested *in vivo*, five significantly altered *in vivo* liver enhancer activity and two of these (rs9999111 and ABCG2RE1*2) were correlated with decreased ABCG2 gene expression in human tissues. Only one of four SNPs expected to increase enhancer activity was confirmed *in vivo*, but all four of the variants that decreased enhancer activity *in vitro* also did so *in vivo*, suggesting that SNPs decreasing enhancer activity *in vitro* might have a more consistent effect *in vivo*. Although there was some correlation between *ABCG2* locus enhancer SNPs and gene expression, further association studies in more diverse cohorts and in additional tissues are needed to validate these findings. Additionally, follow-up is warranted on SNPs that correlate with ABCG2 expression in cohorts of patients receiving treatment with BCRP substrates.

In summary, liver enhancers identified in the *ABCG2* gene locus have many genetic polymorphisms that alter their activity *in vitro*. Several of these SNPs, including rs9999111, rs12508471, rs72873421, rs2725263 and rs149713212 alter enhancer activity *in vivo*. The rs9999111 SNP and ABCG2RE1*2 haplotype were correlated with ABCG2 expression in a tissue-specific manner, with both the rs9999111 and rs12508471 SNPs exhibiting increased binding to nuclear proteins. Taken together, these SNPs could account for some of the reported variability in ABCG2 expression in various tissues and may influence the correlation between ABCG2 and disease risk for cancers or gout. These novel regulatory SNPs may also influence the pharmacokinetics and pharmacodynamics of BCRP substrates. An estimate of the magnitude of this contribution to variability in ABCG2 expression and function warrants further study.

## Supplementary Material

## Acknowledgments

**Source of Funding:**

These studies were supported by the National Institute of Health (NA and DLK). DLK and NA are supported by the National Institute of General Medical Sciences [U01GM0061390]. NA is also supported in part by the National Human Genome Research Institute [R01HG006768 and R01HG005058], National Institute of Child Health & Human Development [R01HD059862], National Institute of Neurological Disorders & Stroke [R01NS079231], National Institute of Diabetes, Digestive & Kidney Diseases [R01DK090382]. RJE, MJK and AC were supported in part by the National Institute of General Medical Sciences Predoctoral Training Grant [T32 GM07175] and all authors were part of the Pharmacogenetics of Membrane Transporters project in the Pharmacogenetics Research Network. RJE was supported by an American Foundation for Pharmaceutical Education Predoctoral fellowship, MJK was supported in part by the University of California San Francisco Quantitative Biosciences Consortium Fellowship for Interdisciplinary Research and the Amgen Research Excellence in Bioengineering and Therapeutic Sciences Fellowship, and RPS was partially supported by a Canadian Institute of Health Research Fellowship in Hepatology.

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Robey RW, To KKK, Polgar O, et al. ABCG2: a perspective. Advanced Drug Delivery Reviews. 2009;61(1):3–13. doi: 10.1016/j.addr.2008.11.003.  [DOI](https://doi.org/10.1016/j.addr.2008.11.003) | [PMC free article](/articles/PMC3105088/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19135109/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Advanced%20Drug%20Delivery%20Reviews&title=ABCG2:%20a%20perspective&author=RW%20Robey&author=KKK%20To&author=O%20Polgar&volume=61&issue=1&publication_year=2009&pages=3-13&pmid=19135109&doi=10.1016/j.addr.2008.11.003&)

2. Jonker JW, Buitelaar M, Wagenaar E, et al. The breast cancer resistance protein protects against a major chlorophyll-derived dietary phototoxin and protoporphyria. Proc. Natl. Acad. Sci. U.S.A. 2002;99(24):15649–15654. doi: 10.1073/pnas.202607599.  [DOI](https://doi.org/10.1073/pnas.202607599) | [PMC free article](/articles/PMC137771/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12429862/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.S.A&title=The%20breast%20cancer%20resistance%20protein%20protects%20against%20a%20major%20chlorophyll-derived%20dietary%20phototoxin%20and%20protoporphyria&author=JW%20Jonker&author=M%20Buitelaar&author=E%20Wagenaar&volume=99&issue=24&publication_year=2002&pages=15649-15654&pmid=12429862&doi=10.1073/pnas.202607599&)

3. Reginato AM, Mount DB, Yang I, Choi HK. The genetics of hyperuricaemia and gout. Nat Rev Rheumatol. 2012;8(10):610–621. doi: 10.1038/nrrheum.2012.144.  [DOI](https://doi.org/10.1038/nrrheum.2012.144) | [PMC free article](/articles/PMC3645862/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22945592/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Rheumatol&title=The%20genetics%20of%20hyperuricaemia%20and%20gout&author=AM%20Reginato&author=DB%20Mount&author=I%20Yang&author=HK%20Choi&volume=8&issue=10&publication_year=2012&pages=610-621&pmid=22945592&doi=10.1038/nrrheum.2012.144&)

4. Cusatis G, Gregorc V, Li J, et al. Pharmacogenetics of ABCG2 and adverse reactions to gefitinib. JNCI Journal of the National Cancer Institute. 2006;98(23):1739–1742. doi: 10.1093/jnci/djj469.  [DOI](https://doi.org/10.1093/jnci/djj469) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17148776/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JNCI%20Journal%20of%20the%20National%20Cancer%20Institute&title=Pharmacogenetics%20of%20ABCG2%20and%20adverse%20reactions%20to%20gefitinib&author=G%20Cusatis&author=V%20Gregorc&author=J%20Li&volume=98&issue=23&publication_year=2006&pages=1739-1742&pmid=17148776&doi=10.1093/jnci/djj469&)

5. Yoh K, Ishii G, Yokose T, et al. Breast cancer resistance protein impacts clinical outcome in platinum-based chemotherapy for advanced non-small cell lung cancer. Clin. Cancer Res. 2004;10(5):1691–1697. doi: 10.1158/1078-0432.ccr-0937-3.  [DOI](https://doi.org/10.1158/1078-0432.ccr-0937-3) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15014021/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Cancer%20Res&title=Breast%20cancer%20resistance%20protein%20impacts%20clinical%20outcome%20in%20platinum-based%20chemotherapy%20for%20advanced%20non-small%20cell%20lung%20cancer&author=K%20Yoh&author=G%20Ishii&author=T%20Yokose&volume=10&issue=5&publication_year=2004&pages=1691-1697&pmid=15014021&doi=10.1158/1078-0432.ccr-0937-3&)

6. Noguchi K, Katayama K, Mitsuhashi J, Sugimoto Y. Functions of the breast cancer resistance protein (BCRP/ABCG2) in chemotherapy. Advanced Drug Delivery Reviews. 2009;61(1):26–33. doi: 10.1016/j.addr.2008.07.003.  [DOI](https://doi.org/10.1016/j.addr.2008.07.003) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19111841/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Advanced%20Drug%20Delivery%20Reviews&title=Functions%20of%20the%20breast%20cancer%20resistance%20protein%20(BCRP/ABCG2)%20in%20chemotherapy&author=K%20Noguchi&author=K%20Katayama&author=J%20Mitsuhashi&author=Y%20Sugimoto&volume=61&issue=1&publication_year=2009&pages=26-33&pmid=19111841&doi=10.1016/j.addr.2008.07.003&)

7. Tsunoda S, Okumura T, Ito T, et al. ABCG2 expression is an independent unfavorable prognostic factor in esophageal squamous cell carcinoma. Oncology. 2006;71(3–4):251–258. doi: 10.1159/000106787.  [DOI](https://doi.org/10.1159/000106787) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17671398/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncology&title=ABCG2%20expression%20is%20an%20independent%20unfavorable%20prognostic%20factor%20in%20esophageal%20squamous%20cell%20carcinoma&author=S%20Tsunoda&author=T%20Okumura&author=T%20Ito&volume=71&issue=3%E2%80%934&publication_year=2006&pages=251-258&pmid=17671398&doi=10.1159/000106787&)

8. Usuda J, Tsunoda Y, Ichinose S, et al. Breast cancer resistant protein (BCRP) is a molecular determinant of the outcome of photodynamic therapy (PDT) for centrally located early lung cancer. Lung Cancer. 2010;67(2):198–204. doi: 10.1016/j.lungcan.2009.04.002.  [DOI](https://doi.org/10.1016/j.lungcan.2009.04.002) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19477032/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lung%20Cancer&title=Breast%20cancer%20resistant%20protein%20(BCRP)%20is%20a%20molecular%20determinant%20of%20the%20outcome%20of%20photodynamic%20therapy%20(PDT)%20for%20centrally%20located%20early%20lung%20cancer&author=J%20Usuda&author=Y%20Tsunoda&author=S%20Ichinose&volume=67&issue=2&publication_year=2010&pages=198-204&pmid=19477032&doi=10.1016/j.lungcan.2009.04.002&)

9. Mao Q, Unadkat JD. Role of the breast cancer resistance protein (BCRP/ABCG2) in drug transport--an update. AAPS J. 2015;17(1):65–82. doi: 10.1208/s12248-014-9668-6.  [DOI](https://doi.org/10.1208/s12248-014-9668-6) | [PMC free article](/articles/PMC4287283/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25236865/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=AAPS%20J&title=Role%20of%20the%20breast%20cancer%20resistance%20protein%20(BCRP/ABCG2)%20in%20drug%20transport--an%20update&author=Q%20Mao&author=JD%20Unadkat&volume=17&issue=1&publication_year=2015&pages=65-82&pmid=25236865&doi=10.1208/s12248-014-9668-6&)

10. Poonkuzhali B, Lamba J, Strom S, et al. Association of breast cancer resistance protein/ABCG2 phenotypes and novel promoter and intron 1 single nucleotide polymorphisms. Drug Metabolism and Disposition. 2008;36(4):780–795. doi: 10.1124/dmd.107.018366.  [DOI](https://doi.org/10.1124/dmd.107.018366) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18180275/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metabolism%20and%20Disposition&title=Association%20of%20breast%20cancer%20resistance%20protein/ABCG2%20phenotypes%20and%20novel%20promoter%20and%20intron%201%20single%20nucleotide%20polymorphisms&author=B%20Poonkuzhali&author=J%20Lamba&author=S%20Strom&volume=36&issue=4&publication_year=2008&pages=780-795&pmid=18180275&doi=10.1124/dmd.107.018366&)

11. Georgitsi M, Zukic B, Pavlovic S, Patrinos GP. Transcriptional regulation and pharmacogenomics. Pharmacogenomics. 2011;12(5):655–673. doi: 10.2217/pgs.10.215.  [DOI](https://doi.org/10.2217/pgs.10.215) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21619428/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Transcriptional%20regulation%20and%20pharmacogenomics&author=M%20Georgitsi&author=B%20Zukic&author=S%20Pavlovic&author=GP%20Patrinos&volume=12&issue=5&publication_year=2011&pages=655-673&pmid=21619428&doi=10.2217/pgs.10.215&)

12. Smith RP, Lam ET, Markova S, Yee SW, Ahituv N. Pharmacogene regulatory elements: from discovery to applications. Genome Med. 2012;4(5):45. doi: 10.1186/gm344.  [DOI](https://doi.org/10.1186/gm344) | [PMC free article](/articles/PMC3506911/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22630332/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Med&title=Pharmacogene%20regulatory%20elements:%20from%20discovery%20to%20applications&author=RP%20Smith&author=ET%20Lam&author=S%20Markova&author=SW%20Yee&author=N%20Ahituv&volume=4&issue=5&publication_year=2012&pages=45&pmid=22630332&doi=10.1186/gm344&)

13. Ge B, Pokholok DK, Kwan T, et al. Global patterns of cis variation in human cells revealed by high-density allelic expression analysis. Nature Genetics. 2009;41(11):1216–1222. doi: 10.1038/ng.473.  [DOI](https://doi.org/10.1038/ng.473) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19838192/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature%20Genetics&title=Global%20patterns%20of%20cis%20variation%20in%20human%20cells%20revealed%20by%20high-density%20allelic%20expression%20analysis&author=B%20Ge&author=DK%20Pokholok&author=T%20Kwan&volume=41&issue=11&publication_year=2009&pages=1216-1222&pmid=19838192&doi=10.1038/ng.473&)

14. Pastinen T. Genome-wide allele-specific analysis: insights into regulatory variation. Nature Reviews Genetics. 2010;11(8):533–538. doi: 10.1038/nrg2815.  [DOI](https://doi.org/10.1038/nrg2815) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20567245/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature%20Reviews%20Genetics&title=Genome-wide%20allele-specific%20analysis:%20insights%20into%20regulatory%20variation&author=T%20Pastinen&volume=11&issue=8&publication_year=2010&pages=533-538&pmid=20567245&doi=10.1038/nrg2815&)

15. Eclov RJ, Kim MJ, Smith RP, Liang X, Ahituv N, Kroetz DL. In Vivo Hepatic Enhancer Elements in the Human ABCG2 Locus. Drug Metabolism and Disposition. 2017;45(2):208–215. doi: 10.1124/dmd.116.072033.  [DOI](https://doi.org/10.1124/dmd.116.072033) | [PMC free article](/articles/PMC5267518/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27856528/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metabolism%20and%20Disposition&title=In%20Vivo%20Hepatic%20Enhancer%20Elements%20in%20the%20Human%20ABCG2%20Locus&author=RJ%20Eclov&author=MJ%20Kim&author=RP%20Smith&author=X%20Liang&author=N%20Ahituv&volume=45&issue=2&publication_year=2017&pages=208-215&pmid=27856528&doi=10.1124/dmd.116.072033&)

16. 1000 Genomes Project Consortium. Abecasis GR, Auton A, et al. An integrated map of genetic variation from 1,092 human genomes. Nature. 2012;491(7422):56–65. doi: 10.1038/nature11632.  [DOI](https://doi.org/10.1038/nature11632) | [PMC free article](/articles/PMC3498066/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23128226/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=An%20integrated%20map%20of%20genetic%20variation%20from%201,092%20human%20genomes&author=GR%20Abecasis&author=A%20Auton&volume=491&issue=7422&publication_year=2012&pages=56-65&pmid=23128226&doi=10.1038/nature11632&)

17. International HapMap Consortium. Frazer KA, Ballinger DG, et al. A second generation human haplotype map of over 3.1 million SNPs. Nature. 2007;449(7164):851–861. doi: 10.1038/nature06258.  [DOI](https://doi.org/10.1038/nature06258) | [PMC free article](/articles/PMC2689609/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17943122/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=A%20second%20generation%20human%20haplotype%20map%20of%20over%203.1%20million%20SNPs&author=KA%20Frazer&author=DG%20Ballinger&volume=449&issue=7164&publication_year=2007&pages=851-861&pmid=17943122&doi=10.1038/nature06258&)

18. Johnson AD, Handsaker RE, Pulit SL, Nizzari MM, O'Donnell CJ, de Bakker PIW. SNAP: a web-based tool for identification and annotation of proxy SNPs using HapMap. Bioinformatics. 2008;24(24):2938–2939. doi: 10.1093/bioinformatics/btn564.  [DOI](https://doi.org/10.1093/bioinformatics/btn564) | [PMC free article](/articles/PMC2720775/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18974171/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=SNAP:%20a%20web-based%20tool%20for%20identification%20and%20annotation%20of%20proxy%20SNPs%20using%20HapMap&author=AD%20Johnson&author=RE%20Handsaker&author=SL%20Pulit&author=MM%20Nizzari&author=CJ%20O'Donnell&volume=24&issue=24&publication_year=2008&pages=2938-2939&pmid=18974171&doi=10.1093/bioinformatics/btn564&)

19. Barrett JC, Fry B, Maller J, Daly MJ. Haploview: analysis and visualization of LD and haplotype maps. Bioinformatics. 2005;21(2):263–265. doi: 10.1093/bioinformatics/bth457.  [DOI](https://doi.org/10.1093/bioinformatics/bth457) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15297300/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=Haploview:%20analysis%20and%20visualization%20of%20LD%20and%20haplotype%20maps&author=JC%20Barrett&author=B%20Fry&author=J%20Maller&author=MJ%20Daly&volume=21&issue=2&publication_year=2005&pages=263-265&pmid=15297300&doi=10.1093/bioinformatics/bth457&)

20. Liu H, Naismith JH. An efficient one-step site-directed deletion, insertion, single and multiple-site plasmid mutagenesis protocol. BMC Biotechnol. 2008;8:91. doi: 10.1186/1472-6750-8-91.  [DOI](https://doi.org/10.1186/1472-6750-8-91) | [PMC free article](/articles/PMC2629768/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19055817/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Biotechnol&title=An%20efficient%20one-step%20site-directed%20deletion,%20insertion,%20single%20and%20multiple-site%20plasmid%20mutagenesis%20protocol&author=H%20Liu&author=JH%20Naismith&volume=8&publication_year=2008&pages=91&pmid=19055817&doi=10.1186/1472-6750-8-91&)

21. Simonet WS, Bucay N, Lauer SJ, Taylor JM. A far-downstream hepatocyte-specific control region directs expression of the linked human apolipoprotein E and C-I genes in transgenic mice. J. Biol. Chem. 1993;268(11):8221–8229.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/7681840/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem&title=A%20far-downstream%20hepatocyte-specific%20control%20region%20directs%20expression%20of%20the%20linked%20human%20apolipoprotein%20E%20and%20C-I%20genes%20in%20transgenic%20mice&author=WS%20Simonet&author=N%20Bucay&author=SJ%20Lauer&author=JM%20Taylor&volume=268&issue=11&publication_year=1993&pages=8221-8229&pmid=7681840&)

22. Kim MJ, Ahituv N. The hydrodynamic tail vein assay as a tool for the study of liver promoters and enhancers. Methods Mol. Biol. 2013;1015:279–289. doi: 10.1007/978-1-62703-435-7_18.  [DOI](https://doi.org/10.1007/978-1-62703-435-7_18) | [PMC free article](/articles/PMC4096022/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23824863/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Methods%20Mol.%20Biol&title=The%20hydrodynamic%20tail%20vein%20assay%20as%20a%20tool%20for%20the%20study%20of%20liver%20promoters%20and%20enhancers&author=MJ%20Kim&author=N%20Ahituv&volume=1015&publication_year=2013&pages=279-289&pmid=23824863&doi=10.1007/978-1-62703-435-7_18&)

23. Kim MJ, Skewes-Cox P, Fukushima H, et al. Functional characterization of liver enhancers that regulate drug-associated transporters. Clinical Pharmacology & Therapeutics. 2011;89(4):571–578. doi: 10.1038/clpt.2010.353.  [DOI](https://doi.org/10.1038/clpt.2010.353) | [PMC free article](/articles/PMC3227682/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21368754/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clinical%20Pharmacology%20&%20Therapeutics&title=Functional%20characterization%20of%20liver%20enhancers%20that%20regulate%20drug-associated%20transporters&author=MJ%20Kim&author=P%20Skewes-Cox&author=H%20Fukushima&volume=89&issue=4&publication_year=2011&pages=571-578&pmid=21368754&doi=10.1038/clpt.2010.353&)

24. Kroetz DL, Yee SW, Giacomini KM. The pharmacogenomics of membrane transporters project: research at the interface of genomics and transporter pharmacology. Clinical Pharmacology & Therapeutics. 2010;87(1):109–116. doi: 10.1038/clpt.2009.226.  [DOI](https://doi.org/10.1038/clpt.2009.226) | [PMC free article](/articles/PMC2923224/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19940846/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clinical%20Pharmacology%20&%20Therapeutics&title=The%20pharmacogenomics%20of%20membrane%20transporters%20project:%20research%20at%20the%20interface%20of%20genomics%20and%20transporter%20pharmacology&author=DL%20Kroetz&author=SW%20Yee&author=KM%20Giacomini&volume=87&issue=1&publication_year=2010&pages=109-116&pmid=19940846&doi=10.1038/clpt.2009.226&)

25. Bolstad BM, Irizarry RA, Astrand M, Speed TP. A comparison of normalization methods for high density oligonucleotide array data based on variance and bias. Bioinformatics. 2003;19(2):185–193. doi: 10.1093/bioinformatics/19.2.185.  [DOI](https://doi.org/10.1093/bioinformatics/19.2.185) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12538238/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=A%20comparison%20of%20normalization%20methods%20for%20high%20density%20oligonucleotide%20array%20data%20based%20on%20variance%20and%20bias&author=BM%20Bolstad&author=RA%20Irizarry&author=M%20Astrand&author=TP%20Speed&volume=19&issue=2&publication_year=2003&pages=185-193&pmid=12538238&doi=10.1093/bioinformatics/19.2.185&)

26. Boes T, Neuhäuser M. Normalization for Affymetrix GeneChips. Methods Inf Med. 2005;44(3):414–417.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/16113766/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Methods%20Inf%20Med&title=Normalization%20for%20Affymetrix%20GeneChips&author=T%20Boes&author=M%20Neuh%C3%A4user&volume=44&issue=3&publication_year=2005&pages=414-417&pmid=16113766&)

27. Schadt EE, Molony C, Chudin E, et al. Mapping the genetic architecture of gene expression in human liver. Plos Biol. 2008;6(5):e107. doi: 10.1371/journal.pbio.0060107.  [DOI](https://doi.org/10.1371/journal.pbio.0060107) | [PMC free article](/articles/PMC2365981/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18462017/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Plos%20Biol&title=Mapping%20the%20genetic%20architecture%20of%20gene%20expression%20in%20human%20liver&author=EE%20Schadt&author=C%20Molony&author=E%20Chudin&volume=6&issue=5&publication_year=2008&pages=e107&pmid=18462017&doi=10.1371/journal.pbio.0060107&)

28. Cancer Genome Atlas Network. Comprehensive molecular portraits of human breast tumours. Nature. 2012;490(7418):61–70. doi: 10.1038/nature11412.  [DOI](https://doi.org/10.1038/nature11412) | [PMC free article](/articles/PMC3465532/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23000897/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Cancer%20Genome%20Atlas%20Network.%20Comprehensive%20molecular%20portraits%20of%20human%20breast%20tumours&volume=490&issue=7418&publication_year=2012&pages=61-70&pmid=23000897&doi=10.1038/nature11412&)

29. Yang T-P, Beazley C, Montgomery SB, et al. Genevar: a database and Java application for the analysis and visualization of SNP-gene associations in eQTL studies. Bioinformatics. 2010;26(19):2474–2476. doi: 10.1093/bioinformatics/btq452.  [DOI](https://doi.org/10.1093/bioinformatics/btq452) | [PMC free article](/articles/PMC2944204/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20702402/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=Genevar:%20a%20database%20and%20Java%20application%20for%20the%20analysis%20and%20visualization%20of%20SNP-gene%20associations%20in%20eQTL%20studies&author=T-P%20Yang&author=C%20Beazley&author=SB%20Montgomery&volume=26&issue=19&publication_year=2010&pages=2474-2476&pmid=20702402&doi=10.1093/bioinformatics/btq452&)

30. Nica AC, Parts L, Glass D, et al. The architecture of gene regulatory variation across multiple human tissues: the MuTHER study. PLoS Genet. 2011;7(2):e1002003. doi: 10.1371/journal.pgen.1002003.  [DOI](https://doi.org/10.1371/journal.pgen.1002003) | [PMC free article](/articles/PMC3033383/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21304890/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet&title=The%20architecture%20of%20gene%20regulatory%20variation%20across%20multiple%20human%20tissues:%20the%20MuTHER%20study&author=AC%20Nica&author=L%20Parts&author=D%20Glass&volume=7&issue=2&publication_year=2011&pages=e1002003&pmid=21304890&doi=10.1371/journal.pgen.1002003&)

31. Grundberg E, Small KS, Hedman ÅK, et al. Mapping cis- and trans-regulatory effects across multiple tissues in twins. Nature Genetics. 2012;44(10):1084–1089. doi: 10.1038/ng.2394.  [DOI](https://doi.org/10.1038/ng.2394) | [PMC free article](/articles/PMC3784328/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22941192/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature%20Genetics&title=Mapping%20cis-%20and%20trans-regulatory%20effects%20across%20multiple%20tissues%20in%20twins&author=E%20Grundberg&author=KS%20Small&author=%C3%85K%20Hedman&volume=44&issue=10&publication_year=2012&pages=1084-1089&pmid=22941192&doi=10.1038/ng.2394&)

32. Stranger BE, Nica AC, Forrest MS, et al. Population genomics of human gene expression. Nature Genetics. 2007;39(10):1217–1224. doi: 10.1038/ng2142.  [DOI](https://doi.org/10.1038/ng2142) | [PMC free article](/articles/PMC2683249/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17873874/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature%20Genetics&title=Population%20genomics%20of%20human%20gene%20expression&author=BE%20Stranger&author=AC%20Nica&author=MS%20Forrest&volume=39&issue=10&publication_year=2007&pages=1217-1224&pmid=17873874&doi=10.1038/ng2142&)

33. Stranger BE, Forrest MS, Clark AG, et al. Genome-wide associations of gene expression variation in humans. PLoS Genet. 2005;1(6):e78. doi: 10.1371/journal.pgen.0010078.  [DOI](https://doi.org/10.1371/journal.pgen.0010078) | [PMC free article](/articles/PMC1315281/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16362079/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet&title=Genome-wide%20associations%20of%20gene%20expression%20variation%20in%20humans&author=BE%20Stranger&author=MS%20Forrest&author=AG%20Clark&volume=1&issue=6&publication_year=2005&pages=e78&pmid=16362079&doi=10.1371/journal.pgen.0010078&)

34. Stranger BE, Forrest MS, Dunning M, et al. Relative impact of nucleotide and copy number variation on gene expression phenotypes. Science. 2007;315(5813):848–853. doi: 10.1126/science.1136678.  [DOI](https://doi.org/10.1126/science.1136678) | [PMC free article](/articles/PMC2665772/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17289997/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=Relative%20impact%20of%20nucleotide%20and%20copy%20number%20variation%20on%20gene%20expression%20phenotypes&author=BE%20Stranger&author=MS%20Forrest&author=M%20Dunning&volume=315&issue=5813&publication_year=2007&pages=848-853&pmid=17289997&doi=10.1126/science.1136678&)

35. Stranger BE, Montgomery SB, Dimas AS, et al. Patterns of cis regulatory variation in diverse human populations. PLoS Genet. 2012;8(4):e1002639. doi: 10.1371/journal.pgen.1002639.  [DOI](https://doi.org/10.1371/journal.pgen.1002639) | [PMC free article](/articles/PMC3330104/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22532805/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet&title=Patterns%20of%20cis%20regulatory%20variation%20in%20diverse%20human%20populations&author=BE%20Stranger&author=SB%20Montgomery&author=AS%20Dimas&volume=8&issue=4&publication_year=2012&pages=e1002639&pmid=22532805&doi=10.1371/journal.pgen.1002639&)

36. Dimas AS, Deutsch S, Stranger BE, et al. Common regulatory variation impacts gene expression in a cell type-dependent manner. Science. 2009;325(5945):1246–1250. doi: 10.1126/science.1174148.  [DOI](https://doi.org/10.1126/science.1174148) | [PMC free article](/articles/PMC2867218/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19644074/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=Common%20regulatory%20variation%20impacts%20gene%20expression%20in%20a%20cell%20type-dependent%20manner&author=AS%20Dimas&author=S%20Deutsch&author=BE%20Stranger&volume=325&issue=5945&publication_year=2009&pages=1246-1250&pmid=19644074&doi=10.1126/science.1174148&)

37. Rudin CM, Liu W, Desai A, et al. Pharmacogenomic and pharmacokinetic determinants of erlotinib toxicity. Journal of Clinical Oncology. 2008;26(7):1119–1127. doi: 10.1200/JCO.2007.13.1128.  [DOI](https://doi.org/10.1200/JCO.2007.13.1128) | [PMC free article](/articles/PMC5846189/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18309947/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Clinical%20Oncology&title=Pharmacogenomic%20and%20pharmacokinetic%20determinants%20of%20erlotinib%20toxicity&author=CM%20Rudin&author=W%20Liu&author=A%20Desai&volume=26&issue=7&publication_year=2008&pages=1119-1127&pmid=18309947&doi=10.1200/JCO.2007.13.1128&)

38. Campa D, Butterbach K, Slager SL, et al. A comprehensive study of polymorphisms in the ABCB1, ABCC2, ABCG2, NR1I2 genes and lymphoma risk. Int. J. Cancer. 2012;131(4):803–812. doi: 10.1002/ijc.26436.  [DOI](https://doi.org/10.1002/ijc.26436) | [PMC free article](/articles/PMC3432449/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21918980/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Cancer&title=A%20comprehensive%20study%20of%20polymorphisms%20in%20the%20ABCB1,%20ABCC2,%20ABCG2,%20NR1I2%20genes%20and%20lymphoma%20risk&author=D%20Campa&author=K%20Butterbach&author=SL%20Slager&volume=131&issue=4&publication_year=2012&pages=803-812&pmid=21918980&doi=10.1002/ijc.26436&)

39. McLeod HL, Sargent DJ, Marsh S, et al. Pharmacogenetic predictors of adverse events and response to chemotherapy in metastatic colorectal cancer: results from North American Gastrointestinal Intergroup Trial N9741. Journal of Clinical Oncology. 2010;28(20):3227–3233. doi: 10.1200/JCO.2009.21.7943.  [DOI](https://doi.org/10.1200/JCO.2009.21.7943) | [PMC free article](/articles/PMC2903324/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20530282/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Clinical%20Oncology&title=Pharmacogenetic%20predictors%20of%20adverse%20events%20and%20response%20to%20chemotherapy%20in%20metastatic%20colorectal%20cancer:%20results%20from%20North%20American%20Gastrointestinal%20Intergroup%20Trial%20N9741&author=HL%20McLeod&author=DJ%20Sargent&author=S%20Marsh&volume=28&issue=20&publication_year=2010&pages=3227-3233&pmid=20530282&doi=10.1200/JCO.2009.21.7943&)

40. Matsson P, Yee SW, Markova S, et al. Discovery of regulatory elements in human ATP-binding cassette transporters through expression quantitative trait mapping. The Pharmacogenomics Journal. 2012;12(3):214–226. doi: 10.1038/tpj.2011.8.  [DOI](https://doi.org/10.1038/tpj.2011.8) | [PMC free article](/articles/PMC3325368/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21383772/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=The%20Pharmacogenomics%20Journal&title=Discovery%20of%20regulatory%20elements%20in%20human%20ATP-binding%20cassette%20transporters%20through%20expression%20quantitative%20trait%20mapping&author=P%20Matsson&author=SW%20Yee&author=S%20Markova&volume=12&issue=3&publication_year=2012&pages=214-226&pmid=21383772&doi=10.1038/tpj.2011.8&)

41. Blazquez AG, Briz O, Romero MR, et al. Characterization of the role of ABCG2 as a bile acid transporter in liver and placenta. Molecular Pharmacology. 2012;81(2):273–283. doi: 10.1124/mol.111.075143.  [DOI](https://doi.org/10.1124/mol.111.075143) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22096226/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Molecular%20Pharmacology&title=Characterization%20of%20the%20role%20of%20ABCG2%20as%20a%20bile%20acid%20transporter%20in%20liver%20and%20placenta&author=AG%20Blazquez&author=O%20Briz&author=MR%20Romero&volume=81&issue=2&publication_year=2012&pages=273-283&pmid=22096226&doi=10.1124/mol.111.075143&)

42. Jonker JW, Merino G, Musters S, et al. The breast cancer resistance protein BCRP (ABCG2) concentrates drugs and carcinogenic xenotoxins into milk. Nature Medicine. 2005;11(2):127–129. doi: 10.1038/nm1186.  [DOI](https://doi.org/10.1038/nm1186) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15685169/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature%20Medicine&title=The%20breast%20cancer%20resistance%20protein%20BCRP%20(ABCG2)%20concentrates%20drugs%20and%20carcinogenic%20xenotoxins%20into%20milk&author=JW%20Jonker&author=G%20Merino&author=S%20Musters&volume=11&issue=2&publication_year=2005&pages=127-129&pmid=15685169&doi=10.1038/nm1186&)

43. Jonker JW, Smit JW, Brinkhuis RF, et al. Role of breast cancer resistance protein in the bioavailability and fetal penetration of topotecan. J. Natl. Cancer Inst. 2000;92(20):1651–1656. doi: 10.1093/jnci/92.20.1651.  [DOI](https://doi.org/10.1093/jnci/92.20.1651) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11036110/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Natl.%20Cancer%20Inst&title=Role%20of%20breast%20cancer%20resistance%20protein%20in%20the%20bioavailability%20and%20fetal%20penetration%20of%20topotecan&author=JW%20Jonker&author=JW%20Smit&author=RF%20Brinkhuis&volume=92&issue=20&publication_year=2000&pages=1651-1656&pmid=11036110&doi=10.1093/jnci/92.20.1651&)

44. Honorat M, Mesnier A, Di Pietro A, et al. Dexamethasone down-regulates ABCG2 expression levels in breast cancer cells. Biochemical and Biophysical Research Communications. 2008;375(3):308–314. doi: 10.1016/j.bbrc.2008.07.149.  [DOI](https://doi.org/10.1016/j.bbrc.2008.07.149) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18692485/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochemical%20and%20Biophysical%20Research%20Communications&title=Dexamethasone%20down-regulates%20ABCG2%20expression%20levels%20in%20breast%20cancer%20cells&author=M%20Honorat&author=A%20Mesnier&author=A%20Di%20Pietro&volume=375&issue=3&publication_year=2008&pages=308-314&pmid=18692485&doi=10.1016/j.bbrc.2008.07.149&)

45. Li Q, Peterson KR, Fang X, Stamatoyannopoulos G. Locus control regions. Blood. 2002;100(9):3077–3086. doi: 10.1182/blood-2002-04-1104.  [DOI](https://doi.org/10.1182/blood-2002-04-1104) | [PMC free article](/articles/PMC2811695/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12384402/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Locus%20control%20regions&author=Q%20Li&author=KR%20Peterson&author=X%20Fang&author=G%20Stamatoyannopoulos&volume=100&issue=9&publication_year=2002&pages=3077-3086&pmid=12384402&doi=10.1182/blood-2002-04-1104&)
