# Association and clinical utility of NAT2 in the prediction of isoniazid-induced liver injury in Singaporean patients

## Metadata
**Authors:** Sze Ling Chan, Angeline Poh Gek Chua, Folefac Aminkeng, Cynthia Bin Eng Chee, Shengnan Jin, Marie Loh, Suay Hong Gan, Yee Tang Wang, Liam R Brunham
**Journal:** PLoS ONE
**Date:** 2017 Oct 16
**DOI:** [10.1371/journal.pone.0186200](https://doi.org/10.1371/journal.pone.0186200)
**PMID:** 29036176
**PMCID:** PMC5642896
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5642896/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC5642896/pdf/pone.0186200.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC5642896/pdf/pone.0186200.pdf)

## Abstract

**Background and aims:** 
Isoniazid (INH) is part of the first-line-therapy for tuberculosis (TB) but can cause drug-induced liver injury (DILI). Several candidate single nucleotide polymorphisms (SNPs) have been previously identified but the clinical utility of these SNPs in the prediction of INH-DILI remains uncertain. The aim of this study was to assess the association between selected candidate SNPs and the risk of INH-DILI and to assess the clinical validity of associated variants in a Singaporean population.

**Methods:** 
This was a case-control study where 24 INH-DILI cases and 79 controls were recruited from the TB control unit in a tertiary hospital. Logistic regression was used to test for the association between candidate SNPs and INH-DILI. NAT2 acetylator status was inferred from genotypes and tested for association with INH-DILI. Finally, clinical validity measures were estimated for significant variants.

**Results:** 
Two SNPs in NAT2 (rs1041983 and rs1495741) and NAT2 slow acetylators (SA) were significantly associated with INH-DILI (OR (95% CI) = 13.86 (4.30–44.70), 0.10 (0.03–0.33) and 9.98 (3.32–33.80), respectively). Based on an INH-DILI prevalence of 10%, the sensitivity, specificity, positive and negative predictive values of NAT2 SA were 75%, 78%, 28% and 97%, respectively. The population attributable fraction (PAF) and number needed to test (NNT) for NAT2 SA were estimated to be 0.67 and 4.08, respectively. A model with clinical and NAT2 acetylator status provided significantly better prediction for INH-DILI than a clinical model alone (area under receiver operating characteristic curve = 0.863 vs. 0.766, respectively, p = 0.027).

**Conclusions:** 
We show the association between NAT2 SA and INH-DILI in a Singaporean population and demonstrated its clinical utility in the prediction of INH-DILI.

### Background and aims

Isoniazid (INH) is part of the first-line-therapy for tuberculosis (TB) but can cause drug-induced liver injury (DILI). Several candidate single nucleotide polymorphisms (SNPs) have been previously identified but the clinical utility of these SNPs in the prediction of INH-DILI remains uncertain. The aim of this study was to assess the association between selected candidate SNPs and the risk of INH-DILI and to assess the clinical validity of associated variants in a Singaporean population.

### Methods

This was a case-control study where 24 INH-DILI cases and 79 controls were recruited from the TB control unit in a tertiary hospital. Logistic regression was used to test for the association between candidate SNPs and INH-DILI. NAT2 acetylator status was inferred from genotypes and tested for association with INH-DILI. Finally, clinical validity measures were estimated for significant variants.

### Results

Two SNPs in *NAT2* (rs1041983 and rs1495741) and NAT2 slow acetylators (SA) were significantly associated with INH-DILI (OR (95% CI) = 13.86 (4.30–44.70), 0.10 (0.03–0.33) and 9.98 (3.32–33.80), respectively). Based on an INH-DILI prevalence of 10%, the sensitivity, specificity, positive and negative predictive values of NAT2 SA were 75%, 78%, 28% and 97%, respectively. The population attributable fraction (PAF) and number needed to test (NNT) for NAT2 SA were estimated to be 0.67 and 4.08, respectively. A model with clinical and NAT2 acetylator status provided significantly better prediction for INH-DILI than a clinical model alone (area under receiver operating characteristic curve = 0.863 vs. 0.766, respectively, p = 0.027).

### Conclusions

We show the association between NAT2 SA and INH-DILI in a Singaporean population and demonstrated its clinical utility in the prediction of INH-DILI.

## Introduction

Tuberculosis (TB) is one of the top 10 causes of death world-wide, with an estimated 10.4 million new cases and 1.8 million TB-related deaths worldwide in 2015 [[1](#pone.0186200.ref001)]. Isoniazid (INH) is an effective anti-mycobacterium agent and is part of first-line 4-drug therapy for TB that also includes rifampicin, pyrazinamide and ethambutol [[1](#pone.0186200.ref001)]. However, INH is associated with drug-induced liver injury (DILI) that can range from asymptomatic elevations in liver enzymes in 10–20% of patients, to clinically significant hepatitis and acute liver failure in 0.5–1% of patients [[2](#pone.0186200.ref002)]. Risk factors for INH-DILI include age, female gender, black race, alcoholism, pre-existing liver disease, and concomitant medications such as rifampicin and pyrazinamide [[2](#pone.0186200.ref002)].

The molecular mechanisms of anti-TB DILI are not fully understood, but it is thought accumulation of toxic intermediate metabolites plays an important role [[3](#pone.0186200.ref003)]. Pharmacogenomic studies have been mainly focused on candidate genes involved in INH biotransformation and metabolism, including N-acetyltransferase 2 (*NAT2*), cytochrome P450 2E1 (*CYP2E1*) and glutathione-S-transferase genes (*GSTT1*, *GSTM1* and *GSTP1*). In addition, INH and some of its metabolites are readily oxidized to reactive species, which may contribute to liver damage [[4](#pone.0186200.ref004)]. INH-DILI typically occurs within 2 weeks to 6 months after commencing therapy [[2](#pone.0186200.ref002)]. The delayed nature of INH-DILI also points to a possible immune component [[4](#pone.0186200.ref004)]. As such, genes in the antioxidant and detoxification pathways [[5](#pone.0186200.ref005)–[8](#pone.0186200.ref008)], human-leukocyte antigen system [[9](#pone.0186200.ref009),[10](#pone.0186200.ref010)] and tumor necrosis factor-α [[11](#pone.0186200.ref011)] have also been studied.

Meta-analyses have consistently shown that NAT2 slow acetylators (SA) have a higher risk of INH-DILI, with an odds ratio (OR) of around 3.1 [[12](#pone.0186200.ref012),[13](#pone.0186200.ref013)]. Seven common coding single nucleotide polymorphisms (SNPs) (rs1801279, rs1041983, rs1801280, rs1799929, rs1799930, rs1208 and rs1799931) define 34 haplotypes, but most studies only genotype a subset to infer acetylator status [[14](#pone.0186200.ref014)]. Recently, a tag SNP, rs1495741, located 14.5kb 3’ of *NAT2* has been associated with INH-DILI, although it may not be a good surrogate for acetylator status [[14](#pone.0186200.ref014),[15](#pone.0186200.ref015)].

In *CYP2E1*, two polymorphisms are most commonly studied (*RsaI/PstI* (rs2031920/rs3813867) and *DraI* (rs6413432)). rs2031920 and rs3813867 are in complete linkage disequilibrium in East Asians [[16](#pone.0186200.ref016)]. A recent meta-analysis found that the *RsaI/PstI* polymorphism *c1/c1* (major allele for both SNPs) confers a higher risk of INH-DILI with an OR of 1.32 compared to the c1/c2 or c2/c2 genotypes, but no increase in risk for the *DraI* polymorphism [[16](#pone.0186200.ref016)].

These candidate SNPs have been studied in diverse populations including several of Asian descent (Chinese, Xinjiang Uyghurs, Japanese, Korean, Indian, Taiwanese and Indonesian) [[15](#pone.0186200.ref015),[17](#pone.0186200.ref017)–[39](#pone.0186200.ref039)]. However, no previous study has been conducted in a Singaporean population. The 3 major ethnic groups in Singapore, Chinese, Malays and Indians, originated largely from Southern China, Peninsula Malaysia and Indonesia, and Southern India, respectively [[40](#pone.0186200.ref040)]. Results derived from the Singaporean population may therefore have broad applicability to populations that have not been well represented in previous candidate gene studies. Malays in particular, have not been included in INH-DILI studies apart from one focused on *NAT2* [[35](#pone.0186200.ref035)]. Representation of these populations is crucial since India, China and Indonesia are among the countries with the greatest burden of TB burden worldwide [[1](#pone.0186200.ref001)].

To facilitate the clinical implementation of prospective pharmacogenomic screening for INH-DILI in these South and Southeast Asian countries, it is imperative to determine the genetic and clinical relevance of INH-DILI pharmacogenomic biomarkers in representative populations. Therefore, the objective of this study was to assess the association between selected candidate SNPs and the risk of INH-DILI, and to assess the clinical validity of associated variants in a multiethnic Singaporean population representative of the Southeast Asian region.

## Results

### Clinical and demographic characteristics

A total of 104 patients were recruited ([S1 Fig](#pone.0186200.s001)). One patient failed genotyping quality control (QC) due to sex mismatch, leaving 103 patients (24 cases and 79 controls) for analysis. No samples were removed due to high identity-by-state (IBS). The clinical characteristics of cases and controls are shown in [Table 1](#pone.0186200.t001). Most cases had grade 2 (41.7%) or grade 3 (54.2%) INH-DILI and only 1 patient had grade 4 INH-DILI. The median onset time for INH-DILI was 17.5 days (range 4–111 days), which is consistent with the typical onset time of 2 weeks to 6 months [[2](#pone.0186200.ref002)]. Cases were more likely to be female than controls (p = 1.80 x 10^−3^) and were more likely to be of ‘Other’ ethnicity ([Table 1](#pone.0186200.t001)).

### Table 1. Demographic and clinical characteristics of Singaporean patients diagnosed with TB and treated with INH.

| Clinical and Demographic Characteristics | All Patients (n = 103) | INH-DILI Cases (n = 24) | Controls (n = 79) | P value* |
| --- | --- | --- | --- | --- |
| Age, mean (sd) | 51.1 (14.5) | 51.6 (16.8) | 50.9 (13.9) | 0.728‡ |
| Male gender, n (%) | 68 (66.0) | 9 (37.5) | 59 (74.7) | 1.80 x 10−3† |
| Ethnicity, n (%) |   |   |   | 0.034† |
| Chinese | 69 (67.0) | 12 (50.0) | 57 (72.2) | 0.051¶ |
| Malays | 15 (14.6) | 5 (20.8) | 10 (12.7) | 0.333¶ |
| Indians | 8 (7.8) | 1 (4.2) | 7 (8.9) | 0.677¶ |
| Others | 11 (10.7) | 6 (25.0) | 5 (6.3) | 0.018¶ |
| BMI, mean (sd) | 21.5 (3.7) | 21.5 (3.8) | 21.4 (3.7) | 0.949§ |
| TB type, n(%) |   |   |   | 0.934† |
| Pulmonary | 73 (70.9) | 17 (70.8) | 56 (70.9) |   |
| Extrapulmonary | 19 (18.4) | 4 (16.7) | 15 (19.0) |   |
| Both | 11 (10.7) | 3 (12.5) | 8 (10.1) |   |
| Drug doses (mg), median (range) |   |   |   |   |
| Isoniazid | 300 (200–386₤) | 300 (200–300) | 300 (200–386₤) | 0.648‡ |
| Rifampicin | 600 (257₤–600) | 600 (300–600) | 600 (257₤–600) | 0.149‡ |
| Pyrazinamide | 1250 (643₤–2000) | 1250 (1000–1750) | 1250 (643₤–2000) | 0.590‡ |
| Ethambutol | 1000 (400–1600) | 950 (600–1400) | 1000 (400–1600) | 0.204‡ |
| Received hepatotoxic concomitant medications^, n(%) | 22 (21.3) | 5 (20.8) | 17 (21.5) | 1† |
| NIH Grade, n (%) |   |   |   |   |
| 0 |   | 0 | 79 (100) | - |
| 2 |   | 10 (41.7) | 0 | - |
| 3 |   | 13 (54.2) | 0 | - |
| 4 |   | 1 (4.2) | 0 | - |
| Baseline LFT, median (range) |   |   |   |   |
| AST (U/L) |   | 22 (14–48) | 21 (11–69) | - |
| ALT (U/L) |   | 18 (8–46) | 18 (6–64) | - |
| Onset or follow up LFT, median (range) |   |   |   |   |
| AST (U/L) |   | 212 (104–1401) | 22 (9–42) | - |
| ALT (U/L) |   | 199.5 (58–684) | 17 (5–42) | - |

Table 1 Caption: P < 0.05 was considered statistically significant and was used for the selection of covariates for the subsequent genomic association analysis *For association between clinical characteristic and case-control status, ‡Mann-Whitney U test, †Fisher’s exact test, ¶each category compared against all others combined, §t-test, ₤one patient on renal dialysis received thrice weekly doses of all drugs. ^Hepatotoxic medications considered here include atorvastatin, simvastatin, ibuprofen, clopidogrel, fenofibrate, paracetamol, acarbose, amitriptyline and tolbutamide. These medications were among drugs with likelihood categories A and B in NIH livertox database. ALT: alanine transaminase, AST: aspartate transaminase, BMI: body mass index, INH-DILI: isoniazid-induced liver injury, LFT: liver function test, NIH: National Institute of Health, mg: milligram, sd: standard deviation, TB: tuberculosis

After removing SNPs with call rate <95% or deviation from Hardy-Weinberg Equilibrium (HWE) within Chinese patients, 536859 SNPs remained. Principal components analysis (PCA) performed using our genome-wide SNP data showed distinct clustering according to self-reported ethnicity, except for ‘Others’ which was spread between the Chinese and Malay clusters ([S2 Fig](#pone.0186200.s002)). Although Mann-Whitney U tests on the first 10 principal components (PCs) by case control status were not statistically significant, we used the first 2 PCs to adjust for ethnic differences in subsequent analyses based on the fact that there were different ethnicities in our cohort, scree plot of the eigenvalues ([S3 Fig](#pone.0186200.s003)) and visual inspection of PCA plot ([S2 Fig](#pone.0186200.s002)).

### Pharmacogenomic association analysis

We performed a comprehensive literature review of INH-DILI and identified 20 candidate SNPs with prior evidence of association with INH-DILI, minor allele frequency (MAF) > 0.05 in at least 1 population and present in our genotyping panel ([S1 Table](#pone.0186200.s006)), which we tested for association in our cohort. Of these 20 candidate SNPs, 2 SNPs (rs1041983 and rs1495741), both in the *NAT2* gene, were significantly associated with INH-DILI in our cohort after correction for multiple correlated tests. The recessive model was most significant for rs1041983 (OR (95% confidence interval (CI)) = 13.86 (4.30–44.70), adj P = 4.754 x 10^−4^), whereas the dominant model was most significant for rs1495741 (OR (95% CI) = 0.10 (0.03–0.33), adj P = 0.004) ([Table 2](#pone.0186200.t002) & [S2 Table](#pone.0186200.s007)). Analyses using the first 5 PCs, 20 PCs or self-reported ethnicity instead of the first 2 PCs yielded very similar results. Association with INH-DILI severity by ordinal logistic regression also yielded very similar results ([S3 Table](#pone.0186200.s008)). Cluster plots of these 2 SNPs showed clear clustering of genotypes ([S4 Fig](#pone.0186200.s004)).

### Table 2. Pharmacogenomic association of NAT2 variants in the Singaporean population.

| Gene | SNP | Maj/ min | MAF |  | Additive |  |  | Dominant |  |  | Recessive |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| Cases | Ctrl | OR (95% CI) | P | Adj P | OR (95% CI) | P | Adj P | OR (95% CI) | P | Adj P |  |  |  |
| NAT2 | rs1041983 | G/A | 0.792 | 0.367 | 6.34 (2.54–15.82) | 7.667 x 10−5 | 0.003 | 5.72 (1.18–27.69) | 0.030 | 0.597 | 13.86 (4.30–44.70) | 1.078 x 10−5 | 4.754 x 10−4 |
| NAT2 | rs1495741 | A/G | 0.167 | 0.551 | 0.21 (0.09–0.52) | 6.267 x 10−4 | 0.024 | 0.10 (0.03–0.33) | 1.084 x 10−4 | 0.004 | 0.25 (0.05–1.21) | 0.085 | 0.901 |
| NAT2 | rs1799929 | G/A | 0.042 | 0.076 | 0.36 (0.07–1.95) | 0.235 | 0.995 | 0.36 (0.07–1.95) | 0.235 | 0.995 | NA* |  |  |
| NAT2 | rs1799930 | G/A | 0.5 | 0.222 | 2.85 (1.32–6.16) | 0.008 | 0.226 | 2.16 (0.74–6.28) | 0.159 | 0.980 | 12.00 (2.49–57.84) | 0.002 | 0.069 |
| NAT2 | rs1799931 | G/A | 0.292 | 0.139 | 2.85 (1.20–6.77) | 0.017 | 0.420 | 3.10 (1.05–9.15) | 0.040 | 0.691 | 7.33 (0.96–56.24) | 0.055 | 0.790 |
| NAT2 | rs1801280 | A/G | 0.042 | 0.095 | 0.30 (0.05–1.62) | 0.161 | 0.978 | 0.30 (0.05–1.68) | 0.169 | 0.982 | NA* |  |  |

Table 2 Caption: This table shows the association results from logistic regression with gender, PC1 and PC2 as covariates. P values were adjusted for 55 multiple correlated tests using the p_ACT procedure. Significant SNPs (Adj P <0.05) are bolded. Only results for NAT2 are shown here. *Logistic regression could not be performed due to absence of patients with homozygous variant in either cases or controls. Adj P: adjusted P value, CI: confidence interval, Ctrl: controls, MAF: minor allele frequency, Maj: major allele, Min: minor allele, OR: odds ratio

We next inferred acetylator phenotype using a 4 SNP panel, which did not include either of the 2 significant SNPs [[14](#pone.0186200.ref014)]. Thirty-five patients (34%) were predicted to be SA. SA status was significantly associated with INH-DILI compared to intermediate acetylator (IA) + rapid acetylator (RA) (OR (95% CI) = 9.98 (3.32–33.80), logistic regression p = 8.36 x 10^−5^). Results from Fisher’s exact test were similar ([S4 Table](#pone.0186200.s009)). The frequencies of the risk alleles showed an increasing trend with DILI grade ([Fig 1](#pone.0186200.g001)).

### Fig 1. Risk allele frequency of associated NAT2 variants by DILI grade.

![Fig 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/eedf/5642896/b28437798da8/pone.0186200.g001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5642896_pone.0186200.g001.jpg)

The frequencies of the risk variants (rs1041983 A, rs1495741 A and NAT2 SA) increase with DILI grade. There was only 1 patient with grade 4 DILI, who was homozygous for the non-risk variants. DILI: drug-induced liver injury, SA: slow acetylators.

Genetic associations may be specific to certain populations. To determine if the associations we observed were consistent across all ethnic groups, we also explored the association between NAT2 acetylator status and the 2 significant SNPs within the 3 major ethnic groups in our cohort. The effect sizes remained similar in Chinese but were smaller and consistent in direction in Malays and Indians ([S5 Table](#pone.0186200.s010)).

### Pharmacogenomic risk prediction analyses

Next, we investigated the accuracy and clinical significance of NAT2 acetylator status and the 2 significant *NAT2* SNPs in predicting INH-DILI. Based on the estimated prevalence of serum aminotransferase elevations of 10% [[2](#pone.0186200.ref002)], we estimated the sensitivity, specificity, positive predictive value (PPV) and negative predictive value (NPV) of predicting INH-DILI at a prevalence of 10% from NAT2 SA status to be 75%, 78%, 28% and 97%, respectively. Interestingly, the 2 significant *NAT2* SNPs also demonstrated similar performance measures, though rs1041983 had slightly lower sensitivity but higher specificity and PPV ([Fig 2](#pone.0186200.g002)). As PPV and NPV are dependent on prevalence of the outcome, we also calculated these values for a range of prevalence from 5–20%. While PPV varied considerably (16–47%), NPV was within a narrow range of 93–98% ([S6 Table](#pone.0186200.s011)). The population attributable fraction (PAF) and number needed to test (NNT) for NAT2 SA were estimated to be 0.67 and 4.08, respectively.

### Fig 2. Clinical validity of NAT2 acetylator status and significant SNPs.

![Fig 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/eedf/5642896/112e70eda8f6/pone.0186200.g002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5642896_pone.0186200.g002.jpg)

Prevalence of 10% is assumed and the risk genotypes for all variants were used for the calculation of clinical validity measures. For both rs1041983 and rs1495741, the homozygous AA genotype is the risk genotype since GA/GG is protective. DOM: dominant, NPV: negative predictive value, PPV: positive predictive value, REC: recessive, SA: slow acetylators.

To examine if *NAT2* significantly improves INH-DILI prediction over clinical factors, we then compared the predictive accuracy of a model using clinical variables alone to models that included NAT2 acetylator status using receiver operating characteristic (ROC) curves ([Fig 3](#pone.0186200.g003)) and Youden’s index. The addition of NAT2 acetylator status provided significantly better prediction over a model with clinical variables alone (area under the curve (AUC) = 0.863 vs. 0.766, respectively, p = 0.027) ([Fig 3](#pone.0186200.g003)). We also assessed the additional contributions of rs1041983 and rs1495741 individually as a comparison to NAT2 acetylator status. Models with clinical factors and the 2 SNPs separately also performed similarly, with AUCs of 0.861 (rs1495741) and 0.853 (rs1041983) ([S5 Fig](#pone.0186200.s005)). Interestingly, both models with clinical factors and rs1495741 or rs1041983 were significantly better than clinical factors alone (p = 0.029 and 0.038, respectively), but not significantly different from one with clinical factors and NAT2 acetylator status (p = 0.678 and 0.592, respectively). The Youden’s indexes for clinical factors alone, clinical and NAT2 acetylator status, rs1495741 or rs1041983 were 0.500, 0.698, 0.672 and 0.656, respectively.

### Fig 3. Predictive value of NAT2 acetylator status.

![Fig 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/eedf/5642896/7ca11d570f0f/pone.0186200.g003.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5642896_pone.0186200.g003.jpg)

Predictive values were evaluated using receiver operating characteristic (ROC) curves and expressed as area-under-curve (AUC), which is a summary measure of the sensitivity and specificity. The clinical model (clin) consists of age, gender and self-reported ethnicity. SA: slow acetylators.

We further explored the correlation between the 2 SNPs and NAT2 acetylator status in order to determine how well these SNPs tag NAT2 acetylator status in the Singaporean populations. We found that the rs1495741 AA genotype had almost complete concordance (98.1%) with the SA phenotype ([S7 Table](#pone.0186200.s012)).

## Discussion

Here we assessed the association of previously identified pharmacogenomic biomarkers for INH-DILI in a multiethnic Singaporean population. The major findings of this study are that we confirmed the association for 2 *NAT2* variants and NAT2 SA status with INH-DILI in this population for the first time. The effect size of 9.98 (95% CI: 3.32–33.8) that we observed was larger than the pooled ORs of 3.1 (3.10 (95% CI: 2.47–3.88) and 3.08 (95% CI: 2.29–4.15)) in prior meta-analyses, although the 95% CIs overlap [[12](#pone.0186200.ref012),[13](#pone.0186200.ref013)]. The effect size we found was also similar to that reported in some studies in Japanese and Iranian populations [[17](#pone.0186200.ref017),[22](#pone.0186200.ref022),[41](#pone.0186200.ref041)]. This is the first study to report on the association of NAT2 variants and INH-DILI in a Singaporean population, and adds to the body of literature on the importance of this marker as a predictor of INH-DILI. We also demonstrate the potential clinical utility of NAT2 genotyping for INH-DILI prediction in these Southeast Asian populations.

Our study had >75% power to detect associations for variants with MAF ≥ 0.15 and allelic OR ≥ 4.5 at a significance level of 0.003 (Bonferroni corrected alpha level for 17 independent SNPs). The MAF of majority of the candidate SNPs outside *NAT2* (except *TNF* rs1800629) were > 0.15, so the lack of replication of these variants in our study was likely because the true effect sizes were too small to be detected.

We further demonstrated the clinical validity of testing for *NAT2* variants for the prediction of INH-DILI in patients receiving anti-TB drugs. Based on the effect size in our study, *NAT2* is an effective test for ruling out INH-DILI if patients are found to not have SA status, owing to the high NPV. Importantly, the NPV is robust to uncertainties in the INH-DILI prevalence. This is potentially clinical significant, as it suggests that patients found to be non-SA could safely undergo treatment with INH with limited monitoring for DILI and high confidence that INH-DILI would not occur. Conversely, patients with SA status would likely benefit from closer surveillance for the development of DILI. Two-thirds of INH-DILI cases can be attributed to the presence of NAT2 SA, and about 4 patients would need to be tested to prevent one case, if followed by an intervention that effectively mitigates the risk of INH-DILI. *NAT2* was also shown to significantly improve INH-DILI prediction over clinical factors alone. To our knowledge, this is the first study to include an assessment of the clinical validity of *NAT2* for INH-DILI, which is a necessary step towards clinical implementation of this drug-gene pair. Future studies will be able to formally assess the cost-effectiveness of NAT2 genotyping based on the clinical validity of the test as determined in our study.

Our results add to the body of evidence supporting the use of NAT2 acetylator status for prediction of INH-DILI. For individuals found to be at high genetic risk of INH-DILI, dose-adjustment based on *NAT2* genotype may represent a strategy to reduce the risk of DILI. A clinical trial of pharmacogenomic-guided INH dosing in a Japanese population demonstrated a significant reduction in INH-DILI when NAT2 SA were administered a reduced, without compromising treatment effectiveness [[42](#pone.0186200.ref042)]. Another pilot study linking *NAT2* genotype and INH serum concentration also supports pharmacogenetic-guided INH dosage regimens [[43](#pone.0186200.ref043)]. Further work is needed to establish evidence-based genotype specific clinical recommendations for NAT2 [[44](#pone.0186200.ref044)].

While NAT2 acetylator status has been consistently associated with INH-DILI, results for individual *NAT2* SNPs are much more variable. For example, rs1799930 was significantly associated with INH-DILI in some studies [[23](#pone.0186200.ref023),[27](#pone.0186200.ref027),[32](#pone.0186200.ref032),[34](#pone.0186200.ref034),[35](#pone.0186200.ref035),[45](#pone.0186200.ref045),[46](#pone.0186200.ref046)] but not others [[24](#pone.0186200.ref024),[29](#pone.0186200.ref029),[31](#pone.0186200.ref031),[47](#pone.0186200.ref047),[48](#pone.0186200.ref048)]. One possibility for this observation could be the imperfect tagging of NAT2 acetylator status by a single SNP, which we also observed in our study. A tag SNP for NAT2 acetylator status (rs1495741) was identified in a bladder cancer GWAS in Europeans [[49](#pone.0186200.ref049)]. Other studies have also found high concordance between rs1495741 and NAT2 acetylator status in Asians, Europeans and admixed Americans, but not in Africans [[15](#pone.0186200.ref015),[50](#pone.0186200.ref050)]. Consistent with these observations, we also found high concordance between rs1495741 and NAT2 acetylator status in our cohort. The presence of African Americans in the study of Hein *et al*. may explain the high misclassification rate for rs1495741 reported in that study [[14](#pone.0186200.ref014)]. Taken together, our results suggest rs1495741 could be used to tag NAT2 SA with high accuracy in Southeast Asian populations.

There are several limitations to our study. Firstly, we did not have sufficient Malay and Indian patients to thoroughly explore the ethnic specificity of the associations in those sub-groups. However, it appears that the effect is consistent across the 3 major ethnic groups though the associations in Malay and Indian populations alone did not reach statistical significance due to small sample size. Secondly, some variants previously associated with INH-DILI were not included due to the nature of our genotyping assay. However, a review of these excluded variants found none of sufficient priority in terms of strength of evidence. Most variants were from single studies or were not significant in meta-analyses (*GSTM1* and *CYP2E1* rs6413432). We therefore chose not to genotype or impute them in our study. Thirdly, our study focused on specific candidate gene variants, and did not identify new genetic variants for INH-DILI. Future studies which combine multiple cohorts will be needed to perform GWAS for INH-DILI with appropriately powered sample sizes. Lastly, it was not possible to identify definitively the causative drug for the DILI as patients were taking 4 drugs simultaneously, but INH has the highest hepatotoxic potential among the drugs. All DILI cases were re-challenged and 80% of them tolerated INH upon re-challenge and completed their treatment. However, this is in line with reports that up to 80% of suspected INH-DILI cases tolerate reintroduction of treatment after the resolution of the initial injury [[2](#pone.0186200.ref002)].

In conclusion, we have reestablished the association between NAT2 SA and INH-DILI in a Singaporean population and demonstrated its clinical validity in prediction of INH-DILI. Clinical studies and pharmacoeconomic analyses are now needed to support the clinical implementation of *NAT2* pharmacogenetic testing for INH-DILI in Singapore.

## Material and methods

### Recruitment and detailed clinical characterization of patients

Cases of INH-DILI were defined as patients with Common Terminology Criteria for Adverse Events (CTCAE) v4.03 grade 2 and above for levels of aspartate transaminase (AST) and/or alanine transaminase (ALT) (i.e. >2.5x upper limit of normal (ULN)) [[51](#pone.0186200.ref051)], and were recruited retrospectively. Patients with active TB from 2014 to January 2016 were identified from the National TB registry, and those treated at the TB Control Unit (TBCU) in Tan Tock Seng Hospital were screened. Potential cases were shortlisted if their TB treatment was interrupted because of drug-induced liver reactions. A trained research nurse reviewed the medical records of these patients to determine if they met the inclusion criteria: i) undergoing treatment of active TB, ii) on anti-TB drugs which should include INH, rifampicin, pyrazinamide, and ethambutol, and iii) 21 to 95 years old. Patients were excluded if they had any of the following: i) unable to provide informed consent, ii) pregnant or breast feeding, iii) alcoholism, iv) chronic hepatitis B or C and other types of hepatitis or v) HIV co-infection. There was no limitation on the co-administration of drugs unrelated to TB treatment but we also assessed the possible contribution of potentially hepatotoxic drugs. For this purpose, drugs classified in categories A (>50 published reports) and B (13–50 published reports) for likelihood of causing DILI were considered hepatotoxic [[52](#pone.0186200.ref052),[53](#pone.0186200.ref053)]. Eligible cases were then recalled for informed consent and blood sampling.

Control patients were recruited from October 2015 to April 2016 and were eligible if they had completed 6 months of TB treatment in TBCU with no symptoms of hepatotoxicity during treatment, and had normal post-treatment liver function tests (LFTs). All patients received their anti-TB treatment under direct observed therapy and all controls were adherent to therapy.

Blood or saliva was collected from recruited patients and deoxyribonucleic acid (DNA) extracted using the prepIT-L2P kit (Genotek, Ottawa, Canada) and DNeasy Blood & Tissue Kit (Qiagen GmbH, Hilden, Germany), respectively, according to manufacturer’s instructions. Clinical data were collected from all patients, including demographics, drug treatment details, baseline and follow-up LFTs, which occurred at the point of onset for cases or 6 months after treatment initiation for controls. All patients gave their written informed consent and the study was approved by the National Healthcare Group Domain Specific Review Board.

### Genotyping and quality control

All samples were genotyped using the Illumina HumanOmniExpress-24 beadchip v1.2 (Illumina, San Diego, CA). To verify the accuracy of all genotyping results, multiple positive and negative (water) controls as well as duplicate samples were included in the assays. No genotyping errors were detected with the use of these controls. To remove any samples or SNPs with genotyping failure or errors, QC filtering was performed to remove samples with high IBS, discordant sex, excess heterozygosity or call rate <95%, and SNPs with call rate <95%, deviation from HWE in Chinese (p <1.0 x 10^−6^). All Chinese patients (cases and controls) were chosen as the population for HWE testing to remove SNPs with genotyping errors in view of the presence of population mixture and small sample size. PCA was performed and we included PCs as covariates to adjust for population structure. Cluster plots of SNPs reaching statistical significance were inspected visually.

### Candidate SNP selection

A comprehensive literature search was performed via PubMed to identify previous pharmacogenomic studies on INH-DILI using the key words (isoniazid OR tuberculosis) AND (liver injury OR hepatitis OR hepatotoxicity) AND (pharmacogen* OR gene OR genetic OR allele* OR variant*) on 7 Oct 2016. We also searched the publication list for INH in the Pharmacogenomics Knowledgebase (PharmGKB) [[54](#pone.0186200.ref054)], a publicly available resource with manually curated knowledge on the impact of human genetic variation on drug response, to supplement the primary literature search. Genetic variants that have been found to be associated with INH-DILI at least once were then screened for their allele frequencies in Chinese, Malays and Indians using the Singapore Genome Variation Project database [[40](#pone.0186200.ref040)] or 1000 genomes (Southern Han Chinese (CHS) and Sri Lankan Tamil from the UK (STU), which has previously been shown to be very similar to Singaporean Indians) [[55](#pone.0186200.ref055),[56](#pone.0186200.ref056)], and their presence in the genotyping panel.

### Statistical analysis

The primary study was designed as a case control study comparing a group of adult TB patients with INH-DILI with a drug matched control group without INH-DILI. We compared clinical characteristics between cases and controls to identify any that were significantly different (P < 0.05) between the 2 groups using t-test, Mann-Whitney U test, chi-square test or Fisher’s exact test as appropriate. The associations between the candidate SNPs and INH-DILI were tested using logistic regression under different genetic models (additive/dominant/recessive), with gender and the first 2 PCs as covariates. Results were presented as P-values and OR with 95% CIs. To adjust for multiple testing, we calculated adjusted P-values using P_ACT_, a computationally efficient method which takes into account correlation due to linkage disequilibrium and different genetic models [[57](#pone.0186200.ref057)]. Adjusted P-values of <0.05 were considered statistically significant and the genetic model with the lowest P-value was used for further analyses. The distribution of the individual alleles and genotypes between cases and controls for the associated variants were also assessed and the P-values and ORs with 95% CIs for each of the respective alleles and genotypes estimated using Fisher’s exact test.

We inferred NAT2 acetylator status using a 4 SNP panel (rs1801280, rs1799930, rs1799931 and rs1801279), where RA were homozygous common for all SNPs, IA were heterozygous for any SNP and SA were heterozygous for ≥2 SNPs or homozygous variant for at least 1 SNP [[14](#pone.0186200.ref014)]. rs1801279 was not included in our candidate SNP list as it was not present on our genotyping panel and expected to be monomorphic in Chinese, Malays and Indians ([S1 Table](#pone.0186200.s006)). We therefore considered this SNP to be homozygous wild type in our cohort. The association between NAT2 acetylator status and INH-DILI was tested using logistic regression with covariates included, as well as Fisher’s exact test. All statistical tests performed were 2-sided.

To evaluate the clinical utility of NAT2 acetylator status and associated SNPs, we calculated the sensitivity, specificity, and estimated the PPV and NPV using Baye’s theorem assuming a prevalence of 10% [[58](#pone.0186200.ref058)]. To take into account uncertainty in the estimation of INH-DILI prevalence, we also performed a sensitivity analysis for the PPV and NPV over a range of prevalences (5–20%). We further calculated the PAF and NNT to detect 1 case of INH-DILI according to the following formulas,

| PAF=IPt−IPoIPt | (1) |
| --- | --- |
| NNT=1ARR | (2) |
| --- | --- |
where *IP*_t_ = incidence proportion in the total population and *IP*_o_ = incidence proportion in the unexposed (IA/RA group), and ARR = absolute risk reduction = *IP*_t_—incidence proportion in exposed (SA).

Finally, we demonstrated the clinical utility of NAT2 acetylator status and SNPs over clinical variables using ROC analysis and compared the AUCs for prediction models based on clinical factors (“clinical”) and “clinical +genetic” risk factors using DeLong's test [[59](#pone.0186200.ref059)]. To estimate the additional contribution of genetic factors in a clinical setting, we used all known clinical factors relevant (not invariant) in our cohort, which were age, gender and self-reported ethnicity. The performance of each prediction model (overall sensitivity and specificity) was calculated using AUC of the ROC curves as well as Youden's index (or Youden’s *J* statistic), where *J* = sensitivity + specificity -1. The maximum value of *J* along all points on the ROC curve represents the optimum sensitivity and specificity [[60](#pone.0186200.ref060)]. All analyses were conducted in PLINK v1.9 [[61](#pone.0186200.ref061)] or R version 3.3.3 [[62](#pone.0186200.ref062)].

## Supporting information

## Acknowledgments

The authors would like to thank all patients who participated in the study. We also thank Ms Sek Onn Lam and Ms Beng Choo Ng for their help in patient recruitment, Ms Madeline Kwek for her assistance in processing the patient samples.

## Data Availability

All relevant data are within the paper and its Supporting Information files.

## Funding Statement

This study was supported by the Biomedical Research Council of the Agency for Science, Technology and Research of Singapore (Strategic Positioning Funds - SPF2014/001) to LRB and by the National University of Singapore. LRB is supported by a Canadian Institutes of Health Research New Investigator award and a Michael Smith Foundation for Health Research Scholar Award. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Data Availability Statement

All relevant data are within the paper and its Supporting Information files.

### Supplementary Materials

### Data Availability Statement

All relevant data are within the paper and its Supporting Information files.

## References

1. World Health Organization. Global Tuberculosis Report 2016 [Internet]. 2016. http://apps.who.int/iris/bitstream/10665/250441/1/9789241565394-eng.pdf?ua=1  [http://apps.who.int/iris/bitstream/10665/250441/1/9789241565394-eng.pdf?ua=1](http://apps.who.int/iris/bitstream/10665/250441/1/9789241565394-eng.pdf?ua=1)

2. National Institutes of Health. Isoniazid [Internet]. Clinical and Research Information on Drug-Induced Liver Toxicity. [cited 2016 Nov 8]. https://livertox.nlm.nih.gov//Isoniazid.htm  [https://livertox.nlm.nih.gov//Isoniazid.htm](https://livertox.nlm.nih.gov//Isoniazid.htm)

3. Preziosi P. Isoniazid: metabolic aspects and toxicological correlates. Curr Drug Metab. 2007. December;8(8):839–51.  [DOI](https://doi.org/10.2174/138920007782798216) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18220565/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Drug%20Metab&title=Isoniazid:%20metabolic%20aspects%20and%20toxicological%20correlates&author=P%20Preziosi&volume=8&issue=8&publication_year=2007&pages=839-51&pmid=18220565&doi=10.2174/138920007782798216&)

4. Metushi IG, Cai P, Zhu X, Nakagawa T, Uetrecht JP. A Fresh Look at the Mechanism of Isoniazid-Induced Hepatotoxicity. Clin Pharmacol Ther. 2011. June 16;89(6):911–4. doi: 10.1038/clpt.2010.355  [DOI](https://doi.org/10.1038/clpt.2010.355) | [DOI](https://doi.org/10.1038/clpt.2010.355) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21412230/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=A%20Fresh%20Look%20at%20the%20Mechanism%20of%20Isoniazid-Induced%20Hepatotoxicity&author=IG%20Metushi&author=P%20Cai&author=X%20Zhu&author=T%20Nakagawa&author=JP%20Uetrecht&volume=89&issue=6&publication_year=2011&pages=911-4&pmid=21412230&doi=10.1038/clpt.2010.355&)

5. Nanashima K, Mawatari T, Tahara N, Higuchi N, Nakaura A, Inamine T, et al. Genetic variants in antioxidant pathway: risk factors for hepatotoxicity in tuberculosis patients. Tuberculosis (Edinb). 2012. May;92(3):253–9.  [DOI](https://doi.org/10.1016/j.tube.2011.12.004) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22341855/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Tuberculosis%20(Edinb)&title=Genetic%20variants%20in%20antioxidant%20pathway:%20risk%20factors%20for%20hepatotoxicity%20in%20tuberculosis%20patients&author=K%20Nanashima&author=T%20Mawatari&author=N%20Tahara&author=N%20Higuchi&author=A%20Nakaura&volume=92&issue=3&publication_year=2012&pages=253-9&pmid=22341855&doi=10.1016/j.tube.2011.12.004&)

6. Wang J, Chen R, Tang S, Lv X, Wu S, Zhang Y, et al. Analysis of IL-6, STAT3 and HSPA1L gene polymorphisms in anti-tuberculosis drug-induced hepatitis in a nested case-control study. PLoS One. 2015;10(3):e0118862 doi: 10.1371/journal.pone.0118862  [DOI](https://doi.org/10.1371/journal.pone.0118862) | [DOI](https://doi.org/10.1371/journal.pone.0118862) | [PMC free article](/articles/PMC4366259/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25789467/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=Analysis%20of%20IL-6,%20STAT3%20and%20HSPA1L%20gene%20polymorphisms%20in%20anti-tuberculosis%20drug-induced%20hepatitis%20in%20a%20nested%20case-control%20study&author=J%20Wang&author=R%20Chen&author=S%20Tang&author=X%20Lv&author=S%20Wu&volume=10&issue=3&publication_year=2015&pages=e0118862&pmid=25789467&doi=10.1371/journal.pone.0118862&)

7. Chang J-C, Liu EH, Lee C-N, Lin Y-C, Yu M-C, Bai K-J, et al. UGT1A1 polymorphisms associated with risk of induced liver disorders by anti-tuberculosis medications. Int J Tuberc Lung Dis. 2012;16(3):376–8. doi: 10.5588/ijtld.11.0404  [DOI](https://doi.org/10.5588/ijtld.11.0404) | [DOI](https://doi.org/10.5588/ijtld.11.0404) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22230213/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Tuberc%20Lung%20Dis&title=UGT1A1%20polymorphisms%20associated%20with%20risk%20of%20induced%20liver%20disorders%20by%20anti-tuberculosis%20medications&author=J-C%20Chang&author=EH%20Liu&author=C-N%20Lee&author=Y-C%20Lin&author=M-C%20Yu&volume=16&issue=3&publication_year=2012&pages=376-8&pmid=22230213&doi=10.5588/ijtld.11.0404&)

8. Huang Y-S, Su W-J, Huang Y-H, Chen C-Y, Chang F-Y, Lin H-C, et al. Genetic polymorphisms of manganese superoxide dismutase, NAD(P)H:quinone oxidoreductase, glutathione S-transferase M1 and T1, and the susceptibility to drug-induced liver injury. J Hepatol. 2007. July;47(1):128–34. doi: 10.1016/j.jhep.2007.02.009  [DOI](https://doi.org/10.1016/j.jhep.2007.02.009) | [DOI](https://doi.org/10.1016/j.jhep.2007.02.009) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17400324/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Hepatol&title=Genetic%20polymorphisms%20of%20manganese%20superoxide%20dismutase,%20NAD(P)H:quinone%20oxidoreductase,%20glutathione%20S-transferase%20M1%20and%20T1,%20and%20the%20susceptibility%20to%20drug-induced%20liver%20injury&author=Y-S%20Huang&author=W-J%20Su&author=Y-H%20Huang&author=C-Y%20Chen&author=F-Y%20Chang&volume=47&issue=1&publication_year=2007&pages=128-34&pmid=17400324&doi=10.1016/j.jhep.2007.02.009&)

9. Chen R, Zhang Y, Tang S, Lv X, Wu S, Sun F, et al. The association between HLA-DQB1 polymorphism and antituberculosis drug-induced liver injury: a Case-Control Study. J Clin Pharm Ther. 2015. February;40(1):110–5. doi: 10.1111/jcpt.12211  [DOI](https://doi.org/10.1111/jcpt.12211) | [DOI](https://doi.org/10.1111/jcpt.12211) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25250564/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Pharm%20Ther&title=The%20association%20between%20HLA-DQB1%20polymorphism%20and%20antituberculosis%20drug-induced%20liver%20injury:%20a%20Case-Control%20Study&author=R%20Chen&author=Y%20Zhang&author=S%20Tang&author=X%20Lv&author=S%20Wu&volume=40&issue=1&publication_year=2015&pages=110-5&pmid=25250564&doi=10.1111/jcpt.12211&)

10. Leiro-Fernández V, Valverde D, Vázquez-Gallardo R, Constenla-Caramés L, Del Campo-Pérez V, Fernández-Villar A. HLA-DQ B1*0201 and A1*0102 Alleles Are Not Responsible for Antituberculosis Drug-Induced Hepatotoxicity Risk in Spanish Population. Front Med. 2016;3:34.  [DOI](https://doi.org/10.3389/fmed.2016.00034) | [PMC free article](/articles/PMC4992991/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27597944/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front%20Med&title=HLA-DQ%20B1*0201%20and%20A1*0102%20Alleles%20Are%20Not%20Responsible%20for%20Antituberculosis%20Drug-Induced%20Hepatotoxicity%20Risk%20in%20Spanish%20Population&author=V%20Leiro-Fern%C3%A1ndez&author=D%20Valverde&author=R%20V%C3%A1zquez-Gallardo&author=L%20Constenla-Caram%C3%A9s&author=V%20Del%20Campo-P%C3%A9rez&volume=3&publication_year=2016&pages=34&pmid=27597944&doi=10.3389/fmed.2016.00034&)

11. Kim S-H, Kim S-H, Yoon HJ, Shin DH, Park SS, Kim Y-S, et al. TNF-α genetic polymorphism -308G/A and antituberculosis drug-induced hepatitis. Liver Int. 2012. May;32(5):809–14. doi: 10.1111/j.1478-3231.2011.02697.x  [DOI](https://doi.org/10.1111/j.1478-3231.2011.02697.x) | [DOI](https://doi.org/10.1111/j.1478-3231.2011.02697.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22151084/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Liver%20Int&title=TNF-%CE%B1%20genetic%20polymorphism%20-308G/A%20and%20antituberculosis%20drug-induced%20hepatitis&author=S-H%20Kim&author=S-H%20Kim&author=HJ%20Yoon&author=DH%20Shin&author=SS%20Park&volume=32&issue=5&publication_year=2012&pages=809-14&pmid=22151084&doi=10.1111/j.1478-3231.2011.02697.x&)

12. Du H, Chen X, Fang Y, Yan O, Xu H, Li L, et al. Slow N-acetyltransferase 2 genotype contributes to anti-tuberculosis drug-induced hepatotoxicity: a meta-analysis. Mol Biol Rep. 2013. May 1;40(5):3591–6. doi: 10.1007/s11033-012-2433-y  [DOI](https://doi.org/10.1007/s11033-012-2433-y) | [DOI](https://doi.org/10.1007/s11033-012-2433-y) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23277397/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Biol%20Rep&title=Slow%20N-acetyltransferase%202%20genotype%20contributes%20to%20anti-tuberculosis%20drug-induced%20hepatotoxicity:%20a%20meta-analysis&author=H%20Du&author=X%20Chen&author=Y%20Fang&author=O%20Yan&author=H%20Xu&volume=40&issue=5&publication_year=2013&pages=3591-6&pmid=23277397&doi=10.1007/s11033-012-2433-y&)

13. Shi J, Xie M, Wang J, Xu Y, Liu X. Susceptibility of N-acetyltransferase 2 slow acetylators to antituberculosis drug-induced liver injury: a meta-analysis. Pharmacogenomics. 2015. December;16(18):2083–97. doi: 10.2217/pgs.15.144  [DOI](https://doi.org/10.2217/pgs.15.144) | [DOI](https://doi.org/10.2217/pgs.15.144) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26616266/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Susceptibility%20of%20N-acetyltransferase%202%20slow%20acetylators%20to%20antituberculosis%20drug-induced%20liver%20injury:%20a%20meta-analysis&author=J%20Shi&author=M%20Xie&author=J%20Wang&author=Y%20Xu&author=X%20Liu&volume=16&issue=18&publication_year=2015&pages=2083-97&pmid=26616266&doi=10.2217/pgs.15.144&)

14. Hein DW, Doll MA. Accuracy of various human NAT2 SNP genotyping panels to infer rapid, intermediate and slow acetylator phenotypes. Pharmacogenomics. 2012. January;13(1):31–41. doi: 10.2217/pgs.11.122  [DOI](https://doi.org/10.2217/pgs.11.122) | [DOI](https://doi.org/10.2217/pgs.11.122) | [PMC free article](/articles/PMC3285565/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22092036/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Accuracy%20of%20various%20human%20NAT2%20SNP%20genotyping%20panels%20to%20infer%20rapid,%20intermediate%20and%20slow%20acetylator%20phenotypes&author=DW%20Hein&author=MA%20Doll&volume=13&issue=1&publication_year=2012&pages=31-41&pmid=22092036&doi=10.2217/pgs.11.122&)

15. Ho H-T, Wang T-H, Hsiong C-H, Perng W-C, Wang N-C, Huang T-Y, et al. The NAT2 tag SNP rs1495741 correlates with the susceptibility of antituberculosis drug-induced hepatotoxicity. Pharmacogenet Genomics. 2013. April;23(4):200–7. doi: 10.1097/FPC.0b013e32835e95e1  [DOI](https://doi.org/10.1097/FPC.0b013e32835e95e1) | [DOI](https://doi.org/10.1097/FPC.0b013e32835e95e1) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23407048/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=The%20NAT2%20tag%20SNP%20rs1495741%20correlates%20with%20the%20susceptibility%20of%20antituberculosis%20drug-induced%20hepatotoxicity&author=H-T%20Ho&author=T-H%20Wang&author=C-H%20Hsiong&author=W-C%20Perng&author=N-C%20Wang&volume=23&issue=4&publication_year=2013&pages=200-7&pmid=23407048&doi=10.1097/FPC.0b013e32835e95e1&)

16. Wang F-J, Wang Y, Niu T, Lu W-X, Sandford AJ, He J-Q. Update meta-analysis of the CYP2E1 RsaI/PstI and DraI polymorphisms and risk of antituberculosis drug-induced hepatotoxicity: evidence from 26 studies. J Clin Pharm Ther. 2016. June;41(3):334–40. doi: 10.1111/jcpt.12388  [DOI](https://doi.org/10.1111/jcpt.12388) | [DOI](https://doi.org/10.1111/jcpt.12388) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27062377/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Pharm%20Ther&title=Update%20meta-analysis%20of%20the%20CYP2E1%20RsaI/PstI%20and%20DraI%20polymorphisms%20and%20risk%20of%20antituberculosis%20drug-induced%20hepatotoxicity:%20evidence%20from%2026%20studies&author=F-J%20Wang&author=Y%20Wang&author=T%20Niu&author=W-X%20Lu&author=AJ%20Sandford&volume=41&issue=3&publication_year=2016&pages=334-40&pmid=27062377&doi=10.1111/jcpt.12388&)

17. Ohno M, Yamaguchi I, Yamamoto I, Fukuda T, Yokota S, Maekura R, et al. Slow N-acetyltransferase 2 genotype affects the incidence of isoniazid and rifampicin-induced hepatotoxicity. Int J Tuberc Lung Dis. 2000. March;4(3):256–61.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/10751073/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Tuberc%20Lung%20Dis&title=Slow%20N-acetyltransferase%202%20genotype%20affects%20the%20incidence%20of%20isoniazid%20and%20rifampicin-induced%20hepatotoxicity&author=M%20Ohno&author=I%20Yamaguchi&author=I%20Yamamoto&author=T%20Fukuda&author=S%20Yokota&volume=4&issue=3&publication_year=2000&pages=256-61&pmid=10751073&)

18. Roy B, Chowdhury A, Kundu S, Santra A, Dey B, Chakraborty M, et al. Increased risk of antituberculosis drug-induced hepatotoxicity in individuals with glutathione S-transferase M1 “null” mutation. J Gastroenterol Hepatol. 2001. September;16(9):1033–7.  [DOI](https://doi.org/10.1046/j.1440-1746.2001.02585.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11595069/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Gastroenterol%20Hepatol&title=Increased%20risk%20of%20antituberculosis%20drug-induced%20hepatotoxicity%20in%20individuals%20with%20glutathione%20S-transferase%20M1%20%E2%80%9Cnull%E2%80%9D%20mutation&author=B%20Roy&author=A%20Chowdhury&author=S%20Kundu&author=A%20Santra&author=B%20Dey&volume=16&issue=9&publication_year=2001&pages=1033-7&pmid=11595069&doi=10.1046/j.1440-1746.2001.02585.x&)

19. Huang Y-S, Chern H-D, Su W-J, Wu J-C, Chang S-C, Chiang C-H, et al. Cytochrome P450 2E1 genotype and the susceptibility to antituberculosis drug-induced hepatitis. Hepatology. 2003. April;37(4):924–30. doi: 10.1053/jhep.2003.50144  [DOI](https://doi.org/10.1053/jhep.2003.50144) | [DOI](https://doi.org/10.1053/jhep.2003.50144) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12668988/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hepatology&title=Cytochrome%20P450%202E1%20genotype%20and%20the%20susceptibility%20to%20antituberculosis%20drug-induced%20hepatitis&author=Y-S%20Huang&author=H-D%20Chern&author=W-J%20Su&author=J-C%20Wu&author=S-C%20Chang&volume=37&issue=4&publication_year=2003&pages=924-30&pmid=12668988&doi=10.1053/jhep.2003.50144&)

20. Shimizu Y, Dobashi K, Mita Y, Endou K, Moriya S, Osano K, et al. DNA microarray genotyping of N-acetyltransferase 2 polymorphism using carbodiimide as the linker for assessment of isoniazid hepatotoxicity. Tuberculosis (Edinb). 2006. September;86(5):374–81.  [DOI](https://doi.org/10.1016/j.tube.2005.09.002) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16246623/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Tuberculosis%20(Edinb)&title=DNA%20microarray%20genotyping%20of%20N-acetyltransferase%202%20polymorphism%20using%20carbodiimide%20as%20the%20linker%20for%20assessment%20of%20isoniazid%20hepatotoxicity&author=Y%20Shimizu&author=K%20Dobashi&author=Y%20Mita&author=K%20Endou&author=S%20Moriya&volume=86&issue=5&publication_year=2006&pages=374-81&pmid=16246623&doi=10.1016/j.tube.2005.09.002&)

21. Cho H-J, Koh W-J, Ryu Y-J, Ki C-S, Nam M-H, Kim J-W, et al. Genetic polymorphisms of NAT2 and CYP2E1 associated with antituberculosis drug-induced hepatotoxicity in Korean patients with pulmonary tuberculosis. Tuberculosis (Edinb). 2007. November;87(6):551–6.  [DOI](https://doi.org/10.1016/j.tube.2007.05.012) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17950035/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Tuberculosis%20(Edinb)&title=Genetic%20polymorphisms%20of%20NAT2%20and%20CYP2E1%20associated%20with%20antituberculosis%20drug-induced%20hepatotoxicity%20in%20Korean%20patients%20with%20pulmonary%20tuberculosis&author=H-J%20Cho&author=W-J%20Koh&author=Y-J%20Ryu&author=C-S%20Ki&author=M-H%20Nam&volume=87&issue=6&publication_year=2007&pages=551-6&pmid=17950035&doi=10.1016/j.tube.2007.05.012&)

22. Higuchi N, Tahara N, Yanagihara K, Fukushima K, Suyama N, Inoue Y, et al. NAT2 6A, a haplotype of the N-acetyltransferase 2 gene, is an important biomarker for risk of anti-tuberculosis drug-induced hepatotoxicity in Japanese patients with tuberculosis. World J Gastroenterol. 2007. December 7;13(45):6003–8.  [DOI](https://doi.org/10.3748/wjg.v13.45.6003) | [PMC free article](/articles/PMC4250881/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18023090/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=World%20J%20Gastroenterol&title=NAT2%206A,%20a%20haplotype%20of%20the%20N-acetyltransferase%202%20gene,%20is%20an%20important%20biomarker%20for%20risk%20of%20anti-tuberculosis%20drug-induced%20hepatotoxicity%20in%20Japanese%20patients%20with%20tuberculosis&author=N%20Higuchi&author=N%20Tahara&author=K%20Yanagihara&author=K%20Fukushima&author=N%20Suyama&volume=13&issue=45&publication_year=2007&pages=6003-8&pmid=18023090&doi=10.3748/wjg.v13.45.6003&)

23. Kim S-H, Kim S-H, Bahn J-W, Kim Y-K, Chang Y-S, Shin E-S, et al. Genetic polymorphisms of drug-metabolizing enzymes and anti-TB drug-induced hepatitis. Pharmacogenomics. 2009. November;10(11):1767–79. doi: 10.2217/pgs.09.100  [DOI](https://doi.org/10.2217/pgs.09.100) | [DOI](https://doi.org/10.2217/pgs.09.100) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19891553/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Genetic%20polymorphisms%20of%20drug-metabolizing%20enzymes%20and%20anti-TB%20drug-induced%20hepatitis&author=S-H%20Kim&author=S-H%20Kim&author=J-W%20Bahn&author=Y-K%20Kim&author=Y-S%20Chang&volume=10&issue=11&publication_year=2009&pages=1767-79&pmid=19891553&doi=10.2217/pgs.09.100&)

24. Lee S-W, Chung LS-C, Huang H-H, Chuang T-Y, Liou Y-H, Wu LS-H. NAT2 and CYP2E1 polymorphisms and susceptibility to first-line anti-tuberculosis drug-induced hepatitis. Int J Tuberc Lung Dis. 2010. May;14(5):622–6.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/20392357/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Tuberc%20Lung%20Dis&title=NAT2%20and%20CYP2E1%20polymorphisms%20and%20susceptibility%20to%20first-line%20anti-tuberculosis%20drug-induced%20hepatitis&author=S-W%20Lee&author=LS-C%20Chung&author=H-H%20Huang&author=T-Y%20Chuang&author=Y-H%20Liou&volume=14&issue=5&publication_year=2010&pages=622-6&pmid=20392357&)

25. Bose PD, Sarma MP, Medhi S, Das BC, Husain SA, Kar P. Role of polymorphic N-acetyl transferase2 and cytochrome P4502E1 gene in antituberculosis treatment-induced hepatitis. J Gastroenterol Hepatol. 2011. February;26(2):312–8. doi: 10.1111/j.1440-1746.2010.06355.x  [DOI](https://doi.org/10.1111/j.1440-1746.2010.06355.x) | [DOI](https://doi.org/10.1111/j.1440-1746.2010.06355.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21261721/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Gastroenterol%20Hepatol&title=Role%20of%20polymorphic%20N-acetyl%20transferase2%20and%20cytochrome%20P4502E1%20gene%20in%20antituberculosis%20treatment-induced%20hepatitis&author=PD%20Bose&author=MP%20Sarma&author=S%20Medhi&author=BC%20Das&author=SA%20Husain&volume=26&issue=2&publication_year=2011&pages=312-8&pmid=21261721&doi=10.1111/j.1440-1746.2010.06355.x&)

26. Sotsuka T, Sasaki Y, Hirai S, Yamagishi F, Ueno K. Association of isoniazid-metabolizing enzyme genotypes and isoniazid-induced hepatotoxicity in tuberculosis patients. In Vivo. 25(5):803–12.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/21753138/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=In%20Vivo&title=Association%20of%20isoniazid-metabolizing%20enzyme%20genotypes%20and%20isoniazid-induced%20hepatotoxicity%20in%20tuberculosis%20patients&author=T%20Sotsuka&author=Y%20Sasaki&author=S%20Hirai&author=F%20Yamagishi&author=K%20Ueno&volume=25&issue=5&pages=803-12&pmid=21753138&)

27. An H-R, Wu X-Q, Wang Z-Y, Zhang J-X, Liang Y. NAT2 and CYP2E1 polymorphisms associated with antituberculosis drug-induced hepatotoxicity in Chinese patients. Clin Exp Pharmacol Physiol. 2012. June;39(6):535–43. doi: 10.1111/j.1440-1681.2012.05713.x  [DOI](https://doi.org/10.1111/j.1440-1681.2012.05713.x) | [DOI](https://doi.org/10.1111/j.1440-1681.2012.05713.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22506592/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Exp%20Pharmacol%20Physiol&title=NAT2%20and%20CYP2E1%20polymorphisms%20associated%20with%20antituberculosis%20drug-induced%20hepatotoxicity%20in%20Chinese%20patients&author=H-R%20An&author=X-Q%20Wu&author=Z-Y%20Wang&author=J-X%20Zhang&author=Y%20Liang&volume=39&issue=6&publication_year=2012&pages=535-43&pmid=22506592&doi=10.1111/j.1440-1681.2012.05713.x&)

28. Tang S, Lv X, Zhang Y, Wu S, Yang Z, Xia Y, et al. Cytochrome P450 2E1 gene polymorphisms/haplotypes and anti-tuberculosis drug-induced hepatitis in a Chinese cohort. PLoS One. 2013;8(2):e57526 doi: 10.1371/journal.pone.0057526  [DOI](https://doi.org/10.1371/journal.pone.0057526) | [DOI](https://doi.org/10.1371/journal.pone.0057526) | [PMC free article](/articles/PMC3583841/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23460870/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=Cytochrome%20P450%202E1%20gene%20polymorphisms/haplotypes%20and%20anti-tuberculosis%20drug-induced%20hepatitis%20in%20a%20Chinese%20cohort&author=S%20Tang&author=X%20Lv&author=Y%20Zhang&author=S%20Wu&author=Z%20Yang&volume=8&issue=2&publication_year=2013&pages=e57526&pmid=23460870&doi=10.1371/journal.pone.0057526&)

29. Gupta VH, Amarapurkar DN, Singh M, Sasi P, Joshi JM, Baijal R, et al. Association of N-acetyltransferase 2 and cytochrome P450 2E1 gene polymorphisms with antituberculosis drug-induced hepatotoxicity in Western India. J Gastroenterol Hepatol. 2013. August;28(8):1368–74. doi: 10.1111/jgh.12194  [DOI](https://doi.org/10.1111/jgh.12194) | [DOI](https://doi.org/10.1111/jgh.12194) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23875638/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Gastroenterol%20Hepatol&title=Association%20of%20N-acetyltransferase%202%20and%20cytochrome%20P450%202E1%20gene%20polymorphisms%20with%20antituberculosis%20drug-induced%20hepatotoxicity%20in%20Western%20India&author=VH%20Gupta&author=DN%20Amarapurkar&author=M%20Singh&author=P%20Sasi&author=JM%20Joshi&volume=28&issue=8&publication_year=2013&pages=1368-74&pmid=23875638&doi=10.1111/jgh.12194&)

30. Rana S V, Sharma SK, Ola RP, Kamboj JK, Malik A, Morya RK, et al. N-acetyltransferase 2, cytochrome P4502E1 and glutathione S-transferase genotypes in antitubercular treatment-induced hepatotoxicity in North Indians. J Clin Pharm Ther. 2014. February;39(1):91–6. doi: 10.1111/jcpt.12105  [DOI](https://doi.org/10.1111/jcpt.12105) | [DOI](https://doi.org/10.1111/jcpt.12105) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24188272/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Pharm%20Ther&title=N-acetyltransferase%202,%20cytochrome%20P4502E1%20and%20glutathione%20S-transferase%20genotypes%20in%20antitubercular%20treatment-induced%20hepatotoxicity%20in%20North%20Indians&author=S%20V%20Rana&author=SK%20Sharma&author=RP%20Ola&author=JK%20Kamboj&author=A%20Malik&volume=39&issue=1&publication_year=2014&pages=91-6&pmid=24188272&doi=10.1111/jcpt.12105&)

31. Xiang Y, Ma L, Wu W, Liu W, Li Y, Zhu X, et al. The incidence of liver injury in Uyghur patients treated for TB in Xinjiang Uyghur autonomous region, China, and its association with hepatic enzyme polymorphisms nat2, cyp2e1, gstm1 and gstt1. PLoS One. 2014;9(1):e85905 doi: 10.1371/journal.pone.0085905  [DOI](https://doi.org/10.1371/journal.pone.0085905) | [DOI](https://doi.org/10.1371/journal.pone.0085905) | [PMC free article](/articles/PMC3900431/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24465778/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=The%20incidence%20of%20liver%20injury%20in%20Uyghur%20patients%20treated%20for%20TB%20in%20Xinjiang%20Uyghur%20autonomous%20region,%20China,%20and%20its%20association%20with%20hepatic%20enzyme%20polymorphisms%20nat2,%20cyp2e1,%20gstm1%20and%20gstt1&author=Y%20Xiang&author=L%20Ma&author=W%20Wu&author=W%20Liu&author=Y%20Li&volume=9&issue=1&publication_year=2014&pages=e85905&pmid=24465778&doi=10.1371/journal.pone.0085905&)

32. Singh M, Gupta VH, Amarapurkar DN, Joshi JM, Baijal R, Ramegowda PH, et al. Association of genetic variants with anti-tuberculosis drug induced hepatotoxicity: a high resolution melting analysis. Infect Genet Evol. 2014. April;23:42–8. doi: 10.1016/j.meegid.2014.01.027  [DOI](https://doi.org/10.1016/j.meegid.2014.01.027) | [DOI](https://doi.org/10.1016/j.meegid.2014.01.027) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24491431/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Infect%20Genet%20Evol&title=Association%20of%20genetic%20variants%20with%20anti-tuberculosis%20drug%20induced%20hepatotoxicity:%20a%20high%20resolution%20melting%20analysis&author=M%20Singh&author=VH%20Gupta&author=DN%20Amarapurkar&author=JM%20Joshi&author=R%20Baijal&volume=23&publication_year=2014&pages=42-8&pmid=24491431&doi=10.1016/j.meegid.2014.01.027&)

33. Singla N, Gupta D, Birbian N, Singh J. Association of NAT2, GST and CYP2E1 polymorphisms and anti-tuberculosis drug-induced hepatotoxicity. Tuberculosis (Edinb). 2014. May;94(3):293–8.  [DOI](https://doi.org/10.1016/j.tube.2014.02.003) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24637014/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Tuberculosis%20(Edinb)&title=Association%20of%20NAT2,%20GST%20and%20CYP2E1%20polymorphisms%20and%20anti-tuberculosis%20drug-induced%20hepatotoxicity&author=N%20Singla&author=D%20Gupta&author=N%20Birbian&author=J%20Singh&volume=94&issue=3&publication_year=2014&pages=293-8&pmid=24637014&doi=10.1016/j.tube.2014.02.003&)

34. Mishra S, Daschakraborty S, Shukla P, Kapoor P, Aggarwal R. N-acetyltransferase and cytochrome P450 2E1 gene polymorphisms and susceptibility to antituberculosis drug hepatotoxicity in an Indian population. Natl Med J India. 26(5):260–5.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/25017831/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Natl%20Med%20J%20India&title=N-acetyltransferase%20and%20cytochrome%20P450%202E1%20gene%20polymorphisms%20and%20susceptibility%20to%20antituberculosis%20drug%20hepatotoxicity%20in%20an%20Indian%20population&author=S%20Mishra&author=S%20Daschakraborty&author=P%20Shukla&author=P%20Kapoor&author=R%20Aggarwal&volume=26&issue=5&pages=260-5&pmid=25017831&)

35. Yuliwulandari R, Susilowati RW, Wicaksono BD, Viyati K, Prayuni K, Razari I, et al. NAT2 variants are associated with drug-induced liver injury caused by anti-tuberculosis drugs in Indonesian patients with tuberculosis. J Hum Genet. 2016. June;61(6):533–7. doi: 10.1038/jhg.2016.10  [DOI](https://doi.org/10.1038/jhg.2016.10) | [DOI](https://doi.org/10.1038/jhg.2016.10) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26911349/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Hum%20Genet&title=NAT2%20variants%20are%20associated%20with%20drug-induced%20liver%20injury%20caused%20by%20anti-tuberculosis%20drugs%20in%20Indonesian%20patients%20with%20tuberculosis&author=R%20Yuliwulandari&author=RW%20Susilowati&author=BD%20Wicaksono&author=K%20Viyati&author=K%20Prayuni&volume=61&issue=6&publication_year=2016&pages=533-7&pmid=26911349&doi=10.1038/jhg.2016.10&)

36. Mushiroda T, Yanai H, Yoshiyama T, Sasaki Y, Okumura M, Ogata H, et al. Development of a prediction system for anti-tuberculosis drug-induced liver injury in Japanese patients. Hum genome Var. 2016;3:16014 doi: 10.1038/hgv.2016.14  [DOI](https://doi.org/10.1038/hgv.2016.14) | [DOI](https://doi.org/10.1038/hgv.2016.14) | [PMC free article](/articles/PMC4917605/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27340556/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20genome%20Var&title=Development%20of%20a%20prediction%20system%20for%20anti-tuberculosis%20drug-induced%20liver%20injury%20in%20Japanese%20patients&author=T%20Mushiroda&author=H%20Yanai&author=T%20Yoshiyama&author=Y%20Sasaki&author=M%20Okumura&volume=3&publication_year=2016&pages=16014&pmid=27340556&doi=10.1038/hgv.2016.14&)

37. Roy B, Ghosh SK, Sutradhar D, Sikdar N, Mazumder S, Barman S. Predisposition of antituberculosis drug induced hepatotoxicity by cytochrome P450 2E1 genotype and haplotype in pediatric patients. J Gastroenterol Hepatol. 2006. April;21(4):784–6. doi: 10.1111/j.1440-1746.2006.04197.x  [DOI](https://doi.org/10.1111/j.1440-1746.2006.04197.x) | [DOI](https://doi.org/10.1111/j.1440-1746.2006.04197.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16677176/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Gastroenterol%20Hepatol&title=Predisposition%20of%20antituberculosis%20drug%20induced%20hepatotoxicity%20by%20cytochrome%20P450%202E1%20genotype%20and%20haplotype%20in%20pediatric%20patients&author=B%20Roy&author=SK%20Ghosh&author=D%20Sutradhar&author=N%20Sikdar&author=S%20Mazumder&volume=21&issue=4&publication_year=2006&pages=784-6&pmid=16677176&doi=10.1111/j.1440-1746.2006.04197.x&)

38. Sharma SK, Jha BK, Sharma A, Sreenivas V, Upadhyay V, Jaisinghani C, et al. Genetic polymorphisms of CYP2E1 and GSTM1 loci and susceptibility to anti-tuberculosis drug-induced hepatotoxicity. Int J Tuberc Lung Dis. 2014. May;18(5):588–93. doi: 10.5588/ijtld.13.0344  [DOI](https://doi.org/10.5588/ijtld.13.0344) | [DOI](https://doi.org/10.5588/ijtld.13.0344) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24903797/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Tuberc%20Lung%20Dis&title=Genetic%20polymorphisms%20of%20CYP2E1%20and%20GSTM1%20loci%20and%20susceptibility%20to%20anti-tuberculosis%20drug-induced%20hepatotoxicity&author=SK%20Sharma&author=BK%20Jha&author=A%20Sharma&author=V%20Sreenivas&author=V%20Upadhyay&volume=18&issue=5&publication_year=2014&pages=588-93&pmid=24903797&doi=10.5588/ijtld.13.0344&)

39. Wang T, Yu HT, Wang W, Pan YY, He LX, Wang ZY. Genetic polymorphisms of cytochrome P450 and glutathione S-transferase associated with antituberculosis drug-induced hepatotoxicity in Chinese tuberculosis patients. J Int Med Res. 38(3):977–86. doi: 10.1177/147323001003800324  [DOI](https://doi.org/10.1177/147323001003800324) | [DOI](https://doi.org/10.1177/147323001003800324) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20819434/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Int%20Med%20Res&title=Genetic%20polymorphisms%20of%20cytochrome%20P450%20and%20glutathione%20S-transferase%20associated%20with%20antituberculosis%20drug-induced%20hepatotoxicity%20in%20Chinese%20tuberculosis%20patients&author=T%20Wang&author=HT%20Yu&author=W%20Wang&author=YY%20Pan&author=LX%20He&volume=38&issue=3&pages=977-86&pmid=20819434&doi=10.1177/147323001003800324&)

40. Teo Y-Y, Sim X, Ong RTH, Tan AKS, Chen J, Tantoso E, et al. Singapore Genome Variation Project: a haplotype map of three Southeast Asian populations. Genome Res. 2009. November;19(11):2154–62. doi: 10.1101/gr.095000.109  [DOI](https://doi.org/10.1101/gr.095000.109) | [DOI](https://doi.org/10.1101/gr.095000.109) | [PMC free article](/articles/PMC2775604/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19700652/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Res&title=Singapore%20Genome%20Variation%20Project:%20a%20haplotype%20map%20of%20three%20Southeast%20Asian%20populations&author=Y-Y%20Teo&author=X%20Sim&author=RTH%20Ong&author=AKS%20Tan&author=J%20Chen&volume=19&issue=11&publication_year=2009&pages=2154-62&pmid=19700652&doi=10.1101/gr.095000.109&)

41. Khalili H, Fouladdel S, Sistanizad M, Hajiabdolbaghi M, Azizi E. Association of N-acetyltransferase-2 genotypes and anti-tuberculosis induced liver injury; first case-controlled study from Iran. Curr Drug Saf. 2011. February 1;6(1):17–22.  [DOI](https://doi.org/10.2174/157488611794479946) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21047300/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Drug%20Saf&title=Association%20of%20N-acetyltransferase-2%20genotypes%20and%20anti-tuberculosis%20induced%20liver%20injury;%20first%20case-controlled%20study%20from%20Iran&author=H%20Khalili&author=S%20Fouladdel&author=M%20Sistanizad&author=M%20Hajiabdolbaghi&author=E%20Azizi&volume=6&issue=1&publication_year=2011&pages=17-22&pmid=21047300&doi=10.2174/157488611794479946&)

42. Azuma J, Ohno M, Kubota R, Yokota S, Nagai T, Tsuyuguchi K, et al. NAT2 genotype guided regimen reduces isoniazid-induced liver injury and early treatment failure in the 6-month four-drug standard treatment of tuberculosis: A randomized controlled trial for pharmacogenetics-based therapy. Eur J Clin Pharmacol. 2013. May 14;69(5):1091–101. doi: 10.1007/s00228-012-1429-9  [DOI](https://doi.org/10.1007/s00228-012-1429-9) | [DOI](https://doi.org/10.1007/s00228-012-1429-9) | [PMC free article](/articles/PMC3641305/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23150149/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Clin%20Pharmacol&title=NAT2%20genotype%20guided%20regimen%20reduces%20isoniazid-induced%20liver%20injury%20and%20early%20treatment%20failure%20in%20the%206-month%20four-drug%20standard%20treatment%20of%20tuberculosis:%20A%20randomized%20controlled%20trial%20for%20pharmacogenetics-based%20therapy&author=J%20Azuma&author=M%20Ohno&author=R%20Kubota&author=S%20Yokota&author=T%20Nagai&volume=69&issue=5&publication_year=2013&pages=1091-101&pmid=23150149&doi=10.1007/s00228-012-1429-9&)

43. Jung JA, Kim T-E, Lee H, Jeong B-H, Park HY, Jeon K, et al. A proposal for an individualized pharmacogenetic-guided isoniazid dosage regimen for patients with tuberculosis. Drug Des Devel Ther. 2015;9:5433–8. doi: 10.2147/DDDT.S87131  [DOI](https://doi.org/10.2147/DDDT.S87131) | [DOI](https://doi.org/10.2147/DDDT.S87131) | [PMC free article](/articles/PMC4598210/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26491254/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Des%20Devel%20Ther&title=A%20proposal%20for%20an%20individualized%20pharmacogenetic-guided%20isoniazid%20dosage%20regimen%20for%20patients%20with%20tuberculosis&author=JA%20Jung&author=T-E%20Kim&author=H%20Lee&author=B-H%20Jeong&author=HY%20Park&volume=9&publication_year=2015&pages=5433-8&pmid=26491254&doi=10.2147/DDDT.S87131&)

44. Kim S, Yun YM, Chae HJ, Cho HJ, Ji M, Kim IS, et al. Clinical Pharmacogenetic Testing and Application: Laboratory Medicine Clinical Practice Guidelines. Ann Lab Med. 2017. March;37(2):180–93. doi: 10.3343/alm.2017.37.2.180  [DOI](https://doi.org/10.3343/alm.2017.37.2.180) | [DOI](https://doi.org/10.3343/alm.2017.37.2.180) | [PMC free article](/articles/PMC5204002/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28029011/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Lab%20Med&title=Clinical%20Pharmacogenetic%20Testing%20and%20Application:%20Laboratory%20Medicine%20Clinical%20Practice%20Guidelines&author=S%20Kim&author=YM%20Yun&author=HJ%20Chae&author=HJ%20Cho&author=M%20Ji&volume=37&issue=2&publication_year=2017&pages=180-93&pmid=28029011&doi=10.3343/alm.2017.37.2.180&)

45. Possuelo LG, Castelan JA, de Brito TC, Ribeiro AW, Cafrune PI, Picon PD, et al. Association of slow N-acetyltransferase 2 profile and anti-TB drug-induced hepatotoxicity in patients from Southern Brazil. Eur J Clin Pharmacol. 2008. July;64(7):673–81. doi: 10.1007/s00228-008-0484-8  [DOI](https://doi.org/10.1007/s00228-008-0484-8) | [DOI](https://doi.org/10.1007/s00228-008-0484-8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18421452/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Clin%20Pharmacol&title=Association%20of%20slow%20N-acetyltransferase%202%20profile%20and%20anti-TB%20drug-induced%20hepatotoxicity%20in%20patients%20from%20Southern%20Brazil&author=LG%20Possuelo&author=JA%20Castelan&author=TC%20de%20Brito&author=AW%20Ribeiro&author=PI%20Cafrune&volume=64&issue=7&publication_year=2008&pages=673-81&pmid=18421452&doi=10.1007/s00228-008-0484-8&)

46. Ben Mahmoud L, Ghozzi H, Kamoun A, Hakim A, Hachicha H, Hammami S, et al. Polymorphism of the N-acetyltransferase 2 gene as a susceptibility risk factor for antituberculosis drug-induced hepatotoxicity in Tunisian patients with tuberculosis. Pathol Biol (Paris). 2012. October;60(5):324–30.  [DOI](https://doi.org/10.1016/j.patbio.2011.07.001) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21856096/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pathol%20Biol%20(Paris)&title=Polymorphism%20of%20the%20N-acetyltransferase%202%20gene%20as%20a%20susceptibility%20risk%20factor%20for%20antituberculosis%20drug-induced%20hepatotoxicity%20in%20Tunisian%20patients%20with%20tuberculosis&author=L%20Ben%20Mahmoud&author=H%20Ghozzi&author=A%20Kamoun&author=A%20Hakim&author=H%20Hachicha&volume=60&issue=5&publication_year=2012&pages=324-30&pmid=21856096&doi=10.1016/j.patbio.2011.07.001&)

47. Lv X, Tang S, Xia Y, Zhang Y, Wu S, Yang Z, et al. NAT2 genetic polymorphisms and anti-tuberculosis drug-induced hepatotoxicity in Chinese community population. Ann Hepatol. 11(5):700–7.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/22947533/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Hepatol&title=NAT2%20genetic%20polymorphisms%20and%20anti-tuberculosis%20drug-induced%20hepatotoxicity%20in%20Chinese%20community%20population&author=X%20Lv&author=S%20Tang&author=Y%20Xia&author=Y%20Zhang&author=S%20Wu&volume=11&issue=5&pages=700-7&pmid=22947533&)

48. Santos NPC, Callegari-Jacques SM, Ribeiro Dos Santos AKC, Silva CA, Vallinoto ACR, Fernandes DCRO, et al. N-acetyl transferase 2 and cytochrome P450 2E1 genes and isoniazid-induced hepatotoxicity in Brazilian patients. Int J Tuberc Lung Dis. 2013. April;17(4):499–504. doi: 10.5588/ijtld.12.0645  [DOI](https://doi.org/10.5588/ijtld.12.0645) | [DOI](https://doi.org/10.5588/ijtld.12.0645) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23394127/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Tuberc%20Lung%20Dis&title=N-acetyl%20transferase%202%20and%20cytochrome%20P450%202E1%20genes%20and%20isoniazid-induced%20hepatotoxicity%20in%20Brazilian%20patients&author=NPC%20Santos&author=SM%20Callegari-Jacques&author=AKC%20Ribeiro%20Dos%20Santos&author=CA%20Silva&author=ACR%20Vallinoto&volume=17&issue=4&publication_year=2013&pages=499-504&pmid=23394127&doi=10.5588/ijtld.12.0645&)

49. Rothman N, Garcia-Closas M, Chatterjee N, Malats N, Wu X, Figueroa JD, et al. A multi-stage genome-wide association study of bladder cancer identifies multiple susceptibility loci. Nat Genet. 2010. November 24;42(11):978–84. doi: 10.1038/ng.687  [DOI](https://doi.org/10.1038/ng.687) | [DOI](https://doi.org/10.1038/ng.687) | [PMC free article](/articles/PMC3049891/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20972438/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=A%20multi-stage%20genome-wide%20association%20study%20of%20bladder%20cancer%20identifies%20multiple%20susceptibility%20loci&author=N%20Rothman&author=M%20Garcia-Closas&author=N%20Chatterjee&author=N%20Malats&author=X%20Wu&volume=42&issue=11&publication_year=2010&pages=978-84&pmid=20972438&doi=10.1038/ng.687&)

50. Suarez-Kurtz G, Fuchshuber-Moraes M, Struchiner CJ, Parra EJ. Single nucleotide polymorphism coverage and inference of N-acetyltransferase-2 acetylator phenotypes in wordwide population groups. Pharmacogenet Genomics. 2016. August;26(8):363–9. doi: 10.1097/FPC.0000000000000225  [DOI](https://doi.org/10.1097/FPC.0000000000000225) | [DOI](https://doi.org/10.1097/FPC.0000000000000225) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27136043/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Single%20nucleotide%20polymorphism%20coverage%20and%20inference%20of%20N-acetyltransferase-2%20acetylator%20phenotypes%20in%20wordwide%20population%20groups&author=G%20Suarez-Kurtz&author=M%20Fuchshuber-Moraes&author=CJ%20Struchiner&author=EJ%20Parra&volume=26&issue=8&publication_year=2016&pages=363-9&pmid=27136043&doi=10.1097/FPC.0000000000000225&)

51. National Institutes of Health. Severity Grading in Drug Induced Liver Injury [Internet]. Clinical and Research Information on Drug-Induced Liver Toxicity; [cited 2016 Nov 8]. https://livertox.nlm.nih.gov/Severity.html  [https://livertox.nlm.nih.gov/Severity.html](https://livertox.nlm.nih.gov/Severity.html) | [Google Scholar](https://scholar.google.com/scholar_lookup?title=Severity%20Grading%20in%20Drug%20Induced%20Liver%20Injury&)

52. National Institutes of Health. Categorization of the likelihood of drug induced liver injury [Internet]. Clinical and Research Information on Drug-Induced Liver Toxicity; [cited 2017 Jan 13]. https://livertox.nih.gov/DrugCategory.html  [https://livertox.nih.gov/DrugCategory.html](https://livertox.nih.gov/DrugCategory.html) | [Google Scholar](https://scholar.google.com/scholar_lookup?title=Categorization%20of%20the%20likelihood%20of%20drug%20induced%20liver%20injury&)

53. Björnsson ES. Hepatotoxicity by Drugs: The Most Common Implicated Agents. Int J Mol Sci. 2016. February 6;17(2):224 doi: 10.3390/ijms17020224  [DOI](https://doi.org/10.3390/ijms17020224) | [DOI](https://doi.org/10.3390/ijms17020224) | [PMC free article](/articles/PMC4783956/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26861310/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Mol%20Sci&title=Hepatotoxicity%20by%20Drugs:%20The%20Most%20Common%20Implicated%20Agents&author=ES%20Bj%C3%B6rnsson&volume=17&issue=2&publication_year=2016&pages=224&pmid=26861310&doi=10.3390/ijms17020224&)

54. Whirl-Carrillo M, McDonagh EM, Hebert JM, Gong L, Sangkuhl K, Thorn CF, et al. Pharmacogenomics knowledge for personalized medicine. Clin Pharmacol Ther. 2012. October;92(4):414–7. doi: 10.1038/clpt.2012.96  [DOI](https://doi.org/10.1038/clpt.2012.96) | [DOI](https://doi.org/10.1038/clpt.2012.96) | [PMC free article](/articles/PMC3660037/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22992668/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Pharmacogenomics%20knowledge%20for%20personalized%20medicine&author=M%20Whirl-Carrillo&author=EM%20McDonagh&author=JM%20Hebert&author=L%20Gong&author=K%20Sangkuhl&volume=92&issue=4&publication_year=2012&pages=414-7&pmid=22992668&doi=10.1038/clpt.2012.96&)

55. 1000 Genomes Project Consortium, Abecasis GR, Auton A, Brooks LD, DePristo MA, Durbin RM, et al. An integrated map of genetic variation from 1,092 human genomes. Nature. 2012;491(7422):56–65. doi: 10.1038/nature11632  [DOI](https://doi.org/10.1038/nature11632) | [DOI](https://doi.org/10.1038/nature11632) | [PMC free article](/articles/PMC3498066/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23128226/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=An%20integrated%20map%20of%20genetic%20variation%20from%201,092%20human%20genomes&author=GR%20Abecasis&author=A%20Auton&author=LD%20Brooks&author=MA%20DePristo&author=RM%20Durbin&volume=491&issue=7422&publication_year=2012&pages=56-65&pmid=23128226&doi=10.1038/nature11632&)

56. Chan SL, Samaranayake N, Ross CJD, Toh MT, Carleton B, Hayden MR, et al. Genetic diversity of variants involved in drug response and metabolism in Sri Lankan populations: implications for clinical implementation of pharmacogenomics. Pharmacogenet Genomics. 2016. January;26(1):28–39. doi: 10.1097/FPC.0000000000000182  [DOI](https://doi.org/10.1097/FPC.0000000000000182) | [DOI](https://doi.org/10.1097/FPC.0000000000000182) | [PMC free article](/articles/PMC4890825/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26444257/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Genetic%20diversity%20of%20variants%20involved%20in%20drug%20response%20and%20metabolism%20in%20Sri%20Lankan%20populations:%20implications%20for%20clinical%20implementation%20of%20pharmacogenomics&author=SL%20Chan&author=N%20Samaranayake&author=CJD%20Ross&author=MT%20Toh&author=B%20Carleton&volume=26&issue=1&publication_year=2016&pages=28-39&pmid=26444257&doi=10.1097/FPC.0000000000000182&)

57. Conneely KN, Boehnke M. So many correlated tests, so little time! Rapid adjustment of P values for multiple correlated tests. Am J Hum Genet. 2007. December;81(6):1158–68. doi: 10.1086/522036  [DOI](https://doi.org/10.1086/522036) | [DOI](https://doi.org/10.1086/522036) | [PMC free article](/articles/PMC2276357/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17966093/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=So%20many%20correlated%20tests,%20so%20little%20time!%20Rapid%20adjustment%20of%20P%20values%20for%20multiple%20correlated%20tests&author=KN%20Conneely&author=M%20Boehnke&volume=81&issue=6&publication_year=2007&pages=1158-68&pmid=17966093&doi=10.1086/522036&)

58. Altman DG, Bland JM. Diagnostic tests 2: Predictive values. BMJ. 1994. July 9;309(6947):102  [DOI](https://doi.org/10.1136/bmj.309.6947.102) | [PMC free article](/articles/PMC2540558/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8038641/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMJ&title=Diagnostic%20tests%202:%20Predictive%20values&author=DG%20Altman&author=JM%20Bland&volume=309&issue=6947&publication_year=1994&pages=102&pmid=8038641&doi=10.1136/bmj.309.6947.102&)

59. DeLong ER, DeLong DM, Clarke-Pearson DL. Comparing the areas under two or more correlated receiver operating characteristic curves: a nonparametric approach. Biometrics. 1988. September;44(3):837–45.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/3203132/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biometrics&title=Comparing%20the%20areas%20under%20two%20or%20more%20correlated%20receiver%20operating%20characteristic%20curves:%20a%20nonparametric%20approach&author=ER%20DeLong&author=DM%20DeLong&author=DL%20Clarke-Pearson&volume=44&issue=3&publication_year=1988&pages=837-45&pmid=3203132&)

60. Youden WJ. Index for rating diagnostic tests. Cancer. 1950. January;3(1):32–5.  [DOI](https://doi.org/10.1002/1097-0142(1950)3:1<32::aid-cncr2820030106>3.0.co;2-3) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15405679/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer&title=Index%20for%20rating%20diagnostic%20tests&author=WJ%20Youden&volume=3&issue=1&publication_year=1950&pages=32-5&pmid=15405679&doi=10.1002/1097-0142(1950)3:1<32::aid-cncr2820030106>3.0.co;2-3&)

61. Chang CC, Chow CC, Tellier LC, Vattikuti S, Purcell SM, Lee JJ. Second-generation PLINK: rising to the challenge of larger and richer datasets. Gigascience. 2015;4:7 doi: 10.1186/s13742-015-0047-8  [DOI](https://doi.org/10.1186/s13742-015-0047-8) | [DOI](https://doi.org/10.1186/s13742-015-0047-8) | [PMC free article](/articles/PMC4342193/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25722852/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gigascience&title=Second-generation%20PLINK:%20rising%20to%20the%20challenge%20of%20larger%20and%20richer%20datasets&author=CC%20Chang&author=CC%20Chow&author=LC%20Tellier&author=S%20Vattikuti&author=SM%20Purcell&volume=4&publication_year=2015&pages=7&pmid=25722852&doi=10.1186/s13742-015-0047-8&)

62. R Core Team. R: A Language and Environment for Statistical Computing [Internet]. R Foundation for Statistical Computing; 2014. http://www.R-project.org  [http://www.R-project.org](http://www.R-project.org) | [Google Scholar](https://scholar.google.com/scholar_lookup?title=R:%20A%20Language%20and%20Environment%20for%20Statistical%20Computing&publication_year=2014&)
