# Functional assessment of CYP3A4 allelic variants on lidocaine metabolism in vitro

## Metadata
**Authors:** Ping Fang, Peng-fei Tang, Ren-ai Xu, Xiang Zheng, Jian Wen, Su-su Bao, Jian-ping Cai, Guo-xin Hu
**Journal:** Drug Design, Development and Therapy
**Date:** 2017 Dec 7
**DOI:** [10.2147/DDDT.S152366](https://doi.org/10.2147/DDDT.S152366)
**PMID:** 29263648
**PMCID:** PMC5724423
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5724423/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC5724423/pdf/dddt-11-3503.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC5724423/pdf/dddt-11-3503.pdf)

## Abstract

**Aim:** 
Human cytochrome P450 3A4 is the most abundant isoform of P450 enzyme in the liver. It plays an important role in the metabolism of wide variety of xenobiotic and endogenous substrates. So far, there are few reports about the functional characterization of CYP3A4 variants in terms of specific substrates. The aim of this study was to systematically investigate the genetic polymorphisms of 23 CYP3A4 alleles and evaluate their catalytic activities on the metabolism of lidocaine in vitro.

**Methods and results:** 
The wild-type and 22 CYP3A4 variants were expressed in Spodoptera frugiperda 21 insect cells. Then the insect microsomes were incubated with the CYP3A4-specific substrate lidocaine. Reactions were performed with 50–3,000 µM for 60 min at 37°C. Lidocaine and its metabolite monoethylglycinexylidide were analyzed by ultra-performance liquid chromatography-tandem mass spectrometry system. Of the 23 CYP3A4 allelic variants tested, 2 variants (CYP3A4*17 and CYP3A4*30) had no detectable enzyme activity; and 5 variants (CYP3A4*2, CYP3A4*5, CYP3A4*9, CYP3A4*16 and CYP3A4*24) showed significantly decreased intrinsic clearance values compared with wild-type CYP3A4*1.

**Conclusion:** 
As the first study of all these CYP3A4 alleles for lidocaine metabolism, our results in vitro assessment may provide novel insights into the allele-specific and substrate-specific activity of CYP3A4 and may also offer a reference to the personalized treatment of lidocaine in a clinical setting.

Keywords: CYP3A4, genetic polymorphism, drug metabolism, lidocaine, personalized treatment

### Aim

Human cytochrome P450 3A4 is the most abundant isoform of P450 enzyme in the liver. It plays an important role in the metabolism of wide variety of xenobiotic and endogenous substrates. So far, there are few reports about the functional characterization of *CYP3A4* variants in terms of specific substrates. The aim of this study was to systematically investigate the genetic polymorphisms of 23 *CYP3A4* alleles and evaluate their catalytic activities on the metabolism of lidocaine in vitro.

### Methods and results

The wild-type and 22 *CYP3A4* variants were expressed in *Spodoptera frugiperda 21* insect cells. Then the insect microsomes were incubated with the CYP3A4-specific substrate lidocaine. Reactions were performed with 50–3,000 µM for 60 min at 37°C. Lidocaine and its metabolite monoethylglycinexylidide were analyzed by ultra-performance liquid chromatography-tandem mass spectrometry system. Of the 23 *CYP3A4* allelic variants tested, 2 variants (*CYP3A4*17* and *CYP3A4*30*) had no detectable enzyme activity; and 5 variants (*CYP3A4*2*, *CYP3A4*5*, *CYP3A4*9*, *CYP3A4*16* and *CYP3A4*24*) showed significantly decreased intrinsic clearance values compared with wild-type *CYP3A4*1*.

### Conclusion

As the first study of all these *CYP3A4* alleles for lidocaine metabolism, our results in vitro assessment may provide novel insights into the allele-specific and substrate-specific activity of *CYP3A4* and may also offer a reference to the personalized treatment of lidocaine in a clinical setting.

## Introduction

Lidocaine, which was the first sodium (Na^+^) channel blocker, is still widely used in clinic as an amide-type local anesthetic agent now.[1](#b1-dddt-11-3503) It inhibits nerve impulse transmission by blocking the fast voltage-gated Na^+^ channels in the neuronal cell membrane thereby producing local anesthetic action. Lidocaine has good activity when it is applied to superficial body sites, penetrating mucous membranes to reduce the sensation of pain.[2](#b2-dddt-11-3503) Additionally, lidocaine is also an antiarrhythmic drug, which is administered intravenously for the prevention and treatment of ventricular arrhythmias in patients with acute myocardial infarction or other cardiac diseases.[3](#b3-dddt-11-3503) Nowadays, lidocaine is prevailing used in cosmetic plastic surgery under local infiltration anesthesia.[4](#b4-dddt-11-3503),[5](#b5-dddt-11-3503)

Lidocaine undergoes extensive metabolism in the liver. Monoethylglycinexylidide (MEGX) is the principle metabolite and 3-OH lidocaine is a minor metabolite.[6](#b6-dddt-11-3503) Based on previous in vitro studies, cytochrome P450 3A4 (CYP3A4) has been regarded as the main CYP isoform responsible for the de-ethylation of lidocaine to its primary metabolite, MEGX.[7](#b7-dddt-11-3503),[8](#b8-dddt-11-3503)

CYP3A4, as a member of P450 enzyme superfamily, is the most important drug-metabolizing enzyme in human liver and gastrointestinal tract.[9](#b9-dddt-11-3503) It is well known that CYP3A4 enzyme plays a major role in the oxidative metabolism of ~30%–40% of clinically used drugs, such as certain antibiotics (erythromycin), antiarrhythmics (quinidine), antidepressants (sertraline), Cardiovascular drugs (verapamil) as well as some endogenous steroids (testosterone and estradiol).[10](#b10-dddt-11-3503),[11](#b11-dddt-11-3503) The expression of CYP3A4 varies 40-fold in individual human livers,[12](#b12-dddt-11-3503) which indicates that CYP3A4 is one of the highly polymorphic enzymes. Recently, increasing evidence has shown that genetic polymorphisms of *CYP3A4* significantly contribute to the wide inter-individual variability of enzyme activity.[13](#b13-dddt-11-3503) The enormous variation observed in drug metabolism may result in undesirable adverse side effects or therapeutic failure, thus more attention should be paid to genetic polymorphism of *CYP3A4*.

To date, 46 different types of *CYP3A4* allelic variants have been identified in the Human CYP Allele Nomenclature Committee website ([http://www.cypalleles.ki.se/cyp3a4.htm](http://www.cypalleles.ki.se/cyp3a4.htm)). In our previous study, Hu et al[16](#b16-dddt-11-3503) sequenced the promoter regions and 13 exon junctions of *CYP3A4* in DNA from 1,114 healthy subjects from the southern region of China. Five previously reported alleles (*CYP3A4*1G*, **4*, **5*, **18B* and **23*) were detected.[14](#b14-dddt-11-3503) In addition, a total of 7 novel exonic variants were also detected and further named as new alleles *CYP3A4*28–*34* by the Human CYP Allele Nomenclature Committee. In this study, lidocaine, as a *CYP3A4*-specific substrate, was used to assess the catalytic activities of wild-type *CYP3A4*1* and 22 *CYP3A4* alleles found in the Chinese Han population (15 alleles were previously reported, and 7 novel alleles with coding regions were newly found).

## Materials and method

### Chemicals and materials

Lidocaine was obtained from Shanghai Canspec Scientific Instruments Co., Ltd (Perfemiker, Shanghai, China) and its metabolite MEGX was purchased from Toronto Research Chemicals Inc (Toronto, Ontario, Canada). The reduced nicotinamide adenine dinucleotide phosphate (NADPH) and procaine (internal standard) were obtained from Sigma (St Louis, MO, USA). Cytochrome b5 microsomes were kind gifts from Beijing Hospital.[24](#b24-dddt-11-3503) *Spodoptera frugiperda 21* (*Sf21*) insect cells, fetal bovine serum, Sf-900™ III SFM insect culture medium, pFastBac Dual vector and Bac-to-Bac Baculovirus Expression System were purchased from Invitrogen (Carlsbad, CA, USA). The CORTECS^®^ ultra-performance liquid chromatography (UPLC)^®^ hydrophilic interaction liquid chromatography (HILIC)^®^ column (2.1×50 mm, 1.6 µM) was obtained from the Waters (Dublin, Ireland). High performance liquid chromatography (HPLC)-grade organic solvents and LC-MS grade acetonitrile were from Merck (Darmstadt, Germany). Ultrapure water was freshly purified by a Milli-Q A10 System (Milli-pore, Billerica, MA, USA). Formic acid (FA, 98% purity) of LC-MS grade was from Sigma-Aldrich (Munich, Germany). All the other chemicals and solvents used were of the analytical grade commercially available.

### Construction of the expression vectors

The dual-expression baculovirus vector pFastBac Dual was used to create the intermediate plasmid pFastBac-OR by inserting the CYPOR cDNA into the cloning sites downstream of the P10 promoter. The open reading frame (ORF) fragments for each *CYP3A4* variants were constructed by the overlap extension polymerase chain reaction (PCR) amplification method using the primers that are listed in [Table 1](#t1-dddt-11-3503), and then isolated ORF fragments by PCR amplification using the forward primer 3A4-SALF (5′-ACCAGTCGACATGG CTCTCATCCCAGAC-3′, introducing 1 Sal site) and the reverse primer 3A4-SR (5′-CCAAACTAGTTCAGGCT CCACTTACGGT-3′, introducing 1 Spe site). These full-length PCR products for each of the *CYP3A4* variants were digested and ligated to the pFastBac-OR receptor vector to produce the ultimate dual-expression vector pFastBac-OR-CYP3A4. To ensure that no errors were introduced during PCR amplification, all of the cDNA regions were confirmed by sequencing the plasmid constructs using the CEQ DTCS Quick Start Kit (Beckman Coulter, Inc, Brea, CA, USA) on the CEQ 8000 Genetic Analysis System.

### Table 1.

| Variants | cDNA changes | Reverse primer (5′→3′) | Forward primer (5′→3′) |
| --- | --- | --- | --- |
| CYP3A4*2 (S222P) | 664T>C | CTGTTATTGGGAGAAAGAA | TTCTTTCTCCCAATAACAG |
| CYP3A4*3 (M445T) | 1334T>C | GCAAACCTCGTGCCAATGC | GCATTGGCACGAGGTTTGC |
| CYP3A4*4 (I118V) | 352A>G | CTATAGAGACGGCACTTTT | AAAAGTGCC |
| CYP3A4*5 (P218R) | 653C>G | AGAAAGAATCGATCCAAA | TTTGGATCGATTCTTTCT |
| CYP3A4*9 (V170I) | 508G>A | TCAAGGTGATAGGCTTGCC | GGCAAGCCTATCACCTTGA |
| CYP3A4*10 (D174H) | 520G>C | CAAAGACGTGTTTCAAGGT | ACCTTGAAACACGTCTTTG |
| CYP3A4*11 (T363M) | 1088C>T | AATCTGAGCATTTCATTCA | TGAATGAAATGCTCAGATT |
| CYP3A4*14 (L15P) | 44T>C | ACCAGTCGACATGGCTCTCATCCCAGAC | TTGGCCATGGAAACCTGGCTTCTCCCGGCTG |
| CYP3A4*15 (R162Q) | 485G>A | TCTGCTTCCTGCCTCAGAT | ATCTGAGGCAGGAAGCAGA |
| CYP3A4*16 (T185S) | 554C>G | GATGTGCTACTGATCACAT | ATGTGATCAGTAGCACATC |
| CYP3A4*17 (F189S) | 566T>C | TTCACTCCAGATGATGTGC | GCACATCATCTGGAGTGAA |
| CYP3A4*18 (L293P) | 878T>C | ACGAGCTCCGGATCGGACA | TGTCCGATCCGGAGCTCGT |
| CYP3A4*19 (P467S) | 1399C>T | CTTTACAAGATTTGAAGGA | TCCTTCAAATCTTGTAAAG |
| CYP3A4*23 (R162W) | 484C>T | CTGCTTCCCACCTCAGATT | AATCTGAGGTGGGAAGCAG |
| CYP3A4*24 (Q200H) | 600A>T | AAAGGGGTCATGTGGATTG | CAATCCACATGACCCCTTT |
| CYP3A4*28 (L22V) | 64C>G | ATAGATAGACGAGCACCAG | CTGGTGCTCGTCTATCTAT |
| CYP3A4*29 (F113I) | 337T>A | TTTTCATAATTCCCACTGG | CCAGTGGGAATTATGAAAA |
| CYP3A4*30 (R130STOP) | 388C>T | GCAATGATCATAATCTCTT | AAGAGATTATGATCATTGC |
| CYP3A4*31 (H324Q) | 972C>A | GACATCAGGTTGAGTGGCC | GGCCACTCAACCTGATGTC |
| CYP3A4*32 (I335T) | 1004T>C | ACTGCATCAGTTTCCTCCT | AGGAGGAAACTGATGCAGT |
| CYP3A4*33 (A370S) | 1108G>T | GTCTCATAGAAATTGGGAA | TTCCCAATTTCTATGAGAC |
| CYP3A4*34 (I427V) | 1279A>G | AAGGATCTACGTTGTCCTT | AAGGACAACGTAGATCCTT |

Table 1 Caption: PCR primers used for the site-mutation of CYP3A4

### Expression of CYP3A4 variant proteins in Sf21 insect cells

According to the manufacturer’s procedure, recombinant baculoviruses were effectively generated in *Sf21* insect cells. And then *Sf21* insect cells were infected with these viruses in Sf-900™ III SFM insect culture medium containing 10% fetal bovine serum, 1× Antibiotic-Antimycotic and 4 µg/mL hemin. Seventy two hours after transfection, the cells were collected and re-suspended with 100 mM phosphate buffer (pH 7.4) containing 1 mM EDTA, 0.25 M sucrose and 0.5 mM phenylmethylsulfonyl fluoride, then the cell suspension was centrifuged at 1,600× *g* for 5 min. After centrifugation, the pellets were re-suspended and sonicated for 50 s on ice and centrifuged at 14,000× *g* at 4°C for 20 min to remove the precipitate. The microsomal fraction was prepared by ultracentrifugation at 100,000× *g* at 4°C for 1 h, and this pellet was re-suspended in stock solution (100 mM KPO4, pH 7.4, containing 20% glycerol) and stored at −80°C before use.

### Protein expression levels determination

Difference spectra of microsomal preparations were measured in UV-Visible spectrophotometer (Thermo Scientific, Waltham, MA, USA) with 1-cm optical path cuvettes. Protein concentration was determined by performing Pierce™ BCA protein assay kit (Thermo Scientific). Microsomal preparations, usually containing 2 mg of protein per mL of 0.1 M phosphate buffer (pH 7.0), were placed in both the sample and reference cells. After recording the base-line, CO was carefully bubbled through the sample for about 20 s, and then reduction of samples with a few milligrams of solid Na_2_S_2_O_4_. CYP3A4 expression levels were determined by measured difference spectra.

### Conditions for enzymatic activity analysis

The incubation mixture included 5 pmol wild-type *CYP3A4*1* or other *CYP3A4* mutants, 5 pmol purified cytochrome b5, lidocaine and 100 mmol/L potassium phosphate buffer (pH 7.4). The addition of b5 was to deliver electrons and support CYP3A4 enzymatic activities. Lidocaine was initially prepared in methanol solution and the final concentration of lidocaine in the mixture for kinetic analysis ranged from 50 to 3,000 µM. The reaction was allowed to preincubate for 5 min at 37°C in a Fisher shaking water bath. Then 10 µL 1 mM NADPH, which plays an important role in the delivery of hydrogen, was added to start the reaction at 37°C in a final volume of 200 µL. The mixture was incubated at 37°C for 60 min. Reactions were terminated by cooling to −80°C immediately. Then 50 µL 0.1 M NaOH and 50 µL 500 µg/mL procaine were added to the incubation mixture, followed by the addition of 1 mL acetic ether. The mixture was vortexed for 2 min and centrifuged at 13,000 rpm at 4°C for 10 min, the organic phase was transferred into a clean tube and it was dried under a nitrogen stream. The resulting residue was dissolved in 100 µL mobile phase and 2 µL of the mixture was injected into the UPLC system for analysis. In addition, the experiments were carried out in triplicate and kept parallel, data presented as the mean ± SD. Additionally, in order to ensure that the metabolic reaction is within the linear range of the given incubation time and protein concentration, the incubation conditions have been optimized.

UPLC was performed on a Waters CORTECS UPLC HILIC column (2.1×50 mm, 1.6 µM). The column temperature was maintained at 30°C. The initial mobile phase consisted of 10 mM ammonium formate with 0.125% FA (solvent A) and acetonitrile (solvent B). The isocratic elution with 10% solvent A +90% solvent B was at a flow rate of 0.8 mL/min by injection volume 2 µL. The total run time of the analytes was 2.5 min. Under previously mentioned conditions, lidocaine, MEGX and procaine were well separated and their retention times were 0.519, 0.721 and 0.868 min, respectively. The column eluent was monitored with an ultraviolet detector at wavelength of 210 nm. A 7-point standard curve, which was prepared using spiked incubation samples, was used to quantify MEGX. In addition, testosterone and its metabolite 6β-hydroxy-testosterone were detected by using Agilent 1260 HPLC with an Agilent RRHD Eclipse Plus C18 column (3.0*100 mm, 1.8 µm, Agilent Technologies, Santa Clara, CA, USA). The mobile phases contained 0.05% trifluoroacetic acid in water (A) and 45% acetonitrile (B). We kept flow rate at 0.35 mL/min and carried out an isocratic elution with 55% solvent A and 45% solvent B for 5 min.

### Statistical analysis

Michaelis–Menten analysis (substrate vs velocity) was estimated using GraphPad Prism 5 (GraphPad Software Inc., San Diego, CA, USA) by non-linear regression curve fitting. And enzyme kinetic parameters (*Km* and *Vmax*) for each variant are presented as the mean ± SD of 3 parallel experiments. The intrinsic clearance (*CLint*) was determined as the ratio of *Vmax*/*Km*. Statistical analysis was carried out with the Statistical Package for the Social Sciences (version 17.0; SPSS Inc., Chicago, IL, USA), and 1-way analysis of variance was used for intergroup comparison together with the post hoc Dunnet test to analyze differences in catalytic activity between wild type and other variants. (**P*<0.05 represents statistically significant).

## Results

To obtain comprehensive data on the enzymatic properties for catalysis of MEGX by all reported *CYP3A4* variants in Chinese population, the wild-type *CYP3A4*1* and 22 allelic variants of *CYP3A4* were assessed in our study. The ratio of *Vmax*/*Km* represents the intrinsic clearance (C*Lint*), which can evaluate the enzymatic activity of each *CYP3A4* allelic variant as an evaluation criteria. Michaelis–Menten plots of lidocaine for the tested 23 *CYP3A4* enzymes are shown in [Figure 1](#f1-dddt-11-3503), and the corresponding kinetic parameters are summarized in [Table 2](#t2-dddt-11-3503).

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0646/5724423/2b70e5d67098/dddt-11-3503Fig1.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5724423_dddt-11-3503Fig1.jpg)

Michaelis–Menten curve of the enzymatic activity of the wild-type and 22 variants toward lidocaine. Notes: Each point represents the mean ± SD of 3 parallel experiments. The variants with designated allele names have been arranged into 6 groups (A–F). Abbreviation: MEGX, monoethylglycinexylidide.

### Table 2.

| Variants | Vmax (pmol/min/pmol P450) | Km (µM) | Intrinsic clearance (Vmax/Km) | Relative clearance (% of wild type) |
| --- | --- | --- | --- | --- |
| CYP3A4*1 | 7,885.00±245.95 | 471.80±62.71 | 16.98±1.63 | 100.00 |
| CYP3A4*2 (S222P) | 4,015.00±30.44** | 842.03±38.09** | 4.73±0.19** | 27.93** |
| CYP3A4*3 (M445T) | 11,350.00±153.08** | 639.57±24.81* | 18.04±0.49 | 106.61 |
| CYP3A4*4 (I118V) | 8,442.00±824.33 | 542.17±169.20 | 14.80±3.21 | 85.95 |
| CYP3A4*5 (P218R) | 6,436.00±265.43** | 1,103.67±124.03** | 5.60±0.41** | 32.99** |
| CYP3A4*9 (V170I) | 8,304.00±272.52 | 713.40±51.53** | 11.52±0.47** | 67.93** |
| CYP3A4*10 (D174H) | 10,730.00±523.55** | 554.43±157.91 | 19.09±3.96 | 109.93 |
| CYP3A4*11 (T363M) | 27,020.00±2,343.67** | 702.57±167.08 | 36.42±5.47** | 213.61** |
| CYP3A4*14 (L15P) | 10,120.00±317.96** | 406.63±27.76 | 25.65±1.00** | 151.58** |
| CYP3A4*15 (R162Q) | 10,730.00±683.04** | 491.07±41.54 | 20.87±0.47* | 123.27* |
| CYP3A4*16 (T185S) | 6,804.00±202.09** | 812.23±56.71** | 8.11±0.52** | 47.91** |
| CYP3A4*17 (F189S) | ND | ND | ND | ND |
| CYP3A4*18 (L293P) | 13,470.00±217.79** | 493.97±32.27 | 27.20±1.55** | 160.51** |
| CYP3A4*19 (P467S) | 9,887.00±325.53** | 466.17±99.43 | 22.40±4.01 | 130.35* |
| CYP3A4*23 (R162W) | 10,560.00±195.53** | 296.73±4.99** | 35.04±0.77** | 206.96** |
| CYP3A4*24 (Q200H) | 304.60±10.90** | 70.83±40.25** | 4.93±2.15** | 30.29** |
| CYP3A4*28 (L22V) | 5,224.00±58.29** | 290.60±25.19** | 17.87±1.73 | 105.29 |
| CYP3A4*29 (F113I) | 7,771.00±344.40 | 310.00±42.34* | 24.08±2.28* | 141.84* |
| CYP3A4*30 (R130STOP) | ND | ND | ND | ND |
| CYP3A4*31 (H324Q) | 7,949.00±99.96 | 237.30±20.46** | 33.58±2.89** | 197.87** |
| CYP3A4*32 (I335T) | 8,743.67±482.94** | 289.60±46.99** | 30.74±5.18** | 182.56** |
| CYP3A4*33 (A370S) | 4,865.00±130.48** | 328.23±24.17* | 14.43±0.70 | 85.16 |
| CYP3A4*34 (I427V) | 12,225.67±1,244.09** | 532.50±343.48** | 28.15±12.35** | 154.49** |

Table 2 Caption: Kinetic parameters for MEGX activities of wild-type and 22 CYP3A4 allelic variants against lidocaine

For the Michaelis–Menten kinetic analysis, the kinetic parameters of *CYP3A4*17* and *CYP3A4*30* could not be determined as no concentration of MEGX was detected, which indicated that *CYP3A4*17* and *CYP3A4*30* showed extremely low activity or no activity. Except for *CYP3A4*17* and *CYP3A4*30*, almost all allelic variants in [Table 2](#t2-dddt-11-3503) showed significant alteration in *Vmax* values compared with the wild-type *CYP3A4*1*: Only 4 variants *CYP3A4*4*, *CYP3A4*9*, *CYP3A4*29*, and *CYP3A4*31* exhibited no statistical difference; 6 variants (*CYP3A4*2*, *CYP3A4*5*, *CYP3A4*16*, *CYP3A4*24*, *CYP3A4*28*, and *CYP3A4*33*) showed mild decrease; and the remaining 10 variants displayed significantly increased *Vmax* values. Meanwhile, most variants showed obvious changes on *Km* values: 6 variants (*CYP3A4*2*, *CYP3A4*3*, *CYP3A4*5*, *CYP3A4*9*, *CYP3A4*16*, and *CYP3A4*34*) exhibited increased *Km* values with ~1.13–2.34-fold compared with *CYP3A4*1* and 7 variants (*CYP3A4*23*, *CYP3A4*24*, *CYP3A4*28*, *CYP3A4*29*, *CYP3A4*31*, *CYP3A4*32*, and *CYP3A4*33*) decreased to 15.01%–69.57%. As a result, according to intrinsic clearance (*Vmax*/*Km*) values of MEGX compared with the wild type, *CYP3A4* alleles could be classified into 3 categories: ten variants (*CYP3A4*11*, **14*, **15*, **18*, **19*, **23*, **29*, **31*, **32*, and **34*) exhibited significantly increased intrinsic clearance values (123.27%–213.61% relative clearance); 5 variants (*CYP3A4*2*, **5*, **9*, **16*, and **24*) showed lower intrinsic clearance values than *CYP3A4*1* (27.93%–67.93% relative clearance). Another 5 variants (*CYP3A4*3*, **4*, **10*, **28*, and **33*) were without statistical significance.

In addition, to investigate the enzymatic abilities of 23 *CYP3A4* variants between different substrate, lidocaine and 1 typical probing substrate for the CYP3A4 enzyme were used in the in vitro functional analysis. As shown in [Figure 2](#f2-dddt-11-3503), [2](#f2-dddt-11-3503) variants, *CYP3A4*17* and *CYP3A4*30*, exhibited extremely low enzymatic activity or null function for both substrates. And 9 variants (*CYP3A4*4*, *CYP3A4*5*, *CYP3A4*11*, *CYP3A4*15*, *CYP3A4*16*, *CYP3A4*17*, *CYP3A4*24*, *CYP3A4*30*, and *CYP3A4*31*) showed similar changed trend of enzymatic activities compared with *CYP3A4*1* toward lidocaine and testosterone. Whereas, the remaining *CYP3A4* variants, especially *CYP3A4*2*, *CYP3A4*14*, *CYP3A4*23*, *CYP3A4*29*, *CYP3A4*32*, and *CYP3A4*34*, displayed obvious different relative clearance between 2 substrates.

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0646/5724423/e23f89703a7c/dddt-11-3503Fig2.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5724423_dddt-11-3503Fig2.jpg)

The catalytic activity of expressed CYP3A4 variants toward typical probe substrates testosterone and lidocaine. Notes: Values were calculated as the percentages of the activities of wild-type protein. Data are presented as the mean ± SD of three independent experiments. *P<0.05; **P<0.01.

## Discussion

CYP3A4 exhibits remarkable individual variation in expression levels, which may be caused by genetic, environmental, pathological, hormonal, and dietary factors.[15](#b15-dddt-11-3503) Increasing evidence suggests that genetic polymorphism is a primary source of inter-individual difference in drug metabolism.[14](#b14-dddt-11-3503),[16](#b16-dddt-11-3503) Genetic polymorphism of *CYP3A4* can greatly influence the rate of elimination of lidocaine, thereby may result in adverse effects or therapeutic failures. Patients with decreased enzymatic activity are at high risk of undesirable adverse effects such as toxicity and delayed emergence from anesthesia. Therefore, it is of great significance to functionally characterize the enzymatic activity of *CYP3A4* variants in the metabolism of lidocaine in vitro.

To better understand the effects of *CYP3A4* variants on the metabolism of lidocaine in vitro, we analyzed the wild-type *CYP3A4*1* and 22 *CYP3A4* variants found in Chinese Han population in detail, of which 7 (*CYP3A4*28–*34*) were new alleles and another 15 alleles (*CYP3A4*2*, **3*, **4*, **5*, **9*, **10*, **11*, **14*, **15*, **16*, **17*, **18*, **19*, **23*, and **24*) had previously been reported. The enzymatic activity of *CYP3A4*1* was used as a control group. As a novel allelic variant, the *CYP3A4*30 (R130STOP)*, which contains the premature stop codon, produces a nonfunctional protein with a shorter peptide. As a consequence, the kinetic parameters of *CYP3A4*30* could not be detected and it showed no catalytic activity on lidocaine metabolism. Similar to *CYP3A4*30*, *CYP3A4*17* exhibited extremely low enzymatic activity toward lidocaine. According to previous reports, *CYP3A4*17* produced an amino acid substitution: F189S in exon 7, and it exhibited decreased catalytic activity compared with the wild type for both testosterone and chlorpyrifos.[17](#b17-dddt-11-3503),[18](#b18-dddt-11-3503)

Aside from *CYP3A4*17* and *CYP3A4*30*, we found that most of *CYP3A4* variants exhibited higher enzymatic activity than wild type, whereas only 5 alleles (*CYP3A4*2*, **5*, **9*, **16*, and **24*) showed a significantly decreased relative clearance compared with *CYP3A4*1* (27.93%–67.93%). The results indicate that these alleles could be classified as poor metabolizer alleles for lidocaine. Therefore, patients carrying the *CYP3A4*2*, **5*, **9*, **16*, **17*, **24*, or **30* should pay more attention to the dose of lidocaine, in order to avoid lidocaine-induced toxicity or delayed emergence from anesthesia.

Four (*CYP3A4*29*, **31*, **32*, and **34*) of novel alleles exhibited significantly increased intrinsic clearance value for lidocaine from 141.84% to 197.87% of wild-type *CYP3A4*1*. *CYP3A4*28* showed both lower *Vmax* and *Km* values so that the intrinsic clearance showed no significant difference to that of the *CYP3A4*1*. Hu et al found that *CYP3A4*28 (L22V)*, *CYP3A4*31 (H324Q)* and *CYP3A4*32 (I335T)* can damage protein function.[16](#b16-dddt-11-3503) However, as for our result that these alleles exhibited higher catalytic activity than wild type in our study, it may be interpreted that His324 and Ile335 are not involved in the active site for drug-substrate binding according to previously reported crystal structures of the CYP3A4 enzyme.[16](#b16-dddt-11-3503)

*CYP3A4*18*, contains a nucleotide substitution of T>C at position 878 of the cDNA of CYP3A4 and results in an amino acid change of Leu to Pro at codon 293, is commonly detected in Asians. According to previous study, *CYP3A4*18* exhibited increased turnover numbers for testosterone and chlorpyrifos in vitro.[17](#b17-dddt-11-3503) Similarly, in our study, *CYP3A4*18*, with a distinctly increased *Vmax*, showed significantly increased relative clearance compared with CYP3A4*1. These results are consistent with the previous study of testosterone and chlorpyrifos,[17](#b17-dddt-11-3503) 2 typical probing substrates for CYP3A4, thus indicating that our CYP expression and in vitro metabolism system were suitable for analyzing the catalytic activities of the *CYP3A4* allelic variants. In addition, *CYP3A4*23* was reported by Hu et al for the first time in the Chinese population. In our work, *CYP3A4*23* exhibited significantly increased intrinsic clearance value (206.96% of CYP3A4*1) owing to marked increase in *Vmax* and huge decrease in *Km*.

According to our previous studies, CYP2D6, CYP2C9 and CYP2C19 not only exhibit significant genetic polymorphisms between individuals, but also had substrate-specificity, which show marked differences on the metabolism of various clinical drugs.[19](#b19-dddt-11-3503)–[21](#b21-dddt-11-3503) Similarly, it has been reported that there is approximately a 10-fold variation in the metabolism of CYP3A4 substrates in vivo, which reveal that CYP3A4 may also exhibit a broad substrate specificity.[22](#b22-dddt-11-3503),[23](#b23-dddt-11-3503) However, thus far, few studies have been performed on *CYP3A4* genetic polymorphisms in the Chinese Han population and this is the first report of all these alleles for lidocaine metabolism. To better understand the enzymatic characteristics of these *CYP3A4* allelic isoforms, testosterone, as a probe substrate of CYP3A4, was also used to detect the catalytic activities of these 23 *CYP3A4* variants. Then we compared lidocaine with testosterone ([Figure 2](#f2-dddt-11-3503)), although some *CYP3A4* variants exhibited similar pattern for the metabolic activity toward testosterone, most of the variants, especially like *CYP3A4*2*, *CYP3A4*29*, *CYP3A4*32*, *CYP3A4*33*, and *CYP3A4*34* showed different relative activities for the probe drug. These results suggested that CYP3A4 also had substrate-specific alterations in metabolic activity and we infer that amino acid substitution in these sites might have different impacts for different substrate of CYP3A4. Moreover, genetic polymorphism and substrate-specificity of *CYP3A4* need further study on daily used drugs in order to help in personalized medicine for patients in a safe and reduced toxicity reaction.

## Conclusion

In summary, this study provides functional assessment of 23 *CYP3A4* variants on lidocaine metabolism in vitro for the first time. Our data revealed that most of these tested variants exhibited significantly increased catalytic activities toward lidocaine, which may offer a reference to the personalized medication for lidocaine in clinical practice. As the first study of all these alleles on lidocaine metabolism, the research may help to improve our understanding of the polymorphism of *CYP3A4*; on the other hand, this study could help clinical assessment of the lidocaine metabolism and aid the application of this knowledge in clinical practice. However, further studies are required to examine a wider range of *CYP3A4* substrates, including clinically used drugs.

## Acknowledgments

This work was supported by the Ministry of Health, National Health and Family Planning Commission of the People’s Republic of China. The authors thank the members of the Beijing Institute of Geriatrics of the Ministry of Health for their advice and assistance.

## Footnotes

## References

1. Turtle JD, Strain MM, Aceves M, et al. Pain input impairs recovery after spinal cord injury: treatment with lidocaine. J Neurotrauma. 2017;34(6):1200–1208. doi: 10.1089/neu.2016.4778.  [DOI](https://doi.org/10.1089/neu.2016.4778) | [PMC free article](/articles/PMC5359686/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27912032/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Neurotrauma&title=Pain%20input%20impairs%20recovery%20after%20spinal%20cord%20injury:%20treatment%20with%20lidocaine&author=JD%20Turtle&author=MM%20Strain&author=M%20Aceves&volume=34&issue=6&publication_year=2017&pages=1200-1208&pmid=27912032&doi=10.1089/neu.2016.4778&)

2. Grigoriev A, Nikitina A, Yaroshenko I, Sidorova A. Development of a HPLC-MS/MS method for the simultaneous determination of nifedipine and lidocaine in human plasma. J Pharm Biomed Anal. 2016;131:13–19. doi: 10.1016/j.jpba.2016.08.004.  [DOI](https://doi.org/10.1016/j.jpba.2016.08.004) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27521985/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharm%20Biomed%20Anal&title=Development%20of%20a%20HPLC-MS/MS%20method%20for%20the%20simultaneous%20determination%20of%20nifedipine%20and%20lidocaine%20in%20human%20plasma&author=A%20Grigoriev&author=A%20Nikitina&author=I%20Yaroshenko&author=A%20Sidorova&volume=131&publication_year=2016&pages=13-19&pmid=27521985&doi=10.1016/j.jpba.2016.08.004&)

3. Orlando R, Piccoli P, De Martin S, Padrini R, Floreani M, Palatini P. Cytochrome P450 1A2 is a major determinant of lidocaine metabolism in vivo: effects of liver function. Clin Pharmacol Ther. 2004;75(1):80–88. doi: 10.1016/j.clpt.2003.09.007.  [DOI](https://doi.org/10.1016/j.clpt.2003.09.007) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14749694/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Cytochrome%20P450%201A2%20is%20a%20major%20determinant%20of%20lidocaine%20metabolism%20in%20vivo:%20effects%20of%20liver%20function&author=R%20Orlando&author=P%20Piccoli&author=S%20De%20Martin&author=R%20Padrini&author=M%20Floreani&volume=75&issue=1&publication_year=2004&pages=80-88&pmid=14749694&doi=10.1016/j.clpt.2003.09.007&)

4. Suh JH, Oh CT, Im SI, Lim JS, Kim BJ, Lee JH. A multicenter, randomized, double-blind clinical study to evaluate the efficacy and safety of a new monophasic hyaluronic acid filler with lidocaine 0.3% in the correction of nasolabial fold. J Cosmet Dermatol. 2017;16(3):327–332. doi: 10.1111/jocd.12310.  [DOI](https://doi.org/10.1111/jocd.12310) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28145073/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Cosmet%20Dermatol&title=A%20multicenter,%20randomized,%20double-blind%20clinical%20study%20to%20evaluate%20the%20efficacy%20and%20safety%20of%20a%20new%20monophasic%20hyaluronic%20acid%20filler%20with%20lidocaine%200.3%%20in%20the%20correction%20of%20nasolabial%20fold&author=JH%20Suh&author=CT%20Oh&author=SI%20Im&author=JS%20Lim&author=BJ%20Kim&volume=16&issue=3&publication_year=2017&pages=327-332&pmid=28145073&doi=10.1111/jocd.12310&)

5. Moradi A, Shirazi A, Moradi-Poehler J, Turner J, Howell DJ. A blinded, randomized, split-face pilot study of bruising and pain with hyaluronic acid for correction of perioral lines using no lidocaine, lidocaine alone, and lidocaine and epinephrine. Aesthet Surg J. 2015;35(4):443–455. doi: 10.1093/asj/sjv043.  [DOI](https://doi.org/10.1093/asj/sjv043) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25908703/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Aesthet%20Surg%20J&title=A%20blinded,%20randomized,%20split-face%20pilot%20study%20of%20bruising%20and%20pain%20with%20hyaluronic%20acid%20for%20correction%20of%20perioral%20lines%20using%20no%20lidocaine,%20lidocaine%20alone,%20and%20lidocaine%20and%20epinephrine&author=A%20Moradi&author=A%20Shirazi&author=J%20Moradi-Poehler&author=J%20Turner&author=DJ%20Howell&volume=35&issue=4&publication_year=2015&pages=443-455&pmid=25908703&doi=10.1093/asj/sjv043&)

6. Hermansson J, Glaumann H, Karlen B, von Bahr C. Metabolism of lidocaine in human liver in vitro. Acta Pharmacol Toxicol (Copenh) 1980;47(1):49–52. doi: 10.1111/j.1600-0773.1980.tb02024.x.  [DOI](https://doi.org/10.1111/j.1600-0773.1980.tb02024.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7395524/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Acta%20Pharmacol%20Toxicol%20(Copenh)&title=Metabolism%20of%20lidocaine%20in%20human%20liver%20in%20vitro&author=J%20Hermansson&author=H%20Glaumann&author=B%20Karlen&author=C%20von%20Bahr&volume=47&issue=1&publication_year=1980&pages=49-52&pmid=7395524&doi=10.1111/j.1600-0773.1980.tb02024.x&)

7. Imaoka S, Enomoto K, Oda Y, et al. Lidocaine metabolism by human cytochrome P-450s purified from hepatic microsomes: comparison of those with rat hepatic cytochrome P-450s. J Pharmacol Exp Ther. 1990;255(3):1385–1391.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/2262908/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharmacol%20Exp%20Ther&title=Lidocaine%20metabolism%20by%20human%20cytochrome%20P-450s%20purified%20from%20hepatic%20microsomes:%20comparison%20of%20those%20with%20rat%20hepatic%20cytochrome%20P-450s&author=S%20Imaoka&author=K%20Enomoto&author=Y%20Oda&volume=255&issue=3&publication_year=1990&pages=1385-1391&pmid=2262908&)

8. Bargetzi MJ, Aoyama T, Gonzalez FJ, Meyer UA. Lidocaine metabolism in human liver microsomes by cytochrome P450IIIA4. Clin Pharmacol Ther. 1989;46(5):521–527. doi: 10.1038/clpt.1989.180.  [DOI](https://doi.org/10.1038/clpt.1989.180) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/2582709/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Lidocaine%20metabolism%20in%20human%20liver%20microsomes%20by%20cytochrome%20P450IIIA4&author=MJ%20Bargetzi&author=T%20Aoyama&author=FJ%20Gonzalez&author=UA%20Meyer&volume=46&issue=5&publication_year=1989&pages=521-527&pmid=2582709&doi=10.1038/clpt.1989.180&)

9. Watkins PB. Noninvasive tests of CYP3A enzymes. Pharmacogenetics. 1994;4(4):171–184. doi: 10.1097/00008571-199408000-00001.  [DOI](https://doi.org/10.1097/00008571-199408000-00001) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7987401/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=Noninvasive%20tests%20of%20CYP3A%20enzymes&author=PB%20Watkins&volume=4&issue=4&publication_year=1994&pages=171-184&pmid=7987401&doi=10.1097/00008571-199408000-00001&)

10. Zanger UM, Turpeinen M, Klein K, Schwab M. Functional pharmacogenetics/genomics of human cytochromes P450 involved in drug biotransformation. Anal Bioanal Chem. 2008;392(6):1093–1108. doi: 10.1007/s00216-008-2291-6.  [DOI](https://doi.org/10.1007/s00216-008-2291-6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18695978/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Anal%20Bioanal%20Chem&title=Functional%20pharmacogenetics/genomics%20of%20human%20cytochromes%20P450%20involved%20in%20drug%20biotransformation&author=UM%20Zanger&author=M%20Turpeinen&author=K%20Klein&author=M%20Schwab&volume=392&issue=6&publication_year=2008&pages=1093-1108&pmid=18695978&doi=10.1007/s00216-008-2291-6&)

11. Liu YT, Hao HP, Liu CX, Wang GJ, Xie HG. Drugs as CYP3A probes, inducers, and inhibitors. Drug Metab Rev. 2007;39(4):699–721. doi: 10.1080/03602530701690374.  [DOI](https://doi.org/10.1080/03602530701690374) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18058330/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Rev&title=Drugs%20as%20CYP3A%20probes,%20inducers,%20and%20inhibitors&author=YT%20Liu&author=HP%20Hao&author=CX%20Liu&author=GJ%20Wang&author=HG%20Xie&volume=39&issue=4&publication_year=2007&pages=699-721&pmid=18058330&doi=10.1080/03602530701690374&)

12. Ortiz de Montellano PR. The 1994 Bernard B. Brodie Award Lecture. Structure, mechanism, and inhibition of cytochrome P450. Drug Metab Dispos. 1995;23(11):1181–1187.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/8591716/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=The%201994%20Bernard%20B.%20Brodie%20Award%20Lecture.%20Structure,%20mechanism,%20and%20inhibition%20of%20cytochrome%20P450&author=PR%20Ortiz%20de%20Montellano&volume=23&issue=11&publication_year=1995&pages=1181-1187&pmid=8591716&)

13. Nicolas JM, Espie P, Molimard M. Gender and interindividual variability in pharmacokinetics. Drug Metab Rev. 2009;41(3):408–421. doi: 10.1080/10837450902891485.  [DOI](https://doi.org/10.1080/10837450902891485) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19601720/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Rev&title=Gender%20and%20interindividual%20variability%20in%20pharmacokinetics&author=JM%20Nicolas&author=P%20Espie&author=M%20Molimard&volume=41&issue=3&publication_year=2009&pages=408-421&pmid=19601720&doi=10.1080/10837450902891485&)

14. Hsieh KP, Lin YY, Cheng CL, et al. Novel mutations of CYP3A4 in Chinese. Drug Metab Dispos. 2001;29(3):268–273.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/11181494/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Novel%20mutations%20of%20CYP3A4%20in%20Chinese&author=KP%20Hsieh&author=YY%20Lin&author=CL%20Cheng&volume=29&issue=3&publication_year=2001&pages=268-273&pmid=11181494&)

15. Westlind A, Lofberg L, Tindberg N, Andersson TB, Ingelman-Sundberg M. Interindividual differences in hepatic expression of CYP3A4: relationship to genetic polymorphism in the 5′-upstream regulatory region. Biochem Biophys Res Commun. 1999;259(1):201–205. doi: 10.1006/bbrc.1999.0752.  [DOI](https://doi.org/10.1006/bbrc.1999.0752) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10334940/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem%20Biophys%20Res%20Commun&title=Interindividual%20differences%20in%20hepatic%20expression%20of%20CYP3A4:%20relationship%20to%20genetic%20polymorphism%20in%20the%205%E2%80%B2-upstream%20regulatory%20region&author=A%20Westlind&author=L%20Lofberg&author=N%20Tindberg&author=TB%20Andersson&author=M%20Ingelman-Sundberg&volume=259&issue=1&publication_year=1999&pages=201-205&pmid=10334940&doi=10.1006/bbrc.1999.0752&)

16. Hu GX, Dai DP, Wang H, et al. Systematic screening for CYP3A4 genetic polymorphisms in a Han Chinese population. Pharmacogenomics. 2017;18(4):369–379. doi: 10.2217/pgs-2016-0179.  [DOI](https://doi.org/10.2217/pgs-2016-0179) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28244811/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Systematic%20screening%20for%20CYP3A4%20genetic%20polymorphisms%20in%20a%20Han%20Chinese%20population&author=GX%20Hu&author=DP%20Dai&author=H%20Wang&volume=18&issue=4&publication_year=2017&pages=369-379&pmid=28244811&doi=10.2217/pgs-2016-0179&)

17. Dai D, Tang J, Rose R, et al. Identification of variants of CYP3A4 and characterization of their abilities to metabolize testosterone and chlorpyrifos. J Pharmacol Exp Ther. 2001;299(3):825–831.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/11714865/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharmacol%20Exp%20Ther&title=Identification%20of%20variants%20of%20CYP3A4%20and%20characterization%20of%20their%20abilities%20to%20metabolize%20testosterone%20and%20chlorpyrifos&author=D%20Dai&author=J%20Tang&author=R%20Rose&volume=299&issue=3&publication_year=2001&pages=825-831&pmid=11714865&)

18. Du J, Xing Q, Xu L, et al. Systematic screening for polymorphisms in the CYP3A4 gene in the Chinese population. Pharmacogenomics. 2006;7(6):831–841. doi: 10.2217/14622416.7.6.831.  [DOI](https://doi.org/10.2217/14622416.7.6.831) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16981844/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Systematic%20screening%20for%20polymorphisms%20in%20the%20CYP3A4%20gene%20in%20the%20Chinese%20population&author=J%20Du&author=Q%20Xing&author=L%20Xu&volume=7&issue=6&publication_year=2006&pages=831-841&pmid=16981844&doi=10.2217/14622416.7.6.831&)

19. Lan T, Yuan LJ, Hu XX, et al. Effects of CYP2C19 variants on methadone metabolism in vitro. Drug Test Anal. 2017;9(4):634–639. doi: 10.1002/dta.1997.  [DOI](https://doi.org/10.1002/dta.1997) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27199033/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Test%20Anal&title=Effects%20of%20CYP2C19%20variants%20on%20methadone%20metabolism%20in%20vitro&author=T%20Lan&author=LJ%20Yuan&author=XX%20Hu&volume=9&issue=4&publication_year=2017&pages=634-639&pmid=27199033&doi=10.1002/dta.1997&)

20. Hu X, Lan T, Dai D, et al. Evaluation of 24 CYP2D6 variants on the metabolism of nebivolol in vitro. Drug Metab Dispos. 2016;44(11):1828–1831. doi: 10.1124/dmd.116.071811.  [DOI](https://doi.org/10.1124/dmd.116.071811) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27538917/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Evaluation%20of%2024%20CYP2D6%20variants%20on%20the%20metabolism%20of%20nebivolol%20in%20vitro&author=X%20Hu&author=T%20Lan&author=D%20Dai&volume=44&issue=11&publication_year=2016&pages=1828-1831&pmid=27538917&doi=10.1124/dmd.116.071811&)

21. Ji Y, Chen S, Zhao L, et al. In vitro assessment of 39 CYP2C9 variants found in the Chinese population on the metabolism of the model substrate fluoxetine and a summary of their effects on other substrates. J Clin Pharm Ther. 2015;40(3):320–327. doi: 10.1111/jcpt.12267.  [DOI](https://doi.org/10.1111/jcpt.12267) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25884291/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Pharm%20Ther&title=In%20vitro%20assessment%20of%2039%20CYP2C9%20variants%20found%20in%20the%20Chinese%20population%20on%20the%20metabolism%20of%20the%20model%20substrate%20fluoxetine%20and%20a%20summary%20of%20their%20effects%20on%20other%20substrates&author=Y%20Ji&author=S%20Chen&author=L%20Zhao&volume=40&issue=3&publication_year=2015&pages=320-327&pmid=25884291&doi=10.1111/jcpt.12267&)

22. Thummel KE, Wilkinson GR. In vitro and in vivo drug interactions involving human CYP3A. Annu Rev Pharmacol Toxicol. 1998;38:389–430. doi: 10.1146/annurev.pharmtox.38.1.389.  [DOI](https://doi.org/10.1146/annurev.pharmtox.38.1.389) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9597161/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Annu%20Rev%20Pharmacol%20Toxicol&title=In%20vitro%20and%20in%20vivo%20drug%20interactions%20involving%20human%20CYP3A&author=KE%20Thummel&author=GR%20Wilkinson&volume=38&publication_year=1998&pages=389-430&pmid=9597161&doi=10.1146/annurev.pharmtox.38.1.389&)

23. Guengerich FP. Cytochrome P-450 3A4: regulation and role in drug metabolism. Annu Rev Pharmacol Toxicol. 1999;39:1–17. doi: 10.1146/annurev.pharmtox.39.1.1.  [DOI](https://doi.org/10.1146/annurev.pharmtox.39.1.1) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10331074/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Annu%20Rev%20Pharmacol%20Toxicol&title=Cytochrome%20P-450%203A4:%20regulation%20and%20role%20in%20drug%20metabolism&author=FP%20Guengerich&volume=39&publication_year=1999&pages=1-17&pmid=10331074&doi=10.1146/annurev.pharmtox.39.1.1&)

24. Cai J, Dai DP, Geng PW, et al. Effects of 22 Novel CYP2D6 Variants Found in the Chinese Population on the Bufuralol and Dextromethorphan Metabolisms In Vitro. Basic & clinical pharmacology & toxicology. 2016;118(3):190–199. doi: 10.1111/bcpt.12478.  [DOI](https://doi.org/10.1111/bcpt.12478) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26310775/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Basic%20&%20clinical%20pharmacology%20&%20toxicology&title=Effects%20of%2022%20Novel%20CYP2D6%20Variants%20Found%20in%20the%20Chinese%20Population%20on%20the%20Bufuralol%20and%20Dextromethorphan%20Metabolisms%20In%20Vitro&author=J%20Cai&author=DP%20Dai&author=PW%20Geng&volume=118&issue=3&publication_year=2016&pages=190-199&pmid=26310775&doi=10.1111/bcpt.12478&)
