# A Longitudinal HbA1c Model Elucidates Genes Linked to Disease Progression on Metformin

## Metadata
**Authors:** Srijib Goswami, Sook Wah Yee, Fei Xu, Sneha B Sridhar, Jonathan JD Mosley, Atsushi Takahashi, Michiaki Kubo, Shiro Maeda, Robert L Davis, Dan M Roden, Monique M Hedderson, Kathleen M Giacomini, Radojka M Savic
**Journal:** Clinical pharmacology and therapeutics
**Date:** 2016 Sep 23
**DOI:** [10.1002/cpt.428](https://doi.org/10.1002/cpt.428)
**PMID:** 27415606
**PMCID:** PMC5534241
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5534241/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC5534241/pdf/nihms803177.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC5534241/pdf/nihms803177.pdf)

## Abstract

One third of type-2 diabetic patients respond poorly to metformin. Despite extensive research, the impact of genetic and non-genetic factors on long-term outcome is unknown. In this study, we combine non-linear mixed effect modeling with computational genetic methodologies to identify predictors of long-term response. 1056 patients contributed their genetic, demographic and long-term HbA1c data. The top 9 variants (of 12,000 variants in 267 candidate genes) accounted for approximately 1/3 of the variability in the disease progression parameter. Average serum creatinine level, age and weight were determinants of symptomatic response, however explaining negligible variability. Two SNPs in CSMD1 gene (rs2617102, rs2954625) and one SNP in pharmacologically relevant SLC22A2 gene (rs316009) influenced disease progression, with minor alleles leading to less and more favorable outcomes respectively. Overall, our study highlights the influence of genetic factors on long-term HbA1c response and provides a computational model, which when validated, may be used to individualize treatment.

## RESULTS

### Summary of data

Baseline characteristics of patients with T2D are summarized in [Table 1](#T1). A total of 7822 HbA1c measurements from 1056 patients were used to develop a mathematical model of longitudinal HbA1c levels.

### Table 1.

| Clinical Site | N (%) |
| --- | --- |
| Total patients | 1056 |
| Kaiser South East | 154 (15%) |
| Marshfield Clinic | 150 (14%) |
| Vanderbilt | 251 (24%) |
| Kaiser Northern California | 501 (47%) |
| Categorical Variable | N (%) |
| Males | 415 (61%) |
| Females | 641 (39%) |
| European Americans1 | 376 (36%) |
| African Americans | 665 (63%) |
| Asian Americans and Others | 15 (1%) |
| Continuous Variable | Median (range) |
| Age (years) | 55 (23–90) |
| Body weight (kg) | 96 (34–212) |
| Average Serum Creatinine (mg/dL) | 0.91 (0.5–2.0) |
| Baseline HbA1c (%) | 7.6 (5.6–17.9) (60 mmol/mol (38–172 mmol/mol) |
| Metformin daily dose (mg) | 1000 (200–2500) |
| # HbA1c samples/patient | 5 (1–45) |
| Years on study | 1.43 (0.28–13.5) |

Table 1 Caption: Baseline characteristics of patients with Type II Diabetes

Of the 7822 total HbA1c measurements, 2928 HbA1c samples (37%) were collected after 2 years following metformin initiation across 344 patients (33%). 1220 HbA1c measurements (15.6%) were collected after 5 years following metformin initiation across 202 patients (19%). 555 HbA1c samples (7%) were collected after 7 years following metformin initiation across 123 patients (12%).

The dataset has a stronger representation of African Americans (63%) compared to European Americans (36%). The average length of time that each patient was under study was 2.78 years (median of 1.43 years, range of 0.28–13.5 years). Mean HbA1c samples provided per patient available for analysis was 7.5 (58 mmol/mol) (median of 5, range of 1–45). Of the 1056 patients, 1220 HbA1c measurements (15.6%) were available for 202 patients (19%) 5 years following metformin initiation. 123 patients (12%) 7 years after metformin initiation, and 28 patients (3%) 10 years after.

### Mathematical model development

A turnover HbA1c model with a reversible metformin effect on the synthesis rate of HbA1c best characterized the data. A reversible (symptomatic) metformin effect was implemented because it was assumed that the drug does not directly impact the disease progression and this structure was supported by the data^22^. The upward trajectory (disease progression) of HbA1c over time was modeled by implementing a separate compartment that represented the HbA1c synthesis rate: K_IN_(t). Model mechanics and the interplay of disease progression, HbA1c synthesis rate and %HbA1c level over time can be viewed in [Supplementary Figure 1](#SD1). In the model structure, K_IN_ was increasing due to disease progression, which is quantified by the disease progression parameter. The disease progression parameter generates a nonlinear increase of K_IN_ over time, especially when the estimate of disease progression is high. A time dependent increase in the HbA1c synthesis rate captured well the upward HbA1c trajectory observed in the data. In the model, between-subject variability (BSV) was estimated for baseline HbA1c, the magnitude of metformin’s effect (an individual’s specific HbA1c relative change from baseline), and disease progression. The inclusion of a full covariance block for all BSV parameters resulted in a significant improvement in the likelihood ratio. Final selection of the model was based on improvements in the objective function value and visual predictive checks of the longitudinal HbA1c data. Through simulations, the “onset’ of disease progression, which is defined by the time point at which HbA1c levels start to increase (i.e. an upward slope in HbA1c levels), was investigated. The model predicted that the onset of disease progression for a typical patient on metformin is approximately 321 days; at which point, HbA1c levels increased at a rate of 0.1% (1.1 mmol/mol) [0.07%–0.13%] per year through the first three years ([Table 2](#T2)). For patients not on metformin, the model predicts that HbA1c levels would increase at a steady state rate of approximately 0.16% (1.7 mmol/mol) [0.08%–0.22%] per year. Mathematical model parameters along with clinically derived parameters are summarized in [Table 2](#T2).

### Table 2.

| Final Model Parameter | Median (%RSE)1 | Median (90% CI)2 |
| --- | --- | --- |
| Baseline HbA1c Level (%) | 7.74 (1) | 7.73 (7.6–7.8) |
| Half Life of Effect (days) | 40.9 (6) | 41.2 (36.8–45.7) |
| Metformin Effect Magnitude EFF | 13.1% (5) | 13.0 (12.1–14.4) |
| Disease Progression Estimate DISPR3 (all patients) | 82.2 (67) | 75.3 (32.6–249) |
| Boxcox transformation parameter on Baseline | 2.38 (9) | 2.41 (1.99–2.78) |
| Boxcox transformation parameter on DISPR | −0.246 (15) | −0.26 (−0.31–−0.20) |
| KLOSS | 0.205 (86%) | 0.266 (0.05–0.657) |
| Between-subject variability (% variance) |  |  |
| Between-subject variability (Baseline) | 16.9 (3) | 16.6 (15.9–17.8) |
| Between-subject variability (Metformin Effect Magnitude METFEFF) | 76.4 (4) | 75.9 (71.7–81.6) |
| Between-subject variability (Disease Progression DISPR) | 324 (17) | 390 (164–418) |
| Covariance of parameters (%) |  |  |
| Correlation Baseline-METFEFF | 0.114 (1) | 0.11 (0.101–0.136) |
| Correlation Baseline-DISPR | 0.033 (3.6) | 0.03 (−0.07–0.14) |
| Correlation DISPR-METFEFF | 0.204 (21) | 0.31 (−0.42–0.95) |
| Residual error model |  |  |
| Proportional error (%) | 0.098 (3) | 0.098 (0.092–0.101) |
| Additive error | 0.1 (FIXED) | 0.1 (NA) |
| Derived Clinical Parameters | Simulated Median (90% CI) |  |
| Estimated onset of disease progression4 | 321 (309–332) days |  |
| Estimated yearly rate of HbA1c increase on Metformin4 | 0.1 %HbA1c (0.07–0.13) |  |
| Estimated yearly rate of HbA1c increase not on Metformin4 | 0.16 %HbA1c (0.08–0.22) |  |

Table 2 Caption: Population pharmacodynamic model derived estimates and bootstrap results for model parameters

### Final demographic/clinical covariate model

As determined by model diagnostics, the demographic-corrected mathematical model adequately described the data ([Figure 2](#F2)). As expected, average serum creatinine level (a likely surrogate for metformin drug exposure) was a significant predictor on Metf_EFFECT_, with higher levels leading to improved HbA1c response. Through simulations, a typical patient with a 0.6 mg/dL creatinine level is expected to result in a 0.77% (8.4 mmol/mol) HbA1c improvement from baseline (at 2 years), whereas a patient with a 1.3 mg/dL creatinine level is expected to result in a 0.96% improvement in HbA1c (10.5 mmol/mol) from baseline. This response characteristic is anticipated as pharmacologically, average exposure of metformin is expected to increase by approximately by 20% with a 0.7 mg/dL increase (from 0.6 to 1.3) in serum creatinine level for males and females of age 50.

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7232/5534241/328f6087b49c/nihms803177f2.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5534241_nihms803177f2.jpg)

Longitudinal HbA1c levels over time and model based visual predictive check. The plot to the left shows raw HbA1c observations over time. On the right plot, a visual predictive check is shown, where the solid black line highlights the median observed profiles. The shaded regions indicate the 95th and 5th percentiles (ends) and the range of median simulated profiles (center) of simulated predictions from the visual predictive check.

Additionally, body weight and clinical site were significant covariates on the Metf_EFFECT_ model parameter. Body weight was inversely related to metformin effect, estimated to result in a 6% decrease in metformin’s effect parameter per 10-kilogram increase in body weight. Clinically, this would result in a 0.99% and 0.80% change in HbA1c (equivalent of 10.8 and 8.7 mmol/mol) from baseline (at 2 years) for patients with body weights of 66 kg and 140 kg (5^th^ and 95^th^ percentile) respectively. For clinical site variable, Vanderbilt and Kaiser Georgia had a 16% and 30% lower estimate on the metformin effect parameter when compared to Kaiser Northern California, respectively.

Age was also a significant covariate on the disease progression model parameter, with a negative correlation observed between age and disease progression. Clinically, this would result in a relative change in HbA1c (at 2 years) to fall between 0.76% and 0.84% improvement for patients between the ages of 49 and 64 years.

### Genetic analysis: hyperlasso methodology on model parameters

A total of 267 genes were selected and approximately 12000 variants within a 50-kilobase region around each gene were extracted for analysis. Of the variants investigated, a total of 16 SNPs were linked to the disease progression parameter by hyperLASSO analysis (with a MAF ≥ 5%). Of the remaining 16 variants, 11 were intronic [*CSMD1*(4), *ADCY5*(1), *PRKAG*(1), *SLC22A2*(1), *EMILIN2*(1), *SULF1*(1), *FTO*(1), *WWOX*(1)], 1 was missense [*SREBF1*], and 4 were located within 50 kilobases upstream or downstream of each gene [*VPS13C*(1), *KCNK16*(1), *PPARG*(1), *FOXN3*(1)].

### Genetic analysis: model-based approach for variant selection

Of the prioritized 16 variants from hyperlasso, a model-based methodology was implemented to verify statistical significance and determine effect sizes on the disease progression parameter. SNPs that passed this test were included in the final mathematical model for simulation purposes. From this step, 7 SNPs were removed due to the defined criteria (see Methods). The 9 remaining variants were statistically significant in the model structure and collectively accounted for approximately one-third of the variability in the predicted disease progression model parameter (reduced the BSV of the disease progression model parameter from 324% to 225%). Of the 9 variants, rs12907856 (*VPS13C*), rs2954625 (*CSMD1*), and rs3160009 (*SLC22A2*) individually accounted for approximately 6%, 5%, and 8% of the variability, respectively. The characteristics of each SNP are shown in [Table 3](#T3).

### Table 3.

| SNP | Chr | Gene | Minor Allele | Major Allele | Feature | Model-based P-Value | Effect size of minor allele on DP parameter | MAF CEU | MAF YRI | Gene Functions (References cited in this table are in the footnote) |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| rs12907856 | 15 | VPS13C - vacuolar protein sorting-associated 13 gene family | G | A | Proximal | 0.0003 | −0.147 (GA) | 0.30 | 0.30 | This gene encodes a member of a vacuolar protein. SNP in this gene was associated with glucose-stimulated insulin response1–3. |
| rs2815022 | 6 | KCNK16 - Potassium Channel, Two Pore Domain Subfamily K, Member 16 | G | A | Proximal | 0.009 | 0.39 (GA) | 0.48 | 0.26 | This gene encodes the TALK-1 channel, the most abundant potassium channel in human beta-cells and it modulates beta-cells electrical excitability, second-phase insulin secretion and glucose homeostasis4, 5 |
| rs2617102 | 8 | CSMD1 - CUB And Sushi Multiple Domains 1 | C | A | Intron | 0.02 | 0.717 (CA) | 0.19 | 0.20 | This gene encodes an integral membrane protein with unknown molecular function. SNP in this gene was associated with congenital hyperinsulinism of infancy6. |
| rs2954625 | 8 | T | C | Intron | 0.029 | 0.23 (TC) | 0.21 | 0.46 |  |  |
| rs316009 | 6 | SLC22A2 - Solute Carrier Family 22 (Organic Cation Transporter), Member 2 | T | C | Intron | 0.04 | −0.44 (TC) | 0.10 | 0.08 | This is a transporter in the kidney that secretes metformin into the urine. This SNP is in linkage disequilibrium to a non-synonymous variant, A270S (rs316019), which was associated with metformin disposition7, 8. |
| rs642887 | 18 | EMILIN2 - Elastin Microfibril Interfacer 2 | A | G | Intron | 0.05 | −0.42 (AG) | 0.14 | 0.21 | An extracellular matrix glycoprotein associated with thrombosis. EMILIN2 is involved in regulating platelet activation important for cardiovascular development9, 10, whereas EMILIN1 may be involved in regeneration of islets, which could play a role in blood glucose lowering11. |
| rs6982250 | 8 | SULF1 - Sulfatase 1 | T | C | Intron | 0.0023 | −0.37 (TC) | 0.17 | 0.28 | This gene encodes an enzyme, which is involved in modulating growth factor signaling. Data from sulfatase knockout mice showed that it plays a role in diabetic nephropathy12. |
| rs7159552 | 14 | FOXN3 - Forkhead Box N3 | T | G | Proximal | 0.009 | −0.25 (TG) | 0.27 | 0.35 | This is a transcriptional repressor, which plays an important role in cell cycle arrest. This gene is localized in chromosome 14q24.3-q31, which is a locus associated with insulin-dependent diabetes mellitus susceptibility13. |
| rs7500549 | 16 | WWOX - WW Domain Containing Oxidoreductase | C | T | Intron | 0.029 | −0.16 (CT) | 0.55 | 0.20 | SNP in this locus was associated with reduced insulin secretion14. |

Table 3 Caption: Summary of Top Genetic Variants Included in Final Population Pharmacodynamic Model of Metformin.

In the final model, several simulations were performed to illustrate the potential clinical impact of each SNP on long-term HbA1c levels. [Figure 3](#F3) quantitatively summarizes the predicted effects of final model genetic and non-genetic factors on HbA1c levels at the 1-year and 5-year mark. Hypothetical gene/gene interactions were also explored and the combinatorial effects of high risk SNPs in the *CSMD1, WWOX*, and *SLC22A2* genes were also explored in [Figure 4](#F4).

### Figure 3.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7232/5534241/1507c6fba6cc/nihms803177f3.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5534241_nihms803177f3.jpg)

Top genetic and demographic covariates on long term HbA1c levels. A. The effect of covariates on the simulated median (bands show 5th and 95th CI of simulated median) of HbA1c levels at the 1-year mark. B. The effect of covariates on the simulated median (5th and 95th CI of simulated median) of HbA1c levels at the 5-year mark. A normal individual here represents a hypothetical patient with no minor alleles of any of the identified variants with median age, body weight, and serum creatinine values.

### Figure 4.

![Figure 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7232/5534241/37ce5cb76810/nihms803177f4.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5534241_nihms803177f4.jpg)

Effect of SNP combinations in CSMD1, SLC22A2, and WWOX on the dynamics of HbA1c levels. A: Simulated median HbA1c levels (with 95% CI bands) over 5 years comparing carriers and non-carriers of CSMD1 minor (risk) alleles. B: Simulated median HbA1c levels over 5 years comparing carriers and non-carriers of SLC22A1/WWOX genes minor alleles. Blue shaded region with solid line: Simulated median for patients carrying no minor alleles with 5th and 95th confidence interval. Red/green shade with dashed line: Simulated median for patients carrying minor alleles of labeled gene(s) with 5th and 95th confidence interval of median.

In the exploratory studies, patients carrying one or more minor alleles of the identified variants in the *CSMD1* gene (rs2617102 (C), rs2954625 (T)) were predicted to have significantly higher long-term HbA1c levels compared to patients not carrying any *CSMD1* minor alleles or patients with homozygous rs3160009 TT (*SLC22A2)* and/or homozygous rs7500549 CC (*WWOX*) genotypes.

### Functional annotation of top variants

Three out of the nine variants (rs12907856, rs316009, and rs7159552) are located in linkage disequilibrium (LD) to a regulatory region, determined by an algorithmic prediction by RegulomeDB^23^. In particular, rs316009 and rs7159552 are located in a transcription factor binding motif as identified by the ENCODE project^24^. The rs316009 variant is in LD to the nonsynonymous variant of SLC22A2 - rs316019 - which is known to play a role in metformin pharmacokinetics^,25,26,27^. Another variant, rs6982250, is in an intronic region of *SULF1*. Several SNPs in SULF1 have been associated with many phenotypes^28^ with one such variant associated with fasting insulin-related traits^29^.

## DISCUSSION

Previous pharmacogenetic studies of metformin response have focused on the effect of selected variants in relevant pharmacogenes on single-time point outcomes of metformin (i.e. HbA1c levels after 90 days, FPG levels, etc.)^15,30,31,32^. Long-term, time-dependent changes of HbA1c have been previously overlooked, resulting in a collapse of valuable biomarker information that may inform disease progression as well as temporal response patterns.

Here, we developed a longitudinal HbA1c model by leveraging a large T2D dataset and subsequently investigated the role of genetic and non-genetic factors on long-term dynamics of HbA1c following metformin initiation. Special focus was given to identifying factors that are responsible for the long-term variance in HbA1c levels.

Three important findings emerged from this analysis: (i) a mathematical model incorporating disease progression and a reversible metformin effect best characterized the long-term HbA1c data in T2D Patients. (ii) The model presented herein predicted that the onset of disease progression for patients on metformin is approximately 321 days, at which point, levels increase, on average, at a rate of 0.1% (1.1 mmol/mol) [0.04%–0.16%] HbA1c per year; HbA1c levels are expected to increase at a steady state rate of approximately 0.16% (1.76 mmol/mol) [0.08%–0.22%] per year in patients not treated with metformin. (iii) Nine variants in 8 genes (of 267 genes interrogated) accounted for approximately one-third of the total estimated variability in the disease progression model parameter. Variants in three of these genes (*CSMD1*, *WWOX*, and *SLC22A2*) were identified as significant influencers of disease progression on metformin therapy.

The development of the final mathematical model resulted from the exploration of several approaches with various empirical and semi-mechanistic considerations. The structural parameters from the model were estimated with high precision. The between-subject variability estimates of baseline, metformin effect, and disease progression were also estimated with relatively high precision (3%, 4%, and 17% relative standard error (RSE), respectively). The high degree of parameter confidence was due to the abundance of available HbA1c data, allowing for the reliable assessment of clinical, demographic, and genetic covariates on disease progression. Disease progression (upward trajectory of HbA1c levels) is a function of both the patient’s underlying disease as well as the build up of metformin resistance. In order to differentiate between the effects of a patient’s biology and a reduction of metformin’s reversible effect, it is necessary to model longitudinal HbA1c data prior to the administration of treatment; unfortunately, this was not possible in our analysis as this would require patients to be off treatment during the duration of the disease. The HbA1c model was however able to adequately predict the dynamics of HbA1c levels, capturing the long-term upward trend observed in this population. The ability to predict long-term HbA1c changes is especially valuable: the onset of disease progression and the rate of HbA1c increase were quantified for patients on metformin therapy (~0.1% increase per year for the first three years after 321 days, which is the estimated onset of disease progression) leveraging the richness of HbA1c data available. This finding was particularly interesting in relation to the study by Winter et al. where the authors noted a slight rise in patients’ HbA1c levels, between 200 and 400 days after metformin initiation; however, they were unable to quantify this upward trend through their simulations – a limitation which resulted from the lack of longitudinal data points available after 400 days^16^. In our analysis, the average length of time in the study was 1014 days and up to 10 years worth of HbA1c measurements were available to inform disease progression – allowing the characterization and quantification of this upward trend with high precision. The robustness in the model enabled the simulation of patient-specific disease progression with an underlying assumption of no metformin administration (approximate increase of 0.16% (1.7 mmol/mol) in HbA1c per year). The ability to separate disease progression and metformin effect is based on early HbA1c data (up to 1 year following metformin initiation). Simulations of disease progression assuming no metformin administration were explored by removing metformin’s estimated effect on the HbA1c synthesis rate within the model structure. The simulations demonstrate that on average, disease progression in patients who are metformin-naive will occur faster than in patients taking metformin for several months. Comparing this estimate to existing literature is problematic since T2D progression is a gradual process that typically takes place over several years and thus allows only a small trajectory of change within the limited time frame available for most studies. In the few studies reported, the rate of HbA1c increase was estimated to be approximately 0.2% (2.2 mmol/mol) per year, a value consistent with our observations^33^.

A stepwise multivariate analysis was performed to identify statistically significant demographic and clinical covariates on model parameters. Average serum creatinine level surfaced as a significant factor that influenced the magnitude of metformin’s effect. This finding was expected since serum creatinine is considered a likely surrogate for metformin exposure. Serum creatinine directly influences a patient’s creatinine clearance, which ultimately influences a patient’s systemic exposure to metformin by affecting the apparent clearance pharmacokinetic parameter. The effect of age was also noted - an inverse relationship was observed between age and the magnitude of disease progression. It is important to note that although age was statistically significant through a stepwise analysis, the effect size was quite small and a reproduction of these results is required to inspire greater conviction of this correlation. Previously, in a study by Williams et al., lower HbA1c levels were reported in African Americans compared to European American individuals^34^. In our analysis, however, there was no significant effect of self-reported ethnicity on any of the model parameters, including disease progression.

We used multiple genetic methods to prioritize influential variants on disease progression. Hyperlasso methodology was selected over a stepwise procedure, as well as several other algorithms. This is because the hyperlasso approach has been shown to be robust when investigated covariates are correlated, which is the case here with strong LD patterns in the genotype data. The final selection of variants was based on the performance of individual variants within the demographic-corrected model so that the correlation across various model parameters may also be taken into consideration.

Nine variants emerged were linked to the progression of HbA1c levels on metformin. Collectively, the variants accounted for approximately one-third of the variance in the disease progression model parameter. It was also observed that these genetic variants had larger effects on HbA1c levels than the demographic and clinical covariates identified from the stepwise analysis.

Of the top genes, minor alleles of two SNPs (rs2617102, rs2954625) in the *CSMD1* (CUB and Sushi multiple domains 1) gene had the strongest impact on disease progression. Although the pharmacological and biological mechanism remains unclear, *CSMD1* has been previously linked to insulin sensitivity and lipid levels^35,36^. *CSMD1* variants may have a significant impact on longitudinal HbA1c levels, especially at the five-year mark when the simulated HbA1c improvement from baseline becomes nominal – especially for homozygous carriers (TT) of rs2617102. The simulated 5-year HbA1c level was very similar to baseline levels ([Figure 4](#F4)) – which means that HbA1c levels rebounded back to its baseline state. Furthermore, the effect on HbA1c levels at the 5-year mark was higher for hypothetical homozygous carriers of both *CSMD1* SNPs (rs2617102, rs2954625) – where HbA1c levels were predicted to be significantly higher than baseline levels.

Minor alleles of SNPs in genes *SLC22A2*, *WWOX, EMILIN2,* and *FOXN3* were associated with more favorable trajectories (lower disease progression) of HbA1c levels compared to major allele carriers. Of these genes, *SLC22A2 (*rs316009 (T)) and *WWOX* (rs7500549 (C)) showed the strongest effect. In contrast to homozygous carriers of *CSMD1* risk alleles, homozygous carriers of both *SLC22A2* and *WWOX* SNPs were predicted to have a favorable clinical outcome - maintaining their peak HbA1c level improvement from baseline through 5 years of metformin therapy. The rs316009 variant is in LD to a nonsynonymous variant of *SLC22A2* (rs316019), a SNP that has been previously shown to alter transporter function as well as modulate metformin pharmacokinetics.^26,27^ Therefore, the clinical expectation that the reduced function rs316009 (T) allele would lead to a more favorable outcome is pharmacologically sound. OCT2 (*SLC22A2*) is predominantly expressed at the basolateral membrane in distal renal tubules and is responsible for the uptake of metformin from circulation into renal epithelial cells, working in concert with other renal transporters to excrete metformin. Though functional studies have been controversial^25^, loss of transporter function is expected to increase plasma levels of metformin, potentially leading to a more favorable pharmacodynamic outcome with relatively low HbA1c levels.

Also of clinical interest, the gene *WWOX* has been previously associated with several T2D traits including body weight, C-reactive protein, insulin, obesity, and lipid levels^37^. *WWOX* encodes for an enzyme that is found in all eukaryotes and has been biologically shown to play an important role in the regulation of a wide variety of cellular functions such as protein degradation, transcription, and RNA splicing. Unlike *SLC22A2,* a pharmacological mechanism for *WWOX* is not clear. However, the clinical impact (if replicated) would mean that carriers of the rs7500549 (C) allele would respond favorably to metformin therapy. Future studies should focus on elucidating the biology of *WWOX* and replicating the genetic findings on disease progression.

Although this computational approach represents a novel way to uncover factors that influence long-term drug response, several important limitations must be highlighted. First, the demographic distribution used for this analysis does not appropriately reflect the national population distribution due to the disproportionate representation of African Americans in this cohort. As a result, it will be critical to replicate both genetic and non-genetic findings in separate cohorts for validation purposes. Furthermore, the retrospective data set lacks a control group and is reflecting multiple studies across multiple sites. As such, validation of the model-based simulations, which quantify metformin’s effect on long term HbA1c dynamics with consideration of impactful covariates, is required.

Overall, our study has successfully integrated robust model-based approaches with genetic analyses methods to uncover genes linked to the progression of HbA1c on metformin therapy in a large T2D cohort. If replicated, these genetic findings may have a significant influence on T2D treatment strategy. Ultimately, the long-term goal of this research is to translate this computational model into clinical practice and enable clinicians to provide data-driven, personalized treatment advice to T2D patients based on individual patient characteristics.

## METHODS

### Patients with type 2 diabetes

Diabetic patients of European American, African American, and Asian American ancestry were recruited into a multicenter retrospective study as described previously^13,32^. All patients were metformin-naive, had HbA1c levels measured before and after initiation of metformin therapy (between 3 and 18 months), and had a medication possession ratio greater than 80%. The institutional review boards (IRBs) of Marshfield Clinic Research Foundation, Kaiser Permanente Northern California, Kaiser Permanente South East, Georgia, approved this study and informed consent was obtained. At Vanderbilt, an opt-out consent model was used. In diabetic patients, metformin was administered for at least three months, so steady state drug concentration levels were achieved, since the half-life of metformin is roughly 5 hours. Patients were in the study for an average of 2.8 years (median = 1.43 years) with on average 7.4 (median = 5) HbA1c measurements. HbA1c results were reported in the NGSP format (National Glycohemoglobin standardization program). The median metformin dose across the patient population was 1000 mg ([Table 1](#T1)). Patients were genotyped using an Illumina OmniExpress genotype array (see [supplementary methods](#SD1) section for further details).

### Development of mathematical model

Patient data were analyzed using non-linear mixed effect modeling (NONMEM 7) with first order conditional estimation method with interaction (FOCE-I). Several semi-mechanistic approaches were explored to best describe the longitudinal HbA1c versus time profiles. Model selection was determined using the objective function value (OFV, −2 times the log of the likelihood) and visual inspection of diagnostic plots. The selected longitudinal HbA1c profiles were described by the following equations:

| d(KIN)dt=KSYN∗(1+DisprEFFECT)-KLOSS∗KIN | (1) |
| --- | --- |
| d(HbA1c)dt=KIN∗(1-MetfEFFECT)-KOUT∗HbA1c | (2) |
| --- | --- |
[Equation 1](#FD1) defines the synthesis rate of HbA1c, which includes a non-linear time sensitive parameter dependent on the baseline synthesis rate and the extent of patient-specific disease progression. In [Equation 1](#FD1), Dispr_EFFECT_, K_LOSS_ and K_SYN_ represent the disease progression effect parameter, loss rate of K_IN_ and synthesis rate of the K_IN_ parameter, respectively. [Equation 2](#FD2) defines the dynamics of HbA1c, parameterized by the synthesis rate of HbA1c (K_IN_(t)), metformin’s effect from baseline (Metf_EFFECT_), and the loss rate of HbA1c (K_OUT_). A more detailed explanation and the model source code can be found in the [supplementary methods](#SD1) section. A simulated demonstration of the dynamics of this model can viewed in [supplementary figure 1](#SD1). A patient’s individual administered doses were taken into consideration by examining the effect of the average daily dose. The average daily dose of metformin was calculated from metformin start day up to the day, where minimum HbA1c levels were achieved between 3–18 months (and before other anti-diabetic drug or insulin was added). Although no drug concentration was directly used for this analysis, surrogate PK information was taken into account in the model structure by investigating the effect of average serum creatinine level (a major predictor on metformin individual clearance) or imputed exposure (based on estimated individual clearance of metformin and average daily dose). Individual clearance was estimated based on the clearance equation previously described^10^. Both average dose and metformin exposure were tested on the Metf_EFFECT_ parameter.

### Demographic analysis

Using the mathematical model described above, agnostic stepwise forward selection (P < 0.05) and backward elimination (P < 0.01) were applied to identify statistically significant demographic and clinical covariates on model parameter estimates, which helped guide the selection of the demographic-corrected final model. The effect of concomitant medications was taken into account by investigating the effect of added drug on model parameters. The subsequent demographic-corrected mathematical model served as a basis to investigate the effect of genetic variants on the variance of long-term response.

### Genetic analyses of model parameters

A comprehensive list of candidate genes was selected using the GWAS Integrator tool on the HuGE Navigator^38^ (details found in [supplementary methods](#SD1) section). A penalized regression-based approach called hyperlasso was implemented to statistically prioritize the variants associated with phenotypes outputted from the mathematical model (e.g. disease progression, metformin effect, and baseline). This methodology was originally proposed by Hoggart et al., and is a generalization of Lasso^39,40^. Further information about the hyperlasso method can be found in [supplementary methods](#SD1) section.

### Model based genetic analysis of identified variants

The top SNPs from hyperlasso were subsequently investigated in the developed demographic-corrected mathematical model described above. Model based analyses are advantageous because they account for correlations across various model parameters as well as potential SNP/SNP interactions. Two key steps were taken to select the final mathematical model; (1) removal of non-significant SNPs, which resulted from a univariate analysis of each variant in the demographic-adjusted mathematical model, and (2) removal of variants from the full genetic model that had very low, clinically irrelevant effect sizes. Details of this step may be found in the [supplemental](#SD1) section.

## Supplementary Material

### What is the current knowledge on the topic?

### What is the current knowledge on the topic?

### What is the current knowledge on the topic?

### What is the current knowledge on the topic?

Previous studies have focused on the effect of genetic polymorphisms in candidate genes on short-term changes in metformin response. Additionally, studies have developed computational models to capture short-term pharmacodynamic changes without consideration of long-term disease progression.

### What question did this study address?

In this study, we combined quantitative pharmacology with computational genetic analysis techniques to investigate the effect of genetic variants in biologically and pharmacologically meaningful genes on long-term disease progression of patients with type 2 diabetes on metformin therapy.

### What this study adds to our knowledge

This study provides evidence that genetic polymorphisms in CSMD1 and membrane transporter gene SLC22A2 are significant influencers of disease progression, affecting the long-term trajectory of HbA1c levels. This study also adds a robust quantitative pharmacology model that predicts long-term changes in HbA1c levels, which if validated, may be used as a valuable tool to predict long-term outcomes for patients.

### How this might change clinical pharmacology or translational science

To date, this is the first study to explore the effect of biologically and pharmacologically relevant genes on long-term disease progression of patients taking metformin. This is also the first study to investigate long-term HbA1c disease trajectories. In the future, combining genotyping of biologically and pharmacologically relevant genes with proper consideration of demographic and clinical predictors may be used to inform metformin therapy in T2D patients.

## Acknowledgments

This research was supported by grants from the National Institutes of Health (NIH) National Institute of General Medical Sciences (GM61390 and GM117163), NIH P30 DK063720, and the NIH Pharmacogenomics Research Network (PGRN)-RIKEN Strategic Alliance.

## Footnotes

## Contributor Information

Srijib Goswami, University of California, San Francisco.

Sook Wah Yee, University of California, San Francisco.

Fei Xu, Kaiser Permanente Northern California.

Sneha B. Sridhar, Kaiser Permanente Northern California

Jonathan JD Mosley, Vanderbilt University.

Atsushi Takahashi, RIKEN Institute, Center for Genomic Medicine.

Michiaki Kubo, RIKEN Institute, Center for Genomic Medicine.

Shiro Maeda, RIKEN Institute, Center for Genomic Medicine.

Robert L. Davis, Kaiser Permanente Georgia

Dan M. Roden, Vanderbilt University

Monique M. Hedderson, Kaiser Permanente Northern California

Kathleen M. Giacomini, University of California, San Francisco

Radojka M. Savic, University of California, San Francisco

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Scarpello JH, Howlett HC. Metformin therapy and clinical uses. Diab Vasc Dis Res. 2008;5:157–67. doi: 10.3132/dvdr.2008.027.  [DOI](https://doi.org/10.3132/dvdr.2008.027) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18777488/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Diab%20Vasc%20Dis%20Res&title=Metformin%20therapy%20and%20clinical%20uses&author=JH%20Scarpello&author=HC%20Howlett&volume=5&publication_year=2008&pages=157-67&pmid=18777488&doi=10.3132/dvdr.2008.027&)

2. Gong L, Goswami S, Giacomini KM, Altman RB, Klein T. Metformin pathways: pharmacokinetics and pharmacodynamics. Pharmacogenet Genomics. 2013;22:820–27. doi: 10.1097/FPC.0b013e3283559b22.  [DOI](https://doi.org/10.1097/FPC.0b013e3283559b22) | [PMC free article](/articles/PMC3651676/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22722338/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Metformin%20pathways:%20pharmacokinetics%20and%20pharmacodynamics&author=L%20Gong&author=S%20Goswami&author=KM%20Giacomini&author=RB%20Altman&author=T%20Klein&volume=22&publication_year=2013&pages=820-27&pmid=22722338&doi=10.1097/FPC.0b013e3283559b22&)

3. Viollet B, Guigas B, Sanz Garcia N, Leclerc J, Foretz M, Andreelli F. Cellular and molecular mechanisms of metformin: an overview. Clin Sci (Lond) 2012;122:253–70. doi: 10.1042/CS20110386.  [DOI](https://doi.org/10.1042/CS20110386) | [PMC free article](/articles/PMC3398862/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22117616/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Sci%20(Lond)&title=Cellular%20and%20molecular%20mechanisms%20of%20metformin:%20an%20overview&author=B%20Viollet&author=B%20Guigas&author=N%20Sanz%20Garcia&author=J%20Leclerc&author=M%20Foretz&volume=122&publication_year=2012&pages=253-70&pmid=22117616&doi=10.1042/CS20110386&)

4. Graham GG, Punt J, Arora M, Day RO, Doogue MP, Duong JK, Furlong TJ, Greenfield JR, Greenup LC, Kirkpatrick CM, Ray JE, Timmins P, Williams KM. Clinical pharmacokinetics of metformin. Clin Pharmacokinet. 2011;50:81–98. doi: 10.2165/11534750-000000000-00000.  [DOI](https://doi.org/10.2165/11534750-000000000-00000) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21241070/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacokinet&title=Clinical%20pharmacokinetics%20of%20metformin&author=GG%20Graham&author=J%20Punt&author=M%20Arora&author=RO%20Day&author=MP%20Doogue&volume=50&publication_year=2011&pages=81-98&pmid=21241070&doi=10.2165/11534750-000000000-00000&)

5. Hamrén B, Björk E, Sunzel M, Karlsson M. Models for plasma glucose, HbA1c, and hemoglobin interrelationships in patients with type 2 diabetes following tesaglitazar treatment. Clin Pharmacol Ther. 2008;84:228–35. doi: 10.1038/clpt.2008.2.  [DOI](https://doi.org/10.1038/clpt.2008.2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18388881/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Models%20for%20plasma%20glucose,%20HbA1c,%20and%20hemoglobin%20interrelationships%20in%20patients%20with%20type%202%20diabetes%20following%20tesaglitazar%20treatment&author=B%20Hamr%C3%A9n&author=E%20Bj%C3%B6rk&author=M%20Sunzel&author=M%20Karlsson&volume=84&publication_year=2008&pages=228-35&pmid=18388881&doi=10.1038/clpt.2008.2&)

6. Menon V, Greene T, Pereira AA, Wang X, Beck GJ, Kusek JW, Collins AJ, Levey AS, Sarnak MJ. Glycosylated hemoglobin and mortality in patients with nondiabetic chronic kidney disease. J Am Soc Nephrol. 2005;16:3411–27. doi: 10.1681/ASN.2005050552.  [DOI](https://doi.org/10.1681/ASN.2005050552) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16207824/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Am%20Soc%20Nephrol&title=Glycosylated%20hemoglobin%20and%20mortality%20in%20patients%20with%20nondiabetic%20chronic%20kidney%20disease&author=V%20Menon&author=T%20Greene&author=AA%20Pereira&author=X%20Wang&author=GJ%20Beck&volume=16&publication_year=2005&pages=3411-27&pmid=16207824&doi=10.1681/ASN.2005050552&)

7. Zoungas S, Chalmers J, Ninomiya T, Li Q, Cooper ME, Colagiuri S, Fulcher G, de Galan BE, Harrap S, Hamet P, Heller S, MacMahon S, Marre M, Poulter N, Travert F, Patel A, Neal B, Woodward M. Association of HbA1c levels with vascular complications and death in patients with type 2 diabetes: evidence of glycaemic thresholds. Diabetologia. 2012;55:636–43. doi: 10.1007/s00125-011-2404-1.  [DOI](https://doi.org/10.1007/s00125-011-2404-1) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22186981/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Diabetologia&title=Association%20of%20HbA1c%20levels%20with%20vascular%20complications%20and%20death%20in%20patients%20with%20type%202%20diabetes:%20evidence%20of%20glycaemic%20thresholds&author=S%20Zoungas&author=J%20Chalmers&author=T%20Ninomiya&author=Q%20Li&author=ME%20Cooper&volume=55&publication_year=2012&pages=636-43&pmid=22186981&doi=10.1007/s00125-011-2404-1&)

8. Andersson C, van Gaal L, Caterson ID, Weeke P, James WPT, Coutinho W, Couthino W, Finer N, Sharma AM, Maggioni AP, Torp-Pedersen C. Relationship between HbA1c levels and risk of cardiovascular adverse outcomes and all-cause mortality in overweight and obese cardiovascular high-risk women and men with type 2 diabetes. Diabetologia. 2012;55:2348–55. doi: 10.1007/s00125-012-2584-3.  [DOI](https://doi.org/10.1007/s00125-012-2584-3) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22638548/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Diabetologia&title=Relationship%20between%20HbA1c%20levels%20and%20risk%20of%20cardiovascular%20adverse%20outcomes%20and%20all-cause%20mortality%20in%20overweight%20and%20obese%20cardiovascular%20high-risk%20women%20and%20men%20with%20type%202%20diabetes&author=C%20Andersson&author=L%20van%20Gaal&author=ID%20Caterson&author=P%20Weeke&author=WPT%20James&volume=55&publication_year=2012&pages=2348-55&pmid=22638548&doi=10.1007/s00125-012-2584-3&)

9. Cook MN, Girman CJ, Stein PP, Alexander CM. Initial monotherapy with either metformin or sulphonylureas often fails to achieve or maintain current glycaemic goals in patients with type 2 diabetes in UK primary care. Diabet Med. 2007;24:350–58. doi: 10.1111/j.1464-5491.2007.02078.x.  [DOI](https://doi.org/10.1111/j.1464-5491.2007.02078.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17335466/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Diabet%20Med&title=Initial%20monotherapy%20with%20either%20metformin%20or%20sulphonylureas%20often%20fails%20to%20achieve%20or%20maintain%20current%20glycaemic%20goals%20in%20patients%20with%20type%202%20diabetes%20in%20UK%20primary%20care&author=MN%20Cook&author=CJ%20Girman&author=PP%20Stein&author=CM%20Alexander&volume=24&publication_year=2007&pages=350-58&pmid=17335466&doi=10.1111/j.1464-5491.2007.02078.x&)

10. Goswami S, Yee SW, Stocker S, Mosley JD, Kubo M, Castro R, Mefford J, Wen C, Liang X, Witte J, Brett C, Maeda S, Simpson MD, Hedderson MM, Davis RL, Roden DM, Giacomini KM, Savic RM. Genetic variants in transcription factors are associated with the pharmacokinetics and pharmacodynamics of metformin. Clin Pharmacol Ther. 2014;96:370–90. doi: 10.1038/clpt.2014.109.  [DOI](https://doi.org/10.1038/clpt.2014.109) | [PMC free article](/articles/PMC4171106/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24853734/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Genetic%20variants%20in%20transcription%20factors%20are%20associated%20with%20the%20pharmacokinetics%20and%20pharmacodynamics%20of%20metformin&author=S%20Goswami&author=SW%20Yee&author=S%20Stocker&author=JD%20Mosley&author=M%20Kubo&volume=96&publication_year=2014&pages=370-90&pmid=24853734&doi=10.1038/clpt.2014.109&)

11. Shu Y, Leabman MK, Feng B, Mangravite LM, Huang CC, Stryke D, Kawamoto M, Johns SJ, DeYoung J, Carlson E, Ferrin TE, Herskowitz I, Giacomini KM. Evolutionary conservation predicts function of variants of the human organic cation transporter, OCT1. Proc Natl Acad Sci U S A. 2003;100:5902–7. doi: 10.1073/pnas.0730858100.  [DOI](https://doi.org/10.1073/pnas.0730858100) | [PMC free article](/articles/PMC156299/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12719534/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&title=Evolutionary%20conservation%20predicts%20function%20of%20variants%20of%20the%20human%20organic%20cation%20transporter,%20OCT1&author=Y%20Shu&author=MK%20Leabman&author=B%20Feng&author=LM%20Mangravite&author=CC%20Huang&volume=100&publication_year=2003&pages=5902-7&pmid=12719534&doi=10.1073/pnas.0730858100&)

12. Shu Y, Sheardown SA, Brown C, Owen RP, Zhang S, Castro RA, Ianculescu AG, Yue L, Lo JC, Burchard EG, Brett CM, Giacomini KM. Effect of genetic variation in the organic cation transporter 1 ( OCT1 ) on metformin action. J Clin Invest. 2007;117:1422–31. doi: 10.1172/JCI30558.  [DOI](https://doi.org/10.1172/JCI30558) | [PMC free article](/articles/PMC1857259/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17476361/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Invest&title=Effect%20of%20genetic%20variation%20in%20the%20organic%20cation%20transporter%201%20(%20OCT1%20)%20on%20metformin%20action&author=Y%20Shu&author=SA%20Sheardown&author=C%20Brown&author=RP%20Owen&author=S%20Zhang&volume=117&publication_year=2007&pages=1422-31&pmid=17476361&doi=10.1172/JCI30558&)

13. Stocker SL, Morrissey KM, Yee SW, Castro R, Xu L, Dahlin A, Ramirez H, Roden DM, Wilke R, McCarty C, Davis RL, Brett CM, Giacomini KM. The effect of novel promoter variants in MATE1 and MATE2 on the pharmacokinetics and pharmacodynamics of metformin. Clin Pharmacol Ther. 2013;93:186–94. doi: 10.1038/clpt.2012.210.  [DOI](https://doi.org/10.1038/clpt.2012.210) | [PMC free article](/articles/PMC3671611/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23267855/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=The%20effect%20of%20novel%20promoter%20variants%20in%20MATE1%20and%20MATE2%20on%20the%20pharmacokinetics%20and%20pharmacodynamics%20of%20metformin&author=SL%20Stocker&author=KM%20Morrissey&author=SW%20Yee&author=R%20Castro&author=L%20Xu&volume=93&publication_year=2013&pages=186-94&pmid=23267855&doi=10.1038/clpt.2012.210&)

14. Christensen MMH, Brasch-Andersen C, Green H, Nielsen F, Damkier P, Beck-Nielsen H, Brosen K. The pharmacogenetics of metformin and its impact on plasma metformin steady-state levels and glycosylated hemoglobin A1c. Pharmacogenet Genomics. 2011;21:837–50. doi: 10.1097/FPC.0b013e32834c0010.  [DOI](https://doi.org/10.1097/FPC.0b013e32834c0010) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21989078/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=The%20pharmacogenetics%20of%20metformin%20and%20its%20impact%20on%20plasma%20metformin%20steady-state%20levels%20and%20glycosylated%20hemoglobin%20A1c&author=MMH%20Christensen&author=C%20Brasch-Andersen&author=H%20Green&author=F%20Nielsen&author=P%20Damkier&volume=21&publication_year=2011&pages=837-50&pmid=21989078&doi=10.1097/FPC.0b013e32834c0010&)

15. Zhou K, Bellenguez C, Spencer CC, Bennett AJ, Coleman RL, Tavendale R, Hawley S, Donnelly L, Schofield C, Groves CJ, Burch L, Carr F, Strange A, Freeman C, Blackwell JM, Bramon E, Brown M, Casas JP, Corvin A, Craddock N, Deloukas P, Dronov S, Duncanson A, Edkins S, Gray E, Hunt S, Jankowski J, Langford C, Markus HS, Mathew CG, Plomin R, Rautanen A, Sawcer SJ, Samani NJ, Trembath R, Viswanathan AC, Wood NW, Harries LW, Hattersley AT, Doney ASF, Colhoun H, Morris AD, Sutherland C, Hardie DG, Peltonen L, McCarthy MI, Holman RR, Palmer CN, Donnelly P, Pearson ER. Common variants near ATM are associated with glycemic response to metformin in type 2 diabetes. Nat Genet. 2011;43:117–20. doi: 10.1038/ng.735.  [DOI](https://doi.org/10.1038/ng.735) | [PMC free article](/articles/PMC3030919/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21186350/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Common%20variants%20near%20ATM%20are%20associated%20with%20glycemic%20response%20to%20metformin%20in%20type%202%20diabetes&author=K%20Zhou&author=C%20Bellenguez&author=CC%20Spencer&author=AJ%20Bennett&author=RL%20Coleman&volume=43&publication_year=2011&pages=117-20&pmid=21186350&doi=10.1038/ng.735&)

16. De Winter W, DeJongh J, Post T, Ploeger B, Urquhart R, Moules I, Eckland D, Danhof M. A mechanism-based disease progression model for comparison of long-term effects of pioglitazone, metformin and gliclazide on disease processes underlying type 2 diabetes mellitus. J Pharmacokinet Pharmacodyn. 2006;33:313–43. doi: 10.1007/s10928-006-9008-2.  [DOI](https://doi.org/10.1007/s10928-006-9008-2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16552630/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharmacokinet%20Pharmacodyn&title=A%20mechanism-based%20disease%20progression%20model%20for%20comparison%20of%20long-term%20effects%20of%20pioglitazone,%20metformin%20and%20gliclazide%20on%20disease%20processes%20underlying%20type%202%20diabetes%20mellitus&author=W%20De%20Winter&author=J%20DeJongh&author=T%20Post&author=B%20Ploeger&author=R%20Urquhart&volume=33&publication_year=2006&pages=313-43&pmid=16552630&doi=10.1007/s10928-006-9008-2&)

17. Ette EI, Williams PJ. Population pharmacokinetics I: background, concepts, and models. Ann Pharmacother. 2004;38:1702–16. doi: 10.1345/aph.1D374.  [DOI](https://doi.org/10.1345/aph.1D374) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15328391/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Pharmacother&title=Population%20pharmacokinetics%20I:%20background,%20concepts,%20and%20models&author=EI%20Ette&author=PJ%20Williams&volume=38&publication_year=2004&pages=1702-16&pmid=15328391&doi=10.1345/aph.1D374&)

18. Ette EI, Williams PJ, Lane JR. Population pharmacokinetics III: design, analysis, and application of population pharmacokinetic studies. Ann Pharmacother. 2004;38:2136–44. doi: 10.1345/aph.1E260.  [DOI](https://doi.org/10.1345/aph.1E260) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15507495/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Pharmacother&title=Population%20pharmacokinetics%20III:%20design,%20analysis,%20and%20application%20of%20population%20pharmacokinetic%20studies&author=EI%20Ette&author=PJ%20Williams&author=JR%20Lane&volume=38&publication_year=2004&pages=2136-44&pmid=15507495&doi=10.1345/aph.1E260&)

19. Pillai GC, Mentré F, Steimer JL. Non-linear mixed effects modeling - from methodology and software development to driving implementation in drug development science. J Pharmacokinet Pharmacodyn. 2005;32:161–83. doi: 10.1007/s10928-005-0062-y.  [DOI](https://doi.org/10.1007/s10928-005-0062-y) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16283536/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharmacokinet%20Pharmacodyn&title=Non-linear%20mixed%20effects%20modeling%20-%20from%20methodology%20and%20software%20development%20to%20driving%20implementation%20in%20drug%20development%20science&author=GC%20Pillai&author=F%20Mentr%C3%A9&author=JL%20Steimer&volume=32&publication_year=2005&pages=161-83&pmid=16283536&doi=10.1007/s10928-005-0062-y&)

20. Hong Y, Rohatagi S, Habtemariam B, Walker JR, Schwartz SL, Mager DE. Population exposure-response modeling of metformin in patients with type 2 diabetes mellitus. J Clin Pharmacol. 2008;48:696–707. doi: 10.1177/0091270008316884.  [DOI](https://doi.org/10.1177/0091270008316884) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18372428/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Pharmacol&title=Population%20exposure-response%20modeling%20of%20metformin%20in%20patients%20with%20type%202%20diabetes%20mellitus&author=Y%20Hong&author=S%20Rohatagi&author=B%20Habtemariam&author=JR%20Walker&author=SL%20Schwartz&volume=48&publication_year=2008&pages=696-707&pmid=18372428&doi=10.1177/0091270008316884&)

21. Møller JB, Kristensen NR, Klim S, Karlsson MO, Ingwersen SH, Kjellsson MC. Methods for predicting diabetes phase III efficacy outcome from early data: superior performance obtained using longitudinal approaches. CPT pharmacometrics Syst Pharmacol. 2014;3:122–30. doi: 10.1038/psp.2014.20.  [DOI](https://doi.org/10.1038/psp.2014.20) | [PMC free article](/articles/PMC4120014/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24988185/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=CPT%20pharmacometrics%20Syst%20Pharmacol&title=Methods%20for%20predicting%20diabetes%20phase%20III%20efficacy%20outcome%20from%20early%20data:%20superior%20performance%20obtained%20using%20longitudinal%20approaches&author=JB%20M%C3%B8ller&author=NR%20Kristensen&author=S%20Klim&author=MO%20Karlsson&author=SH%20Ingwersen&volume=3&publication_year=2014&pages=122-30&pmid=24988185&doi=10.1038/psp.2014.20&)

22. Chan PL, Holford NH. Drug treatment effects on disease progression. Annu Rev Pharmacol Toxicol. 2001;41:625–59. doi: 10.1146/annurev.pharmtox.41.1.625.  [DOI](https://doi.org/10.1146/annurev.pharmtox.41.1.625) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11264471/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Annu%20Rev%20Pharmacol%20Toxicol&title=Drug%20treatment%20effects%20on%20disease%20progression&author=PL%20Chan&author=NH%20Holford&volume=41&publication_year=2001&pages=625-59&pmid=11264471&doi=10.1146/annurev.pharmtox.41.1.625&)

23. Boyle AP, Hong EL, Hariharan M, Cheng Y, Schaub Ma, Kasowski M, Karczewski KJ, Park J, Hitz BC, Weng S, Cherry JM, Snyder M. Annotation of functional variation in personal genomes using RegulomeDB. Genome Res. 2012;22:1790–97. doi: 10.1101/gr.137323.112.  [DOI](https://doi.org/10.1101/gr.137323.112) | [PMC free article](/articles/PMC3431494/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22955989/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Res&title=Annotation%20of%20functional%20variation%20in%20personal%20genomes%20using%20RegulomeDB&author=AP%20Boyle&author=EL%20Hong&author=M%20Hariharan&author=Y%20Cheng&author=Ma%20Schaub&volume=22&publication_year=2012&pages=1790-97&pmid=22955989&doi=10.1101/gr.137323.112&)

24. Myers RM, Stamatoyannopoulos J, Snyder M, Dunham I, Hardison RC, Bernstein BE, Gingeras TR, Good PJ, et al. A user’s guide to the Encyclopedia of DNA elements (ENCODE) PLoS Biol. 2011;9:245–99. doi: 10.1371/journal.pbio.1001046.  [DOI](https://doi.org/10.1371/journal.pbio.1001046) | [PMC free article](/articles/PMC3079585/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21526222/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Biol&title=A%20user%E2%80%99s%20guide%20to%20the%20Encyclopedia%20of%20DNA%20elements%20(ENCODE)&author=RM%20Myers&author=J%20Stamatoyannopoulos&author=M%20Snyder&author=I%20Dunham&author=RC%20Hardison&volume=9&publication_year=2011&pages=245-99&pmid=21526222&doi=10.1371/journal.pbio.1001046&)

25. Chen Y, Li S, Brown C, Cheatham S, Castro RA, Leabman MK, Urban TJ, Chen L, Yee SW, Choi JH, Huang Y, Brett CM, Burchard EG, Giacomini KM. Effect of genetic variation in the organic cation transporter 2 on the renal elimination of metformin. Pharmacogenet Genomics. 2009;19:497–504. doi: 10.1097/FPC.0b013e32832cc7e9.  [DOI](https://doi.org/10.1097/FPC.0b013e32832cc7e9) | [PMC free article](/articles/PMC3104496/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19483665/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Effect%20of%20genetic%20variation%20in%20the%20organic%20cation%20transporter%202%20on%20the%20renal%20elimination%20of%20metformin&author=Y%20Chen&author=S%20Li&author=C%20Brown&author=S%20Cheatham&author=RA%20Castro&volume=19&publication_year=2009&pages=497-504&pmid=19483665&doi=10.1097/FPC.0b013e32832cc7e9&)

26. Wang ZJ, Yin OQP, Tomlinson B, Chow MSS. OCT2 polymorphisms and in-vivo renal functional consequence: studies with metformin and cimetidine. Pharmacogenet Genomics. 2008;18:637–45. doi: 10.1097/FPC.0b013e328302cd41.  [DOI](https://doi.org/10.1097/FPC.0b013e328302cd41) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18551044/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=OCT2%20polymorphisms%20and%20in-vivo%20renal%20functional%20consequence:%20studies%20with%20metformin%20and%20cimetidine&author=ZJ%20Wang&author=OQP%20Yin&author=B%20Tomlinson&author=MSS%20Chow&volume=18&publication_year=2008&pages=637-45&pmid=18551044&doi=10.1097/FPC.0b013e328302cd41&)

27. Christensen MMH, Pedersen RS, Stage TB, Brasch-andersen C, Nielsen F, Damkier P, Beck-nielsen H, Brøsen K. A gene – gene interaction between polymorphisms in the OCT2 and MATE1 genes influences the renal clearance of metformin. Pharmacogenet Genomics. 2013;23:526–34. doi: 10.1097/FPC.0b013e328364a57d.  [DOI](https://doi.org/10.1097/FPC.0b013e328364a57d) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23873119/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=A%20gene%20%E2%80%93%20gene%20interaction%20between%20polymorphisms%20in%20the%20OCT2%20and%20MATE1%20genes%20influences%20the%20renal%20clearance%20of%20metformin&author=MMH%20Christensen&author=RS%20Pedersen&author=TB%20Stage&author=C%20Brasch-andersen&author=F%20Nielsen&volume=23&publication_year=2013&pages=526-34&pmid=23873119&doi=10.1097/FPC.0b013e328364a57d&)

28. Welter D, MacArthur J, Morales J, Burdett T, Hall P, Junkins H, Klemm A, Flicek P, Manolio T, Hindorff L, Parkinson H. The NHGRI GWAS Catalog, a curated resource of SNP-trait associations. Nucleic Acids Res. 2014;42:1001–6. doi: 10.1093/nar/gkt1229.  [DOI](https://doi.org/10.1093/nar/gkt1229) | [PMC free article](/articles/PMC3965119/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24316577/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=The%20NHGRI%20GWAS%20Catalog,%20a%20curated%20resource%20of%20SNP-trait%20associations&author=D%20Welter&author=J%20MacArthur&author=J%20Morales&author=T%20Burdett&author=P%20Hall&volume=42&publication_year=2014&pages=1001-6&pmid=24316577&doi=10.1093/nar/gkt1229&)

29. Manning AK, Hivert MF, Scott RA, Grimsby JL, Bouatia-Naji N, Chen H, Rybin D, Langenberg C, et al. A genome-wide approach accounting for body mass index identifies genetic variants influencing fasting glycemic traits and insulin resistance. Nat Genet. 2012;44:659–69. doi: 10.1038/ng.2274.  [DOI](https://doi.org/10.1038/ng.2274) | [PMC free article](/articles/PMC3613127/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22581228/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=A%20genome-wide%20approach%20accounting%20for%20body%20mass%20index%20identifies%20genetic%20variants%20influencing%20fasting%20glycemic%20traits%20and%20insulin%20resistance&author=AK%20Manning&author=MF%20Hivert&author=RA%20Scott&author=JL%20Grimsby&author=N%20Bouatia-Naji&volume=44&publication_year=2012&pages=659-69&pmid=22581228&doi=10.1038/ng.2274&)

30. Florez JC, Barrett-Connor E, Jablonski K, Knowler WC, Taylor A, Shuldiner AR, Mather K, Pollin TI, Horton E, White NH. The C allele of ATM rs11212617 does not associate with metformin response in the diabetes prevention program. Diabetes Care. 2012;35:1864–67. doi: 10.2337/dc11-2301.  [DOI](https://doi.org/10.2337/dc11-2301) | [PMC free article](/articles/PMC3425006/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22751958/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Diabetes%20Care&title=The%20C%20allele%20of%20ATM%20rs11212617%20does%20not%20associate%20with%20metformin%20response%20in%20the%20diabetes%20prevention%20program&author=JC%20Florez&author=E%20Barrett-Connor&author=K%20Jablonski&author=WC%20Knowler&author=A%20Taylor&volume=35&publication_year=2012&pages=1864-67&pmid=22751958&doi=10.2337/dc11-2301&)

31. Tzvetkov MV, Saadatmand AR, Bokelmann K, Meineke I, Kaiser R, Brockmöller J. Effects of OCT1 polymorphisms on the cellular uptake, plasma concentrations and efficacy of the 5-HT3 antagonists tropisetron and ondansetron. Pharmacogenomics J. 2012;12:22–29. doi: 10.1038/tpj.2010.75.  [DOI](https://doi.org/10.1038/tpj.2010.75) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20921968/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=Effects%20of%20OCT1%20polymorphisms%20on%20the%20cellular%20uptake,%20plasma%20concentrations%20and%20efficacy%20of%20the%205-HT3%20antagonists%20tropisetron%20and%20ondansetron&author=MV%20Tzvetkov&author=AR%20Saadatmand&author=K%20Bokelmann&author=I%20Meineke&author=R%20Kaiser&volume=12&publication_year=2012&pages=22-29&pmid=20921968&doi=10.1038/tpj.2010.75&)

32. Choi JH, Yee SW, Ramirez H, Morrissey KM, Jang GH, Joski PJ, Mefford J, Hesselson SE, Schlessinger A, Jenkins G, Castro R, Johns SJ, Stryke D, Sali A, Ferrin TE, Witte JS, Kwok PY, Roden DM, Wilke R, McCarty C, Davis RL, Giacomini KM. A common 5′-UTR variant in MATE2-K is associated with poor response to metformin. Clin Pharmacol Ther. 2011;90:674–84. doi: 10.1038/clpt.2011.165.  [DOI](https://doi.org/10.1038/clpt.2011.165) | [PMC free article](/articles/PMC3329222/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21956618/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=A%20common%205%E2%80%B2-UTR%20variant%20in%20MATE2-K%20is%20associated%20with%20poor%20response%20to%20metformin&author=JH%20Choi&author=SW%20Yee&author=H%20Ramirez&author=KM%20Morrissey&author=GH%20Jang&volume=90&publication_year=2011&pages=674-84&pmid=21956618&doi=10.1038/clpt.2011.165&)

33. Turner R. Effect of intensive blood-glucose control with metformin on complications in overweight patients with type 2 diabetes (UKPDS 34) Lancet. 1998;352:854–65.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/9742977/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Effect%20of%20intensive%20blood-glucose%20control%20with%20metformin%20on%20complications%20in%20overweight%20patients%20with%20type%202%20diabetes%20(UKPDS%2034)&author=R%20Turner&volume=352&publication_year=1998&pages=854-65&pmid=9742977&)

34. Keoki Williams L, Padhukasahasram B, Ahmedani BK, Peterson EL, Wells KE, González Burchard E, Lanfear DE. Differing effects of metformin on glycemic control by race-ethnicity. J Clin Endocrinol Metab. 2014:39–55. doi: 10.1210/jc.2014-1539.  [DOI](https://doi.org/10.1210/jc.2014-1539) | [PMC free article](/articles/PMC4154100/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24921653/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Endocrinol%20Metab&title=Differing%20effects%20of%20metformin%20on%20glycemic%20control%20by%20race-ethnicity&author=L%20Keoki%20Williams&author=B%20Padhukasahasram&author=BK%20Ahmedani&author=EL%20Peterson&author=KE%20Wells&publication_year=2014&pages=39-55&pmid=24921653&doi=10.1210/jc.2014-1539&)

35. Irvin MR, Wineinger NE, Rice TK, Pajewski NM, Kabagambe EK, Gu CC, Pankow J, North KE, Wilk JB, Freedman BI, Franceschini N, Broeckel U, Tiwari HK, Arnett DK. Genome-wide detection of allele specific copy number variation associated with insulin resistance in african americans from the hyperGEN study. PLoS One. 2011;6:9–11. doi: 10.1371/journal.pone.0024052.  [DOI](https://doi.org/10.1371/journal.pone.0024052) | [PMC free article](/articles/PMC3162025/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21901158/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=Genome-wide%20detection%20of%20allele%20specific%20copy%20number%20variation%20associated%20with%20insulin%20resistance%20in%20african%20americans%20from%20the%20hyperGEN%20study&author=MR%20Irvin&author=NE%20Wineinger&author=TK%20Rice&author=NM%20Pajewski&author=EK%20Kabagambe&volume=6&publication_year=2011&pages=9-11&pmid=21901158&doi=10.1371/journal.pone.0024052&)

36. Hong KW, Go MJ, Jin HS, Lim JE, Lee JY, Han BG, Hwang SY, Lee SH, Park HK, Cho YS, Oh B. Genetic variations in ATP2B1, CSK, ARSG and CSMD1 loci are related to blood pressure and/or hypertension in two Korean cohorts. J Hum Hypertens. 2010;24:367–72. doi: 10.1038/jhh.2009.86.  [DOI](https://doi.org/10.1038/jhh.2009.86) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19960030/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Hum%20Hypertens&title=Genetic%20variations%20in%20ATP2B1,%20CSK,%20ARSG%20and%20CSMD1%20loci%20are%20related%20to%20blood%20pressure%20and/or%20hypertension%20in%20two%20Korean%20cohorts&author=KW%20Hong&author=MJ%20Go&author=HS%20Jin&author=JE%20Lim&author=JY%20Lee&volume=24&publication_year=2010&pages=367-72&pmid=19960030&doi=10.1038/jhh.2009.86&)

37. Aldaz CM, Ferguson BW, Abba MC. WWOX at the crossroads of cancer, metabolic syndrome related traits and CNS pathologies. Biochim Biophys Acta. 2014;1846:188–200. doi: 10.1016/j.bbcan.2014.06.001.  [DOI](https://doi.org/10.1016/j.bbcan.2014.06.001) | [PMC free article](/articles/PMC4151823/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24932569/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochim%20Biophys%20Acta&title=WWOX%20at%20the%20crossroads%20of%20cancer,%20metabolic%20syndrome%20related%20traits%20and%20CNS%20pathologies&author=CM%20Aldaz&author=BW%20Ferguson&author=MC%20Abba&volume=1846&publication_year=2014&pages=188-200&pmid=24932569&doi=10.1016/j.bbcan.2014.06.001&)

38. Yu W, Clyne M, Khoury MJ, Gwinn M. Phenopedia and Genopedia: disease-centered and gene-centered views of the evolving knowledge of human genetic associations. Bioinformatics. 2010;26:145–56. doi: 10.1093/bioinformatics/btp618.  [DOI](https://doi.org/10.1093/bioinformatics/btp618) | [PMC free article](/articles/PMC2796820/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19864262/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=Phenopedia%20and%20Genopedia:%20disease-centered%20and%20gene-centered%20views%20of%20the%20evolving%20knowledge%20of%20human%20genetic%20associations&author=W%20Yu&author=M%20Clyne&author=MJ%20Khoury&author=M%20Gwinn&volume=26&publication_year=2010&pages=145-56&pmid=19864262&doi=10.1093/bioinformatics/btp618&)

39. Hoggart CJ, Whittaker JC, De Iorio M, Balding DJ. Simultaneous analysis of all SNPs in genome-wide and re-sequencing association studies. PLoS Genet. 2008;4:1–8. doi: 10.1371/journal.pgen.1000130.  [DOI](https://doi.org/10.1371/journal.pgen.1000130) | [PMC free article](/articles/PMC2464715/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18654633/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet&title=Simultaneous%20analysis%20of%20all%20SNPs%20in%20genome-wide%20and%20re-sequencing%20association%20studies&author=CJ%20Hoggart&author=JC%20Whittaker&author=M%20De%20Iorio&author=DJ%20Balding&volume=4&publication_year=2008&pages=1-8&pmid=18654633&doi=10.1371/journal.pgen.1000130&)

40. Bertrand J, Balding DJ. Multiple single nucleotide polymorphism analysis using penalized regression in nonlinear mixed-effect pharmacokinetic models. Pharmacogenet Genomics. 2013;23:167–74. doi: 10.1097/FPC.0b013e32835dd22c.  [DOI](https://doi.org/10.1097/FPC.0b013e32835dd22c) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23337849/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Multiple%20single%20nucleotide%20polymorphism%20analysis%20using%20penalized%20regression%20in%20nonlinear%20mixed-effect%20pharmacokinetic%20models&author=J%20Bertrand&author=DJ%20Balding&volume=23&publication_year=2013&pages=167-74&pmid=23337849&doi=10.1097/FPC.0b013e32835dd22c&)
