# HLA-B*15:21 and carbamazepine-induced Stevens-Johnson syndrome: pooled-data and in silico analysis

## Metadata
**Authors:** Kanoot Jaruthamsophon, Varomyalin Tipmanee, Antida Sangiemchoey, Chonlaphat Sukasem, Pornprot Limprasert
**Journal:** Scientific Reports
**Date:** 2017 Mar 30
**DOI:** [10.1038/srep45553](https://doi.org/10.1038/srep45553)
**PMID:** 28358139
**PMCID:** PMC5372085
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5372085/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC5372085/pdf/srep45553.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC5372085/pdf/srep45553.pdf)

## Abstract

HLA-B*15:02 screening before carbamazepine (CBZ) prescription in Asian populations is the recommended practice to prevent CBZ-induced Stevens-Johnson syndrome (CBZ-SJS). However, a number of patients have developed CBZ-SJS even having no HLA-B*15:02. Herein, we present the case of a Thai patient who had a negative HLA-B*15:02 screening result but later developed CBZ-SJS. Further HLA typing revealed HLA-B*15:21/B*13:01. HLA-B*15:21 is a member of the HLA-B75 serotype and is commonly found in Southeast Asian populations. Based on this case, we hypothesised that if all HLA-B*15:02 carriers were prevented from CBZ prescription, another common HLA-B75 serotype marker would show its association with CBZ-SJS. To test this hypothesis, we pooled data from previous association studies in Asian populations, excluded all cases with HLA-B*15:02, and analysed the association significance of HLA-B75 serotype markers. A significant association was found between CBZ-SJS and HLA-B*15:21 and HLA-B*15:11. We also applied an in silico analysis and found that all HLA-B75 serotype molecules shared similar capability in binding the CBZ molecule. In summary, this report provides the first evidence of a positive association between HLA-B*15:21 and CBZ-SJS and the first in silico analysis of CBZ binding sites and details of the molecular behaviour of HLA-B75 molecule to explain its molecular action.

## Introduction

*Human Leukocyte Antigen* (*HLA) -B*15:02* is a genetic marker strongly associated with carbamazepine (CBZ)-induced Stevens-Johnson syndrome and toxic epidermal necrolysis (CBZ-SJS) in Asian populations[1](#b1). Screening for *HLA-B*15:02* before CBZ prescription has proven to prevent CBZ-SJS[2](#b2). However, a number of cases of patients who developed CBZ-SJS while having no *HLA-B*15:02* have been reported[3](#b3),[4](#b4),[5](#b5),[6](#b6). Herein, we report the case of a Thai patient who developed CBZ-SJS while screening negative for *HLA-B*15:02* and was later found to have *HLA-B*15:21*, an HLA-B75 serotype marker similar to *HLA-B*15:02, B*15:11*, and *B*15:08*.

We hypothesised that if all *HLA-B*15:02* carriers were prevented from CBZ prescription, another common HLA-B75 serotype marker – especially *HLA-B*15:21* – would reveal its nature in causing CBZ-SJS. To test this hypothesis, we reviewed published association studies in Asian populations (excluding Japanese and Korean studies, as these populations have been found to have no association with *HLA-B*15:02* [7](#b7),[8](#b8)), pooled genotype data, excluded all cases and controls with *HLA-B*15:02*, and then analysed the association between HLA-B75 serotype markers and CBZ-SJS. Since the serotype manifests in a certain protein form, a tertiary structure of the protein member is necessary for detailed investigation. In our study, we did not only construct a tertiary structure but also performed an *in silico* analysis to evaluate all HLA-B75 structures and the molecular binding between the CBZ molecule and all HLA-B75 serotype molecules compared with the HLA-B*15:01 molecule.

## Results

### Case report

A 14-year-old male patient presented with muscle spasms of his left arm and leg for 6 months. His symptoms were aggravated when he moved quickly. He was diagnosed with paroxysmal kinesigenic dyskinesia and was screened for *HLA-B*15:02* by a validated in-house method[9](#b9). *HLA-B*15:02* screening was negative and the patient was prescribed CBZ 200 mg per day. This patient had neither previous history of food nor drug allergy and had never been prescribed CBZ.

After taking the CBZ for 14 days, the patient developed a maculopapular rash at both arms and trunk, and also had injected eyes and oral ulcers. He was diagnosed with CBZ-SJS, at which time the CBZ was discontinued and he was admitted into the hospital. His symptoms resolved 14 days after the CBZ was discontinued, and he was prescribed topiramate 50 mg per day. A DNA sample was re-examined for HLA typing by the PCR-SSOP method (Luminex^®^IS 100, USA), which revealed *HLA-A*02:03/A*24:02* and *HLA-B*13:01*/*B*15:21*.

### Pooled-data analysis

A search of the HLA and Adverse Drug Reaction Database (accessed on 3 June 2016) found 33 studies evaluating the association between *HLA-B* alleles and antiepileptic drugs[10](#b10). Among these studies, 4 studies did not involve an association between *HLA-B* markers and CBZ-SJS, 9 studies involved European, Japanese, Korean, and Canadian populations, 15 studies had insufficient data for analysis, and the full text of 1 study was not accessible. Finally, only 4 case-control studies were eligible for the pooled-data analysis.

The pooled data of the selected studies are shown in [Table 1](#t1). Among the total of 108 CBZ-SJS cases, 13 cases had no *HLA-B*15:02*. Only 4 cases were found with the HLA-B75 serotype. There were 225 tolerant controls who had no *HLA-B*15:02*. The only reported tolerant control with the HLA-B75 serotype was found to have *HLA-B*15:21* in the Hong Kong Han Chinese study[4](#b4).

### Table 1. Carbamazepine-induced Stevens-Johnson syndrome (CBZ-SJS) cases and tolerant controls without HLA-B*15:02.

| Study (Region) | CBZ-SJS cases |  |  | Tolerant controls |  |
| --- | --- | --- | --- | --- | --- |
| Total | Cases without HLA-B*15:02 | HLA-B genotype of cases without HLA-B*15:02a | Total | Controls without HLA- B*15:02 |  |
| Tassaneeyakul, 2010 (Thailand)3 | 42 | 5 | (1) 18:01/56:04(2) 15:21/48:01(3) 27:06/56:01(4) 15:21/40:01(5) 15:11/46:01 | 42 | 37 |
| Cheung, 2013 (Hong Kong)4 | 26 | 2 | (1) 13:01/54:01(2) 13:01/15:01 | 135 | 119 |
| Khor, 2014 (Malaysia)5 | 5 | 3 | (1) 15:08/56:01(2) 15:01/41:01(3) 40:06/51:06 | 52 | 50 |
| Nguyen, 2015 (Vietnam)6 | 35 | 3 | (1) 13:01/46:01(2) 51:02/54:01(3) 38:02/51:02 | 25 | 19 |
| Total | 108 | 13 |   | 254 | 225 |

Table 1 Caption: The data was obtained from available studies in Asian populations with known associations between HLA-B*15:02 and CBZ-SJS. aUnderlined alleles represented members of the HLA-B75 serotype.

The pooled-data analysis revealed a significant association between CBZ-SJS and both *HLA-B*15:21* (Odds ratio = 40.73, 95% CI = 3.43–484.22, *P* = 0.003) and *HLA-B*15:11* (Odds ratio = 54.12, 95% CI = 2.10–1397.09, *P* = 0.016) ([Table 2](#t2)). However, we were unable to evaluate any possible associations between CBZ-SJS and *HLA-B*15:08* and other common *HLA-B* alleles, as negative association controls, because of insufficient genotype data in the tolerant control group.

### Table 2. Pooled-data analysis for associations between HLA-B*15:21 and B*15:11 and carbamazepine-induced Stevens-Johnson syndrome (CBZ-SJS) in cases without HLA-B*15:02.

| Cases without HLA-B*15:02 | CBZ-SJS cases without B*15:02 (n = 13) | Tolerant controls without B*15:02 (n = 225) | Odds ratio (95% CI), P-value |
| --- | --- | --- | --- |
| B*15:21 | 2 | 1 | 40.73 (3.43–484.22), P = 0.003 |
| No B*15:21 | 11 | 224 |  |
| B*15:11 | 1 | 0a | 54.12 (2.10–1397.09), P = 0.016 |
| No B*15:11 | 12 | 225 |  |

Table 2 Caption: The data were obtained from 4 studies with high-resolution HLA-B typing results. aAs zero is present in the table, 0.5 was added to all cells before calculating the odds ratios.

### In silico analysis

To understand how base changes affect a protein structure and CBZ reacts to the protein structure, a sequence alignment was performed among all HLA-B75 serotype members of interest to evaluate the possibility of modelling the tertiary structures and investigating the molecular information. The sequence alignment, [Fig. 1](#f1), shows a high similarity among the proteins of interest with respect to HLA-B*15:01, in which the tertiary structure is available in the experimental RCSB PDB database (pdb code 1XR9)[11](#b11). At least 95% of the amino acids were conserved, clearly seen from [Fig. 1](#f1).

### Figure 1. Alignment of the amino acid sequences of HLA-B*15:01, B*15:02, B*15:08, B*15:11, and B*15:21.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8c25/5372085/744d3cc3b7c8/srep45553-f1.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5372085_srep45553-f1.jpg)

The data were obtained from the Sequence Alignment Tool found in the IMGT/HLA database version 3.25.0 (accessed on 8 August 2016).

According to the sequence alignment, the other HLA-B75 protein structures can be successfully modelled and simulated using molecular dynamics (MD) simulation to assess the protein conformational details. From our results ([Fig. 2](#f2)), MD sampling showed that a protein conformation of the antigen-presenting area in the HLA-B75 protein remains similar among species except for HLA-B*15:21. In [Fig. 2](#f2), the domain of HLA-B*15:21 can be seen to be larger, compared to other HLA-B75 proteins and HLA-B*15:01 crystal structure.

### Figure 2. Conformation comparison of HLA-B*15:01 and HLA-B75 protein species.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8c25/5372085/75e4e702f1db/srep45553-f2.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5372085_srep45553-f2.jpg)

All simulated structures were aligned with respect to the HLA-B*15:01 crystal structure to visualise the effects due to amino acid changes on protein conformation. The protein conformations at the antigen-presenting area are similar to the HLA-B*15:01 crystal structure (green ribbons), except a larger gap was found in HLA-B*15:21 (purple ribbons with arrow indicated).

Apart from the protein conformation, the selectivity of CBZ binding to the HLA-B75 family was evaluated using the molecular docking approach. In this study, we illustrate the possible CBZ binding sites on HLA-B75 protein on the basis of binding energy. The sites with a lower binding energy (more negative number) indicate a higher possibility and more favourable binding preference. [Table 3](#t3) shows the binding energy results from a docking study between CBZ and HLA-B*15:01 and the various HLA-B75 members. In addition, the structure of the CBZ-HLA-B molecule complex from the molecular docking approach is also illustrated to identify the possible CBZ binding sites in [Figs 3](#f3) and [4](#f4).

### Table 3. Binding energy among CBZ and HLA-B*15: 01 and HLA-B75 protein members.

| HLA-B molecule | Possible observed binding energy (Kcal mol−1) |
| --- | --- |
| HLA-B*15:01 | −7.14 |
| HLA-B*15:02 | −7.89 |
| HLA-B*15:08 | −7.10 |
| HLA-B*15:11 | −6.82, −6.18 |
| HLA-B*15:21 | −6.61 |

Table 3 Caption: Molecular docking was performed between CBZ and ten possible protein structures of each HLA-B molecule obtained from an MD simulation. The binding energy was calculated for each case of CBZ bound to the antigen-presenting area of the protein.

### Figure 3. Possible binding sites of CBZ in HLA-B*15:01.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8c25/5372085/3d2f544aed84/srep45553-f3.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5372085_srep45553-f3.jpg)

(a) HLA-B*15:01 functions via binding the T-cell receptor protein using the antigen-presenting area. The green ribbons represent the HLA-B*15:01 protein structure. (b) The two arrows point at the binding groove of the HLA-B*15:01 molecule, also defined as an antigen-presenting area. A possible binding site was determined by a docked CBZ conformation (red) with the lowest binding energy. In HLA-B*15:01, CBZ possibly binds the protein, however, the binding interactions are caused from the amino acid excluding from the antigen-presenting area. In other words, CBZ could bind HLA-B*15:01 but it could not incorporate into the antigen-presenting area.

### Figure 4. Possible binding site of CBZ in HLA-B75 protein structure.

![Figure 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8c25/5372085/6d1c1f1b3038/srep45553-f4.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5372085_srep45553-f4.jpg)

The green ribbons represent the HLA-B75 protein structure, with the two arrows pointing to the binding groove of CBZ (red), also defined as an antigen-presenting area. In (a) HLA-B*15:02, (b) B*15:08, and (d) B*15:21, CBZ possibly binds the protein using interactions from the antigen-presenting area. In (c) HLA-B*15:11, the best conformation (1) is located outside the antigen-presenting area, but the second best conformation (2) is located at the antigen-presenting area.

## Discussion

Even though *HLA-B*15:02* screening is effective in preventing CBZ-SJS, there have been a number of reported cases of people who developed CBZ-SJS even having neither *HLA-B*15:02* nor the other known associated markers, *HLA-A*31:01* and *B*15:11*[3](#b3),[4](#b4),[5](#b5),[6](#b6). Our patient had screened negative for *HLA-B*15:02*, but we later found *HLA-B*15:21*, another HLA-B75 serotype marker that could be a possible cause of CBZ-SJS. Similarly, the pooled data also included 4 CBZ-SJS cases who had no *HLA-B*15:02* but other HLA-B75 serotype makers: 2 cases with *HLA-B*15:21* and 1 each of *HLA-B*15:08* and *B*15:11* ([Table 1](#t1)).

As HLA frequencies vary greatly among populations, the statistical significance of the association studies has also varied regarding the *HLA-B* allele frequencies. For example, *HLA-B*15:21* is common in Southeast Asian and neighbouring populations whose *HLA-B*15:02* frequency is also high[10](#b10),[12](#b12). It is not surprising, then, that to date, association studies have found no connection between *HLA-B*15:21* and CBZ-SJS, as there are only a small number of cases with *HLA-B*15:21* as the large majority of cases have *HLA-B*15:02*. Only after the *HLA-B*15:02* cases were excluded and cases from a number of studies were pooled did we find statistical significance for *HLA-B*15:21.* Since this pooled data included only a limited number of CBZ-SJS cases without the *HLA-B*15:02* allele, further case-control studies from real-world populations are needed to confirm the associations between CBZ-SJS and the *HLA-B*15:21*, as well as other HLA-B75 serotype markers.

Although it is a significant new finding, the positive association between CBZ and *HLA-B*15:21* is not an unexpected finding, since the *HLA-B*15:21* marker is a member of the HLA-B75 serotype group, similar to *HLA-B*15:02* and *B*15:11*. These HLA-B75 serotype members, including *HLA-B*15:08*, share the 63^rd^ arginine residue ([Fig. 1](#f1)) which is critical for the formation of the drug-HLA molecule complex by a direct pharmacological interaction and is able to induce an *in vitro* lymphocyte cytotoxic response to CBZ[13](#b13).

However, only *HLA-B*15:02* and *B*15:11* have been proven to be associated with CBZ-SJS in real-world association studies[3](#b3),[4](#b4),[5](#b5),[6](#b6),[7](#b7),[8](#b8). In this study, we provide the first positive association between CBZ-SJS and *HLA-B*15:21* but not *HLA-B*15:08*, the HLA-B75 serotype marker with very low frequencies in Asian populations[10](#b10),[12](#b12). If an association between all HLA-B75 serotype members and CBZ-SJS is proven unequivocally, a screening policy for all HLA-B75 serotype will need to be considered in order to enhance the benefit of the *HLA-B* screening before CBZ prescription.

Furthermore, an *in silico* approach was performed to investigate the protein structure of the HLA-B75 members of interest, with respect to the reported HLA-B*15:01 structure as well as the molecular mechanism of CBZ in regard to the HLA-B75 members. Looking into the tertiary structure, the molecular dynamics (MD) simulation illustrated structural similarity among the antigen-presenting domain, except for HLA-B*15:21 ([Fig. 2](#f2)). Our findings suggest that the hypothesis that altered amino acids could affect the structure of the antigen-presenting domain in HLA-B75 serotype members and thus eventually lead to different CBZ binding affinities can be discarded. Instead, it appears that the selectivity of CBZ may be due to a different amino acid side chain.

Aside from the protein structure, the mechanism of CBZ when interacting with the HLA-B75 members is of interest, and it was for this reason the binding information was assessed via the molecular docking approach. In addition to HLA-B*15:02 previously demonstrated to have binding affinity to the CBZ molecule[13](#b13),[14](#b14), the molecular docking study revealed that HLA-B*15:11, HLA-B*15:21, and HLA-B*15:08 are also capable of binding to the CBZ molecule directly. In this study, we observed the three structure most capable of binding possible CBZ structures in the proposed CBZ-protein complex, on the basis of binding energy, so that we could evaluate the possible CBZ binding location.

Our study showed that CBZ bound an HLA-B*15:01, but in this study, the binding took place at a groove not participating in the antigen-presenting domain ([Fig. 3](#f3)). Since the HLA-B molecule has to function via a complex formation with the T-cell receptor using amino acids at an antigen-presenting area as a molecular interface, the presence of CBZ on other sites but this domain could play no effect on the induction of immune response.

On the other hand, CBZ was found to bind the HLA-B75 proteins using amino acids at the antigen-presenting area ([Fig. 4](#f4)). In most cases, namely HLA-B*15:02, B*15:08, and B*15:21, the different possible CBZ position could play a role at the protein interface between the HLA-B75 members and the T-cell receptor[14](#b14). This suggests an action of CBZ at the atomistic level on some HLA-B75 protein members. To note about HLA-B*15:11, even though the best binding, with a relative energy of −6.82 KCal mol^−1^, indicates the CBZ may not affect the antigen-presenting area. An integration of CBZ on the antigen-presenting area of HLA-B*15:11 was raised by the second best binding with a relative of −6.18 KCal mol^−1^. Since these energies were somewhat close, these two CBZ positions could be possible. The former, the position (1) in [Fig. 4c](#f4), can be found more though, and HLA-B*15:11-mediated T cell activation due to CBZ could be in a lesser extent, compared to the above-mentioned HLA-B proteins.

In conclusion, this report provides an additional CBZ-SJS case with *HLA-B*15:21*, but no *HLA-B*15:02*, to raise awareness of possible CBZ-SJS in patients who have negative screening for *HLA-B*15:02*. In addition, the study also provides the first evidence of a positive association between *HLA-B*15:21* and CBZ-SJS along with molecular information regarding how CBZ reacts to the HLA-B75 family.

## Methods

### Pooled-data analysis

In this study, we performed a pooled-data analysis to evaluate the association between the uncommon members of the HLA-B75 serotype and CBZ-SJS, in the absence of cases and controls with *HLA-B*15:02*. We reviewed all studies listed in the HLA Adverse Drug Reaction Database which is a part of the Allele Frequencies Net Database (accessed on 3 June 2016)[10](#b10). Full text and supplementary data of all case-control studies evaluating the association of *HLA-B* alleles and antiepileptic drug-induced severe cutaneous drug reactions were reviewed. The study inclusion criteria for pooling data included: (1) studies in Asian populations other than Japanese and Korean, (2) reported in English language, (3) *HLA-B* genotype, in high resolution, available in all CBZ-SJS cases, and (4) allele frequencies of HLA-B75 serotype members available in a tolerant control group. Only CBZ-SJS cases and CBZ tolerant controls were included in the pooled-data analysis. Thereafter, all cases and controls with *HLA-B*15:02* alleles were excluded from the pooled data.

We analysed the statistical significance of the association between HLA-B75 serotype members and CBZ-SJS, using MedCalc ([www.medcalc.org/calc/odds_ratio.php](http://www.medcalc.org/calc/odds_ratio.php)). The study protocol, including the case report, pooled-data analysis, and *in silico* analysis, was approved by the Institutional Research Ethics Committee, Faculty of Medicine, Prince of Songkla University (REC 59-229-05-1). For the case report, we obtained informed consent from the patient and his parents prior to collecting clinical information. The methods were carried out in accordance with the approved guidelines and relevant regulations.

### In silico analysis

A structure of the protein HLA-B75 family (HLA-B*15:02, B*15:08, B*15:11, and B*15:21) was built based on an HLA-B*15:01 crystal structure (pdb code 1XR9)[11](#b11) as a molecular template. The homology modelling of the HLA-B75 family, except for HLA-B*15:21, was performed using the Rosetta Backrub online web server[15](#b15). In addition, the secondary structure of the protein was independently predicted using the Psipred[16](#b16) and NetSurfP[17](#b17) web server tools to facilitate the selection of the best structure among 20 possible conformations. The best structure was chosen based on a scoring method and secondary structure prediction results. An HLA-B*15:21 model was constructed using Tleap module in the AMBER16 package[18](#b18) to model serine-to-cysteine residue changes ([Fig. 1](#f1)). All missing hydrogen atoms were added into the chosen protein structure using the AMBER force field and a protonation state of all ionisable side chains was evaluated at pH 7.0 using the PDB2PQR online tool[19](#b19),[20](#b20). All structures were written in the RCSB Protein DataBank (PDB) format.

Molecular dynamic (MD) simulation was performed to emulate a fluctuated protein conformation. An implicit solvent model (Generalized Born/Surface Area, GBSA) was exploited to mimic the solvent condition. First, the structure was energy-minimised using the steepest descent method for 1,000 steps and then switched to a conjugate gradient method for 1,000 steps. Second, the minimised structure was sampled at 310 K (37 °C) regulated by a weak-coupling algorithm under an implicit 0.1 M NaCl solution. The MD simulation was carried out with the PMEMD module for 100 ns using a time step of 2 fs. The first 70 ns trajectory was omitted and the 300 equidistant snapshots from the last 30 ns trajectory were taken for a configuration average and conformation analysis.

In the molecular docking study, a CBZ structure was adopted from the PubChem Database, CID 2554. Ten HLA-B75 structures from the MD trajectory were chosen as “receptors” for CBZ. Both CBZ and the protein structures were written in the PDBQT format using the AutoDock Auxiliary Tool (ADT)[21](#b21) based on the Autodock4 Universal Force Field (UFF). The study was first performed via Autogrid4, to cover an antigen-presenting area as a possible binding site, and Lamarckian genetic algorithm sampling implemented in Autodock4 yielded 100,000 possible drug-protein structures equal to (10 protein receptors) * (50 Genetic Algorithm runs) * (a population size of 200). The proposed structure was selected on the basis of lowest binding energy. All sampling variables were set as an Autodock4 default parameter unless stated otherwise.

## Additional Information

**How to cite this article:** Jaruthamsophon, K. *et al. HLA-B*15:21* and carbamazepine-induced Stevens-Johnson syndrome: pooled-data and *in silico* analysis. *Sci. Rep.* **7**, 45553; doi: 10.1038/srep45553 (2017).

**Publisher's note:** Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.

## Acknowledgments

We would like to thank Dr. Utcharee Intusoma for providing the clinical information of the case report and Mr. David Patterson for his English proofreading service. K.J. thanks the Faculty of Medicine, Prince of Songkla University, for a Research Credit Grant for financial support. V.T. thanks Thailand Research Fund (Grant no. RTA5880005) for partial support of the computational system and equipment.

## Footnotes

## References

1. Tangamornsuksan W., Chaiyakunapruk N., Somkrua R., Lohitnavy M. & Tassaneeyakul W. Relationship between the HLA-B*1502 allele and carbamazepine-induced Stevens-Johnson syndrome and toxic epidermal necrolysis: a systematic review and meta-analysis. JAMA Dermatol. 149, 1025–1032 (2013).  [DOI](https://doi.org/10.1001/jamadermatol.2013.4114) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23884208/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA%20Dermatol&title=Relationship%20between%20the%20HLA-B*1502%20allele%20and%20carbamazepine-induced%20Stevens-Johnson%20syndrome%20and%20toxic%20epidermal%20necrolysis:%20a%20systematic%20review%20and%20meta-analysis&author=W.%20Tangamornsuksan&author=N.%20Chaiyakunapruk&author=R.%20Somkrua&author=M.%20Lohitnavy&author=W.%20Tassaneeyakul&volume=149&publication_year=2013&pages=1025-1032&pmid=23884208&doi=10.1001/jamadermatol.2013.4114&)

2. Chen P. et al. Carbamazepine-induced toxic effects and HLA-B*1502 screening in Taiwan. N. Engl. J. Med. 364, 1126–1133 (2011).  [DOI](https://doi.org/10.1056/NEJMoa1009717) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21428768/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N.%20Engl.%20J.%20Med.&title=Carbamazepine-induced%20toxic%20effects%20and%20HLA-B*1502%20screening%20in%20Taiwan&author=P.%20Chen&volume=364&publication_year=2011&pages=1126-1133&pmid=21428768&doi=10.1056/NEJMoa1009717&)

3. Tassaneeyakul W. et al. Association between HLA-B*1502 and carbamazepine-induced severe cutaneous adverse drug reactions in a Thai population. Epilepsia 51, 926–930 (2010).  [DOI](https://doi.org/10.1111/j.1528-1167.2010.02533.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20345939/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Epilepsia&title=Association%20between%20HLA-B*1502%20and%20carbamazepine-induced%20severe%20cutaneous%20adverse%20drug%20reactions%20in%20a%20Thai%20population&author=W.%20Tassaneeyakul&volume=51&publication_year=2010&pages=926-930&pmid=20345939&doi=10.1111/j.1528-1167.2010.02533.x&)

4. Cheung Y. K. et al. HLA-B alleles associated with severe cutaneous reactions to antiepileptic drugs in Han Chinese. Epilepsia 54, 1307–1314 (2013).  [DOI](https://doi.org/10.1111/epi.12217) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23692434/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Epilepsia&title=HLA-B%20alleles%20associated%20with%20severe%20cutaneous%20reactions%20to%20antiepileptic%20drugs%20in%20Han%20Chinese&author=Y.%20K.%20Cheung&volume=54&publication_year=2013&pages=1307-1314&pmid=23692434&doi=10.1111/epi.12217&)

5. Khor A. H., Lim K. S., Tan C. T., Wong S. M. & Ng C. C. HLA-B*15:02 association with carbamazepine-induced Stevens-Johnson syndrome and toxic epidermal necrolysis in an Indian population: a pooled-data analysis and meta-analysis. Epilepsia 55, e120–e124 (2014).  [DOI](https://doi.org/10.1111/epi.12802) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25266342/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Epilepsia&title=HLA-B*15:02%20association%20with%20carbamazepine-induced%20Stevens-Johnson%20syndrome%20and%20toxic%20epidermal%20necrolysis%20in%20an%20Indian%20population:%20a%20pooled-data%20analysis%20and%20meta-analysis&author=A.%20H.%20Khor&author=K.%20S.%20Lim&author=C.%20T.%20Tan&author=S.%20M.%20Wong&author=C.%20C.%20Ng&volume=55&publication_year=2014&pages=e120-e124&pmid=25266342&doi=10.1111/epi.12802&)

6. Nguyen D. V. et al. HLA-B*1502 and carbamazepine-induced severe cutaneous adverse drug reactions in Vietnamese. Asia Pac. Allergy 5, 68–77 (2015).  [DOI](https://doi.org/10.5415/apallergy.2015.5.2.68) | [PMC free article](/articles/PMC4415182/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25938071/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Asia%20Pac.%20Allergy&title=HLA-B*1502%20and%20carbamazepine-induced%20severe%20cutaneous%20adverse%20drug%20reactions%20in%20Vietnamese&author=D.%20V.%20Nguyen&volume=5&publication_year=2015&pages=68-77&pmid=25938071&doi=10.5415/apallergy.2015.5.2.68&)

7. Kaniwa N. et al. HLA-B*1511 is a risk factor for carbamazepine-induced Stevens-Johnson syndrome and toxic epidermal necrolysis in Japanese patients. Epilepsia 51, 2461–2465 (2010).  [DOI](https://doi.org/10.1111/j.1528-1167.2010.02766.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21204807/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Epilepsia&title=HLA-B*1511%20is%20a%20risk%20factor%20for%20carbamazepine-induced%20Stevens-Johnson%20syndrome%20and%20toxic%20epidermal%20necrolysis%20in%20Japanese%20patients&author=N.%20Kaniwa&volume=51&publication_year=2010&pages=2461-2465&pmid=21204807&doi=10.1111/j.1528-1167.2010.02766.x&)

8. Kim S. H. et al. Carbamazepine-induced severe cutaneous adverse reactions and HLA genotypes in Koreans. Epilepsy Res. 97, 190–197 (2011).  [DOI](https://doi.org/10.1016/j.eplepsyres.2011.08.010) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21917426/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Epilepsy%20Res.&title=Carbamazepine-induced%20severe%20cutaneous%20adverse%20reactions%20and%20HLA%20genotypes%20in%20Koreans&author=S.%20H.%20Kim&volume=97&publication_year=2011&pages=190-197&pmid=21917426&doi=10.1016/j.eplepsyres.2011.08.010&)

9. Jaruthamsophon K., Sripo T., Sukasem C. & Limprasert P. Comparison of a new in-house and three published HLA-B*15:02 screening methods for prevention of carbamazepine-induced severe drug reactions. PLoS One 11, e0155907 (2016).  [DOI](https://doi.org/10.1371/journal.pone.0155907) | [PMC free article](/articles/PMC4873226/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27196420/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=Comparison%20of%20a%20new%20in-house%20and%20three%20published%20HLA-B*15:02%20screening%20methods%20for%20prevention%20of%20carbamazepine-induced%20severe%20drug%20reactions&author=K.%20Jaruthamsophon&author=T.%20Sripo&author=C.%20Sukasem&author=P.%20Limprasert&volume=11&publication_year=2016&pages=e0155907&pmid=27196420&doi=10.1371/journal.pone.0155907&)

10. Gonzalez-Galarza F. F. et al. Allele frequency net 2015 update: new features for HLA epitopes, KIR and disease and HLA adverse drug reaction associations. Nucleic Acids Res. 43, D784–D788 (2015).  [DOI](https://doi.org/10.1093/nar/gku1166) | [PMC free article](/articles/PMC4383964/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25414323/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&title=Allele%20frequency%20net%202015%20update:%20new%20features%20for%20HLA%20epitopes,%20KIR%20and%20disease%20and%20HLA%20adverse%20drug%20reaction%20associations&author=F.%20F.%20Gonzalez-Galarza&volume=43&publication_year=2015&pages=D784-D788&pmid=25414323&doi=10.1093/nar/gku1166&)

11. Roder G. et al. Crystal structures of two peptide-HLA-B*1501 complexes; structural characterization of the HLA-B62 supertype. Acta Crystallogr. D. Biol. Crystallogr. 62, 1300–1310 (2006).  [DOI](https://doi.org/10.1107/S0907444906027636) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17057332/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Acta%20Crystallogr.%20D.%20Biol.%20Crystallogr&title=Crystal%20structures%20of%20two%20peptide-HLA-B*1501%20complexes;%20structural%20characterization%20of%20the%20HLA-B62%20supertype&author=G.%20Roder&volume=62&publication_year=2006&pages=1300-1310&pmid=17057332&doi=10.1107/S0907444906027636&)

12. Puangpetch A. et al. HLA-B allele and haplotype diversity among Thai patients identified by PCR-SSOP: evidence for high risk of drug-induced hypersensitivity. Front. Genet. 5, 478 (2015).  [DOI](https://doi.org/10.3389/fgene.2014.00478) | [PMC free article](/articles/PMC4302987/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25657656/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front.%20Genet&title=HLA-B%20allele%20and%20haplotype%20diversity%20among%20Thai%20patients%20identified%20by%20PCR-SSOP:%20evidence%20for%20high%20risk%20of%20drug-induced%20hypersensitivity&author=A.%20Puangpetch&volume=5&publication_year=2015&pages=478&pmid=25657656&doi=10.3389/fgene.2014.00478&)

13. Wei C. Y., Chung W. H., Huang H. W., Chen Y. T. & Hung S. I. Direct interaction between HLA-B and carbamazepine activates T cells in patients with Stevens-Johnson syndrome. J. Allergy Clin. Immunol. 129, 1562–1569 (2012).  [DOI](https://doi.org/10.1016/j.jaci.2011.12.990) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22322005/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Allergy%20Clin.%20Immunol.&title=Direct%20interaction%20between%20HLA-B%20and%20carbamazepine%20activates%20T%20cells%20in%20patients%20with%20Stevens-Johnson%20syndrome&author=C.%20Y.%20Wei&author=W.%20H.%20Chung&author=H.%20W.%20Huang&author=Y.%20T.%20Chen&author=S.%20I.%20Hung&volume=129&publication_year=2012&pages=1562-1569&pmid=22322005&doi=10.1016/j.jaci.2011.12.990&)

14. Zhou P., Zhang S., Wang Y., Yang C. & Huang J. Structural modeling of HLA- B*1502/peptide/carbamazepine/T-cell receptor complex architecture: implication for the molecular mechanism of carbamazepine-induced Stevens-Johnson syndrome/toxic epidermal necrolysis. J. Biomol. Struct. Dyn. 34, 1806–1817 (2016).  [DOI](https://doi.org/10.1080/07391102.2015.1092476) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26488421/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Biomol.%20Struct.%20Dyn.&title=Structural%20modeling%20of%20HLA-%20B*1502/peptide/carbamazepine/T-cell%20receptor%20complex%20architecture:%20implication%20for%20the%20molecular%20mechanism%20of%20carbamazepine-induced%20Stevens-Johnson%20syndrome/toxic%20epidermal%20necrolysis&author=P.%20Zhou&author=S.%20Zhang&author=Y.%20Wang&author=C.%20Yang&author=J.%20Huang&volume=34&publication_year=2016&pages=1806-1817&pmid=26488421&doi=10.1080/07391102.2015.1092476&)

15. Smith C. A. & Kortemme T. Backrub-like backbone simulation recapitulates natural protein conformational variability and improves mutant side-chain prediction. J. Mol. Biol. 380, 742–756 (2008).  [DOI](https://doi.org/10.1016/j.jmb.2008.05.023) | [PMC free article](/articles/PMC2603262/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18547585/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Mol.%20Biol.&title=Backrub-like%20backbone%20simulation%20recapitulates%20natural%20protein%20conformational%20variability%20and%20improves%20mutant%20side-chain%20prediction&author=C.%20A.%20Smith&author=T.%20Kortemme&volume=380&publication_year=2008&pages=742-756&pmid=18547585&doi=10.1016/j.jmb.2008.05.023&)

16. Buchan D. W. A., Minneci F., Nugent T. C. O., Bryson K. & Jones D. T. Scalable web services for the PSIPRED Protein Analysis Workbench. Nucleic Acids Res. 41, W349–W357 (2013).  [DOI](https://doi.org/10.1093/nar/gkt381) | [PMC free article](/articles/PMC3692098/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23748958/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=Scalable%20web%20services%20for%20the%20PSIPRED%20Protein%20Analysis%20Workbench&author=D.%20W.%20A.%20Buchan&author=F.%20Minneci&author=T.%20C.%20O.%20Nugent&author=K.%20Bryson&author=D.%20T.%20Jones&volume=41&publication_year=2013&pages=W349-W357&pmid=23748958&doi=10.1093/nar/gkt381&)

17. Petersen B., Petersen T. N., Andersen P., Nielsen M. & Lundegaard C. A generic method for assignment of reliability scores applied to solvent accessibility predictions. BMC Struct. Biol. 9, 51 (2009).  [DOI](https://doi.org/10.1186/1472-6807-9-51) | [PMC free article](/articles/PMC2725087/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19646261/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Struct.%20Biol.&title=A%20generic%20method%20for%20assignment%20of%20reliability%20scores%20applied%20to%20solvent%20accessibility%20predictions&author=B.%20Petersen&author=T.%20N.%20Petersen&author=P.%20Andersen&author=M.%20Nielsen&author=C.%20Lundegaard&volume=9&publication_year=2009&pages=51&pmid=19646261&doi=10.1186/1472-6807-9-51&)

18. Case D. A. et al. AMBER 16, University of California, San Francisco (2016).  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=AMBER%2016&author=D.%20A.%20Case&publication_year=2016&)

19. Dolinsky T. J., Nielsen J. E., McCammon J. A. & Baker N. A. PDB2PQR: an automated pipeline for the setup of Poisson-Boltzmann electrostatics calculations. Nucleic Acids Res. 32, W665–W667 (2004).  [DOI](https://doi.org/10.1093/nar/gkh381) | [PMC free article](/articles/PMC441519/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15215472/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=PDB2PQR:%20an%20automated%20pipeline%20for%20the%20setup%20of%20Poisson-Boltzmann%20electrostatics%20calculations&author=T.%20J.%20Dolinsky&author=J.%20E.%20Nielsen&author=J.%20A.%20McCammon&author=N.%20A.%20Baker&volume=32&publication_year=2004&pages=W665-W667&pmid=15215472&doi=10.1093/nar/gkh381&)

20. Dolinsky T. J. et al. PDB2PQR: expanding and upgrading automated preparation of biomolecular structures for molecular simulations. Nucleic Acids Res. 35, W522–W525 (2007).  [DOI](https://doi.org/10.1093/nar/gkm276) | [PMC free article](/articles/PMC1933214/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17488841/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=PDB2PQR:%20expanding%20and%20upgrading%20automated%20preparation%20of%20biomolecular%20structures%20for%20molecular%20simulations&author=T.%20J.%20Dolinsky&volume=35&publication_year=2007&pages=W522-W525&pmid=17488841&doi=10.1093/nar/gkm276&)

21. Morris G. M. et al. AutoDock4 and AutoDockTools4: Automated docking with selective receptor flexiblity. J. Comput. Chem. 30, 2785–2791 (2009).  [DOI](https://doi.org/10.1002/jcc.21256) | [PMC free article](/articles/PMC2760638/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19399780/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Comput.%20Chem.&title=AutoDock4%20and%20AutoDockTools4:%20Automated%20docking%20with%20selective%20receptor%20flexiblity&author=G.%20M.%20Morris&volume=30&publication_year=2009&pages=2785-2791&pmid=19399780&doi=10.1002/jcc.21256&)
