# Effects of Human SULT1A3/SULT1A4 Genetic Polymorphisms on the Sulfation of Acetaminophen and Opioid Drugs by the Cytosolic Sulfotransferase SULT1A3

## Metadata
**Authors:** Ahsan F Bairam, Mohammed I Rasool, Fatemah A Alherz, Maryam S Abunnaja, Amal A El Daibani, Katsuhisa Kurogi, Ming-Cheh Liu
**Journal:** Archives of biochemistry and biophysics
**Date:** 2018 Apr 26
**DOI:** [10.1016/j.abb.2018.04.019](https://doi.org/10.1016/j.abb.2018.04.019)
**PMID:** 29705271
**PMCID:** PMC5969526
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5969526/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC5969526/pdf/nihms965997.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC5969526/pdf/nihms965997.pdf)

## Abstract

Sulfoconjugation has been shown to be critically involved in the metabolism of acetaminophen (APAP), morphine, tapentadol and O-desmethyl tramadol (O-DMT). The objective of this study was to investigate the effects of single nucleotide polymorphisms (SNPs) of human SULT1A3 and SULT1A4 genes on the sulfating activity of SULT1A3 allozymes toward these analgesic compounds. Twelve non-synonymous coding SNPs (cSNPs) of SULT1A3/SULT1A4 were investigated, and the corresponding cDNAs were generated by site-directed mutagenesis. SULT1A3 allozymes, bacterially expressed and purified, exhibited differential sulfating activity toward each of the four analgesic compounds tested as substrates. Kinetic analyses of SULT1A3 allozymes further revealed significant differences in binding affinity and catalytic activity toward the four analgesic compounds. Collectively, the results derived from the current study showed clearly the impact of cSNPs of the coding genes, SULT1A3 and SULT1A4, on the sulfating activity of the coded SULT1A3 allozymes toward the tested analgesic compounds. These findings may have implications in the pharmacokinetics as well as the toxicity profiles of these analgesics administered in individuals with distinct SULT1A3 and/or SULT1A4 genotypes.

Keywords: Single nucleotide polymorphisms, human SULT1A3, acetaminophen, tapentadol, morphine, O-desmethyl tramadol

## 1. Introduction

Acetaminophen (APAP) and opioids are frequently used, individually or in combination, for the clinical management of acute and chronic pain [[1](#R1)]. APAP is considered the safest and most popular drug prescribed as an analgesic and antipyretic [[2](#R2)]. Opioids, on the other hand, are the most commonly prescribed drugs in the US for acute pain management. As is widely known, problems concerning diversion, overdose, and addiction, associated with the use of opioids are rising [[3](#R3)]. Of the opioids that are in use, morphine, tapentadol, and tramadol have been shown to exert their action primarily via interaction with μ-opioid receptors [[4](#R4)–[6](#R6)].

Pharmacokinetic studies have revealed that the primary metabolic pathways of APAP in adults are glucuronidation and sulfoconjugation [[7](#R7)]. During prenatal and neonatal stages, however, sulfoconjugation constitutes the primary metabolic pathway of APAP due to low levels of UDP-glucuronosyltransferases (UGTs) [[8](#R8)]. For some opioids, such as morphine, tapentadol, and tramadol, sulfoconjugation has also been shown to be an important metabolic pathway during prenatal and neonatal stages, while glucuronidation plays a quantitatively more important role in adults [[9](#R9)–[12](#R12)]. Studies have demonstrated that sulfate conjugates of APAP and tapentadol are inactive metabolites [[13](#R13), [14](#R14)]. In the case of morphine, morphine-3-sulfate has been shown to exhibit little or no activity, whereas morphine-6-sulfate still possesses some analgesic activity [[15](#R15)]. For tramadol, *O*-desmethyl tramadol (*O*-DMT) has been shown to be an active metabolite, which is inactivated by sulfation with sulfated derivative excreted in the urine [[6](#R6)].

Sulfation as mediated by the cytosolic sulfotransferase (SULT) enzymes is considered a key step in the biotransformation and homeostasis of some key endogenous compounds such as catecholamines and thyroid/steroid hormones, as well as the detoxification of xenobiotics including drugs [[16](#R16)–[18](#R18)]. Of the thirteen known human SULTs [[19](#R19)], SULT1A3 has been shown to be a major enzyme responsible for the sulfation of morphine [[20](#R20)], APAP, tapentadol, and *O*-DMT [[21](#R21)–[23](#R23)]. Genomic studies have revealed that SULT1A3 is coded by two homologous genes, *SULT1A3* and *SULT1A4*, presumably derived from gene duplication during the evolutionary process, and both *SULT1A3* and *SULT1A4* genes are located on chromosome 16 [[24](#R24)–[26](#R26)]. Interestingly, single nucleotide polymorphisms (SNPs) of *SULT1A3* and *SULT1A4* have been reported [[25](#R25)]. It is possible that the SULT1A3 allozymes coded by missense SNPs of *SULT1A3* and *SULT1A4* may have differential sulfating activity toward APAP, morphine, tapentadol and *O*-DMT, thereby affecting their pharmacokinetics and thus their efficacy in individuals with different *SULT1A3* and *SULT1A4* genotypes.

In this study, we performed a comprehensive search for human *SULT1A3* and *SULT1A4* cSNPs. cDNAs corresponding to the thirteen *SULT1A3*/*SULT1A4* cSNPs identified were generated, and the coded SULT1A3 allozymes were bacterially expressed and purified by affinity chromatography. The twelve SULT1A3 allozymes that were successfully purified were analyzed for their enzymatic characteristics with APAP, morphine, tapentadol and *O*-DMT as substrates.

## 2. Materials and Methods

### 2.1. Materials

APAP, adenosine 5’-triphosphate (ATP), dimethyl sulfoxide (DMSO), dithiothreitol (DTT), and N-2-hydroxylpiperazine-N’-2-ethanesulfonic acid (HEPES) were products of Sigma Chemical Company. Morphine, tapentadol, and *O*-DMT were from Cayman Chemical. Cellulose thin-layer chromatography (TLC) plates were from Merck (EMD Millipore Corporation). Carrier-free sodium [^35^S]sulfate was from American Radiolabeled Chemicals. 3’-Phosphoadenosine-5’-phospho[^35^S]sulfate (PAP[^35^S]) was synthesized using ATP and carrier-free [^35^S]sulfate according to a previously established protocol [[27](#R27)]. X-Ray films were from Research Products International Corporation. Prime STAR^®^ GXL DNA Polymerase was a product of Clontech Laboratories, Inc. Protein molecular weight markers were from Bioland Scientific LLC. PCR kit was a product of G-Biosciences. QIAprep^®^ Spin Miniprep Kit was from QIAGEN. Ecolume scintillation cocktail was from MP Biomedical LLC. Glutathione Sepharose^TM^ was a product of GE Healthcare Bio-Sciences. All other chemicals were of the highest grades commercially available.

### 2.2. Database search

Three online databases, located at the websites of, respectively, the U.S. National Center for Biotechnology Information (NCBI), the UniProt Knowledgebase (UniProtKB), and previous genomic studies, were systematically searched for the non-synonymous cSNPs of the human *SULT1A3* and *SULT1A4* genes.

### 2.3. Generation, expression, and purification of SULT1A3 allozymes: Site-directed

mutagenesis, in conjunction with mutagenic primers (cf. [Table 1](#T1)), was employed to generate the cDNAs encoding different SULT1A3 allozymes based on a previously described procedure [[19](#R19)]. Authenticity of the “mutated” SULT1A3 cDNAs, packaged in pGEX-2TK prokaryotic expression vector, was verified by nucleotide sequencing [[28](#R28)]. To express SULT1A3 allozymes, “mutated” SULT1A3 cDNA/pGEX-2TK plasmids were individually transformed into competent BL21 *E. coli* cells. Upon induction of recombinant protein expression with IPTG, the cells were homogenized using an Aminco French Press. Recombinant SULT1A3 allozymes present in cell homogenates were purified using glutathione-Sepharose affinity chromatography based on a previously established procedure [[29](#R29)]. Purified recombinant SULT1A3 allozymes was analyzed for purity using SDS-polyacrylamide gel electrophoresis (SDS-PAGE) [[30](#R30),[31](#R31)]. Protein concentration of purified SULT1A3 allozymes was determined using Bradford protein assay [[32](#R32)].

### Table 1.

| SULT1A3 Allozymes | MAF1 | Nucleotide Change2 | Mutagenic Primers |
| --- | --- | --- | --- |
| SULT1A3-T7P | 0.00035323 | ACC ⇒ CCC | 5′- ATGGAGCTGATCCAGGACCCCTCCCGCCCGCCACTGG -3′5′- CCAGTGGCGGGCGGGAGGGGTCCTGGATCAGCTCCAT -3′ |
| SULT1A3-S8P | 0.00001123 | TCC ⇒ CCC | 5′- GAGCTGATCCAGGACACCCCCCGCCCGCCACTGGAGT -3′5′- ACTCCAGTGGCGGGCGGGGGGTGTCCTGGATCAGCTC -3′ |
| SULT1A3-R9C | 0.000066453 | CGC ⇒ TGC | 5′- CTGATCCAGGACACCTCCTGCCCGCCACTGGAGTACG -3′5′- CGTACTCCAGTGGCGGGCAGGAGGTGTCCTGGATCAG -3′ |
| SULT1A3-P10L | 0.000009143 | CCG ⇒ CTG | 5′- TCCAGGACACCTCCCGCCTGCCACTGGAGTACGTGAA -3′5′- TTCACGTACTCCAGTGGCAGGCGGGAGGTGTCCTGGA -3′ |
| SULT1A3-V15M | 0.000040583 | GTG ⇒ ATG | 5′- CGCCCGCCACTGGAGTACATGAAGGGGGTCCCGCTCA -3′5′- TGAGCGGGACCCCCTTCATGTACTCCAGTGGCGGGCG -3′ |
| SULT1A3-V18F | 0.00023 | GTC ⇒ TTC | 5′- CTGGAGTACGTGAAGGGGTTCCCGCTCATCAAGTACT -3′5′- AGTACTTGATGAGCGGGAACCCCTTCACGTACTCCAG -3′ |
| SULT1A3-P101L | 0.0254 | CCC ⇒ CTC | 5′- CTCTGAAAGACACACCGCTCCCACGGCTCATCAAGTC -3′5′- GACTTGATGAGCCGTGGGAGCGGTGTGTCTTTCAGAG -3′ |
| SULT1A3-P101H | 0.0044 | CCC ⇒ CAC | 5′- CTCTGAAAGACACACCGCACCCACGGCTCATCAAGTC -3′5′- GACTTGATGAGCCGTGGGTGCGGTGTGTCTTTCAGAG -3′ |
| SULT1A3-R144C | 0.0254 | CGT ⇒ TGT | 5′- TCCTACTACCATTTCCACTGTATGGAAAAGGCGCACC -3′5′- GGTGCGCCTTTTCCATACAGTGGAAATGGTAGTAGGA -3′ |
| SULT1A3-K234N | 0.0424 | AAG ⇒AAT | 5′- GTTCAAGGAGATGAAGAATAACCCTATGACCAACTAC -3′5′- GTAGTTGGTCATAGGGTTATTCTTCATCTCCTTGAAC -3′ |
| SULT1A3-N235T | - | AAC ⇒ ACC | 5′- TCAAGGAGATGAAGAAGACCCCTATGACCAACTACAC -3′5′- GTGTAGTTGGTCATAGGGGTCTTCTTCATCTCCTTGA -3′ |
| SULT1A3-S290T | - | AGC ⇒ ACC | 5′- TGGCAGGCTGCAGCCTCACCTTCCGCTCTGAGCTGTG -3′5′- CACAGCTCAGAGCGGAAGGTGAGGCTGCAGCCTGCCA -3′ |

Table 1 Caption: List of human SULT1A3 and SULT1A4 cSNPs, their minor allele frequencies, and mutagenic primer sets designed for the PCR-amplification of the corresponding cDNAs.

### 2.4. Enzymatic assay

The sulfating activity of SULT1A3 allozymes toward APAP, morphine, tapentadol, or *O*-DMT was analyzed using an established enzymatic assay procedure [[20](#R20)]. In an initial screening, three different concentrations of each of the four substrates were used, with radiolabeled PAP[^35^S] as the sulfate donor. The enzymatic assays were performed in 50 mM HEPES, pH 7.4, and allowed to proceed for 10 min at 37ºC, followed by TLC separation of the [^35^S]sulfated product present in the reaction mixture. Upon completion of TLC, autoradiography was performed to locate the [^35^S]sulfated product spot, which was then cut out and subjected to elution by H_2_O. [^35^S]-radioactivity associated with eluted [^35^S]sulfated product was measured using a liquid scintillation counter. The cpm count data obtained were used to calculate the specific activity in unit of nmol of sulfated product/min/mg of enzyme. In kinetic experiments, varying substrate concentrations (0, 50, 66.6, 100, 200, and 500 μM for APAP; 0, 400, 500, 666.6, 1000, and 2000 μM for morphine; 0, 10, 12.5, 16.6, 25, 50, and 100 μM for tapentadol; and 0, 25, 33.3, 50, 100, and 250 μM for *O*-DMT) were used based on the assay procedure described above.

### 2.5. Data analysis

Data obtained from the kinetic experiments were analyzed based on Michaelis-Menten kinetics to calculate the kinetic constants of wild-type and SULT1A3 allozymes in mediating the sulfation of tested substrate compounds. GraphPad Prism 7 software was used in data analysis.

## 3. Results

### 3.1. Analysis of human SULT1A3 and SULT1A4 single nucleotide polymorphisms

A systematic analysis was performed to search for different human *SULT1A3* and *SULT1A4* cSNPs deposited in two online databases located at the websites of the U.S. National Center for Biotechnology Information (NCBI) and the UniProt Knowledgebase (UniProtKB). *SULT1A3*/*SULT1A4* cSNPs reported in previous studies were included in the compiled cSNP list. A total of 10 missense cSNPs was identified for the *SULT1A3* gene, whereas 6 missense cSNPs were found for the *SULT1A4* gene. In between the missense cSNPs found for the two genes, 3 were found to code for same amino acid changes. As a result, 13 distinct missense *SULT1A3*/*SULT1A4* cSNPs remained at the conclusion of the analysis. The designated names and SNP ID numbers of these 13 cSNPs are: SULT1A3-T7P (Reference SNP (rs)776817009/rs754600221), SULT1A3-S8P (rs767263838), SULT1A3-R9C (rs762151655/rs752303630), SULT1A3-P10L (rs757573592), SULT1A3-V15M (rs750575779/ rs758881470), SULT1A3-V18F (rs553050853), SULT1A3-P19L (rs747088850), SULT1A3-P101L (rs751527244), SULT1A3-P101H, SULT1A3-R144C and SULT1A3-K234N [[25](#R25)], SULT1A3-N235T (UniProt [P0DMM9](https://www.ncbi.nlm.nih.gov/protein/P0DMM9)) and SULT1A3-S290T (UniProt [P0DMM9](https://www.ncbi.nlm.nih.gov/protein/P0DMM9)). The reported crystal structure of SULT1A3 [[33](#R33)] was used to demonstrate the location of the amino acid residues associated with these *SULT1A3*/*SULT1A4* cSNPs ([Figure 1](#F1); a color version of this figure is provided in the [Supplementary Data](#SD1) as [Figure S1](#SD1)). It is noted that two of the aforementioned amino acid residues (SULT1A3-K234N and SULT1A3-N235T) are positioned within the three loops, Asp66-Met77, Ser228-Gly259, and Lys85-Pro90, which play an essential role in the configuration of the gate that controls the substrate entry and selectivity [[34](#R34)]. Moreover, to help visualize the binding of the four analgesic substrate compounds with the SULT1A3 molecule, the substrate-binding pocket of SULT1A3 with superimposed dopamine (a prototype substrate) or analgesic substrates (APAP, morphine, tapentadol, and *O*-DMT), as well as the co-substrate, PAPS, were drawn and docked into the active site of the enzyme ([Figure S2A](#SD1)). [Figure S2B](#SD1) shows the hydrophilic and hydrophobic surfaces of the substrate-binding pocket, together with the substrate entry gate.

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a2f7/5969526/7ff12605203e/nihms965997f1.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5969526_nihms965997f1.jpg)

Ribbon diagram of the structure of human SULT1A3-analgesic substrate-PAP complex showing the locations of amino acid residues associated with the SULT1A3/SULT1A4 cSNPs. The structure of SULT1A3 (Protein Data Bank code: 2A3R [33]) was edited using USCF Chimera, a molecular modeling software [57]. Analgesic substrates, acetaminophen, morphine, tapentadol, and O-desmethyltramadol, and PAP in the structure are shown by bond structures. Analgesic substrates superimposed were docked into the active site of SULT1A3 using AutoDock Vina [58]. Loops 1, 2, and 3 refer to Asp66-Met77, Ser228-Gly259, and Lys85-Pro90 segments previously reported to form a gate for substrate entry [34]. Side chains of the amino acid residues associated with the SULT1A3/SULT1A4 cSNPs, Arg9, Pro10, Val15, Val18, Pro101, Arg144, Lys234, Asn235, Ser290, are indicated by bond structures.

### 3.2. Expression and purification of recombinant human SULT1A3 allozymes

SULT1A3 allozyme cDNAs ligated to pGEX-2TK prokaryotic expression vector, prepared via site-directed mutagenesis (see the Materials and Methods) were individually transformed into BL21 *E. coli* cells. Upon induction of recombinant protein expression by IPTG in transformed cells, glutathione-Sepharose affinity chromatography was performed to fractionate the recombinant SULT1A3 allozymes from the *E. coli* cell homogenates. Afterward, bovine thrombin was used to free the recombinant SULT1A3 allozymes from the bound GST fusion proteins. It is noted that of the 13 SULT1A3 allozymes expressed, one was found to be present in the inclusion body form, and thus could not be further purified. The twelve SULT1A3 allozymes that were purified were analyzed by SDS-polyacrylamide gel electrophoresis. As shown in [Figure 2](#F2), the apparent molecular weights of purified SULT1A3 allozymes were similar to that of the wild-type SULT1A3, which has a predicted molecular weight of 34,196.

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a2f7/5969526/92763e84cd60/nihms965997f2.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5969526_nihms965997f2.jpg)

SDS gel electrophoretic pattern of purified human SULT1A3 allozymes. SDS-PAGE was performed on a 12% gel, followed by Coomassie Blue staining. Samples analyzed in lanes 1 through 13 correspond to SULT1A3-WT (wild-type), SULT1A3-T7P, SULT1A3-S8P, SULT1A3-R9C, SULT1A3-P10L, SULT1A3-V15M, SULT1A3-V18F, SULT1A3-P101L, SULT1A3-P101H, SULT1A3-R144C, SULT1A3-K234N, SULT1A3-N235T and SULT1A3-S290T. Positions of protein molecular weight markers are indicated on the right.

### 3.3. Enzymatic characterization of the SULT1A3 allozymes

Purified SULT1A3 allozymes together with the wild-type enzyme were analyzed for their sulfating activity with APAP, morphine, tapentadol, and *O*-DMT as substrates. In an initial study, three different concentrations (one well below reported *Km*, one close to *Km*, and one well above *Km*) of each of the four substrates were tested in the enzymatic assays. The activity data shown in [Figures 3](#F3)–[6](#F6) are described below. It should be pointed out that considering the numerous steps involved in the sulfotransferase assay and the following TLC separation and scintillation counting, the data obtained should not be considered strictly quantitative, but rather semi-quantitative.

### Figure 3.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a2f7/5969526/3a80316989e7/nihms965997f3.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5969526_nihms965997f3.jpg)

Specific activities of the sulfation of APAP by human SULT1A3 allozymes. Concentrations of APAP used in the enzymatic assays were 100 μM (black), 600 μM (gray) and 1500 μM (white). Specific activity refers to nmol APAP sulfated/min/mg of purified allozyme. Data shown represent mean ± standard deviation derived from three determinations. WT refers to wild-type SULT1A3.

### Figure 6.

![Figure 6](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a2f7/5969526/380413bac3e4/nihms965997f6.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5969526_nihms965997f6.jpg)

Specific activities of the sulfation of O-DMT by human SULT1A3 allozymes. Concentrations of O-DMT used in the enzymatic assays were 25 μM (black), 125 μM (gray) and 600 μM (white). Specific activity refers to nmol O-DMT sulfated/min/mg of purified allozyme. Data shown represent mean ± standard deviation derived from three determinations. WT refers to wild-type SULT1A3.

#### With APAP as the substrate

At low and mid substrate concentrations (100 and 600 μM, respectively), similar patterns of APAP-sulfating activities were found for the SULT1A3 allozymes analyzed ([Figure 3](#F3)). Among them, SULT1A3-P101H showed a slightly higher specific activity than the wild-type enzyme, while the specific activities of SULT1A3-V15M and SULT1A3-K234N were comparable to that of the wild-type. The rest of the SULT1A3 allozymes all displayed lower specific activities compared with the wild-type enzyme, with SULT1A3-N235T exhibiting the lowest specific activity. At high substrate concentration (1500 μM), while SULT1A3-P101H still displayed a specific activity that was slightly higher than the wild-type, the specific activities of all other allozymes were lower than the wild-type enzyme, with SULT1A3-N235T exhibiting the lowest specific activity.

#### With morphine as the substrate

At all three substrate concentrations (250, 1000 and 2500 μM, respectively), SULT1A3-P101H displayed a specific activity nearly two times that of the wild-type enzyme ([Figure 4](#F4)). SULT1A3-P101L and SULT1A3-R144C showed specific activities comparable to that of the wild-type. The other nine SULT1A3 allozymes all displayed lower specific activities than the wild-type at varying degrees, with SULT1A3-N235T showing nearly null specific activities.

![Figure 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a2f7/5969526/43199036ce87/nihms965997f4.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5969526_nihms965997f4.jpg)

Specific activities of the sulfation of morphine by human SULT1A3 allozymes. Concentrations of morphine used in the enzymatic assays were 250 μM (black), 1000 μM (gray) and 2500 μM (white). Specific activity refers to nmol morphine sulfated/min/mg of purified allozyme. Data shown represent mean ± standard deviation derived from three determinations. WT refers to wild-type SULT1A3.

#### With tapentadol as the substrate

At all three substrate concentrations (5, 150, and 500 μM), two allozymes, SULT1A3-P101H and SULT1A3-R144C, showed higher specific activities when compared with the wild-type enzyme ([Figure 5](#F5)). The other ten SULT1A3 allozymes all exhibited lower specific activities than the wild-type. Of these ten allozymes, SULT1A3-N235T exhibited the lowest activities, which were approximately 12%, 14%, and 22% that of the wild-type, at 5, 150, and 500 μM tapentadol, respectively.

![Figure 5](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a2f7/5969526/c46fa38795ac/nihms965997f5.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5969526_nihms965997f5.jpg)

Specific activities of the sulfation of tapentadol by human SULT1A3 allozymes. Concentrations of tapentadol used in the enzymatic assays were 5 μM (black), 150 μM (gray) and 500 μM (white). Specific activity refers to nmol tapentadol sulfated/min/mg of purified allozyme. Data shown represent mean ± standard deviation derived from three determinations. WT refers to wild-type SULT1A3.

#### With O-DMT as the substrate

At all three substrate concentrations (25, 125 and 600 μM), SULT1A3-P101H and SULT1A3R144C again showed higher specific activities than the wild-type as well as all other allozymes ([Figure 6](#F6)). SULT1A3-P10L, SULT1A3-P101L and SULT1A3-S290T displayed specific activities comparable to that of the wild-type, whereas the other seven SULT1A3 allozymes displayed lower specific activities than the wild-type with SULT1A3-N235T again showing the lowest specific activity at all three substrate concentrations tested.

### 3.4. Kinetic Analysis

Kinetic experiments were performed to investigate further the differential enzymatic characteristics of the SULT1A3 allozymes. The results were analyzed based on Lineweaver-Burk double reciprocal plots to calculate the kinetic constants: *Km* (reflecting the substrate affinity)*, Vmax* (reflecting the catalytic activity), and *Vmax*/*Km* (reflecting the catalytic efficiency). The kinetic parameters determined for the wild-type and SULT1A3 allozymes are compiled in [Tables 2](#T2)–[5](#T5).

### Table 2.

| SULT1A3 Allozymes | Km (μM) | Vmax (nmol/min/mg) | Vmax/Km |
| --- | --- | --- | --- |
| 1A3-WT1 | 630 ± 40 | 41 ± 3 | 0.07 ± 0.01 |
| 1A3-T7P | 730 ± 40 | 38 ± 4 | 0.05 ± 0.01 |
| 1A3-S8P | 670 ± 40 | 34 ± 3 | 0.05 ± 0.01 |
| 1A3-R9C | 1250 ± 90 | 27 ± 6 | 0.02 ± 0.01 |
| 1A3-P10L | 970 ± 80 | 32 ± 2 | 0.03 ± 0.01 |
| 1A3-V15M | 720 ± 70 | 38 ± 3 | 0.05 ± 0.01 |
| 1A3-V18F | 780 ± 40 | 33 ± 3 | 0.04 ± 0.01 |
| 1A3-P101L | 980 ± 40 | 32 ± 2 | 0.03 ± 0.01 |
| 1A3-P101H | 430 ± 20 | 39 ± 3 | 0.09 ± 0.01 |
| 1A3-R144C | 1970 ± 120 | 28 ± 1 | 0.01 ± 0.01 |
| 1A3-K234N | 630 ± 40 | 29 ± 2 | 0.05 ± 0.02 |
| 1A3-N235T | 4500 ± 640 | 17 ± 1 | 0.004 ± 0.003 |
| 1A3-S290T | 1050 ± 120 | 30 ± 1 | 0.03 ± 0.01 |

Table 2 Caption: Kinetic parameters of the wild-type and SULT1A3 allozymes with acetaminophen as a substrate.

### Table 5.

| SULT1A3 Allozymes | Km (μM) | Vmax (nmol/min/mg) | Vmax/Km |
| --- | --- | --- | --- |
| 1A3-WT1 | 460 ± 50 | 16 ± 0.6 | 0.04 ± 0.01 |
| 1A3-T7P | 500 ± 70 | 6 ± 0.8 | 0.01 ± 0.01 |
| 1A3-S8P | 660 ± 90 | 4 ± 0.4 | 0.006 ± 0.001 |
| 1A3-R9C | 510 ± 40 | 7 ± 0.6 | 0.01 ± 0.01 |
| 1A3-P10L | 530 ± 90 | 16 ± 2 | 0.03 ± 0.01 |
| 1A3-V15M | 570 ± 90 | 12 ± 2 | 0.02 ± 0.01 |
| 1A3-V18F | 500 ± 30 | 6 ± 0.5 | 0.01 ± 0.01 |
| 1A3-P101L | 540 ± 50 | 16 ± 2 | 0.03 ± 0.01 |
| 1A3-P101H | 270 ± 30 | 24 ± 2 | 0.09 ± 0.01 |
| 1A3-R144C | 350 ± 60 | 22 ± 3 | 0.06 ± 0.01 |
| 1A3-K234N | 580 ± 60 | 12.5 ± 3 | 0.02 ± 0.01 |
| 1A3-N235T | 800 ± 70 | 0.03 ± 0.005 | 0.00004 ± 0.00001 |
| 1A3-S290T | 480 ± 60 | 15 ± 2 | 0.03 ± 0.01 |

Table 5 Caption: Kinetic parameters of the wild-type and SULT1A3 allozymes with O-desmethyl tramadol as a substrate.

#### With APAP as the substrate

As shown in [Table 2](#T2), SULT1A3-P101H was the only one among the twelve allozymes showing a lower *Km* value (430 ± 20 μM) than that (630 ± 40 μM) of the wild-type SULT1A3, while SULT1A3-K234N was the only one that exhibited a *Km* value comparable to that of the wild-type enzyme. The other ten SULT1A3 allozymes all showed higher *Km* values than the wild-type SULT1A3. Of them, SULT1A3-R9C, SULT1A3-R144C, SULT1A3-N235T, and SULT1A3-S290T allozymes displayed dramatically higher *Km* values (1250 ± 90, 1970 ± 120, 4500 ± 640, and 1050 ± 120 μM, respectively) when compared with the wild-type enzyme. In regard to *Vmax*, the wild-type SULT1A3 showed the highest value of 41 ± 3 nmol/min/mg. SULT1A3-N235T displayed a *Vmax* value of 17 ± 1 nmol/min/mg that was less than half that of the wild-type enzyme. Based on these results, the calculated *Vmax/Km* values showed that among the twelve SULT1A3 allozymes, only SULT1A3-P101H exhibited a *Vmax/Km* value 1.28 times that of the wild-type enzyme, while the other eleven allozymes displayed lower *Vmax/Km* values than the wild-type SULT1A3. Among these allozymes, SULT1A3-N235T allozyme showed the lowest *Vmax/Km* value, being more than 17 times less efficient than the wild-type enzyme.

#### With morphine as the substrate

The kinetic data shown in [Table 3](#T3) indicated that of the twelve SULT1A3 allozymes, SULT1A3-P101H display a lower *Km* (3800 ± 300 μM) than the wild-type SULT1A3, while all other allozymes showed *Km* values higher than that (4600 ± 400 μM) of the wild-type. Among the latter allozymes, SULT1A3-N235T allozyme displayed the lowest *Km* value (10,000 ± 900 μM). Regarding the *Vmax*, SULT1A3-P101H and SULT1A3- P101L displayed higher *Vmax* values (16 ± 2.5 and 11 ± 0.8 nmol/min/mg, respectively) than that (10 ± 0.6 nmol/min/mg) of the wild-type enzyme. Two allozymes, SULT1A3-T7P and SULT1A3-R144C, showed *Vmax* values (10 ± 1.0 and 10 ± 0.8 nmol/min/mg, respectively) comparable to that of the wild-type SULT1A3, while the remaining eight allozymes showed lower *Vmax* values than the wild-type. Notably, SULT1A3-N235T displayed the lowest *Vmax* values of 0.24 ± 0.03 nmol/min/mg. Based on these results, SULT1A3-P101H allozyme showed a *Vmax/Km* value that was 2 times higher than that of the wild-type SULT1A3, while the rest of allozymes all showed lower *Vmax/Km* values than the wild-type enzyme. Notably, the *Vmax/Km* value of SULT1A3-N235T allozyme was 100 times lower than that of the wild-type SULT1A3.

| SULT1A3 Allozymes | Km (μM) | Vmax (nmol/min/mg) | Vmax/Km |
| --- | --- | --- | --- |
| 1A3-WT1 | 4600 ± 400 | 10 ± 0.6 | 0.002 ± 0.001 |
| 1A3-T7P | 5700 ± 300 | 10 ± 1.0 | 0.002 ± 0.001 |
| 1A3-S8P | 7800 ± 400 | 9 ± 0.5 | 0.001 ± 0.001 |
| 1A3-R9C | 5000 ± 400 | 4 ± 0.2 | 0.0008 ± 0.0003 |
| 1A3-P10L | 5900 ± 500 | 8 ± 0.4 | 0.001 ± 0.001 |
| 1A3-V15M | 5000 ± 300 | 9 ± 0.9 | 0.002 ± 0.001 |
| 1A3-V18F | 5700 ± 600 | 8 ± 0.4 | 0.001 ± 0.001 |
| 1A3-P101L | 6400 ± 600 | 11 ± 0.8 | 0.002 ± 0.001 |
| 1A3-P101H | 3800 ± 300 | 16 ± 2.5 | 0.004 ± 0.001 |
| 1A3-R144C | 7200 ± 300 | 10 ± 0.8 | 0.001 ± 0.00007 |
| 1A3-K234N | 8300 ± 700 | 5 ± 0.4 | 0.0006 ± 0.0001 |
| 1A3-N235T | 10000 ± 900 | 0.24 ± 0.03 | 0.00002 ± 0.00001 |
| 1A3-S290T | 6200 ± 700 | 7 ± 0.7 | 0.001 ± 0.00003 |

Table Caption: Kinetic parameters of the wild-type and SULT1A3 allozymes with morphine as a substrate.

#### With tapentadol as the substrate

As shown in [Table 4](#T4), two allozymes, SULT1A3-P101H and SULT1A3-R144C, showed *Km* values (90 ± 10 and 110 ± 10 μM, respectively) lower than that (150 ± 10 μM) of the wild-type SULT1A3. The remaining ten SULT1A3 allozymes all displayed higher *Km* values than the wild-type enzyme. Among them, SULT1A3-N235T allozyme showed the lowest *Km* value (840 ± 70 μM). In regard to *Vmax*, SULT1A3-P101H and SULT1A3-R144C displayed *Vmax* values (30 ± 3 and 35 ± 2 nmol/min/mg, respectively) higher than that (27 ± 2 nmol/min/mg) of the wild-type enzyme, while the other ten allozymes all exhibited lower *Vmax* values. In particular, SULT1A3-N235T showed a *Vmax* value (13 ± 1 nmol/min/mg) which was less than half of that of the wild-type SULT1A3. Based on these results, two of the twelve SULT1A3 allozymes, SULT1A3-P101H and SULT1A3-R144C, showed higher *Vmax*/*Km* values (being 2 and 1.8 times, respectively) than the wild-type SULT1A3, while the other ten allozymes showed lower *Vmax*/*Km* values. Notably, SULT1A3-N235T exhibited a *Vmax*/*Km* value that was 8.5 times lower than the wild-type enzyme.

| SULT1A3 Allozymes | Km (μM) | Vmax (nmol/min/mg) | Vmax/Km |
| --- | --- | --- | --- |
| 1A3-WT1 | 150 ± 10 | 27 ± 2 | 0.17 ± 0.01 |
| 1A3-T7P | 190 ± 10 | 26 ± 1 | 0.14 ± 0.01 |
| 1A3-S8P | 480 ± 60 | 21 ± 1 | 0.04 ± 0.01 |
| 1A3-R9C | 340 ± 40 | 21 ± 2 | 0.06 ± 0.01 |
| 1A3-P10L | 320 ± 20 | 15 ± 1 | 0.05 ± 0.01 |
| 1A3-V15M | 380 ± 30 | 24 ± 0.8 | 0.06 ± 0.01 |
| 1A3-V18F | 320 ± 20 | 22 ± 0.5 | 0.07 ± 0.01 |
| 1A3-P101L | 240 ± 20 | 26 ± 2 | 0.11 ± 0.01 |
| 1A3-P101H | 90 ± 10 | 30 ± 3 | 0.35 ± 0.01 |
| 1A3-R144C | 110 ± 10 | 35 ± 2 | 0.31 ± 0.02 |
| 1A3-K234N | 550 ± 20 | 18 ± 0.3 | 0.03 ± 0.01 |
| 1A3-N235T | 840 ± 70 | 13 ± 1 | 0.02 ± 0.01 |
| 1A3-S290T | 210 ± 10 | 27 ± 1 | 0.13 ± 0.01 |

Table Caption: Kinetic parameters of the wild-type and SULT1A3 allozymes with tapentadol as a substrate.

#### With O-DMT as the substrate

As shown in [Table 5](#T5), two SULT1A3 allozymes, SULT1A3- P101H and SULT1A3-R144C, exhibited *Km* values (270 ± 30 and 350 ± 60 μM, respectively) lower than that (460 ± 50 μM) of the wild-type enzyme, while the rest of the allozymes all displayed higher *Km* values. Notably, SULT1A3-N235T showed the highest *Km* value (800 ± 65 μM) among these latter allozymes. In regard to *Vmax*, two allozymes, SULT1A3-P101H and SULT1A3-R144C, displayed *Vmax* values (24 ± 2 and 22 ± 3 nmol/min/mg) higher than that (16 ± 0.6 nmol/min/mg) of the wild-type enzyme, and three other allozymes (SULT1A3-P10L and SULT1A3-P101L, and SULT1A3-S290T) showed *Vmax* values comparable to that of the wild- type SULT1A3. The calculated *Vmax* values of the remaining seven allozymes were notably lower than wild-type. SULT1A3-N235T further exhibited the lowest *Vmax* value of only 0.03 ± 0.005 nmol/min/mg. Based on these results, the calculated *Vmax*/*Km* values of SULT1A3-P101H and SULT1A3-R144C allozymes were, respectively, 2.25 and 1.5 times that of the wild-type enzyme. In contrast, the *Vmax*/*Km* values of the other ten allozymes were all lower than the wild- type SULT1A3. Notably, SULT1A3-N235T showed a *Vmax*/*Km* value that was 1,000 times lower than the wild-type enzyme.

## 4. Discussion

APAP and opioids, administered individually or in combination, are the most commonly used analgesics in the United States and Europe [[1](#R1), [3](#R3), [35](#R35), [36](#R36)]. Because of their widespread use and potential adverse effects, it is important to understand better the mechanisms underlying individual differences in the metabolism and hence the efficacy and toxicity of these drugs. Inter-individual and ethnic variations in APAP and opioids metabolism have been reported [[5](#R5), [37](#R37)–[40](#R40)]. Studies have shown that genetic polymorphisms of APAP-metabolizing enzymes could be the cause for the differences in APAP metabolism and toxicity in different ethnic and racial groups [[38](#R38), [41](#R41), [42](#R42)]. It has been demonstrated that the analgesic activity and/or the side effect profiles of morphine and *O*-DMT depended on the genetically polymorphic enzyme cytochrome P450 2D6 [[43](#R43), [44](#R44)]. Previous studies have revealed that sulfation is critically involved in the metabolism of APAP, morphine, tapentadol and *O*-DMT, and that the sulfation pathway is quantitatively more important at pre-and postnatal stages than in adulthood [[8](#R8)–[12](#R12)]. Of the thirteen know human SULTs, SULT1A3 was shown to be a major enzyme responsible for the sulfation of these analgesic drugs [[20](#R20)–[23](#R23)]. In the current study, we first performed a comprehensive database search to identify missense cSNPs of human *SULT1A3* and *SULT1A4* genes that code for the identical SULT1A3 protein. We were able to express and purify twelve of the thirteen SULT1A3 allozymes identified. Purified SULT1A3 allozymes were analyzed for their sulfating activity with APAP, morphine, tapentadol, and *O*-DMT as substrates. Kinetic experiments were performed to further delineate the differential substrate-binding affinity and catalytic activity of these SULT1A3 allozymes.

Specific activity data shown in [Figures 3](#F3)–**6** revealed that compared with the wild-type, SULT1A3 allozymes displayed differential sulfating activities. Among them, SULT1A3-P101H allozyme exhibited consistently higher specific activities than the wild-type enzyme with all four analgesic substrates. A previous study, however, showed SULT1A3-P101H to display a lower activity than the wild-type toward ritodrine, a tocolytic agent [[19](#R19)]. Three other allozymes, SULT1A3-P101L, SULT1A3-R144C and SULT1A3-K234N, examined in the same study also displayed lower sulfating activity with ritodrine [[19](#R19)]. Our study indicated that while SULT1A3-P101L showed lower sulfating activities (approximately half of those of the wild-type enzyme) with APAP and tapentadol, it displayed comparable activity to that of wild-type with morphine and *O*-DMT as substrates. SULT1A3-R144C, on the other hand, exhibited a sulfating activity which was more than two times lower than the wild-type with APAP. Interestingly, it displayed comparable activity with morphine and higher activity with tapentadol or *O*-DMT than the wild-type. In the case of SULT1A3-K234N, while it displayed only a slightly lower sulfating activity with APAP or *O*-DMT, its activity with morphine was much (more than two times) lower with morphine or tapentadol. These differences in sulfating activity among different SULT1A3 allozymes are presumably due to the different chemical structures of the substrate compounds. Among all twelve SULT1A3 allozymes examined, SULT1A3-N235T showed the lowest sulfating activity toward all four analgesic compounds tested as substrates. Notably, this allozyme showed no detectable activity toward morphine or *O*-DMT. With APAP and tapentadol, its activity was 3.8–15.5% and 11.8–21.3%, respectively, those of the wild-type enzyme at the three substrate concentrations tested. Kinetic data compiled in [Tables 2](#T2)–**5** highlighted further the differences between SULT1A3 allozymes in substrate affinity and catalytic activity, and corroborated with the specific activity data shown in [Figures 3](#F3)–**6**.

Several crystal structures of the human SULT1A3 have been reported [[33](#R33), [45](#R45), [46](#R46)]. These studies have unveiled structural elements that are important in the catalysis (residue His108), the PAPS-binding (residues ^45^TYPKSGTT^52^, Arg130, Ser138, and ^257^RKG^259^), the substrate-binding/specificity (residues Asp86 and Glu146) [[33](#R33)], the N-terminal βA- and βB-sheets (residues Leu12-Val15 and Val18-Ile21, respectively) important in the polypeptide folding [[46](#R46), [47](#R47)], and the C-terminal dimerization motif (residues Lys265-Glu274, with a sequence motif KXXXTVXXXE) [[48](#R48)].

Six of the twelve SULT1A3 allozymes analyzed contain amino acid variations in the N-terminal region encompassing the above-mentioned βA- and βB-sheets, which have been proposed to be important in the polypeptide chain folding [[47](#R47)]. All these six allozymes have non-polar amino acids substitutions but with different characteristics, including non-turn-inducing vs. turn-inducing residues (SULT1A3-T7P, SULT1A3-S8P, and SULT1A3-P10L), aliphatic vs. thiol side chains (SULT1A3-R9C), aliphatic vs. S-methyl thioether side chain (SULT1A3-V15M), and non-aromatic vs. aromatic residues (SULT1A3-V18F). The majority of these allozymes showed lower specific activities than the wild-type enzyme with the four analgesic compounds as substrates, except SULT1A3-V15M and SULT1A3-P10L that showed comparable activity toward APAP and *O*-DMT, respectively. The minor variations between these allozymes could be attributed to the differences in the chemical structure of the four analgesic compounds. In SULT1A3-T7P and SULT1A3-S8P, a polar amino acid (Ser or Thr) is substituted with Pro, a turn-inducing amino acid residue. Such non-conservative amino acid substitutions probably induce unnecessary turn formation in the N-terminal region which might weaken the capacity of these two allozymes (SULT1A3-T7P and SULT1A3-S8P) in sulfating the four analgesic substrates. Studies have shown that tolerance to Pro substitution is not easily accommodated and the functional consequence may depend on the position of substitution in the overall structure [[49](#R49)]. SULT1A3-R9C allozyme involves a substitution of Arg with Cys at position 9 in the N-terminal region, which led to lower sulfating activities toward all four substrates. Arg is known to be a positively charged (basic) amino acid residue, which frequently forms salt-bridges with a negatively charged amino acid residue (Asp or Glu) that may be important for maintaining protein conformation and stability [[50](#R50)]. Such a role cannot be fulfilled by a non-polar amino acid like Cys, which potentially may lead to disulfide bond formation with other Cys residues in the same protein molecule or in multi-polypeptide complex as in the case of SULT enzymes [[51](#R51)]. For SULT1A3-P10L, the Pro residue, located at position 10 in the wild-type SULT1A3, is near the edge of the βA sheet in the N-terminal region. Previous studies have demonstrated that Pro is more frequently located at sharp turns such as at the edges of β-sheets, β-strands linking, kinks in transmembrane α-helices or within loops and disordered regions of proteins [[52](#R52)]. Substitution of Pro with Leu in SULT1A3-P10L rendered the allozyme less active than the wild-type enzyme toward three (APAP, morphine and tapentadol) of the four substrates tested. SULT1A3-V15M and SULT1A3-V18F involve the substitution of a valine residue with a S-methyl thioether side chain-containing or aromatic amino acid residue. Both these two allozymes displayed decreased sulfating activities toward the four tested analgesic substrates. It is noted that both Met and Phe possess larger side chains than Val, which may result in more restricted conformations. Collectively, the decreased sulfating activities of SULT1A3 allozymes with amino acid substitutions in the N-terminal region as elaborated above provided further support for the important structural role of the N-terminal βA- and βB-sheets in the SULT1A3 molecule as previously reported [[47](#R47)].

Three of the SULT1A3 allozymes examined, SULT1A3-P101L, SULT1A3-P101H and SULT1A3-R144C, involve amino acid substitutions close to the catalytic residue (His108) and/or substrate binding residues (residues Asp86 and Glu146). The location of the Pro residue at position 101, in a loop connecting α6 and βD, makes it not only close to the catalytic residue (His108) but also a part of the segment 84–104 that has been shown, together with residues 145–154, to be involved in substrate-binding and reshaping of the substrate binding pocket [[33](#R33)]. Ccompared with the wild-type enzyme, SULT1A3-P101L showed comparable sulfating activities with morphine and *O*-DMT as substrates and slightly lower sulfating activities toward APAP and tapentadol. In contrast, SULT1A3-P101H showed higher sulfating activities than the wild-type toward all four substrates tested, indicating that His residue at this location may be important in interacting with the four analgesic compounds tested as substrates. Indeed, SULT1A3-P101H exhibited lower *Km* values, and thus higher affinity, than the wild-type toward the four tested substrates. SULT1A3-R144C showed differential sulfating activity compared with the wild-type, being less active with APAP, equally active with morphine, and more activite with tapentadol and *O*-DMT. As discussed above, an Arg to Cys amino acid substitution may produce phenotype changes depending on the location in the protein molecule. As a result of this substitution, Cys may lead to disulfide-bond formation with other cysteine residues, instead of salt-bridges formed by Arg with negatively charged amino acids residues [[51](#R51)]. The location of Arg144 residue is within the 143–148 segment, which has been shown to play an important role in substrate-binding and catalysis of both human SULT1A1 and SULT1A3 [[53](#R53)]. The R144C substitution thus may lead to structural changes in the substrate-binding pocket, which in turn may affect the binding affinity for the substrate. That the four analgesic substrates tested vary in their chemical structures with differential distribution of functional groups may underscore the differences in sulfating activities of SULT1A3-R144C toward the four analgesic compounds.

Two of the tested allozymes, SULT1A3-K234N and SULT1A3-N235T, have amino acid substitutions close to the PAPS-binding site. SULT1A3-K234N showed lower sulfating activities than the wild-type enzyme with all three opioids (morphine, tapentadol and *O*-DMT), while SULT1A3-N235T exhibited lower sulfating activities with all four substrates. The amino acid substitutions of these two allozymes are located within the α15 sheet which has been proposed to contribute indirectly to the co-substrate (PAPS)-binding as well as restricting the conformations required for substrate-binding when the PAPS is bound to the protein molecule [[47](#R47)]. Moreover, the very low sulfating activity of SULT1A3-N235T might be related to the difficulty in accommodating the bulky side chain of the Thr residue into the α-helical segment of the SULT1A3 molecule. The presence of Asn235 has been shown to be important not only in SULT1A3, but also in SULT1A1 [[54](#R54), [55](#R55)]. In contrast, SULT1A3-S290T, with serine replaced by a hydroxyl group-containing threonine, showed no dramatic differences from the wild-type in sulfating all four analgesic compounds tested.

In summary, the current study aimed to gather information concerning the effects of the genetic polymorphisms of *SULT1A3* and *SULT1A4* on the APAP-, morphine-, tapentadol-, and *O*-DMT-sulfating activity of SULT1A3 allozymes. Specific activity and kinetic data obtained showed clearly the differential sulfating activities of SULT1A3 allozymes toward the four analgesic compounds tested as substrates. These findings may underscore the differential capacity in sulfating APAP, morphine, tapentadol and *O*-DMT in different individuals. Pending further studies, such information may in the future aid in designing personalized regimens of these analgesics to optimize their efficacy and mitigate the side effects for individuals with distinct *SULT1A3*/*SULT1A4* genotypes.

## Supplementary Material

## Acknowledgments

This work was supported in part by a grant from National Institutes of Health (Grant # R03HD071146).

## Abbreviations

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Li Wan Po A, Zhang WY. Systematic overview of co-proxamol to assess analgesic effects of addition of dextropropoxyphene to paracetamol. BMJ. 1997;315:1565–1571. doi: 10.1136/bmj.315.7122.1565.  [DOI](https://doi.org/10.1136/bmj.315.7122.1565) | [PMC free article](/articles/PMC2127964/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9437273/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMJ&title=Systematic%20overview%20of%20co-proxamol%20to%20assess%20analgesic%20effects%20of%20addition%20of%20dextropropoxyphene%20to%20paracetamol&author=A%20Li%20Wan%20Po&author=WY%20Zhang&volume=315&publication_year=1997&pages=1565-1571&pmid=9437273&doi=10.1136/bmj.315.7122.1565&)

2. Bertolini A, Ferrari A, Ottani A, Guerzoni S, Tacchi R, Leone S. Paracetamol: new vistas of an old drug. CNS Drug Rev. 2006;12:250–275. doi: 10.1111/j.1527-3458.2006.00250.x.  [DOI](https://doi.org/10.1111/j.1527-3458.2006.00250.x) | [PMC free article](/articles/PMC6506194/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17227290/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=CNS%20Drug%20Rev&title=Paracetamol:%20new%20vistas%20of%20an%20old%20drug&author=A%20Bertolini&author=A%20Ferrari&author=A%20Ottani&author=S%20Guerzoni&author=R%20Tacchi&volume=12&publication_year=2006&pages=250-275&pmid=17227290&doi=10.1111/j.1527-3458.2006.00250.x&)

3. Volkow ND, McLellan AT. Opioid Abuse in Chronic Pain-Misconceptions and Mitigation Strategies. N Engl J Med. 2016;374:1253–1263. doi: 10.1056/NEJMra1507771.  [DOI](https://doi.org/10.1056/NEJMra1507771) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27028915/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Opioid%20Abuse%20in%20Chronic%20Pain-Misconceptions%20and%20Mitigation%20Strategies&author=ND%20Volkow&author=AT%20McLellan&volume=374&publication_year=2016&pages=1253-1263&pmid=27028915&doi=10.1056/NEJMra1507771&)

4. Kelly E. Efficacy and ligand bias at the μ-opioid receptor. Br J Pharmacol. 2013;169:1430–1446. doi: 10.1111/bph.12222.  [DOI](https://doi.org/10.1111/bph.12222) | [PMC free article](/articles/PMC3724102/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23646826/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Pharmacol&title=Efficacy%20and%20ligand%20bias%20at%20the%20%CE%BC-opioid%20receptor&author=E%20Kelly&volume=169&publication_year=2013&pages=1430-1446&pmid=23646826&doi=10.1111/bph.12222&)

5. Fidman B, Nogid A. Role of Tapentadol Immediate Release (Nucynta) in the Management of Moderate-to-Severe Pain. P T. 2010;35:330–333. 357.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=P%20T&title=Role%20of%20Tapentadol%20Immediate%20Release%20(Nucynta)%20in%20the%20Management%20of%20Moderate-to-Severe%20Pain&author=B%20Fidman&author=A%20Nogid&volume=35&publication_year=2010&pages=330-333&)

6. Hennies HH, Friderichs E, Schneider J. Receptor binding, analgesic and antitussive potency of tramadol and other selected opioids. Arzneimittelforschung. 1988;38:877–880.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/2849950/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arzneimittelforschung&title=Receptor%20binding,%20analgesic%20and%20antitussive%20potency%20of%20tramadol%20and%20other%20selected%20opioids&author=HH%20Hennies&author=E%20Friderichs&author=J%20Schneider&volume=38&publication_year=1988&pages=877-880&pmid=2849950&)

7. Steventon GB, Mitchell SC, Waring RH. Human metabolism of paracetamol (acetaminophen) at different dose levels. Drug Metabol Drug Interact. 1996;13:111–1117. doi: 10.1515/dmdi.1996.13.2.111.  [DOI](https://doi.org/10.1515/dmdi.1996.13.2.111) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8905242/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metabol%20Drug%20Interact&title=Human%20metabolism%20of%20paracetamol%20(acetaminophen)%20at%20different%20dose%20levels&author=GB%20Steventon&author=SC%20Mitchell&author=RH%20Waring&volume=13&publication_year=1996&pages=111-1117&pmid=8905242&doi=10.1515/dmdi.1996.13.2.111&)

8. Adjei AA, Gaedigk A, Simon SD, Weinshilboum RM, Leeder JS. Interindividual variability in acetaminophen sulfation by human fetal liver: implications for pharmacogenetic investigations of drug-induced birth defects. Birth Defects Res A Clin Mol Teratol. 2008;82:155–165. doi: 10.1002/bdra.20535.  [DOI](https://doi.org/10.1002/bdra.20535) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18232020/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Birth%20Defects%20Res%20A%20Clin%20Mol%20Teratol&title=Interindividual%20variability%20in%20acetaminophen%20sulfation%20by%20human%20fetal%20liver:%20implications%20for%20pharmacogenetic%20investigations%20of%20drug-induced%20birth%20defects&author=AA%20Adjei&author=A%20Gaedigk&author=SD%20Simon&author=RM%20Weinshilboum&author=JS%20Leeder&volume=82&publication_year=2008&pages=155-165&pmid=18232020&doi=10.1002/bdra.20535&)

9. Besunder JB, Reed MD, Blumer JL. Principles of drug biodisposition in the neonate. A critical evaluation of the pharmacokinetic-pharmacodynamic interface (Part I) Clin Pharmacokinet. 1988;14:189–216. doi: 10.2165/00003088-198814040-00001.  [DOI](https://doi.org/10.2165/00003088-198814040-00001) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/3292100/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacokinet&title=Principles%20of%20drug%20biodisposition%20in%20the%20neonate.%20A%20critical%20evaluation%20of%20the%20pharmacokinetic-pharmacodynamic%20interface%20(Part%20I)&author=JB%20Besunder&author=MD%20Reed&author=JL%20Blumer&volume=14&publication_year=1988&pages=189-216&pmid=3292100&doi=10.2165/00003088-198814040-00001&)

10. Choonara I, Ekbom Y, Lindström B, Rane A. Morphine sulphation in children. Br J Clin Pharmacol. 1990;30:897–900. doi: 10.1111/j.1365-2125.1990.tb05458.x.  [DOI](https://doi.org/10.1111/j.1365-2125.1990.tb05458.x) | [PMC free article](/articles/PMC1368313/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/2288836/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Clin%20Pharmacol&title=Morphine%20sulphation%20in%20children&author=I%20Choonara&author=Y%20Ekbom&author=B%20Lindstr%C3%B6m&author=A%20Rane&volume=30&publication_year=1990&pages=897-900&pmid=2288836&doi=10.1111/j.1365-2125.1990.tb05458.x&)

11. Terlinden R, Ossig J, Fliegert F, Lange C, Göhler K. Absorption, metabolism, and excretion of 14C-labeled tapentadol HCl in healthy male subjects. Eur J Drug Metab Pharmacokinet. 2007;32:163–169. doi: 10.1007/BF03190478.  [DOI](https://doi.org/10.1007/BF03190478) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18062408/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Drug%20Metab%20Pharmacokinet&title=Absorption,%20metabolism,%20and%20excretion%20of%2014C-labeled%20tapentadol%20HCl%20in%20healthy%20male%20subjects&author=R%20Terlinden&author=J%20Ossig&author=F%20Fliegert&author=C%20Lange&author=K%20G%C3%B6hler&volume=32&publication_year=2007&pages=163-169&pmid=18062408&doi=10.1007/BF03190478&)

12. Wu WN, McKown LA, Liao S. Metabolism of the analgesic drug ULTRAM (tramadol hydrochloride) in humans: API-MS and MS/MS characterization of metabolites. Xenobiotica. 2002;32:411–425. doi: 10.1080/00498250110113230.  [DOI](https://doi.org/10.1080/00498250110113230) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12065063/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Xenobiotica&title=Metabolism%20of%20the%20analgesic%20drug%20ULTRAM%20(tramadol%20hydrochloride)%20in%20humans:%20API-MS%20and%20MS/MS%20characterization%20of%20metabolites&author=WN%20Wu&author=LA%20McKown&author=S%20Liao&volume=32&publication_year=2002&pages=411-425&pmid=12065063&doi=10.1080/00498250110113230&)

13. Prescott LF. Paracetamol overdosage. Pharmacological considerations and clinical management. Drugs. 1983;25:290–314. doi: 10.2165/00003495-198325030-00002.  [DOI](https://doi.org/10.2165/00003495-198325030-00002) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/6343056/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drugs&title=Paracetamol%20overdosage.%20Pharmacological%20considerations%20and%20clinical%20management&author=LF%20Prescott&volume=25&publication_year=1983&pages=290-314&pmid=6343056&doi=10.2165/00003495-198325030-00002&)

14. Schneider J, Jahnel U, Linz K. Neutral effects of the novel analgesic tapentadol on cardiac repolarization due to mixed ion channel inhibitory activities. Drug Dev Res. 2010;71:197–208.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Dev%20Res&title=Neutral%20effects%20of%20the%20novel%20analgesic%20tapentadol%20on%20cardiac%20repolarization%20due%20to%20mixed%20ion%20channel%20inhibitory%20activities&author=J%20Schneider&author=U%20Jahnel&author=K%20Linz&volume=71&publication_year=2010&pages=197-208&)

15. Andersson M, Björkhem-Bergman L, Ekström L, Bergqvist L, Lagercrantz H, Rane A, et al. Detection of morphine-3-sulfate and morphine-6-sulfate in human urine and plasma, and formation in liver cytosol. Pharmacol Res Perspect. 2014;2:e00071. doi: 10.1002/prp2.71.  [DOI](https://doi.org/10.1002/prp2.71) | [PMC free article](/articles/PMC4186450/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25505615/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacol%20Res%20Perspect&title=Detection%20of%20morphine-3-sulfate%20and%20morphine-6-sulfate%20in%20human%20urine%20and%20plasma,%20and%20formation%20in%20liver%20cytosol&author=M%20Andersson&author=L%20Bj%C3%B6rkhem-Bergman&author=L%20Ekstr%C3%B6m&author=L%20Bergqvist&author=H%20Lagercrantz&volume=2&publication_year=2014&pages=e00071&pmid=25505615&doi=10.1002/prp2.71&)

16. Mulder GJ, Jakoby WB. Sulfation in conjugation reactions. In: Mulder GJ, Jakoby WB, editors. Drug Metabolism. Taylor and Francis; London: 1990. pp. 107–161.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Drug%20Metabolism&author=GJ%20Mulder&author=WB%20Jakoby&publication_year=1990&)

17. Falany C, Roth JA. Properties of human cytosolic sulfotransferases involved in drug metabolism. In: Jeffery EH, editor. Human Drug Metabolism; From Molecular Biology to Man. CRC Press; FL: 1993. pp. 101–115.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Human%20Drug%20Metabolism;%20From%20Molecular%20Biology%20to%20Man&author=C%20Falany&author=JA%20Roth&publication_year=1993&)

18. Weinshilboum R, Otterness D. Sulfotransferase enzymes. In: Kaufmann FC, editor. Conjugation-Deconjugation Reactions in Drug Metabolism and Toxicity. Springer-Verlag; Berlin: 1994. pp. 45–78.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Conjugation-Deconjugation%20Reactions%20in%20Drug%20Metabolism%20and%20Toxicity&author=R%20Weinshilboum&author=D%20Otterness&publication_year=1994&)

19. Hui Y, Liu MC. Sulfation of ritodrine by the human cytosolic sulfotransferases (SULTs): Effects of SULT1A3 genetic polymorphism. Eur J Pharmacol. 2015;5:125–129. doi: 10.1016/j.ejphar.2015.04.039.  [DOI](https://doi.org/10.1016/j.ejphar.2015.04.039) | [PMC free article](/articles/PMC4532560/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25941087/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Pharmacol&title=Sulfation%20of%20ritodrine%20by%20the%20human%20cytosolic%20sulfotransferases%20(SULTs):%20Effects%20of%20SULT1A3%20genetic%20polymorphism&author=Y%20Hui&author=MC%20Liu&volume=5&publication_year=2015&pages=125-129&pmid=25941087&doi=10.1016/j.ejphar.2015.04.039&)

20. Kurogi K, Chepak A, Hanrahan MT, Liu MY, Sakakibara Y, Suiko M, Liu MC. Sulfation of opioid drugs by human cytosolic sulfotransferases: metabolic labeling study and enzymatic analysis. Eur J Pharm Sci. 2014;1:40–48. doi: 10.1016/j.ejps.2014.05.003.  [DOI](https://doi.org/10.1016/j.ejps.2014.05.003) | [PMC free article](/articles/PMC4459640/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24832963/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Pharm%20Sci&title=Sulfation%20of%20opioid%20drugs%20by%20human%20cytosolic%20sulfotransferases:%20metabolic%20labeling%20study%20and%20enzymatic%20analysis&author=K%20Kurogi&author=A%20Chepak&author=MT%20Hanrahan&author=MY%20Liu&author=Y%20Sakakibara&volume=1&publication_year=2014&pages=40-48&pmid=24832963&doi=10.1016/j.ejps.2014.05.003&)

21. Yamamoto A, Liu MY, Kurogi K, Sakakibara Y, Saeki Y, Suiko M, Liu MC. Sulphation of acetaminophen by the human cytosolic sulfotransferases: a systematic analysis. J Biochem. 2015;158:497–504. doi: 10.1093/jb/mvv062.  [DOI](https://doi.org/10.1093/jb/mvv062) | [PMC free article](/articles/PMC4819960/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26067475/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biochem&title=Sulphation%20of%20acetaminophen%20by%20the%20human%20cytosolic%20sulfotransferases:%20a%20systematic%20analysis&author=A%20Yamamoto&author=MY%20Liu&author=K%20Kurogi&author=Y%20Sakakibara&author=Y%20Saeki&volume=158&publication_year=2015&pages=497-504&pmid=26067475&doi=10.1093/jb/mvv062&)

22. Bairam AF, Rasool MI, Kurogi K, Liu MC. On the Molecular Basis Underlying the Metabolism of Tapentadol Through Sulfation. Eur J Drug Metab Pharmacokinet. 2017;42:793–800. doi: 10.1007/s13318-016-0392-8.  [DOI](https://doi.org/10.1007/s13318-016-0392-8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28070880/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Drug%20Metab%20Pharmacokinet&title=On%20the%20Molecular%20Basis%20Underlying%20the%20Metabolism%20of%20Tapentadol%20Through%20Sulfation&author=AF%20Bairam&author=MI%20Rasool&author=K%20Kurogi&author=MC%20Liu&volume=42&publication_year=2017&pages=793-800&pmid=28070880&doi=10.1007/s13318-016-0392-8&)

23. Rasool MI, Bairam AF, Kurogi K, Liu MC. On the sulfation of O-desmethyltramadol by human cytosolic sulfotransferases. Pharmacol Rep. 2017;69:953–958. doi: 10.1016/j.pharep.2017.02.014.  [DOI](https://doi.org/10.1016/j.pharep.2017.02.014) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28802998/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacol%20Rep&title=On%20the%20sulfation%20of%20O-desmethyltramadol%20by%20human%20cytosolic%20sulfotransferases&author=MI%20Rasool&author=AF%20Bairam&author=K%20Kurogi&author=MC%20Liu&volume=69&publication_year=2017&pages=953-958&pmid=28802998&doi=10.1016/j.pharep.2017.02.014&)

24. Dooley TP. Cloning of the human phenol sulfotransferase gene family: three genes implicated in the metabolism of catecholamines, thyroid hormones and drugs. Chem Biol Interact. 1998;109:29–41. doi: 10.1016/s0009-2797(97)00118-x.  [DOI](https://doi.org/10.1016/s0009-2797(97)00118-x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9566731/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Chem%20Biol%20Interact&title=Cloning%20of%20the%20human%20phenol%20sulfotransferase%20gene%20family:%20three%20genes%20implicated%20in%20the%20metabolism%20of%20catecholamines,%20thyroid%20hormones%20and%20drugs&author=TP%20Dooley&volume=109&publication_year=1998&pages=29-41&pmid=9566731&doi=10.1016/s0009-2797(97)00118-x&)

25. Hildebrandt MA, Salavaggione OE, Martin YN, Flynn HC, Jalal S, Wieben ED, et al. Human SULT1A3 pharmacogenetics: gene duplication and functional genomic studies. Biochem Biophys Res Commun. 2004;321:870–878. doi: 10.1016/j.bbrc.2004.07.038.  [DOI](https://doi.org/10.1016/j.bbrc.2004.07.038) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15358107/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem%20Biophys%20Res%20Commun&title=Human%20SULT1A3%20pharmacogenetics:%20gene%20duplication%20and%20functional%20genomic%20studies&author=MA%20Hildebrandt&author=OE%20Salavaggione&author=YN%20Martin&author=HC%20Flynn&author=S%20Jalal&volume=321&publication_year=2004&pages=870-878&pmid=15358107&doi=10.1016/j.bbrc.2004.07.038&)

26. Gamage N, Barnett A, Hempel N, Duggleby RG, Windmill KF, Martin JL, et al. Human sulfotransferases and their role in chemical metabolism. Toxicol Sci. 2006;90:5–22. doi: 10.1093/toxsci/kfj061.  [DOI](https://doi.org/10.1093/toxsci/kfj061) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16322073/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Toxicol%20Sci&title=Human%20sulfotransferases%20and%20their%20role%20in%20chemical%20metabolism&author=N%20Gamage&author=A%20Barnett&author=N%20Hempel&author=RG%20Duggleby&author=KF%20Windmill&volume=90&publication_year=2006&pages=5-22&pmid=16322073&doi=10.1093/toxsci/kfj061&)

27. Yanagisawa K, Sakakibara Y, Suiko M, Takami Y, Nakayama T, Nakajima H, et al. cDNA cloning, expression, and characterization of the human bifunctional ATP sulfurylase/adenosine 5'-phosphosulfate kinase enzyme. Biosci Biotechnol Biochem. 1998;62:1037–1040. doi: 10.1271/bbb.62.1037.  [DOI](https://doi.org/10.1271/bbb.62.1037) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9648242/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biosci%20Biotechnol%20Biochem&title=cDNA%20cloning,%20expression,%20and%20characterization%20of%20the%20human%20bifunctional%20ATP%20sulfurylase/adenosine%205'-phosphosulfate%20kinase%20enzyme&author=K%20Yanagisawa&author=Y%20Sakakibara&author=M%20Suiko&author=Y%20Takami&author=T%20Nakayama&volume=62&publication_year=1998&pages=1037-1040&pmid=9648242&doi=10.1271/bbb.62.1037&)

28. Sanger F, Nicklen S, Coulson AR. DNA sequencing with chain-terminating inhibitors. Proc Natl Acad Sci USA. 1977;74:5463–5467. doi: 10.1073/pnas.74.12.5463.  [DOI](https://doi.org/10.1073/pnas.74.12.5463) | [PMC free article](/articles/PMC431765/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/271968/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20USA&title=DNA%20sequencing%20with%20chain-terminating%20inhibitors&author=F%20Sanger&author=S%20Nicklen&author=AR%20Coulson&volume=74&publication_year=1977&pages=5463-5467&pmid=271968&doi=10.1073/pnas.74.12.5463&)

29. Harper S, Speicher DW. Purification of proteins fused to glutathione S-transferase. Methods Mol Biol. 2011;681:259–280. doi: 10.1007/978-1-60761-913-0_14.  [DOI](https://doi.org/10.1007/978-1-60761-913-0_14) | [PMC free article](/articles/PMC3584333/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20978970/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Methods%20Mol%20Biol&title=Purification%20of%20proteins%20fused%20to%20glutathione%20S-transferase&author=S%20Harper&author=DW%20Speicher&volume=681&publication_year=2011&pages=259-280&pmid=20978970&doi=10.1007/978-1-60761-913-0_14&)

30. Shapiro AL, Viñuela E, Maizel JJ. Molecular weight estimation of polypeptide chains by electrophoresis in SDS-polyacrylamide gels. Biochem Biophys Res Commun. 1967;28:815–20. doi: 10.1016/0006-291x(67)90391-9.  [DOI](https://doi.org/10.1016/0006-291x(67)90391-9) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/4861258/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem%20Biophys%20Res%20Commun&title=Molecular%20weight%20estimation%20of%20polypeptide%20chains%20by%20electrophoresis%20in%20SDS-polyacrylamide%20gels&author=AL%20Shapiro&author=E%20Vi%C3%B1uela&author=JJ%20Maizel&volume=28&publication_year=1967&pages=815-20&pmid=4861258&doi=10.1016/0006-291x(67)90391-9&)

31. Laemmli UK. Cleavage of structural proteins during the assembly of the head of bacteriophage T4. Nature. 1970;15:680–685. doi: 10.1038/227680a0.  [DOI](https://doi.org/10.1038/227680a0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/5432063/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Cleavage%20of%20structural%20proteins%20during%20the%20assembly%20of%20the%20head%20of%20bacteriophage%20T4&author=UK%20Laemmli&volume=15&publication_year=1970&pages=680-685&pmid=5432063&doi=10.1038/227680a0&)

32. Bradford MM. A rapid and sensitive method for the quantitation of microgram quantities of protein utilizing the principle of protein-dye binding. Anal Biochem. 1976;7:248–54. doi: 10.1016/0003-2697(76)90527-3.  [DOI](https://doi.org/10.1016/0003-2697(76)90527-3) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/942051/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Anal%20Biochem&title=A%20rapid%20and%20sensitive%20method%20for%20the%20quantitation%20of%20microgram%20quantities%20of%20protein%20utilizing%20the%20principle%20of%20protein-dye%20binding&author=MM%20Bradford&volume=7&publication_year=1976&pages=248-54&pmid=942051&doi=10.1016/0003-2697(76)90527-3&)

33. Lu JH, Li HT, Liu MC, Zhang JP, Li M, An XM, et al. Crystal structure of human sulfotransferase SULT1A3 in complex with dopamine and 3'-phosphoadenosine 5'-phosphate. Biochem Biophys Res Commun. 2005;335:417–423. doi: 10.1016/j.bbrc.2005.07.091.  [DOI](https://doi.org/10.1016/j.bbrc.2005.07.091) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16083857/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem%20Biophys%20Res%20Commun&title=Crystal%20structure%20of%20human%20sulfotransferase%20SULT1A3%20in%20complex%20with%20dopamine%20and%203'-phosphoadenosine%205'-phosphate&author=JH%20Lu&author=HT%20Li&author=MC%20Liu&author=JP%20Zhang&author=M%20Li&volume=335&publication_year=2005&pages=417-423&pmid=16083857&doi=10.1016/j.bbrc.2005.07.091&)

34. Cook I, Wang T, Almo SC, Kim J, Falany CN, Leyh TS. The gate that governs sulfotransferase selectivity. Biochemistry. 2013;52:415–424. doi: 10.1021/bi301492j.  [DOI](https://doi.org/10.1021/bi301492j) | [PMC free article](/articles/PMC3549632/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23256751/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochemistry&title=The%20gate%20that%20governs%20sulfotransferase%20selectivity&author=I%20Cook&author=T%20Wang&author=SC%20Almo&author=J%20Kim&author=CN%20Falany&volume=52&publication_year=2013&pages=415-424&pmid=23256751&doi=10.1021/bi301492j&)

35. Mangus BC, Miller MG. Pharmacology application in athletic training. FA Davis; Pennsylvania: 2005.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Pharmacology%20application%20in%20athletic%20training&author=BC%20Mangus&author=MG%20Miller&publication_year=2005&)

36. Aghababian RV. Essentials of emergency medicine. Jones & Bartlett Publishers; Ontario: 2010.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Essentials%20of%20emergency%20medicine&author=RV%20Aghababian&publication_year=2010&)

37. Zhao L, Pickering G. Paracetamol metabolism and related genetic differences. Drug Metab Rev. 2011;43:41–52. doi: 10.3109/03602532.2010.527984.  [DOI](https://doi.org/10.3109/03602532.2010.527984) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21108564/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Rev&title=Paracetamol%20metabolism%20and%20related%20genetic%20differences&author=L%20Zhao&author=G%20Pickering&volume=43&publication_year=2011&pages=41-52&pmid=21108564&doi=10.3109/03602532.2010.527984&)

38. Critchley JA, Nimmo GR, Gregson CA, Woolhouse NM, Prescott LF. Inter-subject and ethnic differences in paracetamol metabolism. Br J Clin Pharmacol. 1986;22:649–57. doi: 10.1111/j.1365-2125.1986.tb02953.x.  [DOI](https://doi.org/10.1111/j.1365-2125.1986.tb02953.x) | [PMC free article](/articles/PMC1401198/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/3567011/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Clin%20Pharmacol&title=Inter-subject%20and%20ethnic%20differences%20in%20paracetamol%20metabolism&author=JA%20Critchley&author=GR%20Nimmo&author=CA%20Gregson&author=NM%20Woolhouse&author=LF%20Prescott&volume=22&publication_year=1986&pages=649-57&pmid=3567011&doi=10.1111/j.1365-2125.1986.tb02953.x&)

39. Halling J, Weihe P, Brosen K. CYP2D6 polymorphism in relation to tramadol metabolism: a study of faroese patients. Ther Drug Monit. 2008;30:271–275. doi: 10.1097/FTD.0b013e3181666b2f.  [DOI](https://doi.org/10.1097/FTD.0b013e3181666b2f) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18520597/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ther%20Drug%20Monit&title=CYP2D6%20polymorphism%20in%20relation%20to%20tramadol%20metabolism:%20a%20study%20of%20faroese%20patients&author=J%20Halling&author=P%20Weihe&author=K%20Brosen&volume=30&publication_year=2008&pages=271-275&pmid=18520597&doi=10.1097/FTD.0b013e3181666b2f&)

40. Darbari DS, Van Schaik RH, Capparelli EV, Rana S, McCarter R, Van den Anker J. UGT2B7 promoter variant −840G>A contributes to the variability in hepatic clearance of morphine in patients with sickle cell disease. Am J Hematol. 2008;83:200–202. doi: 10.1002/ajh.21051.  [DOI](https://doi.org/10.1002/ajh.21051) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17724700/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hematol&title=UGT2B7%20promoter%20variant%20%E2%88%92840G>A%20contributes%20to%20the%20variability%20in%20hepatic%20clearance%20of%20morphine%20in%20patients%20with%20sickle%20cell%20disease&author=DS%20Darbari&author=RH%20Van%20Schaik&author=EV%20Capparelli&author=S%20Rana&author=R%20McCarter&volume=83&publication_year=2008&pages=200-202&pmid=17724700&doi=10.1002/ajh.21051&)

41. Court MH, Peter I, Hazarika S, Vasiadi M, Greenblatt DJ, Lee WM. Acute Liver Failure Study Group. Candidate gene polymorphisms in patients with acetaminophen-induced acute liver failure. Drug Metab Dispos. 2014;42:28–32. doi: 10.1124/dmd.113.053546.  [DOI](https://doi.org/10.1124/dmd.113.053546) | [PMC free article](/articles/PMC3876784/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24104197/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Candidate%20gene%20polymorphisms%20in%20patients%20with%20acetaminophen-induced%20acute%20liver%20failure&author=MH%20Court&author=I%20Peter&author=S%20Hazarika&author=M%20Vasiadi&author=DJ%20Greenblatt&volume=42&publication_year=2014&pages=28-32&pmid=24104197&doi=10.1124/dmd.113.053546&)

42. Lee HS, Ti TY, Koh YK, Prescott LF. Paracetamol elimination in Chinese and Indians in Singapore. Eur J Clin Pharmacol. 1992;43:81–84. doi: 10.1007/BF02280759.  [DOI](https://doi.org/10.1007/BF02280759) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/1505615/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Clin%20Pharmacol&title=Paracetamol%20elimination%20in%20Chinese%20and%20Indians%20in%20Singapore&author=HS%20Lee&author=TY%20Ti&author=YK%20Koh&author=LF%20Prescott&volume=43&publication_year=1992&pages=81-84&pmid=1505615&doi=10.1007/BF02280759&)

43. Kirchheiner J, Schmidt H, Tzvetkov M, Keulen JT, Lötsch J, Roots I, Brockmöller J. Pharmacokinetics of codeine and its metabolite morphine in ultra-rapid metabolizers due to CYP2D6 duplication. Pharmacogenomics J. 2007;7:257–265. doi: 10.1038/sj.tpj.6500406.  [DOI](https://doi.org/10.1038/sj.tpj.6500406) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16819548/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=Pharmacokinetics%20of%20codeine%20and%20its%20metabolite%20morphine%20in%20ultra-rapid%20metabolizers%20due%20to%20CYP2D6%20duplication&author=J%20Kirchheiner&author=H%20Schmidt&author=M%20Tzvetkov&author=JT%20Keulen&author=J%20L%C3%B6tsch&volume=7&publication_year=2007&pages=257-265&pmid=16819548&doi=10.1038/sj.tpj.6500406&)

44. Williams J, Bonnet F. Analgesics in anesthetic practice. In: Hardman JG, Hopkins PM, Struys MMRF, editors. Oxford Textbook of Anaesthesia. Oxford University Press; Oxford: 2017. p. 314.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Oxford%20Textbook%20of%20Anaesthesia&author=J%20Williams&author=F%20Bonnet&publication_year=2017&)

45. Dajani R, Cleasby A, Neu M, Wonacott AJ, Jhoti H, Hood AM, et al. X-ray crystal structure of human dopamine sulfotransferase, SULT1A3. Molecular modeling and quantitative structure activity relationship analysis demonstrate a molecular basis for sulfotransferase substrate specificity. J Biol Chem. 1999;274:37862–37868. doi: 10.1074/jbc.274.53.37862.  [DOI](https://doi.org/10.1074/jbc.274.53.37862) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10608851/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=X-ray%20crystal%20structure%20of%20human%20dopamine%20sulfotransferase,%20SULT1A3.%20Molecular%20modeling%20and%20quantitative%20structure%20activity%20relationship%20analysis%20demonstrate%20a%20molecular%20basis%20for%20sulfotransferase%20substrate%20specificity&author=R%20Dajani&author=A%20Cleasby&author=M%20Neu&author=AJ%20Wonacott&author=H%20Jhoti&volume=274&publication_year=1999&pages=37862-37868&pmid=10608851&doi=10.1074/jbc.274.53.37862&)

46. Bidwell LM, McManus ME, Gaedigk A, Kakuta Y, Negishi M, Pedersen L, et al. Crystal structure of human catecholamine sulfotransferase. J Mol Biol. 1999;293:521–530. doi: 10.1006/jmbi.1999.3153.  [DOI](https://doi.org/10.1006/jmbi.1999.3153) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10543947/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Mol%20Biol&title=Crystal%20structure%20of%20human%20catecholamine%20sulfotransferase&author=LM%20Bidwell&author=ME%20McManus&author=A%20Gaedigk&author=Y%20Kakuta&author=M%20Negishi&volume=293&publication_year=1999&pages=521-530&pmid=10543947&doi=10.1006/jmbi.1999.3153&)

47. Allali-Hassani A, Pan PW, Dombrovski L, Najmanovich R, Tempel W, Dong A, et al. Structural and Chemical Profiling of the Human Cytosolic Sulfotransferases. PLoS Biol. 2007;5:e97. doi: 10.1371/journal.pbio.0050097.  [DOI](https://doi.org/10.1371/journal.pbio.0050097) | [PMC free article](/articles/PMC1847840/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17425406/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Biol&title=Structural%20and%20Chemical%20Profiling%20of%20the%20Human%20Cytosolic%20Sulfotransferases&author=A%20Allali-Hassani&author=PW%20Pan&author=L%20Dombrovski&author=R%20Najmanovich&author=W%20Tempel&volume=5&publication_year=2007&pages=e97&pmid=17425406&doi=10.1371/journal.pbio.0050097&)

48. Petrotchenko EV, Pedersen LC, Borchers CH, Tomer KB, Negishi M. The dimerization motif of cytosolic sulfotransferases. FEBS Letters. 2001;490:39–43. doi: 10.1016/s0014-5793(01)02129-9.  [DOI](https://doi.org/10.1016/s0014-5793(01)02129-9) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11172807/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=FEBS%20Letters&title=The%20dimerization%20motif%20of%20cytosolic%20sulfotransferases&author=EV%20Petrotchenko&author=LC%20Pedersen&author=CH%20Borchers&author=KB%20Tomer&author=M%20Negishi&volume=490&publication_year=2001&pages=39-43&pmid=11172807&doi=10.1016/s0014-5793(01)02129-9&)

49. Yohannan S, Yang D, Faham S, Boulting G, Whitelegge J, Bowie JU. Proline substitutions are not easily accommodated in a membrane protein. J Mol Biol. 2004;341:1–6. doi: 10.1016/j.jmb.2004.06.025.  [DOI](https://doi.org/10.1016/j.jmb.2004.06.025) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15312757/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Mol%20Biol&title=Proline%20substitutions%20are%20not%20easily%20accommodated%20in%20a%20membrane%20protein&author=S%20Yohannan&author=D%20Yang&author=S%20Faham&author=G%20Boulting&author=J%20Whitelegge&volume=341&publication_year=2004&pages=1-6&pmid=15312757&doi=10.1016/j.jmb.2004.06.025&)

50. Mendes MIS, Smith DEC, Vicente JB, De Almeida IT, Ben-Omran T, Salomons GS, et al. Small aminothiol compounds improve the function of Arg to Cys variant proteins: effect on the human cystathionine β-synthase p.R336C. Human Molecular Genetics. 2015;24:7339–7348. doi: 10.1093/hmg/ddv431.  [DOI](https://doi.org/10.1093/hmg/ddv431) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26464485/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Human%20Molecular%20Genetics&title=Small%20aminothiol%20compounds%20improve%20the%20function%20of%20Arg%20to%20Cys%20variant%20proteins:%20effect%20on%20the%20human%20cystathionine%20%CE%B2-synthase%20p.R336C&author=MIS%20Mendes&author=DEC%20Smith&author=JB%20Vicente&author=IT%20De%20Almeida&author=T%20Ben-Omran&volume=24&publication_year=2015&pages=7339-7348&pmid=26464485&doi=10.1093/hmg/ddv431&)

51. Bass RB, Butler SL, Chervitz SA, Gloor SL, Falke JJ. Use of Site-Directed Cysteine and Disulfide Chemistry to Probe Protein Structure and Dynamics: Applications to Soluble and Transmembrane Receptors of Bacterial Chemotaxis. Methods Enzymol. 2007;423:25–51. doi: 10.1016/S0076-6879(07)23002-2.  [DOI](https://doi.org/10.1016/S0076-6879(07)23002-2) | [PMC free article](/articles/PMC2896970/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17609126/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Methods%20Enzymol&title=Use%20of%20Site-Directed%20Cysteine%20and%20Disulfide%20Chemistry%20to%20Probe%20Protein%20Structure%20and%20Dynamics:%20Applications%20to%20Soluble%20and%20Transmembrane%20Receptors%20of%20Bacterial%20Chemotaxis&author=RB%20Bass&author=SL%20Butler&author=SA%20Chervitz&author=SL%20Gloor&author=JJ%20Falke&volume=423&publication_year=2007&pages=25-51&pmid=17609126&doi=10.1016/S0076-6879(07)23002-2&)

52. Reiersen H, Rees AR. The hunchback and its neighbours: proline as an environmental modulator. Trends Biochem Sci. 2001;26:679–684. doi: 10.1016/s0968-0004(01)01957-0.  [DOI](https://doi.org/10.1016/s0968-0004(01)01957-0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11701327/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Trends%20Biochem%20Sci&title=The%20hunchback%20and%20its%20neighbours:%20proline%20as%20an%20environmental%20modulator&author=H%20Reiersen&author=AR%20Rees&volume=26&publication_year=2001&pages=679-684&pmid=11701327&doi=10.1016/s0968-0004(01)01957-0&)

53. Chen G, Chen X. Arginine Residues in the Active Site of Human Phenol Sulfotransferase (SULT1A1) J Biol Chem. 2003;278:36358–36364. doi: 10.1074/jbc.M306045200.  [DOI](https://doi.org/10.1074/jbc.M306045200) | [PMC free article](/articles/PMC3118444/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12867416/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=Arginine%20Residues%20in%20the%20Active%20Site%20of%20Human%20Phenol%20Sulfotransferase%20(SULT1A1)&author=G%20Chen&author=X%20Chen&volume=278&publication_year=2003&pages=36358-36364&pmid=12867416&doi=10.1074/jbc.M306045200&)

54. Coughtrie MW. Sulfation through the looking glass--recent advances in sulfotransferase research for the curious. Pharmacogenomics J. 2002;2:297–308. doi: 10.1038/sj.tpj.6500117.  [DOI](https://doi.org/10.1038/sj.tpj.6500117) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12439736/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=Sulfation%20through%20the%20looking%20glass--recent%20advances%20in%20sulfotransferase%20research%20for%20the%20curious&author=MW%20Coughtrie&volume=2&publication_year=2002&pages=297-308&pmid=12439736&doi=10.1038/sj.tpj.6500117&)

55. Zhu X, Veronese ME, Iocco P, McManus ME. cDNA cloning and expression of a new form of human aryl sulfotransferase. Int J Biochem Cell Biol. 1996;28:565–571. doi: 10.1016/1357-2725(95)00164-6.  [DOI](https://doi.org/10.1016/1357-2725(95)00164-6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8697101/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Biochem%20Cell%20Biol&title=cDNA%20cloning%20and%20expression%20of%20a%20new%20form%20of%20human%20aryl%20sulfotransferase&author=X%20Zhu&author=ME%20Veronese&author=P%20Iocco&author=ME%20McManus&volume=28&publication_year=1996&pages=565-571&pmid=8697101&doi=10.1016/1357-2725(95)00164-6&)

56. Thomae BA, Rifki OF, Theobald MA, Eckloff BW, Wieben ED, Weinshilboum RM. Human catecholamine sulfotransferase (SULT1A3) pharmacogenetics: functional genetic polymorphism. J Neurochem. 2003;87:809–819. doi: 10.1046/j.1471-4159.2003.02027.x.  [DOI](https://doi.org/10.1046/j.1471-4159.2003.02027.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14622112/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Neurochem&title=Human%20catecholamine%20sulfotransferase%20(SULT1A3)%20pharmacogenetics:%20functional%20genetic%20polymorphism&author=BA%20Thomae&author=OF%20Rifki&author=MA%20Theobald&author=BW%20Eckloff&author=ED%20Wieben&volume=87&publication_year=2003&pages=809-819&pmid=14622112&doi=10.1046/j.1471-4159.2003.02027.x&)

57. Pettersen EF, Goddard TD, Huang CC, Couch GS, Greenblatt DM, Meng EC. UCSF Chimera--a visualization system for exploratory research and analysis. J Comput Chem. 2004;25:1605–1612. doi: 10.1002/jcc.20084.  [DOI](https://doi.org/10.1002/jcc.20084) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15264254/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Comput%20Chem&title=UCSF%20Chimera--a%20visualization%20system%20for%20exploratory%20research%20and%20analysis&author=EF%20Pettersen&author=TD%20Goddard&author=CC%20Huang&author=GS%20Couch&author=DM%20Greenblatt&volume=25&publication_year=2004&pages=1605-1612&pmid=15264254&doi=10.1002/jcc.20084&)

58. Trott O, Olson AJ. AutoDock Vina: improving the speed and accuracy of docking with a new scoring function, efficient optimization, and multithreading. J Comput Chem. 2010;31:455–461. doi: 10.1002/jcc.21334.  [DOI](https://doi.org/10.1002/jcc.21334) | [PMC free article](/articles/PMC3041641/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19499576/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Comput%20Chem&title=AutoDock%20Vina:%20improving%20the%20speed%20and%20accuracy%20of%20docking%20with%20a%20new%20scoring%20function,%20efficient%20optimization,%20and%20multithreading&author=O%20Trott&author=AJ%20Olson&volume=31&publication_year=2010&pages=455-461&pmid=19499576&doi=10.1002/jcc.21334&)
