# Genetic polymorphisms in the long noncoding RNA MIR2052HG offer a pharmacogenomic basis for the response of breast cancer patients to aromatase inhibitor therapy

## Metadata
**Authors:** James N Ingle, Fang Xie, Matthew J Ellis, Paul E Goss, Lois E Shepherd, Judith-Anne W Chapman, Bingshu E Chen, Michiaki Kubo, Yoichi Furukawa, Yukihide Momozawa, Vered Stearns, Kathleen I Pritchard, Poulami Barman, Erin E Carlson, Matthew P Goetz, Richard M Weinshilboum, Krishna R Kalari, Liewei Wang
**Journal:** Cancer research
**Date:** 2016 Oct 10
**DOI:** [10.1158/0008-5472.CAN-16-1371](https://doi.org/10.1158/0008-5472.CAN-16-1371)
**PMID:** 27758888
**PMCID:** PMC5135610
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5135610/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC5135610/pdf/nihms827024.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC5135610/pdf/nihms827024.pdf)

## Abstract

Genetic risks in breast cancer remain only partly understood. Here we report the results of a genome-wide association study of germline DNA from 4,658 women, including 252 women experiencing a breast cancer recurrence, who were entered on the MA.27 adjuvant trial comparing the aromatase inhibitors (AI) anastrozole and exemestane. Single nucleotide polymorphisms (SNP) of top significance were identified in the gene encoding MIR2052HG, a long noncoding RNA of unknown function. Heterozygous or homozygous individuals for variant alleles exhibited a ~40% or ~63% decrease, respectively, in the hazard of breast cancer recurrence relative to homozygous wild-type individuals. Functional genomic studies in lymphoblastoid cell lines and ERα-positive breast cancer cell lines showed that expression from MIR2052HG and the ESR1 gene encoding estrogen receptor-α (ERα) was induced by estrogen and AI in a SNP-dependent manner. Variant SNP genotypes exhibited increased ERα binding to estrogen response elements, relative to wild-type genotypes, a pattern that was reversed by AI treatment. Further, variant SNPs were associated with lower expression of MIR2052HG and ERα. RNAi-mediated silencing of MIR2052HG in breast cancer cell lines decreased ERα expression, cell proliferation and anchorage-independent colony formation. Mechanistic investigations revealed that MIR2052HG sustained ERα levels both by promoting AKT/FOXO3-mediated ESR1 transcription and by limiting ubiquitin-mediated, proteasome-dependent degradation of ERα. Taken together, our results define MIR2052HS as a functionally polymorphic gene that affects risks of breast cancer recurrence in women treated with AI. More broadly, our results offer a pharmacogenomic basis to understand differences in the response of breast cancer patients to AI therapy.

Keywords: GWAS, breast cancer, aromatase inhibitors, functional genomics, MIR2052HG

## Introduction

Breast cancer is the most common form of cancer in women both in the United States ([1](#R1)) and worldwide ([2](#R2)). Endocrine therapy is the most important treatment modality in the majority of women who have estrogen receptor α (ERα) positive breast cancer. Whereas tamoxifen, a selective ER modulator (SERM), has substantial value in reducing the risk of disease recurrence in women with ERα-positive early stage breast cancer ([3](#R3)), a recent meta-analysis demonstrated that, when compared directly as monotherapy, aromatase inhibitors (AIs) were superior to tamoxifen in terms of local recurrence, distant recurrence, contralateral recurrence, breast cancer mortality, and all-cause mortality ([4](#R4)). However, despite the clear efficacy of AIs as adjuvant therapy, this meta-analysis revealed that 19.1% of women treated with an AI, anastrozole or letrozole, experienced a recurrence of their breast cancer at 10 years and there was no indication of a plateau in the time to recurrence curve ([4](#R4)).

Canadian Cancer Trials Group MA.27 ([5](#R5)) is the largest adjuvant endocrine therapy trial that exclusively studied AIs. Postmenopausal women with hormone receptor-positive early stage breast cancer were randomized to the steroidal AI exemestane or the non-steroidal AI anastrozole, and no difference in efficacy was identified ([5](#R5)). We performed a genome-wide association study (GWAS), which indicated that germline genetic variability in SNP genotypes related to a gene encoding a lncRNA, may alter ERα expression and impact outcomes after treatment with AIs.. Functional genomic studies of this lncRNA and the SNPs related to it provided novel mechanisms by which the lncRNA might affect the level of AI benefit.

## Methods

### Source of Patients

Patients were obtained from the MA.27 trial (ClinicalTrials.gov number [NCT00066573](https://clinicaltrials.gov/ct2/show/NCT00066573)). MA.27 included postmenopausal women with histologically confirmed and completely resected stage I–III breast cancer (AJCC Version 6) that was ERα and/or PgR positive. Patients were randomized to five years of anastrozole or exemestane. Only North American patients were offered participation in collection of blood specimens and 5221 of 6827 (76.5%) of the North American patients contributed blood and gave consent for genetic testing. This research was performed after approval by local institutional review boards in accordance with assurances filed with, and approved by, the Department of Health and Human Services.

### Primary Outcome: Breast Cancer-Free Interval

The primary outcome was the STEEP endpoint of BCFI ([6](#R6)), defined as the time from randomization to the first local-regional breast cancer recurrence (including ipsilateral DCIS), distant breast cancer recurrence, contralateral breast cancer (invasive or DCIS) or death with or from breast cancer without a prior recurrence date. Follow-up was censored at non-breast cancer death, or longest follow-up without recurrence.

### Genotyping, Imputation, and Quality Control

Three cohorts of patients from MA.27 were genotyped by the RIKEN Center for Integrative Medical Sciences. Cohort 1 involved patients genotyped as part of a GWAS with musculoskeletal adverse events as the phenotype utilizing the Illumina Human610 Quad Beadchip ([7](#R7)). Cohort 2 involved patients genotyped as part of a GWAS with fragility bone fractures as the phenotype utilizing the Illumina Human OmniExpress platform ([8](#R8)). Cohort 3 involved the remainder of the patients from MA.27 with DNA and consent. The quality control measures for cohorts 1 and 2 have been published ([7](#R7), [8](#R8)). For cohort 3, the following measures were taken for quality control purposes. One case and three controls were randomly chosen as duplicates for quality control of genotype concordance. A Caucasian parent-child Centre d’Etude du Polymorphisme Human trio from the HapMap was included to check for Mendelian transmission of alleles. Genotypes were determined utilizing the Illumina Human OmniExpressExome platform. This platform provided genotype data for 964,193 SNPs of which 2,923 were removed because they were from chromosome Y, mitochondria, or unplaced chromosomes. Additionally, 40,631 SNPs failed genotyping and 250,843 were rare SNPs with MAF<0.01. Imputation was performed using EZimputer ([9](#R9)) across the three cohorts separately. We then combined the genotyping data from all three cohorts to perform the current GWAS. Since each of the three cohorts were genotyped with a different platform, there were some SNPs that were genotyped in some patients but not in others. EZimputer ([9](#R9)) ([http://www.mayo.edu/research/departments-divisions/department-health-sciences-research/division-biomedical-statistics-informatics/software/bioinformatics-software-packages](http://www.mayo.edu/research/departments-divisions/department-health-sciences-research/division-biomedical-statistics-informatics/software/bioinformatics-software-packages)) imputes both un-genotyped SNPs and missing SNPs on a given platform. SNPs selected from imputation by EZimputer ([9](#R9)) had a dosage r squared >0.8 [r squared here is defined as the estimated squared correlation between the estimated allele dosage (0*P(Hom Ref/first) + 1*P(AB) + 2*P(Hom Alt/second)) and the true allele dosage].

The data from this GWAS have been deposited in the Data Base of Genotypes and Phenotypes (dbGaP). The dbGaP Study Accession Number is phs001043 and the URL is [http://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs001043.v1.p1](http://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs001043.v1.p1).

### Deep Sequencing Methodology

We performed deep sequencing of the region containing the top imputed SNPs to determine the quality of the imputation as detailed in [Supplementary Materials](#SD1).

### Statistical Analyses

Exact Fisher tests were used to examine whether there were imbalances between North American patients who were included in this GWAS and those who were not. We also evaluated if any additional clinical variables were significantly associated with breast recurrence event using Stepwise selection method ([10](#R10)). Our analysis employed a stratified genome-wide Cox-proportional hazards model using significant stratification factors and the model was further controlled for additional covariates including treatment arm, cohort, race, ER/PgR status, T-stage, ECOG performance score and bisphosphonate use. To avoid biases that might arise from differences in genetic ancestry (i.e., population stratification), the EigenStrat software was used to determine eigenvalues for the SNP correlation matrix that statistically differed from zero based on Tracy-Widom p-values ([11](#R11)) ([12](#R12)). All the analyses were run using the R statistical computing package ([13](#R13)), PLINK ([14](#R14)), and SAS (SAS Institute, Cary, NC).

Haplotype analysis was performed using the top six SNPs. Haplotype probabilities for individual samples were estimated using haplo.stats package v1.7.7 in R v3.2.0 ([15](#R15)) (details in [Supplementary Materials](#SD1))

### Deep Sequencing Data Analysis

In order to determine the quality of imputation, we performed targeted deep sequencing of the 300kb region (chr8: 75,400,000–75,700,000) surrounding the chromosome 8 GWAS signal in a total of 997 patients (249 with and 748 without a breast event). We removed both 5′ and 3′-end primers with cutadapt-1.7.1 with minimum-length option = 20. The trimmed reads were aligned to the hg19 reference genome using BWA-MEM. SNVs and INDELs were called using HaplotypeCaller and genotyped across all samples with GenotypeGVCF from the Genome Analysis Tool Kit (GATK) ([16](#R16)–[18](#R18)). Variants were annotated with functional features, impact prediction, and clinical significance using SnpEff ([19](#R19)), ClinVar, HGMD, and ExAC population frequencies with the BioR annotation tool ([20](#R20)). To demonstrate the imputation quality of our GWAS SNPs, we correlated the variant calls from deep sequencing with a set of imputed SNPs from the GWAS signal.

### Cell lines

Human breast cancer cell lines CAMA-1, HCC1428, BT474, AU565, BT549 and human embryonic kidney cell line 293T were obtained from American Type Culture Collection (ATCC, Manassus, VA) in 2014 and the identities of all cell lines were confirmed by the medical genome facility at Mayo Clinic Center (Rochester MN) using short tandem repeat profiling upon receipt. The breast cancer cell line MCF7/AC1, stably overexpressing aromatase (stably transfected *CYP19A1* gene), was generously gifted from Angela H. Brodie, Ph. D. (University of Maryland, Baltimore, MD). The cells were authenticated in 2015 by Genetica DNA Laboratories (Cincinnati, OH) using a StemElite ID system that uses short tandem repeat genotyping. All the cells used in our studies were within the initial five passages.

### Functional Genomics Studies

Details with respect to materials used, cell culture and lymphoblastoid cell line (LCLs) techniques, quantitative real-time PCR assay (qRT-PCR), chromatin Immunoprecipitation (ChIP) assays, cell proliferation assays, colony forming assays, western blotting are given in the [Supplementary Materials](#SD1).

## Results

### Population of Patients Studied

The Participant Flow Diagram ([Supplementary Fig. S1](#SD2)) shows the patients included and excluded from the GWAS, and [Supplementary Table 1](#SD1) provides the patient characteristics and analyses revealed good comparability ([Supplementary Materials](#SD1)).

A total of 4,784 patients had sufficient DNA for genotyping but 4,658 patients were utilized for the GWAS following quality control procedures as specified in the [Supplementary Materials](#SD1) and in the Participant Flow Diagram ([21](#R21)) ([Supplementary Fig. S1](#SD2)). The analysis included 252 women with a breast event and 4406 women who had not experienced a breast event ([Table 1](#T1)). We did not find any eigenvectors that were significantly associated with BCFI.

### Table 1.

|   | Breast Event=No (n=4406) | Breast Event=Yes (n=252) | Total (n=4658) |
| --- | --- | --- | --- |
| Genotyped Cohort |  |  |  |
| Cohort 1 | 826 (18.7%) | 43 (17.1%) | 869 (18.7%) |
| Cohort 2 | 834 (18.9%) | 47 (18.7%) | 881 (18.9%) |
| Cohort 3 | 2746 (62.3%) | 162 (64.3%) | 2908 (62.4%) |
| Age at Randomization |  |  |  |
| Median | 64.4 | 64 | 64.4 |
| Q1, Q3 | 58.3, 71.4 | 58.5, 71.3 | 58.4, 71.4 |
| Range | 36.1 – 95.1 | 35.9 – 93.6 | 35.9 – 95.1 |
| Genotypic Race |  |  |  |
| African American | 134 (3.0%) | 6 (2.4%) | 140 (3.0%) |
| Caucasian | 4216 (95.7%) | 243 (96.4%) | 4459 (95.7%) |
| Asian | 56 (1.3%) | 3 (1.2%) | 59 (1.3%) |
| Treatment Arm |  |  |  |
| Anastrozole | 2210 (50.2%) | 127 (50.4%) | 2337 (50.2%) |
| Exemestane | 2196 (49.8%) | 125 (49.6%) | 2321 (49.8%) |
| Adjuvant Chemotherapy |  |  |  |
| No | 3143 (71.3%) | 124 (49.2%) | 3267 (70.1%) |
| Yes | 1263 (28.7%) | 128 (50.8%) | 1391 (29.9%) |
| ECOG Performance Score |  |  |  |
| 0 | 3694 (83.8%) | 190 (75.4%) | 3884 (83.4%) |
| 1+ | 712 (16.2%) | 62 (24.6%) | 774 (16.6%) |
| T-Stage |  |  |  |
| T1/TX | 3317 (75.3%) | 120 (47.6%) | 3437 (73.8%) |
| T2 | 997 (22.6%) | 115 (45.6%) | 1112 (23.9%) |
| T3 | 92 (2.1%) | 17 (6.8%) | 109 (2.3%) |
| Nodal Status |  |  |  |
| Negative (N0) | 3290 (74.7%) | 121 (48.0%) | 3411 (73.2%) |
| Positive (N1-3) | 1037 (23.5%) | 129 (51.2%) | 1166 (25%) |
| Unknown (NX) | 79 (1.79%) | 2 (0.8%) | 81 (1.7%) |
| BMI at baseline |  |  |  |
| Median | 28.5 | 28.4 | 28.5 |
| Q1, Q3 | 25.0, 32.8 | 24.9, 32.4 | 25.0, 32.8 |
| Range | 15.3 – 68.7 | 16.0 – 56.8 | 15.3 – 68.7 |
| Estrogen receptor, progesterone receptor |  |  |  |
| Positive, Positive | 3553 (80.6%) | 181 (71.8%) | 3734 (80.2%) |
| Positive, Negative | 732 (16.6%) | 61 (24.2%) | 793 (17%) |
| Positive, Missing | 99 (2.2%) | 6 (2.4%) | 105 (2.2%) |
| Negative, Positive | 22 (0.5%) | 4 (1.6%) | 26 (0.6%) |
| Bisphosphonate Use |  |  |  |
| No | 2949 (66.9%) | 196 (77.8%) | 3145 (67.5%) |
| Yes | 1457 (33.1%) | 56 (22.2%) | 1513 (32.5%) |
| Trastuzumab use |  |  |  |
| No | 859 (19.5%) | 27 (10.7%) | 886 (19%) |
| Unknown | 3507 (79.6%) | 223 (88.5%) | 3730 (80.1%) |
| Yes | 40 (0.9%) | 2 (0.8%) | 42 (0.9%) |
| Type of Recurrence |  |  |  |
| Any Distant |   | 170 (67.5%) |   |
| Local/Regional +/− Contra (no distant) |   | 42 (16.7%) |   |
| Contralateral Only |   | 40 (15.9%) |   |

Table 1 Caption: Patient Characteristics

[Table 1](#T1) shows that the percentage of patients with a breast event was similar within each of the three genotyped cohorts (Cohort 1: 4.9%, Cohort 2: 5.3%, Cohort 3: 5.6%). Patients with and without a breast event were well balanced for age, treatment arm (anastrozole versus exemestane), and baseline body mass index. There were significant imbalances for adjuvant chemotherapy (P=3.68e-12), T stage (P=<2e-16), lymph node status (p=<2e-16) Eastern Cooperative Oncology Group (ECOG) performance score (P=0.0003), bisphosphonate use (P=0.0004), and ER/progesterone receptor (PgR) (p=0.008) with patients with a breast event having a higher percentage of ER-positive/PgR-negative and a lower percentage of ER-positive/PgR-positive tumors.

### Genotyping and Imputation

Genotyping for Cohorts 1 and 2 has been described previously ([7](#R7), [8](#R8)) utilizing the Illumina Human610 Quad BeadChip and the Illumina HumanOmniExpress platforms, respectively. For Cohort 3, after quality control measures, 669,796 genotyped SNPs were available for combining with genotype data from Cohorts 1 and 2. Imputation was performed using an in-house method, EZimputer ([9](#R9)), which returned a total of 9.57M SNPs (observed plus imputed) with r squared >0.8, of which 7.4M (observed + imputed) SNPs had a MAF>0.01 and were used for the analyses.

### Genome-wide Association Study Analyses

We performed a stratified Cox-proportional hazards analysis utilizing stratification factors and other covariates as detailed in the Methods section. The Manhattan plot ([Fig. 1A](#F1)) shows that the SNPs with the lowest p-values mapped to chromosome 8 and the quantile-quantile plot ([Supplementary Fig. S2](#SD3)) revealed a lambda of 0.999. [Figure 1B](#F1) shows the locus zoom for the region with the top SNPs. Characteristics of the top six SNPs on chromosome 8 are listed in [Table 2](#T2) and these SNPs are all “favorable” in that they were associated with longer BCFI. All six SNPs were in strong linkage disequilibrium with R^2^ values ranging from 0.95 to 0.99. The hazard ratios provided in [Table 2](#T2) are for the presence of a single variant SNP genotype, which would be a heterozygous state. The presence of two variant SNP genotypes would be a homozygous variant state and the hazard ratio would be multiplicative. For example, considering rs4476990, the hazard ratio was 0.61, indicating ~39% reduction in the hazard of a breast cancer event for the heterozygous state relative to the homozygous wild type state, but the hazard ratio for women with the homozygous variant state would be 0.61×0.61=0.37, indicating ~63% reduction in the hazard of a breast cancer event, again relative to those with a homozygous wild type state. The p-values for the top six SNPs ranged from 2.15E-07 to 6.24E-07). Importantly, these variant SNP genotypes were common with MAFs ranging from 0.32 to 0.42.

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1d63/5135610/e93bcb112cae/nihms827024f1.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5135610_nihms827024f1.jpg)

A, GWAS Manhattan Plot. B, Locus zoom of the chromosome 8 region surrounding the MIR2052HG gene.

### Table 2.

| Reference sequence | Base Position | Hazard Ratio* (95% CI) | P-Value | Minor Allele Frequency | Genotyped or Imputed |
| --- | --- | --- | --- | --- | --- |
| rs13260300 | 75476924 | 0.61 (0.50–0.74) | 2.15E-07 | 0.39 | O† |
| rs4476990§ | 75478412 | 0.61 (0.51–0.74) | 2.51E-07 | 0.42 | I‡ |
| rs2891356 | 75473223 | 0.61 (0.51–0.74) | 2.76E-07 | 0.39 | I |
| rs746460 | 75474529 | 0.62 (0.51–0.75) | 3.05E-07 | 0.39 | O |
| rs4735715¶ | 75514554 | 0.59 (0.48–0.73) | 4.47E-07 | 0.32 | I |
| rs3802201§,¶ | 75516199 | 0.60 (0.48–0.74) | 6.24E-07 | 0.32 | I |

Table 2 Caption: SNPs with Lowest P-Values on Chromosome 8

To validate our imputation quality, we used deep sequencing techniques to call variants surrounding the chromosome 8 peak. We compared the deep sequencing variants calls of these SNPs with those obtained from MA-27 data. The correlations between the variants (from deep sequencing) and the MA.27 SNPs were at least 0.9 in all the top SNPs ([Supplementary Table 2](#SD1)), and thus of high quality.

The top SNP (rs13260300) was located 32.4kb 5′ of *MIR2052HG* (other names: *FLJ39080*, LOC441355), a gene located on chromosome 8q21.11, and two of the top SNPs were located in *MIR2052HG. MIR2052HG* encodes a lncRNA whose function is not known and with very few publications referring to this gene ([22](#R22), [23](#R23)).

Because this study involved AIs as therapy for ER-positive breast cancer, we interrogated the top SNPs to determine which were located in or within 500 bp of a putative estrogen response element (ERE), similar to what we have done in previous studies ([24](#R24), [25](#R25)). The distance of 500 bp was chosen as chromatin immunoprecipitation (ChIP) assays become less reliable with greater distances between a SNP and an ERE. Two of the top SNPs fulfilled one of these two criteria with rs4476990 being located 33kb 5′ of *MIR2052HG* and in an ERE, while rs3802201 mapped to intron 1 of *MIR2052HG* and located 16bp from an ERE. We focused on these two SNPs, which are in moderate linkage disequilibrium (R^2^=0.6), in our functional studies. The significant relationships between the genotypes of the two SNPs, rs4476990 and rs3802201, and BCFI are shown in [Supplementary Figs. 3A and 3B](#SD4), respectively. After adjustment for covariates (treatment arm, cohort, race, ER/PR status, T-Stage, ECOG performance status and bisphosphonate use), a stratified Cox model using stratification factors (adjuvant chemotherapy, lymph node status, trastuzumab use) determined p=2.51E-07 for rs4476990 and p=6.24E-07 for rs3802201.

### Haplotype Analysis of the Top Six SNPs

A haplotype analysis of the top six SNPs was performed to determine the degree of association. The results of this analysis are shown in [Supplementary Table 3](#SD1) and reveal that the association for the most significant haplotype (HR 0.58, p=1.23E-06) was not superior to the top SNPs considered individually ([Table 2](#T2)).

### Functional Genomic Studies of Chromosome 8 SNPs

Because of the importance of ERα in endocrine therapy, we interrogated the Cancer Genome Atlas (TCGA) breast cancer data ([26](#R26)) for a possible relationship between the expression of MIR2052HG and ESR1, which encodes ERα, in 485 ER-positive breast cancers. There was a positive correlation (Spearman correlation coefficient 0.370) between the expressions of these two genes. This provided an indication that there might be an important relationship between MIR2052HG and ESR1, which was supported by the functional genomic studies described in subsequent paragraphs. MIR2052HG is expressed in multiple breast cancer cell lines including ERα-positive cell lines, according to the Cancer Genomics Hub ([https://cghub.ucsc.edu](https://cghub.ucsc.edu)).

### MIR2052HG SNPs determine estradiol-dependent MIR2052HG expression and ERα binding to EREs

As mentioned earlier, the rs4476990 SNP was located in a putative ERE and the rs3802201 SNP in intron 1 of *MIR2052HG* was near another ERE ([Fig. 2A](#F2)). To test the possible functional impact of these two SNPs, we utilized a model system consisting of 300 individual human lymphoblastoid cell lines (LCLs) for which we have extensive genomic and transcriptomic data. This LCL model system has repeatedly shown its value in both generating and testing pharmacogenomics hypotheses ([8](#R8), [25](#R25), [27](#R27)). Specifically, we selected 5 LCLs with homozygous wild type (WT) genotypes for both SNPs and 5 LCLs homozygous for variant genotypes for both SNPs to perform estradiol (E2) treatment and ChIP assays. In the presence of E2, cells homozygous for the variant SNP genotypes showed a dose-dependent increase in MIR2052HG expression ([Fig. 2B](#F2)) as well as increased binding of ERα to the EREs shown in [Fig. 2A](#F2) for variant genotypes for both SNPs relative to WT in ChIP assays using ERα antibody ([Figs. 2C and 2D](#F2)). 4-Hydroxy-tamoxifen (4-OH-Tam), a SERM that competes with E2 for ERα binding, could reverse this effect ([Figs. 2C and 2D](#F2)).

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1d63/5135610/8fd571aa4f26/nihms827024f2.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5135610_nihms827024f2.jpg)

A. Schematic of estrogen response elements (EREs) around rs4476990 and rs3802201. The EREs are indicated as boxes and the SNPs are indicated as red circles. B. MIR2052HG mRNA expression in lymphoblastoid cell lines (LCLs) with wild type SNP (W) and variant SNP (V) genotypes for both rs4476990 and rs3802201 after exposure to increasing concentrations of E2. Error bars represent SEM. *P<0.05. C. and D., ChIP assay using six LCLs with known genotypes for rs4476990 and rs3802201 SNPs. Error bars represent SEM of three independent experiments. Representative PCR products are visualized on agarose DNA gels.

### Aromatase inhibitors reverse estradiol-dependent and SNP-dependent MIR2052HG and ESR1 expression

The major function of AIs is to reduce estrogen levels by the inhibition of aromatase, the rate-limiting step in estrogen biosynthesis. The reduction of estrogens could have an effect on ERα-mediated function. As we have shown in our previous studies, estrogens and SERMs can alter gene expression in a SNP-dependent fashion ([25](#R25)). Thus, we proceeded to examine the effect of the AIs anastrozole and exemestane on MIR2052HG and ESR1 expression in the presence of androstenedione, which is aromatized to estrone by aromatase, the target for the AIs under study.

In the presence of androstenedione, LCLs with variant genotypes for both SNPs showed dose-dependent increases in MIR2052HG expression ([Figs. 3A and B](#F3)) that was similar to that for E2 ([Fig. 2B](#F2)). However, the addition an AI, either exemestane ([Fig. 3A](#F3)) or anastrozole ([Fig. 3B](#F3)), to the androstenedione-treated LCLs caused a “reversal” of the expression pattern with increased MIR2052HG expression in LCLs homozygous for the WT genotypes but a marked decrease in LCLs homozygous for variant genotypes. Of particular interest was the observation of a direct correlation between this striking pattern of expression for MIR2052HG and that of ESR1 in the same cell lines ([Figs. 3C and 3D](#F3)), bringing us back to the correlation that we had observed between the expression of MIR2052HG and ESR1 in the TCGA data.

### Figure 3.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1d63/5135610/3f0d71ad3fd0/nihms827024f3.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5135610_nihms827024f3.jpg)

A and B. MIR2052HG mRNA expression in lymphoblastoid cell lines (LCLs) withWT and V genotypes for both rs4476990 and rs3802201 after exposure to androstenedione alone and with increasing concentrations of exemestane or anastrozole. C and D. mRNA expression for ESR1 in LCLs with the same conditions as A and B. *P<0.05, **P<0.01.

### MIR2052HG affects ESR1 and ERα expression and proliferation, colony formation, and response to AIs in ERα-positive breast cancer cell lines

Having determined that, in a SNP and AI-dependent fashion, the expression of MIR2052HG was correlated with that of ESR1 ([Fig. 3](#F3)), we set out to study the possible functional impact of the MIR2052HG lncRNA on AI response and on cell proliferation. When we began our studies, the function of MIR2052HG was not known, but we hypothesized, based on the results shown in [Fig. 3](#F3), that this lncRNA might influence AI response through its effect on the downstream expression of ESR1. To determine the effect of MIR2052HG knock down on ERα levels, we chose two ERα-positive breast cancer cell lines with relatively high endogenous MIR2052HG expression, CAMA1 and the aromatase expressing MCF7/AC1 ([28](#R28)) cell lines. Knock down of MIR2052HG resulted in striking decreases of ERα expression, both at the mRNA and the protein levels ([Figs. 4A and 4B](#F4)), consistent with the TCGA data that showed a positive correlation between the two genes. Furthermore, in MCF7/AC1 cells, knock down of MIR2052HG decreased cell proliferation and colony formation ([Figs. 4C and 4D](#F4)), while overexpression of MIR2052HG increased cell proliferation, colony formation, and ERα expression ([Fig. 4E](#F4)), functionally confirming the positive relationship between ERα and MIR2052HG. We observed the same results in two additional ERα-positive breast cancer cell lines, HCC1428 and BT474, with regard to the effect of MIR2052HG on ERα protein levels and cell proliferation ([Supplementary Fig. S4](#SD5)). Also, it is well known that androstenedione increases MCF7/AC1 proliferation and that this increase in proliferation can be abrogated with AIs ([29](#R29)). However, overexpression of MIR2052HG significantly increased cell proliferation even in the presence of AI treatment ([Fig. 4F](#F4)). Conversely, down-regulation of MIR2052HG inhibited MCF7/AC1 cell proliferation induced by either E2 or androstenedione ([Supplementary Fig. S5](#SD6)). No significant change was observed in the proliferation of ERα-negative cells (AU565 and BT549) after the down-regulation of MIR2052HG ([Supplementary Fig. S6](#SD7)).

### Figure 4.

![Figure 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1d63/5135610/18dc1acab531/nihms827024f4.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5135610_nihms827024f4.jpg)

A, Knock down of MIR2052HG by antisense oligonucleotides (ASO1 and ASO2) down-regulated ERα protein. The histogram shows knock down efficiency in CAMA1 and MCF7/AC1 cells. B, Knock down of MIR2052HG decreased ESR1 mRNA expression levels in CAMA1 and MCF7/AC1 cell lines. C and D. Knock down of MIR2052HG decreased proliferation and colony formation in CAMA1 and MCF7/AC1 cells. The representative colony formation pictures from triplicate experiments are shown. The colony formation rates are quantified as percentages. Error bars represent SEM; ** P< 0.01 compared to baseline (negative control). E, Overexpression of MIR2052HG increased ERα protein levels. Overexpression efficiency was determined by qRT-PCR. Overexpression of MIR2052HG increased the proliferation and colony formation and ERα protein levels in MCF7/AC1 cells. F. Overexpression of MIR2052HG conferred resistance to AIs (exemestane and anastrozole) and 4-hydroxy-tamoxifen treatments in MCF7/AC1 compared to negative controls. The assay was performed as described in C and D. Error bars represent SEM. The concentrations for androstenedione (A), exemestane (EXE, anastrozole (ANA), estradiol (E2) and 4-hydroxy-tamoxifen (TAM) are indicated.

### MIR2052HG regulates ERα expression through transcription and protein degradation pathways

MIR2052HG expression is associated with both ERα mRNA and protein levels ([Fig. 4A](#F4)). To pursue these observations, we began by determining whether MIR2052HG might affect ERα protein stability mediated by protein degradation. Treatment with cycloheximide, resulted in a decrease in the half-life of ERα protein in cells in which MIR2052HG had been knocked down ([Fig. 5A and 5B](#F5)). We also treated MIR2052HG knock down cells with MG132, a proteasome inhibitor, and found that it reversed ERα degradation in these cells ([Fig. 5C](#F5), upper panel). The same phenomenon was observed using bortezomib, another specific proteasome inhibitor ([Fig. 5C](#F5), lower panel). These results indicated that MIR2052HG regulates ERα protein stability through a proteasome-mediated degradation pathway. Furthermore, we observed that ERα ubiquitination increased after knocking down MIR2052HG in 293T cells ([Fig. 5D](#F5)), confirming the involvement of ubiquitin-dependent and proteasome-mediated degradation.

### Figure 5.

![Figure 5](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1d63/5135610/b1700ebe211b/nihms827024f5.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5135610_nihms827024f5.jpg)

A, Knock down of MIR2052HG shortened ERα protein half-life. CAMA1 cells were transfected with MIR2052HG specific antisense oligonucleotides (ASOs) or a negative control ASO and then treated with cycloheximide (CHX). The representative western blotting results from three independent experiments are shown. The knock down efficiency was determined by qRT-PCR. B. Quantitative intensities of ERα are mean values with SEM (error bars) from three independent experiments. C. MIR2052HG regulated ERα stability in a proteasome-dependent manner. MIR2052HG was knocked down with ASOs in CAMA1 cells that were treated with MG132 or bortezomib. D. Knock down of MIR2052HG promoted the ubiquitination of ERα. 293T cells were transfected with HA-Ub plasmid and FLAG-ERα plasmid, and then transfected with either the MIR2052HG specific ASOs or the negative control ASO followed by MG132. ERα proteins were immunoprecipitated and analyzed by western blotting. Knock down efficiency in 293T cells was determined by qRT-PCR. E. MIR2052HG regulated ERα transcription through the AKT-FOXO3 pathway. Knock down of MIR2052HG increased AKT phosphorylation and decreased FOXO3 phosphorylation and FOXO3 total level in MCF7/AC1 and CAMA1 cells. F. Overexpression of FOXO3 in MIR2052HG knocked-down MCF7/AC1 cells reversed ERα protein and mRNA levels. Overexpression of FOXO3 was determined by western blotting. HA: expression tag.

We next examined possible mechanisms by which MIR2052HG was involved in the control of ERα transcription. During these experiments, we observed in both MCF7/AC1 and CAMA1 cells that down-regulation of MIR2052HG resulted in increased phospho-AKT (pAKT) levels at both the Ser473 and Thr308 sites, but total AKT levels did not change ([Fig 5E](#F5)). Because FOXO3 is downstream of AKT and activated AKT phosphorylates FOXO3 resulting in degradation of FOXO3 through a proteasome-dependent process ([30](#R30)), we determined the effects of MIR2052HG knock down on total FOXO3 and phospho-FOXO3 levels (on S318/S321) and observed both of them to be reduced ([Fig. 5E](#F5)), consistent with the known effect of pAKT ([30](#R30)). It is also well known that FOXO3 regulates the expression of ERα ([31](#R31)–[33](#R33)) and that the expression of FOXO3 is directly correlated with the expression of ERα. To further confirm that the regulation of *ESR1* mRNA levels by MIR2052HG is mediated through the regulation of FOXO3, we overexpressed FOXO3 in MCF/AC1 cells, in which MIR2052HG had been knocked down, and observed that FOXO3 overexpression could reverse the down-regulation of *ESR1* mRNA caused by knocking down MIR2052HG ([Fig. 5F](#F5)). In summary, these results indicate that the down-regulation of MIR2052HG can reduce *ESR1* mRNA levels by promoting AKT-mediated down-regulation of FOXO3, which regulates ESR1 transcription. Thus, it appears that tumor expression of MIR2052HG plays a role in the regulation of ERα transcription in addition to ERα protein degradation.

## Discussion

Recurrence of breast cancer in women with early stage disease treated with adjuvant endocrine therapy implies endocrine resistance. Multiple potential mechanisms, mainly focusing on ERα function, have been proposed for this resistance but, by and large, these mechanisms have been related to factors present in the cancers ([34](#R34)–[36](#R36)). Much less attention has been paid to host-related factors for endocrine resistance or, more appropriately, lack of efficacy of endocrine therapy, such as CYP2D6 poor metabolizer genotype in the case of tamoxifen ([37](#R37)). In the current study, our goal was to interrogate the germline genome for SNPs related to breast cancer events in women treated with adjuvant AI therapy, relate those SNPs to genes, and perform functional studies to identify potential mechanisms for the observed associations. We controlled for significant imbalances in three stratification factors by performing stratified Cox analyses and for the effects of baseline factors that impacted BCFI with their forced inclusion in the Cox model. The p-values for the top SNPs were approximately 2E-07, which approaches, but does not reach genome-wide significance. However, because of the importance of the phenotype and the fact that we were studying the largest study that had evaluated AIs and had DNA available, we chose to pursue these signals with functional genomic experiments, with strikingly positive findings. We acknowledge that a replication dataset would have been of value but we considered it important to report our compelling data despite the lack of an available dataset.

Our GWAS identified variant SNPs on chromosome 8 that were protective and in or near a gene (*MIR2052HG*) that encodes a lncRNA. There is increasing appreciation of the role of lncRNAs in regulation of the genome ([38](#R38)). For example, lncRNAs can form extensive networks of ribonucleoprotein complexes with chromatin regulators and modulate them ([39](#R39), [40](#R40)). There is also increasing evidence suggesting lncRNAs play important roles in cancers ([41](#R41), [42](#R42)). For example, HOTAIR (Hox antisense intergenic RNA) is highly induced in about one-quarter of patients with breast cancer ([43](#R43)). The long intergenic non-coding RNA-ROR has been shown to induce epithelial-to-mesenchymal transition and contribute to breast cancer metastasis ([44](#R44)) and to enhance ERα signaling, conferring resistance to tamoxifen ([45](#R45)). A recent study also indicated that a cluster of lncRNAs, termed Eleanors (ESR1 locus enhancing and activating noncoding RNAs) located within the genomic region containing the ESR1 gene can regulate ERα levels through an enhancer function ([46](#R46)). Of note is the fact that the lncRNA SRA1 (steroid receptor RNA activator 1) acted as a coactivator of ERα, and this action depended on the phosphorylation of ERα at Ser118 ([47](#R47)).

The role of lncRNAs in resistance to endocrine therapy is an area of emerging interest ([48](#R48), [49](#R49)). However, our study is, to our knowledge, the first to focus on the impact of a lncRNA on outcomes in a large prospective trial of AI therapy in women with early-stage breast cancer. Our finding of the relationship between SNPs related to the *MIR2052HG* lncRNA and recurrence of breast cancer in women treated with adjuvant AIs led us to perform a series of functional studies to discover potential mechanisms for this association. Using an LCL model system, we showed that, in the presence of E2, these variant SNPs increased both MIR2052HG and ESR1 expression with increased ERα binding to ERE motifs for variant SNP genotypes as shown by ChIP assays ([Figs. 2C, 2D](#F2)). However, when an AI (exemestane or anastrozole) was added, LCLs with the WT SNP genotype displayed a marked up-regulation of MIR2052HG expression and, in parallel, ESR1 expression whereas cells with the variant SNP genotypes showed a clear decrease in both MIR2052HG and ESR1 expression ([Fig. 3](#F3)). That is, the presence of the AI brought about a “reversal” of the SNP-dependent MIR2052HG expression pattern. SNP- and drug-dependent regulation of gene expression has been previously reported by our group in the case of selective estrogen receptor modulators (SERMs), tamoxifen and raloxifene, when given as preventive therapy, which led to the identification of novel mechanisms by which SNPs can regulate gene expression ([25](#R25)). Thus, our previous observations with SERMs ([25](#R25)) provided an impetus to investigate the role of this lncRNA in the efficacy of AI therapy as well as in the regulation of hormone-dependent breast cancer.

*MIR2052HG* is located on chromosome 8 and not in the *ESR1* genomic region on chromosome 6, but it has a significant effect on ERα regulation as noted above. ERα plays an essential role in cell proliferation and survival in estrogen-dependent breast cancers and in AI-treated patients, *ESR1* amplification, resulting in increased ERα expression, is associated with endocrine resistance ([36](#R36)). We showed that overexpression of MIR2052HG increased ERα expression and accelerated cell proliferation of MCF/AC1 cells in the presence of AI treatments ([Figs. 4E, 4F](#F4)). Conversely, down-regulation of MIR2052HG reduced cell proliferation and colony formation even after E2 or androstenedione treatment ([Figs. 4C, 4D](#F4) and [Supplementary Fig. 5](#SD6)). These phenomena are specific for ERα-positive cells, since in ER-negative cell lines, we did not observe an effect of MIR2052HG on cell proliferation ([Supplementary Fig. 6](#SD7)). Based on our observations of the effects of MIR2052HG on both ERα protein and mRNA levels, we hypothesized that MIR2052HG might regulate ERα through a proteasome-mediated pathway, which we experimentally confirmed by treatment with proteasome inhibitors and with ubiquitin assays ([Fig. 5](#F5)). Furthermore, regulation of the transcription of ERα by MIR2052HG was found to be through AKT-dependent FOXO3 regulation; FOXO3 is a known transcription factor for ERα ([Figs. 5E, 5F](#F5)).

Through our GWAS using germline DNA samples from the largest AI clinical trial ([5](#R5)), we have identified a novel lncRNA that potentially plays an important role in the regulation of ERα levels, one of the mechanisms involved in AI resistance. Our GWAS indicated that two variant SNPs (rs4476990 and rs3802201) were common variants with MAF values of 42% and 32%, respectively, and that both were protective, i.e. patients with the variant SNP genotypes had a longer BCFI. The results of our mechanistic studies supported the association in that both variant SNPs down-regulated MIR2052HG expression in the presence of AIs, which was associated with the down-regulation of ERα at both the mRNA level and the protein levels. The presence of markedly increased ESR1 expression in the presence of either anastrozole or exemestane in LCLs with the WT SNP genotype is a potential mechanism for the adverse outcomes in patients carrying the WT SNP genotype. Conversely, down-regulation of ERα in the presence of the variant SNP genotypes after exposure to anastrozole or exemestane might be a factor contributing to their more favorable BCFI.

At the mechanistic level, how MIR2052HG regulates the AKT pathway remains to be further investigated. It could have a direct impact on AKT phosphorylation or more likely, through the regulation of upstream proteins that affect AKT activity. Additionally, the mechanism by which MIR2052HG regulates ERα ubiquitin- and proteasome-mediated degradation also remains unresolved, but previous studies have suggested ER phosphorylation can influence its ubiquitination ([50](#R50)). Therefore, one possibility is that the MIR2052HG could affect ERα levels by regulating various proteins that might affect ERα phosphorylation. Our current findings suggest that MIR2052HG could affect both ERα mRNA and protein levels through different mechanisms.

In summary, we have identified SNP genotypes on chromosome 8 that were associated with breast cancer outcomes in women treated with the AIs anastrozole or exemestane as adjuvant therapy for their early-stage breast cancer, and we related these SNPs to a lncRNA, which, in a SNP-dependent and AI-dependent fashion, regulated ERα expression. The variant SNP genotypes, which are favorable in women treated with anastrozole and exemestane, are potential markers that could identify women for whom either AI would be appropriate therapy, i.e., for those women whose germline carries the variant SNP genotypes. This would require further corroboration with additional clinical testing. However, our GWAS and functional studies have provided initial evidence that germline genetic variability in SNP genotypes related to a gene encoding a lncRNA, *MIR2052HG*, may impact outcomes after treatment with the AIs anastrozole or exemestane.

## Supplementary Material

## Acknowledgments

Financial support: These studies were supported in part by NIH grants U19 GM61388 (The Pharmacogenomics Research Network), P50CA116201 (Mayo Clinic Breast Cancer Specialized Program of Research Excellence), U10CA77202, R01CA196648, CCS 015469 from the Canadian Cancer Society, the Breast Cancer Research Foundation, and the RIKEN Center for Integrative Medical Sciences and the Biobank Japan Project funded by the Ministry of Education, Culture, Sports, Science and Technology, Japan. The MA.27 trial was supported, in part, by Pfizer, Inc.

The authors acknowledge the women who participated in the MA.27 clinical trial and provided DNA and consent for its use in genetic studies.

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Siegel RL, Miller KD, Jemal A. Cancer statistics, 2015. CA Cancer J Clin. 2015;65:5–29. doi: 10.3322/caac.21254.  [DOI](https://doi.org/10.3322/caac.21254) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25559415/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=CA%20Cancer%20J%20Clin&title=Cancer%20statistics,%202015&author=RL%20Siegel&author=KD%20Miller&author=A%20Jemal&volume=65&publication_year=2015&pages=5-29&pmid=25559415&doi=10.3322/caac.21254&)

2. Ferlay J, Soerjomataram I, Dikshit R, Eser S, Mathers C, Rebelo M, et al. Cancer incidence and mortality worldwide: sources, methods and major patterns in GLOBOCAN 2012. Int J Cancer. 2015;136:E359–86. doi: 10.1002/ijc.29210.  [DOI](https://doi.org/10.1002/ijc.29210) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25220842/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Cancer&title=Cancer%20incidence%20and%20mortality%20worldwide:%20sources,%20methods%20and%20major%20patterns%20in%20GLOBOCAN%202012&author=J%20Ferlay&author=I%20Soerjomataram&author=R%20Dikshit&author=S%20Eser&author=C%20Mathers&volume=136&publication_year=2015&pages=E359-86&pmid=25220842&doi=10.1002/ijc.29210&)

3. Davies C, Godwin J, Gray R, Clarke M, Cutter D, Darby S, et al. Relevance of breast cancer hormone receptors and other factors to the efficacy of adjuvant tamoxifen: patient-level meta-analysis of randomised trials. Lancet. 2011;378:771–84. doi: 10.1016/S0140-6736(11)60993-8.  [DOI](https://doi.org/10.1016/S0140-6736(11)60993-8) | [PMC free article](/articles/PMC3163848/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21802721/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Relevance%20of%20breast%20cancer%20hormone%20receptors%20and%20other%20factors%20to%20the%20efficacy%20of%20adjuvant%20tamoxifen:%20patient-level%20meta-analysis%20of%20randomised%20trials&author=C%20Davies&author=J%20Godwin&author=R%20Gray&author=M%20Clarke&author=D%20Cutter&volume=378&publication_year=2011&pages=771-84&pmid=21802721&doi=10.1016/S0140-6736(11)60993-8&)

4. Dowsett M, Forbes JF, Bradley R, Ingle J, Aihara T, Bliss J, et al. Aromatase inhibitors versus tamoxifen in early breast cancer: patient-level meta-analysis of the randomised trials. Lancet. 2015;386:1341–52. doi: 10.1016/S0140-6736(15)61074-1.  [DOI](https://doi.org/10.1016/S0140-6736(15)61074-1) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26211827/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Aromatase%20inhibitors%20versus%20tamoxifen%20in%20early%20breast%20cancer:%20patient-level%20meta-analysis%20of%20the%20randomised%20trials&author=M%20Dowsett&author=JF%20Forbes&author=R%20Bradley&author=J%20Ingle&author=T%20Aihara&volume=386&publication_year=2015&pages=1341-52&pmid=26211827&doi=10.1016/S0140-6736(15)61074-1&)

5. Goss PE, Ingle JN, Pritchard KI, Ellis MJ, Sledge GW, Budd GT, et al. Exemestane versus anastrozole in postmenopausal women with early breast cancer: NCIC CTG MA.27--a randomized controlled phase III trial. J Clin Oncol. 2013;31:1398–404. doi: 10.1200/JCO.2012.44.7805.  [DOI](https://doi.org/10.1200/JCO.2012.44.7805) | [PMC free article](/articles/PMC3612593/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23358971/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Exemestane%20versus%20anastrozole%20in%20postmenopausal%20women%20with%20early%20breast%20cancer:%20NCIC%20CTG%20MA.27--a%20randomized%20controlled%20phase%20III%20trial&author=PE%20Goss&author=JN%20Ingle&author=KI%20Pritchard&author=MJ%20Ellis&author=GW%20Sledge&volume=31&publication_year=2013&pages=1398-404&pmid=23358971&doi=10.1200/JCO.2012.44.7805&)

6. Hudis CA, Barlow WE, Costantino JP, Gray RJ, Pritchard KI, Chapman JA, et al. Proposal for standardized definitions for efficacy end points in adjuvant breast cancer trials: the STEEP system. J Clin Oncol. 2007;25:2127–32. doi: 10.1200/JCO.2006.10.3523.  [DOI](https://doi.org/10.1200/JCO.2006.10.3523) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17513820/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Proposal%20for%20standardized%20definitions%20for%20efficacy%20end%20points%20in%20adjuvant%20breast%20cancer%20trials:%20the%20STEEP%20system&author=CA%20Hudis&author=WE%20Barlow&author=JP%20Costantino&author=RJ%20Gray&author=KI%20Pritchard&volume=25&publication_year=2007&pages=2127-32&pmid=17513820&doi=10.1200/JCO.2006.10.3523&)

7. Ingle JN, Schaid DJ, Goss PE, Liu M, Mushiroda T, Chapman JA, et al. Genome-wide associations and functional genomic studies of musculoskeletal adverse events in women receiving aromatase inhibitors. J Clin Oncol. 2010;28:4674–82. doi: 10.1200/JCO.2010.28.5064.  [DOI](https://doi.org/10.1200/JCO.2010.28.5064) | [PMC free article](/articles/PMC3020700/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20876420/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Genome-wide%20associations%20and%20functional%20genomic%20studies%20of%20musculoskeletal%20adverse%20events%20in%20women%20receiving%20aromatase%20inhibitors&author=JN%20Ingle&author=DJ%20Schaid&author=PE%20Goss&author=M%20Liu&author=T%20Mushiroda&volume=28&publication_year=2010&pages=4674-82&pmid=20876420&doi=10.1200/JCO.2010.28.5064&)

8. Liu M, Goss PE, Ingle JN, Kubo M, Furukawa Y, Batzler A, et al. Aromatase inhibitor-associated bone fractures: a case-cohort GWAS and functional genomics. Mol Endocrinol. 2014;28:1740–51. doi: 10.1210/me.2014-1147.  [DOI](https://doi.org/10.1210/me.2014-1147) | [PMC free article](/articles/PMC4179631/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25148458/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Endocrinol&title=Aromatase%20inhibitor-associated%20bone%20fractures:%20a%20case-cohort%20GWAS%20and%20functional%20genomics&author=M%20Liu&author=PE%20Goss&author=JN%20Ingle&author=M%20Kubo&author=Y%20Furukawa&volume=28&publication_year=2014&pages=1740-51&pmid=25148458&doi=10.1210/me.2014-1147&)

9. Sicotte HNPN, Johnson JA, Lin Y, Decker PA, Eckel-Passow JE, Jenkins RB, Goetz MPR, Weinshilboum R, Boughey JC, Wang L, Kalari KR, McDonnell SK, de Andrade M, Kocher JP. Single Sample Imputation from Next Generation Sequencing (NGS) Exome data can improve genotypes in low-coverage regions. 65th Annual Meeting of The American Society of Human Genetics; Baltimore, MD. 2015.  [Google Scholar](https://scholar.google.com/scholar_lookup?Sicotte%20HNPN,%20Johnson%20JA,%20Lin%20Y,%20Decker%20PA,%20Eckel-Passow%20JE,%20Jenkins%20RB,%20Goetz%20MPR,%20Weinshilboum%20R,%20Boughey%20JC,%20Wang%20L,%20Kalari%20KR,%20McDonnell%20SK,%20de%20Andrade%20M,%20Kocher%20JP.%20Single%20Sample%20Imputation%20from%20Next%20Generation%20Sequencing%20(NGS)%20Exome%20data%20can%20improve%20genotypes%20in%20low-coverage%20regions.%2065th%20Annual%20Meeting%20of%20The%20American%20Society%20of%20Human%20Genetics;%20Baltimore,%20MD.%202015.)

10. PROC PHREG : : SAS Institute Inc. SAS/STAT® 9.2 User’s Guide. Cary, NC: SAS Institute Inc; 2008.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=SAS/STAT%C2%AE%209.2%20User%E2%80%99s%20Guide&publication_year=2008&)

11. Patterson N, Price AL, Reich D. Population structure and eigenanalysis. PLoS Genet. 2006;2:e190. doi: 10.1371/journal.pgen.0020190.  [DOI](https://doi.org/10.1371/journal.pgen.0020190) | [PMC free article](/articles/PMC1713260/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17194218/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet&title=Population%20structure%20and%20eigenanalysis&author=N%20Patterson&author=AL%20Price&author=D%20Reich&volume=2&publication_year=2006&pages=e190&pmid=17194218&doi=10.1371/journal.pgen.0020190&)

12. Price AL, Patterson NJ, Plenge RM, Weinblatt ME, Shadick NA, Reich D. Principal components analysis corrects for stratification in genome-wide association studies. Nat Genet. 2006;38:904–9. doi: 10.1038/ng1847.  [DOI](https://doi.org/10.1038/ng1847) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16862161/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Principal%20components%20analysis%20corrects%20for%20stratification%20in%20genome-wide%20association%20studies&author=AL%20Price&author=NJ%20Patterson&author=RM%20Plenge&author=ME%20Weinblatt&author=NA%20Shadick&volume=38&publication_year=2006&pages=904-9&pmid=16862161&doi=10.1038/ng1847&)

13. Therneau TM. A Package for Survival Analysis in S. version 2.38. 2015 Retrieved from http://CRAN.R-project.org/package=survival.  [http://CRAN.R-project.org/package=survival](http://CRAN.R-project.org/package=survival)

14. Purcell S, Neale B, Todd-Brown K, Thomas L, Ferreira MA, Bender D, et al. PLINK: a tool set for whole-genome association and population-based linkage analyses. Am J Hum Genet. 2007;81:559–75. doi: 10.1086/519795.  [DOI](https://doi.org/10.1086/519795) | [PMC free article](/articles/PMC1950838/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17701901/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=PLINK:%20a%20tool%20set%20for%20whole-genome%20association%20and%20population-based%20linkage%20analyses&author=S%20Purcell&author=B%20Neale&author=K%20Todd-Brown&author=L%20Thomas&author=MA%20Ferreira&volume=81&publication_year=2007&pages=559-75&pmid=17701901&doi=10.1086/519795&)

15. Schaid DJ, Rowland CM, Tines DE, Jacobson RM, Poland GA. Score tests for association between traits and haplotypes when linkage phase is ambiguous. Am J Hum Genet. 2002;70:425–34. doi: 10.1086/338688.  [DOI](https://doi.org/10.1086/338688) | [PMC free article](/articles/PMC384917/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11791212/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=Score%20tests%20for%20association%20between%20traits%20and%20haplotypes%20when%20linkage%20phase%20is%20ambiguous&author=DJ%20Schaid&author=CM%20Rowland&author=DE%20Tines&author=RM%20Jacobson&author=GA%20Poland&volume=70&publication_year=2002&pages=425-34&pmid=11791212&doi=10.1086/338688&)

16. DePristo MA, Banks E, Poplin R, Garimella KV, Maguire JR, Hartl C, et al. A framework for variation discovery and genotyping using next-generation DNA sequencing data. Nat Genet. 2011;43:491–8. doi: 10.1038/ng.806.  [DOI](https://doi.org/10.1038/ng.806) | [PMC free article](/articles/PMC3083463/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21478889/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=A%20framework%20for%20variation%20discovery%20and%20genotyping%20using%20next-generation%20DNA%20sequencing%20data&author=MA%20DePristo&author=E%20Banks&author=R%20Poplin&author=KV%20Garimella&author=JR%20Maguire&volume=43&publication_year=2011&pages=491-8&pmid=21478889&doi=10.1038/ng.806&)

17. McKenna A, Hanna M, Banks E, Sivachenko A, Cibulskis K, Kernytsky A, et al. The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data. Genome Res. 2010;20:1297–303. doi: 10.1101/gr.107524.110.  [DOI](https://doi.org/10.1101/gr.107524.110) | [PMC free article](/articles/PMC2928508/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20644199/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Res&title=The%20Genome%20Analysis%20Toolkit:%20a%20MapReduce%20framework%20for%20analyzing%20next-generation%20DNA%20sequencing%20data&author=A%20McKenna&author=M%20Hanna&author=E%20Banks&author=A%20Sivachenko&author=K%20Cibulskis&volume=20&publication_year=2010&pages=1297-303&pmid=20644199&doi=10.1101/gr.107524.110&)

18. Van der Auwera GA, Carneiro MO, Hartl C, Poplin R, Del Angel G, Levy-Moonshine A, et al. From FastQ data to high confidence variant calls: the Genome Analysis Toolkit best practices pipeline. Curr Protoc Bioinformatics. 2013;43:1101–33. doi: 10.1002/0471250953.bi1110s43.  [DOI](https://doi.org/10.1002/0471250953.bi1110s43) | [PMC free article](/articles/PMC4243306/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25431634/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Protoc%20Bioinformatics&title=From%20FastQ%20data%20to%20high%20confidence%20variant%20calls:%20the%20Genome%20Analysis%20Toolkit%20best%20practices%20pipeline&author=GA%20Van%20der%20Auwera&author=MO%20Carneiro&author=C%20Hartl&author=R%20Poplin&author=G%20Del%20Angel&volume=43&publication_year=2013&pages=1101-33&pmid=25431634&doi=10.1002/0471250953.bi1110s43&)

19. Cingolani P, Platts A, Wang le L, Coon M, Nguyen T, Wang L, et al. A program for annotating and predicting the effects of single nucleotide polymorphisms, SnpEff: SNPs in the genome of Drosophila melanogaster strain w1118; iso-2; iso-3. Fly (Austin) 2012;6:80–92. doi: 10.4161/fly.19695.  [DOI](https://doi.org/10.4161/fly.19695) | [PMC free article](/articles/PMC3679285/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22728672/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Fly%20(Austin)&title=A%20program%20for%20annotating%20and%20predicting%20the%20effects%20of%20single%20nucleotide%20polymorphisms,%20SnpEff:%20SNPs%20in%20the%20genome%20of%20Drosophila%20melanogaster%20strain%20w1118;%20iso-2;%20iso-3&author=P%20Cingolani&author=A%20Platts&author=L%20Wang%20le&author=M%20Coon&author=T%20Nguyen&volume=6&publication_year=2012&pages=80-92&pmid=22728672&doi=10.4161/fly.19695&)

20. Kocher JP, Quest DJ, Duffy P, Meiners MA, Moore RM, Rider D, et al. The Biological Reference Repository (BioR): a rapid and flexible system for genomics annotation. Bioinformatics. 2014;30:1920–2. doi: 10.1093/bioinformatics/btu137.  [DOI](https://doi.org/10.1093/bioinformatics/btu137) | [PMC free article](/articles/PMC4071205/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24618464/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=The%20Biological%20Reference%20Repository%20(BioR):%20a%20rapid%20and%20flexible%20system%20for%20genomics%20annotation&author=JP%20Kocher&author=DJ%20Quest&author=P%20Duffy&author=MA%20Meiners&author=RM%20Moore&volume=30&publication_year=2014&pages=1920-2&pmid=24618464&doi=10.1093/bioinformatics/btu137&)

21. Altman DG, McShane LM, Sauerbrei W, Taube SE. Reporting Recommendations for Tumor Marker Prognostic Studies (REMARK): explanation and elaboration. PLoS Med. 2012;9:e1001216. doi: 10.1371/journal.pmed.1001216.  [DOI](https://doi.org/10.1371/journal.pmed.1001216) | [PMC free article](/articles/PMC3362085/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22675273/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Med&title=Reporting%20Recommendations%20for%20Tumor%20Marker%20Prognostic%20Studies%20(REMARK):%20explanation%20and%20elaboration&author=DG%20Altman&author=LM%20McShane&author=W%20Sauerbrei&author=SE%20Taube&volume=9&publication_year=2012&pages=e1001216&pmid=22675273&doi=10.1371/journal.pmed.1001216&)

22. Melzer D, Perry JR, Hernandez D, Corsi AM, Stevens K, Rafferty I, et al. A genome-wide association study identifies protein quantitative trait loci (pQTLs) PLoS Genet. 2008;4:e1000072. doi: 10.1371/journal.pgen.1000072.  [DOI](https://doi.org/10.1371/journal.pgen.1000072) | [PMC free article](/articles/PMC2362067/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18464913/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet&title=A%20genome-wide%20association%20study%20identifies%20protein%20quantitative%20trait%20loci%20(pQTLs)&author=D%20Melzer&author=JR%20Perry&author=D%20Hernandez&author=AM%20Corsi&author=K%20Stevens&volume=4&publication_year=2008&pages=e1000072&pmid=18464913&doi=10.1371/journal.pgen.1000072&)

23. Ota T, Suzuki Y, Nishikawa T, Otsuki T, Sugiyama T, Irie R, et al. Complete sequencing and characterization of 21,243 full-length human cDNAs. Nat Genet. 2004;36:40–5. doi: 10.1038/ng1285.  [DOI](https://doi.org/10.1038/ng1285) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14702039/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Complete%20sequencing%20and%20characterization%20of%2021,243%20full-length%20human%20cDNAs&author=T%20Ota&author=Y%20Suzuki&author=T%20Nishikawa&author=T%20Otsuki&author=T%20Sugiyama&volume=36&publication_year=2004&pages=40-5&pmid=14702039&doi=10.1038/ng1285&)

24. Ho MF, Bongartz T, Liu M, Kalari KR, Goss PE, Shepherd LE, et al. Estrogen, SNP-Dependent Chemokine Expression and Selective Estrogen Receptor Modulator Regulation. Mol Endocrinol. 2016;30:382–98. doi: 10.1210/me.2015-1267.  [DOI](https://doi.org/10.1210/me.2015-1267) | [PMC free article](/articles/PMC4771694/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26866883/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Endocrinol&title=Estrogen,%20SNP-Dependent%20Chemokine%20Expression%20and%20Selective%20Estrogen%20Receptor%20Modulator%20Regulation&author=MF%20Ho&author=T%20Bongartz&author=M%20Liu&author=KR%20Kalari&author=PE%20Goss&volume=30&publication_year=2016&pages=382-98&pmid=26866883&doi=10.1210/me.2015-1267&)

25. Ingle JN, Liu M, Wickerham DL, Schaid DJ, Wang L, Mushiroda T, et al. Selective estrogen receptor modulators and pharmacogenomic variation in ZNF423 regulation of BRCA1 expression: individualized breast cancer prevention. Cancer Discov. 2013;3:812–25. doi: 10.1158/2159-8290.CD-13-0038.  [DOI](https://doi.org/10.1158/2159-8290.CD-13-0038) | [PMC free article](/articles/PMC3710533/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23764426/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Discov&title=Selective%20estrogen%20receptor%20modulators%20and%20pharmacogenomic%20variation%20in%20ZNF423%20regulation%20of%20BRCA1%20expression:%20individualized%20breast%20cancer%20prevention&author=JN%20Ingle&author=M%20Liu&author=DL%20Wickerham&author=DJ%20Schaid&author=L%20Wang&volume=3&publication_year=2013&pages=812-25&pmid=23764426&doi=10.1158/2159-8290.CD-13-0038&)

26. Comprehensive molecular portraits of human breast tumours. Nature. 2012;490:61–70. doi: 10.1038/nature11412.  [DOI](https://doi.org/10.1038/nature11412) | [PMC free article](/articles/PMC3465532/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23000897/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Comprehensive%20molecular%20portraits%20of%20human%20breast%20tumours&volume=490&publication_year=2012&pages=61-70&pmid=23000897&doi=10.1038/nature11412&)

27. Li L, Fridley B, Kalari K, Jenkins G, Batzler A, Safgren S, et al. Gemcitabine and cytosine arabinoside cytotoxicity: association with lymphoblastoid cell expression. Cancer Res. 2008;68:7050–8. doi: 10.1158/0008-5472.CAN-08-0405.  [DOI](https://doi.org/10.1158/0008-5472.CAN-08-0405) | [PMC free article](/articles/PMC2562356/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18757419/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=Gemcitabine%20and%20cytosine%20arabinoside%20cytotoxicity:%20association%20with%20lymphoblastoid%20cell%20expression&author=L%20Li&author=B%20Fridley&author=K%20Kalari&author=G%20Jenkins&author=A%20Batzler&volume=68&publication_year=2008&pages=7050-8&pmid=18757419&doi=10.1158/0008-5472.CAN-08-0405&)

28. Macedo LF, Guo Z, Tilghman SL, Sabnis GJ, Qiu Y, Brodie A. Role of androgens on MCF-7 breast cancer cell growth and on the inhibitory effect of letrozole. Cancer Res. 2006;66:7775–82. doi: 10.1158/0008-5472.CAN-05-3984.  [DOI](https://doi.org/10.1158/0008-5472.CAN-05-3984) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16885381/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=Role%20of%20androgens%20on%20MCF-7%20breast%20cancer%20cell%20growth%20and%20on%20the%20inhibitory%20effect%20of%20letrozole&author=LF%20Macedo&author=Z%20Guo&author=SL%20Tilghman&author=GJ%20Sabnis&author=Y%20Qiu&volume=66&publication_year=2006&pages=7775-82&pmid=16885381&doi=10.1158/0008-5472.CAN-05-3984&)

29. Yue W, Zhou D, Chen S, Brodie A. A new nude mouse model for postmenopausal breast cancer using MCF-7 cells transfected with the human aromatase gene. Cancer Res. 1994;54:5092–5.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/7923123/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=A%20new%20nude%20mouse%20model%20for%20postmenopausal%20breast%20cancer%20using%20MCF-7%20cells%20transfected%20with%20the%20human%20aromatase%20gene&author=W%20Yue&author=D%20Zhou&author=S%20Chen&author=A%20Brodie&volume=54&publication_year=1994&pages=5092-5&pmid=7923123&)

30. Giamas G, Filipovic A, Jacob J, Messier W, Zhang H, Yang D, et al. Kinome screening for regulators of the estrogen receptor identifies LMTK3 as a new therapeutic target in breast cancer. Nat Med. 2011;17:715–9. doi: 10.1038/nm.2351.  [DOI](https://doi.org/10.1038/nm.2351) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21602804/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Med&title=Kinome%20screening%20for%20regulators%20of%20the%20estrogen%20receptor%20identifies%20LMTK3%20as%20a%20new%20therapeutic%20target%20in%20breast%20cancer&author=G%20Giamas&author=A%20Filipovic&author=J%20Jacob&author=W%20Messier&author=H%20Zhang&volume=17&publication_year=2011&pages=715-9&pmid=21602804&doi=10.1038/nm.2351&)

31. Guo S, Sonenshein GE. Forkhead box transcription factor FOXO3a regulates estrogen receptor alpha expression and is repressed by the Her-2/neu/phosphatidylinositol 3-kinase/Akt signaling pathway. Mol Cell Biol. 2004;24:8681–90. doi: 10.1128/MCB.24.19.8681-8690.2004.  [DOI](https://doi.org/10.1128/MCB.24.19.8681-8690.2004) | [PMC free article](/articles/PMC516736/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15367686/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Cell%20Biol&title=Forkhead%20box%20transcription%20factor%20FOXO3a%20regulates%20estrogen%20receptor%20alpha%20expression%20and%20is%20repressed%20by%20the%20Her-2/neu/phosphatidylinositol%203-kinase/Akt%20signaling%20pathway&author=S%20Guo&author=GE%20Sonenshein&volume=24&publication_year=2004&pages=8681-90&pmid=15367686&doi=10.1128/MCB.24.19.8681-8690.2004&)

32. Zou Y, Tsai WB, Cheng CJ, Hsu C, Chung YM, Li PC, et al. Forkhead box transcription factor FOXO3a suppresses estrogen-dependent breast cancer cell proliferation and tumorigenesis. Breast Cancer Res. 2008;10:R21. doi: 10.1186/bcr1872.  [DOI](https://doi.org/10.1186/bcr1872) | [PMC free article](/articles/PMC2374977/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18312651/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Breast%20Cancer%20Res&title=Forkhead%20box%20transcription%20factor%20FOXO3a%20suppresses%20estrogen-dependent%20breast%20cancer%20cell%20proliferation%20and%20tumorigenesis&author=Y%20Zou&author=WB%20Tsai&author=CJ%20Cheng&author=C%20Hsu&author=YM%20Chung&volume=10&publication_year=2008&pages=R21&pmid=18312651&doi=10.1186/bcr1872&)

33. Morelli C, Lanzino M, Garofalo C, Maris P, Brunelli E, Casaburi I, et al. Akt2 inhibition enables the forkhead transcription factor FoxO3a to have a repressive role in estrogen receptor alpha transcriptional activity in breast cancer cells. Mol Cell Biol. 2010;30:857–70. doi: 10.1128/MCB.00824-09.  [DOI](https://doi.org/10.1128/MCB.00824-09) | [PMC free article](/articles/PMC2812247/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19933843/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Cell%20Biol&title=Akt2%20inhibition%20enables%20the%20forkhead%20transcription%20factor%20FoxO3a%20to%20have%20a%20repressive%20role%20in%20estrogen%20receptor%20alpha%20transcriptional%20activity%20in%20breast%20cancer%20cells&author=C%20Morelli&author=M%20Lanzino&author=C%20Garofalo&author=P%20Maris&author=E%20Brunelli&volume=30&publication_year=2010&pages=857-70&pmid=19933843&doi=10.1128/MCB.00824-09&)

34. Musgrove EA, Sutherland RL. Biological determinants of endocrine resistance in breast cancer. Nat Rev Cancer. 2009;9:631–43. doi: 10.1038/nrc2713.  [DOI](https://doi.org/10.1038/nrc2713) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19701242/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Cancer&title=Biological%20determinants%20of%20endocrine%20resistance%20in%20breast%20cancer&author=EA%20Musgrove&author=RL%20Sutherland&volume=9&publication_year=2009&pages=631-43&pmid=19701242&doi=10.1038/nrc2713&)

35. Osborne CK, Schiff R. Mechanisms of endocrine resistance in breast cancer. Annu Rev Med. 2011;62:233–47. doi: 10.1146/annurev-med-070909-182917.  [DOI](https://doi.org/10.1146/annurev-med-070909-182917) | [PMC free article](/articles/PMC3656649/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20887199/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Annu%20Rev%20Med&title=Mechanisms%20of%20endocrine%20resistance%20in%20breast%20cancer&author=CK%20Osborne&author=R%20Schiff&volume=62&publication_year=2011&pages=233-47&pmid=20887199&doi=10.1146/annurev-med-070909-182917&)

36. Ma CX, Reinert T, Chmielewska I, Ellis MJ. Mechanisms of aromatase inhibitor resistance. Nat Rev Cancer. 2015;15:261–75. doi: 10.1038/nrc3920.  [DOI](https://doi.org/10.1038/nrc3920) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25907219/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Cancer&title=Mechanisms%20of%20aromatase%20inhibitor%20resistance&author=CX%20Ma&author=T%20Reinert&author=I%20Chmielewska&author=MJ%20Ellis&volume=15&publication_year=2015&pages=261-75&pmid=25907219&doi=10.1038/nrc3920&)

37. Schroth W, Goetz MP, Hamann U, Fasching PA, Schmidt M, Winter S, et al. Association between CYP2D6 polymorphisms and outcomes among women with early stage breast cancer treated with tamoxifen. Jama. 2009;302:1429–36. doi: 10.1001/jama.2009.1420.  [DOI](https://doi.org/10.1001/jama.2009.1420) | [PMC free article](/articles/PMC3909953/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19809024/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Jama&title=Association%20between%20CYP2D6%20polymorphisms%20and%20outcomes%20among%20women%20with%20early%20stage%20breast%20cancer%20treated%20with%20tamoxifen&author=W%20Schroth&author=MP%20Goetz&author=U%20Hamann&author=PA%20Fasching&author=M%20Schmidt&volume=302&publication_year=2009&pages=1429-36&pmid=19809024&doi=10.1001/jama.2009.1420&)

38. Iyer MK, Niknafs YS, Malik R, Singhal U, Sahu A, Hosono Y, et al. The landscape of long noncoding RNAs in the human transcriptome. Nat Genet. 2015;47:199–208. doi: 10.1038/ng.3192.  [DOI](https://doi.org/10.1038/ng.3192) | [PMC free article](/articles/PMC4417758/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25599403/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=The%20landscape%20of%20long%20noncoding%20RNAs%20in%20the%20human%20transcriptome&author=MK%20Iyer&author=YS%20Niknafs&author=R%20Malik&author=U%20Singhal&author=A%20Sahu&volume=47&publication_year=2015&pages=199-208&pmid=25599403&doi=10.1038/ng.3192&)

39. Rinn JL, Chang HY. Genome regulation by long noncoding RNAs. Annu Rev Biochem. 2012;81:145–66. doi: 10.1146/annurev-biochem-051410-092902.  [DOI](https://doi.org/10.1146/annurev-biochem-051410-092902) | [PMC free article](/articles/PMC3858397/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22663078/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Annu%20Rev%20Biochem&title=Genome%20regulation%20by%20long%20noncoding%20RNAs&author=JL%20Rinn&author=HY%20Chang&volume=81&publication_year=2012&pages=145-66&pmid=22663078&doi=10.1146/annurev-biochem-051410-092902&)

40. Paci P, Colombo T, Farina L. Computational analysis identifies a sponge interaction network between long non-coding RNAs and messenger RNAs in human breast cancer. BMC Syst Biol. 2014;8:83. doi: 10.1186/1752-0509-8-83.  [DOI](https://doi.org/10.1186/1752-0509-8-83) | [PMC free article](/articles/PMC4113672/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25033876/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Syst%20Biol&title=Computational%20analysis%20identifies%20a%20sponge%20interaction%20network%20between%20long%20non-coding%20RNAs%20and%20messenger%20RNAs%20in%20human%20breast%20cancer&author=P%20Paci&author=T%20Colombo&author=L%20Farina&volume=8&publication_year=2014&pages=83&pmid=25033876&doi=10.1186/1752-0509-8-83&)

41. Tsai MC, Spitale RC, Chang HY. Long intergenic noncoding RNAs: new links in cancer progression. Cancer Res. 2011;71:3–7. doi: 10.1158/0008-5472.CAN-10-2483.  [DOI](https://doi.org/10.1158/0008-5472.CAN-10-2483) | [PMC free article](/articles/PMC3057914/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21199792/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=Long%20intergenic%20noncoding%20RNAs:%20new%20links%20in%20cancer%20progression&author=MC%20Tsai&author=RC%20Spitale&author=HY%20Chang&volume=71&publication_year=2011&pages=3-7&pmid=21199792&doi=10.1158/0008-5472.CAN-10-2483&)

42. Cheetham SW, Gruhl F, Mattick JS, Dinger ME. Long noncoding RNAs and the genetics of cancer. Br J Cancer. 2013;108:2419–25. doi: 10.1038/bjc.2013.233.  [DOI](https://doi.org/10.1038/bjc.2013.233) | [PMC free article](/articles/PMC3694235/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23660942/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Cancer&title=Long%20noncoding%20RNAs%20and%20the%20genetics%20of%20cancer&author=SW%20Cheetham&author=F%20Gruhl&author=JS%20Mattick&author=ME%20Dinger&volume=108&publication_year=2013&pages=2419-25&pmid=23660942&doi=10.1038/bjc.2013.233&)

43. Gupta RA, Shah N, Wang KC, Kim J, Horlings HM, Wong DJ, et al. Long non-coding RNA HOTAIR reprograms chromatin state to promote cancer metastasis. Nature. 2010;464:1071–6. doi: 10.1038/nature08975.  [DOI](https://doi.org/10.1038/nature08975) | [PMC free article](/articles/PMC3049919/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20393566/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Long%20non-coding%20RNA%20HOTAIR%20reprograms%20chromatin%20state%20to%20promote%20cancer%20metastasis&author=RA%20Gupta&author=N%20Shah&author=KC%20Wang&author=J%20Kim&author=HM%20Horlings&volume=464&publication_year=2010&pages=1071-6&pmid=20393566&doi=10.1038/nature08975&)

44. Hou P, Zhao Y, Li Z, Yao R, Ma M, Gao Y, et al. LincRNA-ROR induces epithelial-to-mesenchymal transition and contributes to breast cancer tumorigenesis and metastasis. Cell Death Dis. 2014;5:e1287. doi: 10.1038/cddis.2014.249.  [DOI](https://doi.org/10.1038/cddis.2014.249) | [PMC free article](/articles/PMC4611722/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24922071/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell%20Death%20Dis&title=LincRNA-ROR%20induces%20epithelial-to-mesenchymal%20transition%20and%20contributes%20to%20breast%20cancer%20tumorigenesis%20and%20metastasis&author=P%20Hou&author=Y%20Zhao&author=Z%20Li&author=R%20Yao&author=M%20Ma&volume=5&publication_year=2014&pages=e1287&pmid=24922071&doi=10.1038/cddis.2014.249&)

45. Xue X, Yang YA, Zhang A, Fong KW, Kim J, Song B, et al. LncRNA HOTAIR enhances ER signaling and confers tamoxifen resistance in breast cancer. Oncogene. 2016;35:2746–55. doi: 10.1038/onc.2015.340.  [DOI](https://doi.org/10.1038/onc.2015.340) | [PMC free article](/articles/PMC4791209/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26364613/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncogene&title=LncRNA%20HOTAIR%20enhances%20ER%20signaling%20and%20confers%20tamoxifen%20resistance%20in%20breast%20cancer&author=X%20Xue&author=YA%20Yang&author=A%20Zhang&author=KW%20Fong&author=J%20Kim&volume=35&publication_year=2016&pages=2746-55&pmid=26364613&doi=10.1038/onc.2015.340&)

46. Tomita S, Abdalla MO, Fujiwara S, Matsumori H, Maehara K, Ohkawa Y, et al. A cluster of noncoding RNAs activates the ESR1 locus during breast cancer adaptation. Nat Commun. 2015;6:6966. doi: 10.1038/ncomms7966.  [DOI](https://doi.org/10.1038/ncomms7966) | [PMC free article](/articles/PMC4421845/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25923108/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Commun&title=A%20cluster%20of%20noncoding%20RNAs%20activates%20the%20ESR1%20locus%20during%20breast%20cancer%20adaptation&author=S%20Tomita&author=MO%20Abdalla&author=S%20Fujiwara&author=H%20Matsumori&author=K%20Maehara&volume=6&publication_year=2015&pages=6966&pmid=25923108&doi=10.1038/ncomms7966&)

47. Deblois G, Giguere V. Ligand-independent coactivation of ERalpha AF-1 by steroid receptor RNA activator (SRA) via MAPK activation. J Steroid Biochem Mol Biol. 2003;85:123–31. doi: 10.1016/s0960-0760(03)00225-5.  [DOI](https://doi.org/10.1016/s0960-0760(03)00225-5) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12943696/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Steroid%20Biochem%20Mol%20Biol&title=Ligand-independent%20coactivation%20of%20ERalpha%20AF-1%20by%20steroid%20receptor%20RNA%20activator%20(SRA)%20via%20MAPK%20activation&author=G%20Deblois&author=V%20Giguere&volume=85&publication_year=2003&pages=123-31&pmid=12943696&doi=10.1016/s0960-0760(03)00225-5&)

48. Hayes EL, Lewis-Wambi JS. Mechanisms of endocrine resistance in breast cancer: an overview of the proposed roles of noncoding RNA. Breast Cancer Res. 2015;17:40. doi: 10.1186/s13058-015-0542-y.  [DOI](https://doi.org/10.1186/s13058-015-0542-y) | [PMC free article](/articles/PMC4362832/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25849966/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Breast%20Cancer%20Res&title=Mechanisms%20of%20endocrine%20resistance%20in%20breast%20cancer:%20an%20overview%20of%20the%20proposed%20roles%20of%20noncoding%20RNA&author=EL%20Hayes&author=JS%20Lewis-Wambi&volume=17&publication_year=2015&pages=40&pmid=25849966&doi=10.1186/s13058-015-0542-y&)

49. Adams BD, Anastasiadou E, Esteller M, He L, Slack FJ. The Inescapable Influence of Noncoding RNAs in Cancer. Cancer Res. 2015;75:5206–10. doi: 10.1158/0008-5472.CAN-15-1989.  [DOI](https://doi.org/10.1158/0008-5472.CAN-15-1989) | [PMC free article](/articles/PMC4823153/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26567137/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=The%20Inescapable%20Influence%20of%20Noncoding%20RNAs%20in%20Cancer&author=BD%20Adams&author=E%20Anastasiadou&author=M%20Esteller&author=L%20He&author=FJ%20Slack&volume=75&publication_year=2015&pages=5206-10&pmid=26567137&doi=10.1158/0008-5472.CAN-15-1989&)

50. Reid G, Denger S, Kos M, Gannon F. Human estrogen receptor-alpha: regulation by synthesis, modification and degradation. Cell Mol Life Sci. 2002;59:821–31. doi: 10.1007/s00018-002-8470-2.  [DOI](https://doi.org/10.1007/s00018-002-8470-2) | [PMC free article](/articles/PMC11146101/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12088282/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell%20Mol%20Life%20Sci&title=Human%20estrogen%20receptor-alpha:%20regulation%20by%20synthesis,%20modification%20and%20degradation&author=G%20Reid&author=S%20Denger&author=M%20Kos&author=F%20Gannon&volume=59&publication_year=2002&pages=821-31&pmid=12088282&doi=10.1007/s00018-002-8470-2&)
