# Genetic variants in interferon-λ 4 influences HCV clearance in Chinese Han population

## Metadata
**Authors:** Peng Huang, Yinan Yao, Ming Yue, Ting Tian, Hongbo Chen, Mingzhu Chen, Jie Wang, Yun Zhang, Rongbin Yu
**Journal:** Scientific Reports
**Date:** 2017 Feb 10
**DOI:** [10.1038/srep42408](https://doi.org/10.1038/srep42408)
**PMID:** 28186161
**PMCID:** PMC5301237
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5301237/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC5301237/pdf/srep42408.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC5301237/pdf/srep42408.pdf)

## Abstract

Recent many studies indicated a novel dinucleotide variant in ss469415590 (TT vs. ΔG) of interferon-λ 4 (IFNL4) gene strongly associated with hepatitis C virus clearance. To evaluate the impact and clinical usefulness of IFNL4 ss469415590 genotype on predicting both spontaneous HCV clearance and response to therapy in Chinese population, we genotyped 795 chronic HCV carriers, 460 subjects with HCV natural clearance and 362 patients with pegylated interferon-α and ribavirin (PEG IFN-α/RBV) treatment. IFNL4 ss469415590 variant genotypes significantly decreased host HCV clearance, both spontaneous (dominant model: OR = 0.50, 95% CI = 0.36–0.71) and IFN-α induced (dominant model: OR = 0.32, 95% CI = 0.18–0.56). Multivariate stepwise analysis indicated that ss469415590, rs12979860, the level of baseline HCV RNA and platelet were as independent predictors for sustained virological response (SVR). But the area under the ROC curve (AUC) was only 0.58 for ss469415590, and it was elevated to 0.71 by adding rs12979860, baseline HCV RNA and platelet in the prediction model of SVR. Therefore, these findings underscore that although genetic factors of host and pathogen were commonly important during HCV clearance, ss469415590 may be also a strongly predictive marker in the Chinese population.

## Results

### Participant characteristics

Participant profiles were shown in [Table 1](#t1). Among 795 persistent HCV carriers and 460 subjects with spontaneous clearance, elder, female and subject infected with viral genotype 1 were significantly more likely to be chronic infection (*P* < 0.05 for both comparisons). And patients with spontaneous HCV clearance had normal aspartate aminotransferase (AST) and alanine aminotransferase (ALT) than persistent HCV cases (*P* < 0.001 for AST and *P* = 0.015 for ALT). In this study, 362 chronic infection patients treated with PEG IFN-α/RBV therapy were recruited from a population of former paid-blood donors, which the viral genotype of these CHC were all genotype 1. 65.7% patients achieved SVR. There were more patients with abnormal AST, higher baseline HCV RNA, and lower platelet among none sustained virological response (N-SVR) group (*P* = 0.038, <0.01, 0.039, respectively).

### Table 1. Patient characteristics.

| Variables | Spontaneous clearance (n = 1255) |  |  | Response to treatment (n = 362) |  |  |
| --- | --- | --- | --- | --- | --- | --- |
| Chronic (n = 795) N(%) | Resolver (n = 460) N(%) | P value | N-SVR (n = 125) N(%) | SVR (n = 237) N(%) | P value |  |
| Age, yeara | 52.44 ± 12.30 | 47.05 ± 14.04 | <0.001 | 53.91 ± 8.06 | 53.20 ± 8.81 | 0.456 |
| Gender |   |   | 0.017 |   |   | 0.883 |
| Male | 301 (37.9) | 206 (44.8) |   | 30 (24.0) | 59 (24.9) |   |
| Female | 494 (62.1) | 254 (55.2) |   | 95 (76.0) | 178 (75.1) |   |
| Viral genotype |   |   | 0.002 |   |   |   |
| 1 | 635 (81.9) | 300 (74.1) |   | 124 (100) | 237 (100) |   |
| 2 or 3 | 140 (18.1) | 105 (25.9) |   | 0 | 0 |   |
| AST (IU/L) |   |   | <0.001 |   |   | 0.038 |
| <40 | 468 (58.9) | 340 (73.9) |   | 47 (37.6) | 117 (49.4) |   |
| ≥40 | 327 (41.1) | 120 (26.1) |   | 78 (62.4) | 120 (50.6) |   |
| ALT (IU/L) |   |   | 0.015 |   |   | 0.055 |
| <40 | 509 (64.0) | 338 (73.5) |   | 39 (31.2) | 99 (41.8) |   |
| ≥40 | 286 (36.0) | 122 (26.5) |   | 86 (68.8) | 138 (58.2) |   |
| HCV RNA (IU/L) |   |   |   |   |   | <0.001 |
| <106 |   |   |   | 33 (26.4) | 113 (47.7) |   |
| ≥106 |   |   |   | 92 (73.6) | 124 (52.3) |   |
| Platelets (109/L) |   |   |   |   |   | 0.039 |
| <140 |   |   |   | 81 (64.8) | 126 (53.2) |   |
| ≥140 |   |   |   | 44 (35.2) | 111 (46.8) |   |
| ALB (g/L)b |   |   |   | 43.6 (40.8–46.2) | 43.6 (40.9–46.2) | 0.607 |
| TP (g/L)b |   |   |   | 78 (73.8–82.5) | 78.1 (74.3–81.5) | 0.920 |
| WBC (109/L)b |   |   |   | 4.9 (3.9–6.1) | 4.6 (3.7–5.5) | 0.065 |
| AFP (ng/mL) b |   |   |   | 3.8 (2.4–7.4) | 3.3 (2.0–6.8) | 0.160 |
| HGB (g/L)b |   |   |   | 139 (125–146) | 133.5 (121–145) | 0.056 |
| GLU (mmol/L)b |   |   |   | 5.5 (5.1–6.4) | 5.6 (4.9–6.2) | 0.531 |

Table 1 Caption: Abbreviation: N-SVR, non sustained virological response; SVR, sustained virological response; Non-1b, genotype 2, 3; AST, aspartate transaminase; ALT, alanine aminotransferase; ALB, albumin; TP, total protein; WBC, white blood cell; AFP, alpha fetal protein; HGB, hemoglobin; GLU, glucose. aMean ± standard deviation. bMedian and interquartile range.

The observed genotype frequencies for the two SNPs in subjects with spontaneous HCV clearance were all in Hardy-Weinberg equilibrium (HWE) (*P* > 0.05). But the allele frequencies of rs12979860 were not in accordance with the predicted HWE in patients who attained SVR (*P* < 0.001). The LD was moderate between rs12979860 and ss469415590 both for spontaneous resolver (r^2^ = 0.62) and for patients who attained SVR (r^2^ = 0.79).

### ss469415590 variants and HCV spontaneous clearance

*IFNL4* ss469415590 genotype had a significant impact on spontaneous HCV clearance. Clearance rate was higher in patients carrying the beneficial TT/TT genotype (39.4%) than in patients with TT/ΔG (23.8%) or ΔG/ΔG (18.7%) (*P* < 0.001). Meanwhile, *IL28B* rs12979860 variants were also negatively related with viral clearance and clearance rate was 38.9%, 27.6% and 17.2% for CC, CT and TT genotype (*P* < 0.001). After adjusting for age, gender, and viral genotype, logistic regression analyses showed that *IFNL4* ss469415590 and *IL28B* rs12979860 variant genotypes significantly decreased the ability of host HCV clearance (dominant model: OR = 0.50, 95% CI = 0.36–0.71 for ss469415590; OR = 0.59, 95% CI = 0.42–0.83 for rs12979860) ([Table 2](#t2)). This effect of ss469415590 and rs12979860 on HCV clearance remained in existence after being conditioned on each other (*P* = 0.005 for ss469415590 and *P* = 0.016 for rs12979860) and therefore they was included in further combined analyses by adding up the number of unfavorable alleles of the independent SNPs ss469415590-ΔG and rs12979860-T. The result showed that subjects carrying three to four unfavorable alleles had a 90% decrease in clearance in comparison with those without unfavorable allele (OR = 0.10, 95% CI = 0.02–0.42) ([Table 3](#t3)).

### Table 2. Association of selected polymorphisms with HCV clearance and SVR.

| Spontaneous clearance (n = 1255) |  |  |  |  |  |
| --- | --- | --- | --- | --- | --- |
| Genotype | Chronic N(%) | Resolver N(%) | Clearance rate | OR (95% CI)a | P valuea |
| rs12979860 |  |  |  |  |  |
| CC | 634 (79.7) | 404 (87.8) | 38.9 | 1.00 |   |
| CT | 134 (16.9) | 51 (11.1) | 27.6 | 0.63 (0.44–0.90) | 0.010 |
| TT | 27 (3.4) | 5 (1.1) | 17.2 | 0.37 (0.14–0.99) | 0.049 |
| Dominant |   |   |   | 0.59 (0.42–0.83) | 0.002 |
| Recessive |   |   |   | 0.40 (0.15–1.06) | 0.066 |
| Additive |   |   |   | 0.62 (0.46–0.83) | 0.002 |
| ss469415590 |  |  |  |  |  |
| TT/TT | 628 (79.0) | 409 (88.9) | 39.4 | 1.00 |   |
| TT/ΔG | 154 (19.4) | 48 (10.4) | 23.8 | 0.51 (0.36–0.73) | <0.001 |
| ΔG/ΔG | 13 (1.6) | 3 (0.7) | 18.7 | 0.37 (0.11–1.34) | 0.130 |
| Dominant |   |   |   | 0.50 (0.36–0.71) | <0.001 |
| Recessive |   |   |   | 0.41 (0.12–1.47) | 0.172 |
| Additive |   |   |   | 0.53 (0.39–0.74) | <0.001 |
| Response to treatment (n = 362) |  |  |  |  |  |
| Genotype | N-SVR N(%) | SVR N(%) | SVR rate | OR (95% CI)b | P valueb |
| rs12979860 |  |  |  |  |  |
| CC | 65 (52.0) | 180 (76.0) | 73.5 | 1.00 |   |
| CT | 37 (29.6) | 42 (17.7) | 53.2 | 0.41 (0.24–0.69) | <0.001 |
| TT | 23 (18.4) | 15 (6.3) | 39.5 | 0.24 (0.12–0.48) | <0.001 |
| Dominant |   |   |   | 0.34 (0.22–0.54) | <0.001 |
| Recessive |   |   |   | 0.30 (0.15–0.60) | 0.001 |
| Additive |   |   |   | 0.46 (0.34–0.64) | <0.001 |
| ss469415590 |  |  |  |  |  |
| TT/TT | 91 (72.8) | 212 (89.5) | 70.0 | 1.00 |   |
| TT/ΔG | 31 (24.8) | 24 (10.1) | 43.6 | 0.33 (0.18–0.60) | <0.001 |
| ΔG/ΔG | 3 (2.4) | 1 (0.4) | 25.0 | 0.14 (0.02–1.39) | 0.094 |
| Dominant |   |   |   | 0.32 (0.18–0.56) | <0.001 |
| Recessive |   |   |   | 0.17 (0.02–1.67) | 0.130 |
| Additive |   |   |   | 0.34 (0.20–0.58) | <0.001 |

Table 2 Caption: Abbreviation: N-SVR, non sustained virological response; SVR, sustained virological response. aLogistic regression analyses adjusted for age, gender, and viral genotype. bLogistic regression analyses adjusted for age, gender, HCV RNA and platelets.

### Table 3. Cumulative effects of rs12979860 and ss469415590 on HCV clearance and SVR.

| Variablesa | Spontaneous clearance |  |  | Response to treatment |  |  |
| --- | --- | --- | --- | --- | --- | --- |
| Chronic N(%) | Resolver N(%) | OR (95% CI)b | N-SVR N(%) | SVR N(%) | OR (95% CI)c |  |
| 0 | 572 (71.9) | 385 (83.7) | 1.00 | 61 (48.8) | 174 (73.5) | 1.00 |
| 1 | 111 (14.0) | 39 (8.5) | 0.52 (0.35–0.77) | 16 (12.8) | 34 (14.3) | 0.81 (0.39–1.65) |
| 2 | 82 (10.3) | 34 (7.4) | 0.62 (0.40–0.94) | 42 (33.6) | 23 (9.7) | 0.13 (0.07–0.25) |
| 3–4 | 30 (3.8) | 2 (0.4) | 0.10 (0.02–0.42) | 6 (4.8) | 6 (2.5) | 0.18 (0.05–0.65) |
| Trend |   |   | Pd < 0.001 |   |   | Pd < 0.001 |
| 0 | 572 (71.9) | 385 (83.7) | 1.00 | 61 (48.8) | 174 (73.5) | 1.00 |
| 1–4 | 223 (28.1) | 75 (16.3) | 0.50 (0.37–0.67) | 64 (51.2) | 63 (26.5) | 0.28 (0.17–0.46) |

Table 3 Caption: Abbreviation: N-SVR, non sustained virological response; SVR, sustained virological response. aVariables are numbers of combined unfavorable genotypes (rs12979860 T and ss469415590 ΔG). bLogistic regression analyses adjusted for age, gender, and viral genotype. cLogistic regression analyses adjusted for age, gender, HCV RNA and platelets. dP value of Cochran-Armitage’s trend test.

### ss469415590 variants and response to treatment

In 361 HCV genotype 1 patients, SVR was strongly associated with *IFNL4* ss469415590 genotype: 70.0% of patients with ss469415590 TT/TT had SVR, whereas only 43.6% of patients with TT/ΔG and 16.7% of ΔG/ΔG achieved SVR (*P* < 0.001). Subjects with *IL28B* rs12979860 variant TT were also fewer to achieve SVR ([Table 2](#t2)). Multivariate stepwise analysis indicated that these two SNPs, the level of baseline HCV RNA and platelet were as independent predictors for SVR ([Table 4](#t4)). After adjustment with age, gender, baseline HCV RNA and platelet, multivariate logistic regression showed that the OR of ss469415590 and rs12979860 was 0.32 and 0.34 in dominant model respectively, compared with patients with favorable genotypes (Table). The combined effect of unfavorable alleles on SVR indicated that carrying two unfavorable alleles offered the highest risk effect (OR = 0.13, 95% CI = 0.07–0.25), as showed in [Table 3](#t3).

### Table 4. Multivariate stepwise regression analysis for independent factors of SVR.

| Variables | Coef. | SE | OR (95% CI) | P value |
| --- | --- | --- | --- | --- |
| rs12979860 | −0.133 | 0.041 | 0.43 (0.25–0.75) | 0.003 |
| HCV RNA | −0.249 | 0.051 | 1.65 (1.02–2.67) | 0.043 |
| ss469415590 | −0.222 | 0.067 | 0.29 (0.13–0.60) | <0.001 |
| Platelets | 1.870 | 0.049 | 0.28 (0.15–0.45) | <0.001 |

Table 4 Caption: Abbreviation: Coef, coefficient of variation; SE, standard error; SVR, sustained virological response.

We used decision tree ensembles in the form of a random forest classifier to quantify the relative predictive power of ss469415590, rs12979860, selected demographic characteristics, and clinical features. As a result of this multivariate analysis, the predictive power of a variable was expressed as the Gini. The [Fig. 1A](#f1) showed that the first four stronger factors were ss469415590, baseline HCV RNA, rs12979860 and platelet. The AUC was 0.58 for ss469415590, and was 0.64 for the set of ss469415590 and rs12979860 ([Fig. 1B,C](#f1)). But the AUC was elevated to 0.71 by adding baseline HCV RNA and platelet in the prediction model, suggesting that genetic factors of host and pathogen were commonly important to resolution of virus during treatment ([Fig. 1D](#f1)).

### Figure 1. Predictors of HCV treatment response.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e72e/5301237/a6c8f8e89f41/srep42408-f1.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5301237_srep42408-f1.jpg)

(A) Importance blots of all variables in 362 patients. (B) ROC curve for prediction of treatment response using the IFNL4 ss469415590. (C) ROC curve for prediction of treatment response using the combination of IFNL4 ss469415590 and IL28B rs12979860. (D) ROC curve with the combination of IFNL4 ss469415590, IL28B rs12979860, baseline HCV RNA and platelet to predict HCV treatment response.

Meanwhile ss469415590-ΔG variant (dominant model: OR = 0.26, 95% CI = 0.12–0.54 for RVR; OR = 0.11, 95% CI = 0.06–0.22 for EVR) and rs12979860-T variant (dominant model: OR = 0.37, 95% CI = 0.23–0.62 for RVR; OR = 0.46, 95% CI = 0.26–0.81 for EVR) were also negatively related with RVR and EVR ([Supplemental Table 2](#S1)).

### ss469415590 variants and HCV time-dependent clearance

Patients were stratified according to their *IFNL4* ss469415590 and *IL28B* rs12979860 allele type, and the rate of undetectable HCV-RNA at 4, 8, 12, 24, 48 weeks after the start of therapy and 24 weeks after cessation of treatment were analyzed ([Fig. 2A,B](#f2)). The rate of undetectable HCV RNA was significantly higher in patients with the ss469415590 TT and rs12979860 CC genotype than ss469415590 TT/ΔG or ΔG/ΔG and rs12979860 CT/TT genotype. Especially, the difference was most significant when stratified by ss469415590 genotype: 44.2% vs. 18.6% for 4 weeks, 67.7% vs. 30.5% for 12 weeks and 85.9% vs. 58.3% for 48 weeks.

### Figure 2. Effect of IFNL4 and IL28B variants on time-dependent clearance of HCV.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e72e/5301237/2bf1c4378fa5/srep42408-f2.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5301237_srep42408-f2.jpg)

(A) The rate of undetectable HCV RNA stratified by IFNL4 ss469415590 alleles. (B) The rate of undetectable HCV RNA stratified by IL28B rs12979860 alleles. *P < 0.05, **P < 0.01.

### Meta-analysis

To further study the impact of *IFNL4* ss469415590 on HCV clearance, a meta-analysis was performed by the published data, including four data for HCV spontaneous clearance and 12 data for HCV SVR ([Supplemental Table 3](#S1))[7](#b7),[8](#b8),[11](#b11),[12](#b12),[13](#b13),[14](#b14),[15](#b15),[16](#b16),[17](#b17). [Figure 3](#f3) indicated significant association between *IFNL4* ss469415590 variant and decreased ability of HCV clearance (pooled OR = 0.30, 95% CI = 0.23–0.39 for HCV spontaneous clearance; pooled OR = 0.26, 95% CI = 0.22–0.32 for HCV SVR in dominant model). Subgroup analysis showed that the estimated OR was 0.21 (95% CI = 0.17–0.27) in HCV genotype 1 and 0.36 (95% CI = 0.26–0.48) in other genotypes. These results indicated that patients with ss469415590 variant types may have a lower ability of HCV clearance, especially in viral genotype 1.

### Figure 3. Meta-analysis of the IFNL4 ss469415590 on HCV clearance in dominant genetic model.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e72e/5301237/a1a7c44f8680/srep42408-f3.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5301237_srep42408-f3.jpg)

The position of each symbol indicates the odds ratio (OR) for each included study; the horizontal lines indicate 95% confidence intervals (95%CIs). The vertical dashed lines represent the overall OR of all included studies. Arrows indicate 95%CIs that exceed the x-axis scale.

## Discussion

In this study, we investigated the associations between *IFNL4* ss469415590 and risk of HCV clearance and this is the first study that validated the observations of Prokunina-Olsson *et al*. and of Bibert *et al*. on the prediction of the eradication of HCV, both spontaneous and IFN-α induced, in a large cohort of Chinese Han populations[7](#b7),[8](#b8). The main finding was that the predictive value of *IFNL4* ss469415590 was almost identical to that of *IL28B* rs12979860, which were all consistent with previous studies in other races[12](#b12),[13](#b13),[14](#b14). But, unexpectedly, the linkage disequilibrium among these two SNPs in our study was only moderate, especially in spontaneous resolver (r^2^ = 0.62). Previous studies have reported that ss469415590 and rs12979860 were in strong linkage disequilibrium in chronic hepatitis C virus type 1 or 4 infection in Caucasians[12](#b12),[18](#b18). Both studies were conducted to predict the treatment response of peginterferon alpha and ribavirin. However, the moderate linkage disequilibrium in our study was performed in spontaneous resolver. Besides, the linkage disequilibrium between ss469415590 and rs12979860 in Chinese Han population may be not as strong as in Caucasians.

We performed a mini meta-analysis with a dominant genetic model by pooling the published data to study the effect of *IFNL4* ss469415590 on HCV clearance. The pooled ORs were lower than these in our current research for both spontaneous clearance and IFN-α induced eradication, suggesting the negative effectiveness of *IFNL4* ss469415590 variant was stronger in the meta-analysis. The mainly reason may be the discrepancy of genetic background in different races. The frequency of ss469415590-TT/ΔG or ΔG/ΔG was about 12.2% in Chinese patients with HCV spontaneous clearance of our study, while that was about 69.9% in non-Chinese patients of the meta-analysis. This extreme variation also was found in SVR patients infected with genotype 1 (10.5% vs. 60.8%). But this discrepancy of frequency of ss469415590-TT/ΔG or ΔG/ΔG in SVR patients infected with genotype 1 was reduced among our study and three Japanese studies of the meta-analysis (10.5% vs. 24.7%).

Although GWAS defined the strong association between *IL28B* SNPs and viral eradication in hepatitis C, the exact mechanism or function of *IL28B* has yet to be testified, and even some paradoxes existed in various researches. On one hand, based on different *IL28B* alleles, the level of expression of *IL28B* was dissimilar between in peripheral blood mononuclear cells (PBMCs) and in liver tissue. Many studies identified that the level of expression of *IL28B* was lower in PBMCs from subjects with unfavorable *IL28B* variants in comparison with patients carrying the wild-type gene. However, this findings was fewer confirmed in liver tissue[6](#b6),[19](#b19). On the other hand, the causal impact of IL28A which was a homology with IL28B on HCV clearance was hardly recognized, suggesting that IL28B may not be the true explanation of HCV clearance[9](#b9). Recently, several researches focused on the function of *IFNL4* which may provide the causal mechanism of HCV clearance. First of all, as is known, patients with a high induction of ISG in liver tissue would achieve a poor response to IFN-α. And ISGs upon transient expression in hepatocyte was induced by *IFNL4* gene transfection[20](#b20). In addition, the study of Bibert *et al*. confirmed that the level of IL28B and CXCL10 expression in poly(I:C)-stimulated PBMCs was strongly depended on ss469415590 genotypes but not on rs12979860[8](#b8). Covolo *et al*. validated the observations that circulating level of CXCL10 was significantly higher in patients with ss469415590 mutant alleles[12](#b12).

Although *IFNL4* ss469415590 is strongly associated with HCV clearance, it is uncertain that these genetic markers will be still needed if the upcoming IFN-free therapy will become available. We think this conjecture is not well founded. Firstly, many new treatments such as Daclatasvir and Sofosbuvir were still in clinical trials and such high payment lead it will not be available for the foreseeable future in developing countries[21](#b21). Secondly, with significantly high SVR rates and safety, these IFN-free therapies were researched only in recent years and still needed a long follow-up time to assess the safety and efficacy[22](#b22). Thus, interferon-based therapies will be used for many years in most countries. Finally, in the recent paper on the triple therapy with pegylated IFN-α2b, RBV and telaprevir, 176 patients with *IFNL4* ss469415590 TT/TT had better response rates than those carrying TT/ΔG or ΔG/ΔG alleles, suggesting genetics may still play a role for new therapy[16](#b16).

In this study the environment factors (baseline viral load and platelet) as well as genetic factors independently contribute to the response to treatment. The predict value of ss469415590 for SVR was only moderate (AUC = 0.58) and almost 30% patients carrying with beneficial IFNL4 TT/TT genotype did not achieve SVR. These results were confirmed by some recent findings, and reflected a complex genotype -environment interaction for predicting HCV clearance.

Our study had numerous strengths. First of all, enough statistical power was guaranteed with a relatively large sample size in this study, and it is the first study demonstrating that *IFNL4* ss469415590 variants influence HCV clearance in Chinese Han population, which validated the effect of the novel marker in non-Caucasians. Moreover, patients with treatment came from parts of HCV persistent carriers who were all infected with HCV by blood donation, which may have reduced potential selection bias. Finally, it is rare to study the association between *IFNL4* ss469415590 and HCV clearance, both spontaneous and IFN-α induced in one study. However, little is known regarding the biological mechanism of the significant SNP in the clearance of HCV. Therefore, validations with functional characterizations are warranted.

In conclusion, our study validated that *IFNL4* ss469415590 was also strongly associated with HCV clearance in Chinese Han population.

## Methods

### Ethical statement

This study was approved by the medical ethics committee of Nanjing Medical University. All subjects provided informed consent to participate in the study. The experiment was carried out according to the Declaration of Helsinki.

### Participants

1255 participants for studying spontaneous HCV clearance were recruited from the Nanjing compulsory detoxification center (Nanjing, China) during May and Dec 2006, nine hospital hemodialysis centers in southern China during Oct 2008 and Jan 2010, and a population of former paid-blood donors (Zhenjiang, China) from April 2010 to January 2013. The information about these participants had been described in our previous studies[23](#b23),[24](#b24). 362 treatment-naïve CHC patients were recruited from Jurong People’s Hospital (Jurong, China) from Jan 2011 to Oct 2013, which aimed to evaluate the factors that impacted on response to anti-viral therapy. Eligibility criteria for therapy included age between 18 and 70 years, detectable HCV RNA in serum over a span of more than 6 months of treatment initiation, negative for hepatitis B infection, without other types of liver diseases such as alcoholic diseases, autoimmune liver diseases or metabolic liver diseases.

Each participant was interviewed by a structured questionnaire to collect information on demographic data and environmental exposure history. After interview, an approximately 5 mL venous blood sample was collected from each participant. The serum and peripheral blood mononuclear cells were separated and stored at −70 °C until assay.

### Treatment therapy and outcome

Patients received weekly injections of pegylated IFN-α2a (180 g) and ribavirin (RBV) was administered orally for 48 weeks. The amount of ribavirin was adjusted based on body weight (600 mg for <60 kg, 800 mg for 60–80 kg, 1000 mg for >80 kg).

According to the manufacturers’ instructions, HCV RNA was quantified in all patients at baseline and after 4, 12, 24 and 48 weeks of treatment and 24 weeks after cessation of treatment by Cobas Amplicor HCV Monitor Test, v2.0 (Roche, Basel, Switzerland). In this study, rapid virological response (RVR) was defined as undetectable HCVRNA at 4 weeks during therapy. Early virological response (EVR) was defined as ≥2 log reduction in HCV RNA level compared to baseline HCV RNA level or undetectable HCVRNA at 12 weeks during treatment. SVR was defined as HCV RNA negative 24 weeks after treatment-free follow-up.

### SNP genotyping

DNA extraction was with protease K digestion and phenol-chloroform purification, which was described previously[25](#b25). Genotyping was performed by the TaqMan allelic discrimination assay on ABI PRISM 7900HT Sequence Detection system (Applied Biosystems, San Diego, CA, USA). The information on primers and probes are shown in [Supporting Table 1](#S1). Two blank controls and 5 repeated samples were assigned in each genotyping assay, and a 100% concordant was achieved. The *IL28B* rs12979860 allele C and *IFNL4* ss469415590 alleles TT were defined as wild-type alleles, whereas the T and ΔG as mutant alleles respectively.

### Statistical analysis

Differences in the general demographic characteristics were calculated by the Student *t* test and the chi-square (χ^2^) test. The associations of SNPs with HCV spontaneous clearance, RVR, EVR and SVR were estimated by the odds ratios (ORs) and 95% confidence intervals (CIs) using multivariate logistic regression analysis. Age, gender and viral genotype were adjusted for HCV spontaneous clearance, while, age, gender, baseline HCV RNA and platelet were adjusted for RVR, EVR and SVR during regression analysis. A stepwise forward procedure was used for selecting the final logistic regression model for predicting outcomes of HCV treatment. The trend analysis was assessed with Cochran-Armitage trend test.

Random forest, an advanced tree classifier, was used to evaluate the importance of each variable and to improve the performance of classification. The Gini or information entropy was the standard to classify the partner node into two child nodes. In our study, the area under the curve (AUC) and the plot of variables importance indicated the result of this method[26](#b26).

All the statistical analyses were carried out by STATA 12.0 software (StataCorp LP, College Station, TX, USA), and *P* < 0.05 in a two-sided test was considered as statistical significance.

## Additional Information

**How to cite this article**: Huang, P. *et al*. Genetic variants in interferon-λ 4 influences HCV clearance in Chinese Han population. *Sci. Rep.* **7**, 42408; doi: 10.1038/srep42408 (2017).

**Publisher's note:** Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.

## Supplementary Material

## Acknowledgments

We thank the students who have worked on the study, including Yuangyuan Zhang, Hui Zheng, Wenzhe Ma, Wenjuan Zhang, Xuecheng Song. We also thank doctors and the nurses from the Jurong People’s Hospital for helping sample collection and organization of the field investigation. This research would not have been possible without the consent and help of the participants. The current study was supported in part by National Natural Science Foundation of China (No.81473029 and 81473028), Priority Academic Program Development of Jiangsu Higher Education Institutions (PAPD), Medical Reform Project of Health and Family Planning Commission of Jiangsu Province of China (YG201413), Science and Technology Program of Zhenjiang, China (SH20141).

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Alter M. J. et al. Recommendations for prevention and control of hepatitis C virus (HCV) infection and HCV-related chronic disease. MMWR Morb Mortal Wkly Rep 47 (1998).  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=MMWR%20Morb%20Mortal%20Wkly%20Rep&title=Recommendations%20for%20prevention%20and%20control%20of%20hepatitis%20C%20virus%20(HCV)%20infection%20and%20HCV-related%20chronic%20disease&author=M.%20J.%20Alter&volume=47&publication_year=1998&)

2. Lavanchy D. Evolving epidemiology of hepatitis C virus. Clinical Microbiology and Infection 17, 107–115 (2011).  [DOI](https://doi.org/10.1111/j.1469-0691.2010.03432.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21091831/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clinical%20Microbiology%20and%20Infection&title=Evolving%20epidemiology%20of%20hepatitis%20C%20virus&author=D.%20Lavanchy&volume=17&publication_year=2011&pages=107-115&pmid=21091831&doi=10.1111/j.1469-0691.2010.03432.x&)

3. Seeff L. B. Natural history of chronic hepatitis C. Hepatology 36, S35–S46 (2002).  [DOI](https://doi.org/10.1053/jhep.2002.36806) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12407575/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hepatology&title=Natural%20history%20of%20chronic%20hepatitis%20C&author=L.%20B.%20Seeff&volume=36&publication_year=2002&pages=S35-S46&pmid=12407575&doi=10.1053/jhep.2002.36806&)

4. Zeuzem S. et al. Expert opinion on the treatment of patients with chronic hepatitis C. Journal of viral hepatitis 16, 75–90 (2009).  [DOI](https://doi.org/10.1111/j.1365-2893.2008.01012.x) | [PMC free article](/articles/PMC2759987/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18761607/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20of%20viral%20hepatitis&title=Expert%20opinion%20on%20the%20treatment%20of%20patients%20with%20chronic%20hepatitis%20C&author=S.%20Zeuzem&volume=16&publication_year=2009&pages=75-90&pmid=18761607&doi=10.1111/j.1365-2893.2008.01012.x&)

5. Huang C.-F. et al. Interleukin-28B genetic variants in identification of hepatitis C virus genotype 1 patients responding to 24 weeks peginterferon/ribavirin. Journal of hepatology 56, 34–40 (2012).  [DOI](https://doi.org/10.1016/j.jhep.2011.03.029) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21703176/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20of%20hepatology&title=Interleukin-28B%20genetic%20variants%20in%20identification%20of%20hepatitis%20C%20virus%20genotype%201%20patients%20responding%20to%2024%20weeks%20peginterferon/ribavirin&author=C.-F.%20Huang&volume=56&publication_year=2012&pages=34-40&pmid=21703176&doi=10.1016/j.jhep.2011.03.029&)

6. Tanaka Y. et al. Genome-wide association of IL28B with response to pegylated interferon-α and ribavirin therapy for chronic hepatitis C. Nature genetics 41, 1105–1109 (2009).  [DOI](https://doi.org/10.1038/ng.449) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19749757/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature%20genetics&title=Genome-wide%20association%20of%20IL28B%20with%20response%20to%20pegylated%20interferon-%CE%B1%20and%20ribavirin%20therapy%20for%20chronic%20hepatitis%20C&author=Y.%20Tanaka&volume=41&publication_year=2009&pages=1105-1109&pmid=19749757&doi=10.1038/ng.449&)

7. Prokunina-Olsson L. et al. A variant upstream of IFNL3 (IL28B) creating a new interferon gene IFNL4 is associated with impaired clearance of hepatitis C virus. Nature genetics 45, 164–171 (2013).  [DOI](https://doi.org/10.1038/ng.2521) | [PMC free article](/articles/PMC3793390/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23291588/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature%20genetics&title=A%20variant%20upstream%20of%20IFNL3%20(IL28B)%20creating%20a%20new%20interferon%20gene%20IFNL4%20is%20associated%20with%20impaired%20clearance%20of%20hepatitis%20C%20virus&author=L.%20Prokunina-Olsson&volume=45&publication_year=2013&pages=164-171&pmid=23291588&doi=10.1038/ng.2521&)

8. Bibert S. et al. IL28B expression depends on a novel TT/-G polymorphism which improves HCV clearance prediction. The Journal of experimental medicine 210, 1109–1116 (2013).  [DOI](https://doi.org/10.1084/jem.20130012) | [PMC free article](/articles/PMC3674704/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23712427/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=The%20Journal%20of%20experimental%20medicine&title=IL28B%20expression%20depends%20on%20a%20novel%20TT/-G%20polymorphism%20which%20improves%20HCV%20clearance%20prediction&author=S.%20Bibert&volume=210&publication_year=2013&pages=1109-1116&pmid=23712427&doi=10.1084/jem.20130012&)

9. Booth D. & George J. Loss of function of the new interferon IFN-[lambda] 4 may confer protection from hepatitis C. Nature genetics 45, 119–120 (2013).  [DOI](https://doi.org/10.1038/ng.2537) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23358218/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature%20genetics&title=Loss%20of%20function%20of%20the%20new%20interferon%20IFN-%5Blambda%5D%204%20may%20confer%20protection%20from%20hepatitis%20C&author=D.%20Booth&author=J.%20George&volume=45&publication_year=2013&pages=119-120&pmid=23358218&doi=10.1038/ng.2537&)

10. Urban T., Charlton M. R. & Goldstein D. B. Introduction to the genetics and biology of interleukin‐28B. Hepatology 56, 361–366 (2012).  [DOI](https://doi.org/10.1002/hep.25794) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22511448/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hepatology&title=Introduction%20to%20the%20genetics%20and%20biology%20of%20interleukin%E2%80%9028B&author=T.%20Urban&author=M.%20R.%20Charlton&author=D.%20B.%20Goldstein&volume=56&publication_year=2012&pages=361-366&pmid=22511448&doi=10.1002/hep.25794&)

11. Nozawa Y. et al. Genetic polymorphism in IFNL4 and response to pegylated interferon‐α and ribavirin in Japanese chronic hepatitis C patients. Tissue antigens 83, 45–48 (2014).  [DOI](https://doi.org/10.1111/tan.12264) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24355007/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Tissue%20antigens&title=Genetic%20polymorphism%20in%20IFNL4%20and%20response%20to%20pegylated%20interferon%E2%80%90%CE%B1%20and%20ribavirin%20in%20Japanese%20chronic%20hepatitis%20C%20patients&author=Y.%20Nozawa&volume=83&publication_year=2014&pages=45-48&pmid=24355007&doi=10.1111/tan.12264&)

12. Covolo L. et al. The novel ss469415590 variant predicts virological response to therapy in patients with chronic hepatitis C virus type 1 infection. Alimentary pharmacology & therapeutics 39, 322–330 (2014).  [DOI](https://doi.org/10.1111/apt.12568) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24308755/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Alimentary%20pharmacology%20&%20therapeutics&title=The%20novel%20ss469415590%20variant%20predicts%20virological%20response%20to%20therapy%20in%20patients%20with%20chronic%20hepatitis%20C%20virus%20type%201%20infection&author=L.%20Covolo&volume=39&publication_year=2014&pages=322-330&pmid=24308755&doi=10.1111/apt.12568&)

13. Knapp S. et al. Influence of IFNL3. rs12979860 and IFNL4. ss469415590 polymorphism on clearance of hepatitis C virus infection among Egyptians. Hepatology international 9, 251–257 (2015).  [DOI](https://doi.org/10.1007/s12072-015-9619-z) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25788203/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hepatology%20international&title=Influence%20of%20IFNL3.%20rs12979860%20and%20IFNL4.%20ss469415590%20polymorphism%20on%20clearance%20of%20hepatitis%20C%20virus%20infection%20among%20Egyptians&author=S.%20Knapp&volume=9&publication_year=2015&pages=251-257&pmid=25788203&doi=10.1007/s12072-015-9619-z&)

14. Stättermayer A. et al. Polymorphisms of interferon‐λ4 and IL28B–effects on treatment response to interferon/ribavirin in patients with chronic hepatitis C. Alimentary pharmacology & therapeutics 39, 104–111 (2014).  [DOI](https://doi.org/10.1111/apt.12547) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24205831/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Alimentary%20pharmacology%20&%20therapeutics&title=Polymorphisms%20of%20interferon%E2%80%90%CE%BB4%20and%20IL28B%E2%80%93effects%20on%20treatment%20response%20to%20interferon/ribavirin%20in%20patients%20with%20chronic%20hepatitis%20C&author=A.%20St%C3%A4ttermayer&volume=39&publication_year=2014&pages=104-111&pmid=24205831&doi=10.1111/apt.12547&)

15. Palmieri O. et al. Variation in genes encoding for interferon λ‐3 and λ‐4 in the prediction of HCV‐1 treatment‐induced viral clearance. Liver International 34, 1369–1377 (2014).  [DOI](https://doi.org/10.1111/liv.12411) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25283962/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Liver%20International&title=Variation%20in%20genes%20encoding%20for%20interferon%20%CE%BB%E2%80%903%20and%20%CE%BB%E2%80%904%20in%20the%20prediction%20of%20HCV%E2%80%901%20treatment%E2%80%90induced%20viral%20clearance&author=O.%20Palmieri&volume=34&publication_year=2014&pages=1369-1377&pmid=25283962&doi=10.1111/liv.12411&)

16. Nagaoki Y. et al. Interferon lambda 4 polymorphism affects on outcome of telaprevir, pegylated interferon and ribavirin combination therapy for chronic hepatitis C. Hepatology Research 44, E447–E454 (2014).  [DOI](https://doi.org/10.1111/hepr.12336) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24690180/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hepatology%20Research&title=Interferon%20lambda%204%20polymorphism%20affects%20on%20outcome%20of%20telaprevir,%20pegylated%20interferon%20and%20ribavirin%20combination%20therapy%20for%20chronic%20hepatitis%20C&author=Y.%20Nagaoki&volume=44&publication_year=2014&pages=E447-E454&pmid=24690180&doi=10.1111/hepr.12336&)

17. Miyamura T. et al. IFNL4 ss469415590 Variant Is Associated with Treatment Response in Japanese HCV Genotype 1 Infected Individuals Treated with IFN-Including Regimens. International journal of hepatology 2014 (2014).  [DOI](https://doi.org/10.1155/2014/723868) | [PMC free article](/articles/PMC4274707/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25548683/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=International%20journal%20of%20hepatology&title=IFNL4%20ss469415590%20Variant%20Is%20Associated%20with%20Treatment%20Response%20in%20Japanese%20HCV%20Genotype%201%20Infected%20Individuals%20Treated%20with%20IFN-Including%20Regimens&author=T.%20Miyamura&volume=2014&publication_year=2014&pmid=25548683&doi=10.1155/2014/723868&)

18. Real L. M. et al. IFNL4 ss469415590 variant shows similar performance to rs12979860 as predictor of response to treatment against hepatitis C virus genotype 1 or 4 in Caucasians. PloS one 9, e95515 (2014).  [DOI](https://doi.org/10.1371/journal.pone.0095515) | [PMC free article](/articles/PMC3991683/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24748394/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PloS%20one&title=IFNL4%20ss469415590%20variant%20shows%20similar%20performance%20to%20rs12979860%20as%20predictor%20of%20response%20to%20treatment%20against%20hepatitis%20C%20virus%20genotype%201%20or%204%20in%20Caucasians&author=L.%20M.%20Real&volume=9&publication_year=2014&pages=e95515&pmid=24748394&doi=10.1371/journal.pone.0095515&)

19. Honda M. et al. Hepatic ISG expression is associated with genetic variation in interleukin 28B and the outcome of IFN therapy for chronic hepatitis C. Gastroenterology 139, 499–509 (2010).  [DOI](https://doi.org/10.1053/j.gastro.2010.04.049) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20434452/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gastroenterology&title=Hepatic%20ISG%20expression%20is%20associated%20with%20genetic%20variation%20in%20interleukin%2028B%20and%20the%20outcome%20of%20IFN%20therapy%20for%20chronic%20hepatitis%20C&author=M.%20Honda&volume=139&publication_year=2010&pages=499-509&pmid=20434452&doi=10.1053/j.gastro.2010.04.049&)

20. McGilvray I. et al. Hepatic Cell–Type Specific Gene Expression Better Predicts HCV Treatment Outcome Than IL28B Genotype. Gastroenterology 142, 1122–1131. e1121 (2012).  [DOI](https://doi.org/10.1053/j.gastro.2012.01.028) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22285807/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gastroenterology&title=Hepatic%20Cell%E2%80%93Type%20Specific%20Gene%20Expression%20Better%20Predicts%20HCV%20Treatment%20Outcome%20Than%20IL28B%20Genotype&author=I.%20McGilvray&volume=142&publication_year=2012&pages=1122-1131.%20e1121&pmid=22285807&doi=10.1053/j.gastro.2012.01.028&)

21. Sulkowski M. S. et al. Daclatasvir plus sofosbuvir for previously treated or untreated chronic HCV infection. New England Journal of Medicine 370, 211–221 (2014).  [DOI](https://doi.org/10.1056/NEJMoa1306218) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24428467/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=New%20England%20Journal%20of%20Medicine&title=Daclatasvir%20plus%20sofosbuvir%20for%20previously%20treated%20or%20untreated%20chronic%20HCV%20infection&author=M.%20S.%20Sulkowski&volume=370&publication_year=2014&pages=211-221&pmid=24428467&doi=10.1056/NEJMoa1306218&)

22. Lange C. M. & Zeuzem S. Perspectives and challenges of interferon-free therapy for chronic hepatitis C. Journal of hepatology 58, 583–592 (2013).  [DOI](https://doi.org/10.1016/j.jhep.2012.10.019) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23104162/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20of%20hepatology&title=Perspectives%20and%20challenges%20of%20interferon-free%20therapy%20for%20chronic%20hepatitis%20C&author=C.%20M.%20Lange&author=S.%20Zeuzem&volume=58&publication_year=2013&pages=583-592&pmid=23104162&doi=10.1016/j.jhep.2012.10.019&)

23. Huang P. et al. Association of Polymorphisms in HLA Antigen Presentation-Related Genes with the Outcomes of HCV Infection. PloS one 10 (2015).  [DOI](https://doi.org/10.1371/journal.pone.0123513) | [PMC free article](/articles/PMC4395248/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25874709/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PloS%20one&title=Association%20of%20Polymorphisms%20in%20HLA%20Antigen%20Presentation-Related%20Genes%20with%20the%20Outcomes%20of%20HCV%20Infection&author=P.%20Huang&volume=10&publication_year=2015&pmid=25874709&doi=10.1371/journal.pone.0123513&)

24. Huang P. et al. Genetic variants in antigen presentation-related genes influence susceptibility to hepatitis C virus and viral clearance: a case control study. BMC infectious diseases 14, 3837 (2014).  [DOI](https://doi.org/10.1186/s12879-014-0716-8) | [PMC free article](/articles/PMC4279674/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25528575/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20infectious%20diseases&title=Genetic%20variants%20in%20antigen%20presentation-related%20genes%20influence%20susceptibility%20to%20hepatitis%20C%20virus%20and%20viral%20clearance:%20a%20case%20control%20study&author=P.%20Huang&volume=14&publication_year=2014&pages=3837&pmid=25528575&doi=10.1186/s12879-014-0716-8&)

25. Cui Q. et al. The association between the genetic polymorphisms of LMP2/LMP7 and the outcomes of HCV infection among drug users. Journal of biomedical research 24, 374–380 (2010).  [DOI](https://doi.org/10.1016/S1674-8301(10)60050-4) | [PMC free article](/articles/PMC3596683/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23554652/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20of%20biomedical%20research&title=The%20association%20between%20the%20genetic%20polymorphisms%20of%20LMP2/LMP7%20and%20the%20outcomes%20of%20HCV%20infection%20among%20drug%20users&author=Q.%20Cui&volume=24&publication_year=2010&pages=374-380&pmid=23554652&doi=10.1016/S1674-8301(10)60050-4&)

26. Breiman L. Statistical modeling: The two cultures (with comments and a rejoinder by the author). Statistical Science 16, 199–231 (2001).  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Statistical%20Science&title=Statistical%20modeling:%20The%20two%20cultures%20(with%20comments%20and%20a%20rejoinder%20by%20the%20author)&author=L.%20Breiman&volume=16&publication_year=2001&pages=199-231&)
