# Polymorphic variation in TPMT is the principal determinant of TPMT phenotype: a meta-analysis of three genome-wide association studies

## Metadata
**Authors:** Riin Tamm, Reedik Mägi, Roman Tremmel, Stefan Winter, Evelin Mihailov, Alenka Smid, Anja Möricke, Kathrin Klein, Martin Schrappe, Martin Stanulla, Richard Houlston, Richard Weinshilboum, Irena Mlinarič Raščan, Andres Metspalu, Lili Milani, Matthias Schwab, Elke Schaeffeler
**Journal:** Clinical pharmacology and therapeutics
**Date:** 2017 Feb 1
**DOI:** [10.1002/cpt.540](https://doi.org/10.1002/cpt.540)
**PMID:** 27770449
**PMCID:** PMC5395320
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5395320/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC5395320/pdf/nihms826163.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC5395320/pdf/nihms826163.pdf)

## Abstract

Thiopurine-related hematotoxicity in paediatric acute lymphoblastic leukemia (ALL) and inflammatory bowel diseases has been linked to genetically defined variability in thiopurine S-methyltransferase (TPMT) activity. While gene testing of TPMT is being clinically implemented, it is unclear if additional genetic variation influences TPMT activity with consequences for thiopurine-related toxicity. To examine this possibility, we performed a genome-wide association study (GWAS) of red blood cell TPMT activity in 844 Estonian individuals and 245 paediatric ALL-cases. Additionally, we correlated genome-wide genotypes to human hepatic TPMT activity in 123 samples. Only genetic variants mapping to chromosome 6, including TPMT gene region, were significantly associated with TPMT activity (P<5.0×10−8) in each of the three GWAS and a joint meta-analysis of 1212 cases (top hit P=1.2×10−72). This finding is consistent with TPMT genotype being the primary determinant of TPMT activity, reinforcing the rationale for genetic testing of TPMT alleles in routine clinical practice to individualize MP dosage.

Keywords: Pharmacogenetics, Personalized Medicine, Phase II, Polymorphism, Liver

## Introduction

Mercaptopurine (MP) is important for maintenance therapy of childhood acute lymphoblastic leukemia (ALL).^1^ The S-methylation of MP is catalyzed by the cytosolic enzyme thiopurine S-methyltransferase (TPMT). Over 30 polymorphisms in *TPMT* have been documented which have an effect on the enzymatic activity of TPMT.^2–4^ The most common alleles seen in most ethnicities are *TPMT*3C* (719A>G) and *TPMT*3A* (460G>A and 719A>G). There is a body of evidence that functional polymorphisms in *TPMT* are a determinant of MP-related hematotoxicity,^5^ with myelosuppression seen in patients with TPMT deficiency receiving standard MP therapy.^6,7^ Furthermore, intermediate TPMT activity, only partly explained by TPMT heterozygosity present in 10% of individuals,^8^ is also associated with an increased risk of toxicity.^5,6^ Recently, a prospective clinical trial confirmed a lower incidence of leucopenia in pre-treatment *TPMT* heterozygous-tested patients with inflammatory bowel disease (IBD) after dose-adjusted thiopurine therapy.^9^

While *TPMT* genotyping is advocated clinically by international guidelines using dosage individualization to limit MP-related toxicity,^5,10^ TPMT activity shows considerable variability even in those with low risk *TPMT* genotypes, presumably as a consequence of as yet unidentified additional genetic or non-genetic factors.^11^ Although several studies have been performed trying to identify additional factors apart from the *TPMT* gene that influence TPMT activity,^12–14^ thus far the genome-wide association study (GWAS) approach has not been used to comprehensively investigate the relationship between constitutional genotype and TPMT activity in red blood cells (RBC). Moreover, systematic data regarding the correlation between *TPMT* genotype and TPMT expression or function in human liver, as the predominant site of thiopurine metabolism, is still missing.

To address this deficiency we have conducted the so far largest meta-analysis of three independent genome-wide association studies (GWAS) of TPMT activity comprising 844 individuals of the Estonian population cohort and 245 pediatric ALL cases, as well as 123 human livers.

## Results

### Genome-wide association study in the Estonian population cohort

We first investigated the relationship between genome-wide genotypes and TPMT activity in individuals from the Estonian cohort. TPMT activity measured in RBC of 844 individuals (414 males, 430 females; age range 18–87 years) of the Estonian cohort showed a bimodal distribution and none of the cohort were shown to be TPMT deficient ([Figure 1A](#F1)). Genome-wide SNP genotyping was performed using microarrays. After quality control, European ancestry of all individuals was confirmed using 1000 Genomes as reference, as outlined in detail in the [supplementary methods (Figure S1)](#SD1). Further genetic variants, not covered by the microarray, were imputed using 1000 Genomes as reference (details see methods and [supplementary methods](#SD1)).

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/403e/5395320/2ab9ff1c1ea2/nihms826163f1.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5395320_nihms826163f1.jpg)

A: Histogram of TPMT activity in the Estonian cohort (n=844 individuals included in the GWAS), indicating a bimodal distribution; none of the individuals were shown to be TPMT deficient. B: Results of genome-wide association analysis of TPMT activity in the Estonian cohort, using age and sex as covariates. The Manhattan Plot shows the association P-values (−log10-transformed) of genetic variants across chromosomes 1–22. Significantly associated genetic variants (P<5.0×10−8) are marked in green. C: The regional association plot highlights the genomic region containing TPMT. Recombination rates and linkage disequilibrium estimates (r2) of variants with the TPMT variant 719A>G (rs1142345) are displayed. D: Association of TPMT activity with common non-functional TPMT alleles (TPMT*3A–*3C). E: Manhattan Plot showing association P-values (− log10-transformed) from conditional association analyses considering the most common TPMT alleles (TPMT*3A and TPMT*3C) and correcting for age and sex.

Finally, 8,617,769 genotyped and imputed genetic markers were included in the subsequent genome-wide association analysis, using sex and age as covariates. As shown in the Manhattan plot ([Figure 1B](#F1)) and in the regional association plot ([Figure 1C](#F1)), the most significant association, by several orders of magnitude, was provided by genetic variants mapping to *TPMT* gene region at 6p22.3 in our genome-wide association analysis. In total, 169 genetic variants on chromosome 6 were significant at the genome-wide threshold (*P*<5.0×10^−8^, [Table S1](#SD1)) with a minimal *P*-value of *P*=2.73×10^−76^. In addition, the most important *TPMT* alleles (*TPMT*2,*3A–*3C*) were genotyped separately using real-time PCR (TaqMan) technology. [Figure 1D](#F1) shows the association of *TPMT*3* alleles with TPMT activity, indicating that TPMT activity was significantly lower in heterozygous individuals (median 21, range 13–33 nmol 6-MTG × g^−1^ Hb × h^−1^) compared to wild-type carriers (median 40, range 18–64 nmol 6-MTG × g^−1^ Hb × h^−1^). Further GWAS analyses conditioning on the most frequent non-functional *TPMT* alleles (*TPMT*3A* and **3C*) were performed to identify further genetic factors apart from TPMT. However, no significantly associated variants (*P*<5.0×10^−8^) with TPMT activity in our cohort were found ([Figure 1E](#F1)). In addition, we investigated the contribution of SNP-SNP interactions, but could not identify any further significantly contributing variants ([Figure S2](#SD1)) after correction for multiple testing. Moreover, previously identified candidate SNPs were only negligibly related to TPMT activity in the Estonian cohort based on our GWAS data ([Table 1](#T1)).

### Table 1.

|   |   |   |   |   | Estoniancohort |  | ALLstudy |  | Livercohort |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| gene | rs number | chromo-some | non-effectallele | effectallele | effectsize | P-value | effectsize | P-value | effectsize | P-value |
| PACSIN2§ | rs2413739 | 22 | C | T | 0.06 | 0.23 | −0.07 | 0.48 | −0.02 | 0.91 |
| XDH# | rs494852 | 2 | C | T | 0.07 | 0.36 | −0.13 | 0.26 | 0.004 | 0.98 |
| IMPDH1# | rs4731448 | 7 | A | G | 0.02 | 0.73 | −0.01 | 0.90 | −0.03 | 0.83 |
| SLC28A3# | rs17428030 | 9 | A | G | −0.11 | 0.26 | 0.04 | 0.77 | −0.30 | 0.26 |
| ABCC4# | rs17268122 | 13 | G | T | 0.03 | 0.62 | 0.02 | 0.84 | 0.15 | 0.36 |
| FAM8A6P# | rs1040637 | 6 | A | G | 0.02 | 0.72 | −0.10 | 0.31 | −0.40 | 0.002 |
| HIVEP2,AIG1# | rs200148 | 6 | G | A | 0.05 | 0.27 | −0.13 | 0.12 | 0.13 | 0.34 |
| NUDT15* | rs554405994 | 13 | − | GGAGTC | −0.34 | 0.44 | - | - | - | - |
| NUDT15* | rs186364861 | 13 | G | A | - | - | - | - | - | - |
| NUDT15* | rs116855232 | 13 | C | T | 0.18 | 0.61 | −0.70 | 0.20 | −0.14 | 0.87 |
| NUDT15* | rs147390019 | 13 | G | A | - | - | −0.55 | 0.45 | - | - |

Table 1 Caption: Results for previously identified candidate SNPs associated with TPMT activity and/or thiopurine-related toxicity

### Genome-wide association study in ALL patients

Next, we investigated whether the same relationship between genotypes and TPMT activity was shown in pediatric ALL. The ALL study cohort consists of 245 children of European ancestry ([Figure S1](#SD1)) of the ALL BFM trials,^15^ none of whom had been transfused within 3 months prior to blood sampling and samples were obtained before ALL maintenance therapy. The distribution of TPMT activity is depicted in [Figure 2A](#F2). Two of the cases were previously shown to be TPMT deficient.^7,16^ Next, a genome-wide association study including sex and age as covariates was performed comprising 8,224,478 genotyped and imputed markers. As shown in the Manhattan plot and also in the regional association plot displayed in [Figure 2B–2C](#F2), only genetic variants within the *TPMT* gene region were significantly associated with TPMT activity. All other gene regions were only negligibly associated with TPMT activity. A list of all SNPs that were genome-wide significant (*P*<5.0×10^−8^) is given in [Table S2](#SD1). The common variant 719A>G (rs1142345), which is part of the non-functional *TPMT*3A* and *TPMT*3C* alleles, displayed one of the lowest *P*-values ([Figure 2C](#F2)). [Figure 2D](#F2) shows the association of *TPMT*3A–C* alleles with TPMT activity. In addition, we previously genotyped the most important non-functional *TPMT* alleles using either TaqMan technology or MALDI-TOF MS and thereby identified one deficient patient as carrier of the *TPMT*3A/*11* genotype.^7^ TPMT activity in homozygous carriers of the *TPMT*3A* allele or in compound heterozygous carriers of two non-functional *TPMT* alleles (*TPMT*3A/*11*) displayed an enzyme activity ≤2 nmol 6-MTG × g^−1^ Hb × h^−1^. TPMT wild-type carriers showed a significant higher TPMT activity (median 30, range 9–69 6-MTG × g^−1^ Hb × h^−1^) compared to heterozygous carriers of non-functional TPMT alleles (median 21, range 9–33 6-MTG × g^−1^ Hb × h^−1^). Additional analyses conditioning on the *TPMT*3A* and **3C* alleles revealed no significantly associated variants (*P*<5.0x10^−8^) with TPMT activity in the ALL cohort ([Figure 2E](#F2)). Further, investigation of SNP-SNP interactions suggests lack of interaction effects between the investigated variants ([Figure S2](#SD1)) after correction for multiple testing. In line with these observations, previously identified candidate SNPs were not associated with TPMT activity in the ALL cohort based on our GWAS data ([Table 1](#T1)).

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/403e/5395320/0510dc41c81f/nihms826163f2.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5395320_nihms826163f2.jpg)

A: Histogram of TPMT activity in the pediatric ALL-cohort (n=245), indicating two patients with TPMT deficiency. B: Results of genome-wide association analysis of TPMT activity in the ALL study, using age and sex as covariates. The Manhattan Plot shows the association P-values (−log10-transformed) of genetic variants across chromosomes 1–22. Significantly associated genetic variants (P<5.0×10−8) are marked in green. C: The regional association plot highlights the genomic region containing TPMT. Recombination rates and linkage disequilibrium estimates (r2) of variants with the TPMT variant 719A>G (rs1142345) are displayed. D: Association of TPMT activity with common non-functional TPMT alleles (TPMT*2, TPMT*3A–*3C). E: Manhattan Plot showing association P-values (− log10-transformed) from conditional association analyses considering the most common TPMT alleles (TPMT*3A and TPMT*3C) and correcting for age and sex.

### Genome-wide association study in human liver

Following on from these observations we sought to identify genetic markers influencing TPMT activity in human liver, the most important thiopurine metabolizing tissue. TPMT activity, determined in liver cytosol of 124 individuals showed a bimodal distribution ([Figure 3A](#F3)) and none of the samples was TPMT deficient. Subsequently, we performed a genome-wide association analysis corrected for age and sex including 7,481,872 genotyped and imputed markers. All individuals included in the GWAS were from European ancestry ([Figure S1](#SD1)). As shown in the Manhattan plot and also in the regional association plot displayed in [Figure 3B–3C](#F3), only genetic variants within the *TPMT* region were significantly associated with TPMT activity (*P*<5.0×10^−8^) ([Table S3](#SD1)).

### Figure 3.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/403e/5395320/05066f84ad67/nihms826163f3.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5395320_nihms826163f3.jpg)

A: Histogram of TPMT activity in human liver (n=124), indicating a bimodal distribution; none of the individuals were shown to be TPMT deficient. B: Results of genome-wide association analysis of TPMT activity in human liver, using age and sex as covariates. The Manhattan Plot shows the association P-values (−log10-transformed) across chromosomes 1–22. Significant genetic variants (P<5.0×10−8) are marked in green. C: The regional association plot displays these values focusing on the genomic region on chromosome 6 (hg19), in which TPMT is located. Additionally, it displays recombination rate and linkage disequilibrium estimates (r2) of variants with the TPMT variant 719A>G (rs1142345). D: Association of TPMT activity with common non-functional TPMT alleles (TPMT*2, TPMT*3A–*3C). E: Manhattan Plot showing association P-values (− log10-transformed) from conditional association analyses considering the most common TPMT alleles (TPMT*3A and TPMT*3C) and correcting for age and sex.

The *TPMT* 238G>C, 460G>A, and 719A>G polymorphisms were additionally genotyped by either TaqMan or MALDI-TOF MS technology.^17,18^ Carriers of *TPMT*2* and *TPMT*3A* alleles showed a reduced hepatic TPMT activity ([Figure 3D](#F3)) (median 1.2, range 1.1–1.9 nmol h^−1^mg^−1^) compared to TPMT wild-type patients (median 2.7, range 1.5–4.1 nmol h^−1^ mg^−1^). Moreover, by genotyping 22 functionally relevant *TPMT* alleles using an established and validated MALDI-TOF MS method,^18^ as well as next generation sequencing (NGS) of the *TPMT* coding region, the presence of other functionally relevant TPMT variant alleles could be ruled out. Altogether, results from NGS were in agreement with imputed genotypes ([Table S4](#SD1)).

In addition, analyses conditioning on the most frequent non-functional *TPMT* alleles (*TPMT*3A* and **3C*) revealed no significantly associated variants (*P*<5.0×10^−8^) with TPMT activity in human liver ([Figure 3E](#F3)). Subsequent investigation of SNP-SNP interactions suggests lack of interaction effects between the investigated variants ([Figure S2](#SD1)) after correction for multiple testing. In addition, previously identified candidate SNPs were not associated with hepatic TPMT activity based on our GWAS data ([Table 1](#T1)).

Systematic data on TPMT expression and function in human liver are still missing and therefore we next correlated cytosolic TPMT activity with TPMT protein levels, determined by immunoblotting. As shown in [Figure 4A](#F4), a highly significant correlation was observed (r_s_=0.58, *P*<2.2×10^−16^) between protein expression and enzyme activity. In contrast, TPMT mRNA expression was not significantly correlated with either TPMT activity or TPMT protein level ([Figure4B and 4C](#F4)) in human liver, even after exclusion of *TPMT* variant cases.

### Figure 4.

![Figure 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/403e/5395320/6eda2f9a0044/nihms826163f4.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5395320_nihms826163f4.jpg)

A: Correlation of hepatic TPMT activity and protein levels (rS: Spearman’s correlation coefficient), including representative immunoblot (n=122; TPMT protein could not be reliably quantified through immunoblotting in two samples). B: Correlation of TPMT mRNA and protein expression in human liver. C: Correlation of mRNA and TPMT activity in human liver.

### Joint analyses of GWAS in the Estonian population cohort, ALL patients and human liver

Collectively our findings in three independent cohorts indicate that TPMT activity is primarily determined by genetic variation in the *TPMT* gene region in RBCs as well as in liver tissue. In total, 173 variants with *P*<5.0×10^−8^ could be identified in at least one cohort. Of these, 89% were located on chromosome 6 in the genomic region 18.1–18.2Mb, which includes the *TPMT* gene ([Figure 5A](#F5)). The remaining 11%, which were also located on chromosome 6 (but outside the *TPMT* region), were only significantly associated with TPMT activity in the Estonian cohort ([Figure 5A](#F5)). Noticeably, no significantly associated variants were found on other chromosomes besides chromosome 6.

### Figure 5.

![Figure 5](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/403e/5395320/9b7fa9564c39/nihms826163f5.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5395320_nihms826163f5.jpg)

A: Venn diagram depicting number of genetic variants significantly associated with TPMT activity (P<5.0×10−8) in the three cohorts. All variants located on chromosome 6 in the chromosomal region 18.1–18.2Mb are shown in blue, whereas those outside the region are shown in orange. No loci on other chromosomes were significantly associated with TPMT activity. B: Manhattan Plot showing association p-values (−log10-transformed) of the joint meta-GWAS of all three cohorts, revealing significant results within the TPMT locus only. Sentinel marker was chromosome 6:18103028 (rs73726531; MAF=0.03, P=1.2×10−72; effect size: −2.2), which is linked to TPMT variant 719A>G (rs1142345) (see regional association plot). Altogether 148 markers on chromosome 6 exceeded the genome-wide significance threshold (P<5.0×10−8) in the meta-GWAS (Table S5) and are displayed in the regional association plot.

In addition to the separate analyses of our cohorts, we performed a joint meta-analysis of all three datasets to account on the one hand for potential differences in our cohorts, and on the other hand to increase power and reliability of GWAS analysis. Again, this joint meta-analysis of all three datasets only revealed significantly associated genetic variants on chromosome 6, including the *TPMT* gene region, with the sentinel marker (rs73726531, *P*=1.2×10^−72^, effect size: −2.2; [Figure 5B](#F5); [Table S5](#SD1)). As shown in the regional association plot, the sentinel marker is tightly linked to the *TPMT* variant 719A>G (rs1142345) ([Figure 5B](#F5)).

## Discussion

Based on several clinical studies and various disease entities (e.g. ALL in childhood, IBD, autoimmune diseases)^19,20^ pretreatment determination of the TPMT phenotype and subsequent pharmacogenetically-guided dosing of thiopurines, at least in TPMT deficient individuals, is recommended in routine clinical practice before commencing therapy.^5^ Very recently a landmark randomized clinical trial strongly corroborates the clinical utility of upfront genetic testing for *TPMT* in patients treated with thiopurines to avoid hematotoxicity.^9^ IBD patients who were identified as variant carriers of *TPMT* and subsequently received dose adjusted thiopurine therapy, showed a 10-fold reduction in hematologic side effects.

Nevertheless there are still concerns with regard to whether *TPMT* genetics exclusively determines the marked interindividual variability of TPMT activity measured in RBC or whether other genes may contribute substantially, particularly in patients carrying the wild-type *TPMT* genotype. Clinically it is well-known that patients with *TPMT* wild-type are still at risk for thiopurine-related hematotoxicity and even after consideration of non-genetic factors (e.g. concomitant medication with the XO inhibitor allopurinol or viral infection by cytomegalovirus/parvovirus B19) underlying mechanisms are so far unknown to completely explain thiopurine-related hematotoxicity. Of note, in the retrospective study by Colombel et al.^21^ only 29.3% of 41 patients with IBD and azathioprine-related severe myelosuppression were carriers of *TPMT* variant alleles.

Some studies have already been performed trying to identify additional genetic or non-genetic factors apart from the *TPMT* gene that influence TPMT activity *in vivo*. For example, S-adenosyl-methionine (SAM), the global methyl donor in the human body, has been discovered as a modulator of TPMT activity.^11^ One previous study using HapMap CEU cell lines and two pediatric ALL-patient cohorts indicate that independent from *TPMT* a second gene, *PACSIN2*, which is a member of the ‘protein kinase C and casein kinase substrate in neurons’ family of proteins, also modulates TPMT activity and is associated with the MP-related gastrointestinal toxicity in children with ALL.^14^ Additionally, a GWAS was performed and identified genes in the thiopurine pathway that were associated with thiopurine metabolism in lymphoblastoid cell lines from individuals of different ethnic background and clinical response in pediatric ALL patients.^13^ Altogether, conclusive information is still missing which demonstrates that *TPMT* genetics is the ideal predictor for TPMT phenotype and replication of novel identified non-*TPMT* candidate variants have not been performed in independent large-scale studies.

The present data from our meta-analysis of three independent GWAS clearly indicate that *TPMT* genetics seems to have the strongest impact on TPMT activity in humans and provide little support for the proposal that other genes may significantly contribute to interindividual variability of TPMT activity.

Considering a genome-wide threshold of *P*<5.0×10^−8^, our first GWAS for TPMT activity adjusted for age and sex in 844 volunteer-based samples of the Estonia resident adult population revealed 169 variants on chromosome 6, and 150 of those variants map to the genomic region 18.1–18.2 Mb including the *TPMT* gene. Since this population cohort comprises subjects with various ICD-10 diagnoses^22^ not associated with anemia, we can conclude that different diseases do not appear to significantly alter TPMT activity in the RBC. Data from the second GWAS comprising 245 pediatric ALL children are in accordance to our findings from the Estonian cohort with 77 significantly associated variants mapping to chromosome 6 (18.1–18.2 Mb). Of note, we observed a TPMT genotype-phenotype discordance in ALL patients compared with the Estonia cohort attributed to the disease process of ALL and anemia of patients resulting in lower TPMT activities in RBCs due to degraded TPMT enzyme.^23,24^

Additionally, we investigated for the first time TPMT activity in cytosols from 124 human liver samples. Again the GWAS revealed that only variants on chromosome 6 (81 variants at region 18.1–18.2 Mb) were significantly associated with TPMT activity. Finally, data from our meta-analysis of all three datasets confirmed these findings, since once again all significantly associated genetic variants were located on chromosome 6, including the *TPMT* gene region. Noticeably, the genome-wide significant hits for TPMT activity extend to genes (e.g. *KDM1B*) located next to the *TPMT* gene. As shown in the regional association plot of the meta-GWAS, these variants are linked to the non-functional *TPMT*3* alleles. Further analyses conditioning on the *TPMT*3A* and **3C* alleles were performed, revealing no significantly associated variants (*P*<5.0×10^−8^) with TPMT activity in all three cohorts. Moreover, analyses of SNP-SNP interactions in all three cohorts revealed no significant variants after correction for multiple testing, suggesting a lack of strong interaction effects among the loci tested.

Next we tried to replicate non-*TPMT* related candidate genes previously identified by a genome-wide study approach using HapMapCEU lymphoblastoid cell linies (LCL).^13,14^ Of note, LCLs but even non-HapMap LCLs have been promised as useful model systems for cellular pharmacologic effects as well as biochemical effects and enzymatic reactions.^25^ Several studies have been reported using LCL resources including various whole-genome approaches (e.g. GWAS) to identify novel genetic variants which subsequently were associated with anticancer-related cytotoxicity (e.g. cisplatin,^26^ cytarabine^27^). Matimba et al.^13^ have proposed a minor association (*P*>5.0×10^−4^) between SNPs within “thiopurine-related” genes (*XDH, IMPDH1, SLC28A3, ABCC4*) and “non-thiopurine pathway” SNPs (*FAM8A6P, AJG1/HIVEP2*) with thiopurine cytotoxicity in LCLs with further validation in pediatric ALL patients. However, our findings provide little support for any effect mediated through TPMT, as there was only negligible correlation related to TPMT activity in our cohorts ([Table 1](#T1)).

Moreover, SNPs localizing to *PACSIN2*, previously found to influence TPMT activity and be related to MP-related gastrointestinal toxicity,^14^ were not associated with TPMT activity in our analyses. This finding supports Roberts et al.^28^ who failed to confirm an association between *PACSIN2* genotype and thiopurine-related adverse drug reactions in IBD patients.

Furthermore, very recently the p.Arg139Cys (R139C) variant in the *NUDT15* gene was associated with thiopurine-induced leukopenia firstly in a retrospective cohort of Korean IBD patients^29^ and confirmed by other studies including also children with ALL and MP intolerance.^29–31^ Very recently, Moriyama et al.^30^ provided mechanistically evidence how *NUDT15* variants independent from *TPMT* alter levels of active thiopurine metabolites subsequently resulting in increased thiopurine cytotoxicity. No potential interaction of the *NUDT15* R139C variant (rs116855232) with TPMT activity was found in all of our three cohorts ([Table 1](#T1)) keeping in mind that the allele frequency is extremely low in non-Asians.

Additionally, a genome-wide approach was used to identify novel predictors of TPMT activity in LCLs resulting in 96 genes ranked higher than *TPMT* itself.^12,30^ Except of *SLC22A16*, none of these genes were located on chromosome 6. Since based on our findings only genetic variants on chromosome 6 determine TPMT activity, these genes appear to be of minor importance to predict TPMT phenotype. Noticeably, no significantly associated variants in *SLC22A16* (P<5.0×10^−8^) with TPMT activity were found. Thus, use of LCLs in this investigation and the discovery of novel pharmacogenetic loci may have limitations, and validation of results in large-scale population/patient cohorts is needed.

In summary, for the first time we provide systematic data on TPMT expression (mRNA, protein) and function (activity levels) in a cohort of 124 human liver samples. Human liver is the predominant site of thiopurine metabolism and data are still lacking demonstrating a close correlation between hepatic TPMT protein expression and cytosolic enzyme activity. TPMT mRNA was not significantly correlated with either TPMT activity or TPMT protein level which supports previous findings demonstrating that the common *TPMT* polymorphisms 460G>A and 719A>G affect TPMT activity by post-translational modification and increased protein degradation.^19^ Moreover, no correlation was found between mRNA expression and either TPMT activity or TPMT protein expression measured by Western blotting in subjects carrying the TPMT reference sequence confirmed by next generation sequencing. These data are in contrast to Lindqvist et al.^32^ reporting a significant correlation between normal/high TPMT enzyme activity in RBC and mRNA levels extracted from whole blood in 29 individuals. It remains questionable if the correlation of mRNA derived from white blood cells and TPMT enzyme activity measured in RBC is reasonable.

Limitations of our work may be that this study was designed based on SNP array data and did not investigate gene duplications/deletions or genomic rearrangements which might alter TPMT activity. In addition, to identify novel rare variants in genes other than *TPMT* that are associated with TPMT activity whole genome next-generation sequencing approaches are needed. Moreover, epigenetic regulation of TPMT expression, e.g. by non-coding RNAs such as miRNAs, was not investigated which may be in part a plausible explanation of the missing correlation between TPMT mRNA expression and protein levels in human liver. Finally, our three study cohorts ethnically are restricted to individuals of European ancestry, as confirmed through genetic analyses ([Figure S1](#SD1)), and therefore other ethnic groups like Asians and Africans are not covered. In addition, the present study was not designed to identify genetic variants associated with thiopurine related toxicity. Thus, we can not exclude that in addition to *TPMT* further genes are involved in risk of toxicity or treatment outcome.

In conclusion, based on the largest meta-GWAS including 1212 subjects from different populations, we did not identify any novel associations for TPMT activity, thereby endorsing genetic testing for *TPMT* alleles as advocated by the Clinical Pharmacogenetics Implementation Consortium^5^ and the Dutch pharmacogenetics group^10^ for prediction of patient`s phenotype prior to thiopurine therapy in patients with ALL.

## Methods

### Estonian Population Cohort

Individuals (n=864; 422 males, age: median 34 years, range 18 to 87 years) were randomly chosen from a large cohort (52,000) of collected subjects from the Estonian Biobank,^22^ at the Estonian Genome Center, University of Tartu. The recruitment and sample collection has been described previously,^22^ and all participants have signed a broad informed consent. One inclusion criterion for the TPMT study was that there is no regular use of medications associated with TPMT. The study was approved by the Ethics Review Committee on Human Research, University of Tartu, Estonia. The DNA was extracted from whole blood using the conventional salting out procedure.^33^ Hemolysates were prepared from red blood cells (RBCs) according to established procedures,^34,35^ as described in the [supplementary methods](#SD1).

### ALL-study cohort

ALL-children (n=245; 144 males; age: median 5.6 years, range 1.1 to 17.4 years) of the Berlin-Frankfurt-Münster (BFM) trials were included in the present study,^15^ none of which had been transfused within 3 months prior to blood sampling and samples were obtained before ALL maintenance therapy. Sample collection was performed with informed consent in accordance with the principles of the Declaration of Helsinki and the study was approved by the ethical review board. Genomic DNA, as well as RBC lysates were prepared as previously described.^7,36^

### IKP liver cohort

Histologically normal human liver tissues as well as corresponding blood samples were collected from patients undergoing liver surgery at the Department of General, Visceral and Transplantation Surgery (University Medical Center Charité, Berlin, Germany) as previously described.^37^ The study was approved by the ethics committees of the Charité, Humboldt University (Berlin, Germany) and the University of Tübingen (Tübingen, Germany) in accordance with the principles of the Declaration of Helsinki. The tissue samples were stored at −80°C. Subcellular fractions were prepared according to standard procedures. Briefly, approximately 1 g of tissue was homogenized in 1 mM EDTA, 1 mM DTT, 10 mM HEPES pH7.4, 0.2 mM Pefabloc (Roth, Karlsruhe, Germany) and 0.15 mM KCl and differentially centrifuged at 15 000 g and 105 000 g. The final cytosolic supernatant was immediately frozen in liquid nitrogen in aliquots and stored at −80°C until use. Cytosol used for TPMT activity measurements was available for liver samples of 124 patients (n=124; 54 males, 70 females; age: median 58 years, range 7 to 85 years). Purification of genomic DNA from EDTA blood samples was performed as described previously,^37^ and DNA samples were available for 150 liver samples.^37^

### TPMT activity measurements from hemolyzed red blood cells and liver cytosol

TPMT activity was measured in hemolysates from all samples of the Estonian cohort, and the ALL-study cohort, and in all liver cytosols with a non-radioactive HPLC method as described previously^8^ using 6-TG as a substrate. The cut-off for TPMT deficiency based on TPMT activity measurements in liver cytosol was suggested to be 0 nmol h^−1^ mg^−1^, and the cut-off for TPMT activity in RBCs is ≤2 nmol 6-MTG × g^−1^ Hb × h^−1^.^8^

### Genotyping of TPMT alleles

Genotyping of *TPMT*2* (238G>C, rs1800462), *TPMT*3B* (460G>A, rs1800460) and *TPMT*3C* (719A>G, rs1142345) in the Estonian cohort was carried out by TaqMan Genotyping Assays (Applied Biosystems, Foster City, CA) as previously described. Genotyping for *TPMT*2* and *TPMT*3* alleles in the ALL- and IKP-liver cohort were performed using TaqMan technology and MALDI-TOF MS as previously described.^17,18^

### Whole-genome genotyping and imputation analysis for individuals from the Estonian cohort, the ALL study and the liver cohort

For the 864 samples of the Estonian cohort, the Illumina Human370CNV BeadChips were used for whole-genome genotyping. After quality control, recovery of untyped genotypes using 1000Genomes as reference, and filtering, 8,617,769 markers and 844 individuals were included in the subsequent analysis. Genome-wide genotyping data for the ALL-study was generated by Illumina Human Omni1-Quad arrays. After quality control, imputation and filtering, 8,224,478 markers were included in the subsequent association analyses of 245 patients. Genome-wide genotyping data for liver samples were generated using HumanHap300 Genotyping BeadChip.^38^ After quality control, imputation, and filtering, 7,481,872 markers were included in the subsequent association analyses of 123 samples. Comprehensive details of the methods are provided in [supplementary methods](#SD1).

### Next-Generation sequencing

In addition, the *TPMT* gene region was analyzed systematically in human liver samples for presence of genetic variations by next-generation sequencing (NGS). NGS was conducted at the Center for Genomics and Transcriptomics (CeGaT GmbH, Tübingen, Germany) as previously described.^39^ Details are provided in the [supplementary methods](#SD1).

### TPMT mRNA quantification

High-quality RNA of 150 liver samples was extracted as described previously.^37^ TPMT mRNA was quantified with the TaqMan^®^ Gene Expression Assay Hs00909011_m1 (Applied Biosystems, Foster City, USA). TPMT expression was normalized against *β-actin* expression, which was measured with the HUMAN ACTB (beta actin) Endogenous Control Assay (Applied Biosystems, Foster City, USA). The measurements were conducted on the Fast Real Time PCR System 7900HT (Applied Biosystems, Foster City, USA).

### TPMT protein quantification

TPMT protein expression was quantified by immunoblot analyses of liver cytosol using a specific rabbit anti-TPMT antibody, which was kindly provided by Richard Weinshilboum (Mayo Clinic, Rochester, USA). TPMT protein levels, quantified through immunoblotting, were available for 122 samples.

### Statistical Analysis

#### Analyses of genome-wide association in individual studies

Association study of genome-wide imputed genotypes was performed with SNPTEST v2.5^40^ using the frequentist association test option with expected genotype counts. For this purpose, TPMT activity measurements were first power transformed (Estonian cohort: λ=1.3, ALL study: λ=0.5, liver cohort: λ=1.65) to fulfill Gaussian distribution assumption. Here, the transformation was determined via the MASS_7.3–33 package^41^ within R-3.1.1 ([www.r-project.org](http://www.r-project.org/)). Distribution of transformed measurements was confirmed using a normal quantile-quantile plot and the Shapiro-Wilk test. Finally, the association between each of the genetic variants and the transformed TPMT activity measurements, corrected for sex and age, was analyzed in the additive genetic model. Subsequently, we additionally conditioned on additive dosages of the respective TPMT variants (*TPMT*3A* and *TPMT*3C*) to test for secondary signals. Details about the analyses of SNP-SNP interactions are provided in the [supplementary data](#SD1).

#### Meta-analyses

Results of all three GWAS datasets were combined into a joint meta-analysis using GWAMA software.^42^ Results for the Estonian study and the ALL study, where the TPMT activity was measured in blood were combined in an inverse of variance weighted fixed effect meta-analysis and the results for the IKP liver cohort with activity measurements in liver cytosol were added in a two degrees of freedom test allowing for the differential effect sizes between the two sample types (blood and liver).^43^

#### Visualization of the genome-wide association data

The genome-wide significance level was defined as 5.0×10^−8^. Manhattan plots were drawn using the R-package qqman_0.1.1 and MANH.R script from GWAMA software package.^42^ Regional association plots showing −log_10_ *P* values of genetic variants relative to the *TPMT* gene region were generated using LocusZoom.^44^ Here, LD (r^2^) was calculated from the European population in 1000 Genomes (hg19, Nov 2014).

### Further statistical analyses

Spearman’s correlation test was used to test association between TPMT activity and mRNA or protein expression. R-package beanplot_1.2^45^ was applied to display the association between TPMT activity and common *TPMT* SNPs, with thick black lines representing the medians per group and the dotted line the overall median. All statistical tests were two-sided and statistical significance was defined as *P*<5%.

## Supplementary Material

### What is the current knowledge on the topic?

### What is the current knowledge on the topic?

### What is the current knowledge on the topic?

### What is the current knowledge on the topic?

Clinically it is well-known that patients with TPMT wild-type are still at risk for thiopurine-related hematotoxicity even after consideration of non-genetic factors. Prediction of TPMT phenotype is currently exclusively based on measurement of TPMT activity and/or genotyping.

### What question did this study address?

The largest meta-analysis of GWAS in a population cohort, pediatric ALL patients and human liver samples was designed to assess whether *TPMT* genetics exclusively determines the marked interindividual variability of TPMT activity or if other genes contribute substantially.

### What this study adds to our knowledge?

Only genetic variants mapping to the *TPMT* gene region at chromosome 6 are the primary determinant of TPMT activity in human. Notably this could be demonstrated for human liver as the predominant site of thiopurine metabolism.

### How this might change clinical pharmacology or translational science?

Genetic testing of *TPMT* is sufficient to predict patient`s correct phenotype prior to thiopurine therapy promoting the clinical implementation process of pre-emptive testing. Future studies are warranted to identify so far unidentified TPMT-independent factors to explain thiopurine-related hematotoxicity.

## Acknowledgments

We gratefully acknowledge Britta Klumpp, Monika Elbl, Monika Seiler, Andrea Jarmuth, and Viljo Soo for excellent technical assistance. We would like to thank Dr Krista Fischer for assistance with statistical analyses. We thank Ulrich M. Zanger for kindly providing liver tissue for TPMT analyses. We acknowledge the support of CeGaT GmbH, Tübingen, Germany, for next-generation sequencing.

This work was supported by the Estonian Research Council (Grant IUT20-60), Estonian Science Foundation (ETF9293), the Robert Bosch Foundation (Stuttgart, Germany), the Federal Ministry for Education and Research (BMBF, Berlin, Germany, project “Virtual Liver” 0315755, LiSyM 031L0037), the Horizon 2020-PHC-2015 grant U-PGx 668353, the German Cancer Consortium (DKTK) and German Cancer Research Center (DKFZ, Heidelberg, Germany), the ICEPHA Graduate School Tuebingen-Stuttgart (Germany), and the United States National Institutes of Health (NIH) grants RO1 GM28157 and U19 GM61388.

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Hunger SP, Mullighan CG. Acute Lymphoblastic Leukemia in Children. N Engl J Med. 2015;373(16):1541–1552. doi: 10.1056/NEJMra1400972.  [DOI](https://doi.org/10.1056/NEJMra1400972) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26465987/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Acute%20Lymphoblastic%20Leukemia%20in%20Children&author=SP%20Hunger&author=CG%20Mullighan&volume=373&issue=16&publication_year=2015&pages=1541-1552&pmid=26465987&doi=10.1056/NEJMra1400972&)

2. Feng Q, et al. Thiopurine S-methyltransferase pharmacogenetics: functional characterization of a novel rapidly degraded variant allozyme. Biochem Pharmacol. 2010;79(7):1053–1061. doi: 10.1016/j.bcp.2009.11.016.  [DOI](https://doi.org/10.1016/j.bcp.2009.11.016) | [PMC free article](/articles/PMC2815203/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19945438/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem%20Pharmacol&title=Thiopurine%20S-methyltransferase%20pharmacogenetics:%20functional%20characterization%20of%20a%20novel%20rapidly%20degraded%20variant%20allozyme&author=Q%20Feng&volume=79&issue=7&publication_year=2010&pages=1053-1061&pmid=19945438&doi=10.1016/j.bcp.2009.11.016&)

3. Salavaggione OE, Wang L, Wiepert M, Yee VC, Weinshilboum RM. Thiopurine S-methyltransferase pharmacogenetics: variant allele functional and comparative genomics. Pharmacogenet Genomics. 2005;15(11):801–815. doi: 10.1097/01.fpc.0000174788.69991.6b.  [DOI](https://doi.org/10.1097/01.fpc.0000174788.69991.6b) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16220112/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Thiopurine%20S-methyltransferase%20pharmacogenetics:%20variant%20allele%20functional%20and%20comparative%20genomics&author=OE%20Salavaggione&author=L%20Wang&author=M%20Wiepert&author=VC%20Yee&author=RM%20Weinshilboum&volume=15&issue=11&publication_year=2005&pages=801-815&pmid=16220112&doi=10.1097/01.fpc.0000174788.69991.6b&)

4. Appell ML, et al. Nomenclature for alleles of the thiopurine methyltransferase gene. Pharmacogenet Genomics. 2013;23(4):242–248. doi: 10.1097/FPC.0b013e32835f1cc0.  [DOI](https://doi.org/10.1097/FPC.0b013e32835f1cc0) | [PMC free article](/articles/PMC3727893/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23407052/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Nomenclature%20for%20alleles%20of%20the%20thiopurine%20methyltransferase%20gene&author=ML%20Appell&volume=23&issue=4&publication_year=2013&pages=242-248&pmid=23407052&doi=10.1097/FPC.0b013e32835f1cc0&)

5. Relling MV, et al. Clinical pharmacogenetics implementation consortium guidelines for thiopurine methyltransferase genotype and thiopurine dosing: 2013 update. Clin Pharmacol Ther. 2013;93(4):324–325. doi: 10.1038/clpt.2013.4.  [DOI](https://doi.org/10.1038/clpt.2013.4) | [PMC free article](/articles/PMC3604643/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23422873/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Clinical%20pharmacogenetics%20implementation%20consortium%20guidelines%20for%20thiopurine%20methyltransferase%20genotype%20and%20thiopurine%20dosing:%202013%20update&author=MV%20Relling&volume=93&issue=4&publication_year=2013&pages=324-325&pmid=23422873&doi=10.1038/clpt.2013.4&)

6. Relling MV, et al. Mercaptopurine therapy intolerance and heterozygosity at the thiopurine S-methyltransferase gene locus. J Natl Cancer Inst. 1999;91(23):2001–2008. doi: 10.1093/jnci/91.23.2001.  [DOI](https://doi.org/10.1093/jnci/91.23.2001) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10580024/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Natl%20Cancer%20Inst&title=Mercaptopurine%20therapy%20intolerance%20and%20heterozygosity%20at%20the%20thiopurine%20S-methyltransferase%20gene%20locus&author=MV%20Relling&volume=91&issue=23&publication_year=1999&pages=2001-2008&pmid=10580024&doi=10.1093/jnci/91.23.2001&)

7. Schaeffeler E, et al. A novel TPMT missense mutation associated with TPMT deficiency in a 5-year-old boy with ALL. Leukemia. 2003;17(7):1422–1424. doi: 10.1038/sj.leu.2402981.  [DOI](https://doi.org/10.1038/sj.leu.2402981) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12835738/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Leukemia&title=A%20novel%20TPMT%20missense%20mutation%20associated%20with%20TPMT%20deficiency%20in%20a%205-year-old%20boy%20with%20ALL&author=E%20Schaeffeler&volume=17&issue=7&publication_year=2003&pages=1422-1424&pmid=12835738&doi=10.1038/sj.leu.2402981&)

8. Schaeffeler E, et al. Comprehensive analysis of thiopurine S-methyltransferase phenotype-genotype correlation in a large population of German-Caucasians and identification of novel TPMT variants. Pharmacogenetics. 2004;14(7):407–417. doi: 10.1097/01.fpc.0000114745.08559.db.  [DOI](https://doi.org/10.1097/01.fpc.0000114745.08559.db) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15226673/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=Comprehensive%20analysis%20of%20thiopurine%20S-methyltransferase%20phenotype-genotype%20correlation%20in%20a%20large%20population%20of%20German-Caucasians%20and%20identification%20of%20novel%20TPMT%20variants&author=E%20Schaeffeler&volume=14&issue=7&publication_year=2004&pages=407-417&pmid=15226673&doi=10.1097/01.fpc.0000114745.08559.db&)

9. Coenen MJ, et al. Identification of Patients With Variants in TPMT and Dose Reduction Reduces Hematologic Events During Thiopurine Treatment of Inflammatory Bowel Disease. Gastroenterology. 2015;149(4):907–917. doi: 10.1053/j.gastro.2015.06.002.  [DOI](https://doi.org/10.1053/j.gastro.2015.06.002) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26072396/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gastroenterology&title=Identification%20of%20Patients%20With%20Variants%20in%20TPMT%20and%20Dose%20Reduction%20Reduces%20Hematologic%20Events%20During%20Thiopurine%20Treatment%20of%20Inflammatory%20Bowel%20Disease&author=MJ%20Coenen&volume=149&issue=4&publication_year=2015&pages=907-917&pmid=26072396&doi=10.1053/j.gastro.2015.06.002&)

10. Swen JJ, et al. Pharmacogenetics: from bench to byte--an update of guidelines. Clin Pharmacol Ther. 2011;89(5):662–673. doi: 10.1038/clpt.2011.34.  [DOI](https://doi.org/10.1038/clpt.2011.34) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21412232/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Pharmacogenetics:%20from%20bench%20to%20byte--an%20update%20of%20guidelines&author=JJ%20Swen&volume=89&issue=5&publication_year=2011&pages=662-673&pmid=21412232&doi=10.1038/clpt.2011.34&)

11. Karas-Kuzelicki N, Smid A, Tamm R, Metspalu A, Mlinaric-Rascan I. From pharmacogenetics to pharmacometabolomics: SAM modulates TPMT activity. Pharmacogenomics. 2014;15(11):1437–1449. doi: 10.2217/pgs.14.84.  [DOI](https://doi.org/10.2217/pgs.14.84) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25303295/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=From%20pharmacogenetics%20to%20pharmacometabolomics:%20SAM%20modulates%20TPMT%20activity&author=N%20Karas-Kuzelicki&author=A%20Smid&author=R%20Tamm&author=A%20Metspalu&author=I%20Mlinaric-Rascan&volume=15&issue=11&publication_year=2014&pages=1437-1449&pmid=25303295&doi=10.2217/pgs.14.84&)

12. Jones TS, Yang W, Evans WE, Relling MV. Using HapMap tools in pharmacogenomic discovery: the thiopurine methyltransferase polymorphism. Clin Pharmacol Ther. 2007;81(5):729–734. doi: 10.1038/sj.clpt.6100135.  [DOI](https://doi.org/10.1038/sj.clpt.6100135) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17329987/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Using%20HapMap%20tools%20in%20pharmacogenomic%20discovery:%20the%20thiopurine%20methyltransferase%20polymorphism&author=TS%20Jones&author=W%20Yang&author=WE%20Evans&author=MV%20Relling&volume=81&issue=5&publication_year=2007&pages=729-734&pmid=17329987&doi=10.1038/sj.clpt.6100135&)

13. Matimba A, et al. Thiopurine pharmacogenomics: association of SNPs with clinical response and functional validation of candidate genes. Pharmacogenomics. 2014;15(4):433–447. doi: 10.2217/pgs.13.226.  [DOI](https://doi.org/10.2217/pgs.13.226) | [PMC free article](/articles/PMC4027966/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24624911/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Thiopurine%20pharmacogenomics:%20association%20of%20SNPs%20with%20clinical%20response%20and%20functional%20validation%20of%20candidate%20genes&author=A%20Matimba&volume=15&issue=4&publication_year=2014&pages=433-447&pmid=24624911&doi=10.2217/pgs.13.226&)

14. Stocco G, et al. PACSIN2 polymorphism influences TPMT activity and mercaptopurine-related gastrointestinal toxicity. Hum Mol Genet. 2012;21(21):4793–4804. doi: 10.1093/hmg/dds302.  [DOI](https://doi.org/10.1093/hmg/dds302) | [PMC free article](/articles/PMC3471396/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22846425/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mol%20Genet&title=PACSIN2%20polymorphism%20influences%20TPMT%20activity%20and%20mercaptopurine-related%20gastrointestinal%20toxicity&author=G%20Stocco&volume=21&issue=21&publication_year=2012&pages=4793-4804&pmid=22846425&doi=10.1093/hmg/dds302&)

15. Migliorini G, et al. Variation at 10p12.2 and 10p14 influences risk of childhood B-cell acute lymphoblastic leukemia and phenotype. Blood. 2013;122(19):3298–3307. doi: 10.1182/blood-2013-03-491316.  [DOI](https://doi.org/10.1182/blood-2013-03-491316) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23996088/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Variation%20at%2010p12.2%20and%2010p14%20influences%20risk%20of%20childhood%20B-cell%20acute%20lymphoblastic%20leukemia%20and%20phenotype&author=G%20Migliorini&volume=122&issue=19&publication_year=2013&pages=3298-3307&pmid=23996088&doi=10.1182/blood-2013-03-491316&)

16. Stanulla M, et al. Thiopurine methyltransferase genetics is not a major risk factor for secondary malignant neoplasms after treatment of childhood acute lymphoblastic leukemia on Berlin-Frankfurt-Munster protocols. Blood. 2009;114(7):1314–1318. doi: 10.1182/blood-2008-12-193250.  [DOI](https://doi.org/10.1182/blood-2008-12-193250) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19535798/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Thiopurine%20methyltransferase%20genetics%20is%20not%20a%20major%20risk%20factor%20for%20secondary%20malignant%20neoplasms%20after%20treatment%20of%20childhood%20acute%20lymphoblastic%20leukemia%20on%20Berlin-Frankfurt-Munster%20protocols&author=M%20Stanulla&volume=114&issue=7&publication_year=2009&pages=1314-1318&pmid=19535798&doi=10.1182/blood-2008-12-193250&)

17. Dilger K, et al. Monitoring of thiopurine methyltransferase activity in postsurgical patients with Crohn's disease during 1 year of treatment with azathioprine or mesalazine. Ther Drug Monit. 2007;29(1):1–5. doi: 10.1097/FTD.0b013e3180312b9a.  [DOI](https://doi.org/10.1097/FTD.0b013e3180312b9a) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17304143/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ther%20Drug%20Monit&title=Monitoring%20of%20thiopurine%20methyltransferase%20activity%20in%20postsurgical%20patients%20with%20Crohn's%20disease%20during%201%20year%20of%20treatment%20with%20azathioprine%20or%20mesalazine&author=K%20Dilger&volume=29&issue=1&publication_year=2007&pages=1-5&pmid=17304143&doi=10.1097/FTD.0b013e3180312b9a&)

18. Schaeffeler E, Zanger UM, Eichelbaum M, Asante-Poku S, Shin JG, Schwab M. Highly multiplexed genotyping of thiopurine S-methyltransferase variants using MALDI-TOF mass spectrometry: reliable genotyping in different ethnic groups. Clin Chem. 2008;54(10):1637–1647. doi: 10.1373/clinchem.2008.103457.  [DOI](https://doi.org/10.1373/clinchem.2008.103457) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18687736/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Chem&title=Highly%20multiplexed%20genotyping%20of%20thiopurine%20S-methyltransferase%20variants%20using%20MALDI-TOF%20mass%20spectrometry:%20reliable%20genotyping%20in%20different%20ethnic%20groups&author=E%20Schaeffeler&author=UM%20Zanger&author=M%20Eichelbaum&author=S%20Asante-Poku&author=JG%20Shin&volume=54&issue=10&publication_year=2008&pages=1637-1647&pmid=18687736&doi=10.1373/clinchem.2008.103457&)

19. Moon W, Loftus EV., Jr Review article: recent advances in pharmacogenetics and pharmacokinetics for safe and effective thiopurine therapy in inflammatory bowel disease. Aliment Pharmacol Ther. 2016 doi: 10.1111/apt.13559.  [DOI](https://doi.org/10.1111/apt.13559) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26876431/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Aliment%20Pharmacol%20Ther&title=Review%20article:%20recent%20advances%20in%20pharmacogenetics%20and%20pharmacokinetics%20for%20safe%20and%20effective%20thiopurine%20therapy%20in%20inflammatory%20bowel%20disease&author=W%20Moon&author=EV%20Loftus&publication_year=2016&pmid=26876431&doi=10.1111/apt.13559&)

20. Teml A, Schaeffeler E, Herrlinger KR, Klotz U, Schwab M. Thiopurine treatment in inflammatory bowel disease: clinical pharmacology and implication of pharmacogenetically guided dosing. Clin Pharmacokinet. 2007;46(3):187–208. doi: 10.2165/00003088-200746030-00001.  [DOI](https://doi.org/10.2165/00003088-200746030-00001) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17328579/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacokinet&title=Thiopurine%20treatment%20in%20inflammatory%20bowel%20disease:%20clinical%20pharmacology%20and%20implication%20of%20pharmacogenetically%20guided%20dosing&author=A%20Teml&author=E%20Schaeffeler&author=KR%20Herrlinger&author=U%20Klotz&author=M%20Schwab&volume=46&issue=3&publication_year=2007&pages=187-208&pmid=17328579&doi=10.2165/00003088-200746030-00001&)

21. Colombel JF, et al. Genotypic analysis of thiopurine S-methyltransferase in patients with Crohn's disease and severe myelosuppression during azathioprine therapy. Gastroenterology. 2000;118(6):1025–1030. doi: 10.1016/s0016-5085(00)70354-4.  [DOI](https://doi.org/10.1016/s0016-5085(00)70354-4) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10833476/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gastroenterology&title=Genotypic%20analysis%20of%20thiopurine%20S-methyltransferase%20in%20patients%20with%20Crohn's%20disease%20and%20severe%20myelosuppression%20during%20azathioprine%20therapy&author=JF%20Colombel&volume=118&issue=6&publication_year=2000&pages=1025-1030&pmid=10833476&doi=10.1016/s0016-5085(00)70354-4&)

22. Leitsalu L, et al. Cohort Profile: Estonian Biobank of the Estonian Genome Center, University of Tartu. Int J Epidemiol. 2015;44(4):1137–1147. doi: 10.1093/ije/dyt268.  [DOI](https://doi.org/10.1093/ije/dyt268) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24518929/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Epidemiol&title=Cohort%20Profile:%20Estonian%20Biobank%20of%20the%20Estonian%20Genome%20Center,%20University%20of%20Tartu&author=L%20Leitsalu&volume=44&issue=4&publication_year=2015&pages=1137-1147&pmid=24518929&doi=10.1093/ije/dyt268&)

23. Lennard L, Chew TS, Lilleyman JS. Human thiopurine methyltransferase activity varies with red blood cell age. Br J Clin Pharmacol. 2001;52(5):539–546. doi: 10.1046/j.0306-5251.2001.01497.x.  [DOI](https://doi.org/10.1046/j.0306-5251.2001.01497.x) | [PMC free article](/articles/PMC2014613/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11736862/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Clin%20Pharmacol&title=Human%20thiopurine%20methyltransferase%20activity%20varies%20with%20red%20blood%20cell%20age&author=L%20Lennard&author=TS%20Chew&author=JS%20Lilleyman&volume=52&issue=5&publication_year=2001&pages=539-546&pmid=11736862&doi=10.1046/j.0306-5251.2001.01497.x&)

24. Lennard L, Cartwright CS, Wade R, Richards SM, Vora A. Thiopurine methyltransferase genotype-phenotype discordance and thiopurine active metabolite formation in childhood acute lymphoblastic leukaemia. Br J Clin Pharmacol. 2013;76(1):125–136. doi: 10.1111/bcp.12066.  [DOI](https://doi.org/10.1111/bcp.12066) | [PMC free article](/articles/PMC3703235/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23252716/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Clin%20Pharmacol&title=Thiopurine%20methyltransferase%20genotype-phenotype%20discordance%20and%20thiopurine%20active%20metabolite%20formation%20in%20childhood%20acute%20lymphoblastic%20leukaemia&author=L%20Lennard&author=CS%20Cartwright&author=R%20Wade&author=SM%20Richards&author=A%20Vora&volume=76&issue=1&publication_year=2013&pages=125-136&pmid=23252716&doi=10.1111/bcp.12066&)

25. Zhang W, Dolan ME. Use of cell lines in the investigation of pharmacogenetic loci. Curr Pharm Des. 2009;15(32):3782–3795. doi: 10.2174/138161209789649475.  [DOI](https://doi.org/10.2174/138161209789649475) | [PMC free article](/articles/PMC2782819/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19925429/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Pharm%20Des&title=Use%20of%20cell%20lines%20in%20the%20investigation%20of%20pharmacogenetic%20loci&author=W%20Zhang&author=ME%20Dolan&volume=15&issue=32&publication_year=2009&pages=3782-3795&pmid=19925429&doi=10.2174/138161209789649475&)

26. Huang RS, et al. Identification of genetic variants contributing to cisplatin-induced cytotoxicity by use of a genomewide approach. Am J Hum Genet. 2007;81(3):427–437. doi: 10.1086/519850.  [DOI](https://doi.org/10.1086/519850) | [PMC free article](/articles/PMC1950832/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17701890/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=Identification%20of%20genetic%20variants%20contributing%20to%20cisplatin-induced%20cytotoxicity%20by%20use%20of%20a%20genomewide%20approach&author=RS%20Huang&volume=81&issue=3&publication_year=2007&pages=427-437&pmid=17701890&doi=10.1086/519850&)

27. Hartford CM, et al. Population-specific genetic variants important in susceptibility to cytarabine arabinoside cytotoxicity. Blood. 2009;113(10):2145–2153. doi: 10.1182/blood-2008-05-154302.  [DOI](https://doi.org/10.1182/blood-2008-05-154302) | [PMC free article](/articles/PMC2652364/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19109566/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Population-specific%20genetic%20variants%20important%20in%20susceptibility%20to%20cytarabine%20arabinoside%20cytotoxicity&author=CM%20Hartford&volume=113&issue=10&publication_year=2009&pages=2145-2153&pmid=19109566&doi=10.1182/blood-2008-05-154302&)

28. Roberts RL, et al. PACSIN2 does not influence thiopurine-related toxicity in patients with inflammatory bowel disease. Am J Gastroenterol. 2014;109(6):925–927. doi: 10.1038/ajg.2014.89.  [DOI](https://doi.org/10.1038/ajg.2014.89) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24896764/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Gastroenterol&title=PACSIN2%20does%20not%20influence%20thiopurine-related%20toxicity%20in%20patients%20with%20inflammatory%20bowel%20disease&author=RL%20Roberts&volume=109&issue=6&publication_year=2014&pages=925-927&pmid=24896764&doi=10.1038/ajg.2014.89&)

29. Yang SK, et al. A common missense variant in NUDT15 confers susceptibility to thiopurine-induced leukopenia. Nat Genet. 2014;46(9):1017–1020. doi: 10.1038/ng.3060.  [DOI](https://doi.org/10.1038/ng.3060) | [PMC free article](/articles/PMC4999337/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25108385/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=A%20common%20missense%20variant%20in%20NUDT15%20confers%20susceptibility%20to%20thiopurine-induced%20leukopenia&author=SK%20Yang&volume=46&issue=9&publication_year=2014&pages=1017-1020&pmid=25108385&doi=10.1038/ng.3060&)

30. Moriyama T, et al. NUDT15 polymorphisms alter thiopurine metabolism and hematopoietic toxicity. Nat Genet. 2016;48(4):367–373. doi: 10.1038/ng.3508.  [DOI](https://doi.org/10.1038/ng.3508) | [PMC free article](/articles/PMC5029084/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26878724/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=NUDT15%20polymorphisms%20alter%20thiopurine%20metabolism%20and%20hematopoietic%20toxicity&author=T%20Moriyama&volume=48&issue=4&publication_year=2016&pages=367-373&pmid=26878724&doi=10.1038/ng.3508&)

31. Tanaka Y, et al. Susceptibility to 6-MP toxicity conferred by a NUDT15 variant in Japanese children with acute lymphoblastic leukaemia. Br J Haematol. 2015;171(1):109–115. doi: 10.1111/bjh.13518.  [DOI](https://doi.org/10.1111/bjh.13518) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26033531/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Haematol&title=Susceptibility%20to%206-MP%20toxicity%20conferred%20by%20a%20NUDT15%20variant%20in%20Japanese%20children%20with%20acute%20lymphoblastic%20leukaemia&author=Y%20Tanaka&volume=171&issue=1&publication_year=2015&pages=109-115&pmid=26033531&doi=10.1111/bjh.13518&)

32. Lindqvist M, Almer S, Peterson C, Soderkvist P. Real-time RT-PCR methodology for quantification of thiopurine methyltransferase gene expression. Eur J Clin Pharmacol. 2003;59(3):207–211. doi: 10.1007/s00228-003-0617-z.  [DOI](https://doi.org/10.1007/s00228-003-0617-z) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12819840/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Clin%20Pharmacol&title=Real-time%20RT-PCR%20methodology%20for%20quantification%20of%20thiopurine%20methyltransferase%20gene%20expression&author=M%20Lindqvist&author=S%20Almer&author=C%20Peterson&author=P%20Soderkvist&volume=59&issue=3&publication_year=2003&pages=207-211&pmid=12819840&doi=10.1007/s00228-003-0617-z&)

33. Miller SA, Dykes DD, Polesky HF. A simple salting out procedure for extracting DNA from human nucleated cells. Nucleic Acids Res. 1988;16(3):1215. doi: 10.1093/nar/16.3.1215.  [DOI](https://doi.org/10.1093/nar/16.3.1215) | [PMC free article](/articles/PMC334765/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/3344216/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=A%20simple%20salting%20out%20procedure%20for%20extracting%20DNA%20from%20human%20nucleated%20cells&author=SA%20Miller&author=DD%20Dykes&author=HF%20Polesky&volume=16&issue=3&publication_year=1988&pages=1215&pmid=3344216&doi=10.1093/nar/16.3.1215&)

34. Keizer-Garritsen JJ, et al. Measurement of thiopurine S-methyltransferase activity in human blood samples based on high-performance liquid chromatography: reference values in erythrocytes from children. Ann Clin Biochem. 2003;40(Pt 1):86–93. doi: 10.1258/000456303321016222.  [DOI](https://doi.org/10.1258/000456303321016222) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12542916/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Clin%20Biochem&title=Measurement%20of%20thiopurine%20S-methyltransferase%20activity%20in%20human%20blood%20samples%20based%20on%20high-performance%20liquid%20chromatography:%20reference%20values%20in%20erythrocytes%20from%20children&author=JJ%20Keizer-Garritsen&volume=40&issue=Pt%201&publication_year=2003&pages=86-93&pmid=12542916&doi=10.1258/000456303321016222&)

35. Milek M, Murn J, Jaksic Z, Lukac BJ, Jazbec J, Mlinaric RI. Thiopurine S-methyltransferase pharmacogenetics: genotype to phenotype correlation in the Slovenian population. Pharmacology. 2006;77(3):105–114. doi: 10.1159/000093278.  [DOI](https://doi.org/10.1159/000093278) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16691038/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacology&title=Thiopurine%20S-methyltransferase%20pharmacogenetics:%20genotype%20to%20phenotype%20correlation%20in%20the%20Slovenian%20population&author=M%20Milek&author=J%20Murn&author=Z%20Jaksic&author=BJ%20Lukac&author=J%20Jazbec&volume=77&issue=3&publication_year=2006&pages=105-114&pmid=16691038&doi=10.1159/000093278&)

36. Stanulla M, et al. Thiopurine methyltransferase (TPMT) genotype and early treatment response to mercaptopurine in childhood acute lymphoblastic leukemia. JAMA. 2005;293(12):1485–1489. doi: 10.1001/jama.293.12.1485.  [DOI](https://doi.org/10.1001/jama.293.12.1485) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15784872/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA&title=Thiopurine%20methyltransferase%20(TPMT)%20genotype%20and%20early%20treatment%20response%20to%20mercaptopurine%20in%20childhood%20acute%20lymphoblastic%20leukemia&author=M%20Stanulla&volume=293&issue=12&publication_year=2005&pages=1485-1489&pmid=15784872&doi=10.1001/jama.293.12.1485&)

37. Nies AT, et al. Expression of organic cation transporters OCT1 (SLC22A1) and OCT3 (SLC22A3) is affected by genetic factors and cholestasis in human liver. Hepatology. 2009;50(4):1227–1240. doi: 10.1002/hep.23103.  [DOI](https://doi.org/10.1002/hep.23103) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19591196/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hepatology&title=Expression%20of%20organic%20cation%20transporters%20OCT1%20(SLC22A1)%20and%20OCT3%20(SLC22A3)%20is%20affected%20by%20genetic%20factors%20and%20cholestasis%20in%20human%20liver&author=AT%20Nies&volume=50&issue=4&publication_year=2009&pages=1227-1240&pmid=19591196&doi=10.1002/hep.23103&)

38. Schroder A, et al. Genomics of ADME gene expression: mapping expression quantitative trait loci relevant for absorption, distribution, metabolism and excretion of drugs in human liver. Pharmacogenomics J. 2013;13(1):12–20. doi: 10.1038/tpj.2011.44.  [DOI](https://doi.org/10.1038/tpj.2011.44) | [PMC free article](/articles/PMC3564008/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22006096/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=Genomics%20of%20ADME%20gene%20expression:%20mapping%20expression%20quantitative%20trait%20loci%20relevant%20for%20absorption,%20distribution,%20metabolism%20and%20excretion%20of%20drugs%20in%20human%20liver&author=A%20Schroder&volume=13&issue=1&publication_year=2013&pages=12-20&pmid=22006096&doi=10.1038/tpj.2011.44&)

39. Emami RA, et al. Variability in hepatic expression of organic anion transporter 7/SLC22A9, a novel pravastatin uptake transporter: impact of genetic and regulatory factors. Pharmacogenomics J. 2015;16(4):341–351. doi: 10.1038/tpj.2015.55.  [DOI](https://doi.org/10.1038/tpj.2015.55) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26239079/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=Variability%20in%20hepatic%20expression%20of%20organic%20anion%20transporter%207/SLC22A9,%20a%20novel%20pravastatin%20uptake%20transporter:%20impact%20of%20genetic%20and%20regulatory%20factors&author=RA%20Emami&volume=16&issue=4&publication_year=2015&pages=341-351&pmid=26239079&doi=10.1038/tpj.2015.55&)

40. Marchini J, Howie B, Myers S, McVean G, Donnelly P. A new multipoint method for genome-wide association studies by imputation of genotypes. Nat Genet. 2007;39(7):906–913. doi: 10.1038/ng2088.  [DOI](https://doi.org/10.1038/ng2088) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17572673/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=A%20new%20multipoint%20method%20for%20genome-wide%20association%20studies%20by%20imputation%20of%20genotypes&author=J%20Marchini&author=B%20Howie&author=S%20Myers&author=G%20McVean&author=P%20Donnelly&volume=39&issue=7&publication_year=2007&pages=906-913&pmid=17572673&doi=10.1038/ng2088&)

41. Venables WN, Ripley BD. Modern Applied Statistics with S. 4. Springer-Verlag New York: Copyright Holder Springer Science+Business Media New York; 2002.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Modern%20Applied%20Statistics%20with%20S&author=WN%20Venables&author=BD%20Ripley&publication_year=2002&)

42. Magi R, Morris AP. GWAMA: software for genome-wide association meta-analysis. BMC Bioinformatics. 2010;11:288. doi: 10.1186/1471-2105-11-288.  [DOI](https://doi.org/10.1186/1471-2105-11-288) | [PMC free article](/articles/PMC2893603/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20509871/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Bioinformatics&title=GWAMA:%20software%20for%20genome-wide%20association%20meta-analysis&author=R%20Magi&author=AP%20Morris&volume=11&publication_year=2010&pages=288&pmid=20509871&doi=10.1186/1471-2105-11-288&)

43. Magi R, Lindgren CM, Morris AP. Meta-analysis of sex-specific genome-wide association studies. Genet Epidemiol. 2010;34(8):846–853. doi: 10.1002/gepi.20540.  [DOI](https://doi.org/10.1002/gepi.20540) | [PMC free article](/articles/PMC3410525/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21104887/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genet%20Epidemiol&title=Meta-analysis%20of%20sex-specific%20genome-wide%20association%20studies&author=R%20Magi&author=CM%20Lindgren&author=AP%20Morris&volume=34&issue=8&publication_year=2010&pages=846-853&pmid=21104887&doi=10.1002/gepi.20540&)

44. Pruim RJ, et al. LocusZoom: regional visualization of genome-wide association scan results. Bioinformatics. 2010;26(18):2336–2337. doi: 10.1093/bioinformatics/btq419.  [DOI](https://doi.org/10.1093/bioinformatics/btq419) | [PMC free article](/articles/PMC2935401/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20634204/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=LocusZoom:%20regional%20visualization%20of%20genome-wide%20association%20scan%20results&author=RJ%20Pruim&volume=26&issue=18&publication_year=2010&pages=2336-2337&pmid=20634204&doi=10.1093/bioinformatics/btq419&)

45. Kampstra P. Beanplot: A Boxplot Alternative for Visual Comparison of Distributions. Journal of Statistical Software. 2008;28(1):1–9.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Statistical%20Software&title=Beanplot:%20A%20Boxplot%20Alternative%20for%20Visual%20Comparison%20of%20Distributions&author=P%20Kampstra&volume=28&issue=1&publication_year=2008&pages=1-9&)
