# Variability of the drug response to nonsteroidal anti-inflammatory drugs according to cyclooxygenase-2 genetic polymorphism

## Metadata
**Authors:** Sook Joung Lee, Min Kyu Park, Dong-Seong Shin, Min Ho Chun
**Journal:** Drug Design, Development and Therapy
**Date:** 2017 Sep 13
**DOI:** [10.2147/DDDT.S143807](https://doi.org/10.2147/DDDT.S143807)
**PMID:** 29066864
**PMCID:** PMC5604555
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5604555/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC5604555/pdf/dddt-11-2727.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC5604555/pdf/dddt-11-2727.pdf)

## Abstract

**Purpose:** 
Cyclooxygenase (COX) is the main pharmacodynamic target of nonsteroidal anti-inflammatory drugs (NSAIDs). We investigated the inhibitory effects on COX-2 after NSAIDs administration using a lipopolysaccharide (LPS)-derived COX-2 induction model in whole blood, according to the genotypes of COX-2 single-nucleotide polymorphisms (SNPs).

**Patients and methods:** 
Seven genotypes of COX-2 SNPs were selected from public databases and analyzed in 324 healthy subjects. Two genotypes showing a high percentage of variability were selected. A clinical trial examining pharmacodynamics according to the genotype of two SNPs (rs5275 and rs689466) was conducted. Twenty subjects were administered a single oral dose of 200 mg of celecoxib, and pharmacokinetic and pharmacodynamic analyses were performed.

**Results:** 
In the analysis of the pharmacokinetic parameters, significant differences in drug exposure were not investigated for each SNP genotype. The pharmacodynamic analysis revealed that the maximum effect of COX-2 inhibition was achieved at 2.0 hours for all genotypes of COX-2 SNPs after a single oral administration of 200 mg celecoxib. The inhibitory effects according to the genotype of COX-2 SNPs were investigated, and the area under the effect curve of the rs689466 GG genotype was significantly lower than that for the AA or AG genotype.

**Conclusion:** 
Our results demonstrated that inhibitory effects of celecoxib on COX-2 induction were different according to the genotype of COX-2 SNPs. In the present study, rs689466 is responsible for the variability of the response to celecoxib, suggesting that a subject with the GG genotype of rs689466 would be more responsive to celecoxib in terms of COX-2 inhibition.

Keywords: nonsteroidal anti-inflammatory agent, cyclooxygenase-2, single-nucleotide polymorphism, drug response, pharmacodynamics, clinical trial

### Purpose

Cyclooxygenase (COX) is the main pharmacodynamic target of nonsteroidal anti-inflammatory drugs (NSAIDs). We investigated the inhibitory effects on COX-2 after NSAIDs administration using a lipopolysaccharide (LPS)-derived COX-2 induction model in whole blood, according to the genotypes of COX-2 single-nucleotide polymorphisms (SNPs).

### Patients and methods

Seven genotypes of COX-2 SNPs were selected from public databases and analyzed in 324 healthy subjects. Two genotypes showing a high percentage of variability were selected. A clinical trial examining pharmacodynamics according to the genotype of two SNPs (rs5275 and rs689466) was conducted. Twenty subjects were administered a single oral dose of 200 mg of celecoxib, and pharmacokinetic and pharmacodynamic analyses were performed.

### Results

In the analysis of the pharmacokinetic parameters, significant differences in drug exposure were not investigated for each SNP genotype. The pharmacodynamic analysis revealed that the maximum effect of COX-2 inhibition was achieved at 2.0 hours for all genotypes of COX-2 SNPs after a single oral administration of 200 mg celecoxib. The inhibitory effects according to the genotype of COX-2 SNPs were investigated, and the area under the effect curve of the rs689466 GG genotype was significantly lower than that for the AA or AG genotype.

### Conclusion

Our results demonstrated that inhibitory effects of celecoxib on COX-2 induction were different according to the genotype of COX-2 SNPs. In the present study, rs689466 is responsible for the variability of the response to celecoxib, suggesting that a subject with the GG genotype of rs689466 would be more responsive to celecoxib in terms of COX-2 inhibition.

## Introduction

Nonsteroidal anti-inflammatory drugs (NSAIDs) are the most widely prescribed pharmaceutical agents worldwide and have broad clinical utility in the management of pain, fever, inflammation and inflammatory disease. Cyclooxygenase (COX) is the main pharmacodynamic target of NSAIDs. COX, also known as prostaglandin H synthase, is a major enzymatic source of prostaglandin (PG) production from arachidonic acid and comprises two isoforms, COX-1 and COX-2. Although COX-1 and COX-2 exhibit a high degree of homologous genetic information, their structural stability, enzymatic mechanism and functions differ.[1](#b1-dddt-11-2727),[2](#b2-dddt-11-2727) COX-1 isoform is constitutively expressed in most tissues and cells throughout the body and produces PGs involved in regulating physiological homeostatic processes.[3](#b3-dddt-11-2727)–[5](#b5-dddt-11-2727) In contrast, COX-2 is an inducible isoform that is upregulated by various mitogenic and inflammatory stimuli, such as pro-inflammatory cytokines, growth factors and tumor promoters.[6](#b6-dddt-11-2727)

Although NSAIDs show various beneficial effects, such as antipyretic, analgesic and anti-inflammatory effects, response variability and unwanted adverse drug effects are also observed in patients using NSAIDs. Because the mechanisms of inflammation and pain are complex and vary between individuals, there is interindividual variability in the effectiveness and safety of NSAIDs.

Regarding the pharmacokinetic characteristics of NSAIDs, several studies have investigated the variability of adverse events and the effectiveness of NSAIDs according to the genetic polymorphism of hepatic cytochrome P450 isoenzymes, which are involved in drug metabolism. Indeed, CYP2C subfamilies exhibit many single-nucleotide polymorphisms (SNPs) with different frequencies, and CYP2C9 metabolizes many NSAIDs including ibuprofen, diclofenac, naproxen, piroxicam and tenoxicam.[7](#b7-dddt-11-2727),[8](#b8-dddt-11-2727) However, these previous studies have not investigated the relationship between NSAIDs and COX-2 inhibition. Because the effect of NSAIDs in individual patients is based on pharmaco-dynamic activity related to the induction and continued participation of COX-2 in the inflammatory process, the investigation of various inhibitory effects on COX-2 activity and its duration is necessary to explain the variability of NSAID response.

Regarding the pharmacodynamic target molecule, COX-2 genetic polymorphisms have been reported in various cancers and can change COX-2 expression and alter susceptibility to carcinomas.[9](#b9-dddt-11-2727) In breast cancer, a reduced risk of hormone receptor-positive breast cancer has been reported in subjects with the COX-2 SNP rs5275 using NSAIDs. These findings suggest that the effects of NSAIDs could be modulated by the characteristics of COX-2, such as genetic polymorphisms.

Previous studies have demonstrated a relationship between interindividual variability in COX-2 gene expression and the variable response to celecoxib treatment.[10](#b10-dddt-11-2727)–[12](#b12-dddt-11-2727) However, no comparisons of COX-2 expression associated with genetic polymorphisms have yet been conducted using objective pharmacodynamic parameters.

In the present study, we investigated the characteristics of the inhibitory effects of COX-2 after NSAID administration according to the genotypes of COX-2 SNPs. To objectively evaluate the inhibitory effects of COX-2, we used a previously investigated lipopolysaccharide (LPS)-derived COX-2 induction model in whole blood. Pharmacokinetic and pharmacodynamic analyses were conducted to exclude causative factors other than COX-2 characteristics. Prior to clinical trials, the genotypes of the seven most common COX-2 SNPs were selected and analyzed in a larger population to identify the COX-2 SNP types to be used in the present clinical study. Celecoxib, a type of NSAID designed for COX-2 inhibition that is used clinically to treat inflammatory disease, was employed in this study.

## Patients and methods

### Analysis of COX-2 SNP

#### Study population

The present study, involving a COX-2 SNP analysis, was approved by the institutional review board of our institution, and written informed consent for participation was obtained from all subjects after providing a full explanation of the study procedures (Dong-A University Hospital IRB No: 14-071). In total, 324 subjects were screened and participated in the study without exclusion. All subjects were older than 19 years of age and had no genetic disease according to their medical history and physical examination. Although disease states other than genetic disease were not included in the exclusion criteria, all subjects were healthy.

#### Selection of COX-2 SNPs and genotyping

The human COX-2 gene contains 10 exons, and its molecular weight is approximately 8.3 kilobases.[13](#b13-dddt-11-2727) In the present study, data on COX-2 polymorphisms were collected from available public databases, including dbSNP, and several types of COX-2 SNPs were selected for analysis according to the following criteria: 1) a reported minor allele frequency is 5% or greater and 2) previously reported data or evidence indicating an association with COX-2 function. Based on these criteria, a total of seven SNPs of COX-2 (rs20417, rs5275, rs689466, rs4648276, rs2745557, rs689470 and rs2066826) were selected. After the selection of COX-2 SNPs, genomic DNA was extracted from peripheral blood leukocytes using a commercial QIAamp^®^ DNA blood mini kit (Qiagen, Hilden, Germany) according to the manufacturer’s instructions. Genotyping was conducted via a single-base primer extension assay using the ABI PRISM SNaPShot Multiplex kit (Thermo Fisher Scientific, Waltham, MA, USA). The genomic DNA sequences flanking the SNPs of COX-2 were amplified through polymerase chain reaction (PCR) using forward and reverse primer pairs. Information on the primers is presented in [Table 1](#t1-dddt-11-2727). After amplification, the PCR products were purified and added to a SNaPshot Multiplex Ready reaction mixture for the primer extension reaction. Following the removal of excess fluorescent dye terminators with shrimp alkaline phosphatase, the sequence analysis was performed with GeneMapper software (version 4.0; Thermo Fisher Scientific).

| Gene | rs number | Prime sequence |   | Tm |
| --- | --- | --- | --- | --- |
| PTGS2 | rs5275 | Forward primer | TTTTTACACTGTCGATGTTTCC | 55°C |
| Reverse primer | ATATTCATTTAATAATGCACTGATACCT |   |  |  |
| Genotyping primer | TGTTTTTGTTTGATGACAGAAAAAT |   |  |  |
| PTGS2 | rs689466 | Forward primer | GTATCTCACCCTCACATGCT | 55°C |
| Reverse primer | TGTGACCATGGATCAAAGT |   |  |  |
| Genotyping primer | TGTGACCATGGATCAAAGT |   |  |  |
| PTGS2 | rs4648276 | Forward primer | CCCAGGGAAAATTAGTGTG | 55°C |
| Reverse primer | TTCTGGGTCAAATTTCAGTT |   |  |  |
| Genotyping primer | GATGTGGTAAATGAAAACTCRCACA |   |  |  |
| PTGS2 | rs2745557 | Forward primer | AGAGAAGCTACGTGACTTGG | 55°C |
| Reverse primer | AACATAATCCGGGCTTTC |   |  |  |
| Genotyping primer | GAAAGAGCTTGGACCGCTRGAGT |   |  |  |
| PTGS2 | rs689470 | Forward primer | AGAATGTTTAAGGTTAAGAAAGAAA | 55°C |
| Reverse primer | GGCCAGTCCTAGTTTTGAA |   |  |  |
| Genotyping primer | CCTGGGAATTTGGRTTGTGTATG |   |  |  |
| PTGS2 | rs2066826 | Forward primer | TAAACTGCGCCTTTTCAA | 55°C |
| Reverse primer | TGGCATACATCATCAGACC |   |  |  |
| Genotyping primer | AGGGATTTTWAAATATGRGTATAAG |   |  |  |

Table Caption: Primer sets and Tm for the SNaPshot assay

### Clinical trial

#### Subjects

Based on the results regarding the COX-2 SNP genotypes, a clinical trial examining pharmacodynamics according to the genotypes of two SNPs (rs5275 and rs689466) was conducted. The subjects carrying either rs5275 (T>C) or rs689466 (A>G) were selected for screening, and the 20 participants who passed the screening evaluation were enrolled. The following inclusion criteria were used: age 19–50 years, body mass index (BMI) 18.0–30.0 kg/m^2^ and a lack of clinically significant medical history, physical examination findings or routine laboratory tests (12-lead electrocardiogram [ECG], blood chemistry, hematology, infectious serology and urine analysis). Subjects with gastrointestinal disease that may affect the absorption of the investigational product and those who had ingested any prescribed medication within 2 weeks prior to study initiation were excluded. After a full explanation of the objectives and study procedures, written informed consent was obtained from all subjects. The study protocol was approved by the institutional review board, and the study was conducted according to the Declaration of Helsinki and Korean Good Clinical Practice (Dong-A University Hospital IRB No: 15-148, CRIS No: KCT0002094).

#### Study design

This clinical trial was an open-label, single-oral dose, one-treatment study conducted in Korean male subjects at the Clinical Trials Center of our institution. On Day −1, the enrolled subjects were hospitalized at 6 pm, and each of the subjects was provided with an identical, controlled meal after undergoing baseline laboratory tests (eg, clinical laboratory tests, ECG). The subjects were assigned to each group according to their genotype ([Table 2](#t2-dddt-11-2727)). On Day 1, the subjects received a single oral dose of 200 mg celecoxib with 150 mL of water in the fasting state. Subsequently, the participants remained in a sitting position and continued to fast for 4 hours. Blood sampling for pharmacokinetic and pharmacodynamic analyses was performed over the following 24 hours according to a schedule.

| Parameter | rs5275 (T>C) |  |  | rs689466 (A>G) |  |  |
| --- | --- | --- | --- | --- | --- | --- |
| TT | TC | CC | AA | AG | GG |  |
| Number of subjects | 4 | 4 | 1 | 4 | 4 | 3 |
| Sex: male | 4 | 4 | 1 | 4 | 4 | 3 |
| Age, years | 25.25±3.96 | 20.33±1.89 | 25.0±0.0 | 26.0±7.21 | 25.67±3.79 | 30.67±9.29 |
| Height, cm | 171.15±0.87 | 173.38±4.35 | 172.5±0.0 | 173.48±3.1 | 177.33±5.14 | 174.77±2.1 |
| Weight, kg | 72.18±8.29 | 66.15±3.87 | 66.6±0.0 | 71.75±5.62 | 75.23±13.92 | 65.60±4.03 |
| BMI, kg/m2 | 24.62±2.69 | 22.06±1.81 | 22.4±0.0 | 23.87±2.22 | 23.78±3.15 | 21.47±1.04 |

Table Caption: Demographic data of clinical trial for pharmacokinetic and pharmacodynamic analyses

#### Pharmacokinetic and pharmacodynamic assessment

Venous blood samples for the pharmacokinetic evaluation were collected before dosing (0 hour) and at 0.5, 1, 2, 3, 4, 6, 8, 12 and 24 hours after administration of the study drug to determine the plasma concentration of celecoxib. At the indicated time points, blood was drawn into heparinized tubes and immediately centrifuged at 3,000 rpm for 10 minutes at 4°C. The separated plasma samples were transferred to sterile cryotube vials and stored at −70°C until further analysis.

For the pharmacodynamic evaluation of prostaglandin E_2_ (PGE_2_), which represents COX-2 activity, venous blood samples were collected before dosing (0 hour) and at 2, 4, 8, 12 and 24 hours after study drug administration. Samples of 10 mL blood were drawn into K2 EDTA tubes and stored in a 36°C incubator after treatment with 10 ng/mL LPS for 24 hours. After the appropriate incubation time for COX-2 induction, plasma was separated by centrifugation at 3,000 rpm for 10 minutes at 4°C and frozen at −70°C until PGE_2_ measurement.

#### Assay of celecoxib plasma concentrations

The plasma concentration of celecoxib was measured using high-performance liquid chromatography–tandem mass spectrometry (LC–MS/MS) in human plasma. Method validation was conducted with celecoxib (98% purity) and celecoxib-d7, as an internal standard (IS), which were purchased from Toronto Research Chemicals (North York, ON, Canada) and CDN Isotopes, respectively. For solvent and sample delivery, an Agilent 1,200 system consisting of a G1312A quaternary pump, G1379B degasser, 1367B auto-sampler, G1316A thermostat and G1316A column and an oven (TTC) compartment (Agilent, Santa Clara, CA, USA) was used. An API 4,000 triple-quadrupole mass spectrometer equipped with a TurboIonSpray (EIS; Thermo Fisher Scientific) source was used for mass analysis and detection.

A 50 µL aliquot of 0.2 M zinc sulfate was added to 100 µL of plasma sample, followed by vortex mixing for 10 seconds. This plasma sample was combined with 1 mL of acetonitrile (in celecoxib-d7, 100 ng/mL), and the contents of the tube were vortexed for 30 seconds, followed by centrifugation at 12,000 rpm for 5 minutes. The supernatant was finally transferred to appropriate auto-sampler vials, and an aliquot (5 µL) was injected into an LC–MS/MS instrument for analysis. The mobile phase consisted of acetonitrile–0.1% formic acid (85:15, v/v) at a flow rate of 0.2 mL/min, and the used column was Gemini C18^®^ (50×2.0 mm, 5 µm; Phenomenex, Torrance, CA, USA). The column temperature was maintained at 35°C.

A stock solution of 10 mg of celecoxib in 10 mL of acetonitrile was used to produce seven standard working solutions of 100, 500, 1,000, 2,500, 5,000, 10,000 and 30,000 ng/mL of celecoxib. These solutions were generated by further dilution of the stock solution with acetonitrile–water (50:50, v/v).

The calibration standards were prepared at concentrations of 10, 50, 100, 250, 500, 1,000 and 3,000 ng/mL of celecoxib in plasma, and the corresponding QC samples were prepared at 30, 800 and 2,400 ng/mL. The performance of the proposed method, as the % coefficient of variation (CV) of QC samples, was less than 97.47% and 100.21%. For celecoxib, the calibration curves were linear within the range of 10–3,000 ng/mL.

#### Assay of COX-2 activity

COX-2 activity was estimated by measuring the blood PGE_2_ level according to a previously reported method. Plasma PGE_2_ levels were determined using commercially available enzyme immunoassay kits (Cayman Chemical Company, Ann Arbor, MI, USA) according to the manufacturer’s instructions. The standard and experimental samples were assayed in parallel, and the detection limit of PGE_2_ quantification was 15 pg/mL. The reaction solution, including 50 µL of human plasma in each well with 50 µL of enzyme immunoassay reagent, PGE_2_ and a PGE_2_ monoclonal antibody, was incubated for 1.5 hours. The incubated samples were measured in 96-well plates using a spectrophotometer at 405 nm (SpectraMax 340; Molecular Devices Corp., Sunnyvale, CA, USA).

#### Pharmacokinetic and pharmacodynamic analyses

The pharmacokinetic parameters of celecoxib were estimated using the non-compartmental analysis method of WinNonlin^®^ 6.4 (Pharsight Co, Mountain View, CA, USA), based on the plasma concentrations of celecoxib and the actual sampling time. The C_max_ and the time to C_max_ (T_max_) were estimated directly from the plasma concentration–time profiles. The individual area under the concentration–time curve (AUC_0–t_) from dosing to 24 hours or the AUC_last_ was calculated using the linear-up and log-down trapezoidal method. The elimination rate constant (λz) was estimated from the slope of the terminal log-linear phase, and the terminal elimination half-life (t^1/2^) was calculated as ln(2)/λz. The apparent oral clearance (CL/F) was calculated as dose/AUC.

For objective pharmacodynamic analysis, the pharmacodynamic parameters, including the I_max_, T_max_ and the area under the effect curve (AUEC), were estimated. Therefore, in the present study, the individual AUEC was estimated from the plasma PGE_2_ level–time profiles using the linear trapezoidal approximation method to compare the genotypes of COX-2 SNPs. Additionally, the maximum inhibitory effect on PGE_2_ production and the T_max_ were estimated from the time course of PGE_2_ production.

#### Statistical analysis

For the statistical analyses conducted in the present study, SPSS 22.0 (SPSS Korea, Seoul, Republic of Korea) was used. The analysis of pharmacokinetic parameters was performed for the 20 subjects in whom all procedures for the pharmacokinetic analysis were conducted. The analysis of variance (ANOVA) was performed to compare the AUC_0–t_ and C_max_ pharmacokinetic values between COX-2 SNP types.

For the pharmacodynamic analysis, the AUEC for the inhibition of PGE_2_ production was used to compare COX-2 SNP types, and the Kruskal–Wallis test, followed by Bonferroni’s correction, was employed for statistical analysis. The differences between COX-2 SNP groups were considered significant at the probability level of 0.05.

## Results

### COX-2 genotyping

For COX-2 genotyping, a total of 324 subjects (292 males) were enrolled, and seven COX-2 SNPs were selected and analyzed in whole blood. The mean age of the subjects was 26.47±4.49 years, and their mean body weight was 69.45±10.20 kg. The mean height and BMI of the subjects were 173.16±6.99 cm and 23.09±1.87 kg/m^2^, respectively.

The results for the seven SNPs of COX-2 are presented in [Table 3](#t3-dddt-11-2727). With the exception of the COX-2 SNP rs4648276, the other SNPs were not analyzed in some subjects. The numbers of rs689466, rs20417, rs2745557, rs2066826, rs4648276, rs5275 and rs689470 polymorphisms were 323, 323, 318, 321, 324, 322 and 322, respectively. Based on the seven COX-2 SNPs analyzed, the minor allele frequencies of the rs689466 and rs5275 genotypes were remarkably high, at 46% and 18%, respectively. The minor allele frequencies of the other five genotypes were very low, at approximately 1.1%–4.5%; thus, rs689466 and rs5275 were selected for subsequent clinical trials.

### Table 3.

| Gene no | Reference SNP ID | No of analyzed subjects | Sequence position | Variation | Major allele frequency | Minor allele frequency |
| --- | --- | --- | --- | --- | --- | --- |
| 1 | rs689466 | 323 | −1,290 | A>G | 0.533 | 0.467 |
| 2 | rs20417 | 323 | −765 | G>C | 0.986 | 0.014 |
| 3 | rs2745557 | 318 | 335 | C>T | 0.965 | 0.035 |
| 4 | rs2066826 | 321 | 3,629 | G>A | 0.955 | 0.045 |
| 5 | rs4648276 | 324 | 4,068 | T>C | 0.955 | 0.045 |
| 6 | rs5275 | 322 | 6,498 | T>C | 0.820 | 0.180 |
| 7 | rs689470 | 322 | 8,498 | C>T | 0.989 | 0.011 |

Table 3 Caption: Information regarding genotyped polymorphisms in COX-2

### Clinical trial

In the clinical trial, a total of 20 healthy male subjects were enrolled and completed the study. Twelve subjects were assigned to each allele type of rs5275, and eight subjects were assigned to each allele type of rs689466. The demographic data for each allele type of rs5275 and rs689466 are presented in [Table 2](#t2-dddt-11-2727).

#### Pharmacokinetic parameters of celecoxib

The plasma concentration–time profiles of celecoxib were obtained via high-performance LC–MS/MS. The mean plasma concentration–time profiles of celecoxib are presented in [Figure 1](#f1-dddt-11-2727), and the corresponding pharmacokinetic parameters (C_max_, AUC, etc) are presented in [Table 4](#t4-dddt-11-2727). For rs5275 (T>C), the maximum observed plasma concentration (C_max_) was 434.80±118.33, 350.54±96.19 and 627.88±0.0 ng/L for the TT, TC and CC genotypes, respectively, and the individual AUC_0–t_ from dosing to the last measurable time point (AUC_last_) was 3,360.57±346.58, 2,999.67±700.18 and 4,054.37±0.0 ng·h/L/mg, respectively.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8ee0/5604555/c724781c337a/dddt-11-2727Fig1.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5604555_dddt-11-2727Fig1.jpg)

Mean plasma concentration–time profiles of celecoxib according to genotype of SNPs of COX-2 after a single administration of 200 mg celecoxib. Bars represent SDs. (A) Linear scale. (B) Log-linear scale. Abbreviations: SNPs, single-nucleotide polymorphisms; COX-2, cyclooxygenase-2.

| Parameter | rs5275 |  |  | rs689466 |  |  |
| --- | --- | --- | --- | --- | --- | --- |
| TT (n=4) | TC (n=4) | CC (n=1) | AA (n=4) | AG (n=4) | GG (n=3) |  |
| Tmax (hours) | 3.50 (2.00–4.00) | 1.00 (0.500–2.00) | 1.00±0.0 | 3.00 (1.00–4.00) | 2.00 (1.00–4.00) | 4.00 (2.00–4.00) |
| Cmax (ng/L) | 434.80±118.33 (27.22) | 350.54±96.19 (27.44) | 627.88±0.0 | 498.83±281.46 (56.42) | 464.97±151.83 (32.65) | 511.96±448.08 (87.52) |
| AUC0–t (ng·h/L) | 3,360.57±346.58 (10.31) | 2,999.67±700.18 (23.34) | 4,054.37±0.0 | 4,972.19±3,813.76 (76.70) | 3,270.52±444.42 (13.59) | 4,296.56±2,505.39 (58.31) |
| AUCinf (ng·h/L) | 3,674.56±483.051 (13.15) | 3,511.17±672.144 (19.14) | 5,116.59±0.0 | 5,709.80±3,670.566 (64.29) | 3,822.54±735.29 (19.24) | 4,960.72±2,030.01 (40.92) |
| t1/2 (hours) | 10.02±3.03 (30.27) | 10.07±2.52 (24.97) | 28.72±0.0 | 12.97±4.49 (34.58) | 14.33±8.50 (59.29) | 13.60±2.98 (21.93) |
| Vd/F (L) | 814.30±326.41 (40.09) | 836.19±168.41 (20.14) | 1,619.37±0.0 | 796.86±357.90 (44.91) | 1,020.45±455.95 (44.68) | 910.58±439.98 (48.32) |
| CL/F (L/h) | 55.12±7.01 (12.71) | 58.95±13.88 (23.54) | 39.09±0.0 | 44.44±20.38 (45.86) | 53.81±10.33 (19.20) | 44.32±14.68 (33.12) |

Table Caption: Pharmacokinetic comparisons between genotypes after single oral administration of 200 mg celecoxib

For rs689466 (A>G), C_max_ was 498.83±281.46, 464.97±151.83 and 511.96±448.08 ng/L for the AA, AG and GG genotypes, respectively, and AUC_last_ was 4,972.19±3,813.76, 3,270.52±444.42 and 4,296.56±2,505.39 ng·h/L/mg, respectively. In the analysis of the pharmacokinetic parameters C_max_ and AUC_last_, significant differences in drug exposure were not investigated for each SNP genotype (*p*>0.05).

#### Pharmacodynamic analysis of celecoxib according to COX-2 SNPs

Compared with the pre-dose time point (0 hour), the maximum effect of COX-2 inhibition, based on changes in LPS-induced plasma PGE_2_ levels, was achieved at 2.0 hours (median T_max_) for all genotypes of COX-2 SNPs (rs5275, rs689466). Recovery from the inhibitory effects was observed at 4 hours after celecoxib administration, and the effects almost disappeared at 8 hours after administration in all genotype groups, except for the CC genotype of rs5275. Twelve hours after administration, the inhibitory effect of celecoxib nearly completely disappeared. The mean plasma PGE_2_ level–time profiles according to genotype are presented in [Figure 2](#f2-dddt-11-2727).

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8ee0/5604555/e3dbd64553f5/dddt-11-2727Fig2.jpg)

Inhibition of PGE2 production levels after a single oral administration of 200 mg celecoxib according to different genotypes. Abbreviation: PGE2, prostaglandin E2.

The AUEC was calculated from the plasma PGE_2_ level–time profiles and compared with the genotypes of COX-2 SNPs; the results are described in [Table 5](#t5-dddt-11-2727). Considering the AUEC values between genotypes of rs5275 (T>C), genotype CC tended to be higher, but the statistical significance was not investigated. Among the genotypes of rs689466 (A>G), the AUEC value for GG genotype was significantly lower than those for the AA and GA genotypes (*p*=0.029), but the AUEC values of genotypes AA and GA did not differ. The differences in the AUECs between the genotypes of COX-2 SNPs reflect the time point of recovery from inhibited COX-2 activity, as the degree of the AUEC is compatible with the time point of recovery. For example, the CC genotype of rs5275 showed the latest time point of recovery among the evaluated genotypes ([Figure 2](#f2-dddt-11-2727)).

| Parameter | rs5275 |  |  | p-value | rs689466 |  |  | p-value |
| --- | --- | --- | --- | --- | --- | --- | --- | --- |
| TT (n=4) | TC (n=4) | CC (n=1) | AA (n=4) | AG (n=4) | GG (n=3) |  |  |  |
| Tmax (hours) | 2.00 (2.00–2.00) | 2.00 (2.00–2.00) | 2.00 (2.00–2.00) | – | 2.00 (2.00–2.00) | 2.00 (2.00–2.00) | 2.00 (2.00–2.00) | – |
| Imax (%) | 98.222±1.017 (1.035) | 97.495±1.563 (1.603) | 99.165 | – | 96.920±3.214 (3.316) | 93.756±8.970 (9.560) | 94.490±5.940 (6.286) | – |
| AUEC (pg·h/L/mg) | 388.264±124.109 (31.965) | 303.291±74.553 (24.581) | 676.814 | 0.177 | 454.587±125.046 (27.508) | 573.32±269.299 (46.972) | 226.859±58.135 (25.626) | 0.029 |

Table Caption: Pharmacodynamic comparisons of genotypes after single oral administration of 200 mg celecoxib

#### Safety and tolerability in the clinical trial involving pharmacokinetic and pharmacodynamic analyses

During the course of the present study, no adverse events, serious adverse events or adverse drug responses were reported, and no clinically significant changes in vital signs, physical examination findings, ECG intervals or laboratory test parameters were observed compared with baseline findings.

## Discussion

The major finding of the present study was that when pharmacokinetic characteristics are excluded, COX-2 SNPs affect the drug response to NSAIDs or selective COX-2 inhibitors (such as celecoxib) by modulating COX-2 activity.

### Drug response variability – causative factors

Clinically, interindividual variability in drug responses is not unusual, and the causative mechanisms have also been well established in a previous study.[14](#b14-dddt-11-2727) Regarding pharmacokinetics, genetic polymorphisms in drug-metabolizing enzymes, such as hepatic cytochrome P450 isoenzymes, can lead to clinically significant variability in drug responses.[15](#b15-dddt-11-2727) Because the variability of pharmacokinetic parameters such as C_max_ and AUC suggests that the exposure of the body to the administered drug is variable in each subject, prediction of the variability of drug response is not difficult. However, the pharmacokinetic parameters evaluated in the present study did not differ between the compared groups, which were classified based on the genotype of COX-2 SNPs. This finding indicates that the variability in COX-2 inhibition by celecoxib is not associated with the degree of exposure to celecoxib. When the effect of pharmacokinetic characteristics is excluded, the variability in COX-2 inhibition is only regulated by the corresponding genotype.

### Drug response variability – pharmacodynamic analysis according to the genotype

In the present study, the identified genotype of the COX-2 SNP affecting the variability of the response to celecoxib was rs689466 (A>G). The AUEC for the GG genotype of rs689466 was significantly lower than for the AA or AG genotype, suggesting that a subject with the GG genotype of rs689466 would be more responsive to NSAIDs in terms of COX-2 inhibition.

However, it is difficult to exclude the effect of rs5275 (T>C) on the COX-2 inhibition by celecoxib. Although the significance of the results was not investigated, the inhibitory effects associated with the CC genotype of rs5275 tended to be strong. The lack of significance was most likely caused by the small sample size (only one subject with CC genotype of rs5275). Therefore, we expect that rs5275 (T>C) could be demonstrated to act as a factor that affects the variability of NSAIDs and their COX-2 inhibitory effect if further studies are conducted with larger sample size.

In contrast to the AUEC results, the maximum observed inhibitory effect (I_max_) and time to I_max_ (T_max_) showed no significant differences in the inhibitory effect of celecoxib according to the COX-2 SNP genotype, and the median T_max_ value was identical in all genotypes. It is thought that this finding reflects the sampling time of pharmacodynamic assessment; thus, it will be necessary to compensate for the frequency and time interval for pharmacodynamic assessment in future studies.

### Inhibitory effect of celecoxib according to the time – pharmacodynamic evaluation

In the pharmacodynamic analysis, the time duration of the action of celecoxib was estimated. The maximum effect of COX-2 inhibition, based on changes in LPS-induced plasma PGE_2_ levels, was achieved at 2.0 hours for all genotypes of COX-2 SNPs after a single oral administration of 200 mg celecoxib. The inhibitory effect of celecoxib was reversed after T_max_ and completely disappeared at 12 hours after dosing all genotype groups. However, the clinically recommended dosing interval for 200 mg celecoxib is 24 hours. Therefore, it is necessary to study the appropriateness of the current dosing interval for 200 mg celecoxib.

### Importance of COX-2 genotype in various diseases

Currently, the COX-2 pathway or overexpression of COX-2 is recognized as important in various pathologies, including cancer development and cancer progression,[16](#b16-dddt-11-2727) based on extensive evidence.[17](#b17-dddt-11-2727) Although the causative mechanisms associating COX-2 with tumorigenesis are not predictable, the genetic variations in COX-2 are a remarkable mechanism. For example, the rs2143416 SNP is significantly associated with the risk of breast cancer,[18](#b18-dddt-11-2727) and rs5275 was shown to influence the risk of both lung cancer[19](#b19-dddt-11-2727) and breast cancer.[20](#b20-dddt-11-2727) The rs689466 SNP has also been associated with the risk of digestive system cancers.[21](#b21-dddt-11-2727) Additionally, an association between cardiovascular disease and a COX-2 SNP (rs20417) has also been reported.[22](#b22-dddt-11-2727) Based on these findings, we assumed that COX-2 SNPs may be related to the function of COX-2, which could contribute to the variability of the response to NSAIDs, including COX-2-specific inhibitors.

### Objective measurements of the drug response – ex vivo model for clinical application

To evaluate the variability of drug response, the use of objective parameters for measuring the response to drugs is necessary because subjective parameters, such as the visual analog scale, exhibit poor reproducibility and show high interindividual variability. As the estimated body exposure to a drug must be considered when measuring the drug response, due to aforementioned factors, the method applied in the present study was consistent with that used in the clinical fields to predict or determine the drug response of patients using NSAIDs.[23](#b23-dddt-11-2727)

In this clinical trial, we employed an LPS-derived COX-2 induction model in whole blood, which was easy to use and could objectively estimate the drug response.

### Genotype differences – Koreans vs Caucasians

Another important finding of the present study that is important for the clinical use of NSAIDs, including COX-2-specific inhibitors, is that the allele frequencies of COX-2 SNPs in Koreans are different from those in patients with prostate cancer. According to Dossus et al,[24](#b24-dddt-11-2727) the minor allele frequencies of rs20417, rs689466 and rs5275 were 0.17, 0.20 and 0.34, respectively; however, the minor allele frequencies observed in the present study were 0.014, 0.467 and 0.180, respectively. Because the population assessed by Dossus et al primarily consisted of Caucasians from the US or Europe,[24](#b24-dddt-11-2727) the differences in the allele frequencies of the same COX-2 SNPs could reflect racial differences. Considering that rs689466 is a causative factor associated with the variability of the drug response to NSAIDs, this finding is of clinical interest. If genetic polymorphism is a factor affecting the variability of the response to drugs, then the causative SNP differs according to ethnic group in cases of unexpected drug response.

In this study, the obtained results described the relationship between the variability of the response to celecoxib and COX-2 SNPs. However, there were some limitations in the present work. For example, the clinical study was conducted in a small number of healthy, young, male volunteers, and the sample sizes of the genotype groups were not balanced; thus, the results could not fully reflect the clinical pathophysiology of COX-2 and the inhibitory effect of NSAIDs in patients with pain or inflammatory disease. Additionally, the variability of the response to COX-2 inhibitors was investigated in only two SNP types. Therefore, it is difficult to estimate the relationship of the causative SNP types of COX-2.

## Conclusion

This study evaluated that the inhibitory effect of celecoxib in COX-2 induction was different according to the genotype of COX-2 SNPs. The pharmacodynamic analysis revealed that the maximum effect of COX-2 inhibition, based on changes in LPS-induced plasma PGE_2_ levels, was achieved at 2.0 hours for all genotypes of COX-2 SNPs after a single oral administration of 200 mg celecoxib. The inhibitory effect of celecoxib was reversed after T_max_ and completely disappeared at 12 hours after dosing all genotype groups. The AUEC for the GG genotype of rs689466 was significantly lower than for the AA or AG genotype, suggesting that a subject with the GG genotype of rs689466 would be more responsive to NSAIDs in terms of COX-2 inhibition. Additionally, the allele frequencies of COX-2 SNPs (rs5275, rs689466) showed differences between Korean and Caucasian individuals, indicating ethnic differences.

## Acknowledgments

SJL was supported by a Basic Science Research Program through the National Research Foundation of Korea (NRF) funded by the Ministry of Education (NRF – 2014R1A1A3052625).

## Footnotes

## References

1. Luong C, Miller A, Barnett J, Chow J, Ramesha C, Browner MF. Flexibility of the NSAID binding site in the structure of human cyclooxygenase-2. Nat Struct Biol. 1996;3(11):927–933. doi: 10.1038/nsb1196-927.  [DOI](https://doi.org/10.1038/nsb1196-927) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8901870/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Struct%20Biol&title=Flexibility%20of%20the%20NSAID%20binding%20site%20in%20the%20structure%20of%20human%20cyclooxygenase-2&author=C%20Luong&author=A%20Miller&author=J%20Barnett&author=J%20Chow&author=C%20Ramesha&volume=3&issue=11&publication_year=1996&pages=927-933&pmid=8901870&doi=10.1038/nsb1196-927&)

2. Lu G, Tsai AL, Van Wart HE, Kulmacz RJ. Comparison of the peroxidase reaction kinetics of prostaglandin H synthase-1 and -2. J Biol Chem. 1999;274(23):16162–16167. doi: 10.1074/jbc.274.23.16162.  [DOI](https://doi.org/10.1074/jbc.274.23.16162) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10347169/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=Comparison%20of%20the%20peroxidase%20reaction%20kinetics%20of%20prostaglandin%20H%20synthase-1%20and%20-2&author=G%20Lu&author=AL%20Tsai&author=HE%20Van%20Wart&author=RJ%20Kulmacz&volume=274&issue=23&publication_year=1999&pages=16162-16167&pmid=10347169&doi=10.1074/jbc.274.23.16162&)

3. Smith WL, Langenbach R. Why there are two cyclooxygenase isozymes. J Clin Invest. 2001;107(12):1491–1495. doi: 10.1172/JCI13271.  [DOI](https://doi.org/10.1172/JCI13271) | [PMC free article](/articles/PMC200199/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11413152/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Invest&title=Why%20there%20are%20two%20cyclooxygenase%20isozymes&author=WL%20Smith&author=R%20Langenbach&volume=107&issue=12&publication_year=2001&pages=1491-1495&pmid=11413152&doi=10.1172/JCI13271&)

4. Harizi H, Corcuff JB, Gualde N. Arachidonic-acid-derived eicosanoids: roles in biology and immunopathology. Trends Mol Med. 2008;14(10):461–469. doi: 10.1016/j.molmed.2008.08.005.  [DOI](https://doi.org/10.1016/j.molmed.2008.08.005) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18774339/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Trends%20Mol%20Med&title=Arachidonic-acid-derived%20eicosanoids:%20roles%20in%20biology%20and%20immunopathology&author=H%20Harizi&author=JB%20Corcuff&author=N%20Gualde&volume=14&issue=10&publication_year=2008&pages=461-469&pmid=18774339&doi=10.1016/j.molmed.2008.08.005&)

5. Smyth EM, Grosser T, Wang M, Yu Y, FitzGerald GA. Prostanoids in health and disease. J Lipid Res. 2009;50(Suppl):S423–S428. doi: 10.1194/jlr.R800094-JLR200.  [DOI](https://doi.org/10.1194/jlr.R800094-JLR200) | [PMC free article](/articles/PMC2674745/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19095631/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Lipid%20Res&title=Prostanoids%20in%20health%20and%20disease&author=EM%20Smyth&author=T%20Grosser&author=M%20Wang&author=Y%20Yu&author=GA%20FitzGerald&volume=50&issue=Suppl&publication_year=2009&pages=S423-S428&pmid=19095631&doi=10.1194/jlr.R800094-JLR200&)

6. Simmons DL, Botting RM, Hla T. Cyclooxygenase isozymes: the biology of prostaglandin synthesis and inhibition. Pharmacol Rev. 2004;56(3):387–437. doi: 10.1124/pr.56.3.3.  [DOI](https://doi.org/10.1124/pr.56.3.3) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15317910/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacol%20Rev&title=Cyclooxygenase%20isozymes:%20the%20biology%20of%20prostaglandin%20synthesis%20and%20inhibition&author=DL%20Simmons&author=RM%20Botting&author=T%20Hla&volume=56&issue=3&publication_year=2004&pages=387-437&pmid=15317910&doi=10.1124/pr.56.3.3&)

7. Li-Wan-Po A, Girard T, Farndon P, Cooley C, Lithgow J. Pharmacogenetics of CYP2C19: functional and clinical implications of a new variant CYP2C19*17. Br J Clin Pharmacol. 2010;69(3):222–230. doi: 10.1111/j.1365-2125.2009.03578.x.  [DOI](https://doi.org/10.1111/j.1365-2125.2009.03578.x) | [PMC free article](/articles/PMC2829691/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20233192/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Clin%20Pharmacol&title=Pharmacogenetics%20of%20CYP2C19:%20functional%20and%20clinical%20implications%20of%20a%20new%20variant%20CYP2C19*17&author=A%20Li-Wan-Po&author=T%20Girard&author=P%20Farndon&author=C%20Cooley&author=J%20Lithgow&volume=69&issue=3&publication_year=2010&pages=222-230&pmid=20233192&doi=10.1111/j.1365-2125.2009.03578.x&)

8. Ingelman-Sundberg M, Sim SC, Gomez A, Rodriguez-Antona C. Influence of cytochrome P450 polymorphisms on drug therapies: pharmacogenetic, pharmacoepigenetic and clinical aspects. Pharmacol Ther. 2007;116(3):496–526. doi: 10.1016/j.pharmthera.2007.09.004.  [DOI](https://doi.org/10.1016/j.pharmthera.2007.09.004) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18001838/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacol%20Ther&title=Influence%20of%20cytochrome%20P450%20polymorphisms%20on%20drug%20therapies:%20pharmacogenetic,%20pharmacoepigenetic%20and%20clinical%20aspects&author=M%20Ingelman-Sundberg&author=SC%20Sim&author=A%20Gomez&author=C%20Rodriguez-Antona&volume=116&issue=3&publication_year=2007&pages=496-526&pmid=18001838&doi=10.1016/j.pharmthera.2007.09.004&)

9. Dai ZJ, Shao YP, Ma XB, et al. Association of the three common SNPs of cyclooxygenase-2 gene (rs20417, rs689466, and rs5275) with the susceptibility of breast cancer: an updated meta-analysis involving 34,590 subjects. Dis Markers. 2014;2014:484729. doi: 10.1155/2014/484729.  [DOI](https://doi.org/10.1155/2014/484729) | [PMC free article](/articles/PMC4151597/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25214704/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Dis%20Markers&title=Association%20of%20the%20three%20common%20SNPs%20of%20cyclooxygenase-2%20gene%20(rs20417,%20rs689466,%20and%20rs5275)%20with%20the%20susceptibility%20of%20breast%20cancer:%20an%20updated%20meta-analysis%20involving%2034,590%20subjects&author=ZJ%20Dai&author=YP%20Shao&author=XB%20Ma&volume=2014&publication_year=2014&pages=484729&pmid=25214704&doi=10.1155/2014/484729&)

10. Fritsche E, Baek SJ, King LM, Zeldin DC, Eling TE, Bell DA. Functional characterization of cyclooxygenase-2 polymorphisms. J Pharmacol Exp Ther. 2001;299(2):468–476.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/11602656/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharmacol%20Exp%20Ther&title=Functional%20characterization%20of%20cyclooxygenase-2%20polymorphisms&author=E%20Fritsche&author=SJ%20Baek&author=LM%20King&author=DC%20Zeldin&author=TE%20Eling&volume=299&issue=2&publication_year=2001&pages=468-476&pmid=11602656&)

11. Szczeklik W, Sanak M, Szczeklik A. Functional effects and gender association of COX-2 gene polymorphism G-765C in bronchial asthma. J Allergy Clin Immunol. 2004;114(2):248–253. doi: 10.1016/j.jaci.2004.05.030.  [DOI](https://doi.org/10.1016/j.jaci.2004.05.030) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15316498/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Allergy%20Clin%20Immunol&title=Functional%20effects%20and%20gender%20association%20of%20COX-2%20gene%20polymorphism%20G-765C%20in%20bronchial%20asthma&author=W%20Szczeklik&author=M%20Sanak&author=A%20Szczeklik&volume=114&issue=2&publication_year=2004&pages=248-253&pmid=15316498&doi=10.1016/j.jaci.2004.05.030&)

12. Shi J, Misso NL, Duffy DL, Thompson PJ, Kedda MA. A functional polymorphism in the promoter region of the cyclooxygenase-2 gene is not associated with asthma and atopy in an Australian population. Clin Exp Allergy. 2004;34(11):1714–1718. doi: 10.1111/j.1365-2222.2004.02094.x.  [DOI](https://doi.org/10.1111/j.1365-2222.2004.02094.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15544595/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Exp%20Allergy&title=A%20functional%20polymorphism%20in%20the%20promoter%20region%20of%20the%20cyclooxygenase-2%20gene%20is%20not%20associated%20with%20asthma%20and%20atopy%20in%20an%20Australian%20population&author=J%20Shi&author=NL%20Misso&author=DL%20Duffy&author=PJ%20Thompson&author=MA%20Kedda&volume=34&issue=11&publication_year=2004&pages=1714-1718&pmid=15544595&doi=10.1111/j.1365-2222.2004.02094.x&)

13. Kosaka T, Miyata A, Ihara H, et al. Characterization of the human gene (PTGS2) encoding prostaglandin-endoperoxide synthase 2. Eur J Biochem. 1994;221(3):889–897. doi: 10.1111/j.1432-1033.1994.tb18804.x.  [DOI](https://doi.org/10.1111/j.1432-1033.1994.tb18804.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8181472/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Biochem&title=Characterization%20of%20the%20human%20gene%20(PTGS2)%20encoding%20prostaglandin-endoperoxide%20synthase%202&author=T%20Kosaka&author=A%20Miyata&author=H%20Ihara&volume=221&issue=3&publication_year=1994&pages=889-897&pmid=8181472&doi=10.1111/j.1432-1033.1994.tb18804.x&)

14. Wood AJ. Racial differences in the response to drugs – pointers to genetic differences. N Engl J Med. 2001;344(18):1394–1396. doi: 10.1056/NEJM200105033441811.  [DOI](https://doi.org/10.1056/NEJM200105033441811) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11336055/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Racial%20differences%20in%20the%20response%20to%20drugs%20%E2%80%93%20pointers%20to%20genetic%20differences&author=AJ%20Wood&volume=344&issue=18&publication_year=2001&pages=1394-1396&pmid=11336055&doi=10.1056/NEJM200105033441811&)

15. Bae JW, Choi CI, Jang CG, Lee SY. Effects of CYP2C9*1/*13 on the pharmacokinetics and pharmacodynamics of meloxicam. Br J Clin Pharmacol. 2011;71(4):550–555. doi: 10.1111/j.1365-2125.2010.03853.x.  [DOI](https://doi.org/10.1111/j.1365-2125.2010.03853.x) | [PMC free article](/articles/PMC3080643/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21395648/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Clin%20Pharmacol&title=Effects%20of%20CYP2C9*1/*13%20on%20the%20pharmacokinetics%20and%20pharmacodynamics%20of%20meloxicam&author=JW%20Bae&author=CI%20Choi&author=CG%20Jang&author=SY%20Lee&volume=71&issue=4&publication_year=2011&pages=550-555&pmid=21395648&doi=10.1111/j.1365-2125.2010.03853.x&)

16. Dannenberg AJ, Subbaramaiah K. Targeting cyclooxygenase-2 in human neoplasia: rationale and promise. Cancer Cell. 2003;4(6):431–436. doi: 10.1016/s1535-6108(03)00310-6.  [DOI](https://doi.org/10.1016/s1535-6108(03)00310-6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14706335/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Cell&title=Targeting%20cyclooxygenase-2%20in%20human%20neoplasia:%20rationale%20and%20promise&author=AJ%20Dannenberg&author=K%20Subbaramaiah&volume=4&issue=6&publication_year=2003&pages=431-436&pmid=14706335&doi=10.1016/s1535-6108(03)00310-6&)

17. Harris RE. Cyclooxygenase-2 (cox-2) blockade in the chemoprevention of cancers of the colon, breast, prostate, and lung. Inflammopharmacology. 2009;17(2):55–67. doi: 10.1007/s10787-009-8049-8.  [DOI](https://doi.org/10.1007/s10787-009-8049-8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19340409/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Inflammopharmacology&title=Cyclooxygenase-2%20(cox-2)%20blockade%20in%20the%20chemoprevention%20of%20cancers%20of%20the%20colon,%20breast,%20prostate,%20and%20lung&author=RE%20Harris&volume=17&issue=2&publication_year=2009&pages=55-67&pmid=19340409&doi=10.1007/s10787-009-8049-8&)

18. Gallicchio L, McSorley MA, Newschaffer CJ, et al. Nonsteroidal anti-inflammatory drugs, cyclooxygenase polymorphisms, and the risk of developing breast carcinoma among women with benign breast disease. Cancer. 2006;106(7):1443–1452. doi: 10.1002/cncr.21763.  [DOI](https://doi.org/10.1002/cncr.21763) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16502408/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer&title=Nonsteroidal%20anti-inflammatory%20drugs,%20cyclooxygenase%20polymorphisms,%20and%20the%20risk%20of%20developing%20breast%20carcinoma%20among%20women%20with%20benign%20breast%20disease&author=L%20Gallicchio&author=MA%20McSorley&author=CJ%20Newschaffer&volume=106&issue=7&publication_year=2006&pages=1443-1452&pmid=16502408&doi=10.1002/cncr.21763&)

19. Vogel U, Christensen J, Wallin H, et al. Polymorphisms in genes involved in the inflammatory response and interaction with NSAID use or smoking in relation to lung cancer risk in a prospective study. Mutat Res. 2008;639(1–2):89–100. doi: 10.1016/j.mrfmmm.2007.11.004.  [DOI](https://doi.org/10.1016/j.mrfmmm.2007.11.004) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18164040/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mutat%20Res&title=Polymorphisms%20in%20genes%20involved%20in%20the%20inflammatory%20response%20and%20interaction%20with%20NSAID%20use%20or%20smoking%20in%20relation%20to%20lung%20cancer%20risk%20in%20a%20prospective%20study&author=U%20Vogel&author=J%20Christensen&author=H%20Wallin&volume=639&issue=1%E2%80%932&publication_year=2008&pages=89-100&pmid=18164040&doi=10.1016/j.mrfmmm.2007.11.004&)

20. Cox DG, Buring J, Hankinson SE, Hunter DJ. A polymorphism in the 3′ untranslated region of the gene encoding prostaglandin endoperoxide synthase 2 is not associated with an increase in breast cancer risk: a nested case-control study. Breast Cancer Res. 2007;9(1):R3. doi: 10.1186/bcr1635.  [DOI](https://doi.org/10.1186/bcr1635) | [PMC free article](/articles/PMC1851394/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17214885/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Breast%20Cancer%20Res&title=A%20polymorphism%20in%20the%203%E2%80%B2%20untranslated%20region%20of%20the%20gene%20encoding%20prostaglandin%20endoperoxide%20synthase%202%20is%20not%20associated%20with%20an%20increase%20in%20breast%20cancer%20risk:%20a%20nested%20case-control%20study&author=DG%20Cox&author=J%20Buring&author=SE%20Hankinson&author=DJ%20Hunter&volume=9&issue=1&publication_year=2007&pages=R3&pmid=17214885&doi=10.1186/bcr1635&)

21. Dong J, Dai J, Zhang M, Hu Z, Shen H. Potentially functional COX-2-1195G>A polymorphism increases the risk of digestive system cancers: a meta-analysis. J Gastroenterol Hepatol. 2010;25(6):1042–1050. doi: 10.1111/j.1440-1746.2010.06293.x.  [DOI](https://doi.org/10.1111/j.1440-1746.2010.06293.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20594217/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Gastroenterol%20Hepatol&title=Potentially%20functional%20COX-2-1195G>A%20polymorphism%20increases%20the%20risk%20of%20digestive%20system%20cancers:%20a%20meta-analysis&author=J%20Dong&author=J%20Dai&author=M%20Zhang&author=Z%20Hu&author=H%20Shen&volume=25&issue=6&publication_year=2010&pages=1042-1050&pmid=20594217&doi=10.1111/j.1440-1746.2010.06293.x&)

22. Zhang MM, Xie X, Ma YT, et al. Association of COX-2-765G>C genetic polymorphism with coronary artery disease: a meta-analysis. Int J Clin Exp Med. 2015;8(5):7412–7418.  [PMC free article](/articles/PMC4509228/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26221283/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Clin%20Exp%20Med&title=Association%20of%20COX-2-765G>C%20genetic%20polymorphism%20with%20coronary%20artery%20disease:%20a%20meta-analysis&author=MM%20Zhang&author=X%20Xie&author=YT%20Ma&volume=8&issue=5&publication_year=2015&pages=7412-7418&pmid=26221283&)

23. Shin D, Lee SJ, Ha YM, et al. Pharmacokinetic and pharmacodynamic evaluation according to absorption differences in three formulations of ibuprofen. Drug Des Devel Ther. 2017;11:135–141. doi: 10.2147/DDDT.S121633.  [DOI](https://doi.org/10.2147/DDDT.S121633) | [PMC free article](/articles/PMC5221548/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28115830/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Des%20Devel%20Ther&title=Pharmacokinetic%20and%20pharmacodynamic%20evaluation%20according%20to%20absorption%20differences%20in%20three%20formulations%20of%20ibuprofen&author=D%20Shin&author=SJ%20Lee&author=YM%20Ha&volume=11&publication_year=2017&pages=135-141&pmid=28115830&doi=10.2147/DDDT.S121633&)

24. Dossus L, Kaaks R, Canzian F, et al. PTGS2 and IL6 genetic variation and risk of breast and prostate cancer: results from the Breast and Prostate Cancer Cohort Consortium (BPC3) Carcinogenesis. 2010;31(3):455–461. doi: 10.1093/carcin/bgp307.  [DOI](https://doi.org/10.1093/carcin/bgp307) | [PMC free article](/articles/PMC2832545/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19965896/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Carcinogenesis&title=PTGS2%20and%20IL6%20genetic%20variation%20and%20risk%20of%20breast%20and%20prostate%20cancer:%20results%20from%20the%20Breast%20and%20Prostate%20Cancer%20Cohort%20Consortium%20(BPC3)&author=L%20Dossus&author=R%20Kaaks&author=F%20Canzian&volume=31&issue=3&publication_year=2010&pages=455-461&pmid=19965896&doi=10.1093/carcin/bgp307&)
