# Genetic predictors of the maximum doses patients receive during clinical use of the anti-epileptic drugs carbamazepine and phenytoin

## Metadata
**Authors:** Sarah K Tate, Chantal Depondt, Sanjay M Sisodiya, Gianpiero L Cavalleri, Stephanie Schorge, Nicole Soranzo, Maria Thom, Arjune Sen, Simon D Shorvon, Josemir W Sander, Nicholas W Wood, David B Goldstein
**Journal:** Proceedings of the National Academy of Sciences of the United States of America
**Date:** 2005 Apr 1
**DOI:** [10.1073/pnas.0407346102](https://doi.org/10.1073/pnas.0407346102)
**PMID:** 15805193
**PMCID:** PMC556232
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC556232/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC556232/pdf/pnas-0407346102.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC556232/pdf/pnas-0407346102.pdf)

## Abstract

Phenytoin and carbamazepine are effective and inexpensive anti-epileptic drugs (AEDs). As with many AEDs, a broad range of doses is used, with the final “maintenance” dose normally determined by trial and error. Although many genes could influence response to these medicines, there are obvious candidates. Both drugs target the α-subunit of the sodium channel, encoded by the SCN family of genes. Phenytoin is principally metabolized by CYP2C9, and both are probable substrates of the drug transporter P-glycoprotein. We therefore assessed whether variation in these genes associates with the clinical use of carbamazepine and phenytoin in cohorts of 425 and 281 patients, respectively. We report that a known functional polymorphism in CYP2C9 is highly associated with the maximum dose of phenytoin (P = 0.0066). We also show that an intronic polymorphism in the SCN1A gene shows significant association with maximum doses in regular usage of both carbamazepine and phenytoin (P = 0.0051 and P = 0.014, respectively). This polymorphism disrupts the consensus sequence of the 5′ splice donor site of a highly conserved alternative exon (5N), and it significantly affects the proportions of the alternative transcripts in individuals with a history of epilepsy. These results provide evidence of a drug target polymorphism associated with the clinical use of AEDs and set the stage for a prospective evaluation of how pharmacogenetic diagnostics can be used to improve dosing decisions in the use of phenytoin and carbamazepine. Although the case made here is compelling, our results cannot be considered definitive or ready for clinical application until they are confirmed by independent replication.

Keywords: association study, epilepsy, pharmacogenetics

## Materials and Methods

In this study, we have considered the known functional alleles *2 and *3 at the *CYP2C9* gene and the putatively functional 3435C>T polymorphism in the *ABCB1* gene. Because no common functional variants are known for *SCN1A*, we used a haplotype-tagging strategy ([14](#ref14)). We have related variation in all three genes to the maximum dose of phenytoin in 281 patients treated with phenytoin. The dose measure that we used is the maximum dose patients were exposed to during their regular treatment of epilepsy. In most, but not all, cases, the maximum dose used here will also be the maintenance dose, because starting doses tend to be lower than what is required (see *Discussion*). For carbamazepine, we related variation in both *SCN1A* and *ABCB1* to the maximum dose in 425 patients. Finally, we tested for association with presence or absence of ADRs. There were no significant violations of Hardy–Weinberg equilibrium after Bonferroni corrections for multiple comparisons.

**Subjects.** This study was approved by the relevant institutional Ethics Committees. Patients who self-identified as being of European ancestry with a diagnosis of epilepsy were recruited from the specialized epilepsy clinic of the National Hospital for Neurology and Neurosurgery (London) after written informed consent was obtained. Extensive clinical data were obtained and were stored in a computerized database. The DNA collection contains DNA from patients over the last 3 years and a number of patients who are no longer attending clinic. We identified 448 patients who were treated with phenytoin, of whom 119 were continuing treatment at the time of recruitment; the remainder had stopped treatment. DNA and dose information was available for 281 patients ([Fig. 1](#fig1)). Of these, 115 are female and 166 are male, and age at start of treatment ranges from 1 to 72 years, with a mean of 26 (age data available for 77% of patients). We identified 533 patients who were treated with carbamazepine; DNA and dose information was available for 425 ([Fig. 2](#fig2)). Of these, 202 are female and 223 are male, and age at start of treatment ranges from 3 to 77 years, with a mean of 27 (age data available for 81% of patients). Of the 425 patients exposed to carbamazepine, 240 were not included in the phenytoin analyses.

### Fig. 1.

![Fig. 1.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e6b3/556232/23b7ef4dbe64/zpq0130577980001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=556232_zpq0130577980001.jpg)

Distribution of maximum phenytoin doses.

### Fig. 2.

![Fig. 2.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e6b3/556232/e861a4021999/zpq0130577980002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=556232_zpq0130577980002.jpg)

Distribution of maximum carbamazepine doses.

The following clinical details were recorded whenever available: date when phenytoin/carbamazepine was started and stopped, maximum dosage reached, response, and occurrence of ADRs. Although more details are available in the relevant clinical notes, it has not been possible in all cases to review these notes.

**Brain Tissue.** Thirty-two pairs of surgically resected perilesional temporal neocortex and hippocampus brain tissue were selected at random from the archives of frozen tissue at the National Hospital for Neurology and Neurosurgery. In each case, therapeutic surgery had been undertaken to relieve chronic, drug-resistant epilepsy. All tissue had been flash-frozen in liquid nitrogen within 30 min of resection and stored at –80°C until use. Routine detailed histological examination of the fixed (unaffected) temporal lobe and hippocampus from each patient had shown hippocampal sclerosis and the absence of epileptogenic pathology in the temporal lobe. In each case, written informed consent had been obtained from the patient for the use of resection material for research approved by the institutional Ethics Committee; all samples were irreversibly anonymized before analyses. Twenty-three brain samples from patients with Parkinson's disease were obtained from the Brain Bank at the Institute of Neurology. Samples were anonymized, and ethical permission for this study was obtained from the joint research Ethics Committee of the National Hospital for Neurology and Neurosurgery and the Institute of Neurology.

**Genotyping.** *CYP2C9**2 and *CYP2C9**3 genotyping was performed by using predeveloped Taqman assay reagents for allelic discrimination (Applied Biosystems) according to the manufacturer's instructions.

The four *SCN1A* tagging SNPs (tSNPs) (rs590478, rs8191987, rs3812718, and rs2126152) had been previously genotyped by using Taqman assays (unpublished work). [These tSNPs correspond to SNP1, SNP5, SNP7, and SNP8 in ref. [14](#ref14) and have frequencies of 0.24, 0.13, 0.45, and 0.32, respectively, in 384 unrelated control individuals from the British twin registry ([15](#ref15))]. The researchers found the proportion of haplotype diversity explained by these tSNPs to be 94% (criterion 2 in tagit), and the average haplotype *r*^2^ (criterion 5 in tagit 3.00, available at [www.genome.duke.edu/research/centers/pg2](http://www.genome.duke.edu/research/centers/pg2)) to be 0.8 (using data from ref. [14](#ref14)). The high values for the explained proportion of haplotype diversity and average haplotype *r*^2^ also mean that common SNPs (minor allele frequency > 5–8%) in the gene are generally predicted well by the subset of tSNPs and that little loss of power, therefore, is expected in typing the tSNPs instead of typing directly a causal SNP.

*ABCB1* 3435C>T genotypes had been previously determined by using direct sequencing ([4](#ref4)).

**Nucleic Acid Purification from Brain Samples and RT-PCR.** Genomic DNA was extracted from ≈25 mg of brain tissue by using the Wizard Genomic DNA Purification Kit (Promega) according to the manufacturer's conditions. Total RNA was isolated from ≈30 mg of different tissue from the same brain by using the Lipid Tissue Purification Kit (Qiagen, Valencia, CA) according to manufacturer's conditions. The RNA was quantified spectrophotometrically at 260 nm, and 1 μg RNA of each sample was reverse-transcribed to cDNA by using the High Capacity cDNA Synthesis Kit (Applied Biosystems) in standard conditions. RT-PCR was carried out on a volume of cDNA corresponding to 10 ng of starting RNA.

RT-PCR was used to determine that exon 5N is present in human mRNAs by using commercially obtained human fetal total brain mRNA (Stratagene). Adult human cDNA was obtained from brain tissues and amplified with primers flanking exon 5 designed to amplify exons 5A and 5N equally: forward, CCACCTCTGCCCTGTACATT; reverse, CTCCCACAATGGTTTTCAGG.

The resulting fragment was digested with *Ava*II, which cuts only copies containing exon 5N, for >2 h and separated on 3% agarose gels in the presence of ethidium bromide. The relative intensity of the 5N product and 5A product was measured for each sample by using the syngene package. The ratio of 5A to 5N was corrected for the molecular weight of the two products.

**Statistical Analyses.** All regression analyses were implemented in the usual way by using statistica (StatSoft, Tulsa, OK) with phenytoin/carbamazepine dose as the dependent variable and genotype scores as the independent predictors.

Multilocus association analysis was carried out by using the score test ([16](#ref16)). Briefly, the method uses an expectation maximization algorithm to infer haplotypes from a set of unrelated individuals. A score statistic measuring the association of the inferred haplotypes with the quantitative trait is then estimated. All these analyses were implemented in the software package r 1.9.1.

Standard χ^2^ analyses were used to compare genotype frequencies between individuals with and without ADRs.

A Student *t* test was used to compare ratios of SCN1A with 5A to SCN1A with 5N by *SCN1A* IVS5–91 G>A genotype.

## Results

For carbamazepine, we find that one of the *SCN1A* tSNPs (SNP7, IVS5–91 G>A, or rs3812718) is highly associated with maximum dose. A regression model allowing arbitrary effects for each genotype is significant at the level *P* = 0.0051 (uncorrected). However, the genotypic effects are consistent with additive effects, and, under a regression model restricted to additive effects, the significance is *P* = 0.0014 (uncorrected). These results remain significant after Bonferroni correction for five tests (four in the *SCN1A* gene and one in the *ABCB1* gene). Maximum doses averaged 1,313, 1,225, and 1,083 mg for AA, AG, and GG individuals, with genotype counts of 112, 220, and 93 individuals, respectively ([Fig. 3](#fig3)). A weighted linear haplotype regression using all of the tSNPs does not increase significance.

### Fig. 3.

![Fig. 3.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e6b3/556232/3a33d0b93436/zpq0130577980003.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=556232_zpq0130577980003.jpg)

Distribution of maximum carbamazepine doses for each SCN1A IVS5-91 G>A genotype.

For phenytoin, we find that the *CYP2C9**3 allele shows significant association with maximum dose, with *P* = 0.0066 (uncorrected). Because there was only one individual homozygous for the *****3 allele in our cohort, we excluded this genotype from our regression model (although this single observation follows the same trend of reduction in maximum dose). This value remains significant after Bonferroni correction for seven independent genotyping tests (four tests in the *SCN1A* gene, two tests in *CYP2C9*, and one test in *ABCB1*). Mean phenytoin doses for individuals with 0, 1, or 2 copies of the *3 allele were 354, 309, and 250 mg, respectively (but note that numbers for these three genotypes are 229, 39, and 1, respectively). *CYP2C9**2 did not show a significant association with dose. A multiple regression analysis of combined *2 and *3 genotype did not support a significant role for *2.

The *SCN1A* IVS5–91 G>A polymorphism is also associated with the dosing of phenytoin, with *P* = 0.014 (uncorrected) under an unrestricted regression model and *P* = 0.0045 (uncorrected) under an additive model. The latter model would appear to be indicated from the apparent additive effect of the *SCN1A* genotype on carbamazepine dosing explained above. Under the additive model, significance is retained after correcting for seven independent tests. The unrestricted model shows only a trend after correction. Maximum phenytoin doses averaged 373, 340, and 326 mg for AA, AG, and GG individuals, with genotype counts of 73, 109, and 60, respectively. A weighted linear haplotype regression using all of the tSNPs does not increase significance. When the combined *CYP2C9**3 and *SCN1A* IVS5–91 G>A are considered, the doses range from a mean of 250 mg for the single *3*3/GG individual to 297 mg for the 18 *1*3/AG individuals and 377 mg for the 62 *1*1/AA individuals (*P* = 0.014, uncorrected under unrestricted model). (The single *3*3/GG individual has again been excluded; however, results for this individual followed the same trend).

Of the patients included in the carbamazepine analysis, 185 also had been included in the phenytoin analysis. When these patients are not included, the result remains significant under an additive model, with *P* = 0.0063 (uncorrected). An unrestricted model gives *P* = 0.020 (uncorrected). The phenytoin and carbamazepine results, therefore, provide a functional replication of the effect of the *SCN1A* variant. (See Table 2, which is published as [supporting information](http://www.pnas.org/cgi/content/full/0407346102/DC1) on the PNAS web site.)

*The ABCB1* 3435C>T polymorphism shows no association with dosing for either phenytoin or carbamazepine.

We report here that the IVS5–91 G>A polymorphism in *SCN1A* affects the alternative splicing of exon 5 ([Fig. 4](#fig4)). This polymorphism is located in the 5′ splice donor site of a highly conserved, alternatively spliced exon apparently expressed mainly in fetuses (5N) ([17](#ref17)). The major allele (A) disrupts the consensus sequence of the fetal exon (5N), possibly reducing the expression of this exon relative to the adult exon (5A). A similar splicing event occurs in the German cockroach sodium channel gene, *paraCSMA*, but in a different domain (domain III). Substituting aspartic acid into the S3–S4 linker is associated with altered voltage-gating and sensitivity to the insecticide deltamethrin (although German cockroach sodium channel gene exons have other substitutions in addition to that which is similar to the *SCN1A* exon 5 substitution, which may contribute to differences in pesticide sensitivity) ([18](#ref18)).

### Fig. 4.

![Fig. 4.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e6b3/556232/c2b491c35b89/zpq0130577980004.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=556232_zpq0130577980004.jpg)

The position of the SCN1A IVS5-91 G>A polymorphism and how it correlates with the proportion of 5N in total SCN1A in the temporal lobe and hippocampus of epileptic individuals. (A) Genomic structure of SCN1A surrounding exons 5N and 5A, and regulation of exon 5N in epileptic tissues. (B) Proportion of SCN1A 5N transcript in brain tissue from patients with a history of epilepsy.

We first studied human fetal whole-brain mRNA and confirmed that exon 5N is present (data not shown). Next, we amplified the region including exon 5 in adult human cDNA samples derived from brain tissue with primers flanking that exon. The product of this PCR was digested with the restriction enzyme *Ava*II, which cuts only in copies containing exon 5N. In the mRNA purified from fetal brain (the genotype is unavailable from the commercial mRNA), >60% of the amplified SCN1A mRNA contained exon 5N. We also looked in adult brains derived from a Parkinson's disease brain bank. In these adult brains without a history of epilepsy, the levels of SCN1A with exon 5N were much lower, with 9.5 ± 0.7% (*n* = 23) over all genotypes and with individuals with the AA genotype having slightly, but not significantly, less 5N (8.6 ± 0.75%, *n* = 5) than individuals of either AG (9.94 ± 1.09%, *n* = 14) or GG (9.2 ± 0.96%, *n* = 4) genotype.

There is evidence that seizures can up-regulate the inclusion of exon 5N in other neuronal sodium channels in rodents ([19](#ref19)). Therefore, we also assessed the percentage of SCN1A mRNA containing exons 5A and 5N in brain resection tissue derived from patients undergoing surgery for refractory epilepsy. In these tissues, the amount of SCN1A containing exon 5N was significantly up-regulated in the temporal lobe (TL) relative to the hippocampus (Hipp) in individuals with the permissive (that is, predicted to result in more efficient splicing of 5N) GG genotype (TL 14.6 ± 1.2%, Hipp 11.2 ± 0.8%, *n* = 8, *P* = 0.023) but not in individuals with the AA (TL 11.0 ± 1.5%, Hipp 11.7 ± 2.2%, *n* = 7) or AG (TL 15.1 ± 1.3%, Hipp 13.4 ± 1.9, *n* = 17) genotypes ([Fig. 4B](#fig4)).

Taken together, these results show that seizures influence the proportions of alternative transcripts of the *SCN1A* gene. We also show that the influence of seizures depends on genotype, with the GG permissive genotype resulting in a significant increase of the 5N form in the temporal lobe relative to the hippocampus. These results do not make clear how the IVS5–91 G>A splice site polymorphism influences sensitivity to carbamazepine and phenytoin. Future work focused on precise expression patterns in subregions of the hippocampus and functional assays of drug sensitivity of the channels encoded by 5A and 5N may eventually help to clarify this. Finally, it is also possible that the presence/absence of 5N during development leads to changes in other sodium channels, which change adult sensitivity to sodium channel blockade. This work, however, provides unique evidence of how seizures influence neuronal function in humans; neither rats nor mice possess a functional copy of exon 5N in SCN1A (S.S., unpublished data).

Given the demonstrated association of the IVS5–91 G>A polymorphism and 5N and 5A levels, and its presence in a splice donor consensus site, it is very likely that this polymorphism is itself the causal polymorphism for altered sensitivity to phenytoin and carbamazepine. Because of the extensive linkage disequilibrium throughout the gene, it is formally possible that the causal variant lies elsewhere ([14](#ref14)). We have, however, undertaken exhaustive screening of the *SCN1A* exons and intron–exon boundaries and have not found any other common variants that are predicted to have functional effects (unpublished work).

No associations were found between any genotype and presence of an ADR or presence of the subset of ADRs that are central nervous system-related.

Although all patients in this study self-identified as being of European ancestry, cryptic stratification can still drive spurious associations between polymorphisms and phenotypes, including drug responses ([20](#ref20)). One approach for checking whether stratification is present for a given phenotype, termed “genome control,” assesses the association of that phenotype with markers from elsewhere in the genome that are not linked to the associated polymorphism under study ([21](#ref21)). As part of other projects in the laboratory not related to this work on dosing requirements for carbamazepine and phenytoin, we have typed a series of unlinked polymorphisms. We have assessed the degree of association of each of these polymorphisms with carbamazepine dosing. We find that none of these polymorphisms is significantly associated with maximum dose ([Fig. 5](#fig5)). Therefore, we may set a threshold on the probability that the significant association for the *SCN1A* polymorphism is influenced only by stratification. Under this null hypothesis, the distribution is as described by our set of genome control markers, and the *SCN1A* polymorphism is a significant outlier in this distribution, with *P* < 0.05. As noted in ref. [21](#ref21), this formulation of genomic control is conservative (see Table 3, which is published as [supporting information](http://www.pnas.org/cgi/content/full/0407346102/DC1) on the PNAS web site).

### Fig. 5.

![Fig. 5.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e6b3/556232/9096fc69d55b/zpq0130577980005.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=556232_zpq0130577980005.jpg)

Association of 20 unlinked markers with maximum carbamazepine dose.

We also note that some of the polymorphisms considered were chosen for relevance to epilepsy and so could possibly influence dose requirements, but in the context of using them for genome control, this effect is a conservative one. This analysis, therefore, effectively rules out stratification as an explanation of our results.

## Discussion

We have associated functional polymorphisms with the dose used in regular clinical practice for two leading AEDs, phenytoin and carbamazepine. For phenytoin, a well known low-activity variant in the *CYP2C9* gene associates with dose, as does a functional variant in the *SCN1A* gene, encoding the target of phenytoin. The *SCN1A* variant is also highly associated with dosing of carbamazepine, thus providing functional replication of the effect of this variant. This functional replication, together with the apparent function of this polymorphism, makes a strong case that the association reported here is real. However, although the case made here is compelling, as is usual with association genetics, our results cannot be considered definitive or ready for clinical application until they are confirmed by independent replication. To our knowledge, this *SCN1A* variant is the first polymorphism in a drug target associated with the use of an AED and one of only a handful of target polymorphisms for which there is strong evidence of an effect on clinical drug use ([22](#ref22)). Furthermore, our study demonstrates the pharmacologic significance of alternative splicing in a human sodium channel. Although there are other examples of alternative splicing in other human sodium channel genes, none has been associated with functional effects ([23](#ref23)).

This polymorphism is potentially of more general importance because of the prominence of sodium channel blockade in the treatment of epilepsy (and other neurological conditions). For example, more than half of epilepsy patients treated pharmacologically in the United Kingdom receive a drug that principally targets the sodium channel ([24](#ref24)).

With respect to carbamazepine and phenytoin, this study provides a direction for a dosing scheme to be used in a prospective study to assess how pharmacogenetic diagnostics can improve dosing decisions. In particular, it may be clinically relevant to determine whether some individuals can safely be given more rapid dose increases. Although these polymorphisms explain relatively little of the total variation in the unselected cohort (6.5% and 2.5% for phenytoin and carbamazepine, respectively), it is likely that in more controlled settings, they will have a much larger proportionate effect. Our study did not take into account, for example, other AEDs taken together with phenytoin or carbamazepine, some of which are known to induce (e.g., phenobarbitone) or inhibit (e.g., sodium valproate) cytochrome P450 enzymes. Nevertheless, in our cohort, we see average dose ranges across genotypes from 127 to 230 mg for phenytoin and carbamazepine, respectively, indicating that there is an important effect of these variants on dose. Presumably, in a selected cohort, the effect would be stronger. Therefore, these results suggest the possibility that genetic diagnostics could reduce the time it takes, on average, to control seizures by using phenytoin and carbamazepine.

One limitation of the current study is that the database on which our analyses were based records only the maximum dose received by each patient, rather than maintenance dose. The implications of this limitation are somewhat different for phenytoin and carbamazepine.

When the starting dose is insufficient to control seizures, it may be increased until control is achieved. If the initial dose produces ADRs, on the other hand, it may be lowered. On balance, therefore, upward adjustment of dose is due to lack of efficacy, and downward adjustment is due to ADRs. This means, for example, that the effect of *3 on phenytoin dose is probably estimated conservatively here: Any effect of *3 on dose reduction due to ADRs is not visible in our data. It would appear, however, that any effect of *3 on dose because of ADRs is small; we have observed no direct association between *3 and ADRs. For carbamazepine, recording maximum dose seems less of a limitation, because starting doses are virtually always less than what is finally necessary to control seizures.

Our results support the view that the major target, transporter, and drug metabolizing enzyme are good starting points to study drug response and that pharmacogenetic traits, therefore, are more tractable for genetic analyses than are those for common disease predisposition ([22](#ref22)). We also emphasize that a haplotype-tagging strategy ([14](#ref14)) identified a previously unknown functional variant in the *SCN1A* gene. This functional variant was found 91 bp away from the nearest exon known at the time of the study, illustrating the need for exhaustive tagging.

Overall, our findings suggest that using genotype data may make it possible to safely reduce the time required to reach an effective dose. Therefore, it is also a priority to assess the utility of dose adjustment on the basis of genotype for these medicines in a prospective clinical study. Prospective studies of carbamazepine and phenytoin, informed by a detailed retrospective study, would also serve as a useful model for future pharmacogenetic studies ([25](#ref25)).

## Supplementary Material

## Acknowledgments

D.B.G. is a Wolfson/Royal Society Research Merit Award holder. C.D. was supported by the National Society for Epilepsy, United Kingdom. G.L.C. is supported by the Annals of Human Genetics Scholarship in Human Population Genetics. A.S. is supported by a Medical Research Council studentship. S.K.T. is supported by a Natural Environment Research Council (London) studentship. This work was supported by Medical Research Council Cooperative Group Grant G0400126.

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Kupfer, A. & Preisig, R. (1984) Eur. J. Clin. Pharmacol. 26, 753–759.  [DOI](https://doi.org/10.1007/BF00541938) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/6489416/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur.%20J.%20Clin.%20Pharmacol.&volume=26&publication_year=1984&pages=753&pmid=6489416&doi=10.1007/BF00541938&)

2. Ceulemans, B. P., Claes, L. R. & Lagae, L. G. (2004) Pediatr. Neurol. 30, 236–243.  [DOI](https://doi.org/10.1016/j.pediatrneurol.2003.10.012) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15087100/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pediatr.%20Neurol.&volume=30&publication_year=2004&pages=236&pmid=15087100&doi=10.1016/j.pediatrneurol.2003.10.012&)

3. Hoffmeyer, S., Burk, O., von Richter, O., Arnold, H. P., Brockmoller, J., Johne, A., Cascorbi, I., Gerloff, T., Roots, I., Eichelbaum, M., et al. (2000) Proc. Natl. Acad. Sci. USA 97, 3473–3478.  [DOI](https://doi.org/10.1073/pnas.050585397) | [PMC free article](/articles/PMC16264/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10716719/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&volume=97&publication_year=2000&pages=3473&pmid=10716719&doi=10.1073/pnas.050585397&)

4. Soranzo, N., Cavalleri, G. L., Weale, M. E., Wood, N. W., Depondt, C., Marguerie, R., Sisodiya, S. M. & Goldstein, D. B. (2004) Genome Res. 14, 1333–1344.  [DOI](https://doi.org/10.1101/gr.1965304) | [PMC free article](/articles/PMC442149/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15197162/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Res.&volume=14&publication_year=2004&pages=1333&pmid=15197162&doi=10.1101/gr.1965304&)

5. Siddiqui, A., Kerb, R., Weale, M. E., Brinkmann, U., Smith, A., Goldstein, D. B., Wood, N. W. & Sisodiya, S. M. (2003) N. Engl. J. Med. 348, 1442–1448.  [DOI](https://doi.org/10.1056/NEJMoa021986) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12686700/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N.%20Engl.%20J.%20Med.&volume=348&publication_year=2003&pages=1442&pmid=12686700&doi=10.1056/NEJMoa021986&)

6. Kerb, R., Aynacioglu, A. S., Brockmoller, J., Schlagenhaufer, R., Bauer, S., Szekeres, T., Hamwi, A., Fritzer-Szekeres, M., Baumgartner, C., Ongen, H. Z., et al. (2001) Pharmacogenomics J. 1, 204–210.  [DOI](https://doi.org/10.1038/sj.tpj.6500025) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11908757/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J.&volume=1&publication_year=2001&pages=204&pmid=11908757&doi=10.1038/sj.tpj.6500025&)

7. Wilkinson, G. R. (1996) J. Pharmacokinet. Biopharm. 24, 475–490.  [DOI](https://doi.org/10.1007/BF02353475) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9131486/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Pharmacokinet.%20Biopharm.&volume=24&publication_year=1996&pages=475&pmid=9131486&doi=10.1007/BF02353475&)

8. Ozdemir, V., Kalowa, W., Tang, B. K., Paterson, A. D., Walker, S. E., Endrenyi, L. & Kashuba, A. D. (2000) Pharmacogenetics 10, 373–388.  [DOI](https://doi.org/10.1097/00008571-200007000-00001) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10898107/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&volume=10&publication_year=2000&pages=373&pmid=10898107&doi=10.1097/00008571-200007000-00001&)

9. Hsieh, K. P., Lin, Y. Y., Cheng, C. L., Lai, M. L., Lin, M. S., Siest, J. P. & Huang, J. D. (2001) Drug Metab. Dispos. 29, 268–273.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/11181494/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab.%20Dispos.&volume=29&publication_year=2001&pages=268&pmid=11181494&)

10. Eiselt, R., Domanski, T. L., Zibat, A., Mueller, R., Presecan-Siedel, E., Hustert, E., Zanger, U. M., Brockmoller, J., Klenk, H. P., Meyer, U. A., et al. (2001) Pharmacogenetics 11, 447–458.  [DOI](https://doi.org/10.1097/00008571-200107000-00008) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11470997/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&volume=11&publication_year=2001&pages=447&pmid=11470997&doi=10.1097/00008571-200107000-00008&)

11. Lamba, J. K., Lin, Y. S., Schuetz, E. G. & Thummel, K. E. (2002) Adv. Drug Deliv. Rev. 54, 1271–1294.  [DOI](https://doi.org/10.1016/s0169-409x(02)00066-2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12406645/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Adv.%20Drug%20Deliv.%20Rev.&volume=54&publication_year=2002&pages=1271&pmid=12406645&doi=10.1016/s0169-409x(02)00066-2&)

12. Ball, S. E., Scatina, J., Kao, J., Ferron, G. M., Fruncillo, R., Mayer, P., Weinryb, I., Guida, M., Hopkins, P. J., Warner, N., et al. (1999) Clin. Pharmacol. Ther. 66, 288–294.  [DOI](https://doi.org/10.1016/S0009-9236(99)70037-8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10511065/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&volume=66&publication_year=1999&pages=288&pmid=10511065&doi=10.1016/S0009-9236(99)70037-8&)

13. Potschka, H., Fedrowitz, M. & Loscher, W. (2001) Neuroreport 12, 3557–3560.  [DOI](https://doi.org/10.1097/00001756-200111160-00037) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11733711/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neuroreport&volume=12&publication_year=2001&pages=3557&pmid=11733711&doi=10.1097/00001756-200111160-00037&)

14. Weale, M. E., Depondt, C., Macdonald, S. J., Smith, A., Lai, P. S., Shorvon, S. D., Wood, N. W. & Goldstein, D. B. (2003) Am. J. Hum. Genet. 73, 551–565.  [DOI](https://doi.org/10.1086/378098) | [PMC free article](/articles/PMC1180680/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12900796/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Hum.%20Genet.&volume=73&publication_year=2003&pages=551&pmid=12900796&doi=10.1086/378098&)

15. Andrew, T., Hart, D. J., Snieder, H., de Lange, M., Spector, T. D. & MacGregor, A. J. (2001) Twin Res. 4, 464–477.  [DOI](https://doi.org/10.1375/1369052012803) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11780939/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Twin%20Res.&volume=4&publication_year=2001&pages=464&pmid=11780939&doi=10.1375/1369052012803&)

16. Schaid, D. J., Rowland, C. M., Tines, D. E., Jacobson, R. M. & Poland, G. A. (2002) Am. J. Hum. Genet. 70, 425–434.  [DOI](https://doi.org/10.1086/338688) | [PMC free article](/articles/PMC384917/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11791212/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Hum.%20Genet.&volume=70&publication_year=2002&pages=425&pmid=11791212&doi=10.1086/338688&)

17. Copley, R. R. (2004) Trends Genet. 20, 171–176.  [DOI](https://doi.org/10.1016/j.tig.2004.02.001) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15101391/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Trends%20Genet.&volume=20&publication_year=2004&pages=171&pmid=15101391&doi=10.1016/j.tig.2004.02.001&)

18. Tan, J., Liu, Z., Nomura, Y., Goldin, A. L. & Dong, K. (2002) J. Neurosci. 22, 5300–5309.  [DOI](https://doi.org/10.1523/JNEUROSCI.22-13-05300.2002) | [PMC free article](/articles/PMC3062512/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12097481/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Neurosci.&volume=22&publication_year=2002&pages=5300&pmid=12097481&doi=10.1523/JNEUROSCI.22-13-05300.2002&)

19. Gastaldi, M., Bartolomei, F., Massacrier, A., Planells, R., Robaglia-Schlupp, A. & Cau, P. (1997) Brain Res. Mol. Brain Res. 44, 179–190.  [DOI](https://doi.org/10.1016/s0169-328x(96)00199-4) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9073159/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Brain%20Res.%20Mol.%20Brain%20Res.&volume=44&publication_year=1997&pages=179&pmid=9073159&doi=10.1016/s0169-328x(96)00199-4&)

20. Freedman, M. L., Reich, D., Penney, K. L., McDonald, G. J., Mignault, A. A., Patterson, N., Gabriel, S. B., Topol, E. J., Smoller, J. W., Pato, C. N., et al. (2004) Nat. Genet. 36, 388–393.  [DOI](https://doi.org/10.1038/ng1333) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15052270/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet.&volume=36&publication_year=2004&pages=388&pmid=15052270&doi=10.1038/ng1333&)

21. Reich, D. E. & Goldstein, D. B. (2001) Genet. Epidemiol. 20, 4–16.  [DOI](https://doi.org/10.1002/1098-2272(200101)20:1<4::AID-GEPI2>3.0.CO;2-T) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11119293/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genet.%20Epidemiol.&volume=20&publication_year=2001&pages=4&pmid=11119293&doi=10.1002/1098-2272(200101)20:1<4::AID-GEPI2>3.0.CO;2-T&)

22. Goldstein, D. B., Tate, S. K. & Sisodiya, S. M. (2003) Nat. Rev. Genet. 4, 937–947.  [DOI](https://doi.org/10.1038/nrg1229) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14631354/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Genet.&volume=4&publication_year=2003&pages=937&pmid=14631354&doi=10.1038/nrg1229&)

23. Plummer, N. W. & Meisler, M. H. (1999) Genomics 57, 323–331.  [DOI](https://doi.org/10.1006/geno.1998.5735) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10198179/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genomics&volume=57&publication_year=1999&pages=323&pmid=10198179&doi=10.1006/geno.1998.5735&)

24. Shorvon, S. D., Fish, D. R., Perucca, E., Dodson, W. E. & Avanzini, G. (2004) The Treatment of Epilepsy (Blackwell, Oxford).

25. Ingelman-Sundberg, M. (2004) Trends Pharmacol. Sci. 25, 193–200.  [DOI](https://doi.org/10.1016/j.tips.2004.02.007) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15063083/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Trends%20Pharmacol.%20Sci.&volume=25&publication_year=2004&pages=193&pmid=15063083&doi=10.1016/j.tips.2004.02.007&)

26. Rettie, A. E., Haining, R. L., Bajpai, M. & Levy, R. H. (1999) Epilepsy Res. 35, 253–255.  [DOI](https://doi.org/10.1016/s0920-1211(99)00017-0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10413320/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Epilepsy%20Res.&volume=35&publication_year=1999&pages=253&pmid=10413320&doi=10.1016/s0920-1211(99)00017-0&)

27. Aynacioglu, A. S., Brockmoller, J., Bauer, S., Sachse, C., Guzelbey, P., Ongen, Z., Nacak, M. & Roots, I. (1999) Br. J. Clin. Pharmacol. 48, 409–415.  [DOI](https://doi.org/10.1046/j.1365-2125.1999.00012.x) | [PMC free article](/articles/PMC2014334/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10510154/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br.%20J.%20Clin.%20Pharmacol.&volume=48&publication_year=1999&pages=409&pmid=10510154&doi=10.1046/j.1365-2125.1999.00012.x&)

28. Gotoh, O. (1992) J. Biol. Chem. 267, 83–90.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/1730627/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem.&volume=267&publication_year=1992&pages=83&pmid=1730627&)

29. Takanashi, K., Tainaka, H., Kobayashi, K., Yasumori, T., Hosakawa, M. & Chiba, K. (2000) Pharmacogenetics 10, 95–104.  [DOI](https://doi.org/10.1097/00008571-200003000-00001) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10761997/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&volume=10&publication_year=2000&pages=95&pmid=10761997&doi=10.1097/00008571-200003000-00001&)

30. Odani, A., Hashimoto, Y., Otsuki, Y., Uwai, Y., Hattori, H., Furusho, K. & Inui, K. (1997) Clin. Pharmacol. Ther. 62, 287–292.  [DOI](https://doi.org/10.1016/S0009-9236(97)90031-X) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9333104/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&volume=62&publication_year=1997&pages=287&pmid=9333104&doi=10.1016/S0009-9236(97)90031-X&)

31. Mamiya, K., Ieiri, I., Shimamoto, J., Yukawa, E., Imai, J., Ninomiya, H., Yamada, H., Otsubo, K., Higuchi, S. & Tashiro, N. (1998) Epilepsia 39, 1317–1323.  [DOI](https://doi.org/10.1111/j.1528-1157.1998.tb01330.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9860067/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Epilepsia&volume=39&publication_year=1998&pages=1317&pmid=9860067&doi=10.1111/j.1528-1157.1998.tb01330.x&)

32. Lee, C. R., Goldstein, J. A. & Pieper, J. A. (2002) Pharmacogenetics 12, 251–263.  [DOI](https://doi.org/10.1097/00008571-200204000-00010) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11927841/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&volume=12&publication_year=2002&pages=251&pmid=11927841&doi=10.1097/00008571-200204000-00010&)
