# Genome-wide common and rare variant analysis provides novel insights into clozapine-associated neutropenia

## Metadata
**Authors:** S E Legge, M L Hamshere, S Ripke, A F Pardinas, J I Goldstein, E Rees, A L Richards, G Leonenko, L F Jorskog, Clozapine-Induced Agranulocytosis Consortium, K D Chambert, D A Collier, G Genovese, I Giegling, P Holmans, A Jonasdottir, G Kirov, S A McCarroll, J H MacCabe, K Mantripragada, J L Moran, B M Neale, H Stefansson, D Rujescu, M J Daly, P F Sullivan, M J Owen, M C O'Donovan, J T R Walters
**Journal:** Molecular Psychiatry
**Date:** 2016 Jul 12
**DOI:** [10.1038/mp.2016.97](https://doi.org/10.1038/mp.2016.97)
**PMID:** 27400856
**PMCID:** PMC5065090
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5065090/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC5065090/pdf/mp201697a.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC5065090/pdf/mp201697a.pdf)

## Abstract

The antipsychotic clozapine is uniquely effective in the management of schizophrenia; however, its use is limited by its potential to induce agranulocytosis. The causes of this, and of its precursor neutropenia, are largely unknown, although genetic factors have an important role. We sought risk alleles for clozapine-associated neutropenia in a sample of 66 cases and 5583 clozapine-treated controls, through a genome-wide association study (GWAS), imputed human leukocyte antigen (HLA) alleles, exome array and copy-number variation (CNV) analyses. We then combined associated variants in a meta-analysis with data from the Clozapine-Induced Agranulocytosis Consortium (up to 163 cases and 7970 controls). In the largest combined sample to date, we identified a novel association with rs149104283 (odds ratio (OR)=4.32, P=1.79 × 10−8), intronic to transcripts of SLCO1B3 and SLCO1B7, members of a family of hepatic transporter genes previously implicated in adverse drug reactions including simvastatin-induced myopathy and docetaxel-induced neutropenia. Exome array analysis identified gene-wide associations of uncommon non-synonymous variants within UBAP2 and STARD9. We additionally provide independent replication of a previously identified variant in HLA-DQB1 (OR=15.6, P=0.015, positive predictive value=35.1%). These results implicate biological pathways through which clozapine may act to cause this serious adverse effect.

## Introduction

Clozapine is the only licensed medication for treatment-resistant schizophrenia, defined as a failure to respond to at least two antipsychotic trials of sufficient dose and duration. Although it is the only treatment with proven efficacy in this severely impaired group of patients,^1, 2^ it is substantially under-prescribed^3^ due, at least in part, to the risk of haematological side effects of agranulocytosis and neutropenia (that is, reductions of neutrophils to levels below 500 or 1500 mm^−^^3^, respectively). The cumulative risk of agranulocytosis in those taking clozapine is 0.8% and for neutropenia is 2.9%.^4^ If undetected, compromised immune function secondary to agranulocytosis can be fatal, as happened in a series of patients when the drug was introduced in the 1970s, leading to its widespread withdrawal. Evidence of its marked effectiveness over other antipsychotics led to Federal Drug Agency (USA) approval in 1989 with stipulations about the need for regular blood monitoring to aid early detection of blood abnormalities. The requirement for blood monitoring limits the acceptability of the drug to patients, and poses an obstacle to its use in clinical practice.^5^

The aetiology of clozapine-induced blood disorders is currently unknown, although genetic causes contribute. The first genome-wide association study (GWAS) conducted by the Clozapine-Induced Agranulocytosis Consortium (CIAC) has provided substantial evidence for the role of *HLA-DQB1* and *HLA-B* in clozapine-associated neutropenia.^6^ In this study we report analyses incorporating GWAS, human leukocyte antigen (HLA) allele imputation, exome array and copy-number variation (CNV) to examine genetic associations with clozapine-associated neutropenia. Associated variants were combined in a joint meta-analysis with data from the CIAC study,^6^ giving the largest combined study sample of its kind to date.

## Materials and methods

### Sample description

Study individuals were from CLOZUK (*n*=5493) and CardiffCOGS (Cognition in Schizophrenia, *n*=156) samples. All had clinical or research diagnoses of schizophrenia.^7^ CLOZUK comprises individuals who were prescribed clozapine in the United Kingdom and have a clinical diagnosis of treatment-resistant schizophrenia.^7, 8^ The CLOZUK samples were acquired anonymously by the research team, in accordance with ethics permissions and the UK Human Tissue Act, in collaboration with Novartis, one of the UK suppliers of clozapine. Twelve months after sample acquisition, the research team was informed of those who had developed neutropenia while taking clozapine and, where available, the recorded lowest neutrophil counts of these individuals were supplied. CardiffCOGS is a schizophrenia sample recruited from secondary mental health services in South Wales, UK; for detailed sample description see.^8, 9^ As part of a comprehensive clinical interview, individuals were asked about lifetime clozapine use and occurrence of neutropenia. Clinical case notes were used to confirm neutropenia status, and lowest recorded neutrophil levels were collected.

Clozapine-associated neutropenia cases (*n*=66) developed an absolute neutrophil count (ANC) ⩽1500 mm^−^^3^ during treatment with clozapine. Following the approach of recent studies,^6, 10^ we assessed cases with neutropenia because the success of the monitoring system and preemptive drug withdrawal in the United Kingdom has made agranulocytosis extremely rare. This neutrophil count threshold is used in the United Kingdom as a trigger to discontinue clozapine. Controls (*n*=5583) had received clozapine for a minimum of a year without developing an ANC <2000 mm^−^^3^. Those who had a test result (1500 mm^−^^3^<ANC<2000 mm^−^^3^) were excluded from all analyses (*n*=20). No differences in age or sex were observed between clozapine-associated neutropenia cases and controls ([Supplementary Table 1](#sup1)). All individuals were of European ancestry, as determined by self-report and principal component analysis (PCA) of GWAS data.

### Genotyping

Genotyping was performed at the Broad Institute, Cambridge, MA, USA. CardiffCOGS and part of the CLOZUK sample (40 cases and 3573 controls) were genotyped on Illumina HumanOmniExpressExome-8v1 and the remainder of the CLOZUK sample (26 cases and 2098 controls) were genotyped on both Illumina HumanOmniExpress-12v1 and Illumina HumanExome BeadChip (San Diego, CA, USA).

### Genome-wide association study

Quality-control procedures and imputation were conducted using the Psychiatric Genomics Consortium pipeline.^7^ Imputation was performed using IMPUTE2 (ref. [11](#bib11)) and a reference panel from the full 1000 Genomes Project data set (freeze date August 2012, see [Supplementary Methods](#sup1)). Principal component estimation was conducted using EIGENSTRAT to exclude outliers and assess population stratification^12^ ([Supplementary Figures 1 and 2](#sup1)). We included genotyping array as well as the first three principal components as covariates to account for population structure. Single-nucleotide polymorphisms (SNPs) with allele frequencies that differed between genotyping arrays at *P*<1 × 10^−5^ were excluded ([Supplementary Figure 3](#sup1)). We selected common SNPs for analysis with high imputation quality (imputation INFO score≥0.8, minor allele frequency (MAF) ≥0.01 in cases and controls). Association analysis was performed using logistic regression in PLINK^13^ and SNPs functionally annotated using the Scripps genome advisor.^14^ PLINK^13^ was used to identify index SNPs in relative linkage equilibrium. Further details of quality-control procedures and statistical analyses are provided in [Supplementary Methods](#sup1).

### HLA analysis

Classical *HLA* alleles and amino-acid polymorphisms were imputed using SNP2HLA (version 1.02)^15^ using BEAGLE (version 3.0.4)^16^ from genotyped common variants using a reference data set of 5225 individuals from Type 1 Diabetes Genetics Consortium (T1DGC). We used the same procedures for SNP selection, analysis and covariate selection (three PCAs derived from GWAS, plus genotyping array) as described for the GWAS analysis above. Owing to complex and extended linkage disequilibrium (LD) in the major histocompatibility complex, we did not identify index SNPs in relative linkage equilibrium.

We additionally genotyped a candidate SNP, *HLA-DQB1* 6672G>C (ref. [17](#bib17); rs113332494) in 60 cases and 305 age- and sex-matched controls. This SNP was genotyped separately as it was not imputed with sufficient quality to be reported in the GWAS or HLA analyses, and was a strong candidate variant.^17^ Genotyping was conducted at deCODE genetics using the Centaurus (Nanogen) platform.^18^ Association with clozapine-associated neutropenia was tested using Fisher’s exact test, given low minor allele counts; however, to ensure there was no effect of population stratification, we also conducted a logistic regression including three PCAs derived from GWAS with 5 × 10^8^ permutations to generate empirical *P*-values.

### Exome array analysis

The Illumina exome array is designed to genotype uncommon-to-rare coding variants previously observed in whole-exome sequencing studies. Exome array data were available for 57 cases and 4958 exposed controls. Full details of the quality-control procedures are provided in an open access publication^19^ and [Supplementary Methods](#sup1). PCA was conducted using EIGENSTRAT,^12^ with 14 743 common exome array variants in relative linkage equilibrium (MAF⩾0.05, *r*^2^<0.2) to assess population structure and identify outliers ([Supplementary Figure 4](#sup1)). Owing to the relatively small case sample size in our study we did not apply a frequency filter to variants in this analysis. Single variant association was conducted using logistic regression in PLINK with the first 10 principal components included as covariates. Adaptive permutations (between 10 and 1 × 10^9^) were used to generate empirical *P*-values in logistic regression analyses. PLINK^13^ was used to identify index SNPs in relative linkage equilibrium. To test for the effects of multiple functional variants in genes, we used SKAT-O (ref. [20](#bib20)) with 2 × 10^6^ permutations, including the first 10 principal components, for genes with at least two uncommon (MAF<0.05), non-synonymous (missense, stop or splice) variants.

### Copy-number variation

The identification and quality control of CNV for this sample has been previously described^8^ and is detailed in [Supplementary Methods](#sup1). CNVs were included if they had a frequency ⩽0.01, contained ⩾10 probes and were ⩾100 kb in length. Samples that passed both CNV and GWAS quality control (63 cases and 5456 controls) were used to test genes for enrichment of exon disrupting CNVs using a two-sided Fisher’s exact test. Deletions and duplications were analysed separately.

### Secondary analysis of clozapine-associated neutropenia below ⩽1000 mm−3

We conducted secondary analyses on a subset of the more severely affected cases with ANC ⩽1000 mm^−^^3^ (*n*=18). A total of four of these cases had developed agranulocytosis (ANC ⩽500 mm^−3^). We assessed the association of single variants with clozapine-associated neutropenia below ⩽1000 mm^−^^3^ in GWAS (*N*=18), exome array (*N*=16) and HLA imputation (*N*=18) analyses. All analyses conducted were consistent with methods used for clozapine-associated neutropenia, described above.

### Replication sample and meta-analysis

We obtained summary statistics for associated SNPs from a recently published study by the CIAC.^6^ In this study, Goldstein *et al.*^6^ conducted a comprehensive genetic association study in 163 clozapine-induced neutropenia cases (98 with ANC <500 mm^−3^, 61 with 500⩽ANC⩽1000 mm^−3^ and 4 with ANC ⩽1500 mm^−3^). The CIAC study included the following: GWAS (161 cases with clozapine-induced neutropenia, 249 clozapine-exposed controls without neutropenia and 947 unexposed controls), exome array analysis (148 cases and up to 7970 unexposed controls) and classical *HLA* allele imputation (162 cases and 4319 unexposed controls). These data sets formed the replication samples for the current study and were combined with results for both (i) clozapine-associated neutropenia and (ii) neutropenia ⩽1000 analyses. SNPs that were associated with clozapine-associated neutropenia at *P*<1 × 10^−4^ from our GWAS, or *P*<0.05 from our *HLA* variant analysis, were combined with the replication data in fixed-effects meta-analyses using PLINK to estimate a combined odds ratio (OR) weighted by the study’s inverse standard error (s.e.). If an index SNP was not present in the replication data, a proxy SNP in strong LD (*r*^2^⩾0.8) was substituted and the s.e. weighted (s.e.w.) to account for the lack of information: s.e.w.=s.e./sqrt(*r*^2^).^21^ The variants that were associated with clozapine-associated neutropenia from exome array analyses with *P*<0.01 were combined with the replication data in a *P*-value-based method in METAL,^22^ weighted by the square root of the total sample size. We used different *P*-value replication thresholds for the *HLA* and exome array to arrive at approximately the same number of variants.

## Results

[Figure 1](#fig1) provides a summary of the study design and key results from each analysis.

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c9e1/5622128/9bddec6ee1f0/mp201697f1.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5622128_mp201697f1.jpg)

Study design and key results. To investigate the association of genetic variants with clozapine-associated neutropenia, we conducted (a) a genome-wide association study (GWAS), (b) human leukocyte antigen (HLA) allele imputation and genotyped a candidate variant of interest, HLA-DQB1 6672G>C/ rs113332494, (c) exome array single variant and gene-based analysis and (d) copy-number variation (CNV) analysis. We then took forward the associated variants from GWAS, HLA and exome array analyses to a combined meta-analysis with the Clozapine-Induced Agranulocytosis Consortium (CIAC) study.

### Genome-wide association study

We performed a GWAS of 7 559 010 genotyped and imputed common SNPs (QQ plot in [Supplementary Figure 5](#sup1), *λ*_GC_=0.95). Two SNPs were associated with clozapine-associated neutropenia at the genome-wide significant level of *P*<5 × 10^−8^ ([Supplementary Figure 6](#sup1) and [Supplementary Data 1](#sup1)); rs80208670 on chromosome 13 (OR=8.76, 95% confidence interval (CI): 4.21–18.25, *P*=6.51 × 10^−9^) and rs77897117 on chromosome 1 (OR=4.02, 95% CI: 2.46–6.57, *P*=4.60 × 10^−8^). Our sample size had 80% power to detect an OR>4 for alleles with MAF>0.10 at *P*<5 × 10^−8^ ([Supplementary Figure 7](#sup1)). The genome-wide significant SNPs from the discovery CLOZUK GWAS, rs80208670 and rs77897117, were not significantly associated in CIAC (OR=1.69, *P*=0.27 and OR=0.67, *P*=0.28, respectively).

In total, there were 266 independent (*r*^2^<0.1) SNPs associated with clozapine-associated neutropenia at *P*<1 × 10^−4^ and we sought replication of these SNPs in the CIAC sample (257 of these SNPs were available or had an appropriate proxy, [Supplementary Data 1](#sup1)). [Table 1](#tbl1) lists the 10 most strongly associated SNPs from the meta-analysis. One SNP on chromosome 12 surpassed the GWS threshold for association with clozapine-associated neutropenia (OR=4.32, *P*=1.79 × 10^−8^). rs149104283 is intronic to transcripts of *SLCO1B3* and *SLCO1B7* (solute carrier organic anion transporter family, member 1B3 and member 1B7) and was present in 7.37% of cases versus 1.52% of controls in our sample and 4.20% of cases versus 1.67% of controls in the CIAC sample. For consideration of how this may be translated to risk allele carrier status see [Supplementary Results](#sup1).

### Table 1. GWAS meta-analysis top 10 SNPs.

| CHR | SNP | Position | A1 | CLOZUK |  |  | CIAC |  |  | Meta-analysis |  | Nearest gene | Location |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
|   |   |   |   | P-value | OR | INFO | P-value | OR | INFO | P-value | OR |   |   |
| 12 | rs149104283 | 21 083 862 | T | 4.98 × 10−7 | 6.20 | 0.87 | 3.61 × 10−3 | 2.95 | 0.87 | 1.79 × 10−8 | 4.32 | SLCO1B3, SLCO1B7 | Intronic, intronic |
| 13 | rs80208670 | 84 438 088 | C | 6.51 × 10−9 | 8.76 | 0.83 | 0.2728 | 1.69 | 0.83 | 1.56 × 10−7 | 4.69 | SLITRK1 | 13 kb Downstream |
| 1 | rs184597564 | 82 236 406 | A | 2.48 × 10−5 | 4.03 | 0.89 | 4.99 × 10−3 | 2.40 | 0.94 | 8.01 × 10−7 | 3.06 | ADGRL2 | Intronic |
| 7 | rs78900159 | 76 968 378 | A | 6.94 × 10−6 | 4.01 | 0.99 | 0.0247 | 1.97 | 0.98 | 2.02 × 10−6 | 2.79 | GSAP | Intronic |
| 10 | rs16916041 | 63 146 547 | T | 7.40 × 10−6 | 2.57 | 1.02 | 0.0205 | 1.58 | 0.96 | 2.05 × 10−6 | 1.98 | TMEM26 | 20 kb Downstream |
| 16 | rs11649311 | 25 226 020 | T | 9.70 × 10−5 | 2.07 | 0.89 | 2.62 × 10−3 | 1.51 | 0.89 | 2.26 × 10−6 | 1.68 | AQP8 | 2 kb Upstream |
| 17 | rs117202297 | 53 769 035 | T | 8.48 × 10−6 | 6.25 | 0.82 | 0.1503 | 2.66 | 0.66 | 5.27 × 10−6 | 4.97 | TMEM100 | 28 kb Downstream |
| 17 | rs80282661 | 13 252 073 | T | 1.99 × 10−6 | 6.15 | 0.90 | 0.2632 | 1.85 | 0.89 | 5.49 × 10−6 | 4.17 | HS3ST3A1 | 147 kb Downstream |
| 1 | rs185053659 | 60 704 250 | A | 7.76 × 10−6 | 5.80 | 0.86 | 0.1012 | 2.13 | 0.92 | 8.05 × 10−6 | 3.80 | C1orf87 | 165 kb Upstream |
| 14 | rs78074145 | 40 404 458 | C | 1.13 × 10−6 | 4.05 | 0.96 | 0.2716 | 1.44 | 0.99 | 1.08 × 10−5 | 2.60 | FXB033 | 503 kb Upstream |

Table 1 Caption: Abbreviations: A1, minor reference allele; CHR, chromosome; CIAC, Clozapine-Induced Agranulocytosis Consortium; GWAS, genome-wide association study; OR, odds ratio; SNP, single-nucleotide polymorphism. Results are ordered by meta-analysis P-value. Further details, including minor allele frequencies, are available in Supplementary Data 1.

### HLA analysis

No imputed classical *HLA* allele or amino-acid polymorphism was associated with clozapine-associated neutropenia at the genome-wide significant level (*P*<5 × 10^−8^) in either the discovery analysis (SNPs=7751) or combined meta-analysis (SNPs=102; [Supplementary Data 2](#sup1)). It was not possible to impute the amino-acid polymorphisms *HLA-DQB1* (126Q) and *HLA-B* (158T) implicated in the CIAC study^6^ with sufficient quality (INFO>0.8) in the discovery sample.

We additionally genotyped a previously associated variant, *HLA-DQB1* 6672G>C ^17^ (rs113332494), in 60 cases and 305 age- and sex-matched controls as it was not imputed with sufficient quality. We found independent support for association of *HLA-DQB1* 6672G>C (OR=15.6, 95% CI: 1.6–151.4, *P*=0.015), replicating previous reports of association with clozapine-induced agranulocytosis.^17^ The association strengthened when considering only those with ANC below ⩽1000 mm^−^^3^ (OR=38.1, 95% CI: 3.4–430.9, *P*=0.0079). For the associated ‘G’ allele, we found three heterozygote carriers among 60 cases, and a single heterozygote in 305 controls. Lowest neutrophil counts were available for two of the three ‘G’ case carriers, both of whom had a neutrophil level <1000 mm^−^^3^ (700 and 900). The association remained after adjusting for GWAS PCAs. For further consideration of possible population-specific effects at this locus please refer to [Supplementary Results](#sup1).

### Exome array analysis

In exome array analyses, no single variant exceeded a significance threshold of *P*<4.3 × 10^−7^, corresponding to a Bonferroni correction for 115 000 variants tested, in either the discovery or combined meta-analyses ([Supplementary Data 3](#sup1), QQ plot given in [Supplementary Figure 8](#sup1), *λ*_GC_=1.11). [Table 2](#tbl2) lists the 10 most strongly associated exome variants from the meta-analysis. Of interest is rs1546308 (*P*=1.10 × 10^−6^), a missense variant in *SLCO1B7* and intronic to *SLCO1B3,* that is 92 kb from the SNP that emerged as the only genome-wide significant variant from GWAS meta-analysis. rs1546308 is predicted to be benign and was present in 15.8% of cases versus 5.8% of controls in our sample and 9.3% of cases versus 5.6% of controls in the CIAC sample.

### Table 2. Exome array meta-analysis top 10 variants.

| CHR | Variant | rs ID | Position | A1 | CLOZUK P-value | CIAC P-value | Meta-analysis P-value | Gene | Location | Function |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| 4 | exm387558 | rs201099591 | 7 436 363 | A | 2.48 × 10−3 | 1.31 × 10−4 | 1.10 × 10−6 | PSAPL1, SORCS2 | Exonic, intronic | Missensea |
| 7 | exm622984 | rs17139320 | 63 726 370 | G | 3.96 × 10−3 | 2.87 × 10−4 | 3.70 × 10−6 | ZNF679 | Exonic | Missensea |
| 8 | exm729894 | rs201071539 | 145 003 862 | A | 2.19 × 10−3 | 1.69 × 10−3 | 1.19 × 10−5 | PLEC | Exonic | Missensea |
| 19 | exm1421170 | rs2591594 | 9 076 728 | A | 4.66 × 10−3 | 3.26 × 10−3 | 4.52 × 10−5 | MUC16 | Exonic | Missenseb |
| 12 | exm988839 | rs1546308 | 21 176 135 | C | 1.25 × 10−4 | 0.0679 | 9.13 × 10−5 | SLCO1B7, SLCO1B3 | Exonic, intronic | Missensea |
| 1 | exm23767 | rs12073549 | 17 720 545 | T | 1.60 × 10−3 | 0.0163 | 9.31 × 10−5 | PADI6 | Exonic | Synonymous |
| 15 | exm1186658 | rs117116488 | 89 390 513 | T | 2.76 × 10−4 | 0.0448 | 9.73 × 10−5 | ACAN | Exonic | Missenseb |
| 12 | exm981950 | rs79149293 | 8 975 873 | G | 5.66 × 10−5 | 0.0967 | 1.01 × 10−4 | A2ML1 | Exonic | Missensec |
| 12 | exm995289 | rs138912646 | 42 711 606 | A | 3.94 × 10−5 | 0.1613 | 1.79 × 10−4 | ZCRB1, PPHLN1 | Exonic, intronic | Missensea |
| 12 | exm1051374 | rs143584336 | 130 921 539 | A | 1.22 × 10−4 | 0.1144 | 2.12 × 10−4 | RIMBP2 | Exonic | Missensea |

Table 2 Caption: Abbreviations: A1, minor reference allele; CHR, chromosome; CIAC, Clozapine-Induced Agranulocytosis Consortium. Results are ordered by meta-analysis P-value. Predicted function of non-synonymous variants. aBenign. bPossibly damaging. cProbably damaging. Further details, including minor allele frequencies, are available in Supplementary Data 3.

[Supplementary Table 3](#sup1) displays the 10 most significantly associated genes (including the total allele count and number of variants contributing to each gene) based on the SKAT-O analysis of genes with at least two uncommon (MAF<0.05), non-synonymous variants from the exome array. There was evidence of association for two genes that exceeded a threshold of *P*<2.5 × 10^−6^ (corresponding to a Bonferroni correction of 20 000 genes tested;^23^ QQ plot in [Supplementary Figure 9](#sup1)): *UBAP2* on chromosome 9 (*P*=1.02 × 10^−7^) and *STARD9* on chromosome 15 (*P*=2.85 × 10^−7^). Owing to differing analytical methods used by CIAC, it was not possible to combine our gene-based results in a joint analysis (see [Supplementary Methods](#sup1) for detailed explanation).

### Copy-number variation

In a genome-wide analysis, no individual gene was significantly enriched for large, rare exonic CNVs that exceeded a significance threshold of *P*<2.5 × 10^−6^ ([Supplementary Data 4](#sup1)).

### Associations with rs1546308 and rs149104283 are not independent

We investigated the independence of rs1546308 (missense variant in *SLCO1B7* and intronic in *SLCO1B3* from exome chip analysis) and rs149104283 (GWAS intronic variant in *SLCO1B3* and *SLCO1B7*) in samples with data available for both variants (55 cases and 4834 controls). The LD between the two variants in the sample was *r*^2^=0.15, *D*’=0.84. In a conditional logistic regression, the strength of the association of rs1546308 with clozapine-associated neutropenia was attenuated from OR=3.00 (95% CI: 1.735–5.189, *P*=8.40 × 10^−5^) to OR=2.16 (95% CI: 1.093–4.251, *P*=0.027) after adjusting for rs149104283. Haplotype analysis did not strengthen the association signal. Thus, these two findings are not independent, and the associated region spans *SLCO1B1*, *SLCO1B3* and *SLCO1B7* ([Figure 2](#fig2) and [Supplementary Figure 14](#sup1)).

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c9e1/5622128/11b3ee29c1f6/mp201697f2.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5622128_mp201697f2.jpg)

Association of 12p12.2 with clozapine-associated neutropenia. LocusZoom plot of the region associated with clozapine-associated neutropenia on chromosome 12p12.2 in CLOZUK (discovery) sample. Genes within the region are shown in the lower panel, and the unbroken blue line indicated the recombination rate within the region. Each circle represents the P-value for one SNP in the discovery sample, with the top SNP rs149104283 shown in purple and the SNPs in the region coloured depending on their degree of correlation (r2) with rs149104283 (as estimated by LocusZoom on the basis of CEU HapMap haplotypes).

### Secondary analysis of clozapine-associated neutropenia below ⩽1000 mm−3

Results of the GWAS for the more stringent neutropenia cutoff of ⩽1000 mm^−^^3^ are presented in [Supplementary Results](#sup1). The combined GWAS meta-analysis identified no loci exceeding genome-wide significance. The genome-wide significant SNP from the GWAS meta-analysis, rs149104283, had a MAF of 7.8% in cases with ANC ⩽1000 mm^−3^, and was associated at OR=7.6, *P*=0.0027 in the discovery sample. No variants exceeded relevant significance thresholds in imputed HLA and exome array discovery or combined meta-analyses (see [Supplementary Results](#sup1)).

## Discussion

We have conducted a multifaceted genetic analysis of clozapine-associated neutropenia in the largest combined sample studied to date. Using GWAS, we identify a novel association implicating a family of organic anion transporters involved in drug metabolism, which have been previously associated with adverse drug reactions. We also found evidence for effects of uncommon non-synonymous variants within *UBAP2* and *STARD9* and provide independent replication of a previously identified variant in *HLA-DQB1*.

The primary GWAS finding from the meta-analysis was a genome-wide significant association with neutropenia for rs149104283. The association effect size of this polymorphism is larger in the CLOZUK discovery sample (OR=6.2) than in the CIAC replication data set (OR=2.95), as would be expected from winner’s curse. It follows that the true effect size probably lies closer to the CIAC estimate, although this requires confirmation using independent data. rs149104283 is an intronic SNP for transcripts of both *SLCO1B3* and *SLCO1B7*—the associated region containing a third member of this organic anion transporter family*, SLCO1B1. SLCO1B7* encodes a putative protein (OAT1B7) that is poorly characterised, based on coding sequence prediction, and its functionality is unknown. *SLCO1B3* and *SLCO1B1* share sequence homology and encode liver-specific organic anion transporter polypeptides (OATP1B3 and OATP1B1) that are multipass transmembrane proteins expressed exclusively in the basolateral membrane of hepatocytes.^24^ They facilitate uptake of exogenous substances, including drugs, from the portal vein into hepatocytes, where the substance is subsequently modified either via metabolism with cytochrome (CYP) 450 enzymes or excreted.^25^

Polymorphisms in *SLCO1B1* and *SLCO1B3* have been implicated in adverse reactions with other drugs. In 2008, a GWAS identified a missense variant rs4149056 in *SLCO1B1* that increased the risk of simvastatin-induced myopathy by increasing the area under the curve for simvastatin, particularly in those taking high doses.^26^ This prominent pharmacogenetic finding has been widely replicated and has led to recommendations for its use as a routine preemptive clinical test.^27^ Particularly relevant to the current study are reports of an association between rs11045585, an intronic variant in *SLCO1B3,* and severe leukopenia/neutropenia induced by docetaxel, a chemotherapeutic agent,^28^ and that this may be secondary to alterations in the pharmacokinetics and bioavailability of the drug.^29, 30, 31^ These polymorphisms were not in high LD (*r*^2^<0.1 for both) with the index SNP in this study, although rs11045585 was weakly associated with neutropenia in our discovery sample (OR=1.62, *P*=0.03).

Together, the findings suggest the hypothesis that genetic variants at *SLCO1B3* (and/or *SLCO1B1*) increase the risk of clozapine-associated neutropenia through a pharmacokinetic mechanism. It is unclear whether clozapine plasma levels are associated with the development of neutropenia.^32, 33, 34^ One of the best-supported hypotheses to explain clozapine’s association with agranulocytosis relates to the bioactivation of clozapine, or a stable metabolite, to a chemically reactive nitrenium ion.^35^ The propensity for nitrenium ions to cause apoptosis to neutrophils, or be toxic to neutrophil precursors, is dose-dependent, lending support to the hypothesis that clozapine pharmacokinetics and bioavailability are related to its potential to cause neutropenia.^36, 37^

In analysis of exome chip data we found evidence of association with neutropenia for uncommon non-synonymous variants in *STARD9* and *UBAP2*. *STARD9* is a mitotic kinesin, and *STARD9*-depleted cancer cells have abnormal cellular morphology and undergo apoptosis.^38^ In addition, STARD9 depletion was found to synergise with the chemotherapeutic agent taxol, the use of which is dose-limited because of neutropenia.^38^ The function of *UBAP2* is undetermined, although it has an ubiquitin-associated domain and is widely expressed across tissues including bone marrow. The ubiquitination pathway has been shown to modulate the granulocyte colony-stimulating factor receptor,^39, 40^ a critical regulator of neutrophil production. A recent study reported the association of a missense variant in the ubiquitin gene *USP43* with clozapine-associated neutropenia.^10^

Our final finding adds to the growing evidence implicating *HLA-DQB1* in clozapine-associated neutropenia, supporting the recently published CIAC study.^6^ There have been further reports implicating SNPs within *HLA-DQB1*,^17, 41, 42^ although these samples and those in CIAC are overlapping; thus, we believe we provide the first fully independent replication implicating this locus in clozapine-associated neutropenia/agranulocytosis. The *HLA-DQB1* variant alone has a positive predictive value of 35.1% (see [Supplementary Table 4](#sup1)). Although this is promising, the majority of those that develop neutropenia or agranulocytosis while taking clozapine are not carriers of this risk allele, or indeed the other alleles we have identified in this study. The sensitivity for a test including rs149104283 (GWS intronic variant in *SLCO1B3* and *SLCO1B7*), rs1546308 (missense variant in *SLCO1B7*) and rs113332494 (*HLA-DQB1*) is 29.17%, the specificity 90.61%, the positive predictive value 9.94% and the negative predictive value 97.30% ([Supplementary Table 4](#sup1)). Although the variants identified in this study convey a substantially increased risk for clozapine-associated neutropenia, they are currently on their own unlikely to have clinical utility for pharmacogenetic testing,^43^ particularly as there is currently no alternative treatment for those with treatment-resistant schizophrenia.

An important consideration is that the majority of cases in our analyses had developed neutropenia rather than agranulocytosis. It is now very rare in the United Kingdom to develop agranulocytosis because of the success of the monitoring system and the fact that clozapine is stopped once neutropenia is detected; in fact, only four cases met this threshold in our sample. Despite this, we found that the major findings from our neutropenia analysis extended to the secondary analyses, which was restricted to those with an ANC⩽1000 mm^−3^, indicating that the clozapine-associated neutropenia findings are likely to be applicable to those with severe neutropenia and agranulocytosis.

Our findings provide novel insights into putative biological processes underlying clozapine-associated neutropenia. Furthermore, we have indicated a potential link between the pharmacokinetics of clozapine and risk of neutropenia/agranulocytosis with potentially important clinical implications. The development of such understanding should help widen the availability of clozapine with a beneficial impact on those with treatment-resistant schizophrenia.

## Clozapine-induced Agranulocytosis Consortium (CIAC) members

Jacqueline I Goldstein, SB; L Fredrik Jarskog, MD; Chris Hilliard, BS; Ana Alfirevic, PhD; Laramie Duncan, PhD; Denis Fourches, PhD; Hailiang Huang, PhD; Monkol Lek, PhD; Benjamin M Neale, PhD; Stephan Ripke, MD; Kevin Shianna, PhD; Jin P Szatkiewicz, PhD; Alexander Tropsha, PhD; Edwin JCG van den Oord, PhD; Ingolf Cascorbi, MD, PhD; Michael Dettling, MD, PhD; Ephraim Gazit, MD, PhD; Donald C Goff, MD; Arthur L Holden, MBA; Deanna L Kelly, PharmD, BCPP; Anil K Malhotra, MD; Jimmi Nielsen, PhD; Munir Pirmohamed, MD, PhD; Dan Rujescu, MD, PhD; Thomas Werge, MSc, PhD; Deborah L Levy, PhD; Richard C Josiassen, PhD; James L Kennedy, PhD; Jeffrey A Lieberman, MD; Mark J Daly, PhD; Patrick F Sullivan, MD, FRANZCP.

## Acknowledgments

We thank the participants, clinicians, field team and MRCCNGG lab staff for their help with the CardiffCOGS study. For the CLOZUK sample we thank Novartis for their guidance and cooperation. We also thank staff at The Doctor’s Laboratory. This project has received funding from the European Union’s Seventh Framework Programme for research, technological development and demonstration under grant agreement no. 279227. This publication reflects only the authors’ views and the European Union is not liable for any use that may be made of the information contained therein. This work was supported by a Medical Research Council (MRC) PhD studentship to SL. The work at Cardiff University was funded by Medical Research Council (MRC) Centre (ML/L010305/1) and Program Grants (G0800509). The schizophrenia samples were genotyped at the Broad Institute, USA, and funded by a philanthropic gift to the Stanley Centre for Psychiatric Research.

## Footnotes

## Contributor Information

Clozapine-Induced Agranulocytosis Consortium:

[Jacqueline I Goldstein](https://pubmed.ncbi.nlm.nih.gov/?term=%22Goldstein%20JI%22%5BAuthor%5D), [L Fredrik Jarskog](https://pubmed.ncbi.nlm.nih.gov/?term=%22Jarskog%20LF%22%5BAuthor%5D), [Chris Hilliard](https://pubmed.ncbi.nlm.nih.gov/?term=%22Hilliard%20C%22%5BAuthor%5D), [Ana Alfirevic](https://pubmed.ncbi.nlm.nih.gov/?term=%22Alfirevic%20A%22%5BAuthor%5D), [Laramie Duncan](https://pubmed.ncbi.nlm.nih.gov/?term=%22Duncan%20L%22%5BAuthor%5D), [Denis Fourches](https://pubmed.ncbi.nlm.nih.gov/?term=%22Fourches%20D%22%5BAuthor%5D), [Hailiang Huang](https://pubmed.ncbi.nlm.nih.gov/?term=%22Huang%20H%22%5BAuthor%5D), [Monkol Lek](https://pubmed.ncbi.nlm.nih.gov/?term=%22Lek%20M%22%5BAuthor%5D), [Benjamin M Neale](https://pubmed.ncbi.nlm.nih.gov/?term=%22Neale%20BM%22%5BAuthor%5D), [Stephan Ripke](https://pubmed.ncbi.nlm.nih.gov/?term=%22Ripke%20S%22%5BAuthor%5D), [Kevin Shianna](https://pubmed.ncbi.nlm.nih.gov/?term=%22Shianna%20K%22%5BAuthor%5D), [Jin P Szatkiewicz](https://pubmed.ncbi.nlm.nih.gov/?term=%22Szatkiewicz%20JP%22%5BAuthor%5D), [Alexander Tropsha](https://pubmed.ncbi.nlm.nih.gov/?term=%22Tropsha%20A%22%5BAuthor%5D), [Edwin JCG van den Oord](https://pubmed.ncbi.nlm.nih.gov/?term=%22van%20den%20Oord%20EJCG%22%5BAuthor%5D), [Ingolf Cascorbi](https://pubmed.ncbi.nlm.nih.gov/?term=%22Cascorbi%20I%22%5BAuthor%5D), [Michael Dettling](https://pubmed.ncbi.nlm.nih.gov/?term=%22Dettling%20M%22%5BAuthor%5D), [Ephraim Gazit](https://pubmed.ncbi.nlm.nih.gov/?term=%22Gazit%20E%22%5BAuthor%5D), [Donald C Goff](https://pubmed.ncbi.nlm.nih.gov/?term=%22Goff%20DC%22%5BAuthor%5D), [Arthur L Holden](https://pubmed.ncbi.nlm.nih.gov/?term=%22Holden%20AL%22%5BAuthor%5D), [Deanna L Kelly](https://pubmed.ncbi.nlm.nih.gov/?term=%22Kelly%20DL%22%5BAuthor%5D), [Anil K Malhotra](https://pubmed.ncbi.nlm.nih.gov/?term=%22Malhotra%20AK%22%5BAuthor%5D), [Jimmi Nielsen](https://pubmed.ncbi.nlm.nih.gov/?term=%22Nielsen%20J%22%5BAuthor%5D), [Munir Pirmohamed](https://pubmed.ncbi.nlm.nih.gov/?term=%22Pirmohamed%20M%22%5BAuthor%5D), [Dan Rujescu](https://pubmed.ncbi.nlm.nih.gov/?term=%22Rujescu%20D%22%5BAuthor%5D), [Thomas Werge](https://pubmed.ncbi.nlm.nih.gov/?term=%22Werge%20T%22%5BAuthor%5D), [Deborah L Levy](https://pubmed.ncbi.nlm.nih.gov/?term=%22Levy%20DL%22%5BAuthor%5D), [Richard C Josiassen](https://pubmed.ncbi.nlm.nih.gov/?term=%22Josiassen%20RC%22%5BAuthor%5D), [James L Kennedy](https://pubmed.ncbi.nlm.nih.gov/?term=%22Kennedy%20JL%22%5BAuthor%5D), [Jeffrey A Lieberman](https://pubmed.ncbi.nlm.nih.gov/?term=%22Lieberman%20JA%22%5BAuthor%5D), [Mark J Daly](https://pubmed.ncbi.nlm.nih.gov/?term=%22Daly%20MJ%22%5BAuthor%5D), and [Patrick F Sullivan](https://pubmed.ncbi.nlm.nih.gov/?term=%22Sullivan%20PF%22%5BAuthor%5D)

## Supplementary Material

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Kane J, Honigfeld G, Singer J, Meltzer H. Clozapine for the treatment-resistant schizophrenic. A double-blind comparison with chlorpromazine. Arch Gen Psychiatry 1988; 45: 789–796.  [DOI](https://doi.org/10.1001/archpsyc.1988.01800330013001) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/3046553/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arch%20Gen%20Psychiatry&title=Clozapine%20for%20the%20treatment-resistant%20schizophrenic.%20A%20double-blind%20comparison%20with%20chlorpromazine&volume=45&publication_year=1988&pages=789&pmid=3046553&doi=10.1001/archpsyc.1988.01800330013001&)

2. Leucht S, Corves C, Arbter D, Engel RR, Li C, Davis JM. Second-generation versus first-generation antipsychotic drugs for schizophrenia: a meta-analysis. Lancet 2009; 373: 31–41.  [DOI](https://doi.org/10.1016/S0140-6736(08)61764-X) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19058842/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Second-generation%20versus%20first-generation%20antipsychotic%20drugs%20for%20schizophrenia:%20a%20meta-analysis&volume=373&publication_year=2009&pages=31&pmid=19058842&doi=10.1016/S0140-6736(08)61764-X&)

3. Royal College of Psychiatrists. Report of the National Audit of Schizophrenia (NAS) 2012. Healthcare Quality Improvement Partnership, London, 2012.

4. Atkin K, Kendall F, Gould D, Freeman H, Liberman J, O'Sullivan D. Neutropenia and agranulocytosis in patients receiving clozapine in the UK and Ireland. Br J Psychiatry 1996; 169: 483–488.  [DOI](https://doi.org/10.1192/bjp.169.4.483) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8894200/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Psychiatry&title=Neutropenia%20and%20agranulocytosis%20in%20patients%20receiving%20clozapine%20in%20the%20UK%20and%20Ireland&volume=169&publication_year=1996&pages=483&pmid=8894200&doi=10.1192/bjp.169.4.483&)

5. Patel MX. Clinician hesitation prior to clozapine initiation: is it justifiable? Br J Psychiatry 2012; 201: 425–427.  [DOI](https://doi.org/10.1192/bjp.bp.112.114777) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23209086/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Psychiatry&title=Clinician%20hesitation%20prior%20to%20clozapine%20initiation:%20is%20it%20justifiable&volume=201&publication_year=2012&pages=425&pmid=23209086&doi=10.1192/bjp.bp.112.114777&)

6. Goldstein JI, Fredrik Jarskog L, Hilliard C, Alfirevic A, Duncan L, Fourches D et al. Clozapine-induced agranulocytosis is associated with rare HLA-DQB1 and HLA-B alleles. Nat Commun 2014; 5: 4757.  [DOI](https://doi.org/10.1038/ncomms5757) | [PMC free article](/articles/PMC4155508/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25187353/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Commun&title=Clozapine-induced%20agranulocytosis%20is%20associated%20with%20rare%20HLA-DQB1%20and%20HLA-B%20alleles&volume=5&publication_year=2014&pages=4757&pmid=25187353&doi=10.1038/ncomms5757&)

7. Schizophrenia Working Group of the Psychiatric Genomics Consortium. Biological insights from 108 schizophrenia-associated genetic loci. Nature 2014; 511: 421–427.  [DOI](https://doi.org/10.1038/nature13595) | [PMC free article](/articles/PMC4112379/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25056061/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Biological%20insights%20from%20108%20schizophrenia-associated%20genetic%20loci&volume=511&publication_year=2014&pages=421&pmid=25056061&doi=10.1038/nature13595&)

8. Rees E, Walters JT, Georgieva L, Isles AR, Chambert KD, Richards AL et al. Analysis of copy number variations at 15 schizophrenia-associated loci. Br J Psychiatry 2014; 204: 108–114.  [DOI](https://doi.org/10.1192/bjp.bp.113.131052) | [PMC free article](/articles/PMC3909838/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24311552/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Psychiatry&title=Analysis%20of%20copy%20number%20variations%20at%2015%20schizophrenia-associated%20loci&volume=204&publication_year=2014&pages=108&pmid=24311552&doi=10.1192/bjp.bp.113.131052&)

9. Carroll LS, Williams HJ, Walters J, Kirov G, O'Donovan MC, Owen MJ. Mutation screening of the 3q29 microdeletion syndrome candidate genes DLG1 and PAK2 in schizophrenia. Am J Med Genet B Neuropsychiatr Genet 2011; 156B: 844–849.  [DOI](https://doi.org/10.1002/ajmg.b.31231) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21850710/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Med%20Genet%20B%20Neuropsychiatr%20Genet&title=Mutation%20screening%20of%20the%203q29%20microdeletion%20syndrome%20candidate%20genes%20DLG1%20and%20PAK2%20in%20schizophrenia&volume=156B&publication_year=2011&pages=844&pmid=21850710&doi=10.1002/ajmg.b.31231&)

10. Tiwari AK, Need AC, Lohoff FW, Zai CC, Chowdhury NI, Muller DJ et al. Exome sequence analysis of Finnish patients with clozapine-induced agranulocytosis. Mol Psychiatry 2014; 19: 403–405.  [DOI](https://doi.org/10.1038/mp.2013.74) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23752246/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Psychiatry&title=Exome%20sequence%20analysis%20of%20Finnish%20patients%20with%20clozapine-induced%20agranulocytosis&volume=19&publication_year=2014&pages=403&pmid=23752246&doi=10.1038/mp.2013.74&)

11. Howie BN, Donnelly P, Marchini J. A flexible and accurate genotype imputation method for the next generation of genome-wide association studies. PLoS Genet 2009; 5: e1000529.  [DOI](https://doi.org/10.1371/journal.pgen.1000529) | [PMC free article](/articles/PMC2689936/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19543373/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet&title=A%20flexible%20and%20accurate%20genotype%20imputation%20method%20for%20the%20next%20generation%20of%20genome-wide%20association%20studies&volume=5&publication_year=2009&pages=e1000529&pmid=19543373&doi=10.1371/journal.pgen.1000529&)

12. Price AL, Patterson NJ, Plenge RM, Weinblatt ME, Shadick NA, Reich D. Principal components analysis corrects for stratification in genome-wide association studies. Nat Genet 2006; 38: 904–909.  [DOI](https://doi.org/10.1038/ng1847) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16862161/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Principal%20components%20analysis%20corrects%20for%20stratification%20in%20genome-wide%20association%20studies&volume=38&publication_year=2006&pages=904&pmid=16862161&doi=10.1038/ng1847&)

13. Purcell S, Neale B, Todd-Brown K, Thomas L, Ferreira MA, Bender D et al. PLINK: a tool set for whole-genome association and population-based linkage analyses. Am J Hum Genet 2007; 81: 559–575.  [DOI](https://doi.org/10.1086/519795) | [PMC free article](/articles/PMC1950838/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17701901/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=PLINK:%20a%20tool%20set%20for%20whole-genome%20association%20and%20population-based%20linkage%20analyses&volume=81&publication_year=2007&pages=559&pmid=17701901&doi=10.1086/519795&)

14. Erikson GA, Deshpande N, Kesavan BG, Torkamani A. SG-ADVISER CNV: copy-number variant annotation and interpretation. Genet Med 2015; 17: 714–718.  [DOI](https://doi.org/10.1038/gim.2014.180) | [PMC free article](/articles/PMC4886732/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25521334/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genet%20Med&title=SG-ADVISER%20CNV:%20copy-number%20variant%20annotation%20and%20interpretation&volume=17&publication_year=2015&pages=714&pmid=25521334&doi=10.1038/gim.2014.180&)

15. Jia X, Han B, Onengut-Gumuscu S, Chen WM, Concannon PJ, Rich SS et al. Imputing amino acid polymorphisms in human leukocyte antigens. PloS One 2013; 8: e64683.  [DOI](https://doi.org/10.1371/journal.pone.0064683) | [PMC free article](/articles/PMC3675122/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23762245/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PloS%20One&title=Imputing%20amino%20acid%20polymorphisms%20in%20human%20leukocyte%20antigens&volume=8&publication_year=2013&pages=e64683&pmid=23762245&doi=10.1371/journal.pone.0064683&)

16. Browning SR, Browning BL. Rapid and accurate haplotype phasing and missing-data inference for whole-genome association studies by use of localized haplotype clustering. Am J Hum Genet 2007; 81: 1084–1097.  [DOI](https://doi.org/10.1086/521987) | [PMC free article](/articles/PMC2265661/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17924348/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=Rapid%20and%20accurate%20haplotype%20phasing%20and%20missing-data%20inference%20for%20whole-genome%20association%20studies%20by%20use%20of%20localized%20haplotype%20clustering&volume=81&publication_year=2007&pages=1084&pmid=17924348&doi=10.1086/521987&)

17. Athanasiou MC, Dettling M, Cascorbi I, Mosyagin I, Salisbury BA, Pierz KA et al. Candidate gene analysis identifies a polymorphism in HLA-DQB1 associated with clozapine-induced agranulocytosis. J Clin Psychiatry 2011; 72: 458–463.  [DOI](https://doi.org/10.4088/JCP.09m05527yel) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20868635/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Psychiatry&title=Candidate%20gene%20analysis%20identifies%20a%20polymorphism%20in%20HLA-DQB1%20associated%20with%20clozapine-induced%20agranulocytosis&volume=72&publication_year=2011&pages=458&pmid=20868635&doi=10.4088/JCP.09m05527yel&)

18. Kutyavin IV, Milesi D, Belousov Y, Podyminogin M, Vorobiev A, Gorn V et al. A novel endonuclease IV post-PCR genotyping system. Nucleic Acids Res 2006; 34: e128.  [DOI](https://doi.org/10.1093/nar/gkl679) | [PMC free article](/articles/PMC1636472/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17012270/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=A%20novel%20endonuclease%20IV%20post-PCR%20genotyping%20system&volume=34&publication_year=2006&pages=e128&pmid=17012270&doi=10.1093/nar/gkl679&)

19. Richards AL, Leonenko G, Walters JT, Kavanagh DH, Rees EG, Evans A et al. Exome arrays capture polygenic rare variant contributions to schizophrenia. Hum Mol Genet 2016; 25: 1001–1007.  [DOI](https://doi.org/10.1093/hmg/ddv620) | [PMC free article](/articles/PMC4754044/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26740555/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mol%20Genet&title=Exome%20arrays%20capture%20polygenic%20rare%20variant%20contributions%20to%20schizophrenia&volume=25&publication_year=2016&pages=1001&pmid=26740555&doi=10.1093/hmg/ddv620&)

20. Lee S, Emond MJ, Bamshad MJ, Barnes KC, Rieder MJ, Nickerson DA et al. Optimal unified approach for rare-variant association testing with application to small-sample case-control whole-exome sequencing studies. Am J Hum Genet 2012; 91: 224–237.  [DOI](https://doi.org/10.1016/j.ajhg.2012.06.007) | [PMC free article](/articles/PMC3415556/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22863193/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=Optimal%20unified%20approach%20for%20rare-variant%20association%20testing%20with%20application%20to%20small-sample%20case-control%20whole-exome%20sequencing%20studies&volume=91&publication_year=2012&pages=224&pmid=22863193&doi=10.1016/j.ajhg.2012.06.007&)

21. Green EK, Hamshere M, Forty L, Gordon-Smith K, Fraser C, Russell E et al. Replication of bipolar disorder susceptibility alleles and identification of two novel genome-wide significant associations in a new bipolar disorder case-control sample. Mol Psychiatry 2013; 18: 1302–1307.  [DOI](https://doi.org/10.1038/mp.2012.142) | [PMC free article](/articles/PMC3971368/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23070075/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Psychiatry&title=Replication%20of%20bipolar%20disorder%20susceptibility%20alleles%20and%20identification%20of%20two%20novel%20genome-wide%20significant%20associations%20in%20a%20new%20bipolar%20disorder%20case-control%20sample&volume=18&publication_year=2013&pages=1302&pmid=23070075&doi=10.1038/mp.2012.142&)

22. Willer CJ, Li Y, Abecasis GR. METAL: fast and efficient meta-analysis of genomewide association scans. Bioinformatics 2010; 26: 2190–2191.  [DOI](https://doi.org/10.1093/bioinformatics/btq340) | [PMC free article](/articles/PMC2922887/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20616382/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=METAL:%20fast%20and%20efficient%20meta-analysis%20of%20genomewide%20association%20scans&volume=26&publication_year=2010&pages=2190&pmid=20616382&doi=10.1093/bioinformatics/btq340&)

23. MacArthur DG, Manolio TA, Dimmock DP, Rehm HL, Shendure J, Abecasis GR et al. Guidelines for investigating causality of sequence variants in human disease. Nature 2014; 508: 469–476.  [DOI](https://doi.org/10.1038/nature13127) | [PMC free article](/articles/PMC4180223/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24759409/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Guidelines%20for%20investigating%20causality%20of%20sequence%20variants%20in%20human%20disease&volume=508&publication_year=2014&pages=469&pmid=24759409&doi=10.1038/nature13127&)

24. Konig J, Cui Y, Nies AT, Keppler D. Localization and genomic organization of a new hepatocellular organic anion transporting polypeptide. J Biol Chem 2000; 275: 23161–23168.  [DOI](https://doi.org/10.1074/jbc.M001448200) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10779507/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=Localization%20and%20genomic%20organization%20of%20a%20new%20hepatocellular%20organic%20anion%20transporting%20polypeptide&volume=275&publication_year=2000&pages=23161&pmid=10779507&doi=10.1074/jbc.M001448200&)

25. International Transporter Consortium, Giacomini KM, Huang SM, Tweedie DJ, Benet LZ, Brouwer KL et al. Membrane transporters in drug development. Nat Rev Drug Discov 2010; 9: 215–236.  [DOI](https://doi.org/10.1038/nrd3028) | [PMC free article](/articles/PMC3326076/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20190787/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Drug%20Discov&title=Membrane%20transporters%20in%20drug%20development&volume=9&publication_year=2010&pages=215&pmid=20190787&doi=10.1038/nrd3028&)

26. Search Collaborative Group, Link E, Parish S, Armitage J, Bowman L, Heath S et al. SLCO1B1 variants and statin-induced myopathy—a genomewide study. N Engl J Med 2008; 359: 789–799.  [DOI](https://doi.org/10.1056/NEJMoa0801936) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18650507/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=SLCO1B1%20variants%20and%20statin-induced%20myopathy%E2%80%94a%20genomewide%20study&volume=359&publication_year=2008&pages=789&pmid=18650507&doi=10.1056/NEJMoa0801936&)

27. Ramsey LB, Johnson SG, Caudle KE, Haidar CE, Voora D, Wilke RA et al. The clinical pharmacogenetics implementation consortium guideline for SLCO1B1 and simvastatin-induced myopathy: 2014 update. Clin Pharmacol Ther 2014; 96: 423–428.  [DOI](https://doi.org/10.1038/clpt.2014.125) | [PMC free article](/articles/PMC4169720/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24918167/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=The%20clinical%20pharmacogenetics%20implementation%20consortium%20guideline%20for%20SLCO1B1%20and%20simvastatin-induced%20myopathy:%202014%20update&volume=96&publication_year=2014&pages=423&pmid=24918167&doi=10.1038/clpt.2014.125&)

28. Kiyotani K, Mushiroda T, Kubo M, Zembutsu H, Sugiyama Y, Nakamura Y. Association of genetic polymorphisms in SLCO1B3 and ABCC2 with docetaxel-induced leukopenia. Cancer Sci 2008; 99: 967–972.  [DOI](https://doi.org/10.1111/j.1349-7006.2008.00765.x) | [PMC free article](/articles/PMC11159748/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18294295/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Sci&title=Association%20of%20genetic%20polymorphisms%20in%20SLCO1B3%20and%20ABCC2%20with%20docetaxel-induced%20leukopenia&volume=99&publication_year=2008&pages=967&pmid=18294295&doi=10.1111/j.1349-7006.2008.00765.x&)

29. Nambu T, Hamada A, Nakashima R, Yuki M, Kawaguchi T, Mitsuya H et al. Association of SLCO1B3 polymorphism with intracellular accumulation of imatinib in leukocytes in patients with chronic myeloid leukemia. Biol Pharm Bull 2011; 34: 114–119.  [DOI](https://doi.org/10.1248/bpb.34.114) | [PMC free article](/articles/PMC7384356/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21212528/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biol%20Pharm%20Bull&title=Association%20of%20SLCO1B3%20polymorphism%20with%20intracellular%20accumulation%20of%20imatinib%20in%20leukocytes%20in%20patients%20with%20chronic%20myeloid%20leukemia&volume=34&publication_year=2011&pages=114&pmid=21212528&doi=10.1248/bpb.34.114&)

30. Yamakawa Y, Hamada A, Nakashima R, Yuki M, Hirayama C, Kawaguchi T et al. Association of genetic polymorphisms in the influx transporter SLCO1B3 and the efflux transporter ABCB1 with imatinib pharmacokinetics in patients with chronic myeloid leukemia. Ther Drug Monit 2011; 33: 244–250.  [DOI](https://doi.org/10.1097/FTD.0b013e31820beb02) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21311410/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ther%20Drug%20Monit&title=Association%20of%20genetic%20polymorphisms%20in%20the%20influx%20transporter%20SLCO1B3%20and%20the%20efflux%20transporter%20ABCB1%20with%20imatinib%20pharmacokinetics%20in%20patients%20with%20chronic%20myeloid%20leukemia&volume=33&publication_year=2011&pages=244&pmid=21311410&doi=10.1097/FTD.0b013e31820beb02&)

31. Chew SC, Sandanaraj E, Singh O, Chen X, Tan EH, Lim WT et al. Influence of SLCO1B3 haplotype-tag SNPs on docetaxel disposition in Chinese nasopharyngeal cancer patients. Br J Clin Pharmacol 2012; 73: 606–618.  [DOI](https://doi.org/10.1111/j.1365-2125.2011.04123.x) | [PMC free article](/articles/PMC3376437/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21995462/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Clin%20Pharmacol&title=Influence%20of%20SLCO1B3%20haplotype-tag%20SNPs%20on%20docetaxel%20disposition%20in%20Chinese%20nasopharyngeal%20cancer%20patients&volume=73&publication_year=2012&pages=606&pmid=21995462&doi=10.1111/j.1365-2125.2011.04123.x&)

32. Hasegawa M, Cola PA, Meltzer HY. Plasma clozapine and desmethylclozapine levels in clozapine-induced agranulocytosis. Neuropsychopharmacology 1994; 11: 45–47.  [DOI](https://doi.org/10.1038/npp.1994.34) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7945743/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neuropsychopharmacology&title=Plasma%20clozapine%20and%20desmethylclozapine%20levels%20in%20clozapine-induced%20agranulocytosis&volume=11&publication_year=1994&pages=45&pmid=7945743&doi=10.1038/npp.1994.34&)

33. Centorrino F, Baldessarini RJ, Flood JG, Kando JC, Frankenburg FR. Relation of leukocyte counts during clozapine treatment to serum concentrations of clozapine and metabolites. Am J Psychiatry 1995; 152: 610–612.  [DOI](https://doi.org/10.1176/ajp.152.4.610) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7694912/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Psychiatry&title=Relation%20of%20leukocyte%20counts%20during%20clozapine%20treatment%20to%20serum%20concentrations%20of%20clozapine%20and%20metabolites&volume=152&publication_year=1995&pages=610&pmid=7694912&doi=10.1176/ajp.152.4.610&)

34. Mauri MC, Rudelli R, Bravin S, Gianetti S, Giuliani E, Guerrini A et al. Clozapine metabolism rate as a possible index of drug-induced granulocytopenia. Psychopharmacology 1998; 137: 341–344.  [DOI](https://doi.org/10.1007/s002130050628) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9676893/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Psychopharmacology&title=Clozapine%20metabolism%20rate%20as%20a%20possible%20index%20of%20drug-induced%20granulocytopenia&volume=137&publication_year=1998&pages=341&pmid=9676893&doi=10.1007/s002130050628&)

35. Pirmohamed M, Park K. Mechanism of clozapine-induced agranulocytosis: current status of research and implications for drug development. CNS Drugs 1997; 7: 139–158.  [DOI](https://doi.org/10.2165/00023210-199707020-00005) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23338132/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=CNS%20Drugs&title=Mechanism%20of%20clozapine-induced%20agranulocytosis:%20current%20status%20of%20research%20and%20implications%20for%20drug%20development&volume=7&publication_year=1997&pages=139&pmid=23338132&doi=10.2165/00023210-199707020-00005&)

36. Williams DP, Pirmohamed M, Naisbitt DJ, Uetrecht JP, Park BK. Induction of metabolism-dependent and -independent neutrophil apoptosis by clozapine. Mol Pharmacol 2000; 58: 207–216.  [DOI](https://doi.org/10.1124/mol.58.1.207) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10860943/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Pharmacol&title=Induction%20of%20metabolism-dependent%20and%20-independent%20neutrophil%20apoptosis%20by%20clozapine&volume=58&publication_year=2000&pages=207&pmid=10860943&doi=10.1124/mol.58.1.207&)

37. Pereira A, Dean B. Clozapine bioactivation induces dose-dependent, drug-specific toxicity of human bone marrow stromal cells: a potential in vitro system for the study of agranulocytosis. Biochem Pharmacol 2006; 72: 783–793.  [DOI](https://doi.org/10.1016/j.bcp.2006.06.006) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16844094/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem%20Pharmacol&title=Clozapine%20bioactivation%20induces%20dose-dependent,%20drug-specific%20toxicity%20of%20human%20bone%20marrow%20stromal%20cells:%20a%20potential%20in%20vitro%20system%20for%20the%20study%20of%20agranulocytosis&volume=72&publication_year=2006&pages=783&pmid=16844094&doi=10.1016/j.bcp.2006.06.006&)

38. Torres JZ, Summers MK, Peterson D, Brauer MJ, Lee J, Senese S et al. The STARD9/Kif16a kinesin associates with mitotic microtubules and regulates spindle pole assembly. Cell 2011; 147: 1309–1323.  [DOI](https://doi.org/10.1016/j.cell.2011.11.020) | [PMC free article](/articles/PMC4180425/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22153075/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell&title=The%20STARD9/Kif16a%20kinesin%20associates%20with%20mitotic%20microtubules%20and%20regulates%20spindle%20pole%20assembly&volume=147&publication_year=2011&pages=1309&pmid=22153075&doi=10.1016/j.cell.2011.11.020&)

39. Kindwall-Keller TL, Druhan LJ, Ai J, Hunter MG, Massullo P, Loveland M et al. Role of the proteasome in modulating native G-CSFR expression. Cytokine 2008; 43: 114–123.  [DOI](https://doi.org/10.1016/j.cyto.2008.04.015) | [PMC free article](/articles/PMC2556513/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18554923/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cytokine&title=Role%20of%20the%20proteasome%20in%20modulating%20native%20G-CSFR%20expression&volume=43&publication_year=2008&pages=114&pmid=18554923&doi=10.1016/j.cyto.2008.04.015&)

40. Ai J, Druhan LJ, Loveland MJ, Avalos BR. G-CSFR ubiquitination critically regulates myeloid cell survival and proliferation. PLoS One 2008; 3: e3422.  [DOI](https://doi.org/10.1371/journal.pone.0003422) | [PMC free article](/articles/PMC2561048/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18923646/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=G-CSFR%20ubiquitination%20critically%20regulates%20myeloid%20cell%20survival%20and%20proliferation&volume=3&publication_year=2008&pages=e3422&pmid=18923646&doi=10.1371/journal.pone.0003422&)

41. Yunis JJ, Corzo D, Salazar M, Lieberman JA, Howard A, Yunis EJ. HLA associations in clozapine-induced agranulocytosis. Blood 1995; 86: 1177–1183.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/7620171/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=HLA%20associations%20in%20clozapine-induced%20agranulocytosis&volume=86&publication_year=1995&pages=1177&pmid=7620171&)

42. Dettling M, Schaub RT, Mueller-Oerlinghausen B, Roots I, Cascorbi I. Further evidence of human leukocyte antigen-encoded susceptibility to clozapine-induced agranulocytosis independent of ancestry. Pharmacogenetics 2001; 11: 135–141.  [DOI](https://doi.org/10.1097/00008571-200103000-00004) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11266078/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=Further%20evidence%20of%20human%20leukocyte%20antigen-encoded%20susceptibility%20to%20clozapine-induced%20agranulocytosis%20independent%20of%20ancestry&volume=11&publication_year=2001&pages=135&pmid=11266078&doi=10.1097/00008571-200103000-00004&)

43. Verbelen M, Collier DA, Cohen D, MacCabe JH, Lewis CM. Establishing the characteristics of an effective pharmacogenetic test for clozapine-induced agranulocytosis. Pharmacogenomics J 2015; 15: 461–466.  [DOI](https://doi.org/10.1038/tpj.2015.5) | [PMC free article](/articles/PMC4762904/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25732907/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=Establishing%20the%20characteristics%20of%20an%20effective%20pharmacogenetic%20test%20for%20clozapine-induced%20agranulocytosis&volume=15&publication_year=2015&pages=461&pmid=25732907&doi=10.1038/tpj.2015.5&)
