# Association and cis-mQTL analysis of variants in CHRNA3-A5, CHRNA7, CHRNB2, and CHRNB4 in relation to nicotine dependence in a Chinese Han population

## Metadata
**Authors:** Qiang Liu, Haijun Han, Maiqiu Wang, Yinghao Yao, Li Wen, Keran Jiang, Yunlong Ma, Rongli Fan, Jiali Chen, Kunkai Su, Zhongli Yang, Wenyan Cui, Wenji Yuan, Xianzhong Jiang, Jingjing Li, Thomas J Payne, Jundong Wang, Ming D Li
**Journal:** Translational Psychiatry
**Date:** 2018 Apr 18
**DOI:** [10.1038/s41398-018-0130-x](https://doi.org/10.1038/s41398-018-0130-x)
**PMID:** 29666375
**PMCID:** PMC5904126
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5904126/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC5904126/pdf/41398_2018_Article_130.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC5904126/pdf/41398_2018_Article_130.pdf)

## Abstract

Nicotine dependence (ND) is a worldwide health problem. Numerous genetic studies have demonstrated a significant association of variants in nicotinic acetylcholine receptors (nAChRs) with smoking behaviors. However, most of these studies enrolled only subjects of European or African ancestry. In addition, although an increasing body of evidence implies a causal connection of single-nucleotide polymorphisms (SNPs) and epigenetic regulation of gene expression, few studies of this issue have been reported. In this study, we performed both association and interaction analysis for 67 SNPs in CHRNA3-A5, CHRNA7, CHRNB2, and CHRNB4 with ND in a Chinese Han population (N = 5055). We further analyzed cis-mQTL for the three most significant SNPs and 5580 potential methylation loci within these target gene regions. Our results indicated that the SNPs rs1948 and rs7178270 in CHRNB4 and rs3743075 in CHRNA3 were significantly associated with the Fagerström Test for Nicotine Dependence (FTND) score (p = 6.6 × 10−5; p = 2.0 × 10−4, and p = 7.0 × 10−4, respectively). Haplotype-based association analysis revealed that two major haplotypes, T-G and C-A, formed by rs3743075–rs3743074 in CHRNA3, and other two major haplotypes, A-G-C and G-C-C, formed by rs1948–rs7178270–rs17487223 in CHRNB4, were significantly associated with the FTND score (p ≤ 8.0 × 10−4). Further, we found evidence for the presence of significant interaction among variants within CHRNA3/B4/A5, CHRNA4/B2/A5, and CHRNA7 in affecting ND, with corresponding p values of 5.8 × 10−6, 8.0 × 10−5, and 0.012, respectively. Finally, we identified two CpG sites (CpG_2975 and CpG_3007) in CHRNA3 that are significantly associated with three cis-mQTL SNPs (rs1948, rs7178270, rs3743075) in the CHRNA5/A3/B4 cluster (p ≤ 1.9 × 10−6), which formed four significant CpG–SNP pairs in our sample. Together, we revealed at least three novel SNPs in CHRNA3 and CHRNB4 to be significantly associated with the FTND score. Further, we showed that these significant variants contribute to ND via two methylated sites, and we demonstrated significant interaction affecting ND among variants in CHRNA5/A3/B4, CHRNA7, and CHRNA4/B2/A5. In sum, these findings provide robust evidence that SNPs in nAChR genes convey a risk of ND in the Chinese Han population.

## Introduction

Tobacco smoking is a major public health problem that causes nearly 6 million deaths worldwide every year.^1^ Because of the lack of effective treatment for smoking addiction, the addictive properties of nicotine in tobacco smoke, and lack of awareness of the consequences of smoking in many regions, the worldwide death toll caused by cigarette smoking might well reach 10 million annually by 2020.^2^ Although many developed countries have implemented regulations and laws against tobacco smoking, which have led to dramatic reductions in smoking during recent years, smoking remains a significant issue in many developing countries, especially in Asia.^3,4^ For example, the prevalence of smoking in Chinese men aged 15 or older is an estimated 52.1%,^5^ meaning that male Chinese smokers account for almost one of third of the total number of smokers in the world.^6^

Although environmental factors play an important role in nicotine dependence (ND),^7^ genetics is another important component, as ND has an average heritability of 0.56.^7^ Of the identified susceptibility genes for ND, the most investigated ones are members of the nicotinic acetylcholine receptor (nAChR) gene family, which encodes 12 subunits (i.e., α2–α7; β2–β4) that are widely expressed in many brain regions.^8–10^ Nicotine, a primary component of tobacco smoke, exerts its biological effects on these nAChRs, where it either stimulates them or inactivates them through desensitization.^11^

Numerous studies using approaches such as genome-wide linkage, candidate gene association, and genome-wide association (GWAS) analysis have greatly advanced our knowledge of the genetic architecture underlying ND.^12^ The most replicated susceptibility loci for ND are nAChR subunit genes in the *CHRNA5/A3/B4* cluster on chromosome 15,^12–24^ *CHRNB3/A6* on chromosome 8,^12,25^ and *CYP2A6/A7* on chromosome 19.^12,14,25^ In addition, a significant association has been reported of variants in the *CHRNA5/A3/B4* cluster with ND and lung cancer.^18^

Although the reported associations of variants in *CHRNA7* with ND have not been replicated consistently in independent samples, pharmacological and molecular studies have strongly implicated α7 as an important target for ND and smoking cessation.^26^ Knockout α7 mice show a greater preference for oral nicotine than do their wild-type counterparts.^27^ In addition, local infusion of a highly selective antagonist (α-conotoxin ArIB) of α7 into the nucleus accumbens (NAc) shell or the anterior cingulate cortex increases nicotine self-administration.^28^

Numerous studies have shown that the β2 subunit is the most abundant of the nAChRs expressed in the brain and has high affinity for nicotine and acetylcholine (Ach). Further, many studies have demonstrated that the β2* (* indicates non-β2 subunits forming a functional nAChR by combining with β2) is required for nicotine reinforcement and reward.^29^ In the studies of therapeutics for smoking cessation, both β2-containing and α7-containing nicotine receptors have proved to be targets to curb tobacco addiction. However, α7 appears to act as modulator of nicotine reinforcement in opposition to α4β2* and α6β2*. Melis et al.^30^ showed a modulatory role for α7 in the ventral tegmental area to reduce β2* activation, especially for α4β2* and α6β2*, by stimulating the intracellular signaling cascades that inhibit β2*. These findings indicate that both α7 and β2* activation play an essential role in ND and smoking cessation.

Exogenous factors such as cigarette smoking can alter DNA methylation either locally or globally.^31^ Persistent exposure to smoke stimulates epigenetic reprogramming at the global level by affecting the methylation of repetitive elements.^32^ Although methylation is also under genetic influence, allele-specific DNA methylation is often correlated in related individuals.^33^ It is thus plausible to infer a correlation between these aberrant CpGs and risk variants for cigarette smoking.

A diversity of susceptibility variants for smoking has been identified, but the mechanisms by which these SNPs contribute to smoking-related traits are generally unclear. Benefits from high-throughput next-generation sequencing and high-density array platforms have allowed researchers to find regulatory variants by mapping expression and methylation quantitative trait loci (meQTLs).^34,35^ This approach provides a better way to reveal the mechanisms of significant variants from association studies. For example, considering the chromosome region of 15q25.1 as a well-established susceptibility region for smoking-related phenotypes, Hancock et al.^36^ assessed the number of methylation loci in the region based on the Illumina HumanMethylation27 array, which led to identification of a novel regulatory SNP, rs11636753, in *CHRNA5* that modulates methylation and expression in ND-relevant brain regions in multiancestry groups. Because the Illumina array contains methylation loci only from promoter regions,^37^ it is necessary to perform a fine-mapping analysis of this region to detect more *cis*-meQTLs.

There were three objectives of this study: (1) to determine which individual SNPs or haplotypes in nAChR subunit genes are associated with ND in a Chinese Han smoker sample, a less commonly investigated population; (2) to detect significant interactive effects among these genes in exerting influence on the etiology of ND; and (3) to link risk variants for ND and differential DNA methylation loci by *cis*-mQTLs analysis to explore the underlying mechanisms involved in ND.

## Materials and methods

### Subjects

This study included 5055 unrelated subjects consisting of both smokers (*N* = 2616) and non-smokers (*N* = 2439), who were recruited from local hospitals in Jincheng and Taiyuan in Shanxi Province of China in 2013. Participants with clinically diagnosed psychiatric disorders such as schizophrenia, Alzheimer’s disease, and major depressive disorder were excluded. Because few Chinese women aged 15 years or older smoke (~2.7%),^5^ only male smokers were included.

A set of structured questionnaires on cigarette smoking; demographic information such as age, education, and annual income; drug or substance use history; and neighborhood environment were administered to each participant by trained researchers. The detailed demographic characteristics of this sample are shown in Table [1](#Tab1). The average age of the smokers was 40.4 ± 9.7 years and that of the non-smokers was 37.0 ± 10.9 years. Written informed consent was provided by each participant after receiving a detailed explanation of the project and process of this study. The study and all the questionnaires used in the study were approved by the Ethics Committee of First Affiliated Hospital of Zhejiang University School of Medicine.

### Table 1.

| Characteristic | Smokers | Non-smokers |
| --- | --- | --- |
| Sample size | 2616 | 2439 |
| Age, years (S.D.) | 40.4 (9.7) | 37.0 (10.9) |
| FTND in smokers only (S.D.) | 5.45 (1.41) | NA |
| No. of family members smoke (S.D.) | 1.9 (0.8) | 1.6 (0.7) |
| Income categories (S.D.) | 4.0 (1.2) | 4.0 (1.2) |

Table 1 Caption: Characteristics of samples used in the study

### Phenotype assessment

Non-smokers were defined as persons who had smoked fewer than 100 cigarettes in their lifetimes. Smoking dependence was assessed by the Fagerström Test for Nicotine Dependence (FTND) measure (0–10 scale).^38,39^ We defined the smokers with an FTND score of ≥6 as heavy smokers (*N* = 1243) and those with an FTND score of <6 as light smokers (*N* = 1373) (Supplementary Table [6](#MOESM2)).^40^

### Selection of SNPs for genotyping analysis

Peripheral blood was collected from each participant. Genomic DNA was extracted by the Qiagen DNA purification kit according to the manufacturer’s protocol. A nanodrop was utilized to determine the DNA concentration of each sample based on the optical density (OD) at 260 nm, and the DNA quality of each sample was assessed by the OD 260/280 ratio.

Genotyping was performed with the *Taq*man OpenArray Genotyping Platform (Applied Biosystems, Inc.). For each sample, a mixture of 2 μl of DNA (ca. 100 ng) and 2 μl of 2× *Taq*man OpenArray Genotyping Master Mix was added to a 384-well plate. After the plate was sealed and centrifuged briefly, each plate was loaded into the QuantStudio^TM^ 12 K Flex for PCR amplification. The amplified results were reviewed by *Taq*man Genotyper Software (v. 1.3.1) for SNP calling.

The SNPs examined in this study were three in *CHRNB2*, 14 in *CHRNA3*, 10 in *CHRNB4*, 16 in *CHRNA5*, 23 in *CHNRA7*, and 6 in *CHRNA4*. They were all selected with the SNP Browser software from Applied Biosystems by searching the dbSNP database and published papers. However, because of the calling rate of <95%, SNPs rs6495309 in *CHRNA3*, rs11637890 in *CHRNB4*, rs503464 and rs601079 in *CHRNA5*, and rs6494211 in *CHRNA7* were excluded. After those quality control steps, a total of 67 SNPs remained for the association analysis. All of these SNPs have a minor allele frequency (MAF) of >1% and a *p* value of >1 × 10^−4^ in the Hardy–Weinberg equilibrium test.

### Population structure analysis

We used the Structure program (v. 2.3.4)^41^ to assess population stratification for our samples based on the genotyping data for a panel of 30 ancestry informative markers.^42^ Simulation parameters were set to 100,000 burn-ins and 100,000 iterations. To increase the accuracy when inferring admixture and taking account of the samples being recruited from two sites, we set *K* to 2. Our population structure analysis revealed no evidence of population admixture among the samples from the two recruitment sites, so we performed our association analysis on both sets of samples together with the goal of increasing our statistical power (Supplementary Figure [1](#MOESM1)).

### DNA methylation

DNA methylation data used here were obtained from an ongoing whole-genome bisulfite sequencing project in this laboratory which consisted of 72 subjects selected from the same sample set as used for the abovementioned association analyses on the basis of their age, gender, and smoking status (36 non-smokers; 36 smokers). DNA methylation was identified using the Illumina HiSeq X Ten platform with an average of about 700 million (±75 million) 150-bp paired-end reads per sample. Clean reads were mapped to the hg19 reference genome using Bismark.^43^ We first combined two strands of information of CpG sites and then excluded those CpGs with <5 reads or that overlapped with common variants in the Chinese Han genome (MAF > 0.05). The MAF of each variant was determined by an unpublished Whole Genome Sequencing dataset of a Chinese Han sample (*N* = 1329) in our laboratory.

### Individual SNP-based and haplotype-based association analysis

Individual SNP-based association analysis was performed for both smoking status and FTND score using PLINK (v. 1.07)^44^ under the logistic regression model. Adjusted covariates included age, site (Taiyuan or Jincheng), number of smoking family members, and income. In the haplotype-based association analysis, both pair-wise linkage disequilibrium (LD) and haplotypes were evaluated by Haploview (v. 4.2),^45,46^ and the analysis of those major haplotypes (with a frequency >5%) with each phenotype was performed by HaploStats (v. 1.7.7) in *R* and adjusted for same covariates under the additive model.^47^

### Interaction analysis of variants in CHRNA3, CHRNA4, CHRNA5, CHRNA7, CHRNB2, and CHRNB4

To estimate the epistatic contribution of variants in six nAChR subunit genes to ND, we performed SNP-by-SNP interaction analysis using the GMDR-GPU program developed by our group,^48,49^ performing an exhaustive search of all combinations containing 2–5 SNPs each. The best interaction model was determined according to the following three parameters: (1) the cross-validation consistency (CVC) statistics for the selected variant combination; (2) the predictive accuracy, determined by 10^7^ permutation tests for the selected SNP combinations; and (3) the significant *p* value.

On the basis of chromosomal location and known functional nAChR composition, we separated the genotype file into two: (1) all variants in *CHRNA3, CHRNA5, CHRNB4*, and *CHRNA7*, all of which are located on chromosome 15; and (2) all variants in *CHRNA4, CHRNA5*, and *CHRNB2* because they can form a functional (α4β2)_2_α5 receptor in humans.

### Determination of cis-mQTL

Our *cis-*mQTL analysis was restricted to 250 kb upstream and downstream of each SNP. The intervals for adjacent SNPs were combined if they overlapped. Taken together, a total of 8,915 CpG sites were revealed within the intervals of the target genes. Prior to analysis, we removed the low-quality DNA methylation sites with a calling rate of <80%, which left 5580 highly qualified methylated CpGs for *cis*-mQTL analysis.

We choose three significant SNPs for our *cis*-mQTL association analysis according to individual SNP-based analysis, namely rs1948, rs3743075, and rs7178270. They were all intronic polymorphisms associated with the extent of methylation. We used the Matrix eQTL (v. 2.1.1) R package^50^ to test association of the three SNPs, with linear regression under an additive model and adjusted for age, smoking status, and whether the subject was a coal miner. Bonferroni correction was used to define significant associations (i.e., *p* = 0.05/14,161 = 3.5 × 10^−6^), where 14,161 is the total number of associations tested for the abovementioned three significant SNPs (i.e., 4623 for rs3743075, 4743 for r1948, and 4795 for rs7178270).

## Results

### Individual SNP-based association analysis

As shown in Table [2](#Tab2), for smoking status, we found that eight variants in *CHRNA5* and one variant in *CHRNA4* showed significant associations (*p* = 8.0 × 10^−3^ to 4.4 × 10^−2^). However, they were no longer significant after Bonferroni correction for multiple testing.

### Table 2.

| Gene | SNP | Function | Major allele | Minor allele | MAF | Smoking status |  | FTND |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| OR | p value | OR | p value |  |  |  |  |  |  |
| CHRNA3 | rs3743075 | Synonymous | C | T | 0.47 | 1.1 | 0.056 | 0.8 | 7.0E−04 |
| rs3743074 | Intronic | A | G | 0.46 | 1.2 | 0.083 | 0.8 | 0.002 |  |
| rs6495307 | Intronic | C | T | 0.22 | 1.1 | 0.160 | 0.9 | 0.039 |  |
| rs3743077 | Intronic | C | T | 0.23 | 1.1 | 0.200 | 0.8 | 0.016 |  |
| rs660652 | 3’UTR | G | A | 0.22 | 1.1 | 0.240 | 0.9 | 0.024 |  |
| rs2869546 | Intronic | T | C | 0.23 | 1.1 | 0.270 | 0.8 | 0.014 |  |
| CHRNA5 | rs667282 | Intronic | T | C | 0.46 | 1.1 | 0.008 | 1.2 | 0.005 |
| rs555018 | Intronic | A | G | 0.21 | 1.1 | 0.011 | 0.8 | 0.019 |  |
| rs621849 | Intronic | A | G | 0.27 | 1.1 | 0.020 | 0.9 | 0.024 |  |
| rs680244 | Intronic | C | T | 0.27 | 1.1 | 0.021 | 0.9 | 0.030 |  |
| rs692780 | Intronic | G | C | 0.22 | 1.1 | 0.026 | 0.8 | 0.022 |  |
| rs588765 | Intronic | C | T | 0.21 | 1.1 | 0.032 | 0.9 | 0.025 |  |
| rs6495306 | Intronic | A | G | 0.21 | 1.1 | 0.044 | 0.9 | 0.039 |  |
| rs16969968 | Missense | G | A | 0.03 | 1.3 | 0.042 | 1.4 | 0.041 |  |
| rs514743 | Intronic | A | T | 0.22 | 1.1 | 0.410 | 0.9 | 0.029 |  |
| rs647041 | Intronic | C | T | 0.22 | 1.1 | 0.220 | 0.9 | 0.023 |  |
| rs615470 | 3’UTR | C | T | 0.22 | 1.1 | 0.260 | 0.9 | 0.028 |  |
| CHRNB4 | rs7178270 | Intronic | C | G | 0.40 | 1.1 | 0.110 | 0.7 | 2.0E−04 |
| rs1948 | Synonymous | G | A | 0.47 | 1.1 | 0.120 | 0.8 | 6.6E−05 |  |
| rs950776 | Intronic | T | C | 0.17 | 1.1 | 0.330 | 0.8 | 0.003 |  |
| CHRNA4 | rs3827020 | Intronic | T | C | 0.46 | 1.0 | 0.430 | 1.2 | 0.015 |

Table 2 Caption: SNPs with p < 0.05 in individual SNP association on analyses with smoking status and FTND in Chinese samples

For the FTND phenotype, we found that many variants showed significant associations prior to correction for multiple testing. However, only rs1948 and rs7178270 in *CHRNB4* and rs3743075 in *CHRNA3* remained significant after Bonferroni correction, with *p* values of 6.6 × 10^−5^, 2.0 × 10^−4^, and 7.0 × 10^−4^, the odds ratio (OR) under the additive model being 0.8, 0.7, and 0.8, respectively (Supplementary Table [1](#MOESM2)).

### Haplotype-based association analysis

According to the haplotype block definition of Gabriel et al.^46^ there were seven blocks in the *CHRNA5/A3/B4* cluster, four in *CHRNA7*, and one in *CHRNA4* and *CHRNB2* (*D*′ > 0.90) (see Supplementary Figure [2](#MOESM1) and Supplementary Figure [3](#MOESM1)).

As shown in Table [3](#Tab3), for the ND measured by smoking status, we found one haplotype, G-C-C, formed by rs1948, rs7178270, and rs17487223, to be marginally associated with smoking status (Hap-Score 2.13; *p* = 0.0331). For the FTND measure, we detected five major haplotypes in the *CHRNA5/A3/B4* cluster that were significantly associated with ND. Of them, two major haplotypes, C-A and T-G, formed by rs3743075 and rs3743074, showed significant associations with the FTND score (Hap-Score 3.51; *p* = 4.0 × 10^−4^; Hap-Score −3.37; *p* = 8.0 × 10^−4^). Another haplotype, C-T, formed by rs12914385–rs2869546, also showed significance (Hap-Score −3.02; *p* = 2.6 × 10^−3^). Further, we found two other haplotypes, A-G-C and G-C-C, formed by SNPs rs1948, rs7178270, and rs17487223 with a frequency of 0.40 and 0.48, to be significantly associated with FTND (Hap-Score −3.57; *p* = 4.0 × 10^−4^; Hap-Score 3.79; *p* = 2.0 × 10^−4^, respectively). Importantly, the haplotype G-C-C, formed by rs1948, rs7178270, and rs17487223 in *CHRNA5/A3/B4*, was significantly associated with both smoking status and FTND. In addition, we found a haplotype in *CHRNA4* to be marginally associated with FTND (Supplementary Table [2](#MOESM2)).

### Table 3.

| ND measure | Gene | SNP combination | Haplotype | Hap-Freq | Hap-Score | P-Hap | P-Global |
| --- | --- | --- | --- | --- | --- | --- | --- |
| Smoking status | CHRNA5/A3/B4 | rs1948–rs7178270–rs17487223 | G-C-C | 0.49 | 2.13 | 0.0331 | 0.1122 |
| rs684513–rs667282–rs588765–rs6495306–rs17486278–rs680244–rs621849–rs692780–rs951266 | C-C-C-A-A-C-A-G-G | 0.22 | 2.66 | 0.0078 | 0.0462 |  |  |
| FTND | CHRNA5/A3/B4 | rs684513–rs667282–rs588765–rs6495306–rs17486278–rs680244–rs621849–rs692780–rs951266 | C-T-T-G-A-T-G-C-G | 0.21 | 2.27 | 0.0231 | 0.0197 |
| rs647041–rs16969968–rs514743–rs615470–rs660652–rs578776–rs6495307–rs1051730–rs3743077 | T-G-T-T-A-G-T-G-T | 0.22 | 2.18 | 0.0291 | 0.0258 |  |  |
| rs1948–rs7178270–rs17487223 | A-G-C | 0.40 | −3.57 | 0.0004 | 0.0007 |  |  |
| G-C-C | 0.48 | 3.79 | 0.0002 |   |  |  |  |
| rs12914385–rs2869546 | C-C | 0.23 | 2.47 | 0.0135 | 0.0065 |  |  |
| C-T | 0.50 | −3.02 | 0.0026 |   |  |  |  |
| rs3743075–rs3743074 | T-G | 0.53 | −3.37 | 0.0008 | 0.0009 |  |  |
| C-A | 0.47 | 3.51 | 0.0004 |   |  |  |  |
| CHRNA4 | rs2236196–rs3787137–rs3827020–rs1044393–rs3787140 | A-G-C-G-T | 0.45 | −2.14 | 0.0323 | 0.2009 |  |

Table 3 Caption: Major haplotypes (frequency > 0.05) associated with smoking status and FTND at p < 0.05 in Chinese samples

### SNP-by-SNP interaction analysis

Combinations of different nAChR subunits can form various functional nicotinic receptors, which play various physiological roles in both the peripheral and the central nervous systems. To determine whether there exists any epistatic effect among these nAChR subunits, we performed interaction analysis using our own GMDR-GPU program, which revealed two significant interaction models for FTND. The first model consists of SNPs rs16969968 in *CHRNA5* and rs7178270 in *CHRNB4*, with a CVC 7 of 10, prediction accuracy of 55.4%, and empirical *p* value of 8 × 10^−5^ based on 10^7^ permutations. The second significant model consists of SNPs rs904951 and rs7178176 in *CHRNA7*, with a CVC 10 of 10, prediction accuracy of 56.0%, and empirical *p* value of 5.8 × 10^−6^ from 10^7^ permutations (Supplementary Table [S3](#MOESM2)). Although we also did interaction analysis on SNPs from *CHRNA3*, *CHRNB4*, and *CHRNA5* with *CHRNA7*, we did not find any significant interaction for variants in *CHRNA7* with those from the other nAChR subunit genes.

For smoking status, we detected a best interaction model, which consists of rs16969968 in *CHRNA5*, rs4845378 and rs3811450 in *CHRNB2*, and rs3787137 and rs1044393 in *CHRNA4*. This model has a CVC 7 of 10, prediction accuracy of 52.3%, and empirical *p* value of 0.012 (Supplementary Table [S3](#MOESM2)).

### Relations between genotype and methylation status

All association analysis results of top SNPs with *cis*-mQTLs are shown in Supplementary Table [4](#MOESM2) and Supplementary Figure [4](#MOESM1) (*p* < 5.0 × 10^−4^). Together, we detected four significant associations for SNP–CpG pairs (Fig. [1](#Fig1)). These pairs were formed by three SNPs significant for ND (rs3743075 in *CHRNA3* and rs1948 and rs7178270 in *CHRNB4*) and two distinct CpG sites (CpG_2975, CpG_3007). Significant *cis-*mQTL analysis results and the corresponding *p* values, with a range of 5.2 × 10^−15^ to 1.9 × 10^−06^ (Fig. [2](#Fig2)).

### Fig. 1. Schematic diagram of mQTL analysis and position of investigated SNPs.

![Fig. 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8e73/5904126/1d0e2e8aa0fe/41398_2018_130_Fig1_HTML.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5904126_41398_2018_130_Fig1_HTML.jpg)

The open rectangles represent the exons, black rectangles are untranslated regions, and solid lines indicate introns according to NCBI and UCSC Browsers. Dashed lines refer to the significant SNP–CpG pair. Pos: genome position of SNP and CpG site

### Fig. 2. Association between genotype data of risk variants and extent of methylation in blood samples of 72 participants of Chinese Han origin tested by cis-mQTLs.

![Fig. 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8e73/5904126/90968070ca4e/41398_2018_130_Fig2_HTML.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5904126_41398_2018_130_Fig2_HTML.jpg)

Distribution of the values at the methylation sites is presented for individuals carrying zero, one, and two minor alleles. Only the most significant cis-mQTLs are shown for each variant. The cis-eQTL analysis associated three variants with CHRNA3 expression in human NAc basal ganglia. The cis-eQTL figures were downloaded from GTEX PORTAL (https://gtexportal.org/home/)

Given that the two highly methylated CpG sites are both located in *CHRNA3*, our association analyses of the three SNPs with RNA expression were conducted only for *CHRNA3*. According to the *cis*-eQTL association data from GTEX PORTAL ([https://gtexportal.org/home/](https://gtexportal.org/home/)), we found that the three SNPs linked to ND (i.e., SNPs rs1948, rs7178270, and rs3743075) correlated significantly with two methylation sites that showed allele-specific mRNA expression of *CHRNA3* and *CHRNA5* in several ND-related brain regions of human post-mortem tissue (Supplementary Table [5](#MOESM2)), with *p* values ranging from 3.2 × 10^−6^ to 4.0 × 10^−15^.

In addition, we performed cis-mQTL analysis by adjusting with FTND score and age. Because the FTND score is a ND measure for smokers, we only included smokers in this analysis. We obtained consistent results (Supplementary Table [7](#MOESM2)). The significant CpG–SNP pairs were still formed by CpG_2975 with rs3743075, rs1948 and rs7178270, respectively (*p* = 2.2 × 10^−6^; *p* = 1.8 × 10^−5^; *p* = 2.4 × 10^−5^). These results further support that these three variants are risk variants that influence the extent of methylation in smokers.

## Discussion

To our knowledge, this is the first study exploring the underlying mechanisms of ND using multiple approaches of genetic associations, interaction, and *cis-*mQTLs in a Chinese Han population. Our results revealed that three SNPs and five major haplotypes in *CHRNA5/A3/B4* were significantly associated with the FTND score. We also demonstrated that the significant ND-associated variants (rs1948, rs7178270, and rs3743075) are novel *cis*-meQTLs influencing both the extent of methylation and mRNA expression of *CHRNA3*. In addition, through SNP-by-SNP interaction study, we found several SNPs in *CHRNA5/B4*, *CHRNA5/A4/B2*, and *CHRNA7* that interactively confer susceptibility to ND in our Han sample.

Two haplotypes formed by rs3743074 and rs3743075 were significantly associated with the FTND score. Of them, rs3743075 was significantly and rs3743074 was marginal associated with the score. Further, we showed that the minor allele of rs3743075 in *CHRNA3* was significantly associated with the extent of methylation of CpG_2975 and CpG_3007 (*p* = 5.2 × 10^−15^ and *p* = 1.9 × 10^−6^, respectively) and marginally associated with CpG_3006 (*p* = 4.7 × 10^−4^). In concert with these findings, Hancock et al.^36^ reported that rs3743075 was negatively correlated with a methylation locus of cg22670733 in adult brain (*p* = 7.0 × 10^−6^). According to the annotation by GTEX PORTAL ([https://gtexportal.org/home/](https://gtexportal.org/home/)), rs3743075 has strong evidence of affecting expression of *CHRNA3* in the brain nucleus accumnems (NAc) (*p* = 4.9 × 10^−11^)^51^ and of *CHRNA5* in the NAc, anterior cingulate cortex (BA 24), frontal cortex (BA 9), hippocampus and whole blood (*p* = 2.3 × 10^−10^; *p* = 4.9 × 10^−8^; *p* = 3.0 × 10^−10^; *p* = 1.6 × 10^−5^)^52^ as a *cis*-eQTL in humans. By using the Web-based tool SWISS-PROT ([http://www.uniprot.org/](http://www.uniprot.org/)), we found that rs3743075 represents a part of the conserved transcription factor-binding site for interferon regulatory factor 7 (*IRF-7*) in humans. Previously, we showed that the expression of *IRF-7* mRNA was significantly suppressed by nicotine treatment in mouse RAW264.7 macrophages.^53^ Thus, it is highly likely that rs3743075 is a functional variant regulating the expression of *CHRNA3* by altering the methylation contribution. In light of previous evidence demonstrating that smoking-associated abnormal methylation loci might convey a risk for lung cancer,^54,55^ this may indicate that rs3743075 is a biomarker for lung cancer.

SNPs rs1948 and rs7178270 in *CHRNB4* showed the strongest association with the FTND score in our Chinese Han sample. In addition, we found that two haplotypes, A-G-C and G-C-C (formed by rs1948, rs7178270, and rs17487223), showed significant associations with the FTND score. Moreover, we demonstrated that the minor allele of rs1948 and rs7178270 significantly reduced the methylation of CpG_2975 (*p* = 4.4 × 10^−13^ and *p* = 1.2 × 10^−12^, respectively). Interestingly, these two SNPs increased the expression of *CHRNA3* in the NAc.^51^ These results indicate that SNPs rs1948 and rs7178270 confer susceptibility to ND by suppressing the methylation that leads to increased expression of *CHRNA3*, which is consistent with previous documentation performed with Dutch persons born in The Netherlands.^56^ For *CHRNA5*, there were eight SNPs showing marginal associations with both smoking status and FTND (*p* < 0.05). Among them, two SNPs, rs16969968 and rs667282 in *CHRNA5*, have been widely associated with ND-related phenotypes and lung cancer in subjects of multiancestry, such as Europeans, Africans, and Asian.^12–14,17,19,22,24,57,58^ In addition, there exists one haplotype consisting of nine variants in the *CHRNA5/A3/B4* cluster that is associated marginally with the FTND score and two haplotypes, formed by another nine SNPs in the *CHRNA5/A3/B4* cluster, that are associated marginally with both the FTND and smoking status.

We performed gene–gene interaction analysis of all selected SNPs within *CHRNA5/A3/B4* and *CHRNA7*. We detected the best interaction model between rs7178270 in *CHRNB4* and rs16969968 in *CHRNA5* (*p* = 8.0 × 10^−5^) with FTND, providing genetics-based evidence supporting the view that *CHRNB4* interacts with *CHRNA5* in contributing to ND. Previous molecular studies found that in mice, medial habenula overexpression of *Chrnb4* strongly increases the aversive effect of nicotine, which can be reversed by lentiviral-mediated expression of *Chrna5* D398N.^59^ Moreover, we observed that the interaction model of (α4β2)_2_α5 grouped by five SNPs (rs4845378, rs3811450, rs3787137, rs1044393, and rs16969968) was significantly associated with smoking status (*p* = 0.012). This interactive model was compatible with biological evidence that (α4β2)_2_α5 carrying the risk allele of rs16969968 has a twofold lower maximum response with a nicotinic agonist compared with the α5 carrying a protective allele.^60^ This is the first evidence of an epistatic effect of this gene cluster discovered via genetic methods in the Chinese Han population.

To date, there are far fewer loci in *CHRNA7* reported to be significantly associated with smoking-related phenotypes, in contrast to the numerous reports that the gene is essential in ND in pharmacological studies. In the present work, we did not detect any significant association by either individual SNP-based or haplotype-based association analysis. One possible explanation for the inconsistent observations is that the variants in *CHRNA7* contribute to ND risk through SNP-by-SNP interactions. Concordant with this assumption, we found a significant interaction model formed by rs904951 and rs7178176 associated with FTND. rs7178176 was previously reported in association with an increasing probability of dizziness at first smoke inhalation by adolescents in a Canada sample with a mixed ethnical origins.^61^ For the first time, we provide evidence that an epistatic effect of *CHRNA7* is implicated in ND, which is in accord with the biological fact that (α7)_5_ forms a homomeric pentamer in humans.

There are a few limitations of this study. First, the number of subjects in the methylation cohort might be too small to detect weak signals of risk polymorphisms and mapping *trans*-mQTLs. Considering that *trans*-meQTL is less common and contributes only minor effects to phenotypes,^62^ we concentrated our analysis on identification of *cis*-meQTLs. In spite of the small samples, because we adopted stringent criteria to select high-quality CpG sites, we believe that our *cis*-meQTL analysis results are trustworthy and deserve to be replicated by others if possible. Second, we chose 67 variants within the six genes to perform association analysis for our sample, and the majority of them are common SNPs. Thus, we could not assess the effects of rare variants, which have been thought to exert more influence on traits of interest. Further next-generation sequencing-based studies are warranted to identify more rare variants within these regions for ND in the Chinese Han population.

To sum up, this is the first study to demonstrate the significant effects of *CHRNA3–CHRNB4* variants on ND in the Chinese Han population. Having succeeded in performing integrative data analysis for genetic polymorphisms and DNA methylation, we relate SNP-based association, *cis*-mQTL analysis, and mRNA expression using public data to explore the biological mechanisms of ND. Further, we provide novel evidence of a significant genetic interactive model for *CHRNA7* in affecting ND, which extends our knowledge of the potential biological mechanism for this gene’s actions affecting ND. A complete understanding of the genetic variants will help us find pharmacologic targets to account for the addictive properties of nicotine in Chinese smokers.

## Electronic supplementary material

## Acknowledgements

This study was supported in part by the China Precision Medicine Initiative (2016YFC0906300), Research Center for Air Pollution and Health of Zhejiang University, the State Key Laboratory for Diagnosis and Treatment of Infectious Diseases of the First Affiliated Hospital of Zhejiang University, and Shanxi Key Laboratory of Ecological Animal Science and Environmental Veterinary Medicine of Shanxi Agricultural University. We thank Dr. David L. Bronson for excellent editing of this manuscript.

## Author contributions

H.H., L.W., K.J., Y.M., K.S., Z.Y., W.C., W.Y., and X.J. participated clinical data collection. Q.L., H.H,. M.W., R.F., J.C., and X.J. performed the laboratory experiments. Q.L., L.W., and Y.Y. participated data analysis. Q.L., T.J.P., and J.W. participated in writing of the paper. M.D.L. conceived the study and was involved in every step of the research. All authors approved the final manuscript.

### Conflict of interest

### Conflict of interest

The authors declare that they have no conflict of interest.

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Centers for Disease Control and Prevention. Current cigarette smoking among adults - United States, 2011. MMWR Morb. Mortal. Wkly. Rep. 2012;61:889–894.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/23134971/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=MMWR%20Morb.%20Mortal.%20Wkly.%20Rep.&title=Current%20cigarette%20smoking%20among%20adults%20-%20United%20States,%202011&volume=61&publication_year=2012&pages=889-894&pmid=23134971&)

2. Warren CW, Jones NR, Eriksen MP, Asma S, Global Tobacco Surveillance System collaborative group. Patterns of global tobacco use in young people and implications for future chronic disease burden in adults. Lancet. 2006;367:749–753. doi: 10.1016/S0140-6736(06)68192-0.  [DOI](https://doi.org/10.1016/S0140-6736(06)68192-0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16517275/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Patterns%20of%20global%20tobacco%20use%20in%20young%20people%20and%20implications%20for%20future%20chronic%20disease%20burden%20in%20adults&author=CW%20Warren&author=NR%20Jones&author=MP%20Eriksen&author=S%20Asma&volume=367&publication_year=2006&pages=749-753&pmid=16517275&doi=10.1016/S0140-6736(06)68192-0&)

3. WHO Media centre. Tobacco Fact sheet N 339. http://www.who.int/mediacentre/factsheets/fs339/en/ (2016).  [http://www.who.int/mediacentre/factsheets/fs339/en/](http://www.who.int/mediacentre/factsheets/fs339/en/)

4. Benowitz NL. Clinical pharmacology of nicotine: implications for understanding, preventing, and treating tobacco addiction. Clin. Pharmacol. Ther. 2008;83:531–541. doi: 10.1038/clpt.2008.3.  [DOI](https://doi.org/10.1038/clpt.2008.3) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18305452/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Clinical%20pharmacology%20of%20nicotine:%20implications%20for%20understanding,%20preventing,%20and%20treating%20tobacco%20addiction&author=NL%20Benowitz&volume=83&publication_year=2008&pages=531-541&pmid=18305452&doi=10.1038/clpt.2008.3&)

5. Center for Disease Control. Major finding of 2015 Chinese adults tobacco survery (Chinese Center for Disease Control and Prevention, Beijing, 2015).

6. Liu BQ, et al. Emerging tobacco hazards in China: 1. Retrospective proportional mortality study of one million deaths. BMJ. 1998;317:1411–1422. doi: 10.1136/bmj.317.7170.1411.  [DOI](https://doi.org/10.1136/bmj.317.7170.1411) | [PMC free article](/articles/PMC28719/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9822393/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMJ&title=Emerging%20tobacco%20hazards%20in%20China:%201.%20Retrospective%20proportional%20mortality%20study%20of%20one%20million%20deaths&author=BQ%20Liu&volume=317&publication_year=1998&pages=1411-1422&pmid=9822393&doi=10.1136/bmj.317.7170.1411&)

7. Li MD, Cheng R, Ma JZ, Swan GE. A meta-analysis of estimated genetic and environmental effects on smoking behavior in male and female adult twins. Addiction. 2003;98:23–31. doi: 10.1046/j.1360-0443.2003.00295.x.  [DOI](https://doi.org/10.1046/j.1360-0443.2003.00295.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12492752/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Addiction&title=A%20meta-analysis%20of%20estimated%20genetic%20and%20environmental%20effects%20on%20smoking%20behavior%20in%20male%20and%20female%20adult%20twins&author=MD%20Li&author=R%20Cheng&author=JZ%20Ma&author=GE%20Swan&volume=98&publication_year=2003&pages=23-31&pmid=12492752&doi=10.1046/j.1360-0443.2003.00295.x&)

8. Le Novere N, Corringer PJ, Changeux JP. The diversity of subunit composition in nAChRs: evolutionary origins, physiologic and pharmacologic consequences. J. Neurobiol. 2002;53:447–456. doi: 10.1002/neu.10153.  [DOI](https://doi.org/10.1002/neu.10153) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12436412/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Neurobiol.&title=The%20diversity%20of%20subunit%20composition%20in%20nAChRs:%20evolutionary%20origins,%20physiologic%20and%20pharmacologic%20consequences&author=N%20Le%20Novere&author=PJ%20Corringer&author=JP%20Changeux&volume=53&publication_year=2002&pages=447-456&pmid=12436412&doi=10.1002/neu.10153&)

9. Elgoyhen, A. B. et al. alpha10: a determinant of nicotinic cholinergic receptor function in mammalian vestibular and cochlear mechanosensory hair cells. Proc. Natl Acad Sci USA98, 3501–3506 (2001).  [DOI](https://doi.org/10.1073/pnas.051622798) | [PMC free article](/articles/PMC30682/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11248107/)

10. Elgoyhen AB, Johnson DS, Boulter J, Vetter DE, Heinemann S. Alpha 9: an acetylcholine receptor with novel pharmacological properties expressed in rat cochlear hair cells. Cell. 1994;79:705–715. doi: 10.1016/0092-8674(94)90555-X.  [DOI](https://doi.org/10.1016/0092-8674(94)90555-X) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7954834/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell&title=Alpha%209:%20an%20acetylcholine%20receptor%20with%20novel%20pharmacological%20properties%20expressed%20in%20rat%20cochlear%20hair%20cells&author=AB%20Elgoyhen&author=DS%20Johnson&author=J%20Boulter&author=DE%20Vetter&author=S%20Heinemann&volume=79&publication_year=1994&pages=705-715&pmid=7954834&doi=10.1016/0092-8674(94)90555-X&)

11. Kuryatov A, Berrettini W, Lindstrom J. Acetylcholine receptor (AChR) alpha5 subunit variant associated with risk for nicotine dependence and lung cancer reduces (alpha4beta2)(2)alpha5 AChR function. Mol. Pharmacol. 2011;79:119–125. doi: 10.1124/mol.110.066357.  [DOI](https://doi.org/10.1124/mol.110.066357) | [PMC free article](/articles/PMC3014277/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20881005/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol.%20Pharmacol.&title=Acetylcholine%20receptor%20(AChR)%20alpha5%20subunit%20variant%20associated%20with%20risk%20for%20nicotine%20dependence%20and%20lung%20cancer%20reduces%20(alpha4beta2)(2)alpha5%20AChR%20function&author=A%20Kuryatov&author=W%20Berrettini&author=J%20Lindstrom&volume=79&publication_year=2011&pages=119-125&pmid=20881005&doi=10.1124/mol.110.066357&)

12. Yang J, Li MD. Converging findings from linkage and association analyses on susceptibility genes for smoking and other addictions. Mol. Psychiatry. 2016;21:992–1008. doi: 10.1038/mp.2016.67.  [DOI](https://doi.org/10.1038/mp.2016.67) | [PMC free article](/articles/PMC4956568/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27166759/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol.%20Psychiatry&title=Converging%20findings%20from%20linkage%20and%20association%20analyses%20on%20susceptibility%20genes%20for%20smoking%20and%20other%20addictions&author=J%20Yang&author=MD%20Li&volume=21&publication_year=2016&pages=992-1008&pmid=27166759&doi=10.1038/mp.2016.67&)

13. Saccone NL, et al. The CHRNA5-CHRNA3-CHRNB4 nicotinic receptor subunit gene cluster affects risk for nicotine dependence in African-Americans and in European-Americans. Cancer Res. 2009;69:6848–6856. doi: 10.1158/0008-5472.CAN-09-0786.  [DOI](https://doi.org/10.1158/0008-5472.CAN-09-0786) | [PMC free article](/articles/PMC2874321/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19706762/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res.&title=The%20CHRNA5-CHRNA3-CHRNB4%20nicotinic%20receptor%20subunit%20gene%20cluster%20affects%20risk%20for%20nicotine%20dependence%20in%20African-Americans%20and%20in%20European-Americans&author=NL%20Saccone&volume=69&publication_year=2009&pages=6848-6856&pmid=19706762&doi=10.1158/0008-5472.CAN-09-0786&)

14. Tobacco, Genetics C. Genome-wide meta-analyses identify multiple loci associated with smoking behavior. Nat. Genet. 2010;42:441–447. doi: 10.1038/ng.571.  [DOI](https://doi.org/10.1038/ng.571) | [PMC free article](/articles/PMC2914600/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20418890/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet.&title=Genome-wide%20meta-analyses%20identify%20multiple%20loci%20associated%20with%20smoking%20behavior&volume=42&publication_year=2010&pages=441-447&pmid=20418890&doi=10.1038/ng.571&)

15. Li MD, et al. Association and interaction analysis of variants in CHRNA5/CHRNA3/CHRNB4 gene cluster with nicotine dependence in African and European Americans. Am. J. Med. Genet. B Neuropsychiatr. Genet. 2010;153B:745–756. doi: 10.1002/ajmg.b.31043.  [DOI](https://doi.org/10.1002/ajmg.b.31043) | [PMC free article](/articles/PMC2924635/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19859904/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Med.%20Genet.%20B%20Neuropsychiatr.%20Genet.&title=Association%20and%20interaction%20analysis%20of%20variants%20in%20CHRNA5/CHRNA3/CHRNB4%20gene%20cluster%20with%20nicotine%20dependence%20in%20African%20and%20European%20Americans&author=MD%20Li&volume=153B&publication_year=2010&pages=745-756&pmid=19859904&doi=10.1002/ajmg.b.31043&)

16. Wen L, Jiang K, Yuan W, Cui W, Li MD. Contribution of variants in CHRNA5/A3/B4 gene cluster on chromosome 15 to tobacco smoking: from genetic association to mechanism. Mol. Neurobiol. 2016;53:472–484. doi: 10.1007/s12035-014-8997-x.  [DOI](https://doi.org/10.1007/s12035-014-8997-x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25471942/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol.%20Neurobiol.&title=Contribution%20of%20variants%20in%20CHRNA5/A3/B4%20gene%20cluster%20on%20chromosome%2015%20to%20tobacco%20smoking:%20from%20genetic%20association%20to%20mechanism&author=L%20Wen&author=K%20Jiang&author=W%20Yuan&author=W%20Cui&author=MD%20Li&volume=53&publication_year=2016&pages=472-484&pmid=25471942&doi=10.1007/s12035-014-8997-x&)

17. Berrettini W, et al. Alpha-5/alpha-3 nicotinic receptor subunit alleles increase risk for heavy smoking. Mol. Psychiatry. 2008;13:368–373. doi: 10.1038/sj.mp.4002154.  [DOI](https://doi.org/10.1038/sj.mp.4002154) | [PMC free article](/articles/PMC2507863/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18227835/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol.%20Psychiatry&title=Alpha-5/alpha-3%20nicotinic%20receptor%20subunit%20alleles%20increase%20risk%20for%20heavy%20smoking&author=W%20Berrettini&volume=13&publication_year=2008&pages=368-373&pmid=18227835&doi=10.1038/sj.mp.4002154&)

18. Thorgeirsson TE, et al. A variant associated with nicotine dependence, lung cancer and peripheral arterial disease. Nature. 2008;452:638–642. doi: 10.1038/nature06846.  [DOI](https://doi.org/10.1038/nature06846) | [PMC free article](/articles/PMC4539558/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18385739/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=A%20variant%20associated%20with%20nicotine%20dependence,%20lung%20cancer%20and%20peripheral%20arterial%20disease&author=TE%20Thorgeirsson&volume=452&publication_year=2008&pages=638-642&pmid=18385739&doi=10.1038/nature06846&)

19. Saccone NL, et al. Multiple distinct risk loci for nicotine dependence identified by dense coverage of the complete family of nicotinic receptor subunit (CHRN) genes. Am. J. Med. Genet. B Neuropsychiatr. Genet. 2009;150B:453–466. doi: 10.1002/ajmg.b.30828.  [DOI](https://doi.org/10.1002/ajmg.b.30828) | [PMC free article](/articles/PMC2693307/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19259974/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Med.%20Genet.%20B%20Neuropsychiatr.%20Genet.&title=Multiple%20distinct%20risk%20loci%20for%20nicotine%20dependence%20identified%20by%20dense%20coverage%20of%20the%20complete%20family%20of%20nicotinic%20receptor%20subunit%20(CHRN)%20genes&author=NL%20Saccone&volume=150B&publication_year=2009&pages=453-466&pmid=19259974&doi=10.1002/ajmg.b.30828&)

20. Saccone SF, et al. Cholinergic nicotinic receptor genes implicated in a nicotine dependence association study targeting 348 candidate genes with 3713 SNPs. Hum. Mol. Genet. 2007;16:36–49. doi: 10.1093/hmg/ddl438.  [DOI](https://doi.org/10.1093/hmg/ddl438) | [PMC free article](/articles/PMC2270437/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17135278/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum.%20Mol.%20Genet.&title=Cholinergic%20nicotinic%20receptor%20genes%20implicated%20in%20a%20nicotine%20dependence%20association%20study%20targeting%20348%20candidate%20genes%20with%203713%20SNPs&author=SF%20Saccone&volume=16&publication_year=2007&pages=36-49&pmid=17135278&doi=10.1093/hmg/ddl438&)

21. Lassi G, et al. The CHRNA5-A3-B4 gene cluster and smoking: from discovery to therapeutics. Trends Neurosci. 2016;39:851–861. doi: 10.1016/j.tins.2016.10.005.  [DOI](https://doi.org/10.1016/j.tins.2016.10.005) | [PMC free article](/articles/PMC5152594/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27871728/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Trends%20Neurosci.&title=The%20CHRNA5-A3-B4%20gene%20cluster%20and%20smoking:%20from%20discovery%20to%20therapeutics&author=G%20Lassi&volume=39&publication_year=2016&pages=851-861&pmid=27871728&doi=10.1016/j.tins.2016.10.005&)

22. Li MD, et al. Associations of variants in CHRNA5/A3/B4 gene cluster with smoking behaviors in a Korean population. PLoS ONE. 2010;5:e12183. doi: 10.1371/journal.pone.0012183.  [DOI](https://doi.org/10.1371/journal.pone.0012183) | [PMC free article](/articles/PMC2922326/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20808433/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&title=Associations%20of%20variants%20in%20CHRNA5/A3/B4%20gene%20cluster%20with%20smoking%20behaviors%20in%20a%20Korean%20population&author=MD%20Li&volume=5&publication_year=2010&pages=e12183&pmid=20808433&doi=10.1371/journal.pone.0012183&)

23. Liu JZ, et al. Meta-analysis and imputation refines the association of 15q25 with smoking quantity. Nat. Genet. 2010;42:436–440. doi: 10.1038/ng.572.  [DOI](https://doi.org/10.1038/ng.572) | [PMC free article](/articles/PMC3612983/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20418889/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet.&title=Meta-analysis%20and%20imputation%20refines%20the%20association%20of%2015q25%20with%20smoking%20quantity&author=JZ%20Liu&volume=42&publication_year=2010&pages=436-440&pmid=20418889&doi=10.1038/ng.572&)

24. David SP, et al. Genome-wide meta-analyses of smoking behaviors in African Americans. Transl. Psychiatry. 2012;2:e119. doi: 10.1038/tp.2012.41.  [DOI](https://doi.org/10.1038/tp.2012.41) | [PMC free article](/articles/PMC3365260/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22832964/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Transl.%20Psychiatry&title=Genome-wide%20meta-analyses%20of%20smoking%20behaviors%20in%20African%20Americans&author=SP%20David&volume=2&publication_year=2012&pages=e119&pmid=22832964&doi=10.1038/tp.2012.41&)

25. Thorgeirsson TE, et al. Sequence variants at CHRNB3-CHRNA6 and CYP2A6 affect smoking behavior. Nat. Genet. 2010;42:448–453. doi: 10.1038/ng.573.  [DOI](https://doi.org/10.1038/ng.573) | [PMC free article](/articles/PMC3080600/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20418888/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet.&title=Sequence%20variants%20at%20CHRNB3-CHRNA6%20and%20CYP2A6%20affect%20smoking%20behavior&author=TE%20Thorgeirsson&volume=42&publication_year=2010&pages=448-453&pmid=20418888&doi=10.1038/ng.573&)

26. Ghasemzadeh-Hasankolaei M, Batavani R, Eslaminejad MB, Sayahpour F. Transplantation of autologous bone marrow mesenchymal stem cells into the testes of infertile male rats and new germ cell formation. Int. J. Stem Cells. 2016;9:250–263. doi: 10.15283/ijsc16010.  [DOI](https://doi.org/10.15283/ijsc16010) | [PMC free article](/articles/PMC5155721/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27430978/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Stem%20Cells&title=Transplantation%20of%20autologous%20bone%20marrow%20mesenchymal%20stem%20cells%20into%20the%20testes%20of%20infertile%20male%20rats%20and%20new%20germ%20cell%20formation&author=M%20Ghasemzadeh-Hasankolaei&author=R%20Batavani&author=MB%20Eslaminejad&author=F%20Sayahpour&volume=9&publication_year=2016&pages=250-263&pmid=27430978&doi=10.15283/ijsc16010&)

27. Levin ED, et al. Nicotinic alpha7- or beta2-containing receptor knockout: effects on radial-arm maze learning and long-term nicotine consumption in mice. Behav. Brain Res. 2009;196:207–213. doi: 10.1016/j.bbr.2008.08.048.  [DOI](https://doi.org/10.1016/j.bbr.2008.08.048) | [PMC free article](/articles/PMC2638590/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18831991/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Behav.%20Brain%20Res.&title=Nicotinic%20alpha7-%20or%20beta2-containing%20receptor%20knockout:%20effects%20on%20radial-arm%20maze%20learning%20and%20long-term%20nicotine%20consumption%20in%20mice&author=ED%20Levin&volume=196&publication_year=2009&pages=207-213&pmid=18831991&doi=10.1016/j.bbr.2008.08.048&)

28. Brunzell DH, McIntosh JM. Alpha7 nicotinic acetylcholine receptors modulate motivation to self-administer nicotine: implications for smoking and schizophrenia. Neuropsychopharmacology. 2012;37:1134–1143. doi: 10.1038/npp.2011.299.  [DOI](https://doi.org/10.1038/npp.2011.299) | [PMC free article](/articles/PMC3306875/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22169946/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neuropsychopharmacology&title=Alpha7%20nicotinic%20acetylcholine%20receptors%20modulate%20motivation%20to%20self-administer%20nicotine:%20implications%20for%20smoking%20and%20schizophrenia&author=DH%20Brunzell&author=JM%20McIntosh&volume=37&publication_year=2012&pages=1134-1143&pmid=22169946&doi=10.1038/npp.2011.299&)

29. Brunzell DH, Boschen KE, Hendrick ES, Beardsley PM, McIntosh JM. Alpha-conotoxin MII-sensitive nicotinic acetylcholine receptors in the nucleus accumbens shell regulate progressive ratio responding maintained by nicotine. Neuropsychopharmacology. 2010;35:665–673. doi: 10.1038/npp.2009.171.  [DOI](https://doi.org/10.1038/npp.2009.171) | [PMC free article](/articles/PMC2821821/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19890263/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neuropsychopharmacology&title=Alpha-conotoxin%20MII-sensitive%20nicotinic%20acetylcholine%20receptors%20in%20the%20nucleus%20accumbens%20shell%20regulate%20progressive%20ratio%20responding%20maintained%20by%20nicotine&author=DH%20Brunzell&author=KE%20Boschen&author=ES%20Hendrick&author=PM%20Beardsley&author=JM%20McIntosh&volume=35&publication_year=2010&pages=665-673&pmid=19890263&doi=10.1038/npp.2009.171&)

30. Melis M, et al. PPARalpha regulates cholinergic-driven activity of midbrain dopamine neurons via a novel mechanism involving alpha7 nicotinic acetylcholine receptors. J. Neurosci. 2013;33:6203–6211. doi: 10.1523/JNEUROSCI.4647-12.2013.  [DOI](https://doi.org/10.1523/JNEUROSCI.4647-12.2013) | [PMC free article](/articles/PMC6618938/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23554501/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Neurosci.&title=PPARalpha%20regulates%20cholinergic-driven%20activity%20of%20midbrain%20dopamine%20neurons%20via%20a%20novel%20mechanism%20involving%20alpha7%20nicotinic%20acetylcholine%20receptors&author=M%20Melis&volume=33&publication_year=2013&pages=6203-6211&pmid=23554501&doi=10.1523/JNEUROSCI.4647-12.2013&)

31. Breitling LP, Yang R, Korn B, Burwinkel B, Brenner H. Tobacco-smoking-related differential DNA methylation: 27K discovery and replication. Am. J. Hum. Genet. 2011;88:450–457. doi: 10.1016/j.ajhg.2011.03.003.  [DOI](https://doi.org/10.1016/j.ajhg.2011.03.003) | [PMC free article](/articles/PMC3071918/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21457905/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Hum.%20Genet.&title=Tobacco-smoking-related%20differential%20DNA%20methylation:%2027K%20discovery%20and%20replication&author=LP%20Breitling&author=R%20Yang&author=B%20Korn&author=B%20Burwinkel&author=H%20Brenner&volume=88&publication_year=2011&pages=450-457&pmid=21457905&doi=10.1016/j.ajhg.2011.03.003&)

32. Furniss CS, Marsit CJ, Houseman EA, Eddy K, Kelsey KT. Line region hypomethylation is associated with lifestyle and differs by human papillomavirus status in head and neck squamous cell carcinomas. Cancer Epidemiol. Biomark. Prev. 2008;17:966–971. doi: 10.1158/1055-9965.EPI-07-2775.  [DOI](https://doi.org/10.1158/1055-9965.EPI-07-2775) | [PMC free article](/articles/PMC2633874/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18398038/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Epidemiol.%20Biomark.%20Prev.&title=Line%20region%20hypomethylation%20is%20associated%20with%20lifestyle%20and%20differs%20by%20human%20papillomavirus%20status%20in%20head%20and%20neck%20squamous%20cell%20carcinomas&author=CS%20Furniss&author=CJ%20Marsit&author=EA%20Houseman&author=K%20Eddy&author=KT%20Kelsey&volume=17&publication_year=2008&pages=966-971&pmid=18398038&doi=10.1158/1055-9965.EPI-07-2775&)

33. Bell JT, Spector TD. DNA methylation studies using twins: what are they telling us? Genome Biol. 2012;13:172. doi: 10.1186/gb-2012-13-10-172.  [DOI](https://doi.org/10.1186/gb-2012-13-10-172) | [PMC free article](/articles/PMC3491399/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23078798/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Biol.&title=DNA%20methylation%20studies%20using%20twins:%20what%20are%20they%20telling%20us?&author=JT%20Bell&author=TD%20Spector&volume=13&publication_year=2012&pages=172&pmid=23078798&doi=10.1186/gb-2012-13-10-172&)

34. Liu Y, et al. Epigenome-wide association data implicate DNA methylation as an intermediary of genetic risk in rheumatoid arthritis. Nat. Biotechnol. 2013;31:142–147. doi: 10.1038/nbt.2487.  [DOI](https://doi.org/10.1038/nbt.2487) | [PMC free article](/articles/PMC3598632/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23334450/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Biotechnol.&title=Epigenome-wide%20association%20data%20implicate%20DNA%20methylation%20as%20an%20intermediary%20of%20genetic%20risk%20in%20rheumatoid%20arthritis&author=Y%20Liu&volume=31&publication_year=2013&pages=142-147&pmid=23334450&doi=10.1038/nbt.2487&)

35. Jaffe AE, et al. Mapping DNA methylation across development, genotype and schizophrenia in the human frontal cortex. Nat. Neurosci. 2016;19:40–47. doi: 10.1038/nn.4181.  [DOI](https://doi.org/10.1038/nn.4181) | [PMC free article](/articles/PMC4783176/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26619358/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Neurosci.&title=Mapping%20DNA%20methylation%20across%20development,%20genotype%20and%20schizophrenia%20in%20the%20human%20frontal%20cortex&author=AE%20Jaffe&volume=19&publication_year=2016&pages=40-47&pmid=26619358&doi=10.1038/nn.4181&)

36. Hancock DB, et al. A multiancestry study identifies novel genetic associations with CHRNA5 methylation in human brain and risk of nicotine dependence. Hum. Mol. Genet. 2015;24:5940–5954. doi: 10.1093/hmg/ddv303.  [DOI](https://doi.org/10.1093/hmg/ddv303) | [PMC free article](/articles/PMC4581607/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26220977/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum.%20Mol.%20Genet.&title=A%20multiancestry%20study%20identifies%20novel%20genetic%20associations%20with%20CHRNA5%20methylation%20in%20human%20brain%20and%20risk%20of%20nicotine%20dependence&author=DB%20Hancock&volume=24&publication_year=2015&pages=5940-5954&pmid=26220977&doi=10.1093/hmg/ddv303&)

37. Roadmap Epigenomics C, et al. Integrative analysis of 111 reference human epigenomes. Nature. 2015;518:317–330. doi: 10.1038/nature14248.  [DOI](https://doi.org/10.1038/nature14248) | [PMC free article](/articles/PMC4530010/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25693563/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Integrative%20analysis%20of%20111%20reference%20human%20epigenomes&author=C%20Roadmap%20Epigenomics&volume=518&publication_year=2015&pages=317-330&pmid=25693563&doi=10.1038/nature14248&)

38. Fagerstrom KO. Measuring degree of physical dependence to tobacco smoking with reference to individualization of treatment. Addict. Behav. 1978;3:235–241. doi: 10.1016/0306-4603(78)90024-2.  [DOI](https://doi.org/10.1016/0306-4603(78)90024-2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/735910/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Addict.%20Behav.&title=Measuring%20degree%20of%20physical%20dependence%20to%20tobacco%20smoking%20with%20reference%20to%20individualization%20of%20treatment&author=KO%20Fagerstrom&volume=3&publication_year=1978&pages=235-241&pmid=735910&doi=10.1016/0306-4603(78)90024-2&)

39. Heatherton TF, Kozlowski LT, Frecker RC, Fagerstrom KO. The fagerstrom test for nicotine dependence: a revision of the Fagerstrom Tolerance Questionnaire. Br. J. Addict. 1991;86:1119–1127. doi: 10.1111/j.1360-0443.1991.tb01879.x.  [DOI](https://doi.org/10.1111/j.1360-0443.1991.tb01879.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/1932883/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br.%20J.%20Addict.&title=The%20fagerstrom%20test%20for%20nicotine%20dependence:%20a%20revision%20of%20the%20Fagerstrom%20Tolerance%20Questionnaire&author=TF%20Heatherton&author=LT%20Kozlowski&author=RC%20Frecker&author=KO%20Fagerstrom&volume=86&publication_year=1991&pages=1119-1127&pmid=1932883&doi=10.1111/j.1360-0443.1991.tb01879.x&)

40. Wei J, et al. Association study of 45 candidate genes in nicotine dependence in Han Chinese. Addict. Behav. 2012;37:622–626. doi: 10.1016/j.addbeh.2012.01.009.  [DOI](https://doi.org/10.1016/j.addbeh.2012.01.009) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22309839/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Addict.%20Behav.&title=Association%20study%20of%2045%20candidate%20genes%20in%20nicotine%20dependence%20in%20Han%20Chinese&author=J%20Wei&volume=37&publication_year=2012&pages=622-626&pmid=22309839&doi=10.1016/j.addbeh.2012.01.009&)

41. Falush D, Stephens M, Pritchard JK. Inference of population structure using multilocus genotype data: linked loci and correlated allele frequencies. Genetics. 2003;164:1567–1587. doi: 10.1093/genetics/164.4.1567.  [DOI](https://doi.org/10.1093/genetics/164.4.1567) | [PMC free article](/articles/PMC1462648/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12930761/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genetics&title=Inference%20of%20population%20structure%20using%20multilocus%20genotype%20data:%20linked%20loci%20and%20correlated%20allele%20frequencies&author=D%20Falush&author=M%20Stephens&author=JK%20Pritchard&volume=164&publication_year=2003&pages=1567-1587&pmid=12930761&doi=10.1093/genetics/164.4.1567&)

42. Yang BZ, Zhao H, Kranzler HR, Gelernter J. Practical population group assignment with selected informative markers: characteristics and properties of Bayesian clustering via STRUCTURE. Genet. Epidemiol. 2005;28:302–312. doi: 10.1002/gepi.20070.  [DOI](https://doi.org/10.1002/gepi.20070) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15782414/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genet.%20Epidemiol.&title=Practical%20population%20group%20assignment%20with%20selected%20informative%20markers:%20characteristics%20and%20properties%20of%20Bayesian%20clustering%20via%20STRUCTURE&author=BZ%20Yang&author=H%20Zhao&author=HR%20Kranzler&author=J%20Gelernter&volume=28&publication_year=2005&pages=302-312&pmid=15782414&doi=10.1002/gepi.20070&)

43. Krueger F, Andrews SR. Bismark: a flexible aligner and methylation caller for Bisulfite-Seq applications. Bioinformatics. 2011;27:1571–1572. doi: 10.1093/bioinformatics/btr167.  [DOI](https://doi.org/10.1093/bioinformatics/btr167) | [PMC free article](/articles/PMC3102221/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21493656/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=Bismark:%20a%20flexible%20aligner%20and%20methylation%20caller%20for%20Bisulfite-Seq%20applications&author=F%20Krueger&author=SR%20Andrews&volume=27&publication_year=2011&pages=1571-1572&pmid=21493656&doi=10.1093/bioinformatics/btr167&)

44. Purcell S, et al. PLINK: a tool set for whole-genome association and population-based linkage analyses. Am. J. Hum. Genet. 2007;81:559–575. doi: 10.1086/519795.  [DOI](https://doi.org/10.1086/519795) | [PMC free article](/articles/PMC1950838/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17701901/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Hum.%20Genet.&title=PLINK:%20a%20tool%20set%20for%20whole-genome%20association%20and%20population-based%20linkage%20analyses&author=S%20Purcell&volume=81&publication_year=2007&pages=559-575&pmid=17701901&doi=10.1086/519795&)

45. Barrett JC, Fry B, Maller J, Daly MJ. Haploview: analysis and visualization of LD and haplotype maps. Bioinformatics. 2005;21:263–265. doi: 10.1093/bioinformatics/bth457.  [DOI](https://doi.org/10.1093/bioinformatics/bth457) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15297300/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=Haploview:%20analysis%20and%20visualization%20of%20LD%20and%20haplotype%20maps&author=JC%20Barrett&author=B%20Fry&author=J%20Maller&author=MJ%20Daly&volume=21&publication_year=2005&pages=263-265&pmid=15297300&doi=10.1093/bioinformatics/bth457&)

46. Gabriel SB, et al. The structure of haplotype blocks in the human genome. Science. 2002;296:2225–2229. doi: 10.1126/science.1069424.  [DOI](https://doi.org/10.1126/science.1069424) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12029063/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=The%20structure%20of%20haplotype%20blocks%20in%20the%20human%20genome&author=SB%20Gabriel&volume=296&publication_year=2002&pages=2225-2229&pmid=12029063&doi=10.1126/science.1069424&)

47. Schaid DJ, Rowland CM, Tines DE, Jacobson RM, Poland GA. Score tests for association between traits and haplotypes when linkage phase is ambiguous. Am. J. Hum. Genet. 2002;70:425–434. doi: 10.1086/338688.  [DOI](https://doi.org/10.1086/338688) | [PMC free article](/articles/PMC384917/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11791212/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Hum.%20Genet.&title=Score%20tests%20for%20association%20between%20traits%20and%20haplotypes%20when%20linkage%20phase%20is%20ambiguous&author=DJ%20Schaid&author=CM%20Rowland&author=DE%20Tines&author=RM%20Jacobson&author=GA%20Poland&volume=70&publication_year=2002&pages=425-434&pmid=11791212&doi=10.1086/338688&)

48. Zhu Z, et al. Development of GMDR-GPU for gene-gene interaction analysis and its application to WTCCC GWAS data for type 2 diabetes. PLoS ONE. 2013;8:e61943. doi: 10.1371/journal.pone.0061943.  [DOI](https://doi.org/10.1371/journal.pone.0061943) | [PMC free article](/articles/PMC3633958/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23626757/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&title=Development%20of%20GMDR-GPU%20for%20gene-gene%20interaction%20analysis%20and%20its%20application%20to%20WTCCC%20GWAS%20data%20for%20type%202%20diabetes&author=Z%20Zhu&volume=8&publication_year=2013&pages=e61943&pmid=23626757&doi=10.1371/journal.pone.0061943&)

49. Lou XY, et al. A generalized combinatorial approach for detecting gene-by-gene and gene-by-environment interactions with application to nicotine dependence. Am. J. Hum. Genet. 2007;80:1125–1137. doi: 10.1086/518312.  [DOI](https://doi.org/10.1086/518312) | [PMC free article](/articles/PMC1867100/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17503330/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Hum.%20Genet.&title=A%20generalized%20combinatorial%20approach%20for%20detecting%20gene-by-gene%20and%20gene-by-environment%20interactions%20with%20application%20to%20nicotine%20dependence&author=XY%20Lou&volume=80&publication_year=2007&pages=1125-1137&pmid=17503330&doi=10.1086/518312&)

50. Shabalin AA. Matrix eQTL: ultra fast eQTL analysis via large matrix operations. Bioinformatics. 2012;28:1353–1358. doi: 10.1093/bioinformatics/bts163.  [DOI](https://doi.org/10.1093/bioinformatics/bts163) | [PMC free article](/articles/PMC3348564/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22492648/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=Matrix%20eQTL:%20ultra%20fast%20eQTL%20analysis%20via%20large%20matrix%20operations&author=AA%20Shabalin&volume=28&publication_year=2012&pages=1353-1358&pmid=22492648&doi=10.1093/bioinformatics/bts163&)

51. Consortium GT. Human genomics. The Genotype-Tissue Expression (GTEx) pilot analysis: multitissue gene regulation in humans. Science. 2015;348:648–660. doi: 10.1126/science.1262110.  [DOI](https://doi.org/10.1126/science.1262110) | [PMC free article](/articles/PMC4547484/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25954001/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=Human%20genomics.%20The%20Genotype-Tissue%20Expression%20(GTEx)%20pilot%20analysis:%20multitissue%20gene%20regulation%20in%20humans&author=GT%20Consortium&volume=348&publication_year=2015&pages=648-660&pmid=25954001&doi=10.1126/science.1262110&)

52. Westra HJ, et al. Systematic identification of trans eQTLs as putative drivers of known disease associations. Nat. Genet. 2013;45:1238–1243. doi: 10.1038/ng.2756.  [DOI](https://doi.org/10.1038/ng.2756) | [PMC free article](/articles/PMC3991562/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24013639/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet.&title=Systematic%20identification%20of%20trans%20eQTLs%20as%20putative%20drivers%20of%20known%20disease%20associations&author=HJ%20Westra&volume=45&publication_year=2013&pages=1238-1243&pmid=24013639&doi=10.1038/ng.2756&)

53. Cui WY, et al. Identification and characterization of poly(I:C)-induced molecular responses attenuated by nicotine in mouse macrophages. Mol. Pharmacol. 2013;83:61–72. doi: 10.1124/mol.112.081497.  [DOI](https://doi.org/10.1124/mol.112.081497) | [PMC free article](/articles/PMC3533466/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23028093/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol.%20Pharmacol.&title=Identification%20and%20characterization%20of%20poly(I:C)-induced%20molecular%20responses%20attenuated%20by%20nicotine%20in%20mouse%20macrophages&author=WY%20Cui&volume=83&publication_year=2013&pages=61-72&pmid=23028093&doi=10.1124/mol.112.081497&)

54. Alexandrov LB, et al. Mutational signatures associated with tobacco smoking in human cancer. Science. 2016;354:618–622. doi: 10.1126/science.aag0299.  [DOI](https://doi.org/10.1126/science.aag0299) | [PMC free article](/articles/PMC6141049/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27811275/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=Mutational%20signatures%20associated%20with%20tobacco%20smoking%20in%20human%20cancer&author=LB%20Alexandrov&volume=354&publication_year=2016&pages=618-622&pmid=27811275&doi=10.1126/science.aag0299&)

55. Ma Y, Li MD. Establishment of a strong link between smoking and cancer pathogenesis through DNA methylation analysis. Sci. Rep. 2017;7:1811. doi: 10.1038/s41598-017-01856-4.  [DOI](https://doi.org/10.1038/s41598-017-01856-4) | [PMC free article](/articles/PMC5431893/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28500316/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep.&title=Establishment%20of%20a%20strong%20link%20between%20smoking%20and%20cancer%20pathogenesis%20through%20DNA%20methylation%20analysis&author=Y%20Ma&author=MD%20Li&volume=7&publication_year=2017&pages=1811&pmid=28500316&doi=10.1038/s41598-017-01856-4&)

56. van Eijk KR, et al. Genetic analysis of DNA methylation and gene expression levels in whole blood of healthy human subjects. BMC Genomics. 2012;13:636. doi: 10.1186/1471-2164-13-636.  [DOI](https://doi.org/10.1186/1471-2164-13-636) | [PMC free article](/articles/PMC3583143/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23157493/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Genomics&title=Genetic%20analysis%20of%20DNA%20methylation%20and%20gene%20expression%20levels%20in%20whole%20blood%20of%20healthy%20human%20subjects&author=KR%20van%20Eijk&volume=13&publication_year=2012&pages=636&pmid=23157493&doi=10.1186/1471-2164-13-636&)

57. Wang Y, Peng X, Zhu L, Hu L, Song Y. Genetic variants of CHRNA5-A3 and CHRNB3-A6 predict survival of patients with advanced non-small cell lung cancer. Oncotarget. 2016;7:26436–26443. doi: 10.18632/oncotarget.8510.  [DOI](https://doi.org/10.18632/oncotarget.8510) | [PMC free article](/articles/PMC5041990/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27050379/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncotarget&title=Genetic%20variants%20of%20CHRNA5-A3%20and%20CHRNB3-A6%20predict%20survival%20of%20patients%20with%20advanced%20non-small%20cell%20lung%20cancer&author=Y%20Wang&author=X%20Peng&author=L%20Zhu&author=L%20Hu&author=Y%20Song&volume=7&publication_year=2016&pages=26436-26443&pmid=27050379&doi=10.18632/oncotarget.8510&)

58. Wu C, et al. Genetic variants on chromosome 15q25 associated with lung cancer risk in Chinese populations. Cancer Res. 2009;69:5065–5072. doi: 10.1158/0008-5472.CAN-09-0081.  [DOI](https://doi.org/10.1158/0008-5472.CAN-09-0081) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19491260/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res.&title=Genetic%20variants%20on%20chromosome%2015q25%20associated%20with%20lung%20cancer%20risk%20in%20Chinese%20populations&author=C%20Wu&volume=69&publication_year=2009&pages=5065-5072&pmid=19491260&doi=10.1158/0008-5472.CAN-09-0081&)

59. Frahm S, et al. Aversion to nicotine is regulated by the balanced activity of beta4 and alpha5 nicotinic receptor subunits in the medial habenula. Neuron. 2011;70:522–535. doi: 10.1016/j.neuron.2011.04.013.  [DOI](https://doi.org/10.1016/j.neuron.2011.04.013) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21555077/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neuron&title=Aversion%20to%20nicotine%20is%20regulated%20by%20the%20balanced%20activity%20of%20beta4%20and%20alpha5%20nicotinic%20receptor%20subunits%20in%20the%20medial%20habenula&author=S%20Frahm&volume=70&publication_year=2011&pages=522-535&pmid=21555077&doi=10.1016/j.neuron.2011.04.013&)

60. Bierut LJ, et al. Variants in nicotinic receptors and risk for nicotine dependence. Am. J. Psychiatry. 2008;165:1163–1171. doi: 10.1176/appi.ajp.2008.07111711.  [DOI](https://doi.org/10.1176/appi.ajp.2008.07111711) | [PMC free article](/articles/PMC2574742/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18519524/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Psychiatry&title=Variants%20in%20nicotinic%20receptors%20and%20risk%20for%20nicotine%20dependence&author=LJ%20Bierut&volume=165&publication_year=2008&pages=1163-1171&pmid=18519524&doi=10.1176/appi.ajp.2008.07111711&)

61. Pedneault M, et al. The association between CHRN genetic variants and dizziness at first inhalation of cigarette smoke. Addict. Behav. 2014;39:316–320. doi: 10.1016/j.addbeh.2013.08.033.  [DOI](https://doi.org/10.1016/j.addbeh.2013.08.033) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24119711/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Addict.%20Behav.&title=The%20association%20between%20CHRN%20genetic%20variants%20and%20dizziness%20at%20first%20inhalation%20of%20cigarette%20smoke&author=M%20Pedneault&volume=39&publication_year=2014&pages=316-320&pmid=24119711&doi=10.1016/j.addbeh.2013.08.033&)

62. McClay JL, et al. High density methylation QTL analysis in human blood via next-generation sequencing of the methylated genomic DNA fraction. Genome Biol. 2015;16:291. doi: 10.1186/s13059-015-0842-7.  [DOI](https://doi.org/10.1186/s13059-015-0842-7) | [PMC free article](/articles/PMC4699364/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26699738/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Biol.&title=High%20density%20methylation%20QTL%20analysis%20in%20human%20blood%20via%20next-generation%20sequencing%20of%20the%20methylated%20genomic%20DNA%20fraction&author=JL%20McClay&volume=16&publication_year=2015&pages=291&pmid=26699738&doi=10.1186/s13059-015-0842-7&)
