# Genomic correlates of glatiramer acetate adverse cardiovascular effects lead to a novel locus mediating coronary risk

## Metadata
**Authors:** Ingrid Brænne, Lingyao Zeng, Christina Willenborg, Vinicius Tragante, Thorsten Kessler, CARDIoGRAM Consortium, CARDIoGRAMplusC4D Consortium, Cristen J Willer, Markku Laakso, Lars Wallentin, Paul W Franks, Veikko Salomaa, Abbas Dehghan, Thomas Meitinger, Nilesh J Samani, Folkert W Asselbergs, Jeanette Erdmann, Heribert Schunkert
**Journal:** PLoS ONE
**Date:** 2017 Aug 22
**DOI:** [10.1371/journal.pone.0182999](https://doi.org/10.1371/journal.pone.0182999)
**PMID:** 28829817
**PMCID:** PMC5567477
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5567477/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC5567477/pdf/pone.0182999.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC5567477/pdf/pone.0182999.pdf)

## Abstract

Glatiramer acetate is used therapeutically in multiple sclerosis but also known for adverse effects including elevated coronary artery disease (CAD) risk. The mechanisms underlying the cardiovascular side effects of the medication are unclear. Here, we made use of the chromosomal variation in the genes that are known to be affected by glatiramer treatment. Focusing on genes and gene products reported by drug-gene interaction database to interact with glatiramer acetate we explored a large meta-analysis on CAD genome-wide association studies aiming firstly, to investigate whether variants in these genes also affect cardiovascular risk and secondly, to identify new CAD risk genes. We traced association signals in a 200-kb region around genomic positions of genes interacting with glatiramer in up to 60 801 CAD cases and 123 504 controls. We validated the identified association in additional 21 934 CAD cases and 76 087 controls. We identified three new CAD risk alleles within the TGFB1 region on chromosome 19 that independently affect CAD risk. The lead SNP rs12459996 was genome-wide significantly associated with CAD in the extended meta-analysis (odds ratio 1.09, p = 1.58×10−12). The other two SNPs at the locus were not in linkage disequilibrium with the lead SNP and by a conditional analysis showed p-values of 4.05 × 10−10 and 2.21 × 10−6. Thus, studying genes reported to interact with glatiramer acetate we identified genetic variants that concordantly with the drug increase the risk of CAD. Of these, TGFB1 displayed signal for association. Indeed, the gene has been associated with CAD previously in both in vivo and in vitro studies. Here we establish genome-wide significant association with CAD in large human samples.

## Introduction

Glatiramer acetate (GA), also known under the trade name Copaxone®, is an immunomodulator used in the treatment of relapsing-remitting multiple sclerosis (MS). It is a synthetic peptide consisting of four amino acids[[1](#pone.0182999.ref001), [2](#pone.0182999.ref002)]. GA is assumed to bind major histocompatibility complex (MHC) molecules and compete with various myelin antigens[[2](#pone.0182999.ref002)]. This competitive binding affects the presentation of myelin antigens to T-cells. In addition, GA potentially promotes suppressor T-cells[[1](#pone.0182999.ref001), [3](#pone.0182999.ref003)]. A further mechanism of action is that GA-induced T helper cells secrete high amounts of cytokines such as IL-4/10 and TGF-β[[2](#pone.0182999.ref002)]. In mice, GA is reported to induce transforming growth factor β 1 (TGFB1) in several cell types such as monocytes[[4](#pone.0182999.ref004)], Th2/3 cells[[5](#pone.0182999.ref005)] and brain[[6](#pone.0182999.ref006)]. In addition, it is known that polymorphisms in the TGFB1 gene alter the GA treatment response[[7](#pone.0182999.ref007)].

According to the FDA,[drugs.com](http://drugs.com) and the copaxone own webpage ([copaxone.com](http://copaxone.com)), GA is reported to induce hypertension and increase the risk of coronary artery disease (CAD) and myocardial infarction. The exact mechanisms that explain the increased risk of CAD or hypertension under GA treatment are, however, not fully understood.

Understanding the genetic mechanisms underlying a disease can facilitate the identification of new drug targets. Indeed, several drugs have been developed based on genetic findings [[8](#pone.0182999.ref008)–[10](#pone.0182999.ref010)]. Moreover, if we know which genes or pathways are targeted by a drug, we may also predict adverse effects based on variants in these genes modulating risk[[11](#pone.0182999.ref011), [12](#pone.0182999.ref012)]. Here, we reversed this approach to identify new disease risk genes. We screened genes that are reported to interact with a drug that increases the risk of CAD to identify new CAD risk alleles. In a previous study, we identified new CAD risk genes by studying the pleiotropic effects of cyclooxygenase 2 inhibitors[[13](#pone.0182999.ref013)].

In this work, we first identified genes and gene products reported to interact with GA. These genes were then screened for association in the largest meta-analysis on CAD genome-wide association studies (GWAS), CARDIoGRAMplusC4D 1000G[[14](#pone.0182999.ref014)]. The underlying idea is that single nucleotide polymorphisms (SNPs) in *cis* with these genes may affect expression or structure of these genes in a similar way as the drug. Hence, we expect that some of these SNPs also may increase the risk of CAD, even though the effect size might vary between drug and variant.

## Methods

The first step of the analysis was to identify genes or gene products reported to interact with GA. For this, we used the Drug Gene Interaction Database (DGIdb)[[15](#pone.0182999.ref015)]. Second, in a large CAD GWAS dataset, we identified all SNPs within 200kb surrounding the four genes identified in the first step of this analysis. The 200kb window was selected to also include regulatory SNPs affecting gene expression.

### GWAS dataset

The CARDIoGRAMplusC4D 1000Genomes meta-analysis data set consists of 47 GWAS studies including 60 801 CAD cases and 123 504 controls. Ethical approval was obtained from the appropriate ethics committees and informed consent was obtained from all participants. Specifically, the studies involving genome-wide SNP analysis for CAD were approved by the ethics commissions of the University of Regensburg (02/042), the University of Lübeck (04/041) und the Technical University of Munich (406/15s). The GWAS are imputed with the December 2012 1000Genomes phase I integrated haplotypes ([ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/release/20110521/](https://ftp.1000genomes.ebi.ac.uk/vol1/ftp/release/20110521/)) (for Methods see Nikpay et al.[[14](#pone.0182999.ref014)]).

We validated and later combined the CARDIoGRAMplusC4D 1000Genomes meta-analysis data set[[14](#pone.0182999.ref014)] with GWAS data from CHARGE, deCODE CAD, DILGOM, EPIC, FRISC II GLACIER, METISM, MORGAM FIN, MORGAM FRA, MORGAM GER, MORGAM ITA, MORGAM UNK, PMB, PopGen, SCARF SHEEP, and STR that have been previously reported in references Schunkert et al. [[16](#pone.0182999.ref016)] and/or Deloukas et at. [[17](#pone.0182999.ref017)]. Moreover, we included data from GWAS not reported before, i.e. German MI Family Studies V. See detailed information in [S1 Table](#pone.0182999.s003). In total, this combined dataset consists of 82 735 cases and 199 591 controls. Compared to the CARDIoGRAMplusC4D study, we increase the sample size by 21.934 cases and 76.087 controls.

### Meta-analysis

Logistic regression, assuming an additive model, was performed on all single study data. All analyses were adjusted for sex and age. Age was defined as the recruitment age for controls and the event age for cases. We used the fixed-effect inverse variance-weighted meta-analysis to combine single analyses data. Quality control was performed at individual sites and centrally to assure standardized data formats previously agreed criteria including check of consistency of the given alleles across all studies, quality of the imputation, deviation from Hardy-Weinberg equilibrium and call rate. If individuals or single studies did not pass quality control, they were excluded. SNPs were also excluded from the meta-analysis if present in less than 17 GWAS.

For the meta-analysis, we calculated an 'inverse variance weighting'- fixed-effects and a random effects model[[18](#pone.0182999.ref018)], depending on the heterogeneity between the studies. For heterogeneity calculation, Cochran’s Q was used. The threshold for heterogeneity was phet<0.01. For the combination of the stages (stage1: results of 1000G meta-analysis; stage 2: replication in CARDIoGRAM, CARDIoGRAMplusC4D, GerMIFS V) an 'inverse variance weighting'- the fixed-effects model was calculated and the combined effects and p-values were reported. In total, we evaluated the genomic data from 82 735 cases and 199 591 controls.

### Statistical methods

The number of SNPs tested in the initial screen (for the four genes) is 20,027. We corrected the p-value threshold based on the number of SNPs tested using the Bonferroni correction. Hence, all SNPs with p-values below 2.5x10^-6^ were considered significant.

### Identification of TGFB1 sub-loci

We used Haploreg version 4.1[[19](#pone.0182999.ref019)] with the European 1000G Phase 1 database for LD calculation. We identified LD blocks based on LD > 0.4 to the lead SNPs. In detail, we repeated the following three steps until no sub-loci with a p-value below 1×10^−6^ were found.

To test for independence between the identified sub-loci, we performed conditional analysis using summary statistic data with the GCTA tool[[20](#pone.0182999.ref020)]. As reference data used the GerMIFs II study. For conditional analysis, we used the SNPs identified in the above-described sub-loci analysis. We first performed a joint analysis using the–cojo-joint option and then performed the conditional analysis (—cojo-cond) based on the independent SNPs identified in the joint analysis step.

### Functional annotation of SNPs and TGFB1

To evaluate the functional implication of the SNPs, we identified all SNPs in high LD (r^2^>0.8) with the locus lead SNP using the HaploReg version 4.1 database[[19](#pone.0182999.ref019)]. To estimate the effect of a SNP on gene expression, we identified expression quantitative trait loci (eQTLs) using the publicly available data from Westra et al.[[21](#pone.0182999.ref021)], GTeX[[22](#pone.0182999.ref022)] as well as over 100 studies included in the Genome-Wide Repository of Associations between SNPs and Phenotypes (GRASP) database[[23](#pone.0182999.ref023)]. In addition, we used HaploReg and RegulomeDB[[24](#pone.0182999.ref024)] to functionally annotate SNPs and performed a literature search for gene functions using Pubmed.

## Results

The principle idea of this approach is illustrated in [Fig 1](#pone.0182999.g001). Using DGIdb[[15](#pone.0182999.ref015)], we identified four genes reported to interact with GA; CCR5, HLA-DRB1, IFNAR1, and TGFB1. Of these, only the TGFB1 region displayed signals suggesting an association with CAD risk ([Fig 2](#pone.0182999.g002)). We validated the GA-TGFB1 interaction performing a literature search[[2](#pone.0182999.ref002), [4](#pone.0182999.ref004)–[6](#pone.0182999.ref006)]. Moreover, from a mechanistic point of view, TGFB1 is the only gene with evidence of a functional association with CAD (see [S1 Fig](#pone.0182999.s001)). Hence, we did not examine the other genes further.

### Fig 1. Schematic approach.

![Fig 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a652/5567477/f6f50ba8d031/pone.0182999.g001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5567477_pone.0182999.g001.jpg)

1) Identification of reported adverse effect of GA 2) Identify genes reported to interact with GA. 3) Establish a link between the genes identified in 2. and the adverse effect identified in 1).

### Fig 2. GA interacts with CCR, TGFB1, IFNAR1 and HLA-DRB1 (solid line).

![Fig 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a652/5567477/5aab669afa41/pone.0182999.g002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5567477_pone.0182999.g002.jpg)

Moreover, it is known that GA affects CAD (Coronary Artery Disease) risk (dashed line). In this work, we searched for SNPs associated with CAD in the gene regions representing the GA off target effects (dotted lines). We found a genome-wide significant association for the TGFB1 locus with a p-value of 1.58 × 10−12 (red dotted line). n.s.: non-significant; TGFB1: Transforming Growth Factor, Beta 1; CCR5: Chemokine (C-C Motif) Receptor 5 (Gene/Pseudogene); IFNAR1: Interferon (Alpha, Beta And Omega) Receptor 1; HLA-DRB1: Major Histocompatibility Complex, Class II, DR Beta 1.

In our first GWAS look-up, the lead SNP, rs15052, close to TGFB1 yielded a p-value of 2.21 × 10^−7^. In a replication study with independent samples, rs15052 showed a p-value of 9.97 × 10^−4^, hence validating the initial finding. We next combined the data sets and reran the analysis. In this combined meta-analysis, rs15052 yielded a genome-wide significant p-value of 9.11 × 10^−10^. The new lead SNP in the joint meta-analysis, rs12459996 had a p-value of 1.58 × 10^−12^ and an OR of 1.09 (see [Fig 3](#pone.0182999.g003)).

### Fig 3. Association sub-loci signal for the TGFB1 locus.

![Fig 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a652/5567477/320451356615/pone.0182999.g003.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5567477_pone.0182999.g003.jpg)

The three lead SNPs are shown with the corresponding high-LD blocks (SNPs within r2>0.2.) depicted in orange, red and green. Independent sub-loci were identified with the GCTA conditional analysis tool (see methods). The LD between the lead SNPs indicated and under r2<0.1. The three individual LocusZoom plots are found in the S2 Fig.

The TGFB1 locus presumably harbors several independent sub-loci (see [Table 1](#pone.0182999.t001)). Using the GCTA conditional analysis on summary statistic data, we identified three such independent signals. In addition to the genome-wide significant lead SNP, two sub-loci show significant p-values (rs1056854: 3.30 × 10^−7^ and rs75041078: 3.87 × 10^−7^). We conditioned the two new SNPs on rs12459996 in a stepwise approach. rs1056854 showed the lowest p-value after conditioning with rs12459996 with a p-value of 4.07x10^-10^ and an OR of 1.07. Conditioning rs75041078 on the lead SNPs rs12459996 and rs1056854, we get a p-value for the third SNP of 2.21x10^-6^ with an OR of 1.05.

### Table 1. TGFB1 locus.

| Sub-locus | Lead SNP | CAD risk allele | Frequency risk allele | OR (CI) risk allele | P-value combined analysis | P-value joint analysis GCTA | OR risk allele joint analysis GCTA | SNP effect on TGFB1 | Regulatory function Haploreg | Regulatory function Regulome score |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| I | rs12459996 | T | 0.10 | 1.09 (1.06–1.12) | 1.58x10-12 | 2.73x10-9 | 1.09 | Increased expression | enhancer | 6 |
| II | rs75041078 | A | 0.25 | 1.06 (1.04–1.09) | 3.87x10-7 | 4.05x10-10 | 1.07 | n/a | enhancer | n/a |
| III | rs1056854 | A | 0.86 | 1.06 (1.04–1.08) | 3.30x10-7 | 2.21x10-6 | 1.05 | increased expression | enhancer | 5 |
To evaluate the functional implication of the three potentially independent associations, we performed an in-silico evaluation of the lead SNPs and the SNPs in high LD with these (r^2^ > 0.8) (see [S2 Table](#pone.0182999.s004) for detailed results).

The genome-wide significant lead SNP rs12459996 (p = 1.58 × 10^−12^; OR = 1.09), is found in a regulatory region and is marked as a strong promoter and enhancer in several cell types including smooth muscle and T-helper cells. As the promoter marks are downstream TGFB1, it is more likely that the regulatory effect on TGFB1 is acting through the enhancer. The lead SNP and SNPs in high LD (r^2^>0.8) are reported to have an eQTL effect on TGFB1 in the thyroid gland (p = 3.0 × 10^−8^, OR = 1.58) and the skeletal muscle (p = 5.6 × 10^−6^, 1.36) (GTEX). The CAD risk allele T is associated with increased TGFB1 expression in both cell types.

The second sub-locus has the lead SNP rs75041078 (conditional p = 4.07x10^-10^, conditional OR = 1.07). This SNP is not in high LD with any other SNP (r^2^>0.8). It is located in the intron of TMEM91 and lies in an enhancer histone mark in neutrophils. We did not find any eQTL effect in the databases searched.

The third sub-locus, with the lead SNP rs1056854 (conditional p = 2.21x10^-6^, conditional OR = 1.05), is also located in a regulatory region, which includes a promoter histone mark in various cell types, including H1-hESC cells. The locus is, however, downstream or rather at the 3′ end of the TGFB1 gene and hence the promoter unlikely to affect the TGFB1 gene. SNPs in high LD are also found in enhancer histone marks in multiple cell types, including T cells, T-helper cells and monocytes. Hence, the locus might have a regulatory effect on TGFB1. Supporting this hypothesis, we identified an eQTL effect on TGFB1 in the adrenal gland, suggesting a link between the SNP and the gene also in other tissues. The CAD risk allele A is associated with increased expression of TGFB1 (p = 1.5*10^−6^, OR = 2.2).

## Discussion

Exploring drug-gene interactions, we identified four genes or gene products to be affected by glatiramer acetate, a drug known to increase the risk of severe coronary events. Analyzing the genomic loci harboring these genes, we identified TGFB1 as a new genome-wide significant locus displaying association for CAD. The results of this analysis give rise to the hypothesis that the known interaction of GA with TGFB1 may be responsible for modulating the risk of CAD. Indeed, the results point towards a novel mechanism for the increased risk of CAD under GA treatment.

The pharmacologic mechanisms of GA in the treatment of multiple sclerosis are not fully understood. The general assumption is that the immune-modulatory activity of GA is related to the change of the T-cell antigen reactivity. Through its presumed binding to the MHC class II, GA is thought to alter the presentation of myelin antigens to auto-reactive T-cells and thereby affects the activity of the antigen presenting cells[[25](#pone.0182999.ref025)]. It is also known that GA induces the secretion of cytokines such as IL-4/10 and TGF-β in T-helper cells[[2](#pone.0182999.ref002)], which according to the present data may affect the risk of CAD under GA treatment.

TGFB1 (transforming growth factor, beta1) is a multifunctional peptide controlling multiple cellular functions such as proliferation and differentiation in several cell types. It plays an important role in the pathophysiology of the endothelial and vascular smooth muscle cell. TGFB1 is a very likely CAD candidate gene and has been linked to a range of cardiovascular traits such atherosclerosis, hypertension, inflammation and aneurysm [[26](#pone.0182999.ref026)–[29](#pone.0182999.ref029)]. TGFB1 serum levels are also reported to be higher in CAD patients[[30](#pone.0182999.ref030)]. In addition, several variants within other genes in the TGFB–SMAD signaling pathway have been associated with CAD[[26](#pone.0182999.ref026), [29](#pone.0182999.ref029), [31](#pone.0182999.ref031)–[34](#pone.0182999.ref034)]. It is, hence, also possible, that GAs interaction with TGFB1 influences other reported adverse effects such as the increased risk of hypertension. We cannot exclude that the GA related risk of CAD is secondary to GA induced hypertension, but at the same time we can also not exclude other pathways. TGFB1 is involved in multiple cellular functions suggesting increased CAD risk through multiple pathways.

Here, we identified three independent CAD associated signals within the TGFB1 locus. The TGFB1 locus has not been genome-wide significantly associated with CAD before, most likely due to small effect sizes and hence insufficient power of previous studies. The three variants are not associated with CAD related traits based on a GRASP database search (p<1x10^-4^). However, the overlap of the locus with immune cell histone marks suggest a link to inflammation. The lead SNP (rs12459996) of the TGFB1 locus is reported to increase the expression of the gene, which matches with the direction of effect reported for GA. In addition, the risk allele of the sub-locus rs1056854 is also associated with increased expression of the gene. The identified TGFB1 locus is also associated with expression of other genes (CCDC97, HNRNPUL1, AXL, BCKDHA) (see [S2 Table](#pone.0182999.s004)). Because our study focused on genes that interact with GA, we did not discuss these genes at this locus. It is however possible, that these genes influence the risk of CAD as well. Indeed, several studies have demonstrated that a regulatory SNPs have effects on more than one gene. In a previous study, we found multiple genes per locus where either all SNPs, a subset or only one SNP increase the risk of CAD[[35](#pone.0182999.ref035)].

The molecular effects of TGFB have been extensively studied *in vitro* and *in vivo* models linking the gene to CAD risk. For example, TGFB1 has been associated with several CAD related phenotypes such as thrombosis, inflammation, hypertension and neointima growth[[36](#pone.0182999.ref036)–[44](#pone.0182999.ref044)]. However, the net effect of TGFB may vary. Indeed, TGFB1 may either increase or decrease inflammation, activate or deactivate macrophages, depending on the local cytokine environment[[29](#pone.0182999.ref029)]. This is mainly due to the fact that it acts through several signaling pathways affecting several CAD related phenotypes with partially opposing effect on risk [[29](#pone.0182999.ref029), [33](#pone.0182999.ref033)]. In early stages of the disease, TGFB1 may be atheroprotective and higher levels of TGFB1 have been reported to decrease the risk of atherosclerosis[[29](#pone.0182999.ref029)]. In fact, TGFB1 displayed dosage effects where lower levels of TGFB1 increased proliferation whereas higher levels inhibited proliferation in endothelial cells[[36](#pone.0182999.ref036)]. Vice versa, in the presence of disease increased TGFB1 signaling has been associated with increased restenosis by increasing neointima growth[[37](#pone.0182999.ref037), [38](#pone.0182999.ref038)]. Moreover, TGFB1 has been linked to accelerated thrombus formation by inducing platelet aggregation [[39](#pone.0182999.ref039), [40](#pone.0182999.ref040)] and the expression is increased in rats with traumatic deep vein thrombosis versus control rats [[41](#pone.0182999.ref041)]. TGFB1 was also found to inhibit nitric oxide in vascular endothelial cells linking higher levels of TGFB1 to increased blood pressure[[42](#pone.0182999.ref042), [43](#pone.0182999.ref043)]. In addition, TGFB1 expression was also reported to be increased in patients with hypertension[[44](#pone.0182999.ref044)].

Our study design has several limitations. Our findings are based on associations rather than on functional testing. We thus cannot infer the precise pathway that is affected by the genetic variants at the TGFB1 locus. However, functional effects of the risk allele on expression levels have been previously reported and go in the same direction of TGFB1 expression as the effects reported in the literature for GA. Moreover, the association between the TGFB1 locus and CAD risk reaches genome-wide significance, which can be regarded as a conclusive and scientifically important observation independently of the overlap with GA side effects including enhanced TGFB1 expression. Indeed, the links reported here, first between GA and CAD risk ([drugs.com](http://drugs.com)), second between GA and TGFB1[[2](#pone.0182999.ref002)] and third the genome-wide significant association of TGFB1 SNPs with CAD are each highly conclusive. Together they allow hypothesizing that TGFB1 is involved in the increased risk of CAD under GA treatment.

Taken together, our results imply mechanistic similarities between pharmacologic responses to GA treatment and genetic variants affecting CAD risk. While GA treatment is known to enhance TGFB1 expression and CAD risk, we here associate SNPs within the TGFB1 locus linked with CAD risk and enhanced TGFB1 expression. Thus, both the newly identified CAD risk alleles and GA appear to induce the expression of TGFB1, suggesting that the CAD risk alleles and the drug have similar effects on the gene product and subsequently on CAD risk. With an additive effect, the CAD risk alleles might also explain variable CAD risk under GA treatment. Finally, in this study, we identified TGFB1 as a new genome-wide significant locus affecting CAD risk.

## Supporting information

## Acknowledgments

This work was supported by grants from the Fondation Leducq (*CADgenomics*: Understanding CAD Genes, 12CVD02), the German Federal Ministry of Education and Research (BMBF) within the framework of the e:Med research and funding concept (*e*:*AtheroSysMed*, grant 01ZX1313A-2014 and *SysInflame*, grant 01ZX1306A), and the European Union Seventh Framework Programme FP7/2007-2013 under grant agreement n^o^ HEALTH-F2-2013-601456 (*CVgenes-at-target*). Further grants were received from the DFG as part of the Sonderforschungsbereich CRC 1123 (B2). T.K. was supported by a DZHK Rotation Grant. I.B. was supported by the Deutsche Forschungsgemeinschaft (DFG) cluster of excellence ‘Inflammation at Interfaces’. F.W.A. is supported by a Dekker scholarship-Junior Staff Member 2014T001 - Netherlands Heart Foundation and UCL Hospitals NIHR Biomedical Research Centre.

### CARDIoGRAMplusC4D members

Panos Deloukas1, Stavroula Kanoni1, Christina Willenborg2, Martin Farrall3,4, Themistocles L Assimes5, John R Thompson6, Erik Ingelsson7, Danish Saleheen8–10, Jeanette Erdmann2, Benjamin A Goldstein5, Kathleen Stirrups1, Inke R König11, Jean-Baptiste Cazier4, Åsa Johansson12, Alistair S Hall13, Jong-Young Lee14, Cristen J Willer15,16, John C Chambers17, Tõnu Esko18,19, Lasse Folkersen20,21, Anuj Goel3,4, Elin Grundberg22, Aki SHavulinna23, Weang K Ho10, Jemma C Hopewell24,25, Niclas Eriksson12, Marcus EKleber26,27, Kati Kristiansson23, Per Lundmark28, Leo-Pekka Lyytikäinen29,30, Suzanne Rafelt31, Dmitry Shungin32–34, Rona J Strawbridge20,21, Gudmar Thorleifsson35, Emmi Tikkanen36,37, Natalie Van Zuydam38, Benjamin F Voight39, Lindsay L Waite40, Weihua Zhang17, Andreas Ziegler11, Devin Absher40, David Altshuler41–44, Anthony J Balmforth45, Inês Barroso1,46, Peter SBraund31,47, Christof Burgdorf48, Simone Claudi-Boehm49, David Cox50, Maria Dimitriou51, Ron Do41,43, Alex SF Doney38, Nour Eddine El Mokhtari53, Per Eriksson20,21, Krista Fischer18, Pierre Fontanillas41, Anders Franco-Cereceda54, Bruna Gigante55, Leif Groop56, Stefan Gustafsson7, Jörg Hager57, Göran Hallmans58, Bok-GheeHan14, Sarah EHunt1, Hyun M Kang59, Thomas Illig60, Thorsten Kessler48, Joshua W Knowles5, Genovefa Kolovou61, Johanna Kuusisto62, Claudia Langenberg63, Cordelia Langford1, Karin Leander55, Marja-Liisa Lokki64, Anders Lundmark28, Mark I McCarthy3,65,66, Christa Meisinger67, Olle Melander56, Evelin Mihailov19, Seraya Maouche68, Andrew D Morris38, Martina Müller-Nurasyid69–72, MuTHER Consortium, Kjell Nikus73, John F Peden3, N. William Rayner3, Asif Rasheed9, Silke Rosinger74, Diana Rubin53, Moritz P Rumpf48, Arne Schäfer75, Mohan Sivananthan76,77, Ci Song7, Alexandre F R Stewart78,79, Sian-Tsung Tan80, Gudmundur Thorgeirsson81,82, C Ellen van der Schoot83, Peter J Wagner36,37, Wellcome Trust Case Control Consortium, George A Wells78,79, Philipp S Wild84,85, Tsun-Po Yang1, Philippe Amouyel86, Dominique Arveiler87, Hanneke Basart88, Michael Boehnke59, Eric Boerwinkle89, Paolo Brambilla90, Francois Cambien68, Adrienne L. Cupples91,92, Ulf de Faire55, Abbas Dehghan93, Patrick Diemert94, Stephen E Epstein95, Alun Evans96, Marco M Ferrario97, Jean Ferrières98, Dominique Gauguier3,99, Alan S Go100, Alison H Goodall31,47, Villi Gudnason81,101, Stanley L Hazen102, Hilma Holm35, Carlos Iribarren100, Yangsoo Jang103, Mika Kähönen104, Frank Kee105, Hyo-Soo Kim106, Norman Klopp60, Wolfgang Koenig107, Wolfgang Kratzer108, Kari Kuulasmaa23, Markku Laakso62, Reijo Laaksonen108, Ji-Young Lee14, Lars Lind28, Willem H Ouwehand1,109,110, Sarah Parish24,25, Jeong E Park111, Nancy L Pedersen7, Annette Peters67,112, Thomas Quertermous5, Daniel J Rader113, Veikko Salomaa23, Eric Schadt114, Svati H Shah115,116, Juha Sinisalo117, Klaus Stark118, Kari Stefansson35,81, David-Alexandre Trégouët68, Jarmo Virtamo23, Lars Wallentin12, Nicholas Wareham63, Martina E Zimmermann118, Markku S Nieminen117, Christian Hengstenberg118, Manjinder S Sandhu1,63, Tomi Pastinen119, Ann-Christine Syvänen28, G Kees Hovingh88, George Dedoussis51, Paul W Franks32–34,120, Terho Lehtimäki29,30, Andres Metspalu18,19, Pierre A Zalloua121, Agneta Siegbahn12, Stefan Schreiber75, Samuli Ripatti1,37, Stefan S Blankenberg94, Markus Perola23, Robert Clarke24,25, Bernhard OBoehm74, Christopher O’Donnell93, Muredach P Reilly122,126, Winfried März26,123, Rory Collins24,25,126, Sekar Kathiresan41,124,125,126, Anders Hamsten20,21,126, Jaspal S Kooner80,126, Unnur Thorsteinsdottir35,81,126, John Danesh9,126, Colin NA Palmer38,126, Robert Roberts78,79,126, Hugh Watkins3,4,126, Heribert Schunkert48,126 & Nilesh J Samani31,47,126

### Affiliations CARDIoGRAM plus C4D

1Wellcome Trust Sanger Institute, Hinxton, Cambridge, UK. 2Institut für Integrative und Experimentelle Genomik, Universität zu Lübeck, Lübeck, Germany. 3Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford, UK. 4Cardiovascular Medicine, Radcliffe Department of Medicine, University of Oxford, Oxford, UK. 5Department of Medicine, Stanford University School of Medicine, Stanford, California, USA. 6Department of Health Sciences, University of Leicester, Leicester, UK. 7Department of Medical Epidemiology and Biostatistics, Karolinska Institutet, Stockholm, Sweden. 8Center for Non-Communicable Diseases, Karachi, Pakistan. 9Department of Public Health and Primary Care, University of Cambridge, Cambridge, UK. 10Department of Medicine, University of Pennsylvania, Philadelphia, Pennsylvania, USA. 11Institut für Medizinische Biometrie und Statistik, Universität zu Lübeck, Lübeck, Germany. 12Uppsala Clinical Research Center, Uppsala University, Uppsala, Sweden. 13Division of Cardiovascular and Neuronal Remodelling, Multidisciplinary Cardiovascular Research Centre, Leeds Institute of Genetics, Health and Therapeutics, University of Leeds, Leeds, UK. 14Center for Genome Science, Korea National Institute of Health, Korea Center for Disease Control and Prevention, Yeonje-ri, Chungwon-gun, Korea. 15Division of Cardiovascular Medicine, Department of Internal Medicine, University of Michigan, Ann Arbor, Michigan, USA. 16Department of Human Genetics, University of Michigan, Ann Arbor, Michigan, USA. 17Department of Epidemiology and Biostatistics, Imperial College London, London, UK. 18Estonian Genome Center, University of Tartu, Tartu, Estonia. 19Institute of Molecular and Cell Biology, University of Tartu, Tartu, Estonia. 20Atherosclerosis Research Unit, Department of Medicine, Karolinska Institutet, Stockholm, Sweden. 21Center for Molecular Medicine, Karolinska University Hospital, Stockholm, Sweden. 22Department of Twin Research and Genetic Epidemiology, King’s College London, London, UK. 23Department of Chronic Disease Prevention, National Institute for Health and Welfare, Helsinki, Finland. 24Clinical Trial Service Unit, University of Oxford, Oxford, UK. 25Epidemiological Studies Unit, University of Oxford, Oxford, UK. 26Mannheim Institute of Public Health, Social and Preventive Medicine, Medical Faculty of Mannheim, University of Heidelberg, Mannheim, Germany. 27Ludwigshafen Risk and Cardiovascular Health (LURIC) Study, Freiburg, Germany. 28Department of Medical Sciences, Uppsala University, Uppsala, Sweden. 29Department of Clinical Chemistry, Fimlab Laboratories, Tampere University Hospital, Tampere, Finland. 30Department of Clinical Chemistry, University of Tampere School of Medicine, Tampere, Finland. 31Department of Cardiovascular Sciences, University of Leicester, Glenfield Hospital, Leicester, UK. 32Genetic & Molecular Epidemiology Unit, Department of Clinical Sciences, Lund University Diabetes Center, Skåne University Hospital, Malmö, Sweden. 33Department of Public Health & Clinical Medicine, Genetic Epidemiology & Clinical Research Group, Section for Medicine, Umeå University, Umeå, Sweden. 34Department of Odontology, Umeå University, Umeå, Sweden. 35deCODE Genetics, Reykjavik, Iceland. 36Institute for Molecular Medicine FIMM, University of Helsinki, Helsinki, Finland. 37Public Health Genomics Unit, National Institute for Health and Welfare, Helsinki, Finland. 38Medical Research Institute, University of Dundee, Ninewells Hospital and Medical School, Dundee, UK. 39Department of Pharmacology, University of Pennsylvania, Philadelphia, Pennsylvania, USA. 40HudsonAlpha Institute for Biotechnology, Huntsville, Alabama, USA. 41Broad Institute of Harvard and MIT, Cambridge, Massachusetts, USA. 42Department of Molecular Biology, Massachusetts General Hospital, Boston, Massachusetts, USA. 43Center for Human Genetic Research, Massachusetts General Hospital, Boston, Massachusetts, USA. 44Department of Genetics, Harvard Medical School, Boston, Massachusetts, USA. 45Division of Cardiovascular and Diabetes Research, Multidisciplinary Cardiovascular Research Centre, Leeds Institute of Genetics, Health and Therapeutics, University of Leeds, Leeds, UK. 46University of Cambridge Metabolic Research Laboratories, Institute of Metabolic Science, Addenbrooke’s Hospital, Cambridge, UK. 47National Institute for Health Research (NIHR) Leicester Cardiovascular Biomedical Research Unit, Glenfield Hospital, Leicester, UK. 48Deutsches Herzzentrum München, Technische Universität München, Munich, Germany. 49Practice of Gynecology, Ulm University Medical Centre, Ulm, Germany. 50Biotherapeutics and Bioinnovation Center, Pfizer, South San Francisco, California, USA. 51Department of Dietetics–Nutrition, Harokopio University, Athens, Greece. 53Klinik für Innere Medizin, Kreiskrankenhaus Rendsburg, Rendsburg, Germany. 54Cardiothoracic Surgery Unit, Department of Molecular Medicine and Surgery, Karolinska Institutet, Stockholm, Sweden. 55Division of Cardiovascular Epidemiology, Institute of Environmental Medicine, Karolinska Institutet, Stockholm, Sweden. 56Department of Clinical Sciences, Diabetes and Endocrinology, Lund University, University Hospital Malmö, Malmö, Sweden. 57Commissariat à l′Energie Atomique (CEA)–Genomics Institute, National Genotyping Centre, Paris, France. 58Department of Public Health & Clinical Medicine, Section for Nutritional Research, Umeå University, Umeå, Sweden. 59Department of Biostatistics, Center for Statistical Genetics, University of Michigan, Ann Arbor, Michigan, USA. 60Hannover Unified Biobank, Hannover Medical School, Hannover, Germany. 61First Cardiology Department, Onassis Cardiac Surgery Center 356, Athens, Greece. 62Department of Medicine, University of Eastern Finland and Kuopio University Hospital, Kuopio, Finland. 63Medical Research Council (MRC) Epidemiology Unit, Institute of Metabolic Science, Addenbrooke’s Hospital, Cambridge, UK. 64Transplantation Laboratory, Haartman Institute, University of Helsinki, Helsinki, Finland. 65Oxford Centre for Diabetes, Endocrinology and Metabolism, University of Oxford, Oxford, UK. 66Oxford NIHR Biomedical Research Centre, Churchill Hospital, Oxford, UK. 67Institute of Epidemiology II, Helmholtz Zentrum München–German Research Center for Environmental Health, Neuherberg, Germany. 68Institut National de la Santé et la Recherche Médicale (INSERM) Unité Mixte de Recherche (UMR) S937, Institute for Cardiometabolism and Nutrition (ICAN), Pierre and Marie Curie (Paris 6) University, Paris, France. 69Department of Medicine I, University Hospital Grosshadern, Ludwig-Maximilians-Universität, Munich, Germany. 70Chair of Epidemiology, Institute of Medical Informatics, Biometry and Epidemiology, Ludwig- Maximilians-Universität, Munich, Germany. 71Chair of Genetic Epidemiology, Institute of Medical Informatics, Biometry and Epidemiology, Ludwig-Maximilians-Universität, Munich, Germany. 72Institute of Genetic Epidemiology, Helmholtz Zentrum München–German Research Center for Environmental Health, Neuherberg, Germany. 73Heart Centre, Department of Cardiology, Tampere University Hospital, Tampere, Finland. 74Division of Endocrinology and Diabetes, Department of Internal Medicine, Ulm University Medical Centre, Ulm, Germany. 75Institut für Klinische Molekularbiologie, Christian-Albrechts Universität, Kiel, Germany. 76Division of Epidemiology, Multidisciplinary Cardiovascular Research Centre (MCRC) University of Leeds, Leeds, UK. 77Leeds Institute of Genetics, Health and Therapeutics, University of Leeds, Leeds, UK. 78University of Ottawa Heart Institute, Cardiovascular Research Methods Centre Ontario, Ottawa, Ontario, Canada. 79Ruddy Canadian Cardiovascular Genetics Centre, Ottawa, Ontario, Canada. 80National Heart and Lung Institute (NHLI), Imperial College London, Hammersmith Hospital, London, UK. 81Faculty of Medicine, University of Iceland, Reykjavik, Iceland. 82Department of Medicine, Landspitali University Hospital, Reykjavik, Iceland. 83Department of Experimental Immunohematology, Sanquin, Amsterdam, The Netherlands. 84Center for Thrombosis and Hemostasis, University Medical Center Mainz, Johannes Gutenberg University Mainz, Mainz, Germany. 85Department of Medicine 2, University Medical Center Mainz, Johannes Gutenberg University Mainz, Mainz, Germany. 86Institut Pasteur de Lille, INSERM U744, Université Lille Nord de France, Lille, France. 87Department of Epidemiology and Public Health, EA3430, University of Strasbourg, Strasbourg, France. 88Department of Vascular Medicine, Academic Medical Center, Amsterdam, The Netherlands. 89Human Genetics Center, University of Texas Health Science Center, Houston, Texas, USA. 90Department of Experimental Medicine, University of Milano–Bicocca, Monza, Italy. 91Department of Biostatistics, Boston University School of Public Health, Boston, Massachusetts, USA. 92National Heart, Lung, and Blood Institute’s Framingham Heart Study, Framingham, Massachusetts, USA. 93Department of Epidemiology, Erasmus Medical Center, Rotterdam, The Netherlands. 94Clinic for General and Interventional Cardiology, University Heart Center Hamburg, Hamburg, Germany. 95Cardiovascular Research Institute, Washington Hospital Center, Washington, DC, USA. 96Centre for Public Health, The Queen’s University of Belfast, Belfast, UK. 97Research Centre for Epidemiology and Preventive Medicine (EPIMED), Department of Clinical and Experimental Medicine, University of Insubria, Varese, Italy. 98Department of Cardiology, Toulouse University School of Medicine, Rangueil Hospital, Toulouse, France. 99INSERM UMR S872, Cordeliers Research Centre, Paris, France. 100Division of Research, Kaiser Permanente Northern California, Oakland, California, USA. 101Icelandic Heart Association, Kopavogur, Iceland. 102Lerner Research Institute, Cleveland Clinic, Cleveland, Ohio, USA. 103Cardiology Division, Department of Internal Medicine, Cardiovascular Genome Center, Yonsei University, Seoul, Korea. 104Department of Clinical Physiology, Tampere University Hospital and University of Tampere, Tampere, Finland. 105UK Clinical Research Collaboration (UKCRC) Centre of Excellence for Public Health (Northern Ireland), Queen’s University of Belfast, Belfast, UK. 106Department of Internal Medicine, Cardiovascular Center, Seoul National University Hospital, Seoul, Korea. 107Department of Internal Medicine II–Cardiology, Ulm University Medical Center, Ulm, Germany. 108Science Center, Tampere University Hospital, Tampere, Finland. 109Department of Haematology, University of Cambridge, Cambridge, UK. 110National Health Service (NHS) Blood and Transplant, Cambridge, UK. 111Division of Cardiology, Samsung Medical Center, Seoul, Korea. 112Munich Heart Alliance, Munich, Germany. 113Division of Translational Medicine and Human Genetics, Department of Medicine, Perelman School of Medicine at the University of Pennsylvania, Philadelphia, Pennsylvania, USA. 114Institute for Genomics and Multiscale Biology, Department of Genetics and Genomic Sciences, Mount Sinai School of Medicine, New York, New York, USA. 115Center for Human Genetics, Department of Medicine, Duke University Medical Center, Durham, North Carolina, USA. 116Division of Cardiology, Department of Medicine, Duke University Medical Center, Durham, North Carolina, USA. 117Division of Cardiology, Department of Medicine, Helsinki University Central Hospital (HUCH), Helsinki, Finland. 118Klinik und Poliklinik für Innere Medizin II, Regensburg, Germany. 119Department of Human Genetics, McGill University, Montréal, Québec, Canada. 120Department of Nutrition, Harvard School of Public Health, Boston, Massachusetts, USA. 121Lebanese American University, Chouran, Beirut, Lebanon. 122Cardiovascular Institute, Perelman School of Medicine at the University of Pennsylvania, Philadelphia, Pennsylvania, USA. 123Synlab Academy, Mannheim, Germany. 124Cardiology Division, Center for Human Genetic Research, Massachusetts General Hospital and Harvard Medical School, Boston, Massachusetts, USA. 125Cardiovascular Research Center, Massachusetts General Hospital and Harvard Medical School, Boston, Massachusetts, USA.

### CARDIoGRAM members:

Executive Committee: Sekar Kathiresan1,2,3, Muredach P. Reilly4, Nilesh J. Samani5,6, Heribert Schunkert7,79

Executive Secretary: Jeanette Erdmann7,79

Steering Committee: Themistocles L. Assimes8, Eric Boerwinkle9, Jeanette Erdmann7,79 Alistair Hall10, Christian Hengstenberg11, Sekar Kathiresan1,2,3, Inke R. König12, Reijo Laaksonen13, Ruth McPherson14, Muredach P. Reilly4, Nilesh J. Samani5,6, Heribert Schunkert7,79, John R. Thompson15, Unnur Thorsteinsdottir16,17, Andreas Ziegler12

Statisticians: Inke R. König12 (chair), John R. Thompson15 (chair), Devin Absher18, Li Chen19, L. Adrienne Cupples20,21, Eran Halperin22, Mingyao Li23, Kiran Musunuru1,2,3, Michael Preuss12,7, Arne Schillert12, Gudmar Thorleifsson16, Benjamin F. Voight2,3,24, George A. Wells25

Writing group: Themistocles L. Assimes8, Panos Deloukas26, Jeanette Erdmann7,79, Hilma Holm16, Sekar Kathiresan1,2,3, Inke R. König12, Ruth McPherson14, Muredach P. Reilly4, Robert Roberts14, Nilesh J. Samani5,6, Heribert Schunkert7,79, Alexandre F. R. Stewart14

ADVANCE: Devin Absher18, Themistocles L. Assimes8, Stephen Fortmann8, Alan Go27, Mark Hlatky8, Carlos Iribarren27, Joshua Knowles8, Richard Myers18, Thomas Quertermous8, Steven Sidney27, Neil Risch28, Hua Tang29

CADomics: Stefan Blankenberg30, Tanja Zeller30, Arne Schillert12, Philipp Wild30, Andreas Ziegler12, Renate Schnabel30, Christoph Sinning30, Karl Lackner31, Laurence Tiret32, Viviane Nicaud32, Francois Cambien32, Christoph Bickel30, Hans J. Rupprecht30, Claire Perret32, Carole Proust32, Thomas Münzel30

CHARGE: Maja Barbalic33, Joshua Bis34, Eric Boerwinkle9, Ida Yii-Der Chen35, L. Adrienne Cupples20,21, Abbas Dehghan36, Serkalem Demissie-Banjaw37,21, Aaron Folsom38, Nicole Glazer39, Vilmundur Gudnason40,41, Tamara Harris42, Susan Heckbert43, Daniel Levy21, Thomas Lumley44, Kristin Marciante45, Alanna Morrison46, Christopher J. O´Donnell47, Bruce M. Psaty48, Kenneth Rice49, Jerome I. Rotter35, David S. Siscovick50, Nicholas Smith43, Albert Smith40,41, Kent D. Taylor35, Cornelia van Duijn36, Kelly Volcik46, Jaqueline Whitteman36, Vasan Ramachandran51, Albert Hofman36, Andre Uitterlinden52,36

deCODE: Solveig Gretarsdottir16, Jeffrey R. Gulcher16, Hilma Holm16, Augustine Kong16, Kari Stefansson16,17, Gudmundur Thorgeirsson53,17, Karl Andersen53,17, Gudmar Thorleifsson16, Unnur Thorsteinsdottir16,17

GERMIFS I and II: Jeanette Erdmann7,79, Marcus Fischer11, Anika Grosshennig12,7, Christian Hengstenberg11, Inke R. König12, Wolfgang Lieb54, Patrick Linsel-Nitschke7, Michael Preuss12,7, Klaus Stark11, Stefan Schreiber55, H.-Erich Wichmann56,58,59, Andreas Ziegler12, Heribert Schunkert80

GERMIFS III (KORA): Zouhair Aherrahrou7,79, Petra Bruse7,79, Angela Doering56, Jeanette Erdmann7,79, Christian Hengstenberg11, Thomas Illig56, Norman Klopp56, Inke R. König12, Patrick Diemert7, Christina Loley12,7, Anja Medack7,79, Christina Meisinger56, Thomas Meitinger57,60, Janja Nahrstedt12,7, Annette Peters56, Michael Preuss12,7, Klaus Stark11, Arnika K. Wagner7, H.-Erich Wichmann56,58,59, Christina Willenborg,7,79, Andreas Ziegler12, Heribert Schunkert7,79

LURIC/AtheroRemo: Bernhard O. Böhm61, Harald Dobnig62, Tanja B. Grammer63, Eran Halperin22, Michael M. Hoffmann64, Marcus Kleber65, Reijo Laaksonen13, Winfried März63,66,67, Andreas Meinitzer66, Bernhard R. Winkelmann68, Stefan Pilz62, Wilfried Renner66, Hubert Scharnagl66, Tatjana Stojakovic66, Andreas Tomaschitz62, Karl Winkler64

MIGen: Benjamin F. Voight2,3,24, Kiran Musunuru1,2,3, Candace Guiducci3, Noel Burtt3, Stacey B. Gabriel3, David S. Siscovick50, Christopher J. O’Donnell47, Roberto Elosua69, Leena Peltonen49, Veikko Salomaa70, Stephen M. Schwartz50, Olle Melander26, David Altshuler71,3, Sekar Kathiresan1,2,3

OHGS: Alexandre F. R. Stewart14, Li Chen19, Sonny Dandona14, George A. Wells25, Olga Jarinova14, Ruth McPherson14, Robert Roberts14

PennCATH/MedStar: Muredach P. Reilly4, Mingyao Li23, Liming Qu23, Robert Wilensky4, William Matthai4, Hakon H. Hakonarson72, Joe Devaney73, Mary Susan Burnett73, Augusto D. Pichard73, Kenneth M. Kent73, Lowell Satler73, Joseph M. Lindsay73, Ron Waksman73, Christopher W. Knouff74, Dawn M. Waterworth74, Max C. Walker74, Vincent Mooser74, Stephen E. Epstein73, Daniel J. Rader75,4

WTCCC: Nilesh J. Samani5,6, John R. Thompson15, Peter S. Braund5, Christopher P. Nelson5, Benjamin J. Wright76, Anthony J. Balmforth77, Stephen G. Ball78, Alistair S. Hall10, Wellcome Trust Case Control Consortium

### Affiliations CARDIoGRAM

1 Cardiovascular Research Center and Cardiology Division, Massachusetts General Hospital, Boston, MA, USA; 2 Center for Human Genetic Research, Massachusetts General Hospital, Boston, MA, USA; 3 Program in Medical and Population Genetics, Broad Institute of Harvard and Massachusetts Institute of Technology (MIT), Cambridge, MA, USA; 4 The Cardiovascular Institute, University of Pennsylvania, Philadelphia, PA, USA; 5 Department of Cardiovascular Sciences, University of Leicester, Glenfield Hospital, Leicester, UK; 6 Leicester National Institute for Health Research Biomedical Research Unit in Cardiovascular Disease, Glenfield Hospital, Leicester, LE3 9QP, UK; 7 Institut für integrative und experimentelle Genomik, Universität zu Lübeck, Lübeck, Germany; 8 Department of Medicine, Stanford University School of Medicine, Stanford, CA, USA; 9 University of Texas Health Science Center, Human Genetics Center and Institute of Molecular Medicine, Houston, TX, USA; 10 Division of Cardiovascular and Neuronal Remodelling, Multidisciplinary Cardiovascular Research Centre, Leeds Institute of Genetics, Health and Therapeutics, University of Leeds, UK; 11 Klinik und Poliklinik für Innere Medizin II, Universität Regensburg, Regensburg, Germany; 12 Institut für Medizinische Biometrie und Statistik, Universität zu Lübeck, Lübeck, Germany; 13 Science Center, Tampere University Hospital, Tampere, Finland; 14 The John & Jennifer Ruddy Canadian Cardiovascular Genetics Centre, University of Ottawa Heart Institute, Ottawa, Canada; 15 Department of Health Sciences, University of Leicester, Leicester, UK; 16 deCODE Genetics, 101 Reykjavik, Iceland; 17 University of Iceland, Faculty of Medicine, 101 Reykjavik, Iceland; 18 Hudson Alpha Institute, Huntsville, Alabama, USA; 19 Cardiovascular Research Methods Centre, University of Ottawa Heart Institute, 40 Ruskin Street, Ottawa, Ontario, Canada, K1Y 4W7; 20 Department of Biostatistics, Boston University School of Public Health, Boston, MA USA; 21 National Heart, Lung and Blood Institute's Framingham Heart Study, Framingham, MA, USA; 22 The Blavatnik School of Computer Science and the Department of Molecular Microbiology and Biotechnology, Tel-Aviv University, Tel-Aviv, Israel, and the International Computer Science Institute, Berkeley, CA, USA; 23 Biostatistics and Epidemiology, University of Pennsylvania, Philadelphia, PA, USA; 24 Department of Medicine, Harvard Medical School, Boston, MA, USA; 25 Research Methods, Univ Ottawa Heart Inst; 26 Department of Clinical Sciences, Hypertension and Cardiovascular Diseases, Scania University Hospital, Lund University, Malmö, Sweden; 27 Division of Research, Kaiser Permanente, Oakland, CA, USA; 28 Institute for Human Genetics, University of California, San Francisco, San Francisco, CA, USA; 29 Dept Cardiovascular Medicine, Cleveland Clinic; 30 Medizinische Klinik und Poliklinik, Johannes-Gutenberg Universität Mainz, Universitätsmedizin, Mainz, Germany; 31 Institut für Klinische Chemie und Laboratoriumsmediizin, Johannes-Gutenberg Universität Mainz, Universitätsmedizin, Mainz, Germany; 32 INSERM UMRS 937, Pierre and Marie Curie University (UPMC, Paris 6) and Medical School, Paris, France; 33 University of Texas Health Science Center, Human Genetics Center, Houston, TX, USA; 34 Cardiovascular Health Resarch Unit and Department of Medicine, University of Washington, Seattle, WA USA; 35 Cedars-Sinai Medical Center, Medical Genetics Institute, Los Angeles, CA, USA; 36 Erasmus Medical Center, Department of Epidemiology, Rotterdam, The Netherlands; 37 Boston University, School of Public Health, Boston, MA, USA; 38 University of Minnesota School of Public Health, Division of Epidemiology and Community Health, School of Public Health (A.R.F.), Minneapolis, MN, USA; 39 University of Washington, Cardiovascular Health Research Unit and Department of Medicine, Seattle, WA, USA; 40 Icelandic Heart Association, Kopavogur Iceland; 41 University of Iceland, Reykjavik, Iceland; 42 Laboratory of Epidemiology, Demography, and Biometry, Intramural Research Program, National Institute on Aging, National Institutes of Health, Bethesda MD, USA; 43 University of Washington, Department of Epidemiology, Seattle, WA, USA; 44 University of Washington, Department of Biostatistics, Seattle, WA, USA; 45 University of Washington, Department of Internal Medicine, Seattle, WA, USA; 46 University of Texas, School of Public Health, Houston, TX, USA; 47 National Heart, Lung and Blood Institute, Framingham Heart Study, Framingham, MA and Cardiology Division, Massachusetts General Hospital, Boston, MA, USA; 48 Center for Health Studies, Group Health, Departments of Medicine, Epidemiology, and Health Services, Seattle, WA, USA; 49 The Wellcome Trust Sanger Institute, The Wellcome Trust Genome Campus, Hinxton, Cambridge, UK; 50 Cardiovascular Health Research Unit, Departments of Medicine and Epidemiology, University of Washington, Seattle; 51 Boston University Medical Center, Boston, MA, USA; 52 Department of Internal Medicine, Erasmus Medical Center, Rotterdam, The Netherlands; 53 Department of Medicine, Landspitali University Hospital, 101 Reykjavik, Iceland; 54 Boston University School of Medicine, Framingham Heart Study, Framingham, MA, USA; 55 Institut für Klinische Molekularbiologie, Christian-Albrechts Universität, Kiel, Germany; 56 Institute of Epidemiology, Helmholtz Zentrum München–German Research Center for Environmental Health, Neuherberg, Germany; 57 Institut für Humangenetik, Helmholtz Zentrum München, Deutsches Forschungszentrum für Umwelt und Gesundheit, Neuherberg, Germany; 58 Institute of Medical Information Science, Biometry and Epidemiology, Ludwig-Maximilians-Universität München, Germany; 59 Klinikum Grosshadern, Munich, Germany; 60 Institut für Humangenetik, Technische Universität München, Germany; 61 Division of Endocrinology and Diabetes, Graduate School of Molecular Endocrinology and Diabetes, University of Ulm, Ulm, Germany; 62 Division of Endocrinology, Department of Medicine, Medical University of Graz, Austria; 63 Synlab Center of Laboratory Diagnostics Heidelberg, Heidelberg, Germany; 64 Division of Clinical Chemistry, Department of Medicine, Albert Ludwigs University, Freiburg, Germany; 65 LURIC non profit LLC, Freiburg, Germany; 66 Clinical Institute of Medical and Chemical Laboratory Diagnostics, Medical University Graz, Austria; 67 Institute of Public Health, Social and Preventive Medicine, Medical Faculty Manneim, University of Heidelberg, Germany; 68 Cardiology Group Frankfurt-Sachsenhausen, Frankfurt, Germany; 69 Cardiovascular Epidemiology and Genetics Group, Institut Municipal d’Investigació Mèdica, Barcelona; Ciber Epidemiología y Salud Pública (CIBERSP), Spain; 70 Chronic Disease Epidemiology and Prevention Unit, Department of Chronic Disease Prevention, National Institute for Health and Welfare, Helsinki, Finland; 71 Department of Molecular Biology and Center for Human Genetic Research, Massachusetts General Hospital, Harvard Medical School, Boston, USA; 72 The Center for Applied Genomics, Children’s Hospital of Philadelphia, Philadelphia, Pennsylvania, USA; 73 Cardiovascular Research Institute, Medstar Health Research Institute, Washington Hospital Center, Washington, DC 20010, USA; 74 Genetics Division and Drug Discovery, GlaxoSmithKline, King of Prussia, Pennsylvania 19406, USA; 75 The Institute for Translational Medicine and Therapeutics, School of Medicine, University of Pennsylvania, Philadelphia, PA, USA; 76 Department of Cardiovascular Surgery, University of Leicester, Leicester, UK; 77 Division of Cardiovascular and Diabetes Research, Multidisciplinary Cardiovascular Research Centre, Leeds Institute of Genetics, Health and Therapeutics, University of Leeds, Leeds, LS2 9JT, UK; 78 LIGHT Research Institute, Faculty of Medicine and Health, University of Leeds, Leeds, UK; 79 DZHK (German Centre for Cardiovascular Research), partner site Hamburg/Kiel/Lübeck, Lübeck, Germany; 80: Deutsches Herzzentrum München, Technische Universität München München, Germany

## Data Availability

All relevant data are within the paper and its Supporting Information files.

## Funding Statement

This work was supported by grants from the Fondation Leducq (CADgenomics: Understanding CAD Genes, 12CVD02), the German Federal Ministry of Education and Research (BMBF) within the framework of the e:Med research and funding concept (e:AtheroSysMed, grant 01ZX1313A-2014 and SysInflame, grant 01ZX1306A), and the European Union Seventh Framework Programme FP7/2007-2013 under grant agreement no HEALTH-F2-2013-601456 (CVgenes-at-target). Further grants were received from the DFG as part of the Sonderforschungsbereich CRC 1123 (B2). T.K. was supported by a DZHK Rotation Grant. I.B. was supported by the Deutsche Forschungsgemeinschaft (DFG) cluster of excellence ‘Inflammation at Interfaces’. F.W.A. is supported by a Dekker scholarship-Junior Staff Member 2014T001 -- Netherlands Heart Foundation and UCL Hospitals NIHR Biomedical Research Centre. This work was supported by the German Research Foundation (DFG) and the Technical University of Munich within the funding programme Open Access Publishing.

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Data Availability Statement

All relevant data are within the paper and its Supporting Information files.

### Supplementary Materials

### Data Availability Statement

All relevant data are within the paper and its Supporting Information files.

## References

1. Francis DA. Glatiramer acetate (Copaxone). Int J Clin Pract. 2001;55(6):394–8. .  [PubMed](https://pubmed.ncbi.nlm.nih.gov/11501229/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Clin%20Pract&title=Glatiramer%20acetate%20(Copaxone)&author=DA%20Francis&volume=55&issue=6&publication_year=2001&pages=394-8&pmid=11501229&)

2. Arnon R, Aharoni R. Mechanism of action of glatiramer acetate in multiple sclerosis and its potential for the development of new applications. Proc Natl Acad Sci U S A. 2004;101 Suppl 2:14593–8. doi: 10.1073/pnas.0404887101 ; PubMed Central PMCID: PMCPMC521994.  [DOI](https://doi.org/10.1073/pnas.0404887101) | [DOI](https://doi.org/10.1073/pnas.0404887101) | [PMC free article](/articles/PMC521994/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15371592/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&title=Mechanism%20of%20action%20of%20glatiramer%20acetate%20in%20multiple%20sclerosis%20and%20its%20potential%20for%20the%20development%20of%20new%20applications&author=R%20Arnon&author=R%20Aharoni&volume=101&issue=Suppl%202&publication_year=2004&pages=14593-8&pmid=15371592&doi=10.1073/pnas.0404887101&)

3. Li Q, Milo R, Panitch H, Swoveland P, Bever CT Jr. Glatiramer acetate blocks the activation of THP-1 cells by interferon-gamma. Eur J Pharmacol. 1998;342(2–3):303–10. .  [DOI](https://doi.org/10.1016/s0014-2999(97)01509-4) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9548401/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Pharmacol&title=Glatiramer%20acetate%20blocks%20the%20activation%20of%20THP-1%20cells%20by%20interferon-gamma&author=Q%20Li&author=R%20Milo&author=H%20Panitch&author=P%20Swoveland&author=CT%20Bever&volume=342&issue=2%E2%80%933&publication_year=1998&pages=303-10&pmid=9548401&doi=10.1016/s0014-2999(97)01509-4&)

4. Lalive PH, Neuhaus O, Benkhoucha M, Burger D, Hohlfeld R, Zamvil SS, et al. Glatiramer acetate in the treatment of multiple sclerosis: emerging concepts regarding its mechanism of action. CNS Drugs. 2011;25(5):401–14. doi: 10.2165/11588120-000000000-00000 ; PubMed Central PMCID: PMCPMC3963480.  [DOI](https://doi.org/10.2165/11588120-000000000-00000) | [DOI](https://doi.org/10.2165/11588120-000000000-00000) | [PMC free article](/articles/PMC3963480/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21476611/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=CNS%20Drugs&title=Glatiramer%20acetate%20in%20the%20treatment%20of%20multiple%20sclerosis:%20emerging%20concepts%20regarding%20its%20mechanism%20of%20action&author=PH%20Lalive&author=O%20Neuhaus&author=M%20Benkhoucha&author=D%20Burger&author=R%20Hohlfeld&volume=25&issue=5&publication_year=2011&pages=401-14&pmid=21476611&doi=10.2165/11588120-000000000-00000&)

5. Aharoni R, Kayhan B, Brenner O, Domev H, Labunskay G, Arnon R. Immunomodulatory therapeutic effect of glatiramer acetate on several murine models of inflammatory bowel disease. J Pharmacol Exp Ther. 2006;318(1):68–78. doi: 10.1124/jpet.106.103192 .  [DOI](https://doi.org/10.1124/jpet.106.103192) | [DOI](https://doi.org/10.1124/jpet.106.103192) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16624971/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharmacol%20Exp%20Ther&title=Immunomodulatory%20therapeutic%20effect%20of%20glatiramer%20acetate%20on%20several%20murine%20models%20of%20inflammatory%20bowel%20disease&author=R%20Aharoni&author=B%20Kayhan&author=O%20Brenner&author=H%20Domev&author=G%20Labunskay&volume=318&issue=1&publication_year=2006&pages=68-78&pmid=16624971&doi=10.1124/jpet.106.103192&)

6. Aharoni R, Kayhan B, Eilam R, Sela M, Arnon R. Glatiramer acetate-specific T cells in the brain express T helper 2/3 cytokines and brain-derived neurotrophic factor in situ. Proc Natl Acad Sci U S A. 2003;100(24):14157–62. doi: 10.1073/pnas.2336171100 ; PubMed Central PMCID: PMCPMC283562.  [DOI](https://doi.org/10.1073/pnas.2336171100) | [DOI](https://doi.org/10.1073/pnas.2336171100) | [PMC free article](/articles/PMC283562/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14614135/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&title=Glatiramer%20acetate-specific%20T%20cells%20in%20the%20brain%20express%20T%20helper%202/3%20cytokines%20and%20brain-derived%20neurotrophic%20factor%20in%20situ&author=R%20Aharoni&author=B%20Kayhan&author=R%20Eilam&author=M%20Sela&author=R%20Arnon&volume=100&issue=24&publication_year=2003&pages=14157-62&pmid=14614135&doi=10.1073/pnas.2336171100&)

7. Tsareva EY, Kulakova OG, Boyko AN, Shchur SG, Lvovs D, Favorov AV, et al. Allelic combinations of immune-response genes associated with glatiramer acetate treatment response in Russian multiple sclerosis patients. Pharmacogenomics. 2012;13(1):43–53. doi: 10.2217/pgs.11.136 .  [DOI](https://doi.org/10.2217/pgs.11.136) | [DOI](https://doi.org/10.2217/pgs.11.136) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22111603/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Allelic%20combinations%20of%20immune-response%20genes%20associated%20with%20glatiramer%20acetate%20treatment%20response%20in%20Russian%20multiple%20sclerosis%20patients&author=EY%20Tsareva&author=OG%20Kulakova&author=AN%20Boyko&author=SG%20Shchur&author=D%20Lvovs&volume=13&issue=1&publication_year=2012&pages=43-53&pmid=22111603&doi=10.2217/pgs.11.136&)

8. Abifadel M, Varret M, Rabes JP, Allard D, Ouguerram K, Devillers M, et al. Mutations in PCSK9 cause autosomal dominant hypercholesterolemia. Nat Genet. 2003;34(2):154–6. doi: 10.1038/ng1161 .  [DOI](https://doi.org/10.1038/ng1161) | [DOI](https://doi.org/10.1038/ng1161) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12730697/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Mutations%20in%20PCSK9%20cause%20autosomal%20dominant%20hypercholesterolemia&author=M%20Abifadel&author=M%20Varret&author=JP%20Rabes&author=D%20Allard&author=K%20Ouguerram&volume=34&issue=2&publication_year=2003&pages=154-6&pmid=12730697&doi=10.1038/ng1161&)

9. Tg and Hdl Working Group of the Exome Sequencing Project National Heart Lung Blood Institute, Crosby J, Peloso GM, Auer PL, Crosslin DR, Stitziel NO, et al. Loss-of-function mutations in APOC3, triglycerides, and coronary disease. N Engl J Med. 2014;371(1):22–31. doi: 10.1056/NEJMoa1307095 ; PubMed Central PMCID: PMC4180269.  [DOI](https://doi.org/10.1056/NEJMoa1307095) | [DOI](https://doi.org/10.1056/NEJMoa1307095) | [PMC free article](/articles/PMC4180269/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24941081/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Loss-of-function%20mutations%20in%20APOC3,%20triglycerides,%20and%20coronary%20disease&author=J%20Crosby&author=GM%20Peloso&author=PL%20Auer&author=DR%20Crosslin&author=NO%20Stitziel&volume=371&issue=1&publication_year=2014&pages=22-31&pmid=24941081&doi=10.1056/NEJMoa1307095&)

10. Coding Variation in ANGPTL4, LPL, and SVEP1 and the Risk of Coronary Disease. N Engl J Med. 2016;374(19):1898 doi: 10.1056/NEJMxx160012 .  [DOI](https://doi.org/10.1056/NEJMxx160012) | [DOI](https://doi.org/10.1056/NEJMx160012) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27123876/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Coding%20Variation%20in%20ANGPTL4,%20LPL,%20and%20SVEP1%20and%20the%20Risk%20of%20Coronary%20Disease&volume=374&issue=19&publication_year=2016&pages=1898&pmid=27123876&doi=10.1056/NEJMx160012&)

11. Schunkert H. Brotherhood of genetics and preventive medication. Eur Heart J. 2015;36(25):1566–8. doi: 10.1093/eurheartj/ehv153 .  [DOI](https://doi.org/10.1093/eurheartj/ehv153) | [DOI](https://doi.org/10.1093/eurheartj/ehv153) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25953708/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20Heart%20J&title=Brotherhood%20of%20genetics%20and%20preventive%20medication&author=H%20Schunkert&volume=36&issue=25&publication_year=2015&pages=1566-8&pmid=25953708&doi=10.1093/eurheartj/ehv153&)

12. Johnson JA, Cavallari LH. Pharmacogenetics and cardiovascular disease—implications for personalized medicine. Pharmacol Rev. 2013;65(3):987–1009. doi: 10.1124/pr.112.007252 ; PubMed Central PMCID: PMCPMC3698938.  [DOI](https://doi.org/10.1124/pr.112.007252) | [DOI](https://doi.org/10.1124/pr.112.007252) | [PMC free article](/articles/PMC3698938/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23686351/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacol%20Rev&title=Pharmacogenetics%20and%20cardiovascular%20disease%E2%80%94implications%20for%20personalized%20medicine&author=JA%20Johnson&author=LH%20Cavallari&volume=65&issue=3&publication_year=2013&pages=987-1009&pmid=23686351&doi=10.1124/pr.112.007252&)

13. Brænne I, Willenborg C, Reiz B, Zeng L, Tragante do Ó V, Meitinger T, et al. Pleiotropic molecular targets of coxibs reveals novel genomic loci conferring coronary artery disease risk. Atherosclerosis. 252:e252–e3. doi: 10.1016/j.atherosclerosis.2016.07.066  [DOI](https://doi.org/10.1016/j.atherosclerosis.2016.07.066) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Atherosclerosis&title=Pleiotropic%20molecular%20targets%20of%20coxibs%20reveals%20novel%20genomic%20loci%20conferring%20coronary%20artery%20disease%20risk&author=I%20Br%C3%A6nne&author=C%20Willenborg&author=B%20Reiz&author=L%20Zeng&author=V%20Tragante%20do%20%C3%93&volume=252&pages=e252-e3&)

14. Consortium CAD. A comprehensive 1000 Genomes-based genome-wide association meta-analysis of coronary artery disease. Nat Genet. 2015. doi: 10.1038/ng.3396 .  [DOI](https://doi.org/10.1038/ng.3396) | [DOI](https://doi.org/10.1038/ng.3396) | [PMC free article](/articles/PMC4589895/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26343387/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=A%20comprehensive%201000%20Genomes-based%20genome-wide%20association%20meta-analysis%20of%20coronary%20artery%20disease&publication_year=2015&pmid=26343387&doi=10.1038/ng.3396&)

15. Griffith M, Griffith OL, Coffman AC, Weible JV, McMichael JF, Spies NC, et al. DGIdb: mining the druggable genome. Nat Methods. 2013;10(12):1209–10. doi: 10.1038/nmeth.2689 ; PubMed Central PMCID: PMC3851581.  [DOI](https://doi.org/10.1038/nmeth.2689) | [DOI](https://doi.org/10.1038/nmeth.2689) | [PMC free article](/articles/PMC3851581/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24122041/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Methods&title=DGIdb:%20mining%20the%20druggable%20genome&author=M%20Griffith&author=OL%20Griffith&author=AC%20Coffman&author=JV%20Weible&author=JF%20McMichael&volume=10&issue=12&publication_year=2013&pages=1209-10&pmid=24122041&doi=10.1038/nmeth.2689&)

16. Schunkert H, Konig IR, Kathiresan S, Reilly MP, Assimes TL, Holm H, et al. Large-scale association analysis identifies 13 new susceptibility loci for coronary artery disease. Nat Genet. 2011;43(4):333–8. doi: 10.1038/ng.784 ; PubMed Central PMCID: PMC3119261.  [DOI](https://doi.org/10.1038/ng.784) | [DOI](https://doi.org/10.1038/ng.784) | [PMC free article](/articles/PMC3119261/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21378990/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Large-scale%20association%20analysis%20identifies%2013%20new%20susceptibility%20loci%20for%20coronary%20artery%20disease&author=H%20Schunkert&author=IR%20Konig&author=S%20Kathiresan&author=MP%20Reilly&author=TL%20Assimes&volume=43&issue=4&publication_year=2011&pages=333-8&pmid=21378990&doi=10.1038/ng.784&)

17. CARDIoGRAMplusC4D-Consortium, Deloukas P, Kanoni S, Willenborg C, Farrall M, Assimes TL, et al. Large-scale association analysis identifies new risk loci for coronary artery disease. Nat Genet. 2013;45(1):25–33. doi: 10.1038/ng.2480 ; PubMed Central PMCID: PMC3679547.  [DOI](https://doi.org/10.1038/ng.2480) | [DOI](https://doi.org/10.1038/ng.2480) | [PMC free article](/articles/PMC3679547/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23202125/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Large-scale%20association%20analysis%20identifies%20new%20risk%20loci%20for%20coronary%20artery%20disease&author=P%20Deloukas&author=S%20Kanoni&author=C%20Willenborg&author=M%20Farrall&author=TL%20Assimes&volume=45&issue=1&publication_year=2013&pages=25-33&pmid=23202125&doi=10.1038/ng.2480&)

18. DerSimonian R, Laird N. Meta-analysis in clinical trials. Control Clin Trials. 1986;7(3):177–88. .  [DOI](https://doi.org/10.1016/0197-2456(86)90046-2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/3802833/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Control%20Clin%20Trials&title=Meta-analysis%20in%20clinical%20trials&author=R%20DerSimonian&author=N%20Laird&volume=7&issue=3&publication_year=1986&pages=177-88&pmid=3802833&doi=10.1016/0197-2456(86)90046-2&)

19. Ward LD, Kellis M. HaploReg: a resource for exploring chromatin states, conservation, and regulatory motif alterations within sets of genetically linked variants. Nucleic Acids Res. 2012;40(Database issue):D930–4. doi: 10.1093/nar/gkr917 ; PubMed Central PMCID: PMC3245002.  [DOI](https://doi.org/10.1093/nar/gkr917) | [DOI](https://doi.org/10.1093/nar/gkr917) | [PMC free article](/articles/PMC3245002/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22064851/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=HaploReg:%20a%20resource%20for%20exploring%20chromatin%20states,%20conservation,%20and%20regulatory%20motif%20alterations%20within%20sets%20of%20genetically%20linked%20variants&author=LD%20Ward&author=M%20Kellis&volume=40&publication_year=2012&pages=D930-4&pmid=22064851&doi=10.1093/nar/gkr917&)

20. Yang J, Ferreira T, Morris AP, Medland SE, Genetic Investigation of ATC, Replication DIG, et al. Conditional and joint multiple-SNP analysis of GWAS summary statistics identifies additional variants influencing complex traits. Nat Genet. 2012;44(4):369–75, S1-3. doi: 10.1038/ng.2213 ; PubMed Central PMCID: PMCPMC3593158.  [DOI](https://doi.org/10.1038/ng.2213) | [DOI](https://doi.org/10.1038/ng.2213) | [PMC free article](/articles/PMC3593158/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22426310/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Conditional%20and%20joint%20multiple-SNP%20analysis%20of%20GWAS%20summary%20statistics%20identifies%20additional%20variants%20influencing%20complex%20traits&author=J%20Yang&author=T%20Ferreira&author=AP%20Morris&author=SE%20Medland&volume=44&issue=4&publication_year=2012&pages=369-75&pmid=22426310&doi=10.1038/ng.2213&)

21. Westra HJ, Peters MJ, Esko T, Yaghootkar H, Schurmann C, Kettunen J, et al. Systematic identification of trans eQTLs as putative drivers of known disease associations. Nat Genet. 2013;45(10):1238–43. doi: 10.1038/ng.2756 ; PubMed Central PMCID: PMCPMC3991562.  [DOI](https://doi.org/10.1038/ng.2756) | [DOI](https://doi.org/10.1038/ng.2756) | [PMC free article](/articles/PMC3991562/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24013639/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Systematic%20identification%20of%20trans%20eQTLs%20as%20putative%20drivers%20of%20known%20disease%20associations&author=HJ%20Westra&author=MJ%20Peters&author=T%20Esko&author=H%20Yaghootkar&author=C%20Schurmann&volume=45&issue=10&publication_year=2013&pages=1238-43&pmid=24013639&doi=10.1038/ng.2756&)

22. Consortium GT. Human genomics. The Genotype-Tissue Expression (GTEx) pilot analysis: multitissue gene regulation in humans. Science. 2015;348(6235):648–60. doi: 10.1126/science.1262110 .  [DOI](https://doi.org/10.1126/science.1262110) | [DOI](https://doi.org/10.1126/science.1262110) | [PMC free article](/articles/PMC4547484/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25954001/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=Human%20genomics.%20The%20Genotype-Tissue%20Expression%20(GTEx)%20pilot%20analysis:%20multitissue%20gene%20regulation%20in%20humans&volume=348&issue=6235&publication_year=2015&pages=648-60&pmid=25954001&doi=10.1126/science.1262110&)

23. Leslie R, O'Donnell CJ, Johnson AD. GRASP: analysis of genotype-phenotype results from 1390 genome-wide association studies and corresponding open access database. Bioinformatics. 2014;30(12):i185–94. doi: 10.1093/bioinformatics/btu273 ; PubMed Central PMCID: PMC4072913.  [DOI](https://doi.org/10.1093/bioinformatics/btu273) | [DOI](https://doi.org/10.1093/bioinformatics/btu273) | [PMC free article](/articles/PMC4072913/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24931982/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=GRASP:%20analysis%20of%20genotype-phenotype%20results%20from%201390%20genome-wide%20association%20studies%20and%20corresponding%20open%20access%20database&author=R%20Leslie&author=CJ%20O'Donnell&author=AD%20Johnson&volume=30&issue=12&publication_year=2014&pages=i185-94&pmid=24931982&doi=10.1093/bioinformatics/btu273&)

24. Boyle AP, Hong EL, Hariharan M, Cheng Y, Schaub MA, Kasowski M, et al. Annotation of functional variation in personal genomes using RegulomeDB. Genome Res. 2012;22(9):1790–7. doi: 10.1101/gr.137323.112 ; PubMed Central PMCID: PMCPMC3431494.  [DOI](https://doi.org/10.1101/gr.137323.112) | [DOI](https://doi.org/10.1101/gr.137323.112) | [PMC free article](/articles/PMC3431494/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22955989/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Res&title=Annotation%20of%20functional%20variation%20in%20personal%20genomes%20using%20RegulomeDB&author=AP%20Boyle&author=EL%20Hong&author=M%20Hariharan&author=Y%20Cheng&author=MA%20Schaub&volume=22&issue=9&publication_year=2012&pages=1790-7&pmid=22955989&doi=10.1101/gr.137323.112&)

25. Weber MS, Hohlfeld R, Zamvil SS. Mechanism of action of glatiramer acetate in treatment of multiple sclerosis. Neurotherapeutics. 2007;4(4):647–53. doi: 10.1016/j.nurt.2007.08.002 .  [DOI](https://doi.org/10.1016/j.nurt.2007.08.002) | [DOI](https://doi.org/10.1016/j.nurt.2007.08.002) | [PMC free article](/articles/PMC7479674/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17920545/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neurotherapeutics&title=Mechanism%20of%20action%20of%20glatiramer%20acetate%20in%20treatment%20of%20multiple%20sclerosis&author=MS%20Weber&author=R%20Hohlfeld&author=SS%20Zamvil&volume=4&issue=4&publication_year=2007&pages=647-53&pmid=17920545&doi=10.1016/j.nurt.2007.08.002&)

26. Grainger DJ. TGF-beta and atherosclerosis in man. Cardiovasc Res. 2007;74(2):213–22. doi: 10.1016/j.cardiores.2007.02.022 .  [DOI](https://doi.org/10.1016/j.cardiores.2007.02.022) | [DOI](https://doi.org/10.1016/j.cardiores.2007.02.022) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17382916/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cardiovasc%20Res&title=TGF-beta%20and%20atherosclerosis%20in%20man&author=DJ%20Grainger&volume=74&issue=2&publication_year=2007&pages=213-22&pmid=17382916&doi=10.1016/j.cardiores.2007.02.022&)

27. Mallat Z, Tedgui A. The role of transforming growth factor beta in atherosclerosis: novel insights and future perspectives. Curr Opin Lipidol. 2002;13(5):523–9. .  [DOI](https://doi.org/10.1097/00041433-200210000-00008) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12352016/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Opin%20Lipidol&title=The%20role%20of%20transforming%20growth%20factor%20beta%20in%20atherosclerosis:%20novel%20insights%20and%20future%20perspectives&author=Z%20Mallat&author=A%20Tedgui&volume=13&issue=5&publication_year=2002&pages=523-9&pmid=12352016&doi=10.1097/00041433-200210000-00008&)

28. Grainger DJ. Transforming growth factor beta and atherosclerosis: so far, so good for the protective cytokine hypothesis. Arterioscler Thromb Vasc Biol. 2004;24(3):399–404. doi: 10.1161/01.ATV.0000114567.76772.33 .  [DOI](https://doi.org/10.1161/01.ATV.0000114567.76772.33) | [DOI](https://doi.org/10.1161/01.ATV.0000114567.76772.33) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14699019/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arterioscler%20Thromb%20Vasc%20Biol&title=Transforming%20growth%20factor%20beta%20and%20atherosclerosis:%20so%20far,%20so%20good%20for%20the%20protective%20cytokine%20hypothesis&author=DJ%20Grainger&volume=24&issue=3&publication_year=2004&pages=399-404&pmid=14699019&doi=10.1161/01.ATV.0000114567.76772.33&)

29. Bobik A. Transforming growth factor-betas and vascular disorders. Arterioscler Thromb Vasc Biol. 2006;26(8):1712–20. doi: 10.1161/01.ATV.0000225287.20034.2c .  [DOI](https://doi.org/10.1161/01.ATV.0000225287.20034.2c) | [DOI](https://doi.org/10.1161/01.ATV.0000225287.20034.2c) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16675726/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arterioscler%20Thromb%20Vasc%20Biol&title=Transforming%20growth%20factor-betas%20and%20vascular%20disorders&author=A%20Bobik&volume=26&issue=8&publication_year=2006&pages=1712-20&pmid=16675726&doi=10.1161/01.ATV.0000225287.20034.2c&)

30. Wang XL, Liu SX, Wilcken DE. Circulating transforming growth factor beta 1 and coronary artery disease. Cardiovasc Res. 1997;34(2):404–10. .  [DOI](https://doi.org/10.1016/s0008-6363(97)00033-3) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9205555/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cardiovasc%20Res&title=Circulating%20transforming%20growth%20factor%20beta%201%20and%20coronary%20artery%20disease&author=XL%20Wang&author=SX%20Liu&author=DE%20Wilcken&volume=34&issue=2&publication_year=1997&pages=404-10&pmid=9205555&doi=10.1016/s0008-6363(97)00033-3&)

31. Ruiz-Ortega M, Rodriguez-Vita J, Sanchez-Lopez E, Carvajal G, Egido J. TGF-beta signaling in vascular fibrosis. Cardiovasc Res. 2007;74(2):196–206. doi: 10.1016/j.cardiores.2007.02.008 .  [DOI](https://doi.org/10.1016/j.cardiores.2007.02.008) | [DOI](https://doi.org/10.1016/j.cardiores.2007.02.008) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17376414/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cardiovasc%20Res&title=TGF-beta%20signaling%20in%20vascular%20fibrosis&author=M%20Ruiz-Ortega&author=J%20Rodriguez-Vita&author=E%20Sanchez-Lopez&author=G%20Carvajal&author=J%20Egido&volume=74&issue=2&publication_year=2007&pages=196-206&pmid=17376414&doi=10.1016/j.cardiores.2007.02.008&)

32. Bujak M, Frangogiannis NG. The role of TGF-beta signaling in myocardial infarction and cardiac remodeling. Cardiovasc Res. 2007;74(2):184–95. doi: 10.1016/j.cardiores.2006.10.002 ; PubMed Central PMCID: PMCPMC1924687.  [DOI](https://doi.org/10.1016/j.cardiores.2006.10.002) | [DOI](https://doi.org/10.1016/j.cardiores.2006.10.002) | [PMC free article](/articles/PMC1924687/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17109837/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cardiovasc%20Res&title=The%20role%20of%20TGF-beta%20signaling%20in%20myocardial%20infarction%20and%20cardiac%20remodeling&author=M%20Bujak&author=NG%20Frangogiannis&volume=74&issue=2&publication_year=2007&pages=184-95&pmid=17109837&doi=10.1016/j.cardiores.2006.10.002&)

33. Gordon KJ, Blobe GC. Role of transforming growth factor-beta superfamily signaling pathways in human disease. Biochim Biophys Acta. 2008;1782(4):197–228. doi: 10.1016/j.bbadis.2008.01.006 .  [DOI](https://doi.org/10.1016/j.bbadis.2008.01.006) | [DOI](https://doi.org/10.1016/j.bbadis.2008.01.006) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18313409/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochim%20Biophys%20Acta&title=Role%20of%20transforming%20growth%20factor-beta%20superfamily%20signaling%20pathways%20in%20human%20disease&author=KJ%20Gordon&author=GC%20Blobe&volume=1782&issue=4&publication_year=2008&pages=197-228&pmid=18313409&doi=10.1016/j.bbadis.2008.01.006&)

34. Zeng L, Dang TA, Schunkert H. Genetics links between transforming growth factor beta pathway and coronary disease. Atherosclerosis. 2016;253:237–46. doi: 10.1016/j.atherosclerosis.2016.08.029 .  [DOI](https://doi.org/10.1016/j.atherosclerosis.2016.08.029) | [DOI](https://doi.org/10.1016/j.atherosclerosis.2016.08.029) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27596813/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Atherosclerosis&title=Genetics%20links%20between%20transforming%20growth%20factor%20beta%20pathway%20and%20coronary%20disease&author=L%20Zeng&author=TA%20Dang&author=H%20Schunkert&volume=253&publication_year=2016&pages=237-46&pmid=27596813&doi=10.1016/j.atherosclerosis.2016.08.029&)

35. Braenne I, Civelek M, Vilne B, Di Narzo A, Johnson AD, Zhao Y, et al. Prediction of Causal Candidate Genes in Coronary Artery Disease Loci. Arterioscler Thromb Vasc Biol. 2015;35(10):2207–17. doi: 10.1161/ATVBAHA.115.306108 ; PubMed Central PMCID: PMCPMC4583353.  [DOI](https://doi.org/10.1161/ATVBAHA.115.306108) | [DOI](https://doi.org/10.1161/ATVBAHA.115.306108) | [PMC free article](/articles/PMC4583353/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26293461/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arterioscler%20Thromb%20Vasc%20Biol&title=Prediction%20of%20Causal%20Candidate%20Genes%20in%20Coronary%20Artery%20Disease%20Loci&author=I%20Braenne&author=M%20Civelek&author=B%20Vilne&author=A%20Di%20Narzo&author=AD%20Johnson&volume=35&issue=10&publication_year=2015&pages=2207-17&pmid=26293461&doi=10.1161/ATVBAHA.115.306108&)

36. Lebrin F, Deckers M, Bertolino P, Ten Dijke P. TGF-beta receptor function in the endothelium. Cardiovasc Res. 2005;65(3):599–608. doi: 10.1016/j.cardiores.2004.10.036 .  [DOI](https://doi.org/10.1016/j.cardiores.2004.10.036) | [DOI](https://doi.org/10.1016/j.cardiores.2004.10.036) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15664386/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cardiovasc%20Res&title=TGF-beta%20receptor%20function%20in%20the%20endothelium&author=F%20Lebrin&author=M%20Deckers&author=P%20Bertolino&author=P%20Ten%20Dijke&volume=65&issue=3&publication_year=2005&pages=599-608&pmid=15664386&doi=10.1016/j.cardiores.2004.10.036&)

37. Yamamoto K, Morishita R, Tomita N, Shimozato T, Nakagami H, Kikuchi A, et al. Ribozyme oligonucleotides against transforming growth factor-beta inhibited neointimal formation after vascular injury in rat model: potential application of ribozyme strategy to treat cardiovascular disease. Circulation. 2000;102(11):1308–14. .  [DOI](https://doi.org/10.1161/01.cir.102.11.1308) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10982548/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circulation&title=Ribozyme%20oligonucleotides%20against%20transforming%20growth%20factor-beta%20inhibited%20neointimal%20formation%20after%20vascular%20injury%20in%20rat%20model:%20potential%20application%20of%20ribozyme%20strategy%20to%20treat%20cardiovascular%20disease&author=K%20Yamamoto&author=R%20Morishita&author=N%20Tomita&author=T%20Shimozato&author=H%20Nakagami&volume=102&issue=11&publication_year=2000&pages=1308-14&pmid=10982548&doi=10.1161/01.cir.102.11.1308&)

38. Ward MR, Agrotis A, Kanellakis P, Hall J, Jennings G, Bobik A. Tranilast prevents activation of transforming growth factor-beta system, leukocyte accumulation, and neointimal growth in porcine coronary arteries after stenting. Arterioscler Thromb Vasc Biol. 2002;22(6):940–8. .  [DOI](https://doi.org/10.1161/01.atv.0000019405.84384.9c) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12067902/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arterioscler%20Thromb%20Vasc%20Biol&title=Tranilast%20prevents%20activation%20of%20transforming%20growth%20factor-beta%20system,%20leukocyte%20accumulation,%20and%20neointimal%20growth%20in%20porcine%20coronary%20arteries%20after%20stenting&author=MR%20Ward&author=A%20Agrotis&author=P%20Kanellakis&author=J%20Hall&author=G%20Jennings&volume=22&issue=6&publication_year=2002&pages=940-8&pmid=12067902&doi=10.1161/01.atv.0000019405.84384.9c&)

39. Schini-Kerth VB, Bassus S, Fisslthaler B, Kirchmaier CM, Busse R. Aggregating human platelets stimulate the expression of thrombin receptors in cultured vascular smooth muscle cells via the release of transforming growth factor-beta1 and platelet-derived growth factorAB. Circulation. 1997;96(11):3888–96. .  [DOI](https://doi.org/10.1161/01.cir.96.11.3888) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9403612/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circulation&title=Aggregating%20human%20platelets%20stimulate%20the%20expression%20of%20thrombin%20receptors%20in%20cultured%20vascular%20smooth%20muscle%20cells%20via%20the%20release%20of%20transforming%20growth%20factor-beta1%20and%20platelet-derived%20growth%20factorAB&author=VB%20Schini-Kerth&author=S%20Bassus&author=B%20Fisslthaler&author=CM%20Kirchmaier&author=R%20Busse&volume=96&issue=11&publication_year=1997&pages=3888-96&pmid=9403612&doi=10.1161/01.cir.96.11.3888&)

40. Lev PR, Salim JP, Marta RF, Osorio MJ, Goette NP, Molinas FC. Platelets possess functional TGF-beta receptors and Smad2 protein. Platelets. 2007;18(1):35–42. doi: 10.1080/09537100600800743 .  [DOI](https://doi.org/10.1080/09537100600800743) | [DOI](https://doi.org/10.1080/09537100600800743) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17365852/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Platelets&title=Platelets%20possess%20functional%20TGF-beta%20receptors%20and%20Smad2%20protein&author=PR%20Lev&author=JP%20Salim&author=RF%20Marta&author=MJ%20Osorio&author=NP%20Goette&volume=18&issue=1&publication_year=2007&pages=35-42&pmid=17365852&doi=10.1080/09537100600800743&)

41. Mo JW, Zhang DF, Ji GL, Liu XZ, Fan B. TGF-beta1 and Serpine 1 expression changes in traumatic deep vein thrombosis. Genet Mol Res. 2015;14(4):13835–42. doi: 10.4238/2015.October.29.3 .  [DOI](https://doi.org/10.4238/2015.October.29.3) | [DOI](https://doi.org/10.4238/2015.October.29.3) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26535698/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genet%20Mol%20Res&title=TGF-beta1%20and%20Serpine%201%20expression%20changes%20in%20traumatic%20deep%20vein%20thrombosis&author=JW%20Mo&author=DF%20Zhang&author=GL%20Ji&author=XZ%20Liu&author=B%20Fan&volume=14&issue=4&publication_year=2015&pages=13835-42&pmid=26535698&doi=10.4238/2015.October.29.3&)

42. Lijnen PJ, Petrov VV, Fagard RH. Association between transforming growth factor-beta and hypertension. Am J Hypertens. 2003;16(7):604–11. .  [DOI](https://doi.org/10.1016/s0895-7061(03)00847-1) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12850397/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hypertens&title=Association%20between%20transforming%20growth%20factor-beta%20and%20hypertension&author=PJ%20Lijnen&author=VV%20Petrov&author=RH%20Fagard&volume=16&issue=7&publication_year=2003&pages=604-11&pmid=12850397&doi=10.1016/s0895-7061(03)00847-1&)

43. Kurihara H, Yoshizumi M, Sugiyama T, Takaku F, Yanagisawa M, Masaki T, et al. Transforming growth factor-beta stimulates the expression of endothelin mRNA by vascular endothelial cells. Biochem Biophys Res Commun. 1989;159(3):1435–40. .  [DOI](https://doi.org/10.1016/0006-291x(89)92270-5) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/2649101/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem%20Biophys%20Res%20Commun&title=Transforming%20growth%20factor-beta%20stimulates%20the%20expression%20of%20endothelin%20mRNA%20by%20vascular%20endothelial%20cells&author=H%20Kurihara&author=M%20Yoshizumi&author=T%20Sugiyama&author=F%20Takaku&author=M%20Yanagisawa&volume=159&issue=3&publication_year=1989&pages=1435-40&pmid=2649101&doi=10.1016/0006-291x(89)92270-5&)

44. Stoynev N, Dimova I, Rukova B, Hadjidekova S, Nikolova D, Toncheva D, et al. Gene expression in peripheral blood of patients with hypertension and patients with type 2 diabetes. J Cardiovasc Med (Hagerstown). 2014;15(9):702–9. doi: 10.2459/JCM.0b013e32835dbcc8 .  [DOI](https://doi.org/10.2459/JCM.0b013e32835dbcc8) | [DOI](https://doi.org/10.2459/JCM.0b013e32835dbcc8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23337395/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Cardiovasc%20Med%20(Hagerstown)&title=Gene%20expression%20in%20peripheral%20blood%20of%20patients%20with%20hypertension%20and%20patients%20with%20type%202%20diabetes&author=N%20Stoynev&author=I%20Dimova&author=B%20Rukova&author=S%20Hadjidekova&author=D%20Nikolova&volume=15&issue=9&publication_year=2014&pages=702-9&pmid=23337395&doi=10.2459/JCM.0b013e32835dbcc8&)
