# Genetic polymorphisms of lncRNA-p53 regulatory network genes are associated with concurrent chemoradiotherapy toxicities and efficacy in nasopharyngeal carcinoma patients

## Metadata
**Authors:** Youhong Wang, Zhen Guo, Yu Zhao, Yi Jin, Liang An, Bin Wu, Zhaoqian Liu, Xiaoping Chen, Xiang Chen, Honghao Zhou, Hui Wang, Wei Zhang
**Journal:** Scientific Reports
**Date:** 2017 Aug 16
**DOI:** [10.1038/s41598-017-08890-2](https://doi.org/10.1038/s41598-017-08890-2)
**PMID:** 28814798
**PMCID:** PMC5559481
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5559481/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC5559481/pdf/41598_2017_Article_8890.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC5559481/pdf/41598_2017_Article_8890.pdf)

## Abstract

The relevance of the transcription factor p53 in cancer is inarguable, and numerous lncRNAs are involved in the p53 regulatory network as either regulators or effectors, triggering a transcriptional response that causes either cell arrest or apoptosis following DNA damage in a p53-dependent manner. Despite the fact that the therapeutic response is improved in NPC, heterogeneity among people remains with regard to the susceptibility of adverse effects and the efficacy of treatments. Therefore, we analysed eight potentially functional SNPs of five genes in the lncRNA-p53 regulatory network in a discovery cohort of 505 NPC patients. By performing multivariate logistic regression, the impact of genetic variations on the efficacy and risk of CRT-induced toxicities was investigated. The most dramatic finding was that the MEG3 rs10132552 CC genotype had a greater than three-fold increased risk of developing grade 3–4 anaemia (OR = 3.001, 95%CI = 1.355–6.646, P = 0.007). Furthermore, the rs10132552 CT genotype had a better response to treatment (OR = 0.261, 95%CI = 0.089–0.770, P = 0.015). Individuals carrying LINC-ROR rs2027701 with one or two variant alleles had significant associations with a reduced risk of neutropaenia (OR = 0.503, 95%CI = 0.303–0.835, P = 0.008). In conclusion, our results suggested that genetic polymorphisms of the lncRNA-p53 regulatory network could play a potential role in reducing treatment-related toxicities and improving outcomes for NPC patients.

## Introduction

Nasopharyngeal carcinoma (NPC) is an epithelial malignancy with extremely skewed ethnic and geographic distributions and a particularly high prevalence in southern China^1^. Concurrent chemoradiotherapy (CRT), an important therapeutic milestone, is the standard treatment for locally advanced NPC. Although overall survival has been dramatically improved by the advancement of radiotherapy technology as well as the broader application of chemotherapy, CRT-induced acute toxicity remains a challenge, as it is multifactorial and difficult to predict^2^. Chemoradiotherapy is invariably associated with higher incidences of haematological and non-haematological acute toxic effects compared with radiotherapy alone^3–5^. There is significant variation in prognosis and the risk of toxicities among patients, even if they are exposed to the same therapeutic regimens, suggesting that genetic polymorphisms play a crucial role in individual susceptibility to toxicities and sensitivity to treatments^6,7^.

In support of this notion, many studies have illustrated that single nucleotide polymorphisms (SNPs) may be useful as independent factors for predicting the toxicities and curative efficacy of chemoradiotherapy in human cancers including NPC. For example, the rs1982073 polymorphism of *TGFB1* seems to trend with a higher risk of mucositis in head and neck squamous cell carcinoma when patients underwent platinum-based CRT^8^. As indicated by Ming Jia^9^, the *GADD45B* rs2024144T variant allele correlated with a major event in response to severe haematologic toxicities in individuals with non-small cell lung cancer. Similarly, SNPs in DNA repair pathway genes were correlated with sensitivity to radiotherapy and chemotherapy^10^.

LncRNA is a type of long non-coding RNA with transcripts >200 nt in length that functioning as a master regulator controlling protein-coding and non-coding genes at multiple levels; lncRNAs could drive important cancer phenotypes and serve as a biomarker in diverse cancers such as NPC^11–14^. Undoubtedly, p53, a master human tumour suppressor, participates in all steps of tumour initiation and development by regulating the expression of many downstream genes, whose dysfunction is closely related to the occurrence and progression of NPC^15^. LncRNA has now been added to the p53 regulatory pathway **(**Fig. [1](#Fig1) **)** to form a sophisticated regulatory network^16,17^ that has generated increased attention for its potential to contribute to disease.

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5db1/5559481/32e0efa227b3/41598_2017_8890_Fig1_HTML.jpg)

LncRNAs serve as p53 regulators and effectors, participating in the p53 regulatory pathway. On the one hand, lncRNAs are able to regulate p53 directly or indirectly at the transcriptional or posttranscriptional levels such as MEG3. On the other hand, several lncRNAs could be induced or suppressed by p53 such as LINC-ROR, pR-lncRNA-1, LINC-PINT and TUSC7.

On one hand, lncRNAs act as p53 effectors that are regulated by p53 directly and modulate gene expression downstream of p53. For example, *LINC-ROR*, *pR-lncRNA-1*, *LINC-PINT* and *TUSC7* alter the interaction between p53 and potential p53 response elements (p53REs) when confronted with cellular stresses such as the DNA damage induced by radiation and/or chemotherapeutics by regulating cell proliferation, cell cycle and cell apoptosis^18–21^. On the other hand, lncRNAs such as *MEG3* can serve as p53 regulators by controlling p53 stability^17^. Based on these examples, we selected eight potential SNPs in five genes *MEG3* (rs10132552T > C)*, LINC-ROR* (rs2027701A > G), *pR-lncRNA-1* (rs73594404G > A and rs3743773G > A), *LINC-PINT* (rs1059698A > C and rs2293750T > A) and *TUSC7* (rs1829346C > A and rs36080650T > C) to determine whether genetic polymorphisms of lncRNA-p53 regulatory network genes are associated with toxicities or the therapeutic efficacy of concurrent chemoradiotherapy in NPC in hopes of discovering valuable new biomarkers for personalized CRT among NPC patients.

## Results

### Patient Clinical Characteristics and Genotype Distribution

The studied cohort included 374 males and 131 females, with a mean age of 47.41 (ranging from 15 to 73) years old. Among these patients, 455 individuals (90.1%) were diagnosed at late stages (III and IV), and the other patients (9.9%) were at early stages (I and II). All of the patients were treated with IMRT, and the median total radiation dose was 71.34 Gy. Regarding chemotherapy, regimes of platinum-based inducing and concurrent chemotherapy were given to participants. The demographic characteristics of NPC patients are described in Table [1](#Tab1). Although none of the patients in this study suffered death caused by toxicities, 51 (10.1%), 129 (25.5%), 121 (24.0%), 73 (14.5%) and 94 (18.6%) experienced grade 3–4 dermatitis, mucositis, myelosuppression, leukopaenia and neutropaenia, respectively. Furthermore, 25 (6.0%) and 62 (15.0%) experienced worse treatment efficacy of CRT at the primary tumour and lymph node, respectively, months after treatments.

### Table 1.

| Patient characteristics | N = 505(%) |
| --- | --- |
| Gender |  |
| Male | 374 (74.1) |
| Female | 131 (25.9) |
| Age, years |  |
| Mean ± SD | 47.41 ± 9.15 |
| <47 | 229 (45.3) |
| ≥47 | 276 (54.7) |
| BMI |  |
| <18.5 | 30 (5.9) |
| 18.5 ~ 24 | 274 (54.3) |
| ≥24 | 201 (39.8) |
| Smoking status |  |
| Smoker | 247 (48.9) |
| Nonsmoker | 258 (51.1) |
| Drinking status |  |
| Drinker | 90 (17.8) |
| Nondrinker | 415 (82.2) |
| Histological type |  |
| WHO type II | 214 (42.4) |
| WHO type III | 291 (57.6) |
| Clinical stage a |  |
| I–II | 50 (9.9) |
| III–IV | 455 (90.1) |
| T-staging |  |
| T1–T2 | 246 (48.7) |
| T3–T4 | 259 (51.3) |
| N-staging |  |
| N0–N1 | 93 (18.4) |
| N2–N3 | 412 (81.6) |
| Grade 3–4 toxic reactions |  |
| Dermatitis | 51 (10.1%) |
| Mucositis | 129 (25.5%) |
| Leukopenia | 73 (14.5%) |
| Neutropenia | 94 (18.6%) |
| Myelosuppression | 121 (24.0%) |
| IC regimen |  |
| DP | 200 (39.6) |
| FP | 92 (18.2) |
| TP | 203 (40.2) |
| GP | 10 (2) |
| CRT regimen |  |
| FP | 85 (16.8) |
| TP | 108 (21.4) |
| DDP | 83 (16.4) |
| NDP | 172 (34.1) |
| DP | 57 (11.3) |
| pGTVnx |  |
| Mean ± SD | 71.34 ± 2.79 |
| <71.00 Gy | 261 (51.7) |
| ≥71.00 Gy | 234 (48.3) |

Table 1 Caption: Patient demographics and clinical characteristics.

The characteristics of the 8 SNPs are shown in Table [2](#Tab2). The allelice frequencies of the enrolled SNPs all fit Hardy-Weinberg equilibrium (*P* > 0.05).

### Table 2.

| Gene | SNP | SNP Location | Alleles | Genotype Distributiona | HWE | MAF | Detectable Rate (%) |
| --- | --- | --- | --- | --- | --- | --- | --- |
| LINC-ROR | rs2027701 | chr18:54724945-54725445 | A/G | 174/247/81 | 0.670 | 0.3810 | 99.4 |
| pR-lncRNA-1 | rs73594404 | chr16:53079364-53079864 | G/A | 453/50/2 | 0.625 | 0.0714 | 100 |
| rs3743773 | chr16:53077757-53078257 | G/A | 313/167/25 | 0.656 | 0.2476 | 100 |  |
| LINC-PINT | rs1059698 | chr7:130628844-130629344 | A/C | 223/222/60 | 0.677 | 0.3238 | 100 |
| rs2293750 | chr7:130629522-130630022 | T/A | 155/252/96 | 0.720 | 0.4143 | 99.6 |  |
| MEG3 | rs10132552 | chr14:101300762-101301262 | T/C | 251/208/37 | 0.496 | 0.2952 | 98.2 |
| TUSC7 | rs1829346 | chr3:116428657-116429157 | C/A | 240/219/46 | 0.694 | 0.3476 | 100 |
| rs36080650 | chr3:116431337-116431837 | T/C | 241/219/45 | 0.635 | 0.3571 | 100 |  |

Table 2 Caption: Genotype distribution and MAF of 8 SNPs in LINC-ROR, pR-lncRNA-1, LINC-PINT, MEG3 and TUSC7.

### TUSC7 SNP and the Risk of CRT-induced Dermatitis

We demonstrated that *TUSC7* rs1829346 and rs36080650 were significantly associated with dermatitis **(**Table [3](#Tab3) **)**. Patients carrying the AA genotype of rs1829346 were less resistant to grade 3–4 CRT-induced dermatitis (OR = 2.641, 95%CI = 1.118–6.243, *P* = 0.027). Similarly, the rs36080650 CC genotype was also associated with a prominently higher risk of dermatitis than the CT/TT genotypes (OR = 2.700, 95%CI = 1.141–6.386, *P* = 0.024), which was supported by the recessive model (CC vs CT + TT, OR = 2.544, 95%CI = 1.143–5.662, *P* = 0.022). However, no significant corrections between the risk of oral mucositis and SNPs were found.

### Table 3.

| Toxic reactions | SNP | Genotypes | Toxicity grade |  | OR (95% CI) | P a |
| --- | --- | --- | --- | --- | --- | --- |
| Dermatitis |   |   | Grade ≤ 2 N (%) | Grade > 2 N (%) |   |   |
| rs1829346 | CC | 219 (48.2) | 21 (41.2) | 1.00 (reference) |   |  |
|   | CA | 198 (43.6) | 21 (41.2) | 1.127 (0.597-2.130) | 0.712 |  |
|   | AA | 37 (8.1) | 9 (17.6) | 2.641 (1.118–6.243) | 0.027 |  |
|   | AA + CA vs CC |   |   | 1.407 (0.767–2.584) | 0.270 |  |
|   | AA vs CA + CC |   |   | 2.492 (1.121–5.540) | 0.025 |  |
| rs36080650 | TT | 220 (48.5) | 21 (41.2) | 1.00 (reference) |   |  |
|   | CT | 198 (43.6) | 21 (41.2) | 1.130 (0.598–2.135) | 0.706 |  |
|   | CC | 36 (7.9) | 9 (17.6) | 2.700 (1.141–6.386) | 0.024 |  |
|   | CC + CT vs TT |   |   | 1.414 (0.770–2.595) | 0.264 |  |
|   | CC vs CT + TT |   |   | 2.544 (1.143–5.662) | 0.022 |  |
| Neutropenia |   |   | Grade ≤ 2 N (%) | Grade > 2 N (%) |   |   |
| rs2027701 | AA | 134 (32.6) | 40 (42.6) | 1.00 (reference) |   |  |
|   | GA | 205 (49.9) | 42 (44.7) | 0.545 (0.319–0.930) | 0.026 |  |
|   | GG | 70 (17.0) | 11 (11.7) | 0.378 (0.171–0.836) | 0.016 |  |
|   | GG + GA vs AA |   |   | 0.503 (0.303–0.835) | 0.008 |  |
|   | GG vs GA + AA |   |   | 0.550 (0.264–1.146) | 0.110 |  |
| rs73594404 | GG | 374 (91.0) | 79 (84.0) | 1.00 (reference) |   |  |
|   | GA | 36 (8.8) | 14 (14.9) | 2.118 (1.011–4.440) | 0.047 |  |
|   | AA | 1 (0.2) | 1 (1.1) | 3.484 (0.128–95.079) | 0.459 |  |
|   | AA + GA vs GG |   |   | 2.164 (1.049–4.464) | 0.037 |  |
|   | AA vs GA + GG |   |   | 3.260 (0.121–87.981) | 0.482 |  |
| rs1059698 | AA | 173 (42.1) | 50 (53.2) | 1.00 (reference) |   |  |
|   | CA | 185 (45.0) | 37 (39.4) | 0.780 (0.469–1.295) | 0.336 |  |
|   | CC | 53 (12.9) | 7 (7.4) | 0.395 (0.161–0.971) | 0.043 |  |
|   | CC + CA vs AA |   |   | 0.660 (0.408–1.070) | 0.092 |  |
|   | CC vs CA + AA |   |   | 0.443 (0.185–1.058) | 0.067 |  |
| Anemia |   |   | Grade ≤ 0 N (%) | Grade > 0 N (%) |   |   |
| rs10132552 | TT | 141 (49.0) | 110 (50.7) | 1.00 (reference) |   |  |
|   | CT | 129 (44.8) | 79 (36.4) | 0.764 (0.500–1.169) | 0.215 |  |
|   | CC | 14 (4.9) | 23 (10.6) | 2.653 (1.172–6.008) | 0.019 |  |
|   | CC + CT vs TT |   |   | 0.929 (0.618–1.398) | 0.725 |  |
|   | CC vs CT + TT |   |   | 3.001 (1.355–6.646) | 0.007 |  |
| rs73594404 | GG | 263 (91.3) | 190 (87.6) | 1.00 (reference) |   |  |
|   | GA | 25 (8.7) | 25 (11.5) | 2.109 (1.062–4.188) | 0.033 |  |
|   | AA | 0 (0) | 2 (0.9) | NA | NA |  |
|   | AA + GA vs GG |   |   | 2.239 (1.138–4.405) | 0.020 |  |
|   | AA vs GA + GG |   |   | NA | NA |  |
| Myelosuppression |   |   | Grade ≤ 2 N (%) | Grade > 2 N (%) |   |   |
| rs1059698 | AA | 161 (41.9) | 62 (51.2) | 1.00 (reference) |   |  |
|   | CA | 172 (44.8) | 50 (41.3) | 0.810 (0.510–1.287) | 0.373 |  |
|   | CC | 51 (13.3) | 9 (7.4) | 0.407 (0.180–0.920) | 0.031 |  |
|   | CC + CA vs AA |   |   | 0.708 (0.456–1.098) | 0.123 |  |
|   | CC vs CA + AA |   |   | 0.449 (0.204–0.987) | 0.046 |  |

Table 3 Caption: Associations between genotypes and concurrent chemoradiotherapy-induced toxicities

### Multivariate Analysis of Selected SNPs as Prognostic Factors of Haematological Toxicities

#### Neutropaenia

Three SNPs were significantly associated with neutropaenia: rs2027701, rs73594404 and rs1059698. *LINC*-*ROR* rs2027701 showed an obvious trend towards a superior reaction with toxic effects in patients with one or two variant alleles compared with those with the wild-type genotype (OR = 0.503, 95%CI = 0.303–0.835, *P* = 0.008). As demonstrated, the *LINC-PINT* rs1059698 CC genotype was a protective factor (OR = 0.395, 95%CI = 0.161–0.971, P = 0.043), whereas *LINC-PINT* rs2293750 had no association with the risk of neutropaenia. In contrast to rs2027701 and rs1059698, rs73594404 *pR-lncRNA-1* had a weak correlation with increased risk of adverse reactions when patients possessed the GA genotype (OR = 2.118, 95%CI = 1.011–4.440, *P* = 0.047).

#### Anaemia

Patients with a minor A allele of *pR-lncRNA-1* rs73594404 had an increased risk of anaemia (OR = 2.109, 95%CI = 1.062–4.188, *P* = 0.033). The *MEG3* rs10132552 CC genotype correlated with a significantly inferior toxic reaction (OR = 2.653, 95%CI = 1.172–6.008, *P* = 0.019), which is an independent predictor for prognosis (CC vs CT + TT, OR = 3.001, 95%CI = 1.355–6.646, *P* = 0.007).

#### Myelosuppression

Statistical results indicated that only *LINC-PINT* rs1059698 polymorphisms had a correlation with CRT-induced myelosuppression. Although this SNP was not correlated with dermatitis and anaemia, we found that it had a significant association with not only the risk of neutropaenia (OR = 0.395, 95%CI = 0.161–0.971, *P* = 0.043), but also myelosuppression (OR = 0.407, 95%CI = 0.180–0.920, *P* = 0.031).

According to the multivariate logistic regression analyses, selected 8 SNPs did not interact with leukopaenia and thrombocytopenia in this study.

### Stratification Analysis of the Association between SNPs in lncRNA-p53 Regulatory Network Genes and Toxic Reactions

Supplementary Table [S1](#MOESM1) lists the relevancies between patient-related, tumour-related and therapy-related characteristics and the risk of toxicities generated by CRT. There was a significant dependency between gender, IC regimen, CRT regimen and multifarious toxicities. Female gender was an adverse factor for toxic reactions. Patients treated with TP, DP or DDP during CRT had a higher risk of anaemia compared with those treated with FP. Moreover, both age and BMI impacted individual risks. Patients with advanced T stages had a more than 1.5-fold greater myelosuppression risk compared with those in the early stages.

According to the above evidence, stratified analysis by adjusting for sex, IR regimen and CRT regimen was conducted to estimate the associations between the enrolled SNP polymorphisms and adverse reactions **(**Table [4](#Tab4) **)**. The rs10132552 CT genotype had an increased risk of leukopaenia, neutropaenia and myelosuppression in subgroups of the DP induction protocol. *pR-lncRNA-1* rs73594404 was another vital SNP that showed a strong relationships with toxicities in the IC regimen subgroup (OR = 3.394 and *P* = 0.015 for leukopaenia; OR = 3.540 and *P* = 0.036 for thrombocytopenia; OR = 3.054 and *P* = 0.022 for myelosuppression). Of significant interest, two SNPs, rs2027701 and rs1059698, had a weak correlation with oral mucositis in the subgroups of female (OR = 3.375 and *P* = 0.045) and IC regimen-DP (OR = 0.527 and *P* = 0.049), respectively.

### Table 4.

| Stratified factors | n | SNP | Toxic reactions | OR (95% CI) | P |
| --- | --- | --- | --- | --- | --- |
| IC regimen-TP | 203 | rs2027701 | Dermatitis | 4.721 (1.141–19.536) | 0.032 |
| rs73594404 | Leukopenia | 3.394 (1.263–9.123) | 0.015 |  |  |
| Myelosuppression | 3.054 (1.178–7.922) | 0.022 |  |  |  |
| IC regimen-DP | 200 | rs1059698 | Mucositis | 0.527 (0.279–0.997) | 0.049 |
| rs10132552 | Leukopenia | 4.300 (1.345–13.748) | 0.014 |  |  |
| Neutropenia | 5.462 (1.836–16.251) | 0.002 |  |  |  |
| rs73594404 | Thrombocytopenia | 3.540 (1.089–11.500) | 0.036 |  |  |
| Sex-Female | 131 | rs2027701 | Mucositis | 3.375 (1.025–11.108) | 0.045 |
| rs73594404 | Thrombocytopenia | 10.237 (2.530–41.429) | 0.001 |  |  |
| rs10132552 | Myelosuppression | 4.135 (1.303–13.119) | 0.016 |  |  |
| CCRT regimen-TP | 108 | rs73594404 | Leukopenia | 3.784 (1.086–13.184) | 0.037 |

Table 4 Caption: Stratification analysis of association between SNPs in lncRNA-p53 regulatory network genes and the toxic reactions in NPC patients.

Hence, to further obtain the predictive power of lncRNAs after adjusting for clinical variables, a risk score model was built in accordance with the regression coefficients of variables to predict each patient’s risk of developing toxicities including neutropaenia, anaemia and myelosuppression (factors involved in P < 0.05). Using receiver operating characteristic (ROC) curve analysis, the prognostic power was evaluated **(**Fig. [2](#Fig2) **)**. The area under the curve (AUC) of neutropaenia, anaemia and myelosuppression was 0.731, 0.750 and 0.648, respectively, indicating good performance of the lncRNAs combined with clinical information for predicting toxicities in NPC patients.

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5db1/5559481/46e58b26faf7/41598_2017_8890_Fig2_HTML.jpg)

Analysis of receiver-operating characteristic curve to predict toxicities. ROC analysis showed the AUC of myelosuppression, neutropaenia and anaemia was 0.647, 0.731 and 0.750, respectively.

### Interaction between Selected SNPs and the Efficacy of CRT 3 Months after Treatment

All positive results concerning the relationship between selected SNPs and the efficacy of CRT on the primary tumour and lymph nodes are listed in Table [5](#Tab5). Our results provide a plausible link between *LINC-ROR* rs2027701 polymorphisms and efficacy at the lymph node 3 months after CRT (OR = 2.266, 95%CI = 1.020–5.033, *P* = 0.045), while no significant difference was found on the primary tumour. In contrast, the *MEG3* rs10132552 CT genotype had a better response to treatments for the primary tumour compared with the TT genotype (OR = 0.261, 95%CI = 0.089–0.770, *P* = 0.015). With respect to the lymph node, rs10132552 polymorphisms are nonfunctional. We also analysed the connections between clinical factors and treatment efficacy; positive results are displayed in Supplementary Tables [S2](#MOESM1).

### Table 5.

| SNP |   | Primary tumor 3 months after treatment |  |  | P c | Lymph node 3 months after treatment |  |  | P c |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| Genotype distribution (%) |  | OR (95% CI) | Genotype distribution |  | OR (95% CI) |  |  |  |  |
| Non-sensitivea | Sensitiveb | Non-sensitivea | Sensitiveb |  |  |  |  |  |  |
| rs2027701 | AA | 8 (32.0) | 125 (32.1) | 1.00 (reference) |   | 15 (24.2) | 118 (33.5) | 1.00 (reference) |   |
| GA | 14 (56.0) | 193 (49.6) | 1.289 (0.467–3.558) | 0.624 | 30 (48.4) | 177 (50.3) | 1.280 (0.643–2.549) | 0.483 |  |
| GG | 3 (12.0) | 68 (17.5) | 0.564 (0.131–2.426) | 0.442 | 17 (27.4) | 54 (15.3) | 2.266 (1.020–5.033) | 0.045 |  |
| rs10132552 | TT | 18 (72.0) | 187 (48.1) | 1.00 (reference) |   | 35 (56.5) | 170 (48.3) | 1.00 (reference) |   |
| CT | 6 (24.0) | 169 (43.4) | 0.261 (0.089–0.770) | 0.015 | 23 (37.1) | 152 (43.2) | 0.735 (0.398–1.358) | 0.326 |  |
| CC | 1 (4.0) | 28 (7.2) | 0.219 (0.024–1.990) | 0.177 | 2(3.2) | 27 (7.7) | 0.331 (0.069–1.588) | 0.167 |  |

Table 5 Caption: Association between rs2027701 and rs10132552 and the efficacy of CRT at the primary tumor and lymph node 3 months after treatment in NPC patients

## Discussion

In this study, we estimated the association of 8 SNPs in the lncRNA-p53 regulatory network of genes and the efficacy and toxic reactions in 505 NPC patients. To the best of our knowledge, this is the first study to demonstrate that 2 SNPs (*TUSC7* rs1829346 and rs36080650) were correlated with dermatitis, 3 SNPs (*LINC-ROR* rs2027701, *pR-lncRNA-1* rs73594404 and *LINC-PINT* rs1059698) were related to neutropaenia, 2 SNPs (*MEG3* rs10132552, *pR-lncRNA-1* rs73594404) and 1 SNP (*LINC-PINT* rs1059698) were associated with anaemia and myelosuppression in NPC patients, respectively. Of these, rs2027701 and rs10132552 were related to curative efficacy 3 months after treatment.

The regulation of the p53 tumour-suppressor pathway by lncRNAs, directly or indirectly, has been a hot topic of particularly intense interest. Non-coding mutations contributing to small changes in gene expression can have a large phenotypic impact on carcinoma, perhaps to an even greater degree than currently appreciated. As regulators, lncRNAs could affect p53 expression by influencing p53 mRNA stability or reducing its ability to recognize some of its binding sites, ultimately inhibiting p53 transcription^22,23^. lncRNAs also act as regulators that activate p53 directly by interacting with p53REs, regulating the gene expression of the p53 pathway at multiple levels and even establishing a regulatory feedback loop with p53. It is tempting to speculate that the interaction between p53 and lncRNAs has an impact on cell proliferation, cell cycle and cell apoptosis upon DNA damage, resulting in personalized differences in the toxic reactions and efficacy in NPC.

*TUSC7* was significantly induced in cells expressing wild-type p53, serving as a putative tumour suppressor by inhibiting cell growth, which plays a critical role in cancer prognosis, including oesophageal squamous cell carcinoma (ESCC)^24^, colorectal cancer^25^ and osteosarcoma^26^. Low expression of *TUSC7* was dramatically negatively correlated with the pathologic response to CRT and resulted in a poorer prognosis in cancers than the higher expression group^16^. Similarly, we found that rs1829346 and rs36080650 were significantly correlated with the risk of dermatitis under CRT. Patients carrying a homozygous mutation of these two SNPs were less resistant to grade 3–4 dermatitis. Unfortunately, the combination of the two SNPs did not increase risk. Accumulating studies have revealed that lncRNAs impact cellular functions through various mechanisms, most notably as a ‘sponge’ to titrate miRNAs^27,28^. The variation in the rs1829346 sequence may create a new binding site for miR-1304 **(**Supplementary Table [S3](#MOESM1) **)**, inhibiting the expression of *TUSC7*, regulating cell proliferation proteins and ultimately leading to a higher risk of CRT-induced dermatitis. For rs36080650, although there is no direct evidence to illustrate its function, the strong linkage disequilibrium (LD) with rs1829346 suggested that the regulation of gene expression is not mediated by this SNP but by a variant in rs1829346.

*LINC-ROR*, was first discovered in induced pluripotent stem cells (iPSCs); since it was discovered, the number of studies in this area have increased dramatically. *LINC-ROR* is not only a p53 effector but also a p53 regulator; its depletion would lead to the upregulation of genes involved in the p53 response to DNA damage-inducing agents and other stresses responses^17,29^. In our study, *LINC-ROR* rs2027701 showed an obvious trend towards a superior reaction with toxic effects in patients with one or two variant alleles, which may be explained by Zhang^21^, who found that *LINC-ROR* can significantly suppress p53 during DNA damage. Moreover, *LINC-ROR* suppression of the p53 pathway may account for patients who resist chemotherapy, thus playing a critical role in NPC. Our study found that rs2027701 polymorphisms were correlated with worse CRT efficacy at the lymph node.

*pR-lncRNA-1*, in a similar manner as *LINC-ROR*, is induced by p53 and modulates p53 transcriptional activity by forming an autoregulatory feedback loop with p53, enhancing p53 tumour suppressor activity and ultimately modulating the gene expression response to DNA damage^20^. *pR-lncRNA-1* rs73594404 has a strong connection to both neutropaenia and anaemia. Patients with the GA genotype, but not the AA genotype, have increased risk of CRT-induced toxicities. Although rs73594404 was identified in the intron region, it had enhancer activity and had its maximum signal strength in the region of transcription factor binding sites, suggesting that the rs73594404 mutant may influence gene expression by destroying the site or increasing the affinity of the transcription factor, leading to inactivation of tumour suppressor genes^30,31^. This links to poorer prognosis and additional studies will be required to confirm these observations.

*LINC-PINT* is a bona fide p53 transcriptional target that regulates cell proliferation by inducing cell apoptosis^19,32^. As shown in Table [2](#Tab2), the *LINC-PINT* rs1059698 CC genotype was a protective factor in neutropaenia and myelosuppression. Although studies on *LINC-PINT* are limited, using a database, we found that rs1059698 is located in the predicted active promoter flanking regions. The contribution of rare variants to asthma susceptibility is principally due to noncoding variants in sequences flanking the exons^33^. Similarly, Johnson^34^ revealed that genetic variation in noncoding sequences flanking the CYP3A locus was associated with the risk of breast cancer. Therefore, we speculated that the genetic polymorphism of rs1059698 would control promoter activity, inhibit *LINC-PINT* expression, and regulate a multitude of signalling pathways including the p53 network. Furthermore, rs1059698 is also located in the DNase I Hypersensitivity (DHS) cluster. Mutation of rs1059698 was strongly associated with transcription initiation activity, highlighting the role of rs1059698. However, further investigation is needed to delineate the precise mechanism.

*MEG3* is a p53 regulator that is downregulated by MDM2, a well-known negative p53 regulator, thus controlling p53 stability and regulating downstream genes. *MEG3*, a tumour suppressor, has a great capacity for prognosis in many cancers. The *MEG3* rs7158663 AA genotype has significantly increased colorectal cancer risk, as revealed by Cao^35^. Analogously, we found that the *MEG3* rs10132552 CC genotype correlates with a significantly inferior toxic reaction; however, individuals with the CT genotype had a better response to treatments, suggesting that rs10132552 polymorphisms may impact the expression of *MEG3*, thus influencing p53 and subsequently suppressing cell proliferation or promoting cell apoptosis. Furthermore, functional genomic analyses were designed to provide a potential biological basis for the observed associations. Rs10132552 polymorphisms could create miRNA (mir-564, mir-650 and mir-602) binding sites on *MEG3*, disturbing the lncRNA-miRNA interaction, acting as competing endogenous RNAs (ceRNA) and thereby negatively regulating miRNA expression^36,37^. Interestingly, it is clear that mir-650, mir-602 and mir-564 played roles in cancers such as breast cancer and lung cancer^38–40^. Performing an *in silico* analysis, we found that when the T allele was substituted by the C allele, the structure of the transcript was changed **(**Fig. [3](#Fig3) **)**, leading to a minimum free energy (MFE) change from −150.60 kcal/mol to −153.30 kcal/mol, thus elucidating an important role for SNPs on RNA structure and supporting the idea that even SNPs can alter local RNA folding structure^41^. The altered gene expression and secondary structure may eventually result in different sensitivities of individuals to anaemia.

### Figure 3.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5db1/5559481/f54d0ff52386/41598_2017_8890_Fig3_HTML.jpg)

The predicting rs10132552 on MEG3 secondary structure. (a) Base pair probabilities of the local region rs10132552. (b) The optimal secondary structure of global wild-type sequence. Minimum free energy −150.60 kcal/mol. (c) The optimal secondary structure of global mutant sequence. Minimum free energy −153.30 kcal/mol.

However, genetic polymorphisms are not the only signature of neutropaenia in NPC patients, and gemcitabine plus cisplatin was associated with increased risk of grade 3–4 haematological adverse events such as neutropaenia compared with fluorouracil plus cisplatin^42^, which could be supported by our data (40% had grade 3–4 neutropaenia in the GP group vs 7.6% in the PF group). Furthermore, we combined SNPs (rs2027701, rs73594404 and rs1059698) and clinical information to conduct ROC curves; the AUC of neutropaenia was 0.731, indicating good performance for predicting an adverse effect.

Some strengths of this study should be noted. First, this is the first study to explore the impact of SNPs on genes of the lncRNA-p53 regulatory network and the efficacy and toxic reactions in NPC patients. Second, a well-defined cohort of pathological diagnosed cases and strict inclusion criteria were used to avoid possible confounding factors that could hinder analysis. Finally, we obtained all of the essential clinical data from the included individuals. However, several limitations should not be ignored. First, our study is limited as a retrospective study at a single centre, thus selection bias could not be avoided. Second, the sample size seems to be too small for stratification analysis; therefore, statistical power may be limited. Finally, this pathway is complex-the five genes included in this analysis were insufficient and further study is needed.

In conclusion, we found six potential SNPs in five genes in the lncRNA-p53 regulatory network that are significantly associated with the toxicities and efficacy of CRT in 505 patients with NPC, thus providing new biomarkers that can predict therapeutic effect and acute toxic reactions. This study represents a significant step forward toward a better understanding of the importance of lncRNA-p53 in NPC.

## Methods

### Patient selection

This study consisted of 505 newly diagnosed NPC cases from the Affiliated Cancer Hospital of Xiangya School of Medicine, Central South University between 2014 and 2016. Peripheral blood specimens for genetic analysis were collected from each patient at the time of diagnosis. Patients were enrolled if they met the following criteria: (a) pathologically confirmed NPC; (b) Karnofsky score ≥70; (c) received intensity modulated radiation-therapy (IMRT) and concurrent chemoradiotherapy; and (d) patients without recurrence, metastasis and other malignancy. Patient demographics and clinical characteristics are shown in Table [1](#Tab1). This study was performed with the approval of the Independent Ethical Committee of the Institute of Clinical Pharmacology, Central South University (CTXY-140007–2). At recruitment, written informed consent was obtained from all participants involved in this study. All experiments methods were performed in accordance with the relevant guidelines and regulations.

### Efficacy regimen

All patients were treated with IMRT with the median total radiation dose of 71.34 Gy. The induction chemotherapy and concurrent chemotherapy were all performed with platinum-based chemotherapy regimens, including DP, docetaxel with cisplatin/ nedaplatin; FP, fluorouracil with cisplatin/nedaplatin; TP, paclitaxel with cisplatin/nedaplatin; GP, gemcitabine with cisplatin/ nedaplatin; DDP, cisplatin alone; NDP, nedaplatin alone.

### SNP selection and genotyping assays

We selected the SNPs for *TUSC7, LINC-ROR, pR-lncRNA-1, LINC-PINT* and *MEG3* by using databases including ENCODE, lncRNASNP, Hapmap and Ensembl to analyse their potential function. The SNPs were selected according to the following criteria: (1) with a minor allele frequency (MAF) ≥0.05 in a Southern Han Chinese population; (2) located in the promoter, miRNA binding site and other functional region; (3) SNPs have not been study before, not only NPC, but also other carcinomas. Genomic DNA was extracted from lymphocytes using the QIAamp DNA Blood Mini Kit (Qiagen, Valencia, CA) according to the manufacturer’s instructions, and stored at -80 °C until use. The DNA purity and concentration were determined by spectrophotometric measurement of absorbance at 260 nm and 280 nm. The candidate SNPs were genotyped using the Sequenom MassARRAY iPLEX platform (Sequenom, Inc., San Diego, CA, USA). The detection rate of all SNPs was greater than 98%.

### Evaluations of toxic reactions

Acute CRT-induced toxic reactions including dermatitis, mucositis, leukopaenia, myelosuppression, neutropaenia, anaemia and thrombocytopenia were recorded and evaluated according to the Common Terminology Criteria for Adverse Events (CTCAE 3.0). We defined grade 0–2 as mild toxic reactions and grade 3–4 as severe side effects, except for anaemia and thrombocytopenia, which have a lower incidence rate, and we chose 0 as the cutoff target. All acute toxic reactions were observed once a week from the first day to the end of treatment. RECIST (Response Evaluation Criteria in Solid Tumours) was used to evaluate efficacy three months after treatment.

### Statistical analysis

Deviations from Hardy-Weinberg equilibrium were calculated using χ2 analysis. By computing the odds ratio (OR) and the corresponding 95% confidence interval (CI), and continuous variables such as age and BMI were switched to binary variables, multivariate logistic regression was performed to determine the association of each SNP of *TUSC7, LINC-ROR, pR-lncRNA-1, LINC-PINT* and *MEG3* with the CRT efficacy and toxic reactions, with adjustments for age, sex, clinical stage, treatment modality and other clinical factors. In addition, stratification analyses were performed to characterize the associations between SNPs and toxic reactions in some subgroups when the corresponding clinical factors had an impact on toxic effects. We used receiver operating characteristic (ROC) curve analysis to evaluate the prognostic power of SNPs by comparing area under the curve (AUC) for each ROC when the P value < 0.05 was considered statistically significant. All statistical analyses were performed with SPSS19.0 and the R package. We also used lncRNASNP ([http://bioinfo.life.hust.edu.cn/lncRNASNP/](http://bioinfo.life.hust.edu.cn/lncRNASNP/), accessed 30 January 2017) and RNASNP ([http://rth.dk/resources/rnasnp/](http://rth.dk/resources/rnasnp/), accessed 30 January 2017) to predict the folding structure variants of genes due to SNP genotypes and the gain and loss of function of miRNA-lncRNA interactions through SNP polymorphisms, respectively.

### Data Availability

All data generated or analyzed during this study are included in this published article (and its Supplementary Information files). But the raw data used for the analysis and genotyping data for each patient is private.

## Electronic supplementary material

## Acknowledgements

This research was supported by grants from the National Key Research and Development Program (No. 2016YFC0905000), National High Technology Research and Development Program of China, “863” Project (No. 2012AA02A518), National Scientific Foundation of China (No. 81522048, 81573511, 81472802, 81273595) and Innovation Driven Project of Central South University (No. 2016CX024).

## Author Contributions

Youhong Wang and Zhen Guo conceived and designed the study, performed mutation screening and the data analysis. Youhong Wang wrote the manuscript. Yu Zhao, Yi Jin, Liang An and Bin Wu collected and evaluated the clinical date. Zhaoqian Liu, Xiaoping Chen, Xiang Chen, Honghao Zhou, Hui Wang and Wei Zhang led and coordinated the study. All authors read and approved the final manuscript.

### Competing Interests

### Competing Interests

The authors declare that they have no competing interests.

## Footnotes

## Contributor Information

Hui Wang, Email: wanghui710327@163.com.

Wei Zhang, Email: yjsd2003@163.com.

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Data Availability Statement

All data generated or analyzed during this study are included in this published article (and its Supplementary Information files). But the raw data used for the analysis and genotyping data for each patient is private.

### Supplementary Materials

### Data Availability Statement

All data generated or analyzed during this study are included in this published article (and its Supplementary Information files). But the raw data used for the analysis and genotyping data for each patient is private.

## References

1. Bruce JP, Yip K, Bratman SV, Ito E, Liu FF. Nasopharyngeal Cancer: Molecular Landscape. J Clin Oncol. 2015;33:3346–3355. doi: 10.1200/JCO.2015.60.7846.  [DOI](https://doi.org/10.1200/JCO.2015.60.7846) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26351340/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Nasopharyngeal%20Cancer:%20Molecular%20Landscape&author=JP%20Bruce&author=K%20Yip&author=SV%20Bratman&author=E%20Ito&author=FF%20Liu&volume=33&publication_year=2015&pages=3346-3355&pmid=26351340&doi=10.1200/JCO.2015.60.7846&)

2. Lee AW, Ma BB, Ng WT, Chan AT. Management of Nasopharyngeal Carcinoma: Current Practice and Future Perspective. J Clin Oncol. 2015;33:3356–3364. doi: 10.1200/JCO.2015.60.9347.  [DOI](https://doi.org/10.1200/JCO.2015.60.9347) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26351355/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Management%20of%20Nasopharyngeal%20Carcinoma:%20Current%20Practice%20and%20Future%20Perspective&author=AW%20Lee&author=BB%20Ma&author=WT%20Ng&author=AT%20Chan&volume=33&publication_year=2015&pages=3356-3364&pmid=26351355&doi=10.1200/JCO.2015.60.9347&)

3. Chua ML, Wee JT, Hui EP, Chan AT. Nasopharyngeal carcinoma. Lancet. 2016;387:1012–1024. doi: 10.1016/S0140-6736(15)00055-0.  [DOI](https://doi.org/10.1016/S0140-6736(15)00055-0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26321262/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Nasopharyngeal%20carcinoma&author=ML%20Chua&author=JT%20Wee&author=EP%20Hui&author=AT%20Chan&volume=387&publication_year=2016&pages=1012-1024&pmid=26321262&doi=10.1016/S0140-6736(15)00055-0&)

4. Sun X, et al. Long-term outcomes of intensity-modulated radiotherapy for 868 patients with nasopharyngeal carcinoma: an analysis of survival and treatment toxicities. Radiother Oncol. 2014;110:398–403. doi: 10.1016/j.radonc.2013.10.020.  [DOI](https://doi.org/10.1016/j.radonc.2013.10.020) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24231245/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Radiother%20Oncol&title=Long-term%20outcomes%20of%20intensity-modulated%20radiotherapy%20for%20868%20patients%20with%20nasopharyngeal%20carcinoma:%20an%20analysis%20of%20survival%20and%20treatment%20toxicities&author=X%20Sun&volume=110&publication_year=2014&pages=398-403&pmid=24231245&doi=10.1016/j.radonc.2013.10.020&)

5. Zhang AM, et al. Increased treatment-related mortality with additional cisplatin-based chemotherapy in patients with nasopharyngeal carcinoma treated with standard radiotherapy. Radiother Oncol. 2012;104:279–285. doi: 10.1016/j.radonc.2012.08.022.  [DOI](https://doi.org/10.1016/j.radonc.2012.08.022) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22981609/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Radiother%20Oncol&title=Increased%20treatment-related%20mortality%20with%20additional%20cisplatin-based%20chemotherapy%20in%20patients%20with%20nasopharyngeal%20carcinoma%20treated%20with%20standard%20radiotherapy&author=AM%20Zhang&volume=104&publication_year=2012&pages=279-285&pmid=22981609&doi=10.1016/j.radonc.2012.08.022&)

6. Guo Z, Shu Y, Zhou H, Zhang W, Wang H. Radiogenomics helps to achieve personalized therapy by evaluating patient responses to radiation treatment. Carcinogenesis. 2015;36:307–317. doi: 10.1093/carcin/bgv007.  [DOI](https://doi.org/10.1093/carcin/bgv007) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25604391/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Carcinogenesis&title=Radiogenomics%20helps%20to%20achieve%20personalized%20therapy%20by%20evaluating%20patient%20responses%20to%20radiation%20treatment&author=Z%20Guo&author=Y%20Shu&author=H%20Zhou&author=W%20Zhang&author=H%20Wang&volume=36&publication_year=2015&pages=307-317&pmid=25604391&doi=10.1093/carcin/bgv007&)

7. Feng XP, et al. Identification of biomarkers for predicting nasopharyngeal carcinoma response to radiotherapy by proteomics. Cancer Res. 2010;70:3450–3462. doi: 10.1158/0008-5472.CAN-09-4099.  [DOI](https://doi.org/10.1158/0008-5472.CAN-09-4099) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20406978/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=Identification%20of%20biomarkers%20for%20predicting%20nasopharyngeal%20carcinoma%20response%20to%20radiotherapy%20by%20proteomics&author=XP%20Feng&volume=70&publication_year=2010&pages=3450-3462&pmid=20406978&doi=10.1158/0008-5472.CAN-09-4099&)

8. Lundberg M, Saarilahti K, Mäkitie AA, Mattila PS. TGFbeta1 genetic polymorphism is associated with survival in head and neck squamous cell carcinoma independent of the severity of chemoradiotherapy induced mucositis. Oral Oncol. 2010;46:369–372. doi: 10.1016/j.oraloncology.2010.02.012.  [DOI](https://doi.org/10.1016/j.oraloncology.2010.02.012) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20308003/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oral%20Oncol&title=TGFbeta1%20genetic%20polymorphism%20is%20associated%20with%20survival%20in%20head%20and%20neck%20squamous%20cell%20carcinoma%20independent%20of%20the%20severity%20of%20chemoradiotherapy%20induced%20mucositis&author=M%20Lundberg&author=K%20Saarilahti&author=AA%20M%C3%A4kitie&author=PS%20Mattila&volume=46&publication_year=2010&pages=369-372&pmid=20308003&doi=10.1016/j.oraloncology.2010.02.012&)

9. Jia M, et al. Genetic variants of GADD45A, GADD45B and MAPK14 predict platinum-based chemotherapy-induced toxicities in Chinese patients with non-small cell lung cancer. Oncotarget. 2016;7:25291–25303. doi: 10.18632/oncotarget.8052.  [DOI](https://doi.org/10.18632/oncotarget.8052) | [PMC free article](/articles/PMC5041904/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26993769/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncotarget&title=Genetic%20variants%20of%20GADD45A,%20GADD45B%20and%20MAPK14%20predict%20platinum-based%20chemotherapy-induced%20toxicities%20in%20Chinese%20patients%20with%20non-small%20cell%20lung%20cancer&author=M%20Jia&volume=7&publication_year=2016&pages=25291-25303&pmid=26993769&doi=10.18632/oncotarget.8052&)

10. Wang C, et al. The study of the relation of DNA repair pathway genes SNPs and the sensitivity to radiotherapy and chemotherapy of NSCLC. Sci Rep. 2016;6:26526. doi: 10.1038/srep26526.  [DOI](https://doi.org/10.1038/srep26526) | [PMC free article](/articles/PMC4887885/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27246533/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Sci%20Rep&title=The%20study%20of%20the%20relation%20of%20DNA%20repair%20pathway%20genes%20SNPs%20and%20the%20sensitivity%20to%20radiotherapy%20and%20chemotherapy%20of%20NSCLC&author=C%20Wang&volume=6&publication_year=2016&pages=26526&pmid=27246533&doi=10.1038/srep26526&)

11. Beermann J, Piccoli MT, Viereck J, Thum T. Non-coding RNAs in Development and Disease: Background, Mechanisms, and Therapeutic Approaches. Physiol Rev. 2016;96:1297–1325. doi: 10.1152/physrev.00041.2015.  [DOI](https://doi.org/10.1152/physrev.00041.2015) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27535639/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Physiol%20Rev&title=Non-coding%20RNAs%20in%20Development%20and%20Disease:%20Background,%20Mechanisms,%20and%20Therapeutic%20Approaches&author=J%20Beermann&author=MT%20Piccoli&author=J%20Viereck&author=T%20Thum&volume=96&publication_year=2016&pages=1297-1325&pmid=27535639&doi=10.1152/physrev.00041.2015&)

12. Bo H, et al. Upregulated long non-coding RNA AFAP1-AS1 expression is associated with progression and poor prognosis of nasopharyngeal carcinoma. Oncotarget. 2015;6:20404–20418. doi: 10.18632/oncotarget.4057.  [DOI](https://doi.org/10.18632/oncotarget.4057) | [PMC free article](/articles/PMC4653014/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26246469/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncotarget&title=Upregulated%20long%20non-coding%20RNA%20AFAP1-AS1%20expression%20is%20associated%20with%20progression%20and%20poor%20prognosis%20of%20nasopharyngeal%20carcinoma&author=H%20Bo&volume=6&publication_year=2015&pages=20404-20418&pmid=26246469&doi=10.18632/oncotarget.4057&)

13. Quinn JJ, Chang HY. Unique features of long non-coding RNA biogenesis and function. Nat Rev Genet. 2016;17:47–62. doi: 10.1038/nrg.2015.10.  [DOI](https://doi.org/10.1038/nrg.2015.10) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26666209/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Genet&title=Unique%20features%20of%20long%20non-coding%20RNA%20biogenesis%20and%20function&author=JJ%20Quinn&author=HY%20Chang&volume=17&publication_year=2016&pages=47-62&pmid=26666209&doi=10.1038/nrg.2015.10&)

14. Schmitt AM, Chang HY. Long Noncoding RNAs in Cancer Pathways. Cancer Cell. 2016;29:452–463. doi: 10.1016/j.ccell.2016.03.010.  [DOI](https://doi.org/10.1016/j.ccell.2016.03.010) | [PMC free article](/articles/PMC4831138/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27070700/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Cell&title=Long%20Noncoding%20RNAs%20in%20Cancer%20Pathways&author=AM%20Schmitt&author=HY%20Chang&volume=29&publication_year=2016&pages=452-463&pmid=27070700&doi=10.1016/j.ccell.2016.03.010&)

15. Chou J, et al. Nasopharyngeal carcinoma–review of the molecular mechanisms of tumorigenesis. Head Neck. 2008;30:946–963. doi: 10.1002/hed.20833.  [DOI](https://doi.org/10.1002/hed.20833) | [PMC free article](/articles/PMC3046044/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18446839/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Head%20Neck&title=Nasopharyngeal%20carcinoma%E2%80%93review%20of%20the%20molecular%20mechanisms%20of%20tumorigenesis&author=J%20Chou&volume=30&publication_year=2008&pages=946-963&pmid=18446839&doi=10.1002/hed.20833&)

16. Huarte M, et al. A large intergenic noncoding RNA induced by p53 mediates global gene repression in the p53 response. Cell. 2010;142:409–419. doi: 10.1016/j.cell.2010.06.040.  [DOI](https://doi.org/10.1016/j.cell.2010.06.040) | [PMC free article](/articles/PMC2956184/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20673990/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell&title=A%20large%20intergenic%20noncoding%20RNA%20induced%20by%20p53%20mediates%20global%20gene%20repression%20in%20the%20p53%20response&author=M%20Huarte&volume=142&publication_year=2010&pages=409-419&pmid=20673990&doi=10.1016/j.cell.2010.06.040&)

17. Zhang A, Xu M, Mo YY. Role of the lncRNA-p53 regulatory network in cancer. J Mol Cell Biol. 2014;6:181–191. doi: 10.1093/jmcb/mju013.  [DOI](https://doi.org/10.1093/jmcb/mju013) | [PMC free article](/articles/PMC4034727/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24721780/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Mol%20Cell%20Biol&title=Role%20of%20the%20lncRNA-p53%20regulatory%20network%20in%20cancer&author=A%20Zhang&author=M%20Xu&author=YY%20Mo&volume=6&publication_year=2014&pages=181-191&pmid=24721780&doi=10.1093/jmcb/mju013&)

18. Liu Q, et al. LncRNA loc285194 is a p53-regulated tumor suppressor. Nucleic Acids Res. 2013;41:4976–4987. doi: 10.1093/nar/gkt182.  [DOI](https://doi.org/10.1093/nar/gkt182) | [PMC free article](/articles/PMC3643595/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23558749/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=LncRNA%20loc285194%20is%20a%20p53-regulated%20tumor%20suppressor&author=Q%20Liu&volume=41&publication_year=2013&pages=4976-4987&pmid=23558749&doi=10.1093/nar/gkt182&)

19. Marín-Béjar O, et al. Pint lincRNA connects the p53 pathway with epigenetic silencing by the Polycomb repressive complex 2. Genome Biol. 2013;14:R104. doi: 10.1186/gb-2013-14-9-r104.  [DOI](https://doi.org/10.1186/gb-2013-14-9-r104) | [PMC free article](/articles/PMC4053822/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24070194/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Biol&title=Pint%20lincRNA%20connects%20the%20p53%20pathway%20with%20epigenetic%20silencing%20by%20the%20Polycomb%20repressive%20complex%202&author=O%20Mar%C3%ADn-B%C3%A9jar&volume=14&publication_year=2013&pages=R104&pmid=24070194&doi=10.1186/gb-2013-14-9-r104&)

20. Sánchez Y, et al. Genome-wide analysis of the human p53 transcriptional network unveils a lncRNA tumour suppressor signature. Nat Commun. 2014;5:812. doi: 10.1038/ncomms6812.  [DOI](https://doi.org/10.1038/ncomms6812) | [PMC free article](/articles/PMC4284803/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25524025/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Commun&title=Genome-wide%20analysis%20of%20the%20human%20p53%20transcriptional%20network%20unveils%20a%20lncRNA%20tumour%20suppressor%20signature&author=Y%20S%C3%A1nchez&volume=5&publication_year=2014&pages=812&pmid=25524025&doi=10.1038/ncomms6812&)

21. Zhang A, et al. The human long non-coding RNA-RoR is a p53 repressor in response to DNA damage. Cell Res. 2013;23:340–350. doi: 10.1038/cr.2012.164.  [DOI](https://doi.org/10.1038/cr.2012.164) | [PMC free article](/articles/PMC3587705/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23208419/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell%20Res&title=The%20human%20long%20non-coding%20RNA-RoR%20is%20a%20p53%20repressor%20in%20response%20to%20DNA%20damage&author=A%20Zhang&volume=23&publication_year=2013&pages=340-350&pmid=23208419&doi=10.1038/cr.2012.164&)

22. Li Y, Lu H. Noncoding RNAs: ‘our turn’ to join the p53 network. J Mol Cell Biol. 2014;6:179–180. doi: 10.1093/jmcb/mju022.  [DOI](https://doi.org/10.1093/jmcb/mju022) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24868101/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Mol%20Cell%20Biol&title=Noncoding%20RNAs:%20%E2%80%98our%20turn%E2%80%99%20to%20join%20the%20p53%20network&author=Y%20Li&author=H%20Lu&volume=6&publication_year=2014&pages=179-180&pmid=24868101&doi=10.1093/jmcb/mju022&)

23. Huarte M. p53 partners with RNA in the DNA damage response. Nat Genet. 2016;48:1298–1299. doi: 10.1038/ng.3702.  [DOI](https://doi.org/10.1038/ng.3702) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27787509/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=p53%20partners%20with%20RNA%20in%20the%20DNA%20damage%20response&author=M%20Huarte&volume=48&publication_year=2016&pages=1298-1299&pmid=27787509&doi=10.1038/ng.3702&)

24. Tong YS, et al. Association of decreased expression of long non-coding RNA LOC285194 with chemoradiotherapy resistance and poor prognosis in esophageal squamous cell carcinoma. J Transl Med. 2014;12:233. doi: 10.1186/s12967-014-0233-y.  [DOI](https://doi.org/10.1186/s12967-014-0233-y) | [PMC free article](/articles/PMC4155091/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25169763/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Transl%20Med&title=Association%20of%20decreased%20expression%20of%20long%20non-coding%20RNA%20LOC285194%20with%20chemoradiotherapy%20resistance%20and%20poor%20prognosis%20in%20esophageal%20squamous%20cell%20carcinoma&author=YS%20Tong&volume=12&publication_year=2014&pages=233&pmid=25169763&doi=10.1186/s12967-014-0233-y&)

25. Qi P, et al. Low expression of LOC285194 is associated with poor prognosis in colorectal cancer. J Transl Med. 2013;11:122. doi: 10.1186/1479-5876-11-122.  [DOI](https://doi.org/10.1186/1479-5876-11-122) | [PMC free article](/articles/PMC3665590/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23680400/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Transl%20Med&title=Low%20expression%20of%20LOC285194%20is%20associated%20with%20poor%20prognosis%20in%20colorectal%20cancer&author=P%20Qi&volume=11&publication_year=2013&pages=122&pmid=23680400&doi=10.1186/1479-5876-11-122&)

26. Pasic I, et al. Recurrent focal copy-number changes and loss of heterozygosity implicate two noncoding RNAs and one tumor suppressor gene at chromosome 3q13.31 in osteosarcoma. Cancer Res. 2010;70:160–171. doi: 10.1158/0008-5472.CAN-09-1902.  [DOI](https://doi.org/10.1158/0008-5472.CAN-09-1902) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20048075/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=Recurrent%20focal%20copy-number%20changes%20and%20loss%20of%20heterozygosity%20implicate%20two%20noncoding%20RNAs%20and%20one%20tumor%20suppressor%20gene%20at%20chromosome%203q13.31%20in%20osteosarcoma&author=I%20Pasic&volume=70&publication_year=2010&pages=160-171&pmid=20048075&doi=10.1158/0008-5472.CAN-09-1902&)

27. Hua WF, et al. RBM24 suppresses cancer progression by upregulating miR-25 to target MALAT1 in nasopharyngeal carcinoma. Cell Death Dis. 2016;7:e2352. doi: 10.1038/cddis.2016.252.  [DOI](https://doi.org/10.1038/cddis.2016.252) | [PMC free article](/articles/PMC5059856/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27584791/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell%20Death%20Dis&title=RBM24%20suppresses%20cancer%20progression%20by%20upregulating%20miR-25%20to%20target%20MALAT1%20in%20nasopharyngeal%20carcinoma&author=WF%20Hua&volume=7&publication_year=2016&pages=e2352&pmid=27584791&doi=10.1038/cddis.2016.252&)

28. Zhou X, et al. Linc-RNA-RoR acts as a “sponge” against mediation of the differentiation of endometrial cancer stem cells by microRNA-145. Gynecol Oncol. 2014;133:333–339. doi: 10.1016/j.ygyno.2014.02.033.  [DOI](https://doi.org/10.1016/j.ygyno.2014.02.033) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24589415/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gynecol%20Oncol&title=Linc-RNA-RoR%20acts%20as%20a%20%E2%80%9Csponge%E2%80%9D%20against%20mediation%20of%20the%20differentiation%20of%20endometrial%20cancer%20stem%20cells%20by%20microRNA-145&author=X%20Zhou&volume=133&publication_year=2014&pages=333-339&pmid=24589415&doi=10.1016/j.ygyno.2014.02.033&)

29. Pan Y, et al. The Emerging Roles of Long Noncoding RNA ROR (lincRNA-ROR) and its Possible Mechanisms in Human Cancers. Cell Physiol Biochem. 2016;40:219–229. doi: 10.1159/000452539.  [DOI](https://doi.org/10.1159/000452539) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27855392/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell%20Physiol%20Biochem&title=The%20Emerging%20Roles%20of%20Long%20Noncoding%20RNA%20ROR%20(lincRNA-ROR)%20and%20its%20Possible%20Mechanisms%20in%20Human%20Cancers&author=Y%20Pan&volume=40&publication_year=2016&pages=219-229&pmid=27855392&doi=10.1159/000452539&)

30. Khurana E, et al. Role of non-coding sequence variants in cancer. Nat Rev Genet. 2016;17:93–108. doi: 10.1038/nrg.2015.17.  [DOI](https://doi.org/10.1038/nrg.2015.17) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26781813/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Genet&title=Role%20of%20non-coding%20sequence%20variants%20in%20cancer&author=E%20Khurana&volume=17&publication_year=2016&pages=93-108&pmid=26781813&doi=10.1038/nrg.2015.17&)

31. Melton C, Reuter JA, Spacek DV, Snyder M. Recurrent somatic mutations in regulatory regions of human cancer genomes. Nat Genet. 2015;47:710–716. doi: 10.1038/ng.3332.  [DOI](https://doi.org/10.1038/ng.3332) | [PMC free article](/articles/PMC4485503/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26053494/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Recurrent%20somatic%20mutations%20in%20regulatory%20regions%20of%20human%20cancer%20genomes&author=C%20Melton&author=JA%20Reuter&author=DV%20Spacek&author=M%20Snyder&volume=47&publication_year=2015&pages=710-716&pmid=26053494&doi=10.1038/ng.3332&)

32. Shiekhattar R. PINTing for p53. Genome Biol. 2013;14:132. doi: 10.1186/gb-2013-14-12-r132.  [DOI](https://doi.org/10.1186/gb-2013-14-12-r132) | [PMC free article](/articles/PMC4053842/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24079657/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Biol&title=PINTing%20for%20p53&author=R%20Shiekhattar&volume=14&publication_year=2013&pages=132&pmid=24079657&doi=10.1186/gb-2013-14-12-r132&)

33. Torgerson DG, et al. Resequencing candidate genes implicates rare variants in asthma susceptibility. Am J Hum Genet. 2012;90:273–281. doi: 10.1016/j.ajhg.2012.01.008.  [DOI](https://doi.org/10.1016/j.ajhg.2012.01.008) | [PMC free article](/articles/PMC3276672/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22325360/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=Resequencing%20candidate%20genes%20implicates%20rare%20variants%20in%20asthma%20susceptibility&author=DG%20Torgerson&volume=90&publication_year=2012&pages=273-281&pmid=22325360&doi=10.1016/j.ajhg.2012.01.008&)

34. Johnson N, et al. CYP3A variation, premenopausal estrone levels, and breast cancer risk. J Natl Cancer Inst. 2012;104:657–669. doi: 10.1093/jnci/djs156.  [DOI](https://doi.org/10.1093/jnci/djs156) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22472546/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Natl%20Cancer%20Inst&title=CYP3A%20variation,%20premenopausal%20estrone%20levels,%20and%20breast%20cancer%20risk&author=N%20Johnson&volume=104&publication_year=2012&pages=657-669&pmid=22472546&doi=10.1093/jnci/djs156&)

35. Cao X, et al. Associations between polymorphisms of long non-coding RNA MEG3 and risk of colorectal cancer in Chinese. Oncotarget. 2016;7:19054–19059. doi: 10.18632/oncotarget.7764.  [DOI](https://doi.org/10.18632/oncotarget.7764) | [PMC free article](/articles/PMC4951351/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26934323/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncotarget&title=Associations%20between%20polymorphisms%20of%20long%20non-coding%20RNA%20MEG3%20and%20risk%20of%20colorectal%20cancer%20in%20Chinese&author=X%20Cao&volume=7&publication_year=2016&pages=19054-19059&pmid=26934323&doi=10.18632/oncotarget.7764&)

36. Gao S, et al. ROR functions as a ceRNA to regulate Nanog expression by sponging miR-145 and predicts poor prognosis in pancreatic cancer. Oncotarget. 2016;7:1608–1618. doi: 10.18632/oncotarget.6450.  [DOI](https://doi.org/10.18632/oncotarget.6450) | [PMC free article](/articles/PMC4811484/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26636540/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncotarget&title=ROR%20functions%20as%20a%20ceRNA%20to%20regulate%20Nanog%20expression%20by%20sponging%20miR-145%20and%20predicts%20poor%20prognosis%20in%20pancreatic%20cancer&author=S%20Gao&volume=7&publication_year=2016&pages=1608-1618&pmid=26636540&doi=10.18632/oncotarget.6450&)

37. Liu D, et al. The gain and loss of long noncoding RNA associated-competing endogenous RNAs in prostate cancer. Oncotarget. 2016;7:57228–57238. doi: 10.18632/oncotarget.11128.  [DOI](https://doi.org/10.18632/oncotarget.11128) | [PMC free article](/articles/PMC5302985/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27528026/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncotarget&title=The%20gain%20and%20loss%20of%20long%20noncoding%20RNA%20associated-competing%20endogenous%20RNAs%20in%20prostate%20cancer&author=D%20Liu&volume=7&publication_year=2016&pages=57228-57238&pmid=27528026&doi=10.18632/oncotarget.11128&)

38. Jiang C, et al. MicroRNA-564 is downregulated in glioblastoma and inhibited proliferation and invasion of glioblastoma cells by targeting TGF-beta1. Oncotarget. 2016;7:56200–56208. doi: 10.18632/oncotarget.8987.  [DOI](https://doi.org/10.18632/oncotarget.8987) | [PMC free article](/articles/PMC5302907/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27621042/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncotarget&title=MicroRNA-564%20is%20downregulated%20in%20glioblastoma%20and%20inhibited%20proliferation%20and%20invasion%20of%20glioblastoma%20cells%20by%20targeting%20TGF-beta1&author=C%20Jiang&volume=7&publication_year=2016&pages=56200-56208&pmid=27621042&doi=10.18632/oncotarget.8987&)

39. Mraz M, et al. MicroRNA-650 expression is influenced by immunoglobulin gene rearrangement and affects the biology of chronic lymphocytic leukemia. Blood. 2012;119:2110–2113. doi: 10.1182/blood-2011-11-394874.  [DOI](https://doi.org/10.1182/blood-2011-11-394874) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22234685/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=MicroRNA-650%20expression%20is%20influenced%20by%20immunoglobulin%20gene%20rearrangement%20and%20affects%20the%20biology%20of%20chronic%20lymphocytic%20leukemia&author=M%20Mraz&volume=119&publication_year=2012&pages=2110-2113&pmid=22234685&doi=10.1182/blood-2011-11-394874&)

40. Yang L, et al. MicroRNA-602 regulating tumor suppressive gene RASSF1A is overexpressed in hepatitis B virus-infected liver and hepatocellular carcinoma. Cancer Biol Ther. 2010;9:803–808. doi: 10.4161/cbt.9.10.11440.  [DOI](https://doi.org/10.4161/cbt.9.10.11440) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20364114/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Biol%20Ther&title=MicroRNA-602%20regulating%20tumor%20suppressive%20gene%20RASSF1A%20is%20overexpressed%20in%20hepatitis%20B%20virus-infected%20liver%20and%20hepatocellular%20carcinoma&author=L%20Yang&volume=9&publication_year=2010&pages=803-808&pmid=20364114&doi=10.4161/cbt.9.10.11440&)

41. Li S, et al. Association of genetic variants in lncRNA H19 with risk of colorectal cancer in a Chinese population. Oncotarget. 2016;7:25470–25477. doi: 10.18632/oncotarget.8330.  [DOI](https://doi.org/10.18632/oncotarget.8330) | [PMC free article](/articles/PMC5041918/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27027436/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncotarget&title=Association%20of%20genetic%20variants%20in%20lncRNA%20H19%20with%20risk%20of%20colorectal%20cancer%20in%20a%20Chinese%20population&author=S%20Li&volume=7&publication_year=2016&pages=25470-25477&pmid=27027436&doi=10.18632/oncotarget.8330&)

42. Zhang L, et al. Gemcitabine plus cisplatin versus fluorouracil plus cisplatin in recurrent or metastatic nasopharyngeal carcinoma: a multicentre, randomised, open-label, phase 3 trial. Lancet. 2016;388:1883–1892. doi: 10.1016/S0140-6736(16)31388-5.  [DOI](https://doi.org/10.1016/S0140-6736(16)31388-5) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27567279/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Gemcitabine%20plus%20cisplatin%20versus%20fluorouracil%20plus%20cisplatin%20in%20recurrent%20or%20metastatic%20nasopharyngeal%20carcinoma:%20a%20multicentre,%20randomised,%20open-label,%20phase%203%20trial&author=L%20Zhang&volume=388&publication_year=2016&pages=1883-1892&pmid=27567279&doi=10.1016/S0140-6736(16)31388-5&)
