# Genome-wide study of resistant hypertension identified from electronic health records

## Metadata
**Authors:** Logan Dumitrescu, Marylyn D Ritchie, Joshua C Denny, Nihal M El Rouby, Caitrin W McDonough, Yuki Bradford, Andrea H Ramirez, Suzette J Bielinski, Melissa A Basford, High Seng Chai, Peggy Peissig, David Carrell, Jyotishman Pathak, Luke V Rasmussen, Xiaoming Wang, Jennifer A Pacheco, Abel N Kho, M Geoffrey Hayes, Martha Matsumoto, Maureen E Smith, Rongling Li, Rhonda M Cooper-DeHoff, Iftikhar J Kullo, Christopher G Chute, Rex L Chisholm, Gail P Jarvik, Eric B Larson, David Carey, Catherine A McCarty, Marc S Williams, Dan M Roden, Erwin Bottinger, Julie A Johnson, Mariza de Andrade, Dana C Crawford
**Journal:** PLoS ONE
**Date:** 2017 Feb 21
**DOI:** [10.1371/journal.pone.0171745](https://doi.org/10.1371/journal.pone.0171745)
**PMID:** 28222112
**PMCID:** PMC5319785
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5319785/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC5319785/pdf/pone.0171745.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC5319785/pdf/pone.0171745.pdf)

## Abstract

Resistant hypertension is defined as high blood pressure that remains above treatment goals in spite of the concurrent use of three antihypertensive agents from different classes. Despite the important health consequences of resistant hypertension, few studies of resistant hypertension have been conducted. To perform a genome-wide association study for resistant hypertension, we defined and identified cases of resistant hypertension and hypertensives with treated, controlled hypertension among >47,500 adults residing in the US linked to electronic health records (EHRs) and genotyped as part of the electronic MEdical Records & GEnomics (eMERGE) Network. Electronic selection logic using billing codes, laboratory values, text queries, and medication records was used to identify resistant hypertension cases and controls at each site, and a total of 3,006 cases of resistant hypertension and 876 controlled hypertensives were identified among eMERGE Phase I and II sites. After imputation and quality control, a total of 2,530,150 SNPs were tested for an association among 2,830 multi-ethnic cases of resistant hypertension and 876 controlled hypertensives. No test of association was genome-wide significant in the full dataset or in the dataset limited to European American cases (n = 1,719) and controls (n = 708). The most significant finding was CLNK rs13144136 at p = 1.00x10-6 (odds ratio = 0.68; 95% CI = 0.58–0.80) in the full dataset with similar results in the European American only dataset. We also examined whether SNPs known to influence blood pressure or hypertension also influenced resistant hypertension. None was significant after correction for multiple testing. These data highlight both the difficulties and the potential utility of EHR-linked genomic data to study clinically-relevant traits such as resistant hypertension.

## Introduction

Hypertension, or high blood pressure, affects nearly a third of the US adult population and is a major risk factor for coronary heart disease, stroke, and kidney disease [[1](#pone.0171745.ref001), [2](#pone.0171745.ref002)]. Pharmacological treatment of hypertension has been shown to lower blood pressure and substantially reduce the risk of coronary heart disease and stroke [[3](#pone.0171745.ref003), [4](#pone.0171745.ref004)]. While the number of patients tested and receiving treatment for hypertension has increased substantially over the last 40 years, a majority (~50%) of hypertensive Americans’ blood pressure remains uncontrolled [[1](#pone.0171745.ref001), [5](#pone.0171745.ref005), [6](#pone.0171745.ref006)]. Resistant hypertension is defined as blood pressure that remains ≥140/90 mm Hg despite use of three concurrent antihypertensive agents from different classes, one of which includes a diuretic [[7](#pone.0171745.ref007)]. Other definitions also include individuals who require four or more medications to achieve a blood pressure <140/90 mm Hg. The prevalence of resistant hypertension is progressively increasing and is currently estimated to affect 8–12% of adults with hypertension [[8](#pone.0171745.ref008)].

Along with age and obesity, well-established independent risk factors of resistant hypertension, several other lifestyle and biological risk factors are believed to contribute to resistant hypertension, including excess alcohol use, increased dietary sodium intake, and use of several classes of medications (such as non-steroidal anti-inflammatory drugs, corticosteroids, and calcineurin inhibitors) [[9](#pone.0171745.ref009)]. Genetic factors may also play a role [[10](#pone.0171745.ref010)]. Numerous genetic variants have been identified in studies of the genetic architecture of hypertension and variation in blood pressure [[11](#pone.0171745.ref011), [12](#pone.0171745.ref012)]. Additionally, hypertension pharmacogenomics research has implicated genes important in the inter-individual variability of response to specific antihypertensive drugs [[13](#pone.0171745.ref013)–[15](#pone.0171745.ref015)]. However, due to the small sample sizes, genetic studies of resistant hypertension have been limited in power and in scope (i.e., candidate gene studies) [[16](#pone.0171745.ref016)–[21](#pone.0171745.ref021)].

To help overcome the obstacle of limited sample size, we used data available from seven electronic health record (EHR) systems to identify both resistant hypertension cases and hypertensive individuals who responded well to a single antihypertensive medication (controlled hypertensives) in the electronic MEdical Records & GEnomics (eMERGE) Network [[22](#pone.0171745.ref022), [23](#pone.0171745.ref023)]. We then performed a genome-wide association study (GWAS) to identify common variants associated with resistant hypertension in 2,830 cases and 876 controls. We also assessed whether SNPs previously associated with blood pressure or hypertension in the literature were associated with resistant hypertension, a complex and clinically-relevant phenotype.

## Materials and methods

### eMERGE Network

The eMERGE Network and its studies are approved by the Institutional Review Board at each study site, which include Geisinger Health System, Group Health, Marshfield Clinic, Mayo Clinic, Mount Sinai School of Medicine, Northwestern University, University of Washington, Vanderbilt University [[22](#pone.0171745.ref022), [23](#pone.0171745.ref023)]. Participants at all study sites except for Vanderbilt University provided written, informed consent. Vanderbilt University’s biobank BioVU followed an opt-out model where DNA was extracted from discarded blood extracted for clinical purposes. The DNA linked to de-identified EHRs is considered “non-human subjects” as no personal identifying information is available to the investigators (The Code of Federal Regulations, 45 CFR 46.102 (f)) [[24](#pone.0171745.ref024), [25](#pone.0171745.ref025)].

The eMERGE Network, initially funded in 2007 by the National Human Genome Research Institute (NHGRI), consisted of five EHR-linked biorepositories and a coordinating center at the initiation of this study [[22](#pone.0171745.ref022)]. In brief, the five biorepositories included in eMERGE Phase I were: Group Health/University of Washington (GH/UW), Marshfield Clinic (MFC), Mayo Clinic (MC), Northwestern University (NU), and Vanderbilt University (VU) ([Fig 1](#pone.0171745.g001)). Among the five original biorepositories, two used locally-developed EHR systems (MFC and VU), and three used commercial systems with local modifications (NU, MC, and GHC) [[26](#pone.0171745.ref026)]. A total of 16,029 European Americans and 2,634 African Americans were selected from eMERGE Phase I study sites and genotyped on the Illumina 660-Quad or 1M as previously described [[27](#pone.0171745.ref027)] and briefly described below.

### Fig 1. Resistant hypertension genome-wide association study in the eMERGE Network.

![Fig 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dae3/5319785/b9b8553bb906/pone.0171745.g001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5319785_pone.0171745.g001.jpg)

The eMERGE Network conducted a genome-wide association study for resistant hypertension among adults drawn from the two funding phases of the network. The eMERGE Network I study sites that contributed data are denoted in blue and include Group Health/University of Washington in Seattle, WA; Marshfield Clinic in Marshfield, WI; Mayo Clinic in Rochester, MN; Northwestern University in Chicago, IL; and Vanderbilt University in Nashville, TN. The eMERGE Network II study sites that contributed data are denoted in red and include Geisinger Health System in Danville, PA and Mount Sinai School of Medicine in Manhattan, NY. Denoted in gray are eMERGE Phase II pediatric study sites not included in the present study (Boston Children’s Hospital in Boston, MA; Children’s Hospital of Philadelphia in Philadelphia, PA; and Cincinnati Children’s Hospital in Cincinnati, OH).

In the second phase of eMERGE, the number of study sites expanded to include Children’s Hospital of Philadelphia, Boston Children’s Hospital, Cincinnati Children’s Hospital Medical Center, Geisinger Health System (GHS), and Mount Sinai School of Medicine (MSSM) [[23](#pone.0171745.ref023)]. In this study of adults with resistant hypertension, data from the eMERGE Phase II study sites GHS and MSSM were included ([Fig 1](#pone.0171745.g001)), both of which used a commercial EHR system [[23](#pone.0171745.ref023)].

### Selection of resistant hypertension cases and controls

There are multiple definitions of resistant hypertension in the literature [[28](#pone.0171745.ref028)], including multiple subgroups and classifications of “uncontrolled” and “controlled.” International guidelines such as The National Institute for Health and Care Excellence define uncontrolled resistant hypertension as an individual with systolic or diastolic blood pressure measures >140 or >90 mm Hg, respectively, after use of three classes of anti-hypertensive medications concurrently [[29](#pone.0171745.ref029)]. Controlled resistant hypertension is defined by the American Heart Association Scientific Statement [[7](#pone.0171745.ref007)] as the concurrent use of at least four antihypertensive medication classes.

Defining any resistant hypertension group within the EHR requires a patient’s hypertensive status pre- and post- antihypertensive medication use (based on issued prescriptions) and the medication classes of the anti-hypertensives prescribed. To capture these data, two algorithms were deployed in the eMERGE Network to identify resistant hypertension cases. For the first case algorithm (“controlled case”), each case required the concurrent use of at least four antihypertensive medication classes. Medication classes considered for these definitions included angiotensin converting enzyme inhibitors or angiotensin receptor blockers, beta blockers, non-dihydropyridine calcium channel blockers, dihydropyridine calcium channel blockers, hydralazine, minoxidil, central alpha agonists, direct renin antagonists, aldosterone antagonists, alpha antagonists, and diuretics (thiazides, K-sparing, and loop diuretics) ([S1 Table](#pone.0171745.s009)). Direct alpha antagonists (e.g., phentolamine and phenoxybenzamine) were excluded from the antihypertensive medication class as they are typically given to counteract pathologic adrenergic states (e.g., pheochromocytoma), which were excluded from case and control groups. For eMERGE study sites using natural language processing as part of the data extraction process for medications, the algorithm required a dose, strength, route, or frequency present with the medication name to ensure the medication mentioned represents a prescribed medication.

For the second case algorithm (“uncontrolled case”), each case required the concurrent use of at least three antihypertensive medication classes and a systolic blood pressure >140 mm Hg or diastolic blood pressure >90 mm Hg for at least one month after meeting the medication criteria. For both case definitions, patients were excluded if they had systolic heart failure (defined as an ejection fraction ≤35%) or chronic kidney disease. Chronic kidney disease was defined as an estimated glomerular filtration rate ≤30 ml/min as calculated using the Modification of Diet in Renal Disease formula [[30](#pone.0171745.ref030)]. Other exclusion criteria for cases and controls are given in [S2 Table](#pone.0171745.s010). Patients meeting either resistant hypertension case definitions were combined for all analyses.

Controlled hypertensives (the “controls” for this association study) were defined as patients with a measurement of systolic blood pressure >140 mm Hg or diastolic blood pressure >90 mm Hg prior to meeting the medication criteria or with ICD-9 codes for hypertension (401.*) at any time and have one medication from the medication classes described above (and never have more than one simultaneous medication class, although the medication class can change), and have all systolic and diastolic measurements <135 mg/dl and <90 mg/dl, respectively, one month after blood pressure medications were prescribed (this requires at least one blood pressure measurement). For both cases and controls, simultaneous medication class usage was defined as evidence that the patient was taking the medication concurrently based on the presence of the medications in the same medication list (e.g., problem list, clinic note, or discharge summary) or via medication refill data with accompanying evidence of overlapping prescriptions for each drug. Like the resistant hypertension cases, individuals with systolic heart failure or chronic kidney disease were excluded.

Electronic selection logic using billing codes, laboratory values, text queries, and medication records was used to identify resistant hypertension cases and controls at each site ([S1 File](#pone.0171745.s008)). The initial algorithm used to identify cases and controls for this study of resistant hypertension was created and iteratively refined at VU with input from local experts as well as the eMERGE Phenotyping Workgroup and then deployed at the other study sites. At VU, physicians not associated with the algorithm development reviewed the de-identified clinical record of a fraction of identified cases and controls to assist in the refinement of the algorithm. In this iterative process, the algorithm was refined until a positive predictive value (PPV) of 92% was achieved based on reviews of 50 randomly selected clinical cases and a PPV of 84% for 50 randomly selected controls. Each round of reviews was independent and did not draw from the same set of identified cases and controls. Evaluation at the other four eMERGE I study sites confirmed PPVs of 84%-100%, including some sites that augmented the algorithmic selection with manual review. The general process of phenotype algorithm development in the eMERGE Network has been previously described [[31](#pone.0171745.ref031)], and the final version of workflows and algorithms are available at PheKB [[32](#pone.0171745.ref032)].

### Genotyping

Genotyping was performed for eMERGE I study sites using two Illumina arrays at two genotyping centers. Individuals of self-identified or administratively-assigned European-descent were genotyped on the Illumina 660W-Quad, while individuals of self-identified or administratively-assigned African-descent were genotyped on the Illumina 1M. For the majority of patients, genotyping was performed at one of two centers: the Center for Inherited Disease Research (CIDR) at Johns Hopkins University and the Center for Genotyping and Analysis at the Broad Institute as previously described [[33](#pone.0171745.ref033), [34](#pone.0171745.ref034)]. Existing genotype data available for eMERGE II study sites included data from the Illumina 550 (MC; n = 18), Illumina 610 (MC; n = 7), Illumina HumanOmni Express (GHS; n = 3,111), and Affymetrix 6.0 (MSSM; n = 2,775) [[35](#pone.0171745.ref035)].

Quality control (QC) measures were developed by the eMERGE Genomics Workgroup [[33](#pone.0171745.ref033)] and were implemented by the Coordinating Center. Briefly, the QC process included examination of sample quality and composition (i.e., sex inconsistency checks, sample call rates, sample relatedness, and population stratification), marker quality (i.e., marker call rate, duplicate concordance, minor allele frequency, and Hardy-Weinberg equilibrium), and of batch effects (i.e., average call rates and minor allele frequency per plate) [[33](#pone.0171745.ref033), [34](#pone.0171745.ref034)]. Following QC, analyses were limited to patients with call rates >98% and to SNPs with call rates >99% and minor allele frequencies >5%. Race/ethnicity was self-identified or administratively-assigned, and genetic ancestry was assessed using STRUCTURE [[36](#pone.0171745.ref036)] and EIGENSTRAT [[37](#pone.0171745.ref037)]. As reported previously, administratively-assigned race/ethnicity is highly concordant with genetic ancestry for European and African Americans [[38](#pone.0171745.ref038), [39](#pone.0171745.ref039)].

All genotype data were imputed by Pennsylvania State University Center for Systems Genomics as part of the eMERGE Coordinating Center as previously described [[35](#pone.0171745.ref035)]. Briefly, all autosomes were imputed using the 1000 Genomes cosmopolitan reference panel (n = 1,092) [[40](#pone.0171745.ref040)] using SHAPEIT2 [[41](#pone.0171745.ref041)] and IMPUTE2 [[42](#pone.0171745.ref042)] for phasing and imputation, respectively. Imputation was performed for each study site and genotyping assay separately and then merged for further quality control and analysis.

### Statistical methods

For the GWAS, single-SNP tests of association were performed in PLINK [[43](#pone.0171745.ref043)] and PLATO [[44](#pone.0171745.ref044)] using logistic regression and assuming an additive genetic model. All associations were adjusted for sex, decade of birth, median body mass index (BMI in kg/m^2^), genotyping platform, and genetic ancestry (via ten principal components [PCs] for multi-ethnic analyses and sex-stratified analyses, and three PCs for European-descent only analyses). Analyses were performed in the combined eMERGE I and eMERGE II datasets (3,006 cases and 876 controls). The models did not converge in the combined dataset, and inspection of the distribution of cases and controls by study site and genotyping platform revealed several strata with only case counts but no control counts. Removal of the case-only counts (231) enabled the models to converge.

The results of the GWAS tests of association were plotted using a Manhattan plot. Manhattan plots were generated by the R statistical package using code provided by the blog “Getting Genetics Done” ([http://www.gettinggeneticsdone.com/2011/04/annotated-manhattan-plots-and-qq-plots.html](http://www.gettinggeneticsdone.com/2011/04/annotated-manhattan-plots-and-qq-plots.html)), and regional plots were generated using LocusZoom at default settings [[45](#pone.0171745.ref045)]. Reported p-values were not corrected for multiple comparisons.

SNPs previously associated with blood pressure, systolic blood pressure, diastolic blood pressure, or hypertension among adults at genome-wide significance were drawn from the NHGRI European Bioinformatics Institute (NHGRI-EBI) GWAS Catalog ([http://www.ebi.ac.uk/gwas/](http://www.ebi.ac.uk/gwas/); accessed September 2016) [[46](#pone.0171745.ref046), [47](#pone.0171745.ref047)]. We then performed a look-up of these previously-associated blood pressure variants in the eMERGE Network resistant hypertension multi-ethnic GWAS dataset. Ad hoc power calculations were performed using CaTS [[48](#pone.0171745.ref048)].

## Results

### Identification of resistant hypertensive cases and controls

A total of 3,006 resistant hypertension (uncontrolled and controlled) cases and 876 controlled hypertensives were identified from the EHRs in eMERGE I and II ([Table 1](#pone.0171745.t001)). Slightly more than half the cases (55%) were identified by the first case algorithm (see [Methods](#sec002)). Cases and controls were drawn from seven study sites of eMERGE I and II, with VU (31%) and MFC (21%) contributing the largest percentage of individuals ([S3 Table](#pone.0171745.s011)). For both cases and controls, nearly half were male, and the median BMI was in the overweight category ([Table 1](#pone.0171745.t001)). The median birth decade was 1940 (25% and 75% quartiles were 1930 and 1940, respectively). With respect to race/ethnicity, approximately 65% of the cases were European American, and, in contrast, up 82% of controls were European American.

### Table 1. Characteristics of resistant hypertension cases and controls.

|   | Cases (n = 3,006) | Controls (n = 876) |
| --- | --- | --- |
| Male (%) | 42.81 | 44.98 |
| Median body mass index (kg/m2) | 29.80 | 28.63 |
| African American (%) | 26.51 | 13.93 |
| European American (%) | 64.77 | 81.62 |
| Hispanic (%) | 7.05 | 2.51 |
| Other (%) | 1.66 | 1.94 |

Table 1 Caption: Race/ethnicity is self-reported or administratively-assigned. Here, ethnicity refers to “Hispanic” and race refers to European American, African American, or other.

### Discovery

To identify common variants associated with risk of resistant hypertension, a genome-wide association study of 2,530,150 SNPs was performed among all resistant hypertension cases (n = 2,830) and controls (n = 876) adjusted for sex, decade of birth, median BMI, genotyping platform, and ten PCs. A single association at genome-wide significance was observed between case status and *ESR1* rs9479122 at p = 6.28x10^-11^ with an odds ratio of 0.46 ([S1](#pone.0171745.s001) and [S2](#pone.0171745.s002) Figs). When restricted to European Americans only (n = 1,609 cases and n = 667 controls), *ESR1* rs9479122 remained the most significant association, p = 1.12x10^-16^ (odds ratio = 0.29; [Table 2](#pone.0171745.t002); [S3](#pone.0171745.s003)–[S5](#pone.0171745.s005) Figs).

### Table 2. SNPs associated with resistant hypertension at p<10−6 in the eMERGE Network in the genome-wide association study.

| SNP | Chr:BP | Gene | All eMERGE I and II (ncases = 2,830; ncontrols = 876) |  |  | European Americans only (ncases = 1,719; ncontrols = 708) |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- |
| CAF (CA) | Odds ratio (95% CI) | P-value | CAF (CA) | Odds ratio (95% CI) | P-value |  |  |  |
| rs13144136 | 4:10666909 | CLNK (intronic) | 0.66 (G) | 0.68 (0.58, 0.80) | 1.00E-06 | 0.70 (G) | 0.66 (0.54, 0.79) | 7.42E-06 |
| rs73007615 | 2:134510949 | - | 0.84 (C) | 0.62 (0.51, 0.76) | 2.89E-06 | 0.82 (C) | 0.66 (0.53, 0.83) | 2.48E-04 |
| rs73007604 | 2:134506084 | - | 0.84 (C) | 0.62 (0.51, 0.76) | 3.11E-06 | 0.82 (C) | 0.66 (0.53, 0.83) | 2.48E-04 |
| rs80028883 | 2:134521148 | - | 0.84 (T) | 0.62 (0.51, 0.76) | 3.43E-06 | 0.82 (T) | 0.66 (0.53, 0.83) | 2.50E-04 |
| rs16829231 | 2:134531622 | - | 0.84 (T) | 0.62 (0.51, 0.76) | 3.51E-06 | 0.82 (T) | 0.66 (0.52, 0.82) | 2.37E-04 |
| rs73005736 | 2:134503462 | - | 0.84 (G) | 0.63 (0.51, 0.77) | 4.07E-06 | 0.82 (G) | 0.66 (0.53, 0.83) | 3.12E-04 |
| rs11681656 | 2:134495020 | - | 0.84 (G) | 0.63 (0.52, 0.77) | 5.78E-06 | 0.82 (G) | 0.67 (0.53, 0.84) | 4.41E-04 |
| rs4533516 | 2:134498958 | - | 0.84 (C) | 0.63 (0.52, 0.77) | 6.27E-06 | 0.82 (C) | 0.67 (0.53, 0.84) | 4.41E-04 |
| rs733170 | 2:134528895 | - | 0.84 (T) | 0.64 (0.52, 0.78) | 7.23E-06 | 0.82 (T) | 0.67 (0.53, 0.83) | 3.39E-04 |
| rs7016717 | 8:78630367 | - | 0.89 (G) | 1.77 (1.38, 2.27) | 7.32E-06 | 0.96 (G) | 1.70 (1.10, 2.63) | 1.72E-02 |
| rs12752401 | 1:102967150 | - | 0.93 (C) | 0.50 (0.36, 0.68) | 7.62E-06 | 0.92 (C) | 0.49 (0.35, 0.68) | 2.19E-05 |
| rs6712538 | 2:134473174 | - | 0.83 (C) | 0.64 (0.53, 0.78) | 7.79E-06 | 0.81 (C) | 0.68 (0.54, 0.84) | 3.60E-04 |
| rs6750839 | 2:114941169 | - | 0.92 (T) | 0.43 (0.29, 0.64) | 7.80E-06 | 1.00 (T) | 0.97 (0.03, 28.78) | 9.85E-01 |
| rs1374404 | 2:134495807 | - | 0.84 (T) | 0.64 (0.52, 0.78) | 9.97E-06 | 0.82 (G) | 0.67 (0.53, 0.84) | 4.41E-04 |

Table 2 Caption: After removal of ESR1 rs9479122, single-SNP tests of association were performed for 2,530,149 SNPs in eMERGE I and II using logistic regression, assuming an additive genetic model, adjusted for sex, decade of birth, median body mass index, genotyping platform, and genetic ancestry (principal components 1 through 10). Results are shown for tests of association in the eMERGE I and II Network at p<10−6. Tests of association were repeated for European Americans only using logistic regression, assuming an additive genetic model, adjusted for sex, decade of birth, median body mass index, genotyping platform, and genetic ancestry (principal components 1 through 3). Results are also shown for European Americans for SNPs associated with resistant hypertension at p<10−6 in the eMERGE I and II Network. Abbreviations: basepair (bp), chromosome (chr), coded allele (CA), coded allele frequency (CAF).

To corroborate these potential findings, we performed a look-up for *ESR1* rs9479122 in two independent datasets, INVEST [[21](#pone.0171745.ref021), [49](#pone.0171745.ref049)] and SPS3 [[50](#pone.0171745.ref050)]. INVEST was a large hypertension outcomes trial, and SPS3 was a secondary stroke prevention trial with a blood pressure arm. Both trials conducted genetic sub-studies and resistant hypertension phenotypes were constructed in a manner similar to that constructed in eMERGE. In INVEST and SPS3, *ESR1* rs9479122 is nearly monomorphic (coded allele G frequency 0.996 and 0.99937, respectively), which is in agreement with HapMap CEU estimates. *ESR1* rs9479122 is imputed in this eMERGE dataset [[35](#pone.0171745.ref035)].

Given the differences in the frequency of the imputed variant in eMERGE versus reference data and the look-up datasets, we deemed *ESR1* rs9479122 a likely false positive, removed it from the dataset, and re-generated the Manhattan plots (Figs [2](#pone.0171745.g002) and [3](#pone.0171745.g003)). No tests of association were genome-wide significant in either the overall dataset or among European Americans only ([S6](#pone.0171745.s006) and [S7](#pone.0171745.s007) Figs). In the overall dataset, the most significant finding was *CLNK* rs13144136 (odds ratio = 0.68; p = 1.00x10^-6^). *CLNK* rs13144136 was not associated with resistant hypertension in either the INVEST or SPS3 datasets (p>0.20). A total of 360 associations had nominal significance at p<10^−4^ in the present study, of which 14 were associated at p<10^−5^ ([Table 2](#pone.0171745.t002)).

### Fig 2. Genome-wide association analysis of individuals with resistant hypertension versus controlled hypertensives.

![Fig 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dae3/5319785/62fffd3f284b/pone.0171745.g002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5319785_pone.0171745.g002.jpg)

A total of 2,830 cases of resistant hypertension and 876 controlled hypertensives from eMERGE I and II was available for analysis after quality control. After removing imputed genotypes for ESR1 rs9479122, single-SNP tests of association were performed for 2,530,149 SNPs using logistic regression, assuming an additive genetic model, adjusted for sex, decade of birth, median body mass index, genotyping platform, and genetic ancestry (principal components 1 through 10). Results of each test of association are plotted as p-values (expressed as–log10 on the y-axis) by chromosome (x-axis).

### Fig 3. Q-Q plot of genome-wide association analysis of individuals with resistant hypertension versus controlled hypertensives.

![Fig 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dae3/5319785/f097b22838a5/pone.0171745.g003.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5319785_pone.0171745.g003.jpg)

The Q-Q plots were re-generated after removal of ESR1 rs9479122.

### Association of previously-identified hypertension and blood pressure variants with resistant hypertension

We examined whether SNPs known to influence blood pressure or hypertension also influenced resistant hypertension. The NHGRI-EBI GWAS catalog was used to select SNPs previously associated (p<5.0x10^-8^) with blood pressure, systolic blood pressure, diastolic blood pressure, or hypertension. From published GWAS papers [[51](#pone.0171745.ref051)–[69](#pone.0171745.ref069)], 118 SNPs were selected for examination in the current genome-wide studies ([S4 Table](#pone.0171745.s012)). Of the 118 SNPs, 50 were available in the eMERGE I and II imputed dataset. Of the 50 tests of association performed in the present resistant hypertension study, no one association was significant (p<0.001) and in the same direction compared with the previously published literature. The most significant finding in the present study was *TBX3* rs35444, previously associated with increased diastolic blood pressure in East Asians [[51](#pone.0171745.ref051)], associated with resistant hypertension (eMERGE I and II combined odds ratio = 1.27; p = 1.30x10^-3^) in this multi-ethnic clinical study. With 2,830 cases and 876 controls, we were powered to replicate genetic associations for common variants (allele frequency ≥5%) with an odds ratio ≥1.5, assuming a genetic additive model and a significance threshold of 0.001. As might be expected given that the SNPs tested were from previously reported GWAS, all but one (*SLC39A8* rs13107325) of the 50 SNPs met the minor allele frequency threshold assumed in the power calculations based on the 1000 Genomes Project phase 3 genotype data from 2,500 worldwide individuals [[70](#pone.0171745.ref070)]. Most of the previously-reported associations tested here for associations with resistant hypertension were studies of quantitative traits (such as diastolic and systolic blood pressures) with effect sizes (β) ranging from 0.17 to 5.48 (median = 0.521). Of the few studies reporting odds ratio for associations with hypertension, only one reported an odds ratio >1.5 [[71](#pone.0171745.ref071)]. Although direct comparisons are difficult to make between the present study’s dichotomous outcome and previous studies’ quantitative outcomes ([S4 Table](#pone.0171745.s012)), it is likely that the present study was under-powered to replicate or generalize previous findings from blood pressure GWAS.

## Discussion

We demonstrate here that DNA biobanks linked to EHRs can be used to classify resistant hypertension cases and controls and to perform large-scale genome-wide association studies. Using an algorithm based on a combination of billing codes, laboratory values, text queries, and prescription records, we were able to identify 876 controlled hypertensives and 3,006 cases of resistant hypertension from a sample of >55,000 US adults available in eMERGE Phase I and II [[72](#pone.0171745.ref072)]. Overall, no variants were associated with resistant hypertension at genome-wide significance in either the full dataset or the dataset limited to European Americans. Also, none of the loci previously identified for hypertension and/or blood pressure was associated with resistant hypertension in cases and controls drawn from eMERGE I and II after correction for multiple testing. It is unclear if the lack of associations observed here represent a lack of sufficient power, a lack of accurate phenotyping, or in the case of the hypertension/blood pressure-associated variants, a lack of biological relevance or connection to the phenotype of resistant hypertension.

This present work expands on earlier studies of the genetics of resistant hypertension. The handful of studies that have specifically investigated resistant hypertension have mostly focused on candidate genes: β and γ subunits of the epithelial sodium channel (*ENaC*) [[17](#pone.0171745.ref017)], cytochrome P450 gene (*CYP3A5*) [[18](#pone.0171745.ref018)], glutathione S-transferase mu type 1 gene (*GSTM1*) [[16](#pone.0171745.ref016)], and endothelial nitric oxide synthase gene (*NOS3*) [[19](#pone.0171745.ref019), [20](#pone.0171745.ref020)]. However, these studies suffered from several limitations including small sample sizes (ranging from 11 to 347 cases) and less-than-stringent significance thresholds. Of note is the large-scale candidate gene study in INVEST [[21](#pone.0171745.ref021)] that tested for an association between resistant hypertension (with >500 total cases) and approximately 50,000 common genetic variants targeted by the Illumina HumanCVD Genotyping BeadChip [[73](#pone.0171745.ref073)]. The most significant finding in INVEST (*ATP2B1* rs12817819) did not replicate in an independent dataset, but reached chip-wide significance after meta-analysis [[21](#pone.0171745.ref021)].

### Strengths and limitations

Major strengths of our study include the scale and longitudinal nature of the EHR. The combination of collaboration among research groups with access to biobanks and advances in high-throughput technology allowed us to genotype hundreds of thousands of genetic variants and impute to more than two million SNPs in >2,800 resistant hypertensive patients, enabling us to test for novel associations that may not have been interrogated via traditional candidate gene studies. However, compared with GWAS of common diseases, studies requiring exposure to medications or other treatments face additional challenges [[74](#pone.0171745.ref074)]. While GWAS of common diseases such as type 2 diabetes [[75](#pone.0171745.ref075), [76](#pone.0171745.ref076)] or lipid levels [[77](#pone.0171745.ref077), [78](#pone.0171745.ref078)] are able to accrue tens to hundreds of thousands of individuals for study, obtaining adequate numbers for some clinical traits is challenging as observational cohorts often do not have sufficient data related to drug exposure or treatments, and existing sets of genotyped clinical records are still relatively small. Indeed, even with our fairly large sample size, we were still underpowered to detect small effects at genome-wide significance. Specifically, for common SNPs (minor allele frequency = 0.20), we had 80% power to detect an odds ratio of 1.41.

Statistical power has been a major obstacle for studies designed to identify genetic variants associated with blood pressure in general. Indeed, even contemporary studies with access to genome-wide and imputed data of millions of common variants estimate that one-third to one-half the variability observed for systolic and diastolic blood pressure levels can be attributed to the additive effects of many genetic variants; to date, only ~2% of blood pressure variation can be explained by known blood pressure-associated variants [[79](#pone.0171745.ref079)], each with very small contributions, thus requiring large sample sizes to detect the associations via GWAS [[79](#pone.0171745.ref079)]. Given the genetic and environmental complexity of blood pressure, it is not surprising that the present study was unable to identify novel variants associated with resistant hypertension.

The upcoming NIH Precision Medicine Initiative Cohort Program (“All of Us”), which seeks to accrue >1 million individuals with longitudinal EHR data, may facilitate future studies on drug effects on a scale not possible today [[80](#pone.0171745.ref080), [81](#pone.0171745.ref081)]. Also, longitudinal studies such as those proposed by the NIH Precision Medicine Initiative Cohort Program increase the chances that secondary causes of resistant hypertension will be identified and eliminated to reduce phenotypic heterogeneity. However, it is worth noting that while potentially large, the Precision Medicine Initiative Cohort Program may suffer from similar limitations experienced by the individual study sites of the eMERGE Network. That is, the eMERGE Network is large in overall sample size, but the requirements of repeated blood pressure measurements and prescription information over the course of time severely limited the case and control counts available for the association study. This dramatic loss of sample size and resulting loss of power is a reflection of the fragmented health care system in the United States. Studies requiring extensive longitudinal data may be better powered in health care settings where patients regularly receive the vast majority of his or her care within a single setting such as an integrated managed care consortium (Kaiser Permanente [[82](#pone.0171745.ref082), [83](#pone.0171745.ref083)]) or the Veterans Health Administration [[84](#pone.0171745.ref084)].

Phenotypic heterogeneity in general is a major challenge faced by most studies [[74](#pone.0171745.ref074)]. Our study is no exception. While our algorithm and subsequent review removed potential sources of case misclassification (such as acute myocardial infarction patients on more than one class of medication), it is important to note that elevated blood pressure readings may result from measurement error, other concurrent medical conditions (e.g., pain or anxiety), poor patient compliance, or “white-coat hypertension.” Patients who lack blood pressure control despite appropriate treatment due to the aforementioned reasons may present with resistant hypertension but are actually “pseudoresistant” [[9](#pone.0171745.ref009)]. Potential lifestyle causes of resistant hypertension, such as excessive dietary salt intake and heavy alcohol use, are also not measured and recorded in the EHR. While large-scale biobanks linked to EHRs offer important advantages in research settings, misclassification bias must be recognized and minimized by carefully characterizing the phenotype and exploiting the longitudinal nature of the EHR [[85](#pone.0171745.ref085)].

Of special note is the issue of poor compliance or non-compliance. Most studies involving medications or self-administered treatments have a problem with non-compliance. For hypertension, clinical and epidemiologic surveys continuously suggest that a high proportion of patients do not achieve target blood pressure levels after treatment [[86](#pone.0171745.ref086), [87](#pone.0171745.ref087)]. This observed resistance to treatment can be reflective of true resistance, the outcome of interest in the present study, or apparent resistance which often times is reflective of poor adherence to treatment. Previous retrospective studies have noted that approximately 40% of newly diagnosed hypertensive patients discontinue treatment within a year [[7](#pone.0171745.ref007), [88](#pone.0171745.ref088), [89](#pone.0171745.ref089)]. Multiple factors have been associated with poor adherence to hypertensive treatment or treatment in general including non-patient (such as drug class [[90](#pone.0171745.ref090)]) and patient factors (such as sex, race/ethnicity, and socio-economic status [[91](#pone.0171745.ref091)]), and some data suggest that women prescribed anti-hypertensives are more compliant than men [[92](#pone.0171745.ref092)]. Nearly half the sample in this present study was male. Additional information such as prescription claims data could help clarify this issue of non-compliance.

Finally, a major challenge in designing optimal studies to identify and characterize the genetic architecture of resistant hypertension may be related to the as-of-yet unknown etiology of the phenotype(s) [[10](#pone.0171745.ref010)]. For example, more traditional pharmacogenomics studies of blood pressure and response to anti-hypertensive treatments concentrate on a single class of drug (such as hydrochlorothiazide monotherapy [[93](#pone.0171745.ref093)]) representing a specific pathway or mechanism of action. It could be argued that resistant hypertension represents the dichotomized version of the extremes of the drug-response distribution as the majority of resistant hypertension cases studied here required four medications from different classes. It is unlikely that a single genetic variant affects all pathways represented by these medication classes but rather a complex combination or interaction of variants that phenotypically result in resistant hypertension. This genetic heterogeneity in combination with the aforementioned phenotypic heterogeneity and phenotyping requirements makes the genetic study of resistant hypertension especially challenging.

In conclusion, we describe an approach to identify resistant hypertension cases and controls from seven different institutions’ EHRs. To our knowledge, this is the only GWAS of resistant hypertension in any population. These results highlight the utility of EHR-linked genomic data to further refine our understanding of disease and its subtypes, with the goal of uncovering new therapeutic targets and providing better targeting of current therapies.

## Supporting information

## Data Availability

Data are available in the database of Genotypes and Phenotypes (dbGaP) study accession phs000297.v1.p1.

## Funding Statement

The eMERGE Network is funded by NHGRI, with additional funding from NIGMS through the following grants: U01HG04599 and U01HG006379 to Mayo Clinic; U01HG004610 and U01HG006375 to Group Health Cooperative and University of Washington, Seattle; U01HG004608 to Marshfield Clinic; U01HG006389 to Essentia Institute of Rural Health; U01HG004609 and U01HG006388 to Northwestern University; U01HG04603 and U01HG006378 to Vanderbilt University; U01HG006385 to the Coordinating Center; U01HG006382 to Geisinger Clinic; U01HG006380 to Mount Sinai School of Medicine; U01HG006830 to The Children's Hospital of Philadelphia; and U01HG006828 to Cincinnati Children's Hospital and Boston Children's Hospital. A portion of the dataset(s) used for the analyses described were obtained from Vanderbilt University Medical Center’s BioVU, supported by institutional funding and by the Vanderbilt CTSA grant UL1 TR000445 from NCATS/NIH, and the Mayo Clinic Biobank supported by the Mayo Clinic Center for Individualized Medicine. INVEST-GENES was funded by NIH grants R01 HL074730 and U01 GM074492. SPS3 and SPS3-GENES were funded by NIH grants U01 NS038529, U01 GM074492, and R01 NS073346. CWM was also supported by KL2 TR001429. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Data Availability Statement

Data are available in the database of Genotypes and Phenotypes (dbGaP) study accession phs000297.v1.p1.

### Supplementary Materials

### Data Availability Statement

Data are available in the database of Genotypes and Phenotypes (dbGaP) study accession phs000297.v1.p1.

## References

1. Guo F, He D, Zhang W, Walton RG. Trends in Prevalence, Awareness, Management, and Control of Hypertension Among United States Adults, 1999 to 2010. Journal of the American College of Cardiology. 2012;60(7):599–606. 10.1016/j.jacc.2012.04.026  [DOI](https://doi.org/10.1016/j.jacc.2012.04.026) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22796254/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20of%20the%20American%20College%20of%20Cardiology&title=Trends%20in%20Prevalence,%20Awareness,%20Management,%20and%20Control%20of%20Hypertension%20Among%20United%20States%20Adults,%201999%20to%202010&author=F%20Guo&author=D%20He&author=W%20Zhang&author=RG%20Walton&volume=60&issue=7&publication_year=2012&pages=599-606&pmid=22796254&doi=10.1016/j.jacc.2012.04.026&)

2. Fields LE, Burt VL, Cutler JA, Hughes J, Roccella EJ, Sorlie P. The Burden of Adult Hypertension in the United States 1999 to 2000: A Rising Tide. Hypertension. 2004;44(4):398–404. 10.1161/01.HYP.0000142248.54761.56  [DOI](https://doi.org/10.1161/01.HYP.0000142248.54761.56) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15326093/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hypertension&title=The%20Burden%20of%20Adult%20Hypertension%20in%20the%20United%20States%201999%20to%202000:%20A%20Rising%20Tide&author=LE%20Fields&author=VL%20Burt&author=JA%20Cutler&author=J%20Hughes&author=EJ%20Roccella&volume=44&issue=4&publication_year=2004&pages=398-404&pmid=15326093&doi=10.1161/01.HYP.0000142248.54761.56&)

3. Lenfant C. Reflections on hypertension control rates: A message from the director of the national heart, lung, and blood institute. Archives of Internal Medicine. 2002;162(2):131–2.  [DOI](https://doi.org/10.1001/archinte.162.2.131) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11802745/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Archives%20of%20Internal%20Medicine&title=Reflections%20on%20hypertension%20control%20rates:%20A%20message%20from%20the%20director%20of%20the%20national%20heart,%20lung,%20and%20blood%20institute&author=C%20Lenfant&volume=162&issue=2&publication_year=2002&pages=131-2&pmid=11802745&doi=10.1001/archinte.162.2.131&)

4. Chobanian AV, Bakris GL, Black HR, Cushman WC, Green LA, Izzo JL, et al. Seventh Report of the Joint National Committee on Prevention, Detection, Evaluation, and Treatment of High Blood Pressure. Hypertension. 2003;42(6):1206–52. 10.1161/01.HYP.0000107251.49515.c2  [DOI](https://doi.org/10.1161/01.HYP.0000107251.49515.c2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14656957/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hypertension&title=Seventh%20Report%20of%20the%20Joint%20National%20Committee%20on%20Prevention,%20Detection,%20Evaluation,%20and%20Treatment%20of%20High%20Blood%20Pressure&author=AV%20Chobanian&author=GL%20Bakris&author=HR%20Black&author=WC%20Cushman&author=LA%20Green&volume=42&issue=6&publication_year=2003&pages=1206-52&pmid=14656957&doi=10.1161/01.HYP.0000107251.49515.c2&)

5. Go AS, Mozaffarian D, Roger VL, Benjamin EJ, Berry JD, Blaha MJ, et al. Heart Disease and Stroke Statistics—2014 Update. A Report From the American Heart Association. 2014;129(3):e28–e292.  [DOI](https://doi.org/10.1161/01.cir.0000441139.02102.80) | [PMC free article](/articles/PMC5408159/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24352519/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=A%20Report%20From%20the%20American%20Heart%20Association&title=Heart%20Disease%20and%20Stroke%20Statistics%E2%80%942014%20Update&author=AS%20Go&author=D%20Mozaffarian&author=VL%20Roger&author=EJ%20Benjamin&author=JD%20Berry&volume=129&issue=3&publication_year=2014&pages=e28-e292&pmid=24352519&doi=10.1161/01.cir.0000441139.02102.80&)

6. Bakris G, Sarafidis P, Agarwal R, Ruilope L. Review of blood pressure control rates and outcomes. Journal of the American Society of Hypertension. 2014;8(2):127–41. 10.1016/j.jash.2013.07.009  [DOI](https://doi.org/10.1016/j.jash.2013.07.009) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24309125/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20of%20the%20American%20Society%20of%20Hypertension&title=Review%20of%20blood%20pressure%20control%20rates%20and%20outcomes&author=G%20Bakris&author=P%20Sarafidis&author=R%20Agarwal&author=L%20Ruilope&volume=8&issue=2&publication_year=2014&pages=127-41&pmid=24309125&doi=10.1016/j.jash.2013.07.009&)

7. Calhoun DA, Jones D, Textor S, Goff DC, Murphy TP, Toto RD, et al. Resistant Hypertension: Diagnosis, Evaluation, and Treatment: A Scientific Statement From the American Heart Association Professional Education Committee of the Council for High Blood Pressure Research. Circulation. 2008;117(25):e510–e26. 10.1161/CIRCULATIONAHA.108.189141  [DOI](https://doi.org/10.1161/CIRCULATIONAHA.108.189141) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18574054/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circulation&title=Resistant%20Hypertension:%20Diagnosis,%20Evaluation,%20and%20Treatment:%20A%20Scientific%20Statement%20From%20the%20American%20Heart%20Association%20Professional%20Education%20Committee%20of%20the%20Council%20for%20High%20Blood%20Pressure%20Research&author=DA%20Calhoun&author=D%20Jones&author=S%20Textor&author=DC%20Goff&author=TP%20Murphy&volume=117&issue=25&publication_year=2008&pages=e510-e26&pmid=18574054&doi=10.1161/CIRCULATIONAHA.108.189141&)

8. Sarafidis PA, Georgianos P, Bakris GL. Resistant hypertension—its identification and epidemiology. Nat Rev Nephrol. 2013;9(1):51–8. 10.1038/nrneph.2012.260  [DOI](https://doi.org/10.1038/nrneph.2012.260) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23165303/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Nephrol&title=Resistant%20hypertension%E2%80%94its%20identification%20and%20epidemiology&author=PA%20Sarafidis&author=P%20Georgianos&author=GL%20Bakris&volume=9&issue=1&publication_year=2013&pages=51-8&pmid=23165303&doi=10.1038/nrneph.2012.260&)

9. Sarafidis PA, Bakris GL. Resistant Hypertension: An Overview of Evaluation and Treatment. Journal of the American College of Cardiology. 2008;52(22):1749–57. 10.1016/j.jacc.2008.08.036  [DOI](https://doi.org/10.1016/j.jacc.2008.08.036) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19022154/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20of%20the%20American%20College%20of%20Cardiology&title=Resistant%20Hypertension:%20An%20Overview%20of%20Evaluation%20and%20Treatment&author=PA%20Sarafidis&author=GL%20Bakris&volume=52&issue=22&publication_year=2008&pages=1749-57&pmid=19022154&doi=10.1016/j.jacc.2008.08.036&)

10. El Rouby N, Cooper-DeHoff R. Genetics of Resistant Hypertension: a Novel Pharmacogenomics Phenotype. Curr Hypertens Rep. 2015;17(9):1–11.  [DOI](https://doi.org/10.1007/s11906-015-0583-8) | [PMC free article](/articles/PMC4717470/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26198781/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Hypertens%20Rep&title=Genetics%20of%20Resistant%20Hypertension:%20a%20Novel%20Pharmacogenomics%20Phenotype&author=N%20El%20Rouby&author=R%20Cooper-DeHoff&volume=17&issue=9&publication_year=2015&pages=1-11&pmid=26198781&doi=10.1007/s11906-015-0583-8&)

11. Franceschini N, Reiner AP, Heiss G. Recent Findings in the Genetics of Blood Pressure and Hypertension Traits. American Journal of Hypertension. 2011;24(4):392–400. 10.1038/ajh.2010.218  [DOI](https://doi.org/10.1038/ajh.2010.218) | [PMC free article](/articles/PMC3110743/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20948529/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=American%20Journal%20of%20Hypertension&title=Recent%20Findings%20in%20the%20Genetics%20of%20Blood%20Pressure%20and%20Hypertension%20Traits&author=N%20Franceschini&author=AP%20Reiner&author=G%20Heiss&volume=24&issue=4&publication_year=2011&pages=392-400&pmid=20948529&doi=10.1038/ajh.2010.218&)

12. Binder A. A review of the genetics of essential hypertension. Curr Opin Cardiol. 2007;22(3):176–84. 10.1097/HCO.0b013e3280d357f9  [DOI](https://doi.org/10.1097/HCO.0b013e3280d357f9) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17413273/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Opin%20Cardiol&title=A%20review%20of%20the%20genetics%20of%20essential%20hypertension&author=A%20Binder&volume=22&issue=3&publication_year=2007&pages=176-84&pmid=17413273&doi=10.1097/HCO.0b013e3280d357f9&)

13. Johnson JA. Advancing management of hypertension through pharmacogenomics. Annals of Medicine. 2012;44(sup1):S17–S22.  [DOI](https://doi.org/10.3109/07853890.2011.653399) | [PMC free article](/articles/PMC3686275/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22713143/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Annals%20of%20Medicine&title=Advancing%20management%20of%20hypertension%20through%20pharmacogenomics&author=JA%20Johnson&volume=44&issue=sup1&publication_year=2012&pages=S17-S22&pmid=22713143&doi=10.3109/07853890.2011.653399&)

14. Lupton SJ, Chiu CL, Lind JM. A hypertension gene: are we there yet? Twin Res Hum Genet. 2011;14(4):295–304. 10.1375/twin.14.4.295  [DOI](https://doi.org/10.1375/twin.14.4.295) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21787111/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Twin%20Res%20Hum%20Genet&title=A%20hypertension%20gene:%20are%20we%20there%20yet?&author=SJ%20Lupton&author=CL%20Chiu&author=JM%20Lind&volume=14&issue=4&publication_year=2011&pages=295-304&pmid=21787111&doi=10.1375/twin.14.4.295&)

15. Gong Y, McDonough CW, Beitelshees AL, El Rouby N, Hiltunen TP, O'Connell JR, et al. PTPRD gene associated with blood pressure response to atenolol and resistant hypertension. J Hyertens. 2015;33(11):2278–85.  [DOI](https://doi.org/10.1097/HJH.0000000000000714) | [PMC free article](/articles/PMC4788379/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26425837/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Hyertens&title=PTPRD%20gene%20associated%20with%20blood%20pressure%20response%20to%20atenolol%20and%20resistant%20hypertension&author=Y%20Gong&author=CW%20McDonough&author=AL%20Beitelshees&author=N%20El%20Rouby&author=TP%20Hiltunen&volume=33&issue=11&publication_year=2015&pages=2278-85&pmid=26425837&doi=10.1097/HJH.0000000000000714&)

16. Cruz-Gonzalez I, Corral E, Sanchez-Ledesma M, Sanchez-Rodriguez A, Martin-Luengo C, Gonzalez-Sarmiento R. An association between resistant hypertension and the null GSTM1 genotype. J Hum Hypertens. 2009;23(8):556–8. 10.1038/jhh.2009.19  [DOI](https://doi.org/10.1038/jhh.2009.19) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19279659/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Hum%20Hypertens&title=An%20association%20between%20resistant%20hypertension%20and%20the%20null%20GSTM1%20genotype&author=I%20Cruz-Gonzalez&author=E%20Corral&author=M%20Sanchez-Ledesma&author=A%20Sanchez-Rodriguez&author=C%20Martin-Luengo&volume=23&issue=8&publication_year=2009&pages=556-8&pmid=19279659&doi=10.1038/jhh.2009.19&)

17. Hannila-Handelberg T, Kontula K, Tikkanen I, Tikkanen T, Fyhrquist F, Helin K, et al. Common variants of the beta and gamma subunits of the epithelial sodium channel and their relation to plasma renin and aldosterone levels in essential hypertension. BMC Medical Genetics. 2005;6(1):1–13.  [DOI](https://doi.org/10.1186/1471-2350-6-4) | [PMC free article](/articles/PMC547905/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15661075/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Medical%20Genetics&title=Common%20variants%20of%20the%20beta%20and%20gamma%20subunits%20of%20the%20epithelial%20sodium%20channel%20and%20their%20relation%20to%20plasma%20renin%20and%20aldosterone%20levels%20in%20essential%20hypertension&author=T%20Hannila-Handelberg&author=K%20Kontula&author=I%20Tikkanen&author=T%20Tikkanen&author=F%20Fyhrquist&volume=6&issue=1&publication_year=2005&pages=1-13&pmid=15661075&doi=10.1186/1471-2350-6-4&)

18. Ho H, Pinto A, Hall SD, Flockhart DA, Li L, Skaar TC, et al. Association Between the CYP3A5 Genotype and Blood Pressure. Hypertension. 2005;45(2):294–8. 10.1161/01.HYP.0000151361.31736.96  [DOI](https://doi.org/10.1161/01.HYP.0000151361.31736.96) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15596575/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hypertension&title=Association%20Between%20the%20CYP3A5%20Genotype%20and%20Blood%20Pressure&author=H%20Ho&author=A%20Pinto&author=SD%20Hall&author=DA%20Flockhart&author=L%20Li&volume=45&issue=2&publication_year=2005&pages=294-8&pmid=15596575&doi=10.1161/01.HYP.0000151361.31736.96&)

19. Cruz-González I, Corral E, Sánchez-Ledesma M, Sánchez-Rodríguez A, Martín-Luengo C, González-Sarmiento R. Association between -T786C NOS3 polymorphism and resistant hypertension: a prospective cohort study. BMC Cardiovascular Disorders. 2009;9(1):1–6.  [DOI](https://doi.org/10.1186/1471-2261-9-35) | [PMC free article](/articles/PMC2734743/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19650939/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Cardiovascular%20Disorders&title=Association%20between%20-T786C%20NOS3%20polymorphism%20and%20resistant%20hypertension:%20a%20prospective%20cohort%20study&author=I%20Cruz-Gonz%C3%A1lez&author=E%20Corral&author=M%20S%C3%A1nchez-Ledesma&author=A%20S%C3%A1nchez-Rodr%C3%ADguez&author=C%20Mart%C3%ADn-Luengo&volume=9&issue=1&publication_year=2009&pages=1-6&pmid=19650939&doi=10.1186/1471-2261-9-35&)

20. Jáchymová M, Horký K, Bultas J, Kožich V, Jindra A, Peleška J, et al. Association of the Glu298Asp Polymorphism in the Endothelial Nitric Oxide Synthase Gene with Essential Hypertension Resistant to Conventional Therapy. Biochemical and Biophysical Research Communications. 2001;284(2):426–30. 10.1006/bbrc.2001.5007  [DOI](https://doi.org/10.1006/bbrc.2001.5007) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11394896/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochemical%20and%20Biophysical%20Research%20Communications&title=Association%20of%20the%20Glu298Asp%20Polymorphism%20in%20the%20Endothelial%20Nitric%20Oxide%20Synthase%20Gene%20with%20Essential%20Hypertension%20Resistant%20to%20Conventional%20Therapy&author=M%20J%C3%A1chymov%C3%A1&author=K%20Hork%C3%BD&author=J%20Bultas&author=V%20Ko%C5%BEich&author=A%20Jindra&volume=284&issue=2&publication_year=2001&pages=426-30&pmid=11394896&doi=10.1006/bbrc.2001.5007&)

21. Fontana V, McDonough CW, Gong Y, El Rouby NM, Sa AC, Taylor KD, et al. Large-scale gene-centric analysis identifies polymorphisms for resistant hypertension. J Am Heart Assoc. 2014;3(6):e001398 10.1161/JAHA.114.001398  [DOI](https://doi.org/10.1161/JAHA.114.001398) | [PMC free article](/articles/PMC4338734/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25385345/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Am%20Heart%20Assoc&title=Large-scale%20gene-centric%20analysis%20identifies%20polymorphisms%20for%20resistant%20hypertension&author=V%20Fontana&author=CW%20McDonough&author=Y%20Gong&author=NM%20El%20Rouby&author=AC%20Sa&volume=3&issue=6&publication_year=2014&pages=e001398&pmid=25385345&doi=10.1161/JAHA.114.001398&)

22. McCarty C, Chisholm R, Chute C, Kullo I, Jarvik G, Larson E, et al. The eMERGE Network: A consortium of biorepositories linked to electronic medical records data for conducting genomic studies. BMC Medical Genomics. 2011;4(1):13.  [DOI](https://doi.org/10.1186/1755-8794-4-13) | [PMC free article](/articles/PMC3038887/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21269473/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Medical%20Genomics&title=The%20eMERGE%20Network:%20A%20consortium%20of%20biorepositories%20linked%20to%20electronic%20medical%20records%20data%20for%20conducting%20genomic%20studies&author=C%20McCarty&author=R%20Chisholm&author=C%20Chute&author=I%20Kullo&author=G%20Jarvik&volume=4&issue=1&publication_year=2011&pages=13&pmid=21269473&doi=10.1186/1755-8794-4-13&)

23. Gottesman O, Kuivaniemi H, Tromp G, Faucett WA, Li R, Manolio TA, et al. The Electronic Medical Records and Genomics (eMERGE) Network: past, present, and future. Genet Med. 2013;15(10):761–71. 10.1038/gim.2013.72  [DOI](https://doi.org/10.1038/gim.2013.72) | [PMC free article](/articles/PMC3795928/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23743551/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genet%20Med&title=The%20Electronic%20Medical%20Records%20and%20Genomics%20(eMERGE)%20Network:%20past,%20present,%20and%20future&author=O%20Gottesman&author=H%20Kuivaniemi&author=G%20Tromp&author=WA%20Faucett&author=R%20Li&volume=15&issue=10&publication_year=2013&pages=761-71&pmid=23743551&doi=10.1038/gim.2013.72&)

24. Roden DM, Pulley JM, Basford MA, Bernard GR, Clayton EW, Balser JR, et al. Development of a Large-Scale De-Identified DNA Biobank to Enable Personalized Medicine. Clin Pharmacol Ther. 2008;84(3):362–9. 10.1038/clpt.2008.89  [DOI](https://doi.org/10.1038/clpt.2008.89) | [PMC free article](/articles/PMC3763939/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18500243/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Development%20of%20a%20Large-Scale%20De-Identified%20DNA%20Biobank%20to%20Enable%20Personalized%20Medicine&author=DM%20Roden&author=JM%20Pulley&author=MA%20Basford&author=GR%20Bernard&author=EW%20Clayton&volume=84&issue=3&publication_year=2008&pages=362-9&pmid=18500243&doi=10.1038/clpt.2008.89&)

25. Pulley J, Clayton E, Bernard GR, Roden DM, Masys DR. Principles of Human Subjects Protections Applied in an Opt-Out, De-identified Biobank. Clinical and Translational Science. 2010;3(1):42–8. 10.1111/j.1752-8062.2010.00175.x  [DOI](https://doi.org/10.1111/j.1752-8062.2010.00175.x) | [PMC free article](/articles/PMC3075971/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20443953/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clinical%20and%20Translational%20Science&title=Principles%20of%20Human%20Subjects%20Protections%20Applied%20in%20an%20Opt-Out,%20De-identified%20Biobank&author=J%20Pulley&author=E%20Clayton&author=GR%20Bernard&author=DM%20Roden&author=DR%20Masys&volume=3&issue=1&publication_year=2010&pages=42-8&pmid=20443953&doi=10.1111/j.1752-8062.2010.00175.x&)

26. Kho AN, Pacheco JA, Peissig PL, Rasmussen L, Newton KM, Weston N, et al. Electronic Medical Records for Genetic Research: Results of the eMERGE Consortium. Science Translational Medicine. 2011;3(79):79re1–re1. 10.1126/scitranslmed.3001807  [DOI](https://doi.org/10.1126/scitranslmed.3001807) | [PMC free article](/articles/PMC3690272/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21508311/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science%20Translational%20Medicine&title=Electronic%20Medical%20Records%20for%20Genetic%20Research:%20Results%20of%20the%20eMERGE%20Consortium&author=AN%20Kho&author=JA%20Pacheco&author=PL%20Peissig&author=L%20Rasmussen&author=KM%20Newton&volume=3&issue=79&publication_year=2011&pages=79re1-re1&pmid=21508311&doi=10.1126/scitranslmed.3001807&)

27. Denny JC, Crawford DC, Ritchie MD, Bielinski SJ, Basford MA, Bradford Y, et al. Variants near FOXE1 are associated with hypothyroidism and other thyroid conditions: using electronic medical records for genome- and phenome-wide studies. Am J Hum Genet. 2011;89(4):529–42. 10.1016/j.ajhg.2011.09.008  [DOI](https://doi.org/10.1016/j.ajhg.2011.09.008) | [PMC free article](/articles/PMC3188836/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21981779/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=Variants%20near%20FOXE1%20are%20associated%20with%20hypothyroidism%20and%20other%20thyroid%20conditions:%20using%20electronic%20medical%20records%20for%20genome-%20and%20phenome-wide%20studies&author=JC%20Denny&author=DC%20Crawford&author=MD%20Ritchie&author=SJ%20Bielinski&author=MA%20Basford&volume=89&issue=4&publication_year=2011&pages=529-42&pmid=21981779&doi=10.1016/j.ajhg.2011.09.008&)

28. White WB, Turner JR, Sica DA, Bisognano JD, Calhoun DA, Townsend RR, et al. Detection, evaluation, and treatment of severe and resistant hypertension: Proceedings from an American Society of Hypertension Interactive Forum held in Bethesda, MD, USA, October 10th 2013. Journal of the American Society of Hypertension. 2014;8(10):743–57.  [DOI](https://doi.org/10.1016/j.jash.2014.06.005) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25418497/)

29. Myat A, Redwood SR, Qureshi AC, Spertus JA, Williams B. Resistant hypertension. BMJ. 2012;345.  [DOI](https://doi.org/10.1136/bmj.e7473) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23169802/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMJ&title=Resistant%20hypertension&author=A%20Myat&author=SR%20Redwood&author=AC%20Qureshi&author=JA%20Spertus&author=B%20Williams&volume=345&publication_year=2012&pmid=23169802&doi=10.1136/bmj.e7473&)

30. Levey AS, Bosch JP, Lewis JB, Greene T, Rogers N, Roth D. A More Accurate Method To Estimate Glomerular Filtration Rate from Serum Creatinine: A New Prediction Equation. Annals of Internal Medicine. 1999;130(6):461–70.  [DOI](https://doi.org/10.7326/0003-4819-130-6-199903160-00002) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10075613/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Annals%20of%20Internal%20Medicine&title=A%20More%20Accurate%20Method%20To%20Estimate%20Glomerular%20Filtration%20Rate%20from%20Serum%20Creatinine:%20A%20New%20Prediction%20Equation&author=AS%20Levey&author=JP%20Bosch&author=JB%20Lewis&author=T%20Greene&author=N%20Rogers&volume=130&issue=6&publication_year=1999&pages=461-70&pmid=10075613&doi=10.7326/0003-4819-130-6-199903160-00002&)

31. Newton KM, Peissig PL, Kho AN, Bielinski SJ, Berg RL, Choudhary V, et al. Validation of electronic medical record-based phenotyping algorithms: results and lessons learned from the eMERGE network. Journal of the American Medical Informatics Association. 2013;20(e1):e147–e54. 10.1136/amiajnl-2012-000896  [DOI](https://doi.org/10.1136/amiajnl-2012-000896) | [PMC free article](/articles/PMC3715338/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23531748/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20of%20the%20American%20Medical%20Informatics%20Association&title=Validation%20of%20electronic%20medical%20record-based%20phenotyping%20algorithms:%20results%20and%20lessons%20learned%20from%20the%20eMERGE%20network&author=KM%20Newton&author=PL%20Peissig&author=AN%20Kho&author=SJ%20Bielinski&author=RL%20Berg&volume=20&issue=e1&publication_year=2013&pages=e147-e54&pmid=23531748&doi=10.1136/amiajnl-2012-000896&)

32. Kirby JC, Speltz P, Rasmussen LV, Basford M, Gottesman O, Peissig PL, et al. PheKB: a catalog and workflow for creating electronic phenotype algorithms for transportability. Journal of the American Medical Informatics Association. 2016.  [DOI](https://doi.org/10.1093/jamia/ocv202) | [PMC free article](/articles/PMC5070514/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27026615/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20of%20the%20American%20Medical%20Informatics%20Association&title=PheKB:%20a%20catalog%20and%20workflow%20for%20creating%20electronic%20phenotype%20algorithms%20for%20transportability&author=JC%20Kirby&author=P%20Speltz&author=LV%20Rasmussen&author=M%20Basford&author=O%20Gottesman&publication_year=2016&pmid=27026615&doi=10.1093/jamia/ocv202&)

33. Turner S, Armstrong LL, Bradford Y, Carlson CS, Crawford DC, Crenshaw AT, et al. Quality Control Procedures for Genome-Wide Association Studies. Curr Protoc Hum Genet. 2011;68(1.18):1–19. PMC3066182.  [DOI](https://doi.org/10.1002/0471142905.hg0119s68) | [PMC free article](/articles/PMC3066182/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21234875/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Protoc%20Hum%20Genet&title=Quality%20Control%20Procedures%20for%20Genome-Wide%20Association%20Studies&author=S%20Turner&author=LL%20Armstrong&author=Y%20Bradford&author=CS%20Carlson&author=DC%20Crawford&volume=68&issue=1.18&publication_year=2011&pages=1-19&pmid=21234875&doi=10.1002/0471142905.hg0119s68&)

34. Zuvich RL, Armstrong LL, Bielinski SJ, Bradford Y, Carlson CS, Crawford DC, et al. Pitfalls of merging GWAS data: lessons learned in the eMERGE network and quality control procedures to maintain high data quality. Genetic Epidemiology. 2011;35(8):887–98. 10.1002/gepi.20639  [DOI](https://doi.org/10.1002/gepi.20639) | [PMC free article](/articles/PMC3592376/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22125226/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genetic%20Epidemiology&title=Pitfalls%20of%20merging%20GWAS%20data:%20lessons%20learned%20in%20the%20eMERGE%20network%20and%20quality%20control%20procedures%20to%20maintain%20high%20data%20quality&author=RL%20Zuvich&author=LL%20Armstrong&author=SJ%20Bielinski&author=Y%20Bradford&author=CS%20Carlson&volume=35&issue=8&publication_year=2011&pages=887-98&pmid=22125226&doi=10.1002/gepi.20639&)

35. Verma SS, de Andrade M, Tromp G, Kuivaniemi H, Pugh E, Namjou-Khales B, et al. Imputation and quality control steps for combining multiple genome-wide datasets. Frontiers in Genetics. 2014;5:370 10.3389/fgene.2014.00370  [DOI](https://doi.org/10.3389/fgene.2014.00370) | [PMC free article](/articles/PMC4263197/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25566314/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Genetics&title=Imputation%20and%20quality%20control%20steps%20for%20combining%20multiple%20genome-wide%20datasets&author=SS%20Verma&author=M%20de%20Andrade&author=G%20Tromp&author=H%20Kuivaniemi&author=E%20Pugh&volume=5&publication_year=2014&pages=370&pmid=25566314&doi=10.3389/fgene.2014.00370&)

36. Pritchard JK, Stephens M, Donnelly P. Inference of Population Structure Using Multilocus Genotype Data. Genetics. 2000;155(2):945–59.  [DOI](https://doi.org/10.1093/genetics/155.2.945) | [PMC free article](/articles/PMC1461096/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10835412/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genetics&title=Inference%20of%20Population%20Structure%20Using%20Multilocus%20Genotype%20Data&author=JK%20Pritchard&author=M%20Stephens&author=P%20Donnelly&volume=155&issue=2&publication_year=2000&pages=945-59&pmid=10835412&doi=10.1093/genetics/155.2.945&)

37. Price AL, Patterson NJ, Plenge RM, Weinblatt ME, Shadick NA, Reich D. Principal components analysis corrects for stratification in genome-wide association studies. Nat Genet. 2006;38(8):904–9. 10.1038/ng1847  [DOI](https://doi.org/10.1038/ng1847) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16862161/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Principal%20components%20analysis%20corrects%20for%20stratification%20in%20genome-wide%20association%20studies&author=AL%20Price&author=NJ%20Patterson&author=RM%20Plenge&author=ME%20Weinblatt&author=NA%20Shadick&volume=38&issue=8&publication_year=2006&pages=904-9&pmid=16862161&doi=10.1038/ng1847&)

38. Dumitrescu L, Ritchie MD, Brown-Gentry K, Pulley JM, Basford M, Denny JC, et al. Assessing the accuracy of observer-reported ancestry in a biorepository linked to electronic medical records. Genet Med. 2010;12(10):648–50. PMC2952033. 10.1097/GIM.0b013e3181efe2df  [DOI](https://doi.org/10.1097/GIM.0b013e3181efe2df) | [PMC free article](/articles/PMC2952033/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20733501/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genet%20Med&title=Assessing%20the%20accuracy%20of%20observer-reported%20ancestry%20in%20a%20biorepository%20linked%20to%20electronic%20medical%20records&author=L%20Dumitrescu&author=MD%20Ritchie&author=K%20Brown-Gentry&author=JM%20Pulley&author=M%20Basford&volume=12&issue=10&publication_year=2010&pages=648-50&pmid=20733501&doi=10.1097/GIM.0b013e3181efe2df&)

39. Hall JB, Dumitrescu L, Dilks HH, Crawford DC, Bush WS. Accuracy of Administratively-Assigned Ancestry for Diverse Populations in an Electronic Medical Record-Linked Biobank. PLoS ONE. 2014;9(6):e99161 PMC4045967. 10.1371/journal.pone.0099161  [DOI](https://doi.org/10.1371/journal.pone.0099161) | [PMC free article](/articles/PMC4045967/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24896101/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&title=Accuracy%20of%20Administratively-Assigned%20Ancestry%20for%20Diverse%20Populations%20in%20an%20Electronic%20Medical%20Record-Linked%20Biobank&author=JB%20Hall&author=L%20Dumitrescu&author=HH%20Dilks&author=DC%20Crawford&author=WS%20Bush&volume=9&issue=6&publication_year=2014&pages=e99161&pmid=24896101&doi=10.1371/journal.pone.0099161&)

40. An integrated map of genetic variation from 1,092 human genomes. Nature. 2012;491(7422):56–65. 10.1038/nature11632  [DOI](https://doi.org/10.1038/nature11632) | [PMC free article](/articles/PMC3498066/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23128226/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=An%20integrated%20map%20of%20genetic%20variation%20from%201,092%20human%20genomes&volume=491&issue=7422&publication_year=2012&pages=56-65&pmid=23128226&doi=10.1038/nature11632&)

41. Delaneau O, Zagury JF, Marchini J. Improved whole-chromosome phasing for disease and population genetic studies. Nat Meth. 2013;10(1):5–6.  [DOI](https://doi.org/10.1038/nmeth.2307) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23269371/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Meth&title=Improved%20whole-chromosome%20phasing%20for%20disease%20and%20population%20genetic%20studies&author=O%20Delaneau&author=JF%20Zagury&author=J%20Marchini&volume=10&issue=1&publication_year=2013&pages=5-6&pmid=23269371&doi=10.1038/nmeth.2307&)

42. Howie B, Fuchsberger C, Stephens M, Marchini J, Abecasis GR. Fast and accurate genotype imputation in genome-wide association studies through pre-phasing. Nat Genet. 2012;44(8):955–9. 10.1038/ng.2354  [DOI](https://doi.org/10.1038/ng.2354) | [PMC free article](/articles/PMC3696580/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22820512/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Fast%20and%20accurate%20genotype%20imputation%20in%20genome-wide%20association%20studies%20through%20pre-phasing&author=B%20Howie&author=C%20Fuchsberger&author=M%20Stephens&author=J%20Marchini&author=GR%20Abecasis&volume=44&issue=8&publication_year=2012&pages=955-9&pmid=22820512&doi=10.1038/ng.2354&)

43. Purcell S, Neale B, Todd-Brown K, Thomas L, Ferreira MA, Bender D, et al. PLINK: a tool set for whole-genome association and population-based linkage analysis. Am J Hum Genet. 2007;81(3):559–75. 10.1086/519795  [DOI](https://doi.org/10.1086/519795) | [PMC free article](/articles/PMC1950838/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17701901/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=PLINK:%20a%20tool%20set%20for%20whole-genome%20association%20and%20population-based%20linkage%20analysis&author=S%20Purcell&author=B%20Neale&author=K%20Todd-Brown&author=L%20Thomas&author=MA%20Ferreira&volume=81&issue=3&publication_year=2007&pages=559-75&pmid=17701901&doi=10.1086/519795&)

44. Grady BJ, Torstenson E, Dudek SM, Giles J, Sexton D, Ritchie MD. Finding unique filter sets in PLATO: a precursor to efficient interaction analysis in GWAS data. Pac Symp Biocomput. 2010:315–26.  [PMC free article](/articles/PMC2903053/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19908384/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pac%20Symp%20Biocomput&title=Finding%20unique%20filter%20sets%20in%20PLATO:%20a%20precursor%20to%20efficient%20interaction%20analysis%20in%20GWAS%20data&author=BJ%20Grady&author=E%20Torstenson&author=SM%20Dudek&author=J%20Giles&author=D%20Sexton&publication_year=2010&pages=315-26&pmid=19908384&)

45. Pruim RJ, Welch RP, Sanna S, Teslovich TM, Chines PS, Gliedt TP, et al. LocusZoom: regional visualization of genome-wide association scan results. Bioinformatics. 2010;26(18):2336–7. 10.1093/bioinformatics/btq419  [DOI](https://doi.org/10.1093/bioinformatics/btq419) | [PMC free article](/articles/PMC2935401/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20634204/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=LocusZoom:%20regional%20visualization%20of%20genome-wide%20association%20scan%20results&author=RJ%20Pruim&author=RP%20Welch&author=S%20Sanna&author=TM%20Teslovich&author=PS%20Chines&volume=26&issue=18&publication_year=2010&pages=2336-7&pmid=20634204&doi=10.1093/bioinformatics/btq419&)

46. Hindorff LA, Sethupathy P, Junkins HA, Ramos EM, Mehta JP, Collins FS, et al. Potential etiologic and functional implications of genome-wide association loci for human diseases and traits. Proceedings of the National Academy of Sciences. 2009;106(23):9362–7.  [DOI](https://doi.org/10.1073/pnas.0903103106) | [PMC free article](/articles/PMC2687147/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19474294/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%20National%20Academy%20of%20Sciences&title=Potential%20etiologic%20and%20functional%20implications%20of%20genome-wide%20association%20loci%20for%20human%20diseases%20and%20traits&author=LA%20Hindorff&author=P%20Sethupathy&author=HA%20Junkins&author=EM%20Ramos&author=JP%20Mehta&volume=106&issue=23&publication_year=2009&pages=9362-7&pmid=19474294&doi=10.1073/pnas.0903103106&)

47. Welter D, MacArthur J, Morales J, Burdett T, Hall P, Junkins H, et al. The NHGRI GWAS Catalog, a curated resource of SNP-trait associations. Nucleic Acids Research. 2014;42(D1):D1001–D6.  [DOI](https://doi.org/10.1093/nar/gkt1229) | [PMC free article](/articles/PMC3965119/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24316577/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Research&title=The%20NHGRI%20GWAS%20Catalog,%20a%20curated%20resource%20of%20SNP-trait%20associations&author=D%20Welter&author=J%20MacArthur&author=J%20Morales&author=T%20Burdett&author=P%20Hall&volume=42&issue=D1&publication_year=2014&pages=D1001-D6&pmid=24316577&doi=10.1093/nar/gkt1229&)

48. Skol AD, Scott LJ, Abecasis GR, Boehnke M. Joint analysis is more efficient than replication-based analysis for two-stage genome-wide association studies. Nat Genet. 2006;38(2):209–13. 10.1038/ng1706  [DOI](https://doi.org/10.1038/ng1706) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16415888/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Joint%20analysis%20is%20more%20efficient%20than%20replication-based%20analysis%20for%20two-stage%20genome-wide%20association%20studies&author=AD%20Skol&author=LJ%20Scott&author=GR%20Abecasis&author=M%20Boehnke&volume=38&issue=2&publication_year=2006&pages=209-13&pmid=16415888&doi=10.1038/ng1706&)

49. Pepine CJ, Handberg EM, Cooper-DeHoff RM, et al. A calcium antagonist vs a non–calcium antagonist hypertension treatment strategy for patients with coronary artery disease: The international verapamil-trandolapril study (invest): a randomized controlled trial. JAMA. 2003;290(21):2805–16. 10.1001/jama.290.21.2805  [DOI](https://doi.org/10.1001/jama.290.21.2805) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14657064/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA&title=A%20calcium%20antagonist%20vs%20a%20non%E2%80%93calcium%20antagonist%20hypertension%20treatment%20strategy%20for%20patients%20with%20coronary%20artery%20disease:%20The%20international%20verapamil-trandolapril%20study%20(invest):%20a%20randomized%20controlled%20trial&author=CJ%20Pepine&author=EM%20Handberg&author=RM%20Cooper-DeHoff&volume=290&issue=21&publication_year=2003&pages=2805-16&pmid=14657064&doi=10.1001/jama.290.21.2805&)

50. The SPSSG. Blood-pressure targets in patients with recent lacunar stroke: the SPS3 randomised trial. The Lancet. 382(9891):507–15.  [DOI](https://doi.org/10.1016/S0140-6736(13)60852-1) | [PMC free article](/articles/PMC3979302/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23726159/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=The%20Lancet&title=Blood-pressure%20targets%20in%20patients%20with%20recent%20lacunar%20stroke:%20the%20SPS3%20randomised%20trial&volume=382&issue=9891&pages=507-15&pmid=23726159&doi=10.1016/S0140-6736(13)60852-1&)

51. Kato N, Takeuchi F, Tabara Y, Kelly TN, Go MJ, Sim X, et al. Meta-analysis of genome-wide association studies identifies common variants associated with blood pressure variation in east Asians. Nat Genet. 2011;43(6):531–8. 10.1038/ng.834  [DOI](https://doi.org/10.1038/ng.834) | [PMC free article](/articles/PMC3158568/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21572416/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Meta-analysis%20of%20genome-wide%20association%20studies%20identifies%20common%20variants%20associated%20with%20blood%20pressure%20variation%20in%20east%20Asians&author=N%20Kato&author=F%20Takeuchi&author=Y%20Tabara&author=TN%20Kelly&author=MJ%20Go&volume=43&issue=6&publication_year=2011&pages=531-8&pmid=21572416&doi=10.1038/ng.834&)

52. Levy D, Ehret GB, Rice K, Verwoert GC, Launer LJ, Dehghan A, et al. Genome-wide association study of blood pressure and hypertension. Nat Genet. 2009;41(6):677–87. 10.1038/ng.384  [DOI](https://doi.org/10.1038/ng.384) | [PMC free article](/articles/PMC2998712/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19430479/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Genome-wide%20association%20study%20of%20blood%20pressure%20and%20hypertension&author=D%20Levy&author=GB%20Ehret&author=K%20Rice&author=GC%20Verwoert&author=LJ%20Launer&volume=41&issue=6&publication_year=2009&pages=677-87&pmid=19430479&doi=10.1038/ng.384&)

53. Franceschini N, Fox E, Zhang Z, Edwards T-á, Nalls M-á, Sung Y-á, et al. Genome-wide Association Analysis of Blood-Pressure Traits in African-Ancestry Individuals Reveals Common Associated Genes in African and Non-African Populations. The American Journal of Human Genetics. 2013;93(3):545–54. PMC3769920. 10.1016/j.ajhg.2013.07.010  [DOI](https://doi.org/10.1016/j.ajhg.2013.07.010) | [PMC free article](/articles/PMC3769920/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23972371/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=The%20American%20Journal%20of%20Human%20Genetics&title=Genome-wide%20Association%20Analysis%20of%20Blood-Pressure%20Traits%20in%20African-Ancestry%20Individuals%20Reveals%20Common%20Associated%20Genes%20in%20African%20and%20Non-African%20Populations&author=N%20Franceschini&author=E%20Fox&author=Z%20Zhang&author=T-%C3%A1%20Edwards&author=M-%C3%A1%20Nalls&volume=93&issue=3&publication_year=2013&pages=545-54&pmid=23972371&doi=10.1016/j.ajhg.2013.07.010&)

54. Lu X, Wang L, Lin X, Huang J, Charles Gu C, He M, et al. Genome-wide association study in Chinese identifies novel loci for blood pressure and hypertension. Human Molecular Genetics. 2015;24(3):865–74. 10.1093/hmg/ddu478  [DOI](https://doi.org/10.1093/hmg/ddu478) | [PMC free article](/articles/PMC4303798/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25249183/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Human%20Molecular%20Genetics&title=Genome-wide%20association%20study%20in%20Chinese%20identifies%20novel%20loci%20for%20blood%20pressure%20and%20hypertension&author=X%20Lu&author=L%20Wang&author=X%20Lin&author=J%20Huang&author=C%20Charles%20Gu&volume=24&issue=3&publication_year=2015&pages=865-74&pmid=25249183&doi=10.1093/hmg/ddu478&)

55. Sung YJ, de Las Fuentes L, Schwander KL, Simino J, Rao DC. Gene-smoking interactions identify several novel blood pressure loci in the Framingham Heart Study. Am J Hypertens. 2015;28(3):343–54. PMC4402348. 10.1093/ajh/hpu149  [DOI](https://doi.org/10.1093/ajh/hpu149) | [PMC free article](/articles/PMC4402348/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25189868/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hypertens&title=Gene-smoking%20interactions%20identify%20several%20novel%20blood%20pressure%20loci%20in%20the%20Framingham%20Heart%20Study&author=YJ%20Sung&author=L%20de%20Las%20Fuentes&author=KL%20Schwander&author=J%20Simino&author=DC%20Rao&volume=28&issue=3&publication_year=2015&pages=343-54&pmid=25189868&doi=10.1093/ajh/hpu149&)

56. Simino J, Shi G, Bis Joshua C, Chasman Daniel I, Ehret Georg B, Gu X, et al. Gene-Age Interactions in Blood Pressure Regulation: A Large-Scale Investigation with the CHARGE, Global BPgen, and ICBP Consortia. The American Journal of Human Genetics. 2014;95(1):24–38. PMC4085636. 10.1016/j.ajhg.2014.05.010  [DOI](https://doi.org/10.1016/j.ajhg.2014.05.010) | [PMC free article](/articles/PMC4085636/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24954895/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=The%20American%20Journal%20of%20Human%20Genetics&title=Gene-Age%20Interactions%20in%20Blood%20Pressure%20Regulation:%20A%20Large-Scale%20Investigation%20with%20the%20CHARGE,%20Global%20BPgen,%20and%20ICBP%20Consortia&author=J%20Simino&author=G%20Shi&author=Joshua%20C%20Bis&author=Daniel%20I%20Chasman&author=Georg%20B%20Ehret&volume=95&issue=1&publication_year=2014&pages=24-38&pmid=24954895&doi=10.1016/j.ajhg.2014.05.010&)

57. Kim YK, Kim Y, Hwang MY, Shimokawa K, Won S, Kato N, et al. Identification of a genetic variant at 2q12.1 associated with blood pressure in East-Asians by genome-wide scan including gene-environment interactions. BMC Medical Genetics. 2014;15(1):1–8. PMC4059884.  [DOI](https://doi.org/10.1186/1471-2350-15-65) | [PMC free article](/articles/PMC4059884/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24903457/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Medical%20Genetics&title=Identification%20of%20a%20genetic%20variant%20at%202q12.1%20associated%20with%20blood%20pressure%20in%20East-Asians%20by%20genome-wide%20scan%20including%20gene-environment%20interactions&author=YK%20Kim&author=Y%20Kim&author=MY%20Hwang&author=K%20Shimokawa&author=S%20Won&volume=15&issue=1&publication_year=2014&pages=1-8&pmid=24903457&doi=10.1186/1471-2350-15-65&)

58. He J, Kelly TN, Zhao Q, Li H, Huang J, Wang L, et al. Genome-Wide Association Study Identifies 8 Novel Loci Associated With Blood Pressure Responses to Interventions in Han Chinese. Circulation: Cardiovascular Genetics. 2013;6(6):598–607. PMC3952335. 10.1161/CIRCGENETICS.113.000307  [DOI](https://doi.org/10.1161/CIRCGENETICS.113.000307) | [PMC free article](/articles/PMC3952335/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24165912/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cardiovascular%20Genetics&title=Genome-Wide%20Association%20Study%20Identifies%208%20Novel%20Loci%20Associated%20With%20Blood%20Pressure%20Responses%20to%20Interventions%20in%20Han%20Chinese.%20Circulation&author=J%20He&author=TN%20Kelly&author=Q%20Zhao&author=H%20Li&author=J%20Huang&volume=6&issue=6&publication_year=2013&pages=598-607&pmid=24165912&doi=10.1161/CIRCGENETICS.113.000307&)

59. Hong K-W, Kim SS, Kim Y. Genome-Wide Association Study of Orthostatic Hypotension and Supine-Standing Blood Pressure Changes in Two Korean Populations. Genomics Inform. 2013;11(3):129–34. PMC3794085. 10.5808/GI.2013.11.3.129  [DOI](https://doi.org/10.5808/GI.2013.11.3.129) | [PMC free article](/articles/PMC3794085/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24124408/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genomics%20Inform&title=Genome-Wide%20Association%20Study%20of%20Orthostatic%20Hypotension%20and%20Supine-Standing%20Blood%20Pressure%20Changes%20in%20Two%20Korean%20Populations&author=K-W%20Hong&author=SS%20Kim&author=Y%20Kim&volume=11&issue=3&publication_year=2013&pages=129-34&pmid=24124408&doi=10.5808/GI.2013.11.3.129&)

60. Kelly TN, Takeuchi F, Tabara Y, Edwards TL, Kim YJ, Chen P, et al. Genome-Wide Association Study Meta-Analysis Reveals Transethnic Replication of Mean Arterial and Pulse Pressure Loci. Hypertension. 2013;62(5):853–9. PMC3972802. 10.1161/HYPERTENSIONAHA.113.01148  [DOI](https://doi.org/10.1161/HYPERTENSIONAHA.113.01148) | [PMC free article](/articles/PMC3972802/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24001895/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hypertension&title=Genome-Wide%20Association%20Study%20Meta-Analysis%20Reveals%20Transethnic%20Replication%20of%20Mean%20Arterial%20and%20Pulse%20Pressure%20Loci&author=TN%20Kelly&author=F%20Takeuchi&author=Y%20Tabara&author=TL%20Edwards&author=YJ%20Kim&volume=62&issue=5&publication_year=2013&pages=853-9&pmid=24001895&doi=10.1161/HYPERTENSIONAHA.113.01148&)

61. Melka MG, Bernard M, Mahboubi A, Abrahamowicz M, Paterson AD, Syme C, et al. Genome-wide scan for loci of adolescent obseity and their relationship with blood pressure. J Clin Endocrinol Metab. 2012;97(1):E145–50. 10.1210/jc.2011-1801  [DOI](https://doi.org/10.1210/jc.2011-1801) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22013104/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Endocrinol%20Metab&title=Genome-wide%20scan%20for%20loci%20of%20adolescent%20obseity%20and%20their%20relationship%20with%20blood%20pressure&author=MG%20Melka&author=M%20Bernard&author=A%20Mahboubi&author=M%20Abrahamowicz&author=AD%20Paterson&volume=97&issue=1&publication_year=2012&pages=E145-50&pmid=22013104&doi=10.1210/jc.2011-1801&)

62. Ehret GB, Munroe PB, Rice KM, Bochud M, Johnson AD, Chasman DI, et al. Genetic variants in novel pathways influence blood pressure and cardiovascular disease risk. Nature. 2011;478.  [DOI](https://doi.org/10.1038/nature10405) | [PMC free article](/articles/PMC3340926/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21909115/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Genetic%20variants%20in%20novel%20pathways%20influence%20blood%20pressure%20and%20cardiovascular%20disease%20risk&author=GB%20Ehret&author=PB%20Munroe&author=KM%20Rice&author=M%20Bochud&author=AD%20Johnson&publication_year=2011&pages=478&pmid=21909115&doi=10.1038/nature10405&)

63. Wain LV, Verwoert GC, O'Reilly PF, Shi G, Johnson T, Johnson AD, et al. Genome-wide association study identifies six new loci influencing pulse pressure and mean arterial pressure. Nat Genet. 2011;43(10):1005–11. PMC3445021. 10.1038/ng.922  [DOI](https://doi.org/10.1038/ng.922) | [PMC free article](/articles/PMC3445021/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21909110/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Genome-wide%20association%20study%20identifies%20six%20new%20loci%20influencing%20pulse%20pressure%20and%20mean%20arterial%20pressure&author=LV%20Wain&author=GC%20Verwoert&author=PF%20O'Reilly&author=G%20Shi&author=T%20Johnson&volume=43&issue=10&publication_year=2011&pages=1005-11&pmid=21909110&doi=10.1038/ng.922&)

64. Blech I, Katzenellenbogen M, Katzenellenbogen A, Wainstein J, Rubinstein A, Harman-Boehm I, et al. Predicting Diabetic Nephropathy Using a Multifactorial Genetic Model. PLoS ONE. 2011;6(4):e18743 10.1371/journal.pone.0018743  [DOI](https://doi.org/10.1371/journal.pone.0018743) | [PMC free article](/articles/PMC3077408/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21533139/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&title=Predicting%20Diabetic%20Nephropathy%20Using%20a%20Multifactorial%20Genetic%20Model&author=I%20Blech&author=M%20Katzenellenbogen&author=A%20Katzenellenbogen&author=J%20Wainstein&author=A%20Rubinstein&volume=6&issue=4&publication_year=2011&pages=e18743&pmid=21533139&doi=10.1371/journal.pone.0018743&)

65. Slavin TP, Feng T, Schnell A, Zhu X, Elston RC. Two-marker association tests yield new disease associations for coronary artery disease and hypertension. Human Genetics. 2011;130(6):725–33. PMC3576836. 10.1007/s00439-011-1009-6  [DOI](https://doi.org/10.1007/s00439-011-1009-6) | [PMC free article](/articles/PMC3576836/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21626137/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Human%20Genetics&title=Two-marker%20association%20tests%20yield%20new%20disease%20associations%20for%20coronary%20artery%20disease%20and%20hypertension&author=TP%20Slavin&author=T%20Feng&author=A%20Schnell&author=X%20Zhu&author=RC%20Elston&volume=130&issue=6&publication_year=2011&pages=725-33&pmid=21626137&doi=10.1007/s00439-011-1009-6&)

66. Kraja AT, Vaidya D, Pankow JS, Goodarzi MO, Assimes TL, Kullo IJ, et al. A Bivariate Genome-Wide Approach to Metabolic Syndrome. STAMPEED Consortium. 2011;60(4):1329–39. PMC3064107.  [DOI](https://doi.org/10.2337/db10-1011) | [PMC free article](/articles/PMC3064107/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21386085/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=STAMPEED%20Consortium&title=A%20Bivariate%20Genome-Wide%20Approach%20to%20Metabolic%20Syndrome&author=AT%20Kraja&author=D%20Vaidya&author=JS%20Pankow&author=MO%20Goodarzi&author=TL%20Assimes&volume=60&issue=4&publication_year=2011&pages=1329-39&pmid=21386085&doi=10.2337/db10-1011&)

67. Lettre G, Palmer CD, Young T, Ejebe KG, Allayee H, Benjamin EJ, et al. Genome-Wide Association Study of Coronary Heart Disease and Its Risk Factors in 8,090 African Americans: The NHLBI CARe Project. PLoS Genet. 2011;7(2):e1001300 10.1371/journal.pgen.1001300  [DOI](https://doi.org/10.1371/journal.pgen.1001300) | [PMC free article](/articles/PMC3037413/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21347282/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet&title=Genome-Wide%20Association%20Study%20of%20Coronary%20Heart%20Disease%20and%20Its%20Risk%20Factors%20in%208,090%20African%20Americans:%20The%20NHLBI%20CARe%20Project&author=G%20Lettre&author=CD%20Palmer&author=T%20Young&author=KG%20Ejebe&author=H%20Allayee&volume=7&issue=2&publication_year=2011&pages=e1001300&pmid=21347282&doi=10.1371/journal.pgen.1001300&)

68. Padmanabhan S, Melander O, Johnson T, Di Blasio AM, Lee WK, Gentilini D, et al. Genome-Wide Association Study of Blood Pressure Extremes Identifies Variant near UMOD Associated with Hypertension. PLoS Genet. 2010;6(10):e1001177 PMC2965757. 10.1371/journal.pgen.1001177  [DOI](https://doi.org/10.1371/journal.pgen.1001177) | [PMC free article](/articles/PMC2965757/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21082022/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet&title=Genome-Wide%20Association%20Study%20of%20Blood%20Pressure%20Extremes%20Identifies%20Variant%20near%20UMOD%20Associated%20with%20Hypertension&author=S%20Padmanabhan&author=O%20Melander&author=T%20Johnson&author=AM%20Di%20Blasio&author=WK%20Lee&volume=6&issue=10&publication_year=2010&pages=e1001177&pmid=21082022&doi=10.1371/journal.pgen.1001177&)

69. Newton-Cheh C, Johnson T, Gateva V, Tobin MD, Bochud M, Coin L, et al. Genome-wide association study identifies eight loci associated with blood pressure. Nat Genet. 2009;41. PMC2891673.  [DOI](https://doi.org/10.1038/ng.361) | [PMC free article](/articles/PMC2891673/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19430483/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Genome-wide%20association%20study%20identifies%20eight%20loci%20associated%20with%20blood%20pressure&author=C%20Newton-Cheh&author=T%20Johnson&author=V%20Gateva&author=MD%20Tobin&author=M%20Bochud&volume=41&publication_year=2009&pmid=19430483&doi=10.1038/ng.361&)

70. Consortium GP. A global reference for human genetic variation. Nature. 2015;526(7571):68–74. 10.1038/nature15393  [DOI](https://doi.org/10.1038/nature15393) | [PMC free article](/articles/PMC4750478/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26432245/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=A%20global%20reference%20for%20human%20genetic%20variation&author=GP%20Consortium&volume=526&issue=7571&publication_year=2015&pages=68-74&pmid=26432245&doi=10.1038/nature15393&)

71. Kim SJ, Lee SK, Kim SH, Yun C-H, Kim JH, Thomas RJ, et al. Genetic Association of Short Sleep Duration With Hypertension Incidence—A 6-Year Follow-up in the Korean Genome and Epidemiology Study. Circulation Journal. 2012;76(4):907–13.  [DOI](https://doi.org/10.1253/circj.cj-11-0713) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22322875/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circulation%20Journal&title=Genetic%20Association%20of%20Short%20Sleep%20Duration%20With%20Hypertension%20Incidence%E2%80%94A%206-Year%20Follow-up%20in%20the%20Korean%20Genome%20and%20Epidemiology%20Study&author=SJ%20Kim&author=SK%20Lee&author=SH%20Kim&author=C-H%20Yun&author=JH%20Kim&volume=76&issue=4&publication_year=2012&pages=907-13&pmid=22322875&doi=10.1253/circj.cj-11-0713&)

72. Crawford DC, Crosslin DR, Tromp G, Kullo IJ, Kuivaniemi H, Hayes MG, et al. eMERGEing progress in genomics—the first seven years. Frontiers in Genetics. 2014;5:184 PMC4060012. 10.3389/fgene.2014.00184  [DOI](https://doi.org/10.3389/fgene.2014.00184) | [PMC free article](/articles/PMC4060012/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24987407/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Genetics&title=eMERGEing%20progress%20in%20genomics%E2%80%94the%20first%20seven%20years&author=DC%20Crawford&author=DR%20Crosslin&author=G%20Tromp&author=IJ%20Kullo&author=H%20Kuivaniemi&volume=5&publication_year=2014&pages=184&pmid=24987407&doi=10.3389/fgene.2014.00184&)

73. Keating BJ, Tischfield S, Murray SS, Bhangale T, Price TS, Glessner JT, et al. Concept, Design and Implementation of a Cardiovascular Gene-Centric 50 K SNP Array for Large-Scale Genomic Association Studies. PLoS ONE. 2008;3(10):e3583 10.1371/journal.pone.0003583  [DOI](https://doi.org/10.1371/journal.pone.0003583) | [PMC free article](/articles/PMC2571995/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18974833/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&title=Concept,%20Design%20and%20Implementation%20of%20a%20Cardiovascular%20Gene-Centric%2050%20K%20SNP%20Array%20for%20Large-Scale%20Genomic%20Association%20Studies&author=BJ%20Keating&author=S%20Tischfield&author=SS%20Murray&author=T%20Bhangale&author=TS%20Price&volume=3&issue=10&publication_year=2008&pages=e3583&pmid=18974833&doi=10.1371/journal.pone.0003583&)

74. Daly AK. Using Genome-Wide Association Studies to Identify Genes Important in Serious Adverse Drug Reactions. Annual Review of Pharmacology and Toxicology. 2012;52(1):21–35.  [DOI](https://doi.org/10.1146/annurev-pharmtox-010611-134743) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21819236/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Annual%20Review%20of%20Pharmacology%20and%20Toxicology&title=Using%20Genome-Wide%20Association%20Studies%20to%20Identify%20Genes%20Important%20in%20Serious%20Adverse%20Drug%20Reactions&author=AK%20Daly&volume=52&issue=1&publication_year=2012&pages=21-35&pmid=21819236&doi=10.1146/annurev-pharmtox-010611-134743&)

75. Voight BF, Scott LJ, Steinthorsdottir V, Morris AP, Dina C, Welch RP, et al. Twelve type 2 diabetes susceptibility loci identified through large-scale association analysis. Nat Genet. 2010;42(7):579–89. 10.1038/ng.609  [DOI](https://doi.org/10.1038/ng.609) | [PMC free article](/articles/PMC3080658/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20581827/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Twelve%20type%202%20diabetes%20susceptibility%20loci%20identified%20through%20large-scale%20association%20analysis&author=BF%20Voight&author=LJ%20Scott&author=V%20Steinthorsdottir&author=AP%20Morris&author=C%20Dina&volume=42&issue=7&publication_year=2010&pages=579-89&pmid=20581827&doi=10.1038/ng.609&)

76. Consortium DGRAM-aD, Type AGEN, South AT, Mexican AT, (DGEbN-gsim-ES, Mahajan A, et al. Genome-wide trans-ancestry meta-analysis provides insight into the genetic architecture of type 2 diabetes susceptibility. Nat Genet. 2014;46(3):234–44. 10.1038/ng.2897  [DOI](https://doi.org/10.1038/ng.2897) | [PMC free article](/articles/PMC3969612/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24509480/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Genome-wide%20trans-ancestry%20meta-analysis%20provides%20insight%20into%20the%20genetic%20architecture%20of%20type%202%20diabetes%20susceptibility&author=A%20Mahajan&volume=46&issue=3&publication_year=2014&pages=234-44&pmid=24509480&doi=10.1038/ng.2897&)

77. Teslovich TM, Musunuru K, Smith AV, Edmondson AC, Stylianou IM, Koseki M, et al. Biological, clinical and population relevance of 95 loci for blood lipids. Nature. 2010;466(7307):707–13. 10.1038/nature09270  [DOI](https://doi.org/10.1038/nature09270) | [PMC free article](/articles/PMC3039276/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20686565/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Biological,%20clinical%20and%20population%20relevance%20of%2095%20loci%20for%20blood%20lipids&author=TM%20Teslovich&author=K%20Musunuru&author=AV%20Smith&author=AC%20Edmondson&author=IM%20Stylianou&volume=466&issue=7307&publication_year=2010&pages=707-13&pmid=20686565&doi=10.1038/nature09270&)

78. Global Lipids Genetics C. Discovery and refinement of loci associated with lipid levels. Nat Genet. 2013;45(11):1274–83. http://www.nature.com/ng/journal/v45/n11/abs/ng.2797.html#supplementary-information. 10.1038/ng.2797  [http://www.nature.com/ng/journal/v45/n11/abs/ng.2797.html#supplementary-information](http://www.nature.com/ng/journal/v45/n11/abs/ng.2797.html#supplementary-information) | [DOI](https://doi.org/10.1038/ng.2797) | [PMC free article](/articles/PMC3838666/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24097068/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Discovery%20and%20refinement%20of%20loci%20associated%20with%20lipid%20levels&author=C%20Global%20Lipids%20Genetics&volume=45&issue=11&publication_year=2013&pages=1274-83&pmid=24097068&doi=10.1038/ng.2797&)

79. Salfati E, Morrison AC, Boerwinkle E, Chakravarti A. Direct Estimates of the Genomic Contributions to Blood Pressure Heritability within a Population-Based Cohort (ARIC). PLoS ONE. 2015;10(7):e0133031 10.1371/journal.pone.0133031  [DOI](https://doi.org/10.1371/journal.pone.0133031) | [PMC free article](/articles/PMC4498745/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26162070/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&title=Direct%20Estimates%20of%20the%20Genomic%20Contributions%20to%20Blood%20Pressure%20Heritability%20within%20a%20Population-Based%20Cohort%20(ARIC)&author=E%20Salfati&author=AC%20Morrison&author=E%20Boerwinkle&author=A%20Chakravarti&volume=10&issue=7&publication_year=2015&pages=e0133031&pmid=26162070&doi=10.1371/journal.pone.0133031&)

80. Collins FS, Varmus H. A New Initiative on Precision Medicine. New England Journal of Medicine. 2015;372(9):793–5. 10.1056/NEJMp1500523  [DOI](https://doi.org/10.1056/NEJMp1500523) | [PMC free article](/articles/PMC5101938/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25635347/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=New%20England%20Journal%20of%20Medicine&title=A%20New%20Initiative%20on%20Precision%20Medicine&author=FS%20Collins&author=H%20Varmus&volume=372&issue=9&publication_year=2015&pages=793-5&pmid=25635347&doi=10.1056/NEJMp1500523&)

81. Group PMIPW. The Precision Medicine Initiative Cohort Program—Building a Research Foundation for 21st Century Medicine. 2015 9/17/2015. Report No.

82. Banda Y, Kvale MN, Hoffmann TJ, Hesselson SE, Ranatunga D, Tang H, et al. Characterizing Race/Ethnicity and Genetic Ancestry for 100,000 Subjects in the Genetic Epidemiology Research on Adult Health and Aging (GERA) Cohort. Genetics. 2015;200(4):1285–95. 10.1534/genetics.115.178616  [DOI](https://doi.org/10.1534/genetics.115.178616) | [PMC free article](/articles/PMC4574246/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26092716/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genetics&title=Characterizing%20Race/Ethnicity%20and%20Genetic%20Ancestry%20for%20100,000%20Subjects%20in%20the%20Genetic%20Epidemiology%20Research%20on%20Adult%20Health%20and%20Aging%20(GERA)%20Cohort&author=Y%20Banda&author=MN%20Kvale&author=TJ%20Hoffmann&author=SE%20Hesselson&author=D%20Ranatunga&volume=200&issue=4&publication_year=2015&pages=1285-95&pmid=26092716&doi=10.1534/genetics.115.178616&)

83. Kvale MN, Hesselson S, Hoffmann TJ, Cao Y, Chan D, Connell S, et al. Genotyping Informatics and Quality Control for 100,000 Subjects in the Genetic Epidemiology Research on Adult Health and Aging (GERA) Cohort. Genetics. 2015;200(4):1051–60. 10.1534/genetics.115.178905  [DOI](https://doi.org/10.1534/genetics.115.178905) | [PMC free article](/articles/PMC4574249/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26092718/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genetics&title=Genotyping%20Informatics%20and%20Quality%20Control%20for%20100,000%20Subjects%20in%20the%20Genetic%20Epidemiology%20Research%20on%20Adult%20Health%20and%20Aging%20(GERA)%20Cohort&author=MN%20Kvale&author=S%20Hesselson&author=TJ%20Hoffmann&author=Y%20Cao&author=D%20Chan&volume=200&issue=4&publication_year=2015&pages=1051-60&pmid=26092718&doi=10.1534/genetics.115.178905&)

84. Gaziano JM, Concato J, Brophy M, Fiore L, Pyarajan S, Breeling J, et al. Million Veteran Program: A mega-biobank to study genetic influences on health and disease. Journal of Clinical Epidemiology. 2016.  [DOI](https://doi.org/10.1016/j.jclinepi.2015.09.016) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26441289/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Clinical%20Epidemiology&title=Million%20Veteran%20Program:%20A%20mega-biobank%20to%20study%20genetic%20influences%20on%20health%20and%20disease&author=JM%20Gaziano&author=J%20Concato&author=M%20Brophy&author=L%20Fiore&author=S%20Pyarajan&publication_year=2016&pmid=26441289&doi=10.1016/j.jclinepi.2015.09.016&)

85. Roden DM, Xu H, Denny JC, Wilke RA. Electronic Medical Records as a Tool in Clinical Pharmacology: Opportunities and Challenges. Clinical Pharmacology & Therapeutics. 2012;91(6):1083–6.  [DOI](https://doi.org/10.1038/clpt.2012.42) | [PMC free article](/articles/PMC3819803/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22534870/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clinical%20Pharmacology%20&%20Therapeutics&title=Electronic%20Medical%20Records%20as%20a%20Tool%20in%20Clinical%20Pharmacology:%20Opportunities%20and%20Challenges&author=DM%20Roden&author=H%20Xu&author=JC%20Denny&author=RA%20Wilke&volume=91&issue=6&publication_year=2012&pages=1083-6&pmid=22534870&doi=10.1038/clpt.2012.42&)

86. Ong KL, Cheung BMY, Man YB, Lau CP, Lam KSL. Prevalence, Awareness, Treatment, and Control of Hypertension Among United States Adults 1999–2004. Hypertension. 2007;49(1):69–75. 10.1161/01.HYP.0000252676.46043.18  [DOI](https://doi.org/10.1161/01.HYP.0000252676.46043.18) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17159087/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hypertension&title=Prevalence,%20Awareness,%20Treatment,%20and%20Control%20of%20Hypertension%20Among%20United%20States%20Adults%201999%E2%80%932004&author=KL%20Ong&author=BMY%20Cheung&author=YB%20Man&author=CP%20Lau&author=KSL%20Lam&volume=49&issue=1&publication_year=2007&pages=69-75&pmid=17159087&doi=10.1161/01.HYP.0000252676.46043.18&)

87. Cutler JA, Sorlie PD, Wolz M, Thom T, Fields LE, Roccella EJ. Trends in Hypertension Prevalence, Awareness, Treatment, and Control Rates in United States Adults Between 1988–1994 and 1999–2004. Hypertension. 2008;52(5):818–27. 10.1161/HYPERTENSIONAHA.108.113357  [DOI](https://doi.org/10.1161/HYPERTENSIONAHA.108.113357) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18852389/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hypertension&title=Trends%20in%20Hypertension%20Prevalence,%20Awareness,%20Treatment,%20and%20Control%20Rates%20in%20United%20States%20Adults%20Between%201988%E2%80%931994%20and%201999%E2%80%932004&author=JA%20Cutler&author=PD%20Sorlie&author=M%20Wolz&author=T%20Thom&author=LE%20Fields&volume=52&issue=5&publication_year=2008&pages=818-27&pmid=18852389&doi=10.1161/HYPERTENSIONAHA.108.113357&)

88. Caro JJ, Speckman JL, Salas M, Raggio G, Jackson JD. Effect of initial drug choice on persistence with antihypertensive therapy: the importance of actual practice data. Canadian Medical Association Journal. 1999;160(1):41–6.  [PMC free article](/articles/PMC1229944/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9934342/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Canadian%20Medical%20Association%20Journal&title=Effect%20of%20initial%20drug%20choice%20on%20persistence%20with%20antihypertensive%20therapy:%20the%20importance%20of%20actual%20practice%20data&author=JJ%20Caro&author=JL%20Speckman&author=M%20Salas&author=G%20Raggio&author=JD%20Jackson&volume=160&issue=1&publication_year=1999&pages=41-6&pmid=9934342&)

89. Mazzaglia G, Mantovani LG, Sturkenboom MCJM, Filippi A, Trifiro G, Cricelli C, et al. Patterns of persistence with antihypertensive medications in newly diagnosed hypertensive patients in Italy: a retrospective cohort study in primary care. Journal of Hypertension. 2005;23(11):2093–100.  [DOI](https://doi.org/10.1097/01.hjh.0000186832.41125.8a) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16208153/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Hypertension&title=Patterns%20of%20persistence%20with%20antihypertensive%20medications%20in%20newly%20diagnosed%20hypertensive%20patients%20in%20Italy:%20a%20retrospective%20cohort%20study%20in%20primary%20care&author=G%20Mazzaglia&author=LG%20Mantovani&author=MCJM%20Sturkenboom&author=A%20Filippi&author=G%20Trifiro&volume=23&issue=11&publication_year=2005&pages=2093-100&pmid=16208153&doi=10.1097/01.hjh.0000186832.41125.8a&)

90. Payne KA, Esomonde-White S. Observational studies of anti-hypertensive medication use and compliance: is drug choice a factor in treatment adherence? Curr Hypertens Rep. 2000;2(6):515–24.  [DOI](https://doi.org/10.1007/s11906-996-0035-6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11062596/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Hypertens%20Rep&title=Observational%20studies%20of%20anti-hypertensive%20medication%20use%20and%20compliance:%20is%20drug%20choice%20a%20factor%20in%20treatment%20adherence?&author=KA%20Payne&author=S%20Esomonde-White&volume=2&issue=6&publication_year=2000&pages=515-24&pmid=11062596&doi=10.1007/s11906-996-0035-6&)

91. Rolnick SJ, Pawloski PA, Hedblom BD, Asche SE, Bruzek RJ. Patient Characteristics Associated with Medication Adherence. Clinical Medicine & Research. 2013;11(2):54–65.  [DOI](https://doi.org/10.3121/cmr.2013.1113) | [PMC free article](/articles/PMC3692389/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23580788/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clinical%20Medicine%20&%20Research&title=Patient%20Characteristics%20Associated%20with%20Medication%20Adherence&author=SJ%20Rolnick&author=PA%20Pawloski&author=BD%20Hedblom&author=SE%20Asche&author=RJ%20Bruzek&volume=11&issue=2&publication_year=2013&pages=54-65&pmid=23580788&doi=10.3121/cmr.2013.1113&)

92. Friedman O, McAlister FA, Yun L, Campbell NRC, Tu K. Antihypertensive Drug Persistence and Compliance Among Newly Treated Elderly Hypertensives in Ontario. The American Journal of Medicine. 2010;123(2):173–81. 10.1016/j.amjmed.2009.08.008  [DOI](https://doi.org/10.1016/j.amjmed.2009.08.008) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20103027/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=The%20American%20Journal%20of%20Medicine&title=Antihypertensive%20Drug%20Persistence%20and%20Compliance%20Among%20Newly%20Treated%20Elderly%20Hypertensives%20in%20Ontario&author=O%20Friedman&author=FA%20McAlister&author=L%20Yun&author=NRC%20Campbell&author=K%20Tu&volume=123&issue=2&publication_year=2010&pages=173-81&pmid=20103027&doi=10.1016/j.amjmed.2009.08.008&)

93. Salvi E, Wang Z, Rizzi F, Gong Y, McDonough CW, Padmanabhan S, et al. Genome-Wide and Gene-Based Meta-Analyses Identify Novel Loci Influencing Blood Pressure Response to Hydrochlorothiazide. Hypertension. 2017;69(1):51–9. 10.1161/HYPERTENSIONAHA.116.08267  [DOI](https://doi.org/10.1161/HYPERTENSIONAHA.116.08267) | [PMC free article](/articles/PMC5145728/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27802415/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hypertension&title=Genome-Wide%20and%20Gene-Based%20Meta-Analyses%20Identify%20Novel%20Loci%20Influencing%20Blood%20Pressure%20Response%20to%20Hydrochlorothiazide&author=E%20Salvi&author=Z%20Wang&author=F%20Rizzi&author=Y%20Gong&author=CW%20McDonough&volume=69&issue=1&publication_year=2017&pages=51-9&pmid=27802415&doi=10.1161/HYPERTENSIONAHA.116.08267&)
