# Genome-wide association and replication study of anti-tuberculosis drugs-induced liver toxicity

## Metadata
**Authors:** Zelalem Petros, Ming-Ta Michael Lee, Atsushi Takahashi, Yanfei Zhang, Getnet Yimer, Abiy Habtewold, Wondwossen Amogne, Getachew Aderaye, Ina Schuppe-Koistinen, Taisei Mushiroda, Eyasu Makonnen, Michiaki Kubo, Eleni Aklillu
**Journal:** BMC Genomics
**Date:** 2016 Sep 26
**DOI:** [10.1186/s12864-016-3078-3](https://doi.org/10.1186/s12864-016-3078-3)
**PMID:** 27671213
**PMCID:** PMC5037629
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5037629/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC5037629/pdf/12864_2016_Article_3078.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC5037629/pdf/12864_2016_Article_3078.pdf)

## Abstract

**Background:** 
Drug-induced liver injury (DILI) is a well-recognized adverse event of anti tuberculosis drugs (ATD) possibly associated with genetic variations. The objective of this study was to perform genome-wide association study (GWAS) to identify genetic variants associated with the risk for ATD induced liver toxicity in Ethiopian patients.

**Result:** 
Treatment-naïve newly diagnosed tuberculosis patients (n = 646) were enrolled prospectively and treated with rifampicin based short course anti-tuberculosis therapy. Whole genome genotyping was done using Illumina Omni Express Exome Bead Chip genotyping array with 951,117 single nucleotide polymorphisms (SNPs) on 48 DILI cases and 354 ATD tolerants. Replication study was carried out for 50 SNPs with the lowest P-values (top SNPs) using an independent cohort consisting of 27 DILI cases and 217 ATD tolerants. In the combined analysis, the top SNP identified was rs10946737 (P = 4.4 × 10−6, OR = 3.4, 95 % confidence interval = 2.2–5.3) in the intron of FAM65B in chromosome 6. In addition, we identified a cluster of SNPs with suggestive genome-wide significance in the intron of ATP/GTP binding protein-like 4 (AGBL4).

**Conclusion:** 
We identified genetic variants that are potentially associated with ATD induced liver toxicity. Further studies with larger sample sizes are essential to confirm the findings.

**Electronic supplementary material:** 
The online version of this article (doi:10.1186/s12864-016-3078-3) contains supplementary material, which is available to authorized users.

Keywords: Anti-tuberculosis, Drug induced liver injury, Ethiopian, FAM65B, C6ORF32, GWAS, AGBL4, Hepatotoxicity, Africa, Tuberculosis

### Background

Drug-induced liver injury (DILI) is a well-recognized adverse event of anti tuberculosis drugs (ATD) possibly associated with genetic variations. The objective of this study was to perform genome-wide association study (GWAS) to identify genetic variants associated with the risk for ATD induced liver toxicity in Ethiopian patients.

### Result

Treatment-naïve newly diagnosed tuberculosis patients (*n* = 646) were enrolled prospectively and treated with rifampicin based short course anti-tuberculosis therapy. Whole genome genotyping was done using Illumina Omni Express Exome Bead Chip genotyping array with 951,117 single nucleotide polymorphisms (SNPs) on 48 DILI cases and 354 ATD tolerants. Replication study was carried out for 50 SNPs with the lowest *P*-values (top SNPs) using an independent cohort consisting of 27 DILI cases and 217 ATD tolerants. In the combined analysis, the top SNP identified was rs10946737 (*P* = 4.4 × 10^−6^, OR = 3.4, 95 % confidence interval = 2.2–5.3) in the intron of *FAM65B* in chromosome 6. In addition, we identified a cluster of SNPs with suggestive genome-wide significance in the intron of ATP/GTP binding protein-like 4 (*AGBL4*).

### Conclusion

We identified genetic variants that are potentially associated with ATD induced liver toxicity. Further studies with larger sample sizes are essential to confirm the findings.

### Electronic supplementary material

The online version of this article (doi:10.1186/s12864-016-3078-3) contains supplementary material, which is available to authorized users.

## Background

Liver toxicity associated with drug treatment, known as drug-induced liver injury (DILI) is implicated in most cases of acute liver failure [[1](#CR1)]. It can limit patient access to drugs that might otherwise be beneficial [[2](#CR2)]. DILI is a major adverse event that leads to termination of clinical drug development programs and regulatory measures on approved drugs [[3](#CR3)]. The largest population-based study reported on the incidence of DILI was from Iceland with a crude incidence rate of 19.1 cases per 100,000 inhabitants per year [[4](#CR4)]. Although the causes of DILI can be various, studies have shown that genetic variations in genes involved in drug disposition, cellular stress, and immune response may contribute to DILI susceptibility [[5](#CR5), [6](#CR6)].

Anti-tuberculosis drugs (ATD) are among the most reported anti-microbial drugs incriminated to be potential causes of DILI [[7](#CR7)]. ATD induced liver injury (ATDILI) is one of the most prevalent hepatotoxicities reported in many countries [[8](#CR8)]. A previous study in Ethiopian tuberculosis (TB) patients showed 17.3 % incidence of ATDILI [[9](#CR9)]. Incidence of treatment induced liver toxicity varies between populations. Higher incidence of concomitant ATD and antiretroviral (ARV) drugs induced liver toxicity in Ethiopian (30 %) compared to Tanzanian (10 %) TB and human immunodeficiency virus (HIV) coinfected patients has been reported [[10](#CR10), [11](#CR11)] Among the first line ATD, isoniazid, rifampicin, and pyrazinamide are known to cause DILI [[8](#CR8)]. Genetic variations contribute to inter-individual ATDIL susceptibility [[12](#CR12)]. Polymorphisms in drug metabolizing genes such as N-acetyltransferase 2 (*NAT2*), cytochrome P450 family 2 subfamily E polypeptide 1 (*CYP2E1*), glutathione S-transferase mu 1 (*GSTM1*), uridinediphosphate-glucuronosyltransferase1 family polypeptide A1 (*UGT1A1*) [[8](#CR8), [13](#CR13)], human leukocyte antigen (*HLA*) region [[5](#CR5), [8](#CR8)] and superoxide dismutase-2 mitochondrial (*SOD2*) gene [[8](#CR8), [14](#CR14), [15](#CR15)] have been suggested to play roles in ATDILI.

Sub-Saharan Africa is disproportionally affected by high burden of TB and HIV. According to the latest WHO report, Ethiopia is listed among the top ten high-TB burden countries globally and one of the high multidrug resistant TB (MDR-TB) burden countries [[16](#CR16)]. DILI is one of the important adverse events of anti-TB drugs, particularly during the intensive phase of TB therapy [[17](#CR17)]. Treatment has to be discontinued in those patients who developed severe ATD induced liver toxicity, and treatment interruption may increase the risk for emergence of multidrug-resistant TB. Increased risk of developing MDR-TB in Ethiopian TB patients who encountered adverse events during the first course of TB treatment is reported recently [[18](#CR18)]. Thus identification of genetic markers that predispose patients for ATD induced liver toxicity using GWAS in high TB burden sub-Saharan African countries, such as Ethiopia is imperative.

Using candidate gene approach, we previously identified genetic variation in *NAT2*, *CYP2B6*, and *ABCB1* genes as risk factors for ATD and antiretroviral (ARV) drugs co-treatment induced liver toxicity in TB-HIV co-infected patients [[10](#CR10), [11](#CR11), [19](#CR19)]. Although candidate gene studies contribute to the discovery of genetic risk variants associated with ATDILI, the discovered genetic factors may account only for a proportion of the genetic variations, and some of the studies led to inconsistent results [[20](#CR20)–[22](#CR22)]. Therefore, we aimed to identify additional genetic variants through genome-wide association study (GWAS) for ATDILI in Ethiopian TB patients.

## Results

A total of 646 TB patients participated in this study and 75 (11.6 %) of them met the criteria for DILI diagnosis while on ATD treatment. Whole genome genotyping was done using genomic DNA from 48 DILI cases and 354 ATD tolerants. Replication study for 50 SNPs with lowest *P*-values (top SNPs) was done using genomic DNA from an independent cohort consisting of 27 DILI cases and 217ATD tolerants. The difference between the GWAS and the replication cohorts was based on time of first presentation. The first groups of patients were used for the GWAS, and the subsequent group of patients used for the replication study. The study area, TB diagnostic methods, case definitions, inclusion and exclusion criteria, and ATD treatment regimens used were all the same. The demographics and clinical characteristics of the study participants are presented in Table [1](#Tab1). There were statistically significant differences (*P* < 0.05) in HIV status and liver function test values between cases and treatment tolerants in both the GWAS and replication cohorts. There were statistically significant differences in sex, CD4 count and viral load between cases and treatment tolerants in the GWAS cohort but not in the replication study, which may be attributed to the smaller sample size of the replication cohort. More than one-third of the cases in our study had cholestatic pattern of DILI, and the rest had hepatocellular or mixed pattern.

### Table 1.

| Variables | GWAS |  |  | Replication study |  |  |
| --- | --- | --- | --- | --- | --- | --- |
|   | DILI Cases | Treatment tolerants | P | DILI cases | Treatment tolerants | P |
| No. of patients | 48 | 354 | - | 27 | 217 | - |
| Sex (M, F) | 19, 29 | 203, 151 | 0.02 | 12, 15 | 85, 132 | 0.60 |
| Age (yr), M (SD) | 35.6 (10.4) | 35.7 (11.5) | 0.93 | 32.0 (7.4) | 33.4 (10.3) | 0.48 |
| BMI (kg/m2), M (SD) | 19.0 (3.2) | 19.3 (3.0) | 0.55 | 17.5 (3.0) | 18.9 (3.0) | 0.02 |
| HIV positive, N (%) | 44 (91.7) | 225 (63.6) | <0.01 | 25 (92.6) | 158 (72.8) | 0.03 |
| CD4 count, M (SD) | 96.6 (78.5) | 129.3 (120.8) | 0.03 | 116.8 (98.3) | 138.2 (121.0) | 0.33 |
| Viral load, log M (SD) | 5.3 (0.9) | 4.9 (0.9) | 0.03 | 5.0 (0.8) | 4.9 (0.9) | 0.54 |
| ALT (U/L), M (SD) | 69.7 (37.2) | 30.4 (14.4) | <0.01 | 67.2 (42.1) | 30.7 (14.1) | <0.01 |
| AST (U/L), M (SD) | 101.2 (52.7) | 40.5 (16.2) | <0.01 | 103.6 (71.9) | 38.7 (13.6) | <0.01 |
| ALP (U/L), M (SD) | 187.7 (72.2) | 121.1 (51.7) | <0.01 | 225.8 (139.9) | 114.0 (63.1) | <0.01 |
| T Bil (mg/dL), M (SD) | 1.2 (1.0) | 0.6 (0.4) | <0.01 | 1.1 (0.7) | 0.5 (0.3) | <0.01 |
| DILI pattern, N (%) |  |  |  |  |  |  |
| Cholestatic | 19 (39.6) |   |   | 15 (55.6) |   |   |
| Hepatocellular | 10 (20.8) |   |   | 5 (18.5) |   |   |
| Mixed | 19 (39.6) |   |   | 7 (25.9) |   |   |

Table 1 Caption: Demographics and clinical variables of the study participants

The Quantile-quantile (QQ) plot for the observed versus expected *P*-values, and Manhattan plot for the regression analysis are shown in Figs. [1](#Fig1) and [2](#Fig2), respectively. The top SNP in the GWAS after adjustment for sex, HIV status, CD4 count and HIV viral load was rs10946739 (*P* = 4.1 × 10^−6^, odds ratio (OR) = 3.4, 95 % CI = 2.0–5.6) located in the intron region of family with sequence similarity 65 member B (*FAM65B*), which is also named as chromosome 6 open reading frame 32 (*C6ORF32*) (Additional file [1](#MOESM1): Table S1). The top SNP in the replication study after adjustment for covariates was rs319952 (*P* = 1.0 × 10^−2^, OR = 2.3, 95 % CI = 1.2–4.4) located in the intron of ATP/GTP binding protein-like 4 (*AGBL4*) in chromosome 1 (Additional file [1](#MOESM1): Table S2). In the combined analysis, the top SNP after adjustment for covariates was rs10946737 (*P* = 4.4 × 10^−6^, OR = 3.4, 95 % CI = 2.2–5.3) located in the intron region of *FAM65B* (Table [2](#Tab2)). In addition, four of the top SNPs (rs320035, rs393994, rs319952 and rs320003) were clustered in the intron of *AGBL4*.

### Fig. 1.

![Fig. 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a765/5037629/f100cc9011ee/12864_2016_3078_Fig1_HTML.jpg)

Quantile-quantile (QQ) plot for the observed versus expected P-values in trend test (λGC = 1.00007)

### Fig. 2.

![Fig. 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a765/5037629/3e758aec481e/12864_2016_3078_Fig2_HTML.jpg)

−Log10 P values of logistic regression across chromosomes in the GWAS

### Table 2.

| SNP | Chr (loci) | Alleles (RA) | Study | Cases/controls | MAF | P_min | P_adj | OR (95 % CI) | Nearest gene |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| rs10946737 | 6 (24967240) | A/G (A) | GWAS | 48/354 | 0.10 | 2.0 × 10−5 | 9.7 × 10−6 | 4.3 (2.5–7.4) | FAM65B |
| Rep | 27/216 | 0.10 | 3.8 × 10−2 | 8.6 × 10−2 | 2.2 (0.9–5.4) |  |  |  |  |
| Comb | 75/570 | 0.10 | 6.3 × 10−7 | 4.4 × 10−6 | 3.4 (2.2–5.3) |  |  |  |  |
| rs320035 | 1 (49089197) | A/G (G) | GWAS | 48/354 | 0.48 | 3.5 × 10−6 | 1.3 × 10−4 | 2.4 (1.5–3.8) | AGBL4 |
| Rep | 27/216 | 0.50 | 4.2 × 10−3 | 1.2 × 10−2 | 2.2 (1.9–3.9) |  |  |  |  |
| Comb | 75/570 | 0.49 | 8.2 × 10−7 | 5.1 × 10−6 | 2.3 (1.6–3.3) |  |  |  |  |
| rs10946739 | 6 (24993127) | A/G (A) | GWAS | 48/354 | 0.19 | 9.6 × 10−6 | 4.1 × 10−6 | 3.4 (2.0–5.6) | FAM65B |
| Rep | 25/209 | 0.18 | 1.1 × 10−1 | 1.8 × 10−1 | 1.7 (0.8–3.6) |  |  |  |  |
| Comb | 73/563 | 0.19 | 4.7 × 10−6 | 5.1 × 10−6 | 2.7 (1.8–4.1) |  |  |  |  |
| rs393994 | 1 (49108745) | T/C (C) | GWAS | 48/354 | 0.48 | 6.1 × 10−6 | 1.7 × 10−4 | 2.4 (1.5–3.7) | AGBL4 |
| Rep | 27/216 | 0.50 | 7.9 × 10−3 | 1.4 × 10−2 | 2.1 (1.2–4.0) |  |  |  |  |
| Comb | 75/570 | 0.49 | 1.9 × 10−6 | 7.6 × 10−6 | 2.3 (1.6–3.3) |  |  |  |  |
| rs320003 | 1 (49126778) | A/G (A) | GWAS | 48/354 | 0.48 | 1.7 × 10−5 | 2.3 × 10−4 | 2.3 (1.5–3.7) | AGBL4 |
| Rep | 23/208 | 0.50 | 1.9 × 10−2 | 1.2 × 10−2 | 2.3 (1.2–4.5) |  |  |  |  |
| Comb | 71/562 | 0.49 | 4.6 × 10−6 | 8.3 × 10−6 | 2.3 (1.6–3.4) |  |  |  |  |
| rs319952 | 1 (49113622) | A/G (G) | GWAS | 48/354 | 0.48 | 1.1 × 10−5 | 2.8 × 10−4 | 2.3 (1.5–3.6) | AGBL4 |
| Rep | 26/216 | 0.50 | 1.2 × 10−2 | 1.0 × 10−2 | 2.3 (1.2–4.4) |  |  |  |  |
| Comb | 74/570 | 0.49 | 2.5 × 10−6 | 8.5 × 10−6 | 2.3 (1.6–3.3) |  |  |  |  |
| rs7958375 | 1 (2 111640017) | A/G (A) | GWAS | 48/354 | 0.02 | 8.8 × 10−5 | 1.2 × 10−5 | 11.3 (3.8–33.5) | CUX2 |
| Rep | 27/216 | 0.02 | 1.0 × 10+0 | 7.4 × 10−1 | 1.5 (0.2–13.1) |  |  |  |  |
| Comb | 75/570 | 0.02 | 1.7 × 10−4 | 4.6 × 10−5 | 7.6 (2.9–20.0) |  |  |  |  |

Table 2 Caption: Top SNPs in the combined analysis of the GWAS and the replication study

For the sub-group analysis based on the pattern of liver injury, the top SNPs for cholestatic, hepatocellular and mixed patterns of DILI were rs10182566 (*P* = 4.1 × 10^−6^, OR = 6.0, 95 % CI = 2.8–12.8) in 3′-untranslated region of chromosome 2 open reading frame 71 (*C2orf71*), rs1990046 (*P* = 3.7 × 10^−6^, OR = 28.4, 95 % CI = 6.9–117.3) in the intron of semaphorin3A (*SEMA3A*) in chromosome 7, and rs12603186 (*P* = 8.1 × 10^−6^, OR = 7.2, 95 % CI = 3.0–17.2) in shisa family member 6 (*SHISA6*) in chromosome 17, respectively (Additional file [1](#MOESM1): Table S3).

## Discussion

In this study, we carried out GWAS and replication analysis on a total of 646 patients treated with ATD to identify novel genetic variants associated with DILI. Previously we investigated pharmacogenetic markers for concomitant ARV and ATD co-treatment induced DILI in TB-HIV co-infected patients (*n* = 353) using candidate gene approach [[10](#CR10)]. As a continuation, we conducted a large prospective cohort study in 1060 patients, where we evaluated the patterns of ATD and/or ARV drugs induced liver toxicities [[23](#CR23)]. In the present study, we investigated for possible genetic markers for ATD induced liver toxicity using genome wide association approach in 646 selected study participants from the recent large prospective cohort study by considering DILI cases developed during anti-TB treatment only. Identifying the risk variants could help developing clinical tests to prevent DILI, and to match the patients with alternative, effective and safe medications. To our knowledge, this is the first GWAS for ATDILI in an African population.

The top SNP in the GWAS analysis after adjustment for covariates was in the intron of *FAM65B*. This gene encodes a cytoplasmic protein that plays a role in myoblast differentiation, and it is transiently up-regulated during early stage of the process [[24](#CR24)]. Alternative splicing of this gene results in multiple transcript variants. Inhibition of expression of this gene in myoblasts causes marked decrease in myogenin expression with consequent lack of myoblast fusion; and its over-expression induces formations of cellular protrusions [[25](#CR25)]. It is suggested that *FAM65B* may possibly play a role in myoblast migration, and mutations could affect muscle development and human muscle diseases; however, its exact role is still largely unknown [[25](#CR25)]. According to the human Protein Atlas data, *FAM65B* is also a mitochondrial protein expressed in the liver hepatocytes, gall bladder and bile duct [[26](#CR26)]. Recent studies indicate that *FAM65B* plays a role in cancer and liver inflammation [[27](#CR27)]. Further analysis is necessary to explain functional importance of *FAM65B* gene in ATDILI.

Strong association with ATDILI was identified by a cluster of four SNPs with *P*-values suggestive of genome-wide association significance in the intron of *AGBL4* (*CCP6*). This gene encodes an enzyme that catalyzes deglutamylation of polyglutamate side chains generated by post-translational modification of target proteins like tubulins in microtubules [[28](#CR28)]. Further analysis is required to explain the role of *AGBL4* gene and its contribution to individual differences for susceptibility to ATDILI. The identified genetic risk variants in our study if replicated in larger sample sizes and in other populations, they may serve as genetic biomarkers for ATDILI.

It is increasingly evident that genetic variants can determine an individual’s susceptibility to develop a particular pattern of liver injury [[29](#CR29)]. Therefore, we performed sub-group GWAS analysis based on the pattern of DILI. The SNP (rs1990046) with the smallest *P*-value after adjustment for covariates (*P* = 3.7 × 10^−6^) was identified in the hepatocellular type of DILI. This SNP is located in the intron region of *SEMA3A*, a member of the semaphorin family. This gene encodes a protein with an immunoglobulin-like domain and sema domain, which is vital for normal neuronal pattern development [[30](#CR30)], and also plays a role in the pathogenesis of allergic conditions such as allergic rhinitis [[31](#CR31)]. However, further studies are required to elucidate the role of *SEMA3A* gene in hepatocellular pattern of ATDILI.

In our previous candidate gene study [[10](#CR10)], genetic variants in genes involved in drug metabolism of ATD like *NAT2* were associated with DILI. Variants in other drug metabolizing genes [[8](#CR8), [13](#CR13)], *HLA* region [[5](#CR5), [8](#CR8)], and in genes related to oxidative stress [[32](#CR32)] and autoimmune diseases [[2](#CR2)] were also reported to have association with susceptibility to ATDILI. In our GWAS, we did not find genetic variants that passed genome-wide significance in these genes, which may be related to the limited sample sizes used for the study. But we found possible association SNPs rs12969241 (*P* = 1.1 × 10^−5^) located in the intron region of protein tyrosine phosphatase non-receptor type 2 (*PTPN2*), rs2842997 (*P* = 5.1 × 10^−3^) in the vicinity of *SOD2*, and rs12543818 (*P* = 1.9 × 10^−3^) near *NAT2* for genes related to autoimmune diseases, oxidative stress and pharmacokinetics, respectively (Additional file [1](#MOESM1): Table S4). The SNP rs12969241 in the *PTPN2* gene was also among the top in the GWAS of cholestatic pattern of DILI (*P* = 6.8 × 10^−6^) (Additional file [1](#MOESM1): Table S3). The protein encoded by the *PTPN2* gene is an intracellular tyrosine-specific phosphatase, which is expressed in epithelial cells, fibroblasts or endothelial cells [[33](#CR33)]. This protein was shown to play an important role in the protection of epithelial barrier function during inflammation by acting as negative regulator of pro-inflammatory cytokine interferon-γ [[34](#CR34)]. This finding may indicate the implication of an immune related mechanism in ATDILI. The product of *SOD2* gene, which was identified for genes related to oxidative stress, detoxifies highly reactive superoxide radicals generated by mitochondrial respiration [[35](#CR35)]. This finding is in line with a previous study [[15](#CR15)], which reported common polymorphisms in *SOD2* as predictor of ATDILI. We speculate that ATDILI may be related to the combined effect of the new variants identified, pharmacokinetic, oxidative stress, and immune-related gene variants.

Sub-Saharan African population is the most genetically heterogeneous population globally, characterized by extensive population substructure, and less linkage disequilibrium (LD) among loci compared to non-African populations.[[36](#CR36)]. Although GWAS in populations of African ancestry is challenging due to less degree of LD; the high level of genetic diversity and weak LD with neighboring SNPs in Africans ancestry is considered as a powerful tool for fine mapping causal variants that underlie common diseases or complex traits found globally [[37](#CR37)]. The advantage of conducting GWAS in African ancestry populations in the context of addressing existing and emerging global health conditions is reported recently [[37](#CR37)]. The present study exploring ATDILI risk variants through GWAS in Ethiopia, the second most densely populated country in Africa, will not only provide national genomic information for personalized medicine but also may contribute to the advancement of pharmacogenomics in Africa.

There were some limitations for this study. First, as the DILI cases are rare and were difficult to collect (four years were required to identify 75 ATDILI cases), this resulted in small number of case samples particularly for sub-group analysis based on the pattern of DILI. Second, populations of African ancestry, as in case of our study population, have greater genetic diversity and lower levels of linkage disequilibrium (LD) among chromosomal loci [[38](#CR38)]. The low levels of LD are disadvantageous when screening the genome for disease associations using the current SNP-genotyping approaches that essentially rely on the principle of LD mapping. Therefore, additional studies with higher density SNP array or next generation sequencing may be required to discover susceptibility variants in such population. Ethiopian population display distinct pharmacogenetic variations compared to other black African population [[39](#CR39)–[42](#CR42)], and thus results from this study may not be directly extrapolated to other sub-Saharan African population. However, our exploratory study using homogenous well-characterized clinical samples for the discovery and replication of new DILI biomarkers, represents an important first step in applying GWAS to identify genetic variants for ATDILI. The third limitation is that the current protocol of TB treatment consists of combinations of drugs, thus we cannot affirm that the risk variants identified corresponds only to a single drug or multiple drugs in the treatment regimen.

## Conclusion

Using genome-wide wide associations study, we identified potential genetic variants associated with ATDILI. The results provide evidence that in addition to genetic variants identified by candidate gene studies, other variants also influence the risk of developing DILI by ATD. Further replication studies are essential to confirm the findings.

## Methods

### Study participants and treatment

The participants for the present GWAS were selected from a recent prospective cohort study where patterns of antiretroviral therapy (ART) and/or anti-TB treatment induced liver toxicity was investigated [[23](#CR23)]. In brief newly diagnosed treatment naïve patients enrolled into one of the following study arms were considered for the present study:

Patients were recruited from three health institutions: Kazanchis and Beletshachew health centers and Black Lion specialized referral and teaching university hospital in Addis Ababa, Ethiopia [[23](#CR23)]. Diagnosis of TB was based on sputum smear, fine needle aspirate, clinical and radiological evidences. The eligibility criteria were TB confirmed men and non-pregnant women, age ≥18 years and receiving no other known hepatotoxic drugs concurrently. Patients who had abnormal liver enzyme biochemistry at baseline, positive serological test for either hepatitis B virus surface antigen or anti-hepatitis C virus antibody or known liver injury prior to starting treatment were excluded.. Written informed consent was obtained from all the study participants prior to study enrolment. The study protocol was approved by the Institutional Review Board of College of Health Sciences, Addis Ababa University, Ethiopia; Ethical Review Board of Karolinska Institutet, Sweden; and Ethical Review Committee of RIKEN, Japan.

Drug treatment was initiated according to World Health Organization (WHO) and Ethiopian National TB Treatment Guidelines as described previously [[23](#CR23)]. All patients received short-course ATD treatment consisting of rifampicin (150 mg), isoniazid (75 mg), pyrazinamide (400 mg) and ethambutol (275 mg) for the first two months in fixed dose combinations given daily under direct observed therapy during the intensive phase, followed by rifampicin (150 mg) and isoniazid (75 mg) for the next four months in fixed dose combinations given daily. The treatment dosage was based on the weight of the patient: 20–29 kg (1½ tablets), 30–37 kg (2 tablets), 38–54 kg (3 tablets) and ≥55 kg (4 tablets). Liver function tests were carried out at baseline and on the 1st, 2nd, 4th, 8th, 12th and 24th weeks after initiation of treatment.

### Case definitions

For DILI case definitions, the criteria set by the International DILI expert working group were used [[43](#CR43)]. The upper limit of normal (ULN) for liver biochemical parameters used for the study population were alanine aminotransferase (ALT 33 U/L, male; 29 U/L, female), aspartate aminotransferase (AST, 41 U/L), alkaline phosphatase (ALP, 128 U/L), and 1.0 mg/dL for total bilirubin [[23](#CR23)]. All cases recruited met at least one of the following criteria: − (1) ALT ≥5xULN, (2) ALP ≥2xULN, or (3) ALT ≥3xULN along with total bilirubin ≥2xULN. All cases had a minimum score of three (‘possible’) in Roussel Uclaf Causality Assessment Method (RUCAM) scoring system for DILI. The pattern of liver injury was defined using R-values where, R = (ALT/ULN)/(ALP/ULN). Cases were categorized as having hepatocellular (*R* ≥ 5), cholestatic (*R* ≤ 2), or mixed (2 < R < 5) pattern of DILI [[43](#CR43)]. Treatment tolerants for the study were defined as individuals who were also on short course ATD treatment but did not fulfill the case definitions for DILI, and had not presented clinical signs and symptoms consistent with DILI in the follow up period [[23](#CR23)].

### Whole genome genotyping and quality control

Genomic DNA was isolated from whole blood samples using QIAamp DNA Maxi Kit (QIAGEN GmbH, Hilden, Germany). Genotyping was conducted in RIKEN Center for Integrative Medical Sciences, Yokohama, Japan. Whole genome genotyping was done using Illumina Omni Express Exome Bead Chip genotyping array (Illumina Inc., San Diego, CA, USA) according to the manufacturer’s protocol. This array captures 951,117 single nucleotide polymorphisms (SNPs). To further validate the results of the GWAS analysis, replication study was then carried out for 50 SNPs with the lowest *P*-values (top SNPs) using an independent cohort. Genotyping for the replication study was done using multiplex polymerase chain reaction (PCR) based Invader assay [[44](#CR44)] with ABI PRISM 7900HT Sequence Detection System (Applied Biosystems, Foster City, CA, USA).

For data cleaning, systematic stepwise quality filtering of raw genotyping data was done using PLINK [[45](#CR45)]. From an initial full set, those SNPs not mapped on autosomal chromosomes were filtered out. In addition, SNPs with a call rate less than 99 %, minor allele frequency less than 0.01, or deviated from expected Hardy-Weinberg equilibrium (*P* < 1.0 × 10^−6^) were removed. A total of 660, 206 SNPs that passed the quality filter were used for further analysis. Individuals were checked for gender concordance between recorded clinical data and genotype determined sex. Samples with genotyping call rate greater than 99 % were included in the analysis. Quantile-quantile plot comparing the expected and observed *P*-values was performed in R-statistical environment, and genomic control inflation factor (λ_GC_) was computed to detect population stratification [[46](#CR46)].

#### Statistical analysis

After the quality filter, the tests of associations were done using PLINK v1.07 [[45](#CR45)]. For each SNP, Fisher’s exact test using the three genetic inheritance models (dominant, recessive, allele frequency) were carried out to compare allele and genotype frequencies between DILI cases and treatment tolerants. SNPs were rank-ordered according to the minimum *P*-value in the genetic models. The threshold for genome-wide significance for associated SNPs was determined using Bonferroni correction (*P* < 7.6 × 10^−8^). SNPs with *P*-values below 10^−5^ were considered suggestive of genome-wide significance. Logistic regression analysis adjusted for sex, HIV status, CD4 count and HIV viral load as covariates was performed. These variables were associated with DILI as described previously [[9](#CR9), [10](#CR10)]. Combined analysis of GWAS and replication study was conducted using inverse-variance method [[47](#CR47)]. Manhattan plot was generated using Haploview software to visualize the results [[48](#CR48)]. We also performed sub-group GWAS analysis based on the pattern of liver injury.

## Acknowledgements

We would like to thank to all the study participants. We convey our sincere appreciation to members of Laboratory for International Alliance on Genomic Research for the kind support and technical assistance.

### Funding

Biobank Japan Project, European and Developing Countries Clinical Trials Partnership (NL) (grant number CG_TA.05.40204_005) and from Swedish Research Council (grant number: 2015–03295)

### Authors’ contributions

Conceived and designed the experiments: EA EM MK TM GYAH MTML ZP. Performed the experiments: ZP YZ EA GY AH. Analyzed the data: AT MTML ZP EA. Contributed reagents/materials/analysis tools: EA MK MTML TM AT EM GY AH WA GA ISK. Wrote the paper: ZP MTML EA EM MK TM. All authors read and approved the final manuscript.

### Competing interests

The authors declare that they have no competing interests.

### Additional file

### Additional file

## Footnotes

## Contributor Information

Eyasu Makonnen, Email: eyasumakonnen@yahoo.com.

Michiaki Kubo, Email: mkubo@src.riken.jp.

Eleni Aklillu, Phone: +46 8 5858 7882, Email: Eleni.Aklillu@ki.se.

## References

1. Lee WM. Drug-induced hepatotoxicity. N Engl J Med. 2003;349:474–85. doi: 10.1056/NEJMra021844.  [DOI](https://doi.org/10.1056/NEJMra021844) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12890847/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Drug-induced%20hepatotoxicity&author=WM%20Lee&volume=349&publication_year=2003&pages=474-85&pmid=12890847&doi=10.1056/NEJMra021844&)

2. Urban TJ, Shen Y, Stolz A, Chalasani N, Fontana RJ, Rochon J, et al. Limited contribution of common genetic variants to risk for liver injury due to a variety of drugs. Pharmacogenet Genomics. 2012;22:784–95. doi: 10.1097/FPC.0b013e3283589a76.  [DOI](https://doi.org/10.1097/FPC.0b013e3283589a76) | [PMC free article](/articles/PMC3636716/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22968431/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Limited%20contribution%20of%20common%20genetic%20variants%20to%20risk%20for%20liver%20injury%20due%20to%20a%20variety%20of%20drugs&author=TJ%20Urban&author=Y%20Shen&author=A%20Stolz&author=N%20Chalasani&author=RJ%20Fontana&volume=22&publication_year=2012&pages=784-95&pmid=22968431&doi=10.1097/FPC.0b013e3283589a76&)

3. Watkins PB, Seligman PJ, Pears JS, Avigan MI, Senior JR. Using controlled clinical trials to learn more about acute drug-induced liver injury. Hepatology. 2008;48:1680–9. doi: 10.1002/hep.22633.  [DOI](https://doi.org/10.1002/hep.22633) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18853438/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hepatology&title=Using%20controlled%20clinical%20trials%20to%20learn%20more%20about%20acute%20drug-induced%20liver%20injury&author=PB%20Watkins&author=PJ%20Seligman&author=JS%20Pears&author=MI%20Avigan&author=JR%20Senior&volume=48&publication_year=2008&pages=1680-9&pmid=18853438&doi=10.1002/hep.22633&)

4. Bjornsson ES, Bergmann OM, Bjornsson HK, Kvaran RB, Olafsson S. Incidence, presentation, and outcomes in patients with drug-induced liver injury in the general population of Iceland. Gastroenterology. 2013;144:1419–25. doi: 10.1053/j.gastro.2013.02.006.  [DOI](https://doi.org/10.1053/j.gastro.2013.02.006) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23419359/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gastroenterology&title=Incidence,%20presentation,%20and%20outcomes%20in%20patients%20with%20drug-induced%20liver%20injury%20in%20the%20general%20population%20of%20Iceland&author=ES%20Bjornsson&author=OM%20Bergmann&author=HK%20Bjornsson&author=RB%20Kvaran&author=S%20Olafsson&volume=144&publication_year=2013&pages=1419-25&pmid=23419359&doi=10.1053/j.gastro.2013.02.006&)

5. Daly AK, Day CP. Genetic association studies in drug-induced liver injury. Drug Metab Rev. 2012;44:116–26. doi: 10.3109/03602532.2011.605790.  [DOI](https://doi.org/10.3109/03602532.2011.605790) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21913872/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Rev&title=Genetic%20association%20studies%20in%20drug-induced%20liver%20injury&author=AK%20Daly&author=CP%20Day&volume=44&publication_year=2012&pages=116-26&pmid=21913872&doi=10.3109/03602532.2011.605790&)

6. Chen R, Wang J, Zhang Y, Tang S, Zhan S. Key factors of susceptibility to anti-tuberculosis drug-induced hepatotoxicity. Arch Toxicol. 2015;89:883–97. doi: 10.1007/s00204-015-1473-1.  [DOI](https://doi.org/10.1007/s00204-015-1473-1) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25693865/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arch%20Toxicol&title=Key%20factors%20of%20susceptibility%20to%20anti-tuberculosis%20drug-induced%20hepatotoxicity&author=R%20Chen&author=J%20Wang&author=Y%20Zhang&author=S%20Tang&author=S%20Zhan&volume=89&publication_year=2015&pages=883-97&pmid=25693865&doi=10.1007/s00204-015-1473-1&)

7. Pugh AJ, Barve AJ, Falkner K, Patel M, McClain CJ. Drug-induced hepatotoxicity or drug-induced liver injury. Clin Liver Dis. 2009;13:277–94. doi: 10.1016/j.cld.2009.02.008.  [DOI](https://doi.org/10.1016/j.cld.2009.02.008) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19442919/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Liver%20Dis&title=Drug-induced%20hepatotoxicity%20or%20drug-induced%20liver%20injury&author=AJ%20Pugh&author=AJ%20Barve&author=K%20Falkner&author=M%20Patel&author=CJ%20McClain&volume=13&publication_year=2009&pages=277-94&pmid=19442919&doi=10.1016/j.cld.2009.02.008&)

8. Huang YS. Recent progress in genetic variation and risk of antituberculosis drug-induced liver injury. J Chin Med Assoc. 2014;77:169–73. doi: 10.1016/j.jcma.2014.01.010.  [DOI](https://doi.org/10.1016/j.jcma.2014.01.010) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24593909/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Chin%20Med%20Assoc&title=Recent%20progress%20in%20genetic%20variation%20and%20risk%20of%20antituberculosis%20drug-induced%20liver%20injury&author=YS%20Huang&volume=77&publication_year=2014&pages=169-73&pmid=24593909&doi=10.1016/j.jcma.2014.01.010&)

9. Yimer G, Aderaye G, Amogne W, Makonnen E, Aklillu E, Lindquist L, et al. Anti-tuberculosis therapy-induced hepatotoxicity among Ethiopian HIV-positive and negative patients. PLoS One. 2008;3:e1809. doi: 10.1371/journal.pone.0001809.  [DOI](https://doi.org/10.1371/journal.pone.0001809) | [PMC free article](/articles/PMC2265547/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18350147/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=Anti-tuberculosis%20therapy-induced%20hepatotoxicity%20among%20Ethiopian%20HIV-positive%20and%20negative%20patients&author=G%20Yimer&author=G%20Aderaye&author=W%20Amogne&author=E%20Makonnen&author=E%20Aklillu&volume=3&publication_year=2008&pages=e1809&pmid=18350147&doi=10.1371/journal.pone.0001809&)

10. Yimer G, Ueda N, Habtewold A, Amogne W, Suda A, Riedel KD, et al. Pharmacogenetic & pharmacokinetic biomarker for efavirenz based ARV and rifampicin based anti-TB drug induced liver injury in TB-HIV infected patients. PLoS One. 2011;6:e27810. doi: 10.1371/journal.pone.0027810.  [DOI](https://doi.org/10.1371/journal.pone.0027810) | [PMC free article](/articles/PMC3232196/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22162992/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=Pharmacogenetic%20&%20pharmacokinetic%20biomarker%20for%20efavirenz%20based%20ARV%20and%20rifampicin%20based%20anti-TB%20drug%20induced%20liver%20injury%20in%20TB-HIV%20infected%20patients&author=G%20Yimer&author=N%20Ueda&author=A%20Habtewold&author=W%20Amogne&author=A%20Suda&volume=6&publication_year=2011&pages=e27810&pmid=22162992&doi=10.1371/journal.pone.0027810&)

11. Mugusi S, Ngaimisi E, Janabi M, Minzi O, Bakari M, Riedel KD, et al. Liver enzyme abnormalities and associated risk factors in HIV patients on efavirenz-based HAART with or without tuberculosis co-infection in Tanzania. PLoS One. 2012;7:e40180. doi: 10.1371/journal.pone.0040180.  [DOI](https://doi.org/10.1371/journal.pone.0040180) | [PMC free article](/articles/PMC3394799/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22808112/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=Liver%20enzyme%20abnormalities%20and%20associated%20risk%20factors%20in%20HIV%20patients%20on%20efavirenz-based%20HAART%20with%20or%20without%20tuberculosis%20co-infection%20in%20Tanzania&author=S%20Mugusi&author=E%20Ngaimisi&author=M%20Janabi&author=O%20Minzi&author=M%20Bakari&volume=7&publication_year=2012&pages=e40180&pmid=22808112&doi=10.1371/journal.pone.0040180&)

12. Devarbhavi H. Antituberculous drug-induced liver injury: current perspective. Trop Gastroenterol. 2011;32:167–74.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/22332331/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Trop%20Gastroenterol&title=Antituberculous%20drug-induced%20liver%20injury:%20current%20perspective&author=H%20Devarbhavi&volume=32&publication_year=2011&pages=167-74&pmid=22332331&)

13. Huang YS. Genetic polymorphisms of drug-metabolizing enzymes and the susceptibility to antituberculosis drug-induced liver injury. Expert Opin Drug Metab Toxicol. 2007;3:1–8. doi: 10.1517/17425255.3.1.1.  [DOI](https://doi.org/10.1517/17425255.3.1.1) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17269890/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Expert%20Opin%20Drug%20Metab%20Toxicol&title=Genetic%20polymorphisms%20of%20drug-metabolizing%20enzymes%20and%20the%20susceptibility%20to%20antituberculosis%20drug-induced%20liver%20injury&author=YS%20Huang&volume=3&publication_year=2007&pages=1-8&pmid=17269890&doi=10.1517/17425255.3.1.1&)

14. Daly AK, Day CP. Genetic association studies in drug-induced liver injury. Semin Liver Dis. 2009;29:400–11. doi: 10.1055/s-0029-1240009.  [DOI](https://doi.org/10.1055/s-0029-1240009) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19826974/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Semin%20Liver%20Dis&title=Genetic%20association%20studies%20in%20drug-induced%20liver%20injury&author=AK%20Daly&author=CP%20Day&volume=29&publication_year=2009&pages=400-11&pmid=19826974&doi=10.1055/s-0029-1240009&)

15. Huang YS, Su WJ, Huang YH, Chen CY, Chang FY, Lin HC, et al. Genetic polymorphisms of manganese superoxide dismutase, NAD(P)H:quinone oxidoreductase, glutathione S-transferase M1 and T1, and the susceptibility to drug-induced liver injury. J Hepatol. 2007;47:128–34. doi: 10.1016/j.jhep.2007.02.009.  [DOI](https://doi.org/10.1016/j.jhep.2007.02.009) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17400324/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Hepatol&title=Genetic%20polymorphisms%20of%20manganese%20superoxide%20dismutase,%20NAD(P)H:quinone%20oxidoreductase,%20glutathione%20S-transferase%20M1%20and%20T1,%20and%20the%20susceptibility%20to%20drug-induced%20liver%20injury&author=YS%20Huang&author=WJ%20Su&author=YH%20Huang&author=CY%20Chen&author=FY%20Chang&volume=47&publication_year=2007&pages=128-34&pmid=17400324&doi=10.1016/j.jhep.2007.02.009&)

16. World Health Organization 2015. Global Tuberculosis Report. Geneva: World Health Organization; 2015. http://www.who.int/tb/publications/global_report/gtbr2015_executive_summary.pdf?ua=1 World [last accessed July 2016]  [http://www.who.int/tb/publications/global_report/gtbr2015_executive_summary.pdf?ua=1](http://www.who.int/tb/publications/global_report/gtbr2015_executive_summary.pdf?ua=1)

17. Kumar R, Shalimar, Bhatia V, Khanal S, Sreenivas V, Gupta SD, et al. Antituberculosis therapy-induced acute liver failure: magnitude, profile, prognosis, and predictors of outcome. Hepatology. 2010;51:1665–74. doi: 10.1002/hep.23534.  [DOI](https://doi.org/10.1002/hep.23534) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20196116/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hepatology&title=Antituberculosis%20therapy-induced%20acute%20liver%20failure:%20magnitude,%20profile,%20prognosis,%20and%20predictors%20of%20outcome&author=R%20Kumar&author=%20Shalimar&author=V%20Bhatia&author=S%20Khanal&author=V%20Sreenivas&volume=51&publication_year=2010&pages=1665-74&pmid=20196116&doi=10.1002/hep.23534&)

18. Hirpa S, Medhin G, Girma B, Melese M, Mekonen A, Suarez P, et al. Determinants of multidrug-resistant tuberculosis in patients who underwent first-line treatment in Addis Ababa: a case control study. BMC Public Health. 2013;13:782. doi: 10.1186/1471-2458-13-782.  [DOI](https://doi.org/10.1186/1471-2458-13-782) | [PMC free article](/articles/PMC4015150/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23981845/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Public%20Health&title=Determinants%20of%20multidrug-resistant%20tuberculosis%20in%20patients%20who%20underwent%20first-line%20treatment%20in%20Addis%20Ababa:%20a%20case%20control%20study&author=S%20Hirpa&author=G%20Medhin&author=B%20Girma&author=M%20Melese&author=A%20Mekonen&volume=13&publication_year=2013&pages=782&pmid=23981845&doi=10.1186/1471-2458-13-782&)

19. Yimer G, Amogne W, Habtewold A, Makonnen E, Ueda N, Suda A, et al. High plasma efavirenz level and CYP2B6(*)6 are associated with efavirenz-based HAART-induced liver injury in the treatment of naive HIV patients from Ethiopia: a prospective cohort study. Pharmacogenomics J. 2012;12:499–506. doi: 10.1038/tpj.2011.34.  [DOI](https://doi.org/10.1038/tpj.2011.34) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21862974/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=High%20plasma%20efavirenz%20level%20and%20CYP2B6(*)6%20are%20associated%20with%20efavirenz-based%20HAART-induced%20liver%20injury%20in%20the%20treatment%20of%20naive%20HIV%20patients%20from%20Ethiopia:%20a%20prospective%20cohort%20study&author=G%20Yimer&author=W%20Amogne&author=A%20Habtewold&author=E%20Makonnen&author=N%20Ueda&volume=12&publication_year=2012&pages=499-506&pmid=21862974&doi=10.1038/tpj.2011.34&)

20. Tang S, Lv X, Zhang Y, Wu S, Yang Z, Xia Y, et al. Cytochrome P450 2E1 gene polymorphisms/haplotypes and anti-tuberculosis drug-induced hepatitis in a Chinese cohort. PLoS One. 2013;8:e57526. doi: 10.1371/journal.pone.0057526.  [DOI](https://doi.org/10.1371/journal.pone.0057526) | [PMC free article](/articles/PMC3583841/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23460870/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=Cytochrome%20P450%202E1%20gene%20polymorphisms/haplotypes%20and%20anti-tuberculosis%20drug-induced%20hepatitis%20in%20a%20Chinese%20cohort&author=S%20Tang&author=X%20Lv&author=Y%20Zhang&author=S%20Wu&author=Z%20Yang&volume=8&publication_year=2013&pages=e57526&pmid=23460870&doi=10.1371/journal.pone.0057526&)

21. Yamada S, Tang M, Richardson K, Halaschek-Wiener J, Chan M, Cook VJ, et al. Genetic variations of NAT2 and CYP2E1 and isoniazid hepatotoxicity in a diverse population. Pharmacogenomics. 2009;10:1433–45. doi: 10.2217/pgs.09.66.  [DOI](https://doi.org/10.2217/pgs.09.66) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19761367/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Genetic%20variations%20of%20NAT2%20and%20CYP2E1%20and%20isoniazid%20hepatotoxicity%20in%20a%20diverse%20population&author=S%20Yamada&author=M%20Tang&author=K%20Richardson&author=J%20Halaschek-Wiener&author=M%20Chan&volume=10&publication_year=2009&pages=1433-45&pmid=19761367&doi=10.2217/pgs.09.66&)

22. Vuilleumier N, Rossier MF, Chiappe A, Degoumois F, Dayer P, Mermillod B, et al. CYP2E1 genotype and isoniazid-induced hepatotoxicity in patients treated for latent tuberculosis. Eur J Clin Pharmacol. 2006;62:423–9. doi: 10.1007/s00228-006-0111-5.  [DOI](https://doi.org/10.1007/s00228-006-0111-5) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16770646/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Clin%20Pharmacol&title=CYP2E1%20genotype%20and%20isoniazid-induced%20hepatotoxicity%20in%20patients%20treated%20for%20latent%20tuberculosis&author=N%20Vuilleumier&author=MF%20Rossier&author=A%20Chiappe&author=F%20Degoumois&author=P%20Dayer&volume=62&publication_year=2006&pages=423-9&pmid=16770646&doi=10.1007/s00228-006-0111-5&)

23. Yimer G, Gry M, Amogne W, Makonnen E, Habtewold A, Petros Z, et al. Evaluation of patterns of liver toxicity in patients on antiretroviral and anti-tuberculosis drugs, a prospective four arm observational study in Ethiopian patients. PLoS One. 2014;9:e94271. doi: 10.1371/journal.pone.0094271.  [DOI](https://doi.org/10.1371/journal.pone.0094271) | [PMC free article](/articles/PMC3979833/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24714066/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=Evaluation%20of%20patterns%20of%20liver%20toxicity%20in%20patients%20on%20antiretroviral%20and%20anti-tuberculosis%20drugs,%20a%20prospective%20four%20arm%20observational%20study%20in%20Ethiopian%20patients&author=G%20Yimer&author=M%20Gry&author=W%20Amogne&author=E%20Makonnen&author=A%20Habtewold&volume=9&publication_year=2014&pages=e94271&pmid=24714066&doi=10.1371/journal.pone.0094271&)

24. Yoon S, Molloy MJ, Wu MP, Cowan DB, Gussoni E. C6ORF32 is upregulated during muscle cell differentiation and induces the formation of cellular filopodia. Dev Biol. 2007;301:70–81. doi: 10.1016/j.ydbio.2006.11.002.  [DOI](https://doi.org/10.1016/j.ydbio.2006.11.002) | [PMC free article](/articles/PMC1779902/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17150207/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Dev%20Biol&title=C6ORF32%20is%20upregulated%20during%20muscle%20cell%20differentiation%20and%20induces%20the%20formation%20of%20cellular%20filopodia&author=S%20Yoon&author=MJ%20Molloy&author=MP%20Wu&author=DB%20Cowan&author=E%20Gussoni&volume=301&publication_year=2007&pages=70-81&pmid=17150207&doi=10.1016/j.ydbio.2006.11.002&)

25. Balasubramanian A, Kawahara G, Gupta VA, Rozkalne A, Beauvais A, Kunkel LM, et al. Fam65b is important for formation of the HDAC6-dysferlin protein complex during myogenic cell differentiation. FASEB J. 2014;28:2955–69. doi: 10.1096/fj.13-246470.  [DOI](https://doi.org/10.1096/fj.13-246470) | [PMC free article](/articles/PMC4062822/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24687993/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=FASEB%20J&title=Fam65b%20is%20important%20for%20formation%20of%20the%20HDAC6-dysferlin%20protein%20complex%20during%20myogenic%20cell%20differentiation&author=A%20Balasubramanian&author=G%20Kawahara&author=VA%20Gupta&author=A%20Rozkalne&author=A%20Beauvais&volume=28&publication_year=2014&pages=2955-69&pmid=24687993&doi=10.1096/fj.13-246470&)

26. The human protein atlas: FAM65B: Available form http://www.proteinatlas.org/ENSG00000111913-FAM65B/tissue (Accessed on 17 Dec 2015). Geneva, Switzerland  [http://www.proteinatlas.org/ENSG00000111913-FAM65B/tissue](http://www.proteinatlas.org/ENSG00000111913-FAM65B/tissue)

27. Stoyanov E, Ludwig G, Mizrahi L, Olam D, Schnitzer-Perlman T, Tasika E, et al. Chronic liver inflammation modifies DNA methylation at the precancerous stage of murine hepatocarcinogenesis. Oncotarget. 2015;6:11047–60. doi: 10.18632/oncotarget.3567.  [DOI](https://doi.org/10.18632/oncotarget.3567) | [PMC free article](/articles/PMC4484438/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25918251/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncotarget&title=Chronic%20liver%20inflammation%20modifies%20DNA%20methylation%20at%20the%20precancerous%20stage%20of%20murine%20hepatocarcinogenesis&author=E%20Stoyanov&author=G%20Ludwig&author=L%20Mizrahi&author=D%20Olam&author=T%20Schnitzer-Perlman&volume=6&publication_year=2015&pages=11047-60&pmid=25918251&doi=10.18632/oncotarget.3567&)

28. Rodriguez de la Vega Otazo M, Lorenzo J, Tort O, Aviles FX, Bautista JM. Functional segregation and emerging role of cilia-related cytosolic carboxypeptidases (CCPs) FASEB J. 2013;27:424–31. doi: 10.1096/fj.12-209080.  [DOI](https://doi.org/10.1096/fj.12-209080) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23085998/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=FASEB%20J&title=Functional%20segregation%20and%20emerging%20role%20of%20cilia-related%20cytosolic%20carboxypeptidases%20(CCPs)&author=M%20Rodriguez%20de%20la%20Vega%20Otazo&author=J%20Lorenzo&author=O%20Tort&author=FX%20Aviles&author=JM%20Bautista&volume=27&publication_year=2013&pages=424-31&pmid=23085998&doi=10.1096/fj.12-209080&)

29. Padda MS, Sanchez M, Akhtar AJ, Boyer JL. Drug-induced cholestasis. Hepatology. 2011;53:1377–87. doi: 10.1002/hep.24229.  [DOI](https://doi.org/10.1002/hep.24229) | [PMC free article](/articles/PMC3089004/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21480339/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hepatology&title=Drug-induced%20cholestasis&author=MS%20Padda&author=M%20Sanchez&author=AJ%20Akhtar&author=JL%20Boyer&volume=53&publication_year=2011&pages=1377-87&pmid=21480339&doi=10.1002/hep.24229&)

30. Boczek NJ, Ye D, Johnson EK, Wang W, Crotti L, Tester DJ, et al. Characterization of SEMA3A-encoded semaphorin as a naturally occurring Kv4.3 protein inhibitor and its contribution to Brugada syndrome. Circ Res. 2014;115:460–9. doi: 10.1161/CIRCRESAHA.115.303657.  [DOI](https://doi.org/10.1161/CIRCRESAHA.115.303657) | [PMC free article](/articles/PMC4907370/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24963029/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circ%20Res&title=Characterization%20of%20SEMA3A-encoded%20semaphorin%20as%20a%20naturally%20occurring%20Kv4.3%20protein%20inhibitor%20and%20its%20contribution%20to%20Brugada%20syndrome&author=NJ%20Boczek&author=D%20Ye&author=EK%20Johnson&author=W%20Wang&author=L%20Crotti&volume=115&publication_year=2014&pages=460-9&pmid=24963029&doi=10.1161/CIRCRESAHA.115.303657&)

31. Vadasz Z, Haj T, Toubi E. The role of B regulatory cells and Semaphorin3A in atopic diseases. Int Arch Allergy Immunol. 2014;163:245–51. doi: 10.1159/000360477.  [DOI](https://doi.org/10.1159/000360477) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24713604/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20Arch%20Allergy%20Immunol&title=The%20role%20of%20B%20regulatory%20cells%20and%20Semaphorin3A%20in%20atopic%20diseases&author=Z%20Vadasz&author=T%20Haj&author=E%20Toubi&volume=163&publication_year=2014&pages=245-51&pmid=24713604&doi=10.1159/000360477&)

32. Daly AK, Donaldson PT, Bhatnagar P, Shen Y, Pe’er I, Floratos A, et al. HLA-B*5701 genotype is a major determinant of drug-induced liver injury due to flucloxacillin. Nat Genet. 2009;41:816–9. doi: 10.1038/ng.379.  [DOI](https://doi.org/10.1038/ng.379) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19483685/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=HLA-B*5701%20genotype%20is%20a%20major%20determinant%20of%20drug-induced%20liver%20injury%20due%20to%20flucloxacillin&author=AK%20Daly&author=PT%20Donaldson&author=P%20Bhatnagar&author=Y%20Shen&author=I%20Pe%E2%80%99er&volume=41&publication_year=2009&pages=816-9&pmid=19483685&doi=10.1038/ng.379&)

33. Morón B, Spalinger M, Kasper S, Atrott K, Frey-Wagner I, Fried M, et al. Activation of protein tyrosine phosphatase non-receptor type 2 by spermidine exerts anti-inflammatory effects in human THP-1 monocytes and in a mouse model of acute colitis. PLoS One. 2013;8:e73703. doi: 10.1371/journal.pone.0073703.  [DOI](https://doi.org/10.1371/journal.pone.0073703) | [PMC free article](/articles/PMC3767590/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24040033/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=Activation%20of%20protein%20tyrosine%20phosphatase%20non-receptor%20type%202%20by%20spermidine%20exerts%20anti-inflammatory%20effects%20in%20human%20THP-1%20monocytes%20and%20in%20a%20mouse%20model%20of%20acute%20colitis&author=B%20Mor%C3%B3n&author=M%20Spalinger&author=S%20Kasper&author=K%20Atrott&author=I%20Frey-Wagner&volume=8&publication_year=2013&pages=e73703&pmid=24040033&doi=10.1371/journal.pone.0073703&)

34. Penrose HM, Marchelletta RR, Krishnan M, McCole DF. Spermidine stimulates T cell protein-tyrosine phosphatase-mediated protection of intestinal epithelial barrier function. J Biol Chem. 2013;288:32651–62. doi: 10.1074/jbc.M113.475962.  [DOI](https://doi.org/10.1074/jbc.M113.475962) | [PMC free article](/articles/PMC3820900/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24022492/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=Spermidine%20stimulates%20T%20cell%20protein-tyrosine%20phosphatase-mediated%20protection%20of%20intestinal%20epithelial%20barrier%20function&author=HM%20Penrose&author=RR%20Marchelletta&author=M%20Krishnan&author=DF%20McCole&volume=288&publication_year=2013&pages=32651-62&pmid=24022492&doi=10.1074/jbc.M113.475962&)

35. Candas D, Li JJ. MnSOD in oxidative stress response-potential regulation via mitochondrial protein influx. Antioxid Redox Signal. 2014;20:1599–617. doi: 10.1089/ars.2013.5305.  [DOI](https://doi.org/10.1089/ars.2013.5305) | [PMC free article](/articles/PMC3942709/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23581847/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Antioxid%20Redox%20Signal&title=MnSOD%20in%20oxidative%20stress%20response-potential%20regulation%20via%20mitochondrial%20protein%20influx&author=D%20Candas&author=JJ%20Li&volume=20&publication_year=2014&pages=1599-617&pmid=23581847&doi=10.1089/ars.2013.5305&)

36. Campbell MC, Tishkoff SA. African genetic diversity: implications for human demographic history, modern human origins, and complex disease mapping. Annu Rev Genomics Hum Genet. 2008;9:403–33. doi: 10.1146/annurev.genom.9.081307.164258.  [DOI](https://doi.org/10.1146/annurev.genom.9.081307.164258) | [PMC free article](/articles/PMC2953791/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18593304/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Annu%20Rev%20Genomics%20Hum%20Genet&title=African%20genetic%20diversity:%20implications%20for%20human%20demographic%20history,%20modern%20human%20origins,%20and%20complex%20disease%20mapping&author=MC%20Campbell&author=SA%20Tishkoff&volume=9&publication_year=2008&pages=403-33&pmid=18593304&doi=10.1146/annurev.genom.9.081307.164258&)

37. Peprah E, Xu H, Tekola-Ayele F, Royal CD. Genome-wide association studies in Africans and African Americans: expanding the framework of the genomics of human traits and disease. Public Health Genomics. 2015;18:40–51. doi: 10.1159/000367962.  [DOI](https://doi.org/10.1159/000367962) | [PMC free article](/articles/PMC4418477/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25427668/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Public%20Health%20Genomics&title=Genome-wide%20association%20studies%20in%20Africans%20and%20African%20Americans:%20expanding%20the%20framework%20of%20the%20genomics%20of%20human%20traits%20and%20disease&author=E%20Peprah&author=H%20Xu&author=F%20Tekola-Ayele&author=CD%20Royal&volume=18&publication_year=2015&pages=40-51&pmid=25427668&doi=10.1159/000367962&)

38. Teo Y, Small K, Kwiatkowski D. Methodological challenges of genome-wide association analysis in Africa. Nat Rev Genet. 2010;11:149–60. doi: 10.1038/nrg2731.  [DOI](https://doi.org/10.1038/nrg2731) | [PMC free article](/articles/PMC3769612/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20084087/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Genet&title=Methodological%20challenges%20of%20genome-wide%20association%20analysis%20in%20Africa&author=Y%20Teo&author=K%20Small&author=D%20Kwiatkowski&volume=11&publication_year=2010&pages=149-60&pmid=20084087&doi=10.1038/nrg2731&)

39. Ngaimisi E, Habtewold A, Minzi O, Makonnen E, Mugusi S, Amogne W, et al. Importance of ethnicity, CYP2B6 and ABCB1 genotype for efavirenz pharmacokinetics and treatment outcomes: a parallel-group prospective cohort study in two sub-Saharan Africa populations. PLoS One. 2013;8:e67946. doi: 10.1371/journal.pone.0067946.  [DOI](https://doi.org/10.1371/journal.pone.0067946) | [PMC free article](/articles/PMC3702506/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23861838/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=Importance%20of%20ethnicity,%20CYP2B6%20and%20ABCB1%20genotype%20for%20efavirenz%20pharmacokinetics%20and%20treatment%20outcomes:%20a%20parallel-group%20prospective%20cohort%20study%20in%20two%20sub-Saharan%20Africa%20populations&author=E%20Ngaimisi&author=A%20Habtewold&author=O%20Minzi&author=E%20Makonnen&author=S%20Mugusi&volume=8&publication_year=2013&pages=e67946&pmid=23861838&doi=10.1371/journal.pone.0067946&)

40. Aklillu E, Djordjevic N, Carrillo JA, Makonnen E, Bertilsson L, Ingelman-Sundberg M. High CYP2A6 enzyme activity as measured by a caffeine test and unique distribution of CYP2A6 variant alleles in Ethiopian population. OMICS. 2014;18:446–53. doi: 10.1089/omi.2013.0140.  [DOI](https://doi.org/10.1089/omi.2013.0140) | [PMC free article](/articles/PMC4086241/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24380444/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=OMICS&title=High%20CYP2A6%20enzyme%20activity%20as%20measured%20by%20a%20caffeine%20test%20and%20unique%20distribution%20of%20CYP2A6%20variant%20alleles%20in%20Ethiopian%20population&author=E%20Aklillu&author=N%20Djordjevic&author=JA%20Carrillo&author=E%20Makonnen&author=L%20Bertilsson&volume=18&publication_year=2014&pages=446-53&pmid=24380444&doi=10.1089/omi.2013.0140&)

41. Aklillu E, Mugusi S, Ngaimisi E, Hoffmann MM, Konig S, Ziesenitz V, et al. Frequency of the SLCO1B1 388A > G and the 521 T > C polymorphism in Tanzania genotyped by a new LightCycler(R)-based method. Eur J Clin Pharmacol. 2011;67:1139–45. doi: 10.1007/s00228-011-1065-9.  [DOI](https://doi.org/10.1007/s00228-011-1065-9) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21630030/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Clin%20Pharmacol&title=Frequency%20of%20the%20SLCO1B1%20388A%E2%80%89>%E2%80%89G%20and%20the%20521%C2%A0T%E2%80%89>%E2%80%89C%20polymorphism%20in%20Tanzania%20genotyped%20by%20a%20new%20LightCycler(R)-based%20method&author=E%20Aklillu&author=S%20Mugusi&author=E%20Ngaimisi&author=MM%20Hoffmann&author=S%20Konig&volume=67&publication_year=2011&pages=1139-45&pmid=21630030&doi=10.1007/s00228-011-1065-9&)

42. Gebeyehu E, Engidawork E, Bijnsdorp A, Aminy A, Diczfalusy U, Aklillu E. Sex and CYP3A5 genotype influence total CYP3A activity: high CYP3A activity and a unique distribution of CYP3A5 variant alleles in Ethiopians. Pharmacogenomics J. 2011;11:130–37. doi: 10.1038/tpj.2010.16.  [DOI](https://doi.org/10.1038/tpj.2010.16) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20231858/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=Sex%20and%20CYP3A5%20genotype%20influence%20total%20CYP3A%20activity:%20high%20CYP3A%20activity%20and%20a%20unique%20distribution%20of%20CYP3A5%20variant%20alleles%20in%20Ethiopians&author=E%20Gebeyehu&author=E%20Engidawork&author=A%20Bijnsdorp&author=A%20Aminy&author=U%20Diczfalusy&volume=11&publication_year=2011&pages=130-37&pmid=20231858&doi=10.1038/tpj.2010.16&)

43. Aithal GP, Watkins PB, Andrade RJ, Larrey D, Molokhia M, Takikawa H, et al. Case definition and phenotype standardization in drug-induced liver injury. Clin Pharmacol Ther. 2011;89:806–15. doi: 10.1038/clpt.2011.58.  [DOI](https://doi.org/10.1038/clpt.2011.58) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21544079/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Case%20definition%20and%20phenotype%20standardization%20in%20drug-induced%20liver%20injury&author=GP%20Aithal&author=PB%20Watkins&author=RJ%20Andrade&author=D%20Larrey&author=M%20Molokhia&volume=89&publication_year=2011&pages=806-15&pmid=21544079&doi=10.1038/clpt.2011.58&)

44. Ohnishi Y, Tanaka T, Ozaki K, Yamada R, Suzuki H, Nakamura Y. A high-throughput SNP typing system for genome-wide association studies. J Hum Genet. 2001;46:471–7. doi: 10.1007/s100380170047.  [DOI](https://doi.org/10.1007/s100380170047) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11501945/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Hum%20Genet&title=A%20high-throughput%20SNP%20typing%20system%20for%20genome-wide%20association%20studies&author=Y%20Ohnishi&author=T%20Tanaka&author=K%20Ozaki&author=R%20Yamada&author=H%20Suzuki&volume=46&publication_year=2001&pages=471-7&pmid=11501945&doi=10.1007/s100380170047&)

45. Purcell S, Neale B, Todd-Brown K, Thomas L, Ferreira MA, Bender D, et al. PLINK: a tool set for whole-genome association and population-based linkage analyses. Am J Hum Genet. 2007;81:559–75. doi: 10.1086/519795.  [DOI](https://doi.org/10.1086/519795) | [PMC free article](/articles/PMC1950838/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17701901/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=PLINK:%20a%20tool%20set%20for%20whole-genome%20association%20and%20population-based%20linkage%20analyses&author=S%20Purcell&author=B%20Neale&author=K%20Todd-Brown&author=L%20Thomas&author=MA%20Ferreira&volume=81&publication_year=2007&pages=559-75&pmid=17701901&doi=10.1086/519795&)

46. Devlin B, Roeder K. Genomic control for association studies. Biometrics. 1999;55:997–1004. doi: 10.1111/j.0006-341X.1999.00997.x.  [DOI](https://doi.org/10.1111/j.0006-341X.1999.00997.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11315092/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biometrics&title=Genomic%20control%20for%20association%20studies&author=B%20Devlin&author=K%20Roeder&volume=55&publication_year=1999&pages=997-1004&pmid=11315092&doi=10.1111/j.0006-341X.1999.00997.x&)

47. de Bakker PI, Ferreira MA, Jia X, Neale BM, Raychaudhuri S, Voight BF. Practical aspects of imputation-driven meta-analysis of genome-wide association studies. Hum Mol Genet. 2008;17:R122–8. doi: 10.1093/hmg/ddn288.  [DOI](https://doi.org/10.1093/hmg/ddn288) | [PMC free article](/articles/PMC2782358/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18852200/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mol%20Genet&title=Practical%20aspects%20of%20imputation-driven%20meta-analysis%20of%20genome-wide%20association%20studies&author=PI%20de%20Bakker&author=MA%20Ferreira&author=X%20Jia&author=BM%20Neale&author=S%20Raychaudhuri&volume=17&publication_year=2008&pages=R122-8&pmid=18852200&doi=10.1093/hmg/ddn288&)

48. Barrett JC, Fry B, Maller J, Daly MJ. Haploview: analysis and visualization of LD and haplotype maps. Bioinformatics. 2005;21:263–5. doi: 10.1093/bioinformatics/bth457.  [DOI](https://doi.org/10.1093/bioinformatics/bth457) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15297300/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=Haploview:%20analysis%20and%20visualization%20of%20LD%20and%20haplotype%20maps&author=JC%20Barrett&author=B%20Fry&author=J%20Maller&author=MJ%20Daly&volume=21&publication_year=2005&pages=263-5&pmid=15297300&doi=10.1093/bioinformatics/bth457&)
