# Pathway-Based Analysis of Genome-Wide Association Data Identified SNPs in HMMR as Biomarker for Chemotherapy- Induced Neutropenia in Breast Cancer Patients

## Metadata
**Authors:** Behzad Bidadi, Duan Liu, Krishna R Kalari, Matthias Rubner, Alexander Hein, Matthias W Beckmann, Brigitte Rack, Wolfgang Janni, Peter A Fasching, Richard M Weinshilboum, Liewei Wang
**Journal:** Frontiers in Pharmacology
**Date:** 2018 Mar 13
**DOI:** [10.3389/fphar.2018.00158](https://doi.org/10.3389/fphar.2018.00158)
**PMID:** 29593529
**PMCID:** PMC5859084
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5859084/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC5859084/pdf/fphar-09-00158.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC5859084/pdf/fphar-09-00158.pdf)

## Abstract

Neutropenia secondary to chemotherapy in breast cancer patients can be life-threatening and there are no biomarkers available to predict the risk of drug-induced neutropenia in those patients. We previously performed a genome-wide association study (GWAS) for neutropenia events in women with breast cancer who were treated with 5-fluorouracil, epirubicin and cyclophosphamide and recruited to the SUCCESS-A trial. A genome-wide significant single-nucleotide polymorphism (SNP) signal in the tumor necrosis factor superfamily member 13B (TNFSF13B) gene, encoding the cytokine B-cell activating factor (BAFF), was identified in that GWAS. Taking advantage of these existing GWAS data, in the present study we utilized a pathway-based analysis approach by leveraging knowledge of the pharmacokinetics and pharmacodynamics of drugs and breast cancer pathophysiology to identify additional SNPs/genes associated with the underlying etiology of chemotherapy-induced neutropenia. We identified three SNPs in the hyaluronan mediated motility receptor (HMMR) gene that were significantly associated with neutropenia (p < 1.0E-04). Those three SNPs were trans-expression quantitative trait loci for the expression of TNFSF13B (p < 1.0E-04). The minor allele of these HMMR SNPs was associated with a decreased TNFSF13B mRNA level. Additional functional studies performed with lymphoblastoid cell lines (LCLs) demonstrated that LCLs possessing the minor allele for the HMMR SNPs were more sensitive to drug treatment. Knock-down of TNFSF13B in LCLs and HL-60 promyelocytic cells and treatment of those cells with BAFF modulated the cell sensitivity to chemotherapy treatment. These results demonstrate that HMMR SNP-dependent cytotoxicity of these chemotherapeutic agents might be related to TNFSF13B expression level. In summary, utilizing a pathway-based approach for the analysis of GWAS data, we identified additional SNPs in the HMMR gene that were associated with neutropenia and also were correlated with TNFSF13B expression.

Keywords: breast cancer, neutropenia, HMMR, TNFSF13B, GWAS

## Introduction

Breast cancer is the most common invasive cancer in women worldwide and approximately 1.3 million cases are diagnosed worldwide annually ([Ferlay et al., 2015](#B11)). Those patients with high-risk features require chemotherapy following surgical resection of the tumor. However, administration of chemotherapy regimens such as cytotoxic agents potentially leads to life-threatening adverse drug effects, of which the most common adverse event is chemotherapy-induced neutropenia (CIN) ([Fontanella et al., 2014](#B12)). This adverse event is common in breast cancer patients receiving adjuvant chemotherapy and affects more than 50% of patients ([Holmes et al., 2002](#B15); [Therasse et al., 2003](#B51)). More than 20% of patients with breast cancer receiving adjuvant chemotherapy developed febrile neutropenia, which is a more severe, life-threatening adverse event ([Do et al., 2015](#B9)). CIN may lead to infection and hospitalization, which potentially results in compromised treatment because of the requirement for reduction in dose intensity of the chemotherapy regimen ([Link et al., 2001](#B25); [Fontanella et al., 2014](#B12); [Agiro et al., 2016](#B2)). To prevent CIN, co-administration of white blood cell growth factor such as the granulocyte colony-stimulating growth factor (G-CSF) is recommend with chemotherapy ([Smith et al., 2015](#B46)), and has shown benefit in patients with breast cancer who received chemotherapy ([Holmes et al., 2002](#B15); [Vogel et al., 2005](#B55); [Agiro et al., 2016](#B2)). However, considerable inter-individual variabilities in both risk and severity of CIN, and in the beneficial effect of G-CSF prophylaxis were found among breast cancer patients ([Agiro et al., 2016](#B2)). Sub-classification of patients at increased risk of CIN may allow for improved treatment modifications, such as G-CSF support, dose reduction, or use of chemotherapy regimens with a lower risk of myelosuppression. Clinical risk factors such as older age, poor nutrition and prior chemotherapy have been associated with increased risk of febrile neutropenia ([Aapro et al., 2006](#B1)). However, predictive models using these factors for risk of CIN have shown limited utility ([Lyman et al., 2015](#B29)). Other predictors for CIN risk are urgently needed which may help to improve the individualized care of breast cancer patients during adjuvant chemotherapy.

Germline genetic polymorphisms in patients with cancer have been associated with risk for chemotherapy-related toxicity, including CIN ([Ulrich et al., 2003](#B53); [Pinto et al., 2012](#B40)). Using a candidate gene approach, previous studies have demonstrated that single-nucleotide polymorphisms (SNPs) in genes encoding drug-metabolizing enzymes and transporters were associated with the risk for CIN in breast cancer patients ([Low et al., 2009](#B28); [Yao et al., 2010](#B62); [Okishiro et al., 2012](#B38); [Tang et al., 2013](#B48)). However, those studies included a limited number of SNPs/genes and relatively small sample sizes, which might explain the contradictory results among these studies — e.g., no SNPs/genes were replicated from one study to another even though the same chemotherapy regimens were administrated to those breast cancer patients. To our knowledge, no genome-wide association study (GWAS) for CIN has been reported. Compared to the candidate gene approach, GWAS is an “unbiased” approach that has the capability of revealing novel genetic variants associated with drug response ([Weinshilboum and Wang, 2004](#B59); [Wang et al., 2011](#B58); [Motsinger-Reif et al., 2013](#B34)). We have previously performed GWASs for adverse events in breast cancer patients treated with aromatase inhibitors and identified novel genetic variants associated with musculoskeletal pain ([Ingle et al., 2010](#B18)) and bone fractures ([Liu et al., 2014](#B27)). Recently, we performed a GWAS for CIN in 3,252 women with early stage breast cancer who received adjuvant chemotherapy. Novel SNP signals that were genome-wide significantly associated with CIN were identified in that GWAS. That GWAS will be reported in detail elsewhere.

Although GWAS has been very successful for identifying novel and statistically significant SNPs that were associated with phenotypes, many SNPs/genes that have biological significance could be overlooked in GWAS since those SNPs do not meet the threshold to reach genome-wide significance ([Wang et al., 2007](#B56), [2010](#B57)). To overcome this problem, pathway-based approaches, which consider prior knowledge of genes and pathways contributing to the phenotype of interest has been developed to provide additional insight into the interpretation of GWAS data on complex phenotypes ([Wang et al., 2007](#B56); [Ramanan et al., 2012](#B43)). This approach has been successfully used to analyze and interpret GWAS data for many clinical phenotypes ([Torkamani et al., 2008](#B52); [Baranzini et al., 2009](#B7); [Holmans et al., 2009](#B14); [Ballard et al., 2010](#B6); [Jia et al., 2010](#B20))

In the present study, we took advantage of the GWAS data we generated for CIN in breast cancer patients, and performed pathway-based analysis to identify SNPs/genes that might be associated with CIN. The SNPs identified by this pathway-based analysis were pursued by functional characterization to help us understand underlying mechanisms.

## Materials and Methods

### Patients and the Clinic Trial

Patients in this study were recruited to the SUCCESS-A trial ([ClinicalTrials.gov](http://www.ClinicalTrials.gov) Identifier: [NCT02181101](https://clinicaltrials.gov/ct2/show/NCT02181101)), a randomized Phase III study of response to the treatment of early primary breast cancer with adjuvant therapy after surgical resection ([Rack et al., 2014](#B42); [Widschwendter et al., 2015](#B60)). All patients initially received three cycles of epirubicin, 5-fluorouracil (5-FU), and cyclophosphamide. Patients were then randomized to either receive three cycles of either gemcitabine in addition to docetaxel or docetaxel alone. A second randomization consisted of zoledronate at the conclusion of treatment for either 2 or 5 years. This trial recruited 3754 patients over 18 months in 251 medical centers in Germany, and accrual ended in March 2007. The main study and all pre-specified translational research projects, including the one reported here, were approved by all of the responsible ethics committees and were conducted in accordance with the Declaration of Helsinki. This study was also reviewed and approved by the Mayo Clinic Institutional Review Board. All patients gave written informed consent.

### Case Definition for Neutropenia and GWAS

Cases were defined as patients who had at least one of the following events: leukopenia, neutropenia or febrile neutropenia, during cycles one to three of chemotherapy with epirubicin, 5-fluorouracil (5-FU), and cyclophosphamide. Cases were required to meet the toxicity level of Grade 3 or 4 based on the National Cancer Institute (NCI)’s Common Terminology Criteria for Adverse Events v3.0. A total of 1648 patients fulfilled the criteria for cases. Other patients served as controls for the GWAS analysis. Patient DNA samples were genotyped on the Illumina HumanOmniExpress-12v1 G FFPE array (Illumina, San Diego, CA, United States). After remove of 5 related patients and 9 patients who were non-Caucasian (Asian), a total of 3,252 patients (1,635 controls vs. 1,617 cases) who passed QC for genotyping and whose covariate information (Supplementary Table [S1](#SM2)) was available were included in the GWAS analysis. Imputation was performed based on 1000 Genomes Project data ([Howie et al., 2012](#B17)). Genotyped SNPs which had a *p-*value < 1.0E-05 in the initial GWAS were used as input SNPs to perform imputation. Genotyping and imputation are described in detail in the Supplementary Methods. Data that were generated in the SUCCESS-A trial, including genotype and clinical phenotype data, as well as a further descriptions of materials can be found in the NCBI database of Genotypes and Phenotypes (dbGaP Study Accession: phs000547.v1.p).

### Selection of Pathways, Genes and SNPs

Genes involved in pathways of (1) the pharmacokinetics (PK) and pharmacodynamics (PD) for the chemotherapeutic drugs used in the SUCCESS-A trial, cyclophosphamide, epirubicin and 5-FU were included in this analysis. These candidate genes were identified using the PharmGKB database^1^. Genes involved in the PK/PD of doxorubicin, an analog of epirubicin, were selected because epirubicin is not included in the PharmGKB. (2) Pathways that were shown to be aberrant in breast cancer according to The Cancer Genome Atlas (TCGA) data were included in the analysis ([The Cancer Genome Atlas Network, 2012](#B50)). Those pathways were the PI3K, mTOR, p53, apoptosis, and checkpoint signaling pathways. Genes in these pathways were identified using the Kyoto Encyclopedia of Genes and Genomes (KEGG) database ([Kanehisa and Goto, 2000](#B21)). (3) The PAM50 genes which have been shown to be effective in predicting disease prognosis ([Nielsen et al., 2010](#B35)) were also included. (4) Finally, genes studied in our laboratory shown to be functionally relevant in the AKT/mTOR signaling pathways in chemotherapy response ([Pei et al., 2009](#B39); [Hou et al., 2014](#B16); [Yu et al., 2017](#B63)) and toxicity ([Ingle et al., 2010](#B18)) were also added to the analysis.

SNPs within 50 kb up- and down-stream, and across the selected genes as described above were included in this analysis. SNPs that were located > 50 kb away from a gene might also affect gene transcription and function. Those SNPs will be overlooked in this study. Genotyping information for SNPs in those pathway genes in the SUCCESS-A patients was obtained from our GWAS genotyping data. Imputation was performed based on 1000 Genomes Project data ([Howie et al., 2012](#B17)). See Supplementary Methods for details.

### Association of SNPs With Neutropenia

The analysis for association of SNPs with neutropenia was based on conditional logistic regression to account for the matched design. SNP genotypes were coded as additive effects on the log OR by coding 0, 1, or 2 for the minor allele count. A likelihood ratio test with one degree of freedom for each SNP was then determined. The primary covariates (Supplementary Table [S1](#SM2)) used to match cases and controls were controlled in the conditional logistic regression. Association analyses were performed in R^2^, SAS (SAS Institute Inc.) and PLINK. Logistic regression models were used to calculate Odds ratios (ORs), 95% confidence intervals (CIs), and *p*-values, following corrections for multiple testing.

### eQTL Analysis

Single-nucleotide polymorphisms of interest that were associated with neutropenia were further subjected to expression quantitative trait loci (eQTL) analysis using our lymphoblastoid cell line (LCL) data sets, which have been described in detail in our previous studies ([Li et al., 2008](#B23), [2014](#B24); [Niu et al., 2010](#B36); [Matimba et al., 2014](#B31); [Liu et al., 2017](#B26)). The Human Variation Panel of LCLs from 96 European-American (EA), 95 African-American (AA), and 96 Han Chinese-American (HCA) healthy subjects were obtained from the Coriell Institute. Those LCLs are B-lymphocytes that were immortalized by Epstein Barr Virus (EBV) infection. Genome-wide genotype data and mRNA expression data for these LCLs have been generated in our laboratory by using the Illumina 550K and 510S SNP BeadChips (Illumina, San Diego, CA, United States), and the Affymetrix Human Genome U133 Plus 2.0 GeneChip arrays (Affymetrix, Santa Clara, CA, United States), respectively. Genotype and gene expression data were deposited in the National Center for Biotechnology Information Gene Expression Omnibus (GEO accession: [GSE23120](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE23120)). Expression array data were normalized on a log2 scale using GCRMA ([Wu et al., 2004](#B61)). The normalized expression data were then regressed on gender. Partial Pearson correlations were used to quantify the association between SNPs and mRNA expression. Since the LCLs were from multiple races/ethnic groups, before completing the genetic association analysis, population stratification was assessed using the method developed by [Price et al. (2006)](#B41) as described in detail previously ([Niu et al., 2010](#B36)). These partial correlations were tested using a Wald test. False discovery *q*-values ([Storey and Tibshirani, 2003](#B47)) were also computed for each test.

### Cell Culture and Cytotoxicity Assays

Lymphoblastoid cell lines were cultured in RPMI 1640 media containing 15% fetal bovine serum (FBS). The human promyelocytic leukemia cell line, HL-60, was obtained from the American Type Culture Collection (ATCC, Manassas, VA, United States), and was cultured in RPMI 1640 media containing 10% FBS. Epirubicin (EPI) was purchased from Santa Cruz Biotechnology, Inc. (Santa Cruz, CA, United States). Mafosfamide (MFF) was purchased from Toronto Research Chemicals Inc. (Toronto, Canada). MFF can spontaneously decompose to 4-hydroxy-cyclophosphamide, the active metabolite of cyclophosphamide, when added in culture media ([Mazur et al., 2012](#B32)). Fluorouracil (5-FU) was purchased from Sigma-Aldrich (St. Louis, MO, United States). Cytotoxicity assays with LCLs and HL-60 cells were performed with the CellTiter 96^®^ Aqueous Non-Radioactive Cell Proliferation Assay (Promega Corporation, Madison, WI, United States) in 96 well plates (Corning, Corning, NY, United States) at a density of 5 × 10^5^ cells/mL (100 μL/well). 10 μL of 5-FU (500–0.01 μM), epirubicin (10–0.0005 μM), or mafosfamide (100–0.005 μM) were added into the wells and incubated at 37°C for 72 h. Plates were then added with 20 μL of MTS buffer and read in an Infinite M1000 PRO plate reader (Tecan AG, Switzerland) after incubation for 3 h. Relative cell viability was then plotted against drug concentration to derive a cytotoxicity curve. Experiments were independently repeated at least twice with triplicate wells for each treatment. Independent *t*-test was used to determine statistical significance.

### mRNA Quantification

Messenger RNA levels were quantified by quantitative reverse transcription PCR (qRT-PCR) using Power SYBR^TM^ Green RNA-to-CT^TM^ 1-Step Kit (Applied Biosystems Inc., Foster City, CA, United States). Total RNA was extracted from cells by the RNeasy Plus Mini Kit (QIAGEN, Germany). A total of 100 ng of total RNA was used for each reaction. Specific primers for *TNFSF13B* mRNA were purchased from Integrated DNA Technologies (Coralville, IA, United States). The qRT-PCR reaction was performed using the Stratagene Mx3005P Quantitative Real-Time PCR detection system (Agilent Technologies, Santa Clara, CA, United States). Glyceraldehyde 3-phosphate dehydrogenase (GAPDH) was used as an internal control.

### TNFSF13B KD and Ligand Treatment for Cytotoxicity Assays

LCLs and HL-60 cells were incubated at a density of 1 × 10^6^ cells/mL in RPMI 1640 with 5% charcoal-stripped FBS for 24 h followed by serum free medium for an additional 24 h. For *TNFSF13B* KD, cells were transfected with negative control and *TNFSF13B* siRNAs (Dharmacon, Lafayette, CO, United States) separately, and incubated in RPMI 1640 with 5% charcoal-stripped FBS overnight before cytotoxicity assay. For TNFSF13B ligand (BAFF) treatment, recombinant human BAFF (Cell Signaling Technology, Danvers, MA, United States) reconstituted in PBS containing 200 μM dithiothreitol (DTT) and 0.02% bovine serum albumin was added to the cell culture media at a final concentration of 1 μg/mL and were incubated for 24 h. Vehicle treatment that included DTT and BSA but without BAFF was performed as control. After KD and BAFF treatment, cytotoxicity assays were performed as describe above. Experiments were repeated independently three times.

## Results

A summary of the patients’ characteristics is listed in the Supplementary Table [S1](#SM2). A GWAS for CIN in women with early stage breast cancer who were enrolled in the SUCCESS-A trial has been completed. The Manhattan plot of the GWAS for CIN in women with breast cancer during three cycles of epirubicin, 5-FU, and cyclophosphamide treatment, is shown in Supplementary Figure [S1](#SM1). Two genome-wide significant (*p* < 5.0E-08) SNP signals were identified, including one that mapped to the *TNFSF13B* gene on chromosome 13, with lowest *p-*value of 2.4E-08. *TNFSF13B* encodes a cytokine known as B-cell activating factor (BAFF) which plays an important role in the proliferation and differentiation of B-cells ([Mackay and Browning, 2002](#B30)).

Taking advantages of the existing GWAS data, we utilized a pathway-based approach by considering prior knowledge of these drugs and the disease, to identify additional SNPs/genes that might be associated with neutropenia.

### Pathway-Based Genes and SNPs

Ninety-five candidate genes involved in the PK and PD pathways of the chemotherapeutic drugs used in the SUCCESS-A trial were identified (Supplementary Table [S2](#SM3)). In addition, 311 genes were identified in pathways that were shown to be aberrant in breast cancer according to TCGA data ([The Cancer Genome Atlas Network, 2012](#B50)) were included (Supplementary Table [S3](#SM4)). The 50 genes in the PAM50 gene list (Supplementary Table [S4](#SM5)) which have been shown to be effective in predicting disease prognosis ([Nielsen et al., 2010](#B35)), and 12 genes that had been identified in previous cancer studies in our laboratory (Supplementary Table [S5](#SM6)) were also added to the analysis. A summary of the pathways, number of genes and SNPs is listed in **Table 1**. Several genes were included in multiple pathways and overlapping SNPs were removed during the follow up analysis. Finally, a total of 325, 934 SNPs were included in the analysis for association with CIN in breast cancer patients.

### Table 1.

| Pathway | Pathway gene source | Number of pathway genes | Number of SNPs in pathway genes | Number of SNPs associated with neutropenia (p < 1.0E-04) |
| --- | --- | --- | --- | --- |
| Cyclophosphamide PK/PD | PharmGKB | 40 | 17,548 | 1 |
| Doxorubicin PK/PD | PharmGKB | 30 | 18,879 | 5 |
| 5-Fluorouracil PK/PD | PharmGKB | 28 | 21,063 | 1 |
| Apoptosis | TCGA/KEGG | 72 | 47,321 | 1 |
| Checkpoint | TCGA/KEGG | 46 | 27,072 | 4 |
| P53 signaling | TCGA/KEGG | 88 | 53,318 | 24 |
| mTOR signaling | TCGA/KEGG | 98 | 69,486 | 17 |
| PI3K signaling | TCGA/KEGG | 100 | 63,853 | 15 |
| PAM50 | PAM50 | 50 | 31,550 | 0 |
| AKT/mTOR signaling | “In house” | 12 | 7,394 | 0 |
|   |  |  |  |  |

Table 1 Caption: Pathways and numbers of genes and single-nucleotide polymorphisms (SNPs) involved in pathways included in the analysis and the number of SNPs that were associated with neutropenia.

### SNP Associations With Neutropenia and eQTL Analysis

Genotypes of selected SNPs in the pathway genes were associated with the CIN phenotype in patients recruited in the SUCCESS-A trial. A total of 35 SNPs were identified that were associated with CIN with a *p-*value < 1.0E-04 (**Table 2**). The associations listed in **Table 2** are “nominal” and have not been corrected for multiple comparisons because the purpose was to identify SNPs/genes for functional validation. None of those SNPs was located in an exon or was a splicing variant. We then set out to determine whether those SNPs were either *cis*- or *trans*-eQTLs. To do that, we performed eQTL analysis using the 300 LCLs for which genome-wide SNP genotype and mRNA expression data had been generated in our lab ([Niu et al., 2010](#B36)). Six SNPs with very low MAFs (<0.010) were not applicable for eQTL analysis in those 300 LCLs because the number of LCLs with the variant SNP allele was too small to compare with LCLs with WT genotypes (**Table 2**). For other SNPs that underwent eQTL analysis, none SNPs were *cis*-eQTLs. However, we identified nine SNPs that were *trans*-eQTLs (**Table 2**). Interestingly, three SNPs, rs299293, rs299313, and rs299314, which were located in the hyaluronan mediated motility receptor (*HMMR)* gene, were *trans*-eQTLs for *TNFSF13B* (**Figure 1A** and Supplementary Table [S6](#SM7)), one of the “top” hits identified in the original GWAS for neutropenia (Supplementary Figure [S1](#SM1)). These 3 *HMMR* SNPs were in tight linkage disequilibrium (LD) (*r*^2^ > 0.87) and belonged to the same haplotype. The genotypes for those 3 SNPs in an LCL were matched – e.g., an LCL with homozygous variant genotype for one of those 3 SNPs was also homozygous variant genotype for the other two SNPs. These 3 *HMMR* SNPs were imputed SNPs in the pathway-based analysis, and they were excluded from the GWAS because none of the genotyped SNPs in LD with these three SNPs had a *p-*value < 1.0E-05, the cut-off value for input SNPs to perform imputation for GWAS analysis. The rs299293 SNP was located in intron 12, and the rs299313 and rs299314 SNPs were in intron 16 of *HMMR*. Those 3 SNPs mapped to the 3′ end of the cyclin G1 (*CCNG1*) gene which is involved in the p53 signaling pathway. However, those three SNPs were not eQTLs for either *HMMR* (**Figure 1B**) or *CCNG1* (**Figure 1C**) mRNA expression in LCLs.

### Table 2.

| SNP rs ID | Chr. | Common allele | Minor allele | OR | MAF | p-value | Gene | Pathway | eQTL in LCLs |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| rs144991623 | 13 | C | T | 69.8 | 0.002 | 5.98E-06 | FOXO1 | Checkpoint, PI3K | NA |
| rs9389568 | 6 | T | C | 0.59 | 0.049 | 7.10E-06 | PERP | P53 | Yes |
| rs8110364 | 19 | G | A | 3.09 | 0.011 | 2.23E-05 | PIK3R2 | mTOR, PI3K, P53 | Yes |
| rs56022120 | 19 | C | T | 3.12 | 0.011 | 2.29E-05 | PIK3R2 | mTOR, PI3K, P53 | No |
| rs150688309 | 19 | C | G | 2.87 | 0.013 | 2.67E-05 | PIK3R2 | mTOR, PI3K, P53 | No |
| rs58695150 | 19 | C | T | 3.07 | 0.011 | 3.15E-05 | PIK3R2 | mTOR, PI3K, P53 | Yes |
| rs299293 | 5 | C | T | 1.36 | 0.133 | 3.19E-05 | HMMR/CCNG1 | P53 | Yes |
| rs138602176 | 19 | G | A | 2.97 | 0.011 | 3.24E-05 | PIK3R2 | mTOR, PI3K, P53 | No |
| rs148235907 | 19 | G | A | 3.06 | 0.011 | 3.35E-05 | PIK3R2 | mTOR, PI3K, P53 | No |
| rs117951771 | 19 | C | T | 3.06 | 0.011 | 3.37E-05 | PIK3R2 | mTOR, PI3K, P53 | No |
| rs299314 | 5 | T | C | 1.35 | 0.138 | 3.60E-05 | HMMR/CCNG1 | P53 | Yes |
| rs299313 | 5 | G | A | 1.35 | 0.139 | 3.72E-05 | HMMR/CCNG1 | P53 | Yes |
| rs115457081 | 2 | G | A | 5.04 | 0.012 | 3.89E-05 | IRS1 | mTOR | No |
| rs117341846 | 19 | C | T | 3.03 | 0.011 | 3.93E-05 | PIK3R2 | mTOR, PI3K, P53 | No |
| rs4968187 | 17 | C | T | 0.10 | 0.004 | 4.09E-05 | TP53 | 5FU, Apoptosis, Checkpoint, Doxorubicin, PI3K, P53 | NA |
| rs142244113 | 19 | C | T | 1.72 | 0.039 | 4.59E-05 | INSR | mTOR | No |
| rs79430272 | 19 | C | T | 2.94 | 0.011 | 4.80E-05 | PIK3R2 | mTOR, PI3K, P53 | No |
| rs118129530 | 19 | G | A | 3.00 | 0.011 | 4.82E-05 | PIK3R2 | mTOR, PI3K, P53 | No |
| rs77769901 | 14 | A | G | 0.31 | 0.012 | 4.99E-05 | CCNK | P53 | No |
| rs145623321 | 19 | C | T | 3.01 | 0.010 | 5.35E-05 | PIK3R2 | mTOR, PI3K, P53 | No |
| rs148013902 | 19 | C | T | 3.01 | 0.010 | 5.46E-05 | PIK3R2 | mTOR, PI3K, P53 | No |
| rs55633228 | 19 | C | T | 3.00 | 0.010 | 5.59E-05 | PIK3R2 | mTOR, PI3K, P53 | No |
| rs141084494 | 22 | G | A | 0.05 | 0.003 | 6.26E-05 | RBX1 | Checkpoint | NA |
| rs41412545 | 19 | C | A | 1.70 | 0.038 | 6.36E-05 | INSR | mTOR | No |
| rs3805945 | 6 | T | C | 1.68 | 0.040 | 6.81E-05 | PPP2R5D | mTOR | No |
| rs117101815 | 6 | G | T | 0.63 | 0.051 | 7.17E-05 | PERP | P53 | Yes |
| rs9402944 | 6 | G | T | 0.63 | 0.051 | 7.20E-05 | PERP | P53 | Yes |
| rs181501757 | 17 | G | A | 4.77 | 0.008 | 7.75E-05 | TOP2A | Doxorubicin | NA |
| rs149212925 | 12 | A | G | 0.00 | 0.001 | 8.33E-05 | NOS1 | Doxorubicin | NA |
| rs112783657 | 21 | C | T | 0.49 | 0.027 | 8.65E-05 | CBR3 | Doxorubicin | No |
| rs74743371 | 21 | G | T | 0.49 | 0.027 | 8.68E-05 | CBR3 | Doxorubicin | No |
| rs117458836 | 10 | G | A | 2.53 | 0.014 | 8.85E-05 | CYP2C8 | Cyclophosphamide | No |
| rs118088833 | 11 | C | T | 0.26 | 0.009 | 8.99E-05 | TP53AIP1 | P53 | NA |
| rs112242273 | 3 | C | T | 0.52 | 0.038 | 9.58E-05 | GNL3 | Checkpoint, P53 | No |
| rs78428806 | 6 | G | A | 0.63 | 0.048 | 9.79E-05 | PERP | P53 | Yes |
|   |  |  |  |  |  |  |  |  |  |

Table 2 Caption: Single-nucleotide polymorphisms (SNPs) associated with risk for chemotherapy-induced neutropenia (p < 1.0E-04) in women with breast cancer recruited to the SUCCESS-A trail (n = 3252) and expression quantitative trait loci (eQTL) analysis in lymphoblastoid cell lines (LCLs).

### FIGURE 1.

![FIGURE 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0391/5859084/3b22264c8f6c/fphar-09-00158-g001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5859084_fphar-09-00158-g001.jpg)

Expression quantitative trait loci (eQTL) analysis for hyaluronan mediated motility receptor (HMMR) single-nucleotide polymorphisms (SNPs) (rs299293, rs299313, and rs299314) in lymphoblastoid cell lines (LCLs). These three HMMR SNPs belong to a same haplotype and their genotypes were same in each LCLs. The mRNA level of (A) TNFSF13B, (B) HMMR, and (C) CCNG1 were grouped based on the genotype of HMMR SNPs, and compared by one-way ANOVA. A p-value < 0.05 was considered statistically significant. The mRNA level was measured by the Affymetrix U133 Plus 2.0GeneChip expression array with specific probes as specified. Values are mean ± SEM for mRNA level in LCLs with wild type (W/W, n = 247), heterozygous variant (W/V, n = 35) and homozygous variant (V/V, n = 2) genotypes (showed in red line and error bars), respectively.

### HMMR SNP-Dependent Cytotoxicity

To confirm that the SNPs in the *HMMR* gene were associated with neutropenia, we performed cytotoxicity assays for drugs used in the SUCCESS-A trial using the LCLs with WT and variant genotypes for the *HMMR* SNPs. Six LCLs with variant SNP genotypes and six with WT genotype were chosen randomly for cytotoxicity assay. Since only two LCLs with homozygous variant genotypes for those SNPs were available, four LCLs heterozygous variant for those SNPs were included in the variant group. To confirm genotypes for those three SNPs in *HMMR* which were imputed based on the 1K Genome data, DNA sequencing was performed for those SNPs in these 12 LCLs that were later used in cytotoxicity assays. We found that LCLs possessing the variant *HMMR* SNP genotypes were more sensitive to all three agents, 5-FU, MFF, and EPI (**Figures 2A–C**). This result was consistent with our clinical observations in patients treated with these drugs. Patients with the variant SNP genotype had a higher risk of neutropenia compared to patients with the WT SNP genotype (OR = 1.36) (**Table 2**). The eQTL analysis for those *HMMR* SNPs indicated that the SNPs were *trans*-eQTLs for *TNFSF13B* and variant SNP genotypes were associated with decreased *TNFSF13B* mRNA level (**Figure 1A**). To further determine *TNFSF13B* mRNA levels in those 12 LCLs that were used for cytotoxicity assays, qRT-PCR for *TNFSF13B* mRNA were performed with total RNA extracted from those LCLs. The average mRNA level of *TNFSF13B* in six LCLs with variant SNP alleles was significantly decreased compared with six LCLs with WT genotype (*p* = 0.006) (**Figure 2D**), a result consistent with the eQTL analysis (**Figure 1A**). The mRNA levels of *TNFSF13B*, *HMMR*, and *CCNG1* in 12 LCLs that were determined by mRNA microarray again indicated a decrease in the *TNFSF13B* mRNA in LCLs with variant SNP genotypes but no significant changes in the *HMMR* and *CCNG1* mRNA levels (Supplementary Figure [S2](#SM1)).

### FIGURE 2.

![FIGURE 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0391/5859084/02f8f0a6059f/fphar-09-00158-g002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5859084_fphar-09-00158-g002.jpg)

Cytotoxicity for LCLs with different genotypes for HMMR SNPs. Average cytotoxicity for (A) fluorouracil (5-FU), (B) mafosfamide (MFF), and (C) epirubicin (EPI) in LCLs with W/W genotypes (n = 6) vs. LCLs with V/V and W/V genotypes (n = 2+4). Cell viability was tested by MTS assay after 72 h of drug exposure. Relative cell viability was plotted against drug concentration to derive a cytotoxicity curve. Cytotoxicity for each LCL was averaged from triplicate determinations. Values represent mean ± SEM of LCLs in each genotype group. (D) TNFSF13B mRNA level in LCLs that are used for cytotoxicity assay were determined by qRT-PCR and grouped based on the HMMR SNP genotype. Values are mean ± SEM and compared by unpaired t-test. A p-value < 0.05 was considered statistically significant. ∗∗p < 0.01.

### TNFSF13B and Chemo-Drug Sensitivity

To further demonstrate that *TNFSF13B* expression was important for the sensitivity to these chemotherapeutic agents, we performed a cytotoxicity assay in an LCL with relatively high *TNFSF13B* expression after knock-down (KD) of *TNFSF13B*. The same cytotoxicity assay was also performed with a human promyelocytic leukemia cell line, HL-60, a cell representative of promyelocytes which can differentiate to myelocytes, the major components of white blood cells. These cells were more sensitive to 5-FU and MFF treatment, but not EPI treatment, in both cell lines (**Figures 3A,C**) after *TNFSF13B* KD (**Figures 3B,D**) when compared with the negative control KD. Since *TNFSF13B* encodes a cytokine known as BAFF, we also performed cytotoxicity assays in the LCL and HL-60 cells with BAFF treatment. Both the LCL and HL-60 that were treated with recombinant human BAFF showed dose-dependent resistance to 5-FU and MFF treatment (**Figures 4A,B**, left and middle) but were not significantly changed by EPI treatment (**Figures 4A,B**, right).

### FIGURE 3.

![FIGURE 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0391/5859084/744e55400177/fphar-09-00158-g003.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5859084_fphar-09-00158-g003.jpg)

Cytotoxicity for LCLs and HL-60 after TNFSF13B Knockdown. (A) Cytotoxicity for one of the LCLs, CA11, treated with 5-FU (left), MFF (middle), and EPI (right) after knockdown (KD) of TNFSF13B. (B) TNFSF13B mRNA level in LCL CA11 after 24 h of transfection of TNFSF13B siRNA vs. non-targeting scrambled control (Neg.) siRNA. (C) Cytotoxicity for HL-60 cells treated with 5-FU (left), MFF (middle) and EPI (right) after knockdown (KD) of TNFSF13B. (D) TNFSF13B mRNA level in HL-60 after 24 h of transfection of TNFSF13B siRNA vs. non-targeting scrambled control (Neg.) siRNA. Values represent mean ± SEM from triplicate determinations.

### FIGURE 4.

![FIGURE 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0391/5859084/1e8de60aa113/fphar-09-00158-g004.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5859084_fphar-09-00158-g004.jpg)

Cytotoxicity assays for (A) one of the LCLs, CA11, and for (B) HL-60 that were treated with 5-FU (left), MFF (middle) and EPI (right) with incubation of human recombinant B-cell activating factor (BAFF) at different doses. Values represent mean ± SEM from triplicate determinations.

We did not observe significant changes in EPI sensitivity in both LCL and HL-60 cells after *TNFSF13B* KD or BAFF treatment, while LCLs with variant SNP genotypes were more sensitive to EPI (**Figure 2C**). This might indicate additional mechanisms beyond *TNFSF13B* expression that are related to those SNP genotypes could contribute to EPI cytotoxicity. Although the *HMMR* SNPs were *trans*-eQTLs for *TNFSF13B*, we did not observe any correlation between *HMMR* and *TNFSF13B* mRNA expression based on the mRNA data from the 300 LCLs. We also measured *TNFSF13B* mRNA level after KD of *HMMR* and *CCNG1*, but no change in *TNFSF13B* mRNA was observed (data not shown). The HMMR mRNA level was also quantified after LCLs were treated with 5-FU, EPI, and MFF at the concentration of their IC_50_s, respectively. Compared to vehicle treatment, HMMR mRNA level (relative to GAPDH) was not significantly changed after LCLs were exposed to 5-FU (50 μM) and/or EPI (0.1 μM). Although HMMR mRNA levels were slightly decreased (∼15%) in LCLs after exposure to MFF, no significant differences were observed between WT and variant LCLs (data not shown). The molecular mechanism for SNPs in *HMMR* that was associated with *TNFSF13B* expression needs to be further explored.

In summary, this series of functional genomic experiments confirmed that three SNPs in the *HMMR* gene were associated with the cytotoxicity of drugs used in the SUCCESS-A trial, an effect which is probably mediated through the expression of *TNFSF13B*.

## Discussion

In the present study, we took advantage of GWAS data that we had generated in the SUCCESS-A trial, and utilized a pathway-based approach to identify additional biomarkers and underlying biology for CIN in breast cancer patients. Specifically, SNPs in genes involved in PK and PD pathways for the chemotherapy agents that were used to treat patients, and were involved in disease risk were associated with neutropenia events. We identified additional SNPs with low *p*-value that were associated with CIN, including SNPs in *HMMR* that are *trans*-eQTLs for *TNFSF13B*, one of the genome-wide significant signals identified in the original GWAS (Supplementary Figure [S1](#SM1)). The variant *HMMR* SNP genotype was associated with decreased *TNFSF13B* expression (**Figure 1A**) and higher risk of CIN (**Table 2**). To further eliminate potential false positives results and to understand the underlying biology of these SNPs, we performed functional studies to further validate the association results. We demonstrated that LCLs carrying variant genotypes for the *HMMR* SNPs displayed decreased expression of *TNFSF13B*, and were more sensitive to treatment with 5-FU, mafosfamide and epirubicin (**Figure 2**). Further cytotoxicity assays conducted with *TNFSF13B* KD and TNFSF13B cytokine (BAFF) treatment in both LCL and HL-60 promyelocytic cells suggested that the modulation of *TNFSF13B* expression altered sensitivity to those chemotherapy agents (**Figures 3**, **4**). This study not only enabled us to discover additional SNPs associated with CIN, but also indicated an association with *TNFSF13B* expression.

The three *HMMR* SNPs that were identified by our pathway-based analysis were missed in the GWAS for CIN, probably because those SNPs did not directly contribute to CIN in breast cancer patients but rather through the expression of TNFSF13B. The *HMMR* gene encodes HMMR which is involved in cell motility. *HMMR* has been implicated in the pathogenesis of tumors like breast cancer and inflammatory disorders like osteoarthritis ([Misra et al., 2015](#B33)). We have demonstrated that SNPs within *HMMR* are associated with risk for CIN and that those SNPs are also *trans*-eQTLs for TNFSF13B expression. However, KD of HMMR did not directly affect TNFSF13B expression in the cell lines that we tested. *HMMR* has not been shown to be involved in the mechanism of treatment related neutropenia and the mechanism for how it might be associated with TNFSF13B expression is unclear and should be further explored. The association between *HMMR* SNPs and CIN in breast cancer patients needs to be further validated in an independent data set.

*TNFSF13B* encodes a cytokine known as BAFF, which is synthesized primarily by myeloid cells and is required for survival of B-cells. It is up-regulated in autoimmune diseases and various B-cell malignancies ([Shivakumar and Ansell, 2006](#B45)). Although it is involved primarily in B-cell survival, there has been increasing evidence that it may also be involved in reverse signaling in the activation of myeloid cells as well ([Chang et al., 2006](#B8); [Jeon et al., 2010](#B19)). The role of this growth factor in the activation of myeloid cells is not fully understood and there is conflicting evidence that it is not involved in myeloid cell activation ([Nys et al., 2013](#B37)). Previous studies reported that serum BAFF levels were increased in patients who developed late onset neutropenia after therapy with rituximab ([Terrier et al., 2007](#B49)). However, genetic variation associated with baseline *TNFSF13B* level has not been reported to predispose patients to CIN. Our pathway-based analysis identified variant SNP genotypes in the *HMMR* gene that were associated with decreased *TNFSF13B* expression at baseline and with higher risk of CIN. The potential role of *TNFSF13B* is not only activating B-cells but also myeloid cells, a potential mechanism for how changes in expression of this gene might predispose patients to treatment related neutropenia. The mechanism of *TNFSF13B* expression in association with neutropenia is unclear and needs to be elucidated.

In addition to the *HMMR* SNPs, the pathway-based analysis also identified other SNPs that were significantly associated with CIN in breast cancer patients. However, those SNPs had a very low MAF (≤0.051) (**Table 2**), thus we were unable to perform functional validation for those SNPs using our 300 LCLs. Those SNPs were included in our pathway-based analysis because they are located in genes which are involved in the mTOR, PI3K, P53, and checkpoint pathways (**Table 2**), pathways have been found to be aberrant in breast cancer ([The Cancer Genome Atlas Network, 2012](#B50)). A recent study reported that common genetic variants for breast cancer risk were negatively associated with CIN ([Dorling et al., 2016](#B10)). Although that study used different criteria from ours for selection of SNPs/genes that correlated with breast cancer risk, they also found that the P53 signaling pathway was the most enriched pathway associated with CIN ([Dorling et al., 2016](#B10)). Using a candidate SNPs/genes approach, SNPs in the *TP53* and *MDM2* genes, two most important genes in the P53 signaling pathway, were found to be associated with CIN in Japanese breast cancer patients (*n* = 216) ([Okishiro et al., 2012](#B38)). Of note, the *HMMR* SNPs were included in our pathway-based analysis because those SNPs are located in a gene that is involved in the P53 signaling pathway (**Table 2**). It might be interesting to investigate the relationship between the P53 pathway, TNFSF13B and CIN in future studies.

Several studies have focused on the SNPs/genes involved in the cyclophosphamide PK pathway for associations with CIN in breast cancer patients. These studies reported that SNPs in *GSTP1* ([Yao et al., 2010](#B62)), *ABCC4* ([Low et al., 2009](#B28)) and *CYP3A5* ([Tang et al., 2013](#B48)) were associated with CIN in breast cancer patients. These studies did not include SNPs in *CYP2C8* while our analysis identified a SNP in *CYP2C8* that was associated with CIN (**Table 2**). Together, those studies indicate that genetic polymorphisms in the cyclophosphamide PK pathway may be associated with CIN in breast cancer patients. In addition to anti-cancer drugs, neutropenia can be also induced by other drugs such as clozapine, a dibenzodiazepine antipsychotic agent used for the pharmacotherapy of schizophrenia ([Alvir et al., 1993](#B3); [Atkin et al., 1996](#B5)). Pharmacogenomic studies on clozapine-induced agranulocytosis or neutropenia have often identified SNPs/genes in the clozapine PK pathway and immune response genes ([Athanasiou et al., 2011](#B4); [Goldstein et al., 2014](#B13); [Saito et al., 2016](#B44); [Legge et al., 2017](#B22); [van der Weide et al., 2017](#B54)), a similar scenario to pharmacogenomic studies on CIN in breast cancer patients.

In summary, using a pathway-based approach for analyzing GWAS data, we identified additional genetic makers for CIN in breast cancer patients, including SNPs in the HMMR gene. Further functional genomic studies indicated that *HMMR* SNP-dependent sensitivity to chemotherapy is probably dependent on the expression of *TNFSF13B*.

## Author Contributions

BB and DL conducted the experiments and analyzed the data, and drafted the manuscript. KK did the pathway analysis, genes/SNPs selection, and association study with clinical phenotype. MR, AH, MB, BR, WJ, and PF recruited patients and measured the clinical phenotypes. RW and LW were responsible for the concept generation and entire study design.

## Conflict of Interest Statement

BB is a Merck & Co., Inc. employee. The other authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.

## Supplementary Material

The Supplementary Material for this article can be found online at: [https://www.frontiersin.org/articles/10.3389/fphar.2018.00158/full#supplementary-material](https://www.frontiersin.org/articles/10.3389/fphar.2018.00158/full#supplementary-material)

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Aapro M. S., Cameron D. A., Pettengell R., Bohlius J., Crawford J., Ellis M., et al. (2006). EORTC guidelines for the use of granulocyte-colony stimulating factor to reduce the incidence of chemotherapy-induced febrile neutropenia in adult patients with lymphomas and solid tumours. Eur. J. Cancer 42 2433–2453. 10.1016/j.ejca.2006.05.002  [DOI](https://doi.org/10.1016/j.ejca.2006.05.002) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16750358/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur.%20J.%20Cancer&title=EORTC%20guidelines%20for%20the%20use%20of%20granulocyte-colony%20stimulating%20factor%20to%20reduce%20the%20incidence%20of%20chemotherapy-induced%20febrile%20neutropenia%20in%20adult%20patients%20with%20lymphomas%20and%20solid%20tumours.&author=M.%20S.%20Aapro&author=D.%20A.%20Cameron&author=R.%20Pettengell&author=J.%20Bohlius&author=J.%20Crawford&volume=42&publication_year=2006&pages=2433-2453&pmid=16750358&doi=10.1016/j.ejca.2006.05.002&)

2. Agiro A., Ma Q., Acheson A. K., Wu S.-J., Patt D. A., Barron J. J., et al. (2016). Risk of neutropenia-related hospitalization in patients who received colony-stimulating factors with chemotherapy for breast cancer. J. Clin. Oncol. 34 3872–3879. 10.1200/jco.2016.67.2899  [DOI](https://doi.org/10.1200/jco.2016.67.2899) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27646945/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Oncol.&title=Risk%20of%20neutropenia-related%20hospitalization%20in%20patients%20who%20received%20colony-stimulating%20factors%20with%20chemotherapy%20for%20breast%20cancer.&author=A.%20Agiro&author=Q.%20Ma&author=A.%20K.%20Acheson&author=S.-J.%20Wu&author=D.%20A.%20Patt&volume=34&publication_year=2016&pages=3872-3879&pmid=27646945&doi=10.1200/jco.2016.67.2899&)

3. Alvir J. M., Lieberman J. A., Safferman A. Z., Schwimmer J. L., Schaaf J. A. (1993). Clozapine-induced agranulocytosis. Incidence and risk factors in the United States. N. Engl. J. Med. 329 162–167. 10.1056/NEJM199307153290303  [DOI](https://doi.org/10.1056/NEJM199307153290303) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8515788/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N.%20Engl.%20J.%20Med.&title=Clozapine-induced%20agranulocytosis.%20Incidence%20and%20risk%20factors%20in%20the%20United%20States.&author=J.%20M.%20Alvir&author=J.%20A.%20Lieberman&author=A.%20Z.%20Safferman&author=J.%20L.%20Schwimmer&author=J.%20A.%20Schaaf&volume=329&publication_year=1993&pages=162-167&pmid=8515788&doi=10.1056/NEJM199307153290303&)

4. Athanasiou M. C., Dettling M., Cascorbi I., Mosyagin I., Salisbury B. A., Pierz K. A., et al. (2011). Candidate gene analysis identifies a polymorphism in HLA-DQB1 associated with clozapine-induced agranulocytosis. J. Clin. Psychiatry 72 458–463. 10.4088/JCP.09m05527yel  [DOI](https://doi.org/10.4088/JCP.09m05527yel) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20868635/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Psychiatry&title=Candidate%20gene%20analysis%20identifies%20a%20polymorphism%20in%20HLA-DQB1%20associated%20with%20clozapine-induced%20agranulocytosis.&author=M.%20C.%20Athanasiou&author=M.%20Dettling&author=I.%20Cascorbi&author=I.%20Mosyagin&author=B.%20A.%20Salisbury&volume=72&publication_year=2011&pages=458-463&pmid=20868635&doi=10.4088/JCP.09m05527yel&)

5. Atkin K., Kendall F., Gould D., Freeman H., Liberman J., O’Sullivan D. (1996). Neutropenia and agranulocytosis in patients receiving clozapine in the UK and Ireland. Br. J. Psychiatry 169 483–488. 10.1192/bjp.169.4.483  [DOI](https://doi.org/10.1192/bjp.169.4.483) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8894200/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br.%20J.%20Psychiatry&title=Neutropenia%20and%20agranulocytosis%20in%20patients%20receiving%20clozapine%20in%20the%20UK%20and%20Ireland.&author=K.%20Atkin&author=F.%20Kendall&author=D.%20Gould&author=H.%20Freeman&author=J.%20Liberman&volume=169&publication_year=1996&pages=483-488&pmid=8894200&doi=10.1192/bjp.169.4.483&)

6. Ballard D., Abraham C., Cho J., Zhao H. (2010). Pathway analysis comparison using Crohn’s disease genome wide association studies. BMC Med. Genomics 3:25. 10.1186/1755-8794-3-25  [DOI](https://doi.org/10.1186/1755-8794-3-25) | [PMC free article](/articles/PMC2908056/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20584322/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Med.%20Genomics&title=Pathway%20analysis%20comparison%20using%20Crohn%E2%80%99s%20disease%20genome%20wide%20association%20studies.&author=D.%20Ballard&author=C.%20Abraham&author=J.%20Cho&author=H.%20Zhao&volume=3&issue=25&publication_year=2010&pmid=20584322&doi=10.1186/1755-8794-3-25&)

7. Baranzini S. E., Galwey N. W., Wang J., Khankhanian P., Lindberg R., Pelletier D., et al. (2009). Pathway and network-based analysis of genome-wide association studies in multiple sclerosis. Hum. Mol. Genet. 18 2078–2090. 10.1093/hmg/ddp120  [DOI](https://doi.org/10.1093/hmg/ddp120) | [PMC free article](/articles/PMC2678928/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19286671/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum.%20Mol.%20Genet.&title=Pathway%20and%20network-based%20analysis%20of%20genome-wide%20association%20studies%20in%20multiple%20sclerosis.&author=S.%20E.%20Baranzini&author=N.%20W.%20Galwey&author=J.%20Wang&author=P.%20Khankhanian&author=R.%20Lindberg&volume=18&publication_year=2009&pages=2078-2090&pmid=19286671&doi=10.1093/hmg/ddp120&)

8. Chang S. K., Arendt B. K., Darce J. R., Wu X., Jelinek D. F. (2006). A role for BLyS in the activation of innate immune cells. Blood 108 2687–2694. 10.1182/blood-2005-12-017319  [DOI](https://doi.org/10.1182/blood-2005-12-017319) | [PMC free article](/articles/PMC1895592/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16825497/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=A%20role%20for%20BLyS%20in%20the%20activation%20of%20innate%20immune%20cells.&author=S.%20K.%20Chang&author=B.%20K.%20Arendt&author=J.%20R.%20Darce&author=X.%20Wu&author=D.%20F.%20Jelinek&volume=108&publication_year=2006&pages=2687-2694&pmid=16825497&doi=10.1182/blood-2005-12-017319&)

9. Do T., Medhekar R., Bhat R., Chen H., Niravath P., Trivedi M. V. (2015). The risk of febrile neutropenia and need for G-CSF primary prophylaxis with the docetaxel and cyclophosphamide regimen in early-stage breast cancer patients: a meta-analysis. Breast Cancer Res. Treat. 153 591–597. 10.1007/s10549-015-3531-z  [DOI](https://doi.org/10.1007/s10549-015-3531-z) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26337685/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Breast%20Cancer%20Res.%20Treat.&title=The%20risk%20of%20febrile%20neutropenia%20and%20need%20for%20G-CSF%20primary%20prophylaxis%20with%20the%20docetaxel%20and%20cyclophosphamide%20regimen%20in%20early-stage%20breast%20cancer%20patients:%20a%20meta-analysis.&author=T.%20Do&author=R.%20Medhekar&author=R.%20Bhat&author=H.%20Chen&author=P.%20Niravath&volume=153&publication_year=2015&pages=591-597&pmid=26337685&doi=10.1007/s10549-015-3531-z&)

10. Dorling L., Kar S., Michailidou K., Hiller L., Vallier A. L., Ingle S., et al. (2016). The relationship between common genetic markers of breast cancer risk and chemotherapy-induced toxicity: a case-control study. PLoS One 11:e0158984. 10.1371/journal.pone.0158984  [DOI](https://doi.org/10.1371/journal.pone.0158984) | [PMC free article](/articles/PMC4938564/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27392074/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=The%20relationship%20between%20common%20genetic%20markers%20of%20breast%20cancer%20risk%20and%20chemotherapy-induced%20toxicity:%20a%20case-control%20study.&author=L.%20Dorling&author=S.%20Kar&author=K.%20Michailidou&author=L.%20Hiller&author=A.%20L.%20Vallier&volume=11&issue=e0158984&publication_year=2016&pmid=27392074&doi=10.1371/journal.pone.0158984&)

11. Ferlay J., Soerjomataram I., Dikshit R., Eser S., Mathers C., Rebelo M., et al. (2015). Cancer incidence and mortality worldwide: sources, methods and major patterns in GLOBOCAN 2012. Int. J. Cancer 136 E359–E386. 10.1002/ijc.29210  [DOI](https://doi.org/10.1002/ijc.29210) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25220842/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Cancer&title=Cancer%20incidence%20and%20mortality%20worldwide:%20sources,%20methods%20and%20major%20patterns%20in%20GLOBOCAN%202012.&author=J.%20Ferlay&author=I.%20Soerjomataram&author=R.%20Dikshit&author=S.%20Eser&author=C.%20Mathers&volume=136&publication_year=2015&pages=E359-E386&pmid=25220842&doi=10.1002/ijc.29210&)

12. Fontanella C., Bolzonello S., Lederer B., Aprile G. (2014). Management of breast cancer patients with chemotherapy-induced neutropenia or febrile neutropenia. Breast Care 9 239–245. 10.1159/000366466  [DOI](https://doi.org/10.1159/000366466) | [PMC free article](/articles/PMC4209284/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25404882/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Breast%20Care&title=Management%20of%20breast%20cancer%20patients%20with%20chemotherapy-induced%20neutropenia%20or%20febrile%20neutropenia.&author=C.%20Fontanella&author=S.%20Bolzonello&author=B.%20Lederer&author=G.%20Aprile&volume=9&publication_year=2014&pages=239-245&pmid=25404882&doi=10.1159/000366466&)

13. Goldstein J. I., Jarskog L. F., Hilliard C., Alfirevic A., Duncan L., Fourches D., et al. (2014). Clozapine-induced agranulocytosis is associated with rare HLA-DQB1 and HLA-B alleles. Nat. Commun. 5:4757. 10.1038/ncomms5757  [DOI](https://doi.org/10.1038/ncomms5757) | [PMC free article](/articles/PMC4155508/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25187353/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun.&title=Clozapine-induced%20agranulocytosis%20is%20associated%20with%20rare%20HLA-DQB1%20and%20HLA-B%20alleles.&author=J.%20I.%20Goldstein&author=L.%20F.%20Jarskog&author=C.%20Hilliard&author=A.%20Alfirevic&author=L.%20Duncan&volume=5&issue=4757&publication_year=2014&pmid=25187353&doi=10.1038/ncomms5757&)

14. Holmans P., Green E. K., Pahwa J. S., Ferreira M. A., Purcell S. M., Sklar P., et al. (2009). Gene ontology analysis of GWA study data sets provides insights into the biology of bipolar disorder. Am. J. Hum. Genet. 85 13–24. 10.1016/j.ajhg.2009.05.011  [DOI](https://doi.org/10.1016/j.ajhg.2009.05.011) | [PMC free article](/articles/PMC2706963/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19539887/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Hum.%20Genet.&title=Gene%20ontology%20analysis%20of%20GWA%20study%20data%20sets%20provides%20insights%20into%20the%20biology%20of%20bipolar%20disorder.&author=P.%20Holmans&author=E.%20K.%20Green&author=J.%20S.%20Pahwa&author=M.%20A.%20Ferreira&author=S.%20M.%20Purcell&volume=85&publication_year=2009&pages=13-24&pmid=19539887&doi=10.1016/j.ajhg.2009.05.011&)

15. Holmes F. A., Jones S. E., O’Shaughnessy J., Vukelja S., George T., Savin M., et al. (2002). Comparable efficacy and safety profiles of once-per-cycle pegfilgrastim and daily injection filgrastim in chemotherapy-induced neutropenia: a multicenter dose-finding study in women with breast cancer. Ann. Oncol. 13 903–909. 10.1093/annonc/mdf130  [DOI](https://doi.org/10.1093/annonc/mdf130) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12123336/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann.%20Oncol.&title=Comparable%20efficacy%20and%20safety%20profiles%20of%20once-per-cycle%20pegfilgrastim%20and%20daily%20injection%20filgrastim%20in%20chemotherapy-induced%20neutropenia:%20a%20multicenter%20dose-finding%20study%20in%20women%20with%20breast%20cancer.&author=F.%20A.%20Holmes&author=S.%20E.%20Jones&author=J.%20O%E2%80%99Shaughnessy&author=S.%20Vukelja&author=T.%20George&volume=13&publication_year=2002&pages=903-909&pmid=12123336&doi=10.1093/annonc/mdf130&)

16. Hou J., Bhinge K., Wang L. (2014). “The effect of ERRFI1 as a novel AKT regulator on cell proliferation and response to therapy is cell context dependent,” in Proceedings of the 105th Annual Meeting of the American Association for Cancer Research (San Diego, CA: Cancer Research; ).  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Proceedings%20of%20the%20105th%20Annual%20Meeting%20of%20the%20American%20Association%20for%20Cancer%20Research&author=J.%20Hou&author=K.%20Bhinge&author=L.%20Wang&publication_year=2014&)

17. Howie B., Fuchsberger C., Stephens M., Marchini J., Abecasis G. R. (2012). Fast and accurate genotype imputation in genome-wide association studies through pre-phasing. Nat. Genet. 44 955–959. 10.1038/ng.2354  [DOI](https://doi.org/10.1038/ng.2354) | [PMC free article](/articles/PMC3696580/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22820512/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet.&title=Fast%20and%20accurate%20genotype%20imputation%20in%20genome-wide%20association%20studies%20through%20pre-phasing.&author=B.%20Howie&author=C.%20Fuchsberger&author=M.%20Stephens&author=J.%20Marchini&author=G.%20R.%20Abecasis&volume=44&publication_year=2012&pages=955-959&pmid=22820512&doi=10.1038/ng.2354&)

18. Ingle J. N., Schaid D. J., Goss P. E., Liu M., Mushiroda T., Chapman J. A., et al. (2010). Genome-wide associations and functional genomic studies of musculoskeletal adverse events in women receiving aromatase inhibitors. J. Clin. Oncol. 28 4674–4682. 10.1200/JCO.2010.28.5064  [DOI](https://doi.org/10.1200/JCO.2010.28.5064) | [PMC free article](/articles/PMC3020700/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20876420/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Oncol.&title=Genome-wide%20associations%20and%20functional%20genomic%20studies%20of%20musculoskeletal%20adverse%20events%20in%20women%20receiving%20aromatase%20inhibitors.&author=J.%20N.%20Ingle&author=D.%20J.%20Schaid&author=P.%20E.%20Goss&author=M.%20Liu&author=T.%20Mushiroda&volume=28&publication_year=2010&pages=4674-4682&pmid=20876420&doi=10.1200/JCO.2010.28.5064&)

19. Jeon S. T., Kim W. J., Lee S. M., Lee M. Y., Park S. B., Lee S. H., et al. (2010). Reverse signaling through BAFF differentially regulates the expression of inflammatory mediators and cytoskeletal movements in THP-1 cells. Immunol. Cell Biol. 88 148–156. 10.1038/icb.2009.75  [DOI](https://doi.org/10.1038/icb.2009.75) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19841639/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Immunol.%20Cell%20Biol.&title=Reverse%20signaling%20through%20BAFF%20differentially%20regulates%20the%20expression%20of%20inflammatory%20mediators%20and%20cytoskeletal%20movements%20in%20THP-1%20cells.&author=S.%20T.%20Jeon&author=W.%20J.%20Kim&author=S.%20M.%20Lee&author=M.%20Y.%20Lee&author=S.%20B.%20Park&volume=88&publication_year=2010&pages=148-156&pmid=19841639&doi=10.1038/icb.2009.75&)

20. Jia P., Wang L., Meltzer H. Y., Zhao Z. (2010). Common variants conferring risk of schizophrenia: a pathway analysis of GWAS data. Schizophr. Res. 122 38–42. 10.1016/j.schres.2010.07.001  [DOI](https://doi.org/10.1016/j.schres.2010.07.001) | [PMC free article](/articles/PMC2933424/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20659789/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Schizophr.%20Res.&title=Common%20variants%20conferring%20risk%20of%20schizophrenia:%20a%20pathway%20analysis%20of%20GWAS%20data.&author=P.%20Jia&author=L.%20Wang&author=H.%20Y.%20Meltzer&author=Z.%20Zhao&volume=122&publication_year=2010&pages=38-42&pmid=20659789&doi=10.1016/j.schres.2010.07.001&)

21. Kanehisa M., Goto S. (2000). KEGG: kyoto encyclopedia of genes and genomes. Nucleic Acids Res. 28 27–30. 10.1093/nar/28.1.27  [DOI](https://doi.org/10.1093/nar/28.1.27) | [PMC free article](/articles/PMC102409/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10592173/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&title=KEGG:%20kyoto%20encyclopedia%20of%20genes%20and%20genomes.&author=M.%20Kanehisa&author=S.%20Goto&volume=28&publication_year=2000&pages=27-30&pmid=10592173&doi=10.1093/nar/28.1.27&)

22. Legge S. E., Hamshere M. L., Ripke S., Pardinas A. F., Goldstein J. I., Rees E., et al. (2017). Genome-wide common and rare variant analysis provides novel insights into clozapine-associated neutropenia. Mol. Psychiatry 22 1502–1508. 10.1038/mp.2016.97  [DOI](https://doi.org/10.1038/mp.2016.97) | [PMC free article](/articles/PMC5065090/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27400856/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol.%20Psychiatry&title=Genome-wide%20common%20and%20rare%20variant%20analysis%20provides%20novel%20insights%20into%20clozapine-associated%20neutropenia.&author=S.%20E.%20Legge&author=M.%20L.%20Hamshere&author=S.%20Ripke&author=A.%20F.%20Pardinas&author=J.%20I.%20Goldstein&volume=22&publication_year=2017&pages=1502-1508&pmid=27400856&doi=10.1038/mp.2016.97&)

23. Li L., Fridley B., Kalari K., Jenkins G., Batzler A., Safgren S., et al. (2008). Gemcitabine and cytosine arabinoside cytotoxicity: association with lymphoblastoid cell expression. Cancer Res. 68 7050–7058. 10.1158/0008-5472.CAN-08-0405  [DOI](https://doi.org/10.1158/0008-5472.CAN-08-0405) | [PMC free article](/articles/PMC2562356/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18757419/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res.&title=Gemcitabine%20and%20cytosine%20arabinoside%20cytotoxicity:%20association%20with%20lymphoblastoid%20cell%20expression.&author=L.%20Li&author=B.%20Fridley&author=K.%20Kalari&author=G.%20Jenkins&author=A.%20Batzler&volume=68&publication_year=2008&pages=7050-7058&pmid=18757419&doi=10.1158/0008-5472.CAN-08-0405&)

24. Li L., Fridley B. L., Kalari K., Niu N., Jenkins G., Batzler A., et al. (2014). Discovery of genetic biomarkers contributing to variation in drug response of cytidine analogues using human lymphoblastoid cell lines. BMC Genomics 15:93. 10.1186/1471-2164-15-93  [DOI](https://doi.org/10.1186/1471-2164-15-93) | [PMC free article](/articles/PMC3930546/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24483146/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Genomics&title=Discovery%20of%20genetic%20biomarkers%20contributing%20to%20variation%20in%20drug%20response%20of%20cytidine%20analogues%20using%20human%20lymphoblastoid%20cell%20lines.&author=L.%20Li&author=B.%20L.%20Fridley&author=K.%20Kalari&author=N.%20Niu&author=G.%20Jenkins&volume=15&issue=93&publication_year=2014&pmid=24483146&doi=10.1186/1471-2164-15-93&)

25. Link B. K., Budd G. T., Scott S., Dickman E., Paul D., Lawless G., et al. (2001). Delivering adjuvant chemotherapy to women with early-stage breast carcinoma: current patterns of care. Cancer 92 1354–1367.  [DOI](https://doi.org/10.1002/1097-0142(20010915)92:6&#x0003c;1354::AID-CNCR1458&#x0003e;3.0.CO;2-P) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11745211/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer&title=Delivering%20adjuvant%20chemotherapy%20to%20women%20with%20early-stage%20breast%20carcinoma:%20current%20patterns%20of%20care.&author=B.%20K.%20Link&author=G.%20T.%20Budd&author=S.%20Scott&author=E.%20Dickman&author=D.%20Paul&volume=92&publication_year=2001&pages=1354-1367&pmid=11745211&doi=10.1002/1097-0142(20010915)92:6&#x0003c;1354::AID-CNCR1458&#x0003e;3.0.CO;2-P&)

26. Liu D., Ho M. F., Schaid D. J., Scherer S. E., Kalari K., Liu M., et al. (2017). Breast cancer chemoprevention pharmacogenomics: deep sequencing and functional genomics of the ZNF423 and CTSO genes. NPJ Breast Cancer 3:30. 10.1038/s41523-017-0036-4  [DOI](https://doi.org/10.1038/s41523-017-0036-4) | [PMC free article](/articles/PMC5566425/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28856246/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=NPJ%20Breast%20Cancer&title=Breast%20cancer%20chemoprevention%20pharmacogenomics:%20deep%20sequencing%20and%20functional%20genomics%20of%20the%20ZNF423%20and%20CTSO%20genes.&author=D.%20Liu&author=M.%20F.%20Ho&author=D.%20J.%20Schaid&author=S.%20E.%20Scherer&author=K.%20Kalari&volume=3&issue=30&publication_year=2017&pmid=28856246&doi=10.1038/s41523-017-0036-4&)

27. Liu M., Goss P. E., Ingle J. N., Kubo M., Furukawa Y., Batzler A., et al. (2014). Aromatase inhibitor-associated bone fractures: a case-cohort GWAS and functional genomics. Mol. Endocrinol. 28 1740–1751. 10.1210/me.2014-1147  [DOI](https://doi.org/10.1210/me.2014-1147) | [PMC free article](/articles/PMC4179631/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25148458/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol.%20Endocrinol.&title=Aromatase%20inhibitor-associated%20bone%20fractures:%20a%20case-cohort%20GWAS%20and%20functional%20genomics.&author=M.%20Liu&author=P.%20E.%20Goss&author=J.%20N.%20Ingle&author=M.%20Kubo&author=Y.%20Furukawa&volume=28&publication_year=2014&pages=1740-1751&pmid=25148458&doi=10.1210/me.2014-1147&)

28. Low S. K., Kiyotani K., Mushiroda T., Daigo Y., Nakamura Y., Zembutsu H. (2009). Association study of genetic polymorphism in ABCC4 with cyclophosphamide-induced adverse drug reactions in breast cancer patients. J. Hum. Genet. 54 564–571. 10.1038/jhg.2009.79  [DOI](https://doi.org/10.1038/jhg.2009.79) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19696793/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Hum.%20Genet.&title=Association%20study%20of%20genetic%20polymorphism%20in%20ABCC4%20with%20cyclophosphamide-induced%20adverse%20drug%20reactions%20in%20breast%20cancer%20patients.&author=S.%20K.%20Low&author=K.%20Kiyotani&author=T.%20Mushiroda&author=Y.%20Daigo&author=Y.%20Nakamura&volume=54&publication_year=2009&pages=564-571&pmid=19696793&doi=10.1038/jhg.2009.79&)

29. Lyman G. H., Dale D. C., Legg J. C., Abella E., Morrow P. K., Whittaker S., et al. (2015). Assessing patients’ risk of febrile neutropenia: is there a correlation between physician-assessed risk and model-predicted risk? Cancer Med. 4 1153–1160. 10.1002/cam4.454  [DOI](https://doi.org/10.1002/cam4.454) | [PMC free article](/articles/PMC4559026/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25810005/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Med.&title=Assessing%20patients%E2%80%99%20risk%20of%20febrile%20neutropenia:%20is%20there%20a%20correlation%20between%20physician-assessed%20risk%20and%20model-predicted%20risk?&author=G.%20H.%20Lyman&author=D.%20C.%20Dale&author=J.%20C.%20Legg&author=E.%20Abella&author=P.%20K.%20Morrow&volume=4&publication_year=2015&pages=1153-1160&pmid=25810005&doi=10.1002/cam4.454&)

30. Mackay F., Browning J. L. (2002). BAFF: a fundamental survival factor for B cells. Nat. Rev. Immunol. 2 465–475. 10.1038/nri844  [DOI](https://doi.org/10.1038/nri844) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12094221/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Immunol.&title=BAFF:%20a%20fundamental%20survival%20factor%20for%20B%20cells.&author=F.%20Mackay&author=J.%20L.%20Browning&volume=2&publication_year=2002&pages=465-475&pmid=12094221&doi=10.1038/nri844&)

31. Matimba A., Li F., Livshits A., Cartwright C. S., Scully S., Fridley B. L., et al. (2014). Thiopurine pharmacogenomics: association of SNPs with clinical response and functional validation of candidate genes. Pharmacogenomics 15 433–447. 10.2217/pgs.13.226  [DOI](https://doi.org/10.2217/pgs.13.226) | [PMC free article](/articles/PMC4027966/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24624911/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Thiopurine%20pharmacogenomics:%20association%20of%20SNPs%20with%20clinical%20response%20and%20functional%20validation%20of%20candidate%20genes.&author=A.%20Matimba&author=F.%20Li&author=A.%20Livshits&author=C.%20S.%20Cartwright&author=S.%20Scully&volume=15&publication_year=2014&pages=433-447&pmid=24624911&doi=10.2217/pgs.13.226&)

32. Mazur L., Opydo-Chanek M., Stojak M., Wojcieszek K. (2012). Mafosfamide as a new anticancer agent: preclinical investigations and clinical trials. Anticancer Res. 32 2783–2789.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/22753738/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Anticancer%20Res.&title=Mafosfamide%20as%20a%20new%20anticancer%20agent:%20preclinical%20investigations%20and%20clinical%20trials.&author=L.%20Mazur&author=M.%20Opydo-Chanek&author=M.%20Stojak&author=K.%20Wojcieszek&volume=32&publication_year=2012&pages=2783-2789&pmid=22753738&)

33. Misra S., Hascall V. C., Markwald R. R., Ghatak S. (2015). Interactions between hyaluronan and its receptors (CD44, RHAMM) regulate the activities of inflammation and cancer. Front. Immunol. 6:201. 10.3389/fimmu.2015.00201  [DOI](https://doi.org/10.3389/fimmu.2015.00201) | [PMC free article](/articles/PMC4422082/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25999946/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front.%20Immunol.&title=Interactions%20between%20hyaluronan%20and%20its%20receptors%20(CD44,%20RHAMM)%20regulate%20the%20activities%20of%20inflammation%20and%20cancer.&author=S.%20Misra&author=V.%20C.%20Hascall&author=R.%20R.%20Markwald&author=S.%20Ghatak&volume=6&issue=201&publication_year=2015&pmid=25999946&doi=10.3389/fimmu.2015.00201&)

34. Motsinger-Reif A. A., Jorgenson E., Relling M. V., Kroetz D. L., Weinshilboum R., Cox N. J., et al. (2013). Genome-wide association studies in pharmacogenomics: successes and lessons. Pharmacogenet. Genomics 23 383–394. 10.1097/FPC.0b013e32833d7b45  [DOI](https://doi.org/10.1097/FPC.0b013e32833d7b45) | [PMC free article](/articles/PMC3003940/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20639796/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet.%20Genomics&title=Genome-wide%20association%20studies%20in%20pharmacogenomics:%20successes%20and%20lessons.&author=A.%20A.%20Motsinger-Reif&author=E.%20Jorgenson&author=M.%20V.%20Relling&author=D.%20L.%20Kroetz&author=R.%20Weinshilboum&volume=23&publication_year=2013&pages=383-394&pmid=20639796&doi=10.1097/FPC.0b013e32833d7b45&)

35. Nielsen T. O., Parker J. S., Leung S., Voduc D., Ebbert M., Vickery T., et al. (2010). A comparison of PAM50 intrinsic subtyping with immunohistochemistry and clinical prognostic factors in tamoxifen-treated estrogen receptor-positive breast cancer. Clin. Cancer Res. 16 5222–5232. 10.1158/1078-0432.CCR-10-1282  [DOI](https://doi.org/10.1158/1078-0432.CCR-10-1282) | [PMC free article](/articles/PMC2970720/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20837693/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Cancer%20Res.&title=A%20comparison%20of%20PAM50%20intrinsic%20subtyping%20with%20immunohistochemistry%20and%20clinical%20prognostic%20factors%20in%20tamoxifen-treated%20estrogen%20receptor-positive%20breast%20cancer.&author=T.%20O.%20Nielsen&author=J.%20S.%20Parker&author=S.%20Leung&author=D.%20Voduc&author=M.%20Ebbert&volume=16&publication_year=2010&pages=5222-5232&pmid=20837693&doi=10.1158/1078-0432.CCR-10-1282&)

36. Niu N., Qin Y., Fridley B. L., Hou J., Kalari K. R., Zhu M., et al. (2010). Radiation pharmacogenomics: a genome-wide association approach to identify radiation response biomarkers using human lymphoblastoid cell lines. Genome Res. 20 1482–1492. 10.1101/gr.107672.110  [DOI](https://doi.org/10.1101/gr.107672.110) | [PMC free article](/articles/PMC2963812/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20923822/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Res.&title=Radiation%20pharmacogenomics:%20a%20genome-wide%20association%20approach%20to%20identify%20radiation%20response%20biomarkers%20using%20human%20lymphoblastoid%20cell%20lines.&author=N.%20Niu&author=Y.%20Qin&author=B.%20L.%20Fridley&author=J.%20Hou&author=K.%20R.%20Kalari&volume=20&publication_year=2010&pages=1482-1492&pmid=20923822&doi=10.1101/gr.107672.110&)

37. Nys J., Smulski C. R., Tardivel A., Willen L., Kowalczyk C., Donze O., et al. (2013). No evidence that soluble TACI induces signalling via membrane-expressed BAFF and APRIL in myeloid cells. PLoS One 8:e61350. 10.1371/journal.pone.0061350  [DOI](https://doi.org/10.1371/journal.pone.0061350) | [PMC free article](/articles/PMC3631189/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23620746/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=No%20evidence%20that%20soluble%20TACI%20induces%20signalling%20via%20membrane-expressed%20BAFF%20and%20APRIL%20in%20myeloid%20cells.&author=J.%20Nys&author=C.%20R.%20Smulski&author=A.%20Tardivel&author=L.%20Willen&author=C.%20Kowalczyk&volume=8&issue=e61350&publication_year=2013&pmid=23620746&doi=10.1371/journal.pone.0061350&)

38. Okishiro M., Kim S. J., Tsunashima R., Nakayama T., Shimazu K., Shimomura A., et al. (2012). MDM2 SNP309 and TP53 R72P associated with severe and febrile neutropenia in breast cancer patients treated with 5-FU/epirubicin/cyclophosphamide. Breast Cancer Res. Treat. 132 947–953. 10.1007/s10549-011-1637-5  [DOI](https://doi.org/10.1007/s10549-011-1637-5) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21706156/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Breast%20Cancer%20Res.%20Treat.&title=MDM2%20SNP309%20and%20TP53%20R72P%20associated%20with%20severe%20and%20febrile%20neutropenia%20in%20breast%20cancer%20patients%20treated%20with%205-FU/epirubicin/cyclophosphamide.&author=M.%20Okishiro&author=S.%20J.%20Kim&author=R.%20Tsunashima&author=T.%20Nakayama&author=K.%20Shimazu&volume=132&publication_year=2012&pages=947-953&pmid=21706156&doi=10.1007/s10549-011-1637-5&)

39. Pei H., Li L., Fridley B. L., Jenkins G. D., Kalari K. R., Lingle W., et al. (2009). FKBP51 affects cancer cell response to chemotherapy by negatively regulating Akt. Cancer Cell 16 259–266. 10.1016/j.ccr.2009.07.016  [DOI](https://doi.org/10.1016/j.ccr.2009.07.016) | [PMC free article](/articles/PMC2755578/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19732725/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Cell&title=FKBP51%20affects%20cancer%20cell%20response%20to%20chemotherapy%20by%20negatively%20regulating%20Akt.&author=H.%20Pei&author=L.%20Li&author=B.%20L.%20Fridley&author=G.%20D.%20Jenkins&author=K.%20R.%20Kalari&volume=16&publication_year=2009&pages=259-266&pmid=19732725&doi=10.1016/j.ccr.2009.07.016&)

40. Pinto N., Cohn S. L., Dolan M. E. (2012). Using germline genomics to individualize pediatric cancer treatments. Clin. Cancer Res. 18 2791–2800. 10.1158/1078-0432.CCR-11-1938  [DOI](https://doi.org/10.1158/1078-0432.CCR-11-1938) | [PMC free article](/articles/PMC3354919/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22589487/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Cancer%20Res.&title=Using%20germline%20genomics%20to%20individualize%20pediatric%20cancer%20treatments.&author=N.%20Pinto&author=S.%20L.%20Cohn&author=M.%20E.%20Dolan&volume=18&publication_year=2012&pages=2791-2800&pmid=22589487&doi=10.1158/1078-0432.CCR-11-1938&)

41. Price A. L., Patterson N. J., Plenge R. M., Weinblatt M. E., Shadick N. A., Reich D. (2006). Principal components analysis corrects for stratification in genome-wide association studies. Nat. Genet. 38 904–909. 10.1038/ng1847  [DOI](https://doi.org/10.1038/ng1847) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16862161/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet.&title=Principal%20components%20analysis%20corrects%20for%20stratification%20in%20genome-wide%20association%20studies.&author=A.%20L.%20Price&author=N.%20J.%20Patterson&author=R.%20M.%20Plenge&author=M.%20E.%20Weinblatt&author=N.%20A.%20Shadick&volume=38&publication_year=2006&pages=904-909&pmid=16862161&doi=10.1038/ng1847&)

42. Rack B., Schindlbeck C., Juckstock J., Andergassen U., Hepp P., Zwingers T., et al. (2014). Circulating tumor cells predict survival in early average-to-high risk breast cancer patients. J. Natl. Cancer Inst. 106:dju066. 10.1093/jnci/dju066  [DOI](https://doi.org/10.1093/jnci/dju066) | [PMC free article](/articles/PMC4112925/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24832787/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Natl.%20Cancer%20Inst.&title=Circulating%20tumor%20cells%20predict%20survival%20in%20early%20average-to-high%20risk%20breast%20cancer%20patients.&author=B.%20Rack&author=C.%20Schindlbeck&author=J.%20Juckstock&author=U.%20Andergassen&author=P.%20Hepp&volume=106&issue=dju066&publication_year=2014&pmid=24832787&doi=10.1093/jnci/dju066&)

43. Ramanan V. K., Shen L., Moore J. H., Saykin A. J. (2012). Pathway analysis of genomic data: concepts, methods, and prospects for future development. Trends Genet. 28 323–332. 10.1016/j.tig.2012.03.004  [DOI](https://doi.org/10.1016/j.tig.2012.03.004) | [PMC free article](/articles/PMC3378813/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22480918/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Trends%20Genet.&title=Pathway%20analysis%20of%20genomic%20data:%20concepts,%20methods,%20and%20prospects%20for%20future%20development.&author=V.%20K.%20Ramanan&author=L.%20Shen&author=J.%20H.%20Moore&author=A.%20J.%20Saykin&volume=28&publication_year=2012&pages=323-332&pmid=22480918&doi=10.1016/j.tig.2012.03.004&)

44. Saito T., Ikeda M., Mushiroda T., Ozeki T., Kondo K., Shimasaki A., et al. (2016). Pharmacogenomic study of clozapine-induced agranulocytosis/granulocytopenia in a Japanese population. Biol. Psychiatry 80 636–642. 10.1016/j.biopsych.2015.12.006  [DOI](https://doi.org/10.1016/j.biopsych.2015.12.006) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26876947/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biol.%20Psychiatry&title=Pharmacogenomic%20study%20of%20clozapine-induced%20agranulocytosis/granulocytopenia%20in%20a%20Japanese%20population.&author=T.%20Saito&author=M.%20Ikeda&author=T.%20Mushiroda&author=T.%20Ozeki&author=K.%20Kondo&volume=80&publication_year=2016&pages=636-642&pmid=26876947&doi=10.1016/j.biopsych.2015.12.006&)

45. Shivakumar L., Ansell S. (2006). Targeting B-lymphocyte stimulator/B-cell activating factor and a proliferation-inducing ligand in hematologic malignancies. Clin. Lymphoma Myeloma 7 106–108. 10.3816/CLM.2006.n.046  [DOI](https://doi.org/10.3816/CLM.2006.n.046) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17026820/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Lymphoma%20Myeloma&title=Targeting%20B-lymphocyte%20stimulator/B-cell%20activating%20factor%20and%20a%20proliferation-inducing%20ligand%20in%20hematologic%20malignancies.&author=L.%20Shivakumar&author=S.%20Ansell&volume=7&publication_year=2006&pages=106-108&pmid=17026820&doi=10.3816/CLM.2006.n.046&)

46. Smith T. J., Bohlke K., Lyman G. H., Carson K. R., Crawford J., Cross S. J., et al. (2015). Recommendations for the use of WBC growth factors: American society of clinical oncology clinical practice guideline update. J. Clin. Oncol. 33 3199–3212. 10.1200/jco.2015.62.3488  [DOI](https://doi.org/10.1200/jco.2015.62.3488) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26169616/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Oncol.&title=Recommendations%20for%20the%20use%20of%20WBC%20growth%20factors:%20American%20society%20of%20clinical%20oncology%20clinical%20practice%20guideline%20update.&author=T.%20J.%20Smith&author=K.%20Bohlke&author=G.%20H.%20Lyman&author=K.%20R.%20Carson&author=J.%20Crawford&volume=33&publication_year=2015&pages=3199-3212&pmid=26169616&doi=10.1200/jco.2015.62.3488&)

47. Storey J. D., Tibshirani R. (2003). Statistical significance for genomewide studies. Proc. Natl. Acad. Sci. U.S.A. 100 9440–9445. 10.1073/pnas.1530509100  [DOI](https://doi.org/10.1073/pnas.1530509100) | [PMC free article](/articles/PMC170937/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12883005/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.S.A.&title=Statistical%20significance%20for%20genomewide%20studies.&author=J.%20D.%20Storey&author=R.%20Tibshirani&volume=100&publication_year=2003&pages=9440-9445&pmid=12883005&doi=10.1073/pnas.1530509100&)

48. Tang N. L., Liao C. D., Wang X., Mo F. K., Chan V. T., Ng R., et al. (2013). Role of pharmacogenetics on adjuvant chemotherapy-induced neutropenia in Chinese breast cancer patients. J. Cancer Res. Clin. Oncol. 139 419–427. 10.1007/s00432-012-1345-5  [DOI](https://doi.org/10.1007/s00432-012-1345-5) | [PMC free article](/articles/PMC11824757/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23143606/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Cancer%20Res.%20Clin.%20Oncol.&title=Role%20of%20pharmacogenetics%20on%20adjuvant%20chemotherapy-induced%20neutropenia%20in%20Chinese%20breast%20cancer%20patients.&author=N.%20L.%20Tang&author=C.%20D.%20Liao&author=X.%20Wang&author=F.%20K.%20Mo&author=V.%20T.%20Chan&volume=139&publication_year=2013&pages=419-427&pmid=23143606&doi=10.1007/s00432-012-1345-5&)

49. Terrier B., Ittah M., Tourneur L., Louache F., Soumelis V., Lavie F., et al. (2007). Late-onset neutropenia following rituximab results from a hematopoietic lineage competition due to an excessive BAFF-induced B-cell recovery. Haematologica 92 e20–e23. 10.3324/haematol.11031  [DOI](https://doi.org/10.3324/haematol.11031) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17405749/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Haematologica&title=Late-onset%20neutropenia%20following%20rituximab%20results%20from%20a%20hematopoietic%20lineage%20competition%20due%20to%20an%20excessive%20BAFF-induced%20B-cell%20recovery.&author=B.%20Terrier&author=M.%20Ittah&author=L.%20Tourneur&author=F.%20Louache&author=V.%20Soumelis&volume=92&publication_year=2007&pages=e20-e23&pmid=17405749&doi=10.3324/haematol.11031&)

50. The Cancer Genome Atlas Network (2012). Comprehensive molecular portraits of human breast tumours. Nature 490 61–70. 10.1038/nature11412  [DOI](https://doi.org/10.1038/nature11412) | [PMC free article](/articles/PMC3465532/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23000897/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Comprehensive%20molecular%20portraits%20of%20human%20breast%20tumours.&volume=490&publication_year=2012&pages=61-70&pmid=23000897&doi=10.1038/nature11412&)

51. Therasse P., Mauriac L., Welnicka-Jaskiewicz M., Bruning P., Cufer T., Bonnefoi H., et al. (2003). Final results of a randomized phase III trial comparing cyclophosphamide, epirubicin, and fluorouracil with a dose-intensified epirubicin and cyclophosphamide + filgrastim as neoadjuvant treatment in locally advanced breast cancer: an EORTC-NCIC-SAKK multicenter study. J. Clin. Oncol. 21 843–850. 10.1200/JCO.2003.05.135  [DOI](https://doi.org/10.1200/JCO.2003.05.135) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12610183/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Oncol.&title=Final%20results%20of%20a%20randomized%20phase%20III%20trial%20comparing%20cyclophosphamide,%20epirubicin,%20and%20fluorouracil%20with%20a%20dose-intensified%20epirubicin%20and%20cyclophosphamide%20+%20filgrastim%20as%20neoadjuvant%20treatment%20in%20locally%20advanced%20breast%20cancer:%20an%20EORTC-NCIC-SAKK%20multicenter%20study.&author=P.%20Therasse&author=L.%20Mauriac&author=M.%20Welnicka-Jaskiewicz&author=P.%20Bruning&author=T.%20Cufer&volume=21&publication_year=2003&pages=843-850&pmid=12610183&doi=10.1200/JCO.2003.05.135&)

52. Torkamani A., Topol E. J., Schork N. J. (2008). Pathway analysis of seven common diseases assessed by genome-wide association. Genomics 92 265–272. 10.1016/j.ygeno.2008.07.011  [DOI](https://doi.org/10.1016/j.ygeno.2008.07.011) | [PMC free article](/articles/PMC2602835/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18722519/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genomics&title=Pathway%20analysis%20of%20seven%20common%20diseases%20assessed%20by%20genome-wide%20association.&author=A.%20Torkamani&author=E.%20J.%20Topol&author=N.%20J.%20Schork&volume=92&publication_year=2008&pages=265-272&pmid=18722519&doi=10.1016/j.ygeno.2008.07.011&)

53. Ulrich C. M., Robien K., McLeod H. L. (2003). Cancer pharmacogenetics: polymorphisms, pathways and beyond. Nat. Rev. Cancer 3 912–920. 10.1038/nrc1233  [DOI](https://doi.org/10.1038/nrc1233) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14740638/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Cancer&title=Cancer%20pharmacogenetics:%20polymorphisms,%20pathways%20and%20beyond.&author=C.%20M.%20Ulrich&author=K.%20Robien&author=H.%20L.%20McLeod&volume=3&publication_year=2003&pages=912-920&pmid=14740638&doi=10.1038/nrc1233&)

54. van der Weide K., Loovers H., Pondman K., Bogers J., van der Straaten T., Langemeijer E., et al. (2017). Genetic risk factors for clozapine-induced neutropenia and agranulocytosis in a Dutch psychiatric population. Pharmacogenomics J. 17 471–478. 10.1038/tpj.2016.32  [DOI](https://doi.org/10.1038/tpj.2016.32) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27168101/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J.&title=Genetic%20risk%20factors%20for%20clozapine-induced%20neutropenia%20and%20agranulocytosis%20in%20a%20Dutch%20psychiatric%20population.&author=K.%20van%20der%20Weide&author=H.%20Loovers&author=K.%20Pondman&author=J.%20Bogers&author=T.%20van%20der%20Straaten&volume=17&publication_year=2017&pages=471-478&pmid=27168101&doi=10.1038/tpj.2016.32&)

55. Vogel C. L., Wojtukiewicz M. Z., Carroll R. R., Tjulandin S. A., Barajas-Figueroa L. J., Wiens B. L., et al. (2005). First and subsequent cycle use of pegfilgrastim prevents febrile neutropenia in patients with breast cancer: a multicenter, double-blind, placebo-controlled phase III study. J. Clin. Oncol. 23 1178–1184. 10.1200/jco.2005.09.102  [DOI](https://doi.org/10.1200/jco.2005.09.102) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15718314/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Oncol.&title=First%20and%20subsequent%20cycle%20use%20of%20pegfilgrastim%20prevents%20febrile%20neutropenia%20in%20patients%20with%20breast%20cancer:%20a%20multicenter,%20double-blind,%20placebo-controlled%20phase%20III%20study.&author=C.%20L.%20Vogel&author=M.%20Z.%20Wojtukiewicz&author=R.%20R.%20Carroll&author=S.%20A.%20Tjulandin&author=L.%20J.%20Barajas-Figueroa&volume=23&publication_year=2005&pages=1178-1184&pmid=15718314&doi=10.1200/jco.2005.09.102&)

56. Wang K., Li M., Bucan M. (2007). Pathway-based approaches for analysis of genomewide association studies. Am. J. Hum. Genet. 81 1278–1283. 10.1086/522374  [DOI](https://doi.org/10.1086/522374) | [PMC free article](/articles/PMC2276352/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17966091/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Hum.%20Genet.&title=Pathway-based%20approaches%20for%20analysis%20of%20genomewide%20association%20studies.&author=K.%20Wang&author=M.%20Li&author=M.%20Bucan&volume=81&publication_year=2007&pages=1278-1283&pmid=17966091&doi=10.1086/522374&)

57. Wang K., Li M., Hakonarson H. (2010). Analysing biological pathways in genome-wide association studies. Nat. Rev. Genet. 11 843–854. 10.1038/nrg2884  [DOI](https://doi.org/10.1038/nrg2884) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21085203/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Genet.&title=Analysing%20biological%20pathways%20in%20genome-wide%20association%20studies.&author=K.%20Wang&author=M.%20Li&author=H.%20Hakonarson&volume=11&publication_year=2010&pages=843-854&pmid=21085203&doi=10.1038/nrg2884&)

58. Wang L., McLeod H. L., Weinshilboum R. M. (2011). Genomics and drug response. N. Engl. J. Med. 364 1144–1153. 10.1056/NEJMra1010600  [DOI](https://doi.org/10.1056/NEJMra1010600) | [PMC free article](/articles/PMC3184612/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21428770/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N.%20Engl.%20J.%20Med.&title=Genomics%20and%20drug%20response.&author=L.%20Wang&author=H.%20L.%20McLeod&author=R.%20M.%20Weinshilboum&volume=364&publication_year=2011&pages=1144-1153&pmid=21428770&doi=10.1056/NEJMra1010600&)

59. Weinshilboum R., Wang L. (2004). Pharmacogenomics: bench to bedside. Nat. Rev. Drug Discov. 3 739–748. 10.1038/nrd1497  [DOI](https://doi.org/10.1038/nrd1497) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15340384/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Drug%20Discov.&title=Pharmacogenomics:%20bench%20to%20bedside.&author=R.%20Weinshilboum&author=L.%20Wang&volume=3&publication_year=2004&pages=739-748&pmid=15340384&doi=10.1038/nrd1497&)

60. Widschwendter P., Friedl T. W., Schwentner L., DeGregorio N., Jaeger B., Schramm A., et al. (2015). The influence of obesity on survival in early, high-risk breast cancer: results from the randomized SUCCESS A trial. Breast Cancer Res. 17:129. 10.1186/s13058-015-0639-3  [DOI](https://doi.org/10.1186/s13058-015-0639-3) | [PMC free article](/articles/PMC4575482/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26385214/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Breast%20Cancer%20Res.&title=The%20influence%20of%20obesity%20on%20survival%20in%20early,%20high-risk%20breast%20cancer:%20results%20from%20the%20randomized%20SUCCESS%20A%20trial.&author=P.%20Widschwendter&author=T.%20W.%20Friedl&author=L.%20Schwentner&author=N.%20DeGregorio&author=B.%20Jaeger&volume=17&issue=129&publication_year=2015&pmid=26385214&doi=10.1186/s13058-015-0639-3&)

61. Wu Z., Irizarry R. A., Gentleman R., Martinez-Murillo F., Spencer F. (2004). A model-based background adjustment for oligonucleotide expression arrays. J. Am. Stat. Assoc. 99 909–917. 10.1198/016214504000000683  [DOI](https://doi.org/10.1198/016214504000000683) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Am.%20Stat.%20Assoc.&title=A%20model-based%20background%20adjustment%20for%20oligonucleotide%20expression%20arrays.&author=Z.%20Wu&author=R.%20A.%20Irizarry&author=R.%20Gentleman&author=F.%20Martinez-Murillo&author=F.%20Spencer&volume=99&publication_year=2004&pages=909-917&doi=10.1198/016214504000000683&)

62. Yao S., Barlow W. E., Albain K. S., Choi J. Y., Zhao H., Livingston R. B., et al. (2010). Gene polymorphisms in cyclophosphamide metabolism pathway, treatment-related toxicity, and disease-free survival in SWOG 8897 clinical trial for breast cancer. Clin. Cancer Res. 16 6169–6176. 10.1158/1078-0432.CCR-10-0281  [DOI](https://doi.org/10.1158/1078-0432.CCR-10-0281) | [PMC free article](/articles/PMC3058716/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21169260/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Cancer%20Res.&title=Gene%20polymorphisms%20in%20cyclophosphamide%20metabolism%20pathway,%20treatment-related%20toxicity,%20and%20disease-free%20survival%20in%20SWOG%208897%20clinical%20trial%20for%20breast%20cancer.&author=S.%20Yao&author=W.%20E.%20Barlow&author=K.%20S.%20Albain&author=J.%20Y.%20Choi&author=H.%20Zhao&volume=16&publication_year=2010&pages=6169-6176&pmid=21169260&doi=10.1158/1078-0432.CCR-10-0281&)

63. Yu J., Qin B., Wu F., Qin S., Nowsheen S., Shan S., et al. (2017). Regulation of serine-threonine kinase Akt activation by NAD+-dependent deacetylase SIRT7. Cell Rep. 18 1229–1240. 10.1016/j.celrep.2017.01.009  [DOI](https://doi.org/10.1016/j.celrep.2017.01.009) | [PMC free article](/articles/PMC5298804/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28147277/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell%20Rep.&title=Regulation%20of%20serine-threonine%20kinase%20Akt%20activation%20by%20NAD+-dependent%20deacetylase%20SIRT7.&author=J.%20Yu&author=B.%20Qin&author=F.%20Wu&author=S.%20Qin&author=S.%20Nowsheen&volume=18&publication_year=2017&pages=1229-1240&pmid=28147277&doi=10.1016/j.celrep.2017.01.009&)
