# Integrated analysis of genetic variation and gene expression reveals novel variant for increased warfarin dose requirement in African Americans

## Metadata
**Authors:** W HERNANDEZ, E R GAMAZON, K AQUINO-MICHAELS, E SMITHBERGER, T J O’BRIEN, A F HARRALSON, M TUCK, A BARBOUR, L H CAVALLARI, M A PERERA
**Journal:** Journal of thrombosis and haemostasis : JTH
**Date:** 2017 Mar 25
**DOI:** [10.1111/jth.13639](https://doi.org/10.1111/jth.13639)
**PMID:** 28135054
**PMCID:** PMC5862636
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5862636/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC5862636/pdf/nihms944995.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC5862636/pdf/nihms944995.pdf)

## Abstract

**Background:** 
Warfarin is commonly used to control and prevent thromboembolic disorders. However, because of warfarin’s complex dose–requirement relationship, safe and effective use is challenging. Pharmacogenomics-guided warfarin dosing algorithms that include the well-established VKORC1 and CYP2C9 polymorphisms explain only a small proportion of inter-individual variability in African Americans (AAs).

**Objectives:** 
We aimed to assess whether transcriptomic analyses could be used to identify regulatory variants associated with warfarin dose response in AAs.

**Patients/Methods:** 
We identified a total of 56 expression quantitative trait loci (eQTLs) for CYP2C9, VKORC1 and CALU derived from human livers and evaluated their association with warfarin dose response in two independent AA warfarin patient cohorts.

**Results:** 
We found that rs4889606, a strong cis-eQTL for VKORC1 (log10 Bayes Factor = 12.02), is significantly associated with increased warfarin daily dose requirement (β = 1.1; 95% confidence interval [CI] 0.46 to 1.8) in the discovery cohort (n = 305) and in the replication cohort (β = 1.04; 95% CI 0.33 −1.7; n = 141) after conditioning on relevant covariates and the VKORC1 -1639G>A (rs9923231) variant. Inclusion of rs4889606 genotypes, along with CYP2C9 alleles, rs9923231 genotypes and clinical variables, explained 31% of the inter-patient variability in warfarin dose requirement. We demonstrate different linkage disequilibrium patterns in the region encompassing rs4889606 and rs9923231 between AAs and European Americans, which may explain the increased dose requirement found in AAs.

**Conclusion:** 
Our approach of interrogating eQTLs identified in liver has revealed a novel predictor of warfarin dose response in AAs. Our work highlights the utility of leveraging information from regulatory variants mapped in the liver to uncover novel variants associated with drug response and the importance of population-specific research.

Keywords: African American, pharmacogenetics, quantitative trait loci, thromboembolism, warfarin

### Background

Warfarin is commonly used to control and prevent thromboembolic disorders. However, because of warfarin’s complex dose–requirement relationship, safe and effective use is challenging. Pharmacogenomics-guided warfarin dosing algorithms that include the well-established *VKORC1* and *CYP2C9* polymorphisms explain only a small proportion of inter-individual variability in African Americans (AAs).

### Objectives

We aimed to assess whether transcriptomic analyses could be used to identify regulatory variants associated with warfarin dose response in AAs.

### Patients/Methods

We identified a total of 56 expression quantitative trait loci (eQTLs) for *CYP2C9*, *VKORC1* and *CALU* derived from human livers and evaluated their association with warfarin dose response in two independent AA warfarin patient cohorts.

### Results

We found that rs4889606, a strong *cis*-eQTL for *VKORC1* (log_10_ Bayes Factor = 12.02), is significantly associated with increased warfarin daily dose requirement (β = 1.1; 95% confidence interval [CI] 0.46 to 1.8) in the discovery cohort (*n* = 305) and in the replication cohort (β = 1.04; 95% CI 0.33 −1.7; *n* = 141) after conditioning on relevant covariates and the *VKORC1* -1639G>A (rs9923231) variant. Inclusion of rs4889606 genotypes, along with *CYP2C9* alleles, rs9923231 genotypes and clinical variables, explained 31% of the inter-patient variability in warfarin dose requirement. We demonstrate different linkage disequilibrium patterns in the region encompassing rs4889606 and rs9923231 between AAs and European Americans, which may explain the increased dose requirement found in AAs.

### Conclusion

Our approach of interrogating eQTLs identified in liver has revealed a novel predictor of warfarin dose response in AAs. Our work highlights the utility of leveraging information from regulatory variants mapped in the liver to uncover novel variants associated with drug response and the importance of population-specific research.

## Introduction

Warfarin is an effective anticoagulant for the prevention and treatment of thrombotic disease. However, because of its narrow therapeutic window and large variation in therapeutic warfarin dose requirement between individuals, there can be significant adverse effects [[1](#R1)]. Although new oral anticoagulants (NOACs) have been developed, there are several advantages in continuing to administer warfarin to patients, such as extensive familiarity, well-established form of monitoring (international normalized ratio), treatment of valvular atrial fibrillation and cost [[2](#R2),[3](#R3)]. Furthermore, in many countries outside of the USA, NOACs are covered only for patients with poor anticoagulation control on vitamin K antagonists (VKAs) [[3](#R3)]. Therefore, warfarin will remain a key drug in the treatment of thrombotic disease. As such, identification of polymorphisms associated with warfarin dose response will continue to be of importance in determining safe and effective doses for individual patients.

In the liver, warfarin is primarily metabolized by CYP2C9 and exerts its anticoagulant effect by inhibiting VKOR (encoded by VKORC1), leading to reduced amounts of vitamin K available to serve as a cofactor for clotting proteins [[4](#R4),[5](#R5)]. Polymorphisms in *CYP2C9* and *VKORC1* have been well established in warfarin dose requirement [[6](#R6)]. However, these polymorphisms explain significantly less of the variance in dose among African Americans (AAs) compared with Caucasians (8% vs. 35%, respectively), which may be due to a lower frequency of the minor alleles of *CYP2C9*^*^2, *CYP2C9*^*^3 and *VKORC1* -1639G>A among AAs [[7](#R7)–[10](#R10)]. Several other genetic variants in other genes have also been implicated as influencing warfarin dose requirement. Among these are single nucleotide polymorphisms (SNPs) in *CYP4F2*, which metabolizes vitamin K, thereby limiting its availability, and *CALU*, a cofactor in the VKOR complex [[11](#R11),[12](#R12)]. However, studies have shown a lack of association between *CYP4F2* genotypes and therapeutic warfarin dose in AAs [[13](#R13),[14](#R14)].

Despite substantial efforts, identification of novel polymorphisms associated with warfarin dose response has been limited. Several genome-wide studies have mapped variants that correlate with expression levels of genes known as expression quantitative trait loci (eQTL) [[15](#R15)–[18](#R18)], which may provide a framework for identifying novel variants associated with complex traits, including drug response [[18](#R18)]. Because coagulation factors and warfarin are primarily synthesized and metabolized in the liver, respectively, the study of eQTLs derived from human livers is a more suitable analysis for the identification of regulatory variation that may affect warfarin dose requirement.

With this in mind, we have examined additional common genetic variation (minor allele frequency ≥ 5%) that may be predictive of therapeutic warfarin dose in AAs using eQTL analysis conducted in human livers. We focused on eQTLs for the candidate genes, *CYP2C9*, *VKORC1* and *CALU*, given their role and previous association in warfarin dose response in AAs. The goal of the present study is to utilize regulatory variants derived from human livers in order to identify novel genetic variants associated with warfarin dose requirement in AAs. We performed a two-stage analysis to test the association between previously identified eQTLs for *CYP2C9*, *VKORC1* and *CALU* and warfarin dose response among AAs in a discovery cohort, followed by examination of the most significant eQTLs in an independent replication cohort.

## Materials and methods

### African American patient cohorts

Participants in the discovery (*n* = 306) and replication (*n* = 141) cohorts were unrelated, self-described as AA, age ≥ 18 years, and on a stable dose of warfarin defined as the same dose for at least three consecutive clinic visits that produced an international normalized ratio (INR) within the therapeutic range of 2–3. Data related to warfarin dose requirement included age, height, weight, ethnicity, gender, liver disease and chronic kidney disease defined as an estimated glomerular filtration rate, stable warfarin dose, INR at stable warfarin dose, indication for warfarin therapy, and other medications taken, including potentially interacting drugs. Patients were recruited from The University of Chicago, University of Illinois at Chicago, The George Washington University Medical Faculty Associates and the Veterans Affairs Hospital in Washington, DC. The replication cohort was independent of the discovery cohort and consisted of patients enrolled from the same institutions. The study was conducted according to the principles of the Declaration of Helsinki. The institutional review board approved the study and written informed consent was obtained from all the study participants.

### Identification and selection of liver eQTLs

To determine liver eQTLs for our candidate genes, we conducted a bioinformatics analysis of publicly available data from the Gene Expression Omnibus, accession numbers [GSE9588](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE9588) and [GSE26106](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE26106) ([http://www.ncbi.nlm.nih.gov/geo](http://www.ncbi.nlm.nih.gov/geo)) [[16](#R16),[19](#R19)]. *Cis*- and *trans*-acting eQTLs were identified using a method similar to that previously described by Innocenti *et al.* [[19](#R19)]. Briefly, SNPs within 250 kb to the transcription start site of the associated gene were classified as *cis*-eQTLs. All other associations were considered candidate *trans*-eQTLs. Log_10_-transformed Bayes factor (log_10_BF) was used to infer genotype–trait association rather than a *P*-value in order to assess the evidence that a given SNP is an eQTL for a given gene, as previously described [[20](#R20)].

[GSE9588](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE9588) and [GSE26106](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE26106) are primarily comprised of liver tissue from Caucasian participants. To account for the differences in linkage disequilibrium (LD) structure between AAs and Caucasians, and to identify independent eQTL associations among AAs, we conducted LD pruning with phased genotype data from the African ancestry in the southwest USA (ASW) population in HapMap 3 release 2. Pairwise LD between *VKORC1* -1639G>A rs9923231, the known predictive *VKORC1* SNP for warfarin dose, and the *VKORC1 cis*-eQTLs was calculated in the same fashion utilizing SNAP version 2.2 [[21](#R21)].

### Genotyping

Study participants provided a DNA sample (whole-blood, saliva or mouthwash sample) and genomic DNA was isolated as previously described [[22](#R22)]. eQTLs with a log_10_ BF ≥ 3 were genotyped using either Sequenom Massarray or TaqMan assays according to the manufacturer’s instructions. SNPs, which failed genotyping because of multiplexing constraints, were genotyped via predesigned TaqMan SNP assays (Applied Biosystems, Foster City, CA, USA). Patients were also genotyped for the known genetic variants rs9923231 (*VKORC1 -1639*), rs1799853 (*CYP2C9*2*), rs1057910 (*CYP2C9*3*), rs28371686 (*CYP2C9*5*), rs7900194 (*CYP2C9*8*) and rs28371685 (*CYP2C9*11*) using previously published methods [[23](#R23)–[25](#R25)]. In addition, a panel of 105 ancestry informative markers (AIMs) was genotyped for each patient to determine West African ancestry (WAA) using Sequenom Massarray as previously described [[26](#R26)].

### Quality control

SNPs genotyped in both cohorts were excluded based on genotyping rate <90%, minor allele frequency of <5% and failed Hardy–Weinberg equilibrium (HWE) testing at a *P*-value <0.0001. To test for departures from HWE, we utilized the Fisher’s exact test implemented in PLINK [[27](#R27)]. Three SNPs were removed due to failing quality control (QC) thresholds. Samples with SNP call rates <90% across all SNPs were excluded. Only one individual in the discovery cohort did not meet QC thresholds, resulting in 305 samples included in the analysis. All QC steps were conducted using PLINK [[27](#R27)].

### Statistical analysis

For both cohorts, the Kolmogorov–Smirnov test was used to test for normality and weekly warfarin dose values were log_2_-transformed before statistical modeling to achieve an approximate normal distribution. Pearson’s correlation was used to identify clinical factors associated with weekly warfarin dose (mg). All covariates were tested as single covariates for association with therapeutic weekly warfarin dose using the IBM SPSS Statistics version 19.0.0 package (SPSS, Chicago, IL, USA). In order to identify SNP associations independent of the well-known variants associated with warfarin dose response, analysis of *VKORC1* and *CYP2C9* eQTLs were conditioned for *VKORC1* -1639G>A rs9923231 and presence of *CYP2C9*2*, **3*, **5*, **8* and **11* alleles (*CYP2C9* star alleles), respectively. We combined the *CYP2C9*2*, **3*, **5*, **8* and **11* variants into a single group of *CYP2C9* star alleles, on the basis of earlier data [[28](#R28)]. Individual ancestry estimates were obtained from STRUCTURE 2.3.3 as previously described [[29](#R29)].

Linear regression was used to test the association between SNPs and warfarin dose response adjusting for age, weight and indication of DVT or PE. Statistically significant SNPs in the discovery cohort were then genotyped in the replication cohort for validation. All SNP association analyses were conducted in PLINK [[27](#R27)]. Statistical significance for association refers to a *P*-value ≤ 0.00087 for the discovery cohort reflecting Bonferroni correction (0.05/57) to account for multiple testing and *P*-value ≤ 0.05 for the replication cohort because only one SNP met the criteria for replication. The effects of genetic and non-genetic factors on therapeutic warfarin dose variability were assessed by multivariate analysis. Clinical variables included in the analysis were age, weight and deep vein thrombosis and/or pulmonary embolism (DVT/PE). Presence of *CYP2C9*2*, **3*, **5*, **8* and **11* alleles were coded as 0 for wild-type and 1 for variant carrier; *VKORC1* -1639G>A rs9923231 was coded as 0 for GG, 1 for GA and 2 for AA; and rs4889606 was coded as 0 for AA, 1 for GA and 2 for GG.

## Results

Demographic and clinical characteristics for the discovery and replication cohorts are provided in [Table 1](#T1). Mean therapeutic weekly warfarin dose and INR for the discovery cohort were 44.9 ± 18.7 mg and 2.47 ± 0.36, respectively ([Table 1](#T1)). For the replication cohort, the mean therapeutic weekly warfarin dose and INR were 45.4 ± 15.9 mg and 2.38 ± 0.38, respectively ([Table 1](#T1)). The majority of patients were taking warfarin due to DVT/PE (> 50% in each cohort; [Table 1](#T1)). Mean age was slightly lower for the discovery cohort compared with the replication cohort (57.62 ± 15.7 and 62.12 ± 16.5, respectively; *P* = 0.001). The replication cohort consisted of significantly more men (52.9%, *P* < 0.01) compared with the discovery cohort and may explain the significantly higher mean weight and height observed (100.78 ± 36.0 and 171.27 ± 11.4; *P* = 0.009 and 0.002, respectively; [Table 1](#T1)). Percentage of WAA was significantly higher in the discovery cohort compared with the replication cohort (81.4% and 77.2%, respectively, *P* = 0.001; [Table 1](#T1)). Allele frequencies for *CYP2C9* star alleles and *VKORC1 -1639* are similar between the cohorts and were in HWE ([Table 1](#T1)). Although some demographic characteristics were significantly different between the discovery and the replication cohorts, participants were recruited from the same population and had the same phenotype, furthermore, the same methods were applied.

### Table 1.

| Variable | Discovery cohort (n = 305) | Replication cohort (n = 141) | P-value |
| --- | --- | --- | --- |
| Warfarin dose, mg week−1 | 44.9 ± 18.7 | 45.40 ± 15.9 | 0.81 |
| INR (at stable dose) | 2.47 ± 0.36 | 2.38 ± 0.38 | 0.06 |
| DVT/PE† | 60.0% | 53.3% | 0.18 |
| Male | 30.6% | 52.9% | <0.001 |
| Age, years | 57.62 ± 15.7 | 62.12 ± 16.5 | 0.001 |
| Weight, kg | 92.4 ± 27.8 | 100.78 ± 36.0 | 0.009 |
| Height, cm | 167.83 ± 10.3 | 171.27 ± 11.4 | 0.002 |
| Percentage WAA | 81.4 ± 10.9 | 77.2 ± 9.4 | 0.001 |

Table 1 Caption: Clinical, demographic and genetic characteristics of study cohorts

As expected, age was inversely associated with warfarin dose requirement in both the discovery and replication cohorts (*P* < 0.001; [Table 2](#T2)). We found that warfarin dose requirement increased with increasing weight in the discovery and replication cohorts (*P* < 0.001 and *P* = 0.003, respectively; [Table 2](#T2)). The association between height and warfarin dose requirement was only significant in the discovery cohort (*P* = 0.0004; [Table 2](#T2)). However, this association was lost when we accounted for weight in our analysis. DVT/PE as a primary indication for warfarin therapy was significantly associated with warfarin dose requirement in both cohorts (*P* = 0.001; [Table 2](#T2)). Neither percentage of WAA or sex was associated with warfarin dose requirement (*P* > 0.05) in either cohort ([Table 2](#T2)). Age, weight and DVT/PE were included as covariates for all associations between genetic markers and warfarin dose response.

### Table 2.

|   | Discovery cohort (n = 305) |  | Replication cohort (n = 141) |  |
| --- | --- | --- | --- | --- |
| Variable | Correlation coefficient (95% CI) | P-value | Correlation coefficient (95% CI) | P-value |
| Age, years | −0.02 (−0.02 to −0.01) | 1.80 × 10−12 | −0.01 (−0.02 to −0.05) | 0.0003 |
| Weight, kg | 0.01 (0.004 to 0.01) | 0.000001 | 0.004 (0.001 to 0.01) | 0.003 |
| Height, cm | 0.01 (0.01 to 0.02) | 0.0004 | 0.01 (−0.002 to 0.02) | 0.11 |
| Percentage WAA | 0.42 (−0.21 to 1.04) | 0.19 | −0.32 (−1.47 to 0.84) | 0.59 |
| DVT/PE | 0.25 (0.11 to 0.38) | 0.001 | 0.30 (0.012 to 0.48) | 0.001 |
| Sex, male | −0.13 (−0.28 to 0.02) | 0.09 | −0.07 (−0.26 to −0.12) | 0.44 |
| VKORC1 -1639 A (rs9923231) | −0.20 (−0.36 to 0.03) | 0.02 | −0.34 (−0.55 to −0.14) | 0.004 |
| CYP2C9 star alleles† | −0.14 (−0.30 to 0.24) | 0.10 | −0.37 (−0.60 to −0.15) | 0.003 |

Table 2 Caption: Univariate associations between clinical variables and log2-transformed weekly warfarin dose

A total of 56 eQTLs identified from human livers were genotyped in the discovery cohort ([Tables 3](#T3) and [S1–S3](#SD1)). Analysis of five *cis*- and six *trans*-eQTLs for *CALU* did not uncover a significant association with warfarin dose response ([Table S1](#SD2)). In addition to *CALU* eQTLs, we also genotyped the *CALU* SNP rs339097 because of its previous association with warfarin dose response among AAs [[30](#R30)]. However, rs339097 was not associated with warfarin dose response in the discovery cohort (*P* = 0.61; [Table S1](#SD2)), which is consistent with previous findings [[25](#R25)]. For *CYP2C9*, we evaluated the association between 12 *trans*-eQTLs and warfarin dose response ([Table S2](#SD2)). After conditioning on the presence of any *CYP2C9* star allele, none of the *CYP2C9* eQTLs remained associated with warfarin dose response ([Table S2](#SD2)).

### Table 3.

| Gene | *cis-eQTLs | trans-eQTLs | Total |
| --- | --- | --- | --- |
| CALU | 5 | 6 | 11 |
| CYP2C9 | 0 | 12 | 12 |
| VKORC1 | 15 | 17 | 32 |
| Total eQTLs |   |   | 56 |

Table 3 Caption: Distribution of eQTLs by gene with a log10-transformed Bayes factor ≥ 3

We identified 32 *VKORC1* eQTLs, among which *VKORC1* -1639G>A rs9923231, rs749670, rs4889606 and rs9926533 were strongly associated, *in cis*, with *VKORC1* gene expression (log_10_BF ≥ 12.0; [Table S3](#SD2)). In the discovery and replication cohorts, the known warfarin dose polymorphism *VKORC1* -1639G>A rs9923231 was significantly associated with lower warfarin dose (*P* = 8.46 × 10^−9^ and *P* = 0.0002, respectively; data not shown) after adjusting for covariates. Consequently, for the *VKORC1* eQTLs, we carried out conditional analysis, which included SNP *VKORC1* -1639G>A rs9923231 in addition to age, weight and presence of DVT/PE. Only one eQTL, rs4889606, reached corresponding Bonferroni-corrected significance and was associated with increased warfarin dose requirement in the discovery cohort (*P* = 7.18 × 10^−6^; [Table 4](#T4)). This association was further validated in the replication cohort with matching direction of effect and increased warfarin dose response (*P* = 0.01; [Table 4](#T4)). The association of daily stable warfarin dose requirement and rs4889606 genotypes is shown in [Fig. 1](#F1). The daily warfarin dose was significantly different among the three genotypes of rs4889606, with carriers of the AG or GG genotypes requiring higher doses (6.8 ± 3.1 and 7.6 ± 3.9 mg day^−1^, respectively) compared with AA carriers in the discovery cohort (*β* = 1.1; 95% CI = 0.46–1.8; *P* = 0.0009). This increase in daily warfarin dose requirement for AG or GG carriers (6.8 ± 2.3 and 8.6 ± 3.1, respectively) was also found in the replication cohort (*β* = 1.04; 95% CI = 0.33–1.7; *P* = 0.005; [Fig. 1](#F1)). We also found that SNPs rs4889606 and *VKORC1* -1639G>A rs9923231 were not in LD in either the discovery or replication cohorts (*r*^2^ = 0.17 and *r*^2^ = 0.08, respectively; [Table 4](#T4)), which may explain the increased warfarin dose requirement observed for rs4889606 among AAs. [Table 5](#T5) displays the results of the multiple regression models for the discovery and replication cohorts. The clinical variables alone (age, weight and presence of DVT/PE) explained 20% and 13% of the variability in warfarin dose requirement for the discovery and replication cohorts, respectively (adjusted *R*^2^; [Table 5](#T5)). Inclusion of the *CYP2C9 *2*, **3*, **5*, **8* and **11* alleles and *VKORC1* -1639G>A genotypes in the clinical only model increased the explained variance in warfarin dose requirement to 30% and 28%, respectively. For both the discovery and replication cohorts a modest increase to 31% in explained dose variability was observed in the model with the addition of the rs4889606 genotypes, while accounting for clinical and previously associated genetic predictors ([Table 5](#T5)).

### Table 4.

| eQTL gene | Chr | SNP | Discovery cohort (n = 305) |  |  |  |  |  | Replication cohort (n = 141) |  |  |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| MA | MAF | exp(β) | 95% CI | P-value | r2/D′* | MAF | exp(β) | 95% CI | P-value | r2/D′* |  |  |  |
| VKORC1 (cis) | 16 | rs4889606 | G | 0.21 | 1.28 | 0.14–0.36 | 8.81×10−6 | 0.17/0.68 | 0.22 | 1.19 | 0.04–0.31 | 0.01 | 0.08/0.47 |

Table 4 Caption: Significant allelic associations between eQTLs identified in human livers and warfarin dose requirement among African Americans

### Fig. 1.

![Fig. 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4c17/5862636/6aceb0e86231/nihms944995f1.jpg)

Effect of rs4889606 genotypes on daily dose requirement for warfarin in African Americans. Carriers of AG and GG genotypes had an increased warfarin dose requirement. P-values calculated after controlling for age, weight and deep vein thrombosis/pulmonary embolism (DVT /PE) and conditioned for rs9923231. White diamond within boxes represents the mean. Box plots indicate 25th and 75th percentiles.

### Table 5.

| Model | Discovery cohort Adjusted R2 | Replication cohort Adjusted R2 |
| --- | --- | --- |
| Clinical only | 0.20 | 0.13 |
| Clinical plus CYP2C9 star alleles and VKORC1 rs9923231 | 0.30 | 0.28 |
| Clinical plus CYP2C9 star alleles, VKORC1 rs9923231 and rs4889606 | 0.31 | 0.31 |

Table 5 Caption: Explained variation in therapeutic warfarin dose

Utilizing HapMap data for ASW (African American) and CEU (European American [EA]) populations we generated LD plots for the region between *STX1B2* and *VKORC*1, which encompasses SNPs rs4889606 and *VKORC1* -1639G>A rs9923231. Pairwise LD analysis revealed differences in sizes and patterns of LD blocks (defined as *r*^2^ ≥ 0.8) between ASW and EA populations ([Fig. S1](#SD1)). Among ASW, we found four haplotype blocks across this region ([Fig. S1A](#SD1)). The well-known *VKORC1* eQTLs, *VKORC1* -1639G>A rs9923231 and rs9934438, which are associated with lower warfarin dose requirement, are in high LD and therefore on the same haplotype block extending 10 kb ([Fig. S1A](#SD1)). In contrast, LD is disrupted and SNP rs4889606 is not found on a haplotype block, further supporting our finding that the increased warfarin dose requirement associated with rs4889606 is independent of rs9923231 among ASW ([Fig. S1A](#SD1)). In contrast, among EAs a relative strong LD pattern was observed between rs4889606, *VKORC1* -1639G>A rs9923231 and rs9934438, which resulted in a 98 kb haplotype block encompassing the entire region ([Fig. S1B](#SD1)).

## Discussion

Our study has identified a novel association with increased warfarin dose requirement among AAs. Warfarin dose requirement has been shown to vary by race, with AAs requiring higher doses [[31](#R31),[32](#R32)]. Personalized warfarin dosing, generated from algorithms based on genetic and clinical factors, has been proposed to improve dosing accuracy and reduce the risk of sub- or supra-therapeutic anticoagulation [[7](#R7),[33](#R33)]. The identification of biologically relevant genetic variation in warfarin dose requirement is essential in personalized medicine, and hence the inclusion of transcriptomic data was used to identify novel variants associated with warfarin dose response. In pharmacogenomics, the liver is the most relevant tissue in which to perform eQTL analysis, which can provide candidate loci for prospective clinical studies. As such, genetic variants that are significantly associated with gene expression may have a putative functional role in drug disposition and response [[34](#R34)]. Through publicly available data, we have identified 56 eQTLs derived from human livers that significantly affect *CYP2C9*, *VKORC1* or *CALU* gene expression and tested their association with therapeutic warfarin dose requirement in AAs.

The most significant finding in our study is the association of the minor allele of rs4889606 (G) with higher warfarin dose requirement in two independent AA warfarin patient cohorts, which remained significant after conditioning on *VKORC1* -1639G>A rs9923231 and clinical factors ([Table 4](#T4)). We demonstrate that carriers of the AG and GG genotypes require a significantly higher daily warfarin dose compared with AA carriers in both the discovery and replication cohorts ([Fig. 1](#F1)). The inclusion of rs4889606 genotypes along with clinical variables, *CYP2C9* star alleles and VKORC1 -1639G>A genotype, modestly increased the proportion of variability in warfarin dose requirement explained in AAs ([Table 5](#T5)). This model explained 31% of the inter-patient variability in warfarin dose requirement in both cohorts. In AAs, most of the variability in warfarin dose requirement remains unaccounted for. Our study provides evidence that supports the inclusion of regulatory genetic variants beyond the previously associated *CYP2C9* and *VKORC1* variants in efforts to discover variants associated with warfarin dose response among AAs.

SNP rs4889606 is located within the intron 6 of *STX1B*, which is approximately 98 kb downstream of the *VKORC1* gene and has a minor allele frequency of 21% and 22% in the discovery and replication cohorts, respectively ([Table 4](#T4)). *VKORC1* variation has long been known to be associated with warfarin dose requirement and SNPs *VKORC1* -1639G>A rs9923231 and rs9934438 (which are in high LD) are the most well-established SNPs at this locus associated with lower warfarin dose requirement across ethnicities [[35](#R35)]. We demonstrate that in both the discovery and replication cohorts, there is very low LD between SNPs rs4889606 and *VKORC1* -1639G>A rs9923231 (*r*^2^ ≤ 0.17, [Table 4](#T4)). When comparing LD patterns in the *STX1B* and *VKORC1* region encompassing rs4889606 and *VKORC1* -1639G>A rs9923231 between those with African ancestry in the southwest USA (ASW) and Utah residents with ancestry from northern and western Europe (CEU) (HapMap populations), we observed shorter stretches of LD among ASW resulting in four blocks of LD, but one large LD block is seen that extends from the *STX1B2* gene into the *VKORC1* gene among CEU ([Fig. S1A and 1B](#SD1), respectively). Furthermore, the minor allele (G) at rs4889606 is associated with higher warfarin dose, whereas the *VKORC1* -1639G>A rs9923231 minor allele (A) is associated with lower warfarin dose requirement in our cohorts, suggesting an independent effect of this SNP from *VKORC1* -1639G>A rs9923231. It is important to note that according to the 1000 Genomes data, rs4889606 is in complete LD with rs12716979 (also located in intron 6 of *STX1B*; *r*^2^ = 1) and in high LD with rs11441998 (within Intron 3 of *STX1B*; *r*^2^ = 0.8).

The association of rs4889606 with higher warfarin dose requirement in AAs provides a more comprehensive understanding of the genetic landscape in dose variability and may help explain differences in warfarin dose requirement between populations. A recent genome-wide association study found rs4889606 to be associated with warfarin dose response in a Brazilian cohort (*P* = 1.0 × 10^−29^) but after conditioning on the study’s top signal (rs749671, which is in complete LD with *VKORC1* -1639G>A rs9923231 in Brazilians [*r*^2^ = 0.98]), it was no longer significant [[36](#R36)]. In a more recent study among Egyptians and European Americans on stable warfarin dose, rs4889606 was found to be a significant determinant of warfarin dose requirement in both populations [[37](#R37)]. Once again, rs4889606 was no longer significant after conditioning for rs9923231 as a result of the high LD observed between rs4889606 and rs9923231 among the Egyptian and European American cohorts [[37](#R37)]. Hamadeh *et al.* also report that AIMs for the Egyptian cohort were not available to confirm WAA; nonetheless, they found that the STX1B haplotype block in Egyptians highly resembled that of European Americans [[37](#R37)]. A limitation of our study was the relatively small sample size for both the discovery cohort and the replication cohort. In addition, rs4889606 was identified as a strong *VKORC1 cis*-eQTL from predominantly Caucasian liver samples (~ 90%) and because of the lack of liver eQTL data from AAs, we cannot determine if rs4889606 is also a *VKORC1* liver eQTL in AAs. To identify potentially regulatory functions, we used HaploReg and found that SNP rs4889606 overlaps with promoter and enhancer histone marks in multiple tissues, including liver tissues, and is located in a DNase hypersensitive region [[38](#R38)]. These results strongly suggest that rs4889606 is located in a regulatory sequence.

In conclusion, our unique approach of interrogating liver eQTLs for *CALU*, *CYP2C9* and *VKORC1* led to the identification of a novel SNP associated with higher warfarin dose requirement in AAs. SNP rs4889606 is downstream of *VKORC1* and has a putative regulatory role. By using the decreased LD structure found in populations of African ancestry, we were able to disentangle the association of the well-established *VKORC1* variant, *VKORC1* -1639G>A rs9923231, with other regulatory variants in this population. Our study suggests that an eQTL-based strategy can be an efficient way of discovering novel associations with drug response. Our study utilized known regulatory variation to identify genetic variation associated with warfarin dose requirement and further highlights the unique genetic discoveries that are possible with populations of African ancestry.

## Supplementary Material

## Acknowledgments

This study was supported (in part) by research funding from: the National Collaboration on Aging to T. J. O’Brien A. F. Harralson, and M. Tuck; the American Heart Association Midwest Affiliate Grant-In-Aid (10GRNT3750024) to L. H. Cavallari; the National Heart Lung and Blood Institute (K23 HL089808-01A2 and R21 HL106097-01A1) to M. A. Perera; the National Institute on Minority Health and Health Disparities (1R01MD009217-01) to M. A. Perera; and the Cardiovascular Sciences Training Grant (5T32 HL007381) to W. Hernandez. E. R. Gamazon was supported by R01 MH101820 and R01 MH090937.

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Wysowski DK, Nourjah P, Swartz L. Bleeding complications with warfarin use: a prevalent adverse effect resulting in regulatory action. Arch Intern Med. 2007;13:1414–9. doi: 10.1001/archinte.167.13.1414.  [DOI](https://doi.org/10.1001/archinte.167.13.1414) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17620536/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arch%20Intern%20Med&title=Bleeding%20complications%20with%20warfarin%20use:%20a%20prevalent%20adverse%20effect%20resulting%20in%20regulatory%20action&author=DK%20Wysowski&author=P%20Nourjah&author=L%20Swartz&volume=13&publication_year=2007&pages=1414-9&pmid=17620536&doi=10.1001/archinte.167.13.1414&)

2. Fauchier L, Philippart R, Clementy N, Bourguignon T, Angoulvant D, Ivanes F, Babuty D, Bernard A. How to define valvular atrial fibrillation? Arch Cardiovasc Dis. 2015;10:530–9. doi: 10.1016/j.acvd.2015.06.002.  [DOI](https://doi.org/10.1016/j.acvd.2015.06.002) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26184867/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arch%20Cardiovasc%20Dis&title=How%20to%20define%20valvular%20atrial%20fibrillation?&author=L%20Fauchier&author=R%20Philippart&author=N%20Clementy&author=T%20Bourguignon&author=D%20Angoulvant&volume=10&publication_year=2015&pages=530-9&pmid=26184867&doi=10.1016/j.acvd.2015.06.002&)

3. Hankey GJ, Eikelboom JW. Dabigatran etexilate: a new oral thrombin inhibitor. Circulation. 2011;13:1436–50. doi: 10.1161/CIRCULATIONAHA.110.004424.  [DOI](https://doi.org/10.1161/CIRCULATIONAHA.110.004424) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21464059/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circulation&title=Dabigatran%20etexilate:%20a%20new%20oral%20thrombin%20inhibitor&author=GJ%20Hankey&author=JW%20Eikelboom&volume=13&publication_year=2011&pages=1436-50&pmid=21464059&doi=10.1161/CIRCULATIONAHA.110.004424&)

4. Daly AK, Aithal GP. Genetic regulation of warfarin metabolism and response. Semin Vasc Med. 2003;3:231–8. doi: 10.1055/s-2003-44458.  [DOI](https://doi.org/10.1055/s-2003-44458) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15199455/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Semin%20Vasc%20Med&title=Genetic%20regulation%20of%20warfarin%20metabolism%20and%20response&author=AK%20Daly&author=GP%20Aithal&volume=3&publication_year=2003&pages=231-8&pmid=15199455&doi=10.1055/s-2003-44458&)

5. Rost S, Fregin A, Ivaskevicius V, Conzelmann E, Hortnagel K, Pelz HJ, Lappegard K, Seifried E, Scharrer I, Tuddenham EG, Muller CR, Strom TM, Oldenburg J. Mutations in VKORC1 cause warfarin resistance and multiple coagulation factor deficiency type 2. Nature. 2004;6974:537–41. doi: 10.1038/nature02214.  [DOI](https://doi.org/10.1038/nature02214) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14765194/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Mutations%20in%20VKORC1%20cause%20warfarin%20resistance%20and%20multiple%20coagulation%20factor%20deficiency%20type%202&author=S%20Rost&author=A%20Fregin&author=V%20Ivaskevicius&author=E%20Conzelmann&author=K%20Hortnagel&volume=6974&publication_year=2004&pages=537-41&pmid=14765194&doi=10.1038/nature02214&)

6. Cavallari LH, Shin J, Perera MA. Role of pharmacogenomics in the management of traditional and novel oral anticoagulants. Pharmacotherapy. 2011;12:1192–207. doi: 10.1592/phco.31.12.1192.  [DOI](https://doi.org/10.1592/phco.31.12.1192) | [PMC free article](/articles/PMC3655546/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22122181/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacotherapy&title=Role%20of%20pharmacogenomics%20in%20the%20management%20of%20traditional%20and%20novel%20oral%20anticoagulants&author=LH%20Cavallari&author=J%20Shin&author=MA%20Perera&volume=12&publication_year=2011&pages=1192-207&pmid=22122181&doi=10.1592/phco.31.12.1192&)

7. Gage BF, Eby C, Johnson JA, Deych E, Rieder MJ, Ridker PM, Milligan PE, Grice G, Lenzini P, Rettie AE, Aquilante CL, Grosso L, Marsh S, Langaee T, Farnett LE, Voora D, Veenstra DL, Glynn RJ, Barrett A, McLeod HL. Use of pharmacogenetic and clinical factors to predict the therapeutic dose of warfarin. Clin Pharmacol Ther. 2008;3:326–31. doi: 10.1038/clpt.2008.10.  [DOI](https://doi.org/10.1038/clpt.2008.10) | [PMC free article](/articles/PMC2683977/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18305455/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Use%20of%20pharmacogenetic%20and%20clinical%20factors%20to%20predict%20the%20therapeutic%20dose%20of%20warfarin&author=BF%20Gage&author=C%20Eby&author=JA%20Johnson&author=E%20Deych&author=MJ%20Rieder&volume=3&publication_year=2008&pages=326-31&pmid=18305455&doi=10.1038/clpt.2008.10&)

8. Limdi NA, Beasley TM, Crowley MR, Goldstein JA, Rieder MJ, Flockhart DA, Arnett DK, Acton RT, Liu N. VKORC1 polymorphisms, haplotypes and haplotype groups on warfarin dose among African-Americans and European-Americans. Pharmacogenomics. 2008;10:1445–58. doi: 10.2217/14622416.9.10.1445.  [DOI](https://doi.org/10.2217/14622416.9.10.1445) | [PMC free article](/articles/PMC2586955/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18855533/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=VKORC1%20polymorphisms,%20haplotypes%20and%20haplotype%20groups%20on%20warfarin%20dose%20among%20African-Americans%20and%20European-Americans&author=NA%20Limdi&author=TM%20Beasley&author=MR%20Crowley&author=JA%20Goldstein&author=MJ%20Rieder&volume=10&publication_year=2008&pages=1445-58&pmid=18855533&doi=10.2217/14622416.9.10.1445&)

9. Rieder MJ, Reiner AP, Gage BF, Nickerson DA, Eby CS, McLeod HL, Blough DK, Thummel KE, Veenstra DL, Rettie AE. Effect of VKORC1 haplotypes on transcriptional regulation and warfarin dose. N Engl J Med. 2005;22:2285–93. doi: 10.1056/NEJMoa044503.  [DOI](https://doi.org/10.1056/NEJMoa044503) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15930419/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Effect%20of%20VKORC1%20haplotypes%20on%20transcriptional%20regulation%20and%20warfarin%20dose&author=MJ%20Rieder&author=AP%20Reiner&author=BF%20Gage&author=DA%20Nickerson&author=CS%20Eby&volume=22&publication_year=2005&pages=2285-93&pmid=15930419&doi=10.1056/NEJMoa044503&)

10. Wadelius M, Chen LY, Lindh JD, Eriksson N, Ghori MJ, Bumpstead S, Holm L, McGinnis R, Rane A, Deloukas P. The largest prospective warfarin-treated cohort supports genetic forecasting. Blood. 2009;4:784–92. doi: 10.1182/blood-2008-04-149070.  [DOI](https://doi.org/10.1182/blood-2008-04-149070) | [PMC free article](/articles/PMC2630264/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18574025/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=The%20largest%20prospective%20warfarin-treated%20cohort%20supports%20genetic%20forecasting&author=M%20Wadelius&author=LY%20Chen&author=JD%20Lindh&author=N%20Eriksson&author=MJ%20Ghori&volume=4&publication_year=2009&pages=784-92&pmid=18574025&doi=10.1182/blood-2008-04-149070&)

11. Alvarellos ML, Sangkuhl K, Daneshjou R, Whirl-Carrillo M, Altman RB, Klein TE. PharmGKB summary: very important pharmacogene information for CYP4F2. Pharmacogenet Genomics. 2015;1:41–7. doi: 10.1097/FPC.0000000000000100.  [DOI](https://doi.org/10.1097/FPC.0000000000000100) | [PMC free article](/articles/PMC4261059/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25370453/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=PharmGKB%20summary:%20very%20important%20pharmacogene%20information%20for%20CYP4F2&author=ML%20Alvarellos&author=K%20Sangkuhl&author=R%20Daneshjou&author=M%20Whirl-Carrillo&author=RB%20Altman&volume=1&publication_year=2015&pages=41-7&pmid=25370453&doi=10.1097/FPC.0000000000000100&)

12. Vecsler M, Loebstein R, Almog S, Kurnik D, Goldman B, Halkin H, Gak E. Combined genetic profiles of components and regulators of the vitamin K-dependent gamma-carboxylation system affect individual sensitivity to warfarin. Thromb Haemost. 2006;2:205–11. doi: 10.1160/TH05-06-0446.  [DOI](https://doi.org/10.1160/TH05-06-0446) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16493479/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Thromb%20Haemost&title=Combined%20genetic%20profiles%20of%20components%20and%20regulators%20of%20the%20vitamin%20K-dependent%20gamma-carboxylation%20system%20affect%20individual%20sensitivity%20to%20warfarin&author=M%20Vecsler&author=R%20Loebstein&author=S%20Almog&author=D%20Kurnik&author=B%20Goldman&volume=2&publication_year=2006&pages=205-11&pmid=16493479&doi=10.1160/TH05-06-0446&)

13. Bress A, Patel SR, Perera MA, Campbell RT, Kittles RA, Cavallari LH. Effect of NQO1 and CYP4F2 genotypes on warfarin dose requirements in Hispanic-Americans and African-Americans. Pharmacogenomics. 2012;16:1925–35. doi: 10.2217/pgs.12.164.  [DOI](https://doi.org/10.2217/pgs.12.164) | [PMC free article](/articles/PMC3586586/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23215885/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Effect%20of%20NQO1%20and%20CYP4F2%20genotypes%20on%20warfarin%20dose%20requirements%20in%20Hispanic-Americans%20and%20African-Americans&author=A%20Bress&author=SR%20Patel&author=MA%20Perera&author=RT%20Campbell&author=RA%20Kittles&volume=16&publication_year=2012&pages=1925-35&pmid=23215885&doi=10.2217/pgs.12.164&)

14. Shendre A, Brown TM, Liu N, Hill CE, Beasley TM, Nickerson DA, Limdi NA. Race-specific influence of CYP4F2 on dose and risk of hemorrhage among warfarin users. Pharmacotherapy. 2016;3:263–72. doi: 10.1002/phar.1717.  [DOI](https://doi.org/10.1002/phar.1717) | [PMC free article](/articles/PMC4803610/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26877068/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacotherapy&title=Race-specific%20influence%20of%20CYP4F2%20on%20dose%20and%20risk%20of%20hemorrhage%20among%20warfarin%20users&author=A%20Shendre&author=TM%20Brown&author=N%20Liu&author=CE%20Hill&author=TM%20Beasley&volume=3&publication_year=2016&pages=263-72&pmid=26877068&doi=10.1002/phar.1717&)

15. Dixon AL, Liang L, Moffatt MF, Chen W, Heath S, Wong KC, Taylor J, Burnett E, Gut I, Farrall M, Lathrop GM, Abecasis GR, Cookson WO. A genome-wide association study of global gene expression. Nat Genet. 2007;10:1202–7. doi: 10.1038/ng2109.  [DOI](https://doi.org/10.1038/ng2109) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17873877/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=A%20genome-wide%20association%20study%20of%20global%20gene%20expression&author=AL%20Dixon&author=L%20Liang&author=MF%20Moffatt&author=W%20Chen&author=S%20Heath&volume=10&publication_year=2007&pages=1202-7&pmid=17873877&doi=10.1038/ng2109&)

16. Schadt EE, Molony C, Chudin E, Hao K, Yang X, Lum PY, Kasarskis A, Zhang B, Wang S, Suver C, Zhu J, Millstein J, Sieberts S, Lamb J, GuhaThakurta D, Derry J, Storey JD, Avila-Campillo I, Kruger MJ, Johnson JM. Mapping the genetic architecture of gene expression in human liver. PLoS Biol. 2008;5:e107. doi: 10.1371/journal.pbio.0060107.  [DOI](https://doi.org/10.1371/journal.pbio.0060107) | [PMC free article](/articles/PMC2365981/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18462017/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Biol&title=Mapping%20the%20genetic%20architecture%20of%20gene%20expression%20in%20human%20liver&author=EE%20Schadt&author=C%20Molony&author=E%20Chudin&author=K%20Hao&author=X%20Yang&volume=5&publication_year=2008&pages=e107&pmid=18462017&doi=10.1371/journal.pbio.0060107&)

17. Stranger BE, Montgomery SB, Dimas AS, Parts L, Stegle O, Ingle CE, Sekowska M, Smith GD, Evans D, Gutierrez-Arcelus M, Price A, Raj T, Nisbett J, Nica AC, Beazley C, Durbin R, Deloukas P, Dermitzakis ET. Patterns of cis regulatory variation in diverse human populations. PLoS Genet. 2012;4:e1002639. doi: 10.1371/journal.pgen.1002639.  [DOI](https://doi.org/10.1371/journal.pgen.1002639) | [PMC free article](/articles/PMC3330104/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22532805/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet&title=Patterns%20of%20cis%20regulatory%20variation%20in%20diverse%20human%20populations&author=BE%20Stranger&author=SB%20Montgomery&author=AS%20Dimas&author=L%20Parts&author=O%20Stegle&volume=4&publication_year=2012&pages=e1002639&pmid=22532805&doi=10.1371/journal.pgen.1002639&)

18. Stranger BE, Nica AC, Forrest MS, Dimas A, Bird CP, Beazley C, Ingle CE, Dunning M, Flicek P, Koller D, Montgomery S, Tavare S, Deloukas P, Dermitzakis ET. Population genomics of human gene expression. Nat Genet. 2007;10:1217–24. doi: 10.1038/ng2142.  [DOI](https://doi.org/10.1038/ng2142) | [PMC free article](/articles/PMC2683249/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17873874/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Population%20genomics%20of%20human%20gene%20expression&author=BE%20Stranger&author=AC%20Nica&author=MS%20Forrest&author=A%20Dimas&author=CP%20Bird&volume=10&publication_year=2007&pages=1217-24&pmid=17873874&doi=10.1038/ng2142&)

19. Innocenti F, Cooper GM, Stanaway IB, Gamazon ER, Smith JD, Mirkov S, Ramirez J, Liu W, Lin YS, Moloney C, Aldred SF, Trinklein ND, Schuetz E, Nickerson DA, Thummel KE, Rieder MJ, Rettie AE, Ratain MJ, Cox NJ, Brown CD. Identification, replication, and functional fine-mapping of expression quantitative trait loci in primary human liver tissue. PLoS Genet. 2011;5:e1002078. doi: 10.1371/journal.pgen.1002078.  [DOI](https://doi.org/10.1371/journal.pgen.1002078) | [PMC free article](/articles/PMC3102751/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21637794/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet&title=Identification,%20replication,%20and%20functional%20fine-mapping%20of%20expression%20quantitative%20trait%20loci%20in%20primary%20human%20liver%20tissue&author=F%20Innocenti&author=GM%20Cooper&author=IB%20Stanaway&author=ER%20Gamazon&author=JD%20Smith&volume=5&publication_year=2011&pages=e1002078&pmid=21637794&doi=10.1371/journal.pgen.1002078&)

20. Kass RE, Raftery AE. Bayes Factors. J Am Stat Assoc. 1995;430:773–95.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Am%20Stat%20Assoc&title=Bayes%20Factors&author=RE%20Kass&author=AE%20Raftery&volume=430&publication_year=1995&pages=773-95&)

21. Johnson AD, Handsaker RE, Pulit SL, Nizzari MM, O’Donnell CJ, de Bakker PI. SNAP: a web-based tool for identification and annotation of proxy SNPs using HapMap. Bioinformatics. 2008;24:2938–9. doi: 10.1093/bioinformatics/btn564.  [DOI](https://doi.org/10.1093/bioinformatics/btn564) | [PMC free article](/articles/PMC2720775/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18974171/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=SNAP:%20a%20web-based%20tool%20for%20identification%20and%20annotation%20of%20proxy%20SNPs%20using%20HapMap&author=AD%20Johnson&author=RE%20Handsaker&author=SL%20Pulit&author=MM%20Nizzari&author=CJ%20O%E2%80%99Donnell&volume=24&publication_year=2008&pages=2938-9&pmid=18974171&doi=10.1093/bioinformatics/btn564&)

22. Hernandez W, Gamazon ER, Aquino-Michaels K, Patel S, O’Brien TJ, Harralson AF, Kittles RA, Barbour A, Tuck M, McIntosh SD, Douglas JN, Nicolae D, Cavallari LH, Perera MA. Ethnicity-specific pharmacogenetics: the case of warfarin in African Americans. Pharmacogenomics J. 2014;3:223–8. doi: 10.1038/tpj.2013.34.  [DOI](https://doi.org/10.1038/tpj.2013.34) | [PMC free article](/articles/PMC4016191/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24018621/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=Ethnicity-specific%20pharmacogenetics:%20the%20case%20of%20warfarin%20in%20African%20Americans&author=W%20Hernandez&author=ER%20Gamazon&author=K%20Aquino-Michaels&author=S%20Patel&author=TJ%20O%E2%80%99Brien&volume=3&publication_year=2014&pages=223-8&pmid=24018621&doi=10.1038/tpj.2013.34&)

23. Cavallari LH, Butler C, Langaee TY, Wardak N, Patel SR, Viana MA, Shapiro NL, Nutescu EA. Association of apolipoprotein E genotype with duration of time to achieve a stable warfarin dose in African-American patients. Pharmacotherapy. 2011;8:785–92. doi: 10.1592/phco.31.8.785.  [DOI](https://doi.org/10.1592/phco.31.8.785) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21923605/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacotherapy&title=Association%20of%20apolipoprotein%20E%20genotype%20with%20duration%20of%20time%20to%20achieve%20a%20stable%20warfarin%20dose%20in%20African-American%20patients&author=LH%20Cavallari&author=C%20Butler&author=TY%20Langaee&author=N%20Wardak&author=SR%20Patel&volume=8&publication_year=2011&pages=785-92&pmid=21923605&doi=10.1592/phco.31.8.785&)

24. Hruska MW, Frye RF, Langaee TY. Pyrosequencing method for genotyping cytochrome P450 CYP2C8 and CYP2C9 enzymes. Clin Chem. 2004;12:2392–5. doi: 10.1373/clinchem.2004.040071.  [DOI](https://doi.org/10.1373/clinchem.2004.040071) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15563491/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Chem&title=Pyrosequencing%20method%20for%20genotyping%20cytochrome%20P450%20CYP2C8%20and%20CYP2C9%20enzymes&author=MW%20Hruska&author=RF%20Frye&author=TY%20Langaee&volume=12&publication_year=2004&pages=2392-5&pmid=15563491&doi=10.1373/clinchem.2004.040071&)

25. Perera MA, Gamazon E, Cavallari LH, Patel SR, Poindexter S, Kittles RA, Nicolae D, Cox NJ. The missing association: sequencing-based discovery of novel SNPs in VKORC1 and CYP2C9 that affect warfarin dose in African Americans. Clin Pharmacol Ther. 2011;3:408–15. doi: 10.1038/clpt.2010.322.  [DOI](https://doi.org/10.1038/clpt.2010.322) | [PMC free article](/articles/PMC3625373/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21270790/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=The%20missing%20association:%20sequencing-based%20discovery%20of%20novel%20SNPs%20in%20VKORC1%20and%20CYP2C9%20that%20affect%20warfarin%20dose%20in%20African%20Americans&author=MA%20Perera&author=E%20Gamazon&author=LH%20Cavallari&author=SR%20Patel&author=S%20Poindexter&volume=3&publication_year=2011&pages=408-15&pmid=21270790&doi=10.1038/clpt.2010.322&)

26. Rosenberg NA, Li LM, Ward R, Pritchard JK. Informativeness of genetic markers for inference of ancestry. Am J Hum Genet. 2003;6:1402–22. doi: 10.1086/380416.  [DOI](https://doi.org/10.1086/380416) | [PMC free article](/articles/PMC1180403/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14631557/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=Informativeness%20of%20genetic%20markers%20for%20inference%20of%20ancestry&author=NA%20Rosenberg&author=LM%20Li&author=R%20Ward&author=JK%20Pritchard&volume=6&publication_year=2003&pages=1402-22&pmid=14631557&doi=10.1086/380416&)

27. Purcell S, Neale B, Todd-Brown K, Thomas L, Ferreira MA, Bender D, Maller J, Sklar P, de Bakker PI, Daly MJ, Sham PC. PLINK: a tool set for whole-genome association and population-based linkage analyses. Am J Hum Genet. 2007;3:559–75. doi: 10.1086/519795.  [DOI](https://doi.org/10.1086/519795) | [PMC free article](/articles/PMC1950838/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17701901/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=PLINK:%20a%20tool%20set%20for%20whole-genome%20association%20and%20population-based%20linkage%20analyses&author=S%20Purcell&author=B%20Neale&author=K%20Todd-Brown&author=L%20Thomas&author=MA%20Ferreira&volume=3&publication_year=2007&pages=559-75&pmid=17701901&doi=10.1086/519795&)

28. Cavallari LH, Langaee TY, Momary KM, Shapiro NL, Nutescu EA, Coty WA, Viana MA, Patel SR, Johnson JA. Genetic and clinical predictors of warfarin dose requirements in African Americans. Clin Pharmacol Ther. 2010;4:459–64. doi: 10.1038/clpt.2009.223.  [DOI](https://doi.org/10.1038/clpt.2009.223) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20072124/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Genetic%20and%20clinical%20predictors%20of%20warfarin%20dose%20requirements%20in%20African%20Americans&author=LH%20Cavallari&author=TY%20Langaee&author=KM%20Momary&author=NL%20Shapiro&author=EA%20Nutescu&volume=4&publication_year=2010&pages=459-64&pmid=20072124&doi=10.1038/clpt.2009.223&)

29. Pritchard JK, Stephens M, Donnelly P. Inference of population structure using multilocus genotype data. Genetics. 2000;2:945–59. doi: 10.1093/genetics/155.2.945.  [DOI](https://doi.org/10.1093/genetics/155.2.945) | [PMC free article](/articles/PMC1461096/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10835412/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genetics&title=Inference%20of%20population%20structure%20using%20multilocus%20genotype%20data&author=JK%20Pritchard&author=M%20Stephens&author=P%20Donnelly&volume=2&publication_year=2000&pages=945-59&pmid=10835412&doi=10.1093/genetics/155.2.945&)

30. Voora D, Koboldt DC, King CR, Lenzini PA, Eby CS, Porche-Sorbet R, Deych E, Crankshaw M, Milligan PE, McLeod HL, Patel SR, Cavallari LH, Ridker PM, Grice GR, Miller RD, Gage BF. A polymorphism in the VKORC1 regulator calumenin predicts higher warfarin dose requirements in African Americans. Clin Pharmacol Ther. 2010;4:445–51. doi: 10.1038/clpt.2009.291.  [DOI](https://doi.org/10.1038/clpt.2009.291) | [PMC free article](/articles/PMC2928561/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20200517/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=A%20polymorphism%20in%20the%20VKORC1%20regulator%20calumenin%20predicts%20higher%20warfarin%20dose%20requirements%20in%20African%20Americans&author=D%20Voora&author=DC%20Koboldt&author=CR%20King&author=PA%20Lenzini&author=CS%20Eby&volume=4&publication_year=2010&pages=445-51&pmid=20200517&doi=10.1038/clpt.2009.291&)

31. Cavallari LH, Momary KM, Patel SR, Shapiro NL, Nutescu E, Viana MA. Pharmacogenomics of warfarin dose requirements in Hispanics. Blood Cells Mol Dis. 2011;2:147–50. doi: 10.1016/j.bcmd.2010.11.005.  [DOI](https://doi.org/10.1016/j.bcmd.2010.11.005) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21185752/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood%20Cells%20Mol%20Dis&title=Pharmacogenomics%20of%20warfarin%20dose%20requirements%20in%20Hispanics&author=LH%20Cavallari&author=KM%20Momary&author=SR%20Patel&author=NL%20Shapiro&author=E%20Nutescu&volume=2&publication_year=2011&pages=147-50&pmid=21185752&doi=10.1016/j.bcmd.2010.11.005&)

32. Limdi NA, Wadelius M, Cavallari L, Eriksson N, Crawford DC, Lee MT, Chen CH, Motsinger-Reif A, Sagreiya H, Liu N, Wu AH, Gage BF, Jorgensen A, Pirmohamed M, Shin JG, Suarez-Kurtz G, Kimmel SE, Johnson JA, Klein TE, Wagner MJ. Warfarin pharmacogenetics: a single VKORC1 polymorphism is predictive of dose across 3 racial groups. Blood. 2010;18:3827–34. doi: 10.1182/blood-2009-12-255992.  [DOI](https://doi.org/10.1182/blood-2009-12-255992) | [PMC free article](/articles/PMC2865873/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20203262/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Warfarin%20pharmacogenetics:%20a%20single%20VKORC1%20polymorphism%20is%20predictive%20of%20dose%20across%203%20racial%20groups&author=NA%20Limdi&author=M%20Wadelius&author=L%20Cavallari&author=N%20Eriksson&author=DC%20Crawford&volume=18&publication_year=2010&pages=3827-34&pmid=20203262&doi=10.1182/blood-2009-12-255992&)

33. International Warfarin Pharmacogenetics Consortium. Klein TE, Altman RB, Eriksson N, Gage BF, Kimmel SE, Lee MT, Limdi NA, Page D, Roden DM, Wagner MJ, Caldwell MD, Johnson JA. Estimation of the warfarin dose with clinical and pharmacogenetic data. N Engl J Med. 2009;8:753–64. doi: 10.1056/NEJMoa0809329.  [DOI](https://doi.org/10.1056/NEJMoa0809329) | [PMC free article](/articles/PMC2722908/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19228618/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Estimation%20of%20the%20warfarin%20dose%20with%20clinical%20and%20pharmacogenetic%20data&author=TE%20Klein&author=RB%20Altman&author=N%20Eriksson&author=BF%20Gage&author=SE%20Kimmel&volume=8&publication_year=2009&pages=753-64&pmid=19228618&doi=10.1056/NEJMoa0809329&)

34. Glubb DM, Dholakia N, Innocenti F. Liver expression quantitative trait loci: a foundation for pharmacogenomic research. Front Genet. 2012;3:153. doi: 10.3389/fgene.2012.00153.  [DOI](https://doi.org/10.3389/fgene.2012.00153) | [PMC free article](/articles/PMC3418580/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22912647/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front%20Genet&title=Liver%20expression%20quantitative%20trait%20loci:%20a%20foundation%20for%20pharmacogenomic%20research&author=DM%20Glubb&author=N%20Dholakia&author=F%20Innocenti&volume=3&publication_year=2012&pages=153&pmid=22912647&doi=10.3389/fgene.2012.00153&)

35. Hamadeh IS, Shahin MH, Lima SM, Oliveira F, Wilson L, Khalifa SI, Langaee TY, Cooper-DeHoff RM, Cavallari LH, Johnson JA. Impact of GGCX, STX1B and FPGS polymorphisms on warfarin dose requirements in European-Americans and Egyptians. Clin Transl Sci. 2016;1:36–42. doi: 10.1111/cts.12385.  [DOI](https://doi.org/10.1111/cts.12385) | [PMC free article](/articles/PMC4760888/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26751406/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Transl%20Sci&title=Impact%20of%20GGCX,%20STX1B%20and%20FPGS%20polymorphisms%20on%20warfarin%20dose%20requirements%20in%20European-Americans%20and%20Egyptians&author=IS%20Hamadeh&author=MH%20Shahin&author=SM%20Lima&author=F%20Oliveira&author=L%20Wilson&volume=1&publication_year=2016&pages=36-42&pmid=26751406&doi=10.1111/cts.12385&)

36. Parra EJ, Botton MR, Perini JA, Krithika S, Bourgeois S, Johnson TA, Tsunoda T, Pirmohamed M, Wadelius M, Limdi NA, Cavallari LH, Burmester JK, Rettie AE, Klein TE, Johnson JA, Hutz MH, Suarez-Kurtz G. Genome-wide association study of warfarin maintenance dose in a Brazilian sample. Pharmacogenomics. 2015;11:1–11. doi: 10.2217/PGS.15.73.  [DOI](https://doi.org/10.2217/PGS.15.73) | [PMC free article](/articles/PMC4573240/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26265036/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Genome-wide%20association%20study%20of%20warfarin%20maintenance%20dose%20in%20a%20Brazilian%20sample&author=EJ%20Parra&author=MR%20Botton&author=JA%20Perini&author=S%20Krithika&author=S%20Bourgeois&volume=11&publication_year=2015&pages=1-11&pmid=26265036&doi=10.2217/PGS.15.73&)

37. Hamadeh IS, Shahin MH, Lima SM, Oliveira F, Wilson L, Khalifa SI, Langaee TY, Cooper-DeHoff R, Cavallari LH, Johnson JA. Impact of GGCX, STX1B and FPGS polymorphisms on warfarin dose requirements in European-Americans and Egyptians. Clin Transl Sci. 2016;1:36–42. doi: 10.1111/cts.12385.  [DOI](https://doi.org/10.1111/cts.12385) | [PMC free article](/articles/PMC4760888/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26751406/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Transl%20Sci&title=Impact%20of%20GGCX,%20STX1B%20and%20FPGS%20polymorphisms%20on%20warfarin%20dose%20requirements%20in%20European-Americans%20and%20Egyptians&author=IS%20Hamadeh&author=MH%20Shahin&author=SM%20Lima&author=F%20Oliveira&author=L%20Wilson&volume=1&publication_year=2016&pages=36-42&pmid=26751406&doi=10.1111/cts.12385&)

38. Ward LD, Kellis M. HaploReg v4: systematic mining of putative causal variants, cell types, regulators and target genes for human complex traits and disease. Nucleic Acids Res. 2016;D1:D877–81. doi: 10.1093/nar/gkv1340.  [DOI](https://doi.org/10.1093/nar/gkv1340) | [PMC free article](/articles/PMC4702929/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26657631/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=HaploReg%20v4:%20systematic%20mining%20of%20putative%20causal%20variants,%20cell%20types,%20regulators%20and%20target%20genes%20for%20human%20complex%20traits%20and%20disease&author=LD%20Ward&author=M%20Kellis&volume=D1&publication_year=2016&pages=D877-81&pmid=26657631&doi=10.1093/nar/gkv1340&)
