# Replication study of polymorphisms associated with response to methotrexate in patients with rheumatoid arthritis

## Metadata
**Authors:** Rosario López-Rodríguez, Aida Ferreiro-Iglesias, Aurea Lima, Miguel Bernardes, Andrzej Pawlik, Agnieszka Paradowska-Gorycka, Jerzy Świerkot, Ryszard Slezak, Vita Dolžan, Isidoro González-Álvaro, Javier Narváez, Rafael Cáliz, Eva Pérez-Pampín, Antonio Mera-Varela, Laura Vidal-Bralo, José Gorgonio Acuña Ochoa, Carmen Conde, Juan J Gómez-Reino, Antonio González
**Journal:** Scientific Reports
**Date:** 2018 May 9
**DOI:** [10.1038/s41598-018-25634-y](https://doi.org/10.1038/s41598-018-25634-y)
**PMID:** 29743634
**PMCID:** PMC5943457
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5943457/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC5943457/pdf/41598_2018_Article_25634.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC5943457/pdf/41598_2018_Article_25634.pdf)

## Abstract

About 70 genetic studies have already addressed the need of biomarkers to predict the response of patients with rheumatoid arthritis (RA) to methotrexate (MTX) treatment. However, no genetic biomarker has yet been sufficiently validated. Here, we aimed to replicate a selection of 25 SNPs in the largest collection of patients up to date, which consisted of 915 patients treated with MTX. The change in disease activity (measured as ΔDAS28) from baseline was considered the primary outcome. In addition, response according to widely used criteria (EULAR) was taken as secondary outcome. We considered consistency between outcomes, P values accounting for the number of SNPs, and independence from potential confounders for interpretation of the results. Only the rs1801394 SNP in MTRR fulfilled the high association standards. Its minor allele was associated with less improvement than the major allele according to ΔDAS28 (p = 0.0016), and EULAR response (p = 0.004), with independence of sex, age, baseline DAS28, smoking, seropositivity, concomitant corticosteroid use or previous treatments. In addition, previous evidence suggests the association of this SNP with response to MTX in another autoimmune disease, juvenile idiopathic arthritis, and with high intracellular folate levels, which could contribute to poor response.

## Introduction

Methotrexate (MTX) is the most widely prescribed disease-modifying anti-rheumatic drug (DMARD), and MTX monotherapy is the first treatment recommended for most new patients with rheumatoid arthritis (RA)^1,2^. However, a significant decrease in disease activity is not observed at six months of follow-up in a sizable fraction of patients. This variable response calls for biomarkers to predict patients with low chances of benefiting from MTX monotherapy. The biomarkers have been searched specially in the area of pharmacogenetics^3–5^. About 70 studies have explored polymorphisms in candidate genes including the primary target of MTX, the enzyme dihydrofolate reductase (DHFR), which converts dihydrofolate to tetrahydrofolate. Tetrahydrofolate is essential for purine synthesis, and in the biologically active form, 5-methyl-tetrahydrofolate, is an important cofactor in one-carbon metabolism. Candidate genes are involved in the transport of MTX, generation of its active metabolites, MTX polyglutamates, or in folate one-carbon metabolism, purine synthesis, or in the biosynthetic pathway for adenosine whose accumulation could be the main anti-inflammatory effect of MTX treatment^3,4^. However, no reproducible association has emerged yet from these candidate gene studies. This panorama has been completed recently, thanks to the first GWAS, which was performed in RA patients from the North of India^6^. The authors highlighted 7 SNPs with suggestive evidence of association, none of them located in candidate loci, and several SNPs in candidate genes, which were only apparent after specifically looking for them.

Here, we aimed to replicate 25 of the most promising genetic biomarkers previously associated with MTX response. Fourteen were selected from candidate gene studies^3,4,7–21^, and 11 were selected from the recent GWAS^6^. They were analyzed in the largest collection of patients with RA studied to date and with a focus in reproducibility of the results^22^. A SNP in *MTRR*, rs1801394, showed robust association. None of the other SNPs reached our demanding standards.

## Materials and Methods

### Patients

The study population consisted of 956 patients with RA according to the 1987 American College of Rheumatology criteria^23^. They were of European Caucasian origin, and more specifically of Spanish (269), Slovenian (103), Portuguese (230) and Polish (354) origin. The patients were recruited with written informed consent. The study was approved by the Autonomous Research Ethics Committee of Galicia (Ref. 2014/387). All protocols and methods were conducted according with the relevant guidelines (Declaration of Helsinki and the Belmont Report) and regulations (Spanish Law 14/2007 of Biomedical Research). All patients have received MTX monotherapy for 6 months. Evaluations included Disease Activity Score 28 (DAS28) at the start of treatment and at 6 months. DAS28 is a composite index of RA activity including the number of tender joints and swollen joints (28 joints maximum), erythrocyte sedimentation rate, and global patient health status assessment^24^. DAS28, at the start of the treatment and at 6 months, sex and age were used for all analyses. This information was incomplete in 41 patients that were excluded. Main clinical characteristics of the remaining 915 patients are summarized in Table [1](#Tab1).

### Table 1.

| Characteristic | Value | P ΔDAS28 a | P NR b |
| --- | --- | --- | --- |
| Women, N (%) | 738 (80.6) | 0.002 | ns |
| Age at treatment, mean ± SD | 52.7 ± 13.7 | <10−5 | 0.0003 |
| RF positive, N (%) | 555 (60.6) | ns | ns |
| Anti-CCP positivec, N (%) | 501 (54.7) | 0.001 | 0.03 |
| Seropositivity (RF or antiCCP) | 682 (74.5) | 0.002 | ns |
| Erosive arthritis, N (%) | 479 (52.3) | <10−6 | <10−5 |
| Smoking habitc, N (%) | 195 (21.3) | ns | 0.003 |
| Previous DMARD, N (%) | 185 (20.2) | <10−6 | <10−5 |
| MTX maximum dosec, mean ± SD | 18.7 ± 4.6 | ns | ns |
| Concomitant treatment |  |  |  |
| Corticosteroids, N (%) | 660 (72.1) | 0.01 | 0.01 |
| NSAIDsc, N (%) | 490 (53.5) | 0.0004 | <10−5 |
| Baseline DAS28, mean ± SD | 5.3 ± 1.2 | <10−6 | <10−4 |
| DAS28 at 6 months, mean ± SD | 4.0 ± 1.5 |   |   |
| EULAR response at 6 months, N (%) |  |  |  |
| Good responder | 250 (27.3) |   |   |
| Moderate responder | 354 (38.7) |   |   |
| Non-responder | 311 (34.0) |   |   |

Table 1 Caption: Characteristics of the 915 patients with RA and their association with response to MTX.

### SNP selection and testing

We selected 25 SNP for study (Supplementary Table [S1](#MOESM1) for full details and references): 14 SNPs from candidate gene studies, and 11 SNPS from a GWAS. Genotypes were determined by PCR amplification followed by single-base extension with the SNaPshot Multiplex Kit (Applied Biosystems, Foster City, California). The corresponding primers, probes and detailed protocols are available from the authors. Quality control of the results included duplicate assays of 10% of the samples, genotype call rate ≥95%, concordance with the Hardy–Weinberg equilibrium (p > 0.01) and with the SNP frequencies in the HapMap European collection (Utah residents with Northern and Western European ancestry and Iberian populations in Spain taken from [http://browser.1000genomes.org/index.html](http://browser.1000genomes.org/index.html)). One SNP (rs1051266) was excluded due to low call rate.

### Statistical analyses

Response to MTX was evaluated as change in DAS28 at 6 months of follow-up (ΔDAS28 = DAS28baseline − DAS28follow-up) as the primary outcome. In addition, non-responder classification according to the European League Against Rheumatism (EULAR) criteria was also assessed^25^. The EULAR criteria divide patients into three classes based on change in DAS28 from baseline (ΔDAS28) and DAS28 at the time of evaluation: good responders are those with ΔDAS28 ≥ 1.2 and DAS28 ≤ 3.2; non-responders (NR) are all patients with ΔDAS28 ≤ 0.6 and those with ΔDAS28 > 0.6 but ≤1.2 and with DAS28 > 5.1; all the remaining patients are moderate responders. Here, good and moderate responders were grouped as responders (n = 604) to confront with the NR (n = 311). Linear and logistic regression models for ΔDAS28 and EULAR NR were fitted, respectively. Genotypes were considered in accordance with an additive genetic model of minor allele counts (0, 1 or 2). Therefore, positive regression coefficients indicate a better response associated with minor allele additive effects. Effect of each SNP was adjusted for sex, age, DAS28 at baseline, treatment with a previous DMARD and concomitant treatment with corticosteroids. In a second analysis, additional putative confounders were included: seropositivity and smoking status. Data from the four populations were combined by individual participant data metaanalysis applying two-stage methods. They involved estimation of the association parameters for each population as the first stage and the combination of the results of the four populations in the second stage. A main effects linear regression model for each population was applied to ΔDAS28. The EULAR outcome, in turn, was assessed with separate logistic regression for each population. The combination was done as fixed effects, weighting the effect corresponding to each population with the inverse variance method, and as random effects, according to DerSimonian and Laird. The fixed effects results were preferred except when heterogeneity was moderate (I^2^ > 50), in which case the random effects metaanalysis was selected. Heterogeneity of effect sizes between populations was assessed with Cochran Q test and the inconsistency parameter I^2^. Results were interpreted considering the number of tests following the Bonferroni approach, concordance of results with the two treatment outcomes, independence from confounding variables, and previous evidence of association with response to MTX. Statistica 7.0 (Statsoft, Tulsa, OK, USA) was used for analyses. Post-hoc power analysis was used to assess the study capacity to replicate previous findings. The effect size to replicate was taken from previous studies, considering the EULAR response because this is the most frequently reported outcome in previous studies. The power analysis was conducted using the GPower NT software for the most extreme reported OR^26^.

### Data availability

The datasets generated during the current study are available from the corresponding author on reasonable request

## Results

The 915 RA patients were predominantly female (81%) and DMARD-naïve (80%) with a mean age of 53 years at the start of MTX monotherapy (Table [1](#Tab1)). According to baseline DAS28, many patients presented a high disease activity (DAS28 > 5.1) and most of them received concomitant treatment with corticosteroids or NSAIDs (72.1% and 53.5%, respectively). After 6 months on MTX, only 27.3% of the patients showed a good response according to EULAR criteria, 38.7% showed a moderate response and 34.0% were NR. Several of the clinical and demographic variables were significantly associated with ΔDAS28 or with NR (Table [1](#Tab1)). Specifically, men, seronegative patients, patients lacking erosions, not DMARD-experienced or not receiving concomitant corticosteroids showed larger decrease in DAS28 than the corresponding alternative patient subgroups. Associations with NR were similar (Table [1](#Tab1)).

After genotyping and quality control, one of the 25 SNPs was discarded. Fixed effect metaanalysis of the nested models on the remaining 24 SNPs showed nominal association with ΔDAS28 (Table [2](#Tab2)) of two SNPs from candidate genes (rs2295553 and rs1801394). Only rs1801394 in *MTRR* showed association withstanding Bonferroni correction (*P* = 0.0016). Its minor allele (A) was associated with a lower decrease in DAS28 after MTX treatment than the major allele, specifically, 0.14 less DAS28 units per each A allele according with the additive model. The magnitude of change showed a low level of heterogeneity between the four populations (I^2^ = 24%) that was not significant according with the Q test (P = 0.3) and did not reflect any widely divergent result (Fig. [1](#Fig1)).

### Table 2.

| Locus | SNP | MA | Fixed effects |  | Heterogeneity |  | Random effects |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- |
| β (SE) | P | I2 | P | β (SE) | P |  |  |  |
| Candidate gene |  |  |  |  |  |  |  |  |
| ABCB1 | rs1045642 | A | 0.02 (0.04) | 0.6 | 28 | 0.2 | 0.02 (0.05) | 0.7 |
| ABCC1 | rs35592 | C | 0.03 (0.05) | 0.6 | 71 | 0.02 | −0.04 (0.10) | 0.7 |
| AMPD1 | rs17602729 | A | −0.11 (0.07) | 0.10 | 61 | 0.05 | −0.17 (0.12) | 0.15 |
| ATIC | rs4673990 | G | 0.08 (0.04) | 0.07 | 6 | 0.4 | 0.08 (0.05) | 0.09 |
|   | rs12995526 | A | 0.04 (0.04) | 0.4 | 36 | 0.2 | 0.02 (0.06) | 0.7 |
|   | rs16853834 | T | 0.04 (0.06) | 0.6 | 11 | 0.3 | 0.04 (0.07) | 0.6 |
|   | rs2372536 | G | 0.07 (0.05) | 0.1 | 17 | 0.3 | 0.07 (0.06) | 0.2 |
| ITPA | rs2295553 | C | 0.09 (0.04) | 0.040 | 0 | 0.6 | 0.09 (0.04) | 0.040 |
|   | rs1127354 | A | 0.07 (0.09) | 0.4 | 19 | 0.3 | 0.06 (0.10) | 0.5 |
| MTRR | rs1801394 | A | −0.14 (0.04) | 0.0016 | 24 | 0.3 | −0.15 (0.05) | 0.0056 |
| MTHFD1 | rs2236225 | A | −0.06 (0.05) | 0.2 | 0 | 0.8 | −0.06 (0.05) | 0.2 |
| SLC46A1 | rs2239907 | T | −0.01 (0.04) | 0.7 | 0 | 0.6 | −0.01 (0.04) | 0.7 |
| TYMS | rs699517 | T | −0.03 (0.05) | 0.5 | 57 | 0.07 | 0.00 (0.08) | 1.0 |
| GWAS hit |  |  |  |  |  |  |  |  |
| ARL14/PPM1L | rs7624766 | G | −0.03 (0.05) | 0.5 | 0 | 0.5 | −0.03 (0.05) | 0.5 |
| BMP2 | rs2650972 | T | −0.04 (0.04) | 0.4 | 0 | 0.8 | −0.04 (0.04) | 0.4 |
| DHFR | rs5836788 | del | −0.01 (0.05) | 0.8 | 0 | 0.5 | −0.01 (0.05) | 0.8 |
| 10p15.1 | rs1901633 | G | 0.05 (0.05) | 0.3 | 0 | 0.7 | 0.05 (0.05) | 0.3 |
| 14q13.1 | rs4982133 | A | −0.06 (0.05) | 0.3 | 0 | 1.0 | −0.06 (0.05) | 0.3 |
| 15q26.2 | rs1703794 | C | −0.01 (0.05) | 0.8 | 0 | 0.4 | −0.01 (0.05) | 0.8 |
| 20q13 | rs6064463 | C | −0.06 (0.05) | 0.2 | 11 | 0.3 | −0.06 (0.05) | 0.3 |
| MTRR | rs162040 | C | −0.07 (0.06) | 0.3 | 60 | 0.06 | −0.07 (0.11) | 0.5 |
| PTPRM | rs6506569 | C | 0.02 (0.05) | 0.6 | 0 | 0.8 | 0.02 (0.05) | 0.6 |
| TYMS | rs2244500 | G | 0.02 (0.05) | 0.6 | 64 | 0.04 | 0.07 (0.09) | 0.4 |
|   | rs2847153 | A | 0.02 (0.06) | 0.7 | 0 | 0.5 | 0.02 (0.06) | 0.7 |

Table 2 Caption: Metaanalysis of the association of the 24 SNPs with response to MTX assessed as ΔDAS28 at 6 months in the four populations of RA patients.

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f85d/5943457/16729ab3f797/41598_2018_25634_Fig1_HTML.jpg)

Forest plot of the fixed effects metaanalysis for association of rs1801394 with ΔDAS28 in patients with RA treated with MTX. Square areas are proportional to the weight of the population according to the inverse variance method. Width of the diamond represents the 95% CI of the summary regression coefficient.

The previous results were obtained with adjustment by five clinical variables (sex, age, baseline DAS28, previous DMARD treatment and concomitant corticosteroids), which were available from most of the 915 patients. Multivariate analysis adjusting for two other variables associated with response to MTX (smoking and seropositivity) was only possible in 643 patients (Supplementary Table [S2](#MOESM1)). Therefore, the results were compared at the level of effect sizes, which are independent of the number of samples, not at the level of *P* values, which are very sensitive to the number of patients. The effect size of rs1801394 was almost unaltered (from β = −0.14 to −0.13; *P* = 0.009) after the complete adjusted analysis. No other SNP than rs1801394 showed association (*P* < 0.05) in this set of analyses.

Consideration of the secondary outcome of response to MTX, NR according to EULAR criteria, showed a consistent association of rs1801394 (Table [3](#Tab3)). The A allele showed OR = 1.39 for NR (95% CI 1.11–1.75) without detectable heterogeneity between the populations (I^2^ = 0). There were three other SNPs showing nominal association (rs4673990, rs2372536 and rs2650972) with little heterogeneity between the populations. As for the primary outcome, inclusion of more variables of adjustment did not modify the effect size of rs1801394 (from OR = 1.39 to 1.37) reinforcing the evidence of its independent association with MTX response (Supplementary Table [S2](#MOESM1)). Two of the other SNPs showing association with NR were reinforced in the fully adjusted analysis (rs4673990 and rs2372536), but at the expense of increased heterogeneity between the populations.

### Table 3.

| Locus | SNP | MA | Fixed effects |  | Heterogeneity |  | Random effects |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- |
| OR (95% CI) | P | I2 | P | OR (95% CI) | P |  |  |  |
| Candidate gene |  |  |  |  |  |  |  |  |
| ABCB1 | rs1045642 | A | 0.95 (0.8–1.2) | 0.6 | 22 | 0.3 | 0.97 (0.7–1.2) | 0.8 |
| ABCC1 | rs35592 | C | 1.19 (0.9–1.5) | 0.2 | 61 | 0.05 | 1.24 (0.8–1.9) | 0.3 |
| AMPD1 | rs17602729 | A | 1.08 (0.8–1.5) | 0.6 | 42 | 0.2 | 1.14 (0.7–1.8) | 0.6 |
| ATIC | rs4673990 | G | 0.73 (0.6–0.9) | 0.009 | 0 | 0.5 | 0.73 (0.6–0.9) | 0.009 |
|   | rs12995526 | A | 0.85 (0.7–1.1) | 0.1 | 52 | 0.10 | 0.86 (0.6–1.2) | 0.4 |
|   | rs16853834 | T | 0.91 (0.7–1.3) | 0.6 | 0 | 0.5 | 0.91 (0.7–1.3) | 0.6 |
|   | rs2372536 | G | 0.72 (0.6–0.9) | 0.009 | 8 | 0.4 | 0.72 (0.6–0.9) | 0.014 |
| ITPA | rs2295553 | C | 0.81 (0.6–1.02) | 0.1 | 0 | 1.0 | 0.81 (0.6–1.02) | 0.1 |
|   | rs1127354 | A | 0.74 (0.5–1.1) | 0.2 | 0 | 0.6 | 0.74 (0.5–1.1) | 0.2 |
| MTRR | rs1801394 | A | 1.39 (1.1–1.8) | 0.004 | 0 | 0.6 | 1.39 (1.1–1.8) | 0.004 |
| MTHFD1 | rs2236225 | A | 1.03 (0.8–1.3) | 0.8 | 0 | 0.5 | 1.03 (0.8–1.3) | 0.8 |
| SLC46A1 | rs2239907 | T | 0.96 (0.8–1.2) | 0.7 | 0 | 0.8 | 0.96 (0.8–1.2) | 0.7 |
| TYMS | rs699517 | T | 1.03 (0.8–1.3) | 0.8 | 0 | 0.6 | 1.03 (0.8–1.3) | 0.8 |
| GWAS hit |  |  |  |  |  |  |  |  |
| ARL14/PPM1L | rs7624766 | G | 1.13 (0.9–1.4) | 0.3 | 47 | 0.13 | 1.15 (0.8–1.6) | 0.4 |
| BMP2 | rs2650972 | T | 1.28 (1.02–1.6) | 0.035 | 0 | 0.8 | 1.28 (1.02–1.6) | 0.035 |
| DHFR | rs5836788 | del | 1.00 (0.8–1.3) | 1.0 | 17 | 0.3 | 1.00 (0.8–1.3) | 1.0 |
| 10p15.1 | rs1901633 | G | 0.88 (0.7–1.1) | 0.3 | 0 | 0.9 | 0.88 (0.7–1.1) | 0.3 |
| 14q13.1 | rs4982133 | A | 1.19 (0.9–1.6) | 0.2 | 0 | 0.7 | 1.19 (0.9–1.6) | 0.2 |
| 15q26.2 | rs1703794 | C | 1.12 (0.9–1.5) | 0.4 | 5 | 0.4 | 1.12 (0.8–1.5) | 0.4 |
| 20q13 | rs6064463 | C | 1.17 (0.9–1.5) | 0.2 | 0 | 0.5 | 1.17 (0.9–1.5) | 0.2 |
| MTRR | rs162040 | C | 1.35 (0.98–1.9) | 0.1 | 0 | 1.0 | 1.35 (0.98–1.9) | 0.1 |
| PTPRM | rs6506569 | C | 0.89 (0.7–1.1) | 0.3 | 0 | 0.5 | 0.89 (0.7–1.1) | 0.3 |
| TYMS | rs2244500 | G | 0.89 (0.7–1.1) | 0.4 | 0 | 0.9 | 0.89 (0.7–1.1) | 0.4 |
|   | rs2847153 | A | 1.02 (0.8–1.4) | 0.9 | 0 | 0.7 | 1.02 (0.8–1.4) | 0.9 |

Table 3 Caption: Metaanalysis of the association of the 24 SNPs with response to MTX according to the EULAR criteria at 6 months in the four populations of RA patients.

## Discussion

Here, we have identified a SNP, rs1801394, in *MTRR* associated with response to MTX with all the characteristics we had specified in advance to increase reproducibility^22^. This focus in reproducibility differs from many previous studies on the same field^3–5^. The remaining 23 SNPs from candidate gene studies or from GWAS did not show association at these high standards. These results have the added value of arising from the largest collection of patients analyzed to date in the genetics of response to MTX.

The association of rs1801394 (also called A66G) with response to MTX fulfilled all the requirements of our analysis: its *P* value was low enough to allow for correction by the number of tests, it was consistently observed with the two treatment outcomes (although the NR *P* value was over the corrected threshold), ΔDAS28 and EULAR criteria, and it was independent of the other variables with influence on response to MTX, including differences in disease activity, treatment and sample collection. These requisites were considered necessary to increase reproducibility, which is critical for validation of any biomarker^3–5,22^.

The uncritical use of *P* < 0.05 for claiming association has been shown to lead to many false positive results. Correction by the number of tests, although not without limitations, is widely considered to be very beneficial in genetics of complex diseases^22,27^. Other characteristic that distinguishes our analysis is the adjustment for multiple potential confounding variables. Variables like the baseline level of disease activity or the previous use of DMARDs could have a different distribution in patients with different genotypes and introduce bias in the results as they are strongly associated with MTX response. Another distinctive characteristic of our study is analysis according to the additive genetic model with values 0, 1 and 2 for the number of minor alleles. In selecting the additive model we have followed the approach of the GWAS that have chosen the additive model based in its biological plausibility and statistical reproducibility^27^.

Less consensus exists in the outcome that should be used in the search of biomarkers of response to treatment in RA. ΔDAS28 has all the advantages of quantitative traits in comparison with dichotomous traits for statistics: discrimination, robustness, increased power and reproducibility^28,29^. These considerations motivated our use of ΔDAS28 as the primary outcome for analysis, which is concordant with the large studies and joint initiatives analyzing response to biologic DMARDs^30–32^. The only advantage of dichotomous outcomes is its simplicity. However, this simplicity is misleading because definitions of NR vary widely between studies. In effect, some authors have used the EULAR criteria of response, others have used DAS28-based remission, or low-disease activity, or the ACR 20 and ACR 50 response criteria, and even study-specific definitions of response^4^. As a consequence, it is suspected that heterogeneity in these definitions could contribute to lack of reproducibility. In any case, we decided to include the EULAR criteria as a secondary outcome for completeness.

All the discussed characteristics were intended to increase the reproducibility of our results. However, the association of response to MTX with rs1801394 in *MTRR* is not yet certain. Additional support could be obtained from its credibility as a good candidate^22^. This SNPs causes the substitution of isoleucine for methionine at amino acid 22 of MTRR, 5-methyltetrahydrofolate-homocysteine methyltransferase reductase. This enzyme collaborates with MTR (5-methyltetrahydrofolate-homocysteine methyltransferase) in the transfer of a methyl group from 5-methy tetra-hydro-folic acid to homocysteine, resulting in de novo production of L-methionine and tetra-hydro-folic acid (in a reaction involving vitamin B12 as cofactor). This step in the folic acid cycle precedes the production of the active form of L-methionine, S-adenosyl-methionine, which is the methyl donor in hundreds of biologic transmethylation reactions and the donor of propylamine in polyamine synthesis. Specifically, the A allele, herein associated with poor response to MTX, has been related to a moderate decrease in total plasma levels of homocysteine, which is interpreted as increased capacity to regenerate methionine^33^. More pertinent for our study, patients with RA under MTX treatment displayed folate levels in red blood cells (RBC) that were higher in carriers of the rs1801394 A allele compared to the G/G genotype^16^. This relationship between MTRR and intracellular folate could be linked with the association we have observed because RA patients displaying a decrease in RBC folate (in the form of folate polyglutamates) concentration during MTX treatment showed a marked decrease of DAS28 at 6 months^18^. This indirect relationship seems to indicate that the A allele could decrease response to MTX by leading to increased intracellular folate, which is a factor already suspected due to the interference of folic or folinic acid supplementation with MTX treatment^34–36^. However, the more direct evidence linking rs1801394 with response to MTX comes from previous genetic studies. There have been three studies analyzing rs1801394 as a biomarker of response to MTX in patients with RA^18,19,37^, and two in patients with juvenile idiopathic arthritis (JIA)^38,39^. Two of these studies, one in RA^19^, and one in JIA^38^ have found the same association reported here. The negative studies were either smaller (n = 48 in^18^, and 119 in^39^) or from a different ethnic background (n = 322 Indians in^37^). As the RA study that has found association overlaps with our study^19^, we analyzed the association between rs1801394 and treatment response after excluding the overlapping collection of patients, finding similar effect sizes (n = 664 patients; ΔDAS28 β = −0.19, SE = 0.06, *P* = 0.002; NR OR = 1.33, 95% CI = 1.03–1.7, *P* = 0.030) to those obtained in the total population. Therefore, we can conclude that previous evidence supports a functional relationship of rs1801394 with alterations in the folate pathway and provide suggestive support of its association with response to treatment, although further validation is still needed.

In this study, we analyzed other 23 SNPs from previous candidate gene studies and from a GWAS (Supplementary Table [S1](#MOESM1)). However, none showed an association below the corrected threshold or was associated with the two outcomes at the nominal level of P < 0.05. In addition, three of the four that showed nominal association in any analysis, rs2295553, rs4673990 and rs2372536, were in the opposite direction to previous reports. The only concordant SNP, rs2650972, was just below the 0.05 level for the EULAR outcome (P = 0.035) and far over this threshold for ΔDAS28. This lack of reproducibility is common for candidate gene studies, as signaled by previous systematic revisions and meta-analysis^3–5^. The causes are many, but it is worth to highlight that the candidate gene approach responds to a concept of complex traits that has been demonstrated overly optimistic. In fact, progress in genetics has shown that many complex traits have very polygenic causality with many involved loci, each of them with very small contribution. Therefore, the candidate gene approach has been almost abandoned in favor of GWAS, which are done in much larger sample collections and with strict quality control and requirements to claim association^27^. Unfortunately, the new requirements are hard to fulfill in areas like the response to treatment in RA. This is exemplified by the study of response to TNF inhibitors, where only after multiple consortia it was possible to put together about 3000 patients^30^. The GWAS on response to MTX in comparison only included 457 patients and none of the highlighted loci reached the threshold of significance set for this type of studies^6^. Therefore, it is not surprising that we did not replicate any of the SNPs selected from that GWAS in our study.

Our study shows some limitations that include lack of complete information from all patients, possible confounding by indication common to retrospective studies without randomization, the requirement of 6 months on MTX and insufficient power to detect associations with small effect sizes, which are common in complex traits. However, statistical power of our study was sufficient to replicate previous published effect sizes. Only a SNP did not show power >0.8 to replicate previously findings with *P* < 0.05. This SNP was rs162040, which showed power = 0.71 for detecting association with *P* < 0.05. As the OR of rs162040 in our study was 1.35 without heterogeneity and the OR in the previous study was 1.45 and with similar increase in NR associated with the minor allele^6^, it is possible that lack of power has been an issue in our results for this SNP. This possibility is interesting because rs162040 is also in the *MTRR* locus. The next weakest power was obtained for rs2847153 that showed power = 0.82 for association with *P* < 0.05. However, in this case the effect size in our samples was so low (OR = 1.02) that lack of power did not seem relevant. All the other SNPs showed power ≥0.79 for replication of published effects with *P* < 0.01. Therefore, the association of rs1801394 needs to be replicated in other context not only to confirm its accuracy, but also to know if it extends to other patient groups, including other ethnic backgrounds, and study designs.

In conclusion, our results show only a SNP, rs1801394, in *MTRR* with a robust association with response to MTX monotherapy among the 24 analyzed. This association surpassed all the requirements we have imposed aiming to increase reproducibility and was supported by the study sample size, the largest to date on MTX treatment, and suggestive evidence about the SNP function and previous association data. However, it still needs confirmation. It is to be expected this finding could stimulate further collaborations and efforts to progress in the search of biomarkers predicting the important outcome of MTX treatment.

## Electronic supplementary material

## Acknowledgements

We thank the patients for their generous contribution and Carmen Pena for her excellent technical support and Yolanda Lopez-Golan for her help in the recruitment of patients. This work was supported by the Instituto de Salud Carlos III (Spain) through grants PI14/01651 and RD16/0012/0014 to AG and PI12/01909 to JJG-R. These grants are partially financed by the European Regional Development Fund of the EU (FEDER). R.L.-R. was supported by Instituto de Salud Carlos III (Spain) through a Postdoctoral Contract “Sara Borrell” [CD14/00186].

## Author Contributions

R.L.-R., A.F.-I. and A.G. designed the study. R.L.-R., A.F.-R., A.L., M.B., A.P., A.P.-G., J.S., R.S., V.D., I.G.-A., J.N., R.C., E.P.-P., A.M.-V., L.V.-B., J.G.A.-O., C.C., and J.J.G.-R provided patients or clinical and laboratory data. R.L.-R. and A.G. performed analysis of the results and wrote the first draft of the manuscript. A.F.-R., A.L., M.B., A.P., A.P.-G., J.S., R.S., V.D., I.G.-A., J.N., R.C., E.P.-P., A.M.-V., L.V.-B., J.G.A.-O., C.C., and J.J.G.-R. revised the manuscript for important content. All authors approved the final version of the manuscript.

### Competing Interests

### Competing Interests

The authors declare no competing interests.

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Data Availability Statement

The datasets generated during the current study are available from the corresponding author on reasonable request

### Supplementary Materials

### Data Availability Statement

The datasets generated during the current study are available from the corresponding author on reasonable request

## References

1. Burmester GR, Pope JE. Novel treatment strategies in rheumatoid arthritis. Lancet. 2017;389:2338–2348. doi: 10.1016/S0140-6736(17)31491-5.  [DOI](https://doi.org/10.1016/S0140-6736(17)31491-5) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28612748/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Novel%20treatment%20strategies%20in%20rheumatoid%20arthritis&author=GR%20Burmester&author=JE%20Pope&volume=389&publication_year=2017&pages=2338-2348&pmid=28612748&doi=10.1016/S0140-6736(17)31491-5&)

2. Smolen JS, et al. EULAR recommendations for the management of rheumatoid arthritis with synthetic and biological disease-modifying antirheumatic drugs: 2016 update. Ann Rheum Dis. 2017;76:960–977. doi: 10.1136/annrheumdis-2016-210715.  [DOI](https://doi.org/10.1136/annrheumdis-2016-210715) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28264816/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Rheum%20Dis&title=EULAR%20recommendations%20for%20the%20management%20of%20rheumatoid%20arthritis%20with%20synthetic%20and%20biological%20disease-modifying%20antirheumatic%20drugs:%202016%20update&author=JS%20Smolen&volume=76&publication_year=2017&pages=960-977&pmid=28264816&doi=10.1136/annrheumdis-2016-210715&)

3. Chen Y, Zou K, Sun J, Yang Y, Liu G. Are gene polymorphisms related to treatment outcomes of methotrexate in patients with rheumatoid arthritis? A systematic review and meta-analysis. Pharmacogenomics. 2017;18:175–195. doi: 10.2217/pgs-2016-0158.  [DOI](https://doi.org/10.2217/pgs-2016-0158) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27992285/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Are%20gene%20polymorphisms%20related%20to%20treatment%20outcomes%20of%20methotrexate%20in%20patients%20with%20rheumatoid%20arthritis?%20A%20systematic%20review%20and%20meta-analysis&author=Y%20Chen&author=K%20Zou&author=J%20Sun&author=Y%20Yang&author=G%20Liu&volume=18&publication_year=2017&pages=175-195&pmid=27992285&doi=10.2217/pgs-2016-0158&)

4. Qiu Q, et al. Polymorphisms and Pharmacogenomics for the Clinical Efficacy of Methotrexate in Patients with Rheumatoid Arthritis: A Systematic Review and Meta-analysis. Sci Rep. 2017;7:44015. doi: 10.1038/srep44015.  [DOI](https://doi.org/10.1038/srep44015) | [PMC free article](/articles/PMC5339794/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28266606/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Sci%20Rep&title=Polymorphisms%20and%20Pharmacogenomics%20for%20the%20Clinical%20Efficacy%20of%20Methotrexate%20in%20Patients%20with%20Rheumatoid%20Arthritis:%20A%20Systematic%20Review%20and%20Meta-analysis&author=Q%20Qiu&volume=7&publication_year=2017&pages=44015&pmid=28266606&doi=10.1038/srep44015&)

5. Tarnowski M, et al. The effect of gene polymorphisms on patient responses to rheumatoid arthritis therapy. Expert Opin Drug Metab Toxicol. 2016;12:41–55. doi: 10.1517/17425255.2016.1121233.  [DOI](https://doi.org/10.1517/17425255.2016.1121233) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26609565/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Expert%20Opin%20Drug%20Metab%20Toxicol&title=The%20effect%20of%20gene%20polymorphisms%20on%20patient%20responses%20to%20rheumatoid%20arthritis%20therapy&author=M%20Tarnowski&volume=12&publication_year=2016&pages=41-55&pmid=26609565&doi=10.1517/17425255.2016.1121233&)

6. Senapati S, et al. Genome-wide analysis of methotrexate pharmacogenomics in rheumatoid arthritis shows multiple novel risk variants and leads for TYMS regulation. Pharmacogenet Genomics. 2014;24:211–219. doi: 10.1097/FPC.0000000000000036.  [DOI](https://doi.org/10.1097/FPC.0000000000000036) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24583629/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Genome-wide%20analysis%20of%20methotrexate%20pharmacogenomics%20in%20rheumatoid%20arthritis%20shows%20multiple%20novel%20risk%20variants%20and%20leads%20for%20TYMS%20regulation&author=S%20Senapati&volume=24&publication_year=2014&pages=211-219&pmid=24583629&doi=10.1097/FPC.0000000000000036&)

7. Pawlik A, Wrzesniewska J, Fiedorowicz-Fabrycy I, Gawronska-Szklarz B. The MDR1 3435 polymorphism in patients with rheumatoid arthritis. Int J Clin Pharmacol Ther. 2004;42:496–503. doi: 10.5414/CPP42496.  [DOI](https://doi.org/10.5414/CPP42496) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15487808/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Clin%20Pharmacol%20Ther&title=The%20MDR1%203435%20polymorphism%20in%20patients%20with%20rheumatoid%20arthritis&author=A%20Pawlik&author=J%20Wrzesniewska&author=I%20Fiedorowicz-Fabrycy&author=B%20Gawronska-Szklarz&volume=42&publication_year=2004&pages=496-503&pmid=15487808&doi=10.5414/CPP42496&)

8. Drozdzik M, et al. The effect of 3435C T MDR1 gene polymorphism on rheumatoid arthritis treatment with disease-modifying antirheumatic drugs. Eur J Clin Pharmacol. 2006;62:933–937. doi: 10.1007/s00228-006-0192-1.  [DOI](https://doi.org/10.1007/s00228-006-0192-1) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16932953/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Clin%20Pharmacol&title=The%20effect%20of%203435C%20T%20MDR1%20gene%20polymorphism%20on%20rheumatoid%20arthritis%20treatment%20with%20disease-modifying%20antirheumatic%20drugs&author=M%20Drozdzik&volume=62&publication_year=2006&pages=933-937&pmid=16932953&doi=10.1007/s00228-006-0192-1&)

9. Kooloos WM, et al. Functional polymorphisms and methotrexate treatment outcome in recent-onset rheumatoid arthritis. Pharmacogenomics. 2010;11:163–175. doi: 10.2217/pgs.09.139.  [DOI](https://doi.org/10.2217/pgs.09.139) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20136356/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Functional%20polymorphisms%20and%20methotrexate%20treatment%20outcome%20in%20recent-onset%20rheumatoid%20arthritis&author=WM%20Kooloos&volume=11&publication_year=2010&pages=163-175&pmid=20136356&doi=10.2217/pgs.09.139&)

10. Hinks A, et al. Association of the 5-aminoimidazole-4-carboxamide ribonucleotide transformylase gene with response to methotrexate in juvenile idiopathic arthritis. Ann Rheum Dis. 2011;70:1395–1400. doi: 10.1136/ard.2010.146191.  [DOI](https://doi.org/10.1136/ard.2010.146191) | [PMC free article](/articles/PMC3128324/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21515602/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Rheum%20Dis&title=Association%20of%20the%205-aminoimidazole-4-carboxamide%20ribonucleotide%20transformylase%20gene%20with%20response%20to%20methotrexate%20in%20juvenile%20idiopathic%20arthritis&author=A%20Hinks&volume=70&publication_year=2011&pages=1395-1400&pmid=21515602&doi=10.1136/ard.2010.146191&)

11. Owen SA, et al. Genetic polymorphisms in key methotrexate pathway genes are associated with response to treatment in rheumatoid arthritis patients. Pharmacogenomics J. 2013;13:227–234. doi: 10.1038/tpj.2012.7.  [DOI](https://doi.org/10.1038/tpj.2012.7) | [PMC free article](/articles/PMC5604731/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22450926/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=Genetic%20polymorphisms%20in%20key%20methotrexate%20pathway%20genes%20are%20associated%20with%20response%20to%20treatment%20in%20rheumatoid%20arthritis%20patients&author=SA%20Owen&volume=13&publication_year=2013&pages=227-234&pmid=22450926&doi=10.1038/tpj.2012.7&)

12. Kato T, Hamada A, Mori S, Saito H. Genetic polymorphisms in metabolic and cellular transport pathway of methotrexate impact clinical outcome of methotrexate monotherapy in Japanese patients with rheumatoid arthritis. Drug Metab Pharmacokinet. 2012;27:192–199. doi: 10.2133/dmpk.DMPK-11-RG-066.  [DOI](https://doi.org/10.2133/dmpk.DMPK-11-RG-066) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22104130/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Pharmacokinet&title=Genetic%20polymorphisms%20in%20metabolic%20and%20cellular%20transport%20pathway%20of%20methotrexate%20impact%20clinical%20outcome%20of%20methotrexate%20monotherapy%20in%20Japanese%20patients%20with%20rheumatoid%20arthritis&author=T%20Kato&author=A%20Hamada&author=S%20Mori&author=H%20Saito&volume=27&publication_year=2012&pages=192-199&pmid=22104130&doi=10.2133/dmpk.DMPK-11-RG-066&)

13. Drozdzik M, et al. Reduced folate carrier-1 80G A polymorphism affects methotrexate treatment outcome in rheumatoid arthritis. Pharmacogenomics J. 2007;7:404–407. doi: 10.1038/sj.tpj.6500438.  [DOI](https://doi.org/10.1038/sj.tpj.6500438) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17325736/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=Reduced%20folate%20carrier-1%2080G%20A%20polymorphism%20affects%20methotrexate%20treatment%20outcome%20in%20rheumatoid%20arthritis&author=M%20Drozdzik&volume=7&publication_year=2007&pages=404-407&pmid=17325736&doi=10.1038/sj.tpj.6500438&)

14. Fransen J, et al. Clinical pharmacogenetic model to predict response of MTX monotherapy in patients with established rheumatoid arthritis after DMARD failure. Pharmacogenomics. 2012;13:1087–1094. doi: 10.2217/pgs.12.83.  [DOI](https://doi.org/10.2217/pgs.12.83) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22838954/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Clinical%20pharmacogenetic%20model%20to%20predict%20response%20of%20MTX%20monotherapy%20in%20patients%20with%20established%20rheumatoid%20arthritis%20after%20DMARD%20failure&author=J%20Fransen&volume=13&publication_year=2012&pages=1087-1094&pmid=22838954&doi=10.2217/pgs.12.83&)

15. Wessels JA, et al. A clinical pharmacogenetic model to predict the efficacy of methotrexate monotherapy in recent-onset rheumatoid arthritis. Arthritis Rheum. 2007;56:1765–1775. doi: 10.1002/art.22640.  [DOI](https://doi.org/10.1002/art.22640) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17530705/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arthritis%20Rheum&title=A%20clinical%20pharmacogenetic%20model%20to%20predict%20the%20efficacy%20of%20methotrexate%20monotherapy%20in%20recent-onset%20rheumatoid%20arthritis&author=JA%20Wessels&volume=56&publication_year=2007&pages=1765-1775&pmid=17530705&doi=10.1002/art.22640&)

16. Stamp LK, et al. Polymorphisms within the folate pathway predict folate concentrations but are not associated with disease activity in rheumatoid arthritis patients on methotrexate. Pharmacogenet Genomics. 2010;20:367–376. doi: 10.1097/FPC.0b013e3283398a71.  [DOI](https://doi.org/10.1097/FPC.0b013e3283398a71) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20386493/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Polymorphisms%20within%20the%20folate%20pathway%20predict%20folate%20concentrations%20but%20are%20not%20associated%20with%20disease%20activity%20in%20rheumatoid%20arthritis%20patients%20on%20methotrexate&author=LK%20Stamp&volume=20&publication_year=2010&pages=367-376&pmid=20386493&doi=10.1097/FPC.0b013e3283398a71&)

17. James, H. M. et al. Common polymorphisms in the folate pathway predict efficacy of combination regimens containing methotrexate and sulfasalazine in early rheumatoid arthritis. J Rheumatol 35, 562–571, 08/13/032 [pii] (2008).  [PubMed](https://pubmed.ncbi.nlm.nih.gov/18322994/)

18. Dervieux T, Greenstein N, Kremer J. Pharmacogenomic and metabolic biomarkers in the folate pathway and their association with methotrexate effects during dosage escalation in rheumatoid arthritis. Arthritis Rheum. 2006;54:3095–3103. doi: 10.1002/art.22129.  [DOI](https://doi.org/10.1002/art.22129) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17009228/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arthritis%20Rheum&title=Pharmacogenomic%20and%20metabolic%20biomarkers%20in%20the%20folate%20pathway%20and%20their%20association%20with%20methotrexate%20effects%20during%20dosage%20escalation%20in%20rheumatoid%20arthritis&author=T%20Dervieux&author=N%20Greenstein&author=J%20Kremer&volume=54&publication_year=2006&pages=3095-3103&pmid=17009228&doi=10.1002/art.22129&)

19. Lima A, Bernardes M, Azevedo R, Seabra V, Medeiros R. Moving toward personalized medicine in rheumatoid arthritis: SNPs in methotrexate intracellular pathways are associated with methotrexate therapeutic outcome. Pharmacogenomics. 2016;17:1649–1674. doi: 10.2217/pgs-2016-0067.  [DOI](https://doi.org/10.2217/pgs-2016-0067) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27676277/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Moving%20toward%20personalized%20medicine%20in%20rheumatoid%20arthritis:%20SNPs%20in%20methotrexate%20intracellular%20pathways%20are%20associated%20with%20methotrexate%20therapeutic%20outcome&author=A%20Lima&author=M%20Bernardes&author=R%20Azevedo&author=V%20Seabra&author=R%20Medeiros&volume=17&publication_year=2016&pages=1649-1674&pmid=27676277&doi=10.2217/pgs-2016-0067&)

20. Lima A, Bernardes M, Azevedo R, Medeiros R, Seabra V. Pharmacogenomics of Methotrexate Membrane Transport Pathway: Can Clinical Response to Methotrexate in Rheumatoid Arthritis Be Predicted? Int J Mol Sci. 2015;16:13760–13780. doi: 10.3390/ijms160613760.  [DOI](https://doi.org/10.3390/ijms160613760) | [PMC free article](/articles/PMC4490522/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26086825/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Mol%20Sci&title=Pharmacogenomics%20of%20Methotrexate%20Membrane%20Transport%20Pathway:%20Can%20Clinical%20Response%20to%20Methotrexate%20in%20Rheumatoid%20Arthritis%20Be%20Predicted?&author=A%20Lima&author=M%20Bernardes&author=R%20Azevedo&author=R%20Medeiros&author=V%20Seabra&volume=16&publication_year=2015&pages=13760-13780&pmid=26086825&doi=10.3390/ijms160613760&)

21. Lima A, et al. Prediction of methotrexate clinical response in Portuguese rheumatoid arthritis patients: implication of MTHFR rs1801133 and ATIC rs4673993 polymorphisms. Biomed Res Int. 2014;2014:368681. doi: 10.1155/2014/368681.  [DOI](https://doi.org/10.1155/2014/368681) | [PMC free article](/articles/PMC4055378/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24967362/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biomed%20Res%20Int&title=Prediction%20of%20methotrexate%20clinical%20response%20in%20Portuguese%20rheumatoid%20arthritis%20patients:%20implication%20of%20MTHFR%20rs1801133%20and%20ATIC%20rs4673993%20polymorphisms&author=A%20Lima&volume=2014&publication_year=2014&pages=368681&pmid=24967362&doi=10.1155/2014/368681&)

22. Plenge RM, et al. Recommendations for publication of genetic association studies in Arthritis & Rheumatism. Arthritis Rheum. 2011;63:2839–2847. doi: 10.1002/art.30509.  [DOI](https://doi.org/10.1002/art.30509) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21702018/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arthritis%20Rheum&title=Recommendations%20for%20publication%20of%20genetic%20association%20studies%20in%20Arthritis%20&%20Rheumatism&author=RM%20Plenge&volume=63&publication_year=2011&pages=2839-2847&pmid=21702018&doi=10.1002/art.30509&)

23. Arnett FC, et al. The American Rheumatism Association1987 revised criteria for the classification of rheumatoid arthritis. Arthritis Rheum. 1988;31:315–324. doi: 10.1002/art.1780310302.  [DOI](https://doi.org/10.1002/art.1780310302) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/3358796/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arthritis%20Rheum&title=The%20American%20Rheumatism%20Association1987%20revised%20criteria%20for%20the%20classification%20of%20rheumatoid%20arthritis&author=FC%20Arnett&volume=31&publication_year=1988&pages=315-324&pmid=3358796&doi=10.1002/art.1780310302&)

24. Prevoo ML, et al. Modified disease activity scores that include twenty-eight-joint counts. Development and validation in a prospective longitudinal study of patients with rheumatoid arthritis. Arthritis Rheum. 1995;38:44–48. doi: 10.1002/art.1780380107.  [DOI](https://doi.org/10.1002/art.1780380107) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7818570/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arthritis%20Rheum&title=Modified%20disease%20activity%20scores%20that%20include%20twenty-eight-joint%20counts.%20Development%20and%20validation%20in%20a%20prospective%20longitudinal%20study%20of%20patients%20with%20rheumatoid%20arthritis&author=ML%20Prevoo&volume=38&publication_year=1995&pages=44-48&pmid=7818570&doi=10.1002/art.1780380107&)

25. van Gestel AM, Haagsma CJ, van Riel PL. Validation of rheumatoid arthritis improvement criteria that include simplified joint counts. Arthritis Rheum. 1998;41:1845–1850. doi: 10.1002/1529-0131(199810)41:10<1845::AID-ART17>3.0.CO;2-K.  [DOI](https://doi.org/10.1002/1529-0131(199810)41:10&#x0003c;1845::AID-ART17&#x0003e;3.0.CO;2-K) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9778226/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arthritis%20Rheum&title=Validation%20of%20rheumatoid%20arthritis%20improvement%20criteria%20that%20include%20simplified%20joint%20counts&author=AM%20van%20Gestel&author=CJ%20Haagsma&author=PL%20van%20Riel&volume=41&publication_year=1998&pages=1845-1850&pmid=9778226&doi=10.1002/1529-0131(199810)41:10&#x0003c;1845::AID-ART17&#x0003e;3.0.CO;2-K&)

26. Faul F, Erdfelder E, Lang AG, Buchner A. G*Power 3: a flexible statistical power analysis program for the social, behavioral, and biomedical sciences. Behav Res Methods. 2007;39:175–191. doi: 10.3758/BF03193146.  [DOI](https://doi.org/10.3758/BF03193146) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17695343/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Behav%20Res%20Methods&title=G*Power%203:%20a%20flexible%20statistical%20power%20analysis%20program%20for%20the%20social,%20behavioral,%20and%20biomedical%20sciences&author=F%20Faul&author=E%20Erdfelder&author=AG%20Lang&author=A%20Buchner&volume=39&publication_year=2007&pages=175-191&pmid=17695343&doi=10.3758/BF03193146&)

27. Cantor RM, Lange K, Sinsheimer JS. Prioritizing GWAS results: A review of statistical methods and recommendations for their application. Am J Hum Genet. 2010;86:6–22. doi: 10.1016/j.ajhg.2009.11.017.  [DOI](https://doi.org/10.1016/j.ajhg.2009.11.017) | [PMC free article](/articles/PMC2801749/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20074509/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=Prioritizing%20GWAS%20results:%20A%20review%20of%20statistical%20methods%20and%20recommendations%20for%20their%20application&author=RM%20Cantor&author=K%20Lange&author=JS%20Sinsheimer&volume=86&publication_year=2010&pages=6-22&pmid=20074509&doi=10.1016/j.ajhg.2009.11.017&)

28. Royston P, Altman DG, Sauerbrei W. Dichotomizing continuous predictors in multiple regression: a bad idea. Stat Med. 2006;25:127–141. doi: 10.1002/sim.2331.  [DOI](https://doi.org/10.1002/sim.2331) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16217841/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Stat%20Med&title=Dichotomizing%20continuous%20predictors%20in%20multiple%20regression:%20a%20bad%20idea&author=P%20Royston&author=DG%20Altman&author=W%20Sauerbrei&volume=25&publication_year=2006&pages=127-141&pmid=16217841&doi=10.1002/sim.2331&)

29. MacCallum RC, Zhang S, Preacher KJ, Rucker DD. On the practice of dichotomization of quantitative variables. Psychol Methods. 2002;7:19–40. doi: 10.1037/1082-989X.7.1.19.  [DOI](https://doi.org/10.1037/1082-989X.7.1.19) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11928888/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Psychol%20Methods&title=On%20the%20practice%20of%20dichotomization%20of%20quantitative%20variables&author=RC%20MacCallum&author=S%20Zhang&author=KJ%20Preacher&author=DD%20Rucker&volume=7&publication_year=2002&pages=19-40&pmid=11928888&doi=10.1037/1082-989X.7.1.19&)

30. Sieberts SK, et al. Crowdsourced assessment of common genetic contribution to predicting anti-TNF treatment response in rheumatoid arthritis. Nat Commun. 2016;7:12460. doi: 10.1038/ncomms12460.  [DOI](https://doi.org/10.1038/ncomms12460) | [PMC free article](/articles/PMC4996969/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27549343/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Commun&title=Crowdsourced%20assessment%20of%20common%20genetic%20contribution%20to%20predicting%20anti-TNF%20treatment%20response%20in%20rheumatoid%20arthritis&author=SK%20Sieberts&volume=7&publication_year=2016&pages=12460&pmid=27549343&doi=10.1038/ncomms12460&)

31. Cui J, et al. Genome-wide association study and gene expression analysis identifies CD84 as a predictor of response to etanercept therapy in rheumatoid arthritis. PLoS Genet. 2013;9:e1003394. doi: 10.1371/journal.pgen.1003394.  [DOI](https://doi.org/10.1371/journal.pgen.1003394) | [PMC free article](/articles/PMC3610685/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23555300/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet&title=Genome-wide%20association%20study%20and%20gene%20expression%20analysis%20identifies%20CD84%20as%20a%20predictor%20of%20response%20to%20etanercept%20therapy%20in%20rheumatoid%20arthritis&author=J%20Cui&volume=9&publication_year=2013&pages=e1003394&pmid=23555300&doi=10.1371/journal.pgen.1003394&)

32. Umicevic Mirkov M, et al. Genome-wide association analysis of anti-TNF drug response in patients with rheumatoid arthritis. Ann Rheum Dis. 2013;72:1375–1381. doi: 10.1136/annrheumdis-2012-202405.  [DOI](https://doi.org/10.1136/annrheumdis-2012-202405) | [PMC free article](/articles/PMC4169706/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23233654/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Rheum%20Dis&title=Genome-wide%20association%20analysis%20of%20anti-TNF%20drug%20response%20in%20patients%20with%20rheumatoid%20arthritis&author=M%20Umicevic%20Mirkov&volume=72&publication_year=2013&pages=1375-1381&pmid=23233654&doi=10.1136/annrheumdis-2012-202405&)

33. Gaughan DJ, et al. The methionine synthase reductase (MTRR) A66G polymorphism is a novel genetic determinant of plasma homocysteine concentrations [publishederratum appears in Atherosclerosis 2002; 167:373] Atherosclerosis. 2001;157:451–456. doi: 10.1016/S0021-9150(00)00739-5.  [DOI](https://doi.org/10.1016/S0021-9150(00)00739-5) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11472746/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Atherosclerosis&title=The%20methionine%20synthase%20reductase%20(MTRR)%20A66G%20polymorphism%20is%20a%20novel%20genetic%20determinant%20of%20plasma%20homocysteine%20concentrations%20%5Bpublishederratum%20appears%20in%20Atherosclerosis%202002;%20167:373%5D&author=DJ%20Gaughan&volume=157&publication_year=2001&pages=451-456&pmid=11472746&doi=10.1016/S0021-9150(00)00739-5&)

34. van Ede AE, et al. Effect of folic or folinic acid supplementation on the toxicity and efficacy of methotrexate in rheumatoid arthritis: a forty-eight week, multicenter, randomized, double-blind, placebo-controlled study. Arthritis Rheum. 2001;44:1515–1524. doi: 10.1002/1529-0131(200107)44:7<1515::AID-ART273>3.0.CO;2-7.  [DOI](https://doi.org/10.1002/1529-0131(200107)44:7&#x0003c;1515::AID-ART273&#x0003e;3.0.CO;2-7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11465701/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arthritis%20Rheum&title=Effect%20of%20folic%20or%20folinic%20acid%20supplementation%20on%20the%20toxicity%20and%20efficacy%20of%20methotrexate%20in%20rheumatoid%20arthritis:%20a%20forty-eight%20week,%20multicenter,%20randomized,%20double-blind,%20placebo-controlled%20study&author=AE%20van%20Ede&volume=44&publication_year=2001&pages=1515-1524&pmid=11465701&doi=10.1002/1529-0131(200107)44:7&#x0003c;1515::AID-ART273&#x0003e;3.0.CO;2-7&)

35. Joyce DA, Will RK, Hoffman DM, Laing B, Blackbourn SJ. Exacerbation of rheumatoid arthritis in patients treated with methotrexate after administration of folinic acid. Ann Rheum Dis. 1991;50:913–914. doi: 10.1136/ard.50.12.913.  [DOI](https://doi.org/10.1136/ard.50.12.913) | [PMC free article](/articles/PMC1004580/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/1768158/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Rheum%20Dis&title=Exacerbation%20of%20rheumatoid%20arthritis%20in%20patients%20treated%20with%20methotrexate%20after%20administration%20of%20folinic%20acid&author=DA%20Joyce&author=RK%20Will&author=DM%20Hoffman&author=B%20Laing&author=SJ%20Blackbourn&volume=50&publication_year=1991&pages=913-914&pmid=1768158&doi=10.1136/ard.50.12.913&)

36. Khanna D, et al. Reduction of the efficacy of methotrexate by the use of folic acid: post hoc analysis from two randomized controlled studies. Arthritis Rheum. 2005;52:3030–3038. doi: 10.1002/art.21295.  [DOI](https://doi.org/10.1002/art.21295) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16200612/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arthritis%20Rheum&title=Reduction%20of%20the%20efficacy%20of%20methotrexate%20by%20the%20use%20of%20folic%20acid:%20post%20hoc%20analysis%20from%20two%20randomized%20controlled%20studies&author=D%20Khanna&volume=52&publication_year=2005&pages=3030-3038&pmid=16200612&doi=10.1002/art.21295&)

37. Ghodke-Puranik Y, et al. Folate metabolic pathway single nucleotide polymorphisms: a predictive pharmacogenetic marker of methotrexate response in Indian (Asian) patients with rheumatoid arthritis. Pharmacogenomics. 2015;16:2019–2034. doi: 10.2217/pgs.15.145.  [DOI](https://doi.org/10.2217/pgs.15.145) | [PMC free article](/articles/PMC4976849/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26616421/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Folate%20metabolic%20pathway%20single%20nucleotide%20polymorphisms:%20a%20predictive%20pharmacogenetic%20marker%20of%20methotrexate%20response%20in%20Indian%20(Asian)%20patients%20with%20rheumatoid%20arthritis&author=Y%20Ghodke-Puranik&volume=16&publication_year=2015&pages=2019-2034&pmid=26616421&doi=10.2217/pgs.15.145&)

38. Bulatovic M, et al. Prediction of clinical non-response to methotrexate treatment in juvenile idiopathic arthritis. Ann Rheum Dis. 2012;71:1484–1489. doi: 10.1136/annrheumdis-2011-200942.  [DOI](https://doi.org/10.1136/annrheumdis-2011-200942) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22473625/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Rheum%20Dis&title=Prediction%20of%20clinical%20non-response%20to%20methotrexate%20treatment%20in%20juvenile%20idiopathic%20arthritis&author=M%20Bulatovic&volume=71&publication_year=2012&pages=1484-1489&pmid=22473625&doi=10.1136/annrheumdis-2011-200942&)

39. Zajc Avramovic M, et al. Relationship Between Polymorphisms in Methotrexate Pathway Genes and Outcome of Methotrexate Treatment in a Cohort of 119 Patients with Juvenile Idiopathic Arthritis. J Rheumatol. 2017;44:1216–1223. doi: 10.3899/jrheum.160950.  [DOI](https://doi.org/10.3899/jrheum.160950) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28572465/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Rheumatol&title=Relationship%20Between%20Polymorphisms%20in%20Methotrexate%20Pathway%20Genes%20and%20Outcome%20of%20Methotrexate%20Treatment%20in%20a%20Cohort%20of%20119%20Patients%20with%20Juvenile%20Idiopathic%20Arthritis&author=M%20Zajc%20Avramovic&volume=44&publication_year=2017&pages=1216-1223&pmid=28572465&doi=10.3899/jrheum.160950&)
