# Genetic variants in ALDH1B1 and alcohol dependence risk in a British and Irish population: A bioinformatic and genetic study

## Metadata
**Authors:** Michael J Way, M Adam Ali, Andrew McQuillin, Marsha Y Morgan
**Journal:** PLoS ONE
**Date:** 2017 Jun 8
**DOI:** [10.1371/journal.pone.0177009](https://doi.org/10.1371/journal.pone.0177009)
**PMID:** 28594837
**PMCID:** PMC5464525
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5464525/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC5464525/pdf/pone.0177009.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC5464525/pdf/pone.0177009.pdf)

## Abstract

Alcohol is metabolized in the liver via the enzymes alcohol dehydrogenase (ADH) and aldehyde dehydrogenase (ALDH). Polymorphisms in the genes encoding these enzymes, which are common in East Asian populations, can alter enzyme kinetics and hence the risk of alcohol dependence and its sequelae. One of the most important genetic variants, in this regards, is the single nucleotide polymorphism (SNP) rs671 in ALDH2, the gene encoding the primary acetaldehyde metabolizing enzyme ALDH2. However, the protective allele of rs671 is absent in most Europeans although ALDH1B1, which shares significant sequence homology with ALDH2, contains several, potentially functional, missense SNPs that do occur in European populations. The aims of this study were: (i) to use bioinformatic techniques to characterize the possible effects of selected variants in ALDH1B1 on protein structure and function; and, (ii) to genotype three missense and one stop-gain, protein-altering, non-synonymous SNPs in 1478 alcohol dependent cases and 1254 controls of matched British and Irish ancestry. No significant allelic associations were observed between the three missense SNPs and alcohol dependence risk. The minor allele frequency of rs142427338 (Gln378Ter) was higher in alcohol dependent cases than in controls (allelic P = 0.19, OR = 2.98, [0.62–14.37]) but as this SNP is very rare the study was likely underpowered to detect an association with alcohol dependence risk. This potential association will needs to be further evaluated in other large, independent European populations.

## Introduction

Alcohol is metabolized in the liver to acetaldehyde and acetate *via* the enzymes alcohol dehydrogenase (ADH) and aldehyde dehydrogenase (ALDH). Functional polymorphisms in the genes encoding ADH and ALDH are associated with alterations in enzyme kinetics [[1](#pone.0177009.ref001), [2](#pone.0177009.ref002)], which, in turn, may determine the rates of production and removal of the toxic intermediate metabolite acetaldehyde. High circulating acetaldehyde concentrations are associated with a number of unpleasant systemic effects, including: facial flushing, tachycardia, nausea, headache and even collapse and death [[3](#pone.0177009.ref003), [4](#pone.0177009.ref004)]. In consequence, individuals carrying certain functional *ADH* and *ALDH* variants tend to limit their alcohol consumption and so are, in effect, protected from developing alcohol dependence and the medical consequences of harmful drinking [[5](#pone.0177009.ref005)–[8](#pone.0177009.ref008)].

There are significant differences in the frequencies of these functional *ADH* and *ALDH* variants between ethnic groups. The prevalence of these variants is higher in the Far East and very much lower in Europe. Thus, for example, the missense single nucleotide polymorphism (SNP) rs1229984 (Arg48His) in *ADH1B*, which increases the maximal velocity at which alcohol is oxidized to acetaldehyde by over 100-fold [[3](#pone.0177009.ref003), [9](#pone.0177009.ref009)], is found in 19 to 91% of East-Asians and 10 to 70% of West-Asians [[7](#pone.0177009.ref007)], but at rates ranging from zero to 10% in other populations [[10](#pone.0177009.ref010),[11](#pone.0177009.ref011)]. Nevertheless, there is consistent evidence that the protection against the development of alcohol dependence conferred by carriage of this variant extends to these populations as well [[7](#pone.0177009.ref007),[12](#pone.0177009.ref012)–[15](#pone.0177009.ref015)].

Similarly, a missense SNP, rs671 (Glu504Lys) in *ALDH2*, which inactivates the mitochondrial isoform of ALDH2 causing a loss of up to 90% of its enzymatic activity and 20-fold elevations in circulating acetaldehyde levels following alcohol consumption, is found in 30 to 50% of East-Asians [[16](#pone.0177009.ref016)] but is absent in Europeans [[17](#pone.0177009.ref017)]. However, other genes, for example *ALDH1B1*, have the capacity to oxidize acetaldehyde at physiologically relevant rates [[18](#pone.0177009.ref018)] and may be of importance in relation to alcohol dependence in European populations. *ALDH1B1* is located on chromosome 9 and its encoded protein, ALDH1B1, has 75% amino acid sequence homology with ALDH2. In addition, in common with ALDH2, it is expressed in liver mitochondria and is predicted to form homo-tetramers comprised of four ALDH1B1 monomer subunits [[18](#pone.0177009.ref018)]. Computational analysis predicts that ALDH1B1 monomer subunits preferentially hetero-tetramerize with ALDH2 rather than forming homo-tetramers amongst themselves; this provides a hypothetical explanation for why inactivation of ALDH2 in carriers of the *ALDH2* 504Lys allele is not compensated for by ALDH1B1 [[19](#pone.0177009.ref019)].

A number of missense SNPs have been identified in *ALDH1B1*, which are thought likely to alter the structure of its protein product the enzyme ALDH1B1 [[20](#pone.0177009.ref020)–[22](#pone.0177009.ref022)]. Jackson et al [[19](#pone.0177009.ref019)], undertook computational modelling of human recombinant ALDH1B1 and investigated the functional effects of three non-synonymous SNPs in *ALDH1B1 viz*: rs2073478 (Arg107Leu), rs2228093 (Ala86Val) and rs4878199 (Val253Met) *in vitro*. They showed that presence of the Val86 allele of rs2228093 abolished enzyme activity whereas the other two SNPs appeared to have no effect. The association between two of these SNPs, rs2073478 (Arg107Leu) and rs2228093 (Ala86Val) and alcohol consumption levels and the risk for developing alcohol dependence has been explored but with no robust or replicated findings [[20](#pone.0177009.ref020),[23](#pone.0177009.ref023)–[25](#pone.0177009.ref025)]. Thus, there is some functional evidence suggesting that non-synonymous SNPs in *ALDH1B1* might affect enzyme activity and hence alcohol metabolism. There is also limited and conflicting genetic evidence for their role in modifying the risk for developing alcohol dependence in man.

The primary aims of this study were: (i) to use bioinformatic techniques to identify and characterize non-synonymous SNPs in *ALDH1B1*; and, (ii) to investigate, using a case control design, whether these functional variants associate with alcohol dependence risk in a large, carefully characterized population of British and Irish ancestry.

## Materials and methods

### Bioinformatic analysis

#### Selecting variants in ALDH1B1

A search for nonsense variants with any notated frequency, and missense variants with minor allele frequencies (MAF) exceeding 5% in populations of European ancestry was undertaken in the 1000 Genomes Project build 37 phase I integrated data in the relevant region (chr9: 38,395,746–38,397,299) [[21](#pone.0177009.ref021)].

Four, non-synonymous variants fulfilled the selection criteria: three were missense variants; rs2073478 (Arg107Leu), rs2228093 (Ala86Val) and rs4878199 (Val253Met); and one stop/gain variant, rs142427338 (Gln378Ter). These variants are common in the reference 1000 genomes European ancestry sub-populations [[21](#pone.0177009.ref021)]. The allele frequencies for rs2073478 (Arg107Leu) and rs2228093 (Ala86Val) are largely similar in the British, Toscani and Iberian groups, with the frequency distributions in the Finnish group varying most noticeably from the rest ([S1 Fig](#pone.0177009.s001)).

#### Missense variant functionality prediction

The likelihood that the three missense variants were damaging was investigated using the functional prediction software PolyPhen-2 [[26](#pone.0177009.ref026)]; rs2073478 (Arg107Leu) and rs2228093 (Ala86Val) were predicted to be damaging while the mutation in the third, rs4878199 (Val253Met) was predicted to be benign ([Table 1](#pone.0177009.t001)). This software cannot be used to predict the potential damaging effects of loss of function variants, such as rs142427338 (Gln378Ter), as the rare allele of this SNP encodes a stop codon.

| SNP identity | Amino acid change | PolyPhen-2 score(0–1) | PolyPhen-2 prediction |
| --- | --- | --- | --- |
| rs2073478 | Arg107Leu | 0.46 | Possibly damaging |
| rs2228093 | Ala86Val | 0.99 | Probably damaging |
| rs4878199 | Val253Met | 0 | Benign |

Table Caption: *Adzhubei et al., 2010 [26] Abbreviations: SNP = single nucleotide polymorphism

#### Structural homology modelling

Structural homology models of the monomeric unit of wild type ALDH1B1 (GI: 25777730) were generated using the I-TASSER computational modelling server [[27](#pone.0177009.ref027)–[29](#pone.0177009.ref029)]. The top ranked homology model was derived from an alignment with the human ALDH2 protein structure (PDB 1a4zA) ([S1 Table](#pone.0177009.s002)). The accuracy of this model was estimated by the I-TASSER server using the topology modelling score [[30](#pone.0177009.ref030)]. It was subsequently visualized using UCSF Chimera [[31](#pone.0177009.ref031)]. The model of the wild-type ALDH1B1 was subsequently modified by introducing relevant amino acid substitutions using the rotamer function of UCSF Chimera [[31](#pone.0177009.ref031)] to allow visualization of the positioning of the three missense variants within the overall structure.

Homology modelling was also used to visualize the structural effects of the stop gain variant rs142427338 (Gln378Ter) and hence the potential damaging effects of this SNP.

#### Molecular dynamic simulations

Molecular dynamics simulations were performed on both the wild-type and the rs2228093 (Ala86Val) variant *ALDH1B1* homology models using GROMACS 5.0 [[32](#pone.0177009.ref032), [33](#pone.0177009.ref033)]; the rs2228093 variant was selected for study as it is predicted to be the most damaging of the three missense SNPs. The force-field used in all simulations was OPLA-AA/L all-atom force field [[34](#pone.0177009.ref034)]. The structures were immersed in cubic boxes (size = 10.8 nm3) containing 39842 water molecules. Sodium ions were added for charge neutralization. The Particle Mesh Ewald method [[35](#pone.0177009.ref035)] was used to treat long range electrostatic interactions and the structures relaxed through steepest descent energy minimization runs until the maximum force was less than 1000 kJ.mol-1.nm-2. The solvent and ions were then equilibrated with the starting structure through two steps of molecular dynamics equilibration runs. The first equilibration was run with a constant number of particles, volume and temperature (NVT) over a period of 100 picoseconds; the presence of a stable temperature profile throughout the run confirmed equilibrium. The second equilibration was run with a constant number of particles, pressure and temperature (NPT) over a period of 100 picoseconds; the presence of stable pressure and density over the run confirmed equilibrium. The final production runs of the molecular dynamics were performed over 100 nanoseconds under NPT conditions.

#### Trajectory analysis

The atomic movements within the output trajectory files from the molecular dynamics simulations were analyzed by root mean square deviation (RMSD) and root mean square fluctuation (RMSF) analyses as performed in GROMACS 5.0 [[32](#pone.0177009.ref032),[33](#pone.0177009.ref033)]. RMSF values were calculated between time points of the simulation at equlibrium indicated by a plateau in the trace of the RMSD trajectories. The RMSD and RMSF values of the variant and wild-type structures were directly compared by visualization of the data points over the entire 100 ns of simulation or the amino-acid sequence of ALDH1B1, respectively. The RMSD and RMSF values were also compared by their average values (mean ±1 standard deviation). Trajectories were animated using visual molecular dynamics (VMD) software [[36](#pone.0177009.ref036)].

### Genetic association study

#### Alcohol dependent cases

Individuals attending a variety of UK community and hospital-based services providing support and treatment for alcohol use disorders, between 1997 and 2014, were screened for eligibility. They were included in the study if: (i) they met the criteria for alcohol dependence specified in the *Diagnostic and Statistical Manual of Mental Disorders*, *4th Edition* (DSM-IV) [[37](#pone.0177009.ref037)] or the *International Statistical Classification of Diseases and Related Health Problems*, *10th Revision* (ICD-10) [[38](#pone.0177009.ref038)]; (ii) if they were of English, Scottish, Welsh or Irish but not Jewish descent with a maximum of one grandparent of European ancestry; and, (iii) they were not related to any of the other included subjects.

#### Controls

Ancestrally-matched controls were recruited from London branches of the National Health Service (NHS) blood transfusion service, from General Practitioners’ surgeries, from amongst university students, and from the general public through the National Institute of Health Research (NIHR) funded Mental Health Research Network (MHRN). Individuals were excluded if they met the schedule for affective disorders and schizophrenia life-time version (SADS-L) criteria for a history of depression, bipolar disorder, schizophrenia or alcohol/drug use disorders [[39](#pone.0177009.ref039)]; none of the control subjects had a family history of bipolar disorder, schizophrenia or alcohol dependence; none currently drank alcohol above a weekly maximum of 168 g for men or 112 g for women, nor had done so at any time in the past. In addition, DNA from a separate set of controls of British ancestry, not screened for psychiatric or alcohol use disorders, was purchased from the European Collection of Cell Cultures (ECACC; Health Protection Agency Culture Collections, Salisbury, UK).

None of the cases or controls utilized in this study had participated in any previous studies on ALDH genes polymorphisms or in GWAS studies of alcohol dependence.

#### DNA extraction and genotyping

Genomic DNA was extracted from whole blood from using a standard cell lysis, phenol chloroform technique [[40](#pone.0177009.ref040)]. The ECACC DNA was pre-extracted from transformed lymphoblastoid cell lines.

Primers for genotyping were designed using PrimerPicker software ([http://www.lgcgroup.com/genotyping/](http://www.lgcgroup.com/genotyping/)) ([Table 2](#pone.0177009.t002)). Genotyping for the *ALH1B1* SNPs was carried out, in-house, using fluorescent competitive allele specific PCR reagents (KASPar; LGC Genomics, Hoddesdon, UK). Amplification and detection was undertaken using a LightCycler^®^ 480 real time PCR system (Roche Applied Science, Burgess Hill, UK). Genotype calling was performed automatically by built-in Roche software of cluster plots with some manual editing of calls. Approximately 12% of samples, randomly selected *a priori*, were genotyped in duplicate to ensure consistent allele calling.

| SNP identity | Sequence of primers |
| --- | --- |
| rs2073478 | GAAGGTGACCAAGTTCATGCTCCACTAGGTCTGCCAGGC |
| GAAGGTCGGAGTCAACGGATTGCTCCACTAGGTCTGCCAGGA |  |
| GGTCCCCATGGCGCCGGAT |  |
| AGCGGGGCCGGCTGCTGAA |  |
| rs2228093 | GAAGGTGACCAAGTTCATGCTGTGAAAGCAGCCCGGGAAGC |
| GAAGGTCGGAGTCAACGGATTCGTGAAAGCAGCCCGGGAAGT |  |
| GAGGCATCCATCCGGCGCCAT |  |
| CATGGGGACCCCAGGCGGAA |  |
| rs4878199 | GAAGGTGACCAAGTTCATGCTGAAGGCAACTTTGTCAACATCCAC |
| GAAGGTCGGAGTCAACGGATTGTGAAGGCAACTTTGTCAACATCCAT |  |
| GTGCGGCCATCGCCCAGCA |  |
| CCCAACAGCAGGTGCGGCCAT |  |
| rs142427338 | GAAGGTGACCAAGTTCATGCTAGGCTACATCCAGCTTGGCC |
| GAAGGTCGGAGTCAACGGATTCTAGGCTACATCCAGCTTGGCT |  |
| GCTCTCCGCCACAGAGGAGTTT |  |
| CAGAGGAGTTTTGCGCCCTCCTT |  |

Table Caption: Abbreviations: SNP = Single nucleotide polymorphism

#### Sequencing of rs142427338

Samples identified as carrying the minor allele of the stop gain variant rs142427338 (Gln378Ter), together with samples of uncertain genotype for this SNP, were directly sequenced in order to validate the KASPar genotype calling. Sequencing primers were designed using Primer-BLAST ([http://www.ncbi.nlm.nih.gov/tools/primer-blast/](http://www.ncbi.nlm.nih.gov/tools/primer-blast/)) (Forward–CTGACATGGAGCATGCCGT; Reverse–CAGATCCCGGGTGAACACAG). A standard touchdown PCR protocol was performed [[41](#pone.0177009.ref041)] on 25ng of genomic DNA using a GeneAmp PCR System 9700 machine (Invitrogen, Paisley, UK) with an increased annealing temperature of 68°C to prevent non-specific amplification. PCR amplification success and specificity were determined by visualizing the fragments with agarose gel electrophoresis. Amplified PCR templates were cleaned using PEG precipitation (0.5M NaCl, 1 mM Tris-HCl pH 8.0, 0.1 mM EDTA, PEG 8000 20%, 1.75 mM MgCl2) prior to sequencing. Sequencing reactions were performed using a BigDye terminator kit (Applied Biosystems^®^; Haywards Heath; UK) with an ABI 3730*xl* DNA Analyzer (Applied Biosystems^®^, Life Technologies, California; USA). Sequencing data were read and analyzed using the Staden package [[42](#pone.0177009.ref042)].

#### Data analysis

Test for primary allelic associations, missingness, deviation from Hardy-Weinberg equilibrium (HWE) and linkage disequilbrium were performed using PLINK version 1.9 [[43](#pone.0177009.ref043),[44](#pone.0177009.ref044)]. Samples with conflicting calls (<0.05% of total) were excluded from further analyses.

### Ethics

United Kingdom National Health Service Multicentre Research Ethics Committee approval was granted for this study (MREC/03/11/090). This was ratified by the Research and Development Departments associated with the individual participating centres. Written informed consent was obtained from all subjects prior to inclusion.

## Results

### Bioinformatic analysis

#### Predicted Structure of ALDH1B1

The top ranked predicted structure for the ALDH1B1 monomeric unit had a 95% structural overlap and 76% amino acid sequence identity with the ALDH2 structure 1A4Z. The estimated template modelling score between the homology model structure and the undetermined native structure was 0.69 ± 0.12.

The key structural features of the predicted ALDH1B1 monomer subunit were: (i) a dinucleotide binding domain; (ii) an oligomerization domain; and, (iii) a catalytic domain ([Fig 1A](#pone.0177009.g001); [S1 Video](#pone.0177009.s003)). The three missense variants rs2073478 (Arg107Leu), rs2228093 (Ala86Val) and rs4878199 (Val253Met) are predicted to locate to the co-enzyme domain facing the outer surface of the enzyme ([Fig 1A](#pone.0177009.g001)). The nonsense variant rs142427338 (Gln378Ter) is predicted to result in a premature transcription termination codon within *ALDH1B1* and hence loss of 139 amino acids in the catalytic domain of the enzyme crucial for its function ([Fig 1B](#pone.0177009.g001)).

![Fig 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7096/5464525/03bc67852654/pone.0177009.g001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5464525_pone.0177009.g001.jpg)

This model structure is annotated and rotated to illustrate the positionings of: A) the three misssense variants; and, B) the stop codon variant.

#### Molecular dynamics simulations

The RMSD of the carbon αbackbone of the wild-type ALDH1B1 monomer indicates that the structures attained equilibrium deviation after 10 ns ([Fig 2A](#pone.0177009.g002)). The RMSF indicates that this monomer contains four major regions of structural flexibility *viz*: (i) a region around residues 1–17 in the N-terminal, which lies within the mitochondrial targeting sequence of the enzyme; (ii & iii) two regions around residues 150–170 and 510–517 which, despite being distant in the primary sequence, are sterically close within the monomer structure and encompass the majority of the oligomerization domain; and, (iv) a region around residues 270–280 which contains an alpha-helix α*D*, which is a key component of the dinucleotide binding domain within the Rossman fold ([Fig 2B](#pone.0177009.g002); [S1 Video](#pone.0177009.s003)).

![Fig 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7096/5464525/c9db295f93ea/pone.0177009.g002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5464525_pone.0177009.g002.jpg)

A—The RMSD* of carbon α backbone atoms over the course of the entire 100 ns molecular dynamics simulation for the wild-type (blue) and Val86 variant (red) structure. B—The RMSF** of carbon α backbone atoms over the last 90 ns of the molecular dynamics simulation for the wild-type (blue) and Val86 variant (red) structure. * The root mean squared deviation (RMSD) is a measure the average distance between a number of atomistic positions between a reference structural model and the structural model at a point in time during the simulation. Its variability over the time-course of the simulation indicates the scale of atomistic movement in a structural model and whether it has achieved dynamic equilibrium. ** The root mean squared fluctuation (RMSF) is a calculation of the average RMSD in a well-defined atomistic position, typically atoms present in key regions of the protein structure, over a specified time-period of the simulation. The RMSF highlights differences in movement between in a protein structure over a time period of a simulation and is useful for comparing two or more simulation trajectories.

The simulation trajectory of the Val86 variant monomer differed from that of the wild type (average RMSD ± SD = 0.44 ± 0.055 nm *vs*. average RMSD ± SD = 0.35 ± 0.034 nm respectively), suggesting that the Val86 substitution reduces the structural stability *in silico*. The RMSF data generally mirrored those of the wild-type, except for: (i) a reduction in the flexibility between residues in positions 150–170, which correspond to a region of the oligomerization domain; and, (ii) an increase of the flexibility between residues in positions 270–280, which correspond to a key component of the dinucleotide binding domain.

### Genetic association analysis

#### Genotyping accuracy

Genotyping was completed for all four SNPs with call rates all greater than 95%. All markers followed Hardy Weinberg Equilibrium (HWE; cut-off > 0.05) in cases and controls.

#### Primary allelic associations of ALDH1B1 SNPs

A total of 1478 alcohol dependent cases and 1254 controls were genotyped.

No significant differences were observed in the frequencies of the three missense *ALDH1B1* SNPs between the cases and controls ([Table 3](#pone.0177009.t003)). However, the prevalence of the allele of the rare rs142427338 (Gln378Ter) was higher in the cases (allelic = 0.19, odds ratio [OR] = 2.98, 95% confidence interval [CI] 0.62–14.37) ([Table 3](#pone.0177009.t003)). The variants were in strong linkage disequlibrium.

| SNP | Cohort | Number | Minor allele | Genotype counts |  |  | MAF (%) | Significance(P value)* | OR(95% CI) |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| rs2073478 |   |  |  | TT | TG | GG |   |   |   |
| Case | 1446 | G | 508 | 698 | 240 | 40.7 | 0.75 (1) | 1.03 (0.88–1.20) |  |
| Control | 1244 | 433 | 604 | 207 | 40.9 |  |  |  |  |
| rs2228093 |   |  |  | CC | CT | TT |   |   |   |
| Case | 1450 | T | 1080 | 346 | 24 | 13.6 | 0.91 (1) | 0.99 (0.89–1.11) |  |
| Control | 1245 | 939 | 282 | 24 | 13.3 |  |  |  |  |
| rs4878199 |   |  |  | AA | AG | GG |   |   |   |
| Case | 1415 | G | 1299 | 114 | 2 | 4.2 | 0.42 (1) | 0.90 (0.69–1.17) |  |
| Control | 1232 | 1122 | 106 | 4 | 4.6 |  |  |  |  |
| rs142427338 |   |  |  | CC | CT | TT |   |   |   |
| Case | 1472 | T | 1466 | 6 | 0 | 0.2 | 0.19 (0.77) | 2.98 (0.62–14.37) |  |
| Control | 1252 | 1250 | 2 | 0 | 0.1 |  |  |  |  |

Table Caption: Abbreviations: SNP = Single Nucleotide Polymorphism; MAF = Minor Allele Frequency; χ2 = Chi-squared statistic; OR = Odds Ratio; CI = Confidence Intervals. *The unadjusted significance value of an allelic Fishers exact test; the value in brackets has been adjusted for multiple testing using Bonferroni correction.

## Discussion

The missense variant rs671 in *ALDH2* significantly affects the risk of developing alcohol dependence and its sequelae in East Asian populations but this variant is largely absent in European populations. There is, however, significant sequence homology between *ALDH2* and *ALDH1B1* and similarities in their enzymes cellular positioning within the mitochondrion and in their tendency to forms homo-tetramers *in vitro*. Importantly, however, *ALDH1B1*, unlike *ALDH2*, contains several missense SNPs which are common in European populations. Nevertheless, it is unclear whether these variants affect the processing of acetaldehyde, and hence influence the risk of developing the adverse consequences associated with alcohol consumption. This was addressed in the present study, in which four non-synonymous variants in *ALDH1B1*, three missense and one stop gain, which result in changes to the primary sequence of ALDH1B1, were further characterized using functional prediction software, structural homology modelling and molecular dynamics simulation.

The structure of the ALDH1B1 monomer subunit was predicted using the top-ranked I-TASSER software [[28](#pone.0177009.ref028)]. The output metrics for the final homology model provided a high degree of confidence in the prediction [[30](#pone.0177009.ref030)]. The technique of molecular dynamics simulation was used to infer the flexibility of this predicted ALDH1B1 structure and characterize the effects of the 86Val allelic variant on protein structure and function [[45](#pone.0177009.ref045),[46](#pone.0177009.ref046)]. The simulations revealed four regions of flexibility. The first, located at the N-terminal of the protein (residues 1–17), encompassed the mitochondrial targeting sequence. The simulated flexibility in this mitochondrial targeting sequence agrees with Nuclear Magnetic Resonance (NMR) and crystallography studies, which have experimentally determined freedom of movement in this region in homologous ALDH proteins [[47](#pone.0177009.ref047),[48](#pone.0177009.ref048)]. The second and third regions of flexibility (residues 150–170 and 510–517) lie in the oligomerization domain of ALDH1B1. Flexibility in this domain might reflect the fact that the simulation left this region exposed whereas in the putative ALDH1B1 tetramer the oligomerization domain would likely be buried and hence stabilized within the overall macromolecular structure. The fourth region of flexibility (residues 270–280) contains an alpha-helix α*D* which is a key component of the dinucleotide binding domain. Structural studies of the homologous alpha-helix in ALDH2 comparing the *apo* and *holo* forms of the enzyme, have shown that movement of this alpha-helix is essential for isomerization of bound NAD, a significant and necessary feature of catalysis [[49](#pone.0177009.ref049)].

A comparison of the molecular dynamics trajectory of the predicted structures of the wild-type ALDH1B1 and the missense variant predicted to be most damaging *viz*. rs2228093 (Ala86Val) demonstrates a considerable increase in the flexibility of the 270 to 280 region. As this region of the protein is crucial for the binding of the coenzyme NAD, this observation could explain the catalytic inactivity of this variant previously observed *in vitro* [[19](#pone.0177009.ref019)]. Ultimately, however, this technique does not provide conclusive evidence of a functional effect of this genetic variant on protein structure but does provide a basis for later confirmation by X-ray crystallography or solution NMR. This, not withstanding, other studies have reported [[50](#pone.0177009.ref050)] excellent correlations between molecular dynamics simulation and experimental data.

The variant, rs142427338 (Gln378Ter) in *ALDH1B1* encodes a stop codon rather than another amino-acid residue. Such ‘loss of function’ or ‘nonsense’ variants may exert functional effects at either the transcript level, *via* haploinsufficiency mechanisms, or at the protein level, *via* truncation mechanisms. Haploinsufficiency occurs when the mRNA transcript encoding a nonsense allele is eliminated *via* nonsense mediated decay (NMD). The process of NMD, however, requires splice sites for the detection of premature stop codons and thus does not occur on transcripts of intronless genes [[51](#pone.0177009.ref051)]; as *ALDH1B1* is intronless NMD seems unlikely. Thus, if this transcript is translated and the protein product does not invoke the unfolded protein response, the loss of function allele of *ALDH1B1* could result in loss of a sizeable portion of the final enzyme product encompassing the C-terminal 139 amino acids. This region of the protein contains the enzyme’s catalytic residues and may also be important for tetramer formation. Hence, an ALDH1B1 protein truncated at position 378 would likely be non-functional. Clearly experimental studies would greatly facilitate functional understanding of this variant.

Despite the strength of evidence, from the bioinformatic analyses, for the presence of potentially damaging functional mutations in *ALDH1B1*, these coding variants were not significantly associated with alcohol dependence *per se* in a large well-characterized population of alcohol dependent cases and controls of British and Irish ancestry. However, the prevalence of the 378STOP allele in rs142427338 (Gln378Ter) was higher in cases than in controls although the difference was not significance. The MAFs of the three missense variants ranged from 4.6% to 40.9% and hence this study was sufficiently powered to detect significance between cases and controls. However, the stop gain variant is very rare with a MAF of 0.01; thus this study was likely underpowered to detect genetic association at the current sample size [[52](#pone.0177009.ref052)].

There are very few genetic association data available for comparison. Sherman et al. [[20](#pone.0177009.ref020)], found no association between carriage of either rs2073478 or rs2228093 and alcohol dependence risk in a cohort of 40 people of British ancestry genotyped using PCR-RFLP. Unfortunately, the size of this population severely limits the statistical power at the expected effect size [[53](#pone.0177009.ref053)]. Additionally, the MAFs reported by Sherman et al [[20](#pone.0177009.ref020)] were notably different from those in the present study *viz* 29% and 41% respectively for rs2073478 and 24% and 13.3% respectively for rs2228093; this may reflect inaccuracies in the calculation of the population MAF due to the small size of the cohort and/or to inaccuracies arising from use of PCR-RFLP genotyping, which is prone to partial digestion and genotype reading errors [[54](#pone.0177009.ref054)]. However, the present study is sufficiently powered to allow confidence in the concordant, negative association [[52](#pone.0177009.ref052)].

The relationships between polymorphisms in *ALDH1B1* and alcohol drinking levels in the general population and alcohol-related hypersensitivity reactions have been examined in two large populations of Danish residents of Northern European origin [[23](#pone.0177009.ref023)–[25](#pone.0177009.ref025)]. The reported MAFs for the two SNPs examined were similar to those in the present study *viz* 39% and 41% respectively for rs2073478 and 12% and 13% respectively for rs2228093; thus the studies can be usefully compared.

In the first Danish study Husemoen *et al*. [[23](#pone.0177009.ref023)] obtained self-reported information on alcohol consumption from 1216 participants in the population-based Copenhagen Allergy Study and reported that carriage of the minor allele of rs2228093 was associated with non-drinking and *higher* weekly alcohol intake; these findings are paradoxical as the same directionality of allele effect between these related phenotypes would be expected.

In the second study, Linneberg *et al*. [[25](#pone.0177009.ref025)] combined the information on alcohol consumption obtained from their original cohort of 1216 with similarly collected information from an additional 6784 participants in a replication study. In addition they obtained self-reported information on alcohol hypersensitivity and allergic rhinitis from 957 (79%) of the original 1216 participants and 2419 (36%) of the 6784 replication cohort. In the combined population, carriers of the minor allele of rs2228093 reported significantly *higher* total alcohol consumption and an *increased* prevalence of alcohol hypersensitivity; this finding is again paradoxical as one would expect alcohol hypersensitivity to be associated with lower total alcohol consumption and thus expect a common risk allele between phenotypes. It is unclear why this apparent paradoxical effect occurs, although it is possible that the reported allergic reactions may have been too mild to warrant avoidance of alcohol. However, quantitative information on the severity of the response was not made available.

Emerging evidence suggests that rs2228093 has been under positive selection in East-Asian populations over the past 10,000 years [[55](#pone.0177009.ref055)]. Wang *et al*. [[55](#pone.0177009.ref055)] showed that several functional SNPs in alcohol metabolizing genes *viz* rs1229984 (*ADH1B*), rs671 (*ALDH2*), rs8187929 (*ALDH1A1*), rs3813867 (*CYP2E1*) and rs2031920 (*CYP2E1*), have been under moderate to strong selection, which appears to have been driven by the expansion of agriculture. Although these findings are not directly related to alcohol use phenotypes *per se*, they highlight the inter-related function of alcohol metabolizing genes and in particular the shared factors influencing allele frequencies in these populations. If selection pressures aresimilarly influencing these genes in Europeans, then it is possible that the resultant effects could confound genetic association studies of *ALDH1B1*, potentially explaining the negative findings observed in the present study and the paradoxical findings observed by the Danish groups [[23](#pone.0177009.ref023),[25](#pone.0177009.ref025)]

## Conclusion

In conclusion: bioinformatic techniques have been used to investigate missense SNPs in *ALDH1B1*, in particular the 86Val allele of rs2228093, which is predicted to disrupt the structural flexibility of the protein product, the enzyme ALDH1B1. If, as indicated from the bioinformatic data, these changes influence NAD binding to the catalytic domain of the enzyme, or oligomerization between the enzyme subunits, then they could alter enzyme activity. Functional and molecular cloning studies would be needed to further elucidate the effects of these polymorphisms and, in particular, their physiological role in alcohol pharmacokinetics. Although no genetic associations were identified between the four selected SNPs in *ALDH1B1* and alcohol dependence, the study was likely underpowered in relation to the stop gain variant rs142427338 due to its rarity in this population. Exploration of these *ALDH1B1* variants within other European populations is clearly warranted.

## Supporting information

## Acknowledgments

The research was funded by a PhD studentship award jointly funded by University College London and the Hobson Charity Limited. We thank colleagues associated with the NIHR MHRN for assistance in the identification of cases, obtaining consent and collection of samples at the following NHS trusts: Sandwell Mental Health and Social Care; Northamptonshire Healthcare; Avon and Wiltshire Mental Health Partnership, Sheffield Health and Social Care; Tees Esk and Wear Valleys; Lincolnshire Partnership; Nottinghamshire Healthcare; Central and North West London; South Staffordshire and Shropshire Healthcare; Coventry and Warwickshire; and Dudley and Walsall Mental Health Partnership. Jit Saini, Kush Ruparelia, Radhika Kandaswamy, Alexandra Jarram, Giorgia Quadri, and Niamh O’Brien assisted with the processing of samples and DNA extraction.

## Data Availability

All relevant data are within the paper and its Supporting Information files. Further details may be obtained by contacting the corresponding author (marsha.morgan@ucl.ac.uk).

## Funding Statement

The research was funded by a PhD studentship award jointly funded by University College London and the Hobson Charity Limited.

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Data Availability Statement

All relevant data are within the paper and its Supporting Information files. Further details may be obtained by contacting the corresponding author (marsha.morgan@ucl.ac.uk).

### Supplementary Materials

### Data Availability Statement

All relevant data are within the paper and its Supporting Information files. Further details may be obtained by contacting the corresponding author (marsha.morgan@ucl.ac.uk).

## References

1. Kopun M, Propping P. The kinetics of ethanol absorption and elimination in twins and supplementary repetitive experiments in singleton subjects. Eur J Clin Pharmacol. 1977;11:337–344.  [DOI](https://doi.org/10.1007/BF00566530) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/560303/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Clin%20Pharmacol&title=The%20kinetics%20of%20ethanol%20absorption%20and%20elimination%20in%20twins%20and%20supplementary%20repetitive%20experiments%20in%20singleton%20subjects&author=M%20Kopun&author=P%20Propping&volume=11&publication_year=1977&pages=337-344&pmid=560303&doi=10.1007/BF00566530&)

2. Hurley TD, Edenberg HJ. Genes encoding enzymes involved in ethanol metabolism. Alcohol Res. 2012;34:339–344.  [PMC free article](/articles/PMC3756590/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23134050/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Alcohol%20Res&title=Genes%20encoding%20enzymes%20involved%20in%20ethanol%20metabolism&author=TD%20Hurley&author=HJ%20Edenberg&volume=34&publication_year=2012&pages=339-344&pmid=23134050&)

3. Yin SJ, Bosron WF, Magnes LJ, Li TK. Human liver alcohol dehydrogenase: purification and kinetic characterization of the ß2ß2, ß2ß1, aß2, and ß2γ1 "Oriental" isoenzymes. Biochemistry. 1984;23:5847–5853.  [DOI](https://doi.org/10.1021/bi00319a026) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/6395883/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochemistry&title=Human%20liver%20alcohol%20dehydrogenase:%20purification%20and%20kinetic%20characterization%20of%20the%20%C3%9F2%C3%9F2,%20%C3%9F2%C3%9F1,%20a%C3%9F2,%20and%20%C3%9F2%CE%B31%20%22Oriental%22%20isoenzymes&author=SJ%20Yin&author=WF%20Bosron&author=LJ%20Magnes&author=TK%20Li&volume=23&publication_year=1984&pages=5847-5853&pmid=6395883&doi=10.1021/bi00319a026&)

4. Goedde HW, Agarwal DP. Polymorphisms of aldehyde dehydrogenase and alcohol sensitivity. Enzyme. 1987;37:29–44.  [DOI](https://doi.org/10.1159/000469239) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/3106030/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Enzyme&title=Polymorphisms%20of%20aldehyde%20dehydrogenase%20and%20alcohol%20sensitivity&author=HW%20Goedde&author=DP%20Agarwal&volume=37&publication_year=1987&pages=29-44&pmid=3106030&doi=10.1159/000469239&)

5. Thomasson HR, Edenberg HJ, Crabb DW, Mai XL, Jerome RE, Li TK, et al. Alcohol and aldehyde dehydrogenase genotypes and alcoholism in Chinese men. Am J Hum Genet. 1991;48:677–681.  [PMC free article](/articles/PMC1682953/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/2014795/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=Alcohol%20and%20aldehyde%20dehydrogenase%20genotypes%20and%20alcoholism%20in%20Chinese%20men&author=HR%20Thomasson&author=HJ%20Edenberg&author=DW%20Crabb&author=XL%20Mai&author=RE%20Jerome&volume=48&publication_year=1991&pages=677-681&pmid=2014795&)

6. Chen CC, Lu RB, Chen YC, Wang MF, Chang YC, Li TK, et al. Interaction between the functional polymorphisms of the alcohol-metabolism genes in protection against alcoholism. Am J Hum Genet. 1999;65:795–807. 10.1086/302540  [DOI](https://doi.org/10.1086/302540) | [PMC free article](/articles/PMC1377988/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10441588/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=Interaction%20between%20the%20functional%20polymorphisms%20of%20the%20alcohol-metabolism%20genes%20in%20protection%20against%20alcoholism&author=CC%20Chen&author=RB%20Lu&author=YC%20Chen&author=MF%20Wang&author=YC%20Chang&volume=65&publication_year=1999&pages=795-807&pmid=10441588&doi=10.1086/302540&)

7. Li D, Zhao H, Gelernter J. Strong association of the alcohol dehydrogenase 1B gene (ADH1B) with alcohol dependence and alcohol-induced medical diseases. Biol Psychiatry. 2011;70:504–512. 10.1016/j.biopsych.2011.02.024  [DOI](https://doi.org/10.1016/j.biopsych.2011.02.024) | [PMC free article](/articles/PMC3142297/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21497796/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biol%20Psychiatry&title=Strong%20association%20of%20the%20alcohol%20dehydrogenase%201B%20gene%20(ADH1B)%20with%20alcohol%20dependence%20and%20alcohol-induced%20medical%20diseases&author=D%20Li&author=H%20Zhao&author=J%20Gelernter&volume=70&publication_year=2011&pages=504-512&pmid=21497796&doi=10.1016/j.biopsych.2011.02.024&)

8. Edenberg HJ. Genes contributing to the development of alcoholism: an overview. Alcohol Res. 2012;34:336–338.  [PMC free article](/articles/PMC3711246/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23134049/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Alcohol%20Res&title=Genes%20contributing%20to%20the%20development%20of%20alcoholism:%20an%20overview&author=HJ%20Edenberg&volume=34&publication_year=2012&pages=336-338&pmid=23134049&)

9. Hurley TD, Edenberg HJ, Bosron WF. Expression and kinetic characterization of variants of human b1b1 alcohol dehydrogenase containing substitutions at amino acid 47*. J Biol Chem. 1990;265:16366–16372.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/2398055/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=Expression%20and%20kinetic%20characterization%20of%20variants%20of%20human%20b1b1%20alcohol%20dehydrogenase%20containing%20substitutions%20at%20amino%20acid%2047*&author=TD%20Hurley&author=HJ%20Edenberg&author=WF%20Bosron&volume=265&publication_year=1990&pages=16366-16372&pmid=2398055&)

10. Goedde HW, Agarwal DP, Fritze G, Meier-Tackmann D, Singh S, Beckmann G, et al. Distribution of ADH2 and ALDH2 genotypes in different populations. Hum Genet. 1992;88:344–346.  [DOI](https://doi.org/10.1007/BF00197271) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/1733836/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Genet&title=Distribution%20of%20ADH2%20and%20ALDH2%20genotypes%20in%20different%20populations&author=HW%20Goedde&author=DP%20Agarwal&author=G%20Fritze&author=D%20Meier-Tackmann&author=S%20Singh&volume=88&publication_year=1992&pages=344-346&pmid=1733836&doi=10.1007/BF00197271&)

11. Li H, Mukherjee N, Soundararajan U, Tarnok Z, Barta C, Khaliq S, et al. Geographically separate increases in the frequency of the derived ADH1B*47His allele in eastern and western Asia. Am J Hum Genet. 2007;81:842–846. 10.1086/521201  [DOI](https://doi.org/10.1086/521201) | [PMC free article](/articles/PMC2227934/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17847010/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=Geographically%20separate%20increases%20in%20the%20frequency%20of%20the%20derived%20ADH1B*47His%20allele%20in%20eastern%20and%20western%20Asia&author=H%20Li&author=N%20Mukherjee&author=U%20Soundararajan&author=Z%20Tarnok&author=C%20Barta&volume=81&publication_year=2007&pages=842-846&pmid=17847010&doi=10.1086/521201&)

12. Bierut LJ, Goate AM, Breslau N, Johnson EO, Bertelsen S, Fox L, et al. ADH1B is associated with alcohol dependence and alcohol consumption in populations of European and African ancestry. Mol Psychiatry. 2012;17:445–450. 10.1038/mp.2011.124  [DOI](https://doi.org/10.1038/mp.2011.124) | [PMC free article](/articles/PMC3252425/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21968928/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Psychiatry&title=ADH1B%20is%20associated%20with%20alcohol%20dependence%20and%20alcohol%20consumption%20in%20populations%20of%20European%20and%20African%20ancestry&author=LJ%20Bierut&author=AM%20Goate&author=N%20Breslau&author=EO%20Johnson&author=S%20Bertelsen&volume=17&publication_year=2012&pages=445-450&pmid=21968928&doi=10.1038/mp.2011.124&)

13. Gelernter J, Kranzler HR, Sherva R, Almasy L, Koesterer R, Smith AH, et al. Genome-wide association study of alcohol dependence:significant findings in African- and European-Americans including novel risk loci. Mol Psychiatry. 2014;19:41–49. 10.1038/mp.2013.145  [DOI](https://doi.org/10.1038/mp.2013.145) | [PMC free article](/articles/PMC4165335/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24166409/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Psychiatry&title=Genome-wide%20association%20study%20of%20alcohol%20dependence:significant%20findings%20in%20African-%20and%20European-Americans%20including%20novel%20risk%20loci&author=J%20Gelernter&author=HR%20Kranzler&author=R%20Sherva&author=L%20Almasy&author=R%20Koesterer&volume=19&publication_year=2014&pages=41-49&pmid=24166409&doi=10.1038/mp.2013.145&)

14. Treutlein J, Frank J, Kiefer F, Rietschel M. ADH1B Arg48His allele frequency map: filling in the gap for Central Europe. Biol Psychiatry. 2014;75:e15 10.1016/j.biopsych.2013.05.042  [DOI](https://doi.org/10.1016/j.biopsych.2013.05.042) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24011820/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biol%20Psychiatry&title=ADH1B%20Arg48His%20allele%20frequency%20map:%20filling%20in%20the%20gap%20for%20Central%20Europe&author=J%20Treutlein&author=J%20Frank&author=F%20Kiefer&author=M%20Rietschel&volume=75&publication_year=2014&pages=e15&pmid=24011820&doi=10.1016/j.biopsych.2013.05.042&)

15. Way M, McQuillin A, Saini J, Ruparelia K, Lydall GJ, Guerrini I, et al. Genetic variants in or near ADH1B and ADH1C affect susceptibility to alcohol dependence in a British and Irish population. Addict Biol. 2015;20:594–604. 10.1111/adb.12141  [DOI](https://doi.org/10.1111/adb.12141) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24735490/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Addict%20Biol&title=Genetic%20variants%20in%20or%20near%20ADH1B%20and%20ADH1C%20affect%20susceptibility%20to%20alcohol%20dependence%20in%20a%20British%20and%20Irish%20population&author=M%20Way&author=A%20McQuillin&author=J%20Saini&author=K%20Ruparelia&author=GJ%20Lydall&volume=20&publication_year=2015&pages=594-604&pmid=24735490&doi=10.1111/adb.12141&)

16. Eng MY, Luczak SE, Wall TL. ALDH2, ADH1B, and ADH1C genotypes in Asians: a literature review. Alcohol Res Health. 2007;30:22–27.  [PMC free article](/articles/PMC3860439/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17718397/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Alcohol%20Res%20Health&title=ALDH2,%20ADH1B,%20and%20ADH1C%20genotypes%20in%20Asians:%20a%20literature%20review&author=MY%20Eng&author=SE%20Luczak&author=TL%20Wall&volume=30&publication_year=2007&pages=22-27&pmid=17718397&)

17. Chen YC, Peng GS, Tsao TP, Wang MF, Lu RB, Yin SJ. Pharmacokinetic and pharmacodynamic basis for overcoming acetaldehyde-induced adverse reaction in Asian alcoholics, heterozygous for the variant ALDH2*2 gene allele. Pharmacogenet Genomics. 2009;19:588–599.  [DOI](https://doi.org/10.1097/FPC.0b013e32832ecf2e) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19584771/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Pharmacokinetic%20and%20pharmacodynamic%20basis%20for%20overcoming%20acetaldehyde-induced%20adverse%20reaction%20in%20Asian%20alcoholics,%20heterozygous%20for%20the%20variant%20ALDH2*2%20gene%20allele&author=YC%20Chen&author=GS%20Peng&author=TP%20Tsao&author=MF%20Wang&author=RB%20Lu&volume=19&publication_year=2009&pages=588-599&pmid=19584771&doi=10.1097/FPC.0b013e32832ecf2e&)

18. Stagos D, Chen Y, Brocker C, Donald E, Jackson BC, Orlicky DJ, et al. Aldehyde dehydrogenase 1B1: molecular cloning and characterization of a novel mitochondrial acetaldehyde-metabolizing enzyme. Drug Metab Dispos. 2010;38:1679–1687. 10.1124/dmd.110.034678  [DOI](https://doi.org/10.1124/dmd.110.034678) | [PMC free article](/articles/PMC2957164/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20616185/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Aldehyde%20dehydrogenase%201B1:%20molecular%20cloning%20and%20characterization%20of%20a%20novel%20mitochondrial%20acetaldehyde-metabolizing%20enzyme&author=D%20Stagos&author=Y%20Chen&author=C%20Brocker&author=E%20Donald&author=BC%20Jackson&volume=38&publication_year=2010&pages=1679-1687&pmid=20616185&doi=10.1124/dmd.110.034678&)

19. Jackson BC, Holmes RS, Backos DS, Reigan P, Thompson DC, Vasiliou V. Comparative genomics, molecular evolution and computational modeling of ALDH1B1 and ALDH2. Chem Biol Interact. 2013;202:11–21. 10.1016/j.cbi.2012.11.022  [DOI](https://doi.org/10.1016/j.cbi.2012.11.022) | [PMC free article](/articles/PMC3687035/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23247008/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Chem%20Biol%20Interact&title=Comparative%20genomics,%20molecular%20evolution%20and%20computational%20modeling%20of%20ALDH1B1%20and%20ALDH2&author=BC%20Jackson&author=RS%20Holmes&author=DS%20Backos&author=P%20Reigan&author=DC%20Thompson&volume=202&publication_year=2013&pages=11-21&pmid=23247008&doi=10.1016/j.cbi.2012.11.022&)

20. Sherman D, Davé V, Hsu LC, Peters TJ, Yoshida A. Diverse polymorphism within a short coding region of the human aldehyde dehydrogenase-5 (ALDH5) gene. Hum Genet. 1993;92:477–480.  [DOI](https://doi.org/10.1007/BF00216454) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8244338/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Genet&title=Diverse%20polymorphism%20within%20a%20short%20coding%20region%20of%20the%20human%20aldehyde%20dehydrogenase-5%20(ALDH5)%20gene&author=D%20Sherman&author=V%20Dav%C3%A9&author=LC%20Hsu&author=TJ%20Peters&author=A%20Yoshida&volume=92&publication_year=1993&pages=477-480&pmid=8244338&doi=10.1007/BF00216454&)

21. 1000 Genomes Project Consortium, Abecasis GR, Auton A, Brooks LD, DePristo MA, Durbin RM, et al. An integrated map of genetic variation from 1,092 human genomes. Nature. 2012;491:56–65. 10.1038/nature11632  [DOI](https://doi.org/10.1038/nature11632) | [PMC free article](/articles/PMC3498066/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23128226/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=An%20integrated%20map%20of%20genetic%20variation%20from%201,092%20human%20genomes&author=GR%20Abecasis&author=A%20Auton&author=LD%20Brooks&author=MA%20DePristo&author=RM%20Durbin&volume=491&publication_year=2012&pages=56-65&pmid=23128226&doi=10.1038/nature11632&)

22. Way MJ. Computational modelling of ALDH1B1 tetramer formation and the effect of coding variants. Chem Biol Interact. 2014;207:23 10.1016/j.cbi.2013.10.019  [DOI](https://doi.org/10.1016/j.cbi.2013.10.019) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24211684/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Chem%20Biol%20Interact&title=Computational%20modelling%20of%20ALDH1B1%20tetramer%20formation%20and%20the%20effect%20of%20coding%20variants&author=MJ%20Way&volume=207&publication_year=2014&pages=23&pmid=24211684&doi=10.1016/j.cbi.2013.10.019&)

23. Husemoen LL, Fenger M, Friedrich N, Tolstrup JS, Beenfeldt Fredriksen S, Linneberg A. The association of ADH and ALDH gene variants with alcohol drinking habits and cardiovascular disease risk factors. Alcohol Clin Exp Res. 2008;32:1984–1991. 10.1111/j.1530-0277.2008.00780.x  [DOI](https://doi.org/10.1111/j.1530-0277.2008.00780.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18782342/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Alcohol%20Clin%20Exp%20Res&title=The%20association%20of%20ADH%20and%20ALDH%20gene%20variants%20with%20alcohol%20drinking%20habits%20and%20cardiovascular%20disease%20risk%20factors&author=LL%20Husemoen&author=M%20Fenger&author=N%20Friedrich&author=JS%20Tolstrup&author=S%20Beenfeldt%20Fredriksen&volume=32&publication_year=2008&pages=1984-1991&pmid=18782342&doi=10.1111/j.1530-0277.2008.00780.x&)

24. Husemoen LL, Jørgensen T, Borch-Johnsen K, Hansen T, Pedersen O, Linneberg A. The association of alcohol and alcohol metabolizing gene variants with diabetes and coronary heart disease risk factors in a white population. PloS ONE. 2010;5:e11735 10.1371/journal.pone.0011735  [DOI](https://doi.org/10.1371/journal.pone.0011735) | [PMC free article](/articles/PMC2916825/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20700531/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PloS%20ONE&title=The%20association%20of%20alcohol%20and%20alcohol%20metabolizing%20gene%20variants%20with%20diabetes%20and%20coronary%20heart%20disease%20risk%20factors%20in%20a%20white%20population&author=LL%20Husemoen&author=T%20J%C3%B8rgensen&author=K%20Borch-Johnsen&author=T%20Hansen&author=O%20Pedersen&volume=5&publication_year=2010&pages=e11735&pmid=20700531&doi=10.1371/journal.pone.0011735&)

25. Linneberg A, Gonzalez-Quintela A, Vidal C, Jørgensen T, Fenger M, Hansen T, et al. Genetic determinants of both ethanol and acetaldehyde metabolism influence alcohol hypersensitivity and drinking behaviour among Scandinavians. Clin Exp Allergy. 2010;40:123–130. 10.1111/j.1365-2222.2009.03398.x  [DOI](https://doi.org/10.1111/j.1365-2222.2009.03398.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20205700/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Exp%20Allergy&title=Genetic%20determinants%20of%20both%20ethanol%20and%20acetaldehyde%20metabolism%20influence%20alcohol%20hypersensitivity%20and%20drinking%20behaviour%20among%20Scandinavians&author=A%20Linneberg&author=A%20Gonzalez-Quintela&author=C%20Vidal&author=T%20J%C3%B8rgensen&author=M%20Fenger&volume=40&publication_year=2010&pages=123-130&pmid=20205700&doi=10.1111/j.1365-2222.2009.03398.x&)

26. Adzhubei IA, Schmidt S, Peshkin L, Ramensky VE, Gerasimova A, Bork P, et al. A method and server for predicting damaging missense mutations. Nat Methods. 2010;7:248–249. 10.1038/nmeth0410-248  [DOI](https://doi.org/10.1038/nmeth0410-248) | [PMC free article](/articles/PMC2855889/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20354512/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Methods&title=A%20method%20and%20server%20for%20predicting%20damaging%20missense%20mutations&author=IA%20Adzhubei&author=S%20Schmidt&author=L%20Peshkin&author=VE%20Ramensky&author=A%20Gerasimova&volume=7&publication_year=2010&pages=248-249&pmid=20354512&doi=10.1038/nmeth0410-248&)

27. Zhang Y. I-TASSER server for protein 3D structure prediction. BMC Bioinformatics. 2008;9:40 10.1186/1471-2105-9-40  [DOI](https://doi.org/10.1186/1471-2105-9-40) | [PMC free article](/articles/PMC2245901/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18215316/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Bioinformatics&title=I-TASSER%20server%20for%20protein%203D%20structure%20prediction&author=Y%20Zhang&volume=9&publication_year=2008&pages=40&pmid=18215316&doi=10.1186/1471-2105-9-40&)

28. Roy A, Kucukural A, Zhang Y. I-TASSER: a unified platform for automated protein structure and function prediction. Nat Protoc. 2010;5:725–738. 10.1038/nprot.2010.5  [DOI](https://doi.org/10.1038/nprot.2010.5) | [PMC free article](/articles/PMC2849174/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20360767/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Protoc&title=I-TASSER:%20a%20unified%20platform%20for%20automated%20protein%20structure%20and%20function%20prediction&author=A%20Roy&author=A%20Kucukural&author=Y%20Zhang&volume=5&publication_year=2010&pages=725-738&pmid=20360767&doi=10.1038/nprot.2010.5&)

29. Yang J, Yan R, Roy A, Xu D, Poisson J, Zhang Y. The I-TASSER Suite: protein structure and function prediction. Nat Methods. 2015;12:7–8. 10.1038/nmeth.3213  [DOI](https://doi.org/10.1038/nmeth.3213) | [PMC free article](/articles/PMC4428668/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25549265/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Methods&title=The%20I-TASSER%20Suite:%20protein%20structure%20and%20function%20prediction&author=J%20Yang&author=R%20Yan&author=A%20Roy&author=D%20Xu&author=J%20Poisson&volume=12&publication_year=2015&pages=7-8&pmid=25549265&doi=10.1038/nmeth.3213&)

30. Zhang Y, Skolnick J. Scoring function for automated assessment of protein structure template quality. Proteins. 2004;57:702–710. 10.1002/prot.20264  [DOI](https://doi.org/10.1002/prot.20264) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15476259/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proteins&title=Scoring%20function%20for%20automated%20assessment%20of%20protein%20structure%20template%20quality&author=Y%20Zhang&author=J%20Skolnick&volume=57&publication_year=2004&pages=702-710&pmid=15476259&doi=10.1002/prot.20264&)

31. Pettersen EF, Goddard TD, Huang CC, Couch GS, Greenblatt DM, Meng EC, et al. UCSF Chimera–a visualization system for exploratory research and analysis. J Comput Chem. 2004;25:1605–1612. 10.1002/jcc.20084  [DOI](https://doi.org/10.1002/jcc.20084) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15264254/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Comput%20Chem&title=UCSF%20Chimera%E2%80%93a%20visualization%20system%20for%20exploratory%20research%20and%20analysis&author=EF%20Pettersen&author=TD%20Goddard&author=CC%20Huang&author=GS%20Couch&author=DM%20Greenblatt&volume=25&publication_year=2004&pages=1605-1612&pmid=15264254&doi=10.1002/jcc.20084&)

32. Pronk S, Pall S, Schulz R, Larsson P, Bjelkmar P, Apostolov R, et al. GROMACS 4.5: a high-throughput and highly parallel open source molecular simulation toolkit. Bioinformatics. 2013;29:845–854. 10.1093/bioinformatics/btt055  [DOI](https://doi.org/10.1093/bioinformatics/btt055) | [PMC free article](/articles/PMC3605599/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23407358/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=GROMACS%204.5:%20a%20high-throughput%20and%20highly%20parallel%20open%20source%20molecular%20simulation%20toolkit&author=S%20Pronk&author=S%20Pall&author=R%20Schulz&author=P%20Larsson&author=P%20Bjelkmar&volume=29&publication_year=2013&pages=845-854&pmid=23407358&doi=10.1093/bioinformatics/btt055&)

33. Abraham MJ, Murtola T, Schulz R, Páll S, Smith JC, Hess B, et al. GROMACS: High performance molecular simulations through multi-level parallelism from laptops to supercomputers. SoftwareX. 2015;1:19–25.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=SoftwareX&title=GROMACS:%20High%20performance%20molecular%20simulations%20through%20multi-level%20parallelism%20from%20laptops%20to%20supercomputers&author=MJ%20Abraham&author=T%20Murtola&author=R%20Schulz&author=S%20P%C3%A1ll&author=JC%20Smith&volume=1&publication_year=2015&pages=19-25&)

34. Kaminski GA, Friesner RA, Tirado-Rives J, Jorgensen WL. Evaluation and reparametrization of the OPLS-AA force field for proteins via comparison with accurate quantum chemical calculations on peptides. J Phys Chem B. 2001;105:6474–6487.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Phys%20Chem%20B&title=Evaluation%20and%20reparametrization%20of%20the%20OPLS-AA%20force%20field%20for%20proteins%20via%20comparison%20with%20accurate%20quantum%20chemical%20calculations%20on%20peptides&author=GA%20Kaminski&author=RA%20Friesner&author=J%20Tirado-Rives&author=WL%20Jorgensen&volume=105&publication_year=2001&pages=6474-6487&)

35. Darden T, York D, Pedersen L. Particle mesh Ewald: an Nllog (N) method for Ewald sums in large systems. J Chem Phys. 1993;12:10089–10092.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Chem%20Phys&title=Particle%20mesh%20Ewald:%20an%20Nllog%20(N)%20method%20for%20Ewald%20sums%20in%20large%20systems&author=T%20Darden&author=D%20York&author=L%20Pedersen&volume=12&publication_year=1993&pages=10089-10092&)

36. Humphrey W, Dalke A, Schulten K. VMD: visual molecular dynamics. J Mol Graph. 1996;14:33–8, 27–28.  [DOI](https://doi.org/10.1016/0263-7855(96)00018-5) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8744570/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Mol%20Graph&title=VMD:%20visual%20molecular%20dynamics&author=W%20Humphrey&author=A%20Dalke&author=K%20Schulten&volume=14&publication_year=1996&pages=33-8&pmid=8744570&doi=10.1016/0263-7855(96)00018-5&)

37. American Psychiatric Association. Diagnostic and statistical manual of mental disorders 4th ed Washington DC: American Psychiatric Press; 1994.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Diagnostic%20and%20statistical%20manual%20of%20mental%20disorders&publication_year=1994&)

38. World Health Organisation. The ICD-10 classification of mental and behavioural disorders: clinical descriptions and diagnostic guidelines. Geneva: World Health Organisation; 1992.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=The%20ICD-10%20classification%20of%20mental%20and%20behavioural%20disorders:%20clinical%20descriptions%20and%20diagnostic%20guidelines&publication_year=1992&)

39. Endicott J, Spitzer RL. A diagnostic interview: the schedule for affective disorders and schizophrenia. Arch Gen Psychiatry. 1978;35:837–844.  [DOI](https://doi.org/10.1001/archpsyc.1978.01770310043002) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/678037/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arch%20Gen%20Psychiatry&title=A%20diagnostic%20interview:%20the%20schedule%20for%20affective%20disorders%20and%20schizophrenia&author=J%20Endicott&author=RL%20Spitzer&volume=35&publication_year=1978&pages=837-844&pmid=678037&doi=10.1001/archpsyc.1978.01770310043002&)

40. Sambrook J, Russell DW. Molecular Cloning: a Laboratory Manual Fourth edition ed Cold Spring Harbor, New York: Cold Spring Habour Laboratory Press; 2001.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Cold%20Spring%20Harbor&author=J%20Sambrook&author=DW%20Russell&publication_year=2001&)

41. Korbie DJ, Mattick JS. Touchdown PCR for increased specificity and sensitivity in PCR amplification. Nat Protoc. 2008;3:1452–1456. 10.1038/nprot.2008.133  [DOI](https://doi.org/10.1038/nprot.2008.133) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18772872/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Protoc&title=Touchdown%20PCR%20for%20increased%20specificity%20and%20sensitivity%20in%20PCR%20amplification&author=DJ%20Korbie&author=JS%20Mattick&volume=3&publication_year=2008&pages=1452-1456&pmid=18772872&doi=10.1038/nprot.2008.133&)

42. Bonfield JK, Smith KF, Staden R. A new DNA sequence assembly program. Nucleic Acids Res. 1995;23:4992–4999.  [DOI](https://doi.org/10.1093/nar/23.24.4992) | [PMC free article](/articles/PMC307504/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8559656/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=A%20new%20DNA%20sequence%20assembly%20program&author=JK%20Bonfield&author=KF%20Smith&author=R%20Staden&volume=23&publication_year=1995&pages=4992-4999&pmid=8559656&doi=10.1093/nar/23.24.4992&)

43. Purcell S, Neale B, Todd-Brown K, Thomas L, Ferreira MA, Bender D, et al. PLINK: a tool set for whole-genome association and population-based linkage analyses. Am J Hum Genet. 2007;81:559–575. 10.1086/519795  [DOI](https://doi.org/10.1086/519795) | [PMC free article](/articles/PMC1950838/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17701901/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=PLINK:%20a%20tool%20set%20for%20whole-genome%20association%20and%20population-based%20linkage%20analyses&author=S%20Purcell&author=B%20Neale&author=K%20Todd-Brown&author=L%20Thomas&author=MA%20Ferreira&volume=81&publication_year=2007&pages=559-575&pmid=17701901&doi=10.1086/519795&)

44. Chang CC, Chow CC, Tellier LC, Vattikuti S, Purcell SM, Lee JJ. Second-generation PLINK: rising to the challenge of larger and richer datasets. GigaScience. 2015;4:7 10.1186/s13742-015-0047-8  [DOI](https://doi.org/10.1186/s13742-015-0047-8) | [PMC free article](/articles/PMC4342193/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25722852/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=GigaScience&title=Second-generation%20PLINK:%20rising%20to%20the%20challenge%20of%20larger%20and%20richer%20datasets&author=CC%20Chang&author=CC%20Chow&author=LC%20Tellier&author=S%20Vattikuti&author=SM%20Purcell&volume=4&publication_year=2015&pages=7&pmid=25722852&doi=10.1186/s13742-015-0047-8&)

45. Xin YN, Zhao Y, Lin ZH, Jiang X, Xuan SY, Huang J. Molecular dynamics simulation of PNPLA3 I148M polymorphism reveals reduced substrate access to the catalytic cavity. Proteins. 2013;81:406–414. 10.1002/prot.24199  [DOI](https://doi.org/10.1002/prot.24199) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23042597/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proteins&title=Molecular%20dynamics%20simulation%20of%20PNPLA3%20I148M%20polymorphism%20reveals%20reduced%20substrate%20access%20to%20the%20catalytic%20cavity&author=YN%20Xin&author=Y%20Zhao&author=ZH%20Lin&author=X%20Jiang&author=SY%20Xuan&volume=81&publication_year=2013&pages=406-414&pmid=23042597&doi=10.1002/prot.24199&)

46. Liu M, Wang L, Sun X, Zhao X. Investigating the impact of Asp181 point mutations on interactions between PTP1B and phosphotyrosine substrate. Sci Rep. 2014;28:5095.  [DOI](https://doi.org/10.1038/srep05095) | [PMC free article](/articles/PMC4035576/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24865376/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Sci%20Rep&title=Investigating%20the%20impact%20of%20Asp181%20point%20mutations%20on%20interactions%20between%20PTP1B%20and%20phosphotyrosine%20substrate&author=M%20Liu&author=L%20Wang&author=X%20Sun&author=X%20Zhao&volume=28&publication_year=2014&pages=5095&pmid=24865376&doi=10.1038/srep05095&)

47. Karslake C, Piotto ME, Pak YK, Weiner H, Gorenstein DG. 2D NMR and structural model for a mitochondrial signal peptide bound to a micelle. Biochemistry. 1990;29:9872–9878.  [DOI](https://doi.org/10.1021/bi00494a017) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/2271626/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochemistry&title=2D%20NMR%20and%20structural%20model%20for%20a%20mitochondrial%20signal%20peptide%20bound%20to%20a%20micelle&author=C%20Karslake&author=ME%20Piotto&author=YK%20Pak&author=H%20Weiner&author=DG%20Gorenstein&volume=29&publication_year=1990&pages=9872-9878&pmid=2271626&doi=10.1021/bi00494a017&)

48. Ni L, Zhou J, Hurley TD, Weiner H. Human liver mitochondrial aldehyde dehydrogenase: three-dimensional structure and the restoration of solubility and activity of chimeric forms. Protein Sci. 1999;8:2784–2790. 10.1110/ps.8.12.2784  [DOI](https://doi.org/10.1110/ps.8.12.2784) | [PMC free article](/articles/PMC2144226/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10631996/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Protein%20Sci&title=Human%20liver%20mitochondrial%20aldehyde%20dehydrogenase:%20three-dimensional%20structure%20and%20the%20restoration%20of%20solubility%20and%20activity%20of%20chimeric%20forms&author=L%20Ni&author=J%20Zhou&author=TD%20Hurley&author=H%20Weiner&volume=8&publication_year=1999&pages=2784-2790&pmid=10631996&doi=10.1110/ps.8.12.2784&)

49. Perez-Miller SJ, Hurley TD. Coenzyme isomerization is integral to catalysis in aldehyde dehydrogenase. Biochemistry. 2003;42:7100–7109. 10.1021/bi034182w  [DOI](https://doi.org/10.1021/bi034182w) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12795606/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochemistry&title=Coenzyme%20isomerization%20is%20integral%20to%20catalysis%20in%20aldehyde%20dehydrogenase&author=SJ%20Perez-Miller&author=TD%20Hurley&volume=42&publication_year=2003&pages=7100-7109&pmid=12795606&doi=10.1021/bi034182w&)

50. Smith LJ, Mark AE, Dobson CM, van Gunsteren WF. Comparison of MD simulations and NMR experiments for hen lysozyme. Analysis of local fluctuations, cooperative motions, and global changes. Biochemistry. 1995;34:10918–10931.  [DOI](https://doi.org/10.1021/bi00034a026) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7662673/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochemistry&title=Comparison%20of%20MD%20simulations%20and%20NMR%20experiments%20for%20hen%20lysozyme.%20Analysis%20of%20local%20fluctuations,%20cooperative%20motions,%20and%20global%20changes&author=LJ%20Smith&author=AE%20Mark&author=CM%20Dobson&author=WF%20van%20Gunsteren&volume=34&publication_year=1995&pages=10918-10931&pmid=7662673&doi=10.1021/bi00034a026&)

51. Brocke KS, Neu-Yilik G, Gehring NH, Hentze MW, Kulozik AE. The human intronless melanocortin 4-receptor gene is NMD insensitive. Hum Mol Genet. 2002;11:331–335.  [DOI](https://doi.org/10.1093/hmg/11.3.331) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11823452/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mol%20Genet&title=The%20human%20intronless%20melanocortin%204-receptor%20gene%20is%20NMD%20insensitive&author=KS%20Brocke&author=G%20Neu-Yilik&author=NH%20Gehring&author=MW%20Hentze&author=AE%20Kulozik&volume=11&publication_year=2002&pages=331-335&pmid=11823452&doi=10.1093/hmg/11.3.331&)

52. Skol AD, Scott LJ, Abecasis GR, Boehnke M. Joint analysis is more efficient than replication-based analysis for two-stage genome-wide association studies. Nat Genet. 2006;38:209–213. 10.1038/ng1706  [DOI](https://doi.org/10.1038/ng1706) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16415888/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Joint%20analysis%20is%20more%20efficient%20than%20replication-based%20analysis%20for%20two-stage%20genome-wide%20association%20studies&author=AD%20Skol&author=LJ%20Scott&author=GR%20Abecasis&author=M%20Boehnke&volume=38&publication_year=2006&pages=209-213&pmid=16415888&doi=10.1038/ng1706&)

53. Sham PC, Purcell SM. Statistical power and significance testing in large-scale genetic studies. Nat Rev Genet. 2014;15:335–346. 10.1038/nrg3706  [DOI](https://doi.org/10.1038/nrg3706) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24739678/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Genet&title=Statistical%20power%20and%20significance%20testing%20in%20large-scale%20genetic%20studies&author=PC%20Sham&author=SM%20Purcell&volume=15&publication_year=2014&pages=335-346&pmid=24739678&doi=10.1038/nrg3706&)

54. Wu YY, Delgado R, Costello R, Sunderland T, Dukoff R, Csako G. Quantitative assessment of apolipoprotein E genotypes by image analysis of PCR-RFLP fragments. Clin Chim Acta. 2000;293:213–221.  [DOI](https://doi.org/10.1016/s0009-8981(99)00237-5) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10699435/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Chim%20Acta&title=Quantitative%20assessment%20of%20apolipoprotein%20E%20genotypes%20by%20image%20analysis%20of%20PCR-RFLP%20fragments&author=YY%20Wu&author=R%20Delgado&author=R%20Costello&author=T%20Sunderland&author=R%20Dukoff&volume=293&publication_year=2000&pages=213-221&pmid=10699435&doi=10.1016/s0009-8981(99)00237-5&)

55. Wang LX, Wen S, Wang CC, Zhou B, Li H. Molecular adaption of alcohol metabolism to agriculture in East Asia. Quat Int. 2016;426:187–194.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Quat%20Int&title=Molecular%20adaption%20of%20alcohol%20metabolism%20to%20agriculture%20in%20East%20Asia&author=LX%20Wang&author=S%20Wen&author=CC%20Wang&author=B%20Zhou&author=H%20Li&volume=426&publication_year=2016&pages=187-194&)
