# Influence of HLA alleles and KIR types on Heparin-Induced Thrombocytopenia

## Metadata
**Authors:** Jason H Karnes, Christian M Shaffer, Robert Cronin, Lisa Bastarache, Silvana Gaudieri, Ian James, Rebecca Pavlos, Heidi Steiner, Jonathan D Mosley, Simon Mallal, Joshua C Denny, Elizabeth J Phillips, Dan M Roden
**Journal:** Pharmacotherapy
**Date:** 2017 Sep 4
**DOI:** [10.1002/phar.1983](https://doi.org/10.1002/phar.1983)
**PMID:** 28688202
**PMCID:** PMC5600645
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5600645/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC5600645/pdf/nihms865033.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC5600645/pdf/nihms865033.pdf)

## Abstract

**Objectives:** 
Variation in human leukocyte antigen (HLA) genes is now used to prevent immune-mediated adverse drug reactions. Combinations of HLA alleles and killer cell immunoglobulin-like receptors (KIR) are associated with multiple autoimmune diseases and infections. Heparin-induced thrombocytopenia (HIT) is an unpredictable, life-threatening, immune-mediated reaction to heparin. The objective of this study is to evaluate the association of HLA alleles and KIR types, alone or in the presence of different HLA ligands, with HIT.

**Methods:** 
We identified HIT cases and heparin-exposed controls in BioVU, an electronic health record coupled to a DNA biobank. We performed HLA sequencing and imputed KIR types from Illumina® OMNI-Quad data. We determined odds ratios (ORs) for HLA alleles and KIR types and HLA*KIR interactions using conditional logistic regressions in the overall population and by race/ethnicity. Analysis was restricted to KIR types and HLA alleles with a frequency greater than 0.01. P values for HLA and KIR association were corrected using a false discovery rate (FDR) q<0.05 and HLA*KIR interactions were considered significant at p<0.05.

**Results:** 
We identified 65 HIT cases and 350 matched controls. We observed no statistical differences in baseline characteristics between cases and controls. The HLA-DRB3*01:01 allele was significantly associated with HIT in the overall population (OR 2.81[1.57–5.02], p=2.1x10−4, q=0.02) and in individuals with European ancestry, independent of other alleles. No KIR types were associated with HIT, although we observed a significant interaction between KIR2DS5 and the HLA-C1 KIR binding group (p=0.03).

**Conclusions:** 
We identify the HLA-DRB3*01:01 allele as a potential risk factor for HIT. This class II HLA gene and allele represent biologically plausible candidates for influencing HIT pathogenesis. We found limited evidence of the role of KIR types in HIT pathogenesis. Replication and further study of the HLA-DRB3*01:01 association is necessary.

### Objectives

Variation in human leukocyte antigen (HLA) genes is now used to prevent immune-mediated adverse drug reactions. Combinations of HLA alleles and killer cell immunoglobulin-like receptors (KIR) are associated with multiple autoimmune diseases and infections. Heparin-induced thrombocytopenia (HIT) is an unpredictable, life-threatening, immune-mediated reaction to heparin. The objective of this study is to evaluate the association of HLA alleles and KIR types, alone or in the presence of different HLA ligands, with HIT.

### Methods

We identified HIT cases and heparin-exposed controls in BioVU, an electronic health record coupled to a DNA biobank. We performed HLA sequencing and imputed KIR types from Illumina® OMNI-Quad data. We determined odds ratios (ORs) for HLA alleles and KIR types and HLA*KIR interactions using conditional logistic regressions in the overall population and by race/ethnicity. Analysis was restricted to KIR types and HLA alleles with a frequency greater than 0.01. P values for HLA and KIR association were corrected using a false discovery rate (FDR) q<0.05 and HLA*KIR interactions were considered significant at p<0.05.

### Results

We identified 65 HIT cases and 350 matched controls. We observed no statistical differences in baseline characteristics between cases and controls. The *HLA-DRB3*01:01* allele was significantly associated with HIT in the overall population (OR 2.81[1.57–5.02], p=2.1x10^−4^, q=0.02) and in individuals with European ancestry, independent of other alleles. No KIR types were associated with HIT, although we observed a significant interaction between KIR2DS5 and the HLA-C1 KIR binding group (p=0.03).

### Conclusions

We identify the *HLA-DRB3*01:01* allele as a potential risk factor for HIT. This class II HLA gene and allele represent biologically plausible candidates for influencing HIT pathogenesis. We found limited evidence of the role of KIR types in HIT pathogenesis. Replication and further study of the *HLA-DRB3*01:01* association is necessary.

## Introduction

Heparin-induced thrombocytopenia (HIT) is an antibody-mediated condition of platelet activation in patients receiving unfractionated heparin (UFH) and low molecular weight heparin (LMWH).^1^ HIT develops in 0.5–5% of patients treated with heparin anticoagulants, has a greater than 30% mortality rate, and results in catastrophic thromboembolic complications, including life- and limb-threatening thrombosis.^2–5^ Further complicating the use of heparin anticoagulants, prevention of HIT-related thrombosis is currently possible only after manifestations of HIT are evident and the disease process has already begun.^6, 7^ Clinical algorithms and laboratory tests are inadequate for evaluation of HIT risk prior to heparin administration.^8, 9^ The inability to predict HIT thus represents a liability with heparin administration and *a priori* identification of patients with a high HIT risk could make prevention of HIT possible.

The HLA genes in the major histocompatibility complex (MHC) have emerged as a leading source of biomarkers of immune-mediated adverse drug reactions (ADRs), which can be translated into clinical care.^10, 11^ Alleles from these HLA genes are powerful pharmacogenomic biomarkers for the prevention of previously unpredictable, immune-mediated diseases.^10, 12–16^ HLA alleles are now genotyped to guide drug treatment and prevent immune-mediated ADRs, moving personalized medicine toward reality.^12^ Previous studies have implicated *HLA-DR* variation with the formation of platelet factor 4 (PF4)/heparin antibodies and HIT^17, 18^, suggesting that strong genetic influences on HIT risk might still be identified with other HLA or immune-related genes in the MHC.

In addition to HLA alleles, killer immunoglobulin-like receptor (KIR) types have been observed to associate with disease alone or in interaction with HLA allele ligands.^19–21^ KIRs act as receptors on Natural Killer (NK) cells, play a major role in the rejection of viral-infected or tumor-transformed cells, and are able to interact with particular class I HLA alleles. The objective of this study was to identify HLA alleles, KIR types, and HLA*KIR interactions associated with HIT. We conducted this study using HIT cases and heparin-exposed controls identified in an electronic health record (EHR) linked to a DNA repository.

## Materials and Methods

### Study Population

The study population was identified in BioVU, the Vanderbilt DNA databank that links DNA extracted from discarded blood samples to de-identified EHRs^22^. This study was approved by the Institutional Review Board at Vanderbilt University as described previously^22^. We developed an algorithm to identify individuals with a diagnosis of HIT in the EHR as previously described.^18^ The algorithm used ICD9 diagnostic codes, lab results, and natural language processing^23, 24^ to analyze narrative text in order to identify patients treated with UFH or LMWH who tested positive for PF4/heparin antibodies (optical density [OD] level >0.7 absorbance units [AU]). UFH or LMWH-treated controls were also identified in the EHR using a control algorithm that employed similar techniques. Controls were selected in a 1:5 case:control ratio prior to genomic quality control. HIT cases were confirmed by chart review by at least two physician or pharmacist reviewers. HIT cases were scored with the 4Ts scoring system, which estimates the likelihood of HIT based on timing, magnitude, and potential causes of thrombocytopenia and presence of thrombosis and with a maximum 4Ts score of eight^6, 25^. Controls were matched to cases based on age, STRUCTURE-defined race (as described below), and type of heparin exposure (UFH versus LMWH).

### HLA Sequencing

High resolution, four-digit HLA sequencing was performed for *HLA-A*, *HLA-B*, *HLA-C*, *HLA-DR*, *HLA-DP*, and *HLA-DQ* at the Institute for Immunology and Infectious Diseases (IIID) at Murdoch University in Perth, Australia. The IIID is accredited by the American Society for Histocompatibility and Immunogenetics (ASHI) and the National Association of Testing Authorities (NATA) and has extensive experience with sequencing the HLA genes including in large, multicenter trials using HLA sequence data.^12, 26^ Specific HLA loci were PCR amplified using sample specific MID-tagged primers that amplify polymorphic exons from Class I (A, B, C Exons 2 and 3) and Class II (DQ, Exons 2 and 3; DRB and DPB1, Exon 1) HLA genes. MID tagged primers have been optimized to minimize allele dropouts and primer bias. Amplified DNA products from unique MID tagged products (up to 48 MIDs) were pooled in equimolar ratios and subjected to library preparation, quantitation and emulsion PCR suitable for entry into the 454 FLX sequencing pipeline. Clonally enriched beads were sequenced using 454 Titanium chemistry on a 454 FLX+ sequencer. Sequences were separated by MID tags and alleles called using an in house accredited HLA allele caller software pipeline that minimizes the influence of systematic sequencing errors in 454 data. Alleles were called using the latest international ImMunoGeneTics information system (IMGT) HLA allele database as the allele reference library. Sample to report integrity were tracked and checked using proprietary and accredited Laboratory Information and Management System (LIMS) and HLA analysis reporting software that performs comprehensive allele balance and contamination checks on the final dataset. All samples that were successfully typed were included in the study population.

### KIR Imputation

Samples were genotyped using the Illumina® HumanOmni1-QUAD and HumanOmni5-QUAD BeadChip genome-wide platforms as previously described.^18^ Imputation of KIR types and KIR A versus B haplotype was performed on genotype data imputed from single nucleotide polymorphisms (SNPs) present on the intersection of these platforms (n=730,803 SNPs). Before imputation, data were cleaned using the quality control pipeline developed by the eMERGE Genomics Working Group^27^. KIR imputation was then performed for samples using KIR*IMP ([http://imp.mcri.edu.au/hla/](http://imp.mcri.edu.au/hla/)).^28^ KIR binding groups (HLA-Bw6 or Bw4 and HLA-C1 or C2) were determined for each sample corresponding to sequenced *HLA-B* and *HLA-C* alleles, respectively. KIR*IMP uses a random-forest model to impute KIR data based on reference panels with sequenced KIR loci and genotyped SNP data.^28^ Using this model, KIR alleles and haplotypes can be generated from a population with unknown KIR types but known SNP genotypes. KIR*IMP is validated and highly accurate, allowing for the study of KIR in large cohorts with SNP-level data and enabling detailed investigation of the role of KIR in human disease.

### Definition and Adjustment for Race/Ethnicity

Samples were classified as being of European or African descent using ancestry informative markers (AIMs) input into STRUCTURE using Hapmap reference populations^29^. Principal components (PC) analysis was also performed on AIMs using the EIGNESTRAT method implemented in PLINK^30, 31^. To minimize confounding by population stratification, association analyses were adjusted for the first two PCs and performed in both the overall population and by STRUCTURE-defined race.

### Statistical Analysis

Differences in characteristics between cases and controls were determined using chi square and t-tests. The association of HIT with sequenced HLA alleles, imputed KIR types and KIR haplotype was tested, followed by the interaction of KIR types with HLA-Bw6/Bw4 and HLA-C1/C2. Tests of association with HIT were performed using logistic regression in a dominant model adjusted for age, gender, type of heparin anticoagulant (UFH versus LMWH), and PCs one and two for all SNPs in a total of 462 individuals by STRUCTURE-defined race. Multi-variable logistic regression models were used to estimate odds ratios (ORs) and 95% confidence intervals for HIT, assess HLA*KIR interactions, and calculate synergy indices (SIs) for an estimate of multiplicative HLA*KIR interactions.^32^ In the absence of an interaction effect, the SI equals 1. The same statistical models were employed in an HLA*KIR interaction analysis which included pairwise testing of KIR types by HLA-Bw6/Bw4 subgroup and by HLA-C1/C2 subgroup in multivariate logistic regressions. HLA and KIR association analyses were performed in non-European race/ethnic groups, but we had very little power to detect such associations. In a supplementary analysis, HLA and KIR alleles were also tested for association with 4Ts score among HIT cases using similar multi-variable liner regressions.

HLA alleles and KIR types were tested only if their allele frequency was greater than 0.01. HLA alleles were tested for deviation from Hardy Weinberg Equilibrium (HWE) and alleles with HWE p<0.01 were removed from further analyses. In order to determine linkage of class II allelic associations with *HLA-DRB3*01:01,* large haplotype reconstruction was used to classify carriers of *HLA-DRB1*03:01* and *HLA-DRB1*13:01* as being with or without *HLA-DRB3*01:01*. Logistic regression of the phenotype was then performed with cases split into those carrying *HLA-DRB3*01:01* and those with *HLA-DRB1*03:01* or *HLA-DRB1*13:01* without *HLA-DRB3*01:01*. KIR and HLA associations with a false discovery rate (FDR) adjusted q value<0.05 were considered significant. Significance for interaction analyses was considered at p<0.05. All statistical analyses were performed using PLINK^31^ and SAS 9.3 (SAS, Cary, NC).

## Results

A total of 65 HIT cases were identified from BioVU and matched to 350 controls. ([Table 1](#T1)) The majority of cases were treated with UFH rather than LMWH and the mean 4Ts score for HIT cases was 5.1 (standard deviation [SD] 0.9). No significant differences were observed in baseline characteristics between cases and controls. A total of 99 HLA alleles and 13 KIR types with frequencies greater than 0.01 were identified. No HLA alleles or KIR types significantly departed from HWE (p<0.01). Details of allele frequencies, genotype distributions, and HWE tests are provided in the [Supplemental Materials](#SD1).

### Table 1.

| Characteristic1 | Cases (n=65) | Controls (n=350) | p value2 |
| --- | --- | --- | --- |
| Age (years), mean (SD) | 58.3 (16.0) | 57.1 (16.8) | 0.59 |
| Gender (female) | 33 (51) | 190 (54) | 0.60 |
| Weight (kg), mean (SD) | 80.4 (29.4) | 80.1 (31.9) | 0.96 |
| BMI (kg/m2), mean (SD) | 23.6 (20.0) | 26.9 (25.2) | 0.18 |
| Heparin agent |   |   | 0.92 |
| Heparin | 59 (91) | 319 (91) |   |
| Enoxaparin | 6 (9) | 31 (9) |   |
| Race/ethnicity3 |   |   | 0.97 |
| White | 55 (85) | 298 (85) |   |
| Black | 1 (2) | 6 (2) |   |
| Asian | 0 (0) | 1 (0) |   |
| Other | 9 (14) | 45 (13) |   |
| Percent ancestry3 |  |  |  |
| European (%) | 88 (26) | 89 (27) | 0.88 |
| African (%) | 11 (26) | 10 (25) | 0.68 |
| Native American (%) | 1 (2) | 2 (7) | 0.32 |
| 4Ts Score, mean (SD)4 | 5.1 (0.9) | - | - |

Table 1 Caption: Baseline characteristics in cases versus matched controls.

In the overall population, the *HLA-DRB3*01:01* allele was significantly associated with HIT (OR 2.81 [1.57–5.02], p=2.10x10^−4^, q=0.02). ([Figure 1](#F1)) When the analysis was performed by race, the *HLA-DRB3*01:01* allele was significantly associated with HIT in individuals with STRUCTURE-defined European ancestry (OR 3.00 [1.59–5.62], p=2.67x10^−4^, q=0.015). We found a nominal association between the number of *HLA-DRB3*0101* copies and 4Ts score among HIT cases (β=0.48, p=0.02). Seven other class I and II HLA alleles were nominally associated (p<0.05 but q>0.5) with HIT in the overall population and five other HLA alleles were nominally associated with HIT in individuals with primarily European ancestry. No HLA associations were observed in any non-European race/ethnic group, but we had very little power to detect such associations. Further details of the HLA association analysis are presented in the [Supplemental Materials](#SD1).

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4522/5600645/792a2bb42f3c/nihms865033f1.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5600645_nihms865033f1.jpg)

Due to linkage of *HLA-DRB3*01:01* with *HLA-DRB1*03:01* and *HLA-DRB1*13:01,* large haplotype reconstruction was used to classify carriers of *HLA-DRB1*03:01* and *HLA-DRB1*13:01* as being with or without *HLA-DRB3*01:01*. Logistic regression was performed with the cases split into those carrying *HLA-DRB3*01:01* and those with *HLA-DRB1*03:01* or *HLA-DRB1*13:01* without *HLA-DRB3*01:01*. In this analysis, there was no evidence of an association between HIT and *HLA-DRB1*03:01* or *HLA-DRB1*13:01*. In addition, there no significant interactions of *HLA-DRB3*01:01* with *HLA-DRB1*03:01* or *HLA-DRB1*13:01* on HIT, suggesting that the association for *HLA-DRB3*01:01* with HIT is not linked to carriage of *HLA-DRB1*03:01* or *HLA-DRB1*13:01*.

In the analysis of KIR types, no significant associations were observed for imputed KIR types overall or in any race/ethnic group. Neither were significant associations observed in the analysis of KIR B versus A haplotype (OR 1.20 [0.68–2.13], p=0.53), analysis of HLA-C1 versus C2 KIR binding groups (OR 0.81 [0.42–1.57], p=0.53), nor HLA-Bw4 versus Bw6 KIR binding groups (OR 0.92 [0.49–1.71], p=0.78). No KIR types were associated with 4Ts score among HIT cases. No KIR associations were observed in any non-European race/ethnic group. Further details of the KIR and KIR binding group association analysis are presented in the [Supplemental Materials](#SD1).

In the analysis of HLA*KIR interactions, a significant interaction was observed between KIR2DS5 and the presence of the HLA-C1 KIR binding group in individuals with STRUCTURE-defined European ancestry (synergy index 1.61 [1.04–2.48], p=0.03). The interaction indicated that KIR2DS5 was associated with HIT in subjects with the HLA-C1 KIR binding group (OR 2.10 [1.07–4.12], p=0.03). A trend was observed towards a decreased HIT risk with KIR2DS5 in subjects without the HLA-C1 KIR binding group (OR 0.23 [0.04–1.34], p=0.10). The presence of KIR2DS5 was not associated with HIT when the HLA-C1 binding group was not considered (OR 1.39 [0.76–2.55], p=0.28).

## Discussion

We present an original immunogenetic association study for HIT using an EHR-coupled DNA biobank. Our results provide further evidence of contribution of variation in the *HLA-DR* locus to HIT pathophysiology and implicate a specific HLA allele, *HLA-DRB3*01:01*, as a risk factor for HIT. Our analyses suggests that carriage of *HLA-DRB3*01:01* is associated with HIT and this association is not linked to carriage of *HLA-DRB1*03:01* or *HLA-DRB1*13:01*. Our observed association is consistent with previous reports of association between HIT and class II alleles and our observations provide further insights into the immunopathology of HIT. We found limited evidence that KIR types are associated with HIT, including the observation of an interaction between KIR2DS5 and HLA-C1 KIR ligand groups. To our knowledge, this is the first study to examine the role of KIR in HIT and the first study to implicate a specific HLA allele in HIT risk.

Despite decades of research into the immunopathology of HIT, there is a fundamental knowledge gap regarding the cause of HIT.^33, 34^ The causative immune cell types and the impetus for antibody formation has yet to be identified. The clinical significance of “non-pathogenic” PF4/heparin antibody remains unclear, and the molecular basis that distinguishes them from “pathogenic” PF4/heparin antibody is also unknown.^35^ The role of T cells in HIT and their involvement in PF4/heparin antibody production continues to be controversial and complicated by the fact that HIT displays characteristics of both an innate and adaptive immune reaction. Emerging evidence also suggests a key role of marginal zone B cells in T cell-independent antibody production as well as bacterial infection as a HIT sensitization event.^34, 36^

Our observation of an association between HIT and *HLA-DRB3*01:01* suggests the involvement of T cell-dependent PF4/heparin antibody formation. *HLA-DR* in the MHC region provides a particularly attractive candidate gene for HIT, since class II HLA molecules are directly involved in the adaptive immune response resulting in antibody formation. Class II HLA alleles such as *HLA-DR* are central to antibody formation and HIT is caused by the formation of heparin/PF4 antibodies. *HLA-DRB3*01:01* is associated with fetomaternal alloimmune thrombocytopenia and this allele likely mediates the presentation of HPA-1a peptides and subsequent production of anti-HPA-1a alloantibodies.^37, 38^ PF4 has been shown to promote monocyte survival and induce monocyte differentiation into macrophages that lack the HLA-DR antigen.^39^ We have also previously observed in a genome-wide association study (GWAS) that SNPs in *HLA-DR* are associated with HIT.^18^ However, this GWAS was conducted in the same population as the present study and should not be considered an independent replication of our results. The present study investigates sequenced HLA alleles rather than SNPs, allowing for the identification of obligate HLA alleles or combinations of alleles necessary for HIT. The present study also examines the association of KIR types and HLA*KIR interactions which is not possible using SNP-level GWAS data. Finally, the HLA-DR3 serotype (*HLA-DRB1*03:01*) has been associated with thrombotic thrombocytopenic purpura and *HLA-DR3* expression has been associated with the formation of heparin/PF4 antibodies.^17, 40^

Our observed lack of association between HIT and KIR types is also consistent with a T cell-dependent pathogenic model of HIT. KIR ligands are found exclusively among HLA class I alleles and the association of an HLA class II allele suggests the absence of a role of KIR types and thus NK cells in HIT. Our analysis investigated all available combinations of KIR ligands and HLA-C1/C2 and Bw4/Bw6 binding groups and thus was not based on known predicted receptor/ligand interactions. Our observed interaction between KIR2DS5 and the HLA-C1 binding group should be interpreted with caution as neither the HLA-C1 binding group nor any class I HLA allele was associated with HIT in our study. Neither HLA-C1 nor -C2 has been shown to bind KIR2DS5, which does not have a well-known HLA ligand group. KIR2DS5 was also not associated with HIT, suggesting a limited or non-existent role of KIR types in the immunogenetics of HIT.

There are several limitations in our study. The number of HIT cases in our study was small, which may have limited our power to detect an HLA or KIR association with HIT. We were able to investigate associations only in individuals with primarily European ancestry and our results cannot be generalized to other race/ethnic groups, given the small sample sizes in our study population. However, the presence of the significantly associated *HLA-DRB3*01:01* allele indicates that this number was sufficient to identify HLA associations. In this study, we investigated only common HLA alleles and KIR types and any rare variation with modest effects on HIT pathogenesis would not be identified using our approach. Our HIT cases were also retrospectively identified in an EHR, which precluded the acquisition of confirmatory functional assay testing for HIT. Although our HIT cases were independently reviewed by multiple investigators, we cannot exclude the possibility of classification bias in our observational HIT case and control cohort. Although some misclassification is likely among HIT cases, the use of a high OD cutoff (0.7 AU) for PF4/heparin antibody testing and the prevalence in known HIT risk factors among cases, such as female sex, surgery, durations of exposure over 6 days, high 4Ts scores, and increased age suggest accurate classification among our cohort. The results of our KIR analysis rely on the accuracy of imputation from SNP-level data. Although the KIR genomic region was well covered on our GWAS platforms, some misclassification may be present in the imputation of KIR types. Finally, our results have not been replicated or prospectively validated in an independent population and further research is required to confirm the *HLA-DRB3*01:01* association with HIT.

In summary, we describe an original immunogenetic association study for HIT using an EHR-coupled DNA biobank. We identify the class II allele *HLA-DRB3*01:01* as a potential risk factor for HIT, providing additional evidence for the immune-pathogenesis of this unpredictable, potentially catastrophic consequence of heparin treatment. Our results require replication of the effect of *HLA-DRB3*01:01* in independent populations to confirm our association. We found limited evidence for the involvement of KIR types in HIT pathophysiology. As the inability to predict HIT remains a critical problem with heparin administration, immunogenetic variation might yield biomarkers that enable HIT prediction and pre-emptive genotyping strategies.

## Supplementary Material

## Acknowledgments

The dataset used in the analyses described were obtained from Vanderbilt University Medical Centers BioVU, which is supported by institutional funding and by the Vanderbilt CTSA grant ULTR000445 from NCATS/NIH. Genome-wide genotyping was funded by NIH grants RC2GM092618 from NIGMS/OD, U01HG004603 from NHGRI/NIGMS, and U19HL065962 from NHGRI/NIGMS. BioVU is supported by institutional funding and by the Vanderbilt CTSA grant UL1TR000445 from NCATS/NIH.

JHK has been supported by the VUMC Clinical Pharmacology Training grant (T32 GM07569), the American Heart Association (16SDG29090005 and 15POST22660017), and an ACCP Research Institute Futures Grants Award from the American College of Clinical Pharmacy. Other support was also provided by P50 GM GM115305. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Warkentin TE, Levine MN, Hirsh J, et al. Heparin-induced thrombocytopenia in patients treated with low-molecular-weight heparin or unfractionated heparin. N Engl J Med. 1995;332:1330–5. doi: 10.1056/NEJM199505183322003.  [DOI](https://doi.org/10.1056/NEJM199505183322003) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7715641/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Heparin-induced%20thrombocytopenia%20in%20patients%20treated%20with%20low-molecular-weight%20heparin%20or%20unfractionated%20heparin&author=TE%20Warkentin&author=MN%20Levine&author=J%20Hirsh&volume=332&publication_year=1995&pages=1330-5&pmid=7715641&doi=10.1056/NEJM199505183322003&)

2. Girolami B, Prandoni P, Stefani PM, et al. The incidence of heparin-induced thrombocytopenia in hospitalized medical patients treated with subcutaneous unfractionated heparin: a prospective cohort study. Blood. 2003;101:2955–9. doi: 10.1182/blood-2002-07-2201.  [DOI](https://doi.org/10.1182/blood-2002-07-2201) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12480713/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=The%20incidence%20of%20heparin-induced%20thrombocytopenia%20in%20hospitalized%20medical%20patients%20treated%20with%20subcutaneous%20unfractionated%20heparin:%20a%20prospective%20cohort%20study&author=B%20Girolami&author=P%20Prandoni&author=PM%20Stefani&volume=101&publication_year=2003&pages=2955-9&pmid=12480713&doi=10.1182/blood-2002-07-2201&)

3. Greinacher A, Farner B, Kroll H, Kohlmann T, Warkentin TE, Eichler P. Clinical features of heparin-induced thrombocytopenia including risk factors for thrombosis. A retrospective analysis of 408 patients. Thromb Haemost. 2005;94:132–5. doi: 10.1160/TH04-12-0825.  [DOI](https://doi.org/10.1160/TH04-12-0825) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16113796/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Thromb%20Haemost&title=Clinical%20features%20of%20heparin-induced%20thrombocytopenia%20including%20risk%20factors%20for%20thrombosis.%20A%20retrospective%20analysis%20of%20408%20patients&author=A%20Greinacher&author=B%20Farner&author=H%20Kroll&author=T%20Kohlmann&author=TE%20Warkentin&volume=94&publication_year=2005&pages=132-5&pmid=16113796&doi=10.1160/TH04-12-0825&)

4. Franchini M. Heparin-induced thrombocytopenia: an update. Thromb J. 2005;3:14. doi: 10.1186/1477-9560-3-14.  [DOI](https://doi.org/10.1186/1477-9560-3-14) | [PMC free article](/articles/PMC1262784/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16202170/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Thromb%20J&title=Heparin-induced%20thrombocytopenia:%20an%20update&author=M%20Franchini&volume=3&publication_year=2005&pages=14&pmid=16202170&doi=10.1186/1477-9560-3-14&)

5. Martel N, Lee J, Wells PS. Risk for heparin-induced thrombocytopenia with unfractionated and low-molecular-weight heparin thromboprophylaxis: a meta-analysis. Blood. 2005;106:2710–5. doi: 10.1182/blood-2005-04-1546.  [DOI](https://doi.org/10.1182/blood-2005-04-1546) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15985543/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Risk%20for%20heparin-induced%20thrombocytopenia%20with%20unfractionated%20and%20low-molecular-weight%20heparin%20thromboprophylaxis:%20a%20meta-analysis&author=N%20Martel&author=J%20Lee&author=PS%20Wells&volume=106&publication_year=2005&pages=2710-5&pmid=15985543&doi=10.1182/blood-2005-04-1546&)

6. Linkins LA, Dans AL, Moores LK, et al. Treatment and prevention of heparin-induced thrombocytopenia: Antithrombotic Therapy and Prevention of Thrombosis, 9th ed: American College of Chest Physicians Evidence-Based Clinical Practice Guidelines. Chest. 2012;141:e495S–530S. doi: 10.1378/chest.11-2303.  [DOI](https://doi.org/10.1378/chest.11-2303) | [PMC free article](/articles/PMC3278058/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22315270/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Chest&title=Treatment%20and%20prevention%20of%20heparin-induced%20thrombocytopenia:%20Antithrombotic%20Therapy%20and%20Prevention%20of%20Thrombosis,%209th%20ed:%20American%20College%20of%20Chest%20Physicians%20Evidence-Based%20Clinical%20Practice%20Guidelines&author=LA%20Linkins&author=AL%20Dans&author=LK%20Moores&volume=141&publication_year=2012&pages=e495S-530S&pmid=22315270&doi=10.1378/chest.11-2303&)

7. Watson H, Davidson S, Keeling D. Guidelines on the diagnosis and management of heparin-induced thrombocytopenia: second edition. Br J Haematol. 2012;159:528–40. doi: 10.1111/bjh.12059.  [DOI](https://doi.org/10.1111/bjh.12059) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23043677/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Haematol&title=Guidelines%20on%20the%20diagnosis%20and%20management%20of%20heparin-induced%20thrombocytopenia:%20second%20edition&author=H%20Watson&author=S%20Davidson&author=D%20Keeling&volume=159&publication_year=2012&pages=528-40&pmid=23043677&doi=10.1111/bjh.12059&)

8. Crowther MA, Cook DJ, Albert M, et al. The 4Ts scoring system for heparin-induced thrombocytopenia in medical-surgical intensive care unit patients. J Crit Care. 2010;25:287–93. doi: 10.1016/j.jcrc.2009.12.006.  [DOI](https://doi.org/10.1016/j.jcrc.2009.12.006) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20149589/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Crit%20Care&title=The%204Ts%20scoring%20system%20for%20heparin-induced%20thrombocytopenia%20in%20medical-surgical%20intensive%20care%20unit%20patients&author=MA%20Crowther&author=DJ%20Cook&author=M%20Albert&volume=25&publication_year=2010&pages=287-93&pmid=20149589&doi=10.1016/j.jcrc.2009.12.006&)

9. Warkentin TE, Heddle NM. Laboratory diagnosis of immune heparin-induced thrombocytopenia. Curr Hematol Rep. 2003;2:148–57.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/12901146/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Hematol%20Rep&title=Laboratory%20diagnosis%20of%20immune%20heparin-induced%20thrombocytopenia&author=TE%20Warkentin&author=NM%20Heddle&volume=2&publication_year=2003&pages=148-57&pmid=12901146&)

10. Daly AK, Donaldson PT, Bhatnagar P, et al. HLA-B*5701 genotype is a major determinant of drug-induced liver injury due to flucloxacillin. Nat Genet. 2009;41:816–9. doi: 10.1038/ng.379.  [DOI](https://doi.org/10.1038/ng.379) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19483685/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=HLA-B*5701%20genotype%20is%20a%20major%20determinant%20of%20drug-induced%20liver%20injury%20due%20to%20flucloxacillin&author=AK%20Daly&author=PT%20Donaldson&author=P%20Bhatnagar&volume=41&publication_year=2009&pages=816-9&pmid=19483685&doi=10.1038/ng.379&)

11. Kindmark A, Jawaid A, Harbron CG, et al. Genome-wide pharmacogenetic investigation of a hepatic adverse event without clinical signs of immunopathology suggests an underlying immune pathogenesis. Pharmacogenomics J. 2008;8:186–95. doi: 10.1038/sj.tpj.6500458.  [DOI](https://doi.org/10.1038/sj.tpj.6500458) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17505501/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=Genome-wide%20pharmacogenetic%20investigation%20of%20a%20hepatic%20adverse%20event%20without%20clinical%20signs%20of%20immunopathology%20suggests%20an%20underlying%20immune%20pathogenesis&author=A%20Kindmark&author=A%20Jawaid&author=CG%20Harbron&volume=8&publication_year=2008&pages=186-95&pmid=17505501&doi=10.1038/sj.tpj.6500458&)

12. Mallal S, Phillips E, Carosi G, et al. HLA-B*5701 screening for hypersensitivity to abacavir. N Engl J Med. 2008;358:568–79. doi: 10.1056/NEJMoa0706135.  [DOI](https://doi.org/10.1056/NEJMoa0706135) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18256392/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=HLA-B*5701%20screening%20for%20hypersensitivity%20to%20abacavir&author=S%20Mallal&author=E%20Phillips&author=G%20Carosi&volume=358&publication_year=2008&pages=568-79&pmid=18256392&doi=10.1056/NEJMoa0706135&)

13. Chung WH, Hung SI, Hong HS, et al. Medical genetics: a marker for Stevens-Johnson syndrome. Nature. 2004;428:486. doi: 10.1038/428486a.  [DOI](https://doi.org/10.1038/428486a) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15057820/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Medical%20genetics:%20a%20marker%20for%20Stevens-Johnson%20syndrome&author=WH%20Chung&author=SI%20Hung&author=HS%20Hong&volume=428&publication_year=2004&pages=486&pmid=15057820&doi=10.1038/428486a&)

14. Hung SI, Chung WH, Liou LB, et al. HLA-B*5801 allele as a genetic marker for severe cutaneous adverse reactions caused by allopurinol. Proc Natl Acad Sci U S A. 2005;102:4134–9. doi: 10.1073/pnas.0409500102.  [DOI](https://doi.org/10.1073/pnas.0409500102) | [PMC free article](/articles/PMC554812/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15743917/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&title=HLA-B*5801%20allele%20as%20a%20genetic%20marker%20for%20severe%20cutaneous%20adverse%20reactions%20caused%20by%20allopurinol&author=SI%20Hung&author=WH%20Chung&author=LB%20Liou&volume=102&publication_year=2005&pages=4134-9&pmid=15743917&doi=10.1073/pnas.0409500102&)

15. Wang H, Yan L, Zhang G, et al. Association between HLA-B*1301 and dapsone-induced hypersensitivity reactions among leprosy patients in China. J Invest Dermatol. 2013;133:2642–4. doi: 10.1038/jid.2013.192.  [DOI](https://doi.org/10.1038/jid.2013.192) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23604100/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Invest%20Dermatol&title=Association%20between%20HLA-B*1301%20and%20dapsone-induced%20hypersensitivity%20reactions%20among%20leprosy%20patients%20in%20China&author=H%20Wang&author=L%20Yan&author=G%20Zhang&volume=133&publication_year=2013&pages=2642-4&pmid=23604100&doi=10.1038/jid.2013.192&)

16. Yuan J, Guo S, Hall D, et al. Toxicogenomics of nevirapine-associated cutaneous and hepatic adverse events among populations of African, Asian, and European descent. AIDS. 2011;25:1271–80. doi: 10.1097/QAD.0b013e32834779df.  [DOI](https://doi.org/10.1097/QAD.0b013e32834779df) | [PMC free article](/articles/PMC3387531/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21505298/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=AIDS&title=Toxicogenomics%20of%20nevirapine-associated%20cutaneous%20and%20hepatic%20adverse%20events%20among%20populations%20of%20African,%20Asian,%20and%20European%20descent&author=J%20Yuan&author=S%20Guo&author=D%20Hall&volume=25&publication_year=2011&pages=1271-80&pmid=21505298&doi=10.1097/QAD.0b013e32834779df&)

17. Paparella D, Micelli M, Favoino B, D’Alo M, Fiore T, de Luca Tupputi Schinosa L. Anti-heparin-platelet factor 4 antibodies after cardiopulmonary bypass: role of HLA expression. Haematologica. 2001;86:326–7.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/11255285/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Haematologica&title=Anti-heparin-platelet%20factor%204%20antibodies%20after%20cardiopulmonary%20bypass:%20role%20of%20HLA%20expression&author=D%20Paparella&author=M%20Micelli&author=B%20Favoino&author=M%20D%E2%80%99Alo&author=T%20Fiore&volume=86&publication_year=2001&pages=326-7&pmid=11255285&)

18. Karnes JH, Cronin RM, Rollin J, et al. A genome-wide association study of heparin-induced thrombocytopenia using an electronic medical record. Thromb Haemost. 2015;113:772–81. doi: 10.1160/TH14-08-0670.  [DOI](https://doi.org/10.1160/TH14-08-0670) | [PMC free article](/articles/PMC4433536/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25503805/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Thromb%20Haemost&title=A%20genome-wide%20association%20study%20of%20heparin-induced%20thrombocytopenia%20using%20an%20electronic%20medical%20record&author=JH%20Karnes&author=RM%20Cronin&author=J%20Rollin&volume=113&publication_year=2015&pages=772-81&pmid=25503805&doi=10.1160/TH14-08-0670&)

19. Diaz-Pena R, Vidal-Castineira JR, Moro-Garcia MA, Alonso-Arias R, Castro-Santos P. Significant association of the KIR2DL3/HLA-C1 genotype with susceptibility to Crohn’s disease. Hum Immunol. 2015 doi: 10.1016/j.humimm.2015.10.020.  [DOI](https://doi.org/10.1016/j.humimm.2015.10.020) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26542067/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Immunol&title=Significant%20association%20of%20the%20KIR2DL3/HLA-C1%20genotype%20with%20susceptibility%20to%20Crohn%E2%80%99s%20disease&author=R%20Diaz-Pena&author=JR%20Vidal-Castineira&author=MA%20Moro-Garcia&author=R%20Alonso-Arias&author=P%20Castro-Santos&publication_year=2015&pmid=26542067&doi=10.1016/j.humimm.2015.10.020&)

20. Ridley A, Hatano H, Wong-Baeza I, et al. Activation-Induced Killer Cell Immunoglobulin-like Receptor 3DL2 Binding to HLA-B27 Licenses Pathogenic T Cell Differentiation in Spondyloarthritis. Arthritis Rheumatol. 2016;68:901–14. doi: 10.1002/art.39515.  [DOI](https://doi.org/10.1002/art.39515) | [PMC free article](/articles/PMC4855641/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26841353/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arthritis%20Rheumatol&title=Activation-Induced%20Killer%20Cell%20Immunoglobulin-like%20Receptor%203DL2%20Binding%20to%20HLA-B27%20Licenses%20Pathogenic%20T%20Cell%20Differentiation%20in%20Spondyloarthritis&author=A%20Ridley&author=H%20Hatano&author=I%20Wong-Baeza&volume=68&publication_year=2016&pages=901-14&pmid=26841353&doi=10.1002/art.39515&)

21. Gonzalez-Galarza FF, Takeshita LY, Santos EJ, et al. Allele frequency net 2015 update: new features for HLA epitopes, KIR and disease and HLA adverse drug reaction associations. Nucleic Acids Res. 2015;43:D784–8. doi: 10.1093/nar/gku1166.  [DOI](https://doi.org/10.1093/nar/gku1166) | [PMC free article](/articles/PMC4383964/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25414323/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=Allele%20frequency%20net%202015%20update:%20new%20features%20for%20HLA%20epitopes,%20KIR%20and%20disease%20and%20HLA%20adverse%20drug%20reaction%20associations&author=FF%20Gonzalez-Galarza&author=LY%20Takeshita&author=EJ%20Santos&volume=43&publication_year=2015&pages=D784-8&pmid=25414323&doi=10.1093/nar/gku1166&)

22. Roden DM, Pulley JM, Basford MA, et al. Development of a large-scale de-identified DNA biobank to enable personalized medicine. Clin Pharmacol Ther. 2008;84:362–9. doi: 10.1038/clpt.2008.89.  [DOI](https://doi.org/10.1038/clpt.2008.89) | [PMC free article](/articles/PMC3763939/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18500243/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Development%20of%20a%20large-scale%20de-identified%20DNA%20biobank%20to%20enable%20personalized%20medicine&author=DM%20Roden&author=JM%20Pulley&author=MA%20Basford&volume=84&publication_year=2008&pages=362-9&pmid=18500243&doi=10.1038/clpt.2008.89&)

23. Denny JC, Spickard A, 3rd, Johnson KB, Peterson NB, Peterson JF, Miller RA. Evaluation of a method to identify and categorize section headers in clinical documents. J Am Med Inform Assoc. 2009;16:806–15. doi: 10.1197/jamia.M3037.  [DOI](https://doi.org/10.1197/jamia.M3037) | [PMC free article](/articles/PMC3002123/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19717800/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Am%20Med%20Inform%20Assoc&title=Evaluation%20of%20a%20method%20to%20identify%20and%20categorize%20section%20headers%20in%20clinical%20documents&author=JC%20Denny&author=A%20Spickard&author=KB%20Johnson&author=NB%20Peterson&author=JF%20Peterson&volume=16&publication_year=2009&pages=806-15&pmid=19717800&doi=10.1197/jamia.M3037&)

24. Denny JC, Miller RA, Waitman LR, Arrieta MA, Peterson JF. Identifying QT prolongation from ECG impressions using a general-purpose Natural Language Processor. Int J Med Inform. 2009;78(Suppl 1):S34–42. doi: 10.1016/j.ijmedinf.2008.09.001.  [DOI](https://doi.org/10.1016/j.ijmedinf.2008.09.001) | [PMC free article](/articles/PMC2728459/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18938105/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Med%20Inform&title=Identifying%20QT%20prolongation%20from%20ECG%20impressions%20using%20a%20general-purpose%20Natural%20Language%20Processor&author=JC%20Denny&author=RA%20Miller&author=LR%20Waitman&author=MA%20Arrieta&author=JF%20Peterson&volume=78&issue=Suppl%201&publication_year=2009&pages=S34-42&pmid=18938105&doi=10.1016/j.ijmedinf.2008.09.001&)

25. Cuker A, Gimotty PA, Crowther MA, Warkentin TE. Predictive value of the 4Ts scoring system for heparin-induced thrombocytopenia: a systematic review and meta-analysis. Blood. 2012;120:4160–7. doi: 10.1182/blood-2012-07-443051.  [DOI](https://doi.org/10.1182/blood-2012-07-443051) | [PMC free article](/articles/PMC3501714/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22990018/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Predictive%20value%20of%20the%204Ts%20scoring%20system%20for%20heparin-induced%20thrombocytopenia:%20a%20systematic%20review%20and%20meta-analysis&author=A%20Cuker&author=PA%20Gimotty&author=MA%20Crowther&author=TE%20Warkentin&volume=120&publication_year=2012&pages=4160-7&pmid=22990018&doi=10.1182/blood-2012-07-443051&)

26. Mallal S, Nolan D, Witt C, et al. Association between presence of HLA-B*5701, HLA-DR7, and HLA-DQ3 and hypersensitivity to HIV-1 reverse-transcriptase inhibitor abacavir. Lancet. 2002;359:727–32. doi: 10.1016/s0140-6736(02)07873-x.  [DOI](https://doi.org/10.1016/s0140-6736(02)07873-x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11888582/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Association%20between%20presence%20of%20HLA-B*5701,%20HLA-DR7,%20and%20HLA-DQ3%20and%20hypersensitivity%20to%20HIV-1%20reverse-transcriptase%20inhibitor%20abacavir&author=S%20Mallal&author=D%20Nolan&author=C%20Witt&volume=359&publication_year=2002&pages=727-32&pmid=11888582&doi=10.1016/s0140-6736(02)07873-x&)

27. Zuvich RL, Armstrong LL, Bielinski SJ, et al. Pitfalls of merging GWAS data: lessons learned in the eMERGE network and quality control procedures to maintain high data quality. Genet Epidemiol. 2011;35:887–98. doi: 10.1002/gepi.20639.  [DOI](https://doi.org/10.1002/gepi.20639) | [PMC free article](/articles/PMC3592376/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22125226/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genet%20Epidemiol&title=Pitfalls%20of%20merging%20GWAS%20data:%20lessons%20learned%20in%20the%20eMERGE%20network%20and%20quality%20control%20procedures%20to%20maintain%20high%20data%20quality&author=RL%20Zuvich&author=LL%20Armstrong&author=SJ%20Bielinski&volume=35&publication_year=2011&pages=887-98&pmid=22125226&doi=10.1002/gepi.20639&)

28. Vukcevic D, Traherne JA, Naess S, et al. Imputation of KIR Types from SNP Variation Data. Am J Hum Genet. 2015;97:593–607. doi: 10.1016/j.ajhg.2015.09.005.  [DOI](https://doi.org/10.1016/j.ajhg.2015.09.005) | [PMC free article](/articles/PMC4596914/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26430804/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=Imputation%20of%20KIR%20Types%20from%20SNP%20Variation%20Data&author=D%20Vukcevic&author=JA%20Traherne&author=S%20Naess&volume=97&publication_year=2015&pages=593-607&pmid=26430804&doi=10.1016/j.ajhg.2015.09.005&)

29. Pritchard JK, Stephens M, Donnelly P. Inference of population structure using multilocus genotype data. Genetics. 2000;155:945–59. doi: 10.1093/genetics/155.2.945.  [DOI](https://doi.org/10.1093/genetics/155.2.945) | [PMC free article](/articles/PMC1461096/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10835412/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genetics&title=Inference%20of%20population%20structure%20using%20multilocus%20genotype%20data&author=JK%20Pritchard&author=M%20Stephens&author=P%20Donnelly&volume=155&publication_year=2000&pages=945-59&pmid=10835412&doi=10.1093/genetics/155.2.945&)

30. Price AL, Patterson NJ, Plenge RM, Weinblatt ME, Shadick NA, Reich D. Principal components analysis corrects for stratification in genome-wide association studies. Nat Genet. 2006;38:904–9. doi: 10.1038/ng1847.  [DOI](https://doi.org/10.1038/ng1847) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16862161/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Principal%20components%20analysis%20corrects%20for%20stratification%20in%20genome-wide%20association%20studies&author=AL%20Price&author=NJ%20Patterson&author=RM%20Plenge&author=ME%20Weinblatt&author=NA%20Shadick&volume=38&publication_year=2006&pages=904-9&pmid=16862161&doi=10.1038/ng1847&)

31. Purcell S, Neale B, Todd-Brown K, et al. PLINK: a tool set for whole-genome association and population-based linkage analyses. Am J Hum Genet. 2007;81:559–75. doi: 10.1086/519795.  [DOI](https://doi.org/10.1086/519795) | [PMC free article](/articles/PMC1950838/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17701901/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=PLINK:%20a%20tool%20set%20for%20whole-genome%20association%20and%20population-based%20linkage%20analyses&author=S%20Purcell&author=B%20Neale&author=K%20Todd-Brown&volume=81&publication_year=2007&pages=559-75&pmid=17701901&doi=10.1086/519795&)

32. Khoury MJ, Flanders WD. Nontraditional epidemiologic approaches in the analysis of gene-environment interaction: case-control studies with no controls! Am J Epidemiol. 1996;144:207–13. doi: 10.1093/oxfordjournals.aje.a008915.  [DOI](https://doi.org/10.1093/oxfordjournals.aje.a008915) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8686689/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Epidemiol&title=Nontraditional%20epidemiologic%20approaches%20in%20the%20analysis%20of%20gene-environment%20interaction:%20case-control%20studies%20with%20no%20controls!&author=MJ%20Khoury&author=WD%20Flanders&volume=144&publication_year=1996&pages=207-13&pmid=8686689&doi=10.1093/oxfordjournals.aje.a008915&)

33. Cuker A. Heparin-induced thrombocytopenia: present and future. J Thromb Thrombolysis. 2011;31:353–66. doi: 10.1007/s11239-011-0569-6.  [DOI](https://doi.org/10.1007/s11239-011-0569-6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21327506/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Thromb%20Thrombolysis&title=Heparin-induced%20thrombocytopenia:%20present%20and%20future&author=A%20Cuker&volume=31&publication_year=2011&pages=353-66&pmid=21327506&doi=10.1007/s11239-011-0569-6&)

34. Greinacher A, Holtfreter B, Krauel K, et al. Association of natural anti-platelet factor 4/heparin antibodies with periodontal disease. Blood. 2011;118:1395–401. doi: 10.1182/blood-2011-03-342857.  [DOI](https://doi.org/10.1182/blood-2011-03-342857) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21659541/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Association%20of%20natural%20anti-platelet%20factor%204/heparin%20antibodies%20with%20periodontal%20disease&author=A%20Greinacher&author=B%20Holtfreter&author=K%20Krauel&volume=118&publication_year=2011&pages=1395-401&pmid=21659541&doi=10.1182/blood-2011-03-342857&)

35. Khandelwal S, Arepally GM. Immune pathogenesis of heparin-induced thrombocytopenia. Thromb Haemost. 2016:116. doi: 10.1160/TH16-01-0074.  [DOI](https://doi.org/10.1160/TH16-01-0074) | [PMC free article](/articles/PMC6260829/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27465274/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Thromb%20Haemost&title=Immune%20pathogenesis%20of%20heparin-induced%20thrombocytopenia&author=S%20Khandelwal&author=GM%20Arepally&publication_year=2016&pages=116&pmid=27465274&doi=10.1160/TH16-01-0074&)

36. Zheng Y, Yu M, Podd A, et al. Critical role for mouse marginal zone B cells in PF4/heparin antibody production. Blood. 2013;121:3484–92. doi: 10.1182/blood-2013-01-477091.  [DOI](https://doi.org/10.1182/blood-2013-01-477091) | [PMC free article](/articles/PMC3637017/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23460609/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Critical%20role%20for%20mouse%20marginal%20zone%20B%20cells%20in%20PF4/heparin%20antibody%20production&author=Y%20Zheng&author=M%20Yu&author=A%20Podd&volume=121&publication_year=2013&pages=3484-92&pmid=23460609&doi=10.1182/blood-2013-01-477091&)

37. Valentin N, Vergracht A, Bignon JD, et al. HLA-DRw52a is involved in alloimmunization against PL-A1 antigen. Hum Immunol. 1990;27:73–9. doi: 10.1016/0198-8859(90)90104-w.  [DOI](https://doi.org/10.1016/0198-8859(90)90104-w) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/2298610/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Immunol&title=HLA-DRw52a%20is%20involved%20in%20alloimmunization%20against%20PL-A1%20antigen&author=N%20Valentin&author=A%20Vergracht&author=JD%20Bignon&volume=27&publication_year=1990&pages=73-9&pmid=2298610&doi=10.1016/0198-8859(90)90104-w&)

38. Delbos F, Bertrand G, Croisille L, Ansart-Pirenne H, Bierling P, Kaplan C. Fetal and neonatal alloimmune thrombocytopenia: predictive factors of intracranial hemorrhage. Transfusion. 2016;56:59–66. doi: 10.1111/trf.13274. quiz 58.  [DOI](https://doi.org/10.1111/trf.13274) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26469867/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Transfusion&title=Fetal%20and%20neonatal%20alloimmune%20thrombocytopenia:%20predictive%20factors%20of%20intracranial%20hemorrhage&author=F%20Delbos&author=G%20Bertrand&author=L%20Croisille&author=H%20Ansart-Pirenne&author=P%20Bierling&volume=56&publication_year=2016&pages=59-66&pmid=26469867&doi=10.1111/trf.13274&)

39. Scheuerer B, Ernst M, Durrbaum-Landmann I, et al. The CXC-chemokine platelet factor 4 promotes monocyte survival and induces monocyte differentiation into macrophages. Blood. 2000;95:1158–66.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/10666185/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=The%20CXC-chemokine%20platelet%20factor%204%20promotes%20monocyte%20survival%20and%20induces%20monocyte%20differentiation%20into%20macrophages&author=B%20Scheuerer&author=M%20Ernst&author=I%20Durrbaum-Landmann&volume=95&publication_year=2000&pages=1158-66&pmid=10666185&)

40. Scully M, Brown J, Patel R, McDonald V, Brown CJ, Machin S. Human leukocyte antigen association in idiopathic thrombotic thrombocytopenic purpura: evidence for an immunogenetic link. J Thromb Haemost. 2010;8:257–62. doi: 10.1111/j.1538-7836.2009.03692.x.  [DOI](https://doi.org/10.1111/j.1538-7836.2009.03692.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19922436/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Thromb%20Haemost&title=Human%20leukocyte%20antigen%20association%20in%20idiopathic%20thrombotic%20thrombocytopenic%20purpura:%20evidence%20for%20an%20immunogenetic%20link&author=M%20Scully&author=J%20Brown&author=R%20Patel&author=V%20McDonald&author=CJ%20Brown&volume=8&publication_year=2010&pages=257-62&pmid=19922436&doi=10.1111/j.1538-7836.2009.03692.x&)
