# Functional characterization of wild-type and 24 CYP2D6 allelic variants on gefitinib metabolism in vitro

## Metadata
**Authors:** Ping Fang, Xiang Zheng, Jiayang He, Honglei Ge, Pengfei Tang, Jianping Cai, Guoxin Hu
**Journal:** Drug Design, Development and Therapy
**Date:** 2017 Apr 21
**DOI:** [10.2147/DDDT.S133814](https://doi.org/10.2147/DDDT.S133814)
**PMID:** 28461741
**PMCID:** PMC5407444
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5407444/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC5407444/pdf/dddt-11-1283.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC5407444/pdf/dddt-11-1283.pdf)

## Abstract

**Background:** 
Cytochrome P450 2D6 (CYP2D6), a member of the CYP450 enzyme super family, is a polymorphic enzyme that metabolizes ~25% of therapeutic drugs. CYP2D6 exhibits significant genetic polymorphisms which might cause adverse effects and therapeutic failures of some drugs.

**Objective:** 
The purpose of this study was to evaluate the catalytic activities of 22 novel CYP2D6 alleles (CYP2D6*87, *88, *89, *90, *91, *92, *93, *94, *95, *96, *97, *98, R25Q, F164L, E215K, F219S, V327M, D336N, V342M, R344Q, R440C, R497C) on the metabolism of gefitinib in vitro.

**Methods and results:** 
CYP2D6 variants were incubated with 1–100 μM gefitinib for 60 min at 37°C and the reaction was terminated by cooling to −80°C immediately. Gefitinib and its metabolite O-desmethyl gefitinib were analyzed by an ultra-performance liquid chromatography-tandem mass spectrometry system. Compared to CYP2D6.1, most CYP2D6 variants exhibited significantly decreased relative clearance values (from 3.11% to 79.35%), whereas CYP2D6.92 and CYP2D6.96 displayed no detectable enzyme activity. Only CYP2D6.94 exhibited a markedly increased intrinsic clearance value, and eight variants (CYP2D6.88, CYP2D6.89, CYP2D6.91, CYP2D6.97, V342M, R344Q, F219S, and F164L) showed no significant difference. In addition, 23 CYP2D6 allelic isoforms exhibited substrate inhibition trend toward gefitinib.

**Conclusion:** 
As the first study of all the aforementioned alleles for gefitinib metabolism, these comprehensive data may help in the clinical assessment of the metabolism of gefitinib, and may also offer a reference for personalized treatment with gefitinib in clinical settings.

Keywords: CYP2D6, genetic polymorphism, drug metabolism, gefitinib, enzymatic activity, personalized treatment, NSCLC

### Background

Cytochrome P450 2D6 (CYP2D6), a member of the CYP450 enzyme super family, is a polymorphic enzyme that metabolizes ~25% of therapeutic drugs. CYP2D6 exhibits significant genetic polymorphisms which might cause adverse effects and therapeutic failures of some drugs.

### Objective

The purpose of this study was to evaluate the catalytic activities of 22 novel CYP2D6 alleles (*CYP2D6*87*, **88*, **89*, **90*, **91*, **92*, **93*, **94*, **95*, **96*, **97*, **98*, *R25Q*, *F164L*, *E215K*, *F219S*, *V327M*, *D336N*, *V342M*, *R344Q*, *R440C*, *R497C*) on the metabolism of gefitinib in vitro.

### Methods and results

CYP2D6 variants were incubated with 1–100 μM gefitinib for 60 min at 37°C and the reaction was terminated by cooling to −80°C immediately. Gefitinib and its metabolite *O*-desmethyl gefitinib were analyzed by an ultra-performance liquid chromatography-tandem mass spectrometry system. Compared to CYP2D6.1, most CYP2D6 variants exhibited significantly decreased relative clearance values (from 3.11% to 79.35%), whereas CYP2D6.92 and CYP2D6.96 displayed no detectable enzyme activity. Only CYP2D6.94 exhibited a markedly increased intrinsic clearance value, and eight variants (CYP2D6.88, CYP2D6.89, CYP2D6.91, CYP2D6.97, V342M, R344Q, F219S, and F164L) showed no significant difference. In addition, 23 CYP2D6 allelic isoforms exhibited substrate inhibition trend toward gefitinib.

### Conclusion

As the first study of all the aforementioned alleles for gefitinib metabolism, these comprehensive data may help in the clinical assessment of the metabolism of gefitinib, and may also offer a reference for personalized treatment with gefitinib in clinical settings.

## Introduction

Lung cancer remains the most frequently diagnosed malignancy and the leading cause of cancer-related death worldwide.[1](#b1-dddt-11-1283)–[3](#b3-dddt-11-1283) Approximately 80%–85% of all lung cancers are non-small cell lung cancer (NSCLC) with ~55% of these presenting with locally advanced or metastatic disease causing the 5-year survival rate to be as low as <5%.[1](#b1-dddt-11-1283),[3](#b3-dddt-11-1283) In recent years, the discovery of the epidermal growth factor receptor tyrosine kinase inhibitor (EGFR-TKI) brings a new hope for patients in the treatment of advanced NSCLC. EGFR is a known transmembrane receptor, which upon ligand binding, induces receptor dimerization and tyrosine autophosphorylation and leads to cell proliferation.[4](#b4-dddt-11-1283)–[6](#b6-dddt-11-1283) Mutation in this gene is significantly associated with lung cancer.[7](#b7-dddt-11-1283) In previous studies, the EGFR is detected by immunohistochemistry on 40%–80% of NSCLC.[8](#b8-dddt-11-1283),[9](#b9-dddt-11-1283) Accordingly, promising molecular inhibitors targeting the activating mutants of the EGFR have been intensely pursued in the treatment of EGFR-mutant positive NSCLC.

In the last decade, two small molecules, selective, orally bioavailable and reversible EGFR-TKIs gefitinib and erlotinib have been extensively developed in NSCLC.[3](#b3-dddt-11-1283) Gefitinib ([Figure 1A](#f1-dddt-11-1283)) is the first EGFR-TKI, which has been approved as early as 2002 for the treatment of advanced NSCLC patients. In 2003, gefitinib received accelerated approval by the US Food and Drug Administration (FDA) as third-line treatment for advanced EGFR-mutated NSCLC patients. Then, in 2015 it was approved as the first-line treatment of patients with metastatic NSCLC harboring activating EGFR mutations by FDA.[10](#b10-dddt-11-1283)–[12](#b12-dddt-11-1283) Gefitinib is metabolized by the hepatic cytochrome P450 (CYP450) enzyme CYP2D6, CYP3A4, and CYP3A5. And CYP2D6 is the enzyme primarily responsible for the formation of *O*-desmethyl gefitinib ([Figure 1B](#f1-dddt-11-1283)).[13](#b13-dddt-11-1283) Previous studies indicate that among gefitinib metabolites, only *O*-desmethyl gefitinib can be identified in human plasma, whereas the plasma concentrations of other gefitinib metabolites, such as M537194 and M387783, which are metabolized by CYP3A4 are very low or even undetectable.[2](#b2-dddt-11-1283),[13](#b13-dddt-11-1283)–[16](#b16-dddt-11-1283) There are several studies showing that patients with poor metabolizers (PMs) of CYP2D6 and inhibition of CYP3A4 may lead to prominent overdose of gefitinib, which may finally cause gefitinib-induced hepatotoxicity.[13](#b13-dddt-11-1283),[16](#b16-dddt-11-1283) Therefore, we believe that it is necessary to evaluate the effects of 24 CYP2D6 variants on the metabolism of gefitinib in vitro.

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2637/5407444/5dc50642face/dddt-11-1283Fig1.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5407444_dddt-11-1283Fig1.jpg)

Structure of the analytes and metabolic pathway of gefitinib by CYP2D6 enzyme. Notes: (A) Gefitinib. (B) O-desmethyl gefitinib.

As a member of CYP450 enzyme superfamily, CYP450 2D6 plays a vital role in the metabolism of ~20%–30% of therapeutically important drugs, mainly including antipsychotic drugs, antiarrhythmic agents, tricyclic antidepressants, β-blockers, and opioids.[17](#b17-dddt-11-1283),[18](#b18-dddt-11-1283) Like other CYP450 enzymes, CYP2D6 exhibits significant genetic polymorphisms between individuals. To date, over 100 CYP2D6 alleles have been identified and named by the Human CYP Allele Nomenclature committee ([http://www.cypalleles.ki.se/cyp2d6.htm](http://www.cypalleles.ki.se/cyp2d6.htm)). The genetic polymorphisms in CYP2D6 cause the tremendous variation in CYP2D6 enzyme activities. Thus, individuals can be classified into four phenotypes: PMs, intermediate metabolizers, extensive metabolizers (EMs), or ultrarapid metabolizers. Genetic polymorphisms in CYP2D6 have been shown to have clinical consequences, resulting in toxicity and/or altered efficacy of many drugs. Thus, the identification of the polymorphic allele, especially defective alleles, in different racial or ethnic populations, is of great importance to understand the differences in the clinical response to drugs.[19](#b19-dddt-11-1283)

Previously, 22 new nonsynonymous mutation sites among the 2,129 people of the Chinese Han population were discovered, 12 of them were defined as the novel alleles (*CYP2D6*87*–**93*, **94A*, **94B*, and **95–*98*) by the Human CYP Allele Nomenclature committee.[20](#b20-dddt-11-1283) In our study, we systematically assessed the enzymatic characteristics of these 22 CYP2D6 allelic variants and 2 typical variants (CYP2D6.2 and CYP2D6.10) as well as the wild type CYP2D6.1 toward gefitinib. And we hope that our data can provide valuable information about CYP2D6 genetic polymorphisms and offer a reference to the personalized medication for CYP2D6-metabolized drugs.

## Materials and methods

### Chemicals and reagents

Gefitinib and its metabolite *O*-desmethyl gefitinib were purchased from Shanghai Canspec Scientific Instruments Co., Ltd (Perfemiker, Shanghai, China), the reduced nicotinamide adenine dinucleotide phosphate (NADPH) and carbamazepine (internal standard, IS) were obtained from Sigma-Aldrich Company (St Louis, MO, USA). P450 Cytochrome b5 microsomes and recombinant human CYP2D6 microsomes were kind gifts from Beijing Hospital (Beijing, China).[21](#b21-dddt-11-1283) High performance liquid chromatography (HPLC) grade organic solvents and LC-MS grade acetonitrile were purchased from Merck (Darmstadt, Germany), formic acid of HPLC grade (FA, purity 99.9%) was purchased from J&K scientific Ltd. (Shanghai, China), and all other reagents used were of analytical grade.

### Instrumentation

Samples were analyzed by ultra-performance liquid chromatography-tandem mass spectrometry (UPLC-MS/MS) using a Waters Acquity I-Class and a Waters Xevo TQD triple-quadrupole mass spectrometry (Waters Corp., Milford, MA, USA) with an electrospray ionization source. Instrument control and data acquisition were performed by Masslynx 4.1 software (Waters Corp.).

### Conditions for enzymatic activity analysis

Recombinant microsomes expressing CYP2D6 variants generated in *Spodoptera frugioerda* 21 (Sf21) insect cells were obtained according to the previously reported methods.[21](#b21-dddt-11-1283) Western blotting was performed to determine protein expression levels and the content of microsomes was determined using a BCA Protein Assay Kit with BSA as a standard.[21](#b21-dddt-11-1283) The volume of incubation mixture is 200 μL, which consisted of microsomes (5 pmol CYP2D6.1 or 5 pmol other CYP2D6 mutants, respectively), purified cytochrome b5, gefitinib (1–100 μM) and 0.1 M Tris-HCl buffer (pH 7.4). The reaction was allowed to preincubate for 5 min in a Fisher shaking water bath. Then 1 mM NADPH was added to start the reaction at 37°C. And incubations proceeded at 37°C for 60 min. Reactions were terminated by cooling to −80°C immediately. Then 400 μL acetonitrile and 20 μL carbamazepine (internal standard, 1 μg/mL) were added to the incubation mixture, which was then vortexed for 2 min and centrifuged at 13,000 rpm at 4°C for 10 min. The supernatant was 1:1 diluted with water, and 2 μL of the mixture was injected into the UPLC-MS/MS system for analysis. Incubations were performed in triplicate and data presented as the mean ± standard deviation.

### Chromatographic and mass spectrometric conditions

The chromatographic separation was carried out using an ACQUITY UPLC-MS/MS and performed on a Waters ACQUITY UPLC BEH C18 column (2.1×50 mm, 1.7 μm, Waters Corp.) at 40°C. The initial mobile phase consisted of solvent A (water containing 0.1% formic acid) and solvent B (acetonitrile) with gradient elution at a flow rate of 0.4 mL/min, injection volume 2 μL. A gradient elution program was as follows: 0–0.6 min (60%–10% A), 0.6–1.2 min (10% A), 1.2–1.8 min (10%–60% A). The total run time was 2.5 min. Under above conditions, *O*-desmethyl gefitinib and carbamazepine were well separated and their retention times were 0.35 and 0.77 min, respectively.

A Waters XEVO TQD MS was set to positive electrospray ionization in multiple reaction monitoring mode, with ionization conditions as follows: desolvation temperature 500°C, ionization source temperature 150°C, and capillary voltage 2,000 V. Nitrogen was served as desolvation gas with flow rate of 1,000 L/h. The multiple reaction monitoring transitions were *m*/*z* 447.2→128.1, *m*/*z* 433.1→128.1, and *m*/*z* 237.1→194.2 for gefitinib, *O*-desmethyl gefitinib, and carbamazepine, respectively. The collision energy for gefitinib, *O*-desmethyl gefitinib, and carbamazepine was set at 25, 25, and 20 V, respectively; and the cone voltage for each was set at 40 V.

Prior to the experiments, the UPLC-MS/MS method was validated. A seven-point standard curve in the range of 1–50 μg/mL was used to quantify *O*-desmethyl gefitinib, with a correlation coefficient *r*^2^>0.99. The limit of quantification, defined as a signal/noise ratio of 3, was 0.2 ng/mL for *O*-desmethyl gefitinib. The precision and accuracy of the method were assessed by determination of quality control samples at three concentrations. The relative standard deviations of intra-day and inter-day precisions were both below 10%; the accuracy was from 87.77% to 93.73%. These results demonstrated that the method was accurate and precise.

### Statistical analysis

Michaelis–Menten curves (substrate vs velocity) and enzyme kinetic parameters were obtained by using GraphPad Prism 5 (GraphPad Software Inc., SanDiego, CA, USA). One-way analysis of variance (ANOVA) was used for intergroup comparison, and various variants were considered as factor whereas maximum initial velocity (V_max_), Michaelis constant (K_m_) or intrinsic clearance (C_lint_) values were deemed as dependent list. Statistical analysis was carried out with the Statistical Package for the Social Sciences (version 17.0; SPSS Inc., Chicago, IL, USA) by one-way ANOVA together with the post hoc Dunnet method, with **P*<0.05 considered to be statistically significant.

## Results

In our study, the catalytic activities of the wild-type CYP2D6.1 and 24 allelic variants were detected using gefitinib as substrate and intrinsic clearance (V_max_/K_m_) was used as evaluation criteria for each variant of CYP2D6 in vitro metabolic activity of gefitinib. The Michaelis–Menten curves and substrate inhibition plots for each of the CYP2D6 variants are shown in [Figure 2](#f2-dddt-11-1283), and the corresponding kinetic parameters (V_max_, K_m_, K_si_, intrinsic clearance, and relative clearance) are summarized in [Table 1](#t1-dddt-11-1283).

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2637/5407444/38697bc16e8f/dddt-11-1283Fig2.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5407444_dddt-11-1283Fig2.jpg)

Michaelis–Menten curves of the enzymatic activity of the wild-type and 24 variants toward gefitinib O-demethylation. Notes: Each point represents the mean ± standard deviation of three parallel experiments. The allelic variants with designated allele names have been manually arranged into six different groups: CYP2D6*1, *2, *10, *87, and *88 (A); CYP2D6*1, *94, *95, R25Q, and E215K (B); CYP2D6*1, *90, *97, V327M, and R440C (C); CYP2D6*1, *91, *93, and R497C (D); CYP2D6*1, *89, *98, V342M, and R344Q (E); CPY2D6*1, F164L, F219S, and D336N (F). Abbreviation: CYP2D6, cytochrome P450 2D6.

### Table 1.

| Variants | Allele frequency (%) | Vmax (pmol/min/pmol P450) | Km (μM) | Ksi (μM) | Clint (Vmax/Km) | Relative clearance (% of wild type) |
| --- | --- | --- | --- | --- | --- | --- |
| CYP2D6.1 | 26.56 | 1.25±0.02 | 6.94±0.44 | 158.10±31.16 | 0.18±0.01 | 100.00 |
| CYP2D6.2 (R296C; S486T) | 10.34 | 0.82±0.14 | 19.61±4.69** | 31.61±6.97** | 0.04±0.00** | 23.43** |
| CYP2D6.10 (P34S; S486T) | 42.86 | 0.09±0.00** | 16.83±1.53* | 41.86±1.73** | 0.01±0.00** | 3.11** |
| CYP2D6.87 (A5V) | 0.023 | 1.31±0.03 | 11.02±0.42 | 62.21±4.16** | 0.12±0.00** | 65.82** |
| CYP2D6.88 (V104A) | 0.094 | 2.28±0.21** | 11.80±1.70 | 93.66±15.33** | 0.19±0.01 | 107.23 |
| CYP2D6.89 (L142S) | 0.023 | 2.06±0.06** | 12.65±0.63 | 40.98±2.76** | 0.16±0.00 | 90.21 |
| CYP2D6.90 (K147R) | 0.047 | 1.83±0.08* | 15.17±0.40 | 74.55±6.36** | 0.12±0.00** | 66.97** |
| CYP2D6.91 (C161S) | 0.023 | 1.85±0.08** | 12.42±0.69 | 75.72±6.97** | 0.15±0.00 | 82.68 |
| CYP2D6.92 (218frameshift) | 0.023 | ND | ND | ND | ND | ND |
| CYP2D6.93 (T249P) | 0.023 | 0.07±0.00** | 10.61±0.43 | 88.43±15.92** | 0.01±0.00** | 3.68** |
| CYP2D6.94 (D337G) | 0.164 | 1.46±0.22 | 4.57±1.62 | 113.04±42.64** | 0.34±0.07** | 184.78** |
| CYP2D6.95 (R388H) | 0.047 | 1.27±0.02 | 14.46±0.59 | 88.79±2.62** | 0.09±0.00** | 48.82** |
| CYP2D6.96 (424STOP) | 0.074 | ND | ND | ND | ND | ND |
| CYP2D6.97 (F457L) | 0.047 | 1.42±0.04 | 8.32±0.62 | 58.58±2.45** | 0.17±0.01 | 94.72 |
| CYP2D6.98 (H463D) | 0.023 | 7.86±0.37** | 96.63±5.08** | 5.92±0.30** | 0.08±0.00** | 45.13** |
| R25Q | 0.023 | 1.31±0.48 | 23.67±11.93** | 13.79±5.10** | 0.06±0.01** | 32.30** |
| F164L | 0.023 | 1.68±0.10 | 11.45±0.69 | 74.59±8.44** | 0.15±0.00 | 81.30 |
| E215K | 0.047 | 0.69±0.07* | 39.32±5.27** | 14.46±1.78** | 0.02±0.00** | 9.79** |
| F219S | 0.023 | 1.29±0.05 | 7.71±0.28 | 83.21±12.52** | 0.17±0.00 | 93.16 |
| V327M | 0.023 | 0.94±0.14 | 9.00±2.54 | 67.10±13.51** | 0.11±0.01** | 58.91** |
| D336N | 0.023 | 2.93±0.40** | 24.08±4.91** | 20.31±3.49** | 0.12±0.01** | 67.81** |
| V342M | 0.023 | 2.99±0.24** | 13.71±1.29 | 29.59±3.42** | 0.22±0.00 | 120.96 |
| R344Q | 0.023 | 2.28±0.02** | 15.92±0.29* | 28.90±0.55** | 0.14±0.00 | 79.35 |
| R440C | 0.023 | 1.86±0.31** | 19.93±4.21** | 11.56±2.80** | 0.09±0.00** | 52.07** |
| R497C | 0.023 | 1.21±0.14 | 9.76±2.04 | 77.45±18.65** | 0.13±0.01** | 69.42** |

Table 1 Caption: Kinetic parameters for O-desmethyl gefitinib activities of recombinant wild-type and 24 mutant CYP2D6 proteins against gefitinib

As shown in [Table 1](#t1-dddt-11-1283), among all these variants, CYP2D6.92 and CYP2D6.96 showed no enzymatic activity as no concentration of *O*-desmethyl gefitinib was detected. Except for CYP2D6.92 and CYP2D6.96, the remaining 22 variants displayed considerable differences in K_m_, V_max_, K_si_, or intrinsic clearance values compared with wild-type CYP2D6.1. Three allelic variants (CYP2D6.10, CYP2D6.93, and E215K) exhibited lower V_max_ values and eight variants (CYP2D6.2, CYP2D6.10, CYP2D6.98, R25Q, E215K, D336N, R344Q, and R440C) had higher K_m_ values than that of CYP2D6.1 (**P*<0.05). According to the intrinsic clearance values when compared with the wild-type, only one variant (CYP2D6.94) had increased catalytic activity to 184.78%, whereas the remaining 13 variants (CYP2D6.2, CYP2D6.10, CYP2D6.87, CYP2D6.90, CYP2D6.93, CYP2D6.95, CYP2D6.98, R25Q, E215K, V327M, D336N, R440C, and R497C) had a significant decrease in enzyme activities compared with CYP2D6.1 (**P*<0.05), for which the percentage varied from 3.11% to 69.42%. Especially, the intrinsic clearance values for variants CYP2D6.10, CYP2D6.93, and E215K were decreased to 3.11%, 3.68%, and 9.79% compared with CYP2D6.1, respectively. However, through the analysis of the kinetic parameters, there are eight variants (CYP2D6.88, CYP2D6.89, CYP2D6.91, CYP2D6.97, V342M, R344Q, F219S, and F164L) which showed no marked difference in intrinsic clearance values compared with wild type.

In addition, different from the previous studies,[19](#b19-dddt-11-1283),[21](#b21-dddt-11-1283)–[24](#b24-dddt-11-1283) the metabolic ability toward gefitinib in vitro exhibited substrate inhibition trend, the inhibition constant of 25 tested CYP2D6 variants varied from nearly 5.9 to 158.1 μM.

## Discussion

CYP2D6 plays a major role in the oxidation of ~20%–30% prescription drugs, and it is involved in the metabolism of gefitinib to its active metabolite, *O*-desmethyl gefitinib. According to the research, the formation of *O*-desmethyl gefitinib seems to be dependent on CYP2D6 activity. Therefore, polymorphisms in the CYP2D6 gene can greatly influence the rate of elimination of gefitinib. Individuals who are PMs of CYP2D6 have twofold higher plasma concentrations of gefitinib than EMs.[14](#b14-dddt-11-1283) Up to now, the enzymatic characteristics of CYP2D6 alleles toward gefitinib have not been well studied both in vivo and in vitro.

CYP2D6.2 and CYP2D6.10 tend to undergo the most study of the variants. In order to improve the credibility of our experiment, CYP2D6.2 and CYP2D6.10, two typical variants, served as the positive controls for the functional analyses. CYP2D6*10 is the most common allele in Asians, occurring in 42.86% of the Chinese Han population.[20](#b20-dddt-11-1283),[25](#b25-dddt-11-1283) Several functional analyses have been shown that CYP2D6.10 has higher K_m_, lower V_max_ and behave with the least V_max_/K_m_ values for dextromethorphan, bufuralol, nortriptyline, atomoxetine, and debrisoquine in vitro.[21](#b21-dddt-11-1283),[25](#b25-dddt-11-1283) *CYP2D6*10* is generally considered to be an allele associated with the intermediate metabolizer phenotype. For typical defective allele *CYP2D6*2*, Cai et al found it exhibited lower intrinsic clearance value for dextromethorphan and bufuralol, ~43.85% and 40.41% of wild type, respectively.[21](#b21-dddt-11-1283) As a consequence, in our study the relative clearance of CYP2D6.2 (23.43% of CYP2D6.1) and CYP2D6.10 (3.11% of CYP2D6.1) were both presented significantly decreased. These results were consistent with previous work and indicate that it is appropriate to use our CYP expression and in vitro metabolism system for analyzing the catalytic activities of the other CYP2D6 variants.[21](#b21-dddt-11-1283),[22](#b22-dddt-11-1283),[26](#b26-dddt-11-1283),[27](#b27-dddt-11-1283)

To better understand the enzymatic characteristics of CYP2D6 allelic isoforms and their effects on the metabolism of gefitinib in vitro, we functionally analyzed the enzymatic activity of 22 novel CYP2D6 alleles toward gefitinib using recombinant insect microsomes. We found that some of the variants showed similar trend about the enzyme activity as studies reported,[21](#b21-dddt-11-1283),[27](#b27-dddt-11-1283) but some caused increased or no change in enzyme activity, which is not in accordance with the results of previous research. It indicates that CYP2D6 not only exhibits significant genetic polymorphisms between individuals and among different races and ethnicities, but also showed marked differences on the metabolism of various clinical drugs. Therefore, it is of great necessity to undertake further research on the function of CYP2D6 alleles on the metabolism of gefitinib.

In our study, both CYP2D6.92 and CYP2D6.96 had undetectable enzyme activity in gefitinib metabolism, similar to previous reports.[19](#b19-dddt-11-1283),[21](#b21-dddt-11-1283),[23](#b23-dddt-11-1283),[24](#b24-dddt-11-1283) The reason for this may be because both of these variants were expressed as truncated proteins due to the premature termination in protein translation process, which led to the function loss of enzymatic activity. Particularly, only one variant CYP2D6.94, with a decrease K_m_ and an increase V_max_, showed significant increased relative clearance compared with CYP2D6.1, suggesting higher catalytic activity than CYP2D6.1. On the contrary, as reported in other studies, CYP2D6.94 exhibited markedly different decreased catalytic activity toward propranolol (31.37%), nebivolol (58.23%), dextromethorphan (67.03%), venlafaxine (71.9%), and bufuralol (74.96%) in vitro.[19](#b19-dddt-11-1283),[21](#b21-dddt-11-1283),[23](#b23-dddt-11-1283),[24](#b24-dddt-11-1283) These data suggest that CYP2D6.94 may be reclassified as an ultrarapid metabolizer phenotype for gefitinib. Moreover, CYP2D6.88, CYP2D6.89, CYP2D6.91, CYP2D6.97, F164L, F219S, and V342M were found in Chinese people and exhibited similar metabolic activity of CYP2D6.1 for gefitinib. These results were significantly different from previous studies. Cai et al showed that CYP2D6.91, CYP2D6.97, F164L, F219S, and V342M exhibited markedly decreased catalytic activity toward bufuralol and dextromethorphan. In addition, CYP2D6.88 and CYP2D6.89 exhibited decreased intrinsic clearance for bufuralol as compared with wild type but no significant difference toward dextromethorphan.[21](#b21-dddt-11-1283) These results suggest that CYP2D6 had substrate-specific alterations in metabolic activity and that results obtained from one substrate cannot necessarily be extrapolated to other substrates. We speculate that the different affinity of enzyme and substrate might be responsible for this inconsistency.

It is important to point out that two allelic variants CYP2D6.93 and E215K exhibited a drastic decrease in enzymatic activity. CYP2D6.93 exhibited ~94.4% lower V_max_, 1.53-fold higher K_m_, and 96.32% lower intrinsic clearance values compared with wild type. As for this result, Dai et al speculated that the normal entrance of the CYP2D6 substrate might be blocked, resulting from the replacement of the hydrophilic Thr with the hydrophilic Pro at position 249, which would influence the spatial structure of the adjacent egress channel. *E215K* contains an amino acid switch from Glu to Lys at position 215, which is located in F-helix in the active site cavity of the CYP2D6 protein.[27](#b27-dddt-11-1283) That may be related with E215K’s extremely decreased intrinsic clearance values (9.79%) of CYP2D6.1 owing to high K_m_ (5.67-fold). Our data suggests that CYP2D6.93 and E215K may have to be associated with a PM phenotype in vivo, with gefitinib as a substrate. Several studies have reported that patients with NSCLC who possessed PM phenotype have severe hepatotoxicity after treatment with gefitinib.[16](#b16-dddt-11-1283),[28](#b28-dddt-11-1283) Our work may serve as a reference for rational administration clinically, thereby can avoid adverse drug reactions such as gefitinib-induced hepatotoxicity.

Furthermore, the most meaningful difference between our study and previous reports[19](#b19-dddt-11-1283),[21](#b21-dddt-11-1283)–[24](#b24-dddt-11-1283) is that gefitinib exhibited a substrate inhibition trend of metabolic ability by CYP2D6 in vitro. Twenty-three tested CYP2D6 allelic variants showed this substrate inhibition trend toward gefitinib and their K_si_ values varied from ~6 to 158 μM. As reported in a study by Swaisland et al, suggested that gefitinib produced a little inhibition of CYP2D6 isoenzyme activity which resulted in a small increase in the metoprolol area under the curve and the maximum serum concentration.[29](#b29-dddt-11-1283) Hereby, we know that gefitinib was both an inhibitor and a substrate of CYP2D6. That might be the main reason for this substrate inhibition. In addition, we believe that it is still valuable to pay more consideration to the dose of CYP2D6 substrates which have a narrow therapeutic window when in the presence of gefitinib.

## Conclusion

In summary, we systematically assessed the enzymatic activity of 24 CYP2D6 allelic isoforms on the metabolism of gefitinib. Our results indicate that most of these CYP2D6 variants exhibit significantly decreased enzymatic activities. In addition, it has been reported that inhibition of CYP3A4 leads to a shunt to a CYP2D6-dependent metabolic pathway. Therefore, we speculate that patients with these dysfunctional CYP2D6 alleles should reduce the dosage of gefitinib in order to avoid gefitinib-induced hepatotoxicity when functions of CYP3A4 are also inhibited. Moreover, further studies in vivo or in vitro on the effects of CYP2D6 and CYP3A4 toward gefitinib are required to make sense of the relation between CYPs and gefitinib clearance. As the first study of these alleles for gefitinib metabolism, it may help in guiding individualization and optimization in patient treatment.

## Acknowledgments

This work was supported by the Ministry of Health, National Health and Family Planning Commission of the People’s Republic of China. The authors thank the members of the Beijing Institute of Geriatrics of the Ministry of Health for their advice and assistance.

## Footnotes

## References

1. Cappuzzo F, Morabito A, Normanno N, et al. Efficacy and safety of rechallenge treatment with gefitinib in patients with advanced non-small cell lung cancer. Lung Cancer. 2016;99:31–37. doi: 10.1016/j.lungcan.2016.06.008.  [DOI](https://doi.org/10.1016/j.lungcan.2016.06.008) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27565910/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lung%20Cancer&title=Efficacy%20and%20safety%20of%20rechallenge%20treatment%20with%20gefitinib%20in%20patients%20with%20advanced%20non-small%20cell%20lung%20cancer&author=F%20Cappuzzo&author=A%20Morabito&author=N%20Normanno&volume=99&publication_year=2016&pages=31-37&pmid=27565910&doi=10.1016/j.lungcan.2016.06.008&)

2. Han SY, Zhao HY, Zhou N, Zhou F, Li PP. Marsdenia tenacissima extract inhibits gefitinib metabolism in vitro by interfering with human hepatic CYP3A4 and CYP2D6 enzymes. J Ethnopharmacol. 2014;151(1):210–217. doi: 10.1016/j.jep.2013.10.021.  [DOI](https://doi.org/10.1016/j.jep.2013.10.021) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24157377/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Ethnopharmacol&title=Marsdenia%20tenacissima%20extract%20inhibits%20gefitinib%20metabolism%20in%20vitro%20by%20interfering%20with%20human%20hepatic%20CYP3A4%20and%20CYP2D6%20enzymes&author=SY%20Han&author=HY%20Zhao&author=N%20Zhou&author=F%20Zhou&author=PP%20Li&volume=151&issue=1&publication_year=2014&pages=210-217&pmid=24157377&doi=10.1016/j.jep.2013.10.021&)

3. Nurwidya F, Takahashi F, Takahashi K. Gefitinib in the treatment of nonsmall cell lung cancer with activating epidermal growth factor receptor mutation. J Nat Sci Biol Med. 2016;7(2):119–123. doi: 10.4103/0976-9668.184695.  [DOI](https://doi.org/10.4103/0976-9668.184695) | [PMC free article](/articles/PMC4934098/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27433059/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Nat%20Sci%20Biol%20Med&title=Gefitinib%20in%20the%20treatment%20of%20nonsmall%20cell%20lung%20cancer%20with%20activating%20epidermal%20growth%20factor%20receptor%20mutation&author=F%20Nurwidya&author=F%20Takahashi&author=K%20Takahashi&volume=7&issue=2&publication_year=2016&pages=119-123&pmid=27433059&doi=10.4103/0976-9668.184695&)

4. Bocharov EV, Lesovoy DM, Pavlov KV, Pustovalova YE, Bocharova OV, Arseniev AS. Alternative packing of EGFR transmembrane domain suggests that protein-lipid interactions underlie signal conduction across membrane. Biochim Biophys Acta. 2016;1858(6):1254–1261. doi: 10.1016/j.bbamem.2016.02.023.  [DOI](https://doi.org/10.1016/j.bbamem.2016.02.023) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26903218/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochim%20Biophys%20Acta&title=Alternative%20packing%20of%20EGFR%20transmembrane%20domain%20suggests%20that%20protein-lipid%20interactions%20underlie%20signal%20conduction%20across%20membrane&author=EV%20Bocharov&author=DM%20Lesovoy&author=KV%20Pavlov&author=YE%20Pustovalova&author=OV%20Bocharova&volume=1858&issue=6&publication_year=2016&pages=1254-1261&pmid=26903218&doi=10.1016/j.bbamem.2016.02.023&)

5. Bremer E, Samplonius DF, van Genne L, et al. Simultaneous inhibition of epidermal growth factor receptor (EGFR) signaling and enhanced activation of tumor necrosis factor-related apoptosis-inducing ligand (TRAIL) receptor-mediated apoptosis induction by an scFv:sTRAIL fusion protein with specificity for human EGFR. J Biol Chem. 2005;280(11):10025–10033. doi: 10.1074/jbc.M413673200.  [DOI](https://doi.org/10.1074/jbc.M413673200) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15644326/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=Simultaneous%20inhibition%20of%20epidermal%20growth%20factor%20receptor%20(EGFR)%20signaling%20and%20enhanced%20activation%20of%20tumor%20necrosis%20factor-related%20apoptosis-inducing%20ligand%20(TRAIL)%20receptor-mediated%20apoptosis%20induction%20by%20an%20scFv:sTRAIL%20fusion%20protein%20with%20specificity%20for%20human%20EGFR&author=E%20Bremer&author=DF%20Samplonius&author=L%20van%20Genne&volume=280&issue=11&publication_year=2005&pages=10025-10033&pmid=15644326&doi=10.1074/jbc.M413673200&)

6. Yu X, Sharma KD, Takahashi T, Iwamoto R, Mekada E. Ligand-independent dimer formation of epidermal growth factor receptor (EGFR) is a step separable from ligand-induced EGFR signaling. Mol Biol Cell. 2002;13(7):2547–2557. doi: 10.1091/mbc.01-08-0411.  [DOI](https://doi.org/10.1091/mbc.01-08-0411) | [PMC free article](/articles/PMC117333/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12134089/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Biol%20Cell&title=Ligand-independent%20dimer%20formation%20of%20epidermal%20growth%20factor%20receptor%20(EGFR)%20is%20a%20step%20separable%20from%20ligand-induced%20EGFR%20signaling&author=X%20Yu&author=KD%20Sharma&author=T%20Takahashi&author=R%20Iwamoto&author=E%20Mekada&volume=13&issue=7&publication_year=2002&pages=2547-2557&pmid=12134089&doi=10.1091/mbc.01-08-0411&)

7. Zhang W, Stabile LP, Keohavong P, et al. Mutation and polymorphism in the EGFR-TK domain associated with lung cancer. J Thorac Oncol. 2006;1(7):635–647.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/17409930/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Thorac%20Oncol&title=Mutation%20and%20polymorphism%20in%20the%20EGFR-TK%20domain%20associated%20with%20lung%20cancer&author=W%20Zhang&author=LP%20Stabile&author=P%20Keohavong&volume=1&issue=7&publication_year=2006&pages=635-647&pmid=17409930&)

8. Ettinger DS. Clinical implications of EGFR expression in the development and progression of solid tumors: focus on non-small cell lung cancer. Oncologist. 2006;11(4):358–373. doi: 10.1634/theoncologist.11-4-358.  [DOI](https://doi.org/10.1634/theoncologist.11-4-358) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16614231/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncologist&title=Clinical%20implications%20of%20EGFR%20expression%20in%20the%20development%20and%20progression%20of%20solid%20tumors:%20focus%20on%20non-small%20cell%20lung%20cancer&author=DS%20Ettinger&volume=11&issue=4&publication_year=2006&pages=358-373&pmid=16614231&doi=10.1634/theoncologist.11-4-358&)

9. Kim CH, Kim SH, Park SY, Yoo J, Kim SK, Kim HK. Identification of EGFR Mutations by Immunohistochemistry with EGFR Mutation-Specific Antibodies in Biopsy and Resection Specimens from Pulmonary Adenocarcinoma. Cancer Res Treat. 2015;47(4):653–660. doi: 10.4143/crt.2014.118.  [DOI](https://doi.org/10.4143/crt.2014.118) | [PMC free article](/articles/PMC4614184/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25687872/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res%20Treat&title=Identification%20of%20EGFR%20Mutations%20by%20Immunohistochemistry%20with%20EGFR%20Mutation-Specific%20Antibodies%20in%20Biopsy%20and%20Resection%20Specimens%20from%20Pulmonary%20Adenocarcinoma&author=CH%20Kim&author=SH%20Kim&author=SY%20Park&author=J%20Yoo&author=SK%20Kim&volume=47&issue=4&publication_year=2015&pages=653-660&pmid=25687872&doi=10.4143/crt.2014.118&)

10. Hsiue EH, Lee JH, Lin CC, Yang JC. Safety of gefitinib in non-small cell lung cancer treatment. Expert Opin Drug Saf. 2016;15(7):993–1000. doi: 10.1080/14740338.2016.1192605.  [DOI](https://doi.org/10.1080/14740338.2016.1192605) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27212579/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Expert%20Opin%20Drug%20Saf&title=Safety%20of%20gefitinib%20in%20non-small%20cell%20lung%20cancer%20treatment&author=EH%20Hsiue&author=JH%20Lee&author=CC%20Lin&author=JC%20Yang&volume=15&issue=7&publication_year=2016&pages=993-1000&pmid=27212579&doi=10.1080/14740338.2016.1192605&)

11. Cohen MH, Williams GA, Sridhara R, et al. United States Food and Drug Administration Drug Approval summary: Gefitinib (ZD1839; Iressa) tablets. Clin Cancer Res. 2004;10(4):1212–1218. doi: 10.1158/1078-0432.ccr-03-0564.  [DOI](https://doi.org/10.1158/1078-0432.ccr-03-0564) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14977817/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&title=United%20States%20Food%20and%20Drug%20Administration%20Drug%20Approval%20summary:%20Gefitinib%20(ZD1839;%20Iressa)%20tablets&author=MH%20Cohen&author=GA%20Williams&author=R%20Sridhara&volume=10&issue=4&publication_year=2004&pages=1212-1218&pmid=14977817&doi=10.1158/1078-0432.ccr-03-0564&)

12. Prommer E. Gefitinib: a new agent in palliative care. Am J Hosp Palliati Care. 2004;21(3):222–227. doi: 10.1177/104990910402100313.  [DOI](https://doi.org/10.1177/104990910402100313) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15188923/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hosp%20Palliati%20Care&title=Gefitinib:%20a%20new%20agent%20in%20palliative%20care&author=E%20Prommer&volume=21&issue=3&publication_year=2004&pages=222-227&pmid=15188923&doi=10.1177/104990910402100313&)

13. McKillop D, McCormick AD, Millar A, Miles GS, Phillips PJ, Hutchison M. Cytochrome P450-dependent metabolism of gefitinib. Xenobiotica. 2005;35(1):39–50. doi: 10.1080/00498250400026464.  [DOI](https://doi.org/10.1080/00498250400026464) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15788367/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Xenobiotica&title=Cytochrome%20P450-dependent%20metabolism%20of%20gefitinib&author=D%20McKillop&author=AD%20McCormick&author=A%20Millar&author=GS%20Miles&author=PJ%20Phillips&volume=35&issue=1&publication_year=2005&pages=39-50&pmid=15788367&doi=10.1080/00498250400026464&)

14. Kobayashi H, Sato K, Niioka T, et al. Effects of polymorphisms in CYP2D6 and ABC transporters and side effects induced by gefitinib on the pharmacokinetics of the gefitinib metabolite, O-desmethyl gefitinib. Med Oncol. 2016;33(6):57. doi: 10.1007/s12032-016-0773-5.  [DOI](https://doi.org/10.1007/s12032-016-0773-5) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27154635/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Med%20Oncol&title=Effects%20of%20polymorphisms%20in%20CYP2D6%20and%20ABC%20transporters%20and%20side%20effects%20induced%20by%20gefitinib%20on%20the%20pharmacokinetics%20of%20the%20gefitinib%20metabolite,%20O-desmethyl%20gefitinib&author=H%20Kobayashi&author=K%20Sato&author=T%20Niioka&volume=33&issue=6&publication_year=2016&pages=57&pmid=27154635&doi=10.1007/s12032-016-0773-5&)

15. McKillop D, Guy SP, Spence MP, et al. Minimal contribution of desmethyl-gefitinib, the major human plasma metabolite of gefitinib, to epidermal growth factor receptor (EGFR)-mediated tumour growth inhibition. Xenobiotica. 2006;36(1):29–39. doi: 10.1080/00498250500523253.  [DOI](https://doi.org/10.1080/00498250500523253) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16507511/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Xenobiotica&title=Minimal%20contribution%20of%20desmethyl-gefitinib,%20the%20major%20human%20plasma%20metabolite%20of%20gefitinib,%20to%20epidermal%20growth%20factor%20receptor%20(EGFR)-mediated%20tumour%20growth%20inhibition&author=D%20McKillop&author=SP%20Guy&author=MP%20Spence&volume=36&issue=1&publication_year=2006&pages=29-39&pmid=16507511&doi=10.1080/00498250500523253&)

16. Takimoto T, Kijima T, Otani Y, et al. Polymorphisms of CYP2D6 gene and gefitinib-induced hepatotoxicity. Clin Lung Cancer. 2013;14(5):502–507. doi: 10.1016/j.cllc.2013.03.003.  [DOI](https://doi.org/10.1016/j.cllc.2013.03.003) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23664723/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Lung%20Cancer&title=Polymorphisms%20of%20CYP2D6%20gene%20and%20gefitinib-induced%20hepatotoxicity&author=T%20Takimoto&author=T%20Kijima&author=Y%20Otani&volume=14&issue=5&publication_year=2013&pages=502-507&pmid=23664723&doi=10.1016/j.cllc.2013.03.003&)

17. Zanger UM, Raimundo S, Eichelbaum M. Cytochrome P450 2D6: overview and update on pharmacology, genetics, biochemistry. Naunyn Schmiedebergs Arch Pharmacol. 2004;369(1):23–37. doi: 10.1007/s00210-003-0832-2.  [DOI](https://doi.org/10.1007/s00210-003-0832-2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14618296/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Naunyn%20Schmiedebergs%20Arch%20Pharmacol&title=Cytochrome%20P450%202D6:%20overview%20and%20update%20on%20pharmacology,%20genetics,%20biochemistry&author=UM%20Zanger&author=S%20Raimundo&author=M%20Eichelbaum&volume=369&issue=1&publication_year=2004&pages=23-37&pmid=14618296&doi=10.1007/s00210-003-0832-2&)

18. Zhou SF. Polymorphism of human cytochrome P450 2D6 and its clinical significance: part I. Clin Pharmacokinet. 2009;48(11):689–723. doi: 10.2165/11318030-000000000-00000.  [DOI](https://doi.org/10.2165/11318030-000000000-00000) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19817501/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacokinet&title=Polymorphism%20of%20human%20cytochrome%20P450%202D6%20and%20its%20clinical%20significance:%20part%20I&author=SF%20Zhou&volume=48&issue=11&publication_year=2009&pages=689-723&pmid=19817501&doi=10.2165/11318030-000000000-00000&)

19. Liang B, Zhan Y, Huang X, et al. Effect of 22 novel cytochrome P450 2D6 (CYP2D6) variants found in the Chinese Population on Hemangeol metabolism in vitro. Eur J Drug Metab Pharmacokinet. 2016;41(6):759–765. doi: 10.1007/s13318-015-0307-0.  [DOI](https://doi.org/10.1007/s13318-015-0307-0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26542054/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Drug%20Metab%20Pharmacokinet&title=Effect%20of%2022%20novel%20cytochrome%20P450%202D6%20(CYP2D6)%20variants%20found%20in%20the%20Chinese%20Population%20on%20Hemangeol%20metabolism%20in%20vitro&author=B%20Liang&author=Y%20Zhan&author=X%20Huang&volume=41&issue=6&publication_year=2016&pages=759-765&pmid=26542054&doi=10.1007/s13318-015-0307-0&)

20. Qian JC, Xu XM, Hu GX, et al. Genetic variations of human CYP2D6 in the Chinese Han population. Pharmacogenomics. 2013;14(14):1731–1743. doi: 10.2217/pgs.13.160.  [DOI](https://doi.org/10.2217/pgs.13.160) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24192122/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Genetic%20variations%20of%20human%20CYP2D6%20in%20the%20Chinese%20Han%20population&author=JC%20Qian&author=XM%20Xu&author=GX%20Hu&volume=14&issue=14&publication_year=2013&pages=1731-1743&pmid=24192122&doi=10.2217/pgs.13.160&)

21. Cai J, Dai DP, Geng PW, et al. Effects of 22 novel CYP2D6 variants found in the Chinese Population on the Bufuralol and Dextromethorphan metabolisms in vitro. Basic Clin Pharmacol Toxicol. 2016;118(3):190–199. doi: 10.1111/bcpt.12478.  [DOI](https://doi.org/10.1111/bcpt.12478) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26310775/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Basic%20Clin%20Pharmacol%20Toxicol&title=Effects%20of%2022%20novel%20CYP2D6%20variants%20found%20in%20the%20Chinese%20Population%20on%20the%20Bufuralol%20and%20Dextromethorphan%20metabolisms%20in%20vitro&author=J%20Cai&author=DP%20Dai&author=PW%20Geng&volume=118&issue=3&publication_year=2016&pages=190-199&pmid=26310775&doi=10.1111/bcpt.12478&)

22. Liang B, Zhan Y, Wang Y, et al. Effect of 24 cytochrome P450 2D6 variants found in the Chinese Population on Atomoxetine metabolism in vitro. Pharmacology. 2016;97(1–2):78–83. doi: 10.1159/000442952.  [DOI](https://doi.org/10.1159/000442952) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26666748/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacology&title=Effect%20of%2024%20cytochrome%20P450%202D6%20variants%20found%20in%20the%20Chinese%20Population%20on%20Atomoxetine%20metabolism%20in%20vitro&author=B%20Liang&author=Y%20Zhan&author=Y%20Wang&volume=97&issue=1%E2%80%932&publication_year=2016&pages=78-83&pmid=26666748&doi=10.1159/000442952&)

23. Zhan YY, Liang BQ, Wang H, et al. Effect of CYP2D6 variants on venlafaxine metabolism in vitro. Xenobiotica. 2016;46(5):424–429. doi: 10.3109/00498254.2015.1089364.  [DOI](https://doi.org/10.3109/00498254.2015.1089364) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26406933/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Xenobiotica&title=Effect%20of%20CYP2D6%20variants%20on%20venlafaxine%20metabolism%20in%20vitro&author=YY%20Zhan&author=BQ%20Liang&author=H%20Wang&volume=46&issue=5&publication_year=2016&pages=424-429&pmid=26406933&doi=10.3109/00498254.2015.1089364&)

24. Hu X, Lan T, Dai D, et al. Evaluation of 24 CYP2D6 variants on the metabolism of nebivolol in vitro. Drug Metab Dispos. 2016;44(11):1828–1831. doi: 10.1124/dmd.116.071811.  [DOI](https://doi.org/10.1124/dmd.116.071811) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27538917/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Evaluation%20of%2024%20CYP2D6%20variants%20on%20the%20metabolism%20of%20nebivolol%20in%20vitro&author=X%20Hu&author=T%20Lan&author=D%20Dai&volume=44&issue=11&publication_year=2016&pages=1828-1831&pmid=27538917&doi=10.1124/dmd.116.071811&)

25. Byeon JY, Kim YH, Na HS, et al. Effects of the CYP2D6*10 allele on the pharmacokinetics of atomoxetine and its metabolites. Arch Pharm Res. 2015;38(11):2083–2091. doi: 10.1007/s12272-015-0646-z.  [DOI](https://doi.org/10.1007/s12272-015-0646-z) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26254792/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arch%20Pharm%20Res&title=Effects%20of%20the%20CYP2D6*10%20allele%20on%20the%20pharmacokinetics%20of%20atomoxetine%20and%20its%20metabolites&author=JY%20Byeon&author=YH%20Kim&author=HS%20Na&volume=38&issue=11&publication_year=2015&pages=2083-2091&pmid=26254792&doi=10.1007/s12272-015-0646-z&)

26. Shen H, He MM, Liu H, et al. Comparative metabolic capabilities and inhibitory profiles of CYP2D6.1, CYP2D6.10, and CYP2D6.17. Drug Metab Dispos. 2007;35(8):1292–1300. doi: 10.1124/dmd.107.015354.  [DOI](https://doi.org/10.1124/dmd.107.015354) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17470523/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Comparative%20metabolic%20capabilities%20and%20inhibitory%20profiles%20of%20CYP2D6.1,%20CYP2D6.10,%20and%20CYP2D6.17&author=H%20Shen&author=MM%20He&author=H%20Liu&volume=35&issue=8&publication_year=2007&pages=1292-1300&pmid=17470523&doi=10.1124/dmd.107.015354&)

27. Dai DP, Geng PW, Wang SH, et al. In vitro functional assessment of 22 newly identified CYP2D6 allelic variants in the Chinese population. Basic Clin Pharmacol Toxicol. 2015;117(1):39–43. doi: 10.1111/bcpt.12363.  [DOI](https://doi.org/10.1111/bcpt.12363) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25469868/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Basic%20Clin%20Pharmacol%20Toxicol&title=In%20vitro%20functional%20assessment%20of%2022%20newly%20identified%20CYP2D6%20allelic%20variants%20in%20the%20Chinese%20population&author=DP%20Dai&author=PW%20Geng&author=SH%20Wang&volume=117&issue=1&publication_year=2015&pages=39-43&pmid=25469868&doi=10.1111/bcpt.12363&)

28. Kijima T, Shimizu T, Nonen S, et al. Safe and successful treatment with erlotinib after gefitinib-induced hepatotoxicity: difference in metabolism as a possible mechanism. J Clin Oncol. 2011;29(19):e588–e590. doi: 10.1200/JCO.2010.34.3368.  [DOI](https://doi.org/10.1200/JCO.2010.34.3368) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21502555/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Safe%20and%20successful%20treatment%20with%20erlotinib%20after%20gefitinib-induced%20hepatotoxicity:%20difference%20in%20metabolism%20as%20a%20possible%20mechanism&author=T%20Kijima&author=T%20Shimizu&author=S%20Nonen&volume=29&issue=19&publication_year=2011&pages=e588-e590&pmid=21502555&doi=10.1200/JCO.2010.34.3368&)

29. Swaisland HC, Ranson M, Smith RP, et al. Pharmacokinetic drug interactions of gefitinib with rifampicin, itraconazole and metoprolol. Clin Pharmacokinet. 2005;44(10):1067–1081. doi: 10.2165/00003088-200544100-00005.  [DOI](https://doi.org/10.2165/00003088-200544100-00005) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16176119/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacokinet&title=Pharmacokinetic%20drug%20interactions%20of%20gefitinib%20with%20rifampicin,%20itraconazole%20and%20metoprolol&author=HC%20Swaisland&author=M%20Ranson&author=RP%20Smith&volume=44&issue=10&publication_year=2005&pages=1067-1081&pmid=16176119&doi=10.2165/00003088-200544100-00005&)
