# Cytochrome P450 Genetic Variation Associated with Tamoxifen Biotransformation in American Indian and Alaska Native People

## Metadata
**Authors:** Burhan A Khan, Renee Robinson, Alison E Fohner, LeeAnna I Muzquiz, Brian D Schilling, Julie A Beans, Matthew J Olnes, Laura Trawicki, Holly Frydenlund, Cindi Laukes, Patrick Beatty, Brian Phillips, Deborah Nickerson, Kevin Howlett, Denise A Dillard, Timothy A Thornton, Kenneth E Thummel, Erica L Woodahl
**Journal:** Clinical and Translational Science
**Date:** 2018 Feb 13
**DOI:** [10.1111/cts.12542](https://doi.org/10.1111/cts.12542)
**PMID:** 29436156
**PMCID:** PMC5944577
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5944577/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC5944577/pdf/CTS-11-312.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC5944577/pdf/CTS-11-312.pdf)

## Abstract

Despite evidence that pharmacogenetics can improve tamoxifen pharmacotherapy, there are few studies with American Indian and Alaska Native (AIAN) people. We examined variation in cytochrome P450 (CYP) genes (CYP2D6, CYP3A4, CYP3A5, and CYP2C9) and tamoxifen biotransformation in AIAN patients with breast cancer (n = 42) from the Southcentral Foundation in Alaska and the Confederated Salish and Kootenai Tribes in Montana. We tested for associations between CYP diplotypes and plasma concentrations of tamoxifen and metabolites. Only the CYP2D6 variation was significantly associated with concentrations of endoxifen (P = 0.0008) and 4‐hydroxytamoxifen (P = 0.0074), tamoxifen's principal active metabolites, as well as key metabolic ratios. The CYP2D6 was also the most significant predictor of active metabolites and metabolic ratios in a multivariate regression model, including all four genes as predictors, with minor roles for other CYP genes. In AIAN populations, CYP2D6 is the largest contributor to tamoxifen bioactivation, illustrating the importance of validating pharmacogenetic testing for therapy optimization in an understudied population.

## Study Highlights

## Study Highlights

**WHAT IS THE CURRENT KNOWLEDGE ON THE TOPIC?**

✓ Tamoxifen has been used to reduce the risk of recurrence of estrogen‐positive breast cancer for the past 40 years. Tamoxifen undergoes sequential bioactivation to endoxifen, a metabolite that contributes importantly to its anti‐estrogenic effect. Previous research has shown that the cytochrome P450 CYP2D6 drug‐metabolizing enzyme is critical in tamoxifen bioactivation.

**WHAT QUESTION DID THIS STUDY ADDRESS?**

✓ This tamoxifen pharmacogenetic study asks whether previous results showing a strong association between *CYP2D6* diplotypes and circulating endoxifen concentrations are applicable to diverse populations.

**WHAT DOES THIS STUDY ADD TO OUR KNOWLEDGE?**

✓ Our study is the first to focus on bioactivation of tamoxifen with patients with breast cancer from AIAN populations. This study highlights the key role of *CYP2D6*‐mediated metabolism in formation of the principal activity metabolite, endoxifen.

**HOW THIS MIGHT CHANGE DRUG CLINICAL PHARMACOLOGY OR TRANSLATIONAL SCIENCE**

✓ The research contributes to the discourse regarding pharmacogenetic testing for tamoxifen therapy in AIAN women, an under‐represented group who bear a disproportionally large burden of breast cancer.

## METHODS

### Settings

The Southcentral Foundation (SCF), a tribal‐owned and operated regional health corporation, provides primary health care to 65,000 AIAN people, considered “customer‐owners,” at clinics on the Alaska Native Medical Center (ANMC) campus. The ANMC is co‐owned and managed by the SCF and the Alaska Native Tribal Health Consortium (ANTHC). Patients receiving tamoxifen therapy were managed by oncologists and support staff at the Oncology and Hematology Clinic at the ANMC in Anchorage, Alaska.

The University of Montana and the Confederated Salish and Kootenai Tribes (CSKT) Tribal Health Department (THD) partnered with oncologists and support staff at the Montana Cancer Institute Foundation to identify CSKT patients receiving tamoxifen therapy. The CSKT THD manages seven clinics on the Flathead Indian Reservation in northwestern Montana, providing health care to 8,000 enrolled CSKT members and a large number of descendants.

### Study participants

The *CYP2D6* genetic variation was characterized in a convenience sample of 380 SCF participants. Complete resequencing of *CYP2D6* was performed in 94 participants from this cohort and all 380 samples were genotyped for specific *CYP2D6* single nucleotide variants (SNVs), as described below. A similarly designed *CYP2D6* resequencing study of 187 CSKT participants was published previously.[3](#cts12542-bib-0003)

Participants for the tamoxifen pharmacogenetic study were recruited from SCF and CSKT populations. AIAN women ≥18 years old who were receiving tamoxifen for breast cancer therapy were eligible to participate. Participants completed a survey of demographic information (e.g., gender, date of birth, and tribal affiliation/heritage) and medication information (e.g., tamoxifen dose and concurrent medication usage). Participants provided blood samples for DNA isolation and quantitation of plasma levels of tamoxifen and metabolites. The SCF (*n* = 38) and CSKT (*n* = 4) participants were recruited by research staff members and oncology clinic staff at the ANMC Oncology and Hematology Clinic and the Montana Cancer Institute Foundation Clinic, respectively.

The Alaska Area Institutional Review Board and SCF and ANTHC tribal review boards approved research protocols and forms for written consent prior to initiating research in Alaska. The CSKT Tribal Council, the CSKT THD, and the University of Montana Institutional Review Board approved research conducted in Montana.

### DNA isolation

Genomic DNA was isolated from buffy coat (SCF) or whole blood (CSKT) using QIAamp DNA Blood kits (Qiagen, Valencia, CA). The DNA concentration and quality were determined using a NanoDrop spectrophotometer (Thermo Fisher Scientific, Wilmington, DE).

### Characterization of novel and known variants in CYP2D6 in the SCF population

Complete *CYP2D6* resequencing was performed in a cohort of 94 SCF participants using a method reported previously.[3](#cts12542-bib-0003) Common haplotypes were identified using fastPHASE version 2.0 (University of Chicago, Chicago, IL). The *CYP2D6* allele assignments were made using the Pharmacogene Variation Consortium[16](#cts12542-bib-0016) with [M33388](https://www.ncbi.nlm.nih.gov/nuccore/M33388) as the human reference sequence.

The *CYP2D6* genotyping was performed in a cohort of 380 SCF participants using TaqMan SNP Genotyping Assays (Applied Biosystems, Hercules, CA) on a Fluidigm genotyping platform (Fluidigm, South San Francisco, CA), as reported previously,[4](#cts12542-bib-0004) to confirm allele frequencies in a larger cohort. The genotyping included seven *CYP2D6* SNVs. Seventeen samples were excluded from analysis because the call rates were below 95%.

### CYP genotyping for tamoxifen pharmacogenetic study

Results from *CYP2D6* resequencing, as well as resequencing of other CYPs previously published in CSKT and SCF populations,[3](#cts12542-bib-0003), [4](#cts12542-bib-0004) informed the selection of SNVs included in the genotyping for the tamoxifen pharmacogenetic study. The genotyping included 10 *CYP2D6*, 8 *CYP3A4*, 9 *CYP3A5*, and 9 *CYP2C9* SNVs. The Fluidigm genotyping platform was used as reported previously.[4](#cts12542-bib-0004) The *CYP2D6*5* gene deletion was determine using long‐range polymerase chain reaction with the Expand Long Template PCR System (Roche, Basel, Switzerland), as described previously.[17](#cts12542-bib-0017) All samples had call rates above 95%. The *CYP2D6* diplotypes were translated into activity scores as previously described.[18](#cts12542-bib-0018) These activity scores correspond to those utilized in the Clinical Pharmacogenetics Implementation Consortium (CPIC) gene/drug dosing guidelines for *CYP2D6*.[19](#cts12542-bib-0019), [20](#cts12542-bib-0020), [21](#cts12542-bib-0021), [22](#cts12542-bib-0022), [23](#cts12542-bib-0023), [24](#cts12542-bib-0024)

### Quantification of tamoxifen and metabolites

Steady‐state plasma concentrations of tamoxifen and metabolites (*N*‐dm‐Tam, 4‐OH‐Tam, and endoxifen) were measured using a high‐performance liquid chromatography‐mass spectrometry analytical method. Plasma samples were mixed with internal standards (1 ng/μL tamoxifen‐d5, 1 ng/μL *N*‐dm‐Tam‐d5, 0.5 ng/μl endoxifen‐d5, and 0.025 ng/μl 4‐OH‐Tam‐d5 in methanol) and proteins were precipitated with acetonitrile. After centrifugation, the supernatant was injected on the high‐performance liquid chromatography‐mass spectrometry (liquid chromatography: 1200 series, mass spectrometry: G1956B; Agilent Technologies, Santa Clara, CA) on an Agilent Zorbax SB‐C8 HPLC column (2.1 mm × 150 mm × 3.5 μm) at a flow rate of 0.3 mL/min and column temperature of 50°C. The mobile phase consisted of 10 mM ammonium formate pH = 4 (A) and acetonitrile (B) according to the following gradient: initial 55% A and 45% B for 1 min, 70% B at 5 min, hold 70% B until 5.5 min, and return to 45% B at 5.6 min. The total run time was 10 min. The MS detector was operated in the ESI+ mode. Nitrogen was used for the nebulizer and drying gases (35 psi and 10 L/min, respectively), with drying gas temperature at 350°C. Fragmentor voltage was 120 volts and the capillary voltage was 1,400 volts. Standard curves for each analyte were prepared by spiking blank plasma with calibration standards. Control samples were prepared in batches by spiking blank plasma with calibration standards. Limits of detection (LOD) for tamoxifen, *N*‐dm‐Tam, endoxifen, and 4‐OH‐Tam were 12.5 ng/mL, 12.5 ng/mL, 0.125 ng/mL, and 0.625 ng/mL, respectively; and limits of quantitation were 25 ng/mL, 25 ng/mL, 0.25 ng/mL, and 1.25 ng/mL, respectively. Analyte concentrations below the LOD (6.5% of total analytes) were assigned a value half of the LOD.

### Statistical analysis

All SNVs identified in the resequencing (*n* = 94), genotyping (*n* = 380), and tamoxifen (*n* = 42) cohorts were tested for deviations from Hardy‐Weinberg equilibrium (HWE) using a χ^2^ test. Pairwise linkage disequilibrium (LD) was also estimated in these three cohorts. The r^2^ LD value was determined between every SNV and calculated using the Haploview version 4.2 software.[25](#cts12542-bib-0025)

In the tamoxifen pharmacogenetic study, we characterized associations between *CYP* genetic variation and tamoxifen metabolism. Multivariate linear regression analyses were used to assess associations between genetic variation and plasma concentrations of tamoxifen and its principal metabolites (*N*‐dm‐Tam, 4‐OH‐Tam, and endoxifen), as well as key metabolic ratios used to understand metabolite formation and elimination (endoxifen/tamoxifen, 4‐OH‐Tam/tamoxifen, *N*‐dm‐Tam/tamoxifen, endoxifen/4‐OH‐Tam, and endoxifen/*N*‐dm‐Tam). In the primary analysis, we adjusted for age and recruitment site in the regression models. Weight was not adjusted for because it was not correlated with tamoxifen metabolism. Concomitant medication use was not adjusted for because only one patient reported usage of a medication interacting with CYP metabolism, a selective serotonin reuptake inhibitor (SSRI). Genotype‐phenotype associations were tested with linear regression analyses using RStudio version 3.3.1 (RStudio, Boston, MA). The *CYP2D6* diplotypes were coded as activity scores ranging from 0 to 2 as described previously[18](#cts12542-bib-0018); *CYP3A4*, *CYP3A5*, and *CYP2C9* diplotypes were coded as having either 0, 1, or 2 copies of their respective reference (**1*) allele. To allow for heterogeneous variances of tamoxifen and its metabolites among the diplotype groups, robust standard errors were used to test for significant genotype‐phenotype associations with the multivariate regression models.[26](#cts12542-bib-0026) To account for multiple testing across the four individual genes and nine biomarkers (36 tests), a *P* value of 0.00139 was considered significant after Bonferroni correction (0.05/36). Subsequently, a regression analysis was conducted to assess the combined effects of *CYP2D6*, *CYP3A4*, *CYP3A5*, and *CYP2C9* on the concentrations of tamoxifen, metabolites, and metabolic ratios. In this analysis, all four genes were included as predictors in the multivariate regression model together, along with age and recruitment site.

## RESULTS

### CYP2D6 resequencing and genotyping

To characterize *CYP2D6* variation in the SCF population, we first resequenced *CYP2D6* in 94 participants and then confirmed allele frequencies of the seven most common SNVs in a larger cohort (*n* = 380). Our group previously characterized *CYP2D6* variation in the CSKT population.[3](#cts12542-bib-0003) The *CYP2D6* resequencing identified a total of 54 variants (**Supplementary** [Table S1](#cts12542-supl-0002)). Seven novel variants were identified in *CYP2D6* at low frequencies (<2%), denoted as variants without a reference identification number “rsNA,” with the exception of an intronic 254G>T SNV (7.23%). Some SNVs in the resequencing analysis deviated from HWE (**Supplementary** [Table S1](#cts12542-supl-0002)); however, because the sequencing was used for the identification of SNVs for the tamoxifen study and not in the statistical analyses, it is not of concern. All of the SNVs in the genotyping analysis were tested for deviations from HWE; only rs28371725, the predictive SNV for *CYP2D6*41*, deviated mildly from HWE (χ^2^ = 5.31). This deviation is likely due to low frequency of the homozygous variant diplotype.

The *CYP2D6* allele frequencies for the SCF population were compared with our previously published frequencies in the CSKT population[3](#cts12542-bib-0003) **(Table** [1](#cts12542-tbl-0001) **)**. The *CYP2D6*1* and *CYP2D6*2* were the most common alleles identified in the SCF population at 45.21 and 26.60%, respectively. Both of these alleles confer normal CYP2D6 activity as does the *CYP2D6*35* identified at 5.32%. Decreased activity alleles, *CYP2D6*10* and *CYP2D6*41*, were both identified at 4.26%. The *CYP2D6*4* was the only null, nonfunctional allele identified in the SCF population with a frequency of 14.36%. The *CYP2D6*3*, **9*, **28*, and **33* were not identified. Frequency of the *CYP2D6*5* deletion variant was not determined in this discovery study. Subtle differences were observed in *CYP2D6* allele frequencies between the SCF and CSKT populations. The most common intermediate function allele was *CYP2D6*10* in the SCF population and *CYP2D6*41* in the CSKT population; collectively, intermediate function alleles were less frequent in the SCF (8.52%) than the CSKT (12.57%). The nonfunctional *CYP2D6*4* allele was common in both populations, but found less frequently in the SCF (14.36%) than the CSKT (20.86%).

### Table 1.

|   |   |   |   | Allele frequency, % |  |
| --- | --- | --- | --- | --- | --- |
| Allele a | Nucleotide changes | Protein effect | Activity | SCF | CSKT b |
| CYP2D6*1 | Wild‐type | None | Normal | 45.21 | 37.57 |
| CYP2D6*2 | 2850C>T; 4180G>C | R296C; S486T | Normal | 26.60 | 23.40 |
| CYP2D6*3 | 2549delA | 259 frameshift | No activity | 0 | 0.27 |
| CYP2D6*4 | 100C>T; 1846G>A; 4180G>C | P34S; splicing defect; S486T | No activity | 14.36 | 20.86 |
| CYP2D6*5 | Gene deletion | Gene deletion | No activity | NDc | 1.34 |
| CYP2D6*9 | 2615delAAG | K281del | Reduced activity | 0 | 0.80 |
| CYP2D6*10 | 100C>T; 4180G>C | P34S; S486T | Reduced activity | 4.26 | 1.34 |
| CYP2D6*28 | 19G>A; 1704C>G; 2850C>T; 4180G>C | V7M; Q151E; R296C; S486T | ND | 0 | 0.27 |
| CYP2D6*33 | 2483G>T | A237S | Normal | 0 | 0.53 |
| CYP2D6*35 | 31G>A; 2850C>T; 4180G>C | M11V; R296C; S486T | Normal | 5.32 | 1.07 |
| CYP2D6*41 | 2850C>T; 2988G>A; 4180G>C | R296C; splicing defect; S486T | Reduced activity | 4.26 | 11.23 |

Table 1 Caption: CYP2D6 allele frequencies by resequencing methods in the SCF population (n = 188 chromosomes) compared with frequencies in the CSKT populations (n = 374 chromosomes)

After resequencing *CYP2D6* in the SCF population, we genotyped select variants in a larger cohort of 380 SCF participants to confirm the allele frequencies (**Supplementary** [Table S2](#cts12542-supl-0003)). No meaningful differences were found in the *CYP2D6* allele frequencies between the resequencing and genotyping methods. Frequency of the *CYP2D6*5* deletion variant was also not determined in this cohort.

Of the 54 SNVs identified through *CYP2D6* resequencing (*n* = 94), the LD pattern showed several variants are inherited in blocks (**Supplementary** [Figure S1](#cts12542-supl-0001)). In the larger genotyping cohort (*n* = 380), rs1065852 (defining *CYP2D6*10*) and rs3892097 (defining *CYP2D6*4*) were found in high LD (r^2^ = 0.79).

### Tamoxifen patient characteristics and diplotype frequencies

Demographics of participants in the tamoxifen pharmacogenetic study and diplotype frequencies for *CYP2D6*, *CYP3A4*, *CYP3A5*, and *CYP2C9* are presented in **Table** [2](#cts12542-tbl-0002). Activity scores were assigned to *CYP2D6* diplotypes to designate predicted activity levels.[18](#cts12542-bib-0018) All alleles were in HWE.

### Table 2.

| Characteristic | No. of patients (%) | Activity score |
| --- | --- | --- |
| Age, median (range, years) | 51 (26–100) |   |
| Recruitment site |  |  |
| SCF | 38 (90.5) |   |
| CSKT | 4 (9.5) |   |
| Ethnicity |  |  |
| AI | 6 (14.3) |   |
| AN | 36 (85.7) |   |
| Diplotypes |   |   |
| CYP2D6 |  |  |
| *1/*1 | 29 (69.05) | 2 |
| *1/*10 | 1 (2.38) | 1.5 |
| *1/*41 | 2 (4.76) | 1.5 |
| *1/*4 | 4 (9.52) | 1 |
| *1/*5 | 3 (7.14) | 1 |
| *4/*41 | 1 (2.38) | 0.5 |
| *4/*5 | 2 (4.76) | 0 |
| CYP3A4 |  |  |
| *1/*1 | 33 (78.57) |   |
| *1/*1G | 6 (14.29) |   |
| *1B/*1G | 1 (2.38) |   |
| *1G/*22 | 1 (2.38) |   |
| *1/*22 | 1 (2.38) |   |
| CYP3A5 |  |  |
| *1/*3 | 7 (16.67) |   |
| *3/*3 | 35 (83.33) |   |
| CYP2C9 |  |  |
| *1/*1 | 36 (85.71) |   |
| *1/*2 | 1 (2.38) |   |
| *1/*3 | 2 (4.76) |   |
| *1/N218I | 2 (4.76) |   |
| *1/M1L | 1 (2.38) |   |

Table 2 Caption: Participant characteristics (n = 42) and CYP diplotype frequencies

### Associations between CYP diplotypes with tamoxifen metabolism

Steady‐state plasma concentrations of tamoxifen and metabolites (4‐OH‐Tam, *N*‐dm‐Tam, and endoxifen) were quantified. The *N*‐dm‐Tam plasma concentrations were nearly twofold higher than tamoxifen concentrations (147 ± 92 vs. 85.3 ± 52.6 ng/mL) and endoxifen plasma concentrations were more than five times higher than 4‐OH‐Tam levels (8.22 ± 5.82 vs. 1.53 ± 0.91 ng/mL). Plasma concentrations of tamoxifen and metabolites are presented as a function of CYP2D6 activity scores in **Table** [3](#cts12542-tbl-0003). We observed differences between the Alaska and Montana sites across some biomarkers; therefore, we adjusted for recruitment site in the regression analyses. The reported *P* values are calculated after adjusting for covariates (recruitment site and age).

### Table 3.

|   | Mean plasma concentrations, ng/mL ± SD |  |  |  |
| --- | --- | --- | --- | --- |
| CYP2D6 activity score | Endoxifen | 4‐OH‐Tam | N‐dm‐Tam | Tamoxifen |
| 0 | 0.80 ± 0.14 | 0.45 ± 0.07 | 87.5 ± 3.5 | 33.0 ± 7.1 |
| 0.5a | 3.10 | 2.00 | 407 | 165 |
| 1 | 6.81 ± 4.45 | 1.47 ± 0.87 | 179 ± 91 | 104 ± 61 |
| 1.5 | 5.73 ± 6.04 | 1.03 ± 0.80 | 104 ± 80 | 54.3 ± 46.8 |
| 2 | 9.50 ± 5.92 | 1.65 ± 0.93 | 139 ± 83 | 84.9 ± 50.0 |

Table 3 Caption: Plasma concentrations of tamoxifen and metabolites in AIAN participants (n = 42)

We first performed regression analyses for each gene individually as a predictor of tamoxifen metabolism **(Table** [4](#cts12542-tbl-0004) **)**. The *CYP2D6* gene was significantly associated with plasma concentrations of endoxifen (*P* = 0.0008) and suggestive of significance for 4‐OH‐Tam (*P* = 0.0074); individuals with higher activity scores had increased circulating endoxifen **(Figure** [2](#cts12542-fig-0002) **a)** and 4‐OH‐Tam **(Figure** [2](#cts12542-fig-0002) **b)** levels. The *CYP2D6* variation was not associated with plasma levels of tamoxifen or *N*‐dm‐Tam. Variants in *CYP3A4*, *CYP3A5*, and *CYP2C9* were not associated with tamoxifen or any of the metabolites, including endoxifen. The *CYP2D6* was significantly associated with the endoxifen/tamoxifen metabolic ratio (*P* = 4.0 × 10^−7^; **Figure** [2](#cts12542-fig-0002) **c**), which describes the exposure of the predominant active metabolite to the parent drug and the contributions of CYP2D6 to the formation of endoxifen (tamoxifen‐to‐4‐OH‐Tam and *N*‐dm‐Tam‐to‐endoxifen). The *CYP2D6* variation was also significantly associated with other metabolic ratios: 4‐OH‐Tam/tamoxifen (*P* = 4.5 × 10^−5^; **Figure** [2](#cts12542-fig-0002) **d**), endoxifen/4‐OH‐Tam (*P* = 0.0004; **Figure** [2](#cts12542-fig-0002) **e**), and endoxifen/*N*‐dm‐Tam (*P* = 1.0 × 10^−8^; **Figure** [2](#cts12542-fig-0002) **f**). There was suggestive evidence that *CYP3A5* was associated with the 4‐OH‐Tam/tamoxifen metabolic ratio (*P* = 0.0096), but this was not considered significant after adjusting for multiple comparisons. Variants in *CYP3A4* and *CYP2C9* were not significantly associated with any metabolic ratios.

### Table 4.

|   | P values a |  |  |  |  |  |  |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
|   | Plasma concentrations, ng/mL |  |  |  | Metabolic ratios |  |  |  |  |
|   | Endoxifen | 4‐OH‐Tam | N‐dm‐Tam | Tamoxifen | Endoxifen / tamoxifen | 4‐OH‐Tam / tamoxifen | N‐dm‐Tam / tamoxifen | Endoxifen / 4‐OH‐Tam | Endoxifen / N‐dm‐Tam |
| CYP2D6 | 0.0008* | 0.0074 | 0.5916 | 0.5731 | 4.0 × 10−7 ** | 4.5 × 10−5 ** | 0.0238 | 0.0004* | 1.0 × 10−8 ** |
| CYP3A4 | 0.6818 | 0.8200 | 0.9651 | 0.3876 | 0.8186 | 0.0444 | 0.0431 | 0.0337 | 0.5772 |
| CYP3A5 | 0.9607 | 0.5061 | 0.9713 | 0.7050 | 0.1803 | 0.0096 | 0.2094 | 0.4233 | 0.7375 |
| CYP2C9 | 0.4771 | 0.1983 | 0.5451 | 0.4447 | 0.6031 | 0.0656 | 0.4608 | 0.0497 | 0.5072 |

Table 4 Caption: Associations between CYP variation and tamoxifen biotransformation in AIAN participants (n = 42)

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5903/5944577/44da88866e65/CTS-11-312-g002.jpg)

The cytochrome P450 (CYP)2D6 activity scores associated with tamoxifen metabolites and metabolic ratios. Activity scores for CYP2D6 were associated with steady‐state plasma concentrations of (a) endoxifen and (b) 4‐hydroxytamoxifen (4‐OH‐Tam). The CYP2D6 activity scores were also associated with metabolic ratios of (c) endoxifen/tamoxifen, (d) 4‐OH‐Tam/tamoxifen, (e) endoxifen/4‐OH‐Tam, and (f) endoxifen/N‐desmethyltamoxifen (N‐dm‐Tam). Genotype‐phenotype associations with P < 0.00139 were considered significant after Bonferroni correction.

In subsequent analysis, we included all genes together as predictors in a multivariate regression model **(Table** [5](#cts12542-tbl-0005) **)**. Significant associations with genetic variation and tamoxifen metabolism were identified with *CYP2D6* acting as the major contributor. The *CYP2D6* variation was significantly associated with plasma endoxifen (*P* = 0.0010) and 4‐OH‐Tam (*P* = 0.0088) levels and with the endoxifen/tamoxifen metabolic ratio (*P* = 4.4 × 10^−7^). The *CYP2D6* variation was also significantly associated with other metabolic ratios: 4‐OH‐Tam/tamoxifen (*P* = 5.3 × 10^−7^), *N*‐dm‐Tam/tamoxifen (*P* = 0.0247), endoxifen/4‐OH‐Tam (*P* = 0.0002), and endoxifen/*N*‐dm‐Tam (*P* = 3.0 × 10^−8^). The *CYP3A4* variation was significantly associated with the endoxifen/4‐OH‐Tam ratio (*P* = 0.0028), whereas *CYP3A5* was significantly associated with both the endoxifen/tamoxifen (*P* = 0.0020) and 4‐OH‐Tam/tamoxifen (*P* = 0.0216) ratios. The *CYP2C9* variation was significantly associated the 4‐OH‐Tam/tamoxifen (*P* = 0.0318) and endoxifen/4‐OH‐Tam ratios (*P* = 0.0034).

### Table 5.

| Dependent | Independent | β‐coefficient ± SE | P value a |
| --- | --- | --- | --- |
| Endoxifen | Intercept | 2.84 ± 6.07 | 0.6422 |
|   | CYP2D6 activity | 3.60 ± 1.00 | 0.0010** |
|   | CYP3A4 activity | ‐2.08 ± 2.39 | 0.3895 |
|   | CYP3A5 activity | ‐3.15 ± 3.17 | 0.3278 |
|   | CYP2C9 activity | 0.60 ± 1.42 | 0.6745 |
|   | Age | 0.05 ± 0.06 | 0.4539 |
|   | Recruitment site | ‐0.74 ± 2.18 | 0.7355 |
| 4‐OH‐Tam | Intercept | 0.57 ± 0.90 | 0.5263 |
|   | CYP2D6 activity | 0.45 ± 0.16 | 0.0088* |
|   | CYP3A4 activity | ‐0.19 ± 0.35 | 0.5879 |
|   | CYP3A5 activity | ‐0.54 ± 0.43 | 0.2246 |
|   | CYP2C9 activity | 0.22 ± 0.21 | 0.2898 |
|   | Age | 2.0 × 10−3 ± 0.01 | 0.8166 |
|   | Recruitment site | 1.07 ± 0.45 | 0.0229* |
| N‐dm‐Tam | Intercept | 133.64 ± 95.66 | 0.1712 |
|   | CYP2D6 activity | ‐9.70 ± 19.03 | 0.6134 |
|   | CYP3A4 activity | ‐5.61 ± 34.83 | 0.8729 |
|   | CYP3A5 activity | ‐6.78 ± 49.26 | 0.8914 |
|   | CYP2C9 activity | ‐13.37 ± 24.57 | 0.5897 |
|   | Age | 0.91 ± 0.85 | 0.2897 |
|   | Recruitment site | 171.41 ± 50.89 | 0.0019* |
| Tamoxifen | Intercept | 81.30 ± 64.40 | 0.2151 |
|   | CYP2D6 activity | 5.08 ± 9.78 | 0.6066 |
|   | CYP3A4 activity | ‐19.84 ± 22.65 | 0.3869 |
|   | CYP3A5 activity | ‐16.67 ± 33.95 | 0.6264 |
|   | CYP2C9 activity | ‐10.87 ± 15.26 | 0.4812 |
|   | Age | 0.83 ± 0.64 | 0.2008 |
|   | Recruitment site | 81.46 ± 28.09 | 0.0064* |
| Endoxifen / tamoxifen | Intercept | 0.08 ± 0.05 | 0.1328 |
|   | CYP2D6 activity | 0.04 ± 0.01 | 4.4 × 10−7 *** |
|   | CYP3A4 activity | ‐0.02 ± 0.01 | 0.0874 |
|   | CYP3A5 activity | ‐0.05 ± 0.01 | 0.0020* |
|   | CYP2C9 activity | 4.8 × 10−3 ± 0.02 | 0.7599 |
|   | Age | ‐3.1 × 10−4 ± 4.3 × 10−4 | 0.4857 |
|   | Recruitment site | ‐0.05 ± 0.01 | 0.0001** |
| 4‐OH‐Tam / tamoxifen | Intercept | 0.01 ± 0.01 | 0.1944 |
|   | CYP2D6 activity | 4.6 × 10−3 ± 7.4 × 10−4 | 5.3 × 10−7 *** |
|   | CYP3A4 activity | 5.7 × 10−4 ± 1.1 × 10−3 | 0.6167 |
|   | CYP3A5 activity | ‐4.5 × 10−3 ± 1.9 × 10−3 | 0.0216* |
|   | CYP2C9 activity | 4.5 × 10‐3 ± 2.0 × 10−3 | 0.0318* |
|   | Age | ‐1.1 × 10−4 ± 6.5 × 10−5 | 0.1010 |
|   | Recruitment site | ‐2.9 × 10−3 ± 1.5 × 10−3 | 0.0699 |
| N‐dm‐Tam / tamoxifen | Intercept | 2.08 ± 0.58 | 0.0010** |
|   | CYP2D6 activity | ‐0.32 ± 0.13 | 0.0247* |
|   | CYP3A4 activity | 0.15 ± 0.16 | 0.3435 |
|   | CYP3A5 activity | ‐0.04 ± 0.36 | 0.9048 |
|   | CYP2C9 activity | 0.14 ± 0.10 | 0.1882 |
|   | Age | ‐0.01 ± 4.4 × 10−3 | 0.1700 |
|   | Recruitment site | 0.11 ± 0.16 | 0.4916 |
| Endoxifen / 4‐OH‐Tam | Intercept | 6.36 ± 1.27 | 1.5 × 10−5 *** |
|   | CYP2D6 activity | 1.40 ± 0.33 | 0.0002** |
|   | CYP3A4 activity | ‐1.08 ± 0.33 | 0.0028* |
|   | CYP3A5 activity | ‐0.93 ± 0.69 | 0.1854 |
|   | CYP2C9 activity | ‐1.20 ± 0.38 | 0.0034* |
|   | Age | 0.01 ± 0.01 | 0.1995 |
|   | Recruitment site | ‐2.04 ± 0.36 | 1.9 × 10−6 *** |
| Endoxifen / N‐dm‐Tam | Intercept | 0.03 ± 0.04 | 0.4941 |
|   | CYP2D6 activity | 0.03 ± 4.1 × 10−3 | 3.0 × 10−8 *** |
|   | CYP3A4 activity | ‐0.01 ± 0.01 | 0.1180 |
|   | CYP3A5 activity | ‐0.02 ± 0.01 | 0.1388 |
|   | CYP2C9 activity | 4.2 × 10−3 ± 0.01 | 0.6346 |
|   | Age | 1.3 × 10−4 ± 3.3 × 10−4 | 0.6937 |
|   | Recruitment site | ‐0.03 ± 0.01 | 0.0001** |

Table 5 Caption: Multivariate linear regression analysis for factors affecting tamoxifen biotransformation in AIAN participants (n = 42)

## DISCUSSION

Our research is the first tamoxifen pharmacogenetic study focused on AIAN populations. We identified significant associations between cytochrome P450 genetic variation and tamoxifen metabolism. The *CYP2D6* genetic variation was the largest contributor to interindividual variability in endoxifen levels, the principal active metabolite of tamoxifen. We also found that *CYP2D6* variability was the primary predictor of the key metabolic ratios related to tamoxifen biotransformation. Minor roles for variation in *CYP3A4*, *CYP3A5*, and *CYP2C9* were also identified. In addition, we described the frequencies of genetic variation in the *CYP2D6* gene among AIAN participants in southcentral Alaska and compared the results with our previous research with AI participants in northwestern Montana,[3](#cts12542-bib-0003) contributing important knowledge to the field about pharmacogenetic variation in AIAN people.

The primary goal of the *CYP2D6* resequencing (*n* = 94) and genotyping (*n* = 380) performed in this study was to inform the selection of SNVs to genotype in the tamoxifen pharmacogenetic study (*n* = 42) and for potential clinical applications. We observed modest differences between the *CYP2D6* allele frequencies identified in SCF participants and those previously reported in CSKT members.[3](#cts12542-bib-0003) The most common loss of function, poor metabolizer, allele identified in both groups was *CYP2D6*4*, although the allele frequency was higher in CSKT (20.86%) than SCF (14.36%) participants. The most common intermediate activity allele in SCF participants was *CYP2D6*10* (4.26%), whereas it was *CYP2D6*41* in CSKT participants (11.23%). While CPIC guidelines for tamoxifen are not currently available, CPIC recommends prescribing changes for individuals with ultrarapid, intermediate, and poor CYP2D6 metabolizers in guidelines for other medications.[19](#cts12542-bib-0019), [20](#cts12542-bib-0020), [21](#cts12542-bib-0021), [22](#cts12542-bib-0022), [23](#cts12542-bib-0023), [24](#cts12542-bib-0024) As reported previously in the CSKT population, 3.21% and 5.88% of individuals would be classified as intermediate and poor metabolizers, respectively.[3](#cts12542-bib-0003) Based on allele frequencies in the SCF population described here, 1.06% and 3.19% of individuals would be classified as intermediate and poor metabolizers, respectively. Gene deletion (*CYP2D6*5*) analysis was not performed in the SCF resequencing and genotyping cohorts, but based on the 6% *CYP2D6*5* allele frequency observed in the tamoxifen pharmacogenetics cohort (*n* = 42; **Table** [2](#cts12542-tbl-0002)), the percentage of SCF individuals with decreased activity will not increase considerably.

Our tamoxifen pharmacogenetic study is the first to understand the genetic contributors to the formation of endoxifen, the principal active metabolite of tamoxifen, in AIAN patients with breast cancer. Endoxifen plasma concentrations were more than five times higher than 4‐OH‐Tam levels in our study. This is consistent with other reports that endoxifen is the primary active metabolite, because although endoxifen and 4‐OH‐Tam are equipotent, endoxifen circulates at higher systemic concentrations.[11](#cts12542-bib-0011) The *CYP2D6* diplotypes were highly significant predictors of tamoxifen metabolism in our study. Variation in *CYP2D6* was associated with steady‐state plasma concentrations of endoxifen and 4‐OH‐Tam: concentrations of endoxifen were 11.9‐fold higher in extensive metabolizers (activity score = 2) than poor metabolizers (activity score = 0) and concentrations of 4‐OH‐Tam were 3.7‐fold higher in extensive metabolizers than poor metabolizers. The *CYP2D6* variation was also highly associated with key metabolic ratios used to describe the formation and elimination of the active metabolites. The endoxifen/tamoxifen ratio was 4.3‐fold higher in *CYP2D6* extensive metabolizers than poor metabolizers, which relates steady‐state concentrations of the predominant active metabolite to the parent drug and reflects in significant part the contributions of *CYP2D6* in the biotransformation of tamoxifen to endoxifen. The *CYP2D6* variation was strongly associated with the metabolic pathways in which *CYP2D6* plays a primary role: 4‐OH‐Tam/tamoxifen and endoxifen/*N*‐dm‐Tam. The *CYP2D6* variation was also associated with *N*‐dm‐Tam/tamoxifen and endoxifen/4‐OH‐Tam pathways, in which it is thought to have a more minor role, which is likely due to the strong contribution of *CYP2D6* to 4‐OH‐Tam formation and *N*‐dm‐Tam elimination clearances. Our findings are consistent with other data suggesting *CYP2D6* genetic variation may explain as much as 30–50% of variability in endoxifen concentrations during tamoxifen therapy.[11](#cts12542-bib-0011), [13](#cts12542-bib-0013), [27](#cts12542-bib-0027), [28](#cts12542-bib-0028) Tamoxifen pharmacogenetic studies can be further complicated by concomitant use of certain SSRIs known to inhibit *CYP2D6* activity[10](#cts12542-bib-0010), [13](#cts12542-bib-0013); only one participant in our study reported usage of an SSRI, sertraline, which is a weak/moderate *CYP2D6* inhibitor unlikely to significantly alter endoxifen formation.

Genetic variation in *CYP3A4*, *CYP3A5*, and *CYP2C9* may also influence tamoxifen metabolism, although their role in endoxifen formation is less certain.[10](#cts12542-bib-0010), [11](#cts12542-bib-0011), [29](#cts12542-bib-0029), [30](#cts12542-bib-0030), [31](#cts12542-bib-0031), [32](#cts12542-bib-0032), [33](#cts12542-bib-0033), [34](#cts12542-bib-0034) Previous research has focused largely on genotype‐phenotype associations in common *CYP3A4* (**1B*, **1G*, and **22*), *CYP3A5* (**3*), and *CYP2C9* (**2* and **3*) alleles. Our findings reflect the relative ambiguity with respect to the influence of other CYPs on tamoxifen metabolism. We found significant associations with *CYP3A4*, *CYP3A5*, and *CYP2C9* variation and some of the metabolic ratios, although there were no associations with plasma concentrations of the metabolites. Therefore, our conclusions are that whereas *CYP3A4*, *CYP3A5*, and *CYP2C9* have modest effects on tamoxifen bioactivation, *CYP2D6* is the predominant pharmacogenetic predictor of endoxifen formation.

Although strong evidence suggests that *CYP2D6* poor metabolizer status results in decreased endoxifen concentrations, the association between *CYP2D6* diplotype and breast cancer recurrence is more controversial. Researchers found a significant decrease in time‐to‐recurrence and event‐free‐survival in women who were *CYP2D6* poor metabolizers.[28](#cts12542-bib-0028) The International Tamoxifen Pharmacogenomics Consortium also conducted a meta‐analysis and found that *CYP2D6* poor metabolizers had worse disease‐free survival.[35](#cts12542-bib-0035) Two prospective tamoxifen studies published in 2012, however, found negative associations between *CYP2D6* diplotype and breast cancer recurrence.[36](#cts12542-bib-0036), [37](#cts12542-bib-0037) The following year, a third prospective tamoxifen study was published that found a positive association between *CYP2D6* diplotype and clinical outcomes.[38](#cts12542-bib-0038) Subsequently, it was determined that there may have been methodological issues in the negative studies because of the use of tumor tissues, not healthy tissues, for DNA samples and a failure to test for the *CYP2D6*5* gene deletion.[39](#cts12542-bib-0039), [40](#cts12542-bib-0040), [41](#cts12542-bib-0041) The consequences of *CYP2D6* genotyping from tumor samples may have led to a loss of heterozygosity, substantial deviations from HWE, and misclassification of *CYP2D6* alleles in these negative studies putting the results into doubt.[42](#cts12542-bib-0042), [43](#cts12542-bib-0043), [44](#cts12542-bib-0044) Care must be taken to properly design studies aimed at measuring the effect of *CYP2D6* genetic variation on breast cancer recurrence and disease‐free survival. Additional attention must be paid to broadly screen for *CYP2D6* variation, including identification of function SNVs, insertions/deletions, and structural variations (e.g., gene deletions, duplications, and rearrangements). Finally, a significant limitation of existing tamoxifen pharmacogenetic studies is a focus on common variants identified in populations primarily of European descent, which may not be applicable to diverse populations. Our investigation presents a major step forward to address this limitation by identifying major *CYP2D6* variants present in AIAN populations and demonstrating their associations with tamoxifen biotransformation, in order to inform accurate genotyping strategies for future studies with AIAN patients with breast cancer.

We observed strong associations with *CYP2D6* variation, but we only detected modest associations with variation in other *CYP* genes, in contrast with what others have observed particularly in the case of *CYP2C9*.[11](#cts12542-bib-0011), [33](#cts12542-bib-0033), [34](#cts12542-bib-0034) One of the limitations of our study is the small cohort size and we may have low power to detect associations for less common variants in these genes. Structural variation in *CYP2D6* contributes to variable *CYP2D6* activity, and although we genotyped for the *CYP2D6*5* deletion in the tamoxifen cohort, we did not genotype for duplications or rearrangements. Additionally, inclusion of variants with uncertain function (e.g., *CYP3A4*1B*) could potentially limit the analysis for other *CYP* genes. Finally, stronger representation of participants outside of Alaska would expand the applicability of results as well as reduce the influence of site‐specific differences noted between the two recruitment sites. The findings of our study, however, are consistent with studies in larger patient cohorts, the majority of whom are not AIAN patients; therefore, generalizability is likely.

In summary, our research adds to the growing body of evidence that *CYP2D6* genetic variation is an important predictor of tamoxifen bioactivation to endoxifen and is the first study focused on AIAN women. Our study demonstrates the potential benefit of implementation of pre‐emptive pharmacogenetic testing to guide tamoxifen therapy in AIAN people. The AIAN women who are CYP2D6 poor metabolizers have significantly lower circulating concentrations of endoxifen. Other researchers have had success in normalizing endoxifen levels using pharmacogenetics‐guided tamoxifen dosing, however, normalized levels in poor metabolizers have not been achieved.[45](#cts12542-bib-0045), [46](#cts12542-bib-0046), [47](#cts12542-bib-0047), [48](#cts12542-bib-0048), [49](#cts12542-bib-0049) As a result, these patients may benefit from alternative prescribing strategies during anti‐estrogenic therapy, such as use of aromatase inhibitors or the direct administration of endoxifen. Although endoxifen is not available as a medication, it is currently under investigation with encouraging results from a recent phase I study.[50](#cts12542-bib-0050) By identifying the *CYP2D6* metabolizer status of each patient prior to initiating tamoxifen therapy, prescribing choices can optimize the delivery of precision medicine.

## Conflict of Interest

The authors declared no conflict of interest.

## Supporting information

## Acknowledgments

The authors thank Barbara Kavanaugh, Program Manager for the NWA‐PGRN, for directing programmatic collaborations; Patricia L. Stapleton and Jesse M. Tsai in the Functional Genomics Laboratory at the University of Washington (P30ES007033) for performing the *CYP2D6*5* assay. In Alaska, the authors sincerely thank customer‐owners of the Southcentral Foundation for their input, guidance, and study participation; the ANMC Oncology and Hematology Clinic staff in assisting with patient recruitment; and the SCF Board of Directors and the SCF Research Oversight Committee. In Montana, the authors thank members of the CSKT Community Pharmacogenetics Advisory Council for their input and guidance on the research, and the CSKT Tribal Health Department and CSKT Tribal Council for their approval and support of this project.

## Source of Funding

This work was supported by the Northwest‐Alaska Pharmacogenomics Research Network (NWA‐PGRN) (U01GM092676 and P01GM116691).

## Author Contributions

B.A.K., A.E.F., K.E.T., and E.L.W. wrote the manuscript. K.E.T. and E.L.W. designed the research. B.A.K., R.R., A.E.F., L.I.M., B.D.S., J.A.B., M.J.O., L.T., H.F., C.L., P.B., B.P., D.N., K.H., D.A.D., T.A.T., K.E.T., and E.L.W. performed the research. B.A.K., A.E.F., T.A.T., and E.L.W. analyzed the data. B.P. and D.N. contributed new reagents/analytical tools.

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Boyer, B , Dillard, D. , Woodahl, E.L. , Whitener, R. , Thummel, K.E. & Burke, W . Ethical issues in developing pharmacogenetic research partnerships with American Indigenous communities. Clin. Pharmacol. Ther. 89, 343–345 (2011).  [DOI](https://doi.org/10.1038/clpt.2010.303) | [PMC free article](/articles/PMC3090734/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21326261/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther&title=Ethical%20issues%20in%20developing%20pharmacogenetic%20research%20partnerships%20with%20American%20Indigenous%20communities&author=B%20Boyer&author=D.%20Dillard&author=E.L.%20Woodahl&author=R.%20Whitener&author=K.E.%20Thummel&volume=89&publication_year=2011&pages=343-345&pmid=21326261&doi=10.1038/clpt.2010.303&)

2. Woodahl, E.L. , Lesko, L.J. , Hopkins, S. , Robinson, R.F. , Thummel, K.E. & Burke, W . Pharmacogenetic research in partnership with American Indian and Alaska Native communities. Pharmacogenomics 15, 1235–1241 (2014).  [DOI](https://doi.org/10.2217/pgs.14.91) | [PMC free article](/articles/PMC4201360/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25141898/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Pharmacogenetic%20research%20in%20partnership%20with%20American%20Indian%20and%20Alaska%20Native%20communities&author=E.L.%20Woodahl&author=L.J.%20Lesko&author=S.%20Hopkins&author=R.F.%20Robinson&author=K.E.%20Thummel&volume=15&publication_year=2014&pages=1235-1241&pmid=25141898&doi=10.2217/pgs.14.91&)

3. Fohner, A. et al Pharmacogenetics in American Indian populations: analysis of CYP2D6, CYP3A4, CYP3A5, and CYP2C9 in the Confederated Salish and Kootenai Tribes. Pharmacogenet. Genomics 23, 403–414 (2013).  [DOI](https://doi.org/10.1097/FPC.0b013e3283629ce9) | [PMC free article](/articles/PMC3937472/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23778323/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet.%20Genomics&title=Pharmacogenetics%20in%20American%20Indian%20populations:%20analysis%20of%20CYP2D6,%20CYP3A4,%20CYP3A5,%20and%20CYP2C9%20in%20the%20Confederated%20Salish%20and%20Kootenai%20Tribes&author=A.%20Fohner&volume=23&publication_year=2013&pages=403-414&pmid=23778323&doi=10.1097/FPC.0b013e3283629ce9&)

4. Fohner, A.E. et al Variation in genes controlling warfarin disposition and response in American Indian and Alaska Native people: CYP2C9, VKORC1, CYP4F2, CYP4F11, GGCX. Pharmacogenet. Genomics 25, 343–353 (2015).  [DOI](https://doi.org/10.1097/FPC.0000000000000143) | [PMC free article](/articles/PMC4461509/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25946405/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet.%20Genomics&title=Variation%20in%20genes%20controlling%20warfarin%20disposition%20and%20response%20in%20American%20Indian%20and%20Alaska%20Native%20people:%20CYP2C9,%20VKORC1,%20CYP4F2,%20CYP4F11,%20GGCX&author=A.E.%20Fohner&volume=25&publication_year=2015&pages=343-353&pmid=25946405&doi=10.1097/FPC.0000000000000143&)

5. Roubidoux, M.A. Breast cancer and screening in American Indian and Alaska Native women. J. Cancer Educ. 27 (1 Suppl), S66–S72 (2012).  [DOI](https://doi.org/10.1007/s13187-012-0323-6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22351427/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Cancer%20Educ&title=Breast%20cancer%20and%20screening%20in%20American%20Indian%20and%20Alaska%20Native%20women&author=M.A.%20Roubidoux&volume=27&publication_year=2012&pages=S66-S72&pmid=22351427&doi=10.1007/s13187-012-0323-6&)

6. Wingo, P.A. et al Breast cancer incidence among American Indian and Alaska Native women: US, 1999–2004. Cancer 113 (5 Suppl), 1191–1202 (2008).  [DOI](https://doi.org/10.1002/cncr.23725) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18720389/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer&title=Breast%20cancer%20incidence%20among%20American%20Indian%20and%20Alaska%20Native%20women:%20US,%201999%E2%80%932004&author=P.A.%20Wingo&volume=113%20(5%20Suppl)&publication_year=2008&pages=1191-1202&pmid=18720389&doi=10.1002/cncr.23725&)

7. Moore, S.P. et al Cancer incidence in indigenous people in Australia, New Zealand, Canada, and the USA: a comparative population‐based study. Lancet Oncol. 16, 1483–1492 (2015).  [DOI](https://doi.org/10.1016/S1470-2045(15)00232-6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26476758/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet%20Oncol&title=Cancer%20incidence%20in%20indigenous%20people%20in%20Australia,%20New%20Zealand,%20Canada,%20and%20the%20USA:%20a%20comparative%20population%E2%80%90based%20study&author=S.P.%20Moore&volume=16&publication_year=2015&pages=1483-1492&pmid=26476758&doi=10.1016/S1470-2045(15)00232-6&)

8. DeSantis, C.E. , Ma, J. , Goding Sauer, A. , Newman, L.A. & Jemal, A . Breast cancer statistics, 2017, racial disparity in mortality by state. CA Cancer J. Clin. 67, 439–448 (2017).  [DOI](https://doi.org/10.3322/caac.21412) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28972651/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=CA%20Cancer%20J.%20Clin.&title=Breast%20cancer%20statistics,%202017,%20racial%20disparity%20in%20mortality%20by%20state&author=C.E.%20DeSantis&author=J.%20Ma&author=A.%20Goding%20Sauer&author=L.A.%20Newman&author=A%20Jemal&volume=67&publication_year=2017&pages=439-448&pmid=28972651&doi=10.3322/caac.21412&)

9. Goetz, M.P. Tamoxifen, endoxifen, and CYP2D6: the rules for evaluating a predictive factor. Oncology (Williston Park) 23, 1233–1234, 1236 (2009).  [PubMed](https://pubmed.ncbi.nlm.nih.gov/20120835/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncology%20(Williston%20Park)&title=Tamoxifen,%20endoxifen,%20and%20CYP2D6:%20the%20rules%20for%20evaluating%20a%20predictive%20factor&author=M.P.%20Goetz&volume=23&publication_year=2009&pages=1233-1234&pmid=20120835&)

10. Jin, Y. et al CYP2D6 genotype, antidepressant use, and tamoxifen metabolism during adjuvant breast cancer treatment. J. Natl. Cancer Inst. 97, 30–39 (2005).  [DOI](https://doi.org/10.1093/jnci/dji005) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15632378/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Natl.%20Cancer%20Inst&title=CYP2D6%20genotype,%20antidepressant%20use,%20and%20tamoxifen%20metabolism%20during%20adjuvant%20breast%20cancer%20treatment&author=Y.%20Jin&volume=97&publication_year=2005&pages=30-39&pmid=15632378&doi=10.1093/jnci/dji005&)

11. Mürdter, T.E. et al Activity levels of tamoxifen metabolites at the estrogen receptor and the impact of genetic polymorphisms of phase I and II enzymes on their concentration levels in plasma. Clin. Pharmacol. Ther. 89, 708–717 (2011).  [DOI](https://doi.org/10.1038/clpt.2011.27) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21451508/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther&title=Activity%20levels%20of%20tamoxifen%20metabolites%20at%20the%20estrogen%20receptor%20and%20the%20impact%20of%20genetic%20polymorphisms%20of%20phase%20I%20and%20II%20enzymes%20on%20their%20concentration%20levels%20in%20plasma&author=T.E.%20M%C3%BCrdter&volume=89&publication_year=2011&pages=708-717&pmid=21451508&doi=10.1038/clpt.2011.27&)

12. Buck, M.B. , Coller, J.K. , Mürdter, T.E. , Eichelbaum, M. & Knabbe, C . TGFbeta2 and TbetaRII are valid molecular biomarkers for the antiproliferative effects of tamoxifen and tamoxifen metabolites in breast cancer cells. Breast Cancer Res. Treat. 107, 15–24 (2008).  [DOI](https://doi.org/10.1007/s10549-007-9526-7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18043895/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Breast%20Cancer%20Res.%20Treat.&title=TGFbeta2%20and%20TbetaRII%20are%20valid%20molecular%20biomarkers%20for%20the%20antiproliferative%20effects%20of%20tamoxifen%20and%20tamoxifen%20metabolites%20in%20breast%20cancer%20cells&author=M.B.%20Buck&author=J.K.%20Coller&author=T.E.%20M%C3%BCrdter&author=M.%20Eichelbaum&author=C%20Knabbe&volume=107&publication_year=2008&pages=15-24&pmid=18043895&doi=10.1007/s10549-007-9526-7&)

13. Borges, S. et al Quantitative effect of CYP2D6 genotype and inhibitors on tamoxifen metabolism: implication for optimization of breast cancer treatment. Clin. Pharmacol. Ther. 80, 61–74 (2006).  [DOI](https://doi.org/10.1016/j.clpt.2006.03.013) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16815318/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Quantitative%20effect%20of%20CYP2D6%20genotype%20and%20inhibitors%20on%20tamoxifen%20metabolism:%20implication%20for%20optimization%20of%20breast%20cancer%20treatment&author=S.%20Borges&volume=80&publication_year=2006&pages=61-74&pmid=16815318&doi=10.1016/j.clpt.2006.03.013&)

14. Stearns, V. et al Active tamoxifen metabolite plasma concentrations after coadministration of tamoxifen and the selective serotonin reuptake inhibitor paroxetine. J. Natl. Cancer Inst. 95, 1758–1764 (2003).  [DOI](https://doi.org/10.1093/jnci/djg108) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14652237/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Natl.%20Cancer%20Inst.&title=Active%20tamoxifen%20metabolite%20plasma%20concentrations%20after%20coadministration%20of%20tamoxifen%20and%20the%20selective%20serotonin%20reuptake%20inhibitor%20paroxetine&author=V.%20Stearns&volume=95&publication_year=2003&pages=1758-1764&pmid=14652237&doi=10.1093/jnci/djg108&)

15. Hoskins, J.M. , Carey, L.A. & McLeod, H.L. CYP2D6 and tamoxifen: DNA matters in breast cancer. Nat. Rev. Cancer 9, 576–586 (2009).  [DOI](https://doi.org/10.1038/nrc2683) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19629072/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Cancer&title=CYP2D6%20and%20tamoxifen:%20DNA%20matters%20in%20breast%20cancer&author=J.M.%20Hoskins&author=L.A.%20Carey&author=H.L.%20McLeod&volume=9&publication_year=2009&pages=576-586&pmid=19629072&doi=10.1038/nrc2683&)

16. Gaedigk, A. et al The Pharmacogene Variation (PharmVar) Consortium: incorporation of the Human Cytochrome P450 (CYP) Allele Nomenclature Database. Clin. Pharmacol. Ther. (2017). [Epub ahead of print]  [DOI](https://doi.org/10.1002/cpt.910) | [PMC free article](/articles/PMC5836850/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29134625/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=The%20Pharmacogene%20Variation%20(PharmVar)%20Consortium:%20incorporation%20of%20the%20Human%20Cytochrome%20P450%20(CYP)%20Allele%20Nomenclature%20Database&author=A.%20Gaedigk&publication_year=2017&pmid=29134625&doi=10.1002/cpt.910&)

17. Hersberger, M. , Marti‐Jaun, J. , Rentsch, K. & Hänseler, E. Rapid detection of the CYP2D6*3, CYP2D6*4, and CYP2D6*6 alleles by tetra‐primer PCR and of the CYP2D6*5 allele by multiplex long PCR. Clin. Chem. 46 (8 Pt 1), 1072–1077 (2000).  [PubMed](https://pubmed.ncbi.nlm.nih.gov/10926885/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Chem&title=Rapid%20detection%20of%20the%20CYP2D6*3,%20CYP2D6*4,%20and%20CYP2D6*6%20alleles%20by%20tetra%E2%80%90primer%20PCR%20and%20of%20the%20CYP2D6*5%20allele%20by%20multiplex%20long%20PCR&author=M.%20Hersberger&author=J.%20Marti%E2%80%90Jaun&author=K.%20Rentsch&author=E.%20H%C3%A4nseler&volume=(8%20Pt%201)&publication_year=2000&pages=1072-1077&pmid=10926885&)

18. Gaedigk, A. , Simon, S.D. , Pearce, R.E. , Bradford, L.D. , Kennedy, M.J. & Leeder, J.S. The CYP2D6 activity score: translating genotype information into a qualitative measure of phenotype. Clin. Pharmacol. Ther. 83, 234–242 (2008).  [DOI](https://doi.org/10.1038/sj.clpt.6100406) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17971818/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=The%20CYP2D6%20activity%20score:%20translating%20genotype%20information%20into%20a%20qualitative%20measure%20of%20phenotype&author=A.%20Gaedigk&author=S.D.%20Simon&author=R.E.%20Pearce&author=L.D.%20Bradford&author=M.J.%20Kennedy&volume=83&publication_year=2008&pages=234-242&pmid=17971818&doi=10.1038/sj.clpt.6100406&)

19. Crews, K.R. et al Clinical Pharmacogenetics Implementation Consortium (CPIC) guidelines for codeine therapy in the context of cytochrome P450 2D6 (CYP2D6) genotype. Clin. Pharmacol. Ther. 91, 321–326 (2012).  [DOI](https://doi.org/10.1038/clpt.2011.287) | [PMC free article](/articles/PMC3289963/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22205192/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Clinical%20Pharmacogenetics%20Implementation%20Consortium%20(CPIC)%20guidelines%20for%20codeine%20therapy%20in%20the%20context%20of%20cytochrome%20P450%202D6%20(CYP2D6)%20genotype&author=K.R.%20Crews&volume=91&publication_year=2012&pages=321-326&pmid=22205192&doi=10.1038/clpt.2011.287&)

20. Hicks, J.K. et al Clinical Pharmacogenetics Implementation Consortium (CPIC) guideline for CYP2D6 and CYP2C19 genotypes and dosing of selective serotonin reuptake inhibitors. Clin. Pharmacol. Ther. 98, 127–134 (2015).  [DOI](https://doi.org/10.1002/cpt.147) | [PMC free article](/articles/PMC4512908/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25974703/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Clinical%20Pharmacogenetics%20Implementation%20Consortium%20(CPIC)%20guideline%20for%20CYP2D6%20and%20CYP2C19%20genotypes%20and%20dosing%20of%20selective%20serotonin%20reuptake%20inhibitors&author=J.K.%20Hicks&volume=98&publication_year=2015&pages=127-134&pmid=25974703&doi=10.1002/cpt.147&)

21. Hicks, J.K. et al Clinical Pharmacogenetics Implementation Consortium guideline for CYP2D6 and CYP2C19 genotypes and dosing of tricyclic antidepressants. Clin. Pharmacol. Ther. 93, 402–408 (2013).  [DOI](https://doi.org/10.1038/clpt.2013.2) | [PMC free article](/articles/PMC3689226/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23486447/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Clinical%20Pharmacogenetics%20Implementation%20Consortium%20guideline%20for%20CYP2D6%20and%20CYP2C19%20genotypes%20and%20dosing%20of%20tricyclic%20antidepressants&author=J.K.%20Hicks&volume=93&publication_year=2013&pages=402-408&pmid=23486447&doi=10.1038/clpt.2013.2&)

22. Crews, K.R. et al Clinical Pharmacogenetics Implementation Consortium guidelines for cytochrome P450 2D6 genotype and codeine therapy: 2014 update. Clin. Pharmacol. Ther. 95, 376–382 (2014).  [DOI](https://doi.org/10.1038/clpt.2013.254) | [PMC free article](/articles/PMC3975212/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24458010/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Clinical%20Pharmacogenetics%20Implementation%20Consortium%20guidelines%20for%20cytochrome%20P450%202D6%20genotype%20and%20codeine%20therapy:%202014%20update&author=K.R.%20Crews&volume=95&publication_year=2014&pages=376-382&pmid=24458010&doi=10.1038/clpt.2013.254&)

23. Hicks, J.K. et al Clinical pharmacogenetics implementation consortium guideline (CPIC) for CYP2D6 and CYP2C19 genotypes and dosing of tricyclic antidepressants: 2016 update. Clin. Pharmacol. Ther. 102, 37–44 (2017).  [DOI](https://doi.org/10.1002/cpt.597) | [PMC free article](/articles/PMC5478479/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27997040/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Clinical%20pharmacogenetics%20implementation%20consortium%20guideline%20(CPIC)%20for%20CYP2D6%20and%20CYP2C19%20genotypes%20and%20dosing%20of%20tricyclic%20antidepressants:%202016%20update&author=J.K.%20Hicks&publication_year=2017&pages=37-44&pmid=27997040&doi=10.1002/cpt.597&)

24. Bell, G.C. et al Clinical Pharmacogenetics Implementation Consortium (CPIC) guideline for CYP2D6 genotype and use of ondansetron and tropisetron. Clin. Pharmacol. Ther. 102, 213–218 (2016).  [DOI](https://doi.org/10.1002/cpt.598) | [PMC free article](/articles/PMC5479760/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28002639/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Clinical%20Pharmacogenetics%20Implementation%20Consortium%20(CPIC)%20guideline%20for%20CYP2D6%20genotype%20and%20use%20of%20ondansetron%20and%20tropisetron&author=G.C.%20Bell&publication_year=2016&pages=213-218&pmid=28002639&doi=10.1002/cpt.598&)

25. Barrett, J.C. , Fry, B. , Maller, J. & Daly, M.J. Haploview: analysis and visualization of LD and haplotype maps. Bioinformatics 21, 263–265 (2005).  [DOI](https://doi.org/10.1093/bioinformatics/bth457) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15297300/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=Haploview:%20analysis%20and%20visualization%20of%20LD%20and%20haplotype%20maps&author=J.C.%20Barrett&author=B.%20Fry&author=J.%20Maller&author=M.J.%20Daly&volume=21&publication_year=2005&pages=263-265&pmid=15297300&doi=10.1093/bioinformatics/bth457&)

26. White, H . A heteroskedasticity‐consistent covariance matrix estimator and a direct test for heteroskedasticity. Econometrica 48, 817–838 (1980).  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Econometrica&title=A%20heteroskedasticity%E2%80%90consistent%20covariance%20matrix%20estimator%20and%20a%20direct%20test%20for%20heteroskedasticity&author=H%20White&volume=48&publication_year=1980&pages=817-838&)

27. Lim, H.S. , Ju Lee, H. , Seok Lee, K. , Sook Lee, E. , Jang, I.J. & Ro, J. Clinical implications of CYP2D6 genotypes predictive of tamoxifen pharmacokinetics in metastatic breast cancer. J. Clin. Oncol. 25, 3837–3845 (2007).  [DOI](https://doi.org/10.1200/JCO.2007.11.4850) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17761971/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Oncol.&title=Clinical%20implications%20of%20CYP2D6%20genotypes%20predictive%20of%20tamoxifen%20pharmacokinetics%20in%20metastatic%20breast%20cancer&author=H.S.%20Lim&author=H.%20Ju%20Lee&author=K.%20Seok%20Lee&author=E.%20Sook%20Lee&author=I.J.%20Jang&volume=25&publication_year=2007&pages=3837-3845&pmid=17761971&doi=10.1200/JCO.2007.11.4850&)

28. Schroth, W. et al Association between CYP2D6 polymorphisms and outcomes among women with early stage breast cancer treated with tamoxifen. JAMA 302, 1429–1436 (2009).  [DOI](https://doi.org/10.1001/jama.2009.1420) | [PMC free article](/articles/PMC3909953/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19809024/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA&title=Association%20between%20CYP2D6%20polymorphisms%20and%20outcomes%20among%20women%20with%20early%20stage%20breast%20cancer%20treated%20with%20tamoxifen&author=W.%20Schroth&volume=302&publication_year=2009&pages=1429-1436&pmid=19809024&doi=10.1001/jama.2009.1420&)

29. Teft, W.A. et al CYP3A4 and seasonal variation in vitamin D status in addition to CYP2D6 contribute to therapeutic endoxifen level during tamoxifen therapy. Breast Cancer Res. Treat. 139, 95–105 (2013).  [DOI](https://doi.org/10.1007/s10549-013-2511-4) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23580071/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Breast%20Cancer%20Res.%20Treat.&title=CYP3A4%20and%20seasonal%20variation%20in%20vitamin%20D%20status%20in%20addition%20to%20CYP2D6%20contribute%20to%20therapeutic%20endoxifen%20level%20during%20tamoxifen%20therapy&author=W.A.%20Teft&volume=139&publication_year=2013&pages=95-105&pmid=23580071&doi=10.1007/s10549-013-2511-4&)

30. Lim, J.S. et al Impact of CYP2D6, CYP3A5, CYP2C9 and CYP2C19 polymorphisms on tamoxifen pharmacokinetics in Asian breast cancer patients. Br. J. Clin. Pharmacol. 71, 737–750 (2011).  [DOI](https://doi.org/10.1111/j.1365-2125.2011.03905.x) | [PMC free article](/articles/PMC3093079/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21480951/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br.%20J.%20Clin.%20Pharmacol.&title=Impact%20of%20CYP2D6,%20CYP3A5,%20CYP2C9%20and%20CYP2C19%20polymorphisms%20on%20tamoxifen%20pharmacokinetics%20in%20Asian%20breast%20cancer%20patients&author=J.S.%20Lim&volume=71&publication_year=2011&pages=737-750&pmid=21480951&doi=10.1111/j.1365-2125.2011.03905.x&)

31. Wegman, P. , Elingarami, S. , Carstensen, J. , Stål, O. , Nordenskjöld, B. & Wingren, S . Genetic variants of CYP3A5, CYP2D6, SULT1A1, UGT2B15 and tamoxifen response in postmenopausal patients with breast cancer. Breast Cancer Res. 9, R7 (2007).  [DOI](https://doi.org/10.1186/bcr1640) | [PMC free article](/articles/PMC1851378/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17244352/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Breast%20Cancer%20Res.&title=Genetic%20variants%20of%20CYP3A5,%20CYP2D6,%20SULT1A1,%20UGT2B15%20and%20tamoxifen%20response%20in%20postmenopausal%20patients%20with%20breast%20cancer&author=P.%20Wegman&author=S.%20Elingarami&author=J.%20Carstensen&author=O.%20St%C3%A5l&author=B.%20Nordenskj%C3%B6ld&volume=9&publication_year=2007&pages=R7&pmid=17244352&doi=10.1186/bcr1640&)

32. Schroth, W. et al Breast cancer treatment outcome with adjuvant tamoxifen relative to patient CYP2D6 and CYP2C19 genotypes. J. Clin. Oncol. 25, 5187–5193 (2007).  [DOI](https://doi.org/10.1200/JCO.2007.12.2705) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18024866/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Oncol.&title=Breast%20cancer%20treatment%20outcome%20with%20adjuvant%20tamoxifen%20relative%20to%20patient%20CYP2D6%20and%20CYP2C19%20genotypes&author=W.%20Schroth&volume=25&publication_year=2007&pages=5187-5193&pmid=18024866&doi=10.1200/JCO.2007.12.2705&)

33. Marcath, L.A. et al Comprehensive assessment of cytochromes P450 and transporter genetics with endoxifen concentration during tamoxifen treatment. Pharmacogenet. Genomics 27, 402–409 (2017).  [DOI](https://doi.org/10.1097/FPC.0000000000000311) | [PMC free article](/articles/PMC5659294/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28877533/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet.%20Genomics&title=Comprehensive%20assessment%20of%20cytochromes%20P450%20and%20transporter%20genetics%20with%20endoxifen%20concentration%20during%20tamoxifen%20treatment&author=L.A.%20Marcath&volume=27&publication_year=2017&pages=402-409&pmid=28877533&doi=10.1097/FPC.0000000000000311&)

34. Powers, J.L. et al Multigene and drug interaction approach for tamoxifen metabolite patterns reveals possible involvement of CYP2C9, CYP2C19, and ABCB1. J. Clin. Pharmacol. 56, 1570–1581 (2016).  [DOI](https://doi.org/10.1002/jcph.771) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27198207/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Pharmacol.&title=Multigene%20and%20drug%20interaction%20approach%20for%20tamoxifen%20metabolite%20patterns%20reveals%20possible%20involvement%20of%20CYP2C9,%20CYP2C19,%20and%20ABCB1&author=J.L.%20Powers&volume=56&publication_year=2016&pages=1570-1581&pmid=27198207&doi=10.1002/jcph.771&)

35. Province, M.A. et al CYP2D6 genotype and adjuvant tamoxifen: meta‐analysis of heterogeneous study populations. Clin. Pharmacol. Ther. 95, 216–227 (2014).  [DOI](https://doi.org/10.1038/clpt.2013.186) | [PMC free article](/articles/PMC3904554/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24060820/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=CYP2D6%20genotype%20and%20adjuvant%20tamoxifen:%20meta%E2%80%90analysis%20of%20heterogeneous%20study%20populations&author=M.A.%20Province&volume=95&publication_year=2014&pages=216-227&pmid=24060820&doi=10.1038/clpt.2013.186&)

36. Rae, J.M. et al CYP2D6 and UGT2B7 genotype and risk of recurrence in tamoxifen‐treated breast cancer patients. J. Natl. Cancer Inst. 104, 452–460 (2012).  [DOI](https://doi.org/10.1093/jnci/djs126) | [PMC free article](/articles/PMC3611934/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22395643/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Natl.%20Cancer%20Inst.&title=CYP2D6%20and%20UGT2B7%20genotype%20and%20risk%20of%20recurrence%20in%20tamoxifen%E2%80%90treated%20breast%20cancer%20patients&author=J.M.%20Rae&volume=104&publication_year=2012&pages=452-460&pmid=22395643&doi=10.1093/jnci/djs126&)

37. Regan, M.M. et al CYP2D6 genotype and tamoxifen response in postmenopausal women with endocrine‐responsive breast cancer: the breast international group 1‐98 trial. J. Natl. Cancer Inst. 104, 441–451 (2012).  [DOI](https://doi.org/10.1093/jnci/djs125) | [PMC free article](/articles/PMC3309132/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22395644/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Natl.%20Cancer%20Inst&title=CYP2D6%20genotype%20and%20tamoxifen%20response%20in%20postmenopausal%20women%20with%20endocrine%E2%80%90responsive%20breast%20cancer:%20the%20breast%20international%20group%201%E2%80%9098%20trial&author=M.M.%20Regan&volume=104&publication_year=2012&pages=441-451&pmid=22395644&doi=10.1093/jnci/djs125&)

38. Goetz, M.P. et al CYP2D6 metabolism and patient outcome in the Austrian Breast and Colorectal Cancer Study Group trial (ABCSG) 8. Clin. Cancer Res. 19, 500–507 (2013).  [DOI](https://doi.org/10.1158/1078-0432.CCR-12-2153) | [PMC free article](/articles/PMC3548984/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23213055/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Cancer%20Res&title=CYP2D6%20metabolism%20and%20patient%20outcome%20in%20the%20Austrian%20Breast%20and%20Colorectal%20Cancer%20Study%20Group%20trial%20(ABCSG)%208&author=M.P.%20Goetz&volume=19&publication_year=2013&pages=500-507&pmid=23213055&doi=10.1158/1078-0432.CCR-12-2153&)

39. Stanton, V. Jr. Re: CYP2D6 genotype and tamoxifen response in postmenopausal women with endocrine‐responsive breast cancer: the Breast International Group 1‐98 trial. J. Natl. Cancer Inst. 104, 1265–1266; author reply 1266–1268 (2012).  [DOI](https://doi.org/10.1093/jnci/djs305) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22851267/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Natl.%20Cancer%20Inst.&title=Re:%20CYP2D6%20genotype%20and%20tamoxifen%20response%20in%20postmenopausal%20women%20with%20endocrine%E2%80%90responsive%20breast%20cancer:%20the%20Breast%20International%20Group%201%E2%80%9098%20trial&author=V.%20Stanton&volume=104&publication_year=2012&pages=1265-1266&pmid=22851267&doi=10.1093/jnci/djs305&)

40. Nakamura, Y. , Ratain, M.J. , Cox, N.J. , McLeod, H.L. , Kroetz, D.L. & Flockhart, D.A. Re : CYP2D6 genotype and tamoxifen response in postmenopausal women with endocrine‐responsive breast cancer: the Breast International Group 1‐98 trial. J. Natl. Cancer Inst. 104, 1264; author reply 1266–1268 (2012).  [DOI](https://doi.org/10.1093/jnci/djs304) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22851270/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Natl.%20Cancer%20Inst.&title=CYP2D6%20genotype%20and%20tamoxifen%20response%20in%20postmenopausal%20women%20with%20endocrine%E2%80%90responsive%20breast%20cancer:%20the%20Breast%20International%20Group%201%E2%80%9098%20trial&author=Y.%20Nakamura&author=M.J.%20Ratain&author=N.J.%20Cox&author=H.L.%20McLeod&author=D.L.%20Kroetz&volume=104&publication_year=2012&pages=1266-1268&pmid=22851270&doi=10.1093/jnci/djs304&)

41. Pharoah, P.D. , Abraham, J. & Caldas, C. Re : CYP2D6 genotype and tamoxifen response in postmenopausal women with endocrine‐responsive breast cancer: the Breast International Group 1‐98 trial and Re: CYP2D6 and UGT2B7 genotype and risk of recurrence in tamoxifen‐treated breast cancer patients. J. Natl. Cancer Inst. 104, 1263–1264; author reply 1266–1268 (2012).  [DOI](https://doi.org/10.1093/jnci/djs312) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22851268/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Natl.%20Cancer%20Inst.&title=CYP2D6%20genotype%20and%20tamoxifen%20response%20in%20postmenopausal%20women%20with%20endocrine%E2%80%90responsive%20breast%20cancer:%20the%20Breast%20International%20Group%201%E2%80%9098%20trial%20and%20Re:%20CYP2D6%20and%20UGT2B7%20genotype%20and%20risk%20of%20recurrence%20in%20tamoxifen%E2%80%90treated%20breast%20cancer%20patients&author=P.D.%20Pharoah&author=J.%20Abraham&author=C.%20Re%20Caldas&volume=104&publication_year=2012&pages=1263-1264&pmid=22851268&doi=10.1093/jnci/djs312&)

42. Goetz, M.P. et al Loss of heterozygosity at the CYP2D6 locus in breast cancer: implications for germline pharmacogenetic studies. J. Natl. Cancer Inst. 107, dju401 (2014).  [DOI](https://doi.org/10.1093/jnci/dju401) | [PMC free article](/articles/PMC4565524/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25490892/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Natl.%20Cancer%20Inst.&title=Loss%20of%20heterozygosity%20at%20the%20CYP2D6%20locus%20in%20breast%20cancer:%20implications%20for%20germline%20pharmacogenetic%20studies&author=M.P.%20Goetz&volume=107&publication_year=2014&pmid=25490892&doi=10.1093/jnci/dju401&)

43. Ratain, M.J. , Nakamura, Y. & Cox, N.J. CYP2D6 genotype and tamoxifen activity: understanding interstudy variability in methodological quality. Clin. Pharmacol. Ther. 94, 185–187 (2013).  [DOI](https://doi.org/10.1038/clpt.2013.66) | [PMC free article](/articles/PMC3782290/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23872831/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=CYP2D6%20genotype%20and%20tamoxifen%20activity:%20understanding%20interstudy%20variability%20in%20methodological%20quality&author=M.J.%20Ratain&author=Y.%20Nakamura&author=N.J.%20Cox&volume=94&publication_year=2013&pages=185-187&pmid=23872831&doi=10.1038/clpt.2013.66&)

44. Brauch, H. et al Tamoxifen use in postmenopausal breast cancer: CYP2D6 matters. J. Clin. Oncol. 31, 176–180 (2013).  [DOI](https://doi.org/10.1200/JCO.2012.44.6625) | [PMC free article](/articles/PMC3731938/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23091108/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Oncol.&title=Tamoxifen%20use%20in%20postmenopausal%20breast%20cancer:%20CYP2D6%20matters&author=H.%20Brauch&volume=31&publication_year=2013&pages=176-180&pmid=23091108&doi=10.1200/JCO.2012.44.6625&)

45. Irvin, W.J. Jr et al Genotype‐guided tamoxifen dosing increases active metabolite exposure in women with reduced CYP2D6 metabolism: a multicenter study. J. Clin. Oncol. 29, 3232–3239 (2011).  [DOI](https://doi.org/10.1200/JCO.2010.31.4427) | [PMC free article](/articles/PMC3158597/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21768473/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Oncol.&title=Genotype%E2%80%90guided%20tamoxifen%20dosing%20increases%20active%20metabolite%20exposure%20in%20women%20with%20reduced%20CYP2D6%20metabolism:%20a%20multicenter%20study&author=W.J.%20Irvin&volume=29&publication_year=2011&pages=3232-3239&pmid=21768473&doi=10.1200/JCO.2010.31.4427&)

46. Dezentje, V.O. et al CYP2D6 genotype‐ and endoxifen‐guided tamoxifen dose escalation increases endoxifen serum concentrations without increasing side effects. Breast Cancer Res. Treat. 153, 583–590 (2015).  [DOI](https://doi.org/10.1007/s10549-015-3562-5) | [PMC free article](/articles/PMC4589558/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26369533/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Breast%20Cancer%20Res.%20Treat.&title=CYP2D6%20genotype%E2%80%90%20and%20endoxifen%E2%80%90guided%20tamoxifen%20dose%20escalation%20increases%20endoxifen%20serum%20concentrations%20without%20increasing%20side%20effects&author=V.O.%20Dezentje&volume=153&publication_year=2015&pages=583-590&pmid=26369533&doi=10.1007/s10549-015-3562-5&)

47. Barginear, M.F. et al Increasing tamoxifen dose in breast cancer patients based on CYP2D6 genotypes and endoxifen levels: effect on active metabolite isomers and the antiestrogenic activity score. Clin. Pharmacol. Ther. 90, 605–611 (2011).  [DOI](https://doi.org/10.1038/clpt.2011.153) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21900890/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Increasing%20tamoxifen%20dose%20in%20breast%20cancer%20patients%20based%20on%20CYP2D6%20genotypes%20and%20endoxifen%20levels:%20effect%20on%20active%20metabolite%20isomers%20and%20the%20antiestrogenic%20activity%20score&author=M.F.%20Barginear&volume=90&publication_year=2011&pages=605-611&pmid=21900890&doi=10.1038/clpt.2011.153&)

48. Kiyotani, K. et al Dose‐adjustment study of tamoxifen based on CYP2D6 genotypes in Japanese breast cancer patients. Breast Cancer Res. Treat. 131, 137–145 (2012).  [DOI](https://doi.org/10.1007/s10549-011-1777-7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21947681/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Breast%20Cancer%20Res.%20Treat.&title=Dose%E2%80%90adjustment%20study%20of%20tamoxifen%20based%20on%20CYP2D6%20genotypes%20in%20Japanese%20breast%20cancer%20patients&author=K.%20Kiyotani&volume=131&publication_year=2012&pages=137-145&pmid=21947681&doi=10.1007/s10549-011-1777-7&)

49. Martinez de Duenas, E . et al. Adjusting the dose of tamoxifen in patients with early breast cancer and CYP2D6 poor metabolizer phenotype. Breast 23, 400–406 (2014).  [DOI](https://doi.org/10.1016/j.breast.2014.02.008) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24685597/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Breast&title=et%C2%A0al.%20Adjusting%20the%20dose%20of%20tamoxifen%20in%20patients%20with%20early%20breast%20cancer%20and%20CYP2D6%20poor%20metabolizer%20phenotype&author=E%20Martinez%20de%20Duenas&volume=23&publication_year=2014&pages=400-406&pmid=24685597&doi=10.1016/j.breast.2014.02.008&)

50. Goetz, M.P. et al First‐in‐human phase I study of the tamoxifen metabolite Z‐endoxifen in women with endocrine‐refractory metastatic breast cancer. J. Clin. Oncol. 35, 3391–3400 (2017).  [DOI](https://doi.org/10.1200/JCO.2017.73.3246) | [PMC free article](/articles/PMC5648176/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28854070/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Oncol.&title=First%E2%80%90in%E2%80%90human%20phase%20I%20study%20of%20the%20tamoxifen%20metabolite%20Z%E2%80%90endoxifen%20in%20women%20with%20endocrine%E2%80%90refractory%20metastatic%20breast%20cancer&author=M.P.%20Goetz&volume=35&publication_year=2017&pages=3391-3400&pmid=28854070&doi=10.1200/JCO.2017.73.3246&)
