# Effect of Genetic Variability in the CYP4F2, CYP4F11, and CYP4F12 Genes on Liver mRNA Levels and Warfarin Response

## Metadata
**Authors:** J E Zhang, Kathrin Klein, Andrea L Jorgensen, Ben Francis, Ana Alfirevic, Stephane Bourgeois, Panagiotis Deloukas, Ulrich M Zanger, Munir Pirmohamed
**Journal:** Frontiers in Pharmacology
**Date:** 2017 May 31
**DOI:** [10.3389/fphar.2017.00323](https://doi.org/10.3389/fphar.2017.00323)
**PMID:** 28620303
**PMCID:** PMC5449482
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5449482/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC5449482/pdf/fphar-08-00323.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC5449482/pdf/fphar-08-00323.pdf)

## Abstract

Genetic polymorphisms in the gene encoding cytochrome P450 (CYP) 4F2, a vitamin K oxidase, affect stable warfarin dose requirements and time to therapeutic INR. CYP4F2 is part of the CYP4F gene cluster, which is highly polymorphic and exhibits a high degree of linkage disequilibrium, making it difficult to define causal variants. Our objective was to examine the effect of genetic variability in the CYP4F gene cluster on expression of the individual CYP4F genes and warfarin response. mRNA levels of the CYP4F gene cluster were quantified in human liver samples (n = 149) obtained from a well-characterized liver bank and fine mapping of the CYP4F gene cluster encompassing CYP4F2, CYP4F11, and CYP4F12 was performed. Genome-wide association study (GWAS) data from a prospective cohort of warfarin-treated patients (n = 711) was also analyzed for genetic variations across the CYP4F gene cluster. In addition, SNP-gene expression in human liver tissues and interactions between CYP4F genes were explored in silico using publicly available data repositories. We found that SNPs in CYP4F2, CYP4F11, and CYP4F12 were associated with mRNA expression in the CYP4F gene cluster. In particular, CYP4F2 rs2108622 was associated with increased CYP4F2 expression while CYP4F11 rs1060467 was associated with decreased CYP4F2 expression. Interestingly, these CYP4F2 and CYP4F11 SNPs showed similar effects with warfarin stable dose where CYP4F11 rs1060467 was associated with a reduction in daily warfarin dose requirement (∼1 mg/day, Pc = 0.017), an effect opposite to that previously reported with CYP4F2 (rs2108622). However, inclusion of either or both of these SNPs in a pharmacogenetic algorithm consisting of age, body mass index (BMI), gender, baseline clotting factor II level, CYP2C9∗2 rs1799853, CYP2C9∗3 rs1057910, and VKORC1 rs9923231 improved warfarin dose variability only by 0.5–0.7% with an improvement in dose prediction accuracy of ∼1–2%. Although there is complex regulation across the CYP4F gene cluster, the opposing effects between the two SNPs in the CYP4F gene cluster appear to compensate for each other and their effect on warfarin dose requirement is unlikely to be clinically significant.

Keywords: warfarin, pharmacogenetics, mRNA expression, CYP4F2, CYP4F11, CYP4F12

## Introduction

The *CYP4F* gene subfamily comprises six members, namely *CYP4F2* ([Kikuta et al., 1993](#B25)), *CYP4F3* (*CYP4F3A* and *CYP4F3B*) ([Kikuta et al., 1998](#B24)), *CYP4F8* ([Bylund et al., 2000](#B9)), *CYP4F11* ([Cui et al., 2000](#B11)), *CYP4F12* ([Bylund et al., 2001](#B7); [Hashizume et al., 2001](#B18)), and *CYP4F22* ([Lefevre et al., 2006](#B27)). Structurally, these six *CYP4F* genes are largely similar, with more than 65% amino acid sequence homology. To date, studies have focused on *CYP4F2, CYP4F3, CYP4F8, CYP4F11*, and *CYP4F12* while little is known about the expression and function of *CYP4F22*. The splice sites of *CYP4F2, CYP4F3, CYP4F8, CYP4F11*, and *CYP4F12* are almost identical, suggesting that this cluster of five genes may have evolved by gene duplication ([Bylund et al., 1999](#B8), [2001](#B7); [Kikuta et al., 1999](#B26); [Cui et al., 2000](#B11)).

*CYP4F2, CYP4F3, CYP4F8, CYP4F11*, and *CYP4F12* reside together on chromosome 19p13.1-2, spanning over 320 kb (Supplementary Figure [1](#SM5)). These five members of the *CYP4F* subfamily are all expressed in the liver and are known for their roles in the metabolism of both endogenous and exogenous compounds. They are involved in the catabolism of substrates such as arachidonic acid and its oxygenated derivatives (eicosanoids) such as leukotrienes, prostaglandins (PGs), lipoxins, and hydroxyeicosatetraenoic acids (HETEs) ([Kikuta et al., 1999](#B26); [Bylund et al., 2000](#B9), [2001](#B7); [Hashizume et al., 2001](#B18), [2002](#B19); [Kalsotra et al., 2004](#B23)), and they also catalyze the metabolism of many drugs. For example, *CYP4F2* has also been implicated in the ω-hydroxylation of the tocopherol phytyl side chain in the first step of vitamin E inactivation ([Sontag and Parker, 2002](#B40)). In addition, *CYP4F2* and *CYP4F3B* have been shown to catalyze the initial *O*-demethylation of the anti-parasitic prodrug pafuramidine by human liver and intestinal microsomes ([Wang et al., 2006](#B47), [2007](#B48)). *CYP4F2* has also been reported to be a vitamin K oxidase and plays a role in warfarin response ([McDonald et al., 2009](#B28)). *CYP4F11* is known to be active in the metabolism of several drugs including erythromycin, benzphetamine, ethylmorphine, chlorpromazine, and imipramine ([Kalsotra et al., 2004](#B23)). More recently, a study has reported that *CYP4F11* functions as a vitamin K ω-hydroxylase ([Edson et al., 2013](#B14)). *CYP4F12* has been reported to be involved in the conversion of the antihistaminic prodrug ebastine to the active drug carebastine by hydroxylation ([Hashizume et al., 2001](#B18), [2002](#B19)).

Warfarin is one of the most widely used oral anticoagulants worldwide with proven efficacy in conditions characterized by thromboembolism including atrial fibrillation, deep vein thrombosis, pulmonary embolism, or heart valve prostheses. Despite its efficacy, warfarin is often among the top three drugs that lead to hospitalization from adverse drug reactions ([Budnitz et al., 2007](#B6); [Wysowski et al., 2007](#B49)), owing to its narrow therapeutic window and large inter-individual variability in dose response. Combinations of both non-genetic and genetic factors influence the inter-individual variability in warfarin therapeutic dose requirements. Genetic factors, in particular, single nucleotide polymorphisms (SNPs) in two genes responsible for warfarin pharmacokinetics and pharmacodynamics – cytochrome P450 2C9 (*CYP2C9*) and vitamin K epoxide reductase complex 1 (*VKORC1*) – have repeatedly been found to be significantly associated with warfarin responsiveness, explaining approximately 15 and 25% of dose variability ([Aithal et al., 1999](#B1); [Yuan et al., 2005](#B50); [Gage et al., 2008](#B16); [Wadelius et al., 2009](#B46); [Schwanhausser et al., 2011](#B39)), respectively. Candidate gene(s) and GWAS studies have shown that the *CYP4F2* functional variant, rs2108622, accounts for a small proportion of the variability in warfarin dose requirement (1–7%) ([Caldwell et al., 2008](#B10); [Borgiani et al., 2009](#B4); [Takeuchi et al., 2009](#B44); [Pautas et al., 2010](#B32)). However, some studies have not found an association between rs2108622 and warfarin stable dose ([Zhang et al., 2009](#B52); [Perini et al., 2010](#B34)). A functional study utilizing human liver tissues did not find any association between rs2108622 and *CYP4F2* mRNA but observed a significant association between the rs2108622 variant TT genotype and lower microsomal CYP4F2 protein concentration and reduced vitamin K_1_ oxidation, consistent with its function as a vitamin K_1_ oxidase in catalyzing the ω-hydroxylation of vitamin K_1_ phytyl side chain ([McDonald et al., 2009](#B28)).

We have previously performed fine mapping of the *CYP4F2* region to determine the influence of *CYP4F2* SNPs and haplotypes on various warfarin response outcomes ([Zhang et al., 2009](#B52)). We found an association between rs2189784, a SNP in strong linkage disequilibrium (LD) with rs2108622, with time to achieve therapeutic International Normalized Ratio (INR), but not with stable dose. Given the high degree of homology and LD across the *CYP4F* gene cluster (Supplementary Figure [1](#SM5)), we have undertaken a genotype–phenotype assessment utilizing a well-characterized liver bank and a prospective patient cohort who were followed up for 6 months from the time of intake of warfarin (as summarized in Supplementary Figure [2](#SM6)). *In silico* analysis was also performed to investigate additional SNP-gene associations and the interactions between the *CYP4F* genes.

## Materials and Methods

### Study Populations

Written informed consent in accordance with the Declaration of Helsinki was obtained from all patients recruited to the following cohorts.

#### Liver Surgery Cohort

Blood and liver tissue samples were collected from 149 Caucasian patients undergoing liver surgery at the Department of General, Visceral, and Transplantation Surgery, Campus Virchow, University Medical Centre Charité, Humboldt University, Berlin, Germany, as described previously ([Gomes et al., 2009](#B17)). Normal liver tissues were obtained from adjacent regions of surgically removed liver tumors or metastases or hepatic tissue respected for other reasons. All liver tissue samples were certified to be free of malignant cells by pathological examination. None of these samples were from patients with hepatitis, or cirrhosis, or from those who had chronic alcohol abuse. Clinical patient documentation for all samples included age, gender, medical diagnosis, pre-surgical medication, alcohol use, and smoking. The study was approved by the Research Ethics Committees of the Medical Faculties of the Charité, Humboldt University, Berlin, and of the University of Tuebingen, Tuebingen, Germany.

#### Warfarin-Treated Patient Cohort

Thousand patients starting warfarin therapy were recruited prospectively at two hospitals in Liverpool, United Kingdom (Royal Liverpool and Broadgreen University Hospitals Trust and University Hospital Aintree). The main indications for warfarin therapy were treatment of venous thromboembolism and prophylaxis against systemic emboli in patients with atrial fibrillation. The study was approved by the Birmingham South Research Ethics Committee, United Kingdom.

### Determination of CYP4F2, CYP4F8, CYP4F11, and CYP4F12 mRNA Expression Levels in Human Liver

RNA was extracted from the human liver tissue (*n* = 149) using TRIzol^®^ reagent (Invitrogen, Paisley, United Kingdom) with subsequent RNA clean-up using QIAGEN RNeasy-Mini Kit with on-column DNase treatment. All RNA preparations were of high quality with RNA integrity number (RIN) >7, as measured on the Agilent Bioanalyzer (Nano-Lab Chip Kit, Agilent Technologies, Waldbronn, Germany). Levels of gene expression of over 48,000 mRNA transcripts were assessed by the Human-WG6v2 Expression BeadChip (Illumina, Eindhoven, The Netherlands) as previously described ([Schroder et al., 2013](#B38)). Pre-processing and quality control of the expression data was conducted using the Illumina BeadStudio, version 3.0 (Illumina, San Diego, CA, United States) and the various steps involved are detailed in [Schroder et al. (2013)](#B38). Probe signal intensities corresponding to 15,439 unique genes remain after all pre-processing steps and the data set was log2 transformed. Probe sequences for *CYP4F2*, *CYP4F8*, *CYP4F11*, and *CYP4F12* were confirmed to be specific and expression data were extracted. *CYP4F3* and *CYP4F22* were not further analyzed due to ambiguous probe or gene annotation.

#### Liver Surgery Cohort: CYP4F2, CYP4F11, CYP4F12 SNP Selection, Genotyping, and Haplotype Analysis

Genomic DNA from the liver surgery patients (*n* = 149) was extracted from whole blood using the QIAamp DNA Mini Kit (QIAGEN GmbH, Hilden, Germany) according to the manufacturer’s instructions.

Eighty genetic polymorphisms in the *CYP4F2* gene were selected as previously reported ([Zhang et al., 2009](#B52)). SNPs encompassing *CYP4F11* and *CYP4F12* across the chromosomal 19p13.11 region were chosen on the basis of their functionality, coverage in the CEU population (Utah residents with ancestry from northern and western Europe) available on HapMap data release 27, NCBI build 36 assembly, minor allele frequency (MAF > 1%) and block-tagging ability (*r*^2^ ≥ 0.8). A total of 130 SNPs in the *CYP4F11* and *CYP4F12* region were successfully designed and subdivided into six multiplex assays using Sequenom’s online Human GenoTyping Tools^1^. Primer sequences are available on request. All SNPs were genotyped using the Sequenom MassARRAY iPLEX^TM^ platform (Sequenom, Hamburg, Germany) in accordance with the manufacturer’s instructions. To ensure data quality, 10% DNA replicates and 8 negative controls (water) were included per 384-well plate during genotyping. Markers which deviated from Hardy–Weinberg equilibrium (HWE, *P* < 0.001) (*n* = 10), those with less than 90% call rate (*n* = 27), and those which were monomorphic (*n* = 26), were excluded from downstream analysis (see Supplementary Table [1](#SM1)).

The pattern of pairwise LD between the SNPs was visualized using the program HaploView version 4.2 ([Barrett et al., 2005](#B2)). Haplotype blocks were defined using the default algorithm by [Gabriel et al. (2002)](#B15) in HaploView. The most probable combinations of haplotype-pairs at each block were inferred using the program PHASE version 2.1.1 ([Stephens et al., 2001](#B42); [Stephens and Scheet, 2005](#B41)). Any individuals with a haplotype-pair probability of <90% (*n* = 11) for at least one haplotype block were excluded from tests of association. Within a haplotype block, haplotypes with frequencies <1% were grouped together as a single covariate for analysis.

#### Warfarin-Treated Patient Cohort: Genome-Wide Genotyping and Imputation

Genomic DNA was extracted from whole blood using the standard phenol–chloroform method. Genome-wide genotyping was performed using the Illumina Human610-Quad BeadChip (Illumina, San Diego, CA, United States) at the Wellcome Trust Sanger Institute, United Kingdom. Of the 1000 patients recruited, genome-wide genotype data were available for 752 individuals as previously described ([Bourgeois et al., 2016](#B5)). All quality control measures were performed using PLINK ([Purcell et al., 2007](#B36)). All SNPs with a genotyping success rate <95%, HWE threshold of *P* < 0.0001 and those with MAF <1% were excluded from the dataset. Cryptic relatedness was assessed between individuals and one individual from each pair with an estimated identity by descent (IBD) >0.1875 (i.e., halfway between the expected IBD for third- and second-degree relatives) was removed. Subjects with genotyping success rate <95% were also removed. Principle component analysis was performed to assess genetic markers for ethnicity. Only individuals with genetically matching ethnicity were included into the association analysis (*n* = 711).

For the purpose of this study, genotype data for SNPs across *CYP4F2*, *CYP4F11*, and *CYP4F12* were extracted (*n* = 80) for downstream analyses. After pre-phasing via SHAPEIT ([Delaneau et al., 2012](#B12)), imputation of genotypes at additional SNPs throughout the region encompassing *CYP4F2*, *CYP4F11*, and *CYP4F12* (chr19:15–17 Mb, B36) was carried out using IMPUTE2 ([Howie et al., 2009](#B21)) with the reference genotype data from the European 1000 Genomes Phase I data set (release date June 2014). Imputed variants with an information score <0.8, MAF <1%, HWE <0.0001 and genotyping success rate <95% were excluded using QCTool, leaving a total of 1400 SNPs harboring the *CYP4F12-CYP4F2-CYP4F11* genomic region.

### Statistical Analysis

Statistical analyses were conducted with the software package SPSS, version 18. For each univariate test of association, two tests were performed, one making no assumption on the mode of inheritance while the other assumed an additive mode of inheritance. The minimum *P*-value is referred to in each analysis.

All *P*-values from the genotype–phenotype association tests undertaken in the functional and clinical studies were independently adjusted for multiple testing using false discovery rate (FDR) ([Benjamini et al., 2001](#B3)) in the genetics package of R, version 3.1.2^2^. FDR-corrected *P*-values are denoted as *Pc*-values and values <0.05 were regarded as statistically significant.

The proportion of variability explained by the genetic covariates was calculated using Nagelkerke’s *R*^2^ statistic ([Nagelkerke, 1991](#B31)).

#### Liver mRNA Analysis

Relationships between each of the phenotypic parameters evaluated were examined by Spearman correlation analysis. The mRNA levels of the four *CYP4F* genes were not normally distributed. To enable the use of parametric statistical tests, the expression data were natural log transformed. To evaluate the association of each SNP or haplotype with mRNA expression levels, one-way analysis of variance (ANOVA) and univariate linear regression were conducted.

#### Warfarin Outcome Analysis

Warfarin stable dose was defined as an unchanged daily dose at three or more consecutive clinic visits where INR measurements were within the individual’s target range ([Jorgensen et al., 2009](#B22); [Zhang et al., 2009](#B52)). As the distribution of stable dose was skewed, the outcome was log transformed to achieve normal distribution. To test for the association of SNPs with warfarin stable dose, ANOVA and univariate linear regression were employed. Conditional analysis was conducted by including the SNP of interest into the linear regression model as a covariate. Dosing algorithms were built by incorporating significant (*P* ≤ 0.05) clinical and genetic variables from the univariate analyses into the multiple linear regression models. Supplementary Table [3](#SM3) reports the significant results of the univariate analyses. To assess the predictive accuracy of the dosing algorithms, the mean absolute error was determined by calculating the average of the difference between the predicted and actual stable doses. The percentage of predicted dose which fell within 20% of the actual maintenance dose was also calculated.

### In Silico Analysis to Identify SNPs Associated with mRNA Expression

Putative expression quantitative trait loci (eQTLs) in the *CYP4F* gene cluster were identified using the eQTL browser^3^, a database that summarizes results from large-scale studies which identified eQTLs in the liver ([Schadt et al., 2008](#B37)), brain ([Myers et al., 2007](#B30)), fibroblasts ([Dimas et al., 2009](#B13)), T-cells ([Dimas et al., 2009](#B13)), monocytes ([Zeller et al., 2010](#B51)), and lymphoblastoid cell lines ([Stranger et al., 2007](#B43); [Veyrieras et al., 2008](#B45); [Dimas et al., 2009](#B13); [Montgomery et al., 2010](#B29); [Pickrell et al., 2010](#B35)).

### Network Building to Evaluate Interaction between CYP4F2, CYP4F11, and CYP4F12

The interactions between CYP4F2, CYP4F11, and CYP4F12 were visualized in MetaCore^TM^ (GeneGo, Inc., St. Joseph, MI, United States), an interactive database derived from manually curated literature publications on proteins and small molecules of biological relevance in humans. Results are discussed in the Supplementary Material (Supplementary Figure [3](#SM7), Supplementary Results, and Supplementary Discussion).

## Results

### Correlation between Hepatic CYP4F mRNA Expression Levels

The mRNA levels of the four *CYP4F* genes that were detected by specific probes varied considerably between individuals, ranging from an expression ratio of 2 for *CYP4F8* to an expression ratio of 37 for *CYP4F12* (**Table 1**). Significant correlations among the four *CYP4F* genes are depicted in **Figure 1**. *CYP4F11* and *CYP4F12* mRNA showed significant albeit not very strong correlations with *CYP4F2* mRNA (*r*_s_ = 0.25 and 0.384, respectively, *P* < 0.01; **Figures 1A,B**) and with each other (*r*_s_ = 0.3, *P* < 0.001; **Figure 1D**). *CYP4F8* expression was not significantly correlated to any of the others.

### Table 1.

| Gene | n | mRNA expression (arbitrary unit) |  |
| --- | --- | --- | --- |
|   |   | Range | Ratio (maximum/minimum) |
| CYP4F2 | 149 | 0.21–2.22 | 10 |
| CYP4F8 | 149 | 0.67–1.38 | 2 |
| CYP4F11 | 149 | 0.35–2.53 | 7 |
| CYP4F12 | 149 | 0.08–2.99 | 37 |

Table 1 Caption: Variability of mRNA expression in the CYP4F gene cluster.

### FIGURE 1.

![FIGURE 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/123e/5449482/3d94d2acf709/fphar-08-00323-g001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5449482_fphar-08-00323-g001.jpg)

Statistically significant CYP4F mRNA correlations in 149 Caucasian human liver tissues. (A) CYP4F2 vs. CYP4F11; (B) CYP4F2 vs. CYP4F12; (C) CYP4F8 vs. CYP4F11; (D) CYP4F11 vs. CYP4F12. vs., versus. The Spearman’s rho correlation coefficient (rs) and P-value for each comparison are given. Log2 transformed expression data are presented.

### Genotype–Phenotype Correlation between CYP4F2 Variants and Hepatic mRNA Expression of the CYP4F Gene Cluster

Associations between *CYP4F2* variants and hepatic mRNA expression of the *CYP4F* gene cluster are summarized in **Table 2**. Contrary to a previous report ([McDonald et al., 2009](#B28)), we found a significant association between rs2108622 and liver *CYP4F2* mRNA expression (**Figure 2C**), with subjects homozygous for the rs2108622 minor T allele showing greater *CYP4F2* expression compared to subjects homozygous for the major C allele (TT = 1.47 ± 0.29, CC = 0.97 ± 0.31, *Pc* = 1.72 × 10^-3^, *R*^2^ = 12.6%). Moreover, several other *CYP4F2* variants were also associated with significantly higher *CYP4F2* expression including rs2189784 (*Pc* = 0.030, *R*^2^ = 7.9%, **Figure 2A**), a SNP located 29 kb downstream of the *CYP4F2* gene.

### Table 2.

|   |   |   | mRNA expression |  |  |  |  |  |  |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
|   |   |   | CYP4F2 |  |  | CYP4F11 |  |  | CYP4F12 |  |  |
| Gene | SNP | Localization | P-value | ↑ or ↓ | R2 (%) | P-value | ↑ or ↓ | R2 (%) | P-value | ↑ or ↓ | R2 (%) |
| SNPs in CYP4F2 region |  |   |   |   |   |   |   |   |   |   |   |
| – | rs2189784 | Downstream of CYP4F2 | 0.030 | ↑ | 7.9 | 0.315 | – | – | 0.031 | ↓ | 8.3 |
| CYP4F2 | rs3093209∗ | 3′ near gene | 1.68 × 10-3 | ↑ | 14.6 | 0.097 | – | – | 0.276 | – | – |
| CYP4F2 | rs2108622 | Exon 11, Missense, Met433Val | 1.72 × 10-3 | ↑ | 12.6 | 6.06 × 10-4 | ↓ | 13.7 | 0.509 | – | – |
| CYP4F2 | rs3093173∗ | Intron 9 | 0.485 | – | – | 0.034 | ↓ | 7.2 | 0.481 | – | – |
| CYP4F2 | rs3093169∗ | Intron 9 | 0.655 | – | – | 0.035 | ↓ | 5.8 | 0.275 | – | – |
| CYP4F2 | rs984692 | Intron 3 | 0.519 | – | – | 0.512 | – | – | 0.125 | – | – |
| SNPs in CYP4F11 and CYP4F12 |  |   |   |   |   |   |   |   |   |   |   |
| CYP4F11 | rs1060467 | 3′ UTR | 0.031 | ↓ | 7.2 | 0.310 | – | – | 0.632 | – | – |
| CYP4F11 | rs12977516∗ | Intron 8 | 0.065 | – | – | 0.043 | ↑ | 6.4 | 0.874 | – | – |
| CYP4F11 | rs1471112 | Intron 8 | 0.830 | – | – | 0.867 | – | – | 0.868 | – | – |
| CYP4F11 | rs11086012 | Intron 8 | 0.891 | – | – | 0.715 | – | – | 0.650 | – | – |
| CYP4F11 | rs7249167 | Intron 4 | 0.925 | – | – | 0.479 | – | – | 0.407 | – | – |
| CYP4F12 | rs17682485 | Intron 3 | 0.455 | – | – | 0.989 | – | – | 0.034 | ↓ | 7.9 |
| CYP4F12 | rs12460703 | Intron 3 | 0.478 | – | – | 0.998 | – | – | 0.016 | ↑ | 8.9 |
| CYP4F12 | rs2074568 | Intron 4 | 0.920 | – | – | 0.429 | – | – | 5.43 × 10-5 | ↑ | 19.5 |
| CYP4F12 | rs10409750 | Intron 5 | 0.692 | – | – | 0.844 | – | – | 3.25 × 10-4 | ↓ | 14.8 |
| CYP4F12 | rs10410357 | Intron 9 | 0.761 | – | – | 0.915 | – | – | 6.09 × 10-3 | ↓ | 10.3 |
| CYP4F12 | rs11879787 | Intron 9 | 0.941 | – | – | 0.805 | – | – | 3.22 × 10-5 | ↑ | 17.9 |
| CYP4F12 | rs627971∗ | Intron 9 | 0.497 | – | – | 0.912 | – | – | 0.032 | ↓ | 7.1 |
| CYP4F12 | rs2886476∗ | 3′ near gene | 0.869 | – | – | 0.861 | – | – | 3.84 × 10-5 | ↑ | 18.4 |

Table 2 Caption: Genotype–phenotype correlation of CYP4F2, CYP4F11, and CYP4F12 SNPs and hepatic mRNA expression.

### FIGURE 2.

![FIGURE 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/123e/5449482/ad94c61631bb/fphar-08-00323-g002.jpg)

Levels of CYP4F2, CYP4F11, CYP4F12 mRNA in normal liver tissue donated from 149 patients in relation to corresponding SNPs across the CYP4F2-CYP4F11 locus. (A,B) rs2189784; (C,D) rs2108622; (E,F) rs1060467. FDR-corrected P-values (Pc) are shown in the upper left corner. Each dot represents an individual and the solid lines represent the mean values.

Interestingly, in addition to being associated with increased *CYP4F2* mRNA expression, rs2108622 demonstrated a significant association with decreased *CYP4F11* mRNA levels (*Pc* = 6.06 × 10^-4^, *R*^2^ = 13.7%, **Figure 2D**) while rs2189784 was significantly associated with lower levels of *CYP4F12* mRNA expression (*Pc* = 0.031, *R*^2^ = 8.3%, **Figure 2B**). No associations were found between *CYP4F2* variants and *CYP4F8* mRNA expression (data not shown).

### Genotype–Phenotype Correlation between CYP4F11 and CYP4F12 SNPs and Hepatic mRNA Expression of the CYP4F Gene Cluster

Looking at the region encompassing the *CYP4F* gene cluster on HapMap database (Supplementary Figure [1](#SM5)), high LD is seen in the *CYP4F12-CYP4F2-CYP4F11* locus, suggesting that SNPs across the *CYP4F11* and *CYP4F12* regions could be associated with mRNA expression of *CYP4F2* and possibly other *CYP4F* gene cluster members. To examine the genetic contribution of variants in *CYP4F11* and *CYP4F12* on the hepatic mRNA expression of the *CYP4F* gene cluster, fine mapping of the *CYP4F11* and *CYP4F12* gene regions was conducted and significant associations are summarized in **Table 2**.

rs1060467, a genetic variant located in the 3′ untranslated region (UTR) of *CYP4F11* demonstrated a significant association with decreased *CYP4F2* mRNA expression (*Pc* = 0.031, *R*^2^ = 7.2%, **Figure 2E**); whilst an opposite trend for increasing *CYP4F11* mRNA expression was observed which was not statistically significant after FDR (*Pc* = 0.310, **Figure 2F**).

Eight SNPs spanning *CYP4F12* were significantly associated with *CYP4F12* mRNA expression. No significant association with *CYP4F8* mRNA expression was observed with any SNPs in the *CYP4F11* or *CYP4F12* region (data not shown).

### Association of Haplotypes in the CYP4F12-CYP4F2-CYP4F11 Region on Hepatic mRNA Expression of the CYP4F Gene Cluster

To explore the complex genetic architecture of *CYP4F* locus containing *CYP4F2*, *CYP4F11* and *CYP4F12*, haplotypes across these three genes were constructed based on the genotype data. Ten haplotype blocks were identified as shown in **Figure 3**, with details of haplotypes inferred and their estimated frequencies. Effects of *CYP4F2*, *CYP4F11*, and *CYP4F12* haplotypes on hepatic mRNA expression of the *CYP4F* gene cluster were evaluated and significant associations are reported in **Table 3**.

### FIGURE 3.

![FIGURE 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/123e/5449482/3573513887ae/fphar-08-00323-g003.jpg)

Pairwise LD among polymorphisms in the region of CYP4F12-CYP4F2-CYP4F11 genes in 149 Caucasian samples (r2 ≥ 0.9). The left panel shows 10 distinct haplotype blocks defined by the confidence interval algorithm in HaploView 4.2 and the strength of LD is shown in increasing shades of gray, as depicted by the bars. The right panel details the haplotypes sequences and their frequencies inferred by PHASE 2.1.

### Table 3.

| Gene |   |   |   | mRNA expression |  |  |  |  |  |  |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
|   |   |   |   | CYP4F2 |  |  | CYP4F11 |  |  | CYP4F12 |  |  |
|   | Haplotype identification code (see Figure 3) | SNP code | Haplotype sequence | P-value | ↑ or ↓ | R2 (%) | P-value | ↑ or ↓ | R2 (%) | P-value | ↑ or ↓ | R2 (%) |
| CYP4F12 | 1A | 1112 | AGTG | 0.594 | – | – | 0.864 | – | – | 4.13 × 10-4 | ↓ | 14.4 |
|   | 1C | 2121 | GGAA | 0.894 | – | – | 0.468 | – | – | 4.11 × 10-5 | ↑ | 19.2 |
| CYP4F12 | 2C | 121 | AGT | 0.940 | – | – | 0.804 | – | – | 2.82 × 10-5 | ↑ | 17.9 |
|   | 2D | 212 | GAC | 0.759 | – | – | 0.912 | – | – | 5.60 × 10-3 | ↓ | 10.3 |
| Intergenic | 3D | 1121 | AGTT | 0.332 | – | – | 0.485 | – | – | 2.69 × 10-4 | ↑ | 15.2 |
| CYP4F2 | 4A | 21 | AT | 0.030 | ↑ | 7.9 | 0.313 | – | – | 0.030 | ↓ | 8.3 |
| CYP4F2 | 5A | 121112111 | TGCGGTGGG | 1.71 × 10-3 | ↑ | 12.6 | 5.96 × 10-4 | ↓ | 13.8 | 0.595 | – | – |
| CYP4F2 | 6B | 22 | AA | 0.653 | – | – | 0.033 | ↓ | 5.8 | 0.274 | – | – |
|   | 6C | 21 | AG | 0.352 | – | – | 0.138 | – | – | 0.864 | – | – |
| CYP4F2 | 7D | 11211 | CGGCC | 0.496 | – | – | 0.095 | – | – | 0.604 | – | – |
| CYP4F11 | 8B | 112 | TGC | 0.029 | ↓ | 7.0 | 0.350 | – | – | 0.629 | – | – |
|   | 8D | 121 | TAT | 0.027 | ↑ | 8.8 | 0.054 | – | – | 0.997 | – | – |
| CYP4F11 | 9B | 22 | AG | 0.829 | – | – | 0.865 | – | – | 0.867 | – | – |
|   | 9C | 21 | AT | 0.026 | ↑ | 9.4 | 0.216 | – | – | 0.385 | – | – |

Table 3 Caption: Genotype–phenotype correlation of CYP4F2, CYP4F11, and CYP4F12 haplotypes and hepatic mRNA expression.

Haplotype 4A harboring the sequence ‘AT’ with a frequency of 43.3% was associated with a significant increase in hepatic *CYP4F2* (*Pc* = 0.030, *R*^2^ = 7.9%; **Figure 4A**) and reduced *CYP4F12* (*Pc* = 0.030, *R*^2^ = 8.3%; **Figure 4B**) mRNA expression, mirroring the effect of rs2189784. Corresponding to the effect of rs2108622, haplotype 5A ‘TGCGGTGGG’ (frequency = 28.3%) was significantly associated with increased *CYP4F2* (*Pc* = 1.71 × 10^-3^, *R*^2^ = 12.6%; **Figure 4C**) and decreased *CYP4F11* (*Pc* = 5.96 × 10^-4^, *R*^2^ = 13.8%; **Figure 4D**) mRNA expression. Resembling the effect of rs1060467, haplotype 8B (sequence ‘TGC,’ frequency = 33.2%) was associated with down-regulation of *CYP4F2* (*Pc* = 0.029, *R*^2^ = 7.0%; **Figure 4E**) and showed a non-significant up-regulating effect on *CYP4F11* (*Pc* = 0.350; **Figure 4F**) mRNA expression.

### FIGURE 4.

![FIGURE 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/123e/5449482/d0937e7346ad/fphar-08-00323-g004.jpg)

Levels of hepatic CYP4F2, CYP4F11, CYP4F12 mRNA in normal liver tissue donated from 149 patients in relation to haplotypes across the CYP4F2-CYP4F11 locus. (A,B) Haplotype 4A harboring sequence ‘AT’; (C,D) Haplotype 5A harboring sequence ‘TGCGGTGGG’; (E,F) Haplotype 8B harboring sequence ‘TGC.’ FDR-corrected P-values (Pc) are shown in the upper left corner. Each dot represents an individual and the solid lines represent the mean values.

### Competing Effects of CYP4F11 rs1060467 and CYP4F2 rs2108622 on Warfarin Stable Dose

Linkage disequilibrium analysis of genotypes in our 149 livers revealed that *CYP4F11* rs1060467 and *CYP4F2* rs2108622 were moderately correlated with LD estimates of *r*^2^/D′ = 0.21/1.00. To assess the roles of rs1060467 and rs2108622 in warfarin response, we tested their association with the clinical outcome of warfarin stable dose in our prospective cohort of warfarin-treated patients (*n* = 711). Demographics of the 711 patients are summarized in **Table 4**. Among the 711 patients investigated, 345 achieved warfarin stable dose. **Figure 5A** illustrates warfarin stable dose established in patients, stratified by *CYP4F11* rs1060467 genotype. Patients with a C allele exhibited reduced stable dose requirements (mg/day: TT = 4.6 ± 0.2, TC = 3.9 ± 0.1, CC = 3.8 ± 0.2; *Pc* = 0.017). The proportion of warfarin dose variability explained by rs1060467 was 2.6%. Conversely, as depicted in **Figure 5B**, patients carrying the *CYP4F2* rs2108622 T allele showed increased warfarin stable dose requirements (mg/day: CC = 3.7 ± 0.1, CT = 4.3 ± 0.2, TT = 5.3 ± 0.4; *Pc* = 0.003) and rs2108622 accounted for 4.3% of warfarin dose variance.

### Table 4.

| Characteristic | N (%) |
| --- | --- |
| Gender: male | 394 (55.4) |
| Agea in years, mean (range) | 69 (19-95) |
| BMIb, mean (range) | 28 (13-55) |
| Ethnicity |   |
| White | 710 (99.9) |
| Black-Caribbean | 1 (0.1) |
| Indication for warfarin |   |
| Atrial fibrillation | 469 (66.0) |
| Pulmonary embolism | 110 (15.5) |
| Deep vein thrombosis | 74 (10.4) |
| Cerebrovascular accident and transient ischemic attacks | 44 (6.2) |
| Mechanical heart valve replacement | 8 (1.1) |
| Myocardial infarction | 4 (0.6) |
| Dilated left atrium | 2 (0.3) |
| Otherc | 36 (5.1) |
| Co-morbidity |   |
| Cardiovascular disease | 574 (80.7) |
| Musculoskeletal problems | 426 (59.9) |
| Respiratory disease | 268 (37.7) |
| Gastrointestinal disease | 253 (35.6) |
| Neurological disease | 156 (21.9) |
| Urological condition | 132 (18.6) |
| Renal disease | 75 (10.5) |
| History of falls | 58 (8.2) |
| Hepatic disease | 34 (4.8) |

Table 4 Caption: Clinical profile of 711 warfarin patients.

### FIGURE 5.

![FIGURE 5](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/123e/5449482/4c9a45141605/fphar-08-00323-g005.jpg)

Box and whisker plots showing the distribution of stable warfarin daily doses based on genotype groups in (A) CYP4F11 rs1060467; (B) CYP4F2 rs2108622; (C) intergenic rs7248867; (D) CYP4F12 rs2074568. Boxes represent 25th–75th percentiles of warfarin doses, whiskers represent 5th–95th percentiles, and solid lines represent median dose in each group. Open dots represent outliers. FDR-corrected P-values (Pc) are shown on the upper right corner. Out of the 711 patients recruited prospectively, 345 achieved warfarin stable dose.

By segregating the patients according to their haplotypes for *CYP4F2* rs2108622 and *CYP4F11* rs1060467 as illustrated in **Table 5**, it can be seen that there were small dose changes in patients carrying haplotypes consisting of *CYP4F2* rs2108622 wild-type genotype and *CYP4F11* rs1060467 variant genotype and vice versa.

### Table 5.

| Warfarin dose (mg/day) |  |  |  |
| --- | --- | --- | --- |
| CYP4F2 rs2108622 genotype |  | CYP4F11 rs1060467 genotype |  |
|   |   | TT (n) | CT and CC (n) |
| CC (n) | 3.7 (162) | 4.0 (18) | 3.7 (144) |
| CT and TT (n) | 4.4 (183) | 4.7 (92) | 4.0 (91) |

Table 5 Caption: Relationship between CYP4F2 and CYP4F11 SNPs and stable warfarin dose requirement∗.

### Imputation and Conditional Analysis

To explore the presence of additional signals at the *CYP4F* loci, genotype imputations were carried out across the 380 kb genomic region encompassing the *CYP4F12-CYP4F2-CYP4F11* region. Although additional SNPs showed significant associations with warfarin stable dose, the associations with *CYP4F2* rs2108622 and *CYP4F11* rs1060467 remained the most significant among all the *CYP4F2* and *CYP4F11* SNPs, respectively.

We also performed conditional analyses to evaluate the independence of association between *CYP4F2* rs2108622 and *CYP4F11* rs1060467. When we conditioned on *CYP4F11* rs1060467, a reduction in both magnitude and significance was seen with the association of warfarin stable dose with *CYP4F2* rs2108622 (β_initial_ = 0.078, β_conditional_ = 0.063, *P*_cinitial_ = 0.003, *P*_cconditional_ = 0.05). When we conditioned on *CYP4F2* rs2108622, the association of warfarin stable dose with *CYP4F11* rs1060467 disappeared (*P*_cinitial_ = 0.017, *P*_cconditional_ = 0.418). These results suggest that *CYP4F2* rs2108622 can explain the association signal for *CYP4F11* rs1060467 or vice versa.

### Warfarin Dose Prediction Algorithms

To assess whether the inclusion of *CYP4F2* rs2108622 and/or *CYP4F11* rs1060467 improves warfarin dose predictive accuracy, we developed a clinical algorithm and several pharmacogenetic algorithms as shown in **Table 6**. The clinical algorithm included four predictors which were found significant in the univariate analyses: age, BMI, gender and baseline clotting factor II level, which explained 15.7% of warfarin dose variability. The pharmacogenetic algorithm included *CYP2C9∗2* rs1799853, *CYP2C9∗3* rs1507910, and *VKORC1* rs9923231 genotypes in addition to the clinical factors and accounted for a 32.3% increase in warfarin dose variability, with a marked improvement in dose prediction accuracy. The addition of *CYP4F2* rs2108622 or *CYP4F11* rs1060467 to the pharmacogenetic algorithm explained a further 0.5–0.7% in warfarin dose variability with a modest increase in prediction accuracy (∼1% decrease in MAE and 1.2% increase in the number of predicted dose which fell within ±20% of the observed warfarin dose). When both rs2108622 and rs1060467 were incorporated into the pharmacogenetic algorithm, there was a modest increase in the number of predicted doses which fell within ±20% of the observed warfarin dose (∼2%).

### Table 6.

| Prediction algorithm∗ | Variables included | MAE ± SE (mg/week) | R2 Adj (%) | Within ± 20% of observed dose (%) |
| --- | --- | --- | --- | --- |
| Clinical | Age, BMI, Gender, Baseline Factor II | 9.08 ± 1.06 | 15.7 | 43.7 |
| Clinical + CYP2C9∗2 + CYP2C9∗3 + VKORC1 | Age, BMI, Gender, Baseline Factor II, rs1799853, rs1057910, rs9923231 | 7.32 ± 0.88 | 48.0 | 47.3 |
| Clinical + CYP2C9∗2 + CYP2C9∗3 + VKORC1 + CYP4F2 | Age, BMI, Gender, Baseline Factor II, rs1799853, rs1057910, rs9923231, rs2108622 | 7.26 ± 0.88 | 48.5 | 48.5 |
| Clinical + CYP2C9∗2 + CYP2C9∗3 + VKORC1 + CYP4F11 | Age, BMI, Gender, Baseline Factor II, rs1799853, rs1057910, rs9923231, rs1060467 | 7.24 ± 0.88 | 48.7 | 48.5 |
| Clinical + CYP2C9∗2 + CYP2C9∗3 + VKORC1 + CYP4F2 + CYP4F11 | Age, BMI, Gender, Baseline Factor II, rs1799853, rs1057910, rs9923231, rs2108622, rs1060467 | 7.22 ± 0.88 | 48.6 | 49.4 |

Table 6 Caption: Comparison of predicted stable warfarin doses to actual stable warfarin doses using different prediction algorithms.

### In Silico Genotype–Phenotype Analysis

To explore whether the SNP-gene effects observed in our cohort of human livers was also present in other published studies, we assessed eQTLs in the region encompassing *CYP4F2*, *CYP4F11*, and *CYP4F12* genes using the publicly available eQTL database hosted by the Pritchard laboratories at the University of Chicago. **Table 7** outlines the significant SNP-gene associations available on the eQTL database. Of particular interest is the positive association of rs7248867, a SNP located between *CYP4F12* and *CYP4F2*, with *CYP4F11* transcript levels in livers from individuals of European descent. Using genotype data available on HapMap, LD analysis revealed that this intergenic SNP is in moderate LD with both rs2189784 (*D*′ = 1.0, *r*^2^ = 0.103) and rs2108622 (*D*′ = 1.0, *r*^2^ = 0.046). rs7248867 also tags several SNPs (using *r*^2^ > 0.8) including a *CYP4F12* intronic SNP, rs2074568 (*D*′ = 1.0, *r*^2^ = 0.837) which was analyzed in our cohort of 149 individuals who had donated liver samples. rs2074568 showed a significant association with increased hepatic *CYP4F12* mRNA expression (*Pc* = 1.49 × 10^-5^) but not with *CYP4F11* (*P* = 0.25) and *CYP4F2* (*P* = 0.537).

### Table 7.

| SNP | SNP localization | SNP chromosomal location | Target eQTL gene | P-value | Tissue | Study |
| --- | --- | --- | --- | --- | --- | --- |
| rs7246556 | 5′ upstream of CYP4F12 | 15637511 | SLC35E1 | 9.75439E-05 | Monocytes | [52] |
| rs4808351 | 5′ upstream of CYP4F12 | 15638714 | SLC35E1 | 6.21012E-05 | Monocytes | [52] |
| rs4807967 | 5′ upstream of CYP4F12 | 15638931 | SLC35E1 | 6.21012E-05 | Monocytes | [52] |
| rs4808352 | 5′ upstream of CYP4F12 | 15638996 | SLC35E1 | 0.00011855 | Monocytes | [52] |
| rs10409673 | 5′ upstream of CYP4F12 | 15640453 | SLC35E1 | 6.21012E-05 | Monocytes | [52] |
| rs7251084 | 5′ upstream of CYP4F12 | 15641041 | SLC35E1 | 6.21012E-05 | Monocytes | [52] |
| rs7259028 | 5′ upstream of CYP4F12 | 15641245 | SLC35E1 | 6.21012E-05 | Monocytes | [52] |
| rs7248867 | Intergenic, between CYP4F12 and CYP4F2 | 15731204 | CYP4F11 | 8.23E-05 | Liver | [49] |
| rs2074901 | CYP4F2 | 15858422 | BRD4 | 1.80053E-05 | Monocytes | [52] |
| rs2074902 | CYP4F2 | 15869099 | BRD4 | 1.80053E-05 | Monocytes | [52] |
| rs1060463 | CYP4F11 | 15886176 | ILVBL | 9.96552E-05 | Monocytes | [52] |
| rs6512075 | CYP4F11 | 15899334 | ILVBL | 9.96552E-05 | Monocytes | [52] |
| rs3746154 | CYP4F11 | 15899390 | ILVBL | 9.96552E-05 | Monocytes | [52] |
| rs3746156 | CYP4F11 | 15896494 | ILVBL | 9.96552E-05 | Monocytes | [52] |
| rs2219358 | CYP4F11 | 15896517 | ILVBL | 9.96552E-05 | Monocytes | [52] |
| rs2305803 | CYP4F11 | 15888067 | ILVBL | 9.96552E-05 | Monocytes | [52] |
| rs17641483 | 5′ upstream of CYP4F11 | 15919371 | CYP4F11 | 6.40767E-13 | Monocytes | [52] |

Table 7 Caption: Expression quantitative trait loci (eQTLs) in the CYP4F12-CYP4F2-CYP4F11 gene cluster region.

### Effect of Intergenic rs7248867 and CYP4F12 rs2074568 on Warfarin Stable Dose

Genotypes from the 1000 genomes project were imputed to evaluate the effect of rs7248867 and rs2074568 on warfarin stable dose. As illustrated in **Figure 5C**, patients carrying the minor rs7248867 T-allele required lower warfarin doses compared to patients carrying the major C-allele (mg/day: CC = 4.3 ± 0.1, CT = 3.7 ± 0.2, TT = 3.2 ± 0.4; *Pc* = 0.009). The association of rs2074568 was not significant after FDR but showed a recessive effect on warfarin dose requirements (**Figure 5D**) with the minor A-allele (mg/day: TT = 4.3 ± 0.1, TA = 3.7 ± 0.2, AA = 3.7 ± 0.6; *Pc* = 0.061).

To assess the independence of these two SNPs to *CYP4F2* rs2108622, conditional analyses were performed. When conditioned on rs7248867, the association of *CYP4F2* rs2108622 with warfarin stable dose decreased in both magnitude and significance (β_initial_ = 0.078, β_conditional_ = 0.065, *P*_cinitial_ = 0.003, *P*_cconditional_ = 0.015). When we conditioned on rs2074568, a reduction in magnitude and significance were also observed with *CYP4F2* rs2108622 (β_conditional_ = 0.069, *P*_cconditional_ = 0.009). These results suggest that rs7248867 and rs2074568 are correlated with *CYP4F2* rs2108622.

## Discussion

To elucidate whether the association between genotype and gene expression reflected *cis*-acting regulatory effects on the *CYP4F* gene cluster, we conducted a comprehensive investigation looking at the effects of *CYP4F2*, *CYP4F11*, and *CYP4F12* polymorphisms on the hepatic expression levels of *CYP4F2, CYP4F8, CYP4F11*, and *CYP4F12* mRNA in a Caucasian population. We report for the first time that SNPs and extended haplotypes in *CYP4F2*, *CYP4F11*, and *CYP4F12* affect the mRNA expression levels of *CYP4F2, CYP4F11*, and *CYP4F12* in human liver tissues and that *CYP4F11* plays a role in warfarin response.

Unlike [McDonald et al. (2009)](#B28), our study observed a significant association between the *CYP4F2* rs2108622 SNP and an increase in *CYP4F2* mRNA expression, explaining over 12% of the variability in *CYP4F2* mRNA expression. This may reflect our larger sample size (*n* = 149) of livers compared with the previous study ([McDonald et al., 2009](#B28)). Consistent with the fact that the *CYP4F* genes are highly homologous and show extensive LD, our data show that SNPs in one *CYP4F* gene can have an effect on the expression of another *CYP4F* gene. In fact, rs2108622 in *CYP4F2* was associated with decreased *CYP4F11* mRNA expression accounting for nearly 14% of *CYP4F11* hepatic mRNA expression. Comparatively, the haplotype harboring this *CYP4F2* variant also displayed similar associations. Conversely, a variant in the 3′ UTR of the *CYP4F11* region, rs1060467, was associated with decreased *CYP4F2* mRNA expression, accounting for 7% of the variability in *CYP4F2* mRNA expression. The *CYP4F11* haplotype comprising the minor rs1060467 C-allele also had a corresponding recessive effect on *CYP4F2* mRNA expression.

Given these mutual genotype–phenotype relationships and the fact that both CYP4F2 and CYP4F11 had been identified as equally efficient vitamin K ω-hydroxylases ([Edson et al., 2013](#B14)), we hypothesized that rs1060467 may play a role in warfarin stable dose. Using our GWAS data previously conducted in 711 prospective patients on warfarin therapy, of which 345 patients achieved warfarin stable dose, rs1060467 explained 2.6% of warfarin dose variability, while rs2108622 accounted for 4.3%, similar to previous reports ([Caldwell et al., 2008](#B10); [Borgiani et al., 2009](#B4); [Perez-Andreu et al., 2009](#B33)). Interestingly, the association of rs1060467 with warfarin dose was opposite to that seen with rs2108622, confirming the compensatory effects *CYP4F2* and *CYP4F11* polymorphisms have on hepatic *CYP4F2* mRNA. However, when conditional analyses were performed using SNP rs2108622, the magnitude and significance level for rs1060467 were substantially attenuated, suggesting that rs1060467 and rs2108622 are dependent loci and are both likely to contribute to the same signal at the *CYP4F2-CYP4F11* region. Indeed, our pharmacogenetic algorithms incorporating *CYP4F11* rs1060467 or *CYP4F2* rs2108622 or both *CYP4F11* rs1060467 and *CYP4F2* rs2108622, explained a similar increase in warfarin dose variability with modest improvement in prediction accuracy (1–2%), indicating that just one of these SNPs can explain the effect on warfarin dose variability. The opposing effects between *CYP4F11* rs1060467 and *CYP4F2* rs2108622 in the *CYP4F* gene cluster do not appear to affect warfarin dose requirement.

Our present study also showed a significant association of rs2189784, a SNP located 30 kb downstream of *CYP4F2*, with differences in mRNA expression of *CYP4F2* and *CYP4F12*. Interestingly, we have previously reported this SNP to play a role in time taken to achieve therapeutic INR in patients on prospective warfarin therapy ([Zhang et al., 2009](#B52)). Likewise, the haplotype containing the minor A-allele of variant rs2189784 (haplotype 4A) was also significantly associated with increasing *CYP4F2* and decreasing *CYP4F12* mRNA expression. These results suggest that the previously observed association between rs2189784 and time to therapeutic INR ([Zhang et al., 2009](#B52)) may be mediated through an effect on *CYP4F2* and *CYP4F12* mRNA and SNPs in *CYP4F12* may affect *CYP4F2* mRNA expression. Evaluation of variants across the *CYP4F12* region however, did not show any SNPs to be associated with *CYP4F2* mRNA expression.

*In silico* eQTL analysis provided further insights into the complexity of the regulation of the *CYP4F* gene cluster. *CYP4F11* mRNA expression was associated with an intergenic SNP between *CYP4F12* and *CYP4F2*, rs7248867. This SNP is tagged by a *CYP4F12* intronic SNP (rs2074568) genotyped in our study. These two SNPs were however, not present on the GWAS platform. Imputations were therefore performed and a trend for reduced warfarin stable dose was seen with these two SNPs. However, our conditional analyses suggest that the association signals found with rs7248867 and rs2074568 could be explained by *CYP4F2* rs2108622.

A limitation of our study is that we did not investigate protein expression levels of the different CYP4F isoforms. The reason for this is that the protein sequences of CYP4F2, CYP4F11 and CYP4F12 share 81–93% similarity ([Hirani et al., 2008](#B20)) and currently available antibodies are likely to exhibit high level of cross-reactivity, decreasing the specificity of protein detection. New technologies such as gene editing could be employed to evaluate the function of these *CYP4F* genes.

## Conclusion

We have effectively examined sequence variations across the three *CYP4F* genes – *CYP4F2*, *CYP4F11*, and *CYP4F12* and their effect on mRNA expression. From a clinical perspective, our data show the complexity of gene–gene interactions, where competing effects of different SNPs within the same gene cluster can cancel out the level of *CYP4F2* mRNA and warfarin daily doses required to maintain anticoagulation. As a result, the overall effect of SNPs in *CYP4F2* and *CYP4F11* on warfarin dose variability is very small in our population. However, in other populations with different linkage patterns the influence of *CYP4F* SNPs may be larger. It is possible that additional variants which are rare and functionally active may be important other than the SNPs genotyped in our study, and resequencing of the *CYP4F2*, *CYP4F11* and *CYP4F12* genes in appropriately phenotyped patients on warfarin may help identify these.

## Author Contributions

MP, UZ, and PD designed the research study; JZ, KK, and SB performed the experiments; JZ, KK, AJ, BF, AA, SB, and UZ analyzed the results; JZ, KK, AA, UZ, and MP wrote the manuscript; all authors read and approved the final manuscript.

## Conflict of Interest Statement

The authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.

## Acknowledgments

We thank the clinicians and research nurses involved in recruiting the patients.

## Supplementary Material

The Supplementary Material for this article can be found online at: [http://journal.frontiersin.org/article/10.3389/fphar.2017.00323/full#supplementary-material](http://journal.frontiersin.org/article/10.3389/fphar.2017.00323/full#supplementary-material)

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Aithal G. P., Day C. P., Kesteven P. J. L., Daly A. K. (1999). Association of polymorphisms in the cytochrome P450 CYP2C9 with warfarin dose requirement and risk of bleeding complications. Lancet 353 717–719. 10.1016/S0140-6736(98)04474-2  [DOI](https://doi.org/10.1016/S0140-6736(98)04474-2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10073515/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Association%20of%20polymorphisms%20in%20the%20cytochrome%20P450%20CYP2C9%20with%20warfarin%20dose%20requirement%20and%20risk%20of%20bleeding%20complications.&author=G.%20P.%20Aithal&author=C.%20P.%20Day&author=P.%20J.%20L.%20Kesteven&author=A.%20K.%20Daly&volume=353&publication_year=1999&pages=717-719&pmid=10073515&doi=10.1016/S0140-6736(98)04474-2&)

2. Barrett J. C., Fry B., Maller J., Daly M. J. (2005). Haploview: analysis and visualization of LD and haplotype maps. Bioinformatics 21 263–265. 10.1093/bioinformatics/bth457  [DOI](https://doi.org/10.1093/bioinformatics/bth457) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15297300/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=Haploview:%20analysis%20and%20visualization%20of%20LD%20and%20haplotype%20maps.&author=J.%20C.%20Barrett&author=B.%20Fry&author=J.%20Maller&author=M.%20J.%20Daly&volume=21&publication_year=2005&pages=263-265&pmid=15297300&doi=10.1093/bioinformatics/bth457&)

3. Benjamini Y., Drai D., Elmer G., Kafkafi N., Golani I. (2001). Controlling the false discovery rate in behavior genetics research. Behav. Brain Res. 125 279–284. 10.1016/S0166-4328(01)00297-2  [DOI](https://doi.org/10.1016/S0166-4328(01)00297-2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11682119/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Behav.%20Brain%20Res.&title=Controlling%20the%20false%20discovery%20rate%20in%20behavior%20genetics%20research.&author=Y.%20Benjamini&author=D.%20Drai&author=G.%20Elmer&author=N.%20Kafkafi&author=I.%20Golani&volume=125&publication_year=2001&pages=279-284&pmid=11682119&doi=10.1016/S0166-4328(01)00297-2&)

4. Borgiani P., Ciccacci C., Forte V., Sirianni E., Novelli L., Bramanti P., et al. (2009). CYP4F2 genetic variant (rs2108622) significantly contributes to warfarin dosing variability in the Italian population. Pharmacogenomics 10 261–266. 10.2217/14622416.10.2.261  [DOI](https://doi.org/10.2217/14622416.10.2.261) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19207028/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=CYP4F2%20genetic%20variant%20(rs2108622)%20significantly%20contributes%20to%20warfarin%20dosing%20variability%20in%20the%20Italian%20population.&author=P.%20Borgiani&author=C.%20Ciccacci&author=V.%20Forte&author=E.%20Sirianni&author=L.%20Novelli&volume=10&publication_year=2009&pages=261-266&pmid=19207028&doi=10.2217/14622416.10.2.261&)

5. Bourgeois S., Jorgensen A., Zhang E. J., Hanson A., Gillman M. S., Bumpstead S., et al. (2016). A multi-factorial analysis of response to warfarin in a UK prospective cohort. Genome Med. 8 2 10.1186/s13073-015-0255-y  [DOI](https://doi.org/10.1186/s13073-015-0255-y) | [PMC free article](/articles/PMC4702374/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26739746/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Med.&title=A%20multi-factorial%20analysis%20of%20response%20to%20warfarin%20in%20a%20UK%20prospective%20cohort.&author=S.%20Bourgeois&author=A.%20Jorgensen&author=E.%20J.%20Zhang&author=A.%20Hanson&author=M.%20S.%20Gillman&volume=8&issue=2&publication_year=2016&pmid=26739746&doi=10.1186/s13073-015-0255-y&)

6. Budnitz D. S., Shehab N., Kegler S. R., Richards C. L. (2007). Medication use leading to emergency department visits for adverse drug events in older adults. Ann. Intern. Med. 147 755–765. 10.7326/0003-4819-147-11-200712040-00006  [DOI](https://doi.org/10.7326/0003-4819-147-11-200712040-00006) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18056659/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann.%20Intern.%20Med.&title=Medication%20use%20leading%20to%20emergency%20department%20visits%20for%20adverse%20drug%20events%20in%20older%20adults.&author=D.%20S.%20Budnitz&author=N.%20Shehab&author=S.%20R.%20Kegler&author=C.%20L.%20Richards&volume=147&publication_year=2007&pages=755-765&pmid=18056659&doi=10.7326/0003-4819-147-11-200712040-00006&)

7. Bylund J., Bylund M., Oliw E. H. (2001). cDNA cloning and expression of CYP4F12, a novel human cytochrome P450. Biochem. Biophys. Res. Commun. 280 892–897. 10.1006/bbrc.2000.4191  [DOI](https://doi.org/10.1006/bbrc.2000.4191) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11162607/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem.%20Biophys.%20Res.%20Commun.&title=cDNA%20cloning%20and%20expression%20of%20CYP4F12,%20a%20novel%20human%20cytochrome%20P450.&author=J.%20Bylund&author=M.%20Bylund&author=E.%20H.%20Oliw&volume=280&publication_year=2001&pages=892-897&pmid=11162607&doi=10.1006/bbrc.2000.4191&)

8. Bylund J., Finnstrom N., Oliw E. H. (1999). Gene expression of a novel cytochrome P450 of the CYP4F subfamily in human seminal vesicles. Biochem. Biophys. Res. Commun. 261 169–174. 10.1006/bbrc.1999.1011  [DOI](https://doi.org/10.1006/bbrc.1999.1011) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10405341/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem.%20Biophys.%20Res.%20Commun.&title=Gene%20expression%20of%20a%20novel%20cytochrome%20P450%20of%20the%20CYP4F%20subfamily%20in%20human%20seminal%20vesicles.&author=J.%20Bylund&author=N.%20Finnstrom&author=E.%20H.%20Oliw&volume=261&publication_year=1999&pages=169-174&pmid=10405341&doi=10.1006/bbrc.1999.1011&)

9. Bylund J., Hidestrand M., Ingelman-Sundberg M., Oliw E. H. (2000). Identification of CYP4F8 in human seminal vesicles as a prominent 19-hydroxylase of prostaglandin endoperoxides. J. Biol. Chem. 275 21844–21849. 10.1074/jbc.M001712200  [DOI](https://doi.org/10.1074/jbc.M001712200) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10791960/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem.&title=Identification%20of%20CYP4F8%20in%20human%20seminal%20vesicles%20as%20a%20prominent%2019-hydroxylase%20of%20prostaglandin%20endoperoxides.&author=J.%20Bylund&author=M.%20Hidestrand&author=M.%20Ingelman-Sundberg&author=E.%20H.%20Oliw&volume=275&publication_year=2000&pages=21844-21849&pmid=10791960&doi=10.1074/jbc.M001712200&)

10. Caldwell M. D., Awad T., Johnson J. A., Gage B. F., Falkowski M., Gardina P., et al. (2008). CYP4F2 genetic variant alters required warfarin dose. Blood 111 4106–4112. 10.1182/blood-2007-11-122010  [DOI](https://doi.org/10.1182/blood-2007-11-122010) | [PMC free article](/articles/PMC2288721/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18250228/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=CYP4F2%20genetic%20variant%20alters%20required%20warfarin%20dose.&author=M.%20D.%20Caldwell&author=T.%20Awad&author=J.%20A.%20Johnson&author=B.%20F.%20Gage&author=M.%20Falkowski&volume=111&publication_year=2008&pages=4106-4112&pmid=18250228&doi=10.1182/blood-2007-11-122010&)

11. Cui X., Nelson D. R., Strobel H. W. (2000). A novel human cytochrome P450 4F isoform (CYP4F11): cDNA cloning, expression, and genomic structural characterization. Genomics 68 161–166. 10.1006/geno.2000.6276  [DOI](https://doi.org/10.1006/geno.2000.6276) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10964514/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genomics&title=A%20novel%20human%20cytochrome%20P450%204F%20isoform%20(CYP4F11):%20cDNA%20cloning,%20expression,%20and%20genomic%20structural%20characterization.&author=X.%20Cui&author=D.%20R.%20Nelson&author=H.%20W.%20Strobel&volume=68&publication_year=2000&pages=161-166&pmid=10964514&doi=10.1006/geno.2000.6276&)

12. Delaneau O., Marchini J., Zagury J. F. (2012). A linear complexity phasing method for thousands of genomes. Nat. Methods 9 179–181. 10.1038/nmeth.1785  [DOI](https://doi.org/10.1038/nmeth.1785) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22138821/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Methods&title=A%20linear%20complexity%20phasing%20method%20for%20thousands%20of%20genomes.&author=O.%20Delaneau&author=J.%20Marchini&author=J.%20F.%20Zagury&volume=9&publication_year=2012&pages=179-181&pmid=22138821&doi=10.1038/nmeth.1785&)

13. Dimas A. S., Deutsch S., Stranger B. E., Montgomery S. B., Borel C., Attar-Cohen H., et al. (2009). Common regulatory variation impacts gene expression in a cell type-dependent manner. Science 325 1246–1250. 10.1126/science.1174148  [DOI](https://doi.org/10.1126/science.1174148) | [PMC free article](/articles/PMC2867218/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19644074/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=Common%20regulatory%20variation%20impacts%20gene%20expression%20in%20a%20cell%20type-dependent%20manner.&author=A.%20S.%20Dimas&author=S.%20Deutsch&author=B.%20E.%20Stranger&author=S.%20B.%20Montgomery&author=C.%20Borel&volume=325&publication_year=2009&pages=1246-1250&pmid=19644074&doi=10.1126/science.1174148&)

14. Edson K. Z., Prasad B., Unadkat J. D., Suhara Y., Okano T., Guengerich F. P., et al. (2013). Cytochrome P450-dependent catabolism of vitamin K: omega-hydroxylation catalyzed by human CYP4F2 and CYP4F11. Biochemistry 52 8276–8285. 10.1021/bi401208m  [DOI](https://doi.org/10.1021/bi401208m) | [PMC free article](/articles/PMC3900248/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24138531/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochemistry&title=Cytochrome%20P450-dependent%20catabolism%20of%20vitamin%20K:%20omega-hydroxylation%20catalyzed%20by%20human%20CYP4F2%20and%20CYP4F11.&author=K.%20Z.%20Edson&author=B.%20Prasad&author=J.%20D.%20Unadkat&author=Y.%20Suhara&author=T.%20Okano&volume=52&publication_year=2013&pages=8276-8285&pmid=24138531&doi=10.1021/bi401208m&)

15. Gabriel S. B., Schaffner S. F., Nguyen H., Moore J. M., Roy J., Blumenstiel B., et al. (2002). The structure of haplotype blocks in the human genome. Science 296 2225–2229. 10.1126/science.1069424  [DOI](https://doi.org/10.1126/science.1069424) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12029063/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=The%20structure%20of%20haplotype%20blocks%20in%20the%20human%20genome.&author=S.%20B.%20Gabriel&author=S.%20F.%20Schaffner&author=H.%20Nguyen&author=J.%20M.%20Moore&author=J.%20Roy&volume=296&publication_year=2002&pages=2225-2229&pmid=12029063&doi=10.1126/science.1069424&)

16. Gage B. F., Eby C., Johnson J. A., Deych E., Rieder M. J., Ridker P. M., et al. (2008). Use of pharmacogenetic and clinical factors to predict the therapeutic dose of warfarin. Clin. Pharmacol. Ther. 84 326–331. 10.1038/clpt.2008.10  [DOI](https://doi.org/10.1038/clpt.2008.10) | [PMC free article](/articles/PMC2683977/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18305455/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Use%20of%20pharmacogenetic%20and%20clinical%20factors%20to%20predict%20the%20therapeutic%20dose%20of%20warfarin.&author=B.%20F.%20Gage&author=C.%20Eby&author=J.%20A.%20Johnson&author=E.%20Deych&author=M.%20J.%20Rieder&volume=84&publication_year=2008&pages=326-331&pmid=18305455&doi=10.1038/clpt.2008.10&)

17. Gomes A. M., Winter S., Klein K., Turpeinen M., Schaeffeler E., Schwab M., et al. (2009). Pharmacogenomics of human liver cytochrome P450 oxidoreductase: multifactorial analysis and impact on microsomal drug oxidation. Pharmacogenomics 10 579–599. 10.2217/pgs.09.7  [DOI](https://doi.org/10.2217/pgs.09.7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19374516/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Pharmacogenomics%20of%20human%20liver%20cytochrome%20P450%20oxidoreductase:%20multifactorial%20analysis%20and%20impact%20on%20microsomal%20drug%20oxidation.&author=A.%20M.%20Gomes&author=S.%20Winter&author=K.%20Klein&author=M.%20Turpeinen&author=E.%20Schaeffeler&volume=10&publication_year=2009&pages=579-599&pmid=19374516&doi=10.2217/pgs.09.7&)

18. Hashizume T., Imaoka S., Hiroi T., Terauchi Y., Fujii T., Miyazaki H., et al. (2001). cDNA cloning and expression of a novel cytochrome P450 (CYP4F12) from human small intestine. Biochem. Biophys. Res. Commun. 280 1135–1141. 10.1006/bbrc.2000.4238  [DOI](https://doi.org/10.1006/bbrc.2000.4238) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11162645/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem.%20Biophys.%20Res.%20Commun.&title=cDNA%20cloning%20and%20expression%20of%20a%20novel%20cytochrome%20P450%20(CYP4F12)%20from%20human%20small%20intestine.&author=T.%20Hashizume&author=S.%20Imaoka&author=T.%20Hiroi&author=Y.%20Terauchi&author=T.%20Fujii&volume=280&publication_year=2001&pages=1135-1141&pmid=11162645&doi=10.1006/bbrc.2000.4238&)

19. Hashizume T., Imaoka S., Mise M., Terauchi Y., Fujii T., Miyazaki H., et al. (2002). Involvement of CYP2J2 and CYP4F12 in the metabolism of ebastine in human intestinal microsomes. J. Pharmacol. Exp. Ther. 300 298–304. 10.1124/jpet.300.1.298  [DOI](https://doi.org/10.1124/jpet.300.1.298) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11752129/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Pharmacol.%20Exp.%20Ther.&title=Involvement%20of%20CYP2J2%20and%20CYP4F12%20in%20the%20metabolism%20of%20ebastine%20in%20human%20intestinal%20microsomes.&author=T.%20Hashizume&author=S.%20Imaoka&author=M.%20Mise&author=Y.%20Terauchi&author=T.%20Fujii&volume=300&publication_year=2002&pages=298-304&pmid=11752129&doi=10.1124/jpet.300.1.298&)

20. Hirani V., Yarovoy A., Kozeska A., Magnusson R. P., Lasker J. M. (2008). Expression of CYP4F2 in human liver and kidney: assessment using targeted peptide antibodies. Arch. Biochem. Biophys. 478 59–68. 10.1016/j.abb.2008.06.025  [DOI](https://doi.org/10.1016/j.abb.2008.06.025) | [PMC free article](/articles/PMC2647861/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18662666/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arch.%20Biochem.%20Biophys.&title=Expression%20of%20CYP4F2%20in%20human%20liver%20and%20kidney:%20assessment%20using%20targeted%20peptide%20antibodies.&author=V.%20Hirani&author=A.%20Yarovoy&author=A.%20Kozeska&author=R.%20P.%20Magnusson&author=J.%20M.%20Lasker&volume=478&publication_year=2008&pages=59-68&pmid=18662666&doi=10.1016/j.abb.2008.06.025&)

21. Howie B. N., Donnelly P., Marchini J. (2009). A flexible and accurate genotype imputation method for the next generation of genome-wide association studies. PLoS Genet. 5:e1000529 10.1371/journal.pgen.1000529  [DOI](https://doi.org/10.1371/journal.pgen.1000529) | [PMC free article](/articles/PMC2689936/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19543373/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet.&title=A%20flexible%20and%20accurate%20genotype%20imputation%20method%20for%20the%20next%20generation%20of%20genome-wide%20association%20studies.&author=B.%20N.%20Howie&author=P.%20Donnelly&author=J.%20Marchini&volume=5&issue=e1000529&publication_year=2009&pmid=19543373&doi=10.1371/journal.pgen.1000529&)

22. Jorgensen A. L., Al-Zubiedi S., Zhang J. E., Keniry A., Hanson A., Hughes D. A., et al. (2009). Genetic and environmental factors determining clinical outcomes and cost of warfarin therapy: a prospective study. Pharmacogenet. Genom. 19 800–812. 10.1097/FPC.0b013e3283317ab5  [DOI](https://doi.org/10.1097/FPC.0b013e3283317ab5) | [PMC free article](/articles/PMC3330749/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19752777/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet.%20Genom.&title=Genetic%20and%20environmental%20factors%20determining%20clinical%20outcomes%20and%20cost%20of%20warfarin%20therapy:%20a%20prospective%20study.&author=A.%20L.%20Jorgensen&author=S.%20Al-Zubiedi&author=J.%20E.%20Zhang&author=A.%20Keniry&author=A.%20Hanson&volume=19&publication_year=2009&pages=800-812&pmid=19752777&doi=10.1097/FPC.0b013e3283317ab5&)

23. Kalsotra A., Turman C. M., Kikuta Y., Strobel H. W. (2004). Expression and characterization of human cytochrome P450 4F11: Putative role in the metabolism of therapeutic drugs and eicosanoids. Toxicol. Appl. Pharmacol. 199 295–304. 10.1016/j.taap.2003.12.033  [DOI](https://doi.org/10.1016/j.taap.2003.12.033) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15364545/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Toxicol.%20Appl.%20Pharmacol.&title=Expression%20and%20characterization%20of%20human%20cytochrome%20P450%204F11:%20Putative%20role%20in%20the%20metabolism%20of%20therapeutic%20drugs%20and%20eicosanoids.&author=A.%20Kalsotra&author=C.%20M.%20Turman&author=Y.%20Kikuta&author=H.%20W.%20Strobel&volume=199&publication_year=2004&pages=295-304&pmid=15364545&doi=10.1016/j.taap.2003.12.033&)

24. Kikuta Y., Kato M., Yamashita Y., Miyauchi Y., Tanaka K., Kamada N., et al. (1998). Human leukotriene B4 omega-hydroxylase (CYP4F3) gene: molecular cloning and chromosomal localization. DNA Cell Biol. 17 221–230. 10.1089/dna.1998.17.221  [DOI](https://doi.org/10.1089/dna.1998.17.221) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9539102/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=DNA%20Cell%20Biol.&title=Human%20leukotriene%20B4%20omega-hydroxylase%20(CYP4F3)%20gene:%20molecular%20cloning%20and%20chromosomal%20localization.&author=Y.%20Kikuta&author=M.%20Kato&author=Y.%20Yamashita&author=Y.%20Miyauchi&author=K.%20Tanaka&volume=17&publication_year=1998&pages=221-230&pmid=9539102&doi=10.1089/dna.1998.17.221&)

25. Kikuta Y., Kusunose E., Endo K., Yamamoto S., Sogawa K., Fujii-Kuriyama Y., et al. (1993). A novel form of cytochrome P-450 family 4 in human polymorphonuclear leukocytes. cDNA cloning and expression of leukotriene B4 omega-hydroxylase. J. Biol. Chem. 268 9376–9380.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/8486631/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem.&title=A%20novel%20form%20of%20cytochrome%20P-450%20family%204%20in%20human%20polymorphonuclear%20leukocytes.%20cDNA%20cloning%20and%20expression%20of%20leukotriene%20B4%20omega-hydroxylase.&author=Y.%20Kikuta&author=E.%20Kusunose&author=K.%20Endo&author=S.%20Yamamoto&author=K.%20Sogawa&volume=268&publication_year=1993&pages=9376-9380&pmid=8486631&)

26. Kikuta Y., Miyauchi Y., Kusunose E., Kusunose M. (1999). Expression and molecular cloning of human liver leukotriene B4 omega-hydroxylase (CYP4F2) gene. DNA Cell Biol. 18 723–730. 10.1089/104454999315006  [DOI](https://doi.org/10.1089/104454999315006) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10492403/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=DNA%20Cell%20Biol.&title=Expression%20and%20molecular%20cloning%20of%20human%20liver%20leukotriene%20B4%20omega-hydroxylase%20(CYP4F2)%20gene.&author=Y.%20Kikuta&author=Y.%20Miyauchi&author=E.%20Kusunose&author=M.%20Kusunose&volume=18&publication_year=1999&pages=723-730&pmid=10492403&doi=10.1089/104454999315006&)

27. Lefevre C., Bouadjar B., Ferrand V., Tadini G., Megarbane A., Lathrop M., et al. (2006). Mutations in a new cytochrome P450 gene in lamellar ichthyosis type 3. Hum. Mol. Genet. 15 767–776. 10.1093/hmg/ddi491  [DOI](https://doi.org/10.1093/hmg/ddi491) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16436457/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum.%20Mol.%20Genet.&title=Mutations%20in%20a%20new%20cytochrome%20P450%20gene%20in%20lamellar%20ichthyosis%20type%203.&author=C.%20Lefevre&author=B.%20Bouadjar&author=V.%20Ferrand&author=G.%20Tadini&author=A.%20Megarbane&volume=15&publication_year=2006&pages=767-776&pmid=16436457&doi=10.1093/hmg/ddi491&)

28. McDonald M. G., Rieder M. J., Nakano M., Hsia C. K., Rettie A. E. (2009). CYP4F2 is a vitamin K1 oxidase: an explanation for altered warfarin dose in carriers of the V433M variant. Mol. Pharmacol. 75 1337–1346. 10.1124/mol.109.054833  [DOI](https://doi.org/10.1124/mol.109.054833) | [PMC free article](/articles/PMC2684883/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19297519/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol.%20Pharmacol.&title=CYP4F2%20is%20a%20vitamin%20K1%20oxidase:%20an%20explanation%20for%20altered%20warfarin%20dose%20in%20carriers%20of%20the%20V433M%20variant.&author=M.%20G.%20McDonald&author=M.%20J.%20Rieder&author=M.%20Nakano&author=C.%20K.%20Hsia&author=A.%20E.%20Rettie&volume=75&publication_year=2009&pages=1337-1346&pmid=19297519&doi=10.1124/mol.109.054833&)

29. Montgomery S. B., Sammeth M., Gutierrez-Arcelus M., Lach R. P., Ingle C., Nisbett J., et al. (2010). Transcriptome genetics using second generation sequencing in a Caucasian population. Nature 464 773–777. 10.1038/nature08903  [DOI](https://doi.org/10.1038/nature08903) | [PMC free article](/articles/PMC3836232/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20220756/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Transcriptome%20genetics%20using%20second%20generation%20sequencing%20in%20a%20Caucasian%20population.&author=S.%20B.%20Montgomery&author=M.%20Sammeth&author=M.%20Gutierrez-Arcelus&author=R.%20P.%20Lach&author=C.%20Ingle&volume=464&publication_year=2010&pages=773-777&pmid=20220756&doi=10.1038/nature08903&)

30. Myers A. J., Gibbs J. R., Webster J. A., Rohrer K., Zhao A., Marlowe L., et al. (2007). A survey of genetic human cortical gene expression. Nat. Genet. 39 1494–1499. 10.1038/ng.2007.16  [DOI](https://doi.org/10.1038/ng.2007.16) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17982457/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet.&title=A%20survey%20of%20genetic%20human%20cortical%20gene%20expression.&author=A.%20J.%20Myers&author=J.%20R.%20Gibbs&author=J.%20A.%20Webster&author=K.%20Rohrer&author=A.%20Zhao&volume=39&publication_year=2007&pages=1494-1499&pmid=17982457&doi=10.1038/ng.2007.16&)

31. Nagelkerke N. J. D. (1991). A note on a general definition of the coefficient of determination. Biometrika 78 691–692. 10.1093/biomet/78.3.691  [DOI](https://doi.org/10.1093/biomet/78.3.691) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biometrika&title=A%20note%20on%20a%20general%20definition%20of%20the%20coefficient%20of%20determination.&author=N.%20J.%20D.%20Nagelkerke&volume=78&publication_year=1991&pages=691-692&doi=10.1093/biomet/78.3.691&)

32. Pautas E., Moreau C., Gouin-Thibault I., Golmard J. L., Mahe I., Legendre C., et al. (2010). Genetic factors (VKORC1, CYP2C9, EPHX1, and CYP4F2) are predictor variables for warfarin response in very elderly, frail inpatients. Clin. Pharmacol. Ther. 87 57–64. 10.1038/clpt.2009.178  [DOI](https://doi.org/10.1038/clpt.2009.178) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19794411/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Genetic%20factors%20(VKORC1,%20CYP2C9,%20EPHX1,%20and%20CYP4F2)%20are%20predictor%20variables%20for%20warfarin%20response%20in%20very%20elderly,%20frail%20inpatients.&author=E.%20Pautas&author=C.%20Moreau&author=I.%20Gouin-Thibault&author=J.%20L.%20Golmard&author=I.%20Mahe&volume=87&publication_year=2010&pages=57-64&pmid=19794411&doi=10.1038/clpt.2009.178&)

33. Perez-Andreu V., Roldan V., Anton A. I., Garcia-Barbera N., Corral J., Vicente V., et al. (2009). Pharmacogenetic relevance of CYP4F2 V433M polymorphism on acenocoumarol therapy. Blood 113 4977–4979. 10.1182/blood-2008-09-176222  [DOI](https://doi.org/10.1182/blood-2008-09-176222) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19270263/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Pharmacogenetic%20relevance%20of%20CYP4F2%20V433M%20polymorphism%20on%20acenocoumarol%20therapy.&author=V.%20Perez-Andreu&author=V.%20Roldan&author=A.%20I.%20Anton&author=N.%20Garcia-Barbera&author=J.%20Corral&volume=113&publication_year=2009&pages=4977-4979&pmid=19270263&doi=10.1182/blood-2008-09-176222&)

34. Perini J. A., Struchiner C. J., Silva-Assuncao E., Suarez-Kurtz G. (2010). Impact of CYP4F2 rs2108622 on the stable warfarin dose in an admixed patient cohort. Clin. Pharmacol. Ther. 87 417–420. 10.1038/clpt.2009.307  [DOI](https://doi.org/10.1038/clpt.2009.307) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20182420/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Impact%20of%20CYP4F2%20rs2108622%20on%20the%20stable%20warfarin%20dose%20in%20an%20admixed%20patient%20cohort.&author=J.%20A.%20Perini&author=C.%20J.%20Struchiner&author=E.%20Silva-Assuncao&author=G.%20Suarez-Kurtz&volume=87&publication_year=2010&pages=417-420&pmid=20182420&doi=10.1038/clpt.2009.307&)

35. Pickrell J. K., Marioni J. C., Pai A. A., Degner J. F., Engelhardt B. E., Nkadori E., et al. (2010). Understanding mechanisms underlying human gene expression variation with RNA sequencing. Nature 464 768–772. 10.1038/nature08872  [DOI](https://doi.org/10.1038/nature08872) | [PMC free article](/articles/PMC3089435/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20220758/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Understanding%20mechanisms%20underlying%20human%20gene%20expression%20variation%20with%20RNA%20sequencing.&author=J.%20K.%20Pickrell&author=J.%20C.%20Marioni&author=A.%20A.%20Pai&author=J.%20F.%20Degner&author=B.%20E.%20Engelhardt&volume=464&publication_year=2010&pages=768-772&pmid=20220758&doi=10.1038/nature08872&)

36. Purcell S., Neale B., Todd-Brown K., Thomas L., Ferreira M. A., Bender D., et al. (2007). PLINK: a tool set for whole-genome association and population-based linkage analyses. Am. J. Hum. Genet. 81 559–575. 10.1086/519795  [DOI](https://doi.org/10.1086/519795) | [PMC free article](/articles/PMC1950838/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17701901/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Hum.%20Genet.&title=PLINK:%20a%20tool%20set%20for%20whole-genome%20association%20and%20population-based%20linkage%20analyses.&author=S.%20Purcell&author=B.%20Neale&author=K.%20Todd-Brown&author=L.%20Thomas&author=M.%20A.%20Ferreira&volume=81&publication_year=2007&pages=559-575&pmid=17701901&doi=10.1086/519795&)

37. Schadt E. E., Molony C., Chudin E., Hao K., Yang X., Lum P. Y., et al. (2008). Mapping the genetic architecture of gene expression in human liver. PLoS Biol. 6:e107 10.1371/journal.pbio.0060107  [DOI](https://doi.org/10.1371/journal.pbio.0060107) | [PMC free article](/articles/PMC2365981/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18462017/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Biol.&title=Mapping%20the%20genetic%20architecture%20of%20gene%20expression%20in%20human%20liver.&author=E.%20E.%20Schadt&author=C.%20Molony&author=E.%20Chudin&author=K.%20Hao&author=X.%20Yang&volume=6&issue=e107&publication_year=2008&pmid=18462017&doi=10.1371/journal.pbio.0060107&)

38. Schroder A., Klein K., Winter S., Schwab M., Bonin M., Zell A., et al. (2013). Genomics of ADME gene expression: mapping expression quantitative trait loci relevant for absorption, distribution, metabolism and excretion of drugs in human liver. Pharmacogenomics J. 13 12–20. 10.1038/tpj.2011.44  [DOI](https://doi.org/10.1038/tpj.2011.44) | [PMC free article](/articles/PMC3564008/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22006096/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J.&title=Genomics%20of%20ADME%20gene%20expression:%20mapping%20expression%20quantitative%20trait%20loci%20relevant%20for%20absorption,%20distribution,%20metabolism%20and%20excretion%20of%20drugs%20in%20human%20liver.&author=A.%20Schroder&author=K.%20Klein&author=S.%20Winter&author=M.%20Schwab&author=M.%20Bonin&volume=13&publication_year=2013&pages=12-20&pmid=22006096&doi=10.1038/tpj.2011.44&)

39. Schwanhausser B., Busse D., Li N., Dittmar G., Schuchhardt J., Wolf J., et al. (2011). Global quantification of mammalian gene expression control. Nature 473 337–342. 10.1038/nature10098  [DOI](https://doi.org/10.1038/nature10098) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21593866/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Global%20quantification%20of%20mammalian%20gene%20expression%20control.&author=B.%20Schwanhausser&author=D.%20Busse&author=N.%20Li&author=G.%20Dittmar&author=J.%20Schuchhardt&volume=473&publication_year=2011&pages=337-342&pmid=21593866&doi=10.1038/nature10098&)

40. Sontag T. J., Parker R. S. (2002). Cytochrome P450 omega-hydroxylase pathway of tocopherol catabolism. Novel mechanism of regulation of vitamin E status. J. Biol. Chem. 277 25290–25296. 10.1074/jbc.M201466200  [DOI](https://doi.org/10.1074/jbc.M201466200) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11997390/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem.&title=Cytochrome%20P450%20omega-hydroxylase%20pathway%20of%20tocopherol%20catabolism.%20Novel%20mechanism%20of%20regulation%20of%20vitamin%20E%20status.&author=T.%20J.%20Sontag&author=R.%20S.%20Parker&volume=277&publication_year=2002&pages=25290-25296&pmid=11997390&doi=10.1074/jbc.M201466200&)

41. Stephens M., Scheet P. (2005). Accounting for decay of linkage disequilibrium in haplotype inference and missing-data imputation. Am. J. Hum. Genet. 76 449–462. 10.1086/428594  [DOI](https://doi.org/10.1086/428594) | [PMC free article](/articles/PMC1196397/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15700229/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Hum.%20Genet.&title=Accounting%20for%20decay%20of%20linkage%20disequilibrium%20in%20haplotype%20inference%20and%20missing-data%20imputation.&author=M.%20Stephens&author=P.%20Scheet&volume=76&publication_year=2005&pages=449-462&pmid=15700229&doi=10.1086/428594&)

42. Stephens M., Smith N. J., Donnelly P. (2001). A new statistical method for haplotype reconstruction from population data. Am. J. Hum. Genet. 68 978–989. 10.1086/319501  [DOI](https://doi.org/10.1086/319501) | [PMC free article](/articles/PMC1275651/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11254454/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Hum.%20Genet.&title=A%20new%20statistical%20method%20for%20haplotype%20reconstruction%20from%20population%20data.&author=M.%20Stephens&author=N.%20J.%20Smith&author=P.%20Donnelly&volume=68&publication_year=2001&pages=978-989&pmid=11254454&doi=10.1086/319501&)

43. Stranger B. E., Nica A. C., Forrest M. S., Dimas A., Bird C. P., Beazley C., et al. (2007). Population genomics of human gene expression. Nat. Genet. 39 1217–1224. 10.1038/ng2142  [DOI](https://doi.org/10.1038/ng2142) | [PMC free article](/articles/PMC2683249/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17873874/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet.&title=Population%20genomics%20of%20human%20gene%20expression.&author=B.%20E.%20Stranger&author=A.%20C.%20Nica&author=M.%20S.%20Forrest&author=A.%20Dimas&author=C.%20P.%20Bird&volume=39&publication_year=2007&pages=1217-1224&pmid=17873874&doi=10.1038/ng2142&)

44. Takeuchi F., McGinnis R., Bourgeois S., Barnes C., Eriksson N., Soranzo N., et al. (2009). A genome-wide association study confirms VKORC1, CYP2C9, and CYP4F2 as principal genetic determinants of warfarin dose. PLoS Genet. 5:e1000433 10.1371/journal.pgen.1000433  [DOI](https://doi.org/10.1371/journal.pgen.1000433) | [PMC free article](/articles/PMC2652833/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19300499/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet.&title=A%20genome-wide%20association%20study%20confirms%20VKORC1,%20CYP2C9,%20and%20CYP4F2%20as%20principal%20genetic%20determinants%20of%20warfarin%20dose.&author=F.%20Takeuchi&author=R.%20McGinnis&author=S.%20Bourgeois&author=C.%20Barnes&author=N.%20Eriksson&volume=5&issue=e1000433&publication_year=2009&pmid=19300499&doi=10.1371/journal.pgen.1000433&)

45. Veyrieras J. B., Kudaravalli S., Kim S. Y., Dermitzakis E. T., Gilad Y., Stephens M., et al. (2008). High-resolution mapping of expression-QTLs yields insight into human gene regulation. PLoS Genet. 4:e1000214 10.1371/journal.pgen.1000214  [DOI](https://doi.org/10.1371/journal.pgen.1000214) | [PMC free article](/articles/PMC2556086/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18846210/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet.&title=High-resolution%20mapping%20of%20expression-QTLs%20yields%20insight%20into%20human%20gene%20regulation.&author=J.%20B.%20Veyrieras&author=S.%20Kudaravalli&author=S.%20Y.%20Kim&author=E.%20T.%20Dermitzakis&author=Y.%20Gilad&volume=4&issue=e1000214&publication_year=2008&pmid=18846210&doi=10.1371/journal.pgen.1000214&)

46. Wadelius M., Chen L. Y., Lindh J. D., Eriksson N., Ghori M. J., Bumpstead S., et al. (2009). The largest prospective warfarin-treated cohort supports genetic forecasting. Blood 113 784–792. 10.1182/blood-2008-04-149070  [DOI](https://doi.org/10.1182/blood-2008-04-149070) | [PMC free article](/articles/PMC2630264/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18574025/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=The%20largest%20prospective%20warfarin-treated%20cohort%20supports%20genetic%20forecasting.&author=M.%20Wadelius&author=L.%20Y.%20Chen&author=J.%20D.%20Lindh&author=N.%20Eriksson&author=M.%20J.%20Ghori&volume=113&publication_year=2009&pages=784-792&pmid=18574025&doi=10.1182/blood-2008-04-149070&)

47. Wang M. Z., Saulter J. Y., Usuki E., Cheung Y. L., Hall M., Bridges A. S., et al. (2006). CYP4F enzymes are the major enzymes in human liver microsomes that catalyze the O-demethylation of the antiparasitic prodrug DB289 [2,5-bis(4-amidinophenyl)furan-bis-O-methylamidoxime]. Drug Metab. Dispos. 34 1985–1994. 10.1124/dmd.106.010587  [DOI](https://doi.org/10.1124/dmd.106.010587) | [PMC free article](/articles/PMC2077835/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16997912/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab.%20Dispos.&title=CYP4F%20enzymes%20are%20the%20major%20enzymes%20in%20human%20liver%20microsomes%20that%20catalyze%20the%20O-demethylation%20of%20the%20antiparasitic%20prodrug%20DB289%20%5B2,5-bis(4-amidinophenyl)furan-bis-O-methylamidoxime%5D.&author=M.%20Z.%20Wang&author=J.%20Y.%20Saulter&author=E.%20Usuki&author=Y.%20L.%20Cheung&author=M.%20Hall&volume=34&publication_year=2006&pages=1985-1994&pmid=16997912&doi=10.1124/dmd.106.010587&)

48. Wang M. Z., Wu J. Q., Bridges A. S., Zeldin D. C., Kornbluth S., Tidwell R. R., et al. (2007). Human enteric microsomal CYP4F enzymes O-demethylate the antiparasitic prodrug pafuramidine. Drug Metab. Dispos. 35 2067–2075. 10.1124/dmd.107.016428  [DOI](https://doi.org/10.1124/dmd.107.016428) | [PMC free article](/articles/PMC2364724/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17709372/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab.%20Dispos.&title=Human%20enteric%20microsomal%20CYP4F%20enzymes%20O-demethylate%20the%20antiparasitic%20prodrug%20pafuramidine.&author=M.%20Z.%20Wang&author=J.%20Q.%20Wu&author=A.%20S.%20Bridges&author=D.%20C.%20Zeldin&author=S.%20Kornbluth&volume=35&publication_year=2007&pages=2067-2075&pmid=17709372&doi=10.1124/dmd.107.016428&)

49. Wysowski D. K., Nourjah P., Swartz L. (2007). Bleeding complications with warfarin use: a prevalent adverse effect resulting in regulatory action. Arch. Intern. Med. 167 1414–1419. 10.1001/archinte.167.13.1414  [DOI](https://doi.org/10.1001/archinte.167.13.1414) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17620536/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arch.%20Intern.%20Med.&title=Bleeding%20complications%20with%20warfarin%20use:%20a%20prevalent%20adverse%20effect%20resulting%20in%20regulatory%20action.&author=D.%20K.%20Wysowski&author=P.%20Nourjah&author=L.%20Swartz&volume=167&publication_year=2007&pages=1414-1419&pmid=17620536&doi=10.1001/archinte.167.13.1414&)

50. Yuan H. Y., Chen J. J., Lee M. T., Wung J. C., Chen Y. F., Charng M. J., et al. (2005). A novel functional VKORC1 promoter polymorphism is associated with inter-individual and inter-ethnic differences in warfarin sensitivity. Hum. Mol. Genet. 14 1745–1751. 10.1093/hmg/ddi180  [DOI](https://doi.org/10.1093/hmg/ddi180) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15888487/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum.%20Mol.%20Genet.&title=A%20novel%20functional%20VKORC1%20promoter%20polymorphism%20is%20associated%20with%20inter-individual%20and%20inter-ethnic%20differences%20in%20warfarin%20sensitivity.&author=H.%20Y.%20Yuan&author=J.%20J.%20Chen&author=M.%20T.%20Lee&author=J.%20C.%20Wung&author=Y.%20F.%20Chen&volume=14&publication_year=2005&pages=1745-1751&pmid=15888487&doi=10.1093/hmg/ddi180&)

51. Zeller T., Wild P., Szymczak S., Rotival M., Schillert A., Castagne R., et al. (2010). Genetics and beyond–the transcriptome of human monocytes and disease susceptibility. PLoS ONE 5:e10693 10.1371/journal.pone.0010693  [DOI](https://doi.org/10.1371/journal.pone.0010693) | [PMC free article](/articles/PMC2872668/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20502693/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&title=Genetics%20and%20beyond%E2%80%93the%20transcriptome%20of%20human%20monocytes%20and%20disease%20susceptibility.&author=T.%20Zeller&author=P.%20Wild&author=S.%20Szymczak&author=M.%20Rotival&author=A.%20Schillert&volume=5&issue=e10693&publication_year=2010&pmid=20502693&doi=10.1371/journal.pone.0010693&)

52. Zhang J. E., Jorgensen A. L., Alfirevic A., Williamson P. R., Toh C. H., Park B. K., et al. (2009). Effects of CYP4F2 genetic polymorphisms and haplotypes on clinical outcomes in patients initiated on warfarin therapy. Pharmacogenet. Genom. 19 781–789. 10.1097/FPC.0b013e3283311347  [DOI](https://doi.org/10.1097/FPC.0b013e3283311347) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19741565/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet.%20Genom.&title=Effects%20of%20CYP4F2%20genetic%20polymorphisms%20and%20haplotypes%20on%20clinical%20outcomes%20in%20patients%20initiated%20on%20warfarin%20therapy.&author=J.%20E.%20Zhang&author=A.%20L.%20Jorgensen&author=A.%20Alfirevic&author=P.%20R.%20Williamson&author=C.%20H.%20Toh&volume=19&publication_year=2009&pages=781-789&pmid=19741565&doi=10.1097/FPC.0b013e3283311347&)
