# The impact of CES1 genotypes on the pharmacokinetics of methylphenidate in healthy Danish subjects

## Metadata
**Authors:** Claus Stage, Gesche Jürgens, Louise Schow Guski, Ragnar Thomsen, Ditte Bjerre, Laura Ferrero‐Miliani, Yassine Kamal Lyauk, Henrik Berg Rasmussen, Kim Dalhoff, The INDICES Consortium
**Journal:** British Journal of Clinical Pharmacology
**Date:** 2017 Feb 24
**DOI:** [10.1111/bcp.13237](https://doi.org/10.1111/bcp.13237)
**PMID:** 28087982
**PMCID:** PMC5465325
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5465325/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC5465325/pdf/BCP-83-1506.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC5465325/pdf/BCP-83-1506.pdf)

## Abstract

**Aims:** 
This study investigated the influence of CES1 variations, including the single nucleotide polymorphism (SNP) rs71647871 (G143E) and variation in copy number, on the pharmacokinetics of a single oral dose of 10 mg methylphenidate.

**Methods:** 
CES1 genotype was obtained from 200 healthy Danish Caucasian volunteers. Based on the genotype, 44 (19 males and 25 females) were invited to participate in an open, prospective trial involving six predefined genotypes: three groups with two, three and four CES1 copies, respectively; a group of carriers of the CES1 143E allele; a group of individuals homozygous for CES1A1c (CES1VAR); and a group having three CES1 copies, in which the duplication, CES1A2, had increased transcriptional activity. Plasma concentrations of methylphenidate and its primary metabolites were determined at scheduled time points.

**Results:** 
Median AUC of d‐methylphenidate was significantly larger in the group carrying the 143E allele (53.3 ng ml−1 h−1, range 38.6–93.9) than in the control group (21.4 ng ml−1 h−1, range 15.7–34.9) (P < 0.0001). Median AUC of d‐methylphenidate was significantly larger in the group with four CES1 copies (34.5 ng ml−1 h−1, range 21.3–62.8) than in the control group (P = 0.01) and the group with three CES1 copies (23.8 ng ml−1 h−1, range 15.3–32.0, P = 0.03). There was no difference between the groups with two and three copies of CES1.

**Conclusions:** 
The 143E allele resulted in an increased AUC, suggesting a significantly decreased CES1 enzyme activity. Surprisingly, this was also the case in subjects with homozygous duplication of CES1, perhaps reflecting an undiscovered mutation affecting the activity of the enzyme.

Keywords: carboxylesterase 1, methylphenidate, personalized medicine, pharmacogenetics

### Aims

This study investigated the influence of *CES1* variations, including the single nucleotide polymorphism (SNP) rs71647871 (G143E) and variation in copy number, on the pharmacokinetics of a single oral dose of 10 mg methylphenidate.

### Methods

*CES1* genotype was obtained from 200 healthy Danish Caucasian volunteers. Based on the genotype, 44 (19 males and 25 females) were invited to participate in an open, prospective trial involving six predefined genotypes: three groups with two, three and four *CES1* copies, respectively; a group of carriers of the *CES1* 143E allele; a group of individuals homozygous for *CES1A1c* (*CES1VAR)*; and a group having three *CES1* copies, in which the duplication, *CES1A2*, had increased transcriptional activity. Plasma concentrations of methylphenidate and its primary metabolites were determined at scheduled time points.

### Results

Median AUC of *d*‐methylphenidate was significantly larger in the group carrying the 143E allele (53.3 ng ml^−1^ h^−1^, range 38.6–93.9) than in the control group (21.4 ng ml^−1^ h^−1^, range 15.7–34.9) (*P* < 0.0001). Median AUC of *d*‐methylphenidate was significantly larger in the group with four *CES1* copies (34.5 ng ml^−1^ h^−1^, range 21.3–62.8) than in the control group (*P* = 0.01) and the group with three *CES1* copies (23.8 ng ml^−1^ h^−1^, range 15.3–32.0, *P* = 0.03). There was no difference between the groups with two and three copies of *CES1*.

### Conclusions

The 143E allele resulted in an increased AUC, suggesting a significantly decreased CES1 enzyme activity. Surprisingly, this was also the case in subjects with homozygous duplication of *CES1*, perhaps reflecting an undiscovered mutation affecting the activity of the enzyme.

## What is Already Known about this Subject

## What this Study Adds

## Tables of Links

| TARGETS |
| --- |
| Enzymes 2 |
| Carboxylesterase 1 (CES1) |

Table Caption: These Tables list key protein targets and ligands in this article that are hyperlinked to corresponding entries in http://www.guidetopharmacology.org, the common portal for data from the IUPHAR/BPS Guide to PHARMACOLOGY 1, and are permanently archived in the Concise Guide to PHARMACOLOGY 2015/16 2.

## Introduction

Methylphenidate is a widely prescribed drug for attention deficit hyperactivity disorder (ADHD). From 2003 to 2013 the consumption of methylphenidate in Denmark has increased twentyfold [3](#bcp13237-bib-0003), and the same pattern has been observed in other developed countries [4](#bcp13237-bib-0004), [5](#bcp13237-bib-0005). In the vast majority of formulations, methylphenidate is a racemic mixture consisting of equal amounts of *d*‐ and *l*‐methylphenidate, which are metabolized almost exclusively by carboxylesterase 1 (CES1) to inactive *d*‐ and *l*‐ritalinic acid, respectively. The biologically active compound is *d*‐methylphenidate [6](#bcp13237-bib-0006), [7](#bcp13237-bib-0007).

CES1 is an important enzyme found in large quantities in the human liver. It is estimated to be the tenth most abundant out of 6000 liver enzymes [8](#bcp13237-bib-0008), metabolizing ester‐ and amide‐containing drugs and xenobiotics [9](#bcp13237-bib-0009). The gene encoding CES1 is located on chromosome 16, where four major haplotypes have been reported [10](#bcp13237-bib-0010) (Figure [1](#bcp13237-fig-0001)). Various combinations of the haplotypes give rise to three different diplotypes with two, three or four copies of *CES1*. Two of the haplotypes contain a *CES1*‐related pseudogene (*CES1P1*, also known as *CES1A3* [10](#bcp13237-bib-0010)) which has six exons. *CES1* is subject to duplication, and the duplicated *CES1* variant is called *CES1A2*, while the original *CES1* copy is designated *CES1A1*. The promoter region, exon 1, and the first part of intron 1 in *CES1A2* are homologous to *CES1P1*, but apart from these regions *CES1A1* and *CES1A2* are identical [10](#bcp13237-bib-0010). A number of *CES1A2* haplotypes exist; the most common variant of *CES1A2* containing the *CES1P1* promoter is transcribed to a lesser extent than *CES1A1*, but a variant with increased transcriptional activity has been reported [11](#bcp13237-bib-0011). In addition, there is a variant of *CES1A1* designated *CES1A1c* by Tanimoto *et al.* [12](#bcp13237-bib-0012) (also known as *CES1VAR*) in which exon 1 with flanking sequences has been replaced by the corresponding sequences of *CES1P1*. The *CES1P1*‐derived segment in *CES1A1c* stretches into the Kozak sequence, thus potentially affecting the initiation of the translation.

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7815/5465325/6eee25d9bdd5/BCP-83-1506-g001.jpg)

CES1 gene loci on chromosome 16 spanning about 30 kbp. The four major CES1 haplotypes are designated (A–D). The vertical lines represent the exons of which there are 14. Light and dark green colours indicate regions derived from the original CES1A1 and the pseudogene, CES1P1, respectively

Genetic variants affecting the function of CES1 might be important from a clinical point of view because the enzyme is responsible for metabolizing a wide range of drugs belonging to different therapeutic classes. Some of these medications are prodrugs, that are activated by CES1, including most of the angiotensin converting enzyme (ACE) inhibitors [13](#bcp13237-bib-0013), [14](#bcp13237-bib-0014), dabigatran etexilate [15](#bcp13237-bib-0015), and oseltamivir [16](#bcp13237-bib-0016), while other drugs such as methylphenidate [17](#bcp13237-bib-0017) and clopidogrel [18](#bcp13237-bib-0018) are inactivated by *CES1.*

Variation in *CES1* has the ability to affect the metabolism of methylphenidate, and thus contribute to the substantial individual variation in clinical response encountered when methylphenidate is used in therapy [19](#bcp13237-bib-0019). An example is the non‐synonymous single nucleotide polymorphism (SNP) in *CES1A1*, rs71647871, also known as p.Gly143Glu or G143E, which creates a loss‐of‐function allele with markedly decreased enzymatic activity [20](#bcp13237-bib-0020). This SNP is located in exon 4 and results in a non‐conservative substitution of glycine to glutamic acid. Its impact on the function of CES1 is well documented [21](#bcp13237-bib-0021), [22](#bcp13237-bib-0022), [23](#bcp13237-bib-0023). On the other hand, little is known about how duplication of *CES1* influences drug metabolism.

The objective of this study was to investigate whether specific *CES1* genotypes, including genotypes with duplication of the gene, affect the pharmacokinetic profile of methylphenidate in a population of healthy volunteers.

## Methods

### Selection of subjects and inclusion criteria

Two hundred healthy volunteers recruited from higher educational institutions were enrolled in this study (100 males and 100 females). After being given information about the study design and potential adverse reactions to the study drug, a written consent form was signed by each subject. The study protocol ([ClinicalTrials.gov](http://clinicaltrials.gov) identifier: [NCT02135263](https://clinicaltrials.gov/ct2/show/NCT02135263)) was approved by the Committees on Biomedical Research Ethics for the Capital Region of Denmark (Approval Number H‐1‐2011‐127 dated 10 February 2012) as well as the Danish Health and Medicines Authority. The trial was conducted in accordance with the ICH guideline for Good Clinical Practice and was carried out during the period from 19 November 2012 to 17 October 2013.

The inclusion criteria were: healthy Caucasian males and females aged 18 or above. The exclusion criteria were: pregnancy; known allergy to the study drug; chronic medication apart from birth control devices, ear and eye drops, or creams against dermatitis; chronic disease including heart, liver, and kidney disease as well as high or low blood pressure; diabetes; smoking; and excessive alcohol consumption.

### Screening and genotyping

Saliva samples were obtained from the 200 volunteers, and DNA for *CES1* genotyping was extracted from the samples. The copy number of *CES1*, including the number of copies of *CES1A1* and *CES1A2*, was determined by real‐time duplex PCR. For samples with two *CES1* copies, both *CES1A1*, two overlapping long range PCRs were carried out, allowing the amplification of all *CES1A1* exons. The first of these long PCRs amplified a 12.5 kb fragment containing the promoter and exon 1–5 of the gene. The second amplified a 19.2 kb *CES1A1* fragment containing exon 6–14. For samples with three or four copies, i.e. samples harbouring *CES1A1* as well as *CES1A2*, the 12.5 kb fragment of *CES1A1* and the corresponding *CES1A2* were amplified. The long PCR for amplification of the *CES1A1* fragment containing exon 6–14 does not distinguish *CES1A1* from *CES1A2* and was therefore not applicable for analysis of samples containing both these gene versions. Promoter and coding regions in the amplified fragments were subjected to Sanger sequencing. The sequences of the forward and reverse primers for amplification of the 12.5 kb fragment of *CES1A1* were 5′‐ACTATGGGGGGACGGAGTTCA‐3′ and 5′‐CCAGTCCTGAATTCAGGTATTGTAATCA‐3′. The 12.5 kb fragment of *CES1A2* was amplified using the same reverse primer and a forward primer with the sequence 5′‐CAGGAGCTATTGAGAGATGGAATCAT‐3′. The 19.2 kb fragment of *CES1A1* was amplified using a forward and reverse primer with the sequences 5′‐CTGATTACAATACCTGAATTCAGGAC‐3′ and 5′‐GTATTTCTGCTCATTATGGTCACG‐3′, respectively. The amplifications were performed using Herculase Hotstart DNA Polymerase (Agilent Technologies, Santa Clara, CA, USA). Composition of reaction mixtures and temperature cycling were as recommended by the manufacturer of the DNA polymerase preparation with the exception that reaction mixtures for amplification of the 12.5 kb fragment of *CES1A1* and *CES1A2* were supplemented with an amount of MgCl_2_ that increased the final concentration of this compound by 0.5 mM.

### Selection of subjects for the drug trial

Based on the *CES1* genotype, participants with presumed different metabolism were divided into the following predefined groups: Group 1, a control group (two copies of wild‐type *CES1A1*, i.e. diplotype A/A without any non‐synonymous SNPs in the exons, *n* = 17); group 2, a group with four copies of *CES1* (two with diplotype B/D and three with diplotype B/B, no non‐synonymous SNPs in exon 1–5, *n* = 5); group 3, a group harbouring the 143E allele (four with diplotype A/A and two with A/B, all heterozygous for the mutation, *n* = 6); group 4, a group having three copies of *CES1*, in which the duplication, *CES1A2*, had increased transcriptional activity (*n* = 2); group 5, a group with the *CES1A1c* variant (three with diplotype C/C and one with C/D, *n* = 4); and group 6, a group with three copies of *CES1* in which the duplication, *CES1A2*, had the common promoter with low transcriptional activity (all with diplotype A/B, *n* = 10). Thus, the effect of having two, three or four copies of the gene, as well as having a duplication with increased transcriptional activity, was evaluated. In addition, we investigated the significance of the 143E allele, which has been shown to decrease the activity of CES1, and whether the *CES1A1c* variant is of importance. The genotype distributions of the population were in Hardy–Weinberg equilibrium.

### Clinical study design

An open, prospective, uncontrolled, clinical trial involving the 44 subjects (19 males and 25 females) selected on the basis of *CES1* genotype was undertaken. The subjects were not blinded to their genotype. Study medication was methylphenidate 10 mg in a single dose (Ritalin^®^, Novartis, Basel, Switzerland). All participants were fasting overnight and the study drug was administered 1 h after a standardized breakfast. Two participants received a modified meal due to lactose intolerance and an error in the delivery from the kitchen. Serial blood samples for analysis of plasma concentrations of methylphenidate (*d*‐ and *l*‐methylphenidate) and its metabolites (*d*‐ and *l*‐ritalinic acid) were collected using a 5 ml Na‐Flourid/Na‐Heparin tube (Becton Dickinson) at 0 (pre‐dose), 0.5, 1, 1.5, 2, 2.5, 3, 4, 6, 8, 10, 24 and 33 h after dose administration. After centrifugation of the blood samples for 10 min at 3000 *g* and 4°C, the plasma was transferred to polypropylene screw‐cap tubes and stored at −80°C until analysis of methylphenidate and ritalinic acid levels.

### Pharmacokinetic variables

Non‐compartmental methods were used to determine the pharmacokinetic variables of *d*‐ and *l*‐methylphenidate as well as *d*‐ and *l*‐ritalinic acid plasma concentrations. The pharmacokinetic variables were calculated using the PKSolver extension program (version 2.0) for Microsoft Excel and included *AUC* _0‐inf_, area under the concentration–time curve from time 0 to infinity; *C* _max_, the maximum concentration observed post dose; *t* _max_, time at which *C* _max_ occurs; and *t* _1/2_, terminal elimination half‐life. All AUCs were calculated by the linear‐log trapezoidal method. Plasma concentrations below the limit of quantification (LOQ) were included as zero in the statistical analyses, and sampling times deviating more than 20% from the schedule were not included in the figure showing median concentrations on group level.

### Analytical method

*d,l‐Threo*‐methylphenidate was sourced from Lipomed (Arlesheim, Switzerland), *d,l‐threo*‐ritalinic acid was from Novartis (Basel, Switzerland), while internal standards *d,l‐threo*‐methylphenidate‐d10 and *d,l‐threo*‐ritalinic acid‐d10 were from Toronto Research Chemicals (Toronto, Canada).

The concentrations of *d*‐ and *l*‐*threo* isomers of methylphenidate and ritalinic acid in plasma were quantified by validated chiral high‐pressure liquid chromatography–mass spectrometry using a Quattromicro tandem mass spectrometer (Waters, Milford, MA) operated in positive‐ion electrospray mode and using multiple‐reaction monitoring.

Plasma samples were prepared using solid‐phase extraction as described for whole blood elsewhere [24](#bcp13237-bib-0024), but without the initial protein precipitation step. Lower limit of quantification was 0.205 ng ml^−1^ for methylphenidate isomers and 2.05 ng ml^−1^ for RA isomers. A five‐point calibration curve in spiked plasma in the range 0.205–30.75 ng ml^−1^ for methylphenidate isomers and 2.05–307.5 ng ml^−1^ for RA isomers run in duplicate was used to quantify the analytes. A second order calibration model was employed which provided coefficients of determination (*r* ^2^) better than 0.998 for all four analytes.

Chromatographic separation was performed using the same system as described elsewhere [24](#bcp13237-bib-0024). Briefly, a CHIRALPAK^®^‐AGP column (Sigma‐Aldrich, St. Louis, MO) using isocratic elution was employed to separate the enantiomers of both analytes. The mobile phase consisted of 10 mM ammonium acetate with 0.4% isopropanol and 0.01% formic acid. Column temperature was 30°C and run‐time 8 min.

Precision and accuracy calculated from quality control samples included in each run were ≤9.1% and within ±9.7%, respectively, for all analytes.

### Statistical analyses

Several groups had a small sample size and measurements that were not normally distributed graphically. Accordingly, non‐parametric statistics were chosen and multiple regression with covariates other than genotype was omitted. The Kruskal–Wallis test was used for analysis of variance. In case this analysis produced a significant *P*‐value, the Mann–Whitney test was used for comparisons between the control group and each of the other five groups. For the assessment of the effect of gene duplications, a comparison between the group with four copies of *CES1* and the group with three copies carrying the duplication with the common *CES1A2* promoter was also performed. *P*‐values were not adjusted for multiple comparisons as all statistical comparisons were defined *a priori*. A retrospective power calculation was performed using the formula for minimum sample size determination based on a single mean and standard deviation: *n* > (z_α_ + z_β_)^2^ × σ^2^/(μ − μ_0_)^2^, in which *z* _α_ is the two‐tailed value of *z* related to α (generally α = 0.05), *z* _β_ is the one‐tailed value of *z* related to β (generally β = 0.20), σ is the standard deviation, and μ − μ_0_ is the minimum relevant difference between the AUC of the control group and the other genotype groups [25](#bcp13237-bib-0025).

Statistical analyses were conducted using GraphPad Prism 6.0 (GraphPad Software, San Diego, CA). Differences were considered statistically significant when *P* < 0.05.

## Results

Baseline characteristics of the 43 study participants and the different genotype groups are summarized in Table [1](#bcp13237-tbl-0001). Apart from one subject, who caught a cold with fever, all participants completed the study according to the protocol, and no clinically significant adverse reactions were related to the administration of methylphenidate.

### Table 1.

| Group | Control | 4 copies | G143E | 3 copies active | CES1A1c | 3 copies normal |
| --- | --- | --- | --- | --- | --- | --- |
| Gender (F/M) | 16 (8/8) | 5 (4/1) | 6 (3/3) | 2 (0/2) | 4 (3/1) | 10 (6/4) |
| Age (years) | 24 (21–29) | 24 (20–25) | 23 (22–28) | 22.0 + 22.0a | 23 (21–27) | 24 (22–27) |
| Height (cm) | 171 (160–196) | 175 (165–183) | 178 (174–183) | 189 + 184a | 171 (161–186) | 171 (159–197) |
| Weight (kg) | 66.5 (54.5–104.0) | 70.0 (63.0–77.6) | 69.0 (59.3–80.0) | 63.0 + 71.0a | 64.7 (57.0–77.0) | 61.2 (52.5–80.0) |
| BMI | 24.5 (20.4–31.0) | 24.7 (20.6–25.7) | 21.5 (18.7–24.8) | 17.6 + 21.0a | 22.2 (20.5–24.1) | 21.0 (18.4–22.7) |

Table 1 Caption: Subject characteristics

### Frequency of the evaluated variants of CES1 in a Danish population

In the entire population of 200 Caucasian volunteers, the allele frequency of the haplotype carrying *CES1A2* amounted to 14.7%. Five subjects (2.5%) carried two *CES1A2* (i.e. four copies of *CES1*), and 48 subjects (24.4%) carried one *CES1A2* (i.e. three copies of *CES1*). The promoter variant in *CES1A2* that causes increased transcriptional activity was rare, being found in only two individuals corresponding to a frequency of 0.5% in the population. The frequency of the *CES1A1* chimeric subtype, *CES1A1c*, was 15.5%. The 143E allele was detected in six individuals, who were all heterozygous for this allele, corresponding to an allele frequency of 1.6%. A detailed description of the diversity of the gene encoding CES1 in the overall population including the identification of novel gene variants is the subject of another study.

### Effect of the genetic variations on the metabolism of methylphenidate

The pharmacokinetic results are shown in Table [2](#bcp13237-tbl-0002). In terms of median AUC of *d*‐methylphenidate (Figure [2](#bcp13237-fig-0002)), carrying the 143E allele was associated with a highly significant 149% increase in AUC compared to the control group (*P* < 0.0001). Subjects with four copies of *CES1* had an increased median AUC of *d*‐methylphenidate relative to the control group (61% increase, *P* = 0.011) and the group with three copies of *CES1* carrying the common *CES1A2* promoter (45% increase, *P* = 0.028). There was no difference when comparing the control group and the group with three *CES1* copies carrying the common *CES1A2* (*P* = 0.449). *t* _max_ was not significantly different for any analytes. *C* _max_ was significantly different for both *d*‐methylphenidate (*P* = 0.004) and *d*‐ritalinic acid (*P* = 0.0001) between the group carrying the 143E allele and the control group. In terms of *t* _1/2_, only 143E *vs.* control was significantly different (*P* = 0.0002) for *d*‐methylphenidate.

### Table 2.

| Group | n | AUC0‐inf (ng ml−1 t−1) | C max (ng ml−1) | t max (h) | t 1/2 (h) | AUC d‐RA/AUC d‐MPH |
| --- | --- | --- | --- | --- | --- | --- |
| Dex‐methylphenidate |   |   |   |   |   |   |
| Control | 16 | 21.4 (15.7–34.9) | 5.0 (3.3–8.9) | 2.0 (1.0–3.0) | 2.6 (2.1–3.6) | 28.5 |
| 4 copies | 5 | 34.5 (21.3–62.8)*b | 6.9 (4.6–7.9) | 2.0 (1.0–4.0) | 2.9 (2.6–3.8) | 15.6 |
| G143E | 6 | 53.3 (38.6–93.9)*** | 9.1 (4.7–14.0)** | 1.8 (0.5–4.1) | 4.0 (3.0–4.8)*** | 10.4 |
| 3 copies active | 2 | 24.0 + 24.6a | 3.9 + 7.8a | 1.0 + 4.1a | 2.7 + 2.7a | 24.3 + 25.2a |
| CES1A1c | 4 | 25.2 (14.3–30.4) | 6.4 (3.7–9.7) | 1.8 (1.0–3.1) | 2.6 (2.0–2.6) | 27.2 |
| 3 copies normal | 10 | 23.8 (15.3–32.0) | 5.4 (3.9–7.1) | 1.5 (0.5–2.1) | 2.6 (2.2–3.4) | 27.9 |
| Dex‐ritalinic acid |   |   |   |   |   |   |
| Control | 16 | 608.9 (401.7–863.0) | 69.5 (49.3–95.4) | 2.5 (1.5–4.2) | 8.6 (5.9–10.1) |   |
| 4 copies | 5 | 536.5 (512.1–992.5) | 52.9 (33.4–63.8) | 2.2 (2.0–4.0) | 8.6 (7.5–11.1) |   |
| G143E | 6 | 555.3 (485.9–666.8) | 42.8 (33.8–50.9)*** | 3.1 (2.0–4.1) | 9.4 (8.1–10.3) |   |
| 3 copies active | 2 | 582.9 + 620.9a | 49.0 + 57.3a | 2.0 + 4.1a | 8.6 + 11.3a |   |
| CES1A1c | 4 | 685.5 (618.6–740.3) | 70.5 (57.5–90.3) | 2.0 (2.0–4.1) | 8.0 (7.4–8.7) |   |
| 3 copies normal | 10 | 664.9 (402.8–959.7) | 71.0 (47.2–92.9) | 2.3 (1.2–3.0) | 9.0 (7.0–10.1) |   |
| Levo‐ritalinic acid |   |   |   |   |   |   |
| Control | 16 | 533.9 (406.0–763.4) | 103.2 (72.0–127.9) | 2.3 (1.5–3.1) | 6.9 (2.7–8.0) |   |
| 4 copies | 5 | 547.9 (493.9–577.6) | 82.7 (82.0–111.6) | 2.0 (1.6–4.0) | 5.8 (5.3–6.7) |   |
| G143E | 6 | 534.4 (405.1–627.6) | 92.7 (70.4–131.2) | 2.1 (1.0–2.6) | 6.0 (5.2–7.4) |   |
| 3 copies active | 2 | 436.0 + 606.0a | 75.8 + 89.5a | 1.5 + 4.1a | 7.0 + 7.4a |   |
| CES1A1c | 4 | 614.3 (524.3–643.1) | 114.3 (92.3–140.9) | 1.8 (1.5–3.1) | 6.7 (5.8–7.5) |   |
| 3 copies normal | 10 | 591.0 (334.7–721.4) | 116.7 (75.8–139.8) | 1.8 (1.2–2.1) | 5.8 (3.7–7.5) |   |

Table 2 Caption: Pharmacokinetic parameters on a group level

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7815/5465325/5c6032abc813/BCP-83-1506-g002.jpg)

Median concentrations of d‐methylphenidate in relation to CES1 genotype. Sampling times deviating more than 20% from the schedule are not included in the figure (one plasma sample). Control, two copies of wild‐type CES1; 4 copies, four copies of CES1; G143E, subjects carrying the 143E allele of CES1; 3 copies active, three copies of CES1 in which CES1A2 has increased transcriptional activity; CES1A1c, subjects carrying the CES1A1c variant; 3 copies normal, three copies of CES1, in which CES1A2 possesses the common promoter

Plasma concentrations of *l*‐methylphenidate were below the limit of quantification (LOQ) at every time point for 38 out of 43 participants. Of the five subjects in whom *l*‐methylphenidate was measurable, three participants carried the 143E allele. Compared to the group of controls, the groups with the *CES1A1c* and the *CES1A2* variant with increased transcriptional activity showed no significant differences for any analytes. The group with four copies of *CES1* contains an outlier having a *d*‐methylphenidate AUC of 62.8 ng ml^−1^ t^−1^. Exclusion of this subject in a *post hoc* analysis did not change the statistics, which still showed a significantly larger median (and mean) AUC compared to the control group. The pharmacokinetic data derived from the two subjects, who received a modified meal, were comparable to those receiving the standard meal.

For the retrospective power calculation, we used the mean and standard deviation of *d*‐methylphenidate AUC of the control group (21.86 ng ml^−1^ t^−1^ and 5.27 ng ml^−1^ t^−1^, respectively, data not shown), a significance level of 5%, and a power of 80%. Based on this, we found that the number of subjects in each group required to detect a difference of 30% in the control group AUC was six (*n* > 5.06). The difference of 30% corresponded to a difference in mean AUC of 6.56 ng ml^−1^ t^−1^. To detect a difference of 40%, only three subjects would be required.

## Discussion

The present study investigated the impact of variation in *CES1* on the metabolism of methylphenidate in healthy volunteers. It is the first study to investigate the impact of *CES1* duplication on the metabolism of methylphenidate.

Because of low promoter activity, the transcription of *CES1A2* is approximately 2% of that of *CES1A1* [26](#bcp13237-bib-0026). Consequently, one might expect the presence of *CES1A2* to be without significance for drug metabolism. Surprisingly, we found that four copies of *CES1* increased the AUC of *d*‐methylphenidate compared to both two (control group) and three copies, reflecting a decreased *CES1* activity. There was no difference when comparing the groups with two and three copies. This is in contrast to previously published literature. In a study of cancer patients receiving the cytotoxic drug irinotecan, Sai *et al.* [10](#bcp13237-bib-0010) found a gene‐dose effect of functional *CES1* genes on the formation of the active irinotecan metabolite. Thus, subjects with three or four copies of the gene had an increased *CES1* activity compared to subjects with two copies of the gene. A study by Suzaki *et al.* [27](#bcp13237-bib-0027) found no difference between subjects with three or four copies of *CES1* in the metabolism of oseltamivir in 30 healthy Japanese volunteers. All subjects carried three or four copies of the gene, and therefore an estimate of the effect compared to subjects with no gene duplication is not obtainable. Lastly, a recent *in vitro* study of 102 normal human liver samples showed no significant differences in *CES1* activity between two, three or four functional gene copies in the metabolism of enalapril [28](#bcp13237-bib-0028).

Thus, with regard to duplication of *CES1*, our data diverge from previous studies which examined other substrates. In these studies, duplication had either no impact or an opposite effect to ours [10](#bcp13237-bib-0010), [27](#bcp13237-bib-0027), [28](#bcp13237-bib-0028).

It is possible that our finding of increased AUC in the group with four copies of *CES1* reflects an undiscovered *CES1A1* variant downstream of exon 5 in one or more of the subjects in this group and not an effect of the gene duplication. Due to sequence homology between *CES1A1* and *CES1A2*, sequencing of all exons of *CES1A1* was only possible in individuals with two copies of the gene. In individuals with three and four copies of the gene, a full sequence was only obtained for exons 1–5, and therefore the possibility of clinically significant SNPs downstream of the gene exists. This is supported by a previous study which reported non‐synonymous SNPs with a potential functional impact in *CES1A1* exons downstream of exon 5 [29](#bcp13237-bib-0029).

The outlier in the group with four copies of *CES1* might carry such an undiscovered functional gene variant. However, excluding the outlier from the statistical analysis did not change the significant differences. Thus, apart from a potential functional gene variant, there seems to be other reasons for the decreased enzymatic activity in the group with four copies of *CES1*. Such reasons include accidental differences in lifestyle having an impact on the activity of *CES1*.

The 143E allele was associated with a markedly decreased metabolism of methylphenidate, and this is in accordance with a previous finding [20](#bcp13237-bib-0020). It may not be possible to infer the effect of a change in plasma concentration on an individual level, but a median increase in AUC of 149%, as found in our study, could warrant a dosage reduction due to adverse reactions. Tarkiainen *et al.* found a mean increase in oseltamivir *AUC* _0‐inf_ of 18% and a mean decrease in enalaprilat *AUC* _0‐inf_ of 20% in subjects heterozygous for the 143E allele compared to subjects carrying the wild‐type genotype [22](#bcp13237-bib-0022), [30](#bcp13237-bib-0030). Both results indicated a decreased enzymatic activity. Accordingly, this genetic variant appears to have a larger impact on the pharmacokinetics of methylphenidate than that of enalapril and oseltamivir. The subjects in our study were all heterozygous for the 143E allele, hindering an assessment of a gene‐dosage effect. However, a subject homozygous for this allele has previously been associated with a significantly decreased metabolism of oseltamivir compared to heterozygous subjects, suggesting a gene‐dosage effect [22](#bcp13237-bib-0022).

*A priori*, the group with three copies of *CES1*, in which the duplication, *CES1A2*, had increased transcriptional activity was thought to have an increased metabolism of methylphenidate. In our population, this genetic variant was rare and the pharmacokinetic results of the two subjects with this genotype were alike in terms of AUC of *d*‐methylphenidate but differed in terms of *C* _max_ and *t* _max_. Overall, we found no indication that this promoter enhances the activity of the enzyme.

The *CES1A1c* group showed no pharmacokinetic differences for any analytes compared to the control group. This is consistent with a recent *in vitro* study in which human liver samples from individuals carrying the *CES1A1c* variant (*CES1VAR*) had a 2.6‐fold lower mRNA expression compared to non‐carriers but without detectable changes in protein expression and enzymatic activity [31](#bcp13237-bib-0031).

The first and most important limitation of this study is the small number of individuals in some of the genotype groups, which necessitate a cautious interpretation of the results. Studies estimating the impact of genetic polymorphism of genes on clinical or paraclinical outcomes are generally challenged by the fact that the most relevant alleles only occur at low frequencies. The small number of participants in our groups makes it difficult to correct for the impact of covariates like gender, age and BMI in our statistical analyses, which could have improved the ability to detect the effect of the genetic variants. However, our retrospective power calculation showed that a small subject number in each group is sufficient to detect a difference of 30% or more in *d*‐methylphenidate AUC between our groups.

Secondly, we only used a single dose of methylphenidate. This hampers a direct translation of our results to the clinical environment, in which multiple dosage is the norm. A future study investigating the impact of *CES1* genotypes on multiple dosage regimens is needed.

Before implementation of genotype guided therapy, it has to be established whether or not there is a correlation between plasma concentrations of methylphenidate and therapeutic response. Three previous studies describing the effect of methylphenidate on children with ADHD found no such correlation [32](#bcp13237-bib-0032), while two more recent studies have shown this correlation [33](#bcp13237-bib-0033), [34](#bcp13237-bib-0034).

In summary, carrying the *CES1* 143E allele was found to have a substantial impact on the metabolism of methylphenidate, whereas *CES1A1c* did not appear to influence this metabolism. Moreover, our data revealed an unexpected and unexplained decrease in the metabolism of methylphenidate in subjects carrying four copies of *CES1*. More information about the functional *CES1* variants may contribute to an optimized and personalized methylphenidate therapy.

## Competing Interests

All authors have completed the Unified Competing Interest form at [www.icmje.org/coi_disclosure.pdf](http://www.icmje.org/coi_disclosure.pdf) (available on request from the corresponding author) and declare: the project was funded by the Danish Council for Strategic Research; no financial relationships with any organizations that might have an interest in the submitted work in the previous 3 years; no other relationships or activities that could appear to have influenced the submitted work.

*The present study is part of the Individualized Drug Therapy Based on Pharmacogenomics: Focus on Carboxylesterase 1 (INDICES) project, which was funded by grant 10‐092792/DSF from the Danish Council for Strategic Research, Program Commission on Individuals, Disease and Society. The sponsor had no role in the design and conduct of the study, and no influence on collection, analysis, interpretation, preparation, review or approval of the manuscript.*

## Contributors

K.D., C.S., G.J. and H.B.R. designed the study. C.S. and L.G. performed the clinical trial. C.S., Y.L., L.G., K.D. and G.J. analysed the pharmacokinetic data. D.B., L.F.M. and H.B.R. performed the genetic analyses. R.T. performed the drug concentration analyses. All authors were involved in drafting or reviewing the manuscript and approved the final version.

Stage, C. , Jürgens, G. , Guski, L. S. , Thomsen, R. , Bjerre, D. , Ferrero‐Miliani, L. , Lyauk, Y. K. , Rasmussen, H. B. , Dalhoff, K. , and The INDICES Consortium (2017) The impact of *CES1* genotypes on the pharmacokinetics of methylphenidate in healthy Danish subjects. Br J Clin Pharmacol, 83: 1506–1514. doi: [10.1111/bcp.13237](https://doi.org/10.1111/bcp.13237).

## References

1. Southan C, Sharman JL, Benson HE, Faccenda E, Pawson AJ, Alexander SP, et al. The IUPHAR/BPS Guide to PHARMACOLOGY in 2016: towards curated quantitative interactions between 1300 protein targets and 6000 ligands. Nucl Acids Res 2016; 44: D1054–68.  [DOI](https://doi.org/10.1093/nar/gkv1037) | [PMC free article](/articles/PMC4702778/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26464438/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucl%20Acids%20Res&title=The%20IUPHAR/BPS%20Guide%20to%20PHARMACOLOGY%20in%202016:%20towards%20curated%20quantitative%20interactions%20between%201300%20protein%20targets%20and%206000%20ligands&author=C%20Southan&author=JL%20Sharman&author=HE%20Benson&author=E%20Faccenda&author=AJ%20Pawson&volume=44&publication_year=2016&pages=D1054-68&pmid=26464438&doi=10.1093/nar/gkv1037&)

2. Alexander SPH, Fabbro D, Kelly E, Marrion N, Peters JA, Benson HE, et al. The Concise Guide to PHARMACOLOGY 2015/16: Enzymes. Br J Pharmacol 2015; 172: 6024–6109.  [DOI](https://doi.org/10.1111/bph.13354) | [PMC free article](/articles/PMC4718211/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26650445/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Pharmacol&title=The%20Concise%20Guide%20to%20PHARMACOLOGY%202015/16:%20Enzymes&author=SPH%20Alexander&author=D%20Fabbro&author=E%20Kelly&author=N%20Marrion&author=JA%20Peters&volume=172&publication_year=2015&pages=6024-6109&pmid=26650445&doi=10.1111/bph.13354&)

3. Statens Serum Institut, www.medstat.dk (last accessed 30 September 2016).  [www.medstat.dk](http://www.medstat.dk)

4. Treceño C, Martín Arias LH, Sáinz M, Salado I, García Ortega P, Velasco V, et al. Trends in the consumption of attention deficit hyperactivity disorder medications in Castilla y León (Spain): changes in the consumption pattern following the introduction of extended release methylphenidate. Pharmacoepidemiol Drug Saf 2012; 21: 435–441.  [DOI](https://doi.org/10.1002/pds.2348) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22253017/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacoepidemiol%20Drug%20Saf&title=Trends%20in%20the%20consumption%20of%20attention%20deficit%20hyperactivity%20disorder%20medications%20in%20Castilla%20y%20Le%C3%B3n%20(Spain):%20changes%20in%20the%20consumption%20pattern%20following%20the%20introduction%20of%20extended%20release%20methylphenidate&author=C%20Trece%C3%B1o&author=LH%20Mart%C3%ADn%20Arias&author=M%20S%C3%A1inz&author=I%20Salado&author=P%20Garc%C3%ADa%20Ortega&volume=21&publication_year=2012&pages=435-441&pmid=22253017&doi=10.1002/pds.2348&)

5. Ponizovsky AM, Marom E, Fitoussi I. Trends in attention deficit hyperactivity disorder drugs consumption, Israel, 2005–2012. Pharmacoepidemiol Drug Saf 2014; 23: 534–538.  [DOI](https://doi.org/10.1002/pds.3604) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24610676/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacoepidemiol%20Drug%20Saf&title=Trends%20in%20attention%20deficit%20hyperactivity%20disorder%20drugs%20consumption,%20Israel,%202005%E2%80%932012&author=AM%20Ponizovsky&author=E%20Marom&author=I%20Fitoussi&volume=23&publication_year=2014&pages=534-538&pmid=24610676&doi=10.1002/pds.3604&)

6. Patrick KS, Caldwell RW, Ferris RM, Breese GR. Pharmacology of the enantiomers of threo‐methylphenidate. J Pharmacol Exp Ther 1987; 241: 152–158.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/3572779/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharmacol%20Exp%20Ther&title=Pharmacology%20of%20the%20enantiomers%20of%20threo%E2%80%90methylphenidate&author=KS%20Patrick&author=RW%20Caldwell&author=RM%20Ferris&author=GR%20Breese&volume=241&publication_year=1987&pages=152-158&pmid=3572779&)

7. Ding Y‐S, Fowler JS, Volkow ND, Dewey SL, Wang G‐J, Logan J, et al. Chiral drugs: comparison of the pharmacokinetics of [11C] d‐threo and l‐threo‐methylphenidate in the human and baboon brain. Psychopharmacology (Berl) 1997; 131: 71–78.  [DOI](https://doi.org/10.1007/s002130050267) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9181638/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Psychopharmacology%20(Berl)&title=Chiral%20drugs:%20comparison%20of%20the%20pharmacokinetics%20of%20%5B11C%5D%20d%E2%80%90threo%20and%20l%E2%80%90threo%E2%80%90methylphenidate%20in%20the%20human%20and%20baboon%20brain&author=Y%E2%80%90S%20Ding&author=JS%20Fowler&author=ND%20Volkow&author=SL%20Dewey&author=G%E2%80%90J%20Wang&volume=131&publication_year=1997&pages=71-78&pmid=9181638&doi=10.1007/s002130050267&)

8. Ross MK, Borazjani A, Wang R, Allen Crow J, Xie S. Examination of the carboxylesterase phenotype in human liver. Arch Biochem Biophys 2012; 522: 44–56.  [DOI](https://doi.org/10.1016/j.abb.2012.04.010) | [PMC free article](/articles/PMC3569858/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22525521/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arch%20Biochem%20Biophys&title=Examination%20of%20the%20carboxylesterase%20phenotype%20in%20human%20liver&author=MK%20Ross&author=A%20Borazjani&author=R%20Wang&author=J%20Allen%20Crow&author=S%20Xie&volume=522&publication_year=2012&pages=44-56&pmid=22525521&doi=10.1016/j.abb.2012.04.010&)

9. Hosokawa M. Structure and catalytic properties of carboxylesterase isozymes involved in metabolic activation of prodrugs. Molecules 2008; 13: 412–431.  [DOI](https://doi.org/10.3390/molecules13020412) | [PMC free article](/articles/PMC6245361/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18305428/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Molecules&title=Structure%20and%20catalytic%20properties%20of%20carboxylesterase%20isozymes%20involved%20in%20metabolic%20activation%20of%20prodrugs&author=M%20Hosokawa&volume=13&publication_year=2008&pages=412-431&pmid=18305428&doi=10.3390/molecules13020412&)

10. Sai K, Saito Y, Tatewaki N, Hosokawa M, Kaniwa N, Nishimaki‐Mogami T, et al. Association of carboxylesterase 1A genotypes with irinotecan pharmacokinetics in Japanese cancer patients. Br J Clin Pharmacol 2010; 70: 222–233.  [DOI](https://doi.org/10.1111/j.1365-2125.2010.03695.x) | [PMC free article](/articles/PMC2911552/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20653675/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Clin%20Pharmacol&title=Association%20of%20carboxylesterase%201A%20genotypes%20with%20irinotecan%20pharmacokinetics%20in%20Japanese%20cancer%20patients&author=K%20Sai&author=Y%20Saito&author=N%20Tatewaki&author=M%20Hosokawa&author=N%20Kaniwa&volume=70&publication_year=2010&pages=222-233&pmid=20653675&doi=10.1111/j.1365-2125.2010.03695.x&)

11. Yoshimura M, Kimura T, Ishii M, Ishii K, Matsuura T, Geshi E, et al. Functional polymorphisms in carboxylesterase 1A2 (CES1A2) gene involves specific protein 1 (Sp1) binding sites. Biochem Biophys Res Commun 2008; 369: 939–942.  [DOI](https://doi.org/10.1016/j.bbrc.2008.02.120) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18328811/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem%20Biophys%20Res%20Commun&title=Functional%20polymorphisms%20in%20carboxylesterase%201A2%20(CES1A2)%20gene%20involves%20specific%20protein%201%20(Sp1)%20binding%20sites&author=M%20Yoshimura&author=T%20Kimura&author=M%20Ishii&author=K%20Ishii&author=T%20Matsuura&volume=369&publication_year=2008&pages=939-942&pmid=18328811&doi=10.1016/j.bbrc.2008.02.120&)

12. Tanimoto K, Kaneyasu M, Shimokuni T, Hiyama K, Nishiyama M. Human carboxylesterase 1A2 expressed from carboxylesterase 1A1 and 1A2 genes is a potent predictor of CPT‐11 cytotoxicity in vitro . Pharmacogenet Genomics 2007; 17: 1–10.  [DOI](https://doi.org/10.1097/01.fpc.0000230110.18957.50) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17264798/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Human%20carboxylesterase%201A2%20expressed%20from%20carboxylesterase%201A1%20and%201A2%20genes%20is%20a%20potent%20predictor%20of%20CPT%E2%80%9011%20cytotoxicity%20in%20vitro&author=K%20Tanimoto&author=M%20Kaneyasu&author=T%20Shimokuni&author=K%20Hiyama&author=M%20Nishiyama&volume=17&publication_year=2007&pages=1-10&pmid=17264798&doi=10.1097/01.fpc.0000230110.18957.50&)

13. Takai S, Matsuda A, Usami Y, Adachi T, Sugiyama T, Katagiri Y, et al. Hydrolytic profile for ester‐ or amide‐linkage by carboxylesterases pI 5.3 and 4.5 from human liver. Biol Pharm Bull 1997; 20: 869–873.  [DOI](https://doi.org/10.1248/bpb.20.869) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9300133/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biol%20Pharm%20Bull&title=Hydrolytic%20profile%20for%20ester%E2%80%90%20or%20amide%E2%80%90linkage%20by%20carboxylesterases%20pI%205.3%20and%204.5%20from%20human%20liver&author=S%20Takai&author=A%20Matsuda&author=Y%20Usami&author=T%20Adachi&author=T%20Sugiyama&volume=20&publication_year=1997&pages=869-873&pmid=9300133&doi=10.1248/bpb.20.869&)

14. Thomsen R, Rasmussen HB, Linnet K, The INDICES Consortium . In vitro drug metabolism by human carboxylesterase 1: focus on angiotensin‐converting enzyme inhibitors. Drug Metab Dispos 2013; 42: 126–133.  [DOI](https://doi.org/10.1124/dmd.113.053512) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24141856/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=In%20vitro%20drug%20metabolism%20by%20human%20carboxylesterase%201:%20focus%20on%20angiotensin%E2%80%90converting%20enzyme%20inhibitors&author=R%20Thomsen&author=HB%20Rasmussen&author=K%20Linnet&volume=42&publication_year=2013&pages=126-133&pmid=24141856&doi=10.1124/dmd.113.053512&)

15. Laizure SC, Parker RB, Herring VL, Hu Z‐Y. Identification of carboxylesterase‐dependent dabigatran etexilate hydrolysis. Drug Metab Dispos 2013; 42: 201–206.  [DOI](https://doi.org/10.1124/dmd.113.054353) | [PMC free article](/articles/PMC3912543/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24212379/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Identification%20of%20carboxylesterase%E2%80%90dependent%20dabigatran%20etexilate%20hydrolysis&author=SC%20Laizure&author=RB%20Parker&author=VL%20Herring&author=Z%E2%80%90Y%20Hu&volume=42&publication_year=2013&pages=201-206&pmid=24212379&doi=10.1124/dmd.113.054353&)

16. Shi D, Yang J, Yang D, LeCluyse EL, Black C, You L, et al. Anti‐influenza prodrug oseltamivir is activated by carboxylesterase human carboxylesterase 1, and the activation is inhibited by antiplatelet agent clopidogrel. J Pharmacol Exp Ther 2006; 319: 1477–1484.  [DOI](https://doi.org/10.1124/jpet.106.111807) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16966469/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharmacol%20Exp%20Ther&title=Anti%E2%80%90influenza%20prodrug%20oseltamivir%20is%20activated%20by%20carboxylesterase%20human%20carboxylesterase%201,%20and%20the%20activation%20is%20inhibited%20by%20antiplatelet%20agent%20clopidogrel&author=D%20Shi&author=J%20Yang&author=D%20Yang&author=EL%20LeCluyse&author=C%20Black&volume=319&publication_year=2006&pages=1477-1484&pmid=16966469&doi=10.1124/jpet.106.111807&)

17. Sun Z. Methylphenidate is stereoselectively hydrolyzed by human carboxylesterase CES1A1. J Pharmacol Exp Ther 2004; 310: 469–476.  [DOI](https://doi.org/10.1124/jpet.104.067116) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15082749/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharmacol%20Exp%20Ther&title=Methylphenidate%20is%20stereoselectively%20hydrolyzed%20by%20human%20carboxylesterase%20CES1A1&author=Z%20Sun&volume=310&publication_year=2004&pages=469-476&pmid=15082749&doi=10.1124/jpet.104.067116&)

18. Tang M, Mukundan M, Yang J, Charpentier N, LeCluyse EL, Black C, et al. Antiplatelet agents aspirin and clopidogrel are hydrolyzed by distinct carboxylesterases, and clopidogrel is transesterificated in the presence of ethyl alcohol. J Pharmacol Exp Ther 2006; 319: 1467–1476.  [DOI](https://doi.org/10.1124/jpet.106.110577) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16943252/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharmacol%20Exp%20Ther&title=Antiplatelet%20agents%20aspirin%20and%20clopidogrel%20are%20hydrolyzed%20by%20distinct%20carboxylesterases,%20and%20clopidogrel%20is%20transesterificated%20in%20the%20presence%20of%20ethyl%20alcohol&author=M%20Tang&author=M%20Mukundan&author=J%20Yang&author=N%20Charpentier&author=EL%20LeCluyse&volume=319&publication_year=2006&pages=1467-1476&pmid=16943252&doi=10.1124/jpet.106.110577&)

19. Johnson KA, Barry E, Lambert D, Fitzgerald M, McNicholas F, Kirley A, et al. Methylphenidate side effect profile is influenced by genetic variation in the attention‐deficit/hyperactivity disorder‐associated CES1 gene. J Child Adolesc Psychopharmacol 2013; 23: 655–664.  [DOI](https://doi.org/10.1089/cap.2013.0032) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24350812/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Child%20Adolesc%20Psychopharmacol&title=Methylphenidate%20side%20effect%20profile%20is%20influenced%20by%20genetic%20variation%20in%20the%20attention%E2%80%90deficit/hyperactivity%20disorder%E2%80%90associated%20CES1%20gene&author=KA%20Johnson&author=E%20Barry&author=D%20Lambert&author=M%20Fitzgerald&author=F%20McNicholas&volume=23&publication_year=2013&pages=655-664&pmid=24350812&doi=10.1089/cap.2013.0032&)

20. Zhu H‐J, Patrick KS, Yuan H‐J, Wang J‐S, Donovan JL, DeVane CL, et al. Two CES1 gene mutations lead to dysfunctional carboxylesterase 1 activity in man: clinical significance and molecular basis. Am J Hum Genet 2008; 82: 1241–1248.  [DOI](https://doi.org/10.1016/j.ajhg.2008.04.015) | [PMC free article](/articles/PMC2427248/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18485328/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=Two%20CES1%20gene%20mutations%20lead%20to%20dysfunctional%20carboxylesterase%201%20activity%20in%20man:%20clinical%20significance%20and%20molecular%20basis&author=H%E2%80%90J%20Zhu&author=KS%20Patrick&author=H%E2%80%90J%20Yuan&author=J%E2%80%90S%20Wang&author=JL%20Donovan&volume=82&publication_year=2008&pages=1241-1248&pmid=18485328&doi=10.1016/j.ajhg.2008.04.015&)

21. Zhu H‐J, Markowitz JS. Activation of the antiviral prodrug oseltamivir is impaired by two newly identified carboxylesterase 1 variants. Drug Metab Dispos 2008; 37: 264–267.  [DOI](https://doi.org/10.1124/dmd.108.024943) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19022936/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Activation%20of%20the%20antiviral%20prodrug%20oseltamivir%20is%20impaired%20by%20two%20newly%20identified%20carboxylesterase%201%20variants&author=H%E2%80%90J%20Zhu&author=JS%20Markowitz&volume=37&publication_year=2008&pages=264-267&pmid=19022936&doi=10.1124/dmd.108.024943&)

22. Tarkiainen EK, Backman JT, Neuvonen M, Neuvonen PJ, Schwab M, Niemi M. Carboxylesterase 1 polymorphism impairs oseltamivir bioactivation in humans. Clin Pharmacol Ther 2012; 92: 68–71.  [DOI](https://doi.org/10.1038/clpt.2012.13) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22588607/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Carboxylesterase%201%20polymorphism%20impairs%20oseltamivir%20bioactivation%20in%20humans&author=EK%20Tarkiainen&author=JT%20Backman&author=M%20Neuvonen&author=PJ%20Neuvonen&author=M%20Schwab&volume=92&publication_year=2012&pages=68-71&pmid=22588607&doi=10.1038/clpt.2012.13&)

23. Lewis JP, Horenstein RB, Ryan K, O'Connell JR, Gibson Q, Mitchell BD, et al. The functional G143E variant of carboxylesterase 1 is associated with increased clopidogrel active metabolite levels and greater clopidogrel response. Pharmacogenet Genomics 2013; 23: 1–8.  [DOI](https://doi.org/10.1097/FPC.0b013e32835aa8a2) | [PMC free article](/articles/PMC3682407/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23111421/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=The%20functional%20G143E%20variant%20of%20carboxylesterase%201%20is%20associated%20with%20increased%20clopidogrel%20active%20metabolite%20levels%20and%20greater%20clopidogrel%20response&author=JP%20Lewis&author=RB%20Horenstein&author=K%20Ryan&author=JR%20O'Connell&author=Q%20Gibson&volume=23&publication_year=2013&pages=1-8&pmid=23111421&doi=10.1097/FPC.0b013e32835aa8a2&)

24. Thomsen R, Rasmussen HB, Linnet K, The INDICES Consortium . Enantioselective determination of methylphenidate and ritalinic acid in whole blood from forensic cases using automated solid‐phase extraction and liquid chromatography–tandem mass spectrometry. J Anal Toxicol 2012; 36: 560–568.  [DOI](https://doi.org/10.1093/jat/bks065) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22833645/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Anal%20Toxicol&title=Enantioselective%20determination%20of%20methylphenidate%20and%20ritalinic%20acid%20in%20whole%20blood%20from%20forensic%20cases%20using%20automated%20solid%E2%80%90phase%20extraction%20and%20liquid%20chromatography%E2%80%93tandem%20mass%20spectrometry&author=R%20Thomsen&author=HB%20Rasmussen&author=K%20Linnet&volume=36&publication_year=2012&pages=560-568&pmid=22833645&doi=10.1093/jat/bks065&)

25. Kirkwood B, Sterne J. Essential Medical Statistics, Second edn. Oxford: Blackwell Science, 2003.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Essential%20Medical%20Statistics&author=B%20Kirkwood&author=J%20Sterne&publication_year=2003&)

26. Fukami T, Nakajima M, Maruichi T, Takahashi S, Takamiya M, Aoki Y, et al. Structure and characterization of human carboxylesterase 1A1, 1A2, and 1A3 genes. Pharmacogenet Genomics 2008; 18: 911–920.  [DOI](https://doi.org/10.1097/FPC.0b013e32830b0c5e) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18794728/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Structure%20and%20characterization%20of%20human%20carboxylesterase%201A1,%201A2,%20and%201A3%20genes&author=T%20Fukami&author=M%20Nakajima&author=T%20Maruichi&author=S%20Takahashi&author=M%20Takamiya&volume=18&publication_year=2008&pages=911-920&pmid=18794728&doi=10.1097/FPC.0b013e32830b0c5e&)

27. Suzaki Y, Uemura N, Takada M, Ohyama T, Itohda A, Morimoto T, et al. The effect of carboxylesterase 1 (CES1) polymorphisms on the pharmacokinetics of oseltamivir in humans. Eur J Clin Pharmacol 2012; 69: 21–30.  [DOI](https://doi.org/10.1007/s00228-012-1315-5) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22673926/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Clin%20Pharmacol&title=The%20effect%20of%20carboxylesterase%201%20(CES1)%20polymorphisms%20on%20the%20pharmacokinetics%20of%20oseltamivir%20in%20humans&author=Y%20Suzaki&author=N%20Uemura&author=M%20Takada&author=T%20Ohyama&author=A%20Itohda&volume=69&publication_year=2012&pages=21-30&pmid=22673926&doi=10.1007/s00228-012-1315-5&)

28. Wang X, Wang G, Shi J, Aa J, Comas R, Liang Y, et al. CES1 genetic variation affects the activation of angiotensin‐converting enzyme inhibitors. Pharmacogenomics J 2016; 16: 220–230.  [DOI](https://doi.org/10.1038/tpj.2015.42) | [PMC free article](/articles/PMC6329299/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26076923/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=CES1%20genetic%20variation%20affects%20the%20activation%20of%20angiotensin%E2%80%90converting%20enzyme%20inhibitors&author=X%20Wang&author=G%20Wang&author=J%20Shi&author=J%20Aa&author=R%20Comas&volume=16&publication_year=2016&pages=220-230&pmid=26076923&doi=10.1038/tpj.2015.42&)

29. Yamada S, Richardson K, Tang M, Halaschek‐Wiener J, Cook VJ, Fitzgerald JM, et al. Genetic variation in carboxylesterase genes and susceptibility to isoniazid‐induced hepatotoxicity. Pharmacogenomics J 2010; 10: 524–536.  [DOI](https://doi.org/10.1038/tpj.2010.5) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20195289/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=Genetic%20variation%20in%20carboxylesterase%20genes%20and%20susceptibility%20to%20isoniazid%E2%80%90induced%20hepatotoxicity&author=S%20Yamada&author=K%20Richardson&author=M%20Tang&author=J%20Halaschek%E2%80%90Wiener&author=VJ%20Cook&volume=10&publication_year=2010&pages=524-536&pmid=20195289&doi=10.1038/tpj.2010.5&)

30. Tarkiainen EK, Tornio A, Holmberg MT, Launiainen T, Neuvonen PJ, Backman JT, et al. Effect of carboxylesterase 1 c.428G>A single nucleotide variation on the pharmacokinetics of quinapril and enalapril. Br J Clin Pharmacol 2015; 80: 1131–1138.  [DOI](https://doi.org/10.1111/bcp.12667) | [PMC free article](/articles/PMC4631185/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25919042/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Clin%20Pharmacol&title=Effect%20of%20carboxylesterase%201%20c.428G>A%20single%20nucleotide%20variation%20on%20the%20pharmacokinetics%20of%20quinapril%20and%20enalapril&author=EK%20Tarkiainen&author=A%20Tornio&author=MT%20Holmberg&author=T%20Launiainen&author=PJ%20Neuvonen&volume=80&publication_year=2015&pages=1131-1138&pmid=25919042&doi=10.1111/bcp.12667&)

31. Sanford JC, Wang X, Shi J, Barrie ES, Wang D, Zhu H‐J, et al. Regulatory effects of genomic translocations at the human carboxylesterase‐1 (CES1) gene locus. Pharmacogenet Genomics 2016; 26: 197–207.  [DOI](https://doi.org/10.1097/FPC.0000000000000206) | [PMC free article](/articles/PMC4821783/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26871237/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Regulatory%20effects%20of%20genomic%20translocations%20at%20the%20human%20carboxylesterase%E2%80%901%20(CES1)%20gene%20locus&author=JC%20Sanford&author=X%20Wang&author=J%20Shi&author=ES%20Barrie&author=D%20Wang&volume=26&publication_year=2016&pages=197-207&pmid=26871237&doi=10.1097/FPC.0000000000000206&)

32. Gualtieri CT, Hicks RE, Patrick K, Schroeder SR, Breese GR. Clinical correlates of methylphenidate blood levels. Ther Drug Monit 1984; 6: 379–392.  [DOI](https://doi.org/10.1097/00007691-198412000-00001) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/6515700/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ther%20Drug%20Monit&title=Clinical%20correlates%20of%20methylphenidate%20blood%20levels&author=CT%20Gualtieri&author=RE%20Hicks&author=K%20Patrick&author=SR%20Schroeder&author=GR%20Breese&volume=6&publication_year=1984&pages=379-392&pmid=6515700&doi=10.1097/00007691-198412000-00001&)

33. Teicher MH, Polcari A, Foley M, Valente E, McGreenery CE, Chang W‐W, et al. Methylphenidate blood levels and therapeutic response in children with attention‐deficit hyperactivity disorder I. Effects of different dosing regimens. J Child Adolesc Psychopharmacol 2006; 16: 416–431.  [DOI](https://doi.org/10.1089/cap.2006.16.416) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16958567/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Child%20Adolesc%20Psychopharmacol&title=Methylphenidate%20blood%20levels%20and%20therapeutic%20response%20in%20children%20with%20attention%E2%80%90deficit%20hyperactivity%20disorder%20I.%20Effects%20of%20different%20dosing%20regimens&author=MH%20Teicher&author=A%20Polcari&author=M%20Foley&author=E%20Valente&author=CE%20McGreenery&volume=16&publication_year=2006&pages=416-431&pmid=16958567&doi=10.1089/cap.2006.16.416&)

34. Quinn D, Wigal S, Swanson J, Hirsch S, Ottolini Y, Dariani M, et al. Comparative pharmacodynamics and plasma concentrations of d‐threo‐methylphenidate hydrochloride after single doses of d‐threo‐methylphenidate hydrochloride and d,l‐threo‐methylphenidate hydrochloride in a double‐blind, placebo‐controlled, crossover laboratory school study in children with attention‐deficit/hyperactivity disorder. J Am Acad Child Adolesc Psychiatry 2004; 43: 1422–1429.  [DOI](https://doi.org/10.1097/01.chi.0000140455.96946.2b) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15502602/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Am%20Acad%20Child%20Adolesc%20Psychiatry&title=Comparative%20pharmacodynamics%20and%20plasma%20concentrations%20of%20d%E2%80%90threo%E2%80%90methylphenidate%20hydrochloride%20after%20single%20doses%20of%20d%E2%80%90threo%E2%80%90methylphenidate%20hydrochloride%20and%20d,l%E2%80%90threo%E2%80%90methylphenidate%20hydrochloride%20in%20a%20double%E2%80%90blind,%20placebo%E2%80%90controlled,%20crossover%20laboratory%20school%20study%20in%20children%20with%20attention%E2%80%90deficit/hyperactivity%20disorder&author=D%20Quinn&author=S%20Wigal&author=J%20Swanson&author=S%20Hirsch&author=Y%20Ottolini&volume=43&publication_year=2004&pages=1422-1429&pmid=15502602&doi=10.1097/01.chi.0000140455.96946.2b&)
