# Role of rs3846662 and HMGCR alternative splicing in statin efficacy and baseline lipid levels in familial hypercholesterolemia

## Metadata
**Authors:** Valerie Leduc, Lucienne Bourque, Judes Poirier, Robert Dufour
**Journal:** Pharmacogenetics and genomics
**Date:** 2016 Jan
**DOI:** [10.1097/FPC.0000000000000178](https://doi.org/10.1097/FPC.0000000000000178)
**PMID:** 26466344
**PMCID:** PMC5138058
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5138058/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC5138058/pdf/nihms6053.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC5138058/pdf/nihms6053.pdf)

## Abstract

**Objectives:** 
To assess the contribution of the rs3846662 polymorphism of HMGCR on serum lipid levels and statin efficacy, we measured in vivo HMGCR mRNA and lipid levels in French Canadian individuals affected by heterozygous familial hypercholesterolemia due to the deletion of more than 15 kb of the LDLR gene.

**Results:** 
Men and women carrying the AA genotype at rs3846662, and no APOE4 allele, had higher levels of total cholesterol (5.43 vs. 4.58 mmol/l, P <0.05) and LDL-cholesterol (5.20 vs. 4.39 mmol/l, P <0.05) at baseline. However, with regard to statin efficacy, the penetrance of the AA genotype was sex dependent. Indeed, the percentage reduction in LDL-cholesterol upon statin treatment was significantly decreased in women with the AA genotype compared with women without it (38.4 vs. 46.2%, P <0.05), whereas this was not observed in men. Although both men and women bearing the AA genotype showed a higher ratio of full-length HMGCR mRNA/total HMGCR mRNA compared with individuals without it (n =37, P <0.05), overall transcription of HMGCR was decreased and increased in men and women carrying this genotype, respectively (n =37, P <0.01 and P <0.05). Finally, in our familial hypercholesterolemia cohort, HMGCR alternative splicing explained between 22 and 55% of the variance in statin response.

**Conclusion:** 
rs3846662 polymorphism and the alternative splicing of HMGCR mRNA significantly impact women’s response to statin therapy.

Keywords: alternative splicing, familial hypercholesterolemia, genetic polymorphism, human 3-hydroxy-3-methylglutaryl-CoA reductase, rs3846662, serum lipid levels, statin efficacy

### Objectives

To assess the contribution of the rs3846662 polymorphism of *HMGCR* on serum lipid levels and statin efficacy, we measured *in vivo HMGCR* mRNA and lipid levels in French Canadian individuals affected by heterozygous familial hypercholesterolemia due to the deletion of more than 15 kb of the *LDLR* gene.

### Results

Men and women carrying the AA genotype at rs3846662, and no *APOE4* allele, had higher levels of total cholesterol (5.43 vs. 4.58 mmol/l, *P* <0.05) and LDL-cholesterol (5.20 vs. 4.39 mmol/l, *P* <0.05) at baseline. However, with regard to statin efficacy, the penetrance of the AA genotype was sex dependent. Indeed, the percentage reduction in LDL-cholesterol upon statin treatment was significantly decreased in women with the AA genotype compared with women without it (38.4 vs. 46.2%, *P* <0.05), whereas this was not observed in men. Although both men and women bearing the AA genotype showed a higher ratio of full-length *HMGCR* mRNA/total *HMGCR* mRNA compared with individuals without it (*n* =37, *P* <0.05), overall transcription of *HMGCR* was decreased and increased in men and women carrying this genotype, respectively (*n* =37, *P* <0.01 and *P* <0.05). Finally, in our familial hypercholesterolemia cohort, *HMGCR* alternative splicing explained between 22 and 55% of the variance in statin response.

### Conclusion

rs3846662 polymorphism and the alternative splicing of *HMGCR* mRNA significantly impact women’s response to statin therapy.

## Introduction

Elevated LDL-cholesterol (LDL-C) levels are the primary risk factor for atherosclerosis and atherosclerosis-related diseases (reviewed in Badimon and Vilahur [[1](#R1)]). Consequently, lipid-lowering therapies have become the foundation for prevention and treatment of these disorders [[1](#R1)]. Statins are the most commonly prescribed drugs to lower LDL-C levels. They inhibit the rate-limiting enzyme in cholesterol synthesis – namely, 3-hydroxy-3-methylglutaryl coenzyme A reductase (HMGCR) – thereby reducing hepatic cholesterol production, upregulating hepatic LDL receptors (LDLR), and increasing LDL and LDL-related particle uptake [[2](#R2)]. There are wide interindividual differences in both serum lipid levels and the magnitude of response to statin therapy. Although not a universal finding [[3](#R3)], studies on monozygotic twins reared apart or together suggest that, despite the undeniable environmental influence, the interindividual variations observed in serum lipid levels are predominantly explained by genetic variability [[4](#R4)–[6](#R6)]. As for the differences observed in statin response, they have been attributed to a plethora of genetic and nongenetic factors, including single-nucleotide polymorphisms (SNPs) in lipid metabolism genes (reviewed in Schmitz *et al*. [[7](#R7)]), age, smoking status, body weight, diet, physical activity, and racial ancestry [[8](#R8)–[10](#R10)]. Recent evidence suggests that gene alternative pre-mRNA splicing may also be a likely determinant of serum lipid levels and statin efficacy. Indeed, although *HMGCR* genotypes explained less than 2% of the variance in statin response [[11](#R11)], *HMGCR* alternative splicing explained between 6 and 15% of its variance [[12](#R12),[13](#R13)].

Effectively, *HMGCR* pre-mRNA undergoes alternative splicing of exon 13 and generates two transcripts: full-length (FL) *HMGCR* and Δ13 *HMGCR* [[14](#R14)]. Exon 13 encodes a part of the catalytic/statin-binding domain of the enzyme [[15](#R15),[16](#R16)] and, unsurprisingly, transfection of the Δ13 transcript could not rescue HMGCR activity in UT2 *HMGCR*-deficient cells [[17](#R17)]. Interestingly, two independent groups reported the involvement of the intron 13 SNP rs3846662 (A or G allele) in the regulation of *HMGCR* exon 13 skipping [[12](#R12),[17](#R17)]. This SNP likely alters the binding motif of heterozygous nuclear ribonucleoprotein A1 (HNRNPA1), a molecule that regulates *HMGCR* alternative splicing [[18](#R18)]. The major and ancestral A allele at locus rs3846662 likely engenders the preferential binding of HNRNPA1, which promotes skipping of exon 13 [[18](#R18)]. These findings suggest that an increase in the proportion of Δ13 *HMGCR* mRNA is promoted in individuals carrying the rs3846662 A allele [[12](#R12),[17](#R17)], resulting in an HMGCR protein with possibly lower activity [[13](#R13),[17](#R17)], lower levels of LDL-C at baseline [[19](#R19)–[21](#R21)], and lesser sensitivity and response to statin inhibition [[11](#R11),[13](#R13),[22](#R22),[23](#R23)].

Despite the growing body of in-vitro literature examining the impact of the A/G allele at locus rs3846662, researchers have not yet measured *HMGCR* mRNA levels *in vivo*, in living humans. Furthermore, it has been suggested that the alternative splicing of *HMGCR* may exist primarily to correct small imbalances in cholesterol homeostasis, allowing the cell to dodge the need for stimulation of a robust transcriptional response when cholesterol disequilibrium occurs [[24](#R24)]. Alternative splicing and rs3846662 would thus be expected to have a stronger impact on baseline lipid levels rather than on LDL-C levels following statin treatment [[24](#R24)]. To further assess the contribution of *HMGCR* alternative splicing and rs3846662 on baseline lipid levels and statin response, we quantified these parameters in a homogenous cohort of French Canadian individuals all affected by heterozygous familial hypercholesterolemia (FH) due to the deletion of more than 15 kb of the promoter and first exon of the *LDLR* gene. FH is a monogenic hereditable disorder characterized by large increase in plasma LDL-C levels, cholesterol deposition in tendons, and increased risk for atherosclerosis [[25](#R25)]. Owing to a founder effect, the prevalence of FH in French Canadians of Quebec is twice that observed generally worldwide, with the deletion of more than 15 kb of the *LDLR* gene accounting for 60% of the mutant allele [[26](#R26)]. These patients, in addition to sharing a homogenous genetic background, are functional hemizygotes [[27](#R27)]. Their LDL-C levels are thus likely to be strongly influenced by variations within the functional *LDLR* allele and by the other genetic and environmental factors specific to the individuals [[28](#R28),[29](#R29)]. Notably, sex and apolipoprotein E (*APOE*) genotype have previously been shown to affect lipid traits [[30](#R30)] and response to statin treatment [[31](#R31)] in FH patients of French Canadian descent. Last year, in a population of elderly French Canadians not affected by FH, the effects of rs3846662 on cognitive impairment have also been shown to be modulated by sex and the presence of the *APOE* ε4 allele (APOE4) [[32](#R32)]. Our study thus sought to explore, in FH patients of French Canadian descent, the effects of rs3846662 and *HMGCR* alternative splicing on lipids levels and statin efficacy with respect to sex and *APOE4* presence.

## Methods

### Patient demographics

A total of 106 unrelated patients were selected from the nutrition, metabolism, and atherosclerosis clinic of the Clinical Research Institute of Montreal. All patients were diagnosed with heterozygous FH based on the presence of levels of LDL-C greater than 95th percentile for age and sex norms, a family history of hypercholesterolemia, and the presence of one allele bearing the deletion of more than 15 kb of the promoter and first exon of the *LDLR* gene. Patients were considered for inclusion in the study regardless of the nature or dose of statin currently administered, and most of them received statin with other lipid-lowering therapy (e.g. ezetimibe, bile acid sequestrants, niacin). Height, weight, and BMI were recorded following a standardized protocol. All procedures were approved by the institutional ethics committee, and all patients signed an informed consent form for medical record consultation as well as for plasma and DNA analysis for dyslipidemia-related research.

### Baseline lipid levels study, data collection, and statistical procedure

Medical records were sought to determine baseline lipid profiles. These 12 h-fasting plasma lipid concentrations were measured before any hypolipidemic drug treatment and after introduction of the American Heart Association phase II diet for at least 4 weeks. These data were available for 99 of the 106 patients initially screened ([Table 1](#T1)). Baseline lipid levels were normalized for age and sex by subtracting, from individual value, the 50th percentile lipid value obtained for normal Canadians in the same sex and age group. These reference percentiles were taken from the ‘Canadian Health measures survey: cycle 1 data tables (2007–2009)’ of Statistic Canada. The effect of rs3846662 on the normalized data – namely, delta (Δ) total cholesterol (TC) and Δ LDL-C levels –was then analyzed by means of the Kruskal–Wallis and Mann–Whitney *U*-tests as the values did not follow a normal distribution. To account for its possible effect on baseline lipid levels, data were split by *APOE4* allele status (0, individuals without an *APOE4* allele; 1, individuals with at least one *APOE4* allele).

### Table 1.

|   | Baseline lipid levels study |  | Efficacy study |  | RNA levels study |  |
| --- | --- | --- | --- | --- | --- | --- |
| APOE4 = 0 (N = 59) | APOE4 = 1 (N = 40) | Male (N = 48) | Female (N = 37) | Male (N = 22) | Female (N = 15) |  |
| Age at measurement |  |  |  |  |  |  |
| Mean ± SD (years) | 43.7 ± 11.7 | 41.5 ± 10.6 | 45.5 ± 9.9 | 45.8 ± 12.7 | 61.6 ± 8.8 | 53.2 ± 14.2 |
| BMI |  |  |  |  |  |  |
| Mean ± SD (kg/m2) | 26.1 ± 4.3 | 25.4 ± 3.8 | 26.4 ± 3.8 | 25.8 ± 4.2 | 28.9 ± 5.1 | 28.4 ± 5.2 |
| Sex |  |  |  |  |  |  |
| Female [(n (%)] | 23 (40) | 21 (53) | – | – | – | – |
| Total cholesterol |  |  |  |  |  |  |
| Baseline (mean ± SD, mmol/l) | 9.84 ± 1.34 | 9.57 ± 1.50 | 9.76 ± 1.29 | 9.67 ± 1.64 | 9.91 ± 1.20 | 9.65 ± 1.29 |
| On statin (mean ± SD, mmol/l) | – | – | 6.13 ± 1.22 | 6.09 ± 1.08 | 5.31 ± 1.81 | 6.12 ± 1.81 |
| LDL-C |  |  |  |  |  |  |
| Baseline (mean ± SD, mmol/l) | 7.76 ± 1.28 | 7.62 ± 1.33 | 7.83 ± 1.17 | 7.58 ± 1.48 | 7.99 ± 1.12 | 7.58 ± 1.10 |
| On statin (mean ± SD, mmol/l) | – | – | 4.45 ± 1.08 | 4.24 ± 0.93 | 3.50 ± 1.63 | 4.00 ± 1.59 |
| TG |  |  |  |  |  |  |
| Baseline (mean ± SD, mmol/l) | 1.88 ± 0.75 | 1.52 ± 0.65 | 1.82 ± 0.76 | 1.60 ± 0.75 | 1.82 ± 0.69 | 1.55 ± 0.91 |
| On statin (mean ± SD, mmol/l) | – | – | 1.27 ± 0.65 | 1.22 ± 0.56 | 1.27 ± 0.55 | 1.29 ± 0.78 |
| HDL-C |  |  |  |  |  |  |
| Baseline (mean ± SD, mmol/l) | 1.13 ± 0.34 | 1.18 ± 0.37 | 1.04 ± 0.28 | 1.29 ± 0.38 | 1.01 ± 0.32 | 1.30 ± 0.38 |
| On statin (mean ± SD, mmol/l) | – | – | 1.10 ± 0.25 | 1.30 ± 0.32 | 1.23 ± 0.34 | 1.50 ± 0.40 |
| APOE4 allele |  |  |  |  |  |  |
| Frequency | – | – | 0.21 | 0.23 | 0.17 | 0.18 |
| HMGCR A allele |  |  |  |  |  |  |
| Frequency | 0.60 | 0.60 | 0.63 | 0.66 | 0.48 | 0.50 |

Table 1 Caption: Characteristics of patients with heterozygous familial hypercholesterolemia due to the deletion of more than 15 kb of the LDLR

### Efficacy study, data collection, and statistical procedure

In order to determine the patient’s response to statin therapy, 12 h-fasting plasma lipid concentrations were measured after a hypolipidemic treatment of at least 4 weeks. We again used medical records to retrieve the 12 h-fasting plasma lipid concentrations initially obtained after at least 4 weeks of statin monotherapy. These data were available for 85 of the 99 patients ([Table 1](#T1)), and among them 50 received atorvastatin, 16 received rosuvastatin, 12 received simvastatin, five received lovastatin, and two received pravastatin. Patients were instructed to maintain their cholesterol-lowering diet and their lipid-lowering therapy throughout the study and were seen at 4-week intervals. Patient compliance (or adherence to treatment) was monitored by the medical team through a specific questionnaire. Only patients with good compliance (>80%) were included in the study. The patient’s response to statin therapy was evaluated by comparing the observed decrease in LDL-C following therapy with the expected LDL-C reduction, at the appropriate dose, stated in the statin monographs. Results are expressed as percentage efficacy of statin treatment and percentage reduction in LDL-C, and calculations were as follows:

| Efficacyofstatintreatment(%)=100×[ObservedreductioninLDLCExpectedreductioninLDLC],LDLCreduction(%)=100-[100×(LDLCafter4weeksoftreatmentLDLCbeforetreatment)]. |
| --- |
The effects of rs3846662 on statin efficacy were analyzed by means of univariate analysis of variance models, which included sex and *APOE4* allele status as cofactors, and age, BMI, and baseline LDL-C levels as covariates. The partial *η*^2^ was used to estimate the percentage of variance in statin efficacy explained by the different variables.

### RNA levels study, data collection, and statistical procedure

In order to measure the patient’s *HMGCR* mRNA levels, a fresh blood sample was collected in a PAXgene tube after an overnight fast of at least 12 h. These blood samples were collected from 45 of the 85 patients for which the efficacy of statin treatment had been previously determined ([Table 1](#T1)). The low participation rate (53%) could be attributed to the project being conducted during the summer vacation period. Of the 45 samples collected, eight could not be used because of poor RNA extraction or loss of RNA. Results are presented as *HMGCR* mRNA levels and *HMGCR* mRNA ratios, the latter being calculated as the mRNA levels of the FL or Δ13 *HMGCR* variant divided by the total mRNA levels of *HMGCR* (any *HMGCR* transcripts containing exon boundary 6/7). Whereas the FL, Δ13, or total *HMGCR* mRNA levels reflect the numbers of each mRNA transcripts in a given tissue, the mRNA ratios assess the effects of splicing independently of the effects on overall transcription. The effects of rs3846662 on *HMGCR* mRNA levels or *HMGCR* mRNA ratios were analyzed by means of univariate analysis of variance models, which included sex and *APOE4* allele status as cofactors, and age and BMI as covariates. The partial *η*^2^ was used to estimate the percentage of variance in RNA levels explained by the different variables. Importantly, for safety and ethical reasons, the patients, who were all taking statin with other lipid-lowering therapy (mostly ezetimibe), continued their medication at the time of the blood draw. Therefore, variations in the *HMGCR* mRNA levels cannot be attributed solely to statin use. Finally, the correlation between *HMGCR* mRNA levels and the efficacy of statin treatment was analyzed by means of Pearson’s correlation.

### Determination of plasma lipid concentrations

Blood from patients was drawn under vacuum into EDTA-vacutainer tubes (VWR International, Montreal, Quebec, Canada) after an overnight fast of at least 12 h and kept on ice until separation. Following separation by centrifugation (15 min, 3000 rpm, 4°C), plasma was kept on ice and sent to the laboratory for analysis within 24 h, whereas white blood cells were kept at −80°C until needed. TC, HDL-cholesterol (HDL-C), and triglyceride concentrations were quantified as previously reported [[33](#R33)].

### Determination of LDLR, APOE, and HMGCR genotype

DNA was extracted from white blood cells using an automated 340A DNA extractor (Applied Biosystems, Foster City, California, USA) or a QIAmp Blood Maxi Kit (Qiagen, Toronto, Ontario, Canada) according to the manufacturer’s instructions. In a few cases, DNA was extracted using the Maxwell 16 Cell LEV DNA Purification Kit (Promega, Madison, Wisconsin, USA) on a Maxwell 16 LEV Instrument (Promega). The presence of the deletion of more than 15 kb of the promoter and first exon of the *LDLR* gene was assessed by Southern blotting as previously described [[34](#R34)] or by Sanger sequencing using standard laboratory procedures on a ABI 3130xl Sequencer (Applied Biosystems). Genotype profiling of rs3846662 of the *HMGCR* gene was performed with PCR followed by pyrosequencing as previously described [[32](#R32)]. *APOE* allele determination was also performed by PCR, followed by pyrosequencing using established protocols [[35](#R35)].

### Determination of HMGCR mRNA levels

Blood from patients was drawn under vacuum into PAXgene Blood RNA tubes (VWR International) after an overnight fast of at least 12 h. RNA was extracted using the Maxwell 16 Cell LEV Total RNA Purification Kit (Promega) on a Maxwell 16 LEV Instrument (Promega). The purity and integrity of RNA was estimated using, respectively, the ratio of absorbance values at 260 and 280 nm evaluated on a Biotek Synergy H1 reader (Fisher Scientific, Burlington, Ontario, Canada), and the RNA integrity number determined with a Bio-Rad Experion instrument (Bio-Rad, Hercules, California, USA). All ratios of absorbance were greater than 2, whereas RNA integrity numbers ranged from 6.2 to 10, with 70% of samples greater than 8. Then, cDNA was obtained by reverse transcription on a Multigene thermocycler (Labnet International Inc., Edison, New Jersey, USA) using the High Capacity cDNA RT kit (Applied Biosystems). The thermocycler was set as follows: 10 min at 25°C, 80 min at 37°C, and 5 min at 85°C. Finally, 10 ng of each cDNA was applied in triplicate for real-time PCR performed on a 7500 Fast Real-Time PCR System (Applied Biosystems) with TaqMan Fast Universal PCR Master Mix (Applied Biosystems). The following TaqMan Gene Expression Assays Kits were used: total *HMGCR*, TaqMan assay #Hs00168352_m1 covering exon boundary 6/7; FL *HMGCR* variant, custom assay covering exon boundary 13/14 with forward primer TGTGG CCAGCACCAATAGAG, reverse primer CAACTGGGCCACGAGTCAT, and probe CAGAGCAATAGG TCTTGGT; Δ13 *HMGCR* variant, custom assay covering exon boundary 12/14 with forward primer TCAGAA CCTTCTTCTCTCCAGTACCT, reverse primer CAC CTCCACCAAGCAAGGA, and probe CCTTAC AGGGATTATAATTA; *peptidylprolyl isomerase A* (PPIA), TaqMan assay #Hs99999904_m1; *hypoxanthine phosphoribosyltransferase 1* (HPRT1), TaqMan assay #Hs9999 9909_m1; beta actin (ACTB), TaqMan assay #Hs9999 9903_m1; and *glyceraldehyde-3-phosphate dehydrogenase* (GAPDH), TaqMan assay #Hs99999905_m1. The cycling conditions were 20 s at 95°C, 40 cycles of 3 s at 95°C, and 30 s at 60°C. The relative gene expression of *HMGCR* was determined using the comparative *C*_t_ method (ΔΔ*C*_t_ method, Applied Biosystems User Bulletin #2, December 97). The efficiency of each gene expression assay kit was assessed and corrected accordingly in the equation used to calculate the relative quantities of mRNA [[36](#R36)]. A random sample that was repeated on each qPCR plate was chosen as the calibrator. Selection of the most appropriate endogenous reference gene was performed as previously described [[37](#R37)] on all RNA available, and the geometric mean of *PPIA*, *HPRT1*, and *GAPDH* was used as endogenous reference genes.

## Results

The demographic characteristics of the studied cohort are available in [Table 1](#T1).

### Baseline lipid levels study

Medication-free FH patients carrying the AA genotype had higher levels of TC and LDL-C at baseline (5.43 vs. 4.58 mmol/l for TC, and 5.20 vs. 4.39 mmol/l for LDL-C; *P* < 0.05, [Fig. 1](#F1)). Yet, in individuals with an *APOE4* allele, the effects of the AA genotype at locus rs3846662 were blunted. HDL-C and triglyceride levels remained unchanged by the allele status at rs3846662 (results not shown).

### Fig. 1.

![Fig. 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e854/5138058/329a3c7c1b19/nihms6053f1.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5138058_nihms6053f1.jpg)

Baseline lipid levels of ΔTC (a) and ΔLDL-C (b) in patients with FH (Δ15 kb) according to the rs3846662 genotype and the presence of the APOE4 allele. The Δ values were obtained by subtracting, from the individual lipid value, the estimated 50th percentile lipid value observed in a group of normal Canadians of the same age and sex. Bars represent mean ± SEM * P < 0.05. P values are for Mann–Whitney U-tests. APOE4, apolipoprotein ε4; FH, familial hypercholesterolemia; LDL-C, low-density lipoprotein cholesterol; TC, total cholesterol.

### Efficacy study

As previously reported in an FH population similar to ours [[28](#R28),[38](#R38)], the penetrance function is sex dependent for both the percentage LDL-C reduction (interaction genotype × sex, *P* = 0.008) and the efficacy of statin treatment (interaction genotype × sex, *P* = 0.029). Indeed, we observed that the percentage LDL-C reduction upon statin treatment and the efficacy of statin treatment are significantly decreased in women with the AA genotype compared with women without it (38.4 vs. 46.2% for LDL-C reduction, and 81.3 vs. 98% for efficacy of statin treatment, *P* < 0.05, [Fig. 2](#F2)), whereas the exact opposite tendency can be observed in men (*P* = 0.078, [Fig. 2a](#F2)). Women with the AA genotype thus have a lesser response to statin therapy, whereas we observed no effect in men bearing the AA genotype ([Fig. 2b](#F2)). According to our regression model, the percentage variance in LDL-C reduction and efficacy of statin treatment explained by the interaction sex/AA genotype are 9 and 6%, respectively.

### Fig. 2.

![Fig. 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e854/5138058/ba644521610b/nihms6053f2.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5138058_nihms6053f2.jpg)

LDL-C reduction relative to baseline upon statin treatment (a) and efficacy of statin treatment at reducing LDL-C (b) in patients with FH (Δ15 kb) according to the rs3846662 genotype and sex status. Calculations were performed as follows: % efficacy = [(observed reduction in LDL-C/expected reduction in LDL-C) × 100]; % reduction LDL-C = 100 −[(LDL-C after at least 30 days of treatment/LDL-C before treatment) × 100]. The expected reductions in LDL-C were obtained from each statin monograph. Bars represent mean ± SEM. *P < 0.05, (0.05 > P value < 0.1). P values are for multiple regression models including age, BMI, sex, APOE4 allele status, and baseline LDL-C levels. APOE4, apolipoprotein ε4; FH, familial hypercholesterolemia; LDL-C, low-density lipoprotein cholesterol.

### RNA levels study

Finally, we sought to determine whether the genotype at rs3846662 was associated with changes in the *HMGCR* mRNA levels. Unfortunately, these mRNA levels could not be determined at baseline, but they were efficiently quantified by qPCR using the TaqMan chemistry in FH individuals receiving statin combined with other lipid-lowering therapy (mostly ezetimibe). Our results indicate that the FL *HMGCR* mRNA ratio is significantly increased in patients with the AA genotype (1.0 vs. 1.2, *P* < 0.05, [Fig. 3a](#F3)), whereas only a nonsignificant trend is observed for the Δ13 *HMGCR* mRNA ratio (*P* = 0.084). According to our regression model, the percentage variance in FL *HMGCR* mRNA ratio explained by the AA genotype is 14%. Although no sex-dependent effects were observed on these ratios, one was evident on *HMGCR* overall transcription (interaction genotype × sex for total *HMGCR*, *P* = 0.007; interaction genotype × sex for FL *HMGCR*, *P* = 0.040). Indeed, *HMGCR* mRNA levels ([Fig. 3b–d](#F3)) are increased in women with the AA genotype (significant only for FL *HMGCR* levels, 0.968 vs. 1.446, *P* < 0.05, [Fig. 3c](#F3)), whereas the exact opposite is observed in men with the AA genotype (significant only for total *HMGCR* levels, 0.753 vs. 1.256, *P* < 0.05, [Fig. 3b](#F3)). According to our regression models, the percentage variance in total *HMGCR* and FL *HMGCR* mRNA levels explained by the interaction sex/AA genotype is 24 and 15%, respectively. Although measured at two different points in time, *HMGCR* mRNA levels and the efficacy of statin treatment were correlated ([Fig. 4](#F4)). Interestingly, an increase in the percentage of LDL-C reduction was correlated with an increase in Δ13 *HMGCR* mRNA ratio, irrespective of sex. When the ratios of Δ13 and FL *HMGCR* mRNA are included in our regression models, the percentage variance in LDL-C reduction and efficacy of statin treatment explained by the Δ13 *HMGCR* mRNA ratio are 55% (*P* < 0.001) and 22% (*P* = 0.043), respectively.

### Fig. 3.

![Fig. 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e854/5138058/1db14dfee849/nihms6053f3.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5138058_nihms6053f3.jpg)

Ratios of full-length HMGCR mRNA (a, left) and Δ13 HMGCR mRNA (a, right) in patients with FH (Δ15 kb) treated with statin according to the rs3846662 genotype. mRNA levels of total HMGCR (b), full-length HMGCR variant (c), and Δ13 HMGCR variant (d) in patients with FH (Δ15 kb) treated with statin according to rs3846662 genotype and gender status. Bars represent mean ± SEM. *P < 0.05,**P < 0.01, (0.05 > P value < 0.1). P values are for multiple regression models including age, BMI, sex, and APOE4 allele status. APOE4, apolipoprotein ε4; FH, familial hypercholesterolemia; HMGCR, 3-hydroxy-3-methylglutaryl coenzyme A reductase.

### Fig. 4.

![Fig. 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e854/5138058/900680eb039c/nihms6053f4.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5138058_nihms6053f4.jpg)

Associations of full-length HMGCR mRNA ratio (a) and Δ13 HMGCR mRNA ratio (b) with LDL-C reduction relative to baseline (upper graphs), and efficacy of statin treatment at reducing LDL-C (lower graphs) in patients with FH (Δ15 kb) treated with statin.*P < 0.05. P value is for Pearson’s correlation. FH, familial hypercholesterolemia; FL, full-length; HMGCR, 3-hydroxy-3-methylglutaryl coenzyme A reductase; LDL-C, low-density lipoprotein cholesterol.

## Discussion and conclusion

Current knowledge suggests that response to statin treatment and baseline LDL-C levels of normal, healthy individuals are at their lowest when an A allele is present at locus rs3846662. Indeed, the increased skipping of exon 13 associated with the A allele likely results in lower HMGCR activity. Both a smaller treatment response and a larger uptake of LDL particles via the LDLR pathway are expected when the HMGCR activity levels are smaller ([Fig. 5](#F5)). However, the LDLR pathway being 50% defective in heterozygous FH patients, we expected this equipoise to be disrupted.

### Fig. 5.

![Fig. 5](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e854/5138058/2895e080a26d/nihms6053f5.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5138058_nihms6053f5.jpg)

Expected impact of A or G allele at locus rs3846662 of the HMGCR gene on baseline lipid levels in normal individuals. This model was based on the in-vitro findings reported in Medina and colleagues [12,13,17–19]. Although it is still unclear whether the Δ13 mRNA transcript is translated in vivo into proteins, it does not change its expected result on overall HMGCR activity. If the Δ13 mRNA transcript is not translated into proteins, the decrease in HMGCR activity and sensitivity to statin associated with the A allele would likely result from lower levels of functional HMGCR proteins due to lower levels of FL mRNA transcripts. On the other hand, because the catalytically active HMGCR is found as a tetramer, the translation of the Δ13 mRNA into inactive monomer proteins may lead to the formation of heterotetramers of reduced enzymatic activity and sensitivity to statin [13]. FL, full-length; HMGCR, 3-hydroxy-3-methylglutaryl coenzyme A reductase; LDL-C, low-density lipoprotein cholesterol; LDLR, low-density lipoprotein receptor.

The present study highlighted three particular findings. First, contrary to what is generally observed in normal populations, our results show that *APOE4* does not contribute to variations in plasma baseline lipid concentrations ([Fig. 1](#F1)). This finding is, however, in accordance with several other studies of untreated patients with FH [[31](#R31),[39](#R39)–[41](#R41)]. The higher LDL-C levels seen in normal populations carrying an *APOE4* is not completely understood, but it may result in part from the higher affinity of *APOE4* for the LDLR [[42](#R42),[43](#R43)]. This higher affinity is believed to promote an increased hepatic uptake of VLDL and IDL remnants, resulting in a downregulation of LDLR activity and, consequently, increased LDL-C levels [[44](#R44),[45](#R45)]. As hypothesized by O’Malley and Illingworth, the LDLR in FH patients may be already repressed to a degree that is not further affected by the *APOE4* allele [[31](#R31),[39](#R39)]. That being said, cholesterol requirements of hepatocytes are met by two independent yet inter-related processes: internalization of lipoproteins and de-novo cholesterol synthesis. Given the disruption of the LDLR pathway in FH, we were expecting variations within *HMGCR* to have a considerable impact on baseline lipid levels. Our result indicates that the baseline LDL-C levels of individuals with FH born without an *APOE4* allele show an inverse pattern to what was expected from normal individuals ([Fig. 5](#F5)): patients with FH carrying the AA genotype have higher levels of TC and LDL-C at baseline. In normal individuals, the lower LDL-C levels associated with the AA carriers are believed to result from their lower HMGCR activity that prompts an increased uptake of LDL particles by the LDLR pathway ([Fig. 5](#F5)). In FH, the loss of one functional allele of the *LDLR* gene prevents the LDLR pathway, on which AA carriers rely on, to efficiently clear LDL-C particles from plasma, to exert its beneficial effect. Because the levels of *LDLR* and *HMGCR* transcripts were not quantified in our baseline lipid level study, the reason for the increased LDL-C levels in carriers of the AA genotype with FH remains unclear. Perhaps the lower HMGCR activity combined with the lower LDLR activity that is expected to occur in carriers of the FH-AA genotype prompts the hepatocytes to increase their HMGCR synthesis pathway. In accordance with this hypothesis, the ‘hypercholesterolemic’ effect of the AA genotype is lost in individuals with FH carrying both the AA genotype and an *APOE4* allele, an observation that might be attributed to the fact that individuals carrying an *APOE4* allele absorbed more and synthesized less cholesterol [[46](#R46)–[48](#R48)].

Second, the percentage LDL-C reduction upon statin treatment and the efficacy of statin treatment are significantly decreased in women with the AA genotype compared with women without it, whereas the exact opposite tendency can be observed in men with the AA genotype (although not significantly). Women carrying the AA genotype thus responded similarly to normal individuals ([Fig. 5](#F5)), but men did not. Interestingly, however, both men and women bearing the AA genotype showed higher ratio of FL *HMGCR* mRNA compared with individuals without it ([Fig. 3a](#F3)). Hence, as carrying an AA genotype resulted in decreased response to statin only in women, the effects of rs3846662 on splicing cannot explain the differences observed between men and women. We thus looked at a possible effect of rs3846662 on overall transcription ([Fig. 3b–d](#F3)). We found that, in addition to having a higher ratio of FL *HMGCR* mRNA, women receiving treatment and bearing the AA genotype also showed higher FL *HMGCR* mRNA levels compared with women without it. Hence, the *HMGCR* synthesis pathway appears to be somehow more activated in women carrying the AA genotype compared with women without it, which could explain their lesser response to statin. Once again, the opposite is observed in men. Despite these observations, Pearson’s correlation analyses suggest that, for both men and women, the percentage LDL-C reduction upon statin treatment correlated better with the increase in Δ13 *HMGCR* mRNA ratio rather than with the decrease in FL *HMGCR* mRNA ratio. Although the in-vitro findings reported thus far in the literature have correctly put emphasis on the ratios of *HMGCR* fold changes, our in-vivo study highlights the importance of also reporting *HMGCR* mRNA levels.

The above-mentioned results raise two major questions. First, because the A allele promotes skipping of exon 13 ([Fig. 5](#F5)), we were expecting a lower ratio of FL *HMGCR* mRNA and a higher ratio of Δ13 *HMGCR* mRNA to result in a lower response to statin treatment in carriers of the AA genotype. Rather, it appears as though the ratio and mRNA levels of *HMGCR* are just a mere reflection of the suboptimal statin response observed in women carrying the AA genotype. Thus, our results suggest that the effects mediated by the AA genotype on statin efficacy may not be related solely to its impact on *HMGCR* alternative splicing, at least in FH. Indeed, cholesterol levels are dependent upon the interaction of a plethora of genes (*HMGCR*, *LDLR*, *PCSK9*, etc.), and epistasis between those genes may be responsible for the association between AA and suboptimal statin response in FH women. Evidence for epistasis for rs3846662 has previously been demonstrated: rs3846662 interacts with one locus in hepatic lipase (*LIPC*) to modulate HDL-C levels [[49](#R49)], and rs3846662 modulates Alzheimer’s disease risk in women only [[32](#R32)], a finding suggesting an epistatic interaction between *HMGCR* and genotype defined by the X and Y chromosomes [[50](#R50)]. Of note, Keller *et al*. [[51](#R51)] recently highlighted an association between a polymorphism within the *HMGCR* promoter, rs3761740, and transcriptional activity by SREBP. Yet, we previously confirmed that this polymorphism was in linkage disequilibrium with rs3846662 (linkage disequilibrium 0.97) [[32](#R32)]. These two SNPs might thus act in concert to modulate *HMGCR* mRNA levels, at least in FH, in which the cholesterol metabolism is substantially deregulated. Finally, the penetrance function of the AA genotype appears sex dependent in patients with FH. Even though other genotype sex-dependent differences have been reported previously in patients with FH [[28](#R28),[38](#R38)], our sex difference remains unclear at the moment. Additional studies are required to clarify this issue. However, our sex difference was not unexpected given the fact that the liver is the target organ of statins, and liver metabolism has been shown to be markedly sex-dimorphic [[52](#R52)]. An efficient first-pass uptake by the liver may be more important than a high bioavailability to achieve the statin effect [[53](#R53)]. Thus, sex differences in liver size [[54](#R54)] and metabolism may very well translate into sex differences in statin response, especially for those treated with pro-drugs such as simvastatin.

Third, as was previously reported by others [[12](#R12),[13](#R13)], the variance in statin response is better explained by *HMGCR* alternative splicing rather than by *HMGCR* genotype, especially in patients with FH. Indeed, the percentage variance in LDL-C reduction and efficacy of statin treatment explained by the ratio of Δ13 *HMGCR* mRNA could be as high as 55 and 22%, respectively.

However, several aspects of our study warrant caution, mostly for the third arm of the study (mRNA level study). First, the power of the overall study was not as high as initially planned given lower than expected data retrieval. Given our small sample sizes, our study could only detect reliably large population effects. For example, given an *α* of 0.05 and *N* = 59 (representative of the number of participants in the *APOE4*-negative subgroup), we had a power of 0.50 given an effect size of 0.54 and a power of 0.80 given an effect size of 0.77. This situation increases the likelihood of type 2 errors (failing to obtain a statistical significance when there is a real effect) and the chance of having a large effect size when statistical significance is found [[55](#R55)]. In addition, given our small sample sizes, we performed no type 1 error (obtaining a statistical significance when there is no real effect) correction for multiple testing. Therefore, our findings should be considered as exploratory until they can be cross-validated on an independent study. Second, because the mRNA study did not involve an adjuvant monotherapy arm, we could not evaluate individual ensuing responses to the addition of adjuvant (mostly ezetimibe) on ongoing statin therapy. Thus, for most patients, the result reflects combined therapy (statin/ ezetimibe). Although a review of the literature has revealed no studies suggesting that ezetimibe modifies *HMGCR* mRNA levels, this confounder might still overestimate the percentage of variance in *HMGCR* mRNA levels and statin efficacy explained by the rs3846662 genotype or *HMGCR* mRNA levels. In addition, the purpose of the current study was to investigate the impact of rs3846662 genotype on statin efficacy, no matter which statin was used. While statins are widely different in their chemical structure, metabolism, pharmacological properties, and clinical efficacy, they all block cholesterol synthesis by inhibiting HMGCR [[56](#R56)]. Thus, studies performed with one specific statin should yield results similar to ours. Third, we compared the current *HMGCR* mRNA levels of patients with their past response to statin treatment. In our context, only this procedure allowed an ethical comparison of the *HMGCR* mRNA levels with the efficacy of statin monotherapy. However, as the extent of interindividual variations seems to be largely independent of the dose and drug used [[57](#R57)], two parameters that were modified over the course of years in most of these individuals, we believe that comparison of their current *HMGCR* mRNA levels with their past response to statin therapy is worth investigating.

In conclusion, the genotype at rs3846662 locus contributes to the variability of baseline TC and LDL-C levels observed among individuals with FH carrying the same *LDLR* mutation. The rs3846662 SNP also explains a part of the variance observed in statin response, although *HMGCR* alternative splicing constitutes a better predictor. As reported previously, the rs3846662 SNP appears to be involved in regulation of *HMGCR* exon 13 skipping, but the penetrance function of the AA genotype appears sex dependent in patients with FH. Therefore, the genotype at the rs3846662 locus or *HMGCR* alternative splicing cannot alone predict a patient response to statin therapy, even in a genetically homogenous population such as the French Canadians with FH resulting from the deletion of more than 15 kb of the *LDLR*. Finally, this study highlights the fact that the current model ([Fig. 5](#F5)) explaining the impact of rs3846662 A or G allele is incomplete and cannot be applied in its current form to individuals with FH. Other key players, possibly rs3761740, are implicated in the determination of *HMGCR* mRNA levels and should be further investigated.

## Acknowledgments

The authors thank Martine Gauthier, Hélène L’archevêque, and Jocelyne Fouquette for invaluable assistance in contacting patients and collecting blood samples.

This study was supported by the Canadian Institute of Health Research (V.L., J.P.), the Natural Sciences and Engineering Research Council of Canada (J.P.), the Fondation Levesque (J.P.), the Canadian Stroke Network (V.L.), and the Fondation Leducq Transatlantic Networks of Excellence (L.B., R.D.).

## Footnotes

## References

1. Badimon L, Vilahur G. LDL-cholesterol versus HDL-cholesterol in the atherosclerotic plaque: inflammatory resolution versus thrombotic chaos. Ann N Y Acad Sci. 2012;1254:18–32. doi: 10.1111/j.1749-6632.2012.06480.x.  [DOI](https://doi.org/10.1111/j.1749-6632.2012.06480.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22548566/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20N%20Y%20Acad%20Sci&title=LDL-cholesterol%20versus%20HDL-cholesterol%20in%20the%20atherosclerotic%20plaque:%20inflammatory%20resolution%20versus%20thrombotic%20chaos&author=L%20Badimon&author=G%20Vilahur&volume=1254&publication_year=2012&pages=18-32&pmid=22548566&doi=10.1111/j.1749-6632.2012.06480.x&)

2. Goldstein JL, Brown MS. Regulation of the mevalonate pathway. Nature. 1990;343:425–430. doi: 10.1038/343425a0.  [DOI](https://doi.org/10.1038/343425a0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/1967820/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Regulation%20of%20the%20mevalonate%20pathway&author=JL%20Goldstein&author=MS%20Brown&volume=343&publication_year=1990&pages=425-430&pmid=1967820&doi=10.1038/343425a0&)

3. Jermendy G, Horvath T, Littvay L, Steinbach R, Jermendy AL, Tarnoki AD, et al. Effect of genetic and environmental influences on cardiometabolic risk factors: a twin study. Cardiovasc Diabetol. 2011;10:96. doi: 10.1186/1475-2840-10-96.  [DOI](https://doi.org/10.1186/1475-2840-10-96) | [PMC free article](/articles/PMC3219730/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22050728/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cardiovasc%20Diabetol&title=Effect%20of%20genetic%20and%20environmental%20influences%20on%20cardiometabolic%20risk%20factors:%20a%20twin%20study&author=G%20Jermendy&author=T%20Horvath&author=L%20Littvay&author=R%20Steinbach&author=AL%20Jermendy&volume=10&publication_year=2011&pages=96&pmid=22050728&doi=10.1186/1475-2840-10-96&)

4. Heller DA, de Faire U, Pedersen NL, Dahlen G, McClearn GE. Genetic and environmental influences on serum lipid levels in twins. N Engl J Med. 1993;328:1150–1156. doi: 10.1056/NEJM199304223281603.  [DOI](https://doi.org/10.1056/NEJM199304223281603) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8455681/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Genetic%20and%20environmental%20influences%20on%20serum%20lipid%20levels%20in%20twins&author=DA%20Heller&author=U%20de%20Faire&author=NL%20Pedersen&author=G%20Dahlen&author=GE%20McClearn&volume=328&publication_year=1993&pages=1150-1156&pmid=8455681&doi=10.1056/NEJM199304223281603&)

5. Elder SJ, Lichtenstein AH, Pittas AG, Roberts SB, Fuss PJ, Greenberg AS, et al. Genetic and environmental influences on factors associated with cardiovascular disease and the metabolic syndrome. J Lipid Res. 2009;50:1917–1926. doi: 10.1194/jlr.P900033-JLR200.  [DOI](https://doi.org/10.1194/jlr.P900033-JLR200) | [PMC free article](/articles/PMC2724778/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19372593/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Lipid%20Res&title=Genetic%20and%20environmental%20influences%20on%20factors%20associated%20with%20cardiovascular%20disease%20and%20the%20metabolic%20syndrome&author=SJ%20Elder&author=AH%20Lichtenstein&author=AG%20Pittas&author=SB%20Roberts&author=PJ%20Fuss&volume=50&publication_year=2009&pages=1917-1926&pmid=19372593&doi=10.1194/jlr.P900033-JLR200&)

6. Li S, Duan H, Pang Z, Zhang D, Hjelmborg JV, Tan Q, et al. Heritability of eleven metabolic phenotypes in Danish and Chinese twins: a cross-population comparison. Obesity (Silver Spring) 2013;21:1908–1914. doi: 10.1002/oby.20217.  [DOI](https://doi.org/10.1002/oby.20217) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23686756/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Obesity%20(Silver%20Spring)&title=Heritability%20of%20eleven%20metabolic%20phenotypes%20in%20Danish%20and%20Chinese%20twins:%20a%20cross-population%20comparison&author=S%20Li&author=H%20Duan&author=Z%20Pang&author=D%20Zhang&author=JV%20Hjelmborg&volume=21&publication_year=2013&pages=1908-1914&pmid=23686756&doi=10.1002/oby.20217&)

7. Schmitz G, Schmitz-Madry A, Ugocsai P. Pharmacogenetics and pharmacogenomics of cholesterol-lowering therapy. Curr Opin Lipidol. 2007;18:164–173. doi: 10.1097/MOL.0b013e3280555083.  [DOI](https://doi.org/10.1097/MOL.0b013e3280555083) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17353665/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Opin%20Lipidol&title=Pharmacogenetics%20and%20pharmacogenomics%20of%20cholesterol-lowering%20therapy&author=G%20Schmitz&author=A%20Schmitz-Madry&author=P%20Ugocsai&volume=18&publication_year=2007&pages=164-173&pmid=17353665&doi=10.1097/MOL.0b013e3280555083&)

8. Shear CL, Franklin FA, Stinnett S, Hurley DP, Bradford RH, Chremos AN, et al. Expanded Clinical Evaluation of Lovastatin (EXCEL) study results. Effect of patient characteristics on lovastatin-induced changes in plasma concentrations of lipids and lipoproteins. Circulation. 1992;85:1293–1303. doi: 10.1161/01.cir.85.4.1293.  [DOI](https://doi.org/10.1161/01.cir.85.4.1293) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/1555273/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circulation&title=Expanded%20Clinical%20Evaluation%20of%20Lovastatin%20(EXCEL)%20study%20results.%20Effect%20of%20patient%20characteristics%20on%20lovastatin-induced%20changes%20in%20plasma%20concentrations%20of%20lipids%20and%20lipoproteins&author=CL%20Shear&author=FA%20Franklin&author=S%20Stinnett&author=DP%20Hurley&author=RH%20Bradford&volume=85&publication_year=1992&pages=1293-1303&pmid=1555273&doi=10.1161/01.cir.85.4.1293&)

9. Hunt D, Young P, Simes J, Hague W, Mann S, Owensby D, et al. Benefits of pravastatin on cardiovascular events and mortality in older patients with coronary heart disease are equal to or exceed those seen in younger patients: results from the LIPID trial. Ann Intern Med. 2001;134:931–940. doi: 10.7326/0003-4819-134-10-200105150-00007.  [DOI](https://doi.org/10.7326/0003-4819-134-10-200105150-00007) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11352694/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Intern%20Med&title=Benefits%20of%20pravastatin%20on%20cardiovascular%20events%20and%20mortality%20in%20older%20patients%20with%20coronary%20heart%20disease%20are%20equal%20to%20or%20exceed%20those%20seen%20in%20younger%20patients:%20results%20from%20the%20LIPID%20trial&author=D%20Hunt&author=P%20Young&author=J%20Simes&author=W%20Hague&author=S%20Mann&volume=134&publication_year=2001&pages=931-940&pmid=11352694&doi=10.7326/0003-4819-134-10-200105150-00007&)

10. Simon JA, Lin F, Hulley SB, Blanche PJ, Waters D, Shiboski S, et al. Phenotypic predictors of response to simvastatin therapy among African-Americans and Caucasians: the Cholesterol and Pharmacogenetics (CAP) Study. Am J Cardiol. 2006;97:843–850. doi: 10.1016/j.amjcard.2005.09.134.  [DOI](https://doi.org/10.1016/j.amjcard.2005.09.134) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16516587/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Cardiol&title=Phenotypic%20predictors%20of%20response%20to%20simvastatin%20therapy%20among%20African-Americans%20and%20Caucasians:%20the%20Cholesterol%20and%20Pharmacogenetics%20(CAP)%20Study&author=JA%20Simon&author=F%20Lin&author=SB%20Hulley&author=PJ%20Blanche&author=D%20Waters&volume=97&publication_year=2006&pages=843-850&pmid=16516587&doi=10.1016/j.amjcard.2005.09.134&)

11. Krauss RM, Mangravite LM, Smith JD, Medina MW, Wang D, Guo X, et al. Variation in the 3-hydroxyl-3-methylglutaryl coenzyme a reductase gene is associated with racial differences in low-density lipoprotein cholesterol response to simvastatin treatment. Circulation. 2008;117:1537–1544. doi: 10.1161/CIRCULATIONAHA.107.708388.  [DOI](https://doi.org/10.1161/CIRCULATIONAHA.107.708388) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18332269/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circulation&title=Variation%20in%20the%203-hydroxyl-3-methylglutaryl%20coenzyme%20a%20reductase%20gene%20is%20associated%20with%20racial%20differences%20in%20low-density%20lipoprotein%20cholesterol%20response%20to%20simvastatin%20treatment&author=RM%20Krauss&author=LM%20Mangravite&author=JD%20Smith&author=MW%20Medina&author=D%20Wang&volume=117&publication_year=2008&pages=1537-1544&pmid=18332269&doi=10.1161/CIRCULATIONAHA.107.708388&)

12. Medina MW, Gao F, Ruan W, Rotter JI, Krauss RM. Alternative splicing of 3-hydroxy-3-methylglutaryl coenzyme A reductase is associated with plasma low-density lipoprotein cholesterol response to simvastatin. Circulation. 2008;118:355–362. doi: 10.1161/CIRCULATIONAHA.108.773267.  [DOI](https://doi.org/10.1161/CIRCULATIONAHA.108.773267) | [PMC free article](/articles/PMC2720070/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18559695/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circulation&title=Alternative%20splicing%20of%203-hydroxy-3-methylglutaryl%20coenzyme%20A%20reductase%20is%20associated%20with%20plasma%20low-density%20lipoprotein%20cholesterol%20response%20to%20simvastatin&author=MW%20Medina&author=F%20Gao&author=W%20Ruan&author=JI%20Rotter&author=RM%20Krauss&volume=118&publication_year=2008&pages=355-362&pmid=18559695&doi=10.1161/CIRCULATIONAHA.108.773267&)

13. Medina MW, Krauss RM. The role of HMGCR alternative splicing in statin efficacy. Trends Cardiovasc Med. 2009;19:173–177. doi: 10.1016/j.tcm.2009.10.003.  [DOI](https://doi.org/10.1016/j.tcm.2009.10.003) | [PMC free article](/articles/PMC2805071/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20005478/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Trends%20Cardiovasc%20Med&title=The%20role%20of%20HMGCR%20alternative%20splicing%20in%20statin%20efficacy&author=MW%20Medina&author=RM%20Krauss&volume=19&publication_year=2009&pages=173-177&pmid=20005478&doi=10.1016/j.tcm.2009.10.003&)

14. Johnson JM, Castle J, Garrett-Engele P, Kan Z, Loerch PM, Armour CD, et al. Genome-wide survey of human alternative pre-mRNA splicing with exon junction microarrays. Science. 2003;302:2141–2144. doi: 10.1126/science.1090100.  [DOI](https://doi.org/10.1126/science.1090100) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14684825/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=Genome-wide%20survey%20of%20human%20alternative%20pre-mRNA%20splicing%20with%20exon%20junction%20microarrays&author=JM%20Johnson&author=J%20Castle&author=P%20Garrett-Engele&author=Z%20Kan&author=PM%20Loerch&volume=302&publication_year=2003&pages=2141-2144&pmid=14684825&doi=10.1126/science.1090100&)

15. Istvan ES, Palnitkar M, Buchanan SK, Deisenhofer J. Crystal structure of the catalytic portion of human HMG-CoA reductase: insights into regulation of activity and catalysis. EMBO J. 2000;19:819–830. doi: 10.1093/emboj/19.5.819.  [DOI](https://doi.org/10.1093/emboj/19.5.819) | [PMC free article](/articles/PMC305622/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10698924/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=EMBO%20J&title=Crystal%20structure%20of%20the%20catalytic%20portion%20of%20human%20HMG-CoA%20reductase:%20insights%20into%20regulation%20of%20activity%20and%20catalysis&author=ES%20Istvan&author=M%20Palnitkar&author=SK%20Buchanan&author=J%20Deisenhofer&volume=19&publication_year=2000&pages=819-830&pmid=10698924&doi=10.1093/emboj/19.5.819&)

16. Istvan ES, Deisenhofer J. Structural mechanism for statin inhibition of HMG-CoA reductase. Science. 2001;292:1160–1164. doi: 10.1126/science.1059344.  [DOI](https://doi.org/10.1126/science.1059344) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11349148/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=Structural%20mechanism%20for%20statin%20inhibition%20of%20HMG-CoA%20reductase&author=ES%20Istvan&author=J%20Deisenhofer&volume=292&publication_year=2001&pages=1160-1164&pmid=11349148&doi=10.1126/science.1059344&)

17. Burkhardt R, Kenny EE, Lowe JK, Birkeland A, Josowitz R, Noel M, et al. Common SNPs in HMGCR in micronesians and whites associated with LDL-cholesterol levels affect alternative splicing of exon13. Arterioscler Thromb Vasc Biol. 2008;28:2078–2084. doi: 10.1161/ATVBAHA.108.172288.  [DOI](https://doi.org/10.1161/ATVBAHA.108.172288) | [PMC free article](/articles/PMC2764366/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18802019/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arterioscler%20Thromb%20Vasc%20Biol&title=Common%20SNPs%20in%20HMGCR%20in%20micronesians%20and%20whites%20associated%20with%20LDL-cholesterol%20levels%20affect%20alternative%20splicing%20of%20exon13&author=R%20Burkhardt&author=EE%20Kenny&author=JK%20Lowe&author=A%20Birkeland&author=R%20Josowitz&volume=28&publication_year=2008&pages=2078-2084&pmid=18802019&doi=10.1161/ATVBAHA.108.172288&)

18. Yu CY, Theusch E, Lo K, Mangravite LM, Naidoo D, Kutilova M, et al. HNRNPA1 regulates HMGCR alternative splicing and modulates cellular cholesterol metabolism. Hum Mol Genet. 2014;23:319–332. doi: 10.1093/hmg/ddt422.  [DOI](https://doi.org/10.1093/hmg/ddt422) | [PMC free article](/articles/PMC3869353/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24001602/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mol%20Genet&title=HNRNPA1%20regulates%20HMGCR%20alternative%20splicing%20and%20modulates%20cellular%20cholesterol%20metabolism&author=CY%20Yu&author=E%20Theusch&author=K%20Lo&author=LM%20Mangravite&author=D%20Naidoo&volume=23&publication_year=2014&pages=319-332&pmid=24001602&doi=10.1093/hmg/ddt422&)

19. Aulchenko YS, Ripatti S, Lindqvist I, Boomsma D, Heid IM, Pramstaller PP, et al. Loci influencing lipid levels and coronary heart disease risk in 16 European population cohorts. Nat Genet. 2009;41:47–55. doi: 10.1038/ng.269.  [DOI](https://doi.org/10.1038/ng.269) | [PMC free article](/articles/PMC2687074/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19060911/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Loci%20influencing%20lipid%20levels%20and%20coronary%20heart%20disease%20risk%20in%2016%20European%20population%20cohorts&author=YS%20Aulchenko&author=S%20Ripatti&author=I%20Lindqvist&author=D%20Boomsma&author=IM%20Heid&volume=41&publication_year=2009&pages=47-55&pmid=19060911&doi=10.1038/ng.269&)

20. Hiura Y, Tabara Y, Kokubo Y, Okamura T, Goto Y, Nonogi H, et al. Association of the functional variant in the 3-hydroxy-3-methylglutaryl-coenzyme a reductase gene with low-density lipoprotein-cholesterol in Japanese. Circ J. 2010;74:518–522. doi: 10.1253/circj.cj-09-0790.  [DOI](https://doi.org/10.1253/circj.cj-09-0790) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20145341/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circ%20J&title=Association%20of%20the%20functional%20variant%20in%20the%203-hydroxy-3-methylglutaryl-coenzyme%20a%20reductase%20gene%20with%20low-density%20lipoprotein-cholesterol%20in%20Japanese&author=Y%20Hiura&author=Y%20Tabara&author=Y%20Kokubo&author=T%20Okamura&author=Y%20Goto&volume=74&publication_year=2010&pages=518-522&pmid=20145341&doi=10.1253/circj.cj-09-0790&)

21. Lu Y, Feskens EJ, Boer JM, Imholz S, Verschuren WM, Wijmenga C, et al. Exploring genetic determinants of plasma total cholesterol levels and their predictive value in a longitudinal study. Atherosclerosis. 2010;213:200–205. doi: 10.1016/j.atherosclerosis.2010.08.053.  [DOI](https://doi.org/10.1016/j.atherosclerosis.2010.08.053) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20832063/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Atherosclerosis&title=Exploring%20genetic%20determinants%20of%20plasma%20total%20cholesterol%20levels%20and%20their%20predictive%20value%20in%20a%20longitudinal%20study&author=Y%20Lu&author=EJ%20Feskens&author=JM%20Boer&author=S%20Imholz&author=WM%20Verschuren&volume=213&publication_year=2010&pages=200-205&pmid=20832063&doi=10.1016/j.atherosclerosis.2010.08.053&)

22. Chasman DI, Posada D, Subrahmanyan L, Cook NR, Stanton VP, Jr, Ridker PM. Pharmacogenetic study of statin therapy and cholesterol reduction. JAMA. 2004;291:2821–2827. doi: 10.1001/jama.291.23.2821.  [DOI](https://doi.org/10.1001/jama.291.23.2821) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15199031/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA&title=Pharmacogenetic%20study%20of%20statin%20therapy%20and%20cholesterol%20reduction&author=DI%20Chasman&author=D%20Posada&author=L%20Subrahmanyan&author=NR%20Cook&author=VP%20Stanton&volume=291&publication_year=2004&pages=2821-2827&pmid=15199031&doi=10.1001/jama.291.23.2821&)

23. Chung JY, Cho SK, Oh ES, Lee DH, Lim LA, Jang SB, et al. Effect of HMGCR variant alleles on low-density lipoprotein cholesterol-lowering response to atorvastatin in healthy Korean subjects. J Clin Pharmacol. 2012;52:339–346. doi: 10.1177/0091270011398239.  [DOI](https://doi.org/10.1177/0091270011398239) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21427285/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Pharmacol&title=Effect%20of%20HMGCR%20variant%20alleles%20on%20low-density%20lipoprotein%20cholesterol-lowering%20response%20to%20atorvastatin%20in%20healthy%20Korean%20subjects&author=JY%20Chung&author=SK%20Cho&author=ES%20Oh&author=DH%20Lee&author=LA%20Lim&volume=52&publication_year=2012&pages=339-346&pmid=21427285&doi=10.1177/0091270011398239&)

24. Medina MW, Gao F, Naidoo D, Rudel LL, Temel RE, McDaniel AL, et al. Coordinately regulated alternative splicing of genes involved in cholesterol biosynthesis and uptake. PLoS One. 2011;6:e19420. doi: 10.1371/journal.pone.0019420.  [DOI](https://doi.org/10.1371/journal.pone.0019420) | [PMC free article](/articles/PMC3084847/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21559365/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=Coordinately%20regulated%20alternative%20splicing%20of%20genes%20involved%20in%20cholesterol%20biosynthesis%20and%20uptake&author=MW%20Medina&author=F%20Gao&author=D%20Naidoo&author=LL%20Rudel&author=RE%20Temel&volume=6&publication_year=2011&pages=e19420&pmid=21559365&doi=10.1371/journal.pone.0019420&)

25. Marais AD. Familial hypercholesterolaemia. Clin Biochem Rev. 2004;25:49–68.  [PMC free article](/articles/PMC1853359/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18516203/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Biochem%20Rev&title=Familial%20hypercholesterolaemia&author=AD%20Marais&volume=25&publication_year=2004&pages=49-68&pmid=18516203&)

26. Hobbs HH, Brown MS, Russell DW, Davignon J, Goldstein JL. Deletion in the gene for the low-density-lipoprotein receptor in a majority of French Canadians with familial hypercholesterolemia. N Engl J Med. 1987;317:734–737. doi: 10.1056/NEJM198709173171204.  [DOI](https://doi.org/10.1056/NEJM198709173171204) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/3627182/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Deletion%20in%20the%20gene%20for%20the%20low-density-lipoprotein%20receptor%20in%20a%20majority%20of%20French%20Canadians%20with%20familial%20hypercholesterolemia&author=HH%20Hobbs&author=MS%20Brown&author=DW%20Russell&author=J%20Davignon&author=JL%20Goldstein&volume=317&publication_year=1987&pages=734-737&pmid=3627182&doi=10.1056/NEJM198709173171204&)

27. Hobbs HH, Russell DW, Brown MS, Goldstein JL. The LDL receptor locus in familial hypercholesterolemia: mutational analysis of a membrane protein. Annu Rev Genet. 1990;24:133–170. doi: 10.1146/annurev.ge.24.120190.001025.  [DOI](https://doi.org/10.1146/annurev.ge.24.120190.001025) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/2088165/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Annu%20Rev%20Genet&title=The%20LDL%20receptor%20locus%20in%20familial%20hypercholesterolemia:%20mutational%20analysis%20of%20a%20membrane%20protein&author=HH%20Hobbs&author=DW%20Russell&author=MS%20Brown&author=JL%20Goldstein&volume=24&publication_year=1990&pages=133-170&pmid=2088165&doi=10.1146/annurev.ge.24.120190.001025&)

28. Roy M, Sing CF, Betard C, Davignon J. Impact of a common mutation of the LDL receptor gene, in French-Canadian patients with familial hypercholesterolemia, on means, variances and correlations among traits of lipid metabolism. Clin Genet. 1995;47:59–67. doi: 10.1111/j.1399-0004.1995.tb03925.x.  [DOI](https://doi.org/10.1111/j.1399-0004.1995.tb03925.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7606845/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Genet&title=Impact%20of%20a%20common%20mutation%20of%20the%20LDL%20receptor%20gene,%20in%20French-Canadian%20patients%20with%20familial%20hypercholesterolemia,%20on%20means,%20variances%20and%20correlations%20among%20traits%20of%20lipid%20metabolism&author=M%20Roy&author=CF%20Sing&author=C%20Betard&author=J%20Davignon&volume=47&publication_year=1995&pages=59-67&pmid=7606845&doi=10.1111/j.1399-0004.1995.tb03925.x&)

29. Betard C, Kessling AM, Roy M, Davignon J. Influence of genetic variability in the nondeletion LDL-receptor allele on phenotypic variation in French- Canadian familial hypercholesterolemia heterozygotes sharing a ‘null’ LDL-receptor gene defect. Atherosclerosis. 1996;119:43–55. doi: 10.1016/0021-9150(95)05627-0.  [DOI](https://doi.org/10.1016/0021-9150(95)05627-0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8929255/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Atherosclerosis&title=Influence%20of%20genetic%20variability%20in%20the%20nondeletion%20LDL-receptor%20allele%20on%20phenotypic%20variation%20in%20French-%20Canadian%20familial%20hypercholesterolemia%20heterozygotes%20sharing%20a%20%E2%80%98null%E2%80%99%20LDL-receptor%20gene%20defect&author=C%20Betard&author=AM%20Kessling&author=M%20Roy&author=J%20Davignon&volume=119&publication_year=1996&pages=43-55&pmid=8929255&doi=10.1016/0021-9150(95)05627-0&)

30. Ferrieres J, Sing CF, Roy M, Davignon J, Lussier-Cacan S. Apolipoprotein E polymorphism and heterozygous familial hypercholesterolemia. Sex-specific effects. Arterioscler Thromb. 1994;14:1553–1560. doi: 10.1161/01.atv.14.10.1553.  [DOI](https://doi.org/10.1161/01.atv.14.10.1553) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7918304/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arterioscler%20Thromb&title=Apolipoprotein%20E%20polymorphism%20and%20heterozygous%20familial%20hypercholesterolemia.%20Sex-specific%20effects&author=J%20Ferrieres&author=CF%20Sing&author=M%20Roy&author=J%20Davignon&author=S%20Lussier-Cacan&volume=14&publication_year=1994&pages=1553-1560&pmid=7918304&doi=10.1161/01.atv.14.10.1553&)

31. Carmena R, Roederer G, Mailloux H, Lussier-Cacan S, Davignon J. The response to lovastatin treatment in patients with heterozygous familial hypercholesterolemia is modulated by apolipoprotein E polymorphism. Metabolism. 1993;42:895–901. doi: 10.1016/0026-0495(93)90066-w.  [DOI](https://doi.org/10.1016/0026-0495(93)90066-w) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8345800/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Metabolism&title=The%20response%20to%20lovastatin%20treatment%20in%20patients%20with%20heterozygous%20familial%20hypercholesterolemia%20is%20modulated%20by%20apolipoprotein%20E%20polymorphism&author=R%20Carmena&author=G%20Roederer&author=H%20Mailloux&author=S%20Lussier-Cacan&author=J%20Davignon&volume=42&publication_year=1993&pages=895-901&pmid=8345800&doi=10.1016/0026-0495(93)90066-w&)

32. Leduc V, De Beaumont L, Theroux L, Dea D, Aisen P, Petersen RC, et al. HMGCR is a genetic modifier for risk, age of onset and MCI conversion to Alzheimer’s disease in a three cohorts study. Mol Psychiatry. 2015;20:867–873. doi: 10.1038/mp.2014.81.  [DOI](https://doi.org/10.1038/mp.2014.81) | [PMC free article](/articles/PMC4318698/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25023145/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Psychiatry&title=HMGCR%20is%20a%20genetic%20modifier%20for%20risk,%20age%20of%20onset%20and%20MCI%20conversion%20to%20Alzheimer%E2%80%99s%20disease%20in%20a%20three%20cohorts%20study&author=V%20Leduc&author=L%20De%20Beaumont&author=L%20Theroux&author=D%20Dea&author=P%20Aisen&volume=20&publication_year=2015&pages=867-873&pmid=25023145&doi=10.1038/mp.2014.81&)

33. Saavedra YG, Dufour R, Davignon J, Baass A. PCSK9 R46L, lower LDL, and cardiovascular disease risk in familial hypercholesterolemia: a cross-sectional cohort study. Arterioscler Thromb Vasc Biol. 2014;34:2700–2705. doi: 10.1161/ATVBAHA.114.304406.  [DOI](https://doi.org/10.1161/ATVBAHA.114.304406) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25278291/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arterioscler%20Thromb%20Vasc%20Biol&title=PCSK9%20R46L,%20lower%20LDL,%20and%20cardiovascular%20disease%20risk%20in%20familial%20hypercholesterolemia:%20a%20cross-sectional%20cohort%20study&author=YG%20Saavedra&author=R%20Dufour&author=J%20Davignon&author=A%20Baass&volume=34&publication_year=2014&pages=2700-2705&pmid=25278291&doi=10.1161/ATVBAHA.114.304406&)

34. Ma YH, Betard C, Roy M, Davignon J, Kessling AM. Identification of a second ‘French Canadian’ LDL receptor gene deletion and development of a rapid method to detect both deletions. Clin Genet. 1989;36:219–228. doi: 10.1111/j.1399-0004.1989.tb03194.x.  [DOI](https://doi.org/10.1111/j.1399-0004.1989.tb03194.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/2805380/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Genet&title=Identification%20of%20a%20second%20%E2%80%98French%20Canadian%E2%80%99%20LDL%20receptor%20gene%20deletion%20and%20development%20of%20a%20rapid%20method%20to%20detect%20both%20deletions&author=YH%20Ma&author=C%20Betard&author=M%20Roy&author=J%20Davignon&author=AM%20Kessling&volume=36&publication_year=1989&pages=219-228&pmid=2805380&doi=10.1111/j.1399-0004.1989.tb03194.x&)

35. Petersen RC, Thomas RG, Grundman M, Bennett D, Doody R, Ferris S, et al. Vitamin E and donepezil for the treatment of mild cognitive impairment. N Engl J Med. 2005;352:2379–2388. doi: 10.1056/NEJMoa050151.  [DOI](https://doi.org/10.1056/NEJMoa050151) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15829527/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Vitamin%20E%20and%20donepezil%20for%20the%20treatment%20of%20mild%20cognitive%20impairment&author=RC%20Petersen&author=RG%20Thomas&author=M%20Grundman&author=D%20Bennett&author=R%20Doody&volume=352&publication_year=2005&pages=2379-2388&pmid=15829527&doi=10.1056/NEJMoa050151&)

36. Pfaffl MW, Horgan GW, Dempfle L. Relative expression software tool (REST) for group-wise comparison and statistical analysis of relative expression results in real-time PCR. Nucleic Acids Res. 2002;30:e36. doi: 10.1093/nar/30.9.e36.  [DOI](https://doi.org/10.1093/nar/30.9.e36) | [PMC free article](/articles/PMC113859/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11972351/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=Relative%20expression%20software%20tool%20(REST)%20for%20group-wise%20comparison%20and%20statistical%20analysis%20of%20relative%20expression%20results%20in%20real-time%20PCR&author=MW%20Pfaffl&author=GW%20Horgan&author=L%20Dempfle&volume=30&publication_year=2002&pages=e36&pmid=11972351&doi=10.1093/nar/30.9.e36&)

37. Leduc V, Legault V, Dea D, Poirier J. Normalization of gene expression using SYBR green qPCR: a case for paraoxonase 1 and 2 in Alzheimer’s disease brains. J Neurosci Methods. 2011;200:14–19. doi: 10.1016/j.jneumeth.2011.05.026.  [DOI](https://doi.org/10.1016/j.jneumeth.2011.05.026) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21672555/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Neurosci%20Methods&title=Normalization%20of%20gene%20expression%20using%20SYBR%20green%20qPCR:%20a%20case%20for%20paraoxonase%201%20and%202%20in%20Alzheimer%E2%80%99s%20disease%20brains&author=V%20Leduc&author=V%20Legault&author=D%20Dea&author=J%20Poirier&volume=200&publication_year=2011&pages=14-19&pmid=21672555&doi=10.1016/j.jneumeth.2011.05.026&)

38. Ferrieres J, Lambert J, Lussier-Cacan S, Davignon J. Coronary artery disease in heterozygous familial hypercholesterolemia patients with the same LDL receptor gene mutation. Circulation. 1995;92:290–295. doi: 10.1161/01.cir.92.3.290.  [DOI](https://doi.org/10.1161/01.cir.92.3.290) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7634440/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circulation&title=Coronary%20artery%20disease%20in%20heterozygous%20familial%20hypercholesterolemia%20patients%20with%20the%20same%20LDL%20receptor%20gene%20mutation&author=J%20Ferrieres&author=J%20Lambert&author=S%20Lussier-Cacan&author=J%20Davignon&volume=92&publication_year=1995&pages=290-295&pmid=7634440&doi=10.1161/01.cir.92.3.290&)

39. O’Malley JP, Illingworth DR. The influence of apolipoprotein E phenotype on the response to lovastatin therapy in patients with heterozygous familial hypercholesterolemia. Metabolism. 1990;39:150–154. doi: 10.1016/0026-0495(90)90068-n.  [DOI](https://doi.org/10.1016/0026-0495(90)90068-n) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/2299987/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Metabolism&title=The%20influence%20of%20apolipoprotein%20E%20phenotype%20on%20the%20response%20to%20lovastatin%20therapy%20in%20patients%20with%20heterozygous%20familial%20hypercholesterolemia&author=JP%20O%E2%80%99Malley&author=DR%20Illingworth&volume=39&publication_year=1990&pages=150-154&pmid=2299987&doi=10.1016/0026-0495(90)90068-n&)

40. De Knijff P, Stalenhoef AF, Mol MJ, Gevers Leuven JA, Smit J, Erkelens DW, et al. Influence of apo E polymorphism on the response to simvastatin treatment in patients with heterozygous familial hypercholesterolemia. Atherosclerosis. 1990;83:89–97. doi: 10.1016/0021-9150(90)90134-5.  [DOI](https://doi.org/10.1016/0021-9150(90)90134-5) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/2390138/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Atherosclerosis&title=Influence%20of%20apo%20E%20polymorphism%20on%20the%20response%20to%20simvastatin%20treatment%20in%20patients%20with%20heterozygous%20familial%20hypercholesterolemia&author=P%20De%20Knijff&author=AF%20Stalenhoef&author=MJ%20Mol&author=JA%20Gevers%20Leuven&author=J%20Smit&volume=83&publication_year=1990&pages=89-97&pmid=2390138&doi=10.1016/0021-9150(90)90134-5&)

41. Dallongeville J, Roy M, Leboeuf N, Xhignesse M, Davignon J, Lussier-Cacan S. Apolipoprotein E polymorphism association with lipoprotein profile in endogenous hypertriglyceridemia and familial hypercholesterolemia. Arterioscler Thromb. 1991;11:272–278. doi: 10.1161/01.atv.11.2.272.  [DOI](https://doi.org/10.1161/01.atv.11.2.272) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/1998645/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arterioscler%20Thromb&title=Apolipoprotein%20E%20polymorphism%20association%20with%20lipoprotein%20profile%20in%20endogenous%20hypertriglyceridemia%20and%20familial%20hypercholesterolemia&author=J%20Dallongeville&author=M%20Roy&author=N%20Leboeuf&author=M%20Xhignesse&author=J%20Davignon&volume=11&publication_year=1991&pages=272-278&pmid=1998645&doi=10.1161/01.atv.11.2.272&)

42. Innerarity TL, Friedlander EJ, Rall SC, Jr, Weisgraber KH, Mahley RW. The receptor-binding domain of human apolipoprotein E. Binding of apolipoprotein E fragments. J Biol Chem. 1983;258:12341–12347.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/6313652/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=The%20receptor-binding%20domain%20of%20human%20apolipoprotein%20E.%20Binding%20of%20apolipoprotein%20E%20fragments&author=TL%20Innerarity&author=EJ%20Friedlander&author=SC%20Rall&author=KH%20Weisgraber&author=RW%20Mahley&volume=258&publication_year=1983&pages=12341-12347&pmid=6313652&)

43. Yamamoto T, Choi HW, Ryan RO. Apolipoprotein E isoform-specific binding to the low-density lipoprotein receptor. Anal Biochem. 2008;372:222–226. doi: 10.1016/j.ab.2007.09.005.  [DOI](https://doi.org/10.1016/j.ab.2007.09.005) | [PMC free article](/articles/PMC2174959/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17923100/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Anal%20Biochem&title=Apolipoprotein%20E%20isoform-specific%20binding%20to%20the%20low-density%20lipoprotein%20receptor&author=T%20Yamamoto&author=HW%20Choi&author=RO%20Ryan&volume=372&publication_year=2008&pages=222-226&pmid=17923100&doi=10.1016/j.ab.2007.09.005&)

44. Nestruck AC, Bouthillier D, Sing CF, Davignon J. Apolipoprotein E polymorphism and plasma cholesterol response to probucol. Metabolism. 1987;36:743–747. doi: 10.1016/0026-0495(87)90110-7.  [DOI](https://doi.org/10.1016/0026-0495(87)90110-7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/3600286/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Metabolism&title=Apolipoprotein%20E%20polymorphism%20and%20plasma%20cholesterol%20response%20to%20probucol&author=AC%20Nestruck&author=D%20Bouthillier&author=CF%20Sing&author=J%20Davignon&volume=36&publication_year=1987&pages=743-747&pmid=3600286&doi=10.1016/0026-0495(87)90110-7&)

45. Kwiterovich POJ. The Johns Hopkins textbook of dyslipidemia. 1. New York: Lippincott Williams & Wilkins; 2012.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=The%20Johns%20Hopkins%20textbook%20of%20dyslipidemia&author=POJ%20Kwiterovich&publication_year=2012&)

46. Kesaniemi YA, Ehnholm C, Miettinen TA. Intestinal cholesterol absorption efficiency in man is related to apoprotein E phenotype. J Clin Invest. 1987;80:578–581. doi: 10.1172/JCI113107.  [DOI](https://doi.org/10.1172/JCI113107) | [PMC free article](/articles/PMC442272/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/3611358/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Invest&title=Intestinal%20cholesterol%20absorption%20efficiency%20in%20man%20is%20related%20to%20apoprotein%20E%20phenotype&author=YA%20Kesaniemi&author=C%20Ehnholm&author=TA%20Miettinen&volume=80&publication_year=1987&pages=578-581&pmid=3611358&doi=10.1172/JCI113107&)

47. Sarkkinen E, Korhonen M, Erkkila A, Ebeling T, Uusitupa M. Effect of apolipoprotein E polymorphism on serum lipid response to the separate modification of dietary fat and dietary cholesterol. Am J Clin Nutr. 1998;68:1215–1222. doi: 10.1093/ajcn/68.6.1215.  [DOI](https://doi.org/10.1093/ajcn/68.6.1215) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9846849/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Clin%20Nutr&title=Effect%20of%20apolipoprotein%20E%20polymorphism%20on%20serum%20lipid%20response%20to%20the%20separate%20modification%20of%20dietary%20fat%20and%20dietary%20cholesterol&author=E%20Sarkkinen&author=M%20Korhonen&author=A%20Erkkila&author=T%20Ebeling&author=M%20Uusitupa&volume=68&publication_year=1998&pages=1215-1222&pmid=9846849&doi=10.1093/ajcn/68.6.1215&)

48. O’Neill FH, Patel DD, Knight BL, Neuwirth CK, Bourbon M, Soutar AK, et al. Determinants of variable response to statin treatment in patients with refractory familial hypercholesterolemia. Arterioscler Thromb Vasc Biol. 2001;21:832–837. doi: 10.1161/01.atv.21.5.832.  [DOI](https://doi.org/10.1161/01.atv.21.5.832) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11348882/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arterioscler%20Thromb%20Vasc%20Biol&title=Determinants%20of%20variable%20response%20to%20statin%20treatment%20in%20patients%20with%20refractory%20familial%20hypercholesterolemia&author=FH%20O%E2%80%99Neill&author=DD%20Patel&author=BL%20Knight&author=CK%20Neuwirth&author=M%20Bourbon&volume=21&publication_year=2001&pages=832-837&pmid=11348882&doi=10.1161/01.atv.21.5.832&)

49. Ma L, Brautbar A, Boerwinkle E, Sing CF, Clark AG, Keinan A. Knowledge-driven analysis identifies a gene-gene interaction affecting high-density lipoprotein cholesterol levels in multi-ethnic populations. PLoS Genet. 2012;8:e1002714. doi: 10.1371/journal.pgen.1002714.  [DOI](https://doi.org/10.1371/journal.pgen.1002714) | [PMC free article](/articles/PMC3359971/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22654671/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet&title=Knowledge-driven%20analysis%20identifies%20a%20gene-gene%20interaction%20affecting%20high-density%20lipoprotein%20cholesterol%20levels%20in%20multi-ethnic%20populations&author=L%20Ma&author=A%20Brautbar&author=E%20Boerwinkle&author=CF%20Sing&author=AG%20Clark&volume=8&publication_year=2012&pages=e1002714&pmid=22654671&doi=10.1371/journal.pgen.1002714&)

50. Wolf JB, Brodie ED, Wade MJ. Epistasis and the evolutionary process. Oxford, UK; New York: Oxford University Press; 2000.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Epistasis%20and%20the%20evolutionary%20process&author=JB%20Wolf&author=ED%20Brodie&author=MJ%20Wade&publication_year=2000&)

51. Keller L, Murphy C, Wang HX, Fratiglioni L, Olin M, Gafvels M, et al. A functional polymorphism in the HMGCR promoter affects transcriptional activity but not the risk for Alzheimer disease in Swedish populations. Brain Res. 2010;1344:185–191. doi: 10.1016/j.brainres.2010.04.073.  [DOI](https://doi.org/10.1016/j.brainres.2010.04.073) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20450896/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Brain%20Res&title=A%20functional%20polymorphism%20in%20the%20HMGCR%20promoter%20affects%20transcriptional%20activity%20but%20not%20the%20risk%20for%20Alzheimer%20disease%20in%20Swedish%20populations&author=L%20Keller&author=C%20Murphy&author=HX%20Wang&author=L%20Fratiglioni&author=M%20Olin&volume=1344&publication_year=2010&pages=185-191&pmid=20450896&doi=10.1016/j.brainres.2010.04.073&)

52. Mode A, Gustafsson JA. Sex and the liver – a journey through five decades. Drug Metab Rev. 2006;38:197–207. doi: 10.1080/03602530600570057.  [DOI](https://doi.org/10.1080/03602530600570057) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16684657/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Rev&title=Sex%20and%20the%20liver%20%E2%80%93%20a%20journey%20through%20five%20decades&author=A%20Mode&author=JA%20Gustafsson&volume=38&publication_year=2006&pages=197-207&pmid=16684657&doi=10.1080/03602530600570057&)

53. Gazzerro P, Proto MC, Gangemi G, Malfitano AM, Ciaglia E, Pisanti S, et al. Pharmacological actions of statins: a critical appraisal in the management of cancer. Pharmacol Rev. 2012;64:102–146. doi: 10.1124/pr.111.004994.  [DOI](https://doi.org/10.1124/pr.111.004994) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22106090/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacol%20Rev&title=Pharmacological%20actions%20of%20statins:%20a%20critical%20appraisal%20in%20the%20management%20of%20cancer&author=P%20Gazzerro&author=MC%20Proto&author=G%20Gangemi&author=AM%20Malfitano&author=E%20Ciaglia&volume=64&publication_year=2012&pages=102-146&pmid=22106090&doi=10.1124/pr.111.004994&)

54. Chouker A, Martignoni A, Dugas M, Eisenmenger W, Schauer R, Kaufmann I, et al. Estimation of liver size for liver transplantation: the impact of age and gender. Liver Transpl. 2004;10:678–685. doi: 10.1002/lt.20113.  [DOI](https://doi.org/10.1002/lt.20113) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15108261/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Liver%20Transpl&title=Estimation%20of%20liver%20size%20for%20liver%20transplantation:%20the%20impact%20of%20age%20and%20gender&author=A%20Chouker&author=A%20Martignoni&author=M%20Dugas&author=W%20Eisenmenger&author=R%20Schauer&volume=10&publication_year=2004&pages=678-685&pmid=15108261&doi=10.1002/lt.20113&)

55. Gelman A, Weakliem D. Of beauty, sex, and power. American Scientist. 2009;97:310–316.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=American%20Scientist&title=Of%20beauty,%20sex,%20and%20power&author=A%20Gelman&author=D%20Weakliem&volume=97&publication_year=2009&pages=310-316&)

56. Kapur NK, Musunuru K. Clinical efficacy and safety of statins in managing cardiovascular risk. Vasc Health Risk Manag. 2008;4:341–353. doi: 10.2147/vhrm.s1653.  [DOI](https://doi.org/10.2147/vhrm.s1653) | [PMC free article](/articles/PMC2496987/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18561510/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Vasc%20Health%20Risk%20Manag&title=Clinical%20efficacy%20and%20safety%20of%20statins%20in%20managing%20cardiovascular%20risk&author=NK%20Kapur&author=K%20Musunuru&volume=4&publication_year=2008&pages=341-353&pmid=18561510&doi=10.2147/vhrm.s1653&)

57. Naoumova RP, Marais AD, Mountney J, Firth JC, Rendell NB, Taylor GW, et al. Plasma mevalonic acid, an index of cholesterol synthesis in vivo, and responsiveness to HMG-CoA reductase inhibitors in familial hypercholesterolaemia. Atherosclerosis. 1996;119:203–213. doi: 10.1016/0021-9150(95)05649-1.  [DOI](https://doi.org/10.1016/0021-9150(95)05649-1) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8808497/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Atherosclerosis&title=Plasma%20mevalonic%20acid,%20an%20index%20of%20cholesterol%20synthesis%20in%20vivo,%20and%20responsiveness%20to%20HMG-CoA%20reductase%20inhibitors%20in%20familial%20hypercholesterolaemia&author=RP%20Naoumova&author=AD%20Marais&author=J%20Mountney&author=JC%20Firth&author=NB%20Rendell&volume=119&publication_year=1996&pages=203-213&pmid=8808497&doi=10.1016/0021-9150(95)05649-1&)
