# Dihydropyrimidine dehydrogenase pharmacogenetics for predicting fluoropyrimidine-related toxicity in the randomised, phase III adjuvant TOSCA trial in high-risk colon cancer patients

## Metadata
**Authors:** A Ruzzo, F Graziano, Fabio Galli, Francesca Galli, E Rulli, S Lonardi, M Ronzoni, B Massidda, V Zagonel, N Pella, C Mucciarini, R Labianca, M T Ionta, I Bagaloni, E Veltri, P Sozzi, S Barni, V Ricci, L Foltran, M Nicolini, E Biondi, A Bramati, D Turci, S Lazzarelli, C Verusio, F Bergamo, A Sobrero, L Frontini, M Menghi, M Magnani
**Journal:** British Journal of Cancer
**Date:** 2017 Aug 24
**DOI:** [10.1038/bjc.2017.289](https://doi.org/10.1038/bjc.2017.289)
**PMID:** 29065426
**PMCID:** PMC5709672
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5709672/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC5709672/pdf/bjc2017289a.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC5709672/pdf/bjc2017289a.pdf)

## Abstract

**Background:** 
Dihydropyrimidine dehydrogenase (DPD) catabolises ∼85% of the administered dose of fluoropyrimidines. Functional DPYD gene variants cause reduced/abrogated DPD activity. DPYD variants analysis may help for defining individual patients’ risk of fluoropyrimidine-related severe toxicity.

**Methods:** 
The TOSCA Italian randomised trial enrolled colon cancer patients for 3 or 6 months of either FOLFOX-4 or XELOX adjuvant chemotherapy. In an ancillary pharmacogenetic study, 10 DPYD variants (*2A rs3918290 G>A, *13 rs55886062 T>G, rs67376798 A>T, *4 rs1801158 G>A, *5 rs1801159 A>G, *6 rs1801160 G>A, *9A rs1801265 T>C, rs2297595 A>G, rs17376848 T>C, and rs75017182 C>G), were retrospectively tested for associations with ⩾grade 3 fluoropyrimidine-related adverse events (FAEs). An association analysis and a time-to-toxicity (TTT) analysis were planned. To adjust for multiple testing, the Benjamini and Hochberg’s False Discovery Rate (FDR) procedure was used.

**Results:** 
FAEs occurred in 194 out of 508 assessable patients (38.2%). In the association analysis, FAEs occurred more frequently in *6 rs1801160 A allele carriers (FDR=0.0083). At multivariate TTT analysis, significant associations were found for *6 rs1801160 A allele carriers (FDR<0.0001), *2A rs3918290 A allele carriers (FDR<0.0001), and rs2297595 GG genotype carriers (FDR=0.0014). Neutropenia was the most common FAEs (28.5%). *6 rs1801160 (FDR<0.0001), and *2A rs3918290 (FDR=0.0004) variant alleles were significantly associated with time to neutropenia.

**Conclusions:** 
This study adds evidence on the role of DPYD pharmacogenetics for safety of patients undergoing fluoropyrimidine-based chemotherapy.

Keywords: colorectal cancer, pharmacogenetics, toxicity, chemotherapy, dihydropyrimidine dehydrogenase, polymorphisms, 5-fluorouracil, capecitabine

### Background:

Dihydropyrimidine dehydrogenase (DPD) catabolises ∼85% of the administered dose of fluoropyrimidines. Functional *DPYD* gene variants cause reduced/abrogated DPD activity. *DPYD* variants analysis may help for defining individual patients’ risk of fluoropyrimidine-related severe toxicity.

### Methods:

The TOSCA Italian randomised trial enrolled colon cancer patients for 3 or 6 months of either FOLFOX-4 or XELOX adjuvant chemotherapy. In an ancillary pharmacogenetic study, 10 *DPYD* variants (*2A rs3918290 G>A, *13 rs55886062 T>G, rs67376798 A>T, *4 rs1801158 G>A, *5 rs1801159 A>G, *6 rs1801160 G>A, *9A rs1801265 T>C, rs2297595 A>G, rs17376848 T>C, and rs75017182 C>G), were retrospectively tested for associations with ⩾grade 3 fluoropyrimidine-related adverse events (FAEs). An association analysis and a time-to-toxicity (TTT) analysis were planned. To adjust for multiple testing, the Benjamini and Hochberg’s False Discovery Rate (FDR) procedure was used.

### Results:

FAEs occurred in 194 out of 508 assessable patients (38.2%). In the association analysis, FAEs occurred more frequently in *6 rs1801160 *A* allele carriers (FDR=0.0083). At multivariate TTT analysis, significant associations were found for *6 rs1801160 *A* allele carriers (FDR<0.0001), *2A rs3918290 *A* allele carriers (FDR<0.0001), and rs2297595 *GG* genotype carriers (FDR=0.0014). Neutropenia was the most common FAEs (28.5%). *6 rs1801160 (FDR<0.0001), and *2A rs3918290 (FDR=0.0004) variant alleles were significantly associated with time to neutropenia.

### Conclusions:

This study adds evidence on the role of *DPYD* pharmacogenetics for safety of patients undergoing fluoropyrimidine-based chemotherapy.

## Materials and methods

### The TOSCA trial

Patients included in this study represent a subgroup of the 3.759 patients with surgically resected, stage III and high-risk stage II colorectal cancer recruited in TOSCA trial between 2007 and 2011 ([Lonardi et al, 2016](#bib18)). This is an Italian intergroup, multicentre, randomised, non-inferiority phase III study in high-risk stage II and stage III colon cancer patients treated with 3 or 6 months of either FOLFOX-4 (intravenous oxaliplatin 85 mg/m^2^ on day 1 and a 2-hour infusion of L-folinic acid 100 mg/m^2^ followed by bolus 5-FU 400 mg/m^2^ and a 22-hour continuous infusion of 5-FU 600 mg/m^2^ for two consecutive days with treatment repeated every two weeks) or XELOX (intravenous oxaliplatin 130 mg/m^2^ on day 1, followed by capecitabine 1000 mg/m^2^ per os twice daily on days 1–14 with cycles were repeated every 21 days) adjuvant chemotherapy, sponsored by GISCAD (Italian Group For The Study Of Gastrointestinal Cancer) and supported by Italian Medicines Agency (AIFA; [Lonardi et al, 2016](#bib18)). Selected haematologic and non-haematologic toxicities (anaemia, leukopenia, neutropenia, thrombocytopenia, asthenia, diarrhoea, mucositis stomatitis, vomiting, nausea, hepatic toxicity, skin toxicity, and neurotoxicity) were assessed at the start of each cycle using Common Toxicity Criteria for Adverse Events (CTCAE) version 3.0.

All adverse events at any time were monitored and reported. Toxicity was managed as follows; in case of grade ⩾3 haematologic toxicity or persistent grade 2 the dose of all drugs was reduced by 25%. In case of grade ⩾3 non-haematologic toxicity the dose of the related drugs was reduced by 50%. In case of grade ⩾3 or persistent grade 2 neurotoxicity, oxaliplatin dose was reduced by 20%. Oxaliplatin was definitely stopped if grade ⩾2 neurosensory symptoms persisted between cycles.

Patients eligible for the TOSCA study were asked to give further and specific written informed consent to be enrolled in the pharmacogenetic studies. All experiments were performed in accordance with relevant guidelines and regulations and the Local Ethics Committee of each Institution approved the Study.

### DPYD assessments

The retrospective *DPYD* analysis in the ancillary pharmacogenetic study to the TOSCA clinical trial was planned in 2014 after the publication of the pharmacogenetic analysis in the QUASAR2 study ([Rosmarin et al, 2014](#bib24)). Genetic markers of toxicity for capecitabine monotherapy were selected after systemic review and then investigated in the QUASAR2 patients population of Caucasian individuals. We aimed at re-evaluating the *DPYD* panel of the QUASAR2 study (*4 rs1801158, *5 rs1801159, *6 rs1801160, *9A rs1801265, rs2297595, *2A rs3918290, *13 rs55886062, and rs67376798) in the homogeneous population of patients of the TOSCA trial who underwent adjuvant fluoropyrimidine/oxaliplatin combination chemotherapy. At the time of the planning of our *DPYD* analysis, two additional variants, rs17376848 and rs75017182, showed promising predictive role for fluoropyrimidine-related toxicity ([van Kuilenburg et al, 2010](#bib32); [Teh et al, 2013](#bib28); [Froehlich et al, 2015](#bib12)). These genetic variants were included in our panel ([Table 1](#tbl1)) considering that: (A) the polymorphisms had some degree of likelihood to alter the structure or the expression of the gene in a biologically relevant manner; (B) the ‘q’ allele frequency was expected to be >1% and (C) the polymorphisms were established and well-documented.

### Table 1. Characteristics of the DPYD studied variants with observed genotypes.

|   |   |   |   |   | Genotype (number of patients) |  |  | Allele frequency |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| rs_numbers | Nucleotide change | Effect | CPIC code | N | p2 | pq | q2 | p | q |
| rs1801158 | 1601 G>A | Ser 534 Asn | DPYD*4 | 497 | 472 | 24 | 1 | 0.974 | 0.026 |
| rs1801159 | 1627 A>G | Ile 543 Val | DPYD*5 | 496 | 318 | 156 | 22 | 0.798 | 0.202 |
| rs1801160 | 2194 G>A | Val 732 Ile | DPYD*6 | 497 | 427 | 65 | 5 | 0.924 | 0.075 |
| rs1801265 | 85 T>C | Cys 29 Arg | DPYD*9A | 497 | 311 | 169 | 17 | 0.796 | 0.204 |
| rs2297595 | 496 A>G | Met 166 Val | — | 493 | 395 | 94 | 4 | 0.897 | 0.103 |
| rs3918290 | 1905+ 1G>A | Exon skipping | DPYD*2A | 494 | 491 | 3 | 0 | 0.997 | 0.003 |
| rs17376848 | 1896 T>C | Phe 632 Phe | — | 497 | 465 | 31 | 1 | 0.967 | 0.033 |
| rs55886062 | 1679 T>G | Ile 56 Ser | DPYD*13 | 496 | 496 | 0 | 0 | 1 | 0 |
| rs67376798 | 2846 A>T | Asp 949 Val | — | 497 | 491 | 6 | 0 | 0.994 | 0.006 |
| rs75017182 | 1129–5923 C>G | Aberrant splicing | — | 504 | 494 | 10 | 0 | 0.99 | 0.01 |

Table 1 Caption: Abbreviations: CPIC code=Clinical Pharmacogenetics Implementation Consortium (https://cpicpgx.org); N=number of patients; p2=major allele homozygous genotype; pq=heterozygous genotype; q2=minor allele homozygous genotypes; rs_number=reference SNP ID number.

Genomic DNA was extracted from 2 ml whole blood by using the QiaAmp kit (Qiagen, Valencia, CA, USA). rs75017182 was analysed by Real-Time PCR assay using the Easy *DPYD* kit (Diatech Pharmacogenetics, Jesi, Italy), while the other nine *DPYD* variants were all included in the MYRIAPOD ADMET kit (Diatech Pharmacogenetics), and analysed on the MassARRAY System (Agena Bioscience). The MassARRAY protocol is characterised by three main steps: polymerase chain reaction (PCR), single-base primer extension (SBE), and separation of the products on a matrix-loaded silicon chip by matrix-assisted laser desorption ionization time of life mass spectrometry (MALDI-TOF MS). After the amplification of the region of interest, a primer extension reaction with oligos that bind adjacent to the targeted polymorphic site and all four nucleotide terminators (iPLEX) was carried out. The extension reaction generated different products for different alleles: primers extended with the terminator dNTP complementary to the targeted polymorphic site. All iPLEX products, each with its unique mass, were then identified using mass spectrometry. PCR and SBE reactions were performed in a thermal cycler (Labcycler, SensoQuest), whereas the extension products were analysed using the MALDI-TOF MassARRAY Analyzer 4 (Agena Bioscience), according to the MYRIAPOD ADMET kit’s instructions for use and using all reagents and consumables contained in the SQ TYPING 960 Kit (Diatech Pharmacogenetics). The genotype call was performed with the iGENETICS MYRIAPOD software (Diatech Pharmacogenetics).

All laboratory analyses were performed blind to the patients' treatment and clinical outcomes. Genetic data were then transferred to and independently analysed at IRCCS Istituto di Ricerche Farmacologiche ‘Mario Negri’.

### Statistics

Conforming to previously FAEs definition ([Lee et al, 2014](#bib16); [Boige et al, 2016](#bib3)) and to the planned management of toxicity in the TOSCA trial, grade ⩾3 neutropenia, diarrhoea, asthenia, nausea, vomiting, leukopenia, thrombocytopenia, mucositis, stomatitis, and skin toxicity were deemed as severe FAEs. The treatment compliance was described in terms of treatment interruption and dose intensity, defined as the dose given in mg per m^2^ per week.

According to the results of *DPYD* analysis, patients were categorised in three genotype groups: carriers of the homozygous wild type (*p*^2^); heterozygous (pq); and homozygous variant (*q*^2^). The possible association of *DPYD* variant with FAEs was analysed in the codominant model (*p*^2^, pq, and *q*^2^ genotypes considered separately) and in a dominant model with merged heterozygous (pq) and homozygous (*q*^2^) risk variant genotype carriers.

To test the effect of *DPYD* genotypes on toxicity, two analyses were planned: an association analysis and a time-to-toxicity (TTT) analysis. This choice was made because a conventional analysis with a binary outcome describing only the occurrence of severe toxicity may be inaccurate in the case of few observations (due to the rarity of some genotypes), and it may not capture potential clinically meaningful differences also in terms of time of toxicity onset ([Thanarajasingam et al, 2016](#bib29)). The association analysis compared the rate of FAEs across *DPYD* genotypes by means of a Fisher’s test in contingency tables. The TTT was defined as the time from date of randomisation in TOSCA trial to the date of severe FAEs occurrence. Subjects without severe FAEs at the time of analysis were censored at the date they were last known to be event-free while on treatment. TTT curves were estimated using the Kaplan–Meier method. Cox proportional hazard models stratified for treatment duration (6 or 3 months) were used to assess the effects of *DPYD* genotypes on TTT. Multivariate analysis stratified for treatment duration was performed to adjust the identified effect for age, gender, stage and treatment (FOLFOX-4 or XELOX). Results were provided as the hazard ratio (HR) with 95% confidence interval (95% CI).

All reported *P-*values were two-sided with *P*<0.05 value considered statistically significant. However, to adjust the analyses for multiple testing, the Benjamini and Hochberg’s False Discovery Rate (FDR) procedure was used, considering both the dominant and codominant model.

Assuming the prevalence of a high-risk allele of at least 10% and FAEs in about one-third of the study population, 188 events would allow the detection of a HR of at least 2 associated to the group with unfavorable genotypes (90% power and 5% type I error in a bilateral test). Detection of significant association for the three ‘core’ variants (*2A rs3918290, *13 rs55886062, and rs67376798) would require higher HR values given the expected frequencies of their risk alleles below 10%.

A *χ*^2^ test was used for checking the Hardy–Weinberg equilibrium. Linkage disequilibrium (LD), defined as a non-random association of alleles adjacent loci, was assessed and both *D*′ and *r*^2^ measures were provided. *D*′ can take any value from 0 (random co-inheritance of alleles) to 1 (complete LD); *r*^2^ also ranges from 0 (random co-inheritance of alleles) to 1 (perfect LD). Values of *r*^2^<0.33 suggest absence of strong LD ([Ardlie et al, 2002](#bib2)). Analyses were performed with SAS 9.4 (SAS Institute, Cary, NC, USA) and the SNPStats package ([Solè et al, 2006](#bib27)).

## Results

### Patient characteristics and toxicity

From July 2007 to October 2011, 534 patients from 26 experimental centers entered the study. This figure represents 81% of patients randomised in the same period and by the same centers in the main study. Twenty-six patients were not assessable for the following reason: 5 patients were never treated, for 2 patients the treatment data were unavailable, and for 19 patients the blood sampling was not assessable due to technical problems. Therefore, the analysis was conducted in 508 patients.

Characteristics of the 508 patients are shown in [Table 2](#tbl2). Patients’ baseline characteristics were consistent with those of the whole trial population ([Lonardi et al, 2016](#bib18)). Most patients were randomised to FOLFOX-4 because option for XELOX regimen was introduced in TOSCA trial only during the late phase of accrual of this ancillary study. Toxicity related to adjuvant chemotherapy is reported in [Table 3](#tbl3). Again, the spectrum and the frequency of toxicities did not differ from those observed in whole trial population ([Lonardi et al, 2016](#bib18)). One hundred ninety-four (38.2%) patients experienced at least one FAE. Neutropenia was the commonest among FAEs occurring in 145 patients (28.5%). As shown in [Supplementary Table S1](#sup1), analysis of dose intensity did not show differences across treatment arms.

### Table 2. Demographic and clinical characteristics of the enrolled patients.

|   | All sample (N=508) |
| --- | --- |
| Arm, n (%) |  |
| Folfox-4 (6 months) | 183 (36.0) |
| Folfox-4 (3 months) | 187 (36.8) |
| Xelox (24 weeks) | 70 (13.8) |
| Xelox (12 weeks) | 68 (13.4) |
| Age, years |  |
| Median (Q1–Q3) | 64.2 (57.4–70.7) |
| Female sex—n (%) | 217 (42.7) |
| ECOG performance status, n (%) |  |
| 0 | 488 (96.1) |
| 1 | 20 (3.9) |
| Tumour site, n (%) |  |
| Multiple site | 23 (4.5) |
| Single site: | 485 (95.5) |
| Ascending colon | 137 (28.3) |
| Sigmoid-rectum colon | 201 (41.4) |
| Descending colon | 66 (13.6) |
| Trasverse colon | 32 (6.6) |
| Splenic flexure | 27 (5.6) |
| Hepatic flexure | 22 (4.5) |
| Histology, n (%) |  |
| Adenocarcinoma | 437 (86.0) |
| Mucoid adenocarcinoma | 65 (12.8) |
| Other | 6 (1.2) |
| Stage, n (%) |  |
| II | 184 (36.2) |
| III | 324 (63.8) |
| Grade, n (%) |  |
| Gx | 4 (0.8) |
| G1-2 | 340 (67.6) |
| G3-4 | 159 (31.6) |
| Missing | 5 |
| T stage, n (%) |  |
| pTx | 1 (0.2) |
| pT1 | 12 (2.4) |
| pT2 | 41 (6.1) |
| pT3 | 380 (74.8) |
| pT4 | 84 (16.5) |
| N stage, n (%) |  |
| pN0 | 184 (36.2) |
| pN1 | 233 (45.9) |
| pN2 | 91 (17.9) |

Table 2 Caption: Abbreviations: n=number; Q1=first quartile; Q3=third quartile.

### Table 3. Grade ⩾3 adverse events occurred in the study population.

|   | All sample N=508 |
| --- | --- |
| All grade >3 adverse events |   |
| Neutropenia | 145 (28.5) |
| Grade ⩾2 neurological toxicity | 131 (25.8) |
| Diarrhoea | 33 (6.5) |
| Asthenia | 16 (3.1) |
| Nausea | 14 (2.8) |
| Vomiting | 11 (2.2) |
| Leukopenia | 11 (2.2) |
| Thrombocytopenia | 6 (1.2) |
| Hepatic toxicity | 6 (1.2) |
| Mucositis | 4 (0.8) |
| Stomatitis | 2 (0.4) |
| Anaemia | 2 (0.4) |
| Skin toxicity | 1 (0.2) |
| First grade ⩾3 FAEs occurred | 194 (38.2) |
| Neutropenia | 130 (67.0) |
| Diarrhoea | 25 (12.9) |
| Leukopenia | 10 (5.2) |
| Asthenia | 8 (4.1) |
| Nausea | 8 (4.1) |
| Thrombocytopenia | 4 (2.1) |
| Mucositis | 4 (2.1) |
| Vomiting | 3 (1.6) |
| Stomatitis | 1 (0.5) |
| Skin toxicity | 1 (0.5) |

Table 3 Caption: Abbreviations: FAEs: fluoropyrimidine-related adverse events.

### Genetic assessments

[Table 1](#tbl1) lists the studied genetic variants and the distribution of genotypes of patients successfully assessed for each polymorphism. Consistent with previous observations, genotype frequency did not differ from those observed in Caucasian population. The *13 rs55886062 G allele was not found in the studied population and therefore, this variant was excluded from subsequent analyses. Allele frequencies of the remaining polymorphisms were consistent with the Hardy–Weinberg equilibrium (*P*>0.05). Results of LD analyses are shown in [Supplementary Table S2](#sup1).

### DPYD variants and FAEs

The prevalence of *DPYD* high-risk alleles was heterogeneous, ranging from 0% of the *13 rs55886062 G allele to 37.5% of the *9A rs1801265 C allele. Therefore, 194 events would allow detection of an HR of at least 8.3 and an HR of at least 1.5 for a prevalence of a high-risk allele equal to 1% and to 35%, respectively (power of 90% and a I type error of 5%, for a bilateral test). A statistically significant association was found between *6 rs1801160 genotypes and FAEs (FDR=0.0083 in both the dominant and codominant models). No additional significant associations were detected (data not shown).

Results about the effect of *DPYD* variants on TTT are shown in [Table 4](#tbl4). At univariate analysis, *6 rs1801160 (codominant model: FDR=0.0022), rs2297595 (codominant model: FDR=0.0413), *2A rs3918290 (codominant model: FDR=0.0001) correlated with TTT. Specifically, *6 rs1801160 *GA* genotype carriers and *A* allele carriers were at risk for shorter TTT (HR 1.99, 95% CI 1.38–2.86, FDR=0.0002 and HR 2.01, 95% CI 1.42–2.86, FDR=0.0006, respectively). Median TTT for *6 rs1801160 *GG, GA* and *AA* genotype carriers were 7.0, 3.0 and 2.1 months, respectively. Also, the rs2297595 *GG* genotype (HR 4.28, 95% CI 1.35–13.55, FDR=0.0136) and the *2A rs3918290 *GA* genotype (HR 15.34, 95% CI 4.72–49.89, FDR=0.0001) showed a shorter TTT. Median TTT for rs2297595 *AA, AG* and *GG* genotype carriers were 7.0, 6.6 and 1.2 months, respectively. Median TTT for *2A rs3918290 *GG* and *GA* genotype carriers were 7.0 and 0.9 months, respectively. [Figure 1](#fig1) depicts Kaplan–Meier curves of ‘q’ allele carriers *vs* ‘p^2^’ genotype carriers of rs2297595 and *6 rs1801160. At multivariate analyses the associations with *DPYD* variants identified in the univariate analyses were confirmed.

### Table 4. Effect of DPYD variants on TTT for FAEs.

|   | Univariate analysisa |  | Multivariate analysisa,b |  |
| --- | --- | --- | --- | --- |
|   | HR (95% CI) | FDR | HR (95% CI) | FDR |
| *4 rs1801158 |  |  |  |  |
| Overall codominant: |   | 0.9831 |   |   |
| Dominant: |   |   |   |   |
| G/A or A/A vs G/G | 0.87 (0.43–1.78) | 0.8874 |   |   |
| *5 rs1801159 |  |  |  |  |
| Overall codominant: |   | 0.2910 |   |   |
| Dominant: |   |   |   |   |
| G/A or G/G vs A/A | 0.85 (0.63–1.15) | 0.4400 |   |   |
| *6 rs1801160 |  |  |  |  |
| Overall codominant: |   | 0.0022 |   | 0.0002 |
| G/G | 1.00 |   | 1.00 |   |
| G/A | 1.99 (1.38–2.86) | 0.0002 | 2.06 (1.43–2.96) | 0.0001 |
| A/A | 2.40 (0.76–7.60) | 0.1375 | 2.53 (0.79–8.09) | 0.1189 |
| Dominant: |   |   |   |   |
| G/A or A/A vs G/G | 2.01 (1.42–2.86) | 0.0006 | 2.09 (1.47–2.97) | <0.0001 |
| *9A rs1801265 |  |  |  |  |
| Overall codominant: |   | 0.2181 |   |   |
| Dominant: |   |   |   |   |
| C/T or C/C vs T/T | 1.10 (0.82–1.47) | 0.7640 |   |   |
| rs2297595 |  |  |  |  |
| Overall codominant: |   | 0.0413 |   | 0.0032 |
| A/A | 1.00 |   | 1.00 |   |
| G/A | 1.40 (0.99–1.97) | 0.0569 | 1.26 (0.89–1.78) | 0.1950 |
| G/G | 4.28 (1.35–13.55) | 0.0136 | 6.77 (2.10–21.84) | 0.0014 |
| Dominant: |   |   |   |   |
| G/A or G/G vs A/A | 1.46 (1.05–2.05) | 0.0722 | 1.33 (0.95–1.87) | 0.0942 |
| *2A rs3918290 |  |  |  |  |
| Overall codominant: |   |   |   |   |
| G/A vs G/G | 15.34 (4.72–49.89) | 0.0001 | 14.98 (4.39–51.09) | <0.0001 |
| rs17376848 |  |  |  |  |
| Overall codominant: |   | 0.9652 |   |   |
| Dominant: |   |   |   |   |
| C/T or C/C vs T/T | 1.15 (0.65–2.02) | 0.8386 |   |   |
| rs67376798 |  |  |  |  |
| Overall codominant: |   |   |   |   |
| T/A vs A/A | 3.02 (1.12–8.16) | 0.0722 |   |   |
| rs75017182 |  |  |  |  |
| Overall codominant: |   |   |   |   |
| C/G vs Cs/C | 0.99 (0.37–2.67) | 0.9831 |   |   |

Table 4 Caption: Abbreviations: 95% CI=confidence interval at 95% DPYD=dihydropyrimidine dehydrogenase; FAEs=fluoropyrimidine-related adverse events; FDR=False Discovery Rate; HR=hazard ratio; TTT=time-to-toxicity. aCox proportional hazard models. bAdjusted for age, gender, stage, and treatment.

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3b22/5709672/53204cd85f99/bjc2017289f1.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5709672_bjc2017289f1.jpg)

Kaplan-Meier curves. (A) TTT curves of the *6 rs1801160 minor A allele carriers (merged heterozygous plus homozygous minor allele carriers) and homozygous GG genotype carriers. (B) TTT curves of the rs2297595 minor G allele carriers (merged heterozygous plus homozygous minor allele carriers) and homozygous AA genotype carrier.

Neutropenia was the commonest FAEs, occurring in 145 patients (28.5%). The second one was diarrhoea, which occurred in 33 patients (6.5%). Therefore, univariate and multivariate Cox analyses to address the effect of *DPYD* variants on TTT for specific FAEs were performed only for neutropenia ([Table 5](#tbl5)). At univariate analysis, associations with time to neutropenia were found for *6 rs1801160 and *2A rs3918290. In detail, *6 rs1801160 *GA* genotype carriers in the codominant model and *A* allele carriers in the dominant model were at risk for shorter time to neutropenia (HR 2.19, 95% CI 1.46–3.28, FDR=0.0002 and HR 2.18, 95% CI 1.47–3.24, FDR=0.0024, respectively). The codominant model analysis for *2A rs3918290 showed significant association with short time to neutropenia for *GA* variant genotype carriers (HR 10.74, 95% CI 2.59–44.61, FDR=0.0054). The impact of all this *DPYD* variants was confirmed at multivariate analysis.

### Table 5. Effect of DPYD variants on TTT for neutropenia.

|   | Univariate analysisa |  | Multivariate analysisa,b |  |
| --- | --- | --- | --- | --- |
|   | HR (95% CI) | FDR | HR (95% CI) | FDR |
| *4 rs1801158 |  |  |  |  |
| Overall codominant: |   | 0.9137 |   |   |
| Dominant: |   |   |   |   |
| G/A or A/A vs G/G | 0.74 (0.30–1.80) | 0.5937 |   |   |
| *5 rs1801159 |  |  |  |  |
| Overall codominant: |   | 0.3509 |   |   |
| Dominant: |   |   |   |   |
| G/A or G/G vs A/A | 0.76 (0.53–1.08) | 0.2837 |   |   |
| *6 rs1801160 |  |  |  |  |
| Overall codominant: |   | 0.0054 |   | 0.0003 |
| G/G | 1.00 |   | 1.00 |   |
| G/A | 2.19 (1.46–3.28) | 0.0002 | 2.30 (1.53–3.46) | <0.0001 |
| A/A | 2.07 (0.51–8.45) | 0.3107 | 2.00 (0.49–8.26) | 0.3364 |
| Dominant: |   |   |   |   |
| G/A or A/A vs G/G | 2.18 (1.47–3.24) | 0.0024 | 2.28 (1.53–3.40) | <0.0001 |
| *9A rs1801265 |  |  |  |  |
| Overall codominant: |   | 0.5133 |   |   |
| Dominant: |   |   |   |   |
| C/T or C/C vs T/T | 1.00 (0.71–1.41) | 0.9847 |   |   |
| rs2297595 |  |  |  |  |
| Overall codominant: |   | 0.1661 |   |   |
| Dominant: |   |   |   |   |
| G/A or G/G vs A/A | 1.55 (1.06–2.26) | 0.0958 |   |   |
| *2A rs3918290 |  |  |  |  |
| Overall codominant: |   |   |   |   |
| G/A vs G/G | 10.74 (2.59–44.61) | 0.0054 | 14.72 (3.35–64.72) | 0.0004 |
| rs17376848 |  |  |  |  |
| Overall codominant: |   | 0.6299 |   |   |
| Dominant: |   |   |   |   |
| C/T or C/C vs T/T | 1.34 (0.73–2.49) | 0.5133 |   |   |
| rs67376798 |  |  |  |  |
| Overall codominant: |   |   |   |   |
| T/A vs A/A |   | 0.5133 |   |   |
| rs75017182 |  |  |  |  |
| Overall codominant: |   |   |   |   |
| C/G vs C/C |   | 0.3509 |   |   |

Table 5 Caption: Abbreviations: 95% CI=confidence interval at 95; DPYD=dihydropyrimidine dehydrogenase; FDR=False Discovery Rate; HR=hazard ratio; TTT=time-to-toxicity. aCox proportional hazard models. bAdjusted for age, gender, stage, and treatment.

## Discussion

As shown in [Table 6](#tbl6), this study is added to previous pharmacogenetic analyses for *DPYD*, which were incorporated in randomised clinical trials of fluoropyrimidine-based chemotherapy in colorectal cancer ([Deenen et al, 2011](#bib7); [Lee et al, 2014](#bib16); [Rosmarin et al, 2014](#bib24); [Del Re et al, 2015](#bib9); [Boige et al, 2016](#bib3); [Lee et al, 2016](#bib17)). These studies offer a unique opportunity for performing pharmacogenetics in an optimal setting, where the genotyped patient population is well characterised and uniformly assessed for clinical/pathologic characteristics and the monitoring of toxicity. Unfortunately, these studies cannot be uniformly evaluated because of the substantial differences in disease stage (adjuvant *vs* metastatic), chemotherapy protocols (often with biologics), panels of *DPYD* variants, and methodology for assessing putative pharmacogenetics associations. To this regard, we introduced the TTT analysis in addition to a standard genotypes/FAEs distribution analysis, which was commonly adopted in studies listed in [Table 6](#tbl6). The TTT analysis for detecting pharmacogenetic associations with FAEs may help to disclose potential clinical impact of *DPYD* variants, which could be lost in a common binary analysis of genotype frequencies in contingency tables. The TTT analysis adds the dimension of time, and therefore, it allows for detection of ‘more and early’ toxicity events ([Thanarajasingam et al, 2016](#bib29)). In fact, if severe toxicity occurs after multiple cycles of chemotherapy, it may also represent a cumulative effect and the stress of the system after several doses of the drugs. On the contrary, if severe toxicity events occur early, they are more likely related to innate defects, often linked with catabolic pathways ([Sahota et al, 2016](#bib26)). Notably, some clinical analyses on *DPYD* variants and fluoropyrimidine-related toxicity were based on FAEs occurring within the first 3 cycles of therapy ([Gross et al, 2008](#bib14); [Deenen et al, 2011](#bib7), [Froehlich et al, 2015](#bib12)). The TTT approach avoids the need of defining such a cut-point and it may better characterise a gene-linked toxicity profile. Also, it should be considered that some functional *DPYD* variants may not induce a dramatic loss of enzyme function like the *2A rs3918290, and therefore, in these cases, TTT analysis may be more sensitive for detecting the risk of toxicity determined by *DPYD* variants with moderate functional effects.

### Table 6. Summary of randomised controlled clinical trials with dedicated DPYD pharmacogenetic analyses.

| Trial (reference) | Setting | Treatment arms (N) | Number of DPYD studied variants | Toxicity outcomes (%) | Significant associations |
| --- | --- | --- | --- | --- | --- |
| QUASAR2 (Rosmarin et al, 2014) | Adjuvant | Cap (436) Cap+Bev (491) | 12 | Grade ⩾3 FAEs (32.4%) | rs67376798 |
| CAIRO-2 (Deenen et al, 2011) | Metastatic | Cap/Oxa/Bev (281) Cap/Oxa/Bev/Cetux (287) | 29 | Grade ⩾3 diarrhoea (24.4%) Any grade ⩾3 toxicity (85.3%) Hand-foot grade ⩾2 (43.1%) | rs3918290 (DPYD*2A), rs1801160 (DPYD*6), rs56038477 noa no |
| NCCTG (Lee et al, 2014, 2016) | Adjuvant | FOLFOX (2384) FOLFIRI (210) CT plus Cetux (1191) CT without Cetux (1403) | 25+1b | Grade ⩾3 FAEs (33%) | rs3918290 (DPYD*2A), rs67376798 |
| PETACC-8 (Boige et al, 2016) | Adjuvant | FOLFOX (780) FOLFOX+Cetux (765) | 25 | Grade ⩾3 FAEs (49.5%) | rs1801160 (DPYD*6), rs67376798 |
| TRIBE (Del Re et al, 2015) | Metastatic | FOLFOXIRI+Bev (220) FOLFIRI+Bev (220) | 2 | Grade ⩾3 FAEs | rs3918290 (DPYD*2A) plus rs67376798c |
| TOSCA—ancillary | Adjuvant | FOLFOX (370) Cap/Oxa (138) | 10 | Grade ⩾3 FAEs (32.4%) | rs3918290 (DPYD*2A), rs1801160 (DPYD*6), rs2297595 |

Table 6 Caption: Abbreviations: Bev=bevacizumab; Cap=capecitabine; Cetux=cetuximab; DPYD=dihydropyrimidine dehydrogenase; FAEs=fluoropyrimidine-related adverse events; FOLFOX=bolus/infusional 5-fluorouracil plus oxaliplatin; FOLFIRI=bolus/infusional 5-fluorouracil plus Irinotecan; FOLFOXIRI=bolus/infusional 5-fluorouracil plus oxaliplatin and irinotecan; N=number of patients; Oxa=oxaliplatin. aIn the CAIRO-2 analysis, *2A rs3918290 G>A did not meet criteria for statistical significant thresholds in the overall analysis of toxicity, but all carriers of the *2A rs3918290 A allele developed grade 3–4 toxicity with 1 death possibly related to the capecitabine treatment. bA second pharmacogenetic assessment in the NCCTG trial added to the original 25 DPYD genotypes the novel rs75017182 C>G genetic variant. cA combined analysis of the two genotypes for association with FAE was performed.

In the present study population, potential baseline confounders for early toxicity could be excluded since the administration of adjuvant combination chemotherapy was per-protocol proposed to high-risk colon cancer patients without evidence of metastatic disease, no major comorbidity, long life expectancy, and good performance status. Furthermore, only 2 patients interrupted treatment due to disease progression and in these patients no fluoropyrimidine-related toxicity was observed.

In our population of patients, the observed frequencies of the rare deleterious *DPYD* variant alleles *2A rs3918290, *13 rs55886062, and rs67376798 were 0.6%, 0%, and 1.2%, respectively. Only *2A rs3918290 showed significant association with FAEs in the TTT analysis achieving an HR equal to 14.98, and a significant impact on time to neutropenia ([Tables 4](#tbl4) and [5](#tbl5), respectively). However, even if they all had shown significant HRs for FAEs, they cannot explain the overall estimated contribution of functional *DPYD* variants in causing severe fluoropyrimidine toxicity. DPD deficiency has been described in ∼40–60% of patients with ⩾3 grade fluoropyrimidine-induced toxicity ([Meulendijks et al, 2015](#bib21)). However, DPD deficiency cannot always be traced back to a currently known *DPYD* variant associated with reduced enzyme activity ([Meulendijks et al, 2015](#bib21)). Therefore, other detrimental variants should be identified to improve sensitivity of *DPYD* genotyping ([Gentile et al, 2016](#bib13)). Indeed, among the seven additional *DPYD* studied variants, two (*6 rs1801160 and rs2297595) showed associations with FAEs.

The *DPYD* *6 rs1801160 was analysed within the *DPYD* panel of three studies listed in [Table 6](#tbl6) ([Deenen et al, 2011](#bib7); [Rosmarin et al, 2014](#bib24); [Boige et al, 2016](#bib3)). Notably, in the large PETACC-8 study, *6 rs1801160 showed statistically significant association with grade 3 or greater FAEs and neutropenia in particular ([Boige et al, 2016](#bib3)). In the QUASAR2 ([Rosmarin et al, 2014](#bib24)) and the CAIRO-2 ([Deenen et al, 2011](#bib7)) studies, *6 rs1801160 did not show predictive role for FAEs. However, it should be considered that the QUASAR2 analysis ([Rosmarin et al, 2014](#bib24)) was performed in patients treated with capecitabine mono-chemotherapy only. As far as the CAIRO-2 is concerned, the high probability of developing FAEs (85%) was considered as a major reason for not detecting significant associations between FAEs and all tested *DPYD* variants in this study ([Deenen et al, 2011](#bib7)). If we look at risk associations between **6* rs1801160 and FAEs in the present study and the PETACC-8 study ([Boige et al, 2016](#bib3)), it should be noted a significant but moderate effect size attributed to the **6* rs1801160 A risk allele. Results from the pharmacogenetics analysis by Kleibl *et al* suggested an impact of the *6 rs1801160 A allele in determining fluoropyrimidine toxicity especially in the context of specific *DPYD* haplotypes ([Kleibl et al, 2009](#bib15)). Notably, in the whole *DPYD* panel, the *6 rs1801160 locus did not show strong LD, thus excluding that the association of the variant with toxicity may be only the results of LD with a neighboring etiologic variant. These aspects would suggest direct but mild impact on phenotype of the *6 rs1801160, which cumulates with other variants and/or emerges in specific chemotherapy regimen because of toxicity synergy between fluoropyrimidine and other drugs (i.e., oxaliplatin; [Offer and Diasio, 2016](#bib23)). In the sub-type analysis of FAEs, the *6 rs1801160 variant showed detrimental effect on time to neutropenia. We observed grade ⩾3 neutropenia in the 28.5% of patients and this figure is slightly lower than the toxicity rates previously reported in patients treated with XELOX and FOLFOX regimens (up to 40% [Eng (2009)](#bib11)). These figures would exceed the expected frequency of ⩾3 grade neutropenia if the sum of neutropenia rates in single-agent studies of oxaliplatin, capecitabine and bolus/infusional 5-FU (<10% of patients) would be applied for prediction. The array of interactions and synergisms between fluoropyrimidines and oxaliplatin in humans may explain this discrepancy. In this context, a *DPYD* variant, which depresses, but does not abrogate the enzyme function may significantly increase the risk of severe toxicity (neutropenia) when the fluorpyrimidine is combined with other drugs.

*DPYD* pharmacogenetics in the PETACC-8 study ([Boige et al, 2016](#bib3)) included the rs2297595, but without detecting significant associations with FAEs. In the present study, the homozygous rs2297595 *GG* genotype was associated with a significant relatively large effect (HR 6.77) in the TTT analysis, whereas the heterozygous genotype did not. This behavior would suggest an ‘allele-dosage’ effect and a clinically meaningful DPD deficient phenotype in carriers of the ‘q^2^’ genotype. This hypothesis parallels previous findings in a retrospective pharmacogenetic study by Gross *et al* ([2008](#bib14)), and it is compatible with the putative functional effect of the rs2297595 variant. The methionine-valine exchange, as consequence of the non-synonymous sequence variation occurs in a highly conserved site during evolution, which may be critical to enzyme structure and function ([Mattison et al, 2002](#bib19)). Moreover, LD analyses showed that the *6 rs1801160 and rs2297595 loci are not co-inherited, and therefore they may act independently.

The analysis of the median TTT values contributes to the understanding of the clinical impact of the *6 rs1801160, *2A rs3918290, and rs2297595 variants. Median TTT was 7 months among common homozygous genotypes carriers, whereas it was significantly shortened (between 0.9 and 2.1 months) in carriers of the homozygous variant *6 rs1801160 and rs2297595 and the *2A rs3918290 heterozygous genotypes. Notably, shortened TTT was detectable in *6 rs1801160, but not rs2297595 heterozygous genotype carriers, thus corroborating the hypothesis of a different effect of the two variants in depressing/altering the DPD function. The early onset of toxicity corroborates the hypothesis of an underlying enzymatic defect and the opportunity of verifying *DPYD* variants/DPD status in patients with early severe FAEs after fluoropyrimidine exposure.

As far as ethnicity is concerned, the frequency of the *6 rs1801160 A risk allele seems comparable in Caucasian, Middle-Eastern, and African-American, whereas it seems less frequent in Asian populations ([Caudle et al, 2013](#bib5)). The clinical impact of the rs2297595 variant may be more relevant to populations of African ancestry, where its frequency seems to double in comparison with Caucasian populations ([Aminkeng et al, 2014](#bib1)).

It is still matter of debate whether *DPYD* genotyping should be incorporated in the routine pre-treatment screening of patients undergoing fluoropyrimidine-based chemotherapy. To this regard, the recent guidelines of the European Society for Medical Oncology (ESMO) consider the testing as an option, which is indicated in the case of patients who experience severe toxicity and before the fluoropyrimidine is re-introduced ([van Cutsem et al, 2016](#bib30)). We disagree with statement, especially when possible cautions could be adopted in treatment settings with narrow therapeutic window. Since the *DPYD* assessment was not incorporated in our original study plan, we could not perform a reliable cost-effectiveness analysis. However, available analyses suggest that *DPYD*-genotype guided dosing according to *2A rs3918290 ([Deenen et al, 2016](#bib8)), or *2A rs3918290, *13 rs55886062, and rs67376798 ([Cortejoso et al, 2016](#bib6)) may significantly improve safety of fluoropyrimidine therapy and being cost saving.

It should be considered that additional tests have been developed for assessing the activity of the DPD enzyme (DPD activity in peripheral blood monuclear cells, Uracil breath test, endogenous plasma/urine Uracil/Dihydrouracil, sampling PK model after 5-fluoruracil test dose; [van Staveren et al, 2013](#bib33), [2016](#bib34); [Del Re et al, 2017](#bib10)). These phenotyping tests seem to possess better predictivity then genotyping for fluoropyrimidine toxicity ([van Staveren et al, 2013](#bib33), [2016](#bib34)). In a recent analysis in 550 patients, Meulendijks *et al* found that high pre-treatment uracil concentrations were strongly associated with severe fluoropyrimidine-related toxicity, whereas *DPYD* genotypes did not ([Meulendijks et al, 2017](#bib22)). However, in this study, *DPYD* genotyping was limited to rs67376798, *13 rs55886062, rs75017182, and *4 rs1801158. In general, as with the genotyping strategy, the phenotyping tests suffer from suboptimal sensitivity and specificity. Notably, a test for detecting DPD deficiency and preventing fluoropyrimidine toxicity requires high sensitivity. On the other side, low specificity may cause unnecessary dose reduction and suboptimal exposure to effective chemotherapy. To this end, as pointed out by Boisdron-Celle *et al* ([2007](#bib4)), *DPYD* genotyping and DPD phenotyping tests could be integrated in a two-step strategy for screening selected patients.

In conclusion, this study remarks the role of *DPYD* *2A rs3918290 for fluoropyrimidine-related toxicity. It also indicates that *6 rs1801160 and rs2297595 produce additional *DPYD* genotypes, which may be predictive of toxicity in the same setting. TTT analysis in pharmacogenetic studies may help to characterise the clinical impact of risk alleles causing reduced DPD function.

## Acknowledgments

This work was partially supported by the TERPAGE Project ‘POR MARCHE FESR 2007–2013’. AR, Francesco G, and MM conceived and performed the study design, performed the manuscript preparation and data interpretation. Fabio G performed coordination study. Francesca G, Fabio G, and ER performed statistical analysis, data interpretation, and manuscript preparation. SL, MR, BM, VZ, NP, CM, RL, MTI, EV, PS, SB, V R, IB, LF, MN, EB, AB, DT, SL, CV, FB,AS, and LF collected samples and patients’ data, and commented the manuscript. RL, LF, and AS participated in the study design and data interpretation, and helped to draft the manuscript. All authors reviewed the manuscript.

## Footnotes

## Supplementary Material

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Aminkeng F, Ross CJ, Rassekh SR, Brunham LR, Sistonen J, Dube MP, Ibrahim M, Nyambo TB, Omar SA, Froment A, Bodo JM, Tishkoff S, Carleton BC, Hayden MR (2014) Canadian Pharmacogenomics Network for Drug Safety Consortium. Higher frequency of genetic variants conferring increased risk for ADRs for commonly used drugs treating cancer, AIDS and tuberculosis in persons of African descent. Pharmacogenomics J 14(2): 160–170.  [DOI](https://doi.org/10.1038/tpj.2013.13) | [PMC free article](/articles/PMC4684079/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23588107/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=Canadian%20Pharmacogenomics%20Network%20for%20Drug%20Safety%20Consortium.%20Higher%20frequency%20of%20genetic%20variants%20conferring%20increased%20risk%20for%20ADRs%20for%20commonly%20used%20drugs%20treating%20cancer,%20AIDS%20and%20tuberculosis%20in%20persons%20of%20African%20descent&volume=14&publication_year=2014&pages=160&pmid=23588107&doi=10.1038/tpj.2013.13&)

2. Ardlie KG, Kruglyak L, Seielstad M (2002) Patterns of linkage disequilibrium in the human genome. Nat Rev Genet 3: 299–309.  [DOI](https://doi.org/10.1038/nrg777) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11967554/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Genet&title=Patterns%20of%20linkage%20disequilibrium%20in%20the%20human%20genome&volume=3&publication_year=2002&pages=299&pmid=11967554&doi=10.1038/nrg777&)

3. Boige V, Vincent M, Alexandre P, Tejpar S, Landolfi S, Le Malicot K, Greil R, Cuyle PJ, Yilmaz M, Faroux R, Matzdorff A, Salazar R, Lepage C, Taieb J, Laurent-Puig P (2016) DPYD genotyping to predict adverse events following treatment with flourouracil-based adjuvant chemotherapy in patients with stage III colon cancer: a secondary analysis of the PETACC-8 randomized clinical trial. JAMA Oncol 2: 655–662.  [DOI](https://doi.org/10.1001/jamaoncol.2015.5392) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26794347/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA%20Oncol&title=DPYD%20genotyping%20to%20predict%20adverse%20events%20following%20treatment%20with%20flourouracil-based%20adjuvant%20chemotherapy%20in%20patients%20with%20stage%20III%20colon%20cancer:%20a%20secondary%20analysis%20of%20the%20PETACC-8%20randomized%20clinical%20trial&volume=2&publication_year=2016&pages=655&pmid=26794347&doi=10.1001/jamaoncol.2015.5392&)

4. Boisdron-Celle M, Remaud G, Traore S, Poirier AL, Gamelin L, Morel A, Gamelin E (2007) 5-Fluorouracil-related severe toxicity: a comparison of different methods for the pretherapeutic detection of dihydropyrimidine dehydrogenase deficiency. Cancer Lett 249(2): 271–282.  [DOI](https://doi.org/10.1016/j.canlet.2006.09.006) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17064846/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Lett&title=5-Fluorouracil-related%20severe%20toxicity:%20a%20comparison%20of%20different%20methods%20for%20the%20pretherapeutic%20detection%20of%20dihydropyrimidine%20dehydrogenase%20deficiency&volume=249&publication_year=2007&pages=271&pmid=17064846&doi=10.1016/j.canlet.2006.09.006&)

5. Caudle KE, Thorn CF, Klein TE, Swen JJ, McLeod HL, Diasio RB, Schwab M (2013) Clinical pharmacogenetics implementation consortium guidelines for dihydropyrimidine dehydrogenase genotype and fluoropyrimidine dosing. Clin Pharmacol Ther 94: 640–645.  [DOI](https://doi.org/10.1038/clpt.2013.172) | [PMC free article](/articles/PMC3831181/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23988873/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Clinical%20pharmacogenetics%20implementation%20consortium%20guidelines%20for%20dihydropyrimidine%20dehydrogenase%20genotype%20and%20fluoropyrimidine%20dosing&volume=94&publication_year=2013&pages=640&pmid=23988873&doi=10.1038/clpt.2013.172&)

6. Cortejoso L, García-González X, García MI, García-Alfonso P, Sanjurjo M, López-Fernández LA (2016) Cost-effectiveness of screening for DPYD polymorphisms to prevent neutropenia in cancer patients treated with fluoropyrimidines. Pharmacogenomics 17(9): 979–984.  [DOI](https://doi.org/10.2217/pgs-2016-0006) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27248859/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Cost-effectiveness%20of%20screening%20for%20DPYD%20polymorphisms%20to%20prevent%20neutropenia%20in%20cancer%20patients%20treated%20with%20fluoropyrimidines&volume=17&publication_year=2016&pages=979&pmid=27248859&doi=10.2217/pgs-2016-0006&)

7. Deenen MJ, Tol J, Burylo AM, Doodeman VD, de Boer A, Vincent A, Guchelaar HJ, Smits PH, Beijnen JH, Punt CJ, Schellens JH, Cats A (2011) Relationship between single nucleotide polymorphisms and haplotypes in DPYD and toxicity and efficacy of capecitabine in advanced colorectal cancer. Clin Cancer Res 17: 3455–3468.  [DOI](https://doi.org/10.1158/1078-0432.CCR-10-2209) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21498394/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&title=Relationship%20between%20single%20nucleotide%20polymorphisms%20and%20haplotypes%20in%20DPYD%20and%20toxicity%20and%20efficacy%20of%20capecitabine%20in%20advanced%20colorectal%20cancer&volume=17&publication_year=2011&pages=3455&pmid=21498394&doi=10.1158/1078-0432.CCR-10-2209&)

8. Deenen MJ, Meulendijks D, Cats A, Sechterberger MK, Severens JL, Boot H, Smits PH, Rosing H, Mandigers CM, Soesan M, Beijnen JH, Schellens JH (2016) Upfront genotyping of DPYD*2A to individualize fluoropyrimidine therapy: a safety and cost analysis. J Clin Oncol 34: 227–234.  [DOI](https://doi.org/10.1200/JCO.2015.63.1325) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26573078/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Upfront%20genotyping%20of%20DPYD*2A%20to%20individualize%20fluoropyrimidine%20therapy:%20a%20safety%20and%20cost%20analysis&volume=34&publication_year=2016&pages=227&pmid=26573078&doi=10.1200/JCO.2015.63.1325&)

9. Del Re M, Cremolini C, Loupakis F, Marmorino F, Citi V, Palombi M, Bergamo F, Schirripa M, Rossini D, Cortesi E, Tomasello G, Spadi R, Buonadonna A, Amoroso D, Vitello S, Di Donato S, Granetto C, D'Amico M, Danesi R, Falcone A (2015) DPYD c.1905+1G>A and c.2846A>T and UGT1A1*28 allelic variants as predictors of toxicity: Pharmacogenetic translational analysis from the phase III TRIBE study in metastatic colorectal cancer. J Clin Oncol 33, (suppl): abstr 3532. http://meetinglibrary.asco.org/record/112386/abstract.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=DPYD%20c.1905+1G>A%20and%20c.2846A>T%20and%20UGT1A1*28%20allelic%20variants%20as%20predictors%20of%20toxicity:%20Pharmacogenetic%20translational%20analysis%20from%20the%20phase%20III%20TRIBE%20study%20in%20metastatic%20colorectal%20cancer&volume=33&publication_year=2015&)

10. Del Re M, Restante G, Di Paolo A, Crucitta S, Rofi E, Danesi R (2017) Pharmacogenetics and metabolism from science to implementation in clinical practice: the example of dihydropyrimidine dehydrogenase. Curr Pharm Des 23(14): 2028–2034.  [DOI](https://doi.org/10.2174/1381612823666170125155530) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28128059/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Pharm%20Des&title=Pharmacogenetics%20and%20metabolism%20from%20science%20to%20implementation%20in%20clinical%20practice:%20the%20example%20of%20dihydropyrimidine%20dehydrogenase&volume=23&publication_year=2017&pages=2028&pmid=28128059&doi=10.2174/1381612823666170125155530&)

11. Eng C (2009) Toxic effects and their management: daily clinical challenges in the treatment of colorectal cancer. Nat Rev Clin Oncol 6(4): 207–218.  [DOI](https://doi.org/10.1038/nrclinonc.2009.16) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19333227/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Clin%20Oncol&title=Toxic%20effects%20and%20their%20management:%20daily%20clinical%20challenges%20in%20the%20treatment%20of%20colorectal%20cancer&volume=6&publication_year=2009&pages=207&pmid=19333227&doi=10.1038/nrclinonc.2009.16&)

12. Froehlich TK, Amstutz U, Aebi S, Joerger M, Largiadèr CR (2015) Clinical importance of risk variants in the dihydropyrimidine dehydrogenase gene for the prediction of early-onset fluoropyrimidine toxicity. Int J Cancer 136(3): 730–739.  [DOI](https://doi.org/10.1002/ijc.29025) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24923815/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Cancer&title=Clinical%20importance%20of%20risk%20variants%20in%20the%20dihydropyrimidine%20dehydrogenase%20gene%20for%20the%20prediction%20of%20early-onset%20fluoropyrimidine%20toxicity&volume=136&publication_year=2015&pages=730&pmid=24923815&doi=10.1002/ijc.29025&)

13. Gentile G, Botticelli A, Lionetto L, Mazzuca F, Simmaco M, Marchetti P, Borro M (2016) Genotype-phenotype correlations in 5-fluorouracil metabolism: a candidate DPYD haplotype to improve toxicity prediction. Pharmacogenomics J 16: 320–325.  [DOI](https://doi.org/10.1038/tpj.2015.56) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26216193/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=Genotype-phenotype%20correlations%20in%205-fluorouracil%20metabolism:%20a%20candidate%20DPYD%20haplotype%20to%20improve%20toxicity%20prediction&volume=16&publication_year=2016&pages=320&pmid=26216193&doi=10.1038/tpj.2015.56&)

14. Gross E, Busse B, Riemenschneider M, Neubauer S, Seck K, Klein HG, Kiechle M, Lordick F, Meindl A (2008) Strong association of a common dihydropyrimidine dehydrogenase gene polymorphism with fluoropyrimidine-related toxicity in cancer patients. PLoS One 3(12): e4003.  [DOI](https://doi.org/10.1371/journal.pone.0004003) | [PMC free article](/articles/PMC2602733/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19104657/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=Strong%20association%20of%20a%20common%20dihydropyrimidine%20dehydrogenase%20gene%20polymorphism%20with%20fluoropyrimidine-related%20toxicity%20in%20cancer%20patients&volume=3&publication_year=2008&pages=e4003&pmid=19104657&doi=10.1371/journal.pone.0004003&)

15. Kleibl Z, Fidlerova J, Kleiblova P, Kormunda S, Bilek M, Bouskova K, Sevcik J, Novotny J (2009) Influence of dihydropyrimidine dehydrogenase gene (DPYD) coding sequence variants on the development of fluoropyrimidine-related toxicity in patients with high-grade toxicity and patients with excellent tolerance of fluoropyrimidine-based chemotherapy. Neoplasma 56: 303–316.  [DOI](https://doi.org/10.4149/neo_2009_04_303) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19473056/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neoplasma&title=Influence%20of%20dihydropyrimidine%20dehydrogenase%20gene%20(DPYD)%20coding%20sequence%20variants%20on%20the%20development%20of%20fluoropyrimidine-related%20toxicity%20in%20patients%20with%20high-grade%20toxicity%20and%20patients%20with%20excellent%20tolerance%20of%20fluoropyrimidine-based%20chemotherapy&volume=56&publication_year=2009&pages=303&pmid=19473056&doi=10.4149/neo_2009_04_303&)

16. Lee AM, Shi Q, Pavey E, Alberts SR, Sargent DJ, Sinicrope FA, Berenberg JL, Goldberg RM, Diasio RB (2014) DPYD variants as predictors of 5-fluorouracil toxicity in adjuvant colon cancer treatment (NCCTG N0147). J Natl Cancer Inst 106: dju298.  [DOI](https://doi.org/10.1093/jnci/dju298) | [PMC free article](/articles/PMC4271081/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25381393/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Natl%20Cancer%20Inst&title=DPYD%20variants%20as%20predictors%20of%205-fluorouracil%20toxicity%20in%20adjuvant%20colon%20cancer%20treatment%20(NCCTG%20N0147)&volume=106&publication_year=2014&pages=dju298&pmid=25381393&doi=10.1093/jnci/dju298&)

17. Lee AM, Shi Q, Alberts SR, Sargent DJ, Sinicrope FA, Berenberg JL, Grothey A, Polite B, Chan E, Gill S, Kahlenberg MS, Nair SG, Shields AF, Goldberg RM, Diasio RB (2016) Association between DPYD c.1129-5923 C>G/hapB3 and severe toxicity to 5-fluorouracil-based chemotherapy in stage III colon cancer patients: NCCTG N0147(Alliance). Pharmacogenet Genomics 26: 133–137.  [DOI](https://doi.org/10.1097/FPC.0000000000000197) | [PMC free article](/articles/PMC4738010/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26658227/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Association%20between%20DPYD%20c.1129-5923%20C>G/hapB3%20and%20severe%20toxicity%20to%205-fluorouracil-based%20chemotherapy%20in%20stage%20III%20colon%20cancer%20patients:%20NCCTG%20N0147(Alliance)&volume=26&publication_year=2016&pages=133&pmid=26658227&doi=10.1097/FPC.0000000000000197&)

18. Lonardi S, Sobrero A, Rosati G, Di Bartolomeo M, Ronzoni M, Aprile G, Scartozzi M, Banzi M, Zampino MG, Pasini F, Marchetti P, Cantore M, Zaniboni A, Rimassa L, Ciuffreda L, Ferrari D, Barni S, Zagonel V, Maiello E, Rulli E, Labianca R (2016) TOSCA (Three or Six Colon Adjuvant) Investigators. Phase III trial comparing 3-6 months of adjuvant FOLFOX4/XELOX in stage II-III colon cancer: safety and compliance in the TOSCA trial. Ann Oncol 27: 2074–2081.  [DOI](https://doi.org/10.1093/annonc/mdw404) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27573560/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Oncol&title=TOSCA%20(Three%20or%20Six%20Colon%20Adjuvant)%20Investigators.%20Phase%20III%20trial%20comparing%203-6%20months%20of%20adjuvant%20FOLFOX4/XELOX%20in%20stage%20II-III%20colon%20cancer:%20safety%20and%20compliance%20in%20the%20TOSCA%20trial&volume=27&publication_year=2016&pages=2074&pmid=27573560&doi=10.1093/annonc/mdw404&)

19. Mattison LK, Johnson MR, Diasio RB (2002) A comparative analysis of translated dihydropyrimidine dehydrogenase cDNA; conservation of functional domains and relevance to genetic polymorphisms. Pharmacogenetics 12(2): 133–144.  [DOI](https://doi.org/10.1097/00008571-200203000-00007) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11875367/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=A%20comparative%20analysis%20of%20translated%20dihydropyrimidine%20dehydrogenase%20cDNA;%20conservation%20of%20functional%20domains%20and%20relevance%20to%20genetic%20polymorphisms&volume=12&publication_year=2002&pages=133&pmid=11875367&doi=10.1097/00008571-200203000-00007&)

20. Meulendijks D, Cats A, Beijnen JH, Schellens JH (2016) Improving safety of fluoropyrimidine chemotherapy by individualizing treatment based on dihydropyrimidine dehydrogenase activity—ready for clinical practice? Cancer Treat Rev 50: 23–34.  [DOI](https://doi.org/10.1016/j.ctrv.2016.08.002) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27589829/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Treat%20Rev&title=Improving%20safety%20of%20fluoropyrimidine%20chemotherapy%20by%20individualizing%20treatment%20based%20on%20dihydropyrimidine%20dehydrogenase%20activity%E2%80%94ready%20for%20clinical%20practice&volume=50&publication_year=2016&pages=23&pmid=27589829&doi=10.1016/j.ctrv.2016.08.002&)

21. Meulendijks D, Henricks LM, Sonke GS, Deenen MJ, Froehlich TK, Amstutz U, Largiadèr CR, Jennings BA, Marinaki AM, Sanderson JD, Kleibl Z, Kleiblova P, Schwab M, Zanger UM, Palles C, Tomlinson I, Gross E, van Kuilenburg AB, Punt CJ, Koopman M, Beijnen JH, Cats A, Schellens JH (2015) Clinical relevance of DPYD variants c.1679T>G, c.1236G>A/HapB3, and c.1601G>A as predictors of severe fluoropyrimidine-associated toxicity: a systematic review and meta-analysis of individual patient data. Lancet Oncol 16: 1639–1650.  [DOI](https://doi.org/10.1016/S1470-2045(15)00286-7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26603945/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet%20Oncol&title=Clinical%20relevance%20of%20DPYD%20variants%20c.1679T>G,%20c.1236G>A/HapB3,%20and%20c.1601G>A%20as%20predictors%20of%20severe%20fluoropyrimidine-associated%20toxicity:%20a%20systematic%20review%20and%20meta-analysis%20of%20individual%20patient%20data&volume=16&publication_year=2015&pages=1639&pmid=26603945&doi=10.1016/S1470-2045(15)00286-7&)

22. Meulendijks D, Henricks LM, Jacobs BAW, Aliev A, Deenen MJ, de Vries N, Rosing H, van Werkhoven E, de Boer A, Beijnen JH, Mandigers CMPW, Soesan M, Cats A, Schellens JHM (2017) Pretreatment serum uracil concentration as a predictor of severe and fatal fluoropyrimidine-associated toxicity. Br J Cancer 116(11): 1415–1424.  [DOI](https://doi.org/10.1038/bjc.2017.94) | [PMC free article](/articles/PMC5520099/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28427087/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Cancer&title=Pretreatment%20serum%20uracil%20concentration%20as%20a%20predictor%20of%20severe%20and%20fatal%20fluoropyrimidine-associated%20toxicity&volume=116&publication_year=2017&pages=1415&pmid=28427087&doi=10.1038/bjc.2017.94&)

23. Offer SM, Diasio RB (2016) Biomarkers of fluorouracil toxicity: insight from the PETACC-8 trial. JAMA Oncol 2: 662–663.  [DOI](https://doi.org/10.1001/jamaoncol.2015.5463) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26797288/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA%20Oncol&title=Biomarkers%20of%20fluorouracil%20toxicity:%20insight%20from%20the%20PETACC-8%20trial&volume=2&publication_year=2016&pages=662&pmid=26797288&doi=10.1001/jamaoncol.2015.5463&)

24. Rosmarin D, Palles C, Church D, Domingo E, Jones A, Johnstone E, Wang H, Love S, Julier P, Scudder C, Nicholson G, Gonzalez-Neira A, Martin M, Sargent D, Green E, McLeod H, Zanger UM, Schwab M, Braun M, Seymour M, Thompson L, Lacas B, Boige V, Ribelles N, Afzal S, Enghusen H, Jensen SA, Etienne-Grimaldi MC, Milano G, Wadelius M, Glimelius B, Garmo H, Gusella M, Lecomte T, Laurent-Puig P, Martinez-Balibrea E, Sharma R, Garcia-Foncillas J, Kleibl Z, Morel A, Pignon JP, Midgley R, Kerr D, Tomlinson I (2014) Genetic markers of toxicity from capecitabine and other fluorouracil-based regimens: investigation in the QUASAR2 study, systematic review, and meta-analysis. J Clin Oncol 32: 1031–1039.  [DOI](https://doi.org/10.1200/JCO.2013.51.1857) | [PMC free article](/articles/PMC4879695/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24590654/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Genetic%20markers%20of%20toxicity%20from%20capecitabine%20and%20other%20fluorouracil-based%20regimens:%20investigation%20in%20the%20QUASAR2%20study,%20systematic%20review,%20and%20meta-analysis&volume=32&publication_year=2014&pages=1031&pmid=24590654&doi=10.1200/JCO.2013.51.1857&)

25. Ruzzo A, Graziano F, Galli F, Giacomini E, Floriani I, Galli F, Rulli E, Lonardi S, Ronzoni M, Massidda B, Zagonel V, Pella N, Mucciarini C, Labianca R, Ionta MT, Veltri E, Sozzi P, Barni S, Ricci V, Foltran L, Nicolini M, Biondi E, Bramati A, Turci D, Lazzarelli S, Verusio C, Bergamo F, Sobrero A, Frontini L, Magnani M (2014) Genetic markers for toxicity of adjuvant oxaliplatin and fluoropyrimidines in the phase III TOSCA trial in high-risk colon cancer patients. Sci Rep 5: 6828.  [DOI](https://doi.org/10.1038/srep06828) | [PMC free article](/articles/PMC4220280/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25370899/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Sci%20Rep&title=Genetic%20markers%20for%20toxicity%20of%20adjuvant%20oxaliplatin%20and%20fluoropyrimidines%20in%20the%20phase%20III%20TOSCA%20trial%20in%20high-risk%20colon%20cancer%20patients&volume=5&publication_year=2014&pages=6828&pmid=25370899&doi=10.1038/srep06828&)

26. Sahota T, Danhof M, Della Pasqua O (2016) Pharmacology-based toxicity assessment: towards quantitative risk prediction in humans. Mutagenesis 31: 359–374.  [DOI](https://doi.org/10.1093/mutage/gev081) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26970519/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mutagenesis&title=Pharmacology-based%20toxicity%20assessment:%20towards%20quantitative%20risk%20prediction%20in%20humans&volume=31&publication_year=2016&pages=359&pmid=26970519&doi=10.1093/mutage/gev081&)

27. Solè X, Guinò E, Valls J, Iniesta R, Moreno V (2006) SNPStats: a web tool for the analysis of association studies. Bioinformatics 22: 1928–1929.  [DOI](https://doi.org/10.1093/bioinformatics/btl268) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16720584/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=SNPStats:%20a%20web%20tool%20for%20the%20analysis%20of%20association%20studies&volume=22&publication_year=2006&pages=1928&pmid=16720584&doi=10.1093/bioinformatics/btl268&)

28. Teh LK, Hamzah S, Hashim H, Bannur Z, Zakaria ZA, Hasbullani Z, Shia JK, Fijeraid H, Md Nor A, Zailani M, Ramasamy P, Ngow H, Sood S, Salleh MZ (2013) Potential of dihydropyrimidine dehydrogenase genotypes in personalizing 5-fluorouracil therapy among colorectal cancer patients. Ther Drug Monit 35(5): 624–630.  [DOI](https://doi.org/10.1097/FTD.0b013e318290acd2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23942539/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ther%20Drug%20Monit&title=Potential%20of%20dihydropyrimidine%20dehydrogenase%20genotypes%20in%20personalizing%205-fluorouracil%20therapy%20among%20colorectal%20cancer%20patients&volume=35&publication_year=2013&pages=624&pmid=23942539&doi=10.1097/FTD.0b013e318290acd2&)

29. Thanarajasingam G, Atherton PJ, Novotny PJ, Loprinzi CL, Sloan JA, Grothey A (2016) Longitudinal adverse event assessment in oncology clinical trials: the Toxicity over Time (ToxT) analysis of Alliance trials NCCTG N9741 and 979254. Lancet Oncol 17: 663–670.  [DOI](https://doi.org/10.1016/S1470-2045(16)00038-3) | [PMC free article](/articles/PMC4910515/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27083333/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet%20Oncol&title=Longitudinal%20adverse%20event%20assessment%20in%20oncology%20clinical%20trials:%20the%20Toxicity%20over%20Time%20(ToxT)%20analysis%20of%20Alliance%20trials%20NCCTG%20N9741%20and%20979254&volume=17&publication_year=2016&pages=663&pmid=27083333&doi=10.1016/S1470-2045(16)00038-3&)

30. van Cutsem E, Cervantes A, Adam R, Sobrero A, Van Krieken JH, Aderka D, Aranda Aguilar E, Bardelli A, Benson A, Bodoky G, Ciardiello F, D'Hoore A, Diaz-Rubio E, Douillard JY, Ducreux M, Falcone A, Grothey A, Gruenberger T, Haustermans K, Heinemann V, Hoff P, Köhne CH, Labianca R, Laurent-Puig P, Ma B, Maughan T, Muro K, Normanno N, Österlund P, Oyen WJ, Papamichael D, Pentheroudakis G, Pfeiffer P, Price TJ, Punt C, Ricke J, Roth A, Salazar R, Scheithauer W, Schmoll HJ, Tabernero J, Taïeb J, Tejpar S, Wasan H, Yoshino T, Zaanan A, Arnold D (2016) ESMO consensus guidelines for the management of patients with metastatic colorectal cancer. Ann Oncol 27(8): 1386–1422.  [DOI](https://doi.org/10.1093/annonc/mdw235) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27380959/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Oncol&title=ESMO%20consensus%20guidelines%20for%20the%20management%20of%20patients%20with%20metastatic%20colorectal%20cancer&volume=27&publication_year=2016&pages=1386&pmid=27380959&doi=10.1093/annonc/mdw235&)

31. van Kuilenburg AB, van Lenthe H, Blom MJ, Mul EP, Van Gennip AH (1999) Profound variation in dihydropyrimidine dehydrogenase activity in human blood cells: major implications for the detection of partly deficient patients. Br J Cancer 79(3–4): 620–626.  [DOI](https://doi.org/10.1038/sj.bjc.6690097) | [PMC free article](/articles/PMC2362418/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10027339/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Cancer&title=Profound%20variation%20in%20dihydropyrimidine%20dehydrogenase%20activity%20in%20human%20blood%20cells:%20major%20implications%20for%20the%20detection%20of%20partly%20deficient%20patients&volume=79&publication_year=1999&pages=620&pmid=10027339&doi=10.1038/sj.bjc.6690097&)

32. van Kuilenburg AB, Meijer J, Gökcay G, Baykal T, Rubio-Gozalbo ME, Mul AN, de Die-Smulders CE, Weber P, Mori AC, Bierau J, Fowler B, Macke K, Sass JO, Meinsma R, Hennermann JB, Miny P, Zoetekouw L, Roelofsen J, Vijzelaar R, Nicolai J, Hennekam RC (2010) Dihydropyrimidine dehydrogenase deficiency caused by a novel genomic deletion c.505_513del of DPYD. Nucleosides Nucleotides Nucleic Acids29: 509–514.  [DOI](https://doi.org/10.1080/15257771003730227) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20544545/)

33. van Staveren MC, Guchelaar HJ, van Kuilenburg AB, Gelderblom H, Maring JG (2013) Evaluation of predictive tests for screening for dihydropyrimidine dehydrogenase deficiency. Pharmacogenomics J 13(5): 389–395, Review. Erratum in: (2014) Pharmacogenomics J 14(4): 400.  [DOI](https://doi.org/10.1038/tpj.2013.25) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23856855/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=Evaluation%20of%20predictive%20tests%20for%20screening%20for%20dihydropyrimidine%20dehydrogenase%20deficiency&volume=13&publication_year=2013&pages=389&pmid=23856855&doi=10.1038/tpj.2013.25&)

34. van Staveren MC, van Kuilenburg AB, Guchelaar HJ, Meijer J, Punt CJ, de Jong RS, Gelderblom H, Maring JG (2016) Evaluation of an oral uracil loading test to identify DPD-deficient patients using a limited sampling strategy. Br J Clin Pharmacol 81(3): 553–561.  [DOI](https://doi.org/10.1111/bcp.12821) | [PMC free article](/articles/PMC4767209/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26551538/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Clin%20Pharmacol&title=Evaluation%20of%20an%20oral%20uracil%20loading%20test%20to%20identify%20DPD-deficient%20patients%20using%20a%20limited%20sampling%20strategy&volume=81&publication_year=2016&pages=553&pmid=26551538&doi=10.1111/bcp.12821&)
