# A genome-wide association study identifies a candidate gene associated with atazanavir exposure measured in hair

## Metadata
**Authors:** Bani Tamraz, Yong Huang, Audrey L French, Seble Kassaye, Kathryn Anastos, Marek J Nowicki, Stephen Gange, Deborah R Gustafson, Peter Bacchetti, Ruth M Greenblatt, Pirro G Hysi, Bradley E Aouizerat, Women’s Interagency HIV Study
**Journal:** Clinical pharmacology and therapeutics
**Date:** 2018 Feb 8
**DOI:** [10.1002/cpt.1014](https://doi.org/10.1002/cpt.1014)
**PMID:** 29315502
**PMCID:** PMC6037621
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6037621/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC6037621/pdf/nihms933666.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC6037621/pdf/nihms933666.pdf)

## Abstract

Hair provides a direct measure of long-term exposure of atazanavir (ATV). We report the results of the first genome-wide association study (GWAS) of ATV exposure measured in hair in an observational cohort representative of U.S. women living with HIV; the Women’s Interagency HIV Study. Approximately 14.1 million SNPs were analyzed in linear regression-based GWAS, with replication, adjusted for non-genetic predictors collected under conditions of actual use of ATV in 398 participants. Lastly, the PharmGKB database was used to identify pharmacogene associations with ATV exposure. The rs73208473, within intron 1 of SORCS2, resulted in 0.46 fold decreased in ATV exposure, with strongest association (P = 1.71×10−8) in GWAS. A priori pharmacogene screening did not identify additional variants statistically significantly associated with ATV exposure, including those previously published in ATV plasma candidate pharmacogene studies. The findings demonstrate the potential value of pharmacogenomic GWAS in ethnically diverse populations under conditions of actual use.

Keywords: GWAS, Atazanavir, Pharmacogenomics, Pharmacogenetics, WIHS, Admixed population, Hair concentrations, HIV, Women, Antiretroviral

## INTRODUCTION

Twenty to forty percent of patients using combination antiretroviral therapy (cART) fail to achieve sustained virological suppression.([1](#R1)–[5](#R5)) With over one million individuals currently living with HIV in the U.S.([6](#R6)), this implies 200,000–400,000 Americans are at risk for suboptimal HIV treatment outcomes. Therapeutic and toxic effects of most drugs, including antiretrovirals (ARVs), are directly related to drug exposure.([7](#R7), [8](#R8)) In addition to host genetics, drug exposure and response are influenced by a variety of non-genetic factors including adherence, comorbidities and concomitant medications.([9](#R9)) Most studies that have measured pharmacokinetics (PK) and pharmacodynamics (PD) of ARVs observe this variability, but are unable to account for a large proportion of the observed variability.

Given the current expectation that cART will be indefinite in duration, ([10](#R10), [11](#R11)) it is crucial to develop models that can predict outcomes in the setting of comorbidities and in genetically diverse populations. Much of the information we have on the determinants of ARV exposure is based on studies that enroll participants with fewer comorbidities and concurrent medications, more limited ethnic diversity and fewer women as compared to patients in clinical care.([12](#R12)) The gap between clinical trial-eligible participants and clinical populations often limits generalizability and may bias studies against detection of important factors that influence ARV exposure.

Atazanavir (ATV), a potent inhibitor of HIV-1 protease,([13](#R13)) has been a common component of cART regimens. Despite its favorable pharmacologic properties that enable once a day dosing, the PK of ATV are associated with high inter- and intra-individual variabilities in exposure.([14](#R14)) Innate factors that influence ATV plasma concentration and pharmacokinetic parameters described in the literature include variations in *ABCB1*([15](#R15)–[18](#R18)), *NR1I2*([19](#R19), [20](#R20)), *CYP3A4/A5*([21](#R21), [22](#R22)) from candidate gene studies, and non-genetic factors that currently include adherence([21](#R21)), race/ethnicity([22](#R22)), and sex([18](#R18)). There is a paucity of published literature evaluating, ethnicity, concurrent medications, other comorbidities, and comprehensive evaluation of candidate genes associated with ATV exposure. An understanding of genetic and non-genetic factors, especially under conditions of actual use of ATV, is important for optimizing and maintaining ATV concentrations for consistent inhibition of viral replication. The Women’s Interagency HIV Study (WIHS) has conducted extensive pharmacologic and genetic characterization in representative, understudied and ethnically diverse women. The current study utilizes a hypothesis generating design that examines genome-wide data (including pharmacogenome) to find genetic variants that influence ATV exposure; this approach can identify novel variants and previously unconsidered candidate pharmacogenes.

## RESULTS

[Table 1](#T1) summarizes the characteristics of women who contributed data to these analyses. Self-reported adherence for all participants at the visit with the highest measured concentration of ATV in hair was >=95%. [Figure 1A](#F1) depicts ATV concentrations measured in hair for all participants over all visits. The mean ATV level was 4.34 ± 4.41 ng/mg of hair. Participants reported concurrent use of ritonavir with ATV in the majority of visits (85%). WIHS has previously published on the existence of a relationship between ATV level in hair and virologic suppression ([23](#R23)), and a similar trend exists in the current sample ([Figure 1B](#F1)). Although ATV dose was not recorded, we assumed that any participant who is on ritonavir is on a dose of 300 mg of ATV, boosted with 100 mg Ritonavir (A standard dose). The percent of participants on ATV/ritonavir regiment in each hair quintile with undetectable viral load (i.e., <80 copies/ml) is comparable to values calculated for all participants in this study ([Figure 1B](#F1)).

### Table 1.

| Characteristic | Hair data group£ (n=398) |
| --- | --- |
| Age (years)¥ | 43.9 ± 8.7 |
|   |  |
| BMI (kg/m2) | 28.5 ± 7.12 |
|   |  |
| Height (m) ¥ | 1.61 ± 0.07 |
|   |  |
| Self-reported ethnicity, n (%) |  |
| White (Non-Hispanic) | 64 (16.1%) |
| African-American (Non-Hispanic) | 194 (48.7%) |
| Hispanic | 130 (32.7%) |
| Asians, Pacific Islanders and others | 10(2.51%) |
|   |  |
| CrCl (n=392), n (%)¥ |  |
| Normal (>90 ml/min) | 89 (22.7%) |
| Mild impairment (60–90 ml/min) | 216 (55.1%) |
| Moderate impairment (30–59 ml/min) | 79 (20.1%) |
| Severe impairment (<30 ml/min) | 8 (2.0%) |
|   |  |
| EGFR* (mL/min/1.73 m2) ¥ | 91.7 ± 26.4 |
|   |  |
| Self-reported adherence, n (%) |  |
| >=95% | 398 (100%) |
| 75–94% | 0 (0) |
| <75% | 0 (0) |
|   |  |
| HBV antigen positive (n=394), n (%) | 6 (1.3%) |
|   |  |
| HCV RNA positive, n (%) | 69 (17.3%) |
|   |  |
| Liver enzymes (IU) |  |
| ALT | 26.9 ± 29.3 |
| AST | 29.4 ± 20.1 |
| GGT¥ | 45.3 ± 64.2 |
|   |  |
| Bilirubin, total (mg/dl) | 1.7 ± 1.2 |
|   |  |
| Crack use (n=394), n (%) | 18 (4.5%) |
|   |  |
| Cocaine use (n=394), n (%)¥ | 3 (0.8%) |
|   |  |
| Heroin use (n=394), n (%) | 5 (1.2%) |
|   |  |
| Smoker, n (n=394) (%) | 160 (40.1%) |
|   |  |
| Alcohol use (n=394), n (%) |  |
| Abstain | 259 (65.7%) |
| >0–7 drinks/week | 121(30.7%) |
| >7–12 drinks/week | 9 (2.3%) |
| >12 drinks/week | 5 (1.3%) |
|   |  |
| Drug interaction variables, n (%) |  |
| Medications that Decrease ATV exposure | 170 (42.7%) |
| Antacids | 21 (5.3%) |
| Ritonavir¥ | 339 (85.2%) |
|   |  |
| Not pregnant (n=397), n (%) | 394 (99.2%) |

Table 1 Caption: Demographics and clinical characteristics of study participants.

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c1e3/6037621/627ac5ad71da/nihms933666f1.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=6037621_nihms933666f1.jpg)

Atazanavir (ATV) hair concentrations measured in hair (n=1,575) (panel A) and percent of participants in each hair quintile with undetectable viral load (i.e., <80 copies/ml) (panel B).

Of 398 individuals providing ATV measures, 379 participants passed all QC criteria for inclusion in the GWAS: 190 African-Americans (AA), 57 Caucasians (WT), and 53 Hispanic (HIS). The additional 79 subjects that were either ethnically admixed or did not fit in any particular ethnic cluster were retained for replication analysis. Three strata were sufficiently large for GWAS analysis (i.e., AA, WT, HIS). The genomic control factor inflation was low for each group (1.004, 1.052 and 1.005 respectively), suggesting that there was no major admixture or stratification effects left uncontrolled in the samples.

Combining the three strata identified a strong association between ATV level and a cluster of 16 SNPs on chromosome 4p16.1 ([Figure 2](#F2)). The most statistically significant SNP in this cluster (rs73208473, p= 1.71x10^−8^, [Table 2](#T2)), is a non-coding SNP located within a linkage disequilibrium (LD) block that contains intron 1 of the *SORCS2* gene ([Table 2](#T2), [Figures 2](#F2) and [3](#F3)). On average, each copy of the rs73208473 minor “A” allele was associated with a 46% decrease in ATV levels (95%CI 0.35–0.61, [Figure 3](#F3)). Although an imputed SNP, rs73208473 had an excellent imputation score (rsq= 0.973) and was in high LD with other directly measured SNPs showing strong association with ATV levels (r^2^= 0.86 across our entire multi-ethnic sample with rs73208470, association with ATV level p= 8.04x10^−08^). SNP rs73208473 is relatively common, with allele frequencies of 0.118, 0.319 and 0.119 in the AA, WT and HIS participants respectively.

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c1e3/6037621/4ab75e7cc4f8/nihms933666f2.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=6037621_nihms933666f2.jpg)

Manhattan plot of the GWAS results from the meta-analysis of ATV exposure. The genomic position for each SNP for 22 autosomes are plotted along the x-axis, and the values on the y-axis denote the –log10-transformed and λ-adjusted p-values from the meta-analysis of the associations with ATV hair level for each of the SNPs present in ≥ 240 WIHS participants. The green line denotes the a priori threshold for genome-wide significance (p = 5x10−8).

### Table 2.

| rsID | Genotyped or Imputed | Chr. | Position* | A1/A2 allele | MAF | Ethnic group (n) | Fold difference (95% CI) | P |
| --- | --- | --- | --- | --- | --- | --- | --- | --- |
| rs73208473 | Imputed | 4 | 7324596 | A/C | 0.118 | AA (194) | 0.54 (0.37–0.80) | 0.0025 |
|   |   |   |   |   | 0.319 | WT (57) | 0.34 (0.24–0.51) | 3.09x10−6 |
|   |   |   |   |   | 0.119 | HIS (53) | 0.91 (0.34–2.37) | 0.84 |
|   |   |   |   |   |   | Meta-analysis (300) | 0.46 (0.35–0.61) | 1.71x10−8 |
|   |  |  |  |  |  |  |  |  |
| rs73208470 | Genotypes | 4 | 7323913 | G/C | 0.143 | AA (194) | 0.61 (0.44–0.86) | 0.0055 |
|   |   |   |   |   | 0.324 | WT (57) | 0.34 (0.24–0.51) | 3.09x10−6 |
|   |   |   |   |   | 0.119 | HIS (53) | 0.91 (0.34–2.37) | 0.84 |
|   |   |   |   |   |   | Meta-analysis (300) | 0.50 (0.39–0.65) | 8.04x10−8 |

Table 2 Caption: Results from GWAS, meta-analysis and replication for association with ATV hair level in WIHS.

### Figure 3.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c1e3/6037621/01cda2168e36/nihms933666f3.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=6037621_nihms933666f3.jpg)

Regional association plots and recombination rates of the locus associated with ATV hair level. Data of both directly genotyped and imputed SNPs are presented. The SNP with the most significant association is denoted with a purple diamond. The plot was created using LocusZoom (49). The color coding of all other SNPs in linkage disequilibrium (LD) with rs73208473, estimated by CEU r2 from phase II HapMap is provided, with an LD of: red, r2 ≥ 0.8; yellow, 0.6 ≤ r2 < 0.8; green, 0.4 ≤ r2 < 0.6; cyan, 0.2 ≤ r2 < 0.4,; blue r2 < 0.2; and gray, r2 unknown, respectively. The left y axis (points) represents –log10 P values for each SNP association, the right y axis (lines, mostly obscured) represents the recombination rate, estimated from the International HapMap Project (50) and the x axis represents base-pair positions along the chromosome 4 based on human genome build 37.

Replication in independent samples is standard practice for genome-wide association studies.([24](#R24)) Given the unavailability of additional ATV measurements in hair, we fit a linear regression model in the 79 remaining participants who were not included in the initial stratified analysis. While the effect was in the same direction as with those observed in the stratified discovery analysis: Each additional copy of “A” allele was associated with a 0.87 fold reduction of ATV hair levels (95%CI 0.50 – 1.49, [Figure 4](#F4)), this finding was not statistically significant (p=0.31).

### Figure 4.

![Figure 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c1e3/6037621/1bbbe4491243/nihms933666f4.jpg)

Forest plot of rs7320847 SNP in SORCS2 for association with ATV hair level. Each point represents the estimated effect size for the rs7320847, A allele, transformed to the linear scale (using the function exp(linear regression beta) for its calculation) for each stratum of the analysis (AA, African American; WT, White; HIS, Hispanic). “Meta-GWAS” refers to the meta-analysis of the discovery cohorts, whereas “Replication” refers to the 79 multi-ethnic subjects analyzed separately. The bars indicate 95% confidence intervals.

We evaluated SNPs listed in the PharmGKB database to assess for associations of any of the variants previously cataloged for any pharmacogenomic effects with ATV exposure in the current study. [Table S3](#SD4) (online) describes the associations of the 136 variants that were below the nominal statistical threshold of significance (p < 0.05). While variants in numerous genes previously unknown to impact ATV concentration were associated with a decrease or increase in ATV level in hair (i.e., *RRM1*, rs12806698, fold difference = 0.57, 95% CI, 0.42–0.78; *ABCC2,* fold difference = 1.4, 95% CI, 1.11–1.78), none exceeded the Bonferroni threshold (p<1.29x10^−05^) and must be viewed preliminary.

We also evaluated variants in the *ABCB1*([15](#R15)–[18](#R18)), *NR1I2*([19](#R19), [20](#R20)) and *CYP3A5*([21](#R21), [22](#R22)) genes reported in previous ATV pharmacogenetic literature ([Table S4](#SD5) online). None of those variants substantially influenced ATV concentration in hair, but confidence intervals were too wide to rule out the possibility of an effect ([Table S4](#SD5) online). While other variants in these genes (i.e., *ABCB1*, rs2373586, fold difference = 1.36, 95% CI, 0.94–1.98; *CYP3A4/A5*, rs2246709, fold difference = 1.26, 95% CI, 0.99–1.59; *NR1I2*, rs3732356, fold difference = 1.22, 95% CI, 0.93–1.61)) were associated with ATV concentration in hair, the confidence intervals were too wide to rule out the possibility of no effect. The associations among these 3 gene variants and hair ATV levels had large p values (p>0.05) in the sample after adjusting for multiple testing.

## DISCUSSION

We carried out what is, to our knowledge, the first GWAS of ATV concentration measured in hair. The WIHS cohort is composed of participants who are representative of female HIV patients in the United States. These participants were assessed under the condition of actual ATV use and we adjusted for potential confounders. We built on the diversity of this cohort to test genome-wide in each ancestral group independently and then applied meta-analysis methods, demonstrating the value of GWAS methods applied to diverse populations in identifying new genetic factors that influence drug exposure. ARV exposure in hair is of interest because it is a readily accessible biomatrix representing longer duration of exposures and arguably longer-term ATV exposure (approximately 6 weeks) as compared with plasma. Furthermore, while hair is not the site of action for HIV medication, ATV concentrations in hair are strongly associated with viral suppression ([23](#R23)). As such, we believe the underlying model, that hair levels are a stable measurement depending primarily on long-term average serum levels is a reasonable one. This study provides evidence that common variants within the *SORCS2* gene are associated with decreased ATV concentration measured in hair.

The Sortilin-related VPS10 domain containing receptor 2 (*SORCS2*) is a transmembrane glycoprotein receptor and a member of vacuolar protein sorting 10 family proteins (VPS10Ps) that play roles in protein trafficking and intracellular and intercellular signaling.([25](#R25)) It has been implicated in amyotrophic lateral sclerosis (ALS) ([26](#R26)) and bipolar disease ([27](#R27), [28](#R28)). More interestingly, in line with our findings, previous work reported that three variants in this gene mediated *CYP3A4* induction by St. John’s Wort, but they found no evidence for *NR1I2* association with CYP3A4 overexpression ([29](#R29)). Given that ATV is primarily metabolized through CYP3A4/3A5 enzymes,([30](#R30)) our finding of *SORCS2* reducing ATV hair concentration would be consistent with overexpression of CYP3A4 enzyme. However, the functional properties of the *SORCS2* gene, its protein product and their eventual connection to CYP3A4/3A5 or ATV metabolism, warrants further investigation.

Additionally, within the same region of our finding (i.e., the intron 1 of *SORCS2)* there lies the microRNA MIR4798 ([Figure 3](#F3)). In general these short non-coding RNAs are involved in post-transcriptional regulation of gene expression that normally results in silencing mRNA expression. Although the functional properties of MIR4798 within the context of ATV expression or otherwise are unknown, the NR1I2 mRNA is one of the predicted targets of this particular miRNA ([http://www.targetscan.org/cgi-bin/targetscan/vert_71/targetscan.cgi?mirg=hsa-miR-4798-3p](http://www.targetscan.org/cgi-bin/targetscan/vert_71/targetscan.cgi?mirg=hsa-miR-4798-3p)). This finding is intriguing because the encoded protein from *NR1I2* (a.k.a, *PXR*) is a transcriptional regulator of *CYP3A4*([31](#R31), [32](#R32)) and *ABCB1*([32](#R32)), which are involved in metabolism of ATV.

While there are no directly comparable quantitative values reported in ATV pharmacogenetic literature, we can use plasma reporting as examples for such comparison, recognizing that plasma concentrations may not be directly and reliably related to hair levels. For example, carrying either one or two copies of the rs1045642 (C3435T) minor allele “T” (“CT” or “TT” genotypes), a coding SNP of *ABCB1*, was associated with decreased ATV minimum concentration in plasma (“CT”/”TT” genotypes: 376 ng/ml (interquartile range (IQR), 221–722 vs. “CC” genotype: 939 ng/ml (IQR, 492–1266), p= 0.001).([17](#R17)) In our study, presence of the “T” allele had a modest and statistically nonsignificant (p= 0.51) association with increased ATV hair level. While the confidence interval was too wide to rule out the possibility of substantial effect (Fold difference: 1.09, 95% CI: 0.85–1.41) in hair, it does provide evidence against an effect as large as 0.40-fold observed in plasma.

Remarkably, genetic variants previously reported in ATV pharmacogenetic studies, such as those in *ABCB1*([15](#R15)–[18](#R18)), *NR1I2*([19](#R19), [20](#R20)), and *CYP3A4/A5*([21](#R21), [22](#R22)), were not substantially and statistically significantly associated with ATV level in hair, but confidence intervals for some were too wide to rule out the possibility of a substantial effect. Possible explanations may relate largely to differences in what was measured (i.e., oral clearance ([21](#R21)), minimum plasma concentration ([22](#R22))) and study compartments (i.e., hair vs. plasma), as well as clinical and demographic differences. Another explanation could be due to unaccounted factors other than long-term ATV exposure that could impact ATV hair level. Our measured phenotypic and environmental covariates, while broad, may not completely control for variation in long-term ATV exposure in hair. Notably, actual adherence may have varied despite uniformly high self-reported adherence.

There are potential limitations to the study that need to be considered. First, the observed association should be treated as preliminary until they can be replicated in additional independent studies, as is common practice for all GWAS results. While the effect size in the replication sample is in the same direction as discovery analysis and interestingly similar to the HIS group used in the stratified discovery stage ([Figure 4](#F4)), the replication is not formally statistically significant (p=0.31) and therefore inconclusive. In addition to obvious precision limitation of the replication sample, the ethnic heterogeneity may also be a further limiting factor behind this result. Second, our findings were obtained from a relatively small initial sample, a situation which has in the past produced spurious results([33](#R33)). Finally, the considerable diversity in ethnic origin and the corresponding differences in allele frequencies and LD patterns across multi-ethnic samples could present a challenge with having sufficient power to detect some variants and overlook true genetic associations.

In summary, this work represents the first genome-wide study to elucidate genetic bases of ATV level in hair. Our findings suggest a complex combination between genetic and non-genetic factors. We have also identified a new candidate gene associated with ATV hair levels with a reasonable mechanistic basis. The results also suggest that specific genetic polymorphisms associated with ATV exposure may differ when assessed in hair among phenotypically and behaviorally complex HIV positive women (e.g., those with HCV infection, concurrent alcohol consumption, smoking, and users of illicit and other prescribed medication) under conditions of actual use compared to those in plasma. Our findings in hair should not be interpreted as refutation of existing literature on ATV pharmacogenetics conducted in plasma, because of the differences in patient population characteristics and other differences in effect sizes of risk factors that may need larger sample sizes for replication. To truly translate genetics into a change in clinical practice, our findings support the importance of conducting pharmacogenetic studies under conditions that best represent the actual use of a therapy and that consider the underlying ethnic background of the population, which represents not only genetic differences but unmeasured cultural factors.

## METHODS

### STUDY POPULATION

The WIHS is a multi-center observational cohort study of women with and at risk for HIV infection with follow-up visits every six months. Informed consent was provided by all participants via protocols approved by institutional review committees at each affiliated institution. Consent to studies of host genetics was specifically obtained for women who contributed to this study. Detailed methods have been published previously ([34](#R34), [35](#R35)). WIHS participants are representative of women with HIV infection in the United States a group that is not well represented in antiretroviral drug clinical trials.([36](#R36)) Over a period of 5 years (2003–2008), 466 WIHS participants reported at least 6 weeks of taking an ATV containing cART regimen and provided 1,575 hair samples for measurement of ATV concentration. For individuals with more than 1 measurement, the visit with highest level in hair was selected for this analysis. A subset of 398 individuals had available covariate and GWAS data for this analysis ([Table 1](#T1)). The only exclusion criteria for this study were not receiving ATV for at least 6 weeks, incomplete covariate and genotyping data.

### CONCENTRATION OF ATV IN HAIR

The method for extraction and analysis of ATV in hair using high performance liquid chromatography tandem mass spectrometry (HPLC-MS) has been described previously.([23](#R23))

### GENETIC AND STATISTICAL ANALYSES

This study is a linear regression-based genome-wide association study (GWAS) of ATV hair concentration. Given the need to control for potential bias arising from population structure due to WIHS participants’ ethnic diversity, a stratified GWAS was conducted in two stages. In the first stage, association analyses were performed in each strata of samples representing the most numerically large homogeneous ethnic groups, using within-group correction for the main Principal Components, as suggested elsewhere([37](#R37)). In the second phase, results from separate strata were combined. Validation was sought through analyzing samples that were not included in the first step (i.e., WIHS participants who could not be assigned to an ethnic group stratum), after PC-based adjustment for genetic admixture.

### OUTCOME

The main outcome was ATV level in hair, evaluated as a biomarker of ATV exposure, which provides a direct measure of long term ATV exposure. To control for non-genetic factors that can cause significant variability in blood ATV and thus hair, over multiple visits for the same patients, we selected as primary outcome of our analyses the highest ATV concentration in hair (in ng/mg) for a given participant. The rationale for this choice is that the genetic effects remain constant over time, while the confounding from variable factors (e.g., renal function, self-reported adherence, hepatic function) was at their minimum of their fluctuating influence.

Levels in hair below the limit of detection (i.e., <0.05 ng/mg of hair) were assigned a value of 0.05 ng/ml. Of 1,593 hair samples, 58 had concentrations below limit of quantification (BLOQ), but only 8 participants in the final analysis had atazanavir concentration that were BLOQ. The empirical distribution of the ATV concentration in hair is not normally distributed. Thus, for the purpose of statistical analyses, this outcome was normalized employing transformation (i.e., natural logarithm) prior to inclusion in regression models.

### INDEPENDENT VARIABLES

For the GWAS, the main predictor was the number of the rarer allele carried at each locus, ranging from 0 to 2. In addition to genotypes, we assessed a variety of non-genetic factors ([Table 1](#T1)) that may exert non-specific influences over ATV pharmacology, including age, body mass index (BMI), alcohol use, pregnancy status, self-reported race/ethnicity, self-reported adherence to cART, self-reported use of other medications, self-reported use of cocaine, heroin, or other injectable drugs, and current smoking. To better control for variation of ATV levels in hair associated with non-genetic factors, we additionally assessed clinical laboratory measures including estimated glomerular filtration rate (EGFR - calculated using the CKD-EPI equation([38](#R38))), estimated creatinine clearance (calculated using Cockcroft-Gault equation) serum aspartate aminotransferase (AST), alanine aminotransferase (ALT), gamma-glutamyl transferase (GGT), total bilirubin, laboratory evidence of chronic hepatitis B and/or C. Concurrently used medications were assessed for known impact on ATV pharmacokinetics (e.g., ritonavir, tenofovir, antacids) ([Table S1](#SD2) online) and drugs that had these effects were included as covariates in GWAS analysis. The strongest among these factors, which could increase error in the genetic analyses, were identified and then included as GWAS covariates. In brief, non-genetic predictors were selected following a stepwise backward regression that retained only variables showing a nominally statistically significant association (p<0.05) in the final multivariable model. As a result, the following non-genetic predictors were included as covariates in the GWAS: age, creatinine clearance category (as an ordinal variable), GGT, EGFR, cocaine use, body height and concurrent ritonavir use ([Table S2](#SD3) online).

### GENOTYPING AND IMPUTATION

Genomic DNA isolated from blood was genotyped on the Illumina Omni2.5 Array (San Diego, CA) platform (n ~ 2.5 million SNPs). Rigorous individual level and SNP level quality control were applied.([39](#R39)) Participants were excluded if they exhibited array-wide call rates <95%, excess heterozygosity or relatedness to other WIHS participants. Phase information from the 1000 Genomes Project (Phase 3 v5) panel was used to impute these genotypic data (n ~ 14 million SNPs). Among others, phasing was performed using the ShapeIT software([40](#R40)) and imputation using minimac3([41](#R41)). Only individuals with more than 90% of the genotypes available were included and hard calls from only SNPs with sufficient imputation quality scores (RSQ>0.5) and with minor allele frequencies (MAF)>0.05 and Hardy-Weinberg equilibrium (HWE) p-values>0.001 within each ethnic group were used for the analyses (n ~ 11.4 million SNPs). In total, approximately 14 million SNPs were available for analysis.

### LINEAR REGRESSION

In the stratified analysis, only ethnic groups comprised of more than 50 subjects among our genotyped samples were considered as a stratum, which were analyzed separately for association with ATV exposure in hair using PLINK 1.9 ([https://www.cog-genomics.org/plink2](https://www.cog-genomics.org/plink2)). This threshold of inclusion (n=50) corresponds to a minimum of 5 heterozygous participants for the lowest MAF analyzed (MAF=0.05). To further control for potential residual population structure within these ethnically homogeneous strata, we adjusted for the first three principal components ([37](#R37)) ([Figure S1](#SD1) online) as well as 7 clinical and demographic covariates selected as described above.

Results from each stratum were combined using GWAMA ([42](#R42)), using a fixed-effect inverse-variance model on results adjusted for the genomic control inflation factor (GC, or λ) ([43](#R43)) observed in each stratum. In addition to prior within-ethnic group inclusion thresholds, only results observed from sufficiently large samples (defined as >240 participants, or 80% of the original sample) were considered further. Cochran’s Q test and *I*^2^ were used to assess heterogeneity across strata and results with large effect heterogeneity (defined as *I2*> 75%) were removed from further consideration. For the GWAS the conventional significance threshold of 5x10^−8^ was adopted.([44](#R44)) Results were annotated to the nearest gene using SNPnexus ([45](#R45)–[47](#R47)).

#### PharmGKB database lookup

The pharmacogenomics knowledgebase (pharmGKB) database([48](#R48)) is a curated database of pharmacogenomically important variants and their impact on any known drug response. The reference sequence identifiers (rsID) ([https://www.pharmgkb.org/downloads/](https://www.pharmgkb.org/downloads/); Accession date: May 2017) of 5,098 variants in this database were downloaded and used as an unbiased source to annotate and evaluate for pharmacogene variant associations with ATV hair concentrations in this study. Additionally variants in *ABCB1* ([15](#R15)–[17](#R17)), *NR1I2* ([19](#R19), [20](#R20)), *CYP3A5* ([21](#R21), [22](#R22)) which are reported in pharmacogenetic studies of ATV in plasma were assessed for association with ATV concentration measured in hair. Of the 5,098 variants present in the pharmGKB at the time of the query, 3,877 were sufficiently common and passed quality controls to be assessed in the current study. For evaluation of associations among these loci, a Bonferroni adjusted significance threshold of 0.05/3,877 =1.29x10^−05^ was considered while noting that SNPs are not always independent and this threshold may be too conservative.

## Supplementary Material

### WHAT IS THE CURRENT KNOWLEDGE ON THE TOPIC?

### WHAT IS THE CURRENT KNOWLEDGE ON THE TOPIC?

### WHAT IS THE CURRENT KNOWLEDGE ON THE TOPIC?

### WHAT IS THE CURRENT KNOWLEDGE ON THE TOPIC?

Candidate gene studies have identified pharmacogenetic variants that influence pharmacokinetics of atazanavir (ATV) in plasma. However, these findings are derived from study samples poorly representing conditions and populations actually using ATV.

### WHAT QUESTION DID THIS STUDY ADDRESS?

This study evaluated women and treatment related variables collected under conditions of actual use of ATV to identify genetic associations with ATV concentration in hair; a measure of long-term exposure.

### WHAT DOES THIS STUDY ADD TO OUR KNOWLEDGE?

The association of *SORCS2* rs73208473 with reduced ATV exposure reached genome-wide significance. The results suggest that specific genetic polymorphisms associated with ATV exposure may differ when assessed in hair among phenotypically and behaviorally complex HIV positive women under conditions of actual use.

### HOW MIGHT THIS CHANGE CLINICAL PHARMACOLOGY OR TRANSLATIONAL SCIENCE?

The gap between clinical trials participants and clinical populations limits generalizability of clinical trial-based pharmacogenetic studies and may bias studies against detection of significant predictors of response. A better understanding of factors that contribute to medication outcomes is critical for optimizing treatment for all patients.

## Acknowledgments

**Funding:** UCSF-Gladstone Center For AIDS Research Grant: Bani Tamraz P30 A1027763; HHS | NIH | National Institute of Allergy and Infectious Diseases (NIAID): Kathryn Anastos U01-AI-035004; HHS | NIH | National Institute of Allergy and Infectious Diseases (NIAID): Deborah (R) Gustafson U01-AI-031834; HHS | NIH | National Institute of Allergy and Infectious Diseases (NIAID): Audrey (L) French U01-AI-034993; HHS | NIH | National Institute of Allergy and Infectious Diseases (NIAID): Seble Kassaye U01-AI-034994; HHS | NIH | National Institute of Allergy and Infectious Diseases (NIAID): Ruth (M.) Greenblatt, Bradley (E) Aouizerat U01-AI-034989; HHS | NIH | National Institute of Allergy and Infectious Diseases (NIAID): Stephen (J.) Gange U01-AI-042590; HHS | NIH | National Institute of Child Health and Human Development (NICHD): Marek (J) Nowicki U01-HD-032632; UCSF CTSA: Peter Bacchetti UL1-TR000004

Data in this manuscript were collected by the Women’s Interagency HIV Study (WIHS). The contents of this publication are solely the responsibility of the authors and do not represent the official views of the National Institutes of Health (NIH).

WIHS (Principal Investigators): Bronx WIHS (Kathryn Anastos), U01-AI-035004; Brooklyn WIHS (Howard Minkoff and Deborah Gustafson), U01-AI-031834; Chicago WIHS (Mardge Cohen and Audrey French), U01-AI-034993; Metropolitan Washington WIHS (Seble Kassaye), U01-AI-034994; Connie Wofsy Women’s HIV Study, Northern California (Ruth Greenblatt, Bradley Aouizerat, and Phyllis Tien), U01-AI-034989; WIHS Data Management and Analysis Center (Stephen Gange and Elizabeth Golub), U01-AI-042590; Southern California WIHS (Joel Milam), U01-HD-032632 (WIHS I - WIHS IV). The WIHS is funded primarily by the National Institute of Allergy and Infectious Diseases (NIAID), with additional co-funding from the *Eunice Kennedy Shriver* National Institute of Child Health and Human Development (NICHD), the National Cancer Institute (NCI), the National Institute on Drug Abuse (NIDA), and the National Institute on Mental Health (NIMH). Targeted supplemental funding for specific projects is also provided by the National Institute on Alcohol Abuse and Alcoholism (NIAAA), and the NIH Office of Research on Women’s Health. WIHS data collection and statistical analysis is also supported by UL1-TR000004 (UCSF CTSA) and data collection by UL1-TR000454 (Atlanta CTSA). Bani Tamraz was also supported by UCSF-Gladstone Center For AIDS Research grant (P30 A1027763).

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Moore RD, Bartlett JG. Dramatic decline in the HIV-1 RNA level over calendar time in a large urban HIV practice. Clin Infect Dis. 2011;53:600–4. doi: 10.1093/cid/cir467.  [DOI](https://doi.org/10.1093/cid/cir467) | [PMC free article](/articles/PMC3202317/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21844006/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Infect%20Dis&title=Dramatic%20decline%20in%20the%20HIV-1%20RNA%20level%20over%20calendar%20time%20in%20a%20large%20urban%20HIV%20practice&author=RD%20Moore&author=JG%20Bartlett&volume=53&publication_year=2011&pages=600-4&pmid=21844006&doi=10.1093/cid/cir467&)

2. Napravnik S, Eron JJ, Sterling TR, Juday T, Uy J, Moore RD. Outcomes of second combination antiretroviral therapy regimens among HIV-infected persons in clinical care: a multicenter cohort study. AIDS Res Hum Retroviruse. 2013;29:574–80. doi: 10.1089/aid.2012.0210.  [DOI](https://doi.org/10.1089/aid.2012.0210) | [PMC free article](/articles/PMC3581035/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23072322/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=AIDS%20Res%20Hum%20Retroviruse&title=Outcomes%20of%20second%20combination%20antiretroviral%20therapy%20regimens%20among%20HIV-infected%20persons%20in%20clinical%20care:%20a%20multicenter%20cohort%20study&author=S%20Napravnik&author=JJ%20Eron&author=TR%20Sterling&author=T%20Juday&author=J%20Uy&volume=29&publication_year=2013&pages=574-80&pmid=23072322&doi=10.1089/aid.2012.0210&)

3. Dray-Spira R, Spire B, Heard I, Lert F. Heterogeneous response to HAART across a diverse population of people living with HIV: results from the ANRS-EN12-VESPA Study. AIDS. 2007;21(Suppl 1):S5–12. doi: 10.1097/01.aids.0000255079.39352.9b.  [DOI](https://doi.org/10.1097/01.aids.0000255079.39352.9b) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17159588/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=AIDS&title=Heterogeneous%20response%20to%20HAART%20across%20a%20diverse%20population%20of%20people%20living%20with%20HIV:%20results%20from%20the%20ANRS-EN12-VESPA%20Study&author=R%20Dray-Spira&author=B%20Spire&author=I%20Heard&author=F%20Lert&volume=21&issue=Suppl%201&publication_year=2007&pages=S5-12&pmid=17159588&doi=10.1097/01.aids.0000255079.39352.9b&)

4. Pillay P, Ford N, Shubber Z, Ferrand RA. Outcomes for efavirenz versus nevirapine-containing regimens for treatment of HIV-1 infection: a systematic review and meta-analysis. PLoS One. 2013;8:e68995. doi: 10.1371/journal.pone.0068995.  [DOI](https://doi.org/10.1371/journal.pone.0068995) | [PMC free article](/articles/PMC3718822/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23894391/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=Outcomes%20for%20efavirenz%20versus%20nevirapine-containing%20regimens%20for%20treatment%20of%20HIV-1%20infection:%20a%20systematic%20review%20and%20meta-analysis&author=P%20Pillay&author=N%20Ford&author=Z%20Shubber&author=RA%20Ferrand&volume=8&publication_year=2013&pages=e68995&pmid=23894391&doi=10.1371/journal.pone.0068995&)

5. Bartlett JA, et al. Minimizing resistance consequences after virologic failure on initial combination therapy: a systematic overview. J Acquir Immune Defic Syndr. 2006;41:323–31. doi: 10.1097/01.qai.0000197070.69859.f3.  [DOI](https://doi.org/10.1097/01.qai.0000197070.69859.f3) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16540933/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Acquir%20Immune%20Defic%20Syndr&title=Minimizing%20resistance%20consequences%20after%20virologic%20failure%20on%20initial%20combination%20therapy:%20a%20systematic%20overview&author=JA%20Bartlett&volume=41&publication_year=2006&pages=323-31&pmid=16540933&doi=10.1097/01.qai.0000197070.69859.f3&)

6. CDC. HIV in the United States: At a Glance. Nov, 2013.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=HIV%20in%20the%20United%20States:%20At%20a%20Glance&publication_year=2013&)

7. Gallego L, et al. Analyzing sleep abnormalities in HIV-infected patients treated with Efavirenz. Clin Infect Dis. 2004;38:430–2. doi: 10.1086/380791.  [DOI](https://doi.org/10.1086/380791) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14727217/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Infect%20Dis&title=Analyzing%20sleep%20abnormalities%20in%20HIV-infected%20patients%20treated%20with%20Efavirenz&author=L%20Gallego&volume=38&publication_year=2004&pages=430-2&pmid=14727217&doi=10.1086/380791&)

8. Haas DW, et al. Pharmacogenetics of efavirenz and central nervous system side effects: an Adult AIDS Clinical Trials Group study. AIDS. 2004;18:2391–400.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/15622315/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=AIDS&title=Pharmacogenetics%20of%20efavirenz%20and%20central%20nervous%20system%20side%20effects:%20an%20Adult%20AIDS%20Clinical%20Trials%20Group%20study&author=DW%20Haas&volume=18&publication_year=2004&pages=2391-400&pmid=15622315&)

9. Gandhi M, et al. A single-nucleotide polymorphism in CYP2B6 leads to >3-fold increases in efavirenz concentrations in plasma and hair among HIV-infected women. J Infect Dis. 2012;206:1453–61. doi: 10.1093/infdis/jis508.  [DOI](https://doi.org/10.1093/infdis/jis508) | [PMC free article](/articles/PMC3466997/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22927450/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Infect%20Dis&title=A%20single-nucleotide%20polymorphism%20in%20CYP2B6%20leads%20to%20>3-fold%20increases%20in%20efavirenz%20concentrations%20in%20plasma%20and%20hair%20among%20HIV-infected%20women&author=M%20Gandhi&volume=206&publication_year=2012&pages=1453-61&pmid=22927450&doi=10.1093/infdis/jis508&)

10. Lohse N, et al. Survival of persons with and without HIV infection in Denmark, 1995–2005. Ann Intern Med. 2007;146:87–95. doi: 10.7326/0003-4819-146-2-200701160-00003.  [DOI](https://doi.org/10.7326/0003-4819-146-2-200701160-00003) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17227932/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Intern%20Med&title=Survival%20of%20persons%20with%20and%20without%20HIV%20infection%20in%20Denmark,%201995%E2%80%932005&author=N%20Lohse&volume=146&publication_year=2007&pages=87-95&pmid=17227932&doi=10.7326/0003-4819-146-2-200701160-00003&)

11. Mahungu TW, Rodger AJ, Johnson MA. HIV as a chronic disease. Clin Med. 2009;9:125–8. doi: 10.7861/clinmedicine.9-2-125.  [DOI](https://doi.org/10.7861/clinmedicine.9-2-125) | [PMC free article](/articles/PMC4952661/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19435115/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Med&title=HIV%20as%20a%20chronic%20disease&author=TW%20Mahungu&author=AJ%20Rodger&author=MA%20Johnson&volume=9&publication_year=2009&pages=125-8&pmid=19435115&doi=10.7861/clinmedicine.9-2-125&)

12. Loutfy MR, et al. Caring for women living with HIV: gaps in the evidence. J Int AIDS Soc. 2013;16:18509. doi: 10.7448/IAS.16.1.18509.  [DOI](https://doi.org/10.7448/IAS.16.1.18509) | [PMC free article](/articles/PMC3789211/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24088395/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Int%20AIDS%20Soc&title=Caring%20for%20women%20living%20with%20HIV:%20gaps%20in%20the%20evidence&author=MR%20Loutfy&volume=16&publication_year=2013&pages=18509&pmid=24088395&doi=10.7448/IAS.16.1.18509&)

13. Croom KF, Dhillon S, Keam SJ. Atazanavir: a review of its use in the management of HIV-1 infection. Drugs. 2009;69:1107–40. doi: 10.2165/00003495-200969080-00009.  [DOI](https://doi.org/10.2165/00003495-200969080-00009) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19496633/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drugs&title=Atazanavir:%20a%20review%20of%20its%20use%20in%20the%20management%20of%20HIV-1%20infection&author=KF%20Croom&author=S%20Dhillon&author=SJ%20Keam&volume=69&publication_year=2009&pages=1107-40&pmid=19496633&doi=10.2165/00003495-200969080-00009&)

14. Piliero PJ. Atazanavir: a novel HIV-1 protease inhibitor. Expert Opin Investig Drugs. 2002;11:1295–301. doi: 10.1517/13543784.11.9.1295.  [DOI](https://doi.org/10.1517/13543784.11.9.1295) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12225250/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Expert%20Opin%20Investig%20Drugs&title=Atazanavir:%20a%20novel%20HIV-1%20protease%20inhibitor&author=PJ%20Piliero&volume=11&publication_year=2002&pages=1295-301&pmid=12225250&doi=10.1517/13543784.11.9.1295&)

15. Bonora S, et al. Successful pharmacogenetics-based optimization of unboosted atazanavir plasma exposure in HIV-positive patients: a randomized, controlled, pilot study (the REYAGEN study) J Antimicrob Chemother. 2015 doi: 10.1093/jac/dkv208.  [DOI](https://doi.org/10.1093/jac/dkv208) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26174719/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Antimicrob%20Chemother&title=Successful%20pharmacogenetics-based%20optimization%20of%20unboosted%20atazanavir%20plasma%20exposure%20in%20HIV-positive%20patients:%20a%20randomized,%20controlled,%20pilot%20study%20(the%20REYAGEN%20study)&author=S%20Bonora&publication_year=2015&pmid=26174719&doi=10.1093/jac/dkv208&)

16. D’Avolio A, et al. Intracellular accumulation of atazanavir/ritonavir according to plasma concentrations and OATP1B1, ABCB1 and PXR genetic polymorphisms. J Antimicrob Chemother. 2014;69:3061–6. doi: 10.1093/jac/dku234.  [DOI](https://doi.org/10.1093/jac/dku234) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24997317/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Antimicrob%20Chemother&title=Intracellular%20accumulation%20of%20atazanavir/ritonavir%20according%20to%20plasma%20concentrations%20and%20OATP1B1,%20ABCB1%20and%20PXR%20genetic%20polymorphisms&author=A%20D%E2%80%99Avolio&volume=69&publication_year=2014&pages=3061-6&pmid=24997317&doi=10.1093/jac/dku234&)

17. Rodriguez-Novoa S, et al. Genetic factors influencing atazanavir plasma concentrations and the risk of severe hyperbilirubinemia. AIDS. 2007;21:41–6. doi: 10.1097/QAD.0b013e328011d7c1.  [DOI](https://doi.org/10.1097/QAD.0b013e328011d7c1) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17148966/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=AIDS&title=Genetic%20factors%20influencing%20atazanavir%20plasma%20concentrations%20and%20the%20risk%20of%20severe%20hyperbilirubinemia&author=S%20Rodriguez-Novoa&volume=21&publication_year=2007&pages=41-6&pmid=17148966&doi=10.1097/QAD.0b013e328011d7c1&)

18. Rodriguez Novoa S, et al. Plasma levels of atazanavir and the risk of hyperbilirubinemia are predicted by the 3435C-->T polymorphism at the multidrug resistance gene 1. Clin Infect Dis. 2006;42:291–5. doi: 10.1086/499056.  [DOI](https://doi.org/10.1086/499056) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16355344/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Infect%20Dis&title=Plasma%20levels%20of%20atazanavir%20and%20the%20risk%20of%20hyperbilirubinemia%20are%20predicted%20by%20the%203435C-->T%20polymorphism%20at%20the%20multidrug%20resistance%20gene%201&author=S%20Rodriguez%20Novoa&volume=42&publication_year=2006&pages=291-5&pmid=16355344&doi=10.1086/499056&)

19. Siccardi M, et al. Association of a single-nucleotide polymorphism in the pregnane X receptor (PXR 63396C-->T) with reduced concentrations of unboosted atazanavir. Clin Infect Dis. 2008;47:1222–5. doi: 10.1086/592304.  [DOI](https://doi.org/10.1086/592304) | [PMC free article](/articles/PMC3672984/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18831695/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Infect%20Dis&title=Association%20of%20a%20single-nucleotide%20polymorphism%20in%20the%20pregnane%20X%20receptor%20(PXR%2063396C-->T)%20with%20reduced%20concentrations%20of%20unboosted%20atazanavir&author=M%20Siccardi&volume=47&publication_year=2008&pages=1222-5&pmid=18831695&doi=10.1086/592304&)

20. Schipani A, et al. Population pharmacokinetic modeling of the association between 63396C->T pregnane X receptor polymorphism and unboosted atazanavir clearance. Antimicrob Agents Chemother. 2010;54:5242–50. doi: 10.1128/AAC.00781-10.  [DOI](https://doi.org/10.1128/AAC.00781-10) | [PMC free article](/articles/PMC2981241/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20921307/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Antimicrob%20Agents%20Chemother&title=Population%20pharmacokinetic%20modeling%20of%20the%20association%20between%2063396C->T%20pregnane%20X%20receptor%20polymorphism%20and%20unboosted%20atazanavir%20clearance&author=A%20Schipani&volume=54&publication_year=2010&pages=5242-50&pmid=20921307&doi=10.1128/AAC.00781-10&)

21. Savic RM, et al. Effect of adherence as measured by MEMS, ritonavir boosting, and CYP3A5 genotype on atazanavir pharmacokinetics in treatment-naive HIV-infected patients. Clin Pharmacol Ther. 2012;92:575–83. doi: 10.1038/clpt.2012.137.  [DOI](https://doi.org/10.1038/clpt.2012.137) | [PMC free article](/articles/PMC3939416/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23033116/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Effect%20of%20adherence%20as%20measured%20by%20MEMS,%20ritonavir%20boosting,%20and%20CYP3A5%20genotype%20on%20atazanavir%20pharmacokinetics%20in%20treatment-naive%20HIV-infected%20patients&author=RM%20Savic&volume=92&publication_year=2012&pages=575-83&pmid=23033116&doi=10.1038/clpt.2012.137&)

22. Anderson PL, et al. Atazanavir pharmacokinetics in genetically determined CYP3A5 expressors versus non-expressors. J Antimicrob Chemother. 2009;64:1071–9. doi: 10.1093/jac/dkp317.  [DOI](https://doi.org/10.1093/jac/dkp317) | [PMC free article](/articles/PMC2760462/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19710077/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Antimicrob%20Chemother&title=Atazanavir%20pharmacokinetics%20in%20genetically%20determined%20CYP3A5%20expressors%20versus%20non-expressors&author=PL%20Anderson&volume=64&publication_year=2009&pages=1071-9&pmid=19710077&doi=10.1093/jac/dkp317&)

23. Gandhi M, et al. Atazanavir concentration in hair is the strongest predictor of outcomes on antiretroviral therapy. Clin Infect Dis. 2011;52:1267–75. doi: 10.1093/cid/cir131.  [DOI](https://doi.org/10.1093/cid/cir131) | [PMC free article](/articles/PMC3079399/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21507924/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Infect%20Dis&title=Atazanavir%20concentration%20in%20hair%20is%20the%20strongest%20predictor%20of%20outcomes%20on%20antiretroviral%20therapy&author=M%20Gandhi&volume=52&publication_year=2011&pages=1267-75&pmid=21507924&doi=10.1093/cid/cir131&)

24. McCarthy MI, et al. Genome-wide association studies for complex traits: consensus, uncertainty and challenges. Nat Rev Genet. 2008;9:356–69. doi: 10.1038/nrg2344.  [DOI](https://doi.org/10.1038/nrg2344) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18398418/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Genet&title=Genome-wide%20association%20studies%20for%20complex%20traits:%20consensus,%20uncertainty%20and%20challenges&author=MI%20McCarthy&volume=9&publication_year=2008&pages=356-69&pmid=18398418&doi=10.1038/nrg2344&)

25. Lane RF, St George-Hyslop P, Hempstead BL, Small SA, Strittmatter SM, Gandy S. Vps10 family proteins and the retromer complex in aging-related neurodegeneration and diabetes. J Neurosci. 2012;32:14080–6. doi: 10.1523/JNEUROSCI.3359-12.2012.  [DOI](https://doi.org/10.1523/JNEUROSCI.3359-12.2012) | [PMC free article](/articles/PMC3576841/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23055476/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Neurosci&title=Vps10%20family%20proteins%20and%20the%20retromer%20complex%20in%20aging-related%20neurodegeneration%20and%20diabetes&author=RF%20Lane&author=P%20St%20George-Hyslop&author=BL%20Hempstead&author=SA%20Small&author=SM%20Strittmatter&volume=32&publication_year=2012&pages=14080-6&pmid=23055476&doi=10.1523/JNEUROSCI.3359-12.2012&)

26. Mori F, et al. Sortilin-related receptor CNS expressed 2 (SorCS2) is localized to Bunina bodies in amyotrophic lateral sclerosis. Neurosci Lett. 2015;608:6–11. doi: 10.1016/j.neulet.2015.09.030.  [DOI](https://doi.org/10.1016/j.neulet.2015.09.030) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26420026/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neurosci%20Lett&title=Sortilin-related%20receptor%20CNS%20expressed%202%20(SorCS2)%20is%20localized%20to%20Bunina%20bodies%20in%20amyotrophic%20lateral%20sclerosis&author=F%20Mori&volume=608&publication_year=2015&pages=6-11&pmid=26420026&doi=10.1016/j.neulet.2015.09.030&)

27. Ollila HM, et al. Findings from bipolar disorder genome-wide association studies replicate in a Finnish bipolar family-cohort. Mol Psychiatry. 2009;14:351–3. doi: 10.1038/mp.2008.122.  [DOI](https://doi.org/10.1038/mp.2008.122) | [PMC free article](/articles/PMC2694406/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19308021/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Psychiatry&title=Findings%20from%20bipolar%20disorder%20genome-wide%20association%20studies%20replicate%20in%20a%20Finnish%20bipolar%20family-cohort&author=HM%20Ollila&volume=14&publication_year=2009&pages=351-3&pmid=19308021&doi=10.1038/mp.2008.122&)

28. Baum AE, et al. A genome-wide association study implicates diacylglycerol kinase eta (DGKH) and several other genes in the etiology of bipolar disorder. Mol Psychiatry. 2008;13:197–207. doi: 10.1038/sj.mp.4002012.  [DOI](https://doi.org/10.1038/sj.mp.4002012) | [PMC free article](/articles/PMC2527618/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17486107/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Psychiatry&title=A%20genome-wide%20association%20study%20implicates%20diacylglycerol%20kinase%20eta%20(DGKH)%20and%20several%20other%20genes%20in%20the%20etiology%20of%20bipolar%20disorder&author=AE%20Baum&volume=13&publication_year=2008&pages=197-207&pmid=17486107&doi=10.1038/sj.mp.4002012&)

29. Rahmioglu N, et al. Genome-wide association study reveals a complex genetic architecture underpinning-induced CYP3A4 enzyme activity. Eur J Drug Metab Pharmacokinet. 2013;38:63–7. doi: 10.1007/s13318-012-0103-z.  [DOI](https://doi.org/10.1007/s13318-012-0103-z) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22945461/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Drug%20Metab%20Pharmacokinet&title=Genome-wide%20association%20study%20reveals%20a%20complex%20genetic%20architecture%20underpinning-induced%20CYP3A4%20enzyme%20activity&author=N%20Rahmioglu&volume=38&publication_year=2013&pages=63-7&pmid=22945461&doi=10.1007/s13318-012-0103-z&)

30. Reyataz (Atazanavir) package insert. Bristol-Myers Squibb; Updated 09/2016.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Reyataz%20(Atazanavir)%20package%20insert&)

31. Lehmann JM, McKee DD, Watson MA, Willson TM, Moore JT, Kliewer SA. The human orphan nuclear receptor PXR is activated by compounds that regulate CYP3A4 gene expression and cause drug interactions. J Clin Invest. 1998;102:1016–23. doi: 10.1172/JCI3703.  [DOI](https://doi.org/10.1172/JCI3703) | [PMC free article](/articles/PMC508967/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9727070/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Invest&title=The%20human%20orphan%20nuclear%20receptor%20PXR%20is%20activated%20by%20compounds%20that%20regulate%20CYP3A4%20gene%20expression%20and%20cause%20drug%20interactions&author=JM%20Lehmann&author=DD%20McKee&author=MA%20Watson&author=TM%20Willson&author=JT%20Moore&volume=102&publication_year=1998&pages=1016-23&pmid=9727070&doi=10.1172/JCI3703&)

32. Fukuen S, et al. Identification of the novel splicing variants for the hPXR in human livers. Biochem Biophys Res Commun. 2002;298:433–8. doi: 10.1016/s0006-291x(02)02469-5.  [DOI](https://doi.org/10.1016/s0006-291x(02)02469-5) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12413960/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem%20Biophys%20Res%20Commun&title=Identification%20of%20the%20novel%20splicing%20variants%20for%20the%20hPXR%20in%20human%20livers&author=S%20Fukuen&volume=298&publication_year=2002&pages=433-8&pmid=12413960&doi=10.1016/s0006-291x(02)02469-5&)

33. Button KS, et al. Power failure: why small sample size undermines the reliability of neuroscience. Nat Rev Neurosci. 2013;14:365–76. doi: 10.1038/nrn3475.  [DOI](https://doi.org/10.1038/nrn3475) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23571845/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Neurosci&title=Power%20failure:%20why%20small%20sample%20size%20undermines%20the%20reliability%20of%20neuroscience&author=KS%20Button&volume=14&publication_year=2013&pages=365-76&pmid=23571845&doi=10.1038/nrn3475&)

34. Barkan SE, et al. The Women’s Interagency HIV Study. WIHS Collaborative Study Group. Epidemiology. 1998;9:117–25.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/9504278/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Epidemiology&title=The%20Women%E2%80%99s%20Interagency%20HIV%20Study.%20WIHS%20Collaborative%20Study%20Group&author=SE%20Barkan&volume=9&publication_year=1998&pages=117-25&pmid=9504278&)

35. Bacon MC, et al. The Women’s Interagency HIV Study: an observational cohort brings clinical sciences to the bench. Clin Diagn Lab Immunol. 2005;12:1013–9. doi: 10.1128/CDLI.12.9.1013-1019.2005.  [DOI](https://doi.org/10.1128/CDLI.12.9.1013-1019.2005) | [PMC free article](/articles/PMC1235804/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16148165/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Diagn%20Lab%20Immunol&title=The%20Women%E2%80%99s%20Interagency%20HIV%20Study:%20an%20observational%20cohort%20brings%20clinical%20sciences%20to%20the%20bench&author=MC%20Bacon&volume=12&publication_year=2005&pages=1013-9&pmid=16148165&doi=10.1128/CDLI.12.9.1013-1019.2005&)

36. Gandhi M, et al. Eligibility criteria for HIV clinical trials and generalizability of results: the gap between published reports and study protocols. AIDS. 2005;19:1885–96. doi: 10.1097/01.aids.0000189866.67182.f7.  [DOI](https://doi.org/10.1097/01.aids.0000189866.67182.f7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16227797/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=AIDS&title=Eligibility%20criteria%20for%20HIV%20clinical%20trials%20and%20generalizability%20of%20results:%20the%20gap%20between%20published%20reports%20and%20study%20protocols&author=M%20Gandhi&volume=19&publication_year=2005&pages=1885-96&pmid=16227797&doi=10.1097/01.aids.0000189866.67182.f7&)

37. Price AL, Patterson NJ, Plenge RM, Weinblatt ME, Shadick NA, Reich D. Principal components analysis corrects for stratification in genome-wide association studies. Nat Genet. 2006;38:904–9. doi: 10.1038/ng1847.  [DOI](https://doi.org/10.1038/ng1847) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16862161/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Principal%20components%20analysis%20corrects%20for%20stratification%20in%20genome-wide%20association%20studies&author=AL%20Price&author=NJ%20Patterson&author=RM%20Plenge&author=ME%20Weinblatt&author=NA%20Shadick&volume=38&publication_year=2006&pages=904-9&pmid=16862161&doi=10.1038/ng1847&)

38. Levey AS, et al. A new equation to estimate glomerular filtration rate. Ann Intern Med. 2009;150:604–12. doi: 10.7326/0003-4819-150-9-200905050-00006.  [DOI](https://doi.org/10.7326/0003-4819-150-9-200905050-00006) | [PMC free article](/articles/PMC2763564/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19414839/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Intern%20Med&title=A%20new%20equation%20to%20estimate%20glomerular%20filtration%20rate&author=AS%20Levey&volume=150&publication_year=2009&pages=604-12&pmid=19414839&doi=10.7326/0003-4819-150-9-200905050-00006&)

39. Alonso N, Lucas G, Hysi P. Big data challenges in bone research: genome-wide association studies and next-generation sequencing. Bonekey Rep. 2015;4:635. doi: 10.1038/bonekey.2015.2.  [DOI](https://doi.org/10.1038/bonekey.2015.2) | [PMC free article](/articles/PMC4325556/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25709812/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bonekey%20Rep&title=Big%20data%20challenges%20in%20bone%20research:%20genome-wide%20association%20studies%20and%20next-generation%20sequencing&author=N%20Alonso&author=G%20Lucas&author=P%20Hysi&volume=4&publication_year=2015&pages=635&pmid=25709812&doi=10.1038/bonekey.2015.2&)

40. Delaneau O, Marchini J, Zagury JF. A linear complexity phasing method for thousands of genomes. Nat Methods. 2011;9:179–81. doi: 10.1038/nmeth.1785.  [DOI](https://doi.org/10.1038/nmeth.1785) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22138821/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Methods&title=A%20linear%20complexity%20phasing%20method%20for%20thousands%20of%20genomes&author=O%20Delaneau&author=J%20Marchini&author=JF%20Zagury&volume=9&publication_year=2011&pages=179-81&pmid=22138821&doi=10.1038/nmeth.1785&)

41. Das S, et al. Next-generation genotype imputation service and methods. Nat Genet. 2016;48:1284–7. doi: 10.1038/ng.3656.  [DOI](https://doi.org/10.1038/ng.3656) | [PMC free article](/articles/PMC5157836/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27571263/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Next-generation%20genotype%20imputation%20service%20and%20methods&author=S%20Das&volume=48&publication_year=2016&pages=1284-7&pmid=27571263&doi=10.1038/ng.3656&)

42. Magi R, Morris AP. GWAMA: software for genome-wide association meta-analysis. BMC Bioinformatics. 2010;11:288. doi: 10.1186/1471-2105-11-288.  [DOI](https://doi.org/10.1186/1471-2105-11-288) | [PMC free article](/articles/PMC2893603/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20509871/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Bioinformatics&title=GWAMA:%20software%20for%20genome-wide%20association%20meta-analysis&author=R%20Magi&author=AP%20Morris&volume=11&publication_year=2010&pages=288&pmid=20509871&doi=10.1186/1471-2105-11-288&)

43. Devlin B, Roeder K. Genomic control for association studies. Biometrics. 1999;55:997–1004. doi: 10.1111/j.0006-341x.1999.00997.x.  [DOI](https://doi.org/10.1111/j.0006-341x.1999.00997.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11315092/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biometrics&title=Genomic%20control%20for%20association%20studies&author=B%20Devlin&author=K%20Roeder&volume=55&publication_year=1999&pages=997-1004&pmid=11315092&doi=10.1111/j.0006-341x.1999.00997.x&)

44. Dudbridge F, Gusnanto A. Estimation of significance thresholds for genomewide association scans. Genet Epidemiol. 2008;32:227–34. doi: 10.1002/gepi.20297.  [DOI](https://doi.org/10.1002/gepi.20297) | [PMC free article](/articles/PMC2573032/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18300295/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genet%20Epidemiol&title=Estimation%20of%20significance%20thresholds%20for%20genomewide%20association%20scans&author=F%20Dudbridge&author=A%20Gusnanto&volume=32&publication_year=2008&pages=227-34&pmid=18300295&doi=10.1002/gepi.20297&)

45. Chelala C, Khan A, Lemoine NR. SNPnexus: a web database for functional annotation of newly discovered and public domain single nucleotide polymorphisms. Bioinformatics. 2009;25:655–61. doi: 10.1093/bioinformatics/btn653.  [DOI](https://doi.org/10.1093/bioinformatics/btn653) | [PMC free article](/articles/PMC2647830/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19098027/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=SNPnexus:%20a%20web%20database%20for%20functional%20annotation%20of%20newly%20discovered%20and%20public%20domain%20single%20nucleotide%20polymorphisms&author=C%20Chelala&author=A%20Khan&author=NR%20Lemoine&volume=25&publication_year=2009&pages=655-61&pmid=19098027&doi=10.1093/bioinformatics/btn653&)

46. Dayem Ullah AZ, Lemoine NR, Chelala C. SNPnexus: a web server for functional annotation of novel and publicly known genetic variants (2012 update) Nucleic Acids Res. 2012;40:W65–70. doi: 10.1093/nar/gks364.  [DOI](https://doi.org/10.1093/nar/gks364) | [PMC free article](/articles/PMC3394262/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22544707/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=SNPnexus:%20a%20web%20server%20for%20functional%20annotation%20of%20novel%20and%20publicly%20known%20genetic%20variants%20(2012%20update)&author=AZ%20Dayem%20Ullah&author=NR%20Lemoine&author=C%20Chelala&volume=40&publication_year=2012&pages=W65-70&pmid=22544707&doi=10.1093/nar/gks364&)

47. Dayem Ullah AZ, Lemoine NR, Chelala C. A practical guide for the functional annotation of genetic variations using SNPnexus. Brief Bioinform. 2013;14:437–47. doi: 10.1093/bib/bbt004.  [DOI](https://doi.org/10.1093/bib/bbt004) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23395730/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Brief%20Bioinform&title=A%20practical%20guide%20for%20the%20functional%20annotation%20of%20genetic%20variations%20using%20SNPnexus&author=AZ%20Dayem%20Ullah&author=NR%20Lemoine&author=C%20Chelala&volume=14&publication_year=2013&pages=437-47&pmid=23395730&doi=10.1093/bib/bbt004&)

48. Whirl-Carrillo M, et al. Pharmacogenomics knowledge for personalized medicine. Clin Pharmacol Ther. 2012;92:414–7. doi: 10.1038/clpt.2012.96.  [DOI](https://doi.org/10.1038/clpt.2012.96) | [PMC free article](/articles/PMC3660037/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22992668/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Pharmacogenomics%20knowledge%20for%20personalized%20medicine&author=M%20Whirl-Carrillo&volume=92&publication_year=2012&pages=414-7&pmid=22992668&doi=10.1038/clpt.2012.96&)

49. Pruim RJ, et al. LocusZoom: regional visualization of genome-wide association scan results. Bioinformatics. 2010;26:2336–7. doi: 10.1093/bioinformatics/btq419.  [DOI](https://doi.org/10.1093/bioinformatics/btq419) | [PMC free article](/articles/PMC2935401/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20634204/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=LocusZoom:%20regional%20visualization%20of%20genome-wide%20association%20scan%20results&author=RJ%20Pruim&volume=26&publication_year=2010&pages=2336-7&pmid=20634204&doi=10.1093/bioinformatics/btq419&)

50. International HapMap C. The International HapMap Project. Nature. 2003;426:789–96. doi: 10.1038/nature02168.  [DOI](https://doi.org/10.1038/nature02168) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14685227/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=The%20International%20HapMap%20Project&volume=426&publication_year=2003&pages=789-96&pmid=14685227&doi=10.1038/nature02168&)
