# Dopamine gene variants in opioid addiction: comparison of dependent patients, nondependent users and healthy controls

## Metadata
**Authors:** Matthew Randesi, Wim van den Brink, Orna Levran, Vadim Yuferov, Peter Blanken, Jan M van Ree, Jurg Ott, Mary Jeanne Kreek
**Journal:** Pharmacogenomics
**Date:** 2017 Dec 6
**DOI:** [10.2217/pgs-2017-0134](https://doi.org/10.2217/pgs-2017-0134)
**PMID:** 29210332
**PMCID:** PMC6826863
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6826863/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC6826863/pdf/pgs-19-95.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC6826863/pdf/pgs-19-95.pdf)

## Abstract

**Aim:** 
To determine whether specific dopaminergic system gene variants are associated with opioid dependence.

**Patients & methods:** 
Subjects included 153 healthy controls, 163 opioid exposed, but not dependent and 281 opioid dependent. Genotypes of 90 variants in 13 genes were examined.

**Results:** 
The most significant results were obtained for DA β-hydroxylase variants, rs2073837 and rs1611131, which were associated with protection from addiction (q = 0.0172, 0.0415, respectively) and the functional TH variant, rs2070762, was associated with more risk (q = 0.0387). The three variants also showed a combined effect that remained significant after correction for multiple testing (pfinal = 0.0039).

**Conclusion:** 
These data offer support that dopaminergic gene variants have a role in opioid dependence and warrant further study.

Keywords: : DBH, drug addiction, TH

### Aim:

To determine whether specific dopaminergic system gene variants are associated with opioid dependence.

### Patients & methods:

Subjects included 153 healthy controls, 163 opioid exposed, but not dependent and 281 opioid dependent. Genotypes of 90 variants in 13 genes were examined.

### Results:

The most significant results were obtained for *DA β-hydroxylase* variants, rs2073837 and rs1611131, which were associated with protection from addiction (q = 0.0172, 0.0415, respectively) and the functional *TH* variant, rs2070762, was associated with more risk (q = 0.0387). The three variants also showed a combined effect that remained significant after correction for multiple testing (p_final_ = 0.0039).

### Conclusion:

These data offer support that dopaminergic gene variants have a role in opioid dependence and warrant further study.

## Materials & methods

### Subjects

Three groups of subjects were ascertained in The Netherlands, (n = 795; 30% females), as previously reported [[15–17](#B15)] ([Table 1](#T1)A). Briefly;

### Table 1. . Subject characteristics.

| Treatment group | Recruited | Caucasian by self-report | >70% Caucasian by AIMs | Excluded (DNA low quality) | Included in analysis | Mean age ± SD | Male (%) |
| --- | --- | --- | --- | --- | --- | --- | --- |
| HC | 197 | 168 | 158 | 5 | 153 | 39.0 ± 10.4 | 56 |
|   |  |  |  |  |  |  |  |
| NOD | 198 | 171 | 166 | 3 | 163 | 40.1 ± 9.0 | 65 |
|   |  |  |  |  |  |  |  |
| OD | 400 | 289 | 285 | 4 | 281 | 43.4 ± 8.3 | 75 |

Table 1 Caption: AIM: Ancestry informative marker; HC: Healthy control; NOD: Not-opioid dependent; OD: Opioid dependent; SD: Standard deviation.

All participants were at least 25 years of age.

The Central Committee on Research Involving Human Subjects in The Netherlands (protocol number P04.0156C) approved the study of heroin-assisted and methadone maintenance treatments and the human molecular genetics study for all study groups. The genetics study was also approved by The Rockefeller University's Institutional Review Board. All subjects signed informed consent for the genetics research.

### Socio-demographic & drug use assessment

Trained investigators performed extensive interviews of all subjects. Standard questionnaires were used for collection of age, gender and country of origin information. The DSM-IV was used for diagnosis of opioid dependence.

In addition, subjects were administered the Kreek–McHugh–Schluger–Kellogg (KMSK) scale [[18](#B18)], a rapid and quantitative instrument to assess a subject's exposure to opioids, cocaine, alcohol and nicotine. The KMSK scale assesses the frequency, amount and duration of exposure to each substance during a person's period of greatest use (lifetime score). In prior studies, the results of KMSK assessments were evaluated using receiver operator characteristics analysis for the optimal cut-point score for alcohol, cocaine and opiate dependence/addiction diagnoses using an ethnically diverse population [[18](#B18)].

### Genotyping

Genomic DNA was isolated from the blood specimens of all subjects received from The Netherlands using standard techniques. The DNA was genotyped for 118 SNPs in 13 dopaminergic system genes ([Table 2](#T2) & [Supplementary Table 1](http:///doi/suppl/10.2217/pgs-2017-0134/suppl_file/pgs-19-95-s1.docx)) using a 1536-plex Illumina Golden Gate Custom Panel (GS0013101-OPA; Illumina, CA, USA), which is a modification of the ‘addiction array’ that has been previously described [[10](#B10),[19](#B19)]. Data analysis was performed with BeadStudio v2.3.43 software (Illumina). Genotype data were visually inspected and filtered to include only SNPs with call rates >90%, a well-defined cluster separation and minor allele frequency (MAF) >0.05.

### Table 2. . Dopaminergic system genes.

| Symbol | Name | SNPs |
| --- | --- | --- |
| ANKK1 | Ankyrin repeat and kinase domain containing 1 | 4 |
|   |  |  |
| COMT | Catechol-O-methyltransferase | 13 |
|   |  |  |
| CSNK1E | Casein kinase 1, epsilon | 9 |
|   |  |  |
| DBH | Dopamine β-hydroxylase | 21 |
|   |  |  |
| DDC | Dopa decarboxylase | 17 |
|   |  |  |
| DRD1 | Dopamine receptor D1 | 3 |
|   |  |  |
| DRD2 | Dopamine receptor D2 | 16 |
|   |  |  |
| DRD3 | Dopamine receptor D3 | 14 |
|   |  |  |
| DRD4 | Dopamine receptor D4 | 3 |
|   |  |  |
| DRD5 | Dopamine receptor D5 | 3 |
|   |  |  |
| PPP1R1B | Protein phosphatase 1, regulatory (inhibitor) subunit 1B (DARPP-32) | 3 |
|   |  |  |
| SLC6A3 | Solute carrier family 6 (neurotransmitter transporter), member 3 (DAT) | 10 |
|   |  |  |
| TH | Tyrosine hydroxylase | 2 |
### Assessment of percentage of European ancestry

Ethnicity was initially assigned based on self-reported family origin data with 628 self-identified Caucasian subjects as previously reported [[3](#B3)]. Based on 155 ancestry informative markers, the fraction of genetic affiliation of an individual in each of seven clusters was calculated with *STRUCTURE* v2.2 [[20](#B20)]. Each subject was anchored against 1051 samples from 51 worldwide populations represented in the Human Genome Diversity Cell Line Panel, as described [[21](#B21)]. For the current study, the inclusion criteria was set to 70% or greater European ancestry contribution estimate to minimize population stratification.

### Statistical analysis

We followed the recommendation to test for deviations from Hardy–Weinberg equilibrium (HWE) in control individuals [[22](#B22)]. Using Plink v1.9 [[23](#B23)] we computed p-values for deviation from HWE with mid-p adjustment [[24](#B24)] and obtained Bonferroni-corrected p-values for the 90 SNPs as *pBon* = 1 − (1 − p)^90^. The smallest corrected value was *pBon* = 0.1314. The fact that genotype frequencies of controls conform to HWE provides a surrogate of genetic association studies quality in terms of design and conduct [[22](#B22)]; thus, the clear absence of significant deviations from HWE serves as our control for population stratification. Pairwise linkage disequilibrium (LD) was estimated using Haploview v4.2 [[25](#B25)].

Initially, a standard genetic association test was performed for comparisons of HC versus NOD and NOD versus OD. If no significant differences are found for one or both comparisons, the compared groups can be combined to increase power. Two types of analyses for case/control association were performed for each SNP separately, and combination effect of multiple SNPs.

Analysis (1): individual SNP association analysis was performed using the maxstat program [[26](#B26)]. For each of the 90 SNPs, the basic test statistic was the maximum of chi-squares, recessive and dominant allele action. Such maximum tests have been shown to have superior power [[17](#B17)]. Based on 40,000 permutation replicates, an empirical significance level was computed, p_0_ = nominal p-value for the given SNP. To control for the false discovery rate, a q-value was calculated for each p-value [[27](#B27),[28](#B28)].

Analysis (2): the sumstat program [[26](#B26)] was used to look for any combined effect from more than one SNP. SNPs were ranked by their resulting test statistics and sequential sums were computed, such that s_2_ = sum of largest two test statistics, s_3_ = sum of largest three test statistics and so on. Each sum had an associated significance level, p_sum_. Generally, these sum-specific significance levels tend to drop as more and more test statistics are added to the sums, and after a minimum significance level, p_min_, they tend to increase again. The number of test statistics in the sum furnishing p_min_ is regarded as the number of SNPs potentially contributing to disease in a combined manner. To allow for testing of multiple sums, p_min_ is taken as a new test statistic whose significance level, p_final_, is corrected for testing multiple SNPs and sums. If a single SNPs p-value is less than the p_final_ (p < p_final_) there is evidently no validity in considering sums of test statistics as the best single SNP is more significant than any sum.

## Results

### Sample characteristics

Subjects with <70% estimated proportion of European ancestry based on 155 ancestry informative markers were excluded from the analyses. Twelve subjects were removed due to the low quantity and/or poor quality of DNA. The remaining 597 subjects were included in the association analyses ([Table 1](#T1)A).

Of the 118 SNPs genotyped, 21 SNPs were excluded based on MAF <0.05 and seven SNPs were excluded due to call rate <90%. Genotypes of 90 SNPs from 13 genes were analyzed for association with nondependent opioid use (NOD) and opioid dependence (OD; [Table 2](#T2) & [Supplementary Table 1](http:///doi/suppl/10.2217/pgs-2017-0134/suppl_file/pgs-19-95-s1.docx)). None of the SNPs significantly violated HWE in the control group (HC). Among the OD subjects, there was a strong deviation from HWE for *DBH* SNP rs2073837 (more CT genotypes and fewer CC genotypes than expected) but this deviation may be interpreted as evidence for association. Further, LD analysis of all SNPs, in the HC sample, revealed 21 SNP pairs that were in strong LD, r^2^ > 0.8 ([Supplementary Figure 1](http:///doi/suppl/10.2217/pgs-2017-0134/suppl_file/pgs-19-95-s1.docx)). LD analysis of the *DBH* gene ([Figure 1](#F0001)) shows there are three main LD blocks. The third LD block includes SNPs rs1611131 and rs2073837, which are in strong LD (r^2^ = 0.85).

### Figure 1. . Pairwise linkage dsequilibrium for DA β-hydroxylase.

![Figure 1. ](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2f78/6826863/4059dfcb9e38/pgs-19-95-g1.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=6826863_pgs-19-95-g1.jpg)

The pairwise correlation between single nucleotide polymorphisms measured in D’ (red) and r2 (black). The values shown are (×100) in each box. The color scheme indicates the magnitude of the value where the darker the color indicates the greater linkage dsequilibrium. When the value is equal to 1.0 the box is empty.

### Association analysis testing each SNP individually (1)

#### Comparison of HC versus NOD

Basic association analysis testing for each of the SNPs one at a time found no significant difference (after correction for multiple testing) between HC versus NOD (see [Supplementary Table 4](http:///doi/suppl/10.2217/pgs-2017-0134/suppl_file/pgs-19-95-s1.docx)). In an earlier report with a separate set of genes, significant differences were found between HC versus NOD and therefore they were not combined [[3](#B3)]. However, for the current study in the absence of significant differences, we combined the HC and NOD groups for comparison to the group of OD.

#### Comparison of NOD versus OD

Individual SNP analysis for the comparison of NOD versus OD ([Table 3](#T3)A) showed three SNPs with nominally significant association using a recessive model of inheritance, *DBH* SNPs rs2073837 and rs1611131 and *TH* SNP, rs2070762 (p_0_ = 0.0009, p_0_ = 0.0039 and p_0_ = 0.0062, respectively). The signal from rs2073837 remained significant after correction (q = 0.0474; OR = 0.26; 95% CI: 0.11–0.58).

| SNP | Gene | Chr | Position | Alleles | Location | MAF | Test | Stat | p0 | q± | OR (95% CI) |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| NOD versus OD |  |  |  |  |  |  |  |  |  |  |  |
|   |  |  |  |  |  |  |  |  |  |  |  |
| rs2073837† | DBH | 9 | 133657805 | C/T | Intron | 0.3 | Rec | 11.88 | 0.0009 | 0.0474 | 0.26 (0.11–0.58) |
|   |  |  |  |  |  |  |  |  |  |  |  |
| rs1611131† | DBH | 9 | 133657064 | A/G | Intron | 0.29 | Rec | 9.35 | 0.0039 | 0.0978 | 0.29 (0.13–0.63) |
|   |  |  |  |  |  |  |  |  |  |  |  |
| rs2070762 | TH | 11 | 2165104 | T/C | Intron | 0.49 | Rec | 9.25 | 0.0062 | 0.1038 | 2.07 (1.27–3.42) |
|   |  |  |  |  |  |  |  |  |  |  |  |
| HC versus OD |  |  |  |  |  |  |  |  |  |  |  |
|   |  |  |  |  |  |  |  |  |  |  |  |
| rs2073837† | DBH | 9 | 133657805 | C/T | Intron | 0.3 | Rec | 12.91 | 0.001 | 0.0419 | 0.24 (0.11–0.54) |
|   |  |  |  |  |  |  |  |  |  |  |  |
| rs1611131† | DBH | 9 | 133657064 | A/G | Intron | 0.29 | Rec | 10.27 | 0.0031 | 0.1866 | 0.29 (0.13–0.63) |
|   |  |  |  |  |  |  |  |  |  |  |  |
| rs2070762 | TH | 11 | 2165104 | T/C | Intron | 0.49 | Rec | 9.59 | 0.0044 | 0.2437 | 2.08 (1.27–3.42) |
|   |  |  |  |  |  |  |  |  |  |  |  |
| (HC + NOD) versus OD |  |  |  |  |  |  |  |  |  |  |  |
|   |  |  |  |  |  |  |  |  |  |  |  |
| rs2073837† | DBH | 9 | 133657805 | C/T | Intron | 0.3 | Rec | 16.96 | 0.0001 | 0.0172 | 0.25 (0.12–0.52) |
|   |  |  |  |  |  |  |  |  |  |  |  |
| rs2070762 | TH | 11 | 2165104 | T/C | Intron | 0.49 | Rec | 14.28 | 0.0004 | 0.0387 | 2.12 (1.43–3.14) |
|   |  |  |  |  |  |  |  |  |  |  |  |
| rs1611131† | DBH | 9 | 133657064 | A/G | Intron | 0.29 | Rec | 13.51 | 0.0007 | 0.0415 | 0.29 (0.14–0.59) |

Table Caption: FDR adjustment assumed 90 SNPs and 3 group comparisons. q±, false discovery rate (FDR) associated p-value, with FDR set to 0.05. †SNPs in strong linkage disequilibrium (r2 = 0.85). Chr: Chromosome; HC: Healthy control; MAF: Minor allele frequency; NOD: Nondependent opioid user; OD: Opioid dependent; OR: Odds ratio; p0: Nominal p-value; ; Rec: Recessive; Stat: Test statistic.

#### Comparison of HC versus OD

Individual SNP analysis for the comparison of HC versus OD ([Table 3](#T3)B) showed the same three SNPs with nominally significant association using a recessive model of inheritance, *DBH* SNPs rs2073837 and rs1611131 and *TH* SNP, rs2070762 (p_0_ = 0.0010, p_0_ = 0.0031 and p_0_ = 0.0044, respectively). Again, the signal from rs2073837 remained significant after correction (q = 0.0419; OR = 0.24; 95% CI: 0.11–0.54).

#### Comparison of (HC + NOD) versus OD

Since we found now significant difference (after multiple testing correction) when we compared HC versus NOD we combined the groups and compared the larger group to OD. Again, the same three SNPs, rs2073837 (*DBH)*, rs2070762 (*TH*) and rs1611131 (*DBH)*, were found to be significant using a recessive model of inheritance ([Table 3](#T3)C). After correction, all three signals remained significant (q = 0.0172, OR = 0.25; 95% CI: 0.12–0.52; q = 0.0387, OR = 2.12; 95% CI: 1.43–3.14; and q = 0.0415, OR = 0.29; 95% CI: 0.14–0.59, respectively).

### Association analysis testing the combined action of multiple SNPs (2)

Forming sums of the test statistics revealed that there is a combined effect of the top three SNPs (rs2073837, rs1611131 and rs2070762) for the comparisons, HC versus OD and (HC + NOD) versus OD.

#### Comparison of HC versus OD

The sum of the largest three test statistics furnished a p_min_ = 0.0319 ([Table 4](#T4)A). Taking p_min_ as a new test statistic and calculating p_final_ = 0.0595, which is corrected for testing multiple SNPs and sums. Since p_final_>p-value for the best individual SNP result (p = 0.0486), in this case, calculating sums did not seem to contribute more that the best SNP does by itself.

| Rank | SNP | Gene | Chr | Position | Alleles | Location | Stat | Sum | p0 | pStat | pSum |   |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| HC versus OD |  |  |  |  |  |  |  |  |  |  |  |  |
|   |  |  |  |  |  |  |  |  |  |  |  |  |
| 1 | rs2073837 | DBH | 9 | 133657805 | C/T | Intron | 12.91 | 12.91 | 0.001 | 0.0486 | 0.0486 |   |
|   |  |  |  |  |  |  |  |  |  |  |  |  |
| 2 | rs1611131 | DBH | 9 | 133657064 | A/G | Intron | 10.27 | 23.18 | 0.0031 | 0.1866 | 0.0406 |   |
|   |  |  |  |  |  |  |  |  |  |  |  |  |
| 3 | rs2070762 | TH | 11 | 2165104 | T/C | Intron | 9.59 | 32.77 | 0.0044 | 0.2437 | 0.0319 | (pmin) |
|   |  |  |  |  |  |  |  |  |  |  |  |  |
|   |   |   |   |   |   |   |   |   |   |   |   | pfinal = 0.0595 |
|   |  |  |  |  |  |  |  |  |  |  |  |  |
| (HC + NOD) versus OD |  |  |  |  |  |  |  |  |  |  |  |  |
|   |  |  |  |  |  |  |  |  |  |  |  |  |
| 1 | rs2073837 | DBH | 9 | 133657805 | C/T | Intron | 16.96 | 16.96 | 0.0001 | 0.0048 | 0.0059 |   |
|   |  |  |  |  |  |  |  |  |  |  |  |  |
| 2 | rs2070762 | TH | 11 | 2165104 | T/C | Intron | 14.28 | 31.24 | 0.0004 | 0.0234 | 0.0027 |   |
|   |  |  |  |  |  |  |  |  |  |  |  |  |
| 3 | rs1611131 | DBH | 9 | 133657064 | A/G | Intron | 13.51 | 44.75 | 0.0007 | 0.0346 | 0.0016 | (pmin) |
|   |  |  |  |  |  |  |  |  |  |  |  |  |
|   |   |   |   |   |   |   |   |   |   |   |   | pfinal = 0.0039 |

Table Caption: Chr: Chromosome; HC: Healthy control; NOD: Nondependent opioid user; OD: Opioid dependent; p0: Nominal p-value; pfinal: pmin corrected for testing multiple SNP and sum; pmin: Smallest psum for a given comparison; psum: Sum-specific significance level; pstat: p-value corrected for testing multiple SNPs and two models of inheritance per SNP; Stat: Test statistic; Sum: Sum of largest n test statistics.

#### Comparison of (HC + NOD) versus OD

The sum of the largest three test statistics furnished a p_min_ = 0.0016 ([Table 4](#T4)B), which is highly significant with a p_final_ = 0.0039, corrected for testing multiple SNPs and sums. Since p_final_<p-value for the best individual SNP result (p = 0.0048), sum statistics suggest that these three SNPs (rs2073837, rs2070762 and rs1611131) appear to contribute jointly to disease association. As noted earlier, the two *DBH* SNPs, rs2073837 and rs1611131 were found to be in strong LD; therefore, the signal detected is most likely coming from just one of the SNPs.

Genotype frequencies of *DBH* SNPs, rs2073837 and rs1611131, and the *TH* SNP rs2070762 for the treatment groups (HC, NOD and OD) are shown in [Supplementary Table 2](http:///doi/suppl/10.2217/pgs-2017-0134/suppl_file/pgs-19-95-s1.docx).

## Discussion

In this study, we have tested 90 SNPs in 13 dopaminergic system genes for association with nondependence opioid use and heroin dependence. We found two *DBH* SNPs in strong LD, rs2073837 and rs1611131, to be associated with protection from OD, and one *TH* SNP, rs2070762, to be associated with a risk or vulnerability to develop OD. The three SNPs have not been previously reported to be associated with OD. The *DBH* variant rs1611131 has been reported to be linked to dependence symptoms of cocaine users, with carriers of the G allele having more dependence symptoms [[29](#B29)]. This is in contrast to the current report which found that persons with the variant allele for rs1611131 should be protected from OD.

However, comparing the two studies is difficult since the earlier report was looking at DA genes effect on dependence symptoms among cocaine users and not opioid users as in the current study.

*DBH* has 12 exons and spans 23 kb and there are two known splice variants [[30](#B30)]. Since the *DBH* SNPs are in intronic regions they may be involved in splicing processes with formation of new *DBH* variants or to be targets for miRNA activity, possibly leading to the production of an enzyme with reduced or no activity.

DBH converts DA to norepinephrine, thereby playing a direct role in determining the ratio of DA to norepinephrine in noradrenergic neurons ([Figure 2](#F0002)). It is known that plasma DBH activity varies widely between individuals [[31](#B31)]. Another SNP in *DBH*, rs1611115 (1021C/T), that was not found to be significant in the current study, is functional and located in the 5′ flanking region of the gene, has been shown to account for ‘35–50%’ of levels of plasma DBH activity in populations of European origin [[32](#B32)]. Several other functional polymorphisms in the *DBH* gene such as rs2519152 and rs6271 have been reported by different investigators. Association between *DBH* SNPs and plasma DBH activity, as well as cocaine-induced paranoia were reported [[31](#B31),[33](#B33)].

### Figure 2. . Catecholamine synthesis pathway.

![Figure 2. ](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2f78/6826863/3fd511a89c4b/pgs-19-95-g2.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=6826863_pgs-19-95-g2.jpg)

TH is the rate-limiting enzyme in the synthesis of catecholamines; including DA, noradrenaline and adrenaline. TH converts tyrosine to L-DOPA that is in turn converted to DA by DOPA decarboxylase. Several variants in *TH* have been tested for association with affective disorders, such as bipolar disorder and schizophrenia. A polymorphic four base repeat in *TH*, that was not included in this study, was found associated with schizophrenia in Japanese females, but not males [[34](#B34)]. The functional intronic *TH* SNP rs2070762, identified in this study, was previously found to confer risk of hypertension in a northern Chinese Han population [[35](#B35)]. SNP rs2070762 was also found nominally associated with cocaine dependence in a Spanish sample [[36](#B36)]. The current study is the first to report an association of *TH* rs2070762-C allele with the vulnerability to develop opiate dependence.

As stated above, our findings indicate that persons having the *DBH* variant alleles for SNPs rs2073837 and rs1611131 would be more ‘protected’ from the vulnerability of developing OD than someone having the wild-type alleles. Further, the findings indicate persons with the variant allele for the *TH* SNP rs2070762 would have more risk or be more vulnerable to the development of OD. Mechanistically, one can envision how this might occur, tyrosine hydroxylase converts L-tyrosine into L-DOPA, precursor to DA, a variant *TH* would result in a less active enzyme and in turn less free DA therefore more vulnerable to addiction. By the same reasoning, since DBH catalyzes the conversion of DA to noradrenaline, a variant *DBH* would result in higher levels of free DA therefore less vulnerable to development of addiction. Of course this scenario can only occur when two assumptions are made; the specific gene variant is functional and results in reduced levels of the gene product and that increasing free DA levels would result in a decrease in the vulnerability to develop an addiction.

Although we found no variants with significant differences that survived correction for multiple testing in their genotype frequencies between HC and NOD subjects with these genes and their variants, recently we reported differences for several variants in genes of the opioid system [[3](#B3)].

There are some important limitations for the current study that should be considered when interpreting the findings. Drug use assessment using the KMSK ([Supplementary Table 3](http:///doi/suppl/10.2217/pgs-2017-0134/suppl_file/pgs-19-95-s1.docx)) revealed that among the NOD subjects 33% reported heavy cocaine use, 6% reported regular opioid use and 68% reported alcohol use, all of which would be classified as dependence by KMSK criteria. In addition, the KMSK confirmed that the HC subjects were not using cocaine or heroin, but 31% did report excess alcohol use consistent with dependence by the KMSK criteria. The KMSK also indicated that 76% of the OD subjects were abusing cocaine and 70% were abusing alcohol. Regrettably, the numbers of subjects in the groups were too small to allow statistical analysis for concomitant cocaine, alcohol or cocaine with alcohol. It is of interest that the genes identified in the current study are well known to be coding for enzymes involved in reward pathways for cocaine and alcohol. The high percentage of subjects with polydrug use could very well be confounding the results presented.

The deviation from HWE in the OD sample for *DBH* SNP rs2073837 can be interpreted as evidence for association. However, one must consider the possibility that this deviation is the result of population stratification. We tested for deviations from HWE in control individuals, where no deviation was found. The fact that genotype frequencies of controls conform to HWE provides a surrogate of genetic association studies quality in terms of design and conduct [[22](#B22)]; thus, the clear absence of significant deviations from HWE served as our control for population stratification.

Our current study provides the first evidence that *DBH* SNPs, rs2073837 and rs1611131 and *TH* SNP rs2070762 play a role in opioid dependence. The relatively small number of subjects in the current study may have limited detection of additional significant differences between groups. Further studies with greater statistical power are warranted to corroborate the results and to assess the clinical significance of the findings.

## Supplementary Material

## Acknowledgements

We would like to thank and acknowledge the contribution of P-H Shen, D Goldman and C Zhao for genotyping and structure analysis.

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Kendler KS, Jacobson KC, Prescott CA, Neale MC. Specificity of genetic and environmental risk factors for use and abuse/dependence of cannabis, cocaine, hallucinogens, sedatives, stimulants, and opiates in male twins. Am. J. Psychiatry. 2003;160(4):687–695. doi: 10.1176/appi.ajp.160.4.687.  [DOI](https://doi.org/10.1176/appi.ajp.160.4.687) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12668357/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Psychiatry&title=Specificity%20of%20genetic%20and%20environmental%20risk%20factors%20for%20use%20and%20abuse/dependence%20of%20cannabis,%20cocaine,%20hallucinogens,%20sedatives,%20stimulants,%20and%20opiates%20in%20male%20twins&author=KS%20Kendler&author=KC%20Jacobson&author=CA%20Prescott&author=MC%20Neale&volume=160&issue=4&publication_year=2003&pages=687-695&pmid=12668357&doi=10.1176/appi.ajp.160.4.687&)

2. Tsuang MT, Lyons MJ, Eisen SA, et al. Genetic influences on DSM-III-R drug abuse and dependence: a study of 3372 twin pairs. Am. J. Med. Genet. 1996;67(5):473–477. doi: 10.1002/(SICI)1096-8628(19960920)67:5<473::AID-AJMG6>3.0.CO;2-L.  [DOI](https://doi.org/10.1002/(SICI)1096-8628(19960920)67:5<473::AID-AJMG6>3.0.CO;2-L) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8886164/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Med.%20Genet.&title=Genetic%20influences%20on%20DSM-III-R%20drug%20abuse%20and%20dependence:%20a%20study%20of%203372%20twin%20pairs&author=MT%20Tsuang&author=MJ%20Lyons&author=SA%20Eisen&volume=67&issue=5&publication_year=1996&pages=473-477&pmid=8886164&doi=10.1002/(SICI)1096-8628(19960920)67:5<473::AID-AJMG6>3.0.CO;2-L&)

3. Randesi M, van den Brink W, Levran O, et al. Variants of opioid system genes are associated with non-dependent opioid use and heroin dependence. Drug Alcohol Depend. 2016;168:164–169. doi: 10.1016/j.drugalcdep.2016.08.634.  [DOI](https://doi.org/10.1016/j.drugalcdep.2016.08.634) | [PMC free article](/articles/PMC6842569/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27664554/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Alcohol%20Depend.&title=Variants%20of%20opioid%20system%20genes%20are%20associated%20with%20non-dependent%20opioid%20use%20and%20heroin%20dependence&author=M%20Randesi&author=W%20van%20den%20Brink&author=O%20Levran&volume=168&publication_year=2016&pages=164-169&pmid=27664554&doi=10.1016/j.drugalcdep.2016.08.634&)

4. Xi ZX, Stein EA. GABAergic mechanisms of opiate reinforcement. Alcohol Alcohol. 2002;37(5):485–494. doi: 10.1093/alcalc/37.5.485.  [DOI](https://doi.org/10.1093/alcalc/37.5.485) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12217944/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Alcohol%20Alcohol.&title=GABAergic%20mechanisms%20of%20opiate%20reinforcement&author=ZX%20Xi&author=EA%20Stein&volume=37&issue=5&publication_year=2002&pages=485-494&pmid=12217944&doi=10.1093/alcalc/37.5.485&)

5. Clarke TK, Weiss AR, Ferarro TN, et al. The dopamine receptor D2 (DRD2) SNP rs1076560 is associated with opioid addiction. Ann. Hum. Genet. 2014;78(1):33–39. doi: 10.1111/ahg.12046.  [DOI](https://doi.org/10.1111/ahg.12046) | [PMC free article](/articles/PMC4013426/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24359476/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann.%20Hum.%20Genet.&title=The%20dopamine%20receptor%20D2%20(DRD2)%20SNP%20rs1076560%20is%20associated%20with%20opioid%20addiction&author=TK%20Clarke&author=AR%20Weiss&author=TN%20Ferarro&volume=78&issue=1&publication_year=2014&pages=33-39&pmid=24359476&doi=10.1111/ahg.12046&)

6. Levran O, Peles E, Randesi M, et al. Dopaminergic pathway polymorphisms and heroin addiction: further support for association of CSNK1E variants. Pharmacogenomics. 2014;15(16):2001–2009. doi: 10.2217/pgs.14.145.  [DOI](https://doi.org/10.2217/pgs.14.145) | [PMC free article](/articles/PMC4288976/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25521358/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Dopaminergic%20pathway%20polymorphisms%20and%20heroin%20addiction:%20further%20support%20for%20association%20of%20CSNK1E%20variants&author=O%20Levran&author=E%20Peles&author=M%20Randesi&volume=15&issue=16&publication_year=2014&pages=2001-2009&pmid=25521358&doi=10.2217/pgs.14.145&)

7. Levran O, Randesi M, Da Rosa JC, et al. Overlapping dopaminergic pathway genetic susceptibility to heroin and cocaine addictions in African Americans. Ann. Hum. Genet. 2015;79(3):188–198. doi: 10.1111/ahg.12104.  [DOI](https://doi.org/10.1111/ahg.12104) | [PMC free article](/articles/PMC4399004/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25875614/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann.%20Hum.%20Genet.&title=Overlapping%20dopaminergic%20pathway%20genetic%20susceptibility%20to%20heroin%20and%20cocaine%20addictions%20in%20African%20Americans&author=O%20Levran&author=M%20Randesi&author=JC%20Da%20Rosa&volume=79&issue=3&publication_year=2015&pages=188-198&pmid=25875614&doi=10.1111/ahg.12104&)

8. Nelson EC, Lynskey MT, Heath AC, et al. ANKK1, TTC12 and NCAM1 polymorphisms and heroin dependence: importance of considering drug exposure. JAMA Psychiat. 2013;70(3):325–333. doi: 10.1001/jamapsychiatry.2013.282.  [DOI](https://doi.org/10.1001/jamapsychiatry.2013.282) | [PMC free article](/articles/PMC3789525/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23303482/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA%20Psychiat.&title=ANKK1,%20TTC12%20and%20NCAM1%20polymorphisms%20and%20heroin%20dependence:%20importance%20of%20considering%20drug%20exposure&author=EC%20Nelson&author=MT%20Lynskey&author=AC%20Heath&volume=70&issue=3&publication_year=2013&pages=325-333&pmid=23303482&doi=10.1001/jamapsychiatry.2013.282&)

9. Jacobs MM, Okvist A, Horvath M, et al. Dopamine receptor D1 and postsynaptic density gene variants associate with opiate abuse and striatal expression levels. Mol. Psychiat. 2013;18(11):1205–1210. doi: 10.1038/mp.2012.140.  [DOI](https://doi.org/10.1038/mp.2012.140) | [PMC free article](/articles/PMC3637428/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23044706/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol.%20Psychiat.&title=Dopamine%20receptor%20D1%20and%20postsynaptic%20density%20gene%20variants%20associate%20with%20opiate%20abuse%20and%20striatal%20expression%20levels&author=MM%20Jacobs&author=A%20Okvist&author=M%20Horvath&volume=18&issue=11&publication_year=2013&pages=1205-1210&pmid=23044706&doi=10.1038/mp.2012.140&)

10. Levran O, Londono D, O'Hara K, et al. Genetic susceptibility to heroin addiction: a candidate gene association study. Genes Brain Behav. 2008;7(7):720–729. doi: 10.1111/j.1601-183X.2008.00410.x.  [DOI](https://doi.org/10.1111/j.1601-183X.2008.00410.x) | [PMC free article](/articles/PMC2885890/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18518925/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genes%20Brain%20Behav.&title=Genetic%20susceptibility%20to%20heroin%20addiction:%20a%20candidate%20gene%20association%20study&author=O%20Levran&author=D%20Londono&author=K%20O'Hara&volume=7&issue=7&publication_year=2008&pages=720-729&pmid=18518925&doi=10.1111/j.1601-183X.2008.00410.x&)

11. Xie X, Xu L, Liu H, et al. Positive association between--1021TT genotype of dopamine beta hydroxylase gene and progressive behavior of injection heroin users. Neurosci. Lett. 2013;541:258–262. doi: 10.1016/j.neulet.2013.02.049.  [DOI](https://doi.org/10.1016/j.neulet.2013.02.049) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23510745/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neurosci.%20Lett.&title=Positive%20association%20between--1021TT%20genotype%20of%20dopamine%20beta%20hydroxylase%20gene%20and%20progressive%20behavior%20of%20injection%20heroin%20users&author=X%20Xie&author=L%20Xu&author=H%20Liu&volume=541&publication_year=2013&pages=258-262&pmid=23510745&doi=10.1016/j.neulet.2013.02.049&)

12. Leventhal AM, Lee W, Bergen AW, et al. Nicotine dependence as a moderator of genetic influences on smoking cessation treatment outcome. Drug Alcohol Depend. 2014;138:109–117. doi: 10.1016/j.drugalcdep.2014.02.016.  [DOI](https://doi.org/10.1016/j.drugalcdep.2014.02.016) | [PMC free article](/articles/PMC4095777/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24667010/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Alcohol%20Depend.&title=Nicotine%20dependence%20as%20a%20moderator%20of%20genetic%20influences%20on%20smoking%20cessation%20treatment%20outcome&author=AM%20Leventhal&author=W%20Lee&author=AW%20Bergen&volume=138&publication_year=2014&pages=109-117&pmid=24667010&doi=10.1016/j.drugalcdep.2014.02.016&)

13. Ittiwut R, Listman JB, Ittiwut C, et al. Association between polymorphisms in catechol-O-methyltransferase (COMT) and cocaine-induced paranoia in European–American and African–American populations. Am. J. Med. Genet. B Neuropsychiatr. Genet. 2011;156B(6):651–660. doi: 10.1002/ajmg.b.31205.  [DOI](https://doi.org/10.1002/ajmg.b.31205) | [PMC free article](/articles/PMC3864552/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21656904/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Med.%20Genet.%20B%20Neuropsychiatr.%20Genet.&title=Association%20between%20polymorphisms%20in%20catechol-O-methyltransferase%20(COMT)%20and%20cocaine-induced%20paranoia%20in%20European%E2%80%93American%20and%20African%E2%80%93American%20populations&author=R%20Ittiwut&author=JB%20Listman&author=C%20Ittiwut&volume=156B&issue=6&publication_year=2011&pages=651-660&pmid=21656904&doi=10.1002/ajmg.b.31205&)

14. Shiels MS, Huang HY, Hoffman SC, et al. A community-based study of cigarette smoking behavior in relation to variation in three genes involved in dopamine metabolism: catechol-O-methyltransferase (COMT), dopamine beta-hydroxylase (DBH) and monoamine oxidase-A (MAO-A) Prev. Med. 2008;47(1):116–122. doi: 10.1016/j.ypmed.2008.03.013.  [DOI](https://doi.org/10.1016/j.ypmed.2008.03.013) | [PMC free article](/articles/PMC2577854/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18486967/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Prev.%20Med.&title=A%20community-based%20study%20of%20cigarette%20smoking%20behavior%20in%20relation%20to%20variation%20in%20three%20genes%20involved%20in%20dopamine%20metabolism:%20catechol-O-methyltransferase%20(COMT),%20dopamine%20beta-hydroxylase%20(DBH)%20and%20monoamine%20oxidase-A%20(MAO-A)&author=MS%20Shiels&author=HY%20Huang&author=SC%20Hoffman&volume=47&issue=1&publication_year=2008&pages=116-122&pmid=18486967&doi=10.1016/j.ypmed.2008.03.013&)

15. Zaaijer ER, Bruijel J, Blanken P, et al. Personality as a risk factor for illicit opioid use and a protective factor for illicit opioid dependence. Drug Alcohol Depend. 2014;145:101–105. doi: 10.1016/j.drugalcdep.2014.09.783.  [DOI](https://doi.org/10.1016/j.drugalcdep.2014.09.783) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25454407/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Alcohol%20Depend.&title=Personality%20as%20a%20risk%20factor%20for%20illicit%20opioid%20use%20and%20a%20protective%20factor%20for%20illicit%20opioid%20dependence&author=ER%20Zaaijer&author=J%20Bruijel&author=P%20Blanken&volume=145&publication_year=2014&pages=101-105&pmid=25454407&doi=10.1016/j.drugalcdep.2014.09.783&)

16. Blanken P, van den Brink W, Hendriks VM, et al. Heroin-assisted treatment in The Netherlands: history, findings, and international context. Eur. Neuropsychopharmacol. 2010;20(Suppl. 2):S105–S158. doi: 10.1016/S0924-977X(10)70001-8.  [DOI](https://doi.org/10.1016/S0924-977X(10)70001-8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20362236/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur.%20Neuropsychopharmacol.&title=Heroin-assisted%20treatment%20in%20The%20Netherlands:%20history,%20findings,%20and%20international%20context&author=P%20Blanken&author=W%20van%20den%20Brink&author=VM%20Hendriks&volume=20&issue=Suppl.%202&publication_year=2010&pages=S105-S158&pmid=20362236&doi=10.1016/S0924-977X(10)70001-8&)

17. Zheng G, Freidlin B, Gastwirth JL. Comparison of robust tests for genetic association using case–control studies. IMS Lecture Notes-Monograph Series. 2006;49:253–265.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=IMS%20Lecture%20Notes-Monograph%20Series&title=Comparison%20of%20robust%20tests%20for%20genetic%20association%20using%20case%E2%80%93control%20studies&author=G%20Zheng&author=B%20Freidlin&author=JL%20Gastwirth&volume=49&publication_year=2006&pages=253-265&)

18. Kellogg SH, Mchugh PF, Bell K, et al. The Kreek–McHugh–Schluger–Kellogg scale: a new, rapid method for quantifying substance abuse and its possible applications. Drug Alcohol Depend. 2003;69(2):137–150. doi: 10.1016/s0376-8716(02)00308-3.  [DOI](https://doi.org/10.1016/s0376-8716(02)00308-3) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12609695/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Alcohol%20Depend.&title=The%20Kreek%E2%80%93McHugh%E2%80%93Schluger%E2%80%93Kellogg%20scale:%20a%20new,%20rapid%20method%20for%20quantifying%20substance%20abuse%20and%20its%20possible%20applications&author=SH%20Kellogg&author=PF%20Mchugh&author=K%20Bell&volume=69&issue=2&publication_year=2003&pages=137-150&pmid=12609695&doi=10.1016/s0376-8716(02)00308-3&)

19. Hodgkinson CA, Yuan Q, Xu K, et al. Addictions biology: haplotype-based analysis for 130 candidate genes on a single array. Alcohol Alcohol. 2008;43(5):505–515. doi: 10.1093/alcalc/agn032.  [DOI](https://doi.org/10.1093/alcalc/agn032) | [PMC free article](/articles/PMC2724863/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18477577/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Alcohol%20Alcohol.&title=Addictions%20biology:%20haplotype-based%20analysis%20for%20130%20candidate%20genes%20on%20a%20single%20array&author=CA%20Hodgkinson&author=Q%20Yuan&author=K%20Xu&volume=43&issue=5&publication_year=2008&pages=505-515&pmid=18477577&doi=10.1093/alcalc/agn032&)

20. Pritchard JK, Stephens M, Donnelly P. Inference of population structure using multilocus genotype data. Genetics. 2000;155(2):945–959. doi: 10.1093/genetics/155.2.945.  [DOI](https://doi.org/10.1093/genetics/155.2.945) | [PMC free article](/articles/PMC1461096/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10835412/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genetics&title=Inference%20of%20population%20structure%20using%20multilocus%20genotype%20data&author=JK%20Pritchard&author=M%20Stephens&author=P%20Donnelly&volume=155&issue=2&publication_year=2000&pages=945-959&pmid=10835412&doi=10.1093/genetics/155.2.945&)

21. Ducci F, Roy A, Shen PH, et al. Association of substance use disorders with childhood trauma but not African genetic heritage in an African American cohort. Am. J. Psychiat. 2009;166(9):1031–1040. doi: 10.1176/appi.ajp.2009.08071068.  [DOI](https://doi.org/10.1176/appi.ajp.2009.08071068) | [PMC free article](/articles/PMC2768270/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19605534/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Psychiat.&title=Association%20of%20substance%20use%20disorders%20with%20childhood%20trauma%20but%20not%20African%20genetic%20heritage%20in%20an%20African%20American%20cohort&author=F%20Ducci&author=A%20Roy&author=PH%20Shen&volume=166&issue=9&publication_year=2009&pages=1031-1040&pmid=19605534&doi=10.1176/appi.ajp.2009.08071068&)

22. Zintzaras E. Impact of Hardy–Weinberg equilibrium deviation on allele-based risk effect of genetic association studies and meta-analysis. Eur. J. Epidemiol. 2010;25(8):553–560. doi: 10.1007/s10654-010-9467-z.  [DOI](https://doi.org/10.1007/s10654-010-9467-z) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20526652/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur.%20J.%20Epidemiol.&title=Impact%20of%20Hardy%E2%80%93Weinberg%20equilibrium%20deviation%20on%20allele-based%20risk%20effect%20of%20genetic%20association%20studies%20and%20meta-analysis&author=E%20Zintzaras&volume=25&issue=8&publication_year=2010&pages=553-560&pmid=20526652&doi=10.1007/s10654-010-9467-z&)

23. Purcell S, Neale B, Todd-Brown K, et al. PLINK: a tool set for whole-genome association and population-based linkage analyses. Am. J. Hum. Genet. 2007;81(3):559–575. doi: 10.1086/519795.  [DOI](https://doi.org/10.1086/519795) | [PMC free article](/articles/PMC1950838/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17701901/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Hum.%20Genet.&title=PLINK:%20a%20tool%20set%20for%20whole-genome%20association%20and%20population-based%20linkage%20analyses&author=S%20Purcell&author=B%20Neale&author=K%20Todd-Brown&volume=81&issue=3&publication_year=2007&pages=559-575&pmid=17701901&doi=10.1086/519795&)

24. https://mathdept.iut.ac.ir/sites/mathdept.iut.ac.ir/files/AGRESTI.PDF AGRESTI.PDF. Categorical data analysis.  [https://mathdept.iut.ac.ir/sites/mathdept.iut.ac.ir/files/AGRESTI.PDF](https://mathdept.iut.ac.ir/sites/mathdept.iut.ac.ir/files/AGRESTI.PDF)

25. Barrett JC, Fry B, Maller J, Daly MJ. Haploview: analysis and visualization of LD and haplotype maps. Bioinformatics. 2005;21(2):263–265. doi: 10.1093/bioinformatics/bth457.  [DOI](https://doi.org/10.1093/bioinformatics/bth457) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15297300/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=Haploview:%20analysis%20and%20visualization%20of%20LD%20and%20haplotype%20maps&author=JC%20Barrett&author=B%20Fry&author=J%20Maller&author=MJ%20Daly&volume=21&issue=2&publication_year=2005&pages=263-265&pmid=15297300&doi=10.1093/bioinformatics/bth457&)

26. Hoh J, Wille A, Ott J. Trimming, weighting, and grouping SNPs in human case–control association studies. Genome Res. 2001;11(12):2115–2119. doi: 10.1101/gr.204001.  [DOI](https://doi.org/10.1101/gr.204001) | [PMC free article](/articles/PMC311222/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11731502/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Res.&title=Trimming,%20weighting,%20and%20grouping%20SNPs%20in%20human%20case%E2%80%93control%20association%20studies&author=J%20Hoh&author=A%20Wille&author=J%20Ott&volume=11&issue=12&publication_year=2001&pages=2115-2119&pmid=11731502&doi=10.1101/gr.204001&)

27. Storey JD. Statistical significance for genomewide studies. Proc. Natl Acad. Sci. USA. 2004;100:9440–9445. doi: 10.1073/pnas.1530509100.  [DOI](https://doi.org/10.1073/pnas.1530509100) | [PMC free article](/articles/PMC170937/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12883005/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl%20Acad.%20Sci.%20USA&title=Statistical%20significance%20for%20genomewide%20studies&author=JD%20Storey&volume=100&publication_year=2004&pages=9440-9445&pmid=12883005&doi=10.1073/pnas.1530509100&)

28. Storey JD, Tibshirani R. Statistical significance for genomewide studies. Proc. Natl Acad. Sci. USA. 2003;100(16):9440–9445. doi: 10.1073/pnas.1530509100.  [DOI](https://doi.org/10.1073/pnas.1530509100) | [PMC free article](/articles/PMC170937/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12883005/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl%20Acad.%20Sci.%20USA&title=Statistical%20significance%20for%20genomewide%20studies&author=JD%20Storey&author=R%20Tibshirani&volume=100&issue=16&publication_year=2003&pages=9440-9445&pmid=12883005&doi=10.1073/pnas.1530509100&)

29. Derringer J, Krueger RF, Dick DM, et al. The aggregate effect of dopamine genes on dependence symptoms among cocaine users: cross-validation of a candidate system scoring approach. Behav. Genet. 2012;42(4):626–635. doi: 10.1007/s10519-012-9531-4.  [DOI](https://doi.org/10.1007/s10519-012-9531-4) | [PMC free article](/articles/PMC3416038/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22358648/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Behav.%20Genet.&title=The%20aggregate%20effect%20of%20dopamine%20genes%20on%20dependence%20symptoms%20among%20cocaine%20users:%20cross-validation%20of%20a%20candidate%20system%20scoring%20approach&author=J%20Derringer&author=RF%20Krueger&author=DM%20Dick&volume=42&issue=4&publication_year=2012&pages=626-635&pmid=22358648&doi=10.1007/s10519-012-9531-4&)

30. Kobayashi K, Kurosawa Y, Fujita K, Nagatsu T. Human dopamine beta-hydroxylase gene: two mRNA types having different 3′-terminal regions are produced through alternative polyadenylation. Nucleic Acids Res. 1989;17(3):1089–1102. doi: 10.1093/nar/17.3.1089.  [DOI](https://doi.org/10.1093/nar/17.3.1089) | [PMC free article](/articles/PMC331724/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/2922261/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&title=Human%20dopamine%20beta-hydroxylase%20gene:%20two%20mRNA%20types%20having%20different%203%E2%80%B2-terminal%20regions%20are%20produced%20through%20alternative%20polyadenylation&author=K%20Kobayashi&author=Y%20Kurosawa&author=K%20Fujita&author=T%20Nagatsu&volume=17&issue=3&publication_year=1989&pages=1089-1102&pmid=2922261&doi=10.1093/nar/17.3.1089&)

31. Cubells JF, Sun X, Li W, et al. Linkage analysis of plasma dopamine beta-hydroxylase activity in families of patients with schizophrenia. Hum. Genet. 2011;130(5):635–643. doi: 10.1007/s00439-011-0989-6.  [DOI](https://doi.org/10.1007/s00439-011-0989-6) | [PMC free article](/articles/PMC3193571/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21509519/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum.%20Genet.&title=Linkage%20analysis%20of%20plasma%20dopamine%20beta-hydroxylase%20activity%20in%20families%20of%20patients%20with%20schizophrenia&author=JF%20Cubells&author=X%20Sun&author=W%20Li&volume=130&issue=5&publication_year=2011&pages=635-643&pmid=21509519&doi=10.1007/s00439-011-0989-6&)

32. Zabetian CP, Anderson GM, Buxbaum SG, et al. A quantitative-trait analysis of human plasma-dopamine beta-hydroxylase activity: evidence for a major functional polymorphism at the DBH locus. Am. J. Hum. Genet. 2001;68(2):515–522. doi: 10.1086/318198.  [DOI](https://doi.org/10.1086/318198) | [PMC free article](/articles/PMC1235285/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11170900/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Hum.%20Genet.&title=A%20quantitative-trait%20analysis%20of%20human%20plasma-dopamine%20beta-hydroxylase%20activity:%20evidence%20for%20a%20major%20functional%20polymorphism%20at%20the%20DBH%20locus&author=CP%20Zabetian&author=GM%20Anderson&author=SG%20Buxbaum&volume=68&issue=2&publication_year=2001&pages=515-522&pmid=11170900&doi=10.1086/318198&)

33. Cubells JF, Kranzler HR, Mccance-Katz E, et al. A haplotype at the DBH locus, associated with low plasma dopamine beta-hydroxylase activity, also associates with cocaine-induced paranoia. Mol. Psychiat. 2000;5(1):56–63. doi: 10.1038/sj.mp.4000657.  [DOI](https://doi.org/10.1038/sj.mp.4000657) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10673769/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol.%20Psychiat.&title=A%20haplotype%20at%20the%20DBH%20locus,%20associated%20with%20low%20plasma%20dopamine%20beta-hydroxylase%20activity,%20also%20associates%20with%20cocaine-induced%20paranoia&author=JF%20Cubells&author=HR%20Kranzler&author=E%20Mccance-Katz&volume=5&issue=1&publication_year=2000&pages=56-63&pmid=10673769&doi=10.1038/sj.mp.4000657&)

34. Kurumaji A, Kuroda T, Yamada K, Yoshikawa T, Toru M. An association of the polymorphic repeat of tetranucleotide (TCAT) in the first intron of the human tyrosine hydroxylase gene with schizophrenia in a Japanese sample. J. Neural. Transm. (Vienna) 2001;108:489–495. doi: 10.1007/s007020170069.  [DOI](https://doi.org/10.1007/s007020170069) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11475015/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Neural.%20Transm.%20(Vienna)&title=An%20association%20of%20the%20polymorphic%20repeat%20of%20tetranucleotide%20(TCAT)%20in%20the%20first%20intron%20of%20the%20human%20tyrosine%20hydroxylase%20gene%20with%20schizophrenia%20in%20a%20Japanese%20sample&author=A%20Kurumaji&author=T%20Kuroda&author=K%20Yamada&author=T%20Yoshikawa&author=M%20Toru&volume=108&publication_year=2001&pages=489-495&pmid=11475015&doi=10.1007/s007020170069&)

35. Wang L, Li B, Lu X, et al. A functional intronic variant in the tyrosine hydroxylase (TH) gene confers risk of essential hypertension in the Northern Chinese Han population. Clin. Sci. (Lond.) 2008;115(5):151–158. doi: 10.1042/CS20070335.  [DOI](https://doi.org/10.1042/CS20070335) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18208403/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Sci.%20(Lond.)&title=A%20functional%20intronic%20variant%20in%20the%20tyrosine%20hydroxylase%20(TH)%20gene%20confers%20risk%20of%20essential%20hypertension%20in%20the%20Northern%20Chinese%20Han%20population&author=L%20Wang&author=B%20Li&author=X%20Lu&volume=115&issue=5&publication_year=2008&pages=151-158&pmid=18208403&doi=10.1042/CS20070335&)

36. Fernandez-Castillo N, Roncero C, Grau-Lopez L, et al. Association study of 37 genes related to serotonin and dopamine neurotransmission and neurotrophic factors in cocaine dependence. Genes Brain Behav. 2013;12(1):39–46. doi: 10.1111/gbb.12013.  [DOI](https://doi.org/10.1111/gbb.12013) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23241418/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genes%20Brain%20Behav.&title=Association%20study%20of%2037%20genes%20related%20to%20serotonin%20and%20dopamine%20neurotransmission%20and%20neurotrophic%20factors%20in%20cocaine%20dependence&author=N%20Fernandez-Castillo&author=C%20Roncero&author=L%20Grau-Lopez&volume=12&issue=1&publication_year=2013&pages=39-46&pmid=23241418&doi=10.1111/gbb.12013&)
