# Pharmacogenomics of statin-related myopathy: Meta-analysis of rare variants from whole-exome sequencing

## Metadata
**Authors:** James S Floyd, Katarzyna M Bloch, Jennifer A Brody, Cyrielle Maroteau, Moneeza K Siddiqui, Richard Gregory, Daniel F Carr, Mariam Molokhia, Xiaoming Liu, Joshua C Bis, Ammar Ahmed, Xuan Liu, Pär Hallberg, Qun-Ying Yue, Patrik K E Magnusson, Diane Brisson, Kerri L Wiggins, Alanna C Morrison, Etienne Khoury, Paul McKeigue, Bruno H Stricker, Maryse Lapeyre-Mestre, Susan R Heckbert, Arlene M Gallagher, Hector Chinoy, Richard A Gibbs, Emmanuelle Bondon-Guitton, Russell Tracy, Eric Boerwinkle, Daniel Gaudet, Anita Conforti, Tjeerd van Staa, Colleen M Sitlani, Kenneth M Rice, Anke-Hilse Maitland-van der Zee, Mia Wadelius, Andrew P Morris, Munir Pirmohamed, Colin A N Palmer, Bruce M Psaty, Ana Alfirevic, on behalf of the PREDICTION-ADR Consortium and EUDRAGENE
**Journal:** PLoS ONE
**Date:** 2019 Jun 26
**DOI:** [10.1371/journal.pone.0218115](https://doi.org/10.1371/journal.pone.0218115)
**PMID:** 31242253
**PMCID:** PMC6594672
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6594672/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC6594672/pdf/pone.0218115.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC6594672/pdf/pone.0218115.pdf)

## Abstract

**Aims:** 
Statin-related myopathy (SRM), which includes rhabdomyolysis, is an uncommon but important adverse drug reaction because the number of people prescribed statins world-wide is large. Previous association studies of common genetic variants have had limited success in identifying a genetic basis for this adverse drug reaction. We conducted a multi-site whole-exome sequencing study to investigate whether rare coding variants confer an increased risk of SRM.

**Methods and results:** 
SRM 3–5 cases (N = 505) and statin treatment-tolerant controls (N = 2047) were recruited from multiple sites in North America and Europe. SRM 3–5 was defined as symptoms consistent with muscle injury and an elevated creatine phosphokinase level >4 times upper limit of normal without another likely cause of muscle injury. Whole-exome sequencing and variant calling was coordinated from two analysis centres, and results of single-variant and gene-based burden tests were meta-analysed. No genome-wide significant associations were identified. Given the large number of cases, we had 80% power to identify a variant with minor allele frequency of 0.01 that increases the risk of SRM 6-fold at genome-wide significance.

**Conclusions:** 
In this large whole-exome sequencing study of severe statin-related muscle injury conducted to date, we did not find evidence that rare coding variants are responsible for this adverse drug reaction. Larger sample sizes would be required to identify rare variants with small effects, but it is unclear whether such findings would be clinically actionable.

### Aims

Statin-related myopathy (SRM), which includes rhabdomyolysis, is an uncommon but important adverse drug reaction because the number of people prescribed statins world-wide is large. Previous association studies of common genetic variants have had limited success in identifying a genetic basis for this adverse drug reaction. We conducted a multi-site whole-exome sequencing study to investigate whether rare coding variants confer an increased risk of SRM.

### Methods and results

SRM 3–5 cases (N = 505) and statin treatment-tolerant controls (N = 2047) were recruited from multiple sites in North America and Europe. SRM 3–5 was defined as symptoms consistent with muscle injury and an elevated creatine phosphokinase level >4 times upper limit of normal without another likely cause of muscle injury. Whole-exome sequencing and variant calling was coordinated from two analysis centres, and results of single-variant and gene-based burden tests were meta-analysed. No genome-wide significant associations were identified. Given the large number of cases, we had 80% power to identify a variant with minor allele frequency of 0.01 that increases the risk of SRM 6-fold at genome-wide significance.

### Conclusions

In this large whole-exome sequencing study of severe statin-related muscle injury conducted to date, we did not find evidence that rare coding variants are responsible for this adverse drug reaction. Larger sample sizes would be required to identify rare variants with small effects, but it is unclear whether such findings would be clinically actionable.

## Introduction

Lipid-lowering drugs that inhibit 5-hydroxy-3-methylglutaryl-coenzyme A (HMG-CoA), known as statins, are widely used for the primary and secondary prevention of cardiovascular disease (CVD). All statins can cause muscle toxicity that ranges in severity from mild muscle pain to severe myopathy or rhabdomyolysis, which can lead to kidney failure and death [[1](#pone.0218115.ref001)]. The most severe forms of statin-related muscle injury are uncommon, suggesting a genetic predisposition. For instance, severe statin-related myopathy (SRM) restricted to cases with creatine phosphokinase (CK) levels >10x the upper limit of normal (ULN) (SRM 4) requiring hospitalisation, occurs in approximately 1–10 out of every 10,000 individuals taking standard statin doses [[2](#pone.0218115.ref002)].

Previous genome-wide association studies (GWAS) have identified a common variant in the drug transporter gene *SLCO1B1* that increases the risk of SRM defined by varying CK elevation criteria, 2- to 4-fold through a pharmacokinetic mechanism, but the underlying causes of this adverse drug reaction remain largely unexplained [[3](#pone.0218115.ref003)–[5](#pone.0218115.ref005)]. To investigate whether rare variants in coding regions of the genome may cause SRM, we conducted whole-exome sequencing on cases of SRM identified from multiple sites throughout North America and Europe, harmonized variant calling and analysis at two centres, and meta-analysed results from single-variant and gene-based burden tests across two large case-control studies.

## Materials and methods

### Study population and overall study design

Identification and recruitment of case and control subjects, whole-exome sequencing, variant calling, and statistical analyses were coordinated by two analysis centers: one based at the University of Washington that included 291 cases and 1540 controls from North America and the United Kingdom (“US-UK”) and one based in Liverpool that included 214 cases and 507 controls from Europe (“PREDICTION-ADR”). Within each of these two large case-control studies, participants were recruited from multiple sites. Because there were few non-European ancestry participants in either case-control study, and many sites had none, we restricted analyses to participants of European ancestry. The details of recruitment at each site are described in the **Supporting Information files**. Whole-exome sequencing and variant calling were harmonized within each of the two case-control studies, and investigators from both studies agreed prospectively to implement a coordinated statistical analysis plan, which included protocols for meta-analysis of results from both single-variant and gene-based burden tests. The University of Washington Human Subjects Division, East of Scotland Research Ethics Service REC1 and REC2, North West–Liverpool Central Research Ethics Committee, Regional Ethical Review Boards in Uppsala (2008/213 and 2010/231) and Stockholm (2007/644-31 and 2011/463-32) and the Medical Ethics Committee from the University Medical Center in Utrecht reviewed and approved this study.

### Case definition

Using a previously published case definition ([S1 Table](#pone.0218115.s002)), SRM 3–5 cases received statin treatment at the time of onset of muscle injury symptoms, had a peak CK level > 4x ULN (SRM 3) or supporting clinical evidence of rhabdomyolysis (SRM 5), and had no other likely cause of muscle injury other than statin treatment.[[1](#pone.0218115.ref001)] In each study, non-statin causes of muscle injury were excluded based on a review of medical records and/or interview of patients by physicians or study staff. Severe SRM cases (SRM 4) sometimes called severe myopathy and SRM 5, sometimes called rhabdomyolysis in some previous studies [[6](#pone.0218115.ref006)], met these criteria and had a peak CK level > 10x ULN ([S1 Table](#pone.0218115.s002)). SRM cases with myopathy and evidence of anti-3-hydroxy-3-methyl-glutaryl-coenzyme A reductase (HMGCR) autoantibodies (SRM 6), which are suggestive of autoimmune myopathy, were excluded at all sites [[7](#pone.0218115.ref007)]. In both case-control studies, statin treatment-tolerant controls subjects were users of statins who had no evidence of muscle injury during follow up.

### Whole-exome sequencing, variant calling, quality control

Exome sequencing, variant calling, and quality control of sequence data were performed separately in each case-control study ([S1 Appendix](#pone.0218115.s001) and [S2 Table](#pone.0218115.s003)). Each study conducted sequencing and called variants for cases and controls together, to eliminate batch effects across case-control sampling. We created a combined variant annotation file including all quality-controlled variant sites observed in either study. Variants were annotated using ANNOVAR [[8](#pone.0218115.ref008)] and dbNSFP v3.0 [[9](#pone.0218115.ref009)] according to the reference genome GRCh37.

### Statistical analysis

Within each case-control study, which included only European ancestry participants, population structure was adjusted for using principal components derived from a genetic relationship matrix. To estimate associations between single variants and the risk of SRM, the Firth bias-corrected score test was used [[10](#pone.0218115.ref010)]. The primary single-variant analysis included all SRM cases and all control subjects. Single variant analyses were restricted to variants with at least 10 copies of the minor allele, but not further restricted by annotated variant function. Results from the two case-control studies were combined using a Z-based meta-analysis [[11](#pone.0218115.ref011)]. The following secondary analyses were also conducted: restriction to users of simvastatin, atorvastatin, and cerivastatin (the three most commonly-used statins in these studies); stratification of SRM cases into moderate (SRM 3: CK ≥ 4x ULN and <10x ULN) and severe (SRM 4 and 5: CK ≥ 10x ULN) categories; and inclusion of all missense variants. In the US-UK case-control study, information on fibrate use was collected. Because fibrates can cause drug-drug interactions with statins through inhibition of the metabolism of statins, and because strong drug-drug interactions can mask the presence of a genuine drug-gene interaction that may be less potent, we conducted secondary analyses restricted to non-users of fibrates.

Rare variants (MAF < 1%) were collapsed into gene-burden scores and tested for association with the risk of SRM [[12](#pone.0218115.ref012)]. Gene-burden results from the two case-control studies were combined using a Z-based meta-analysis. The primary gene-based analysis included variants with MAF < 1% that are missense, bioinformatically predicted to be damaging (MetaLR > .5) [[13](#pone.0218115.ref013)], stop-gain/loss, coding indels, or splice site variants. Secondary gene-burden analyses restricted to users of simvastatin or atorvastatin, stratified SRM cases into moderate (SRM 3: CK > = 4x ULN and <10x ULN) and severe (SRM 4 and 5: CK > = 10x ULN) categories, and included rare missense variants regardless of their prediction score for function.

For each analysis, the Bonferroni-corrected threshold for statistical significance was P = 0.05 / number of single variants or genes tested. For the primary single-variant analysis, we conducted *post-hoc* power calculations using QUANTO 1.1 [[14](#pone.0218115.ref014), [15](#pone.0218115.ref015)] to determine study power in the primary analysis population across a range of effect sizes and MAFs, based on α = 0.05 / number of single variants included in the primary analysis and an estimated disease probability among control subjects of 0.0001 [[16](#pone.0218115.ref016)].

The full results from each meta-analysis will be available on dbGaP within the next release of Cohort for Heart and Aging Research in Genomic Epidemiology Results CHARGE Consortium summary results (accession phs000930) ([http://www.chargeconsortium.com/main/results](http://www.chargeconsortium.com/main/results)). Information on how to access individual-level study data is provided in the Supporting Information files.

## Results

Across the two case-control studies, there were 505 SRM cases and 2,047 treatment-tolerant controls that passed sample-level quality control ([Table 1](#pone.0218115.t001)). Most of the US-UK cases (mean CK 159x ULN) were severe (65%) and used cerivastatin or simvastatin, while most of the PREDICTION-ADR cases (mean CK 32x ULN) were moderate (61%) and used simvastatin or atorvastatin.

### Table 1. Characteristics of statin-related myopathy cases and statin-tolerant controls from two international case-control studies.

|   | US-UK CasesN = 291 | US-UK ControlsN = 1540 | PREDICTION-ADR CasesN = 214 | PREDICTION-ADR ControlsN = 507 |
| --- | --- | --- | --- | --- |
| Demographic data |   |   |   |   |
| Mean age, years (SD) | 65 (10.4) | 62 (8.3) | 66 (11) | 69 (9.5) |
| Age range, years | [34, 91] | [44, 87] | [21, 92] | [39, 93] |
| Gender, % female | 44.0% | 53.8% | 34.1% | 36.5% |
| Statin Type, % |   |   |   |   |
| Atorvastatin | 32 (11%) | 121 (8%) | 57 (27%) | 129 (25%) |
| Cerivastatin | 146 (50%) | 10 (1%) | 1 (0%) | 0 |
| Fluvastatin | 3 (1%) | 1 (0%) | 4 (2%) | 4 (1%) |
| Pravastatin | 7 (2%) | 172 | 13 (6%) | 26 (5%) |
| Rosuvastatin | 8 (3%) | 130 (12%) | 12 (6%) | 7 (1%) |
| Simvastatin | 93 (33%) | 82 (53%) | 127 (59%) | 341 (67%) |
| Lovastatin | 0 | 28 (2%) | 0 | 0 |
| Unknown type | 2 (1%) | 996 (65%) | 0 | 0 |
| CK/ULN, mean (SD) | 159 (325) | N/A | 32 (74) | N/A |
| CK/ULN, range | [4, 529] | N/A | [4, 542] | N/A |
| Myopathy type |   |   |   |   |
| Moderate (SRM 3) | 101 (35%) | N/A | 132 (61%) | N/A |
| Severe (SRM 4 or 5) | 190 (65%) | N/A | 82 (39%) | N/A |
| Severe, no fibrates | 98 (34%) | N/A | 53 (25%) | N/A |
### Single-variant results

The mean sequencing depth in US-UK case-control study was 78x, and in PREDICTION-ADR it was 56x. After restricting to rare (MAF < 0.01) likely damaging variants, 162,813 variants passed quality control in one or both studies and were included in meta-analyses ([Fig 1](#pone.0218115.g001)). In primary analyses that included all SRM cases and all statin-tolerant controls, there were no variants that met the Bonferroni-corrected threshold for statistical significance (P < 3.07 x 10^−7^) ([Table 2](#pone.0218115.t002)). The widely replicated locus for statin-related muscle injury in *SLCO1B1* (rs4149056) was associated with a 1.59-fold increased risk of SRM (95% confidence interval, 1.41–1.77), but this finding was not significant after correction for the number of single variants tested.

### Fig 1. Single variant analysis.

![Fig 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dce5/6594672/ef11c5dfc95a/pone.0218115.g001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=6594672_pone.0218115.g001.jpg)

Single variants included in primarily analysis by frequency (panel A) and type (panel B).

### Table 2. Top associations from meta-analysis of primary single-variant results from US-UK and PREDICTION-ADR case-control studies of statin-related myopathy.

| Ch:position | Coded/referent allele | MAF | P value | US-UKBeta (SE) | PREDICTIONBeta (SE) | Annotation |
| --- | --- | --- | --- | --- | --- | --- |
| 6:44221262 | G/A | 0.6% | 4.59E-07 | 2.89 (0.60) | NA | Nonsynonymous:HSP90AB1 |
| 22:20117046 | C/T | 4.3% | 1.04E-06 | 1.00 (0.24) | 0.66 (0.24) | Intergenic |
| 12:21331549 | C/T | 17.8% | 1.26E-06 | 0.59 (0.12) | 0.25 (0.15) | Nonsynonymous:SLCO1B1 |
| 17:4337283 | A/G | 16.8% | 1.84E-06 | -0.68 (0.15) | NA | Synonymous:SPNS3 |
| 6:30918065 | C/T | 0.6% | 5.35E-06 | 2.23 (0.48) | NA | Synonymous:DPCR1 |
| 6:42236725 | GCT/G | 0.3% | 6.95E-06 | 3.08 (0.74) | NA | Frameshift:TRERF1 |
| 7:21778429 | C/T | 5.5% | 1.79E-05 | -0.845 (0.26) | -0.73 (0.30) | Nonsynonymous:DNAH11 |
| 7:36552656 | A/G | 37.9% | 2.49E-05 | NA | -0.55 (0.13) | Utr3:AOAH |
| 6:32261014 | A/C | 1.4% | 2.67E-05 | 1.02 (0.37) | 1.44 (0.45) | Nonsynonymous:C6orf10 |

Table 2 Caption: MAF = minor (coded) allele frequency

Secondary analyses restricted by statin type and stratified by moderate or severe cases of SRM did not yield any associations that met criteria for statistical significance ([S3 Table](#pone.0218115.s004)). Among non-users of fibrates in the US-UK case-control study, the *SLCO1B1* variant rs4149056 was associated with a 4.01-fold increased risk of SRM (95% confidence interval, 2.61–6.17), and this association met criteria for statistical significance (P = 5.46 x 10^−11^) ([S4 Table](#pone.0218115.s005)).

### Gene-burden results

In total, 10,244 genes passed quality control thresholds in one or both studies and were included in the meta-analyses of burden test results. In primary analyses of all SRM cases and all controls, no genes met the Bonferroni-corrected threshold for statistical significance (P < 4.9 x 10^−6^) ([Table 3](#pone.0218115.t003)). Similarly, none of the secondary burden tests analyses yielded genome-wide significant findings ([S5 Table](#pone.0218115.s006)).

### Table 3. Top associations from meta-analysis of primary gene burden results from US-UK and PREDICTION-ADR case-control studies of statin-related myopathy.

| Gene | cMAF | P value | US-UKBeta (SE) | PREDICTIONBeta (SE) |
| --- | --- | --- | --- | --- |
| TRERF1 | 0.5% | 0.00017 | 2.78 (0.63) | 0.98 (2.31) |
| FOXP4 | 0.8% | 0.00066 | 1.31 (0.39) | 1.22 (1.03) |
| BAAT | 1.3% | 0.0011 | 0.72 (0.34) | 1.19 (0.47) |
| EPB41 | 1.8% | 0.0014 | 0.69 (0.25) | 1.15 (0.67) |
| ESYT3 | 3.5% | 0.0024 | 0.62 (0.20) | 0.46 (0.35) |
| NR4A3 | 2.2% | 0.0028 | 0.41 (0.34) | 1.12 (0.40) |
| TMEM239 | 0.6% | 0.0032 | 1.89 (0.79) | 0.98 (0.56) |
| FAN1 | 2.8% | 0.0035 | 0.52 (0.22) | 1.09 (0.63) |
| MYBPH | 0.9% | 0.0035 | 1.15 (0.38) | 0.68 (0.84) |

Table 3 Caption: cMAF = cumulative minor allele frequency

### Post-hoc power calculations

We conducted *post-hoc* power calculations to estimate study power in the primary analysis population to identify single-variants associated with SRM given a range of MAFs and effect sizes ([Fig 2](#pone.0218115.g002)). For a variant with a MAF of 0.01, we had 98% power to detect an OR of 5, and for a variant with a MAF of 0.005, we had 84% power to detect an OR of 6.

### Fig 2. Power calculation.

![Fig 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dce5/6594672/0f2870e92dfa/pone.0218115.g002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=6594672_pone.0218115.g002.jpg)

Post-hoc power calculations to identify a single variant associated with statin-related myopathy in primary analysis population given effect size (odds ratio) and minor allele frequency (MAF), α = 0.05 / 162,813.

## Discussion

We recruited more than 500 patients with statin-related muscle injury from multiple sites in North America and Europe to conduct the largest whole-exome sequencing study of this adverse drug reaction to date. Using high-throughput sequencing technology, joint variant calling at two experienced analysis centres, and rigorous statistical methods, we did not identify any novel coding variants associated with SRM 3–5. We also evaluated hypotheses about the burden of rare coding variants within genes, muscle injury due to specific statin drugs, and the severity of muscle injury; the findings from these analyses were null as well.

In 2008, Link *et al*. found that a common nonsynonymous variant in the drug transporter *SLCO1B1* (rs4149056) was associated with a 4-fold increase in the risk of myopathy among users of high dose simvastatin at genome-wide levels of statistical significance [[4](#pone.0218115.ref004)]. This finding has been widely replicated in other settings for other statin drugs [[3](#pone.0218115.ref003), [5](#pone.0218115.ref005), [17](#pone.0218115.ref017)–[19](#pone.0218115.ref019)], and in our study this common coding variant was associated with a 4-fold increased risk of SRM 5 among non-users of fibrates. Interestingly, we did not find an association between the *SLCO1B1* polymorphisms and mild forms of SRM. Attempts to discover additional common variants related to statin-related muscle injury have failed to yield replicable findings [[20](#pone.0218115.ref020)–[22](#pone.0218115.ref022)]. A recent study from the Czech Republic conducted whole-exome sequencing on 88 patients with mild statin-associated muscle toxicity, reporting an increased burden of rare variation in 24 genes [[23](#pone.0218115.ref023)]. We did not replicate these findings in our study, perhaps due to different phenotypes as most of our patients have more severe SRM. The validity of these findings may be limited because of the use of publicly available sequencing repositories that do not have information on medication use as the referent group rather than statin-tolerant control subjects.

Strengths of our study include the recruitment of patients from multiple international sites; the use of a standardized case-definition for different forms of myopathy in our study (SRM 3–5) with the mildest being CK>4x ULN (SRM 3) and rhabdomyolysis ranging from CK>10 to >40xULN with end-organ impairment and muscle symptoms[[1](#pone.0218115.ref001)], the depth of whole-exome sequencing in both case-control studies; the use of statin exposed controls to minimize genetic associations with indications for statin therapy, the coordinated statistical analysis plan with correction for multiple testing to reduce the risk of false positive findings; and the prospective plan for a meta-analysis of results across two large-case control studies.

There were several limitations. As with any adverse drug reaction, it is difficult to attribute causality to a specific drug, even with prospectively developed case definitions and expert adjudication of cases. In addition, the presence of unknown drug-drug interactions with statin that increase the risk of SRM may have masked our ability to identify genuine drug-gene interactions. We recognize that our inability to recruit a large number of cases from diverse ancestries limits the generalizability of our findings to variants observed in European ancestry participants. Half of the SRM cases in the US-UK case-control study were due to the drug cerivastatin, and we were unable to identify a large number of controls subjects with WES who used this drug, since it was removed from the market in 2001. If there is heterogeneity in the effects of genetic variants on the risk of SRM by the type of statin drug or severity of SRM, this might have reduced our overall power to identify genetic loci for this adverse drug reaction in our primary analysis, which included all samples. Finally, using our stringent correction for multiple comparisons, we may have missed an association with rare, loss of function variants in biologically plausible metabolising enzyme or transporter genes that could have led to large increase in drug exposure and therefore muscle damage in some individuals [[24](#pone.0218115.ref024)].

While our post-hoc calculations demonstrated that our study was well-powered to identify rare coding variants (MAF 1%) with large effect sizes (OR 5), we had limited power to detect rare variants with small- or moderate-sized effects, or to detect even rarer variants with large effects. For instance, to have 90% power to identify a coding variant with a frequency of 0.001 among controls that increases the risk of SRM 6-fold, we would have to recruit and conduct whole-exome sequencing on approximately 2,700 cases with this rare adverse drug reaction, matched with 10,800 treatment-tolerant controls.

It is possible that rare variants exist that increase the risk of SRM, perhaps in non-coding regions of the genome, but they are unlikely to be detected with the sample sizes available in ongoing studies. Based on the null findings from our study, genotyping rare coding alleles for the prediction of severe statin-related muscle injury seems unlikely to yield clinically actionable findings.

## Supporting information

## Acknowledgments

We thank research nurses Anita Hanson, Ruth Westhead and Diane Van Eker, and administrators Joe Fahy, Emma McDonough and Angela Foxcroft in Liverpool, for recruiting patients and administering the clinical database. We thank the Centre for Genomic Research lead by Prof Christiane Hertz-Fowler, which is the MRC sequencing hub at the University of Liverpool, UK.

We thank research nurses Ulrica Ramqvist and Elisabeth Stjernberg and research assistants Sofie Collin, Eva Prado Lopez, Agnes Kataja Knight, Agnes Wadelius, and Martha Wadelius, Department of Medical Sciences, Clinical Pharmacology, Uppsala University, Uppsala, Sweden, for recruiting and interviewing cases and for administering the phenotype database. We acknowledge Barbro Sandin at the Swedish Twin Registry for access to data for the Swedish controls.

The lead author from PREDICTION-ADR is Prof Colin AN Palmer (c.n.a.palmer@dundee.ac.uk),

PREDICTION ADR consortium members: Colin AN Palmer, Ana Alfirevic, Mia Wadelius, Anke-Hilse Maitland-van der Zee, Ekaterina V Baranova, Niclas Eriksson, Cyrielle Maroteau, Katarzyna M Bloch, Moneeza K Siddiqui, Daniel F Carr, Pär Hallberg, Munir Pirmohamed, Anu Aaspollu, Alun McCarthy, Olivier Delrieu.

The lead author from EUDRAGENE is Dr Molokhia (mariam.molokhia@kcl.ac.uk),

EUDRAGEN consortium members: Mariam Molokhia, Paul McKeigue, Bruno H Stricker, Maryse Lapeyre-Mestre, Anita Conforti, Emmanuelle Bondon-Guitton.

## Data Availability

The full results from each meta-analysis will be available on dbGaP within the next release of Cohort for Heart and Aging Research in Genomic Epidemiology Results CHARGE Consortium summary results (accession phs000930) ([http://www.chargeconsortium.com/main/results](http://www.chargeconsortium.com/main/results)). For the individual-level data, it is impossible to provide individual level data needed to replicate findings on publicly available databases. Most of the individual studies are prohibited from sharing individual-level data with any kind of public database. Information on how to access individual-level study data is provided in the Supporting Information files. Interested researchers are able to re-create the minimal data set (with the individual level data) through the information currently provided within the paper and Supporting Information files.

## Funding Statement

Support for collection and sequencing of statin-related rhabdomyolysis cases in the US-UK study was provided by R01HL078888 from the National Heart, Lung, and Blood Institute. Collection and sequencing of controls for the US-UK study was supported by the ARIC Study, the Cardiovascular Health Study, and the CHARGE-S project. The ARIC Study is carried out as a collaborative study supported by National Heart, Lung, and Blood Institute (NHLBI) contracts (HHSN268201100005C, HHSN268201100006C, HHSN268201100007C, HHSN268201100008C, HHSN268201100009C, HHSN268201100010C, HHSN268201100011C, and HHSN268201100012C). CHS research was supported by National Heart, Lung, and Blood Institute grants and contracts HHSN268201200036C, HHSN268200800007C, HHSN268201800001C, N01HC55222, N01HC85079, N01HC85080, N01HC85081, N01HC85082, N01HC85083, N01HC85086, U01HL080295, R01HL087652, R01HL105756, R01HL103612, R01HL120393, U01HL130114, and R01AG023629, as well as additional contributions from the National Institute of Neurological Disorders and Stroke. Funding support for "Building on GWAS for NHLBI-diseases: the U.S. CHARGE Consortium" was provided by the NIH through the American Recovery and Reinvestment Act of 2009 (ARRA) (5RC2HL102419). Sequencing was carried out at the Baylor Genome Center (U54 HG003273). The content is solely the responsibility of the authors and does not necessarily represent the official views of the National Institutes of Health. PREDICTION-ADR has received funding form the European Community’s Seventh Framework Programme (FP7/2007-2013) under Grant Agreement no. 602108. Work in Sweden was supported by the Swedish Research Council (Medicine 521-2011-2440 and 521-2014-3370); Swedish Heart and Lung Foundation (20120557, 20140291 and 20170711); Selander’s foundation; Thuréus’ foundation; the Swedish Medical Products Agency; the Clinical Research Support (ALF) at Uppsala University. The STAGE study samples (PI: Pirmohamed), recruited via the UK Clinical Practice Research Datalink, were from a grant funded by the e-Health Initiative, a joint venture between the Medical Research Council (reference: MC_qA137929), the Wellcome Trust, the Engineering and Physical Sciences Research Council, and The Economic and Social Research Council. MP would also like to thank the MRC Centre for Drug Safety Science for support, and is an Emeritus National Institute for Health Research Senior Investigator. This study also includes funding from the Medical Research Council (MR/N003322/1) and the NIHR Biomedical Research Centre Funding Scheme. The views expressed in this publication are those of the authors and not necessarily those of the NHS, the National Institute for Health Research, or the Department of Health. The EUDRAGENE project was supported with a concerted action grant from the European Commission 5th Framework QLRI-CT-2002- 02757, by the Serious Adverse Events Consortium (SAEC), and the National Institute for Health Research (NIHR) Biomedical Research Centre at Guy’s and St Thomas’ NHS Foundation Trust and King’s College London. The views expressed are those of the author(s) and not necessarily those of the NHS, the NIHR or the Department of Health. We thank research nurses Anita Hanson, Ruth Westhead and Diane Van Eker, and administrators Joe Fahy, Emma McDonough and Angela Foxcroft in Liverpool, for recruiting patients and administering the clinical database. We thank the Centre for Genomic Research lead by Prof Christiane Hertz-Fowler, which is the MRC sequencing hub at the University of Liverpool, UK. We thank research nurses Ulrica Ramqvist and Elisabeth Stjernberg and research assistants Sofie Collin, Eva Prado Lopez, Agnes Kataja Knight, Agnes Wadelius, and Martha Wadelius, Department of Medical Sciences, Clinical Pharmacology, Uppsala University, Uppsala, Sweden, for recruiting and interviewing cases and for administering the phenotype database. We acknowledge Barbro Sandin at the Swedish Twin Registry for access to data for the Swedish controls. The Swedish Twin Registry is managed by Karolinska Institutet and receives funding through the Swedish Research Council under grant no. 2017-00641. Sequencing of the Swedish controls was performed by the SNP&SEQ Technology Platform in Uppsala. The facility is part of the National Genomics Infrastructure (NGI) Sweden and Science for Life Laboratory, and is supported by the Swedish Research Council and the Knut and Alice Wallenberg Foundation. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Data Availability Statement

The full results from each meta-analysis will be available on dbGaP within the next release of Cohort for Heart and Aging Research in Genomic Epidemiology Results CHARGE Consortium summary results (accession phs000930) ([http://www.chargeconsortium.com/main/results](http://www.chargeconsortium.com/main/results)). For the individual-level data, it is impossible to provide individual level data needed to replicate findings on publicly available databases. Most of the individual studies are prohibited from sharing individual-level data with any kind of public database. Information on how to access individual-level study data is provided in the Supporting Information files. Interested researchers are able to re-create the minimal data set (with the individual level data) through the information currently provided within the paper and Supporting Information files.

### Supplementary Materials

### Data Availability Statement

The full results from each meta-analysis will be available on dbGaP within the next release of Cohort for Heart and Aging Research in Genomic Epidemiology Results CHARGE Consortium summary results (accession phs000930) ([http://www.chargeconsortium.com/main/results](http://www.chargeconsortium.com/main/results)). For the individual-level data, it is impossible to provide individual level data needed to replicate findings on publicly available databases. Most of the individual studies are prohibited from sharing individual-level data with any kind of public database. Information on how to access individual-level study data is provided in the Supporting Information files. Interested researchers are able to re-create the minimal data set (with the individual level data) through the information currently provided within the paper and Supporting Information files.

## References

1. Alfirevic A, Neely D, Armitage J, Chinoy H, Cooper RG, Laaksonen R, et al. Phenotype standardization for statin-induced myotoxicity. Clin Pharmacol Ther. 2014;96(4):470–6. 10.1038/clpt.2014.121 .  [DOI](https://doi.org/10.1038/clpt.2014.121) | [PMC free article](/articles/PMC4172546/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24897241/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Phenotype%20standardization%20for%20statin-induced%20myotoxicity&author=A%20Alfirevic&author=D%20Neely&author=J%20Armitage&author=H%20Chinoy&author=RG%20Cooper&volume=96&issue=4&publication_year=2014&pages=470-6&pmid=24897241&doi=10.1038/clpt.2014.121&)

2. Graham DJ, Staffa JA, Shatin D, Andrade SE, Schech SD, La Grenade L, et al. Incidence of hospitalized rhabdomyolysis in patients treated with lipid-lowering drugs. JAMA. 2004;292(21):2585–90. 10.1001/jama.292.21.2585 .  [DOI](https://doi.org/10.1001/jama.292.21.2585) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15572716/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA&title=Incidence%20of%20hospitalized%20rhabdomyolysis%20in%20patients%20treated%20with%20lipid-lowering%20drugs&author=DJ%20Graham&author=JA%20Staffa&author=D%20Shatin&author=SE%20Andrade&author=SD%20Schech&volume=292&issue=21&publication_year=2004&pages=2585-90&pmid=15572716&doi=10.1001/jama.292.21.2585&)

3. Carr DF, O’Meara H, Jorgensen AL, Campbell J, Hobbs M, McCann G, et al. SLCO1B1 Genetic Variant Associated With Statin-Induced Myopathy: A Proof-of-Concept Study Using the Clinical Practice Research Datalink. Clin Pharmacol Ther. 2013;94(6):695–701. 10.1038/clpt.2013.161 .  [DOI](https://doi.org/10.1038/clpt.2013.161) | [PMC free article](/articles/PMC3831180/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23942138/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=SLCO1B1%20Genetic%20Variant%20Associated%20With%20Statin-Induced%20Myopathy:%20A%20Proof-of-Concept%20Study%20Using%20the%20Clinical%20Practice%20Research%20Datalink&author=DF%20Carr&author=H%20O%E2%80%99Meara&author=AL%20Jorgensen&author=J%20Campbell&author=M%20Hobbs&volume=94&issue=6&publication_year=2013&pages=695-701&pmid=23942138&doi=10.1038/clpt.2013.161&)

4. Link E, Parish S, Armitage J, Bowman L, Heath S, Matsuda F, et al. SLCO1B1 variants and statin-induced myopathy—a genomewide study. The New England journal of medicine. 2008;359(8):789–99. Epub 2008/07/25. 10.1056/NEJMoa0801936 .  [DOI](https://doi.org/10.1056/NEJMoa0801936) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18650507/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=The%20New%20England%20journal%20of%20medicine&title=SLCO1B1%20variants%20and%20statin-induced%20myopathy%E2%80%94a%20genomewide%20study&author=E%20Link&author=S%20Parish&author=J%20Armitage&author=L%20Bowman&author=S%20Heath&volume=359&issue=8&publication_year=2008&pages=789-99&pmid=18650507&doi=10.1056/NEJMoa0801936&)

5. Marciante KD, Durda JP, Heckbert SR, Lumley T, Rice K, McKnight B, et al. Cerivastatin, genetic variants, and the risk of rhabdomyolysis. Pharmacogenetics and genomics. 2011;21(5):280–8. 10.1097/FPC.0b013e328343dd7d .  [DOI](https://doi.org/10.1097/FPC.0b013e328343dd7d) | [PMC free article](/articles/PMC3076530/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21386754/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics%20and%20genomics&title=Cerivastatin,%20genetic%20variants,%20and%20the%20risk%20of%20rhabdomyolysis&author=KD%20Marciante&author=JP%20Durda&author=SR%20Heckbert&author=T%20Lumley&author=K%20Rice&volume=21&issue=5&publication_year=2011&pages=280-8&pmid=21386754&doi=10.1097/FPC.0b013e328343dd7d&)

6. Floyd JS, Kaspera R, Marciante KD, Weiss NS, Heckbert SR, Lumley T, et al. A screening study of drug-drug interactions in cerivastatin users: an adverse effect of clopidogrel. Clin Pharmacol Ther. 2012;91(5):896–904. 10.1038/clpt.2011.295 .  [DOI](https://doi.org/10.1038/clpt.2011.295) | [PMC free article](/articles/PMC3830936/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22419147/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=A%20screening%20study%20of%20drug-drug%20interactions%20in%20cerivastatin%20users:%20an%20adverse%20effect%20of%20clopidogrel&author=JS%20Floyd&author=R%20Kaspera&author=KD%20Marciante&author=NS%20Weiss&author=SR%20Heckbert&volume=91&issue=5&publication_year=2012&pages=896-904&pmid=22419147&doi=10.1038/clpt.2011.295&)

7. Mammen AL, Chung T, Christopher-Stine L, Rosen P, Rosen A, Doering KR, et al. Autoantibodies against 3-hydroxy-3-methylglutaryl-coenzyme A reductase in patients with statin-associated autoimmune myopathy. Arthritis Rheum. 2011;63(3):713–21. 10.1002/art.30156 .  [DOI](https://doi.org/10.1002/art.30156) | [PMC free article](/articles/PMC3335400/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21360500/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arthritis%20Rheum&title=Autoantibodies%20against%203-hydroxy-3-methylglutaryl-coenzyme%20A%20reductase%20in%20patients%20with%20statin-associated%20autoimmune%20myopathy&author=AL%20Mammen&author=T%20Chung&author=L%20Christopher-Stine&author=P%20Rosen&author=A%20Rosen&volume=63&issue=3&publication_year=2011&pages=713-21&pmid=21360500&doi=10.1002/art.30156&)

8. Wang K, Li M, Hakonarson H. ANNOVAR: functional annotation of genetic variants from high-throughput sequencing data. Nucleic Acids Res. 2010;38(16):e164 10.1093/nar/gkq603 .  [DOI](https://doi.org/10.1093/nar/gkq603) | [PMC free article](/articles/PMC2938201/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20601685/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=ANNOVAR:%20functional%20annotation%20of%20genetic%20variants%20from%20high-throughput%20sequencing%20data&author=K%20Wang&author=M%20Li&author=H%20Hakonarson&volume=38&issue=16&publication_year=2010&pages=e164&pmid=20601685&doi=10.1093/nar/gkq603&)

9. Liu X, Wu C, Li C, Boerwinkle E. dbNSFP v3.0: A One-Stop Database of Functional Predictions and Annotations for Human Nonsynonymous and Splice-Site SNVs. Human mutation. 2016;37(3):235–41. 10.1002/humu.22932 .  [DOI](https://doi.org/10.1002/humu.22932) | [PMC free article](/articles/PMC4752381/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26555599/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Human%20mutation&title=dbNSFP%20v3.0:%20A%20One-Stop%20Database%20of%20Functional%20Predictions%20and%20Annotations%20for%20Human%20Nonsynonymous%20and%20Splice-Site%20SNVs&author=X%20Liu&author=C%20Wu&author=C%20Li&author=E%20Boerwinkle&volume=37&issue=3&publication_year=2016&pages=235-41&pmid=26555599&doi=10.1002/humu.22932&)

10. Ma C, Blackwell T, Boehnke M, Scott LJ, Go TDi. Recommended joint and meta-analysis strategies for case-control association testing of single low-count variants. Genetic epidemiology. 2013;37(6):539–50. 10.1002/gepi.21742 .  [DOI](https://doi.org/10.1002/gepi.21742) | [PMC free article](/articles/PMC4049324/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23788246/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genetic%20epidemiology&title=Recommended%20joint%20and%20meta-analysis%20strategies%20for%20case-control%20association%20testing%20of%20single%20low-count%20variants&author=C%20Ma&author=T%20Blackwell&author=M%20Boehnke&author=LJ%20Scott&author=TDi%20Go&volume=37&issue=6&publication_year=2013&pages=539-50&pmid=23788246&doi=10.1002/gepi.21742&)

11. Willer CJ, Li Y, Abecasis GR. METAL: fast and efficient meta-analysis of genomewide association scans. Bioinformatics. 2010;26(17):2190–1. Epub 2010/07/10. 10.1093/bioinformatics/btq340 .  [DOI](https://doi.org/10.1093/bioinformatics/btq340) | [PMC free article](/articles/PMC2922887/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20616382/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=METAL:%20fast%20and%20efficient%20meta-analysis%20of%20genomewide%20association%20scans&author=CJ%20Willer&author=Y%20Li&author=GR%20Abecasis&volume=26&issue=17&publication_year=2010&pages=2190-1&pmid=20616382&doi=10.1093/bioinformatics/btq340&)

12. Li B, Leal SM. Methods for detecting associations with rare variants for common diseases: application to analysis of sequence data. Am J Hum Genet. 2008;83(3):311–21. 10.1016/j.ajhg.2008.06.024 .  [DOI](https://doi.org/10.1016/j.ajhg.2008.06.024) | [PMC free article](/articles/PMC2842185/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18691683/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=Methods%20for%20detecting%20associations%20with%20rare%20variants%20for%20common%20diseases:%20application%20to%20analysis%20of%20sequence%20data&author=B%20Li&author=SM%20Leal&volume=83&issue=3&publication_year=2008&pages=311-21&pmid=18691683&doi=10.1016/j.ajhg.2008.06.024&)

13. Dong C, Wei P, Jian X, Gibbs R, Boerwinkle E, Wang K, et al. Comparison and integration of deleteriousness prediction methods for nonsynonymous SNVs in whole exome sequencing studies. Hum Mol Genet. 2015;24(8):2125–37. 10.1093/hmg/ddu733 .  [DOI](https://doi.org/10.1093/hmg/ddu733) | [PMC free article](/articles/PMC4375422/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25552646/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mol%20Genet&title=Comparison%20and%20integration%20of%20deleteriousness%20prediction%20methods%20for%20nonsynonymous%20SNVs%20in%20whole%20exome%20sequencing%20studies&author=C%20Dong&author=P%20Wei&author=X%20Jian&author=R%20Gibbs&author=E%20Boerwinkle&volume=24&issue=8&publication_year=2015&pages=2125-37&pmid=25552646&doi=10.1093/hmg/ddu733&)

14. Gauderman WJ. Sample size requirements for matched case-control studies of gene-environment interaction. Stat Med. 2002;21(1):35–50. Epub 2002/01/10. .  [DOI](https://doi.org/10.1002/sim.973) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11782049/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Stat%20Med&title=Sample%20size%20requirements%20for%20matched%20case-control%20studies%20of%20gene-environment%20interaction&author=WJ%20Gauderman&volume=21&issue=1&publication_year=2002&pages=35-50&pmid=11782049&doi=10.1002/sim.973&)

15. Gauderman WJ, Morrison JM. QUANTO 1.1: A computer program for power and sample size calculations for genetic-epidemiology studies, http://hydra.usc.edu/gxe, 2006.  [http://hydra.usc.edu/gxe](http://hydra.usc.edu/gxe)

16. Graham DJ, Staffa JA, Shatin D, Andrade SE, Schech SD, La Grenade L, et al. Incidence of hospitalized rhabdomyolysis in patients treated with lipid-lowering drugs. JAMA: the journal of the American Medical Association. 2004;292(21):2585–90. Epub 2004/12/02. 10.1001/jama.292.21.2585 .  [DOI](https://doi.org/10.1001/jama.292.21.2585) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15572716/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA:%20the%20journal%20of%20the%20American%20Medical%20Association&title=Incidence%20of%20hospitalized%20rhabdomyolysis%20in%20patients%20treated%20with%20lipid-lowering%20drugs&author=DJ%20Graham&author=JA%20Staffa&author=D%20Shatin&author=SE%20Andrade&author=SD%20Schech&volume=292&issue=21&publication_year=2004&pages=2585-90&pmid=15572716&doi=10.1001/jama.292.21.2585&)

17. Brunham LR, Lansberg PJ, Zhang L, Miao F, Carter C, Hovingh GK, et al. Differential effect of the rs4149056 variant in SLCO1B1 on myopathy associated with simvastatin and atorvastatin. The pharmacogenomics journal. 2012;12(3):233–7. 10.1038/tpj.2010.92 .  [DOI](https://doi.org/10.1038/tpj.2010.92) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21243006/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=The%20pharmacogenomics%20journal&title=Differential%20effect%20of%20the%20rs4149056%20variant%20in%20SLCO1B1%20on%20myopathy%20associated%20with%20simvastatin%20and%20atorvastatin&author=LR%20Brunham&author=PJ%20Lansberg&author=L%20Zhang&author=F%20Miao&author=C%20Carter&volume=12&issue=3&publication_year=2012&pages=233-7&pmid=21243006&doi=10.1038/tpj.2010.92&)

18. Linde R, Peng L, Desai M, Feldman D. The role of vitamin D and SLCO1B1*5 gene polymorphism in statin-associated myalgias. Dermato-endocrinology. 2010;2(2):77–84. 10.4161/derm.2.2.13509 .  [DOI](https://doi.org/10.4161/derm.2.2.13509) | [PMC free article](/articles/PMC3081682/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21547103/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Dermato-endocrinology&title=The%20role%20of%20vitamin%20D%20and%20SLCO1B1*5%20gene%20polymorphism%20in%20statin-associated%20myalgias&author=R%20Linde&author=L%20Peng&author=M%20Desai&author=D%20Feldman&volume=2&issue=2&publication_year=2010&pages=77-84&pmid=21547103&doi=10.4161/derm.2.2.13509&)

19. Voora D, Shah SH, Spasojevic I, Ali S, Reed CR, Salisbury BA, et al. The SLCO1B1*5 genetic variant is associated with statin-induced side effects. Journal of the American College of Cardiology. 2009;54(17):1609–16. 10.1016/j.jacc.2009.04.053 .  [DOI](https://doi.org/10.1016/j.jacc.2009.04.053) | [PMC free article](/articles/PMC3417133/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19833260/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20of%20the%20American%20College%20of%20Cardiology&title=The%20SLCO1B1*5%20genetic%20variant%20is%20associated%20with%20statin-induced%20side%20effects&author=D%20Voora&author=SH%20Shah&author=I%20Spasojevic&author=S%20Ali&author=CR%20Reed&volume=54&issue=17&publication_year=2009&pages=1609-16&pmid=19833260&doi=10.1016/j.jacc.2009.04.053&)

20. Carr DF, Alfirevic A, Johnson R, Chinoy H, van Staa T, Pirmohamed M. GATM gene variants and statin myopathy risk. Nature. 2014;513(7518):E1 Epub 2014/09/19. 10.1038/nature13628 .  [DOI](https://doi.org/10.1038/nature13628) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25230669/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=GATM%20gene%20variants%20and%20statin%20myopathy%20risk&author=DF%20Carr&author=A%20Alfirevic&author=R%20Johnson&author=H%20Chinoy&author=T%20van%20Staa&volume=513&issue=7518&publication_year=2014&pages=E1&pmid=25230669&doi=10.1038/nature13628&)

21. Floyd JS, Bis JC, Brody JA, Heckbert SR, Rice K, Psaty BM. GATM locus does not replicate in rhabdomyolysis study. Nature. 2014;513(7518):E1–3. Epub 2014/09/19. 10.1038/nature13629 .  [DOI](https://doi.org/10.1038/nature13629) | [PMC free article](/articles/PMC4230441/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25230668/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=GATM%20locus%20does%20not%20replicate%20in%20rhabdomyolysis%20study&author=JS%20Floyd&author=JC%20Bis&author=JA%20Brody&author=SR%20Heckbert&author=K%20Rice&volume=513&issue=7518&publication_year=2014&pages=E1-3&pmid=25230668&doi=10.1038/nature13629&)

22. Mangravite LM, Engelhardt BE, Medina MW, Smith JD, Brown CD, Chasman DI, et al. A statin-dependent QTL for GATM expression is associated with statin-induced myopathy. Nature. 2013;502(7471):377–80. Epub 2013/09/03. 10.1038/nature12508 .  [DOI](https://doi.org/10.1038/nature12508) | [PMC free article](/articles/PMC3933266/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23995691/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=A%20statin-dependent%20QTL%20for%20GATM%20expression%20is%20associated%20with%20statin-induced%20myopathy&author=LM%20Mangravite&author=BE%20Engelhardt&author=MW%20Medina&author=JD%20Smith&author=CD%20Brown&volume=502&issue=7471&publication_year=2013&pages=377-80&pmid=23995691&doi=10.1038/nature12508&)

23. Neroldova M, Stranecky V, Hodanova K, Hartmannova H, Piherova L, Pristoupilova A, et al. Rare variants in known and novel candidate genes predisposing to statin-associated myopathy. Pharmacogenomics. 2016;17(13):1405–14. 10.2217/pgs-2016-0071 .  [DOI](https://doi.org/10.2217/pgs-2016-0071) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27296017/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Rare%20variants%20in%20known%20and%20novel%20candidate%20genes%20predisposing%20to%20statin-associated%20myopathy&author=M%20Neroldova&author=V%20Stranecky&author=K%20Hodanova&author=H%20Hartmannova&author=L%20Piherova&volume=17&issue=13&publication_year=2016&pages=1405-14&pmid=27296017&doi=10.2217/pgs-2016-0071&)

24. Teslovich TM, Musunuru K, Smith AV, Edmondson AC, Stylianou IM, Koseki M, et al. Biological, clinical and population relevance of 95 loci for blood lipids. Nature. 2010;466(7307):707–13. 10.1038/nature09270 .  [DOI](https://doi.org/10.1038/nature09270) | [PMC free article](/articles/PMC3039276/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20686565/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Biological,%20clinical%20and%20population%20relevance%20of%2095%20loci%20for%20blood%20lipids&author=TM%20Teslovich&author=K%20Musunuru&author=AV%20Smith&author=AC%20Edmondson&author=IM%20Stylianou&volume=466&issue=7307&publication_year=2010&pages=707-13&pmid=20686565&doi=10.1038/nature09270&)
