# LPA variants are associated with residual cardiovascular risk in patients receiving statins

## Metadata
**Authors:** Wei-Qi Wei, Xiaohui Li, Qiping Feng, Michiaki Kubo, Iftikhar J Kullo, Peggy L Peissig, Elizabeth W Karlson, Gail P Jarvik, Ming Ta Michael Lee, Ning Shang, Eric A Larson, Todd Edwards, Christian Shaffer, Jonathan D Mosley, Shiro Maeda, Momoko Horikoshi, Marylyn Ritchie, Marc S Williams, Eric B Larson, David R Crosslin, Sarah T Bland, Jennifer A Pacheco, Laura J Rasmussen-Torvik, David Cronkite, George Hripcsak, Nancy J Cox, Russell A Wilke, C Michael Stein, Jerome I Rotter, Yukihide Momozawa, Dan M Roden, Ronald M Krauss, Joshua C Denny
**Journal:** Circulation
**Date:** 2018 Oct 23
**DOI:** [10.1161/CIRCULATIONAHA.117.031356](https://doi.org/10.1161/CIRCULATIONAHA.117.031356)
**PMID:** 29703846
**PMCID:** PMC6202211
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6202211/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC6202211/pdf/nihms-967912.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC6202211/pdf/nihms-967912.pdf)

## Abstract

**Background:** 
Coronary heart disease (CHD) is a leading cause of death globally. Although therapy with HMG-CoA reductase inhibitors (statins) decreases circulating levels of low-density lipoprotein cholesterol (LDL-C) and the incidence of CHD, additional events occur despite statin therapy in some individuals. The genetic determinants of this residual cardiovascular risk remain unknown.

**Method:** 
We performed a two-stage genome-wide association study (GWAS) of CHD events during statin therapy. We first identified 3,099 cases who experienced CHD events (defined as acute myocardial infarction or the need for coronary revascularization) during statin therapy and 7,681 controls without CHD events during comparable intensity and duration of statin therapy from four sites in the Electronic Medical Records and Genomics (eMERGE) Network. We then sought replication of candidate variants in another 160 cases and 1112 controls from a fifth eMERGE site, which joined the network after the initial GWAS. Finally, we performed a phenome-wide association study (PheWAS) for other traits linked to the most significant locus.

**Results:** 
The meta-analysis identified seven SNPs at a genome-wide level of significance within the LPA/PLG locus associated with CHD events on statin treatment. The most significant association was for an intronic SNP within LPA/PLG (rs10455872, MAF=0.069, Odds Ratio [OR]=1.58, 95% CI [1.35–1.86], P=2.6×10−10). In the replication cohort, rs10455872 was also associated with CHD events (OR=1.71, 95% CI [1.14–2.57], p=0.009). The association of this SNP with CHD events was independent of statin-induced change in LDL-C (OR=1.62, 95% CI [1.17–2.24], p=0.004) and persisted in individuals with LDL-C≤70mg/dL (OR=2.43, 95% CI [1.18–4.75], p=0.015). PheWAS supported the effect of this region on coronary heart disease and did not identify non-cardiovascular phenotypes.

**Conclusions:** 
Genetic variations at the LPA locus is associated with CHD events during statin therapy independent of the extent of LDL-C lowering. This finding provides support for exploring strategies targeting circulating concentrations of lipoprotein(a) to reduce CHD events in patients receiving statins.

Keywords: LPA, statin, CHD, EHR, LDL-C

### Background:

Coronary heart disease (CHD) is a leading cause of death globally. Although therapy with HMG-CoA reductase inhibitors (statins) decreases circulating levels of low-density lipoprotein cholesterol (LDL-C) and the incidence of CHD, additional events occur despite statin therapy in some individuals. The genetic determinants of this residual cardiovascular risk remain unknown.

### Method:

We performed a two-stage genome-wide association study (GWAS) of CHD events during statin therapy. We first identified 3,099 cases who experienced CHD events (defined as acute myocardial infarction or the need for coronary revascularization) during statin therapy and 7,681 controls without CHD events during comparable intensity and duration of statin therapy from four sites in the Electronic Medical Records and Genomics (eMERGE) Network. We then sought replication of candidate variants in another 160 cases and 1112 controls from a fifth eMERGE site, which joined the network after the initial GWAS. Finally, we performed a phenome-wide association study (PheWAS) for other traits linked to the most significant locus.

### Results:

The meta-analysis identified seven SNPs at a genome-wide level of significance within the *LPA/PLG* locus associated with CHD events on statin treatment. The most significant association was for an intronic SNP within *LPA/PLG* (rs10455872, MAF=0.069, Odds Ratio [OR]=1.58, 95% CI [1.35–1.86], P=2.6×10^−10^). In the replication cohort, rs10455872 was also associated with CHD events (OR=1.71, 95% CI [1.14–2.57], p=0.009). The association of this SNP with CHD events was independent of statin-induced change in LDL-C (OR=1.62, 95% CI [1.17–2.24], p=0.004) and persisted in individuals with LDL-C≤70mg/dL (OR=2.43, 95% CI [1.18–4.75], p=0.015). PheWAS supported the effect of this region on coronary heart disease and did not identify non-cardiovascular phenotypes.

### Conclusions:

Genetic variations at the *LPA* locus is associated with CHD events during statin therapy independent of the extent of LDL-C lowering. This finding provides support for exploring strategies targeting circulating concentrations of lipoprotein(a) to reduce CHD events in patients receiving statins.

**Keywords:** *LPA*, statin, CHD, EHR, LDL-C

## Introduction

Coronary heart disease (CHD) affects more than 80 million Americans and remains the leading cause of mortality worldwide.^1^ Statins (3-hydroxymethyl-3-methyglutaryl coenzyme A [HMG-CoA] reductase inhibitors) reduce the incidence of CHD events. The major cardiovascular benefit of statin treatment is achieved through its ability to reduce circulating levels of low-density lipoprotein cholesterol (LDL-C). A meta-analysis of 26 randomized trials from 170,000 participants demonstrated that statin treatment significantly reduced the five-year incidence of major CHD events by ~20% for every 1 mmol/L (39mg/dL) reduction in circulating levels of LDL-C.^2^ However, clinical trials and retrospective observational cohort studies have reported considerable inter-individual variability in LDL-C response to statins.^3–5^ Recent findings from the Genomic Investigation of Statin Therapy (GIST) consortium have supported earlier evidence^6, 7^ that genetic factors contribute to this variation.^8^ In their genome-wide association study (GWAS), GIST investigators identified single nucleotide polymorphisms (SNPs) at four loci significantly associated with the magnitude of statin-induced LDL-C reduction (*LPA, APOE, SLCO1B1,* and *SORT1/CELSR2/PSRC1).*

Although statin therapy decreases the incidence of CHD events,^2, 9, 10^ events continue to occur despite lower LDL-C levels.^9, 11–13^ For example, a recent clinical trial suggested no additional benefit of LDL-C reduction with respect to major adverse cardiovascular events involving lower cardiovascular risk patients.^12^ The contribution of genetic variation to this residual CHD risk during statin therapy remains unknown, which impedes the development of an optimum approach to long-term reduction of CHD events.^7, 14, 15^ We therefore conducted a multisite case-control GWAS to assess the genetic determinants of CHD events, defined as either acute myocardial infarction (AMI) or the need for revascularization, occurring during statin therapy, and the extent to which risk was dependent on change in LDL-C. Phenotypic information for cases and controls was ascertained across multiple sites of the Electronic Medical Records and Genomics (eMERGE) network using a validated algorithm.^16^

## Methods:

### Availability of data

Data from eMERGE network have been submitted to dgGaP (phs000360, phs000944.v1.p1). The authors declare that other genotyped and phenotypic data will be made available to other researchers through dbGaP for purposes of reproducing the results.

### Research Participants

We performed a two-stage genome wide association study within the eMERGE Network.^17, 18^ The eMERGE Network is a consortium of U.S. cohorts with DNA samples linked to electronic health record (EHR) data for conducting large-scale, high-throughput genetic research. The current phase of the eMERGE Network has twelve member sites. Dense genotypic data coupled to EHRs is in place at each eMERGE site for individuals selected for a range of initial phenotypes.^17^ Each participating site obtained Institutional Review Board approval.

#### Discovery Cohorts:

Our primary analysis was a meta-analysis of cases and controls identified at four eMERGE sites: (1) Vanderbilt University Medical Center’s BioVU resource,^19^ (2) Geisinger Health System, (3) Mayo Clinic, and (4) Marshfield Clinic. We identified cases and controls with extant GWAS data at these four sites. At the same time, an additional large cohort was identified from Vanderbilt’s BioVU resource and was genotyped at the RIKEN Center for Genomic Medicine (BioVU-RIKEN) under an existing alliance with the Pharmacogenomics Research Network (PGRN).^20^

#### Validation Cohort:

To replicate the initial findings, we identified cases and controls from the Partners HealthCare Biobank, which joined the eMERGE network after the initial GWAS was underway. The Partners Biobank is a recontactable EHR-linked DNA biobank with 60,528 consented individuals. Among these individuals, 4,930 had been genotyped at the time of this study. Our replication was limited to only genome-wide significant associations from the initial study.

#### Cohort for Phenome Wide Association Study (PheWAS):

After completing our GWAS, we conducted a PheWAS to investigate other potential associations with our found variant. PheWAS is a systematic approach to replicate and discover relationships between targeted genotypes and multiple phenotypes.^21, 22^ We used 11,566 individuals of European ancestry with genome-wide genotyping data available in BioVU, excluding those in the discovery cohort.

### Phenotyping

#### Identification of CHD events:

We defined a CHD event as either AMI or the need for revascularization. Our algorithm used EHR data, including International Classification of Diseases, Ninth Revision, Clinical Modification (ICD-9-CM) codes, Current Procedural Terminology (CPT) codes, and laboratory test results to identify a CHD event. We defined a CHD event while on statins as one that occurred at least 180 days following the earliest recorded date of statin use. The algorithm is published on PheKB.org and has been validated at three sites of the eMERGE network using manual chart validation.^16^ Manual validation showed a 96% to 100% positive predictive value (PPV) for the identification of CHD events during statin therapy (cases)^16^ and 100% PPV for controls. The BioVU-RIKEN cohort was limited to individuals identified as “White” in the EHR. We identified 1,758 cases and matched them to 3,516 controls for sex and age of statin initiation at a 1:2 ratio. We then added 726 controls based on their availability in BioVU. For the other cohorts, we identified cases and controls from all individuals who had been genotyped.

We also collected information about type 2 diabetes (T2D), hypertension, smoking status, and prior CHD history for each individual. We used the previously validated algorithm for T2D posted at PheKB.org.^23^ We applied previously validated natural language processing (NLP) and machine learning algorithms to determine ever/never smoking status.^24^ We used presence or absence of ICD-9-CM codes to ascertain each individual’s hypertension status (401.*) and whether or not the individual had a history of a CHD (410–414). These covariates were used to adjust analyses.

#### Extraction of LDL-C response to statin treatment:

We used the definition of statin response adopted by the GIST consortium^8^ with modifications as follows. Statin medication exposure and dose and LDL-C measures were extracted from each individual’s EHR by applying NLP algorithms that we have developed and validated.^4^ To qualify, a participant was required to have at least one off-treatment LDL-C measurement and at least one on-treatment LDL-C measurement. We defined the off-treatment LDL-C as the median value of all LDL-C measures before the first mention of statin therapy in the EHR. We defined the on-treatment LDL as the median value of all LDL-C measures after the first mention of statin use. For cases, we only used the LDL-C results prior to the first CHD event during treatment. We calculated the magnitude of LDL-C lowering effect of statin therapy and used it as a covariate.

#### PheWAS:

Following established protocols used in past PheWAS,^25, 26^ we grouped each individual’s ICD-9-CM codes into 1,837 disease phecodes. To be a case for each phecode, an individual needed to have relevant ICD-9-CM codes on two or more different days. Individuals who had only one relevant ICD-9-CM code for a phecode were neither cases nor controls. Controls were remaining individuals who also lacked related ICD-9-CM codes to the phecode (e.g., an individual with ischemic heart disease does not serve as a control for an individual with an acute myocardial infarction). We analyzed all 1,083 phecodes occurring in more than 20 patients.

### Genotyping and imputation

All genotyping was conducted using commercially available genome-wide SNP arrays with quality control criteria for variants before imputation listed in [Supplementary Table 1](#SD1). The eMERGE-Phase-1 cohort, generated during the initial period of eMERGE, included data from Marshfield Clinic and partial data from Geisinger, Mayo, and Vanderbilt BioVU for this study. Geisinger, Mayo and BioVU cohorts represented data collected subsequently. Genotyping for the Geisinger, Mayo Clinic, and Marshfield Clinic and the other Vanderbilt samples was conducted within the eMERGE network. Genotyping for the BioVU-RIKEN set was conducted at RIKEN. Genotyping of Partners Biobank participants was conducted separately. All datasets are exclusive.

Genotype data were curated for quality control using PLINK.^27^ For the BioVU-RIKEN cohort, results were filtered using minor allele frequency (MAF) ≥ 0.01. For other cohorts, we removed samples with (1) per-individual call rate < 95%; (2) per-individual autosome heterozygosity > 5 s.d. from the mean; (3) wrongly assigned gender; (4) one of each pair of individuals with a cryptic relationship closer than a third-degree relative (proportion identity by descent PI_HAT >=0.125)^28^ or both individuals from a duplicated pair (PI_HAT ≥ 0.95); (5) SNPs with a genotyping call rate < 95%; (6) SNPs with estimated allele frequencies in controls that were > 10% different from the population estimate from the 1000 Genomes Project. We also aligned alleles to the genomic forward strand using 1000 Genomes Project allele frequency estimates. The remaining samples were assessed for population stratification using principal component analyses implemented in EIGENSOFT. ^29, 30^

To increase the power and coverage of the GWAS, we performed whole genome imputation. We pre-phased haplotypes from post-QC, strand-aligned genotype data using SHAPEIT2.^31^ We then used IMPUTE2^32^ to perform genotype imputation to the 1000 Genomes Project Phase 3 reference haplotypes (October 2014). Approximately 10 million directly or imputed SNPs passed the quality control filters and were evaluated for association.

### Statistical analysis

We assessed the relationship between genetic variation and the risk of developing a CHD event after statin exposure using the software package SNPTESTv2.2.0. ^33^ We assumed an additive effect of SNP alleles on risk and applied logistic regression with the frequentist test, adjusting for age, sex, T2D, hypertension, smoking status, prior CHD history, and the top 10 principal components for ancestry. SNPs with an info score < 0.4 were removed. The analysis was run on each discovery cohort individually, followed by a meta-analysis using METAL^34^ combining the results from all discovery cohorts and adjusting for multiple testing. We only evaluated the SNPs when there are two or more cohorts with available information. The replication analysis was run separately. Regional association plots were generated using LocusZoom ([http://locuszoom.sph.umich.edu/locuszoom/](http://locuszoom.sph.umich.edu/locuszoom/)).

To evaluate the effect of changes in LDL-C on the top hits, we conducted further analyses adjusting for LDL-C change (defined as the difference between the median LDL-C before and after statin treatment) using individuals for whom this information was available from the largest study cohort (BioVU-RIKEN). We also conducted a survival analysis on the BioVU-RIKEN cohort (the only data set in which time to event data were known), with the endpoint defined as the first CHD event during statin treatment by computing Kaplan-Meier curves for the top hit. The survival analysis was done using the R statistical language 3.3.0. PheWAS was performed using the R PheWAS package^35^ using an additive genetic model and adjusted for age, sex, and principal components. In addition, the PheWAS was repeated also adjusting for statin use and the median LDL-C value.

## Results

Deploying validated algorithms across the discovery set identified 3,099 cases with CHD events on statin and 7,681 controls. The replication cohort from Partners biobank contributed another 160 cases and 1,112 controls. The characteristics of these cohorts are listed in [Table 1](#T1) and [Supplementary Table 1](#SD1).

### Table 1.

|   |   | Number | Sex (M/F ratio) | White (%) | Age (year) | T2DM (%) | Hypertension (%) | Smoker (%) | LDL (mg/dL) |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| Discovery cohort |  |  |  |  |  |  |  |  |  |
| BioVU-RIKEN* | case | 1758 | 2.26 | 100% | 73.59±11.47 | 33% | 87% | 22% | 87.19±31.86 |
| control | 4242 | 1.65 | 100% | 68.94±11.19 | 17% | 36% | 41% | 102.09±31.60 |  |
| eMERGE-Phase-1 | case | 528 | 1.89 | 98% | 82.91±10.92 | 37% | 95% | 46% | 100.77±27.75 |
| control | 1199 | 0.85 | 96% | 75.30±11.15 | 25% | 56% | 40% | 112.74±25.80 |  |
| BioVU | case | 321 | 2.32 | 100% | 73.40±12.26 | 34% | 92% | 46% | 86.66±26.22 |
| control | 134 | 0.71 | 100% | 69.47±12.59 | 19% | 68% | 22% | 109.59±31.87 |  |
| Geisinger | case | 424 | 3.46 | 100% | 77.51±9.52 | 38% | 93% | 86% | 94.04±24.15 |
| control | 1264 | 0.81 | 99% | 68.49±13.50 | 40% | 77% | 62% | 101.80±24.62 |  |
| Mayo | case | 68 | 2.17 | 99% | 74.18±10.58 | 15% | 98% | 60% | 95.67±19.86 |
| control | 842 | 1.05 | 96% | 81.81±10.33 | 8% | 42% | 57% | 112.83±25.23 |  |
| Replication cohort |  |  |  |  |  |  |  |  |  |
| Partners | case | 160 | 2.4 | 87% | 72.85±9.98 | 41% | 98% | 60% | 92.11±29.14 |
| control | 1112 | 0.8 | 89% | 65.41±10.97 | 19% | 67% | 42% | 112.47±28.91 |  |

Table 1 Caption: Demographic Characteristics of Discovery and Replication Sets.

### Primary analysis

The meta-analysis identified seven SNPs within *LPA/PLG* locus that were associated with CHD events while on statin treatment ([Figure 1](#F1); [Table 2](#T2)) at genome-wide significance (p<5×10 ^−8^). The most significant association (*P*=2.6×10^−10^) was for rs10455872 at the *LPA/PLG* locus on chromosome 6 ([Figure 1](#F1), [Table 2](#T2)). The MAF of our cohort for rs10455872 is 7.8%, which is consistent with the 7% MAF in the European population according to 1000 Genome Project.^36^ Carriers of the minor allele were more likely to have CHD events while on statin treatment than non-carriers (odds ratio [OR]=1.58, 95% confidence interval [95% CI] = 1.35–1.86). An additional six variants were associated with case status within the *LPA/PLG* region: rs74617384, rs55730499, rs118039278, rs4252185, rs56393506 and rs2315065. All these variants were in strong linkage with the most strongly associated SNP rs10455872 within *LPA* ([Figure 2](#F2)).

### Figure 1.

![Figure 1.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5477/6202211/2c10ed36415c/nihms-967912-f0001.jpg)

Results of the GWAS meta-analysis. (a) Manhattan plot presenting the –log10 P values from the meta-analysis (n=10,780; 3,099 cases and 7,681 controls) on association with CHD events on statin. P values were generated using logistic regression analysis. (b) Forest plot of association of rs10455872 with CHD from each discovery cohort.

### Table 2.

| Chr | Position | SNP | Gene | Minor Allele | Frequency | OR (95% confidence interval) | Direction | P-value |
| --- | --- | --- | --- | --- | --- | --- | --- | --- |
| 6 | 161010118 | rs10455872 | LPA | G | 0.078 | 1.58 (1.35–1.86) | +++−+ | 2.6 × 10−10 |
| 6 | 160997118 | rs74617384 | LPA | T | 0.078 | 1.58 (1.35–1.86) | +++−+ | 3.2 × 10−10 |
| 6 | 161005610 | rs55730499 | LPA | T | 0.080 | 1.56 (1.33–1.83) | +++−+ | 6.7 × 10−10 |
| 6 | 160985526 | rs118039278 | LPA | A | 0.078 | 1.56 (1.33–1.83) | +++−+ | 8.5 × 10−10 |
| 6 | 161123451 | rs4252185 | PLG | C | 0.078 | 1.69 (1.43–2.01) | ++−−+ | 1.1 × 10−9 |
| 6 | 161089307 | rs56393506 | LPA | T | 0.181 | 1.31 (1.17–1.47) | +++++ | 1.1 × 10−8 |
| 6 | 161108144 | rs2315065 | LPA / PLG | A | 0.088 | 1.53 (1.31–1.78) | ++−++ | 2.6 × 10−8 |

Table 2 Caption: Genome-wide significant associations in discovery meta-analysis.

### Figure 2.

![Figure 2.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5477/6202211/3ff31a6bf39d/nihms-967912-f0002.jpg)

Regional association plots of the LPA locus on association with CHD events during statin treatment. The color of the SNPs is based on the linkage disequilibrium with the lead SNP (shown in purple). RefSeq genes in the region are shown in lower panel. P values were generated using logistic regression analysis.

The *LPA* locus was the only one of the four loci identified by the GIST consortium as being associated with LDL-C response to statin therapy to also be associated with the risk of CHD events at genome-wide significance (<10^−8^) in this study (data for the other 3 GIST loci are presented in [Supplementary Table 2](#SD1)).

### Replication

We tested the top associations at the *LPA/PLG* locus in the cohort from the Partners biobank ([Table 3](#T3)). The top SNP from the primary analysis, rs10455872, was associated with CHD events on statins (OR=1.71, 95% CI=1.14–2.57, p=0.009). Two other top SNPs, rs74617384 and rs55730499, were also replicated. The effect size and direction were similar to those in the discovery set.

### Table 3.

|   |   | Discovery Cohorts |  | Validation Cohort |
| --- | --- | --- | --- | --- |
|   |   | Without adjustment for LDL-C changeOR (95% confidence interval [CI]) | With adjustment for LDL-C change OR (95% CI) | OR (95% CI, P value) |
| rs10455872 | LPA | 1.58 (1.35–1.86) | 1.62 (1.17–2.24) | 1.71 (1.14–2.57, 0.0093) |
| rs74617384 | LPA | 1.58 (1.35–1.86) | 1.62 (1.17–2.24) | 1.71 (1.14–2.57, 0.0093) |
| rs55730499 | LPA | 1.56 (1.33–1.83) | 1.57 (1.14–2.17) | 1.67 (1.11–2.50, 0.0134) |
| rs118039278 | LPA | 1.56 (1.33–1.83) | 1.60 (1.16–2.21) | 1.55 (1.01–2.39, 0.0456) |
| rs4252185 | PLG | 1.69 (1.43–2.01) | 1.73 (1.23–2.43) | 1.34 (0.87–2.07, 0.1903) |
| rs56393506 | LPA | 1.31 (1.17–1.47) | 1.28 (1.02–1.60) | 1.17 (0.86–1.61, 0.3198) |
| rs2315065 | LPA / PLG | 1.53 (1.31–1.78) | 1.58 (1.16–2.16) | 1.44 (0.98–2.14, 0.0660) |

Table 3 Caption: Replication results.

### Effects of LDL-C changes on the associations of LPA SNPs with CHD

To account for the effect of LDL-lowering on the association of top findings in the *LPA* locus with CHD events, we extracted LDL-C changes for 474 cases and 832 controls from the BioVU-RIKEN cohort and repeated the analysis adjusting for statin-induced change in LDL-C. As shown in [Table 3](#T3), the *LPA* locus was associated with CHD events while on statin treatment independent of LDL-C changes. The effect size was unchanged after adjustment for LDL-C change (rs10455872, OR before adjustment 1.58 and 1.62 after adjustment, [Table 3](#T3)).

### LDL-C stratified analysis

We then collected data of individuals from both discovery and validation cohorts and performed a stratified analysis for individuals with various LDL-C levels. Based on their available LDL-C results, we classified individuals into two distinct groups, i.e. a group with mean LDL-C≤70 mg/dL (n=480) and a group with mean LDL-C>70 mg/dL (n=4,069). As shown in [Table 4](#T4), rs10455872 was significantly associated with CHD in individuals with LDL-C≤70 mg/dL (OR=2.43, P=0.015) and was similar regardless of adjustment for age, sex, and race.

### Table 4.

| LDL-C (mg/dL) | N (cases/controls) | no adjustment |  | Adjusted by age, sex, and race |  |
| --- | --- | --- | --- | --- | --- |
| P | OR (95% CI) | P | OR (95% CI) |  |  |
| ≤70 | 480 (187/293) | 0.016 | 2.34 (1.18–4.75) | 0.015 | 2.43 (1.19–5.07) |
| >70 | 4,069 (947/3,122) | <0.001 | 1.42 (1.16–1.73) | <0.001 | 1.48 (1.20–1.82) |

Table 4 Caption: Sub-analyses of individuals with different LDL-C

Furthermore, from the cohort of 6,000 BioVU-RIKEN study individuals with detailed longitudinal EHR data, we identified 67 cases and 69 controls with mean LDL-C≤70 mg/dL prior to the CHD event; all were white males. The p-values for association with CHD events were 0.008 (without adjustment) and 0.0078 (adjusted for age) in these individuals with LDL-C≤70 mg/dL prior to their CHD events.

### Survival analysis

We conducted a survival analysis of rs10455872 in the BioVU-RIKEN cohort ([Figure 3](#F3)). The group with two copies of G (n_g=2_=27) developed CHD earlier than the group with one copy (n_g=1_=593) and no copy (n_g=0_=4,721). In the multivariate Cox regression analysis, age (P<2×10^−16^), sex (P=1.90×10^−4^), smoking status (P<2×10^−16^), and rs10455872 (P=2.05×10^−6^) were significantly associated with CHD events during statin treatment. We also found similar results for those individuals with mean LDL-C≤70 mg/dL (n_g=0_=3,086; n_g=1_=519; n_g=2_=20; P=0.01).

### Figure 3.

![Figure 3.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5477/6202211/5ece5c7e94dc/nihms-967912-f0003.jpg)

Kaplan-Meier curves by rs10455872 with CHD events. Table shows survival probabilities across up to 3,000 days. The P value from log rank test is 1.92×10-6.

### PheWAS analysis

We performed a PheWAS for rs10455872 in 11,566 individuals of European ancestry from three Illumina genome-wide SNP arrays available in BioVU (HumanCoreExome, MEGA, and OncoArray). The analysis result showed significant associations between rs10455872 and coronary disease, including the phenotypes for coronary atherosclerosis, chronic ischemic heart disease, unstable angina, and myocardial infarction ([Figure 4](#F4)). The signals also remained significant after adjusting by median LDL-C value and statin use.

### Figure 4.

![Figure 4.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5477/6202211/186a9e860b0c/nihms-967912-f0004.jpg)

PheWAS results of rs10455872 on 11,566 additional individuals of European ancestry. Coronary atherosclerosis, other chronic ischemic heart disease, and unstable angina were top hits associated with the SNP adjusted by sex and age (left). Other chronic ischemic heart disease, coronary atherosclerosis, unstable angina, and myocardial infarction remained significant adjusted by sex, age, median LDL-C value, and statin usage (right).

## Discussion

To identify genetic variants associated with CHD events during statin treatment, we conducted a multi-site case-control GWAS (>10,700 statin users) in the context of routine clinical care. We identified variants in the *LPA/PLG* locus associated with risk of CHD events while on statin therapy. Each copy of the risk allele G at the lead variant, rs10455872, was associated with a 58% increased risk of CHD events. The MAF for rs10455872 in European populations is 7%. The association was independent of the LDL-C lowering effect of statin treatment and was also present in individuals with low LDL-C (≤70mg/dL).

The *LPA* gene encodes apolipoprotein (a), a liver-derived protein with homology to plasminogen. Apolipoprotein (a) is covalently bound to apoprotein B on an LDL particle, forming a particle designated Lp(a). Circulating Lp(a) levels vary widely across individuals and ethnic groups, and >70% of the variation can be attributed to variants at the *LPA* locus, including Kringle IV repeats.^37–39^ In a previous study of 1,822 individuals, the minor allele G of rs10455872 was associated with an increase in circulating Lp(a) levels of approximately 25%.^40^

Plasma Lp(a) level is an independent predictor for CHD. The Copenhagen City Heart Study reported a stepwise increase of AMI risk associated with elevated Lp(a) concentration.^41^ Bennet and colleagues^42^ conducted a meta-analysis of 9,870 individuals with CHD cases. They found individuals in the top tertile of Lp(a) levels were 1.45 times more likely to develop CHD than those in the bottom tertile (OR=1.45, 95% 1.32–1.58). The association changed slightly after adjusting for smoking, lipids, blood pressure, diabetes and body mass index (BMI).^43, 44^ Notably, a Mendelian randomization study of *LPA* variants associated with both Lp(a) levels and CHD risk provided further evidence for a causal role of Lp(a) in the pathogenesis of CHD.^40^

We observed that the minor allele of rs10455872 was associated with a 58% increase of CHD risk in statin-treated patients, which is similar to the 47% increase in CHD in carriers of this allele reported for non-statin treated patients^40^. Our findings are also consistent with the finding of an association of this SNP with CHD in statin-treated patients in an *LPA* candidate gene study (OR 1.41, 95% CI 1.17–1.68).^14^ Although this variant has been previously associated with less LDL-C reduction in response to statin therapy^8^, the change in LDL-C levels could not explain all the increased CHD risk. Donnelly et al. reported that the minor allele of rs10455872 was associated with a 0.10 mmol/L LDL reduction^14^, corresponding to only a ~2% increase of CHD risk.^2, 45, 46^ Our follow-up analysis adjusting for LDL-C change further supports the independent role of rs10455872 in predicting on-treatment risk of a CHD event.

Given the known association of rs10455872 with circulating Lp(a) levels, these data suggest a causal role for Lp(a) in residual CHD risk for individuals on statins. Previously, a meta-analysis of nine clinical trials reported an association of atorvastatin treatment with a decrease of Lp(a) levels.^47^ A similar effect was also observed in a small study of both atorvastatin and rosuvastatin.^44^ However, the JUPITER trial reported zero median change in Lp(a) with rosuvastatin and placebo.^43^ Further study is needed to clarify the degree to which the association seen in our work was due to statin mediated change in Lp(a) levels; current data suggests that the effect is small and may vary by statin.

Despite substantial evidence that Lp(a) promotes the progression of atherosclerosis and increases the risk of thrombosis in individuals with high plaque burden^48^, a recent report from the DALoutcome trial showed no association between Lp(a) level and risk of ischemic cardiovascular events after acute coronary syndrome (ACS).^49^ The disparate findings may be due to differences in study cohorts and outcome definitions. Subjects in DALoutcomes had a recent ACS while ours included all statin users regardless of CHD history. In addition, our definition of an event contains both AMI and the need for revascularization. Nevertheless, a limitation of our study was that we were not capable of measuring Lp(a) in subjects. In the future, a Mendelian randomization study of *LPA* in a mix of statin and non-statin users could further elucidate this effect.

The variant rs10455872 may influence circulating levels of Lp(a) by altering *LPA* expression. Lu et al. reported that the carriers of rs10455872 have a higher level of *LPA* mRNA than non-carriers.^50^ However, we cannot rule out other roles of this variant in regulating circulating Lp(a) levels. By querying the Genotype-Tissure Expression (GTEx)^51, 52^ databases, we found that rs10455872 is an expression quantitative trait loci (eQTL) for *SLC22A3,* a gene located upstream of *LPA*. *SLC22A3* is a polyspecific organic cation transporter that is expressed in the liver, kidney, intestine. Further work is needed to determine its involvement in lipid metabolism and CHD.

PheWAS also supported the association between rs10455872 and coronary heart disease. This association was independent of statin use and median LDL-C value, further supporting the primary findings from the GWAS. We did not observe other significant signals in PheWAS, which suggests that drugs mediating Lp(a) may not have other significant effects (positive or negative), making Lp(a) a desirable target for further drug development

The finding in this genome-wide study adds to the evidence for an important role of Lp(a) in contributing to cardiovascular risk in patients on statin therapy. The potential for lowering Lp(a) with existing and emerging therapeutic agents thus holds promise for further reducing CHD events in statin-treated patients.^53^

### Limitations

Our analysis combined data for all statins and statin doses due to differing practice patterns across study sites. While most individuals were receiving atorvastatin or simvastatin, many received two or more different statins and/or doses ([Supplementary Table 3](#SD1)). In addition, although individuals were taking the medication based on their refill records in the EHR, we were not able to definitively ascertain compliance. Most of our data were based on populations of European ancestry. Further study is needed to determine whether rs10455872 is associated with similar residual CHD risk in other ethnic populations since Lp(a) levels vary widely across ethnic groups. Our validation analysis focused on the top signal rs10455872. Although rs10455872 explains approximately 25% of the variance in circulating Lp(a) levels, a future study using variations within the *LPA* gene will be of interest. Furthermore, we cannot rule out a possible role of *PLG* in the risk of CHD events while on statin. *PLG* encodes plasminogen, which is critical for both intravascular and extravascular fibrinolysis,^54^ and patients with plasminogen deficiency have an increased risk of thrombosis.^55–57^ Follow-up study is needed to evaluate the relationship between *PLG* and statin treatment. Though we imputed the genotype data, we cannot rule out the possibility that important rare or low frequency genetic variants were missed. We did not examine aspirin therapy in this study due to its high use as a secondary prevention strategy in our cohorts. For example, in the Vanderbilt cohort, 99% of cases had aspirin documented in their EHRs. Given the very small number of patients not using aspirin in this cohort, we lack the statistical power to rigorously quantify the relative contribution of aspirin as a covariate. Finally, our definition of CHD events included both AMI and the need for revascularization. We were not able to conduct an analysis using AMI alone due to limited statistical power.

## Conclusion

Our GWAS demonstrates that genetic variants in *LPA* are associated with CHD events while on statin therapy, highlighting *LPA* as an important contributor to residual CHD risk.

## Supplementary Material

## Clinical Perspective

## Clinical Perspective

## Clinical Perspective

## Clinical Perspective

### What is new?

### What are clinical implications?

## Acknowledgments

**Funding Sources:**

The project was supported by eMERGE Network (Phase III) This phase of the eMERGE Network was initiated and funded by the National Human Genome Research Institute (NHGRI) through the following grants: U01HG8657 (Group Health Cooperative/University of Washington); U01HG8685 (Brigham and Women’s Hospital); U01HG8672 (Vanderbilt University Medical Center); U01HG8666 (Cincinnati Children’s Hospital Medical Center); U01HG6379 (Mayo Clinic); U01HG8679 (Geisinger Clinic); U01HG8680 (Columbia University Health Sciences); U01HG8684 (Children’s Hospital of Philadelphia); U01HG8673 (Northwestern University); U01HG8701 (Vanderbilt University Medical Center serving as the Coordinating Center); U01HG8676 (Partners Healthcare/Broad Institute); and U01HG8664 (Baylor College of Medicine) . The project was supported by NIH grant 1 R01 LM 010685, P50 GM115305, R01 HL133786, R01 GM103859, U01 HG008672, R01 GM120523 and American Heart Association (AHA) Scientist Development Grant 16SDG27490014.

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Mozaffarian D, Benjamin EJ, Go AS, Arnett DK, Blaha MJ, Cushman M, Das SR, de Ferranti S, Despres JP, Fullerton HJ, Howard VJ, Huffman MD, Isasi CR, Jimenez MC, Judd SE, Kissela BM, Lichtman JH, Lisabeth LD, Liu S, Mackey RH, Magid DJ, McGuire DK, Mohler ER 3rd, Moy CS, Muntner P, Mussolino ME, Nasir K, Neumar RW, Nichol G, Palaniappan L, Pandey DK, Reeves MJ, Rodriguez CJ, Rosamond W, Sorlie PD, Stein J, Towfighi A, Turan TN, Virani SS, Woo D, Yeh RWand Turner MB. Heart Disease and Stroke Statistics-2016 Update: A Report From the American Heart Association. Circulation. 2016;133:e38-60.  [DOI](https://doi.org/10.1161/CIR.0000000000000350) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26673558/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circulation&title=&author=D%20Mozaffarian&author=EJ%20Benjamin&author=AS%20Go&author=DK%20Arnett&author=MJ%20Blaha&volume=133&publication_year=2016&pages=e38-60&pmid=26673558&doi=10.1161/CIR.0000000000000350&)

2. Cholesterol Treatment Trialists C, Baigent C, Blackwell L, Emberson J, Holland LE, Reith C, Bhala N, Peto R, Barnes EH, Keech A, Simes J and Collins R. Efficacy and safety of more intensive lowering of LDL cholesterol: a meta-analysis of data from 170,000 participants in 26 randomised trials. Lancet. 2010;376:1670–1681.  [DOI](https://doi.org/10.1016/S0140-6736(10)61350-5) | [PMC free article](/articles/PMC2988224/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21067804/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Efficacy%20and%20safety%20of%20more%20intensive%20lowering%20of%20LDL%20cholesterol:%20a%20meta-analysis%20of%20data%20from%20170,000%20participants%20in%2026%20randomised%20trials&author=C%20Cholesterol%20Treatment%20Trialists&author=C%20Baigent&author=L%20Blackwell&author=J%20Emberson&author=LE%20Holland&volume=376&publication_year=2010&pages=1670-1681&pmid=21067804&doi=10.1016/S0140-6736(10)61350-5&)

3. Pearson TA, Laurora I, Chu H and Kafonek S. The lipid treatment assessment project (L-TAP): a multicenter survey to evaluate the percentages of dyslipidemic patients receiving lipid-lowering therapy and achieving low-density lipoprotein cholesterol goals. Arch Intern Med. 2000;160:459–467.  [DOI](https://doi.org/10.1001/archinte.160.4.459) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10695686/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arch%20Intern%20Med&title=The%20lipid%20treatment%20assessment%20project%20(L-TAP):%20a%20multicenter%20survey%20to%20evaluate%20the%20percentages%20of%20dyslipidemic%20patients%20receiving%20lipid-lowering%20therapy%20and%20achieving%20low-density%20lipoprotein%20cholesterol%20goals&author=TA%20Pearson&author=I%20Laurora&author=H%20Chu&author=S%20Kafonek&volume=160&publication_year=2000&pages=459-467&pmid=10695686&doi=10.1001/archinte.160.4.459&)

4. Wei WQ, Feng Q, Jiang L, Waitara MS, Iwuchukwu OF, Roden DM, Jiang M, Xu H, Krauss RM, Rotter JI, Nickerson DA, Davis RL, Berg RL, Peissig PL, McCarty CA, Wilke RA and Denny JC. Characterization of statin dose response in electronic medical records. Clin Pharmacol Ther. 2014;95:331–338.  [DOI](https://doi.org/10.1038/clpt.2013.202) | [PMC free article](/articles/PMC3944214/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24096969/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Characterization%20of%20statin%20dose%20response%20in%20electronic%20medical%20records&author=WQ%20Wei&author=Q%20Feng&author=L%20Jiang&author=MS%20Waitara&author=OF%20Iwuchukwu&volume=95&publication_year=2014&pages=331-338&pmid=24096969&doi=10.1038/clpt.2013.202&)

5. Simon JA, Lin F, Hulley SB, Blanche PJ, Waters D, Shiboski S, Rotter JI, Nickerson DA, Yang H, Saad M and Krauss RM. Phenotypic predictors of response to simvastatin therapy among African-Americans and Caucasians: the Cholesterol and Pharmacogenetics (CAP) Study. Am J Cardiol. 2006;97:843–850.  [DOI](https://doi.org/10.1016/j.amjcard.2005.09.134) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16516587/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Cardiol&title=Phenotypic%20predictors%20of%20response%20to%20simvastatin%20therapy%20among%20African-Americans%20and%20Caucasians:%20the%20Cholesterol%20and%20Pharmacogenetics%20(CAP)%20Study&author=JA%20Simon&author=F%20Lin&author=SB%20Hulley&author=PJ%20Blanche&author=D%20Waters&volume=97&publication_year=2006&pages=843-850&pmid=16516587&doi=10.1016/j.amjcard.2005.09.134&)

6. Chasman DI, Posada D, Subrahmanyan L, Cook NR, Stanton VP Jr. and Ridker PM. Pharmacogenetic study of statin therapy and cholesterol reduction. JAMA. 2004;291:2821–2827.  [DOI](https://doi.org/10.1001/jama.291.23.2821) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15199031/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA&title=Pharmacogenetic%20study%20of%20statin%20therapy%20and%20cholesterol%20reduction&author=DI%20Chasman&author=D%20Posada&author=L%20Subrahmanyan&author=NR%20Cook&author=VP%20Stanton&volume=291&publication_year=2004&pages=2821-2827&pmid=15199031&doi=10.1001/jama.291.23.2821&)

7. Hopewell JC, Parish S, Offer A, Link E, Clarke R, Lathrop M, Armitage J, Collins R and Group MBHPSC . Impact of common genetic variation on response to simvastatin therapy among 18 705 participants in the Heart Protection Study. Eur Heart J. 2013;34:982–992.  [DOI](https://doi.org/10.1093/eurheartj/ehs344) | [PMC free article](/articles/PMC3612775/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23100282/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20Heart%20J&title=.%20Impact%20of%20common%20genetic%20variation%20on%20response%20to%20simvastatin%20therapy%20among%2018%20705%20participants%20in%20the%20Heart%20Protection%20Study&author=JC%20Hopewell&author=S%20Parish&author=A%20Offer&author=E%20Link&author=R%20Clarke&volume=34&publication_year=2013&pages=982-992&pmid=23100282&doi=10.1093/eurheartj/ehs344&)

8. Postmus I, Trompet S, Deshmukh HA, Barnes MR, Li X, Warren HR, Chasman DI, Zhou K, Arsenault BJ, Donnelly LA, Wiggins KL, Avery CL, Griffin P, Feng Q, Taylor KD, Li G, Evans DS, Smith AV, de Keyser CE, Johnson AD, de Craen AJ, Stott DJ, Buckley BM, Ford I, Westendorp RG, Slagboom PE, Sattar N, Munroe PB, Sever P, Poulter N, Stanton A, Shields DC, O’Brien E, Shaw-Hawkins S, Chen YD, Nickerson DA, Smith JD, Dube MP, Boekholdt SM, Hovingh GK, Kastelein JJ, McKeigue PM, Betteridge J, Neil A, Durrington PN, Doney A, Carr F, Morris A, McCarthy MI, Groop L, Ahlqvist E, Welcome Trust Case Control C, Bis JC, Rice K, Smith NL, Lumley T, Whitsel EA, Sturmer T, Boerwinkle E, Ngwa JS, O’Donnell CJ, Vasan RS, Wei WQ, Wilke RA, Liu CT, Sun F, Guo X, Heckbert SR, Post W, Sotoodehnia N, Arnold AM, Stafford JM, Ding J, Herrington DM, Kritchevsky SB, Eiriksdottir G, Launer LJ, Harris TB, Chu AY, Giulianini F, MacFadyen JG, Barratt BJ, Nyberg F, Stricker BH, Uitterlinden AG, Hofman A, Rivadeneira F, Emilsson V, Franco OH, Ridker PM, Gudnason V, Liu Y, Denny JC, Ballantyne CM, Rotter JI, Adrienne Cupples L, Psaty BM, Palmer CN, Tardif JC, Colhoun HM, Hitman G, Krauss RM, Wouter Jukema J and Caulfield MJ. Pharmacogenetic meta-analysis of genome-wide association studies of LDL cholesterol response to statins. Nat Commun. 2014;5:5068.  [DOI](https://doi.org/10.1038/ncomms6068) | [PMC free article](/articles/PMC4220464/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25350695/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Commun&title=Pharmacogenetic%20meta-analysis%20of%20genome-wide%20association%20studies%20of%20LDL%20cholesterol%20response%20to%20statins&author=I%20Postmus&author=S%20Trompet&author=HA%20Deshmukh&author=MR%20Barnes&author=X%20Li&volume=5&publication_year=2014&pages=5068&pmid=25350695&doi=10.1038/ncomms6068&)

9. Ridker PM, Danielson E, Fonseca FA, Genest J, Gotto AM Jr., Kastelein JJ, Koenig W, Libby P, Lorenzatti AJ, MacFadyen JG, Nordestgaard BG, Shepherd J, Willerson JT, Glynn RJ and Group JS. Rosuvastatin to prevent vascular events in men and women with elevated C-reactive protein. N Engl J Med. 2008;359:2195–2207.  [DOI](https://doi.org/10.1056/NEJMoa0807646) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18997196/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Rosuvastatin%20to%20prevent%20vascular%20events%20in%20men%20and%20women%20with%20elevated%20C-reactive%20protein&author=PM%20Ridker&author=E%20Danielson&author=FA%20Fonseca&author=J%20Genest&author=AM%20Gotto&volume=359&publication_year=2008&pages=2195-2207&pmid=18997196&doi=10.1056/NEJMoa0807646&)

10. Cholesterol Treatment Trialists C, Fulcher J, O’Connell R, Voysey M, Emberson J, Blackwell L, Mihaylova B, Simes J, Collins R, Kirby A, Colhoun H, Braunwald E, La Rosa J, Pedersen TR, Tonkin A, Davis B, Sleight P, Franzosi MG, Baigent C and Keech A. Efficacy and safety of LDL-lowering therapy among men and women: meta-analysis of individual data from 174,000 participants in 27 randomised trials. Lancet. 2015;385:1397–1405.  [DOI](https://doi.org/10.1016/S0140-6736(14)61368-4) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25579834/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Efficacy%20and%20safety%20of%20LDL-lowering%20therapy%20among%20men%20and%20women:%20meta-analysis%20of%20individual%20data%20from%20174,000%20participants%20in%2027%20randomised%20trials&author=J%20Fulcher&author=R%20O%E2%80%99Connell&author=M%20Voysey&author=J%20Emberson&author=L%20Blackwell&volume=385&publication_year=2015&pages=1397-1405&pmid=25579834&doi=10.1016/S0140-6736(14)61368-4&)

11. Sampson UK, Fazio S and Linton MF. Residual cardiovascular risk despite optimal LDL cholesterol reduction with statins: the evidence, etiology, and therapeutic challenges. Curr Atheroscler Rep. 2012;14:1–10.  [DOI](https://doi.org/10.1007/s11883-011-0219-7) | [PMC free article](/articles/PMC3697085/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22102062/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Atheroscler%20Rep&title=Residual%20cardiovascular%20risk%20despite%20optimal%20LDL%20cholesterol%20reduction%20with%20statins:%20the%20evidence,%20etiology,%20and%20therapeutic%20challenges&author=UK%20Sampson&author=S%20Fazio&author=MF%20Linton&volume=14&publication_year=2012&pages=1-10&pmid=22102062&doi=10.1007/s11883-011-0219-7&)

12. Ridker PM, Revkin J, Amarenco P, Brunell R, Curto M, Civeira F, Flather M, Glynn RJ, Gregoire J, Jukema JW, Karpov Y, Kastelein JJP, Koenig W, Lorenzatti A, Manga P, Masiukiewicz U, Miller M, Mosterd A, Murin J, Nicolau JC, Nissen S, Ponikowski P, Santos RD, Schwartz PF, Soran H, White H, Wright RS, Vrablik M, Yunis C, Shear CL, Tardif JC and Investigators SCO. Cardiovascular Efficacy and Safety of Bococizumab in High-Risk Patients. N Engl J Med. 2017;376:1527–1539.  [DOI](https://doi.org/10.1056/NEJMoa1701488) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28304242/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Cardiovascular%20Efficacy%20and%20Safety%20of%20Bococizumab%20in%20High-Risk%20Patients&author=PM%20Ridker&author=J%20Revkin&author=P%20Amarenco&author=R%20Brunell&author=M%20Curto&volume=376&publication_year=2017&pages=1527-1539&pmid=28304242&doi=10.1056/NEJMoa1701488&)

13. Sabatine MS, Giugliano RP, Keech AC, Honarpour N, Wiviott SD, Murphy SA, Kuder JF, Wang H, Liu T, Wasserman SM, Sever PS, Pedersen TR, Committee FS and Investigators. Evolocumab and Clinical Outcomes in Patients with Cardiovascular Disease. N Engl J Med. 2017;376:1713–1722.  [DOI](https://doi.org/10.1056/NEJMoa1615664) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28304224/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Evolocumab%20and%20Clinical%20Outcomes%20in%20Patients%20with%20Cardiovascular%20Disease&author=MS%20Sabatine&author=RP%20Giugliano&author=AC%20Keech&author=N%20Honarpour&author=SD%20Wiviott&volume=376&publication_year=2017&pages=1713-1722&pmid=28304224&doi=10.1056/NEJMoa1615664&)

14. Donnelly LA, van Zuydam NR, Zhou K, Tavendale R, Carr F, Maitland-van der Zee AH, Leusink M, de Boer A, Doevendans PA, Asselbergs FW, Morris AD, Pearson ER, Klungel OH, Doney AS and Palmer CN. Robust association of the LPA locus with low-density lipoprotein cholesterol lowering response to statin treatment in a meta-analysis of 30 467 individuals from both randomized control trials and observational studies and association with coronary artery disease outcome during statin treatment. Pharmacogenet Genomics. 2013;23:518–525.  [DOI](https://doi.org/10.1097/FPC.0b013e3283642fd6) | [PMC free article](/articles/PMC4110714/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23903772/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Robust%20association%20of%20the%20LPA%20locus%20with%20low-density%20lipoprotein%20cholesterol%20lowering%20response%20to%20statin%20treatment%20in%20a%20meta-analysis%20of%2030%20467%20individuals%20from%20both%20randomized%20control%20trials%20and%20observational%20studies%20and%20association%20with%20coronary%20artery%20disease%20outcome%20during%20statin%20treatment&author=LA%20Donnelly&author=NR%20van%20Zuydam&author=K%20Zhou&author=R%20Tavendale&author=F%20Carr&volume=23&publication_year=2013&pages=518-525&pmid=23903772&doi=10.1097/FPC.0b013e3283642fd6&)

15. Li JH, Suchindran S, Shah SH, Kraus WE, Ginsburg GS and Voora D. SLCO1B1 genetic variants, long-term low-density lipoprotein cholesterol levels and clinical events in patients following cardiac catheterization. Pharmacogenomics. 2015;16:449–458.  [DOI](https://doi.org/10.2217/pgs.15.2) | [PMC free article](/articles/PMC4719145/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25916517/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=SLCO1B1%20genetic%20variants,%20long-term%20low-density%20lipoprotein%20cholesterol%20levels%20and%20clinical%20events%20in%20patients%20following%20cardiac%20catheterization&author=JH%20Li&author=S%20Suchindran&author=SH%20Shah&author=WE%20Kraus&author=GS%20Ginsburg&volume=16&publication_year=2015&pages=449-458&pmid=25916517&doi=10.2217/pgs.15.2&)

16. Wei WQ, Feng Q, Weeke P, Bush W, Waitara MS, Iwuchukwu OF, Roden DM, Wilke RA, Stein CM and Denny JC. Creation and Validation of an EMR-based Algorithm for Identifying Major Adverse Cardiac Events while on Statins. AMIA Jt Summits Transl Sci Proc. 2014;2014:112–119.  [PMC free article](/articles/PMC4333709/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25717410/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=AMIA%20Jt%20Summits%20Transl%20Sci%20Proc&title=Creation%20and%20Validation%20of%20an%20EMR-based%20Algorithm%20for%20Identifying%20Major%20Adverse%20Cardiac%20Events%20while%20on%20Statins&author=WQ%20Wei&author=Q%20Feng&author=P%20Weeke&author=W%20Bush&author=MS%20Waitara&volume=2014&publication_year=2014&pages=112-119&pmid=25717410&)

17. Gottesman O, Kuivaniemi H, Tromp G, Faucett WA, Li R, Manolio TA, Sanderson SC, Kannry J, Zinberg R, Basford MA, Brilliant M, Carey DJ, Chisholm RL, Chute CG, Connolly JJ, Crosslin D, Denny JC, Gallego CJ, Haines JL, Hakonarson H, Harley J, Jarvik GP, Kohane I, Kullo IJ, Larson EB, McCarty C, Ritchie MD, Roden DM, Smith ME, Bottinger EP, Williams MS and e MN. The Electronic Medical Records and Genomics (eMERGE) Network: past, present, and future. Genet Med. 2013;15:761–771.  [DOI](https://doi.org/10.1038/gim.2013.72) | [PMC free article](/articles/PMC3795928/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23743551/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genet%20Med&title=The%20Electronic%20Medical%20Records%20and%20Genomics%20(eMERGE)%20Network:%20past,%20present,%20and%20future&author=O%20Gottesman&author=H%20Kuivaniemi&author=G%20Tromp&author=WA%20Faucett&author=R%20Li&volume=15&publication_year=2013&pages=761-771&pmid=23743551&doi=10.1038/gim.2013.72&)

18. McCarty CA, Chisholm RL, Chute CG, Kullo IJ, Jarvik GP, Larson EB, Li R, Masys DR, Ritchie MD, Roden DM, Struewing JP, Wolf WA and e MT. The eMERGE Network: a consortium of biorepositories linked to electronic medical records data for conducting genomic studies. BMC Med Genomics. 2011;4:13.  [DOI](https://doi.org/10.1186/1755-8794-4-13) | [PMC free article](/articles/PMC3038887/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21269473/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Med%20Genomics&title=The%20eMERGE%20Network:%20a%20consortium%20of%20biorepositories%20linked%20to%20electronic%20medical%20records%20data%20for%20conducting%20genomic%20studies&author=CA%20McCarty&author=RL%20Chisholm&author=CG%20Chute&author=IJ%20Kullo&author=GP%20Jarvik&volume=4&publication_year=2011&pages=13&pmid=21269473&doi=10.1186/1755-8794-4-13&)

19. Roden DM, Pulley JM, Basford MA, Bernard GR, Clayton EW, Balser JR and Masys DR. Development of a large-scale de-identified DNA biobank to enable personalized medicine. Clin Pharmacol Ther. 2008;84:362–369.  [DOI](https://doi.org/10.1038/clpt.2008.89) | [PMC free article](/articles/PMC3763939/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18500243/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Development%20of%20a%20large-scale%20de-identified%20DNA%20biobank%20to%20enable%20personalized%20medicine&author=DM%20Roden&author=JM%20Pulley&author=MA%20Basford&author=GR%20Bernard&author=EW%20Clayton&volume=84&publication_year=2008&pages=362-369&pmid=18500243&doi=10.1038/clpt.2008.89&)

20. Yee SW, Momozawa Y, Kamatani Y, Tyndale RF, Weinshilboum RM, Ratain MJ, Giacomini KM and Kubo M. Genomewide Association Studies in Pharmacogenomics: Meeting Report of the NIH Pharmacogenomics Research Network-RIKEN (PGRN-RIKEN) Collaboration. Clin Pharmacol Ther. 2016;100:423–426.  [DOI](https://doi.org/10.1002/cpt.405) | [PMC free article](/articles/PMC5056156/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27256705/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Genomewide%20Association%20Studies%20in%20Pharmacogenomics:%20Meeting%20Report%20of%20the%20NIH%20Pharmacogenomics%20Research%20Network-RIKEN%20(PGRN-RIKEN)%20Collaboration&author=SW%20Yee&author=Y%20Momozawa&author=Y%20Kamatani&author=RF%20Tyndale&author=RM%20Weinshilboum&volume=100&publication_year=2016&pages=423-426&pmid=27256705&doi=10.1002/cpt.405&)

21. Bush WS, Oetjens MT and Crawford DC. Unravelling the human genome-phenome relationship using phenome-wide association studies. Nat Rev Genet. 2016;17:129–145.  [DOI](https://doi.org/10.1038/nrg.2015.36) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26875678/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Genet&title=Unravelling%20the%20human%20genome-phenome%20relationship%20using%20phenome-wide%20association%20studies&author=WS%20Bush&author=MT%20Oetjens&author=DC%20Crawford&volume=17&publication_year=2016&pages=129-145&pmid=26875678&doi=10.1038/nrg.2015.36&)

22. Denny JC, Bastarache L, Ritchie MD, Carroll RJ, Zink R, Mosley JD, Field JR, Pulley JM, Ramirez AH, Bowton E, Basford MA, Carrell DS, Peissig PL, Kho AN, Pacheco JA, Rasmussen LV, Crosslin DR, Crane PK, Pathak J, Bielinski SJ, Pendergrass SA, Xu H, Hindorff LA, Li R, Manolio TA, Chute CG, Chisholm RL, Larson EB, Jarvik GP, Brilliant MH, McCarty CA, Kullo IJ, Haines JL, Crawford DC, Masys DR and Roden DM. Systematic comparison of phenome-wide association study of electronic medical record data and genome-wide association study data. Nat Biotechnol. 2013;31:1102–1110.  [DOI](https://doi.org/10.1038/nbt.2749) | [PMC free article](/articles/PMC3969265/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24270849/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Biotechnol&title=Systematic%20comparison%20of%20phenome-wide%20association%20study%20of%20electronic%20medical%20record%20data%20and%20genome-wide%20association%20study%20data&author=JC%20Denny&author=L%20Bastarache&author=MD%20Ritchie&author=RJ%20Carroll&author=R%20Zink&volume=31&publication_year=2013&pages=1102-1110&pmid=24270849&doi=10.1038/nbt.2749&)

23. Kho AN, Hayes MG, Rasmussen-Torvik L, Pacheco JA, Thompson WK, Armstrong LL, Denny JC, Peissig PL, Miller AW, Wei WQ, Bielinski SJ, Chute CG, Leibson CL, Jarvik GP, Crosslin DR, Carlson CS, Newton KM, Wolf WA, Chisholm RL and Lowe WL. Use of diverse electronic medical record systems to identify genetic risk for type 2 diabetes within a genome-wide association study. J Am Med Inform Assoc. 2012;19:212–218.  [DOI](https://doi.org/10.1136/amiajnl-2011-000439) | [PMC free article](/articles/PMC3277617/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22101970/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Am%20Med%20Inform%20Assoc&title=Use%20of%20diverse%20electronic%20medical%20record%20systems%20to%20identify%20genetic%20risk%20for%20type%202%20diabetes%20within%20a%20genome-wide%20association%20study&author=AN%20Kho&author=MG%20Hayes&author=L%20Rasmussen-Torvik&author=JA%20Pacheco&author=WK%20Thompson&volume=19&publication_year=2012&pages=212-218&pmid=22101970&doi=10.1136/amiajnl-2011-000439&)

24. Liu M, Shah A, Jiang M, Peterson NB, Dai Q, Aldrich MC, Chen Q, Bowton EA, Liu H, Denny JC and Xu H. A study of transportability of an existing smoking status detection module across institutions. AMIA Annu Symp Proc. 2012;2012:577–586.  [PMC free article](/articles/PMC3540509/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23304330/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=AMIA%20Annu%20Symp%20Proc&title=A%20study%20of%20transportability%20of%20an%20existing%20smoking%20status%20detection%20module%20across%20institutions&author=M%20Liu&author=A%20Shah&author=M%20Jiang&author=NB%20Peterson&author=Q%20Dai&volume=2012&publication_year=2012&pages=577-586&pmid=23304330&)

25. Karnes JH, Bastarache L, Shaffer CM, Gaudieri S, Xu Y, Glazer AM, Mosley JD, Zhao S, Raychaudhuri S, Mallal S, Ye Z, Mayer JG, Brilliant MH, Hebbring SJ, Roden DM, Phillips EJ and Denny JC. Phenome-wide scanning identifies multiple diseases and disease severity phenotypes associated with HLA variants. Sci Transl Med. 2017;9 DOI: 10.1126/scitranslmed.aai8708.  [DOI](https://doi.org/10.1126/scitranslmed.aai8708) | [PMC free article](/articles/PMC5563969/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28490672/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Sci%20Transl%20Med&title=Phenome-wide%20scanning%20identifies%20multiple%20diseases%20and%20disease%20severity%20phenotypes%20associated%20with%20HLA%20variants&author=JH%20Karnes&author=L%20Bastarache&author=CM%20Shaffer&author=S%20Gaudieri&author=Y%20Xu&volume=9&publication_year=2017&pmid=28490672&doi=10.1126/scitranslmed.aai8708&)

26. Wei WQ, Bastarache LA, Carroll RJ, Marlo JE, Osterman TJ, Gamazon ER, Cox NJ, Roden DM and Denny JC. Evaluating phecodes, clinical classification software, and ICD-9-CM codes for phenome-wide association studies in the electronic health record. PloS one. 2017;12:e0175508.  [DOI](https://doi.org/10.1371/journal.pone.0175508) | [PMC free article](/articles/PMC5501393/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28686612/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PloS%20one&title=Evaluating%20phecodes,%20clinical%20classification%20software,%20and%20ICD-9-CM%20codes%20for%20phenome-wide%20association%20studies%20in%20the%20electronic%20health%20record&author=WQ%20Wei&author=LA%20Bastarache&author=RJ%20Carroll&author=JE%20Marlo&author=TJ%20Osterman&volume=12&publication_year=2017&pages=e0175508&pmid=28686612&doi=10.1371/journal.pone.0175508&)

27. Purcell S, Neale B, Todd-Brown K, Thomas L, Ferreira MA, Bender D, Maller J, Sklar P, de Bakker PI, Daly MJ and Sham PC. PLINK: a tool set for whole-genome association and population-based linkage analyses. Am J Hum Genet. 2007;81:559–575.  [DOI](https://doi.org/10.1086/519795) | [PMC free article](/articles/PMC1950838/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17701901/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=PLINK:%20a%20tool%20set%20for%20whole-genome%20association%20and%20population-based%20linkage%20analyses&author=S%20Purcell&author=B%20Neale&author=K%20Todd-Brown&author=L%20Thomas&author=MA%20Ferreira&volume=81&publication_year=2007&pages=559-575&pmid=17701901&doi=10.1086/519795&)

28. Hancock DB, Eijgelsheim M, Wilk JB, Gharib SA, Loehr LR, Marciante KD, Franceschini N, van Durme YM, Chen TH, Barr RG, Schabath MB, Couper DJ, Brusselle GG, Psaty BM, van Duijn CM, Rotter JI, Uitterlinden AG, Hofman A, Punjabi NM, Rivadeneira F, Morrison AC, Enright PL, North KE, Heckbert SR, Lumley T, Stricker BH, O’Connor GT and London SJ. Meta-analyses of genome-wide association studies identify multiple loci associated with pulmonary function. Nat Genet. 2010;42:45–52.  [DOI](https://doi.org/10.1038/ng.500) | [PMC free article](/articles/PMC2832852/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20010835/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Meta-analyses%20of%20genome-wide%20association%20studies%20identify%20multiple%20loci%20associated%20with%20pulmonary%20function&author=DB%20Hancock&author=M%20Eijgelsheim&author=JB%20Wilk&author=SA%20Gharib&author=LR%20Loehr&volume=42&publication_year=2010&pages=45-52&pmid=20010835&doi=10.1038/ng.500&)

29. Patterson N, Price AL and Reich D. Population structure and eigenanalysis. PLoS Genet. 2006;2:e190.  [DOI](https://doi.org/10.1371/journal.pgen.0020190) | [PMC free article](/articles/PMC1713260/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17194218/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet&title=Population%20structure%20and%20eigenanalysis&author=N%20Patterson&author=AL%20Price&author=D%20Reich&volume=2&publication_year=2006&pages=e190&pmid=17194218&doi=10.1371/journal.pgen.0020190&)

30. Price AL, Patterson NJ, Plenge RM, Weinblatt ME, Shadick NA and Reich D. Principal components analysis corrects for stratification in genome-wide association studies. Nat Genet. 2006;38:904–909.  [DOI](https://doi.org/10.1038/ng1847) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16862161/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Principal%20components%20analysis%20corrects%20for%20stratification%20in%20genome-wide%20association%20studies&author=AL%20Price&author=NJ%20Patterson&author=RM%20Plenge&author=ME%20Weinblatt&author=NA%20Shadick&volume=38&publication_year=2006&pages=904-909&pmid=16862161&doi=10.1038/ng1847&)

31. O’Connell J, Gurdasani D, Delaneau O, Pirastu N, Ulivi S, Cocca M, Traglia M, Huang J, Huffman JE, Rudan I, McQuillan R, Fraser RM, Campbell H, Polasek O, Asiki G, Ekoru K, Hayward C, Wright AF, Vitart V, Navarro P, Zagury JF, Wilson JF, Toniolo D, Gasparini P, Soranzo N, Sandhu MS and Marchini J. A general approach for haplotype phasing across the full spectrum of relatedness. PLoS Genet. 2014;10:e1004234.  [DOI](https://doi.org/10.1371/journal.pgen.1004234) | [PMC free article](/articles/PMC3990520/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24743097/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet&title=A%20general%20approach%20for%20haplotype%20phasing%20across%20the%20full%20spectrum%20of%20relatedness&author=J%20O%E2%80%99Connell&author=D%20Gurdasani&author=O%20Delaneau&author=N%20Pirastu&author=S%20Ulivi&volume=10&publication_year=2014&pages=e1004234&pmid=24743097&doi=10.1371/journal.pgen.1004234&)

32. Howie B, Fuchsberger C, Stephens M, Marchini J and Abecasis GR. Fast and accurate genotype imputation in genome-wide association studies through pre-phasing. Nat Genet. 2012;44:955–959.  [DOI](https://doi.org/10.1038/ng.2354) | [PMC free article](/articles/PMC3696580/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22820512/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Fast%20and%20accurate%20genotype%20imputation%20in%20genome-wide%20association%20studies%20through%20pre-phasing&author=B%20Howie&author=C%20Fuchsberger&author=M%20Stephens&author=J%20Marchini&author=GR%20Abecasis&volume=44&publication_year=2012&pages=955-959&pmid=22820512&doi=10.1038/ng.2354&)

33. Marchini J, Howie B, Myers S, McVean G and Donnelly P. A new multipoint method for genome-wide association studies by imputation of genotypes. Nat Genet. 2007;39:906–913.  [DOI](https://doi.org/10.1038/ng2088) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17572673/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=A%20new%20multipoint%20method%20for%20genome-wide%20association%20studies%20by%20imputation%20of%20genotypes&author=J%20Marchini&author=B%20Howie&author=S%20Myers&author=G%20McVean&author=P%20Donnelly&volume=39&publication_year=2007&pages=906-913&pmid=17572673&doi=10.1038/ng2088&)

34. Willer CJ, Li Y and Abecasis GR. METAL: fast and efficient meta-analysis of genomewide association scans. Bioinformatics. 2010;26:2190–2191.  [DOI](https://doi.org/10.1093/bioinformatics/btq340) | [PMC free article](/articles/PMC2922887/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20616382/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=METAL:%20fast%20and%20efficient%20meta-analysis%20of%20genomewide%20association%20scans&author=CJ%20Willer&author=Y%20Li&author=GR%20Abecasis&volume=26&publication_year=2010&pages=2190-2191&pmid=20616382&doi=10.1093/bioinformatics/btq340&)

35. Carroll RJ, Bastarache L and Denny JC. R PheWAS: data analysis and plotting tools for phenome-wide association studies in the R environment. Bioinformatics. 2014;30:2375–2376.  [DOI](https://doi.org/10.1093/bioinformatics/btu197) | [PMC free article](/articles/PMC4133579/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24733291/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=R%20PheWAS:%20data%20analysis%20and%20plotting%20tools%20for%20phenome-wide%20association%20studies%20in%20the%20R%20environment&author=RJ%20Carroll&author=L%20Bastarache&author=JC%20Denny&volume=30&publication_year=2014&pages=2375-2376&pmid=24733291&doi=10.1093/bioinformatics/btu197&)

36. Genomes Project C, Auton A, Brooks LD, Durbin RM, Garrison EP, Kang HM, Korbel JO, Marchini JL, McCarthy S, McVean GA and Abecasis GR. A global reference for human genetic variation. Nature. 2015;526:68–74.  [DOI](https://doi.org/10.1038/nature15393) | [PMC free article](/articles/PMC4750478/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26432245/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=A%20global%20reference%20for%20human%20genetic%20variation&author=C%20Genomes%20Project&author=A%20Auton&author=LD%20Brooks&author=RM%20Durbin&author=EP%20Garrison&volume=526&publication_year=2015&pages=68-74&pmid=26432245&doi=10.1038/nature15393&)

37. Barlera S, Specchia C, Farrall M, Chiodini BD, Franzosi MG, Rust S, Green F, Nicolis EB, Peden J, Assmann G, Collins R, Hamsten A, Tognoni G, Watkins H and Consortium P. Multiple QTL influence the serum Lp(a) concentration: a genome-wide linkage screen in the PROCARDIS study. Eur J Hum Genet. 2007;15:221–227.  [DOI](https://doi.org/10.1038/sj.ejhg.5201732) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17133260/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Hum%20Genet&title=Multiple%20QTL%20influence%20the%20serum%20Lp(a)%20concentration:%20a%20genome-wide%20linkage%20screen%20in%20the%20PROCARDIS%20study&author=S%20Barlera&author=C%20Specchia&author=M%20Farrall&author=BD%20Chiodini&author=MG%20Franzosi&volume=15&publication_year=2007&pages=221-227&pmid=17133260&doi=10.1038/sj.ejhg.5201732&)

38. Berglund L and Ramakrishnan R. Lipoprotein(a): an elusive cardiovascular risk factor. Arterioscler Thromb Vasc Biol. 2004;24:2219–2226.  [DOI](https://doi.org/10.1161/01.ATV.0000144010.55563.63) | [PMC free article](/articles/PMC3275911/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15345512/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arterioscler%20Thromb%20Vasc%20Biol&title=Lipoprotein(a):%20an%20elusive%20cardiovascular%20risk%20factor&author=L%20Berglund&author=R%20Ramakrishnan&volume=24&publication_year=2004&pages=2219-2226&pmid=15345512&doi=10.1161/01.ATV.0000144010.55563.63&)

39. Sandholzer C, Hallman DM, Saha N, Sigurdsson G, Lackner C, Csaszar A, Boerwinkle E and Utermann G. Effects of the apolipoprotein(a) size polymorphism on the lipoprotein(a) concentration in 7 ethnic groups. Hum Genet. 1991;86:607–614.  [DOI](https://doi.org/10.1007/BF00201550) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/2026424/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Genet&title=Effects%20of%20the%20apolipoprotein(a)%20size%20polymorphism%20on%20the%20lipoprotein(a)%20concentration%20in%207%20ethnic%20groups&author=C%20Sandholzer&author=DM%20Hallman&author=N%20Saha&author=G%20Sigurdsson&author=C%20Lackner&volume=86&publication_year=1991&pages=607-614&pmid=2026424&doi=10.1007/BF00201550&)

40. Clarke R, Peden JF, Hopewell JC, Kyriakou T, Goel A, Heath SC, Parish S, Barlera S, Franzosi MG, Rust S, Bennett D, Silveira A, Malarstig A, Green FR, Lathrop M, Gigante B, Leander K, de Faire U, Seedorf U, Hamsten A, Collins R, Watkins H, Farrall M and Consortium P. Genetic variants associated with Lp(a) lipoprotein level and coronary disease. N Engl J Med. 2009;361:2518–2528.  [DOI](https://doi.org/10.1056/NEJMoa0902604) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20032323/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Genetic%20variants%20associated%20with%20Lp(a)%20lipoprotein%20level%20and%20coronary%20disease&author=R%20Clarke&author=JF%20Peden&author=JC%20Hopewell&author=T%20Kyriakou&author=A%20Goel&volume=361&publication_year=2009&pages=2518-2528&pmid=20032323&doi=10.1056/NEJMoa0902604&)

41. Kamstrup PR, Benn M, Tybjaerg-Hansen A and Nordestgaard BG. Extreme lipoprotein(a) levels and risk of myocardial infarction in the general population: the Copenhagen City Heart Study. Circulation. 2008;117:176–184.  [DOI](https://doi.org/10.1161/CIRCULATIONAHA.107.715698) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18086931/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circulation&title=Extreme%20lipoprotein(a)%20levels%20and%20risk%20of%20myocardial%20infarction%20in%20the%20general%20population:%20the%20Copenhagen%20City%20Heart%20Study&author=PR%20Kamstrup&author=M%20Benn&author=A%20Tybjaerg-Hansen&author=BG%20Nordestgaard&volume=117&publication_year=2008&pages=176-184&pmid=18086931&doi=10.1161/CIRCULATIONAHA.107.715698&)

42. Bennet A, Di Angelantonio E, Erqou S, Eiriksdottir G, Sigurdsson G, Woodward M, Rumley A, Lowe GD, Danesh J and Gudnason V. Lipoprotein(a) levels and risk of future coronary heart disease: large-scale prospective data. Arch Intern Med. 2008;168:598–608.  [DOI](https://doi.org/10.1001/archinte.168.6.598) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18362252/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arch%20Intern%20Med&title=Lipoprotein(a)%20levels%20and%20risk%20of%20future%20coronary%20heart%20disease:%20large-scale%20prospective%20data&author=A%20Bennet&author=E%20Di%20Angelantonio&author=S%20Erqou&author=G%20Eiriksdottir&author=G%20Sigurdsson&volume=168&publication_year=2008&pages=598-608&pmid=18362252&doi=10.1001/archinte.168.6.598&)

43. Khera AV, Everett BM, Caulfield MP, Hantash FM, Wohlgemuth J, Ridker PM and Mora S. Lipoprotein(a) concentrations, rosuvastatin therapy, and residual vascular risk: an analysis from the JUPITER Trial (Justification for the Use of Statins in Prevention: an Intervention Trial Evaluating Rosuvastatin). Circulation. 2014;129:635–642.  [DOI](https://doi.org/10.1161/CIRCULATIONAHA.113.004406) | [PMC free article](/articles/PMC3946056/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24243886/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circulation&title=Lipoprotein(a)%20concentrations,%20rosuvastatin%20therapy,%20and%20residual%20vascular%20risk:%20an%20analysis%20from%20the%20JUPITER%20Trial%20(Justification%20for%20the%20Use%20of%20Statins%20in%20Prevention:%20an%20Intervention%20Trial%20Evaluating%20Rosuvastatin)&author=AV%20Khera&author=BM%20Everett&author=MP%20Caulfield&author=FM%20Hantash&author=J%20Wohlgemuth&volume=129&publication_year=2014&pages=635-642&pmid=24243886&doi=10.1161/CIRCULATIONAHA.113.004406&)

44. Vavlukis M, Mladenovska K, Daka A, Dimovski A, Domazetovska S, Kuzmanovska S and Kedev S. Effects of Rosuvastatin Versus Atorvastatin, Alone or in Combination, on Lipoprotein (a). Ann Pharmacother. 2016;50:609–615.  [DOI](https://doi.org/10.1177/1060028016652415) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27242350/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Pharmacother&title=Effects%20of%20Rosuvastatin%20Versus%20Atorvastatin,%20Alone%20or%20in%20Combination,%20on%20Lipoprotein%20(a)&author=M%20Vavlukis&author=K%20Mladenovska&author=A%20Daka&author=A%20Dimovski&author=S%20Domazetovska&volume=50&publication_year=2016&pages=609-615&pmid=27242350&doi=10.1177/1060028016652415&)

45. Baigent C, Keech A, Kearney PM, Blackwell L, Buck G, Pollicino C, Kirby A, Sourjina T, Peto R, Collins R, Simes R and Cholesterol Treatment Trialists C. Efficacy and safety of cholesterol-lowering treatment: prospective meta-analysis of data from 90,056 participants in 14 randomised trials of statins. Lancet. 2005;366:1267–1278.  [DOI](https://doi.org/10.1016/S0140-6736(05)67394-1) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16214597/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Efficacy%20and%20safety%20of%20cholesterol-lowering%20treatment:%20prospective%20meta-analysis%20of%20data%20from%2090,056%20participants%20in%2014%20randomised%20trials%20of%20statins&author=C%20Baigent&author=A%20Keech&author=PM%20Kearney&author=L%20Blackwell&author=G%20Buck&volume=366&publication_year=2005&pages=1267-1278&pmid=16214597&doi=10.1016/S0140-6736(05)67394-1&)

46. Goff DC Jr., Lloyd-Jones DM, Bennett G, Coady S, D’Agostino RB , Gibbons R, Greenland P, Lackland DT, Levy D, O’Donnell CJ, Robinson JG, Schwartz JS, Shero ST, Smith SC Jr., Sorlie P, Stone NJ, Wilson PW, Jordan HS, Nevo L, Wnek J, Anderson JL, Halperin JL, Albert NM, Bozkurt B, Brindis RG, Curtis LH, DeMets D, Hochman JS, Kovacs RJ, Ohman EM, Pressler SJ, Sellke FW, Shen WK, Smith SC Jr., Tomaselli GF and American College of Cardiology/American Heart Association Task Force on Practice G. 2013 ACC/AHA guideline on the assessment of cardiovascular risk: a report of the American College of Cardiology/American Heart Association Task Force on Practice Guidelines. Circulation. 2014;129:S49-73.  [DOI](https://doi.org/10.1161/01.cir.0000437741.48606.98) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24222018/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circulation&title=,&author=DC%20Goff&author=DM%20Lloyd-Jones&author=G%20Bennett&author=S%20Coady&author=RB%20D%E2%80%99Agostino&volume=129&publication_year=2014&pages=S49-73&pmid=24222018&doi=10.1161/01.cir.0000437741.48606.98&)

47. Takagi H and Umemoto T. Atorvastatin decreases lipoprotein(a): a meta-analysis of randomized trials. Int J Cardiol. 2012;154:183–186.  [DOI](https://doi.org/10.1016/j.ijcard.2011.09.060) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21996415/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Cardiol&title=Atorvastatin%20decreases%20lipoprotein(a):%20a%20meta-analysis%20of%20randomized%20trials&author=H%20Takagi&author=T%20Umemoto&volume=154&publication_year=2012&pages=183-186&pmid=21996415&doi=10.1016/j.ijcard.2011.09.060&)

48. Tsimikas S A Test in Context: Lipoprotein(a): Diagnosis, Prognosis, Controversies, and Emerging Therapies. J Am Coll Cardiol. 2017;69:692–711.  [DOI](https://doi.org/10.1016/j.jacc.2016.11.042) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28183512/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Am%20Coll%20Cardiol&title=A%20Test%20in%20Context:%20Lipoprotein(a):%20Diagnosis,%20Prognosis,%20Controversies,%20and%20Emerging%20Therapies&author=S%20Tsimikas&volume=69&publication_year=2017&pages=692-711&pmid=28183512&doi=10.1016/j.jacc.2016.11.042&)

49. Schwartz GG, Ballantyne CM, Barter PJ, Kallend D, Leiter LA, Leitersdorf E, McMurray JJV, Nicholls SJ, Olsson AG, Shah PK, Tardif JC and Kittelson J. Association of Lipoprotein(a) With Risk of Recurrent Ischemic Events Following Acute Coronary Syndrome: Analysis of the dal-Outcomes Randomized Clinical Trial. JAMA Cardiol. 2018;3:164–168  [DOI](https://doi.org/10.1001/jamacardio.2017.3833) | [PMC free article](/articles/PMC5838593/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29071331/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA%20Cardiol&title=Association%20of%20Lipoprotein(a)%20With%20Risk%20of%20Recurrent%20Ischemic%20Events%20Following%20Acute%20Coronary%20Syndrome:%20Analysis%20of%20the%20dal-Outcomes%20Randomized%20Clinical%20Trial&author=GG%20Schwartz&author=CM%20Ballantyne&author=PJ%20Barter&author=D%20Kallend&author=LA%20Leiter&volume=3&publication_year=2018&pages=164-168&pmid=29071331&doi=10.1001/jamacardio.2017.3833&)

50. Lu W, Cheng YC, Chen K, Wang H, Gerhard GS, Still CD, Chu X, Yang R, Parihar A, O’Connell JR, Pollin TI, Angles-Cano E, Quon MJ, Mitchell BD, Shuldiner AR and Fu M. Evidence for several independent genetic variants affecting lipoprotein (a) cholesterol levels. Hum Mol Genet. 2015;24:2390–400.  [DOI](https://doi.org/10.1093/hmg/ddu731) | [PMC free article](/articles/PMC4380064/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25575512/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mol%20Genet&title=Evidence%20for%20several%20independent%20genetic%20variants%20affecting%20lipoprotein%20(a)%20cholesterol%20levels&author=W%20Lu&author=YC%20Cheng&author=K%20Chen&author=H%20Wang&author=GS%20Gerhard&volume=24&publication_year=2015&pages=2390-400&pmid=25575512&doi=10.1093/hmg/ddu731&)

51. Consortium GT. Human genomics. The Genotype-Tissue Expression (GTEx) pilot analysis: multitissue gene regulation in humans. Science. 2015;348:648–660.  [DOI](https://doi.org/10.1126/science.1262110) | [PMC free article](/articles/PMC4547484/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25954001/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=Human%20genomics.%20The%20Genotype-Tissue%20Expression%20(GTEx)%20pilot%20analysis:%20multitissue%20gene%20regulation%20in%20humans&author=GT%20Consortium&volume=348&publication_year=2015&pages=648-660&pmid=25954001&doi=10.1126/science.1262110&)

52. Gamazon ER, Wheeler HE, Shah KP, Mozaffari SV, Aquino-Michaels K, Carroll RJ, Eyler AE, Denny JC, Consortium GT, Nicolae DL, Cox NJ and Im HK. A gene-based association method for mapping traits using reference transcriptome data. Nat Genet. 2015;47:1091–1098.  [DOI](https://doi.org/10.1038/ng.3367) | [PMC free article](/articles/PMC4552594/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26258848/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=A%20gene-based%20association%20method%20for%20mapping%20traits%20using%20reference%20transcriptome%20data&author=ER%20Gamazon&author=HE%20Wheeler&author=KP%20Shah&author=SV%20Mozaffari&author=K%20Aquino-Michaels&volume=47&publication_year=2015&pages=1091-1098&pmid=26258848&doi=10.1038/ng.3367&)

53. Boffa MB. Emerging Therapeutic Options for Lowering of Lipoprotein(a): Implications for Prevention of Cardiovascular Disease. Curr Atheroscler Rep. 2016;18:69.  [DOI](https://doi.org/10.1007/s11883-016-0622-1) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27761705/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Atheroscler%20Rep&title=Emerging%20Therapeutic%20Options%20for%20Lowering%20of%20Lipoprotein(a):%20Implications%20for%20Prevention%20of%20Cardiovascular%20Disease&author=MB%20Boffa&volume=18&publication_year=2016&pages=69&pmid=27761705&doi=10.1007/s11883-016-0622-1&)

54. Schuster V, Hugle B and Tefs K. Plasminogen deficiency. J Thromb Haemost. 2007;5:2315–22.  [DOI](https://doi.org/10.1111/j.1538-7836.2007.02776.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17900274/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Thromb%20Haemost&title=Plasminogen%20deficiency&author=V%20Schuster&author=B%20Hugle&author=K%20Tefs&volume=5&publication_year=2007&pages=2315-22&pmid=17900274&doi=10.1111/j.1538-7836.2007.02776.x&)

55. McColl MD, Tait RC, Walker ID, McCall F, Conkie JA and Perry DJ. Asymptomatic coinheritance of heterozygous plasminogen deficiency and the factor VLeiden mutation. Blood Coagul Fibrinolysis. 1997;8:195–199.  [DOI](https://doi.org/10.1097/00001721-199704000-00007) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9167021/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood%20Coagul%20Fibrinolysis&title=Asymptomatic%20coinheritance%20of%20heterozygous%20plasminogen%20deficiency%20and%20the%20factor%20VLeiden%20mutation&author=MD%20McColl&author=RC%20Tait&author=ID%20Walker&author=F%20McCall&author=JA%20Conkie&volume=8&publication_year=1997&pages=195-199&pmid=9167021&doi=10.1097/00001721-199704000-00007&)

56. Sartori MT, Simioni P, Patrassi GM, Theodoridis P, Tormene D and Girolami A. Combined heterozygous plasminogen deficiency and factor V Leiden defect in the same kindred. Clin Appl Thromb Hemost. 2000;6:36–40.  [DOI](https://doi.org/10.1177/107602960000600106) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10726047/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Appl%20Thromb%20Hemost&title=Combined%20heterozygous%20plasminogen%20deficiency%20and%20factor%20V%20Leiden%20defect%20in%20the%20same%20kindred&author=MT%20Sartori&author=P%20Simioni&author=GM%20Patrassi&author=P%20Theodoridis&author=D%20Tormene&volume=6&publication_year=2000&pages=36-40&pmid=10726047&doi=10.1177/107602960000600106&)

57. Cheng X, Wang M, Jiang M, Bhugul PA, Hao X and Yang L. A protein C and plasminogen compound heterozygous mutation and a compound heterozygote of protein C in two related Chinese families. Blood Coagul Fibrinolysis. 2016; 27:838–844.  [DOI](https://doi.org/10.1097/MBC.0000000000000579) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27314862/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood%20Coagul%20Fibrinolysis&title=A%20protein%20C%20and%20plasminogen%20compound%20heterozygous%20mutation%20and%20a%20compound%20heterozygote%20of%20protein%20C%20in%20two%20related%20Chinese%20families&author=X%20Cheng&author=M%20Wang&author=M%20Jiang&author=PA%20Bhugul&author=X%20Hao&volume=27&publication_year=2016&pages=838-844&pmid=27314862&doi=10.1097/MBC.0000000000000579&)
