# Association of Genetic Variants in NUDT15 With Thiopurine-Induced Myelosuppression in Patients With Inflammatory Bowel Disease

## Metadata
**Authors:** Gareth J Walker, James W Harrison, Graham A Heap, Michiel D Voskuil, Vibeke Andersen, Carl A Anderson, Ashwin N Ananthakrishnan, Jeffrey C Barrett, Laurent Beaugerie, Claire M Bewshea, Andy T Cole, Fraser R Cummings, Mark J Daly, Pierre Ellul, Richard N Fedorak, Eleonora A M Festen, Timothy H Florin, Daniel R Gaya, Jonas Halfvarson, Ailsa L Hart, Neel M Heerasing, Peter Hendy, Peter M Irving, Samuel E Jones, Jukka Koskela, James O Lindsay, John C Mansfield, Dermot McGovern, Miles Parkes, Richard C G Pollok, Subramaniam Ramakrishnan, David S Rampton, Manuel A Rivas, Richard K Russell, Michael Schultz, Shaji Sebastian, Philippe Seksik, Abhey Singh, Kenji So, Harry Sokol, Kavitha Subramaniam, Anthony Todd, Vito Annese, Rinse K Weersma, Ramnik Xavier, Rebecca Ward, Michael N Weedon, James R Goodhand, Nicholas A Kennedy, Tariq Ahmad
**Journal:** JAMA
**Date:** 2019 Feb 26
**DOI:** [10.1001/jama.2019.0709](https://doi.org/10.1001/jama.2019.0709)
**PMID:** 30806694
**PMCID:** PMC6439872
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6439872/

## Abstract

**Question:** 
What genetic variants are associated with thiopurine-induced myelosuppression among patients of European ancestry with inflammatory bowel disease?

**Findings:** 
In this case-control study that used whole-exome sequence data from 961 thiopurine-exposed patients of European ancestry with inflammatory bowel disease, 3 coding NUDT15 variants, including a 6–base pair in-frame deletion (odds ratio, 38.2), were identified that were associated with thiopurine-induced myelosuppression.

**Meaning:** 
Among patients of European ancestry with inflammatory bowel disease, variants in NUDT15 were associated with increased risk of thiopurine-induced myelosuppression.

### Question

What genetic variants are associated with thiopurine-induced myelosuppression among patients of European ancestry with inflammatory bowel disease?

### Findings

In this case-control study that used whole-exome sequence data from 961 thiopurine-exposed patients of European ancestry with inflammatory bowel disease, 3 coding *NUDT15* variants, including a 6–base pair in-frame deletion (odds ratio, 38.2), were identified that were associated with thiopurine-induced myelosuppression.

### Meaning

Among patients of European ancestry with inflammatory bowel disease, variants in *NUDT15* were associated with increased risk of thiopurine-induced myelosuppression.

### Importance

Use of thiopurines may be limited by myelosuppression. *TPMT* pharmacogenetic testing identifies only 25% of at-risk patients of European ancestry. Among patients of East Asian ancestry, *NUDT15* variants are associated with thiopurine-induced myelosuppression (TIM).

### Objective

To identify genetic variants associated with TIM among patients of European ancestry with inflammatory bowel disease (IBD).

### Design, Setting, and Participants

Case-control study of 491 patients affected by TIM and 679 thiopurine-tolerant unaffected patients who were recruited from 89 international sites between March 2012 and November 2015. Genome-wide association studies (GWAS) and exome-wide association studies (EWAS) were conducted in patients of European ancestry. The replication cohort comprised 73 patients affected by TIM and 840 thiopurine-tolerant unaffected patients.

### Exposures

Genetic variants associated with TIM.

### Main Outcomes and Measures

Thiopurine-induced myelosuppression, defined as a decline in absolute white blood cell count to 2.5 × 10^9^/L or less or a decline in absolute neutrophil cell count to 1.0 × 10^9^/L or less leading to a dose reduction or drug withdrawal.

### Results

Among 1077 patients (398 affected and 679 unaffected; median age at IBD diagnosis, 31.0 years [interquartile range, 21.2 to 44.1 years]; 540 [50%] women; 602 [56%] diagnosed as having Crohn disease), 919 (311 affected and 608 unaffected) were included in the GWAS analysis and 961 (328 affected and 633 unaffected) in the EWAS analysis. The GWAS analysis confirmed association of *TPMT* (chromosome 6, [rs11969064](http://www.ncbi.nlm.nih.gov/snp/11969064)) with TIM (30.5% [95/311] affected vs 16.4% [100/608] unaffected patients; odds ratio [OR], 2.3 [95% CI, 1.7 to 3.1], *P* = 5.2 × 10^−9^). The EWAS analysis demonstrated an association with an in-frame deletion in *NUDT15* (chromosome 13, [rs746071566](http://www.ncbi.nlm.nih.gov/snp/746071566)) and TIM (5.8% [19/328] affected vs 0.2% [1/633] unaffected patients; OR, 38.2 [95% CI, 5.1 to 286.1], *P* = 1.3 × 10^−8^), which was replicated in a different cohort (2.7% [2/73] affected vs 0.2% [2/840] unaffected patients; OR, 11.8 [95% CI, 1.6 to 85.0], *P* = .03). Carriage of any of 3 coding *NUDT15* variants was associated with an increased risk (OR, 27.3 [95% CI, 9.3 to 116.7], *P* = 1.1 × 10^−7^) of TIM, independent of *TPMT* genotype and thiopurine dose.

### Conclusions and Relevance

Among patients of European ancestry with IBD, variants in *NUDT15* were associated with increased risk of TIM. These findings suggest that *NUDT15* genotyping may be considered prior to initiation of thiopurine therapy; however, further study including additional validation in independent cohorts is required.

## Introduction

Thiopurines (mercaptopurine and its prodrug azathioprine) are commonly used in the management of inflammatory bowel disease (IBD). However, approximately 15% of patients develop adverse drug reactions that necessitate drug withdrawal.^1,2^ Thiopurine-induced myelosuppression (TIM) has a cumulative incidence of 7% and usually occurs within a few weeks of starting the drug.^1^ Most patients are asymptomatic, but serious opportunistic infections may occur and there is an estimated mortality of 1%.^1^

The enzyme thiopurine S-methyltransferase (TPMT) converts thiopurines to methylated metabolites, reducing the production of active 6-thioguanine nucleotides.^3^ Genetic variation in the *TPMT* gene (RefSeqGene [NG_012137](http://www.ncbi.nlm.nih.gov/nuccore/1994123092).2) can result in decreased TPMT enzyme activity and higher production of 6-thioguanine nucleotides, predisposing patients to bone marrow suppression.^1,3,4^ Pretreatment testing of *TPMT* is recommended by the US Food and Drug Administration to identify patients at risk of TIM.^5^ Among patients with reduced TPMT activity, the drug may be avoided or the dose reduced.^6^ However, *TPMT* variants are only found in 25% of patients of European ancestry affected by TIM, suggesting the presence of other genetic and environmental determinants.^6,7^ Studies in patients of East Asian ancestry^8,9^ and other populations^10,11,12,13,14^ have identified variants in nudix hydrolase 15 (*NUDT15*; RefSeqGene [NG_047021](http://www.ncbi.nlm.nih.gov/nuccore/1024846744).1) as risk factors for TIM. Although a novel *NUDT15* variant ([rs746071566](http://www.ncbi.nlm.nih.gov/snp/746071566), p.Gly17_Val18del) was described by Moriyama et al^10^ in a single pediatric patient of European ancestry affected by TIM, the association of the *NUDT15* genetic variation with TIM in this population has not been fully evaluated.

The primary objective of this study was to investigate the association between genetic variants and TIM in patients of European ancestry with IBD. It was hypothesized that the frequency of these variants would be increased among patients affected by TIM and enriched in those with early-onset TIM (≤8 weeks from start of maximum dose).^1^

## Methods

### Study Design and Setting

The protocol was approved by the National Research Ethics Committee (11/SW/0222, Exeter pharmacogenetic PRED4 program and STB1, Exeter IBD Genetics cohort, England). All participants provided informed written consent. A retrospective case-control study of the association of genetic variants with TIM was designed as part of the Exeter pharmacogenetic PRED4 program, which aims to investigate the genetic basis of serious adverse reactions among patients prescribed commonly used drugs in gastroenterology ([http://www.ibdresearch.co.uk](http://www.ibdresearch.co.uk)).^15,16^ Platforms for both genome-wide association studies (GWAS) and exome-wide association studies (EWAS) were used to investigate common and rare genetic variation, respectively.

### Study Populations and Case Definition

Thiopurine-induced myelosuppression cases (affected patients) were recruited from 82 sites within the United Kingdom and from 7 sites outside the United Kingdom between March 2012 and November 2015 and not followed up after their single study visit. Individuals affected by TIM were identified through clinical encounters, systematic searches of electronic records, recall via the Medicines and Healthcare Products Regulatory Agency Yellow Card Scheme, and by direct advertising to patients.

Inclusion criteria included all of the following: diagnosis of IBD, history of thiopurine exposure during the 7 days prior to the onset of TIM, decline in absolute white blood cell count to 2.5 × 10^9^/L or less or decline in absolute neutrophil cell count to 1.0 × 10^9^/L or less, and determination by the treating physician that use of a thiopurine was the likely cause of TIM and the dose was reduced or the drug was withdrawn.

Investigators at each site completed a custom-designed case report form (eAppendix 1 in the [Supplement](#note-JOI190011-1-s)) that captured the following data: patient demographics (age, weight, height, ethnicity, and smoking history), adverse drug reaction data (type of thiopurine, dose, drug start and stop date, full blood cell count parameters before, during, and after exposure to the drug, and full blood cell count normal range reference values), and IBD phenotype. Each patient was diagnosed as having IBD by his or her gastroenterologist using endoscopic data, histological data, radiological data, or a combination of these, and phenotyped according to the Montreal classification of IBD.^17^ The Montreal system classified the extent of ulcerative colitis as limited to the rectum (E1), distal to the splenic flexure (E2), or proximal to the splenic flexure (E3). For Crohn disease, patients were categorized by age at disease onset (A1: <17 years, A2: 17-40 years, or A3: >40 years), location of disease (L1: ileal, L2: colonic, or L3: ileocolonic), and disease behavior (B1: nonstricturing and nonpenetrating, B2: stricturing, or B3: penetrating).

Consistent with our prior pharmacogenetics studies,^15,16^ the case report forms of all recruited patients affected by TIM were reviewed independently by at least 4 gastroenterologists and assigned an adjudication category (eAppendices 2-4, eFigure 1, and eMethods in the [Supplement](#note-JOI190011-1-s)).^18^ Only patients assigned as definitely or probably affected by TIM were included in the discovery and replication analyses.

Thiopurine-exposed controls without TIM (unaffected patients) were identified from the Exeter IBD Genetics cohort recruited at the Royal Devon and Exeter Hospital (additional details appear in the eMethods in the [Supplement](#note-JOI190011-1-s)). In the final analyses, only patients with an absolute white blood cell count of 3.0 × 10^9^/L or greater and an absolute neutrophil cell count of 1.5 × 10^9^/L or greater for the duration of their treatment with a thiopurine were included.

The replication cohort met the identical inclusion criteria and included nonoverlapping patients from the same central study site (Royal Devon and Exeter Hospital) and patients from 4 new sites (Saint-Antoine Hospital in France, University Medical Center Groningen in the Netherlands, Cedars-Sinai Medical Center in the United States, and Massachusetts General Hospital in the United States). Patients at these new sites were identified from searches of preexisting genetics cohorts in April 2017. These sites had started recruitment in 2005 (Massachusetts General Hospital and Cedars-Sinai Medical Center), 2011 (University Medical Center Groningen), and 2013 (Saint-Antoine Hospital).

### Genetic Analysis

Details of genetic data generation and quality control prior to the GWAS and EWAS analyses appear in the eMethods in the [Supplement](#note-JOI190011-1-s). For the GWAS analysis, 245 185 variants were genotyped using the Illumina Infinium G4L GWAS array. Patients were excluded if they had variants with a call rate of less than 98%, had variants with a minor allele frequency of less than 1%, or had variants with a Hardy Weinberg equilibrium of *P* < 1 × 10^−6^ among unaffected patients. The principal component analysis was carried out using Genome-wide Complex Trait Analysis version 1.24^19^ to assess data from the 1000 Genomes Project.^20^ Only data from patients clustering with non-Finnish European patients were included. This process minimized the potential confounding effects of population stratification, which might have resulted in an association with variants and TIM even though the association was with a specific ethnicity, which was by chance overrepresentated or underrepresented among affected patients compared with unaffected patients.

We excluded patients of Finnish ancestry because their unique genetic background, which has occurred as a consequence of geographical and cultural isolation, has led to the enrichment of some disease-causing gene variants and losses of others. Other quality control measures included a sex-mismatch check (a method that used X chromosome homozygosity rates to determine sex and identify patients for whom the sex recorded in the case report form or phenotype database did not match the predicted sex based on genetic data) and relatedness checking (in which sample and pedigree integrity were both simultaneously examined by reconciling genomic data with self-reported relationships between patients).

After prephasing with the Eagle2 algorithm,^21^ imputation with the positional Burrows-Wheeler transform^22^ method was performed using phase 3 of the 1000 Genomes Project^20^ reference panel and the Wellcome Trust Sanger Imputation Service. Only single-nucleotide polymorphisms with a postimputation information score of less than 0.85 or a minor allele frequency of less than 0.01 were included. After all the quality control measures were implemented, 6 272 335 variants remained.

For the EWAS analysis, exonic regions were sequenced using the Illumina HiSeq platform (150 base-paired reads) and reads mapped to the Genome Reference Consortium human build 37 using the Burrows-Wheeler Alignment MEM algorithm.^23^ Each sample was sequenced to an average depth of 34 times with approximately 99% of the targeted regions covered by 1 time or greater, approximately 92% covered by 10 times or greater, and approximately 70% covered by 25 times or greater. Variants with a Hardy Weinberg equilibrium of *P* < 1 × 10^−6^ were excluded along with any variants that had a genotyping success rate of less than 0.98, a read depth of less than 10 times, or a genotype skew of *P* < 5 × 10^−9^ (binomial test). For quality control after the EWAS analysis, a quantile-quantile plot was used (eFigure 2 in the [Supplement](#note-JOI190011-1-s)).

### Statistical Analysis

#### Phenotype Comparisons

Continuous data were summarized using medians and interquartile ranges (IQRs) and were compared using the Mann-Whitney test. The estimate of the median difference between patients affected by TIM and unaffected patients and 95% CIs also were calculated using R version 3.5.1 (R Foundation for Statistical Computing). Categorical data were summarized as the number and percentage and compared using the Fisher exact test.

#### Primary Analyses

Associations for both the GWAS and EWAS analyses were determined using the Fisher exact test and PLINK version 1.9 (Cog Genomics). Manhattan plots were generated using R software to display negative log_10_ *P* values at each single-nucleotide polymorphism. A genome-wide significance threshold of *P* < 5 × 10^−8^ was deemed significant. Gene burden tests using PLINK sequence 0.10 and sequence kernel association tests^24^ were used to evaluate if an association existed between sets of rare variants across individual candidate genes and patients affected by TIM. Technical validation of the variants was carried out using Sanger sequencing (eMethods in the [Supplement](#note-JOI190011-1-s)). For the replication cohort, case adjudication, genotype data generation, genetic quality control, and other analyses were undertaken using the same platforms and methods as the discovery cohort. Replicated variants with a Fisher exact *P* < .05 were considered significant.

#### Exploratory Analyses

After finding an association with a variant and TIM, the final data set was examined for any other nonmonomorphic variants within *NUDT15* annotated as missense or as loss of function in the Genome Aggregation Database (gnomAD).^25^ Further missense variants were evaluated using in silico Protein Variation Effect Analyzer.^26^ Because the functional significance of this modeling is uncertain, only replicated *NUDT15* variants and those previously described in other TIM cohorts^8,9^ were used in subsequent genotype-phenotype analyses, multivariable logistic regression analyses, and clinical usefulness analyses.

Combinations of *TPMT* variants on the same chromosome have been reported as haplotypes; these were reconstructed using the Eagle2 algorithm^21^ and matched to the Clinical Pharmacogenetics Implementation Consortium definitions.^27^ Categorical *TPMT* enzyme activity (ie, absent, low, normal, or high) was measured in red blood cells using radiometric high-performance liquid chromatography as part of routine clinical practice. The relationship between *TPMT* haplotypes and enzyme activity was determined.

Genotype-phenotype interactions were explored using the Mann-Whitney test and the Fisher exact test. All statistical tests were 2-sided and *P* < .05 was considered significant. No adjustment of the *P *value was made for multiple comparisons of phenotype data; therefore, the results of these analyses should be considered exploratory. Weight-adjusted dose (milligrams per kilograms) was calculated using the following formulas for mercaptopurine (mercaptopurine dose in milligrams × 2.08/weight in kilograms) and azathioprine (azathioprine dose in milligrams/weight in kilograms).

A multivariable logistic regression analysis was undertaken to assess the independent associations of *NUDT15*, *TPMT*, and weight-adjusted thiopurine dose with risk of TIM. Time to TIM (stratified by genotype) was analyzed using the Mann-Whitney test.

The potential for clinical usefulness (sensitivity, specificity, negative and positive predictive values) of genotyping for variants associated with TIM was estimated using methods adapted from Tonk et al^28^ and de Graaff et al^29^ (eMethods in the [Supplement](#note-JOI190011-1-s)). The estimates assumed an overall risk of TIM of 7%,^1^ either avoidance of the drug (reducing risk of TIM to 0) or target dose reduction in those patients carrying deleterious variants (reducing risk of TIM to that seen in patients with the reference haplotype or genotype), the non-Finnish European population variant carrier frequency from gnomAD,^25^ and the odds ratio (OR) of TIM for the variant in multivariable logistic regression analysis. The 95% CIs for the number of patients needed to genotype were estimated using 10 000 bootstraps from the case-control cohort, from randomly generated estimates of the population with carriage of *NUDT15* and *TPMT* variants, and from TIM rates that were based on sampling from binomial distributions. For *TPMT*, detailed information appears in eMethods in the [Supplement](#note-JOI190011-1-s).

The prevalence of *NUDT15* variants in patients of other ancestry was explored using all affected patients and population data from gnomAD.^25^

## Results

### Study Overview

Participant flow through the study appears in [Figure 1](#joi190011f1). In this case-control study, 491 patients with IBD and TIM (affected patients) were recruited from 82 UK and 7 international sites between March 2012 and November 2015. One UK center recruited 843 thiopurine-exposed patients with IBD and no history of TIM (unaffected patients). Following the adjudication process, 1077 patients (398 affected and 679 unaffected patients) entered the final analysis.

### Figure 1. Flow Diagram and Study Overview of Case and Control Cohorts.

![Figure 1. ](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2f41/6439872/d2749d09dbec/jama-e190709-g001.jpg)

aIndicates non-Finnish European ancestry based on principal component analysis. bIndicates patients too closely related to each other. cIndicates a discrepancy between genetically determined sex and phenotype data.

After assessment using the genetic quality control measures, 70 affected patients were excluded (68 for ethnicity, 1 for relatedness, and 1 for sex mismatch) and 46 unaffected patients were excluded (31 for ethnicity, 13 for relatedness, and 2 for sex mismatch). In addition, for the GWAS analysis, 17 affected patients were excluded (10 due to failure of quality control genotyping and 7 to failure of genotyping) and 25 unaffected patients were excluded (23 due to failure of quality control genotyping and 2 to failure of genotyping). Thus, 919 patients (311 affected and 608 unaffected patients) were included in the GWAS and 961 patients (328 affected and 633 unaffected patients) were included in the EWAS analysis. Replication was conducted in 73 affected and 840 unaffected patients recruited from 5 international sites.

### Phenotype Comparisons

There were no differences in sex when comparing affected and unaffected patients (female 53.0% [211/398] vs 48.5% [329/679], respectively, *P* = .17; [Table 1](#joi190011t1)). There were no differences when comparing affected and unaffected patients by type of IBD diagnosis: Crohn disease (57.8% [230/398] vs 54.8% [372/679], respectively), ulcerative colitis (39.7% [158/398] vs 44.0% [299/679]), and IBD-unclassified (2.5% [10/398] vs 1.2% [8/679], *P* = .12).

### Table 1. Characteristics of Patients With Inflammatory Bowel Disease (IBD) by Thiopurine-Induced Myelosuppression (TIM) Status.

| Characteristic | Individuals Affected by TIM(n = 398)a | Thiopurine-TolerantUnaffected Individuals(n = 679) |
| --- | --- | --- |
| Sex, No. (%) |   |   |
| Female | 211 (53.0) | 329 (48.5) |
| Male | 187 (47.0) | 350 (51.5) |
| Type of IBD diagnosis, No. (%) |   |   |
| Crohn disease | 230 (57.8) | 372 (54.8) |
| Ulcerative colitis | 158 (39.7) | 299 (44.0) |
| IBD-unclassified | 10 (2.5) | 8 (1.2) |
| Age at diagnosis of IBD, median (IQR), yb | 30.1 (19.3-43.1) | 31.6 (22.2-44.7) |
| Weight-adjusted thiopurine dose, median (IQR), mg/kgc | 2.07 (1.69-2.45) | 1.84 (1.48-2.19) |
| Montreal Classification of IBD, No./total (%) d |  |  |
| Age range at diagnosis |   |   |
| A1: <17 y | 52/229 (22.7) | 23/300 (7.7) |
| A2: 17-40 y | 122/229 (53.3) | 235/300 (78.3) |
| A3: >40 y | 55/229 (24.0) | 42/300 (14.0) |
| Disease location |   |   |
| L1: ileal | 57/229 (24.9) | 132/300 (44.0) |
| L2: colonic | 74/229 (32.3) | 79/300 (26.3) |
| L3: ileocolonic | 98/229 (42.8) | 89/300 (29.7) |
| Behavior of IBD |   |   |
| B1: nonstricturing and nonpenetrating | 123/213 (57.7) | 175/297 (58.9) |
| B2: stricturing | 62/213 (29.1) | 82/297 (27.6) |
| B3: penetrating | 28/213 (13.1) | 40/297 (13.5) |
| Extent of ulcerative colitis and IBD-unclassified |   |   |
| E1: limited to the rectum | 15/160 (9.4) | 14/234 (6.0) |
| E2: distal to the splenic flexure | 73/160 (45.6) | 112/234 (47.9) |
| E3: proximal to the splenic flexure | 72/160 (45.0) | 108/234 (46.2) |

Table 1 Caption: Abbreviation: IQR, interquartile range. a Adjudicated prior to genomic quality control. b The estimate of difference was 2.3 (95% CI, 0.4-4.2). c Represents the maximum mercaptopurine or azathioprine equivalent dose prior to TIM and adjusted for weight: (mercaptopurine dose in milligrams × 2.08)/weight in kilograms) or (azathioprine dose in milligrams/weight in kilograms). The estimate of difference was −0.24 (95% CI, −0.32 to −0.17). d Details on the Montreal classification of IBD system reported by Silverberg et al.17

There were no differences in behavior of IBD when comparing affected and unaffected patients using the Montreal classification of IBD: B1 (nonstricturing and nonpenetrating, 57.7% [123/213] vs 58.9% [175/297], respectively), B2 (stricturing, 29.1% [62/213] vs 27.6% [82/297]), and B3 (penetrating, 13.1% [28/213] vs 13.5% [40/297], *P* = .94). There were no differences in the extent of ulcerative colitis and IBD-unclassified when comparing affected and unaffected patients using the Montreal Classification system: E1 (limited to the rectum, 9.4% [15/160] vs 6.0% [14/234], respectively), E2 (distal to the splenic flexure, 45.6% [73/160] vs 47.9% [112/234]), and E3 (proximal to the splenic flexure, 45.0% [72/160] vs 46.2% [108/234], *P* = .46).

In contrast, affected patients were younger at the time of IBD diagnosis (median, 30.1 years [IQR, 19.3-43.1 years]) compared with unaffected patients (median, 31.6 years [IQR, 22.2-44.7 years], *P* = .02) and received a higher weight-adjusted thiopurine dose (median, 2.07 mg/kg [IQR, 1.69-2.45 mg/kg] vs 1.84 mg/kg [IQR, 1.48-2.19 mg/kg], respectively, *P* < .001). In addition, affected patients with Crohn disease were more likely to have colonic or ileo-colonic disease than unaffected patients (L1 [ileal]: 24.9% [57/229] vs 44.0% [132/300], respectively; L2 [colonic]: 32.3% [74/229] vs 26.3% [79/300], and L3 [ileocolonic]: 42.8% [98/229] vs 29.7% [89/300], *P* < .001).

Among the 398 affected patients, 143 (36%) episodes of TIM occurred within 8 weeks of therapy with the maximum dose of thiopurine (eTable 1 in the [Supplement](#note-JOI190011-1-s)). The median time from commencement of thiopurine to TIM was 28.3 weeks (IQR, 9.0-81.1 weeks) and the median time from maximum dose of thiopurine to TIM was 14.7 weeks (IQR, 5.9-37.9 weeks). Phenotype data for the replication cohort appear in eTable 2 in the [Supplement](#note-JOI190011-1-s).

### Primary Analyses

#### GWAS Analysis

Data from 311 affected and 608 unaffected patients (eTable 3 in the [Supplement](#note-JOI190011-1-s)) were included in the GWAS discovery cohort. The association of TIM with *TPMT* ([rs11969064](http://www.ncbi.nlm.nih.gov/snp/11969064)) was confirmed in 30.5% (95/311) of affected patients compared with 16.4% (100/608) of unaffected patients (OR, 2.3 [95% CI, 1.7 to 3.1], *P* = 5.2 × 10^−9^; eFigure 3 in the [Supplement](#note-JOI190011-1-s)). This association was enriched in patients affected early (≤8 weeks of starting maximum thiopurine dose; OR, 4.0 [95% CI, 2.8 to 5.8], *P* = 1.8 × 10^−15^) compared with those affected later (OR, 1.6 [95% CI, 1.1 to 2.2], *P* = .01; eFigure 4 in the [Supplement](#note-JOI190011-1-s)). No other genetic associations with TIM exceeded the a priori threshold for statistical significance.

#### EWAS Analysis

Data from 328 affected and 633 unaffected patients were included in the EWAS discovery cohort (eTable 4 in the [Supplement](#note-JOI190011-1-s)). The EWAS analysis, which was performed to investigate the role of rare coding variants, revealed a TIM association with a 6–base pair in-frame deletion at position 48611918 of chromosome 13 in exon 1 of *NUDT15* ([rs746071566](http://www.ncbi.nlm.nih.gov/snp/746071566), p.Gly17_Val18del) for 5.8% (19/328) of the affected patients compared with 0.2% (1/633) of the unaffected patients (OR, 38.2 [95% CI, 5.1 to 286.1], *P* = 1.3 × 10^−8^; [Figure 2](#joi190011f2)). Compared with unaffected patients, the OR was 74.2 (95% CI, 9.6 to 573.5, *P* = 8.2 × 10^−10^) for patients affected with early-onset TIM and was 20.9 (95% CI, 2.6 to 170.1, *P* = 4.2 × 10^−4^) for patients affected with late-onset TIM. The patients affected with early-onset TIM were significantly enriched with the variant (OR, 3.6 [95% CI, 1.4 to 9.2], *P* = .005; eTable 5 in the [Supplement](#note-JOI190011-1-s)).

![Figure 2. ](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2f41/6439872/0f464972a1f5/jama-e190709-g002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=6439872_jama-e190709-g002.jpg)

Each colored dot represents a single variant within each respective chromosome. The negative log10 P value represents a Fisher exact test analysis between affected and unaffected patients. The orange dotted horizontal line indicates genome-wide significance at Fisher exact P = 5.0 × 10−8. Gene names correspond to the gene in closest proximity to the variant with the lowest P value at each locus if within 50 kilobase pairs.

The association of the p.Gly17_Val18del variant and TIM was confirmed in the replication cohort analysis among 2.7% (2/73) of the affected patients with IBD compared with 0.2% (2/840) of the unaffected patients (OR, 11.8 [95% CI, 1.6 to 85.0], *P* = .03). A duplication at this multiallelic site within *NUDT15* ([rs746071566](http://www.ncbi.nlm.nih.gov/snp/746071566), p.Gly17_Val18dup [also annotated as p.Val18_Val19insGlyVal]) was noted; however, the duplication did not meet genome-wide significance (1.5% [5/328] of affected patients vs 0.3% [2/633] of unaffected patients; OR, 5.2 [95% CI, 1.0 to 26.6], *P* = .04; [Table 2](#joi190011t2)). The only variant outside of *NUDT15* significantly associated with TIM in the exome sequencing data was [rs1800460](http://www.ncbi.nlm.nih.gov/snp/1800460) in *TPMT* (OR, 3.0 [95% CI, 2.0 to 4.3], *P* = 2.0 × 10^−8^). Gene burden testing did not identify any novel associations beyond *TPMT* and *NUDT15* (eTable 6 in the [Supplement](#note-JOI190011-1-s)).

| NUDT15 Identifiers |  |  |  |  | Individuals Affected by Thiopurine-Induced Myelosuppression (n = 328) |  |  | Thiopurine-TolerantUnaffected Individuals (n = 633) |  |  | Odds Ratio(95% CI) | P Valueb |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| Chromosome13 Position | SNP ID No. or Chromosome Position | Protein Sequence | VariantAllele | ReferenceAllele | No. ofVariant Heterozygotes | No. ofReference Homozygotes | Minor Allele Frequencya | No. ofVariant Heterozygotes | No. ofReference Homozygotes | Minor Allele Frequencya |  |  |
| 48611918c | rs746071566 | p.Gly17_Val18del | A | AGGAGTC | 19 | 304 | 0.029 | 1 | 630 | 7.9 × 10−4 | 38.2(5.1-286.1) | 1.3 × 10−8 |
| 48619855 | rs116855232 | p.Arg139Cys | T | C | 8 | 320 | 0.012 | 0 | 633 | 0 | NA | 1.8 × 10−4 |
| 48611918c | rs746071566 | p.Gly17_Val18dupd | AGGAGTCGGAGTC | AGGAGTC | 5 | 304 | 0.008 | 2 | 630 | 0.002 | 5.2(1.0-26.6) | .04 |
| 48611979 | rs768057637 | p.Lys33Glu | G | A | 1 | 327 | 0.002 | 0 | 633 | 0 | NA | .34 |
| 48615121 | 13:48615121 | p.Val75Gly | G | T | 1 | 327 | 0.002 | 0 | 633 | 0 | NA | .34 |
| 48611961 | rs777311140 | p.Cys28GlyfsTer28 | CGCGG | C | 0 | 328 | 0 | 1 | 632 | 7.9 × 10−4 | NA | >.99 |
| 48611883 | 13:48611883 | p.Met1? | C | A | 0 | 328 | 0 | 1 | 632 | 7.9 × 10−4 | NA | >.99 |

Table Caption: Abbreviations: NA, not applicable; SNP, single-nucleotide polymorphism. a Calculated as (No. of variant alleles in population)/(2 × No. of participants). No affected or unaffected patients were homozygous for NUDT15 variant alleles. b Calculated using the Fisher exact test. A genome-wide threshold of P < 5 × 10−8 was considered significant. c This site is multiallelic and both of these variants occur at the same chromosome position (48611918). Nineteen patients were heterozygous for p.Gly17_Val18del, 5 affected patients were heterozygous for p.Gly17_Val18dup, and 304 affected patients were considered reference homozygous (328 patients in total). d Previously annotated as p.Val18_Val19insGlyVal.

### Exploratory Analyses

Sequence data for *NUDT15* were examined for the presence of all coding variants either previously associated with TIM^8,9,10^ or identified in gnomAD^25^ and predicted as deleterious using the Protein Variation Effect Analyzer^26^ ([Table 2](#joi190011t2) and eFigure 5 in the [Supplement](#note-JOI190011-1-s)). However, 4 (p.Lys33Glu, p.Val75Gly, p.Cys28GlyfsTer28, and p.Met1?) of the 7 *NUDT15* variants were each only found in a single individual. Therefore, only variants either meeting genome-wide association in this analysis (p.Gly17_Val18del) or previously associated with TIM in other analyses (p.Arg139Cys and p.Gly17_Val18dup) were included for subsequent exploratory analyses.

Overall, 9.5% (31/328) of the non-Finnish European TIM discovery cohort carry any of the 3 *NUDT15* coding variants compared with 0.5% (3/633) of unaffected patients (OR, 20.9 [95% CI, 6.4 to 68.6], *P* = 1.5 × 10^−12^). The association with these *NUDT15* variants was enriched in patients affected with early-onset vs late-onset TIM (OR, 3.3 [95% CI, 1.6 to 6.9], *P* < .001).

Of the included patients in the EWAS analysis (discovery cohort), 75% (717/961) had TPMT activity levels available for analysis. All 10 patients with absent TPMT activity and 73% of patients (80/109) with low TPMT activity carried variant *TPMT* haplotypes (eFigure 6 and eTables 7-9 in the [Supplement](#note-JOI190011-1-s)). Overall, 4.9% (16/328) of affected patients and 0.2% (1/633) of unaffected patients had 2 TIM-associated *TPMT* variant haplotypes.

#### Genotype-Phenotype Analyses

Among all affected patients in the EWAS analysis, the median time to TIM was 15 weeks (IQR, 6-41 weeks) and 34% (111/328) experienced early-onset TIM. Of note, 18% (59/328) presented with an opportunistic infection, 23% (77/328) were admitted to a hospital with a median length of stay of 6 days (IQR, 2-9 days), and 9% (31/328) required granulocyte colony–stimulating factor rescue therapy.

The median time to TIM was shorter in affected patients who carried *NUDT15* variants compared with affected patients without risk variants (7.7 weeks [IQR, 5.7-20.0 weeks] vs 20.0 weeks [IQR, 7.6-48.3 weeks], respectively; *P* = .009) and in those who carried double *TPMT* variants (6.1 weeks [IQR, 4.2-7.6 weeks] vs 20.0 weeks [IQR, 7.6-48.3 weeks], respectively; *P* = .002).

The median time to TIM was shortest in patients with both *TPMT* and *NUDT15* variants compared with affected patients without risk variants (2.5 weeks [IQR, 1.5-4.1 weeks] vs 20.0 weeks [IQR, 7.6-48.3 weeks], respectively, *P* < .001; [Figure 3](#joi190011f3) and eFigure 6 in the [Supplement](#note-JOI190011-1-s)). No difference in time to TIM was seen in patients carrying 1 variant *TPMT* haplotype compared with affected patients without risk variants (13.9 weeks [IQR, 5.9-40.4 weeks] vs 20.0 weeks [IQR, 7.6-48.3 weeks], respectively, *P* = .14).

![Figure 3. ](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2f41/6439872/63a48cb89725/jama-e190709-g003.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=6439872_jama-e190709-g003.jpg)

Data points are each represented by a dot. The lower and upper boundaries of the box correspond to the first and third quartiles. The line within the box represents the median. The upper whisker extends from the upper boundary of the box to the largest value no further than 1.5 × the interquartile range (IQR). The lower whisker extends from the lower boundary of the box to the lowest value, at most no further than 1.5 × the IQR. The time to thiopurine-induced myelosuppression (TIM) was calculated using the following formula: (time to TIM in weeks) = date meeting entry criteria for TIM minus start date of highest dose prior to TIM). The median values and IQRs are provided to facilitate interpretation of time to TIM. One TIM case carried 2 NUDT15 variants (rs746071566 [p.Gly17_Val18dup] and rs116855232 [p.Arg139Cys]); however, it was unknown if this represented a compound heterozygote or a heterozygote (*2 NUDT15 haplotype). For the purpose of the analysis, this patient was grouped with NUDT15 heterozygotes and annotated as NUDT15var/*. One TIM case was TPMTvar/var and NUDT15var/ref; for the purpose of the analysis, this patient was grouped with 5 others who carried single NUDT15 and TPMT variants (TPMTvar/ref and NUDT15var/ref). Compared with the leftmost group, the Mann-Whitney P values for the differences in time to onset of TIM were .14, .009, .002, and <.001, respectively. Ref indicates reference genotype or haplotype; var, variant.

Patients carrying either *NUDT15* or *TPMT* and those with variants for both genes developed lower neutrophil counts than affected patients without these variants (median, 0.8 × 10^9^/L [IQR, 0.4-1.1 × 10^9^/L] vs median, 1.0 × 10^9^/L [IQR, 0.7-1.2 × 10^9^/L], respectively, *P* < .001), were more likely to be admitted to the hospital (40% [39/97] vs 17% [38/231], *P* < .001), and were more likely to receive granulocyte colony–stimulating factor rescue therapy (20% [19/97] vs 5.2% [12/231], *P* < .001; eTables 10-11 in the [Supplement](#note-JOI190011-1-s)).

The success of another challenge with thiopurine according to genotype also was explored. Among the 51% (167/328) of affected patients rechallenged, 57% (95/167) were able to tolerate a lower dose (median successful rechallenge dose, 1.2 mg/kg [IQR, 0.9-1.5 mg/kg]). Neither weight-adjusted dose, type of thiopurine, patient age, *TPMT* genotype, nor *NUDT15* genotype were associated with subsequent tolerance after rechallenge (eTable 12 in the [Supplement](#note-JOI190011-1-s)).

#### Multivariable Logistic Regression Analysis

In a multivariable logistic regression model analysis, the odds of TIM among patients with any of 3 coding variants in *NUDT15* (OR, 27.3 [95% CI, 9.3 to 116.7]; *P* = 1.1 × 10^−7^) and in *TPMT* (in heterozygotes: OR, 2.2 [95% CI, 1.4 to 3.3], *P* = 3.5 × 10^−4^; in homozygotes: OR, 53.4 [95% CI, 10.4 to 980.1], *P* = 1.5 × 10^−4^) were independent of thiopurine-weight adjusted dose (OR, 2.2 [95% CI, 1.8 to 2.8], *P* = 5.3 × 10^−11^; [Table 3](#joi190011t3)).

| Variable | Odds Ratio (95% CI) | P Valueb |
| --- | --- | --- |
| Weight-adjusted thiopurine dosec | 2.2 (1.8 to 2.8) | 5.3 × 10−11 |
| NUDT15 genotype |   |   |
| Reference genotype/reference genotype | 1 [Reference] |   |
| Variant genotype/*d | 27.3 (9.3 to 116.7) | 1.1 × 10−7 |
| TPMT haplotype |   |   |
| Reference haplotype/reference haplotype | 1 [Reference] |   |
| Reference haplotype/variant haplotype | 2.2 (1.4 to 3.3) | 3.5 × 10−4 |
| Variant haplotype/variant haplotype | 53.4 (10.4 to 980.1) | 1.5 × 10−4 |

Table Caption: a There were 42 observations missing. b From logistic regression with all 3 variables included. P < .05 deemed statistically significant. c For every 1 mg/kg increase in azathioprine equivalent dose. Represents the maximum azathioprine equivalent dose prior to thiopurine-induced myelosuppression adjusted for weight (milligrams per kilograms). d Carriage of 1 or more of 3 NUDT15 variants: rs746071566 [p.Gly17_Val18del], rs746071566 [p.Gly17_Val18dup], and rs116855232 [p.Arg139Cys]. One patient with TIM possessed 2 NUDT15 variants (rs746071566 [p.Gly17_Val18dup] and rs116855232 [p.Arg139Cys]); however, it was not possible to ascertain if this represented a compound heterozygote or 2 variants on the same strand (*2 NUDT15 haplotype). For the purpose of the analysis, this case was considered as a single NUDT15 variant carrier (NUDT15 var/*).

#### Estimated Potential Clinical Effectiveness

For *NUDT15,* the estimated number of patients needed to genotype to prevent 1 patient from developing TIM was 95 patients (95% CI, 62-143 patients). For every 10 000 patients genotyped, 164 would test positive for a *NUDT15* variant, and of these patients, 105 would have developed TIM if they had not received an alternative treatment (positive predictive value, 64% [95% CI, 43%-100%]; eMethods in the [Supplement](#note-JOI190011-1-s)). Genotyping 10 000 patients for *NUDT15* would prevent 105 cases of TIM, which is 95 patients genotyped for every case prevented. The number needed to genotype assumed a cumulative incidence for TIM of 7% (95% CI, 6%-8%) based on a meta-analysis of 8302 patients,^1^ a drug avoidance strategy in *NUDT15* variant carriers, a population carriage frequency of 1.6% (95% CI, 1.5%-1.8%), and ORs derived from bootstrapping the affected and unaffected population (sampling with replacement to estimate the variability of the OR). If a dose reduction strategy was used in *NUDT15* variant carriers instead, thus reducing risk of TIM to that of patients with the reference genotype (absolute risk, 6% [95% CI, 5%-7%]), the number needed to genotype would be 105 patients (95% CI, 65-168 patients).

For *TPMT,* the estimated number needed to genotype was 123 patients (95% CI, 75-235 patients). For every 10 000 patients genotyped, 996 would test positive for a *TPMT* variant and need to receive an alternative therapy to prevent TIM in 81 patients (95% CI, 43-133 patients). Genotyping 10 000 patients for *TPMT* would prevent 81 cases of TIM, which is 123 genotyped for every case prevented. This assumed the following for patients carrying 2 *TPMT* variant haplotypes: drug avoidance, a population carrier frequency^30^ of 0.26% (95% CI, 0.19%-0.34%), and an OR of 53.4 (95% CI, 10.4-980.1). For patients carrying 1 *TPMT* haplotype, this assumed the following: a thiopurine dose reduction, a population carrier frequency of 9.7% (95% CI, 8.4%-11.0%), and an OR of 2.2 (95% CI, 1.4-3.3).

In the wider cohort of 398 affected patients who were adjudicated and when including patients of non-European ancestry (who had been excluded from the GWAS and EWAS analyses), carriage of *NUDT15* variants was more frequent than in patients of non-Finnish European ancestry (100% [4/4] for South Asian patients vs 9% [31/328] for non-Finnish European patients, *P* = 1.1 × 10^−4^; and 56% [23/41] for East Asian patients vs 9% [31/328] non-Finnish European patients, *P* = 2.0 × 10^−11^; eTable 13 in the [Supplement](#note-JOI190011-1-s)).

Estimates of the rate of carrying 1 or more *NUDT15* risk alleles in the general population using the gnomAD reference database ranged from 0.7% in patients of African ancestry to 29.2% in patients of East Asian ancestry (eTable 14 in the [Supplement](#note-JOI190011-1-s)).

## Discussion

In this case-control study involving both GWAS and EWAS analyses, an association between an *NUDT15* variant (p.Gly17_Val18del) and TIM has been identified and replicated in independent cohorts of patients of non-Finnish European ancestry. In total, 3 *NUDT15* coding variants, including p.Gly17_Val18del, were identified and collectively associated with TIM independent of *TPMT* genotype and thiopurine dose. Patients with variants of either *NUDT15* or *TPMT,* or among those with variants of both genes, had a faster onset of TIM, more severe TIM, and had a greater need for granulocyte colony–stimulating factor rescue therapy.

To our knowledge, this is the first study to describe the association of an *NUDT15* variant with TIM in patients of European ancestry at genome-wide significance. This extends previous work by Moriyama et al^10^ that first described this p.Gly17_Val18del variant in 2 pediatric patients with acute lymphoblastic leukemia and TIM; one of whom was of European, and the second, of African ancestry.

The p.Arg139Cys variant has previously been associated with TIM in a North American IBD cohort study for which the minor allele frequency reported was 2.7% in affected patients and 0.3% in unaffected patients (OR, 9.50; *P* = 4.6 × 10^−4^).^9^ In contrast, to our knowledge, the p.Gly17_Val18dup variant had only been reported in cohorts of East Asian ancestry.^8^

*NUDT15* is hypothesized to hydrolyse nucleoside triphosphate active metabolites (6-thio-dGTP, 6-thio-GTP, and dGTP) thus preventing their incorporation into DNA where they would otherwise lead to futile mismatch repair and apoptosis.^8,9,31^ Functional experiments confirm that *NUDT15* variants result in lower enzymatic activity, leading to higher levels of thiopurine active metabolites and a greater risk of TIM.^8,9,10,31^ The p.Gly17_Val18dup variant reduces *NUDT15* activity to approximately 15% of normal activity, whereas p.Gly17_Val18del and p.Arg139Cys are nearly void of enzyme activity, suggesting that patients with these variants may be particularly sensitive to thiopurines.^8,10^

Given the widespread use of the thiopurines, these findings may have ramifications beyond the management of IBD in patients of European ancestry. Although *NUDT15* variants were first associated with TIM in East Asian patients with IBD,^9^ this phenomenon has now been demonstrated in oncology and other immune-mediated diseases^32,33^ as well as in other populations.^9,10,11,12,13,14^ For population stratification reasons, patients of non-European ancestry were excluded from the genetic analyses of this study. However, it is interesting to note in the absence of *TPMT* variants, the frequency of variant *NUDT15* haplotypes is 29.2% in populations of East Asian ancestry compared with 20.7% in Latin American populations, 13.4% in South Asian populations, and 1.6% in non-Finnish European populations.^25^

As expected, in the wider cohort of affected patients who were adjudicated, patients of non-European descent demonstrated a higher carriage frequency of *NUDT15* variants and a lower carriage frequency of *TPMT* variants. If replicated in additional studies, these findings suggest that *NUDT15* testing may be considered prior to thiopurine therapy irrespective of the ethnic background of the patient.

The positive predictive value of *NUDT15* genotyping estimated in this study together with the recent development of alternative but more expensive therapies suggests potential clinical utility of pretreatment testing and drug avoidance in genetically at-risk patients. Recommendations regarding pretreatment *NUDT15* genotyping based on data from East Asian populations are under review by the Clinical Pharmacogenetics Implementation Consortium.^27^ Our data suggest that pretreatment sequencing of the *NUDT15* gene, including the p.Gly17_Val18del deletion, may also be considered in patients of European ancestry. However, this will not obviate the requirement for regular monitoring with blood tests for the duration of treatment among patients deemed at low risk of TIM.

The estimated number of patients needed to genotype for *NUDT15* is 95, similar to the number needed to genotype reported herein and by others^34^ for *TPMT* (123 and 100, respectively). However, further validation studies including a cost-effectiveness analysis should be conducted prior to implementation of pretreatment *NUDT15* genotyping.

### Limitations

This study has several limitations. First, inclusion was restricted to patients with IBD of non-Finnish European ancestry. Further research is required to evaluate the association of these variants with TIM in other ancestries and disease groups. Second, the replication cohort was not exclusively recruited from independent sites. The central site recruited affected and unaffected patients to the discovery cohort and then additional patients to the replication cohorts.

Third, in keeping with all case-control studies, the data are likely to be susceptible to recall bias and there was greater recruitment of more severely affected patients. We estimate that our affected patients represent 5% of the total eligible patients with IBD and an episode of TIM. This is based on a UK IBD prevalence of 388 patients per 100 000 population,^35^ a thiopurine exposure rate of 31%,^36^ and a rate of TIM of 7%.^1^ This recall bias might explain the IBD phenotype differences observed between cases and controls and an overestimate of the risk associated with *NUDT15* variants and TIM.

Fourth, 4.9% (16/328) of affected and 0.2% (1/633) of unaffected patients had 2 of the known TIM-associated *TPMT* variant haplotypes despite the recommended practice of pretreatment measurement of TPMT enzyme activity and thiopurine avoidance in patients deficient of TPMT enzyme activity. These patients arguably should not have received treatment with a thiopurine, regardless of the presence of *NUDT15* variants.

Fifth, the proposed mitigation strategy of thiopurine avoidance rather than dose reduction in patients with *NUDT15* coding variants may be overly cautious. Previous studies in patients of East Asian ancestry have shown that even patients with 2 low-functioning *NUDT15* alleles may successfully tolerate a dose reduction of thiopurine by 90%.^8,31,33^ Likewise, in *NUDT15* knockout mice models, accumulation of thiopurine metabolites was noted to be in an mercaptopurine dose–related fashion, suggesting that dose reduction might be an effective strategy.^31^ However, as discussed above, not all variants affect *NUDT15* enzymatic function to the same extent and the magnitude of the deleterious effect of individual variants may differ across ethnic groups.^37^ Furthermore, it is unknown whether such a marked dose reduction would compromise the therapeutic effect of thiopurines in patients with IBD. In our study of patients of non-Finnish European ancestry, almost 50% of patients with a single variant did not tolerate a rechallenge with thiopurine at a lower dose. These arguments may justify the use of alternative, more expensive therapies in this small group of patients at high risk of TIM. However, further data are needed to explore whether thiopurine dose reduction with enhanced monitoring or drug avoidance is the safer, less expensive, and more clinically effective strategy.

## Conclusions

Among patients of European ancestry with IBD, variants in *NUDT15* were associated with increased risk of TIM. These findings suggest that *NUDT15* genotyping may be considered prior to initiation of thiopurine therapy; however, further study including additional validation in independent cohorts is required.

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Gisbert JP, Gomollón F. Thiopurine-induced myelotoxicity in patients with inflammatory bowel disease: a review. Am J Gastroenterol. 2008;103(7):1783-1800. doi: 10.1111/j.1572-0241.2008.01848.x  [DOI](https://doi.org/10.1111/j.1572-0241.2008.01848.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18557712/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Gastroenterol&title=Thiopurine-induced%20myelotoxicity%20in%20patients%20with%20inflammatory%20bowel%20disease:%20a%20review&volume=103&issue=7&publication_year=2008&pages=1783-1800&pmid=18557712&doi=10.1111/j.1572-0241.2008.01848.x&)

2. Chaparro M, Ordás I, Cabré E, et al. Safety of thiopurine therapy in inflammatory bowel disease: long-term follow-up study of 3931 patients. Inflamm Bowel Dis. 2013;19(7):1404-1410. doi: 10.1097/MIB.0b013e318281f28f  [DOI](https://doi.org/10.1097/MIB.0b013e318281f28f) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23665964/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Inflamm%20Bowel%20Dis&title=Safety%20of%20thiopurine%20therapy%20in%20inflammatory%20bowel%20disease:%20long-term%20follow-up%20study%20of%203931%20patients&volume=19&issue=7&publication_year=2013&pages=1404-1410&pmid=23665964&doi=10.1097/MIB.0b013e318281f28f&)

3. Gearry RB, Barclay ML. Azathioprine and 6-mercaptopurine pharmacogenetics and metabolite monitoring in inflammatory bowel disease. J Gastroenterol Hepatol. 2005;20(8):1149-1157. doi: 10.1111/j.1440-1746.2005.03832.x  [DOI](https://doi.org/10.1111/j.1440-1746.2005.03832.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16048561/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Gastroenterol%20Hepatol&title=Azathioprine%20and%206-mercaptopurine%20pharmacogenetics%20and%20metabolite%20monitoring%20in%20inflammatory%20bowel%20disease&volume=20&issue=8&publication_year=2005&pages=1149-1157&pmid=16048561&doi=10.1111/j.1440-1746.2005.03832.x&)

4. Teml A, Schaeffeler E, Herrlinger KR, Klotz U, Schwab M. Thiopurine treatment in inflammatory bowel disease: clinical pharmacology and implication of pharmacogenetically guided dosing. Clin Pharmacokinet. 2007;46(3):187-208. doi: 10.2165/00003088-200746030-00001  [DOI](https://doi.org/10.2165/00003088-200746030-00001) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17328579/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacokinet&title=Thiopurine%20treatment%20in%20inflammatory%20bowel%20disease:%20clinical%20pharmacology%20and%20implication%20of%20pharmacogenetically%20guided%20dosing&volume=46&issue=3&publication_year=2007&pages=187-208&pmid=17328579&doi=10.2165/00003088-200746030-00001&)

5. US Food and Drug Administration . Approved drug products with therapeutic equivalence evaluations (Orange Book). https://www.fda.gov/Drugs/InformationOnDrugs/ucm129662.htm. Accessed March 1, 2018.  [https://www.fda.gov/Drugs/InformationOnDrugs/ucm129662.htm](https://www.fda.gov/Drugs/InformationOnDrugs/ucm129662.htm)

6. Colombel JF, Ferrari N, Debuysere H, et al. Genotypic analysis of thiopurine S-methyltransferase in patients with Crohn’s disease and severe myelosuppression during azathioprine therapy. Gastroenterology. 2000;118(6):1025-1030. doi: 10.1016/S0016-5085(00)70354-4  [DOI](https://doi.org/10.1016/S0016-5085(00)70354-4) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10833476/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gastroenterology&title=Genotypic%20analysis%20of%20thiopurine%20S-methyltransferase%20in%20patients%20with%20Crohn%E2%80%99s%20disease%20and%20severe%20myelosuppression%20during%20azathioprine%20therapy&volume=118&issue=6&publication_year=2000&pages=1025-1030&pmid=10833476&doi=10.1016/S0016-5085(00)70354-4&)

7. Dewit O, Moreels T, Baert F, et al. ; Belgian Inflammatory Bowel Disease Research Group (BIRD) . Limitations of extensive TPMT genotyping in the management of azathioprine-induced myelosuppression in IBD patients. Clin Biochem. 2011;44(13):1062-1066. doi: 10.1016/j.clinbiochem.2011.06.079  [DOI](https://doi.org/10.1016/j.clinbiochem.2011.06.079) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21723857/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Biochem&title=Limitations%20of%20extensive%20TPMT%20genotyping%20in%20the%20management%20of%20azathioprine-induced%20myelosuppression%20in%20IBD%20patients&volume=44&issue=13&publication_year=2011&pages=1062-1066&pmid=21723857&doi=10.1016/j.clinbiochem.2011.06.079&)

8. Moriyama T, Nishii R, Perez-Andreu V, et al. NUDT15 polymorphisms alter thiopurine metabolism and hematopoietic toxicity. Nat Genet. 2016;48(4):367-373. doi: 10.1038/ng.3508  [DOI](https://doi.org/10.1038/ng.3508) | [PMC free article](/articles/PMC5029084/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26878724/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=NUDT15%20polymorphisms%20alter%20thiopurine%20metabolism%20and%20hematopoietic%20toxicity&volume=48&issue=4&publication_year=2016&pages=367-373&pmid=26878724&doi=10.1038/ng.3508&)

9. Yang S-K, Hong M, Baek J, et al. A common missense variant in NUDT15 confers susceptibility to thiopurine-induced leukopenia. Nat Genet. 2014;46(9):1017-1020. doi: 10.1038/ng.3060  [DOI](https://doi.org/10.1038/ng.3060) | [PMC free article](/articles/PMC4999337/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25108385/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=A%20common%20missense%20variant%20in%20NUDT15%20confers%20susceptibility%20to%20thiopurine-induced%20leukopenia&volume=46&issue=9&publication_year=2014&pages=1017-1020&pmid=25108385&doi=10.1038/ng.3060&)

10. Moriyama T, Yang YL, Nishii R, et al. Novel variants in NUDT15 and thiopurine intolerance in children with acute lymphoblastic leukemia from diverse ancestry. Blood. 2017;130(10):1209-1212. doi: 10.1182/blood-2017-05-782383  [DOI](https://doi.org/10.1182/blood-2017-05-782383) | [PMC free article](/articles/PMC5606007/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28659275/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Novel%20variants%20in%20NUDT15%20and%20thiopurine%20intolerance%20in%20children%20with%20acute%20lymphoblastic%20leukemia%20from%20diverse%20ancestry&volume=130&issue=10&publication_year=2017&pages=1209-1212&pmid=28659275&doi=10.1182/blood-2017-05-782383&)

11. Kakuta Y, Naito T, Onodera M, et al. NUDT15 R139C causes thiopurine-induced early severe hair loss and leukopenia in Japanese patients with IBD. Pharmacogenomics J. 2016;16(3):280-285. doi: 10.1038/tpj.2015.43  [DOI](https://doi.org/10.1038/tpj.2015.43) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26076924/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=NUDT15%20R139C%20causes%20thiopurine-induced%20early%20severe%20hair%20loss%20and%20leukopenia%20in%20Japanese%20patients%20with%20IBD&volume=16&issue=3&publication_year=2016&pages=280-285&pmid=26076924&doi=10.1038/tpj.2015.43&)

12. Zhu X, Wang X-D, Chao K, et al. NUDT15 polymorphisms are better than thiopurine S-methyltransferase as predictor of risk for thiopurine-induced leukopenia in Chinese patients with Crohn’s disease. Aliment Pharmacol Ther. 2016;44(9):967-975. doi: 10.1111/apt.13796  [DOI](https://doi.org/10.1111/apt.13796) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27604507/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Aliment%20Pharmacol%20Ther&title=NUDT15%20polymorphisms%20are%20better%20than%20thiopurine%20S-methyltransferase%20as%20predictor%20of%20risk%20for%20thiopurine-induced%20leukopenia%20in%20Chinese%20patients%20with%20Crohn%E2%80%99s%20disease&volume=44&issue=9&publication_year=2016&pages=967-975&pmid=27604507&doi=10.1111/apt.13796&)

13. Soler AM, Olano N, Méndez Y, et al. TPMT and NUDT15 genes are both related to mercaptopurine intolerance in acute lymphoblastic leukaemia patients from Uruguay. Br J Haematol. 2018;181(2):252-255. doi: 10.1111/bjh.14532  [DOI](https://doi.org/10.1111/bjh.14532) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28146264/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Haematol&title=TPMT%20and%20NUDT15%20genes%20are%20both%20related%20to%20mercaptopurine%20intolerance%20in%20acute%20lymphoblastic%20leukaemia%20patients%20from%20Uruguay&volume=181&issue=2&publication_year=2018&pages=252-255&pmid=28146264&doi=10.1111/bjh.14532&)

14. Shah SAV, Paradkar M, Desai D, Ashavaid TF. Nucleoside diphosphate-linked moiety X-type motif 15 C415T variant as a predictor for thiopurine-induced toxicity in Indian patients. J Gastroenterol Hepatol. 2017;32(3):620-624. doi: 10.1111/jgh.13494  [DOI](https://doi.org/10.1111/jgh.13494) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27416873/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Gastroenterol%20Hepatol&title=Nucleoside%20diphosphate-linked%20moiety%20X-type%20motif%2015%20C415T%20variant%20as%20a%20predictor%20for%20thiopurine-induced%20toxicity%20in%20Indian%20patients&volume=32&issue=3&publication_year=2017&pages=620-624&pmid=27416873&doi=10.1111/jgh.13494&)

15. Heap GA, So K, Weedon M, et al. Clinical features and HLA association of 5-aminosalicylate (5-ASA)-induced nephrotoxicity in inflammatory bowel disease. J Crohn’s Colitis. 2016;10(2)149-158. doi: 10.1093/ecco-jcc/jjv219  [DOI](https://doi.org/10.1093/ecco-jcc/jjv219) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26619893/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Crohn%E2%80%99s%20Colitis&title=Clinical%20features%20and%20HLA%20association%20of%205-aminosalicylate%20(5-ASA)-induced%20nephrotoxicity%20in%20inflammatory%20bowel%20disease&volume=10&issue=2&publication_year=2016&pages=149-158&pmid=26619893&doi=10.1093/ecco-jcc/jjv219&)

16. Heap GA, Weedon MN, Bewshea CM, et al. ; International Serious Adverse Events Consortium; IBD Pharmacogenetics Study Group . HLA-DQA1-HLA-DRB1 variants confer susceptibility to pancreatitis induced by thiopurine immunosuppressants. Nat Genet. 2014;46(10):1131-1134. doi: 10.1038/ng.3093  [DOI](https://doi.org/10.1038/ng.3093) | [PMC free article](/articles/PMC6379053/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25217962/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=HLA-DQA1-HLA-DRB1%20variants%20confer%20susceptibility%20to%20pancreatitis%20induced%20by%20thiopurine%20immunosuppressants&volume=46&issue=10&publication_year=2014&pages=1131-1134&pmid=25217962&doi=10.1038/ng.3093&)

17. Silverberg MS, Satsangi J, Ahmad T, et al. Toward an integrated clinical, molecular and serological classification of inflammatory bowel disease: report of a Working Party of the 2005 Montreal World Congress of Gastroenterology. Can J Gastroenterol. 2005;19(suppl A):5A-36A. doi: 10.1155/2005/269076  [DOI](https://doi.org/10.1155/2005/269076) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16151544/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Can%20J%20Gastroenterol&title=Toward%20an%20integrated%20clinical,%20molecular%20and%20serological%20classification%20of%20inflammatory%20bowel%20disease:%20report%20of%20a%20Working%20Party%20of%20the%202005%20Montreal%20World%20Congress%20of%20Gastroenterology&volume=19&issue=suppl%20A&publication_year=2005&pages=5A-36A&pmid=16151544&doi=10.1155/2005/269076&)

18. Gallagher RM, Kirkham JJ, Mason JR, et al. Development and inter-rater reliability of the Liverpool adverse drug reaction causality assessment tool. PLoS One. 2011;6(12):e28096. doi: 10.1371/journal.pone.0028096  [DOI](https://doi.org/10.1371/journal.pone.0028096) | [PMC free article](/articles/PMC3237416/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22194808/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=Development%20and%20inter-rater%20reliability%20of%20the%20Liverpool%20adverse%20drug%20reaction%20causality%20assessment%20tool&volume=6&issue=12&publication_year=2011&pages=e28096&pmid=22194808&doi=10.1371/journal.pone.0028096&)

19. Yang J, Lee SH, Goddard ME, Visscher PM. GCTA: a tool for genome-wide complex trait analysis. Am J Hum Genet. 2011;88(1):76-82. doi: 10.1016/j.ajhg.2010.11.011  [DOI](https://doi.org/10.1016/j.ajhg.2010.11.011) | [PMC free article](/articles/PMC3014363/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21167468/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=GCTA:%20a%20tool%20for%20genome-wide%20complex%20trait%20analysis&volume=88&issue=1&publication_year=2011&pages=76-82&pmid=21167468&doi=10.1016/j.ajhg.2010.11.011&)

20. Auton A, Brooks LD, Durbin RM, et al. ; 1000 Genomes Project Consortium . A global reference for human genetic variation. Nature. 2015;526(7571):68-74. doi: 10.1038/nature15393  [DOI](https://doi.org/10.1038/nature15393) | [PMC free article](/articles/PMC4750478/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26432245/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=A%20global%20reference%20for%20human%20genetic%20variation&volume=526&issue=7571&publication_year=2015&pages=68-74&pmid=26432245&doi=10.1038/nature15393&)

21. Loh P-R, Danecek P, Palamara PF, et al. Reference-based phasing using the Haplotype Reference Consortium panel. Nat Genet. 2016;48(11):1443-1448. doi: 10.1038/ng.3679  [DOI](https://doi.org/10.1038/ng.3679) | [PMC free article](/articles/PMC5096458/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27694958/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Reference-based%20phasing%20using%20the%20Haplotype%20Reference%20Consortium%20panel&volume=48&issue=11&publication_year=2016&pages=1443-1448&pmid=27694958&doi=10.1038/ng.3679&)

22. Durbin R. Efficient haplotype matching and storage using the positional Burrows-Wheeler transform (PBWT). Bioinformatics. 2014;30(9):1266-1272. doi: 10.1093/bioinformatics/btu014  [DOI](https://doi.org/10.1093/bioinformatics/btu014) | [PMC free article](/articles/PMC3998136/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24413527/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=Efficient%20haplotype%20matching%20and%20storage%20using%20the%20positional%20Burrows-Wheeler%20transform%20(PBWT)&volume=30&issue=9&publication_year=2014&pages=1266-1272&pmid=24413527&doi=10.1093/bioinformatics/btu014&)

23. Li H, Durbin R. Fast and accurate short read alignment with Burrows-Wheeler transform. Bioinformatics. 2009;25(14):1754-1760. doi: 10.1093/bioinformatics/btp324  [DOI](https://doi.org/10.1093/bioinformatics/btp324) | [PMC free article](/articles/PMC2705234/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19451168/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=Fast%20and%20accurate%20short%20read%20alignment%20with%20Burrows-Wheeler%20transform&volume=25&issue=14&publication_year=2009&pages=1754-1760&pmid=19451168&doi=10.1093/bioinformatics/btp324&)

24. Ionita-Laza I, Lee S, Makarov V, Buxbaum JD, Lin X. Sequence kernel association tests for the combined effect of rare and common variants. Am J Hum Genet. 2013;92(6):841-853. doi: 10.1016/j.ajhg.2013.04.015  [DOI](https://doi.org/10.1016/j.ajhg.2013.04.015) | [PMC free article](/articles/PMC3675243/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23684009/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=Sequence%20kernel%20association%20tests%20for%20the%20combined%20effect%20of%20rare%20and%20common%20variants&volume=92&issue=6&publication_year=2013&pages=841-853&pmid=23684009&doi=10.1016/j.ajhg.2013.04.015&)

25. Lek M, Karczewski KJ, Minikel EV, et al. ; Exome Aggregation Consortium . Analysis of protein-coding genetic variation in 60,706 humans. Nature. 2016;536(7616):285-291. doi: 10.1038/nature19057  [DOI](https://doi.org/10.1038/nature19057) | [PMC free article](/articles/PMC5018207/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27535533/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Analysis%20of%20protein-coding%20genetic%20variation%20in%2060,706%20humans&volume=536&issue=7616&publication_year=2016&pages=285-291&pmid=27535533&doi=10.1038/nature19057&)

26. Choi Y, Sims GE, Murphy S, Miller JR, Chan AP. Predicting the functional effect of amino acid substitutions and indels. PLoS One. 2012;7(10):e46688. doi: 10.1371/journal.pone.0046688  [DOI](https://doi.org/10.1371/journal.pone.0046688) | [PMC free article](/articles/PMC3466303/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23056405/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=Predicting%20the%20functional%20effect%20of%20amino%20acid%20substitutions%20and%20indels&volume=7&issue=10&publication_year=2012&pages=e46688&pmid=23056405&doi=10.1371/journal.pone.0046688&)

27. Clinical Pharmacogenetics Implementation Consortium . Clinical Pharmacogenetics Implementation Consortium website. https://cpicpgx.org/. Accessed January 25, 2019.  [https://cpicpgx.org/](https://cpicpgx.org/)

28. Tonk ECM, Gurwitz D, Maitland-van der Zee A-H, Janssens ACJW. Assessment of pharmacogenetic tests: presenting measures of clinical validity and potential population impact in association studies. Pharmacogenomics J. 2017;17(4):386-392. doi: 10.1038/tpj.2016.34  [DOI](https://doi.org/10.1038/tpj.2016.34) | [PMC free article](/articles/PMC5549182/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27168098/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=Assessment%20of%20pharmacogenetic%20tests:%20presenting%20measures%20of%20clinical%20validity%20and%20potential%20population%20impact%20in%20association%20studies&volume=17&issue=4&publication_year=2017&pages=386-392&pmid=27168098&doi=10.1038/tpj.2016.34&)

29. de Graaff LC, van Schaik RH, van Gelder T. A clinical approach to pharmacogenetics. Neth J Med. 2013;71(3):145-152.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/23712814/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neth%20J%20Med&title=A%20clinical%20approach%20to%20pharmacogenetics&volume=71&issue=3&publication_year=2013&pages=145-152&pmid=23712814&)

30. Lennard L, Cartwright CS, Wade R, Vora A. Thiopurine methyltransferase and treatment outcome in the UK acute lymphoblastic leukaemia trial ALL2003. Br J Haematol. 2015;170(4):550-558. doi: 10.1111/bjh.13469  [DOI](https://doi.org/10.1111/bjh.13469) | [PMC free article](/articles/PMC4687427/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25940902/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Haematol&title=Thiopurine%20methyltransferase%20and%20treatment%20outcome%20in%20the%20UK%20acute%20lymphoblastic%20leukaemia%20trial%20ALL2003&volume=170&issue=4&publication_year=2015&pages=550-558&pmid=25940902&doi=10.1111/bjh.13469&)

31. Nishii R, Moriyama T, Janke LJ, et al. Preclinical evaluation of NUDT15-guided thiopurine therapy and its effects on toxicity and antileukemic efficacy. Blood. 2018;131(22):2466-2474. doi: 10.1182/blood-2017-11-815506  [DOI](https://doi.org/10.1182/blood-2017-11-815506) | [PMC free article](/articles/PMC5981167/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29572377/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Preclinical%20evaluation%20of%20NUDT15-guided%20thiopurine%20therapy%20and%20its%20effects%20on%20toxicity%20and%20antileukemic%20efficacy&volume=131&issue=22&publication_year=2018&pages=2466-2474&pmid=29572377&doi=10.1182/blood-2017-11-815506&)

32. Moriyama T, Nishii R, Lin T-N, et al. The effects of inherited NUDT15 polymorphisms on thiopurine active metabolites in Japanese children with acute lymphoblastic leukemia. Pharmacogenet Genomics. 2017;27(6):236-239. doi: 10.1097/FPC.0000000000000282  [DOI](https://doi.org/10.1097/FPC.0000000000000282) | [PMC free article](/articles/PMC5510236/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28445187/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=The%20effects%20of%20inherited%20NUDT15%20polymorphisms%20on%20thiopurine%20active%20metabolites%20in%20Japanese%20children%20with%20acute%20lymphoblastic%20leukemia&volume=27&issue=6&publication_year=2017&pages=236-239&pmid=28445187&doi=10.1097/FPC.0000000000000282&)

33. Yang JJ, Landier W, Yang W, et al. Inherited NUDT15 variant is a genetic determinant of mercaptopurine intolerance in children with acute lymphoblastic leukemia. J Clin Oncol. 2015;33(11):1235-1242. doi: 10.1200/JCO.2014.59.4671  [DOI](https://doi.org/10.1200/JCO.2014.59.4671) | [PMC free article](/articles/PMC4375304/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25624441/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Inherited%20NUDT15%20variant%20is%20a%20genetic%20determinant%20of%20mercaptopurine%20intolerance%20in%20children%20with%20acute%20lymphoblastic%20leukemia&volume=33&issue=11&publication_year=2015&pages=1235-1242&pmid=25624441&doi=10.1200/JCO.2014.59.4671&)

34. Winter J, Walker A, Shapiro D, Gaffney D, Spooner RJ, Mills PR. Cost-effectiveness of thiopurine methyltransferase genotype screening in patients about to commence azathioprine therapy for treatment of inflammatory bowel disease. Aliment Pharmacol Ther. 2004;20(6):593-599. doi: 10.1111/j.1365-2036.2004.02124.x  [DOI](https://doi.org/10.1111/j.1365-2036.2004.02124.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15352906/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Aliment%20Pharmacol%20Ther&title=Cost-effectiveness%20of%20thiopurine%20methyltransferase%20genotype%20screening%20in%20patients%20about%20to%20commence%20azathioprine%20therapy%20for%20treatment%20of%20inflammatory%20bowel%20disease&volume=20&issue=6&publication_year=2004&pages=593-599&pmid=15352906&doi=10.1111/j.1365-2036.2004.02124.x&)

35. Rubin GP, Hungin AP, Kelly PJ, Ling J. Inflammatory bowel disease: epidemiology and management in an English general practice population. Aliment Pharmacol Ther. 2000;14(12):1553-1559. doi: 10.1046/j.1365-2036.2000.00886.x  [DOI](https://doi.org/10.1046/j.1365-2036.2000.00886.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11121902/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Aliment%20Pharmacol%20Ther&title=Inflammatory%20bowel%20disease:%20epidemiology%20and%20management%20in%20an%20English%20general%20practice%20population&volume=14&issue=12&publication_year=2000&pages=1553-1559&pmid=11121902&doi=10.1046/j.1365-2036.2000.00886.x&)

36. Kirchgesner J, Lemaitre M, Carrat F, Zureik M, Carbonnel F, Dray-Spira R. Risk of serious and opportunistic infections associated with treatment of inflammatory bowel diseases. Gastroenterology. 2018;155(2):337-346.e10. doi: 10.1053/j.gastro.2018.04.012  [DOI](https://doi.org/10.1053/j.gastro.2018.04.012) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29655835/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gastroenterology&title=Risk%20of%20serious%20and%20opportunistic%20infections%20associated%20with%20treatment%20of%20inflammatory%20bowel%20diseases&volume=155&issue=2&publication_year=2018&pages=337-346.e10&pmid=29655835&doi=10.1053/j.gastro.2018.04.012&)

37. Ye BD, McGovern DPB. Genetic variation in IBD: progress, clues to pathogenesis and possible clinical utility. Expert Rev Clin Immunol. 2016;12(10):1091-1107. doi: 10.1080/1744666X.2016.1184972  [DOI](https://doi.org/10.1080/1744666X.2016.1184972) | [PMC free article](/articles/PMC5083126/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27156530/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Expert%20Rev%20Clin%20Immunol&title=Genetic%20variation%20in%20IBD:%20progress,%20clues%20to%20pathogenesis%20and%20possible%20clinical%20utility&volume=12&issue=10&publication_year=2016&pages=1091-1107&pmid=27156530&doi=10.1080/1744666X.2016.1184972&)
